<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Pds5p Is Essential Chromosomal Protein Required Sister <br /> Chromatid Cohesion Condensation <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-15'  ></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br /> Theresa Hartman,* Kristen Stead,* Douglas Koshland,&#8225; Vincent Guacci* <br /> *Fox Chase <span id='am-63' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-65'  ></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Cancer" datatype="xsd:string"></span>Cancer</span> Center, Basic Science Division, Philadelphia, Pennsylvania 19111; &#8225;Carnegie Institution Washington, <br /> Department Embryology, Baltimore, Maryland 21210 <br />  <br />  <br />  <br />  <br /> Abstract. The PDS5 gene (precocious dissociation                           sequences bound Mcd1p. Furthermore, Pds5p local- <br /> sisters) identified genetic screen designed                       ization chromosomes dependent Mcd1p. Thus, <br /> identify genes important chromosome structure.                            Pds5p, like cohesin complex members, compo- <br /> PDS5 essential gene homologues                            nent molecular glue mediates sister chroma- <br /> from yeast humans. Pds5p function important                         tid cohesion. However, Mcd1p localization chromo- <br /> viability S phase mitosis localizes                       somes independent Pds5p, reflect <br /> chromosomes cell cycle window, en-                          differences roles cohesion. Finally, Pds5p is <br /> compasses times sister chromatid cohesion                            required condensation cohesion, which <br /> exists. Pds5p required maintain cohesion cen-                        confirms link processes revealed <br /> tromere proximal distal sequences. These proper-                          analysis Mcd1p (Guacci, V., D. Koshland, <br /> ties identical cohesion complex                      A. Strunnikov. 1997. Cell. 91:47&#8211;57). Therefore, the <br /> members Mcd1p/Scc1p, Smc1p, Smc3p, Scc3p/Irr1p                            link cohesion condensation general <br /> (Guacci, V., D. Koshland, A. Strunnikov. 1997. Cell.                      property yeast chromosomes. <br /> 91:47&#8211;57; Michaelis, C., R. Ciosk, K. Nasmyth. <br /> 1997. Cell. 91:35&#8211;45; Toth, A., R. Ciosk, F. Uhlmann, M.                      Key words: mitosis &#8226; chromosomes &#8226; cohesion &#8226; con- <br /> Galova, A. Schleiffer, K. Nasmyth. 1999. Genes                            densation &#8226; yeast <br /> Dev. 13:307&#8211;319). Pds5p binds centromeric arm <br />  <br />  <br />  <br /> Introduction <br /> After replication, duplicated chromatids (sister chro-                    rescent situ hybridization (FISH)1 indicated both <br /> matids) associated entire length. During                      mammals yeast, sister chromatids associated along <br /> mitosis, association cohesion sister chro-                     lengths time replication anaphase <br /> matids maintained sisters condense, attach                    (Wilson, 1925; Selig et al., 1992; Guacci et al., 1993, 1994). <br /> the mitotic spindle bipolar orientation, congress                    Mammalian yeast chromosomes undergo cell cy- <br /> to metaphase plate. At metaphase anaphase                          cle&#8211;dependent condensation (Wilson, 1925; Lawrence et <br /> transition, sister chromatid cohesion dissolved synchro-                   al., 1988; Trask et al., 1989; Guacci et al., 1994). The simi- <br /> nously chromosomes sisters segregate oppo-                      larity yeast mammalian chromosomes established <br /> site poles microtubule-dependent movement. Finally,                       budding yeast model study sister chromatid <br /> the segregated sister chromatids decondense.                                  cohesion condensation (Guacci et al., 1993, 1994). A <br />    Eukaryotic chromosomes large dynamic structures,                       method green fluorescent protein (GFP)&#8211;tagging chro- <br /> making daunting task identify components re-                      mosomal loci vivo enhanced yeast system <br /> sponsible cohesion condensation. Insights                    (Straight et al., 1996; Michaelis et al., 1997). The use of <br /> timing, distribution, mechanism sister chromatid                       FISH- GFP-tagged loci cytological tools combined <br /> cohesion provided studies mamma-                         powerful genetic molecular tools available in <br /> lian yeast cells. Classical cytological analyses fluo-                yeast led isolation mitotic regulators and <br />                                                                               chromosomal structural proteins (Guacci et al., 1993, 1994, <br />  <br /> Address correspondence Vincent Guacci, Fox Chase Cancer Center, <br /> Basic Science Division, Room W462, 7701 Burholme Avenue, Philadel-            1 <br />                                                                                 Abbreviations used paper: ChIP, chromatin immunoprecipitation; <br /> phia, PA 19111. Tel.: (215) 728-5632. Fax: (215) 728-3616. E-mail:            FISH, fluorescent situ hybridization; GFP, green fluorescent protein; <br /> va_guacci@fccc.edu                                                            HA, hemagglutinin. <br />  <br />  <br />  <br />  <br /> &#63721; The Rockefeller University Press, 0021-9525/2000/10/613/14 $5.00 <br /> The Journal Cell Biology, Volume 151, Number 3, October 30, 2000 613&#8211;626 <br /> http://www.jcb.org/cgi/content/full/151/3/613                                 613 <br />  1997b; Strunnikov et al., 1995; Funabiki et al., 1996; Yama-       al., 1999). In addition roles cohesion con- <br /> moto et al., 1996a,b; Michaelis et al., 1997; Furuya et al.,       densation, members Smc protein <span id='am-44' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-45' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> been <br /> 1998; Skibbens et al., 1999; Toth et al., 1999; Ciosk et al.,      implicated dosage compensation, recombination, and <br /> 2000; Tanaka et al., 2000).                                        transcriptional silencing (Chuang et al., 1994; Jessberger et <br />    Chromosomal proteins essential sister chromatid co-         al., 1996; Donze et al., 1999). One prominent characteristic <br /> hesion identified yeast vertebrates               Smc protein family members possess large <br /> (Guacci et al., 1997b; Michaelis et al., 1997; Furuya et al.,      coiled-coil domains span length (re- <br /> 1998; Losada et al., 1998; Skibbens et al., 1999; Toth et al.,     viewed Koshland Strunnikov, 1996). It that <br /> 1999; Ciosk et al., 2000; Tanaka et al., 2000). In budding         broad range chromosomal functions conse- <br /> yeast, cohesion factors Mcd1p (also known             quence specific interactions variety proteins with <br /> Scc1p), Smc1p, Smc3p, Ctf7p (also known Eco1p), Scc2p,          Smc members coiled-coil domain. <br /> Scc3p (also known Irr1p), Scc4p (Kurlandzka et al.,            The cohesin complex proteins molecular <br /> 1995; Guacci et al., 1997b; Michaelis et al., 1997; Skibbens et    glue mediates sister chromatid cohesion mito- <br /> al., 1999; Toth et al., 1999; Ciosk et al., 2000). These factors   sis. However, sister chromatid cohesion complex and <br /> can functionally divided mediate cohesion       highly regulated process cohesin complex only <br /> (i.e., form molecular &#8220;glue&#65533;?) required      story (reviewed Koshland Guacci, 2000). <br /> to establish cohesion glue.                The establishment cohesion requires load- <br />    Several lines evidence indicate Mcd1p, Smc1p,           ing cohesin complex chromosomes, likely <br /> Smc3p, Scc3p molecular glue. First,            coupling DNA replication. The identification the <br /> these proteins form soluble complex termed co-              budding yeast Trf4p DNA polymerase required for <br /> hesin, localized chromosomes early S phase           sister chromatid cohesion supports connection be- <br /> (Losada et al., 1998). Second, Mcd1p required cohe-         tween cohesion replication (Wang et al., 2000). Fur- <br /> sion cell viability soon S phase, indicat-     thermore, Drosophila MEI-S322 protein loads onto <br /> ing essential role time cohesion established          centromeric regions chromosomes prometaphase and <br /> (Guacci et al., 1997b; Uhlmann Nasmyth, 1998). Fur-            thought function maintenance cohesion at <br /> thermore, Mcd1p essential cohesion mitosis, dem-         centromere (Kerrebrock et al., 1995; Moore et al., <br /> onstrating required maintain cohesion (Guacci        1998). Thus, region-specific cohesion factors that <br /> et al., 1997b). Third, functional site cohesion           load chromosomes DNA replication. Fi- <br /> mapped yeast CEN DNA vivo (Megee                     nally, Mcd1p required condensation cohe- <br /> Koshland, 1999). Subsequently, cohesin complex members             sion (Guacci et al., 1997b). Therefore, continued our <br /> Mcd1p Smc1p shown localize yeast CEN                screen identify additional components required for <br /> DNA, providing direct link cohesin complex           sister chromatid cohesion help elucidate mecha- <br /> and functional site cohesion (Blat Kleckner, 1999;        nisms sister chromatid cohesion condensation. <br /> Megee et al., 1999; Tanaka et al., 1999).                          Here identified new budding yeast protein, <br />    Several lines evidence indicate Ctf7p, Scc2p,       Pds5p (precocious dissociation sister chromatids). <br /> Scc4p required establishment cohesion.              Pds5p 1,277 amino acids shares homology with <br /> First, required cohesion cell viability dur-      proteins <span id='am-35' about='obo:NCBITaxon_4751' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-36' property="oboInOwl:hasDbXref" content="PMID:12684019" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="oboInOwl:hasDbXref" content="PMID:11062127" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="fungi" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Fungi" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="fungi" datatype="xsd:string"></span>fungi</span> (<span id='am-52' about='obo:NCBITaxon_5052' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5042 obo:NCBITaxon_33154 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Aspergillus" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasAlternativeId" content="NCBITaxon:37503" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57'  ></span>Aspergillus</span> nidulans {BimD6} and <br /> ing soon S phase, required S <br /> phase (Skibbens et al., 1999; Toth et al., 1999; Ciosk et al., <br /> 2000). Second, Scc2p Scc4p required loading            Table I. Yeast Strains Used Study <br /> the cohesin complex chromosomes, colo-             Strain                                  Genotype <br /> calize complex (Toth et al., 1999; Ciosk et al., <br />                                                                    VG958-7C              Mata pds5-3 trp1 leu2 bar1 gal1 <br /> 2000). Third, genetic interactions Ctf7p and <br />                                                                    VG982-3A              Mat&#65533;?&#65533; trp1 leu2 ura3 bar1 gal1 <br /> Pol30p (PCNA), known replication protein, Ctf18p,            VG982-7B              Mat&#65533;?&#65533; trp1 leu2 bar1 gal1 <br /> a replication factor C homologue, indicate cohesion           VG982-6A              Mata trp1 ura3 bar1 gal1 <br /> and replication intrinsically linked (Skibbens et al.,         VG986-5B              Mata pds5-1 trp1 ura3 bar1 gal1 <br /> 1999). Taken together, Scc2p, Scc4p, Ctf7p function         VG987-5C              Mata pds5-2 trp1 ura3 bar1 gal1 <br /> load proteins physically mediate cohesion link        VG988-1C              Mata pds5-3 trp1 ura3 bar1 gal1 <br /> the establishment cohesion DNA replication.                VG1328-2C             Mat&#65533;?&#65533; cdc15-2 trp1 ura3 gal1 <br />    Chromosomal proteins essential chromosome con-              VG2043-8C             Mata trp1 lys2-801 bar1 gal1 <br /> densation identified yeast Xenopus           VG2049-3A             Mata MCD1-GFP ura3 bar1 gal1 <br /> (Hirano Mitchison, 1994; Saitoh et al., 1994; Saka et          VG2050-1A             Mata MCD1-6HA ura3 bar1 gal1 <br />                                                                    VG2061-1D             Mat&#65533;?&#65533; MCD1-GFP cdc15-2 ura3 leu2 gal1 <br /> al., 1994; Strunnikov et al., 1995; Hirano et al., 1997; Sutani <br />                                                                    VG2063-5C             Mata PDS5-GFP trp1 ura3 lys2-801 bar1 gal1 <br /> and Yanagida, 1997; Lavoie et al., 2000; Ouspenski et al.,         VG2064-2A             Mata PDS5-6HA trp1 ura3 lys2-801 bar1 gal1 <br /> 2000). In budding yeast, Smc2p Brn1p required              VG2069-1B             Mata PDS5-GFP cdc15-2 trp1 ura3 gal1 <br /> for chromosome condensation (Strunnikov et al., 1995;              VG2072-2D             Mata mcd1-1 PDS5-6HA trp1 ura3 bar1 gal1 <br /> Lavoie et al., 2000; Ouspenski et al., 2000). Homologues        VG2073-7D             Mata pds5-1 MCD1-6HA leu2 ura3 bar1 gal1 <br /> Smc2p, Smc4p, Barren pro-              VG2074-6C             Mata pds5-2 MCD1-6HA leu2 bar1 gal1 <br /> teins form soluble complex termed condensin (Hirano et           VG2073-7D             Mata pds5-3 MCD1-6HA leu2 ura3 bar1 gal1 <br /> al., 1997; Sutani et al., 1999). While mechanism con-       VG1325                Mata trp1 leu2 ura3 ade2 bar1 gal1 <br /> densation determined, condensin com-                                    Mat&#65533;?&#65533; trp1 leu2 ura3 ade3 bar1?gal1 can1 <br /> plex function induce supercoils DNA facili-           VG1359                Mata pds5::URA3 trp1 leu2 ura3 ade2 bar1 galI <br />                                                                                          Mat&#65533;?&#65533; trp1 leu2 ura3 ade3 bar1?gal1 can1 <br /> tate condensation (Kimura Hirano, 1997; Kimura et <br />  <br />  <br /> The Journal Cell Biology, Volume 151, 2000                      614 <br />  <span id='am-47' about='obo:NCBITaxon_5146' typeof='obo:NCBITaxon_222544 owl:Thing obo:NCBITaxon_5139 obo:NCBITaxon_1 obo:NCBITaxon_5148 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_147550 obo:NCBITaxon_715989 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_2759'><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50'  ></span><span id='am-51' property="rdfs:label" content="Sordaria" datatype="xsd:string"></span>Sordaria</span> {Spo76}) putative proteins Schizo-                      virtually spores germinated attempted cell divisions <br /> <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24'  ></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>saccharomyces</span> pombe humans (Denison et al., 1993;                           dying. Finally, strain VG1359 transformed plasmid <br />                                                                                pTH10, centromere plasmid bearing PDS5 andTRP1, sporulated, and <br /> van Heemst et al., 1999). The BimD6 Spo76 proteins                         dissected. Spore viability increased 65%. Of 10 tetrads dissected, 1 was <br /> are essential mitotic meiotic chromosome segrega-                      4:0 (viable:inviable), 4 3:1, 5 2:2. Viable Ura&#11001; spores were <br /> tion, respectively (Denison et al., 1993; van Heemst et al.,                   obtained Trp&#11001;, indicating spores containing the <br /> 1999). Here demonstrate Pds5p essential chromo-                       pds5::URA3 allele grew contained plasmid pTH10. These <span id='am-98' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-102'  ></span>data</span> <br />                                                                                demonstrate PDS5 essential gene. <br /> somal protein required chromosome segregation and <br /> sister chromatid cohesion mitosis. Thus, Pds5p                      COOH-terminal Tagging Pds5p Mcd1p <br /> functional homology BimD6 Spo76. Further- <br /> more, localization Pds5p chromosomes depen-                       Strains constructed contain COOH-terminally tagged <br />                                                                                Pds5p (Pds5p-6HA Pds5p-GFP) Mcd1p (Mcd1p-6HA Mcd1p- <br /> dent cohesin complex member Mcd1p. Finally,                          GFP) control endogenous promoters sole source of <br /> show Pds5p required chromosome conden-                        Pds5p Mcd1p, respectively. Haploids tagged Pds5p as <br /> sation mitosis, confirming sister chromatid co-                    follows: BamHI site generated amino acid PDS5 <br /> hesion condensation linked.                                            ORF PCR 0.26-kb BamHI fragment bearing tandem he- <br />                                                                                magglutinin (HA) epitopes 0.75-kb BamHI fragment bearing GFP was <br />                                                                                inserted form plasmid pVG264 pVG263, respectively. These <span id='am-108' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-109' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span> <br /> Materials Methods                                                          pRS406 backbone, PDS5 promoter, ORF tag 3&#11032;UTR. <br />                                                                                Plasmids pVG264 pVG263 linearized AflII, transformed into <br />                                                                                haploid VG982-3A, plated Ura&#11002; media create tandem inte- <br /> Reagents Media                                                             grants PDS5 genomic locus. Ura&#11001; transformants streaked on <br /> Reagents media described previously (Guacci et al., 1997b;            <span id='am-1' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5'  ></span>FOA</span>-containing plates Ura&#11002; cells (popouts) screened PCR to <br /> Yamamoto et al., 1996b). Benomyl gift Dupont. Yeast strains         identify retaining tagged PDS5 alleles. These crossed haploid <br /> are listed Table I.                                                         VG2043-8C resulting diploids sporulated dissected gener- <br />                                                                                ate VG2064-2A (PDS5-6HA) VG2063-5C (PDS5-GFP). Haploid <br />                                                                                VG2063-5C crossed VG1328-2C resulting diploid sporu- <br /> Identification Mutants Exhibiting Mitotic Lethality                         lated dissected generate haploid VG2069-1B (cdc15-2 PDS5-GFP). <br /> A collection temperature-sensitive mutants (Ts&#11002;) patched               To make haploids containing tagged Mcd1p, BamHI site gener- <br /> YEPD plates grown 23&#11034;C. Cells replica plated media con-         ated PCR amino acid MCD1 ORF the <br /> taining &#65533;?&#65533; factor (10&#11002;8 M) benomyl (70 &#65533;?&#65533;g/ml) incubated        6HA GFP fragment inserted form plasmid pVG269 pVG270, re- <br /> 23&#11034;C cells arrested G1 mid-M phase, respectively. Plates    spectively. These plasmids YIplac211 backbone, MCD1 pro- <br /> were transferred 37&#11034;C 3 h inactivate mutant gene product         moter, ORF tag, 3&#11032;UTR. Plasmids pVG269 pVG270 <br /> while cells arrested, replica plated YEPD incubated       linearized AgeI, transformed haploid VG982-3A. Cells <br /> at 23&#11034;C 48 h release cells arrest allow growth. Mutants        bearing MCD1-6HA MCD1-GFP alleles identified described <br /> showing enhanced lethality transient arrest mid-M phase com-          tagged Pds5p. Strains bearing tagged Pds5p Mcd1p grew rates <br /> pared G1 phase chosen study.                                     similar isogenic wild-type strains, indicating tag did com- <br />                                                                                promise protein function. <br /> Cloning PDS5 Gene <br />                                                                                Yeast Cell Culture Conditions <br /> Three mutants enhanced mitotic lethality precocious dissociation <br /> of sister chromatids fell single complementation group based       Yeast cells grown YPD 23&#11034;C mid-log phase treated as <br /> failure grow 37&#11034;C mated, renamed pds5-1, pds5-2,        follows. <br /> and pds5-3. Strains back-crossed Ts&#11002; mitotic lethality           Arrest G1, S Mid-M Phase. &#65533;?&#65533; factor (10&#11002;8 M), hydroxyurea (HU; <br /> phenotypes segregated 2&#11001;:2&#11002; cosegregated. For analysis pds5        100 mM), nocodazole (15 &#65533;?&#65533;g/ml) added arrest cells G1, S, or <br /> phenotypes, original isolates backcrossed times wild-type         mid-M phase, respectively, described (Guacci et al., 1997b). <br /> strains generate haploids VG986-5B (pds5-1), VG987-5C (pds5-2),            Synchronous Populations Cycling Cells Released From S Phase. <br /> VG988-1C (pds5-1). The PDS5 gene cloned transforming haploid            Cells treated described (Guacci et al., 1997b), &#65533;?&#65533; factor <br /> VG958-7C (pds5-3) pRS314 (CEN ARS TRP1) vector-based yeast              (10&#11002;8 M) added media release S phase. <br /> genomic library (provided P. Hieter, University British Columbia,           Synchronous Populations Mid-M Phase Cells Obtained Release <br /> Vancouver, BC, Canada). Transformed cells plated Trp&#11002; me-            G1 Phase. Cells treated described (Guacci et al., 1997b). <br /> dia, incubated 22 h 23&#11034;C, transferred 37&#11034;C 3 d. Five <br /> clones grew 37&#11034;C contained identical plasmids (8B). A 5.8-kb ClaI       Fluorescence In Situ Hybridization <br /> fragment plasmid 8B complemented Ts&#11002; mitotic lethal <br />                                                                                FISH performed described described (Guacci et al., 1994, <br /> phenotypes pds5 mutants. The ClaI insert ends sequenced and <br />                                                                                1997a). Probes used analysis cohesion condensation de- <br /> contained ORF YMR076C chromosome XIII. <br />                                                                                scribed (Guacci et al., 1994, 1997a,b). <br />  <br /> PDS5 Is Essential Gene                                                      Chromosome Spreads <br /> Plasmid pVG177 contains 5.8-kb ClaI fragment bearing PDS5 vector          Chromosome spreads performed described (Michaelis et al., <br /> pRS314. The URA3 gene 1.2-kb HindIII fragment (blunt ended using          1997). HA-tagged proteins detected using mouse anti&#8211;HA antibody <br /> Klenow) inserted plasmid pVG177 BstEII site (blunt             16B12 (Babco) (1:2,500) Cy3-conjugated goat anti&#8211;mouse antibody <br /> ended using Klenow) form plasmid, pVG199, URA3 inserted           (Jackson ImmunoResearch Laboratories) (1:3,000). <br /> after amino acid 114 PDS5 ORF (pds5::URA3). Plasmid pVG199 <br /> was digested ClaI transformed diploid VG1325. Ura&#11001; trans- <br /> formants analyzed Southern blot identify clones pds5::         Chromatin Immunoprecipitation <br /> URA3 chromosome wild-type PDS5 homologue,              Chromatin immunoprecipitation (ChIP) PCR analysis per- <br /> were renamed diploid VG1359.                                                formed described (Megee et al., 1999). <br />    To determine PDS5 essential gene, wild-type parent <br /> VG1325 (PDS5/ PDS5) VG1359 (pds5::URA3/PDS5) diploid cells                 Fluorescence Microscopy Flow Cytometry <br /> were sporulated dissected. VG1325 (PDS5/ PDS5) 98% spore <br /> viability. In contrast, VG1359 (pds5::URA3/PDS5) 47% spore          Indirect immunofluorescence flow cytometry described (Yama- <br /> viability. Of 29 tetrads dissected, 25 2:2 (viable:inviable) 4   moto et al., 1996a). Images collected using epi-fluorescence mi- <br /> 1:3 tetrads. All viable spores Ura&#11002; indicating contained    croscope (E800; Nikon) recorded digitally using Hamamatso cooled <br /> pds5::URA3 allele. Microscopic examination spores failed form       CCD <span id='am-73' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-77'  ></span>camera</span> processing software Phase 3 Imaging, al- <br /> colonies revealed 98% cell bodies, demonstrating          lowed image superimposition. <br />  <br />  <br />  <br /> Hartman et al. Pds5p Is Required Cohesion Condensation                 615 <br />  Results                                                          using &#65533;?&#65533; factor, hydroxyurea, nocodazole, respec- <br />                                                                  tively, incubated 2 h 37&#11034;C arrested <br /> PDS5 Was Isolated Screen Designed Identify               (see Materials Methods). Aliquots cells were <br /> Genes Encoding Proteins Important Sister                     washed free arresting agent plated YEPD to <br /> Chromatid Cohesion                                               determine cell viability (Fig. 1 B). In G1 phase, pds5 cells <br />                                                                  maintained viability shifted 37&#11034;C. In S phase, pds5 <br /> We previously used cytogenetic screen isolate mutants       cells showed 1.5-fold decrease viability in- <br /> defective sister chromatid cohesion identified pds1       cubated 37&#11034;C. However, mid-M phase, pds5 cells <br /> and mcd1 mutants (Guacci et al., 1993, 1997b; Yamamoto           showed 3-fold decrease viability 23&#11034;C 40-fold <br /> et al., 1996a,b). Here characterization       decrease 2% viability shifted 37&#11034;C. Thus, Pds5p <br /> pds5 (precocious dissociation sister chromatids), an-         function required G1 phase, essential in <br /> other mutant identified screen. In brief, collection   mid-M phase. The weak effect S phase pds5 cells indi- <br /> of mutants temperature sensitive growth (Ts&#11002;)           cates Pds5p function required during <br /> screened identify exhibiting enhanced lethality af-     replication soon after, Pds5p localizes chromo- <br /> ter transient arrest mid-M phase compared G1          somes early S phase (see below). <br /> phase. Mutants enhanced mitotic lethality were <br /> screened FISH identify exhibiting precocious <br /> dissociation sister chromatids (see below). Three mu- <br />                                                                  PDS5 Is Essential Gene Required Proper <br /> tants formed complementation group named <br />                                                                  Chromosome Segregation <br /> pds5-1, pds5-2, pds5-3. All mutants showed de-         The PDS5 gene cloned complementation the <br /> creased viability 30&#11034;C growth 37&#11034;C, indicating      Ts&#11002; mitotic lethal phenotypes pds5 cells (Fig. 1 C). <br /> that allele defective function high tempera-     Standard yeast genetic techniques used that <br /> ture (Fig. 1 A). Even 23&#11034;C, pds5 allele         PDS5 essential gene (Materials Methods). The <br /> growth defects pds5 cells form heterogeneously sized       mitotic lethality pds5 cells suggested function for <br /> colonies &#11601;80% pds5 cells form colonies            Pds5p chromosome structure segregation. To deter- <br /> YEPD plates, compared 100% wild-type             role Pds5p cell cycle progression chro- <br /> cells (data shown).                                          mosome segregation, wild-type pds5 haploid cells <br />   To precisely assess stage cell cycle           grown mid-log phase 23&#11034;C shifted 37&#11034;C, and <br /> where Pds5p function essential, wild-type pds5 hap-       cell, spindle, DNA morphologies DNA <br /> loid cells arrested G1, S, mid-M phases        content scored. At 23&#11034;C, FACS&#174; (Becton Dickinson) <br />  <br />  <br />  <br />  <br />                                                                                               Figure 1. Characterization of <br />                                                                                               pds5 mutants. (A) Ts&#11002; pheno- <br />                                                                                               type pds5 cells. Wild-type <br />                                                                                               (VG982-6A), pds5-1 (VG986- <br />                                                                                               5B), pds5-2 (VG987-5C), and <br />                                                                                               pds5-3 (VG988-1C) cells grown <br />                                                                                               23&#11034;C YPD liquid were <br />                                                                                               plated 10-fold serial dilutions <br />                                                                                               YPD incubated 23&#11034;, <br />                                                                                               30&#11034;, 37&#11034;C. (B) Cell cycle&#8211; <br />                                                                                               dependent lethality pds5 <br />                                                                                               cells. Strains A arrested <br />                                                                                               23&#11034;C G1, S, mid-M <br />                                                                                               phase, incubated 37&#11034;C while <br />                                                                                               arrested, plated on <br />                                                                                               YPD 23&#11034;C determine per- <br />                                                                                               cent viability (Materials and <br />                                                                                               Methods). Data three <br />                                                                                               independent experiments was <br />                                                                                               used generate error bars. (C) <br />                                                                                               Cloning PDS5 gene. <br />                                                                                               Strains A transformed <br />                                                                                               CEN vector alone <br />                                                                                               bearing PDS5 ORF. <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 151, 2000                    616 <br />                                                                                                 Figure 2. Cell cycle progression <br />                                                                                                wild-type pds5 mutant <br />                                                                                                cells. (A) DNA content cells. <br />                                                                                                Wild-type (VG982-6A) and <br />                                                                                                pds5-3 (VG988-1C) cells were <br />                                                                                                grown asynchronously 23&#11034;C <br />                                                                                                shifted 37&#11034;C 3 h, and <br />                                                                                                subjected FACS&#174; analy- <br />                                                                                                sis measure DNA content <br />                                                                                                (Materials Methods). (B) <br />                                                                                                Micrographs showing mi- <br />                                                                                                totic spindle cells 1 h re- <br />                                                                                                lease S phase 37&#11034;C. <br />                                                                                                Strains A synchronously <br />                                                                                                released 37&#11034;C S phase <br />                                                                                                arrest, and, 1 h, cells were <br />                                                                                                processed immunofluores- <br />                                                                                                cence. Chromosomal DNA <br />                                                                                                (gray) Tubulin (white). (C) <br />                                                                                                Quantitation cell DNA <br />                                                                                                morphologies release from <br />                                                                                                S phase 37&#11034;C. Strains B <br />                                                                                                scored cell DNA <br />                                                                                                morphologies S phase ar- <br />                                                                                                rested cells 37&#11034;C [S phase <br />                                                                                                (HU)] cells 2 h re- <br />                                                                                                lease 37&#11034;C (2 h S phase). <br />                                                                                                Data 400 cells two <br />                                                                                                independent experiments were <br />                                                                                                scored generate error bars. <br />  <br />  <br /> profiles pds5 cells similar wild-type cells (Fig. 2   tures wild-type pds5 cells grown mid-log <br /> A). However, 3 6 h 37&#11034;C, pds5 cultures      phase 23&#11034;C, arrested S phase addition hydroxy- <br /> enriched cells 2C DNA content showed              urea, transferred 37&#11034;C 1 h pres- <br /> twofold increase frequency large budded cells          ence hydroxyurea inactivate mutant Pds5p S <br /> with short partially elongated spindles (Fig. 2 A         phase (Materials Methods). Finally, cells re- <br /> data shown). These phenotypes indicate pds5 cells       leased S phase transferring YEPD media con- <br /> accumulate G2 M phases cell cycle             taining &#65533;?&#65533; factor 37&#11034;C. <span id='am-68' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-69'  ></span><span id='am-70' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> regimen allows cells to <br /> grown 37&#11034;C. This result consistent defect        progress mitosis nonpermissive temperature <br /> chromosome structure delaying cell cycle progression             pds5, arrests cells G1 phase prevent <br /> through mitosis.                                                 progression new cell cycle (Materials Meth- <br />   Pds5 mutant cells characterized using syn-        ods). In S phase arrested cells 23&#11034; 37&#11034;C, wild- <br /> chronous populations cycling cells obtained re-         type pds5 cells 1C DNA content, large bud, un- <br /> lease S phase arrest. Asynchronously growing cul-           divided DNA mass, short spindle (Fig. 2 C, data <br />  <br />  <br />  <br />  <br /> Hartman et al. Pds5p Is Required Cohesion Condensation   617 <br />                                                                                            Figure 3. Sister chromatid cohe- <br />                                                                                           sion wild-type pds5 cells. <br />                                                                                           (A) Micrographs mid-M <br />                                                                                           phase cells subjected FISH <br />                                                                                           using chromosome XVI CEN- <br />                                                                                           distal probe distal (295 kb from <br />                                                                                           CEN16). Wild-type (VG982- <br />                                                                                           6A) pds5-2 (VG987-5D) <br />                                                                                           cells arrested G1 phase <br />                                                                                           [G1 (&#65533;?&#65533;F) 23&#11034;C], shifted 37&#11034;C, <br />                                                                                           released G1 and <br />                                                                                           arrested mid-M phase {M (no- <br />                                                                                           codazole) 37&#11034;C}. Mid-M cells <br />                                                                                           processed FISH. Chro- <br />                                                                                           mosomal DNA (gray) hy- <br />                                                                                           bridized probe (white). Bar, 5 <br />                                                                                           &#65533;?&#65533;m. (B) Quantitation FISH <br />                                                                                           G1 mid-M phase cells. <br />                                                                                           Wild-type (VG982-6A), pds5-1 <br />                                                                                           (VG986-5B), pds5-2 (VG987- <br />                                                                                           5C), pds5-3 (VG988-1C) <br />                                                                                           arrested subjected to <br />                                                                                           FISH described A. The <br />                                                                                           number FISH signals each <br />                                                                                           DNA mass determined and <br />                                                                                           plotted percentage total <br />                                                                                           nuclei. 200 nuclei G1 and <br />                                                                                           mid-M cells scored gen- <br />                                                                                           erate data. (C) DNA content of <br />                                                                                           mid-M phase cells. Aliquots of <br />                                                                                           cells strains B fixed <br />                                                                                           processed flow cytome- <br />                                                                                           try analyze DNA content <br />                                                                                           mid-M phase&#8211;arrested cells at <br />                                                                                           37&#11034;C. <br />  <br />  <br /> not shown). 1 h release S phase 37&#11034;C,     Pds5p Is Required Sister Chromatid Cohesion <br /> wild-type pds5 cells completed DNA replication       We assayed pds5 cells defective sister <br /> as indicated 2C DNA content (data shown). Most      chromatid cohesion. Wild-type, pds5-1, pds5-2, pds5-3 <br /> wild-type cells completed chromosome segrega-       haploid strains arrested G1 phase 23&#11034;C incuba- <br /> tion elongated spindle segregated      tion YEPD liquid containing &#65533;?&#65533; factor, trans- <br /> DNA masses (Fig. 2 B). In contrast, pds5 cells    ferred 37&#11034;C 1 h inactivate mutant Pds5p G1 <br /> stretched nuclear DNA mass partially elongated spin-     phase (Materials Methods). Cultures re- <br /> dle, indicative chromosome segregation defect (Fig. 2   leased G1 phase arrest 37&#11034;C transferring cells to <br /> B). 2 h release, 75% wild-type cells unbud-    YEPD liquid-containing nocodazole arrest cells mid- <br /> ded, demonstrated exited mitosis     M phase 37&#11034;C (Materials Methods). In proto- <br /> arrested G1 phase (Fig. 2 C). In contrast, 25%    col, cells nonpermissive temperature the <br /> pds5-3 cells unbudded majority remained       pds5 mutation S, G2, mid-M phases, which <br /> in mitosis large budded cells anaphase   encompasses time sister chromatid cohesion is <br /> stretched DNA mass telophase separated DNA        established maintained. Aliquots G1 mid-M <br /> masses (Fig. 2 C). After 4 h, number unbudded         phase cells processed FISH using chromosome <br /> pds5-3 cells increased 63%. Similar results ob-      XVI centromere-distal probe assay cohesion site <br /> tained pds5-1 pds5-2 cells (data shown).        295 kb centromere (Materials Methods). <br /> These data suggest pds5 inactivation causes mitotic   Consistent previous studies, wild-type cells <br /> defect disrupts chromosome segregation delays,      mid-M phase FISH signal nuclear DNA <br /> but does prevent, exit mitosis.                     mass, demonstrating sister chromatids paired <br />  <br /> The Journal Cell Biology, Volume 151, 2000                618 <br />                                                                          replication new bud formation (Fig. 3 C, data not <br />                                                                         shown). Therefore, precocious dissociation sister <br />                                                                         chromatids pds5 cells defect cell-cycle <br />                                                                         regulation, reflects potential role Pds5p sister <br />                                                                         chromatid cohesion. <br />                                                                           While shown pds5 mutants defective <br />                                                                         cohesion, previous regimen does distinguish <br />                                                                         defect establishment mainte- <br />                                                                         nance cohesion. We used different regimen distin- <br />                                                                         guish possibilities inactivating mu- <br />                                                                         tant Pds5p cohesion established. Wild-type <br />                                                                         pds5-3 haploid cells arrested G1 phase 23&#11034;C <br />                                                                         using &#65533;?&#65533; factor, released G1 phase arrested in <br />                                                                         mid-M phase 23&#11034;C treatment nocodazole, and <br />                                                                         transferred 37&#11034;C inactivate mutant Pds5-3p <br />                                                                         mid-M phase (Materials Methods). Aliquots of <br />                                                                         mid-M phase cells 23&#11034; 37&#11034;C plated cell via- <br />                                                                         bility processed FISH using centromere-distal <br />                                                                         probe chromosome XVI. As expected, wild- <br />                                                                         type cells mid-M phase 23&#11034; 37&#11034;C viable and <br />                                                                         FISH signal (Fig. 4, A B). Similarly, most <br />                                                                         pds5-3 cells mid-M phase 23&#11034;C viable had <br />                                                                         FISH signal. However, shift mid-M phase <br />                                                                         pds5-3 cells 37&#11034;C, viability decreased number of <br />                                                                         cells FISH signals increased (Fig. 4, A B). <br />                                                                         Similar results obtained using FISH probes cen- <br />                                                                         tromere-proximal regions chromosomes IV XVI <br /> Figure 4. Maintenance sister chromatid cohesion wild-type         (data shown). These results demonstrate Pds5p <br /> and pds5 cells arrested mid-M phase. Wild-type (VG982-6A)            function required maintain sister chromatid cohesion <br /> and pds5-3 (VG988-1C) cells arrested G1 phase                mid-M phase centromere proximal distal chro- <br /> 23&#11034;C, released G1 phase arrested mid-M             mosomal regions. <br /> phase 23&#11034;C. Cells transferred 37&#11034;C 1 h ar- <br /> rested. Aliquots mid-M phase cells 23&#11034; 37&#11034;C plated <br /> to determine cell viability (A) processed FISH (B). (A)         Pds5p Is Required Chromosome Condensation <br /> Cell viability. Mid-M phase cells 23&#11034; 37&#11034;C plated onto <br /> YEPD grown 3 d 23&#11034;C, percentage cell viabil-         We previously shown mcd1 mutant defec- <br /> ity calculated (Materials Methods). Data generated          tive chromosome condensation sister chro- <br /> from independent experiments. (B) Sister chromatid cohe-            matid cohesion (Guacci et al., 1997b). Therefore, used <br /> sion. Mid-M phase cells 23&#11034; 37&#11034;C processed FISH         FISH determine pds5 mutants defec- <br /> using chromosome XVI centromere distal probe (cosmid                  tive condensation. Wild-type, pds5-1, pds5-2, pds5-3 <br /> 70912). The number FISH signals DNA mass                 haploid cells arrested G1 phase 23&#11034;C using &#65533;?&#65533; fac- <br /> quantitated described Fig. 3 B.                                   <span id='am-93' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-97'  ></span>tor</span>, incubated 37&#11034;C 1 h, released G1 <br />                                                                         phase arrested mid-M phase 37&#11034;C using nocoda- <br />                                                                         zole processed FISH (Materials Methods). <br /> (Fig. 3, A B). In small number DNA masses,                 First, examined chromosome condensation the <br /> FISH signals detected low level                 rDNA locus, 500-kb block repetitive DNA. In G1 <br /> precocious sister chromatid dissociation spurious back-              phase cells, amorphous rDNA FISH characteristic a <br /> ground (Fig. 3 B). In contrast, pds5 cells mid-M                decondensed chromosome detected, mid-M <br /> phase 37&#11034;C FISH signals, indicating sister              phase haploid cells, FISH signal formed single line, <br /> chromatids precociously dissociated (Fig. 3, A B).              characteristic condensed chromosome paired sis- <br /> The second FISH signal mid-M phase pds5 cells                ter chromatids (Guacci et al., 1994). As expected, wild- <br /> due preexisting aneuploidy G1 phase cells              type cells arrested mid-M phase, FISH signal formed <br /> had FISH signal (Fig. 3 B). The FISH signals               single line edge DNA masses (Fig. 5 A). In <br /> detected nuclei G1 phase wild-type pds5               contrast, pds5-1, pds5-2, pds5-3 cells mid-M phase, <br /> cells likely spurious background. Similar re-               amorphous FISH signal reminiscent decondensed <br /> sults obtained using probes CEN-proximal re-                  rDNA detected (Fig. 5 A, data shown). An <br /> gions chromosomes I, IV, XVI (data shown).                   amorphous FISH signal expected sisters chroma- <br /> Thus, Pds5p required sister chromatid cohesion                tids simply dissociated wild-type cells, preco- <br /> both CEN-proximal distal chromosomal regions.                       ciously dissociated sisters detected line-like sig- <br />    Cells defective mitotic checkpoint precociously               nals (Guacci et al., 1997b). Furthermore, anaphase nuclei <br /> dissociate sister chromatids consequence            cycling wild-type cells, rDNA line-like sis- <br /> failure arrest response spindle damage (Straight et            ter chromatids separated segregated (Guacci et <br /> al., 1996; Yamamoto et al., 1996b). However, pds5 mutants               al., 1994). Thus, pds5 cells mid-M phase 37&#11034;C have <br /> have functional mitotic checkpoint nocodazole-                  rDNA morphology indicative defect condensation <br /> treated pds5 cells undergo new rounds DNA                     merely dissociation sister chromatids. <br />  <br /> Hartman et al. Pds5p Is Required Cohesion Condensation          619 <br />                                                                                               Figure 5. Analysis chromo- <br />                                                                                              condensation wild- <br />                                                                                              type pds5 cells. Wild- <br />                                                                                              type (VG982-6A) pds5-3 <br />                                                                                              (VG988-1C) cells synchro- <br />                                                                                              nously arrested mid-M at <br />                                                                                              37&#11034;C, described Fig. 3 A. <br />                                                                                              (A) Micrographs showing FISH <br />                                                                                              rDNA. Mid-M phase cells <br />                                                                                              37&#11034;C subjected FISH using <br />                                                                                              rDNA probe. Chromosomal <br />                                                                                              DNA (PI) hybridized probe <br />                                                                                              (FITC) shown. Arrows indi- <br />                                                                                              cate line-like rDNA wild-type <br />                                                                                              cells. Bar, 5 &#65533;?&#65533;m. (B) Micro- <br />                                                                                              graphs showing FISH chro- <br />                                                                                              mosome VIII region. Mid-M <br />                                                                                              phase cells 37&#11034;C subjected to <br />                                                                                              FISH using mixture six <br />                                                                                              chromosome VIII probes. Chro- <br />                                                                                              mosomal DNA (gray) hy- <br />                                                                                              bridized probe (white) are <br />                                                                                              shown. Bar, 5 &#65533;?&#65533;m. <br />  <br />  <br />    We determined Pds5p required chro-      signals detected DNA masses (Fig. 5 B, and <br /> mosome condensation unique chromosomal regions.             data shown). Some increased numbers FISH <br /> The number spacing FISH signals cells <span id='am-103' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-104' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-107'  ></span>hybrid</span>-        signals expected precocious sister chromatid <br /> ized mixture chromosome VIII XVI probes           dissociation. However, sister chromatids remained con- <br /> provides qualitative measure chromosome condensa-         densed tight lines clusters FISH <br /> tion. In haploid G1 phase cells hybridized chro-     signals, separated condensed sister <br /> mosome XVI probes, widely spaced FISH signals         chromatid. Instead, dispersed FISH signals remi- <br /> often detected nucleus, mid-M phase hap-        niscent separated decondensed chromosomes <br /> loid cells short line-like FISH signal       (Fig. 5 B). Similar results obtained pds5-1 and <br /> closely associated FISH signals detected (Guacci et al.,    pds5-2 cells (data shown). Taken together, results <br /> 1994). These patterns indicate change decon-          FISH using rDNA chromosome VIII XVI <br /> densed chromosome G1 phase condensed chromo-           probes indicate pds5 cells exhibit defects chromo- <br /> some paired sister chromatids mid-M phase. Wild-       condensation sister chromatid cohesion. <br /> type pds5-3 haploid cells arrested mid-M <br /> phase cells 37&#11034;C, described rDNA FISH, <br /> and subjected FISH using mixture probes <br />                                                                Pds5p Localizes Chromosomes Cell <br /> from chromosome VIII XVI (Materials and <br />                                                                Cycle&#8211;dependent Manner Its Binding to <br /> Methods). In wild-type cells, single line-like FISH signal <br />                                                                Chromosomes Is Dependent Mcd1p Function <br /> or closely associated FISH signals detected      The defects mitotic chromosome structure detected in <br /> most nuclei (Fig. 5 B, data shown). By contrast,    pds5 mutants consistent role Pds5p chro- <br /> pds5-3 cells, larger number widely spaced FISH       mosomal structural protein. To determine Pds5p <br />  <br />  <br /> The Journal Cell Biology, Volume 151, 2000                  620 <br />                                                                        Figure 6. Cell cycle&#8211;dependent localization Pds5p Mcd1p <br />                                                                       chromosomes. (A) Localization Pds5p chromosomes. <br />                                                                       Strain 2064-2A (PDS5-6HA) grown 23&#11034;C, ar- <br />                                                                       rested G1, S, mid-M phase treatment &#65533;?&#65533; factor, <br />                                                                       hydroxyurea, nocodazole, respectively. Aliquots cells were <br />                                                                       fixed processed chromosome spreads (Materials and <br />                                                                       Methods). Chromosomal DNA (DAPI) Pds5p (Pds5-6HAp) <br />                                                                       shown. *Anaphase cell asynchronous population. <br />                                                                       Bar, 5 &#65533;?&#65533;m. (B) Localization Mcd1p chromosomes. Strain <br />                                                                       VG2050-1A (MCD1-6HA) grown processed chro- <br /> mosome spreads described A. Chromosomal DNA (DAPI) Mcd1p (Mcd1-6HAp) shown. *Anaphase cell asyn- <br /> chronous population. Bar, 5 &#65533;?&#65533;m. (C) Enlargement mid-M phase cells cultures A B. Bar, 1.25 &#65533;?&#65533;m. (D) Binding Pds5p <br /> and Mcd1p CEN3 LEU2 loci. Strains A grown mid-log phase subjected ChIP using anti&#8211;HA antibody <br /> C12A5, analyzed PCR presence CEN3, LEU2, ADE3 chromosomal loci (Materials Methods). (Total) <br /> PCR total chromatin equivalent 1% used ChIP; positive control PCR. (E) Assay determine Mcd1p function <br /> is required Pds5p localization chromosomes. Wild-type (VG2064-2A) mcd1-1 mutant (VG2072-2D) strains containing Pds5p- <br /> 6HA arrested G1 phase 23&#11034;C, transferred 37&#11034;C, arrested mid-M phase 37&#11034;C processed chromosome <br /> spreads (Materials Methods). Chromosomal DNA (DAPI) Pds5p (Pds5-6HAp) shown. Bar, 5 &#65533;?&#65533;m. (F) Assay determine <br /> whether Pds5p function required Mcd1p localization chromosomes. Wild-type (VG2050-1A) pds5 mutant ({VG2073-7D; <br /> pds5-1}, {VG2074-6C; pds5-2}, {VG2075-6D; pds5-3} strains bearing Mcd1p-6HA treated described E. Chromosomal <br /> DNA (DAPI) Mcd1p (Mcd1-6HAp) shown. Bar, 5 &#65533;?&#65533;m. <br />  <br />  <br />  <br /> Hartman et al. Pds5p Is Required Cohesion Condensation        621 <br />  binds chromosomes, asynchronous culture haploid          asynchronously 23&#11034;C, Pds5p bound chromosomes in <br /> strain VG2064-2A arrested G1, S, mid-M                 42% cells, similar wild-type, bound in <br /> phases. This strain contains PDS5-6HA allele con-         43% cells. Thus, localization Pds5p chromo- <br /> trol PDS5 promoter sole source Pds5p,            somes dependent Mcd1p function. We assessed <br /> which enabled assess chromosomal localization        Mcd1p localization chromosomes depen- <br /> Pds5p using chromosome &#8220;spread&#65533;? technique (Materials            dent Pds5p function. Wild-type VG2050-1A (Mcd1p- <br /> and Methods). In G1 phase cells, little Pds5p detected        6HA) pds5 mutant strains VG2073-7D (pds5-1 Mcd1p- <br /> on chromosomes (Fig. 6 A). In contrast, bright Pds5p sig-       6HA), VG2074-6C (pds5-2 Mcd1p-6HA) VG2075-6D <br /> nal detected S mid-M phase cells (Fig. 6          (pds5-3 Mcd1p-6HA) arrested mid-M phase at <br /> A). In asynchronous cells, general types         37&#11034;C processed chromosome spreads Mcd1p as <br /> DNA masses, elongated dumbbell-shaped DNA masses                  described above. Mcd1p showed strong chromosomal lo- <br /> from late anaphase cells roughly spherical DNA                calization wild-type pds5 mutant strains, indi- <br /> masses cells stages cell cycle. Pds5p    cating Pds5p function required Mcd1p bind- <br /> was absent elongated DNA masses late                  ing chromosomes (Fig. 6 F). However, slight <br /> anaphase cells (Fig. 6 A). For spherical DNA masses, 42%          difference patterns Mcd1p staining more <br /> had prominent Pds5p staining, remaining lit-        punctate pds5 mutant strains wild-type cells. <br /> tle Pds5p staining, consistent cell cycle&#8211;depen-     This subtle difference consequence con- <br /> dent chromosomal binding. Finally, cells arrested           densation defect pds5 mutants or, alternatively, re- <br /> telophase cdc15 mutation, little Pds5-6HAp de-         flect decrease Mcd1p binding specific sites. We are <br /> tected chromosomes (data shown). Mcd1p, like               conducting experiments determine possibility is <br /> Pds5p, required sister chromatid cohesion        correct. These results demonstrate global localiza- <br /> and condensation (Guacci et al., 1997b). Therefore, an-        tion Pds5p chromosomes dependent wild-type <br /> alyzed localization Mcd1p using haploid strain             Mcd1p function. In contrast, bulk Mcd1p localization to <br /> VG2050-1A, contains Mcd1-6HAp sole                   chromosomes does depend Pds5p function. <br /> source Mcd1p, compared Pds5p (Ma-             The chromosome spreads revealed little <br /> terials Methods). Mcd1p bound chromosomes             Pds5p chromosomes G1 phase, late anaphase, or <br /> cell cycle&#8211;dependent manner indistinguishable        telophase cells. We wanted determine was <br /> Pds5p (Fig. 6 B). The staining Pds5p Mcd1p           absence Pds5p cells exclusion of <br /> appeared brighter chromosomes S mid-M         Pds5p chromatin. To address question, asyn- <br /> phase. Interestingly, mid-M phase cells, Pds5p         chronous culture haploid strain VG2063-5C (PDS5- <br /> Mcd1p staining diffuse DAPI ap-             GFP) arrested G1, S, mid-M phases lo- <br /> peared line-like cells (Fig. 6 C). This pattern sug-   calization Pds5p directly determined intact cells by <br /> gestive staining chromosomal <span id='am-78' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-82'  ></span>axis</span>. These re-       fluorescence microscopy (Materials Methods). Pds5- <br /> sults demonstrate Pds5p localizes chromosomes        GFPp detected single prominent spherical signal <br /> cell cycle&#8211;dependent manner identical Mcd1p.           G1, S, mid-M phase cells (Fig. 7 A). The intensity of <br />    Pds5p Mcd1p localize chromosomes            Pds5p fluorescence lower G1 phase cells ei- <br /> stages cell cycle required cohesion       ther S mid-M phase cells. When DNA cells was <br /> and condensation. This suggests proteins           stained DAPI, Pds5p localize the <br /> may function binding chromosomal sites.            region DNA, indicating Pds5p nu- <br /> To test possibility, compared binding Pds5p        clear (data shown). An asynchronous culture hap- <br /> and Mcd1p discrete chromosomal loci. Asynchro-           loid strain VG2069-1B (PDS5-GFP cdc15-2) grown at <br /> nous cultures VG2064-2A (Pds5-6HAp) VG2050-                23&#11034;C, transferred 37&#11034;C 3 h arrest cells <br /> 1A (Mcd1-6HAp) subjected ChIP using anti&#8211;HA               telophase cdc15 mutation (Pringle and <br /> antibodies. PCR analysis ChIP revealed Pds5p          Hartwell, 1981). Two spherical Pds5-GFPp signals were <br /> and Mcd1p bound CEN3 DNA sequence near               observed opposite ends cell (Fig. 7 A). When <br /> LEU2 gene, protein bound region near           DNA cells stained DAPI, Pds5p localized <br /> the ADE3 gene (Fig. 6 D, Megee et al., 1999). Thus,           region DNA masses opposite ends, <br /> Pds5p Mcd1p bind specific chromo-             indicating nuclear cells (data shown). <br /> somal loci, suggests function common           We examined Mcd1p arresting haploid strain VG2049- <br /> chromosomal sites mediate sister chromatid cohesion.           3A (MCD1-GFP) G1, S, mid-M phase (Materials <br />    We determined chromosomal binding             Methods). Mcd1-GFPp detected G1 phase <br /> of Pds5p dependent Mcd1p function. Asynchro-               cells, formed single prominent round signal S and <br /> nously growing cultures strains VG2072-2D (mcd1-1              mid-M phase cells (Fig. 7 B). When haploid strain <br /> Pds5-6HAp) VG2064-2A (Pds5-6HAp) arrested                VG2061-1D (MCD1-GFP cdc15-2) arrested telo- <br /> in G1 phase 23&#11034;C, transferred 37&#11034;C inactivate        phase described above, faint Mcd1-GFPp signal <br /> mutant mcd1p, released G1 arrested           detected, indicating Mcd1p ab- <br /> mid-M phase 37&#11034;C (Materials Methods). Mid-M                sent cells. Western blot analysis total protein from <br /> phase cells 37&#11034;C subjected chromosome spreads          cells bearing Pds5-6HAp Mcd1-6HAp confirmed <br /> cells detect Pds5p (Materials Methods). As ex-             cell cycle&#8211;dependent changes Pds5p Mcd1p lev- <br /> pected, Pds5p widely distributed chromosomes            els obtained GFP-tagged proteins (data not <br /> wild-type cells (Fig. 6 E). In contrast, Pds5p failed bind     shown). These results demonstrate Pds5p Mcd1p <br /> to chromosomal regions mcd1 mutant mid-M            bound chromosomes similar stages cell cy- <br /> phase 37&#11034;C (Fig. 6 E). When mcd1 mutant grown          cle. However, Pds5p present nucleus stages <br />  <br />  <br /> The Journal Cell Biology, Volume 151, 2000                     622 <br />                                                                       times sister chromatid cohesion exists. Furthermore, <br />                                                                      Pds5p required maintain cohesion late mid-M <br />                                                                      phase. In contrast, Ctf7p, Scc2p, Scc4p, nec- <br />                                                                      essary establishment cohesion, required for <br />                                                                      sister chromatid cohesion cell viability S phase but <br />                                                                      mid-M phase (Skibbens et al., 1999; Toth et al., <br />                                                                      1999; Ciosk et al., 2000). Thus, Pds5p proper- <br />                                                                      ties cohesin complex members Mcd1p, Smc1p, <br />                                                                      Smc3p, Scc3p. These results demonstrate Pds5p is <br />                                                                      component molecular glue mediates sister <br />                                                                      chromatid cohesion maintenance cohesion <br />                                                                      requires cohesin complex proteins. <br />                                                                         What relationship Pds5p cohesin <br />                                                                      complex? First, shown Pds5p Mcd1p lo- <br />                                                                      calize chromosomes cell cycle stages. Inter- <br />                                                                      estingly, localization Pds5p Mcd1p mid-M <br />                                                                      phase cells tighter DAPI, suggesting that <br />                                                                      proteins core chromosomal axis <br />                                                                      (this study). Second, chromatin immunoprecipitation of <br />                                                                      Pds5p reveals binds chromosomal DNA sites in <br />                                                                      CEN3 near LEU2, near ADE3. This binding <br />                                                                      pattern identical Mcd1p (Megee et al., 1999). <br />                                                                      In addition, global localization Pds5p chromo- <br />                                                                      somes depends Mcd1p function, Mcd1p localiza- <br />                                                                      tion chromosomes independent Pds5p function. <br /> Figure 7. Localization Pds5p Mcd1p intact yeast cells.     Third, mcd1 pds5 mutants synthetically lethal (V. <br /> Haploid strains VG2063-5C (PDS5-GFP) VG2049-3A <br />                                                                      Guacci, unpublished results). Moreover, bimD6 mu- <br /> (MCD1-GFP) grown mid-log phase 23&#11034;C, ar- <br /> rested G1, S, mid-M phase. Haploid strains VG2069-1B <br />                                                                      tant suppressed mutation SudA gene, en- <br /> (cdc15-1 PDS5-GFP) VG2061-1D (cdc15-2 MCD1-GFP)                  codes A. nidulans homologues Pds5p Smc3p, <br /> were grown mid-log phase 23&#11034;C, arrested telo-      respectively (Holt May, 1996). These results reveal <br /> phase incubation 37&#11034;C 3 h. Arrested cells fixed   Pds5p cohesin complex proteins functionally <br /> the localization GFP-tagged proteins determined intact cells   interact. Furthermore, fact Mcd1p binding to <br /> by fluorescence microscopy (Materials Methods). Localiza-        chromosomes independent Pds5p function suggests <br /> tion (A) Pds5p (B) Mcd1p cells different cell     cohesin complex serve load Pds5p onto <br /> cycle stages.                                                        chromosomal sites, enables Pds5p mediate sister <br />                                                                      chromatid cohesion. <br /> of cell cycle, Mcd1p prominent levels           Two homologues Pds5p, BimD6 protein A. <br /> times bound chromosomes. The fact                 nidulans Spo76 protein Sordaria, been <br /> Pds5p binds chromosomes times Mcd1p               shown essential mitotic meiotic chromosome <br /> present consistent data showing Pds5p binding            segregation, respectively. BimD6 mutant cells exhibited <br /> to chromosomes dependent Mcd1p function.                       lethal defects mitotic chromosome segregation had <br />                                                                      fewer kinetochore microtubules wild-type cells (Den- <br />                                                                      ison et al., 1993). The Spo76p localizes meiotic chromo- <br /> Discussion                                                           somes length, spo76-1 mutant arrests <br /> We executed cytogenetic screen temperature                 meiosis exhibits defects sister chromatid cohe- <br /> sensitive mutants identify exhibited preco-            sion condensation (van Heemst et al., 1999). During <br /> cious dissociation sister chromatids. This screen led          mitosis, spo76-1 mutant transient defect in <br /> the identification pds1 mcd1 mutants (Yama-               sister chromatid cohesion condensation, ex- <br /> moto et al., 1996a,b; Guacci et al., 1997b). Here identi-         hibited growth abnormalities. These results indicate <br /> fied characterized PDS5, gene identified              Spo76p serve prominent nonessen- <br /> this screen. PDS5 essential gene vegetative                tial function mitosis spo76-1 allele has <br /> growth. Pds5p required sister chromatid cohesion           normal mitotic function. Alternatively, may <br /> centromere proximal centromere distal regions. Pds5p             Pds5p homologue Sordaria performs <br /> is required chromosome condensation re-             essential mitotic chromosomal function. Thus, Pds5p <br /> petitive unique chromosomal regions. Pds5 mutant                 homologues play important roles chromosome <br /> cells functional mitotic checkpoint, demon-             structure function eukaryotes. <br /> strates defects cohesion condensation              Previously, proposed cohesion condensa- <br /> due failure cell cycle arrest. The cell cycle stages         tion linked mcd1 mutants defects both <br /> when Pds5p required viability loaded chro-           cohesion condensation (Guacci et al., 1997b). The fact <br /> mosomes provides insights role cohesion. Pds5p           pds5 alleles defects cohesion and <br /> function important viability soon af-        condensation confirms link. Thus, link between <br /> ter S phase anaphase localizes chromosomes              cohesion condensation general property yeast <br /> during cell cycle window, encompasses                 chromosomes. Since, Pds5p Mcd1p homologues <br />  <br />  <br /> Hartman et al. Pds5p Is Required Cohesion Condensation       623 <br />  in eukaryotes, likely cohesion&#8211;conden-        cant, fundamental similarities structure the <br /> sation link conserved. Support <span id='am-58' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-60'  ></span><span id='am-61' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>idea</span>         mechanism cohesion condensation. <br /> comes Sordaria, spo76-1 mutant exhibited               Second, interactions cohesin condensin <br /> defects cohesion condensation (van Heemst et           complexes exist, missed. The cell cy- <br /> al., 1999). Unlike budding yeast, mitotic chromo-           cle window condensin cohesin complex mem- <br /> somes Sordaria detected individual rod-like           bers interact short involve small amount <br /> chromosomes reminiscent eukaryotic chromo-                total protein. In addition, putative interactions <br /> somes. The conservation chromosome structure eu-             occur proteins bound chromo- <br /> karyotes surprising chromosomal proteins              somes. Finally, interactions soluble com- <br /> involved condensation conserved eu-              plexes weak preserved immu- <br /> karyotes. This conservation likely reflects common          noprecipitations fractionations. In fact, biochemical <br /> chromosomal architecture required execute struc-         evidence yeast supports interaction cohesin <br /> tural changes eukaryotic chromosomes undergo dur-             condensin complex members. Smc1p Smc2p, <br /> ing mitosis.                                                       members cohesin condensin complexes, respec- <br />    It proposed cohesion condensation             tively, co-immunoprecipitate overex- <br /> are distinct chromosomal processes soluble             pressed (Strunnikov et al., 1995). In addition, other <br /> condensin cohesin complexes yeast, Xenopus,                 yeast proteins, Trf4p Hec1p, coimmunoprecipitate <br /> and mammals copurify temporally coexist               Smc1p Smc2p (Castano et al., 1996; Zheng <br /> chromosomes (Hirano et al., 1997; Losada et al., 1998;             et al., 1999). These results interactions between <br /> Schmiesing et al., 1998; Toth et al., 1999). Furthermore, im-      condensin cohesin members occur medi- <br /> munodepletion cohesin complex members interphase             ated common component(s). <br /> Xenopus egg extracts resulted cohesion defects                Finally, discrepancy simply reflect different <br /> chromosomes entered mitosis rod like chromosomes         aspects chromosome condensation monitored. To <br /> formed, suggested condensation unaffected           explain cohesion&#8211;condensation link, proposed that <br /> (Losada et al., 1998). These results contradict        Mcd1p-containing cohesin complexes cohesion site <br /> link cohesion condensation revealed analy-          associate cohesion site the <br /> sis budding yeast Sordaria mutants. There       chromosome (Guacci et al., 1997b). This loop out <br /> ways explain seemingly contradictory results.               intervening DNA chromosomal axis and <br />    First, chromosome structure funda-          shorten chromosomes. The DNA adjacent cohe- <br /> mentally different yeast higher eukaryotes. Several         sion sites (looped-out regions) packaged, pre- <br /> lines evidence make unlikely. The proteins required        sumably condensin complex. Thus, fully condensed <br /> for cohesion condensation conserved             mitotic chromosomes require interaction co- <br /> eukaryotes (reviewed Koshland Strunnikov, 1996;             hesion proteins adjacent cohesion sites chro- <br /> Koshland Guacci, 2000). In addition, cohesin complex           mosome condensin-mediated packaging inter- <br /> members present chromosomes similar cell cycle           vening DNA loops (Fig. 8 A). The association adjacent <br /> stages budding yeast, Xenopus, human cells. Co-             cohesion sites occur loop packaging after <br /> hesin complex members bound chromosomes                  packaging loops brought adjacent cohesion sites <br /> during interphase remain chromosomes, albeit             close proximity. The mechanism different, end <br /> lower levels, mitosis (Losada et al., 1998, 2000;           result, shortening chromosomal axis, the <br /> Schmiesing et al., 1998; I. Waizenegger, S. Hauf, J.M.         same. If Mcd1p Pds5p function perturbed, inter- <br /> Peters, unpublished observations). A similar pattern            actions adjacent cohesion sites chroma- <br /> seen budding yeast (see below). Budding yeast               tid altered result axial shortening <br /> mammalian chromosomes undergo cell cycle&#8211;dependent                 (Fig. 8 B). However, loops packaged the <br /> condensation, yeast exhibits smaller level           condensin complex factors rod-shaped <br /> condensation (Guacci et al., 1994). Cohesin binding sites          chromosomes form longer nor- <br /> have mapped large chromosomal regions                 mal metaphase chromosomes. We used FISH assess the <br /> budding yeast recent PCR-based chromosomal walk-               length spacing DNA sequences individual <br /> ing revealed sites occur 8&#8211;9 kb non-       chromosome arms overall length and <br /> centromeric regions (Blat Kleckner, 1999; Laloraya,            spacing increased pds5 mcd1 mutants (this <br /> S., V. Guacci, D. Koshland, manuscript submitted           study Guacci et al., 1997b). In contrast, formation <br /> publication). Cohesin binding sites mapped           rod-like chromosomes taken evidence con- <br /> in higher eukaryotes, likely sites      densation unaffected immunodepletion co- <br /> more widely spaced metaphase chromosomes appear              hesin complex Xenopus extracts (Losada et al., <br /> to contain 50&#8211;250-kb DNA loops (Earnshaw Laem-                 1998). However, axial length individual chromo- <br /> mli, 1983; Gasser et al., 1986). Thus, likely cohesin   somes measured, leaves open possibility <br /> binding sites distributed eukaryotic chromo-             axial shortening chromosomes actually per- <br /> somes, different spacing budding yeast             turbed immunodepletion. Clearly, questions <br /> higher eukaryotes. Such decreased spacing (i.e., higher          answered cohesion&#8211;condensation link <br /> density) cohesion sites proposed explain             fully understood. <br /> lower levels chromosome condensation budding                    In Xenopus, cohesin complex members present at <br /> yeast (Guacci et al., 1997b). In summary, chromosomes              high levels chromosomes interphase, decrease to <br /> from budding yeast higher eukaryotes signifi-             low levels chromosomes mitosis (Losada et al., 1998, <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 151, 2000                      624 <br />                                                                      parts chromosomes metaphase anaphase <br />                                                                     transition, results dissolution cohesion <br />                                                                     (Michaelis et al., 1997; Uhlmann et al., 1999). It also <br />                                                                     tempting speculate loss Pds5p occurs the <br />                                                                     metaphase anaphase transition crucial dis- <br />                                                                     solution cohesion. However, chromosomes decon- <br />                                                                     dense late anaphase Pds5p Mcd1p re- <br />                                                                     quired condensation (this study; Guacci et al., 1994, <br />                                                                     1997b). Therefore, essential correlate precise <br />                                                                     time Pds5p Mcd1p depart chromosomes <br />                                                                     state chromosome condensation. Unfortu- <br />                                                                     nately, budding yeast, metaphase anaphase tran- <br />                                                                     sition easily detected individual chro- <br />                                                                     mosomes visible conventional microscopy. <br />                                                                     Yeast cells S phase midmitosis round <br />                                                                     DNA mass short spindle. Just metaphase to <br />                                                                     anaphase, centromere proximal sequences sister chro- <br />                                                                     matids rapidly reach opposite poles prominent <br />                                                                     elongation DNA mass spindle occurs yeast <br />                                                                     (Guacci et al., 1997a; Straight et al., 1997). It until <br />                                                                     metaphase anaphase transition, cells <br />                                                                     elongated spindles DNA masses, one <br />                                                                     sure sister chromatids separated. It in <br />                                                                     late anaphase cells Pds5p and <br />                                                                     Mcd1p detected chromosomes (this study; <br /> Figure 8. Model chromosome condensation. (A) Condensa-          Michaelis et al., 1997; Uhlmann et al., 1999). Therefore, it <br /> tion wild-type cells. (Top) Decondensed chromosome sis-     known departure Pds5p Mcd1p <br /> ter chromatid cohesion. Chromosome condensation begins ei-       occurs precise time sister cohesion dissolved <br /> ther packaging DNA cohesion sites (middle left)       (the metaphase anaphase transition) oc- <br /> by association adjacent cohesion sites (middle right). The   curs later mitosis chromosomes decondense (late <br /> fully condensed mitotic chromosome (bottom) requires       anaphase telophase). Until precise timing deter- <br /> association adjacent cohesion sites packaging     mined departure Pds5p Mcd1p chro- <br /> DNA cohesion sites. (B) Condensation pds5 mcd1        mosomes, effect departure chromo- <br /> mutants. (Top) Decondensed chromosome dissociated sister <br />                                                                     structure remains open question. <br /> chromatids. When chromosomes undergo condensation, the <br /> DNA cohesion sites packaged, adjacent         Our finding Pds5p molecular glue re- <br /> cohesion sites associate.                                    sponsible maintenance sister chromatid cohe- <br />                                                                     sion demonstrates cohesin complex proteins <br />                                                                     represent mechanism mediates co- <br /> 2000). Pds5p Mcd1p present high levels yeast          hesion. Further characterization Pds5p relation- <br /> chromosomes S mid-M phase (this study; Michaelis             ship cohesin complex provide valuable insights <br /> et al., 1997). Our qualitative analysis Pds5p Mcd1p          cohesion maintained ultimately dissolved. <br /> levels chromosome spreads suggests levels            Towards end, examining Pds5p a <br /> these proteins bound chromosomes decrease               new member cohesin complex, loads <br /> from S mid-M phase. While result con-               chromosomes complex added after- <br /> firmed quantitative methods, suggests            wards. We testing genetic interactions between <br /> yeast Xenopus, chromosomal levels proteins              Pds5p cohesin condensin complex members in <br /> mediate sister chromatid cohesion decrease mitosis.          yeast help dissect link cohesion con- <br /> Since chromosomes condensed mitosis, de-              densation. <br /> crease cohesin complex correlates condensation. It <br />                                                                     We thank P. Megee B. Lavoie comments manuscript. <br /> is possible decrease required proper <br />                                                                        D. Koshland supported Howard Hughes Medical Institute, <br /> chromosome condensation. Yeast chromosomes undergo                  V. Guacci, T. Hartman, K. Stead supported National <br /> less condensation mitosis mammalian chromo-             Institutes Health grant CA06927 Commonwealth Pennsyl- <br /> somes (Guacci et al., 1994). Therefore, possible     <span id='am-30' about='obo:NCBITaxon_165384' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_1394505 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Vania" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-34'  ></span>vania</span>. <br /> more dramatic decrease cohesin complex Xenopus <br />                                                                     Submitted: 28 June 2000 <br /> may consequence higher level chromosome <br />                                                                     Revised: 25 September 2000 <br /> condensation occurs budding yeast.                          Accepted: 25 September 2000 <br />    Pds5p bound chromosomes mid-M phase cells <br /> (nocodazole arrested), sister cohesion present, but <br />                                                                     References <br /> is absent chromosomes late anaphase, sisters <br /> are separated. Similar results observed Mcd1p              Blat, Y., N. Kleckner. 1999. Cohesins bind preferential sites yeast <br />                                                                        chromosome III, differential regulation arms versus centric <br /> (this study; Michaelis et al., 1997; Uhlmann et al., 1999). It         region. Cell. 98:249&#8211;259. <br /> has suggested Mcd1p/Scc1p cleaved de-              Castano, I., P. Brzoska, B. Sadoff, H. Chen, M. Christman. 1996. Mitotic <br />  <br />  <br />  <br />  <br /> Hartman et al. Pds5p Is Required Cohesion Condensation      625 <br />     chromosome condensation rDNA requires TRF4 DNA topo-                 Moore, D.P., A.W. Page, T.T. Tang, A.W. Kerrebrock, T.L. Orr-Weaver. <br />    isomerase I Saccharomyces cerevisiae. Genes Dev. 10:2564&#8211;2576.                  1998. The cohesion protein MEI-S332 localizes condensed meiotic and <br /> Chuang, P.T., D.G. Albertson, B.J. Meyer. 1994. DPY-27: chromosome              mitotic centromeres sister chromatids separate. J. Cell Biol. 140:1003&#8211; <br />    condensation protein homolog regulates C. elegans dosage compensa-            1012. <br />    tion association X chromosome. Cell. 79:459&#8211;474.               Ouspenski, I.I., O.A. Cabello, B.R. Brinkley. 2000. Chromosome conden- <br /> Ciosk, R., M. Shirayama, A. Shevchenko, T. Tanaka, A. Toth, A. Shevchenko,            sation factor brn1p required chromatid separation mitosis. Mol. <br />    K. Nasmyth. 2000. Cohesin&#8217;s binding chromosomes depends sep-           Biol. Cell. 11:1305&#8211;1313. <br />    arate complex consisting Scc2 Scc4 proteins. Mol. Cell. 5:243&#8211;254.       Pringle, J.R., L.H. Hartwell. 1981. The Saccharomyces cerevisiae cell cycle. <br /> Denison, S.H., E. Kafer, G.S. May. 1993. Mutation bimD gene As-         In The Molecular Biology Yeast Saccharomyces: Life Cycle In- <br />    pergillus nidulans confers conditional mitotic block sensitivity DNA      heritance. J.N. Strathern, E.W. Jones, J.R. Broach, editors. Cold Spring <br />    damaging agents. Genetics. 134:1085&#8211;1096.                                          Harbor Laboratory, Cold Spring Harbor, NY. 97&#8211;142. <br /> Donze, D., C.R. Adams, J. Rine, R.T. Kamakaka. 1999. The boundaries         Saitoh, N., I.G. Goldberg, E.R. Wood, W.C. Earnshaw. 1994. ScII: an <br />    silenced HMR domain Saccharomyces cerevisiae. Genes Dev. 13:698&#8211;            abundant chromosome scaffold protein member family puta- <br />    708.                                                                               tive ATPases unusual predicted tertiary structure. J. Cell Biol. 127: <br /> Earnshaw, W.C., U.K. Laemmli. 1983. Architecture metaphase chromo-             303&#8211;318. <br />    somes chromosome scaffolds. J. Cell Biol. 96:84&#8211;93.                         Saka, Y., T. Sutani, Y. Yamashita, S. Saitoh, M. Takeuchi, Y. Nakaseko, M. <br /> Funabiki, H., H. Yamano, K. Kumada, K. Nagao, T. Hunt, M. Yanagida.               Yanagida. 1994. Fission yeast cut3 cut14, members ubiquitous <br />    1996. Cut2 proteolysis required sister chromatid separation fission         protein family, required chromosome condensation segregation <br />    yeast. Nature. 381:438&#8211;441.                                                        mitosis. EMBO (Eur. Mol. Biol. Organ.) J. 13:4938&#8211;4952. <br /> Furuya, K., K. Takahashi, M. Yanagida. 1998. Faithful anaphase ensured      Schmiesing, J.A., A.R. Ball, H.C. Gregson, J.M. Alderton, S. Zhou, K. <br />    Mis4, sister chromatid cohesion molecule required S phase          Yokomori. 1998. Identification distinct human SMC protein com- <br />    destroyed G1 phase. Genes Dev. 12:3408&#8211;3418.                                    plexes involved mitotic chromosome dynamics. Proc. Natl. Acad. Sci. <br /> Gasser, S.M., T. Laroche, J. Falquet, E. Boy la Tour, U.K. Laemmli.            USA. 95:12906&#8211;12911. <br />    1986. Metaphase chromosome structure. Involvement topoisomerase II.          Selig, S., K. Oku <br /> </body></html>