<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A Network Conserved Damage Survival Pathways <br /> Revealed Genomic RNAi Screen <br /> Dashnamoorthy Ravi1, Amy M. Wiles1, Selvaraj Bhavani1, Jianhua Ruan2, Philip Leder3, Alexander J. R. <br /> Bishop1,3* <br /> 1 Greehey Children&#8217;s <span id='am-34' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-38'  ></span>Cancer</span> Research Institute, University Texas Health Science Center San Antonio, San Antonio, Texas, United States America, 2 Department of <br /> Computer Science, University Texas San Antonio, San Antonio, Texas, United States America, 3 Harvard Medical School, Department Genetics, Harvard <br /> University, Boston, Massachusetts, United States America <br />  <br />  <br />  <br />      Abstract <br />      Damage initiates pleiotropic cellular response aimed cellular survival appropriate. To identify genes required for <br />      damage survival, used cell-based RNAi screen Drosophila genome alkylating agent methyl <br />      methanesulphonate (MMS). Similar studies performed model organisms report damage response involve <br />      pleiotropic cellular processes central DNA repair components, intuitive systems level view the <br />      cellular components required damage survival, interrelationship, contextual importance lacking. <br />      Further, comparing <span id='am-82' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-84'  ></span><span id='am-85' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> different model organisms, identification conserved presumably core survival <br />      components forthcoming. We identified 307 genes, representing 13 signaling, metabolic, enzymatic pathways, <br />      affecting cellular survival MMS&#8211;induced damage. As expected, majority pathways involved DNA repair; <br />      however, pathways diverse biological functions identified, including <span id='am-72' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-75'  ></span><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>TOR</span> pathway, <br />      transcription, translation, proteasome, glutathione synthesis, ATP synthesis, Notch signaling, equally <br />      important damage survival. Comparison genomic screen data <span id='am-7' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10'  ></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> revealed overlap <br />      enrichment individual genes <span id='am-56' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>, conservation pathways. To demonstrate functional <br />      conservation pathways, tested Drosophila mouse cells, pathway responding alkylation <br />      damage species. Using protein interactome, significant level connectivity observed Drosophila <br />      MMS survival proteins, suggesting higher <span id='am-59' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-60' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> relationship. <span id='am-51' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-54'  ></span><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> connectivity dramatically improved incorporating <br />      components 13 identified pathways network. Grouping proteins &#8216;&#8216;pathway nodes&#8217;&#8217; qualitatively <br />      improved interactome organization, revealing highly organized &#8216;&#8216;MMS survival network.&#8217;&#8217; We conclude that <br />      identification pathways facilitate comparative biology analysis direct gene/orthologue comparisons fail. A <br />      biologically intuitive, highly interconnected MMS survival network revealed incorporated pathway data our <br />      interactome analysis. <br />  <br />   Citation: Ravi D, Wiles AM, Bhavani S, Ruan J, Leder P, et al. (2009) A Network Conserved Damage Survival Pathways Revealed Genomic RNAi Screen. PLoS <br />   Genet 5(6): e1000527. doi:10.1371/journal.pgen.1000527 <br />   Editor: Amy Kiger, University <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>California</span> San Diego, United States America <br />   Received January 5, 2009; Accepted May 19, 2009; Published June 19, 2009 <br />   Copyright: &#223; 2009 Ravi et al. This open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported NIEHS (K22-ES12264; AJRB). JR supported NIH grant 1SC3GM086305-01. We thank Drosophila RNAi <br />   Screening Center (http://flyrnai.org/), supported NIGMS (2R01-GM067761-05). The funders role study design, data collection analysis, <br />   decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: bishopa@uthscsa.edu <br />  <br />  <br />  <br />  <br /> Introduction                                                                            function consequences dysfunction. <br />                                                                                         Recent reports suggest DDR involve pleiotropic cellular <br />    Cellular damage normal component life, constant                         processes central DDR components [3,4], an <br /> damage exposure endogenous exogenous sources.                             intuitive systems level view cellular components required for <br /> Damage DNA considered biologically relevant                        damage survival, interrelationship, contextual <br /> lesion potential mutagenic results,                             importance lacking. The comprehensive attempt at <br /> exogenous agents potential damage components                           understanding interrelationship damage response and <br /> of cell. Responding appropriately insults,                           survival components yeast systems level mapping <br /> mitigating cellular toxicity initiating appropriate cell death                    identified genes network integrating general biological <br /> response, critical, particularly multi-<span id='am-62' about='obo:NCBITaxon_131567' typeof='owl:Thing obo:NCBITaxon_1'><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="biota" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="cellular organisms" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cellular organisms</span>.                        processes distinct modules [5]. It suggested the <br /> Inappropriate responses facilitate deleterious effects,                     inclusion well-defined biological pathways protein networks <br /> a destabilized genome diseases cancer [1]. As such,                         provide better understanding biological interactions <br /> DNA damage response (DDR) components critical suppressors                           [6,7]. <br /> of deleterious effects genotoxic agents controlling cell cycle                       In order identify genes involved damage response an <br /> progression, DNA repair, apoptosis [2]. For reason,                      unbiased manner functional context, used <br /> have investigations using variety model organisms                        RNAi library [8] knock-down predicted protein the <br /> to identify components DDR subsequently determine                             <span id='am-44' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-45' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-46'  ></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Drosophila melanogaster</span> genome assessed knock-down of <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                        1                             June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                         Damage Survival Pathways <br />  <br />  <br />  <br />   Author Summary                                                           Results <br />   Cellular damage known elicit pleiotropic response,               Genome-wide RNAi screen validation <br />   relative importance constituent compo-                       An RNAi screen designed identify genes that <br />   nents cell survival poorly understood. To provide               modulate cellular survival following exposure level MMS- <br />   unbiased identification proteins utilized damage               induced damage resulted limited cell <br />   survival, performed RNAi survival screen fly cells              death. A linear decrease cell viability observed 0.002% <br />   methyl methanesulfonate (MMS). The genes identi-                    (w/v) MMS 0.008% (w/v) MMS (Figure S1). A dose of <br />   fied involved 13 biologically diverse pathways.                   0.004% (w/v; 40 mg/mL) MMS chosen, resulted a <br />   Comparison analogous yeast data demonstrated                      statistically significant decrease approximately 65% viability, <br />   lack conservation individual MMS survival genes                allowing additional, measurable decrease cell survival. <br />   conservation pathways. We went                            RNAi screens performed using Drosophila RNAi <br />   demonstrate MMS responsiveness representa-                  Screening Center (DRSC) version 1 library (about 23,000 D. <br />   tive pathways fly mouse cells. We conclude <br />                                                                            <span id='am-77' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span><span id='am-80'  ></span><span id='am-81' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>melanogaster</span> open reading frames) [8]. Kc167 Drosophila cells were <br />   identification pathways facilitate comparative <br />   biology analysis direct gene/orthologue compari-                    exposed days dsRNA allow protein knock-down, <br />   sons fail. Incorporation pathway data interactome                  followed days growth absence presence <br />   analysis improved connectivity and, importantly,               MMS exposure (Figure 1A). <br />   revealed biologically intuitive, highly inter-connected                   To identify genes required cell viability following <br />   &#8216;&#8216;MMS survival network.&#8217;&#8217; This pathway conservation                  MMS treatment, viability results MMS treated RNAi <br />   inter-connectivity implies extensive interaction                 screen compared untreated (control) screen <br />   pathways; diseases cancer, crosstalk                previously described [18]. 1,398 different open reading frames <br />   dictate disparate cellular responses necessarily expect-             identified affected MMS survival continuous <br />   ed confound treatments tailored                  distribution cell survival values (see [18] http://gccri. <br />   individual molecular context.                                            uthscsa.edu/ABPublished_Data.asp original data), which <br />                                                                            996 unique assigned FlyBase gene number (FBgn; denotes <br />                                                                            known genes) (see Figure 1B Table 1). Of 996 genes, the <br /> individual proteins significantly altered cell viability following         537 selected validation analysis previously <br /> methyl methanesulfonate (MMS) exposure. MMS prototypical              described, stringent validation method [18]. Whereas [18] we <br /> alkylating agent attacks nucleophilic groups,           validated normalization methods, independently validated <br /> found nucleic acid [9]. The resultant base methylation                  individual genes using dsRNA targeting different region each <br /> destabilizes glycosidic bond, facilitating production       gene. 202 protein knock-downs validated significant MMS <br /> most biologically relevant cellular lesion MMS, abasic site       viability effect, 55 notable trend effect without <br /> DNA. Other common sources alkylation damage include                     meeting stringent statistical criteria (Table S1 Table S2). <br /> endogenous S-adenoysyl methionine [10], tobacco carcinogen             An examination gene ontology (GO) 202 validated MMS <br /> N-nitorsoamine [11] chemotherapeutics temozolo-                survival genes revealed significant enrichment genes involved <br /> mide [12], carmustine (BCNU) [9], cyclophosphamide [13].               DNA metabolism, gene transcription, cellular metabolism <br /> Considering physiological relevance alkylation damage,              (Table 2). The overall variety GO categories observed quite <br /> several recent studies using yeast model organism investigated          broad, consistent findings reported analogous yeast <br /> the global effects alkylation damage induced MMS. Viability          screen [3] (data shown). Surprisingly though, significant <br /> [14], gene transcription [15] protein expression [16]             enrichment gene orthologues organisms <br /> measured studies, provided insights         observed (G-test p-values 0.29 0.057 Fisher Exact <br /> diverse nature biological responses alkylation damage.               test p-values 0.37 0.08, yeast fly, respectively). To <br /> However transcriptional responses various environmen-              test this, examined MMS sensitivity following <br /> tal stresses compared mammalian yeast cells               knock-down 183 fly genes orthologues 118 yeast <br /> [17], Murray et al. reported clear distinctions suggested         MMS survival genes [3], conservation MMS <br /> this result different selective pressures          survival phenotype 20 (Table S3). This apparent lack gene <br /> multi-cellular organisms single-cell organisms. In                 enrichment species reported when <br /> attempt understand diverse responses required damage                comparing transcriptional profiles mammalian yeast <br /> survival evolutionarily closer mammals              cells response variety stresses [17]. Overall, results <br /> than yeast, performed genome-wide, RNAi-based screen                  suggest conservation biological processes <br /> using cells derived Drosophila melanogaster. Our experiments          respond MMS individual genes. <br /> were based &#8216;&#8216;loss function&#8217;&#8217; assessment cellular <br /> viability following exposure MMS. Here present results               Identification pathways responsive MMS <br /> from MMS survival screen, pathways                         Assuming biological function informative <br /> identified, comparative analysis pathways understand           measure damage response requirement individual <br /> conservation pathways yeast mouse cells.                   genes, endeavoured identify MMS survival <br /> Additionally, present novel approach assembling                  proteins known signaling, metabolic, enzymatic <br /> protein interaction network based defined biological path-              pathways. Using priori knowledge KegArray, we <br /> ways, facilitates network consolidation. By including                identified 13 pathways included 41 MMS survival <br /> biological pathways genomic data protein network                proteins (Figures 1, 2, 3 Figures S2, S3, S4, S5), them, <br /> analysis, able determine commonality                 DNA repair pathways (Base Excision Repair; Nucleotide <br /> species, obscured direct orthologue comparison,               Excision Repair; Mismatch Repair; Homologous Recombination <br /> and provide simplified representation global survival             Repair; RECQ). Many pathways statistically <br /> responses identified.                                              significant number MMS survival proteins (Table S5), this <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                             2                            June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                           Damage Survival Pathways <br />  <br />  <br />  <br />  <br /> Figure 1. The experimental design Drosophila RNAi MMS survival screen summary workflow results. (A) A <br /> schematic representation experimental design used screen genes involved survival MMS exposure using RNAi Drosophila. (B) A <br /> schematic representation workflow initial RNAi screen, normalization validation results, follow-up work conducted, <br /> with summary number dsRNA amplicons genes (FBgn; FlyBase gene number) involved step process. Also provided are <br /> the final numbers MMS survival proteins MMS survival pathways (BER: base excision repair; NER: nucleotide excision repair; MMR: mismatch <br /> repair; HRR: homologous recombination repair; RECQ: RecQ helicases; DDR: DNA damage response; Proteasome; GSH: glutathione synthesis; TOR: <br /> TOR pathway; Transcription: basal transcription; Ribosome; ATPase; Notch: Notch signaling pathway). <br /> doi:10.1371/journal.pgen.1000527.g001 <br />  <br />  <br /> is accounting pathway &#8216;&#8216;branching&#8217;&#8217; &#8216;&#8216;subdivision.&#8217;&#8217;          hypothesis identified pathways involved in <br /> Considering likely role pathways MMS survival,            MMS survival. Though result highlights possibility false <br /> used RNAi assay method, sensitive                  negative screen data, compared previous false negative rate <br /> stringent original screen [18], determine          23.6% randomly selected data, observe significant <br /> members 13 identified pathway affected MMS survival              enrichment false negatives pertinent pathways ((x2 <br /> (Table S1). By individually interrogating 346 pathway                    p = 3.9E-5). In total identified 146 MMS survival genes 13 <br /> members 13 pathways identified                   &#8216;&#8216;MMS survival pathways,&#8217;&#8217; encompassing 25% 86% non- <br /> original screen, additional 105 MMS survival genes                   essential genes pathway (Table S4). <br /> discovered. This observation significantly enriched number               Using genes mapped MMS survival pathways, we <br /> MMS survival genes 13 pathways (Figures 1, 2, 3               compared pathways identified Drosophila MMS <br /> and Figures S2, S3, S4, S5) provides additional support           screen analogous screen performed yeast [14] using <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              3                            June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                               Damage Survival Pathways <br />  <br />  <br />  <br />  Table 1. Summary MMS survival genes identified                              component XRCC1 resulted increased AP sites <br />  genome-wide RNAi screen.                                                          itself (p#3E-3), following MMS treatment, the <br />                                                                                    AP sites absence XRCC1 increased <br />                                                                                    (Figure 4A; compared untreated XRCC1 knock- <br />  Normalized RNAi screen results                                                    down, p#3.3E-8; compared MMS treated luciferase (Luc) <br />                                                                                    control, p#3.7E-3); MMS produces expected form <br />                      Death           Resistance              Total <br />                                                                                    DNA damage BER appropriately responds. p53, a <br />  BS_QN               1183            40                      1223                  central component DDR pathway, regulated at <br />  cellHTS2            594             40                      634                   expression, protein stabilization, posttranslational modifica- <br />                                                                                    tion levels. As expected, p53 gene expression increased response <br />  total               1326            72                      1398 <br />                                                                                    MMS exposure (Figure 4B; p#1.6E-3). <br />  FBgn                935             61                      996 <br />                                                                                       We examined MMS survival pathways not <br />  Numbers death resistance hits background subtraction              canonical DDR DNA repair process: glutathione <br />  followed quantile normalization (BSQN) cellHTS2 given. The total       metabolism, proteasome, TOR pathway. An increased <br />  number unique hits total number hits amplicons map         activity glutathione synthesis pathway following MMS <br />  genes FlyBase gene number (FBgn) given.                             exposure demonstrated measuring total glutathione <br />  doi:10.1371/journal.pgen.1000527.t001 <br />                                                                                    content milligram protein lysate (Figure 4C; p#3.4E-3), <br />                                                                                    examining expression rate-limiting enzyme <br /> genes responsive MMS. As previously noted,                     glutathione synthesis, GCLc [22] (p#8.8E-4). Therefore, as <br /> we did observe significant overlap screens                   expected, knock-down protein, GCLc, significantly <br /> when comparing MMS survival gene orthologues, however,                        reduced total glutathione present compared with <br /> this pathway comparison, mapped orthologues yeast MMS                        control (p#3.4E-3), knock-down prevented cells <br /> survival genes 10 13 Drosophila MMS survival pathways                    increasing level glutathione response MMS <br /> (Figures 2 3; Figures S2, S3, S4, S5; Table S6). The                     exposure (Figure 4C). Since glutathione synthesis considered to <br /> Drosophila MMS survival pathways yeast MMS survival                        oxidative stress response, demonstrated MMS <br /> genes conserved yeast                  resulted dose-dependent increase level 8-oxoguanine, <br /> pathway components essential viability yeast,                      DNA lesion normally associated oxidative damage (Figure <br /> making refractory MMS viability analysis (Table S5). The                   S7 Text S1). Thus appears MMS results only <br /> gene enrichment pathway species highly                        alkylation damage damage oxidative stress. <br /> supportive conservation processes involved MMS                             For analysis proteasome degradation pathway, we <br /> survival. A notable absentee Drosophila screen,                   measured proteasome activity increased <br /> observed yeast, 3-methyladenine-DNA-glycosylase (MAG1/                       following MMS treatment (Figure 4D; p#4.2E-2). Protein <br /> AlkA), principal protection MMS-induced DNA                            knock-down proteasome components Rpn2 Pros26.4 <br /> damage yeast, direct orthologues exist animals. One                   significantly reduced proteasome activity compared cells <br /> of glycosylase Drosophila act step               knock-down unexposed (Rpn2: <br /> in BER, Thd1, required survival                   p#1.7E-2; Pros26.4: p#3.7E-3) exposed MMS (Rpn2: <br /> MMS treatment. Furthermore, pathways observed                       p#1.4E-4; Pros26.4: p#7.0E-5). Following Rpn2 knock-down, <br /> in species, proteasome [19], TOR pathway [20],                    cells unable mount detectable increase proteasome <br /> and DNA repair pathways [21], shown                                activity following MMS exposure, unable to <br /> functionally responsive MMS yeast. Altogether, results                 demonstrate Pros26.4 knock-down (p#1.7E-3). Overall, it <br /> suggest conservation pathway function, individual genes,               appears MMS exposure results increased proteasome <br /> in response MMS species.                                                activity. <br />    To demonstrate identified Drosophila pathways                         Finally, investigate TOR pathway activity, S6K phosphor- <br /> functioning expected manner response MMS,                   ylation status monitored. TOR kinase phosphorylates <br /> selected examination &#8211; base excision repair (BER),                     S6K promote growth ribosome biogenesis is <br /> DDR, glutathione metabolism, proteolysis proteasomal degra-                     negatively regulated tumor suppressor protein Tsc1 [23]. <br /> dation (proteasome), TOR pathway. Two                             MMS exposure resulted dose-dependent decrease S6K <br /> pathways, BER DDR, expected play role MMS                          phosphorylation, suggesting inhibition TOR activity <br /> survival [9], selected apparent                    (Figure 4E). This MMS-induced decrease S6K phosphorylation <br /> breadth function non-canonical role damage                         dependent Tsc1 (Figure 4E). MMS exposure therefore <br /> survival. For pathway, appropriate functional assay                    elicited down-regulation growth promoting TOR <br /> chosen, possible, appropriate upstream MMS                             pathway, suggesting pathway coordinated <br /> survival protein pathway knocked-down (Figure                      component DDR similar observations yeast [20]. This <br /> S6A) demonstrate modulation MMS induced response.                        supports previously published observation Matsuoka et al. [4], <br />                                                                                    showed components mammalian TOR <br /> Functional conservation MMS survival pathways                                pathway phosphorylated following ionizing radiation expo- <br /> Drosophila mouse                                                               sure. In conclusion, results functional assays validate <br />    First, tested expected MMS survival pathways BER                     identification &#8216;&#8216;MMS survival pathways&#8217;&#8217; RNAi <br /> and DDR. MMS-induced DNA damage results production                          screening functionality pathways, lack <br /> of apurinic/apyrimidic (AP) sites, DNA damage typically                          thereof, affects MMS survival. <br /> repaired BER [9]. We quantified AP                         Having identified 13 Drosophila MMS survival pathways, we <br /> sites microgram genomic DNA following MMS treatment                         went investigate functional conservation mammals. <br /> and observed statistically significant increase compared                      Using primary mouse embryonic fibroblasts (MEFs), examined <br /> control (Figure 4A; p#7.6E-8). Knock-down BER                               pathways modulated following MMS <br />  <br />  <br />          PLoS Genetics | www.plosgenetics.org                                  4                          June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                                       Damage Survival Pathways <br />  <br />  <br />  <br /> Table 2. Categories Gene Ontology enrichment validated MMS survival genes. <br />  <br />  <br /> Gene Ontology enrichment <br />  <br /> Biological Process <br />  <br /> Adjusted P-value               GO Attribute <br />  <br /> ,0.001                         0006139: nucleobase, nucleoside, nucleotide nucleic acid metabolic process <br /> ,0.001                         0006351: transcription, DNA-dependent <br /> ,0.001                         0032774: RNA biosynthetic process <br /> ,0.001                         0016070: RNA metabolic process <br /> ,0.001                         0045449: regulation transcription <br /> ,0.001                         0065004: protein-DNA complex assembly <br /> ,0.001                         0016043: cellular component organization biogenesis <br /> ,0.001                         0051252: regulation RNA metabolic process <br /> ,0.001                         0006367: transcription initiation RNA polymerase II promoter <br /> ,0.001                         0006355: regulation transcription, DNA-dependent <br /> ,0.001                         0065003: macromolecular complex assembly <br /> ,0.001                         0008283: cell proliferation <br /> 0.001                          0035080: heat shock-mediated polytene chromosome puffing <br /> 0.014                          0006911: phagocytosis, engulfment <br /> 0.018                          0045165: cell fate commitment <br /> 0.034                          0030154: cell differentiation <br /> 0.049                          0006259: DNA metabolic process <br />  <br /> Molecular Function <br />  <br /> Adjusted P-value               GO Attribute <br />  <br /> ,0.001                         0030528: transcription regulator activity <br /> ,0.001                         0003676: nucleic acid binding <br /> ,0.001                         0016251: general RNA polymerase II transcription factor activity <br /> ,0.001                         0003702: RNA polymerase II transcription factor activity <br /> 0.003                          0005515: protein binding <br /> 0.003                          0043565: sequence-specific DNA binding <br />  <br /> Cellular Component <br />  <br /> Adjusted P-value               GO Attribute <br />  <br /> ,0.001                         0005634: nucleus <br /> ,0.001                         0005654: nucleoplasm <br /> ,0.001                         0043227: membrane-bounded organelle <br /> ,0.001                         0031981: nuclear lumen <br /> ,0.001                         0016591: DNA-directed RNA polymerase II, holoenzyme <br /> ,0.001                         0031974: membrane-enclosed lumen <br /> ,0.001                         0043234: protein complex <br /> ,0.001                         0032991: macromolecular complex <br /> ,0.001                         0005667: transcription factor complex <br /> 0.003                          0000119: mediator complex <br /> 0.012                          0008540: proteasome regulatory particle, base subcomplex <br /> 0.015                          0005665: DNA-directed RNA polymerase II, core complex <br /> 0.032                          0005675: holo TFIIH complex <br /> 0.038                          0000123: histone acetyltransferase complex <br />  <br /> An examination 202 validated screen MMS survival genes category enrichment Gene Ontologies; Biological Process, 46 genes having functional <br /> annotation; Molecular Function, 40 genes having functional annotation; Cellular Component, 68 genes having cellular component annotation, using <br /> FuncAssociate. <br /> doi:10.1371/journal.pgen.1000527.t002 <br />  <br />  <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                    5                               June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                             Damage Survival Pathways <br />  <br />  <br />  <br />  <br /> PLoS Genetics | www.plosgenetics.org   6   June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                            Damage Survival Pathways <br />  <br />  <br />  <br /> Figure 2. Base excision repair, DNA damage response, proteasome pathways utilized MMS survival. CG numbers given for <br /> each Drosophila pathway component, protein names complex names human orthologues. Pathway entry points noted <br /> with Roman numerals top, end points bottom. A key following symbols provided. Symbols encircled lines <br /> represent proteins act complex, symbols encircled lines represent paralogues proteins substitute for <br /> one another. Proteins affect MMS survival noted (death) triangles (resistance). Statistically significant proteins are <br /> indicated black triangles, trend hits indicated grey triangles. Essential genes noted bar downwards <br /> or upwards pointing boxes validated conferring death resistance, respectively, MMS knock-down. Shaded squares are <br /> proteins hits validation, open squares tested validation. Yeast orthologues previously required <br /> for MMS survival [3] noted dot symbol. (A) Base Excision Repair. (B) DNA Damage Response. (C) Proteasome. <br /> doi:10.1371/journal.pgen.1000527.g002 <br />  <br /> exposure Drosophila. In general observed comparable results             activator (RBP-Jk) [26], result demonstrates a <br /> in MEFs Drosophila (compare Figures 4 5). MMS increased                functional Notch pathway response alkylation damage expo- <br /> the AP sites MEFs (Figure 5A); absence                 sure. Taken together, data suggest functional conservation <br /> XRCC1, however, observed significant increase AP sites                MMS survival pathways responses similarly <br /> following MMS exposure (Figure 5A; p#3.7E-4). For DDR                      acting agents. <br /> MEFs, assessed phosphorylation status Chk1 (a kinase <br /> that, phosphorylated, phosphorylates p53) [24], total p53                Network integration MMS survival pathways <br /> protein, phosphorylation status p53 (Figure 5B),               Of 307 identified MMS survival genes (202 validated screen <br /> examine p53 expression levels, obtain result equivalent           hits plus 105 pathway analyses), able assign 146 to <br /> that Drosophila result: DDR activated MMS exposure.             13 pathways (Figures 1, 2, 3 Figures S2, S3, S4, S5). Because <br /> Also, similar observation Drosophila cells, MEFs                161 MMS survival protein remain unassigned, re-examined the <br /> increased proteasome activity following MMS treatment                         inter-relationship identified MMS survival proteins <br /> (Figure 5D; p#2.0E-4); knock-down proteasome                        using currently available Drosophila protein:protein interactome <br /> component, PSMC1 PSMD1 (orthologues Rpn2                            (PPI) map [27&#8211;29]. To measure connectivity MMS <br /> Pros26.4, respectively), resulted decreased proteasome activity            survival genes, took largest connected components the <br /> following MMS exposure (PSMC1: p#8.1E-5; PSMD1: p#4.1E-                       PPI network discarded other, smaller components. The <br /> 4). These results comparable Drosophila: clear            largest connected component contained 7364 7504 proteins <br /> proteasome activity responsive MMS MEFs. It              original PPI network. Taking 202 original validated <br /> be noted that, able demonstrate dose-              MMS screen hits unbiased starting point, determined the <br /> dependent decrease S6K phosphorylation following MMS                       number proteins directly connected another, <br /> exposure MEFs seen Drosophila cells (Figure 5E)             compared random set number proteins from <br /> knock-down TSC1 disrupted MMS-induced effect,                     PPI, observed significant enrichment (Figure 6A and <br /> disruption evident observed Drosophila, probably           Table S7; p#2.1E-10). Similar results obtained other <br /> due inefficient knock-down TSC1 protein MEFs (data               assessments network connectivity (Table S7). Because MMS <br /> not shown). Having observed functionality pathways            screen hits connections average set random <br /> response MMS ability siRNA modulate                 genes, biased analysis, also <br /> the MMS response respective pathway, examined                compared connectivity MMS screen hits real PPI <br /> the effect knock-down MMS survival MEFs. Three                  network connectivity set proteins a <br /> of protein knock-downs, GCLC, PSMC1 PSMD1,                            randomly rewired PPI network. Degrees preserved the <br /> affected MMS survival MEFs (data shown). Overall,                   random rewiring process. As shown Table S7, MMS screen <br /> would appear Drosophila used accurately predict              hits statistically significantly direct interactions than <br /> MMS survival pathways mouse.                                               expected randomly rewired network (p = 0.01). On <br />                                                                               hand, randomized network smaller average <br /> Utilization MMS survival pathways response                       distance higher global efficiency real network, which <br /> Temozolomide                                                                  attributed well-known property real-world <br />    Given MMS causes alkylation damage,                       networks: usually slightly longer average distances (and <br /> reasonable suppose identified biological pathways                  correspondingly, lower global coefficients) degree- <br /> generally involved response alkylating agents.                preserving randomly rewired counterparts [30]. <br /> Temozolomide alkylating agent used clinically                       Because unable assess effect essential proteins <br /> treatment cancer [12,25]. It demonstrated                 (proteins knock-down resulted cell death regardless of <br /> that BER TOR necessary Temozolomide survival                      treatment) MMS survival, repeated connectivity analysis <br /> [12,25]. Temozolomide causes DNA damage increasing                     including essential proteins connected two <br /> number AP sites, similar MMS results, inhibition             MMS survival proteins, ones most <br /> BER increases sensitivity temozolomide [12]. It              likely functional role MMS survival. This increased <br /> reported rapamycin inhibition TOR pathway increased               size significantly improved connectivity the <br /> cellular sensitivity temozolomide [25]. To demonstrate             resultant network (Figure 6B Table S7; p#1.3E-26). After <br /> the utilization MMS survival pathways response                       pathway analysis, identified additional 105 MMS pathway <br /> temozolomide, examined DDR pathway, glutathione                        hits, validated. In order expand network to <br /> levels, proteasome activity MEFs observed                 include proteins relevant pathways, hits were <br /> responses observed following MMS exposure (Figure S8).                     incorporated larger, equally well-connected <br /> Further, using HEK293 cells, observed MMS                    network observed (Figure 6C Table S7; p#4.6E-26). <br /> temozolomide exposure repressed Notch reporter activity (Figure               Considering apparent importance pathways the <br /> S9 Text S1). Absence functional Notch protein                 individual genes, included components 13 <br /> shown repress activity downstream transcriptional                 identified MMS survival pathways observed the <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                               7                            June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                                 Damage Survival Pathways <br />  <br />  <br />  <br />  <br /> Figure 3. Glutathione TOR pathways utilized MMS survival. CG numbers given Drosophila pathway component, well <br /> as protein names complex names human orthologues. Pathway entry points noted Roman numerals top, end <br /> points bottom. A key following symbols provided. Symbols encircled lines represent proteins act a <br /> complex, symbols encircled lines represent paralogues proteins substitute another. Proteins affect MMS <br /> survival noted (death) triangles (resistance). Statistically significant proteins indicated black triangles, trend hits are <br /> indicated grey triangles. Essential genes noted bar downwards upwards pointing boxes validated as <br /> conferring death resistance, respectively, MMS knock-down. Shaded squares proteins hits validation, open <br /> squares tested validation. Yeast orthologues previously required MMS survival [3] noted dot the <br /> symbol. (A) Glutathione (an expanded list GST <span id='am-31' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> members given Table S4) (B) TOR. <br /> doi:10.1371/journal.pgen.1000527.g003 <br />  <br />  <br /> connectivity resultant enlarged network significantly                 essential genes pathway hits randomly rewired <br /> improved (Figure 6D Table S7; p#4.6E-52). Similar                  network consisting nodes. Although lower <br /> to analysis 202 validated MMS screen hits,                    statistical significance, general trend observed (Table <br /> compared connectivity subnetworks containing                      S7). This inclusive network allows view interactions <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                  8                             June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                                Damage Survival Pathways <br />  <br />  <br />  <br />  <br /> Figure 4. An examination functionality of pathways utilized MMS survival Drosophila cells. (A) MMS exposure results in <br /> an increase AP sites. dsRNA knock-down BER component XRCC1 increases quantity AP sites observed following MMS exposure. (B) MMS <br /> exposure results increased p53 expression quantitative real-time PCR analysis; expression level provided fold change compared an <br /> endogenous control (CG6905). (C) MMS results increased total glutathione, increase dependent rate limiting <br /> glutathione metabolizing enzyme glutamate cysteine ligase (GCLc). The intracellular glutathione concentration expressed mmoles/L normalized per <br /> mg protein. This GCLc dependency correlates increased level GCLc expression observed following MMS exposure. (D) Proteasome activity is <br /> increased following MMS exposure manner dependent proteasome components Rpn2 Pros26.4. Proteasome activity expressed as <br /> units activity/cell, normalizing activity number cells using parallel viability assessment. (E) MMS exposure results dose-dependent <br /> decrease phosphorylation dTor component p70S6K. This decrease dependent negative regulator dTor, Tsc1. Statistically <br /> significant differences unexposed MMS exposed cells denoted (*; **; ***; p-values ,0.05; ,0.01; ,0.001 respectively) are <br /> differences Luc control dsRNA knock-down (#; ##; ###; p-values ,0.05; ,0.01; ,0.001 respectively). <br /> doi:10.1371/journal.pgen.1000527.g004 <br />  <br />  <br />  <br />  <br /> Figure 5. Functional conservation mouse embryonic fibroblasts pathways utilized MMS survival in <br /> Drosophila cells. (A) MMS exposure results increase AP sites, siRNA based XRCC1 knock-down results increased quantity AP sites <br /> following MMS exposure. (B) MMS exposure results phosphorylation CHK1 p53 accumulation total p53 levels. (C) MMS exposure <br /> results increased total glutathione increase partially dependent GCLc. The intracellular glutathione concentration is <br /> expressed mmoles/L normalized mg protein. This GCLc dependency correlates increased level GCLc expression observed following <br /> MMS exposure. (D) Proteasome activity increased following MMS exposure manner dependent proteasome components Pmsc1 <br /> and Psmd1. Proteasome activity expressed units activity/cell, normalizing activity number cells using parallel viability <br /> assessment. (E) MMS exposure results dose-dependent decrease phosphorylation p70S6K. Statistically significant differences between <br /> unexposed MMS exposed cells denoted (*; **; ***; p-values ,0.05; ,0.01; ,0.001 respectively) differences NT <br /> control siRNA knock-down (#; ##; ###; p-values ,0.05; ,0.01; ,0.001 respectively). <br /> doi:10.1371/journal.pgen.1000527.g005 <br />  <br />        PLoS Genetics | www.plosgenetics.org                                 9                             June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                              Damage Survival Pathways <br />  <br />  <br />  <br />  <br /> PLoS Genetics | www.plosgenetics.org   10   June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                             Damage Survival Pathways <br />  <br />  <br />  <br /> Figure 6. Protein network MMS survival proteins reveals integrated, highly connected MMS survival network. (A) 150 the <br /> validated MMS survival proteins (red nodes) represented known Drosophila interactome, protein:protein interactions represented by <br /> grey edges; 104 validated hits &#8216;&#8216;orphans,&#8217;&#8217; connected proteins. (B) Nodes orphans (A), including 11 proteins <br /> essential viability (black nodes) connected validated hits, improves connectivity DNA damage survival network. <br /> In network, 93 hits orphans larger network 36 proteins, containing 29 validated hits, observed. (C) Nodes orphans from <br /> (B), including additional pathway hits screen (green nodes). In network, 86 screen hits additional 54 pathway hits orphans <br /> and larger network 63 proteins, containing 33 validated screen hits, 20 additional pathway hits 10 essential proteins. (D) Nodes orphans <br /> from (C), including non-hit members 13 MMS survival pathways (open nodes). In network, 60 screen hits additional 40 pathway <br /> hits orphans larger network 247 proteins, containing 79 validated screen hits (26 pathways), 35 additional pathway <br /> hits, 101 non-hit pathway proteins, 32 essential proteins. (E) Network proteins validated required MMS treatment (red nodes), <br /> proteins essential viability (black nodes), 13 pathways (blue pathway nodes) validated involved MMS survival. All proteins each <br /> pathway, regardless hit, represented metanode, node size representing protein number. After pathway components are <br /> used create network, 54 proteins orphan, 96 proteins contained main interconnected network. Network <br /> connectivity coefficients given lower right-hand corner (A&#8211;D). <br /> doi:10.1371/journal.pgen.1000527.g006 <br />  <br /> both connecting pathway, pathway                   recent studies demonstrated role proteasome in <br /> components critical survival MMS. To               regulating DNA repair pathways (reviewed [34]), <br /> qualitatively visualize result, protein known             supporting observation increased proteasome activity in <br /> pathway grouped assigned &#8216;&#8216;pathway node&#8217;&#8217;                response MMS. Thus, screen pathway identification have <br /> (a single node interactome retains interactions            revealed conserved set MMS survival pathways. <br /> of constitutive proteins proteins external                Our Drosophila based study provided novel insights the <br /> pathway). This resulted highly connected interactome,                  global cellular response alkylation damage identifying <br /> &#8216;&#8216;MMS survival network&#8217;&#8217; (Figure 6E), encompassed 179                 biological pathways functions required survival <br /> of 233 MMS survival proteins present PPI                   damage. The analogous genome-wide, loss-of- <br /> network (77%). Of remaining 54 orphan proteins, 47                function screen MMS survival genes performed yeast <br /> one protein removed MMS survival network,                 [14]. That study highlighted general biological processes <br /> proteins away, connected all. At step           required MMS survival based gene ontology integrated <br /> this analysis, observed improved network, enlarged             identified proteins disorganized network [14]. <br /> better connected, comparing entire set                    Considering pathways, signaling, metabolic, or <br /> proteins network random set proteins PPI,           enzymatic, long identifiable entities, logical to <br /> also randomizing additional proteins added step               consider units network. Thus experiments <br /> (data shown). Overall, observations MMS survival              focused identifying pathways required exposure to <br /> network suggest despite general interconnectivity              damage validating biological responsiveness pathways <br /> protein interactomes, pathway analysis highly relevant            following damage exposure. Our results confirmed these <br /> it improve interactome connectivity simplifies systems            biological pathways functional yeast, Drosophila, and <br /> biology overview.                                                              mouse cells functional contribution these <br />                                                                                biological pathways pertinent damage response network <br /> Discussion                                                                     representation. <br />                                                                                   In addition functional conservation survival <br />    Studies performed using yeast model organism predict                pathway response alkylation damage, biological <br /> network response MMS reported astounding involvement                     pathways appear roles response types of <br /> of diverse biological pathways [31]. Considering genes                damages. It interesting compare results elegant <br /> that respond environmental stress differ mammalian                  study Matsuoka et al. [4], identified proteins are <br /> and yeast cells [17], presumed damage response                   phosphorylated following ionizing radiation human cells. Their <br /> be different complex higher eukaryotes, especially                  study identified proteins components of 13 <br /> considering presence paralogues increased                      MMS survival pathways, including DNA repair <br /> genetic redundancy. We performed genome-wide,                      pathways, DDR, mTOR, proteasome, basal transcription, and <br /> RNAi based screen Drosophila cells investigate                   ribosome. These results suggest different types damage, not <br /> genes essential survival following damage exposure                just alkylation damage, utilize different components DNA <br /> MMS. We able identify validate 307 MMS survival                    damage survival network functionally conserved manner and <br /> genes, majority previously associated                reemphasize functional conservation pathways, the <br /> with alkylation damage survival. Of genes, 146                      individual genes, species. <br /> components 13 different MMS survival pathways. With                        Our emphasis reorganize MMS survival network based <br /> five pathways examined detail, observed                   pathways facilitate observation biologically <br /> functionally conserved yeast conserved mouse                relevant interactions. Often protein:protein interaction networks <br /> with regard utilization following MMS treatment                       appear chaotic, interrogated simple sub- <br /> (Figure 5). In yeast, experimental validation response                   networks associated protein(s) pathways [21]. <br /> MMS proteasome [19], TOR pathway [20], DNA                          However, working pleiotropic responses that <br /> repair pathways [21] previously reported. Similarly,                       encompass different biological processes, as <br /> glutathione response MMS observed mammalian cells                    DDR, chaotic network representation appears non-intuitive <br /> [32], observation increase GCLc expression                    (Figure 6D). Thus, integration pathways the <br /> provides underlying mechanism phenomenon. Our                      conceptual framework systems biology networking logical. <br /> demonstration dose-dependent increase 8-oxoguanine               An additional advantage including pathways highlighted by <br /> MMS exposure indicates MMS results oxidative stress               demonstration MMS survival protein enrichment by <br /> damage, previous studies suggested [33]. Additionally,              detailed examination pathways. Even detailed <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                               11                           June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                        Damage Survival Pathways <br />  <br />  <br /> analysis convincing evidence pathways               Methods <br /> functioning expected, able assign protein <br /> within pathway &#8216;&#8216;MMS survival protein.&#8217;&#8217; There                Tissue culture <br /> many possible explanations this, nonetheless, considering               Kc167 cells grown Schneider medium (Invitrogen, <br /> the pathways provide framework                       Carlsbad, CA) supplemented 10% heat inactivated fetal <br /> network highlights novel interactions, cross-talk,                  bovine serum (65uC, 10 minutes), penicillin streptomycin at <br /> identified proteins mapped canonical pathway,                   22uC humidified chamber. <br /> present network, unlikely observed                     Primary MEFs obtained harvesting 14.5 day C57BL/6 <br /> &#8216;&#8216;chaotic network view&#8217;&#8217; encourages investigation. This            embryos previously described [48]. Briefly, fetal liver head <br /> approach similar computational clustering networks                  removed remainder embryo mechanically <br /> based signaling pathways using interactome datasets [35],             disaggregated plating medium. A suspension single-cells was <br /> our approach includes identified proteins pathway                   plated Dulbecco&#8217;s Modified Eagle&#8217;s Medium (DMEM) <br /> components. Our representation simplified, using single                 supplemented 10% fetal calf serum, 2 mM glutamine, <br /> node represent entire pathway complex                   100 U/mL penicillin, 100 mg/mL streptomycin. MEFs were <br /> display interactions component pathway [36].               grown passages freezing aliquots. Aliquots were <br /> This simplified pathway inclusive representation reveals highly            taken expanded needed experiment. <br /> organized network, consistent requirement each <br /> pathway cellular survival (Figure 6E), provides                   Genome-wide viability screen following MMS exposure <br /> effective strategy integrate modular components                     Methods adapted [49], 1.26104 Kc167 cells <br /> network [37] inferring biological properties [38].               suspended 10 mL serum-free Schneider medium (Invitrogen) <br />    The interconnectivity survival pathways                       added sixty-three 384-well plates. Each well <br /> (Figure 5E) suggests potential pathway cross-talk. If cross-            contains 0.25 mg double-stranded (ds) RNA, with <br /> talk exists, highly pertinent cancer therapy.                 22,915 individual dsRNA represented library <br /> Recent studies demonstrated utility global level               (Version 1 dsRNA). Cells incubated 1 h 22uC, allowed <br /> analysis, allowing identification altered pathway function             phagocytic uptake dsRNA, 20 mL serum (15% heat- <br /> complex diseases Notch pathway pancreatic                     inactivated fetal bovine serum) containing medium added <br /> cancer [39] similarly DDR pathway breast                      plates incubated 72 h. Medium was <br /> colorectal cancers [40]. Considering identification Notch,            exchanged fresh serum containing medium, without <br /> TOR, DDR, proteasome &#8216;&#8216;survival pathways,&#8217;&#8217;                0.004% (w/v) MMS (Sigma-Aldrich, St. Louis, MO). Following <br /> which currently explored targets cancer therapy             72 h additional incubation, medium removed number <br /> [41,42], identified survival network suggest                   viable cells assessed using pre-diluted Celltiter-Glo per <br /> possibility combining pathway-specific pharmacological agents             manufacturers instructions (Promega, Madison, WI). Screens were <br /> in cancer therapy. Some pathway connections                       performed duplicate, replicate screen initiated a <br /> potential cross-talk represented survival network                     different day. <br /> (Figure 6E) observed. For example, protein <br /> phosphatase, PP2A, downstream component TOR                         Production dsRNA <br /> pathway [23], interacts DDR component regulate                      dsRNA produced similarly previously described [8,18]. <br /> phosphorylation ATM ATR [43] vice versa [4];                      Briefly, dsRNA synthesized using cDNA prepared gene <br /> DDR interacts BER CHK2 XRCC1 [44]; Notch                        specific amplification reverse transcribed cDNA. For cDNA <br /> interacts DDR Mastermind p53 [45];                          preparation, RNA harvested adult D. melanogaster using <br /> glutathione pathway interacts nucleotide excision repair            Trizol reagent. For gene specific amplification, primers designed <br /> pathway (NER) [46]; proteasome interacts various                DRSC target genes custom synthesized <br /> DNA repair components [34,47]. Together, appear                overhanging T7 promoter sequences (Invitrogen). DRSC <br /> our model integrated network conserved damage                       validation primer sequences obtained DRSC for <br /> survival pathways valid biologically relevant.                   targeting genes interest, oligonucleotides purchased <br />    In conclusion, identified network pathways              (Invitrogen). First round PCR amplifications performed using <br /> have functional role damage response affecting viability;         gene specific primers, amplified products gel <br /> also demonstrated functional conservation species             purified, using agarose gel purification kit (Qiagen, <span id='am-67' about='obo:NCBITaxon_195854' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_8087 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_28738 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_7898 obo:NCBITaxon_195853 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-68'  ></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Valencia" datatype="xsd:string"></span>Valencia</span>, <br /> the MMS survival pathways. By considering protein                        CA) 96-well gel isolation kit (Invitrogen). The purified PCR <br /> interactions MMS survival proteins incorpo-               products used second round amplification using <br /> rating MMS survival pathways, highly interconnected                    gene specific primers; amplified products purified <br /> damage survival network observed encompasses                using Millipore PCR purification 96-well plate (Millipore, <br /> 58% identified MMS survival proteins directly. This                   Billercia, MA). The purified products used dsRNA <br /> interconnectivity suggests strong functional interrelationship             synthesis using T7 Ribomax express (Promega) and <br /> between constitutive components survival network              purified using PCR purification plate (Millipore). The <br /> possibility pathway cross-talk coordination level greater        quality purified dsRNA verified agarose gel electropho- <br /> than just instigation DDR pathway. Although               resis and, following spectrophotometric quantification, stored at <br /> MMS survival pathways implicated MMS                    220uC. <br /> damage response, identified seemingly disparate <br /> pathways single screen, network analysis,             Validation analysis <br /> lends direct connection pathways response                    Validation MMS survival genes performed previously <br /> damage. The pleiotropic effects alkylation damage               described [18], exception DRSC &#8216;&#8216;validation&#8217;&#8217; <br /> require wide-variety functioning pathways order cell         dsRNA (targeting gene mRNA transcript different location <br /> to survive.                                                                  library amplicon), designed DRSC produced <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                              12                          June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                           Damage Survival Pathways <br />  <br />  <br /> in-house, preferentially used study. Briefly, dsRNA              Bradford&#8217;s reagent (Biorad). For estimation glutathione, equal <br /> were validated quadruplicate, set plates treated          amounts protein lysate de-proteinized using metapho- <br /> MMS set used untreated control. Plates                  sphoric acid (MPA) final concentration MPA adjusted <br /> without MMS dsRNA compared.                           5%. The precipitated proteins cleared centrifugation at <br />    Genes required MMS treatment genes knock-                  4000 g 10 min 4uC, supernatant used the <br /> down conferred resistance MMS validated validation                 assay. Reduced glutathione (Sigma) used standard, and <br /> amplicons, available. When validation amplicons                   samples arrayed 96-well plate. A two-step reaction was <br /> available, amplicon used original library used             conducted, thioesterification intracellular thiols using 4-choloro- <br /> validation. In instances, validation library                    1-methyl-7-trifluoromethyl-quinolinium methylsulfate followed by <br /> amplicons used validation experiments. In cases,                alkaline conversion glutathione-thioester chromophoric <br /> the knock-down result validation amplicon preferred              thione, followed detection total glutathione absorbtion <br /> over library amplicon, difference evident. If        400 nm. <br /> gene annotated pathway been <br /> determined involved MMS response, gene                       Quantitative Reverse-Transcriptase PCR <br /> considered hit regardless version amplicon validated.              dsRNA used target gene interest; dsRNA was <br />                                                                               used validate genes knock-down. The <br /> Gene ontology                                                                 level RNAi mediated silencing gene expression was <br />    Gene ontology retrieved FlyBase (http://flybase.org/).            monitored quantitative real time RT&#8211;PCR using QuantiTect <br /> Enrichment analysis conducted using FuncAssociate (http://                SYBR Green RT-PCR kit (Qiagen GmbH) ABI 7500 Real <br /> llama.med.harvard.edu/cgi/func/funcassociate/) ontol-               Time PCR System (Applied Biosystems, Foster City, CA). For <br /> ogies over-represented p-value 0.05. Those                experiments, 18.616104 Kc167 cells dispensed 24- <br /> ontology categories overlapped consolidated.                        tissue culture plate containing 6.2 mg dsRNA well. <br />                                                                               Following 1 h incubation serum-free condition allow the <br /> Westerns                                                                      uptake dsRNA, serum replenished final concentration <br />    Protein lysates prepared using radioimmunoprecipitation               10%. On Day 3, RNA isolated using RNeasy Mini kit <br /> assay (RIPA) buffer containing 5% sodium deoxycholate, 0.1%                   (Qiagen) quantified using ND-1000 Spectrophotometer <br /> SDS, 0.1% Igepal PBS cocktail protease inhibitors                (Nanodrop, Wilmington, DE). For PCR amplifications, distinct <br /> (1 mM PMSF, 1 mM Sodium Orthovandate 30 uL/mL                             primers encompassed dsRNA used to <br /> Aprotinin (Sigma)). Protein concentration determined using                target gene used. For gene expression analysis following <br /> Bradford Protein Assay Reagent (Biorad, Hercules, CA). Equal                  MMS exposure, RNA isolated Day 4. For experiments <br /> amounts protein resolved using 10% SDS polyacrylamide                 Kc167 cells, CG6905, expression remained <br /> gel transferred nitrocellulose membrane (Hybond-                   unaltered following MMS exposure, used endogenous <br /> ECL, GE Healthcare Lifesciences, Piscataway, NJ). Using                       control. For experiments MEFs, isolated RNA was <br /> appropriate dilutions primary secondary antibodies,                    reverse transcribed using ImProm-II Reverse Transcription <br /> immunodetection protein performed using ECL                    System (Promega) cDNA used TaqMan Gene <br /> plus (GE Healthcare Lifesciences). Primary antibodies                  Expression Assay kit GCLc, purchased (Applied <br /> include P-Chk1 (T68) (Abcam) total CHK1 (Abcam), P-                       Biosystems, CA) TaqMan Universal Master Mix (Applied <br /> p70S6K (T389) (Cell Signaling Technology) total p70S6K                    Biosystems). Mouse b-Actin TaqMan Gene Expression Assay kit <br /> (Cell Signaling Technology), P-p53 (S15) (Cell Signaling                  used endogenous control. The level gene expression <br /> Technology) total p53 (Cell Signaling Technology).                        determined using nnCt method [50]. <br />  <br /> Apurinic sites measurement                                                    Proteasome measurement <br />    Apurinic apyrmidinic sites (AP) generated following              Experiments performed 384-well plates using same <br /> DNA damage measured using colorimetric assay kit                   layout timing described RNAi validation <br /> DNA damage quantification (Oxford Biomedical Science, Oxford,                 experiments, [18], proteasome activity measured <br /> MI), following manufacturer&#8217;s protocol. Briefly, DNA isolated             Day 4 experiment, 24 h MMS exposure. For <br /> from experimental cells using Genomic DNA isolation kit                 proteasome measurement, Proteasome-Glo assay kit used <br /> (Oxford Biomedical Science). Equal amounts DNA                   (Promega), instructions. To normalize proteasome <br /> labelled using biotinylated aldehyde reactive probe. Labelling              activity cell density, parallel experiment performed a <br /> was followed purification colorimetric quantification using            separate 384-well plate, cell density estimated using <br /> streptavidin-horse radish peroxidase (HRP) conjugate HRP-                 CellTiter-Glo (Promega) described above. Proteasome activity <br /> dependent substrate supplied kit. The aldehyde reactive              normalized relative number cells present. <br /> probe labelled DNA standard, supplied kit, used to <br /> determine number AP sites 100 kilobase pairs DNA                Orthologue identification <br /> in experimental samples.                                                    Orthologues Drosophila genes human, mouse, <br />                                                                               yeast, obtained Ensembl49 (http://ensembl.org/). <br /> Glutathione measurement <br />    Intracellular glutathione levels measured using colori-             siRNA targeting MEFs <br /> metric assay kit glutathione (Oxford Biomedical Science),                    SMARTPool siRNAs purchased Dharmacon RNA <br /> following manufacturer&#8217;s protocol. Briefly, cells RNAi                   technologies (Lafayette, CO). For siRNA transfection MEFs, <br /> experiments resuspended ice-cold PBS homogenized                  cells harvested trypsinization, washed serum free <br /> using sonicator; lysate cleared centrifugation           medium, cell density adjusted 16106 cells 100 mL of <br /> protein concentration supernatant determined using                 MEF-2 Nucleofector Solution (Amaxa, Gaithersburg, MD), <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              13                           June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                                Damage Survival Pathways <br />  <br />  <br /> containing 0.5 mg siRNA, followed transfection                            itself) edges them. The clustering coefficient vi <br /> electroporation using Nucleofector II Device (Amaxa). The                          measured total number edges gi divided the <br /> transfected cells seeded 96-well 384-well                    maximum number edges possibly exist gi. The <br /> tissue culture plate, required experiment. For non-                      clustering coefficient network given average the <br /> specific control, scrambled, non-targeting siRNA used. The                     clustering coefficient vertex, high clustering <br /> fluorescent probe siGLO red (Dharmacon) used monitor                        coefficient indicating distinction real network from <br /> the efficiency siRNA uptake, efficiency protein                      random one. To assess statistical significance each <br /> knock-down determined western blot analysis (Figure S3B).                   measurement, number proteins proteins pairs, as <br /> Additional validation SMARTPool siRNA experiments                         appropriate, randomly sampled PPI PPI <br /> performed using minimum independent duplex siRNA                         subnetwork. This sampling repeated 1,000 times estimate <br /> for gene (Figure S10).                                                        p-value difference expected chance. <br />  <br /> Pathway identification                                                             Statistical analyses <br />    Pathway analysis conducted priori identification                 To determine knock-down genes resulted increased <br /> protein/pathway relationships KegArray, yielding                    sensitivity MMS, raw data obtained viability assays <br /> we included DNA damage response, glutathione metabolism,                       normalized statistically analyzed described previously <br /> TOR pathway, proteasome, DNA repair pathways. For                            [18]. A T-test performed normalized quadruplicates <br /> systematic analysis MMS hits, KegArray (with default                        determine significant difference treated and <br /> settings) used (http://genome.jp/download/), yielding Notch                    untreated wells. Percent control survival MMS treatment <br /> signaling, ATPase, basal transcription, ribosome, proteasome                   estimated experimental gene knock-down as <br /> glutathione metabolism. These pathways examined                       described previously [18], second T-test performed on <br /> number hits identified pathway versus number total                percent viability compared luciferase control within <br /> pathway components. Pathways included prior <br />                                                                                    plate described [18]. From analyses, <br /> knowledge pathways retrieved KegArray due <br />                                                                                    significant hits selected death hits greater <br /> to relatively small size completeness KEGG <br />                                                                                    55% viability treated wells compared untreated p- <br /> database fly. <br />                                                                                    value 0.05 resulted T-tests. For <br />                                                                                    genes essential phenotype 40% viability <br /> Interactome analysis                                                               non-targeting dsRNA, stringent requirements made <br />    Protein interactome data obtained IntAct [27]                         viability effect MMS treatment, 30&#8211;40% <br /> (http://www.ebi.ac.uk/intact/), Database Interacting                        viability untreated needed 15% viability MMS <br /> Proteins [28] (DIP, http://dip.doe-mbi.ucla.edu/) Bio- <br />                                                                                    treatment, 20&#8211;30% needed 10% treatment, 10&#8211;20% needed <br /> molecular Interaction Network Database [29] (BIND, http:// <br />                                                                                    5% treatment, 0&#8211;10% dead determine if <br /> bond.unleashedinformatics.com/). Data databases <br />                                                                                    MMS effect. Genes considered &#8216;&#8216;trend death hits&#8217;&#8217; if <br /> were combined single interactome, using CG number of <br />                                                                                    exhibited 55% viability treatment did not <br /> each gene identifier protein. Interactomes were <br />                                                                                    p-value significance exhibited 55&#8211;65% <br /> visualized using Cytoscape 2.6 (http://www.cytoscape.org/). <br />                                                                                    viability MMS treatment significant p-value. Knock- <br /> Pathway nodes created external Cytoscape renaming <br />                                                                                    genes resulting resistance MMS confirmed as <br /> all nodes representing protein pathways of <br />                                                                                    greater 85% viability treatment compared <br /> the pathway. If protein multiple pathways, is <br /> represented relevant pathway nodes. Interactions                    untreated wells, p-values 0. 0001 <br /> pathways trimmed interaction                    viability treatment 60% non- <br /> existed pathways. For instance, protein:protein                   targeting knock-down. <br /> interaction occurs members NER pathway,                            To determine gene essential viability, T-test was <br /> and proteins exist BER pathway,                     performed untreated wells comparing values non-targeting <br /> interaction likely pathway specific cross-talk               dsRNA luciferase dsRNA targeting gene. If the <br /> between pathways removed.                                        targeting dsRNA resulted 70% viability luciferase <br />                                                                                    p-value 0.00001, gene deemed essential. <br /> Connectivity analyses                                                                 Comparisons predicted observed numbers of <br />    Four measurements connectivity made: (1) Counting                   MMS survival genes yeast Drosophila studies were <br /> number pairs MMS survival proteins directly interact                 standard G-test [52]. The G-test equivalent a <br /> the interaction network, (2) computing average geodesic                        contingency Chi-square test allows classes zero <br /> distance (i.e., number edges shortest path)               events. <br /> pair MMS survival proteins interaction network [51], (3)                    To determine protein enrichment pathway there <br /> the global efficiency network [51], (4) clustering                  unbalanced distribution data numbers of <br /> coefficient [51]. The number direct interactions provides                    proteins pathway compared total number genes <br /> intuitive measure connectivity subnetwork,                   genome, Fisher&#8217;s Exact test employed [53]. <br /> average distance measures global connectivity sub-                         For connectivity measurements, case clustering <br /> network. Global efficiency provides similar measure average                   coefficient, p-values estimated using Z-test, given the <br /> distance, allows disconnected                                              connectivity measurement random subnetworks approxi- <br />                                  1 <br />                                       P components.    1 <br />                                                                The global <br /> efficiency measured N &#240;N{1   &#222;   i, j~1::N, i=j dij , n        mately follows normal distribution (Table S6). The p-value for <br /> number vertices network dij geodesic distance                 clustering coefficient measurement estimated simply <br /> between vertices vi vj. The clustering coefficient measure                counting frequency clustering coefficient a <br /> of local connectivity network. For vertex vi, let gi            randomly sampled sub-network high of <br /> subnetwork consists direct neighbours vi (excluding vi                  real sub-network. <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                   14                           June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                        Damage Survival Pathways <br />  <br />  <br />  <br /> Supporting Information                                                      knock-down resulted significant difference MMS <br />                                                                             survival control shown black (death) stripped <br /> Figure S1 The MMS dose response measured cell survival.               (resistance), trend effect shown grey. A <br /> Viability Drosophila Kc167 cells following exposure increasing        complete list proteins human yeast <br /> dose MMS determined using CellTiter-Glo (relative light units            orthologues given Table S6. (A) Basal Transcription. (B) <br /> (ordinate) increasing dose MMS (abscissa)).                          Ribosome. <br /> Found at: doi:10.1371/journal.pgen.1000527.s001 (0.54 MB EPS)               Found at: doi:10.1371/journal.pgen.1000527.s003 (3.26 MB EPS) <br /> Figure S2 Four additional pathways utilized                Figure S4 Two additional pathways utilized for <br /> MMS survival. CG numbers given Drosophila pathway              MMS survival. CG numbers given Drosophila pathway <br /> component, protein names complex names                component, protein names complex names for <br /> their human orthologues. Pathway entry points noted                human orthologues. Pathway entry points noted with <br /> Roman numerals top, end points bottom. A              Roman numerals top, end points bottom. A <br /> key following symbols provided. Symbols encircled           key following symbols provided. Symbols encircled with <br /> thick lines represent proteins act complex,           lines represent proteins act complex, <br /> while symbols encircled lines represent paralogues             symbols encircled lines represent paralogues or <br /> proteins substitute another. Proteins             proteins substitute another. Proteins to <br /> affect MMS survival noted (death) triangles               affect MMS survival noted (death) triangles <br /> (resistance). Statistically significant proteins indicated         (resistance). Statistically significant proteins indicated with <br /> black triangles, trend hits indicated grey triangles.        black triangles, trend hits indicated grey triangles. <br /> Essential genes noted bar                     Essential genes noted bar with <br /> downwards upwards pointing boxes validated                  downwards upwards pointing boxes validated as <br /> conferring death resistance, respectively, MMS knock-            conferring death resistance, respectively, MMS knock- <br /> down. Shaded squares proteins hits                down. Shaded squares proteins hits after <br /> validation, open squares tested validation.             validation, open squares tested validation. <br /> Yeast orthologues previously required MMS                   Yeast orthologues previously required MMS <br /> survival [3] noted dot symbol. An example           survival [3] noted dot symbol. An example of <br /> average percent untreated control survival validated hits          average percent untreated control survival validated hits is <br /> shown pathway, represent               shown pathway, represent the <br /> actual control data point graph. Error bars         actual control data point graph. Error bars are <br /> the standard deviation quadruplicates. Survival control cells         standard deviation quadruplicates. Survival control cells <br /> with dsRNA targeting luciferase shown open bar, protein            dsRNA targeting luciferase shown open bar, protein <br /> knock-down resulted significant difference MMS                 knock-down resulted significant difference MMS <br /> survival control shown black (death) stripped           survival control shown black (death) stripped <br /> (resistance), trend effect shown grey. A            (resistance), trend effect shown grey. A <br /> complete list proteins human yeast                   complete list proteins human yeast <br /> orthologues given Table S6. (A) Nucleotide Excision Repair.           orthologues given Table S6. (A) ATPases. (B) Notch. <br /> (B) Mismatch Repair. (C) Homologous Recombination Repair. (D)               Found at: doi:10.1371/journal.pgen.1000527.s004 (2.47 MB EPS) <br /> RecQ Helicases. <br />                                                                             Figure S5 Average percent untreated control survival of <br /> Found at: doi:10.1371/journal.pgen.1000527.s002 (1.40 MB EPS) <br />                                                                             validated hits pathway represented Figures 2 3. <br /> Figure S3 Two additional pathways utilized                 Error bars standard deviation quadruplicates. Survival <br /> MMS survival. CG numbers given Drosophila pathway              control cells dsRNA targeting luciferase shown an <br /> component, protein names complex names                open bar average plates general reference. It <br /> their human orthologues. Pathway entry points noted                noted plate luciferase controls <br /> Roman numerals top, end points bottom. A              plate values compared. Protein knock-down <br /> key following symbols provided. Symbols encircled           resulted significant difference MMS survival their <br /> thick lines represent proteins act complex,           internal plate control shown black (death) dashed <br /> while symbols encircled lines represent paralogues             (resistance), trend effect shown grey. A <br /> proteins substitute another. Proteins             complete list proteins human yeast <br /> affect MMS survival noted (death) triangles               orthologues given Table S6. <br /> (resistance). Statistically significant proteins indicated         Found at: doi:10.1371/journal.pgen.1000527.s005 (4.48 MB EPS) <br /> black triangles, trend hits indicated grey triangles. <br />                                                                             Figure S6 Efficiency RNAi mediated silencing gene <br /> Essential genes noted bar with <br />                                                                             expression. (A) RNAi transfection resulted decreased expression <br /> downwards upwards pointing boxes validated as <br />                                                                             target mRNA Drosophila Kc167 cells, measured quantitative <br /> conferring death resistance, respectively, MMS knock- <br />                                                                             real time PCR indicated percent control gene expression <br /> down. Shaded squares proteins hits after <br />                                                                             (ordinate) genes (abscissa) tested, expression <br /> validation, open squares tested validation. <br />                                                                             normalized endogenous control (CG6905). (B) Western blot <br /> Yeast orthologues previously required MMS <br />                                                                             analysis efficiency RNAi siRNA transfected primary <br /> survival [3] noted dot symbol. An example of <br />                                                                             mouse embryonic fibroblast cells. <br /> average percent untreated control survival validated hits is <br />                                                                             Found at: doi:10.1371/journal.pgen.1000527.s006 (0.74 MB EPS) <br /> shown pathway, represent the <br /> actual control data point graph. Error bars         Figure S7 MMS exposure results dose-dependent increase <br /> the standard deviation quadruplicates. Survival control cells         8-oxoguanine DNA modifications. The MMS-dependent <br /> with dsRNA targeting luciferase shown open bar, protein            increase 8-oxoguanine (A) observed qualitatively by <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                             15                          June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                              Damage Survival Pathways <br />  <br />  <br /> microscopic examination 8-oxoguanine containing cells                    Table S3 List yeast MMS hits fly orthologues. Yeast <br /> observed fluorescence FITC channel (B)                 MMS hits determined Begley et al. [3], fly <br /> be quantified percentage fluorescent field                   orthologues hit MMS screen a <br /> (ordinate) increasing dose MMS (abscissa).                             pathway identified screen. Also given fly <br /> Found at: doi:10.1371/journal.pgen.1000527.s007 (7.71 MB EPS)                 gene essential, resistance hit, death hit, death tread <br />                                                                               validation independent dsRNA. <br /> Figure S8      Temozolomide exposure results functional re- <br />                                                                               Found at: doi:10.1371/journal.pgen.1000527.s013 (0.04 MB <br /> sponse MMS survival pathways mouse embryonic <br />                                                                               XLS) <br /> fibroblasts. (A) Temozolomide exposure results phosphorylation <br /> of p53 accumulation total p53 levels. (B)                        Table S4 Drosophila GST family members involvement <br /> Temozolomide exposure results increased total                 MMS survival. The GST families component <br /> glutathione. The intracellular glutathione concentration ex-               members. Provided gene name, corresponding CG number, <br /> pressed units activity/cell. (C) Proteasome activity                 validated significant MMS survival gene (death), <br /> increased following temozolomide exposure, normalizing                    trend (death trend) involved MMS survival (no). For <br /> activity number cells using parallel viability assessment.        validation observed effect, type amplicon, <br /> Proteasome activity expressed units activity/cell,                   library validation given. It noted validated <br /> normalizing activity number cells using parallel              MMS survival gene effect observed the <br /> viability assessment.                                                         screen. If validation performed noted (not <br /> Found at: doi:10.1371/journal.pgen.1000527.s008 (0.71 MB EPS)                 tested). <br />                                                                               Found at: doi:10.1371/journal.pgen.1000527.s014 (0.02 MB <br /> Figure S9 MMS Temozolomide exposure results                            XLS) <br /> functional response notch signalling pathway HEK 293 <br /> cells. MMS temozolomide exposure results decreased                     Table S5 Summary number MMS survival genes <br /> luciferase activity notch reporter RBP-Jk, normalized              identified MMS survival pathway (BER: base excision repair; <br /> transfection efficiency HEK 293 cells using <span id='am-39' about='obo:NCBITaxon_6134' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_6073 obo:NCBITaxon_33208 obo:NCBITaxon_1 obo:NCBITaxon_37509 obo:NCBITaxon_6133 obo:NCBITaxon_131567 obo:NCBITaxon_6132 obo:NCBITaxon_6101 obo:NCBITaxon_86567 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="rdfs:label" content="Renilla" datatype="xsd:string"></span>renilla</span> luciferase.            DDR: DNA damage response; Proteasome; GSH: glutathione <br /> Found at: doi:10.1371/journal.pgen.1000527.s009 (0.41 MB EPS)                 synthesis; TOR: TOR pathway; NER: nucleotide excision repair; <br />                                                                               MMR: mismatch repair; HRR: homologous recombination <br /> Figure S10     Validation pathway functions additional                repair; RECQ: RecQ helicases; Transcription: basal transcription; <br /> siRNA knock mouse embryonic fibroblasts. (A) Knock-                   Ribosome; ATPase; Notch: Notch signaling pathway), the <br /> down GCLc different siRNA results decreased                   percentage non-essential genes pathway this <br /> GCLc expression mouse cells, quantitative real-time PCR                 represents. Protein enrichment pathway compared <br /> analysis; expression level provided fold-change compared             number genes validated (202 13826 prior to <br /> an endogenous control (mouse b-actin). (B) MMS results                  pathway analysis 307 13826 including pathway analysis) is <br /> increased total glutathione, increase                   determined using Fisher&#8217;s Exact Test (NS: Not Significant; NA: <br /> dependent rate limiting glutathione metabolizing enzyme              Not Applicable). <br /> glutamate-cysteine ligase (GCLc). (C) Knock-down proteasome                Found at: doi:10.1371/journal.pgen.1000527.s015 (0.51 MB PDF) <br /> components Psmc1 Psmd1 different siRNA results in <br /> decreased gene expression mouse cells, quantitative real-time           Table S6 The orthologous relationship MMS <br /> PCR analysis; expression level provided fold-change                     survival genes identified Drosophila yeast MMS survival <br /> compared endogenous control (mouse b-actin). (D)                        screens. Provided 13 MMS survival pathway, gene <br /> Proteasome activity increased following MMS exposure                  names Drosophila component corresponding CG <br /> manner dependent proteasome components Pmsc1                     number. For Drosophila genes, yeast ORF is <br /> and Psmd1.                                                                    provided identifiable yeast orthologue was <br /> Found at: doi:10.1371/journal.pgen.1000527.s010 (0.81 MB EPS)                 observed involved MMS survival (hit) S. cerevisiae by <br />                                                                               Begley et al. [3], identified (not hit) essential <br /> Table S1 Summary MMS screen hits pathway genes                         (essential). Where orthologues species known and <br /> selected validation. The 534 Drosophila MMS screen hits                   taking account yeast essential genes be <br /> selected validation analysis 298 pathway genes tested             assayed effect MMS survival, clear <br /> pathway analysis, provided FlyBase gene number, corre-                   enrichment MMS survival genes majority of <br /> sponding CG number, dsRNA used validation, noted                  pathways examined species. A complete list of <br /> by DRSC identification number, library dsRNA                        proteins human orthologues given. <br /> validation dsRNA. For genes validation                      Found at: doi:10.1371/journal.pgen.1000527.s016 (0.11 MB <br /> amplicon designed library amplicon                      XLS) <br /> potential off-target effects 19 nt, data [18] used <br /> (asterisks). For dsRNA, significant death, trend death                Table S7 Connectivity analysis MMS survival genes identified <br /> significant resistance MMS treatment noted (MMS survival).              RNAi genomic screen following validation analyses. <br />                                                                               Provided number proteins identified, number of <br /> Found at: doi:10.1371/journal.pgen.1000527.s011 (0.94 MB <br />                                                                               proteins protein:protein interactome <br /> XLS) <br />                                                                               (PPI), number direct connections proteins, <br /> Table S2    Raw, normalized, survival data validation                 average distance possible pair proteins within <br /> experiment. For data normalization, raw data untreated control             network compared expected values, global efficiency <br /> and MMS experiments normalized using luciferase (Luc)                clustering coefficient. For measurement expected <br /> high MMS controls statistical significance determined              number direct interactions derived number of <br /> described [18].                                                               proteins randomly selected 1000 times PPI. Analyses are <br /> Found at: doi:10.1371/journal.pgen.1000527.s012 (0.23 MB                      provided proteins identified MMS screen prior to <br /> XLS)                                                                          validation (Screen hits), screen hits actually validated <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              16                             June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                                                  Damage Survival Pathways <br />  <br />  <br /> (Validated hits), validated hits essential proteins                      Acknowledgments <br /> are connected hits (Validated+essential connec- <br /> tors), total number validated MMS survival proteins                                   We thank Thomas Begley, University Albany sharing yeast MMS <br />                                                                                              screen data. From UTHSCSA Center Epidemiology Biostatistics, <br /> identified screen pathway validation (Validated+path- <br />                                                                                              thank Gary Chisholm performed initial statistical analyses and <br /> way hits), Validated pathway hits essential                                  John Cornell Jonathon Gelfond discussions guidance. We <br /> proteins connected hits (Validated+path-                             thank Fen Zhou Montserrat Michelman Leder laboratory <br /> way hits+essential connectors), total number validated MMS                            cell line maintenance. We thank Kent Nybakken, Ram Dasgupta, and <br /> survival proteins components 13 MMS survival                            Herve&#180; Agaisse Norbert Perrimon&#8217;s laboratory cells, <span id='am-87' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-88' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span>, and <br /> pathways (Validated+all pathways), finally total number                           technical discussions Lisa Ching helping production of <br /> of validated MMS survival proteins, components                              dsRNA. David Rodrigez GCCRI, UTHSCSA, provided support for <br /> 13 MMS survival pathways essential proteins                                 production line art. Thank members Bishop and <br /> connected hits pathway proteins (Validated+all                             Leder laboratories, Dr. Don McEwen particularly Dr. <br />                                                                                              Vivienne Rebel constructive criticism manuscript. <br /> in pathways+essential connectors). Analyses provided to <br /> compare connectivity real randomly rewired PPI. <br /> Found at: doi:10.1371/journal.pgen.1000527.s017 (0.19 MB                                     Author Contributions <br /> DOC)                                                                                         Conceived designed experiments: DR PL AJRB. Performed the <br />                                                                                              experiments: DR SB AJRB. Analyzed data: DR AMW SB JR AJRB. <br /> Text S1       Methods 8-oxo-guanine assay notch reporter                             Contributed reagents/materials/analysis tools: PL AJRB. Wrote paper: <br /> analysis.                                                                                    DR AMW JR AJRB. Reviewed manuscript: PL. <br /> Found at: doi:10.1371/journal.pgen.1000527.s018 (0.03 MB <br /> DOC) <br />  <br /> References <br />  1. Altieri F, Grillo C, Maceroni M, Chichiarelli S (2008) DNA damage repair:            20. Shen C, Lancaster CS, Shi B, Guo H, Thimmaiah P, et al. (2007) TOR <br />     molecular mechanisms health implications. Antioxid Redox Signal 10:                  signaling determinant cell survival response DNA damage. Mol Cell <br />     891&#8211;937.                                                                                     Biol 27: 7007&#8211;7017. <br />  2. Shimada M, Nakanishi M (2006) DNA damage checkpoints cancer. J Mol                   21. Rusyn I, Fry RC, Begley TJ, Klapacz J, Svensson JP, et al. (2007) <br />     Histol 37: 253&#8211;260.                                                                          Transcriptional networks S. cerevisiae linked accumulation base <br />  3. Begley TJ, Rosenbach AS, Ideker T, Samson LD (2004) Hot spots                            excision repair intermediates. PLoS ONE 2: e1252. doi:10.1371/journal.- <br />     modulating toxicity identified genomic phenotyping localization                       pone.0001252. <br />     mapping. Mol Cell 16: 117&#8211;125.                                                           22. Dickinson DA, Levonen AL, Moellering DR, Arnold EK, Zhang H, et al. (2004) <br />  4. Matsuoka S, Ballif BA, Smogorzewska A, McDonald ER 3rd, Hurov KE, et al.                     Human glutamate cysteine ligase gene regulation electrophile <br />     (2007) ATM ATR substrate analysis reveals extensive protein networks                     response element. Free Radic Biol Med 37: 1152&#8211;1159. <br />     responsive DNA damage. Science 316: 1160&#8211;1166.                                        23. Dann SG, Thomas G (2006) The amino acid sensitive TOR pathway yeast <br />  5. Workman CT, Mak HC, McCuine S, Tagne JB, Agarwal M, et al. (2006) A                          mammals. FEBS Lett 580: 2821&#8211;2829. <br />     systems approach mapping DNA damage response pathways. Science 312:                   24. Ou YH, Chung PH, Sun TP, Shieh SY (2005) p53 C-terminal phosphorylation <br />     1054&#8211;1059.                                                                                   CHK1 CHK2 participates regulation DNA-damage-induced C- <br />  6. Kelley R, Ideker T (2005) Systematic interpretation genetic interactions using            terminal acetylation. Mol Biol Cell 16: 1684&#8211;1695. <br />     protein networks. Nat Biotechnol 23: 561&#8211;566.                                            25. Sinnberg T, Lasithiotakis K, Niessner H, Schittek B, Flaherty KT, et al. (2008) <br />  7. Ng A, Bursteinas B, Gao Q, Mollison E, Zvelebil M (2006) Resources                       Inhibition PI3K-AKT-mTOR Signaling Sensitizes Melanoma Cells to <br />     integrative systems biology: data databases networks                     Cisplatin Temozolomide. J Invest Dermatol. <br />     dynamic models. Brief Bioinform 7: 318&#8211;330.                                       26. Kao HY, Ordentlich P, Koyano-Nakagawa N, Tang Z, Downes M, et al. (1998) <br />  8. Ramadan N, Flockhart I, Booker M, Perrimon N, Mathey-Prevot B (2007)                         A histone deacetylase corepressor complex regulates Notch signal <br />     Design implementation high-throughput RNAi screens cultured                        transduction pathway. Genes Dev 12: 2269&#8211;2277. <br />     Drosophila cells. Nat Protoc 2: 2245&#8211;2264.                                               27. Kerrien S, Alam-Faruque Y, Aranda B, Bancarz I, Bridge A, et al. (2007) <br />                                                                                                  IntAct&#8211;open source resource molecular interaction data. Nucleic Acids Res <br />  9. Drablos F, Feyzi E, Aas PA, Vaagbo CB, Kavli B, et al. (2004) Alkylation <br />                                                                                                  35: D561&#8211;565. <br />     damage DNA RNA&#8211;repair mechanisms medical significance. DNA <br />                                                                                              28. Salwinski L, Miller CS, Smith AJ, Pettit FK, Bowie JU, et al. (2004) The <br />     Repair (Amst) 3: 1389&#8211;1407. <br />                                                                                                  Database Interacting Proteins: 2004 update. Nucleic Acids Res 32: <br /> 10. Rydberg B, Lindahl T (1982) Nonenzymatic methylation DNA the <br />                                                                                                  D449&#8211;451. <br />     intracellular methyl group donor S-adenosyl-L-methionine potentially <br />                                                                                              29. Alfarano C, Andrade CE, Anthony K, Bahroos N, Bajec M, et al. (2005) The <br />     mutagenic reaction. Embo J 1: 211&#8211;216. <br />                                                                                                  Biomolecular Interaction Network Database related tools 2005 update. <br /> 11. Hecht SS (1999) DNA adduct formation tobacco-specific N-nitrosamines.                   Nucleic Acids Res 33: D418&#8211;424. <br />     Mutat Res 424: 127&#8211;142.                                                                  30. Watts DJ, Strogatz SH (1998) Collective dynamics &#8216;small-world&#8217; networks. <br /> 12. Liu L, Taverna P, Whitacre CM, Chatterjee S, Gerson SL (1999) Pharmacologic                  Nature 393: 440&#8211;442. <br />     disruption base excision repair sensitizes mismatch repair-deficient -            31. Begley TJ, Samson LD (2004) Network responses DNA damaging agents. <br />     proficient colon cancer cells methylating agents. Clin Cancer Res 5:                      DNA Repair (Amst) 3: 1123&#8211;1132. <br />     2908&#8211;2917.                                                                               32. Lynn S, Yew FH, Hwang JW, Tseng MJ, Jan KY (1994) Glutathione rescue <br /> 13. Malet-Martino M, Gilard V, Martino R (1999) The analysis cyclophospha-                    inhibitory effects nickel DNA ligation repair synthesis. <br />     mide metabolites. Curr Pharm Des 5: 561&#8211;586.                                         Carcinogenesis 15: 2811&#8211;2816. <br /> 14. Begley TJ, Rosenbach AS, Ideker T, Samson LD (2002) Damage recovery                      33. Mizumoto K, Glascott PA Jr, Farber JL (1993) Roles oxidative stress and <br />     pathways <span id='am-24' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-25' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae revealed genomic phenotyping                     poly(ADP-ribosyl)ation killing cultured hepatocytes methyl <br />     interactome mapping. Mol Cancer Res 1: 103&#8211;112.                                              methanesulfonate. Biochem Pharmacol 46: 1811&#8211;1818. <br /> 15. Jelinsky SA, Samson LD (1999) Global response Saccharomyces cerevisiae             34. Harper JW, Elledge SJ (2007) The DNA damage response: years after. Mol <br />     alkylating agent. Proc Natl Acad Sci U S A 96: 1486&#8211;1491.                                 Cell 28: 739&#8211;745. <br /> 16. Lee MW, Kim BJ, Choi HK, Ryu MJ, Kim SB, et al. (2007) Global protein                    35. Baudot A, Angelelli JB, Guenoche A, Jacq B, Brun C (2008) Defining modular <br />     expression profiling budding yeast response DNA damage. Yeast 24:                   signalling network fly interactome. BMC Syst Biol 2: 45. <br />     145&#8211;154.                                                                                 36. Barrios-Rodiles M, Brown KR, Ozdamar B, Bose R, Liu Z, et al. (2005) High- <br /> 17. Murray JI, Whitfield ML, Trinklein ND, Myers RM, Brown PO, et al. (2004)                     throughput mapping dynamic signaling network mammalian cells. <br />     Diverse specific gene expression responses stresses cultured human                 Science 307: 1621&#8211;1625. <br />     cells. Mol Biol Cell 15: 2361&#8211;2374.                                                      37. Hwang D, Smith JJ, Leslie DM, Weston AD, Rust AG, et al. (2005) A data <br /> 18. Wiles AM, Ravi D, Bhavani S, Bishop AJ (2008) An analysis normalization                   integration methodology systems biology: experimental verification. Proc <br />     methods Drosophila RNAi genomic screens development robust                      Natl Acad Sci U S A 102: 17302&#8211;17307. <br />     validation scheme. J Biomol Screen 13: 777&#8211;784.                                          38. Vidal M (2005) Interactome modeling. FEBS Lett 579: 1834&#8211;1838. <br /> 19. Burgis NE, Samson LD (2007) The protein degradation response                          39. Jones S, Zhang X, Parsons DW, Lin JC, Leary RJ, et al. (2008) Core signaling <br />     Saccharomyces cerevisiae classical DNA-damaging agents. Chem Res                          pathways human pancreatic cancers revealed global genomic analyses. <br />     Toxicol 20: 1843&#8211;1853.                                                                       Science 321: 1801&#8211;1806. <br />  <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                            17                                June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                                                Damage Survival Pathways <br />  <br />  <br /> 40. Lin J, Gan CM, Zhang X, Jones S, Sjoblom T, et al.  <br /> </body></html>