<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Oct. 1997, p. 6105&#8211;6113                                                                          Vol. 17, No. 10 <br /> 0270-7306/97/$04.0010 <br /> Copyright &#169; 1997, American Society Microbiology <br />  <br />  <br />  <br />             Identification RNR4, Encoding Second Essential Small <br />                       Subunit Ribonucleotide Reductase in <br />                               <span id='am-11' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-25'  ></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br />                                          MINGXIA HUANG           AND    STEPHEN J. ELLEDGE* <br />        Verna Mars McLean Department Biochemistry, Howard Hughes Medical Institute, Baylor College Medicine, <br />                                                  Houston, Texas 77030 <br />                         Received 3 February 1997/Returned modification 26 March 1997/Accepted 24 June 1997 <br />  <br />             Ribonucleotide reductase (RNR), catalyzes rate-limiting step deoxyribonucleotide production <br />           required DNA synthesis, a2b2 tetramer consisting large small subunits. RNR2 encodes <br />           small subunit essential mitotic viability <span id='am-28' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-30'  ></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae. We cloned second <br />           essential gene encoding homologous small subunit, RNR4. RNR4 RNR2 appear nonoverlapping <br />           functions substitute overproduced. The lethality RNR4 deletion <br />           mutations suppressed overexpression RNR1 RNR3, genes encoding large subunit of <br />           RNR enzyme, indicating genetic interactions RNR genes. RNR2 RNR4 present the <br />           reductase complex vivo, coimmunoprecipitate cell extracts. Like RNR genes, <br />           RNR4 inducible DNA-damaging agents signal transduction pathway involving MEC1, <br />           RAD53, DUN1 kinase genes. Analysis DNA damage inducibility RNR2 RNR4 revealed partial <br />           inducibility dun1 mutants, indicating DUN1-independent branch transcriptional response DNA <br />           damage. <br />  <br />  <br />    Ribonucleotide reductases (RNRs) catalyze reduction                 differentially regulated. Immunocytochemical studies <br /> all ribonucleoside diphosphates deoxyribonucleoside                 showed R1 R2 subunits mammalian RNR <br /> diphosphates play central role controlling levels             enzyme localized cytoplasm (11) some <br /> cellular deoxynucleoside triphosphates (dNTPs), es-                 cell types appear concentrated near nuclear region <br /> sential high-fidelity DNA replication DNA repair pro-                 (28). The R2 herpes simplex virus type 1 RNR enzyme <br /> cesses. Three classes RNR enzymes character-                     concentrated discrete cytoplasmic foci close nu- <br /> ized; unrelated protein structures, contain              cleus, R1 exhibits diffuse cytoplasmic localiza- <br /> organic free radical crucial catalytic activity,          tion (4). The T4 phage RNRs multienzyme <br /> all allosterically regulated (for reviews, references 23              complex called dNTP synthetase complex, lo- <br /> and 31).                                                                      cated near sites DNA replication (37). <br />    <span id='am-57' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-58' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Class</span> I enzymes commonly best                          Two common features shared RNRs all <br /> studied. They exist eukaryotes prokaryotes,                organisms examined. First, expression enzy- <br /> with aerobic <span id='am-70' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-85'  ></span>Escherichia coli</span> RNR prototype. The                   matic activity RNR enzymes fluctuate cell <br /> functional class I enzyme complex heterotetramer (a2b2)                  cycle, maximal activity S phase (3, 7, 9, 10, 12, 13, 33). <br /> between nonidentical homodimers (20), large subunit                   Second, expression RNR genes inducible DNA- <br /> (a R1) small subunit (b R2),                   damaging agents replication blocks (6, 7, 14), the <br /> inactive itself. Each monomer R1 dimer (two 90-kDa                  mechanisms regulation differ. Such conservation <br /> monomers) contains substrate-binding site redox-                     regulation RNR activity evolution underscores <br /> active thiols separate allosterically regulated binding               importance dNTP level regulation cellular function. <br /> sites dNTPs. One site binds ATP dATP controls                     There known genes encoding RNR subunits in <br /> the overall enzyme activity ensure proper balance be-                  Saccharomyces cerevisiae, RNR1, RNR2, RNR3. The large <br /> tween level NTPs RNA synthesis dNTPs                   subunit encoded highly homologous genes, RNR1 and <br /> for DNA synthesis; site controls substrate spec-                RNR3 (8). RNR1 essential mitotic viability, whereas <br /> ificity maintain balanced dNTP pools necessary                RNR3 nonessential. RNR1 transcription tightly cell cycle <br /> high-fidelity DNA replication. Each monomer R2 dimer                   regulated moderately inducible DNA damage. The <br /> (two 45-kDa monomers) contains tyrosyl free radical                 RNR3 transcript level low normal conditions is <br /> essential catalytic activity. <span id='am-60' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-62'  ></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> tyrosyl free radical gen-           highly inducible DNA damage, increasing 100-fold <br /> erated maintained oxygen-linked binuclear ferric                    (8). The small subunit encoded RNR2, both <br /> iron center buried inside protein (22). The chemothera-                   essential DNA damage inducible (5). Transcriptional reg- <br /> peutic agent hydroxyurea (HU) scavenges tyrosyl radical                  ulation RNR genes extensively studied; how- <br /> and inactivates RNR activity (2). The flexible carboxyl-              ever, little known stoichiometry RNR en- <br /> terminal tail R2 protein essential binding R1            zyme complex regulation assembly activity in <br /> protein assumes rigid structure binding (18).                 vivo. <br />    The localization assembly RNR subunits appear                      Here report isolation characterization sec- <br />                                                                               ond gene encoding RNR small subunit S. cerevisiae, <br />                                                                               RNR4. Our drawn RNR4 initially a <br />   * Corresponding author.                                                     missense mutation results constitutively high transcrip- <br />  <br />                                                                        6105 <br />  6106       HUANG AND ELLEDGE                                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />                                                              TABLE 1. Yeast strains used study <br />   Strain                                                                         Genotype                                                                     Reference <br />  <br /> Y203                    MATa ade2-1 his3 leu2-3,113 lys2 trp1 ura3-D100 r nr3::RNR3-URA3-TRP1                                                                38 <br /> Y205                    MATa ade2-1 his3 leu2-3,113 lys2 trp1 ura3-D100 r nr3::RNR3-URA3-LEU2                                                                39 <br /> Y254                    As Y205, dun1-3                                                                                                                      39 <br /> Y300                    MATa can1-100 ade2-1 his3-11 leu2-3,112 trp1 ura3-1                                                                                  1 <br /> Y323                    MATa/a can1-100/can1-100 ade2-1/ade2-1 his3-11/his3-11 leu2-3,112/leu2-3,112 trp1/trp1 ura3-1/ura3-1                                 1 <br /> TWY397                  MATa ura3 his7 leu2 trp1                                                                                                             36 <br /> TWY312                  MATa mec2-1 (rad53-1) ura3 trp1                                                                                                      36 <br /> TWY308                  MATa mec1-1 ura2 trp1                                                                                                                36 <br /> Y566                    As Y203, rnr4-99                                                                                                                     This   study <br /> Y291                    As Y203, rnr4-99 dun1-3                                                                                                              This   study <br /> Y567                    As TWY397, dun1-D100::HIS3                                                                                                           This   study <br /> Y568                    As Y300, rnr4-99                                                                                                                     This   study <br /> Y569                    As Y300, rnr4-99 dun1-D100::HIS3 1 pMH140 (URA3)                                                                                     This   study <br /> Y570                    As Y300, rnr4-99 rad53-21 1 pMH140 (URA3)                                                                                            This   study <br /> Y571                    As Y300, rnr4-99 mec1-21 1 pMH140 (URA3)                                                                                             This   study <br /> Y572                    As Y323, RNR4/rnr4-D127::LEU2                                                                                                        This   study <br /> Y573                    As Y323, RNR2/rnr2-312::TRP1                                                                                                         This   study <br /> Y574                    Y572 1 pBAD70 (TRP1, GAP-RNR1)                                                                                                       This   study <br /> Y575                    Y572 1 pBAD79 (TRP1, GAP-RNR3)                                                                                                       This   study <br /> Y576                    Y572 1 pBAD52 (TRP1, GAP)                                                                                                            This   study <br />  <br />  <br />  <br /> tion RNR3 absence DNA damage (Crt phe-                                    Immunoprecipitation Western blots. Yeast cells containing GAL expres- <br /> notype [38]). We RNR2 RNR4 essential                             sion <span id='am-112' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-113' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span> grown SC-tryptophan-uracil containing 2% raffinose an <br />                                                                                         optical density 600 nm 0.4, induced galactose final concentration <br /> for mitotic viability present RNR <br />                                                                                         2% (wt/vol) 2 h, harvested. Cell extracts prepared glass <br /> enzyme complex. We lethality RNR4 null                            bead disruption method described previously (7). Yeast protein extracts (60 <br /> mutations suppressed overexpression RNR1                                mg reaction) diluted 500 ml immunoprecipitation buffer (25 <br /> RNR3, suggesting large subunit rate-lim-                            mM HEPES [pH 7.5], 100 mM NaCl, 5 mM MgCl2, 0.1 mM EDTA, 0.1% Tween <br /> iting factor assembly functional RNR enzyme.                                  20, 1 mM phenylmethylsulfonyl fluoride, 0.1 mM benzamidine, 1 mM aprotinin, <br />                                                                                         0.1 mg leupeptin, soybean trypsin inhibitor, pepstatin, antipain per <br />                                                                                         ml) incubated 5 ml mouse anti-HA (12C5) anti-Myc (9E10) <br />                          MATERIALS AND METHODS                                          monoclonal antibody (BAbCO) 1 h 4&#176;C. The antibody-protein complexes <br />                                                                                         precipitated absorption protein A-Sepharose beads 1 h 4&#176;C and <br />    Strains. The yeast strains used study listed Table 1. Y568 was <br />                                                                                         washed twice high-salt buffer (12 mM HEPES [pH 7.9], 500 mM NaCl, 2 <br /> obtained outcrossing rnr4-99 allele times Y300. <br />                                                                                         mM EDTA, 0.01% Tween 20) room temperature. Western blot analysis <br />    Cloning RNR4. A yeast genomic library TRP1 centromeric (CEN) <br /> plasmid (a gift F. Spencer) transformed Y216 selected              performed ECL (Amersham). <br /> SC-tryptophan&#8211;5-fluoroorotic acid (5-<span id='am-1' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-2' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>FOA</span>) plates. Two plasmids 3.4 kb              Immunofluorescence staining flow cytometry. Yeast cells expressing GAL- <br /> overlapping genomic sequence chromosome VII, pMH77 pMH78,                   driven, Myc-tagged Rnr proteins grown SC-uracil 2% raffinose as <br /> recovered E. coli, reintroduced Y216, complement Crt2          carbon source early log phase, induced addition galactose 2%, and <br /> phenotype. The Crt2-complementing activity localized 2.7-kb            harvested 3 h later. Cells fixed 0.1 M potassium phosphate buffer (pH <br /> NheI-AvrII fragment present CEN plasmid, pMH131 (HIS3). The sequence               6.5) 4% formaldehyde room temperature 1 h digested with <br /> of 3.4-kb genomic DNA obtained S. cerevisiae genome <span id='am-107' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-108' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-110'  ></span><span id='am-111' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span>-              Zymolyase 100000 100 mg/ml 0.1 M potassium phosphate buffer (pH <br /> base, extensive open reading frame (ORF), encoding RNR4,                7.0)&#8211;1.2 M sorbitol 37&#176;C 40 min. All following incubations done <br /> detected region. To confirm mutated locus crt3-99 allelic    room temperature phosphate-buffered saline&#8211;1% bovine serum albumin: <br /> to RNR4, LEU2 marker inserted wild-type RNR4 locus               antibody (9E10) 1:50 dilution 3 h, fluorescein isothiocyanate- <br /> haploid homologous integration mated crt3-99 haploid. From 12             conjugated secondary antibody 1:20 dilution 1.5 h, 1 mg DAPI <br /> tetrads dissected, LEU2 marker HU sensitivity segregated             (49,6-diamidino-2-phenylindole) ml 3 min. The flow cytometry (fluores- <br /> exclusively 2:2 fashion, mapping crt3-99 mutation RNR4 locus.      cence-activated cell sorting [FACS]) assay performed described previously <br />    Plasmids. pMH120 constructed cloning 3.1-kb NheI-PvuII fragment           (1). <br /> from pMH78 SpeI-EcoRV sites pBluescript KSII(2). pMH127, the <br /> rnr4D::LEU2 disruption plasmid, inserting LEU2-Kmr cassette <br /> between StuI MscI sites RNR4 pMH120, replacing entire <br /> coding region RNR4. NheI-XhoI cleavage pMH127 results 5.4-kb <br /> fragment transplacement. pMH140 cloning 3.6-kb NheI- <br /> BamHI fragment XbaI BamHI sites pRS416 (URA3) (29). <br />    The hemagglutinin (HA)- Myc-tagged RNR gene expression constructs <br /> were creating convenient restriction enzyme sites outside RNR <br /> ORFs vitro mutagenesis (Bio-Rad Mut-a-Gene kit), inserting HA a <br /> tandem triple Myc epitope-encoding sequence double-stranded oligonucleo- <br /> tides amino termini, subsequent subcloning CEN <span id='am-93' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>vectors</span> under <br /> the control GAL promoter. pMH172 pMH178 contain Rnr2HA and <br /> Rnr1HA TRP1 plasmid, respectively; pMH176, pMH180, pMH195 <br /> contain Rnr43XMyc, Rnr13XMyc, Rnr23XMyc URA3 plasmid, respectively. <br />    The construction 2mm plasmids pBAD70, -71, -79, -86 described <br /> elsewhere (4a). pBAD70 pBAD71 2mm plasmids contain RNR1                   FIG. 1. Northern hybridization analysis transcripts RNR3-URA3 in <br /> ORF control GAP promoter contain TRP1 URA3,                    wild-type (Y203), rnr4-99 (Y566), dun1-3 (Y254), rnr4 dun1-3 (Y291) mutant <br /> respectively. pBAD79 pBAD86 2mm plasmids contain RNR3                  strains. Individual cultures grown early log phase (optical density 600 <br /> ORF control GAP promoter contain TRP1 HIS3                     nm 5 0.4) 30&#176;C kept growing drug treatment (2) treated <br /> markers, respectively.                                                                  150 mM HU 0.01% MMS 2 h 30&#176;C. Transcription RNR3 <br />    RNA blots. Total RNA isolated hot acid-phenol method (16),                promoter visualized Northern blots URA3 probe. The higher- <br /> resolved formaldehyde&#8211;1% agarose gels, hybridized DNA probes               mobility band corresponds endogenous ura3-D100 transcript; lower- <br /> described previously (26).                                                              mobility band corresponds URA3 transcript RNR3 promoter. <br />  VOL. 17, 1997                                                                 RIBONUCLEOTIDE REDUCTASE SMALL-SUBUNIT GENE                                       6107 <br />  <br />  <br />  <br />  <br />    FIG. 2. RNR4 locus alignment Rnr4, Rnr2, human R2 protein. (A) Position RNR4 chromosome VII. Arrows indicate direction of <br /> transcription. RNR4 located ORFs, YGR179C YGR181W. The TATA-like sequences indicated dots. Restriction sites: N, NheI; S, StuI, <br /> M, MscI. The position replacement RNR4 coding sequence LEU2-Kmr cassette indicated. (B) Alignment Rnr4, Rnr2, human (Hu) <br /> R2 protein. Residues identical proteins black boxes, residues identical proteins gray boxes. Asterisks human R2 indicate <br /> the residues conserved R2 proteins mouse, clam, herpes simplex virus, Epstein-Barr virus, E. coli, bacteriophage T4 (5). Sc, S. cerevisiae. <br />  <br />  <br />  <br />  <br />                                  RESULTS                                             mutants (Fig. 1). Therefore, crt3-99 allele act upstream of <br />                                                                                      exclusively DUN1 activate RNR3 transcription. <br />    Identification RNR4 complemention crt3-99, mis-                           We cloned CRT3 gene complementation the <br /> sense mutation results Crt2 phenotype. The crt3-99                         Crt2 phenotype HU sensitivity crt3-99 mutant. <br /> allele identified screen Crt2 mutants. In addition                    The extensive ORF 3.4-kb complementing frag- <br /> to causing constitutive RNR3 transcription, crt3-99 mutants                      ment chromosome VII gene extensive homol- <br /> also extremely sensitive HU (38). To determine epistasis                      ogy RNR2 (Fig. 2). On basis, named gene RNR4. <br /> between CRT3 DUN1, signal-transducing kinase gene                            We confirmed mutated locus crt3-99 mutant is <br /> that controls RNR gene transcriptional response DNA                        allelic RNR4 linkage analysis markers integrated at <br /> damage (39), double mutants carrying crt3-99                             RNR4 locus (see Materials Methods). Therefore, the <br /> dun1-3, apparent null allele DUN1. crt3-99 dun1-3 double                       gene mutation referred RNR4 rnr4-99, <br /> mutants exhibit intermediate RNR3 transcript level, higher                        respectively. Sequence analysis rnr4-99 allele revealed a <br /> than wild type lower crt3-99 single                     GCA-to-GAA change <span id='am-65' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-66' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-67'  ></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> 184, replacing <span id='am-102' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-103'  ></span><span id='am-104' property="rdfs:label" content="Ala" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Ala</span> Glu. <br />  6108       HUANG AND ELLEDGE                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />                                                                                   small-subunit gene suc221 gives rise transcripts, <br />                                                                                   longer transcript induced HU (13, 14). <br />                                                                                      DNA damage inducibility RNR genes controlled by <br />                                                                                   signal-transducing kinase genes MEC1, RAD53, and <br />                                                                                   DUN1. The DNA damage signal transduction pathway yeast <br />                                                                                   includes known kinase genes transducers, central <br />                                                                                   transducer genes MEC1 RAD53, control cell <br />                                                                                   cycle arrest transcriptional response, DUN1, is <br />                                                                                   involved transcriptional response (1, 27, 36, 39). <br />                                                                                   The DNA inducibility RNR2, RNR3, RNR4 wild-type <br />                                                                                   mec1-1, rad53-1, dun1 null mutants examined. <br />                                                                                   The DNA damage inducibility RNR genes re- <br />                                                                                   duced dun1 null mutants abolished rad53-1 <br />                                                                                   mec1-1 mutants (Fig. 3A). The minor increase RNR <br />                                                                                   levels rad53-1 mutants treated HU to <br />                                                                                   S-phase synchronization cell population residual wild- <br />    FIG. 3. Northern analyses RNR genes. (A) Northern hybridization analysis    type Rad53 activity rad53-1 mutants. <br /> of RNR2, RNR3, RNR4 wild-type (WT) (TW397), dun1-D100::HIS3                   rnr4-99 delays S-phase progression synthetically le- <br /> (Y567), rad53-1 (TWY312), mec1-1 (TWY308) strains. Cells grown        thal mutant alleles DUN1, RAD53, MEC1. North- <br /> early log phase treated 200 mM HU 0.01% MMS 2 h at <br /> 30&#176;C. 2, RNA untreated cells. mRNAs detected different RNR           ern analysis showed RNR3 level rnr4-99 dun1-3 <br /> gene probes ACT1 probe. (B) Northern blot samples       double mutants higher wild-type cells <br /> panel A probed 118-bp fragment TATA-like sequences       lower rnr4-99 single mutant cells (Fig. 1). <br /> in 59 untranslated region RNR4. The numbers lane     Since dun1-3 apparent null allele DUN1 (39), these <br /> indicate relative intensity RNR4 signal, normalized ACT1 <br /> signal.                                                                           results indicate DUN1-independent branch for <br />                                                                                   RNR3 induction. The transcript levels RNR2 RNR4 also <br />                                                                                   increase HU MMS treatment DUN1 <br />                                                                                   deletion mutants (Fig. 3A), suggesting pathway indepen- <br />    The RNR4 ORF encodes predicted polypeptide 345                            dent DUN1 involved DNA damage-induced tran- <br /> amino acids. RNR2 RNR4 extensive sequence identity                       scription RNR genes. To exclude possibility residual <br /> in coding region, 56% amino acid level 69%                      DUN1 activity dun1-3 mutants, attempted make an <br /> the nucleic acid level (Fig. 2). However, RNR4 product                     rnr4-99 dun1-D100::HIS3 double mutant. No nonparental di- <br /> about 50 amino acids shorter RNR2                     type tetrads viable spores tetratype <br /> mammalian small subunits N terminus. Another unusual                       tetrads viable spores recovered, suggesting that <br /> feature RNR4 product diverges                    rnr4-99 dun1-D100::HIS lethal. The synthetic lethality was <br /> known R2 sequences conserved residues, 6                      confirmed failure rnr4-99 dun1D mutants carrying <br /> of 16 residues absolutely conserved E. coli                  RNR4 URA3 plasmid grow 5-FOA plates, appar- <br /> mammals (5) changed. The striking change His                     ently lethal event losing wild-type RNR4 copy. <br /> residue (amino acid 127) coordinated iron center                   Similar analyses revealed rnr4-99 synthetically le- <br /> required catalytic activity replaced Tyr Rnr4. A                   thal rad53-21 mec1-21 alleles (Fig. 4A). rnr4-99, like <br /> search S. cerevisiae gene bank confirmed RNR4                     mutations RNR1 RNR2 (38), likely lie upstream of <br /> RNR2 yeast genes encoding RNR small                          damage signal transducer genes causing dNTP pool deple- <br /> subunits class II- class III-related reduc-                  tion generating constitutive replication-blocking signal, <br /> tases appear exist.                                                            properly relayed mec1-21, rad53-21, or <br />    The RNR4 gene transcripts, highly                      dun1D mutants. FACS analysis revealed rnr4-99 mutant <br /> inducible DNA damage replication blocks. One com-                          cells mid-log-phase culture accumulate S phase (Fig. 4B). <br /> mon feature known RNR genes transcription                    When released G1 arrest a-factor, rnr4-99 mutant cells <br /> is upregulated DNA damage replication blocks. We                          exhibit 30-min delay exit S phase compared wild- <br /> tested RNR4 DNA damage inducible North-                        type cells (Fig. 4B). These findings consistent <span id='am-49' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-52'  ></span><span id='am-53' property="rdfs:label" content="Idea" datatype="xsd:string"></span>idea</span> <br /> ern blot analysis. Two transcripts 1 kb detected                    rnr4-99 mutants defective DNA synthesis. <br /> by probe covering full-length RNR4 ORF. The longer                             RNR2 RNR4 essential mitotic viability. To <br /> transcript induced alkylating reagent methyl                       determine null phenotype RNR4 mutants, an <br /> methanesulfonate (MMS) HU, shorter                             rnr4D::LEU2 deletion allele (Fig. 2A) introduced a <br /> transcript (Fig. 3A). Neither RNR4 transcripts                      diploid strain, Y323, transplacement (25). Disruptions were <br /> shows obvious fluctuation cell cycle (data                         verified Southern blot analysis. Tetrad analysis per- <br /> shown). Examination promoter region RNR4 revealed                       formed independently derived diploids heterozygous <br /> two TATA-like sequences near translational start (data                    rnr4D::LEU2 disruption. None 26 tetrads analyzed <br /> shown), suggesting existence different transcrip-                      gave rise viable spores, viable <br /> tional initiation sites. A probe corresponding 118-bp                      spores Leu2 auxotrophs (Table 2). The inviable spores <br /> sequence TATA-like sequences detected                        did germinate undergo rounds cell division <br /> the longer, DNA damage-inducible transcript Northern                           arrest cells elongated buds. Therefore, RNR4 is <br /> blotting (Fig. 3B). Therefore, likely more-up-                     essential mitotic viability. <br /> stream TATA box gives rise longer transcript                         RNR2 shown previous studies essential <br /> the more-downstream TATA box gives rise shorter                            mitotic viability, strains different Y323 <br /> transcript. The function differentially regulated RNR4                     (5). rnr2D::TRP1 disruptants Y323 the <br /> transcripts unclear, encode pro-                    manner described previously (5), similar termi- <br /> tein. It interesting <span id='am-35' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-36' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span>Schizosaccharomyces</span> pombe RNR                    nally arrested phenotype observed rnr2D::TRP1 hap- <br />  VOL. 17, 1997                                                                  RIBONUCLEOTIDE REDUCTASE SMALL-SUBUNIT GENE                         6109 <br />  <br />  <br />                                                                                     TABLE 2. Tetrad analysis RNR4/rnr4D-127 RNR2/rnr2D- <br />                                                                                                   312 diploid strains Y572 Y573 <br />                                                                                                  No. viable     No. tetrads <br />                                                                                     Strain                                                  Genotype <br />                                                                                                spores tetrad     observed <br />  <br />                                                                                                                                      trp2         TRP1 <br />                                                                                     Y572              4                  0              0           0 <br />                                                                                                       3                  0              0           0 <br />                                                                                                       2                 26            52            0 <br />                                                                                                       1                  2              2           0 <br />  <br />                                                                                     Y573                                             ura2         URA1 <br />                                                                                                       4                  0             0            0 <br />                                                                                                       3                  0             0            0 <br />                                                                                                       2                 25            50            0 <br />                                                                                                       1                  3             3            0 <br />  <br />  <br />  <br />  <br />                                                                                     5 RNR2, b9 5 RNR4). These isoforms different <br />                                                                                     specificities localizations. To determine RNR2 RNR4 <br />                                                                                     form complex vivo, coexpression GAL-driven, <br />                                                                                     epitope-tagged RNR2 RNR4 induced addition of <br />                                                                                     galactose immunoprecipitation performed. Rnr2HA <br />                                                                                     detected anti-Myc immunoprecipitates Rnr43XMyc by <br />                                                                                     immunoblotting anti-HA (Fig. 5C, lane 5). Rnr2HA was <br />                                                                                     coimmunoprecipitated anti-Myc extracts cells <br />                                                                                     expressing Rnr43XMyc (Fig. 5C, lane 4). However, little <br />                                                                                     Rnr43XMyc detected immunoprecipitates Rnr2HA <br />                                                                                     (Fig. 5C, lane 8). It possible anti-HA binding Rnr2HA <br />                                                                                     interferes Rnr2-Rnr4 interaction Rnr4 binding to <br />                                                                                     Rnr2HA obscures HA epitope. We tested possible <br />                                                                                     interactions Rnr1 Rnr2 Rnr4. Although both <br />                                                                                     Rnr13XMyc Rnr43XMyc immunoprecipitated anti- <br />                                                                                     Myc antibody (Fig. 5C, lanes 10 12), Rnr2HA (lane 4) <br />                                                                                     Rnr1HA (lane 6) detected immunoprecipitates. <br />                                                                                     Rnr1HA immunoprecipitated anti-HA antibody <br />                                                                                     conditions (Fig. 5C, lane 3). The levels of <br />                                                                                     GAL-driven RNR2 RNR4 expression comparable to <br />                                                                                     wild-type cells DNA damage replication <br />                                                                                     block (Fig. 5A, lanes 2, 3, 5) the <br />                                                                                     normal physiological range RNR levels. <br />                                                                                        The lethality RNR4 deletion suppressed over- <br />                                                                                     expression large-subunit genes RNR1 RNR3. It is <br />    FIG. 4. rnr4-99 mutants exhibits delay S-phase progression, rnr4-99     unclear subunit(s) RNR rate-limiting en- <br /> is synthetically lethal mutant alleles dun1, rad53, mec1. (A) rnr4-99   zyme complex assembly. One possible mechanism le- <br /> is synthetically lethal dun1-D100::HIS3, rad53-21, mec1-21. All strains <br /> shown carry plasmid pMH140 (URA3 RNR4) streaked a <br />                                                                                     thality deletion RNR2 RNR4 mutant <br /> 5-FOA plate. Failure grow 5-FOA plates indicates lethality re-       cells lower level functional RNR complex, resulting <br /> sulted loss wild-type RNR4 copy URA3 plasmid. The            dNTP levels threshold critical mitotic viability. <br /> strains Y568 (rnr4-99), Y569 (rnr4-99 dun1-D100::HIS3), Y570 (rnr4-99           We tested increasing level large-subunit <br /> rad53-21), Y571 (rnr4-99 mec1-21). (B) FACS profiles wild-type (Y300) <br /> and rnr4-99 (Y568) cells asynchronously grown cultures (Asyn) cells     genes suppress lethality caused deficiency the <br /> different time points release a-factor-induced G1 arrest.                small-subunit genes plasmid shuffle tetrad analyses. <br />                                                                                     Both RNR1 RNR3 overexpression suppress rnr4D le- <br />                                                                                     thality (Fig. 6), suppress rnr2D lethality (data <br />                                                                                     shown). RNR1 overexpression suppressed lethality <br />                                                                                     rnr4-99 dun1-D100::HIS3 double mutants (data shown). <br /> loids. Thus, RNR2 RNR4 essential mitotic                              Subcellular localization large small subunits of <br /> viability. The essentiality small-subunit gene                     RNR enzymes. The subcellular localization RNR <br /> be complemented presence multiple copies                           subunits examined immunofluorescence studies the <br /> other, suggesting unique function (data                         GAL-driven, Myc3-tagged Rnr proteins. Both Rnr13XMyc and <br /> shown).                                                                             Rnr23XMyc showed stronger staining cytoplasm in <br />    RNR4 RNR2 present reductase                               nucleus, Rnr43XMyc showed stronger staining in <br /> complex vivo. The finding RNR2 RNR4                            nucleus (Fig. 7). No apparent change staining pat- <br /> essential suggests functional isoforms                    tern observed DNA damage replication blocks <br /> of RNR vivo, a2b2, a2b92, a2bb9 (a 5 RNR1, b                         (data shown). <br />  6110       HUANG AND ELLEDGE                                                                                                                 MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                                                                        FIG. 6. Overexpression RNR1 RNR3 suppresses lethality RNR4 <br />                                                                                     deletion mutations. rnr4D::LEU2/RNR4, pMH140 (URA3 RNR4) diploid strains <br />                                                                                     containing vector (pBAD52, Y576) (A), plasmid expressing RNR1 <br />                                                                                     (pBAD70, Y574) (B), plasmid expressing RNR3 (pBAD79, Y575) the <br />                                                                                     GAP promoter (C) sporulated. Tetrads dissected, spores were <br />                                                                                     incubated 30&#176;C 3 4 days. (D) Plasmid shuffle analysis. rnr4D::LEU2, <br />                                                                                     pMH140 haploid strains carrying pBAD52, pBAD70, pBAD79 grown on <br />                                                                                     5-FOA plates. Growth 5-FOA plates indicates rnr4D strain can <br />                                                                                     survive absence wild-type copy RNR4 URA3 plasmid <br />    FIG. 5. Rnr4 Rnr2 form complex vivo. (A) Northern analysis of <br />                                                                                     pMH140. <br /> endogenous GAL-driven RNR2 RNR4 transcription. Wild-type cells <br /> (WT) (Y300) grown early log phase, kept untreated (2) treated with <br /> 200 mM HU 0.01% MMS 2 h 30&#176;C, harvested RNA extraction. <br /> Wild-type cells carrying GAL-driven RNR2 (pMH172) RNR4 (pMH176) <br /> plasmids grown SC-tryptophan-uracil medium plus 2% raffinose (Raf)       low temperatures. The difference likely to the <br /> early log phase, induced addition galactose 2% growth 2 h       different genetic backgrounds stains used two <br /> 30&#176;C (Gal), harvested preparation RNA. The numbers bottom <br /> of lane indicate relative abundances RNR2 RNR4, normalized          studies, used replacement construct in <br /> against ACT1. (B) Western analysis GAL-driven Rnr13XMyc (pMH180),                strain obtained results similar theirs (15a). <br /> Rnr1HA (pMH178), Rnr2HA (pMH172), Rnr43XMyc (pMH176). Cells                   There plausible explanations yeast cells hav- <br /> treated described panel A, 60 mg protein extract loaded   ing essential RNR small-subunit genes. (i) The level of <br /> lane. There cross-reacting bands anti-HA (a-HA) blot, near <br /> the position Rnr2HA band the position Rnr1HA      RNR2 fluctuates twofold cell cycle (8), <br /> band. (C) Immunoprecipitation (IP) Rnr proteins. Sixty micrograms extract     RNR4 does not. Thus, RNR4 compensate <br /> from wild-type strain (Y300) contains plasmids expressing HA-        RNR2 activity RNR2 low outside S phase. The <br /> Myc-tagged Rnr proteins immunoprecipitated a-HA (lanes 1 3         evidence explanation plasmid constitu- <br /> 7 9) a-Myc (lanes 4 6 10 12) immunoblotted a-HA <br /> (lanes 1 6) a-Myc (lanes 6 12). IgG, heavy chain immunoglobulin G.      tively overexpressing RNR2 (pMH172 GAL-Rnr2HA) com- <br /> The asterisk indicates unknown protein cross-reacts secondary      plement rnr2 mutants rescue rnr4 deletion mutants. <br /> antibody.                                                                           (ii) It possible small subunits contribute the <br />                                                                                     regulation enzyme different RNR isoforms have <br />                                                                                     different substrate specificities enzymatic activities vivo. <br />                                DISCUSSION                                           Therefore, disruption RNR2 RNR4 perturb <br />                                                                                     balance dNTP pools, detrimental mitotic <br />    Why essential RNR small-subunit genes                           viability. It important note overexpression of <br /> yeast? The RNR enzymes prokaryotic eukary-                            RNR1 suppressed rnr4D lethality, RNR3 levels likely be <br /> otic organisms long known a2b2 protein                         high well. This leads possibility reductases the <br /> structure, consisting large (R1) small (R2)                          composition aa9b2, a2b2, a92b2 exist circum- <br /> subunits (reviewed reference 23). In budding yeast,                      stances contribute viability. (iii) It possible RNR <br /> large subunit encoded genes, RNR1 RNR3. Un-                           yeast generally form a2bb9 enzymes the <br /> der normal vegetative conditions, enzyme thought                       form a2b2 a2b29 inactive assemble poorly. (iv) <br /> mostly a12b2 form, RNR1 transcript detect-                     Since Rnr2 Rnr4 partially differentially localized <br /> able. Upon DNA damage, RNR3 highly induced,                             inside cell, Rnr2 cytoplasm Rnr4 <br /> the enzyme isoforms, a92b2 aa9b2 (8).                         nucleus, possible play essential <br /> The small subunit RNR contains tyrosyl free radical                          roles different subcellular compartments. One caveat that <br /> coordinated binuclear iron center essential                      immunofluorescence analyses performed GAL- <br /> enzymatic activity. RNR2 essential gene encodes                     driven, epitope-tagged Rnr proteins not <br /> small subunit S. cerevisiae. In paper described                     represent localization endogenous protein. For ex- <br /> the identification second essential gene encoding small                    ample, Rnr1 require association limiting factor, <br /> subunit RNR, RNR4. In accompanying paper (35) Wang                           e.g., Rnr4, enter nucleus, overproduction give <br /> et al. independent isolation RNR4 gene                      artifactual results. <br /> show rnr4 null mutants viable sensitive high                      One intriguing feature RNR4 product sev- <br />  VOL. 17, 1997                                                               RIBONUCLEOTIDE REDUCTASE SMALL-SUBUNIT GENE                                     6111 <br />  <br />  <br />  <br />  <br />   FIG. 7. Immunofluorescence staining Myc-tagged Rnr1, Rnr2, Rnr4. Y300 containing GAL-driven Rnr13XMyc (pMH180), Rnr23XMyc (pMH195), or <br /> Rnr43XMyc (pMH176) grown SC-uracil containing 2% raffinose induced 3 h 2% galactose prior fixing staining described Materials <br /> and Methods. <br />  <br />  <br />  <br />  <br /> eral changes residues completely con-                        play structural regulatory nonenzymatic role in <br /> served R2 subunits evolution. One example                         enzyme activity. <br /> that conserved His (residue 127) coordinated                       Why does RNR1 RNR3 overexpression suppress the <br /> iron center R2 essential catalytic activity replaced                 lethality rnr4 deletion? The RNR enzyme activity re- <br /> by tyrosine RNR4 product. These changes raise                         quires large small subunits. It unclear the <br /> question RNR4 product acts functional                       formation functional RNR enzyme regulated. The finding <br /> tyrosol free radical provider vivo. We feel given                    overexpression large subunit suppress le- <br /> changes, highly likely RNR4 product cata-                    thality rnr4 deletion mutation suggests large <br /> lytically active likely act structural capacity. In                 subunit rate-limiting step assembly functional <br /> support this, Wang et al. (35) substitution Phe                   RNR enzyme vivo. The data Wang et al. shown <br /> for conserved Tyr (residue 131), generates free                      large subunit rate-limiting vitro (35). This is <br /> radical R2 subunit, complements rnr4 deletion                     consistent fact RNR1 mRNA tightly cell <br /> mutants. This indicates Rnr4 need catalytically                        cycle regulated (8). rnr4 null mutants lower-than- <br /> active carry essential function. However, rnr4-99                 critical concentration certain dNTPs globally all <br /> mutants exhibit delay S-phase progression                        dNTPs regionally (e.g., near DNA polymerases). An increase <br /> sensitive HU, RNR4&#8217;s role likely involve dNTP synthe-                     level large subunit forces assembly the <br /> sis likely act support RNR2 RNR1 activity.                     enzyme complex, a2b2 configuration, com- <br /> In support this, Wang et al. (35)                        pensates decrease dNTP levels RNR4 loss. <br /> endogenous levels Rnr activity vitro low                       However, overexpression RNR1 RNR3 sup- <br /> could significantly increased addition bacterially ex-                    press rnr2 null mutations. It possible rnr2 deletion makes <br /> pressed Rnr1 protein. However, extracts rnr4 mutants                          level small subunit rate-limiting factor <br /> supplemented Rnr1 reached 15% wild-type                           RNR assembly, compensated an <br /> RNR activity levels, indicating defect Rnr function. Addi-                    increase large subunit. Alternatively, Rnr2 <br /> tion bacterially expressed Rnr4 extracts re-                     catalytically active Rnr4 inactive. Since RNR4 and <br /> store RNR activity. These results, ob-                      RNR2 coimmunoprecipitate, feel likely exist <br /> servations Rnr2 Rnr4 physically associate, suggest                        bb9 dimer b (Rnr2) catalytic activity b9 <br /> that Rnr4 structural component RNR enzyme. It                          (Rnr4) plays largely structural role. Thus, yeast enzyme is <br />  6112       HUANG AND ELLEDGE                                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br />                                                                                    completely distinct mechanism. There evidence for <br />                                                                                    redundant function DUN1 RAD53, a <br />                                                                                    rad53-21 mutation lethal combination deletion of <br />                                                                                    DUN1 (15b). It remains seen constitutes par- <br />                                                                                    allel pathway. However, clear transcriptional re- <br />                                                                                    sponse DNA damage complex initially per- <br />                                                                                    ceived likely involve additional genes to <br />                                                                                    implicated response. <br />  <br />                                                                                                              ACKNOWLEDGMENTS <br />                                                                                       We thank Z. Zhou members Elledge lab helpful <br />                                                                                    discussions and/or reagents, thank Tim Huffaker sharing <br />                                                                                    results prior publication. <br />                                                                                       This work supported grant GM44664 National <br />                                                                                    Institutes Health grant Q1187 Robert A. Welch <br />                                                                                    Foundation S.J.E. M.H. Robert A. Welch postdoctoral fellow <br />                                                                                    currently supported postdoctoral fellowship the <br />                                                                                    <span id='am-6' about='obo:NCBITaxon_317678' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6854 obo:NCBITaxon_317663 obo:NCBITaxon_6843 obo:NCBITaxon_317662 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_61977 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="rdfs:label" content="Damon" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Damon</span> Runyon-Walter Winchell Foundation. S.J.E. PEW Scholar <br />                                                                                    Biomedical Sciences Investigator Howard Hughes <br />                                                                                    Medical Institute. <br />  <br />    FIG. 8. Model DNA damage signal transduction pathway yeast. Sen-                                          REFERENCES <br /> sors DNA damage signals G1 G2-M phases Rad9, Rad17, Rad24, <br /> and Mec3 (19). Putative sensors DNA damage replication blocks         1. Allen, J. B., Z. Zhou, W. Siede, E. C. Friedberg, S. J. Elledge. 1994. The <br /> S phase Pol2 (21) Rfc5 (32). The signal generated sensors          SAD1/RAD53 protein kinase controls multiple checkpoints DNA dam- <br /> transduced kinases Mec1 Tel1, leading phosphorylation               age-induced transcription yeast. Genes Dev. 8:2401&#8211;2424. <br /> activation Rad53 (marked Rad53*) (27, 34). Rad53* causes cell cycle arrest    2. Barlow, T. R. Eliasson, A. Platz, P. Reichard, B. M. Sjoberg. 1983. <br /> and transcriptional induction. The transcriptional response mediated         Enzymic modification tyrosine residue stable free radical ribo- <br /> two mechanisms, activation Dun1 kinase (marked Dun1*) (1)            nucleotide reductase. Proc. Natl. Acad. Sci. USA 80:1492&#8211;1495. <br /> and DUN1-independent mechanism.                                 3. Bjorklund, S., S. Skog, B. Tribukait, L. Thelander. 1990. S-phase- <br />                                                                                        specific expression mammalian ribonucleotide reductase R1 R2 sub- <br />                                                                                        unit mRNAs. Biochemistry 29:5452&#8211;5458. <br />                                                                                     4. Conner, J., J. Murray, A. Cross, J. B. Clements, J. S. Marsden. 1995. <br />                                                                                        Intracellular localization herpes simplex virus type 1 ribonucleotide re- <br />                                                                                        ductase subunits infection cultured cell. Virology 213:615&#8211;623. <br /> likely structure a2bb9, structure unique                       4a.Desany, B., S. J. Elledge. Unpublished data. <br /> RNRs.                                                                               5. Elledge, S. J., R. W. Davis. 1987. Identification isolation gene <br />    A DUN1-independent pathway RNR transcriptional in-                              encoding small subunit ribonucleotide reductase Saccharomyces <br /> duction DNA damage. Mutations RNR genes,                                 cerevisiae: DNA damage-inducible gene required mitotic viability. Mol. <br />                                                                                        Cell. Biol. 7:2783&#8211;2793. <br /> RNR1 (CRT7), RNR2 (CRT6), RNR4 (CRT3)                                 6. Elledge, S. J., R. W. Davis. 1989. DNA damage induction ribonucle- <br /> isolated Crt2 screen (38). These mutations, like muta-                          otide reductase. Mol. Cell. Biol. 9:4932&#8211;4940. <br /> tions genes involved dNTP metabolism, likely                     7. Elledge, S. J., R. W. Davis. 1989. Identification damage-regulatory <br /> lead endogenous DNA damage replication block sig-                             element RNR2 evidence distinct proteins bind it. Mol. <br />                                                                                        Cell. Bio. 9:5373&#8211;5386. <br /> nal dNTP pool depletion, activates DNA                             8. Elledge, S. J., R. W. Davis. 1990. Two genes differentially regulated in <br /> damage signaling pathway resulting constitutive transcrip-                        cell cycle DNA-damaging agents encode alternate regulatory <br /> tional response, Crt2 phenotype. Epistasis analysis sug-                           subunits ribonucleotide reductase. Genes Dev. 4:740&#8211;751. <br /> gests rnr4-99 acts partially completely                        9. Elledge, S. J., Z. Zhou, J. B. Allen, T. A. Navas. 1993. DNA damage and <br />                                                                                        cell cycle regulation ribonucleotide reductase. Bioessays 15:333&#8211;339. <br /> Dun1 kinase, RNR3 level rnr4-99 dun1-3 double                         10. Engstrom, Y., S. Eriksson, I. Jildevik, S. Skog, L. Thelander, B. Tribu- <br /> mutants intermediate rnr4-99 mutants                           kait. 1985. Cell cycle-dependent expression mammalian ribonucleotide <br /> the wild type. Northern analysis shows signif-                     reductase. J. Biol. Chem. 260:9114&#8211;9116. <br /> icant RNR2 RNR4 induction HU MMS treat-                              11. Engstrom, Y., B. Rozell. 1988. Immunocytochemical evidence the <br />                                                                                        cytoplasmic localization differential expression cell cycle of <br /> ment dun1 null mutant (Fig. 3A), induction                          M1 M2 subunits mammalian ribonucleotide reductase. EMBO J. <br /> further decreased mec1 rad53 mutants. These data in-                            7:1615&#8211;1620. <br /> dicate regulation RNR2 RNR4 transcription                       12. Eriksson, S., A. Graslund, S. Skog, L. Thelander, B. Tribukait. 1984. <br /> not completely controlled Dun1, illustrated model                         Cell cycle-dependent regulation mammalian ribonucleotide reductase. <br />                                                                                        J. Biol. Chem. 259:11695&#8211;11700. <br /> in Fig. 8.                                                                         13. Fernandez-Sarabia, M.-J., C. McInerny, P. Harris, C. Gordon, P. <br />    Therefore, DUN1-independent mechanism                                Fantes. 1993. The cell cycle genes cdc221 suc221 fission yeast <br /> RNR2 RNR4 regulation response DNA damage.                                    <span id='am-40' about='obo:NCBITaxon_4896' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_1 obo:NCBITaxon_147554 obo:NCBITaxon_34346 obo:NCBITaxon_4890 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4895 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces malidevorans" datatype="xsd:string"></span><span id='am-43'  ></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces pombeP" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45042" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Schizosaccharomyces pombe" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="fission yeast" datatype="xsd:string"></span>Schizosaccharomyces pombe</span> encode large small subunits ribonu- <br /> Perhaps exists gene carries DUN1&#8217;s function.                          cleotide reductase. Mol. Gen. Genet. 238:241&#8211;251. <br />                                                                                    14. Harris, P., P. J. Kersey, C. J. McInerny, P. A. Fante. 1996. Cell cycle, <br /> There DUN1-related gene, MRE4/MEK1, struc-                               DNA damage, hear shock regulate suc221 expression fission yeast. <br /> turally similar DUN1 kinase domain                           Mol. Gen. Genet. 252:284&#8211;291. <br /> regulatory region, FHA domain (15).                             15. Hofmann, K., P. Bucher. 1995. The FHA domain: putative nuclear <br /> However, MRE4 meiosis specific (17, 24)                            signaling domain protein kinases transcription factors. Trends. <br />                                                                                        Biochem. Sci. 20:347&#8211;349. <br /> unlikely perform DUN1-redundant role. Rad53 con-                         15a.Huang, M. Unpublished data. <br /> tains forkhead-associated (FHA) domain closely re-                        15b.Jones, W., S. J. Elledge. Unpublished data. <br /> lated Dun1 kinase domain, calm-                        16. Kohrer, K., H. Domdey. 1991. Preparation high molecular weight <br /> odulin-dependent kinase protein kinase <span id='am-54' about='obo:NCBITaxon_subfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="subfamily" datatype="xsd:string"></span>subfamily</span> (30, 39). It                       RNA. Methods Enzymol. 194:398&#8211;405. <br />                                                                                    17. Leem, S.-H., H. Ogawa. 1991. The MRE4 gene encodes novel protein <br /> possible Rad53 itself activate RNR transcription                           kinase homologue required meiotic recombination Saccharomyces <br /> two mechanisms, indirectly activating Dun1 (1)                          cerevisiae. Nucleic Acids Res. 20:449&#8211;457. <br /> directly recognizing Dun1 substrates                        18. Lycksell, P. O., R. Ingemarson, R. Davis, A. Graslund, L. Thelander. <br />  VOL. 17, 1997                                                                        RIBONUCLEOTIDE REDUCTASE SMALL-SUBUNIT GENE                                      6113 <br />  <br />  <br />       1994. 1H NMR studies mouse ribonucleotide reductase: R2 protein                    host strains designed efficient manipulation DNA Sacchromyces <br />       carboxyl-terminal tail, essential subunit interaction, highly flexible         cerevisiae. Genetics 122:19&#8211;27. <br />       rigid presence protein R1. Biochemistry 33:2838&#8211;2842.             30.   Stern, D., P. Zheng, D. R. Berdler, C. Zerillo. 1991. Spk1, new kinase <br /> 19.   Lydall, D., T. Weinert. 1995. Yeast checkpoint genes DNA damage                    Sacchromyces cerevisiae, phosphorylates proteins serine, threonine, <br />       processing: implications repair arrest. Science 270:1488&#8211;1491.                    tyrosine. Mol. Cell Biol. 11:987&#8211;1001. <br /> 20.   Mathews, C. K., B. M. Sjoberg, P. Reichard. 1987. Ribonucleotide                31.   Stubbe, J. 1990. Ribonucleotide reductases: amazing confusing. J. Biol. <br />       reductase <span id='am-86' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-87'  ></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia</span> coli. Cross-linking agents probes quaternary               Chem. 265:5329&#8211;5332. <br />       quinary structure. Eur. J. Biochem. 166:279&#8211;285.                                32.   Sugimoto, K., T. Shimomura, K. Hashimoto, H. Araki, A. Sugino, K. <br /> 21.   Navas, T. A., Z. Zhou, S. J. Elledge. 1995. DNA polymerase &#949; links                Matsumoto. 1996. Rfc5, small subunit replication factor C complex, <br />       DNA replication machinery S phase checkpoint. Cell 80:29&#8211;39.                       couples DNA replication mitosis budding yeast. Proc. Natl. Acad. Sci. <br /> 22.   Petersson, L., A. Graslund, A. Ehrenberg, B. Sjoberg, P. Reichard. 1980.              USA 93:7048&#8211;7052. <br />       The iron center ribonucleotide reductase Escherichia coli. J. Biol.         33.   Sun, L., J. A. Fuchs. 1992. Escherichia coli ribonucleotide reductase <br />       Chem. 255:6706&#8211;6712.                                                                      expression cell cycle regulated. Mol. Biol. Cell 3:1095&#8211;1105. <br /> 23.   Reichard, P. 1993. From RNA DNA, ribonucleotide reduc-               34.   Sun, Z., D. S. Fay, F. Marini, M. Foiani, D. F. Stern. 1996. Spk1/Rad53 <br />       tases? Science 260:1773&#8211;1777.                                                             regulated Mec1-dependent protein phosphorylation DNA replication <br /> 24.   Rockmill, B., G. S. Roeder. 1991. A meiosis-specific protein kinase                   damage checkpoint pathways. Genes Dev. 10:395&#8211;406. <br />       homolog required chromosome <span id='am-97' about='obo:NCBITaxon_324792' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91897 obo:NCBITaxon_3398 obo:NCBITaxon_4143 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-98'  ></span><span id='am-99' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="Synapsis" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>synapsis</span> recombination. Genes Dev.              35.   Wang, P. J., A. Chabes, R. Casagrande, X. C. Tian, L. Thelander, T. C. <br />       5:2392&#8211;2404.                                                                              Huffaker. 1997. RNR4, novel ribonucleotide reductase small-subunit pro- <br /> 25. <br /> </body></html>