<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>  <br />                               Mammalian Rif1 contributes to replication stress <br />   <br />  <br />  <br />                               R <br />                                     if1, originally recognized for its role at telomeres in                        poisons. Rif1 deficiency does not alter the activation of the <br />                                     budding yeast, has been implicated in a wide vari&#173;                             DNA replication checkpoint but rather affects the execution <br />                                     ety of cellular processes in mammals, including pluri&#173;                         of repair. RNA interference to human Rif1 decreases the <br />                               potency of stem cells, response to double-strand breaks,                             efficiency of homology-directed repair (HDR), and Rif1 <br /> THE JOURNAL OF CELL BIOLOGY <br />  <br />  <br />  <br />  <br />                               and breast <span id='am-26' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-28'  ></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Cancer" datatype="xsd:string"></span>cancer</span> development. As the molecular function                             deficiency results in aberrant aggregates of the HDR fac&#173; <br />                               of Rif1 is not known, we examined the consequences of                                <span id='am-74' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-75'  ></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Tor" datatype="xsd:string"></span>tor</span> Rad51. Consistent with a role in S-phase progression, <br />                               Rif1 deficiency in mouse cells. Rif1 deficiency leads to fail&#173;                       Rif1 accumulates at stalled replication forks, preferentially <br />                               ure in embryonic development, and conditional deletion                               around pericentromeric heterochromatin. Collectively, these <br />                               of Rif1 from mouse embryo fibroblasts affects S-phase                                findings reveal a function for Rif1 in the repair of stalled forks <br />                               progression, rendering cells hypersensitive to replication                           by facilitating HDR. <br />  <br />  <br />  <br />  <br />                               Introduction <br />                               In mammalian cells, two major checkpoints ensure the fidelity                        DNA damage, and genes required for the DNA replication <br />                               of DNA replication. The intra&#8211;S phase checkpoint is elicited by                      checkpoint are essential (Liu et al., 1994; Brown and Baltimore, <br />                               exogenously inflicted DNA damage that creates double-strand                          2000; Takai et al., 2000; Weiss et al., 2000; Yamane et al., 2002; <br />                               breaks (DSBs) and is controlled by the PI3-like kinase ataxia                        Budzowska et al., 2004; Hopkins et al., 2004). Activation of the <br />                               telangiectasia mutated (ATM). ATM can be activated in every                          DNA replication checkpoint arrests the cell cycle and activates <br />                               phase of the cell cycle, including S phase, and the main signal                      the homologous recombination pathway, which mediates <br />                               transducer is Chk2 (for review see Cimprich and Cortez, 2008).                       the restart of the arrested replication fork (for review see <br />                               Neither Chk2 nor ATM are essential genes (Barlow et al., 1996;                       Lambert et al., 2007). <br />                               Elson et al., 1996; Xu et al., 1996; Hirao et al., 2002; Takai                              We have previously shown that Rif1 participates in the <br />                               et al., 2002). The second and essential checkpoint (hitherto re-                     intra&#8211;S phase checkpoint, contributing to the inhibition of DNA <br />                               ferred as the DNA replication checkpoint) monitors the process                       replication associated with the activation of ATM (Silverman <br />                               of DNA replication itself and is activated by single-stranded                        et al., 2004). Human Rif1 localizes to DSBs induced by a vari- <br />                               DNA (ssDNA) accumulated at stalled replication forks. The ac-                        ety of clastogenic agents but not to DNA lesions generated by <br />                               tivation of the checkpoint depends on another PI3-like kinase,                       UV radiation. <span id='am-31' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-34'  ></span><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> association with DSBs is dependent on the <br />                               ataxia telangiectasia and Rad3 related (ATR), and its main signal                    activation of the ATM kinase and the DNA damage response <br />                               transducer is Chk1 (for review see Cimprich and Cortez, 2008).                       factor 53BP1 (Schultz et al., 2000; Silverman et al., 2004). <br />                               Errors in DNA replication are the main source of endogenous                                 Rif1 was originally identified in budding yeast based on its <br />                                                                                                                    ability to interact with Rap1, a protein which plays a key role <br />                               Correspondence to Sara B.C. Buonomo: sara.buonomo@embl.it <br />                                                                                                                    at telomeres (Hardy et al., 1992). By tethering Rif1 and a sec- <br />                               S.B.C. Buonomo&#8217;s present address is Mouse Biology Unit, European Molecular <br />                               Biology Laboratory Monterotondo, 00015 Monterotondo, Italy.                          ond Rap1-interacting factor, Rif2, Rap1 can limit the action of <br />                               Abbreviations list: ATM, ataxia telangiectasia mutated; ATR, ataxia telangiecta-     telomerase in <span id='am-69' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71'  ></span><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Cis" datatype="xsd:string"></span>cis</span> and thus establish telomere length homeostasis <br />                               <span id='am-84' about='obo:NCBITaxon_208766' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_62771 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6994 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_70905 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_6993 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_33341 obo:NCBITaxon_50557 obo:NCBITaxon_6072 obo:NCBITaxon_6960'><span id='am-85'  ></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Sia" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>sia</span> and Rad3 related; DSB, double-strand break; ES, embryonic stem; HDR, <br />                               homology-directed repair; HU, hydroxyurea; IF, immunofluorescence; IR, ionizing <br />                                                                                                                    &#169; 2009 Buonomo et al.&#8195; This article is distributed under the terms of an Attribution&#8211; <br />                               radiation; LTGC, long tract gene conversion; MEF, mouse embryonic fibroblast;        Noncommercial&#8211;Share Alike&#8211;No Mirror Sites license for the first six months after the publica- <br />                               MMC, mitomycin C; mTOR, mammalian target of rapamycin; NeoR, Neomycin                tion date (see http://www.jcb.org/misc/terms.shtml). After six months it is available under a <br />                               resistance; shRNA, short hairpin RNA; SKY, spectral karyotyping analysis; ssDNA,     Creative Commons License (Attribution&#8211;Noncommercial&#8211;Share Alike 3.0 Unported license, <br />                               single-stranded DNA; SV40LT, SV40 large T antigen.                                   as described at http://creativecommons.org/licenses/by-nc-sa/3.0/). <br />  <br />  <br />  <br />  <br />                               The Rockefeller University Press&#8195;&#8202;$30.00 <br />                               J. Cell Biol. Vol. 187 No. 3&#8195; 385&#8211;398 <br />                               www.jcb.org/cgi/doi/10.1083/jcb.200902039                                                                                                                                     JCB    385 <br />            (Marcand et al., 1997; Levy and Blackburn, 2004; Bianchi and          detected for wild-type Rif1 (unpublished <span id='am-89' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-90' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-91'  ></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span>). Intercrosses of <br />           Shore, 2008). Rif2 is a diverged version of ORC4 that can bind        Rif1GT/+ mice yielded homozygous Rif1GT/GT offspring at a se- <br />           to Rif1 as well as Rap1 (Wotton and Shore, 1997; Marcand              verely reduced frequency (Fig. S1 F), suggesting a critical role <br />           et al., 2008). No other Rif1 interacting partners have been iden-     for Rif1 during mouse development. In addition, examination <br />           tified in yeast, and the mechanism by which Rif1 enforces the         of the genotypes of embryonic day (E) 12.5 embryos showed <br />           inhibition of telomerase has not been established. Orthologues        only one third of the expected number of Rif1GT/GT embryos <br />           of Rif1 and Rap1 (but not Rif2) have been recognized in               (Fig. S1 F), indicating that Rif1 contributes to embryonic de- <br />           <span id='am-7' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Schizosaccharomyces</span> pombe and in vertebrates (Li et al., 2000;        velopment at early as well as late stages. In addition, a con- <br />           Kanoh and Ishikawa, 2001; Li and de Lange, 2003; Adams                ventional knockout allele of Rif1, in which exons 8 and 9 were <br />           and McLaren, 2004; Silverman et al., 2004). Although fission          replaced with a Neomycin resistance (NeoR) cassette, resulted <br />           yeast Rif1 does not bind Rap1, it does interact with telomeres        in embryonic lethality on a pure C57BL/6J background (un- <br />           and contribute to telomere length homeostasis (Kanoh and              published data). These results are in contrast with the findings <br />           Ishikawa, 2001).                                                      with ATM and 53BP1 knockout mice (Barlow et al., 1996; <br />                   So far there has been no indication of a conserved role for   Elson et al., 1996; Xu et al., 1996; Morales et al., 2003; Ward <br />           yeast and mammalian Rif1 orthologues. Mammalian Rif1 does             et al., 2003), which are born at the expected Mendelian ratios <br />           not appear to bind normal telomeres nor to have a role in telo-       and are fully viable. Therefore, the embryonic lethality indi- <br />           mere homeostasis (Silverman et al., 2004; Xu and Blackburn,           cates that Rif1 has functions in addition to those downstream <br />           2004; this study). Although rif1&#57412; budding yeast looses chromo-        of ATM and 53BP1. Furthermore, the recovery of the Rif1- <br />           somes at a slightly increased rate (Wotton and Shore, 1997;           deficient embryos at E12.5 (Fig. S1 F) argues that Rif1 is not <br />           Banerjee and Myung, 2004), Rif1 deficiency does not affect the        essential for the function of Nanog, whose absence results in <br />           rate of gross chromosomal rearrangements (Myung et al., 2001),        lethality around E5.5 (Mitsui et al., 2003). <br />           which are largely caused by the repair of S-phase damage. Sim-               The few Rif1GT/GT mice born survived for at least 1 yr but <br />           ilarly, there is no data indicating that fission yeast Rif1 plays a   appeared slightly retarded in their growth and were less fertile <br />           prominent role in the response to S-phase damage.                     than their littermates (unpublished data). Because these rare <br />                   In addition to regulation of telomere maintenance in bud-     Rif1GT/GT survivors are likely reflecting some type of compen- <br />           ding yeast and response to DSBs in human cells, Rif1 has also         sation for the essential function of Rif1, we did not study <br />           been implicated in transcriptional silencing at yeast telomeres       them further. <br />           and ribosomal DNA (Hardy et al., 1992; Smith et al., 1999; Teng <br />           et al., 2000; Chan et al., 2001; Silverman et al., 2004; Teixeira     Conditional deletion of Rif1 induces <br />           et al., 2004) and in controlling mouse embryonic stem (ES) cells&#8217;     an S-phase defect <br />           identity (Loh et al., 2006; Wang et al., 2006; Fazzio et al., 2008;   Given the diminished viability of Rif1-deficient embryos and <br />           Liang et al., 2008). Rif1 has no informative protein motifs or in-    the associated possibility of selection for compensation, we <br />           teracting partners that could point to the molecular mechanism        generated a conditional allele of Rif1. In the conditional allele <br />           by which Rif1 contributes to these different pathways.                (Rif1F), LoxP target sites for the Cre recombinase were posi- <br />                   To gain further insight into the function of this enigmatic   tioned 5&#57401; of exon 5 and 3&#57401; of exon 7 (Fig. 1 A). Mice <span id='am-36' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38'  ></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Homo" datatype="xsd:string"></span>homo</span>&#173; <br />           protein, we targeted Rif1 in the mouse, generating a setting in       zygous for the Rif1F/F allele were born at the expected frequency, <br />           which the consequences of the absence of Rif1 could be stud-          viable, healthy, and fertile (unpublished data). Fibroblasts were <br />           ied. Our data reveal a new role for mammalian Rif1 in the con-        prepared from E12.5 Rif1F/F, and wild-type littermate embryos <br />           text of the repair machinery that is mobilized by the DNA             were derived from Rif1F/+ intercrosses. Cells were either used <br />           replication checkpoint.                                               as primary cultures (pMEFs) or immortalized with SV40 large <br />                                                                                 T antigen (SV40LT; MEFs). Infection with a Cre-expressing <br />           Results                                                               retrovirus efficiently generated the Rif1-null allele (Fig. 1, B and C). <br />                                                                                 In the absence of Rif1, primary and SV40LT immortalized <br />           Rif1 deficiency impairs embryonic                                     cells proliferated significantly slower than the controls (Fig. 1 D <br />           development                                                           and not depicted), which is consistent with the findings from <br />           To determine the consequences of Rif1 deficiency, we initially        the Rif1GT/GT cells (Fig. S1 G). Rif1-deficient cells, like Rif1GT/GT <br />           used a mouse ES cell line from a gene trap collection (Hansen         pMEFs, also showed reduced uptake of BrdU during a 30-min <br />           et al., 2003) in which a pT1&#57442;-GEO vector was integrated be-           pulse, indicating that the proliferation phenotype is (at least in <br />           tween exons 21 and 22 of the Rif1 gene (referred to as Rif1GT;        part) caused by a defect in entry and/or progression through <br />           Fig. S1, A&#8211;D). This insertion generates a truncated allele, cod-      S phase (Fig. 1 E, Fig. S2 A, and not depicted). We could not <br />           ing for a Rif1/&#57442;-GEO fusion protein which lacks a nuclear             detect any significant effect of Rif1 deletion on the amount of <br />           local&#173;ization signal (Fig. S1 E). Immunoblots of Rif1GT/+ mouse       cells in G2/M based on FACS analysis of phosphorylated Ser10 <br />           embryonic fibroblasts (MEFs) showed diminished expression             histone H3 (Fig. 1 F). Consistent with previous data on human <br />           of the Rif1/&#57442;-GEO protein compared with wild type, sug-               Rif1 (Silverman et al., 2004), Rif1&#57394;/&#57394; cells revealed no obvious <br />           gesting that the truncated protein is unstable (Fig. S1 E). Fur-      telomere defects such as telomere fusions or loss of telomeric <br />           thermore, as expected from the lack of a nuclear localization         DNA (Fig. S1 H and not depicted), arguing that their dimin- <br />           signal, Rif1/&#57442;-GEO failed to show the specific nuclear signals        ished proliferation is not telomere related. Thus, unlike the <br />  <br />  <br />     386   JCB &#8226; VOLUME 187 &#8226; NUMBER 3 &#8226; 2009 <br />                                                                                                 Figure 1.&#8195; Conditional deletion of mouse Rif1 <br />                                                                                                affects cell growth. (A) Schematic diagram <br />                                                                                                of the mouse Rif1 locus, the targeting vector <br />                                                                                                (pTV), the conditional Rif1 allele (FLOX), and <br />                                                                                                the null allele (DEL). LoxP sites (triangles) flank <br />                                                                                                exons 5, 6, and 7. FRT sites (circles) flank the <br />                                                                                                NeoR gene. SacI (S) and BamHI (B) fragment <br />                                                                                                sizes are indicated for each genotype, and <br />                                                                                                the probes p1 and p2 are shown. f1, r1, and <br />                                                                                                r2 are primers for genomic PCR. (B) Southern <br />                                                                                                blots from MEF clones. Genomic DNA digested <br />                                                                                                with SacI and probed with p1. The MEFs <br />                                                                                                were infected with retrovirus carrying either <br />                                                                                                an empty vector (&#57394;Cre) or pWZL-Cre (+Cre). <br />                                                                                                Length is indicated in kilobases. (C) Western <br />                                                                                                blot of extracts from MEFs of the indicated <br />                                                                                                genotypes probed with mouse Rif1 antibody <br />                                                                                                1240. mTOR was used as a loading control. <br />                                                                                                (D) Growth curves of Rif1 wild type (WT) and <br />                                                                                                two independent Rif1F/F (F/F) MEF lines with <br />                                                                                                and without treatment with Cre. (E) Changes in <br />                                                                                                the S-phase index upon infection with Cre deter- <br />                                                                                                mined based on BrdU uptake during a 30-min <br />                                                                                                pulse of asynchronous populations of Rif1- <br />                                                                                                proficient and -deficient cells. The changes were <br />                                                                                                quantified by calculating the percentage of <br />                                                                                                BrdU-positive cells in (&#57394;Cre) &#57394; (+Cre)/(&#57394;Cre). <br />                                                                                                (F) The same quantification as in E was per- <br />                                                                                                formed for the G2/M cells by staining for his- <br />                                                                                                tone H3 phospho-Ser10 and FACS analysis. <br />                                                                                                (D&#8211;F) Error bars represent SD. <br />  <br />  <br />  <br />  <br /> fungal Rif1 genes, mammalian Rif1 is required for normal              et al., 1984). In wild-type cells, this treatment induces chro- <br /> cell proliferation.                                                   matid breaks and chromosomal fragments. The frequency of <br />                                                                       these anomalies was significantly elevated (threefold) in Rif1- <br /> Rif1 deficiency exacerbates sensitivity                               deficient cells, suggesting an increase in fork-stalling frequency, <br /> to aphidicolin                                                        a decrease in the stability of stalled forks, or a defective restart <br /> The reduced proliferation of Rif1-null cells and their dimin-         of the stalled replication fork. Other aberrations such as trans- <br /> ished BrdU uptake suggested a role for Rif1 in S phase that           locations and radial chromosomes were not significantly altered <br /> is distinct from its ATM-dependent response to DSBs. There-           in the Rif1-null setting (Fig. 2 B). In the absence of aphidi- <br /> fore, we asked whether Rif1 deficiency affected the response          colin treatment, Rif1-null cells showed a spontaneous eleva- <br /> to the DNA polymerase inhibitor aphidicolin. The effect of            tion of chromatid breaks and chromosomal fragments over the <br /> aphidicolin treatment was evaluated on metaphase chromo-              negligible background detected in Cre-treated wild-type cells. <br /> somes from Rif1-deficient immortalized MEFs or Cre-treated            A similar breakage phenotype occurred in Rif1GT/GT MEFs (un- <br /> wild-type control cells (Fig. 2, A and B). The standard dose of       published data). <br /> aphidicolin used for this experiment does not inhibit cell cycle             Because chromosome spreads showed that replication forks <br /> progression but slows replication fork progression and ren-           in Rif1-null cells are more sensitive to low doses of aphidicolin, <br /> ders it more prone to stalling, especially at fragile sites (Glover   we examined how Rif1 deficiency influenced the ability of cells to <br />  <br />  <br />                                                                              Rif1 modulates HDR at stalled forks &#8226; Buonomo et al.                    387 <br />            Figure 2.&#8195; Rif1-null cells are hypersensitive to aphidicolin. (A) Representative examples of metaphase chromosomes from wild-type (left) or Rif1F/F (right) <br />           cells treated with Cre and aphidicolin. Open arrows indicate chromatid breaks, the closed arrow indicates a fragment, and arrowheads indicate radial <br />           chromosomes. (B) Table summarizing chromosome anomalies scored by SKY analysis of metaphase spreads from MEFs of the indicated genotypes. Values <br />           in parentheses indicate percentages of chromosomes with the relevant abnormality. aph, aphidicolin. (C and D) Graphs of colony survival of MEFs with <br />           the indicated genotypes treated with aphidicolin (C) or MMC (D). Error bars represent SD. Bar, 10 &#181;m. <br />  <br />  <br />           survive aphidicolin treatment (Fig. 2 C). Two independent Rif1F/F               shown in Fig. S2 D), which is consistent with previous findings <br />           MEF lines and a control wild-type line were treated with Cre re-                with RNAi to Rif1 in human cells (Silverman et al., 2004). <br />           combinase and examined for their sensitivity to aphidicolin. Rif1 <br />           deletion resulted in significantly diminished survival compared                 Rif1-null cells accumulate <br />           with Rif1-proficient cells. The sensitivity to agents interfering di-           S phase&#8211;induced DNA damage <br />           rectly with DNA polymerases appeared to be specific. Rif1-null                  Aphidicolin treatment causes the activation of the DNA replica- <br />           cells also showed an approximate sixfold increase in cell death after           tion checkpoint, which ensures both the stability of the stalled <br />           low doses of hydroxyurea (HU; Fig. S2 B). In contrast, treatment                forks and the activation of the DNA repair pathways responsible <br />           with the interstrand cross-linking agent mitomycin C (MMC;                      for fork restart (Casper et al., 2002; Cha and Kleckner, 2002; <br />           Fig. 2 D) or the topoisomerase inhibitor ICRF-193 (Fig. S2 C) did               Sogo et al., 2002; Zachos et al., 2003; Calzada et al., 2005). <br />           not affect cells in a Rif1-dependent manner. Ionizing radiation                 The hypersensitivity to the aphidicolin treatment of the Rif1- <br />           (IR)&#8211;induced DSBs only marginally affected the survival of Rif1-                null cells could be explained either by a defect in some aspect <br />           null cells (1.3- and 2.5-fold reduced survival compared with Rif1-              of the ATR kinase cascade or by a problem during repair. To test <br />           proficient cells when irradiated with 1 Gy or 5 Gy, respectively, as            the first possibility, we monitored the phosphorylation state of <br />  <br />  <br />     388   JCB &#8226; VOLUME 187 &#8226; NUMBER 3 &#8226; 2009 <br />  Figure 3.&#8195; Rif1 deletion causes accumulation of DNA damage. (A and B) Western blotting for the phosphorylation state of Chk1 and -2. MEFs of the <br /> indicated genotypes were treated with aphidicolin (A) or analyzed untreated (B). Proteins were extracted and analyzed for total levels of Chk1, Chk1 <br /> phosphorylation on Ser345, Chk2 phosphorylation, and levels of cyclin A. As a control, NIH3T3 cells were treated with aphidicolin (aph) or &#57447;-irradiated <br /> (IR). (C and D) Cells were treated or not (no APH) with aphidicolin for 3 h and 30 min. The presence of the DNA damage is identified by IF for &#57447;-H2AX and <br /> is quantified as the mean intensity of specific signal in the nuclei. The percentage of each cell population that showed a value of mean intensity signal &gt;50 <br /> was calculated for each time point. Data represent the mean &#177; SEM of two independent Rif1F/F + Cre and the mean &#177; SD of three wild type (WT; Rif1 WT, <br /> Rif WT + Cre, and Rif1F/F). (C) Mean intensity of &#57447;-H2AX signal after 3 h and 30 min of aphidicolin (APH) treatment is comparable between wild-type and <br /> Rif1-null cells. (D) Time course monitoring the variation of &#57447;-H2AX signal intensity after release from aphidicolin-induced S-phase arrest. Data were plotted <br /> as fold induction over time 0 mean value for each clone. Each time point represents the mean value of two independent Rif1F/F + Cre clones and three wild <br /> type (Rif1 WT, Rif WT + Cre, and Rif1F/F). The breaks in the lines indicate that the &#8220;no APH&#8221; is not a time point in a time course but the basal signal level. <br /> (E) Rif1GT/GT pMEF and wild-type littermate controls were synchronized in G0 and released in BrdU to monitor S-phase progression. At the indicated time <br /> point, cells were collected and analyzed for BrdU content by FACS. One representative experiment out of two is shown. <br />  <br />  <br /> the ATR target Chk1. Aphidicolin treatment of Rif1-null cells                    elicited a low level of Chk1 phosphorylation, whereas the <span id='am-94' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-96'  ></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-98' property="rdfs:label" content="Phos" datatype="xsd:string"></span>phos</span>- <br /> induced the same level of Chk1 phosphorylation as in the Rif1-                   phorylation of Chk2 was not obviously elevated compared with <br /> proficient controls (Fig. 3 A), indicating that the main signal-                 wild-type cells (Fig. 3 B). The specific phosphorylation of Chk1 <br /> ing cascade of the DNA replication checkpoint does not require                   and not Chk2 in Rif1-null cells is consistent with a defect in the <br /> Rif1. In addition, Rif1 deletion itself, without additional treatment,           repair of DNA damage in S phase. Rif1-null cells did not have a <br />  <br />  <br />                                                                                           Rif1 modulates HDR at stalled forks &#8226; Buonomo et al.                   389 <br />                                                                                             long tract gene conversion (LTGC). It contains the two halves <br />                                                                                            of the blasticidin resistance gene, such that I-SceI&#8211;stimulated <br />                                                                                            LTGC leads to the reconstitution of the wild-type gene. To re- <br />                                                                                            duce Rif1 levels, we used two Rif1-specific siRNAs and an <br />                                                                                            irrelevant siRNA to luciferase as a control (Fig. 4 A). Quan- <br />                                                                                            titative immunoblots indicated that the siRNAs reduced Rif1 <br />                                                                                            levels by 30&#8211;60% compared with the level in the luciferase <br />                                                                                            siRNA control cells (Fig. 4 A). Despite the modest knockdown <br />                                                                                            of Rif1, we detected a significant effect on the efficiency of <br />                                                                                            HDR. Cells transfected with siRNA #6 showed a threefold de- <br />                                                                                            crease in HDR, whereas siRNA #4 had a lesser effect (Fig. 4 B). <br />                                                                                            As it has been shown for two other genes involved in HDR, <br />                                                                                            BRCA1 and -2, depletion of Rif1 leads to a reduced S phase <br />                                                                                            (Fig. S3 A; Patel et al., 1998; Kumaraswamy and Shiekhattar, <br />                                                                                            2007). Diminished Rif1 levels also affected the efficiency of <br />                                                                                            LTGC to an extent proportional to the magnitude of the knock- <br />                                                                                            down achieved by the different siRNAs. <br />  <br />                                                                                            Rif1 localizes at sites of replication <br />                                                                                            fork stalling, mostly at pericentromeric <br />                                                                                            heterochromatin <br />           Figure 4.&#8195; Rif1 promotes HDR. (A and B) Knockdown of Rif1 decreases <br />           HDR efficiency. (A) Rif1 levels were reduced by two specific siRNAs, #4          Given the possibility of a role for Rif1 at stalled replication forks, <br />           and #6, in U2OS#18. siRNA against luciferase (Luc si) was used as a              we examined the distribution of Rif1 during an unperturbed cell <br />           control. Western blot showing a titration to validate the reduction of Rif1 <br />                                                                                            cycle. Cells were briefly labeled with BrdU such that sites of <br />           levels upon treatment with specific siRNAs. mTOR was used as a loading <br />           control. (B) Upon DSB induction by I-SceI transfection, HDR was evaluated        DNA replication could be detected with anti-BrdU antibodies <br />           by quantifying the amount of GFP-positive cells by FACS. The effect on HDR       under denaturing conditions. This technique allows the identifi- <br />           is expressed as the percentage of the luciferase siRNA. P-values are &lt;0.05 <br />                                                                                            cation of cells in different stages of S phase (Fig. 5; Nakamura <br />           for both siRNAs. (C) LTGC efficiency was calculated by counting the num- <br />           ber of colonies growing in the presence of blasticidin and normalized for        et al., 1986; Dimitrova and Berezney, 2002). Immunofluores- <br />           plating efficiency. The effect of LTGC is expressed relative to the luciferase   cence (IF) for Rif1 showed a punctuate nuclear staining pattern <br />           siRNA control. (B and C) Error bars represent SD. <br />                                                                                            in late G1 with more prominent signals at the pericentromeric <br />                                                                                            heterochromatin. In early and mid S phase, Rif1 progressively <br />           defect in their initial response to replication stress, as measured              concentrated at pericentromeric heterochromatin. The Rif1 sig- <br />           based on the level of &#57447;-H2AX in cells treated with aphidicolin                   nal never overlapped with BrdU or proliferating cell nuclear <br />           (Fig. 3 C). However, the aphidicolin-induced &#57447;-H2AX response                     antigen (unpublished data) except when pericentromeric hetero&#173; <br />           persisted in Rif1-null cells, which is consistent with a defect                  chromatin started to be replicated in mid S phase. At that <br />           in the repair of aphidicolin-induced DNA damage (Fig. 3 D).                      stage, some BrdU foci colocalized with Rif1, again mostly at <br />           To test more directly the requirements for Rif1 during normal                    pericentromeric heterochromatin. In mid&#8211;late and late S phase, <br />           S-phase progression, we synchronized in G0 Rif1GT/GT pMEFs                       Rif1 appeared to have left the chromatin, showing a diffuse nu- <br />           and analyzed their progression through one cell cycle. Rif1-null                 clear staining pattern. These data raised the possibility that Rif1 <br />           cells appeared to transit through S phase reproducibly slower                    has a specific function in the replication of certain regions of the <br />           than wild-type controls (Fig. 3 E and Fig. S2 E), which is con-                  genome, including the pericentromeric heterochromatin. <br />           sistent with the presence of replication-induced DNA damage.                            To determine whether the sites of BrdU&#8211;Rif1 colocaliza- <br />                                                                                            tion represent sites of fork stalling, we visualized stalled repli- <br />           Rif1 stimulates homology-directed                                                cation forks in aphidicolin-treated cells based on the presence <br />           repair (HDR)                                                                     of ssDNA using BrdU staining without denaturation (Fig. 6, A <br />           To further explore the role of Rif1 in the repair of S-phase dam-                and B). Rif1 colocalized with ssDNA in only &#57466;12% of the un- <br />           age, we used a previously established assay for HDR, the main                    treated and 27% of the aphidicolin-treated BrdU-positive cells <br />           pathway for the repair of replication-associated DNA damage                      (Fig. 6 A and Fig. S5 B). In these cells, only a subset of the <br />           (Johnson and Jasin, 2000; Puget et al., 2005; for reviews see                    stalled forks showed the presence of Rif1 (Fig. 6 A), mostly, but <br />           Cromie et al., 2001; Helleday, 2003). The HDR assay is based                     not exclusively, at or near the pericentromeric heterochromatin <br />           on a human U2OS cell line containing two tandem mutant                           (Fig. 6 A, bottom). Control experiments with Rif1&#57394;/&#57394; cells did <br />           copies of the gene encoding GFP. The first copy is promoterless                  not show foci of Rif1 coinciding with BrdU, establishing the <br />           and contains a small deletion in the 5&#57401; end; the second has a                    specificity of the antibody (Fig. S3 B). To further verify the lo- <br />           promoter but has an I-SceI site interrupting the GFP ORF. Upon                   calization pattern of Rif1, we executed dual IF for the ssDNA- <br />           introduction of I-SceI, an intact GFP gene can be generated                      binding protein RPA34 and Rif1 (Fig. S3 C). Consistent with <br />           through DSB repair by unequal sister or intrachromatid recom-                    the BrdU data, there was significant colocalization of Rif1 with <br />           bination. The same reporter allows for assaying the efficiency of                a subset of the RPA34 foci. These data support the <span id='am-21' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23'  ></span><span id='am-24' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>idea</span> that the <br />  <br />  <br />     390   JCB &#8226; VOLUME 187 &#8226; NUMBER 3 &#8226; 2009 <br />  Figure 5.&#8195; Rif1 binds chromatin before and during DNA replication but only seldom colocalizes with the replication fork in mid&#8211;late S phase and around <br /> pericentromeric heterochromatin. IF on wild-type MEFs for Rif1 (red) and BrdU visualized by denaturing BrdU staining (green). Late G1, early, mid, <br /> mid&#8211;late, and late S were identified by the BrdU staining pattern. Insets contain enlarged details of the indicated areas. <br />  <br />  <br /> Rif1&#8211;BrdU colocalization at pericentromeric heterochromatin                  centromeric heterochromatin. On the contrary, Rif1 is not re- <br /> of unperturbed cells reflects sites of fork stalling.                        quired for 53BP1 or BLM localization at stalled replication <br />                                                                              forks (Fig. S5 D). This finding is in agreement with the fact that <br /> Rif1 localization at stalled replication forks                               Rif1-deficient cells do not show an increase in sister chromatid <br /> requires ATR and 53BP1                                                       exchanges (unpublished data). <br /> To further characterize Rif1 foci, we analyzed the presence of <br /> other fork-stalling response factors. The Rif1 foci at ssDNA                 Aberrant Rad51 aggregates in <br /> always contained 53BP1 (Fig. S4 A), although there were ssDNA                Rif1-deficient cells <br /> sites containing 53BP1 but no Rif1. Similarly, Rif1 foci at                  Because we found that Rif1 contributes to the efficiency of HDR, <br /> stalled forks colocalized with the BLM RecQ helicase, but not                we examined the behavior of one of the main effectors of homolo- <br /> all BLM foci coincided with Rif1 signals (Fig. S4 B).                        gous recombination, the protein Rad51, in Rif1-deficient MEFs. <br />       The localization of Rif1 at IR-induced DSBs is depen-                  Co-IF for endogenous Rad51 and 53BP1 revealed that 15% of the <br /> dent on ATM and 53BP1 (Silverman et al., 2004). In contrast,                 Rad51-positive cells accumulated large aberrant Rad51 aggregates <br /> ATM&#57394;/&#57394; MEFs retained the ability to accumulate Rif1 at                       during S phase in the absence of Rif1 (Fig. 7 A). These types of <br /> aphidicolin-induced ssDNA, whereas the absence of 53BP1                      structures were present exclusively in cells that were in S phase <br /> abrogated this response (Fig. 6 B). This suggested that the ATR              (Fig. 7 B; Starborg et al., 1996) and were virtually absent from wild- <br /> kinase might be involved in providing the signal responsible                 type cells (Fig. 7, A and D). In contrast, the overall percentage of <br /> for the localization of 53BP1 and Rif1 at the stalled forks. Con-            Rad51-positive cells was not significantly different between wild- <br /> sistent with this, the Rif1 pattern was significantly altered by             type and Rif1-null cells (Fig. 7 C). The presence of these aggregates <br /> treating the cells with the PI3-like kinase inhibitor caffeine               could be explained if Rif1 had a direct or indirect role in promoting <br /> (Fig. 6 B) or by inhibiting ATR with a specific short hairpin                late steps of Rad51 turnover at the sites of recombination. Alterna- <br /> RNA (shRNA; Fig. S5, A&#8211;C). Together, the data suggest that                   tively, their presence could be an indirect consequence of the accu- <br /> Rif1 requires ATR and 53BP1 to localize to a subset of stalled               mulation of damage during the replication of specific regions of the <br /> replication forks, very often located in the proximity of peri-              genome that cannot be repaired in the absence of Rif1. <br />  <br />  <br />                                                                                      Rif1 modulates HDR at stalled forks &#8226; Buonomo et al.                391 <br />            Figure 6.&#8195; Rif1 localizes to sites of replication stress. (A) IF on wild-type MEFs for Rif1 (red) and ssDNA visualized by nondenaturing BrdU staining (green). <br />           The bottom row shows an example of Rif1 localized at stalled replication forks near heterochromatin. (B) Rif1 localization at aphidicolin-induced sites of <br />           ssDNA is abolished in 53BP1-null cells and upon caffeine treatment. IF for Rif1 (red) and ssDNA (green) on ATM&#57394;/&#57394;, 53BP1&#57394;/&#57394;, and caffeine-treated wild- <br />           type MEFs. All cells were treated with aphidicolin. (A and B) Insets contain enlarged details of the indicated areas. <br />  <br />  <br />           Discussion                                                                       for Rif1 for cellular and embryonic survival can be explained <br />                                                                                            from the S-phase function of Rif1 documented in this study. <br />           Our results identify Rif1 as a novel component of the DNA                        We show that upon ATR activation and 53BP1 recruitment, <br />           replication checkpoint, essential for embryonic development                      Rif1 localizes at a subset of stalled replication forks, where it <br />           and normal DNA replication. We had previously shown that                         could be important to regulate HDR during replication. In- <br />           Rif1 was contributing to the inhibition of DNA replication in                    deed, Rif1 deletion induces accumulation of chromatid-type <br />           response to ATM activation by DSBs (Silverman et al., 2004).                     genome instability and, as a consequence, renders cells hyper- <br />           However, Rif1 is essential, whereas ATM is not. The requirement                  sensitive to aphidicolin and HU. We have also shown that Rif1 <br />  <br />  <br />     392   JCB &#8226; VOLUME 187 &#8226; NUMBER 3 &#8226; 2009 <br />  Figure 7.&#8195; Aberrant Rad51 aggregates in Rif1-deficient cells. (A) IF for Rad51 (green) and 53BP1 (red) in Rif1F/F (left) and wild-type (right) MEFs treated <br /> with Cre and aphidicolin. Arrowheads point to aberrant Rad51 aggregates. Insets contain enlarged details of the indicated areas. (B) IF for Rad51 (green), <br /> Ki67 (red), and DAPI staining (blue) on Rif1F/F MEFs treated with Cre and aphidicolin (aph). (C and D) Quantification of the percentage of Rad51-positive <br /> cells (C) and of the percentage of Rad51-positive (pos) cells containing Rad51 large aggregates (D). One representative experiment out of two is shown. <br /> wt, wild type. <br />  <br />  <br />  <br /> localizes at stalled replication forks preferentially during                    (Nakamura et al., 2008). Therefore, we hypothesize that Rif1 <br /> mid&#8211;late S phase, often at/around replicating pericentromeric                   regulates Rad51-dependent homologous recombination at <br /> heterochromatin. Centromeres, like telomeres, are highly                        replication forks stalled in regions of the genome that are dif- <br /> repetitive sequences and are highly recombinogenic (Jaco                        ficult to replicate like repetitive sequences or tight protein&#8211; <br /> et al., 2008; Nakamura et al., 2008). In particular, it was re-                 DNA complexes. <br /> cently shown that Rad51 function is essential to suppress gross                       These findings can be reconciled with observations in <br /> chromosomal rearrangement at centromeres in fission yeast                       yeast, in which Rif1 has a telomere-specific function. Budding <br />  <br />  <br />                                                                                         Rif1 modulates HDR at stalled forks &#8226; Buonomo et al.                  393 <br />            yeast strains that survive without telomerase maintain their           Materials and methods <br />           telomeres either through Rad51-dependent HDR (type I <br />           survivors) or through a less understood Rad50-dependent                Gene targeting <br />                                                                                  The mouse Rif1 locus contains 36 exons, with the second exon containing <br />           pathway (type II survivors; Lundblad and Blackburn, 1993;              the ATG for the 2,426&#8211;amino acid ORF. The targeting strategy allows <br />           Lendvay et al., 1996; Le et al., 1999; Teng and Zakian, 1999).         Cre-mediated deletion of exons 5, 6, and 7; the splicing of exon 4 to <br />           In the absence of Rif1, the frequency of type I survivors is           exon 8 generates a frame shift. The targeting vector was generated by <br />                                                                                  recombineering (Lee et al., 2001) using bacterial artificial chromosome <br />           diminished (Teng et al., 2000), which is consistent with a             RCPI-23 395O2 (American Type Culture Collection) in <span id='am-46' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span>Escherichia coli</span> <br />           role for Rif1 in the Rad51 pathway. Data from fission yeast            strain EL350. The first LoxP site was introduced in the unique EcoRV site <br />           Rif1 can also be explained from an HDR perspective. When               upstream of exon 4 by inserting an oligonucleotide that also introduced <br />                                                                                  SacI and BamHI restriction sites used for the analysis of targeting in ES <br />           the telomeric protein Taz1 is absent, fission yeast telomeres          cells. The unique EcoRI site 3&#57401; of exon 7 was used to insert a LoxP-FRT- <br />           can become entangled, presumably reflecting a pathological             NeomycinR(NeoR)-FRT cassette (gift from D. O&#8217;Carroll, European Molecu- <br />           product of an HDR event taking place after replication fork            lar Biology Laboratory Monterotondo, Monterotondo, Italy). The construct <br />                                                                                  was subcloned into the pDTA-TKIII vector, which allows double negative <br />           arrest (Miller and Cooper, 2003; Miller et al., 2006). Rif1 de-        selection (DTA and TK genes), linearized with NotI, and electroporated <br />           ficiency suppresses the lethality associated with this process,        into Bruce4 C57BL/6 ES cells. Four independent ES cell clones were <br />           which is consistent with a role in HDR (Miller et al., 2005).          injected by standard techniques into albino C57BL/6J blastocysts, and <br />                                                                                  chimeras were evaluated based on coat color. Chimeric founders were <br />           Therefore, modulation of Rad51-dependent HDR could be                  crossed to C57BL/6J females, and the mice were kept in a pure C57BL/6J <br />           Rif1&#8217;s function conserved during evolution. The finding that           background. The NeoR cassette was removed by crossing the F1 mice <br />           Rif1 depends on 53BP1 and positively contributes to HDR                with an FLPe deleter mouse strain (Jackson ImmunoResearch Laboratories, <br />                                                                                  Inc.). Genotyping of mice and derived cells was performed by PCR with <br />           execution is in apparent conflict with observations on 53BP1-          the following primers: f1, 5&#57401;-TTAGAGGAACTGAGGGGTAGGTAG-3&#57401;; <br />           null MEFs. Deletion of 53BP1 does not sensitize cells to               r1, 5&#57401;-AACTGCAACTCTACTGAGGGAAG-3&#57401;; and r2, 5&#57401;-TGAAACCG- <br />           aphidicolin (unpublished data) and promotes HDR (Adams                 TAGCCAGAAACTG-3&#57401;. PCR generated a 78-bp product for the wild-type <br />                                                                                  allele, a 138-bp product for the LoxP-modified (FLOX) allele, and a 246-bp <br />           et al., 2005; Xie et al., 2007). This discrepancy could be ex-         product for the deleted (DEL) allele. The southern probes p1 and p2 were <br />           plained if Rif1 is one of several factors that act downstream          generated by PCR from genomic DNA with the following primers: p1 E3s, <br />           of 53BP1 to navigate replication problems such that its dele-          5&#57401;-GAATTGTAGTTAATGATGGGCC-3&#57401;; and E3a 5&#57401;-TCTATGGATGTAGCT- <br />                                                                                  TAGTGGTACA-3&#57401;; and p2 E8a, 5&#57401;-TGGTTCACATCTACGATCCCA-3&#57401;; and <br />           tion creates an imbalance in HDR that leads to pathologi-              E8s 5&#57401;-GAATTGTAGTTAATGATGGGCC-3&#57401;. <br />           cal products. One interesting possibility is that Rif1 inhibits                Rif1GT mice were generated from an ES cell clone carrying a <br />           (directly or not) BLM, which has been proposed to have a               trapped Rif1 allele (A045A01) available from the German Gene Trap <br />                                                                                  Consortium (http://genetrap.gsf.de/). Chimeric founders were crossed to <br />           53BP1-regulated role at stalled forks (Sengupta et al., 2004;          C57BL/6J females, and the mice were kept in a mixed 129/C57BL/6J <br />           Bugreev, 2007; Tripathi et al., 2007, 2008). Inhibition of BLM         background. Genotyping of mice and derived cells was performed by PCR <br />           by Rif1 would be consistent with genetic data on the survi-            with the following primers: f2, 5&#57401;-GCTAGTCTAGGCACCAGGAACC-3&#57401;; <br />                                                                                  f3, 5&#57401;-AGGAGAAAATACCGCATCAGG-3&#57401;; and r3, 5&#57401;-TCCTCAGCAT- <br />           vor pathway in yeast, which allows budding yeast to prolifer-          CAACCAAAGAG-3&#57401;. The PCR products were 135 bp and 190 bp for <br />           ate without telomerase (McEachern and Haber, 2006). The                the wild-type and GT allele, respectively. The southern probe p3 was <br />           yeast orthologue of BLM, Sgs1, is required for the type II             generated by PCR from genomic DNA with the following primers: E22s, <br />                                                                                  5&#57401;-TCAGATTGGTCCAGAAAGAA-3&#57401;; and E24a, 5&#57401;-CATTCCACAA- <br />           (Rad50 dependent) survivor pathway (Johnson et al., 2001),             CAGAGCACTCT-3&#57401;. The Neo probe was obtained by digesting the plas- <br />           but this requirement is alleviated by the deletion of Rif1             mid pZero2-LoxP-Neo-LoxP with PstI and NcoI and isolating the fragment <br />           (Tsai et al., 2006). Because Rif1 localizes only to a subset           corresponding to part of the NeoR ORF. <br />  <br />           of the stalled replication forks, the interplay between BLM <br />                                                                                  Generation of MEFs, conditional deletion of Rif1, and cell manipulation <br />           and Rif1 could direct the choice between different types of            MEFs were generated from E12.5 embryos according to standard pro- <br />           HDR at single forks, depending on the DNA sequence and                 tocols and propagated in DME high glucose, 15% heat-inactivated FBS <br />           chromatin context.                                                     (Invitrogen), 100 U/ml penicillin, 0.1 mg/ml streptomycin, 0.2 mM l-Gln, <br />                                                                                  0.1 mM nonessential amino acids, and 50 &#181;M &#57442;-mercaptoethanol. To <br />                  Finally, the finding that Rif1 plays a role in HDR is rele-     immortalize primary MEFs, early passage (P2) cells were infected with <br />           vant to the recent identification of Rif1 mutations in human           pBabeSV40LT (gift from G. Hannon, Cold Spring Harbor Laboratory, Cold <br />           breast cancer. One breast cancer cell line (HCC2218) was found         Spring Harbor, NY). <br />                                                                                          ATM&#57394;/&#57394; MEFs were prepared from heterozygous intercrosses of <br />           to have two mutations in Rif1 (E1784K and D1955H; Sj&#246;blom              ATM+/&#57394; mice (129S6/SvEvTac-Atmtm1Awb/J; Jackson ImmunoResearch <br />           et al., 2006), and a second breast cancer cell line (HCC1806)          Laboratories, Inc.). 53BP1&#57394;/&#57394; MEFs were prepared from heterozygous <br />           carries a translocation that truncates Rif1 after exon 21              inter&#173;crosses of mice provided by J. Chen (Yale University School of Medi- <br />                                                                                  cine, New Haven, CT). MEFs were immortalized with SV40LT as described <br />           (Howarth et al., 2008). According to our data on Rif1/&#57442;-GEO,           in the previous paragraph. <br />           this truncation is expected to be nonfunctional. The possible in-              For Cre-mediated deletion of the Rif1 gene, cells were infected six <br />           activation of Rif1 in sporadic cases of breast cancer is of interest   times at 6&#8211;8-h intervals with the supernatant from <span id='am-1' about='obo:NCBITaxon_4719' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4710 obo:NCBITaxon_4734 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_131221 obo:NCBITaxon_169748 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_169700 obo:NCBITaxon_40551 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Phoenix L." datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Phoenix" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Phoenix</span> packaging cells <br />                                                                                  (American Type Culture Collection) transfected by standard calcium phos- <br />           with regard to the BRCA2 gene, whose inactivation predisposes          phate protocol with either empty pWZL-hygro (gift from S. Lowe, Cold <br />           to breast and ovarian cancer. Like Rif1, BRCA2 is implicated in        Spring Harbor Laboratory, Cold Spring Harbor, NY) or pWZL-hygro-CRE. <br />           the Rad51 pathway (for review see Thorslund and West, 2007).           Cells were split into medium containing hygromycin B (Sigma-Aldrich) at <br />                                                                                  90 &#181;g/ml and grown for 3 d. <br />           Given the current interest in treating BRCA2-deficient breast                  Growth curves were determined by plating 34,000 or 68,000 cells <br />           cancer with therapies based on their inability to execute HDR          per well of a 6-well plate in triplicate for each day of the experiment, for <br />           efficiently (Bryant et al., 2005; Farmer et al., 2005), it may be      immortalized and primary cells, respectively. Cells were trypsinized and <br />                                                                                  counted with a Z2 counter (Beckman Coulter). <br />           useful to consider sporadic tumors with Rif1 mutations for this                Aphidicolin sensitivity was measured by plating 1,400 and 1,000 <br />           treatment approach.                                                    cells per well of a 6-well plate for Rif1F/F and wild-type immortalized MEFs, <br />  <br />  <br />     394   JCB &#8226; VOLUME 187 &#8226; NUMBER 3 &#8226; 2009 <br />  respectively, in triplicate. After 24 h, cells were either left untreated or    Analytical Technologies). After application of the threshold in channel 1 <br /> treated for 24 h with the different doses of aphidicolin, washed three times    (DAPI), objects were identified and masked. The images and masks were <br /> with PBS, and incubated in growth media for about 1 wk. After rinsing with      visually inspected for accuracy. Then, the percentage of cells with a mean <br /> PBS, cells were fixed for 10 min in a solution containing 50% methanol,         intensity signal of channel 2 &gt;50 was calculated (n &gt; 100) for each condi- <br /> 7% glacial acetic acid, and 0.1% Coomassie brillant blue. The number of         tion and normalized to the percentage of the same cell line at time 0. <br /> colonies with &gt;20 cells was counted. MMC sensitivity was tested in the <br /> same conditions, except that the drug treatment was performed only for          Western blotting <br /> 2 h and 30 min.                                                                 To monitor Chk1 Ser345 phosphorylation in response to aphidicolin, cells <br />        G0 synchronization of pMEFs was obtained by 5 d of contact inhibi-       were treated for 3 h and 30 min with 5 &#181;M aphidicolin and collected. <br /> tion and 48 h of serum starvation (0.1% FBS). Cells were pulsed with            Control irradiated NIH3T3 were treated with 5 Gy and allowed to recover <br /> 10 &#181;M BrdU for 1 h and subsequently released in 15% FBS medium con-             for 30 min before preparing protein extracts. Cells were collected by tryp- <br /> taining 10 &#181;M BrdU by plating 1.5 &#215; 106 cells per 10 cm.                        sinization, rinsed in PBS, directly lysed in 2&#215; Laemmli buffer, and passed <br />                                                                                 through a 29-gauge insulin syringe. Alternatively, cells were lysed in <br /> BrdU staining for FACS analysis                                                 50 mM Tris-HCl, pH 7.4, 1% Triton X-100, 0.1% SDS, 150 mM NaCl, <br /> Asynchronous cell populations were pulsed for 30 min with 10 &#181;M BrdU.           1 mM EDTA, phosphatase, and protease inhibitors. Chromatin-bound pro- <br /> After fixation in 70% ethanol at &#57394;20&#176;C, 3 &#215; 105 cells were stained accord-      teins were extracted by increasing the NaCl concentration to 500 mM. The <br /> ing to the BD protocol, with 10&#8211;15 &#181;l per sample of anti-BrdU FITC conju-       extract was centrifuged for 10 min at 13,200 rpm at 4&#176;C, and the super- <br /> gated (BD). The samples were analyzed by FACS on a FACSCalibur System           natant was recovered. Proteins were separated by gel electrophoresis and <br /> One <span id='am-79' about='obo:NCBITaxon_62990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_241799 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_241789 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="Laser" datatype="xsd:string"></span><span id='am-83'  ></span>laser</span> (488 nm; BD), and the data were analyzed with FlowJo software         blotted according to standard protocols. All of the Western blots, with the <br /> (Tree Star, Inc.).                                                              exception of phospho&#8211;Ser345 Chk1, were blocked in 5% milk/PBS/0.1% <br />                                                                                 Tween, and the primary antibody was incubated in 1% milk/PBS/0.1% <br /> Aphidicolin treatment, metaphase spreads, and spectral karyotyping              Tween. For the anti-phospho&#8211;Ser345 Chk1 antibody (Cell Signaling Tech- <br /> analysis (SKY)                                                                  nology), the manufacturer&#8217;s instructions were followed. <br /> Cells were treated for 24 h with or without 0.3 &#181;M aphidicolin, followed <br /> by 4 h with 0.2 &#181;g/ml colcemid, and then collected by trypsinization and        shRNA, siRNA, and sister chromatid recombination assay <br /> treated according to standard protocols to prepare metaphase chromo-            Knockdown of ATR in wild-type or ATM&#57394;/&#57394; MEFs was performed as de- <br /> somes. Metaphase spreads were analyzed either by DAPI staining or by            scribed previously (Denchi and de Lange, 2007). In brief, cells plated on <br /> SKY (Liyanage et al., 1996) using paint mixtures from Applied Spectral          coverslips were infected four times in 24 h with retroviruses carrying <br /> Imaging according to the manufacturer&#8217;s protocol. Imaging was per-              either the ATR or the control luciferase shRNA. After 24 h in 2 &#181;g/ml puro- <br /> formed on a microscope (BX-61; Olympus) equipped with an interferome-           mycin and 3 &#181;M BrdU, cells were incubated for 3 h in 5 &#181;M aphidicolin <br /> ter driven by a desktop computer and SpectraView software (Applied              and processed for IF as described in the IF section. Sister chromatid re- <br /> Spectral Imaging).                                                              combination assay was performed using U2OS#18 (gift from R. Scully, <br />                                                                                 Harvard University, Cambridge, MA) as described previously, with minor <br /> IF                                                                              modifications (Puget et al., 2005). In brief, 50,000 cells were plated per <br /> Cells grown on coverslips were rinsed with PBS and extracted for 10 min         well of a 12-well plate in triplicate for each siRNA. siRNA transfections <br /> at 4&#176;C in Triton buffer (0.5% Triton X-100, 20 mM Hepes, pH 7.9, 50 mM          were performed as described previously (Silverman et al., 2004), except <br /> NaCl, 3 mM MgCl2, and 300 mM sucrose), fixed for 10 min in 3% para-             for the use of DharmaFECT Duo (Thermo Fisher Scientific) instead of <br /> formaldehyde/2% sucrose at room temperature, and permeabilized                  Oligofectamine (Invitrogen). The sequences of siRNA #4 and #6 and <br /> again for 10 min at 4&#176;C in Triton buffer. IF was performed as described         luciferase were published previously (Silverman et al., 2004). The siRNAs <br /> previously (Zhu et al., 2000), with minor modifications. In brief, cells were   were synthesized either by Thermo Fisher Scientific or Mycrosynth. 1 d after <br /> blocked in PBG buffer (0.2% cold fish gelatin [Sigma-Aldrich] and 0.5%          the second transfection, the medium was changed, and cells were col- <br /> BSA) for 30 min at room temperature. The primary antibodies were di-            lected for Western blot analysis and for blasticidin selection. 250,000 <br /> luted in PBG and incubated for 1 h at room temperature. The fluorescent-        cells were plated in duplicate in a 10-cm dish and, after 24 h, were <br /> conjugated secondary antibodies were also diluted in PBG and incubated          switched to full medium containing 5 &#181;g/ml blasticidin. 200 cells were <br /> for 45 min.                                                                     also plated in a 6-cm dish in triplicate to calculate plating efficiency. Cells <br />         Nondenaturing BrdU IF was performed according to Raderschall            were collected for GFP FACS at 48 h after the second transfection. During <br /> et al. (1999). In brief, cells were labeled with 3 &#181;M BrdU for 25 h. After      the FACS analysis, at least 30,000 events per sample were acquired. <br /> harvesting, IF was performed as described (Raderschall et al., 1999),           Statistical analysis was performed using a standard t test with a two- <br /> using the BD anti-BrdU antibody at 1:1,000. An appropriate secondary            tailed distribution and equal variance. <br /> fluorescent-conjugated anti&#8211;mouse antibody was used to detect the BrdU. <br />         Denaturing BrdU IF was performed by pulsing cells for 30 min with       Antibodies <br /> 10 &#181;M BrdU. Cells were pre-extracted and fixed as described in the previ-       Rabbit anti&#8211;mouse Rif1 serum 1240 was raised against a keyhole limpet <br /> ous paragraph. 100 U/ml DNaseI and 3 mM MgCl2 were added to the                 hemocyanin&#8211;conjugated peptide spanning amino acids 401&#8211;424 of the <br /> primary antibody mix to denature the DNA.                                       mouse Rif1 sequence. The antibody was affinity purified and used at a <br />         For the triple staining, either one or both primary rabbit antibodies   1:10,000 dilution for IF and Western blotting. Other antibodies used in <br /> were directly conjugated to the fluorophore using the Zenon kit (Invitrogen)    this study were anti-53BP1 (1:1,000; Novus Biologicals), a mouse anti- <br /> according to the manufacturer&#8217;s instructions. Anti-BrdU antibody was de-        53BP1 serum generated against the C-terminal 312 amino acids of human <br /> tected using a Cy5-conjugated anti&#8211;mouse secondary antibody (Millipore).        53BP1 and fused to GST (1:2,000), anti-BLM (1:200; Abcam), anti-RPA <br /> To assay the effect of PI3-like kinase&#8217;s inhibition on Rif1 localization at     (1:500; GeneTex, Inc.), anti-Ki67 (1:1,000; BD), anti&#8211;&#57447;-H2AX (1:1,000; <br /> stalled replication forks, wild-type MEFs were incubated for 3 h with 10 mM     Bethyl Laboratories, Inc.), and anti-Rad51 (1:2,500 for IF; gift from <br /> caffeine before aphidicolin treatment. Caffeine was maintained during           R. Kanaar, Erasmus Medical Center, Rotterdam, Netherlands; van Veelen <br /> the aphidicolin treatment. Slides were mounted in embedding medium              et al., 2005). For Western blotting, anti-Chk2 (BD) was used at 1:1,000, <br /> (1 mg/ml p-phenylene diamine [Sigma-Aldrich] in 1&#215; PBS and 90% glyc-            anti-Chk1 (Santa Cruz Biotechnology, Inc.) was used at 1:200; anti- <br /> erol). Images were acquired with a microscope (Axioplan 2 [Carl Zeiss,          phospho&#8211;Ser345 Chk1 (Cell Signaling Technology) was used at 1:1,000, anti- <br /> Inc.], DM6000 [Leica], or confocal TCS SP5 [Leica]) with a 63&#215; 1.4 NA oil       ATR (Santa Cruz Biotechnology, Inc.) was used at 1:500, anti&#8211;mammalian <br /> objective. The <span id='am-41' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-42' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45'  ></span>camera</span> used was either a C4742-95 (Hamamatsu Photonics)          target of rapamycin (mTOR; Cell Signaling Technology) was used at <br /> or a DFC340FX (Leica). The acquisition software used was either Open-           1:1,000, and anti&#8211;cyclin A (Cell Sciences) was used at 1:1,000. <br /> Lab (PerkinElmer) or LAS AF (Leica). Contrast adjustment and cropping was <br /> performed in Photoshop (Adobe), and figures were composed in Illustrator        Online supplemental material <br /> (Adobe). For the confocal images (Fig. 7), maximum projection is shown.         Fig. S1 shows that insertional inactivation of Rif1 leads to embryonic <br />         In the case of the quantification of the &#57447;-H2AX signal over the time    lethality and reduced proliferation of MEFs. Fig. S2 exemplifies that Rif1- <br /> course (Fig. 3, C and D), cells from two Rif1F/F + Cre and three wild type      null cells exhibit a reduced S-phase index and are hypersensitive to HU, <br /> were imaged using a wide-field DM6000 microscope with a 63&#215; objective           as sensitive as wild-type cells to ICRF-193, and only mildly sensitive to <br /> (30 fields from each time point). Two channels were imaged: channel 1 to        &#57447;-irradiation. Fig. S3 shows a decreased S-phase index in cells treated with <br /> find the nuclei (DAPI) and channel 2 to image the &#57447;-H2AX signal. The mean       Rif1-specific siRNAs and demonstrates the specificity of IF with anti&#8211;mouse Rif1 <br /> intensity of channel 2 was quantified by using MetaMorph software (MDS          antibody 1240 and Rif1&#8211;RPA34 colocalization. Fig. S4 illustrates that Rif1 <br />  <br />  <br />                        <br />         colocalizes with BLM and 53BP1 at stalled replication forks. Fig. S5 shows           <br />           that the Rif1 localization at stalled replication forks depends on ATR and                <br />           that Rif1 is not required for 53BP1 and BLM localization at stalled rep-                <br />           lication forks. Online supplemental material is available at http://www              <br />           .jcb.org/cgi/content/full/jcb.200902039/DC1.                                        <br />           We gratefully acknowledge Devon White for mouse husbandry, The Rocke-                <br />           feller University (RU) and Memorial Sloan-Kettering Cancer Center Transgenic            <br />           facilities for ES cell injections, the RU FACS facility, Daniel Bilbao at the Euro-        <br />           pean Molecular Biology Laboratory (EMBL) FACS facility, and Valeria Berno             <br />           at EMBL Monterotondo for confocal images and precious support in the                       <br />           quantification of the images. We thank Donal O&#8217;Carroll and Alexander                      <br />           Tarakhovsky for invaluable help with mouse protocols and scientific input.                  <br />           We greatly appreciate Eros Lazzerini Denchi, Hiro Takai, and Agnel Sfeir for      <br />           their support with helpful discussions and protocols. R. Scully provided the                 <br />           U2OS#18 line, R. Kanaar provided the Rad51 antibody, and J. Chen pro-                      <br />           vided the 53BP1-null mice. <br />   <br /> </body></html>