<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>J protein cochaperone mitochondrial inner <br /> membrane required protein import the <br /> mitochondrial matrix <br /> Patrick D. D&#8217;Silva*, <span id='am-74' about='obo:NCBITaxon_139583' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30102 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_35567 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7524 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_139578 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33365 obo:NCBITaxon_33342 obo:NCBITaxon_33368 obo:NCBITaxon_50557'><span id='am-75'  ></span><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Brenda" datatype="xsd:string"></span>Brenda</span> Schilke*, William Walter*, Amy Andrew*&#8224;, Elizabeth A. Craig*&#8225; <br /> *Department Biochemistry, 433 Babcock Drive, University Wisconsin, Madison WI 53706; &#8224;Graduate Program Biomolecular Chemistry, <br /> 1300 University Avenue, University Wisconsin, Madison, WI 53706 <br />  <br /> Contributed Elizabeth A. Craig, September 24, 2003 <br />  <br /> The major Hsp70 mitochondrial matrix (Ssc1 yeast)               associated J protein Sec63 stimulate ATPase activity the <br /> critically important translocation proteins              luminal Hsp70 Kar2 essential translocation proteins <br /> cytosol, mitochondrial inner membrane,               cytosol endoplasmic reticulum lumen (9&#8211;11). <br /> matrix. Tim44, peripheral inner membrane protein limited                 No J protein mitochondria reported involved <br /> sequence similarity J domain J-type cochaperones, tethers          protein translocation. However, mitochondrial J pro- <br /> Ssc1 import channel. Here report that, unlike J protein,         teins studied yeast <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-17'  ></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>. <br /> Tim44 does stimulate ATPase activity Ssc1, does            The soluble matrix J protein Mdj1 known function Ssc1 <br /> affect stimulation known mitochondrial J protein          folding imported mitochondrially encoded proteins but <br /> a peptide substrate. Thus, conclude Tim44 does                   does play significant role import (12, 13). The J protein <br /> function J protein cochaperone Ssc1; rather, tethers Ssc1         Jac1 functions minor Hsp70 matrix, Ssq1, the <br /> to import channel interactions independent              biogenesis Fe&#20862;S clusters (14, 15). Mdj2 J protein the <br /> critical J protein function. However, previously unstudied             inner membrane. However, deletion MDJ2 little no <br /> essential gene, PAM18, encodes 18-kDa protein contains             phenotypic effect; thus, function remains obscure (16). <br /> J domain localized mitochondrial inner membrane.                  Tim44, mitochondrial inner membrane protein, been <br /> Pam18 stimulates ATPase activity Ssc1; depletion Pam18             considered possible J protein partner Ssc1. Tim44, which <br /> in vivo disrupts import proteins mitochondrial matrix.           serves tether Ssc1 import channel, shows weak <br /> We propose Pam18 J protein partner Ssc1               similarity J domain Sec63, deletion region <br /> import channel critical Ssc1&#8217;s function protein import.        disrupts interaction Tim44 Ssc1 mitochondria (17). <br />                                                                              Although controversial, recent reports indicate Tim44 <br />                                                                              interacts ATPase domain Ssc1, does J domain <br /> M       ost proteins mitochondrial matrix synthesized on <br />         cytosolic ribosomes imported <br /> across outer inner mitochondrial membranes. Translo- <br />                                                                              known J proteins (18, 19). However, regardless it <br />                                                                              acts J protein partner Ssc1, Tim44 regulates inter- <br /> cation inner membrane occurs inner                    action Ssc1 import channel. Tim44 tethers ATP- <br /> membrane channel driven membrane potential                 bound Ssc1 import channel. When polypeptide substrates <br /> an import motor (1&#8211;3). Three critical components motor               bind Ssc1[ATP], Ssc1 released Tim44 absence <br /> are major Hsp70 molecular chaperone mitochondria                      ATP hydrolysis, permitting binding another <br /> (mtHsp70; Ssc1 yeast); peripheral inner membrane pro-                 Ssc1[ATP] channel second cycle interaction (20). <br /> tein Tim44, tethers Ssc1 import channel;                   Because importance import proteins into <br /> nucleotide release factor Ssc1, Mge1. Multiple cycles Ssc1            mitochondria cell function, set test <span id='am-30' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-31'  ></span><span id='am-32' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>idea</span> that <br /> binding release translocating polypeptide, driving               Tim44, analogous Sec63, functions J protein partner of <br /> the import process, required import proteins             Ssc1 protein import. We Tim44 does have <br /> mitochondrial matrix (1&#8211;3).                                                  activities associated prototypical J protein. Rather, a <br />    Ssc1, like Hsp70s, contains C-terminal domain                previously unstudied essential gene, PAM18, encodes J protein <br /> binds short segments unfolded polypeptides rich hydro-                 inner membrane important protein transloca- <br /> phobic amino acids (4, 5). The N-terminal ATPase domain                      tion vivo J protein partner Ssc1. <br /> regulates binding interaction adenine nucleo- <br /> tides. In turn, binding peptide segment C-terminal             Materials Methods <br /> domain stimulates ATPase activity N terminus. In              Yeast Strains, <span id='am-105' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-106' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Plasmids</span>, Genetic Techniques. PAM18 ob- <br /> ATP-bound state, interaction peptide substrate unstable,             tained PCR-amplifying genomic DNA position &#11002;303 to <br /> with fast rates. In ADP-bound state,                 &#11001;702 using Pfu Turbo polymerase (Stratagene) cloned <br /> interaction relatively stable, slow rates.                pRS316 (21). To obtain null allele PAM18, coding <br />                                                                                                                                                        BIOCHEMISTRY <br />  <br />  <br />  <br /> Therefore, thought ATP-bound form Hsp70                    sequences &#11001;71 &#11001;542 replaced TRP1 gene. <br /> initiates interaction polypeptide substrate,              A heterozygous PAM18&#20862;pam18 diploid PJ53 (22) carrying <br /> stabilized hydrolysis ATP. Exchange ATP bound               PAM18 plasmid sporulated. &#9004;pam18 haploids carrying <br /> ADP results release peptide, completing cycle.            PAM18 pRS316 isolated dissecting tetrads YP <br />    Hsp70s rarely, ever, function independently. Rather,              medium (1% yeast extract&#20862;2% peptone). Mutants PAM18 <br /> function cochaperones. J proteins, named                 constructed site-directed mutagenesis using <br /> presence signature J domain interacts                   QuikChange protocol (Stratagene) cloned pRS315. <br /> ATPase domain, stimulate Hsp70&#8217;s ATPase activity, facil-                   Pam18 epitope-tagged C terminus tandem <br /> itating interaction substrate polypeptides (6, 7). <span id='am-35' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> activ- <br /> ity critical feature J protein function, amino acid <br /> alterations J domain decrease ATPase stimulation             Abbreviation: HA, hemagglutinin. <br /> functionally defective vivo (8). For example, lumen             &#8225;To   correspondence addressed. E-mail: ecraig@wisc.edu. <br /> the endoplasmic reticulum, ability membrane-                      &#169; 2003 The National Academy Sciences USA <br />  <br />  <br />  <br /> www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.1936150100                                             PNAS &#20841; November 25, 2003 &#20841; vol. 100 &#20841; no. 24 &#20841; 13839 &#8211;13844 <br />  copies hemagglutinin (HA) peptide introducing a <br /> NotI site termination <span id='am-40' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>codon</span> subcloning HA <br /> tag NotI fragment pGTEP1 (23). The tagged protein <br /> was functional; cells carrying tagged protein only <br /> PAM18 protein grew wild-type cells variety of <br /> conditions. For purposes purification, entire ORF of <br /> PAM18 cloned glutathione S-transferase expressing <br /> vector pGEX-KG (24), forming fusion protein Pam18 <br /> at C terminus. To modulate expression Pam18, entire <br /> ORF (&#11001;1 &#11001;507) cloned pCM189 (25), placing <br /> PAM18 control tetracycline-regulatable promoter, <br /> resulting repression transcription addition           Fig. 1. Effect Tim44 ATPase activity Ssc1. Ssc1:ATP complex (&#11015;0.2 <br /> tetracycline derivative doxycycline.                                    &#65533;?&#65533;M) incubated 23&#176;C presence various concentrations Mdj1 <br />    Complete synthetic media lacking specific amino acids             peptide P5. The rate conversion ADP measured plotted the <br /> containing 5-fluoroorotic acid prepared described (26).         fold stimulation basal rate determined absence additional <br /> All chemicals obtained Sigma unless stated.         components. (A) Mdj1, peptide P5, Tim44, indicated. (B) Reactions with <br />                                                                         1 &#65533;?&#65533;M Mdj1 (Left) 5 &#65533;?&#65533;M peptide P5 (Right) carried presence <br />                                                                         indicated concentrations Tim44. <br /> Fractionation Mitochondria. Mitochondria fractionated <br /> essentially described ref. 27. Briefly, 25 &#65533;?&#65533;g mitochondria <br /> were swollen 500 &#65533;?&#65533;l EM buffer (10 mM <span id='am-100' about='obo:NCBITaxon_258862' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_9436 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-101' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="Mops" datatype="xsd:string"></span><span id='am-104'  ></span>Mops</span> KOH, pH                 purified Ssc1 antibodies crosslinked protein A- <br /> 7.2&#20862;1 mM EDTA) incubated ice 15 min. Efficiency <br />                                                                         Sepharose beads. Six micrograms Ssc1 incubated with <br /> of disruption outer membrane assessed incubation <br />                                                                         Ssc1 antibody beads 4&#176;C 1 h. Under condition, &#11015; 90% <br /> with proteinase K 50 &#65533;?&#65533;g&#20862;ml ice 15 min. As control, <br />                                                                         Ssc1 typically immunoprecipitated. After washes, <br /> sodium deoxycholate added 1% disrupt mitochon- <br /> drial inner membrane. Mitochondria separated mem-             Tim44 (0.6 &#65533;?&#65533;g) added. The mixture incubated final <br /> brane soluble fractions described ref. 28. Mitochondrial      volume 150 &#65533;?&#65533;l 23&#176;C 30 min point 1.5 &#65533;?&#65533;l a <br /> fractions subjected immunoblot analysis using            1 mg&#20862;ml solution peptide P5 added reactions, and <br /> ECL (Amersham Pharmacia Biotech) according                incubation continued additional 30 min. Tim44:Ssc1 <br /> manufacturer&#8217;s suggestion, using polyclonal antibodies spe-          complex formation robust, forming 0&#176;C a <br /> cific Tim44, cytochrome b2, Mge1 monoclonal               variety salt buffer conditions (<span id='am-95' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-96' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-97'  ></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> shown). The final <br /> antibody specific HA.                                               concentrations Ssc1, Tim44, peptide P5 0.6, 0.09, <br />                                                                         10 &#65533;?&#65533;M, respectively. After centrifugation, beads were <br /> Protein Purification. Pam18 purified bacterial strain C41      washed times mitochondrial lysis buffer (250 mM <br /> (29). Expression induced 3 h 37&#176;C addition 1 mM        sucrose&#20862;80 mM KCl&#20862;20 mM Mops-KOH, pH 7.2&#20862;0.2% Triton <br /> isopropyl-&#65533;?&#65533;-D-thiogalactopyranoside. After affinity chromatog-          X-100&#20862;1 mM PMSF) containing appropriate nucleotide. <br /> raphy using glutathione agarose, biotinylated thrombin              Tim44 Ssc1 bound beads detected SDS&#20862; <br /> added mixture incubated 3 h 20&#176;C. The cleaved        PAGE immunoblot analysis using polyclonal antibodies <br /> Pam18 product recovered removal glutathione               specific Tim44 Ssc1. <br /> agarose beads centrifugation removal thrombin by <br /> streptavidin&#20862;agarose (catalog no. 69022-3, Novagen).                    Peptide Binding Assay. Briefly, fluorescence anisotropy assays for <br />    Tim44His (20), Mdj1His (30), Ssc1His (31), Ssq1His (31)     binding peptide carried described refs. 20 and <br /> purified described. All His-tagged proteins previously          32. Peptide P5 labeled fluorescein generate F-P5 as <br /> shown able rescue absence normal protein vivo.   described (33). Various concentrations Ssc1 incubated <br />                                                                         F-P5 (10 nM) 23&#176;C buffer A (25 mM Hepes-KOH, pH <br /> ATPase Assay. Isolation Hsp70:ATP complexes ATPase               7.4&#20862;100 mM KCl&#20862;11 mM Mg(OAC)2&#20862;10% glycerol). After <br /> assays carried essentially described refs. 20 32,    binding reached equilibrium, anisotropy measurements were <br /> with changes. To obtain radiolabeled ATP:Ssc1 complexes,           Beacon 2000 Fluorescence Polarization system <br /> 50 &#65533;?&#65533;g Hsp70 incubated 100 &#65533;?&#65533;Ci [&#65533;?&#65533;-32P]ATP (3,000          (Panvera, Madison, WI) 23&#176;C excitation 490 nm and <br /> Ci&#20862;mmol, DuPont) buffer A (25 mM Hepes-KOH, pH                       emission 535 nm. <br /> 7.4&#20862;100 mM KCl&#20862;11 mM Mg(OAC)2&#20862;10% glycerol) containing <br /> 25 &#65533;?&#65533;M ATP 100 &#65533;?&#65533;l ice 30 sec. The complex isolated         Results <br /> immediately NICK column (Amersham Pharmacia Bio- <br />                                                                         Tim44 Does Not Stimulate ATPase Activity Ssc1. To address the <br /> tech) frozen aliquots &#11002;70&#176;C. For ATPase assays, <br />                                                                         possibility Tim44 acts J protein cochaperone the <br /> aliquots thawed ice; time 0 time shift 23&#176;C, <br />                                                                         import channel, tested ability affect ATPase activity <br /> when additions peptide P5 (CALLLSAPRR), Tim44, Mdj1, <br /> and&#20862;or Pam18 samples. At indicated                Ssc1 using single-turnover conditions. A complex between <br /> times, 3-&#65533;?&#65533;l aliquots removed mixed 1 &#65533;?&#65533;l stop           Ssc1 radiolabeled ATP isolated, rate ADP <br /> solution containing 4 M formic acid, 2 M LiCl, 36 mM ATP.           formation determined incubation presence and <br /> The mixture spotted polyethyleneimine-cellulose              absence Tim44. The basal rate ATP hydrolysis Ssc1 was <br /> TLC plate developed 1 M formic acid 0.5 M LiCl.              0.07 min&#11002;1, consistent previous results (32). No effect on <br /> After drying, TLC plate exposed PhosphorImager             ATPase activity Ssc1 observed addition of <br /> screen (Molecular Dynamics), ATP ADP              Tim44 range concentrations, 35 &#65533;?&#65533;M, 140 <br /> was quantified PhosphorImager (Molecular Dynam-             molar excess Ssc1 (Fig. 1A). As control, increasing <br /> ics). The percentage hydrolysis 0 time (&#11015;20%) sub-            concentrations soluble mitochondrial J protein <br /> tracted time points.                                           Mdj1 peptide substrate added. Both stimulated the <br />                                                                         hydrolysis ATP. For example, 10-fold molar excess (2 &#65533;?&#65533;M) <br /> Coimmunoprecipitation Assay. The Ssc1 coimmunoprecipitation             Mdj1, activity stimulated 7-fold; 100-fold excess (20 <br /> assay carried essentially described ref. 20. Affinity-    &#65533;?&#65533;M), peptide activity stimulated 6.5-fold. <br />  <br /> 13840 &#20841; www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.1936150100                                                                                      D&#8217;Silva et al. <br />  Fig. 2. Functional defect Ssc1A503D. (A) ATPase activity. Ssc1:ATP Ssc1A503D:ATP complexes (&#11015;0.2 &#65533;?&#65533;M) incubated 23&#176;C presence Mdj1 <br /> or peptide P5. The basal rates hydrolysis mutant wild-type protein indistinguishable. (B) Peptide binding measured fluorescence anisotropy: <br /> Fluorescein-labeled F-P5 (10 nM) incubated presence indicated concentrations wild-type A503D mutant protein ADP. Anisotropy <br /> measurements taken 25&#176;C; raw polarization values fitted one-site binding (hyperbola) equation determine Kd. (C) Effect peptide on <br /> Ssc1:Tim44 interaction. Tim44 (0.09 &#65533;?&#65533;M) incubated Ssc1 (0.6 &#65533;?&#65533;M) presence ATP ADP, indicated, 30 min. In cases, 10 &#65533;?&#65533;M <br /> peptide P5 added, incubation continued additional 30 min. The mixtures immunoprecipitated using Ssc1-specific antibodies described <br /> in Materials Methods. The presence Tim44 Ssc1 precipitate assessed immunoblot analysis using antibodies specific respective <br /> proteins. <br />  <br />  <br />  <br /> Tim44 Interaction Ssc1 Is Distinct That J Protein. To              rate Ssc1 observed, vitro, presence high <br /> assess Tim44 interacting Ssc1 sites similar                concentration peptide substrate turnover minute. <br /> or overlapping contacting J protein, tested                   This rate hydrolysis clearly sufficient drive import in <br /> effect increasing concentrations Tim44 ability                    vivo, rate cycle interaction would <br /> Mdj1 stimulate ATPase activity Ssc1. No effect                       occur imported polypeptide. Assuming rate of <br /> stimulation seen Tim44 added concentrations                       ATP hydrolysis Ssc1 observe vitro assays is <br /> between 5 35 &#65533;?&#65533;M, Mdj1 held constant 1 &#65533;?&#65533;M                         reflective natural state protein, cofactor Ssc1 <br /> (Fig. 1B). Similar experiments carried presence                   stimulates ATP hydrolysis exist. <br /> of peptide substrates. Again, effect seen peptide                          Because Tim44 does stimulate ATPase activity Ssc1, <br /> stimulation ATPase activity addition Tim44. These                      searched yeast genome database (www.yeastgenome.org) <br /> results indicate Tim44 does act J protein partner                potential J protein partners Ssc1. The ORF YLR008C <br /> Ssc1, does interfere binding substrate peptide.                  (PAM18) essential gene that <br />    To probe differences Tim44 J protein                   encodes protein containing predicted internal membrane <br /> interactions, used mutant isolated search                   spanning domain (Fig. 3A) J domain 57% identical <br /> for temperature-sensitive point mutations SSC1. Ssc1A503D                      J protein mitochondrial inner membrane <br /> has single amino acid alteration peptide-binding domain.                 Mdj2 (16). We confirmed deletion PAM18 W303 <br /> The ATPase activity A503D mutant protein stimu-                        strain background resulted inviability, deletion MDJ2 <br /> lated Mdj1 (Fig. 2A). However, interactions Tim44                     little phenotypic effect (data shown). <br /> a peptide substrate normal. Ssc1A503D bound fluorescein-                        To determine J domain Pam18 important for <br /> tagged peptide affinity wild-type protein,                  function, constructed mutants causing alterations in <br /> a dissociation constant (Kd) 0.3 &#65533;?&#65533;M (20), measured                       J domain. Because hallmark J domain highly <br /> fluorescence anisotropy (32) (Fig. 2B). A peptide substrate                       conserved tripeptide, histidine, proline, aspartic acid <br /> stimulated ATPase activity Ssc1A503D nearly                     (HPD), constructed mutation results replace- <br /> wild-type protein (4.0- 4.2-fold, respectively) peptide                  ment HPD alanines (Pam18A3). In addition, we <br /> concentration 15 &#65533;?&#65533;M (Fig. 2 A). Tim44 bound                            constructed mutation results substitution glu- <br /> Ssc1[ATP] Ssc1A503D[ADP], and, wild-type pro-                         tamine place histidine (Pam18H141Q). To test whether <br /> tein, peptide able stimulate release Tim44                       mutants rescue inviability &#9004;pam18 strain, <br /> Ssc1A503D[ATP] complex (ref. 20 Fig. 2C). Thus, interac-                      plasmids carrying PAM18 mutants transformed a <br /> tion Tim44 peptide substrate, J protein                      &#9004;pam18 strain carrying wild-type copy PAM18 cen- <br />                                                                                                                                                                     BIOCHEMISTRY <br />  <br />  <br />  <br /> Mdj1, normal mutant Ssc1 protein. Because Tim44                      tromeric plasmid having URA3 gene. When resulting <br /> cannot compete binding Mdj1 peptide interacts                      strains plated media containing 5-fluoroorotic acid, which <br /> normally Ssc1 A503D, conclude Tim44 J                       selects cells having lost URA3-containing plasmid carrying <br /> protein does bind sites Ssc1 prevent                       wild-type PAM18 gene, viable cells recovered (Fig. 3B). <br /> interaction known J protein.                                               Thus, J domain essential PAM18 function. <br />                                                                                      For Pam18 J protein partner Ssc1 vivo, must <br /> Essential J Protein Mitochondrial Inner Membrane, Pam18.                   localized mitochondria J domain interact <br /> Multiple binding release cycles Ssc1 incoming                      Ssc1, matrix. To determine cellular <br /> polypeptide chain required import (1&#8211;3). It                  location Pam18, mitochondria cells expressing Pam18 <br /> estimated that, average, vivo, polypeptide chains pass                HA tag C terminus (Pam18-HA) isolated. <br /> through import channel minute (34). Thus, turnover                   These mitochondria contained protein reacted the <br /> rate ATP hydrolysis Ssc1 sufficiently fast allow                 HA-specific antibody present mitochondria <br /> numerous interaction cycles time frame. The turnover                  expressing Pam18 lacking HA tag (data shown). Mito- <br />  <br /> D&#8217;Silva et al.                                                                                      PNAS &#20841; November 25, 2003 &#20841; vol. 100 &#20841; no. 24 &#20841; 13841 <br />                                                                                     localize Pam18, mitochondria having intact inner <br />                                                                                    membrane, disrupted outer membrane, formed by <br />                                                                                    swelling mitochondria buffer low osmotic strength. The <br />                                                                                    swollen mitochondria treated protease, were <br />                                                                                    intact mitochondria (Fig. 3C Left). Like Tim44, intermem- <br />                                                                                    brane space protein cytochrome b2, matrix protein Mge1, <br />                                                                                    Pam18 substantially affected protease digestion of <br />                                                                                    intact mitochondria, suggesting Pam18 outer <br />                                                                                    membrane. However, treatment swollen mitochondria with <br />                                                                                    protease resulted production predominant cleavage <br />                                                                                    fragment containing HA tag. These results suggest that <br />                                                                                    Pam18 located inner membrane. In addition, because <br />                                                                                    HA tag C terminus resistant protease attack, <br />                                                                                    J domain C-terminal putative membrane- <br />                                                                                    spanning domain predicted matrix the <br />                                                                                    inner membrane, N terminus protruding the <br />                                                                                    intermembrane space. Such orientation make J <br />                                                                                    domain accessible Ssc1 matrix. <br />                                                                                       To assess Pam18 role mitochondrial protein <br />                                                                                    import vivo, PAM18 placed control the <br />                                                                                    tetracycline-repressible promoter. Drug added turn off <br />                                                                                    expression, accumulation precursor form nuclear- <br />                                                                                    encoded mitochondrial protein, Hsp60, monitored. As the <br />                                                                                    case nuclear-encoded matrix proteins, prese- <br />                                                                                    quence Hsp60 cleaved matrix processing protease <br />                                                                                    immediately traversing inner membrane (35); the <br />                                                                                    accumulation normally undetectable precursor form has <br />                                                                                    used previously vivo <span id='am-1' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-2' property="rdfs:label" content="Indicator" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5'  ></span>indicator</span> function the <br />                                                                                    import process (28, 36). Cells harvested 24 h drug <br />                                                                                    addition, growth tester strain begun slow <br />                                                                                    compared control strain. The precursor form Hsp60 <br />                                                                                    accumulated strain Pam18 expression was <br />                                                                                    repressed, control strain. This accumulation is <br />                                                                                    indicative role Pam18 import proteins the <br />                                                                                    mitochondrial matrix vivo. <br />  <br />                                                                                    Pam18 Specifically Stimulates Ssc1 ATPase Activity. The results <br />                                                                                    described previous section suggest Pam18 acts J <br />                                                                                    protein mitochondrial import. We purified PAM18 <br />                                                                                    wild-type mutant proteins tested ability stim- <br /> Fig. 3. In vivo function J protein Pam18. (A) Predicted amino acid          ulate ATPase activity Ssc1 using single-turnover assay <br /> sequence Pam18. A box drawn putative membrane-spanning            described above. Wild-type Pam18 stimulated ATPase ac- <br /> region. The region having sequence identity J domain underlined;       tivity 7-fold present 7-fold excess (Fig. 4A). Pam18 and <br /> highly conserved HPD indicated ***. (B) Requirement vivo J domain      Mdj1 similar concentration dependences <span id='am-69' about='obo:NCBITaxon_1113472' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_40096 obo:NCBITaxon_40092 obo:NCBITaxon_33208 obo:NCBITaxon_40093 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-70' property="rdfs:label" content="Stimula" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-73'  ></span>stimula</span>- <br /> of Pam18. Strains carrying plasmids expressing protein (&#11002;), wild-type           tion Ssc1 ATPase activity, suggesting proteins <br /> Pam18, mutant Pam18, Pam18AAA Pam18H141Q (Pam18HI/Q) spot-             interact similarly Ssc1. As case Mdj1, addition <br /> ted 1- fold dilution series media lacking (Left) containing (Right) <br />                                                                                    excess Tim44 did inhibit ability Pam18 stimulate <br /> 5-fluoroorotic acid. Plates incubated 3 days 30&#176;C. (C) Fractionation <br /> of mitochondria. Mitochondria purified cells expressing Pam18 <br />                                                                                    Ssc1 (Fig. 4B), indicating binding Tim44 Pam18 are <br /> having HA tag C terminus. Mitochondria separated               mutually exclusive. <br /> membrane soluble components (Right). Mitochondrial protein (T)                To test Pam18 stimulates Ssc1 ATPase activity a <br /> fractional equivalents membrane pellet (P) supernatant (S)             prototypical J protein, used J domain mutants described <br /> analyzed. Mitochondria (M) subjected swelling disrupt outer         previous section, Ssc1 mutant a <br /> membrane (SW) (Left). Equivalent amounts treated proteinase K            defective interaction known J protein Mdj1. Pam18 <br /> and cases treated detergent (sodium deoxycholate), indicated.      mutant proteins altered J domains unable to <br /> The fractions subjected immunoblot analysis using antibodies            function vivo failed stimulate Ssc1&#8217;s ATPase activity <br /> specific HA tag Pam18, antibody specific known <br />                                                                                    (Fig. 4A). In addition, tested ability wild-type Pam18 <br /> proteins intermembrane space (cytochrome b2), inner membrane <br /> (Tim44), matrix (Mge1). (D) &#9004;pam18 cells carrying Pam18 control of <br />                                                                                    stimulate ATPase activity A503D Ssc1 mutant (Fig. <br /> the tetracycline regulatable promoter grown overnight rich media,       4C). Like Mdj1 (Fig. 2D), Pam18 did stimulate mutant <br /> which point subcultured presence absence <span id='am-90' about='obo:NCBITaxon_1310384' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_83136 obo:NCBITaxon_1 obo:NCBITaxon_83139 obo:NCBITaxon_131567 obo:NCBITaxon_6854 obo:NCBITaxon_426515 obo:NCBITaxon_6843 obo:NCBITaxon_426501 obo:NCBITaxon_33154 obo:NCBITaxon_6933 obo:NCBITaxon_6656 obo:NCBITaxon_70332 obo:NCBITaxon_6946 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_70320 obo:NCBITaxon_6947 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-91'  ></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="Tetra" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>tetra</span>-         protein. Also, stimulation Pam18 specific, it <br /> cycline analogue doxycycline. After 24 h whole-cell lysates prepared      failed stimulate ATPase activity Hsp70 the <br /> separated SDS&#20862;PAGE, followed immunoblot analysis using antibodies         mitochondrial matrix Ssq1 (Fig. 4D). Thus, conclude that <br /> specific Hsp60.                                                                Pam18 J protein partner Ssc1. <br />  <br />                                                                                    Discussion <br /> chondria containing Pam18-HA separated membrane                          The data presented allow better understanding the <br /> and soluble fractions, subjected immunoblot                     mechanism translocation proteins mitochondrial <br /> analysis using HA-specific antibodies. Pam18-HA present                     inner membrane. First, Tim44 tethers Ssc1 import <br /> in membrane fraction, Tim44 (Fig. 3C Right). To                         channel, does function J protein cochaperone for <br />  <br /> 13842 &#20841; www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.1936150100                                                                                       D&#8217;Silva et al. <br />  Fig. 4. Stimulation Ssc1 ATPase activity Pam18. Hsp70:ATP complexes prepared, rate hydrolysis measured 23&#176;C presence <br /> of various concentrations Mdj1, wild-type Pam18, mutant Pam18, presence absence Tim44. The rate conversion ADP measured and <br /> is plotted fold stimulation basal rate. In panel, Hsp70 ATP hydrolysis measured indicated asterisk. (A) Ssc1:ATP <br /> (&#11015;0.2 &#65533;?&#65533;M) incubated presence various concentrations wild-type Pam18, Mdj1, Pam 18 mutant proteins Pam18AAA Pam18H141Q. (B) <br /> Reactions 1 &#65533;?&#65533;M Pam18 carried presence indicated concentrations Tim44. (C) Ssc1:ATP Ssc1A503D:ATP complexes (&#11015;0.2 &#65533;?&#65533;M) were <br /> incubated presence excess Pam18. (D) Ssq1:ATP complexes (0.1 &#65533;?&#65533;M) presence various concentrations Pam18. <br />  <br />  <br />  <br /> mitochondrial Hsp70 Ssc1, does bind site(s)                 induced interaction peptide substrate rapidly de- <br /> Hsp70 J protein. Second, identified J protein                      stabilizes Ssc1:Tim44 interaction (20). Likewise, binding of <br /> partner Ssc1 inner membrane stimulates Ssc1&#8217;s                     Mge1 Ssc1[ADP] destabilizes interaction Tim44 (20, 37). <br /> ATPase activity. These results lead model Ssc1,                     Thus, Tim44 evolved conformational-sensitive switch <br /> tethered import channel, stimulated J protein                  governing association Ssc1 import channel. <br /> in close proximity, ensuring rapid binding incoming                      The fact Tim44 J protein identification of <br /> polypeptide chain.                                                                new J protein partner Ssc1 allow forward revised <br />    Determining mechanism action Tim44 protein                        model protein import. In contrast endoplasmic retic- <br /> translocation important, central initial                  ulum, tethering import channel stimulation of <br /> interaction translocating protein import motor. The                 ATPase activity executed protein, J protein Sec63 <br /> inability Tim44 stimulate ATPase activity Ssc1,              (38), mitochondria separate critical functions two, or <br /> concentrations 20-fold higher required 5-fold                     more, proteins. Tim44 regulates Ssc1&#8217;s interaction with <br /> stimulation known J protein partner Ssc1, Mdj1, argues                  import channel, Pam18 stimulates Ssc1&#8217;s ATPase activ- <br /> strongly Tim44 functioning J cochaperone Ssc1. The                ity. Occupancy biologically relevant form Ssc1, <br /> original idea Tim44 function J-type cochaperone               Ssc1[ATP], predominates import channel (20). Interaction <br /> Ssc1 based limited sequence homology J domains                  incoming polypeptide stimulates dissociation Ssc1 <br /> of J proteins (17). It possible J-like region evolved            Tim44 and, combination Pam18, stimulates ATP <br /> from true J domain facilitate interaction Ssc1                   hydrolysis, stabilizing Ssc1:polypeptide substrate interaction. <br /> Tim44. In fact, 18-<span id='am-84' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-89'  ></span>aa</span> deletion completely removing region                 Interestingly, Ssc1&#8217;s release Tim44 substrate binding <br /> resulted synthesis stable mutant protein inserted                independent ATP hydrolysis. Because Pam18&#8217;s ability to <br /> correctly membrane defective interaction                   stimulate ATPase activity Ssc1 affected Tim44, <br /> Ssc1. These results suggested region Tim44 interacts                 determining relative rates release Tim44 ATP <br /> directly Ssc1 (17). However, scenario Tim44 having                   hydrolysis needed definitive model of <br /> evolved J-like domain interacts Ssc1 J domain                  import forth. However, noted that <br /> binding site unlikely competition functional                    mechanistically important ATP hydrolysis temporally <br /> interaction Tim44 true J proteins Mdj1 Pam18                  coupled release, failure hydrolyze ATP result <br /> were observed. Thus, interaction sites J proteins Tim44                dissociation polypeptide chain Ssc1 the <br /> are likely distinct; detailed structural information                 extreme instability Ssc1:substrate interaction Ssc1 is <br /> required clearly delineate exact sites interactions               bound ATP. Pam18 surely plays critical role stimulating <br /> these proteins.                                                                   hydrolysis rate. It play role coordinating the <br />    Our results indicate Pam18 prime candidate                 timing hydrolysis, allowing efficient coupling release from <br /> J protein partner Ssc1 mitochondrial import. As case                Tim44 ATP hydrolysis. <br />                                                                                      Determination exact mechanistic details protein import <br /> for components import machinery, Tim44, <br />                                                                                   coordination action Ssc1 cofactors awaits further <br /> Ssc1, Mge1, Pam18 essential. It located inner <br />                                                                                   analysis. However, precedent additional proteins <br />                                                                                                                                                                    BIOCHEMISTRY <br />  <br />  <br />  <br /> membrane, J domain extending matrix, well <br />                                                                                   mediating interactions Hsp70s&#20862;J proteins&#20862;substrate <br /> positioned interact Ssc1 tethered import <br />                                                                                   polypeptides. In <span id='am-45' about='obo:NCBITaxon_274' typeof='owl:Thing obo:NCBITaxon_188786 obo:NCBITaxon_188787 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_68933 obo:NCBITaxon_270 obo:NCBITaxon_1297'><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Thermua flavus" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Thermus aquaticus (SUBSP. FLAVUS)" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Thermus aquaticus (SUBSP. THERMOPHILUS)" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasAlternativeId" content="NCBITaxon:273" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Thermus themophilus" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="PMID:4334342" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Thermus aquaticus subsp. thermophilus" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasRelatedSynonym" content="Thermus aquaticus flavus" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Thermus flavus" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="PMID:8590676" datatype="xsd:string"></span><span id='am-58'  ></span><span id='am-59' property="rdfs:label" content="Thermus thermophilus" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="PMID:11594628" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Thermus aquaticus thermophilus" datatype="xsd:string"></span>Thermus thermophilus</span> stable complex between <br /> channel. As expected bona fide J protein partner, can <br />                                                                                   Hsp70 J protein mediated protein, DafA (39). <br /> stimulate ATPase activity Hsp70 Ssc1, it <br />                                                                                   Upon binding substrate polypeptide, DafA displaced from <br /> functions, does stimulate ATPase activity of <br />                                                                                   complex. Interactions adapter proteins Tim44 and <br /> another matrix Hsp70, Ssq1. This stimulation affected by <br />                                                                                   DafA common presently appreciated serve <br /> the presence Tim44. Thus, Pam18 able stimulate                     important roles regulating interactions Hsp70s, J proteins, <br /> the ATPase activity Ssc1 tethered import                    substrate polypeptides. <br /> channel Tim44. <br />    However, Tim44 considered simply membrane                      We thank Jarek Marszalek helpful discussions thoughtful <br /> anchor Ssc1. The interaction proteins                   comments manuscript. This work supported National <br /> to evolved exquisitely sensitive conformation                   Institutes Health Grants GM278709 (to E.A.C.) 5T326M08349 <br /> of Ssc1. For example, conformational change Ssc1[ATP]                        (to A.A.). <br />  <br />  <br /> D&#8217;Silva et al.                                                                                     PNAS &#20841; November 25, 2003 &#20841; vol. 100 &#20841; no. 24 &#20841; 13843 <br />   1.   Neupert, W. &amp; Brunner, M. (2002) Nat. Rev. Mol. Cell Biol. 8, 555&#8211;565.               22.   James, P., Pfund, C. &amp; Craig, E. (1997) Science 275, 387&#8211;389. <br />  2.   Pfanner, N. &amp; Geissler, A. (2001) Nat. Rev. Mol. Cell Biol. 5, 339&#8211;349.              23.   Roof, D. M., Meluh, P. B. &amp; Rose, M. D. (1992) J. Cell Biol. 118, 95&#8211;108. <br />  3.   Matouschek, A., Pfanner, N. &amp; Voos, W. (2000) EMBO Rep. 5, 404&#8211;410.                  24.   Guan, K. L. &amp; Dixon, J. E. (1991) Anal. Biochem. 192, 262&#8211;267. <br />  4.   Bukau, B. &amp; Horwich, A. L. (1998) Cell 92, 351&#8211;366.                                  25.   <span id='am-79' about='obo:NCBITaxon_457750' typeof='owl:Thing obo:NCBITaxon_98297 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6580 obo:NCBITaxon_6544 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_6599 obo:NCBITaxon_61354 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-80'  ></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="Gari" datatype="xsd:string"></span>Gari</span>, E., Piedrafita, L., Aldea, M. &amp; Herrero, E. (1997) Yeast 13, 837&#8211;848. <br />  5.   Hartl, F. &amp; Hayer-Hartl, M. (2002) Science 295, 1852&#8211;1858.                           26.   Sikorski, R. S. &amp; Boeke, J. D. (1991) Metholds Enzymology: Guide Yeast <br />  6.   Cheetham, M. E. &amp; Caplan, A. J. (1998) Cell Stress Chaperones 3, 28&#8211;36.                    Genetics Molecular Biology, eds. Guthrie, C. &amp; Fink, G. R., Vol. 194, pp. <br />  7.   Kelley, W. (1999) Curr. Biol. 9, R305&#8211;R308.                                                302&#8211;318. <br />  8.   Genevaux, P., Schwager, F., Georgopoulos, C. &amp; Kelley, W. L. (2002) Genetics         27.   Ryan, M. T., Voos, W. &amp; Pfanner, N. (2001) Methods Cell Biol. 65, 189&#8211;215. <br />       162, 1045&#8211;1053.                                                                      28.   Kang, P. J., Ostermann, J., Shilling, J., Neupert, W., Craig, E. A. &amp; Pfanner, <br />  9.   Brodsky, J. L. &amp; Schekman, R. (1993) J. Cell Biol. 123, 1355&#8211;1363.                         N. (1990) Nature 348, 137&#8211;143. <br /> 10.   Matlack, K. E. S., Plath, K., Misselwitz, B. &amp; Rapoport, T. A. (1997) Science        29.   Miroux, B. &amp; Walker, J. E. (1996) J. Mol. Biol. 260, 289&#8211;298. <br />       277, 938&#8211;941.                                                                        30.   Horst, M., Oppliger, W., Rospert, S., Schonfeld, H.-J., Schatz, G. &amp; Azem, A. <br /> 11.   Corsi, A. K. &amp; Schekman, R. (1997) J. Cell Biol. 137, 1483&#8211;1493.                           (1997) EMBO J. 16, 1842&#8211;1849. <br /> 12.   Hermann, J., Stuart, R., Craig, E. &amp; Neupert, W. (1994) J. Cell Biol. 127,           31.   Dutkiewicz, R., Schilke, B., Knieszner, H., Walter, W., Craig, E. A. &amp; <br />       893&#8211;902.                                                                                   Marszalek, J. (2003) J. Biol. Chem. 278, 29719&#8211;29727. <br /> 13.   Rowley, N., Prip-Buus, C., Westermann, B., Brown, C., Schwarz, E., Barrell, B.       32.   Liu, Q., Krzewska, J., Liberek, K. &amp; Craig, E. A. (2001) J. Biol. Chem. 276, <br />       &amp; Neupert, W. (1994) Cell 77, 249&#8211;259.                                                     6112&#8211;6118. <br /> 14.   Lutz, T., Westermann, B., Neupert, W. &amp; Herrmann, J. (2001) J. Mol. Biol. 307,       33.   Montgomery, D., Morimoto, R. &amp; Gierasch, L. (1999) J. Mol. Biol. 286, <br />       815&#8211;825.                                                                                   915&#8211;932. <br /> 15.   Voisine, C., Cheng, Y. C., Ohlson, M., Schilke, B., Hoff, K., Beinert, H.,           34.   Lim, J. H., Martin, F., Guiard, B., Pfanner, N. &amp; Voos, W. (2001) EMBO J. 20, <br />       Marszalek, J. &amp; Craig, E. (2001) Proc. Natl. Acad. Sci. USA 98, 1483&#8211;1488.                 941&#8211;950. <br /> 16.   Westermann, B. &amp; Neupert, W. (1997) J. Mol. Biol. 272, 477&#8211;483.                      35.   Cheng, M., Hartl, F., Martin, J., Pollock, R., Kalousek, F., Neupert, W., <br /> 17.   Merlin, A., Voos, W., Maarse, A., Meijer, M., Pfanner, N. &amp; Rassow, J. (1999)              Hallberg, E., Hallberg, R. &amp; Horwich, A. (1989) Nature 337, 620&#8211;625. <br />       J. Cell Biol. 145, 961&#8211;972.                                                          36.   Gambill, B. D., Voos, W., Kang, P. J., Miao, B., Langer, T., Craig, E. A. &amp; <br /> 18.   Krimmer, T., Rassow, J., Kunau, W. H., Voos, W. &amp; Pfanner, N. (2000) Mol.                  Pfanner, N. (1993) J. Cell Biol. 123, 109&#8211;117. <br />       Cell. Biol. 20, 5879&#8211;5887.                                                           37.   Schneider, H. C., Westermann, B., Neupert, W. &amp; Brunner, M. (1996) EMBO <br /> 19.   Strub, A., Rottgers, K. &amp; Voos, W. (2002) EMBO J. 21, 2626&#8211;2635.                           J. 15, 5796&#8211;5803. <br /> 20.   Liu, Q., D&#8217;Silva, P., Walter, W., Marszalek, J. &amp; Craig, E. A. (2003) Science 300,   38.   Rapoport, T., Matlack, K., Plath, K., Misselwitz, B. &amp; Staeck, O. (1999) Biol. <br />       139&#8211;141.                                                                                   Chem. 380, 1143&#8211;1150. <br /> 21.   Sikorski, R. S. &amp; Hieter, P. (1989) Genetics 122, 19&#8211;27.                             39.   Klostermeier, D., Seidel, R. &amp; Reinstein, J. (1999) J. Mol. Biol. 287, 511&#8211;525. <br />  <br />  <br />  <br />  <br /> 13844 &#20841; www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.1936150100                                                             <br /> </body></html>