<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Microbiology                                                                                                                         BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> Dxr essential <span id='am-1' about='obo:NCBITaxon_1763' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1762 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasDbXref" content="PMID:1883713" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="PMID:7547304" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Mycobacterium" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="PMID:7547284" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="PMID:8863452" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="PMID:1380284" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium Lehmann and Neumann 1896" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="PMID:2275850" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="PMID:16014496" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="PMID:7907223" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="PMID:1581193" datatype="xsd:string"></span>Mycobacterium</span> tuberculosis fosmidomycin <br /> resistance lack uptake <br /> Amanda C Brown Tanya Parish* <br />  <br /> Address: Institute Cell Molecular Science, Barts London School Medicine Dentistry, London, UK <br /> Email: Amanda C Brown - a.c.brown@qmul.ac.uk; Tanya Parish* - t.parish@qmul.ac.uk <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 20 May 2008                                                        Received: 10 September 2007 <br />                                                                               Accepted: 20 May 2008 <br /> BMC Microbiology 2008, 8:78   doi:10.1186/1471-2180-8-78 <br /> <span id='am-61' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-62' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-63'  ></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> article available from: http://www.biomedcentral.com/1471-2180/8/78 <br /> &#169; 2008 Brown Parish; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  Abstract <br />                  Fosmidomycin phosphonic antibiotic inhibits 1-deoxy-D-xylulose 5-phosphate <br />                  reductoisomerase (Dxr), committed step non-mevalonate pathway isoprenoid <br />                  biosynthesis. In <span id='am-43' about='obo:NCBITaxon_1773' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1762 obo:NCBITaxon_1763 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_77643'><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Bacterium tuberculosis" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Bacillus tuberculosis" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium tuberculosis typus humanus" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium tuberculosis var. hominis" datatype="xsd:string"></span><span id='am-49'  ></span><span id='am-50' property="rdfs:label" content="Mycobacterium tuberculosis" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Mycobacterium tuberculosis</span> Dxr encoded Rv2870c, antibiotic <br />                  shown inhibit recombinant enzyme [1], mycobacteria intrinsically resistant to <br />                  fosmidomycin cell level. Fosmidomycin hydrophilic molecule bacteria <br />                  uptake active process involving cAMP dependent glycerol-3-phosphate transporter <br />                  (GlpT). The fact glpT homologue M. tuberculosis genome highly <br />                  impervious nature hydrophobic mycobacterial cell wall suggests resistance due <br />                  lack cellular penetration. <br />                  Results: We demonstrated dxr (Rv2780c) essential gene M. tuberculosis, could <br />                  delete chromosomal copy unless second functional copy provided integrating <br />                  vector. This confirmed intracellular target fosmidomycin essential as <br />                  sensitive. We looked uptake fosmidomycin mycobacterial <span id='am-71' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-72' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>, slow- <br />                  growing pathogenic M. tuberculosis fast-growing, saprophytic <span id='am-35' about='obo:NCBITaxon_1772' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1762 obo:NCBITaxon_1763 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003'><span id='am-36'  ></span><span id='am-37' property="rdfs:label" content="Mycobacterium smegmatis" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Bacterium smegmatis" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium paratuberculosis smegmatis" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Bacillus smegmatis" datatype="xsd:string"></span>Mycobacterium smegmatis</span>; both <br />                  species resistant fosmidomycin high level. Fosmidomycin accumulated intra- <br />                  cellularly M. tuberculosis M. smegmatis remained extra-cellular medium. In contrast, <br />                  fosmidomycin uptake confirmed sensitive organism, <span id='am-92' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-93' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-106'  ></span><span id='am-107' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span>Escherichia coli</span>. We established that <br />                  lack intra-cellular accumulation efflux, efflux pump inhibitors no <br />                  effect fosmidomycin resistance. Finally, demonstrated fosmidomycin modified <br />                  mycobacterial cells extracts remained fully functional state. <br />                  Conclusion: Taken together, <span id='am-150' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-151' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-152' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-154'  ></span>data</span> demonstrate fosmidomycin resistance M. <br />                  tuberculosis M. smegmatis results lack penetration antibiotic site the <br />                  sensitive target. <br />  <br />  <br />  <br />  <br /> Background                                                                    terium tuberculosis, (the main causative agent human <br /> The mycobacteria contain number important patho-                         tuberculosis), resulting million deaths per <br /> gens infect animals humans. The World                          year [2]. Currently available vaccine live, <br /> Health Organisation (WHO) estimated mil-                       attenuated <span id='am-16' about='obo:NCBITaxon_1765' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1762 obo:NCBITaxon_1763 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_77643'><span id='am-17' property="rdfs:label" content="Mycobacterium bovis" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="PMID:14657105" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium tuberculosis var. bovis" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="PMID:11931153" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="PMID:10618079" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium tuberculosis typus bovinus" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-25'  ></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="NCBITaxon:115863" datatype="xsd:string"></span>Mycobacterium bovis</span> Calmette-Guerin (BCG) <br /> lion humans annum newly infected Mycobac-                        strain; however, BCG does offer complete immunity, <br />  <br />                                                                                                                                          Page 1 9 <br />                                                                                                                  (page number citation purposes) <br />  BMC Microbiology 2008, 8:78                                                   http://www.biomedcentral.com/1471-2180/8/78 <br />  <br />  <br />  <br /> and protection highly variable wide range         Fosmidomycin (sodium hydrogen 3-(N-hydroxyfoma- <br /> social, economic environmental factors. Opportunis-          mido)propylphosphonate) phosphonate antibiotic <br /> tic mycobacterial pathogens identified caus-        inhibits enzymes DOXP pathway <br /> ing disseminated disease HIV-infected            &#8211; DOXP reductoisomerase (Dxr) [10]. However, anti- <br /> immuno-compromised individuals. In addition                    microbial activity shown limited Gram- <br /> number diseases, including leprosy          negative species, fosmidomycin effective <br /> Buruli ulcer, caused mycobacteria. Many          Gram-positive cocci anaerobic species [11,12]. <br /> infections increase and, effective ther-     In <span id='am-108' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-109' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-110'  ></span><span id='am-111' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span>Escherichia</span> coli, fosmidomycin transported the <br /> apy exists diseases, increase multi-     cell glycerol-3-phosphate transporter, GlpT, and <br /> drug resistance strains jeopardises ability treat         glpT deletion mutants fosmidomycin resistant [13]. A <br /> them. Current research trends focused produc-         fosmidomycin resistance gene (fsr) identi- <br /> tion improved vaccine, identification new drug          fied bacteria including E. coli <span id='am-85' about='obo:NCBITaxon_234' typeof='owl:Thing obo:NCBITaxon_118882 obo:NCBITaxon_356 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_28211'><span id='am-86'  ></span><span id='am-87' property="oboInOwl:hasDbXref" content="PMID:8573514" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasRelatedSynonym" content="Brucella Meyer and Shaw 1920" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Brucella" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Brucella</span> spp. <br /> targets, development new anti-mycobacterials.         [14,15]. Fsr member <span id='am-58' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> permeases, suggest- <br /> All activities benefited greatly avail-   ing resistance results rapid efflux drug <br /> ability complete genome sequence M. tuberculosis       cell. Fosmidomycin currently considered <br /> [3].                                                             treatment malaria [16-18]. <br />  <br /> The need new antibiotics effective myco-         Mycobacteria, including M. tuberculosis, highly resist- <br /> bacteria greater. In addition search     ant fosmidomycin cell level, although <br /> new antibiotics, renewed exam-        recombinant M. tuberculosis Dxr sensitive fosmid- <br /> ining existing compounds efficacy anti-mycobacte-         omycin, mechanism resistance unknown [1]. <br /> rial agents. Mycobacteria relatively antibiotic resistant    In study, characterized basis intrinsic <br /> and susceptible commonly used antibi-            resistance M. tuberculosis M. smegmatis fosmid- <br /> otic groups, penicillins. This intrinsic resist-     omycin. We demonstrated dxr gene essential in <br /> ance attributed largely nature            M. tuberculosis, confirming organism sensi- <br /> mycobacterial cell wall, rich long-chain fatty       tive vulnerable target. We investigated uptake fos- <br /> acids including C60 C90 mycolic acids,          midomycin mycobacteria E. coli using bioassay <br /> covalently linked arabinogalactan-peptidogylcan           fosmidomycin. Efflux pump inhibitors used to <br /> layer. Porins, (water-rich channel proteins allow          assess role efflux mediating fosmidomycin resist- <br /> hydrophilic molecules enter cell diffusion),      ance. Inactivation fosmidomycin mycobacterial cells <br /> rare mycobacteria [4] function         cell-free extracts tested. <br /> at considerably reduced rate comparison porins in <br /> Gram-negative bacteria Mycobacterium smegmatis [5-7].         Results <br /> Therefore antibiotic resistance physical     Essentiality Dxr <br /> properties cell wall forming impermeable barrier       We interested determining basis mycobac- <br /> [8], drug inactivation assumed       terial resistance fosmidomycin. The target fosmid- <br /> that bacteria susceptible intracellular targets,     omycin enzyme Dxr (DOXP reductoisomerase, <br /> that drugs modified allow cellular entry     Rv2870c), committed step non-mevalonate <br /> would effective.                                          pathway isoprenoid biosynthesis. M. tuberculosis Dxr <br />                                                                  expressed recombinant protein read- <br /> Isopentenyl disphosphate (IPP) common precursor          ily inhibited fosmidomycin [1], indicating resist- <br /> the biosynthesis isoprenoid compounds. This               ance lack susceptible target. One <br /> includes polyprenyl phosphate, involved          possibility account lack cell sensitivity <br /> synthesis covalently linked peptidoglycan-arabi-          Dxr required mycobacterial growth. The <br /> nogalactan-mycolic acid complex, lipomannan                  non-mevalonate pathway known pathway of <br /> lipoarabinomannan [9]. Isoprenoids synthesized            isoprenoid biosynthesis mycobacteria [19], seems <br /> by pathways; mevalonate pathway (which                unlikely case. However, work using saturat- <br /> present humans), non-mevalonate 1-deoxy-           ing transposon mutagenesis [20] gave rise predic- <br /> D-xylulose 5-phosphate (DOXP) pathway,                 tion dxr essential gene. <br /> been bacteria parasites. Genome <br /> sequencing M. tuberculosis shown non-            In <span id='am-74' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-75' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> address question, used two-step <br /> mevalonate pathway sole pathway present [3]. Since        homologous recombination method attempt gener- <br /> this pathway absent humans represents              ate defined mutant dxr M. tuberculosis [21]. A dele- <br /> attractive target drug development.                          tion delivery vector, pORTAL3- defined deletion of <br />                                                                  dxr, introduced M. tuberculosis single cross <br />  <br />  <br />                                                                                                                        Page 2 9 <br />                                                                                                (page number citation purposes) <br />  BMC Microbiology 2008, 8:78                                                      http://www.biomedcentral.com/1471-2180/8/78 <br />  <br />  <br />  <br /> overs (SCOs) obtained. Double cross-overs (DCOs)               Resistance fosmidomycin efflux <br /> were generated SCOs screened PCR               We confirmed mycobacteria highly resistant to <br /> to determine wild type deletion allele            fosmidomycin resistance culture conditions. We <br /> present. Of 40 DCOs screened               looked growth M. smegmatis M. tuberculosis <br /> wild type, suggesting essentiality dxr.                          liquid medium presence fosmidomycin. No <br />                                                                     inhibition growth seen concentration to <br /> Since unable isolate deletion mutant            1 mg/ml (Figure 2A &amp; B), confirming species are <br /> wild type background, constructed merodiploid                  resistant high level. Since dxr essential normal <br /> strain carrying entire dxr operon native pro-          culture conditions Dxr sensitive fosmidomy- <br /> moter mycobacteriophage L5 based integrating vec-              cin inhibition [1], explanation for <br /> <span id='am-140' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-141' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-142' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-143'  ></span><span id='am-144' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>tor</span> (pOPW: dxrintgm). DCOs generated                   high level resistance. <br /> background screened PCR. Of 24 <br /> DCOs screened 12 deletion allele (50%), indicat-            Resistance fosmidomycin attributed Fsr <br /> ing able isolate chromosomal deletion            efflux pump micro-organisms [14,15]. Myco- <br /> dxr background. The expected genotype                  bacteria known posses efflux pumps which <br /> number strains, termed &quot;del-                 confer intrinsic antibiotic resistance [27-32]. We investi- <br /> inquents&quot; i.e. deleted copy chromosome              gated efflux plays role fosmidomycin resist- <br /> integrated, functional gene confirmed Southern               ance mycobacteria, utilising efflux pump <br /> analysis (Figure 1B). This data confirms dxr               inhibitors, reserpine, verapamil carbonyl cyanide m- <br /> operon essential (since complemented                 chlorophenylhydrazone (CCCP). The concentrations of <br /> whole operon).                                                      efflux pump inhibitors used taken previ- <br />                                                                     ous study M. smegmatis [27]. As before, reserpine and <br /> Since dxr operon (Figure 1A), possible      verapamil 12 40 &#956;g/ml respectively inhib- <br /> inability isolate mutants             itory growth. However, CCCP concentration 15 <br /> downstream genes disrupted mutant. Analy-              &#956;g/ml seen significantly inhibit growth M. <br /> sis mutation created pORTAL3            smegmatis M. tuberculosis. Therefore, tested lower <br /> delivery vector suggests disrupt promoter           concentrations CCCP; 1.25 &#956;g/ml highest per- <br /> region start operon gene down-           mitted concentration growth rate notably <br /> stream dxr (Rv2869c), non-essential intramembrane-             affected, inhibition seen. <br /> cleaving protease (iCLIP) [22,23]. <br />                                                                     Growth mycobacteria presence efflux pump <br /> In order confirm dxr essential, used          inhibitors fosmidomycin assayed. All cultures <br /> gene switching strategy. This relies fact efficient     grew OD 1.0 (approx 109 cell ml), indi- <br /> replacement resident integrated vector easily           cating significant effects growth rate <br /> achieved M. tuberculosis transformation selec-            final OD reached. No difference sensitivity of <br /> tion second integrating plasmid carrying different          species fosmidomycin seen presence <br /> antibiotic resistance marker [24-26]. We use            inhibitors, indicating efflux mecha- <br /> delinquent strain carrying complete operon (pOPW)               nism resistance. <br /> (dxr&#916;, dxrint gm) tested ability plasmid carry- <br /> ing operon dxr (pOPD; Figure 1A)                Transport fosmidomycin cells <br /> complement chromosomal mutation. The delinquent                 Since discounted efflux resistance mecha- <br /> strain transformed pOPD plated                    nism, determined fosmidomycin trans- <br /> hygromycin plates select incoming vector. No             ported mycobacterial cells place. <br /> hygromycin resistant transformants obtained,                   Uptake antibiotics typically measured using radi- <br /> although control vector transformed cells          olabelled derivative. However, radio-labelled fosmidomy- <br /> efficiency 1.4 &#215; 106per &#956;g. In contrast replacement           cin available commercially additional <br /> pOPW pOPD wild-type background easily                   associated hazards connected using radio-isotopes <br /> achieved efficiency 1.2 &#215; 106 &#956;g. These data            Category Three level containment UK. A <br /> indicate vector lacking dxr does comple-            microbiological bioassay, based inhibition E. <br /> ment chromosomal mutation, dxr               coli active compounds, previously used to <br /> itself essential. It possible genes   measure fosmidomycin concentrations [11,33]. In this <br /> operon essential strategy         assay, active fosmidomycin measured inhibition of <br /> future extended examining essentiality           E. coli growth disks. A standard curve using known <br /> genes operon.                                               concentrations fosmidomycin generated and <br />  <br />  <br />  <br />                                                                                                                          Page 3 9 <br />                                                                                                  (page number citation purposes) <br />  BMC Microbiology 2008, 8:78                                                         http://www.biomedcentral.com/1471-2180/8/78 <br />  <br />  <br />  <br />  <br />                           (A) <br />                                Region deleted pORTAL3 <br />  <br />  <br />  <br />                                           dxr             Rv2869c      gcpE           Rv2867c <br />  <br />  <br />                                 Region deleted pOPD <br />  <br />  <br />                                                  Region present pOPW <br />  <br />  <br />                                                   Region present pOPD <br />  <br />                              (B) <br />                NotI                                                          NotI                wild-type <br />                                          probe <br />  <br />                                                          dxr          Rv2869c       gcpE          Rv2867c <br />                                         6.4 kb <br />  <br />                NotI                              NotI          NotI                              deletion <br />                                          probe <br />  <br />                                                          Rv2869c      gcpE           Rv2867c <br />                                4.0 kb <br />  <br />  <br />  <br />  <br />                                                            1 2           3 <br />  <br />                                                                                       10 kb <br />                                    wt 6.4kb <br />                                                                                       6 kb <br />                                 del 4.0 kb                                            4 kb <br />                                                                                       3 kb <br />                                  int 2.6 kb                                           2.5 kb <br />  <br />  <br />  <br />  <br /> Figure 1 <br /> Demonstration   essentiality dxr <br /> Demonstration essentiality dxr. (A) The chromosomal location dxr operon, region deleted the <br /> delivery vector (pORTAL3), regions present complementing <span id='am-120' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-121' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>vectors</span> pOPW (complete operon) pOPD <br /> (operon dxr) shown. (B) The deletion delivery vector pORTAL3 introduced chromosome homol- <br /> ogous recombination followed site-specific recombination complementing vector pOPW generate merodip- <br /> loid strain. DCOs generated background analysed. Southern analysis representative DCOs isolated the <br /> merodiploid background shown. Genomic DNA digested NotI hybridised indicated probe. The sizes for <br /> the wild-type deletion alleles shown diagrammatically &#8211; replacement wild-type gene results additional NotI <br /> site chromosome. Lane 1 &#8211; wild-type DCO (dxrwt, dxrint), Lane 2 &#8211; Deletion DCO (dxr&#916;, dxrint), Lane 3 &#8211; markers. The <br /> bands wild-type, deletion integrated copies dxr indicated. <br />  <br />  <br />  <br />  <br />                                                                                                                           Page 4 9 <br />                                                                                                   (page number citation purposes) <br />  BMC Microbiology 2008, 8:78                                                    http://www.biomedcentral.com/1471-2180/8/78 <br />  <br />  <br />  <br />                                                                   The limit detection uptake assay 0.1 &#956;g/ <br />                                                                   ml, 10,000-fold level antibiotic in <br />                                                                   medium. One limitations using high con- <br />                                                                   centration fosmidomycin E. coli the <br />                                                                   antibiotic supernatant released <br />                                                                   cells antibiotic-induced lysis. However, this <br />                                                                   lead underestimate uptake, over- <br />                                                                   estimate it. <br />  <br />                                                                   Detoxification/modification <br />                                                                   The results transport assay strongly suggested <br />                                                                   fosmidomycin resistance mycobacteria results <br />                                                                   lack active transport. However, possi- <br />                                                                   ble tell antibiotic modified pre- <br />                                                                   vious assay. This high concentration of <br />                                                                   antibiotic used; small proportion fos- <br />                                                                   midomycin modified, been <br />                                                                   detected loss fosmidomycin activity superna- <br />                                                                   tant. Thus, possibility cells modify- <br />                                                                   ing antibiotic non-toxic form existed. To <br />                                                                   address question repeated exposure assay <br />                                                                   using lower concentration fosmidomycin which <br />                                                                   allowed precise quantification active antibi- <br />                                                                   otic supernatant. M. smegmatis E. coli cells were <br />                                                                   incubated 20 &#956;g/ml fosmidomycin 60 min and <br />                                                                   active fosmidomycin assayed supernatant (Figure <br />                                                                   3B). In supernatant exposed M. smegmatis, 2-fold <br />                                                                   reduction activity (from 20 10 &#956;g/ml) seen. <br /> Figure <br /> Efflux <br /> ance  2 <br />        pump  inhibitors effect mycobacterial resist- <br />          fosmidomycin                                             Given limitation assay, difference <br /> Efflux pump inhibitors effect mycobacte-               result change zone inhibition less <br /> rial resistance fosmidomycin. Mycobacteria                1 mm, likely to lack sensitivity in <br /> grown presence 100 &#956;g/ml fosmidomycin (Fos), 12         assay. In addition, detoxification occurring we <br /> &#956;g/ml reserpine (RES), 40 &#956;g/ml verapamil (VER) 1.25 &#956;g/      expect complete reduction fosmidomy- <br /> ml carbonyl cyanide m-chlorophenylhydrazone (CCCP)                cin activity. In contrast 100-fold reduction fosmid- <br /> where indicated. (A) M. tuberculosis grown 28 d       omycin activity seen E. coli supernatants (due <br /> (B) M. smegmatis 16 h. Error bars represent standard          uptake extra-cellular environment the <br /> deviation triplicate samples. <br />                                                                   cell). Thus, modification antibiotic M. smeg- <br />                                                                   matis seen. <br />  <br />                                                                   Although fosmidomycin taken myco- <br /> used quantify fosmidomycin [see Additional file 1]             bacterial cells, intra-cellular enzymes <br /> [34].                                                             capable detoxification. In order address this, we <br />                                                                   determined extracts M. smegmatis capable <br /> M. smegmatis, M. tuberculosis E. coli cells incu-        inactivating fosmidomycin. M. smegmatis cell-free <br /> bated 1 mg/ml fosmidomycin 30 min (Figure             extracts incubated fosmidomycin (20 &#956;g/ml <br /> 3A). Active fosmidomycin supernatant (extra-cellu-         200 &#956;g/ml) 10 60 minutes. No decrease fos- <br /> lar) cell-free extracts (intra-cellular) measured.        midomycin activity seen time, indicating <br /> No active fosmidomycin present cell-free               detoxifying enzymes present (Figure 4). These <br /> extracts generated M. tuberculosis M. smegmatis;      data confirmed resistance modification <br /> of fosmidomycin remained supernatant. In con-          fosmidomycin. <br /> trast, E. coli extracts contained 150 &#956;g/ml and <br /> a reduction fosmidomycin super-           Discussion <br /> natant seen. These data confirmed fosmidomycin                Using two-step recombination process, demon- <br /> uptake E. coli, conversely lack         strated Rv2870c (dxr) essential growth M. <br /> uptake mycobacteria.                                           tuberculosis vitro. This contradictory previous <br />  <br />  <br />                                                                                                                         Page 5 9 <br />                                                                                                 (page number citation purposes) <br />  BMC Microbiology 2008, 8:78                                                    http://www.biomedcentral.com/1471-2180/8/78 <br />  <br />  <br />  <br />  <br /> Figureof3fosmidomycin uptake <br /> Assay <br /> Assay fosmidomycin uptake. Uptake fosmidomycin measured mycobacteria E. coli. Cells subjected to <br /> (A) 1 mg/ml fosmidomycin 30 min; (B) 20 &#956;g/ml fosmidomycin 60 min. Cell-free extracts (CFE) culture superna- <br /> tants (SN) prepared serial dilutions applied paper disks described methods. For assay, stand- <br /> ard curve generated using known concentrations fosmidomycin. For test samples, zones inhibition E. coli were <br /> measured antibiotic concentration calculated standard curve. <br />  <br />  <br /> reports TraSH analysis, predicted dxr         data dxr suggests prediction non-essen- <br /> non-essential gene M. tuberculosis [20]. In TraSH         tial incorrect, ratio obtained close <br /> system predictions gene essentiality based ratio     cut-off point. Our data demonstrate is <br /> generated microarray analysis transposon               case, able construct a <br /> library, arbitrary cut-off point used distinguish     chromosomal deletion dxr merodiploid back- <br /> essential non-essential genes. A closer inspection        ground. However, able demonstrate <br />  <br />  <br />                                                                                                                        Page 6 9 <br />                                                                                                (page number citation purposes) <br />  BMC Microbiology 2008, 8:78                                                     http://www.biomedcentral.com/1471-2180/8/78 <br />  <br />  <br />  <br />                                                                    10% v/v OADC (7H10/OADC). M. smegmatis mc2155 was <br />                                                                    grown Lemco media [39]. X-Gal (5-bromo-4-chloro-3- <br />                                                                    indolyl-&#946;-D-galactopyranoside) used 50 &#956;g/ml; <br />                                                                    IPTG (isopropyl-<span id='am-52' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-56'  ></span><span id='am-57' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span>beta</span>-D-thiogalactopyranoside) 0.5 <br />                                                                    mM; kanamycin 20 &#956;g/ml; hygromycin B 100 &#956;g/ml; <br />                                                                    sucrose 2% w/v. Fosmidomycin (sodium salt) was <br />                                                                    purchased Invitrogen. Efflux pump inhibitors: reser- <br />                                                                    pine used 12 &#956;g/ml; verapamil 40 &#956;g/ml; and <br />                                                                    carbonyl cyanide m-chlorophenylhydrazone (CCCP) at <br />                                                                    1.25 &#956;g/ml. M. tuberculosis growth curves conducted <br />                                                                    4 ml 7H9/Tw/OADC media 16 mm glass tubes <br />                                                                    containing 8 mm magnetic stirrer bar, stirring 150 <br />                                                                    rpm. <br />  <br />                                                                    Attempt construct deletion mutants dxr <br /> Figureof4fosmidomycin detoxification cell free extracts <br /> Assay <br />                                                                    A deletion delivery vector dxr (pORTAL3) con- <br /> Assay fosmidomycin detoxification cell free <br /> extracts. Cell free extracts (CFE) tested inactiva-       structed follows (Figure 1): upstream flanking <br /> tion fosmidomycin. Fosmidomycin, final assay concen-       region amplified M. tuberculosis genomic DNA <br /> tration 20 200 &#956;g/ml, incubated sterile 10 mM       using primer pair DXR US F (PstI) 5&apos;-TGGGCTGCAG- <br /> Tris pH8 (control), M. smegmatis cell-free extract 10       CAACCCGCTAAGAAC-3, DXR US Rev (NotI) 5&apos;- <br /> or 60 min. A 100 &#956;l aliquot test sample              CCCGCGGCCGCTTGATGCTAAGATGCCATGC-3&apos; and <br /> applied sterile paper disc aseptically placed media    downstream flanking region amplified using the <br /> containing E. coli. The diameter zone growth inhibi-     primer pair DXR DS F (NotI) 5&apos;-CCCGCGGCCGCTGTTT- <br /> tion measured. Error bars standard deviation        GTTACCGGCATTGTG-3&apos; DXR DS Rev (HindIII) 5&apos;- <br /> triplicate samples.                                                CCCAAGCTTGGGCCAAGAAGAACCAGAAC-3&apos;.                      Frag- <br />                                                                    ments cloned p2NIL (PstI-HindIII) using the <br />                                                                    underlined restriction sites. The 6.3 kb PacI cassette from <br /> that dxr essential growth vitro,        pGOAL19 (hyg, sacB, lacZ) cloned sole <br /> truly validated drug target needs proved essen-      PacI site. The vector verified restriction digest and <br /> tial vivo. Whilst extremely challenging [35],       sequencing. 5 &#956;g UV pre-treated plasmid DNA [40] was <br /> use inducible systems generating conditional                electroporated competent M. tuberculosis SCO <br /> strains, recently described, allow           transformants selected 7H10/OADC medium <br /> carried [36-38].                                               containing kanamycin hygromycin. DCOs <span id='am-145' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-146'  ></span><span id='am-147' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-148' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>iso</span>- <br />                                                                    lated streaking cells plates lacking antibiotics and <br /> Conclusion                                                         selected/screened media containing sucrose X- <br /> Our data demonstrate fosmidomycin taken             gal previously described [21]. PCR screening DCOs <br /> by mycobacteria, resistance results intra-cel-        carried using primers DXR OL F (5&apos;-TTGGACGA- <br /> lular target inaccessible drug. Since,        TAGATCGACACC-3&apos;) DXR OL Rev (5&apos;-GATGGCATA- <br /> validated dxr essential gene, possible   GATCAGCACCA-3&apos;). <br /> could prove useful drug target inhibitors which <br /> enter cell developed. Alternatively fosmidomycin           A complementing vector carrying entire dxr operon <br /> could used conjunction compound                 native promoter constructed amplifying a <br /> increases permeability. Other approaches involve             4.7 kb region genomic H37Rv DNA primers <br /> nano-particulation fosmidomycin conversion            DXR OP C&apos; F 5&apos;-TTAATTAACACCTCGCGGTA- <br /> fosmidomycin pro-drug, readily               CAGCTCGTC-3&apos; DXR OP C&apos; Rev 5&apos;-TTAAT- <br /> taken cell hydrolysed active            TAAGTCGTCAGCGCTGATATGCCC-3&apos;. The product was <br /> form inside cell structure.                               cloned pSC-A (Stratagene); dxr_op/pSCA, and <br />                                                                    Gm-int cassette pUC-Gm-Int [41] intro- <br /> Methods                                                            duced HindIII fragment pOPW (Figure 1). <br /> Culture mycobacteria <br /> M. tuberculosis H37Rv cultured Middlebrook 7H9              Construction merodiploid strain confirmation of <br /> liquid medium supplemented 10% v/v OADC (oleic                essentiality <br /> acid, bovine serum albumin, D-glucose, catalase; Becton            We constructed merodiploid strain electroporating <br /> Dickinson) 0.05 % w/v Tween 80 (7H9/Tw/OADC)                   dxr SCO pOPW isolating kanamycin/hygro- <br /> or solid Middlebrook 7H10 agar supplemented                mycin/gentamicin resistant transformants. DCOs were <br />  <br />  <br />                                                                                                                         Page 7 9 <br />                                                                                                 (page number citation purposes) <br />  BMC Microbiology 2008, 8:78                                                          http://www.biomedcentral.com/1471-2180/8/78 <br />  <br />  <br />  <br /> isolated PCR-screened before. Strains con-           ducted experimental work, participated design <br /> firmed Southern analysis using AlkPhos Direct sys-        drafted manuscript. All authors read and <br /> tem (GE Healthcare) <span id='am-139' about='rdfs:label' typeof='owl:Thing'>label</span> detect DNA. A deletion          approved final manuscript. <br /> derivative dxr_op/pSCA constructed the <br /> 1.2 kb dxr gene removed site directed mutagenesis         Additional material <br /> with primers DXR KO SDM F 5&apos;-GTGACCAACTCGAC- <br /> CGACGTATCTGGTATGGCTTCG-3&apos; DXR KO SDM <br /> Rev      5&apos;-CGAAGCCATACCAGATACGTCGGTCGAGTT-                           Additional file 1 <br />                                                                       Schematic representation bioassay used quantify fosmidomy- <br /> GGTCAC-3&apos;. The Hyg-Int cassette pUC-Hyg-Int [41]                 cin. Cells exposed fosmidomycin; cell-free extracts culture fil- <br /> was cloned HindIII digest. The resulting plasmid,            trates prepared applied sterile paper discs. The discs were <br /> pOPD (Figure 1), used gene switching experi-                 aseptically placed media containing E. coli, test organism. The <br /> ment [25]. Strains electroporated integrating               diameter zone growth inhibition measured following incuba- <br /> vectors transformants isolated appropriate                 tion. Using known amounts fosmidomycin standard curve growth <br />                                                                       inhibition constructed used quantify fosmidomycin the <br /> antibiotic selection (for incoming vector). Transform- <br />                                                                       samples. <br /> ants patch tested antibiotic resistance required.         Click file <br />                                                                       [http://www.biomedcentral.com/content/supplementary/1471- <br /> Fosmidomycin uptake assays                                            2180-8-78-S1.tiff] <br /> Bacteria grown late log phase (OD600 = 1.0, <br /> approximately 109 cells ml), pelleted centrifugation <br /> at 3 000 &#215; g 5 minutes re-suspended 50 mM Tris <br /> buffer pH 7.0 approximately 1011 cells ml. One ml         Acknowledgements <br /> of suspension added 0.5 ml fosmidomycin solu-             This work funded EU Project QLRT-2001-00887 BBSRC <br /> tion 3.5 ml LB media E. coli M. smegmatis         Project BB/D000181. We like thank Karen Adie technical <br />                                                                  assistance, Paul Carroll manuscript suggestions Dean Crick val- <br /> Middlebrook 7H9/Tw/OADC medium M. tuberculosis <br />                                                                  uable discussions. <br /> and incubated 37&#176;C 30 60 min shaking at <br /> 100 rpm (E. coli, M. smegmatis) standing (M. tuberculo- <br />                                                                  References <br /> sis). Cells pelleted centrifugation 3 000 &#215; g     1.      Dhiman RK, Schaeffer ML, Bailey AM, Testa CA, Scherman H, Crick <br /> 5 minutes, supernatant collected, filter sterilized              DC: 1-Deoxy-D-Xylulose 5-Phosphate Reductoisomerase <br /> through 0.2 &#956;m syringe filter stored ice. Cells                 (IspC) Mycobacterium tuberculosis : under- <br />                                                                          standing mycobacterial resistance fosmidomycin. J Bacte- <br /> were washed twice 5 ml 50 mM Tris pH 7.0, re-<span id='am-66' about='obo:NCBITaxon_9822' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9821 obo:NCBITaxon_35497 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-67' property="rdfs:label" content="Sus" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-69'  ></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>sus</span>-                riol 2005, 187(24):8395-8402. <br /> pended 1 ml 10 mM Tris pH 7.0, transferred Fast       2.      Dye C, Scheele S, Dolin P, Pathania V, Raviglione MC: Global bur- <br />                                                                          den tuberculosis. Estimated incidence, prevalence, and <br /> Prep Lysing Matrix B tube (QBiogene) incubated                    mortality country. Journal American Medical Association 1999, <br /> ice 5 min. Cell-free extracts prepared using                282:677-686. <br /> Fast Prep speed 4 30 seconds (QBiogene). Samples          3.      Cole ST, Brosch R, Parkhill J, Garnier T, Churcher C, Harris D, Gor- <br />                                                                          don SV, Eiglmeier K, Gas S, Barry CE, et al.: Deciphering biol- <br /> were incubated ice 5 min, cell debris pelleted,               ogy Mycobacterium tuberculosis complete <br /> cleared lysate recovered, filter sterilized stored ice.           genome sequence. Nature 1998, 393(6685):537-544. <br />                                                                  4.      <span id='am-115' about='obo:NCBITaxon_426861' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158332 obo:NCBITaxon_158426 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_158406 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-116' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="Trias" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-119'  ></span>Trias</span> J, Jarlier V, Benz R: Porins cell wall mycobacteria. <br />                                                                          Science 1992, 258(5087):1479-1481. <br /> Fosmidomycin quantitation                                        5.      Engelhardt H, Heinz C, Niederweis M: A tetrameric porin limits <br /> Fosmidomycin measured using microbiological                        cell wall permeability Mycobacterium smegmatis. J Biol <br />                                                                          Chem 2002, 277(40):37567-37572. <br /> assay [34]. 100 ml cell-free extracts supernatants         6.      Mukhopadhyay S, Basu D, Chakrabarti P: Characterization a <br /> (and serial dilutions) applied sterile 14 mm filter            porin Mycobacterium smegmatis.                  J Bacteriol 1997, <br /> paper disc (Whatman). The discs allowed dry                   179(19):6205-6207. <br />                                                                  7.      Stephan J, Bender J, Wolschendorf F, Hoffmann C, Roth E, Mailander <br /> room temperature 15 minutes transferral                  C, Engelhardt H, Niederweis M: The growth rate Mycobacte- <br /> individual 90 mm agar plates containing 0.5% v/v E.                rium smegmatis depends sufficient porin-mediated influx <br />                                                                          nutrients. Molecular Microbiology 2005, 58(3):714-730. <br /> coli overnight culture. Plates incubated 20 hours       8.      Nguyen L, Thompson CJ: Foundations antibiotic resistance in <br /> at 37&#176;C diameter zone bacterial growth                 bacterial physiology: mycobacterial paradigm. Trends in <br /> inhibition measured. Discs known concentra-                     Microbiology 2006, 14(7):304-312. <br />                                                                  9.      Crick DC, Schulbach MC, Zink EE, Macchia M, Barontini S, Besra GS, <br /> tions fosmidomycin used make standard                       Brennan PJ: Polyprenyl phosphate biosynthesis Mycobacte- <br /> curve, calculate concentration fosmidomycin                 rium tuberculosis Mycobacterium smegmatis. J Bacteriol <br /> the test solutions. The concentration fosmidomycin                 2000, 182(20):5771-5778. <br />                                                                  10.     Kuzuyama T, Shimizu T, Takahashi S, Seto H: Fosmidomycin, a <br /> the samples calculated standard curve.                 specific inhibitor 1-deoxy-D-xylulose 5-phosphate reduc- <br />                                                                          toisomerase nonmevalonate pathway terpenoid <br />                                                                          biosynthesis. Tetrahedron Letters 1998, 39(43):7913-7916. <br /> Authors&apos; contributions                                           11.     Neu HC, Kamimura T: In vitro vivo antibacterial activity <br /> TP conceived study, participated design                FR-3 phosphonic acid antimicrobial agent. Antimicrob <br /> and coordination drafted manuscript. ACB con-                    Agents Chemother 1564, 19(6):1013-1023. <br />  <br />  <br />  <br />                                                                                                                                    Page 8 9 <br />                                                                                                           (page number citation purposes) <br />  BMC Microbiology 2008, 8:78                                                                        http://www.biomedcentral.com/1471-2180/8/78 <br />  <br />  <br />  <br /> 12.   Neu HC, Kamimura T: Synergy fosmidomycin (FR-31564)                          <span id='am-27' about='obo:NCBITaxon_1766' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1762 obo:NCBITaxon_1763 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003'><span id='am-28' property="oboInOwl:hasDbXref" content="PMID:10028244" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium minetti" datatype="xsd:string"></span><span id='am-30'  ></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium giae" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Mycobacterium fortuitum" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Mycobacterium fortuitum</span>. J Antimicrob Chemother 2006, <br />       antimicrobial agents. Antimicrob Agents Chemother                     57(2):252-259. <br />       1982, 22(4):560-563.                                                      33.   Cheoymang A, Hudchinton D, Kioy D, Na-Bangchang K: Bioassay <br /> 13.   Sakamoto Y, Furukawa S, Ogihara H, Yamasaki M: Fosmidomycin                     determination fosmidomycin plasma urine: <br />       resistance adenylate cyclase deficient (cya) mutants                      Application pharmacokinetic dose optimisation. Journal of <br />       Escherichia coli. Biosci Biotechnol Biochem 2003, 67(9):2030-2033.              Microbiological Methods 2007, 69(1):65-69. <br /> 14.   Fujisaki S, Ohnuma S, Horiuchi T, Takahashi I, Tsukui S, Nishimura Y,     34.   Kojo H, Shigi Y, Nishida M: FR-31564 new phosphonic acid <br />       Nishino T, Kitabatake M, Inokuchi H: Cloning gene                     antibiotic: bacterial resistance membrane permeability. <br />       Escherichia coli confers resistance fosmidomycin                   J Antibiot (Tokyo) 1980, 33(1):44-48. <br />       consequence amplification. Gene 1996, 175(1&#8211;2):83-87.                  35.   Thomson CJ, Power E, Ruebsamen-Waigmann H, Labischinski H: <br /> 15.   Halling SM, Peterson-Burch BD, Bricker BJ, Zuerner RL, Qing Z, Li L-            Antibacterial research development 21st Century <br />       L, Kapur V, Alt DP, Olsen SC: Completion genome                          &#8211; industry perspective challenges. Current Opinion in <br />       sequence <span id='am-77' about='obo:NCBITaxon_235' typeof='owl:Thing obo:NCBITaxon_118882 obo:NCBITaxon_356 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_28211 obo:NCBITaxon_234'><span id='am-78' property="rdfs:label" content="Brucella abortus" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Bacterium abortus" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Brucella melitensis biovar Abortus" datatype="xsd:string"></span><span id='am-82'  ></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasExactSynonym" content="Brucella melitensis bv. Abortus" datatype="xsd:string"></span>Brucella abortus</span> comparison highly                       Microbiology 2004, 7(5):445-450. <br />       similar genomes <span id='am-131' about='obo:NCBITaxon_29459' typeof='owl:Thing obo:NCBITaxon_118882 obo:NCBITaxon_356 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_28211 obo:NCBITaxon_234'><span id='am-132' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasDbXref" content="PMID:8573514" datatype="xsd:string"></span><span id='am-134'  ></span><span id='am-135' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasRelatedSynonym" content="Streptococcus Miletensis" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasRelatedSynonym" content="Micrococcus melitensis" datatype="xsd:string"></span><span id='am-138' property="rdfs:label" content="Brucella melitensis" datatype="xsd:string"></span>Brucella melitensis</span> <span id='am-124' about='obo:NCBITaxon_29461' typeof='owl:Thing obo:NCBITaxon_118882 obo:NCBITaxon_356 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_28211 obo:NCBITaxon_234'><span id='am-125' property="rdfs:label" content="Brucella suis" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasExactSynonym" content="Brucella melitensis bv. Suis" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-128'  ></span><span id='am-129' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="Brucella melitensis biovar Suis" datatype="xsd:string"></span>Brucella suis</span>. J Bac-          36.   Blokpoel MCJ, Murphy HN, O&apos;Toole R, Wiles S, Runn ESC, Stewart <br />       teriol 2005, 187(8):2715-2726.                             <br /> </body></html>