<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>INFECTION AND IMMUNITY, July 2010, p. 3118&#8211;3128                                                                                    Vol. 78, No. 7 <br /> 0019-9567/10/$12.00 doi:10.1128/IAI.00146-10 <br /> Copyright &#169; 2010, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />         Directed Screen <span id='am-87' about='obo:NCBITaxon_262' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_34064 obo:NCBITaxon_1236 obo:NCBITaxon_72273'><span id='am-88' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="Francisella" datatype="xsd:string"></span><span id='am-90'  ></span><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="PMID:8123561" datatype="xsd:string"></span>Francisella</span> novicida Virulence Determinants <br />                        Using <span id='am-60' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-61' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-64'  ></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span>Drosophila melanogaster</span>&#17188;&#8224; <br />                    Monika K. &#197;hlund,1&#8225; Patrik Ryde&#180;n,2 Anders Sjo <br />                                                                 &#168;stedt,1 Svenja Sto <br />                                                                                       &#168;ven1* <br />       Department Clinical Microbiology, Ume&#229; University, SE, 90185 Ume&#229;, Sweden,1 Department Mathematics and <br />                       Mathematical Statistics, Department Statistics, Computational Life Science Cluster, <br />                                            Ume&#229; University, SE, 90185 Ume&#229;, Sweden2 <br />                            Received 11 February 2010/Returned modification 2 April 2010/Accepted 5 May 2010 <br />  <br />              <span id='am-93' about='obo:NCBITaxon_263' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_34064 obo:NCBITaxon_1236 obo:NCBITaxon_72273 obo:NCBITaxon_262'><span id='am-94' property="oboInOwl:hasRelatedSynonym" content="Pasteurella tularensis" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-96'  ></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="Bacterium tularense" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasRelatedSynonym" content="Francisella tularense" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="Francisella tularensis" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="Brucella tularensis" datatype="xsd:string"></span>Francisella tularensis</span> highly virulent, facultative intracellular human pathogen virulence mecha- <br />           nisms understood. Occasional outbreaks tularemia potential use F. tularensis a <br />           bioterrorist agent warrant better knowledge pathogenicity bacterium. Thus far, genome-wide <br />           vivo screens virulence factors performed mice, restricted necessity to <br />           apply competition-based, negative-selection assays. We wanted individually evaluate putative virulence <br />           determinants suggested assays performed directed screening 249 F. novicida transposon <br />           insertion mutants using survival infected fruit flies measure bacterial virulence. Some 20% the <br />           genes tested required normal virulence flies; previously investigated in <br />           vitro vivo. We characterized involvement bacterial proliferation pathogenicity <br />           flies mouse macrophages. Hierarchical cluster analysis mutant phenotypes indicated functional <br />           linkage clustered genes. One cluster grouped genes Francisella pathogenicity island <br />           loci required intracellular survival. We identified genes involved adaptation oxidative <br />           stress genes induce host energy wasting. Several genes related type IV pilus formation <br />           demonstrated hypervirulent mutant phenotypes. Collectively, <span id='am-175' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-176' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-177' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-179'  ></span>data</span> demonstrate bacteria part <br />           use similar virulence mechanisms mammals Drosophila <span id='am-170' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-171'  ></span><span id='am-172' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-173' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-174' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span>melanogaster</span> considerable propor- <br />           tion virulence factors active mammals dispensable pathogenicity insect model. <br />  <br />  <br />    Francisella tularensis causative agent tularemia,                   The tularemia treatment date relies antibiotics. <br /> zoonotic disease affecting wide variety small vertebrates               To enable development vaccines new antimicrobial <br /> well humans (49). The severity clinical manifesta-                   drugs, vital understand molecular mechanisms be- <br /> tions disease highly dependent infecting strain               hind interaction pathogen humans. F. tularensis <br /> and route entry. If inhaled, 10 bacteria                   escapes host cell phagosome propagates the <br /> cause infection humans, untreated, mortality rate                 cytosol (16). Multiplication results cell death release of <br /> can reach 60% (45). To date, <span id='am-162' about='obo:NCBITaxon_subspecies' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-163' property="rdfs:label" content="subspecies" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>subspecies</span> cause disease                  bacteria (25), allowing spread regional lymph nodes <br /> in humans, F. tularensis subspecies holarctica F. tularensis                colonize spleen, liver, lung (52). A substantial <br /> subspecies tularensis, identified. A closely related                  proportion bacterial burden persist extracellularly in <br /> <span id='am-72' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>, F. novicida, environmental pathogen appears                  bloodstream (14, 59). <br /> not affect healthy humans, F. novicida infections                    Despite knowledge vivo life cycle, genome se- <br /> been reported exclusively immunocompromised in-                       quence data, techniques mutant generation, still <br /> dividuals (8, 19, 58), recognized relevant                   know little specific virulence determinants F. tularen- <br /> when Francisella virulence various mouse infection models                 sis. Factors known play role involved in <br /> investigated. Genome comparisons revealed high sequence                         lipopolysaccharide biosynthesis intracellular survival. A fo- <br /> similarities F. novicida isolate U112 clin-                 cus attention genomic region called Fran- <br /> ically important F. tularensis subspecies tularensis strain SCHU                cisella pathogenicity island (FPI), required escape <br /> S4. The represents evolutionarily oldest                    phagosome proliferation inside cytosol and <br /> complete Francisella genome, supporting good metabolic com-                     encodes putative type VI secretion (reviewed in <br /> petence, human-pathogenic strains, adaptation                    references 2, 13, 35). <br /> intracellular niche, lost genes genetic drift                     Lately, libraries transposon insertion mutants different <br /> are fastidious (26, 43).                                              Francisella reference strains used screen virulence <br />                                                                                 factors various mammalian vitro vivo infection <br />                                                                                 models (23, 31, 38, 51, 53, 57). In vivo screens mice applied <br />    * Corresponding author. Mailing address: Department Clinical              competition-based, negative-selection strategy identify <br /> Microbiology, Ume&#229; University, SE, 90185 Ume&#229;, Sweden. Phone: 46                bacterial mutants survive and/or colonize tar- <br /> 90 7851143. Fax: 46 90 7852225. E-mail: svenja.stoven@climi.umu.se.             organ (23, 51, 57). Results studies suggested a <br />    &#8225; Present address: Department Medical Chemistry Biophys-              large number genes involved Francisella pathoge- <br /> ics, Ume&#229; University, SE, 90185 Ume&#229;, Sweden. <br />    &#8224; Supplemental material article http://iai <br />                                                                                 nicity. While strategy sensitive efficient way to <br /> .asm.org/.                                                                      screen bacterial genome, probably overestimates <br />    &#17188; <br />      Published ahead print 17 May 2010.                                   number virulence genes importance as <br />  <br />                                                                          3118 <br />  VOL. 78, 2010                                                                          FLY SCREEN OF FRANCISELLA VIRULENCE FACTORS                                    3119 <br />  <br />  <br /> providing limited information specific roles indi-                           examined independent replicates. The median life length (MLL) <br /> vidual genes pathogenesis.                                                             replicate calculated linear extrapolation values <br />                                                                                           surrounding 50% survival. In cases 50% survival reached within <br />    Thus, far, mouse preferred model host                             10-day observation period, data censored MLL set to <br /> in vivo studies Francisella. Recently, however, analysis                            10 days. Only sets experiments MLL values two <br /> various human pathogens model organisms like Caenorhab-                                wild-type controls differed 1 day accepted. <br /> ditis elegans Drosophila melanogaster demonstrated                                   Intracellular growth assay. The mouse macrophage-like cell line J774A.1 was <br />                                                                                           cultured standard Dulbecco&#8217;s modified Eagle&#8217;s medium (DMEM) containing <br /> that bacteria large extent rely virulence <br />                                                                                           10% fetal bovine serum kept 37&#176;C 5% CO2. Cells infected as <br /> strategies invertebrates humans (24). Because                           described previously (17), using 1.5 &#11003; 105 cells 24-well plates an <br /> simplicity genetic tools available, nonmammalian mod-                             approximate multiplicity infection 100. The release lactate dehydroge- <br /> els offer unique opportunity unravel basis host-                              nase <span id='am-35' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-36'  ></span><span id='am-37' property="rdfs:label" content="Indicator" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>indicator</span> cytotoxicity measured using commercial kit (Pro- <br /> pathogen interactions great detail. Our previous work,                              mega). All analyses twice, duplicates second round. <br />                                                                                              Data analysis: fly survival. For mutant replicate, determined <br /> which introduced D. melanogaster model host Fran-                             difference MLL compared corresponding U112 wild-type <br /> cisella infections, suggested fruit fly valuable                        control calculated median replicates (m&#9004;MLL). To identify <br /> for identification characterization virulence deter-                           mutant m&#9004;MLL values significantly different wild type, we <br /> minants (56). In addition, blood-feeding arthropods, like ticks,                          obtained cutoff criteria using simulation approach based the <br />                                                                                           wild-type MLL values experiments. The difference two <br /> mosquitoes, biting flies, long acknowledged as <br />                                                                                           controls, &#9004;MLLwt, calculated experiments, assumed these <br /> <span id='am-146' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-147' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-148' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> tularemia (34, 37), implying bacterium                            observations normally distributed. The distribution parameters esti- <br /> evolved strategies persistence replication or-                            mated sample mean standard deviation &#9004;MLLwt data. The <br /> ganisms. Drosophila melanogaster represent natural                            estimated normal distribution used simulate 10,000 pseudo-m&#9004;MLLwt <br /> host Francisella tularensis, highly relevant model                        observations, replicates observation. The empirical distribution of <br />                                                                                           simulated data estimates null distribution mutants <br /> for immune mechanisms basic physiology arthropods. <br />                                                                                           behave wild type. The 0.0025 0.9925 quantiles simulated data <br />    Here individually analyzed nearly 250 genes                                selected cutoff values (&#11006;1.5 days) corresponded significance <br /> previously suggested candidate virulence determinants                             level 0.5% (P &#11005; 0.005). Some mutants did replicates, by <br /> by negative-selection screens mice <span id='am-75' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-76' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> confirm                           considering numbers replicates simulation procedure, obtained <br /> importance pathogenicity robust infection model. By                              P values. All mutants outside &#11006;1.5-day interval significant P <br />                                                                                           value 0.005, mutant inside &#11006;1.5-day interval significant <span id='am-67' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-68'  ></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> <br /> using survival infected flies measure bacterial viru-                          level. A mutant strain called attenuated median &#9004;MLL &#11022;1.5 days <br /> lence, identified 49 genes required normal                                hypervirulent &#65533;?1.5 days. <br /> virulence flies. These genes investigated                                Bacterial growth vitro. Based values linear range growth <br /> their role bacterial proliferation flies mouse mac-                          curve (0.5 &#65533;? OD600 &#65533;? 1.8), growth rate, &#9004;OD600/h, calculated each <br />                                                                                           mutant replicate. The mean growth rate U112 wild type was <br /> rophage-like cells. All mutant phenotypes analyzed by <br />                                                                                           0.510 &#11006; 0.057 &#9004;OD600/h (n &#11005; 24). The ratio mutant growth rate and <br /> hierarchical cluster analysis provide new insights func-                          corresponding wild-type control defined vitro prolifer- <br /> tional relationships corresponding genes. Our col-                              ation index. <br /> lected data comparison bacterial mutants model                                 Spearman&#8217;s rank correlation (RS), conservative significance level P &#65533;? <br /> systems allowed evaluation D. melanogaster                            0.001, used determine results different assays were <br />                                                                                           statistically significantly correlated other. <br /> screening model.                                                                             Gene information gathered http://www.francisella.org/. <br />  <br />  <br />                          MATERIALS AND METHODS <br />                                                                                                                             RESULTS <br />    Bacterial strains, fly strains, culture media. Infection experiments D. <br /> melanogaster Oregon R performed using F. novicida U112 wild-type <br /> control U112-derived transposon insertion mutants two-allele li- <br />                                                                                              Our group recently demonstrated LVS kills D. melano- <br /> brary provided Colin Manoil&#8217;s group (15). Mutants grown initially                 gaster dose-dependent manner 50% flies <br /> directly 96-well plates containing library later single-            infected 3 &#11003; 105 bacteria succumb infection within <br /> colony stocks. Other strains used live vaccine strain (LVS) F. tula-          5 6 days (56). We showed mutants single FPI <br /> rensis subspecies holarctica. Bacteria cultured modified GC agar plates <br />                                                                                           genes virulent flies. <br /> (Difco GC medium base [Becton Dickinson] complemented hemoglobin <br /> and <span id='am-165' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-166' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-167' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-169'  ></span>Iso</span>-Vitalex) supplemented TSB medium (Trypticase soy broth, 0.1%                   For analysis putative virulence factors, chose F. <br /> L-cysteine HCl, 0.2% dextrose) 37&#176;C 5% CO2. For transposon mutants,                novicida transposon insertion mutants well-described <br /> media contained 15 &#65533;?&#65533;g/ml kanamycin; viable count analysis, agar plates           two-allele library (15). This library contains distinct strains <br /> also supplemented 50 &#65533;?&#65533;g/ml polymyxin B. For vitro proliferation test,         1,490 loci transposon insertions were <br /> bacteria inoculated optical density 600 nm (OD600) 0.05 a <br /> standard overnight culture incubated 29&#176;C 200 rpm. Data points for <br />                                                                                           verified provides internal controls, two <br /> growth curves collected OD600 reached 2. Viable count analysis             alleles gene result similar phenotypes. We <br /> from infected flies carried described previously (56), sample             established experimental conditions infections F. <br /> preparation days 1, 3, 5 postinfection attenuated mutants days          novicida. Advantageous screen, U112-infected flies <br /> 1, 2, 3, 4, 5 postinfection hypervirulent mutants. All experiments were <br />                                                                                           died quickly did LVS-infected flies (see Fig. <br /> repeated times. Prior infection, flies reared standard corn <br /> meal agar room temperature.                                                            S1A supplemental material), observation has <br />    Fly infection. In test, 20 adult flies (10 females 10 males, 2 8 days      mouse infection models (22). Infection by <br /> old) infected pricked glass needle dipped bacterial               pricking reproducibly delivered inoculum &#11011;104 CFU per <br /> colonies. Flies incubated 29&#176;C transferred fresh food day.           fly, killing flies approximately 7 days. Furthermore, <br /> Living animals counted day 10 days, results were <br /> recorded percentage number living animals relation the <br />                                                                                           ascertained U112 mutant known virulence de- <br /> number flies recorded 1 day postpricking. Bacterial mutants tested sets        terminant, MglA, attenuated phenotype similar in <br /> of 10 15 strains plus wild-type replicates set. Each mutant allele         terms fly survival previously demonstrated an <br />  3120     &#197;HLUND ET AL.                                                                                                     INFECT. IMMUN. <br />  <br />  <br />                                                                        FTN_1325 (pdpD), FTN_1326 (anmK), behaved like the <br />                                                                        wild-type control flies. Surprisingly, mu- <br />                                                                        tants hypervirulent flies (FTN_0119, <span id='am-1' about='obo:NCBITaxon_255477' typeof='owl:Thing obo:NCBITaxon_6475 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_75116 obo:NCBITaxon_54973 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_6072'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="rdfs:label" content="Pila" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>pilA</span>, pckA, fimT, <br />                                                                        pilB, pilM). <br />                                                                           For 49 loci identified significantly altered fly sur- <br />                                                                        vival, wanted confirm mutant phenotype testing <br />                                                                        sister alleles library. For 42 candidates, library <br />                                                                        provided second allele, transposon inserted <br />                                                                        downstream ORF. For one, could <br />                                                                        confirm effect bacterial gene knockout fly survival <br />                                                                        (see Table S1 supplemental material). The correlation <br />                                                                        median &#9004;MLL allele the <br />                                                                        second allele high (R &#11005; 0.923) (see Fig. S3 supple- <br />                                                                        mental material), ensuring candidates true <br />                                                                        &#8220;hits.&#65533;? In case FTN_0672, second allele produced a <br />                                                                        different, nonattenuated phenotype. Sequencing confirmed <br />                                                                        alleles harbored transposon expected po- <br />                                                                        sition genome. However, nonattenuated allele the <br />                                                                        transposon inserted 9 bp end ORF and <br />                                                                        probably does disrupt gene function. In cases of <br />                                                                        FTN_0119 greA, test sister allele, <br />   FIG. 1. Flow chart visualizing sequence experiments. The      mutants downstream genes FTN_0118 fimT, respec- <br /> various assays analysis methods marked rectangular boxes,   tively, demonstrated phenotype different from <br /> and groups bacterial mutants marked ovals. Assay readouts <br />                                                                        FTN_0119 greA mutants (see Table S1 the <br /> used multivariate analysis indicated dotted arrows. <br />                                                                        supplemental material). We checked available in- <br />                                                                        formation insertion sites mutant strains but <br />                                                                        indications off-target effects downstream <br /> LVS mutant gene (see Fig. S1B supplemen-            genes operon, so-called polar effects, possible <br /> tal material compare reference 56).                             exceptions insertions tig FTN_0772 (see Dis- <br />    Assessment putative virulence determinants fly sur-           cussion). Since tig genes putative operon <br /> vival. We performed directed screen genes           (tig-clpP-clpX-lon), possible transposon insertion in <br /> been implicated pathogenicity recent negative-selec-         tig disrupts expression entire operon creates <br /> tion screens mice. These studies identified          additive phenotype. With respect fly survival, mutants of <br /> U112 transposon insertion mutants unable spread              genes operon mildly attenuated did <br /> spleen 48 h intraperitoneal infection (57) LVS mutants        differ significantly wild type, SCHU S4 mu- <br /> deficient persistence lung 7 days intranasal           tants tig lon attenuated mice (21). <br /> infection (51). A total 240 mutants absent           For large number mutant strains median <br /> target organs studies, corresponding genes        &#9004;MLL differed 1.5 days wild type <br /> were considered candidate virulence factors           significant, possible transposon fails to <br /> mouse. Of genes, 222 represented U112 two-            disrupt gene function (see example given above). However, <br /> allele library; residual loci covered library,      bacterial strains two-allele library originally been <br /> since belong large group candidate essential            selected carry transposon insertions 5 70% ORF length <br /> genes required bacterial survival broth 37&#176;C        increase chance null alleles (15). Furthermore, mu- <br /> (15). In addition 222 loci, included number genes       tants 11 genes nonsignificant phenotype flies were <br /> in screen suggested putative virulence            negatively selected mouse infection model, used the <br /> factors investigations.                                       transposon library did (23). So, differences between <br />    Altogether, individually tested mutants 249 genes          bacterial mutant phenotypes flies mice ex- <br /> flies using mutant allele transposon inserted       plained partly assumption competitive selection in <br /> the 5&#11032; half open reading frame (ORF) (see study outline         mice sensitive assay individual testing flies <br /> in Fig. 1). For mutant, monitored survival infected         assay detect subtle phenotypes. In ad- <br /> flies (see examples Fig. 2A) calculated median              dition, conservative cutoff criteria (&#11022;1.5-day difference, <br /> difference median life length compared U112-            P &#65533;? 0.005) used define attenuated hypervirulent <br /> infected flies (see Table S1 Fig. S2 supplemental           mutants contributed differences (see Discussion). <br /> material). We used nonparametric statistical test            We decided conservative cutoff criteria avoid high <br /> determine mutant killed flies significantly             percentage false-positive hits. Consequently, fly model <br /> slowly quickly did U112 (see Materials Methods).           robust, false-negative hits oc- <br /> We identified 43 loci attenuated killing Drosoph-        curred. <br /> ila. These included FPI genes, grouped             Importance candidate genes bacterial proliferation in <br /> an average median &#9004;MLL 4.3 &#11006; 0.7 days. Only mutants              flies. Next, wanted know mutants&#8217; effect on <br /> four FPI genes, FTN_1319 (pdpC), FTN_1320 (pdpE),                      fly survival dependent bacterial proliferation. Before <br />  VOL. 78, 2010                                                          FLY SCREEN OF FRANCISELLA VIRULENCE FACTORS                             3121 <br />  <br />  <br />  <br />  <br />    FIG. 2. Phenotypic features attenuated hypervirulent (FTN_0119) F. novicida transposon insertion mutants nonannotated genes, as <br /> determined various assays. If noted, representative results experiment shown mutants, averages are <br /> shown U112. The legend applies panels A, B, E, numbers refer FTN designations (e.g., 0096 represents the <br /> FTN_0096 mutant). (A) Survival infected flies. (B) Bacterial growth liquid culture 29&#176;C. The averages standard errors means <br /> (SEM) shown mutants (n &#11005; 3). (C) Bacterial proliferation adult flies day 1 (white), day 3 (gray), day 5 (black) postinfection. <br /> Day 5 values determined FTN_1412 mutant. (D) Bacterial proliferation J774 cells 45 60 min (white) 24 h (black) after <br /> infection. Note FTN_0119 mutant, bacterial numbers higher 16 h (8.05 log10 CFU/well). (E) Bacterial cytotoxicity, measured <br /> by release lactate dehydrogenase J774 cells culture medium. <br />  <br />  <br />  <br /> we carried viable count determination infected flies,              hypervirulent mutant strains 29&#176;C (see examples in <br /> we needed ensure attenuation flies caused             Fig. 2B). For mutant, determined vitro prolif- <br /> a general growth defect mutants experimental               eration index relation U112 (see Materials and <br /> conditions. As mentioned above, two-allele library does             Methods). Only mutants grew considerably slowly <br /> include mutants general growth defect 37&#176;C. Never-                U112, i.e., glgB, sspA, manB, FTN_0848, and <br /> theless, FTN_0665, FTN_0821, FTN_1661 mutant strains                    FTN_1412 mutants, confirming cases low <br /> grew slowly agar media 37&#176;C hands                 temperature did limit proliferation (Tables 1 2; also <br /> investigated further. Since carried fly infection experi-            Fig. S4A supplemental material). <br /> ments 29&#176;C, highest temperature D. melano-                    We followed bacterial proliferation flies first <br /> gaster kept extended period, monitored                 5 days infection 46 strains (examples Fig. 2C). The <br /> vitro growth remaining 40 attenuated mutant strains              bacterial load day 3 chosen readout, pro- <br />  3122      &#197;HLUND ET AL.                                                                                                                           INFECT. IMMUN. <br />  <br />  <br />                                      TABLE 1. Features attenuated F. novicida transposon insertion mutantsf <br />                                                                                                           D. melanogaster                       J774 cells <br />                                                                                      In vitro <br /> Gene       Locus tag                     Gene description                     proliferation        Fly          Bacterial       Intracellular      Cytotoxicity <br />                                                                                   index (n &#11005; 3)      survivalb        loadc          proliferation        indexe <br />                                                                                                     (n &#11005; 2&#8211;7)      (n &#11005; 2 3)     indexd (n &#11005; 2)       (n &#11005; 2) <br />  <br />   pyrB         FTN_0019        Aspartate carbamoyltransferase                          1.03            3.57           20.35              0.12                1.02 <br />   carB         FTN_0020        Carbamoyl-phosphate synthase large                      1.02            3.60           20.74              0.20                0.93 <br />                                  chain <br />   carA         FTN_0021        Carbamoyl-phosphate synthase small                      1.01            2.18           20.21              0.13                0.93 <br />                                  chain <br />   pyrF         FTN_0035        Orotidine-5&#11032;-phosphate decarboxylase                    0.94            2.68           20.31              0.22                1.01 <br />   pyrD         FTN_0036        Dihydroorotate oxidase                                  0.93            2.00           20.49              0.27                1.02 <br />   feoB         FTN_0066        Ferrous iron transport protein B                        1.04            2.18           18.86              1.05                1.00 <br />                FTN_0096        Conserved hypothetical membrane                         1.11            1.59           20.39              0.54                1.12 <br />                                  protein <br />                FTN_0109        Novel protein unknown function                       1.10            4.06            7.68              0.01                0.12 <br />   ribC         FTN_0113        Riboflavin synthase alpha chain                         1.10            4.38           20.07              0.31                0.79 <br />   minD         FTN_0330        Septum formation inhibitor-activating                   0.81            3.00           17.31              0.46                0.50 <br />                                   ATPase <br />   minC         FTN_0331        Septum formation inhibitor                              0.87            2.60           16.82              0.38                0.99 <br />   slt          FTN_0496        Soluble lytic murein transglycosylase                   0.89            3.63           14.49              0.04                0.84 <br />   glgB         FTN_0513        1,4-&#65533;?&#65533;-Glucan branching enzyme                           0.75            4.44            8.26              0.24                0.97 <br />   sspA         FTN_0549        Stringent starvation protein A                          0.44            3.43           17.93              1.76                0.69 <br />   secA         FTN_0672        Preprotein translocase, subunit A                       0.92            2.06           16.19              0.10                1.16 <br />                FTN_0772        Conserved protein unknown function                   0.95            4.16           17.06              0.01                0.41 <br />                FTN_0806        Glycosyl hydrolase <span id='am-40' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-41' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> 3                             1.03            1.85           21.00              0.76                1.12 <br />                FTN_0848        Amino acid antiporter                                   0.65            2.33           21.18              0.00                0.25 <br />   tig          FTN_1058        Trigger factor (TF) protein                             0.87            1.95           20.58              0.40                1.11 <br />                FTN_1240        BolA family protein                                     0.86            1.86           19.33              0.14                0.92 <br />   pdpA         FTN_1309        Protein unknown function                             1.08            4.01           18.85              0.00                0.04 <br />   icmFa        FTN_1310        Conserved protein unknown function                   1.04            5.50           19.07              0.00                0.03 <br />   iglEa        FTN_1311        Protein unknown function                             1.04            4.53           19.34              0.01                0.04 <br />   vgrGa        FTN_1312        Conserved hypothetical protein                          1.03            3.03           20.22              0.01                0.03 <br />   iglFa        FTN_1313        Hypothetical protein                                    1.00            3.07           20.08              0.02                0.03 <br />   iglGa        FTN_1314        Conserved hypothetical protein                          1.05            3.53           17.74              0.00                0.03 <br />   iglHa        FTN_1315        Protein unknown function                             0.99            4.25           18.13              0.01                0.03 <br />   dotUa        FTN_1316        Conserved protein unknown function                   0.95            4.71           17.31              0.00                0.03 <br />   iglIa        FTN_1317        Protein unknown function                             0.97            4.71           13.70              0.01                0.03 <br />   iglJa        FTN_1318        Hypothetical protein                                    0.98            4.71           18.49              0.01                0.03 <br />   iglD         FTN_1321        Intracellular growth locus protein D                    0.98            4.70           17.87              0.01                0.03 <br />   iglC         FTN_1322        Intracellular growth locus protein C                    0.98            4.70           19.14              0.01                0.04 <br />   iglB         FTN_1323        Conserved protein unknown function                   1.00            4.56           20.15              0.01                0.04 <br />   iglA         FTN_1324        Conserved protein unknown function                   0.97            4.71           18.37              0.01                0.04 <br />   recB         FTN_1357        ATP-dependent exoDNase &#65533;?&#65533; subunit                        0.97            1.49           21.88              0.85                1.16 <br />   feoA         FTN_1368        Fe2&#11001; transport protein A                         1.05            1.98           19.05              0.85                1.10 <br />                FTN_1412        DNA-directed RNA polymerase &#65533;?&#65533;                           0.42            3.78           14.52              0.24                0.97 <br />                                  subunit <br />   manB         FTN_1417        Phosphomannomutase                                      0.73            2.19           19.88              0.00                0.72 <br />   dsbB         FTN_1608        Disulfide bond formation protein                        1.01            4.43           15.90              0.00                0.12 <br />   clpB         FTN_1743        Chaperone ClpB                                          1.00            3.69           18.48              0.09                0.05 <br />  a <br />    Nomenclature according reference 29. <br />  b <br />    Fly survival given difference median life length (&#9004;MLL) mutant- U112-infected flies (in days). <br />  c <br />    Bacterial load 3 days postinfection given log2 CFU/fly. The mean bacterial load U112-infected flies 21.7 &#11006; 1.2 (n &#11005; 18). <br />  d <br />    Intracellular proliferation index 24 h postinfection. <br />  e <br />    Cytotoxicity index 48 h postinfection. <br />  f <br />    Median values given assays; n, number experiments. <br />  <br />  <br />  <br />  <br /> liferation reached stationary phase time                          Importance candidate genes infection mouse mac- <br /> point (Tables 1 2; Fig. S4B supplemental                        rophages. Only genes suggested as <br /> material). Overall, strong significant negative                     putative virulence factors negative-selection screens in <br /> correlation bacterial load fly survival (Table 3).                      mice investigated individually. Since intra- <br /> Hypervirulent mutant strains grew numbers                           cellular survival F. tularensis prerequisite viru- <br /> the U112 average, strongly attenuated mutants                         lence, sought establish candidate genes were <br /> reduced growth vivo. However, attenuated                            required virulence mechanism. We tested the <br /> mutants, quite strains grew numbers similar                       46 mutant strains intracellular growth cytotoxicity in <br /> for U112, indicating Francisella virulence flies                  mouse macrophage-like J774 cells. The proliferation time <br /> depend bacterial burden host.                                    uptake 16 h percent cytotoxicity 24 h were <br />  VOL. 78, 2010                                                                    FLY SCREEN OF FRANCISELLA VIRULENCE FACTORS                                    3123 <br />  <br />  <br />                                     TABLE 2. Features hypervirulent F. novicida transposon insertion mutantse <br />                                                                                                            D. melanogaster                       J774 cells <br />                                                                                       In vitro <br /> Gene        Locus tag                     Gene description                     proliferation         Fly          Bacterial      Intracellular       Cytotoxicity <br />                                                                                    index (n &#11005; 3)       survivala        loadb         proliferation         indexd <br />                                                                                                      (n &#11005; 3 4)      (n &#11005; 3)       indexc (n &#11005; 2)        (n &#11005; 2) <br />  <br />                 FTN_0119         Conserved outer membrane protein                   1.03             &#11002;1.88           22.71             0.24                3.66 <br />                                    unknown function <br />       pilA      FTN_0415         Type IV pili, pilus assembly protein                  1.06             &#11002;2.66           22.68             0.03                3.36 <br />       pckA      FTN_0540         Phosphoenolpyruvate carboxykinase                     0.94             &#11002;1.96           22.79             1.07                0.70 <br />       fimT      FTN_0664         Type IV pili, pilus assembly protein                  0.97             &#11002;1.58           21.84             0.10                2.64 <br />       pilB      FTN_1115         Type IV pili ATPase                                   1.03             &#11002;1.98           23.38             0.15                2.77 <br />       pilM      FTN_1141         Type IV pili, pilus assembly protein                  1.06             &#11002;2.16           20.22             0.06                3.52 <br />   a <br />     Fly survival given difference median life length (&#9004;MLL) mutant- U112-infected flies (in days). <br />   b <br />     Bacterial load 3 days postinfection given log2 CFU/fly. The mean bacterial load U112-infected flies 21.7 &#11006; 1.2 (n &#11005; 18). <br />   c <br />     Intracellular proliferation index 16 h postinfection. <br />   d <br />     Cytotoxicity index 24 h postinfection. <br />   e <br />     Median values given assays; n, number experiments. <br />  <br />  <br />  <br />  <br /> chosen readouts hypervirulent strains; prolifer-                      features determined different assays correlated <br /> ation time uptake 24 h percent cytotoxicity                       another. Apart correlation bacte- <br /> at 48 h chosen readouts attenuated strains                           rial load virulence model systems, strong <br /> (examples Fig. 2D E). These time points demonstrated                          significant correlations fly survival intracel- <br /> the best distinction mutant wild-type infections.                        lular proliferation fly survival cytotoxicity <br /> Again, related mutants&#8217; values corre-                         (Table 3). This correlation readouts two <br /> sponding U112 control expressed results indices                           model systems suggests survival infected flies indeed <br /> (Tables 1 2). In concordance negative selection                          good indicator bacterial virulence mice. Encouraged by <br /> mice reduced virulence flies, attenuated mutants                         result, thought multivariate analysis mu- <br /> had low intracellular proliferation indices,                       tants&#8217; phenotypic features allow conclusions regard- <br /> also demonstrated reduced cytotoxicity (see Fig. S4C D                        ing functional relationships corresponding genes. <br /> the supplemental material). Despite strong significant                         Based readouts listed Tables 1 2, per- <br /> correlation intracellular proliferation cytotoxicity                     formed separate hierarchical cluster analyses attenu- <br /> (Table 3), mutants strongly reduced intracellular                          ated hypervirulent mutants (Fig. 3). At arbitrary <br /> growth fully cytotoxic, i.e., pyrB, carA, secA,                     height 10, clusters loci attenuated mutant <br /> FTN_1240 mutants. Similarly conclusion mentioned                              phenotype clearly distinguishable. Cluster 1 mutants dif- <br /> above, results indicate cytotoxicity depend                     fered little U112 wild type, biggest deviation <br /> on bacterial burden alone.                                                           seen intracellular proliferation. Candidate virulence genes <br />    Five hypervirulent mutants 16-h proliferation index                         group function primarily basic metabolic processes, <br /> that far U112 (Table 2). Since strains                       like energy metabolism (ribC) arginine pyrimidine <br /> were highly cytotoxic 24 h postinfection, re-                       biosynthesis (carAB, pyrB, pyrFD), be <br /> duced proliferation index probably premature                              important bacterial adaptation oxidative stress. For in- <br /> death host cells.                                                             stance, CarA, CarB, PyrB required suppression <br />    Identification functionally linked genes multivariate                       oxidative burst neutrophils (46). FTN_1240 ho- <br /> analysis. Finally, asked mutants&#8217; phenotypic                          mology BolA morphogens, switch cell morphology <br />                                                                                      type induced carbon star- <br />                                                                                      vation acidic oxidative stress (44). Tig, called trigger <br />       TABLE 3. Spearman&#8217;s rank correlations (RS)                      factor, act posttranslational protein modification in <br />                   readouts discussed study                                    periplasm; suggested putative virulence deter- <br />              Group readouts compared                       RS      P value       minant extensive bioinformatic analysis Francisella <br />                                                                                      genomes (43). Interestingly, mutants FTN_0848, putative <br /> All mutants (n &#11005; 46)                                                                 amino acid transporter, manB, phosphomannomutase, <br />   Fly survival/in vitro proliferation                       &#11002;0.03      0.8257 <br />   Fly survival/bacterial load flies                      &#11002;0.65      0.0000        formed subgroup cluster 1; did proliferate <br />   In vitro proliferation/bacterial load flies             0.23      0.1280        vitro macrophage-like cells grew normally in <br />                                                                                      flies. Knowing Francisella able grow extracellularly in <br /> Attenuated mutants (n &#11005; 40)                                                          fly (56), assume fly hemolymph, par- <br />   Fly survival/intracellular proliferation                  &#11002;0.59      0.0002 <br />   Fly survival/cytotoxicity                                 &#11002;0.71      0.0000 <br />                                                                                      ticularly rich amino acids organic acids citric <br />   In vitro proliferation/intracellular proliferation        &#11002;0.08      0.5927        acid cycle (11), compensates deficiency two <br />   In vitro proliferation/cytotoxicity                       &#11002;0.17      0.2770        gene functions. <br />   Bacterial load flies/intracellular proliferation        0.19      0.2458           Phenotypic features genes cluster 2 overall similar <br />   Bacterial load flies/cytotoxicity                       0.30      0.0641        group, mutants&#8217; growth flies was <br />   Intracellular proliferation/cytotoxicity                   0.73      0.0000 <br />                                                                                      reduced. The functions genes related cell <br />  3124      &#197;HLUND ET AL.                                                                                                           INFECT. IMMUN. <br />  <br />  <br />  <br />  <br />    FIG. 3. Hierarchical cluster analysis phenotypic features attenuated hypervirulent (inset) F. novicida mutants. Readout variables from <br /> all analyses listed Tables 1 2 standardized. The Manhattan distance used calculate distances mutants, and <br /> Ward&#8217;s method used calculate distances clusters. Circled numbers identify clusters phenotypically similar attenuated <br /> mutants, numbers used identify genes refer FTN designations (e.g., 1240 represents FTN_1240). See text details. <br />  <br />  <br />  <br />  <br /> division and, again, oxidative stress. MinC MinD                 rectly regulates expression 30 genes, among <br /> homologous proteins required chromosome partitioning                 genes FPI (6). <br /> during cell division components oxyanion pump.                    The mutants cluster 5 extremely reduced their <br /> Based findings Francisella minD mutant sen-              growth flies strongly attenuated killing flies, suggest- <br /> sitive serum oxidative killing, Anthony et al. hypoth-            ing particularly sensitive fly&#8217;s immune <br /> esized locus forms ion pump extrude toxic            defense. The FTN_0109 mutant strongly affected in <br /> ions radicals required maintain cell wall integrity            macrophage-like cells. FTN_0109 novel protein un- <br /> (1). Slt murein-degrading enzyme probably impor-                known function associated outer mem- <br /> tant cytokinesis. Like Tig, Slt listed putative             brane; GlgB annotated 1,4-&#65533;?&#65533;-glucan branching enzyme <br /> virulence factor Rohmer et al. (43). The gene product                 involved glycogen biosynthesis. <br /> FTN_1412 annotated DNA-dependent RNA polymer-                          The mutant strains cluster 6 formed tight group, <br /> ase. SecA takes conserved protein secretion process               strong effects virulence flies J774 cells. Most <br /> across inner membrane.                                                  corresponding genes belong FPI. Six these <br />    Cluster 3 mutants weakly attenuated flies                genes, icmF, dotU, vgrG, iglA, iglB, iglD, sug- <br /> attenuated J774 cells, indicating targeted genes            gested function components type VI secretion system <br /> important intracellular survival. The feoA feoB genes,              (reviewed references 3 13), iglC gene product <br /> which participate iron transport, recB, involved           depends components proper localization the <br /> DNA repair, related defense oxidative stress.            outer membrane (29). Two additional cluster 6 loci, clpB and <br /> Taken together, observations suggest cluster 3 genes             dsbB, involved protein modification important <br /> play role extracellular period Francisella life         proliferation cells. ClpB belongs highly conserved <br /> cycle. In insect hemolymph, reactive oxygen nitrogen                ring-forming AAA&#11001; <span id='am-143' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-144' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-145' property="rdfs:label" content="superfamily" datatype="xsd:string"></span>superfamily</span> ATPases, associ- <br /> species contribute antimicrobial defense.                            ated virulence pathogens, including F. tularensis <br />    The sspA mutant (cluster 4) separated                     (7, 54, 60, 62). ClpB proteins disaggregate reactivate <br /> groups, slow growth vitro surprisingly high           strongly aggregated proteins appear critically re- <br /> intracellular proliferation index. However, absolute num-               quired adaptation various forms stress. DsbB, to- <br /> bers intracellular bacteria immediately uptake              gether DsbA components disulfide bond <br /> 24 h fact 10% levels U112 (data                 formation system, required efficient protein secretion <br /> shown). SspA, stringent starvation protein A, thought              periplasm number bacterial pathogens (27, 33). <br /> involved intracellular survival; MglA, di-              Recent studies demonstrated importance F. tularensis <br />  VOL. 78, 2010                                                         FLY SCREEN OF FRANCISELLA VIRULENCE FACTORS                      3125 <br />  <br />  <br /> virulence mice (39, 40, 50). We tested mutant F.             attenuated mutant phenotype mice. There possi- <br /> novicida homolog dsbA, FTN_0771. This mutant                  ble explanations phenotypic differences mutants mice <br /> attenuated flies (see Table S1 supplemental material),         flies. We mentioned Results, and <br /> which transposon insertion close 3&#11032; end           lie differences mutant strains <br /> of ORF. Instead, FTN_0772 mutants at-                 tested (types mutation subspecies). Despite <br /> tenuated closely linked dsbB strain multi-           97% sequence identity U112, LVS, and <br /> variate analysis. One possible explanation             SCHU S4, differences numbers protein-encoding <br /> transposon insertion FTN_0772 allele generated polar            genes ORF lengths exist affect vivo life cycle <br /> effect downstream gene FTN_0771. Interestingly,               host immune response (12, 26, 43). For example, <br /> short FTN_0772 ORF 29% identical FTN_0771                  16 200 nonsignificant F. novicida genes either <br /> and shows homology FK506 binding protein-like peptidyl-             strongly truncated missing live vaccine strain <br /> prolyl <span id='am-138' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-139' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-142'  ></span>cis</span>-trans isomerase 1 similar shown DsbA.              (see Table S1 supplemental material). <br /> Therefore, excluded phenotype              However, differences mutant phenotypes reflect <br /> FTN_0772 alleles truly reflects deficiency gene func-          physiological differences model organisms <br /> tion.                                                                    compared here. For instance, noteworthy genes in- <br />    The hypervirulent mutants formed relatively tight cluster           volved purine metabolism (purM, purF, purL, relA) and <br /> characterized increased virulence flies J774 cells,         basic metabolic functions (glpX, FTN_0720, aroG, ggt, <br /> with pckA mutant outlier, wild-type-like            FTN_1213) important mice flies <br /> behavior cell culture assay (inset Fig. 3). In concor-         (see Table S1 supplemental material). Assuming that <br /> dance results, pckA shown important             intracellular conditions Francisella overall similar <br /> for virulence mice (21). pckA outlier terms          host models, insect hemolymph probably repre- <br /> gene function, loci appear contribute        sents biggest difference. It possible that, similarly our <br /> to type IV pili surface structures bacteria (see      observations regarding FTN_0848 manB, correspond- <br /> Discussion).                                                             ing deficiencies compensated rich composition of <br />    In summary, multivariate analysis provided ample exam-            Drosophila hemolymph (11). Another group genes re- <br /> ples mutants similar phenotypes, mutants clustered          quired virulence mice flies encode enzymes of <br /> tightly, mutants targeted genes function-          lipopolysaccharide capsule biosynthesis pathways <br /> ally linked.                                                             (capB, capC, capA, wbtA). Despite high degree con- <br />                                                                          servation components innate immune response <br />                                                                          insects mammals, differences systems <br />                          DISCUSSION <br />                                                                          exist contribute outcome screen. <br />    We previously established fruit fly model or-            Pathogen recognition, instance, based different patho- <br /> ganism infections F. tularensis, employed           gen moieties different host receptors. <br /> this model identify F. novicida virulence factors individual          Induction virulence factors shift mammalian <br /> testing large number bacterial mutants. Combining fly            body temperature known pathogens, one <br /> experiments mouse cell culture assays, identified genes           assume lower temperature insect host <br /> involved virulence different host systems genes            affect hits retrieved model. Indeed, <br /> necessary intracellular survival cytopathogenicity. A            mutations 11 genes induced shift 26&#176; to <br /> strong correlation results model systems            37&#176;C did generate phenotype flies (20). However, five <br /> evaluated D. melanogaster reliable model host.                      upregulated genes, pyrB, carB, carA, clpB, pilB, were <br />    Of 249 Francisella loci suggested putative virulence de-           required normal virulence flies. Similarly, <br /> terminants mice, 49 required normal                genes upregulated infection mice (compared to <br /> virulence flies. Six act virulence modifi-        expression growth broth 37&#176;C) did a <br /> ers, corresponding transposon mutant strains              mutant phenotype flies, exception iglC and <br /> hypervirulent (see below). For 49 candidates,         FTN_0119 (55). <br /> recent literature provides confirmatory data infections             Comparing vivo screens mice, evident that <br /> mice U112, LVS, SCHU S4 mutants showing               performed widely varying experimental con- <br /> genes required normal virulence insect         ditions (types mutant library, routes infection, etc.) and <br /> in mammalian model organism (see Table S1 sup-                documented different readouts (target organs, time <br /> plemental material references).                                      postinfection, detection mutants) (23, 51, 57). These <br />    Of 200 genes significant phenotype flies,            differences explain small overlap candidate virulence <br /> about 20% tested individual mutant analysis              factors obtained studies. The reports we <br /> mice researchers (see Table S1 supplemental              based mutant selection 19 candidate genes in <br /> material references). From analyses, 26 genes             common, corresponding 12% (57) 20% (51) overlap. <br /> shown important virulence U112, LVS,                 Compared numbers, 19% overlap our <br /> SCHU S4 mice. Thus, genes considered false-              results original mouse screens appears <br /> positive hits original negative-selection screens, al-          expected range. Mutations FTN_0848, iglA, <br /> though them, importance virulence differed de-            iglB, iglC, clpB caused attenuation mouse <br /> pending genetic background. The residual 19 loci,               screens flies. A recent study Kraemer et al., in <br /> tested various backgrounds, confirmed                 F. novicida two-allele library screened an <br />  3126     &#197;HLUND ET AL.                                                                                                   INFECT. IMMUN. <br />  <br />  <br /> aerosol infection model, identified genes         scribed previously (9, 10, 48). In mammalian systems, Fran- <br /> picked previous screens; these, iglD         cisella demonstrated interfere Akt signaling <br /> vgrG, generated attenuated phenotype flies (23).                (5, 41). We previously speculated energy wasting or <br />    By analysis bacterial growth vitro vivo    similar disruptions host physiology contribute fly <br /> of cytopathogenicity cultured mouse macrophage-like cells,         death LVS infection (56). Despite overall good cor- <br /> we characterized candidates attenuated         relation bacterial load virulence flies, here <br /> a hypervirulent phenotype flies. Consistent observa-          identified genes appear contribute host death apart <br /> tions mice, significant correlation bac-        simple linkage bacterial numbers, i.e., loci in <br /> terial proliferation virulence vivo fly model    cluster 1; FTN_0096, FTN_0806, recB cluster 3; iglE, <br /> the cell culture model (47). More importantly, experiments        iglB, vgrG, iglF FPI. While mutants genes <br /> bridge insect mammalian models.                               proliferated normally flies, hampered their <br />    The strong correlation mutant phenotypes flies          proliferation mouse macrophage-like cells. Nevertheless, <br /> and cultured mouse macrophages proves survival              demonstrated nearly wild-type cytotoxicity in <br /> infected flies reliable indicator bacterial cytopatho-     cells. This discrimination cell culture system <br /> genicity mouse model. It demonstrates impor-          vivo situation suggests targeted genes affect <br /> tance cytopathogenicity virulence mechanism             systemic host processes cell-autonomous re- <br /> insect host. This consistent previous findings       sponse. <br /> LVS able replicate inside hemocytes, phagocytes con-           In addition genes required virulence, identified <br /> sidered functional equivalent mammalian macro-           genes considered &#8220;virulence modifiers,&#65533;? since <br /> phages, flies phagocytosis inhibited suc-       generated hypervirulent mutant phenotype. Zogaj et al. <br /> cumbed quickly infection (56).                                forward <span id='am-55' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-56' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58'  ></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>idea</span> PilA virulence modifier after <br />    By grouping mutants clusters phenotypically sim-       noted length PilA various Francisella <br /> ilar genes, multivariate analysis provides additional insight     species strains reciprocal virulence bacteria <br /> into gene function. Half candidate virulence factors       (61). As mentioned above, genes hypervirulent group <br /> involved cell division basic metabolic processes,       likely contribute formation type IV pili. The <br /> possible role responses oxidative stress. These genes al-       validity results supported finding F. novi- <br /> most exclusively formed left branch cluster tree,          cida pilA, pilB, pilC mutants exhibited increased virulence <br /> while genes involved intracellular survival segregated      mice (18). These authors demonstrated PilA <br /> right branch. Within clusters, examples tightly       PilB fact required secretion number of <br /> linked genes known expected act        substrates. Hager et al. hypothesized type IV pilus-medi- <br /> cellular process. This linkage primes hypothesis un-         ated secretion metalloprotease PepO inhibits spreading <br /> known genes involved functions related        bacteria mammalian vascular (18). PepO <br /> better characterized nearby neighbor(s) cluster.               displays homology mammalian endothelin converting <br />    We 14 18 genes FPI required          enzyme 1, induces vasoconstriction (32). Type IV pilus- <br /> full virulence flies proliferation J774 cells. For      mediated secretion mutants easily dis- <br /> iglE, iglF, iglG, iglH, dotU, iglI, data far     seminate blood circulatory system. Although theory <br /> been available, genes believed play          explain hypervirulent phenotype fly, is <br /> role intracellular survival. There indirect evidence   interesting note D. melanogaster expresses ho- <br /> the importance FPI genes protozoan model, mu-           mologues endothelin converting enzyme 1, closest being <br /> tants iglC major regulator FPI gene expression,         neprilysin 3 (4, 30). Two observations interesting in <br /> mglA, proliferate <span id='am-43' about='obo:NCBITaxon_5754' typeof='owl:Thing obo:NCBITaxon_555280 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_555407 obo:NCBITaxon_2759 obo:NCBITaxon_33677 obo:NCBITaxon_1485168 obo:NCBITaxon_554915'><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Acanthamoeba" datatype="xsd:string"></span><span id='am-46'  ></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Acanthamoeba</span> castellanii (28).       respect. First, expression pilB upregulated the <br /> In screen, FPI genes did affect virulence       attenuated live vaccine strain exposure 37&#176;C (20), and <br /> in flies pdpC, pdpE, pdpD, anmK. Consistent         second, FTN_0119 gene product expressed 2- 16- <br /> findings, Read et al. reported pdpC pdpD             fold-lower levels SCHU S4 virulent strains F. <br /> required intracellular proliferation insect cell line       tularensis subspecies holarctica (36). From experiments, <br /> (42). Similar results obtained pdpD mouse macro-          Zogaj et al. concluded that, PilA PilB form <br /> phages, mice PdpD             secretion apparatus, gene products FTN_0389 (pilE4) <br /> AnmK plays role virulence F. novicida (29). Neverthe-         FTN_1622 (pilT) PilB form type IV pili (61). <br /> less, requirement FPI genes Francisella survival        They exclude pil genes important <br /> organisms diverse amoebae, flies, mice indicates       pilus formation. Information role fimT pilM <br /> this bacterium adapted sustain itself variety host      incomplete; were, however, listed candidate virulence <br /> cells. This adaptation key ability Fran-       factors Rohmer et al. (43). From observations, it <br /> cisella survive environment provides explana-        type IV pili mask true Francisella <br /> tion occasional outbreaks tularemia seen worldwide.        virulence slow infection process. <br />    In addition intracellular proliferation, interference         We shown D. melanogaster valuable host sys- <br /> systemic physiological processes play role Fran-        tem vivo studies Francisella. Our screening protocol <br /> cisella pathogenicity. &#8220;Energy wasting&#65533;? well-known effect        reliably efficiently identified attenuated hypervirulent <br /> of chronic inflammation humans investi-          mutants basis fly survival. Thanks availability <br /> gated fruit fly, infection-induced inhibition     mapped transposon mutant libraries, genome-wide screen <br /> the Akt kinase insulin receptor pathway de-           identify novel Francisella virulence determinants be <br />  VOL. 78, 2010                                                                         FLY SCREEN OF FRANCISELLA VIRULENCE FACTORS                                        3127 <br />  <br />  <br /> performed using protocol. From observations,                             16. Golovliov, I., V. Baranov, Z. Krocova, H. Kovarova, A. Sjo       &#168;stedt. 2003. <br />                                                                                              An attenuated strain facultative intracellular bacterium Francisella <br /> predict determinants required successful mam-                               tularensis escape phagosome monocytic cells. Infect. Immun. <br /> malian infections identified screen. However,                              71:5940&#8211;5950. <br /> we seen important strategies, intracellular                            17. Golovliov, I., M. Ericsson, G. Sandstro  &#168;m, A. Ta &#168;rnvik, A. Sjo&#168;stedt. 1997. <br />                                                                                              Identification proteins Francisella tularensis induced growth in <br /> survival adaptation oxidative stress, active                           macrophages cloning gene encoding prominently induced 23- <br /> insect host. Hits screen D. melanogaster                            kilodalton protein. Infect. Immun. 65:2183&#8211;2189. <br /> most likely universally relevant. In contrast experiments                          18. Hager, A. J., D. L. Bolton, M. R. Pelletier, M. J. Brittnacher, L. A. Gallagher, <br />                                                                                              R. Kaul, S. J. Skerrett, S. I. Miller, T. Guina. 2006. Type IV pili- <br /> in cell culture, vivo fly model offers opportunity                             mediated secretion modulates Francisella virulence. Mol. Microbiol. 62:227&#8211; <br /> discover bacterial interference systemic host processes. It                             237. <br />                                                                                          19. Hollis, D. G., R. E. Weaver, A. G. Steigerwalt, J. D. Wenger, C. W. Moss, and <br /> is worth noting high-throughput screening pro-                                 D. J. Brenner. 1989. <span id='am-150' about='obo:NCBITaxon_28110' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_34064 obo:NCBITaxon_1236 obo:NCBITaxon_72273 obo:NCBITaxon_262'><span id='am-151' property="oboInOwl:hasDbXref" content="PMID:2671019" datatype="xsd:string"></span><span id='am-152' property="rdfs:label" content="Francisella philomiragia" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasDbXref" content="PMID:17766855" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasDbXref" content="PMID:5361214" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasDbXref" content="PMID:2541646" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasRelatedSynonym" content="Yersinia philomiragia" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasAlternativeId" content="NCBITaxon:452326" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasRelatedSynonym" content="&apos;Philomiragia&apos; bacterium" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-161'  ></span>Francisella philomiragia</span> comb. nov. (formerly Yersinia <br /> cedure permits analysis mutant individually                                philomiragia) Francisella tularensis biogroup novicida (formerly Fran- <br /> more rapid ethically economically feasible way                               cisella novicida) associated human disease. J. Clin. Microbiol. 27:1601&#8211; <br />                                                                                              1608. <br /> mammalian model systems. Thus, approach surely                              20. Horzempa, J., P. E. Carlson, Jr., D. M. O&#8217;Dee, R. M. Shanks, G. J. Nau. <br /> aid ongoing quest identify new targets drug                                   2008. Global transcriptional response mammalian temperature provides <br /> vaccine development needed treatment prevention                                  new insight Francisella tularensis pathogenesis. BMC Microbiol. 8:172. <br />                                                                                          21. Kadzhaev, K., C. Zingmark, I. Golovliov, M. Bolanowski, H. Shen, W. Con- <br /> of F. tularensis infections.                                                                 lan, A. Sjostedt. 2009. Identification genes contributing viru- <br />                                                                                              lence Francisella tularensis SCHU S4 mouse intradermal infection <br />                                                                                              model. PLoS One 4:e5463. <br />                           ACKNOWLEDGMENTS                                                22. Kieffer, T. L., S. Cowley, F. E. Nano, K. L. Elkins. 2003. Francisella <br />                                                                                              novicida LPS greater immunobiological activity mice F. tularensis <br />   We thank Colin Manoil providing F. novicida transposon                             LPS, contributes F. novicida murine pathogenesis. Microbes Infect. <br /> mutant library, thank There&#180;se Norrsken technical assis-                          5:397&#8211;403. <br /> tance. We thank members Sjo  &#168;stedt lab technical advice                  23. Kraemer, P. S., A. Mitchell, M. R. Pelletier, L. A. Gallagher, M. Wasnick, L. <br /> and discussions. We grateful Dan Hultmark sharing                         Rohmer, M. J. Brittnacher, C. Manoil, S. J. Skerett, N. R. Salama. 2009. <br /> laboratory space.                                                                            Genome-wide screen Francisella novicida genes required pulmo- <br />   This work supported grant Swedish Research                               nary systemic infection mice. Infect. Immun. 77:232&#8211;244. <br /> Council S.S. (no. 70285201).                                                          24. Kurz, C. L., J. J. Ewbank. 2007. Infection dish: high-throughput <br />                                                                                              analyses bacterial pathogenesis. Curr. Opin. Microbiol. 10:10&#8211;16. <br />                                                                                          25. Lai, X.-H., I. Golovliov, A. Sjo &#168;stedt. 2001. Francisella tularensis induces <br />                                  REFERENCES                                                  cytopathogenicity apoptosis murine macrophages mechanism <br />  1. Anthony, L. S., S. C. Cowley, K. E. Mdluli, F. E. Nano. 1994. Isolation              requires intracellular bacterial multiplication. Infect. Immun. 69:4691&#8211; <br />     Francisella tularensis mutant sensitive serum oxidative              4694. <br />     killing avirulent mice: correlation loss MinD homologue        26. Larsson, P., D. Elfsmark, K. Svensson, P. Wikstrom, M. Forsman, T. Bret- <br />     expression. FEMS Microbiol. Lett. 124:157&#8211;165.                                           tin, P. Keim, A. Johansson. 2009. Molecular evolutionary consequences <br />  2. Barker, J. R., K. E. Klose. 2007. Molecular genetic basis patho-              niche restriction Francisella tularensis, facultative intracellular patho- <br />     genesis Francisella tularensis. Ann. N. Y. Acad. Sci. 1105:138&#8211;159.                   gen. PLoS Pathog. 5:e1000472. <br />  3. Bingle, L. E., C. M. Bailey, M. J. Pallen. 2008. Type VI secretion:            27. Lasica, A. M., E. K. Jagusztyn-Krynicka. 2007. The role Dsb proteins <br />     beginner&#8217;s guide. Curr. Opin. Microbiol. 11:3&#8211;8.                                         Gram-negative bacteria process pathogenesis. FEMS Microbiol. <br />  4. Bland, N. D., J. W. Pinney, J. E. Thomas, A. J. Turner, R. E. Isaac. 2008.           Rev. 31:626&#8211;636. <br />     Bioinformatic analysis neprilysin (M13) family peptidases reveals          28. Lauriano, C. M., J. R. Barker, S.-S. Yoon, F. E. Nano, B. P. Arulanandam, <br />     complex evolutionary functional relationships. BMC Evol. Biol. 8:16.                 D. J. Hassett, K. E. Klose. 2004. MglA regulates transcription viru- <br />  5. Butchar, J. P., T. J. Cremer, C. D. Clay, M. A. Gavrilin, M. D. Wewers, C. B.            lence factors necessary Francisella tularensis intraamoebae intra- <br />     Marsh, L. S. Schlesinger, S. Tridandapani. 2008. Microarray analysis              macrophage survival. Proc. Natl. Acad. Sci. U. S. A. 101:4246&#8211;4249. <br />     human monocytes infected Francisella tularensis identifies new targets          29. Ludu, J. S., O. M. Bruin, B. N. Duplantis, C. L. Schmerk, A. Y. Chou, <br />     host response subversion. PLoS One 3:e2924.                                           K. L. Elkins, F. E. Nano. 2008. The Francisella pathogenicity island <br />  6. Charity, J. C., M. M. Costante-Hamm, E. L. Balon, D. H. Boyd, E. J. Rubin,               protein PdpD required virulence associates homologues <br />     S. L. Dove. 2007. Twin RNA polymerase-associated proteins control                    type VI secretion system. J. Bacteriol. 190:4584&#8211;4595. <br />     virulence gene expression Francisella tularensis. PLoS Pathog. 3:e84.             30. Macours, N., K. Hens. 2004. Zinc-metalloproteases insects: ACE and <br />  7. Chastanet, A., I. Derre, S. Nair, T. Msadek. 2004. clpB, novel member              ECE. Insect Biochem. Mol. Biol. 34:501&#8211;510. <br />     <span id='am-6' about='obo:NCBITaxon_1639' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1637 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186820'><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="PMID:8782698" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Bacterium monocytogenes hominis" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Bacterium monocytogenes" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Corynebacterium parvulum" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Listerella hepatolytica" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="PMID:1906732" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="PMID:17773427" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Erysipelothrix monocytogenes" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Lysteria monocytogenes" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Listeria monocytogenes" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Corynebacterium infantisepticum" datatype="xsd:string"></span>Listeria monocytogenes</span> CtsR regulon, involved virulence         31. Maier, T. M., M. S. Casey, R. H. Becker, C. W. Dorsey, E. M. Glass, N. <br />     general stress tolerance. J. Bacteriol. 186:1165&#8211;1174.                                Maltsev, T. C. Zahrt, D. W. Frank. 2007. Identification Francisella <br />  8. Clarridge, J. E., III, T. J. Raich, A. Sjosted, G. Sandstrom, R. O. Darouiche,           tularensis Himar1-based transposon mutants defective replication in <br />     R. M. Shawar, P. R. Georghiou, C. Osting, L. Vo. 1996. Characterization              macrophages. Infect. Immun. 75:5376&#8211;5389. <br />     unusual clinically significant Francisella strains. J. Clin. Microbiol.       32. Masaki, T. 1998. The discovery endothelins. Cardiovasc. Res. 39:530&#8211;533. <br />     34:1995&#8211;2000.                                                                        33. Messens, J., J. F. Collet. 2006. Pathways disulfide bond formation in <br />  9. Diangelo, J. R., M. L. Bland, S. Bambina, S. Cherry, M. J. Birnbaum.                 <span id='am-102' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-103' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-104'  ></span><span id='am-105' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span>Escherichia coli</span>. Int. J. Biochem. Cell Biol. 38:1050&#8211;1062. <br />     2009. The immune response attenuates growth nutrient storage Dro-             34. Morner, T. 1992. The ecology tularaemia. Rev. Sci. Tech. 11:1123&#8211;1130. <br />     sophila reducing insulin signaling. Proc. Natl. Acad. Sci. U. S. A. 106:          35. Nano, F. E., C. Schmerk. 2007. The Francisella pathogenicity island. <br />     20853&#8211;20858.                                                                             Ann. N. Y. Acad. Sci. 1105:122&#8211;137. <br /> 10. Dionne, M. S., L. N. Pham, M. Shirasu-Hiza, D. S. Schneider. 2006. Akt           36. Pavkova, I., M. Reichelova, P. Larsson, M. Hubalek, J. Vackova, A. Fors- <br />     FOXO dysregulation contribute infection-induced wasting Dro-                   berg, J. Stulik. 2006. Comparative proteome analysis fractions en- <br />     sophila. Curr. Biol. 16:1977&#8211;1985.                                                       riched membrane-associated proteins Francisella tularensis subsp. <br /> 11. Echalier, G. 1997. Drosophila cells culture, p. 3&#8211;11. Academic Press, New             tularensis F. tularensis subsp. holarctica strains. J. Proteome Res. <br />     York, NY.                                                                                5:3125&#8211;3134. <br /> 12. Elkins, K. L., S. C. Cowley, C. M. Bosio. 2007. Innate adaptive              37. Petersen, J. M., P. S. Mead, M. E. Schriefer. 2009. Francisella tularensis: <br />     immunity Francisella. Ann. N. Y. Acad. Sci. 1105:284&#8211;324.                             arthropod-borne pathogen. Vet. Res. 40:7. <br /> 13. Filloux, A., A. Hachani, S. Bleves. 2008. The bacterial type VI secretion        38. Qin, A., B. J. Mann. 2006. Identification transposon insertion mutants <br />     machine: player protein transport membranes. Micro-               Francisella tularensis tularensis strain Schu S4 deficient intracellular <br />     biology 154:1570&#8211;1583.                                                                   replication hepatic cell line HepG2. BMC Microbiol. 6:69. <br /> 14. Forestal, C. A., M. Malik, S. V. Catlett, A. G. Savitt, J. L. Benach, T. J.          39. Qin, A., D. W. Scott, B. J. Mann. 2008. Francisella tularensis subsp. <br />     Sellati, M. B. Furie. 2007. Francisella tularensis significant extra-          tularensis Schu S4 disulfide bond formation protein B, RND-type <br />     cellular phase infected mice. J. Infect. Dis. 196:134&#8211;137.                            efflux pump, required virulence. Infect. Immun. 76:3086&#8211;3092. <br /> 15. Gallagher, L. A., E. Ramage, M. A. Jacobs, R. Kaul, M. Brittnacher, C.           40. Qin, A., D. W. Scott, J. A. Thompson, B. J. Mann. 2009. Identification <br />     Manoil. 2007. A comprehensive transposon mutant library Francisella                   essential <span id='am-78' about='obo:NCBITaxon_119856' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_34064 obo:NCBITaxon_1236 obo:NCBITaxon_72273 obo:NCBITaxon_262 obo:NCBITaxon_263'><span id='am-79' property="oboInOwl:hasDbXref" content="PMID:5462048" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Francisella tularensis subsp. nearctica" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="Francisella tularensis subsp. tularensis" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Francisella tularensis var. tularemia" datatype="xsd:string"></span><span id='am-85'  ></span><span id='am-86' property="oboInOwl:hasAlternativeId" content="NCBITaxon:111848" datatype="xsd:string"></span>Francisella tularensis subsp. tularensis</span> virulence factor. Infect. <br />     novicida, bioweapon surrogate. Proc. Natl. Acad. Sci. U. S. A. 104:1009&#8211;               Immun. 77:152&#8211;161. <br />     1014.                                                                                41. Rajaram, M. V., L. P. Ganesan, K. V. Parsa, J. P. Butchar, J. S. Gunn, and <br />  3128         &#197;HLUND ET AL.                                                                                                                           INFECT. IMMUN. <br />  <br />       S. Tridandapani. 2006. Akt/protein kinase B modulates macrophage inflam-        51. Su, J., J. Yang, D. Zhao, T. H. Kawula, J. A. Banas, J. R. Zhang. 2007. <br />       matory response Francisella infection confers survival advantage        Genome-wide identification Francisella tularensis virulence determinants. <br />       mice. J. Immunol. 177:6317&#8211;6324.                                                    Infect. Immun. 75:3089&#8211;3101. <br /> 42.   Read, A., S. J. Vogl, K. Hueffer, L. A. Gallagher, G. M. Happ. 2008.        52. Tarnvik, A., L. Berglund. 2003. Tularaemia. Eur. Respir. J. 21:361&#8211;373. <br />       Francisella genes required replication mosquito cells. J. Med. Ento-     53. Tempel, R., X. H. Lai, L. Crosa, B. Kozlowicz, F. Heffron. 2006. Atten- <br />       mol. 45:1108&#8211;1116.                                                                  uated Francisella novicida transposon mutants protect mice wild- <br /> 43.   Rohmer, L., C. Fong, S. Abmayr, M. Wasnick, T. J. Larson Freeman, M.                type challenge. Infect. Immun. 74:5095&#8211;5105. <br />       Radey, T. Guina, K. Svensson, H. S. Hayden, M. Jacobs, L. A. Gallagher, C.      54. Turner, A. K., M. A. Lovell, S. D. Hulme, L. Zhang-Barber, P. A. <br />       Manoil, R. K. Ernst, B. Drees, D. Buckley, E. Haugen, D. Bovee, Y. Zhou,            Barrow. 1998. Identification <span id='am-125' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-126' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-135'  ></span><span id='am-136' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span>Salmonella</span> typhimurium genes required for <br />       J. Chang, R. Levy, R. Lim, W. Gillett, D. Guenthener, A. Kang, S. A. Shaffer,       colonization chicken alimentary tract virulence newly <br />       G. Taylor, J. Chen, B. Gallis, D. A. D&#8217;Argenio, M. Forsman, M. V. Olson,            hatched chicks. Infect. Immun. 66:2099&#8211;2106. <br />       D. R. Goodlett, R. Kaul, S. I. Miller, M. J. Brittnacher. 2007. Compar-     55. Twine, S. M., N. C. Mykytczuk, M. D. Petit, H. Shen, A. Sjostedt, J. W. <br />       ison Francisella tularensis genomes reveals evolutiona <br /> </body></html>