<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Systems Biology                                                                                                                      BioMed Central <br />  <br />  <br />  <br /> Methodology article                                                                                                                    Open Access <br /> Machine learning based analyses metabolic networks supports <br /> high-throughput knockout screens <br /> Kitiporn Plaimas1,2, Jan-Phillip Mallm1, Marcus Oswald3, Fabian Svara4, <br /> Victor Sourjik4, Roland Eils*1,2 Rainer K&#246;nig1,2 <br />  <br /> Address: 1Department Bioinformatics Functional Genomics, Institute Pharmacy Molecular Biotechnology, Bioquant, University of <br /> Heidelberg, Im Neuenheimer Feld 267, 69120 Heidelberg, Germany, 2Division Theoretical Bioinformatics, German <span id='am-46' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-47' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49'  ></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Cancer</span> Research Center <br /> (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany, 3Department Discrete Optimization, Interdisciplinary Center Scientific <br /> Computing, University Heidelberg, Im Neuenheimer Feld 368, 69120 Heidelberg, Germany 4Center Molecular Biology Heidelberg <br /> (ZMBH), Im Neuenheimer Feld 282, 69120 Heidelberg, Germany <br /> Email: Kitiporn Plaimas - k.plaimas@dkfz.de; Jan-Phillip Mallm - jpmallm@gmx.de; Marcus Oswald - Marcus.Oswald@Informatik.Uni- <br /> Heidelberg.de; Fabian Svara - fabian.svara@urz.uni-heidelberg.de; Victor Sourjik - v.sourjik@zmbh.uni-heidelberg.de; <br /> Roland Eils* - r.eils@dkfz.de; Rainer K&#246;nig - r.koenig@dkfz.de <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 24 July 2008                                                       Received: 29 April 2008 <br />                                                                               Accepted: 24 July 2008 <br /> BMC Systems Biology 2008, 2:67   doi:10.1186/1752-0509-2-67 <br /> <span id='am-59' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63'  ></span>This</span> article available from: http://www.biomedcentral.com/1752-0509/2/67 <br /> &#169; 2008 Plaimas et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  Abstract <br />                  Background: Computational identification new drug targets major goal pharmaceutical <br />                  bioinformatics. <br />                  Results: This paper presents machine learning strategy study validate essential enzymes <br />                  metabolic network. Each single enzyme characterized local network topology, gene <br />                  homologies co-expression, flux balance analyses. A machine learning trained <br />                  distinguish essential non-essential reactions. It validated comprehensive <br />                  experimental dataset, consists phenotypic outcomes single knockout mutants of <br />                  <span id='am-111' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-112' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-123'  ></span><span id='am-124' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-125' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span>Escherichia coli</span> (KEIO collection). We yielded reliable results high accuracy (93%) and <br />                  precision (90%). We topologic, genomic transcriptomic features describing the <br />                  network sufficient defining essentiality reaction. These features substantially <br />                  depend specific media conditions enabled apply approach specific <br />                  media conditions, like lysogeny broth rich medium. <br />                  Conclusion: Our analysis feasible validate experimental knockout <span id='am-158' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-159' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-160'  ></span><span id='am-161' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-162' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> high throughput <br />                  screens, used improve flux balance analyses supports experimental knockout screens <br />                  define drug targets. <br />  <br />  <br />  <br />  <br /> Background                                                                     silico helps identify enzymes essential the <br /> Defining drug targets drug design major                      survival organism [2,3]. A general model the <br /> goals biomedical research. In particular, metabolic                         metabolic network described graph theoreti- <br /> enzymes successfully targeted specific drugs                      cal approaches applied identify drug targets in <br /> to inhibit essential processes pathogenic organisms                      pathogenic organisms [4]. The term &apos;damage&apos; used to <br /> the human host [1]. Analyzing metabolic network                         assess enzymes serve drug targets their <br />  <br />  <br />                                                                                                                                        Page 1 11 <br />                                                                                                                  (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                              http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />  <br /> inhibition influences substantial number down-            targets. Supporting claim, 19 37 predictions for <br /> stream metabolic reactions products [5]. Further-          novel targets literature reported <br /> more, concepts choke points load points            experimental evidence. <br /> successfully applied estimate essentiality an <br /> enzyme [2,3]. Load points defined hot spots     Methods <br /> metabolic network (enzymes/metabolites) based           Network reconstruction <br /> ratio number k-shortest paths passing      The data metabolic network taken pre- <br /> metabolite/enzyme (in/out), number nearest          vious study reconstructed way (iAF1260, <br /> neighbor links (in/out) attached it. This ratio com-    Feist co-workers [8]). Basically, metabolic net- <br /> pared average load value network [2]. Choke      work represented undirected bipartite graph <br /> points uniquely consume produce certain metabolite,       consisting metabolites reactions alternating <br /> which make indispensable. For example,             nodes. This network taken flux balance analy- <br /> <span id='am-34' about='obo:NCBITaxon_5820' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5794 obo:NCBITaxon_131567 obo:NCBITaxon_422676 obo:NCBITaxon_2759 obo:NCBITaxon_5819 obo:NCBITaxon_33630'><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="Plasmodium" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Plasmodium</span> falciparum d-aminolevulinate dehydratase            ses. For analyses, unspecific compounds as <br /> (ALAD) considered choke point [3]        water, ATP, etc. discarded. <br /> proven experimentally serve valid antimalarial tar- <br /> get [6].                                                       The gold standard <br />                                                                In <span id='am-69' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-70' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> demonstrate efficiency approach we <br /> Flux balance analyses (FBA) widely used          used data KEIO collection [10] gold stand- <br /> established method assess essentiality genes         ard. The dataset consisted phenotypic outcomes <br /> [7,8]. However, FBA approaches need clear definitions       set knockout mutants single genes was <br /> nutrition availability biomass production spe-       used define classes &quot;essential&quot; &quot;non-essential&quot; <br /> cifically given environmental conditions (for good over-     reactions. Genes knocked in-frame <br /> view aspects e.g. [9]). High-throughput           replacement PCR product containing kanamycin <br /> experiments performed investigate essen-      resistance gene. The start-<span id='am-64' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-65' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67'  ></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> up-stream trans- <br /> tiality major portion genes organism [10-    lational signal replaced fully intact. After <br /> 12]. For <span id='am-127' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-128' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-132' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-133'  ></span>Escherichia</span> coli, essentiality virtually   kanamycin treatment, in-frame single gene deletions were <br /> open reading frames observed comprehensive            verified PCR loci specific primers. When were <br /> knockout screen (KEIO collection [10]). This data enables      unable create mutant formed colonies plate, <br /> to test performance silico metabolic model        mutated gene considered essential. Knock- <br /> that predicts essential genes. Analyzing flux balances         experiments performed LB rich medium and <br /> under aerobic glucose condition using COBRA tool-          glucose minimal medium, resulting datasets <br /> box [13] newly reconstructed metabolic network        (denoted rich medium glucose minimal medium, <br /> E. coli yielded 92% accuracy predicting essenti-      respectively). For rich medium, 4,288 tested <br /> ality genes [8]. Feist co-workers compared        genes, 303 genes mutants therefore <br /> predictions KEIO collection yielded 88%       defined essential. Genes considered to <br /> rich media conditions. In study, FBA cor-      essential rich medium condition con- <br /> responding experimental knockout screen performed          sidered essential glucose minimal medium con- <br /> to study opportunistic behavior pathogen Pseu-      dition. Additionally, genes, 119 genes were <br /> domonas aeruginosa systems view [14].                   assigned essential glucose minimal medium as <br />                                                                showed slow growth minimal media (growth <br /> In paper propose integrative machine learning       rate &#8804; 0.0926 24 hours). Experimental criteria gene <br /> approach applying broad list described tools. The     essentiality glucose minimal medium described in <br /> machine learning supplied qualitative          [8,12]. Genes mapped corresponding <br /> and quantitative descriptors derived biochemical          proteins, enzymes reactions using gene-protein- <br /> knowledge, genomic transcriptomic data, flux           reaction Table Feist et al. [8]. The reaction(s) associ- <br /> balance analyses. Using KEIO collection [10]        ated gene defined essential non- <br /> gold standard, yielded overall accuracy 93%       essential way activate reac- <br /> rich media conditions. Comparative analysis        tion(s) genes coding gene experi- <br /> flux balance approach machine learning                 mentally essential non-essential, respectively. <br /> approach yielded improvements FBA,          Otherwise discarded training test- <br /> consider aminyl-tRNA reactions modeling. Predictions        ing analysis. Furthermore, 133 reactions discarded <br /> that contradicted KEIO collection experimentally      analysis, corresponding genes couldn&apos;t be <br /> tested successfully used detect errors experi-   defined. Finally, 303 essential genes determined <br /> mental data. Predicted reactions matching experimen-       set 231 essential 1125 non-essential reactions <br /> tal screen strengthen candidacy potential drug        rich medium. Out 1125 non-essential reac- <br />  <br />  <br />                                                                                                                    Page 2 11 <br />                                                                                              (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                                               http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />  <br /> tions rich medium, 107 reactions defined                       Topology based features <br /> essential glucose minimal medium. In total, 1356                       We set breadth algorithm investigate net- <br /> reactions used experimental results (KEIO)                      work single reaction blocked. We defined a <br /> for essentiality taken <span id='am-51' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-52' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> labels reac-               reaction essential survival basically the <br /> tions (samples) training validating classifiers.                 mutated network yield products reac- <br /> Note that, didn&apos;t use experimental data fea-                 tion upstream substrates reaction. Hence, fea- <br /> tures reactions.                                                      tures defined knocked reaction <br />                                                                              substantial producing downstream metabolites <br /> Defining features                                                        products produced path- <br /> A list relevant features obtained differ-                  ways. The investigation tested knocked reac- <br /> ent aspects: network topology, genomics flux balance                     tion performed following algorithm. <br /> analyses. Table 1 gives overview features their <br /> abbreviations. <br /> Table 1: List features <br />  <br />  Short form Explanation <br />  <br />                 Topology features: local structures <br />  RUPa           Reachable/Unreachable Products (RUP): equal product produced blocking reaction <br />  PUP            Percentage Unreachable Products (PUP): percentage products produced blocking reaction <br />  NSa            Number Substrates (NS) <br />  NPa            Number Products (NP) <br />  NNRa           Number Neighbouring Reactions (NNR) <br />  NNNRa          Number Neighbours Neighbouring Reactions (NNNR) <br />  CCVa           Clustering Coefficient Value (CCV): clustering coefficient reaction <br />  <span id='am-168' about='obo:NCBITaxon_366224' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_150879 obo:NCBITaxon_33415 obo:NCBITaxon_42282 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-169'  ></span><span id='am-170' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-171' property="rdfs:label" content="Dira" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>DIRa</span>           Directionality reaction (DIR) <br />  <br />                 Topology features: deviations, choke points, load scores damage <br />  ND             Number Deviations (ND) <br />  APL            Average Path Length (APL): average path length deviations <br />  LSPa           Length Shortest Path (LSP): length shortest path deviations <br />  CP             Choke Point (CP): reaction choke point (Rahman et al, 2006) <br />  LSa            Load Score (LS): load score reaction (Rahman et al, 2006) <br />  NDRa           Number Damaged Reactions (NDR): number damaged reactions blocking reaction (Lemke et al, 2004) <br />  NDCa           Number Damaged Compounds (NDC): number damaged compounds blocking reaction (Lemke et al, 2004) <br />  NDRDa          Number Damaged Reactions having Deviations (NDRD): number damaged reactions alternative paths <br />                 reached blocking reaction <br />  NDCDa          Number Damaged Compounds having Deviations (NDCD): number damaged compounds alternative <br />                 paths reached blocking reaction <br />  NDCRa          Number Damaged Choke point Reactions (NDCR): number damaged choke point reactions blocking reaction <br />  NDCCa          Number Damaged Choke point Compounds (NDCC): number damaged choke point compounds blocking reaction <br />  NDCRDa         Number Damaged Choke point Reactions having Deviations (NDCRD): number damaged choke point reactions that <br />                 alternative paths reached blocking reaction <br />  NDCCDa         Number Damaged Choke point Compounds having Deviations (NDCCD): number damaged choke point compounds <br />                 alternative paths reached blocking reaction <br />  <br />                 Gene expression data, genomic data miscellaneous <br />  NCGa           Number Coding Genes (NCG): number coding genes reaction <br />  H10a           Homology 10-10 (H10): number homologous genes e-value cutoff 10-10 <br />  H7             Homology 10-7 (H7): number homologous genes e-value cutoff 10-7 <br />  H5a            Homology 10-5 (H5): number homologous genes e-value cutoff 10-5 <br />  H3a            Homology 10-3 (H3): number homologous genes e-value cutoff 10-3 <br />  NRSGa,b        Number Reactions Same Genes (NRSG): number reactions derived genes <br />  NRSEa,b        Number Reaction having Similar Expression <br />                 (NRSE): number reactions similar expression (correlation coefficient &gt;0.8) <br />  MCCa,b         Maximum Correlation Coefficients (MCC): maximum value correlation coefficients neighbouring reactions <br />  <br />                 Flux distribution <br />  BFVa           Biomass Flux Value (BFV): biomass flux value blocking reaction (under aerobic glucose condition) <br />  <br />  athe   optimized features, bfeatures gene expression <br />  <br />  <br />                                                                                                                                       Page 3 11 <br />                                                                                                                 (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                               http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />  <br /> i. All metabolites acting input nodes (substrates)       blocked. The organism pathways pro- <br /> output nodes (products) knocked reaction        duce products making robust. Thus, <br /> selected. The set substrates S defined input nodes       counted number possible alternative paths <br /> and set products P defined output nodes. To      yielding feature ND (ND, number deviations). <br /> a broader list available substrates integrated several <br /> other substrates S. We included substrates      Choke points, load points damage <br /> upstream reactions products downstream           A reaction uniquely consumes produces certain <br /> reactions sets S P, respectively. Substrates    metabolite metabolic network considered a <br /> reactions substrates S sub-   choke point. Such reaction shows high potential for <br /> strate included S. Further, substrates reactions    essentiality [2,3]. We checked observed reaction was <br /> that metabolite P substrate         choke point (CP, choke points). According con- <br /> included S.                                                cept load scores [2], computed load score of <br />                                                                 reaction average number pathways passing <br /> ii. Reactions selected used available com-      reaction, comparison number of <br /> pounds substrates.                                           pathways metabolites network. We used the <br />                                                                 <span id='am-157' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> damaged compounds/reactions reported by <br /> iii. These selected reactions products incor-    [5]. Basically, damage defined determining the <br /> porated network. These products set new        potentially effected metabolites reactions down- <br /> available metabolites network.                           stream knocked reaction. We applied def- <br />                                                                 inition calculating features NDR (NDR, number of <br /> iv. Steps ii iii repeated reactions   damaged reactions) NDC (number damaged com- <br /> could identified incorporation.                          pounds). In turn, damaged compounds/reactions <br />                                                                 produced alternative pathways. <br /> v. The output nodes produced counted         Therefore, calculated number damaged com- <br /> (reachable products P).                                         pounds/reactions did alternative way to <br />                                                                 reached substrates knocked reac- <br /> After finishing process, used number defined      tion (NDRD, number damaged reactions having no <br /> output nodes produced mutated          deviations; NDCD, number damaged compounds hav- <br /> network features, i.e. quality feature defining    ing deviations). In addition analysis dam- <br /> at product produced (RUP, reacha-        aged compounds/reactions, included number <br /> ble/unreachable products), percentage prod-          damaged choke points (NDCR, number damaged <br /> ucts produced (PUP, percentage             choke point reactions; NDCC, number damaged choke <br /> unreachable products).                                          point compounds; NDCRD, number damaged choke <br />                                                                 point reactions having deviations; NDCCD, number of <br /> We run breadth search network esti-     damaged choke point compounds having deviations). <br /> mate possible deviations. This time focused rele- <br /> vant pathways using similarity measure          Local topology features <br /> SIMCOMP software [15]. SIMCOMP used define               The number substrates products knocked out <br /> the relevant substrates products reac-         reaction counted (NS, number substrates, NP, <br /> tion. Starting S, breadth search explored    number products, respectively). Further, defined <br /> network finding direct products knocked      features number neighbouring reactions (NNR, <br /> reaction. When algorithm visited products,         number neighbouring reactions), number neigh- <br /> stored corresponding pathway continued              bours neighbouring reactions (NNNR, number of <br /> search alternative paths network      neighbours neighbouring reactions) clustering <br /> was entirely explored maximal path length 10 reac-      coefficient (CCV, clustering coefficient values) [16,17] of <br /> tions reached. We took average path length (APL,        knocked reaction. The reaction direction (DIR, <br /> average path length) shortest path length (LSP,         directionality reaction) taken model <br /> length shortest path) deviations features      Feist et al.[8]. <br /> the classifier. The deviation features used find <br /> alternative pathways produce products knocked         Gene expression data, genomic data miscellaneous <br /> out reaction substrates S. In metabolic network,     For case study, collected gene expression data from <br /> these substrates consumed reactions        study observing regulation oxygen depriva- <br /> yielding products etc. Therefore, kept track        tion [18]. This dataset taken unspe- <br /> alternative paths metabolic network poten-       cific regulation, i.e. small band broad <br /> tial organism survive reaction             range effected metabolic pathways. The gene expression <br />  <br />  <br />                                                                                                                    Page 4 11 <br />                                                                                              (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                                 http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />  <br /> data data-set mapped corresponding          diction value (number true negatives/number nega- <br /> reactions. For reaction catalysed complex        tively predicted reactions). <br /> of proteins, took mean gene expression val- <br /> ues corresponding genes (for details            Feature selection <br /> [19]). Genes pathway co-regula-           The feature selection top-down approach. <br /> tion [20]. Therefore, maximum correlation coefficient        We trained Support Vector Machines terms max- <br /> of neighboring reactions knocked reaction         imizing overall accuracy using features. Each single <br /> (MCC, maximum correlation coefficient) number            feature discarded data set perform- <br /> of reactions having similar gene expression (correlation         ance machine observed. Testing perform- <br /> coefficient &gt; 0.8) calculated (NRSE, number reac-        ance machine leave-one-out cross <br /> tions having similar expression). Together              validation. The accuracies machines missing one <br /> number reactions coming gene (NRSG,             feature compared best machine kept the <br /> number reactions genes), features             iteration. This repeated accuracy did not <br /> served machine estimating knocked reac-       increase. The machine best accuracy selected <br /> tion biosynthesis degradation pathway. We            best classifier features optimized fea- <br /> also included number homologous genes                ture set. <br /> might taken function knocked out <br /> gene. Homologous genes searched using Blast [21]            Experimental protocol knockout verification <br /> against open reading frames E. coli differ-     Knockout mutations verified PCR amplification <br /> ent e-value cutoffs, i.e. 10-3, 10-5, 10-7, 10-10 yielding   genomic loci expected contain 1327 base pair <br /> the features H3, H5, H7 H10, respectively. The               gene replacement cassette specific primers (Table S4 <br /> method flux balance simulations described           Additional file 1). Primers chosen equal <br /> Results discussion (section Comparing perform-           predicted melting temperatures ~60&#176;C hybridised <br /> ance performance flux balance analyses).               specific distances upstream downstream tar- <br />                                                                  gene. PCR reactions performed directly from <br /> Machine learning                                                 freshly grown bacterial colonies 30 cycles the <br /> We applied Support Vector Machines R package            annealing temperature 54&#176;C. The product sizes <br /> e1071 [22] classify essential non-essential       obtained KEIO collection strains compared <br /> reactions metabolic network. A radial basis func-         wild-type E. coli strain MG1655 1% <br /> tion used kernel function. Parameter optimiza-        agarose gels. <br /> tion performed regularization term that <br /> defined costs false classifications (5 steps each,   Assembling list drug targets <br /> range: 2n, n = -4, -2, 0, 2, 4). The range taken    To map enzymes drug targets, drugs corre- <br /> the kernel width &#947;. This optimization realized            sponding drug targets selected drug data- <br /> training grid search combinations       base Drugbank [23]. We took drugs account that <br /> parameters [22]. The sizes classes differed sig-      affected organism excepting humans mam- <br /> nificantly data set (essential: 17%, non-essential:       mals. Entries metabolites reaction <br /> 83%). For broad spectrum different precisions           KEGG database [24] discarded restrict our <br /> sensitivities, varied weight factor positive      drug list non-endogenous compounds. The targets&apos; <br /> instances data set optimized feature set       annotated EC numbers remaining drugs col- <br /> in range 0.1 5.0. We performed leave-one-out         lected validated drug targets. <br /> cross validation measure effectiveness the <br /> machine learning method. A single reaction selected          Results discussion <br /> as validation data predicted remaining         An overview machine learning procedure shown <br /> reactions training data. This repeated       Figure 1. The machine learning trained and <br /> reaction data set. For assessing performance       validated large set features. Firstly, local topology <br /> the classifiers, calculated standard measures accu-       based features used qualitatively possi- <br /> racy (number correctly predicted reactions/number          ble flux deviations. Secondly, choke load points were <br /> all predicted reactions), sensitivity (number true posi-      defined damage used qualitative <br /> tives/(number true positives + number false nega-          flux load stream effects knocked down <br /> tives)), specificity (number true negatives/(number        reaction. Thirdly, functional genomics data, co- <br /> true negatives + number false positives)), positive pre-      expression genes up- down-stream reactions <br /> diction value precision (number true positives/            used indicate conjoint reactions pathway. In <br /> number positively predicted reactions), negative pre-         addition features, considered existence of <br />                                                                  homologous genes corresponding knockout reac- <br />  <br />  <br />                                                                                                                       Page 5 11 <br />                                                                                                 (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                                     http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />  <br />  <br />                        Metabolic Network                           1) Topology based features <br />                                                                              - Local topology <br />                                                                              - Deviations <br />                                                                              - Choke load points <br />                                                                              - Damages <br />                                                                    2) Genomic transcriptomic data <br />                                                                                              ATCGAGTTGCAAT <br />                                                                                              ATAGAA &#8211; TGCA-T <br />                                                                                             - Gene expression <br />                                                                                             - Homologous genes <br />                          reaction                                 3) Flux balance analysis <br />                          metabolite <br />                                                                                        - Biomass objective flux <br />  <br />  <br />                                                                              Feature creation <br />                                                                                     Features <br />                                                                                            R <br />                                                                                                           Essentiality <br />                                                                              P UP S P NR NN V             classes <br />                                                                            RU P N N N N CC . . . <br />                                                                        1                                    E   E: Essential <br />                                                                                                             N   N: Non-essential <br />                                                                                                           N <br />                                                            Reactions <br />                                                                          ...   3 1    4   ...             E <br />                                                                                                             . <br />                                                                                                             . <br />                                                                        . <br />                                                                                                             . <br />                                                                        . <br />                                                                        . <br />  <br />                                                                                  Reaction profile <br />  <br />  <br />       The machine learning (SVM) <br />  <br />              Essential             &#966; <br />                o o                                           o                                    Trained classifier <br />                 o o x                                     oo <br />               o                                         o o o <br />                        x x                                x xx <br />               x ox                                                                             Prediction the <br />                         Non-essential                    x x <br />                                                                                             reaction essential <br />                  Input space                           Space kernel <br />  <br />  <br /> Figure 1 overview method <br /> Schematic <br /> Schematic overview method. For reaction, features calculated describing 1) local topology the <br /> metabolic network, 2) genomic transcriptomic relations, 3) biomass production using flux balance analyses when <br /> discarding reaction. E.g. given reaction educts, product 4 neighbors, feature NS (number sub- <br /> strates), NP (number products), NNR (number neighboring reactions) set 3, 1 4, respectively. A table <br /> for reaction profiles created used training classifier distinguish essential non-essential reac- <br /> tions. A Support Vector Machine (SVM) applied classifier optimal separating hyperplane. <br />  <br />  <br />  <br />  <br />                                                                                                                              Page 6 11 <br />                                                                                                        (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                                   http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />  <br /> tion, expected function           More positive instances recognized their <br /> the knocked gene. Finally, used flux balance anal-          weight factor increased. The highest specificity (99%) and <br /> yses estimate network&apos;s biomass production follow-          best precision (95%) yielded data <br /> ing knocking reaction. A list features given    point weight factor 0.1. This beneficial when <br /> Table 1.                                                           predicting drug targets high reliability. Alternatively, <br />                                                                    avoid overlooking potential targets, increased sensitiv- <br /> Performance machine learning algorithm                      ity achieved raising weight factor 1.0 <br /> Due small data set 1356 reactions training           (sensitivity = 75%). In following, analyses were <br /> and validation, performed leave-one-out cross vali-           performed weight factor one. <br /> dation measure effectiveness machine learn- <br /> ing system. Using KEIO-collection data rich                 Identifying drug targets <br /> medium reference, gained overall accuracy          We compared enzymes predictions the <br /> 92% features taken (Table 2). To increase            results KEIO collection comprehensive list of <br /> the performance, did systematic feature reduction             valid drug targets Drugbank database [23] (Fig- <br /> within top-down procedure. We yielded better result            ure 3). Surprisingly, 80% drug target enzymes were <br /> with optimized feature set 25 features (accuracy =           KEIO high-throughput screen by <br /> 93%, Table 2). These 25 features regarded            machine. This fact drug <br /> the dominating factors leading good performance.          targets broad range organisms having different <br /> To relevant,              topologies metabolic networks dif- <br /> started procedure, stopping          ferent alternative pathways corresponding drug tar- <br /> step. For features, compared accuracy          gets. It noted study focused on <br /> classifier lacking feature, respectively. It turned     reactions essential rich media conditions. We <br /> that loosing feature NNNR yielded worst classifi-          suggest 37 promising drug target enzymes in <br /> cation performance (accuracy -0.89 compared clas-           Drugbank database, validated inter- <br /> sifier features) hinting      section predictions results exper- <br /> most relevant feature. This feature followed NRSE           imental KEIO screen. A list reactions with <br /> (-0.82), BFV (-0.74), NNR (-0.52) H10 (-0.52). Inter-          references reported experimental evidences given in <br /> estingly, features span set     Table S1 [in Additional file 1]. <br /> of feature categories (NNNR NNR: network <br /> toplogy; NRSE: gene expression, genomics; H10: homol-              Comparing performance machine learning <br /> ogy, genomics; BFV: flux balance analysis).                    approach flux balance analyses <br />                                                                    We performed single reaction deletion network <br /> The sizes classes &quot;essential reactions&quot; &quot;non-       calculated flux values FBA using Cobra Tool- <br /> essential reactions&quot; differed significantly data set        box [13] assess essential reactions aerobic glu- <br /> (essential: 17%, non-essential: 83%). For obtaining differ-        cose minimal media conditions (as described the <br /> ent stringencies, weighted positive instances          supplementary material Feist et al. 2007). In anal- <br /> factor ranging 0.1 5.0 step size 0.1 (Fig-       ysis, reaction assessed essential respective <br /> ure 2). The sensitivity increased significantly smaller       prediction mutated network&apos;s maximal biomass <br /> to higher weights, reaching plateau weight factors        production &lt; 1% wildtype&apos;s biomass produc- <br /> 1.0 more. As expected, smaller weight classi-        tion. The biomass objective function used analysis <br /> fier tended overwhelmed large negative class.         taken explained [8]. Note simulating <br />  <br /> Table 2: Performance machine learning based predictions rich media condition <br />  <br />                                                                          Machine Learning                          Machine Learning <br />                                                                           (all 30 features)                   (25 optimized features) <br />  <br />  true positives                                                                         168                                        174 <br />  true negatives                                                                        1078                                       1092 <br />  false positives                                                                         47                                         33 <br />  false negatives                                                                         63                                         57 <br />  sensitivity (recall)                                                                72.73%                                     75.32% <br />  specificity                                                                         95.82%                                     97.07% <br />  positive predictive values (precision)                                              78.14%                                     84.06% <br />  negative predictive values                                                          94.48%                                     95.04% <br />  overall accuracy                                                                    91.89%                                     93.36% <br />  <br />  <br />  <br />  <br />                                                                                                                         Page 7 11 <br />                                                                                                   (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                                         http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />  <br />                                                                        medium according experimental criteria gene <br />                                               ROC Plot                 essentiality glucose minimal media KEIO col- <br />                     90%                                                lection [8,10,12]. 996 reactions identified non- <br />                     80%                                                essential. The remaining reactions associated gene, <br />                     70%                                                exchange reactions, clearly identi- <br />  sensitivity (%) <br />  <br />  <br />  <br />  <br />                     60%                                                fied. The FBA approach detected essentiality reac- <br />                                                                        tion aerobic glucose minimal condition an <br />                     50%                                                accuracy 86%, sensitivity 52% specificity of <br />                     40%                                                98%. We performed machine learning glucose <br />                     30%                                                minimal conditions BFV (Biomass flux <br />                     20%                                                value FBA simulation) BFV improve <br />                     10%                                                results (Table 3). With BFV, approach yielded 90% <br />                                                                        precision 79% recall experimental results, com- <br />                      0% <br />                         0% 1% 2% 3% 4% 5% 6% 7%                        pared FBA results 87% precision 51% recall. <br />                                                                        Results reactions given Table S2 [Additional <br />                                       100 &#8211; specificity (%)            file 2]. <br />  <br /> ROC-curve <br /> Figurefactors <br /> weight         positive <br />          2 showing    prediction <br />                             instances results different           Improving flux balance simulations <br /> ROC-curve showing prediction results dif-                     Figure 4 shows comparison method FBA by <br /> ferent weight factors positive instances. Each (blue)              categorizing results according KEGG pathways <br /> diamond shows result different weight. From left          [24]. The essential reactions our <br /> right, weight increased 0.1 5.0, step size <br />                                                                        machine learning approach FBA were <br /> of 0.1. When weight factors higher 1.0, the <br /> sensitivity remained constant. The dotted line manually            reactions amino acid metabolism lipid <br /> fitted.                                                                metabolism. In amino acid metabolism, tRNA trans- <br />                                                                        ferases amino acids essen- <br />                                                                        tial machine learning approach FBA. We <br /> rich media conditions challenging, difficult            improved FBA simulations adding corresponding <br /> characterize uptake rates compound rich              aminyl-tRNA reactions products biomass <br /> medium (Adam Feist, personal communication 2008).                      objective function. This ensured tRNA transferases <br /> Because this, compared performances                   predicted essential FBA method. The <br /> approach FBA glucose minimal media. 338                    simulations subsequently gained better results cor- <br /> reactions essential glucose minimal                rectly predicting essentiality aminyl-tRNA reac- <br />                                                                        tions. <br />  <br />                                                                        Using approach means validate the <br />                                             Machine Learning <br />                                                                        experimental knockout screen <br />                                  8                                     Predicting different outcome experimental high <br />                                                                        throughput screen (KEIO) error in <br />  <br />                             37         17                              Table 3: Comparison machine learning method Flux <br />                                  15                                    Balance Analyses glucose minimal media condition <br />                                                                  183 <br />                        22        16         125                         Performance                         ML\BFV1          ML2         FBA3 <br />  <br />                                                                         true positives                          192          266          174 <br />                                                   Drug targets          true negatives                          932          968          971 <br />                    KEIO (rich medium)                                   false positives                          64           28           25 <br />                                                                         false negatives                         146           72          164 <br />                                                                         sensitivity                          56.80%       78.70%       51.48% <br /> The <br /> Figureinvestigated <br />           3        enzymes                                              specificity                          93.57%       97.19%       97.49% <br /> The investigated enzymes. Three sets shown: essen-                  positive predictive values           75.00%       90.48%       87.44% <br /> tial enzymes machine learning approach (top                negative predictive values           86.46%       93.08%       85.55% <br /> circle), essential enzymes KEIO knockout                   overall accuracy                     84.26%       92.50%       85.83% <br /> screen [10] (bottom-left circle) enzymes valid <br />                                                                         1machine learning feature BFV (biomass flux value from <br /> drug targets taken database Drugbank [23] (bot- <br /> tom-right).                                                             FBA). <br />                                                                         2machine learning including feature BFV <br />                                                                         3Flux Balance Analysis <br />  <br />  <br />  <br />  <br />                                                                                                                                Page 8 11 <br />                                                                                                          (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                                       http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />  <br />                                                                         4 genes list, mutations were <br />                                                      FBA <br />                                                         Non essential   stated 100% correct Baba et al. Indeed, of <br />                                                         Essential <br />                                                      ML                 genes (aspC, epd, luxS, thiE) correct PCR <br />           Metabolism                                 Non essential <br />   Cofactors Vitamins                                Essential <br />                                                                         product corresponding knockout allele was <br />   Amino Acid Metabolism                              Experiment         observed. <br />                                                        Nonessential <br />    Nucleotide Metabolism                               Essential <br />                                                                         Conclusion <br />         Lipid Metabolism <br />                                                                         Defining drug targets challenging task. Many experi- <br />  Carbohydrate Metabolism                                                ments rely conditional essentiality screen genes to <br />       Not pathway                                                define associated enzymes possible drug targets. <br />                                                                         Machine learning methods help validate exper- <br />                  Others <br />                                                                         imental data. Our approach used experimental knock- <br />                            0   50 100 150 200 250 300 350               data E. coli KEIO [10]. The machine was <br />                                Number reactions                      trained data predicted quite accurately the <br />                                                                         experimental outcomes. Most methods based graphi- <br /> Figure <br /> Comparison <br /> the experimental <br />         4  data,  according <br />                      machine  learning <br />                                    different <br />                                        predictions, <br />                                                 pathways <br />                                                     FBA             cal networks aim finding weak points net- <br /> Comparison machine learning predictions,                         work. We set machine learning integrates <br /> FBA experimental data, according differ-                     features describing network topology functional <br /> ent pathways. For pathway KEGG [24], lowest                 genomics properties elaborated way. By we <br /> bars represent experimental result (KEIO), reactions are <br />                                                                         gained valuable insights. Firstly, the <br /> grouped non-essential (left, blue) essential (right, <br /> magenta). The mid bars prediction results              topologic, genomic transcriptomic data describing <br /> of machine learning approach flux balance analyses,             network attributes sufficient defining the <br /> respectively. Larger differences machine learn-             essentiality certain reaction. For pathogens often <br /> ing prediction FBA amino acid metabo-               hard define environmental parameters are <br /> lism lipid metabolism.                                              complex changeable e.g. intestinal infections. <br />                                                                         Our approach can, principle, handle media condi- <br />                                                                         tions, shown rich media conditions study. <br /> our algorithm, error experimental knock-               Rich media conditions better reflect situation of <br /> out screen. We examined lists false positives                pathogens host (like e.g. gut), compar- <br /> false negatives experimental set-ups. Our list                ison minimal media conditions clearly defined <br /> false negatives contained 71 genes algorithm                  carbon sources flux balance analyses well <br /> predicted non-essential glucose minimal con-                adapted. A second benefit study experimental <br /> dition contradiction outcome KEIO exper-               validation support estimations potential drug <br /> iment [10]. For 33 obtained corresponding                    targets. When regarding intersection results and <br /> knockout clones KEIO library (growing rich                  KEIO collection, 37 potential targets for <br /> media), grew M9 glucose medium. Indeed,                     novel drugs, 19 some <br /> we able grow 9 33 clones good growth                reported experimental evidence literature. An <br /> rates (OD600 &#8805; 0.2 48 hours) 3 clones rea-               advantage machine learning approaches easily <br /> sonable growth rates (OD600 0.07 0.2                  change stringency parameter, e.g. increasing preci- <br /> 48 hours). The complete list given Table S3 [in Addi-             sion avoid loosing potential candidates, weight fac- <br /> tional file 1]. In turn, tested list false posi-         <span id='am-139' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-140' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-141'  ></span><span id='am-142' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>tor</span> positive instances increased. We used <br /> tives, algorithm predicted 33 genes                 gene expression data E. coli wild-type single <br /> essential, contrast experimental high throughput              knock strains. The single knock outs regulators <br /> screen. We assumed genes weren&apos;t                     respiration effecting large number genes also <br /> knocked correctly. Baba et al. (2006) provided valid-             treatment unspecific (growth oxygen rich <br /> ity estimation clones. We compared results             deprived conditions). Hence, large portion net- <br /> their estimations selected 6 genes, mutants               work pathways metabolic network differen- <br /> they estimated equal 37.5% correct.               tially expressed [25]. Within presented approach, data <br /> For 5 6 genes (alaS, coaA, coaE, glyS                  pathway unspecific examinations suited let <br /> hemE) PCR specific primer pairs (Table S4 Addi-                 classifier learn neighboring enzymes jointly <br /> tional file 1) yielded products sizes correspond-              work together. Therefore, multiple gene co-expres- <br /> ing wild-type knockout alleles, respectively. This               sion datasets variety conditions suit for <br /> indicated genes correctly knocked                 approach. However, needs investigated <br /> and wild-type gene present. No PCR product                gene expression data suits best optimize per- <br /> was observed ileS knockout. Additionally tested              formance. <br />  <br />  <br />                                                                                                                             Page 9 11 <br />                                                                                                       (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                                               http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />  <br /> We presented broadly applied                    SYS consortium Viroquant (# 0313923), Deutscher Akademischer Aus- <br /> to systems seeking potential drug targets variety                  landsdienst, Helmholtz Alliance Systems Biology Cancer the <br /> substantial bacterial infections organisms. For                   Commission Higher Education (CHE) Thailand. <br /> E. coli benefited rich data pool including well <br /> elaborated metabolic network, genome wide knock out <br />                                                                             References <br />                                                                             1.    Hopkins AL, Groom CR: The druggable genome. Nature reviews <br /> viability screen, genome sequence feasible gene                         2002, 1(9):727-730. <br /> expression dataset. Nowadays, genomic sequence                      2.    Rahman SA, Schomburg D: Observing local global proper- <br />                                                                                   ties metabolic pathways: &apos;load points&apos; &apos;choke points&apos; in <br /> not limiting factor applications                             metabolic networks. Bioinformatics 2006, 22(14):1767-1774. <br /> remarkable number genomes sequenced                          3.    Yeh I, Hanekamp T, Tsoka S, Karp PD, Altman RB: Computational <br /> will sequenced future. As approach uses                        analysis <span id='am-39' about='obo:NCBITaxon_5833' typeof='obo:NCBITaxon_5820 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5794 obo:NCBITaxon_131567 obo:NCBITaxon_418107 obo:NCBITaxon_422676 obo:NCBITaxon_2759 obo:NCBITaxon_5819 obo:NCBITaxon_33630'><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="rdfs:label" content="Plasmodium falciparum" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="malaria parasite P. falciparum" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Plasmodium (Laverania) falciparum" datatype="xsd:string"></span>Plasmodium falciparum</span> metabolism: organizing <br />                                                                                   genomic information facilitate drug discovery. Genome Res <br /> unspecific gene expression data obtained                         2004, 14(5):917-924. <br /> from publically available resources obtained                   4.    Schuster S, Fell DA, Dandekar T: A general definition <span id='am-134' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-135' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-138'  ></span>meta</span>- <br /> straightforward experiments. Very elaborated meta-                           bolic pathways useful systematic organization analy- <br />                                                                                   sis complex metabolic networks. Nat Biotechnol 2000, <br /> bolic networks assembled organisms                             18(3):326-332. <br /> (e.g. B. subtilis [11], H. pylori [26], M. barkeri [27], M.                 5.    Lemke N, Heredia F, Barcellos CK, Dos Reis AN, Mombach JC: <br />                                                                                   Essentiality damage metabolic networks. Bioinformatics <br /> tuberculosis [28], S. cerevisiae [29]) expect                    2004, 20(1):115-119. <br /> our method transferred major difficulties.                   6.    Bonday ZQ, Dhanasekaran S, Rangarajan PN, Padmanaban G: Import <br /> Further networks received large                            host <span id='am-144' about='obo:NCBITaxon_998453' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_50638 obo:NCBITaxon_7438 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7434 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_34725 obo:NCBITaxon_50557'><span id='am-145' property="rdfs:label" content="Delta" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-148'  ></span>delta</span>-aminolevulinate dehydratase malarial <br />                                                                                   parasite: identification new drug target. Nat Med 2000, <br /> organisms existing excellent databases like BioCyc                           6(8):898-903. <br /> [30] Kegg [24]. It challenging exploit                 7.    Edwards JS, Palsson BO: Metabolic flux balance analysis the <br />                                                                                   silico analysis <span id='am-163' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-164' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-165' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span>Escherichia coli K-12</span> gene deletions. BMC <br /> networks method. Finally, now,                             Bioinformatics 2000, 1:1. <br /> approach genome wide essentiality screen sub-                  8.    Feist AM, Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD, <br /> stantial laborious. A methodological challenging                         Broadbelt LJ, Hatzimanikatis V, Palsson BO: A genome-scale met- <br />                                                                                   abolic reconstruction Escherichia coli K-12 MG1655 that <br /> task remains employ approach different                              accounts 1260 ORFs thermodynamic information. <br /> organisms, e.g. using essentiality screen                           Molecular systems biology 2007, 3:121. <br /> organism infer information another.                               9.    Schuetz R, Kuepfer L, Sauer U: Systematic evaluation objec- <br />                                                                                   tive functions predicting intracellular fluxes Escherichia <br />                                                                                   coli. Molecular systems biology 2007, 3:119. <br /> Authors&apos; contributions                                                      10.   Baba T, <span id='am-54' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-55' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58'  ></span>Ara</span> T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko <br />                                                                                   KA, Tomita M, Wanner BL, Mori H: Construction Escherichia <br /> KP, J&#8211;PM, RE RK general concept                                coli K-12 in-frame, single-gene knockout mutants: Keio <br /> design study. KP carried data analysis. MO                         collection. Molecular systems biology 2006, 2:2006 0008. <br /> and KP conceptualized implemented breath                      11.   Oh YK, Palsson BO, Park SM, Schilling CH, Mahadevan R: Genome- <br />                                                                                   scale reconstruction metabolic network <span id='am-1' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-2' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span>Bacillus subtilis</span> <br /> search algorithm. FS VS validated model predic-                           based high-throughput phenotyping gene essentiality <br /> tions experimentally lab. KP RK drafted                            data. J Biol Chem 2007, 282(39):28791-28799. <br />                                                                             12.   Joyce AR, Reed JL, White A, Edwards R, Osterman A, Baba T, Mori <br /> manuscript. All authors read approved final man-                          H, Lesely SA, Palsson BO, Agarwalla S: Experimental compu- <br /> uscript.                                                                          tational assessment conditionally essential genes in <br />                                                                                   Escherichia coli. J Bacteriol 2006, 188(23):8259-8271. <br />                                                                             13.   Becker SA, Feist AM, Mo ML, Hannum G, Palsson BO, Herrgard MJ: <br /> Additional material                                                               Quantitative prediction cellular metabolism con- <br />                                                                                   straint-based models: COBRA Toolbox. Nature protocols <br />                                                                                   2007, 2(3):727-738. <br />  Additional File 1                                                          14.   Oberhardt MA, Puchalka J, Fryer KE, Dos Santos VA, Papin JA: <br />                                                                                   Genome-scale metabolic network analysis opportun- <br />  Supplementary tables. Tables S1, S3 S4. <br />                                                                                   istic pathogen <span id='am-72' about='obo:NCBITaxon_286' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621'><span id='am-73' property="oboInOwl:hasDbXref" content="PMID:7727274" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="PMID:23918787" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="PMID:18048745" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Pseudomonas" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="RNA similarity group I" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="PMID:15950132" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="PMID:10758879" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="PMID:10939664" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Liquidomonas" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="PMID:9103607" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasRelatedSynonym" content="Peudomonas" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasRelatedSynonym" content="Loefflerella" datatype="xsd:string"></span>Pseudomonas</span> aeruginosa PAO1. J Bacteriol 2008. <br />  Click file                                                        15.   Hattori M, Okuno Y, Goto S, Kanehisa M: Development a <br />  [http://www.biomedcentral.com/content/supplementary/1752-                        chemical structure comparison method integrated anal- <br />  0509-2-67-S1.pdf]                                                                ysis chemical genomic information metabolic <br />                                      <br /> </body></html>