<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Concurrent Growth Rate Transcript Analyses Reveal <br /> Essential Gene Stringency <span id='am-118' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-128' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-132'  ></span><span id='am-133' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span>Escherichia coli</span> <br /> Shan Goh1,2, Jaroslaw M. Boberek2, Nobutaka Nakashima3, Jem Stach4, Liam Good1,2* <br /> 1 Department Cell Molecular Biology, Karolinska Institute, Stockholm, Sweden, 2 Department Pathology Infectious Diseases, Royal Veterinary College, <br /> University London, London, United <span id='am-152' about='obo:NCBITaxon_kingdom' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-153' property="rdfs:label" content="kingdom" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Kingdom</span>, 3 Research Institute Genome-based biofactory, Toyohira-Ku, Sapporo, Japan, 4 School Biology, University of <br /> Newcastle, Newcastle Tyne, United Kingdom <br />  <br />  <br />  <br />      Abstract <br />      Background: Genes essential bacterial growth particular scientific interest. Many putative essential genes have <br />      identified predicted <span id='am-102' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-103' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-104' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>, however, little known gene expression requirement stringency, <br />      important aspect bacterial physiology likely determining factor drug target development. <br />  <br />      Methodology/Principal Findings: Working premise essential genes differ absolute requirement growth, <br />      silencing putative essential genes E. coli obtain titration declining growth rates transcript levels <br />      using antisense peptide nucleic acids (PNA) expressed antisense RNA. The relationship mRNA decline and <br />      growth rate decline reflects degree essentiality, stringency, essential gene, defined the <br />      minimum transcript level 50% reduction growth rate (MTL50). When applied growth essential genes, both <br />      RNA silencing methods resulted MTL50 values reveal acpP stringently required genes <br />      examined, ftsZ stringently required. The established antibacterial targets murA fabI less <br />      stringently required. <br />  <br />      Conclusions: RNA silencing reveal stringent requirements gene expression respect growth. <span id='am-87' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-88'  ></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> method may <br />      used validate existing essential genes quantify drug target requirement. <br />  <br />   Citation: Goh S, Boberek JM, Nakashima N, Stach J, Good L (2009) Concurrent Growth Rate Transcript Analyses Reveal Essential Gene Stringency in <br />   <span id='am-134' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-135'  ></span><span id='am-136' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span>Escherichia</span> coli. PLoS ONE 4(6): e6061. doi:10.1371/journal.pone.0006061 <br />   Editor: Laurent Re&#180;<span id='am-147' about='obo:NCBITaxon_155002' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_5338 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_155000 obo:NCBITaxon_2759 obo:NCBITaxon_155619'><span id='am-148' property="rdfs:label" content="Nia" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-151'  ></span>nia</span>, BMSI-A*STAR, Singapore <br />   Received November 3, 2008; Accepted June 2, 2009; Published June 26, 2009 <br />   Copyright: &#223; 2009 Goh et al. This open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: The Swedish Institute Guest Scholarship SG, Bloomsbury Colleges scholarship JMB, Swedish Research Council, University London <br />   Central Research Fund, Sasakawa Foundation. The funders role study design, <span id='am-155' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-156'  ></span><span id='am-157' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> collection analysis, decision publish, preparation of <br />   manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: lgood@rvc.ac.uk <br />  <br />  <br />  <br /> Introduction                                                                             We hypothesize growth essential genes differ require- <br />                                                                                       ment stringencies, differences revealed by <br />    Progress antimicrobial discovery slow recent                    measuring relationships specific mRNA decreases and <br /> decades [1] despite large-scale efforts identify genes essential               bacterial growth rate decline. The aim study measure <br /> growth conserved pathways Escherichia coli [2] <span id='am-1' about='obo:NCBITaxon_1279' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1385 obo:NCBITaxon_90964 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-2' property="oboInOwl:hasDbXref" content="PMID:10319495" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="PMID:9734063" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Aurococcus" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="PMID:10425778" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="PMID:12656157" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasDbXref" content="PMID:10319469" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Staphylococcus" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="PMID:17220435" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="PMID:10758876" datatype="xsd:string"></span>Staphylococcus</span>               growth stringency requirement E. coli genes using antisense <br /> aureus [3]. The drug discovery process involves essential gene                        gene silencing, evaluate degree essentiality cell <br /> identification various methods chromosomal                            viability. Two antisense gene control strategies bacteria are <br /> deletions [2] experimentally reduced genomes [4], followed                        used - synthetic antisense peptide nucleic acids (PNAs) plasmid <br /> by studies gene product interactions tandem affinity                       expression antisense RNA sequences. Both methods capable <br /> purification [5] and/or mathematical models [6]. Potential drug                       modulating essential genes situ E. coli [11,12] S. aureus <br /> targets subjected high-throughput screening                           [13,14], certain antisense agents inhibit Mycobacteria <br /> inhibitors, fail cellular level &#8216;&#8216;hit-to-lead&#8217;&#8217;                 [15,16]. Therefore, RNA silencing using specific antisense <br /> stages development factors [7], gene                        sequences provides approach measure gene requirement <br /> function redundancy [8]. Also, certain essential genes suspect                 stringency. <br /> that low degree gene expression requirement                        Four genes, acpP, fabI, ftsZ murA selected study <br /> bacterial viability. Despite evidence differential requirement                 following reasons: (a) gene shown <br /> growth essential genes [9], initial efforts scoring essentiality               experimentally essential growth E. coli <br /> [2,10], quantitative method determine                     [2,4,17,18,19,20], (b) studied antimicrobial <br /> expression requirement essential genes stringently                   targets [21,22,23], (c) gene products different cellular <br /> required targets considered drug development. This                    functions (d) gene silenced potentially without <br /> study focuses measuring differences suspected essential                    downstream effects [24,25]. Interactions FabI its <br /> genes terms degree requirement cell viability,                     specific inhibitor, triclosan, understood [26]. The fabI <br /> referred stringency, E. coli.                                           gene encodes enoyl ACP reductase, catalyzes fatty acid <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                 1                                    June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                               E. coli Gene Stringency <br />  <br />  <br /> elongation [17]. Another well-studied drug target UDP-N-                    antisense expressed <span id='am-165' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-166' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-167' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-169' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span>, constructed four <br /> acetylglucosamine enolpyruvyl transferase (encoded murA),               DH5a strains containing pBAD-acpP+pHNA, pBAD- <br /> which catalyzes synthesis peptidoglycan N-acetylglu-               fabI+pHN682, pBAD-ftsZ+pHNZ pBAD-murA+pHNM. Each <br /> cosamine acid phosphoenoylpyruvate [10,18,27]. MurA                     strain grown presence IPTG induction of <br /> specifically inhibited phosphomycin [28], mechanism                 expressed antisense, addition L- <br /> of inhibition understood [29]. The protein function FtsZ            arabinose induce target gene over-expression. Without L- <br /> has studied detail, FtsZ focus assay           arabinose, growth strains inhibited, L-arabinose <br /> development discover effective inhibitors [30]. The ftsZ gene               addition complemented gene silencing allowed strains to <br /> encodes tubulin-like protein polymerizes form Z-ring            grow presence expressed antisense (Figure 1). The results <br /> a scaffold cell division [31]. ACP acyl carrier protein              rescue experiments demonstrate silencers <br /> encoded acpP. ACP interesting drug target because,                    display gene selective growth inhibition. <br /> although shown central bacterial fatty acid <br /> biosynthesis [32,33], specific protein inhibitors            Titration essential gene expression effect of <br /> discovered. Nevertheless, acpP shown effective                    growth rate <br /> antisense target treatment bacterial infections vivo [34,35]            To test hypothesis essential genes differentially <br /> and gene product inhibited [36]. <br />                                                                                required maintain similar level growth, titrated down <br />    In principle, extent partial mRNA inhibition limits            growth rate E. coli using antisense PNA expressed <br /> growth reflect stringently required targeted mRNA               antisense determined mRNA levels targeted essential <br /> is bacterial viability. The specificities antisense PNA             gene. To enable parallel analyses multiple genes study, <br /> expressed antisense RNA silencing selected target genes <br />                                                                                used RT-PCR quantify mRNA levels assume a <br /> were assessed conducting rescue experiments, over- <br />                                                                                proxy protein measure bacteria [39]. For PNA-mediated <br /> expression target genes different strains comple- <br />                                                                                antisense effects, used hyper-permeable E. coli strain AS19 <br /> mented effects cognate silencers. To titrate down <br />                                                                                obtain efficient uptake [40]. Six doses antisense PNA <br /> essential gene expression obtain measured responses E. coli, <br />                                                                                selected <span id='am-105' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-106' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> achieve titration E. coli growth rate <br /> growth, used gene specific antisense PNA IPTG-induced <br />                                                                                reduction. Hence, dose range antisense PNA was <br /> expression antisense sequences. This concept enables quantita- <br />                                                                                unique, resulting growth inhibition dose dependent manner <br /> tive analyses mRNA silencing growth reduction, are <br />                                                                                (Figure 2). For expressed antisense, plasmids transformed <br /> not possible using knockout approaches. We the <br />                                                                                TOP10 E. coli cells (Table 2) antisense RNA expression <br /> essential genes differed mRNA inhibition growth inhibition <br />                                                                                induced using IPTG concentrations provided a <br /> profiles, demonstrating relationship transcript <br />                                                                                titration decreasing growth rates (not shown). <br /> levels growth rate reflects requirement stringencies of <br />                                                                                   Both antisense PNA expressed antisense specifically <br /> essential genes. <br />                                                                                silenced mRNA essential genes acpP, fabI, ftsZ murA, as <br />                                                                                determined analyses transcript levels appropriate <br /> Results                                                                        range inhibitory lethal doses (Figure 3). Decreases in <br /> Specificity antisense RNA PNA expressed                                 essential gene transcripts corresponded decreases bacterial <br />                                                                                growth rates (PNA data: R2 acpP, fabI, ftsZ, murA = 0.87, 0.63, <br /> antisense RNA <br />                                                                                0.89, 0.45), response non-linear. It be <br />    To silence selected target genes using different <br />                                                                                noted growth profiles observed ftsZ-inhibited cultures, <br /> silencing methods, needed design, test validate <br />                                                                                displayed rise fall culture turbidity, were <br /> several new silencers. Fortunately, able use previously <br />                                                                                consistent cell-division effect observed previously, where <br /> developed silencers acpP fabI, used similar design <br />                                                                                cells elongate dividing [41,42]. Phenotypic effects ftsZ- <br /> guidelines additional silencers needed study. We <br />                                                                                inhibited cultures confirmed microscopy, revealing <br /> found silencers able inhibit E. coli growth. Also, <br />                                                                                extremely elongated cells compared untreated cells cells <br /> both PNA expressed antisense RNA silencers displayed gene <br />                                                                                treated PNAs. Cell phenotypes antisense-expressing <br /> and sequence selective effects. Nevertheless, ensure all <br />                                                                                cells similar antisense PNA-treated cells (Figure 4). <br /> silence mainly target gene, carried set rescue <br /> experiments RNA silencers used conditions <br /> that inhibited growth tested growth                    Essential gene stringencies minimum level transcript <br /> rescued over-expression target gene open reading frame               50 measure <br /> (ORF) plasmid. This transcomplementation strategy                          The relationship decrease mRNA growth <br /> provides strict control specificity relative controls             rate determined silencing methods indicates the <br /> that involve sense, scrambled irrelevant sequences,              requirement essential genes unequal, some <br /> takes account transcript sequences present bacterial           genes stringently required others. Indeed, expressing <br /> cell effector sequences remain unchanged. We designed                  relative growth rate function relative mRNA level (Figure 5) <br /> antisense PNA specific acpP (Ec108), fabI (Ec107), ftsZ (Ec326),           showed minimum level mRNA needed 50% cell <br /> and murA (Ec330) using optimal parameters gene silencing                   viability order acpP .ftsZ.murA.fabI antisense PNA. <br /> [37,38] (Table 1). To test PNA specificity, constructed                That is, large number acpP transcripts (approximately 0.65 <br /> DH5a strains containing pBAD-acpP, pBAD-fabI, pBAD-                     relative value 65% normal cell) required maintain 50% <br /> ftsZ pBAD-murA. Each strain grown presence                    cell viability, compared ftsZ (38%), murA (30%) fabI (20%) <br /> antisense PNA specific target gene cloned pBAD,                transcripts. In words, small decrease acpP mRNA led to <br /> either addition L-arabinose target gene             large decrease bacterial growth rates, decreases ftsZ <br /> over-expression. Without L-arabinose, strains did grow                 mRNA proportional effects growth rates large <br /> upon induction L-arabinose, resistant growth                    decreases fabI murA mRNA required obtain a <br /> inhibitory effects PNA (Figure 1). To test specificity               small decrease growth rates. Gene stringency order <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          2                              June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                            E. coli Gene Stringency <br />  <br />  <br />  <br />  <br /> Figure 1. Transcomplementation antisense PNA expressed antisense RNA effects growth target gene over-expression. <br /> Essential gene over-expression induced L-arabinose indicated concentrations. PNAs added final concentration 4 mM. <br /> Concentrations IPTG used induction acpP-, fabI-, ftsZ- murA-antisense expression 200 mM, 1 mM, 100 mM 75 mM, respectively. <br /> doi:10.1371/journal.pone.0006061.g001 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                      3                               June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                                E. coli Gene Stringency <br />  <br />  <br />  <br />  Table 1. Properties PNA used study. <br />  <br />  <br />  PNA         Sequence                                       Target         Target location* length                 Reference/source <br />  <br />  Ec107       (KFF)3K-eg1-cccatagctt                         fabI           25 +5 (10 nt.)                           [11] <br />  Ec108       (KFF)3K-eg1-ctcatactct                         acpP           25 +5 (10 nt.)                           [57] <br />  Ec326       (KFF)3K-eg1-tcaaacatag                         ftsZ           22 +8 (10 nt.)                           This study <br />  Ec330       (KFF)3K-eg-1-tttagtttgt                        murA           29 +1 (10 nt.)                           This study <br />  <br />  * <br />   Antisense target locations indicated relative start <span id='am-92' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-95'  ></span><span id='am-96' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span>. <br />  doi:10.1371/journal.pone.0006061.t001 <br />  <br />  <br /> acpP.ftsZ.fabI.murA expressed antisense (Figure 5),                   dsRNA-mediated cleavage target site. Ideally, protein <br /> differs PNA data murA stringently                quantification carried parallel RT-PCR, <br /> required essential gene growth fabI.                           however, biochemical assays confirm antisense inhibition of <br />    Based data, propose minimum transcript level                acpP, fabI, ftsZ murA carried technical <br /> 50 (MTL50) measure stringent requirement, somewhat                     difficulties involved harvesting sufficient culture biomass and <br /> similar IC50 measure enzyme inhibition. MTL50 values                 quantitative protein assays lacking genes interest. <br /> directly reflect relationship level particular            Current methods limited use antibodies [33,43] or <br /> mRNA growth rate decline, indicating stringently                  intracellular peptide expression [44]. Nevertheless, correlation <br /> required essential gene growth. By plotting values            mRNA protein levels bacteria been <br /> relative growth rate relative mRNA level gene              demonstrated genes [25,39,45,46,47], suggesting that <br /> the graph, able estimate value relative                  MTL values reported provide useful measure gene <br /> mRNA (MTL50) 50% growth rate (Figure 5). We observed                   requirement. <br /> a shift right indicates greater stringency minimum                  Four genes compared study acpP does <br /> level transcript needed sustain 50% cell viability increases.            encode enzyme. However, finding acpP being <br /> Hence, high MTL50 value indicates high gene stringency. This              critically required ftsZ, fabI murA entirely <br /> because value indicates minimum required level                      surprising. ACP known interact large number of <br /> transcript viability, minimum inhibition level                  proteins fatty acid biosynthesis pathway [5] reduction <br /> transcript, differs IC50 respect. Compar-               levels active ACP results cell toxicity [33,36]. In <br /> isons MTL50 values, determined different gene-               addition, ACP interacts proteins involved lipid <br /> silencing methods, shown Figure 6. In general, MTL50                    synthesis [48]. Therefore, ACP needed large quantities and <br /> values higher expressed antisense cultures compared             slight decrease ACP levels amplified its <br /> antisense PNA treated cultures. Both methods               interacting partners, leading widespread physiological effects. <br /> stringently required essential gene acpP, followed ftsZ.                Indeed, acpP transcripts protein products abundant E. <br /> Antisense PNA silencing showed fabI stringent,                 coli [49,50]. FtsZ recently GTPase activity <br /> whereas expressed antisense silencing murA significantly             [51], however, interacts proteins a <br /> less stringently required, relative essential genes          component complex divisome [52]. From point of <br /> examined (Figures 5 6).                                                    view gene product regulation, FtsZ concentrations dictate the <br />                                                                                initiation cell division interaction FtsA FtsQ <br /> Discussion                                                                     [53]. In contrast, FabI MurA interact fewer proteins <br />                                                                                [5] possibly catalyze specific reactions. Also, FabI is <br />    Cell growth mRNA profiles following expressed antisense                 catalytic end point fatty acid elongation subject to <br /> RNA silencing differed somewhat following antisense                 feedback inhibition acyl-ACP fatty acid elongation <br /> PNA silencing, possibly reflecting different mechanisms                     cycle [54], MurA catalytic starting point in <br /> silencing. Nevertheless, data reject null hypothesis equal          peptidoglycan assembly [27] regulated through <br /> stringent requirement growth essential genes support               feedback inhibition UDP-N-acetylmuramic acid, down- <br /> premise essential genes differ level requirement          stream product pathway [55]. Hence, argued <br /> growth. Indeed, RNA silencing methods indicate significant                changes ACP FtsZ levels tolerated than <br /> differences gene stringency reveal acpP                     changes FabI MurA levels. <br /> stringently required essential gene, followed ftsZ,                 Specific control single gene expression bacterial <br /> four studied. As MTL50 value gene                     operon remains challenge polycistronic nature of <br /> calculated relative appropriate unsilenced controls, differences            transcripts. Indeed, antisense PNAs targeting genes lac <br /> in basal expression different essential genes affect       operon polar effects transcript stability translation <br /> stringency ranking.                                                            [39]. To avoid problem, selected genes likely <br />    The stringency ranking murA fabI differed PNA-               affected: fabI murA predicted RegulonDB <br /> treated cells antisense-expressing cells, probably              (regulondb.ccg.unam.mx) EcoCyc (http://ecocyc.org/) be <br /> differences PNA RNA chemistry, size differences                    individually transcribed, ftsZ, 39 end operon <br /> between types silencers subtle differences               [24], acpP, transcribed fabF [25], <br /> mechanisms gene silencing. For example, antisense PNAs act                  non-essential gene involved thermal regulation fatty acid <br /> directly steric hindrance translation initiation            biosynthesis [56]. We note potency acpP-specific <br /> indirectly trigger decay repressed transcripts, antisense           PNA inhibition fabF expression well, although <br /> RNAs act steric hindrance direct activation              unlikely, reported tandem mutation <br />  <br />  <br />          PLoS ONE | www.plosone.org                                        4                              June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                                 E. coli Gene Stringency <br />  <br />  <br />  <br />  <br /> Figure 2. Dose dependent growth inhibition antisense PNA-treated AS19 cells. Overnight AS19 cultures sub-cultured fresh media <br /> containing different PNA doses gene, growth monitored turbidity. Six doses chosen PNA, obtain titration in <br /> growth inhibition. Growth profiles terminated cultures harvested DOD550,0.1 untreated culture, transcript analyses. <br /> doi:10.1371/journal.pone.0006061.g002 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         5                                June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                                              E. coli Gene Stringency <br />  <br />  <br />  <br />  Table 2. Plasmids used study. <br />  <br />  <br />                                                                                                              Antisense target location*             Reference/ <br />  Plasmid           Relevant characteristic                         Purpose                                   length                             source <br />  <br />  pBAD/HisA         Arabinose inducible promoter (PBAD), AmpR       Inducible expression essential genes   n.a.                                   Invitrogen <br />  pBAD-fabI         fabI ORF                                        Inducible over-expression fabI         n.a                                    This study <br />  pBAD-acpP         acpP ORF                                        Inducible over-expression acpP         n.a.                                   This study <br />  pBAD-ftsZ         ftsZ ORF                                        Inducible over-expression ftsZ         n.a                                    This study <br />  pBAD-murA         murA ORF                                        Inducible over-expression murA         n.a.                                   This study <br />  pHN678            IPTG inducible promoter (Ptrc),                 Stabilized antisense expression vector    n.a.                                   [12] <br />                    paired-termini flanking MCS, CamR <br />  pHN682            fabI antisense insert                           Inducible expression fabI antisense    274 +86 fabI (160 nt.)           [12] <br />  pHNA              acpP antisense insert                           Inducible expression acpP antisense    242 +85 acpP (127 nt.)           This study <br />  pHNZ              ftsZ antisense insert                           Inducible expression ftsZ antisense    253 +76 ftsZ (129 nt.)           This study <br />  pHNM              murA antisense insert                           Inducible expression murA antisense    254 +76 murA (130 nt.)           This study <br />  <br />  * <br />   Antisense target locations indicated relative start codon. <br />  doi:10.1371/journal.pone.0006061.t002 <br />  <br />  <br /> in gene (fabB) needed growth inhibition [56],                       Plasmids strains used testing specificity of <br /> anti-acpP effects rescued plasmid acpP transcomplementa-                        antisense agents <br /> tion [37].                                                                                The ORFs essential genes cloned the <br />    In summary, results essential genes differentially                     multiple cloning site (MCS) pBAD/HisA vector (Invitrogen), <br /> required requirement level appears reflect extent                     allowing gene over-expression L-arabinose-inducible <br /> of predicted protein-protein interactions. Both PNA                                promoter. Sequences amplified K12 genomic DNA <br /> expressed antisense gene silencing methods gave similar results,                       using primers acpP-OF/R, ftsZ-OF/R, murA-OF/R fabI- <br /> despite having different inhibitory mechanisms. Therefore RNA                          OF/R, specific acpP, ftsZ, murA fabI, respectively (Table <br /> silencing used confirm quantify essential gene                           S1). Amplicons digested NcoI XhoI (Fermentas) and <br /> requirement serve molecular pre-screen,                                cloned pBAD/HisA vector containing similarly digested <br /> investing search specific inhibitors. It provide                     ends. Recombinant plasmids transformed E. coli DH5a <br /> a useful tool identifying reference genes quantitative                         cells selected LB plates supplemented ampicillin <br /> transcript profiling.                                                                  (100 mg/ml). Clones used testing specificity the <br />                                                                                        PNA treatment. For testing specificity expressed antisense, <br /> Materials Methods                                                                  clones containing pBAD-fabI, pBAD-acpP, pBAD-ftsZ or <br />                                                                                        pBAD-murA transformed respective antisense- <br /> Bacteria growth conditions specificity testing                              carrying plasmid, i.e. pHN682, pHNA, pHNZ pHNM (Table 2 <br /> antisense agents                                                                       below), selected LB plates supplemented with <br />    E. coli strain DH5a (Invitrogen) transformed                               ampicillin (100 mg/ml) chloramphenicol (30 mg/ml). <br /> recombinant plasmids pBAD-fabI, pBAD-acpP, pBAD-ftsZ and <br /> pBAD-murA (Table 2 below) clones used testing                         Bacteria, growth conditions, PNA treatment and <br /> the specificity gene silencing PNA treatment                           antisense expression gene silencing <br /> antisense expression. Overnight E. coli cultures standardized                        E. coli strain AS19 hyper-permeable phenotype used <br /> by OD550 readings approximately 26104 cfu/ml. To test                           PNA treatments (total 6 doses; Table 1) [40]. AS19 cultures <br /> specificity antisense PNA, clones grown Mueller-Hinton                      Mueller-Hinton broth (DIFCO) used PNA (Panagene, <br /> broth (MHB; Oxoid) supplemented ampicillin (100 mg/ml)                            Korea) treatments prepared described previously [57], and <br /> and 4 mM gene-specific antisense PNA aqueous solution,                           PNA concentrations (0&#8211;110 nM) optimized titration in <br /> either 0.02&#8211;0.2% L-arabinose (Sigma)                               growth inhibition. TOP10 E. coli (Invitrogen) used in <br /> induction essential gene over-expression. Concentrations L-                      transformation reactions plasmids expressing antisense sequenc- <br /> arabinose optimized clone.                                               es (Table 2). Antisense expression induced various <br />    To test expressed antisense specificity, clones containing                     concentrations IPTG (total 6 doses) added LB <br /> essential gene over-expressing plasmid antisense-expressing                        supplemented chloramphenicol (30 mg/ml; Sigma). Over- <br /> plasmid (see below) grown MHB supplemented                                night cultures antisense-expressing clones standardized by <br /> ampicillin (100 mg/ml) plus chloramphenicol (30 mg/ml) 0.1&#8211;                        OD550 readings approximately 26105 cfu/ml IPTG <br /> 75 mM IPTG (for induction expressed antisense),                      induction. For antisense PNA treatment IPTG induction, <br /> without 0.02&#8211;0.2% L-arabinose induction essential gene                          20 ml appropriate PNA IPTG concentration aqueous <br /> over-expression. Concentrations IPTG L-arabinose                           solution deposited 96 plate the <br /> optimized clone. Bacterial cultures grown Bio-                      addition E. coli liquid culture final volume 200 ml. <br /> Tek PowerWave X340 spectrophotometer 37uC agitation                            Bacterial cultures grown VERSAmax spectropho- <br /> every 5 min 200 ml volumes 96-well plate. Growth                           tometer 37uC agitation 5 s 5 min. Growth was <br /> monitored OD550 readings 5 min. Each experiment                           monitored OD550 readings 5 min cultures were <br /> performed triplicate.                                                               harvested untreated control increased turbidity of <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                 6                                June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                                 E. coli Gene Stringency <br />  <br />  <br />  <br />  <br /> Figure 3. Gene silencing E. coli obtain titration declining growth rates transcript levels antisense PNA expressed <br /> antisense. E. coli AS19 treated PNA doses, TOP10 clones containing antisense-expressing plasmids induced various IPTG <br /> concentrations shown Figure 2, obtain growth inhibition dose-dependent manner. Total RNA E. coli cultures harvested the <br /> time control culture increased OD550 0.1. RNA used cDNA synthesis quantitative real-time PCR determine mRNA levels by <br /> normalization reference gene rpoA, calculated relative unsilenced (untreated) control. Growth rates determined OD550 <br /> readings time. The relationship mRNA growth rate decline indicated growth requirement stringency gene. <br /> doi:10.1371/journal.pone.0006061.g003 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         7                                June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                                 E. coli Gene Stringency <br />  <br />  <br />  <br />  <br /> Figure 4. Effect gene silencers cell morphology. Cells stained DAPI fluorescent microscopy. Left panel: AS19 untreated <br /> or treated antisense PNA Ec108 (20 nM) targeting acpP, Ec107 (80 nM) targeting fabI, Ec326 (70 nM) targeting ftsZ Ec330 (55 nM) targeting <br /> murA. Gross elongation cells observed cultures treated ftsZ-specific PNA, Ec326. Right panel: TOP10 clones uninduced induced <br /> with IPTG antisense RNA expression. Concentrations IPTG used induction acpP-, fabI-, ftsZ- murA-antisense expression 160 mM, <br /> 130 mM, 80 mM 80 mM, respectively. Cells expressing ftsZ-antisense grossly elongated compared cells. <br /> doi:10.1371/journal.pone.0006061.g004 <br />  <br />  <br />  <br />  <br /> Figure 5. Determining MTL50 value gene. For gene, mean relative mRNA growth rate values Figure 3 were <br /> transformed mean relative mRNA values plotted corresponding mean values relative growth rates obtain curve. The curve <br /> of gene allows estimation transcript value (on x-<span id='am-97' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-100'  ></span><span id='am-101' property="rdfs:label" content="Axis" datatype="xsd:string"></span>axis</span>) growth reduction 50% (y = 0.5), is, MTL50 value gene of <br /> interest. The MTL50value reveals stringency essential gene growth. <br /> doi:10.1371/journal.pone.0006061.g005 <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         8                                June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                           E. coli Gene Stringency <br />  <br />  <br />  <br />  <br /> Figure 6. Analysis essential gene MTL50 values. Three curves triplicate experiments gene plotted obtain three <br /> MTL50 values statistical analyses. Significant difference (** = p,0.01, * = p,0.05) MTL50 genes determined one-way ANOVA <br /> and Tukey HSD Test. Antisense PNA silencing (left panel) showed MTL50 acpP significantly greater genes. Expressed <br /> antisense RNA silencing (right panel) showed MTL50 acpP ftsZ significantly greater murA only. <br /> doi:10.1371/journal.pone.0006061.g006 <br />  <br /> ,0.1. Wells containing treatment pooled harvest           PNA viewed fluorescence microscopy. Similarly, E. <br /> and cells pelleted RNA extraction. Growth rate                coli TOP10 cultures left uninduced induced IPTG <br /> calculated DOD/Dt.                                                      moderate (dose 3) high expression (dose 6) antisense <br />                                                                            sequences viewed microscopically. Cells grown and <br /> Design antisense PNA gene silencing                                 harvested described above, 1 ml cells pooled and <br />    Antisense PNA specific acpP (Ec108), fabI (Ec107), ftsZ             pelleted. Cells washed equal volume 16PBS, <br /> (Ec326) murA (Ec330) designed using optimal parameters            suspended 100 ml 16PBS, stained DAPI (1 mM) for <br /> for gene silencing [37,38] (Table 1). Briefly, parameters are:       5 min. Cells viewed 10006magnification Leica <br /> 9&#8211;12 mer PNA antisense sequence targeting region              DMRA2 microscope images captured processed <br /> 210 +10 start codon conjugated cell wall-           using Openlab software version 3.1.4. <br /> permeating peptide (KFF)3K. Candidate antisense sequences <br /> within stated start codon region checked sequence <br /> uniqueness similar regions using E. coli database (http:// <br />                                                                            RNA extraction, reverse transcription qPCR <br /> genolist.pasteur.fr/<span id='am-72' about='obo:NCBITaxon_214662' typeof='owl:Thing obo:NCBITaxon_9241 obo:NCBITaxon_9242 obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-73'  ></span><span id='am-74' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="Colibri" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Colibri</span>/) testing antisense inhibition          Extraction RNA bacterial cells, followed DNase I <br /> growth gene expression, described below.                  treatment carried using RiboPure Bacteria Kit <br />                                                                            (Ambion) according manufacturer&#8217;s protocol. RNA <br />                                                                            (200&#8211;500 ng) converted cDNA 25 ml reaction <br /> Plasmid expressing antisense gene silencing                            consisting 16RT reaction buffer, 5.5 mM MgCl2, 0.5 mM <br />    Antisense sequences chosen based length, location (120&#8211;         dNTP, 2.5 mM random hexamers, 0.4 U/ml RNase <br /> 160 nt start codon include promoter coding               inhibitor, 1.25 U/ml MultiScribe Reverse Transcriptase <br /> regions) [12] minimal secondary structures,             (Applied Biosystems). Each 25 ml PCR reaction contained <br /> predicted RNAfold (http://www.tbi.univie.ac.at/RNA/RNA- <br />                                                                            12.5 ml SYBR Green PCR buffer (Eurogentec), 100 nM of <br /> fold.html). Antisense sequences cloned MCS of <br />                                                                            primer (Biomers) 5 ml cDNA. Relative qPCR was <br /> pHN678, flanked 38 bp paired-termini stabilize <br />                                                                            carried primers acpP-F/R, fabI-F/R, ftsZ-F/R, <br /> antisense RNA inserts [12]. Antisense sequences essential genes <br />                                                                            murA-F/R specific acpP, fabI, ftsZ murA, respectively, <br /> were amplified K12 genomic DNA primers acpP-XF1/R2, <br />                                                                            target gene, primers rpoA-F/R specific rpoA as <br /> ftsZ-XF5/R5 murA-XF/R specific acpP, ftsZ murA, <br />                                                                            reference gene (Table S1). Target reference primer <br /> respectively (Table S1). Amplicons digested NcoI and <br />                                                                            pairs validated amplification efficiencies real time <br /> XhoI cloned pHN678 similarly digested ends. <br />                                                                            data analyses, 2-DDCT method [58] primers <br /> Recombinant plasmids transformed TOP10 cells selected <br />                                                                            similar efficiencies, 2DCT (target)/2DCT (reference) <br /> on LB supplemented chloramphenicol (30 mg/ml). The fabI <br />                                                                            method primers different efficiencies [59] relative <br /> antisense expressing plasmid pHN682 described [12]. <br />                                                                            qPCR. Quantitation acpP, fabI, ftsZ murA mRNA was <br />                                                                            normalized rpoA mRNA calculated relative the <br /> Fluorescence microscopy                                                    untreated sample. Mean values error bars, representing <br />   E. coli AS19 cultures left untreated treated         standard deviation experimental replicates, were <br /> sub-inhibitory (dose 3) inhibitory (dose 6) concentrations          plotted PNA IPTG doses. <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       9                              June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                                                               E. coli Gene Stringency <br />  <br />  <br />  <br /> MTL50 calculation                                                                               Supporting Information <br />    For gene, data points DOD/Dt PNA IPTG <br /> concentration (Figure 2) plotted relative mRNA                                 Table S1 Primers used study. <br /> PNA IPTG concentration (Figure 2) new graph. This                                       Found at: doi:10.1371/journal.pone.0006061.s001 (0.06 MB <br /> results curve relative DOD/Dt relative mRNA                                 DOC) <br /> each gene, MTL50 estimated. The x-axis value <br /> where curve intersects y = 0.5 (for 50% reduction growth)                                Acknowledgments <br /> gives MTL50 value. As experiment carried                                    We thank Swedish Institute Guest Scholarship SG, Bloomsbury <br /> triplicate, gene curves PNA treatment                                   Colleges scholarship JMB Swedish Research Council, the <br /> three curves expressed antisense experiments. For simplicity,                              University London Central Research Fund, Sasakawa <br /> only mean data resulting average curve gene                                   Foundation support. We thank Rikard Dryselius, Mark Schreiber, <br /> shown Figure 3. Triplicate MTL50 values, estimated                                      Abbas Nikravesh, Omid Faradani help comments. <br /> triplicate curves, used statistical analysis Figure 6. <br />                                                                                                 Author Contributions <br /> Statistical analysis                                                                            Conceived designed experiments: LG. Performed experiments: <br />   Statistical analyses R2 correlation, mean6SD                                       SG. Analyzed data: SG LG. Wrote paper: SG LG. Carried out <br /> ANOVA carried using Excel. Tukey HSD Test                                          cloning fabI, ftsZ murA over-expressing strains antisense <br /> post-ANOVA comparisons calculated http://faculty.vassar.                                 RNA rescue experiments: JMB. Carried cloning acpP over- <br /> edu/lowry/hsd.html                                                                              expressing strain: NN. Helped design supervise rescue experiments: JS. <br />  <br /> References <br />  1. Vergidis PI, Falagas ME (2008) Multidrug-resistant Gram-negative bacterial                  20. De Lay NR, Cronan JE (2006) Gene-specific random mutagenesis Escherichia <br />     infections: emerging threat potential novel treatment options. Curr Opin                coli vivo: isolation temperature-sensitive mutations acyl carrier protein <br />     Investig Drugs 9: 176&#8211;183.                                                                      fatty acid synthesis. J Bacteriol 188: 287&#8211;296. <br />  2. Baba T, <span id='am-82' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-84'  ></span><span id='am-85' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="Ara" datatype="xsd:string"></span>Ara</span> T, Hasegawa M, Takai Y, Okumura Y, et al. (2006) Construction                   21. Eschenburg S, Priestman MA, Abdul-Latif FA, Delachaume C, Fassy F, et al. <br />     <span id='am-160' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-161' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-164' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span>Escherichia coli K-12</span> in-frame, single-gene knockout mutants: Keio                       (2005) A novel inhibitor suspends induced fit mechanism UDP-N- <br />     collection. Mol Syst Biol 2: 2006 0008.                                                         acetylglucosamine enolpyruvyl transferase (MurA). J Biol Chem 280: <br />  3. Forsyth RA, Haselbeck RJ, Ohlsen KL, Yamamoto RT, Xu H, et al. (2002) A                         14070&#8211;14075. <br />     genome-wide strategy identification essential genes Staphylococcus            22. Jaiswal R, Beuria TK, Mohan R, Mahajan SK, <span id='am-77' about='obo:NCBITaxon_212257' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3646 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_124867 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-78' property="rdfs:label" content="Panda" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80'  ></span><span id='am-81' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Panda</span> D (2007) Totarol inhibits <br />     aureus. Mol Microbiol 43: 1387&#8211;1400.                                                            bacterial cytokinesis perturbing assembly dynamics FtsZ. Biochemistry <br />  4. Yu BJ, Sung BH, Koob MD, Lee CH, Lee JH, et al. (2002) Minimization                      46: 4211&#8211;4220. <br />     Escherichia coli genome using Tn5-targeted Cre/loxP excision system. Nat                  23. Tasdemir D, Topaloglu B, Perozzo R, Brun R, O&#8217;Neill R, et al. (2007) Marine <br />     Biotechnol 20: 1018&#8211;1023.                                                                       natural products Turkish sponge <span id='am-108' about='obo:NCBITaxon_72714' typeof='obo:NCBITaxon_33154 obo:NCBITaxon_85771 owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_6042 obo:NCBITaxon_1 obo:NCBITaxon_6040 obo:NCBITaxon_131567 obo:NCBITaxon_2759 obo:NCBITaxon_85807'><span id='am-109' property="rdfs:label" content="Agelas" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-111'  ></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Agelas</span> oroides inhibit enoyl <br />  5. Butland G, Peregrin-Alvarez JM, Li J, Yang W, Yang X, et al. (2005) Interaction                 reductases <span id='am-60' about='obo:NCBITaxon_5820' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5794 obo:NCBITaxon_131567 obo:NCBITaxon_422676 obo:NCBITaxon_2759 obo:NCBITaxon_5819 obo:NCBITaxon_33630'><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62'  ></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="Plasmodium" datatype="xsd:string"></span>Plasmodium</span> falciparum, <span id='am-28' about='obo:NCBITaxon_1763' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1762 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003'><span id='am-29' property="rdfs:label" content="Mycobacterium" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="PMID:16014496" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="PMID:7547284" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium Lehmann and Neumann 1896" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="PMID:1380284" datatype="xsd:string"></span><span id='am-34'  ></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="PMID:1883713" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="PMID:7907223" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="PMID:1581193" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="PMID:2275850" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="PMID:7547304" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="PMID:8863452" datatype="xsd:string"></span>Mycobacterium</span> tuberculosis Escherichia coli. <br />     network containing conserved essential protein complexes Escherichia coli.               Bioorg Med Chem 15: 6834&#8211;6845. <br />     Nature 433: 531&#8211;537.                                                                        24. Flardh K, Garrido T, Vicente M (1997) Contribution individual promoters in <br />  6. Yellaboina S, Goyal K, Mande SC (2007) Inferring genome-wide functional                         ddlB-ftsZ region transcription essential cell-division gene ftsZ in <br />     linkages E. coli combining improved genome context methods: comparison                    Escherichia coli. Mol Microbiol 24: 927&#8211;936. <br />     high-throughput experimental data. Genome Res 17: 527&#8211;535.                             25. Zhang Y, Cronan JE Jr (1996) Polar allele duplication transcriptional analysis <br />  7. Cheng AC, Coleman RG, Smyth KT, Cao Q, Soulard P, et al. (2007) Structure-                      consecutive essential genes: application cluster Escherichia coli fatty acid <br />     based maximal affinity model predicts small-molecule druggability. Nat                          biosynthetic genes. J Bacteriol 178: 3614&#8211;3620. <br />     Biotechnol 25: 71&#8211;75.                                                                       26. Heath RJ, Rubin JR, Holland DR, Zhang E, Snow ME, et al. (1999) Mechanism <br />  8. Myllykallio H, Leduc D, Filee J, Liebl U (2003) Life dihydrofolate                      triclosan inhibition bacterial fatty acid synthesis. J Biol Chem 274: <br />                                                                                                     11110&#8211;11114. <br />     reductase FolA. Trends Microbiol 11: 220&#8211;223. <br />                                                                                                 27. Marquardt JL, Siegele DA, Kolter R, Walsh CT (1992) Cloning sequencing <br />  9. Badarinarayana V, Estep PW 3rd, Shendure J, Edwards J, Tavazoie S, et al. <br />                                                                                                     Escherichia coli murZ purification product, UDP-N-acetylglucosa- <br />     (2001) Selection analyses insertional mutants using subgenic-resolution arrays. <br />                                                                                                     enolpyruvyl transferase. J Bacteriol 174: 5748&#8211;5752. <br />     Nat Biotechnol 19: 1060&#8211;1065. <br />                                                                                                 28. Marquardt JL, Brown ED, Lane WS, Haley TM, Ichikawa Y, et al. (1994) <br /> 10. Herring CD, Blattner FR (2004) Conditional lethal amber mutations essential <br />                                                                                                     Kinetics, stoichiometry, identification reactive thiolate the <br />     Escherichia coli genes. J Bacteriol 186: 2673&#8211;2681. <br />                                                                                                     inactivation UDP-GlcNAc enolpyruvoyl transferase antibiotic <br /> 11. Dryselius R, Nekhotiaeva N, Good L (2005) Antimicrobial synergy                         fosfomycin. Biochemistry 33: 10646&#8211;10651. <br />     mRNA- protein-level inhibitors. J Antimicrob Chemother 56: 97&#8211;103.                      29. Schonbrunn E, Eschenburg S, Krekel F, Luger K, Amrhein N (2000) Role the <br /> 12. Nakashima N, Tamura T, Good L (2006) Paired termini stabilize antisense                         loop containing residue 115 induced-fit mechanism bacterial cell <br />     RNAs enhance conditional gene silencing Escherichia coli. Nucleic Acids                  wall biosynthetic enzyme MurA. Biochemistry 39: 2164&#8211;2173. <br />     Res 34: e138.                                                                               30. Tsao DH, Sutherland AG, Jennings LD, Li Y, Rush TS 3rd, et al. (2006) <br /> 13. Nekhotiaeva N, Awasthi SK, Nielsen PE, Good L (2004) Inhibition                              Discovery novel inhibitors ZipA/FtsZ complex NMR fragment <br />     <span id='am-14' about='obo:NCBITaxon_1280' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1385 obo:NCBITaxon_90964 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_1279'><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Staphylococus aureus" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Micrococcus pyogenes" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Staphilococcus aureus" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="PMID:8573498" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Micrococcus aureus" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Staphylococcus aureus" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Streptococcus aureus" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-24'  ></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Staphlococcus pyogenes citreus" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Staphylococcus pyogenes aureus" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="NCBITaxon:325213" datatype="xsd:string"></span>Staphylococcus aureus</span> gene expression growth using antisense peptide nucleic                screening coupled structure-based design. Bioorg Med Chem 14: <br />     acids. Mol Ther 10: 652&#8211;659.                                                                    7953&#8211;7961. <br /> 14. Pragman <span id='am-141' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-142' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-144' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-146'  ></span>AA</span>, Ji Y, Schlievert PM (2007) Repression Staphylococcus aureus SrrAB            31. Bi EF, Lutkenhaus J (1991) FtsZ ring structure associated division in <br />     using inducible antisense srrA alters growth virulence factor transcript levels.            Escherichia coli. Nature 354: 161&#8211;164. <br />     Biochemistry 46: 314&#8211;321.                                                                   32. Majerus PW, Alberts AW, Vagelos PR (1964) The acyl carrier protein fatty <br /> 15. Kulyte A, Nekhotiaeva N, Awasthi SK, Good L (2005) Inhibition                                acid synthesis: purification, physical properties, substrate binding site. Proc <br />     <span id='am-43' about='obo:NCBITaxon_1772' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1762 obo:NCBITaxon_1763 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003'><span id='am-44' property="rdfs:label" content="Mycobacterium smegmatis" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium paratuberculosis smegmatis" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-47'  ></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Bacterium smegmatis" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Bacillus smegmatis" datatype="xsd:string"></span>Mycobacterium smegmatis</span> gene expression growth using antisense peptide                      Natl Acad Sci U S A 51: 1231&#8211;1238. <br />     nucleic acids. J Mol Microbiol Biotechnol 9: 101&#8211;109.                                       33. Jackowski S, Rock CO (1983) Ratio active inactive forms acyl carrier <br /> 16. Harth G, Zamecnik PC, Tabatadze D, Pierson K, Horwitz MA (2007) Hairpin                         protein Escherichia coli. J Biol Chem 258: 15186&#8211;15191. <br />     extensions enhance efficacy mycolyl transferase-specific antisense                   34. Tan XX, Actor JK, Chen Y (2005) Peptide nucleic acid antisense oligomer a <br />     oligonucleotides targeting <span id='am-51' about='obo:NCBITaxon_1773' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1762 obo:NCBITaxon_1763 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_77643'><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium tuberculosis typus humanus" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium tuberculosis var. hominis" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Bacillus tuberculosis" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Bacterium tuberculosis" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58'  ></span><span id='am-59' property="rdfs:label" content="Mycobacterium tuberculosis" datatype="xsd:string"></span>Mycobacterium tuberculosis</span>. Proc Natl Acad Sci U S A                 therapeutic strategy bacterial infection: proof principle using mouse <br />     104: 7199&#8211;7204.                                                                                 intraperitoneal infection. Antimicrob Agents Chemother 49: 3203&#8211;3207. <br /> 17. Heath RJ, Rock CO (1995) Enoyl-acyl carrier protein reductase (FabI) plays                35. Tilley LD, Mellbye BL, Puckett SE, Iversen PL, Geller BL (2007) Antisense <br />     determinant role completing cycles fatty acid elo <br /> </body></html>