<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Dec. 1987, p. 5408-5415                                                                   Vol. 169, No. 12 <br /> 0021-9193/87/125408-08$02.00/0 <br /> Copyright &#169; 1987, American Society Microbiology <br />  <br />  <br />       Sequence Analysis, Transcriptional Organization, Insertional <br />             Mutagenesis envA Gene <span id='am-24' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-25'  ></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia coli</span> <br />                                       BERNARD BEALL AND JOE LUTKENHAUS* <br />                 Department Microbiology, University Kansas Medical Center, Kansas City, Kahsas 66103 <br />                                          Received 29 June 1987/Accepted 2 September 1987 <br />  <br />               The Eschenichia coli cel permeability-cell separation gene envA region immediately downstream <br />            sequenced. The envA gene consistd 305 codons encoded 34-kilodalton polypeptide lacked <br />            signal sequence hydrophobic membrane-spanning regions. The envAl mutation determined a <br />            missense mutation <span id='am-16' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-17'  ></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> 19 resulting change amino acid sequence histidine tyrosine. <br />            Located 299 base pairs downstream envA gene uidentified open reading frame consisting 147 <br />            codons. <span id='am-11' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> open reading frame followed additional open reading frame starting 59 base pairs <br />            downstream corresponded secA gene. A transcription terminator located just <br />            downstream envA fragment contained sequence corresponding typical rho-independent <br />            terminator. Transcription envA upstreamfts genes terminated terminator probably <br />            uncoupled downstream genes, including secA. Gene disruption experiments indicated envA <br />            gene essential gene. <br />  <br />    The envA gene maps large cell envelope-cell                 The envA gene identified basis single <br /> division gene cluster located 2 min <span id='am-40' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-41' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-43'  ></span><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia</span> coli       non-temperature-sensitive mutation results pleio- <br /> genetic map (1). This region received considerable               tropic phenotype (18, 19). The phenotype associated with <br /> attention contains severalfts genes, including        envAl mutation includes increased sensitivity hydro- <br /> ftsZ gene, apparently pivotal role cell division      phobic hydrophilic antibiotics defect cell <br /> (31). In addition, region contains secA gene,         separation resulting chain growth. This phenotype <br /> plays essential role protein export cytoplas-       correlated decreased N-acetylmuramyl-L- <br /> mic membrane (22), number peptidylglycan biosyn-            alanine amidase activity (33). The antibiotic hypersensitivity <br /> thetic genes (1). These genes tightly clustered         hyperpermeability associated lower <br /> are transcribed direction (13, 14, 28). Nonethe-         lipopolysaccharide/protein ratio outer membrane (5). <br /> less, promoters associated            One extragenic suppressor antibiotic sensitivity, sefA, <br /> with gene, transcription terminators           results increase outer membrane protein content and <br /> located region (23, 36).                                      cells incomplete septa (5, 20). <br />    The envA gene immediately downstream cell                  To aid determining function envA gene <br /> division genes ftsQ, ftsA, ftsZ separated        product analysis gene expression region, <br /> 3&apos; end ftsZ gene 100 base pairs (bp) (35, 36).             completed sequence envA gene the <br /> Downstream envA gene secA gene,               known envA mutation. We demonstrated through <br /> transcribed direction envA (22). It          gene disruption experiments envA essential gene. <br /> noted approximately 900 bp unassigned              Further sequence analysis revealed additional short open <br /> DNA sequence envA secA code                  reading frame starting 299 bp downstream envA, is <br /> for moderately sized protein. Promoter analysis suggests           followed start secA gene. A strong transcrip- <br /> that gene does exist, transcribed       tional terminator located 299-bp intergenic <br /> direction envA secA (28). Immediately upstream             region downstream envA. <br /> thefts genes lie number murein biosynthetic genes. Two <br /> of genes, murC ddl, located immediately                         MATERIALS AND METHODS <br /> upstream ftsQ, transcribed direction              Bacterial phage strains. All bacterial strains used in <br /> ftsQ (13, 14). Sequence analysis revealed ddl           work derivatives E. coli K-12 listed in <br /> ftsQ separated 2 bp (24) ftsQ           Table 1. Strain GIA86 used complementation tests to <br /> ftsA genes actually overlap 2 bp (23, 36). In contrast,ftsA       assess envA allele carried <span id='am-73' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-74' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span>. The transducing <br /> and ftsZ separated 60 bp include sequence             phages X16-2 X16-3 described previously (12). <br /> that form hairpin structure terminator      They identical X16-2 carries wild-type <br /> the reverse direction (35, 36).                                      envA allele X16-3 carries envAL. In addition, phage <br />    The fts genes identified temperature-                carry ftsZ, ftsA, ftsQ, murC, ddl genes. <br /> sensitive mutations result filamentous growth               Plasmid constructions. Plasmid pBL1 constructed by <br /> eventual cell death nonpermissive temperature (2, 13,         ligating Hindlll-digested X16-2 pBR322 DNAs trans- <br /> 29). In addition, mutations render cells resistant       forming JFL101 [ftsZ84(Ts)J ampicillin resistance and <br /> SOS-inducible cell division inhibitor <span id='am-78' about='obo:NCBITaxon_32641' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_9205 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_30446 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-79'  ></span><span id='am-80' property="rdfs:label" content="Sula" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>sulA</span> located         temperature resistance. Plasmid pBL2 carrying envAl <br /> within ftsZ gene (8, 11).                                        mutation constructed manner using X16-3 <br />                                                                      DNA. Both plasmids contained 3.5-kilobase (kb) HindIII <br />                                                                      fragment containing entire ftsZ gene wild-type <br />   *   Corresponding author.                                          envA gene mutant envAl allele. These HindIII frag- <br />                                                               5408 <br />  VOL. 169, 1987                                                                                                                                                 envA GENE OF E. COLI            5409 <br />  <br />                                                         TABLE 1. Bacterial phage strains used <br /> Bacterial strain             Relevant marker                                                                            Genotype                                               Source reference <br /> GIA86                envAl                                               thr thi pyrF thyA ilvA arg lac tonA tsx                                                      12 <br /> JFL101               ftsZ84(Ts)                                          recA ilv deo <span id='am-6' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-7' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>ara</span>(Am) lacZl25(Am) galU42(Am)                                                      12 <br />                                                                            trp(Am) tyrT [supFA81(Ts)] <br /> JC7623               recB21 recC22 sbcBl5                                arg ara leu pro thr                                                                          31 <br /> NK6923               leu::TnlO                                           thy                                                                                              M. Singer <br /> BL7623(X16-2)        envA::kanr leu::TnlO                                As JC7623                                                                                        This study <br /> W3110                                                                    Prototroph                                                                                       Laboratory collection <br /> JM101                A(lac proAB)/F&apos; traD36 proAB                                                                                                                         16 <br /> NK5549               F&apos; Tn9 lacIq                                                                                                                                         Nancy Kleckner <br />  <br /> ments cloned pACYC184, yielding pBL3                                                      striction fragments phages determined the <br /> (envA) pBL4 (envAl).                                                                                 C test described previously (17). Products sequenc- <br />    Plasmid pBL5 consists 3.5-kb Hindlll fragment                                               ing reactions analyzed use buffer gradient gels (3). <br /> X16-2 cloned low-copy-number plasmid vector pGB2                                                   Inactivation chromosomal envA gene. A strain con- <br /> (4). Plasmid pBL5K, containing disrupted envA gene,                                                taining disrupted envA gene constructed the <br /> constructed replacing 209-bp ClaI fragment pBL5                                                procedure Winans et al. (32). First, strain JC7623 was <br /> with AccI fragment carrying 1.5-kb kanamyacin                                                    lysogenized X16-2 provide additional copy the <br /> resistance cassette pUC4K (see Fig. 6).                                                             envA gene. This lysogen transformed pBL5K that <br />    To test possible transcription terminators, pKO (16)                                            linearized EcoRI digestion. A Kanr Spcs <br /> derivative, pSR132 (Rockenbach Lutkenhaus, unpub-                                                    transformant transduced Tetr P1 grown strain <br /> lished), used. This plasmid contains EcoRI fragment                                               NK6923 (leu: :TnJO). Among Tetr transductants, 53% had <br /> containing promoter located near 5&apos; end ftsQ                                                lost Kanr, indicating Kanr linked leu the <br /> gene cloned upstream galK gene. The plasmid                                                   chromosomal copy envA inactivated this <br /> several restriction sites located promoter                                               transformant. One Tetr transformants re- <br /> the galK gene fragments cloned test                                               tained Kanr designated BL7623(A16-2) used in <br /> terminator activity (see Fig. 5).                                                                        P1 transductions. <br />    Marker rescue envAl mutation. Strain GIA                                                          Labeling proteins synthesized maxicells. Strain JFL101 <br /> 86(F&apos;::Tn9 lacIq) infected 2x YT medium (17)                                                 transformed various plasmids labeled by <br /> recombinant M13 phage carrying different segments                                                 maxicell described Sancar et al. (26). The <br /> wild-type envA allele. Adequate time allowed recom-                                              labeled proteins analyzed sodium dodecyl sulfate- <br /> bination expression wild-type envA allele. Rescue                                             polyacrylamide gel electrophoresis autoradiography. <br /> of envAl mutation scored plating infected <br /> cells L-agar plates containing rifampin (5 ,ug/ml).                                                                         RESULTS <br />    DNA sequencing. DNA sequenced dideoxy <br /> method strain JM101 host M13mpl8                                                       Sequence wild-type mutant envA alleles. The <br /> M13mpl9 derivatives (34). The orientations cloned re-                                                 envA gene located 2.5-kb EcoRI fragment that <br />  <br />                                HE        E ap           ES               Bg                  H           E                          C       CP   P   P    E       H <br />                                           II II         I I <br />                                                         II                                   I           I <br />                                                                                                                                II <br />                                   murC            ddl         ftsQ                ftsA                           ftsZ               envA                 geneX   soT <br />                                                                      I             lkb           I           / <br />  <br />                                                                                                      / <br />  <br />  <br />                                                                                   /              C       CP             P Pv P HpEA                       Hp H <br />                                                                               I          I <br />                                                                                                                           Tt <br />                                                                                                                                         I                 11 <br />                                                                               0-                     envA                                   geneX          secA <br />  <br />  <br />                                                                                                  1-4 <br />                                                                                                  4-                                              i <br />                                                                          *                       [AC                                                 --4 <br />                                                                                                                                                       - <br />  <br />  <br />                                                                                              4 <br />  <br />  <br />  <br />  <br />    FIG. 1. Sequencing strategy envA region. The line indicates restriction map gene <span id='am-21' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> 2-min region E. <br /> coli genetic map. The expanded region restriction map DNA sequenced study. The positions consensus <br /> promoter sequences indicated arrowheads, terminator indicated t. The horizontal arrows indicate lengths and <br /> directions sequences determined dideoxy sequencing method. The fragments marked asterisks indicated fragments that <br /> could rescue envAl mutation. Abbreviations: A, AsuII; Bg, BglII; C, Clal; E, EcoRI; H, Hindlll; Hp, HpaI; P, PstI; Pv, PvuII; AC, <br /> deletion 209-bp ClaI fragment. <br />  5410    BEALL AND LUTKENHAUS                                                                                              J. BACTERIOL. <br />  <br />  <br />  <br />                                             (End FtsZ] <br />  Phe Leu Arg Lys Gln <span id='am-51' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Ala" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55'  ></span>Ala</span> Asp ---               -35                   -10 <br />  TTC CTG CGT AAG CM GCT GAT TAA GAATTGACTGGMTTTGGGTTTCGATTCTCTTTGTGCTMACTGGCC                                                    1571 <br />                                                                           (Start EnvA] <br />                                                       Hot Iie Lys Gin Arg Thr <br />  CGCCGAATGTATAGTACACTTCGGTTGGATAGGTAATTTGGCGAGATMTACG ATG ATC AAA CM AGG ACA                                                     1642 <br />  Leu Lys Arg Ile Val Gln Ala Thr Gly Val Gly Lou IHis Thr Gly Lys Lys Val Thr Leu <br />  CTT <span id='am-57' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-58' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-61'  ></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>AA</span> CGT ATC GTT CAG GCG ACG GGT GTC GGT TTA CAT ACC GGC MG MA GTC ACC CTG                                                    1702 <br />  Thr Leu Arg Pro Ala Pro Ala Aen Thr Gly Val Ile Tyr Arg Arg Thr Asp Lou Aen Pro <br />  ACG TTA CGC CCT GCG CCG GCC MC ACC GGG GTC ATC TAT CGT CGC ACC GAC TTG MT CCA                                                   1762 <br />  Pro Val Asp Phe Pro Ala Asp Ala Lys Ser Val Arg Asp Thr Hot Lou Cys Thr Cys Leu <br />  CCG GTA GAT TTC CCG GCC GAT GCC MA TCT GTG CGT GAT ACC ATG CTC TGT ACG TGT CTG                                                  1822 <br />  Val Asn Glu His Asp V41 Arg Ile Ser Thr Val Glu His Lou Aen Ala Ala Lou Ala Gly <br />  GTC MC GAG CAT GAT GTA CGG ATT TCA ACC GTA GAG CAC CTC MT GCT GCT CTC GCG GGC                                                   1882 <br />  Leu Gly Ile Asp Aen Ile Val Ile Glu Val Asn                          Ala Pro Glu Ile Pro I1e Hot Asp Gly <br />  TTG GGC ATC GAT MC ATT GTT ATC GM GTT MC                             GCG CCG GAM ATC CCG ATC ATG GAC GGC                        1942 <br />           ^ClaI                        ^HpaI <br />  Ser Ala Ala Pro Phe Val Tyr Leu Leu Lou Asp                          Ala Gly Ile Asp Glu Lou Asn Cys Ala <br />  AGC GCC GCT CCG TTT GTA TAC CTG CTG CTT GAC                          GCC GGT ATC GAC GAG TTG MC TGC GCC                         2002 <br />  <br />  Lys Lys Phe Val Arg Ile Lys Glu Thr Val Arg Val Glu Asp Gly Asp Lys Trp Ala Glu <br />  MA MA TTT GTT CGC ATC MA GAG ACT GTT CGT GTC GAM GAT GGC GAT MG TGG GCT GM 2062 <br />  Phe Lys Pro Tyr Aen Gly Phe Ser Lou Asp Phe Thr Ile Asp Phe Ash His Pro Ala Ile <br />  TTT MG CCG TAC MT GGT TTT TCG CTG GAT TTC ACC ATC GAT TTT MC CAT CCG GCT ATT                                                    2122 <br />                                                   ^ClaI <br />  Asp Ser Ser Asn Gln Arg Tyr Ala met Asn Phe Ser Ala Asp Ala Phe Met Arg Gln Ile <br />  GAT TCC AGC MC CAG CGC TAT GCG ATG MC TTC TCC GCT GAT GCG TTT ATG CGC CAG ATC                                                   2182 <br />  Ser Arg Ala Arg Thr Phe Gly Phe Met Arg Asp Ile Glu Tyr Lou Gln Ser Arg Gly Lou <br />  AGC CGT GCG CGT ACG TTC GGT TTC ATG CGT GAT ATC GAM TAT CTG CAG TCC CGT GGT TTG                                                 2242 <br />                                                                                             ^PstI <br />  Cys Leu Gly Gly Ser Phe Asp Cys Ala Ile Val Val Asp                               Asp Tyr Arg Val Leu Asn Glu <br />  TGC CTG GGC GGC AGC TTC GAT TGT GCC ATC GTT GTT GAC GAT TAT CGC GTA CTG MC GM 2302 <br />  Asp Gly Leu Arg Phe Glu Asp Glu Phe Val Arg His Lys Hot Lou Asp Ala 1l1 Gly Asp <br />  GAC GGC CTG CGT TTT GM GAC GM TTT GTG CGT CAC MA ATG CTC GAT GCG ATC GGT GAC                                                    2362 <br />  Leu Phe Het Cys Gly His Asn Ile Ile Gly Ala Phe Thr Ala Tyr Lys Ser Gly His Ala <br />  TTG TTC ATG TGT GGT CAC MT ATT ATT GGT GCA TTT ACC GCT TAT MA TCC GGT CAT GCA                                                   2422 <br />  Leu Asn Aen Lys Leu Leu Gln Ala Val Lou Ala Lys Gln Glu Ala Trp Glu Tyr Val Thr <br />  CTG MT MC AA CTG CTG CAG GCT GTC CTG GCG AAA CAG GM GCC TGG GM TAT GTG ACC                                                      2482 <br />                           APstI                                       ( End EnvA ] <br />  Phe Gln Asp Asp Ala Glu Leu Pro Leu Ala Phe Lys Ala Pro Ser Ala Val Leu Ala                                             --- <br />  <br />  <br />  <br />  TTC CAG GAC GAC GCA GM CTG CCG TTG GCC TTC AAA GCG CCT TCA GCT GTA CTG GCA TM                                                   2542 <br />                                                                                                 &apos;^PvuII <br />   FIG. 2. Nucleotide sequence envA region. The deduced amino acid sequences end ftsZ, envA, gene X the <br /> beginning secA  given nucleotide sequence. The sequence envAl mutation box. Putative promoter <br /> sequences underlined, location hyphenated dyad symmetry indicated arrows. The numbering consistent with <br /> the previously published sequence ftsZ (35). <br />   VOL. 169, 1987                                                                                envA GENE OF E. COLI           5411 <br />  <br />   CGACATTTATACTGTCGTATAAAATTCGACTGGCAAATCTGGCACTCTCTCCGGCCAGGTGAACCAGTCGTTTTTTTTT                                      2621 <br />  <br />  <br />   GAATTTTATAAGAGCTATAAAAAACGGTGCGAACGCTGTTTTCTTAAGCACTTTTCCGCACAACTTATCTTCATTCGTG                                      2700 <br />  <br />                                                                         -35                            -10 <br />   CTGTGGACTGCAGGCTTTAATGATAAGATTTGTGCGCTAAATACGTTTGAATATGATCGGGATGGCAATAACGTGAGTG                                      2779 <br />              ^PstI <br />                                                                                              Met Val Ala Ala <br />   GAATACTGACGCGCTGGCGACAGTTTGGTAAACGCTACTTCTGGCCGCATCTCTTATTAGGG ATG GTT GCG GCG                                       2853 <br />  <br />   Ser Leu Gly Leu Pro Ala Leu Ser Asn Ala Ala Glu Pro Asn Ala Pro Ala Lys Ala Thr <br />   AGT TTA GGT TTG CCT GCG CTC AGC AAC GCC GCC GAA CCA AAC GCG CCC GCA AAA GCG ACA                                      2913 <br />  <br />   Thr Arg Asn His Glu Pro Ser Ala Lys Val Asn Phe Gly Gln Leu Ala Leu Leu Glu Ala <br />   ACC CGC AAC CAC GAG CCT TCA GCC AAA GTT AAC TTT GGT CAA TTG GCC TTG CTG GAA GCG                                      2973 <br />                                                            &apos;HpaI <br />   Asn Thr Arg Arg Pro Asn Ser Asn Tyr Ser Val Asp Tyr Trp His Gln His Ala Ile Arg <br />   AAC ACA CGC CGC CCG AAT TCG AAC TAT TCC GTT GAT TAC TGG CAT CAA CAT GCC ATT CGC                                      3033 <br />                     ^EcoRI^AsUuII <br />   Thr Val Ile Arg His Leu Ser Phe Ala Met Ala Pro Gln Thr Leu Pro Val Ala Glu Glu <br />   ACG GTA ATC CGT CAT CTT TCT TTC GCA ATG GCA CCG CAA ACA CTG C0C GTT GCT GAA GAA                                      3093 <br />  <br />   Ser Leu Pro Leu Gln Ala Gln His Leu Ala Leu Leu Asp Thr Leu Ser Ala Leu Leu Thr <br />   TCT TTC CCT CTT CAG GCG CAA CAT CTT GCA TTA CTG GAT ACG CTC AGC GCG CTG CTG ACC                                      3153 <br />  <br />   Gln Glu Gly Thr Pro Ser Glu Lys Gly Tyr Arg Ile Asp Tyr Ala His Phe Thr Pro Gln <br />   CAG GAA GGC ACG CCG TCT GAA AAG GGT TAT CGC ATT GAT TAT GCG CAT TTT ACC CCA CAA                                      3213 <br />   Ala Lys Phe Ser Thr Pro Val Trp Ile Ser Gln Ala Gln Gly Ile Arg Ala Gly Pro Gln <br />   GCA AAA TTC AGC ACG CCC GTC TGG ATA AGC CAG GCG CAA GGC ATC CGT GCT GGC CCT CAA                                      3273 <br />                                                                                                      [Start SecA] <br />   Arg Leu Thr       -   -   - <br />                                                                                Met <br />   CGC CTC ACC TAACAACAATAAACCTTTACTTCATTTTATTAACTCCGCAACGCGGGGCGTTTGAGATTTTATT ATG 3349 <br />  <br />   Leu Ile Lys Leu Leu Thr              Lys Val Phe Gly Ser Arg Asn Asp Arg Thr Leu Arg Arg Met <br />   CTA ATC AAA TTG TTA ACT              AAA GTT TTC GGT AGT CGT AAC GAT CGC ACC CTG CGC CGG ATG                         3409 <br />                    ^HpaI <br />   Arg Lys Val Val Asn Ile              Ile Asn Ala Met Glu Pro Glu Met Glu Lys Leu Ser Asp Glu <br />   CGC AAA GTC GTO AAC ATC              ATC AAT GCC ATG GAA CCG GAG ATG GAA AAA CTC TCC GAC GAA                         3469 <br />   Glu Leu Lys Gly Lys Thr Ala Glu Phe Arg Ala Arg Leu Glu Lys Gly Glu Val Leu Glu <br />   GAM CTG AAA      A  ACC GCA GAG TTT CGT GCA CGT CTG GAA AAA GGC GCM GTG CTG GAA                                      3529 <br />  <br />  <br />   Asn Leu Ile Pro Glu Ala <br />   AAT CTG ATC CCG GAA GCT T                3548 <br />                                 &apos;HindIII <br /> also contains 3&apos; end ftsZ gene (Fig. 1) (13,       Analysis sequence <span id='am-63' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-65'  ></span><span id='am-66' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> revealed env-A gene <br /> 35). By promoter fusion experiments nucleotide se-             consisted 305 codons yielding predicted protein a <br /> quence analysis, orientation transctiption deter-       molecular weight 34,000 (34K). The gene preceded by <br /> mined theftsZ gene (28). Previously      Shine-Dalgarno sequence (27), GAG, located 8 bp up- <br /> the 5&apos; end envA gene sequenced               stream initiation codon. Located 57 bp upstream the <br /> determination nucleotide sequence ftsZ gene          initiation codon just i4 bp 3&apos; end ftsZ <br /> (35). To complete sequence envA gene, used           gene sequence showed strong homology the <br /> plasmids obtained subcloning envA gene X16-2           consensus E. coli promoter sequence (6). Just downstream <br /> (see Materials Methods). The strategy sequencing           envA gene stretch T residues preceded <br /> the envA region shown Fig. 1, results shown      region hyphenated dyad symmetry. Such sequences <br /> in Fig. 2.                                                         rho-independent terminators (25). <br />  5412             BEALL AND LUTKENHAUS                                                                                         J. BACTERIOL. <br />  <br />    The predicted envA gene product contained 22% charged <br /> residues distributed length                                    1    2        3 <br /> protein. The protein lacked amino-terminal signal se- <br /> quence long hydrophobic stretches (Fig. 3). Comparison <br /> of predicted amino acid sequence envA gene <br /> product protein sequences present Protein Iden-                                                        &lt; ftsZ <br /> tification Resource National Biomedical Research                                                   *   _     aenvA <br /> Foundation revealed significant homologies. <br />    Marker rescue envA mutation strain GIA86 <br /> revealed phage containing DNA left the <br /> first ClaI site envA gene capable rescuing <br /> the envA mutation (Fig. 1). These results confirmed by <br /> the observation pZAQ, contained wild-type DNA <br /> to left ClaI site inserted pBR322, also <br /> rescue envAl high frequency (data shown). Se-                      FIG. 4. Comparison gene products envA EnvAl <br /> quence analysis mutant allele 5&apos; end              alleles. Maxicells containing plasmids labeled [35S]methi- <br /> ClaI site revealed mutation codon 19 resulting                 onine analyzed polyacrylamide gel electrophoresis and <br /> change amino acid sequence histidine tyrosine             autoradiography. Lane 1, No plasmid; lane 2, pBL3 (envA); lane 3, <br /> (Fig. 2). A missense mutation envAl unexpected,               pBL4 (envAl). <br /> because envA envAl code proteins the <br /> same molecular weight (Fig. 4). <br />    Seqqence analysis region 3&apos; end envA.            terminator antisense orientation (unpublished obser- <br /> The secA gene located downstream envA                    vations). Previously, Sullivan Donachie (28) used pro- <br /> gene, 5&apos; end located 450-bp EcoRI-HindIII                moter fusion experiments determine position pro- <br /> fragment (Fig. 1). In addition, promoter           moters envA region. In experiments did not <br /> located downstream envA upstream secA (28). This               detect transcription terminators immediatfly down- <br /> promoter appears sufficiently upstream secA             stream envA. However, inspection sequence Fig. <br /> additional gene located promoter                  3 raised possibility terminator existed just down- <br /> secA.                                                                    stream envA, proximity promoter for <br />    Analysis sequence data revealed open reading                gene X detected unless elements <br /> frame starting positions 3347 3349 extending            separated. <br /> end fragment. The location reading frame                  To test presence transcription terminator, the <br /> consistent beginning secA gene.               PstI fragment containing suspected terminator not <br /> Upstream reading frame additional open                    gene X promoter (Fig. 1) examined cloning a <br /> reading frame starting 299 bp downstream envA                     pKO vector, pSR132, contained promoter there- <br /> consisting 147 codons. The predicted protein 16K               fore expressed galK. Since PstI fragment already <br /> size. A consensus promoterlike sequence occurred posi-                cloned M13mpl9 sequencing, able take <br /> tions 3747 3774.                                                      advantage flanking restriction sites directionally sub- <br />    Transcription termination. Earlier results indicated              clone fragment pSR132 orientations (Fig. 5). <br /> that transcription terminators existed region be-              The results clearly expected orientation a <br /> tween ddl envA extend secA.                     strong terminator, pSR132 fragment this <br /> Although sequence existed ftsA ftsZ                   orientation (pBL11) gave white colonies MacConkey- <br /> could form stable hairpin structure, acted                galactose <span id='am-1' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Indicator" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5'  ></span>indicator</span> plates. The opposite orientation (pBL10) <br />                                                                          did significantly affect galK expression. Thus, transcrip- <br />                                                                          tion envA terminate terminator affect <br />                                                                          expression downstream secA gene. <br />                                                                             Construction null mutation envA. Our strategy for <br />                                                                          obtaining null mutation envA gene place a <br />                                                                          selectable marker envA gene plasmid and <br /> 1I <br /> P4 - <br />      - <br />                    11                                                    cross mutation chromosome (Fig. 6). <br />       ti    Pi     -O    JI <br />                         f   iw <br />                                                                          Strain JC7623 lysogenized X16-2 provide an <br />                                                                          additional copy envA. This lysogen transformed with <br />                                                                          pBL5K linearized EcoRI digestion. To determine the <br />       -rI                     T <br />                               rl <br />                                II                               JII   location kan marker, phage induced several <br />             ,Ht&quot;                     &apos;j   \14r- I&apos;   5 <br />                                                              Ii   YI <br />                                                                          Kanr Spcs colonies screened ability transduce <br />       4- <br />                    ~~~ <br />                     ~~~                1~~~~~~~                       cells Kanr. If phage unable transduce cells to <br /> .,q <br />                                                                          Kanr, corresponding lysogen candidate having <br /> -H <br /> 10 <br />  0 <br />       4- <br />                                                                          kan marker chromosomal envA gene the <br /> 0 <br /> 1:4 <br />       -r1                                                                X16-2 envA locus. The location kan marker such <br /> ~0 <br /> la <br />       -4-                                                                transformant tested P1 transduction closely <br /> :z:                                                                      linked TnJO insertion leu. Of Tetr transductants, 53% <br />    FIG. 3. Hydrophilicity-hydrophobicity plot EnvA pro-          Kans, demonstrating linkage kan leu (Table 2). <br /> tein. The abscissa divided increments 10 amino acids.         One Kanr Tetr transductant, designated BL6723(X16-2), was <br /> Hydrophilicity indicated positive value ordinate,     infected P1 transduction strains having one <br /> hydrophobicity indicated negative value. The values         (W3110) [W3110(X16-2)] copies envA gene. <br /> determined using range residues (7).                         When selected marker Kanr, strain two <br />  VOL. 169, 1987                                                                                                            envA GENE OF E. COLI                      5413 <br />  <br />                B p      p   H                          B p        pH <br />  <br />                        PV                                    Pv <br />  <br />                                        BamHI <br />                                        Hind 1ll <br />  <br />  <br />                                                                                                                                                           -E <br />                                 O-W,              H <br />  <br />  <br />  <br />  <br />                                                                                                                                                 EcoRI <br />                                        PSR132                 galK <br />  <br />                                                                                                                                             kanr <br />  <br />                                                                                                                               E                           EE <br />  <br />                                                                                                                                         envA&apos; envA&quot; <br />  <br />  <br />  <br />                 B p <br />                                                         \                                             tetr <br />  <br />                                                                                                    leu::TnIO <br />                                                                                                                                    x <br />  <br />                                                                                                                                              envA <br />                                                                                                                                                   x <br />  <br />  <br />  <br />  <br />                        Pv <br />                                   H               le <br />                                                                   Pv <br />                                                                                                       tetr <br />                                                                                                           57 <br />                                                                                                                                recombination~~~~~~~kanr <br />                                                                                                                                                w <br />  <br />  <br />                                                                                                    1eu::TnlO                         envA&apos;     envA&quot; <br />                pBLIO              galK                        pSI11 <br />                                                                                              FIG. 6. Construction null allele envA. The small ClaI <br />                                                                                            fragment pBL5 replaced Accl fragment containing <br />                                                                                            kan cassette pUC4K. The resultant plasmid pBL5K was <br />                                                                                            digested EcoRI used transform JC7623(X16-2) Kanr. <br />                                                                                            Homologous recombination results replacement chromo- <br />   FIG. 5. Plasmid constructions analysis terminator activity.                       somal gene disrupted allele. The solid line represents <br /> The PstI fragment contained orientations M13mp19                            chromosomal DNA, open box represents plasmid DNA. <br /> cloned pSR132 using BamHI (B) Hindlll (H) sites. The                           Abbreviations: H, Hindlll; E, EcoRI; C, ClaI; C/A, ClaI-AccI <br /> resultant plasmids tested galK expression. PftsQ, ftsQ                            <span id='am-68' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-69' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-71'  ></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>hybrid</span> site; A, AccI. <br /> promoters. See Fig. 1 legend abbreviations. <br />  <br /> copies envA yielded transductants, did high                                 The nucleotide sequence obtained our <br /> frequency. In addition, transductants Kanr                                 results published results helped eluci- <br /> was shown linked Tetr, expected. In contrast,                                  date organization expression genes envA <br /> selection Tetr resulted transductants equal                                region. From sequence analysis, appear two <br /> frequency strains. However, strain                                  consensuslike promoters region, just up- <br /> two copies envA showed cotransduction Kanr, indicat-                                 stream envA upstream gene X. This agrees <br /> ing inactivation single copy envA gene                              promoter fusion ekperiments Sullivan and <br /> lethal (Table 2). These results indicate envA                                   Donachie (25), located promoters within <br /> essential gene.                                                                            region. By using runoff transcription experiments, they <br />                                                                                            able position promoters relative restriction sites. <br />                         DISCUSSION                                                         Our location envA promoter agrees exactly their <br />                                                                                            results; however, positioning gene X promoter is <br />   The envA gene of pleiotropic                                  different. From results, indicated pro- <br /> phenotypes known envA mutation loca-                                   moter located 295 bp upstream EcoRI site <br /> tion large cell envelope-cell division gene cluster                             (approximately position 2680), just upstream proximal <br /> occurring 2 min E. coli genetic map.                                             PstI site. However, observed PstI fragment <br />  <br />                                                                   TABLE 2. P1 transduction leu::TnJO kan <br />           Donor                                               Recipient                 Selected marker            Unselected marker               Cotransduction (%) <br /> NK6923 (leu::TnlO)                                    Kanr transformant                leu::TnlO                   Kans                                 53 <br />                                                         JC7623(A16-2) <br /> BL7623(X16-2) (leu::TnlO                              W3110                               leu::TnJO                   Kanr                                     Oa <br />   envA: :kan)                                                                              Kanr                         b <br /> BL7623(X16-2) (leu::TnlO                              W3110(X16-2)                        leu::TnJO                   Kanr                                 34 <br />   envA::kan)                                                                               Kanr                       leu::TnlO                            74 <br />   a500 Tetr colonies screened cotransduction Kanr. <br />  b-, No transductants. <br />  5414     BEALL AND LUTKENHAUS                                                                                                  J. BACTERIOL. <br />  <br /> containing region promoter activity fact                 gradient gels 35S <span id='am-56' about='rdfs:label' typeof='owl:Thing'>label</span> aid rapid DNA sequence <br /> had terminator activity. Therefore, believe position                 determination. Proc. Natl. Acad. Sci. USA 80:3963-3965. <br /> just downstream PstI site positions 2747 2774            4. Churchward, G., D. Berlin, Y. Nagamine. 1984. A pSC101- <br /> be better estimate gene X promoter. It appears               derived plasmid shows sequence homology other <br /> from sequence promoter serve                   commonly used cloning <span id='am-47' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="vectors" datatype="xsd:string"></span>vectors</span>. Gene 31:165-171. <br />                                                                          5. Grunstrom, T. S., S. Normark, K. Magnusson. 1980. Over- <br /> expression secA, does appear                     production outer membrane protein suppresses envA- <br /> terminator gene X secA.                                         mediated hyperpermeability. J. Bacteriol. 144:8848890. <br />    The open reading frame corresponding gene X indicated              6. Hawley, D. K., W. R. McClure. 1983. Compilation and <br /> in Fig. 2 longest possible open reading frame.                   analysis Escherichia coli promoter DNA sequences. Nucleic <br /> There&apos; possible GTG initiation codons phase                  Acids Res. 11:2237-2255. <br /> upstream open reading frame &apos;we indicated;                   7. Hopp, T. P., K. R. Woods. 1981. Prediction protein <br /> howeyer, favor ATG codon positions 2842 2844.                  antigenic determinants amino acid sequences. Proc. Natl. <br /> The major argument promoter sequence overlaps                  Acad. Sci. USA 78:3824-3828. <br />                                                                          8. Jones, C. A., I. B. Holland. 1984. Inactivation essential <br /> one GTG codons downstream one,                   division genes, ftsA, ftsZ, suppresses mutations sfiB, locus <br /> precluding presence transcript. The initiation                 mediating division inhibition SOS response E. coli. <br /> codon chosen possible in-frame initiation               EMBO J. 3:1181-1186. <br /> codon lacks easily recQgnizable Shine-                 9. Jones, C. A., 1. B. Holland. 1985. Role SfiB (FtsZ) <br /> Dalgarno sequence. We unable detect correspond-                   protein division inhibition SOS response E. coli: <br /> ing gene product maxicells (Fig. 3),                   FtsZ stabilizes inhibitor SfiA maxicells. Proc. Natl. Acad. <br /> proof actually translated.                                       Sci. USA 82:6045-6049. <br />    The location transcription terminator downstream             10. Kyte, J., R. F. Doolittle. 1982. A simple method for <br /> envA revealed use terminator-probe vector.                  displaying hydropathic character protein. J. Mol. Biol. <br />                                                                             157:105-132. <br /> Recent Si analysis confirmed location tran-              11. Lutkenhaus, J. F. 1983. Coupling DNA replication cell <br /> scription terminator region hyphenated dyad sym-                  division: sulB allele ftsZ. J. Bacteriol. 154:1339-1346. <br /> metry followed run T&apos;s located just downstream               12. Lutkenhaus, J. F., W. B. Donachie. 1979. Identification of <br /> the envA gene (Corton Lutkenhaus, unpublished). This                    fsA gene product. J. Bacteriol. 137:1088-1094. <br /> transcription terminator terminator discovered          13. Lutkenhaus, J. F., H. Wolf-Watz, W. D. Donachie. 1980. <br /> </body></html>