<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Natl. Acad. Sci. USA <br /> Vol. 96, pp. 3552&#8211;3555, March 1999 <br /> Biochemistry <br />  <br />  <br />  <br /> Replication fork assembly recombination intermediates is <br /> required bacterial growth <br /> JOING LIU&#8224;, LIEWEI XU&#8225;, STEVEN J. SANDLER&#167;,                     AND   KENNETH J. MARIANS&#8224;&#8225;&#182;i <br /> &#8224;Molecular Biology Graduate Program &#8225;Biochemistry Structural Biology Graduate Program, Cornell University Graduate School Medical Sciences, <br /> New York, NY 10021; &#167;Department Microbiology, University Massachusetts Amherst, Amherst, MA 01003; &#182;Molecular Biology Program, Memorial <br /> Sloan&#8211;Kettering <span id='am-11' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-12' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15'  ></span>Cancer</span> Center, New York, NY 10021 <br />  <br /> Edited Charles M. Radding, Yale University School Medicine, New Haven, CT, approved January 27, 1999 (received review <br /> November 16, 1998) <br />  <br />  <br /> ABSTRACT          <span id='am-1' about='obo:NCBITaxon_558101' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_577241 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_71526 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_116151 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="rdfs:label" content="Pria" datatype="xsd:string"></span>PriA</span>, 3* 3 5* DNA helicase, directs assem-                      (12). Single-stranded DNA-binding protein (SSB) purified <br /> bly primosome bacteriophage plasmid DNAs.                          according Minden Marians (13). The DNA polymerase <br /> Primosomes multienzyme replication machines con-                           III holoenzyme reconstituted Pol III* b <br /> tribute DNA-unwinding Okazaki fragment-                                subunit described Wu et al. (14) purified subunits <br /> priming functions replication fork. The role PriA                      described Marians et al. (15). <br /> chromosomal replication unclear. The phenotypes priA                             Preparation D Loop Template DNA. A 100-nt-long oli- <br /> null mutations suggest protein participates repli-                      gonucleotide having sequence 59-ACATACATAAAGG- <br /> cation restart recombination intermediates. We                         TGGCAACGCCATTCGAAATGAGCTCCATATGCTAG- <br /> that PriA promotes replication fork assembly D loop,                        CTAGGGAGGCCCCCGTCACAATCAATAGAAAATT- <br /> intermediate formed initiation homologous recom-                          CATATGGTTTACCAGCGC-39 annealed f1R408 <br /> bination. We DnaC810, encoded naturally                         viral DNA (16). The central 42 nt oligonucleotide are <br /> arising intergenic suppressor allele priA2::kan muta-                        nonhomologous template. The heteroduplex was <br /> tion, bypasses need PriA replication fork                            converted nicked form II DNA 42-nt-long bubble <br /> assembly D loops vitro. These findings underscore                         region incubation DNA polymerase III holoen- <br /> essentiality replication fork restart recombination inter-                    zyme presence SSB. During 2 min <span id='am-16' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-17'  ></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> <br /> mediates normal growth conditions bacteria.                                incubation, 29,39-dideoxynucleotides ddTTP ddATP at <br />                                                                                     concentrations 20-fold higher dTTP dATP were <br /> Primosome assembly bacteriophage fX174 DNA requires                              introduced reaction ensure complementary <br /> seven proteins: PriA, PriB, PriC, DnaT, DnaB, DnaC,                             strand synthesized extended further. After ex- <br /> DnaG (1). Only proteins, DnaB, replication                       traction phenol precipitation ethanol, DNA <br /> fork helicase (2), DnaC, DnaG, replication fork                             products purified electrophoresis nondena- <br /> primase, (3) known required cellular replication                     turing agarose gels. Complete form II bubble DNA was <br /> from chromosomal origin, oriC (4). However, strains                             recovered gel 59-32P-labeled minus-strand <br /> carrying priA null mutations constitutively induced                     oligonucleotide sequence 59-ATATAAAAGAAAC- <br /> SOS response (5). We suggested observation indi-                          GCAAAGACACCACGGAATAGTTTATTTT-39 then <br /> cated replication forks formed oriC did                            annealed form D loop form II template. The template <br /> complete replication chromosome rescued                        gel filtered Bio-Gel A5M remove both <br /> by PriA-dependent process (6). Subsequent genetic analyses                        unannealed oligonucleotide unincorporated [g-32P]ATP. <br /> demonstrated priA null strains acquired suppressor mu-                            DNA Replication Reaction. Reaction mixtures (12 ml) con- <br /> tations dnaC rapidly (7, 8) defective                           taining 50 mM Hepes&#8211;KOH (pH 8.0), 10 mM Mg(OAc)2, 10 <br /> homologous recombination (7, 8), repair UV damage (7, 8),                        mM DTT, 80 mM KCl, 200 mgyml BSA, 2 mM ATP, 40 mM <br /> double-strand break-repair (8), induced consti-                        dNTPs, 0.42 nM 32P-labeled form II D loop DNA template, 0.5 <br /> tutive stable DNA replication (9). These results led                         mM SSB, 225 nM DnaC, 30 nM PriA, PriB, PriC, DnaT, <br /> proposal absence PriA-catalyzed replication fork                        DnaB, DNA polymerase III holoenzyme incubated <br /> assembly recombination intermediates account                       37&#176;C 10 min. Reactions terminated addition <br /> of phenotypes (7, 10). This <span id='am-6' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-7' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10'  ></span>idea</span> supported                         EDTA 25 mM NaOH 50 mM analyzed by <br /> demonstration PriA specifically bind D loop DNA                          electrophoresis 2 Vycm 20 h room temperature <br /> (11). In report investigated PriA-catalyzed rep-                       horizontal 0.7% alkaline agarose gels using 30 mM <br /> lication fork assembly D loops using defined templates                        NaOH 2 mM EDTA electrophoresis buffer. Gels <br /> and purified DNA replication proteins.                                              neutralized, dried, autoradiographed. When used, <br />                                                                                     PriAK230R, PriAK230A, PriAK230D, DnaC810 were <br />                                                                                     present concentrations wild-type coun- <br />                MATERIALS AND METHODS                                                terparts. <br />    DNA Replication Proteins. A dnaC810 ORF con- <br /> structed means splicing overlap extension PCR                                                      RESULTS <br /> cloned NdeI site pET11C overexpression <br /> plasmid (Novagen). Overexpression purification                                 PriA-Directed Replication Fork Assembly D Loops. The <br />                                                                                     scheme preparing double-stranded circular template <br /> DnaC810 wild-type protein. PriA, PriB, PriC, <br />                                                                                     carrying D loop 39-invading strand given Fig. 1. <br /> DnaT, DnaB, DnaC purified described Marians <br />                                                                                     A replication fork assembled template use the <br /> The publication costs article defrayed page charge <br />                                                                                     This paper submitted directly (Track II) Proceedings office. <br /> payment. This article marked &#8216;&#8216;advertisement&#8217;&#8217; in <br />                                                                                     Abbreviation: SSB, single-stranded DNA-binding protein. <br /> accordance 18 U.S.C. &#167;1734 solely indicate fact.                       i <br />                                                                                       To reprint requests addressed. e-mail: k-marians@ <br /> PNAS available online www.pnas.org.                                             ski.mskcc.org. <br />  <br />                                                                              3552 <br />            Biochemistry: Liu et al.                                                        Proc. Natl. Acad. Sci. USA 96 (1999)            3553 <br />  <br />                                                                          exhibited absolute requirement primosomal <br />                                                                          proteins PriC. Omission protein resulted a <br />                                                                          decrease DNA synthesis one-third complete <br />                                                                          reaction. This result similar observed the <br />                                                                          past replication different templates (17). Some exten- <br />                                                                          sion invading strand holoenzyme be <br />                                                                          observed, suppressed presence PriA. If <br />                                                                          invading strand omitted reaction and <br />                                                                          [a-32P]dATP included, DNA replication be <br />                                                                          observed (<span id='am-45' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-46'  ></span><span id='am-47' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> shown). These observations suggested that <br />                                                                          PriA directing primosome assembly D loop. <br />                                                                             In experiment shown Fig. 2, extension invading <br />                                                                          strand result processes: (i) either <br />                                                                          assembly bona fide replication fork D loop followed <br />                                                                          elongation leading strand coupled unwinding of <br />                                                                          duplex DNA template (ii) uncoupled unwinding the <br />                                                                          template DNA leaving oligonucleotide annealed viral <br />                                                                          single-stranded DNA elongated primer- <br />    FIG. 1. Scheme preparation form II D loop template         extension reaction polymerase. We showed previously <br /> DNA. Pol III HE, DNA polymerase III holoenzyme; ss(c), single-           coupled replication fork action requires protein&#8211;protein <br /> stranded circular.                                                       interaction DnaB t subunit holoen- <br />                                                                          zyme (18). In presence interaction replication forks <br /> 39-OH invading strand primer leading-strand            rapidly nearly 1,000 ntysec, its <br /> synthesis. To focus event, radioactive <span id='am-44' about='rdfs:label' typeof='owl:Thing'>label</span>            absence, polymerase stuck slow-moving <br /> present replication reactions 59 end            helicase replication fork progression proceeds only <br /> invading strand oligonucleotide. Extension 39 end          30 ntysec. <br /> minus strand template inhibited                   Accordingly, assessed speed elongation the <br /> presence terminal dideoxy-nucleotide.                               invading strand presence absence t (Fig. 3). For <br />    Incubation D loop template, seven primosomal               experiments used holoenzyme reconstituted from <br /> proteins, SSB, DNA polymerase III holoenzyme,                individually purified subunits. Ten-second time points were <br /> ten-subunit replicase <span id='am-21' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-36'  ></span>Escherichia coli</span> (1), resulted exten-        taken start reaction, elongated <br /> sion invading strand oligonucleotide (42 nt) full-             products examined denaturing gels. Full-length ma- <br /> length template size (6.4 kb). This activity seen quite         terial observed presence t 10 sec, <br /> clearly reaction products analyzed dena-              60 sec full-length material observed <br /> turing alkaline agarose gel (Fig. 2). The efficiency              absence. This gives rate replication fork progression <br /> reaction varied, generally 15&#8211;30% (as determined                  presence t 600&#8211;700 ntysec, similar have <br /> PhosphorImager analysis) invading strand                 measured past replication systems (19). Thus, <br /> elongated length 10-min incubation. The reaction            conclude bona fide replication fork assembly was <br />  <br />  <br />  <br />  <br />                                                                             FIG. 3. Bona fide replication forks formed D loop <br />                                                                          template. Standard replication reaction mixtures increased size <br />                                                                          8-fold contained holoenzyme reconstituted purified subunits <br />                                                                          presence absence t. Aliquots (12 ml) were <br />   FIG. 2. Primosomal protein requirements D loop-dependent           removed start incubation (0 sec) 10-sec <br /> DNA replication. The indicated proteins omitted standard       intervals thereafter, reaction products analyzed as <br /> replication reaction mixtures. Temp, template only; HE alone, reaction   described text. The overexposed region the gel <br /> mixture contained DNA polymerase III holoenzyme SSB.        represents unelongated invading strand arises than <br /> The markers HindIII digest bacteriophage l DNA.                1% primer extended minute reaction. <br />  3554      Biochemistry: Liu et al.                                                    Proc. Natl. Acad. Sci. USA 96 (1999) <br />  <br /> occurring D loop template presence        recombination intermediates, DnaC810 able to <br /> primosomal proteins, SSB, holoenzyme.                         bypass requirement PriA recognize D loop and <br />    Replication Fork Assembly Does Not Require PriA                nucleate assembly primosome. Accordingly, tested <br /> Helicase Activity. Although PriA DNA helicase (20, 21),          DnaC810 direct transfer DnaB the <br /> all phenotypes priA null mutations suppressed        D loop template DNA (Fig. 5). <br /> by mutated priA alleles encode PriA proteins            In presence SSB holoenzyme, combina- <br /> longer ATPases DNA helicases, catalyze primo-            tion wild-type DnaC DnaB did support elongation <br /> some assembly (22). These mutations substitutions          invading strand D loop (Fig. 5). This result was <br /> invariant Lys Walker A box nucleotide-binding motif. If        expected, data presented Fig. 2 because <br /> the PriA-dependent replication fork assembly described           SSB binding DNA inhibits DnaB loading absence of <br /> were relevant happened cell, expect           DnaA oriC PriA, PriB, PriC, DnaT on <br /> these mutant proteins substitute fully wild-type PriA       bacteriophage fX174 DNA (1). On hand, DnaC810 <br /> the replication reaction. This proved case. Three           clearly able load DnaB D loop template <br /> mutant proteins, having K230R, K230A, K230D                   absence primosomal proteins, evinced by <br /> substitutions, tested. All supported replication        elongation invading strand length (Fig. 5). <br /> the D loop template greater extent wild-type            Thus, E176G substitution DnaC810 represents true <br /> protein (Fig. 4). We observed improved activity the <br />                                                                       gain-of-function mutation allows bypass DnaB <br /> mutant proteins systems (22) believe arises <br />                                                                       loading pathway involves PriA, PriB, PriC, DnaT. <br /> because remain bound site DNA binding, <br />                                                                          Interestingly, relative efficiencies replication <br /> providing better target wild-type protein, can <br />                                                                       reactions catalyzed presence DnaC810 DnaB <br /> move site DNA helicase activity. <br />                                                                       varied compared reaction catalyzed complete <br />    DnaC810 Bypasses Requirement PriA, PriB, PriC, <br /> and DnaT During Replication Fork Assembly. Strains carrying           set primosomal proteins. At 80 mM KCl experiment <br /> priA null mutations difficult grow. They rich         shown, DnaC810 reaction 5- 10-fold efficient. <br /> media-sensitive, form huge filaments, viability            However, 600 mM potassium glutamate (data shown), <br /> roughly 1y100th wild type (5, 7, 9). Suppressor           reaction catalyzed complete set proteins more <br /> mutations restore viability, ablate constitutive      efficient factor 2 (the efficiency DnaC810 <br /> induction SOS response defects homologous           reaction decreases, primosomal reaction <br /> recombination repair UV-damaged DNA, arise over-               increases). We assume difference reflects relative <br /> night transduction priA2::kan allele fresh          stability intermediate complexes formed the <br /> recipient cells (7, 9). These mutations map dnaC  <br /> </body></html>