<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Kinase requirements in human cells: II. Genetic <br /> interaction screens identify kinase requirements <br /> following HPV16 E7 expression in <span id='am-19' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-22'  ></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>cancer</span> cells <br />  <br />  <br /> <span id='am-1' about='obo:NCBITaxon_10566' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_151340 obo:NCBITaxon_35237 obo:NCBITaxon_10239 obo:NCBITaxon_333774 obo:NCBITaxon_173087'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Human Papilloma Virus" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Human papillomavirus" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="human papillomavirus HPV" datatype="xsd:string"></span>Human papillomavirus</span> (HPV) oncoproteins subvert cellular signaling            <span id='am-29' about='obo:NCBITaxon_267112' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_267089 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7505 obo:NCBITaxon_6970 obo:NCBITaxon_7504 obo:NCBITaxon_33341 obo:NCBITaxon_50557 obo:NCBITaxon_6072 obo:NCBITaxon_6960'><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-32'  ></span><span id='am-33' property="rdfs:label" content="Vates" datatype="xsd:string"></span>vates</span> the tumor suppressor protein p53 by accelerating its <br /> pathways, including kinase pathways, during the carcinogenic pro-             degradation through the proteasome, whereas the HPV E7 <br /> cess. To identify kinases targeted by the HPV16 E7 oncoprotein,               protein inactivates the retinoblastoma tumor suppressor protein <br /> shRNA kinase screens were performed in RKO colorectal carcinoma               pRb and the related p107/p130 <span id='am-16' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> members through a similar <br /> cell lines that differ only in their expression of HPV16 E7. Our screens      mechanism. In addition, high-risk HPV E6 and E7 oncoproteins <br /> identified kinases that were essential for the survival of RKO cells, but     have a number of additional cellular targets that contribute to <br /> not essential for RKO cells expressing HPV16 E7. These kinases include        their oncogenic activities (reviewed in ref. 4). <br /> CDK6, ERBB3, FYN, AAK1, and TSSK2. We show that, as predicted,                   The p53 and pRb tumor suppressor pathways are commonly <br /> CDK6 knockdown inhibits pRb phosphorylation and induces S-phase               rendered dysfunctional in most human carcinomas, and their <br /> depletion, thereby inhibiting cell viability. Knockdown of ERBB3, FYN, <br />                                                                               inactivation by high-risk HPV E6 and E7 proteins generates the <br /> AAK1, and TSSK2 induces a similar loss of cell viability through an <br />                                                                               functional equivalent of loss of p53 or pRb by mutation. The RKO <br /> unknown mechanism. Expression of the HPV16 E7 oncoprotein, <br />                                                                               colon carcinoma cell line used here has intact pRb and p53 tumor <br /> known to bind and degrade pRb, relieves the requirement of these <br />                                                                               suppressor pathways, and previous work has shown that pRb <br /> kinases. These studies demonstate that expression of a single onco- <br /> protein can dramatically alter kinase sensitivity in human cells. The <br />                                                                               functions are lost in the RKO cells that express HPV16 E7 (RKO <br /> shRNA screens used here perform analogously to genetic interaction            E7) (5). <br /> screens commonly used in genetically tractable organisms such as                 Although we anticipate many different nuances to emerge <br /> yeast, and thus represent an exciting method for unbiased identifi-           over time, the human cells used here behave much in the same <br /> cation of cellular signaling pathways targeted by cancer mutations.           way as yeast cells do in simple genetic screens. As reported in our <br />                                                                               preceding study (6), we found numerous differences in kinase <br /> essential kinases &#20841; shRNA screening &#20841; E7 oncoprotein &#20841;                        requirements when we analyzed a wide range of human tumor <br /> retinoblastoma tumor suppressor                                               cell lines. Although somewhat expected, it was interesting to note <br />                                                                               that kinase requirements were quite similar in cells from the <br />                                                                               same tissue type or in cell lines that differ only by expression of <br /> E   xpression of an oncoprotein and loss of a tumor suppressor <br />     protein are well known changes that provide essential steps <br /> in the development of a cancer. We understand in detail many <br />                                                                               a single protein. Specifically, when the kinase requirements of <br />                                                                               RKO colorectal carcinoma cells expressing the HPV16 E7 <br /> of the phenotypic changes that are induced by the activation of               oncoprotein were compared with the RKO parental cells, we <br /> well studied oncogenes or the loss of tumor suppressor genes                  observed that only a distinct subset of kinases were differentially <br /> (reviewed in ref. 1). In many cases, molecular consequences of                required (6). These findings prompted us to characterize in <br /> these tumorigenic events have been determined, but such studies               greater detail the differential kinase requirements in cancer cells <br /> are often limited to recording changes primarily in those path-               that differed only in expression of the HPV16 E7 oncoprotein. <br /> ways in which these proteins have been shown to act. It has been              CDK6, ERBB3, FYN, AAK1, and TSSK2 were identified as <br /> difficult to comprehensively determine the molecular alterations              essential kinases in RKO cells but were dispensable for viability <br /> that are induced by specific oncogenic events. One potential                  of RKO E7 cells. The differential requirement of CDK6 in RKO <br /> approach to determine how cells are altered by expression of an               versus RKO E7 cells most likely reflects the ability of HPV16 E7 <br /> oncoprotein or loss of a tumor suppressor protein in an unbiased,             to target the pRb tumor suppressor protein for degradation. <br /> more comprehensive way, is the use of comparative RNAi                        ERBB3, FYN, AAK1, and TSSK2 may represent previously <br /> screens. Such RNAi screens can be perceived as genetic inter-                 <span id='am-53' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-54' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-68'  ></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span>unidentified</span> essential regulators of pRb activity in RKO cells, or <br /> action screens in which the action of the RNAi knockdown is                   these kinases may be rendered dispensable for RKO E7 viability <br /> analyzed in combination with the action of the cancer-promoting               as consequence of HPV16 E7 targeting other signal transduction <br /> event. Here, we report a proof-of-principle genetic interaction               pathways in these cells. Our results suggest a general method to <br /> screen in human cells in which the action of the human papil-                 study the changes in cellular signaling networks induced by a <br /> lomavirus (HPV) 16 E7 oncoprotein is studied in conjunction                   cancer-promoting event, specifically the expression of a single <br /> with a kinase loss-of-function RNAi screen. <br />    High-risk HPVs are causative agents of cervical carcinoma, a <br /> leading cause of death in women, worldwide. A number of other                 Author contributions: A.B., E.H., K.M., and D.A.G. designed research; A.B., W.L., M.G., and <br /> anogenital tract carcinomas, and a fraction of oropharyngeal                  D.A.G. performed research; A.B., J.P., and D.A.G. contributed new reagents/analytic tools; <br /> carcinomas, are also associated with high-risk HPV infections                 A.B., W.L., and D.A.G. analyzed <span id='am-38' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span>; and A.B., E.H., K.M., and D.A.G. wrote the paper. <br /> (reviewed in (2, 3)). High-risk HPV E6 and E7 proteins are                    The authors declare no conflict of interest. <br /> consistently expressed in cervical cancers, and their expression              &#8225;To   whom correspondence should be addressed. E-mail: eharlow@hms.harvard.edu. <br /> is necessary for induction and maintenance of the transformed                 <span id='am-24' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-25'  ></span><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> article contains supporting information online at www.pnas.org/cgi/content/full/ <br /> state. The functional roles of the HPV E6 and E7 proteins have                0806195105/DCSupplemental. <br /> been studied extensively. The high-risk HPV E6 protein inacti-                &#169; 2008 by The National Academy of Sciences of the USA <br />  <br />  <br />  <br /> 16478 &#8211;16483 &#20841; PNAS &#20841; October 28, 2008 &#20841; vol. 105 &#20841; no. 43                                                       www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.0806195105 <br />                                                                                       knockdown in each of the cell lines. Differential kinase require- <br />                                                                                      ments resulting from expression of the HPV16 E7 oncoprotein <br />                                                                                      in RKO cells can be distinguished. Knockdowns of 19 unique <br />                                                                                      kinases induced a 30% or greater differential in cell survival <br />                                                                                      between RKO and RKO E7 cells (see Fig. 1). A complete list <br />                                                                                      detailing percentage of loss of cell viability and the resulting <br />                                                                                      survival differentials in RKO versus RKO E7 cells can be found <br />                                                                                      in supporting information (SI) Table S1. These results demon- <br />                                                                                      strate that shRNA screens can be used to determine specific <br />                                                                                      differences in cellular kinase requirements induced by expres- <br />                                                                                      sion of the HPV16 E7 oncoprotein. <br />  <br />                                                                                      HPV16 E7 Expression Rescues the Lethality of Kinase Loss. To verify <br />                                                                                      and explore the differential kinase requirements identified in <br />                                                                                      our initial screen, we chose to further investigate 8 of the 19 <br />                                                                                      kinases exhibiting the largest survival differentials in RKO <br />                                                                                      versus RKO E7 cells. In addition, three kinases exhibiting no <br />                                                                                      significant survival differentials between the two cell popula- <br />                                                                                      tions were included as negative controls. First, to minimize the <br />                                                                                      chance that the cell survival differentials were the result of off <br />                                                                                      target effects, we tested four to five shRNAs from the original <br />                                                                                      RNAi Consortium library targeting each of the selected kinases. <br />                                                                                      For five of the eight kinases tested, we identified multiple <br />                                                                                      shRNAs that resulted in a &#11022;30% survival differential. These <br />                                                                                      kinases include CDK6, ERBB3, FYN, AAK1, and TSSK2. The <br />                                                                                      remaining three kinases, TLK1, JNK3, and Cke1, did not pass <br />                                                                                      this validation step and were consequently not included for <br />                                                                                      further analysis. None of the available shRNAs targeting the <br />                                                                                      three control kinases exhibited differential killing between RKO <br />                                                                                      and RKO E7 cells (data not shown). <br />                                                                                         Second, we performed a time course of kinase knockdown <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 APPLIED BIOLOGICAL <br />                                                                                      using multiple shRNAs targeting each of the five kinases that <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      SCIENCES <br /> Fig. 1. Expression of the HPV16 E7 oncoproteins alters the essential kinase          passed the first validation step. Cells were assayed for survival <br /> signature in RKO colorectal carcinoma cells. RKO and RKO E7 cells were               using Alamar blue at 1, 3, 5, and 8 days after infection with the <br /> infected with each of 100 lentiviral shRNA expression <span id='am-34' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-35' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> targeting 80           shRNA lentiviruses, and were subsequently fixed and stained <br /> unique kinases and were subsequently assessed for cell viability using Alamar        with crystal violet for microscopic evaluation. The 5-day time- <br /> blue. Percentage of viability was determined after normalization to a scram-         point was determined to be optimal for determining cell survival <br /> bled shRNA control vector. Results from two independent shRNA screens, each <br />                                                                                      differentials, as Alamar blue readings exhibited maximal differ- <br /> performed in quadruplicate, are represented here in a heat map. Each column <br /> represents a cell line, each row represents an individual hairpin (hairpin names <br />                                                                                      ences in signal without overcrowding of cells in the well. Rep- <br /> are given on the right). Green indicates the most cell viability and red indicates   resentative photomicrographs of the cells 5 days after infection <br /> the least cell viability. Data are sorted using unsupervised clustering and          with two shRNAs targeting each of the five kinases are shown <br /> Euclidean distance.                                                                  and demonstrate cell survival differentials consistent with the <br />                                                                                      Alamar blue measurements (Fig. 2A). Each of the shRNAs <br />                                                                                      dramatically inhibited the growth of RKO cells without markedly <br /> oncoprotein, and establishes a reliable method to do functional                      affecting the proliferation of RKO E7 cells. <br /> pathway mapping in human cells.                                                         Next, we used a range of viral multiplicities of infection to deliver <br />                                                                                      varying doses of shRNAs to the cells. Infections were performed <br /> Results                                                                              using dilutions of the viral supernatants, from 1.25 to 20 &#9262;l, and cells <br /> Expression of the HPV16 E7 Oncoprotein Induces Differential Kinase                   were assayed by Alamar blue for cell viability 5 days after infection. <br /> Requirements in Cancer Cells. To determine differential kinase                       Percentage of reduction in cell viability resulting from kinase <br /> requirements induced by expression of the HPV16 E7 onco-                             knockdown was calculated after normalization to a scrambled <br /> protein, we performed essential kinase screens in both the RKO                       shRNA control. Cell viability values are graphed for each shRNA <br /> parental cell line and RKO cells stably expressing HPV16 E7                          at multiple viral concentrations. Red lines with squares indicate <br /> (RKO E7) (5). Both cell lines were infected with lentiviral                          percentage of loss of cell viability in RKO E7 cells and blue lines <br /> vectors expressing 100 individual shRNAs targeting 80 unique                         with circles indicate percentage of loss of cell viability in RKO <br /> kinases. These correspond to the top 100 kinase &#8216;&#8216;hits&#8217;&#8217; that are                    control cells. As in Fig. 2 A, the results with two shRNAs targeting <br /> essential for viability of HeLa and 293T cells, as identified in our                 different portions of each kinase are shown (Fig. 2B). Each of the <br /> primary screen (6). Cells were subsequently assayed for viability                    shRNAs inhibited the proliferation of RKO control cells by &#11022;30% <br /> 5 days after infection. Percentage of cell viability was determined                  over that of RKO E7 cells. <br /> using Alamar blue, a dye that measures mitochondrial fitness (7,                        As our final validation step, we assessed kinase mRNA <br /> 8). Values were calculated as a percentage of loss of viability,                     knockdown 60 h after infection by RT-PCR, normalizing all <br /> normalized to a scrambled control shRNA. Two independent                             values to GAPDH mRNA levels. As a control, we included <br /> screens were performed, each in quadruplicate, with a resulting                      shRNAs against ALK, a kinase that did not demonstrate a <br /> correlation coefficient between screens of 0.95. Kinase signa-                       significant percentage of survival differential between RKO and <br /> tures from both screens are displayed in a heat map; green                           RKO E7 cells. TSSK2 mRNA levels could not be adequately <br /> denotes the greatest cell viability, and red denotes the least cell                  assessed, because we could not identify effective primer pairs. <br /> viability (Fig. 1). Each column represents data from a specific                      Percentage of RNA knockdown was calculated and is shown in <br /> cell line, and each row illustrates results from individual kinase                   Fig. 2C. GAPDH levels were also measured as a report of cell <br />  <br /> Baldwin et al.                                                                                         PNAS &#20841; October 28, 2008 &#20841; vol. 105 &#20841; no. 43 &#20841; 16479 <br />  Fig. 2. HPV16 E7 expression abrogates the requirement of CDK6, ERBB3, FYN, AAK1, and TSSK2 for viability of RKO cells. (A) Photomicrographs of RKO and <br /> RKO E7 cells after infection with two different shRNA expression vectors targeting each kinase. Cells were fixed and stained with crystal violet 5 days after <br /> infection. (B) Viability of RKO and RKO E7 cells after infection with different concentrations of shRNA expression vectors. Cells were infected with 1.25 to 20 &#9262;l <br /> of the corresponding shRNA expression vector and assayed by Alamar blue for cell survival 5 days after infection. Percentage of loss of cell viability resulting from <br /> kinase knockdown is graphed. Values were normalized to a scrambled hairpin control. Red lines with squares indicate values from RKO E7 cells, and blue lines <br /> with circles indicate values from RKO control cells. Results with two hairpins targeting different portions of each kinase are shown. (C) Efficiency of kinase mRNA <br /> knockdown in RKO and RKO E7 cells. Quantitative RT PCR analysis was used to determine expression of each kinase 60 h after infection with the corresponding <br /> shRNA expression vectors. Values shown are normalized GAPDH levels. <br />  <br />  <br />  <br /> fitness, and the relative mRNA levels of each of the kinases in                     where pRb function is compromised because of HPV16 E7- <br /> RKO and RKO E7 cells were also measured (Fig. S1). GAPDH                            mediated degradation (5). Given the differential requirement of <br /> mRNA levels did not change significantly after viral infection,                     CDK6 in RKO and RKO E7 cells, we postulated that upon <br /> arguing that the change in kinase mRNA levels do not adversely                      CDK6 knockdown, RKO cells would undergo G1/S cell cycle <br /> affect overall cellular viability (Fig. S1 A). As for the overall                   arrest, whereas RKO E7 cells may continue to progress through <br /> endogenous mRNA levels of five kinases tested in RKO cells,                         the cell cycle. To test this hypothesis, cell cycle profiles were <br /> ALK, CDK6, and FYN kinase levels did not vary greatly with E7                       determined upon CDK6 knockdown in each of the cell lines. We <br /> expression. However, relative endogenous mRNA levels of                             also tested shRNAs targeting ERBB3 and included shRNAs <br /> AAK1 and ERBB3 are approximately fivefold less with E7                              targeting JNK3, which do not differentially affect viability of <br /> expression (see Fig. S1B). Taken together, these experiments                        RKO and RKO E7 cells, as a negative control. Cells were <br /> demonstrate that HPV16 E7 can complement the requirement                            collected 96 h after infection for FACS analysis, and the resulting <br /> of CDK6, ERBB3, FYN, AAK1, and TSSK2 in RKO colorectal                              profiles are shown (Fig. 3). For RKO cells expressing the E7 <br /> carcinoma cells.                                                                    oncoprotein, no change in the cell cycle profiles was detected <br />                                                                                     upon CDK6 or ERBB3 knockdown (see Fig. 3 Lower). In <br /> S-Phase Depletion Induced by Kinase Knockdown is Prevented by                       contrast, however, we observed a striking reduction of S-phase <br /> Expression of HPV16 E7. Because the action of CDK6 on the cell                      upon CDK6 and ERBB3 knockdown in RKO cells. As expected, <br /> cycle through phosphorylation of pRb has been extensively                           JNK3 knockdown did not affect the cell cycle profiles of RKO <br /> studied and is well known (reviewed in ref. 9), we sought to                        or RKO E7 cells (see Fig. 3). These results indicate that <br /> compare the effects of CDK6 knockdown on the cell cycle in                          expression of the HPV16 E7 oncoprotein abrogates cell cycle <br /> RKO cells, which contain functional pRb, and RKO E7 cells,                          perturbations induced by loss of CDK6 and ERBB3. <br />  <br /> 16480 &#20841; www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.0806195105                                                                                                   Baldwin et al. <br />                                                                                                            Fig. 3. HPV16 E7 expression in RKO cells <br />                                                                                                           abrogates S-phase depletion induced by <br />                                                                                                           CDK6 and ERBB3 knockdown. RKO and RKO <br />                                                                                                           E7 cells were infected with two shRNA ex- <br />                                                                                                           pression vectors targeting CDK6, ERBB3, and <br />                                                                                                           JNK3 or a scrambled shRNA control vector. <br />                                                                                                           Cells were harvested 96 h after infection, <br />                                                                                                           stained with propidium iodide, and DNA con- <br />                                                                                                           tent was analyzed by FACS. Percentage of <br />                                                                                                           cells in S-phase is given. Data shown is repre- <br />                                                                                                           sentative of three independent experiments. <br />  <br />  <br />  <br /> CDK6 Depletion Reduces pRb Phosphorylation. Given the striking         panels of siRNAs or shRNAs, provide approaches that closely <br /> reduction in S-phase in response to CDK6 knockdown in RKO              resemble genetic screens for mammalian tissue culture cells (11, <br /> cells, and CDK6&#8217;s known function to phosphorylate pRb (re-             12). In the studies reported here, we have performed the equivalent <br /> viewed in ref. 9), we sought to determine whether CDK6                 of a genetic interaction screen and determined how the expression <br /> knockdown altered phospho-pRb levels in RKO cells. A <span id='am-48' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-49' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-52'  ></span>phos</span>-             of the HPV16 E7 oncoprotein has changed the patterns of kinase <br /> pho-specific antibody was used to analyze pRb phosphorylation          requirements of a colon carcinoma cell line. In our preceding study <br /> at serines 807 and 811, which had previously been identified as        (6), we report the comparison of essential kinase signatures from <br /> CDK4/CDK6-preferred phosphorylation sites (10). RKO and <br /> RKO E7 cells were infected with an shRNAs targeting CDK6 <br />  <br />  <br />  <br />  <br />                                                                                                                                                             APPLIED BIOLOGICAL <br /> and a scrambled shRNA as a control. Cells were lysed for <br /> Western analysis 96 h after infection. Samples were analyzed for <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  SCIENCES <br /> total pRb, phosphoserine 807/811-pRb, tubulin, CDK6, and E7 <br /> levels using a Kodak Imager (Fig. 4A). Total pRb and phospho- <br /> serine 807/811-pRb levels were normalized to the tubulin loading <br /> control, and relative protein levels after CDK6 knockdown were <br /> calculated as a percentage of control. Phosphoserine 807/811- <br /> pRb levels (P-Rb, gray bars) as a fraction of total pRb levels <br /> (Total Rb, black bars) were calculated and are indicated (Fig. <br /> 4B). The ratios of phosphoserine 807/811-pRb were as expected: <br /> CDK6 knockdown reduces phosphoserine 807/811-pRb levels in <br /> RKO cells. <br />  <br /> Discussion <br /> Mammalian cell biology has been limited by the absence of <br /> methods to perform genetic screens that have been immensely <br /> powerful in genetically tractable model systems. Components of <br /> mammalian signal transduction pathways are commonly identi- <br /> fied by any number of approaches, including protein purification, <br /> homologies from model systems, and reverse genetics ap- <br /> proaches, where interesting genes are identified from cloning of <br /> disease genes. Proteins identified using these approaches can be <br /> linked to molecular machines of all sizes by identifying associated <br /> proteins and integration into signaling pathways by finding <br /> immediate upstream and downstream partners. Additionally, <br /> their roles in contributing to general cellular phenotypes can be <br /> inferred either by analyzing how they affect cell behavior when <br /> their levels or activities are altered, or by determining their <br /> subcellular localization in comparison with other proteins of <br /> interest. These methods have provided the basis of our views of <br /> modern mammalian cell biology. However, it would be a pow- <br /> erful addition to these tools to be able to use genetic-type screens   Fig. 4. HPV16 E7 expression in RKO cells prevents reduction of phsopho pRb <br />                                                                        levels upon CDK6 knockdown. (A) Immunoblot analysis of phospho pRb (Ser <br /> in mammalian cells. When done to saturation, genetic screens           807/811) and total pRb levels in RKO colorectal carcinoma cells 72 h after <br /> can provide a list that approaches an unbiased and comprehen-          infection with CDK6 specific shRNA expression vectors or a scrambled shRNA <br /> sive compendium of genes that play roles in a particular process       control vector. CDK6, E7 and &#9251;-tubulin levels are also shown. (B) Relative levels <br /> or that affect the action of other proteins. We and others have        of total pRb (black) versus phospho-pRb (Ser 807/811) (gray) upon CDK6 <br /> commented previously that RNAi screens, done with various              knockdown were quantified using a Kodak Imager. <br />  <br />  <br /> Baldwin et al.                                                                             PNAS &#20841; October 28, 2008 &#20841; vol. 105 &#20841; no. 43 &#20841; 16481 <br />  tumor cell lines that carry various mutations acquired by the cells     mRNA in RKO E7 may reflect the fact that their expression is <br /> during carcinogenesis. Initially, we expected that these experiments    not essential for viability of these cells. Even though the steady <br /> might enable us to correlate specific alterations in essential kinase   state mRNA levels of AAK1 and ERBB3 are reduced in RKO <br /> signatures to defined oncogenic mutations common among these            E7 cells, they are present at levels sufficient to observe a further <br /> cancer lines. However, the resulting kinase signatures were highly      decrease of each upon specific shRNA expression (see Fig. 2C). <br /> complex, because they presumably reflect the many different                CDK6&#8217;s role in directly phosphorylating pRb at the G1/S phase <br /> cellular variations acquired during tumor development and adap-         transition of the cell division cycle is well established (reviewed in <br /> tation to culture. However, when we compared cell lines that            ref. 9); therefore, we tested whether CDK6 knockdown in RKO <br /> differed only by expression of a single oncoprotein, HPV16 E7, the      cells affected cell cycle progression. As predicted, CDK6 knock- <br /> differences in kinase requirements were fewer and relatively            down blocked RKO cells from entering S phase with a concomitant <br /> straightforward to discern (6).                                         decrease in pRb phosphorylation on serine residues 807 and 811 <br />    In the work reported here, we compared an established                (see Figs. 3 and 4). These results argue that CDK6 has an obligatory <br /> colorectal carcinoma cell line, RKO, to a derivative that ex-           function in the regulation of G1/S transition in RKO cells and that <br /> presses the HPV16 E7 oncoprotein, RKO E7. From a total of 80            this function is abrogated in RKO E7 cells, presumably because of <br /> kinases that were evaluated in this study, our initial screen           HPV16 E7-mediated degradation of pRb family members. The <br /> tentatively identified 19 kinases essential for viability of the        exquisite sensitivity of RKO cells to CDK6 knockdown is somewhat <br /> parental RKO line but dispensable for viability of RKO E7 cells.        surprising because CDK4 is generally thought to provide redun- <br /> Somewhat surprisingly, this screen did not identify any kinases         dancy for CDK6 in pRb phosphorylation. However, evidence that <br /> that showed a synthetic lethal interaction with HPV16 E7                these kinases are not entirely functionally redundant and may have <br /> expression in RKO cells. These results suggest that in this cellular    distinct and overlapping functions has been reported in previous <br /> background, subversion of cellular signaling pathways targeted          studies (13&#8211;17). The results described here also support the notion <br /> by HPV16 E7, such as the pRb pathway, might not induce new              that CDK4 must provide only limited redundancy for CDK6 loss in <br /> sensitive points but rather relieve existing rate-limiting steps.       RKO cells. <br /> This would be consistent with pRb&#8217;s role as an inhibitor of                Similar to what we observed for CDK6, ERBB3 knockdown <br /> proliferation and pRb family member&#8217;s ability to act on a number        causes S-phase depletion in RKO cells, but to a much lesser <br /> of key transcription steps, making it unlikely that any one event       extent in RKO E7 cells (see Fig. 3). ERBB3 is a member of the <br /> might be sufficient to overcome the large number of important           EGF receptor tyrosine kinase family and is not directly involved <br /> pRb/p107/p130 regulated events.                                         in pRb phosphorylation. Nonetheless, expression of cyclin D1, a <br />    Because the 80 kinases evaluated in this study were originally       regulatory subunit of CDK4 and CDK6, is regulated by growth <br /> identified as essential for the viability of HeLa or 293T cells, one    factor signaling (18), and it is therefore conceivable that ERBB3 <br /> might expect that HeLa and RKOE7 cells would respond                    knockdown in RKO cells may indirectly affect CDK6 activity and <br /> similarly in essential kinase screens, because they are both cancer     thus pRb phosphorylation. Alternatively, ERBB3 may also <br /> cell lines that express high-risk HPV E7. Although these cell lines     modulate other signal transduction pathways involved in the <br /> did respond similarly to a majority of the kinases tested, we did       G1/S cell cycle transition that are abrogated by HPV16 E7 <br /> observe a subset of kinases whose essentiality was altered by           expression through mechanisms other than pRb inactivation. <br /> HPV16 E7 expression as compared with RKO parental cells and             The latter possibility is supported by the finding that ERBB3 <br /> HeLa cells. The differences in kinase requirements we have              knockdown results in a dramatic increase of cells with subG1 <br /> observed in RKO E7 versus HeLa cells might be because there             DNA content, suggestive of apoptotic cell death, whereas CDK6 <br /> is no E6 expression in the RKO E7 cells, and we are thus                knockdown did not have the same effect. <br /> observing the effects solely a result of E7 expression. Alterna-           HPV16 E7 expression in RKO cells can likewise rescue loss of <br /> tively, these differences in kinase essentiality may be caused by       viability as a consequence of FYN, AAK1, and TSSK2 knock- <br /> differences in the genetic background of the cells. It is important     down. However, we do not know whether this is related to ability <br /> to note that kinases targeted by the E7 oncoprotein in RKO cells        of HPV16 E7 to target pRb family members or whether it <br /> might be very different from those identified when assayed in           reflects inactivation of other cellular HPV16 E7 targets (Fig. 5). <br /> other cancer cell lines or in primary epithelial cells.                    The proof-of-concept experiments that are reported here <br />    From the initial list of 19 kinases that were tentatively            provide evidence that unbiased shRNA screens can be produc- <br /> identified in the primary screen, a subset of 8 kinases that showed     tively used to determine how cellular signal transduction path- <br /> the most dramatic survival differentials in RKO versus RKO E7           ways change during carcinogenesis. Moreover, we demonstrate <br /> cells were chosen for more detailed follow-up. For each of these        that shRNA screens can reveal changes in kinase requirements <br /> kinases, we tested multiple shRNAs targeting different regions          resulting from expression of a single oncoprotein. Previous <br /> of the respective mRNA. To qualify for more detailed studies,           methods have been effective in identifying cancer mutations and <br /> similar cell survival differentials had to be detected with two or      characterizing the corresponding oncoproteins or tumor sup- <br /> more shRNAs targeting each kinase. Of the eight kinases that            pressor proteins. The approach exemplified here provides a <br /> were tested, five kinases, CDK6, ERBB3, AAK1, TSSK2, and                practical method to identify major cellular changes that occur <br /> FYN, continued to score with multiple shRNAs, demonstrating             well beyond the immediate biochemical alterations caused by the <br /> consistent and robust responses similar to what we observed in          cancer mutations themselves. Genetic interaction screens per- <br /> our initial screen. Those shRNAs that inhibited the proliferation       formed in closely related cellular backgrounds raise the possi- <br /> of RKO cells also showed concomitant decreases in target kinase         bility of identifying essential proteins whose requirements are <br /> mRNA or protein levels (see Figs. 2 A and C and 4A). It is              induced by the action of a distantly acting oncoprotein or tumor <br /> interesting to note that the relative endogenous mRNA levels of         suppressor. Such proteins would be difficult to identify through <br /> AAK1 and ERBB3 are reduced approximately fivefold in RKO                standard biochemical methods. It will be interesting to deter- <br /> E7 cells when compared with RKO control cells. Conversely,              mine how these newly identified protein kinases are wired to the <br /> endogenous mRNA levels of CDK6 and FYN, in addition to                  molecular targets of HPV16 E7 and whether they may be useful <br /> CDK6 protein levels, do not significantly vary between the two          as therapeutic cancer targets. <br /> cell lines (see Fig. 4A and Fig. S1B). These results demonstrate <br /> that the decreased sensitivity to knockdown of these kinases is         Materials and Methods <br /> not a consequence of high-level expression in RKO E7 cells.             Tissue Culture. RKO cells stably expressing the pcDNA3 control vector or HPV16 <br /> Indeed, the observed reduced expression of AAK1 and ERBB3               E7 have been described in ref. 5 and were maintained in McCoy&#8217;s medium <br />  <br />  <br /> 16482 &#20841; www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.0806195105                                                                                    Baldwin et al. <br />                                                                                                from 6-well plates 72 h after infection for Western analysis and 96 h after <br />                                                                                               infection for cell cycle analysis. <br />  <br />                                                                                               Alamar Blue Assay. Five days after infection, media was removed from each <br />                                                                                               well of the 96-well plates, and 100-&#9262;l Alamar blue reagent (Biosource/ <br />                                                                                               Invitrogen, diluted 10-fold in supplemented McCoy&#8217;s medium) was added to <br />                                                                                               each well (7, 8). Plates were incubated for 2 to 4 h at 37&#176;C then read at 595 nm <br />                                                                                               on a microtiter well plate reader, Spectrafluor Plus (Tecan). Raw values were <br />                                                                                               normalized to a scrambled control. <br />  <br />                                                                                               Crystal Violet Staining. Five days after infection, media was removed from each <br />                                                                                               well of the 96-well plates. Cells were washed with PBS and fixed for 20 min <br />                                                                                               with 10% acetic acid and 10% methanol. Cells were washed once with PBS, <br />                                                                                               incubated for 30 min with 0.4% crystal violet in 20% ethanol, and were <br />                                                                                               subsequently washed with PBS twice. Phase contrast Images were acquired <br />                                                                                               with an inverted microscope at a magnification of 100&#11003; (Nikon). <br />  <br />                                                                                               Cell Cycle Analysis. Infected cells were trypsinized and centrifuged with their <br /> Fig. 5. Kinases required for survival of RKO cells that may act upstream of Rb.               media, then washed with PBS. After washing, cells were resuspended in 0.5 ml <br /> CDK6, ERBB3, FYN, AAK1, and TSSK2 are necessary for the survival of RKO                       PBS. Cells were fixed by adding 0.5 ml of 80% ethanol and were incubated at <br /> colorectal carcinoma cells. This requirement is relieved by expression of the                 4&#176;C overnight. After fixation, cells were washed with PBS and resuspended in <br /> HPV16 E7 oncoprotein and subsequent reduction of Rb levels. CDK6 loss                         200 &#9262;l of 0.5 mg/ml RNAseA in PBS. After incubation at 37&#176;C for 30 min, 200 <br /> inhibits phosphorylation of Rb, induces S-phase loss, and decreases cell sur-                 &#9262;l of 70-&#9262;M propidium iodide in 38-mM sodium citrate, pH 7.3, was added and <br /> vival. Knockdown of ERBB3, FYN, AAK1, and TSSK2 results in a similar loss of                  incubated at 37&#176;C for an additional 30 min. Cells were stored at 4&#176;C overnight <br /> cell survival through an unknown mechanism. These kinases may function                        and were analyzed on FACS Calibur (Becton Dickinson). <br /> through direct or indirect action on the Rb protein or alternatively through <br /> other cell cycle control pathways that are subverted by HPV16 E7 expression.                  Western Analysis. Cells were scraped in RIPA buffer containing protease inhibitor <br />                                                                                               mixture (Roche) and 25 mM NEM. Fifty-microliter samples were prepared in 5&#11003; <br />                                                                                               SDS loading buffer, resolved on 4 to 12% Nupage bis-Tris gradient gels (Invitro- <br /> supplemented with 10% calf serum, 100 units/ml penicillin, 100 &#9262;g/ml strep-                   gen), and transferred onto PVDF membrane (Perkin&#8211;Elmer). Antibodies: ErbB3 <br /> tomycin, and 0.5 mg/ml G418 (19).                                                             (1B2 from Cell Signaling), Rb (M-153 from Santa Cruz), phospho-Rb (Ser-807/811 <br />                                                                                               from Cell Signaling), alpha-tubulin (Sigma), CDK6 (DCS83 from Cell Signaling), <br /> 100 &#8216;&#8216;Hits&#8217;&#8217; DNA and Virus Production. High quality DNA preparations for 100                  HPV 16 E7 (a mixture of 8C9 from Zymed and ED17 from Santa Cruz), antirabbit <br />                                                                                               and antimouse HRP conjugates (GE Healthcare). Images were captured and <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                 APPLIED BIOLOGICAL <br /> &#8216;&#8216;hits&#8217;&#8217; were obtained using a large-scale plasmid purification kit (Qiagen). For <br /> high-throughput lentiviral production in a 96-well format, 293T packaging                     quantified on a Kodak 4000R digital imager. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                      SCIENCES <br /> cells were cotransfected with shRNA-encoding replication deficient viral vec- <br /> tors and the necessary helper <span id='am-43' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-44' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span> for virus production. The virus was                    RT-PCR. Lentiviral vectors expressing specific shRNAs were produced and RKO <br /> pseudotyped with the envelope glycoprotein from <span id='am-8' about='obo:NCBITaxon_11276' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_11157 obo:NCBITaxon_10239 obo:NCBITaxon_439488 obo:NCBITaxon_11270 obo:NCBITaxon_11271 obo:NCBITaxon_35301 obo:NCBITaxon_327822'><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="vesicular stomatitis virus, VSV" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasAlternativeId" content="NCBITaxon:12872" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Vesicular stomatitis virus" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="vesicular stomatitis virus VSV" datatype="xsd:string"></span>vesicular stomatitis virus</span>                    and RKO E7 cells were infected as described above in a 96-well format. RNA <br /> (VSV-g), as previously described (20 &#8211;22).                                                    was isolated using the RNeasy 96 Kit (Qiagen) at 36 and 60 h time points after <br />                                                                                               infection. cDNA synthesis was performed using the SuperScript III Reverse <br /> Infections in 96-well and 6-well Formats. For cell survival assays, RKO and RKO               Transcriptase Kit (Invitrogen) with 5 &#9262;l of RNA. Primer-probe sets were de- <br /> E7 cell lines were seeded into four 96-well plates each at 2,200 cells per well               signed using the Assay Design Center from Roche. Gene specific primers were <br /> using in a final volume of 50 &#9262;l per well. For FACS and Western analysis, cells               obtained from Integrated DNA Technologies, and gene specific probes were <br />                                                                                               obtained from Roche. Real-time PCR reactions were performed in duplicate in <br /> were seeded in 6-well dishes at 30% confluency in a final volume of 1.5 ml per <br />                                                                                               a 96-well format using the LightCycler 480 (Roche). <br /> well. Twenty-four hours after plating, viral supernatants were added (ranging <br /> in amounts from 1.25 to 20 &#9262;l for the 96-well format, and 150 to 250 &#9262;l for the <br /> 6-well format), in the presence of 8 &#9262;g/ml Polybrene. Plates were centrifuged                 <br /> at 1,178 &#11003; g for 22 min. Media was replaced 12 h after infection. For cell <br />                                                                                              <br /> survival assays, 2 &#9262;g/ml puromycin was added to two of the four 96-well plates.             <br /> Cells were harvested 5 days after infection from 96-well plates for Alamar blue               <br /> measurements and for crystal violet image acquisition. Cells were harvested                   <br /> </body></html>