<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Systems Biology 4; Article number 174; doi:10.1038/msb.2008.10 <br /> Citation: Molecular Systems Biology 4:174 <br /> &amp; 2008 EMBO Nature Publishing Group All rights reserved 1744-4292/08 <br /> www.molecularsystemsbiology.com <br />  <br />  <br /> A complete collection single-gene deletion mutants <br /> of <span id='am-173' about='obo:NCBITaxon_469' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_468 obo:NCBITaxon_1236 obo:NCBITaxon_72274'><span id='am-174'  ></span><span id='am-175' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasDbXref" content="PMID:8934907" datatype="xsd:string"></span><span id='am-177' property="rdfs:label" content="Acinetobacter" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasDbXref" content="PMID:9336926" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasDbXref" content="PMID:10028249" datatype="xsd:string"></span><span id='am-181' property="oboInOwl:hasDbXref" content="PMID:7520730" datatype="xsd:string"></span><span id='am-182' property="oboInOwl:hasDbXref" content="PMID:9226915" datatype="xsd:string"></span>Acinetobacter</span> baylyi ADP1 <br /> Ve&#180;ronique Berardinis1,2,*, David Vallenet1,2, Vanina Castelli1,2, Marielle Besnard1,2, Agne`s Pinet1,2, Corinne Cruaud1, <br /> Sumitta Samair1, Christophe Lechaplais1,2, Gabor Gyapay1,2, Ce&#180;line Richez1, Maxime Durot1,2, Annett Kreimeyer1,2, <br /> Franc&#184;ois Le Fe`vre1,2, Vincent Scha&#168;chter1,2, Vale&#180;rie Pezo1, Volker Do&#168;ring3, Claude Scarpelli1, Claudine Me&#180;digue1,2, Georges N Cohen1, <br /> Philippe Marlie`re3, Marcel Salanoubat1,2, Jean Weissenbach1,2 <br /> 1 <br />   Genoscope, Institut Ge&#180;nomique (<span id='am-1' about='obo:NCBITaxon_1107390' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_272274 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7422 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_272201 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7423 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Cea" datatype="xsd:string"></span>CEA</span>), Evry, France, 2 UMR8030 CNRS-Genoscope-Universite&#180; d&#8217;Evry, Evry, France 3 Isthmus, Evry, France <br /> * Corresponding author. V Berardinis, Genoscope Institut Ge&#180;nomique (CEA), 2 rue Gaston Cre&#180;mieux, 91057 Evry, France. <br /> Tel: &#254; 33 01 60 87 45 87; Fax: &#254; 33 01 60 87 25 14; E-mail: vberard@genoscope.fr <br />  <br /> Received 5.9.07; accepted 22.1.08 <br />  <br />  <br />  <br />  <br />                We constructed collection single-gene deletion mutants dispensable genes soil <br />                bacterium <span id='am-427' about='obo:NCBITaxon_202950' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_468 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_469'><span id='am-428' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-429' property="rdfs:label" content="Acinetobacter baylyi" datatype="xsd:string"></span><span id='am-430'  ></span><span id='am-431' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-432' property="oboInOwl:hasDbXref" content="PMID:12892111" datatype="xsd:string"></span>Acinetobacter baylyi</span> ADP1. A total 2594 deletion mutants obtained, 499 <br />                (16%) not, candidate essential genes life minimal medium. <span id='am-131' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-132' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-134' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-135'  ></span>This</span> <br />                essentiality <span id='am-438' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-439' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-440' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-441' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-442'  ></span>data</span> set 88% consistent <span id='am-350' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-351' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-352' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-353' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-354' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-355' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-356' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-357' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-358' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-359' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-360' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-361' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-362' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-363' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-364'  ></span><span id='am-365' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span>Escherichia coli</span> data set inferred Keio <br />                mutant collection profiled growth minimal medium, 80% orthologous genes <br />                described essential <span id='am-272' about='obo:NCBITaxon_286' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621'><span id='am-273'  ></span><span id='am-274' property="oboInOwl:hasDbXref" content="PMID:9103607" datatype="xsd:string"></span><span id='am-275' property="oboInOwl:hasDbXref" content="PMID:23918787" datatype="xsd:string"></span><span id='am-276' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-277' property="oboInOwl:hasRelatedSynonym" content="Peudomonas" datatype="xsd:string"></span><span id='am-278' property="oboInOwl:hasDbXref" content="PMID:10939664" datatype="xsd:string"></span><span id='am-279' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-280' property="oboInOwl:hasDbXref" content="PMID:7727274" datatype="xsd:string"></span><span id='am-281' property="oboInOwl:hasRelatedSynonym" content="Loefflerella" datatype="xsd:string"></span><span id='am-282' property="oboInOwl:hasDbXref" content="PMID:18048745" datatype="xsd:string"></span><span id='am-283' property="oboInOwl:hasRelatedSynonym" content="Liquidomonas" datatype="xsd:string"></span><span id='am-284' property="rdfs:label" content="Pseudomonas" datatype="xsd:string"></span><span id='am-285' property="oboInOwl:hasDbXref" content="PMID:10758879" datatype="xsd:string"></span><span id='am-286' property="oboInOwl:hasRelatedSynonym" content="RNA similarity group I" datatype="xsd:string"></span><span id='am-287' property="oboInOwl:hasDbXref" content="PMID:15950132" datatype="xsd:string"></span>Pseudomonas</span> aeruginosa essential ADP1. Several strategies were <br />                undertaken investigate ADP1 metabolism (1) searching discrepancies our <br />                essentiality data current metabolic knowledge, (2) comparing essentiality data set those <br />                organisms, (3) systematic phenotyping mutant collection variety carbon <br />                sources (quinate, 2-3 butanediol, glucose, etc.). This collection provides new resource study <br />                gene function forward reverse genetic approaches constitutes robust experimental <br />                data source systems biology approaches. <br />                Molecular Systems Biology 4 March 2008; doi:10.1038/msb.2008.10 <br />                Subject Categories: functional genomics; cellular metabolism <br />                Keywords: Acinetobacter baylyi ADP1; deletion mutants; essential genes; growth phenotype; <br />                metabolism; systems biology <br />  <br />                This open-access article distributed terms Creative Commons Attribution Licence, <br />                permits distribution reproduction medium, provided original author source are <br />                credited. Creation derivative works permitted resulting work distributed the <br />                similar licence one. This licence does permit commercial exploitation specific <br />                permission. <br />  <br />  <br /> Introduction                                                                   genes. For years, analysis mutant phenotypes has <br />                                                                                powerful tool understanding gene function, until <br /> The genome sequences approximately 500 bacteria                        recently, production mutants performed a <br /> already released public databases, providing                 gene-to-gene basis. Recently, availability complete <br /> information contributes understanding prokar-                   genome sequences permitted construction large- <br /> yotic physiology. However, function large fraction                 scale collections mutants. Two different methods been <br /> these bacterial genes unknown. Experimental                       used construct mutant libraries genomic scale. The <br /> in silico analyses elucidated role numerous genes                  frequently used method random approach <br /> although 13.9% <span id='am-366' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-367' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-368' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-369' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-370' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-371'  ></span><span id='am-372' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span>Escherichia</span> coli genes                         whole-genome transposon mutagenesis for <br /> assigned function additional 32% predicted                  <span id='am-311' about='obo:NCBITaxon_727' typeof='owl:Thing obo:NCBITaxon_135625 obo:NCBITaxon_712 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_724'><span id='am-312' property="oboInOwl:hasRelatedSynonym" content="Coccobacillus pfeifferi" datatype="xsd:string"></span><span id='am-313' property="oboInOwl:hasRelatedSynonym" content="Influenza-bacillus" datatype="xsd:string"></span><span id='am-314' property="rdfs:label" content="Haemophilus influenzae" datatype="xsd:string"></span><span id='am-315' property="oboInOwl:hasRelatedSynonym" content="Haemophilus meningitidis" datatype="xsd:string"></span><span id='am-316' property="oboInOwl:hasRelatedSynonym" content="Bacterium influenzae" datatype="xsd:string"></span><span id='am-317' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-318' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium influenzae" datatype="xsd:string"></span><span id='am-319' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-320'  ></span>Haemophilus influenzae</span> (Akerley et al, 2002), Pseudomonas <br /> function (Riley et al, 2006). It likely number               aeruginosa PA14 (Liberati et al, 2006) PA01 (Jacobs et al, <br /> biological processes remain understood,                     2003), E. coli (Gerdes et al, 2003) <span id='am-78' about='obo:NCBITaxon_1716' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_1653'><span id='am-79' property="oboInOwl:hasDbXref" content="PMID:7547291" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Corynebacterium" datatype="xsd:string"></span><span id='am-82'  ></span><span id='am-83' property="oboInOwl:hasDbXref" content="PMID:10425754" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="PMID:7547293" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="PMID:7547284" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="PMID:19661509" datatype="xsd:string"></span>Corynebacterium</span> <br /> identification gene function represents big                        glutamicum (Suzuki et al, 2006). An alternative strategy has <br /> challenge forthcoming years. Furthermore,                         represented directed approaches genome- <br /> processes ubiquitous and, important,                      wide gene-by-gene transposon mutagenesis (Kang et al, 2004) <br /> only shared subset bacterial <span id='am-141' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-142' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> (or strains)             complete gene deletion method used <span id='am-116' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-117' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-122'  ></span>Saccharomyces</span> <br /> not well-studied models. Understanding modelling                       cerevisiae (Giaever et al, 2002), <span id='am-45' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-59'  ></span><span id='am-60' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span>Bacillus subtilis</span> (Kobayashi <br /> complexity living organism require global elucidation                  et al, 2003) E. coli (Baba et al, 2006). As random <br /> gene function identification essential                   approach cost time effective, progressively <br />  <br /> &amp; 2008 EMBO Nature Publishing Group                                                                                  Molecular Systems Biology 2008 1 <br />  Collection A. baylyi ADP1 deletion mutants <br /> V Berardinis et al <br />  <br />  <br />  <br />  <br /> become popular one, suffers           convenient complementary model studying metabolism <br /> main drawbacks: (1) heterogeneous distribution          major reasons: (1) does metabolic ambiguity <br /> transposon insertions chromosome, production            (aerobic/anaerobic), leads difficult inter- <br /> of mutants nonessential gene turns difficult,      pretations, (2) known present important <br /> (2) insertion transposon gene strictly       metabolic differences E. coli specificity their <br /> correlated gene inactivation, (3) growth             respective ecological niches (Juni, 1978; Barbe et al, 2004; Young <br /> mutant occurs mixed population, leads               et al, 2005). Moreover, presence smaller number genes <br /> counter-selection decreased fitness. The           ADP1 genome compared bacteria (around 3300 <br /> laborious expensive directed approaches suffer             genes versus B4000 genes E. coli B5600 genes for <br /> from limitations appear error-prone         P. aeruginosa) suggests equally smaller number redundant <br /> (Gerdes et al, 2006).                                                 genes, increase detection single-gene deletion <br />    Despite numerous studies genes enzymatic                phenotypes. <br /> activities corresponding proteins, gaps      Here, report construction nearly complete <br /> some metabolic pathways, large number genes              collection ADP1 mutants single-gene deletion on <br /> which function assigned. Recently, complete           minimal media. Of 3197 annotated protein-encoding genes <br /> collection single-gene knockout mutants E. coli        ADP1, essential/dispensable status assigned 3093 <br /> reported Keio University (Baba et al, 2006). This resource         genes (B97% total). We started explore ADP1 <br /> constitutes powerful tool elucidation parts           metabolism analyzing mutant data respect our <br /> metabolism remain characterized. However,               current knowledge metabolism, based genome <br /> E. coli lives specific environment (e.g., human gut),   annotation, comparison essentiality data from <br /> it important extend analysis prokaryotic <span id='am-373' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-374'  ></span><span id='am-375' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-376' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-377' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>meta</span>-           organisms, E. coli phylogenetically closer <br /> bolism organisms living different environments.           bacterium, P. aeruginosa. Genomic comparative analyses with <br />    Species <span id='am-378' about='obo:NCBITaxon_genus' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-379' property="rdfs:label" content="genus" datatype="xsd:string"></span><span id='am-380' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>genus</span> Acinetobacter (a g-proteobacterium)               diverse genomes used complementary <br /> commonly aquatic soil environments capable           approach facilitate prediction physiolo- <br /> of utilizing diverse range compounds,               gical function genes located synteny bacterial <br /> phytochemicals (e.g., aromatic compounds), sources              genomes. Furthermore, number mutants correspond to <br /> carbon/energy. Furthermore, robust metabolism                genes involved catabolic processes. Indeed, whole <br /> responsible high capacity adaptation. For reason,      mutant collection grown defined metabolite carbon <br /> they aroused growing possible          nitrogen source, genes involved assimilation <br /> biotechnological environmental applications. Some                 (enzymes, regulators, transporters, etc.) appear as <br /> strains genus known involved          essential genes. To characterize ADP1 catabolic pathways, we <br /> biodegradation number different pollutants (biphenyl,         initiated systematic phenotyping mutant <br /> chlorinated biphenyl, benzoate, crude oil, on)          collection grown diverse carbon sources. <br /> the removal phosphate heavy metals. They well <br /> represented production number extra- and <br /> intracellular economical products (lipases, proteases, cyano-         Results discussion <br /> phycine, bioemulsifiers, on) kinds of <br /> biopolymers (Abd El-Haleem, 2003).                                    Construction ADP1 mutant collection <br />    Acinetobacter baylyi ADP1 (ADP1) considered               The approach used gene deletion ADP1 been <br /> particularly interesting remarkable competence         described (Metzgar et al, 2004) based ADP1 natural <br /> for natural transformation. This highly transformable strain          competence transformation combined homology- <br /> was obtained mutagenesis strain isolated        directed recombination linear fragments. The gene deletion <br /> from soil. ADP1 sequenced annotated Genoscope              strategy, summarized Figure 1, comprises following steps: <br /> with special focus reconstruction metabolism (Barbe             (1) cassette carrying positive (KanR) negative (tdk <br /> et al, 2004). Annotation data available               gene used generate unmarked deletions construct <br /> Magnifying Genome platform (MaGe; http://www.genoscope.fr/            double mutants) selection markers amplified PCR <br /> agc/mage/; Vallenet et al, 2006). The expert annotation            amplification; (2) approximately 300 bp regions (R1 and <br /> an ongoing process, updated releases annotation        R2) located upstream downstream targeted gene <br /> regularly deposited public databases. ADP1 relatively        amplified using primers included 50 -extension <br /> small genome (3.6 Mb) genes encoding catabolic          complementary primers used cassette amplifica- <br /> functions clustered genetic islands (Young et al,      tion; (3) PCR products combined re- <br /> 2005). Moreover, ideal organism high-throughput          amplified generate linear fragment containing cassette <br /> genetic analysis (Metzgar et al, 2004), shows high            flanked specific regions; (4) resulting product was <br /> competence natural transformation (Palmen Hellingwerf,        used directly transform ADP1 cells selected minimal <br /> 1997) easily manipulated homology-directed              medium agar plates supplemented kanamycin. <br /> recombination linear fragments (de Vries Wackernagel, <br /> 2002). In nature, new genes functional be <br /> recovered ADP1 using DNA strains isolated soil           Selection primers <br /> or natural environments. ADP1, strict aerobe, shares          As general rule, primers named P4 P5 designed <br /> about one-third genes E. coli, providing           delete gene start stop codons <br />  <br /> 2 Molecular Systems Biology 2008                                                                     &amp; 2008 EMBO Nature Publishing Group <br />                                                                                                                         Collection A. baylyi ADP1 deletion mutants <br />                                                                                                                                                  V Berardinis et al <br />  <br />  <br />  <br />  <br />                     Amplification integration cassette                Amplification upstream downstream regions target <br />                               pEVL186                                                   gene genomic DNA <br />                                                                                                                        P2&#8242; <br />                                                                                   P3            AUG                            P5 <br />                            P1 <br />  <br />                                         Tdk/KanR                                                    Target gene deletion <br />  <br />                                 Pt5                                                                                       STOP             P6 <br />                                                                                                P4 <br />                                                          P2 <br />                                                                                                       P1&#8242; <br />                                       PCR1                                       PCR2                                        PCR3 <br />  <br />                                                                                         R1                                          R2 <br />                                        Tdk/KanR <br />                                 Pt5 <br />  <br />  <br />                                                                                              Multiplex PCR <br />  <br />                                                               R1                                                  R2 <br />                                                                                     Tdk/KanR <br />           Transformation A. baylyi ADP1 <br />           homologous recombination                                 AUG <br />                                                                            Target gene deletion <br />  <br />                                                                                                     STOP <br />                                         P7 <br />           Selection kanamycin <br />           verification                                                          Tdk/KanR <br />                                                                                                                                          P8 <br />                                                                             S1                         S2 <br />  <br /> Figure 1 Method construction single-gene deletion mutants creation spliced PCR integration cassette. P1&#8211;P6 used integration cassette <br /> construction P7, P8, S1 S2 verifications. The kanR integration cassette obtained PCR amplification using P1 P2 primers pEVL186 DNA <br /> template. The flanking regions, specific target gene, amplified wild-type DNA template P3/P4 primers (R1 region) P5/P6 primers (R2 region). <br /> Designations followed prime (0 ) represent reverse complement sequences. The primers P7 P8 used external PCR verification correct replacement <br /> of targeted gene integrative cassette. The primers S1 S2 located kanR cassette used sequence junctions cassette P7/P8 <br /> PCR product. <br />  <br />  <br />  <br /> (Figure 1). For overlapping genes genes separated short                      Construction validation deletion mutants <br /> intergenic region, primers designed preserve                       For gene deletion experiment, picked KanR colonies <br /> expression flanking genes (see Materials                              re-isolated twice. Two correct clones each <br /> methods). According genome annotation, 21% ADP1                            mutant corresponding nonessential genes (dispensable <br /> genes overlapping 70% overlaps span                            genes) obtained set experiments about <br /> than nucleotides. Moreover, 36% genes                                90% cases. A second set mutations produced all <br /> separated intergenic regions shorter 20 nucleotides.                         cases mutants obtained, amplification of <br />                                                                                      PCR product failed second clone missing. <br />                                                                                      Finally, mutants obtained high- <br /> Choice selective medium mutant library                                    throughput process reattempted individually (up five <br /> construction                                                                         times genes unknown function). <br /> Four different media supplemented kanamycin (LB, 2YT,                              Mutants checked correct structure size by <br /> YT, minimal medium supplemented succinate)                                 PCR amplifications combining locus-specific, kanamycin <br /> tested deletion 96 genes (one 96-well plate)                          (P1/P6 P3/P2) external primers (P7/P8) (Figure 1). A <br /> number mutants obtained scored condition.                            clone validated PCR products correct <br /> Surprisingly, higher rate mutant recovery obtained                       size. However, cases, clone validated only <br /> minimal medium, suggest soil bacterium                            correct PCR products PCR products generated <br /> ADP1 adapted lifestyle carbon sources                        locus-specific kanamycin primers missing. <br /> rare scarce. This influence competence                             The analysis PCR products generated primers <br /> ability recover transformation process. To optimize                      P7/P8 resulted detection 196 mutants, showed <br /> our mutation screen, chosen realize                         amplification products, corresponding genomic <br /> mutant library minimal medium supplemented                               region intact target gene genomic <br /> succinate, efficient carbon source ADP1,                             region harboring correct replacement targeted gene <br /> directly assimilated central metabolism. A number                         cassette. These mutants called &#8216;Double Band&#8217; <br /> mutants obtained minimal medium (e.g.,                          mutants (DB mutants). Using Comparative Genomic <br /> genes involved biosynthesis amino acids, cofactors,                        Hybridization microarray, shown 9 10 <br /> and on), recovered medium                            DB mutants tested present duplication ranging 300 kb <br /> ad hoc supplementation.                                                              1 Mb (Gyapay et al, preparation). This result <br />  <br /> &amp; 2008 EMBO Nature Publishing Group                                                                                             Molecular Systems Biology 2008 3 <br />  Collection A. baylyi ADP1 deletion mutants <br /> V Berardinis et al <br />  <br />  <br />  <br />  <br /> could explained high plasticity ADP1 genome                      92% them. About 85% mutants obtained after <br /> shown previously Reams Neidle (2004)                             day incubation 92% grew 3 <br /> construction mutants carrying large tandem head-to-tail                       days. Mutants obtained produced an <br /> repeats (less 260 kb). For construction Keio                     incorrect structure (e.g., DB mutants) 499 genes (15.6% of <br /> collection (Baba et al, 2006), mutants carrying bona fide                 targeted genes). These genes represent set of <br /> deletion intact copy deleted gene localized                        candidate genes essential growth minimal medium <br /> elsewhere genome suggested cause                    supplemented succinate sole carbon source <br /> rare recovery mutants essential genes (frequency                          (Figure 3; Supplementary Table I). These essential genes are <br /> estimated 1%). As frequency significantly                   distributed chromosome regions: <br /> higher case (B7.5% mutants), assessed                       archipelago catabolic islands, prophagic region <br /> presence/absence targeted gene using internal primers                     siderophore region. In ADP1, CDSs asymmetry <br /> on subset 360 mutants including (1) 220 mutants                        distribution chromosome the <br /> clones appearing 3 days growth, (2) 120                       leading lagging strands (60 versus 40%). The <br /> mutants corresponding genes ortholog described                     distribution essential genes biased two <br /> essential analysis mutant collections, i.e.                   strands (15 versus 13% genes). The mutant collection is <br /> E. coli (Gerdes et al, 2003; Baba et al, 2006), P. aeruginosa                    currently extended mutants biosynthetic genes <br /> (Jacobs et al, 2003; Liberati et al, 2006) B. subtilis                        using media supplemented appropriate component <br /> (Kobayashi et al, 2003) (3) 20 mutants flanked essential                  (arginine, histidine, leucine, methionine, tryptophan), which <br /> genes ADP1. We confirmed deletion targeted gene                    resulted production 30 auxotrophic <br /> in 350 mutants, present 10 mutants. As                   mutants. The 74 mutants generated to <br /> these 10 genes size similar integration cassette                   various technical problems produced manually. <br /> (B1.7 kb), P7/P8 PCR products undistin-                            As mentioned above, pool 499 essential genes includes <br /> guishable electrophoresis. These 10 real DB mutants                      206 DB mutants (41% essential genes). A large proportion of <br /> raise 206, total number DB mutants.                                    genes resulting DB mutants present evidence of <br />    To double-check integration sites, vast majority                   essentiality. Indeed, 121 orthologs E. coli and <br /> PCR products obtained primers P7/P8 sequenced                          P. aeruginosa, 87% correspond candidate <br /> using internal primers, S1 S2, located cassette                   essential genes organisms <br /> (Figure 1). We detected incorrect structure small gene,                 (Supplementary Table II; Supplementary Annex 1). To investi- <br /> which showed integration cassette lacking R2                        gate essentiality genes, started to <br /> flanking region targeted gene.                                        generate mutants corresponding genes involved in <br />                                                                                  amino-acid biosynthesis complementing basic medium <br />                                                                                  suitable amino acid. Mutants obtained a <br /> Preliminary analysis collection                                           correct structure, amino-acid auxotrophy con- <br /> From 3310 coding sequences (CDS) ADP1, 113 genes                          firmed genes tested far (metE, metR, metW, argB, <br /> previously annotated <span id='am-88' about='obo:NCBITaxon_2673' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="Insertion" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="insertion sequences" datatype="xsd:string"></span>insertion sequences</span> (IS), gene                           argC, argF, arG, argJ, leuB, leuC, leuD, trpF). <br /> remnants doubtful CDSs discarded study, <br /> while remaining 3197 targeted deletion. Primers <br />                                                                                  Essentiality studies: comparison ADP1 <br /> could designed using standard automated procedure <br /> for 30 genes later developed manually. The gene                      prokaryotes <br /> deletion procedure carried set 3167 genes.                  A comparative analysis essential genes different <br />    We generated total 2594 mutants correct                         organisms reveal differences metabolic net- <br /> structure (Figure 2) obtained independent clones                     works and, therefore, define set essential genes shared <br />  <br />  <br />                                                                                         3197 genes <br />  <br />  <br />  <br />                                                              3167 mutants attempted                     30 genes primers <br />  <br />  <br />  <br />                                 293 mutants                   2800 mutants            74 mutants obtained <br />                                    obtained                         obtained               technical reasons <br />  <br />  <br />                                                           206 &#171; double <br />                                                          band &#187; mutants <br />  <br />  <br />  <br />  <br />                                                  499 candidate                   2594 validated         104 technical problems <br />                                                 essential genes                     mutants <br />  <br /> Figure 2   Flow diagram ADP1 mutant collection. <br />  <br /> 4 Molecular Systems Biology 2008                                                                                    &amp; 2008 EMBO Nature Publishing Group <br />                                                                                                                              Collection A. baylyi ADP1 deletion mutants <br />                                                                                                                                                       V Berardinis et al <br />  <br />  <br />  <br />  <br /> Figure 3 Chromosomal representation ADP1 mutant collection. Circles display (from outside): (1) GC percent deviation (GC window&#8212;mean GC) 1000- <br /> bp window. (2) CDSs blue classified biosynthesis TIGR role categories: (i) amino acid, (ii) cofactors, prosthetic groups, carriers, (iii) purines, <br /> pyrimidines, nucleosides, nucleotides. (3) CDSs red corresponds ADP1 essential genes. (4) ADP1 CDSs purple sharing orthologs E. coli K12 <br /> genes, described essential (Keio collection, green) conditionally essential (mutants growing poorly glucose minimal media (ODo0.1)). (5) GC skew <br /> (G &#254; C/G C) 1000-bp window. <br />  <br />  <br />  <br /> by bacteria living diverse environments. It noted                           data consistent previous reports the <br /> that interpretations differences pools essential                           majority essential genes known genes (Baba et al, 2006; <br /> genes performed cautiously, data                                 Liberati et al, 2006). However, number essential genes is <br /> crucially dependent technique medium used                             far higher analysis estimated previous <br /> generating mutant collection (Gerdes et al, 2006). Only                              studies bacterial mutant collections E. coli <br /> restricted sets gene essentiality data available date.                         (303 mutants 7% collection; Baba et al, 2006), <br /> A general analysis orthologous gene essentiality data                                 P. aeruginosa (335 mutants 6% collection; (Liberati <br /> between ADP1 bacteria (E. coli, P. aeruginosa                              et al, 2006)) B. subtilis (271 mutants 6.6% the <br /> B. subtilis) summarized Figure 4. Among 499                                    collection; Kobayashi et al, 2003). All collections were <br /> candidate essential genes, 350 shared                            constructed rich media, sets essential genes are <br /> other organisms, E. coli P. aeruginosa. Moreover, 69                           restricted genes required viability favorable <br /> ADP1 genes obvious orthologs organisms                                  conditions, present study, genes involved in <br /> happen essential encode proteins unknown                                 biosynthesis essential compounds, necessary <br /> putative function (82%).                                                                 growth minimal medium, added the <br />                                                                                          aforementioned number essential genes (Supplementary <br />                                                                                          Table III). <br /> Functional categories                                                                       ADP1 CDS classified different functional roles <br /> Of 499 candidate essential genes, 397 (79.6%) correspond                             according TIGR classification. The distribution of <br /> to known genes, 56 (11.2%) genes putative function                             essential genes ADP1, E. coli (Baba et al, 2006) and <br /> and 46 (9.2%) genes unknown function (Table I). These                              P. aeruginosa (Liberati et al, 2006) functional categories is <br />  <br /> &amp; 2008 EMBO Nature Publishing Group                                                                                                 Molecular Systems Biology 2008 5 <br />  Collection A. baylyi ADP1 deletion mutants <br /> V Berardinis et al <br />  <br />  <br />  <br />  <br />                                                                      Acinetobacter baylyi ADP1 <br />  <br />  <br />  <br />  <br />                                        <span id='am-288' about='obo:NCBITaxon_287' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_286 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_136841 obo:NCBITaxon_135621'><span id='am-289' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas pyocyanea" datatype="xsd:string"></span><span id='am-290' property="oboInOwl:hasAlternativeId" content="NCBITaxon:932477" datatype="xsd:string"></span><span id='am-291' property="oboInOwl:hasRelatedSynonym" content="probable synonym or variety: &quot;Pseudomonas polycolor&quot; Clara 1930" datatype="xsd:string"></span><span id='am-292' property="oboInOwl:hasRelatedSynonym" content="Bacterium aeruginosum" datatype="xsd:string"></span><span id='am-293' property="rdfs:label" content="Pseudomonas aeruginosa" datatype="xsd:string"></span><span id='am-294' property="oboInOwl:hasRelatedSynonym" content="Peudomonas aeruginosa" datatype="xsd:string"></span><span id='am-295' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931955" datatype="xsd:string"></span><span id='am-296' property="oboInOwl:hasRelatedSynonym" content="Bacterium pyocyaneum" datatype="xsd:string"></span><span id='am-297' property="oboInOwl:hasRelatedSynonym" content="Bacillus aeruginosus" datatype="xsd:string"></span><span id='am-298' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931956" datatype="xsd:string"></span><span id='am-299' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1437768" datatype="xsd:string"></span><span id='am-300' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-301' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224290" datatype="xsd:string"></span><span id='am-302'  ></span><span id='am-303' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931958" datatype="xsd:string"></span><span id='am-304' property="oboInOwl:hasRelatedSynonym" content="Micrococcus pyocyaneus" datatype="xsd:string"></span><span id='am-305' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931957" datatype="xsd:string"></span><span id='am-306' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-307' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1437770" datatype="xsd:string"></span><span id='am-308' property="oboInOwl:hasRelatedSynonym" content="Bacillus pyocyaneus" datatype="xsd:string"></span><span id='am-309' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas polycolor" datatype="xsd:string"></span><span id='am-310' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1437769" datatype="xsd:string"></span>Pseudomonas aeruginosa</span> PA14                                           Bacillus subtilis <br />                                             (1655 orthologs)                      95 genes                   (704 orthologs) <br />                                                                            Ess(18 A; 0 E; 6 P; 8 B) <br />                                           526 genes                                                                   52 genes <br />                                    Ess(45 A; 0 E; 22 P; 0 B)                                                    Ess(4 A; 0 E; 0 P; 2 B) <br />                                                                             520 genes <br />                                                                    Ess(251 A; 156 E; 125 P; 168 B) <br />  <br />  <br />                                                      514 genes <br />                                               Ess(99 A; 61 E; 62 P; 0 B)                               37 genes <br />                                                                                                 Ess(5 A; 2 E; 0 P; 3 B) <br />  <br />  <br />                                                                             106 genes <br />                                                                       Ess(8 A; 4 E; 0 P; 0 B) <br />  <br />                                                                        Escherichia coli K12 <br />                                   1347 genes                             (1177 orthologs) <br />                             Ess(69 A; 0 E; 0 P; 0 B) <br />  <br />  <br />  <br />  <br /> Figure 4 Analysis orthologous gene essentiality data ADP1 bacteria (E. coli, P. aeruginosa B. subtilis). The number of <br /> orthologous genes comparison number genes essential organisms, reported Ess(A: ADP1, E: E. coli, P: P. aeruginosa, <br /> B: B. subtilis). <br />  <br />  <br /> Table I Summary A. baylyi ADP1 knockout collection <br />  <br />                                            Number genes          Number attempted mutants                 Number mutants          Number essential genes <br />  <br /> Protein known function                       1160                              1150                                 731                           397 <br /> Protein putative function                     928                               920                                835                              56 <br /> Conserved hypothetical proteinsa                   889                               881                                825                              36 <br /> Hypothetical proteinsb                             220                               216                                203                              10 <br /> All categories                                    3197                           3167 (97%)                          2594 (82%)                     499 (16%) <br />  <br /> a <br /> Homologs previously reported genes unknown function. <br /> b <br /> No homology previously reported sequences. <br />  <br />  <br />  <br /> shown Figure 5A. For functional categories (bio-                                example, salvage pathway nucleosides nucleotides, <br /> synthesis amino acids, cofactors/prosthetic groups                                does exist ADP1, involves 20 genes in <br /> purines/pyrimidines/nucleosides/nucleotides), percentage                             E. coli. The functional categories notably <br /> of essential genes significantly higher ADP1                           influenced medium used mutant construction. <br /> other species, reflect different media used in <br /> the construction libraries (minimal versus rich <br /> medium). We, therefore, compared data extended                                 Comparison Pseudomonas aeruginosa essential <br /> set essential genes included conditional essential genes                         genes <br /> (or CEG; mutants growing poorly (ODo0.1) minimal                                 P. aeruginosa phylogenetically closest organism ADP1 <br /> medium supplemented glucose), available                               collections transposon mutants available <br /> E. coli Keio collection (EEGS &#8216;Extended Essential                             (Jacobs et al, 2003; Liberati et al, 2006). These mutant <br /> Gene Set&#8217;; Figure 5B). For categories                            collections obtained rich medium (LB) set of <br /> essential genes appeared significantly lower E. coli,                        335 essential genes defined genes transposon <br /> comparison EEGS resulted ratio number                             integration library. P. aeruginosa ADP1 share <br /> essential genes similar ones observed                           1655 orthologous genes, including 209 described as <br /> ADP1. For category (synthesis purines, pyrimidines,                          essential P. aeruginosa, essentiality data ADP1. <br /> nucleosides nucleotides), number                                     A large majority genes (80%) essential in <br /> essential genes similar, percentage difference                            ADP1. In contrast, 246 essential genes ADP1 described <br /> reduced, mainly higher number genes                                    dispensable P. aeruginosa (Supplementary Table IV). <br /> assigned category E. coli (80 versus 44 ADP1). For                         Among 246 genes, 100 genes associated with <br />  <br /> 6 Molecular Systems Biology 2008                                                                                                 &amp; 2008 EMBO Nature Publishing Group <br />                                                                                                                                   Collection A. baylyi ADP1 deletion mutants <br />                                                                                                                                                            V Berardinis et al <br />  <br />  <br />  <br />  <br />                                                                      A.baylyi ADP1                           E. coli                                P. aeruginosa <br />                                                                     minimal medium                        rich medium                                 rich medium <br />                           Transport binding proteins <br />                                              Transcription <br />                                      Regulatory functions <br />       Purines, pyrimidines, nucleosides nucleotides <br />                                         Protein synthesis <br />                                               Protein fate <br />                                        Unknown function <br />                  Fatty acid phospholipid metabolism <br />                                       Energy metabolism <br />                                         DNA metabolism <br />                         Central intermediary metabolism <br />                                        Cellular processes <br />                                             Cell envelope <br /> Biosynthesis cofactors, prosthetic groups carriers <br />                                 Amino-acid biosynthesis <br />                                                          0%   20%     40%   60%   80%   100%   0%   20%   40%    60%    80%   100%     0%     20%    40%    60%     80%   100% <br />  <br />  <br />  <br />                                                                      A.baylyi ADP1 <br />                                                                                                            E. coli <br />                                                                     minimal medium <br />                           Transport binding proteins <br />                                              Transcription <br />                                      Regulatory functions <br />       Purines, pyrimidines, nucleosides nucleotides <br />                                         Protein synthesis <br />                                               Protein fate <br />                                        Unknown function <br />                  Fatty acid phospholipid metabolism <br />                                       Energy metabolism <br />                                         DNA metabolism <br />                         Central intermediary metabolism <br />                                        Cellular processes <br />                                             Cell envelope <br /> Biosynthesis cofactors, prosthetic groups carriers <br />                                 Amino-acid biosynthesis <br />                                                          0%   20%     40%   60%   80%   100%   0%   20%   40%    60%    80%   100% <br />  <br /> Figure 5 Comparison ADP1, E. coli P. aeruginosa essentiality data according TIGR role classification (A) medium used obtain mutants, (B) <br /> on minimal medium. White blue areas correspond nonessential essential genes, respectively. The red area concerns main categories notably impacted by <br /> the medium used mutant construction. <br />  <br /> biosynthesis function (e.g., amino acids, cofactors, etc.)                                  gene deletion method better determination multi- <br /> are expected essential minimal media. Surpris-                                    functional protein essentiality. <br /> ingly, unexpectedly high number discrepancies (33 <br /> cases) genes involved protein synthesis. <br /> Most genes encode tRNA synthetases ribosomal                                        Comparison Escherichia coli essential genes <br /> proteins essential life previously described                               The Keio collection E. coli library a <br /> number organisms. They represent back-                                        detailed comparison essential gene data sets be <br /> ground false dispensable genes detected random                                        performed accuracy, sets based single-gene <br /> transposon mutagenesis approach.                                                                knockout mutant collections obtained profiled for <br />    Another example background caused use                                       growth minimal media. A status assigned our <br /> a transposon library described illustrates                                     mutant collection 1144 genes 1177 orthologous <br /> a similar gene essentiality status mutant                                           genes E. coli ADP1. We compared set of <br /> collections conceal different features. In riboflavin                                       essential genes EEGS Keio collection, the <br /> biosynthesis, ribA ribB encode GTP cyclohydrolase                                         results consistent 88% cases (i.e., genes to <br /> and riboflavin synthase, respectively. Both genes                                 essential dispensable sets; Table II). In <br /> bifunctional ribB/A gene ADP1; ribB                                             comparison, Baba et al. (2006) reported 67% their <br /> dispensable expected, surprisingly ribA appears                                    predicted essential genes overlap 620 essential genes <br /> essential, suggesting RibB/A does complement                                       estimated Gerdes et al. (2003) E. coli transposition <br /> RibA function. In P. aeruginosa, ribA ribB/A                                     mutant collection. This result highlights importance the <br /> are present, ribA essential, despite absence ribB,                                techniques conditions used construction the <br /> ribB/A unexpectedly reported dispensable. A close                                         library prediction essential genes. <br /> examination insertion sites gene shows                                     The essentiality status different 134 genes the <br /> all insertions located ribA region ribB/A,                                    Keio collection (Table II; Supplementary Table V). Of <br /> whereas insertion ribB region, revealing                                 these, 98 genes essential ADP1 dispensable <br /> essentiality. Taken together, observations clearly                                   E. coli, 36 dispensable ADP1, essential in <br /> the essentiality RibA RibB functions                                   E. coli. Although exclude cases <br /> bacteria cast doubt commonly annotated function                                    undetected technical problems occurred <br /> of ribA-like region ribB/A. In agreement                                        deletion process polar effect, differences <br /> hypothesis, genes ribB/A <span id='am-248' about='obo:NCBITaxon_210' typeof='owl:Thing obo:NCBITaxon_72293 obo:NCBITaxon_213849 obo:NCBITaxon_209 obo:NCBITaxon_1 obo:NCBITaxon_68525 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_29547'><span id='am-249' property="oboInOwl:hasRelatedSynonym" content="Campylobacter pyloridis" datatype="xsd:string"></span><span id='am-250' property="oboInOwl:hasDbXref" content="PMID:8186097" datatype="xsd:string"></span><span id='am-251' property="oboInOwl:hasAlternativeId" content="NCBITaxon:219" datatype="xsd:string"></span><span id='am-252' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-253' property="oboInOwl:hasDbXref" content="PMID:11931154" datatype="xsd:string"></span><span id='am-254' property="oboInOwl:hasRelatedSynonym" content="Campylobacter pylori subsp. pylori" datatype="xsd:string"></span><span id='am-255' property="oboInOwl:hasDbXref" content="PMID:8494747" datatype="xsd:string"></span><span id='am-256' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-257' property="oboInOwl:hasDbXref" content="PMID:1995031" datatype="xsd:string"></span><span id='am-258' property="rdfs:label" content="Helicobacter pylori" datatype="xsd:string"></span><span id='am-259' property="oboInOwl:hasRelatedSynonym" content="Helicobacter nemestrinae" datatype="xsd:string"></span><span id='am-260' property="oboInOwl:hasRelatedSynonym" content="Campylobacter pylori" datatype="xsd:string"></span><span id='am-261'  ></span>Helicobacter pylori</span>                                      explained major factors isofunctional <br /> strain P1 (Fassbinder et al, 2000) <span id='am-6' about='obo:NCBITaxon_1883' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_2062 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_85011'><span id='am-7' property="oboInOwl:hasDbXref" content="PMID:4886644" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="PMID:8934906" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="PMID:1371059" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="PMID:8590678" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1973" datatype="xsd:string"></span><span id='am-13'  ></span><span id='am-14' property="oboInOwl:hasDbXref" content="PMID:15950131" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="PMID:9731302" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="PMID:16560709" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="NCBITaxon:65506" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="NCBITaxon:31945" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="PMID:10843062" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Chainia" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Streptomyces" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="PMID:11411701" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="PMID:9336904" datatype="xsd:string"></span>Streptomyces</span> davawensis                                  genes. Therefore, discrepancies mainly reflect the <br /> (Grill et al, 2007) fail complement E. coli mutant lacking                                following points. (1) Lifestyle differences (strict aerobe/ <br /> RibA function. This example illustrates advantage                                    facultative aerobe) shown instance dispensability <br />  <br /> &amp; 2008 EMBO Nature Publishing Group                                                                                                      Molecular Systems Biology 2008 7 <br />  Collection A. baylyi ADP1 deletion mutants <br /> V Berardinis et al <br />  <br />  <br />  <br />  <br /> Table II Orthologous genes A. baylyi ADP1 Escherichia coli       require investigation, genes carbon <br /> essentiality E. coli                                                           storage regulator (csrA), ribonuclease III (rnc) rmpA (the <br />                                                    E. coli                        50S ribosomal subunit protein L27), described as <br />                                                                                   essential E. coli. <br /> Acinetobacter baylyi ADP1          Dispensable          Essential genes <br />                                       genes               (E+CEG) <br />  <br /> Dispensable genes                      745         36 (24 E+12 CEG)               Exploration gene function pathways <br /> (781 genes)a                                                                      mutant status <br /> Essential genes (363                   98          265 (189 E+76 CEG) <br /> genes)a                                                                           Acinetobacter genome annotation prerequisite the <br /> All categories                         843         301 (213 E+88 CEG)             expert reconstruction metabolic pathways, can <br />                                                                                   provide global view metabolism. The comparison <br /> E, essential gene Keio collection data corresponding mutants not <br /> obtained rich medium; CEG, conditional essential gene Keio            current knowledge metabolism experimental <br /> collection data corresponding mutants growing poorly glucose           data set raises questions validity annotation-based <br /> (ODo0.1); discrepancies ADP1 E.coli written bold characters.   knowledge. However, number exceptions that <br /> a <br />  For category, number ADP1 genes ortholog E. coli is <br /> reported brackets. This number corresponds sum number         allowed formulate new hypotheses, including two <br /> dispensable essential genes E. coli.                                       examples described below. To help comparison, an <br />                                                                                   ADP1 Genome Browser implemented present data of <br />                                                                                   mutant collection genomic context (KEGG and <br /> of ATP synthase subunits E. coli. (2) Various carbon <br />                                                                                   BioCyc metabolic pathways, SEED subsystems, operon pre- <br /> sources (glucose versus succinate) used obtain/grow the <br />                                                                                   dictions, orthologous genes essentiality data several <br /> mutants illustrated essentiality succinate <br />                                                                                   organisms, phenotyping data high-throughput experi- <br /> dehydrogenase subunits succinate transporter in <br />                                                                                   ments), available (http://www.genoscope.cns.fr/ <br /> ADP1 phosphoenolpyruvate carboxylase (ppc) <br />                                                                                   cgi-bin/ggb/thesaurus/gbrowse/thesaurus). <br /> in E. coli clones grown glucose (this gene dispensable <br />                                                                                      The step Coenzyme-A biosynthesis involves the <br /> when mutant grown presence intermediate of <br />                                                                                   formation b-alanine aspartate. The panD gene encodes <br /> the tricarboxylic acid cycle). (3) The presence isofunctional <br />                                                                                   1-aspartate decarboxylase activity, essential for <br /> genes genes involved gluconeogenesis and <br />                                                                                   growth minimal medium large number prokaryotes <br /> the pentose phosphate pathway (gpmI, fbp, rpiA) E. coli, <br />                                                                                   including E. coli. In ADP1 panD mutant obtained, <br /> and map, folD, icd, ribB, lip ADP1. (4) The redundancy of <br />                                                                                   suggesting presence gene could <br /> metabolic pathways. This point illustrated example <br />                                                                                   perform activity alternative pathway leading to <br /> by biosynthesis iron-sulfur clusters (ISC). E. coli <br />                                                                                   b-alanine biosynthesis. Similarly, dispensability erytho- <br /> contains well-described biosynthetic systems, <br />                                                                                   nate-4-phosphate dehydrogenase (pdxB) suggests pre- <br /> ISC (Iron Sulfur Cluster) SUF (sulfur assimilation) (Barras <br />                                                                                   sence isofunctional gene alternative pathway <br /> et al, 2005). Both isc suf operons encode proteins with <br />                                                                                   resulting formation pyridoxal 5-phosphate <br /> similar biochemical activities vitro (Fontecave et al, 2005), <br />                                                                                   cofactor. <br /> and pathway dispensable E. coli, confirmed the <br />                                                                                      In novo biosynthesis pyrimidine ribonucleotides, two <br /> data Keio collection. In ADP1, similar Pseudomonas <br />                                                                                   dihydroorotases, ACIAD1150 (pyrC) ACIAD1419, which <br /> species, isc operon genome. Moreover, <br />                                                                                   share homology, identified. We delete pyrC, <br /> the essentiality iscSUA-hscBA-fdx genes two <br />                                                                                   indicating gene complemented other <br /> species reinforces hypothesis pathway is <br />                                                                                   dihydroorotase vivo. In Pseudomonas, presence two <br /> active exists. (5) Variations metabolic pathways, <br />                                                                                   functional dihydroorotases PyrC PyrC2, orthologs of <br /> such linear/cyclic arginine biosynthetic pathway, are <br />                                                                                   ACIAD1150 ACIAD1419, respectively, experimentally <br /> illustrated dispensability <span id='am-381' about='obo:NCBITaxon_95269' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_85772 obo:NCBITaxon_6656 obo:NCBITaxon_85773 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7399 obo:NCBITaxon_50557 obo:NCBITaxon_6072 obo:NCBITaxon_6960'><span id='am-382' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-383'  ></span><span id='am-384' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-385' property="rdfs:label" content="Arge" datatype="xsd:string"></span>argE</span> argA, raises <br />                                                                                   demonstrated. It shown pyrC constitutively <br /> questions role ADP1. (6) Physiological <br />                                                                                   expressed, pyrC2 expressed pyrC mutant <br /> differences. For instance, number genes responsible for <br />                                                                                   background possible induction carbamoyl <br /> the formation rod-shaped cell polar differentiation <br />                                                                                   L-aspartate accumulation (Brichta et al, 2004). In ADP1, we <br /> (mreBCD, minCDE, pbpA/mrdA rodA/mrdB) consid- <br />                                                                                   hypothesize pyrC essentiality explained a <br /> ered essential Keio collection PEC database <br />                                                                                   delay pyrC2 induction, incompatible <br /> (except minCDE; www.shigen.nig.ac.jp/ecoli/pecv3/). In <br />                                                                                   survival cell depleted dihydroorotase activity <br /> E. coli, cells depleted mreB, mreC mreD become <br />                                                                                   construction mutant library. <br /> spherical, enlarge finally undergo lysis (Kruse et al, 2005). <br /> We observed phenomenon ADP1, which <br /> the deletion genes produces dramatic change in <br /> cell morphology normal rod shape spherical                          Methionine biosynthesis <br /> form, lysis observed 24h-culture (Figure 6).                   Prokaryotic organisms synthesize methionine differ- <br /> A spherical form observed deletion pbpA                     ent biosynthetic pathways homoserine (Figure 7). The <br /> and rodA genes ADP1. Moreover, deletion minCDE                          step corresponds acylation homoserine using <br /> involved Z-ring formation (Zhou Lutkenhaus, 2005)                          succinate acetate generate O-succinyl-L-homoserine <br /> leads formation giant long cells ADP1 (Figure 6). In                 mediated metA E. coli O-acetyl-L-homoserine <br /> some cases, dispensability genes ADP1                        mediated concerted action metX metW as <br />  <br /> 8 Molecular Systems Biology 2008                                                                                &amp; 2008 EMBO Nature Publishing Group <br />                                                                                                                    Collection A. baylyi ADP1 deletion mutants <br />                                                                                                                                             V Berardinis et al <br />  <br />  <br />  <br />  <br />                        Wild-type strain                        pbpA                                  rodA <br />  <br />  <br />  <br />  <br />                         mreB                                   mreC                                  mreD <br />  <br />  <br />  <br />  <br />                         minE                                  minD                                   minC <br />  <br />  <br />  <br />  <br /> Figure 6 Comparison mutants corresponding deletion genes responsible formation rod shape polar differentiation wild-type strain <br /> phenotypes observed microscopy (   1000). The mreBCD, pbpA rodA genes involved cell-shape determination rod-shaped bacteria, the <br /> minCDE genes involved cellular division. The cells grown overnight liquid media (minimal medium complemented succinate carbon source). <br />  <br />  <br /> in <span id='am-207' about='obo:NCBITaxon_317' typeof='owl:Thing obo:NCBITaxon_136849 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_286 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621 obo:NCBITaxon_251695'><span id='am-208'  ></span><span id='am-209' property="oboInOwl:hasDbXref" content="PMID:1805313" datatype="xsd:string"></span><span id='am-210' property="oboInOwl:hasAlternativeId" content="NCBITaxon:936736" datatype="xsd:string"></span><span id='am-211' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasDbXref" content="PMID:10552044" datatype="xsd:string"></span><span id='am-213' property="rdfs:label" content="Pseudomonas syringae" datatype="xsd:string"></span><span id='am-214' property="oboInOwl:hasDbXref" content="PMID:10319466" datatype="xsd:string"></span><span id='am-215' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-216' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas syringae subsp. syringae" datatype="xsd:string"></span>Pseudomonas syringae</span> (Andersen et al, 1998) Pseudo-                          The final step methionine pathway performed by <br /> <span id='am-413' about='obo:NCBITaxon_267819' typeof='owl:Thing obo:NCBITaxon_96792 obo:NCBITaxon_1 obo:NCBITaxon_98651 obo:NCBITaxon_131567 obo:NCBITaxon_33634 obo:NCBITaxon_2759 obo:NCBITaxon_2825'><span id='am-414' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-415' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-416' property="rdfs:label" content="Monas" datatype="xsd:string"></span><span id='am-417'  ></span>monas</span> putida (Alaminos Ramos, 2001). Little known                         methionine synthase encoded B12 dependent metH or <br /> about metW, function remains unclear (Picardeau et al,                     B12 independent metE. As expected, metH mutant is <br /> 2003). In ADP1, shown metX metW mutants                         prototrophic, metE mutant, obtained presence <br /> are auxotrophic methionine (Table III) P. syringae. In                 coenzyme-B12, auxotrophic methionine. In <br /> addition, E. coli metA complements mutants, confirming                      Enterobacteria, metE usually flanked transcriptional <br /> that metX metW involved homoserine acylation.                         regulator metR, ADP1, metR located another <br /> Surprisingly, shown overexpression metX                      position genome, gene unknown function <br /> can complement deletion metX                        (ACIAD3524) metE. We shown the <br /> deletion metW metX/metW, metW comple-                          ACIAD3524 mutant auxotrophic methionine, whereas <br /> ment metW mutant (Table IV). These results strongly                     prototrophy restored complementation ACIAD3524. <br /> suggest constitutive expression metX compen-                     The colocalization metE ACIAD3524 conserved in <br /> sate absence metW MetW essential                      <span id='am-25' about='obo:NCBITaxon_1224' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-26' property="oboInOwl:hasExactSynonym" content="purple non-sulfur bacteria" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="PMID:11321122" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="PMID:16280474" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="PMID:11837318" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="purple bacteria and relatives" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="Proteobacteria" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="purple photosynthetic bacteria and relatives" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="proteobacteria" datatype="xsd:string"></span><span id='am-34'  ></span><span id='am-35' property="oboInOwl:hasDbXref" content="PMID:11542017" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="purple photosynthetic bacteria" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="purple bacteria" datatype="xsd:string"></span>proteobacteria</span>, Pseudomonas (but not <br /> acetylation homoserine vivo direct indirect                 P. aeruginosa), <span id='am-197' about='obo:NCBITaxon_338' typeof='owl:Thing obo:NCBITaxon_32033 obo:NCBITaxon_135614 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236'><span id='am-198' property="rdfs:label" content="Xanthomonas" datatype="xsd:string"></span><span id='am-199' property="oboInOwl:hasDbXref" content="PMID:2275852" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-201' property="oboInOwl:hasDbXref" content="PMID:10758874" datatype="xsd:string"></span><span id='am-202' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-203' property="oboInOwl:hasDbXref" content="PMID:11931143" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:hasDbXref" content="PMID:18048743" datatype="xsd:string"></span><span id='am-205'  ></span><span id='am-206' property="oboInOwl:hasDbXref" content="PMID:9103617" datatype="xsd:string"></span>Xanthomonas</span>, <span id='am-183' about='obo:NCBITaxon_357' typeof='owl:Thing obo:NCBITaxon_356 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_82115 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_28211 obo:NCBITaxon_227290'><span id='am-184' property="oboInOwl:hasDbXref" content="PMID:11760945" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasDbXref" content="PMID:13130069" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasDbXref" content="PMID:16560572" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-188'  ></span><span id='am-189' property="oboInOwl:hasDbXref" content="PMID:8494742" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasDbXref" content="PMID:10758890" datatype="xsd:string"></span><span id='am-191' property="oboInOwl:hasDbXref" content="PMID:13130068" datatype="xsd:string"></span><span id='am-192' property="oboInOwl:hasDbXref" content="PMID:11211278" datatype="xsd:string"></span><span id='am-193' property="oboInOwl:hasDbXref" content="PMID:8240952" datatype="xsd:string"></span><span id='am-194' property="oboInOwl:hasRelatedSynonym" content="Polymonas" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-196' property="rdfs:label" content="Agrobacterium" datatype="xsd:string"></span>Agrobacterium</span>, <span id='am-152' about='obo:NCBITaxon_441' typeof='owl:Thing obo:NCBITaxon_433 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_28211 obo:NCBITaxon_204441'><span id='am-153' property="oboInOwl:hasRelatedSynonym" content="Gluconobacter Asai 1935" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-155' property="rdfs:label" content="Gluconobacter" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasRelatedSynonym" content="&quot;Acetomonas&quot; Leifson 1954" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:hasRelatedSynonym" content="Acetomonas" datatype="xsd:string"></span><span id='am-159'  ></span>Gluconobacter</span>, <br /> action MetX.                                                                  <span id='am-386' about='obo:NCBITaxon_22' typeof='owl:Thing obo:NCBITaxon_267890 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135622'><span id='am-387' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-388' property="rdfs:label" content="Shewanella" datatype="xsd:string"></span><span id='am-389' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-390'  ></span><span id='am-391' property="oboInOwl:hasRelatedSynonym" content="Shewanella MacDonell and Colwell 1986" datatype="xsd:string"></span>Shewanella</span> <span id='am-335' about='obo:NCBITaxon_662' typeof='owl:Thing obo:NCBITaxon_641 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135623'><span id='am-336' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-337' property="oboInOwl:hasDbXref" content="PMID:17978204" datatype="xsd:string"></span><span id='am-338' property="oboInOwl:hasAlternativeId" content="NCBITaxon:705" datatype="xsd:string"></span><span id='am-339' property="rdfs:label" content="Vibrio" datatype="xsd:string"></span><span id='am-340' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-341' property="oboInOwl:hasDbXref" content="PMID:4935323" datatype="xsd:string"></span><span id='am-342' property="oboInOwl:hasDbXref" content="PMID:1371064" datatype="xsd:string"></span><span id='am-343' property="oboInOwl:hasDbXref" content="PMID:7520733" datatype="xsd:string"></span><span id='am-344'  ></span><span id='am-345' property="oboInOwl:hasRelatedSynonym" content="Pacinia" datatype="xsd:string"></span><span id='am-346' property="oboInOwl:hasRelatedSynonym" content="Listonella" datatype="xsd:string"></span><span id='am-347' property="oboInOwl:hasDbXref" content="PMID:21296930" datatype="xsd:string"></span><span id='am-348' property="oboInOwl:hasRelatedSynonym" content="Microspira" datatype="xsd:string"></span><span id='am-349' property="oboInOwl:hasDbXref" content="PMID:8590667" datatype="xsd:string"></span>Vibrio</span> genera. These findings suggest a <br />    The second step methionine biosynthesis generates                          role gene methionine biosynthesis regulation. <br /> homocysteine major pathways microorganisms: <br /> (1) transsulfuration pathway cystathionine (CTT) <br /> by CTT g-synthase (metB Enterobacteria metZ                         Ubiquinone biosynthesis <br /> Pseudomonales) CTT b-lyase (metC); (2) sulfhy-                       A second example pathway exploration focuses the <br /> drylation pathway leading formation homocysteine                    ubiquinone biosynthesis chorismate (Supplementary <br /> one step metY and/or metZ. Only sequenced bacteria                      Figure 1), previously described E. coli <br /> have pathways (Alaminos Ramos, 2001; Hwang et al,                       (Meganathan, 2001) widely shared by <br /> 2002; Picardeau et al, 2003; Kobayashi et al, 2003), and,                        g-proteobacteria. The gene encoding UbiC appears to <br /> according genome annotation, ADP1 both.                     dispensable ADP1 E. coli, indicates <br /> As expected, deletion metY metC lethal                      existence enzyme perform the <br /> intriguingly, metZ mutant methionine auxotroph,                         chorismate pyruvate lyase reaction. Moreover, presence of <br /> suggesting MetY does catalyze direct sulfhydryla-                   essential gene unknown function (ACIAD0383), is <br /> tion conditions. Further analyses initiated                  localized ubiB ubiE ADP1 (yigp E. coli), <br /> define MetZ catalyzes direct sulfhydrylation                     suggests potential implication ubiquinone biosynthesis. <br /> route absent ADP1.                                                         In addition, absence 3-octaprenyl-4-hydroxybenzoate <br />  <br /> &amp; 2008 EMBO Nature Publishing Group                                                                                       Molecular Systems Biology 2008 9 <br />  Collection A. baylyi ADP1 deletion mutants <br /> V Berardinis et al <br />  <br />  <br />  <br />  <br />                                      A. baylyi ADP1                                                                                E. coli <br />                                              NH2                                                                                      NH2 <br />  <br />                                                      OH                                                                                            OH <br />                               HO                                                                                       HO <br />  <br />                                                  O                                                                                   O <br />                                          Homoserine                                                                          Homoserine <br />                                                   Acetyl CoA                                                                         Succinyl CoA <br />                          metW/metX                                                                                       metA <br />                                                    CoenzymeA                                                                         CoenzymeA <br />                                                                                                                   O                             O <br />                                     O                                                                                                 O <br />                                                                                                             HO                                                       OH <br />                                                      O <br />                               HO                                                                                       NH2            O <br />                                                                                                                          O-Succinyl-L-homoserine <br />                                        NH2             O <br />                                     O-Acetyl-L-homoserine                                                             L-Cysteine <br />                         H2S                                            L-Cysteine                                                                  metB <br />  <br />                                    metY                    metZ                                                       Succinate                   O            OH <br />                   Acetate          metZ                                  Acetate                                        O <br />  <br />                                                                                                                                              S <br />                                                                                            O   OH                HO                                            NH2 <br />                         NH2                                       O                                                                 Cystathionine <br />                                                 metC                                                                         NH2 <br />                                 OH                                                 S                                                         H2O <br />           HS                                                 HO                                 NH2 <br />                                                                                                                                                  Pyruvate <br />                          O                 Pyruvate H2O               NH2                                                   metC <br />                  Homocysteine                                                                                                                 NH3 <br />                                                                          Cystathionine <br />                                             NH3                                                                                              NH2 <br />  <br />                                                                                                                                                           OH <br />                                metE                                                                                        HS <br />                                metH                                                                                          Homocysteine <br />                                                                                                                                                    O <br />  <br />                                    NH2                                                                                        metE <br />                                                                                                                               metH <br />                                            OH                                                                                                    NH2 <br />                     S <br />                                                                                                                                                            OH <br />                          Methionine O                                                                                        S <br />                                                                                                                                   Methionine <br />                                                                                                                                                       O <br />  <br /> Figure 7 Methionine biosynthesis. The steps methionine biosynthesis ADP1 E. coli. Essential genes shown red, nonessential genes are <br /> in green. <br />  <br />  <br /> Table III Media used deletion genes involved methionine                  Table IV Functional complementation experiments A. baylyi ADP1 mutants <br /> biosynthesis                                                                           involved acetylation L-homoserine <br />  <br /> Mutants                                            Media                               Genes used              DmetX               DmetW                     DmetX/metW <br />                                                                                        complementation <br /> DmetX                                              Succinate+methionine <br /> DmetW                                              Succinate+methionine                pEVL174a                                                                             <br /> DmetX/metW                                         Succinate+methionine                pEVL174/metXW                   +                     +                            + <br /> DmetY                                              Succinate                           pEVL174/metAb                   +                     +                            + <br /> DmetZ                                              Succinate+methionine                pEVL174/metX                    +                     +                            + <br /> DmetC                                              Succinate                           pEVL174/metW                                          +                              <br /> DmetR                                              Succinate+methionine <br /> DACIAD3524                                         Succinate+methionine                a <br />                                                                                        E. coli-A. baylyi ADP1 shuttle expression vector. <br /> DmetE                                              Succinate+coenzyme-B12              b <br />                                                                                        metA gene E. coli. <br /> DmetH                                              Succinate <br />  <br />                                                                                        possibility presence other <br />                                                                                        decarboxylase fulfilling function. <br /> decarboxylase particularly intriguing. In general, this <br /> activity fulfilled isofunctional genes, ubiX ubiD. <br /> Although genes colocalized, are <br /> always present given genome (see ubiquinone subsystem <br />                                                                                        Analysis gene function pathways using <br /> on SEED platform; http://seed-viewer.theseed.org/). Never-                             growth phenotypes carbon sources <br /> theless, bacteria like <span id='am-160' about='obo:NCBITaxon_442' typeof='owl:Thing obo:NCBITaxon_433 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_441 obo:NCBITaxon_28211 obo:NCBITaxon_204441'><span id='am-161' property="oboInOwl:hasAlternativeId" content="NCBITaxon:136161" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasAlternativeId" content="NCBITaxon:443" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasRelatedSynonym" content="Bacillus oxydans" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasAlternativeId" content="NCBITaxon:89582" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasRelatedSynonym" content="Acetobacter oxydans" datatype="xsd:string"></span><span id='am-168' property="rdfs:label" content="Gluconobacter oxydans" datatype="xsd:string"></span><span id='am-169'  ></span><span id='am-170' property="oboInOwl:hasRelatedSynonym" content="Bacterium oxydans" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasRelatedSynonym" content="Acetomonas oxydans" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasDbXref" content="PMID:15754250" datatype="xsd:string"></span>Gluconobacter oxydans</span>                                  The nearly complete collection mutants analyzed for <br /> Xanthomonas species genes, reinforcing                               growth different carbon sources. Out 220 carbon <br />  <br /> 10 Molecular Systems Biology 2008                                                                                            &amp; 2008 EMBO Nature Publishing Group <br />                                                                                                                  Collection A. baylyi ADP1 deletion mutants <br />                                                                                                                                           V Berardinis et al <br />  <br />  <br />  <br />  <br /> sources tested Phenotype MicroArrays (PM1&amp;2 plates                        final steps Entner&#8211;Doudoroff pathway confirms that <br /> Biolog Inc-CA-, Bochner et al, 2001) in-house growth                        route glucose dissimilation (Supplementary <br /> experiments, 50 led significant growth used                      Annex 2). Moreover, mutants genes unknown <br /> define preliminary set carbon sources. The growth                        function able grow glucose require more <br /> profiling approximately 2450 mutants performed                        investigation. <br /> both solid liquid media (Supplementary Table VI). <br />  <br />                                                                                 Catabolism 2,3-butanediol <br /> Evaluation approach                                                     Using growth phenotyping strategy, carefully <br /> To evaluate specificity sensitivity growth                       explored degradation 2,3-butanediol, histori- <br /> phenotyping approach detecting genes involved cata-                      cally important ADP1, strain initially selected <br /> bolic pathways, tested carbon sources assimilated                        ability use compound carbon source. <br /> through known pathways ADP1. One best character-                      However, catabolic pathway entirely under- <br /> ized catabolic pathways dissimilation quinate,                  stood (for review, Xiao Xu, 2007). Two different <br /> occurs protocatechuate, central metabolite                      routes 2,3-butanediol dissimilation described to <br /> number significant phytochemical degradation pathways                        date: (1) 2,3-butanediol cycle involving 2,3-butanediol <br /> (quinate, benzoate, vanillate, shikimate, on),                   dehydrogenase acetoin dehydrogenase (Juni Heym, <br /> the b-ketoadipate pathway (Stanier Ornston, 1973; Neidle                    1956), generates acetate oxidized (2) <br /> et al, 1991). Quinate dissimilation performed 13 genes                    degradation 2,3-butanediol acetoin dehydrogen- <br /> corresponding 10 enzymatic activities transporters                   ase complex likely generating acetyl-CoA acetalde- <br /> (Supplementary Figure 2). The results obtained                           hyde (Xiao Xu, 2007). To obtain clearer picture this <br /> consistent current knowledge ADP1 quinate                           pathway, mutants profiled growth minimal <br /> assimilation (Supplementary Annex 2). However,                             medium supplemented 2,3-butanediol acetate. A total <br /> mutants (e.g. mutants NADH dehydrogenase subunits,                           21 mutants reproducibly unable grow 2,3- <br /> panD) growth defect explained current                     butanediol, including 8 mutants unable grow acetate <br /> knowledge quinate metabolism require                          (Table V). These results clearly confirm degradation of <br /> more investigation.                                                             2,3-butanediol dependent acetate utilisation, genes of <br />    The glucose dissimilation described biochemically                   glyoxilic shunt (isocitrate lyase/ACIAD1084 malate <br /> by Juni 1978, genes associated                      synthase/ACIAD2335) essential growth 2,3- <br /> known enzymatic activities genome annotation                         butanediol acetate. Moreover, putative transcriptional <br /> process (Barbe et al, 2004). This pathway involves                          regulator (ACIAD1082) essential growth on <br /> transporters (glucose gluconate) genes corre-                      carbon sources, suggesting potential implication in <br /> sponding enzymatic activities (Supplementary Figure 3).                 regulation glyoxilic shunt. Our findings show <br /> Our phenotypic screening growth glucose confirms                     essentiality genes clustered genome the <br /> dissimilation periplasmic oxidation, leading <span id='am-424' about='obo:NCBITaxon_forma' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-425' property="rdfs:label" content="forma" datatype="xsd:string"></span><span id='am-426' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>forma</span>-                   putative transcriptional regulator acetoin metabolism <br /> tion gluconate, assimilated                     (ACIAD1014), homology acoR three <br />  <br />  <br /> Table V Summary 2,3 butanediol acetate growth analysis based high throughput data confirmed individual analysis <br />  <br /> Name               Gene product                                                                                              2,3 butanediol           Acetate <br />  <br /> ACIAD0772          Putative ATP-binding protein                                                                                                          + <br /> ACIAD0808          Putative protease                                                                                                                     + <br /> ACIAD1014          Putative transcriptional regulator                                                                                                    + <br /> ACIAD1017          Acetoin:2,6-dichlorophenolindophenol oxidoreductase subunit (acoA)                                                                  + <br /> ACIAD1018          Acetoin:2,6-dichlorophenolindophenol oxidoreductase b subunit (acoB)                                                                  + <br /> ACIAD1019          Dihydrolipoamide acetyltransferase (acoC)                                                                                             + <br /> ACIAD1021          Putative (R,R)-butanediol dehydrogenase                                                                   Growth defect               + <br /> ACIAD1052          Putative membrane-bound protein GNT I transport                                                                             + <br /> ACIAD1550          Putative oxidoreductase, short-chain dehydrogenase/reductase <span id='am-123' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-124' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-125' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span>                                                                   + <br /> ACIAD2018          Aldehyde dehydrogenase (ald1)                                                                                                         + <br /> ACIAD2084          Putative transcriptional regulator                                                                                                    + <br /> ACIAD2346          Integration host factor (himA)                                                                                                        + <br /> ACIAD3071          Cysteine synthase B (cysM)                                                                                                            + <br /> ACIAD3549          g-Glutamate-cysteine ligase (gshA)                                                                                                    + <br /> ACIAD0273          DnaK suppressor protein (dksA)                                                                            Growth defect                 <br /> ACIAD0922          Putative D-amino acid oxidase                                                                                                           <br /> ACIAD1082          Putative transcriptional regulator                                                                                                      <br /> ACIAD1084          Isocitrate lyase (aceA)                                                                                                                 <br /> ACIAD1135          Putative NADH pyrophosphatase (NUDIX hydrolase family)                                                                                  <br /> ACIAD1951          Putative thioesterase                                                                                                                   <br /> ACIAD2236          Putative bifunctional protein (dGTP-pyrophosphohydrolase; thiamine phosphate synthase)                                                  <br /> ACIAD2335          Malate synthase G (glcB)                                                                                                                <br /> ACIAD3469          Putative two-component response regulator                                                                                               <br />  <br />  <br /> &amp; 2008 EMBO Nature Publishing Group                                                                                   Molecular Systems Biology 2008 11 <br />  Collection A. baylyi ADP1 deletion mutants <br /> V Berardinis et al <br />  <br />  <br />  <br />  <br />                       <span id='am-328' about='obo:NCBITaxon_636' typeof='owl:Thing obo:NCBITaxon_635 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-329' property="oboInOwl:hasRelatedSynonym" content="Paracolobactrum anguillimortiferum" datatype="xsd:string"></span><span id='am-330' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-331' property="rdfs:label" content="Edwardsiella tarda" datatype="xsd:string"></span><span id='am-332' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-333'  ></span><span id='am-334' property="oboInOwl:hasRelatedSynonym" content="Edwardsiella anguillimortifera" datatype="xsd:string"></span>Edwardsiella tarda</span> <br />                                             P       A B   C       D   E       F       G       H            I       J   K   L   M   N       O <br />  <br />  <br />  <br />  <br />                       Acinetobacter baylyi ADP1 <br />                                                 J             O           G           F   D       C   B        A       H           K   M           N <br />  <br />  <br />  <br />                                        ACIAD2681                                                                                                           ACIAD2699 <br />  <br />  <br />  <br />  <br />                       Acinetobacter baumanii AYE <br />                                                               N           M       K           H        J               O           G   F       D       C   B     A <br />  <br />  <br />                                            ABAYE2399 <br />                                                                                                                                                                ABAYE2416 <br />  <br /> Figure 8 Acinetobacter genomic organization evp homolgous gene cluster involved type VI protein secretion E. tarda. Genes green are <br /> those homologous E. tarda genes; genes orange conserved domain (COG PUF domain) E. tarda. Genes gray genes known <br /> function annotated conserved hypothetical proteins Acinetobacter species. <br />  <br />  <br />  <br />  <br /> of subunits acetoin dehydrogenase complex                                             ACIAD2681&#8211;2699 characterized large number genes <br /> (acoABC/ACIAD1017-1019). The dispensability acoL                                               unknown function (panel B). Moreover, closer study of <br /> gene (ACIAD1020) encoding dihydrolipoamide dehydrogen-                                              cluster revealed 12 genes present a <br /> ase explained presence Lpd, isofunctional                                       genomic organization similar evp operon Edwardsiella <br /> enzyme involved pyruvate/oxoglutarate dehydrogenase                                            tarda (Figure 8), encodes type VI protein secretion <br /> complex. The deletion putative butanediol dehydrogenase                                        (Zheng Leung, 2007). Furthermore, ADP1 <br /> (ACIAD1021) results severe growth impairment                                                genes synteny Acinetobacter baumanii AYE <br /> mutant, indicating dehydrogenase perform                                             (Vallenet et al, 2008), <span id='am-406' about='obo:NCBITaxon_48736' typeof='owl:Thing obo:NCBITaxon_80840 obo:NCBITaxon_28216 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_119060 obo:NCBITaxon_2'><span id='am-407' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-408' property="oboInOwl:hasRelatedSynonym" content="Ralstonia Yabuuchi et al. 1996" datatype="xsd:string"></span><span id='am-409'  ></span><span id='am-410' property="oboInOwl:hasDbXref" content="PMID:8657018" datatype="xsd:string"></span><span id='am-411' property="rdfs:label" content="Ralstonia" datatype="xsd:string"></span><span id='am-412' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Ralstonia</span> solanaecerum, <span id='am-144' about='obo:NCBITaxon_73029' typeof='owl:Thing obo:NCBITaxon_206389 obo:NCBITaxon_28216 obo:NCBITaxon_75787 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2'><span id='am-145' property="oboInOwl:hasRelatedSynonym" content="Dechlorimonas" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasRelatedSynonym" content="Dechloromonas Achenbach et al. 2001" datatype="xsd:string"></span><span id='am-147'  ></span><span id='am-148' property="oboInOwl:hasDbXref" content="PMID:11321099" datatype="xsd:string"></span><span id='am-149' property="rdfs:label" content="Dechloromonas" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Dechloromonas</span> <br /> this function, albeit weakly. The data clearly                                          aromatica <span id='am-464' about='obo:NCBITaxon_32008' typeof='owl:Thing obo:NCBITaxon_80840 obo:NCBITaxon_28216 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_119060 obo:NCBITaxon_2'><span id='am-465' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-466' property="oboInOwl:hasDbXref" content="PMID:10555347" datatype="xsd:string"></span><span id='am-467' property="oboInOwl:hasRelatedSynonym" content="Burkholderia Yabuuchi et al. 1993 emend. Gillis et al. 1995" datatype="xsd:string"></span><span id='am-468' property="rdfs:label" content="Burkholderia" datatype="xsd:string"></span><span id='am-469' property="oboInOwl:hasDbXref" content="PMID:1283774" datatype="xsd:string"></span><span id='am-470' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas RNA homology group II" datatype="xsd:string"></span><span id='am-471' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-472'  ></span>Burkholderia</span> species showing <span id='am-136' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-137' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-138' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-140'  ></span>homo</span>- <br /> putative acetaldehyde dehydrogenase (ACIAD2018)                                               logies 50%. Mutants additional genes (ACIAD0852, <br /> involved 2,3-butanediol degradation. Altogether                                              ACIAD0854, ACIAD1205 ACIAD2603) encoding proteins <br /> results strongly suggest 2,3-butanediol degraded                                         related bacterioferritin produced larger colonies on <br /> two C2 compounds, similar Bacillus subtilis (Lopez et al,                                       solid media. <br /> 1975), <span id='am-392' about='obo:NCBITaxon_19' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_68525 obo:NCBITaxon_131567 obo:NCBITaxon_69541 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_28221 obo:NCBITaxon_213423 obo:NCBITaxon_18'><span id='am-393' property="rdfs:label" content="Pelobacter carbinolicus" datatype="xsd:string"></span><span id='am-394' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-395'  ></span><span id='am-396' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Pelobacter carbinolicus</span> (Oppermann et al, 1991) and <br /> <span id='am-40' about='obo:NCBITaxon_1485' typeof='owl:Thing obo:NCBITaxon_186802 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_186801 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_31979'><span id='am-41' property="rdfs:label" content="Clostridium" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Clostridium</span> magnum (Kruger et al, 1994). Moreover, the <br /> acetoin dehydrogenase (budC/ACIAD1022) present ADP1                                             Conclusion <br /> and deletion does impair growth 2,3-butanediol                                         We took advantage natural competence ADP1 <br /> on acetate, reinforcing probable absence 2,3-                                              combined homology-directed recombination linear <br /> butanediol cycle ADP1. Furthermore, detected                                               DNA construct complete collection gene-by-gene <br /> several genes encoding proteins putative functions                                         deletion mutants selected minimal medium. A total of <br /> that essential growth 2,3-butanediol,                                           2594 genes covering 81% predicted genes ADP1 were <br /> acetate. The implication genes specific                                              successfully disrupted, 499 genes, including 47 genes of <br /> catabolic pathway needs analyzed complete                                            unknown function, essential prototrophic <br /> our understanding 2,3-butanediol degradation.                                                      life. The library currently completed auxotrophic <br />                                                                                                       mutants, genes involved amino-acid <br />                                                                                                       biosynthetic pathways. A simple examination essential/ <br /> Growth phenotype solid media                                                                       dispensable genes highlighted discrepancies <br /> The analysis mutant collection growth phenotypes                                           experimental data, current knowledge <br /> on agar plates supplemented various carbon sources                                           metabolic pathways, led identification of <br /> allowed identification mutants generate colonies                                          interesting new features differences known pathways <br /> with size twice average. This phenotype                                              bacteria. It noted the <br /> suggest membrane cell adhesion disorder. The distribution                                        discrepancies automatically detected using the <br /> of genes, gathered main clusters genome,                                       computational model ADP1 metabolic network (Durot <br /> shown Supplementary Figure 4. One cluster located                                               et al, preparation), confirmed efficiency and <br /> between ACIAD0068 ACIAD0102 composed                                                    importance silico models identify quickly the <br /> genes involved exopolysaccharide biosynthesis (panel A).                                           differences annotation-based knowledge of <br /> When grown liquid medium, mutants usually                                                    metabolism set experimental data derived a <br /> aggregate leading apparent null OD. The cluster,                                          mutant library. <br />  <br /> 12 Molecular Systems Biology 2008                                                                                                                  &amp; 2008 EMBO Nature Publishing Group <br />                                                                                                               Collection A. baylyi ADP1 deletion mutants <br />                                                                                                                                        V Berardinis et al <br />  <br />  <br />  <br />  <br />    This collection provides powerful tool                      AT-rich regions (AT% higher 70%). The primers P4 P5 were <br /> exploration gene function, particular, catabolic pathways               extended complementary sequences integration cassette <br />                                                                               amplification primers described (Metzgar et al, 2004). Primers <br /> through analysis growth phenotypes defined                          P3 P6 amplification flanking regions chosen to <br /> compounds assimilated. A number essential                      obtain regions 300&#177;50 bp. The &#8216;external primers&#8217; P7 P8, <br /> genes catabolic pathways                           used verification correct locus integration, chosen to <br /> identified pathways remain investigated                     800&#177;200 pb upstream downstream target gene, <br />                                                                               respectively. The primers used mutant construction are <br /> using different carbon nitrogen sources. Furthermore,                  available Artemis interface MaGe platform. <br /> clustering similar growth phenotypic profiles different <br /> mutants allow identification potential role for <br /> genes unknown function comparison well-                          Generation integration cassette <br /> known genes (Ito et al, 2005; Dudley et al, 2005).                            The strategy followed obtain gene deletions depicted Figure 1. <br />    This new collection mutants likely valuable             Splicing PCR performed described previously (Murphy et al, <br /> resource bacteria, ADP1 shares large number                 2000; Metzgar et al, 2004) generate recombinant DNA fragments <br />                                                                               consisting tdk/kan cassette flanked 250&#8211;350 pb chromoso- <br /> genes bacteria. Genetic tools developed ADP1,                  mal regions upstream downstream gene deleted. PCR <br /> such multiple gene deletions existence E. coli/                 amplify tdk/kan markers positive negative (tdk gene <br /> ADP1 shuttle <span id='am-402' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-403' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-404' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-405' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> (Metzgar et al, 2004), allow                      used generate unmarked deletions construct double mutants) <br /> heterologous functional complementation. ADP1                      selection performed plasmid pEVL386. We constructed <br />                                                                               <span id='am-453' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-454' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-455' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-456' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-457' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> pUC18-derived E. coli/ADP1 shuttle vector <br /> be alternative complementary model study                    pKT40 (Metzgar et al, 2004). The plasmid pEVL174 constructed <br /> gene function bacteria, especially                pKT40 insertion tdk gene expression cassette, <br /> easily amenable genetic manipulations.                                     containing E. coli tdk gene control bacteriophage <br />    In conclusion, collection important               T5 promoter EcoRI/SacI sites. The plasmid pEVL386 derivative <br /> identification gene function, alternative pathways                     plasmid pEVL174 containing ahp gene kanamycin resistance <br />                                                                               inserted XbaI/PstI restriction sites. PCR amplify upstream <br /> regulatory networks, genome-scale reconstruc-                downstream flanking regions recombinant PCR were <br /> tion ADP1 metabolism based high throughput phenotyping                  carried following published protocols (Metzgar et al, 2004). PCRs <br /> and essentiality data (Oh et al, 2007; Durot et al, preparation).          performed 96-well microplates using 1.5 ml overnight <br /> Moreover, precisely analyze effect genetic perturbations            culture ADP1. At step, 96 PCR products run 1% <br />                                                                               agarose electrophoresis gel using 0.5   TBE buffer analyzed <br /> at metabolite level, initiated large-scale                    manually length. <br /> systematic study mutant collection using mass spectro- <br /> metry aiming collecting information metabolome of <br /> several mutants. The comparison metabolic                    Transformation mutant selection <br /> profiles provide information effect well-defined             A 5 ml volume overnight culture ADP1 added 500 ml <br /> mutation quantitative/qualitative variation metabolites                 MAS 96-well microtiter plates. After 2 h culture, 100 ml the <br /> and great importance collecting                      recombinant PCR product added culture. The microtiter <br /> information genes unknown function, providing                plates incubated shaking 301C 24 h spread onto <br />                                                                               MASK agar plates. The number clones time clone <br /> a dynamic global view ADP1 metabolism.                                 appearance stored database. For mutant, colonies <br />                                                                               isolated twice verifications. Three PCRs were <br />                                                                               carried test correct structure mutant. Two use <br /> Materials methods                                                         combination locus-specific kanamycin primers (P3/P2 and <br />                                                                               P6/P1) uses couple external primers (P7/P8). PCR <br /> Bacteria, media, chemicals reagents                                       reactions carried described previously, this <br /> Cells routinely grown MA (Medium Acinetobacter) minimal            case 1 ml overnight mutant culture used. The P7/P8 PCR <br /> medium (31 mM Na2HPO4, 25 mM KH2PO4, 18 mM NH4Cl, 41 mM                       products sequenced primers S1 (50 -CTCCTTCATTACA <br /> nitrilotriacetic acid, 2 mM MgSO4, 0.45 mM CaCl2, 3 mM FeCl3, 1 mM            GAAACGGC) S2 (50 -CTTACCCGCATTCATTGCGG) specific for <br /> MnCl2, 1 mM ZnCl2, 0.3 mM (CrCl3, H3BO3, CoCl2, CuCl2, NiCl2,                 integration cassette (see Figure 1), integration site was <br /> Na2MoO2, Na2SeO3)) supplemented 25 mM carbon source                   verified. Mutants stored individually microplates  801C <br /> and used (MASK) (MAS) kanamycin 30 mg/ml. The              MASK supplemented 10% glycerol. <br /> carbon sources chemicals Sigma-Aldrich. In <br /> special cases (e.g., deletion genes involved amino acid <br /> biosynthesis), MAS/MASK complemented 1 mM                 Auxotrophy complementation assays <br /> appropriate amino acid compound.                                              About 30 mutants genes involved amino acid biosynthesis not <br />                                                                               obtained MASK medium high-throughput process were <br />                                                                               obtained presence 1 mM appropriate amino acid (arginine, <br /> PCR primers                                                                   methionine, histidine, leucine, tryptophan) auxotrophy was <br />                                                                               confirmed MASK. To perform complementation tests, the <br /> The choice primers P3&#8211;P8 (Figure 1) based Primer3 <br />                                                                               coding sequences E. coli metA metX, metW, metXW, metYand of <br /> program (Rozen Skaletsky, 2000). All primers chosen as <br />                                                                               ACIAD3524 ADP1 cloned PacI/NotI sites plasmid <br /> close possible 20 bp melting temperature 551C maintain <br />                                                                               pEVL174 (primers shown Supplementary Table VII). The correct <br /> consistent PCR conditions constructions. When two <br />                                                                               sequence plasmid construction confirmed before <br /> neighboring genes orientation, primers P4 P5 <br />                                                                               transformation. <br /> used deletion target gene designed preserve least <br /> 25 nucleotides 50 region flanking gene ribosome- <br /> binding site intact. The preserved region extended 75 nucleotides <br /> when genes located opposite strands, preserve             Growth tests <br /> ribosome-binding site include promoter region. These rules         For growth tests, overnight preculture inoculated 5 ml of <br /> were relaxed 100 nucleotides inside target gene          glycerol stock solution. Liquid growth studies carried in <br />  <br /> &amp; 2008 EMBO Nature Publishing Group                                                                                Molecular Systems Biology 2008 13 <br />  Collection A. baylyi ADP1 deletion mutants <br /> V Berardinis et al <br />  <br />  <br />  <br /> duplicate solid growth triplicate, starting independent           <span id='am-443' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-444' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-445' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-446' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-447' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span>Escherichia coli K-12</span> in-frame, single-gene knockout mutants: the <br /> precultures. Mutants tested growth 200 ml MA              Keio collection. Mol Syst Biol 2: 2006.0008 <br /> 25 mM carbon source directly inoculated preculture      Barbe V, Vallenet D, Fonknechten N, Kreimeyer A, Oztas S, Labarre L, <br /> using 96 inoculation pins (Genetix) incubated 301C 24 h       Cruveiller S, Robert C, Duprat S, Wincker P, Ornston LN, <br /> shaking. The plates closed air pore pad (Qiagen) correct <br />                                                                               Weissenbach J, Marliere P, Cohen GN, Medigue C (2004) Unique <br /> aeration. The 301C room equipped Ku&#168;hner shakers allows <br />                                                                               features revealed genome sequence Acinetobacter sp. <br /> incubation 240 plates simultaneously (four carbon/nitrogen sources <br /> in duplicate experiments instance). The absorbance 600 nm           ADP1, versatile naturally transformation competent <br /> the cultures measured plate reader (Spectramax 384plus,            bacterium. Nucleic Acids Res 32: 5766&#8211;5779 <br /> Molecular Devices). Normalization OD data performed       Barras F, Loiseau L, Py B (2005) How Escherichia coli and <br /> plate. Discrepancies replications noted data           <span id='am-99' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-114' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-115'  ></span>Saccharomyces cerevisiae</span> build Fe/S proteins. Adv Microb Physiol <br /> were excluded analyses. The mutants auxotrophic         50: 41&#8211;101 <br /> acetate or/and 2,3-butanediol high-throughput analyses       Bochner BR, Gadzinski P, Panomitros E (2001) Phenotype microarrays <br /> were confirmed individually. Solid growth phenotypes scored           high-throughput phenotypic testing assay gene function. <br /> replica printing entire collection agar containing MA           Genome Res 11: 1246&#8211;1255 <br /> 25 mM carbon sources. Twelve 96-well overnight culture plates          Brichta DM, Azad KN, Ralli P, O&#8217;Donovan GA (2004) Pseudomonas <br /> could replicated agar plate (20   20 cm) using colony-            aeruginosa dihydroorotases: tale pyrCs. Arch Microbiol <br /> picking robot (Qbot Genetix) 96-pin head. The replicas            182: 7&#8211;17 <br /> were incubated 24 36 h 301C, photographed analyzed          Chakravorty A, Klovstad M, Peterson G, Lindeman R, Gregg-Jolly LA <br /> automatically using picking program robot. Normalization        (2008) Sensitivity Acinetobacter baylyi mpl mutant DNA <br /> colony diameter performed using 12 wild-type colonies spotted          damage. Appl Environ Microbiol 74: 1273&#8211;1275 <br /> each plate.                                                               Vries J, Wackernagel W (2002) Integration foreign DNA during <br />                                                                               natural transformation Acinetobacter sp. homology-facilitated <br />                                                                               illegitimate recombination. Proc Natl Acad Sci USA 99: 2094&#8211;2099 <br /> Distribution ADP1 mutant collection                                Dudley AM, Janse DM, Tanay A, Shamir R, Church GM (2005) A global <br />                                                                               view pleiotropy phenotypically derived gene function in <br /> Individual mutants distributed used several <br /> studies DNA damage (Chakravorty et al, 2008) analysis             yeast. Mol Syst Biol 1: 2005.0001 <br /> of metabolism (Aghaie et al, preparation). The mutant distribution     Fassbinder F, Kist M, Bereswill S (2000) Structural functional <br /> is handled Genoscope web page including                       analysis riboflavin synthesis genes encoding GTP <br /> primers data used deletions (http://www.genoscope.cns.fr/spip/            cyclohydrolase II (ribA), DHBP synthase (ribBA), riboflavin <br /> Strain-request-for-mutants-of,749.html).                                      synthase (ribC), riboflavin deaminase/reductase (ribD) from <br />                                                                               <span id='am-262' about='obo:NCBITaxon_209' typeof='owl:Thing obo:NCBITaxon_72293 obo:NCBITaxon_213849 obo:NCBITaxon_1 obo:NCBITaxon_68525 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_29547'><span id='am-263'  ></span><span id='am-264' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-265' property="oboInOwl:hasDbXref" content="PMID:11156001" datatype="xsd:string"></span><span id='am-266' property="oboInOwl:hasAlternativeId" content="NCBITaxon:28203" datatype="xsd:string"></span><span id='am-267' property="rdfs:label" content="Helicobacter" datatype="xsd:string"></span><span id='am-268' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-269' property="oboInOwl:hasDbXref" content="PMID:1704793" datatype="xsd:string"></span><span id='am-270' property="oboInOwl:hasDbXref" content="PMID:15143020" datatype="xsd:string"></span><span id='am-271' property="oboInOwl:hasDbXref" content="PMID:17329766" datatype="xsd:string"></span>Helicobacter</span> pylori strain P1. FEMS Microbiol Lett 191: 191&#8211;197 <br />                                                                           Fontecave M, Choudens SO, Py B, Barras F (2005) Mechanisms iron- <br />                                                                               sulfur cluster assembly: SUF machinery. J Biol Inorg Chem 10: <br /> Supplementary information                                                     713&#8211;721 <br /> Supplementary information available Molecular Systems           Gerdes S, Edwards R, Kubal M, Fonstein M, Stevens R, Osterman A <br /> Biology website (www.nature.com/msb).                                         (2006) Essential genes metabolic maps. Curr Opin Biotechnol 17: <br />                                                                               448&#8211;456 <br />                                                                           Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, Daugherty <br />                                                                               MD, Somera AL, Kyrpides NC, Anderson I, Gelfand MS., <br /> Acknowledgements                                                              Bhattacharya A, Kapatral V, D0 Souza M, Baev MV, Grechkin Y, <br /> We thank Genoscope bioinformatic team support              Mseeh F, Fonstein MY, Overbeek R, Barabasi AL, Oltvai ZN, <br /> the project. We grateful AGC team dynamic reactive       Osterman AL (2003) Experimental determination level <br /> support development MaGe platform. We especially thank          analysis essential genes Escherichia coli MG1655. J Bacteriol <br /> Genoscope services team daily support Jean-Louis Petit,        185: 5673&#8211;5684 <br /> Ce&#180;line Orvain-Durand Laurie Bertrand help. We thank        Giaever G et al. (2002) Functional profiling Saccharomyces <br /> Vale&#180;rie Barbe, Nuria Fonknechten Alain Perret critical reading       cerevisiae genome. Nature 418: 387&#8211;391 <br /> and Susan Cure correcting manuscript. We deeply indebted      Grill S, Yamaguchi H, Wagner H, Zwahlen L, Kusch U, Mack M (2007) <br /> to Jamile Hazan improving manuscript. This project                Identification characterization <span id='am-433' about='obo:NCBITaxon_348043' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_2062 obo:NCBITaxon_201174 obo:NCBITaxon_1883 obo:NCBITaxon_85003 obo:NCBITaxon_85011'><span id='am-434' property="rdfs:label" content="Streptomyces davawensis" datatype="xsd:string"></span><span id='am-435' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-436'  ></span><span id='am-437' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Streptomyces davawensis</span> <br /> financially supported Consortium National Recherche en                  riboflavin biosynthesis gene clusters. Arch Microbiol 188: 377&#8211;387, <br /> Ge&#180;nomique.                                                                   advance online publication 1 June 2007; doi: advance online <br />                                                                               publication 8 July 2004; doi: 10.1007/s00203-007-0258-1 <br />                                                                           Hwang BJ, Yeom HJ, Kim Y, Lee HS (2002) Corynebacterium <br />                                                                               glutamicum utilizes transsulfuration direct <br /> References                                                                    sulfhydrylation pathways methionine biosynthesis. J Bacteriol <br />                                                                               184: 1277&#8211;1286 <br /> Abd El-Haleem D (2003) Acinetobacter: Environmental                   Ito M, Baba T, Mori H, Mori H (2005) Functional analysis 1440 <br />    Biotechnological Applications. Af J Biotechnol 2: 71&#8211;74                    Escherichia coli genes using combination knock-out library <br /> Akerley BJ, Rubin EJ, Novick VL, Amaya K, Judson N, Mekalanos JJ              phenotype microarrays. Metab Eng 7: 318&#8211;327 <br />    (2002) A genome-scale analysis identification genes required    Jacobs MA, Alwood A, Thaipisuttikul I, Spencer D, Haugen E, Ernst S, <br />    growth survival <span id='am-321' about='obo:NCBITaxon_724' typeof='owl:Thing obo:NCBITaxon_135625 obo:NCBITaxon_712 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236'><span id='am-322' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-323' property="rdfs:label" content="Haemophilus" datatype="xsd:string"></span><span id='am-324' property="oboInOwl:hasRelatedSynonym" content="Haemophilus Winslow et al. 1917" datatype="xsd:string"></span><span id='am-325'  ></span><span id='am-326' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-327' property="oboInOwl:hasDbXref" content="PMID:1736960" datatype="xsd:string"></span>Haemophilus</span> influenzae. Proc Natl Acad           Will O, Kaul R, Raymond C, Levy R, Chun-Rong L, Guenthner D, <br />    Sci USA 99: 966&#8211;971                                                        Bovee D, Olson MV, Manoil C (2003) Comprehensive transposon <br /> Alaminos M, Ramos JL (2001) The methionine biosynthetic pathway               mutant library Pseudomonas aeruginosa. Proc Natl Acad Sci USA <br />    homoserine <span id='am-217' about='obo:NCBITaxon_303' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_286 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621 obo:NCBITaxon_136845'><span id='am-218' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931964" datatype="xsd:string"></span><span id='am-219' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas arvilla" datatype="xsd:string"></span><span id='am-220' property="oboInOwl:hasDbXref" content="PMID:11211254" datatype="xsd:string"></span><span id='am-221' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas rugosa" datatype="xsd:string"></span><span id='am-222' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1416793" datatype="xsd:string"></span><span id='am-223' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931954" datatype="xsd:string"></span><span id='am-224' property="oboInOwl:hasAlternativeId" content="NCBITaxon:126995" datatype="xsd:string"></span><span id='am-225' property="oboInOwl:hasRelatedSynonym" content="Bacillus fluorescens putidus" datatype="xsd:string"></span><span id='am-226' property="oboInOwl:hasAlternativeId" content="NCBITaxon:105847" datatype="xsd:string"></span><span id='am-227' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas convexa" datatype="xsd:string"></span><span id='am-228' property="oboInOwl:hasAlternativeId" content="NCBITaxon:691260" datatype="xsd:string"></span><span id='am-229' property="oboInOwl:hasDbXref" content="PMID:9734035" datatype="xsd:string"></span><span id='am-230' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas eisenbergii" datatype="xsd:string"></span><span id='am-231' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas sp. PX-A-Ps.sp2" datatype="xsd:string"></span><span id='am-232' property="oboInOwl:hasDbXref" content="PMID:1177780" datatype="xsd:string"></span><span id='am-233' property="oboInOwl:hasAlternativeId" content="NCBITaxon:760256" datatype="xsd:string"></span><span id='am-234' property="oboInOwl:hasRelatedSynonym" content="Bacillus putidus" datatype="xsd:string"></span><span id='am-235' property="oboInOwl:hasAlternativeId" content="NCBITaxon:79218" datatype="xsd:string"></span><span id='am-236' property="oboInOwl:hasAlternativeId" content="NCBITaxon:29437" datatype="xsd:string"></span><span id='am-237' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas striata" datatype="xsd:string"></span><span id='am-238' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-239'  ></span><span id='am-240' property="rdfs:label" content="Pseudomonas putida" datatype="xsd:string"></span><span id='am-241' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas ovalis" datatype="xsd:string"></span><span id='am-242' property="oboInOwl:hasRelatedSynonym" content="Arthrobacter siderocapsulatus" datatype="xsd:string"></span><span id='am-243' property="oboInOwl:hasAlternativeId" content="NCBITaxon:668614" datatype="xsd:string"></span><span id='am-244' property="oboInOwl:hasDbXref" content="PMID:17978216" datatype="xsd:string"></span><span id='am-245' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-246' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas incognita" datatype="xsd:string"></span><span id='am-247' property="oboInOwl:hasRelatedSynonym" content="Pseudomanas putida" datatype="xsd:string"></span>Pseudomonas putida</span> involves metW, metX,             100: 14339&#8211;14344 <br />    metZ, metH metE gene products. Arch Microbiol 176: 151&#8211;154         Juni E (1978) Genetics physiology Acinetobacter. Annu Rev <br /> Andersen GL, Beattie GA, Lindow SE (1998) Molecular                           Microbiol 32: 349&#8211;371 <br />    characterization sequence methionine biosynthetic locus       Juni E, Heym GA (1956) A cyclic pathway bacterial <br />    Pseudomonas syringae. J Bacteriol 180: 4497&#8211;4507                      dissimilation 2,3-butanediol, acetylmethylcarbinol, and <br /> Baba T, <span id='am-126' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-127' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-130'  ></span>Ara</span> T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko               diacetyl. I. General aspects 2,3-butanediol cycle. J Bacteriol <br />    KA, Tomita M, Wanner BL, Mori H (2006) Construction                     71: 425&#8211;432 <br />  <br /> 14 Molecular Systems Biology 2008                                                                           &amp; 2008 EMBO Nature Publishing Group <br />                                                                                                        Collection A. baylyi ADP1 deletion mutants <br />                                                                                                                                 V Berardinis et al <br />  <br />  <br />  <br />  <br /> Kang Y, Durfee T, Glasner JD, Qiu Y, Frisch D, Winterberg KM, Blattner   Reams AB, Neidle EL (2004) Selection gene clustering tandem <br />    FR (2004) Systematic mutagenesis Escherichia coli genome.         duplication. Annu Rev Microbiol 58: 119&#8211;142 <br />    J Bacteriol 186: 4921&#8211;4930                                            Riley M, Abe T, Arnaud MB, Berlyn MK, Blattner FR, Chaudhuri RR, <br /> Kobayashi K, Kondo M, Fukuda H, Nishimura M, Ohta H (2003)                  Glasner JD, Horiuchi T, Keseler IM, Kosuge T, Mori H, <span id='am-448' about='obo:NCBITaxon_38948' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6547 obo:NCBITaxon_106220 obo:NCBITaxon_6546 obo:NCBITaxon_6545 obo:NCBITaxon_6544 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_301959 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-449' property="rdfs:label" content="Perna" datatype="xsd:string"></span><span id='am-450'  ></span><span id='am-451' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-452' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Perna</span> NT, <br />    Essential Bacillus subtilis genes. Proc Natl Acad Sci USA 100:           Plunkett III G, Rudd KE, Serres MH, Thomas GH, Thomson NR, <br />    4678&#8211;4683                                                                Wishart D, Wanner BL (2006) Escherichia coli K-12: cooperatively <br /> Kruger N, Oppermann FB, Lorenzl H, Steinbuchel A (1994)                     developed annotation snapshot&#8212;2005. Nucleic Acids Res 34: 1&#8211;9 <br />    Biochemical molecular characterization Clostridium         Rozen S, Skaletsky H (2000) Primer3 WWW general users <br />    magnum acetoin dehydrogenase enzyme system. J Bacteriol 176:             biologist programmers. Methods Mol Biol 132: 365&#8211;386 <br />    3614&#8211;3630                                                             Srinivasa Rao PS, Yamada Y, Leung KY (2003) A major catalase (KatB) <br /> Kruse T, Bork-Jensen J, Gerdes K (2005) The morphogenetic MreBCD            required resistance H2O2 phagocyte-mediated <br />    proteins Escherichia coli form essential membrane-bound            killing Edwardsiella tarda. Microbiology 149: 2635&#8211;2644 <br />    complex. Mol Microbiol 55: 78&#8211;89                                      Stanier RY, Ornston LN (1973) The <span id='am-93' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-96'  ></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>beta</span>-ketoadipate pathway. Adv <br /> Liberati NT, Urbach JM, Miyata S, Lee DG, Drenkard E, Wu G,                 Microb Physiol 9: 89&#8211;151 <br />    Villanueva J, Wei T, Ausubel FM (2006) An ordered, nonredundant       Suzuki N, Okai N, Nonaka H, Tsuge Y, Inui M, Yukawa H (2006) High- <br />    library Pseudomonas aeruginosa strain PA14 transposon                 throughput transposon mutagenesis Corynebacterium <br />    insertion mutants. Proc Natl Acad Sci USA 103: 2833&#8211;2838                 glutamicum construction single-gene disruptant mutant <br /> Lopez JM, Thoms B, Rehbein H (1975) Acetoin degradation Bacillus         library. Appl Environ Microbiol 72: 3750&#8211;3755 <br />    subtilis direct oxidative cleavage. Eur J Biochem 57: 425&#8211;430      Val <br /> </body></html>