<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Interplay Mre11 Nuclease Dna2 plus Sgs1 in <br /> Rad51-Dependent Recombinational Repair <br /> Martin E. Budd, Judith L. Campbell* <br /> Divisions Biology Chemistry, Caltech, Braun Laboratories, Pasadena, <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-6'  ></span>California</span>, United States America <br />  <br />  <br />  <br />      Abstract <br />      The Mre11/Rad50/Xrs2 complex initiates IR repair binding end double-strand break, resulting 59 39 <br />      exonuclease degradation creating single-stranded 39 overhang competent strand invasion unbroken <br />      chromosome. The nuclease(s) involved understood. Mre11 encodes nuclease, 39 59, 59 <br />      39 activity. Furthermore, mutations inactivate nuclease activity Mre11 repair functions, <br />      mre11-D56N mre11-H125N, resistant IR. <span id='am-76' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-77' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79'  ></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> suggests nuclease catalyze 59 39 degradation. <br />      One candidate nuclease tested date encoded essential gene Dna2 helicase/ <br />      nuclease. We recently reported ability suppress lethality dna2D pif1D. The dna2D pif1D mutant IR- <br />      resistant. We determined dna2D pif1D mre11-D56N dna2D pif1D mre11-H125N strains equally sensitive <br />      IR mre11D strains, suggesting absence Dna2, Mre11 nuclease carries repair. The dna2D pif1D mre11- <br />      D56N triple mutant complemented <span id='am-110' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-111' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-114' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> expressing Mre11, Dna2 dna2K1080E, mutant defective helicase <br />      functional nuclease, demonstrating nuclease Dna2 compensates absence Mre11 nuclease IR <br />      repair, presumably 59 39 degradation DSB ends. We sgs1D mre11-H125N, sgs1D, very <br />      sensitive IR, implicating Sgs1 helicase Dna2-mediated pathway. <br />  <br />   Citation: Budd ME, Campbell JL (2009) Interplay Mre11 Nuclease Dna2 plus Sgs1 Rad51-Dependent Recombinational Repair. PLoS ONE 4(1): e4267. <br />   doi:10.1371/journal.pone.0004267 <br />   Editor: Michael Lichten, National <span id='am-66' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-67' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-68'  ></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Cancer</span> Institute, United States America <br />   Received December 11, 2008; Accepted December 22, 2008; Published January 23, 2009 <br />   Copyright: &#223; 2009 Campbell et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported NIH GM087666, Philip Morris External Research Program, Ellison Foundation. The funders role study <br />   design, <span id='am-105' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-106' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-108'  ></span><span id='am-109' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: jcampbel@caltech.edu <br />  <br />  <br />  <br /> Introduction                                                                          activity double-stranded DNA single-strand endonuclease <br />                                                                                       activity DNA hairpins [6&#8211;8]. Rad50 ATP binding protein, <br />    Double-strand breaks (DSBs) major result endogenous                       Walker A motif N terminus Walker B motif in <br /> DNA replication errors exogenous DNA damaging agents,                          C terminus separated long heptad repeat domain [9]. A <br /> and failure repair breaks leads gross chromosomal                      signature (arginine finger) region C terminus of <br /> rearrangements characteristic tumor cells. In yeast, homologous                    ATP binding motif [10]. Mutations ATP binding motif <br /> recombination (HR), carried proteins encoded                            result strains meiotic recombination null phenotypes <br /> RAD52 epistasis group, used repair DSBs induced ionizing                     [9,11]. The Xrs2 protein (known Nbs1 mammals) contains a <br /> radiation (IR). The time course IR repair vivo                         forkhead domain N terminus binding domain Tel1 <br /> assayed observing post-irradiation appearance foci                   C-terminus. Tel1 kinase involved telomere <br /> various HR repair proteins tagged green fluorescent protein                      maintenance checkpoint signaling. Mre11/Rad50/Nbs1 has <br /> [1]. Mre11, Rad50, Xrs2 form complex (MRX)                          weak ATP-stimulated DNA unwinding activity [12]. <br /> first arrivals DSB formation. The MRX complex regulates                            In bacterial recombinational repair, strand invasion <br /> the processing end DSB, resulting resection                   intermediate generated RecBCD helicase/nuclease a <br /> 59 end leaving 39 single-stranded overhang [2]. RPA binds                       complicated reaction requiring 59 39 helicase, 39 59 helicase, <br /> the single-stranded DNA, followed Rad52 Ddc2/Mec1                              59 39 nuclease, 39 59 nuclease, specific DNA sequence <br /> binding. Ddc2/Mec1 apical checkpoint-activating protein                        called chi. A counterpart complex identified in <br /> kinase. Rad52 mediate exchange RPA Rad51                           eukaryotic cells. The highly radioresistant organism known, <br /> strand exchange protein, generating 39 terminal filament                     <span id='am-12' about='obo:NCBITaxon_1269' typeof='owl:Thing obo:NCBITaxon_85006 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_1268'><span id='am-13' property="oboInOwl:hasDbXref" content="PMID:11931177" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Micrococcus" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="PMID:7547287" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Micrococcus Cohn 1872 emend. Stackebrandt et al. 1995" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Micrococcus Cohn 1872 (Approved Lists 1980) emend. Wieser et al. 2002" datatype="xsd:string"></span><span id='am-20'  ></span>Micrococcus</span> radiodurans, does RecBCD helicase/nuclease, <br /> initiates invasion intact sister homologous chromosome                    does possess complete RecF pathway [13]. A major question <br /> [3]. Rad51 association followed binding Rad54                        pathway eukaryotes. MRE11 a <br /> additional strand exchange proteins, Rad55 Rad57.                                 nuclease involved DSB repair mre11D strains extremely <br />    Mechanistically, initial steps poorly understood. Our                      sensitive irradiation. After 17 krads survival mre11D <br /> studies address function MRX complex, ATP                        strain 0.06%. If assumes 2 krads results 1 DSB per <br /> binding, ATPase, adenylate kinase, nuclease, DNA end-                             haploid cell [14], survival 0.02% be <br /> bridging activity [4,5]. Mre11 houses nuclease activity                      expected absence repair. Therefore, repair nearly <br /> domain containing phosphodiesterase motifs homologous                         absent mre11D strain. mre11-D65N mre11-H125N more <br /> the E. coli sbcD exonuclease. Mre11 39 59 exonuclease                          sensitive IR MRE11 strains resistant than <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                1                               January 2009 | Volume 4 | Issue 1 | e4267 <br />                                                                                                                     Mre11 Nuclease &amp; Dna2 &amp; Sgs1 <br />  <br />  <br /> mre11D strains. At high dose 70 krads 75 DSBs                Pif1, helicase orthologous E. coli RecD helicase, that <br /> induced, predict survival value e235 10213%             functions DNA replication telomere homeostasis [34,35]. <br /> in complete absence repair. The survival mre11-D56N           Using dna2D pif1D strain mre11 nuclease defective mutants, <br /> mre11-H125N mutants 70 krads 5 10 fold             nucleases Dna2 Mre11 required in <br /> MRE11 strains (3% survival), higher                  repair IR-induced damage, presumably 59 resection, that <br /> predicted survival absence DSB repair [15,16]. In                 compensate other&#8217;s loss. In addition, show <br /> addition, paradoxically, Mre11 39 59 exonuclease activity            helicase Dna2 dispensable Sgs1 helicase <br /> rather 59 39 nuclease expected 59 resection [6&#8211;8].          required Dna2 nuclease able compensate lack of <br /> Furthermore, resection reduced mre11 mutants defective            Mre11 nuclease. <br /> in nuclease active site proficient functions of <br /> Mre11 [17,18]. This led proposal nucleases            Results <br /> are required compensate absence the <br /> Mre11 nuclease. Sae2 nuclease appears                 Dna2 nuclease compensate loss Mre11 <br /> associated MRX complex, 39 59 activity             nuclease DSB repair vice versa <br /> [19]. Exo1, 59 39 nuclease Fen1 <span id='am-60' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> involved                 The relative importance repair proteins measured by <br /> mismatch repair, play role. Overproduction Exo1             determining survival mutants defective respective <br /> increases IR resistance mre11D strain [20,21]. However,           proteins increasing doses DNA damaging agent. We <br /> exo1D mre11D exoD mre11-nuclease deficient double mutants           used assays measure survival acute dose IR <br /> viable, sensitive IR mre11D mre11-nuclease           followed growth absence DNA damaging agent. <br /> deficient mutants, respectively, carry gene conversion       We surprised dna2D pif1D mutant resistant X- <br /> efficiently vivo EXO1 MRE11 strains [20]. This suggests          rays (Figure 1, sensitivity mre11D mre11D pif1D shown <br /> yet nuclease compensate Mre11 nuclease                     controls), dna2 hypomorphic mutants sensitive <br /> deficiency.                                                                 IR [31,36]. This suggested absence Pif1, another <br />    Dna2 candidate role. Dna2 59 39 helicase, 59       protein carry role Dna2 repair. Both <br /> to 39 exo/endonuclease, 39 59 exo/endonuclease, single           hypomorphic dna2 mutants dna2D pif1D strain are <br /> strand DNA annealing strand exchange activity [22&#8211;28].                  synthetically lethal mutants affecting nuclease mutants <br /> Dna2 plays role Okazaki fragment processing (OFP), assisting           defective DSB repair, mre11D, sae2D, exo1D [31,33,34]. We <br /> FEN1 RNA primer removal [26,29]. dna2 mutants, however,                  reasoned Mre11 nuclease compensating the <br /> are extremely sensitive bleomycin IR, consistent             loss Dna2 X-ray repair dna2D pif1D strain. <br /> additional role DSB repair [30&#8211;32]. Also, showed                    To test hypothesis Mre11 Dna2 compensatory <br /> years ago dna2-2 rad50-5 mutants synergistically defective         functions, wished compare X-ray sensitivity dna2D pif1D <br /> in IR repair [31]. Finally, Dna2 forms hub genetic network           dna2D pif1D mre11-nuclease defective strains. We created dna2D <br /> of 322 proteins preserve genome stability [33]. Since              pif1D mre11D strains carrying MRE11, mre11-D56N mre11-H125N <br /> DNA2 gene essential, however, difficult test          centromeric plasmids. The D56N H125N mutations lie the <br /> hypothesis Dna2 nuclease participates generation 39          conserved phosphodiesterase motif Mre11, mutant <br /> overhang DSB repair. Recently,                     proteins vitro endonuclease activity [18,37]; however, <br /> inviability dna2D mutants efficiently suppressed deletion          able form complex Rad50 Xrs2. As shown in <br />  <br />  <br />  <br />  <br /> Figure 1. X-ray resistance dna2D pif1D strain compared sensitivity mre11D pif1D. The strains MB120-5A (WT), MB121- pif1D, <br /> MB161B- dna2D pif1D, MB122-17C- mre11D, MB124-2D- mre11D pif1D harvested mid-log phase, resuspended water, irradiated, serially <br /> diluted (1:10), plated. <br /> doi:10.1371/journal.pone.0004267.g001 <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        2                             January 2009 | Volume 4 | Issue 1 | e4267 <br />                                                                                                                       Mre11 Nuclease &amp; Dna2 &amp; Sgs1 <br />  <br />  <br />  <br />  <br /> Figure 2. dna2D pif1D mre11D (pRS414mre11-D56N) dna2D pif1D mre11D (pRS414mre11-H125N) strains exhibit similar sensitivity <br /> to X-rays mre11D pif1D. These strains created dissecting tetrads following crosses: MATa dna2D pif1D trp1D bar1D/ MATa <br /> mre11D trp1D (pRS414MRE11), MATa dna2D pif1D trp1D bar1D/ MATa mre11D trp1D (pRS414mre11-D56N) MATa dna2D pif1D trp1D bar1D/ MATa <br /> mre11D trp1D (pRS414mre11-H125N) diploids. Trp+ spores replica plated, irradiated. Control strains mre11D pif1D (pRS414MRE11) and <br /> dna2D pif1D mre11D (pRS414MRE11) identified respective cross X-ray resistant. dna2D pif1D mre11D (pRS414mre11D56N) or <br /> dna2D pif1D mre11D (pRS414mre11H125N), respective crosses radiation sensitive. To compare radiation sensitivity the <br /> dna2D pif1D mre11D (pRS414mre11-D56N) dna2D pif1D mre11D (pRS414mre11-H125N) (bottom panel) mre11D pif1D (pRS414mre11-D56N) <br /> and mre11D pif1D (pRS414mre11-H125N) strains (top panel), serially diluted cells irradiated 26 krads plate allowed grow for <br /> three days. Note dna2D pif1D mre11D strain carrying pRS vector inviable, mre11D pif1D strain carrying pRS vector is <br /> shown control, indicated * panel. <br /> doi:10.1371/journal.pone.0004267.g002 <br />  <br />  <br /> Figure 2, wild-type nuclease-defective alleles MRE11              conclude nucleolytic activities Mre11 Dna2 that <br /> suppress lethality dna2D pif1D mre11D strains. As predicted,           complementary. <br /> however, strains deficient Dna2 Mre11 nuclease               mre11-D56N mre11-H125N mutants sensitive to <br /> are X-ray sensitive, strains expressing wild-type Mre11             DNA damaging agents allow direct scoring genetic crosses. <br /> resistant IR. Thus, absence Dna2, Mre11 nuclease              Therefore, compare DNA2 PIF1 mre11 nuclease-minus the <br /> required X-ray repair. By contrast, X-ray sensitivity pif1D        dna2D pif1D mre11 nuclease-minus mutants, triple mutants also <br /> mre11D, functional Dna2 nuclease,                      constructed introduction genetically tagged copy mre11 <br /> complemented wild-type MRE11 nuclease                 nuclease mutant genes chromosome, described the <br /> defective mre11-D56N mre11-H125N. These results suggest               legend Figure 4. dna2D pif1D MRE11 X-ray resistant (see <br /> Dna2 nuclease function X-ray repair absence Mre11            Figure 1). As before, dna2D pif1D mre11D::NatR::306 mre11- <br /> nuclease, complete absence Mre11. It says         D56N::URA3 dna2D pif1D mre11D::NatR::306 mre11-H125N::URA3 <br /> Mre11 nuclease function X-ray repair absence Dna2.           highly sensitive IR compared mre11D::NatR::306 <br />    To insure X-ray sensitivities nuclease-dead mre11          mre11D56N::URA3 mre11D::NatR::306 mre11H125N::URA3 <br /> mutants result inefficient expression employing        strains (Figure 4 A,B). Clearly, absence Dna2 nuclease, <br /> extrachromosomal plasmids express mre11 alleles, mre11             Mre11 nuclease essential repair IR-induced damage. <br /> nuclease defective genes integrated chromosome by <br /> two-step gene replacement. The dna2D pif1D mre11-D56N dna2D               Sgs1 Required Dna2-mediated Repair Pathway <br /> pif1D mre11-H125N clones X-ray sensitive mre11D pif1D             Based results, hypothesized Dna2 nuclease <br /> strain (compare Figure 3A Figure 3B Figure 1). Importantly,         play role resection 59 end IR-induced breaks. <br /> both dna2D pif1D mre11-D56N dna2D pif1D mre11-H125N X-ray                 Since Dna2 helicase required, asked helicase <br /> sensitive clones, X-ray resistance restored introduction           participate convert DSB substrate recognized <br /> plasmid expressing DNA2 MRE11 (Figure 3A,B). The                    Dna2, recognizes single-stranded termini. Sgs1, <br /> helicase activity Dna2 essential complementation,        encodes helicase RecQ family, likely <br /> the X-ray resistance dna2D pif1D mre11-H125N clone                 candidate, helicase defective dna2-2 mutants defective in <br /> restored plasmids expressing Dna2 mutations          X-ray repair double sgs1D dna2-2 mutant significantly <br /> the helicase domain: pRS314dna2K1080E plasmid,                    sensitive IR dna2-2, sgs1D itself X-ray <br /> mutation conserved Walker P loop functional                  resistant [31,38]. Furthermore, recently demonstrated that <br /> nuclease, pRS314dna2R1253Q (dna2-2) (Figure 3B). We                       human BLM, RecQ family helicase, suppress MMS and <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         3                            January 2009 | Volume 4 | Issue 1 | e4267 <br />                                                                                                                    Mre11 Nuclease &amp; Dna2 &amp; Sgs1 <br />  <br />  <br />  <br />  <br /> Figure 3. (A) X-ray sensitivity dna2D pif1D mre11-D56N strain complementation CEN plasmids containing Mre11 Dna2. <br /> DNA fragments containing mre11-D56N mre11-H125N mutant genes 800 bp 59 140 bp 39 gene cloned pRS306URA3, <br /> cleaved mre11 promoter transformed dna2D pif1D MRE11 strain allow integration MRE11 locus. The resulting transformants <br /> were propagated YPD media grown 5-<span id='am-7' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-11'  ></span>FOA</span> containing media excise MRE11 allele. Approximately 15% pRS306mre11-D56N <br /> and pRS306mre11-H125N Ura2 pop outs X-ray sensitive. Strains MB161B-56- dna2D pif1D mre11D56N, MB161B-56 (pRS414MRE11), MB161B <br /> (pRS314DNA2) compared X-ray sensitivity Figure 1. (B) X-ray sensitivity dna2D pif1D mre11-H125N strain and <br /> complementation plasmids containing MRE11 nuclease proficient DNA2 genes. Strain MB161B-125 &#8211; dna2D pif1D mre11H125N, <br /> MB161B-125 (pRS414MRE11), MB161B-125 (pR314DNA2), MB161B (pRS314dna2K1080E), MB161B-125 (pRS314dna2-2) compared X-ray <br /> sensitivity described Figure 1. <br /> doi:10.1371/journal.pone.0004267.g003 <br />  <br />  <br /> bleomycin sensitivity dna2-2 mutants [39,40]. We                  Sae2 nuclease functions MRX complex and <br /> investigated yeast Sgs1 involved Dna2               function nuclease [19]. Thus, wanted test <br /> repair pathway. We found, others, sgs1D mutants            sae2D sgs1D strains survival sgs1D mre11- <br /> as resistant IR wildtype (Figure 5). We constructed           H125N, dna2D mre11-H125N, mre11D strains. As illustrated in <br /> sgs1D mre11-H125N double mutant. As predicted, double                Figure 5, sae2D sgs1D strains sensitive IR sgs1D <br /> mutant sensitive IR dna2D mre11-H125N mutant               mre11-H125N strains. These results Mre11 nuclease Sae2 <br /> (compare Figures 3 4 Figure 5). We conclude Sgs1           nuclease pathway Dna2 nuclease Sgs1 helicase in <br /> required <span id='am-81' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-82' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> Dna2 compensate loss Mre11              pathway initiation exonucleolytic degradation <br /> nuclease repair IR-induced damage.                                DSB formation. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                       4                            January 2009 | Volume 4 | Issue 1 | e4267 <br />                                                                                                                  Mre11 Nuclease &amp; Dna2 &amp; Sgs1 <br />  <br />  <br />  <br />  <br /> Figure 4. (A) Comparison X-ray sensitivity mre11-D56N pif1D strain dna2D mre11-D56N pif1D strain. The integrating <br /> plasmids pRS306mre11-D56N::URA3 pRS306mre11-H125N::URA3 cut Sph1 cleaves 300 bp 59 ATG MRE11 and <br /> transformed MATa mre11D::natR strain insertion MRE11 locus selected G418 plates lacking uracil. These strains were <br /> mated MB161B-MATa dna2D pif1D MRE11, diploids sporulated dissected. When resulting spores scored, NatR <br /> gene segregated URA3 gene, demonstrating pRS306mre11nuclease minus plasmids integrated MRE11 loci the <br /> respective strains. Strains MB126-mre11D::natR::mre11D56N::URA3, MB128- mre11D::natR::mre11-D56N::URA3 pif1D, MB133 - dna2D pif1D <br /> mre11D::natR::mre11-D56N::URA3 treated Figure 1 determine IR sensitivity. (B) Comparison X-ray sensitivity mre11- <br /> H125N pif1D strain dna2D mre11-H125N pif1D strain. Strains constructed Figure 4A. Strains MB127-mre11D::natR::mre11- <br /> H125N::URA3, MB129-mre11D::natR::mre11-H125N::URA3 pif1D, MB134 - dna2D mre11D::natR::mre11-H125N::URA3 pif1D, treated in <br /> Figure 1. <br /> doi:10.1371/journal.pone.0004267.g004 <br />  <br />  <br /> Dna2 Helicase Activity produces Requirement                       undetectable nuclease activity intact helicase ATPase <br /> Dna2 Nuclease Activity                                                    activity [25,41]. The plasmid, pGAL::DNA2, high copy vector <br />    Separation function MRX ATP binding nuclease                 containing GAL10 inducible Dna2 expresses sufficient Dna2 <br /> functions investigated extensively attempted             induction complement dna2-1 mutants 37uC on <br /> similar analysis Dna2. The nuclease Dna2 maps region           glucose plates [25]. The plasmids pGAL::dna2-K1080E, <br /> N-terminal helicase, amino acids 649&#8211;744              pGAL::dna2-D657A pGAL::dna2-E675A complement a <br /> domain containing conserved E. coli RecB nuclease motifs. Two             dna2-1 mutant glucose plates 37uC [25]. Also, found <br /> mutants Dna2, D657A E675A, specifically affecting RecB             here, dna2-K1080E, dna2-D657A, dna2-E675A genes <br /> active site motifs, result Dna2 proteins greatly reduced       expressed centromeric plasmids fail complement dna2D <br />  <br />  <br />       PLoS ONE | www.plosone.org                                      5                            January 2009 | Volume 4 | Issue 1 | e4267 <br />                                                                                                                 Mre11 Nuclease &amp; Dna2 &amp; Sgs1 <br />  <br />  <br />  <br />  <br /> Figure 5. Comparison X-ray sensitivity strain sgs1D sgs1D mre11-H125N sgs1D sae2D. Strains MB137 sgs1D, MB138 <br /> sgs1D mre11-H125N MB135 sae2D MB136 sgs1D sae2D constructed study treated Figure 1. <br /> doi:10.1371/journal.pone.0004267.g005 <br />  <br /> PIF1 mutant (data shown). This indicates ATP                 fact dna2D pif1D (RS314dna2-K1080E) strain grows at <br /> binding nuclease functions Dna2 essential.                     37uC, unlike dna2D pif1D strain [34]. Interestingly, the <br />    To verify active Dna2 nuclease necessary IR            pRS314dna2E675A transformant (helicase plus, nuclease minus) <br /> repair absence Mre11 nuclease, compared growth           did form colonies 30uC (Figure 6). However, the <br /> and X-ray sensitivity dna2D pif1D strains transformed             pRS314dna2K1080E,E675A (helicase minus, nuclease minus) did <br /> centromeric plasmids carrying DNA2, dna2K1080E,                       support viability 30uC. We conclude temperature the <br /> dna2E675A, dna2K1080E,E675A. The dna2K1080E,E675A                   absence nuclease deleterious normal DNA <br /> doubly deficient helicase nuclease. When dna2D            replication, furthermore Dna2 helicase activity <br /> pif1D transformants incubated 23uC tryptophan                  generates need Dna2 nuclease. <br /> deficient plates transformants grew, illustrated              Next, dna2D pif1D transformants irradiated after <br /> Figure 6. At 30uC, transformants containing plasmids              plating incubated 23uC. The pRS314, pRS314DNA2, <br /> pRS314, pRS314DNA2, pRS314dna2K1080E (helicase minus,                     pRS314dna2K1080E, pRS314dna2K1080E,E675A transfor- <br /> nuclease plus) formed viable colonies. Thus, pif1D suppresses         mants resistant IR, transformant containing <br /> inviability dna2D PIF1 (pRS314dna2-K1080E) strain,            pRS314dna2E675A plasmid sensitive IR. Thus, with <br />  <br />  <br />  <br />  <br /> Figure 6. Expression nuclease deficient Dna2 toxic dna2D pif1D strain. Strain MB21-dna2D pif1D carrying pRS314DNA2, Hel+, <br /> Nuc+; pRS314, Vector; pRS314dna2K1080E, Hel2Nuc+; pRS314dna2E675A, Hel+Nuc2 pRS314dna2K1080E,dna2E675A, Hel2Nuc2 plasmids was <br /> grown, serially diluted 10 fold plated trptophan deficient plates 26 krads IR, allowed grow days the <br /> temperatures indicated. <br /> doi:10.1371/journal.pone.0004267.g006 <br />  <br />  <br />       PLoS ONE | www.plosone.org                                      6                           January 2009 | Volume 4 | Issue 1 | e4267 <br />                                                                                                                   Mre11 Nuclease &amp; Dna2 &amp; Sgs1 <br />  <br />  <br />  <br />  <br /> Figure 7. Inactivation Dna2 nuclease lethal pif1D mre11-H125N strain. A dna2D pif1D mre11-H125N trp1D (pGAL::DNA2URA3) <br /> strain transformed Trp+ plasmids pRS314DNA2, Hel+, Nuc+; pRS314, Vector; pRS314dna2K1080E, Hel2Nuc+; pRS314dna2E675A, Hel+Nuc2 <br /> or pRS314dna2K1080E,dna2E675A, Hel2Nuc2. The Trp+Ura+ transformants grown tryptophan deficient media spotted tryptophan <br /> minus (left) tryptophan minus, 5-FOA containing (right) media 23uC photographed 4 days 23uC. <br /> doi:10.1371/journal.pone.0004267.g007 <br />  <br /> growth restrictive temperature 30uC, IR, Dna2         result dna2D pif1D strain resistant X-rays the <br /> nuclease minus strain sensitive damage,             dna2D pif1D mre11-H125N mutant sensitive X-rays an <br /> presence active Dna2 helicase. We conclude                 mre11D strain, showing absence Mre11 <br /> activities Dna2 helicase nuclease properly             nuclease Dna2 nuclease, cells nearly blocked repair. <br /> coordinated DNA replication repair.                          Thus, strikingly, nuclease support repair efficiently <br />    The requirement coordination Dna2 helicase              (Figure 3). The direct mechanistic inference Dna2 and <br /> nuclease examined dna2D pif1D mre11-H125N              Mre11 performing partially overlapping nucleolytic <br /> mutant. The dna2D pif1D mre11-H125N (pGAL::DNA2)                      functions processing ends DSBs. Since Mre11 39 59 <br /> transformed set Dna2 mutant plasmids used             nuclease 59 39 activity vitro, question arises as <br /> Figure 6, yield double transformants, dna2D pif1D mre11-H125N          perform 59 39 nucleolytic degradation the <br /> (pGAL::DNA2) (pRS414-dna2 mutant). Each double transformant               absence Dna2. Mre11 achieve absence Dna2 <br /> was grown tryptophan deficient liquid media select          cleaving hairpin structures ends unwound an <br /> pRS314dna2 mutant allow loss URA3 containing            unknown helicase [8]. Another interpretation Mre11 <br /> pGAL::DNA2 plasmid. The resulting cultures spotted                nuclease Dna2 function sequentially second <br /> tryptophan-deficient tryptophan-deficient media containing 5-          nuclease, exonuclease 1, substitute Dna2 nuclease, when <br /> FOA grown 23uC determine ability dna2                Mre11 present. In absence Mre11 nuclease, however, <br /> mutants support growth. dna2D pif1D mre11-H125N containing             Dna2 Exo1, overcome requirement Mre11 <br /> the plasmids pRS314, pRS314DNA2, pRS314dna2K1080E grew,                   nucleolytic step. We favor interpretation (see below). <br /> but strains containing plasmids pRS314dna2E675A (nuclease                While studies Dna2 nuclease compensate <br /> minus) did grow 5-FOA-containing plates (Figure 7). In        entirely, terms repair survival, absence Mre11 <br /> Mre11-nuclease deficient background, reduced                nuclease, Dna2 compensate absence Mre11. Mre11 <br /> viability transformant containing pRS314dna2-                  clearly required form MRX end processing complex. <br /> K1080E,E675A (nuclease minus, helicase minus). We conclude                   The interaction Mre11 Dna2 DSB damage is <br /> that viability Dna2-nuclease defective dna2D pif1D <br />                                                                           paralleled fact Dna2, like Mre11, telomeres, <br /> (pRS314dna2E675A) 23uC shown (Figure 6) depends a <br />                                                                           linear ends chromosomes resemble DSBs [32]. The <br /> functional Mre11 nuclease, suggesting complementarity not <br />                                                                           Mre11 nuclease required novo synthesis of <br /> only IR repair DNA replication repair of <br />                                                                           telomeres DSB telomere seed sequence, an <br /> replication errors. This consistent model Dna2 <br />                                                                           mre11 nuclease deficient mutant does short telomeres <br /> helicase makes potentially toxic structure DNA <br />                                                                           [42]. Dna2 appears required extending short <br /> replication preferentially coordinately processed by <br />                                                                           telomere, nucleolytic function play [32,43,44]. <br /> the Dna2 nuclease, processed Mre11 <br />                                                                           Interestingly, dna2D pif1D mutants short telomeres compared <br /> nuclease. Inactivation Dna2 helicase sufficient remove the <br /> requirement nucleolytic processing absence Dna2          pif1D mutants [34]. <br /> but insufficient absence Dna2 Mre11                    Unexpectedly, study supplied genetic evidence that <br /> nuclease. In dna2D pif1D mre11-H12N mutant, residual              supports biochemical studies suggesting Dna2 helicase and <br /> MRX helicase contribute generating structure                 nuclease activities coupled. dna2D pif1D expressing Dna2- <br /> needing nuclease processing.                                              E675A (helicase plus, nuclease minus) protein reduced <br />                                                                           maximum permissive growth temperature sensitive to <br />                                                                           IR. However, dna2D pif1D expressing helicase minus, nuclease <br /> Discussion <br />                                                                           minus double Dna2 mutant protein growth repair <br />   We shown Dna2 59 39 nuclease function           defect, i.e. inactivation helicase removes requirement for <br /> IR repair absence active Mre11 nuclease. The key             nuclease (Figure 6). This suppression occurs Mre11 <br />  <br />  <br />       PLoS ONE | www.plosone.org                                      7                            January 2009 | Volume 4 | Issue 1 | e4267 <br />                                                                                                                        Mre11 Nuclease &amp; Dna2 &amp; Sgs1 <br />  <br />  <br /> nuclease functional (Figure 7). Therefore, propose              does fully compensate defective Dna2 helicase. We have <br /> Dna2 helicase creates structure DNA replication                   previously shown BLM, human counterpart Sgs1 <br /> DNA repair potentially toxic unless coordinately processed             helicase, complements MMS sensitivity dna2-2 mutants as <br /> by nuclease Dna2 or, absence, Mre11. Further,             lethality dna2-1 mutants [39], suppression <br /> the observed lethality temperatures absence               attributed suppression defects DSB processing. <br /> DNA damage mre11-nuclease minus dna2D pif1D expressing                      We expect overexpression BLM suppress the <br /> nuclease defective Dna2 leads hypothesis Mre11                     IR-sensitivity dna2-2 mutants. Genetic evidence suggests Pif1 <br /> nuclease play role Okazaki fragment processing                  helicase inhibitor Sgs1 helicase [49]. Another possible <br /> repair lethal endogenous DNA damage. A role Mre11                    role Dna2 counteract Pif1 inhibition Sgs1 during <br /> OFP consistent previous results showing mre11-H125N               59 39 processing step. <br /> rad27 mutants inviable [18].                                                   What results tell role Dna2 in <br />    Also worth emphasizing fact findings suggest             endogenous damage repair? We previously identified the <br /> additional role Dna2, 59 resection, X-ray repair. dna2D         DNA replication fork pause replication fork barrier (RFB) in <br /> pif1D strains resistant IR wildtype (this work),          ribosomal DNA specific site endogenous DNA damage <br /> dna2D PIF1 overexpressing FEN1 maintain viability X-ray               requiring combined activities Dna2 Sgs1 repair. In <br /> sensitive rad52D [31]. Therefore, IR repair, Pif1                presence Fob1, DNA replication forks pause RFB and <br /> contribute formation structure lethal unless processed         DSB arises, occurrence elevated dna2-2 and <br /> by Dna2, function complemented MRE11, EXO1,                 sgs1D mutants [40,50]. Interestingly, Pif1 required for <br /> or SGS1. Furthermore, dna2-2 PIF1 mutant (nuclease plus, helicase            normal levels pausing RFB subsequent chromosomal <br /> defective) 100 times sensitive IR DNA2 PIF1 (at                breakage [51]. These endogenous DSBs inability repair <br /> 75 krads) [31]. However, dna2-2 gene complements X-ray                 contribute synthetic sickness/lethality dna2-2 <br /> sensitivity dna2D pif1D mre11-H125N strain (Figure 3B). These           sgs1D strain, sickness/lethality suppressed fob1D <br /> latter results appear odds explained Dna2-2             mutation [40,50]. A similar situation occurs Ctf4, a <br /> protein perform resection function necessary            replication/cohesion protein, dna2-2 ctf4D mutant is <br /> for X-ray resistance presence PIF1, resection          lethal, dna2-2 ctf4D fob1D viable slow growing. The <br /> or strand exchange.                                                     dna2-2 ctf4D fob1D mutant significantly sensitive IR than <br />    Mechanistic studies published work                  dna2-2 fob1D ctf4 fob1D mutants [33]. Presumably dna2-2 <br /> completed dna2 mutants sgs1 mutants exhibit               ctf4D inviability results failure repair endogenous <br /> significantly delayed long-range resection Rad51-independent              DSBs RFB. Thus, Dna2 plays role repairing <br /> single-strand annealing (SSA) repair reaction enzymatically-             endogenous breaks collaboration Sgs1 Ctf4 this <br /> induced DSB [45,46]. Based studies enzymatic                  likely reflection role Dna2 Sgs1 resection. It is <br /> activities Dna2 Sgs1, propose results imply              probably role Dna2 accounts remarkable <br /> interaction Mre11 nuclease Dna2/Sgs1                        observation increasing gene dosage Dna2 copy <br /> resection Rad51-dependent homologous recombination. At                  leads partial genome stabilization absolutely required for <br /> same time, however, new results raise conundrum. The                viability mec1-21 strain [52]. <br /> reduced exonuclease resection dna2D pif1m2 mutant observed                  In conclusion, results Zhu et al. [46] <br /> in SSA studies does appear cause defect subsequent             clarify multiple events occurring early repair DSBs, <br /> Rad51-dependent repair [46], dna2D pif1D                especially previously unappreciated centrality Dna2, they <br /> strain resistant IR wild-type strain 70 krads IR (35       highlight fundamental questions future: First, <br /> DSB 2 17 chromosomes). Similarly, sgs1D strains,                 resection sufficient repair rate- <br /> they reduced SSA, resistant IR wild-type strains,            limiting step repair? Second, mechanism resection <br /> even 75 krads. Thus, mechanisms work long-             se? How Mre11 Dna2, form link between <br /> range resection SSA entirely interchangeable            DNA replication repair, coordinate repair endogenous <br /> resection pathway leading strand exchange. In regard             DSBs occurring replication fork failure? <br /> interesting Exo1, 59 39 nuclease, appears able to <br /> compensate Dna2 Sgs1 deficiency SSA [45,46], does                Materials Methods <br /> not appear able IR repair, dna2 mre11-H125N <br /> and sgs1 mre11-H125N mutants sensitive IR mre11D,                 Strains Plasmids <br /> even Exo1 present normal levels mre11                      The       plasmids      pRS314DNA2,         pRS314dna2K1080E, <br /> nuclease-defective exo1 double mutants significantly              pRS314dna2E675A, pRS314dna2K1080E,E675A 6 kb <br /> sensitive IR mre11 nuclease deficient mutants. Previous studies        EcoRI fragment containing Dna2 Dna2 indicated <br /> showed 100&#8211;300 bp homology required Rad51-                mutations cloned pRS314CEN TRP EcoR1 site. The <br /> dependent gene conversion [47,48], differs SSA              construction Dna2 containing fragment construc- <br /> gene conversion requires strand invasion homologous sister                tion mutants described previously [25]. The plasmid <br /> chromatid homolog. We propose Dna2/Sgs1 pathway                 pSEY18GALDNA2, referred pGAL::DNA2, was <br /> the major pathway resection homologous recombination.                    described previously [25], work DNA2 gene <br />    The observed IR sensitivity sgs1 mre11-H125N mutant               carried additional FLAG tag, MDYKDDDK, N terminus <br /> report here, combined defect resection sgs1D                    protein. <br /> demonstrated others, help explain                         The plasmids pRS414MRE11, pRS414mre11-D56N, and <br /> previous current results. We shown helicase                  pRSmre11-H125N (gift K. Lewis, Texas State University, San <br /> defective dna2-2 mutants defective X-ray repair,           Marcos, TX) contain MRE11 genes 800 bp 59 the <br /> double sgs1D dna2-2 mutant significantly sensitive IR               ATG 136 bp 39 TAG contained Kpn1 Not1- <br /> than dna2-2, sgs1D itself X-ray resistant [31]. Thus,           fragment. The KpnI NotI fragments containing mre11-D56N and <br /> Dna2 compensate absence Sgs1 helicase, Sgs1                 mre11-H125N cloned KpnI NotI site integrating <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          8                            January 2009 | Volume 4 | Issue 1 | e4267 <br />                                                                                                                               Mre11 Nuclease &amp; Dna2 &amp; Sgs1 <br />  <br />  <br />  <br />  Table 1. Strains. <br />  <br />  <br />  BY4741                       MATa his3D1 leu2D0 met15D0 ura3D0 <br />  BY4742                       MATa his3D1 leu2D0 met15D0 ura3D0 <br />  MB120-5A                     4742 MATa bar1D::kanMX trp1D <br />  MB121                        4741 MATa pif1D::HIS3 bar1D::kanMX <br />  MB122-17C                    4741 MATa mre11D::natR bar1D::kanMX trp1D <br />  MB123-1D                     4742 MATa mre11D::natR trp1D <br />  MB124-2D                     4741MATa mre11D::natR pif1D::HIS3 bar1D::kanMX trp1D <br />  MB126                        4742 MATa mre11D::natR mre11-D56N::URA3 trp1D <br />  MB127                        4741 MATa mre11D::natR mre11-H125N::URA3 trp1D <br />  MB128                        4741MATa mre11D::natR mre11-D56N::URA3 pif1D::HIS3 bar1D::kanMX <br />  MB129                        4741 MATa mre11D::natR mre11-H125N::URA3 pif1D::HIS3 bar1D::kanMX <br />  MB203                        4741 MATa dna2D::kanMX pif1D::HIS3 trp1D <br />  MB213                        4741 MATa dna2D::natR pif1D::HIS3 trp1D <br />  MB161B                       4741 MATa dna2D::kanMX pif1D::HIS3 trp1D bar1D::kanMX <br />  MB161B-56                    4741 MATa dna2D::kanMX pif1D::HIS3 trp1D bar1D::kanMX mre11-D56N <br />  MB161B-56                    (pRS414MRE11) <br />  MB161B-56                    (pRS314DNA2) <br />  MB161B-125                   4741 MATa dna2D::kanMX pif1D::HIS3 trp1D bar1D::kanMX mre11-H125N <br />  MB161B-125                   (pRS414MRE11) <br />  MB161B-125                   (pRS314DNA2) <br />  MB161B-125                   (pRS314dna2-K1080E) <br />  MB161B-125                   (pRS314dna2-E675A) <br />  MB161B-125                   (pRS314dna2-K1080E,K673A) <br />  MB161B-125                   (pRS314dna2-2) <br />  MB130                        4741 MATa dna2D::kanMX pif1D::HIS3 trp1D bar1D::kanMX mre11D::natR (pRS414MRE11) <br />  MB131                        4741 MATa dna2D::kanMX pif1D::HIS3 trp1D bar1D::kanMX mre11D::natR (pRS414mre11-D56N) <br />  MB132                        4741 MATa dna2D::kanMX pif1D::HIS3 trp1D bar1D::kanMX mre11D::natR (pRS414mre11-H125N) <br />  MB133                        4741 MATa dna2D::kanMX pif1D::HIS3 trp1D bar1D::kanMX mre11D::natR mre11-D56N::URA3 <br />  MB134                        4741 MATa dna2D::kanMX pif1D::HIS3 trp1D bar1D::kanMX mre11D::natR mre11-H125N::URA3 <br />  MB135                        4742 MATa sgs1D::kanMX sae2D::natR <br />  MB136                        4742 MATa sae2D::natR <br />  MB137                        4742 MATa sgs1D::kanMX <br />  MB138                        4741 MATa sgs1D::kanMX mre11D::natR mre11-H125N::URA3 <br />  <br />  doi:10.1371/journal.pone.0004267.t001 <br />  <br />  <br /> plasmid, pRS306. The pRS306mre11-D56N pRS306mre11-                               ion chamber (model 10X5-0.6, serial no. 9352) connected a <br /> H125N plasmids cut SphI transformed yeast.                        Radiation Monitor Controller (model 9010; serial no. 90-2910). <br />    All MRE11 DNA2 alleles used study expressed <br /> under control native promoters.                                         Acknowledgments <br />    Strains used described Table 1. <br />                                                                                      We thank Kevin Lewis cloned wild-type mutant MRE11 genes. <br />  <br /> X-ray Treatment                                                                      Author Contributions <br />   Cells treated X-rays previously described using a <br /> Pantak MKII X-ray machine operated 20 mA 70 kev [31].                         Conceived designed experiments: JLC MEB. Performed the <br />                                                                                      experiments: MEB. Analyzed data: JLC MEB. Wrote paper: JLC <br /> The dosimetry determined using Radcal; 0.6 cubic centimeter                    MEB. <br />  <br /> References <br /> 1. Lisby M, Barlow JH, Burgess RC, Rothstein R (2004) Choreography             3. Sung P (1997) Function yeast Rad52 protein     mediator between <br />    DNA damage response: Spatiotemporal relationships checkpoint                replication protein A Rad51 recombinase. J     Biol Chem 272: <br />    repair proteins. Cell 118: 699&#8211;713.                                                   28194&#8211;28197. <br /> 2. Ivanov EJ, Sugawara N, White CI, Fabre F, Haber JE (1994) Mutants XRS2          4. D&#8217;Amours D, Jackson SP (2002) The Mre11 complex:       crossroads <br />    RAD50 delay, prevent, mating-type switching <span id='am-53' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-54' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-59'  ></span>Saccharomyces</span>           dna repair checkpoint signalling. Nat Rev Mol   Cell Biol 3: 317&#8211; <br />    cerevisiae. Mol Cell Biol 14: 3414&#8211;3425.                                              327. <br />  <br />  <br />  <br />  <br />       PLoS ONE | www.plosone.org                                                 9                             January 2009 | Volume 4 | Issue 1 | e4267 <br />                                                                                                                                                Mre11 Nuclease &amp; Dna2 &amp; Sgs1 <br />  <br />  <br />  5. D&#8217;Amours D, Jackson SP (2001) The yeast Xrs2 complex functions S phase                 28. Budd ME, Campbell JL (1995) A new yeast gene required DNA replication <br />     checkpoint regulation. Genes Dev 15: 2238&#8211;2249.                                               encodes protein homology DNA helicases. Proc Natl Acad Sci USA 92: <br />  6. Paull TT, Gellert M (1998) The 39 59 exonuclease activity Mre11 facilitates             7642&#8211;7646. <br />     repair DNA double-strand breaks. Mol Cell 1: 969&#8211;979.                                  29. Bae SH, Bae K-H, Kim JA, Seo YS (2001) RPA governs endonuclease switching <br />  7. Trujillo KM, Yuan SS, Lee EY, Sung P (1998) Nuclease activities complex                  processing Okazaki fragments eukaryotes. Nature 412: 456&#8211;461. <br />     human recombination DNA repair factors Rad50, Mre11, p95. J Biol               30. <span id='am-92' about='obo:NCBITaxon_225842' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_117743 obo:NCBITaxon_200644 obo:NCBITaxon_49546 obo:NCBITaxon_976 obo:NCBITaxon_68336'><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="Pseudocytophaga" datatype="xsd:string"></span><span id='am-94'  ></span><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasDbXref" content="PMID:15143012" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="PMID:16403882" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="Formosa" datatype="xsd:string"></span>Formosa</span> T, Nitiss T (1999) Dna2 mutants reveal interactions DNA <br />     Chem 273: 21447&#8211;21450.                                                                        polymerase alpha Ctf4, Pol accessory factor, DNA2 <br />  8. Trujillo KM, Sung P (2001) DNA structure-specific nuclease activities                  helicase activity essential growth. Genetics 151: 1459&#8211;1470. <br />     <span id='am-36' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-37' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-41'  ></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> Rad50*Mre11 complex. J Biol Chem 276:                            31. Budd ME, Campbell JL (2000) The pattern sensitivity yeast dna2 mutants to <br />     35458&#8211;35464.                                                                                  DNA damaging agents suggests role DSB postreplication repair <br />  9. Hopfner KP, Karcher A, Shin DS, Craig L, Arthur LM, et al. (2000) Structural                  pathways. Mutat Res 459: 173&#8211;186. <br />     biology Rad50 ATPase: ATP-driven conformational control DNA double-                 32. Choe W, Budd M, Imamura O, Hoopes L, Campbell JL (2002) Dynamic <br />     strand break repair ABC-ATPase <span id='am-84' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-85' property="rdfs:label" content="superfamily" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>superfamily</span>. Cell 101: 789&#8211;800.                        localization Okazaki fragment processing protein suggests novel role in <br /> 10. Moncalian G, Lengsfeld B, Bhaskara V, Hopfner KP, Karcher A, et al. (2004)                    telomere replication. Mol Cell Biol 22: 2002&#8211;2017. <br />     The rad50 signature motif: essential ATP binding biological function.              33. Budd ME, Tong AH, Polaczek P, Peng X, Boone C, et al. (2005) A network of <br />                                                                                                   multi-Tasking proteins DNA replication fork preserves genome stability. <br />     J Mol Biol 335: 937&#8211;951. <br />                                                                                                   PLoS Genet 1: 634&#8211;650. <br /> 11. Alani E, Subbiah S, Kleckner N (1989) The yeast RAD50 gene encodes a <br />                                                                                               34. Budd ME, Reis CC, Smith S, Myung K, Campbell JL (2006) Evidence <br />     predicted 153-kD protein containing purine nucleotide-binding domain and <br />                                                                                                   suggesting Pif1 helicase functions DNA replication Dna2 <br />     large heptad-repeat regions. Genetics 122: 47&#8211;57. <br />                                                                                                   helicase/nuclease DNA polymerase <span id='am-100' about='obo:NCBITaxon_998453' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_50638 obo:NCBITaxon_7438 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7434 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_34725 obo:NCBITaxon_50557'><span id='am-101'  ></span><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-104' property="rdfs:label" content="Delta" datatype="xsd:string"></span>delta</span>. Mol Cell Biol 26: 2490&#8211;2500. <br /> 12. Paull TT, Gellert M (1999) Nbs1 potentiates ATP-driven DNA unwinding                  35. Bessler JB, Torre JZ, Zakian VA (2001) The Pif1p <span id='am-63' about='obo:NCBITaxon_subfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="subfamily" datatype="xsd:string"></span>subfamily</span> helicases: region- <br />     endonuclease cleavage Mre11/Rad50 complex. Genes Dev 13:                               specific DNA helicases? Trends Cell Biology 11: 60. <br />     1276&#8211;1288.                                                                                36. Fiorentino DF, Crabtree GR (1997) Characterization Saccharomyces cerevisiae <br /> 13. White O, Eisen JA, Heidelberg JF, Hickey EK, Peterson JD, et al. (1999)                       dna2 mutants suggests role helicase late S phase. Mol Biol Cell 8: <br />     Genome sequence radioresistant bacterium <span id='am-21' about='obo:NCBITaxon_1298' typeof='owl:Thing obo:NCBITaxon_188787 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_118964 obo:NCBITaxon_183710 obo:NCBITaxon_1297'><span id='am-22'  ></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Deinobacter" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Deinococcus" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="PMID:9103641" datatype="xsd:string"></span>Deinococcus</span> radiodurans R1.                   2519&#8211;2537. <br />     Science 286: 1571&#8211;1577.                                                                   37. Hopfner KP, Karcher A, Craig L, Woo TT, Carney JP, et al. (2001) Structural <br /> 14. Resnick MA, Martin P (1976) The repair double-strand breaks nuclear                 biochemistry interaction architecture DNA double-strand break <br />     DNA Saccharomyces cerevisiae genetic control. Mol Gen Genet 143:                   repair Mre11 nuclease Rad50-ATPase. Cell 105: 473&#8211;485. <br />     119&#8211;129.                                                                                  38. Gangloff S, Massy B, Arthur L, Rothstein R, Fabre F (1999) The essential <br /> 15. Moreau S, Morgan EA, Symington LS (2001) Overlapping functions                         role yeast topoisomerase III meiosis depends recombination. Embo J 18: <br />     Saccharomyces cerevisiae Mre11, Exo1 Rad27 nucleases DNA                               1701&#8211;1711. <br />     metabolism. Genetics 159: 1423&#8211;1433.                                                      39. Imamura O, Campbell JL (2003) The human Bloom syndrome gene suppresses <br /> 16. Lewis LK, Storici F, Van Komen S, Calero S, Sung P, et al. (2004) Role                 DNA replication repair defects yeast dna2 mutants. Proc Natl Acad <br />     nuclease activity Saccharomyces cerevisiae Mre11 repair DNA double-strand            Sci USA 100: 8193&#8211;8198. <br />     breaks mitotic cells. Genetics 166: 1701&#8211;1713.                                         40. Weitao T, Budd M, Campbell JL (2003) Evidence yeast SGS1, DNA2, SRS2, <br /> 17. Llorente B, Symington LS (2004) The Mre11 nuclease required 59 39               FOB1 interact maintain rDNA stability. Mutation Res 532: 157&#8211;172. <br />     resection multiple HO-induced double-strand breaks. Mol Cell Biol 24:                  41. Lee K-H, Kim DW, Bae S-H, Kim J-A, Ryu G-H, et al. (2000) The <br />     9682&#8211;9694.                                                                                    endonuclease activity yeast Dna2 enzyme essential vivo. Nucl Acids <br /> 18. Moreau S, Ferguson JR, Symington LS (1999) The nuclease activity Mre11                  Res 28: 2873&#8211;2881. <br />     required meiosis mating type switching, end joining, telomere          42. Frank CJ, Hyde M, Greider CW (2006) Regulation telomere elongation by <br />     maintenance. Mol Cell Biol 19: 556&#8211;566.                                                       cyclin-dependent kinase CDK1. Mol Cell 24: 423&#8211;432. <br /> 19. Lengsfeld BM, Rattray AJ, Bhaskara V, Ghirlando R, Paull TT (2007) Sae2                43. Tomita K, Kibe T, Kang H-Y, Seo Y-S, Uritani M, et al. (2004) Fission yeast <br />     endonuclease processes hairpin DNA cooperatively Mre11/                      Dna2 required generation telomeric single-strand overhang. Mol <br />     Rad50/Xrs2 complex. Mol Cell 28: 638&#8211;651.                                                     Cell Biol 24: 9557&#8211;9567. <br /> 20. Moreau S, Morgan EA, Symington LS (2001) Overlapping functions                     44. Parenteau J, Wellinger RJ (1999) Accumulation single-stranded DNA and <br />     Saccharomyces cerevisiae Mre11, Exo1 Rad27 nucleases DNA metabolism.                   destabilization telomeric repeats yeast mutant strains carrying deletion of <br />     Genetics 159: 1423&#8211;1433.                                                                      RAD27. Mol Cell Biol 19: 4143&#8211;4152. <br /> 21. Lewis LK, Karthikeyan G, Westmoreland JW, Resnick MA (2002) Differential                  45. Mimitou EP, Symington LS (2008) Sae2, Exo1 Sgs1 collaborate DNA <br />     suppression DNA repair deficiencies Yeast rad50, mre11 xrs2 mutants              double-strand break processing. Nature 455: 770&#8211;774. <br />                                                                                               46. Zhu Z, Chung W-H, Shi EY, Lee SE, <span id='am-71' about='obo:NCBITaxon_717364' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_82596 obo:NCBITaxon_82593 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_82592 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-72'  ></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="Ira" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Ira</span> G (2008) Sgs1 helicase two <br />     EXO1 TLC1 (the RNA component telomerase). Genetics 160: 49&#8211;62. <br />                                                                                                   nucleases Dna2 Exo1 resect DNA double-strand break ends. Cell 134: <br /> 22. Bae S-H, Seo Y-S (2000) Characterization enzymatic properties the <br />                                                                                                   981&#8211;994. <br />     yeast Dna2 helicase/endonuclease suggests new model Okazaki fragment <br />                                                                                               47. Jinks-Robertson S, Michelitch M, Ramcharan S (1993) Substrate length <br />     processing. J Biol Chem 275: 38022&#8211;38031. <br />                                                                                                   requirements efficient mitotic recombination Saccharomyces cerevisiae. <br /> 23. Bae S-H, Choi E, Lee K, Park J, Lee S, et al. (1 <br /> </body></html>