<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Transcriptional profiling endogenous germ layer <br /> precursor cells identifies dusp4 essential <br /> gene zebrafish endoderm specification <br /> Jamie L. Brown*, Mirit Snir*, Houtan Noushmehr*&#8224;, Martha Kirby&#8225;, Sung-Kook Hong&#167;, Abdel G. Elkahloun&#182;, <br /> and Benjamin Feldman*&#20648; <br /> *Medical Genetics Branch, &#8225;Genetics Molecular Biology Branch, &#182;Genome Technology Branch, National Human Genome Research Institute; <br /> and &#167;Program Genomics Differentiation, <span id='am-32' about='obo:NCBITaxon_73379' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_104737 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_105389 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_65495 obo:NCBITaxon_6341 obo:NCBITaxon_33317 obo:NCBITaxon_105390 obo:NCBITaxon_2759 obo:NCBITaxon_6340 obo:NCBITaxon_6072'><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Eunice" datatype="xsd:string"></span><span id='am-36'  ></span>Eunice</span> Kennedy Shriver National Institute Child Health Human Development, National <br /> Institutes Health, Bethesda, MD 20892 <br />  <br /> Communicated Francis S. Collins, National Institutes Health, Bethesda, MD, June 10, 2008 (received review February 21, 2008) <br />  <br /> A major goal developmental biologists define behav-                  A reproducible map embryonic cell fates zebrafish is <br /> iors molecular contents differentiating cells. We                   established &#11015;5 h post-fertilization (hpf), late blastula stage <br /> devised strategy isolating cells diverse embryonic                   (6). At time, embryonic cell layers arranged <br /> regions stages zebrafish, using computer-guided <span id='am-47' about='obo:NCBITaxon_62990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_241799 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_241789 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50'  ></span><span id='am-51' property="rdfs:label" content="Laser" datatype="xsd:string"></span>laser</span>                dome surrounds hemisphere single large yolk <br /> photoconversion injected Kaede protein flow cytometry.                   cell (Fig. 1A). The endoderm precursor field lies the <br /> <span id='am-22' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-23'  ></span><span id='am-24' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> strategy enabled perform genome-wide transcriptome                 lowest tiers dome (Fig. 1 A B) (7). The mesoderm <br /> comparison germ layer precursor cells. Mesendoderm ec-                   precursor field overlaps endoderm precursor field and <br /> toderm precursors cells isolated method differentiated                  extends higher tiers (6). The cells comprise over- <br /> appropriately transplantation assays. Microarray analysis                 lapping fields collectively termed mesendoderm precursors. <br /> these cells reidentified known genes efficiently                 Above sixth tier, staggered transition meso- <br /> previously reported strategies relied artificial mesend-                derm fates ectoderm fates, ectoderm precursors con- <br /> oderm activation inhibition. We identified large set               tinuing embryo (6). <br /> uncharacterized mesendoderm-enriched genes ecto-                        Traditional methods isolating early mesendoderm and <br /> derm-enriched genes. Loss-of-function studies revealed              ectoderm cells technically challenging (excising tops of <br /> these genes, MAP kinase inhibitor dusp4, essential early             embryos eyebrow hair explantation individual cells <br /> development. Embryos injected antisense morpholino oligo-                  micropipet) studies published <br /> nucleotides targeted Dusp4 displayed necrosis head tissues. <br />                                                                                 examined fate changes undergo transplanted <br /> Marker analysis late gastrulation revealed specific loss of <br />                                                                                 host embryos (8, 9) explanted cultured (10). <br />                                                                                 Transgenic zebrafish lines mesoderm-, endoderm-, ec- <br /> sox17, endoderm markers, analysis later <br />                                                                                 toderm-restricted fluorescent protein expression re- <br /> stages revealed loss foregut pancreatic endoderm. This <br />                                                                                 ported (11, 12), late-blastula stage, are <br /> specific loss sox17 establishes new <span id='am-19' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-20' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> endoderm speci- <br />                                                                                 bright FACS-based purification (J.L.B. B.F., <br /> fication defect. <br />                                                                                 unpublished observations). <br />                                                                                    To understanding germ-layer differentiation, <br /> microarray &#20841; zgc:55423 &#20841; gastrulation &#20841; mkp2 &#20841; sox17 <br />                                                                                 developed FACS-assisted microdissection photola- <br />                                                                                 beled (FAM-P) cells, strategy isolating zebrafish embryonic <br />  <br /> T   establishment germ layers coordinated <br />     rearrangement gastrulation core developmental <br /> process (1). In organisms ranging humans flatworms, <br />                                                                                 cells. FAM-P uses recombinant photoconvertible Kaede protein, <br />                                                                                 overcomes spatiotemporal limitations transgeni- <br />                                                                                 cally expressed fluorescent proteins injected mRNA and <br /> three primary germ layers arise: mesoderm forms deep                        allows quantitative isolation early embryonic zebrafish <br /> tissues blood muscle, endoderm produces viscera                 cells user-defined regions interest. We isolated endog- <br /> like digestive tract liver, ectoderm forms                  enous mesendoderm ectoderm precursors FAM-P and <br /> epidermis central nervous system. Cell lineage tracing                      transplanted host embryos, contributed to <br /> studies mapped precursor cells fated form                  similar range tissues mesendoderm ectoderm pre- <br /> these germ layers, key commonalities fate                  cursors traditionally explanted micropipet. DNA <br /> maps different vertebrates (2). Germ layer precursor cells               microarray analysis FAM-P-isolated mesendoderm ecto- <br /> typically distributed extended fields, making quantitative <br /> isolation manual microsurgery particularly challenging. <br />                                                                                 Author contributions: J.L.B., M.S., M.K., A.G.E., B.F. designed research; J.L.B., M.S., M.K., <br />    Fluorescent proteins opened door new microdis-                   S.-K.H., A.G.E. performed research; J.L.B., M.S., H.N., S.-K.H., B.F. analyzed <span id='am-57' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-60'  ></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span>; <br /> section approaches developmental biology. Placing different                  B.F. wrote paper. <br /> fluorescent proteins different promoters, distinct colors                The authors declare conflict interest. <br /> have expressed cell types comprising nascent skin                Data deposition: The data reported paper deposited Gene <br /> late gastrula endoderm developing mice (3, 4). In                      Expression Omnibus (GEO) database, www.ncbi.nlm.nih.gov/geo (accession no. GSE8654). <br /> studies, fluorescently labeled cells newborn                     &#8224;Presentaddress: Eli Edythe Broad Center Regenerative Medicine Stem Cell <br /> embryonic mice dissociated, separated FACS,                          Research, Keck School Medicine, University Southern <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-6'  ></span>California</span>, Los Angeles, CA <br />                                                                                  90089. <br /> mRNA harvested identify tissue-specific genes mi- <br />                                                                                 &#20648;To <br />                                                                                                                                                                                     DEVELOPMENTAL <br />  <br />  <br />  <br />  <br /> croarray analysis. More recently, photoconvertible fluorescent                      correspondence addressed at: Vertebrate Embryology Section, Med- <br />                                                                                 ical Genetics Branch, National Human Genome Research Institute, National Institutes of <br /> proteins Kaede (5) available, allowing <br />                                                                                                                                                                                        BIOLOGY <br />  <br />  <br />  <br />  <br />                                                                                 Health, Building 35 Room 1B205, 35 Convent Drive, MSC 3717, Bethesda, MD 20892-3717. <br /> researchers interactively convert expressing cells                  E-mail: bfeldman@mail.nih.gov. <br /> color another. Several promising uses photoconvertible                    This article contains supporting information online www.pnas.org/cgi/content/full/ <br /> proteins embryology described; however,                  0805589105/DCSupplemental. <br /> not used facilitate tissue-specific microdissection.                &#169; 2008 The National Academy Sciences USA <br />  <br />  <br />  <br /> www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.0805589105                                                    PNAS &#20841; August 26, 2008 &#20841; vol. 105 &#20841; no. 34 &#20841; 12337&#8211;12342 <br />  Fig. 1. Use Kaede protein <span id='am-46' about='rdfs:label' typeof='owl:Thing'>label</span> embryonic cells trace lineages. (A) Schematic late blastula stage (40% epiboly) embryo. The embryonic layer <br /> (EL) color coded according germ layer precursor distribution, dark red representing endoderm precursor-enriched, red representing mesoderm <br /> precursors, green representing ectoderm precursors. Also indicated yolk cell (YC, tan), enveloping layer (EVL, purple), yolk syncytial layer <br /> (YSL, light blue). For clarity, yolk cell represented separated overlying tissues. (B) Fate map late-blastula stage embryo Kimmel et al. <br /> (6) using color scheme A including precursor domains indicated outcomes H. (C) Coomassie-blue stained polyacrylamide gel, showing <br /> a 34-kDa monomer bacterially expressed Kaede column purification. (D) Four-cell-stage (1 hpf) embryo shortly injection 3.5 ng purified <br /> Kaede protein, lateral view. (E) 40% epiboly-stage (5 hpf) embryo just photolabeling margin, animal pole view. (F) Same embryo E, just <br /> after photolabeling margin, animal pole view. (G) Close-up Kaede protein-injected embryo, directly photolabeling margin, lateral <br /> view. Only optical sections near surface shown. Brackets numbers indicate lowest cellular tiers. (H) Same embryo G, at <br /> pharyngula stage (24 hpf), lateral view, anterior left. Representative tissues indicated H relative pixel values (% red total red &#11001; green) match <br /> expected outcomes follows: 1 (22%), 2 (25%), 3 (52%), 4 (52%), 5 (77%), 6 (78%). <br />  <br />  <br />  <br /> derm precursors revealed continue express molecular                          generated recombinant Kaede protein (Fig. 1C). Injection of <br /> markers specific regions origin. Microarray analysis                     purified Kaede protein nontoxic brightly labeled em- <br /> also identified dozens new mesendoderm precursor-enriched                         bryos time injection (Fig. 1D) late <br /> genes. Morpholino knockdown genes, dusp4, pro-                          blastula stage (Fig. 1 E&#8211;G) persisted 24 hpf (Fig. 1H), <br /> duced loss-of-function phenotype, characterized down-                       time signal gradually faded (data shown), <br /> regulation single endoderm marker gastrulation,                          presumably cellular catabolism. <br /> sox17, endoderm markers expressed nor-                              To specifically label mesendoderm precursors, microin- <br /> mally, loss foregut pancreatic endoderm later                        jected Kaede protein fertilized eggs late blastula <br /> stages. Our studies illustrate just ways FAM-P                      stage embryos mounted scanning confocal <br /> can applied experimental embryology. Given spatial                         microscope, embryo nearest <span id='am-13' about='obo:NCBITaxon_3863' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_163743 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-14'  ></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Lens" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Lens Mill." datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>lens</span> (Fig. <br /> and temporal flexibility FAM-P offers, useful                      1E). Orientation multiple specimens facilitated mount- <br /> for experiments zebrafish possibly                        ing embryos, transparent chorions intact, arrayed conical <br /> model organisms.                                                                     molds (13, 14). A circular swath defined comprising the <br />                                                                                      outermost, lowest, tiers cells, mes- <br /> Results                                                                              endoderm precursors photoconverted green red by <br /> Differential Labeling Germ Layer Precursors Kaede Pro-                   computer-controlled scanning near-UV laser (Fig. 1F). <br /> tein. The mesendoderm precursor field zebrafish late blastula                     Using method, label mesendoderm precursors of <br /> stage (40&#8211;50% epiboly, &#11015;5 hpf) embryos consists narrow                          8&#8211;12 embryos single experiment. <br /> band cells encircle delicate yolk cell (Fig. 1 A B).                      Traditional fate mapping requires lineage tracing single <br /> Because efficient isolation cells using traditional                   cells dozens embryos. Labeling entire margin visualized <br /> microsurgical tools untenable, set develop method                     key features fate map (Fig. 1B) single embryo. Image <br /> for fluorescently labeling cells isolating                   analysis day 2 shows tissues normally derived lower tiers <br /> FACS. For purpose, selected photoconvertible                             red pixels, tissues normally higher tiers with <br /> fluorescent reagent Kaede, stony coral protein fluoresces                     green pixels, tissues normally derived near the <br /> green (518 nm) full-length form red (582&#8211;627 nm)                    margin/nonmargin boundary similar numbers red and <br /> photocleavage near-UV light (405 nm) (5). We discovered                           green pixels, presumably representing mixture red, green, <br /> that nascent fluorescent protein maturation slow relative                      half-labeled yellow cells (Fig. 1H). This analysis demon- <br /> zebrafish development, late blastula-stage fluores-                        strates correctly target mesendoderm precur- <br /> cence injected kaede mRNA [see supporting information                           sor field indicates Kaede protein labeling does alter <br /> (SI) Fig. S1] zygotically expressed transgenic fluorescent                       cellular destinies. We similarly used Kaede protein label <br /> proteins (J.L.B. B.F., unpublished observations)                        trace outcomes embryonic regions (Fig. S1). Such <br /> bright FACS. To circumvent difficulty,                            lineage tracing experiments facilitated ability cells <br />  <br /> 12338 &#20841; www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.0805589105                                                                                                      Brown et al. <br />  labeled Kaede protein hold color time. When <br /> kaede mRNA injected, photolabeled lineages signif- <br /> icantly yellowed newly synthesized green Kaede <br /> protein (see Fig. S1). <br />  <br /> Isolation Mesendoderm Ectoderm Precursors. To quantita- <br /> tively purify mesendoderm ectoderm precursors, disso- <br /> ciated labeled embryos single cell suspensions sorted <br /> cells FACS (Fig. S2). To minimize losses labeled cells <br /> during processing (e.g., pipetting, centrifugation, FACS <br /> steps), mixed unlabeled cells uninjected <br /> sibling embryos point trypsinization. Forward side <br /> scatter values single embryonic cells determined, and <br /> events gates sorted relative red <br /> and green fluorescence. Using parameters, purify <br /> homogenous populations green red cells, recovery <br /> rate input embryonic cells close 75% 99% viability <br /> (data shown). <br />    A critical test FAM-P&#8217;s utility determine whether <br /> sorted cells retain developmental potentials biases. It <br /> has previously shown ectoderm precursors trans- <br /> planted animal pole reliably incorporated host <br /> ectoderm (8, 9). We true FAM-P-isolated <br /> ectoderm precursors, contributed ectoderm fates, such <br /> as retinal neurons forebrain cells (Fig. S2 Table S1). It <br /> was previously shown early-stage mesendoderm precursors <br /> (4.7&#8211;5 hpf) readily reprogrammed ectoderm (8, 9), but <br /> shortly [beyond 50% epiboly (5.3 hpf)] com- <br /> mitted mesendoderm fates (8). Our FAM-P purified mesen- <br /> doderm cells come intermediate stage [40&#8211;50% epiboly <br /> (5.0&#8211;5.3 hpf)] consistent this, contributed to <br /> ectoderm- mesendoderm-derived tissues, did marginal <br /> cells stage traditionally transplanted directly <br /> from embryo (Fig. S2 Table S1). We <br /> conclude FAM-P-purified mesendoderm ectoderm <br /> precursors maintain endogenous commitment status and <br /> developmental potential. <br />                                                                       Fig. 2. Transcriptome analysis mesendoderm neurectoderm precur- <br />                                                                       sor cells. (A B) Volcano plots showing 31,356 (A) 25,837 (B) oligos each. <br /> Identification Previously Characterized Uncharacterized Germ <br />                                                                       Each oligo mapped y <span id='am-27' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-28' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31'  ></span>axis</span> according relative enrichment in <br /> Layer-Specific Genes. To study transcriptomes         direct comparison source tissues (average normalized Log2 the <br /> mesendoderm ectoderm precursors, harvested am-             hybridization signal ratio) x axis according significance <br /> plified RNA purified precursor cells cohybridized          (negative Log2 scale uncorrected P value). (A) FAM-P isolated mesendoderm <br /> them oligo chips microarray analysis (Fig. 2A). As         precursor cells (&#11022; 0 Y axis) vs. cognate FAM-P isolated ectoderm precursor <br /> control photolabeling effects, prepared           cells (&#65533;? 0 Y axis). (B) Kaede-injected photoconverted embryo cells (&#11022; 0 <br /> cohybridized amplified RNA probes green red control          Y axis) vs. cells Kaede-injected embryos photoconverted <br /> cells, yielding substantially enriched genes (Fig. 2B). Thus,      (&#65533;? 0 Y axis). (C D) Bar graphs showing representation selected <br /> the expression profile embryonic cells substantially        molecular functions (C) signaling pathways (D) 60 most- <br />                                                                       enriched genes (Table S2). Mes, mesendoderm precursor enriched; Ect, ecto- <br /> altered photolabeling se. We identified 188 full-length <br />                                                                       derm precursor enriched; Con, red control cell enriched; E, Enzyme; LAR, <br /> cDNAs significantly enriched (P &#65533;? 0.05)              ligand (agonist), antagonistic ligand receptor; TF, transcription factor; N, <br /> mesendoderm precursor pool 106 similarly enriched cDNAs           Nodal pathway; W, Wnt pathway; F, FGF pathway; RA, retinoic acid pathway. <br /> in ectoderm precursor pool.                                       Genes acting multiple pathways arbitrarily assigned pathway <br />    We compiled cohorts 60 unique genes           following priority: RA &#11022; F &#11022; W &#11022; N. <br /> comparison minimum annotated information was <br /> available (Table S2) examined over-represented <br /> gene ontology terms (Fig. 2C Table S2). A striking percent-       mesendoderm precursors (15) known ectoderm- <br /> age (45%) mesendoderm precursor genes encode tran-             specific genes. We 38.3% mesendoderm <br /> scription factors (Fig. 2C). The frequency transcription factors   precursor cohort genes fall four <br /> on entire microarray, contrast, 10%, similar    pathways, ectoderm control cohort incidences were <br /> frequency seen control cohort (Fig. 2C). We noted      8.3% 1.7%, respectively (Fig. 2D). Although identified <br /> an excess ligands, agonists, receptors paucity        far fewer highly enriched ectoderm genes, slightly increased <br /> enzymes mesendoderm precursors compared                false discovery filter 5.5% reveals substantial enrichment of <br /> the control cohort (Fig. 2C). An elevated number transcrip-        characterized neurectoderm genes, sox3 (16), <br />                                                                                                                                                           DEVELOPMENTAL <br />  <br />  <br />  <br />  <br /> tion factors (32%) seen ectoderm precursors            sox 19a (17), lhx5 (18) otx2 (19), nonneural <br /> (Fig. 2C), molecular class           ectoderm gene foxi1 (20) (Fig. 2C Table S2). <br />                                                                                                                                                              BIOLOGY <br />  <br />  <br />  <br />  <br /> underrepresented.                                                        A number mesendoderm-enriched genes identified <br />    We scored mesendoderm, ectoderm, control              reported expression late blastula stage. We <br /> cohorts membership signaling pathways&#8212;Nodal,              characterized 21 genes mesendoderm precursor <br /> WNT, FGF, retinoic acid&#8212;which known active              pool cloning performing mount situ hybridiza- <br />  <br /> Brown et al.                                                                               PNAS &#20841; August 26, 2008 &#20841; vol. 105 &#20841; no. 34 &#20841; 12339 <br />                                                                                   expression mesoderm endoderm markers Dusp4 <br />                                                                                  morphants. We observed drastic reduction expression of <br />                                                                                  sox17 gastrula-stage endoderm cells specialized <br />                                                                                  group cells termed dorsal forerunners (Fig. 3 L M) <br />                                                                                  (22). Intriguingly, stage, endoderm markers as <br />                                                                                  foxa2 (23) (Fig. 3 N O) expressed normally Dusp4 <br />                                                                                  morphants. These phenotypes specific able to <br />                                                                                  phenocopy morphological molecular defects a <br />                                                                                  second, nonoverlapping MO Dusp4 (data shown) <br />                                                                                  rescue coinjection dusp4 mRNA (Fig. <br />                                                                                  3K). Analysis differentiated endoderm probes for <br />                                                                                  foxa2 (24) insulin (24) transcripts revealed loss foregut <br />                                                                                  (Fig. 3 P Q) pancreatic (Fig. 3 R S) endoderm. <br />  <br />                                                                                  Discussion <br />                                                                                  This paper introduces method viably isolating embryonic <br />                                                                                  cells user-defined regions stages interest. Injected <br />                                                                                  Kaede protein brightly label cells earlier exoge- <br />                                                                                  nous mRNA, problem novo synthesis over <br />                                                                                  course lineage tracing experiments. Caged fluorophores <br />                                                                                  suitable alternatives Kaede protein; however, these <br />                                                                                  reagents commercially unavailable years. <br />                                                                                  Another possible alternative recombinant photoconvertible <br />                                                                                  EosFP protein, used achieve early cell labeling <br />                                                                                  frog embryos (25). <br />                                                                                     For studies described here, microdissected embryos <br />                                                                                  1 h labeling; however, labeled cells allowed to <br />                                                                                  differentiate isolation. For instance, easily <br />                                                                                  isolated early-labeled mesendoderm precursors mid- and <br /> Fig. 3. Expression function new mesendoderm genes. (A&#8211;H) Whole            late-gastrula stages (data shown). Thus, diverse range of <br /> mount situ hybridizations (WISH) late-blastula-stage (5 hpf) embryos,      embryonic cell types labeled purified FAM-P. One <br /> validating mesendoderm-specific expression mesendoderm pre-         topological limitation noted. The cells labeled were <br /> cursor-enriched genes, indicated. Animal pole views shown, C&#8211;F        conveniently arranged shallow half dome. Differential <br /> have dorsal sides right (black arrows), confirmed double      labeling deep superficial cells feasible the <br /> stains chordin gene (data shown). (I&#8211;K) Embryos injected     single-photon apparatus used achievable with <br /> control morpholino (Ctrl MO), Dusp4 morpholino (Dusp4 MO), rescuing <br />                                                                                  multiphoton systems, reported photoconversion <br /> combination Dusp4MO dusp4 mRNA, shown pharyngula stage <br /> (28 hpf). (L&#8211;O) WISH late-gastrula-stage (80% epiboly) control Dusp4 <br />                                                                                  KikGR (26). <br /> MO-injected embryos, visualizing expression endoderm markers sox17           To knowledge, microarray analysis such <br /> (L M, dorsal views) foxa2 (N O, lateral views). Black arrowheads     study endogenous germ layer precursors zebrafish; how- <br /> indicate sample endoderm cells, open arrows point stained dorsal          ever, papers reported transcriptomes of <br /> forerunner cells. (P&#8211;S) WISH 18 somite-stage control (P), pharyngula-stage    zebrafish embryos forced over- underproduce mesend- <br /> (26 hpf) Dusp4 MO-injected (Q), pharyngula-stage (30 hpf) control        oderm alterations Nodal signaling (27&#8211;29). Each these <br /> morphant (R S) embryos, visualizing foxa2 (P Q) insulin (R S).   studies yielded important complementary data, is <br /> Embryos P Q stage matched time matched account        useful perform detailed comparison assess endoge- <br /> for developmental delays morphants. Black arrows P Q point to <br />                                                                                  nous approach. Our method best identifying non-Nodal <br /> posterior endoderm, including foregut region. Black arrows magnifica- <br /> tions R S highlight pancreatic stains. <br />                                                                                  mesendoderm pathway members (i.e., FGF, WNT, retinoic <br />                                                                                  acid), 12 60 genes categories, <br />                                                                                  compared 9 98 (28) 6 75 (27) studies <br /> tions late blastula-stage embryos. Ten genes showed                  presented nonvalidated hits (Table S2). Our approach <br /> margin-specific staining late blastula embryos, validating              successful identifying Nodal pathway genes. The <br /> enrichment mesendoderm precursors (Fig. 3 A&#8211;H                          highest number validated Nodal-regulated genes emerging <br /> data shown). We validated ectoderm-specific ex-                     studies 34 curated list 72 (29). Our 188 <br /> pression previously uncharacterized ectoderm precur-                  mesendoderm genes included 26 genes list have <br /> sor genes (data shown).                                                      dozens more, Fig. 3, pending validation (Table <br />    We used antisense morpholino oligonucleotides (MOs)                        S3). Thus, screening endogenous cells yielded overall greater <br />                                                                                  diversity pathway-specific genes. Our proposed explanation <br /> block translation validated mesendoderm precur- <br />                                                                                  difference hyperactivation Nodal signaling <br /> sor genes associate distinct and <br />                                                                                  extinguishes expression Nodal non-Nodal pathway <br /> reproducible phenotype: zgc:55423, 86% homology                        genes uniquely expressed lower endogenous Nodal <br /> (Clustal alignment) human dual-specificity phosphatase 4                      signaling levels. <br /> (DUSP4, alias: MKP2) accordingly termed                           Many mesendoderm precursor-enriched genes iden- <br /> dusp4. Dusps attenuate mitogen-activated protein kinase                          tified previously shown expressed mesend- <br /> (MAPK) signaling de-phosphorylation key residues,                     oderm precursors, half tested showed margin- <br /> with human DUSP4 reported target MAPKs ERK                            enriched expression situ, including dusp4. The loss of <br /> JNK (21). At pharyngula stage, Dusp4-depleted embryos                        expression sox17, foxa2, observed Dusp4 <br /> displayed developmental delays small necrotic heads,                     morphants gastrulation unusual define new <br /> suggesting essential role Dusp4 anterior development                   class endoderm defect; reduction sox17 been <br /> (Fig. 3J). Given dusp4&#8217;s expression margin, investigated               seen zebrafish sqt, cyc, oep, bon, fau, cas mutants, foxa2 <br />  <br /> 12340 &#20841; www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.0805589105                                                                                     Brown et al. <br />  has seen coordinately down-regulated exam-                                   media cell viability assays transplants. Methods transplantation <br /> ined (30). We report loss foregut pancreatic                                    experiments legend Fig. S2. <br /> endoderm Dusp4 morphants. This outcome consistent with <br /> sox17 decreases, mouse knockout Sox17                                        RNA Isolation, Amplification, Cy Labeling. Cells sorted TRIzol, <br />                                                                                               vortexed, frozen dry ice, stored &#11002;80&#176;C, extracted according the <br /> reported lack gut endoderm (31). <br />                                                                                               manufacturer&#8217;s protocol slight modifications. Approximately 300 ng 1 <br />    In summary, ability microdissect wide range early                              &#65533;?&#65533;g RNA linearly amplified using Amino Allyl MessageAmp II <span id='am-52' about='obo:NCBITaxon_993333' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_27548 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Arna" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-56'  ></span>aRNA</span> <br /> cells zebrafish opens numerous experimental possibilities                               Amplification kit (Ambion) yields ranging 12 30 &#65533;?&#65533;g aRNA. <br /> for developmental biologists. Future studies include,                               aRNA samples split labeled, half Cy3 mono NHS ester half <br /> instance, comparisons dorsal vs. ventral younger vs. older                              Cy5 mono NHS ester (CyDyes GE Healthcare; post-labeling reagents <br /> embryonic cell populations. We shown FAM-P-                                         MessageAmp II kit). Labeled aRNAs fragmented incubation <br /> purified zebrafish cells used transplantation assays                            94&#176;C 15 min fragmentation buffer (Affymetrix). Paired probes were <br /> microarray studies, applications envi-                                  prepared independent experiments microdissecting mesendoderm <br /> sioned, direct transcriptome sequencing, proteome anal-                               ectoderm precursors independent experiments using control <br /> ysis, establishment new embryonic cell lines.                                       red control green cells prepared follows. A number embryos were <br />                                                                                               injected Kaede photoconverted red toto, equivalent <br /> Materials Methods                                                                         number Kaede-injected embryos left green-fluorescent state. <br />                                                                                               These red-labeled green-labeled embryos pooled together <br /> Microinjection Kaede Photoconversion. Embryos obtained WT <br />                                                                                               unlabeled stage-matched embryos, disaggregated, entire cell <br /> adult zebrafish (AB, WIK, EK) natural mating. pKaede-S1 vector (MBL <br />                                                                                               population sorted isolate red green control cells. <br /> International) subcloned pRSET B (Invitrogen), expressed E. coli, <br /> and purified using Ni-NTA agarose affinity, sizing, hydroxyapatite col- <br />                                                                                               Microarray Hybridization, Transcriptomes Analysis, Validation. In-house <br /> umns (expression purification ProteinOne Inc.). The protein stored <br />                                                                                               microarray chips used (13). Four biological replicates mesendoderm <br /> in 5 mM Hepes (pH 7) 0.2 M KCl buffer &#11002;80&#176;C. For kaede dusp4 <br />                                                                                               ectoderm precursor probes cohybridized, dye swap, <br /> mRNA, NotI-linearized Kaede-pCS2&#11001; (32) dusp4-pCS2 used tem- <br />                                                                                               total hybridizations. Similarly, biological replicates of <br /> plate mMessage mMachine synthesis using SP6 RNA polymerase (Ambion). <br />                                                                                               red-cell green-cell probes dye swap cohybridized total <br /> Kaede protein injections 3.5 ng/embryo mRNA injections 300 pg/ <br />                                                                                               additional hybridizations. Hybridizations performed overnight at <br /> embryo carried 1&#8211;2 cell stage embryos subsequently <br />                                                                                               45&#176;C Maui Mixer FL hybridization chambers (BioMicro Systems). Post- <br /> incubated 28&#176;C. Embryos stage mounted their <br />                                                                                               hybridization processing scanning previously described (13). Data <br /> chorions agarose array conical imprints (13, 14). By using laser <br />                                                                                               points quality values &#65533;?0.85 eliminated, dataset was <br /> scanning microscope (Carl Zeiss LSM 510 <span id='am-37' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-38' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41'  ></span>META</span>), Kaede photoconverted <br />                                                                                               normalized median shift following log2 transformation using the <br /> in regions interest, margin, 405-nm laser imaged via <br />                                                                                               Avadis 4.0 software program. P values average log ratios deter- <br /> 488-nm 543-nm excitation green red Kaede, respectively. Emitted <br />                                                                                               mined oligo array described ref. 13 predicted genes, <br /> light respectively filtered 505&#8211;530-nm BP 560-nm LP filters. <br />                                                                                               ESTs, duplicates disregarded. Representation molecular catego- <br /> For loss-of-function studies, 1&#8211;3 ng antisense splicing MO (Gene Tools), <br />                                                                                               ries enriched genes mesendoderm precursor vs. ectoderm <br /> targeting zygotic dusp4 translation (5&#11032;-TAGCCCCCTGTAAAAGTGGAAAAGG- <br />                                                                                               precursor comparisons red vs. green control-cell comparisons was <br /> 3&#11032;), injected cell-stage embryos. A similar, nonoverlapping MO <br />                                                                                               preliminarily assessed GeneSifter software (VizX Labs), looking ap- <br /> (5&#11032;-TGCTGCTTGTGTATTTACCTGGTCG-3&#11032;) yielded indistinguishable results. <br />                                                                                               proximately 100 genes available RefSeq IDs. This assessment was <br /> For control embryos, 5 ng standard control MO (5&#11032;-TAGTTAAGC- <br />                                                                                               followed hand-curated analysis 60 most-enriched genes each <br /> CTAGCTCTCATAAACT-3&#11032;) injected. For rescue, 50-pg full-length dusp4 <br />                                                                                               category (Table S2). <br /> mRNA coinjected. <br />                                                                                                  For validation studies, primers designed amplify coding se- <br />                                                                                               quence 5&#11032; 3&#11032; UTRs genes selected study cloned by <br /> Embryonic Cell Sorting Transplantation. Embryos dechorionated with <br />                                                                                               PCR TOPO TA <span id='am-42' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-43' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>vectors</span> (Invitrogen). DIG-labeled RNA probes pre- <br /> Pronase (P8811, Sigma) transferred siliconized glass vials. Uninjected <br />                                                                                               pared using PCR-generated DNA templates DIG RNA labeling kit with <br /> embryos added samples &#65533;?20 labeled embryos bring total <br />                                                                                               SP6 T7 polymerase (Roche), situ hybridizations carried <br /> number 20 &#8211;30. For dissociation, embryos incubated 6 min 300 <br />                                                                                               essentially described ref. 33, post-hybridization washes per- <br /> &#65533;?&#65533;l trypsin solution (27) triturated transfer pipette midin- <br />                                                                                               formed automated liquid exchanger (Biolane HTI, Ho&#168;lle &amp; Hu&#168;ttner). <br /> cubation. After dissociation trypsin, 30 &#65533;?&#65533;l heat-inactivated FBS (HI-FBS) <br />                                                                                                  All zebrafish work approved National Human Genome Research <br /> (Gibco) added vial flooded L-15 Medium (Sigma) <br />                                                                                               Institute&#8217;s Animal Care Use Committee. <br /> containing 0.1% BSA, 0.8 mM CaCl2, 2 mM L-glutamine, 100 U/ml penicillin, <br /> and 0.1 mg/ml streptomycin. The cells centrifuged 100 &#11003; g 4 min <br /> at 4&#176;C &#11015;700 &#65533;?&#65533;l media removed. The cells transferred                   ACKNOWLEDGMENTS. We thank Darryl Leja artwork expert assis- <br />                                                                                               tance graphics; <span id='am-7' about='obo:NCBITaxon_381266' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Alexandra Bunge" datatype="xsd:string"></span><span id='am-10'  ></span><span id='am-11' property="rdfs:label" content="Alexandra" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Alexandra</span> Joyner, Paul Liu, David Bodine, Igor Dawid, <br /> to Titertube Micro Tube (Bio-Rad) inside 5-ml polystyrene tube, 70 <br />                                                                                               Mary LaMarca helpful comments manuscript; Erich Roessler and <br /> &#65533;?&#65533;l HI-FBS added. The cells transported ice flow cytometer                 Stacie Anderson valuable discussions; Atsushi Miyawaki sharing <br /> (FACSDiVa Vantage, BD Biosciences) sorted based                      Kaede-pCS2&#11001; vector. This research supported Intramural <br /> forward scatter vs. scatter green vs. red fluorescence using                    Research Program National Human Genome Research Institute, Na- <br /> 100-&#65533;?&#65533;m tip 8 &#8211;10 psi TRIzol (Invitrogen) RNA isolation cell             tional Institutes Health. <br />  <br />  <br />  1. Haeckel E (1874) Die gastraea-theorie: Die phylogenetische classification des thier-       9. Ho RK, Kimmel CB (1993) Commitment cell fate early zebrafish embryo. <br />     reichs und die homologie der keimbla&#168;tter. Jenaische Zeitschrift fu&#168;r Naturwissen-            Science 261:109 &#8211;111. <br />     schaften 8:1&#8211;55.                                                                          10. Sagerstrom CG, Gammill LS, Veale R, Sive H (2005) Specification enveloping <br />  2. Hatada Y, Stern CD (1994) A fate map epiblast early chick embryo.               layer lack autoneuralization zebrafish embryonic explants. Dev Dyn <br />     Development 120:2879 &#8211;2889.                                                                   232:85&#8211;97. <br />  3. Rendl M, Lewis L, Fuchs E (2005) Molecular dissection mesenchymal-epithelial           11. Ellingsen S, et al. (2005) Large-scale enhancer detection zebrafish genome. <br />     interactions hair follicle. PLoS Biol 3:e331.                                          Development 132:3799 &#8211;3811. <br />  4. Sherwood RI, et al. (2007) Prospective isolation global gene expression analysis   12. Fan X, et al. (2007) Nodal signals mediate interactions extra-embryonic <br />     definitive visceral endoderm. Dev Biol 304:541&#8211;555.                                       embryonic tissues zebrafish. Dev Biol 310:363&#8211;378. <br />  5. Ando R, Hama H, Yamamoto-Hino M, Mizuno H, Miyawaki A (2002) An optical marker            13. Pei W, et al. (2007) An early requirement maternal FoxH1 zebrafish gastru- <br />     based UV-induced green-to-red photoconversion fluorescent protein. Proc           lation. Dev Biol 310:10 &#8211;22. <br />                                                    <br /> </body></html>