<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />    QIP, a Component of the Vegetative RNA Silencing Pathway, Is Essential <br />       for Meiosis and Suppresses Meiotic Silencing in <span id='am-16' about='obo:NCBITaxon_5140' typeof='obo:NCBITaxon_222544 owl:Thing obo:NCBITaxon_5139 obo:NCBITaxon_1 obo:NCBITaxon_5148 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_147550 obo:NCBITaxon_715989 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_2759'><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Neurospora" datatype="xsd:string"></span><span id='am-20'  ></span>Neurospora</span> <span id='am-6' about='obo:NCBITaxon_688987' typeof='owl:Thing obo:NCBITaxon_37581 obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_116123 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_57992 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Crassa" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>crassa</span> <br />  <br />   <br />                                                               ABSTRACT <br />                 Among the processes that play essential roles in both genome defense and organism survival are those <br />              involved in chromosome comparison. They are acutely active in the meiotic cells of <span id='am-21' about='obo:NCBITaxon_5141' typeof='obo:NCBITaxon_222544 owl:Thing obo:NCBITaxon_5139 obo:NCBITaxon_1 obo:NCBITaxon_5148 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_147550 obo:NCBITaxon_715989 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_5140'><span id='am-22' property="rdfs:label" content="Neurospora crassa" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Chrysonilia crassa" datatype="xsd:string"></span><span id='am-26'  ></span>Neurospora crassa</span>, <br />              where they evaluate the mutual identity of homologs by a process we call trans-sensing. When non- <br />              symmetrical regions are found, they are silenced. The known molecular components of <span id='am-27' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-31'  ></span>this</span> meiotic <br />              silencing machinery are related to RNA-dependent RNA polymerases, Argonautes and Dicers, suggesting <br />              that the mechanisms of how heterologous chromosomal regions are silenced involves, at some stage, the <br />              production of small interfering RNAs. Neurospora has two active and clearly distinct RNA interference <br />              pathways: quelling (vegetative specific) and meiotic silencing (meiosis specific). Both pathways require a <br />              common set of protein types like RNA-dependent RNA polymerases, Argonautes and Dicers. In this work <br />              we demonstrate the involvement of quelling defective-2 interacting protein (qip1), a Neurospora gene whose <br />              function is essential to silencing by quelling, in meiotic silencing, and normal sexual development. Our <br />              observations reinforce the molecular connection between these two silencing pathways. <br />  <br />  <br />  <br />  <br /> E    UKARYOTIC genomes require the activity of com- <br />        plex mechanisms aimed at maintaining and pre- <br /> serving their molecular integrity. At least four distinct <br />                                                                             vation was that mutants carrying a deletion of the <br />                                                                             transcription factor Ascospore maturation-1 (asm-11) were <br />                                                                             viable, and yet, when crossed to wild type, the progeny <br /> but potentially interrelated such processes are known                       was composed of dead sexual spores (Aramayo and <br /> in Neurospora crassa: DNA methylation, quelling, repeat-                    Metzenberg 1996; Aramayo et al. 1996). The connec- <br /> induced point mutation (RIP), and meiotic silencing                         tion between this position-effect phenomenon to RNA <br /> (Galagan et al. 2003; Borkovich et al. 2004).                               silencing was made by the characterization of a suppres- <br />    Quelling, or vegetative RNA silencing, was observed                      sor of meiotic silencing, Suppressor of ascus dominance-1 <br /> at the very dawn of the RNA silencing field (Cogoni                         (Sad-1) (Shiu et al. 2001), which implicated SAD-1, <br /> et al. 1996). Like in other organisms, its discovery was                    an RNA-dependent RNA polymerase in the pathway. <br /> born by the need to explain why having two copies of                        Further characterization of the pathway revealed the in- <br /> a gene was equivalent to having none (Cogoni et al.                         volvement of the Argonaute Suppressor of meiotic silencing- <br /> 1994). Genetic dissection of this pathway uncovered the                     2 (Sms-2) and of dicer-like-1/Suppressor of meiotic silencing-3 <br /> genes quelling defective-1 (qde-11), -2 (qde-21), -3 (qde-31),              (Dcl-1/Sms-3) (Galagan et al. 2003; Lee et al. 2003b; <br /> encoding for an RNA-dependent RNA polymerase                                Alexander et al. 2008). Unlike quelling, the molecular <br /> (QDE-1), an Argonaute (QDE-2), and a RecQ DNA                               properties required to trigger meiotic silencing are well <br /> helicase (QDE-3), respectively (Cogoni and Macino                           understood (Kutil et al. 2003; Lee et al. 2003b; Lee et al. <br /> 1997, 1999a, 1999b; Catalanotto et al. 2000).                               2004; Pratt et al. 2004). Exactly how this symmetry- <br />    Like the discovery of quelling, meiotic silencing was                    evaluation process works is still a mystery, but we know <br /> also born out of the need to explain an odd observa-                        that it involves the evaluation of identity at the most <br /> tion: How could a gene be dominant by being absent?                         detailed level (Pratt et al. 2004). A model for meiotic <br /> (Aramayo and Metzenberg 1996). Our initial obser-                           silencing has been described previously (Aramayo and <br />                                                                             Pratt 2010). It involves the recognition of nonhomol- <br />                                                                             ogy between homologous chromosomes by unknown <br />   Supporting information is available online at http://www.genetics.org/ <br />                                                                             molecular players. Once identified, heterologous re- <br /> cgi/content/full/genetics.110.118422/DC1.                                   gions are predicted to produce some form of aberrant <br />   This work is dedicated to the memory of David Dexter Perkins (1919&#8211;       RNA, which is converted into dsRNA by the action of the <br /> 2007), whose leadership and spirit guided and inspired our work. David,     RdRP, SAD-1. Once produced, dsRNA triggers the ini- <br /> your light shall illuminate our work forever. <br />   1 <br />                                                                             tiation step of the pathway, which is predicted to involve <br />    Corresponding author: Department of Biology, College of Science, Texas <br /> A&amp;M University, Room 415, Bldg. BSBW, College Station, TX 77843-            the conversion of the dsRNA trigger into siRNAs. This <br /> 3258. E-mail: raramayo@<span id='am-40' about='obo:NCBITaxon_114495' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6547 obo:NCBITaxon_106220 obo:NCBITaxon_6546 obo:NCBITaxon_6545 obo:NCBITaxon_6544 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_117558 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-41' property="rdfs:label" content="Tamu" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>tamu</span>.edu                                             step is most likely executed by the DCL-1/SMS-3 Dicer. <br /> Genetics 186: 127&#8211;133 (September 2010) <br />  128                                               D. W. Lee, R. Millimaki and R. Aramayo <br />  <br />                                                                     TABLE 1 <br />                                                        Fungal strains used in this study <br />  <br /> <span id='am-1' about='obo:NCBITaxon_389777' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_105691 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_6894 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Namea" datatype="xsd:string"></span><span id='am-5'  ></span>Namea</span>                                          Genotypeb                                                           Origina,c <br /> 367-4                          QipDThph1 A                                                  From Yi Liu (Maiti et al. 2007) <br /> DLNCR517                       f l; Asm-1-Ama-11Thph1 A                                     Progeny from DLNCR455 3 DLNCR469 <br /> DLNCR518                       f l; Asm-1-Ama-11Thph1 a                                     Progeny from DLNCR455 3 DLNCR469 <br /> DLNCR553                       gle-11-sgfp1Thph1 A                                          Progeny from RANCR05A 3 DLNCT545 <br /> DLNCR554                       gle-11-sgfp1Thph1 a                                          Progeny from RANCR05A 3 DLNCT545 <br /> DLNCR707                       A                                                            Progeny from RANCR05A 3 RANCR49A <br /> DLNCR708                       a                                                            Progeny from RANCR05A 3 RANCR49A <br /> DLNCR750                       QipDThph1 a                                                  Progeny from 367-4 3 KBNCR06A <br /> DLNCR769                       ridRIP1; qip1-sgfp1Thph1, mus-53RIP A                        Progeny from DLNCT756 3 DLNCR708 <br /> DLNCR770                       qip1-sgfp1Thph1, mus-53RIP a                                 Progeny from DLNCT756 3 DLNCR708 <br /> DLNCT807                       ridRIP1, his-31Tccg-1(p)-qde-2-sgfp1; inl A                  Transformation of DLNCR243 with pDLAM384 <br /> DLNCT808                       ridRIP2, his-31Tccg-1(p)-qde-2-sgfp1; inl a                  Transformation of DLNCR244 with pDLAM384 <br /> KBNCR05A                       his-3, RspRIP93; f l A                                       Progeny from DLNCR93 3 RANCR50A <br /> KBNCR06A                       RspRIP93; f l a                                              Progeny from DLNCR93 3 RANCR50A <br /> RMNCR113                       Qip DThph1; gle-11-sgfp1Thph1 A                              Progeny from 367-4 3 DLNCR554 <br /> RMNCR114                       Qip DThph1; gle-11-sgfp1Thph1 a                              Progeny from 367-4 3 DLNCR554 <br />    a <br />      DLNC, KBNC, RANC, and RMNC indicate strains constructed or provided for this study by Dong W. Lee, Kevin Baker, Rodolfo <br /> Aramayo and Ryan Millimaki, respectively. <br />    b <br />      Allele numbers or designations are: mating type A, A; mating type a, a; Ascospore maturation-africana-1, Ama-1; Ascospore maturation-1, <br /> Asm-1; clock-controlled gene-1 promoter, ccg-1(p); fluffy, fl (P); <span id='am-11' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-12' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-13'  ></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>glycine</span>-leucine-phenylalanine-glycine lethal-1, gle-1; histidine-3, his-3 (1-234- <br /> 723); hygromycin B phosphotransferase, hph; inositol, inl (89601); mutagen-sensitive-53, mus-53; quelling-deficient-2, qde-2; quelling-deficient- <br /> 2-interacting protein, qip; repeat-induced-deficient, rid; Roundspore, Rsp; S65T green fluorescent protein, sgfp. <br />    c <br />      Information regarding the construction of parental strains and/or <span id='am-73' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-74' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span> used will be provided upon request. <br />  <br />  <br /> The maintenance of silencing probably involves ampli-                        sponding protein complexes involved must then share <br /> fication, also through SAD-1, and degradation of the                         as many molecular components as possible. When the <br /> target mRNAs via an RNA-induced silencing complex                            Neurospora genome became available we found that <br /> (RISC), of which the Argonaute-like protein SMS-2 is                         many components involved in quelling had correspond- <br /> predicted to be an essential component. Interestingly,                       ing paralogs, and, as predicted, at least some of them <br /> the SAD-2 protein is also required for meiotic silencing,                    were involved in meiotic silencing (Lee et al. 2003b). <br /> but its role appears to be to anchor or recruit the SAD-1                    The unexpected presence of paralogs was explained to <br /> RdRP to the cytoplasmic face of the nuclear periphery                        be due to the complexity of meiotic silencing. <br /> (Shiu et al. 2006) (D. W. Lee and R. Aramayo, un-                               A common hallmark of the conserved RNA interfer- <br /> published results). This indicates that SAD-1&#8217;s RdRP                         ence pathways (RNAi) in all organisms is the activation <br /> activity is required outside of the nucleus, which is never                  of the RISC by Argonaute proteins, a process that <br /> completely dissolved in Neurospora meiosis. RNAs                             requires the separation of the siRNA duplex into single <br /> might be processed by SAD-1 as they transit nuclear                          strands (Paroo et al. 2007). In the Neurospora quelling <br /> pores as a sort of quality check. In any case, RdRP activity                 pathway, QIP, the product of the quelling defective-2 <br /> is likely a downstream effector of meiotic silencing and                     interacting protein (qip1) gene, is essential for this pur- <br /> not a component of the trans-sensing mechanism that                          pose and acts as an exonuclease that cleaves and re- <br /> must necessarily be nuclear. An &#8216;&#8216;RNA quality&#8217;&#8217; monitor-                     moves the nicked passenger strand from the siRNA <br /> ing system, however, implies that deficient pairing                          duplex in a QDE-2-dependent manner (Maiti et al. <br /> triggers production of aberrant transcripts to alert the                     2007). Here we report the involvement of qip1 in meiotic <br /> system, and hence that proficient recognition and                            silencing and sexual development. This establishes QIP <br /> pairing suppresses this transcription.                                       as a common component of quelling and meiotic silenc- <br />    Clearly, the presence of mechanisms that evolved to                       ing and reinforces the notion that quelling and meiotic <br /> preserve genome integrity have had profound evolu-                           silencing are intrinsically connected (Catalanotto et al. <br /> tionary consequences for Neurospora and shaped its                           2000; Cogoni 2001; Lee et al. 2004). <br /> genome. Currently, Neurospora has the lowest number <br /> of paralogous genes in comparison to other organisms <br /> (Galagan et al. 2003; Borkovich et al. 2004), suggest- <br /> ing that if related molecular mechanisms operate at                                          MATERIALS AND METHODS <br /> different stages of its life cycle (e.g., RNA silencing op-                    Molecular biology: Basic procedures for DNA cloning, <br /> erating in vegetative cells vs. meiotic cells), the corre-                   analysis, sequencing, Southern blot, and other nucleic acid <br />                                                  Exonucleases in Sex and Meiotic Silencing                                             129 <br />  <br />                                                                  TABLE 2 <br />                                                     QIP is involved in meiotic silencing <br />  <br />       Linkage <br />        group:                                                                     Total    Wild-type                       Suppression <br /> Cross Relevant        I       III        VI                                    ascospores ascospores                            of <br /> no. genotypea        rsp      qip       asm-1                Parents           examined      (%)b         Observations      silencing? <br />                       1         1        1 <br /> 1                  rsp        qip    asm-1          DLNCR707 3 DLNCR708           5161      86.8 6 0.5 Control                 n/ac <br />                    rsp1       qip1   asm-11 <br /> 2                  RspRIP93   qip1   asm-11         KBNCR05 3 KBNCR06             2541         0        Control                n/ac <br />                    RspRIP93   qip1   asm-11 <br /> 3                  rsp1       qip1   asm-11         DLNCR707 3 KBNCR06            2743       6.1 6 0.9 Meiotic silencing       No <br />                    RspRIP93   qip1   asm-11 <br /> 4                  rsp1       QipD   asm-11         367-4 3 KBNCR06               4139      14.1 6 1.1 Experimental            Yes <br />                    RspRIP93   qip1   asm-11 <br /> 5                  rsp1       qip1   Asm-1-Ama-11   DLNCR517 3 DLNCR518           8382      73.1 6 1.3 Control                 n/ac <br />                    rsp1       qip1   Asm-1-Ama-11 <br /> 6                  rsp1       qip1   Asm-1-Ama-11   DLNCR517 3 DLNCR708           7187       9.2 6 5.4 Meiotic silencing       No <br />                    rsp1       qip1   asm-11 <br /> 7                  rsp1       QipD   asm-11         367-4 3 DLNCR518              7034      36.6 6 2.1 Experimental            Yes <br />                    rsp1       qip1   Asm-1-Ama-11 <br />     a <br />       Complete genotype described in Table 1. <br />     b <br />       Scoring done as described in Figure 1. <br />     c <br />       Not applicable as meiotic silencing is not induced. <br />  <br />  <br /> manipulations were performed as described (Pratt and                     transferred to a hemocytometer and covered with a glass cover <br /> Aramayo 2002; Pratt et al. 2004).                                        slip. Six pictures were taken from each cross and the num- <br />    Strain description and construction: <span id='am-45' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-53'  ></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span>Escherichia coli</span> K12             ber of round and spindle ascospores were counted. For both <br /> XL1-Blue MR (Stratagene) was the host for all bacterial                  Asm-1- and Rsp-silencing experiments, two or three duplicate <br /> manipulations. All N. crassa strains used in this study are              crosses were performed per experiment. Multiple duplicate <br /> described in Table 1. The formulas for the Vogel&#8217;s medium N,             crosses were performed to account for plate-to-plate variation. <br /> the Westergaard&#8217;s medium, and the sugar mixture of Brock-                The percentage of wild-type spores was averaged between the <br /> man and de Serres have been described by Davis and De                    crosses and a standard deviation was determined to provide <br /> Serres (1970). Similarly, growth conditions, conidial sphero-            error bars. <br /> plast preparation, and fungal transformation were performed <br /> as described (Pratt and Aramayo 2002). Homokaryon <br /> purification was performed as described (Pratt and Aramayo                               RESULTS AND DISCUSSION <br /> 2002; Lee et al. 2003a). <br />    Preparation of perithecial tissues for microscopy: Perithe-              QIP is involved in meiotic silencing: To test the role <br /> cia were collected (4 days after fertilization), fixed (90 mm            of QIP in meiotic silencing we used a strain carrying a <br /> Pipes pH 6.9, 10 mm EGTA pH 7.5, 10 mm MgSO4, 0.3% Triton                deletion of the gene or its derivatives (Table 1). Testing <br /> X-100, and 3.7% formaldehyde for 30 min), washed (twice                  its involvement in the homozygous loss-of-function con- <br /> with 13 PBS), and observed (while suspended in 13 PBS, 15% <br /> glycerol, pH 7.0 solution containing 0.75 mg/ml DAPI) at                 dition proved to be impossible due to the barren <br /> 6003 magnification. Pictures were taken with Photometrics                phenotype of these crosses (see below). We therefore <br /> CoolSnap HQ2 <span id='am-32' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-36'  ></span>camera</span> using RSimage software from Roper                    assayed meiotic silencing in the heterozygous condition <br /> Scientific.                                                              following a very-well-established protocol (Lee et al. <br />    Assaying meiotic silencing: All crosses are listed in Table 2.        2003b, Figure 1, and Table 2). For this we used the <br /> For quantification of Asm-1 silencing, progeny from direc- <br /> tional crosses were allowed to shoot onto petri dish lids. At            best-characterized and sensitive inducers of meiotic <br /> 15 days postfertilization (dpf), spores were collected from the          silencing, the Roundspore (Rsp) allele RspRIP93 (Pratt <br /> lid with 1 ml of water. An aliquot of this population was placed         et al. 2004) and a chimeric fusion where the promoter of <br /> on a hemocytometer and covered with a glass cover slip. Six              the Ascospore maturation-1 (Asm-1) gene of N. crassa is <br /> pictures were taken from each cross and the number of white              fused to the coding and trailer region of the Ascospore <br /> and black spores was determined from these pictures. Quan- <br /> tification of Rsp silencing was done as follows: at 15 dpf,              maturation-1 (Ama-1) gene of N. africana (Asm-1-Ama-11). <br /> ascospores were collected in 1 ml water from the lid of the              Unlike the RspRIP93 allele, which is nonfunctional and <br /> plate then transferred to a microfuge tube. Round-shaped                 methylated, the Asm-1-Ama-11 allele is both functional <br /> spores are not ejected as efficiently as wild-type spores and            and not methylated (<span id='am-68' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-72'  ></span>data</span> not shown), but induces <br /> tend to ooze out of the perithecia, so the surface of crossing           meiotic silencing due to a series of polymorphisms <br /> plate was scraped using a glass rod and 3 ml of water. Collected <br /> spores were combined and filtered through Miracloth. The                 present in both its coding and 39-untranslated regions. <br /> combined spores were centrifuged at 13,500 rpm for 10 min to             As expected, the crosses between the &#8216;&#8216;wild-type&#8217;&#8217; strains <br /> concentrate. An aliquot of ascospores from each cross was                rsp1 3 rsp1 or Asm-1-Ama-11 3 Asm-1-Ama-11 produced <br />  130                                          D. W. Lee, R. Millimaki and R. Aramayo <br />  <br />                                                                                               Figure 1.&#8212;QIP is required for effi- <br />                                                                                            cient meiotic silencing. The involve- <br />                                                                                            ment of qip1 in the silencing induced <br />                                                                                            by homeologous and indel alleles was <br />                                                                                            determined by quantifying the ratio of <br />                                                                                            wild-type to mutant ascospores in each <br />                                                                                            cross. Efficient meiotic silencing results <br />                                                                                            in the production of ascospores that are <br />                                                                                            round instead of spindle shaped for Rsp <br />                                                                                            and white instead of black for Asm-1. <br />                                                                                            The weaker the silencing the higher <br />                                                                                            the percentage of wild-type ascospores <br />                                                                                            produced in the cross. Note that the ab- <br />                                                                                            sence of spindle spores in cross 2 is not <br />                                                                                            due to silencing but to the loss of Rsp <br /> function. Similarly, the number of wild-type ascospores observed in cross 5 is due to the nature of the Asm-1&#8211;Ama-11 fusion allele. <br /> Crosses are described in Table 2. See text and Pratt et al. (2004) for details. <br />  <br /> ascospores with the wild-type phenotype (86.8 6 0.5%                 (his-3) chromosomal location (Freitag et al. 2004). In <br /> and 73.1 6 1.3%; Figure 1, crosses 1 and 5, respectively),           homozygous crosses we were unable to observe perinu- <br /> while crosses between the mutant strains RspRIP93 3                  clear localization of QDE-2&#8211;SGFP in meiosis I (support- <br /> RspRIP93 produced only round ascospores (Figure 1, cross             ing information, Figure S1, A&#8211;H). Taken together, these <br /> 2). Unpairing Rsp in an otherwise wild-type background               observations are consistent with QIP, an Argonaute- <br /> results in the production of ascospores with a mutant                interacting protein, working at the perinuclear region in <br /> round (instead of the wild-type spindle) phenotype                   meiosis, the same region where the Argonaute SMS-2, <br /> (Pratt et al. 2004) (6.1 6 0.5%; Figure 1, cross 3).                 not QDE-2, is located. While these observations re- <br /> Silencing is compromised when RspRIP93 is unpaired in                inforce the notion that QIP&#8211;sGFP might be working <br /> the presence of an unpaired copy of Qip (14.1 6 1.1%;                with SMS-2, they do not establish a direct physical <br /> Figure 1, cross 4). We also induced meiotic silencing by             interaction. <br /> crossing strains carrying the Asm-1-Ama-11 allele with                  QIP is essential for early sexual development: The <br /> those carrying the wild-type asm-11. In heterozygous                 homozygous barren phenotype of qip loss-of-function <br /> crosses (i.e., asm-11 3 Asm-1-Ama-11), Asm-1 unpairing               mutants was further investigated. Sexual development <br /> results in efficient silencing as measured by the low <br /> production of black, mature ascospores (9.2 6 5.4%; <br /> Figure 1, cross 6). As expected, this silencing is sup- <br /> pressed by unpairing the same reporter gene in the <br /> presence of an unpaired copy of Qip (36.6 6 2.1%; <br /> Figure 1, cross 7). Combined, these results suggest that, <br /> under these conditions, QIP is involved in meiotic <br /> silencing. <br />    QIP localization is perinuclear in meiosis: Previous <br /> studies have established that components of the meiotic <br /> silencing machinery are perinuclearly localized (Shiu <br /> et al. 2006; Alexander et al. 2008). Their common lo- <br /> calization and function suggests that they work together, <br /> possibly through direct interactions (Bardiya et al. <br /> 2008). It has been previously established that QIP <br /> interacts with QDE-2 (Maiti et al. 2007). To understand <br /> how QIP works in meiotic silencing we first determined <br /> its subcellular localization in meiosis. We constructed a <br /> fusion between qip1 and the reporter gene sgfp1 and <br /> inserted the resulting fusion (qip1&#8211;sgfp1) at the canon- <br /> ical chromosomal location by gene replacement. Ho- <br /> mozygous crosses between these strains were fertile,                   Figure 2.&#8212;Localization of QIP&#8211;sGFP in meiosis. Pictures <br /> suggesting the fusion is functional. We observed that                show representative asci photographed at the pachytene stage <br /> QIP&#8211;sGFP has a perinuclear localization in meiosis I                 of meiosis I. Columns 1 and 2 show the QIP&#8211;sGFP (green) <br /> (Figure 2, A&#8211;K). To determine if QDE-2 had a similar                 and DAPI (red) signals, respectively. Column 3 shows the cor- <br />                                                                      responding merged images of columns 1 and 2. Tissues were <br /> localization, we then constructed a qde-21&#8211;sgfp1 fusion              harvested at 4 days postfertilization and processed as described <br /> gene under the control of the clock controlled gene-1                in materials and methods The bar on A, column 1, <br /> promoter (ccg-1(p)) and inserted it at the histidine-3               equals 20 mm. <br />                                              Exonucleases in Sex and Meiotic Silencing                                            131 <br />  <br />                                                                                                             Figure 3.&#8212;QIP is essen- <br />                                                                                                          tial for the completion of <br />                                                                                                          early sexual development. <br />                                                                                                          (A) Sexual development in <br />                                                                                                          N. crassa. The cartoon dia- <br />                                                                                                          gram represents the three <br />                                                                                                          main stages (early develop- <br />                                                                                                          ment, middle development, <br />                                                                                                          and late development), <br />                                                                                                          from fertilization to asco- <br />                                                                                                          spore maturation observed <br />                                                                                                          during sexual develop- <br />                                                                                                          ment. Drawings are not to <br />                                                                                                          scale. Cartoon was inspired <br />                                                                                                          on the work of Raju <br />                                                                                                          (1980). (B). Early wild-type <br />                                                                                                          development.       Maternal <br />                                                                                                          and ascogenous tissue cor- <br />                                                                                                          responding to a wild-type <br />                                                                                                          cross     (DLNCR553       3 <br />                                                                                                          DLNCR554; Table 2) was <br />                                                                                                          examined. In 1A, 1B, and <br />                                                                                                          2A 8A, GLE-1&#8211;sGFP signal <br />                                                                                                          is green, whereas DAPI sig- <br />                                                                                                          nal is red. 1A shows repre- <br />                                                                                                          sentative maternal tissue. <br />                                                                                                          1B shows a 103 enlarge- <br />                                                                                                          ment of one of the nuclei <br />                                                                                                          present in 1A. 2A 8A show <br />                                                                                                          merge images of develop- <br />                                                                                                          ing croziers as they go <br />                                                                                                          through formation, conju- <br />                                                                                                          gate mitosis, karyogamy, <br />                                                                                                          and early meiosis I. 2B 8B <br />                                                                                                          show light images. (C). <br />                                                                                                          Early QipD development. <br />                                                                                                          Maternal and ascogenous <br />                                                                                                          tissue corresponding to a <br />                                                                                                          mutant Qip loss-of-function <br />                                                                                                          cross     (RMNCR113       3 <br />                                                                                                          RMNCR114; Table 1) was <br />                                                                                                          examined. In 1A, 1B, 1C, <br /> and 1D and 2A, 2B 9A, 9B, GLE-1&#8211;sGFP signal is green. Whereas in 1A, 1B, 1C, and 1D and 2A, 2C 9A, 9C, DAPI signal is red. <br /> 1A and 1C show representative maternal tissue from days 3 and 5 postfertilization, respectively. 1B and 1D show a 103 enlargement <br /> of one of the nuclei present in 1A and 1C, respectively. 2A to 9A show merge images of developing croziers as they go through de- <br /> velopment. 2B to 9B show GLE-1&#8211;sGFP localization. 2C 9C show DAPI signal. 2D 9D show light images. All tissues were harvested at <br /> 3 and 4 days postfertilization, unless otherwise noted, and processed as described in materials and methods. Pictures were taken at <br /> 6003. The bar on 1A equals 20 mm. <br />  <br />  <br /> in Neurospora is complex and can be divided into three               tissue (Figure 3A, stage I, and Figure 3B, 1A and 1B), to <br /> stages (Figure 3A). Following fertilization early sexual             the large nuclei observed in ascogenous tissue (Figure 3A, <br /> development is difficult to follow. This is due to the               stages I to III, and Figure 3B[2A 8A]). The nuclei reach <br /> syncytial nature of both the maternal and ascogenous                 their maximal size at the pachytene stage of meiosis I <br /> tissue involved and to their colocalization inside the               (Figure 4[1] ). Following meiosis I and II (Figure 4[2&#8211; <br /> incipient perithecium. To distinguish between these                  9]), nuclei regain &#8216;&#8216;normal&#8217;&#8217; GLE-1 signal after mitosis I <br /> tissues and their corresponding nuclei, we constructed               (Figure 4[10]). In contrast to the wild-type condition, <br /> strains carrying canonical fusions of the glycine&#8211;leucine&#8211;           crosses homozygous for strains carrying GLE-1&#8211;sGFP in <br /> phenylalanine&#8211;glycine lethal-1 (gle-11, NCU01198) nucleo-            a qipD mutant background (Table 1) were severely <br /> pore complex component gene (Murphy and Wente                        developmentally compromised. In no circumstances <br /> 1996) and sgfp1. GLE-1 is predicted to be localized to the           did we observe the formation of large asci or ascospores. <br /> nuclear membrane and this was confirmed in all tissues               Although the formation of maternal tissue appeared to <br /> examined (Figures 3 and 4). Note that during sexual                  be normal (Figure 3C[1]), the development of ascoge- <br /> development the size of the nucleus undergoes dramatic               nous tissue was compromised at an early stage (Figure <br /> changes; from the small nuclei observed in maternal                  3C[2&#8211;9]). A hallmark of ascogenous tissue is the <br />  132                                          D. W. Lee, R. Millimaki and R. Aramayo <br />  <br />                                                                   an efficient repeat-inactivating mechanism. This redun- <br />                                                                   dancy suggests that both quelling and meiotic silencing <br />                                                                   might have biological functions that go beyond simple <br />                                                                   repeat recognition and silencing. In evolutionary terms, <br />                                                                   it is safe to think that meiosis is a &#8216;&#8216;late&#8217;&#8217; evolutionary <br />                                                                   invention (i.e., meiosis vs. mitosis). As such, it is likely <br />                                                                   that the meiotic silencing machinery was built upon <br />                                                                   an already existing RNA silencing apparatus. While the <br />                                                                   exact evolutionary <span id='am-37' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-38' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> of emergence between the <br />                                                                   primitive RNA silencing machineries and RIP is not <br />                                                                   known, the evolutionary pressures for function imposed <br />                                                                   by RIP must have selected against the presence of pro- <br />                                                                   teins specific for each pathway while favoring sharing of <br />                                                                   proteins between both pathways. Our finding of an ad- <br />                                                                   ditional common component reinforces this hypothesis. <br />  <br />   Figure 4.&#8212;Localization of GLE-1&#8211;sGFP in normal meiosis.      <br /> The cellular localization of GLE-1&#8211;sGFP was followed during          <br /> meiosis in a cross between strains (DLNCR553 3 DLNCR554;          <br /> Table 1), representing cells in (1) pachytene, (2) metaphase,          <br /> (3) telophase, (4) interphase, (5) prophase II, (6) anaphase II       <br /> through telophase II, (7) early interphase II, (8) late inter-        <br /> phase II, (9) metaphase IV, and (10) postmitosis II. Stages        <br /> were assigned following Raju (1980). All stages show the           <br /> merge image of GLE-1&#8211;sGFP (green) and DAPI (red) signals.             <br /> Tissues were harvested at 5 and 7 days postfertilization and           <br /> processed as described in materials and methods. The             <br /> bar in l 1 equals 20 mm. Pictures were taken at 6003.         <br /> formation of the crozier, a three-celled apparatus con-          <br /> sisting of a basal, a tip, and a penultimate cell (see Figure         <br /> 3A for details). Our observations suggest that while                 <br /> being formed, crozier development seems to be com-             <br /> promised in qip loss-of-function mutants. In summary,             <br /> our results show that QIP is essential for early sexual           <br /> development and suggest that it is also required for           <br /> meiotic silencing.                                            <br />    A substantial component of all eukaryotic genomes                  <br /> studied to date is formed of repeated sequences (Hsieh             <br /> and Fire 2000). The genome of N. crassa is intriguing                <br /> in this regard because it has mechanisms, like quelling               <br /> and meiotic silencing, that arguably have evolved to              <br /> cope with, and to respond to, the presence of genomic-            <br /> repeated elements. This is despite the presence of RIP,               <br />  <br />  <br />  <br /> QIP, a Component of the Vegetative RNA Silencing Pathway, Is Essential <br />    for Meiosis and Suppresses Meiotic Silencing in Neurospora crassa <br />  <br />  <br />  <br />              Dong Whan Lee, Ryan Millimaki and Rodolfo Aramayo <br />  <br />  <br />  <br />  <br />             Copyright &#211; 2010 by the Genetics Society of America <br />                     DOI: 10.1534/genetics.110.118422 <br />  2 SI                                                   D. W. Lee et al. <br />  <br />  <br />  <br />  <br />    FIGURE S1.&#8212;Localization of QDE-2-sGFP in Meiosis. Picture shows representative asci photographed at the Pachytene stage <br /> of Meiosis I (Panels A to H). Columns 1 and 2 show the GLE-1-sGFP (green), and DAPI (red) signals, respectively. Column 3 <br /> shows the corresponding merged images of Columns 1 and 2. Perithecia were collected, fixed, washed and observed as described <br /> in Materials and Methods. The bar present on Panel 1A equals 20 &#956;. <br />   <br /> </body></html>