<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The <span id='am-65' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-66'  ></span><span id='am-67' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>TOR</span> (target of rapamycin) signal transduction pathway <br /> regulates the stability of translation initiation factor eIF4G <br /> in the yeast <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-10'  ></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br />      (cell cycleystarvationyprotein degradation) <br />  <br /> ABSTRACT           Initiation factor eIF4G is an essential pro-                     identified in mammalian cells (for a review, see ref. 13). <br /> tein required for initiation of mRNA translation via the 5*                         Inhibition of signaling through <span id='am-30' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-31'  ></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> pathway blocks the <br /> cap-dependent pathway. It interacts with eIF4E (the mRNA 5*                         p70(S6)yp85(S6) kinase, inhibits the phosphorylation of ribo- <br /> cap-binding protein) and serves as an anchor for the assembly                       somal protein S6 and of eIF4E binding proteins (4E-BPs), and <br /> of further initiation factors. With treatment of <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-21'  ></span><span id='am-22' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span>                      promotes the association of 4E-BPs with eIF4E (14&#8211;18). <br /> cerevisiae with rapamycin or with entry of cells into the diauxic                   Dephosphorylated 4E-BPs inhibit the interaction of eIF4E <br /> phase, eIF4G is rapidly degraded, whereas initiation factors                        with eIF4G through competition for the same binding site on <br /> eIF4E and eIF4A remain stable. We propose that nutritional                          eIF4E (19, 20). Blocking of the signaling pathway with <span id='am-25' about='obo:NCBITaxon_644105' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_57611 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_39475 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28'  ></span><span id='am-29' property="rdfs:label" content="Rapa" datatype="xsd:string"></span>rapa</span>- <br /> deprivation or interruption of the TOR signal transduction                          mycin affects mostly translation of mRNAs containing a <br /> pathway induces eIF4G degradation.                                                  polypyrimidine tract in their 59 untranslated region; among <br />                                                                                     those, mRNAs encoding ribosomal proteins, translation elon- <br /> The availability of nutrients plays a central role in growth                        gation factors (14, 21), and proteins involved in growth control <br /> regulation of the yeast Saccharomyces cerevisiae. Starvation of                     (22). <br /> cells for carbon or nitrogen, a common condition in natural                            eIF4G, which in Saccharomyces cerevisiae is encoded by the <br /> habitats (1), leads to lowering of transcription and translation of                 genes TIF4631 and TIF4632 (23), is an essential initiation factor <br /> most genes and cessation of growth in the G1 phase of the cell                      required for mRNA translation via the 59 cap structure (24). <br /> cycle (2). As an exception, expression of genes that allow entry                    Mammalian eIF4G serves as an anchor for the binding of further <br /> into the stationary or G0 phase is enhanced (for reviews, see refs.                 initiation factors such as eIF3 and eIF4A (25) and the 40S <br /> 1 and 3). Entry into the G0 phase in response to starvation is                      ribosomal subunit (for a review, see ref. 26). Here, we report that <br /> accompanied by and depends on reduced expression of the G1                          initiation factor eIF4G is rapidly degraded in S. cerevisiae on entry <br /> phase cyclins. Cells that express stable cyclin2 (Cln2-1p) enter S                  in the diauxic growth phase (transition from fermentative to <br /> phase despite nutritional deprivation (4, 5) and lose viability. G0                 oxidative metabolism) or on treatment with rapamycin. <br /> phase cells are able to survive for long periods of time under <br />                                                                                                     MATERIALS AND METHODS <br /> conditions of nutritional deprivation and to resume growth when <br /> nutrients become available again (1, 3).                                               Yeast Strains. Strain CBY12 was obtained by crossing a <br />    A response similar to starvation is triggered by treatment of                    spontaneous ura32 mutant of strain CBY2.1 (transformed with <br /> yeast cells with the immunosuppressant rapamycin (6, 7).                            pCB2) with strain CBY1.2. Haploid spores carrying the desired <br /> Rapamycin interacts with the FK506-binding protein (FKBP;                           auxotrophic markers were selected by replica-plating on minimal <br /> a proline rotamase) (6), and the resulting complex binds to and                     medium plates (0.67% yeast nitrogen base) containing 2% ga- <br /> inhibits the proteins Tor1p and Tor2p (6, 8), two phosphati-                        lactose. The plasmid pCB2 contains the entire TIF4631 cDNA <br /> dylinositol kinase homologs. This results in G1 phase arrest,                       (EcoRI fragment) (23) in the vector p301-URA3 under the <br /> inhibition of translation initiation, and acquirement of a G0                       GAL1y10 promoter. For further details on the characteristics of <br /> phase-like phenotype reminiscent of nutrient-starved cells (7).                     yeast strains and relevant <span id='am-75' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span> used, see Table 1. <br /> G1 arrest in rapamycin-treated cells is probably caused by                             Growth Conditions. Unless indicated otherwise, cells were <br /> reduced CLN3 (cyclin3) mRNA translation. Cln3p induces the                          grown in YPD (1% yeast extract, 2% peptone, 2% glucose) or <br /> expression of other G1 phase cyclins that promote G1 to S                           YPGal (1% yeast extract, 2% peptone, 2% galactose) at 30&#176;C. <br /> phase transition (for reviews, see refs. 9 and 10). A gene                             Manipulation of Yeast Cells. Yeast cells were transformed <br /> construct carrying the promoter and the 59 untranslated region                      by using the lithium acetate method (27). <br /> of UBI4 (the polyubiquitin-encoding gene) fused to the CLN3                            DNA Manipulations. DNA manipulations were carried out <br /> ORF renders translation of CLN3 mRNA less dependent on                              according to Sambrook et al. (28) by using <span id='am-35' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-49'  ></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span>Escherichia coli</span> <br /> initiation factor eIF4E (eukaryotic initiation factor 4E),                          strain XL2B (Stratagene) for subclonings and maintenance of <br /> thereby suppressing the rapamycin-induced G1 phase arrest                           plasmid DNA. <br /> and conferring starvation sensitivity to the cells (7). This result                    Drug Treatment. Final concentrations of drugs added to <br /> indicates that Tor1p and Tor2p control the activity of eIF4E                        exponentially growing cell cultures were as follows: 0.2 mgyml <br /> or eIF4E-associated protein(s).                                                     rapamycin (dissolved in vehicle: 90% ethanoly10% Tween-20), <br />    A similar rapamycin-sensitive transduction pathway signal-                       10 mgyml cycloheximide (Sigma), 2 mM a-factor (Sigma), 0.1 <br /> ing through a Tor-like protein (mTORyFRAPyRAFT1) (11,                               M hydroxyurea (Fluka), 25 mgyml nocodazole (Fluka; dis- <br /> 12) and regulating the activity of translation factors has been                     solved in dimethyl sulfoxide). Samples (10&#8211;20 ml cell cultures) <br />                                                                                     were harvested at different time points after addition of drugs, <br /> The publication costs of this article were defrayed <span id='am-85' about='ncbitaxon:in_part' typeof='owl:Thing'>in part</span> by page charge <br />                                                                                     This paper was submitted directly (Track II) to the Proceedings office. <br /> payment. This article must therefore be hereby marked &#8216;&#8216;advertisement&#8217;&#8217; in <br />                                                                                     Abbreviations: eIF, eukaryotic initiation factor; GST, glutathione <br /> accordance with 18 U.S.C. &#167;1734 solely to indicate this fact. <br />                                                                                     S-transferase; BP, binding protein. <br /> &#169; 1998 by The National Academy of Sciences 0027-8424y98y954264-6$2.00y0             *To whom reprint requests should be addressed. e-mail: michael. <br /> PNAS is available online at http:yywww.pnas.org.                                     altmann@mci.unibe.ch. <br />  <br />                                                                              4264 <br />            Biochemistry: Berset et al.                                                      Proc. Natl. Acad. Sci. USA 95 (1998)            4265 <br />  <br /> and cells were collected by centrifugation at 5,000 3 g for 3            baking of the membrane for 1 h at 80&#176;C. Prehybridization of the <br /> min, washed once, and immediately processed for protein                  membranes was done for 1 h at 42&#176;C in 50% formamide, 53 SSC <br /> analysis (or frozen at 280&#176;C for later processing).                      (750 mM sodium chloridey75 mM sodium citrate, pH 7.5), 53 <br />    SDSyPAGE and Western Blot Analysis. Cell pellets were                 Denhardt&#8217;s reagent [0.5 g of Ficoll (type 400, Pharmacia)y0.5 g <br /> resuspended in 50&#8211;100 ml of buffer A (20 mM TriszHCl, pH                 polyvinylpyrrolidoney0.5 g of BSA (fraction V; Sigma)y0.5% <br /> 7.5y100 mM KCly2 mM MgCl2y0.1 mM EDTAy7 mM 2-mer-                        SDS]. The blots were probed with 32P labeled TIF4631 cDNA <br /> captoethanol) and mixed with 0.2&#8211;0.4 g glass beads rinsed with           (23) by incubating overnight at 42&#176;C. They were extensively <br /> buffer A. Cells were lysed by vortex mixing eight times for 20 s.        washed with 53 SSCy0.1% SDS, 23 SSCy0.1% SDS, and 0.53 <br /> After centrifugation at 12,000 3 g for 5&#8211;10 min, the ribosome            SSCy0.1% SDS at room temperature, and 0.23 SSCy0.1% SDS <br /> content of the supernatants was determined by measuring the              at 42&#176;C. They were then exposed to x-ray films. <br /> absorption at 260 nm. Unless indicated otherwise, equal <br /> amounts of proteins were loaded on 12.5% SDSyPAGE and                                                RESULTS <br /> blotted onto nitrocellulose for 45 min at 60 V in a Mini Trans              Rapamycin Treatment Causes the Disappearance of eIF4G. <br /> Blot Cell (Bio-Rad). Blots were saturated with 2.5% BSA in               Treatment of S. cerevisiae with the immunosuppressant rapamy- <br /> TBS (10 mM TriszHCl, pH 7.5y150 mM NaCl) for 5 min at                    cin induces inhibition of translation at the level of initiation, arrest <br /> room temperature and incubated overnight with rat polyclonal             of growth in the G1 phase of the cell cycle, and expression of G0 <br /> antibodies (1:1,000 dilutions in TBS containing 0.5% BSA).               phase-specific genes (6, 7). We asked whether factors involved in <br /> After washing with TBS blots were decorated for 1 h with                 59 cap-dependent translation such as eIF4E and eIF4G were <br /> peroxidase-conjugated rabbit anti-rat Igs (Dako) and stained             affected by rapamycin treatment. Exponentially growing wild- <br /> with 0.018% chloronaphthol and 0.006% H2O2 in TBS. Equal                 type cells (strain CWO4; Table 1) were treated either with <br /> protein loading was verified by Coomassie blue staining.                 rapamycin or drug vehicle as a control. Aliquots of treated <br /> Amounts corresponding to 0.8 A260 were loaded for the                    cultures were harvested at hourly intervals and analyzed by <br /> detection of eIF4G1, 1.2 A260 for eIF4G2, 0.2 A260 for eIF4E,            Western blotting (Fig. 1A). Between 1 and 2 h after rapamycin <br /> and 0.08 A260 for eIF4A.                                                 addition, eIF4G levels were reduced and were hardly detectable <br />    Flow Cytometry. Cell culture samples (300 ml culture) were            after 3 h of drug treatment (Fig. 1 A, first row, lanes 4&#8211;6). In some <br /> kept in 70% ethanol at 4&#176;C overnight or longer. Washed cells             experiments, we detected the degradation of most eIF4G only <br /> (dH2O) were resuspended in 500 ml of 50 mM sodium citrate (pH            after 3 h of rapamycin treatment (Fig. 1B). In contrast, initiation <br /> 7.4) containing 0.25 mgyml RNase A, sonicated for 3 min, and             factors eIF4A (Fig. 1 A, second row) and eIF4E (Fig. 1 A, third <br /> incubated for 1 h at 37&#176;C. DNA was stained by addition of 500 ml         row) as well as eIF4G from cells incubated with vehicle (Fig. 1 A, <br /> of citrate buffer containing 16 mgyml propidium iodide. For each         first row, lanes 1&#8211;3) were not affected by the treatment. The <br /> time point, 10,000 events were analyzed for DNA content by using         antibody used to detect eIF4G [rat polyclonal antibody against a <br /> a Becton Dickinson FACScan machine, and <span id='am-70' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-71' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74'  ></span>data</span> were processed              glutathione S-transferase (GST)-eIF4G1y160&#8211;492 (<span id='am-59' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-64'  ></span>aa</span> 160&#8211;492) <br /> by using CELL QUEST software (Cell Quest, Lincoln Park, NJ).             fusion protein (29)] recognizes eIF4G1 (encoded by TIF4631) <br />    RNA Preparation and Northern Blot Analysis. All solutions             and, with somewhat lower affinity, also eIF4G2 (encoded by <br /> were prepared by using diethylpyrocarbonate (DEPC)-treated               TIF4632). Fig. 1 A Lower shows that eIF4G also disappears in a <br /> ddH2O. Cell pellets (10 ml cultures) were resuspended in 500 ml          strain that only expresses eIF4G2 (strain CBY1.1; Table 1) upon <br /> TEL buffer (10 mM TriszHCl, pH 7.5y2 mM EDTAy150 mM                      treatment of cells with rapamycin. The two lower bands are <br /> LiCl) plus 1% lithium dodecyl sulfate. Cell extracts were prepared       probably degradation products of eIF4G2. They accumulate after <br /> by vortex mixing with 0.5 g glass beads in the presence of 0.5 ml        1 h of rapamycin treatment (lane 2) and then disappear gradually <br /> phenolychloroform (1:1) and centrifuged at 12,000 3 g for 10             (lanes 3 and 4). Identical results were obtained when cells were <br /> min. Phenolic phases were reextracted with 200 ml TEL buffer,            directly boiled in SDS sample buffer. The disappearance of intact <br /> and the aqueous phases were reextracted twice with phenoly               eIF4G was also observed when Western blots were decorated <br /> chloroform (1:1). RNA was precipitated from the combined                 with other antibodies: rat polyclonal anti-eIF4F antibody (30), rat <br /> aqueous phases with 2.5 vol of absolute ethanol in the presence          polyclonal antibody against GST-eIF4G1y542&#8211;883 (aa 542&#8211;883), <br /> of 0.1 M KAc. RNA pellets were washed once with 70% ethanol,             or anti-HA antibody for a strain carrying a construct encoding <br /> air dried, and resuspended in 10 ml (DEPC-treated) ddH2O. The            HA-tagged Tif4632p. We conclude that levels of both eIF4G1 <br /> RNA yield was determined by measuring the optical density at             and eIF4G2 are dramatically reduced after 1&#8211;3 h of rapamycin <br /> 260 nm (1 A260 5 40 mg total RNA). Equal amounts of RNA (15              treatment, at a time when levels of other initiation factors such as <br /> mg) were dissolved in FR buffer (100 mM Mopsy50 mM sodium                eIF4E and eIF4A are almost unchanged. <br /> acetate, pH 7.0y5 mM EDTA), 6% formaldehyde, and 50%                        The Disappearance of eIF4G Is Regulated by the TOR <br /> formamide, incubated for 15 min at 65&#176;C, and loaded directly on          Signaling Pathway. Rapamycin (complexed with FKBP) exerts <br /> a 0.7% agarose gel containing FR buffer and 6% formaldehyde.             its effects on the cell cycle through interaction with Tor1p and <br /> Gel electrophoresis was carried out for 3&#8211;4 h at 30 V. The agarose       Tor2p. Mutation of a serine residue in the lipid-kinase domain <br /> gels were blotted on <span id='am-80' about='obo:NCBITaxon_743421' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_50638 obo:NCBITaxon_7438 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7434 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_34725 obo:NCBITaxon_50557'><span id='am-81' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-82'  ></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="Zeta" datatype="xsd:string"></span>Zeta</span>-probe membrane (Schleicher &amp; Schu    &#168;ll)      of TOR1 or TOR2 abolishes binding of the FKBP&#8211;rapamycin <br /> overnight. Blotted RNA was fixed for 2 min by UV treatment and           complex and confers rapamycin resistance to the cells (31, 32). <br />  <br />                     Table 1.   Relevant yeast strains and plasmids <br />                      Strain                                  Genotype                                      Sourceyref. <br />                     CWO4                   MATa ade2-1 his3-11,15 leu2-3,112 trp1-1 ura3                        45 <br />                     ASZ1                   Isogenic to CWO4, MATa                                               45 <br />                     CBY1.1                 Isogenic to CWO4, MATa tif4631::LEU2                         This study, ref.   23 <br />                     CBY1.2                 Isogenic to CWO4, MATa tif4631::LEU2                         This study, ref.   23 <br />                     CBY2.1                 Isogenic to CWO4, MATa tif4632::URA3                         This study, ref.   23 <br />                     CBY12                  CWO4 tif4631::LEU2 tif4632::ura3 ^pCB2&amp;                      This study, ref.   23 <br />                     PDY4                   cdc63-1 trp1 ura3                                                This study <br />                     Plasmid <br />                       pPW2                 ^tor1-1; URA3&amp;                                                       31 <br />                       pCB2                 ^p301-TIF4631cDNA; URA3&amp;                                     This study, ref. 23 <br />  4266      Biochemistry: Berset et al.                                                         Proc. Natl. Acad. Sci. USA 95 (1998) <br />  <br /> To test whether the disappearance of eIF4G was caused by                    1B, lanes 4 and 5). We conclude from these data that the <br /> blockage of the TOR signal transduction pathway, we trans-                  disappearance of eIF4G in rapamycin-treated cells is caused by <br /> formed wild-type yeast cells with the tor1-1 allele (plasmid                the blockage of the TOR signal transduction pathway. <br /> pPW2; Table 1), which confers partial rapamycin resistance,                    The Disappearance of eIF4G Is Caused by Degradation. To <br /> and analyzed eIF4G levels by Western blotting after treatment               determine whether the disappearance of eIF4G after rapamycin <br /> of the cells with rapamycin (Fig. 1B). In this strain, rapamycin            treatment was caused by enhanced degradation or reduced <br /> treatment did not lead to the disappearance of eIF4G (first                 synthesis of the protein, we attempted to measure its half-life. <br /> row, lanes 1 and 2). The levels of eIF4A (second row) and                   Because several attempts to <span id='am-58' about='rdfs:label' typeof='owl:Thing'>label</span> eIF4G with [35S]methionine <br /> eIF4E (third row) also remained unchanged. Interestingly, the               were not successful, we began to study eIF4G in the presence or <br /> amount of eIF4G was lower in rapamycin-treated cells than in                absence of rapamycin under conditions where de novo protein <br /> vehicle-treated cells (compare lanes 1 and 2 with lane 3). This             synthesis is blocked. Three hours after addition of the translation <br /> may be due to the fact that the tor1-1 allele confers only partial          elongation inhibitor cycloheximide, the eIF4G level was not <br /> resistance to rapamycin (6). In a control experiment, the same              affected (Fig. 2A, lane 2). Simultaneous treatment of yeast cells <br /> yeast strain was transformed with a plasmid carrying the same               with rapamycin and cycloheximide prevented rapamycin- <br /> auxotrophic marker as pPW2 but without tor1-1. In this case                 mediated eIF4G degradation (Fig. 2 A; compare lane 1 with lanes <br /> rapamycin treatment led to the disappearance of eIF4G (Fig.                 3 and 4). When cycloheximide was added 15 min to 1 h (but not <br />                                                                             2 h) after the addition of rapamycin, it inhibited degradation of <br />                                                                             eIF4G (Fig. 2 A, lanes 5&#8211;8). <br />  <br />  <br />  <br />  <br />                                                                                FIG. 2. Stability of eIF4G. (A) Effect of rapamycin treatment on <br />                                                                             eIF4G (strain CWO4) when de novo protein synthesis is blocked with <br />                                                                             cycloheximide. Rapamycin was added to exponentially growing cells at <br />                                                                             time point 0 h, except for lane 2 (no rapamycin added) and lane 3 where <br />                                                                             it was added 15 min after cycloheximide addition. The time points of <br />                                                                             cycloheximide addition are indicated. The total duration of incubation in <br />                                                                             the presence of the drugs was 3 h. Shown are Western blots from extracts <br />    FIG. 1. Rapamycin causes the disappearance of eIF4G. (A) Ex-             decorated for eIF4G. (B) Effect of rapamycin (lanes 1&#8211;3) or vehicle (lanes <br /> ponentially growing cells from strains CWO4 (Upper) or CBY1.1               4&#8211;6) on eIF4G stability in the mutant strain PDY4 (cdc63-1) grown at <br /> (Lower) were treated with rapamycin or vehicle. Aliquots (20 ml) were       25&#176;C (first row) and after shift from 25&#176;C to 37&#176;C (second row). Time <br /> harvested at the times indicated after addition of the drug (or vehicle).   points after addition of the drug (or vehicle) are indicated. Shown are <br /> Cell extracts were fractionated by SDSyPAGE, blotted onto nitrocel-         Western blots decorated for eIF4G. (C) Exponentially growing cells in <br /> lulose, and decorated with rat polyclonal antibodies against eIF4G          YPGal (strain CBY12) were shifted to fresh medium containing YPGal <br /> (Upper and Lower, first row), eIF4A (Upper and Lower, second row),          (lanes 1 and 2) or YPD (lanes 3&#8211;8) and incubated with rapamycin or <br /> and eIF4E (Upper, third row). (B) Cell extracts from strain CWO4            vehicle for the times indicated. First row shows Western blot decorated <br /> transformed with the plasmid pPW2 carrying the dominant tor1-1              for eIF4G. Equal volumes of cell cultures were loaded. Second row shows <br /> allele (lanes 1&#8211;3) or a vector without tor1-1 (lanes 4&#8211;6) were treated      Northern blot of total RNA isolated from cells harvested at the time <br /> as described in A with rapamycin (lanes 1, 2, 4 and 5) or vehicle (lanes    points indicated and probed with a cDNA corresponding to the whole <br /> 3 and 6) and analyzed on Western blots for eIF4G (first row), eIF4A         TIF4631 ORF (23). Third row shows ethidium bromide staining of 28S <br /> (second row), or eIF4E content (third row).                                 and 18S rRNAs used for the Northern blot analysis. <br />            Biochemistry: Berset et al.                                                   Proc. Natl. Acad. Sci. USA 95 (1998)              4267 <br />  <br />    In a further experiment protein synthesis was blocked by <br /> incubating PDY4 cells carrying the temperature sensitive cdc63-1 <br /> allele [CDC63 encodes Prt1p, a subunit of translation initiation <br /> factor eIF3 (33&#8211;35)] at the nonpermissive temperature (37&#176;C). <br /> After 3 h at 37&#176;C, eIF4G was still present (Fig. 2B, second row, <br /> lanes 4&#8211;6). Although rapamycin (Fig. 2B, first row, lanes 1&#8211;3) but <br /> not vehicle (Fig. 2B, first row, lanes 4&#8211;6) induced eIF4G degra- <br /> dation at 25&#176;C, incubation of cdc63-1 cells at 37&#176;C led to stabi- <br /> lization of eIF4G in the presence of rapamycin (Fig. 2B, second <br /> row, lanes 1&#8211;3). Taken together the data in Fig. 2 A and B suggest <br /> that eIF4G is a stable protein and that the rapamycin-mediated <br /> destabilization requires ongoing protein synthesis. <br />    To verify that eIF4G is an intrinsically stable protein we <br /> constructed a conditionally lethal strain in which eIF4G is ex- <br /> pressed from TIF4631 (as its only eIF4G source) under the <br /> control of the galactose-regulatable GAL1y10 promoter (strain <br /> CBY12; Table 1). Expression of eIF4G is shut off when cells are <br /> transferred from galactose- to glucose-containing medium. <br /> Through dilution with fresh medium the cells were kept at low <br /> density. Cells at different time points after transfer to fresh <br /> glucose-containing medium (or to galactose-containing medium <br /> as a control) and addition of rapamycin or vehicle were analyzed <br /> for eIF4G protein and eIF4G mRNA content (Fig. 2C). Already <br /> 5 min after shift to glucose, hardly any eIF4G1 mRNA could be <br /> detected (Fig. 2C, second row, lanes 3&#8211;5). Despite the very quick <br /> loss of eIF4G mRNA, eIF4G1 protein levels did not decrease <br /> significantly (Fig. 2C, first row, lanes 3&#8211;5). For the Western blot, <br /> equal volumes of cell cultures were loaded (instead of equal A260 <br /> units) to compensate for residual growth of the cells during the <br /> first hours after the shift to glucose-containing medium despite <br /> the shut off of TIF4631 gene expression. Rapamycin treatment of <br /> cells maintained in galactose had little effect on eIF4G1 mRNA <br /> decay (Fig. 2C, second row, lane 2) but led to eIF4G protein <br /> degradation (Fig. 2C, first row, lane 2). In the cells shifted to <br /> glucose-containing medium and additionally treated with rapa- <br /> mycin, eIF4G mRNA faded as expected (Fig. 2C, second row, <br /> lanes 6&#8211;8) and the protein levels decreased rapidly (Fig. 2C, first <br /> row, lanes 6&#8211;8). It is somewhat difficult to estimate the half-life <br /> of eIF4G from these experiments. Nevertheless, in absence of <br /> rapamycin, the protein level was only slightly reduced 3 h after the <br /> disappearance of eIF4G mRNA (Fig. 2C, first and second rows, <br /> lane 5). In contrast, more than 50% of the protein was degraded          FIG. 3. eIF4G degradation is induced upon entry into the diauxic <br /> after 1 h of rapamycin treatment (Fig. 2C, lane 7), suggesting a       phase. (A) Strain CWO4 was allowed to grow to the diauxic phase <br /> half-life shorter than 1 h. These results show that in exponentially   (points 2 and 3), diluted into fresh medium, and allowed to resume <br /> growing cells eIF4G is a stable protein and strongly suggest that      exponential growth (points 4&#8211;6). Strain CBY1.1 (TIF4631 deletion) <br /> the disappearance of eIF4G is due to its accelerated degradation       was allowed to grow to the diauxic phase (points c and d). Time points <br /> upon rapamycin treatment.                                              indicated (1&#8211;6) correspond to CWO4 cells harvested at a cell density <br />    Degradation of eIF4G at the Diauxic Shift. Because rapamycin        (OD600) of 1.302 (time point 1), 4.66 (time point 2), 6.6 (time point 3), <br />                                                                        0.178 (time point 4), 0.445 (time point 5), and 1.10 (time point 6) and <br /> treatment of yeast cells simulates nutritional deprivation (7) we      to CBY1.1 cells with an OD600 of 1.81 (time point a), 3.29 (time point <br /> looked for eIF4G degradation under starvation conditions. Fig.         b), 6.19 (time point c), and 6.88 (time point d). (B) Flow cytometry <br /> 3A depicts growth curves of wild-type cells (strain CWO4, time         analysis (graphs) of CWO4 cells harvested at time points 1&#8211;6 in A. (C) <br /> points 1&#8211;6) and cells deleted for TIF4631 (strain CBY1.1, time         Western blots of extracts from CWO4 cells harvested at time points <br /> points a&#8211;d) grown in YPD to high density (a stage at which most        1&#8211;6 (lanes 1&#8211;6) and CBY1.1 (lanes a&#8211;d) and decorated with antibodies <br /> of the glucose is metabolized) and then diluted back into fresh        against eIF4G, eIF4A, and eIF4E. <br /> medium to replenish glucose. When cells reached high density <br /> and slowed down growth in the diauxic phase (Fig. 3A, time points      with anti-eIF4F or anti-eIF4G1y542&#8211;883 antibody. These data <br /> 2 and 3, and c and d) most of them had a 1n DNA content (1n            show that exponentially growing cells contain eIF4G1 and <br /> referring to the DNA content of a haploid cell before replication)     eIF4G2 and that both are degraded upon entry into the diauxic <br /> (Fig. 3B, graphs 2 and 3) and lost eIF4G (Fig. 3C, Upper, lane 3)      phase. <br /> and eIFG2 (Fig. 3C, Lower, lanes c and d). Under these conditions        Rapamycin-Induced eIF4G Degradation Is Not Limited to <br /> translation initiation factors eIF4A and eIF4E still persisted (Fig.   the G1 Phase of the Cell Cycle. To test whether the degradation <br /> 3C, Upper, second and third rows). When cells were kept for            of eIF4G in response to rapamycin treatment was dependent <br /> longer times at high density (e.g., 12 h), eIF4A- and eIF4E-levels     on a specific stage of the cell cycle, we arrested cells in different <br /> also declined [in contrast to the ribosome content which remained      phases and treated them with rapamycin. When cells were <br /> constant (results not shown)]. When cells were diluted back into       arrested in the G1 phase by treatment with a-factor, eIF4G was <br /> fresh medium they accumulated eIF4G again (Fig. 3C, Upper,             degraded in the presence of rapamycin (Fig. 4A, lane 2). The <br /> lanes 5 and 6), entered S phase, as measured by the appearance         same was observed when cells were arrested in the S phase with <br /> of cells with a 2n DNA content (Fig. 3B, graphs 5 and 6), and          hydroxyurea, although eIF4G degradation seems to be some- <br /> resumed exponential growth (Fig. 3A, points 5 and 6). Identical        what less efficient (Fig. 4A, lane 4). To test for degradation of <br /> results were obtained when Western blots were decorated either         eIF4G in the G2yM (mitosis) phase we arrested cells with the <br />  4268       Biochemistry: Berset et al.                                                         Proc. Natl. Acad. Sci. USA 95 (1998) <br />  <br />                                                                                and 120 min after release in the presence of rapamycin. At this <br />                                                                                stage most cells have a 2n DNA content confirming that eIF4G <br />                                                                                degradation also happens at the G2yM stage. We conclude <br />                                                                                from these data that rapamycin-induced eIF4G degradation <br />                                                                                can happen at different stages of the cell cycle. <br />  <br />                                                                                                         DISCUSSION <br />                                                                                The data presented in this paper show that eIF4G is a stable <br />                                                                                protein under conditions of exponential growth and becomes <br />                                                                                unstable with a considerably shorter half-life upon rapamycin <br />                                                                                treatment or during diauxic phase. The degradation pathway for <br />                                                                                eIF4G is not known. We show that it is initiated by inhibition of <br />                                                                                the Tor kinases. The components of the TOR signal transduction <br />                                                                                pathway upstream and downstream of Tor are unknown in yeast. <br />                                                                                Because rapamycin induces a response similar to starvation (7), <br />                                                                                we assume that degradation of eIF4G is mediated by inhibition <br />                                                                                of the same transduction pathway. The results presented in Fig. <br />                                                                                2 demonstrate that ongoing translation is necessary for rapamy- <br />                                                                                cin-induced eIF4G degradation. Because both rapamycin and <br />                                                                                cycloheximide block protein synthesis, it may appear surprising <br />                                                                                that rapamycin can cause eIF4G degradation. This can be ex- <br />                                                                                plained by the fact that inhibition of translation by cycloheximide <br />                                                                                is much faster than by rapamycin (7). In yeast, cellular proteins are <br />                                                                                degraded either in the vacuole or by the proteasome. Preliminary <br />                                                                                experiments with phenylmethylsulfonyl fluoride (an inhibitor of <br />                                                                                vacuolar proteases) or with proteasome mutants suggest that <br />                                                                                eIF4G degradation happens in the vacuole (Diana Dominguez, <br />                                                                                personal communication). Ongoing translation may be required <br />                                                                                for the synthesis of a receptor protein that allows import of eIF4G <br />                                                                                into the vacuole as it has been proposed for fructose 1,6- <br />                                                                                bisphosphatase (36). We are currently investigating which deter- <br />                                                                                minants on eIF4G target the protein for degradation. Sequence <br />                                                                                analysis revealed the presence of PEST sequences [proline (P), <br />                                                                                glutamate (E), serine (S), and threonine (T) rich] in eIF4G (23, <br />    FIG. 4. eIF4G degradation is not confined to a specific stage of the cell <br />                                                                                26) which could act as signals for proteolysis. <br /> cycle. (A Upper) Exponentially growing cells (strain ASZ1) were arrested          We speculated that a check point in G1 would control eIF4G <br /> at an OD600 of 0.2&#8211;0.3yml by treating them either with a-factor for 2 h        degradation and therefore that rapamycin could exert its effect <br /> (lanes 1 and 2), with hydroxyurea for 2 h (lanes 3 and 4), or with             only during a specific stage of the cell cycle. The results <br /> nocodazole for 1 h (lanes 5 and 6). Vehicle (lanes 1, 3, and 5) or rapamycin   presented in Fig. 4 show that this is not the case. It seems that <br /> (lanes 2, 4, and 6) was added to the arrested cultures and incubation was      the machinery used for the degradation of eIF4G by rapamycin <br /> continued for 2 h (except nocodazole-treated cells for 1.5 h). Cell extracts   is present during all phases of the cell cycle. However, the <br /> were analyzed on Western blots for eIF4G content (lanes 1&#8211;6). (A               efficiency of rapamycin-induced eIF4G degradation may vary <br /> Lower)The graphs show the flow cytometry analysis of cells arrested with       somewhat with the different phases of the cell cycle. <br /> the different drugs. (B) Exponentially growing ASZ1 cells were arrested <br /> with a-factor (2 h treatment) and subsequently released by incubating <br />                                                                                   Is degradation of eIF4G the primary cause or a secondary <br /> with fresh YPD in the absence (Upper) or in the presence of rapamycin          effect of the translational down-regulation caused by starvation? <br /> (Lower). At the time points indicated (0, 30, 60, 90, and 120 min) cell        Dissociation of eIF4G from eIF4E might precede degradation of <br /> cultures (10 ml) were harvested, aliquots were prepared for flow cytom-        eIF4G. In mammalian cells, rapamycin leads to the dephosphor- <br /> etry analysis, and cell extracts were analyzed for eIF4G content by            ylation of 4E-BPs and the enhancement of their affinity for eIF4E <br /> Western blotting and immunodecoration.                                         (37, 38). Unfortunately, it was not tested and we therefore do not <br />                                                                                know whether this leads to the dissociation of eIF4E from eIF4G. <br /> drug nocodazole and subsequently treated them with rapamy-                     In temperature-sensitive yeast strains exhibiting a weakened <br /> cin. Because cells could not be arrested for longer periods in                 eIF4E&#8211;eIF4G interaction caused by mutations in the eIF4E gene, <br /> the G2yM phase of the cell cycle with nocodazole, they were                    we observed degradation of eIF4G caused by shift to the non- <br /> treated for only 1.5 h with rapamycin. We observe partial                      permissive temperature (results not shown). This result indicates <br /> eIF4G degradation under these conditions (Fig. 4A, lane 6).                    that free eIF4G is more susceptible to degradation than eIF4G in <br /> Because of this limitation, we do not know whether degrada-                    a complex with eIF4E. This may also be true for eIF4G in <br /> tion of eIF4G in the G2yM phase is as efficient as in other                    mammalian cells where overexpression of eIF4E antisense RNA <br /> phases of the cell cycle. The arrest of the cells in the G1, S                 caused reduced eIF4E expression and consequently a decrease in <br /> phase, or in the G2yM phase was verified by flow cytometry                     the cellular levels of eIF4G (39). In S. cerevisiae p20, an eIF4E-BP <br /> depicting a 1n peak for a-factor-arrested cells (Fig. 4A, Lower                (40, 41) has been shown to compete with eIF4G for binding to <br /> Left), a broad peak of fluorescence between 1n and 2n for S                    eIF4E (29). However, we have no evidence that rapamycin <br /> phase-arrested cells (Fig. 4A, Lower Center) and predomi-                      induces dissociation of eIF4G from eIF4E in yeast cells. It <br /> nantly 2n cells for G2yM-phase-arrested cells (Fig. 4A, Lower                  remains to be elucidated whether p20 is modified after rapamycin <br /> Right). We extended this analysis by releasing cells synchro-                  treatment of yeast cells and whether it is involved in the disso- <br /> nized with a-factor from the block in the presence or absence                  ciation of the eIF4E&#8211;eIF4G complex in vivo. Preliminary results <br /> of rapamycin. As presented in Fig. 4B, most cells had dupli-                   indicate that a mutant which does not express p20 (29) shows very <br /> cated their DNA content after 60 min after release from G1                     similar kinetics of eIF4G degradation like wild-type cells. <br /> arrest, both in the absence and presence of rapamycin. Al-                        At present, we can only speculate on the physiological role of <br /> though eIF4G remained intact during progression of the cell                    eIF4G degradation upon glucose starvation of yeast cells in the <br /> cycle in the absence of rapamycin, it was degraded between 60                  diauxic phase. eIF4G is most likely the limiting factor in cap- <br />  <br /> dependent translation in yeast, as we detected only &apos;0.1 mol                <br /> eIF4G per mol ribosomes, whereas the concentration of other                         <br /> initiation factors such as eIF4E and eIF4A is at least in the range            <br /> of 1 mol factor per mol ribosomes (unpublished data). Degra-                  <br />                                                                                     <br /> dation of eIF4G may serve to down-regulate translation and                      <br /> arrest growth in the G1 phase of the cell cycle. Indeed, strain                    <br /> CBY12 (whose only eIF4G gene copy is under the control of the                       <br /> galactose-regulatable GAL1y10 promoter; Table 1) continued               <br /> growing and dividing normally for approximately four genera-                    <br /> tions after transfer from galactose- to glucose-containing medium            <br /> until most eIF4G significantly decreased because of factor dilu-                    <br /> tion and turnover. The cells arrested in the G1 phase (even at low                   <br />                                                                               <br /> cell density) indicating that eIF4G activity is required for G1 to                 <br /> S phase transition and that its down-regulation causes G1 arrest                   <br /> (results not shown). In addition, degradation of this factor could           <br /> be required for proper entry of the cells into the stationary phase.                <br /> It would be interesting to check whether yeast cells containing a        <br /> degradation-resistant form of eIF4G are unable to correctly enter                  <br /> the G0 phase and would consequently lose viability upon starva-                <br /> tion.                                                                                <br />                                                                                  <br />    An intriguing question is how cells synthesize eIF4G when                  <br /> the conditions are favorable to resume growth after a period                       <br /> of starvation. Cap-dependent translation requires intact                            <br /> eIF4G and eIF4E&#8211;eIF4G interaction (29, 42). In contrast,                        <br /> internal initiation (42, 43), translation of uncapped mRNAs                   <br /> (42), and translation of certain capped mRNAs (7, 29) do not.                       <br /> It is possible that translation of eIF4G mRNA itself does not                        <br /> need intact eIF4G. However, it could require a fragment of                     <br />                                                                              <br /> eIF4G. In mammalian cells, the translation of eIF4G mRNA                      <br /> is initiated internally (44) and is therefore expected to depend             <br /> only on a C-terminal fragment of eIF4G. In yeast, we have not                    <br /> yet been able to detect a specific fragment of eIF4G that would               <br /> persist after degradation of the protein. It is also possible that           <br /> residual levels of eIF4G (undetected in our assays) are pro-                    <br /> tected from degradation and initiate eIF4G mRNA translation                      <br /> when nutritional conditions are benign.                                        <br />                                                                            <br />   <br /> </body></html>