<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Mlc1p Is Light Chain Unconventional Myosin <br /> Myo2p <span id='am-11' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-24'  ></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br /> Richard C. Stevens Trisha N. Davis <br /> Department Biochemistry, University Washington, Seattle, Washington 98195-7350 <br />  <br />  <br />  <br />  <br /> Abstract. In <span id='am-28' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33'  ></span><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae, unconven-                         haploinsufficiency. Reduced levels MYO2 overcome <br /> tional myosin Myo2p fundamental importance                          haploinsufficiency MLC1. The mutant MYO2- <br /> polarized growth. We explore role neck re-                        D6IQ able suppress haploinsufficiency de- <br /> gion associated light chains regulating Myo2p                     letion MLC1. We used modified gel overlay assay <br /> function. Surprisingly, precise deletion                     demonstrate direct interaction Mlc1p and <br /> the IQ sites neck region results myosin,                     neck Myo2p. Overexpression MYO2 toxic, <br /> Myo2-D6IQp, support growth yeast                           causing severe decrease growth rate. When MYO2 <br /> strain 90% rate wild-type isogenic strain. We                    overexpressed, Myo2p fourfold stable a <br /> exploit mutant characterization light                       wild-type strain. High copies MLC1 completely over- <br /> chains Myo2p. First, demonstrate local-                       come growth defects increase stability of <br /> ization calmodulin sites polarized growth                           Myo2p. Our results suggest Mlc1p responsible for <br /> largely depends IQ sites neck Myo2p.                        stabilizing myosin binding neck region. <br /> Second, demonstrate previously uncharacter- <br /> ized protein, Mlc1p, myosin light chain Myo2p.                       Key words: myosin &#8226; polarized &#8226; stability &#8226; Myo4 <br /> MLC1 (YGL106w) essential gene exhibits                            &#8226; cytokinesis <br />  <br />  <br />  <br />                                                                              Mooseker, 1992; Rayment et al., 1993). IQ sites z25&#8211; <br />  <br /> T <br />        Saccharomyces cerevisiae unconventional myo- <br />        sin Myo2p member <span id='am-48' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-49' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> V myosins                      amino acid residue motifs bind calmodulin myosin <br />        implicated vesicle movement polar-                    light chains. In cases, binding small <br /> ized growth (Johnston et al., 1991; Govindan et al., 1995).                  EF-hand proteins activates Mg21ATPase activity of <br /> Homologues Myo2p include mouse dilute, plays                      myosins. For example, calmodulin required the <br /> role melanosome transport (Provance et al., 1996;                         Mg21ATPase activity chicken myosin V (Espindola et <br /> Nascimento et al., 1997), chicken myosin V,                    al., 1992). Furthermore, light chains perform structural <br /> a role neuron filopodial extension (Wang et al., 1996).                   roles affecting myosin head orientation orien- <br /> Myo2p essential growth S. cerevisiae, local-                tation light chains other. As example, the <br /> izes bud tip bud formation bud                      light chains scallop myosin II required stabilize <br /> neck cytokinesis (Brockerhoff et al., 1994; Lillie                    Ca21 binding myosin head domain alter the <br /> and Brown, 1994). A temperature-sensitive mutation                        myosin head orientation (Fromherz Szent-Gyorgyi, <br /> MYO2 confers defects polarized growth vacuole                      1995). Myo2p tandem IQ sites (Johnston et al., <br /> inheritance general secretion (Johnston et al.,                   1991). <br /> 1991; Govindan et al., 1995; Hill et al., 1996).                                We previously shown calmodulin binds to <br />   All myosins light chain binds                    Myo2p perform essential function polarized <br /> the myosin heavy chain light chain binding motif                       growth (Brockerhoff et al., 1994). Not calmodulin <br /> called IQ site (Cheney Mooseker, 1992; Xie et al.,                    present sites polarized growth (Brockerhoff and <br /> 1994). IQ sites tandem repeats                    Davis, 1992), calmodulin Myo2p direct phys- <br /> the head-motor domain tail domain (Cheney                        ical contact IQ sites neck Myo2p <br />                                                                              (Brockerhoff et al., 1994). CMD1 mutants allele-spe- <br />                                                                              cific synthetic lethality mutant myo2-66, sug- <br />                                                                              gesting calmodulin Myo2p share essential <br /> Address correspondence Trisha N. Davis, Department Biochemis- <br />                                                                              function. <br /> try, Box 357350, University Washington, Seattle, WA 98195-7350. Tel.:        Interestingly, class V myosins S. cerevisiae, <br /> (206) 543-5345. Fax: (206) 685-1792. E-mail: tdavis@u.washington.edu         Myo2p Myo4p, confer deleterious effects cell <br />  <br />  <br />  <br /> &#63721; The Rockefeller University Press, 0021-9525/98/08/711/12 $2.00 <br /> The Journal Cell Biology, Volume 142, Number 3, August 10, 1998 711&#8211;722 <br /> http://www.jcb.org                                                           711 <br />  Table I. <span id='am-122' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-123' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-126' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>Plasmids</span> Used <span id='am-56' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-59'  ></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> Study <br /> Plasmid                                  Parent vector                            Relevant markers construction*                              Reference source <br />  <br /> pBluescriptII KS (1)                                                        r <br />                                                                        amp f1 origin                                                        Stratagene <br /> pGEX-2T                                                                Ptac gene encoding GST                                          Pharmacia <br /> pQE30                                                                  Ptac 6XHis polylinker                                           Qiagen <br /> pRS306                                                                 URA3 f1 origin                                                       (Sikorski Hieter, 1989) <br /> pRS315                                                                 CEN6 ARSH4 LEU2 f1 origin                                            (Sikorski Hieter, 1989) <br /> pRS316                                                                 CEN6 ARSH4 URA3 f1 origin                                            (Sikorski Hieter, 1989) <br /> pGF27                              pRS304                              2-mm origin (YEp24 fragment) inserted AatII site                  G. Zhu <br /> pGF29                              pRS306                              2-mm origin (YEp24 fragment) inserted AatII site                  G. Zhu <br /> pJG28                              pSB5                                ampr trc promoter, CMD1                                              J. Geiser <br /> pJP10-2B                           YCp50                               URA3 MYO2 CEN4 ARS1                                                  (Johnston et al., 1991) <br /> pLI831                             pBluescriptII SK (1)                ADE3                                                                 E. Muller <br /> myo4D::URA3                        pBluescriptII KS (1)                myo4D::URA3                                                          S. Brown <br /> pRS23                              pBluescriptII SK (1)                5.6-kb ClaI fragment MYO2                                         This study <br /> pRS28                              pRS27                               myo2D::TRP1                                                          This study <br /> pRS29                              pRS25                               5.6-kb ClaI fragment MYO2                                         This study <br /> pRS31                              pGF29                               5.6-kb ClaI fragment MYO2                                         This study <br /> pRS43                              pRS29                               NheI site removed                                                    This study <br /> pRS50                              pRS37                               2-mm origin (YEp24 fragment) inserted NotI site                   This study <br /> pRS72                              pRS43                               NcoI site added <span id='am-71' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-73'  ></span><span id='am-74' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> 1 MYO2                                   This study <br /> pRS172                             pRS43                               Internal deletion MYO2 (S787G                                     This study <br />                                                                           Daa788&#8211;927) known MYO2-D6IQ <br /> pRS174                             pRS306                              5.4-kb ClaI fragment MYO2-D6IQ                                    This study <br /> pRS221                             pGF29                               5.4-kb ClaI fragment MYO2-D6IQ                                    This study <br /> pRS276                             pBluescriptII KS (1)                3.3-kb BamHI fragment carrying MLC1                                  This study <br /> pRS286                             pRS285                              mlc1D::TRP1                                                          This study <br /> pRS289                             pRS316                              1.4-kb BamHI-SacI fragment MLC1                                   This study <br /> pRS290                             pGF27                               1.4-kb BamHI-SacI fragment MLC1                                   This study <br /> pRS296                             pQE30                               ampr promoter 6XHis-MLC1                                             This study <br /> pRS321                             pRS315                              1.4-kb BamHI-SacI fragment MLC1                                   This study <br /> pSB6                               pACYC177                            kanr lysis genes lambda                                           (Brockerhoff et al., 1994) <br /> pSB20                              pGEX-3X                             MYO2 (aa740&#8211;1457) fused GST                                       (Brockerhoff et al., 1994) <br /> pSB21                              pGEX-3X                             MYO2 (aa740&#8211;1116) fused GST                                       (Brockerhoff et al., 1994) <br /> pSB24                              pGEX-3X                             MYO2 (aa790&#8211;924) fused GST                                        (Brockerhoff et al., 1994) <br /> pSB25                              pGEX-3X                             MYO2 (aa740&#8211;833) fused GST                                        (Brockerhoff et al., 1994) <br /> pSB27                              pGEX-3X                             MYO2 (aa247&#8211;740) fused GST                                        (Brockerhoff et al., 1994) <br /> pTD28                              pTD17                               2-mm origin LYS2 CMD1                                                (Davis Thorner, 1989) <br /> pTD29                              YEp24                               2-mm origin LYS2                                                     (Geiser et al., 1993) <br /> *Unless stated, markers parent plasmid present new construct. MYO2-D6IQ encodes internal deletion Myo2p mutation S787G and <br /> missing amino acids 787&#8211;927. MLC1 identical Saccharomyces cerevisiae open reading frame YGL106w. <br />  <br />  <br />  <br />  <br /> growth overexpressed (Haarer et al., 1994). Myo4p                                  cloned ClaI site pBluescriptII KS1 (Stratagene, La Jolla, CA) <br /> localizes bud essential polarized distri-                         make pRS23. In pRS24, 4.1-kb NdeI-AflII fragment MYO2 was <br />                                                                                         removed. Plasmid pRS24 digested EcoRI, ends filled in <br /> bution asymmetry determinant, Ash1p (Bobola et al.,                              Klenow fragment DNA polymerase destroy EcoRI site, <br /> 1996; Jansen et al., 1996). MYO4 essential growth                            ends ligated T4 DNA ligase make pRS26. The re- <br /> (Haarer et al., 1994). The cause growth defects ei-                             maining coding sequence MYO2 pRS26 replaced EcoRI <br /> ther MYO2 MYO4 overexpressed unknown.                                          site oligonucleotide-directed mutagenesis (Kunkel et al., 1987) using <br />                                                                                         primer MYO2D (59-CTCGCGCCATCAGTTGAATTCTATACG- <br />    Here investigate role light chains function                         CTGTAATATCTGCTG-39), creating plasmid pRS27. In plasmid <br /> of Myo2p. Localization calmodulin sites cell growth                            pRS28, 0.8-kb EcoRI fragment containing TRP1 (Davis et al., 1986) was <br /> depends neck region Myo2p. The S. cerevisiae ge-                              cloned EcoRI site pRS27. <br /> nome sequencing project revealed small protein simi-                                The plasmid shuffle used study required plasmid encoding <br /> larities calmodulin myosin light chains. We                            ADE3 MYO2. A 5.6-kb ClaI fragment MYO2 cloned the <br />                                                                                         SmaI site pLI831 (Geiser et al., 1993) make pRS37. The shuttle vec- <br /> this protein, Mlc1p (myosin light chain), binds                           <span id='am-107' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-108' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-109'  ></span><span id='am-110' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>tor</span> pRS50 contains 2-mm origin 2.1-kb EcoRI fragment YEp24 <br /> to myosin neck regulates stability Myo2p.                                ligated NotI site pRS37. In plasmids, 59 overhangs <br />                                                                                         filled treatment Klenow fragment DNA poly- <br />                                                                                         merase. <br /> Materials Methods                                                                      Additional plasmids containing MYO2 carrying URA3 <br />                                                                                         selectable marker. Plasmid pRS25 derivative pRS316 (Sikorski and <br /> Plasmids                                                                                Hieter, 1989) HindIII-NotI fragment polylinker was <br />                                                                                         removed, 59 overhangs filled Klenow, plasmid <br /> The plasmids used study listed Table I.                                  recircularized T4 DNA ligase. The plasmid pRS29 contains the <br />    Plasmid pRS28, used precisely delete MYO2,                    5.6-kb ClaI fragment MYO2 plasmid pJP10-2B cloned the <br /> steps. The 5.6-kb ClaI fragment pJP10-2B, containing MYO2,                       ClaI site pRS25. In pRS29, coding sequence MYO2 runs in <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 142, 1998                                           712 <br />  the direction f1 origin. For later studies, NheI site           MMY28 carrying integrated form GFP&#8211;CMD1 (Moser et al., 1997) <br /> pRS29 removed digestion NheI, filling Klenow,              make diploid strain RSY33. A myo4D::URA3 construct (a gift from <br /> reclosing T4 DNA ligase create pRS43. The high-copy number                 Susan Brown, University Michigan, Ann Arbor, MI) transformed <br /> plasmid pRS31 created cloning 5.6-kb ClaI fragment MYO2              RSY33 create strain RSY38, heterozygous GFP&#8211;CMD1, <br /> into ClaI site pGF29, 2-mm plasmid.                                       MYO2-D6IQ, myo4D. <br />    Deletion region encoding neck MYO2 oligo-                Strain RSY105 carrying deletion MLC1 difficult create be- <br /> nucleotide-directed mutagenesis (Kunkel et al., 1987). Because              cause MLC1 displays haploinsufficiency. To create disruption, first <br /> large size MYO2, cassette created containing middle one-              transformed JGY46 plasmid pRS289, carrying MLC1 gene. The <br /> third gene. The 2.6-kb BamHI-EcoRI fragment MYO2                     resulting strain functioning copies MLC1. If copy of <br /> cloned BamHI EcoRI sites pBluescriptII KS1 make                 MLC1 removed one-step gene replacement, resulting <br /> pRS78. The step deleting IQ sites introduce           strain copies gene. The 1.7-kb BamHI-NdeI <br /> NarI restriction endonuclease cleavage sites coding sequence           fragment plasmid pRS286 encoding mlc1D::TRP1 transformed into <br /> MYO2 end region encoding IQ sites. The oligonucle-            JGY46(pRS289). Of 42 colonies obtained, unable lose <br /> otide NAR1 (59-GCAATAAGATGCATAATGGCGCCATTGTTATGA-                                  URA3-marked MLC1 plasmid pRS289 streaked minimal <br /> TCCAGAAG-39) results addition <span id='am-6' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>glycine</span> amino acid               medium containing 59-fluoro orotic acid (59-<span id='am-1' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-2' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5'  ></span>FOA</span>).1 These colonies <br /> residues 786 787 mutation S787A. The oligonucleotide            likely candidates heterozygous mlc1D strains. To test hy- <br /> NAR4       (59-GAAATTGAAACAATTGAAGGGCGCCGACGCTAA-                                  pothesis, strains transformed LEU2-marked MLC1 <br /> ATCAGTTAATC-39) results addition glycine amino                 plasmid pRS321 streaked minimal medium containing 59-FOA. <br /> acids 927 928. After mutating pRS78 using NAR1 NAR4, re-               Three strains able lose pRS289 given alterna- <br /> sulting plasmid digested NarI reclosed using T4 DNA ligase.           tive source MLC1 pRS321. RSY105 shown heterozy- <br /> This mutant cassette ligated BamHI- EcoRI-cut pRS43             gous MLC1/mlc1D::TRP1 Southern blot analysis. RSY105 carry a <br /> create plasmid pRS172, encodes Myo2p S787G muta-               plasmid source MLC1 survive. <br /> tion lacks amino acids 788&#8211;927. This mutant named MYO2-                  To study null phenotype mlc1D, strains RSY105(pRS289) and <br /> D6IQ. Plasmid pRS221 5.4-kb Asp718-SacI fragment containing                JGY46 sporulated SpoIII medium, 50 tetrads dissected <br /> MYO2-D6IQ cloned pGF29.                                                       strain. After 24 h, spores examined germination and <br />    The plasmid pRS174 created integrate MYO2-D6IQ ge-              colony growth. Images spores colonies obtained described <br /> nome S. cerevisiae. pRS174 ligating 5.4-kb ClaI frag-           using fluorescent microscope (model Axioplan; Carl Zeiss, Inc., <br /> ment pRS172 containing MYO2-D6IQ ClaI site pRS306.                  Thornwood, NY) 0.3-s, low-light exposures. Germinating spores <br />    The gene YGL106w encodes 149&#8211;amino acid protein similarities             transferred 20 ml minimal medium containing 3.7% formalde- <br /> to calmodulin. Based work described paper, re-              hyde. After 60 min room temperature, 4,69-diamidino-2-phenylindole <br /> named YGL106w MLC1. The 3.3-kb BamHI fragment lambda PM-                     (DAPI) added final concentration 50 mg/ml, nuclei were <br /> 3353 (American Type Culture Collection, Rockville, MD) cloned             examined described below. <br /> the BamHI site pBluescriptII KS1, creating plasmid pRS276. Plasmid <br /> pRS287 derivative pRS276, 2.0-kb NdeI-NotI fragment <br /> was removed, 59 overhangs filled Klenow, plasmid              Immunoblot Analysis <br /> recircularized T4 DNA ligase. In plasmid pRS287, MLC1 only <br /> open reading frame remaining pBluescript polylinker.                        SDS-PAGE immunoblot analysis performed described (Gei- <br />    The plasmid delete MLC1 precisely similar way           ser et al., 1991) following exceptions. Strains analyzed were <br /> myo2D construct. Plasmid pRS284 digesting pRS276                  grown 50 Klett units appropriate medium. After centrifugation, <br /> EcoRI, filling 59 overhangs Klenow, recircularizing            cells washed lysis buffer (50 mM Tris, pH 8.0, 13 proteinase inhib- <br /> plasmid T4 DNA ligase. The coding sequence MLC1 replaced               itor cocktail [Drubin et al., 1988]). Cells lysed mixing equal vol- <br /> with EcoRI site oligonucleotide-directed mutagenesis (Kunkel et al.,         umes cells, glass beads (Sigma Chemical Co., St. Louis, MO), lysis <br /> 1987) using primer MLC1D (59-CTAAATTTGCAGTTCCGCACTCT-                              buffer vortexing times 1 min alternating 1-min incuba- <br /> CAGAATTCTGTTATTCTATGTATGTGCG-39), creating pRS285. A                               tions ice. After centrifugation 15 min microcentrifuge 48C, <br /> 0.8-kb EcoRI fragment containing TRP1 (Davis et al., 1986) cloned              pellets washed lysis buffer solubilized using cracking <br /> into EcoRI site pRS285 make pRS286. In pRS286, TRP1                  buffer (10 mM sodium phosphate, pH 7.2, 1% b-mercaptoethanol, 1% <br /> gene orientation original MLC1.                              SDS, 6 M urea). For detection Myo2p, 30 mg protein loaded <br />    Plasmid pRS296, designed express 6XHis-tagged MLC1 Escheri-               lane 6% SDS&#8211;polyacrylamide gel. Proteins transferred <br /> chia coli, steps. An NcoI site placed         nitrocellulose membranes using wet-transfer apparatus (Bio-Rad <br /> codon MLC1 oligonucleotide-directed mutagenesis using primer                 Laboratories, Hercules, CA) following instructions provided the <br /> MLC1-NCOI            (59-GGCTCTGGTGGCTGCCATGGTTATTCTAT-                            manufacturer. The membranes incubated 1:400 dilution af- <br /> GTATGTGC-39) (Kunkel et al., 1987) make pRS291. This mutagenesis                finity-purified anti-Myo2p antibody (Lillie Brown, 1994). Secondary <br /> also created mutation S2A. The MLC1 fragment digest-               antibody blotting grade goat anti&#8211;rabbit IgG horseradish peroxidase <br /> ing pRS291 NcoI, filling using Klenow fragment DNA poly-            conjugate (Bio-Rad Laboratories) 1:5,000 dilution. Signal de- <br /> merase, digesting SacI, purifying 0.8-kb fragment. Plasmid            tected using Renaissance luminol reagent (DuPont/NEN, Boston, MA) <br /> pRS296 digesting pQE30 (Qiagen, Inc., Chatsworth,                 Hyperfilm-MP (Amersham Corp., Arlington Heights, IL). <br /> CA) BamHI, filling using Klenow fragment DNA poly- <br /> merase, digesting SacI, ligating 0.8-kb MLC1 fragment. <br />                                                                                    Gel Overlays <br />                                                                                    Production 35S-labeled 6XHis-Mlc1p follows. The E. coli strain <br /> Media Strains                                                                  GM1 (Coulondre Miller, 1977) containing plasmids pRS296 and <br />                                                                                    pSB6 grown 30 ml M9 medium (Miller, 1972) 1.0 mM <br /> Media growth S. cerevisiae E. coli described previously <br />                                                                                    MgCl2 (instead MgSO4), 0.4% glucose, 1 mg/ml thiamine, 50 mg/ml <br /> (Zhu et al., 1993). The yeast strains used study listed Table II. <br />                                                                                    ampicillin, 15 mg/ml kanamycin, 8 mCi carrier-free Na35SO4 (DuPont/ <br /> Genetic manipulations transformations performed essentially as <br />                                                                                    NEN) 20 Klett units induced adding isopropyl-b-d-thioga- <br /> described previously (Sherman et al., 1986). RSY2 carrying deletion of <br />                                                                                    lactopyranoside final concentration 2 mM. After 5 h 378C, the <br /> MYO2 created one-step gene replacement (Rothstein, 1991) using <br />                                                                                    cells collected centrifugation resuspended 400 ml lysis <br /> the plasmid pRS28 digested BglII ClaI transformed RSY1. <br />                                                                                    buffer (50 mM Tris, pH 7.5, 1 mM MgCl2, 1 mM phenylmethylsulfonyl <br /> Because strains containing ade3D-100 histidine auxotrophs, ADE3 on <br />                                                                                    fluoride). The sample frozen thawed times, alternating <br /> a plasmid selected histidine prototrophy (Jones Fink, 1982). <br /> RSY2 transformed pRS50 carrying ADE3 MYO2. The strain <br /> RSY2-60B MYO2 deleted genome carries pRS50. <br />    The mutant MYO2-D6IQ integrated yeast two-step gene <br /> replacement (Rothstein, 1991) using plasmid pRS174 digested                   1. Abbreviations used paper: 59-FOA, 59-fluoro orotic acid; DAPI, <br /> BamHI. Strains RSY21 RSY22 shown contain MYO2-D6IQ                     49,6-diamidino-2-phenylindole; GFP, green fluorescent protein; GST, glu- <br /> by PCR Southern blot analysis. RSY22 mated strain                    tathione-S-transferase. <br />  <br />  <br />  <br />  <br /> Stevens Davis M1c1p Is Myosin Light Chain Myo2p                          713 <br />  Table II. Yeast Strains <br /> Strain                                                           Genotype*                                                            Reference <br />  <br /> CRY1                MATa ade2-1oc can1-100 his3-11,15 leu2-3,112 trp1-1 ura3-1                                           R. Fuller (Stanford University, CA) <br /> JGY46               MATa/MATa ade2-1oc/ade2-1oc can1-100/can1-100 his3-                                                  (Geiser et al., 1991) <br />                      11,15/his3-11,15 leu2-3,112/leu2-3,112 trp1-1/trp1-1 ura3-1/ura3-1 <br /> EMY55-5D            MATa ade2-1oc ade3D-100 can1-100 cyh2r his3-11,15 leu2-3,112 lys2D::HIS3 trp1-1 ura3-1               E. Muller (University of <br />                                                                                                                             Washington, Seattle, WA) <br /> EMY55-6B            MATa ade2-1oc ade3D-100 can1-100 cyh2S his3-11,15 leu2-3,112 lys2D::HIS3 trp1-1 ura3-1               E. Muller (University of <br />                                                                                                                          Washington) <br /> MMY28               MATa ade2-1oc ade3D-100 can1-100 (S65T-GFP)&#8211;CMD1 his3-                                               (Moser et al., 1997) <br />                       11,15 leu2-3,112 lys2D::HIS3 trp1-1 ura3-1 <br /> RSY1                EMY55-5D X EMY55-6B                                                                                  This study <br /> RSY2                MATa/MATa ade2-1oc/ade2-1oc ade3D-100/ade3D-100 can1-100/can1-100 cyh2r/cyh2S                        This study <br />                        his3-11,15/his3-11,15 leu2-3,112/leu2-3,112 lys2D::HIS3/lys2D::HIS3 MYO2/myo2D::TRP1 <br />                       trp1-1/-trp1-1 ura3-1/ura3-1 <br /> RSY2-60B            MATa ade2-1oc ade3 D-100 can1-100 his3-11,15 leu2-3,112                                              This study <br />                       lys2D::HIS3 myo2D::TRP1 trp1-1 ura3-1 carrying plasmid pRS50 <br /> RSY21               MATa ade2-1oc can1-100 his3-11,15 leu2-3,112 MYO2-D6IQ trp1-1 ura3-1                                 This study <br /> RSY22               MATa ade2-1oc can1-100 his3-11,15 leu2-3,112 MYO2-D6IQ trp1-1 ura3-1                                 This study <br /> RSY38               MATa/MATa ade2-1oc/ade2-1oc ADE3/ade3D-100 can1-100/can1-100 CMD1/GFP&#8211;CMD1                           This study <br />                        his3-11,15/his3-11,15 leu2-3,112/leu2-3,112 MYO2/MYO2-D6IQ MYO4/myo4D::URA3 <br />                       trp1-1/trp1-1 ura3-1/ura3-1 <br /> RSY38-9D            MATa ade2-1oc ade3D-100 can1-100 GFP&#8211;CMD1 his3-11,15                                                 This study <br />                       leu2-3112 lys2D::HIS3 MYO2-D6IQ trp1-1 ura3-1 <br /> RSY38-16A           MATa ade2-1oc ade3D-100 can1-100 GFP&#8211;CMD1 his3-11,15                                                 This study <br />                       leu2-3,112 lys2D::HIS3 myo4D::URA3 trp1-1 ura3-1 <br /> RSY38-16C           MATa ade2-1oc ade3D-100 can1-100 his3-11,15 leu2-3,112                                               This study <br />                       lys2D::HIS3 MYO2-D6IQ myo4D::URA3 trp1-1 ura3-1 <br /> RSY38-17C           MATa ade2-1oc ade3D-100 can1-100 GFP&#8211;CMD1 his3-11,15                                                 This study <br />                       leu2-3,112 lys2D::HIS3 trp1-1 ura3-1 <br /> RSY105              MATa/MATa ade2-1 oc/ade2-1 oc can1-100/can1-100 his3-                                                This study <br />                       11,15/his3-11,15 leu2-3,112/leu2-3,112 MLC1/mcl1D::TRP1 <br />                       trp1-1/trp1-1 ura3-1/ura3-1 carrying plasmid pRS289 <br /> RSY105-6A           MATa ade2-1oc can1-100 his3-11,15 leu2-3,112 mcl1D::TRP1                                             This study <br />                       trp1-1 ura3-1 carrying plasmid pRS289 <br /> RSY107              RSY21 X RSY105-6A carrying plasmid pRS289                                                            This study <br /> RSY112              MATa/MATa ade2-1oc/ade2-1oc ade3D-100/ade3D-100 can1-100/can1-100 his3-11,15/his3-11,15              This study <br />                        leu2-3,112/leu2-3,112 lys2D::HIS3/lys2D::HIS3 MLC1/mcl1D::TRP1 <br />                       MYO2/myo2D::TRP1 trp1-1/trp1-1 ura3-1/ura3-1 carrying plasmid pRS289 <br /> *GFP&#8211;CMD1 encodes fusion protein green fluorescent protein calmodulin. <br />  <br />  <br />  <br />  <br /> between 220 208C lyse cells, centrifuged 10,000 g          branes probed 16 h 218C 140 mM 35S-labeled <br /> for 30 min 48C microcentrifuge. The supernatant fraction batch           6XHis-Mlc1p 156 mM 35S-labeled calmodulin buffer C (buffer A <br /> mixed 10 min 700 ml preequilibrated Ni-NTA beads (Qiagen,                5 mM CaCl2 0.05% Tween-20). The membranes then <br /> Inc.). The beads poured form column washed 10 ml ly-           washed 40 min (four changes) buffer C. Next, membranes were <br /> sis buffer containing 25 mM imidazole. Purified 6XHis-Mlc1p eluted            air dried, dipped 7% 2,5-diphenyloxazole acetone, air dried, ex- <br /> from column using lysis buffer containing 500 mM imidazole. The imi-          posed Hyperfilm-MP (Amersham Corp.) 5&#8211;8 d. <br /> dazole buffer exchanged 100 mM ammonium bicarbonate a <br /> PD-10 G25 column (Pharmacia Biotech, Piscataway, NJ). Protein                 Northern Analysis MYO2 Transcript <br /> stored 2808C needed. Protein concentration determined using <br /> bicinchoninic acid (Sigma Chemical Co.). The purified 35S-labeled 6XHis-          Total mRNA purified strains CRY1 carrying pRS43, <br /> Mlc1p specific activity 27 dpm/fmol.                                     pRS172, pRS31, pRS221 using previously described procedure (Wise, <br />     Purified 35S-labeled calmodulin previously described (Brock-      1991). From 500-ml cultures, z7.0 mg RNA obtained. Total mRNA <br /> erhoff et al., 1992), 8 mCi Na35SO4 used labeling.    purified samples using Promega PolyA Tract mRNA <br /> The purified 35S-labeled calmodulin specific activity 39 dpm/fmol.       Isolation System I (Madison, WI) MagneSphere particles. Approxi- <br />     The gel overlay assays calmodulin Mlc1p binding per-             mately 50 mg poly A&#8211;containing mRNA purified culture. <br /> formed previously described (Brockerhoff et al., 1994) slight mod-           Northern blot analysis performed using standard procedures (Sam- <br /> ifications. In brief, extracts E. coli strain GM1 (Cou-        brook et al., 1989) following modifications. 10 mg mRNA was <br /> londre Miller, 1977) containing plasmid pGEX-2T, pSB20,                loaded lane prerun formaldehyde gel. After removing the <br /> pSB21, pSB24, pSB25, pSB27. Approximately 30 mg total protein              formaldehyde washing 23 DEPC dH2O, mRNA transferred to <br /> loaded lane SDS-PAGE. Proteins transferred Immo-            nitrocellulose membrane using capillary transfer (Sambrook et al., <br /> bilon membranes (Millipore Corp., Bedford, MA) using wet-transfer               1989). The membrane baked 658C 1 h. Probes using <br /> apparatus (Bio-Rad Laboratories) following instructions provided           3.4-kb NcoI-EcoRI fragment MYO2 pRS72 PCR prod- <br /> the manufacturer. After transfer, proteins renatured washing          uct containing 0.5-kb fragment TRX2. After incubating mem- <br /> the Immobilon membranes total 40 min (four changes) buffer A          brane z1 3 107 cpm probe (pooled) 48 h, membrane <br /> (20 mM Hepes, pH 7.2, 100 mM NaCl) blocked 6 h buffer A                washed analyzed using Molecular Dynamics PhosphorImager <br /> containing 3% bovine serum albumin 0.05% Tween-20. The mem-                   (model 400S; Sunnyvale, CA) using 176-micron pixel size. <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 142, 1998                                     714 <br />  Indirect Immunofluorescence Green Fluorescent                               Myo2-D6IQp stably produced wild-type <br /> Protein Fusions                                                                 Myo2p present strain (Fig. 1 C). The mutant cells <br /> The immunolocalization Myo2p strains CRY1 RSY21 per- <br />                                                                                 normal budding index average 10% <br /> formed previously described (Brockerhoff et al., 1994; Lillie            larger size. The distribution bud sizes similar to <br /> Brown, 1994).                                                                   seen wild-type cultures. The mutant cultures con- <br />    The localization GFP&#8211;Cmd1p strains RSY38-9D, RSY38-16A,                tained 5% large unbudded cells the <br /> RSY38-16C, RSY38-17C performed using fluorescent micro- <br />                                                                                 wild-type cultures. A strain carrying MYO2-D6IQ does <br /> scope (model Axioplan; Carl Zeiss, Inc.) (Moser et al., 1997). 200 cells from <br /> each strain scored presence polarized calmodulin. Calmod-       grow 38.58C, isogenic wild-type strain <br /> ulin judged polarized fluorescent signal stronger   grows poorly. A strain carrying MYO2-D6IQ grows well <br /> bud mother portion cell. Images captured using         medium 1 M sorbitol (<span id='am-112' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-113'  ></span><span id='am-114' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> shown). <br /> an Imagepoint-cooled CCD video <span id='am-66' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-67' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-70'  ></span>camera</span> (Photometrics, Tucson, AZ) fit-             The neckless mutant protein localizes indistinguishably <br /> ted microscope conjunction IP Lab software (Signal Analyt- <br /> ics, Vienna, VA). Images GFP&#8211;calmodulin 5-s exposures. Previous <br />                                                                                 wild-type Myo2p (Fig. 2). In cases, un- <br /> studies shown expression GFP yeast resulted          budded cells localization small patch of <br /> uniform distribution fluorescence cell, excluding         Myo2p. In small- medium-budded cells, Myo2p is <br /> vacuole (Niedenthal et al., 1996).                                              present concentrated region bud tip. In large- <br />                                                                                 budded cells, Myo2p bud tip the <br /> Results                                                                         bud neck (Fig. 2). These results consistent previ- <br />                                                                                 ous analysis Myo2p localization (Brockerhoff et al., <br /> Myo2p Functions Neck Domain                                           1994; Lillie Brown, 1994). <br /> The neck Myo2p contains IQ sites tandem array <br /> spanning amino acids 790&#8211;921 (Johnston et al., 1991). <br />                                                                                 Neckless MYO2-D6IQ Affects Calmodulin Localization <br /> Myo2p essential growth, IQ sites not.                    Our previous results strongly suggest calmodulin light <br /> The mutant MYO2-D6IQ, lacking sequence encoding                             chain Myo2p. Calmodulin localizes sites cell <br /> amino acids 787&#8211;927, supports nearly normal growth as <br /> the source Myo2p, causing 10% reduction in <br /> growth rate (Fig. 1). An immunoblot confirmed neck- <br />  <br />  <br />  <br />  <br /> Figure 1. The mutant MYO2-D6IQ allows growth. Strains carry- <br /> ing wild-type MYO2 (CRY1) MYO2-D6IQ (RSY21) were <br /> plated YPD medium incubated 3 d 30 378C (A). <br /> Log-phase cultures diluted 10,000 cells/ml YPD me- <br /> dium, growth monitored 4 h. Cell number de- <br /> termined Coulter counter (Coulter Corporation, Hialeah, <br /> FL). (B) j, wild-type MYO2 strain, CRY1; d, MYO2-D6IQ                           Figure 2. Comparison localization Myo2p Myo2- <br /> strain, RSY21. An immunoblot extracts log-phase                    D6IQp. Cells stained affinity-purified anti-Myo2p anti- <br /> cultures strains CRY1 RSY21 (C). The antibody anti-                  body described Materials Methods. Cells simulta- <br /> Myo2p (Lillie Brown, 1994).                                                 neously stained DAPI stain DNA. Bar, 5 mm. <br />  <br />  <br />  <br />  <br /> Stevens Davis M1c1p Is Myosin Light Chain Myo2p                       715 <br />  Figure 3. Localization GFP&#8211;calmodulin myo4D strains. Cultures RSY38-17c (WT), RSY38-16A (myo4D MYO2), RSY38-9D <br /> (MYO4 MYO2-D6IQ), RSY38-16C (myo4D MYO2-D6IQ) grown log phase. Samples collected, mixed equal vol- <br /> umes 1% agarose dissolved SDC, mounted microscope slides image collection described Materials Meth- <br /> ods. Bar, 5 mm. <br />  <br />  <br /> growth, calmodulin mutants display defects polar-           MYO2 localization GFP&#8211;calmodulin as <br /> ized growth (Brockerhoff Davis, 1992; Davis, 1992;             wild-type strain containing MYO4 MYO2. <br /> Ohya Botstein, 1994). Mutations CMD1 exacerbate             However, strain carrying MYO2-D6IQ myo4D, <br /> defects caused myo2-66 allele-specific manner.            GFP&#8211;calmodulin polarized 4% cells (Fig. 3 <br /> The viability MYO2-D6IQ strain allows test         Table III). Thus, calmodulin localization sites of <br /> the localization calmodulin sites cell growth de-         polarized growth largely dependent IQ sites of <br /> pends IQ sites Myo2p. Localization fu-          Myo2p, Myo4p contributes small extent. <br /> sion GFP calmodulin examined wild-type and <br /> neckless mutant strains. GFP&#8211;calmodulin functionally re-           MLC1, New Light Chain <br /> places calmodulin localizes properly (Moser et al.,            The newly completed genome sequence S. cerevisiae <br /> 1997). In strain carrying wild-type MYO2, GFP&#8211;calmod-            identified potential myosin light chain. YGL106w en- <br /> ulin sites cell growth 84% cells (Fig.    codes 149&#8211;amino acid polypeptide significant ho- <br /> 3 Table III). This number equivalent values ob-          mology calmodulin myosin light chains. Based on <br /> tained immunolocalization (Brockerhoff Davis,               results described below, renamed gene <br /> 1992). The cells polarized calmodulin           MLC1. An alignment vertebrate calmodulin the <br /> large-budded cells process moving calmodulin             similar light chain reveals homology the <br /> from bud tip bud neck unbudded cells that <br /> have started bud formation. In strain carrying <br /> MYO2-D6IQ, 15% cells polarized cal-               Table III. Calmodulin Localization Myosin Mutants <br /> modulin greatly reduced diffuse signal even <br />                                                                    Genotype                                    Polarized calmodulin (%) <br /> in cells. Myo4p localizes diffusely yeast bud <br /> (Jansen et al., 1996). We tested remaining             Wild type                                             84 <br /> GFP&#8211;calmodulin bud MYO2-D6IQ strain              myo2-D6IQ                                             15 <br /> dependent Myo4p, essential growth.             myo4D                                                 80 <br /> Strains deleted MYO4 containing wild-type                  myo4D myo2-D6IQ                                        4 <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 142, 1998                      716 <br />                                                                                                  Figure 4. Alignment the <br />                                                                                                 predicted amino acid se- <br />                                                                                                 quence MLC1 three <br />                                                                                                 small EF-hand proteins. <br />                                                                                                 Vertebrate calmodulin (<span id='am-61' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-62'  ></span><span id='am-63' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Homo</span> <br />                                                                                                 sapiens) amino acids 7&#8211;149, a <br />                                                                                                 chicken light chain (<span id='am-51' about='obo:NCBITaxon_9030' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_8976 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_9005 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_9072 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Gallus" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Gallus</span> <br />                                                                                                 gallus, MLE1) amino acids <br />                                                                                                 45&#8211;190, <span id='am-38' about='obo:NCBITaxon_5754' typeof='owl:Thing obo:NCBITaxon_555280 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_555407 obo:NCBITaxon_2759 obo:NCBITaxon_33677 obo:NCBITaxon_1485168 obo:NCBITaxon_554915'><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Acanthamoeba" datatype="xsd:string"></span>Acanthamoeba</span> <br />                                                                                                 myosin IC light chain (A. cas- <br />                                                                                                 tellanii, MICLC) amino acids <br />                                                                                                 7&#8211;149 compared the <br />                                                                                                 complete amino acid sequence <br />                                                                                                 Mlc1p. Amino acid residues <br />                                                                                                 sharing identity between <br />                                                                                                 Mlc1p the <br />                                                                                                 sequences shaded. <br />                                                                                                 Amino acids potential <br />                                                                                                 Ca21-binding loops la- <br />                                                                                                 beled solid line. <br />  <br />  <br />  <br />  <br /> sequences (Fig. 4). Mlc1p shares 35% identity verte-       Haploid spores contain mlc1D gene disruption re- <br /> brate calmodulin, second yeast calmodulin         quire MLC1 plasmid (data shown). Therefore, <br /> yeast proteins (Davis Thorner, 1986). Mlc1p 35%          MLC1 haploinsufficient essential. <br /> identical chicken light chain (MLE1) (Matsuda et al.,         To begin characterizing phenotype mlc1D cells, <br /> 1981). Mlc1p 31% identical Acanthamoeba myosin 1C         examined spores hemizygous diploid strain. The <br /> light chain, unconventional myosin light chain          diploid requires plasmid copy MLC1 survive. Any <br /> (Wang et al., 1997).                                            spores receiving plasmid grow colonies <br />                                                                 carried genomic copy mlc1D. The mlc1D <br /> MLC1 Is Essential Displays Haploinsufficiency               spores did inherit plasmid identified as <br /> Initial attempts make diploid hemizygous MLC1          20% spores germinated did form <br /> were unsuccessful. Southern blot analysis revealed         colonies. The mlc1D cells grew elongated buds (Fig. 6). <br /> one-step gene replacement (Rothstein, 1991) resulted         Cytokinesis complete removal cell <br /> strains containing wild-type copies MLC1        wall treatment zymolyase did result separa- <br /> for copy deletion construct. One possible      tion buds mother cells. DAPI staining re- <br /> explanation effect copy MLC1        vealed elongated cells multiple nuclei (Fig. 6 <br /> diploid genome insufficient viability. This phenome-     C). Less 1% spores control strain <br /> non called haploinsufficiency (Wilkie, 1994). We tested      formed elongated cells. <br /> if MLC1 haploinsufficient repeating gene disrup- <br /> tion strain carrying copies MLC1 genome <br />                                                                 Mlc1p Interacts Neck Myo2p <br /> plus copy MLC1 plasmid. In case,       Two lines evidence suggest Mlc1p light chain of <br /> able delete copy MLC1 genome de-          Myo2p. First, haploinsufficiency exhibited MLC1 is <br /> scribed Materials Methods. The hemizygous dip-       suppressed reduced copies MYO2 (Fig. 5). A diploid <br /> loid requires plasmid carrying MLC1 viability (Fig. 5).   strain hemizygous MYO2 MLC1 viable. <br />  <br />  <br />  <br />  <br />                                                                                                 Figure 5. MLC1 displays <br />                                                                                                 haploinsufficiency.      Strains <br />                                                                                                 JGY46, RSY105, RSY107, <br />                                                                                                 RSY112, carrying <br />                                                                                                 MLC1 plasmid pRS289, <br />                                                                                                 streaked min- <br />                                                                                                 imal medium lacking uracil <br />                                                                                                 minimal medium contain- <br />                                                                                                 ing 59-FOA (which kill <br />                                                                                                 cells require plas- <br />                                                                                                 mid pRS289). The plates were <br />                                                                                                 incubated 308C 3 d. <br />  <br />  <br />  <br />  <br /> Stevens Davis M1c1p Is Myosin Light Chain Myo2p       717 <br />                                                                       Figure 7. Glutathione-S-transferase (GST)&#8211;Myo2p fusion pro- <br />                                                                      teins. <br />  <br />  <br />  <br />                                                                         Second, Mlc1p binds directly IQ sites Myo2p as <br />                                                                      assayed gel overlay protocol. The constructs used in <br />                                                                      experiment presented Fig. 7. 6XHis-Mlc1p <br />                                                                      binds specifically fusion proteins contain IQ <br />                                                                      sites (Fig. 8 B). As control, calmodulin also <br />                                                                      binds specifically fusion proteins contain IQ sites <br />                                                                      (Fig. 8 C). Both calmodulin 6XHis-Mlc1p bind equally <br />                                                                      presence Ca21 EGTA (data shown). <br />                                                                      These data demonstrate Mlc1p direct interac- <br />                                                                      tion IQ sites Myo2p. <br />  <br />                                                                      The Toxicity Overexpressed MYO2 Is Suppressed by <br />                                                                      MLC1 Not CMD1 <br />                                                                      We looked functional interaction Mlc1p and <br />                                                                      Myo2p. A yeast strain carrying high-copy number plas- <br />                                                                      mid MYO2 grows 40% slower strains normal <br />                                                                      levels MYO2 (data shown). Microscopy re- <br />                                                                      vealed 5% cells MYO2 overexpression strain <br /> Figure 6. mlc1D spores defects cell separation. Strains      abnormal bud necks multiple buds (data not <br /> JGY46 RSY105(pRS289) sporulated tetrads            shown). A similar defect high-copy number MYO4 <br /> dissected. After 24 h, spores characterized classes: (1)   strains caused neck region Myo4p, and <br /> germinated formed colonies, (2) germinated did           overexpression calmodulin does suppress effect <br /> form colonies, (3) did germinate. Examples class      (Haarer et al., 1994). One possibility high copies of <br /> are shown (A). 200 spores strain classified (B).      myosin light chains overcome defects associated with <br /> Germinated spores RSY105(pRS289) stained              MYO2 overexpression. <br /> DAPI. Elongated spores DAPI staining shown (C). <br />                                                                         A strain overexpressing MYO2 shows noticeable re- <br /> Bars, 10 mm. <br />                                                                      duction growth rate (Fig. 9 A). The strain over- <br />                                                                      expressing MYO2-D6IQ grows normally. Northern blot <br /> Suppression haploinsufficiency hemizygous                  analysis reveals MYO2 MYO2-D6IQ are <br /> MYO2 demonstrates Mlc1p ordinarily                      transcribed similar steady-state levels two <br /> present minimum level necessary cell growth.              strains fivefold normal levels (Fig. 9 B). <br /> MYO2-D6IQ suppress haploinsufficiency                Surprisingly, immunoblot strain overexpressing <br /> MLC1 (Fig. 5). However, MYO2-D6IQ bypass                  MYO2 does higher levels Myo2p (Fig. 9, C <br /> requirement Mlc1p haploid strain (data                  D). In fact, Myo2p actually decrease the <br /> shown), essential func-          strain overexpressing MYO2. Overexpressing MYO2- <br /> tion Mlc1p yeast.                                              D6IQ results high levels Myo2-D6IQp (Fig. 9, C D). <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 142, 1998                        718 <br />  Figure 8. Gel-overlay assays vitro binding Mlc1p calmodulin IQ sites Myo2p. Myo2p fusion proteins ex- <br /> pressed subjected SDS-PAGE described Materials Methods. Polyacrylamide gel stained Coomassie blue. The full- <br /> length fusion protein extract marked arrow (A). Autoradiographs membranes incubated 35S-labeled 6XHis- <br /> Mlc1p (B) 35S-labeled calmodulin (C). The gel overlay assays performed described Materials Methods. Lanes 1&#8211;6 <br /> contain extracts IPTG-induced E. coli strain GM1 containing plasmids pGEX-2T, pSB20, pSB27, pSB21, pSB24, pSB25, respec- <br /> tively. See Fig. 7 Table I description fusion proteins. <br />  <br />  <br />  <br />   We hypothesized high levels calmodulin                mal end head 20-nm neck likely corre- <br /> Mlc1p counteract toxicity overexpressed               sponds IQ sites (Cheney et al., 1993). This region <br /> MYO2. Overexpressing CMD1 unable overcome                predicted a-<span id='am-43' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-45'  ></span><span id='am-46' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>helix</span> neck class II myo- <br /> decreased growth rate (Fig. 9 A) low levels              sins. In class II myosins, neck region stabilized the <br /> Myo2p (Fig. 9 D) caused overexpressed MYO2. In con-             binding myosin light chains (Rayment et al., 1993; <br /> trast, high-copy number plasmid carrying MLC1 able            Houdusse Cohen, 1996). Calmodulin previously <br /> to fully suppress overexpression MYO2 defects re-          identified light chain chicken yeast <br /> sults strain high levels Myo2p (Fig. 9, A     myosin V (Cheney et al., 1993; Brockerhoff et al., 1994). <br /> C). These results indicate neck region MYO2         However, calmodulin does stabilize yeast myosin. <br /> indeed responsible overexpression growth defect             Here, present evidence MLC1 (YGL106w) en- <br /> and toxicity overcome high levels Mlc1p.        codes new light chain yeast class V myosin <br />                                                                    Myo2p. Our results argue Mlc1p binds neck re- <br /> Mlc1p Stabilizes Myo2p                                             gion stabilizes Myo2p. <br /> Because high copies MLC1 able overcome de-                  When overexpressed, Myo2p decreased half-life <br /> fects caused overexpression MYO2, tested              toxic yeast cells. Overproduction calmodulin <br /> hypothesis Mlc1p stabilizes Myo2p. Yeast cultures             does affect stability Myo2p, overproduction <br /> were analyzed stability Myo2p following              Mlc1p stabilizes Myo2p ameliorates toxicity. <br /> Myo2p breakdown immunoblot analysis (Fig. 10). In             The requirement Mlc1p stabilize Myo2p bypassed <br /> wild-type strain, Myo2p half-life greater 8 h.       deleting IQ sites Myo2p. Thus, presence <br /> In strain carrying MYO2-D6IQ, Myo2-D6IQp               IQ sites destabilizes Myo2p unless sufficient Mlc1p <br /> half-life greater 8 h. The neckless Myo2-D6IQp          present. <br /> at stable wild-type Myo2p. When overex-                   Removal IQ sites Myo2p causes slightly <br /> pressed, half-life Myo2p 2&#8211;3 h. These data          slower growth rate slightly larger cells. A mutant Dic- <br /> explain Myo2p levels overexpression strain          tyostelium myosin II heavy chain DBLCBS, lacks <br /> are increased. In strain overexpressing MYO2             IQ sites, maintains 20% wild-type motor activity <br /> carrying high copies MLC1, half-life Myo2p 4 h.       measured sliding filament assay (Uyeda et al., <br /> High copies MLC1 able stabilize Myo2p,              1996). The DBLCBS myosin allows cell division vivo but <br /> suggesting Mlc1p light chain responsible stabilizing      demonstrates slightly slowed furrow formation cy- <br /> the neck region Myo2p.                                          tokinesis (Zang et al., 1997). Thus, Myo2-D6IQp and <br />                                                                    DBLCBS myosin confer relatively minor defects. <br />                                                                    These results indicate efficiency myosin <br />                                                                    essential myosin function vivo. <br /> Discussion                                                            MLC1 essential haploinsufficient. Haploinsuffi- <br /> <span id='am-102' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-103' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-104' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-106'  ></span>Electron</span> micrographs chicken myosin V molecule              ciency rarely observed yeast genes. There several <br /> show heads connected 30-nm stalk. At proxi-           reasons reduction number gene copies have <br />  <br />  <br />  <br />  <br /> Stevens Davis M1c1p Is Myosin Light Chain Myo2p          719 <br />                                                                                                   Figure 9. High copies of <br />                                                                                                  MLC1, high copies of <br />                                                                                                  CMD1, overcome toxic <br />                                                                                                  effects overexpressed <br />                                                                                                  MYO2. Plasmids used in <br />                                                                                                  experiments listed <br />                                                                                                  Table I. EMY55-5D was <br />                                                                                                  transformed plasmids <br />                                                                                                  encoding high copies of <br />                                                                                                  MYO2, MYO2-D6IQ, CMD1, <br />                                                                                                  MLC1. Plates incu- <br />                                                                                                  bated 3 d 308C (A). <br />                                                                                                  CRY1 transformed with <br />                                                                                                  pRS43, pRS172, pRS31, or <br />                                                                                                  pRS221, mRNA col- <br />                                                                                                  lected log-phase cul- <br />                                                                                                  tures. 10 mg polyA-con- <br />                                                                                                  taining RNA samples were <br />                                                                                                  analyzed described Ma- <br />                                                                                                  terials Methods. Blots <br />                                                                                                  simultaneously <span id='am-117' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-118' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-119'  ></span><span id='am-120' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-121' property="rdfs:label" content="hybrid" datatype="xsd:string"></span>hybrid</span>- <br />                                                                                                  ized MYO2 TRX2 <br />                                                                                                  32 <br />                                                                                                    P-labeled probes (B). CRY1 <br />                                                                                                  transformed in- <br />                                                                                                  dicated constructs study <br />                                                                                                  effects high copies of <br />                                                                                                  MLC1 (pRS290) strains <br />                                                                                                  extra copy of <br />                                                                                                  MYO2 (pRS43) ex- <br />                                                                                                  tra copy MYO2-D6IQ <br />                                                                                                  (pRS172), overexpressing <br />                                                                                                  MYO2 (pRS31) MYO2- <br />                                                                                                  D6IQ (pRS221) (C). CRY1 <br />                                                                                                  transformed in- <br />                                                                                                  dicated constructs study <br />                                                                                                  effects high copies of <br />                                                                                                  CMD1 (pTD28) strains <br />                                                                                                  overexpressing MYO2 (pRS31) <br />                                                                                                  MYO2-D6IQ (pRS221) <br />                                                                                                  (D). Log-phase extracts were <br />                                                                                                  described Materi- <br />                                                                                                  als Methods. Samples <br />                                                                                                  assayed immunoblot <br />                                                                                                  analysis using anti-Myo2p <br />                                                                                                  antibody (Lillie Brown, <br />                                                                                                  1994) detect steady-state <br />                                                                                                  levels Myo2p. <br />  <br />  <br />  <br /> deleterious effects. In cases, reduced gene copy            eliminated adding extra copies MLC1. Finally, re- <br /> number affects regulatory genes working threshold           moval IQ binding sites Myo2p rescues hap- <br /> level (Wilkie, 1994). An example form regulation      loinsufficiency exhibited MLC1. <br /> is dosage-dependent sex determination Drosophila. Al-            Myo2-D6IQp does overcome requirement of <br /> ternately, proteins produced minimum          Mlc1p cell viability. This observation does address <br /> level proper function. The ACT1 gene displays            essential nature interaction Mlc1p and <br /> temperature-sensitive growth defects increased osmo-         Myo2p. It merely demonstrates Mlc1p at <br /> sensitivity present low levels (Shortle et al., 1984).   essential function does involve binding <br /> Finally, stoichiometry various protein components         neck Myo2p. None myosins in <br /> may important. Several lines evidence suggest       yeast essential growth (Brown, 1997), suggesting <br /> fixed ratio Mlc1p Myo2p required con-          Mlc1p essential function unrelated myosins. <br /> fer normal growth. First, diploid yeast strain carrying        Inviable mlc1D cells striking defect cytokinesis, <br /> one copy MLC1 grow          resulting enlarged, multinucleate cells. This phenotype <br /> one copy wild-type MYO2. Second, growth defects           identical defects detected cells deleted IQG1, <br /> associated yeast strain overexpressing MYO2           encodes newly discovered IQGAP <span id='am-35' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> member <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 142, 1998                    720 <br />  Figure 10. Degradation Myo2p. Cycloheximide added final concentration 100 mg/ml log-phase cultures. Cultures were <br /> allowed continue shaking, samples collected immunoblot analysis described Materials Methods. CRY1 con- <br /> taining plasmids (m), plasmids pRS31 (high copies MYO2) pRS290 (high copies MLC1) (j), pRS31 (high copies <br /> of MYO2) pGF27 (vector alone) (d) examined. Bands immunoblots probed anti-Myo2p antibody quantified, <br /> and values normalized loading using Coomassie blue&#8211;stained band demonstrated high stability (A). The error bars rep- <br /> resent standard deviation (wild type), (with high copies MYO2 high copies MLC1), (with high cop- <br /> ies MYO2) independent experiments. The immunoblots probed anti-Myo2p antibody demonstrate Myo2p stability CRY1 <br /> carrying pRS31 pRS290 (B) pRS31 pGF27 (C), CRY1 (MYO2 strain) (D) RSY21 (MYO2-D6IQ <br /> strain) (E). Numbers represent hours cycloheximide addition. <br />  <br />  <br />  <br />  <br /> in yeast (Epp Chant, 1997; Lippincott Li, 1998).              In conclusion, presented lines evidence <br /> Iqg1p contains IQ sites act binding sites        Myo2p light chains, calmodulin a <br /> for Mlc1p. We propose essential function                newly characterized protein, Mlc1p. Mlc1p regulates the <br /> Mlc1p act light chain IQGAP.                         stability Myo2p binding neck region. Our <br />    Calmodulin additional light chain Myo2p. We            characterization Mlc1p aid identification of <br /> have previously shown calmodulin Myo2p coim-               additional light chains chicken class V myosins. <br /> munoprecipitate yeast cell extracts interact            We thank Susan Brown plasmid myo4D::URA3. <br /> vitro. A mutation actin-binding site Myo2p           This work supported National Institutes Health grant <br /> and interferes Myo2p function dramatically worsens             GM40506 (T.N. Davis). R.C. Stevens supported Public Health Ser- <br /> the phenotype conferred mutations calmodulin                  vices National Research Service Award T32 GM07270. <br /> (Brockerhoff et al., 1994). Here localiza-         Received publication 18 September 1997 revised form 15 June <br /> tion calmodulin sites cell growth predominately         1998. <br /> dependent IQ sites Myo2p small con- <br /> tribution yeast class V myosin Myo4p. In a <br />                                                                     References <br /> MYO2-D6IQ strain lacking MYO4, calmodulin localizes to <br /> the spindle pole body fast moving patches             Bobola, N., R.P. Jansen, T.H. Shin, K. Nasmyth. 1996. Asymmetric accu- <br />                                                                       mulation Ash1p postanaphase nuclei depends myosin restricts <br /> cell surface. The protein target binds calmodulin             yeast mating-type switching mother cells. Cell. 84:699&#8211;709. <br /> these fast moving patches identified. The             Brockerhoff, S.E., T.N. Davis. 1992. Calmodulin concentrates regions of <br /> IQGAP protein yeast localizes actin ring             cell growth Saccharomyces cerevisiae. J. Cell Biol. 118:619&#8211;629. <br />                                                                     Brockerhoff, S.E., C.G. Edmonds, T.N. Davis. 1992. Structural analysis of <br /> bud neck cytokinesis (Epp Chant, 1997; Lip-                wild-type mutant yeast calmodulins limited proteolysis electro- <br /> pincott Li, 1998). Because calmodulin localization             spray ionization mass spectrometry. Protein Sci. 1:504&#8211;516. <br />                                                                     Brockerhoff, S.E., R.C. Stevens, T.N. Davis. 1994. The unconventional my- <br /> the bud neck cytokinesis depends Myo2p,              osin, Myo2p, calmodulin target sites cell growth Saccharomyces <br /> calmodulin does bind yeast IQGAP calmodulin                 cerevisiae. J. Cell Biol. 124:315&#8211;323. <br /> binds IQGAP levels detection.                              Brown, S.S. 1997. Myosins yeast. Curr. Opin. Cell Biol. 9:44&#8211;48. <br />                                                                     Cheney, R.E., M.S. Mooseker. 1992. Unconventional myosins. Curr. Opin. <br />    Only noncalmodulin unconventional myosin                 Cell Biol. 4:27&#8211;35. <br /> light chain characterized. Biochemical studies             Cheney, R.E., M.K. O&#8217;Shea, J.E. Heuser, M.V. Coelho, J.S. Wolenski, E.M. <br /> identified MICLC light chain Acanthamoeba myo-               Espreafico, P. Forscher, R.E. Larson, M.S. Mooseker. 1993. Brain myo- <br />                                                                       sin-V two-headed unconventional myosin motor activity. Cell. 75: <br /> sin IC (Wang et al., 1997). Mlc1p MICLC share 31%                 13&#8211;23. <br /> sequence identity, proteins share significant              Coulondre, C., J.H. Miller. 1977. Genetic studies lac repressor: IV. <br />                                                                       Mutagenic specificity lacI gene <span id='am-76' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-91'  ></span>Escherichia coli</span>. J. Mol. Biol. 117: <br /> identity calmodulin/EF-hand <span id='am-99' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-100' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="superfamily" datatype="xsd:string"></span>superfamily</span> mem-               577&#8211;606. <br /> bers. The chicken class V myosin light           Davis, T.N. 1992. A temperature-sensitive calmodulin mutant loses viability <br /> chains, calmodulin additional uncharacterized                 mitosis. J. Cell Biol. 118:607&#8211;617. <br />                                                                     Davis, T.N., J. Thorner. 1986. Calmodulin calcium-binding pro- <br /> small proteins copurify myosin (Cheney et al.,          teins yeast. In Yeast Cell Biology. Vol. 33. J.B. Hicks, e <br /> </body></html>