<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Conserved Genes Act Modifiers Invertebrate SMN <br /> Loss Function Defects <br /> Maria Dimitriadi1., James N. Sleigh2,3., Amy Walker2., Howard C. Chang4, Anindya Sen4, Geetika <br /> Kalloo1, Jevede Harris2, Tom Barsby2,3, <span id='am-118' about='obo:NCBITaxon_39170' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_216706 obo:NCBITaxon_216718 obo:NCBITaxon_3398 obo:NCBITaxon_4136 obo:NCBITaxon_4143 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-119' property="rdfs:label" content="Melissa" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-121'  ></span><span id='am-122' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Melissa</span> B. Walsh5, John S. Satterlee2, Chris Li6, David Van <br /> Vactor4, Spyros Artavanis-Tsakonas4,7, Anne C. Hart1* <br /> 1 Department Neuroscience, Brown University, Providence, <span id='am-83' about='obo:NCBITaxon_483892' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_75054 obo:NCBITaxon_1 obo:NCBITaxon_74970 obo:NCBITaxon_131567 obo:NCBITaxon_75055 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-84'  ></span><span id='am-85' property="rdfs:label" content="Rhode" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Rhode</span> Island, United States America, 2 Center <span id='am-20' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-21' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23'  ></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Cancer</span> Research, Massachusetts General Hospital, <br /> Department Pathology, Harvard Medical School, Boston, Massachusetts, United States America, 3 Department Biology Biochemistry, University Bath, Bath, <br /> United <span id='am-100' about='obo:NCBITaxon_kingdom' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-101' property="rdfs:label" content="kingdom" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Kingdom</span>, 4 Department Cell Biology, Harvard Medical School, Boston, Massachusetts, United States America, 5 Department Molecular Cellular Biology, <br /> Brown University, Providence, Rhode Island, United States America, 6 Department Biology, City College &#8211; City University New York, New York, New York, United <br /> States America, 7 Colle&#180;ge France, <span id='am-93' about='obo:NCBITaxon_49669' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_50362 obo:NCBITaxon_4447 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_4667 obo:NCBITaxon_3193'><span id='am-94' property="rdfs:label" content="Paris" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-97'  ></span><span id='am-98' property="oboInOwl:hasRelatedSynonym" content="Paris L." datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasAlternativeId" content="NCBITaxon:49665" datatype="xsd:string"></span>Paris</span>, France <br />  <br />  <br />  <br />      Abstract <br />      Spinal Muscular Atrophy (SMA) caused diminished function Survival Motor Neuron (SMN) protein, the <br />      molecular pathways critical SMA pathology remain elusive. We used genetic approaches invertebrate models to <br />      identify conserved SMN loss function modifier genes. <span id='am-40' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-41'  ></span><span id='am-42' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Drosophila melanogaster</span> <span id='am-25' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-26' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-31'  ></span>Caenorhabditis elegans</span> a <br />      single gene encoding protein orthologous human SMN; diminished function invertebrate genes causes <br />      lethality neuromuscular defects. To genes modulate SMN function defects <span id='am-75' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-76' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>, approaches <br />      used. First, genome-wide RNAi screen C. elegans SMN modifier genes undertaken, yielding genes. <br />      Second, tested conservation modifier gene function species; genes identified invertebrate model <br />      tested function invertebrate model. Drosophila orthologs genes, identified <br />      originally C. elegans, modified Drosophila SMN loss function defects. C. elegans orthologs genes, were <br />      originally identified previous Drosophila screen, modified C. elegans SMN loss function defects. Bioinformatic analysis <br />      conserved, cross-species, modifier genes suggests conserved cellular pathways, specifically endocytosis and <br />      mRNA regulation, act critical genetic modifiers SMN loss function defects species. <br />  <br />   Citation: Dimitriadi M, Sleigh JN, Walker A, Chang HC, Sen A, et al. (2010) Conserved Genes Act Modifiers Invertebrate SMN Loss Function Defects. PLoS <br />   Genet 6(10): e1001172. doi:10.1371/journal.pgen.1001172 <br />   Editor: Gregory S. Barsh, Stanford University, United States America <br />   Received February 4, 2010; Accepted September 21, 2010; Published October 28, 2010 <br />   <span id='am-55' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-56' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58'  ></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> open-access article distributed terms Creative Commons Public Domain declaration stipulates that, placed public <br />   domain, work freely reproduced, distributed, transmitted, modified, built upon, used lawful purpose. <br />   Funding: This work supported Spinal Muscular Atrophy Foundation (DVV, SA-T, ACH), National Institutes Health (NIH R01 NS42459) NIH/ <br />   NCRR/RCMI Grant G12-RR03060 (CL), City College New York, respectively. Also NIH R01 NS26084 (SA-T) Families SMA Postdoctoral <br />   Fellowship (AS). Some nematode strains used work provided <span id='am-32' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-33'  ></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Caenorhabditis</span> Genetics Center, funded NIH National Center for <br />   Research Resources. The funders role study design, <span id='am-113' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-114' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-116' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-117'  ></span>data</span> collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: anne_hart@brown.edu <br />   . These authors contributed equally work. <br />  <br />  <br />  <br />  <br /> Introduction                                                                          proteins [11&#8211;19]. Roles SMN small nucleolar RNA <br />                                                                                       (snoRNA) microRNA (miRNA) pathways been <br />    Decreased Survival Motor Neuron (SMN) protein function                          suggested [20&#8211;22]. The relative contributions SMN these <br /> underlies Spinal Muscular Atrophy (SMA) cases [1]. The                           various compartments relative importance SMN <br /> SMN protein ubiquitously expressed [2,3], SMA pathology                        function neurons muscles SMA pathology been <br /> is remarkably specific. Patients lose spinal a-motorneurons                       difficult determine. Various tissue requirements SMN <br /> experience muscular dysfunction atrophy. Mild cases result                    function observed different SMA model systems <br /> slowly progressing muscular weakness, severe cases dramat-                      [23&#8211;27]. The diverse subcellular SMN localization varied <br /> ically perturb proximal neuromuscular function resulting                           cellular requirements SMN function suggest protein <br /> childhood death [4]. There effective treatment SMA                          act multiple cellular compartments including the <br /> and 1 40 people US population carriers                      neuromuscular junction (NMJ) [28]. <br /> SMN loss function disease alleles [5&#8211;7].                                              To determine unbiased fashion cellular and <br />    The SMN protein component well-characterized                           molecular pathways particularly relevant SMA pathology, <br /> Gemin complex, assembles splicing machinery eukary-                          researchers turned recently genetic approaches in <br /> otes [8&#8211;10]. SMN associates b-actin mRNA                             vertebrates invertebrates. The identification SMN loss of <br /> anterograde transport neuronal processes suggesting role                     function modifier genes reveal important biochemical <br /> SMN mRNA transport, sub-cellular localization and/or local                         pathways SMA pathology. Studies patients already <br /> translation [11&#8211;15]. In addition, SMN post-synaptic                       identified genes act modifiers SMA: SMN2 and <br /> densities Z-discs muscles RNA processing                      Plastin 3 (PLS3). <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                      1                          October 2010 | Volume 6 | Issue 10 | e1001172 <br />                                                                                                                       Invertebrate Modifiers SMN <br />  <br />  <br />  <br />   Author Summary                                                            conservation species, genes act SMN loss function <br />                                                                             modifiers invertebrates important SMA pathology <br />   Spinal Muscular Atrophy (SMA) common, untreatable,                   humans (e.g. PLS3) [37]. <br />   fatal neuromuscular disease predominately                          In recent study, twenty-seven P-element transposon insertion <br />   caused reduced Survival Motor Neuron (SMN) protein                     lines identified Drosophila modified SMN loss function <br />   function. Here, use invertebrate models identify                defects, role TGF-<span id='am-11' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-15'  ></span><span id='am-16' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>beta</span> pathway SMN loss of <br />   validate conserved genes play critical role SMN                 function pathology delineated [26]. However, remains unclear <br />   loss function neuromuscular defects. Decreased SMN                     P-element lines Drosophila gene near transposon <br />   function causes growth defects nematode Caenor-                    insertion site responsible modulating SMN phenotypic defects. <br />   habditis elegans fruit fly Drosophila melanoga-                The Drosophila P-element lines carried inducible GAL4-UAS that <br />   ster&#8212;as behavioral synaptic connectivity defects <br />                                                                             drive over-expression antisense RNAi expression of <br />   neurons muscles, respectively. We that <br />   genetic modifier SMA patients, plastin, affects              neighboring genes depending transposon insertion site. Addi- <br />   SMN function invertebrate models. We undertook                   tionally, insertion P-element itself perturb gene function. <br />   genome-wide RNAi screen identify genes                         Eliminating ambiguity regarding modifier gene identity would <br />   perturbation alters growth defects C. elegans lacking              increase utility Drosophila study. <br />   SMN. These genes validated neuromuscular assays                      To explore genetic circuitry affecting SMN activity C. <br />   nematode fly models SMA. Additionally,                       elegans, Cesmn-1(lf) growth defect phenotype used a <br />   used C. elegans model test SMN modifier genes                      metric rapid large-scale genetic screen. Growth be <br />   previously identified Drosophila SMA model.                        affected variety changes, body length longevity. <br />   Combined, cross-species approaches identified                       Subsequently, modifier genes tested using C. elegans <br />   genes important species                     behavioral assay, pharyngeal pumping, likely more <br />   SMN function decreased. Related mammalian proteins                     pertinent SMN loss function neuromuscular defects. In <br />   pathways act (including endocytosis                 addition, identify conserved invertebrate SMN modifier genes, <br />   RNA transport/translational control) likely impor-                utilized previously described Drosophila assays assess genetic <br />   tant players SMA.                                                      interaction DmSmn Drosophila orthologs C. elegans modifier <br />                                                                             genes. In study Chang co-workers, DmSmn lethal <br />    Two genes encode human SMN protein: SMN1 SMN2.                       phenotype correlated NMJ defects virtually DmSmn <br /> The SMN1 gene encodes full-length SMN protein                modifier genes, suggesting lethality neuromuscular <br /> SMN2 gene encodes different transcripts; 10% SMN2                    bouton number effective measures genetic interaction with <br /> transcripts encode full-length SMN protein identical               Drosophila SMN ortholog [26]. <br /> SMN1 gene product. However, change splice                      Here, define conserved genetic modifiers SMN loss of <br /> consensus sequence, 90% SMN2 transcripts contain stop                  function using C. elegans Drosophila. We PLS3 <br /> <span id='am-65' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-69'  ></span>codon</span> beginning exon 7 and, therefore, encode                   orthologs act SMN modifier genes invertebrate species. <br /> truncated protein (called SMNdeltaEx7 D7SMN) diminished               A genome-wide RNAi screen C. elegans identified new SMN <br /> function stability [1,29&#8211;31]. Humans various numbers            modifier genes, including ncbp-2 flp-4, modify SMN <br /> SMN2 genes; patients copies SMN2 generally                loss function defects Drosophila. Candidate SMN modifier <br /> later onset/less severe symptoms patients fewer copies         genes identified previous Drosophila screen tested C. <br /> SMN2. Decreased severity delayed onset usually attributed            elegans yielding cross-species modifier genes. Examination of <br /> to increased full-length SMN levels SMN2 vivo [17,32&#8211;36].           literature genes suggested specific cellular pathways <br />    PLS3 modulate severity SMA. In families,              critical genetic modifiers SMN function: endocytosis <br /> daughters lack SMN1 over-express PLS3 remarkably               RNA processing. These pathways pertinent to <br /> unaffected [37]. PLS3 encodes conserved calcium-binding, actin-           SMN loss function defects patients SMA. <br /> bundling/stabilizing protein broadly expressed various <br /> tissues including blood, muscles neurons [38&#8211;40]. Loss           Results <br /> yeast PLS3 ortholog, Sac6p, results defective endocytosis [41,42]. <br /> Altering PLS3 levels modified SMN loss function defects                  The previously described Cesmn-1(ok355) deletion allele causes a <br /> zebrafish motorneurons consistent results human families            complete loss Cesmn-1 function referred as <br /> and PLS3 co-precipitated SMN neuronal tissues [37].               Cesmn-1(lf) [44]. Cesmn-1(lf) recessive; heterozygous animals are <br /> However, increased PLS3 (due profilin knockdown) did                 overtly normal. To facilitate identification heterozygous versus <br /> decrease defects SMA mouse model remains unclear           homozygous animals, utilized balanced strain hT2(bli- <br /> how PLS3 modify SMN neuromuscular defects [43].                       4(e937) let-?(q782) qIs48[myo-2p::GFP])/Cesmn-1(lf) (abbreviated +/ <br />    Modifier genes identified patient populations clearly             Cesmn-1(lf) [44,46]. Heterozygous +/Cesmn-1(lf) animals express <br /> pertinent SMA pathology. However, studies humans                  pharyngeal GFP, homozygous Cesmn-1(lf) progeny express <br /> limited kindred sizes considerations. As SMN                   GFP, progeny homozygous hT2 balancer die GFP- <br /> orthologs C. elegans Drosophila <span id='am-108' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-109'  ></span><span id='am-110' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-112' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span>melanogaster</span>,       expressing embryos. <br /> be efficient identify SMA modifier genes powerful             Although complete loss SMN function causes lethality, C. <br /> invertebrate models. SMN loss function models               elegans homozygous mutant Cesmn-1(lf) survive for <br /> been defined C. elegans Drosophila [18,25,44,45]. Loss            days partial maternal rescue. It suggested <br /> Drosophila Smn (DmSmn) causes larval lethality NMJ defects;             +/Cesmn-1(lf) hermaphrodites load sufficient Cesmn-1 maternal <br /> DmSmn function required neurons muscles flies [26].            protein and/or mRNA oocytes support develop- <br /> Loss C. elegans SMN-1 (Cesmn-1) causes neuromuscular                ment embryogenesis early larval stages [44]. <br /> function deficits followed larval lethality [44]. Expression          Accordingly, homozygous Cesmn-1(lf) larvae initially resemble wild <br /> Cesmn-1 neurons dramatically restores neuromuscular function,            type animals. Eventually maternally-loaded Cesmn-1 product lost; <br /> whereas expression muscles little effect [44]. Given SMN             Cesmn-1(lf) animals grow slowly +/Cesmn-1(lf) siblings, <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                              2                       October 2010 | Volume 6 | Issue 10 | e1001172 <br />                                                                                                                         Invertebrate Modifiers SMN <br />  <br />  <br /> are shorter, sterile, Cesmn-1(lf) animals die reaching        heterozygous populations (Table 1, Rows 1 &amp; 2). Initially, it <br /> adulthood (Figure 1A). Combined, defects decrease                   counter-intuitive defects Cesmn-1(lf) animals are <br /> average size Cesmn-1(lf) population versus control animals;            exacerbated Cesmn-1(RNAi). However, scenario, <br /> decreased average population size referred             transcripts somatic tissues germline +/Cesmn- <br /> growth defect. This growth defect harnessed automated               1(lf) heterozygous animals targeted and, consequently, <br /> assay identify Cesmn-1(lf) modifier genes genome-wide screen.         maternally-loaded Cesmn-1 transcript protein depleted in <br />                                                                               homozygous Cesmn-1(lf) progeny, abrogating partially observed <br /> C. elegans growth assay                                                       maternal rescue. The ability Cesmn-1(RNAi) exacerbate <br />    To validate growth assay SMN modifier gene                       Cesmn-1(lf) defects suggests effects modifier genes be <br /> identification, demonstrated RNAi knockdown                  assessed using RNAi feeding. <br /> Cesmn-1 invertebrate ortholog Plastin 3 (PLS3) altered                 Knockdown C. elegans PLS3 ortholog, plst-1, increased the <br /> Cesmn-1(lf) growth. The C. elegans gene plst-1 (PLaSTin (actin bundling       average length +/Cesmn-1(lf) population, did not <br /> protein) homolog-1) encodes predicted protein similar PLS3.              significantly alter average length Cesmn-1(lf) animals <br />    To knockdown gene function, C. elegans reared bacteria             (Table 1, Rows 1 &amp; 3). Genetic interaction plst-1 further <br /> producing double stranded RNA corresponding gene                    confirmed using plst-1 (tm4255) mutant allele (Table 2). The <br /> interest, strategy known &#8216;feeding RNAi&#8217; [47]. Feeding RNAi               average length +/Cesmn-1(lf);plst-1(tm4255) adult animals was <br /> decreases gene transcripts C. elegans tissues                significantly increased relation +/Cesmn-1(lf) animals. In <br /> knockdown neurons generally effective knockdown               contrast, average length homozygous Cesmn-1(lf);plst- <br /> in muscles, germline, tissues [48&#8211;50]. Here, animals                1(tm4255) altered, recapitulating results plst- <br /> were reared generations solid media RNAi feeding               1(RNAi). Increased average adult length overall growth <br /> bacteria corresponding Cesmn-1 plst-1, allowing knockdown            metric thzat encompass variety changes; decreased plst-1 <br /> maternal zygotic transcripts (Figure 1B). Bacteria containing             function, RNAi mutant allele, increase length, cause <br /> the RNAi feeding vector used negative control                 sterility, and/or increase longevity +/Cesmn-1(lf) control <br /> (empty(RNAi)).                                                                animals. It appears loss Cesmn-1 function suppresses the <br />    An automated used simultaneously measure                     effects decreased plst-1 function (i.e. increased length not <br /> growth determine genotype progeny +/Cesmn-1(lf)                observed Cesmn-1(lf);plst-1(tm4255) homozygous mutant ani- <br /> animals (Figure 1C). The COPAS BioSorter (Union Biometrica,                   mals). The genetic functional relationship SMN and <br /> Holliston, MA) measures C. elegans length &#8216;time-of-flight&#8217;,          PLS3 bears examination; plst-1 Cesmn-1 have <br /> is time required animal pass                       opposing effects growth assay Cesmn-1(lf);plst- <br /> fluorescence-detection chamber [51]. Cesmn-1(lf) homozygous                   1(tm4255) animals resemble Cesmn-1(lf) single mutants, Cesmn-1 <br /> animals express GFP +/Cesmn-1(lf) heterozygous                   act downstream plst-1 growth assay [52]. <br /> animals express GFP longer Cesmn-1 homozygous <br /> animals late larval adult stage. Animals smaller          Growth modifier genes identified C. elegans <br /> the L2 larval stage excluded analysis avoid                 genome-wide screen <br /> bacterial debris. The percentage large adult animals                     To identify additional genes modify SMN loss function <br /> determined genotype RNAi treatment.                              defects, large-scale genome-wide screen enhancers and <br />    RNAi knockdown Cesmn-1 decreased percentage large                suppressors Cesmn-1(lf) growth defect undertaken. The <br /> animals Cesmn-1(lf) homozygous +/Cesmn-1(lf)                      growth assay adapted higher-throughput 96-well, liquid <br />  <br />  <br />  <br />  <br /> Figure 1. Survival average length Cesmn-1(lf) animals decreased. Homozygous loss Cesmn-1(lf) results slower growth and <br /> most animals die larval stages. Animals reach adult stage short-lived sterile. A) Image age-matched heterozygous <br /> hT2(lethal)[myo-2p::GFP]/Cesmn-1(lf) (+/Cesmn-1(lf), described text) homozygous Cesmn-1(lf) individuals (below). Scale bar indicates 150 <br /> microns. B) To test impact candidate modifier genes growth rates size, C. elegans growth assay established. +/Cesmn-1(lf) animals <br /> were collected unhatched eggs, reared 5 days/2 generations standard C. elegans plates RNAi bacterial strains, illustrated flow <br /> chart. Bacterial cultures expressed dsRNA corresponding gene RNAi knockdown; bacterial cultures containing vector no <br /> insert used control (&#8216;empty&#8217;). C) Using automated system, length fluorescence animals determined [51]; smallest <br /> larvae culture indistinguishable debris excluded analysis illustrated graph. Genotypes were <br /> discriminated GFP fluorescence length determined &#8216;time-of-flight&#8217; <span id='am-103' about='obo:NCBITaxon_62990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_241799 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_241789 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-104' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-105' property="rdfs:label" content="Laser" datatype="xsd:string"></span><span id='am-106'  ></span><span id='am-107' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>laser</span> chamber; dot graph represents an <br /> individual animal. The disparate growth/survival rates Cesmn-1(lf) +/Cesmn-1(lf) animals alters percentage &#8216;large&#8217; animals mixed <br /> stage cultures. As illustrated graphically, +/Cesmn-1(lf) Cesmn-1(lf) animals sorted blue red boxes, respectively; animals <br /> designated &#8216;large&#8217; fall shaded boxes. The fraction large animals calculated (# animals shaded box/# animals shaded and <br /> unshaded box genotype) reported % large standard error mean (S.E.M.); significance determined Mann-Whitney <br /> U two-tailed test p#0.05. At independent cultures 200 animals scored targeted gene. <br /> doi:10.1371/journal.pgen.1001172.g001 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                               3                        October 2010 | Volume 6 | Issue 10 | e1001172 <br />                                                                                                                                       Invertebrate Modifiers SMN <br />  <br />  <br />  <br />  Table 1. Conserved genes modify Cesmn-1(lf) defects                                  animals, growth ratio large small animals determined <br />  growth assay.                                                                               clone Cesmn-1(lf) +/Cesmn-1(lf) genotypes. If <br />                                                                                              RNAi clone growth ratio 2 standard deviations <br />                                                                                              away mean Cesmn-1(lf) animals 0.7 <br />  Ce gene          Dm gene            Cesmn-1(lf)               +/Cesmn-1(lf)                 standard deviations mean +/Cesmn-1(lf) animals at <br />                                                                                              40% independent trials, corresponding gene was <br />                                      % large &#177; SEM             % large &#177; SEM                 designated Cesmn-1(lf) modifier (Figure 2B). In primary <br />                            1862                      4661                          high-throughput screen, suppressors found, genes <br />  Cesmn-1          Smn                664*                      3161*                         identified enhancers (Figure 2B). RNAi knockdown of <br />                                                                                              genes exacerbated homozygous Cesmn-1(lf) growth defects <br />  plst-1           Fimbrin            1563                      5563* <br />                                                                                              did significantly alter growth heterozygous +/Cesmn- <br />  uso-1            p115               764*                      4461                          1(lf) animals: ncbp-2, T02G5.3, grk-2, flp-4. ncbp-2 encodes the <br />  nhr-85           Eip75B             865*                      4461                          C. elegans Cap Binding Protein 20 (CBP20 Cbp20) ortholog <br />  egl-15           Breathless         965*                      4561                          [54]. T02G5.3 encodes predicted protein unknown function <br />  atf-6            Atf6               1167*                     4064                          vertebrate orthologs based BLAST analysis. grk-2 <br />                                                                                              encodes G-protein coupled receptor kinases. flp-4 <br />  ape-1            CG18375            1169*                     5062 <br />                                                                                              encodes FMRFamide <span id='am-17' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-18' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> neuropeptide protein. The low <br />  kcnl-2           SK                 1261*                     4261 <br />                                                                                              number modifiers identified screen versus previous <br />  nekl-3           Nek2               1363*                     4362                          Drosophila screen reflect stringent criterion utilized or <br />  atn-1            Actinin            2364*                     4361                          inefficiency RNAi feeding neurons. <br />  cash-1           CG33172            2665                      64611*                           To determine decreased adult body length accounts the <br />  dlc-1            cut             1568                      6663* <br />                                                                                              enhanced Cesmn-1(lf) growth defect knockdown ncbp-2, <br />                                                                                              T02G5.3, grk-2 flp-4, average body length Cesmn-1(lf) <br />  C. elegans modifier genes listed column 1 orthologous Drosophila                 young animals determined (Text S1 Table S4, panel). <br />  genes listed column 2. Only genes significantly altered                     Only ncbp-2(RNAi) significantly reduced average body length of <br />  percentage large Cesmn-1(lf) homozygous animals (column 3) +/Cesmn- <br />                                                                                              Cesmn-1(lf) animals suggesting enhanced growth defect <br />  1(lf) heterozygous animals (column 4) presented. Animals reared for <br />  generations bacterial RNAi strains targeting C. elegans gene prior            caused ncbp-2(RNAi) attributed Cesmn-1(lf) <br />  growth analysis described Figure 1. The percentage large animals               shorter body size. The enhancer genes alter <br />  determined genotype RNAi knockdown indicated C.                       survival growth adult animals. <br />  elegans gene reported standard error mean (S.E.M.). Large                   SMA neuromuscular disease and, therefore, objective was <br />  animals include adults, L4 L3 stage larvae. This assay does distinguish <br />  slow growth, changes relative length, altered viability. Three            identification modifier genes impact SMN neuromuscular <br />  independent determinations undertaken genotype/RNAi                       function. We examined impact Cesmn-1 growth modifier <br />  culture percentage large animals trial averaged.                     genes Cesmn-1 loss function neuromuscular defects using RNAi <br />  Significant changes empty(RNAi) RNAi/genotype using two-                  and, available, loss function alleles modifier genes. <br />  tailed Mann-Whitney U test (p,0.05) indicated asterisk. See Table <br />  S1 results genes tested Materials Methods details. <br />  doi:10.1371/journal.pgen.1001172.t001                                                       RNAi knockdown ncbp-2, grk-2, T02G5.3 modified <br />                                                                                              Cesmn-1(lf) neuromuscular defects <br /> culture format previously described genome-wide C. elegans                                A recent study Sattelle laboratory demonstrated loss <br /> RNAi feeding library used gene knockdown (Figure 2A)                                 Cesmn-1 function causes progressive defects C. elegans <br /> [53]. Progeny +/Cesmn-1(lf) animals reared weeks                             neuromuscular function pharyngeal pumping [44]. C. elegans <br /> (more 2 generations) RNAi feeding bacterial strains                           feeds bacteria microorganisms using small, discrete <br /> assessment growth using COPAS Biosorter [51]. To identify                             subset neurons muscles contained pharynx (Figure 3A) <br /> RNAi clones specifically altered growth Cesmn-1(lf)                              [55]. Pharyngeal cell specification, neuronal development, and <br />                                                                                              myoblast fusion completed hours hatching [56,57]. The <br />                                                                                              pharynx pumps continuously symmetrically 250 beats <br />  Table 2. C. elegans orthologs PLS3, GRK, FMRFamide <br />                                                                                              minute wild type animals food present larval <br />  modify Cesmn-1(lf) growth defects. <br />                                                                                              pumping interrupted molting standard culture <br />                                                                                              conditions. We confirmed previous report [44] early larval <br />                                                                                              stages, pumping rates Cesmn-1(lf) animals indistinguish- <br />  Ce genotype           Ce gene          Cesmn-1(lf)             +/Cesmn-1(lf)) <br />                                                                                              able control animals, later larval stages Cesmn-1(lf) <br />                        RNAi target      % large &#177; SEM           % large &#177; SEM                pumping rates drop (Figure 3B).Cesmn-1(lf) animals progressive <br />                                    2762                    4462                         defects pharyngeal pumping, occur earlier reported <br />                                                                                              locomotion defects. At day 2, 62% Cesmn-1(lf) animals moving <br />  plst-1(tm4255)                    3965                    6762* <br />                                                                                              spontaneously, pumping rates dropped dramatically <br />  grk-2(rt97)                       1061*                   4161                         (Figure 3B). Restoration Cesmn-1 function neurons almost <br />  flp-4(yn35)                       2062                    5162                         completely restores pumping rates suggesting Cesmn-1 is <br />                                                                                              required neurons behavior [44]. <br />  The growth defects +/Cesmn-1(lf) Cesmn-1(lf) animals examined in <br />  presence grk-2(rt97), flp-4(yn35) plst-1(tm4255) loss function alleles;          The efficacy RNAi feeding neuromuscular assay <br />  alleles modifer gene listed column 1. The percentage large               assessed Cesmn-1 plst-1 using Cesmn-1(lf) +/Cesmn- <br />  animals determined Cesmn-1(lf) homozygous animals (column 3)                     1(lf) animals. Animals allowed hatch RNAi feeding <br />  +/Cesmn-1(lf) heterozygous animals (column 4). At independent                   plates pumping rates determined days. Either <br />  determinations undertaken genotype. Significance versus control <br />  genotypes (+/Cesmn-1(lf) Cesmn-1(lf)) determined using Chi-square <br />                                                                                              plst-1(RNAi) Cesmn-1(RNAi) decreased Cesmn-1(lf) pumping <br />  analysis (p,0.05 indicated asterisk). S.E.M. reported.                      rates, +/Cesmn-1(lf) pumping rates (Figure 3C). In <br />  doi:10.1371/journal.pgen.1001172.t002                                                       addition, plst-1(lf) significantly decreased pumping rates of <br />  <br />  <br />           PLoS Genetics | www.plosgenetics.org                                           4                       October 2010 | Volume 6 | Issue 10 | e1001172 <br />                                                                                                                           Invertebrate Modifiers SMN <br />  <br />  <br />  <br />  <br /> Figure 2. Genome-wide RNAi screen Cesmn-1(lf) modifier genes. A) To identify genes knockdown modifies growth Cesmn- <br /> 1(lf) animals, progeny hT2(lethal)[myo-2p::GFP]/Cesmn-1(lf) (+/Cesmn-1(lf), described text) reared 2 generations 96-well, <br /> liquid culture format bacteria expressing dsRNA corresponding 16,500 C. elegans genes. Modifier genes identified assay tested <br /> subsequently neuromuscular assays C. elegans Drosophila. B) Cesmn-1(lf) +/Cesmn-1(lf) length measured RNAi clone a <br /> &#8216;growth&#8217; ratio large:small animals determined genotype. Representative graphs illustrate distribution RNAi clone growth ratios. <br /> Candidate enhancer genes growth ratio 2 standard deviations mean Cesmn-1(lf) (shaded right graph) <br /> and 0.7 standard deviations mean +/Cesmn-1(lf) (shaded left graph) 96-well plate. No suppressors identified using <br /> similar criteria. Two independent determinations clone original screen. Candidate genes retested least <br /> quadruplicate enhanced growth 40% trials designation growth modifier genes; average growth ratios enhancers from <br /> the C. elegans screen indicated. <br /> doi:10.1371/journal.pgen.1001172.g002 <br />  <br /> Cesmn-1(lf) animals, validating genetic interaction plst-1            To test genetic interaction flp-4 Cesmn-1, we <br /> Cesmn-1 neuromuscular pharyngeal pumping assay                          identified flp-4 loss function allele, flp-4(yn35), using PCR <br /> (Figure 3D). This exacerbation Cesmn-1 loss function defects             based screening techniques [61&#8211;65]. The flp-4(yn35) deletion <br /> by plst-1 manipulation consistent results organisms           removes sequences encoding FLP-4 FMRFamide neuropep- <br /> [37]. The ability Cesmn-1(RNAi) plst-1(RNAi) alter                   tides likely causes complete loss flp-4 function ([66] C. <br /> pumping homozygous mutant Cesmn-1(lf) animals suggests                 Li, preparation). Although flp-4(yn35) reduced percentage of <br /> candidate modifier genes assessed using RNAi knockdown                  Cesmn-1(lf) large animals growth assay, difference was <br /> in neuromuscular assay.                                                   statistical significant different (Table 2). Similar results were <br />    The modifier genes C. elegans growth screen              obtained using pharyngeal pumping assay. The pumping rates <br /> tested function Cesmn-1 neuromuscular modifier genes using          flp-4(lf) animals slightly lower, significantly <br /> pharyngeal pumping assay. Results summarized Figure 4A.                 different control animals. Loss flp-4 function decreased <br /> ncbp-2(RNAi) T02G5.3(RNAi) enhanced suppressed                     pumping rates Cesmn-1(lf) animals independent trials, but <br /> pharyngeal pumping defects Cesmn-1(lf) animals, respectively; flp-          difference statistically significant (p = 0.236, Figure 4B). <br /> 4(RNAi) grk-2(RNAi) significant effect compared                  Either flp-4 bona fide modifier flp-4(RNAi) act off- <br /> controls. We suggest ncbp-2 T02G5.3 likely modifiers           target decreasing function 32 C. <br /> Cesmn-1(lf) neuromuscular defects based RNAi results.                       elegans FMRFamide genes [67]. <br />  <br /> Loss grk-2 enhances Cesmn-1(lf) growth                                  Drosophila orthologs plst-1, ncbp-2 flp-4 modify <br /> neuromuscular defects                                                          SMN neuromuscular defects <br />    RNAi knockdown C. elegans genes feeding robust                     SMN modifier genes conserved species be <br /> virtually cell types inefficient result        considerable interest. Three candidate genes identified in <br /> only partial loss gene function, especially nervous           C. elegans screen encode conserved proteins clear orthologs <br /> [58,59]. To address specificity genetic modifiers,              species: grk-2, flp-4, ncbp-2. To determine their <br /> RNAi results confirmed using mutant alleles                       orthologs modify SMN loss function defects, turned the <br /> possible; alleles ncbp-2 T02G5.3 available.                    fruit fly Drosophila. Decreased function Drosophila SMN <br />    A grk-2 loss function allele previously described, grk-         ortholog Smn (DmSmn) results growth defects, early pupal <br /> 2(rt97) [60]. Loss grk-2 significantly enhanced growth defects          arrest, NMJ synaptic defects [26]. We utilized pre-existing <br /> of Cesmn-1(lf) animals (Table 2). Additionally, pumping rates           Drosophila loss function alleles previously described Drosophila <br /> grk-2(lf) animals derived hT2 parents significantly              assays assess genetic interaction DmSmn Drosophila <br /> lower control animals, average pumping rates             orthologs C. elegans modifier genes [18,25,26]. <br /> of Cesmn-1(lf) grk-2(lf) double mutant animals significantly                 First, determined Fimbrin (Fim), Drosophila ortholog of <br /> lower pumping rates single mutant (Figure 4B).              PLS3, modifies DmSmn loss function defects growth and <br /> This suggests grk-2 loss enhances Cesmn-1(lf) growth             NMJ assays. It shown RNAi knockdown DmSmn <br /> pharyngeal pumping defects. A grk-2 gain function allele             (DmSmn RNAi) results 44% lethality early pupal stages with <br /> available transgenes unstable +/Cesmn-1(lf) animals                 56% lethality late pupal stages [26]. Loss Fim does not <br /> (unpublished results [44]).                                                cause larval pupal lethality (data shown). Three Fim loss of <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                5                        October 2010 | Volume 6 | Issue 10 | e1001172 <br />                                                                                                                         Invertebrate Modifiers SMN <br />  <br />  <br />  <br />  <br /> Figure 3. Loss C. elegans PLS3 ortholog enhances pharyngeal pumping defects Cesmn-1(lf) animals. During feeding C. <br /> elegans pharynx contracts 200 times minute capture grind bacteria. A) Pharyngeal pumping rates determined by <br /> videotaping feeding counting contractions movement grinder (indicated arrow) slower playback speeds; scale bar indicates 40 <br /> microns. B) The progeny hT2(lethal)[myo-2p::GFP]/Cesmn-1(lf) (+/Cesmn-1(lf)) animals allowed hatch bacterial strains standard C. <br /> elegans plates. Pumping rates +/Cesmn-1(lf) heterozygous Cesmn-1(lf) homozygous animals determined different days (day 1&#8211;4) a <br /> control &#8216;empty vector&#8217; bacterial strain. Decreased locomotion (sinusoidal movement) scored uncoordinated (Unc). At day 1, pumping <br /> rates locomotion Cesmn-1(lf) animals identical +/Cesmn-1(lf) wild type animals. By day 2, average pumping rates dropped <br /> dramatically (as previously reported [44]) roughly 20% animals uncoordinated (column right). By day 3 4, pumping rates and <br /> locomotion majority surviving animals defective. Expressing Cesmn-1 neurons sufficient restore pumping rates near normal <br /> levels [44]. The behavior Cesmn-1(lf) animals initially normal maternal loading Cesmn-1 gene products, maternal contribution <br /> is depleted, loss Cesmn-1 causes progressive defects neuromuscular function. C) Pumping rates +/Cesmn-1(lf) Cesmn-1(lf) animals were <br /> determined 3 days post-hatching bacterial feeding strains expressing dsRNA corresponding Cesmn-1 plst-1 genes; &#8216;empty vector&#8217; <br /> bacterial cultures used control (&#8216;empty&#8217;). plst-1(RNAi) Cesmn-1(RNAi) significantly descreased Cesmn-1(lf) pumping rates, +/Cesmn- <br /> 1(lf) pharyngeal pumps. D) Decreasing plst-1 function effect pumping rates heterozygous +/Cesmn-1(lf) animals, pumping rates were <br /> significantly decreased Cesmn-1(lf); plst-1(tm4255) double mutant animals. At 20 animals scored genotype 2 <br /> independent trials. Pumping rates reported average animals trials. Because known impact chromosomal pairing on <br /> histone methylation gene expression [96], animals tested progeny animals carrying hT2 balancer chromosome that <br /> prevents recombination. S.E.M. reported. Significance p#0.05 indicated asterisk determined using unpaired two- <br /> sample t-test Mann-Whitney U two-tailed test according sample-specific parameters [100]. <br /> doi:10.1371/journal.pgen.1001172.g003 <br />  <br />  <br /> function alleles crossed DmSmn(RNAi) background                 decreased suggesting loss Fim function exacerbated DmSmn <br /> and accelerated death compared DmSmn(RNAi) control                    loss function defects, consistent studies vertebrate <br /> animals (Figure 5A).                                                          models SMA [37]. These results suggest PLS3 cross- <br />    In Drosophila, loss SMN function results dose-dependant            species modifier SMN function. <br /> decrease process arborization NMJ diminished numbers               Next, Drosophila orthologs candidate SMN modifier genes <br /> of synaptic specializations, termed synaptic boutons [26]. Boutons        C. elegans examined. Cbp20 Fmrf selected as <br /> visualized coincident pre-synaptic synaptotagmin post-                 Drosophila orthologs ncbp-2 flp-4, respectively, based on <br /> synaptic Discs large protein immunoreactivity. The number                  similarity Drosophila loss function alleles obtained. <br /> synaptic boutons Drosophila neurons muscles                 (There 32 genes C. elegans encoding 32 FMRFamide-related <br /> provides simple readily quantifiable assessment phenotypic           neuropeptides, contrast, FMRFamide genes exist in <br /> severity. We determined Drosophila PLS3 ortholog Fim             Drosophila. There redundancy FMRFamide gene <br /> also modify NMJ defects DmSmn. RNAi knockdown                       function Drosophila [68,69,70]). <br /> DmSmn using ubiquitous tubulin promoter (TubGAL4;SmnR-                       Heterozygous loss DmSmn function +/Smn73Ao or <br /> NAi) modestly decreased synaptic innervation Drosophila larvae             +/Smnf01109 animals significant effect bouton number <br /> (reported bouton numbers muscle area, Figure 5B). Loss              expected, Smn73Ao/Smnf01109 animals dramatically <br /> Drosophila Fim function Fimd02114 animals modestly decreased          decreased bouton numbers (Figure 6). Loss copy of <br /> bouton density. We effects Fimd02114 DmSmn                  Cbc20 Fmrf modestly decreased synaptic bouton number <br /> knockdown synergistic; bouton numbers significantly                 compared control animals. However, simultaneous loss one <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                               6                        October 2010 | Volume 6 | Issue 10 | e1001172 <br />                                                                                                                          Invertebrate Modifiers SMN <br />  <br />  <br />  <br />  <br /> Figure 4. C. elegans RNAi screen identifies conserved SMN modifier genes. A) Four genes identified modifiers Cesmn-1(lf) growth <br /> defects C. elegans RNAi screen: ncbp-2, T02G5.3, grk-2 flp-4. The pharyngeal pumping rates homozygous mutant Cesmn-1(lf) and <br /> heterozygous control +/Cesmn-1(lf) animals determined day 3 described Figure 3. B) For C. elegans candidate modifier genes, flp-4 <br /> and grk-2, loss function alleles available. grk-2(rt97) significantly decreased pumping rates Cesmn-1(lf) animals, flp-4(yn35) <br /> lowered Cesmn-1(lf) pumping rates, did reach statistical significance (p = 0.012 p = 0.236 two-tailed Mann-Whitney U test, respectively). <br /> Pumping rates determined day 3 post-hatching; 25 animals scored genotype 3 independent trials. Pumping <br /> rates reported average animals trials. Significance versus Cesmn-1(lf) indicated asterisk. To control genetic <br /> background, animals tested progeny animals carrying balancer chromosome hT2[myo-2p::GFP]; control animals hT2/Cesmn-1(lf) <br /> siblings. <br /> doi:10.1371/journal.pgen.1001172.g004 <br />  <br />  <br /> copy DmSmn copy modifier gene resulted                 significantly alter percentage large heterozygous +/Cesmn- <br /> further synaptic bouton loss (Figure 6). The genetic interaction            1(lf) animals. Knockdown C. elegans ortholog atn-1 significantly <br /> trans-heterozygous animals consistent strong genetic                 suppressed growth defects Cesmn-1(lf) animals altering <br /> interaction Smn modifier genes. We                    percentage large +/Cesmn-1(lf) siblings. Finally, C. elegans <br /> unable obtain classical alleles grk-2 Drosophila ortholog. We        orthologs Drosophila genes identified, genetic <br /> conclude Cbp20 FMRFamide conserved invertebrate                   interaction Cesmn-1 resembled interaction plst-1 with <br /> enhancers Smn loss function defects genetic                Cesmn-1: cash-1 dlc-1. RNAi knockdown genes <br /> interaction conserved species.                                       increased percentage large animals +/Cesmn-1(lf) <br />                                                                                population altering Cesmn-1(lf) population. Growth <br /> Testing Drosophila modifier genes C. elegans assays                         assay results genes Table 1 (Rows 4 <br />    A previous Drosophila screen identified twenty-seven P-element              13), results orthologs tested Table S1, <br /> insertion lines altered Drosophila SMN (DmSmn) loss function           discussion modifier gene function presented Text S2. <br /> defects [26]. Cross-species validation genes help             For bona fide cross-species modifier genes, impact modifier <br /> elucidate conserved pathways critical SMN loss                  genes SMN loss function defects conserved across <br /> function pathology. However, genes flanked P-element               species (i.e. enhancer genes enhance species). For six <br />                                                                                cross-species genes, impact modifier gene loss DmSmn <br /> insertion site modifier lines precise DmSmn <br />                                                                                Cesmn-1 loss function defects conserved expected. <br /> modifier gene unambiguously identified. Therefore, 40 <br />                                                                                Specifically, enhancement Cesmn-1(lf) defects RNAi <br /> candidate modifier genes reported [26]. We identified the <br />                                                                                knockdown nhr-85, egl-15, kcnl-2 consistent the <br /> likely C. elegans orthologs 32 40 genes using reciprocal <br />                                                                                effects corresponding Drosophila modifier genes DmSmn <br /> BLAST similarity searching (Table S1). The ability genes to <br />                                                                                [26]; corresponding Drosophila insertion lines (d09801, f02864, <br /> modify Cesmn-1(lf) growth defects assessed feeding Cesmn-1(lf) <br />                                                                                d03336) enhanced DmSmn defects transposon <br /> and +/Cesmn-1(lf) animals bacteria expressing corresponding <br />                                                                                insertion lines predicted decrease function. The <br /> dsRNA; RNAi feeding clones constructed B0432.13, dhs-22 <br />                                                                                results Drosophila orthologs C. elegans uso-1 nekl-3 were <br /> and ugt-49 [53]. Twelve genes crossed species modified Cesmn- <br />                                                                                consistent species. The exacerbation Cesmn-1(lf) <br /> 1(lf) defects C. elegans assays.                                growth defects observed uso-1(RNAi) nekl-3(RNAi) knock- <br />                                                                                consistent suppression DmSmn defects <br /> Orthologs Drosophila genes modified Cesmn-1(lf)                         observed over-expression cognate Drosophila genes. <br /> growth defects                                                                 There good concordance effect actinin <br />    Knockdown seven C. elegans genes (uso-1, nhr-85, egl-15, atf-6,          orthologs invertebrate species. The d00712 Drosophila <br /> ape-1, kcnl-2 nekl-3) orthologous DmSmn modifier genes                  insertion line likely drives over-expression Drosophila gene <br /> specifically enhanced Cesmn-1(lf) growth defects, did                  Actinin (Actn) enhances DmSmn defects [26,71,72], while <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                7                       October 2010 | Volume 6 | Issue 10 | e1001172 <br />                                                                                                                          Invertebrate Modifiers SMN <br />  <br />  <br />  <br />  <br /> Figure 5. Loss PLS3 orthologs enhances SMN loss function defects invertebrates. Previous studies vertebrates suggested that <br /> PLS3 orthologs C. elegans Drosophila modify SMN loss function defects [37]. A) A genetic interaction the <br /> Drosophila PLS3 ortholog (Fimbrin Fim) DmSmn using previously described DmSmn RNAi knockdown lines [26] Fim loss function <br /> alleles. Percentage early late larval lethality reported. Ubiquitous RNAi knockdown DmSmn using tubulinGAL4 (TubGAL4) driver results in <br /> pupal death; modifier genes alter percentage animals die early versus late pupal stages (day 7 versus day 9 [26]). Loss Fim function <br /> significantly increased percentage animals died early pupae (p#0.05 Chi-square analysis); Fim enhancer DmSmn loss of <br /> function growth/survival defects. At 100 animals genotype scored replicates lethality assay. No significant variation <br /> was observed control experimental crosses independent trial. B) The Drosophila d02114 allele likely eliminates Fim function <br /> and homozygous animals modestly perturbs NMJ morphology larval muscle 4. Fimd02114 enhances DmSmn loss function NMJ defects <br /> consistent studies vertebrates [37]. All strains carry ubiquitous TubGAL4 driver. Significance p#0.05 versus single mutant strains was <br /> determined ANOVA indicated asterisk; S.E.M. shown. In representative images, red corresponds anti-Discs Large (DLG), green is <br /> anti-synaptotagmin blue DAPI; scale bar indicates 15 microns. <br /> doi:10.1371/journal.pgen.1001172.g005 <br />  <br /> suppression Cesmn-1(lf) growth defects RNAi knockdown                 pharyngeal pumping assay. We RNAi knockdown daf-4 <br /> C. elegans atn-1 observed here.                                            enhanced Cesmn-1(lf) pharyngeal pumping defects, knock- <br />    For genes, unclear results Drosophila orthologs       kncl-2 nhr-25 suppressed Cesmn-1(lf) pumping defects <br /> are concordant species: atf-6, ape-1, dlc-1 cash-1. For atf-        (Table 3, rows 3 5). <br /> 6 ape-1, corresponding Drosophila <span id='am-1' about='obo:NCBITaxon_2387' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Transposon" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="transposons" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>transposons</span> (d05057                daf-4 encodes C. elegans TGF-beta receptor subunits <br /> d05779) inserted 1st intron                     orthologous Drosophila Wit (Witless). In C. elegans, daf-4 and <br /> transcripts predicted orthologous Drosophila genes; accord-            TGF-beta/Dpp pathway function required cell specification <br /> ingly, transposons perturb gene function drive                numerous stages transit stress-resistant, <br /> over-expression predicted 2nd shorter transcript. For               long-lived dauer stage [73]. RNAi knockdown daf-4 exacerbated <br /> genes complex genetic interactions Cesmn-1 (i.e. dlc-1           Cesmn-1(lf) pumping defects, consistent effect TGF-beta <br /> cash-1), function Drosophila orthologs ctp CKA likely           pathway manipulation Drosophila [26]. <br /> decreased Drosophila insertion lines f02345 f04448,                 RNAi knockdown C. elegans genes diminished Cesmn-1(lf) <br /> suppressed enhanced DmSmn defects, respectively [26].                      pumping defects: kcnl-2 nhr-25. kcnl-2 encodes likely C. elegans <br /> Overall, genes modified DmSmn growth defects                   SK channel subunit nhr-25 C. elegans proteins <br /> are clearly concordant C. elegans growth data, suggesting             similar Drosophila Usp (Ultraspiracle). No clear ortholog of <br /> conserved roles SMN loss function modifiers.                             Usp C. elegans genome. The corresponding <br />                                                                                Drosophila d00712 transposon insertion line likely drives over- <br /> Orthologs Drosophila genes modified Cesmn-1(lf)                       expression Usp resulting enhancement DmSmn defects [26]. <br /> neuromuscular defects                                                          This consistent C. elegans results. By contrast, impact of <br />   C. elegans orthologs DmSmn modifier genes identified               SK/kcnl-2 loss growth versus pumping assays discordant. The <br /> previous Drosophila screen [26] rescreened using                 d03336 transposon insertion located SK gene, likely <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                8                       October 2010 | Volume 6 | Issue 10 | e1001172 <br />                                                                                                                     Invertebrate Modifiers SMN <br />  <br />  <br />  <br />  <br /> Figure 6. Loss Drosophila Cbp20 FMRF enhances DmSmn loss function neuromuscular defects. For C. elegans modifier <br /> genes, loss function alleles available orthologous Drosophila genes: Cbp20 FMRF [68&#8211;70]. Loss Drosophila gene enhanced <br /> the NMJ defects DmSmn animals. The number synaptic boutons A2 segment muscles 6 7 counted instar larvae <br /> (visualized using Discs large synaptotagmin immunoreactivity, shown red green, respectively). Homozygous loss DmSmn in <br /> DmSmnf1109/DmSmn73Ao animals dramatically decreases bouton number loss copy DmSmn, Cbp20 FMRF little effect. Loss one <br /> copy Cbp20 FMRF animals heterozygous DmSmn allele significantly decreased bouton numbers. This nonallelic <br /> noncomplementation suggests strong genetic interaction conserved modifier genes DmSmn. Scale bar indicates 10 <br /> microns representative images; S.E.M. shown graph; transheterozygous combinations significantly different corresponding <br /> single heterozygous controls p#0.05 ANOVA indicated asterisk. <br /> doi:10.1371/journal.pgen.1001172.g006 <br />  <br />  <br /> perturbs SK function, enhances DmSmn growth Drosophila             A subset cross-species modifier genes interact, directly or <br /> NMJ defects [26]. This consistent kcnl-2(RNAi) enhance-            indirectly, previously described neurological neuromuscular <br /> ment C. elegans growth defects described above. The suppression         disease proteins suggesting common neurodegenerative pathways <br /> of Cesmn-1(lf) pumping defects observed kcnl-2                  work (i.e. ATF6 VAPB/ALS8 GPRK2 and <br /> knockdown reflect differences requirement kcnl-2            SMN1 FMRP) [75&#8211;77]. To determine specific cellular <br /> function neuromuscular tissue and/or relative inefficiency       mechanisms implicated SMN loss function pathology, <br /> RNAi knockdown neurons.                                                 published literature public databases examined for <br />                                                                            physical and/or functional interactions cross-species SMN <br /> Modifier gene interactions implicate specific pathways                     modifier genes, SMN neuromuscular disease genes. A protein/ <br />                                                                            genetic interaction map assembled presented Figure 7 <br /> critical neurodegenerative disease <br />                                                                            references. We note genes implicated endocytosis and <br />    To address specificity invertebrate SMN modifier <br />                                                                            mRNA translational regulation unexpectedly predominate this <br /> genes, impact RNAi knockdown examined an <br />                                                                            interaction map. These cellular pathways pertinent to <br /> unrelated pharyngeal pumping defective strain. Loss egl-30 <br />                                                                            SMN loss function pathology. <br /> (ad805), perturbs Gqa function C. elegans, decreases their <br /> pharyngeal pumping rates [74]. RNAi knockdown did not <br /> significantly alter egl-30 pharyngeal pumping rates modifier <br />                                                                            Discussion <br /> gene (Table S3), suggesting genes likely specific             Enormous effort decades resulted the <br /> modifiers SMN loss function defects.                                 successful identification numerous neurodegenerative disease <br />    Combined results described define conserved             genes proteins encode. However, cases there <br /> genes modify invertebrate SMN ortholog function           remains considerable controversy perturbation these <br /> one assay C. elegans Drosophila (summarized Table 4).       genes results neurodegeneration [78&#8211;81]. SMN plays well- <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                            9                       October 2010 | Volume 6 | Issue 10 | e1001172 <br />                                                                                                                                        Invertebrate Modifiers SMN <br />  <br />  <br />  <br />  Table 3. RNAi knockdown candidate genes alters Cesmn-1(lf) neuromuscular defects pharyngeal pumping assay. <br />  <br />  <br />                                                    Cesmn-1(lf)                    +/Cesmn-1(lf)                 Cesmn-1(lf)                  Cesmn-1(lf) <br />  <br />                                                    % change vs. control RNAi                                    empty(RNAi)                  candidate(RNAi) <br />  Ce gene                  Dm gene                                                                               Pumps/min <br />  <br />  Cesmn-1                  Smn                      56619*                         10162                         5369                         2766 <br />  plst-1                   Fim                      86621*                         10661                         5369                         4368 <br />  daf-4                    Wit                      71627*                         9362                          3867                         2666 <br />  kcnl-2                   SK                       147650*                        9061                          4267                         4868 <br />  nhr-25                   Usp                      155645*                        9165                          4868                         67611 <br />  <br />  C. elegans orthologs DmSmn modifier genes RNAi knockdown altered Cesmn-1(lf) pharyngeal pumping defects listed column 1 column 2, <br />  respectively. The percent (%) change pumping rates vector RNAi versus candidate gene RNAi determined independently separate experiments <br />  reported S.E.M. Cesmn-1(lf) homozygous mutant +/Cesmn-1(lf) heterozygous control animals (columns 3 4). As pumping rates each <br />  genotype/treatment varied day-to-day (due food thickness, etc.), % change versus vector RNAi controls reported used determine significance. <br />  The overall pharyngeal pumping rates columns 5 6 calculated pooling Cesmn-1(lf) animals reared RNAi bacteria indicated (n.40 animals). <br />  Animals allowed hatch bacterial cultures carrying RNAi vector (empty) RNAi constructs targeting C. elegans genes; pumping rates determined <br />  3 days early adult late L4 larval stage illustrated Figure 2. Significance (indicated asterisk) determined two-sample t-test/Mann- <br />  Whitney U test according sample-specific parameters (p#0.05). Two additional genes modified growth CG33172/cash-1 CG18375/ape-1 (Table S2), reached <br />  significance pair-wise comparisons controls. However, knockdown enhanced Cesmn-1(lf) pumping defects trials suppressed trials <br />  resulting significant change overall pumping rates. See Table S2 results genes tested Materials Methods details. <br />  doi:10.1371/journal.pgen.1001172.t003 <br />  <br />  <br /> described ubiquitous role Gemin complex snRNP                            suggested invertebrate modifier screens have <br /> assembly [8&#8211;10], SMA specifically affects neuromuscular                             identified conserved human disease-related genes and/or func- <br /> function, motorneuron survival, leads muscle atrophy.                            tional pathways [83&#8211;88]. <br /> Given neuromuscular specificity, likely loss                         Thus far, published human SMA modifier <br /> SMN function impacts cellular pathways outside Gemin                             genes: Plastin 3 (PLS3) SMN2. The role SMN2 clear it <br /> complex. In addition, given complexity cellular signaling                        provides modicum functional SMN protein. However, the <br /> pathways, genetic pathways directly involved SMN                        role PLS3 SMA controversial unclear PLS3 <br /> activity impact SMN loss function pathology. To identify                         levels modulate severity SMA patients [37,43]. We find <br /> SMN modifier pathways, used genetic approach.                                 invertebrate PLS3 orthologs act modifiers C. elegans and <br /> Unbiased genetic screens powerful tools utilize                             Drosophila models. This cross-species interaction PLS3 SMN <br /> functional criteria identification genes critical cellular               increases confidence invertebrate models suggests <br /> function. In case SMN loss function, genetic screens                      plastin-associated pathways important SMN function <br /> reveal conserved genes pathways important                              fundamental level multiple contexts. <br /> neuromuscular dysfunction pathology independent initial                             In bioinformatic analysis presented Figure 7, we <br /> assumptions roles SMN neurons muscles. The                          independently identified cellular pathways connect multiple <br /> identification hitherto unsuspected molecular pathways                          modifier genes SMN: endocytosis RNA processing/ <br /> modulate SMN neuropathology, directly indirectly, expected                        translational control pathways. Regarding former, note <br /> to widen range targets SMA therapy development.                              yeast ortholog PLS3, Sac6p, key player endocytosis <br />    Conserved genes modify SMN loss function defects                          Sac6p levels critical expanded polyglutamine <br /> disparate species likely represent pathways important                      neurodegenerative disease proteins expressed system <br /> SMN loss function defects pathology. C. elegans Drosophila                    [42,89&#8211;91]. We suggest 1) cellular mechanisms may <br /> models used identify SMN loss function                             particular importance SMA pathology 2) that <br /> modifier genes &#8216;cross species&#8217;. It difficult estimate                    unexpected intimate connections exist two <br /> many modifiers SMN loss function missed                               pathways. A pair recently published studies the <br /> genome-wide C. elegans RNAi screen. &#8216;Growth&#8217; encompasses                              microRNA regulatory RISC complex endocytosis physically <br /> variety factors; slow progression larvae stages,                         functionally coupled non-neuronal cells [92,93]. Interestingly, <br /> reduced growth adult stage, longevity, body size, different                      RISC complex contains Gemin complex proteins; the <br /> culture format (liquid versus plates), combination these.                       function Gemin RISC complexes related, directly <br /> Additionally, identification genetic modifiers null allele                 indirectly [20]. We speculate normal animals, physically <br /> be challenging compared identification genetic                            coupling seemingly disparate pathways endocytosis local <br /> modifiers partial loss function alleles [82]. No genetic screen                  translational regulation help coordinate synaptic activity and <br /> can identify modifier genes pertinent pathway                              receptor signaling protein translation synaptic <br /> important players missed (e.g. miRNAs). Despite this,                      development neuron maintenance [94]. Defects endocytosis <br /> is excellent concordance modifier gene action C. elegans                      suggested previously play pivotal role in <br /> Drosophila. In cases, genes enhanced SMN loss                              neurodegenerative diseases numerous scenarios. In diseases, <br /> function defects Drosophila enhanced SMN loss function                       including SMA, perturbation endocytosis result RNA <br /> defects C. elegans vice versa. This concordance suggests                    translational control defects, vice versa [92,93]. A recent study has <br /> the genetic relationships SMN modifier genes                          demonstrated impaired synaptic vesicle release NMJs severe <br /> are conserved species. Orthologous genes likely                         SMA mice consistent defects synaptic vesicle endocytosis/ <br /> important SMN loss function pathology vertebrate species,                      recycling and/or defects active zone organization [94]. Further <br />  <br />  <br />           PLoS Genetics | www.plosgenetics.org                                     10                          October 2010 | Volume 6 | Issue 10 | e1001172 <br />                                                                                                                                   Invertebrate Modifiers SMN <br />  <br />  <br />  <br />  Table 4. Summary: Invertebrate modifiers SMN loss                                 animals tested progeny hT2(lethal)[myo- <br />  function.                                                                               2p::GFP] parents. The use RNAi sensitive C. elegans mutant <br />                                                                                          strains avoided behavior normal assays <br />                                                                                          (Hart, unpublished observations) SMN complex/Sm <br />                                                                       Change/            proteins implicated miRNA pathways [20,97,98]. <br />  Ce         Dm        Hs                            Change/           Affect             We note RNAi knockdown effective. To control <br />  gene       gene      gene                          Affect Ce         Dm                 genetic background effects, animals tested studies were <br />  plst-1     Fim       Plastin 3 (PLS3)              RNAi/Cmpx Gr,     Lof/Enh            heterozygous hT2 balancer chromosome progeny of <br />                                                     Enh Pump                             hT2 parents unless noted. <br />                                                                                             plst-1(tm4255) animals obtained Japanese <br />  daf-4      Wit       TGFb receptor (BMPRII)        RNAi/Enh Pump Lof/Enh <br />                                                                                          National Bioresource Project backcrossed times <br />  kcnl-2     SK        SK channel (KCNN3)            RNAi/Enh Gr,      Lof/Enh            study. The tm4255 allele 368 base pair deletion <br />                                                     Sup Pump? <br />                                                                                          removes calponin-like, actin-binding homology <br />  nhr-25     Usp       NHR LRH-1 (NR5A2)             RNAi/Sup Pump OE/Enh                 (CH) domains; plst-1(tm4255) likely partial loss function <br />  uso-1      p115      Vesicle docking (USO1)        RNAi/Enh Gr       OE/Sup             allele. To test genetic interaction plst-1 Cesmn-1, the <br />  nhr-85     Eip75B    NHR RevErb (NR1D2)            RNAi/Enh Gr       Lof/Enh            backcrossed plst-1(tm4255) allele used create double <br />  atf-6      Atf6      Atf6 trans. factor (ATF6)     RNAi/Enh Gr       ?/Sup              mutant Cesmn-1(lf). The flp-4(yn35) deletion allele isolated <br />  egl-15     Btl       FGF receptor (FGFR3)          RNAi/Enh Gr       Lof/Enh <br />                                                                                          PCR-based screening EMS-mutagenized animals. The yn35 <br />                                                                                          allele 928 base pair deletion removes exon 3 flp-4 gene <br />  ape-1      CG18375 p53 inhibition (PPP1R13)        RNAi/Enh Gr       ?/Enh <br />                                                                                          59 sequences (flanking sequences, ttctgaaaaacttttaataa <br />  nekl-3     Nek2      NIMA family kinase (NEK7)     RNAi/Enh Gr       OE/Sup             agctcgccgagccgagtctt) [66]. The grk-2(rt97) loss function <br />  atn-1      Actinin   a-actinin (ACTN)              RNAi/Sup Gr       OE/Enh             allele previously characterized [60]. <br />  cash-1     CG33172 Striatin (STRN)                 RNAi/Cmpx Gr      Lof/Enh               Drosophila stocks maintained standard cornmeal/yeast/ <br />  dlc-1      cut    Dynein light chain (DYNLL2)   RNAi/Cmpx Gr      Lof/Sup            molasses/agar medium 25uC. The mutations Smn73Ao and <br />                                                                                          Smnf01109 described previously [25]. Cbp20e02787 a <br />  ncbp-2     CBC20     Cap binding (CBP20)           RNAi/Enh Gr &amp;     Lof/Enh            Piggy-Bac insertion mutation Exelixis collection. The <br />                                                     Pump                                 insertion location 59 upstream adjacent start codon of <br />  grk-2      Gprk      GRK kinase (ADRBK1)           RNAi/Enh Gr &amp;     n.d.               Cbp20 transcript. FmrfKG1300 Fim alleles loss of <br />                                                     Pump                                 function alleles (Flybase). The line d03334 unlinked <br />  flp-4      FMRF      Neuropeptide (NPFF)           RNAi/Enh Gr       Lof/Enh            lethal mutation chromosome. Fimd02114 and <br />  T02G5.3                                  RNAi/Enh Gr,      n.d.               SmnRNAi; Fimd02114 Tubulin:Gal4 background; this <br />                                                     Sup Pump                             Gal4 transgene does alter Smn defects (data shown). <br />  The summary table divided sections corresponding potential <br />  Cesmn-1(lf) modifier genes originally identified humans, Drosophila, C.           Bioinformatics <br />  elegans (1 Hs, 40 Dm, 4 Ce, respectively). Species gene names             C. elegans orthologs Drosophila human genes were <br />  listed columns. The change gene expression             identified BLAST searching NCBI. When clear ortholog <br />  Drosophila GAL4-driven transposon insertion lines C. elegans RNAi <br />                                                                                          identified reciprocal BLAST analysis, similar <br />  knockdown listed columns 4 5; effect Drosophila gene function is <br />  predicted based solely transposon insertion sites cases. (RNAi: RNAi         C. elegans genes generally tested. plst-1 corresponds exons of <br />  knockdown; Lof: loss function; decreased function antisense; OE: over-            predicted adjacent genes Y104H12BR.1 Y104H12BL.1 based <br />  expression; ?: unclear; Enh: enhanced SMN loss function defects; Sup:                similarity searching. T02G5.3 corresponds exons T02G5.3, <br />  suppressed SMN loss function defects, Cmpx: complex genetic interaction; <br />                                                                                          T02G5.2, T02G5.1 based high-throughput cDNA <br />  determined, n.d.). Column 4 includes C. elegans assay revealing modifier <br />  activity (Gr: growth; Pump: pharyngeal pumping.) RNAi knockdown                     sequencing gene prediction programs [99]. New gene <br />  sufficient modify pumping rates T02G5.3 ncbp-2; mutant alleles            predictions reported Wormbase. To assemble the <br />  grk-2 exacerbated Cesmn-1(lf) pumping defects. flp-4 loss decreased Cesmn-1(lf)         interaction map Figure 7, literature pertaining modifier <br />  pumping rates 4 5 trials, did reach significance overall. <br />                                                                                          gene examined NCBI, AceView, C. elegans yeast on-line <br />  Discordance Drosophila C. elegans results indicated ?. <br />  doi:10.1371/journal.pgen.1001172.t004                                                   databases (Wormbase SGD) identify functional direct <br />                                                                                          interactions modifier genes neurodegenerative <br /> studies warranted ascertain interdependence endocy-                        disease genes. <br /> tosis translational control pathways explore relevance <br /> of pathways neurodegenerative disease.                                          Construction RNAi feeding clones B0432.13, dhs-22 <br />                                                                                          ugt-49 <br /> Materials Methods                                                                       The L4440 vector [47] used clone PCR products <br /> Genetic analysis                                                                         corresponding B0432.13, dhs-22 ugt-49 genes. <span id='am-128' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-129' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-132' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>Plasmids</span> were <br />    Cesmn-1(lf) homozygous strains maintained                            transformed bacterial strain HT115(DE3) [47,49]. Primers <br /> infertility; hT2(lethal)[myo-2p::GFP]/Cesmn-1(lf) (hT2[bli-4(e937) let-                  used cloning were: B0432.13 forward 59-acaagctctcgacatcgctg- <br /> ?(q782)qIs48] (I;III)) animals fertile maintained using                     39, reverse 59- ttaatcgccgcatcctcttg -39; dhs-22 forward 59-tatgctgtg- <br /> standard techniques [95]. The lack homologous pairing                         cagaagcgaag-39, reverse 59-ctgcttgattcctggtgtattc-39; ugt-49 forward <br /> rearranged chromosomes LGI LGIII hT2 animals likely                               59-acgtggatgtagctgaatgg-39, reverse 59- acgtgaagaacagcaacgaac-39. <br /> results increased maternal/zygotic expression Cesmn-1 and <br /> other balanced genes [96]. As expected, progeny                        C. elegans growth studies <br /> of hT2 animals relatively resistant Cesmn-1(RNAi)                                   For analysis modifier genes, animals reared two <br /> compared wild type control strains assays (data                            generations/5 days plates spread bacterial RNAi strains <br /> shown). Consequently, genetic background invariant,                          Ahringer Vidal RNAi libraries [53]. RNAi clones <br />  <br />  <br />           PLoS Genetics | www.plosgenetics.org                                      11                       October 2010 | Volume 6 | Issue 10 | e1001172 <br />                                                                                                                            Invertebrate Modifiers SMN <br />  <br />  <br />  <br />  <br /> Figure 7. Interaction map SMN modifier genes. A) Numerous published physical and/or functional interactions literature <br /> that connect invertebrate SMN modifier genes. The names vertebrate genes used figure, interactions drawn literature <br /> in animal species. Each type interaction represented different colors connecting lines references provided table B. <br /> Not interactions indicated including interactions FGF, TGF-beta neuropeptide signaling pathways endocytosis/cytoskeletal <br /> pathways, dynein interactions APP [101] possible connection VAPB, ALS8 locus, SK channels riluzole [102&#8211;106]. <br /> The interaction analysis exhaustive additional interactions exist. B) The type interaction indicated table [107&#8211;155]: P <br /> pull-down, Y yeast 2-<span id='am-123' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-124' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-125'  ></span><span id='am-126' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>hybrid</span>, F functional/genetic, D direct physical interaction. Genes shaded blue cross-species invertebrate modifiers; genes <br /> shaded yellow mammalian genes perturbation cause neurodegeneration; genes outlined grey SMA modifiers drug studies <br /> or patients. Genes pertinent endocytosis include CIN85, EndoA, Alix, PLS3, CAV1, GPRK, Ataxin2, Profilin, EPS15, Phocein, Dynamin, Striatin; <br /> genes pertinent RNA processing include PABP, CBC20, DcpS, SMN, FMRP. Combined, results published studies support model in <br /> which endocytosis RNA translational control pathways physically functionally coupled [92,93]. In normal animals, coupling these <br /> seemingly disparate pathways help coordinate synaptic activity protein synthesis neuromuscular system. <br /> doi:10.1371/journal.pgen.1001172.g007 <br />  <br /> corresponding modifier genes Table 4 sequenced                    genotype [51]. Three independent determinations were <br /> confirm accuracy. The hlh-4(RNAi) clone feeding library              undertaken genotype/RNAi culture. Significant changes <br /> incorrect. A Cesmn-1(ok355);hlh-4(tm604) double mutant strain               empty(RNAi) calculated RNAi/genotype using <br /> generated. hlh-4(tm604) did affect pharyngeal pumping                   two-tailed Mann-Whitney U test. <br /> rates Cesmn-(lf) (data shown) hlh-4 excluded from <br /> further analysis. Length GFP fluorescence determined                    C. elegans pharyngeal pumping <br /> using COPAS Biosorter (Union Biometrica, Holliston, MA)                       The average pharyngeal pumping rates animals were <br /> and percentage large animals determined                     determined 3 days (at 25, 25 20uC) post-hatching on <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                12                        October 2010 | Volume 6 | Issue 10 | e1001172 <br />                                                                                                                                               Invertebrate Modifiers SMN <br />  <br />  <br /> empty vector (empty(RNAi)) candidate gene RNAi bacterial                                larvae dissected fixed 5 minutes Bouin&#8217;s fixative. <br /> feeding strains. Animals videotaped feeding                                 Stained specimens mounted FluoroGuard Antifade <br /> 10 seconds AxioCam ICc1 <span id='am-60' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-63'  ></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>camera</span> Zeiss Stemi                                    Reagent (Bio-Rad), images obtained Zeiss <br /> SV11 20 666 magnification. Movies slowed                                 LSM510 confocal microscope. Bouton numbers counted <br /> counting pumping rates. Pharyngeal grinder movements                                based Discs large Synaptotagmin staining A2 <br /> <span id='am-70' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-71' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-72'  ></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>axis</span> scored pumping event. Average pumping rates (6                              segment muscles 6 7 muscle 4 indicated. The <br /> standard error mean, S.E.M) genotype/treatment                             ratio muscle area various genotypes normalized to <br /> were calculated independently separate experi-                              wild type. At 10&#8211;12 animals genotype dissected <br /> ments. The percent change pumping rate vector versus                           bouton analysis. The ANOVA multiple comparison test <br /> candidate gene RNAi determined trial                                 used statistical analysis. <br /> Cesmn-1(lf) homozygous +/Cesmn-1(lf) heterozygous animals <br /> and used calculate mean, S.E.M, significance.                                   Supporting Information <br />  <br /> C. elegans genome-wide RNAi screen                                                         Table S1 Summary C. elegans growth assays. <br />    hT2(bli-4(e937) let-?(q782) qIs48[myo-2p::GFP]) (I;III) animals                         Found at: doi:10.1371/journal.pgen.1001172.s001 (0.09 MB <br /> were reared liquid cultures 96-well plate format RNAi                           DOC) <br /> feeding strains [53]. At independent cultures corre-                             Table S2 Summary C. elegans pharyngeal pumping assays. <br /> sponding C. elegans RNAi feeding clone established.                           Found at: doi:10.1371/journal.pgen.1001172.s002 (0.09 MB <br /> Concentrated dsRNA expressing bacteria added cultures                            DOC) <br /> necessary prevent starvation. Cultures maintained 8 <br /> days 25uC generate sufficient animals analysis. Length                           Table S3 Invertebrate modifier genes specifically affect Cesmn- <br /> and fluorescence determined using COPAS BioSorter                                 1(lf) pharyngeal pumping defects. <br /> (Union Biometrica, Holliston, MA). Data exported Excel                              Found at: doi:10.1371/journal.pgen.1001172.s003 (0.05 MB <br /> (Microsoft Corp.) analysis. Thirty-one clones identified                          DOC) <br /> that modified average length Cesmn-1(lf) animals relative                        Table S4 Body length determinations Cesmn-1(lf) modifier <br /> +/Cesmn-1(lf) siblings trials. Four genes altered                         genes. <br /> Cesmn-1(lf) size relative +/Cesmn-1(lf) siblings 40%                     Found at: doi:10.1371/journal.pgen.1001172.s004 (0.05 MB <br /> subsequent trials selected candidate modifier                            DOC) <br /> genes neuromuscular analysis described text. <br />                                                                                            Text S1 Methods supplementary files. <br />                                                                                            Found at: doi:10.1371/journal.pgen.1001172.s005 (0.02 MB <br /> Drosophila pupal lethality <br />                                                                                            DOC) <br />    Three males virgin females placed fresh food <br /> at 25uC day 1. Eggs collected 2 days (Set 1),                         Text S2 Supplementary discussion invertebrate modifier <br /> the parents transferred fresh food. Eggs collected                             genes. <br /> another 2 days (Set 2), parents discarded. The F1 animals                          Found at: doi:10.1371/journal.pgen.1001172.s006 (0.33 MB <br /> were scored 15 days,- 16th day set,                     DOC) <br /> 19th day second set, day 1. White pupae scored <br /> as early stage death black pupae scored late stage                             Acknowledgments <br /> death. Control crosses tubGAL4:FL26B(Smn RNAi) out- <br /> crossed wild type strain used control                           We grateful advice and/or reagents C. elegans knockout <br />                                                                                            consortia numerous members C. elegans community including <br /> experimental set. Significance determined Chi-square <br />                                                                                            Prof. D. B. Sattelle sharing information Cesmn-1(lf) pumping defects <br /> analysis.                                                                                  prior publication. Some nematode strains used work were <br />                                                                                            provided Caenorhabditis Genetics Center. <br /> Drosophila NMJ analysis <br />   Primary antibodies used following dilutions:                                 Author Contributions <br /> monoclonal anti-DLG (1:500) (Developmental Studies Hybridoma <br />                                                                                            Conceived designed experiments: MD JNS AW HCC AS GK JH <br /> Bank), polyclonal anti-Synaptotagmin (1:1000) (a gift Hugo <br />                                                                                            TB MBW JSS CL DVV SAT ACH. Performed experiments: MD JNS <br /> Bellen). FITC- (1:40) Cy5- (1:40) conjugated anti-rabbit                           AW HCC AS GK JH TB MBW JSS SAT ACH. Analyzed data: MD <br /> anti-mouse secondary antibodies purchased Jackson                                JNS AW HCC AS GK JH TB MBW JSS DVV SAT ACH. Contributed <br /> Immunoresearch Laboratories. Anti-Disc large used 1:100                                 reagents/materials/analysis tools: CL DVV. Wrote paper: MD JNS <br /> (Hybridoma) anti-HRP used 1:1000 (Cappell). 3rd instar                              AW HCC AS ACH. <br />  <br /> References <br />   1. Lefebvre S, Burglen L, Reboullet S, Clermont O, Burlet P, et al. (1995)                 6. McAndrew PE, Parsons DW, Simard LR, Rochette C, Ray PN, et al. (1997) <br />      Identification characterization spinal muscular atrophy-determining               Identification proximal spinal muscular atrophy carriers patients analysis <br />      gene. Cell 80: 155&#8211;165.                                                                    SMNT SMNC gene copy number. Am J Hum Genet 60: 1411&#8211;1422. <br />   2. Boda B, Mas C, Giudicelli C, Nepote V, Guimiot F, et al. (2004) Survival motor          7. Ogino S, Leonard DG, Rennert H, Ewens WJ, Wilson RB (2002) Genetic risk <br />      neuron SMN1 SMN2 gene promoters: identical sequences differential                  assessment carrier testing spinal muscular atrophy. Am J Med Genet 110: <br />      expression neurons non-neuronal cells. Eur J Hum Genet 12: 729&#8211;737.                 301&#8211;307. <br />   3. Monani UR, McPherson JD, Burghes AH (1999) Promoter analysis                     8. Gubitz AK, Feng W, Dreyfuss G (2004) The SMN complex. Exp Cell Res 296: <br />      human centromeric telomeric survival motor neuron genes (SMNC                      51&#8211;56. <br />      SMNT). Biochim Biophys Acta 1445: 330&#8211;336.                                              9. Ohn T, Kedersha N, Hickman T, Tisdale S, Anderson P (2008) A functional <br />   4. Pearn J (1980) Classification spinal muscular atrophies. Lancet 1: 919&#8211;922.             RNAi screen links O-GlcNAc modification ribosomal proteins stress <br />   5. Cusin V, Clermont O, Gerard B, Chantereau D, Elion J (2003) Prevalence                  granule processing body assembly. Nat Cell Biol 10: 1224&#8211;1231. <br />      SMN1 deletion duplication carrier normal populations: implication           10. Hua Y, Zhou J (2004) Survival motor neuron protein facilitates assembly of <br />      genetic counselling. J Med Genet 40: e39.                                              stress granules. FEBS Lett 572: 69&#8211;74. <br />  <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                           13                            October 2010 | Volume 6 | Issue 10 | e1001172 <br />                                                                                                                                                Invertebrate Modifiers SMN <br />  <br />  <br /> 11. Fan L, Simard LR (2002) Survival motor neuron (SMN) protein: role neurite              37. Oprea GE, Krober S, McWhorter ML, Rossoll W, Muller S, et al. (2008) <br />     outgrowth neuromuscular maturation neuronal differentiation                    Plastin 3 protective modifier autosomal recessive spinal muscular <br />     development. Hum Mol Genet 11: 1605&#8211;1614.                                                     atrophy. Science 320: 524&#8211;527. <br /> 12. Rossoll W, Jablonka S, Andreassi C, Kroning AK, Karle K, et al. (2003) Smn,               38. Bretscher A (1981) Fimbrin cytoskeletal protein crosslinks F-actin in <br />     spinal muscular atrophy-determining gene product, modulates axon growth                   vitro. Proc Natl Acad Sci U S A 78: 6849&#8211;6853. <br />     localization beta-actin mRNA growth cones motoneurons. J Cell                39. Bretscher A, Weber K (1980) Fimbrin, new microfilament-associated protein <br />     Biol 163: 801&#8211;812.                                                                            present microvilli cell surface structures. J Cell Biol 86: 335&#8211;340. <br /> 13. Francis JW, Sandrock AW, Bhide PG, Vonsattel JP, Brown RH, Jr. (1998)                     40. Glenney JR, Jr., Kaulfus P, Matsudaira P, Weber K (1981) F-actin binding and <br />     Heterogeneity subcellular localization electrophoretic mobility                     bundling properties fimbrin, major cytoskeletal protein microvillus core <br />     survival motor neuron (SMN) protein mammalian neural cells tissues.                    filaments. J Biol Chem 256: 9283&#8211;9288. <br />     Proc Natl Acad Sci U S A 95: 6492&#8211;6497.                                                   41. Adams AE, Botstein D, Drubin DG (1989) A yeast actin-binding protein is <br /> 14. Giavazzi A, Setola V, Simonati A, Battaglia G (2006) Neuronal-specific roles               encoded SAC6, gene suppression actin mutation. Science <br />     survival motor neuron protein: evidence survival motor neuron                        243: 231&#8211;233. <br />     expression patterns developing human central nervous system.                       42. Kubler E, Riezman H (1993) Actin fimbrin required the <br />     J Neuropathol Exp Neurol 65: 267&#8211;277.                                                         internalization step endocytosis yeast. EMBO J 12: 2855&#8211;2862. <br /> 15. Pagliardini S, Giavazzi A, Setola V, Lizier C, Di Luca M, et al. (2000)                   43. Bowerman M, Anderson CL, Beauvais A, Boyl PP, Witke W, et al. (2009) <br />     Subcellular localization axonal transport survival motor neuron                    SMN, profilin IIa plastin 3: link deregulation actin <br />     (SMN) protein developing rat spinal cord. Hum Mol Genet 9: 47&#8211;56.                      dynamics SMA pathogenesis. Mol Cell Neurosci 42: 66&#8211;74. <br /> 16. Gennarelli M, Lucarelli M, Capon F, Pizzuti A, Merlini L, et al. (1995) Survival          44. Briese M, Esmaeili B, Fraboulet S, Burt EC, Christodoulou S, et al. (2009) <br />     motor neuron gene transcript analysis muscles spinal muscular atrophy                 Deletion smn-1, Caenorhabditis elegans ortholog spinal muscular <br />     patients. Biochem Biophys Res Commun 213: 342&#8211;348.                                            atrophy gene, results locomotor dysfunction reduced lifespan. Hum Mol <br /> 17. Coovert DD, Le TT, McAndrew PE, Strasswimmer J, Crawford TO, et al.                           Genet 18: 97&#8211;104. <br />     (1997) The survival motor neuron protein spinal muscular atrophy. Hum                  45. Miguel-Aliaga I, Culetto E, Walker DS, Baylis HA, Sattelle DB, et al. (1999) <br />     Mol Genet 6: 1205&#8211;1214.                                                                       The Caenorhabditis elegans orthologue human gene responsible spinal <br /> 18. Rajendra TK, Gonsalvez GB, Walker MP, Shpargel KB, Salz HK, et al. (2007)                     muscular atrophy maternal product critical germline maturation and <br />     A Drosophila melanogaster model spinal muscular atrophy reveals function             embryonic viability. Hum Mol Genet 8: 2133&#8211;2143. <br />     SMN striated muscle. J Cell Biol 176: 831&#8211;841.                                         46. McKim KS, Peters K, Rose AM (1993) Two types sites required meiotic <br /> 19. Walker MP, Rajendra TK, Saieva L, Fuentes JL, Pellizzoni L, et al. (2008) The                 chromosome pairing Caenorhabditis elegans. Genetics 134: 749&#8211;768. <br />     SMN complex localizes sarcomeric Z-disc proteolytic target             47. Timmons L, Fire A (1998) Specific interference ingested dsRNA. Nature <br />     calpain. Hum Mol Genet.                                                                       395: 854. <br /> 20. Mourelatos Z, Dostie J, Paushkin S, Sharma A, Charroux B, et al. (2002)                   48. Kamath RS, Martinez-Campos M, Zipperlen P, Fraser AG, Ahringer J (2001) <br />     miRNPs: novel <span id='am-37' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> ribonucleoproteins containing numerous microRNAs.                    Effectiveness specific RNA-mediated interference ingested double- <br />     Genes Dev 16: 720&#8211;728.                                                                        stranded RNA Caenorhabditis elegans. Genome Biol 2: RESEARCH0002. <br /> 21. Pellizzoni L, Baccon J, Charroux B, Dreyfuss G (2001) The survival motor               49. Timmons L, Court DL, Fire A (2001) Ingestion bacterially expressed <br />     neurons (SMN) protein interacts snoRNP proteins fibrillarin                      dsRNAs produce specific potent genetic interference Caenorhabditis <br />     GAR1. Curr Biol 11: 1079&#8211;1088.                                                                elegans. Gene 263: 103&#8211;112. <br /> 22. Whitehead SE, Jones KW, Zhang X, Cheng X, Terns RM, et al. (2002)                         50. Rual JF, Ceron J, Koreth J, Hao T, Nicot AS, et al. (2004) Toward improving <br />     Determinants interaction spinal muscular atrophy disease protein                Caenorhabditis elegans phenome mapping ORFeome-based RNAi library. <br />     SMN dimethylarginine-modified box H/ACA small nucleolar                              Genome Res 14: 2162&#8211;2168. <br />     ribonucleoprotein GAR1. J Biol Chem 277: 48087&#8211;48093.                                     51. Pulak R (2006) Techniques analysis, sorting, dispensing C. elegans on <br /> 23. Cifuentes-Diaz C, Frugier T, Tiziano FD, Lacene E, Roblot N, et al. (2001)                    COPAS flow-sorting system. Methods Mol Biol 351: 275&#8211;286. <br />     Deletion murine SMN exon 7 directed skeletal muscle leads severe                 52. Huang LS, Sternberg PW (2006) Genetic dissection developmental <br />     muscular dystrophy. J Cell Biol 152: 1107&#8211;1114.                                               pathways. WormBook. pp 1&#8211;19. <br /> 24. Shafey D, Cote PD, Kothary R (2005) Hypomorphic Smn knockdown C2C12                       53. Kamath RS, Ahringer J (2003) Genome-wide RNAi screening Caenorhabditis <br />     myoblasts reveal intrinsic defects myoblast fusion myotube morphology.                 elegans. Methods 30: 313&#8211;321. <br />     Exp Cell Res 311: 49&#8211;61.                                                                  54. Lall S, Piano F, Davis RE (2005) Caenorhabditis elegans decapping proteins: <br /> 25. Chan YB, Miguel-Aliaga I, Franks C, Thomas N, Trulzsch B, et al. (2003)                       localization functional analysis Dcp1, Dcp2, DcpS during <br />     Neuromuscular defects Drosophila survival motor neuron gene mutant.                      embryogenesis. Mol Biol Cell 16: 5880&#8211;5890. <br />     Hum Mol Genet 12: 1367&#8211;1376.                                                              55. Avery L (1993) The genetics feeding Caenorhabditis elegans. Genetics 133: <br /> 26. Chang HC, Dimlich DN, Yokokura T, Mukherjee A, Kankel MW, et al. (2008)                       897&#8211;917. <br />     Modeling spinal muscular atrophy Drosophila. PLoS ONE 3: e3209.                        56. Sulston JE, Horvitz HR (1977) Post-embryonic cell lineages nematode, <br />     doi:10.1371/journal.pone.0003209.                                                             Caenorhabditis elegans. Dev Biol 56: 110&#8211;156. <br /> 27. McWhorter ML, Monani UR, Burghes AH, Beattie CE (2003) Knockdown                       57. Sulston JE, Schierenberg E, White JG, Thomson JN (1983) The embryonic cell <br />     survival motor neuron (Smn) protein zebrafish causes defects motor                  lineage nematode Caenorhabditis elegans. Dev Biol 100: 64&#8211;119. <br />     axon outgrowth pathfinding. J Cell Biol 162: 919&#8211;931.                                 58. Simmer F, Moorman C, van der Linden AM, Kuijk E, van den Berghe PV, <br /> 28. Kariya S, Park GH, Maeno-Hikichi Y, Leykekhman O, Lutz C, et al. (2008)                       et al. (2003) Genome-wide RNAi C. elegans using hypersensitive rrf-3 <br />     Reduced SMN protein impairs maturation neuromuscular junctions                      strain reveals novel gene functions. PLoS Biol 1: e12. doi:10.1371/journal.- <br />     mouse models spinal muscular atrophy. Hum Mol Genet 17: 2552&#8211;2569.                         pbio.0000012. <br /> 29. Lorson CL, Androphy EJ (2000) An exonic enhancer required inclusion                59. Kennedy S, Wang D, Ruvkun G (2004) A conserved siRNA-degrading RNase <br />     essential exon SMA-determining gene SMN. Hum Mol Genet 9:                        negatively regulates RNA interference C. elegans. Nature 427: 645&#8211;649. <br />     259&#8211;265.                                                                                  60. Fukuto HS, Ferkey DM, Apicella AJ, Lans H, Sharmeen T, et al. (2004) G <br /> 30. Monani UR, Coovert DD, Burghes AH (2000) Animal models spinal                              protein-coupled receptor kinase function essential chemosensation C. <br />     muscular atrophy. Hum Mol Genet 9: 2451&#8211;2457.                                                 elegans. Neuron 42: 581&#8211;593. <br /> 31. Monani UR, Lorson CL, Parsons DW, Prior TW, Androphy EJ, et al. (1999) A                  61. Barrett PL, Fleming JT, Gobel V (2004) Targeted gene alteration in <br />     single nucleotide difference alters splicing patterns distinguishes SMA              Caenorhabditis elegans gene conversion. Nat Genet 36: 1231&#8211;1237. <br />     gene SMN1 copy gene SMN2. Hum Mol Genet 8: 1177&#8211;1183.                            62. Gengyo-Ando K, Mitani S (2000) Characterization mutations induced by <br /> 32. Lefebvre S, Burlet P, Liu Q, Bertrandy S, Clermont O, et al. (1997) Correlation               ethyl methanesulfonate, UV, trimethylpsoralen nematode <br />     severity SMN protein level spinal muscular atrophy. Nat Genet                  Caenorhabditis elegans. Biochem Biophys Res Commun 269: 64&#8211;69. <br />     16: 265&#8211;269.                                                                              63. Gilchrist EJ, O&#8217;Neil NJ, Rose AM, Zetka MC, Haughn GW (2006) TILLING <br /> 33. Feldkotter M, Schwarzer V, Wirth R, Wienker TF, Wirth B (2002)                                effective reverse genetics technique Caenorhabditis elegans. BMC <br />     Quantitative analyses SMN1 SMN2 based real-time lightCycler                         Genomics 7: 262. <br />     PCR: fast highly reliable carrier testing prediction severity spinal        64. Jansen G, Hazendonk E, Thijssen KL, Plasterk RH (1997) Reverse genetics by <br />     muscular atrophy. Am J Hum Genet 70: 358&#8211;368.                                                 chemical mutagenesis Caenorhabditis elegans. Nat Genet 17: 119&#8211;121. <br /> 34. Wirth B, Herz M, Wetter A, Moskau S, Hahnen E, et al. (1999) Quantitative                 65. Lesa GM (2006) Isolation Caenorhabditis elegans gene knockouts PCR <br />     analysis survival motor neuron copies: identification subtle SMN1                       screening chemically mutagenized libraries. Nat Protoc 1: 2231&#8211;2240. <br />     mutations patients spinal muscular atrophy, genotype-phenotype                    66. Liu T, Kim K, Li C, Barr MM (2007) FMRFamide-like neuropeptides and <br />     correlation, implications genetic counseling. Am J Hum Genet 64:                      mechanosensory touch receptor neurons regulate male sexual turning behavior <br />     1340&#8211;1356.                                                                                    Caenorhabditis elegans. J Neurosci 27: 7174&#8211;7182. <br /> 35. Lefebvre S, Burglen L, Frezal J, Munnich A, Melki J (1998) The role                67. Li C, Kim K (2008) Neuropeptides. WormBook. pp 1&#8211;36. <br />     SMN gene proximal spinal muscular atrophy. Hum Mol Genet 7:                            68. Mount SM, Salz HK (2000) Pre-messenger RNA processing factors the <br />     1531&#8211;1536.                                                                                    Drosophila genome. J Cell Biol 150: F37&#8211;44. <br /> 36. Burghes AH (1997) When deletion deletion? When converted.                69. Lasko P (2000) The Drosophila melanogaster genome: translation factors RNA <br />     Am J Hum Genet 61: 9&#8211;15.                                                                      binding proteins. J Cell Biol 150: F51&#8211;56. <br />  <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                              14                          October 2010 | Volume 6 | Issue 10 | e1001172 <br />                                                                                                                                                  Invertebrate Modifiers SMN <br />  <br />  <br />  70. Nambu JR, Murphy-Erdosh C, Andrews PC, Feistner GJ, Scheller RH (1988)                    103. Bensimon G, Lacomblez L, Meininger V (1994) A controlled trial riluzole in <br />      Isolation characterization Drosophila neuropeptide gene. Neuron 1:                    amyotrophic lateral sclerosis. ALS/Riluzole Study Group. N Engl J Med 330: <br />      55&#8211;61.                                                                                         585&#8211;591. <br />  71. Roulier EM, Fyrberg C, Fyrberg E (1992) Perturbations Drosophila alpha-                104. Couratier P, Sindou P, Esclaire F, Louvel E, Hugon J (1994) Neuroprotective <br />      actinin cause muscle paralysis, weakness, atrophy confer                        effects riluzole ALS CSF toxicity. Neuroreport 5: 1012&#8211;1014. <br />      obvious nonmuscle phenotypes. J Cell Biol 116: 911&#8211;922.                                   105. Haddad H, Cifuentes-Diaz C, Miroglio A, Roblot N, Joshi V, et al. (2003) <br />  72. Fyrberg E, Kelly M, Ball E, Fyrberg C, Reedy MC (1990) Molecular genetics                   Riluzole attenuates spinal muscular atrophy disease progression mouse <br />      Drosophila alpha-actinin: mutant alleles disrupt Z disc integrity muscle                   model. Muscle Nerve 28: 432&#8211;437. <br />      insertions. J Cell Biol 110: 1999&#8211;2011.                                                   106. Nishimura AL, Mitne-Neto M, Silva HC, Richieri-Costa A, Middleton S, et al. <br />  73. Inoue T, Thomas JH (2000) Targets TGF-beta signaling Caenorhabditis                      (2004) A mutation vesicle-trafficking protein VAPB causes late-onset <br />      elegans dauer formation. Dev Biol 217: 192&#8211;204.                                                spinal muscular atrophy amyotrophic lateral sclerosis. Am J Hum Genet <br />  74. Brundage L, Avery L, Katz A, Kim UJ, Mendel JE, et al. (1996) Mutations                   75: 822&#8211;831. <br />      C. elegans Gqalpha gene disrupt movement, egg laying, viability. Neuron               107. Shen V, Liu H, Liu SW, Jiao X, Kiledjian M (2008) DcpS scavenger decapping <br />      16: 999&#8211;1009.                                                                                  enzyme modulate pre-mRNA splicing. RNA 14: 1132&#8211;1142. <br />  75. Gkogkas C, Middleton S, Kremer AM, Wardrope C, Hannah M, et al. (2008)                    108. Singh J, Salcius M, Liu SW, Staker BL, Mishra R, et al. (2008) DcpS a <br />      VAPB interacts modulates activity ATF6. Hum Mol Genet 17:                      therapeutic target spinal muscular atrophy. ACS Chem Biol 3: 711&#8211;722. <br />      1517&#8211;1526.                                                                                109. Nonis D, Schmidt MH, van Loo S, Eich F, Dikic I, et al. (2008) Ataxin-2 <br />  76. Wang H, Wu LJ, Kim SS, Lee FJ, Gong B, et al. (2008) FMRP acts key                        associates endocytosis complex affects EGF receptor trafficking. <br />      messenger dopamine modulation forebrain. Neuron 59: 634&#8211;647.                        Cell Signal 20: 1725&#8211;1739. <br />  77. Piazzon N, Rage F, Schlotter F, Moine H, Branlant C, et al. (2008) In vitro           110. Lim J, Hao T, Shaw C, Patel AJ, Szabo G, et al. (2006) A protein-protein <br />      cellulo evidences association survival motor neuron complex              interaction network human inherited ataxias disorders Purkinje cell <br />      fragile X mental retardation protein. J Biol Chem 283: 5598&#8211;5610.                          degeneration. Cell 125: 801&#8211;814. <br />  78. Bauer PO, Nukina N (2009) The pathogenic mechanisms polyglutamine                      111. Li S, Armstrong CM, Bertin N, Ge H, Milstein S, et al. (2004) A map the <br />      diseases current therapeutic strategies. J Neurochem 110: 1737&#8211;1765.                       interactome network metazoan C. elegans. Science 303: 540&#8211;543. <br />  79. Crawford TO, Pardo CA (1996) The neurobiology childhood spinal                         112. Jeong H, Then F, <span id='am-88' about='obo:NCBITaxon_43708' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_43707 obo:NCBITaxon_41937 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-89' property="rdfs:label" content="Melia" datatype="xsd:string"></span><span id='am-90'  ></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Melia</span> TJ, Jr., Mazzulli JR, Cui L, et al. (2009) Acetylation <br />      muscular atrophy. Neurobiol Dis 3: 97&#8211;110.                                                     targets mutant huntingtin autophagosomes degradation. Cell 137: 60&#8211;72. <br />  80. Goedert M, Spillantini MG (2006) A century Alzheimer&#8217;s disease. Science                113. Jia K, Hart AC, Levine B (2007) Autophagy genes protect disease <br />      314: 777&#8211;781.                                                                                  caused polyglutamine expansion proteins Caenorhabditis elegans. Autophagy <br />  81. Lang AE, Lozano AM (1998) Parkinson&#8217;s disease. First parts.                             3: 21&#8211;25. <br />      N Engl J Med 339: 1044&#8211;1053.                                                              114. Serra HG, Duvick L, <span id='am-6' about='obo:NCBITaxon_143324' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_81386 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_81393 obo:NCBITaxon_7776 obo:NCBITaxon_123357 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Zu" datatype="xsd:string"></span>Zu</span> T, Carlson K, Stevens S, et al. (2006) RORalpha- <br />  82. Jorgensen EM, Mango SE (2002) The art design genetic screens:                           mediated Purkinje cell development determines disease severity adult SCA1 <br />      Caenorhabditis elegans. Nat Rev Genet 3: 356&#8211;369.                                              mice. Cell 127: 697&#8211;708. <br />  83. Dimitriadi M, Hart AC (2010) Neurodegenerative disorders: Insights               115. Petrelli A, Gilestro GF, Lanzardo S, Comoglio PM, Migone N, et al. (2002) <br />      nematode Caenorhabditis elegans. Neurobiol Dis.                                                The endophilin-CIN85-Cbl complex mediates ligand-dependent downregula- <br />  84. Fiuza UM, Arias AM (2007) Cell molecular biology Notch. J Endocrinol                    tion c-Met. Nature 416: 187&#8211;190. <br />      194: 459&#8211;474.                                                                             116. Kaneko T, Maeda A, Takefuji M, Aoyama H, Nakayama M, et al. (2005) Rho <br />  85. Kim SK (2007) Common aging pathways worms, flies, mice humans.                          mediates endocytosis epidermal growth factor receptor phosphor- <br />      J Exp Biol 210: 1607&#8211;1612.                                                                     ylation endophilin A1 Rho-kinase. Genes Cells 10: 973&#8211;987. <br />  86. Leicht DT, Balan V, Kaplun A, Singh-Gupta V, Kaplun L, et al. (2007) Raf                  117. Guillaumond F, Dardente H, Giguere V, Cermakian N (2005) Differential <br />      kinases: function, regulation role human cancer. Biochim Biophys Acta                   control Bmal1 circadian transcription REV-ERB ROR nuclear <br />      1773: 1196&#8211;1212.                                                                               receptors. J Biol Rhythms 20: 391&#8211;403. <br />  87. Schlegel A, Stainier DY (2007) Lessons &#8216;&#8216;lower&#8217;&#8217; organisms: worms,              118. Forman BM, Chen J, Blumberg B, Kliewer SA, Henshaw R, et al. (1994) <br />      flies, zebrafish teach human energy metabolism. PLoS Genet                    Cross-talk ROR alpha 1 Rev-erb family orphan nuclear <br />      3: e199. doi:10.1371/journal.pgen.0030199.                                                     receptors. Mol Endocrinol 8: 1253&#8211;1261. <br />  88. Silverman GA, Luke CJ, <span id='am-78' about='obo:NCBITaxon_706860' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30102 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_35567 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7524 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_33372 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33365 obo:NCBITaxon_33342 obo:NCBITaxon_33368 obo:NCBITaxon_50557'><span id='am-79' property="rdfs:label" content="Bhatia" datatype="xsd:string"></span><span id='am-80'  ></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Bhatia</span> SR, Long OS, Vetica AC, et al. (2009)                       119. Freeman JL, Pitcher JA, Li X, Bennett V, Lefkowitz RJ (2000) alpha-Actinin is <br />      Modeling molecular cellular aspects human disease using nematode                    potent regulator G protein-coupled receptor kinase activity substrate <br />      Caenorhabditis elegans. Pediatr Res 65: 10&#8211;18.                                                 specificity vitro. FEBS Lett 473: 280&#8211;284. <br />  89. Kaksonen M, Toret CP, Drubin DG (2005) A modular design clathrin-                 120. Chatellard-Causse C, Blot B, Cristina N, Torch S, Missotten M, et al. (2002) <br />      actin-mediated endocytosis machinery. Cell 123: 305&#8211;320.                                   Alix (ALG-2-interacting protein X), protein involved apoptosis, binds to <br />  90. Ralser M, Nonhoff U, Albrecht M, Lengauer T, Wanker EE, et al. (2005)                          endophilins induces cytoplasmic vacuolization. J Biol Chem 277: <br />      Ataxin-2 huntingtin interact endophilin-A complexes function                    29108&#8211;29115. <br />      plastin-associated pathways. Hum Mol Genet 14: 2893&#8211;2909.                                 121. Lu L, Zhang Q, Timofeyev V, Zhang Z, Young JN, et al. (2007) Molecular <br />  91. Singer-Kruger B, Nemoto Y, Daniell L, Ferro-Novick S, De Camilli P (1998)                      coupling Ca2+-activated K+ channel L-type Ca2+ channels alpha- <br />      Synaptojanin family members implicated endocytic membrane traffic                    actinin2. Circ Res 100: 112&#8211;120. <br />      yeast. J Cell Sci 111 ( Pt 22): 3347&#8211;3356.                                                122. Chen B, Borinstein SC, Gillis J, Sykes VW, Bogler O (2000) The glioma- <br />  92. Gibbings DJ, Ciaudo C, Erhardt M, Voinnet O (2009) Multivesicular bodies                       associated protein SETA interacts AIP1/Alix ALG-2 modulates <br />      associate components miRNA effector complexes modulate                             apoptosis astrocytes. J Biol Chem 275: 19275&#8211;19281. <br />      miRNA activity. Nat Cell Biol 11: 1143&#8211;1149.                                              123. Schmidt MH, Hoeller D, Yu J, Furnari FB, Cavenee WK, et al. (2004) Alix/ <br />  93. Lee YS, Pressman S, Andress AP, Kim K, White JL, et al. (2009) Silencing                    AIP1 antagonizes epidermal growth factor receptor downregulation the <br />      small RNAs linked endosomal trafficking. Nat Cell Biol 11: 1150&#8211;1156.                    Cbl-SETA/CIN85 complex. Mol Cell Biol 24: 8981&#8211;8993. <br />  94. Kong L, Wang X, Choe DW, Polley M, Burnett BG, et al. (2009) Impaired                     124. Carman CV, Lisanti MP, Benovic JL (1999) Regulation G protein-coupled <br />      synaptic vesicle release immaturity neuromuscular junctions spinal                   receptor kinases caveolin. J Biol Chem 274: 8858&#8211;8864. <br />      muscular atrophy mice. J Neurosci 29: 842&#8211;851.                                            125. Ralser M, Albrecht M, Nonhoff U, Lengauer T, Lehrach H, et al. (2005) An <br />  95. Brenner S (1974) The genetics Caenorhabditis elegans. Genetics 77: 71&#8211;94.                   integrative approach gain insights cellular function human ataxin- <br />  96. Bean CJ, Schaner CE, Kelly WG (2004) Meiotic pairing imprinted X                           2. J Mol Biol 346: 203&#8211;214. <br />      chromatin assembly Caenorhabditis elegans. Nat Genet 36: 100&#8211;105.                      126. Lessing D, Bonini NM (2008) Polyglutamine genes interact modulate the <br />  97. Bilinski SM, Jaglarz MK, Szymanska B, Etkin LD, Kloc M (2004) Sm proteins,                     severity progression neurodegeneration Drosophila. PLoS Biol 6: e29. <br />      constituents spliceosome, components nuage mitochon-                     doi:10.1371/journal.pbio.0060029. <br />      drial cement Xenopus oocytes. Exp Cell Res 299: 171&#8211;178.                               127. Ciosk R, DePalma M, Priess JR (2004) ATX-2, C. elegans ortholog ataxin <br />  98. Dostie J, Mourelatos Z, Yang M, Sharma A, Dreyfuss G (2003) Numerous                           2, functions translational regulation germline. Development 131: <br />      microRNPs neuronal cells containing novel microRNAs. RNA 9: 180&#8211;186.                        4831&#8211;4841. <br />  99. Hillier LW, Reinke V, Green P, Hirst M, Marra MA, et al. (2009) Massively                 128. Satterfield TF, Pallanck LJ (2006) Ataxin-2 Drosophila homolog, ATX2, <br />      parallel sequencing polyadenylated transcriptome C. elegans. Genome                  physically assemble polyribosomes. Hum Mol Genet 15: 2523&#8211;2532. <br />      Res 19: 657&#8211;666.                                                                          129. Nonhoff U, Ralser M, Welzel F, Piccini I, Balzereit D, et al. (2007) Ataxin-2 <br /> 100. Townend J (2002) Practical statistics environmental biological scientists.             interacts DEAD/H-box RNA helicase DDX6 interferes P- <br />      Chichester, UK: John Wiley Sons Ltd.                                                       bodies stress granules. Mol Biol Cell 18: 1385&#8211;1396. <br /> 101. Gunawardena S, Goldstein LS (2001) Disruption axonal transport                     130. Kozlov G, Trempe JF, Khaleghpour K, Kahvejian A, Ekiel I, et al. (2001) <br />      neuronal viability amyloid precursor protein mutations Drosophila. Neuron                Structure function C-terminal PABC domain human poly(A)- <br />      32: 389&#8211;401.                                                                                   binding protein. Proc Natl Acad Sci U S A 98: 4409&#8211;4413. <br /> 102. Cao YJ, Dreixler JC, Couey JJ, Houamed KM (2002) Modulation                            131. Ikezu T, Trapp BD, Song KS, Schlegel A, Lisanti MP, et al. (1998) Caveolae, <br />      recombinant native neuronal SK channels neuroprotective drug                        plasma membrane microdomains alpha-secretase-mediated processing of <br />      riluzole. Eur J Pharmacol 449: 47&#8211;54.                                                          amyloid precursor protein. J Biol Chem 273: 10485&#8211;10495. <br />  <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                              15                           October 2010 | Volume 6 | Issue 10 | e1001172 <br />                                                                                                                                                 Invertebrate Modifiers SMN <br />  <br />  <br /> 132. Massenet S, Pellizzoni L, Paushkin S, Mattaj IW, Dreyfuss G (2002) The SMN               144. Shao J, Welch WJ, Diprospero NA, Diamond MI (2008) Phosphorylation of <br />      complex associated snRNPs cytoplasmic assembly                       profilin ROCK1 regulates polyglutamine aggregation. Mol Cell Biol 28: <br />      pathway. Mol Cell Biol 22: 6533&#8211;6541.                                                         5196&#8211;5208. <br /> 133. Gareus R, Di Nardo A, Rybin V, Witke W (2006) Mouse profilin 2 regulates                 145. Baillat G, Gaillard S, Castets F, Monneron A (2002) Interactions phocein <br />      endocytosis competes SH3 ligand binding dynamin 1. J Biol Chem                    nucleoside-diphosphate kinase, Eps15, Dynamin I. J Biol Chem 277: <br />      281: 2803&#8211;2811.                                                                               18961&#8211;18966. <br /> 134. Schafer DA, Weed SA, Binns D, Karginov AV, Parsons JT, et al. (2002)                     146. Sittler A, Walter S, Wedemeyer N, Hasenbank R, Scherzinger E, et al. (1998) <br />      Dynamin2 cortactin regulate actin assembly filament organization.                     SH3GL3 associates Huntingtin exon 1 protein promotes the <br />      Curr Biol 12: 1852&#8211;1857.                                                                      formation polygln-containing protein aggregates. Mol Cell 2: 427&#8211;436. <br /> 135. Witke W, Podtelejnikov AV, Di Nardo A, Sutherland JD, Gurniak CB, et al.                 147. Qin ZH, Wang Y, Sapp E, Cuiffo B, Wanker E, et al. (2004) Huntingtin bodies <br />      (1998) In mouse brain profilin I profilin II associate regulators             sequester vesicle-associated proteins polyproline-dependent interaction. <br />      endocytic pathway actin assembly. EMBO J 17: 967&#8211;976.                                     J Neurosci 24: 269&#8211;281. <br /> 136. Al-Ramahi I, Perez AM, Lim J, Zhang M, Sorensen R, et al. (2007) dAtaxin-2               148. Fallon L, Belanger CM, Corera AT, Kontogiannea M, Regan-Klapisz E, et al. <br />      mediates expanded Ataxin-1-induced neurodegeneration Drosophila model                    (2006) A regulated interaction UIM protein Eps15 implicates parkin in <br />                                                                                                    EGF receptor trafficking PI(3)K-Akt signalling. Nat Cell Biol 8: 834&#8211;842. <br />      SCA1. PLoS Genet 3: e234. doi:10.1371/journal.pgen.0030234. <br />                                                                                               149. Salcini AE, Hilliard MA, Croce A, Arbucci S, Luzzi P, et al. (2001) The Eps15 <br /> 137. Ghosh-Roy A, Desai BS, Ray K (2005) Dynein light chain 1 regulates <br />                                                                                                    C. elegans homologue EHS-1 implicated synaptic vesicle recycling. Nat Cell <br />      dynamin-mediated F-actin assembly sperm individualization in <br />                                                                                                    Biol 3: 755&#8211;760. <br />      Drosophila. Mol Biol Cell 16: 3107&#8211;3116. <br />                                                                                               150. Krishnan KS, Rikhy R, Rao S, Shivalkar M, Mosko M, et al. (2001) Nucleoside <br /> 138. Navarro-Lerida I, Martinez Moreno M, Roncal F, Gavilanes F, Albar JP, et al.                  diphosphate kinase, source GTP, required dynamin-dependent <br />      (2004) Proteomic identification brain proteins interact dynein light             synaptic vesicle recycling. Neuron 30: 197&#8211;210. <br />      chain LC8. Proteomics 4: 339&#8211;346.                                                        151. Sharma A, Lambrechts A, Hao le T, Le TT, Sewry CA, et al. (2005) A role for <br /> 139. <span id='am-47' about='obo:NCBITaxon_9596' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasAlternativeId" content="NCBITaxon:317028" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasAlternativeId" content="NCBITaxon:445518" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasExactSynonym" content="chimpanzees" datatype="xsd:string"></span><span id='am-53'  ></span><span id='am-54' property="rdfs:label" content="Pan" datatype="xsd:string"></span>Pan</span> S, Wang R, Zhou X, He G, Koomen J, et al. (2006) Involvement                       complexes survival motor neurons (SMN) protein gemins profilin <br />      conserved adaptor protein Alix actin cytoskeleton assembly. J Biol Chem                    neurite-like cytoplasmic extensions cultured nerve cells. Exp Cell Res 309: <br />      281: 34640&#8211;34650.                                                                             185&#8211;197. <br /> 140. Gaillard S, Bartoli M, Castets F, Monneron A (2001) Striatin, calmodulin-              152. Giesemann T, Rathke-Hartlieb S, Rothkegel M, Bartsch JW, Buchmeier S, <br />      dependent scaffolding protein, directly binds caveolin-1. FEBS Lett 508: 49&#8211;52.               et al. (1999) A role polyproline motifs spinal muscular atrophy protein <br /> 141. Khandjian EW, Huot ME, Tremblay S, Davidovic L, Mazroui R, et al. (2004)                      SMN. Profilins bind colocalize smn nuclear gems. J Biol Chem <br />      Biochemical evidence association fragile X mental retardation protein              274: 3 <br /> </body></html>