<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Identification of Spt5 Target Genes in Zebrafish <br /> Development Reveals Its Dual Activity In Vivo <br />  <br />      Abstract <br />      Spt5 is a conserved essential protein that represses or stimulates transcription elongation in vitro. Immunolocalization <br />      studies on Drosophila polytene chromosomes suggest that Spt5 is associated with many loci throughout the genome. <br />      However, little is known about the prevalence and identity of Spt5 target genes in vivo during development. Here, we <br />      identify direct target genes of Spt5 using fogsk8 zebrafish mutant, which disrupts the foggy/spt5 gene. We identified that <br />      fogsk8 and their wildtype siblings differentially express less than 5% of genes examined. These genes participate in diverse <br />      biological processes from stress response to cell fate specification. Up-regulated genes exhibit shorter overall gene length <br />      compared to all genes examined. Through chromatin immunoprecipitation in zebrafish embryos, we identified a subset of <br />      developmentally critical genes that are bound by both Spt5 and RNA polymerase II. The protein occupancy patterns on <br />      these genes are characteristic of both repressive and stimulatory elongation regulation. Together our findings establish Spt5 <br />      as a dual regulator of transcription elongation in vivo and identify a small but diverse set of target genes critically <br />      dependent on Spt5 during development. <br />  <br />  <br />  <br />  <br /> Introduction                                                                          Inducing Factor), which has both repressive and stimulatory <br />                                                                                       activity on transcription elongation in vitro [21]. Spt5 has also been <br />    The production of functional mRNA involves multiple steps                          identified independently as a protein required for HIV-1 Tat <br /> that include transcription initiation, elongation, and termination.                   trans-activation in fractionated extracts [22]. In addition, Spt5 <br /> The initiation step is thought to be the major regulatory point for                   interacts and modulates activity of factors implicated in mRNA <br /> most genes in vivo, despite that promoter-proximal stalled RNA                        maturation and surveillance [23&#8211;25]. <br /> polymerase II (RNAPII) has been detected at several loci including                       At the phenotypic level, the in vivo role of Spt5 in multi-cellular <br /> hsp70 [1], HIVLTR [2], c-myc [3], and c-fos [4]. Recently, it has                     organisms has been revealed through genetic analyses in C. elegans, <br /> been observed that promoters of many genes are poised for                             Drosophila, and zebrafish. Inactivation of Spt5 through RNAi in C. <br /> activation, as assessed by chromatin status and RNAPII occupancy                      elegans arrest embryonic development prior to gastrulation [26]. In <br /> in human Hela cells, mammalian embryonic stem cells, and                              Drosophila, a hypomorphic mutation in spt5 displays defects in the <br /> Drosophila embryos [5&#8211;9]. These observations suggest that                             expression of segmental patterning genes [27]. At the molecular <br /> regulation at the level of transcription elongation may be more                       level, Spt5 localization on polytene chromosomes and chromatin <br /> prevalent than previously thought. However, the distinct role of                      immunoprecipitation studies reveal broad distribution of Spt5 at <br /> individual transcription elongation factors in critically regulating                  sites of active transcription as well as at the un-induced heat shock <br /> elongation of target genes remains unclear.                                           gene promoters [28&#8211;30]. In zebrafish, whereas a hypomorphic <br />    In recent years, biochemical studies have identified over a dozen                  mutation in spt5 (the fogm806 allele) displays distinct defects in <br /> proteins and a small nuclear RNA, which regulate RNAPII                               neuronal development [31], severe truncation (the fogsk8 allele) or <br /> elongation in vitro [10&#8211;17]. Among these molecules, Spt5, an                          deletion (the fogs30 allele) of the gene product causes broad deficits <br /> evolutionarily conserved protein, is the focus of <span id='am-36' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-37'  ></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> study. Spt5,                   in embryonic development [32]. Together, these analyses establish <br /> together with Spt4 and Spt6, was initially discovered by genetic                      functional significance of Spt5 in vivo, implicate intricate regulatory <br /> studies in <span id='am-7' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-22'  ></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> [18], and later found to be essential             mechanisms that remain to be elucidated, and suggest that Spt5 <br /> for transcription via modulating chromatin structure [19,20]. A                       may regulate a large number of target genes in vivo. Despite these <br /> search for factors that inhibit transcription elongation in the                       insights, only the heat shock protein-encoding hsp70 and HIV viral <br /> presence of the nucleotide analog 5,6-dichloro-1-b-D-ribofurano-                      genes, have been demonstrated to be regulated at the level of <br /> sylbenzimidazole (DRB) led to the biochemical purification of Spt4                    transcription elongation by Spt5 in intact cells [29,33], whereas <br /> and Spt5 as a tight complex named DSIF (DRB Sensitivity                               induction of the immediate early gene c-fos appears to depend on <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                 1                           November 2008 | Volume 3 | Issue 11 | e3621 <br />                                                                                                                          In Vivo Target Genes of Spt5 <br />  <br />  <br /> the stimulatory activity of Spt5 [34]. Thus, an important question           a. <br /> remains: What other genes, if any, are critically dependent on Spt5 <br />                                                                              A                                      B <br /> in vivo? <br />    Here we report expression profiling of over 10,000 protein- <br /> coding genes using ,24 hours post fertilization (hpf) zebrafish <br /> embryos. Our results identified a surprisingly small fraction (,5%) <br /> of genes that were differentially expressed between fogsk8 mutants <br /> and their wildtype (WT) siblings. Pathway analyses revealed that <br /> these differentially expressed genes are involved in diverse                 WT-10hpf                    Spt5       WT-10hpf                 RNAPII <br /> biological pathways that range from stress response to cell fate <br /> specification. A gene structure analysis showed that genes                    C <br /> upregulated in fogsk8 embryos had a significantly shorter length, <br />                                                                                                                    D <br />                                                                                                                         brain <br /> compared to the average gene length in zebrafish. Furthermore, in                 brain <br /> vivo chromatin immunoprecipitation (ChIP) on selected set of <br /> differentially expressed genes in 24 hpf WT embryos uncovered <br /> Spt5 target genes: among the target genes repressed by Spt5 <br /> (including gadd45b, c-fos, hsp70, smoothened, and follistatin), RNAPII                        eye                           eye <br /> occupancy was higher at the 59 than 39 end of these genes; among <br /> the target genes activated by Spt5 (including lunatic fringe and <br /> tropomyosin a), RNAPII occupancy was detected at both their 59 and           WT-24 hpf                   Spt5      fogsk8 -24 hpf              Spt5 <br /> 39 ends. ChIP analyses in fogsk8 mutant embryos identified gadd45b <br /> that exhibited significantly increased RNAPII occupancy at the 39             E                     19 hpf                24 hpf <br /> end compared to that in WT embryos, whereas other target genes                                  WT       fogsk8        WT          fogsk8 <br /> unexpectedly showed reduced RNAPII occupancy at both 59 and <br /> 39 ends. This observation highlights the dynamic and complex                           Spt5                                                 ~150kDa <br /> nature of elongation regulation in vivo. Taken together, our results <br /> establish Spt5 as a dual regulator of transcription elongation in vivo <br /> and reveal a small but diverse set of direct target genes.                            Actin                                                 ~45kDa <br />  <br /> Results <br />                                                                               F                                    G <br /> Spt5 protein is significantly reduced in 24 hpf fogsk8 <br />                                                                                      brain                                brain <br /> mutant embryos <br />    The foggy/spt5 transcript is broadly distributed throughout the <br /> developing embryo [31,32]. To gain insights into the level and                                 yolk                             yolk <br /> distribution of Spt5 protein, an antibody directed against the C- <br /> terminal end of the zebrafish Spt5 protein was generated. Double- <br /> labeling with 8WG16, an antibody that recognizes RNAPII,                                                                    tail <br /> demonstrated prominent nuclear distribution of Spt5 (Fig. 1A&#8211;B).                   tail <br /> The fogsk8 mutation (Fig. 1F&#8211;G) is predicted to yield a non- <br /> functional zygotic gene product, and the maternal transcript is              WT-24 hpf                             fogsk8 -24 hpf <br /> undetectable after 10 hours post fertilization (hpf) [32]. However, <br /> the status of the maternal protein was unclear. The sk8 mutant               Figure 1. Spt5 protein is significantly decreased in fogsk8 <br /> phenotype is first visible around 19 hpf, suggesting the presence of         embryos. (A&#8211;B) Whole mount immunofluorescence detection of <br /> maternal Foggy protein at earlier stages of development.                     Spt5 and RNAPII in zebrafish embryos, showing the nuclear localization <br /> Immunostaining (Fig. 1C&#8211;D) showed that maternal Spt5 protein                 pattern of Spt5 protein. (C&#8211;D) Whole mount immunofluorescence <br />                                                                              detection of Spt5 in WT and fogsk8 embryos at 24 hpf, showing overall <br /> was significantly reduced in fogsk8 mutant embryos as compared to <br />                                                                              decrease of Spt5 protein in fogsk8 embryos. Confocal images of lateral <br /> WT siblings. Western blotting (Fig. 1E) detected two bands around            views are shown, with the anterior to the left and dorsal up. (E) Western <br /> the predicted size of zebrafish Spt5 in WT embryos. Since Spt5 is            blot of wild-type (WT) and fogsk8 embryonic extracts at 19 hpf and <br /> known to be phosphorylated [14,34&#8211;38] and methylated [39],                   24 hpf shows substantial decrease in maternal Spt5 protein in the <br /> these bands may represent forms of Spt5 with different post-                 mutant. (F&#8211;G) Light microscopic view of WT and sk8 embryos at 24 hpf, <br /> translational modifications. Although the intensity of both bands            showing the gross morphological phenotype of the mutant embryos. <br />                                                                              doi:10.1371/journal.pone.0003621.g001 <br /> was reduced in the fogsk8 mutant, the faster migrating form <br /> appeared more severely affected (Figure 1E). This result indicates <br /> that fogsk8 mutant at 24 hpf exhibits a significant reduction but not        background signal-to-noise ratio (as determined by microarray <br /> complete absence of Spt5 protein.                                            absent-present calls), suggesting that a majority of genes on the <br />                                                                              array have detectable expression at this stage. Interestingly <br /> Expression profiling reveals a small number of                               however, only 4.5% of the genes were differentially expressed <br /> differentially expressed genes in 24 hpf fogsk8 embryos                      (absolute fold greater than 2, student t-test P,0.05) between fogsk8 <br />    To identify potential downstream target genes of Spt5, we                 and WT siblings. The 2-fold cutoff was used because an expression <br /> performed microarray analysis by hybridizing zebrafish Affymetrix            change of less than 2 fold was often not validated by quantitative <br /> oligoarrays, which contain over 10,000 protein-coding genes, with            PCR analysis (<span id='am-52' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-55'  ></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> not shown). Among the differentially expressed <br /> biotinylated cRNA probes from 24 hpf fogsk8 mutant and WT                    genes, 97 (,1% of the total genes examined) were up-regulated <br /> sibling embryos. 75% of the probe sets displayed above-                      (ranging from +2- to +10-fold), and 358 (,3.5% of the total genes <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        2                          November 2008 | Volume 3 | Issue 11 | e3621 <br />                                                                                                                                                                                                                 In Vivo Target Genes of Spt5 <br />  <br />  <br />  <br />  <br /> A <br />            sk8 <br />   wt/fog          microarray <br />     chip1 chip2 chip3                             Down-regulated <br />                                                                                                             Number Number            Permute <br />                                                                              GO Name                                         Z Score                                          GO Type                         Corresponding Genes <br />                                                                                                             Changed Measured            P <br />                                                       extracellular matrix structural constituent              5                      7           8.03             0          function              col1a1, col1a2, col1a3, col2a1a, matn4 <br />                                                           tetrapyrrole metabolic process                       4                       6          6.90          0             process                     alas2, hmbs, hmox1, urod <br />                                                               Notch signaling pathway                          6                      15          6.20          0             process                  dla, dlb, dlc, dld, her4.1, notch3 <br />                                                       ATP hydrolysis coupled proton transport                  3                       5          5.62       0.0015           process                   atp1a1a.4, atp1a2a, atp1a1b <br />                                                                                                                                                                                                cdh11, cdh15, cmlc1, dl(a,b,c,d), hsp90b1, lcp1, <br />                                                                      calcium ion binding                       23                 167             5.26             0          function             matn4, myl7, mylz(2,3), notch3, pvalb1, <br />                                                                                                                                                                                                                pvalb(2,5,8), tnnc <br />                                                              oxygen transporter activity                       5                      16          4.80       0.0005           function                 hbae1, hbae3, hbbe3, mb, vim <br />                                                      vascular endothelial growth factor receptor <br />                                                                                                                3                      7           4.58       0.0005           function                               kdr, pdgfra, ttnl <br />                                                                       activity <br />                                                                                                                                                                                          ascl1a, dld, lhx1b, rtn1b, tbr1, tlx3a, tnw, <br />                                                                         Neurogenesis                           14                 113             4.56             0          Panther <br />                                                                                                                                                                                                   zgc:113283, zgc:64199 <br />                                                              Dr_DNA_replication_Reactome                       5                      23          4.29        0.001       GenMAPP          mcm2, mcm3, mcm4, mcm5, orc5l <br />                                                                     porin activity                             3                       8          4.20        0.003        function               aqp3, aqp8, zgc:85890 <br />                                                                     actin binding                              6                      27          4.09          0          function         gsnl1, lcp1, sptb, svil, tmsb, tpma <br />                                                                  Muscle development                            5                      30          3.50        0.004        Panther         boc, im:6899248, myl7, pvalb8, ttnl <br />                                                                                                                                                                                      atoh1a, atp1a1a.2, atp1a2a, boc, ccnd1, cx43, <br />                                                         anatomical structure morphogenesis                     17                 159             3.32             0       process  dld, gap43, igfbp1, kdr, lbr, notch3, oep, plasticin, <br />                                                                                                                                                                                                       ptf1a, sox9b, tnc <br />                                                                         somitogenesis                          5                      27          3.21       0.0025        process             dlc, dld, her4.1, myhz2, oep <br />                                                                                                                                                                                        epb41, prox1, sptb, svil, tmsb, tuba2, tubb5, <br />                                                      cytoskeleton organization and biogenesis                  8                      60          2.95        0.002        process <br />                                                                                                                                                                                                          zgc:112335 <br />                                                                     cell migration                             6                      53          2.10        0.018        process           boc, lbr, nrarpa, oep, supt5h, tnc <br />                                                         protein tyrosine phosphatase activity                  4                      31          2.01        0.041        function          ptprn2, ptprn, ptpn6, zgc:63553 <br />  <br />                                                   Up-regulated <br />                                                                         viral envelope                         3                   4              13.25         0         component                   zgc:114014, zp2.2, zp2.4, zp3.2 <br />                                                                   response to cadmium ion                      3                   8               9.22         0          process                           gadd45b, hsp70, jun <br />                                                                  pattern specification process                 5                  105              3.27      0.0035        process                atp1a1a.1, fst, gadd45b, gadd45bl, smo <br />                                                                                                                                                                                                bapx1, fos, foxd5, hox(b6a, b6b, b10a, c5a, 6a, <br />                                                              sequence-specific DNA binding                     13                 288             5.14             0          function <br />                                                                                                                                                                                                        d11a, d13a), jun, nkx2.7, six4.1 <br />                                                                  G-protein mediated signaling                  3                      79          3.28        0.044           Panther                        opn1sw2, smo, tpbgl <br />  <br />  <br /> down-regulated                 up-regulated <br />  <br />  <br />  <br />  <br /> B <br />            Author: Alex ander C. Zambon <br />            Maintained by: GenMAPP.org <br />            E-mail: azambon@gladstone.uc sf.edu <br />                                                                                              G1 to S cell cycle control <br />            Last modified: This pathway was inferred from human by homology <br />  <br />  <br />                                          G1 Phase                                                                                                                                                       G1/S transition Phase <br />                                                                                    CDKN2B                   cdkn1c <br />               G1 Cyclin                  Cyclin Ds                                                          CDKN1B <br />                                                                                    CDKN2A                                                                                                               G1/S transition Cyclin <br />           Dependent Kinases                                                                                                                                                                                                        Cyclin Es <br />                                           ccnd1 -2.145                             CDKN2C                   CDKN1A                                                                                       Dependent Kinases <br />                  CDK4                     CCND2                                                                                                                                                                                      ccne <br />                                                                                    CDKN2D                                                                                                                      zgc:56598 <br />                  CDK6                     CCND3                                                                                                                                                                                    zgc:86724 <br />                                                                                                                     e                      ATM <br />                                                                                                                                                                 u <br />                                                                                                         b                                  p                  u                inactive              wee1 <br />                                                                                   zgc:55680                                                                   u <br />                               CyclinD                  p          CyclinD                                                                  mdm2                                                      MYT1 <br />                                                                               p                                         tp53                               tp53                  ccne                                 ccne <br />                                Cdk       zgc:85821                 Cdk <br />                                                                                          e                                                                                p      CDK            p                     CDK <br />                                          zgc:114132                                                                      G1/S Cell Cycle                  Degredation <br />                                          zgc:101599                                           gadd45a                     Checkpoint <br />  <br />                                                              p                                                                                                                                      CDC25A <br />                                                                                                                                                                                                                     phosphorylation of proteins <br />                                                                                                         b                                                                                                           involved in G1/S transition <br />  <br />                                                                        E2f         Rb <br />                                         dp1             RB1            E2F4        RB1                                         pcna <br />                                       TFDP1                E2f         E2F1       RBL1 <br />                                                                        E2F5 <br />                                        tfdp2 <br />                                                                        E2F2                                                                                                               DNA replication <br />                         RB1      p                                     E2F6                                                                                                        zgc:56264                mcm2      -3.406 <br />                                                                                                                                                      origen recognition <br />                                                                        E2F3                                                                                                                                 mcm3      -2.584 <br />                                                                                                                                                                                DNA polymerases <br />                                                                                                                                                            complex <br />                                   E2f                                                                                                                                                                       mcm4      -2.523 <br />                                                                                                                                                              zgc:85772              POLE2                   mcm5      -2.643 <br />                                 TFDP1                 Cell cycle DNA replication genes <br />                                                                                                     Gene Value: sk8-fold change                              zgc:77827 -2.294       POLE <br />                                                                                                                                                                                                           MCM6 <br />                                TTTCC/GCGC                                                                                                                         orc6l            zgc:77841 <br />                                                                                                     Legend:                                                       orc1l <br />                                                                                                                                                                                                           MCM7 <br />                                   G2 Phase-M phase                                                      Increased sk8                                                                   rpa1 <br />                                                                                                                                                               ORC2L                                      CDC2A <br />                                                                                                         Decreased sk8                                                                   rpa2 <br />                               CDC2A                  ccng2                                                                                                        orc3l                                     CDK4 <br />                                                                                                                                                                                    zgc:56310 <br />                               ccna1                                                                     No criteria met                                                                                     CDK6 <br />                               ccnb1                                                                     Not found                                                                                       zgc:85821 <br />  <br />                                                                                                                                                                                                             prim1 <br />                                                                                                                                                                                                         zgc:110727 <br />  <br />  <br />  <br /> Figure 2. Gene expression profiling reveals that the expression of a small percentage of genes is affected in fogsk8 mutant embryos. <br /> (A) HOPACH cluster diagram of the microarray data are shown for individual replicates compared with the mean for 24 hpf embryos. 615 probe sets <br /> with fold change .2 and t test P-value ,0.05 are displayed along with Gene Ontology terms and pathways over-represented as determined by a <br />  <br />  <br />  <br />       PLoS ONE | www.plosone.org                                                                                           3                                              November 2008 | Volume 3 | Issue 11 | e3621 <br />                                                                                                                         In Vivo Target Genes of Spt5 <br />  <br />  <br />  <br /> Fisher&#8217;s exact test permutation based P-value. The intensity of down-regulation (green) and up-regulation (red) reflects the magnitude of fold <br /> change. Representative genes within the GO categories are shown in the last column. Genes subjected to further analyses in this study are <br /> represented in blue color in the last column. (B) G1 to S Cell Cycle pathway is shown as an example of GenMAPP pathway analysis. The pathway map <br /> shows genes that are regulated as a group (e.g. mcm genes) and genes that are not differentially expressed in fogsk8 embryos but are present on the <br /> array (grey color). <br /> doi:10.1371/journal.pone.0003621.g002 <br />  <br />  <br /> examined) were down-regulated (from 22- to 2300-fold) in fogsk8               analysis also showed a significant increase in exon length for <br /> embryos (Fig. 2A). Many of the differentially expressed genes have            upregulated genes, the difference was relatively small (1.26 fold). <br /> previously been shown to display dynamic spatial and temporal                 None of the gene structure data differed between downregulated <br /> expression patterns [40]. Four genes (gch, pvalb, hoxb6, dlx2) were           genes and all other genes on the array (Fig. 3B). Taken together, <br /> validated through whole mount in situ analysis (Supplemental Fig.             this analysis suggests that genes repressed by Spt5 tend to be <br /> S1) and 28 other genes were further validated through quantitative            shorter in length. <br /> RT-PCR analysis (Supplemental Tables S1 and S2). <br />    To identify biological pathways that might be regulated by Spt5,           In vivo chromatin immunoprecipitation (ChIP) reveals <br /> we employed GenMAPP [41] and associated tools [42] (http://                   Spt5 occupancy patterns on differentially expressed <br /> www.genmapp.org/go_elite) to analyze pathway and gene ontol- <br />                                                                               genes in 24 hpf WT embryos <br /> ogy over-representation in our differentially expressed gene <br />                                                                                  As microarray experiments measure the steady-state levels of <br /> dataset. This analysis found that genes induced by environmental <br />                                                                               mRNA and Spt5 is involved in many aspects of mRNA stability by <br /> or cellular stress (such as gadd45b and hsp70) or encoding certain <br />                                                                               virtue of its association with many components of the transcriptional <br /> transcription factors (such as foxd5 and fos) were upregulated, <br />                                                                               machinery , many of the genes differentially expressed in fogsk8 <br /> whereas genes that are constituents of the extracellular matrix (e.g. <br />                                                                               mutants might represent mRNA processing targets of Spt5. In <span id='am-49' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> <br /> collagen) and sodium/potassium ion transporters (e.g. atp1a1a) were <br />                                                                               to better identify genes that are regulated at the elongation level at <br /> down-regulated in fogsk8 embryos (Fig. 2A). The expression of <br />                                                                               24 hpf, we performed chromatin immunoprecipitation (ChIP) <br /> genes previously suggested to be regulated at the level of <br />                                                                               assays using antibodies against RNAPII and Foggy. <br /> elongation [10], including hsp70, fos, globin, tubulin, and opsin, were <br /> found altered in fogsk8 embryos. Moreover, the expression of                     Based on the current model of regulation at the level of <br /> multiple genes in the cell cycle pathway was reduced in fogsk8                transcription elongation [16,17], the following patterns of Spt5 <br /> embryos (Fig. 2B). In contrast, genes from major developmental                and RNAPII occupancy are expected on their target genes (Fig. 4): <br /> pathways (e.g. fibroblast growth factor, wingless), although present          A) Spt5 target genes that are in a state of productive transcription <br /> on the array, were largely unaffected in fogsk8 embryos (Supple-              elongation shall display Spt5 and RNAPII occupancy at both 59 and <br /> mentary Table S3). In addition, RNA processing genes and                      39 ends of their chromatin. B) Spt5 target genes that are repressed by <br /> protein biosynthesis genes (ribosome constituents) were under-                a promoter-proximal transcription elongation block shall have <br /> represented in our differentially expressed gene dataset (data not            minimal Spt5 and RNAPII occupancy at 39 end but significantly <br /> shown). It is worth pointing out that many differentially expressed           more Spt5 and RNAPII occupancy at 59 end. C) Genes that are not <br /> were novel, and were thus not assignable to specific GenMAPP                  Spt5 targets or are transcriptionally inactive shall lack detectable <br /> pathways. Taken together, these results indicate that the lack of             Spt5 or RNAPII respectively throughout their chromatin. <br /> zygotic Spt5 protein only affects the expression of a small                      Based on the significance of expression fold changes, constraints <br /> percentage of genes at 24 hpf, and these genes are involved in a              on availability of reliable genomic structure (exon/intron) <br /> number of distinct biological pathways.                                       information, and known gene functionality, 11 upregulated genes, <br />                                                                               7 downregulated genes, and 3 control genes were selected for <br />                                                                               ChIP analysis (Table 1 and Supplementary Table S4) in 24 hpf <br /> Genes that are upregulated in fogsk8 embryos are shorter <br />                                                                               WT embryos. Among the 11 upregulated genes, 8 (gadd45b, fos, <br /> in gene length compared to all genes examined                                 hsp70, smo, atf3, fst, tpbgl, foxd5) showed high Spt5 occupancy at 59 <br />    Little is known as to what signatures may subject a gene to Spt5           end but little at 39 end, whereas the remaining three genes did not <br /> regulation. To determine if differentially expressed genes have <br />                                                                               show significant Spt5 occupancy on either end (Fig. 5A and <br /> common features in gene structures, we performed bioinformatics <br />                                                                               Table 1). Among the 7 downregulated genes, 3 (lfng, tpma, a2bp1- <br /> analysis on these gene sets. To assess the extent and overall <br />                                                                               like) showed significant Spt5 occupancy at both 59 and 39 ends, <br /> likelihood of a difference between the length of fogsk8 &#8211;dependent <br />                                                                               whereas the remaining 4 genes did not (Fig. 5B and Table 1). <br /> genes and that of all other genes in the array, probe sets from the <br />                                                                               Genes that showed significant Spt5 occupancy are likely to be <br /> microarray were linked to gene structure data from the Ensembl <br />                                                                               targets of Spt5, whereas those that did not have significant Spt5 <br /> database, the mean of different gene features (e.g., genomic length) <br />                                                                               occupancy were either transcriptionally inactive or not directly <br /> was computed, and fold change and P values were calculated via a <br />                                                                               regulated by Spt5. Two (<span id='am-1' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-6'  ></span>beta</span>-actin 1 and cyclin E) of the three <br /> permutation based analysis (Fig. 3A). <br />                                                                               control genes showed significant Spt5 occupancy (Fig. 5C and <br />    Using this method, we found that genes upregulated in fogsk8 <br />                                                                               Table 1), whose expression was largely unaltered in 24 hpf fogsk8 <br /> embryos had on average ,2.39 fold decrease in overall gene <br />                                                                               embryos, suggesting that these transcripts might be regulated by <br /> length and ,1.69 fold decrease in intron length, as compared to <br />                                                                               additional elongation factors. <br /> all gene-linked probe sets on the array (P,0.05) (Fig. 3B). These <br /> differences were unlikely to occur due to chance, because a fold <br /> change larger than or equal to the gene length difference (2.39               ChIP analysis using the 8WG16 antibody in 24 hpf WT <br /> fold) was not observed in 10,000 random permutations of all                   embryos reveals RNAPII distribution patterns on the <br /> microarray probe sets, and that larger than or equal to the intron            differentially expressed genes <br /> length difference (1.69 fold) only occurred twice over 10,000                    We next examined RNAPII distribution in 24 hpf WT embryos <br /> random permutations (P = 0.0002), hence emphasizing the                       to determine the transcriptional status of the genes. The 8WG16 <br /> unlikelihood of such events occurring at random. Although this                antibody was used in our experiments, as it has been successfully <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         4                        November 2008 | Volume 3 | Issue 11 | e3621 <br />                                                                                                                         In Vivo Target Genes of Spt5 <br />  <br /> A <br />        Import Expression Data                                             Import Annotations <br />       Microarray Data                                                        probe set-Ens      Affymetrix-csv <br />                                   probe set <br />                               Sk8 +/+ mean exp <br />                               Sk8 mt fold change                             probe set-Ens      BioMart-Ensembl <br />                                Sk8 mt ttest pval <br />                                                                         Ens-gene length         BioMart-Ensembl <br />                                                                         Ens-exon length          (gene variables) <br />                                                                         Ens-intron length <br />                                                                         Ens-exon number <br />  <br />  <br />                            Link microarray data to gene/exon/intron annotations                   Step 1 <br />  <br />                                   Calculate Summary Statistics <br />  <br />                                         All probe    Up       Down <br />                                        sets (APS) regulated regulated <br />                        Store all unique gene variable values for each probe set list                   Step 2 <br />  <br />                     Calculate the mean gene variable value for all genes in each list                     Step 3 <br />  <br />         Repeat steps 2 and 3 for i &amp; j random                      Calculate fold &amp; ttest p by comparing <br />          elements from APS, 10,000 times                          filtered to unfiltered gene variables lists <br />  <br />  <br />        Determine the probability of each mean                   i = number of up regulated probeset list <br />           occuring by chance (permute p)                        j = number of down regulated probeset list <br />  <br />  <br />  <br /> B <br />                                              all                 sk8-down                             sk8-up <br />     Groups                                Mean             Fold          PermuteP              Fold         PermuteP <br />     Gene length (bp)                     18233             1.01              0.8576           -2.39                0 <br />     Exon number                             10             1.12              0.7372           -1.45            0.8956 <br />     Exon length (bp)                       182             1.06              0.1648            1.26            0.0158 <br />     Intron length (bp)                    2165             -1.17             0.0516           -1.69            0.0002 <br />     Genes regulated                       9955                        358                                 97 <br />     Probesets regulated                  15617                        485                                129 <br /> Figure 3. Permutation analysis uncovers a significant shorter length of those genes that are up-regulated in fogsk8 mutant embryos. <br /> (A) Algorithm design for assessing global changes in overall gene length. Unique Ensembl genes linked to probesets were obtained from the <br /> Affymetrix website and BioMart, and genomic gene, exon and intron length were also obtained from BioMart and summarized for up-, down- <br /> regulated and all array probesets (AAPs). Folds relative to AAPs linked genes were calculated as well as permutation based p-values by randomly <br /> selecting input probeset lists of the same size as the regulated sets from AAPs, over 10,000 iterations. (B) Table summarizing the permutation based <br /> analysis of the structural features of genes differentially expressed in fogsk8 embryos. Up-regulated genes have significantly shorter gene length, <br /> shorter intron length and longer exon length than other genes on the array. Down-regulated genes do not show significantly different gene structure <br /> requirements compared to other genes. <br /> doi:10.1371/journal.pone.0003621.g003 <br />  <br />        PLoS ONE | www.plosone.org                                        5                        November 2008 | Volume 3 | Issue 11 | e3621 <br />                                                                                                                                                 In Vivo Target Genes of Spt5 <br />  <br />  <br />                                                                                                      occupancy at different target genes were not clear (see Discussion). <br />  A. Spt5 or RNAPII occupancy at both 5&#8217; and 3&#8217;:                                                      Taken together, our in vivo expression profiling and ChIP analyses <br />     Active transcription                                       1       2       3       4 <br />                                                                                                      indicate that Spt5 exerts dual activity in regulating mRNA <br />                                                       5&#8217;                                   3&#8217;&#8217;       expression and RNAPII occupancy in vivo and in a gene-specific <br />  B. Higher Spt5 or RNAPII occupancy at 5&#8217; end:                                                       manner. <br />     Poised transcription                                   1       2       3       4 <br />                                                      5&#8217;                                    3&#8217;        gadd45b mRNA is globally increased leading to increased <br />  C. Little Spt5 or RNAPII occupancy at either end: <br />    Little transcription <br />                                                            1       2       3       4                 protein products in fogsk8 embryos <br />                                                       5&#8217;                                   3&#8217; <br />                                                                                                         The one gene that did show increased RNAPII occupancy in <br />                                                                                                      the fogsk8 mutant, gadd45b, is regulated by multiple important <br />                                                                                                      cellular signaling pathways including NF-kB and JNK [47] and is <br /> Figure 4. Schemata showing predicted transcriptional status <br /> based on Spt5 and RNAPII occupancy.                                                                  involved in zebrafish somitogenesis [48]. To probe the significance <br /> doi:10.1371/journal.pone.0003621.g004                                                                of Spt5 regulation of this gene, we asked whether and where in the <br />                                                                                                      fogsk8 embryos the de-repression of gadd45b results in overall <br /> used to recognize RNAPII occupancy on the chromatin at both                                          mRNA increase, through whole-mount in situ hybridization. In <br /> the 59 and 39 ends of genes in other <span id='am-46' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-47' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> [43]. In general, the                                   16 hpf WT embryos, gadd45b was strongly expressed in the <br />                                                                                                      developing somites, and such expression was largely normal in the <br /> RNAPII occupancy pattern correlated well with that of Spt5. <br />                                                                                                      fogsk8 mutant (Fig. 8A&#8211;B). In 24 hpf WT embryos, gadd45b <br /> Among those that were upregulated in the fogsk8 mutant embryo <br />                                                                                                      expression was detected in cells of pronephric ducts and weakly <br /> and also displayed significant Spt5 occupancy (indicating that their <br />                                                                                                      in posterior paraxial mesoderm and minimally elsewhere (Fig. 8C, <br /> expression is repressed by Spt5), more RNAPII was detected at 59 <br />                                                                                                      8E, 8G), revealing the dynamic nature of gadd45b expression in <br /> than 39 end (Fig. 6A, 6D, and Table 1). This finding is consistent <br />                                                                                                      development. In 24 hpf fogsk8 embryos, a global increase of gadd45b <br /> with the <span id='am-31' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-35'  ></span>idea</span> that the genes repressed by Spt5 are in a poised state <br />                                                                                                      RNA (Fig. 8D, 8F, 8H), and protein (Fig. 8I) was detected, thus <br /> of transcription. Among those that were downregulated in the <br />                                                                                                      demonstrating the significance of Spt5 regulation of this gene. <br /> fogsk8 mutant embryo and also displayed significant Spt5 <br /> occupancy, significant occupancy of RNAPII was also detected <br /> at both 59 and 39 ends (Fig. 6B, 6D, and Table 1), indicating that                                   Discussion <br /> these genes require the stimulatory activity of Spt5 for productive                                     Spt5 has a demonstrated dual activity in regulating transcription <br /> transcription in 24 hpf WT embryos. The genes that exhibited                                         elongation in vitro [21], and may regulate a large number of genes <br /> little Spt5 occupancy also showed little RNAPII occupancy,                                           in vivo [28,29,49]. However, the prevalence and identity of genes <br /> confirming that these genes are not transcriptionally active in                                      whose expression is critically dependent on Spt5 are poorly <br /> 24 hpf WT embryos. Taken together, our results identify gadd45b,                                     defined in any in vivo system. In this study, we have carried out <br /> fos, hsp70, tpbgl, smo, atf3, fst, and foxd5 as target genes that are                                expression profiling of over 10,000 genes using a zebrafish zygotic <br /> repressed by Spt5 at the elongation step, and lfng, tpma, and a2bp1-                                 null mutant, fogsk8. Maternally deposited foggy/spt5 gene products <br /> like as target genes that are activated by Spt5 at the elongation                                    have sustained fogsk8 embryos to a developmental stage when <br /> step. Though genes such as fos and hsp70 have been known to be                                       sufficient amount of tissues are experimentally available and foggy/ <br /> regulated by elongation, our results identify multiple critical                                      spt5 gene products become significantly depleted. Our results show <br /> developmental regulators such as smo (regulating Hedgehog                                            that a surprisingly small fraction of genes (,5%) are differentially <br /> signaling) [44], fst (regulating Transforming Growth Factor beta                                     expressed between WT and fogsk8 embryos at 24 hpf. In vivo ChIP <br /> signaling) [45], and lfng (regulating Notch signaling) [46] as novel                                 analyses with both Spt5 and RNAPII antibodies on a select set of <br /> elongation targets of Spt5.                                                                          genes further identify targets that are regulated either by the <br />                                                                                                      repressive or by the stimulatory activity of Spt5 at the level of <br /> Analysis of RNAPII occupancy on putative Spt5 target                                                 transcription elongation. The observation that Spt5 associates with <br /> genes in 24 hpf fogsk8 embryos                                                                       control genes confirm that Spt5 occupancy does not equate to <br />    Based on the above results and current model of transcription                                     functional necessity. Moreover, RNAPII occupancy patterns <br /> elongation, one might expect, in the fogsk8 embryos, an increased                                    observed in fogsk8 embryos emphasize the dynamic and complex <br /> occupancy of RNAPII at 39 end of the genes whose transcription                                       nature of transcription regulation by Spt5 in a gene-specific <br /> elongation is repressed by Spt5 in WT embryos. Alternatively,                                        manner in vivo. <br /> depletion of Spt5 may lead to a decreased association of RNAPII <br /> with the 59 end of genes, due to a reduced level of promoter-                                        The prevalence of target genes that require Spt5 <br /> proximal pausing, as has been observed in the case of NELF                                           regulation in vertebrate development <br /> depletion [8]. To test these predictions, in vivo ChIP was carried                                      Previous immunolocalization studies using Drosophila polytene <br /> out using RNAPII antibody in 24 hpf fogsk8 embryos. Due to                                           chromosomes show that non-productively elongating RNAPII, <br /> limited availability of fogsk8 embryos, we focused on five genes,                                    together with Spt5, labels many chromosomal sites [28,29]. <br /> hsp70, fos, gadd45b, fst, and smo. The patterns of RNAPII                                            Recent genome-wide analyses also find significant RNAPII <br /> occupancy in 24 hpf fogsk8 embryos were shown in Fig. 7. For                                         occupancy at the 59 end of many genes, whether or not their <br /> most genes tested, we observed significantly decreased RNAPII                                        transcripts can be detected [5&#8211;7]. These studies imply that most, if <br /> occupancy at 59 end and no increase of occupancy at 39 end, in                                       not all, genes may be dependent on Spt5 for transcription <br /> comparison to that in WT embryos, consistent with the second                                         regulation. In contrast to this view, our survey of over 10,000 <br /> prediction. One gene, gadd45b, however, fitted with the first                                        genes at one developmental stage (24 hpf) reveals less than 5% of <br /> prediction: RNAPII occupancy at its 59 end was similar between                                       differentially expressed genes in fogsk8 embryos. Several possibilities <br /> WT and fogsk8 mutant embryos, but RNAPII occupancy at its 39                                         may explain why such a small fraction of genes show altered <br /> end was significantly increased in fogsk8 embryos as compared to                                     expression in fogsk8 embryos: First, as localization does not always <br /> WT (Fig. 7). Reasons behind such differential patterns of RNAPII                                     reflect functional dependence, it is possible that, although a much <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                               6                        November 2008 | Volume 3 | Issue 11 | e3621 <br />                                                Table 1. Summary of Genes Used for in vivo CHIP Analysis. <br />  <br />  <br />                                                         Gene                                                                               Array/qRT-PCR         Genomic         Transcript    No. of   Degree of Spt5   Degree of RNAPII <br />                                               No.       Symbol      Gene Name                                      GO Terms*               Fold Change           length (kb)     length (kb)   Exons    occupancy        occupancy <br />  <br />                                                                                                                            Genes Up-regulated in 24 hpf fogsk8 Mutant Embryos <br />  <br />  <br />  <br />  <br /> PLoS ONE | www.plosone.org <br />                                               1         gadd45b     Growth arrest and DNA-damage-inducible, beta Stress response           6.34/6.07             1.6             1.2           4        ++++             ++++ <br />                                               2         fos         v-fos FBJ murine osteosarcoma viral oncogene   Transcription Factor    5.68/8.44             2.1             1.7           4        ++++             ++++ <br />                                               3         hsp70       Heat shock protein 70                          Stress response         5.6/2.32              3.4             2.3           2        +++              +++ <br />                                               4         fst         Follistatin                                    Pattern specification   2.25/2.07             5.5             3.4           5        +++              ++ <br />                                               5         smo         Smoothened homolog                             Pattern specification   2.02/2.12             5.6             3.0           12       +++              ++++ <br />                                               6         atf3        Activating transcription factor 3              Transcription Factor    2/1.89                6.1             2.6           4        +++              ++ <br />                                               7         tpbgl       Trophoblast glycoprotein-like                  Receptor                3.24/2.37             2.6             1.7           2        ++               +++ <br />                                               8         foxd5       Forkhead box D5                                Transcription Factor    9.83/10.77            4.1             1.1           2        +                + <br />                                               9         bapx        Bagpipe homeobox 1/ NK3 homeobox 2             Transcription Factor    4.62/4.62             0.5             0.3           2        2                2 <br />                                               10        zp2.4       Zona pellucida glycoprotein 2.4                Viral envelope          3.74/6.02             1.8             1.3           8        2                2 <br />  <br />  <br />  <br />  <br /> 7 <br />                                               11        opn1sw2     Opsin 1, short-wave-sensitive 2                Signal transduction     2.72/7.81             2.6             1.3           5        2                2 <br />                                                                                                                          Genes Down-regulated in 24 hpf fogsk8 Mutant Embryos <br />                                               1         lfng        Lunatic fringe homolog                         Signaling               3/3.68                9.4             2.9           8        ++++             ++++ <br />                                               2         tpma        Alpha tropomyosin                              Actin binding           2.86/3.41             4.5             1.3           10       +++              +++ <br />                                               3         a2bp1l      ataxin 2-binding protein 1-like                Nucleotide binding      10.69/7.83            18.5            1.1           12       ++               ++ <br />                                               4         ndrg1       N-myc downstream regulated gene 1              Cell differentiation    6.04/8.47             19.9            2.3           16       2                + <br />                                               5         ldb3l       LIM domain binding 3 like                      Protein binding         3.64/3.44             39.3            1.6           12       2                2 <br />                                               6         pvalb8      Parvalbumin 8                                  Calcium ion binding     17.62/12.23           174.7           0.8           5        2                2 <br />                                               7         atp1a1a.4   ATPase, Na+/K+ transporting, alpha 1a.4        Sodium\potassium        3.37/62.85            7.5             3.2           22       2                2 <br />                                                                                                                    ATPase <br />                                                                                                                                Genes Unchanged in 24 hpf fogsk8 Mutant Embryos <br />                                               1         bactin1     Beta Actin 1                                   Cytoskeleton            1.26/21.8             3.6             1.7           6        +++              ++++ <br />                                               2         ccne        Cyclin E                                       Cell cycle              1/1.87                9.8             2.0           12       ++               2 <br />                                               3         fkbp5       FK506 binding protein 5                        Protein folding         1/1.04                9.9             2.4           11       2                2 <br />  <br />                                               * <br />                                                GO/MAPP program-suggested gene function. <br />                                               doi:10.1371/journal.pone.0003621.t001 <br />                                                                                                                                                                                                                                             In Vivo Target Genes of Spt5 <br />  <br />  <br />  <br />  <br /> November 2008 | Volume 3 | Issue 11 | e3621 <br />                                                                              In Vivo Target Genes of Spt5 <br />  <br />  <br />  <br />                                  Figure 5. In vivo ChIP analysis using Spt5 antibody on <br />                                  differentially expressed genes reveals putative target genes <br />                                  directly bound by Spt5. (A&#8211;C) Spt5 occupancy on upregulated, <br />                                  downregulated, and control genes. Y- <span id='am-41' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-42' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45'  ></span>axis</span> values in the graphs <br />                                  represent fold Foggy occupancy over fold pre-immune serum <br />                                  occupancy levels. Arbitrary cutoff for detectable Foggy fold occupancy <br />                                  over control serum is set at 5. Thick black bars highlight the genes that <br />                                  display significant Spt5 occupancy at either 59 or 39 end of the <br />                                  chromatin. The results shown are the average of at least three <br />                                  independent replicates. Error bars are SEM values. Note that the scales <br />                                  are different for each of the graphs. <br />                                  doi:10.1371/journal.pone.0003621.g005 <br />  <br />                                  larger number of genes are bound by Spt5, only a fraction of them <br />                                  are critically dependent on its activity. Indeed, our ChIP analyses <br />                                  have found that Spt5 are associated with two genes whose <br />                                  expression is largely unchanged in fogsk8 embryos. Second, despite <br />                                  significant reduction of Spt5 protein level in fogsk8 embryos, some <br />                                  maternal proteins still remain, which could mask the identification <br />                                  of those genes whose expression is not particularly sensitive to Spt5 <br />                                  protein reduction. Although removal of maternal Spt5 will allow <br />                                  the test of this hypothesis, it may potentially lead to early death of <br />                                  embryos, rendering the experiments impossible to carry out. <br />                                  Third, since this expression profiling experiment cannot distin- <br />                                  guish the relative contribution of maternal versus zygotic mRNA <br />                                  to the steady state mRNA levels, it remains possible that maternal <br />                                  mRNAs that are not yet degraded by 24 hpf could mask changes <br />                                  of Spt5-dependent zygotic gene expression. Finally, since only one <br />                                  developmental stage is examined in this study, it remains possible <br />                                  that a different set of genes and/or a much larger number of them <br />                                  may show altered expression in the mutant at other developmental <br />                                  stages. Interestingly, when we analyzed the expression of 21 <br />                                  candidate target genes in a hypomorphic mutation (the fogm806 <br />                                  allele) at 48 hpf, 15 of the genes tested were unchanged while four <br />                                  (fos, ndrg1, atp1a1a.4 and pvalb) showed similar trend as in fogsk8 <br />                                  mutant (Supplemental Table S5). This observation suggests that <br />                                  the fogm806 allele is likely to affect even smaller number of genes. <br />                                  Taken together, we suggest that, at the late segmentation stage of <br />                                  zebrafish development, only a small fraction of genes is critically <br />                                  dependent on Spt5 for transcription regulation, even though a <br />                                  large number of genes may recruit Spt5 to their chromatin. <br />  <br />                                  Target genes that are positively or negatively regulated <br />                                  by Spt5 <br />                                     In addition to previously known categories of inducible genes, <br />                                  our pathway analysis revealed that Spt5 target genes belong to <br />                                  diverse biological pathways and functions. Further studies are <br />                                  required to bridge the gap between the known function of the <br />                                  genes to the mechanism of their gene regulation at the elongation <br />                                  level, particularly to developmentally regulated genes. Our <br />                                  bioinformatics analyses reveal that genes repressed by Spt5 are <br />                                  significantly shorter in gene length. Two possibilities, not mutually <br />                                  exclusive, may account for this observation: First, as one distinct <br />                                  advantage of regulating transcription elongation is to allow rapid <br />                                  gene induction in response to extrinsic signals, genes that are <br />                                  regulated in this manner may have been selected during evolution <br />                                  to have shorter length, in order to further accommodate the need <br />                                  for accelerated transcription. Interestingly, genes with shorter <br />                                  introns and overall shorter gene length are hallmarks of immediate <br />                                  early (IE) genes. Since only three of the genes up-regulated in the <br />                                  fogsk8 appear to be well- characterized IE genes (gadd45b, fos and <br />                                  jun), further studies are required to determine if other genes <br />                                  represented in this category also function as IE genes. Second, <br />                                  since the severe depletion of Spt5 in fogsk8 embryos lead to deficits <br />                                  of not only the repressive but also the stimulatory activity, it is <br />  <br />  <br />  <br />  <br /> Figure 6. Distinct patterns of RNAPII occupancy at putative Spt5 target genes. (A&#8211;D) ChIP data showing RNAPII occupancy in 24 hpf WT <br /> embryos on the genes that are up-regulated in fogsk8 embryos (A), genes that are down-regulated in fogsk8 embryos (B), control genes (C), and the <br /> ratio of RNAPII occupancy at 59 vs 39 end (D). Y-axis values in the graphs represent fold RNAPII occupancy over fold mouse control serum occupancy. <br /> Arbitrary cutoff for detectable RNAPII fold occupancy over control serum is set at 5. Thick black bars highlight the genes that display significant <br /> RNAPII occupancy at either 59 or 39 end of the chromatin. The results shown are the average of at least three independent replicates. Error bars are <br /> SEM values. <br /> doi:10.1371/journal.pone.0003621.g006 <br />  <br /> possible that the genes with longer length are not up-regulated in            induction also depends on Spt5 [32]. Thus, one gene may require <br /> fogsk8 embryos because of their dependency on both the repressive             both the repressive and the stimulatory activity of Spt5, depending <br /> as well as the stimulatory activity of Spt5.                                  on developmental states or surrounding environmental signals. <br />    Our expression profiling together with in vivo ChIP analyses at            Spt5 is shown to be phosphorylated by the positive elongation <br /> one developmental stage have identified two non-overlapping sets              kinase P-TEFb [14,34,36&#8211;38], and such phosphorylation is <br /> of target genes: those that are negatively regulated by Spt5 and              proposed to act as a switch that converts Spt5 from a repressor <br /> those that are positively regulated by Spt5. Are there indeed two             to an activator [34]. Although such model remains to be tested in <br /> distinct populations of genes that differentially require Spt5                vivo, it supports the idea that Spt5 and transcription elongation are <br /> activity in vivo? We suggest not. First, this study found that hsp70          dynamically regulated in vivo, and same set of genes may be <br /> belongs to the group that is negatively regulated by Spt5. Under              regulated by the repressive and stimulatory activity of foggy/Stp5, <br /> heat stimulus, however, it has been previously shown that hsp70               depending on the in vivo states. <br />  <br />  <br />       gadd45b               fos            hsp70                 fst              smo <br /> Figure 7. RNAPII occupancy on direct target genes of Spt5 in fogsk8 mutant embryos reveals unexpected complexity and increased <br /> 39 occupancy at the gadd45b locus. ChIP with RNAPII antibody in WT and fogsk8 embryos on gadd45b, fos, hsp70, fst, and smo genes show that in <br /> fogsk8 embryos, RNAPII occupancy is significantly increased at 39 end of gadd45b without increase at 59 end. Other genes tested showed reduction of <br /> RNAPII occupancy at 59 ends but no change at 39 ends in fogsk8 embryos compared to WT. The results shown are the average of at least three <br /> independent replicates. Error bars are SEM values. <br /> doi:10.1371/journal.pone.0003621.g007 <br />  <br />  <br /> Dynamics and complexity of regulated transcription                           late segmentation stages, fogsk8 embryos do not show obvious <br /> elongation in vivo                                                           segmentation defects and no reduction in myoD transcript levels <br />    Our ChIP analyses in fogsk8 embryos reveal decreased RNAPII               was observed in our expression profiling analysis. Thus, under- <br /> occupancy along the chromatin of four genes (fos, hsp70, fst, and            standing the contribution of overexpressed gadd45b or any other <br /> smo) among total five genes tested. Since the transcripts of these           target genes to the fogsk8 mutant phenotype would require a precise <br /> genes are increased in fogsk8 embryos at 24 hpf, together with the           determination of temporal profiles of their expression in the fogsk8 <br /> demonstrated RNAPII and Spt5 occupancy on these genes in WT                  mutant, followed by conditional knockdown or overexpression of <br /> embryos, one possible explanation is that increased 39 RNAPII                these target genes with temporal control. <br /> occupancy may have occurred at an earlier developmental stage,                  How might the control of gadd45b expression by Spt5 fit into the <br /> coinciding with the onset of transcript increase from these genes.           known regulation of gadd45b? Gadd45b is regulated by NF-kappaB <br /> Because of Spt5 depletion, promoter-proximal association of                  [51] and TGF-beta [52] in mammalian cell cultures, though its in <br /> RNAPII was reduced subsequently in fogsk8 mutant embryos,                    vivo regulation is poorly understood. Future studies are required to <br /> resulting in observed decrease of RNAPII occupancy at 24 hpf.                elucidate any potential interaction between Spt5 and the NF-kB <br /> Indeed, it has been shown in yeast that Spt4-5 complex is                    and/or TGF-beta pathway members to understand the interplay <br /> important for stabilizing RNAPII on the chromatin template                   between DNA binding transcription factors and elongation <br /> under normal and DNA-damage inducing conditions [50]. Similar                regulation of gadd45b. It has been previously suggested that gadd45 <br /> reduction of promoter-proximal associated RNAPII has also been               falls into the category of &#8216;preset&#8217; genes whose upregulation is <br /> observed after NELF knockdown in Drosophila [8].                             independent of changes in chromatin structure (by binding of <br />    Among all the up-regulated genes subjected to ChIP analysis,              transcription factors), or synthesis of new transcription factors, or <br /> gadd45b showed increased RNAPII occupancy at the 39 end in the               stimuli-induced DNA-protein interactions [53]. Hence, it is <br /> fogsk8 mutant embryos. It also exhibited highest Spt5 occupancy at           tempting to speculate that the elongation control of gadd45b by <br /> the 59 end in WT (Figure 5A). In addition, the onset of gadd45b              Foggy/Spt5 is the switch that regulates these &#8216;preset&#8217; genes. <br /> mRNA increase in fogsk8 embryos may be rather close to 24 hpf,               Transcription elongation control of such &#8216;preset&#8217; genes might be <br /> since no increase in gadd45b mRNA was detected in 16 hpf fogsk8              crucial to integrate signals from multiple pathways (gadd45b <br /> embryos. These observations may explain why gadd45b displays                 controlled by NF-kB and TGF-b) to coordinate regulation of <br /> increased RNAPII occupancy patterns in fogsk8 embryos. The                   important signaling pathways that ultimately determine cell fate <br /> significantly up-regulated Gadd45b protein levels suggest that the           and survival. <br /> produced gadd45b transcripts can be translated in vivo. In zebrafish,           Taken together, our findings provide the first comprehensive <br /> overexpression of gadd45b by mRNA injection at the 1-cell stage              view of in vivo target genes whose transcription is critically <br /> leads to somite segmentation defect and a down-regulation of myoD            dependent on Spt5 both positively and negatively during <br /> [48]. Since gadd45b was not misexpressed in fogsk8 embryos until             vertebrate development. Further studies are required to elucidate <br />  <br />  <br />        PLoS ONE | www.plosone.org                                       10                        November 2008 | Volume 3 | Issue 11 | e3621 <br />                                                                                                                          In Vivo Target Genes of Spt5 <br />  <br />  <br />  <br />                                                                                 Materials and Methods <br />                                                                                 Zebrafish strains and husbandry <br />                                                                                    Wild-type AB and fogsk8 embryos were obtained through natural <br />                                                                                 spawning and were maintained and staged according to <br />                                                                                 established procedures [54]. The use of zebrafish was approved <br />                                                                                 by UCSF ethics committee. <br />  <br />                                                                                 Total RNA isolation and quantitative RT-PCR <br />                                                                                    Equal number of embryos (WT siblings vs. fogsk8 mutants) at <br />                                                                                 24 hpf was dechorionated manually or by Pronase K treatment. <br />                                                                                 Trizol (Invitrogen) was used to extract total RNA that was DNase- <br />                                                                                 treated (Qiagen) to minimize contamination by genomic DNA <br />                                                                                 during qRT-PCR. To obtain cDNA, total RNA was reverse- <br />                                                                                 transcribed using oligodT and Superscript (Invitrogen). Equal <br />                                                                                 amount of cDNA was used in SybrGreen qPCR mastermix system <br />                                                                                 (Applied Biosystems) with appropriate primers (Supplemental <br />                                                                                 Table S2). All plates contained at least two-well duplicates for <br />                                                                                 each primer and cDNA condition. Cycle values were averaged <br />                                                                                 between the duplicates and normalized to control primer values. <br />                                                                                 WT fold values were compared to fogsk8 mutant fold values and <br />                                                                                 expressed as fold upregulated or downregulated in fogsk8 embryos. <br />                                                                                 At least two independent replicates were performed for each <br />                                                                                 primer set. <br />  <br />                                                                                 Microarray Data Analysis <br />                                                                                    Microarray hybridization was performed at the NINDS <br />                                                                                 microarray consortium using zebrafish Affymetrix oligonucleotide <br />                                                                                 arrays. Three independent replicates for each sample were <br />                                                                                 performed. Gene intensity values for the Affymetrix Zebrafish <br />                                                                                 Genome Array CEL files were obtained using the GC-RMA <br />                                                                                 package from Bioconductor [55]. To identify genes with relative <br />                                                                                 changes in steady-state mRNA levels between WT and fogsk8 <br />                                                                                 mutants, a fold change was calculated from these log2 intensity <br />                                                                                 values along with a student t test p value. Genes with 2 fold change <br />                                                                                 in expression and p,0.05 were clustered using the program <br />                                                                                 HOPACH (hierarchical ordered partitioning and collapsing <br />                                                                                 <span id='am-57' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-58'  ></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="hybrid" datatype="xsd:string"></span>hybrid</span>), with uncentered correlation distance [56,57]. Associations <br />                                                                                 with Gene Ontology, GenMAPP [58] and PANTHER [59]path- <br />                                                                                 ways were obtained using MAPPFinder 2.0 [42], a part of the <br />                                                                                 GenMAPP 2.1 [41] application package and further filtered and <br />                                                                                 annotated using the program GO-Elite (http://www.genmapp.org/ <br />                                                                                 go_elite/go_elite.html). The raw image and processed expression of <br />                                                                                 our microarray dataset is posted at NCBI&#8217;s Gene Expression <br />                                                                                 Omnibus database (GSE12826). To determine relative differences <br />                                                                                 in gene length parameters, a custom <span id='am-62' about='obo:NCBITaxon_37579' typeof='owl:Thing obo:NCBITaxon_1329950 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8504 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8509 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_1329912 obo:NCBITaxon_33208 obo:NCBITaxon_1329961 obo:NCBITaxon_1329911 obo:NCBITaxon_7711 obo:NCBITaxon_8570 obo:NCBITaxon_34979 obo:NCBITaxon_7776 obo:NCBITaxon_34984 obo:NCBITaxon_32561 obo:NCBITaxon_8457'><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasAlternativeId" content="NCBITaxon:51884" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="Python" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67'  ></span>python</span> script was written that <br />                                                                                 links microarray probe set data to gene structural information <br />                                                                                 from the Ensembl database (PMID: 17148474) obtained through <br />                                                                                 BioMart [60]. Probe set annotations to Ensembl genes obtained <br /> Figure 8. Global increase of gadd45 mRNA and increased                          from BioMart were augmented with those directly from <br /> Gadd45b protein in fogsk8 mutant embryos. (A&#8211;H) Whole-mount                     Affymetrix (http://www.affymetrix.com). In this script, fold <br /> in situ hybridization using the gadd45b antisense probe in WT and               changes were calculated by comparing the mean length of distinct <br /> fogsk8 embryos at 16 hpf and 24 hpf. gadd45b expression is not                  gene features (genomic, exon and intron lengths/number) or gene <br /> increased at 16 hpf but shows widespread increase at 24 hpf in fogsk8 <br /> embryos. (I) Western blot analysis of WT and fogsk8 embryo extracts with        expression between upregulated or downregulated Ensembl-linked <br /> commercially available anti-Gadd45b antibody and anti-Actin antibody            genes compared to all genes on the microarray. For each <br /> shows significant increase of Gadd45b protein in fogsk8 embryos at              parameter, a permutation p value was calculated by randomizing <br /> 24 hpf.                                                                         the selection of input probe sets 10,000 times and re-calculating <br /> doi:10.1371/journal.pone.0003621.g008                                           and storing the mean fold changes for both up and down regulated <br />                                                                                 sets to determine the probability that the original fold occurs by <br />                                                                                 chance alone. <br />  <br /> the complexities of elongation regulation through Spt5 in vivo, as              Chromatin Immunoprecipitation (ChIP) Assay <br /> well as to determine the significance of such regulation on                        WT embryos at 24 hpf were dechorionated by Pronase <br /> individual target genes and their associated pathways.                          treatment and washed thoroughly. Embryos were fixed in 1.5% <br />  <br />  <br />  <br /> formaldehyde for 20 minutes and then homogenized 25 times in                                 secondary antibody conjugated to horse-radish peroxidase (HRP) <br /> 56 extraction buffer with protease inhibitors on ice. Sonication                             were used. The ECL kit (Amersham) was used to visualize the <br /> was carried out with Branson sonifier 250 on an ice bath for 10                              protein bands transferred from the blot to film. <br /> times, 10 seconds each with 20 second breaks. Sonicated samples <br /> were then centrifuged at 14k rpm at 4uC for 30 minutes to                                 <br /> separate the pellet from the supernatant. The supernatant was <br /> immunoprecipitated with the following antibodies: 8WG16 Ab-                                  <br /> 1:40 (Covance Research Products), Anti-Spt5 antibody 1:10, and                             <br /> control sera -Normal mouse serum-1:100 (Sigma) and Rabbit pre-                             <br /> immune serum (1:10) for 2&#8211;4 hours in the cold room. Protein A                              <br /> Sepharose (Sigma) beads were added to the samples and left on                              <br /> overnight. The next day, extensive washing with different buffers                         <br /> were performed before eluting the beads at 65uC for 20 minutes. <br /> Proteinase K and RNase were added to the eluates and reverse-                              <br /> crosslinked overnight at 65uC. Phenol/chloroform extraction                               <br /> followed by ethanol precipitation was performed. The immuno-                               <br /> precipitated DNA was immediately used for qPCRs with                                       <br /> appropriate primers (Supplemental Table S4). All cycle values                                <br /> were first normalized to input values before comparing control                         <br /> samples to immunoprecipitated samples. These values were <br /> represented as fold occupancy over control values. The procedure                           <br /> was performed similarly for fogsk8 embryos.                                              <br /> Whole mount in situ hybridization and                                                     <br /> immunocytochemistry                                                                         <br />    Whole-mount in situ hybridization was performed as previously                           <br /> described [61]. Plasmid containing pBS-gadd45b was obtained from <br /> ZIRC, and antisense probe was made using T7 RNA polymerase                              <br /> (Ambion). Whole-mount immunocytochemistry was performed <br />                                                                                 <br /> similarly with appropriate antibodies (8WG16 ab- 1:100; Anti- <br />                                                                                             <br /> zfSpt5 antibody &#8211; 1:5000; CTD4H8 &#8211;AF488 &#8211; 1:2500) and                                      <br /> fluorescent secondary antibodies. Images were taken with Zeiss                               <br /> epi-fluorescent and confocal microscopes.                                                    <br /> Western Blot                                                                                <br />    Dechorionated embryos were fixed in 1.5% formaldehyde for <br /> 20 minutes and homogenized before addition of 26 SDS loading                             <br /> buffer. Equal amounts of the extracts were loaded on to the wells                        <br /> of the gel. Primary antibodies (Anti-Foggy antibody-1:6000; Anti-                          <br /> Actin antibody &#8211; 1:2000; Anti-Gadd45b antibody &#8211; 1:100) and                             <br />  <br /> </body></html>