<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Interaction ZPR1 Translation Elongation <br /> Factor-1a Proliferating Cells <br /> Laxman Gangwani, Monique Mikrut, Zoya Galcheva-Gargova, <span id='am-159' about='obo:NCBITaxon_1009818' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_263623 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_71526 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_7080 obo:NCBITaxon_50557 obo:NCBITaxon_115347 obo:NCBITaxon_7041'><span id='am-160' property="rdfs:label" content="Roger" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-163'  ></span>Roger</span> J. Davis <br /> Howard Hughes Medical Institute Program Molecular Medicine, Department Biochemistry Molecular Biology, <br /> University Massachusetts Medical School, Worcester, Massachusetts 01605 <br />  <br />  <br />  <br />  <br /> Abstract. The zinc finger protein ZPR1 present                          essential viability S. cerevisiae. The yeast <br /> the cytoplasm quiescent mammalian cells trans-                         ZPR1 protein redistributes cytoplasm the <br /> locates nucleus treatment mitogens,                          nucleus response nutrient stimulation. Disruption <br /> including epidermal growth factor (EGF). Homologues                           binding ZPR1 eEF-1a mutational analy- <br /> of ZPR1 identified yeast mammals. These                           sis resulted accumulation cells G2/M <br /> ZPR1 proteins bind eukaryotic translation elonga-                          phase cell cycle defective growth. Reconstitution <br /> tion factor-1a (eEF-1a). Studies mammalian cells                           ZPR1 interaction eEF-1a restored normal <br /> demonstrated EGF treatment induces interac-                          growth. We conclude ZPR1 essential cell via- <br /> tion ZPR1 eEF-1a redistribution                            bility interaction eEF-1a contributes <br /> both proteins nucleus. In yeast Saccharomy-                        normal cellular proliferation. <br /> ces cerevisiae, genetic analysis demonstrated ZPR1 <br /> is essential gene. Deletion analysis demonstrated                          Key words: cell cycle &#8226; eEF-1a &#8226; EGF &#8226; Saccharomy- <br /> that NH2-terminal region ZPR1 required                          ces cerevisiae &#8226; zinc finger <br /> normal growth COOH-terminal region <br />  <br />  <br />  <br />  <br /> T <br />         zinc finger protein ZPR1 located                         mitogen-stimulated release ZPR1 receptors <br />         cytoplasm quiescent mammalian cells (Galcheva-                     trigger accumulation ZPR1 nucleus. Al- <br />         Gargova et al., 1996). Treatment mitogens,                       plausible hypothesis, appears <span id='am-103' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-104' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-106'  ></span><span id='am-107' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> <br /> including epidermal growth factor (EGF), causes re-                       possible mechanism represents oversimplification of <br /> distribution ZPR1 cytoplasm nucleus                        processes control subcellular localization of <br /> (Galcheva-Gargova et al., 1996). The function ZPR1                      ZPR1. First, unclear relative abundance <br /> the nucleus defined. However, nuclear                        receptors ZPR1 sufficient account cyto- <br /> redistribution ZPR1 proliferating cells suggests                   plasmic sequestration ZPR1 quiescent cells. Second, <br /> this protein act signaling molecule communi-                    ZPR1 lacks obvious nuclear localization sequence that <br /> cates mitogenic signals cytoplasm nucleus.                    direct nuclear accumulation ZPR1 released <br />    The mechanism cytoplasmic retention ZPR1                          receptor complexes. Third, present study, we <br /> quiescent cells unclear. ZPR1 binds group ty-                      nuclear localization ZPR1 regulated in <br /> rosine kinase receptors, including EGF platelet-                      cells lack tyrosine kinase receptors. These consider- <br /> derived growth factor receptors (Galcheva-Gargova et al.,                     ations suggest interaction ZPR1 bio- <br /> 1996). The binding ZPR1 receptors blocked                      molecules contribute regulation subcel- <br /> after mitogen stimulation (Galcheva-Gargova et al., 1996).                    lular distribution ZPR1. <br /> It possible ZPR1 sequestered                         Although known ZPR1 binds cell surface re- <br /> the cytoplasm receptors quiescent cells                    ceptor molecules quiescent cells, established <br />                                                                               effect mitogens inhibit ZPR1/recep- <br />                                                                               <span id='am-149' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-150' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-153'  ></span>tor</span> interaction represents formation novel ZPR1 <br />                                                                               complex different protein. The purpose study <br />                                                                               described report identify protein inter- <br />                                                                               acts ZPR1 treatment cells mitogens. This <br /> Address correspondence R.J. Davis, Howard Hughes Medical Institute, <br /> Program Molecular Medicine, University Massachusetts Medical            study facilitated identification ZPR1 <span id='am-108' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-109'  ></span><span id='am-110' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>homo</span>- <br /> School, 373 Plantation St., Worcester, MA 01605. Tel.: (508) 856-6054.        logues mammals yeast. <br /> Fax: (508) 856-3210. E-mail: roger.davis@ummed.edu                               We report ZPR1 binds eukaryotic translation <br />  <br />  <br />  <br /> &#63721; The Rockefeller University Press, 0021-9525/98/12/1471/14 $2.00 <br /> The Journal Cell Biology, Volume 143, Number 6, December 14, 1998 1471&#8211;1484 <br /> http://www.jcb.org                                                             1471 <br />  Table I. <span id='am-169' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-170' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-171' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-173' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Plasmids</span> <br /> Plasmids                                                 Description                                                    Source <br />  <br /> YCp50                                   URA3, ARS1, CEN4                                                   Rose et al., 1987 <br /> pRS317                                  LYS2, ARSH4, CEN6                                                  Sikorski Boeke, 1991 <br /> pRS405                                  LEU2                                                               Sikorski Hieter, 1989 <br /> pRS426                                  URA3, 2 m                                                          Christianson et al., 1992 <br /> p413ADH                                 HIS3, ARSH4, CEN6 ADH promoter                                American Type Culture Collection <br /> pLG1                                    YCp50 genomic cZPR1                                           This study <br /> pLG2                                    pBS cZPR1                                                     This study <br /> pLP1                                    pGEX-3X cZPR1                                                 This study <br /> pLP2                                    pGEX-5X-1 TEF1                                                This study <br /> pLP3                                    pCAL-n cZPR1                                                  This study <br /> pLP4                                    pCAL-n mZPR1 (1&#8211;774)                                          This study <br /> pLPNT                                   pCAL-n cZPR1 (1&#8211;780)                                          This study <br /> pLPCT                                   pCAL-n cZPR1 (781&#8211;1,458)                                      This study <br /> pLPD1                                   pCAL-n cZPR1 D (664&#8211;723)                                      This study <br /> pLPD2                                   pCAL-n cZPR1 D (664&#8211;783)                                      This study <br /> pLDP3                                   pCAL-n cZPR1 D (604&#8211;723)                                      This study <br /> pLDP4                                   pCAL-n cZPR1 D (604&#8211;783)                                      This study <br /> pLPD5                                   pCAL-n cZPR1 D (604&#8211;663)                                      This study <br /> pLC1                                    p413ADH cZPR1                                                 This study <br /> pLCNT                                   p413ADH cZPR1 (1&#8211;780)                                         This study <br /> pLCCT                                   p413ADH cZPR1 (781&#8211;1,458)                                     This study <br /> pLCCT2                                  pRS317 ADH promoter, cZPR1 (781&#8211;1,458)                        This study <br /> pLCD1                                   p413ADH cZPR1 D (664&#8211;723)                                     This study <br /> pLCD2                                   p413ADH cZPR1 D (664&#8211;783)                                     This study <br /> pLCD3                                   p413ADH cZPR1 D (604&#8211;723)                                     This study <br /> pLCD4                                   p413ADH cZPR1 D (604&#8211;783)                                     This study <br /> pLCD5                                   p413ADH cZPR1 D (604&#8211;663)                                     This study <br /> pLCNT2                                  p413ADH cZPR1 (1&#8211;780) D (604&#8211;663)                             This study <br /> pLCG1                                   p413ADH cZPR1 D (604&#8211;663), GFP                                This study <br /> pLCG2                                   p413ADH cZPR1 (1&#8211;780), GFP                                    This study <br /> pLCG3                                   p413ADH cZPR1 (781&#8211;1,485), GFP                                This study <br /> pMM6                                    pRS405 cZPR1 (2785 210), (970&#8211;1,471)                       This study <br /> pMM13                                   pRS426 GAL promoter, cZPR1                                    This study <br /> pMM29                                   p413ADH cZPR1 (294, 297 C/A)                                  This study <br /> pMM36                                   p413ADH cZPR1 (54, 57 C/A)                                    This study <br /> pMM38                                   pRS317 ADH promoter, cZPR1 (1&#8211;780)                            This study <br /> pMM48                                   p413ADH GFP                                                   This study <br /> pMM49                                   p413ADH cZPR1, GFP                                            This study <br /> pZPP                                    pGEX-3X pZPR1                                                 This study <br /> pLZM1&#8211;7                                 pGEX-5X-1 mZPR1 (fragments 1&#8211;7)                               Galcheva-Gargova et al., 1996 <br /> pZM2                                    pCDNA3 Flag-mZPR1                                             Galcheva-Gargova et al., 1996 <br /> pZPM                                    pGEX-3X mZPR1                                                 Galcheva-Gargova et al., 1996 <br />  <br />  <br /> elongation factor-1a (eEF-1a).1 EGF stimulation mam-                    binding ZPR1 eEF-1a mutational analysis <br /> malian cells triggers formation ZPR1/eEF-1a com-                       caused accumulation cells G2/M phase the <br /> plexes translocation ZPR1 eEF-1a                       cell cycle defective growth. Reconstitution the <br /> from cytoplasm nucleus. Similarly, yeast                 ZPR1 interaction eEF-1a inter-allelic complemen- <br /> S. cerevisiae, ZPR1 redistributes cytoplasm                    tation restored normal growth. These <span id='am-164' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-165' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-167'  ></span><span id='am-168' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> establish role <br /> and nucleus response nutrient stimulation. Dis-                  formation nuclear ZPR1/eEF-1a complexes in <br /> ruption ZPR1 gene lethal S. cerevisiae. Dele-                 normal growth proliferating cells. <br /> tion analysis demonstrated ZPR1 divided                      Previous studies demonstrated eEF-1a plays a <br /> two functional domains. The NH2-terminal region                         pivotal role protein biosynthesis (Moldave, 1985). In ad- <br /> ZPR1 required normal growth COOH-                       dition, eEF-1a known involved diverse cellular <br /> terminal region essential viability. Disruption              processes, including embryogenesis (Krieg et al., 1989), se- <br />                                                                            nescence (Shepherd et al., 1989), oncogenic transforma- <br />                                                                            tion (Tatsuka et al., 1992), cell proliferation (Sanders et <br /> 1. Abbreviations used paper: CBP, calmodulin-binding protein;      al., 1996), organization cytoskeleton (Condee- <br /> CT, COOH-terminal region ZPR1; cZPR1, S. cerevisiae ZPR1; DAPI,         lis, 1995; Edmonds et al., 1996). eEF-1a present mainly <br /> 496-diamidino-2-phenylindole; eEF-1a, eukaryotic translation elongation    cytoplasm cells (Edmonds et al., 1996; Minella et <br /> factor-1a; GFP, green fluorescent protein; GST, glutathione-S-trans- <br /> ferase; mZPR1, mouse ZPR1; NT, NH2-terminal region ZPR1; PMA, <br />                                                                            al., 1996), small population eEF-1a molecules has <br /> phorbol 12-myristate 13-acetate; PVDF, polyvinylene difluoride; pZPR1,     previously identified nucleus (Colling et al., <br /> S. pombe ZPR1.                                                             1994; Janssen et al., 1994; Barberse et al., 1995; Billaut- <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 143, 1998                              1472 <br />  Mulot et al., 1996; Sanders et al., 1996). Recently, eEF-1a                  ADH SmaI/EcoRI sites create plasmid pMM48. A cZPR1 <br /> was shown bind viral RNA (Blackwell Brinton,                          fragment (BglII-SmaI) amplified PCR cloned pMM48 the <br />                                                                              BamHI/SmaI sites create plasmid pMM49. The S. cerevisiae gene <br /> 1997) RNA polymerases (Blumenthal Carmi-                             encoding translation elongation factor eEF-1a (TEF1) amplified by <br /> chael, 1979; Das et al., 1998).                                              PCR using genomic DNA template cloned pGEX-5X-1 <br />   The involvement eEF-1a nuclear processes re-                         (Pharmacia Biotech) BamHI site. In-frame deletions prepared <br /> mains established. The results present study                    using standard PCR-based procedures. The plasmids used study are <br />                                                                              summarized Table I. <br /> suggest physiological functions eEF-1a                       Recombinant plasmids express fusion proteins calmodulin- <br /> nucleus include complex formation ZPR1 mi-                       binding protein (CBP) glutathione-S-transferase (GST) ex- <br /> togen-treated cells.                                                         pressed BL21 (DE3) bacteria BL21 bacteria, respectively. The <br />                                                                              fusion proteins induced 1 mM IPTG 2 h 308C were <br />                                                                              purified affinity chromatography. <br /> Materials Methods <br />                                                                              Disruption ZPR1 Gene <br /> Preparation Recombinant Plasmids Proteins                             Yeast manipulated according standard procedures. The diploid <br /> The yeast ZPR1 gene isolated screening S. cerevisiae genomic       strain (MY2) heterozygous ZPR1 gene disruption constructed <br /> library (prepared Rose et al., 1987) using colony screening. A 300-bp     using established techniques (Rothstein, 1993). The recombinant plasmid <br /> fragment, amplified using polymerase chain reaction (PCR),      pMM6 prepared inserting fragments cZPR1 gene (2785 <br /> right arm chromosome VII S. cerevisiae used preparing      bp 210 bp 1970 bp 11,471 bp) flanking LEU2 gene the <br /> [32P]labeled probe hybridization (Prime-It II; Stratagene, La Jolla,     plasmid pRS405 (Sikorski Hieter, 1989). The ScaI-SpeI fragment <br /> CA). Positive clones examined automated sequencing (model            pMM6 containing disrupted cZPR1 gene transformed into <br /> 373A; Applied Biosystems, Foster City, CA). The S. pombe zpr11 gene          diploid yeast strain CY246 (obtained C.L. Peterson, University <br /> was isolated using similar procedure (Galcheva-Gargova et al., 1998).      Massachusetts Medical School, Worcester, MA). Southern blots the <br /> Subsequently, open reading frame encoding S. cerevisiae ZPR1             Leu1 transformants identified colonies heterozygous cZPR1 dis- <br /> (YGR211w) identified S. cerevisiae genome (GenBank/EMBL/          ruption. The heterozygote MY2 sporulated tetrads dis- <br /> DDBJ accession number Z72996). The nucleotide sequence cloned         sected. The spores showed 2:2 segregation viability via- <br /> ZPR1 gene YGR211w identical. Standard recombinant DNA               ble spores Leu1, indicating disruption cZPR1 gene <br /> techniques used construct fusion proteins. The coding region S.   lethal. To isolate haploid yeast cZPR1 disrupted gene (zpr1:: <br /> cerevisiae ZPR1 (cZPR1) amplified PCR cZPR1 genomic          LEU2), yeast strain MY2 (diploid) transformed plasmid <br /> clone (pLG1) template subcloned plasmid pBS            pMM13 (ZPR1 control GAL promoter URA1 plasmid), <br /> (Stratagene) XhoI fragment SalI site (pLG2). This cZPR1         sporulated, tetrads dissected. Leu1 Ura1 haploid cells <br /> fragment cloned XhoI (blunt-ended)-EcoRI fragment (from            (zpr1::LEU2) complemented wild-type cZPR1 plasmid pMM13 <br /> pLG2) pGEX-3X (Pharmacia Biotech, Piscataway, NJ) SmaI/            selected designated strain MY7. Mutant cZPR1 genes were <br /> EcoRI sites. Full-length (wild-type) truncated cZPR1, amplified       introduced strain MY7 plasmid shuffling (Sikorski Boeke, <br /> PCR, cloned pCAL-n (Stratagene) p413ADH (American                1991) using 59-fluoroorotic acid (Toronto Research Chemicals, North <br /> Type Culture Collection, Rockville, MD) expression <span id='am-139' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-140' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-141' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> SmaI/      York, Ontario, Canada). Yeast strains wild-type mutant cZPR1 <br /> EcoRI sites. Mouse ZPR1 (mZPR1), full-length truncated fragments         listed Table II. <br /> amplified PCR, cloned pCAL-n EcoRI/XhoI sites. <br /> S. pombe ZPR1 (pZPR1) PCR amplified cloned pGEX-3X             Mammalian Cell Culture <br /> the BamHI/EcoRI sites. Mutant green fluorescent protein (GFP) (S65T, <br /> V163A; Anderson et al., 1996) PCR amplified cloned p413           COS-7 A431 cells maintained Dulbecco&#8217;s modified Eagle&#8217;s <br />  <br />  <br /> Table II. <span id='am-41' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-43'  ></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> Strains <br /> Strain                                                        Genotype                                                                   Source <br />  <br /> L40        MATa his3D200 trp1-901 leu2-3,112 ade2 LYS2:: (lexAop)4-His URA3::(lexAop)8-LacZ                                 Vojtek et al., 1993 <br /> CY246      MATa/MATa lys2-801/lys2-801 ade2-101/ade2-101 leu2-D1/leu2-D1 his3-D200/his3-D200 ura3-D99/ura3-D99              C.L. Peterson, <br />                                                                                                                             University Massachusetts <br />                                                                                                                             Medical School, <br />                                                                                                                             Worcester, MA <br /> MY2        MATa/MATa lys2-801/lys2-801 ade2-101/ade2-101 leu2-D1/leu2-D1/his3-D200/his3-D200 ura3-D99/ura3-D99              This study <br />             zpr1::LEU2/ZPR1 <br /> MY7        MATa lys2-801 ade2-101 leu2-D1 his3-D200 ura3-D99 zpr1::LEU2 pMM13                                               This study <br /> MY28       MATa lys2-801 ade2-101 leu2-D1 his3-D200 ura3-D99 zpr1::LEU2 pMM49                                               This study <br /> LY1        MATa lys2-801 ade2-101 leu2-D1 his3-D200 ura3-D99 zpr1::LEU2 pLC1                                                This study <br /> LY2        MATa lys2-801 ade2-101 leu2-D1 his3-D200 ura3-D99 zpr1::LEU2 pLCD1                                               This study <br /> LY3        MATa lys2-801 ade2-101 leu2-D1 his3-D200 ura3-D99 zpr1::LEU2 pLCD2                                               This study <br /> LY4        MATa lys2-801 ade2-101 leu2-D1 his3-D200 ura3-D99 zpr1::LEU2 pLCD3                                               This study <br /> LY5        MATa lys2-801 ade2-101 leu2-D1 his3-D200 ura3-D99 zpr1::LEU2 pLCD4                                               This study <br /> LY6        MATa lys2-801 ade2-101 leu2-D1 his3-D200 ura3-D99 zpr1::LEU2 pLCD5                                               This study <br /> LY7        MATa lys2-801 ade2-101 leu2-D1 his3-D200 ura3-D99 zpr1::LEU2 pLCD3, pMM38                                        This study <br /> LY8        MATa lys2-801 ade2-101 leu2-D1 his3-D200 ura3-D99 zpr1::LEU2 pLCD5, pMM38                                        This study <br /> LY9        MATa lys2-801 ade2-101 leu2-D1 his3-D200 ura3-D99 zpr1::LEU2 pLCCT                                               This study <br /> LY10       MATa lys2-801 ade2-101 leu2-D1 his3-D200 ura3-D99 zpr1::LEU2 pLCCT, pMM38                                        This study <br /> LY11       MATa lys2-801 ade2-101 leu2-D1 his3-D200 ura3-D99 zpr1::LEU2 pLCG1                                               This study <br /> LY12       MATa lys2-801 ade2-101 leu2-D1 his3-D200 ura3-D99 zpr1::LEU2 pMM13, pLCG2                                        This study <br /> LY13       MATa lys2-801 ade2-101 leu2-D1 his3-D200 ura3-D99 zpr1::LEU2 pLCCT2, pLCNT2                                      This study <br /> LY14       MATa lys2-801 ade2-101 leu2-D1 his3-D200 ura3-D99 zpr1::LEU2 pLCG3, pMM38                                        This study <br /> MY17       MATa lys2-801 ade2-101 leu2-D1 his3-D200 ura3-D99 zpr1::LEU2 pMM36                                               This study <br />  <br />  <br />  <br />  <br /> Gangwani et al. Interaction ZPR1 eEF-1a                               1473 <br />  medium, 2 mM glutamine, 100 U/ml penicillin 100 U/ml streptomycin          Upstate Biotechnology. The cells washed PBS-T (0.2% <br /> supplemented 10% heat-inactivated fetal bovine serum (COS-7 cells)        Tween-20) times incubated conjugated secondary anti- <br /> or 5% calf serum (A431 cells) (Life Technologies, Gaithersburg, MD).           bodies (goat cyanine [Cy3]-labeled anti-rabbit IgG goat fluorescein <br /> The cells grown 378C humidified atmosphere 5% CO2.           [FITC]-labeled anti-mouse IgG; Jackson ImmunoResearch Laboratories, <br /> Transfection plasmid pCDNA3-Flag-mZPR1 (Galcheva-Gar-                 West Grove, PA) 1 h washed times PBS-T (0.2% <br /> gova et al., 1996) performed using Lipofectamine (Life Technologies)       Tween-20). Coverslips mounted slides using Vectashield (Vector <br /> according manufacturer&#8217;s instructions. A431 cells grown         Laboratories, Burlingame, CA) examined double-<span id='am-148' about='rdfs:label' typeof='owl:Thing'>label</span> immuno- <br /> 50&#8211;60% confluence starved serum-free medium 24 h. The cells         fluorescence microscopy using conventional microscope (Carl Zeiss Inc., <br /> were treated 100 nM EGF PMA 5&#8211;60 min 378C.                      Thornwood, NY). <br />                                                                                   Haploid yeast cells disrupted zpr1::LEU2 gene trans- <br /> Binding Assays                                                                 formed plasmids expressing wild-type cZPR1, mutated cZPR1, or <br />                                                                                cZPR1 fusion proteins GFP. Cells grown synthetic media <br /> Binding assays performed using CBP GST fusions [35S]me-          glucose (2%). Log-phase cultures diluted (0.5 OD600) and <br /> thionine-labeled -unlabeled cell lysates. The cell extracts pre-      starved glucose-free medium 12 h 308C. Glucose (2%) added <br /> pared lysis buffer (50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 10 mM            cells incubated 308C 2 h. The cells harvested and <br /> b-mercaptoethanol, 1.0 mM MgCl2, 1.0 mM CaCl2, 1.0 mM imidazole,               stained 496-diamidino-2-phenylindole (DAPI). The GFP DAPI <br /> 0.1% Triton X-100, 1.0 mM PMSF, 1.0 mM benzamidine, 1 mM sodium                fluorescence examined conventional microscopy (Carl Zeiss Inc.). <br /> orthovanadate, 10 mg/ml leupeptin, 10 mg/ml aprotinin, 10 mg/ml bestatin,      Image analysis performed using software NIH Image <br /> 10 mg/ml antipain, 1.0 mg/ml pepstatin A, 1.0 mg/ml chymostatin).          (v 1.61/PPC; Bethesda, MD). <br /> CBP GST fusions (5 mg) immobilized calmodulin affinity <br /> resin (Stratagene) glutathione agarose beads (Pharmacia Biotech), re- <br /> spectively. The beads (20 ml) washed twice 48C incubated with <br />                                                                                Flow Cytometry <br /> cell extracts (300&#8211;500 mg protein) 2 h 500 ml lysis buffer 48C   Samples prepared flow cytometry (Lew et al., 1992). Cells (106) <br /> then washed times. The beads boiled Laemmli sample             pelleted, washed twice PBS, resuspended 100 ml PBS, and <br /> buffer 3 min, centrifuged, supernatant examined        added dropwise 70% ethanol (1.5 ml) vortexing. The cells <br /> SDS-PAGE.                                                                      fixed overnight 70% ethanol 48C, washed PBS, then <br />                                                                                digested 1 mg/ml RNase 50 mM Tris-HCl, pH 8.0, 1 h 378C <br /> Mass Spectroscopy                                                              followed 5 mg/ml pepsin 55 mM HCl 30 min 378C. The cells <br />                                                                                washed PBS, stained propidium iodide, analyzed. <br /> A 50-kD ZPR1 binding protein purified human A431 epider- <br /> moid carcinoma cells S. cerevisiae strain L40 affinity chromatogra- <br /> phy using immobilized CBP-ZPR1 (residues 1&#8211;268). Bound proteins           Results <br /> separated SDS-PAGE electrotransferred polyvinylene di- <br /> fluoride (PVDF) membrane (Applied Biosystems). The membrane was <br /> stained Ponceau S 50-kD protein excised, rinsed with <br />                                                                                The Zinc Finger Protein ZPR1 Is Conserved in <br /> water, air dried. The protein digested l-tosylamido-         Mammals Yeast <br /> 2-phenyl ethyl chloromethyl ketone-treated trypsin subjected mass <br /> spectroscopic analysis using matrix-assisted <span id='am-154' about='obo:NCBITaxon_62990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_241799 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_241789 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-155' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-157' property="rdfs:label" content="Laser" datatype="xsd:string"></span><span id='am-158'  ></span>laser</span> desorption ioniza- <br />                                                                                Homologues mammalian zinc finger protein ZPR1 <br /> tion time flight (MALDI-TOF) technique. Masses poly-            identified cloned yeast S. cerevisiae and <br /> peptides obtained compared MS-Fit data base (Ludwig              <span id='am-65' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-66' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-69'  ></span>Schizosaccharomyces</span> pombe. The sequence mouse, <br /> Institute <span id='am-79' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-80'  ></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Cancer</span> Research University College London mass spec-         S. cerevisiae S. pombe ZPR1 proteins deduced <br /> troscopy facility, London, UK).                                                nucleotide sequence DNA clones (Fig. 1 A). <br />                                                                                Comparison ZPR1 proteins indicates conserved <br /> Immunoprecipitation Immunoblotting                                         structure including similar zinc fingers (ZnF1 and <br /> Cell extracts prepared using Triton lysis buffer (20 mM Tris-HCl, pH 7.4,      ZnF2) similar regions zinc finger (Fig. 1 B, <br /> 137 mM NaCl, 2 mM EDTA, 25 mM b-glycerophosphate, 2 mM sodium                  A B). The mouse ZPR1 (mZPR1) protein 46% <br /> pyrophosphate, 1% Triton X-100, 10% glycerol, 1 mM PMSF, 1 mM so- <br /> dium orthovanadate, 10 mg/ml leupeptin, 10 mg/ml aprotinin) were <br />                                                                                identical 72% similar S. cerevisiae ZPR1 (cZPR1) <br /> used immunoprecipitation studies. Antibodies mouse ZPR1 (Gal-           43% identical 70% similar S. pombe ZPR1 <br /> cheva-Gargova et al., 1996), eEF-1a (Upstate Biotechnology, Lake Placid,       (pZPR1). The yeast proteins 55% identical and <br /> NY) Flag (M2; Eastman-Kodak, Rochester, NY) prebound               78% similar. The high level homology yeast <br /> 20 ml protein A&#8211;Sepharose (Pharmacia Biotech). The beads incu- <br />                                                                                mammalian ZPR1 suggests overall structure <br /> bated cell lysate 2 h, washed times, boiled with <br /> Laemmli sample buffer gel electrophoresis. Bound proteins sepa-         protein conserved lower higher eu- <br /> rated SDS-PAGE (10%) electrotransferred PVDF membranes              karyotes. <br /> (0.45 mm; Millipore, Waters Chromatography, Milford, MA). Antibody <br /> incubations carried Tris-buffered saline 0.5% Tween-20        ZPR1 Is Essential Gene S. cerevisiae <br /> and 20% horse serum. The rabbit anti-ZPR1 mouse anti&#8211;eEF-1a an- <br /> tibodies diluted 1:400 1:1,000, respectively. HRP-conjugated          We examined gene encoding cZPR1 es- <br /> donkey anti&#8211;rabbit IgG sheep anti&#8211;mouse IgG (1:5,000) secondary            sential cell viability S. cerevisiae. A DNA fragment <br /> antibodies detected enhanced chemiluminescence (Amersham <br /> International, Little Chalfont, UK). <br />                                                                                containing cZPR1 gene disrupted LEU2 gene <br />                                                                                (Fig. 2 A) transformed diploid yeast strain <br /> Fluorescence Microscopy                                                        CY246. Colonies selected plates lacking leucine <br />                                                                                disruption genomic copy cZPR1 was <br /> A431 cells cultured glass coverslips coated poly-l-lysine. The <br />                                                                                confirmed Southern blot analysis (Fig. 2 B). The dip- <br /> cells incubated serum-free medium 24 h, starved, then <br /> treated 100 nM EGF PMA 378C 15 min. Coverslips             loid yeast strain MY2, heterozygous wild-type <br /> rinsed phosphate-buffered saline (PBS) briefly chilled      cZPR1 (ZPR1/zpr1::LEU2), sporulated tet- <br /> methanol. The cells fixed 2208C methanol 5 min,           rads dissected. In tetrads analyzed, viable <br /> then acetone 2 min. The coverslips blocked 10% horse        progeny segregated 2:2, surviving spores <br /> serum PBS 0.5% Tween-20 (PBS-T) 30 min 258C Coplin <br /> jars incubated primary antibodies 1 h 258C. The rabbit <br />                                                                                Leu1 (disrupted cZPR1). These data indicate <br /> polyclonal antibody ZPR1 described (Galcheva-Gargova et            cZPR1 gene essential viability (Fig. 2 C). To <br /> al., 1996). The mouse monoclonal antibody eEF-1a purchased              establish loss viability disruption <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 143, 1998                                  1474 <br />                                                                      Figure 2. ZPR1 essential gene yeast S. cerevisiae. (A) <br />                                                                     The S. cerevisiae diploid WT strain (CY246) used create <br />                                                                     heterozygous strain MY2 (ZPR1/zpr1::LEU2). The struc- <br />                                                                     tures wild-type disrupted cZPR1 genes illustrated. <br />                                                                     (B) Disruption ZPR1 allele confirmed Southern <br />                                                                     blot analysis PvuI restricted genomic DNA isolated the <br />                                                                     diploid WT strain CY246 heterozygous strain MY2 <br />                                                                     (ZPR1/zpr1::LEU2). (C) The heterozygous diploid strain MY2 <br />                                                                     (ZPR1/zpr1::LEU2) sporulated randomly selected <br />                                                                     asci dissected examined viability YEPD agar <br />                                                                     media. <br />  <br />  <br /> Figure 1. Identification ZPR1 homologues mammals and <br /> yeast. (A) The sequence S. cerevisiae, S. pombe, mouse <br /> ZPR1 proteins deduced nucleotide sequence DNA           mouse, S. cerevisiae S. pombe ZPR1 proteins (Fig. 3 <br /> clones presented. Residues identical sequence mouse    A). Similar results obtained binding assays using <br /> ZPR1 indicated period (.). Gaps introduced     [35S]methionine extracts prepared COS-7, CHO, and <br /> the sequences optimize alignment (-). Overline, zinc fin-    S. cerevisiae (strain L40) cells (data shown). These <br /> gers; asterisks, Cys residues. The sequences mouse ZPR1, S. <br />                                                                     data indicate yeast mammalian ZPR1 pro- <br /> pombe zpr1 S. cerevisiae ZPR1 deposited with <br /> GenBank/EMBL/DDBJ accession numbers U41287, AF019768                teins similar biochemical properties, including <br /> and AF019769, respectively. (B) Schematic representation     interaction 50-kD protein. <br /> predicted domain structure ZPR1. The regions required in-       Deletion analysis mouse ZPR1 using [35S]methionine <br /> teraction EGF receptor indicated (EGF-R).              cell extracts demonstrated NH2-terminal mZPR1 <br />                                                                     fragment (ZnF1-A; residues 1&#8211;268) sufficient bind- <br />                                                                     ing 50-kD protein (data shown). This fragment of <br />                                                                     mZPR1 used isolate 50-kD protein human <br /> of cZPR1 gene, complemented haploid yeast               (A431) S. cerevisiae (L40) cell extracts affinity <br /> the disrupted cZPR1 gene (zpr1::LEU2) using plasmid               chromatography. The bound proteins resolved by <br /> expressing wild-type cZPR1. These data demonstrated                 SDS-PAGE electrotransferred PVDF mem- <br /> that cZPR1 gene essential viability S. cerevi-       brane. The 50-kD protein digested l-tosylamido- <br /> siae.                                                               2-phenyl ethyl chloromethyl ketone-treated trypsin and <br />                                                                     examined mass spectroscopy using MALDI-TOF <br /> Identification eEF-1a ZPR1-binding Protein                  technique. A comparison masses polypep- <br /> To study interaction ZPR1 proteins,            tides obtained MS-Fit data base led identi- <br /> performed vitro binding assays using [35S]methionine             fication 50-kD protein eEF-1a (Table III). <br /> human A431 epidermoid carcinoma cell extracts im-                  To confirm identification 50-kD protein as <br /> mobilized GST&#8211;ZPR1 fusion proteins. As expected,                eEF-1a, examined proteins bound mZPR1 <br /> mouse ZPR1 protein bound 180-kD EGF receptor                 Western blot analysis using antibody eEF-1a. Im- <br /> (Fig. 3 A). In addition, [35S]methionine pro-         mobilized recombinant mZPR1 bind eEF-1a <br /> teins observed bind ZPR1. In particular, 50-kD            (Fig. 3 B, lane 8). Similarly, immobilized recombinant <br /> protein prominently detected binding assays using            eEF-1a bound mZPR1 (Fig. 3 C). Deletion analysis <br />  <br />  <br />  <br />  <br /> Gangwani et al. Interaction ZPR1 eEF-1a                      1475 <br />                                                                                                         Figure 3. ZPR1 binds eEF- <br />                                                                                                        1a vitro. (A) A431 cells <br />                                                                                                        labeled [35S]me- <br />                                                                                                        thionine. Soluble extracts <br />                                                                                                        prepared incubated <br />                                                                                                           immobilized      ZPR1 <br />                                                                                                        (GST&#8211;ZPR1) proteins from <br />                                                                                                        <span id='am-113' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-114' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>: S. cerevisiae, S. <br />                                                                                                        pombe, mouse. Bound <br />                                                                                                        proteins examined by <br />                                                                                                        SDS-PAGE autoradiog- <br />                                                                                                        raphy. Arrows, binding the <br />                                                                                                        180-kD EGF receptor to <br />                                                                                                        mouse ZPR1 promi- <br />                                                                                                        nent 50-kD protein binds <br />                                                                                                        mouse yeast ZPR1 <br />                                                                                                        proteins. (B) Deletion analy- <br />                                                                                                        sis mZPR1 sequences re- <br />                                                                                                        quired interaction with <br />                                                                                                        eEF-1a. Immobilized GST <br />                                                                                                        GST&#8211;mZPR1 fusion pro- <br />                                                                                                        teins incubated cell <br />                                                                                                        lysates. Bound eEF-1a was <br /> detected protein immunoblot analysis using monoclonal antibody eEF-1a. The binding eEF-1a full-length (lane 8) and <br /> truncated (lanes 1&#8211;7) mZPR1 proteins presented. The mZPR1 amino acid residues truncated constructs are: 1&#8211;99 (1); 94&#8211;268 <br /> (2); 263&#8211;306 (3); 302&#8211;469 (4); 1&#8211;268 (5); 263&#8211;469 (6); 1&#8211;306 (7); 1&#8211;469 (8). (C) Recombinant eEF-1a binds mZPR1 vitro. Im- <br /> mobilized GST GST&#8211;eEF-1a incubated cell lysates. Bound mZPR1 detected protein immunoblot analysis using a <br /> rabbit polyclonal antibody mZPR1. The multiple bands characteristic electrophoretic mobility mZPR1 (Galcheva-Gar- <br /> gova et al., 1996). (D) Binding recombinant eEF-1a cZPR1. Immobilized GST GST&#8211;eEF-1a incubated purified <br /> CBP&#8211;cZPR1. Bound cZPR1 detected protein immunoblot analysis. <br />  <br />  <br /> demonstrated NH2-terminal region mZPR1                                    (data shown). To test ZPR1 eEF-1a bind <br /> was sufficient binding eEF-1a (Fig. 3 B). Deletion                                directly, performed vitro assays using puri- <br /> analysis eEF-1a demonstrated NH2-terminal                                 fied recombinant proteins. This analysis demonstrated <br /> region (residues 1&#8211;300) required binding ZPR1                                 eEF-1a directly binds cZPR1 (Fig. 3 D). <br />  <br /> Table III. Identification 50-kD ZPR1-binding Protein Mass Spectroscopy <br /> Mass submitted                            Mass matched                                         Peptide sequences                                       Protein matched <br />  <br /> <span id='am-58' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-59' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62'  ></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae <br />   1026.0000                                1026.2286                      (K) IGGIGTVPVGR (V) <br />   1494.0000                                1492.7675                      (K) YAWVLDKLKAER (E) <br />   1563.0000                                1561.7876                      (K) SHINVVVIGHVDSGK (S) <br />   1861.0000                                1860.1303                      (K) YQVTVIDAPGHRDFIK (N) <br />   2326.0000                                2326.7513                      (K) FVPSKPMCVEAFSEYPPLGR (F)                                                    eEF-1a <br />   2365.0000                                2364.7137                      (K) TLLEAIDAIEQPSRPTDKKPLR (L)                                               (50032.9 kD) <br />   2432.0000                                2431.9076                      (R) VETGVIKPGMVVTFAPAGVTTEVK (S) <br />   2552.0000                                2551.8068                      (K) SVEMHHEQLEQGVPGDNVGFNVK (N) <br />   2911.0000                                2910.2887                      (K) TVPFVPISGWNGDNMIEATTNAPWYK (G) <br />   3320.0000                                3321.8552                      (K) TLLEAIDAIEQPSRPTDKPLRLPLQDVYK (I) <br />  <br /> Human <br />    815.6000                                 816.9392                      (K) EKIDRR (S) <br />    959.0000                                 960.1287                      (K) IDRRSGKK (L) <br />    959.0000                                 960.1256                      (K) SAQKAQKAK (&#8211;) <br />   1026.0000                                1026.2286                      (K) IGGIGTVPVGR (V) <br />   1121.0000                                1121.2830                      (K) STTTGHLIYK (C)                                                              eEF-1a <br />   1315.5000                                1314.5320                      (K) GSFKYAWVLDK (L)                                                          (50127.1 kD) <br />   1315.5000                                1315.5608                      (R) EHALLAYTLGVK (Q) <br />   1405.8000                                1405.6019                      (K) YYVTIIDAPGHR (D) <br />   1405.8000                                1405.6461                      (K) FAELKEKIDRR (S) <br />   1590.3000                                1589.8416                      (K) THINIVVIGHVDSGK (S) <br />   2517.6001                                2517.0136                      (R) VETGVLKPGMVVTFAPVNVTTEVK (S) <br />   3011.8000                                3011.4769                      (R) KSGDAIVDMVPGKPMCVESPSDYPPLGR (F) <br /> The 50-kD protein purified affinity chromatography SDS-PAGE. The protein electroblotted PVDF membrane, digested L-tosylamido-2-phenyl ethyl <br /> chloromethyl ketone-treated trypsin, subjected mass spectroscopy using MALDI-TOF technique. Peptide masses obtained compared MS-Fit data- <br /> base. <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 143, 1998                                         1476 <br />  Interaction eEF-1a ZPR1 In Vivo                              Fig. 4 A shows eEF-1a coimmunoprecipitated <br />                                                                     ZPR1. Control experiments using cells did not <br /> To test eEF-1a ZPR1 interact vivo, <br />                                                                     express epitope-tagged mZPR1 demonstrated coimmu- <br /> we performed coimmunoprecipitation analysis. Epitope- <br />                                                                     noprecipitation eEF-1a. These data indicate ZPR1 <br /> tagged mZPR1 immunoprecipitated serum- <br />                                                                     eEF-1a interact vivo. <br /> treated COS-7 cells presence eEF-1a im- <br />                                                                        ZPR1 shown bind EGF receptor qui- <br /> munoprecipitates examined Western blot analysis. <br />                                                                     escent cells released receptor upon <br />                                                                     treatment EGF (Galcheva-Gargova et al. 1996). EGF <br />                                                                     regulate interaction ZPR1 eEF- <br />                                                                     1a. To examine effect EGF, starved A431 cells <br />                                                                     serum-free medium 24 h treated cells <br />                                                                     100 nM EGF. eEF-1a immunoprecipitated and <br />                                                                     presence ZPR1 detected Western blot analy- <br />                                                                     sis. Coimmunoprecipitation ZPR1 eEF-1a was <br />                                                                     detected EGF-treated cells, serum- <br />                                                                     starved cells (Fig. 4 B). The coimmunoprecipi- <br />                                                                     tated ZPR1 sustained longer periods treat- <br />                                                                     ment EGF (Fig. 4 C). These data indicate EGF <br />                                                                     induces interaction ZPR1 eEF-1a. To test <br />                                                                     interaction ZPR1 eEF-1a can <br />                                                                     induced stimuli, studied effect serum. <br />                                                                     Treatment serum-starved cells fetal calf serum <br />                                                                     (10%) resulted coimmunoprecipitation ZPR1 <br />                                                                     eEF-1a (Fig. 4, B C). Based observa- <br />                                                                     tions, propose growth signals mitogen-activated <br />                                                                     cells induce formation eEF-1a/ZPR1 com- <br />                                                                     plexes vivo. Specificity formation eEF-1a/ <br />                                                                     ZPR1 complex established observation that <br />                                                                     treatment cells tumor promoter phorbol <br />                                                                     12-myristate 13-acetate (PMA) did cause coimmuno- <br />                                                                     precipitation ZPR1 eEF-1a (Fig. 4 B). <br />  <br />                                                                     Redistribution ZPR1 eEF-1a Nucleus of <br />                                                                     Mitogen-activated Mammalian Cells <br />                                                                     We examined subcellular localization ZPR1 and <br />                                                                     eEF-1a mammalian cells double-label immunofluo- <br />                                                                     rescence microscopy. Consistent previous studies <br /> Figure 4. Coimmunoprecipitation analysis demonstrates          (Galcheva-Gargova et al., 1996) observed ZPR1 <br /> eEF-1a ZPR1 interact vivo. (A) COS-7 cells mock-        (red) redistributed cytoplasm nucleus upon <br /> transfected (CONTROL) transfected vector expressing       EGF stimulation serum-starved cells (Fig. 5). eEF-1a is <br /> epitope-tagged ZPR1. Flag-ZPR1 immunoprecipitated          abundant protein present mainly cyto- <br /> the M2 monoclonal antibody. eEF-1a immunoprecipitates        plasm cells (Edmonds et al., 1996). Fig. 5 shows the <br /> (IP) detected protein immunoblot analysis mono-       effect EGF treatment subcellular distribution <br /> clonal antibody eEF-1a. (B) Coimmunoprecipitation ZPR1        eEF-1a (green). In serum-starved cells, eEF-1a was <br /> with eEF-1a cells treated serum EGF. A431 epider-      present cytoplasm. Upon treatment EGF, a <br /> moid carcinoma cells incubated serum-free medium for <br />                                                                     fraction eEF-1a molecules observed redis- <br /> 24 h. The cells untreated treated 5% calf serum, 100 <br /> nM EGF, 100 nM PMA 15 min. Extracts prepared            tribute cytoplasm nucleus. The distribution <br /> from cells eEF-1a isolated immunoprecipitation.      eEF-1a ZPR1 nucleus treatment cells <br /> Specificity immunoprecipitation examined using mock      EGF similar, represented yellow fluores- <br /> immunoprecipitates prepared using protein A&#8211;Sepharose beads         cence merged images ZPR1 eEF-1a (Fig. 5). <br /> without antibody eEF-1a (CONTROL). ZPR1 immuno-           Similar results obtained treatment serum- <br /> precipitates detected protein immunoblot analysis using    starved cells 10% calf serum (data shown). Quan- <br /> rabbit polyclonal antibody. (C) EGF causes rapid formation      titative analysis fluorescence images demonstrated <br /> of ZPR1/eEF-1a complexes detected coimmunopre-          ZPR1 z98% cytoplasmic serum-starved cells <br /> cipitation analysis. A431 cells serum starved 24 h. The    z96% nuclear EGF-treated cells. Similar anal- <br /> cells incubated 100 nM EGF various times, extracts <br />                                                                     ysis demonstrated eEF-1a detected nu- <br /> were prepared, eEF-1a isolated immunopre- <br /> cipitation. Specificity immunoprecipitation examined     cleus serum-starved cells z5% total <br /> using mock immunoprecipitates prepared using protein A&#8211;Seph-        eEF-1a located nucleus stimulation with <br /> arose beads antibody eEF-1a (CONTROL). The pres-         EGF. These data indicate ZPR1 eEF-1a are <br /> ence ZPR1 immunoprecipitates examined pro-         located nucleus mitogen-treated cells. The punc- <br /> tein immunoblot analysis rabbit polyclonal antibody.         tate distribution ZPR1 nucleus previously <br />  <br />  <br />  <br />  <br /> Gangwani et al. Interaction ZPR1 eEF-1a                      1477 <br />                                                                                              Figure 5. Redistribution of <br />                                                                                             eEF-1a ZPR1 the <br />                                                                                             cytoplasm nucleus in <br />                                                                                             EGF-treated cells. A431 <br />                                                                                             cells incubated se- <br />                                                                                             rum-free medium 24 h <br />                                                                                             (starved) treated <br />                                                                                             100 nM EGF 100 nM <br />                                                                                             PMA 15 min 378C. The <br />                                                                                             cells fixed, permeabi- <br />                                                                                             lized, examined by <br />                                                                                             double-label immunofluores- <br />                                                                                             cence microscopy. Fluores- <br />                                                                                             cein (green) cyanine <br />                                                                                             (red) represent eEF-1a and <br />                                                                                             ZPR1, respectively. Yellow, <br />                                                                                             colocalization eEF-1a and <br />                                                                                             ZPR1 nucleus upon <br />                                                                                             EGF treatment merged <br />                                                                                             images. Bar, 50 mm. <br />  <br />  <br />  <br /> demonstrated correspond accumulation         (Venema et al., 1991). However, PMA did cause punc- <br /> nucleoli (Galcheva-Gargova et al., 1996). This nucleolar     tate nuclear accumulation eEF-1a ZPR1 (Fig. 5). <br /> accumulation characterized colocalization ZPR1      These data consistent results coimmuno- <br /> with fibrillarin RNA polymerase I, splic-   precipitation analysis treatment PMA did not <br /> ing factor SC35 p80 coilin (Galcheva-Gargova et al.,      induce formation ZPR1/eEF-1a complexes (Fig. 4 <br /> 1998).                                                       B). We conclude ZPR1 responds selective extracel- <br />   We studied effect PMA subcellular distri-    lular stimuli. <br /> bution eEF-1a ZPR1 confirm specificity of <br /> the interaction eEF-1a ZPR1 response <br /> to extracellular stimulation. PMA known induce the <br />                                                              Nuclear Redistribution ZPR1 Yeast <br /> phosphorylation eEF-1a activating protein kinase C     To study localization cZPR1 yeast S. cerevi- <br />  <br />  <br />  <br />                                                                                             Figure 6. Redistribution of <br />                                                                                             ZPR1 yeast S. cerevi- <br />                                                                                             siae. The haploid yeast strain <br />                                                                                             MY28 (Table III) examined <br />                                                                                             fluorescence microscopy. <br />                                                                                             This yeast strain expresses <br />                                                                                             cZPR1&#8211;GFP absence <br />                                                                                             wild-type cZPR1. The cells <br />                                                                                             grown synthetic me- <br />                                                                                             dium glucose (CON- <br />                                                                                             TROL) starved glu- <br />                                                                                             cose-free medium 12 h <br />                                                                                             (STARVED). Glucose (2%) <br />                                                                                             added starved cells <br />                                                                                             incubated 2 h (GLU- <br />                                                                                             COSE). The GFP (green) <br />                                                                                             DAPI (blue) represents <br />                                                                                             ZPR1 DNA, respec- <br />                                                                                             tively. Bar, 10 mm. <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 143, 1998                1478 <br />  siae, recombinant cZPR1&#8211;GFP fusion gene con- <br /> structed. The cZPR1&#8211;GFP gene complemented loss of <br /> viability yeast disrupted zpr1::LEU2 gene. <br /> These haploid yeast expressing cZPR1&#8211;GFP fusion <br /> protein examined fluorescence microscopy. Con- <br /> trol experiments demonstrated GFP sequestered <br /> in vacuole yeast expressing GFP (data not <br /> shown). Fig. 6 shows cZPR1&#8211;GFP fusion protein <br /> (green) present mainly nucleus S. cerevisiae <br /> (CONTROL). Starvation yeast glucose-free me- <br /> dium caused redistribution cZPR1 nucleus <br /> to cytoplasm (Fig. 6). Readdition glucose the <br /> starved cells induced translocation cZPR1 the <br /> cytoplasm nucleus. The localization redistribu- <br /> tion ZPR1 response nutrients S. cerevisiae ap- <br /> pears similar observed starved mito- <br /> gen-treated mammalian cells. Therefore, possible that <br /> cZPR1 function signaling molecule S. cerevi- <br /> siae responds proliferation signals. <br />  <br /> Identification Mutant ZPR1 Molecules That Fail to <br /> Interact eEF-1a <br /> To examine interaction ZPR1 eEF-1a, per- <br /> formed deletion analysis cZPR1. Studies using NH2-ter- <br /> minal (NT) COOH-terminal (CT) fragments cZPR1 <br /> demonstrated eEF-1a&#8211;binding site located in <br /> the NH2-terminal region cZPR1 (residues 1&#8211;261) (Fig. <br /> 7, A B). We constructed series COOH-terminal <br /> truncated mutants NT fragment cZPR1. Binding <br /> assays indicated cZPR1 residues 202&#8211;261 re- <br /> quired interaction eEF-1a cZPR1 (data not <br /> shown). Subsequently, series in-frame deletion mu- <br /> tants full-length cZPR1 (D1-D4) prepared (Fig. 7 <br /> A). These internal deletion mutants expressed as <br /> CBP fusions bacteria purified. The binding of <br /> eEF-1a cZPR1 proteins detected immuno- <br /> blot analysis. The in-frame deletions D1 (residues 222&#8211; <br /> 241) D2 (residues 222&#8211;261) effect eEF-1a <br /> binding, deletions D3 (residues 202&#8211;241) D4 <br /> (residues 202&#8211;261) eliminated binding eEF-1a to <br /> cZPR1 (Fig. 7 C). These data suggest sequence re- <br /> quired eEF-1a binding present D3 region (res- <br /> idues 202&#8211;241) cZPR1 (Fig. 7 A). To delineate       Figure 7. A small region ZPR1 required interaction with <br /> the cZPR1 sequence required binding eEF-1a,         eEF-1a yeast S. cerevisiae. (A) Schematic representation of <br /> the smaller in-frame deletion mutant D5 (residues 202&#8211;         S. cerevisiae ZPR1. NH2-terminal (NT; residues 1&#8211;261) and <br /> 221) prepared tested eEF-1a binding. Immuno-       COOH-terminal (CT; residues 262&#8211;486) fragments cZPR1 are <br /> blot analysis demonstrated cZPR1 deletion             illustrated. In-frame deletions full-length cZPR1 pro- <br />                                                                tein constructed: D1 (residues 222&#8211;241), D2 (residues 222&#8211; <br /> mutant D5 did bind eEF-1a (Fig. 7 D). Therefore, the <br />                                                                261), D3 (residues 202&#8211;241), D4 (residues 202&#8211;261), D5 (res- <br /> 20-amino acid region D5 (residues 202&#8211;221) required        idues 202&#8211;221). The sequence D5 region mouse and <br /> to present full-length truncated ZPR1 eEF-       yeast ZPR1 proteins illustrated. (B&#8211;D) Recombinant cZPR1 <br /> 1a binding. This region located A-domain      proteins expressed CBP fusion protein bacteria. The <br /> the cZPR1 protein conserved mouse, S. cerevisiae     CBP fusion proteins purified, immobilized, used <br /> and S. pombe (Fig. 7 A).                                       binding assays using cell lysates. Bound eEF-1a detected <br />                                                                protein immunoblot analysis. The binding eEF-1a NT <br /> Mutational Analysis ZPR1 Function                           CT fragments cZPR1 examined (B). The effect in- <br />                                                                frame deletions (D1, D2, D3, D4) cZPR1 investigated <br /> We examined effect cZPR1 mutations disrupt         (C). The effect D5 in-frame deletion mutation cZPR1 <br /> eEF-1a binding viability S. cerevisiae. The in-      examined (D). <br /> frame deletion mutants D1, D2, D3, D4, D5 comple- <br /> mented loss viability caused disruption the <br /> cZPR1 gene S. cerevisiae (Table IV). These data indi- <br /> cate deletion sequences required eEF-1a bind- <br />  <br />  <br />  <br />  <br /> Gangwani et al. Interaction ZPR1 eEF-1a                 1479 <br />  Table IV. Mutational Analysis Effect ZPR1 Yeast                          periments demonstrated cZPR1 mutants D1 and <br /> Viability Interaction ZPR1 eEF-1a                                     D2, bind eEF-1a, exhibited growth defect (Fig. 8 <br /> ZPR1                                   Complementation             eEF-1a binding     A). In contrast, yeast strains (Table II) expressing mu- <br />                                                                                       tant cZPR1 (D3, D5, CT) proteins bind <br /> Empty plasmid p413 ADH                         2                          2           eEF-1a (Table IV) grow 20-fold <br /> WT                                             1                          1 <br />                                                                                       slower yeast expressing wild-type cZPR1 (Fig. 8, A <br /> D1                                             1                          1 <br /> D2                                             1                          1 <br />                                                                                       B). <br /> D3                                             1                          2              The morphology yeast strains expressing wild- <br /> D3 1 NT                                        1                          1           type mutant cZPR1 proteins examined light <br /> D5                                             1                          2           microscopy. Fig. 8 C shows images yeast expressing <br /> D5 1 NT                                        1                          1           wild-type cZPR1 mutant D5 cZPR1 does bind <br /> NT                                             2                          1           eEF-1a. Cells mutant cZPR1 (D5) larger and <br /> CT                                             1                          2           grow chain-like clusters. The failure cells sepa- <br /> CT 1 NT                                        1                          1           rate larger size suggest slow-growing <br /> CT 1 NT (DD5)                                  1                          2           yeast accumulate G2/M phase cell cycle. <br /> MtZnF1                                         1                          1 <br />                                                                                       These cells examined DNA content by <br /> MtZnF2                                         2                          1 <br />                                                                                       flow cytometry. Fig. 8 C shows wild-type cells are <br /> Viability haploid yeast (zpr1::LEU2) transformed plasmid expression vectors   distributed G1 G2 phases cell cy- <br /> for wild-type mutated ZPR1 proteins scored positive (1) negative (2). <br /> Binding recombinant ZPR1 eEF-1a detected vitro (1, 2). <br />                                                                                       cle, corresponding 1N 2N DNA content, respec- <br />                                                                                       tively (Fig. 8 C). In contrast, majority mutant <br />                                                                                       yeast cells contained 2N DNA (Fig. 8 D), consistent with <br />                                                                                       accumulation G2/M phase cell cycle (Fig. 8 <br /> ing did affect essential role ZPR1 required                            C). These data suggest interaction cZPR1 with <br /> viability. Therefore, binding cZPR1 eEF-1a                              eEF-1a required normal cell cycle progression. <br /> not required viability S. cerevisiae.                                             To examine absence binding eEF-1a is <br />    We performed deletion analysis identify region                           relevant defective growth yeast strains ex- <br /> cZPR1 required yeast viability. Expression                            press mutant cZPR1 molecules (D3, D5, CT), per- <br /> an NT fragment cZPR1 (residues 1&#8211;261), in-                                   formed interallelic complementation assay. The NT <br /> cludes zinc finger region required in-                          fragment cZPR1 binds eEF-1a, sufficient for <br /> teraction eEF-1a, did confer viability (Table IV).                           viability. However, expression NT fragment of <br /> In contrast, expression CT fragment cZPR1 (resi-                              cZPR1 D3, D5, CT mutant ZPR1 yeast strains <br /> dues 262&#8211;486), includes second zinc finger                              restored normal proliferation (Fig. 8, A B). To con- <br /> does interact eEF-1a, able confer viability                           firm restoration normal growth in- <br /> (Table IV). These data demonstrate functions                              teraction ZPR1 eEF-1a, yeast strain expressing <br /> cZPR1 bind eEF-1a confer viability encoded                              CT fragment transformed plasmid express- <br /> within separate regions cZPR1 gene. eEF-1a bind-                               ing NT (DD5) fragment cZPR1 does bind <br /> ing requires NT fragment cZPR1 viability re-                               eEF-1a. Expression NT (DD5) did restore growth <br /> quires CT fragment cZPR1.                                                      yeast complemented CT fragment of <br />    To examine sequences present CT                                 cZPR1 (Fig. 8 B). These data consistent hy- <br /> fragment cZPR1 required viability, ex-                             pothesis interaction cZPR1 eEF-1a was <br /> amined effect mutations cZPR1 zinc fingers.                             required normal growth yeast. The specificity of <br /> The zinc fingers disrupted replacing                               D5 mutation cause reduced growth tested by <br /> Cys residues <span id='am-143' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-144' property="rdfs:label" content="Ala" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-147'  ></span>Ala</span>. Mutation zinc finger 1 (located                          comparison effect mutation zinc fin- <br /> the NT fragment cZPR1) complemented loss via-                               ger cZPR1 (replacement Cys residues with <br /> bility caused disruption ZPR1 gene. In contrast,                            Ala). Mutation zinc finger caused slow growth <br /> mutation second zinc finger (present CT                                 associated accumulation G2/M <br /> region cZPR1) did confer viability assay (Ta-                          phase cell cycle (data shown). Thus, G2/M ac- <br /> ble IV). These data indicate sequences located                            cumulation caused D5 mutation does represent <br /> CT required viability include second zinc                            nonspecific consequence defects cZPR1 function. <br /> finger.                                                                                  To examine effect NT fragment of <br />                                                                                       cZPR1 restore normal growth yeast express <br /> Interaction ZPR1 eEF-1a Is Required                                       D5 mutant cZPR1, studied effect cZPR1 mu- <br /> Normal Growth                                                                         tations subcellular localization cZPR1 S. cere- <br /> The interaction cZPR1 eEF-1a did appear                                visiae. The GFP-labeled NT CT fragments cZPR1 <br /> be essential viability yeast S. cerevisiae (Table                          (NT&#8211;GFP CT&#8211;GFP), like wild-type cZPR1-GFP, were <br /> IV). However, ZPR1 eEF-1a interact response                                 accumulate nucleus (Fig. 8 E). The dis- <br /> extracellular growth stimuli (Fig. 4, B C). Therefore,                         tribution D5 mutant cZPR1-GFP (D5&#8211;GFP) was <br /> was possible interaction play role                             similar wild-type cZPR1&#8211;GFP (Fig. 8 E). Therefore, de- <br /> normal process cellular proliferation. To examine                              letion D5 region did affect nuclear distribu- <br /> function ZPR1/eEF-1a complex cellular growth,                               tion cZPR1 growing cells. These data demonstrate <br /> we investigated effect cZPR1 mutations disrupt                            cZPR1 accumulate nucleus independently <br /> eEF-1a binding growth S. cerevisiae. Control ex-                            eEF-1a. <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 143, 1998                                         1480 <br />  Figure 8. Mutational analysis effect ZPR1 growth yeast S. cerevisiae. (A) The growth haploid yeast strains <br /> (zpr1::LEU2) complemented plasmid expression vectors wild-type (WT) mutated (D1, D2, D3, D5, NT) cZPR1 was <br /> examined. Liquid cultures yeast (0.2 OD600) serially diluted 5-, 10-, 50-, 100- 1,000-fold, spotted YEPD plates, <br /> and incubated 308C. (B) The growth haploid yeast strains (zpr1::LEU2) complemented plasmid expression vectors for <br /> wild-type (WT) mutated (CT, NT, NTDD5) cZPR1 examined described above. (C) Morphology haploid yeast strains <br /> (zpr1::LEU2) complemented plasmid expression vectors WT cZPR1 (strain LY1) mutant D5 cZPR1 (strain LY6) ex- <br /> amined microscopy using Nomarski optics. DNA detected DAPI staining fluorescence microscopy. (D) FACS&#174; analysis <br /> of haploid yeast strains LY1 LY6 expressing wild-type mutant (D5) cZPR1, respectively. (E) Mutational analysis the <br /> sub-cellular localization cZPR1 S. cerevisiae. The location wild-type cZPR1 (cZPR1-GFP), D5 mutant cZPR1 (D5-GFP), the <br /> NT fragment cZPR1 (NT-GFP), CT fragment cZPR1 (CT-GFP) examined fluorescence microscopy yeast strains <br /> MY28, LY11, LY12, LY14, respectively (Table II). Bars: (C) 50 mm; (E) 10 mm. <br />  <br />  <br />  <br />  <br /> Gangwani et al. Interaction ZPR1 eEF-1a                        1481 <br />  Discussion                                                       polymerase I, splicing factor SC35 p80 <br />                                                                  coilin (Galcheva-Gargova et al., 1998). The colocalization <br /> ZPR1 Is Conserved Mammals Yeast                           fibrillarin RNA polymerase I suggests ZPR1 <br />                                                                  accumulates nucleoli. <br /> ZPR1 zinc finger protein initially identified     Although ZPR1 known redistribute the <br /> mammals (Galcheva-Gargova et al., 1996). This protein            cytoplasm nucleus mammalian cells (Galcheva- <br /> contains zinc fingers primary structure: Cys-       Gargova et al., 1996), unclear subcellu- <br /> Xaa2-Cys-Xaa25-Cys-Xaa2-Cys. Atomic absorption spec-             lar localization ZPR1 regulated yeast. Here we <br /> troscopy confirmed presence zinc atoms        demonstrate ZPR1 redistributes cyto- <br /> ZPR1 molecule (Galcheva-Gargova et al., 1996). Here           plasm nucleus yeast S. cerevisiae. In prolifer- <br /> demonstrate ZPR1 protein conserved bud-           ating yeast, ZPR1 protein located preferentially in <br /> ding yeast (S. cerevisiae) fission yeast (S. pombe). A       nucleus. In contrast, nutrient deprivation yeast, like <br /> homologue ZPR1 identified nematode            serum starvation mammalian cells, caused redistri- <br /> <span id='am-91' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-92' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-95'  ></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Caenorhabditis elegans</span> searching GenBank data-            bution ZPR1 nucleus cytoplasm. These <br /> base (Wilson et al., 1994; GenBank/EMBL/DDBJ acces-              data indicate ZPR1 responds proliferative signals <br /> sion number AF022981) using BLAST File Server                mammals yeast. Further studies required <br /> (National Center Biotechnology Information, Be-              determine mechanism nuclear exclusion nu- <br /> thesda, MD). The conserved structure ZPR1 pro-            clear accumulation response altered proliferative <br /> teins diverse species indicates ZPR1         states. <br /> protein similar functions higher lower eu- <br /> karyotes. <br />    To examine function ZPR1 eukaryotic cells, we <br /> studied role yeast S. cerevisiae. Gene disruption     Interaction ZPR1 eEF-1a in <br /> experiments demonstrated ZPR1 essential vi-         Mitogen-treated Cells <br /> ability S. cerevisiae. Deletion analysis genetic com-     The ZPR1 protein bind eEF-1a. Coimmuno- <br /> plementation assays demonstrated COOH-termi-            precipitation analysis demonstrated ZPR1 did not <br /> nal region ZPR1 (including second zinc finger)        interact eEF-1a serum-starved mammalian cells. <br /> required viability. In contrast, NH2-terminal region     In contrast, treatment EGF serum caused rapid in- <br /> of ZPR1 (including zinc finger) dispensable        duction ZPR1 complex formation eEF-1a. Control <br /> for yeast viability.                                             studies demonstrated formation ZPR1/eEF-1a <br />    The requirement COOH-terminal region               complexes selective treatment cells <br /> cZPR1 yeast viability evolutionary signifi-         phorbol ester did cause coimmunoprecipitation <br /> cance. Prokaryotic homologues ZPR1              eEF-1a ZPR1. These data indicate whereas <br /> described. However, prokaryotes contain homo-              ZPR1 interacts cell surface receptors quiescent <br /> logue COOH-terminal region ZPR1. Conceptual            cells (Galcheva-Gargova et al., 1996), ZPR1 binds eEF- <br /> translation genomic sequences bacteria,        1a treatment cells certain extracellular <br /> including <span id='am-116' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-117' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-124'  ></span><span id='am-125' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span>Escherichia coli</span> (Taura et al., 1992; GenBank/         stimuli. <br /> EMBL/DDBJ accession number P25538), <span id='am-15' about='obo:NCBITaxon_1129' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_1301283 obo:NCBITaxon_1118 obo:NCBITaxon_1117'><span id='am-16' property="oboInOwl:hasDbXref" content="PMID:11411708" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Anacystis" datatype="xsd:string"></span><span id='am-18'  ></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Synechococcus" datatype="xsd:string"></span>Synechococcus</span>                   eEF-1a preferentially located cytoplasm cells <br /> sp. (Sugita et al., 1995; GenBank/EMBL/DDBJ accession            (Edmonds et al., 1996; Minella et al., 1996). In contrast, <br /> number P46227), <span id='am-22' about='obo:NCBITaxon_2184' typeof='owl:Thing obo:NCBITaxon_183939 obo:NCBITaxon_2183 obo:NCBITaxon_1 obo:NCBITaxon_2182 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_28890'><span id='am-23'  ></span><span id='am-24' property="oboInOwl:hasDbXref" content="PMID:15653928" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="PMID:10319480" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Methanococcus" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="PMID:8782682" datatype="xsd:string"></span>Methanococcus</span> jannaschii (Bult et al.,           ZPR1 located nucleus mitogen-activated cells <br /> 1996; GenBank/EMBL/DDBJ accession number O57950),                (Galcheva-Gargova et al., 1996). The cytoplasmic location <br /> <span id='am-30' about='obo:NCBITaxon_2234' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_2233 obo:NCBITaxon_2232 obo:NCBITaxon_183980 obo:NCBITaxon_2231 obo:NCBITaxon_28890'><span id='am-31'  ></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="PMID:11540072" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Archaeoglobus fulgidus" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Archaeoglobus fulgidus</span> (Klenk et al., 1997; GenBank/             eEF-1a incompatible evidence nuclear <br /> EMBL/DDBJ accession number AE00150), Methano-                ZPR1 mitogen-activated cells forms complexes with <br /> bacterium thermoautotrophicum (Smith et al., 1997; Gen-          eEF-1a. As eEF-1a abundant cellular protein, we <br /> Bank/EMBL/DDBJ accession number AE000936) indi-                  considered possibility population eEF-1a <br /> cates presence conserved hypothetical protein           molecules form complexes ZPR1 represents only <br /> with zinc finger similar COOH-terminal region         fraction total eEF-1a present cell. To <br /> of ZPR1. Whether hypothetical prokaryotic zinc fin-        test hypothesis, examined nuclear accumula- <br /> ger proteins share functions similar eukary-     tion eEF-1a serum-starved EGF-treated cells. <br /> otic ZPR1 proteins presently unclear, warrants fur-       This analysis demonstrated nuclear eEF-1a not <br /> ther study.                                                      detected starved cells, detected EGF-treated <br />                                                                  cells. Interestingly, nuclear eEF-1a exhibited punc- <br />                                                                  tate immunofluorescence appearance similar to <br />                                                                  ZPR1. These data consistent hypothesis that <br /> Nuclear Redistribution ZPR1 Proliferating Cells            mitogen-treatment cells caused formation nu- <br /> The ZPR1 protein located cytoplasm quiescent        clear complexes ZPR1 eEF-1a. Further evidence <br /> mammalian cells redistributes nucleus           support interpretation obtained ob- <br /> treatment mitogens (Galcheva-Gargova et al., 1996).         servation treatment cells phorbol ester, <br /> Immunofluorescence analysis demonstrated nu-            does cause formation ZPR1/eEF-1a <br /> clear ZPR1 protein exhibited punctate appearance. Dou-         complexes, did cause punctate nuclear localization of <br /> ble-label immunofluorescence microscopy demonstrated             eEF-1a. <br /> that nuclear ZPR1 colocalized fibrillarin RNA              The preferential location eEF-1a cytoplasm is <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 143, 1998                    1482 <br />  established (Edmonds et al., 1996; Minella et al., 1996)                    Nolan, L.A. Herzenberg. 1996. Simultaneous fluorescence-activated cell <br />                                                                                 sorter analysis distinct transcriptional elements single cell us- <br /> the presence nuclear subpopulation eEF-1a                           ing engineered green fluorescence proteins. Proc. Natl. Acad. Sci. USA. 93: <br /> been described (Collig et al., 1994; Janssen et al., 1994;                      8508&#8211;8511. <br /> Barberse et al., 1995; Billaut-Mulot et al., 1996; Sanders et                Barbarese, E., D.E. Koppel, M.P. Deutscher, C.L. Smith, K. Ainger, F. Morgan, <br />                                                                                 J.H. Carson. 1995. Protein translation components colocalized in <br /> al., 1996). These populations eEF-1a play dif-                       granules oligodendrocytes. J. Cell Sci. 108:2781&#8211;2790. <br /> ferent physiological roles. The results study demon-                 Billaut-Mulot, O., R. Fernandez-Gomez, M. Loyens, A. Ouaissi. 1996. Try- <br />                                                                                 panosoma cruzi elongation factor-1a: nuclear localization parasites under- <br /> strate nuclear eEF-1a includes complexes eEF-1a                         going apoptosis. Gene. 174:19&#8211;26. <br /> with ZPR1 proliferating cells. Previous studies eEF-                   Blackwell, J.L., M.A. Brinton. 1997. Translation elongation factor-1 alpha <br /> 1a indicate bind RNA (Blackwell Brinton,                        interacts 39 stem-loop region <span id='am-9' about='obo:NCBITaxon_11082' typeof='owl:Thing obo:NCBITaxon_11051 obo:NCBITaxon_11071 obo:NCBITaxon_1 obo:NCBITaxon_10239 obo:NCBITaxon_11050 obo:NCBITaxon_439488 obo:NCBITaxon_35278'><span id='am-10' property="oboInOwl:hasBroadSynonym" content="WNV" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="West Nile virus" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-13'  ></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>West Nile virus</span> genomic RNA. J. Virol. <br />                                                                                 71:6433&#8211;6444. <br /> 1997) interacts RNA polymerase poly-                        Blumenthal, T., G.G. Carmichael. 1979. RNA replication: function and <br /> merases (Blumenthal Carmichael, 1979; Das et al.,                           structure Qb-replicase. Ann. Rev. Biochem. 48:528&#8211;548. <br /> 1998). One possible role nucleolar complex eEF-                    Bult, C.J., O. White, G.J. Olsen, L. Zhou, R.D. Fleischmann, G.G. Sutton, J.A. <br />                                                                                 Blake, L.M. Fitzgerald, R.A. Clayton, J.D. Gocayne, et al. 1996. Complete <br /> 1a ZPR1 functionally interact, directly in-                       genome sequence methanogenic archaeon Methanococcus jannaschii. <br /> directly, RNA. The eEF-1a GTPase activity                              Science. 273:1058&#8211;1073. <br />                                                                              Christianson, T.W., R.S. Sikorski, M. Dante, J.H. Shero, P. Hieter. 1992. <br /> contribute putative regulatory function                          Multifunctional yeast high-copy number shuttle vectors. Gene. 110:119&#8211;122. <br /> ZPR1/eEF-1a complex.                                                         Colling, D.A., G.O. Wastenays, M. Miyazaki, R.E. Williamson. 1994. Elon- <br />                                                                                 gation factor 1a component subcortical actin bundles characean <br />                                                                                 algae. Cell Biol. Int. 18:1019&#8211;1024. <br /> Interaction ZPR1 eEF-1a Is Required                              Condeelis, J. 1995. Elongation factor 1 alpha, translation cytoskeleton. <br /> Normal Cellular Proliferation                                                   Trends Biochem. Sci. 20:169&#8211;170. <br />                                                                              Das, T., M. Mathur, A.K. Gupta, G.M.C. Janssen, A.K. Banerjee. 1998. <br /> Deletion analysis demonstrated short segment                          RNA polymerase <span id='am-1' about='obo:NCBITaxon_11276' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_11157 obo:NCBITaxon_10239 obo:NCBITaxon_439488 obo:NCBITaxon_11270 obo:NCBITaxon_11271 obo:NCBITaxon_35301 obo:NCBITaxon_327822'><span id='am-2' property="oboInOwl:hasAlternativeId" content="NCBITaxon:12872" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Vesicular stomatitis virus" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5'  ></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="vesicular stomatitis virus, VSV" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="vesicular stomatitis virus VSV" datatype="xsd:string"></span>vesicular stomatitis virus</span> specifically associates with <br />                                                                                 translation elongation factor-1abg activity. Proc. Natl. Acad. Sci. <br /> the NH2-terminal region ZPR1 required in-                        USA. 95:1449&#8211;1454. <br /> teraction ZPR1 eEF-1a. Since COOH-termi-                         Edmonds, B.T., J. Wyckoff, Y.-G. Yeung, Y. Wang, R.E. Stanley, J. Jones, J. <br /> nal region ZPR1 sufficient yeast viability,                       Segall, J. Condeelis. 1996. Elongation factor-1a overexpressed ac- <br />                                                                                 tin binding protein metastatic rat mammary adenocarcinoma. J. Cell Sci. <br /> conclude interaction ZPR1 eEF-1a                        109:2705&#8211;2714. <br /> required viability. This conclusion strongly sup-                    Galcheva-Gargova, Z., K.N. Konstantinov, H.-I. Wu, F.G. Klier, T. Barrett, <br /> ported observation small in-frame deletions,                        R.J. Davis. 1996. Binding zinc finger protein ZPR1 epidermal <br />                                                                                 growth factor receptor. Science. 272:1797&#8211;1802. <br /> which disrupt binding eEF-1a, did affect ability                  Galcheva-Gargova, Z., L. Gangwani, K.N. Konstantinou, M. Mikrut, S.J. The- <br /> of ZPR1 confer viability. However, yeast strains ex-                         roux, T. Enoch, R.J. Davis. 1998. The cytoplasmic zinc finger protein <br />                                                                                 ZPR1 accumulates nucleolus proliferating cells. Mol. Biol. Cell. <br /> pressing ZPR1 proteins did bind eEF-1a                            9:2963&#8211;2971. <br /> found exhibit slow growth, aberrant morphology,                    Janssen, G.M.C., H.T.F. Van Damme, J. Kriek, R. Amons, W. Moller. <br /> accumulate G2/M phase cell cycle. These data                      1994. The subunit structure elongation factor 1 <span id='am-84' about='obo:NCBITaxon_6660' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_38009 obo:NCBITaxon_116556 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_6659 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_88770 obo:NCBITaxon_6658 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-85'  ></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasExactSynonym" content="brine shrimps" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasExactSynonym" content="sea monkeys" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Artemia" datatype="xsd:string"></span>Artemia</span> <span id='am-174' about='obo:NCBITaxon_33337' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_730330 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_79707 obo:NCBITaxon_79744 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33336 obo:NCBITaxon_30001 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960'><span id='am-175' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-176'  ></span><span id='am-177' property="rdfs:label" content="Salina" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>salina</span>; why <br />                                                                                 a-chains complex. J. Biol. Chem. 269:1&#8211;8. <br /> indicate interaction ZPR1 eEF-1a                     Klenk, H.P., R.A. Clayton, J. Tomb, O. White, K.E. Nelson, K.A. Ketchum, <br /> required normal proliferation yeast. We tested                       R.J. Dodson, M. Gwinn, E.K. Hickey, J.D. Peterson, et al. 1997. The com- <br /> this hypothesis performing interallelic complemen-                        plete sequence hyperthermophilic, sulphate-reducing, archaeon Ar- <br />                                                                                 chaeoglobus fulgidus. Nature. 390:364&#8211;370. <br /> tation assay using NH2-terminal fragment ZPR1                          Krieg, P.A., S.M. Varnum, W.M. Wormington, D.A. Melton. 1989. The <br /> which binds eEF-1a, insufficient viability. This                  mRNA encoding elongation factor-1a (eEF-1a) major transcript the <br />                                                                                 midblastula transition Xenopus. Dev. Biol. 133:93&#8211;100. <br /> fragment ZPR1 complemented defective growth                        Lew, D.J., N.J. Marini, S.I. Reed. 1992. Different G1 cyclins control the <br /> yeast strains express ZPR1 proteins bind                       timing cell cycle commitment mother daughter cells budding <br /> eEF-1a. In contrast, deletion sequences required                         yeast S. cerevisiae. Cell. 69:317&#8211;327. <br />                                                                              Minella, O., O. Mulner-Lorillon, V. De Smedt, S. Hourdez, P. Cormier, R. <br /> eEF-1a binding NH2-terminal fragment ZPR1                             Belle. 1996. Major intracellular localization elongation factor-1. Cell Mol. <br /> (NT DD5) blocked interallelic complementation. These                        Biol. 42:805&#8211;810. <br /> data provide strong support hypothesis                      Moldave, K. 1985. Eukaryotic protein synthesis. Ann. Rev. Biochem. 54:1109&#8211; <br />                                                                                 1149. <br /> binding ZPR1 eEF-1a required normal cellular                    Rose, M.D., P. Novick, J.H. Thomos, D. Botstein, G.R. Fink. 1987. Saccha- <br /> proliferation. Further studies required determine                        romyces cerevisiae genomic plasmid bank based centromere containing <br />                                                                                 shuttle vector. Gene. 50:237&#8211;243. <br /> whether ZPR1/eEF-1a complex required prolif-                      Rothstein, R.J. 1983. One-step gene disruption yeast. Methods Enzymol. 101: <br /> eration mammalian cells. In addition, studies re-                        202&#8211;211. <br /> quired establish molecular mechanism action                     Sanders, J., M. Brandsma, G.M.C. Janssen, J. Dijk, W. Moller. 1996. Immu- <br />                                                                                 nofluorescence studies human fibroblasts demonstrate presence the <br /> the ZPR1/eEF-1a complex vivo.                                          <br /> </body></html>