<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Why Do Hubs Yeast Protein Interaction Network <br /> Tend To Be Essential: Reexamining Connection <br /> between Network Topology Essentiality <br /> Elena Zotenko1, Julian Mestre1, Dianne P. O&#8217;Leary2,3, Teresa M. Przytycka4* <br /> 1 Max-Planck Institute Informatics, Saarbruecken, Germany, 2 Department Computer Science, University Maryland, College Park, Maryland, United States of <br /> America, 3 Institute Advanced Computer Studies, University Maryland, College Park, Maryland, United States America, 4 National Center Biotechnology <br /> Information, National Library Medicine, National Institutes Health, Bethesda, Maryland, United States America <br />  <br />  <br />  <br />      Abstract <br />      The centrality-lethality rule, notes high-degree nodes protein interaction network tend correspond to <br />      proteins essential, suggests topological prominence protein protein interaction network a <br />      good predictor biological importance. Even correlation degree essentiality confirmed by <br />      independent studies, reason correlation remains illusive. Several hypotheses putative connections <br />      essentiality hubs topology protein&#8211;protein interaction networks proposed, we <br />      demonstrate, explanations supported properties protein interaction networks. To identify main <br />      topological determinant essentiality provide biological explanation connection network <br />      topology essentiality, performed rigorous analysis variants genomewide protein interaction network <br />      <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> obtained using different techniques. We demonstrated majority hubs essential <br />      involvement Essential Complex Biological Modules, group densely connected proteins shared <br />      biological function enriched essential proteins. Moreover, rejected previously proposed explanations for <br />      centrality-lethality rule, relating essentiality hubs role overall network connectivity another <br />      relying recently published essential protein interactions model. <br />  <br />   Citation: Zotenko E, Mestre J, O&#8217;Leary DP, Przytycka TM (2008) Why Do Hubs Yeast Protein Interaction Network Tend To Be Essential: Reexamining the <br />   Connection Network Topology Essentiality. PLoS Comput Biol 4(8): e1000140. doi:10.1371/journal.pcbi.1000140 <br />   Editor: Burkhard Rost, Columbia University, United States America <br />   Received December 12, 2007; Accepted June 23, 2008; Published August 1, 2008 <br />   <span id='am-35' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> open-access article distributed terms Creative Commons Public Domain declaration stipulates that, placed public <br />   domain, work freely reproduced, distributed, transmitted, modified, built upon, used lawful purpose. <br />   Funding: TMP EZ supported Intramural Research Program NIH National Library Medicine. DPO supported NSF Grant <br />   CCF-0514213. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: przytyck@ncbi.nlm.nih.gov <br />  <br />  <br />  <br /> Introduction                                                                        removal equivalent number random nodes <br />                                                                                     [3,8]. Therefore, assumption organism&#8217;s <br />    An intriguing question analysis biological networks                 function depends connectivity various parts its <br /> whether biological characteristics protein, essentiality,              interactome, hubs predominantly essential they <br /> can explained placement network, i.e.,                     play central role maintaining connectivity. <br /> topological prominence implies biological importance. One the <br />                                                                                        Recently, He colleagues challenged hypothesis of <br /> first connections context protein <br />                                                                                     essentiality function global network structure and <br /> interaction network, so-called centrality-lethality rule, was <br />                                                                                     proposed majority proteins essential their <br /> observed Jeong colleagues [3], demonstrated that <br />                                                                                     involvement essential protein&#8211;protein interactions that <br /> high-degree nodes hubs protein interaction network of <br />                                                                                     distributed uniformly random network edges [9]. <br /> <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-28'  ></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae contain essential proteins would <br /> be expected chance. Since correlation degree                    Under hypothesis, hubs proposed predominantly <br /> and essentiality confirmed studies [4&#8211;7],                    essential involved interactions thus <br /> recently systematic attempt examine reasons                     likely involved essential. <br /> for correlation. In particular, main topological                      In work carefully evaluate proposed <br /> determinant essentiality? Is number immediate                          explanations centrality-lethality rule. Recently several <br /> neighbors other, global topological property                      hypotheses linked structural properties protein interaction <br /> essential proteins protein interaction network?                       networks biological phenomena come scrutiny, with <br />    Jeong colleagues [3] suggested overrepresentation                    main concern observed properties to <br /> essential proteins high-degree nodes attributed                     experimental artifacts and/or biases present networks <br /> the central role hubs play mediating interactions                     lack biological implication. To limit impact of <br /> numerous, connected proteins. Indeed, removal hubs                      biases results reported study use variants of <br /> disrupts connectivity network, measured                        genomewide protein interaction network Saccharomyces cerevisiae <br /> network diameter size largest connected component,                    compiled diverse sources interaction evidence [10&#8211;15]. <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                        1                            August 2008 | Volume 4 | Issue 8 | e1000140 <br />                                                                                                                          Essentiality Hub Proteins <br />  <br />  <br />  <br />   Author Summary                                                             proteins, COmplex BIological Modules (or <br />                                                                              COBIMs). We demonstrate uneven distribution <br />   Analysis protein interaction networks budding                    essential proteins COBIMs majority essential <br />   yeast Saccharomyces cerevisiae revealed small                   proteins lie COBIMs enriched essential <br />   number proteins, so-called hubs, interact                    proteins, Essential COmplex BIological Modules (or <br />   disproportionately large number proteins. Fur-                    ECOBIMs). <br />   thermore, hub proteins shown                             By <span id='am-48' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>, ECOBIMs contain, relative size, <br />   essential survival cell&#8212;that is, optimal                     essential nodes random group proteins same <br />   conditions, yeast grow multiply them. <br />                                                                              size. But fraction essential hubs members such <br />   This relation essentiality number of <br />                                                                              ECOBIMs? How does number relate expected by <br />   neighbors protein&#8211;protein interaction network has <br />   termed centrality-lethality rule. However,                chance? In fact, does enrichment hubs are <br />   hubs essential? Jeong colleagues [1] suggested                    members/nonmembers ECOBIMs essential proteins relate to <br />   overrepresentation essential proteins high-                  enrichment values expected chance suitable <br />   degree nodes attributed central role                    randomization protocol? We propose membership in <br />   hubs play mediating interactions numerous,                   ECOBIMs largely accounts enrichment hubs essential <br />   connected proteins. Another view, proposed He                       proteins. In support hypothesis, fraction <br />   Zhang, suggested that majority proteins                    essential proteins non-ECOBIM hubs is, depending on <br />   essential involvement essential                network, 13&#8211;35%, low network <br />   protein&#8211;protein interactions distributed uniformly                average. Furthermore essentiality nodes not <br />   random network edges [2]. We                   members ECOBIMs weakly correlated degree. <br />   reasons determines essentiality. Instead,                 Finally, using randomization experiment demonstrated that <br />   majority hubs essential involvement                 properties characteristic protein interaction <br />   Essential Complex Biological Modules, group densely                   network unlikely corresponding randomized network. <br />   connected proteins shared biological function that <br />   enriched essential proteins. This study sheds new <br />   light topological complexity protein interaction <br />                                                                              Results <br />   networks.                                                                  Our Study Uses Six Protein Interaction Networks <br />                                                                                 Our source protein interaction <span id='am-49' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52'  ></span><span id='am-53' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> yeast Saccharomyces <br />                                                                              cerevisiae numerous small-scale studies seven high-throughput <br />    To assess essentiality hubs related role       experiments [15,18&#8211;23]. Interactions reported targeted studies are <br /> in maintaining network connectivity performed tests. First,           believed biologically relevant usually subjected a <br /> if case, expect essential hubs             variety validation methods. Recently, Reguly et al. [11] curated <br /> more important maintaining network connectivity                     30,000 literature abstracts compile network protein <br /> nonessential hubs. We case. Next,              interactions reported small-scale experiments. We refer this <br /> addition node degree, consider measures               network LC network (Literature Curated network). <br /> topological prominence, demonstrate                    It suggested centrality-lethality phenomenon an <br /> are better predictors role node plays network               artifact possible <span id='am-1' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Bias" datatype="xsd:string"></span>bias</span> present networks mainly derived <br /> connectivity node degree. Thus, essentiality related         small-scale experiments [24]. Namely, essential proteins are <br /> maintaining network connectivity, expect                      focus studies tend higher <br /> essentiality better correlated centrality measures          degree networks. Therefore, complement LC <br /> than node degree. However, node degree             network, included study networks contain <br /> a better predictor essentiality measure tested.            interactions reported small-scale studies high- <br />    To reject essential protein interaction model [9], used          throughput experiments. The DIP CORE network derived from <br /> hypothesis testing approach. Namely, observed model             pool protein interactions deposited DIP database <br /> implies probability protein essential                  using computational method Deane et al. [10] recruits <br /> independent probability noninteracting protein           evolutionary information filter unreliable interactions. The <br /> is essential. However, tested networks essentiality            HC network (the High Confidence network) recently published by <br /> noninteracting proteins share interaction partners correlat-         Batada et al. [12] derived intersecting small-scale data with <br /> ed. Thus, reject independence assumption and, result,            above-mentioned seven high-throughput datasets. More <br /> the essential protein interaction model high confidence.                specifically, interaction included final network if <br />    Motivated findings propose alternative explana-              independently reported twice. <br /> tion centrality-lethality rule. Our explanation draws              We include networks derived solely high- <br /> growing realization phenotypic effect gene-knockout                  throughput experimental data. The Y2H network obtained from <br /> experiments function group functionally related genes,          genomewide yeast-two-<span id='am-59' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-60' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-63'  ></span>hybrid</span> interaction screen Ito et al. <br /> such genes gene products members                    [15] contains high-confidence interactions exper- <br /> multiprotein complex [16]. It known densely connected           imentally detected times. Recently, Collins et al. [13] <br /> subnetworks enriched proteins share biological                   published statistical scoring scheme maps raw complex <br /> function. Therefore, expect dense subnetworks              purification experimental data interaction confidence scores. <br /> protein interaction networks enriched depleted           The authors applied method raw purification data from <br /> in essential proteins. Indeed, Hart colleagues observed             recent genomewide complex purification experiments [22,23]. <br /> essential proteins distributed evenly set               We refer network contains interactions a <br /> automatically indentified multiprotein complexes [17]. In               confidence score certain threshold TAP-MS network. <br /> work observe phenomenon holds potentially                  Finally, include network interactions predicted silico <br /> larger groups densely connected functionally related                  using computational approach Jansen et al. [14]. The <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                 2                         August 2008 | Volume 4 | Issue 8 | e1000140 <br />                                                                                                                                        Essentiality Hub Proteins <br />  <br />  <br />  <br />  Table 1. Structural properties tested protein                                    connected proteins. Consequently, high-degree <br />  interaction networks.                                                                   nodes referred hubs, observed phenomenon is <br />                                                                                          known centrality-lethality rule. <br />                                                                                             To confirm centrality-lethality rule tested networks we <br />                 Number          Number           Average        Average clustering       used results systematic gene deletion screen [25] which <br />                 nodes        edges         degree         coefficient              1,105 yeast genes essential growth rich <br />  DIP CORE       2,316           5,569            4.81           0.30 <br />                                                                                          glucose media. There numerous ways exposing positive <br />                                                                                          correlation degree essentiality, used in <br />  LC             3,224           11,291           7.00           0.36 <br />                                                                                          paper. First, ask hubs, nodes degree <br />  HC             2,752           9,097            6.61           0.37                     greater equal certain threshold, likely be <br />  TAP-MS         1,994           15,819           15.87          0.60                     essential average network node, i.e., fraction of <br />  BAYESIAN       4,135           20,984           10.15          0.26                     essential proteins hubs greater network average. <br />  Y2H            400             491              2.45           0.09                     To choose appropriate threshold value relied Figure 1A, <br />                                                                                          shows enrichment values nodes degree greater <br />  doi:10.1371/journal.pcbi.1000140.t001                                                   equal k function k. In networks steady <br />                                                                                          increase enrichment values interrupted large values k. <br /> method trains Bayesian network combines variety                              Therefore, chose threshold value approximately 20% <br /> genomic features mRNA coexpression, colocalization, etc.                         network nodes hubs. (For DIP CORE network the <br /> to derive interaction confidence scores protein pairs. The                           value k 7, LC network 10, HC network 10, <br /> authors used protein interactions derived set manually                         TAP-MS network 24, BAYESIAN network is <br /> curated protein complexes set positive training examples                       12, Y2H network 3.) However, repeated the <br /> and pairs proteins localized different cellular compartments                       experiments hubs defined 10% (data shown) found <br /> as set negative training examples. We refer network                       conclusions robust specific choice threshold. <br /> as BAYESIAN network. <br />    Table 1 summarizes structural properties networks <br /> just described. (Here paper analyze the <br /> largest connected component protein interaction network.) <br /> Table 2 shows overlap, fraction interactions common, <br /> between networks. Given differences experimental <br /> techniques used construct networks fact the <br /> edges TAP-MS BAYESIAN networks correspond to <br /> membership multiprotein complexes, Y2H physical <br /> contacts, DIP CORE, LC, HC networks mix <br /> of things, surprising networks differ <br /> significantly terms density, cliquishness, parame- <br /> ters. The biggest outlier Y2H network. In fact, this <br /> network, relation essentiality lethality less <br /> prominent discussed section. <br />  <br /> The Centrality-Lethality Rule Holds Six Networks <br />    In influential paper, Jeong et al. [3] observed the <br /> degree node yeast protein interaction network correlates <br /> with phenotypic effect deletion. More specifically, the <br /> authors observed high-degree nodes times more <br /> likely essential nodes having interaction partners. It <br /> was hypothesized high-degree nodes tend be <br /> essential central role play maintaining the <br /> overall connectivity network mediating interactions <br />  <br />  Table 2. Amount overlap tested networks. <br />  <br />  <br />  DIP CORE        0.58      0.62         0.25             0.61             0.02 <br />  0.28            LC        0.53         0.26             0.39             0.01 <br />  0.38            0.65      HC           0.47             0.47             0.02 <br />  0.09            0.18      0.27         TAP-MS           0.36             0.00 <br />                                                                                          Figure 1. Relationship degree essentiality the <br />  0.16            0.21      0.20         0.27             BAYESIAN         0.02           tested networks. (A) For tested network fraction essential <br />  0.26            0.18      0.31         0.10             0.97             Y2H            nodes nodes highest degree (hubs) shown. The <br />                                                                                          horizontal <span id='am-40' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-43'  ></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>axis</span> shows fraction total network nodes that <br />  Each row table corresponds single network shows fraction           designated hubs. (B) Correlation degree and <br />  edges contained tested networks. Thus, example, 58%             essentiality assessed Kendall&#8217;s tau Spearman&#8217;s rho rank <br />  edges DIP CORE network present LC network.                       correlation coefficients. <br />  doi:10.1371/journal.pcbi.1000140.t002                                                   doi:10.1371/journal.pcbi.1000140.g001 <br />  <br />  <br />         PLoS Computational Biology | www.ploscompbiol.org                            3                          August 2008 | Volume 4 | Issue 8 | e1000140 <br />                                                                                                                                    Essentiality Hub Proteins <br />  <br />  <br />    From Figure 1A clear enrichment values increase with <br /> k. Therefore, use nonparametric measure association, <br /> such Kendall&#8217;s tau Spearman&#8217;s rho rank correlation coefficients [26], to <br /> assess correlation degree essentiality all <br /> network nodes. As shown Figure 1B measures agree in <br /> their estimates strength correlation; further <br /> evaluations Kendall&#8217;s tau rank correlation <br /> coefficient. Then, assess correlation centrality <br /> measures essentiality correcting correlation degree, <br /> we used partial Kendall&#8217;s tau rank correlation. <br />    It noted contrast networks Y2H <br /> network exhibits weak correlation degree and <br /> essentiality. This agreement study Batada et al. [4]. <br /> They observed highly significant difference average degree <br /> of essential nonessential proteins LC network found <br /> that difference disappears analysis restricted <br /> to interactions detected yeast-two-hybrid experiments. <br />  <br /> Essential Hubs Are Not More Important Maintaining                                  Figure 2. Centrality measures demonstrated toy network. <br />                                                                                       Here demonstrate difference centrality measures a <br /> the Overall Network Connectivity Than Nonessential                                    toy network. (A) The toy network consists cliques: K50 nodes <br /> Hubs                                                                                  A1&#8211;A50 K10 nodes B1&#8211;B10. The cliques interconnected <br />     A network centrality index assigns centrality value                     edge (A1, B1) additional vertex D. Additional node <br /> node network quantifies topological prominence.                       C attaches network A2. (B) As measures assign <br /> Topological prominence defined number ways,                        centrality values based different network properties rank <br />                                                                                       nodes differently. Briefly, eigenvector centrality measure (EC) will <br /> over years centrality indices introduced                           assign high-centrality values nodes close other <br /> emphasize different aspects network topology [27]. In local                      central nodes network. The subgraph centrality measure (SC) <br /> centrality index, node&#8217;s centrality value mainly influenced                 assigns centrality values node based number closed walks <br /> the topology local neighborhood. A known example                     originate node. The shortest path betweenness centrality <br /> local centrality index degree centrality, node&#8217;s                         measure (SPBC) assigns node centrality value based fraction <br /> centrality value equal number immediate neighbors.                   shortest paths pass node averaged pairs of <br />                                                                                       nodes network. The current-flow betweenness centrality measure <br /> Betweenness indices, hand, assign centrality values                      (CFC) generalizes SPBC measure including additional paths, not <br /> based node&#8217;s role maintaining connectivity                      just shortest paths, computation. Here, difference <br /> pairs nodes network. A well-known example                        measures exemplified rankings produce <br /> betweenness centrality index shortest-path betweenness central-                    toy network nodes. <br /> ity, node&#8217;s centrality value proportional fraction                doi:10.1371/journal.pcbi.1000140.g002 <br /> of shortest paths pass it. <br />     Even degree centrality local centrality index,                network, local betweenness <br /> networks hubs play important role maintaining                           indices HC BAYESIAN networks. The local measures <br /> overall connectivity network. For example,                              strongly agree ranking network nodes networks <br /> demonstrated scale-free networks removal                          Y2H network. The agreement strongest the <br /> hubs affects ability nodes communicate                      TAP-MS network; result curves EC SC <br /> than removal random nodes [8]. To clarify topological                      measures overlap completely Figure 3D. <br /> role hubs tested networks, compared degree centrality                       While removal set nodes disconnect various <br /> to local indices (eigenvector centrality (EC) [28]                      parts network, impair significantly &#8216;&#8216;quality of <br /> subgraph centrality (SC) [29]) betweenness indices                         communication&#8217;&#8217; them. For example, an <br /> (shortest-path betweenness centrality (SPBC) [30] current-flow                    increase length shortest path decrease number <br /> betweenness centrality (CFC) [31]). (See Figure 2                              alternative paths pairs nodes network. <br /> illustration, Materials Methods detailed                           Therefore, introduced additional measures, call <br /> description centrality measures used study.)                           network integrity measures, capture various aspects effect of <br />     Since betweenness indices rank nodes based role                       nodes&#8217; removal ability nodes communicate. <br /> mediating communication pairs nodes                           (See Materials Methods description network integrity <br /> network, interesting compare effectiveness high-                      measures.) We sensitive measures are <br /> degree nodes nodes high betweenness centrality                            used observations disruptive power of <br /> disconnecting network. One common way measure                              hubs relative central proteins hold (Table S1). <br /> impact nodes&#8217; removal network connectivity                           Next, examined disruption power hubs comes <br /> monitoring decrease size largest connected                          mainly essential hubs. First, observe removal of <br /> component. Figure 3A&#8211;F shows, protein interaction                         essential proteins huge connected component less <br /> networks, removal central nodes, random                           disruptive removal equivalent number most <br /> nodes, essential proteins affects network connectivity. As                    central nodes according index (Figure 3A&#8211;F). Moreover, as <br /> expected, removing nodes high local centrality values                    shown Table 3, removal essential nodes more <br /> less disruptive removing high betweenness                             disruptive removal equivalent number random <br /> centrality values. Interestingly, degree centrality efficient                nonessential nodes degree distribution. We <br /> shattering network betweenness DIP CORE, LC,                        conclude networks, DIP CORE, LC, <br /> Y2H networks, inefficient local indices TAP-MS                    HC, Y2H networks, removal high-degree nodes is <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                          4                         August 2008 | Volume 4 | Issue 8 | e1000140 <br />                                                                            Essentiality Hub Proteins <br />  <br />  <br />  <br />  <br /> PLoS Computational Biology | www.ploscompbiol.org   5   August 2008 | Volume 4 | Issue 8 | e1000140 <br />                                                                                                                                    Essentiality Hub Proteins <br />  <br />  <br />  <br /> Figure 3. Vulnerability attack central proteins. (A&#8211;F) The impact node removal quantified fraction nodes the <br /> largest connected component. There curve centrality measure shows fraction nodes largest connected component <br /> as function fraction central nodes removed. We impact node removal random <span id='am-45' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-46' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> size the <br /> largest connected component essential proteins removed. <br /> doi:10.1371/journal.pcbi.1000140.g003 <br />  <br />  <br /> disruptive, disruption related essentiality                We Reject Essential Protein Interaction Model <br /> nodes. On contrary, essential genes indistinguishable                           Recently He colleagues [9] proposed explanation the <br /> that respect random nonessential genes                          centrality-lethality rule terms essential protein interactions: a <br /> degree distribution.                                                                   protein essential involvement more <br />                                                                                        essential protein interactions factors. The authors <br /> There Is No Relationship Disruptive Power                                  argue determination protein essentiality protein <br /> of Centrality Index Its Enrichment Essential                                  interaction network captured simple random process: <br /> Proteins                                                                               (i) distribute essential protein interactions edges the <br />    Above demonstrated various centrality indices vary                          network uniformly random probability a; (ii) distribute <br /> considerably ability predict disruption overall                     essential proteins nodes network uniformly at <br /> connectivity network. Next asked difference                  random probability b. Thus, according model, the <br /> reflected enrichment levels. Figure 4 shows fraction                     probability (PE) protein k neighbors essential is <br /> essential proteins hubs equivalent number                         PE = 12(12a)k(12b), natural logarithm fraction of <br /> central proteins according centrality measures. We observe                     nonessential proteins proteins degree k linear <br /> that local centrality indices enrichment levels comparable                 dependency k: log(12PE) = log(12a)k+log(12b). <br /> those betweenness indices cases higher. But                       We note assumptions essential protein <br /> notably, degree centrality fares better centrality index                interaction model follows proteins interact <br /> in networks narrowly beaten shortest-path centrality                essentiality protein pair does depend <br /> the Y2H network. The superiority degree centrality                     essentiality protein. Furthermore, this <br /> apparent Kendall&#8217;s tau rank correlation coefficient used                    independence observed proteins share <br /> measure correlation centrality values essentiality                    interaction neighbors. To test holds real data, we <br /> all network nodes (compare Table 2 Table 4).                                        computed number nonadjacent protein pairs, or <br />    As considerable correlation degree centrality                      neighbors (one neighbors Y2H network), that <br /> and centrality indices, used Kendall&#8217;s tau partial rank                       essential nonessential tested networks <br /> correlation coefficient indices                           compared numbers expected number such <br /> correlated essentiality correlation degree                        pairs model. (The model parameters estimated <br /> centrality index. We that, controlling correlation                       using different strategies described Materials and <br /> with degree, correlation essentiality reduced                           Methods. In paper, He et al. point model may <br /> statistically insignificant values betweenness centrality indices                  work networks edges represent membership in <br /> and greatly reduced local indices (Table 4).                                    protein complex. Thus, excluded TAP-MS and <br />    The observations indicate main topological                           BAYESIAN networks analysis.) As shown Table 5, the <br /> determinant essentiality node&#8217;s local neighborhood                    model does capture correlation essentiality observed in <br /> than role maintaining overall connectivity network. <br /> In particular, removing nodes high <br /> betweenness centrality indices effective shattering <br /> some protein interaction networks, correlation with <br /> essentiality reduced statistically insignificant levels by <br /> subtracting correlation degree centrality. <br />  <br />  Table 3. Impact removal essential proteins as <br />  compared removal equivalent number random <br />  nonessential proteins degree distribution. <br />  <br />  <br />                        Essential          Random nonessential <br />  <br />  DIP CORE              0.519              0.50460.007 <br />  LC                    0.578              0.55160.010 <br />  HC                    0.521              0.52560.005 <br />  TAP-MS                0.512              0.51260.011 <br />  BAYESIAN              0.685              0.62560.006 <br />  Y2H                   0.410              0.39760.046                                  Figure 4. Enrichment hubs equivalent number of <br />                                                                                        central nodes according centrality measures in <br />  The impact removal set proteins measured fraction nodes       essential proteins. Fraction essential proteins hubs an <br />  largest connected component. For network effect                equivalent number central nodes according other <br />  removal essential proteins removal equivalent number              centrality measures. The fraction essential proteins nodes <br />  random nonessential proteins degree shown.                           network shown ntwk.avg. <br />  doi:10.1371/journal.pcbi.1000140.t003                                                 doi:10.1371/journal.pcbi.1000140.g004 <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                           6                         August 2008 | Volume 4 | Issue 8 | e1000140 <br />                                                                                                                                                  Essentiality Hub Proteins <br />  <br />  <br />  <br />  Table 4. Correlation centrality indices essentiality. <br />  <br />  <br />                               Eigenvector centrality                                                    Subgraph centrality <br />  <br />                               tess                               tess.dc                                tess                                 tess.dc <br />  <br />  DIP CORE                     0.15 (3.5e-19)                     0.064 (8.6e-05)                        0.17 (1.2e-24)                       0.059 (2.5e-04) <br />  LC                           0.23 (7.9e-56)                     0.094 (3.6e-11)                        0.23 (1.2e-55)                       0.093 (4.9e-11) <br />  HC                           0.24 (1.8e-54)                     0.107 (2.9e-12)                        0.24 (7.9e-55)                       0.102 (3.4e-11) <br />  TAP-MS                       0.12 (8.42e-11)                    20.007 (6.5e-01)                       0.12 (8.42e-11)                      20.007 (6.5e-01) <br />  BAYESIAN                     0.17 (5.7e-39)                     0.046 (1.5e-04)                        0.17 (5.1e-41)                       0.051 (3.1e-05) <br />  Y2H                          0.05 (1.1e-01)                     0.027 (2.5e-01)                        0.03 (2.0e-01)                       20.024 (7.2e-01) <br />  <br />                               Shortest-path betweenness centrality                                      Current-flow betweenness <br />  <br />                               tess                               tess.dc                                tess                                 tess.dc <br />  <br />  DIP CORE                     0.15 (3.2e-18)                     20.002 (5.5e-01)                       0.19 (2.7e-27)                       0.012 (2.5e-01) <br />  LC                           0.21 (1.4e-46)                     0.003 (4.25e-01)                       0.26 (3.7e-70)                       20.007 (6.8e-01) <br />  HC                           0.20 (1.9e-36)                     0.005 (3.7e-01)                        0.24 (2.6e-53)                       20.005 (6.2e-01) <br />  TAP-MS                       0.12 (3.5e-11)                     0.018 (1.8e-01)                        0.16 (3.3e-18)                       0.017 (1.8e-01) <br />  BAYESIAN                     0.18 (2.4e-41)                     0.005 (3.43e-01)                       0.23 (2.7e-69)                       0.018 (8.1e-02) <br />  Y2H                          0.10 (1.2e-02)                     0.048 (1.4e-01)                        0.10 (1.4e-02)                       0.041 (1.8e-01) <br />  <br />  The correlation centrality measures essentiality (tess) measured Kendall&#8217;s tau rank correlation coefficient. The correlation essentiality, controlling <br />  correlation degree centrality, measured using partial Kendall&#8217;s tau rank correlation coefficient (tess.dc). The p-values derived Kendall&#8217;s tau z- <br />  scores shown parentheses. <br />  doi:10.1371/journal.pcbi.1000140.t004 <br />  <br /> the tested networks; i.e., statistically significant difference                proteins. Some large multiprotein complexes, example, those <br /> between number pairs observed real data                            involved transcription regulation, known highly enriched <br /> number expected model. Consequently, essential                              essential proteins, general phenomenon it <br /> interaction model rejected high confidence.                                       account centrality-lethality rule? <br />                                                                                              To investigate question, introduce notion of <br /> We Propose Alternative Explanation Centrality-                                 Essential Complex Biological Modules, groups proteins with <br /> Lethality Rule                                                                            shared biological function extensively interact other <br />    In previous section showed proteins share                             enriched essential proteins. First, automatic <br /> neighbors likely essentiality (be                          method extraction ECOBIMs protein interaction <br /> essential nonessential) expected essential PPI                     network. Next, argue membership ECOBIMs accounts <br /> model. Moreover, observed study essential                          large extent centrality-lethality rule tested networks. <br /> proteins distributed uniformly set                                Finally, address statistical issues related selection procedure <br /> automatically derived multiprotein complexes [17]. This suggests                          applying suitable randomization protocols. <br /> that densely connected subnetworks polarized                                We developed automatic method extraction ECOBIMs <br /> either highly enriched significantly depleted essential proteins.                   protein interaction network. In work proteins deemed <br /> Furthermore, known densely connected subnetworks                          share biological function annotated GO <br /> are enriched proteins share biological function. Therefore,                       biological process term set 192 terms selected by <br /> one expect protein interaction networks contain densely                       group experts represent relevant aspects molecular biology <br /> connected functional modules highly enriched essential                        [32]. Therefore, method applied subnetworks induced by <br />  <br />  Table 5. Difference observed expected number pairs proteins essential nonessential. <br />  <br />  <br />                     Total number pairs            Number pairs type               Expected number pairs type <br />  <br />                                                                                                     Simulation               Line fitting              Weighted line fitting <br />  <br />  DIP CORE           1,849                            1,135                                          945 (3.6e-10)            928 (8.6e-12)             938 (8.0e-11) <br />  LC                 10,777                           6,143                                          5,691 (6.6e-10)          5.556 (1.1e-15)           5.589 (3.9e-14) <br />  HC                 5,907                            3,516                                          3,213 (2.0e-08)          2,997 (2.2e-16)           2,994 (2.2e-16) <br />  Y2H                3,254                            2,167                                          1,976 (9.6e-07)          2,025 (2.6e-04)           2,052 (3.3e-03) <br />  <br />  The total number pairs refers number nonadjacent protein pairs common neighbors network. (Due sparsity Y2H <br />  network, statistics calculated nonadjacent pairs having neighbors common.) The nodes pair &#8216;&#8216;the type&#8217;&#8217; both <br />  essential nonessential. <br />  doi:10.1371/journal.pcbi.1000140.t005 <br />  <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                              7                               August 2008 | Volume 4 | Issue 8 | e1000140 <br />                                                                                                                            Essentiality Hub Proteins <br />  <br />  <br /> proteins annotated GO biological process term,              LC network contains multiprotein complexes involved the <br /> subnetwork time. The high-level <span id='am-30' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-31' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33'  ></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>idea</span> method             process RNA polymerase 2 transcription [33], RNA <br /> first identify groups densely connected proteins,            polymerase 2, general transcription factors, mediator complex, <br /> Complex Biological Modules (or COBIMs), identify subset         etc. The ECOBIMs 20 members shown Table 7; <br /> COBIMs ECOBIMs based distribution essential                      ECOBIMs member proteins given Table S2. <br /> proteins COBIM nodes. More specifically, heuristic                 Moreover, ECOBIMs remarkably different non- <br /> selects subset COBIMs enriched essential proteins.           ECOBIM COBIMs. As shown Table 8, distribution of <br /> (The method schematically shown Figure 5 described            essential proteins COBIM nodes highly uneven. In <br /> detail Materials Methods section. Figure S1 shows              particular, observed difference fractions essential <br /> fraction nodes members r COBIMs various            proteins ECOBIM nodes non-ECOBIM <br /> values r.)                                                                 COBIM nodes accounted degrees of <br />    To examine extent membership ECOBIMs                        essential COBIM nodes particular ECOBIM selection <br /> accounts centrality-lethality rule partitioned hubs       procedure. The claim validated performing 100,000 <br /> groups, members ECOBIMs                         randomized assignments essential proteins preserve degrees <br /> (ECOBIM hubs) (non-ECOBIM hubs),                   number essential COBIM nodes, selecting the <br /> compared enrichment values. As shown Figure 6 ECOBIM                 ECOBIMs computing corresponding fractions. As shown <br /> hubs highly enriched essential proteins, non-                  Table 8, values obtained true assignment of <br /> ECOBIM hubs depleted essential proteins compared                 essential proteins significantly different obtained <br /> the network average enrichment values. But importantly,               randomized assignment essential proteins. <br /> discussed paragraph, difference fraction               So far, demonstrated high correlation degree <br /> essential proteins ECOBIM hubs non-ECOBIM hubs                   essentially predominantly attributed ECOBIMs. In <br /> not result greedy ECOBIM selection procedure particular           addition, known certain functions essential to <br /> degree sequence essential proteins network. We asked           cell, example, transcription regulation cell-cycle regulation, <br /> whether correlation degree lethality               rely large multiprotein complexes. Indeed, GO terms <br /> network nodes members ECOBIMs. As shown                overrepresented ECOBIM nodes type, as <br /> Table 6 correlation essentiality degree non-              seen Figure 7. Do ECOBIMs play distinguished role those <br /> ECOBIM nodes network nodes.                    essential processes, merely byproduct above- <br />    One ask extent difference behavior               mentioned observation? In particular, difference the <br /> ECOBIM hubs non-ECOBIM hubs particular                      enrichment essential proteins exclusively fact some <br /> selection procedure employ identify putative                   essential GO processes contain ECOBIMs not? <br /> ECOBIMs. More specifically, concerns need                  To elucidate role ECOBIMs examined GO <br /> be addressed. First, method guided enrichment                processes contain ECOBIM. Table 9 shows the <br /> essential proteins selecting ECOBIMs COBIMs.                        results DIP core network sorted percentage of <br /> Therefore, expected fraction essential proteins       essential proteins given GO process. (The data other <br /> ECOBIM hubs higher non-ECOBIM                       networks given Table S3.) Observe enrichment of <br /> hubs. Second, method considers annotated yeast genes.                ECOBIMs essential genes typically higher the <br /> Therefore, argue difference behavior          average enrichment corresponding GO. Thus, the <br /> the fact ECOBIM hubs necessarily annotated non-                ECOBIMs merely representatives average structure <br /> ECOBIM hubs include annotated unannotated genes.                 corresponding GO subnetwork. The uneven distribution of <br />    To address concern performed control experiment             essential proteins observed corresponding <br /> where essential proteins assigned random set nodes               GO process extremely enriched essential proteins as <br /> having degree distribution true set essential              rRNA metabolic process (GO:0016072) transcription initiation <br /> proteins network. (A total 100,000 random assignments               (GO:0006352). The percentage essential proteins among <br /> were performed, resulted 100,000 sets ECOBIMs.) To                network nodes annotated processes is <br /> address second concern, restricted random assignment               80%, COBIMs selected ECOBIMs. The <br /> to annotated genes only. As shown Table 6, ECOBIMs                     process highest percentage essential proteins, <br /> resulting true assignment essential proteins                 transcription RNA polymerase III promoter (GO:0006383), <br /> dramatically different properties resulting               contains types COBIMs. Interestingly, process, all <br /> random assignment essential proteins. In particular, fraction          ECOBIM nodes essential, remaining COBIM <br /> of essential proteins non-ECOBIM hubs true                    nodes is. In fact, GO process contains ECOBIM and <br /> assignment essential proteins significantly lower             non-ECOBIM COBIMS, polarization frequent albeit <br /> under randomized assignment essential proteins,                   rarely extreme (Table 9). Removal protein a <br /> though selection procedure used cases.                    Complex Biological Module expected perturb disable <br /> Therefore, conclude observed difference result          module. Thus, large spectrum essential GO <br /> the particular distribution essential proteins nodes          processes, cell tolerate large perturbations modules <br /> the network artifact selection procedure. The               little perturbations ECOBIMs. <br /> same holds reduction correlation degree                   This observation explain poor correlation <br /> essentiality non-ECOBIM nodes.                                            degree essentiality Y2H networks, indicates that <br />                                                                               ECOBIMs likely contain large, stable multiprotein modules, <br /> We Identify Properties ECOBIMs                                         typically multiprotein complexes. However, interactions recovered <br />   The identified ECOBIMs correspond large essential                 Y2H technique correspond physical contacts such <br /> multiprotein complexes anaphase promoting complex                 encompass members complex. Moreover, its <br /> (APC) DAM1 protein complex exclusively                        binary nature, Y2H technique completely miss interactions <br /> complexes. For example, largest ECOBIMs identified                 complexes require cooperative binding [34]. <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                  8                          August 2008 | Volume 4 | Issue 8 | e1000140 <br />                                                                                                                       Essentiality Hub Proteins <br />  <br />  <br />  <br />  <br /> Figure 5. The automatic method extraction ECOBIMs. Here demonstrate major steps method HC network. The <br /> input method protein interaction network, GO annotation, set essential nodes, shown red. The method considers <br /> subnetworks induced proteins annotated GO biological process term, subnetwork time, identify densely connected <br /> regions COBIMs. The COBIMs shown COBIM intersection graph, nodes correspond COBIMs (the size node proportional <br /> to number genes corresponding COBIM) edge pair COBIMs proteins common. <br /> The COBIMs enriched essential proteins selected ECOBIMs, shown green. <br /> doi:10.1371/journal.pcbi.1000140.g005 <br />        PLoS Computational Biology | www.ploscompbiol.org               9                         August 2008 | Volume 4 | Issue 8 | e1000140 <br />                                                                                                                                         Essentiality Hub Proteins <br />  <br />  <br />  <br />  <br /> Figure 6. Enrichment ECOBIM non-ECOBIM hubs essential proteins. Fraction essential proteins various types hubs: all <br /> hubs, hubs members ECOBIMs (ECOBIM hubs), hubs members ECOBIMs (non-ECOBIM hubs). The fraction essential <br /> proteins proteins network shown (ntwk.avg.). The numbers bars number essential hubs total <br /> number hubs type ECOBIM non-ECOBIM hubs. <br /> doi:10.1371/journal.pcbi.1000140.g006 <br />  <br />  <br /> Discussion                                                                              determinant essentiality biological explanation for <br />                                                                                         connection network topology essentiality. <br />   The enrichment high-degree nodes essential proteins, known                         To address question performed rigorous analysis six <br /> as centrality-lethality rule, suggests topological                         protein interaction networks Saccharomyces cerevisiae compiled from <br /> prominence protein protein interaction network                       diverse sources interaction evidence. To clarify topological <br /> good predictor biological importance. There exist numerous                       roles essential proteins general essential hubs particular, <br /> measures topological prominence, called network centrality                           compared degree centrality local betweenness <br /> indices; local centrality indices assign centrality values based                 centrality indices. We networks high-degree <br /> topology node&#8217;s local neighborhood, betweenness                          nodes important maintaining overall network <br /> centrality indices assign centrality values based node&#8217;s role                 connectivity nodes having high betweenness centrality values, <br /> maintaining connectivity pairs nodes                        property essential proteins. On contrary, <br /> network. Even definition degree centrality local                         essential proteins indistinguishable respect from <br /> measure, depending structure network, hubs play                       nonessential proteins having degree distribution. We also <br /> an important role maintaining overall connectivity                        degree centrality better predictor essentiality than <br /> network. In paper sought identify main topological                       measure tested correlation betweenness indices <br />  <br />  <br />  Table 6. Membership ECOBIMs centrality-lethality rule. <br />  <br />  <br />                     Enrichment ECOBIM hubs                    Enrichment non-ECOBIM hubs              Corr. degree vs. essentiality non-ECOBIM hubs <br />  <br />                     Obs.        Rand.        p-value             Obs.       Rand.           p-value         Obs.               Rand.        p-value <br />  <br />  DIP CORE           0.80        0.67         1.98e-03            0.26       0.43            ,1.00e-05       0.08               0.18         ,1.00e-05 <br />  LC                 0.80        0.69         1.88e-03            0.32       0.48            ,1.00e-05       0.17               0.27         ,1.00e-05 <br />  HC                 0.83        0.70         4.00e-05            0.35       0.51            ,1.00e-05       0.17               0.27         ,1.00e-05 <br />  TAP-MS             0.76        0.62         1.00e-05            0.24       0.40            ,1.00e-05       0.12               0.20         ,1.00e-05 <br />  BAYESIAN           0.77        0.65         ,1.00e-05           0.18       0.36            ,1.00e-05       0.09               0.20         ,1.00e-05 <br />  Y2H                0.85        0.66         5.81e-02            0.13       0.25            2.00e-05        20.04              0.05         2.00e-04 <br />  <br />  For quantity values shown: value true assignment essential proteins (Obs.), mean value randomized assignment of <br />  essential proteins (Rand.), fraction randomized assignments resulted values stronger (either smaller larger depending context) than <br />  obtained true assignment essential proteins (p-value). <br />  doi:10.1371/journal.pcbi.1000140.t006 <br />  <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                           10                         August 2008 | Volume 4 | Issue 8 | e1000140 <br />                                                                                                                                      Essentiality Hub Proteins <br />  <br />  <br />  <br /> Table 7. Largest ECOBIMs extracted tested                                       essentiality entirely correlation degree <br /> networks.                                                                                centrality. Thus, conclude topological determinant of <br />                                                                                          essentiality node&#8217;s local neighborhood role in <br />                                                                                          maintaining overall connectivity network. <br /> The DIP CORE network                                                                        Next examined essential interactions model, <br /> GO:0006508 proteolysis                                            27 35 0.77 <br />                                                                                          recently proposed explain centrality-lethality rule, valid in <br />                                                                                          tested networks. We model&#8217;s central assumption <br /> GO:0042254 ribosome biogenesis assembly                       27 32 0.84 <br />                                                                                          majority proteins essential involvement <br /> GO:0016192 vesicle mediated transport                             21 30 0.70             essential protein interactions, distributed <br /> GO:0016071 mRNA metabolic process                                 18 28 0.64             uniformly random edges network, violates basic <br /> GO:0015931 nucleobase, nucleoside, nucleotide nucleic         15 24 0.62             clustering patterns essential proteins networks we <br /> acid transport GO:0051236 establishment RNA localization                              examined. The uniform distribution essential protein interac- <br /> GO:0016072 rRNA metabolic process                                 18 21 0.86             tions implies that, long proteins interact, the <br /> GO:0008380 RNA splicing                                           16 21 0.76             essentiality protein pair independent the <br />                                                                                          essentiality protein. However, real protein <br /> The LC network <br />                                                                                          interaction networks essentiality pairs proteins share <br /> GO:0042254 ribosome biogenesis assembly                       88 107 0.82 <br />                                                                                          neighbors correlated, number nonadjacent <br /> GO:0016071 mRNA metabolic process                                 37 58 0.64             protein pairs share neighbors either <br /> GO:0008380 RNA splicing                                           35 52 0.67             essential nonessential significantly deviates the <br /> GO:0015931 nucleobase, nucleoside, nucleotide nucleic         16 26 0.62             expected number pairs model. Consequently, we <br /> acid transport GO:0051236 establishment RNA localization                              rejected essential interactions explanation high confi- <br /> GO:0006508 proteolysis                                            17 24 0.71             dence. We stress reject existence essential <br /> The HC network <br />                                                                                          protein interactions assumption these <br />                                                                                          interactions evenly distributed edges network <br /> GO:0042254 ribosome biogenesis assembly                       84 100 0.84 <br />                                                                                          explain degree distribution essential proteins. <br /> GO:0016071 mRNA metabolic process                                 49 71 0.69                The observations led propose alternative <br /> GO:0016072 rRNA metabolic process                                 63 71 0.89             explanation centrality-lethality rule. Our explanation builds <br /> GO:0008380 RNA splicing                                           46 63 0.73             growing body evidence gene knock-out phenotypes for <br /> GO:0006508 proteolysis                                            28 35 0.80             genes gene products members multiprotein <br />                                                                                          complex correlated [16,17]. In particular, Hart et al. <br /> The TAP-MS network <br />                                                                                          demonstrated essential proteins distributed evenly <br /> GO:0042254 ribosome biogenesis assembly                       90 120 0.75 <br />                                                                                          set automatically identified multiprotein complexes; <br /> GO:0016071 mRNA metabolic process                                 46 66 0.70             &#8216;&#8216;surprisingly&#8217;&#8217; complexes majority <br /> GO:0008380 RNA splicing                                           45 62 0.73             members essential &#8216;&#8216;surprisingly&#8217;&#8217; complexes where <br /> GO:0016072 rRNA metabolic process                                 37 41 0.90             majority members essential [17]. Here we <br /> GO:0016072 rRNA metabolic process                                 30 32 0.94 <br />                                                                                          hypothesized computationally confirmed same <br />                                                                                          phenomenon holds potentially larger groups densely <br /> GO:0006508 proteolysis                                            17 22 0.77 <br />                                                                                          connected functionally related proteins called Complex <br /> The BAYESIAN network                                                                     Biological Modules abbreviated COBIMs. But more <br /> GO:0042254 ribosome biogenesis assembly                       119 152 0.78           importantly, able demonstrate membership in <br /> GO:0016072 rRNA metabolic process                                 93 106 0.88            ECOBIMs, COBIMs enriched essential proteins, <br /> GO:0008380 RNA splicing GO:0016071 mRNA metabolic                 40 50 0.80             provides good explanation correlation degree and <br /> process                                                                                  essentiality protein interaction networks considered this <br /> GO:0006366 transcription RNA polymerase II promoter          23 42 0.55             study. In particular, showed non-ECOBIM hubs are <br />                                                                                          depleted essential proteins non-ECOBIM proteins the <br /> GO:0006508 proteolysis                                            28 37 0.76 <br />                                                                                          correlation degree essentiality greatly reduced. <br /> GO:0006913 nucleocytoplasmic transport                            17 31 0.55 <br />                                                                                          Moreover, applying suitable randomization protocols showed <br /> GO:0006412 translation                                            18 27 0.67             different characteristics ECOBIM non-ECOBIM <br /> GO:0051169 nuclear transport                                      15 27 0.55             hubs (or general ECOBIM non-ECOBIM proteins) a <br /> GO:0045184 establishment protein localization                  15 27 0.55             mere consequence degrees particular computational <br /> The Y2H network <br />                                                                                          method adopted selecting ECOBIMs. <br />                                                                                             In past, attempts classify high-degree <br /> GO:0007010 cytoskeleton organization biogenesis               9    11 0.82 <br />                                                                                          nodes using additional biological data obtain deeper insight <br /> GO:0006366 transcription RNA polymerase II promoter          7    11 0.64           biological physiological properties hubs were <br /> GO:0045184 establishment protein localization                  6    10 0.60           reported possess. Here discuss findings fit the <br /> GO:0006913 nucleocytoplasmic transport GO:0051169 nuclear         6    10 0.60           results reported studies [35,36]. Han et al. utilized <br /> transport                                                                                mRNA expression data classify hubs party date hubs, <br />                                                                                          party hubs significant agreement mRNA <br /> For tested protein interaction network list ECOBIMs least <br /> 20 members; Y2H network, ECOBIMs 10 members are <br />                                                                                          expression levels, coexpressed, interacting <br /> listed. For ECOBIM following information shown: corresponding            partners, date hubs coexpressed their <br /> GO biological process term, number essential genes, number genes,              neighbors [35]. The removal date hubs observed to <br /> fraction essential genes. For list ECOBIMs member genes        shatter network efficiently removal of <br /> Table S2. <br /> doi:10.1371/journal.pcbi.1000140.t007 <br />                                                                                          party hubs. On basis observations the <br />                                                                                          paper, date hubs proposed &#8216;&#8216;&#8230;participate wide range <br />  <br />  <br />       PLoS Computational Biology | www.ploscompbiol.org                             11                         August 2008 | Volume 4 | Issue 8 | e1000140 <br />                                                                                                                                            Essentiality Hub Proteins <br />  <br />  <br />  <br />  Table 8. ECOBIMs contain large fraction essential COBIM proteins. <br />  <br />  <br />                                           Enrich. ECOBIM proteins                                                 Enrich. non-ECOBIM COBIM proteins <br />  <br />                                           Obs.             Rand.             p-value                              Obs.             Rand.             p-value <br />  <br />  DIP CORE                                 0.77             0.65              ,1.0e-05                             0.06             0.21              ,1.0e-05 <br />  LC                                       0.77             0.65              1.00e-05                             0.10             0.17              1.56e-03 <br />  HC                                       0.81             0.68              ,1.00e-05                            0.12             0.18              2.31e-02 <br />  TAP-MS                                   0.74             0.64              ,1.00e-05                            0.09             0.17              1.87e-03 <br />  BAYESIAN                                 0.76             0.65              ,1.00e-05                            0.08             0.18              ,1.00e-05 <br />  Y2H                                      0.79             0.63              9.93e-03                             0.06             0.17              3.00e-05 <br />  <br />  For network enrichment essential proteins ECOBIM nodes enrichment COBIM nodes members ECOBIMs shown. For <br />  group values listed: fraction true assignment essential proteins (Obs.), mean fraction randomized assignment essential <br />  proteins (Rand.), p-value difference. <br />  doi:10.1371/journal.pcbi.1000140.t008 <br />  <br />  <br /> of integrated connections required global organization                       COBIMs hubs explicitly defined hubs members <br /> biological modules proteome network&#8230;&#8217;&#8217; However,                        highly connected modules. Similarly party hubs, average <br /> fraction essential proteins party hubs slightly                   enrichment COBIM hubs essential proteins slightly higher <br /> higher date hubs. This consistent                    non-COBIM hubs (data shown). We also <br /> the conclusions paper, namely, essentiality                       demonstrated essential proteins clearly cluster ECOBIMs <br /> byproduct node&#8217;s ability maintain overall connectivity                 uniformly distributed COBIMs. <br /> the network. Furthermore, proposed &#8216;&#8216;party hubs                            In second study Kim et al. utilized structural data classify <br /> represent integral elements distinct modules&#8217;&#8217; &#8216;&#8216;tend                     hubs singlish-interface multiinterface hubs, singlish- <br /> function lower level organization proteome&#8217;&#8217; [35].                   interface hubs interact partners or <br /> Such description consistent properties COBIM hubs                         distinct interfaces, multiinterface hubs would <br />                                                                                         interact partners distinct interfaces <br />                                                                                         [36]. In case, however, classification produced significantly <br />                                                                                         different enrichment levels, multiinterface hub twice as <br />                                                                                         likely essential singlish-interface hub average network <br />                                                                                         node. The authors suggested multiinterface hubs likely <br />                                                                                         correspond members large stable multiprotein complexes. <br />                                                                                         Consequently, imply stable multiprotein complexes <br />                                                                                         enriched essential proteins. This view consistent the <br />                                                                                         results paper additional caveats discussed below. <br />                                                                                             It known certain biological functions essential the <br />                                                                                         cell depend large multiprotein complexes. (Consider, for <br />                                                                                         example, RNA Polymerase II transcription machinery [33] or <br />                                                                                         ribosome biogenesis assembly [37].) Indeed, ECOBIMs <br />                                                                                         indentified approach associated processes. <br />                                                                                         However, essential processes, ECOBIMs <br />                                                                                         distinguish enriched essential proteins <br />                                                                                         remaining proteins process. The <br />                                                                                         enrichment essential proteins non-ECOBIM COBIMs is <br />                                                                                         usually level frequently significantly lower the <br />                                                                                         average enrichment corresponding GO process. Thus, <br />                                                                                         large spectrum essential GO processes, cell can <br />                                                                                         tolerate large perturbations non-ECOBIM modules very <br />                                                                                         little perturbation ECOBIMs. Some COBIMs contain <br />                                                                                         essential proteins. In case, module be <br />                                                                                         nonessential, fact cell tolerate removal of <br />                                                                                         member COBIM does exclude possibility <br />                                                                                         COBIM corresponds stable complex. <br />  <br />                                                                                         Materials Methods <br /> Figure 7. GO terms overrepresented ECOBIM <br /> nodes. For network GO terms overrepresented                          Network Centrality Indices <br /> among ECOBIM nodes shown. The overrepresentation GO term                          In work compare degree centrality measure two <br /> is quantified natural logarithm p-value, p-value is <br /> the probability number ECOBIM genes would <br />                                                                                         local measures (eigenvector centrality (EC) [28] and <br /> belong GO term ECOBIM genes selected uniformly                      subgraph centrality (SC) [29]) betweenness measures <br /> at random network genes.                                                       (shortest-path betweenness centrality (SPBC) [30] current-flow <br /> doi:10.1371/journal.pcbi.1000140.g007                                                   betweenness centrality (CFC) [31]). <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                          12                            August 2008 | Volume 4 | Issue 8 | e1000140 <br />                                                                                                                                Essentiality Hub Proteins <br />  <br />  <br />  <br />  Table 9. Enrichment ECOBIM non-ECOBIM COBIM nodes GO subnetworks DIP CORE network. <br />  <br />  <br />  GO term                                                              Subnetwork nodes            ECOBIM nodes              Non-ECOBIM COBIM nodes <br />  <br />  GO:0016072 rRNA metabolic process                                    0.83                        0.91                      n/a <br />  GO:0006352 transcription initiation                                  0.82                        1.00                      n/a <br />  GO:0006383 transcription RNA polymerase III pro                 0.77                        1.00                      0.00 <br />  GO:0042254 ribosome biogenesis assembly                          0.72                        0.87                      n/a <br />  GO:0008380 RNA splicing                                              0.71                        0.79                      0.50 <br />  GO:0006839 mitochondrial transport                                   0.64                        0.80                      n/a <br />  GO:0006360 transcription RNA polymerase I pro                   0.64                        0.80                      0.00 <br />  GO:0016071 mRNA metabolic process                                    0.63                        0.75                      0.40 <br />  GO:0006260 DNA replication                                           0.61                        0.93                      n/a <br />  GO:0031123 RNA 39-end processing                                     0.59                        0.93                      0.29 <br />  GO:0006399 tRNA metabolic process                                    0.50                        1.00                      0.00 <br />  GO:0007059 chromosome segregation                                    0.49                        0.76                      n/a <br />  GO:0006944 membrane fusion                                           0.48                        0.75                      0.22 <br />  GO:0006508 proteolysis                                               0.46                        0.77                      n/a <br />  GO:0051169 nuclear transport                                         0.44                        0.80                      0.47 <br />  GO:0006997 nuclear organization biogenesis                       0.43                        1.00                      0.33 <br />  GO:0000278 mitotic cell cycle                                        0.43                        0.81                      0.19 <br />  GO:0015931 nucleobase, nucleoside, nucleotide n                  0.42                        0.63                      n/a <br />  GO:0006913 nucleocytoplasmic transport                               0.42                        0.80                      0.41 <br />  GO:0051236 establishment RNA localization                         0.42                        0.63                      n/a <br />  GO:0006366 transcription RNA polymerase II pro                  0.40                        0.75                      0.29 <br />  GO:0007010 cytoskeleton organization biogenesis                  0.40                        0.78                      0.00 <br />  GO:0048308 organelle inheritance                                     0.39                        0.86                      n/a <br />  GO:0006401 RNA catabolic process                                     0.38                        0.83                      0.41 <br />  GO:0006461 protein complex assembly                                  0.38                        1.00                      n/a <br />  GO:0045184 establishment protein localization                     0.37                        0.89                      0.38 <br />  GO:0009100 glycoprotein metabolic process                            0.37                        0.63                      n/a <br />  GO:0006412 translation                                               0.36                        0.85                      0.00 <br />  GO:0007005 mitochondrion organization biogenes                   0.35                        0.91                      n/a <br />  GO:0006512 ubiquitin cycle                                           0.34                        0.82                      n/a <br />  GO:0051325 interphase                                                0.33                        0.83                      0.00 <br />  GO:0016192 vesicle-mediated transport                                0.31                        0.71                      0.18 <br />  GO:0000074 regulation progression cell cycl               0.31                        0.73                      0.18 <br />  GO:0000279 M phase                                                   0.30                        0.80                      0.17 <br />  GO:0006974 response DNA damage stimulus                           0.28                        0.67                      0.11 <br />  GO:0006323 DNA packaging                                             0.26                        1.00                      0.16 <br />  GO:0006417 regulation translation                                 0.26                        0.80                      n/a <br />  GO:0016481 negative regulation transcription                      0.25                        1.00                      0.13 <br />  GO:0007001 chromosome organization biogenesi                     0.22                        0.79                      0.16 <br />  GO:0016458 gene silencing                                            0.22                        1.00                      0.00 <br />  GO:0040029 regulation gene expression, epigenet                   0.21                        1.00                      0.00 <br />  GO:0007047 cell wall organization biogenesis                     0.17                        0.75                      n/a <br />  <br />  For GO subnetwork contributed ECOBIM, fractions essential proteins subnetwork nodes, subnetwork ECOBIM nodes, and <br />  subnetwork non-ECOBIM COBIM nodes shown. <br />  doi:10.1371/journal.pcbi.1000140.t009 <br />  <br />  <br />  <br />                                                                                                       P <br />    The computation eigenvector centrality values cast               iteration xkz1 <br />                                                                                                  ~ j xkj normalize scores <br /> as iterative process: (i) start initial vector centrality           xkz1 ~~ <br />                                                                                   ~       xkz1 =j~ <br />                                                                                                  xkz1 j. It shown process converges <br />        x0 ~&#240;x0 . . . xn &#222;; (ii) iteration k+1 update centrality <br /> scores ~                                                                          eigenvector corresponds largest eigenvalue the <br /> score node using scores neighbors previous             adjacency matrix network. <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                     13                          August 2008 | Volume 4 | Issue 8 | e1000140 <br />                                                                                                                                  Essentiality Hub Proteins <br />  <br />  <br />    The subgraph centrality value node equal number                 biological process terms, selected group experts to <br /> of closed walks start terminate node. As              represent relevant aspects molecular biology. Thus, method <br /> infinite number walks, obtain finite index values                  applied 192 subnetworks, subnetwork time. <br /> number closed walks length k weighted 1/k!. Therefore,                   From GO subnetwork method extracts groups of <br /> short walks dominate subgraph centrality values.                              densely connected proteins. An ideal dense network clique, a <br />    For shortest-path betweenness index, node&#8217;s centrality                 complete network pair nodes adjacent. Over the <br /> value equal average fraction shortest paths pass                years various generalizations clique concept proposed <br /> through node.                                                                 literature model wider set dense networks. Here we <br />    The current-flow centrality measure extends shortest-path                  adopt generalization based k-connectivity. We say a <br /> centrality measure taking account paths addition              pair nodes k-connected k node-disjoint paths the <br /> shortest paths. This achieved current-flow paradigm                  network them. We say network k-connected if <br /> where network viewed resistor network edge                  pair nodes k-connected. For example, (k+1)-clique, a <br /> having unit capacity. For pair nodes s t, unit              clique (k+1) nodes, k-connected. In fact, smallest k- <br /> current shipped s t, centrality node set        connected graph. <br /> average current passes node.                             Our method utilizes following approach regions of <br />    We demonstrate difference centrality                      GO subnetworks k-connected: start seed a <br /> measures toy network Figure 1A. In network                       (k+1)-clique iteratively extend seed addition of <br /> cliques K50 K10 interconnected edge (A1, B1)                    proteins k neighbors seed. In <br /> through node D. The nodes K50 labeled A1&#8230;A50,                    addition k-connected COBIMs satisfy following <br /> nodes K10 labeled B1&#8230;B10. An additional node C attaches                    property: nodes removed COBIM one such <br /> to K50 A2. Figure 1B shows ranking network nodes                   network induced remaining nodes k- <br /> based centrality values assigned centrality                    connected. We note k-connected network this <br /> measures.                                                                         property. Consider, example, cycle. The cycle 2-connected. <br />                                                                                   However, removal node results path 1- <br /> Network Integrity Measures                                                        connected. The value parameter k chosen the <br />    We introduced measures, network integrity                    fraction COBIM nodes 25% number nodes in <br /> measures, capture various effects node removal ability               network. As shown Table S5 results following <br /> of nodes communicate. An integrity measure maps set                 values k tested networks: DIP CORE network <br /> nodes, S, value 0 1, value 0                   k = 3, LC HC networks k = 4, TAP-MS network <br /> assigned removal S completely disrupts                            k = 11, BAYESIAN network k = 4, Y2H <br /> communication value 1 assigned causes                    network k = 1. We sampled values k neighborhood of <br /> no disruption. Our measure, shortest-path integrity, quantifies             selected values results reported paper are <br /> the increase P <br />                     length shortest path removal             robust respect selected value k. We note <br />                          max&#240;C{dS &#240;s,t&#222;,0&#222;                                        approaches similar previously used <span id='am-54' about='obo:NCBITaxon_366134' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_215748 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_42261 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Palla" datatype="xsd:string"></span><span id='am-57'  ></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Palla</span> <br /> S given Ps,t=[S max&#240;C{d &#240;s,t&#222;,0&#222; , d(s,t) length of <br />                         s,t= <br />                            [S                                                     et al [38] Chesler et al. [39]. <br /> the shortest path s t original network, ds(s,t)                Once COBIMs computed, method selects subset of <br /> the length shortest path s t removal              COBIMs based distribution essential proteins the <br /> S, C constant. In work chose value C                COBIM nodes. Namely, heuristic selects COBIMs a <br /> twice diameter original network. Our second measure,                   fraction essential proteins significantly higher what <br /> edge-disjoint paths integrity, quantifiesPthe decrease number              expected uniform distribution essential genes <br />                                            [S fS &#240;s,t&#222;                            COBIM nodes. More specifically, COBIM n <br /> of edge-disjoint paths given P f &#240;s,t&#222; , fs(s,t) the <br />                                         s,t= <br />  <br />                                              s,t= <br />                                                 [S                                nodes m essential nodes selected iff: <br /> number edge-disjoint paths s t modified <br /> network f(s,t) value original network.                                                                             <br />                                                                                                     Pn           M           N{M <br />                                                                                                        k~m <br /> Estimating Parameters Essential Protein                                                               k            n{k <br />                                                                                                                                         v0:5, <br /> Interactions Model                                                                                                   N <br />    To evaluate model tested networks used                                                        n <br /> strategies estimate model&#8217;s parameters: network simulation <br /> procedure, line fitting points (log(12PE), k) k#k0,                    N total number COBIM nodes M the <br /> weighted line fitting points (log(12PE), k) values k. (In           number essential COBIM nodes. <br /> weighted line fitting, contribution (log(12PE), k) error <br /> function weighted fraction nodes having degree k.) The               Supporting Information <br /> first strategies described He et al. [9]. They deem the <br />                                                                                   Table S1 Using network integrity measures evaluate effect <br /> agreement parameter values estimated using network <br />                                                                                   removal hubs equivalent number central <br /> simulation line fitting strategies strongest <br />                                                                                   nodes according centrality measures <br /> indications validity model. But tested networks <br /> the parameter values estimated using different strategies, shown               Found at: doi:10.1371/journal.pcbi.1000140.s001 (0.04 MB <br /> in Table S4, vary considerably.                                                   DOC) <br />                                                                                   Figure S1 Membership COBIMs. The overlap <br /> The Method Automatic Identification ECOBIMs                                COBIMs quantified showing fraction nodes <br />   Our method automatic extraction putative ECOBIMs                      members COBIMs. <br /> applied subnetworks induced proteins annotated                Found at: doi:10.1371/journal.pcbi.1000140.s002 (0.05 MB <br /> biological process GO term. In work used set 192                     DOC) <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                     14                          August 2008 | Volume 4 | Issue 8 | e1000140 <br />                                                                                                                                                     Essentiality Hub Proteins <br />  <br />  <br /> Table S2 ECOBIMs member genes. For tested                                     COBIMs (ECOBIMs) depends value the <br /> protein interaction network list automatically identified                              parameter k. For small values k COBIMs (ECOBIMs) <br /> ECOBIMs. For ECOBIM following information                                         output algorithm larger COBIMs (ECOBIMs) <br /> shown: corresponding GO biological process term/terms,                                    identified bigger values k contain more <br /> number essential genes, number genes, names                                  network nodes. Here exact dependency shown range of <br /> member genes.                                                                                 parameter values. For protein interaction network the <br /> Found at: doi:10.1371/journal.pcbi.1000140.s003 (0.05 MB XLS)                                 fraction network nodes members more <br /> Table S3 Enrichment ECOBIM non-ECOBIM COBIM                                            COBIMs (ECOBIMs) shown. For network selected a <br /> nodes GO subnetworks LC, HC, TAP-MS, BAYESIAN,                                     value k results approximately 25% network nodes <br /> and Y2H networks. For GO subnetwork contributed                                  members COBIMs; resulting fractions shown <br /> least ECOBIM fraction essential proteins                                 bold. <br /> subnetwork nodes, subnetwork ECOBIM nodes subnetwork                                      Found at: doi:10.1371/journal.pcbi.1000140.s006 (0.03 MB <br /> non-ECOBIM COBIM nodes shown.                                                              DOC) <br /> Found at: doi:10.1371/journal.pcbi.1000140.s004 (0.15 MB XLS) <br /> Table S4      The parameters essential protein interaction                             Acknowledgments <br /> model. We use strategies estimate parameters,                              The authors thank Eugene Koonin (NCBI) valuable comments the <br /> b, essential protein interaction model: network                                    manuscript. <br /> simulation described original paper (simulation), line <br /> fitting points described original paper (line fitting),                      Author Contributions <br /> and weighted line fitting points values k (weighted line <br />                                                                                               Conceived designed experiments: EZ TMP. Performed the <br /> fitting).                                                                                     experiments: EZ. Analyzed data: EZ DPO TMP. Wrote paper: EZ. <br /> Found at: doi:10.1371/journal.pcbi.1000140.s005 (0.03 MB                                      Contributed algorithm efficient computation edge-disjoint path <br /> DOC)                                                                                          integrity measure: JM. Contributed designing experiments and <br /> Table S5 The number COBIM ECOBIMs nodes                                           writing paper: DPO. Participated writing paper: TP. <br /> function parameter . The number nodes belong to <br />  <br /> References <br />  1. Jeong H, Mason SP, Baraba&#180;si AL, Oltvai ZN (2001) Lethality centrality                   examine two-hybrid interactions possible combinations yeast <br />     protein networks. Nature 411(6833): 41&#8211;42.                                                      proteins. Proc Natl Acad Sci U S A 97(3): 1143&#8211;1147. <br />  2. He X, Zhang J (2006) Why hubs tend essential protein networks?                19.   Uetz P, Giot L, Cagney G, Mansfield TA, Judson RS, et al. (2000) A <br />     PLoS Genet 2(6): e88. doi:10.1371/journal.pgen.0020088.                                         comprehensive analysis protein-protein interactions Saccharomyces <br />  3. Jeong H, Mason SP, Baraba&#180;si AL, Oltvai ZN (2001) Lethality centrality                   cerevisiae. Nature 403(6770): 623&#8211;627. <br />     protein networks. Nature 411(6833): 41&#8211;42.                                                20.   Gavin AC, Bo&#168;sche M, Krause R, Grandi P, Marzioch M, et al. (2002) <br />  4. Batada NN, Hurst LD, Tyers M (2006) Evolutionary physiological                              Functional organization yeast proteome systematic analysis protein <br />     importance hub proteins. PLoS Comput Biol 2(7): e88. doi:10.1371/                            complexes. Nature 415(6868): 141&#8211;147. <br />     journal.pcbi.0020088.                                                                     21.   Ho Y, Gruhler A, Heilbut A, Bader GD, Moore L, et al. (2002) Systematic <br />  5. Hahn MW, Kern AD (2005) Comparative genomics centrality essentiality                     identification protein complexes Saccharomyces cerevisiae mass <br />     eukaryotic protein-interaction networks. Mol Biol Evol 22(4): 803&#8211;806.                 spectrometry. Nature 415(6868): 180&#8211;183. <br />  6. Yu H, Greenbaum D, Xin Lu H, Zhu X, Gerstein M (2004) Genomic analysis                 22.   Gavin AC, Al <br /> </body></html>