<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Genetic Physiological Activation Osmosensitive <br /> Gene Expression Mimics Transcriptional Signatures of <br /> Pathogen Infection C. elegans <br /> Anne-Katrin Rohlfing1, Yana Miteva1, Sridhar Hannenhalli2, Todd Lamitina1* <br /> 1 Department Physiology, University Pennsylvania, Philadelphia, Pennsylvania, United States America, 2 Department Genetics, Penn Center Bioinformatics, <br /> University Pennsylvania, Philadelphia, Pennsylvania, United States America <br />  <br />  <br />  <br />      Abstract <br />      The soil-dwelling nematode C. elegans powerful comparative molecular analyses environmental stress <br />      response mechanisms. Infection worms bacterial fungal pathogens causes activation well-characterized <br />      innate immune transcriptional programs pathogen-exposed hypodermal intestinal tissues. However, the <br />      pathophysiological events drive transcriptional responses understood. Here, infection- <br />      activated transcriptional responses are, large part, recapitulated physiological genetic activation the <br />      osmotic stress response. Microarray profiling wild type worms exposed non-lethal hypertonicity identified suite of <br />      genes regulated infection. Expression profiles different osmotic stress resistant (osr) mutants under <br />      isotonic conditions reiterated wild type transcriptional response osmotic stress showed substantial similarity <br />      infection-induced gene expression isotonic conditions. Computational, transgenic, functional approaches <br />      revealed GATA transcription factors previously implicated infection-induced transcriptional responses, elt-2 and <br />      elt-3, essential coordinated tissue-specific activation osmosensitive gene expression promote survival <br />      osmotically stressful conditions. Together, <span id='am-97' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-98'  ></span><span id='am-99' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> suggest infection osmotic adaptation share previously <br />      unappreciated transcriptional similarities controlled regulation tissue-specific GATA transcription <br />      factors. <br />  <br />   Citation: Rohlfing A-K, Miteva Y, Hannenhalli S, Lamitina T (2010) Genetic Physiological Activation Osmosensitive Gene Expression Mimics Transcriptional <br />   Signatures Pathogen Infection C. elegans. PLoS ONE 5(2): e9010. doi:10.1371/journal.pone.0009010 <br />   Editor: Matt Kaeberlein, University Washington, United States America <br />   Received December 2, 2009; Accepted January 13, 2010; Published February 2, 2010 <br />   Copyright: &#223; 2010 Rohlfing et al. <span id='am-19' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-20' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23'  ></span>This</span> open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: 1R01AA017580 T.L. R01GM085226 S.H. P30 ES013508 University Pennsylvania Center Excellence Environmental Toxicology (T.L. <br />   member). Some nematode strains used work provided <span id='am-11' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-13'  ></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span>Caenorhabditis</span> Genetics Center, funded National Institutes Health <br />   (NIH) National Center Research Resources (NCRR). The funders role study design, data collection analysis, decision publish, preparation of <br />   manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: lamitina@mail.med.upenn.edu <br />  <br />  <br />  <br /> Introduction                                                                           activated infection required proper infection inducible <br />                                                                                        gene expression, toxin defenses, normal lifespan [8,9,10]. <br />    The innate immune evolutionarily ancient response                      While conserved signaling pathway plays critical role in <br /> to pathogen infection [1,2]. The detection pathogens provokes                       immune responses, upstream sensory mechanisms activate <br /> the upregulation numerous antimicrobial peptides                                p38 MAP kinase signaling response infection poorly <br /> cytokines, activation specific adaptive                         understood. <br /> immune mechanisms, diverse signaling pathways [3]. Molec-                             In contrast clear roles p38 MAP kinase signaling <br /> ular mechanisms innate immunity appear highly                                 pathways regulation innate immune responses C. <br /> conserved, mammals, flies, worms utilize similar                         elegans, transcription factors functioning downstream p38 <br /> signaling pathways activate innate immune responses.                                MAPKs execute infection regulated gene expression programs <br />    The nematode C. elegans proven powerful model                           defined. Recent work [11,12,13] suggested that <br /> system genetic genomic analysis innate immunity [4]. C.                     C. elegans GATA-type transcription factors, elt-2 elt-3, may <br /> elegans infected intestine hypodermis                      promote pathogen resistance pathogen-inducible gene <br /> by bacterial fungal pathogens, including human                         expression intestine hypodermis, respectively. <br /> opportunistic pathogen <span id='am-32' about='obo:NCBITaxon_286' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621'><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Liquidomonas" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="PMID:18048745" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="PMID:10939664" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Loefflerella" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-38'  ></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="PMID:9103607" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="PMID:15950132" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="PMID:23918787" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Peudomonas" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="PMID:10758879" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Pseudomonas" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="PMID:7727274" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="RNA similarity group I" datatype="xsd:string"></span>Pseudomonas</span> aeruginosa [5]. Unlike infection                    [11,12,13]. GATA type transcription factors zinc finger <br /> in humans, infection killing C. elegans bacterial                       containing proteins bind highly conserved DNA sequence <br /> pathogens does involve cell invasion, replication, lysis [6].                  motif (&#8216;WGATAR&#8217;). GATA factors conserved yeast to <br /> Several studies revealed prominent role bacterial                           humans play crucial roles numerous developmental and <br /> produced toxins nematode killing [7,8], patho-                         physiological contexts. [14]. The specific role(s) GATA factors <br /> physiological mechanisms killing C. elegans                   immunity evolutionarily conserved GATAs in <br /> clear. Nonetheless, genetic screens led identification                     Drosophila humans play functionally important roles <br /> of conserved ASK/p38 MAP kinase signaling pathway                            pathogen resistance [12,15,16]. In Drosophila, GATAs function <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                 1                               February 2010 | Volume 5 | Issue 2 | e9010 <br />                                                                                                                        Stress Immunity Worms <br />  <br />  <br /> together rel-type transcription factors, Dif, Dorsal,         independently derived biological samples. In cases, qPCR <br /> Relish promote immune responsive gene expression [15]. Rel                  data correlated microarray results (Figure S1). Since <br /> <span id='am-1' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-2' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> transcription factors mediate responses wide                  osmotic stress causes rapid significant shrinkage cells to <br /> variety cellular stressors [17]. Interestingly, C. elegans genome       osmotically induced water loss [19], considered possibility <br /> does express known homologs rel-type transcription factors,             ORGs represent secondary response to <br /> suggesting GATA factors function key integrators                osmotically-induced tissue/cellular damage. To separate damage <br /> cellular stressors immune responses. Currently, unclear              responsive genes osmotic response genes, compared the <br /> whether C. elegans GATA factors involved response                ORG list genes regulated heavy metal cadmium, which <br /> specific pathogen signals general pathogen-evoked                   used previously control transcriptional effects <br /> disruptions host cell homeostatic processes.                                tissue damage [8,12]. Similar previous studies [12], <br />    In work, osmotic stress, natural environmental          ORGs exhibited significant overlap genes regulated Cd2+ <br /> stress C. elegans, mimics previously described transcriptional             (78/324 genes, 3.1x enrichment, p,0.001). However, 78 <br /> responses bacterial fungal infection. Transcriptional profiling         genes, 42.3% (33/78) showed concordance direction of <br /> of different osmotic stress resistant (osr) mutants reiterated        transcriptional regulation. Furthermore, makeup Cd2+ <br /> wild type response osmotic stress exhibited substantial          osmotic stress responses functionally distinct. For <br /> overlap numerous infection models, suggesting overlap            example, Cd2+ induced expression 14 heat shock proteins, <br /> is genetically regulated. Using bioinformatics, transgenic promoter            osmotic stress failed induce HSPs repressed <br /> analysis, loss-of-function studies, GATA                 expression HSPs. In addition differences <br /> factors, elt-2 elt-3, required mediate osmosensitive gene           ORGs Cd2+ responsive genes, ORGs did exhibit <br /> expression intestine hypodermis, respectively. We              significant overlap transcriptional responses ER <br /> find simultaneous inhibition elt-2 elt-3 suppresses           stress hypoxia, additional stressors thought to <br /> physiological genetic resistance animals osmotic stress.             cause tissue damage [20,21]. (Figure S2). Together, data <br /> These data add understanding mechanisms infection             majority transcriptional response osmotic <br /> by suggesting osmotic dyshomeostasis represent                      stress largely distinct composition function from <br /> significant pathophysiological consequence infection C. elegans.         damage-inducing environmental stressors. <br /> Furthermore, findings implicate GATA factors key <br /> post-developmental integrators diverse stress responses C.               OSR Mutants Activate ORG Transcriptional Program <br /> elegans. Given highly conserved links innate immunity              Absence Osmotic Stress <br /> and osmoregulation mammals, understanding molecular                        To determine activation ORGs was <br /> mechanisms osmosensing osmosensitive gene expression                 responsive tissue damage genetically regulated activation <br /> C. elegans shed new light mechanisms pathogen-                  osmosensitive signaling pathways, performed microarray <br /> induced cellular toxicity.                                                     analysis osmotic stress resistant (osr) mutants grown under <br />                                                                                isotonic conditions. osr mutants exhibit physiological <br /> Results                                                                        characteristic wild type animals adapted hyperosmotic stress, <br />                                                                                high gpdh-1 expression, glycerol accumulation, and <br /> Microarray Analysis Osmotic Stress Response                          resistance normally lethal levels hypertonicity [18,22]. <br /> C. elegans                                                                     However, unlike wild type animals, osr mutants mis-express these <br />    As soil-dwelling organism, C. elegans exposed unstable              phenotypes isotonic conditions, suggesting osr mutants <br /> osmotic environments adapt environments                   encode genes negative regulators osmosensitive <br /> survive. Previously, showed upregulation glycerol                 signaling pathways loss function mutations osr <br /> biosynthetic enzyme resulted osmotically induced glycerol                   genes cause constitutive activation pathways. The osr <br /> accumulation, required proper adaptation                       mutants osm-7 osm-11 encode homologous secreted proteins <br /> hyperosmotic environments [18]. To better understand                 function Notch ligands [22,23], dpy-9 dpy-10 <br /> adaptive responses transcriptionally specified define           encode extracellular matrix collagens, osm-8 encodes a <br /> physiological processes osmotic stress responses             secreted mucin-like protein (Rohlfing Lamitina, prepara- <br /> involved, carried time-course microarray analysis                  tion). osr mutants exhibit obvious signs cellular tissue <br /> synchronized young adult worms transferred isotonic                       damage isotonic conditions [22] (our unpublished observa- <br /> (50 mM NaCl Nematode Growth Media(NGM); 170 mOsm)                           tions). Therefore, genes transcriptionally regulated osr <br /> non-lethal hypertonic (200 mM NaCl NGM; ,500 mOsm)                             mutants likely responsive osmosensitive signaling pathways <br /> growth conditions. Labeled cDNA hybridized Affymetrix                   secondarily responsive tissue damage. <br /> Genechips containing probes 22,500 transcripts. To                        Using stringent statistical criteria (see &#8216;Methods&#8217;), that <br /> capture acute chronic responses hypertonicity,                  genes regulated osr mutants largely overlapping with <br /> measured gene expression changes 15 minutes, 1 hour,                     wild type ORGs (Figure 1B, Tables S2, S3, <br /> 6 hours, 1 generation growth (,96 hours) 200 mM                 S4, S5, S6). Activation repression gene expression tended to <br /> NaCl NGM plates. Our previous studies shown                    robust osr mutants compared wild type <br /> levels osmotic stress result activation osmoregulatory                animals exposed 200 mM NaCl (Figure 1A), suggesting osr <br /> adaptive responses, accumulation organic                    mutants cause strong constitutive osmoresponsive gene <br /> osmolyte glycerol upregulation glycerol biosynthetic                expression. Of genes physiologically regulated osmotic <br /> enzyme gpdh-1, causing significant lethality [18,19]                   stress wild type animals, 63.5% (206/324) regulated <br />    Using stringent statistical criteria (see &#8216;Methods&#8217;), identified         osr mutants isotonic conditions (Figure 1B), which <br /> 324 genes expression altered $3-fold                  highly significant enrichment (p,0.001). The transcriptional <br /> timepoint (Figure 1A, Table S1) refer Osmotically              profiles osr mutants fell distinct patterns, with <br /> Regulated Genes (ORGs). We validated microarray data                    Notch ligands mutants osm-7 osm-11 mucin protein <br /> performing real-time quantitative PCR subset ORGs                 mutant osm-8 exhibiting high degree similarity wild type <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          2                           February 2010 | Volume 5 | Issue 2 | e9010 <br />                                                                                                                      Stress Immunity Worms <br />  <br />  <br />  <br />  <br /> Figure 1. Expression profiling hypertonic stress response C. elegans. (A) Hierarchical clustered heat map gene expression <br /> changes associated physiological activation osmosensitive gene expression (200 mM NaCl indicated time interval) regulation <br /> of genes various osr mutants normal conditions (50 mM NaCl). For filtering statistical criteria, &#8216;Methods&#8217;. (B) Venn diagram <br /> showing overlap genes differentially regulated osmotic stress, osm mutants, dpy mutants. <br /> doi:10.1371/journal.pone.0009010.g001 <br />  <br /> ORG dataset collagen dpy-9, dpy-10 regulating                 osmotic stress osr mutants (Figure 2C). To test whether <br /> partially overlapping larger set transcripts           ORGs overlapped broadly infection regulated genes, we <br /> less similar wild type ORG dataset (Figure S3). Genes                compiled list genes known regulated bacterial or <br /> specifically regulated osm-7 osm-11 mutants             fungal infection C. elegans compared list ORG <br /> enriched known Notch signaling targets [24], suggesting            dataset [10,11,12,25,26]. We 52.4% ORGs (170/324) <br /> these proteins regulatory roles addition         differentially regulated model infection, which <br /> roles Notch signaling. osr mutant transcriptional profiles did       represented significant enrichment expected two <br /> exhibit significant overlap transcriptional responses heat          randomly distributed datasets (1.4 fold enrichment, p,0.001; <br /> shock, hypoxia, ER stress, demonstrating osr mutant         Figure 2A). ORGs exhibited significant overlap infection <br /> activation osmosensitive gene expression highly specific (data        regulated genes 6/8 datasets representing exposure seven <br /> not shown). Taken together, data suggest osr mutants         different pathogen models (Table S7). Among pathogen datasets, <br /> genetically activate transcriptional program                 ORGs exhibited significant enrichment genes regulated <br /> physiologically activated hyperosmotic stress wild type               infection gram positive pathogen M. nematophilium, which <br /> C. elegans.                                                                 induces protective inflammatory cell swelling response post- <br />                                                                             rectal epithelial cells [27]. ORGs exhibited highly significant <br /> OSR Mutants Osmotic Stress Mimic Innate Immune                          enrichment genes regulated P. aeuroginosa, mechanism of <br /> Transcriptional Responses                                                   killing strongly influenced environmental osmolarity [7]. While <br />    Among genes regulated osmotic stress osr             expression ORGs infection regulated genes, as <br /> mutants, noticed enrichment genes gene classes        nlp, cnc, clec genes, exhibited concordance regulation <br /> previously associated microbial infection C. elegans. These         osmotic stress infection conditions, genes, as <br /> included neuropeptide like proteins (nlp), caenicin (cnc), c-type       organic osmolyte accumulation enzyme gpdh-1, encodes a <br /> lectin (clec) gene classes (Figure 2B). These genes thought          glycerol-3-phosphate dehydrogenase, strongly upregulated by <br /> function antimicrobial peptides C. elegans induced       osmotic stress downregulated models infection. These <br /> by bacterial fungal infection [11,25]. We               data genes transcriptionally regulated osmotic <br /> virtually immune effectors strongly upregulated        stress infection highly similar respects. However, <br />  <br />  <br />        PLoS ONE | www.plosone.org                                       3                           February 2010 | Volume 5 | Issue 2 | e9010 <br />                                                                                                                             Stress Immunity Worms <br />  <br />  <br />                                                                                   C. elegans discriminate osmotic stress infection, evidenced <br />                                                                                   specific upregulation gpdh-1 hyperosmotic stress. <br />  <br />                                                                                   The Promoters ORGs Are Enriched GATA <br />                                                                                   Transcription Factor Binding Motifs <br />                                                                                      We used single gene RNA interference test vivo functional <br />                                                                                   requirements 280/324 ORGs failed observed significant <br />                                                                                   affects survival, growth, reproductive capacity osmotically <br />                                                                                   challenged wild type animals various sensitized genetic <br />                                                                                   backgrounds (rrf-3, gpdh-1, gpdh-2, gpdh-1;gpdh-2, osm-11; data <br />                                                                                   shown). This suggested that, like complex physiological <br />                                                                                   processes ageing [28], transcriptional responses of <br />                                                                                   individual genes play subtle roles response hyperosmotic <br />                                                                                   stress. Given results, hypothesized regulatory genes, <br />                                                                                   transcription factors, exhibit robust phenotypes <br />                                                                                   regards activation osmosensitive gene expression and <br />                                                                                   vivo osmotic stress resistance. To identify transcription factors that <br />                                                                                   regulate osmotic stress responses vivo, performed an <br />                                                                                   unbiased search promoters (1Kb) upregulated ORGs <br />                                                                                   C. briggsae orthologs transcription factor binding sites <br />                                                                                   using binding motifs described TRANSFAC motif <br />                                                                                   database (www.gene-regulation.com; &#8216;Methods&#8217;). Using this <br />                                                                                   approach, identified transcription factor binding motifs <br />                                                                                   enriched ORG dataset (Figure 3A). The broadest <br />                                                                                   statistically significant motif enrichment the <br />                                                                                   GATA type transcription factor binding motif (Figure 3A). The <br />                                                                                   consensus GATA motif promoters ORGs highly <br />                                                                                   similar previously published motif C. elegans elt-2 (Figure 3B; <br />                                                                                   &#8216;ACTGATAAGG&#8217;) [29]. Within promoter strongly <br />                                                                                   regulated ORG gpdh-1, GATA sites located ,250 bp <br />                                                                                   upstream ATG start <span id='am-24' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-28'  ></span>codon</span>. This region shows high sequence <br />                                                                                   conservation multiple nematode <span id='am-29' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-30' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> (Figure 3C), <br />                                                                                   suggesting GATA sites and/or additional motifs play <br />                                                                                   functionally important roles activation osmosensitive gpdh-1 <br />                                                                                   expression. <br />  <br />                                                                                   GATA Sites Are Required Physiological Genetic <br />                                                                                   Regulation gpdh-1 Expression Intestine and <br />                                                                                   Hypodermis <br />                                                                                      To determine GATA binding sequences required <br />                                                                                   genetic physiological activation gpdh-1 expression, we <br />                                                                                   constructed series gpdh-1 promoter deletions joined the <br />                                                                                   coding sequence GFP analyzed ability mediate <br />                                                                                   osmosensitive GFP expression live transgenic animals. Con- <br />                                                                                   structs containing 251 bp base pairs promoter sequence, which <br />                                                                                   contain conserved GATA binding sites, sufficient to <br />                                                                                   mediate osmosensitive GFP expression (Figure S4). However, <br />                                                                                   constructs containing 114 bp promoter sequence, did not <br />                                                                                   include consensus GATA sites, failed induce GFP exposed <br />                                                                                   hypertonic conditions, suggesting sequences the <br />                                                                                   2114 2250 region required activation osmosensitive <br />                                                                                   gpdh-1 expression. Since region contains predicted <br />                                                                                   GATA binding sites, hypothesized sites were <br />                                                                                   required <span id='am-76' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-77' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80'  ></span>cis</span> mediate osmosensitive gpdh-1 expression. To test <br /> Figure 2. Osmotically regulated genes exhibit significant <br /> overlap pathogen regulated genes. (A) Venn diagram showing                   hypothesis, utilized site directed mutagenesis convert the <br /> the genes common osmotic stress compilation               consensus &#8216;GATA&#8217; sequence &#8216;CAAA&#8217; context full <br /> pathogen microarray data. Pathogen data compiled existing               length (3.0 Kb) gpdh-1 promoter predicted GATA <br /> microarray datasets [10,12,25,26] obtain 5095 unique genes           binding sites (Figure 4A). Similar alterations shown to <br /> significantly regulated model infection. (B) Enrichment        inhibit GATA-DNA interactions block transcriptional trans- <br /> of indicated gene <span id='am-16' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> genes regulated osmotic stress in <br />                                                                                   activation [30]. Transgenic animals expressing wild type or <br /> wild type isotonic conditions osm dpy mutants. (C) <br /> Hierarchical clustering antimicrobial cnc, nlp, clec gene expression       mutated GATA constructs analyzed ability to <br /> in osmotically stressed wild type unstressed osm dpy mutants.              respond physiological activation high salt (Figure 4B,C) <br /> doi:10.1371/journal.pone.0009010.g002                                             genetic activation osr mutants (Figure 4D,E). The wild type <br />  <br />  <br />        PLoS ONE | www.plosone.org                                             4                            February 2010 | Volume 5 | Issue 2 | e9010 <br />                                                                                                                       Stress Immunity Worms <br />  <br />  <br />                                                                                activation gpdh-1p::GFP expression intestine partially <br />                                                                                required genetic activation gpdh-1p::GFP hypodermis. <br />  <br />                                                                                Tissue-Specific GATA Factors elt-2 elt-3 Function <br />                                                                                Together Mediate In Vivo Osmosensitive Gene <br />                                                                                Expression Whole Animal Osmotic Stress Resistance <br />                                                                                    The data suggested GATA factors may <br />                                                                                play functionally important roles regulation osmosensitive <br />                                                                                gene expression vivo osmotic stress resistance C. elegans. <br />                                                                                Previous studies demonstrated GATA factor elt-3 is <br />                                                                                necessary osmotically induced gene expression the <br />                                                                                hypodermis [11] confirmed result (Figure 5). To <br />                                                                                determine additional GATA factors regulate intestinal <br />                                                                                gpdh-1 expression, used RNAi post-embryonically inhibit the <br />                                                                                expression 10/14 C. elegans GATA factors animals expressing <br />                                                                                gpdh-1p::GFP. We RNAi GATA factor, elt-2, <br />                                                                                attenuated osmotically induced GFP expression intestine <br />                                                                                substantially affecting hypodermal GFP expression <br />                                                                                (Figure 5). elt-2(RNAi) suppressed genetic activation of <br />                                                                                intestinal, hypodermal, gpdh-1p::GFP expression osr <br />                                                                                mutant osm-7 (Figure 5). The effect elt-2(RNAi) osmotic stress <br />                                                                                responses simply general defect intestinal gene <br />                                                                                expression, elt-2(RNAi) exhibited normal expression an <br />                                                                                intestinal ER-stress reporter (hsp-4p::GFP) (data shown). <br />                                                                                    Previous studies shown elt-3 elt-2 play <br />                                                                                functionally important roles innate immune responses to <br />                                                                                fungal bacterial pathogens [11,12,13]. Since osmotic <br />                                                                                stress induces robust transcriptional responses the <br />                                                                                hypodermis intestine, hypothesized hypodermal elt-3 <br />                                                                                intestinal elt-2 play additive roles mediating vivo <br />                                                                                osmotic stress resistance. To test hypothesis, measured the <br /> Figure 3. GATA transcription factor binding sites over-                    survival elt-2(RNAi), elt-3(gk121), double mutant elt-2(RNAi); <br /> represented upregulated ORGs. (A) TRANSFAC-annotated                     elt-3(gk121) animals following exposure hyperosmotic stress. <br /> transcription factor DNA binding motifs enrichment scores .1.5            Each single elt mutants survived hyperosmotic challenge as <br /> and p-values,0.05 C. elegans C. briggsae ORGs. The number          wild type animals (Figure 6A). However, elt-2; elt-3 <br /> of ORGs motif plotted. For details enrichment analysis, <br />                                                                                double mutant showed greater sensitivity hyperosmotic stress <br /> see &#8216;Methods&#8217;. Only binding motifs p-values ,0.05 C. <br /> elegans C. briggsae datasets shown. GATA - GATA binding                single mutants (Figure 6A). We tested <br /> protein 2; TATA - TATA sequence binding protein; USF - Upstream                elt-2 and/or elt-3 required genetically endowed <br /> stimulatory factor (USF1 USF2); Clock:BMAL - brain-muscle-ARNT-            osmotic stress resistance using osr mutant osm-8. The chronic <br /> like protein 2 (ARNT-2); mtTFA - mitochondrial transcription factor A;         osmotic stress resistance phenotype osm-8(n1518) mutants was <br /> HLF - hepatic leukemia factor; N-Myc - neuroblastoma MYC oncogene;             significantly suppressed elt-2(RNAi) elt-3(gk121) <br /> c-Myc - c-myc proto-oncogene; Evi-1 - Ecotropic viral integration site 1 <br /> transcription factor. (B) Consensus GATA2 motif 181 occurrences <br />                                                                                backgrounds (Figure 6B). However, osm-8(n1604); elt-2(RNAi); elt- <br /> in C. elegans ORG dataset. (C) Analysis gpdh-1 promoter        3(gk121) triple mutants exhibited significant reduction chronic <br /> C. elegans, C. briggsae, C. remanei. Regions containing .85%               osmotic stress resistance (Figure 6B). The phenotype triple <br /> identify promoters indicated black shading.             mutants defects general health the <br /> doi:10.1371/journal.pone.0009010.g003                                          animals, survival isotonic conditions indistin- <br />                                                                                guishable control animals (data shown). These data show <br /> gpdh-1p::GFP expression robustly induced hypodermis                 GATA factors elt-2 elt-3 play overlapping and/or <br /> and intestine wild type animals following brief (4 h 8 h)                redundant roles physiological osm-8-induced genetic <br /> exposure 200 mM NaCl (Figure 4B,C). RNAi inactivation                    regulation vivo osmotic stress resistance C. elegans. <br /> both osm-7 osm-11, negatively regulate gpdh-1 expression <br /> [18], resulted robust GFP expression intestine and <br />                                                                                Discussion <br /> hypodermis isotonic conditions (Figure 4D,E). In contrast, <br /> all DGATA-gpdh-1p::GFP transgenic lines failed express                         Previous studies infection C. elegans noted that <br /> significant levels GFP 8 hours exposure                    transcriptional responses infection appear highly specific and <br /> 200 mM NaCl (Figure 4B,C). Genetic activation DGATA-gpdh-                   exhibit little significant overlap forms well-studied <br /> 1p::GFP inhibition osm-7 osm-11 strongly                    environmental stress, heat shock ER stress [12,13]. <br /> attenuated. Interestingly, response genetic activation,                  However, infection diverse types pathogens gives rise to <br /> intestinal expression completely eliminated DGATA-gpdh-                 common transcriptional signatures [26], etiology is <br /> 1p::GFP, hypodermal expression frequently persisted                clear. Here, activation osmotic stress <br /> mutants (Figure 4D,E), suggesting additional factors likely           response C. elegans, physiological genetic <br /> to participate regulation gpdh-1 expression                   mechanisms, mimic common transcriptional responses a <br /> hypodermis. These data demonstrate GATA motifs                        diverse array infection models C. elegans. The mechanism(s) by <br /> absolutely required cis robust physiological genetic                activation osmotic stress responses mimic infection is <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          5                          February 2010 | Volume 5 | Issue 2 | e9010 <br />                                                                                                                      Stress Immunity Worms <br />  <br />  <br />  <br />  <br /> Figure 4. GATA binding sites required cis physiological genetic activation gpdh-1 expression. (A) Schematic diagram <br /> of gpdh-1 promoter constructs, location GATA binding sites mutagenized sequences indicated arrows. (B) Animals <br /> expressing wild type (top) DGATA gpdh-1p::GFP transgene (bottom) exposed 200 mM NaCl 4 8 hours being <br /> imaged (B) quantified using COPAS Biosort (C). For COPAS data, n.100 young adult animals independent wild type lines six <br /> independent DGATA lines. (D) Animals carrying wild type (top) DGATA gpdh-1p::GFP transgene (bottom) fed osm-7(RNAi) or <br /> osm-11(RNAi) generations. The RNAi treatment highly effective lines, 100% osm-7(RNAi) osm-11(RNAi) animals exhibited an <br /> osmotic stress resistance (osr) phenotype. (E) The induction GFP following EV, osm-7, osm-11 RNAi measured COPAS Biosort; n.100 <br /> for lines. Horizontal lines indicate data independent transgenic lines. <br /> doi:10.1371/journal.pone.0009010.g004 <br />  <br />  <br /> currently unclear intriguing mechanism relate                 elegans reduced following infection toxin-compromised <br /> infection-induced disruptions osmotic homeostasis. For exam-              pathogens. <br /> ple, pathogens, production pore-forming toxins                Functional links infection osmotic stress also <br /> and Type III secretion systems, create ion water permeable               observed systems. For example, mammals, T-cell <br /> pores plasma membrane [31]. Due standing                       activation <span id='am-81' about='obo:NCBITaxon_49990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_216706 obo:NCBITaxon_216718 obo:NCBITaxon_3398 obo:NCBITaxon_4136 obo:NCBITaxon_4143 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-82' property="rdfs:label" content="Thymus" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-85'  ></span>thymus</span> occurs hypertonic environments, the <br /> electrochemical ion gradients present cells, insertion         maintenance requires rel-type transcription factor <br /> these relatively non-selective pores allow gradients            NFAT5/TonEBP [34]. NFAT5/TonEBP essential <br /> collapse, producing osmotic dyshomeostasis [32,33]. In                  mediator hypertonicity-induced transcriptional responses in <br /> regard, osmotic stress mimic early pathophysiological                    diverse cell types, including extremely hypertonic environment <br /> responses infection. Consistent hypothesis,            inner medulla mammalian kidney [35,36]. In <br /> also substantial overlap ORGs identified               Drosophila, salt stress induces broad transcriptional response that <br /> study genes regulated exposure C. elegans purified            includes induction numerous infection regulated genes [37]. <br /> pore-forming toxin [8]. In future, interesting test       Along present studies C. elegans, data suggest the <br /> whether common transcriptional signatures infection C.                 intriguing hypothesis cellular osmotic stress an <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        6                          February 2010 | Volume 5 | Issue 2 | e9010 <br />                                                                                                                          Stress Immunity Worms <br />  <br />  <br />  <br />  <br /> Figure 5. The GATA factors elt-2 elt-3 mediate cell-type specific activation gpdh-1 expression following physiological or <br /> genetic activation. (A) gpdh-1p::GFP animals exposed indicated RNAi treatments exposed 200 mM NaCl 24 hours GFP was <br /> imaged. Scale bar = 75 m. (B) osm-7(n1515); gpdh-1p::GFP animals exposed indicated RNAi treatments isotonic conditions GFP <br /> was imaged. Scale bar = 75 m. Open arrows point intestine closed arrowheads point hypodermis. <br /> doi:10.1371/journal.pone.0009010.g005 <br />  <br /> evolutionarily conserved physiologically significant activator              role elt-2 studies developmentally required <br /> of immune responses. Interestingly, C. elegans genome does              transcription factor, skn-1, required oxidative stress <br /> contain rel-type transcription factors similar NFAT5/TonEBP,                 responses C. elegans intestine. Multiple signaling pathways <br /> suggesting functional links osmotic stress                 converge skn-1, integrates signals coordinate <br /> immunity preceded evolution family                      appropriate responses ageing, insulin signaling, caloric restriction, <br /> transcription factors. C. elegans provide outstanding                 oxidative stressors [43,44,45,46,47]. Further studies be <br /> model better explore functional, molecular,              required determine elt-2, like skn-1, exhibits direct <br /> and evolutionary relationships immunity osmotic                     regulation localization, DNA binding, and/or activity response <br /> stress.                                                                         osmotic stress and/or infection. <br />    In animals, molecular mechanisms regulating osmosensing                 In conclusion, findings osmotic stress and <br /> osmosensitive gene expression understood. As mentioned             pathogen infection regulate similar transcriptional targets C. <br /> above, rel factor NFAT5 essential osmosensitive gene                 elegans. Tissue-specific GATA-type transcription factors elt-2 elt- <br /> expression mammals, C. elegans does express homolog             3 play major role coordinating gene expression <br /> this rel factor. Therefore, transcription factor(s)            programs required response osmotic stress. Our <br /> controlling osmosensitive gene expression C. elegans remained           studies functionally link cellular osmotic stress responses innate <br /> mysterious. Our data reveal important role GATA-type                 immunity suggest osmotic dyshomeostasis represent <br /> transcription factors, elt-2 elt-3, process. Both GATA              new evolutionarily significant paradigm physiological <br /> factors required developmental non-                      basis host cell pathogen sensing. <br /> developmental processes [11,12,13,38,39,40,41,42], suggesting that <br /> additional factors likely coordinate elt-2 elt-3               Materials Methods <br /> determine specific responses. For example, elt-2 required <br /> for osmosensitive infection-inducible gene expression,             C. elegans Strains Transgenics <br /> sek-1/pmk-1 p38 MAP kinase signaling pathway required                  All strains maintained using standard culture methods and <br /> pathogen-induced transcriptional responses osmosensitive                fed E. coli strain OP50. NGM media hypertonic <br /> gene expression [11] (our unpublished observations). Therefore,                 addition NaCl indicated concentration. The <br /> multiple upstream signaling pathways converge GATA                       following strains used: LGI: kbIs5(gpdh-1p::GFP; rol-6); LGII: <br /> factors determine specific transcriptional outputs. Such               dpy-10(e128), osm-8(n1518); LGIII: osm-7(n1515); LGIV: dpy- <br /> convergence lead enrichment GATA factors                        9(e12); LGX: osm-11(n1604), elt-3(gk121). Unless noted, <br /> signaling-appropriate promoters, possibly interactions             strains grown 20uC. The wild type N2 strain was <br /> specific transcriptional cofactors signaling components. Such          obtained <span id='am-4' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-5'  ></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Caenorhabditis elegans</span> Genetic Stock Center. <br /> a mechanism explain inducible activation GATA                         gpdh-1::GFP reporter constructs created using PCR site <br /> factors regulate restrictive set transcriptional targets.       directed mutagenesis construct described [18]. Site <br /> It intriguing note parallels findings         directed mutagenesis carried using QuickChange <br />  <br />  <br />        PLoS ONE | www.plosone.org                                           7                           February 2010 | Volume 5 | Issue 2 | e9010 <br />                                                                                                                          Stress Immunity Worms <br />  <br />  <br />                                                                                  placed NGM plates containing 200 mM NaCl 15 <br />                                                                                  minutes, 1 hour, 6 hours. For steady state samples, <br />                                                                                  synchronized L1 stage animals placed NGM plates <br />                                                                                  containing 200 mM NaCl allowed grow young adult <br />                                                                                  stage. For samples, 1000 adult worms collected into <br />                                                                                  1.5 ml tubes using gating criteria COPAS Biosort that <br />                                                                                  excluded L4 younger animals. Immediately following sorting, <br />                                                                                  worms pelleted 2,000 RPM supernatant was <br />                                                                                  aspirated, leaving ,100 ml worm pellet. 400 ml of <br />                                                                                  Trizol added solution vortexed minutes. <br />                                                                                  Worms stored -80u RNA isolation. For each <br />                                                                                  timepoint, individual replicates performed. The best <br />                                                                                  preparations (as determined Agilent Bioanalyzer analysis <br />                                                                                  spectrophotometric readings) used RNA labeling and <br />                                                                                  hybridization. The remaining preparations used for <br />                                                                                  qPCR validation microarray results. <br />  <br />                                                                                  RNA Isolation Microarray Analysis <br />                                                                                      Total RNA extracted using combined Trizol (Invitrogen)/ <br />                                                                                  RNAeasy (Qiagen, <span id='am-86' about='obo:NCBITaxon_195854' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_8087 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_28738 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_7898 obo:NCBITaxon_195853 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="Valencia" datatype="xsd:string"></span><span id='am-90'  ></span>Valencia</span>, CA) column method. Purified total <br />                                                                                  RNA submitted University Pennsylvania Microarray <br />                                                                                  Core facility RNA quality control, labeling, hybridization <br />                                                                                  Affymetrix C. elegans GeneChips. All protocols conducted <br />                                                                                  described Affymetrix GeneChip Expression Analysis <br />                                                                                  Technical Manual. Briefly, 2 mg total RNA converted to <br />                                                                                  first-strand cDNA using Superscript II reverse transcriptase <br />                                                                                  primed poly(T) oligomer incorporated T7 promoter. <br />                                                                                  Second-strand cDNA synthesis followed vitro transcrip- <br />                                                                                  tion linear amplification transcript incorporation of <br />                                                                                  biotinylated CTP UTP. The cRNA products fragmented <br />                                                                                  200 nucleotides less, heated 99uC 5 min and <br />                                                                                  hybridized 16 h 45uC. The microarrays washed <br />                                                                                  low (6X SSPE) high (100 mM MES, 0.1M NaCl) stringency <br />                                                                                  stained streptavidin-phycoerythrin. Fluorescence was <br /> Figure 6. The GATA factors elt-2 elt-3 required to <br />                                                                                  amplified adding biotinylated anti-streptavidin an <br /> mediate physiological genetic resistance hyperosmotic <br /> stress vivo. (A) Wild type elt-3(gk121) animals exposed            additional aliquot streptavidin-phycoerythrin stain. A confocal <br /> either control (EV) elt-2(RNAi) treatment described &#8216;Methods&#8217;.          scanner used collect fluorescence signal 3 um resolution <br /> Young adult animals transferred 500 mM NaCl plates                   excitation 570 nm. The average signal sequential <br /> survival measured 24 hour period. Animals died                scans calculated microarray feature. <br /> bagging crawling edge plate censored time               Affymetrix .cel files arrays uploaded Partek <br /> of event. N.50 animals genotype. (B) osm-8(n1518) osm- <br /> 8(n1518); elt-3(gk121) animals exposed control (EV) elt- <br />                                                                                  Genomics Suite intensity values normalized using the <br /> 2(RNAi) treatment described &#8216;Methods&#8217;. Young adult animals            GC-RMA algorithm. Using Affymetrix present/absent calls, <br /> transferred 500 mM NaCl survival measured 24 hour                13,866/22,625 probes &#8216;present&#8217; 3/30 datasets, <br /> period. Animals died bagging crawling edge            probes used subsequent statistical analysis. <br /> plate censored time event. N.40 animals                   Data wild type series analyzed using the <br /> genotype. * - p,0.05.                                                            Significance analysis Microarrays (SAM) algorithm (to calculate <br /> doi:10.1371/journal.pone.0009010.g006 <br />                                                                                  False Discovery Rate (FDR)[49]) genes #1% FDR <br />                                                                                  $3-fold change considered Osmotic Regulated Genes <br /> System (Stratagene) mutations verified DNA                       (ORGs). Data manually curated add annotation or <br /> sequencing. For deletion DGATA transgenics, wild type                   remove pseudogenes genes longer predicted expressed <br /> animals injected 150 ng/ml myo3p::dsRed2 plasmid                  using data available Wormbase (WS190), resulting final <br /> and 30 ng/ml relevant gpdh-1 construct using standard                     list 324 ORGs. All microarray data MIAME compliant and <br /> microinjection methods [48]. For quantification, animals carrying                deposited Gene Expression Omnibus (GEO) <br /> the extrachromosomal arrays placed COPAS Biosort                     database accession number GSE19310. <br /> (Union Biometrica) animals expressing dsRed separated <br /> from non-transgenic animals gating window. GFP                            Quantitative PCR <br /> expression dsRed+ animals normalized worm size                          cDNA reverse synthesized 1 mg total RNA (Super- <br /> (time-of-flight) sample normalized mean               script II kit, Invitrogen, Carlsbad, CA) qPCR performed <br /> corresponding control, indicated figure.                              using ABI7300 thermocycler TaqMan probe sets (Applied <br />                                                                                  Biosystems, Foster City, CA). Each reaction performed in <br /> C. elegans Growth Collection Microarray Analysis                         20 ml reactions technical triplicate quadruplicate from <br />   Hypochlorite synchronized L1 stage wild type animals                      2&#8211;3 biological replicates. Data normalized expression <br /> grown young adult stage standard NGM plates. Worm                      levels cpt-6, encodes carnitine palmitoyl transferase <br /> were washed plates using isotonic M9 solution (,170 mOsm)                    expression levels unchanged hypertonicity. Initial test <br />  <br />  <br />        PLoS ONE | www.plosone.org                                            8                           February 2010 | Volume 5 | Issue 2 | e9010 <br />                                                                                                                   Stress Immunity Worms <br />  <br />  <br /> showed data normalization using probes clh-3 lam-           microarrays) total number genes. P-values,0.05 were <br /> 3, expression unchanged hypertonicity, showed              taken indicate statistical significance. <br /> similar results achieved cpt-6 (data shown). <br />                                                                              Supporting Information <br /> Survival Assays, RNA Interference, COPAS Biosort <br />                                                                              Figure S1 qPCR validation hypertonic stress microarray data. <br /> Analysis                                                                     cDNA prepared independent biological replicates used <br />     Survival assay performed placing 10&#8211;20 young adult               template quantitative RT-PCR (see methods). Data shown <br /> animals OP50-seeded NGM plates containing indicated                 mean fold changes (hypertonic/isotonic) 6 S.D. Microarray <br /> amount NaCl. Animals scored survival 24 hours.             data mean fold changes 3 independent samples. <br /> Worms considered dead failed respond harsh              Found at: doi:10.1371/journal.pone.0009010.s001 (0.47 MB TIF) <br /> plate tapping light touch platinum wire. Animals died <br /> as result internally hatched progeny (&#8216;&#8216;bag-of-worms&#8217;&#8217;                Figure S2    ORGs exhibit significant overlap other <br /> phenotype) censored analysis. Each NaCl concentra-             stress-induced transcriptional responses. ORGs compared to <br /> tion repeated quadruplicate assay repeated 2&#8211;3           previously published microarray studies endoplasmic <br /> times. Figures illustrate data representative experiment.       reticulum unfolded protein response (UPR) response to <br />     For elt-2 RNAi experiments, L1 stage animals fed                hypoxia [20,21]. The enrichment score hypergeometric p- <br /> control RNAi bacteria (HT115 containing pPD129.36                  value enrichment calculated described previously [49]. <br /> vector (EV) plasmid) 24 hours 16u, washed plates,            Found at: doi:10.1371/journal.pone.0009010.s002 (0.38 MB TIF) <br /> distributed EV elt-2(RNAi)(from MRC RNAi library;                      Figure S3 Pearson Correlation analysis WT ORGs and <br /> sequence confirmed) plates 72 hours 16u prior               osr mutants. The pairwise Pearson Correlation coefficient was <br /> survival assays COPAS measurements. Using conditions,               determined dataset. Each experimental replicate is <br /> we able circumvent developmental arrest phenotype             shown indicated genotype. <br /> elt-2(RNAi) achieve ,90% knockdown elt-2 mRNA,               Found at: doi:10.1371/journal.pone.0009010.s003 (0.77 MB TIF) <br /> assessed qPCR (data shown). For survival assays, EV(RNAi) <br />                                                                              Figure S4 Truncation analysis gpdh-1 promoter. <br /> and elt-2(RNAi) animals transferred NGM RNAi plates <br />                                                                              Transgenic animals containing indicated lengths the <br /> containing indicated levels NaCl survival measured <br />                                                                              gpdh-1 promoter (from start ATG) fused GFP using a <br /> as described above. For COPAS assays, kbIs5 animals were <br />                                                                              PCR fusion method [52]. GFP expression transgenic animals <br /> washed new RNAi plates containing indicated RNAi <br />                                                                              quantified COPAS Biosort following exposure either <br /> clone 50 mM 200 mM NaCl. At indicated <br />                                                                              50 mM NaCl 200 mM NaCl. N.150 animals each <br /> timepoints, animals analyzed COPAS Biosort time-               measurement. <br /> of-flight (TOF) GFP expression. Raw data filtered                Found at: doi:10.1371/journal.pone.0009010.s004 (0.42 MB TIF) <br /> select adult animals (TOF.300) GFP data was <br /> normalized TOF measurement (GFP/TOF).                                 Table S1 Osmotically regulated genes wild type following 15 <br />                                                                              minute, 1 hour, 6 hour, 1 generation exposure 200 mM <br /> Motif Analysis                                                               NaCl regulation osr mutants (Excel file). <br />    1 kb promoter sequence (upstream start ATG)                Found at: doi:10.1371/journal.pone.0009010.s005 (0.10 MB <br /> each gene Groups 2&#8211;5 C. elegans ORG dataset                        XLS) <br /> (foreground dataset) C. briggsae orthologs searched           Table S2 Genes regulated osm-7(n1515) mutants their <br /> for presence transcription factor binding motif using           regulation osr mutants (Excel file). <br /> 811 motifs Positional weight Matrices (PWM) TRANSFAC                   Found at: doi:10.1371/journal.pone.0009010.s006 (0.06 MB <br /> 10.2 (www.gene-regulaiton.com). We performed search                 XLS) <br /> using 8451/8956 C. elegans/C. briggsae ortholog pairs exhibited <br />                                                                              Table S3 Genes regulated osm-8(n1518) mutants their <br /> detectable expression microarray data (background dataset) <br />                                                                              regulation osr mutants (Excel file). <br /> The motif scan using previously described tool, <br />                                                                              Found at: doi:10.1371/journal.pone.0009010.s007 (0.23 MB <br /> PWM_SCAN [50]. A stringent percentile score threshold 0.95 <br />                                                                              XLS) <br /> was used consider motif &#8216;&#8216;present&#8217;&#8217; gene promoter. The <br /> percentile score 0.95 means score 95th             Table S4 Genes regulated osm-11(n1604) mutants their <br /> percentile range scores achievable PWM. For motif,              regulation osr mutants (Excel file). <br /> given number matches foreground dataset (ORGs),                Found at: doi:10.1371/journal.pone.0009010.s008 (0.09 MB <br /> the p-value corresponding motif occurrence estimated              XLS) <br /> based 1000 random samplings background dataset                 Table S5 Genes regulated dpy-9(e12) mutants their <br /> computing fraction times motif occurence equaled             regulation osr mutants (Excel file). <br /> or exceeded occurrences foreground dataset. Only motifs           Found at: doi:10.1371/journal.pone.0009010.s009 (0.59 MB <br /> with p-values,0.05 C. elegans C. briggsae datasets           XLS) <br /> were considered significant. <br />                                                                              Table S6 Genes regulated dpy-10(e120) mutants their <br /> Statistical Analysis                                                         regulation osr mutants (Excel file). <br />   Data presented means 6 S.E. Survival qPCR gene                  Found at: doi:10.1371/journal.pone.0009010.s010 (0.70 MB <br /> expression data analyzed using Student&#8217;s T-test           XLS) <br /> one-way ANOVA, implemented Graphpad Prism (San                         Table S7 Overlap ORGs genes regulated eight <br /> Diego, CA). Enrichment scores enrichment significance               different models pathogen infection (Excel file). <br /> calculated previously described [51]. For enrichment analysis,            Found at: doi:10.1371/journal.pone.0009010.s011 (0.02 MB <br /> we used 13,866 (the number expressed genes ORG                     XLS) <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        9                         February 2010 | Volume 5 | Issue 2 | e9010 <br />                                                                                                                                              Stress Immunity Worms <br />  <br />  <br />  <br /> Acknowledgments                                                                             Author Contributions <br /> We thank Sara Cherry, Jim McGhee, members Sundaram                           Conceived designed experiments: TL. Performed experiments: <br /> Raizen labs valuable discussions.                                                       AKR YM TL. Analyzed data: AKR YM SH TL. Contributed <br />                                                                                             reagents/materials/analysis tools: SH TL. Wrote paper: TL. <br />  <br /> References <br />  1. Kim DH, Ausubel FM (2005) Evolutionary perspectives innate immunity                  27. Hodgkin J, Kuwabara PE, Corneliussen B (2000) A novel bacterial pathogen, <br />     study Caenorhabditis elegans. Curr Opin Immunol 17: 4&#8211;10.                       <span id='am-91' about='obo:NCBITaxon_151262' typeof='owl:Thing obo:NCBITaxon_85006 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_85023 obo:NCBITaxon_201174 obo:NCBITaxon_33882 obo:NCBITaxon_85003'><span id='am-92'  ></span><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="&apos;Microbacterium nematophilum&apos;" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Microbacterium nematophilum" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Microbacterium nematophilum</span>, induces morphological change nematode <br />  2. Akira S, Uematsu S, Takeuchi O (2006) Pathogen recognition innate                       C. elegans. Curr Biol 10: 1615&#8211;1618. <br />     immunity. Cell 124: 783&#8211;801.                                                            28. Murphy CT, McCarroll SA, Bargmann CI, Fraser A, Kamath RS, et al. (2003) <br />  3. Matsuzawa A, Saegusa K, Noguchi T, Sadamitsu C, Nishitoh H, et al. (2005)                   Genes act downstream DAF-16 influence lifespan of <br />     ROS-dependent activation TRAF6-ASK1-p38 pathway selectively                       Caenorhabditis elegans. Nature 424: 277&#8211;283. <br />     required TLR4-mediated innate immunity. Nat Immunol 6: 587&#8211;592.                     29. McGhee JD, Sleumer MC, Bilenky M, Wong K, McKay SJ, et al. (2007) The <br />  4. Powell JR, Ausubel FM (2008) Models Caenorhabditis elegans infection                  ELT-2 GATA-factor global regulation transcription C. elegans <br />     bacterial fungal pathogens. Methods Mol Biol 415: 403&#8211;427.                              intestine. Dev Biol 302: 627&#8211;645. <br />  5. Tan MW, Mahajan-Miklos S, Ausubel FM (1999) Killing Caenorhabditis                   30. Fukushige T, Hawkins MG, McGhee JD (1998) The GATA-factor elt-2 is <br />     elegans <span id='am-48' about='obo:NCBITaxon_287' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_286 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_136841 obo:NCBITaxon_135621'><span id='am-49' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931955" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Pseudomonas aeruginosa" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas polycolor" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasAlternativeId" content="NCBITaxon:932477" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasRelatedSynonym" content="probable synonym or variety: &quot;Pseudomonas polycolor&quot; Clara 1930" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931958" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Micrococcus pyocyaneus" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasRelatedSynonym" content="Peudomonas aeruginosa" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224290" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1437769" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Bacterium pyocyaneum" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1437770" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1437768" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas pyocyanea" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Bacterium aeruginosum" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931956" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="Bacillus aeruginosus" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931957" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Bacillus pyocyaneus" datatype="xsd:string"></span><span id='am-70'  ></span>Pseudomonas aeruginosa</span> used model mammalian bacterial                         essential formation Caenorhabditis elegans intestine. Dev Biol 198: <br />     pathogenesis. Proc Natl Acad Sci U S A 96: 715&#8211;720.                                         286&#8211;302. <br />  6. Kurz CL, Ewbank JJ (2007) Infection dish: high-throughput analyses              31. Tardy F, Homble F, Neyt C, Wattiez R, Cornelis GR, et al. (1999) Yersinia <br />     bacterial pathogenesis. Curr Opin Microbiol 10: 10&#8211;16.                                      enterocolitica type III secretion-translocation system: channel formation by <br />  7. Mahajan-Miklos S, Tan MW, Rahme LG, Ausubel FM (1999) Molecular                             secreted Yops. Embo J 18: 6793&#8211;6799. <br />     mechanisms bacterial virulence elucidated using Pseudomonas aeruginosa-            32. Aroian R, van der Goot FG (2007) Pore-forming toxins cellular non- <br />     Caenorhabditis elegans pathogenesis model. Cell 96: 47&#8211;56.                                  immune defenses (CNIDs). Curr Opin Microbiol 10: 57&#8211;61. <br />  8. Huffman DL, Abrami L, Sasik R, Corbeil J, van der Goot FG, et al. (2004)                33. Moraes TF, Spreter T, Strynadka NC (2008) Piecing type III <br />     Mitogen-activated protein kinase pathways defend bacterial pore-                    injectisome bacterial pathogens. Curr Opin Struct Biol 18: 258&#8211;266. <br />     forming toxins. Proc Natl Acad Sci U S A 101: 10995&#8211;11000.                              34. Go WY, Liu X, Roti MA, Liu F, Ho SN (2004) NFAT5/TonEBP mutant mice <br />  9. Kim DH, Liberati NT, Mizuno T, Inoue H, Hisamoto N, et al. (2004)                           define osmotic stress critical feature lymphoid microenvironment. <br />     Integration Caenorhabditis elegans MAPK pathways mediating immunity                      Proc Natl Acad Sci U S A 101: 10673&#8211;10678. <br />     stress resistance MEK-1 MAPK kinase VHP-1 MAPK phosphatase.                  35. <span id='am-71' about='obo:NCBITaxon_96544' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_33380 obo:NCBITaxon_6072 obo:NCBITaxon_7524 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_33385 obo:NCBITaxon_33373 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_96541 obo:NCBITaxon_6656 obo:NCBITaxon_143961 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33342 obo:NCBITaxon_50557'><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Trama" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-75'  ></span>Trama</span> J, Go WY, Ho SN (2002) The osmoprotective function NFAT5 <br />     Proc Natl Acad Sci U S A 101: 10990&#8211;10994.                                                  transcription factor T cell development activation. J Immunol 169: <br /> 10. Troemel ER, Chu SW, Reinke V, Lee SS, Ausubel FM, et al. (2006) p38 MAPK                    5477&#8211;5488. <br />     regulates expression immune response genes contributes longevity C.        36. Miyakawa H, Woo SK, Dahl SC, Handler JS, Kwon HM (1999) Tonicity- <br />     elegans. PLoS Genet 2: e183.                                                                responsive enhancer binding protein, rel-like protein stimulates <br /> 11. Pujol N, Zugasti O, Wong D, Couillault C, Kurz CL, et al. (2008) Anti-fungal                transcription response hypertonicity. Proc Natl Acad Sci U S A 96: <br />     innate immunity C. elegans enhanced evolutionary diversification                2538&#8211;2542. <br />     antimicrobial peptides. PLoS Pathog 4: e1000105.                                        37. Stergiopoulos K, Cabrero P, Davies SA, Dow JA (2009) Salty dog, SLC5 <br /> 12. Shapira M, Hamlin BJ, Rong J, Chen K, Ronen M, et al. (2006) A conserved                    symporter, modulates Drosophila response salt stress. Physiol Genomics 37: <br />     role GATA transcription factor regulating epithelial innate immune                 1&#8211;11. <br />     responses. Proc Natl Acad Sci U S A 103: 14086&#8211;14091.                                   38. McGhee JD, Fukushige T, Krause MW, Minnema SE, Goszczynski B, et al. <br /> 13. Kerry S, TeKippe M, Gaddis NC, Aballay A (2006) GATA transcription factor                   (2009) ELT-2 predominant transcription factor controlling differentiation <br />     required immunity bacterial fungal pathogens. PLoS ONE 1: e77.                   function C. elegans intestine, embryo adult. Dev Biol 327: <br /> 14. Patient RK, McGhee JD (2002) The GATA family (vertebrates                               551&#8211;565. <br />     invertebrates). Curr Opin Genet Dev 12: 416&#8211;422.                                        39. Burmeister C, Luersen K, Heinick A, Hussein A, Domagalski M, et al. (2008) <br /> 15. Senger K, Harris K, Levine M (2006) GATA factors participate tissue-specific             Oxidative stress Caenorhabditis elegans: protective effects Omega class <br />     immune responses Drosophila larvae. Proc Natl Acad Sci U S A 103:                        glutathione transferase (GSTO-1). Faseb J 22: 343&#8211;354. <br />     15957&#8211;15962.                                                                            40. Romney SJ, Thacker C, Leibold EA (2008) An iron enhancer element the <br /> 16. Petersen UM, Kadalayil L, Rehorn KP, Hoshizaki DK, Reuter R, et al. (1999)                  FTN-1 gene directs iron-dependent expression Caenorhabditis elegans <br />     Serpent regulates Drosophila immunity genes larval fat body               intestine. J Biol Chem 283: 716&#8211;725. <br />     essential GATA motif. Embo J 18: 4013&#8211;4022.                                             41. Moilanen LH, Fukushige T, Freedman JH (1999) Regulation metallothionein <br /> 17. P <br /> </body></html>