<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, May 1993,        p.   2970-2979                                                                 Vol. 175, No. 10 <br /> 0021-9193/93/102970-10$02.00/0 <br /> Copyright X 1993, American Society Microbiology <br />  <br />  <br />           The murI Gene <span id='am-52' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-53'  ></span><span id='am-54' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span>Escherichia coli</span> Is Essential Gene That <br />                    Encodes Glutamate Racemase Activity <br />                          <span id='am-124' about='obo:NCBITaxon_344662' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_344711 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_42315 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-125' property="rdfs:label" content="Patricia" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasRelatedSynonym" content="Patricia Fox 1940" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-129'  ></span>PATRICIA</span> DOUBLET,1 JEAN vAN HEIJENOORT,1 JEAN-PIERRE BOHIN,2 <br />                                       AND DOMINIQUE MENGIN LECREULX1* <br />            URA 1131 du Centre National la Recherche Scientifique (CNRS), Biochimie Molkulaire et Cellulaire, <br />            Universite <span id='am-107' about='obo:NCBITaxon_49669' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_50362 obo:NCBITaxon_4447 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_4667 obo:NCBITaxon_3193'><span id='am-108'  ></span><span id='am-109' property="oboInOwl:hasRelatedSynonym" content="Paris L." datatype="xsd:string"></span><span id='am-110' property="rdfs:label" content="Paris" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasAlternativeId" content="NCBITaxon:49665" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Paris</span>-Sud, Batiment 432, 91405 Orsay Cedex, UMR 111 du CNRS, Laboratoire Chimie <br />              Biologique, Universite des Sciences et Technologies Lille, 59655 Villeneuve d&apos;Ascq Cedex2 France <br />                                            Received 16 December 1992/Accepted 14 March 1993 <br />  <br />                The murI gene <span id='am-68' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-72'  ></span><span id='am-73' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span>Escherichia</span> coli recently identified basis ability complement only <br />             mutant requiring D-glutamic acid growth described date: strain WM335 E. coli B/r (P. <br />             Doublet, J. van Heienoort, D. Mengin-Lecreulx, J. Bacteriol. 174:5772-5779, 1992). We report <br />             experiments insertional mutagenesis murI gene demonstrate gene essential the <br />             biosynthesis D-glutamic acid, specific components cell wall peptidoglycan. A special strategy was <br />             used construction strains disrupted copy murI, limited capability E. coli <br />             strains grown rich medium internalize D-glutamic acid. The murI gene product overproduced and <br />             identified glutamate racemase activity. UDP-N-acetylmuramoyl-L-alanine (UDP-MurNAc-L-<span id='am-114' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-115' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-116' property="rdfs:label" content="Ala" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-118'  ></span>Ala</span>), which <br />             nucleotide substrate D-glutamic-acid-adding enzyme (the murD gene product) catalyzing the <br />             subsequent step pathway peptidoglycan synthesis, appears effector racemase activity. <br />  <br />    The biosynthesis bacterial cell wall peptidoglycan                 DNA     sequence chromosomal region (6), murI <br />  (murein) complex process involving different                  gene   product predicted protein 289 amino <br /> cytoplasmic membrane steps (18, 37). In Eschenichia                  acids molecular weight 31,500. <br /> coli, D-glutamic acid specific components              It previously shown murI gene <br /> peptidoglycan structure. It incorporated specifically            capable complementing specific defect strain <br /> <span id='am-39' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-41'  ></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> metabolic pathway addition UDP-N-acetylmu-                WM335 (11). However, observation suffi- <br /> ramoyl-L-alanine (UDP-MurNAc-L-Ala), reaction cata-                    cient assess gene product sustain <br /> lyzed D-glutamic-acid-adding enzyme (the murD gene                D-glutamic acid synthesis E. coli. Two genes alanine <br /> product) (28, 35). The enzymatic mechanism                      racemases genes D-alanine:D-alanine <br /> D-glutamic acid produced bacteria investigated            ligases, catalyzed reactions metabolic <br /> for limited number <span id='am-49' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>. Two different routes                pathway, previously identified E. coli <span id='am-20' about='obo:NCBITaxon_8028' typeof='owl:Thing obo:NCBITaxon_33511 obo:NCBITaxon_186623 obo:NCBITaxon_41705 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_7711 obo:NCBITaxon_1 obo:NCBITaxon_1489388 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_8015 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_504568 obo:NCBITaxon_1489341 obo:NCBITaxon_33154 obo:NCBITaxon_7776 obo:NCBITaxon_8006 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Salmothymus" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="NCBITaxon:62064" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="NCBITaxon:82302" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Acantholingua" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="rdfs:label" content="Salmo" datatype="xsd:string"></span>Salmo</span>- <br /> D-glutamic acid biosynthesis identified: (i) trans-          nella typhimurium (43, 44, 47). We report experiments of <br /> amination process catalyzed D-amino acid transaminase               insertional mutagenesis murI gene demonstrating that <br /> with D-alanine a-ketoglutarate, demonstrated Ba-               essential biosynthesis D-glutamic acid E. <br /> cillus species (24, 40), (ii) direct conversion L-glu-           coli. A special strategy construction strains a <br /> tamic acid D-glutamic acid catalyzed glutamate                   disrupted copy murI chromosome used, <br /> racemase, observed <span id='am-6' about='obo:NCBITaxon_1578' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_33958 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-7' property="oboInOwl:hasDbXref" content="PMID:21724957" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="PMID:19244449" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Lactobacillus" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="PMID:19060088" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="PMID:18048724" datatype="xsd:string"></span><span id='am-14'  ></span>Lactobacillus</span> <span id='am-1' about='obo:NCBITaxon_1253' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_33958 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="rdfs:label" content="Pediococcus" datatype="xsd:string"></span>Pediococcus</span>                 apparently limited capability E. coli strains <br /> species (33, 34). However, attempts identify             D-glutamic acid rich medium. Furthermore, the <br /> two enzymatic activities E. coli present               murI gene product identified glutamate racemase <br /> unsuccessful (11, 22, 33, 40).                                           activity. <br />    We recently investigated problem and <br /> in particular genetics D-glutamic acid biosynthesis in <br /> E. coli (11). For purpose, used known                                  MATERIALS AND METHODS <br /> mutant requiring D-glutamic acid described to <br /> date: strain WM335 E. coli B/r (17, 22), resembled                 Bacterial strains, phages, <span id='am-145' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-146' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-147' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span>. The E. coli strains <br /> other previously described mutants defects pepti-                used study listed Table 1. Plasmid pPDM3 <br /> doglycan synthesis lytic phenotype exhibited             carrying lamB gene E. coli coding lambda <br /> restrictive growth conditions. In particular, identified          receptor cI857 gene codes thermosensitive <br /> gene named murI alteration apparently provoked <br />                                                                          form lambda repressor previously described (11). <br /> the auxotrophy D-glutamic acid mutant strain (11).            pUC18, pUC19, pTTQ19 <span id='am-103' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-104' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> Kanr genblock <br /> The gene unambiguously mapped 90-min region                originating pUC4K plasmid purchased from <br /> the E. coli chromosome corresponded previously                  Pharmacia, plasmid pBTacl obtained Boehr- <br /> sequenced open reading frame ORFi (6), flanked left               inger. Plasmid pSKS114 (39) used source the <br /> by btuB gene encoding vitamin B12 outer membrane                 cat cartridge. Plasmid pMAK705 (15) bearing thermosen- <br /> receptor protein (1, 16) right rrnB operon             sitive replicon obtained S. R. Kushner Y. <br /> of genes 16S, 23S, 5S rRNAs (5, 6, 14). From                 Mechulam (Laboratoire Biochimie, Ecole Polytechnique, <br />                                                                          Palaiseau, France). Phage Xdarg13 carrying 20-kb chromo- <br />                                                                          somal fragment E. coli extending fromppc murI was <br />                                                                          previously described (25). Phage clone XE11C11 (clone 534) <br />   * <br />       Corresponding author.                                              E. coli miniset library Kohara et al. (20) initially <br />                                                                   2970 <br />  VOL. 175, 1993                                        murI FROM E. COLI ENCODES GLUTAMATE RACEMASE ACTIVITY                                         2971 <br />  <br />                                                    TABLE 1. E. coli strains used study <br />          Strain                                                Genotype marker                                                Source referencea <br /> E. coli K-12 strains <br />   JM83                          araA(lac-proAB) rpsL thi +80 dlacZAM15                                                      46 <br />   JM109                         recAl A(lac-proAB) endAl gyrA96 <span id='am-15' about='obo:NCBITaxon_557274' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_557273 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_197562 obo:NCBITaxon_116704 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_6774'><span id='am-16'  ></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Thia" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>thiA</span> hsdRl7 supE44 reLA1 (F&apos; traD36                         46 <br />                                   proAB laclqZAM15) <br />   AB1133                        F- thr-1 leuB6 A(gpt-proA)62 hisG4 argE3 thi-1 lacYl galK2 <span id='am-34' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>ara</span>-14                           2 <br />                                   xyl-5 mtl-1 rpsL-31 supE44 <br />   JC7623                        AB1133 tsx-33 recB21 recC22 sbcB15                                                          45 <br />   NFB366                        JC7623 (X199 Xdarg13)                                                                   This work <br />   FB8                           F- prototroph                                                                               F. Blasi <br />   JE1011                        F- thr leu trp thy thi lac gal xyl mtl rpsL azi                                         32 <br />   ST5                           JE1011 mrbA (murB)                                                                          B. Bachmann     CGSC 6442 <br />   NK5139                        thi-39::TnlO IN(rrnD-rrnE)1                                                                 B. Bachmann     CGSC 6165 <br />   BW6175                        Hfr X- thr-1 leuB6 azi-15 tonA21 lacYl supE44 argE86::TnlO thi-1                            B. Bachmann     CGSC 6763 <br />   BW6165                        Hfr ara-41, lacYl, lacY40 Xind- xyl-7 mtl-2 argE86::TnlO                                 B. Bachmann     CGSC 6760 <br />   NFB279                        thr-1 leuB6 A(gpt-proA) 62 hisG4 argE3 thi-1 ara-14 lacYl galK2 xyl-5                       This work <br />                                   mtl-1 btuB202::TnlO rpsL31 supE44 <br />   P180                          MG1655 <span id='am-88' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-90'  ></span><span id='am-91' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>metA</span> A::Cmr                                                                          C. Richaud <br />   GRA83                         JM83 murI::kan (pMAKI)                                                                      This work <br /> E. coli B/r strains <br />   WM301                         leu pro trp arg thyA deoB met lac gal xyl ara mal lam phx rpsL                          W. Messer PC 2341 <br />                                  hsdS-K12 <br />   WM335                         WM301 glt (murI-335)                                                                        W. Messer PC 2342 <br />   GRA101                        WM335 murI-335 argE86::TnlO                                                                 This work <br />   GRA102                        WM335 murI-335 btuB202::TnlO                                                                This work <br />   GRA103                        WM335 murI-335 thi-39::TnlO                                                                 This work <br />   GRA104                        WM335 murI+ metAA::Cmr                                                                      This work <br />   a <br />     CGSC, E. coli Genetic Stock Center, Yale University, New Haven, Conn.; PC, Phabagen Collection, Department Molecular Cell Biology, State University <br /> of Utrecht, The Netherlands. <br />  <br />  <br /> selected complementation D-glutamic-acid-requir-                    amounts isomers glutamic acid estimated in <br /> ing mutant strain used DNA source                           cell extracts reaction mixtures (see assays <span id='am-119' about='obo:NCBITaxon_173684' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_41937 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_4011 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-120' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-121' property="rdfs:label" content="Gluta" datatype="xsd:string"></span><span id='am-122'  ></span><span id='am-123' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>gluta</span>- <br /> cloning murI gene, previously described (11).                        mate racemase activity below) recently developed <br />    Growth conditions. Unless noted, 2YT (31)                    high-performance liquid chromatography (HPLC) tech- <br /> rich medium M63 (31) supplemented 0.4% glucose                        nique, derivatization aliquots Marfey&apos;s chiral <br /> as defined minimal medium used growing cells.                       reagent. Diastereoisomers L- D-glutamic acid ob- <br /> Growth monitored 600 nm spectrophotometer                        tained reaction 1-fluoro-2,4-dinitrophenyl-5-L- <br /> (model 240; Gilford Instrument Laboratories, Inc., Oberlin,                    alanine-amide described Marfey (23) separated <br /> Ohio). D-Glutamic acid (100 ,ug. ml-&apos;), thymine (100 jig                       HPLC described legend Fig. 5. <br /> ml-&apos;), thiamine (1 ,ug. ml-&apos;), L-amino acids (50 pg                           Isolation sacculi quantitation peptidoglycan. Ex- <br /> ml-1) added case specific strain require-                  ponential-phase cells GRA83 grown 30&#176;C first <br /> ments. For strains carrying drug resistance genes, antibiotics                 30&#176;C 43&#176;C described above. Harvested cells <br /> were used following concentrations (in micrograms                       washed cold 0.85% NaCl solution centri- <br /> per milliliter): ampicillin (100), streptomycin (50), kanamycin                fuged again. Bacteria rapidly suspended under <br /> (30), chloramphenicol (25), tetracycline (15). Broth                   vigorous stirring 40 ml hot (95 100&#176;C) aqueous 4% <br /> plates solidified 1.5% agar, plasmid                         sodium dodecyl sulfate (SDS) solution 30 min. After <br /> inserts screened absence ot-complementation,                   standing overnight rQom temperature, suspensions <br /> 5-bromo-4-chloro-3-indolyl-,3-D-galactopyranoside (X-Gal)                      centrifuged 30 min 200,000 x g, pellets <br /> was added 40 ,g ml-&apos;.                                                       washed times water. Final suspensions <br />    Pool levels peptidoglycan precursors. Cells GRA83                     5 ml water homogenized brief sonication. <br /> (1-liter cultures) grown exponentially 30&#176;C 2YT                     Aliquots hydrolyzed analyzed previously de- <br /> medium. At appropriate cell concentration (3 x 106.                        scribed (26, 30), peptidoglycan content sacculi <br /> ml-&apos;), temperature culture maintained                    expressed terms diaminopimelic acid content. <br /> at 30&#176;C increased 43&#176;C. Incubation continued                   The specific incorporation radioactive meso-diami- <br /> the optical density culture 43&#176;C reached plateau                   nopimelic acid peptidoglycan lysA derivative <br /> value 0.5 3 h later (see Fig. 2). At time,                       strain (29) used similar results. <br /> preceding onset cell lysis minutes, cells                    General DNA techniques E. coli cell transformation. <br /> were rapidly chilled 0&#176;C harvested cold. As                    The isolation phages extraction purification of <br /> control, similar cultures strain JM83                           DNAs carried standard techniques (9, 38). <br /> (pMAKI) harboring intact chromosomal copy murI.                          Small- large-scale plasmid isolations carried by <br /> The extraction peptidoglycan precursors analyti-                    alkaline lysis method (38), plasmids eventually <br /> cal procedure used quantitation previously                   purified cesium chloride-ethidium bromide gra- <br /> described (26, 27).                                                            dients. Standard procedures endonuclease digestions, <br />    Separation isomers glutamic acid. The relative                        ligation, filling 5&apos;-protruding ends using Klenow <br />     2972      DOUBLET ET AL.                                                                                                        J. BACrERIOL. <br />  <br />                          89.5                              90                                              0    a <br />                                                                                                                      91I   min <br />                                   argECBH           btuB muri rrnB murB birA            rpoB,C   thi                metA <br />  <br />  <br />  <br />  <br />                                          E    F.I      BP <br />                                                        I <br />                                                                    N       D       HI            HE <br />                                                                            I                      I <br />  <br />                                              IbtuB      IvnUrIN            PI 1 59 rrsB          II <br />                                                                                                  - <br />  <br />  <br />                                          E    F.       BP <br />                                                        ..  a <br />                                                                    N       D       ZUk2JJ pDME <br />                                            I <br />                                        *10.xx                                                    l-    PPDM25 <br />                                                 F    N     D <br />                                              *110\\\\\\\\1]                    pPDM27 <br />                                                   BP N <br />                                                                                pPDM30 <br />                                                                Kanr <br />  <br />  <br />                                               OF       B       P       r   D <br />                                                                                p PDM 31 <br />      FIG. 1. Localization murI gene 90-min region E. coli chromosome. Locations genes (3) used our <br />   transduction experiments indicated top. Bacterial DNAs present phage clone X534 library Kohara et al. (20) in <br />   derived plasmid inserts indicated below. The murI gene represented hatched region, position lac promoter relative <br /> X insert pUC18- pUC19-derived plasmid indicated arrow. Positions cleavage sites shown BstEII (B), DraI <br />   (D), EcoRI (E), EcoRV (F), HindIII (H), NaeI (N), PstI (P). <br />  <br />  <br />   fragment DNA polymerase I, agarose electrophoresis                      D-glutamic acid incubated 370C. Growth ob- <br />   used (38). Usually, E. coli cells competent                     served 24 h incubation. <br />   transformation plasmid DNA method de-                             Construction plasmids. DNA phage XE11C11 <br />   scribed Dagert Ehrlich (8), technique                        (clone 534) collection Kohara et al. (20) used <br />   electroporation preferentially used strain WM335                      starting material construction different <br />   E. coli B/r transformed.                                                plasmids used study (see Fig. 1). Plasmid pPDM25 <br />      Complementation tests D-glutamate auxotrophy. (i)                    carrying 2.5-kb EcoRI chromosomal fragment con- <br />   Adarg13 phage particles. A 200-pI volume suspension                    tains murI gene E. coli previously described <br />   WM335(pPDM3) cells 10 mM MgSO4 incubated 1                          (11). The size murI-containing insert reduced to <br />   h 30&apos;C shaking lysate containing mixture                   1.3-kb EcoRV-DraI 0.96-kb BstEII-DraI fragments that <br />   X199 (carrying cI857 gene) Xdarg13 (a X199-                         cloned HincII site pUC18 vector to <br />   transducing derivative) phage particles (1010. ml-&apos;,                      generate plasmids pPDM27 pPDM30, respectively (Fig. <br />   multiplicity infection 10). After addition 400 VA                  1). For expression murI control the <br />   2YT medium supplemented D-glutamic acid, mix-                         lambda PR promoter, plasmid pPDM33 constructed by <br />   ture incubated 30&apos;C shaking additional 2-h                  inserting 2.4-kb XhoII lambda fragment (carrying the <br />   period expression phage genes. Cells recovered                     structural gene cI857 encoding thermosensitive form the <br />   centrifugation, washed times 2YT medium                     lambda cI repressor, strongpR promoter, Shine- <br />   remove residual D-glutamic acid, plated 2YT                        Dalgarno sequence cro) unique BamHI site of <br />   plates supplemented supplemented D-glutamic                        plasmid pPDM30. Plasmid pPDM34 constructed by <br />   acid. Growth observed 24 h incubation 30&apos;C                     inserting initial 2.5-kb EcoRI fragment corre- <br />   plates lacking D-glutamic acid, clones                        sponding site pTTQ19 vector, orientation <br />   tested lysogenization lambda phages (thermosensitive                    dependent tac promoter control. A cat cartridge <br />   growth).                                                                       originating plasmid pSKS114 inserted both <br />      (ii) Plasmids. WM335 cells competent de-                       possible orientations unique NaeI site plasmid <br />   scribed transformed various plasmids                    pPDM34 (lying middle murI gene sequence) to <br />   tested. The cell suspension (200 pA) mixed DNA                        generate plasmids pPDM35 pPDM36. Similarly, kan <br />   kept ice 2 h heated 3 min 420C.                     cartridge originating pUC4K inserted both <br />   Then, 400 pA 2YT medium supplemented D-glutamic                        possible orientations NaeI site plasmid pPDM27, <br />   acid added, cells incubated 370C 1                    generating pPDM31 pPDM32 (Fig. 1). The pMAK~kan <br />   h expression plasmid genes. Cells recovered                     plasmid obtained insertion 2.5-kb EcoRI-SphI <br />   centrifugation washed times 2YT medium.                       fragment pPDM31 HindIII site pMAK705 <br />   A final suspension 200 p1 2YT medium                            (15). The pMAKI plasmid isolated described the <br />   separated 2 parts plated 2YT-                         text excision process followed integration of <br />   antibiotic plates supplemented supplemented                        pMAKIkan plasmid chromosomal murI locus. The <br />  VOL. 175, 1993                                murl FROM E. COLI ENCODES GLUTAMATE RACEMASE ACTIVITY                         2973 <br />  <br /> construction pTACI2 plasmid carrying murI gene         Bq; Amersham), 0.04 mM UDP-MurNAc-L-Ala, crude <br /> under control tac promoter diagrammed Fig.       enzyme (30 300 pug protein) final volume 80 jul. <br /> 3.                                                                The mixtures incubated 370C 1 h, reactions <br />    Disruption chromosomal murI gene. The wild-type         terminated addition 20 pl acetic acid. <br /> chromosomal copy murI gene E. coli strains          Reaction products separated high-voltage electro- <br /> replaced disrupted following alternative          phoresis Whatman 3MM filter paper 2% formic acid <br /> methods. (i) The procedure described Hamilton et al.           (pH 1.9) 45 min 40 V/cm LT36 apparatus <br /> (15), uses pMAK705, plasmid bearing thermosen-          (Savant Instruments, Hicksville, N.Y.), radioactive <br /> sitive replicon, used. pMAK.Ian, pMAK705 deriva-            spots located overnight autoradiography type <br /> tive carrying disrupted murI gene, transformed       R2 films (3M, St. Paul, Minn.) radioactivity scanner <br /> JM83. Integration plasmid chromosome          (model Multi-Tracermaster LB285; Berthold France, Elan- <br /> selected plating cells 440C 2YT plates containing    court, France). The radioactive spots (glutamic acid and <br /> 20 ,ug chloramphenicol ml-&apos;. Several clones picked        UDP-MurNAc-dipeptide) cut counted an <br /> up, integration pMAKIkan murI locus         Intertechnique SL 30 liquid scintillation spectrophotometer <br /> verified Southern analysis (<span id='am-130' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-131' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-132'  ></span><span id='am-133' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> shown). The plasmid       solvent consisting 2 ml water 13 ml <br /> was excised chromosome follows. Cointe-          Aqualyte mixture (J. T. Baker Chemicals, Deventer, The <br /> grants grown 300C 2YT-chloramphenicol           Netherlands). It verified [D-14C]glutamic acid in <br /> least 30 doubling times subsequently plated           cases quantitatively converted nucleotide <br /> 30&apos;C. Individual clones screened sensitivity     form D-glutamic-acid-adding activity. (ii) By direct <br /> chloramphenicol 440C, indicative plasmid           assay, UDP-MurNAc-L-Ala ATP-Mg + omitted <br /> excision. The structures excised plasmids sev-        reaction mixture, ratio L- D-isomer of <br /> eral clones determined DNA restriction analysis.          glutamic acid determined HPLC procedure <br /> One clone containing plasmid bearing wild-type murI         described above. <br /> gene (called pMAKI) chosen. Finally, replacement <br /> in strain (named GRA83) chromosomal murI copy                                    RESULTS <br /> by inactivated verified Southern analysis <br /> (data shown). (ii) The procedure described Winans et         Attempts transduce murI mutation bacterial strains <br /> al. (45) used. Plasmid pPDM35 pPDM36 harboring           D-glutamic-acid-requiring mutant strain <br /> disrupted copy murI gene (by cat cartridge)          WM335. The murI gene E. coli, required the <br /> linearized BamHI used transform strain JC7623,        biosynthesis D-glutamic acid, recently identified in <br /> a recB recC sbcB strain transformed linear         90-min region chromosome (11). This function was <br /> DNA (45). Exchange mutated copy murI           apparently altered auxotrophic mutant described <br /> chromosomal wild-type gene selected plating         date, strain WM335 E. coli B/r (17, 22). We tried to <br /> transformed bacteria 2YT plates presence             transduce mutation (named murI-335) E. coli K-12. <br /> chloramphenicol D-glutamic acid.                              This using <span id='am-93' about='obo:NCBITaxon_95269' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_85772 obo:NCBITaxon_6656 obo:NCBITaxon_85773 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7399 obo:NCBITaxon_50557 obo:NCBITaxon_6072 obo:NCBITaxon_6960'><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-95'  ></span><span id='am-96' property="rdfs:label" content="Arge" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>argE</span>, btuB, thi markers (with the <br />    Preparation crude enzyme. Cells JM109(pBTacl)          TnlO transposon inserted), chosen high frequencies <br /> and JM109(pTACI2) (constructs diagrammed Fig. 3)          cotransduction wild-type murI+ gene (35, 95, and <br /> grown exponentially (1-liter cultures, duplicate) 370C   75%, respectively). In case, clones WM335 that <br /> LB medium (31) supplemented ampicillin. When             acquired tetracycline resistance remained auxotrophic <br /> optical density cultures reached 0.1, isopropyl-13-D-      D-glutamic acid isolated (strains GRA101 to <br /> thiogalactopyranoside (IPTG) added cul-         GRA103). P1 phage lysates produced strains were <br /> tures final concentration 1 mM, growth            used transduce TnlO transposon prototro- <br /> continued 3 h. Cells harvested cold           phic strain, FB8, E. coli K-12. Transductants were <br /> washed 40 ml cold 0.02 M potassium phosphate              selected 2YT plates supplemented tetracycline and <br /> buffer (pH 7) containing 0.1% 3-mercaptoethanol (standard         D-glutamic acid. Surprisingly, isolated argE::TnlO, <br /> buffer). The wet cell pellet (ca. 1.5 g) suspended 12 ml   btuB::TnlO, thi::TnlO clones (more 300 each <br /> of buffer disrupted sonication (Sonicator         case) appeared prototrophic D-glutamic acid. This <br /> 150; T. S. Ultrasons, Annemasse, France) 10 min          unexpected finding difference B <br /> cooling. The resulting suspension centrifuged 40C      K-12 species organization genes this <br /> 30 min 200,000 x g Beckman TL100 centrifuge. The        region chromosome, observed if <br /> supernatant eventually dialyzed overnight 40C      parental strain WM301 E. coli B/r used the <br /> 100 volumes standard buffer, resulting solution    recipient strain. It verified phage stocks carried <br /> (300 mg protein 11 ml), designated crude enzyme,       murI-335 allele. Strain WM335 converted to <br /> was stored -20&apos;C. SDS-polyacrylamide gel electrophore-         prototrophy acquisition murI+ gene, was <br /> sis (PAGE) analysis proteins crude        cotransduced case 25% met4::Cm marker. <br /> extracts performed previously described (21)          Then, observed clone (GRA104) returned <br /> 13% polyacrylamide gels.                                          initial auxotrophic status transduced Tetr by <br />    Enzymatic assays. Glutamate racemase activity crude         phage stocks Tetr murI-335 described above. <br /> enzyme preparations determined different pro-            These results interpreted ways. (i) The <br /> cedures. (i) In coupled assay, D-glutamic acid pro-       murl mutation effectively transferred ge- <br /> duced L-isomer glutamate racemase              netic backgrounds, effect D-glutamic acid <br /> converted UDP-MurNAc-L-Ala-D-Glu D-glutam-              biosynthesis, result indicative presence another <br /> ic-acid-adding enzyme present high level activity    gene E. coli chromosome compensating murI <br /> in crude extracts (30, 35). The standard assay mixture            mutation. However, hypothesis unlikely, since <br /> contained 0.1 M Tris-hydrochloride buffer (pH 8.6), 20 mM         previous experiments complementation WM335 <br /> MgCl2, 10 mM ATP, 0.02 mM [L-14C]glutamic acid (100,000           defect various DNA sources allowed identify only <br />  2974     DOUBLET ET AL.                                                                                                J. BACT1ERIOL. <br />  <br /> the murI gene. (ii) The murI mutation trans-             L-glutamic acid particular permease encoded by <br /> ferred strains WM335 mutant, probably            gltS gene previously shown transport <br /> a result additional mutation strain     D-isomer (10, 19, 42). <br /> prerequisite isolation mutants altered murI             Inactivation chromosomal murI gene effect on <br /> gene. This second hypothesis corroborated studying             peptidoglycan metabolism. To inactivate murI gene on <br /> the cotransduction frequencies markers equidistant           chromosome, used procedure described by <br /> from region (Fig. 1): argE, murI, murB, codes          Hamilton et al. (15) particularly adapted the <br /> for UDP-GlcNAc-enolpyruvate reductase (12, 36). A                 case essential genes. First, murI gene-coding se- <br /> thermosensitive ST5 strain E. coli K-12 (32) altered        quence carried pPDM27 plasmid disrupted the <br /> murB gene transduced tetracycline resistance                insertion NaeI site 1.24-kb kanamycin resis- <br /> using different phage stocks: specifying argE::TnlO           tance gene pUC4K, generating plasmid pPDM31 (Fig. <br /> murI+ murB+ specifying argE::TnlO murI-335                    1). Then, total insert pPDM31 plasmid was <br /> murB+. Arginine-requiring clones selected 30&apos;C             inserted pMAK705 vector bears thermosen- <br /> 2YT plates supplemented tetracycline D-glutamic              sitive replicon. The resulting plasmid, pMAKLkan, used <br /> acid. Although case, 30% Tetr ST5                 described Materials Methods construction of <br /> derivatives isolated thermoresistant (murB+),             strain derived JM83, GRA83 (JM83 murI::kan <br /> none Tetr clones selected experiment              [pMAKI]), having inactivated murI gene chromo- <br /> could grow 420C, result suggesting impossible        wild-type allele pMAK705 vector. At <br /> to transfer murB+ gene mutated copy            nonpermissive temperature plasmid replication (43 to <br /> the murI gene.                                                        440C), GRA83 failed grow 2YT plates, indicating that <br />    A second mutation present strain WM335               strain disrupted copy murI gene the <br /> essential isolation murI mutation. To confirm        chromosome viable presence trans a <br /> involvement sole murI gene synthesis                 plasmid carrying murI+ gene. As control, GRA83 was <br /> D-glutamic acid E. coli, tried inactivate gene       infected P1 transduction strains having one <br /> the chromosome using procedure described Win-               (JC7623) (NFB366 [JC7623 Xdargl3J) copies the <br /> ans et al. (45). Cartridges expressing resistance chloram-         murI gene. When selected marker Kanr, the <br /> phenicol kanamycin inserted various murI-                strain copies murI yielded transductants, it <br /> carrying plasmids NaeI site located middle                did high frequency, result confirming murI an <br /> position sequence. The resulting plasmids           essential gene. <br /> disrupted copy murI linearized enzymes                 Since plasmid pMAKI bears thermosensitive repli- <br /> cut polylinker vectors used            con, effects inactivation murI gene were <br /> transform strain JC7623 E. coli K-12, recB recC sbcB             observed shifting cultures GRA83 cells growing 30&#176;C <br /> strain transformed linear DNA (45). All         2YT medium nonpermissive temperature 43&#176;C. <br /> the Cmr Kanr clones selected presence D-glu-             As shown Fig. 2A, cells grew normally 43&#176;C about <br /> tamic acid fact ampicillin resistant            3 h rapidly lysed. The delay preceding cell lysis <br /> contained original intact plasmid. By method,             (about 9 generation times) corresponds time needed <br /> were unable exchange chromosomal copy murI              copies pMAKI plasmid cured, <br /> gene disrupted one, lengths                followed time preexisting MurI molecules to <br /> homologous DNAs present plasmids sides               progressively diluted remaining internal pool <br /> of inserted cartridge spite presence high      D-glutamic acid consumed. To demonstrate that <br /> concentration D-glutamic acid growth medium.                lysis occurred result depletion pool of <br />    The results experiments described              D-glutamic acid, cells harvested just onset of <br /> that altered murI gene strain WM335 untrans-             cell lysis, contents peptidoglycan nucleotide <br /> ferable E. coli strains chromosomal             precursors estimated. As previously shown the <br /> murI gene disrupted affecting cell               WM335 mutant deprived D-glutamic acid (11), GRA83 <br /> viability, D-glutamic acid present high             cells accumulated 43&#176;C large amounts UDP-MurNAc- <br /> concentration growth medium. This suggested                L-Ala, pools precursors located downstream in <br /> presence strain WM335 second mutation                pathway progressively depleted (data shown). <br /> essential expression phenotype requiring D-glu-          As result, cell lysis observed cell peptidogly- <br /> tamic acid. The likely hypothesis wild-type E.          content reduced critical value representing 70% <br /> coli strains did efficiently internalize D-glutamic acid,         normal cell content. <br /> i.e., rate sufficient fulfill specific require-      Furthermore, D-glutamic acid shown restore the <br /> ments peptidoglycan synthesis. Transductants ac-              normal phenotype GRA83 cells 43&#176;C growth was <br /> quired disrupted copy murI gene chromosome            performed minimal medium (Fig. 2B). This result further <br /> probably lysed expression period, spite          indicated uptake D-amino acid wild-type <br /> presence D-amino acid growth medium.                    strains blocked complex growth media were <br />    We observed labeled [DL-14C]glutamic acid inter-          used. It finally proved phenomenon related <br /> nalized poorly WM335 WM301 strains                   presence rich medium high concentration of <br /> when added exponential-phase cells growing rich                 L-glutamic acid competes D-isomer the <br /> medium (data shown). However, slightly higher rate           permease (data shown). <br /> uptake observed mutant cells. This probably                   Overproduction murI gene product. Previous at- <br /> explains capability D-amino acid complement             tempts determine biochemical mechanism which <br /> the murI defect strain. It aim             murI gene product synthesizes D-glutamic acid E. coli <br /> present study analyze additional muta-          unsuccessful (11, 22, 33, 40). The overproduc- <br /> tion increased uptake D-glutamic acid strain           tion murI gene product considered a <br /> WM335. Likely candidates permeases specific               prerequisite facilitated identification activity. <br />  VOL. 175, 1993                                   murl FROM E. COLI ENCODES GLUTAMATE RACEMASE ACITIVITY                                 2975 <br />  <br />  <br />  <br />  <br />                            305 <br />                             0j <br />  <br />  <br />  <br />  <br />                              0.01- <br />  <br />  <br />  <br />  <br />                                          2        4        6   h <br />                                                                  hours               4         8         12 <br />   FIG. 2. Effect inactivation murI gene growth E. coli K-12. GRA83 cells grown exponentially 30&apos;C either <br /> 2YT (A) minimal medium supplemented 0.4% glucose (B). At time indicated arrow, temperature culture either <br /> maintained 30&apos;C shifted 430C. Symbols: *, growth 300C absence D-glutamic acid; 0, growth 43&apos;C presence of <br /> D-glutamic acid; 0, growth 43&apos;C absence D-glutamic acid. O.D., optical density. <br />  <br />                                                                                      EcoRl                             PstI <br />    For purpose, plasmids expressing gene                      5&apos; P-AATTCATGAGACAGTCTATGGCTACCAAACTGCA 3&apos; <br />                                                                                  3&apos;        GTACTCTGTCAGATACCGATGGTTTG-P     5&apos; <br /> control strong promoters constructed. A plas- <br /> mid, pPDM33, contained BstEII-DraI fragment ex- <br /> pressed control lambda PR promoter. <br /> However, exponentially growing cells JM83 <br /> (pPDM33) shifted 420C (for derepression PR                                                  I            P   N       D     H <br /> promoter), overproduction protein detected                                                             Lour <br /> in SDS-PAGE analysis corresponding extracts. Baliko <br /> et al. (4) previously tried determine biological function <br /> of gene, supposed related downstream rrnB <br /> operon rRNA genes (5, 14). In particular, reported <br /> that attempts achieve significant overproduction of <br /> the protein merely exchanging promoter ORFi <br /> gene (our murl gene) stronger promoter (in case, <br /> the P2 promoter rmB) failed completely, suc- <br /> ceeded constructing overproducing plasmid by <br /> exploiting principle translational coupling (4). <br />    We constructed plasmid expressing murI gene <br /> directly control strong tac promoter (Fig. 3). <br /> In construct, chromosomal DNA sequence <br /> preceding ATG initiation <span id='am-44' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-47'  ></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> eliminated. We took <br /> advantage fact unique PstI site lies the <br /> gene sequence 25 bp putative start codon <br /> (Fig. 1). The 1.3-kb PstI-HindIII fragment contains the <br /> truncated murI gene purified ligated pBTacl <br /> vector (cut EcoRI HindIII), presence a <br /> synthetic phosphorylated double-strand linker correspond- <br /> ing beginning gene (from ATG initiation <br /> codon PstI site) preceded EcoRI site (Fig. 3).               FIG. 3. Construction plasmid pTACI2. The 1.3-kb PstI- <br /> The D-glutamic-acid-requiring strain WM355 trans-                  HindIII fragment contains murI gene truncated 25 <br /> formed ligation mixture electroporation. By               bases ligated pBTacl vector cut EcoRI HindIII in <br />                                                                        presence phosphorylated synthetic oligonucleotides <br /> selecting directly complementation, isolated 25 clones          sequences shown top. When hybridized together, <br /> that carried expected plasmid, called pTACI2 (Fig. 3).         oligonucleotides reconstitute 5&apos; end gene, the <br />    Recombination-deficient strain JM109 carrying copy              initiation codon (underlined) PstI site, preceded EcoRI <br /> of lacPq repressor gene F&apos; transformed             site. Positions cleavage sites indicated BamHI (B), DraI <br /> with plasmid. Exponential-phase JM109 cells harboring             (D), EcoRI (E), HindIII (H), NaeI (N), PstI (P), SmaI (S). <br />  2976     DOUBLET ET AL.                                                                                                  J. BACTERIOL. <br />  <br />                  MW     A B C D                      MW <br />                                                                         assay D-glutamic acid produced the <br />                                                                         L-isomer converted UDP-MurNAc-L-Ala-D-Glu, <br />                                                                         taking advantage activity D-glutamic-acid- <br />                                                                         adding enzyme present extracts. This procedure <br />           94                                                            allowed detect high level glutamate racemase <br />                                                                         activity extracts strain JM1O9(pTACI2), compared <br />                                                                         practically undetectable level extracts strain <br />           67                                                            JM109(pBTacl) wild-type strains previously inves- <br />                                                                         tigated (11). The specific activities glutamate racemase (in <br />                                                                         nanomoles UDP-MurNAc-L-Ala-[D-14C]Glu synthesized <br />                                                                         hour milligram protein) follows crude <br />                                                                         extracts cells grown LB medium: JM1O9(pBTac1) <br />          43                                                             IPTG, 0.02; JM109(pTACI2) IPTG, <br />                                                                         3.8 (overproduction factor, 190); JM1O9(pTACI2) with <br />                                                                         IPTG, 4.6 (overproduction factor, 230). Evidence that <br />                                                                         [D-14C]glutamic acid synthesized direct racemization <br />                                                                         L-isomer action D-alanine:D- <br />                                                                         glutamic acid transaminase provided different exper- <br />           30                                                            iments. (i) The racemase activity detected an <br />                                                                         extensive <span id='am-29' about='obo:NCBITaxon_124307' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7147 obo:NCBITaxon_7203 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_92613 obo:NCBITaxon_43736 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_125599 obo:NCBITaxon_33392 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_50557 obo:NCBITaxon_6960 obo:NCBITaxon_6072'><span id='am-30'  ></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Dialysis" datatype="xsd:string"></span>dialysis</span> extract eliminated cosub- <br />                                                                         strates putative transaminase activities. It noted <br />                                                                         dialysis accompanied significant loss race- <br />                                                                         mase activity (more 50%), verified was <br />                                                                         related instability enzyme loss <br />                                                                         eventual cofactors substrates. (ii) Addition a-keto- <br />           20                                                            glutarate (1 mM) reaction mixture did significantly <br />                                                                         affect transfer radioactivity isomers glu- <br />    FIG. 4. Overproduction 31K (31,000-molecular-weight) pro-       tamic acid, result indicating clearly a-ketoglutarate <br /> tein crude extracts strain JM109(pTACI2). Strains JM109         intermediate process. (iii) Addition of <br /> (pTACI2) JM109 harboring control vector pBTacl does        D-cycloserine (1 mM) slightly inhibited reaction, <br /> not contain murI gene grown exponentially 370C           previously shown D-alanine:D-gluta- <br /> LB-ampicillin medium. When necessary, IPTG added 1 mM          mate transaminase Bacillus species (24) other <br /> final concentration optical density culture reached <br /> 0.1. Cells harvested 3 h later disrupted sonication.   enzymes D-alanine substrate completely inhib- <br /> As described Materials Methods, protein contents      ited micromolar concentrations antibiotic. <br /> soluble particulate fractions obtained high-speed centrifu-      When direct assay carried UDP- <br /> gation corresponding crude extracts analyzed SDS-        MurNAc-L-Ala, [D- 4C]glutamic acid detectable by <br /> PAGE. Molecular weight (MW) standards (in thousands) indicated          HPLC analysis. The reaction mixture devoid ATP <br /> on left follows: phosphorylase b (94), bovine serum          Mg2&quot;, required conversion D-glu- <br /> albumin (67), ovalbumin (43), carbonic anhydrase (30), soybean      tamic acid nucleotide UDP-MurNAc-L-Ala-D-Glu. <br /> trypsin inhibitor (20). Lanes: A B, analysis soluble         We hypothesized D-glutamic acid good <br /> fractions JM109(pBTacl) cells grown absence       substrate racemase, Km value lower of <br /> presence IPTG, respectively; C D, analysis soluble <br /> fractions JM109(pTACI2) cells grown absence       L-isomer, implying molecule D-glutamic acid <br /> presence IPTG, respectively. The 31K protein corresponding        produced assay rapidly reused the <br /> the murI gene product, highly overproduced extract          reverse synthesis L-isomer, immediately incor- <br /> strain JM109(pTACI2), indicated arrow right. This      porated nucleotide precursor. In presence of <br /> band completely absent corresponding particulate frac-       low concentrations UDP-MurNAc-L-Ala (5 ,uM), at <br /> tions (data shown).                                                 30% radioactivity L-glutamic acid was <br />                                                                         recovered D-isomer form. This effect specific, <br />                                                                         nucleotide precursor UDP-GlcNAc to <br /> this plasmid overproduced relatively high levels murI            UDP-MurNAc-pentapeptide effector added at <br /> gene   product (Fig. 4), detected SDS-PAGE band                 0.1 mM. A purification glutamate racemase now <br /> migrating 31- 32-kDa species, finding agreement            required determine kinetic parameters forward <br /> with value 31,500 calculated DNA sequence               reverse reactions catalyzed enzyme to <br /> (6). This band detected exclusively soluble                  investigate UDP-MurNAc-L-Ala effect detail. <br /> fraction extract. A high-level overproduction                Effects overproduction glutamate racemase the <br /> protein observed absence IPTG (Fig.               pool D-glutamic acid. The overexpression glutamate <br /> 4), probably single copy lacIq did produce             racemase activity JM109(pTACI2) cells ap- <br /> enough repressor molecules efficiently repress tran-             parent effect cell growth bacterial shape. We as- <br /> scription murI gene high-copy-number                   sumed overproduction enzymatic activity in <br /> plasmid.                                                                cell content resulted modification the <br />    Identification murI gene product glutamate               pool levels isomers glutamic acid. It effec- <br /> racemase activity. The apparent overproduction murI              tively observed pool glutamic acid (L plus D) in <br /> gene product prompted assay corresponding enzy-               cells growing exponentially 2YT medium significantly <br /> matic extracts glutamate racemase D-amino acid                  increased presence plasmid (from 19,000 to <br /> transaminase activities. To detect eventual racemase                 27,000 nmol expressed g bacterial dry weight). <br /> activity crude extracts, used convenient coupled          Moreover, D-isomer, normally represents small <br />  VOL. 175, 1993                                     murl FROM E. COLI ENCODES GLUTAMATE RACEMASE ACTIVITY                          2977 <br />  <br />                                                                         D-glutamic acid growth described date: <br />                                                                         strain WM335 E. coli B/r (17, 22). The altered gene, <br />                                                                         named murI, identified previously sequenced open <br />                                                                         reading frame, ORFi (6), lying 90-min region the <br />                                                                         chromosome, btuB gene encoding vitamin <br />                                                                         B12 receptor protein (16) rrsB gene coding 16S <br />                                                                         rRNA (6). It noteworthy btuB murI genes <br />                                                                         considerable 65-bp overlap (1, 6). Interestingly, <br />                                                                         mutations affecting peptidoglycan metabolism were <br />                                                                         earlier located region chromosome near argH <br />  2                                                                      (32, 41), consequently named mrb, murein <br />                                                                         region b, including, particular, mutation conferring <br />                                                                         resistance fosfomycin (41), interpreted an <br />                                                                         alteration murA gene coding PEP:UDP-GlcNAc- <br />                                                                         enolpyruvyl transferase, mutation characterized <br />                                                                         thermosensitive lytic phenotype accumulation <br />                                                                         pool UDP-GlcNAc-enolpyruvate restrictive <br />                                                                         growth temperature (strain ST5 reference 32). In latter <br />      Ci                                                                 case, corresponding altered gene murB encodes the <br />                                                                         UDP-GlcNAc-enolpyruvate reductase recently identi- <br />                                                                         fied (12, 36), 6 kb clockwise respect murI gene, <br />                                                                         rrnB operon birA gene required for <br />                                                                         biotin synthesis (Fig. 1). The cellular functions of the <br />                                                                         genes particular region btuB rpoBC have <br />                                                                         established. The murA gene identi- <br />                                                                         fied date, clear murA, murB, and <br />                                                                         murI genes belong cluster tightly packed genes <br />                                                                         related functions case mur genes <br />                                                                         2-min region E. coli chromosome (28). <br />                                                                            The murI gene E. coli gene selected <br />                                                                         complementation specific defect strain WM335. <br />           0                 10                   20                     Evidence provided essential gene whose <br />                                                                         inactivation results D-glutamic acid auxotrophy. We were <br />                                   minutes                               able construct strain, GRA83, chromosomal <br />    FIG. 5. Separation L- D-isomers (L D, respectively)    murI gene disrupted viable presence of <br /> glutamic acid HPLC derivatization Marfey&apos;s reagent.       plasmid carrying wild-type gene. Since plasmid <br /> Diastereoisomers obtained reaction 1-fluoro-2,4-dinitro-     bears thermosensitive replicon, effects inactiva- <br /> phenyl-5-L-alanine amide separated HPLC LiChro-            tion easily visualized shifting exponentially growing <br /> spher 100CH 18/2 column (Merck). Elution 12 mM                 cells nonpermissive temperature. Cells shown to <br /> ammonium phosphate (pH 5.6) containing 10% acetonitrile. The            accumulate UDP-MurNAc-L-Ala pools nucle- <br /> flow rate 1 ml. mink, column temperature 370C.          otide precursors located downstream pathway were <br /> Elution monitored 340 nm sensitivity 0.04 absorbance <br /> units (full scale). (a) Analysis standard mixture containing 0.5   rapidly depleted. As result, peptidoglycan synthesis was <br /> nmol isomer glutamic acid; (b c) analysis      inhibited cell lysis finally occurred, previously ob- <br /> same conditions pool free glutamic acid isolated         served initial WM335 mutant depleted of <br /> JM109(pBTacl) JM109(pTACI2) cells, respectively. O.D., op-          D-glutamic acid (11). It clear inactivation the <br /> tical density.                                                           sole murI gene sufficient expression D-glutamic- <br />                                                                          acid-requiring phenotype. <br />                                                                            Furthermore, data possible to <br /> percentage (3% case) pool, detected              transduce altered murI gene WM335 or <br /> as 20% total glutamic acid (Fig. 5).                    GRA83 species genetic backgrounds, spite of <br />                                                                          presence high concentration D-glutamic acid the <br />                         DISCUSSION                                       growth medium. Only strains harboring copies the <br />                                                                         murI gene efficient recipients experiments. <br />   D-Glutamic acid essential component bacterial cell            The reason discrepancy wild-type E. coli <br /> wall peptidoglycan practically exclusively encoun-                strains poorly D-glutamic acid, i.e., rate <br /> tered therein. Identification enzymatic process                apparently sufficient fulfill specific <br /> which D-amino acid synthesized bacteria there-             requirements peptidoglycan synthesis. Although D-glu- <br /> fore greatest interest, considered            tamic acid unable complement murl defect of <br /> potential target search new antibacterial com-                GRA83 cells growing 430C rich medium, longer <br /> pounds. In early studies, biosynthesis D-glutamic acid            failed minimal medium. This the <br /> was investigated Bacillus, Lactobacillus, Pediococ-               absence synthetic minimal medium compounds as <br /> cus species, little attention paid             L-glutamic acid compete D-glutamic acid <br /> problem enteric bacteria E. coli medi-              permeation system. As control, shown that <br /> cally important bacterial species.                                       murI defect transduced genetic back- <br />   The genetics D-glutamic acid biosynthesis E. coli            ground chosen, soon transductants selected in <br /> recently investigated mutant requiring           minimal medium supplemented D-glutamic acid. These <br />  2978       DOUBLET ET AL.                                                                                                J. BACTERIOL. <br />  <br /> results confirmed assumption original                              ACKNOWLEDGMENTS <br /> WM335 mutant, initially selected rich medium,            We thank B. Bachmann supplying E. coli strains <br /> necessarily carries additional mutation specific glu-      used work R. Aufrere helpful discussions. <br /> tamate permease gene. Recently, Dougherty et al. (13)                 This work supported grants Centre National la <br /> sequenced gltS gene strain WM335 showed          Recherche Scientifique (URA 1131) Institut National la <br />                                                                    Sante et la Recherche Medicale (900314). P. Doublet was <br /> it effectively altered missense mutations. The          supported scholarship Ministere la Recherche et de <br /> isolation double mutant unfavorable,       la Technologie. <br /> and explain known D-glutam- <br /> ic-acid-requiring strain described date.                                      REFERENCES <br />    Attempts detect glutamate racemase              1. Aufrere, R., M. Tempite, J.-P. Bohin. 1986. Un cas de <br />                                                                        chevauchement cadres lecture chez Eschenchia coli. C.R. <br /> D-alanine:D-glutamate transaminase wild-type E. coli                Acad. Sci. 303:49-54. <br /> strains previously met failure (11, 22, 33, 40). It    2. Bachmann, B. J. 1972. Pedigrees mutant strains of <br /> was assumed enzyme probably expressed                <span id='am-135' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-136' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-138' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia coli K-12</span>. Bacteriol. Rev. 36:525-557. <br /> very low level E. coli cells, confirmed data     3. Bachmann, B. J. 1983. Linkage map Escherichia coli K-12, <br /> from Baliko et al. (4) showed ORFi-encoded              edition 7. Microbiol. Rev. 47:180-230. <br /> protein observed limit detectability cells       4. Baliko, G., A. Raukas, I. <span id='am-98' about='obo:NCBITaxon_295959' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_295956 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="Boros" datatype="xsd:string"></span><span id='am-102'  ></span>Boros</span>, P. Venetianer. 1988. An <br /> were labeled [L-35S]methionine. Evidence provided <br />                                                                        Escherichia coli gene search function. Mol. Gen. Genet. <br />                                                                        211:326-331. <br /> in present study murI gene encodes glutamate         5. Boros, I., A. Kiss, B. Sain, G. Somlyai, P. Venetianer. 1983. <br /> racemase activity. This possible constructing              Cloning promoters Escherichia coli rRNA gene. <br /> strains overproduce murI gene product high               New experimental study regulation rRNA <br /> level developing efficient sensitive assay           transcription. Gene 22:191-201. <br /> for glutamate racemase activity. The enzyme             6. Brosius, J., T. J. Dull, D. D. Sleeter, H. F. Noller. 1981. <br /> greatly unstable, overnight dialysis crude extract        Gene organization primary structure ribosomal RNA <br />                                                                        operon Escherichia coli. J. Mol. Biol. 148:107-127. <br /> was accompanied important loss activity. In         7. Choi, S.-Y., N. Esaki, T. Yoshimura, K. Soda. 1991. <br /> the purification steps glutamate racemase Pedi-            Overproduction glutamate racemase Pediococcuspentosa- <br /> ococcuspentosaceus, Soda coworkers (7, 33, 34) used              ceus Escherichia coli clone cells purification. Prot. <br /> standard buffer containing 1 mM DL-glutamic acid, 10%                  Expression Purification 2:90-93. <br /> glycerol, 0.1% P-mercaptoethanol stabilization       8. Dagert, M., S. D. Ehrlich. 1979. Prolonged incubation in <br /> enzymatic activity. These authors recently reported (7)           calcium chloride improves competence Escherichia coli <br /> the glutamate racemase P. pentosaceus composed              cells. Gene 6:23-28. <br />                                                                     9. Davis, R. W., D. Botstein, J. R. Roth. 1972. Advanced <br /> 265 amino acids molecular weight 29,140. It           bacterial genetics. Cold Spring Harbor Laboratory, Cold Spring <br /> be noted values similar calcn-            Harbor, N.Y. <br /> lated E. coli murI gene product. The search          10. Deguchi, Y., I. Yamato, Y. Anraku. 1989. Molecular cloning <br /> eventual homology amino acid sequences                  gltS gltP, encode glutamate carriers Escherichia <br /> both racemase species required DNA sequence                 coli B. J. Bacteriol. 171:1314-1319. <br /> Pediococcus gene, published date.            11. Doublet, P., J. van Heijenoort, D. Mengin-Lecreulx. 1992. <br />    D-Glutamic acid specific component peptidogly-          Identification Eschenchia coli murI gene, is <br />                                                                        required biosynthesis D-glutamic acid, specific <br /> can structure, macromolecule normally accounts for <br />                                                                        component bacterial peptidoglycan. J. Bacteriol. 174:5772- <br /> 1 2% dry cell weight (30). This minor cellular          5779. <br /> utilization compared L-isomer, important       12. Doublet, P., J. van Heiienoort, D. Mengin-Lecreulx. 1992. <br /> metabolite used pathways particular               Identification murI murB genes coding cytoplasmic <br /> protein synthesis. The pool level D-glutamic acid         peptidoglycan synthetases 90-min region E. coli <br /> very low compared L-isomer,             chromosome, p. 139-146. In M. A. De Pedro, J. V. Holtje, and <br /> case cells overproducing glutamate racemase activity.           W. Loffelhardt (ed.), Bacterial growth lysis: metabolism <br /> It tempting imagine physiological mechanism              structure bacterial sacculus. Plenum Publishing <br />                                                                        Company, London. <br /> exists controls activity glutamate racemase     13. Dougherty, T. J., J. A. Thanassi, M. J. Pucci. 1993. The <br /> avoid excessive conversion L-glutamic acid                Escherichia coli mutant requiring D-glutamic acid result of <br /> D-isomer. An adjustment synthesis D-glutamic acid            mutations distinct genetic loci. J. Bacteriol. 175:111-116. <br /> to requirements peptidoglycan synthesis,             14. Erdei, S., I. Boros, G. Szabo, P. Venetianer. 1983. A novel <br /> particular flow UDP-MurNAc derivatives            type bacterial transcription unit, specifying mRNA, rRNA, <br /> main reaction sequence pathway,                  tRNA. Mol. Gen. Genet. 191:162-164. <br /> considered. In fact, activity glutamate racemase        15. Hamilton, C. M., M. Aldea, B. K. Washburn, P. Babitzke, and <br />                                                                        S. R. Kushner. 1989. New method generating deletions and <br /> was detectable direct-assay conditions UDP- <br />                                                                        gene replacements Escherichia coli. J. Bacteriol. 171:4617- <br /> MurNAc-L-Ala added reaction mixture. Interest-              4622. <br /> ingly, effect specific nucleotide,       16. Heller, K., R. J. Kadner. 1985. Nucleotide sequence the <br /> other precursor UDP-GlcNAc UDP-MurNAc-pen-                     gene vitamin B12 receptor protein outer membrane <br /> tapeptide effector. This result important,         Escherichia coli. J. Bacteriol. 161:904-908. <br /> since suggests activity glutamate racemase      17. Hoffman, B., W. Messer, U. Schwarz. 1972. Regulation of <br /> could modulated vivo concentration           polar cap formation life cycle Escherichia coli. J. <br /> substrate D-glutamic-acid-adding enzyme cata-             Supramol. Struct. 1:29-37. <br />                                                                    18. Holtje, J. V., U. Schwarz. 1985. Biosynthesis growth of <br /> lyzes subsequent step pathway peptidoglycan             murein sacculus, p. 77-119. In N. Nanninga (ed.), Molecular <br /> synthesis. A purification enzyme progress              cytology Escherichia coli. Academic Press, Inc., New York. <br /> control phenomena observed crude enzyme prepa-            19. Kalman, M., D. R. Gentry, M. Cashel. 1991. Characteriza- <br /> rations.                                                               tion Escherichia coli K12 gltS glutamate permease gene. <br />  VOL. 175, 1993                                     murl FROM E. COLI ENCODES GLUTAMATE RACEMASE ACTIVITY                                  2979 <br />  <br />      Mol. Gen. Genet. 225:379-386.                                            characterization enzyme Pediococcus pen- <br /> 20. Kohara, Y., K. Akiyama, K. Isono. 1987. The physical map              tosaceus. Agric. Biol. Chem. 52:3099-3104. <br />      E. coli chromosome: application new strategy      35. Pratviel-Sosa, F., D. Mengin-Lecreulx, J. van HeUenoort. <br />      rapid analysis sorting large genomic library. Cell          1991. Overproduction, purification properties uridine <br />      50:495-508.                                                              diphosphate N-acetylmuramoyl-L-alanine:D-glutamate ligase <br /> 21. Laemmli, U. K., M. Favre. 1973. Maturation head             Escherichia coli. Eur. J. Biochem. 202:1169-1176. <br />      bacteriophage T4. J. Mol. Biol. 80:575-599.                         36. Pucci, M. J., L. F. Discotto, T. J. Dougherty. 1992. Cloning <br /> 22. Lugtenberg, E. J. J., H. J. W. Wjsman, D. van Zaane. 1973.            identification Escherichia coli murB DNA sequence, <br />      Properties D-glutamic acid-requiring mutant Escherichia          encodes UDP-N-acetylenolpyruvoyl-glucosamine reduc- <br />      coli. J. Bacteriol. 114:499-506.                                         tase. J. Bacteriol. 174:1690-1693. <br /> 23. Marfey, P. 1984. Determination D-amino acids. II. Use        37. Rogers, H. J., H. R. Perkins, J. B. Ward. 1980. Biosynthesis <br />      bifunctional reagent, 1,5-difluoro-2,4-dinitrobenzene. Carlsberg         peptidoglycan, p. 239-297. In Microbial cell walls mem- <br />      Res. Commun. 49:591-596.                                                 branes. Chapman &amp; Hall, Ltd., London. <br /> 24. Martinez-Carrion, M., W. T. Jenkins. 1965. D-Alanine-D-          38. Sambrook, J., E. F. Fritsch, T. Maniatis. 1989. Molecular <br />      glutamate transaminase. II. Inhibitors mechanism              cloning: laboratory manual, 2nd ed. Cold Spring Harbor <br />      transamination D-amino acids. J. Biol. Chem. 240:3547-3552.           Laboratory, Cold Spring Harbor, N.Y. <br /> 25. Mazaitis, A. J., S. Palchaudhuri, N. Glansdorff, W. K.           39. Shapira, S. K., J. Chou, F. V. Richand, M. J. Casadaban. <br />      Maas. 1976. Isolation characterization XdargECBH trans-           1983. New versatile plasmid vectors expression <span id='am-140' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-141' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-142' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-144'  ></span>hybrid</span> <br />      ducing phages heteroduplex analysis argECBH                   proteins coded cloned gene fused lacZ gene sequences <br />      cluster. Mol. Gen. Genet. 143:185-196.                                   encoding enzymatically active carboxy-terminal portion of <br /> 26. Mengin-Lecreulx, D., B. Flouret, J. van HeUenoort. 1982. <br />      Cytoplasmic steps peptidoglycan synthesis Escherichia              0-galactosidase. Gene 25:71-82. <br />      coli. J. Bacteriol. 151:1109-1117.                                  40. Tanizawa, K., S. Asano, Y. Masu, S. Kuramitsu, H. Kag- <br /> 27. Mengin-Lecreulx, D., B. Flouret, J. van Heuenoort. 1983.              amiyama, H. Tanaka, K. Soda. 1989. The primary structure <br />      Pool levels UDP-N-acetylglucosamine UDP-N-acetylglu-              thermostable D-amino acid aminotransferase thermo- <br />      cosamine-enolpyruvate Escherichia coli correlation           philic Bacillus species correlation L-amino acid <br />      peptidoglycan synthesis. J. Bacteriol. 154:1284-1290.                    aminotransferases. J. Biol. Chem. 264:2450-2454. <br /> 28. Mengin-Lecreulx, D., C. Parquet, L. R. Desviat, J. Pla, B.           41. Venkateswaran, P. S., H. C. Wu. 1972. Isolation and <br />      Flouret, J. A. Ayala, J. van Heijenoort. 1989. Organization          characterization phosphonomycin-resistant mutant Esch- <br />      murE-murG region Escherrchia coli: identification           erichia coli K-12. J. Bacteriol. 110:935-944. <br />      murD gene encoding D-glutamic-acid-adding enzyme. J.        42. Wallace, B., Y.-J. Yang, J. Hong, D. Lum. 1990. Cloning <br />      Bacteriol. 171:6126-6134.                                                sequencing gene encoding glutamate aspartate <br /> 29. Mengin-Lecreulx, D., E. Siegel, J. van Heienoort. 1989.               carrier Escherichia coli K-12. J. Bacteriol. 172:3214-3220. <br />      Variations UDP-N-acetylglucosamine UDP-N-acetyl-             43. Wasserman, S. A., C. T. Walsh, D. Botstein. 1983. Two <br />      muramyl-pentapeptide pools Escherichia coli inhibition          alanine racemase genes <span id='am-75' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-76' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-77'  ></span><span id='am-78' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Salmonella</span> typhimunium differ in <br />      protein synthesis. J. Bacteriol. 171:3282-3287.                       structure function. J. Bacteriol. 153:1439-1450. <br /> 30. Mengin-Lecreulx, D., J. van Heienoort. 1985. Effect           44. Wild, J., J. Hennig, M. Lobocka, W. Walczak, T. Klopo- <br />      growth conditions peptidoglycan content cytoplasmic               towski. 1985. Identification dadX gene coding the <br />      steps biosynthesis Escherichia coli. J. Bacteriol. 163:        predominant isozyme alanine racemase Eschenichia coli <br />      208-212.                                                                 K12. Mol. Gen. Genet. 198:315-322. <br /> 31. Miller, J. H. 1972. Experiments molecular genetics. Cold          45. Winans, S. T., S. J. Elledge, J. H. Krueger, G. C. Walker. <br />       Spring Harbor Laboratory, Cold Spring Harbor, N.Y.                       1985. Site-directed insertion deletion mutagenesis with <br /> 32. Miyakawa, T., H. Matsuzawa, M. Matsuhashi, Y. Sugino.                 cloned fragments Eschenchia coli. J. Bacteriol. 161:1219- <br />       1972. Cell wall peptidoglycan mutants Escherichia coli K-12:          1221. <br />       existence clusters genes, mra mrb, cell wall     46. Yanisch-Perron, C., J. Vieira, J.  <br /> </body></html>