<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>  <br />  <br />  <br />                            Systematic genetic analysis of virulence in the human fungal <br />                            pathogen <span id='am-36' about='obo:NCBITaxon_5207' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_5204 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5215 obo:NCBITaxon_5206 obo:NCBITaxon_5234 obo:NCBITaxon_2759 obo:NCBITaxon_552466 obo:NCBITaxon_155616'><span id='am-37' property="rdfs:label" content="Cryptococcus neoformans" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Filobaxidiella neoformans" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces neoformans" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Debaryomyces neoformans" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Torulopsis neoformans" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Filobasidiella neoformans" datatype="xsd:string"></span><span id='am-44'  ></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Torula neoformans" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Filobasidiella neoformornans" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Lipomyces neoformans" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Blastomyces neoformans" datatype="xsd:string"></span>Cryptococcus neoformans</span> <br />  <br />                             SUMMARY <br />                                 The fungus Cryptococcus neoformans is a leading cause of mortality and morbidity among HIV- <br />                                 infected individuals. We utilized the completed genome sequence and optimized methods for <br />                                 homologous DNA replacement using high-velocity particle bombardment to engineer 1,201 gene <br />                                 knockout mutants. We screened <span id='am-61' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-64'  ></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>this</span> resource in vivo for proliferation in murine lung tissue and in <br />                                 vitro for three well-recognized virulence attributes &#8212; polysaccharide capsule formation, <br />                                 melanization, and growth at body temperature. We identified dozens of previously uncharacterized <br />                                 genes that affect these known attributes as well as 40 infectivity mutants without obvious defects in <br />                                 these traits. The latter mutants affect predicted regulatory factors, secreted proteins, and immune- <br />                                 related factors, and represent powerful tools for elucidating novel virulence mechanisms. In <br />                                 particular, we describe a GATA <span id='am-33' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> transcription factor that inhibits phagocytosis by murine <br />                                 macrophages independently of the capsule, indicating a previously unknown mechanism of innate <br />                                 immune modulation. <br />  <br />  <br />                            INTRODUCTION <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Systematic profiling of gene deletion strain collections has been essential for the <br />                                               comprehensive study of biological processes in the nonpathogenic yeast <span id='am-17' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-18' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-21'  ></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span> <br />                                               cerevisiae (Bader et al., 2003). Likewise, forward genetic screens of large mutant collections <br />                                               of bacterial pathogens have been extremely fruitful in identifying virulence pathways in <br />                                               prokaryotic organisms (Saenz and Dehio, 2005). In contrast, the study of fungal pathogenesis <br />                                               has historically been hindered by a relative lack of genetic tools and techniques in these <br />                                               eukaryotic <span id='am-71' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>. With the release of complete genome sequences for numerous fungal <br />                                               pathogens (Jones et al., 2004; Loftus et al., 2005; Nierman et al., 2005), there are now <br />                                               unprecedented opportunities to create genome-scale resources and to apply systematic methods <br />                                               to the study of fungal virulence. <br />  <br />  <br />  <br />                            4 Correspondence, hiten@biochem.ucsf.edu tel (415) 514-0594, fax (415) 502-4315. <br />                            Publisher&apos;s Disclaimer: This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers <br />                            we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting <br />                            proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could <br />                            affect the content, and all legal disclaimers that apply to the journal pertain. <br />                             Liu et al.                                                                                                 Page 2 <br />  <br />  <br />                                            Cryptococcus neoformans is the most common cause of systemic fungal infections in people <br />                                            infected with HIV (Bicanic and Harrison, 2004). Whereas primary pulmonary infections are <br />                                            typically mildly symptomatic and self-resolving in patients with intact immune systems, these <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            infections commonly disseminate through the bloodstream to infect the meninges in HIV- <br />                                            positive patients. In sub-Saharan Africa and Southeast Asia, where over 80% of the world&#8217;s <br />                                            HIV-infected population lives, studies indicate that 10&#8211;40% of HIV-related deaths are caused <br />                                            by cryptococcal meningitis (Corbett et al., 2002; French et al., 2002). The high cost and limited <br />                                            availability of antifungal drugs often prohibit effective medical treatment in these areas <br />                                            (Warnock, 2006) and mortality rates in parts of sub-Saharan African approach 100% (Bekondi <br />                                            et al., 2006). <br />  <br />                                            There are three well-established mediators of C. neoformans virulence: a polysaccharide <br />                                            capsule, cell wall-associated melanin, and the ability to grow at human body temperature. The <br />                                            capsule has been shown to have both anti-phagocytic and immunomodulatory properties (Del <br />                                            <span id='am-53' about='obo:NCBITaxon_1126785' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_95194 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-54' property="rdfs:label" content="Poeta" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57'  ></span>Poeta</span>, 2004; Monari et al., 2006). Melanin is believed to protect the fungus from a broad range <br />                                            of toxic insults, both in the environment and the host (Nosanchuk and Casadevall, 2003). <br />  <br />                                            Molecular studies of C. neoformans virulence pathways have focused almost exclusively on <br />                                            the regulation of capsule and melanization (Table S1). Attempts to identify novel virulence <br />                                            mechanisms using random insertional mutagenesis have been hampered by biases in the <br />                                            insertion sites, chromosomal rearrangements, and relatively low ORF hit rates (Idnurm et al., <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            2004;Nelson et al., 2001). The release of the complete C. neoformans genome sequence now <br />                                            makes it possible to use directed gene knockout mutants to screen for novel virulence <br />                                            mechanisms. To this end, we constructed a large library of 1,201 signature-tagged, targeted <br />                                            gene deletion strains. Below, we describe insights gained from systematic profiling of this <br />                                            library in a murine infection model and in three in vitro assays for virulence-associated <br />                                            attributes (Figure 1). We also describe detailed analysis of a key transcriptional regulator that <br />                                            has pointed to the existence of an unsuspected capsule-independent mechanism for inhibition <br />                                            of phagocytosis. <br />  <br />                                RESULTS <br />                                Generation of signature-tagged C. neoformans gene deletion collection <br />                                            Approximately 1,500 genes in C. neoformans were targeted for deletion by homologous <br />                                            recombination. The majority (~900) were selected because of the absence of obvious homologs <br />                                            in the nonpathogenic fungus S. cerevisiae. The remainder encode proteins with sequence motifs <br />                                            implicating them in a broad variety of cellular processes. Because these targets were chosen <br />                                            prior to the availability of detailed gene models and were based on BLAST-based annotations <br />                                            (Chow et al., 2007), there was a <span id='am-6' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Bias" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10'  ></span>bias</span> against genes that are C. neoformans-specific. We utilized <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            fusion PCR to create deletion constructs in a high-throughput manner and optimized biolistic <br />                                            transformation for efficient recombination. For reasons described below, each mutant strain <br />                                            contains one of 48 unique signature tag DNA sequences adjacent to the deletion marker. The <br />                                            resulting set of 1,201 verified mutants represents deletions of 1,180 genes (in some cases, <br />                                            different domains of the same gene were targeted separately). For comparison, the genome of <br />                                            C. neoformans is predicted to contain ~6,500 genes (Loftus et al., 2005), and ~190 targeted <br />                                            gene deletions in C. neoformans have previously been reported in the literature (Table S1). <br />                                            The deletion collection is currently available without restriction through the Fungal Genetic <br />                                            Stock Center (www.fgsc.org) or ATCC (www.atcc.org). <br />  <br />                                Quantitative screen for infectivity mutants in a murine inhalation model of disease <br />                                            To identify C. neoformans genes involved in infectivity, we adopted signature-tagged <br />                                            mutagenesis (STM), a strategy that enables high-throughput, parallel analysis of pathogen <br />  <br />  <br />                                                    Cell. Author manuscript; available in PMC 2009 October 3. <br />                             Liu et al.                                                                                                     Page 3 <br />  <br />  <br />                                            mutants in experimental infections (Saenz and Dehio, 2005). In this method, individual mutants <br />                                            are marked in their genomes with a unique DNA sequence (termed a signature tag or a barcode). <br />                                            This allows a pool of many mutants to be tested in a single animal infection by comparing the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            relative representation of each mutant in the pool before and after selection in the host. In <br />                                            practice, this is accomplished by isolating DNA from mutant pathogen pools before and after <br />                                            infection and quantifying the abundance of each tag in the sample. <br />  <br />                                            We developed a robust, quantitative STM assay utilizing a previously established mouse model <br />                                            of pulmonary cryptococcal infection (Cox et al., 2000). C. neoformans deletion mutants were <br />                                            grown independently and then pooled into groups of 48 for intranasal infection of mice. <br />                                            Infections were allowed to progress until the mice displayed signs of significant morbidity <br />                                            (~5&#8211;6 weeks), at which time the mice were sacrificed and C. neoformans cells were recovered <br />                                            from the lung tissue. Quantitative PCR using tag-specific primers together with a common <br />                                            primer in the gene deletion cassette was used to measure the relative abundance of each mutant <br />                                            in both the input and recovered pools. The ratio of these two values ([recovered] / [input]) was <br />                                            calculated on a log2 scale and termed the STM score (see Experimental Procedures). As shown <br />                                            in Figure 2A, STM scores demonstrated remarkable consistency among the three mice infected <br />                                            with each pool of signature-tagged mutants. In total, 1,100 deletion mutants were screened <br />                                            using the STM assay. <br />  <br />                                            We next determined if the STM score for a given mutant was affected by the composition of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            mutants in that pool. 104 mutants were removed from their original pools and sorted into new <br />                                            pools containing different mutants. Infections were repeated and STM scores were recalculated <br />                                            for each mutant. The original STM scores were highly correlated with the re-pooled STM <br />                                            scores (R2=0.84), demonstrating that the STM scores were largely independent of the identities <br />                                            of the competing mutants and were highly reproducible from pool to pool (Figure 2B). <br />  <br />                                            As an additional control for reproducibility of the assay, we constructed 43 independent <br />                                            disruptions of the gene SXI1, which is required for mating but dispensable for virulence (Hull <br />                                            et al., 2004); each mutant was marked with a different signature tag. The 43 sxi1&#916; mutant <br />                                            strains were tested a total of 293 times throughout the STM screen and, as expected, the <br />                                            distribution of STM scores for the sxi1&#916; strains was centered near zero (median=0.1) and <br />                                            showed a tight range (SD=0.5) (Figure 2C). The distribution of STM scores for the deletion <br />                                            library as a whole was also centered on zero (median=0.0); however, in this case, the <br />                                            distribution of scores was much broader (SD=3.3), suggesting a wide range of fitness in the <br />                                            assay. <br />  <br />                                STM phenotypes correlate with virulence <br />                                            STM screens are designed to measure the relative competitive fitness of mutants in a mixed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            infection. Thus, the term &#8220;infectivity&#8221; in this context refers to this ability to proliferate/survive <br />                                            within host tissue. &#8220;Virulence,&#8221; on the other hand, is classically defined as the ability of an <br />                                            individual strain not only to proliferate, but to cause disease as well. To assess the ability of <br />                                            our STM screen phenotypes to predict virulence defects, we performed two additional tests. <br />  <br />                                            First, we asked whether previously identified virulence mutants included in our deletion <br />                                            collection were detected by the screen. Among 24 previously-reported virulence mutants, 21 <br />                                            displayed STM scores significantly below zero, ranging from &#8722;12.8 to &#8722;1.9 (labeled on the x- <br />                                            <span id='am-66' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-70'  ></span>axis</span> of Figure 2C and Table S2). Two published virulence mutants, lac1&#916; and ure1&#916;, had <br />                                            normal STM scores. These mutants, however, are known to replicate normally in mouse lungs <br />                                            and have late defects in dissemination (Noverr et al., 2004; Olszewski et al., 2004); such <br />                                            mutants would not be anticipated to have defects in our screen. The final virulence mutant, <br />                                            scp1&#916;, was retested in our assay using a different pool of mutants, and a significantly lower <br />                                            STM score was detected (&#8722;3.6 vs. &#8722;0.8). These results demonstrate a robust sensitivity of the <br />  <br />  <br />                                                     Cell. Author manuscript; available in PMC 2009 October 3. <br />                             Liu et al.                                                                                                 Page 4 <br />  <br />  <br />                                            STM screen for virulence genes known to affect propagation in pulmonary tissue (21 mutants <br />                                            identified of 22 mutants with known lung defects). Conversely, the sxi1&#916; STM <span id='am-87' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-88' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-89'  ></span><span id='am-90' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> described <br />                                            above did not yield a single negative outlier and all seven mutants in the collection containing <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            deletions of genes known to be dispensable for virulence had normal STM scores suggesting <br />                                            a screen specificity for infectivity defects approaching 100% (Figure 2C, Table S2). <br />  <br />                                            In a second test, we conducted classical time-to-endpoint survival analyses for seven <br />                                            previously-uncharacterized mutants with a wide range of STM scores. In these experiments, <br />                                            virulence was directly measured by determining the rate at which mice infected with a single <br />                                            strain of C. neoformans reach a defined disease endpoint (see Supplementary Materials). <br />                                            Monotypic intranasal infections of 8&#8211;10 mice per mutant and a wild-type control were <br />                                            performed and the time to significant morbidity was assessed for each mouse. Survival curves <br />                                            were plotted for the mutant strains and compared to that of the wild-type control. Two mutants <br />                                            with STM scores close to zero were indistinguishable from wild-type in these survival curves <br />                                            (Figure 2D). Four mutants with STM scores significantly below zero were attenuated for <br />                                            virulence (Figure 2E) with the degree of attenuation closely correlated with the magnitude of <br />                                            the STM defect. Interestingly, a mutant with an STM score significantly above zero <br />                                            (zap103&#916;) was hypervirulent (Figure 2E). Together, these studies suggest that the STM assay <br />                                            is a sensitive and specific measure for infectivity and that in this pulmonary infection model <br />                                            of cryptococcosis, altered infectivity correlates well with altered virulence. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Based on these results, we chose STM scores of &lt; &#8722;2.5 and &gt; +2.5 as the cutoff points for <br />                                            reduced infectivity and increased infectivity, respectively. Application of these criteria to our <br />                                            data yielded 164 (15%) mutants in the former category and 33 (3%) in the latter. <br />  <br />                                Systematic in vitro phenotyping of the deletion collection identifies mutants defective in <br />                                known virulence attributes <br />                                            To complement the in vivo STM screen, we systematically assayed the deletion collection in <br />                                            vitro for three known virulence-associated phenotypes: 1) growth in minimal medium at 37&#176; <br />                                            C, 2) melanization, and 3) synthesis of the polysaccharide capsule. <br />  <br />                                            A high-throughput, 96-well format assay was developed to measure the growth of each mutant <br />                                            in synthetic minimal medium (YNB) at 37&#176;C. As expected, a large overlap existed between <br />                                            mutants that displayed poor growth at 37&#176;C and mutants with STM defects (P=8.2E-30) (Figure <br />                                            2F). Using hypergeometric distribution analysis, we determined that mutants that displayed <br />                                            growth scores greater than &#8722;0.5 did not show a significant enrichment for mutants with <br />                                            virulence phenotypes (P &gt; 0.01) (Figure S1); thus, we used a growth score of &#8722;0.5 as a cutoff <br />                                            for determining which mutants were categorized as having significant growth defects. Some <br />                                            mutants displayed severe growth defects but not STM defects. We attributed at least a portion <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            of this inconsistency to the fact that minimal medium does not perfectly mimic the nutritional <br />                                            environment found within the host. For example, mutants in peroxisome function (pex1&#916; and <br />                                            pex6&#916;) have been shown previously to have normal virulence despite being unable to grow on <br />                                            YNB with glucose as the sole carbon source (Idnurm et al., 2007). <br />  <br />                                            To identify mutants defective in melanin synthesis, we grew the deletion collection on agar <br />                                            plates containing L-DOPA (a diphenolic substrate). Wild-type cells turned dark brown over <br />                                            2&#8211;3 days as they synthesized and accumulated melanin. Thirty-eight deletion mutants <br />                                            displayed decreased levels of melanization compared to wild-type, including known melanin <br />                                            mutants such as lac1&#916;, cac1&#916;, and pka1&#916;.(Alspaugh et al., 2002; D&apos;Souza et al., 2001; Noverr <br />                                            et al., 2004). As noted above, the lac1&#916; mutant (which lacks the laccase primarily responsible <br />                                            for melanin synthesis) has late defects in dissemination from the lungs but grows normally in <br />                                            lung tissue (Noverr et al., 2004); as such, it was not identified by the STM screen. Likewise, <br />                                            a mutant containing a deletion of the second C. neoformans laccase gene, LAC2, as well as a <br />  <br />  <br />                                                    Cell. Author manuscript; available in PMC 2009 October 3. <br />                             Liu et al.                                                                                               Page 5 <br />  <br />  <br />                                           double knockout mutant, lac1&#916; lac2&#916;, did not display STM phenotypes (Table S2). This is <br />                                           consistent with melanization playing a role in dissemination, but not lung infectivity. Thus, we <br />                                           were surprised to note that overall there was a significant overlap between mutants with <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           melanization defects and those with STM defects in the lungs (Figure 2F). This association <br />                                           remained significant (P =1.4E-11) even after excluding mutants with growth defects from the <br />                                           analysis. Given that the laccases were dispensable for lung infection, additional defects beyond <br />                                           poor pigment production must be proposed in this subset of melanin-defective mutants to <br />                                           account for their lung infectivity defects. These deficits may affect co-regulated pathways that <br />                                           function independently of and/or redundantly with melanization to promote lung proliferation. <br />  <br />                                           To identify mutants defective in polysaccharide capsule formation, we screened for mutants <br />                                           with a dry colony appearance on rich medium (YPAD) agar plates. A dry colony morphology <br />                                           is characteristic of most known C. neoformans mutants with absent or greatly reduced amounts <br />                                           of capsule (Griffith et al., 2004; Moyrand et al., 2004; Moyrand and Janbon, 2004). Using this <br />                                           simple assay, we identified eleven potentially capsule-defective mutants including six strains <br />                                           (cap59&#916;, cap64&#916;, cap60&#916;, cap10&#916;, cas35&#916;, and nrg1&#916;) that contain deletions in genes <br />                                           previously implicated in capsule synthesis (Cramer et al., 2006; Moyrand et al., 2004; <br />                                           Okabayashi et al., 2007). In addition, PBX1 and PBX2 were first identified in this screen; <br />                                           deletion of PBX1 or PBX2 affects the fidelity of polysaccharide synthesis and their detailed <br />                                           analysis has been described elsewhere (Liu et al., 2007). As expected, all mutants with dry <br />                                           colony morphologies also had STM defects, consistent with the well-established importance <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           of the capsule for causing disease (Figure 2F). <br />  <br />                                           Certain mutants, such as pka1&#916; and cac1&#916;, are known to produce reduced or structurally- <br />                                           altered capsules (Alspaugh et al., 2002; D&apos;Souza et al., 2001), but do not have dry colony <br />                                           morphologies. To identify this <span id='am-58' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-59' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> of mutant, we tested a subset of mutants in the collection <br />                                           for the ability to induce capsule formation using an India ink assay. This stain yields a <br />                                           characteristic halo around cells with enlarged capsules (we note this analysis does not rule out <br />                                           subtle capsule structure defects that would otherwise only be detectable by NMR analysis). <br />                                           We tested 82 mutants that displayed melanin and/or STM defects but normal colony <br />                                           morphology. Only one mutant, containing a deletion of GAT201 (which encodes a GATA- <br />                                           family zinc finger DNA-binding transcriptional regulator, Figure S2), produced no visible <br />                                           capsule by India ink staining under capsule-inducing conditions (Figure 4C). Further <br />                                           investigation of the role of GAT201 in C. neoformans pathogenesis is described below. <br />  <br />                                           In total, among 164 mutants with STM defects, 85 also had defects in growth at 37&#176;C, <br />                                           polysaccharide capsule formation, and/or melanization. The remaining 79 mutants appeared <br />                                           normal in all three in vitro assays (Figure 2F). To verify that the observed phenotypes were <br />                                           linked to the targeted deletions, we reconstructed 107 deletion strains and retested for the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           original phenotype: 5 of 5 reconstructed strains reproduced the original capsule phenotype; 33 <br />                                           of 35 reproduced the original melanin phenotype; 53 of 67 reproduced an STM score <br />                                           significantly below zero; and 8 of 11 reproduced an STM score significantly above zero (Figure <br />                                           S3). The results overall indicate strong linkage between the targeted mutation and the observed <br />                                           phenotypes. Slightly higher variability in reproduction of STM phenotypes may reflect both <br />                                           higher variability of the assay and the potential for a larger number of unlinked mutations to <br />                                           affect this phenotype. <br />  <br />                                Novel melanin and capsule genes <br />                                           Thirty-three novel genes that regulate melanization and five novel genes that regulate capsule <br />                                           formation were identified and verified by knockout reconstruction and retesting (Figure 3, <br />                                           Table 1). As described previously (Chow et al., 2007), gene names were assigned based on <br />                                           homology to known S. cerevisiae genes when available. Four melanin genes and one capsule <br />                                           gene could not be named using this approach and, instead, were designated MLN1&#8211;MLN4 and <br />  <br />                                                   Cell. Author manuscript; available in PMC 2009 October 3. <br />                             Liu et al.                                                                                                 Page 6 <br />  <br />  <br />                                            CPL1 respectively. The melanin and capsule genes identified in these screens implicate a <br />                                            number of novel pathways in the regulation of these two virulence attributes. For example, <br />                                            mutants that lack the transcription factor Rim101 and its predicted upstream regulator Rim20 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            both display melanization defects (Figure S3). Similarly, deletion of components of the Set3 <br />                                            deacetylase complex resulted in significantly larger capsules compared to wild-type as well as <br />                                            defects in melanization (Figure S3&#8211;S4). <br />  <br />                                Novel infectivity genes <br />                                            A particularly interesting category of mutants identified in these screens had defects in lung <br />                                            infectivity but were normal for growth, capsule, and melanin. Forty of these mutants were <br />                                            verified by knockout reconstruction and retesting using the STM assay (Figure 3, Table 2). <br />                                            Again, gene names were assigned based on homology to known S. cerevisiae genes when <br />                                            available. Fifteen unnamed genes were designated LIV1&#8211;LIV15 (Lung InfectiVity). <br />  <br />                                            In addition to capsule, melanin, and growth, a number of other less-studied factors have been <br />                                            implicated in C. neoformans virulence based on genetic studies. These include urease, <br />                                            alternative oxidase, phospholipase, reductive iron uptake, glucosylceramide synthesis, and <br />                                            resistance to nitrogen and oxygen species (Akhter et al., 2003; Cox et al., 2003; Cox et al., <br />                                            2001; Cox et al., 2000; de Jesus-Berrios et al., 2003; Jung et al., 2008; Shea et al., 2006). In <br />                                            most cases, mutants in these pathways display growth defects on certain media. For example, <br />                                            mutants lacking the phospholipase gene (PLB1) are sensitive to SDS (Siafakas et al., 2007), <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            mutants lacking the flavohemoglobin gene (FHB1) are sensitive to acidified nitrite, and <br />                                            mutants lacking the alternative oxidase gene (AOX1) are sensitive to potassium cyanide (KCN). <br />                                            Thus, in <span id='am-74' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-75' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> to identify potential roles for these infectivity genes, we systematically tested <br />                                            the 40 mutants described above for growth on five media conditions as well as for urease <br />                                            activity using an enzymatic assay. This analysis revealed five mutants with sensitivity to <br />                                            acidified nitrite, two with sensitivity to SDS, and two with sensitivity to KCN (Table 2, Figure <br />                                            S5&#8211;S7). In contrast, none of the mutants displayed decreased growth in pH 4 media (Figure <br />                                            S5) or in low ferrous iron conditions (Figure S8) and none of the mutants displayed a defect <br />                                            in urease activity (Figures S9). <br />  <br />                                            Two of the mutants that displayed increased sensitivity to acidified nitrite contain deletions of <br />                                            DNA repair genes (rad54&#916; and rad23&#916;), suggesting that unrepaired DNA damage, perhaps <br />                                            via free radical attack by phagocytes, can be deleterious to infection (Figure S5). Notably, a <br />                                            mutant containing a deletion of LIV8, which encodes an RGS (Regulator of G-protein <br />                                            Signaling) domain, was exquisitely sensitive to acidified nitrite, SDS, and KCN (Figures S5&#8211; <br />                                            S7). We note that since these media assays are relatively nonspecific, the sensitivities observed <br />                                            may or may not explain the virulence defects of particular mutants. For example, a mutant <br />                                            containing a deletion of the virulence gene SRE1, which has been shown to be involved in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            ergosterol biosynthesis (Chang et al., 2007; Chun et al., 2007), displayed sensitivity to both <br />                                            SDS and KCN (Figures S6&#8211;S7). <br />  <br />                                            Nonetheless, the 33 mutants that did not display phenotypes in any of these in vitro assays <br />                                            represent the genes that are the strongest candidates for novel C. neoformans virulence factors. <br />                                            Seven genes in this category encode predicted sequence-specific transcription factors. Another <br />                                            five genes encode factors likely involved in chromatin regulation. Of particular note, three <br />                                            genes encode factors with strong homology to proteins previously implicated in pathogenesis <br />                                            or regulation of the human immune response (Figure 3, Table 2). LIV5 encodes a protein <br />                                            homologous to one required for pathogenesis by the plant fungal pathogen <span id='am-1' about='obo:NCBITaxon_148303' typeof='obo:NCBITaxon_222544 owl:Thing obo:NCBITaxon_81093 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_147550 obo:NCBITaxon_715989 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_639021 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_2759'><span id='am-2' property="rdfs:label" content="Magnaporthe" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Magnaporthe</span> <br />                                            grisea (Fujimoto et al., 2002). LIV7 encodes a putative secreted protein that is homologous to <br />                                            a human protein induced by lipopolysaccharide. Finally, LIV6 encodes a protein highly <br />                                            homologous to the human T-cell immunomodulatory protein (TIP). Secreted TIP has been <br />                                            shown to directly regulate cytokine production by T-cells (Fiscella et al., 2003). Protein <br />  <br />                                                    Cell. Author manuscript; available in PMC 2009 October 3. <br />                             Liu et al.                                                                                                   Page 7 <br />  <br />  <br />                                            structure prediction indicated that Liv6 contains FG-GAP repeats, which are commonly found <br />                                            in alpha integrins, as well as a signal peptide sequence, suggesting that it may be secreted <br />                                            (Figure S2). Deletion of LIV5 and LIV6 were confirmed to result in virulence defects by <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            survival curve analysis (Figure 2E, Figure S10). <br />  <br />                                            We also identified mutants that reproducibly showed significant increases in the STM assay. <br />                                            Eight mutants were confirmed by knockout reconstruction and retesting to display enhanced <br />                                            colonization of lung tissue (Table 2). These include deletions of genes encoding two members <br />                                            of Rho signaling pathways (RGD1 and RHO104), two kinases (RCK2 and KSP1), and a <br />                                            transcription factor (ZAP103). One unnamed gene was designated LIV16. Deletions of <br />                                            ZAP103 and RGD1 were confirmed to result in increased virulence by survival curve analysis <br />                                            (Figure 2E, Figure S10). <br />  <br />                                Gat201: a key regulator of virulence and capsule-independent anti-phagocytic mechanisms <br />                                            Among mutants with normal growth at 37&#176;C, the strain containing a deletion of GAT201 <br />                                            exhibited the largest decrease in the STM screen (STM score = &#8722;18.3). Indeed, the gat201&#916; <br />                                            mutant strain was avirulent in monotypic infection of mice using our standard inoculum (Figure <br />                                            4A). This mutant was chosen for further investigation. <br />  <br />                                            As described above, capsules of gat201&#916; cells could not be visualized by India ink staining <br />                                            under capsule-inducing conditions (Figure 4B). However, a more sensitive assay using a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            monoclonal antibody to stain the main capsular polysaccharide, glucoronoxylomannan, <br />                                            detected capsule polysaccharide on the cell surface of gat201&#916; cells (Figure 5B). These data <br />                                            suggest GAT201 is not required for production of basal levels of capsule polysaccharide but is <br />                                            needed for proper induction of large capsules. gat201&#916; cells were also hypermelanized when <br />                                            grown on L-DOPA plates (Figure 4C). Previously-identified hypermelanized mutants have <br />                                            been associated with both increased and decreased virulence and often display additional <br />                                            phenotypes such as altered capsule production (Bahn et al., 2005; Wang et al., 2004). Similarly, <br />                                            our studies indicate GAT201 participates directly or indirectly in the regulation of both capsule <br />                                            induction and melanization. <br />  <br />                                            Consistent with this notion, overexpression of GAT201 in wild-type cells using the promoter <br />                                            of a constitutive ribosomal protein gene (pRPL2b) was sufficient to induce large capsule <br />                                            formation, even in normally non-inducing growth conditions (Figure 4B, Figure S11). As <br />                                            shown in Figure 4C, the pRPL2b::GAT201 strain also displayed a slight melanization defect. <br />                                            The pRPL2b::GAT201 strain was hypervirulent in mice after monotypic intranasal infection <br />                                            (Figure 4A), demonstrating that overexpression of GAT201 is sufficient to activate key <br />                                            virulence pathways. A reconstructed gat201&#916; strain recapitulated the phenotypes of the original <br />                                            gat201&#916; strain in these assays (Figure 4A&#8211;4C). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Whole-genome, microarray-based expression profiling of the pRPL2b::GAT201 strain <br />                                            identified 543 genes that were significantly upregulated and 24 genes that were significantly <br />                                            downregulated compared to a wild-type strain (Figure 4D). Consistent with the enhanced <br />                                            nutritional and metabolic requirements necessary for ectopic polysaccharide capsule <br />                                            formation, the induced set contained a substantial number of genes encoding proteins involved <br />                                            in carbohydrate regulation, including sugar transporters (40 genes), glycosyl hydrolases (8 <br />                                            genes), and glycosyl transferases (5 genes). In addition, twenty-five genes that encode DNA- <br />                                            binding transcription factors were upregulated in the pRPL2b::GAT201 strain, suggesting that <br />                                            Gat201 exerts at least a portion of its effects by activating the expression of other transcription <br />                                            factors. Using a previously described annotation database (Chow et al., 2007), we identified <br />                                            C. neoformans genes that do not have homologs in nine sequenced ascomycetes and two <br />                                            sequenced basidiomycetes. Over 29% of Gat201-upregulated genes (159 genes) fall into this <br />                                            C. neoformans-specific gene class, which represents a significant enrichment over their general <br />  <br />  <br />                                                    Cell. Author manuscript; available in PMC 2009 October 3. <br />                             Liu et al.                                                                                              Page 8 <br />  <br />  <br />                                         representation on the microarray (P =1.2E-35). Sixteen of these genes are predicted to produce <br />                                         proteins with signal sequences but no transmembrane domain; these potential secreted effectors <br />                                         include all members of a unique four-gene family (CNF2230, CNF2240, CNF2250, and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         CND5620). <br />  <br />                                         To further probe the role of Gat201 in virulence, we examined the interaction of gat201&#916; cells <br />                                         with RAW264.7 mouse macrophages. Phagocytic cells play a central role in the host immune <br />                                         response against C. neoformans, and the ability of C. neoformans to avoid phagocytosis and <br />                                         killing by these cells is crucial to pathogenesis (Del Poeta, 2004). In contrast to most <span id='am-24' about='obo:NCBITaxon_4751' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-25' property="oboInOwl:hasExactSynonym" content="fungi" datatype="xsd:string"></span><span id='am-26'  ></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="fungi" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="PMID:11062127" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="Fungi" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="PMID:12684019" datatype="xsd:string"></span>fungi</span>, <br />                                         such as C. albicans and S. cerevisiae, wild-type C. neoformans undergoes negligible <br />                                         phagocytosis by cultured macrophages in the absence of opsonizing antibodies or complement <br />                                         (Kozel and Mastroianni, 1976). The anti-phagocytic properties of C. neoformans have largely <br />                                         been attributed to the polysaccharide capsule (Del Poeta, 2004). We recapitulated these results <br />                                         with wild-type C. neoformans, observing both robust induction of capsule and a very low rate <br />                                         of phagocytosis after 24 hours of co-culturing with RAW264.7 macrophages (&lt;2% of <br />                                         macrophages contained internalized wild-type C. neoformans, Figures 5A&#8211;B, Figure S12). <br />                                         Deletion of the capsule-associated genes CAP10, CAP60, or CAP64 resulted in approximately <br />                                         a three- to seven-fold increase in the number of macrophages containing C. neoformans (4&#8211; <br />                                         14%). As previously reported (Moyrand and Janbon, 2004), cap10&#916;, cap60&#916;, and cap64&#916; <br />                                         mutants had no detectable capsular material, either by antibody or India ink staining (Figure <br />                                         5C). Unexpectedly, a dramatic increase in the amount of phagocytosis was seen when <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         RAW264.7 macrophages were co-cultured with gat201&#916; cells; 66% of macrophages contained <br />                                         internalized C. neoformans after 24 hours (Figure 5A&#8211;B). Since gat201&#916; mutants were <br />                                         phagocytosed more readily than CAP gene mutants, yet displayed a weaker capsule defect, the <br />                                         phagocytosis phenotype of gat201&#916; cells could not easily be explained by its hypocapsular <br />                                         phenotype. To examine the role of GAT201 in the absence of capsule, we asked whether double <br />                                         mutants lacking both GAT201 and one of the CAP genes were phagocytosed to a greater extent <br />                                         than the CAP gene single mutants. Strikingly, we found this to be the case: high levels of <br />                                         phagocytosis were seen in gat201&#916; cap10&#916; (47%), gat201&#916; cap60&#916; (57%), and gat201&#916; <br />                                         cap64&#916; (52%) double mutant strains (Figure 5B). Unlike gat201&#916; single mutants, these double <br />                                         mutant strains had no detectable capsules by antibody or India ink staining and displayed <br />                                         clumpy phenotypes characteristic of cap10&#916;, cap60&#916;, and cap64&#916; single mutants (Figure 5C), <br />                                         indicating that the capsule is not the basis for the difference in phagocytosis between <br />                                         gat201&#916; and the CAP mutants. <br />  <br />                                DISCUSSION <br />                                         The incidence of invasive fungal infections in humans has risen dramatically in the last two <br />                                         decades (Warnock, 2006), yet our understanding of the molecular mechanisms relevant to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         fungal pathogenesis remains limited. The analysis of the C. neoformans gene deletion <br />                                         collection described here demonstrates the potential for harnessing genome sequencing efforts <br />                                         to implement systematic approaches for identifying and dissecting fungal virulence <br />                                         mechanisms. Since biolistic gene knockout methods for C. neoformans were introduced in <br />                                         1993, ~190 targeted deletions have been reported in the literature (Table S1 summarizes the <br />                                         phenotypes of previously-described gene deletions). We report the construction and analysis <br />                                         of 1,201 targeted gene deletion strains, significantly extending the genetic resources for this <br />                                         pathogen. Systematic profiling of the deletion collection yielded a number of insights into C. <br />                                         neoformans virulence: <br />                                             1.   Melanization: By identifying and analyzing numerous novel factors required for <br />                                                  melanization, we obtained strong evidence that virulence and melanization are co- <br />                                                  regulated, but that the lung proliferation defects observed in a large subset of melanin <br />                                                  mutants result from defects beyond poor melanization. <br />  <br />  <br />                                                  Cell. Author manuscript; available in PMC 2009 October 3. <br />                             Liu et al.                                                                                                   Page 9 <br />  <br />  <br />                                                2.   Polysaccharide capsule: We confirmed the role of the polysaccharide capsule in <br />                                                     virulence. Our screen identified four novel hypocapsular mutants, all of which <br />                                                     displayed defects in infection. Gat201, following Nrg1 (Cramer et al., 2006), is the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     second DNA-binding transcription factor found to regulate capsule formation. Gat201 <br />                                                     is normally limiting for synthesis of this virulence factor since overexpression of <br />                                                     GAT201 is sufficient to induce capsule synthesis under classically non-inducing <br />                                                     conditions. As Gat201 also controls melanization, inhibition of phagocytosis, and <br />                                                     virulence, it appears to be a central point of regulation for cryptococcal pathogenesis. <br />                                                3.   Capsule-independent anti-phagocytic pathways: The anti-phagocytic properties of <br />                                                     C. neoformans have largely been attributed to its polysaccharide capsule (Del Poeta, <br />                                                     2004). Our analysis of the Gat201 transcriptional regulator has identified a previously <br />                                                     unsuspected capsule-independent mechanism of anti-phagocytosis since deletion of <br />                                                     GAT201, both in wild-type and capsule-deficient genetic backgrounds, resulted in a <br />                                                     dramatic increase in phagocytosis of untreated C. neoformans by macrophages. Some <br />                                                     CAP mutants do produce the minor capsule polysaccharide, galactoxylomannan <br />                                                     (GalXM), which is detectable in culture supernatant (Vaishnav et al., 1998). Whether <br />                                                     GAT201 functions by controlling the synthesis of GalXM, protein factors, and/or <br />                                                     lipids, and whether the effect seen in gat201&#916; mutants is due to the lack of an anti- <br />                                                     phagocytic factor or the presence of a pro-phagocytic factor remain unclear. <br />                                                     Transcriptional profiling of a GAT201 overexpression strain detected a significant <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     enrichment (P =1.2E-35) for transcripts of C. neoformans genes that are not found in <br />                                                     other sequenced fungi including a number of predicted secreted proteins&#8212;some of <br />                                                     these could contribute to C. neoformans-specific virulence pathways, including ones <br />                                                     involved in inhibiting phagocytosis by macrophages. Finally, 25 genes encoding <br />                                                     DNA-binding transcription factors were upregulated in the overexpression strain, <br />                                                     suggesting that Gat201 may control a large, virulence-associated transcriptional <br />                                                     regulatory circuit. <br />                                                4.   Potential novel virulence pathways: Molecular studies of C. neoformans virulence <br />                                                     have focused primarily on the regulation of the capsule and melanization. The <br />                                                     discovery of 48 previously undescribed genes (40 mutants with decreased infectivity, <br />                                                     8 mutants with increased infectivity), that affect proliferation in lung tissue, but not <br />                                                     capsule, growth or melanin, strongly suggests the existence of additional unknown <br />                                                     virulence mechanisms and provides tools for their identification. In principle, these <br />                                                     genes could code for 1) regulators of novel virulence pathways or 2) direct effectors <br />                                                     of virulence. Potential regulators include seven predicted transcription factors, whose <br />                                                     targets presumably include pathogenesis factors. Possible direct effectors of virulence <br />                                                     include three factors with homology to proteins previously linked to pathogenesis <br />                                                     and/or immune modulation (Liv5, Liv6 and Liv7). A substantial number of proteins <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     required for infectivity were identified whose function cannot be inferred from <br />                                                     sequence relationships, indicating that novel molecular functions yet to be assigned <br />                                                     to specific proteins in any organism are likely to play a role in fungal virulence. Taken <br />                                                     together, the identification and categorization of infectivity mutants in a large deletion <br />                                                     library highlight the advantages of using a multipronged, systematic genetic analysis <br />                                                     to dissect pathogenic mechanisms. <br />  <br />                                EXPERIMENTAL PROCEDURES <br />                                Generation of gene deletion strains <br />                                            <span id='am-77' about='obo:NCBITaxon_178876' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5234 obo:NCBITaxon_552466 obo:NCBITaxon_33154 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_5207 obo:NCBITaxon_5215 obo:NCBITaxon_5206 obo:NCBITaxon_2759 obo:NCBITaxon_155616'><span id='am-78' property="oboInOwl:hasAlternativeId" content="NCBITaxon:427679" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Cryptococcus neoformans serotype A" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="Cryptococcus neoformans var. grubii" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Cryptococcus neoformans A" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Filobasidiella neoformans serotype A" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasRelatedSynonym" content="Filobasidiella neoformans var. grubii" datatype="xsd:string"></span><span id='am-86'  ></span>Cryptococcus neoformans var. grubii</span> serotype A clinical strain H99 (a gift from J. Lodge) was <br />                                            used as wild-type. A previously described strategy for gene deletion (Chun et al., 2007) utilizing <br />                                            overlap fusion PCR and biolistic techniques was used to generate knockout strains (see <br />  <br />  <br />                                                     Cell. Author manuscript; available in PMC 2009 October 3. <br />                             Liu et al.                                                                                                 Page 10 <br />  <br />  <br />                                             Supplementary Materials). Details on the deletion collection including the targeted regions in <br />                                             each mutant can be found in Table S4. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Signature-tagged mutagenesis <br />                                             Groups of 48 C. neoformans deletion strains were individually grown in liquid YPAD and <br />                                             combined into a single pool. These pooled inocula were used to intranasally infect three mice <br />                                             each. Three aliquots of each inoculum (~5 &#215; 105 cells) were also plated on Sabouraud agar <br />                                             plates containing 40 &#181;g/mL gentamicin and 50 &#181;g/mL carbenicillin. Genomic DNA (Input <br />                                             DNA) was purified independently from each plate using hexadecyltrimethyl ammonium <br />                                             bromide (CTAB, Sigma). Infected mice were monitored and sacrificed at the disease endpoint. <br />                                             Lungs were removed and homogenized in 5 mL sterile PBS. Serial dilutions of each organ <br />                                             sample were plated on Sabouraud agar plates containing 40 &#181;g/mL gentamicin and 50 &#181;g/mL <br />                                             carbenicillin. Genomic DNA (Output DNA) from each set of lungs was purified using CTAB. <br />  <br />                                             The amounts of each signature tag in a genomic DNA prep were measured using 48 <br />                                             simultaneous quantitative PCR reactions. Each reaction was identical except for a different <br />                                             tag-specific primer (Table S7). The threshold cycle (Ct) was determined for each reaction and <br />                                             an STM score was calculated for each mutant using a variation of the 2&#8722;&#916;&#916;Ct method for <br />                                             quantitation analysis (Livak and Schmittgen, 2001). STM scores from three mice were <br />                                             averaged to determine a final STM score for each mutant. A detailed description of this STM <br />                                             method can be found in the Supplementary Materials. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                In vitro phenotyping <br />                                             Polysaccharide capsule formation, melanin synthesis, urease activity, and growth on acidified <br />                                             nitrite, SDS, potassium cyanide, pH 4, and low iron medium were assayed using standard <br />                                             conditions (see Supplementary Materials). Growth in minimal medium was quantified by <br />                                             measuring the change in optical density at 600 nm (OD600) after 24 hours growth using an <br />                                             optical plate reader (Molecular Devices). Growth scores were normalized to the median growth <br />                                             in groups of 48 (see Supplementary Materials). <br />  <br />                                Statistical analysis <br />                                             Significances of overlaps between STM, capsule, and melanin phenotypes were measured <br />                                             using a hypergeometric test. Correlation of STM and growth defects was analyzed by <br />                                             measuring enrichment for STM defects in a 25-mutant moving average across growth scores <br />                                             using a hypergeometric test. <br />  <br />                                             Significances of mutant STM scores were calculated by measuring the divergence of a mutant <br />                                             STM score from the distribution of sxi&#916; STM scores using a z-score calculation. For retesting <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             of STM phenotypes in reconstructed knockouts, an STM score of &#8722;1.5 was used as a threshold <br />                                             for significance (P &lt; .001). <br />  <br />                                             Survival curves were analyzed using the Kaplan-Meier method and log rank test. <br />  <br />                                Macrophage assays <br />                                             To test for phagocytosis of untreated C. neoformans, RAW264.7 macrophages (4&#215;104 cells/ <br />                                             well) were seeded in 96-well tissue culture-treated glass bottom plates (Nunc) in DMEM media <br />                                             and allowed to adhere overnight. C. neoformans cells from overnight cultures grown in liquid <br />                                             YPAD were washed three times with PBS and then lightly sonicated using a Sonifer 450 <br />                                             equipped with a microtip (Branson, two one-second pulses at a power setting of 2) to break up <br />                                             any clumps of cells prior to incubation with macrophages. Sonication did not affect the viability <br />                                             of cells and similar results were obtained in phagocytosis assays that utilized wild-type and <br />                                             gat201&#916; cells that were not sonicated (Figure S13). The yeast cells were resuspended in PBS <br />  <br />                                                       Cell. Author manuscript; available in PMC 2009 October 3. <br />                             Liu et al.                                                                                                                      Page 11 <br />  <br />  <br />                                              to a concentration of 107 cells/mL, and 10 &#181;L (105 cells) were co-incubated with RAW264.7 <br />                                              cells (in 200 &#181;L fresh DMEM/well) for an MOI of ~ 2:1. Following 24 hours of co-incubation, <br />                                              unphagocytosed C. neoformans cells were removed with three washes of PBS, and yeast and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              macrophages were fixed with 1% formaldehyde/PBS. Remaining extracellular yeast were <br />                                              labeled with a mixture of an anti-glucoronxylomannan antibody, mAb 339 (Belay et al., <br />                                              1997), and an anti-<span id='am-11' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span>beta</span>-glucan mouse monoclonal antibody (Biosupplies Australia) in PBS, a <br />                                              combination which labels both encapsulated and acapsular strains. Following three washes <br />                                              with PBS, the cells were then incubated with FITC-conjugated donkey anti-mouse secondary <br />                                              antibody (Jackson ImmunoResearch) in PBS. Phagocytosis of C. neoformans was quantified <br />                                              on an Axiovert 200M (Zeiss) microscope running Axiovision software. Percentage of <br />                                              macrophages with phagocytosed yeast was determined by counting the number of macrophages <br />                                              with internalized yeast (as defined by the absence of FITC-labeling). At least 200 macrophages <br />                                              were assayed per well, and each strain was tested in triplicate. <br />  <br />                                Microarrays <br />                                              Four replicate cultures each of H99 and pRPL2b::GAT201 were grown overnight to saturation <br />                                              in YNB at 37&#176;C. Cultures were diluted to an OD600 of 0.1 in YNB (pre-warmed to 37&#176;C), <br />                                              grown to an OD600 of 1.0, and then harvested by centrifugation and snap freezing. RNA <br />                                              isolation, array hybridizations, and data analysis using SAM analysis were performed as <br />                                              previously described (Chun et al., 2007, see Supplementary Materials). Microarray data can <br />                                              be found at the Gene Expression Omnibus (GSE10750) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />                                         Figure 1. <br />                                         Schematic overview of systematic phenotypic profiling strategy utilized in this work. <br />  <br />  <br />  <br />  <br />                                              <br />  <br />  <br />  <br />                                         Figure 2. In vivo signature-tagged mutagenesis (STM) analysis of gene deletions <br />                                         (A) Mutants were pooled in groups of 48 and used to intranasally infect three mice. Quantitative <br />                                         PCR with signature tag-specific primers was used to calculate an STM score for each mutant. <br />                                         Red-dotted lines indicate cutoffs used to identify STM phenotypes of interest. (B) 104 mutants <br />                                         assayed by STM were sorted into new pools containing different competitors and retested. <br />                                         Original STM scores show a strong correlation with repooled STM scores. (C) The frequencies <br />                                         of STM scores observed in this screen were plotted for both the deletion collection and the <br />                                         sxi1&#916; control strains. Red-dotted lines indicate cutoffs used to identify STM phenotypes of <br />                                         interest. Indicated below are the STM scores of gene knockouts in the collection previously <br />                                         reported to have (red) or not to have (black) virulence defects. (D&#8211;E) Survival curve analyses <br />  <br />                                                 Cell. Author manuscript; available in PMC 2009 October 3. <br />                             Liu et al.                                                                                          Page 17 <br />  <br />  <br />                                         of mice infected with individual strains demonstrate the correlation between STM phenotypes <br />                                         and virulence phenotypes. Both hypovirulent and hypervirulent strains were accurately <br />                                         identified by STM. P-values indicate the significance of the virulence phenotype. (F) Shown <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         is the distribution of in vivo and in vitro phenotypes in the deletion collection. Significant <br />                                         overlaps were seen between mutants with defects in the three virulence attributes and mutants <br />                                         with in vivo infectivity defects. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Cell. Author manuscript; available in PMC 2009 October 3. <br />                             Liu et al.                                                                                              Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 3. Proteins identified in this work that regulate capsule formation, melanization, and/or <br />                                         infectivity <br />                                         Factors previously reported to affect these phenotypes and factors that affect growth in YNB <br />                                         are not included. Proteins predicted to encode signal peptides are outlined in red. The <br />                                         relationships between melanization, dissemination, and lung infectivity are discussed in the <br />                                         text. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Cell. Author manuscript; available in PMC 2009 October 3. <br />                             Liu et al.                                                                                           Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 4. Gat201 is a regulator of multiple virulence pathways <br />                                         (A) gat201&#916; mutant strains (two independent knockout strains) were avirulent during <br />                                         monotypic infection and a GAT201 overexpression strain was hypervirulent. (B) gat201&#916; <br />                                         mutant strains did not induce an observable capsule by India ink staining in capsule-inducing <br />                                         conditions, though capsule polysaccharide on the cell surface was detected by antibody staining <br />                                         (Figure 5B). GAT201 overexpression resulted in ectopic capsule formation in normally non- <br />                                         inducing conditions. Quantification of capsule size is shown in Figure S11. (C) gat201&#916; <br />                                         mutants strains showed increased melanization while a GAT201 overexpression strain showed <br />                                         decreased melanization. (D) Transcriptional profiles of pRPL2b::GAT201 and wild-type <br />  <br />  <br />                                                 Cell. Author manuscript; available in PMC 2009 October 3. <br />                             Liu et al.                                                                                 Page 20 <br />  <br />  <br />                                         grown at 37&#176;C in minimal medium were compared using whole-genome microarrays. Values <br />                                         represent the average of four microarrays. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Cell. Author manuscript; available in PMC 2009 October 3. <br />                             Liu et al.                                                                                          Page 21 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 5. Gat201 inhibits phagocytosis independent of capsule <br />                                         (A&#8211;B) Phagocytosis of untreated wild-type (WT) and gat201&#916; cells by RAW264.7 <br />                                         macrophages was measured in both WT and capsule-deficient backgrounds. The percentage <br />                                         of macrophages containing internalized C. neoformans cells was determined after 24 hours of <br />                                         co-incubation. Data are represented as mean +/&#8722; SD from triplicate well s. (C) GXM can be <br />                                         detected on the cell surface of gat201&#916; cells, but not in single and double mutants containing <br />                                         deletions of CAP genes. (D) Model of GAT201 regulation of phagocytosis. &#8220;X&#8221; represents a <br />                                         novel capsule-independent mechanism for inhibiting phagocytosis by macrophages. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Cell. Author manuscript; available in PMC 2009 October 3. <br />                                                              NIH-PA Author Manuscript                                     NIH-PA Author Manuscript                                       NIH-PA Author Manuscript <br />                                                                                                                                                 TABLE 1 <br />                                                                                  Novel capsule and melanin mutants <br />                                                                                  Genes identified in this work that regulate capsule formation, melanization, and/or infectivity. Previously characterized genes can be <br />                                                                                  found in Table S2. <br />                                                                                                                                                                                                                                        Liu et al. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  Gene Deletion Phenotype <br />  <br />                                                               KO#                 Gene Name           Broad Annotation   Description                                                 STM Score   Growth Score      Capsule   Melanin <br />  <br />  <br />                                                               Capsule <br />                                                               Transcription <br />                                                                D429                GAT201             CNAG_01551         GATA family DNA-binding regulator                           &#8722;18.3&#8595;*     &#8722;0.18                &#8722;        ++ <br />                                                                D226                SSN801             CNAG_00440         Cyclin subunit of Mediator subcomplex                       &#8722;1.4        0.18                 ++       ++ <br />                                                               Chromatin <br />                                                                D110                HOS2               CNAG_05563         Class II HDAC; component of Set3 complex in S. cerevisiae   &#8722;16.1&#8595;      &#8722;1.24&#8595;               ++       &#8722; <br />                                                                D1487               SET302             CNAG_06591         SET domain protein; homolog of S. cerevisiae Set3           &#8722;10.9&#8595;      &#8722;0.86&#8595;               ++       &#8722; <br />                                                               Putative secreted proteins <br />                                                                D667                CPL1               CNAG_02797&#8224;        Potential secreted protein of unknown function              &#8722;5.3&#8595;*      0.04                 &#8722;        + <br />  <br />                                                               Melanin <br />                                                               Transcription <br />                                                                D1248              ZAP104              CNAG_05392         Zinc finger DNA-binding regulator                           &#8722;7.1&#8595;*      0.06                 +         &#8722; <br />                                                                D1512              MBP102              CNAG_06721         APSES family DNA-binding regulator                          &#8722;3.5&#8595;       0.08                 +         &#8722; <br />                                                                D528               NHP6B02             CNAG_02115         Nonhistone chromosomal protein                              &#8722;0.9        &#8722;1.86&#8595;               +        +/&#8722; <br />                                                                D1218              MLN1                CNAG_04837         bHLH family DNA-binding regulator                           &#8722;0.3        &#8722;0.04                +         &#8722; <br />                                                                D228               NHP6B01             CNAG_00445         Nonhistone chromosomal protein                              1.1         0.00                 +         &#8722; <br />                                                                D356               PAN1                CNAG_01173         LAG1 family DNA-binding regulator                           1.4         &#8722;0.40                +         &#8722; <br />                                                                D1258              RIM101              CNAG_05431         Zn finger DNA-binding regulator regulated by proteolysis    2.8&#8593;        &#8722;0.07                +         &#8722; <br />                                                               Chromatin <br />                                                                D1498              RTF1                CNAG_06648         Component of PAF complex required for histone H2B           &#8722;12.7&#8595;      &#8722;1.66&#8595;               +        &#8722; <br />                                                                                                                          monoubiquitination <br />                                                                D78                 SET202             CNAG_03188         Histone H3 lysine 36 methyltransferase                      &#8722;8.1&#8595;       &#8722;0.14                +         &#8722; <br />                                                                D373                SET101             CNAG_01243         Histone H3 lysine 4 methyltransferase                       &#8722;4.5&#8595;*      &#8722;0.38                +        +/&#8722; <br />                                                               Protein sorting / Protein degradation <br />                                                                D282                DOA4               CNAG_00757         Ubiquitin hydrolase                                         &#8722;15.6&#8595;      &#8722;0.06                +         &#8722; <br />                                                                D1250               VAM6               CNAG_05395         Vacuolar membrane fusion factor                             &#8722;14.3&#8595;      &#8722;0.61&#8595;               +         &#8722; <br />  <br />  <br />  <br />  <br /> Cell. Author manuscript; available in PMC 2009 October 3. <br />                                                                D1222               VPS25              CNAG_04863         ESCRT-II Complex subunit                                    &#8722;13.8&#8595;      &#8722;0.06                +         &#8722; <br />                                                                D1314               UBP14              CNAG_05708         Ubiquitin carboxyl-terminal hydrolase                       &#8722;12.3&#8595;      0.27                 +        +/&#8722; <br />                                                                D454                CSN1201            CNAG_01697         COP9 signalosome subunit 12; PINT domain                    &#8722;8.9&#8595;       &#8722;0.58&#8595;               +        +/&#8722; <br />                                                                D936                CSN7               CNAG_04177         COP9 signalosome subunit 7; PINT domain                     &#8722;1.7        &#8722;0.07                +        +/&#8722; <br />                                                                D598                CSN1               CNAG_02513         COP9 signalosome subunit 1; PINT domain                     &#8722;1.1        &#8722;0.16                +        +/&#8722; <br />                                                                D737                CSN4               CNAG_03151         COP9 signalosome subunit 4; PINT domain                     &#8722;0.2        0.00                 +        +/&#8722; <br />                                                                D1349               HSE102             CNAG_05882         Class E VPS factor involved in endosomal sorting            &#8722;0.2        0.00                 +         &#8722; <br />                                                                D671                RPS3102            CNAG_02827         NEDD8 ubiquitin-like protein that regulates Cullins         4.1&#8593;        &#8722;1.68&#8595;               +         &#8722; <br />                                                               Signaling <br />                                                                D245                YCK2               CNAG_00556         Casein kinase homolog involved in endocytosis               &#8722;15.4&#8595;      &#8722;0.44                +         &#8722; <br />                                                                D1476               SNF102             CNAG_06552         AMP-activated kinase; ortholog of S. cerevisiae Snf1        &#8722;15.0&#8595;*     &#8722;0.34                +         &#8722; <br />                                                                D731                CHO2               CNAG_03139         Putative protein-S-isoprenylcysteine methyltransferase      &#8722;12.3&#8595;      0.27                 +         &#8722; <br />                                                                D221                CDC2801            CNAG_00415         Cyclin-dependent kinase of PITSLRE <span id='am-50' about='obo:NCBITaxon_subfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="subfamily" datatype="xsd:string"></span>subfamily</span>                &#8722;9.9&#8595;       &#8722;0.11                +        +/&#8722; <br />                                                                D825                RIM20              CNAG_03582         ESCRT-III-associated regulator of Rim101                    &#8722;0.3        0.09                 +         &#8722; <br />                                                               Nuclear pore <br />                                                                D891                NUP75              CNAG_03917         Nuclear pore protein homologous to human Nup75              &#8722;14.2&#8595;      &#8722;1.83&#8595;               +        +/&#8722; <br />                                                               Unknown function <br />                                                                D1408               MLN2               CNAG_06224         CS domain protein of unknown function                       &#8722;1.4        &#8722;1.24&#8595;               +         &#8722; <br />                                                                D1453               MLN3               CNAG_06430         Mus7 domain-containing protein conserved in fungi           &#8722;1.6        &#8722;0.78&#8595;               +        +/&#8722; <br />                                                                                                                                                                                                                                        Page 22 <br />                                                              NIH-PA Author Manuscript                                            NIH-PA Author Manuscript                                 NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                                                                   Gene Deletion Phenotype <br />  <br />                                                               KO#                  Gene Name           Broad Annotation          Description                                          STM Score   Growth Score      Capsule   Melanin <br />                                                                                                                                                                                                                                         Liu et al. <br />  <br />  <br />  <br />  <br />                                                                D445                MLN4                CNAG_01644                Protein of unknown function; C. neoformansspecific   &#8722;1.4        &#8722;0.18                +        &#8722; <br />  <br />  <br />                                                               Down arrows (&#8595;) indicate significant decreases compared to wild-type. <br />  <br />                                                               Up arrows (&#8593;) indicate significant increases compared to wild-type. <br />  <br />                                                               Asterisks (*) indicate that the STM phenotype was verified in a reconstructed knockout strain. <br />  <br />                                                               Crosses (&#8224;) indicate gene models that are predicted to encode signal peptides. <br />  <br />  <br />  <br />  <br /> Cell. Author manuscript; available in PMC 2009 October 3. <br />                                                                                                                                                                                                                                         Page 23 <br />                                                              NIH-PA Author Manuscript                                        NIH-PA Author Manuscript                                        NIH-PA Author Manuscript <br />                                                                                                                                                 TABLE 2 <br />                                                                                  Infectivity mutants without defects in capsule formation, melanization, or growth in YNB <br />                                                                                  Table 2 includes only infectivity mutants that did not display defects in capsule formation, melanization, or growth in YNB. These mutants <br />                                                                                  were tested for six additional phenotypes: decreased urease activity and sensitivity to acidified nitrite (NO), SDS, potassium cyanide <br />                                                                                  (KCN), pH 4, and low-iron. <br />                                                                                                                                                                                                                                                 Liu et al. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                   Gene Deletion Phenotype <br />  <br />                                                               KO#                    Gene Name           Broad Annotation       Description                                       STM Score      Growth Score         Additional Phenotypes <br />  <br />  <br />                                                               Decreased Infectivity (STM Score &lt; &#8722;2.5) <br />                                                               Transcription <br />                                                                D883                  PDR802              CNAG_03894             Binuclear cluster DNA-binding regulator           &#8722;7.2&#8595;*         &#8722;0.08                      None detected <br />                                                                D231                  LIV1                CNAG_00460             bHLH family DNA-binding regulator                 &#8722;6.1&#8595;*         &#8722;0.35                      None detected <br />                                                                D725                  HCM1                CNAG_03116             Forkhead DNA-binding regulator                    &#8722;4.2&#8595;*         &#8722;0.46                      None detected <br />                                                                D140                  SNT1                CNAG_00051             SANT domain protein related to human NCOR         &#8722;4.0&#8595;*         &#8722;0.10                      None detected <br />                                                                                                                                 corepressor subunit <br />                                                                D177                   LIV2               CNAG_00201             LAG1 family DNA-binding regulator                 &#8722;3.9&#8595;*         0.06                       None detected <br />                                                                D1338                  LIV3               CNAG_05835             WOR1 family DNA-binding regulator                 &#8722;3.7&#8595;*         0.03                       None detected <br />                                                                D1420                  LIV4               CNAG_03116             Myb family DNA-binding regulator                  &#8722;2.5&#8595;*         &#8722;0.03                      None detected <br />                                                               Chromatin <br />                                                                D111                   RPD304             CNAG_05690             Class II histone deacetylase                      &#8722;11.5&#8595;*        &#8722;0.3                      None detected <br />                                                                D1296                  RPH1               CNAG_05622             Histone H3 lysine 36 demethylase                  &#8722;8.0&#8595;*         &#8722;0.25                     None detected <br />                                                                D933                   HIRA               CNAG_04158             Histone chaperone                                 &#8722;6.7&#8595;*         &#8722;0.15                 Inc. sensitivity to NO <br />                                                                D790                   SPP101             CNAG_03406             PHD domain protein of unknown function            &#8722;6.1&#8595;*         &#8722;0.09                     None detected <br />                                                                D1428                  SAS3               CNAG_06322             Histone acetyltransferase                         &#8722;4.4&#8595;*         &#8722;0.18                     None detected <br />                                                                D62                    HST302             CNAG_02085             Homolog of Sir2; Class III histone deacetylase    &#8722;2.8&#8595;*         &#8722;0.11                     None detected <br />                                                               Protein sorting / Protein degradation <br />                                                                D1266                  HRD1               CNAG_05469             Transmembrane ubiquitin ligase required for       &#8722;8.0&#8595;*         &#8722;0.38                 Inc. sensitivity to NO <br />                                                                                                                                 ER-associated degradation <br />                                                                D317                  UBA4                CNAG_00986             Urm1 activating protein                           &#8722;6.0&#8595;*         &#8722;0.2                       None detected <br />                                                                D399                  RMD5                CNAG_01350             Ubiquitin ligase of unknown function              &#8722;4.3&#8595;*         0.02                       None detected <br />                                                                D729                  CUL3                CNAG_03138             Cullin subunit of SCF-like ubiquitin ligase       &#8722;3.6&#8595;*         &#8722;0.11                      None detected <br />                                                                D1166                 UBC8                CNAG_04611             Ubiquitin conjugating enzyme                      &#8722;3.5&#8595;*         0.16                       None detected <br />  <br />  <br />  <br />  <br /> Cell. Author manuscript; available in PMC 2009 October 3. <br />                                                                D1278                 JJJ1                CNAG_05538             DnaJ-like cochaperone; contains U1 snRNA-         &#8722;3.4&#8595;*         0.11                       None detected <br />                                                                                                                                 type Zn finger <br />                                                               DNA Repair <br />                                                                D352                  RAD54               CNAG_01163             SWI/SNF family ATPase that promotes               &#8722;9.3&#8595;*         &#8722;0.49                 Inc. sensitivity to NO <br />                                                                                                                                 recombinational DNA repair <br />                                                                D288                  RAD23               CNAG_00772             Ubiquitin-binding protein required for            &#8722;7.6&#8595;*         0.09                  Inc. sensitivity to NO <br />                                                                                                                                 nucleotide excision repair <br />                                                                D832                  YKU80               CNAG_03637             Double strand break repair factor and silencing   &#8722;4.9&#8595;*         0.04                       None detected <br />                                                                                                                                 regulator <br />                                                                D632                  RAD502              CNAG_02630             SWI/SNF family ATPase that promotes               &#8722;2.7&#8595;*         &#8722;0.04                      None detected <br />                                                                                                                                 recombinational DNA repair <br />                                                                D855                  RINT1               CNAG_03760             Rad50-interacting protein implicated in DNA       &#8722;2.5&#8595;*         0.01                 Inc. sensitivity to KCN <br />                                                                                                                                 repair <br />                                                               Pathogenesis / Immunity <br />                                                                D576                 LIV5                 CNAG_02409             TB2/DP1 family member homologous to M.            &#8722;12.6&#8595;*        &#8722;0.09                      None detected <br />                                                                                                                                 grisea pathogenicity protein <br />                                                                D166                  LIV6                CNAG_00170&#8224;            Related to human T-cell immunomodulatory          &#8722;6.2&#8595;*         0.14                       None detected <br />                                                                                                                                 protein; FG-GAP repeats <br />                                                                                                                                                                                                                                                 Page 24 <br />                                                              NIH-PA Author Manuscript                                            NIH-PA Author Manuscript                                              NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                                                                             Gene Deletion Phenotype <br />  <br />                                                               KO#                    Gene Name              Broad Annotation             Description                                        STM Score      Growth Score         Additional Phenotypes <br />                                                                                                                                                                                                                                                              Liu et al. <br />  <br />  <br />  <br />  <br />                                                                D1466                 LIV7                   CNAG_06464&#8224;                  Protein of unknown function; human homolog         &#8722;3.2&#8595;*         &#8722;0.17                      None detected <br />                                                                                                                                          is LPS-inducible <br />                                                               Signaling <br />                                                                D972                  HSV2                   CNAG_04371                   WD40 protein related to 3-phosphorylated           &#8722;15.9&#8595;*        0.18                       None detected <br />                                                                                                                                          inositide-binding protein <br />                                                                D280                  HRK1                   CNAG_00745                   Protein kinase; S. cerevisiae ortholog regulates   &#8722;6.8&#8595;*         0.14                 Inc. sensitivity to SDS <br />                                                                                                                                          metabolite transport <br />                                                                D437                  LIV8                   CNAG_01611                   RGS protein homolog - G protein GAP                &#8722;2.7&#8595;*         0.10               Inc. sensitivity to NO, SDS, <br />                                                                                                                                                                                                                                           KCN <br />                                                               Unknown <br />                                                                D70                   HOS4                   CNAG_02727                   Ankyrin repeat protein of unknown function         &#8722;7.6&#8595;*         0.09                       None detected <br />                                                                D736                  LIV9                   CNAG_03149                   RING and ubiquitin-binding protein with            &#8722;7.3&#8595;*         &#8722;0.04                      None detected <br />                                                                                                                                          BRAP2 domain <br />                                                                D686                  LIV10                  CNAG_02930                   DUF1765 domain protein of unknown function         &#8722;7.1&#8595;*         &#8722;0.19                      None detected <br />                                                                D171                  FYV10                  CNAG_00181                   LisH, CTLH domain protein of unknown               &#8722;6.5&#8595;*         &#8722;0.06                      None detected <br />                                                                                                                                          function <br />                                                                D1256                 LIV11                  CNAG_05422                   Protein of unknown function                        &#8722;5.0&#8595;*         &#8722;0.49                      None detected <br />                                                                D1035                 LIV12                  CNAG_05105                   Protein of unknown function                        &#8722;3.9&#8595;*         &#8722;0.11                      None detected <br />                                                                D660                  LIV13                  CNAG_02753                   UPF0121 domain protein of unknown function         &#8722;3.2&#8595;*         &#8722;0.34                      None detected <br />                                                                D1448                 LIV14                  CNAG_06397                   Protein of unknown function                        &#8722;2.9&#8595;*         0.06                       None detected <br />                                                                D1357                 LIV15                  CNAG_05934                   Translin domain protein of unknown function        &#8722;2.8&#8595;*         0.09                       None detected <br />                                                               Transporter <br />                                                                D1091                 ENA2                   CNAG_00531                   Sodium pump                                        &#8722;11.1&#8595;*        0.11                       None detected <br />  <br />                                                               Increased Infectivity (STM Score &gt; 2.5) <br />                                                               Transcription <br />                                                                D88                   ZAP103                 CNAG_04352                   Zinc finger DNA-binding regulator                  3.2&#8593;*          &#8722;0.14 <br />                                                               Signaling <br />                                                                D1340                 RGD1                   CNAG_05838                   Rho GAP homolog                                    5.3&#8593;*          0.07 <br />                                                                D1490                 RHO104                 CNAG_06606                   Rho family GTPase                                  4.4&#8593;*          &#8722;0.33 <br />                                                                D1156                 RCK2                   CNAG_00130                   Ca-CaM kinase homolog                              3.7&#8593;*          0.03 <br />                                                                D488                  KSP1                   CNAG_01905                   Protein kinase                                     3.4&#8593;*          &#8722;0.04 <br />  <br />  <br />  <br />  <br /> Cell. Author manuscript; available in PMC 2009 October 3. <br />                                                               Cell wall <br />                                                                D61                   BNI4                   CNAG_02071                   S. cerevisiae ortholog is a scaffold protein       4.8&#8593;*          0.27 <br />                                                                                                                                          involved in cell wall assembly <br />                                                               Actin-related <br />                                                                D359                  LSB1                   CNAG_01183                   BAR-SH3 domain; S. cerevisiae homolog binds        3.6&#8593;*          0.11 <br />                                                                                                                                          actin nucleator WASP <br />                                                               Unknown <br />                                                                D1179                 LIV16                  CNAG_07193&#8224;                  Conserved protein of unknown function              3.8&#8593;*          0.05 <br />  <br />  <br />                                                               Down arrows (&#8595;) indicate significant decreases compared to wild-type. <br />  <br />                                                               Up arrows (&#8593;) indicate significant increases compared to wild-type. <br />  <br />                                                               Asterisks (*) indicate that the STM phenotype was verified in a reconstructed knockout strain. <br />  <br />                                                               Crosses (&#8224;) indicate gene models that are predicted to encode signal peptides. <br />                                                                                                                                                                                                                                                              Page 25 <br />   <br /> </body></html>