<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Bioinformatics                                                                                                                       BioMed Central <br />  <br />  <br />  <br /> Database                                                                                                                               Open Access <br /> A PATO-compliant zebrafish screening database (MODB): <br /> management morpholino knockdown screen information <br /> Michelle N Knowlton*1,6, Tongbin Li2, Yongliang Ren2, Brent R Bill1, <br /> Lynda BM Ellis3,4 Stephen C Ekker1,4,5 <br />  <br /> Address: 1Department Genetics, Cell Biology Development, University Minnesota, Minneapolis, MN 55455, USA, 2Department of <br /> Neuroscience, University Minnesota, Minneapolis, MN 55455, USA, 3Department Laboratory Medicine Pathology, University of <br /> Minneapolis, Minneapolis, MN 55455, USA, 4University Minnesota Arnold Mabel Beckman Center Transposon Research, Minneapolis, <br /> MN 55455, USA, 5Department Biochemistry Molecular Biology, Mayo Clinic, Rochester, MN 55905, USA 6Box 45, The Jackson <br /> Laboratory, 600 Main Street, Bar Harbor, Maine 04609, USA <br /> Email: Michelle N Knowlton* - mnk@informatics.jax.org; Tongbin Li - toli@biocompute.umn.edu; <br /> Yongliang Ren - yongliang@biocompute.umn.edu; Brent R Bill - bill0115@umn.edu; Lynda BM Ellis - lynda@umn.edu; <br /> Stephen C Ekker - ekker.stephen@mayo.edu <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 7 January 2008                                                    Received: 28 April 2007 <br />                                                                              Accepted: 7 January 2008 <br /> BMC Bioinformatics 2008, 9:7   doi:10.1186/1471-2105-9-7 <br /> <span id='am-11' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-12' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-13'  ></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> article available from: http://www.biomedcentral.com/1471-2105/9/7 <br /> &#169; 2008 Knowlton et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  Abstract <br />                  Background: The zebrafish powerful model vertebrate amenable high throughput vivo <br />                  genetic analyses. Examples include reverse genetic screens using morpholino knockdown, <br />                  expression-based screening using enhancer trapping forward genetic screening using <br />                  transposon insertional mutagenesis. We created database facilitate web-based <br />                  distribution <span id='am-35' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> genetic studies. <br />                  Description: The MOrpholino DataBase MySQL relational database online, PHP <br />                  interface. Multiple quality control levels allow differential access data raw finished formats. <br />                  MODBv1 includes sequence information relating 800 morpholinos targets and <br />                  phenotypic data regarding dose effect morpholino (mortality, toxicity defects). To <br />                  improve searchability database, incorporated fixed-vocabulary defect <br />                  ontology allows organization morpholino affects based anatomical structure <br />                  affected defect produced. This allows comparison <span id='am-16' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> utilizing Phenotypic <br />                  Attribute Trait Ontology (PATO) designated terminology. MODB cross-linked ZFIN, <br />                  allowing searches databases. MODB offers users ability retrieve <br />                  morpholino data sequence morpholino target, target, anatomical structure <br />                  affected defect produced. <br />                  Conclusion: MODB data used functional genomic analysis morpholino design to <br />                  maximize efficacy minimize toxicity. MODB serves template future sequence- <br />                  based functional genetic screen databases, currently used model creation <br />                  mutagenic insertional transposon database. <br />  <br />  <br />  <br />  <br />                                                                                                                                        Page 1 10 <br />                                                                                                                  (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                  http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br />  <br /> Background                                                     pholinos. Ensembl limited ability store dis- <br /> Advances scientific technology availability      play experimental data, ZFIN contains wealth <br /> genomic sequence increasing number species           information published results lacks specific infor- <br /> have led paradigm shift identifying sequence      mation dosage searchable phenotype data for <br /> gene product function. Large functional genetic screens        published morpholinos phenotypic data largely <br /> are increasingly prevalent require new          unpublished morpholinos. <br /> way thinking data facilitate cross-species com- <br /> parison deciphering gene product function differ-       Many model organism communities developed <br /> ent organisms. Zebrafish particularly amenable          online databases access results mutagenic, knock- <br /> large screens tremendous ability repro-      down/out over-expression studies. While mouse <br /> duce, embryos finish development         knockout mutant [15,16] C. elegans RNAi [17,18] <br /> organ rudiments 2 days post-fertilization [1]. Their        experimental result databases phenotypic content <br /> semi-transparent embryos ready availability         exist, morpholino-specific database available is <br /> transgenic lines facilitate functional annotation       pilot Xenopus morpholino screen 202 genes <br /> many genes required numerous organ systems             targeted subsequent phenotypes displayed a <br /> pathways. While historically common use          webpage [19,20]. The Xenopus tropicalis Morpholino <br /> genetic screenings primarily restricted ENU        Screen database flat format website allows for <br /> mutagenesis screens [2-4] retroviral mutagenesis [5],      records accessed viewed gene targeted, syn- <br /> the availability morpholinos [6], antisense knockdown       phenotype group specific defect. Notably database <br /> oligonucleotides, insertional mutagenic <span id='am-1' about='obo:NCBITaxon_2387' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Transposon" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="transposons" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>transposons</span>        does include search functions does offer a <br /> [7] opened new avenues functional genetic             means input outside sources. However, the <br /> research.                                                      intent database stated authors display <br />                                                                results specific screen [19,20]. <br /> Recently, morpholino screens pub- <br /> lished collaboration Ekker laboratory [8,9].       Given intent database display large <br /> The largest them, Secretome Screen [9,10],           data generated on-going collabora- <br /> identified likely secreted genes available transcrip-     tive screens, developed scalable, centralized <br /> tome databases using bioinformatics approach. The Ver-       easily accessible worldwide, MOrpholino DataBase <br /> tebrate Secretome DataBase (VSDB) [4] established       (MODB) [3] store facilitate retrieval zebrafish <br /> identify co-translationally translocated (CTT) proteins     morpholino screening results. Additional access been <br /> several model organism species [10]. Using combina-          provided active links MODB ZFIN. <br /> tion signal sequence, transmembrane domain, ini- <br /> tiation site predictors homology known           MODB scalable, searchable readily adaptable <br /> secreted proteins, subset zebrafish CTT genes        online MySQL database PHP interface. MODB <br /> identified (0.3&#215; genome coverage based zebrafish            includes storage retrieval results generated our <br /> loaded predicted protein sequences CTTome           screens public access published unpublished <br /> database Genescan gene prediction estimates         information morpholinos MODB allows dis- <br /> zebrafish genome [4,5]). 150 gene targets selected     semination target-specific information effect of <br /> random morpholino design, carried using             knockdown, facilitates analysis morpholino- <br /> Assisted MOrpholino Design tool (AMOD) [11,12]. Each           specific data toxicity, dosage, mortality and <br /> gene knocked zebrafish morpholino-             design. MODB new, online, publicly accessible data- <br /> injection screened defects general morphology      base data-mining zebrafish morpholino knockdown <br /> and organ systems including ocular, renal, pig-        effects morpholino non-specific effects. <br /> ment, vascular, hematopoietic, cranio-facial, motor <br /> [9]. A second morpholino knockdown screen, Hemat-          Construction Content <br /> opoietic Stem Cell Screen, conducted using zebrafish       Construction <br /> homologues human genes differentially expressed          The MODB web project currently running Apache <br /> hematopoietic stem cell differentiation microarray          web server [21] PHP [1] Linux (Intel <br /> analysis [8].                                                  Xeon, 2.8 GHz, CentOS release 4.4). The project architec- <br />                                                                ture developed according Model-View-Control- <br /> Currently, zebrafish community stores published        ler (MVC) design paradigm. The Model (database tables) <br /> sequence-related results web-based databases.       stored MySQL relational database [22]. Major tables <br /> The principal database zebrafish-related research   include Morpholino, Staff, Lab, Test, Test_Result and <br /> Zebrafish Information Network (ZFIN) [13,14]. Ensembl          store information pertinent morpholino and <br /> [5], likewise, stores sequences targeted published mor-     users (Figure 1). The View developed help of <br />  <br />  <br />                                                                                                                    Page 2 10 <br />                                                                                              (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                    http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br />  <br /> Figure 1 <br /> Entity-Relationship diagram MODB <br /> Entity-Relationship diagram MODB. The organization data contained MODB centered unique mor- <br /> pholino sequence. Tables pertaining morpholino-specific information located upper left corner (solid box). <br /> The tables lower left corner establish relations laboratories, users security level designation (dashed <br /> line box). The tables right half diagram represent tests test components anatomical structure, <br /> screen defect observed (unboxed area). <br />  <br />  <br />  <br />  <br /> Smarty [23], template engine PHP helps             anatomy browser written PHP incorporated in <br /> designer make modifications layout break-         MODB. <br /> ing PHP logic. PHP scripts control connections <br /> between Model View. The GD graphics library          Content <br /> (a PHP module) [24] used graphic functions dis-       The unique identifier MODB sequence each <br /> played summary page.                                      morpholino. Many morpholino targets screens are <br />                                                                  derived transcriptome sequence data The Insti- <br /> NCBI BLAST 2.2.13 [25] invoked PHP sys-            tute Genomic Research (TIGR) [27] GenBank [28] <br /> tem calls sequence search functions. The MODB             subject change new sequence data becomes <br /> BLAST target database updated automatically            available. For unique morpholino entered the <br /> new morpholino sequence entered database.            database, accompanying set tables devoted <br /> The ZFIN anatomy ontology [26] downloaded                    information pertaining morpholino as <br /> (release date 08/16/2007) ZFIN web site          name, designator, laboratory created in, contact person, <br /> OBO format [26] transferred database. An            target name, target sequence, gene targeted, reference <br />  <br />  <br />                                                                                                                      Page 3 10 <br />                                                                                                (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                   http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br />  <br /> Figure 2 <br /> Morpholino-specific information <br /> Morpholino-specific information. Information morpholino sequence, laboratory morpholino designa- <br /> <span id='am-30' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-33'  ></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>tor</span> (box number), creating laboratory, initiating user reference images associated morpholino displayed <br /> on initial morpholino page morpholino. <br />  <br />  <br />  <br />  <br /> images number tests created (Figure 2). This con-        tests, caused toxic effects high cell <br /> cise format allows key information regarding mor-           death and/or mortality, produced pleiotropic effects we <br /> pholino viewed once.                                   decipher. This information use sci- <br />                                                                 entific community optimizing morpholino design and <br /> Early 2007, MODB contained 700 unique mor-              learning dosage data overall effect previously <br /> pholino sequences generated screens, indi-         designed tested morpholinos. <br /> vidual projects control morpholinos. The first <br /> generation Secretome Screen [9] largest overall con-     Since MODB living database, new data regularly <br /> tributor morpholino sequences. From 150 candidate            submitted. In coming year 50 150 <br /> genes sequences, 209 morpholinos designed. Each            morpholinos added database contin- <br /> morpholino subjected variety morphological          uing Hematopoietic Stem Cell secretome screens. To <br /> and molecular assays determine effect down-reg-       allow release data defined formats quality, <br /> ulating particular gene product 5-days    MODB data sharing quality control levels. Users <br /> zebrafish development [9]. This Secretome Screen identi-        capable accessing modifying data related to <br /> fied 26/150 gene targets produced reproducible             designated morpholinos. To facilitate collaborative <br /> effects specific aspects early development early       efforts, shared designation developed allow <br /> morphology variety organ systems including vis-        members consortium laboratories access data related <br /> ual, renal, cardiovascular, cranio-facial, hematopoiesis,       morpholinos designed user laborato- <br /> and pigment. The focused Hematopoietic Stem                ries connected relationship MODB. <br /> Cells Screen identified 16/64 targets necessary    The designation (&apos;public&apos;) allows morpholino data <br /> proper blood development [8]. The bulk data              used generate summary reports available to <br /> entered MODB (124/150 target gene morpholinos           open scientific community MODB website. <br /> the Secretome Screen 48/64 Hematopoietic             Nearly morpholinos (779/784) MODB, the <br /> Stem Cell Screen) consists morpholinos           exception non-zebrafish morpholinos, designated <br /> observed developmentally specific effect screening       public MODBv1. <br />  <br />  <br />                                                                                                                     Page 4 10 <br />                                                                                               (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                 http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br />  <br /> Figure 3 defect check box <br /> Common <br /> Common defect check box. Entry observations following morpholino injection zebrafish embryos cata- <br /> logued using fixed vocabulary checkbox form. Penetrance observed defect recorded subsequent analysis. <br />  <br />  <br />  <br />  <br />                                                                                                                  Page 5 10 <br />                                                                                            (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                 http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br />  <br /> Figure 4 <br /> Correspondence    defect description PATO designation <br /> Correspondence defect description PATO designation. As work ongoing establishing cross- <br /> species phenotypic vocabulary PATO, terminology constantly evolving. Defect names used maintain con- <br /> sistency rooted PATO designations translation table. When PATO term exists adequately the <br /> defect observed, proposed term submitted NCBO ZFIN. <br />  <br />  <br />  <br /> Utility                                                       administrators enter new listings provide basic <br /> Data Entry                                                    information morpholino. Authorized users sub- <br /> A major function MODB data compilation mor-          mit morpholino sequence results specific mor- <br /> pholino-specific information generated primary             phological molecular experiments given <br /> research. There main ways add data. Currently,     observation point dose. Researchers apply an <br />  <br />  <br />                                                                                                                  Page 6 10 <br />                                                                                            (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                  http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br /> authorized-user identification password (see author        search keyword, anatomical structure fixed defects <br /> contact information).                                          term allows future integration fixed- <br />                                                                vocabulary phenotype databases WormBase and <br /> These results called &apos;tests&apos;. In early 2007, MODB con-     Mouse Phenome database [15,29]. To utilize data <br /> tained 9000 individual test entries. A free text notes    entered MODB meaningful manner, devel- <br /> field permits detailed description defects observed mor-    oped variety search functions. In addition the <br /> pholino-injected embryos fixed vocabulary defect        parameters mentioned (laboratory designator, <br /> term selected. The defects ontology check box         morpholino name, morpholino target name, gene tar- <br /> was established common scientific terms used       geted, laboratory created in, keyword), MODB be <br /> screening protocols (Figure 3). Specific in-laboratory sci-    queried controlled defects vocabulary. This defects <br /> entific descriptions correspond established defects         ontology developed conjunction ZFIN to <br /> ontology. The reduction free text promotion      model current status Phenotypic Attribute Trait <br /> fixed vocabularies increases searchability MODB     Ontology (PATO) developed National Center for <br />  <br />  <br />  <br />  <br /> Figure search <br /> MODB   5      functions <br /> MODB search functions. A. Data retrieval MODB simplified morpholino gene targeted. Results be <br /> sorted morpholino gene targeted. B. An additional search public interface allows searches based PATO <br /> term anatomical structure affected. C. An additional sequence-based search using morpholino target sequence be <br /> performed. <br />  <br />  <br />  <br />  <br />                                                                                                                    Page 7 10 <br />                                                                                              (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                 http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br /> Biomedical Ontology (NCBO) [30]. To facilitate use        nology (Figure 4). Initially, PATO designed to <br /> of common language, MODB users assess phenotypes         defect observed anatomical structure (the <br /> using defect descriptions linked current PATO termi-       entity, e.g. tail) using trait (e.g. relative length) and <br />  <br />  <br />  <br />  <br /> Figuresummary <br /> Public  6       page <br /> Public summary page. A publicly accessible summary page available released morpholinos MODB. This page con- <br /> tains morpholino target information name, sequence etc. review knockdown results dosing for <br /> mortality normalcy. A list tests performed, defects observed images displayed. <br />  <br />  <br />  <br />  <br />                                                                                                                  Page 8 10 <br />                                                                                            (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                  http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br /> attribute trait (e.g. short length). This   information (reviewed [2,6,7,31-33]). Efficient data <br /> subsequently replaced entity quality          storage critical allow retrieval results mean- <br /> designation (e.g. tail, short length). MODB&apos;s defects          ingful fashion facilitate data mining. MODB a <br /> ontology contains version PATO. When          non-proprietary, relational database online accessi- <br /> PATO term exists adequately defect             bility acts storehouse critical information <br /> observed, proposed term submitted NCBO ZFIN        relating sequence-specific morpholino knockdown <br /> to assist development cross-species phenotypic     screens. MODB is, knowledge, unique the <br /> ontology. The importance common phenotypic                available databases offers searchable mor- <br /> vocabulary resides ability perform cross-species     pholino-specific data fixed-vocabulary defects onto- <br /> analysis gene function conserved pathways.              logical cataloging morphants allows cross- <br />                                                                species analysis using common, PATO-derived ontol- <br /> Data retrieval                                                 ogy. Cross-linking ZFIN provides additional value of <br /> The usefulness data stored database       MODB research community. <br /> subject retrieval methods available. MODB has <br /> been designed numerous search sort functions          Continued screening efforts zebrafish using mor- <br /> that allow retrieval morpholino sequence, mor-          pholinos sequence-specific knockdown or <br /> pholino gene targeted (Figure 5A). An                 knockout technologies expand knowledge of <br /> advanced search feature public interface allows         gene function. MODB provides central repository for <br /> users search anatomy phenotype identify           morpholino information. This schema potentially <br /> morpholinos given defect (Figure 5B). A sequence-       applicable sequence-based screening approach in <br /> based search available, using target mor-   zebrafish related systems. For example, MODB cur- <br /> pholino sequence sequence morpholino             rently adapted creation transposon data- <br /> (Figure 5C). By offering numerous search sorting           base includes insertional mutagenesis expression <br /> options, results nearly 800 morpholinos       analysis data. <br /> 9000 tests accessed analyzed scien- <br /> tific insight knockdown effect specific mor-     Conclusion <br /> pholino target, genes effecting developmental         By offering public access schema data con- <br /> pathways, non-specific morpholino effects.                 tained MODB, hope aid advancement <br />                                                                functional genetic studies gene annotation well <br /> MODB v1 Public Interface                                       improvements morpholino analysis design. The <br /> To facilitate display digestion large     dissemination valuable knowledge gained large <br /> data stored MODB, summary sheet mor-         screens Secretome [9] Hematopoietic <br /> pholino created (see construction content)      Stem Cell [8] screens <span id='am-19' about='obo:NCBITaxon_376232' typeof='obo:NCBITaxon_34667 obo:NCBITaxon_79514 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71528 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_192380 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-20' property="rdfs:label" content="Goes" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22'  ></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>goes</span> published gene target <br /> publicly available. Each morpholino summary contains           characterization provides foundation future <br /> sequence target information survival        research. <br /> normalcy (percent normal embryos observed overall or <br /> for specific screen types) curves based dose (Figure 6).    Availability requirements <br /> Several search functions used access infor-     MODB accessible web http://www.secre <br /> mation including gene targeted, BLAST                tomes.umn.edu/MODB/. To added authorized <br /> advanced search function facilitates searching        user, contact corresponding author M.K. S.C.E. <br /> morpholinos PATO designation anatomical struc-          Morpholinos readily available purchase through <br /> ture affected (Figure 5B). MODB presents users       Gene Tools [34] Open Biosystems [35]. The authors <br /> the valuable knowledge, mortality dose             affiliation, financial otherwise, cor- <br /> curves, effectiveness morphant phenotype. Currently,       porate entity. <br /> 101 morpholinos contain hyperlinks specific mor- <br /> pholino entries ZFIN hyperlinks provided        Authors&apos; contributions <br /> gene targets MODBv1.                                        MK designed, tested managed database. TL and <br />                                                                YR responsible programming, debugging and <br /> Discussion                                                     informatics support. BB aided conception, design <br /> With advent whole-genome sequencing technology,           testing database. EL SE aided concep- <br /> greater emphasis gene function emerging. Func-           tion design database coordinated the <br /> tional genomic screening tools available zebrafish     project whole. All authors read approved final <br /> as ENU mutagenesis, GripNAs, RNAi, insertional muta-           manuscript. <br /> genesis morpholinos allow large functional gene <br /> screens produce enormous biological <br />  <br />  <br />                                                                                                                     Page 9 10 <br />                                                                                               (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                                       http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br />  <br /> Acknowledgements                                                              20.   Rana <span id='am-24' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-25' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-29'  ></span>AA</span>, Collart C, Gilchrist MJ, Smith JC: Defining synphenotype <br /> The authors like acknowledge numerous collaborations              groups Xenopus tropicalis use antisense morpholino <br />                                                                                     oligonucleotides. PLoS Genet 2006, 2(11):e193. <br /> have contributed development MODB laboratories       21.   Apache Web Server [http://www.apache.org] <br /> Drs. Farber, Hammerschmidt, Pickart, Schimmenti, Sivasubbu, Verfail-      22.   Nasevicius A, Ekker SC: Effective targeted gene &apos;knockdown&apos; in <br /> lie. We like acknowledge bioinformaticians               zebrafish. Nat Genet 2000, 26(2):216-220. <br /> worked project including Dr. Eric Klee Kyong Jin Shim. Funding    23.   Smarty [http://smarty.php.net] <br />                                                                               24.   GD Graphics [http://www.boutell.com/gd/] <br /> for project obtained NIH SCE (GM63904                25.   Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ: Basic local <br /> CA65493). This work supported University Minnesota            alignment search tool. J Mol Biol 1990, 215(3):403-410. <br /> Supercomputing Institute.                                                     26.   ZFIN Anatomy Ontology                  [http://www.geneontology.org/ <br />                                                                                     GO.format.shtml] <br />                                                                               27.   The Institute Genomic Research (TIGR)                    [http:// <br /> References                                                                          www.tigr.org] <br /> 1.    Kimmel CB, Ballard WW, Kimmel SR, Ullmann B, Schilling TF: Stages       28.   GenBank [http://www.ncbi.nlm.nih.gov/Genbank/index.html] <br />       embryonic development zebrafish. Dev Dyn 1995,                29.   WormBase [http://www.wormbase.org/] <br />       203(3):253-310.                                                         30.   National Center Biomedical Ontology (NCBO) [http:// <br /> 2.    Geisler R, Rauch GJ, Geiger-Rudolph S, Albrecht A, van Bebber F,              www.bioontology.org/] <br />       Berger A, Busch-Nentwich E, Dahm R, Dekens MP, Dooley C, et al.:        31.   Amsterdam A: Insertional mutagenesis zebrafish: genes for <br />       Large-scale mapping mutations affecting zebrafish devel-                   development, genes disease. Brief Funct Genomic Proteomic <br />       opment. BMC Genomics 2007, 8(1):11.                                           2006, 5(1):19-23. <br /> 3.    Draper BW, McCallum CM, Stout JL, Slade AJ, Moens CB: A high-           32.   Amsterdam A, Hopkins N: Mutagenesis strategies zebrafish <br />       throughput method identifying N-ethyl-N-nitrosourea                       identifying genes involved development disease. <br />       (ENU)-induced point mutations zebrafish. Methods Cell Biol                 Trends Genet 2006, 22(9):473-478. <br />       2004, 77:91-112.                                                        33.   Dahm R, Geisler R: Learning small fry: zebrafish a <br /> 4.    Wienholds E, van Eeden F, Kosters M, Mudde J, Plasterk RH, Cuppen             genetic model organism aquaculture fish species. Mar Bio- <br />       E: Efficient target-selected mutagenesis zebrafish. Genome                 technol (NY) 2006, 8(4):329-345. <br />       Res 2003, 13(12):2700-2707.                                             34.   Gene Tools [http://www.gene-tools.com/] <br /> 5.    Amsterdam A, Burgess S, Golling G, Chen W, Sun Z, Townsend K,           35.   Open Biosystems [http://www.openbiosystems.com/] <br />       Farrington S, Haldi M, Hopkins N: A large-scale insertional muta- <br />       genesis screen zebrafish. Genes Dev 1999, 13(20):2713-2724. <br /> 6.    Pickart MA, Sivasubbu S, Nielsen AL, Shriram S, King RA, Ekker SC: <br />       Functional genomics tools analysis zebrafish pig- <br />       ment. Pigment Cell Res 2004, 17(5):461-470. <br /> 7.    Taylor MR, Kikkawa S, Diez-Juan A, Ramamurthy V, Kawakami K, <br />       Carmeliet P, Brockerhoff SE: The zebrafish pob gene encodes a <br />       novel protein required survival red cone photo- <br />       receptor cells. Genetics 2005, 170(1):263-273. <br /> 8.    Eckfeldt CE, Mendenhall EM, Flynn CM, Wang TF, Pickart MA, Grin- <br />       dle SM, Ekker SC, Verfaillie CM: Functional analysis human <br />       hematopoietic stem cell gene expression using zebrafish. <br />       PLoS Biol 2005, 3(8):e254. <br /> 9.    Michael A, Pickart EWK, Nielsen Aubrey L, Sivasubbu Sridhar, <br />       Mendenhall Eric M, Bill Brent R, Chen Eleanor, Eckfeldt Craig E, <br />       Knowlton Michelle, Robu Mara E, Larson Jon D, Deng Yun, Schim- <br />       menti Lisa A, Ellis Lynda BM, Verfaillie Catherine M, Hammerschmidt <br />       Matthias, Farber Steven A, Ekker Stephen C: Genome-Wide <br />       Reverse Genetics Framework Identify Novel Functions of <br />       Vertebrate Secretome. PLoS One 2006, 1(1):e104. <br /> 10.   Klee EW, Ellis L BM: Evaluating eukaryotic secreted protein <br />       prediction. BMC Bioinformatics 2005, 6:256. <br /> 11.   Sadl VS, Sing A, Mar L, Jin F, Cordes SP: Analysis hindbrain pat- <br />       terning defects caused kreisler(enu) mutation reveals <br />       multiple roles Kreisler hindbrain segmentation. Dev Dyn <br />       2003, 227(1):134-142. <br /> 12.   Klee EW, Shim KJ, Pickart MA, Ekker SC, Ellis LB: AMOD: mor- <br />       pholino oligonucleotide selection tool. Nucleic Acids Res <br />       2005:W506-511. <br /> 13.   Miller-Bertoglio V, Carmany-Rampey A, Furthauer M, Gonzalez EM, <br />       Thisse C, Thisse B, Halpern ME, Solnica-Krezel L: Maternal and <br />       zygotic activity zebrafish ogon locus antagonizes BMP <br />       signaling. Dev Biol 1999, 214(1):72-86. <br />                                                                                         Publish Bio Med Central every <br /> 14.   Sprague J, Bayraktaroglu L, Clements D, Conlin T, Fashena D, Frazer              scientist read work free charge <br />       K, Haendel M, Howe DG, Mani P, Ramachandran S, et al.: The                    &quot;BioMed Central significant development for <br />       Zebrafish Information Network: zebrafish model organ- <br />                                                                                     disseminating results biomedical researc h lifetime.&quot; <br />       ism database. Nucleic Acids Res 2006:D581-585. <br /> 15.   Mouse Phenome Database [http://phenome.jax.org/pub-cgi/phe                        Sir Paul Nurse, <span id='am-6' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Cancer</span> Research UK <br />       nome/mpdcgi?rtn=docs/home] <br /> 16.   Bogue MA, Grubb SC, Maddatu TP, Bult CJ: Mouse Phenome                          Your research papers be: <br />       Database (MPD). Nucleic Acids Res 2007:D643-649.                                  available free charge entire biomedical community <br /> 17.   Horn T, Arziman Z, Berger J, Boutros M: GenomeRNAi: data-                       peer reviewed published immediately acceptance <br />       base cell-based RNAi phenotypes. Nucleic Acids Res <br />       2007:D492-497.                                                                    cited PubMed archived PubMed Central <br /> 18.   GenomeRNAi [http://www.dkfz.de/signaling2/rnai/ernai.html]                        &#8212; copyright <br /> 19.   Xenopus tropicalis Morpholino Screen                  [http://www.gur <br />       don.cam.ac.uk/~smithlab/screens/Xenopus-morpholino-pilot/mor             Submit manuscript here:                                BioMedcentral <br />       pholino-screen-lists.html#synphenotypes]                                 http://www.biomedcentral.com/info/publishing_adv.asp <br />  <br />  <br />  <br />  <br />                                                                <br /> </body></html>