<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Methods Enzymol. 2010 ; 470: 181&#8211;204. doi:10.1016/S0076-6879(10)70008-2. <br />  <br />  <br />  <br />                            Making temperature-sensitive mutants <br />  <br />                            Shay Ben-Aroya1,*, Xuewen Pan2,*, Jef D. Boeke3, Philip Hieter1 <br />                            1 Michael Smith Laboratories, Department Medical Genetics, University British Columbia, <br />  <br />                            Vancouver, BC Canada <br />                            2Verna Marrs McLean Department Biochemistry Molecular Biology, Department of <br />                            Molecular Human Genetics, Baylor College Medicine, One Baylor Plaza, Houston, Texas <br />                            77030, USA <br />                            3High Throughput Biology Center, Johns Hopkins University school Medicine, Baltimore MD <br />                            21205 <br />  <br />                            Abstract <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  The study temperature sensitive (Ts) mutant phenotypes fundamental gene identification and <br />                                  dissecting essential gene function. In chapter &#8220;shuffling&#65533;? methods for <br />                                  producing Ts mutants using combination PCR, vivo recombination, transformation of <br />                                  diploid strains heterozygous knockout desired mutation. The main difference between <br />                                  methods type strain produced. In &#8220;plasmid&#65533;? version, product knockout <br />                                  mutant carrying centromeric plasmid carrying Ts mutant. In &#8220;chromosomal&#65533;? version, The <br />                                  Ts allele integrated directly endogenous locus, albeit entirely native configuration. <br />                                  Both variations ir strengths weaknesses, discussed here. <br />  <br />  <br />                            Introduction <br />                                              The study temperature sensitive (Ts) mutant phenotypes proven fundamental <br />                                              approach identification gene sets essential various aspects biology and <br />                                              obtaining detailed understanding essential gene function. While observation that <br />                                              temperature sensitive mutations represent general <span id='am-39' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-40' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> mutation recognized the <br />                                              &#8216;50s (Horowitz, 1950), targeted screen, isolation, analysis Ts mutants (382 <br />                                              mutations located 37 genes scattered widely bacteriophage T4 genome) by <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Edgar Lielausis 1963 (Edgar Lielausis, 1964). Lee Hartwell, 1967, reported the <br />                                              isolation 400 Ts mutations S. cerevisiae, caused defects essential processes <br />                                              including cell division, protein, RNA, DNA synthesis (Hartwell, 1967). Over past <br />                                              40 years, isolation analysis Ts mutations essential genes linchpin <br />                                              technology investigating genetics molecular biology essential processes all <br />                                              experimental organisms. <br />  <br />                                              Ts mutations typically missense mutations, retain function specific essential <br />                                              gene standard (permissive) low temperature, lack function defined high (non- <br />                                              permissive) temperature, exhibit partial (hypomorphic) function intermediate (semi- <br />                                              permissive) temperature. Such mutants make possible analysis physiologic changes that <br />                                              follow controlled inactivation gene gene product shifting cells non-permissive <br />                                              temperature, offering powerful approach analysis gene function. <br />  <br />  <br />  <br />  <br />                            *These authors contributed equally work <br />                             Ben-Aroya et al.                                                                                               Page 2 <br />  <br />  <br />                                               Essential genes, <span id='am-85' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>, encode critical cellular functions buffered by <br />                                               redundant functions pathways (Hartman et al., 2001). Essential genes shown to <br />                                               highly dense hubs genetic interaction networks involved aspects of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               basic cellular function (Jeong et al., 2001). Furthermore, essential genes tend highly <br />                                               conserved evolution; 38% essential yeast proteins easily identifiable counterparts <br />                                               humans, versus 20% nonessential genes (Hughes, 2002). <br />  <br />                                               Despite importance, functions essential yeast proteins studied. <br />                                               <span id='am-96' about='ncbitaxon:in_part' typeof='owl:Thing'>In part</span>, absence essential gene representation genome wide haploid <br />                                               mutant collections, cover ~5000 non-essential yeast genes. Thus, no <br />                                               comparable systematic haploid mutant collection currently exists ~1000 essential genes <br />                                               S. cerevisiae. The frequency sites mutable reduced conditional function highly <br />                                               gene-specific; example, highly conserved proteins, random single missense mutation <br />                                               expected, vast majority positions protein, cause complete loss <br />                                               function mutated. Therefore, genetic screens using random mutagenesis approach <br />                                               rarely reach saturation &#8220;mutability&#65533;? varies widely genes. <br />  <br />                                               Here report detailed protocols methodologies allow systematic isolation of <br />                                               Ts alleles essential genes interest. The method plasmid-based, second is <br />                                               genome integration based, specific advantages depending application. <br />                                               Both methods exploit features &#8220;haploid convertible&#65533;? heterozygous diploid collection, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               allows introduction library mutagenized essential gene copies the <br />                                               heterozygous diploid subsequent direct selection haploids deleted the <br />                                               target essential gene carry individual members mutagenized essential gene <br />                                               library, using &#8220;diploid shuffle&#65533;? technique (see below). Several useful corollary <br />                                               methods transferring extant Ts alleles specific gene constructs (eg, fusion proteins) are <br />                                               presented. Finally, note laboratories process generating complete <br />                                               set Ts alleles essential genes S. cerevisiae, distributed a <br />                                               resource scientific community completed (see Ben Aroya etal 2008 details). <br />                                               For specific essential genes study individual laboratories, however, useful, <br />                                               using methods described here, generate additional series independent Ts alleles for <br />                                               detailed functional analysis. Furthermore, mutagenized libraries specific essential non- <br />                                               essential genes screened conditional viability variety conditions are <br />                                               normally sub-lethal wildtype strain (eg, sublethal doses drugs) using methods <br />                                               described here. <br />  <br />                               Part I. Diploid shuffle &#8211; plasmid method <br />                               General description diploid shuffle &#8211; plasmid method <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Much like traditional plasmid-shuffling methods, mutants generated version the <br />                                               diploid shuffle plasmid-borne alleles easily transferred tested in <br />                                               different strain backgrounds. The experimental procedure outlined Fig. 1. First, the <br />                                               endogenous promoter (including 5&#8242; untranslated region, 5&#8242;UTR, ~500bp) terminator <br />                                               (3&#8242;UTR, ~500bp) gene (or favorite gene, YFG) PCR amplified and <br />                                               cloned tandem centromere-based yeast-E. coli shuttle vector, contains URA3 <br />                                               selectable marker yeast cells, pRS416. The resultant promoter/terminator clone <br />                                               subsequently linearized endonuclease, typically NotI, cuts site pre- <br />                                               engineered promoter terminator. Simultaneously, sequence YFG, <br />                                               including open reading frame (ORF), complete promoter terminator <br />                                               regions, randomly mutagenized error prone PCR. The linearized promoter/terminator <br />                                               plasmid PCR products combined co-transformed haploid-convertible <br />                                               heterozygous diploid deletion mutant (YFG/yfg&#916;::kanMX4) gene being <br />                                               mutagenized. The mutagenized PCR product cloned URA3 plasmid via <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                             Page 3 <br />  <br />  <br />                                               recombination mediated terminal homologous DNA sequences PCR products <br />                                               linearized vector. The Ura+ transformants subsequently cultured sporulation <br />                                               medium converted haploid cells growing medium allow growth only <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               haploid MATa G418R Ura+ cells. In cells, chromosomal wild-type copy YFG is <br />                                               deleted, allowing direct observation phenotypes plasmid-borne alleles. To screen <br />                                               conditional alleles, haploid cells grown permissive condition as <br />                                               low temperature. Colonies formed subsequently replica-plated fresh plates permissive <br />                                               nonpermissive conditions. Conditional alleles identified grow the <br />                                               permissive non-permissive condition subsequently verified. <span id='am-50' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-51'  ></span><span id='am-52' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> method has <br />                                               used create thermosensitive (Ts) alleles multiple essential genes large <br />                                               collection methyl methanesulfonate (MMS) hypersensitive alleles POL30 (Huang et al., <br />                                               2008;Lin et al., 2008). Here outline detailed methods generating verifying <br />                                               Ts alleles essential gene. <br />  <br />                               Materials <br />                                               Media&#8212;Haploid selection synthetic medium SC&#8211;Ura&#8211;Leu&#8211;His&#8211;Arg+G418+Can: dextrose, <br />                                               20 g/L; yeast nitrogen base amino acids ammonium sulfate, 1.7 g/L; SC&#8211;Ura&#8211; <br />                                               Leu&#8211;His&#8211;A dropout mix, 2g/L; sodium glutamate, 1g/L; G418, 200 mg/L; L-canavanine <br />                                               (Sigma, Cat# C1625), 60 mg/L; Agar, 2%. The sodium glutamate substituted ammonium <br />                                               sulfate nitrogen source makes G418 selection reliable minimal <br />                                               medium. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               SC&#8211;Ura: dextrose, 20g/L; yeast nitrogen base amino acids ammonium sulfate, <br />                                               1.7g/L; SC&#8211;Ura dropout mix, 2g/L; ammonium sulfate, 5g/L; Agar, 2%. <br />  <br />                                               Liquid YPD: yeast extract, 10g/L; peptone, 20g/L; dextrose, 20g/L. <br />  <br />                                               Solid liquid sporulation medium: potassium acetate, 10g/L; zinc acetate 0.05g/L, or <br />                                               2% agar respectively. <br />  <br />                                               Solid <span id='am-5' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-6' property="rdfs:label" content="Luria" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Luria</span> Broth (LB) plus carbenicillin: yeast extract, 10g/L; Tryptone, 5g/L; sodium <br />                                               chrolide, 10g/L; carbenicillin, 50mg/L; Agar, 2%. <br />  <br />                                               Liquid LB plus ampicillin: yeast extract, 10g/L; Tryptone, 5g/L; sodium chrolide, 10g/L; <br />                                               ampicillin, 50mg/L. <br />  <br />                                               Yeast Strains&#8212;The haploid-convertible heterozygous diploid knockout mutants (MAT a&#945;/ <br />                                               ura3&#916;0/ura3&#916;0 leu2&#916;0/leu2&#916;0 his3&#916;1/his3&#916;1 lys2&#916;0/LYS2 met15&#916;0/MET15 can1&#916;::LEU2- <br />                                               MFA1pr::His3/CAN1 YFG/yfg&#916;::KanMX; OpenBiosystems Cat# YSC4428) (<span id='am-42' about='obo:NCBITaxon_9596' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-44'  ></span><span id='am-45' property="oboInOwl:hasAlternativeId" content="NCBITaxon:317028" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasAlternativeId" content="NCBITaxon:445518" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Pan" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="chimpanzees" datatype="xsd:string"></span>Pan</span> et al., <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               2006) used screen Ts alleles. Chemically competent DH5&#945; cells prepared described <br />                                               (Inoue et al., 1990) used cloning plasmid recovering yeast. <br />  <br />                                               <span id='am-91' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>Plasmids</span>&#8212;It essential method use plasmid vector contains YFG <br />                                               promoter terminator separated unique endonuclease (typically NotI) recognition site. <br />                                               Due limited auxotrophic markers available haploid-convertible heterozygous <br />                                               diploid knockout mutants, normally use plasmids containing URA3 selectable marker <br />                                               pRS416 (Brachmann et al., 1998; Sikorski Hieter, 1989) YCplac33 (Gietz and <br />                                               Sugino, 1988). Other URA3 CEN <span id='am-81' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-82' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> work. <br />  <br />                                               Yeast genomic DNA&#8212;A genomic DNA sample isolated wild-type yeast strain <br />                                               BY4743 MATa/&#945; (Brachmann et al., 1998) used template cloning promoter <br />                                               terminator YFG mutagenizing entire sequence PCR. <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                              Page 4 <br />  <br />  <br />                               Methods <br />                                               Constructing promoter/terminator clone&#8212;In past, used <br />                                               endonuclease restriction enzyme digestion ligation construct promoter/terminator <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               clone. First, promoter terminator YFG separately PCR-amplified using primers <br />                                               contain endonuclease recognition sites, example, HindIII/NotI promoter and <br />                                               NotI/BamHI terminator. The PCR products digested HindIII/NotI and <br />                                               NotI/BamHI, respectively, ligated pRS416 (or YCplac33) digested HindIII/BamHI <br />                                               3-piece ligation reaction. The ligation products transformed DH5&#945; competent cells <br />                                               candidate clones selected solid LB plus carbenicillin. More recently, have <br />                                               adopted modified version sequence ligation independent cloning (SLIC) procedure <br />                                               (Li Elledge, 2007) (Figure 2). This method does require endonuclease digestion the <br />                                               inserts greatly simplifies primer designs experimental procedures, especially <br />                                               large number genes processed simultaneously. This relatively new <br />                                               cloning technique described greater detail. <br />                                                   1. Design PCR primers: promoter forward (PF), promoter reverse (PR), terminator <br />                                                   forward (TF), terminator reverse (TR). In addition gene specific sequences their <br />                                                   3&#8242; termini, PF TR primers contains 30 bp sequence 5&#8242; end either <br />                                                   identical complimentary ends pRS416 (or YCplac33) linearized the <br />                                                   multicloning site (Fig. 2). The PR TF primers completely complementary both <br />                                                   contain parts: ~20 bp promoter- terminator-specific sequences ends and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   NotI recognition site (8 bp) middle (Fig. 2). <br />                                                   2a. PCR-amplify promoter terminator according conditions listed Table <br />                                                   1. Here genomic DNA BY4743a/&#945; used PCR template. Platinum Pfx DNA <br />                                                   polymerase (Invitrogen, Cat# 11708) used relative robustness high fidelity. <br />                                                   Others enzymes similar features Phusion (New England Biolabs, NEB, <br />                                                   Cat# F-540) KOD (Novagen, Cat# 71085-3) DNA polymerases used. <br />                                                   2b. Digest ~1ug pRS416, YCplac33, URA3 plasmid endonuclease <br />                                                   multicloning site generate ends competent cloning PCR products <br />                                                   SLIC. <br />                                                   3. Gel-purify PCR products digested vector DNA using gel extraction kit <br />                                                   (Qiagen, Cat# 28706 equivalent) following manufacturer&#8217;s instruction. Here the <br />                                                   PCR products promoter terminator combined purification. Elute <br />                                                   purified sample 25&#956;l provided elution buffer. <br />                                                   4. Set reaction according Table 2. Here DNA resection 3&#8242; 5&#8242; direction (by <br />                                                   T4 DNA polymerase, NEB, Cat# M0203) annealing complementary single <br />                                                   strand overhangs occurs reaction. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   5. Immediately use 2&#956;l reaction transform 20&#956;l chemically competent <br />                                                   DH5&#945; cells following standard protocol. This includes incubating cell/DNA <br />                                                   mixture sequentially ice 30 minutes, 42&#176;C 90 seconds, ice 2 <br />                                                   minutes, 37&#176;C 30 minutes (in presence 200 &#956;l liquid LB medium). <br />                                                   6. Plate 100&#956;l transformation mixture solid LB plus carbenicillin select for <br />                                                   single colony transformants. <br />                                                   7. Screen positive clones using colony-PCR PF PR primers according to <br />                                                   Table 1 subtle modifications. Here cells single colonies instead yeast <br />                                                   genomic DNA used provide PCR templates. <br />                                                   8. Prepare plasmid DNA samples 2&#8211;3 positive clones using mini-spin kit (Qiagen, <br />                                                   Cat# 27106 equivalent) following manufacture&#8217;s instructions. <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                              Page 5 <br />  <br />  <br />                                                   9. Verify plasmid DNA sequencing using primers read the <br />                                                   inserts (promoter terminator) directions vector backbone. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Mutagenizing YFG error prone PCR&#8212;Mutagenesis YFG using error-prone PCR <br />                                               performed essentially described previously (Leung et al, 1989). Again, genomic DNA <br />                                               sample wild-type yeast strain BY4743a/&#945; used DNA template. The PF TR <br />                                               primers described used amplify full-length gene ~500bp flanking <br />                                               sequences. TaKaRa Ex Taq (Cat# RR001A) LA Taq (Cat# RR002B), ~4 times <br />                                               accurate normal Taq polymerase, used robustness. Induction <br />                                               mutation rates achieved adding Mn2+ PCR final concentration 10&#8211;150 <br />                                               &#956;M arbitrarily defined, higher concentrations smaller genes (genes&#8217; sizes range <br />                                               0.5&#8211;5kb). <br />                                                   1.   Set independent reactions gene according Table 3 reduce <br />                                                        potential founder effects. <br />                                                   2.   After PCR, pool samples. <br />                                                   3.   Examine PCR products agarose gel electrophoresis using 1&#8211;2 &#956;l sample <br />                                                        ensure successful PCR amplification. <br />  <br />                                               Linearizing plasmid DNA promoter/terminator clone&#8212;Two &#956;g plasmid <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               DNA promoter/terminator clone digested NotI (NEB, Cat# R0189) NEB buffer <br />                                               3 20&#956;l reaction. The reaction incubated 37&#176;C overnight subsequently at <br />                                               65&#176;C 20 minutes inactivate NotI. A small aliquot digestion product examined <br />                                               agarose gel electrophoresis ensure complete digestion plasmid. <br />  <br />                                               Combining concentrating PCR products digested vector&#8212;The <br />                                               mutagenized PCR products (approximately 10&#8211;20 &#956;g 200 &#956;l) linearized plasmid DNA <br />                                               promoter/terminator clone (~2 &#956;g) combined concentrated ethanol <br />                                               precipitation. <br />                                                   1.   Transfer mutagenesis PCR products NotI-digested promoter/terminator <br />                                                        plasmid DNA 1.7 ml microcentrifuge tube adjust volume ~200 &#956;l by <br />                                                        adding ddH2O. <br />                                                   2.   Add 5 &#956;l 4M ammonium acetate (pH 7.0) 500 &#956;l 100% ethanol DNA <br />                                                        sample mix briefly vortexing. Place ice 10 minutes. <br />                                                   3.   Precipitate DNA spinning &gt;12,000rpm microcentrifuge 7 minutes. There <br />                                                        tiny whitish DNA pellet the tube. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   4.   Carefully aspirate liquid wash DNA pellet 300 &#956;l 70% <br />                                                        ethanol. <br />                                                   5.   Spin &gt;12,000rpm microcentrifuge 3 minutes carefully aspirate ethanol. <br />                                                   6.   Dry DNA pellet speed vac. <br />                                                   7.   Resuspend DNA 28 &#956;l sterile ddH2O. <br />  <br />                                               Transforming yeast cells&#8212;The concentrated DNA sample PCR products the <br />                                               linearized vector transformed corresponding haploid-convertible heterozygous <br />                                               diploid yeast knockout mutant create mutagenized library YFG. <br />                                                   1.   Inoculate haploid-convertible YFG/yfg&#916;::kanMX4 heterozygous diploid mutant <br />                                                        5 ml liquid YPD incubate 30&#176;C overnight roller drum. <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                              Page 6 <br />  <br />  <br />                                                   2.   Transfer aliquot overnight culture 50 ml YPD liquid (starting 0.125 <br />                                                        OD600nm/ml 250 ml Erlenmeyer flask) incubate 30&#176;C shaking (at 200 <br />                                                        rpm) cell density ~0.5 OD600nm/ml. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   3.   Harvest culture 50 ml conical tube spinning 4000 rpm 3 minutes in <br />                                                        bench centrifuge discard medium. <br />                                                   4.   Resuspend cells 10 ml sterile ddH2O, centrifuge described previous <br />                                                        step, discard supernatant. <br />                                                   5.   Resuspend cells 10 ml 0.1 M lithium acetate (LiOAc), centrifuge, discard <br />                                                        supernatant before. <br />                                                   6.   Resuspend cells residual 0.1 M LiOAc total volume 100 &#956;l 1.7 ml <br />                                                        microcentrifuge tube. <br />                                                   7.   Make transformation mixture <span id='am-55' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-56' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> mix well: 480 &#956;l 50% polyethylene <br />                                                        glycol (PEG-3350, JTBaker, Cat# JTU221-9), 72 &#956;l 1M lithium acetate, 40 &#956;l of <br />                                                        heat-denatured herring sperm DNA (10mg/ml, Sigma, Cat# D6898) 28 &#956;l DNA <br />                                                        sample transformed (error-prone PCR products linearized promoter/ <br />                                                        terminator clone). <br />                                                   8.   Add transformation mixture yeast competent cells prepared step 6 and <br />                                                        immediately mixed pipetting P1000 pipettor followed vortexing <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                        (VMR, Vortexer 2) speed 5&#8211;10 seconds. <br />                                                   9.   Incubate transformation reaction 30&#176;C incubator 30 minutes. <br />                                                   10. Add 72 &#956;l dimethyl sulfoxide (DMSO; Qbiogene DMSO0001, Molecular Biology <br />                                                       Grade) transformation reaction immediately mixed thoroughly vortexing <br />                                                       speed 5&#8211;10 seconds. DMSO intrinsically sterile sterilization <br />                                                       needed. <br />                                                   11. Incubate transformation reaction 42&#176;C water bath 13 minutes. A <br />                                                   12. Spin cells 3600 rpm microcentrifuge 30 seconds pellet cells. <br />                                                   13. Aspirate supernatant resuspend cells 1 ml sterile ddH2O. <br />                                                   14. Take 0.2 &#956;l (1/5000) resuspended cells plate SC&#8211;Ura plate determine <br />                                                       transformation yield. A successful reaction typically yields library 105 106 <br />                                                       independent Ura+ transformants, &gt;90% recombinants PCR <br />                                                       products linearized plasmid DNA promoter/terminator clone. The <br />                                                       remainder primarily vector molecules cut place <br />                                                       rejoined nonhomologous end joining. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Sporulation&#8212;The rest yeast transformation reaction incubated 50 ml of <br />                                               fresh liquid SC&#8211;Ura 30&#176;C 2 days allow propagation library sporulated <br />                                               immediately convert transformants library haploid spores harbor mutant <br />                                               alleles YFG plasmid (see below). <br />                                                   1.   Grow yeast transformants nonselectively 50 ml liquid YPD incubating at <br />                                                        30&#176;C shaking (180&#8211;200 rpm) 3 hours refresh cells. <br />                                                   2.   Harvest cells 50 ml conical tube spinning 4000 rpm 3 minutes bench <br />                                                        centrifuge discard medium. <br />                                                   3.   Resuspend cells 40 ml sterile ddH2O spin 4000 rpm 3 minutes. <br />                                                   4.   Discard supernatant resuspend cells 50 ml liquid sporulation medium. <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                             Page 7 <br />  <br />  <br />                                                   5.   Incubate sporulation culture 250 ml flask 25&#176;C 4&#8211;6 days shaking <br />                                                        (180&#8211;200rpm). This typically rise 20&#8211;40% sporulation efficiency when <br />                                                        checked microscope. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   6.   Repeat Steps 2 3. <br />                                                   7.   Discard supernatant resuspend cells 10 ml sterile ddH2O. <br />                                                   8.   Spread aliquots (200 &#956;l) 10 &#215; serial dilutions sporulation culture onto <br />                                                        individual plates haploid selection medium SC&#8211;Ura&#8211;Leu&#8211;His&#8211;Arg+G418 <br />                                                        +Can incubate 25&#176;C 2&#8211;3 days determine efficiency producing <br />                                                        MATa G418R Ura+ haploid cells. <br />                                                   9.   Store rest spores ddH2O 4&#176;C later use. The viability spores can <br />                                                        maintained weeks way. <br />  <br />                                               Screening Ts alleles&#8212;After titer MATa G418R Ura+ haploid cells determined, <br />                                               library screened potential Ts mutants. We typically screen ~4,000 clones each <br />                                               gene. <br />                                                   1.   Spread spores solid SC&#8211;Ura&#8211;Leu&#8211;His&#8211;Arg+G418 +Can, aiming density <br />                                                        ~400 MATa G418R Ura+ haploid colonies formed plates. Store the <br />                                                        rest spores 4&#176;C backup. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   2.   Incubate plates 25&#176;C 3 days allow formation colonies ~2 mm in <br />                                                        diameter. <br />                                                   3.   Replica-plate colonies plate fresh plates haploid <br />                                                        selection medium mark orientations plates. It best pre-warm the <br />                                                        &#8220;nonpermissive&#65533;? plate 37&#176;C prior replica plating. <br />                                                   4.   Incubate daughter plate 25&#176;C 37&#176;C 1 day. <br />                                                   5.   Compare growth colony plates assess potential Ts phenotype and <br />                                                        select alleles form relatively robust colonies 25&#176;C ghost colonies 37&#176;C <br />                                                        candidate Ts mutants. If Ts colony detected, need screen more <br />                                                        clones using backup spores stored 4&#176;C, using 38&#176;C restrictive <br />                                                        temperature, both. <br />  <br />                                               Confirming Ts mutants&#8212;Candidate mutants picked re-streaked same <br />                                               haploid selection media re-tested Ts phenotypes incubating 25&#176;C 37&#176;C. <br />                                               The plasmids recovered confirmed Ts initial assay re- <br />                                               introduced individually haploid-convertible heterozygous diploid mutant test <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Ts phenotype linked plasmids. <br />                                                   1.   Grow candidate Ts mutant 1.5 ml liquid SC&#8211;Ura 25&#176;C saturated, <br />                                                        typically takes 1&#8211;2 days, depending particular alleles. <br />                                                   2.   Harvest cells strain microcentrifuge tube spinning 4000 rpm 1 <br />                                                        minute discard medium. <br />                                                   3.   Resuspend cells 500&#8211;1000 &#956;l sterile ddH2O repeat Step 2. <br />                                                   4.   Resuspend cells 40 &#956;l Lysis Buffer (50 mM Tris-HCl 7.5, 10 mM EDTA) <br />                                                        containing 5 mg/ml Zymolyase 100T (MP Biomedicals, Cat# 320931) incubate <br />                                                        37&#176;C 1 hour shaking 15-minute intervals. <br />                                                   5.   Add 40 &#956;l 10% SDS mix vortexing pipetting. <br />                                                   6.   Add 160 &#956;l 7.5M ammonium acetate mix vortexing pipetting. <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                            Page 8 <br />  <br />  <br />                                                   7.   Incubate sample &#8722;80&#176;C 15 minutes. <br />                                                   8.   Centrifuge &gt;12,000rpm 5 minutes 4&#176;C. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   9.   Carefully transfer 100 &#956;l clear supernatant new tube contains 75 &#956;l of <br />                                                        isopropanol mix inverting tube times. <br />                                                   10. Centrifuge &gt;12,000rpm 7 minutes room temperature precipitate DNA. <br />                                                   11. Wash DNA pellet 100 &#956;l 70% ethanol. <br />                                                   12. Carefully aspirate ethanol dry DNA pellet speed vac. <br />                                                   13. Resuspend DNA pellet 20 &#956;l sterile ddH2O. <br />                                                   14. Use 2 &#956;l DNA sample transform chemically competent DH5&#945; cells as <br />                                                       mentioned &#8220;constructing promoter/terminator clone&#65533;? section. <br />                                                   15. Purify plasmid DNA representative bacterial transformant using kit (Qiagen, <br />                                                       Cat# 27106 equivalent). <br />                                                   16. Individually transform plasmid haploid-convertible heterozygous <br />                                                       diploid knockout mutant described previously (Gietz et al., 1995) select for <br />                                                       Ura+ transformants. <br />                                                   17. Patch representative Ura+ transformants plasmid solid SC&#8211;Ura and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                       incubate 30&#176;C overnight. <br />                                                   18. Transfer cells sporulation plates incubate room temperature 4&#8211;6 days. <br />                                                   19. Resuspend cells sporulated culture sterile ddH2O. <br />                                                   20. Spot aliquots 10x serial dilutions SC&#8211;Ura&#8211;Leu&#8211;His&#8211;Arg+G418+Can <br />                                                       plates. <br />                                                   21. Incubate plate 25&#176;C 37&#176;C 2&#8211;3 days. <br />                                                   22. Compare growth MATa G418R Ura+ cells temperatures. True Ts alleles <br />                                                       form colonies 25&#176;C 37&#176;C. <br />  <br />                               Part II. Diploid shuffle &#8211; chromosome method <br />                               A general description <br />                                               The &#8220;diploid shuffle&#65533;? chromosome method used systematically screen for <br />                                               missense mutations result temperature sensitive (Ts) alleles hundreds essential <br />                                               genes, allele directly integrated endogenous chromosomal location flanked <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               &#8220;barcodes&#65533;? corresponding yeast knockout mutant (Fig 3; (Ben-Aroya et al., <br />                                               2008), unpublished <span id='am-86' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-87'  ></span><span id='am-88' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span>). First, YFG, including promoter terminator regions, is <br />                                               mutagenized error-prone PCR (Fig. 3A). The mutagenized PCR product cloned <br />                                               SB221+Topo-TA (Fig. 3B). This plasmid contains URA3 gene flanked 5&#8242; and <br />                                               3&#8242; regions KanMX. The Topo-TA cloning site (Invitrogen) inserted the <br />                                               KanMX 5&#8242; end URA3 gene. This site allows direct cloning of PCR products, <br />                                               need modifications. The result cloning step library of <br />                                               mutagenized YFG, transformed E. coli, digested release linear <br />                                               fragments (following DNA purification) (Fig. 3B). The linear fragments directly <br />                                               transformed corresponding strain haploid-convertible heterozygous YFG/ <br />                                               yfg&#916;::kanMX diploid YKO collection selecting Ura+ transformants (Fig. 3C). The <br />                                               ~700bp KanMX5&#8242; KanMX3&#8242; fragments direct mutagenized YFG library the <br />                                               yfg&#916;::KanMX genomic locus homologous recombination, retention original bar <br />                                               codes flanking gene (Fig. 3C). Pools Ura+ cells containing mutant alleles are <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                                Page 9 <br />  <br />  <br />                                               sporulated (Fig. 3E). Spores formed spread haploid selective medium and <br />                                               incubated 25&#176;C colony formation. Only haploid MATa Ura+ spores containing the <br />                                               integrated mutant allele YFG grow medium. Colonies formed selective <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               medium replica-plated incubated 37&#176;C (Fig. 3F). Colonies growing 25&#176;C not <br />                                               37&#176;C selected potential Ts alleles, retested. In summary, final product the <br />                                               diploid shuffle approach confirmed MATa strain YKO collection genetic <br />                                               background containing URA3 marked Ts allele specific gene integrated its <br />                                               endogenous locus flanked barcodes. In addition, strain contains LEU2- <br />                                               MFA1pr-HIS3 reporter integrated CAN1 locus. <br />  <br />                                               In addition creating Ts alleles, diploid shuffle-chromosomal method used to <br />                                               transfer existing alleles knockout strain background strain backgrounds and <br />                                               vice versa. Using Topo-TA plasmid protocol described Fig. 1, extant Ts allele <br />                                               easily transferred deletion collection genetic background (referred &#8220;allele <br />                                               transfer-in&#65533;?). The result integrated allele, marked URA3, flanked appropriate <br />                                               barcodes. Moreover, using Topo-TA plasmid, PCR product (mutant allele, fusion <br />                                               protein, heterologous gene expression cassette, etc.) introduced 6000 <br />                                               genomic sites carrying KanMX replacement cassette integration site, depending the <br />                                               specific deletion mutant chosen recipient strain. By using primers external to <br />                                               primers used original mutagenesis, URA3 marked Ts-allele gene construct <br />                                               easily transferred deletion set genetic background strain interest, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               replace wild-type copy recipient strain homologous recombination (referred <br />                                               &#8220;allele transfer-out&#65533;?, Fig. 4). Thus, Ts mutation gene construct analyzed <br />                                               specific phenotypes variety genetic contexts. <br />  <br />                               Materials <br />                                               Media&#8212;Haploid selection medium SC&#8211;Ura&#8211;Leu&#8211;His&#8211;Arg+Can: dextrose, 20g/L; yeast <br />                                               nitrogen base amino acids ammonium sulfate, 1.7g/L; SC-Ura-Leu-His-Arg <br />                                               dropout mix, 2g/L; sodium glutamate, 1g/L; L-canavanine, 60mg/L; Agar, 2%. YPD: yeast <br />                                               extract, 10g/L; peptone, 20g/L; dextrose, 20g/L; Agar, 2%. YPD+G418: yeast extract, 10g/L; <br />                                               peptone, 20g/L; dextrose, 20g/L; Agar, 2%; G418, 200 mg/L. <br />  <br />                                               Diploid selection medium SC-Ura+ClonNAT: dextrose, 20 g/L; yeast nitrogen base without <br />                                               amino acids ammonium sulfate, 1.7 g/L; SC&#8211;Ura dropout mix, 2g/L; sodium glutamate, <br />                                               1g/L; ClonNAT, 200 mg/L; Agar, 2%. The sodium glutamate substituted ammonium <br />                                               sulfate nitrogen source makes ClonNAT selection reliable minimal <br />                                               medium. <br />  <br />                                               Others described Part I. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Strains&#8212;The genotype haploid-convertible heterozygous diploid strains similar to <br />                                               described Part I. Haploid Ts strains following genotype: MATa ura3&#916;0 <br />                                               leu2&#916;0 his3&#916;1 lys2&#916;0 (or LYS2) met15&#916;0 (or MET15) can1&#916;::LEU2-MFA1pr::HIS3 yfg- <br />                                               ts::URA3. <br />  <br />                                               BY4742-ade2101-NatMX MAT&#945; wild-type haploid strain NatMX gene is <br />                                               linked ade2-101 ochre mutation. This strain used confirm MATa Ura+ Ts <br />                                               candidate: MAT&#945; his3&#916;1 leu2&#916;0 lys2&#916;0 ura3&#916;0 ade2-101-NatMX <br />  <br />                                               OneShot&#174; TOP 10 Electrocomp Cells (Invitrogen Cat #C4040-52) <br />  <br />                                               Plasmids&#8212;SB221 derived M4758 (Voth et al., 2003). In M4758, BglII/KpnI <br />                                               SphI/EcoRI fragments contain TEF promoter (388bp) terminator (262bp) <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                               Page 10 <br />  <br />  <br />                                               respectively Ashbya gossypii. In SB221 fragments replaced BamHI/KpnI <br />                                               PCR fragment (731bp) containing TEF promoter plus half KanMX gene, SphI/ <br />                                               EcoRI fragment (751bp) containing half KanMX gene TEF terminator. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               In cases template PCR products KanMX gene used constructing the <br />                                               heterozygous diploid collection. Finally BamHI site SB221, adjusted Topo- <br />                                               TA site (invitrogen) create SB221-Topo-TA. <br />  <br />                               Methods <br />                                               PCR mutagenesis&#8212;This carried described Part I. However, have <br />                                               successfully used slightly different condition experiments using diploid shuffle <br />                                               chromosomal method. Here exclusively used LA Taq DNA polymerase (Cat# <br />                                               RR002B) 150 &#956;M MnCl2. One PCR 50 &#956;l, instead four, normally set each <br />                                               gene. Two primers, allow amplification entire coding region, 250&#8211;300bp the <br />                                               5&#8242;UTR, 150&#8211;200bp 3&#8242;UTR gene, used PCR. <br />  <br />                                               Cloning PCR products&#8212;The mutagenized PCR products purified cloned into <br />                                               E. coli cells electroporation. <br />                                                   1.   Purify PCR product using ChargeSwitch&#174; Clean-up Kit (Invitrogen Cat <br />                                                        #CS12000). <br />                                                   2.   Set ligation reaction includes following components: 0.5&#956;l SB221 TOPO- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                        TA, 1.0 &#956;l ligation Buffer (300mM NaCl, 15mM MgCl2), purified PCR product <br />                                                        (100ng/1kb), ddH2O total volume 6 &#956;l. <br />                                                   3.   Incubate reaction room temperature overnight. <br />                                                   4.   Store reaction 4&#176;C (if using soon) (&#8722;20&#176;C). <br />                                                   5.   Take 1.5 &#956;l ligation products transform aliquot 50 &#956;l <br />                                                        OneShot&#174; TOP 10 Electrocomp Cells (Invitrogen Cat #C4040-52) via <br />                                                        electroporation. <br />                                                   6.   Add 200 &#956;l LB transformation incubate 37&#176;C 1 hour. <br />                                                   7.   Plate 1 &#956;l aliquot LB plus ampicillin (LB-amp) plate estimate the <br />                                                        transformation efficiency. <br />                                                   8.   Transfer 100 &#956;l transformation suspension flask containing 50 ml LB- <br />                                                        amp liquid media incubate 37&#176;C ~12h amplify plasmid library. If <br />                                                        necessary, store rest transformation suspension (approximately 150 &#956;l) in <br />                                                        20% glycerol &#8722;70&#176;C later use. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   9.   Count number colonies 1 &#956;l plate multiply microliters <br />                                                        inoculated LB-amp liquid media estimate complexity library. <br />                                                        (The total number colonies ~150,000&#8211;200,000) <br />                                                   10. Purify plasmid DNA library using Plasmid Midi prep kit (Qiagen Cat #12745). <br />                                                   11. Digest 2 &#956;l 10x diluted library DNA sample NotI (10U/&#956;l, NEB, Cat#R018S) <br />                                                       10 &#956;l reaction examine digests agarose gel electrophoresis. This will <br />                                                       allow estimation overall ligation efficiency. 2.5kb+2.7kb fragments represent <br />                                                       vector (Fig. 5A), successful cloning reaction indicated band <br />                                                       shift 2.7kb fragment (in accordance insert size) (Fig. 5B). <br />  <br />                                               Yeast transformation&#8212;Purified plasmid DNA sample random mutagenesis library <br />                                               digested NotI EcoRI (NEB, Cat#R0101T, 100u/&#956;l) transformed the <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                               Page 11 <br />  <br />  <br />                                               corresponding haploid-convertible heterozygous knockout diploid mutant. The mutant alleles <br />                                               integrated endogenous locus homologous recombination. <br />                                                   1.   Digest 40 &#956;g library plasmid DNA (over night 37&#176;C) total volume 400ul <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                        using restriction enzymes mentioned above. <br />                                                   2.   Reduce reaction volume 100 &#956;l using speed-vac. <br />                                                   3.   Use digested library DNA transform corresponding heterozygous diploid <br />                                                        strain. Split digested DNA 50 &#956;l aliquots, use prepare <br />                                                        separate transformation suspension described (Pan et al., 2004). <br />                                                   4.   Combine transformation suspensions (~1284 &#956;l) plate 1 &#956;l SC&#8211; <br />                                                        Ura plate estimate transformation yield. <br />                                                   5.   Transfer rest transformation suspension 50 ml liquid SC&#8211;Ura incubate <br />                                                        30&#176;C 2 days amplify yeast library. <br />                                                   6.   Count number colonies 1 &#956;l plate multiply microliters <br />                                                        inoculated SC&#8211;Ura estimate total transformation yield. This number <br />                                                        15,000. Otherwise, repeat yeast transformation. <br />  <br />                                               Yeast sporulation&#8212;Amplified yeast library sporulated convert heterozygous <br />                                               diploid haploid spores similarly described Part I. However, slightly different haploid <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               selection medium, SC&#8211;Ura&#8211;Leu&#8211;His&#8211;Arg+Can, used determine efficiency of <br />                                               producing haploid MATa Ura+ cells sporulation culture. <br />  <br />                                               Screening Ts alleles&#8212;After plating efficiency determined, typically ~6000 <br />                                               haploid MATa Ura+ colonies screened candidate Ts alleles gene. <br />                                                   1.   Spread sporulation culture 15 plates SC&#8211;Ura&#8211;Leu&#8211;His&#8211;Arg+Can ~400 <br />                                                        colonies plate. <br />                                                   2.   Incubate plates 25&#176;C 3&#8211;4 days. <br />                                                   3.   Replica-plate colonies plate fresh plate haploid selection <br />                                                        media mark orientation mother daughter plates. <br />                                                   4.   Incubate mother plates 25&#176;C daughter plates 37&#176;C. <br />                                                   5.   Assess Ts phenotype clone comparing growth mother and <br />                                                        daughter plates day. <br />  <br />                                               Confirming Ts phenotypes&#8212;Candidate Ts mutants re-streaked haploid <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               selection media plates, incubate 25&#176;C 37&#176;C. Once Ts phenotype confirmed, <br />                                               backcross MATa Ura+ Ts candidate wild-type BY4742-ade2101-NatMX MAT&#945; strain. <br />                                               In strain NatMX gene (which provides resistance drug ClonNAT) linked to <br />                                               ade2-101 ochre mutation. Diploid cells selected streaking SC-Ura+ClonNAT <br />                                               media, sporulated, dissected. The dissected tetrads replicated YPD (25&#176;C 37&#176; <br />                                               C), SC-Ura (25&#176;C), YPD supplemented G418 (25&#176;C). This confirms that: 1) the <br />                                               temperature sensitivity segregates Mendelian manner (2:2), indicates Ts <br />                                               phenotype depends single mutated gene, 2) Ts phenotype linked URA3 gene, <br />                                               co-segregates mutated PCR product, 3) mutagenized PCR product <br />                                               integrated correct genomic locus rendering cells G418 sensitive (5&#8242;kanMX::yfg- <br />                                               ts-URA3::3&#8242;kanMX). <br />  <br />                                               Allele transfer-in&#8212;This carried similarly described subtle <br />                                               modifications. A pre-existing Ts allele (or gene construct) PCR-amplified with <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                                Page 12 <br />  <br />  <br />                                               appropriate plasmid genomic DNA template using proofreading-competent <br />                                               polymerase (e.g. LA Taq DNA polymerase) generate &#8220;A&#65533;? overhang 3&#8242; end of <br />                                               PCR product. The PCR products cloned SB221 using Topo TA cloning. The cloned <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               PCR products released URA3 marker KanMX5&#8242; and <br />                                               KanMX3&#8242; fragments vector backbone trans formed corresponding haploid <br />                                               convertible heterozygous diploid mutant. Ura+ yeast transformants sporulated a <br />                                               population plated SC&#8211;Ura&#8211;Leu&#8211;His&#8211;Arg+Can select haploid MATa Ura+ cells <br />                                               appropriate colony density. Single colonies tested phenotype interest <br />                                               (e.g. temperature sensitivity). Candidate clones backcrossed wild-type MAT&#945; <br />                                               strain analyzed tetrad dissection confirm phenotype. <br />  <br />                                               Allele transfer-out&#8212;Here transfer Ts allele generated diploid <br />                                               shuffle method ura3 strain background. Unless stated, methodologies <br />                                               mentioned above. <br />                                                   1.   Design PCR primers amplify 5&#8242;kanMX::yfg-ts-URA3::3&#8242;kanMX <br />                                                        cassette. These primers specific 5&#8242; 3&#8242; regions YFG recipient <br />                                                        strain. They external primers originally used generate the <br />                                                        Ts allele donor strain (Fig. 4A). <br />                                                   2.   Set PCR using genomic DNA Ts mutant DNA template. <br />                                                        Transform PCR product strain select Ura+ <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   3. <br />                                                        transformants. <br />                                                   4.   If recipient strain haploid, screen colonies Ts phenotype. <br />                                                   5.   Backcross wild-type strain opposite mating type. <br />                                                   6.   Verify Ts phenotype linked Ura+ tetrad analysis. <br />                                               The recipient strain diploid. In case, Ura+ transformants selected Step <br />                                               3, sporulated, characterized tetrad analysis ensure Ts Ura+ phenotypes <br />                                               co-segregate. This allow testing Ts allele viable particular genetic <br />                                               context recipient strain. It possible allele longer Ts strain <br />                                               background. If so, representative Ura+ transformants need characterized with <br />                                               diagnostic PCR sequencing ensure 5&#8242;kanMX::yfg-ts-URA3::3&#8242;kanMX cassette is <br />                                               integrated right locus. <br />  <br />  <br />                                               Perspectives <br />                                               The variations &#8220;diploid shuffle&#65533;? highly efficient methods making Ts <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               mutants. It essentially possible, screening mutants using methods <br />                                               outlined here, alleles. An adaptation methods outlined be <br />                                               required making Ts mutants large essential genes (in case, gene would <br />                                               mutagenized sections). The relative advantages methods described are <br />                                               summarized Table 4. <br />                                               We chosen forward chromosomal method generation a <br />                                               genome-wide collection Ts mutants community resource felt that <br />                                               users prefer integrated copies fluctuate lost a <br />                                               subpopulation cells division episome. <br />                                               The ability generate genome wide collection compares favorably attempts <br />                                               generate genome wide resources study essential genes, Tet-regulated <br />                                               alleles (Hartman et al., 2001; Mnaimneh et al., 2004) dAMP (Schuldiner et al., 2005). <br />                                               Both approaches, having distinct advantages, produced well- <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                                    Page 13 <br />  <br />  <br />                                               behaved alleles 30% cases. Disadvantages Ts mutants include fact <br />                                               uniform regard nonpermissive temperature leakiness, fact <br />                                               needed temperature shifts induce heat shocks effects could <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               potentially cloud phenotypic analyses. Nevertheless alleles bastion of <br />                                               traditional genetic analyses essential genes. The Ts alleles sequenced include a <br />                                               mix single amino acid substituions multi amino acid substitutions; however, have <br />                                               sequenced develop extensive statistics this. Studies collections <br />                                               Ts mutants variety genes allowed empirical determination typical <br />                                               characteristics. On basis, schemes predicting Ts mutants developed (Ye, <br />                                               P., Dymond, J., Shi, X., Lin, Y.-Y., Pan, X., Boeke J.D. Bader, J.S., submitted). This <br />                                               likely prove useful designing Ts mutants use organisms which <br />                                               extensive screening impractical. <br />  <br />  <br />                               References <br />                                               Ben-Aroya S, et al. Toward Comprehensive Temperature-Sensitive Mutant Repository Essential <br />                                                  Genes <span id='am-10' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-18'  ></span><span id='am-19' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>. Mol Cell 2008;30:248&#8211;58. [PubMed: 18439903] <br />                                               Brachmann CB, et al. Designer deletion strains derived <span id='am-1' about='obo:NCBITaxon_559292' typeof='owl:Thing obo:NCBITaxon_4932 obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Saccharomyces cerevisiae S288c" datatype="xsd:string"></span>Saccharomyces cerevisiae S288C</span>: useful <br />                                                  set strains plasmids PCR-mediated gene disruption applications. Yeast <br />                                                  1998;14:115&#8211;32. [PubMed: 9483801] <br />                                               Edgar RS, Lielausis I. Temperature-Sensitive Mutants Bacteriophage T4d: Their Isolation Genetic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Characterization. Genetics 1964;49:649&#8211;62. [PubMed: 14156925] <br />                                               Gietz RD, et al. Studies transformation intact yeast cells LiAc/SS-DNA/PEG procedure. <br />                                                  Yeast 1995;11:355&#8211;60. [PubMed: 7785336] <br />                                               Gietz RD, Sugino A. New yeast-<span id='am-58' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-66'  ></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span>Escherichia coli</span> shuttle vectors constructed vitro mutagenized <br />                                                  yeast genes lacking six-base pair restriction sites. Gene 1988;74:527&#8211;34. [PubMed: 3073106] <br />                                               Hartman, JLt, et al. Principles buffering genetic variation. Science 2001;291:1001&#8211;4. [PubMed: <br />                                                  11232561] <br />                                               Hartwell LH. Macromolecule synthesis temperature-sensitive mutants yeast. J Bacteriol <br />                                                  1967;93:1662&#8211;70. [PubMed: 5337848] <br />                                               Horowitz NH. Biochemical genetics <span id='am-34' about='obo:NCBITaxon_5140' typeof='obo:NCBITaxon_222544 owl:Thing obo:NCBITaxon_5139 obo:NCBITaxon_1 obo:NCBITaxon_5148 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_147550 obo:NCBITaxon_715989 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_2759'><span id='am-35' property="rdfs:label" content="Neurospora" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38'  ></span>Neurospora</span>. Adv Genet 1950;3:33&#8211;71. [PubMed: 15425388] <br />                                               Huang Z, et al. Plasmid-chromosome shuffling non-deletion alleles yeast. Nat Methods <br />                                                  2008;5:167&#8211;9. [PubMed: 18193055] <br />                                               Hughes TR. Yeast drug discovery. Funct Integr Genomics 2002;2:199&#8211;211. [PubMed: 12192593] <br />                                               Inoue H, et al. High efficiency transformation <span id='am-74' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-77'  ></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span>Escherichia</span> coli plasmids. Gene 1990;96:23&#8211;8. <br />                                                   [PubMed: 2265755] <br />                                               Jeong H, et al. Lethality centrality protein networks. Nature 2001;411:41&#8211;2. [PubMed: 11333967] <br />                                               Li MZ, Elledge SJ. Harnessing homologous recombination vitro generate recombinant DNA via <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   SLIC. Nat Methods 2007;4:251&#8211;6. [PubMed: 17293868] <br />                                               Leung DW, Chen E, Goeddel DV. A method random mutagenesis defined DNA segment using <br />                                                   modified polymerase chain reaction. J Cell Mol Biol 1989;1:11&#8211;15. <br />                                               Lin YY, et al. A comprehensive synthetic genetic interaction network governing yeast histone acetylation <br />                                                   deacetylation. Genes Dev 2008;22:2062&#8211;74. [PubMed: 18676811] <br />                                               Mnaimneh S, et al. Exploration essential gene functions titratable promoter alleles. Cell <br />                                                   2004;118:31&#8211;44. [PubMed: 15242642] <br />                                               Pan X, et al. A DNA integrity network yeast <span id='am-27' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-33'  ></span>Saccharomyces</span> cerevisiae. Cell 2006;124:1069&#8211;81. <br />                                                   [PubMed: 16487579] <br />                                               Pan X, et al. A robust toolkit functional profiling yeast genome. Mol Cell 2004;16:487&#8211;96. <br />                                                   [PubMed: 15525520] <br />                                               Schuldiner M, et al. Exploration function organization yeast early secretory pathway <br />                                                   epistatic miniarray profile. Cell 2005;123:507&#8211;19. [PubMed: 16269340] <br />  <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                              Page 14 <br />  <br />  <br />                                               Sikorski RS, Hieter P. A shuttle vectors yeast host strains designed efficient <br />                                                   manipulation DNA Saccharomyces cerevisiae. Genetics 1989;122:19&#8211;27. [PubMed: 2659436] <br />                                               Voth WP, et al. New &#8216;marker swap&#8217; plasmids converting selectable markers budding yeast gene <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   disruptions plasmids. Yeast 2003;20:985&#8211;93. [PubMed: 12898713] <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                           Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Fig. 1. A schematic creating conditional alleles using plasmid-chromosome shuffle <br />                                               The promoter (5&#8242;) terminator (3&#8242;) YFG separately PCR-amplified primer pairs <br />                                               PF/PR TF/TR, respectively, cloned centromeric (CEN) yeast-E. coli <br />                                               shuttling vector. Here PF stands promoter forward, PR promoter reverse, TF for <br />                                               terminator forward, TR terminator reverse. A NotI recognition site engineered <br />                                               promoter terminator. The resultant promoter/terminator clone linearized <br />                                               NotI digestion. In meanwhile, entire sequence YFG gene, including coding <br />                                               region promoter terminator sequences, mutagenized using error-prone PCR with <br />                                               primer pair PF/TR. The mutagenesis PCR products linearized promoter/terminator <br />                                               plasmid DNA mixed transformed haploid-convertible heterozygous <br />                                               diploid knockout mutant gene (MATa/&#945; YFG/yfg&#916;::kanMX4 CAN1/can1&#916;::LEU2- <br />                                               MFA1pr-HIS3). The linearized promoter/terminator clones repaired inside yeast cells <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               homologous recombination using co-transformed mutagenesis products (or <br />                                               YFG* alleles) templates. Due extensive homology ends PCR <br />                                               product vector, &gt;105 recombinant clones easily generated. This pool of <br />                                               recombinants sporulated. Haploid MATa G418R Ura+ cells selected a <br />                                               permissive condition solid SC-Ura-Leu-His-Arg+G418+Can medium single colonies, <br />                                               replica-plated fresh plates incubated permissive non- <br />                                               permissive conditions. Candidate alleles grow permissive condition the <br />                                               non-permissive condition. <br />  <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                            Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Fig. 2. Constructing promoter/terminator clone using sequence ligation independent cloning <br />                                               (SLIC) <br />                                               The promoter (5&#8242;) terminator (3&#8242;) YFG separately PCR-amplified yeast genomic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               DNA primer pairs PF/PR TF/TR, respectively. In meanwhile, centromeric yeast- <br />                                               E. coli shuttling vector linearized endonuclease digestion multicloning site <br />                                               (MCS). The PCR products linear vector plasmid mixed processed with <br />                                               T4 DNA polymerase create 5&#8242; single-stranded overhangs. The PCR primers designed in <br />                                               way PCR products vector assembled homology-mediated <br />                                               single strand annealing process. A NotI site engineered cloned promoter and <br />                                               terminator. An aliquot annealing reaction transformed E. coli competent cells. <br />                                               This modified version SLIC procedure originally described Li Elledge <br />                                               (2007). <br />  <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                             Page 17 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Fig. 3. A diagram &#8220;Diploid shuffle&#65533;? method generating temperature sensitive alleles <br />                                               (A.) Genomic DNA containing YFG 5&#8242; 3&#8242; regions used template PCR <br />                                               mutagenesis. Two black horizontal arrows represent gene-specific primers used. The <br />                                               mutagenized PCR products cloned vector SB221+Topo-TA (mutations are <br />                                               represented black stars). The Topo-TA cloning site represented black T, A <br />                                               overhang protruding PCR product represented black A. Left gray bar represents <br />                                               5&#8242; half KanMX selectable marker (Kan), right gray bar represents half <br />                                               KanMX selectable marker (MX). The NotI restriction sites indicated 2 diagonal <br />                                               black arrows. (B.) The product cloning step library mutagenized YFEG. The <br />                                               library transformed E. coli, digested NotI release linear fragments <br />                                               (following DNA purification). (C.) The linearized library transformed corresponding <br />                                               heterozygous diploid strain. Bars flank KanMX knockout represent Barcodes. <br />                                               (D.) Heterozygous diploid transformants sporulated (following meiosis), MATa Ura+ <br />                                               haploids spores selected haploid selective medium 25&#176;C. (E.) Selection temperature <br />                                               sensitive candidates following replica plating incubating 25&#176;C C 37&#176;C. Back <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               arrows identify potential Ts allele. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                                 Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Fig. 4. Allele transfer-out <br />                                               Unless stated, symbols Fig. 3 (A.) In specific example, genomic <br />                                               DNA containing Ts allele generated diploid shuffle method PCR amplified. <br />                                               Two black arrows represent primers used amplification. These primers specific to <br />                                               5&#8242; 3&#8242; regions YFG recipient strain. They external primers <br />                                               originally used generate Ts allele donor strain (represented broken arrows). <br />                                               (B.) The PCR product transformed strain interest. The specific example shows allele <br />                                               transfer haploid strain. However, desirable transfer allele diploid strain <br />                                               first, followed sporulation tetrad dissection, Ts allele transferred inviable <br />                                               specific genetic context interest. The Ts allele replaces wild-type YFG by <br />                                               homologous recombination (represented dashed lines) rise Ura+ transformants <br />                                               (indicated Ts phenotype). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                         Page 19 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Fig. 5. Testing Topo-TA cloning efficiency SB221 <br />                                               Unless stated, symbols Fig. 3 (A.) The SB221 vector of <br />                                               URA3 gene flanked fragment containing TEF promoter plus half KanMX <br />                                               gene (Kan), fragment containing half KanMX gene TEF <br />                                               terminator (Mx). This 2.7kb fragment excised 2.5kb backbone restriction <br />                                               digestion NotI. (B.) Restriction digest NotI following successful Topo-TA cloning <br />                                               reaction indicated 2.5kb backbone band shift 2.7kb fragment, in <br />                                               accordance insert size. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                                     Page 20 <br />  <br />  <br />  <br />  <br />                                                                                               Table 1 <br />                            PCR amplification promoters terminators. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Component                                       Volume/reaction <br />                             10 &#215; Platinum Pfx DNA Polymerase buffer               2.5 &#956;l <br />  <br />                             dNTP (2.5 mM each)                                     2 &#956;l <br />  <br />                             Primer mix (5 &#956;M each)                                 2 &#956;l <br />  <br />                             Yeast genomic DNA (~200 ng/&#956;l)                         1 &#956;l <br />  <br />                             ddH2O                                                17.25 &#956;l <br />  <br />                             Platinum Pfx DNA Polymerase (2.5 units/&#956;l)           0.25 &#956;l <br />  <br />                             Total                                                 25 &#956;l <br />  <br />                            PCR conditions: 94&#176;C 4 min; 30 &#215; (94&#176;C 30 sec, 55&#176;C 30 sec, 72&#176;C 45 sec); 72&#176;C 7 min; hold 4&#176;C. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                      Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                                         Page 21 <br />  <br />  <br />  <br />  <br />                                                                                                    Table 2 <br />                            A T4 DNA polymerase resection reaction. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Component                              Volume <br />                             10 &#215; NEB BamHI Buffer                    2 &#956;l <br />  <br />                             10 &#215; BSA                                 2 &#956;l <br />  <br />                             ddH2O                                   3.7 &#956;l <br />  <br />                             T4 DNA polymerase (3 units/&#956;l)          0.3 &#956;l <br />  <br />                             Vector (100 ng/&#956;l)                       2 &#956;l <br />  <br />                             PCR product (50 ng/&#956;l)                  10 &#956;l <br />  <br />                             Total                                   20 &#956;l <br />  <br />                            Upon set up, reaction incubated 25&#176;C 30 minutes. After that, sit ice E. coli transformation. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                        Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                                                      Page 22 <br />  <br />  <br />  <br />  <br />                                                                                                 Table 3 <br />                            Error-prone PCR conditions. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Component                                 Volume/reaction <br />  <br />                             10 &#215; Ex Taq buffer                               5 &#956;l <br />  <br />                             dNTP (2.5 mM each)                               4 &#956;l <br />  <br />                             Primer mixtur <br /> </body></html>