<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Feb. i991, p. 1376-1381                                                                      Vol. 173, No. 4 <br /> 0021-9193/91/041376-06$02.00/0 <br /> Copyright &#169; 1991, American Society Microbiology <br />  <br />  <br />          RNase T Affects <span id='am-29' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32'  ></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span>Escherichia coli</span> Growth Recovery from <br />                               Metabolic Stress <br />                          KRISTINE PALAS PADMANABHA AND MURRAY P. DEUTSCHER* <br />             Department Biochemistry, University Connecticut Health Center, Farmington, Connecticut 06432 <br />                                         Received 21 September 1990/Accepted 5 December 1990 <br />  <br />              To determine essentialty role RNase T RNA metabolism, constructed <span id='am-45' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-46' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-49'  ></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia</span> col <br />            chromosomal rnt.:kan mutation using gene replacement disrupted, plasmid-borne copy rnt <br />            gene. Cell extracts strain mutations RNases BN, D, H, I interuppted rnt gene were <br />            devoid RNase T activity, retained low level (&lt;10%) exonucleolytic activity on <br />            tRNA-C-C-[]4CJA <span id='am-67' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-71'  ></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span>unidentified</span> RNases. A mutant lacking tRNA nucleotidyltanserase in <br />            addition aforementioned RNases accumulated 5% defective tRNA did RNase <br />            T-positive cells, indicating RNase responsible essentialiy tRNA end turnover E. coi. tRNA <br />            rnt::kan strains displayed slightly reduced capacity aminoacylated, raising possibility that <br />            RNase T participate tRNA processing. Strains devoid RNase T displayed slower growth rates <br />            did wild type, phenotype accentuated absence exoribonludeases. A <br />            strain lacking RNase T RNases displayed a, normal response UV irradiation growth of <br />            bacteriophages severely affected ability recover starvation regimen. The <span id='am-57' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-58'  ></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> <br />            demonstrate absence RNase T affects normal functioning E. coil, compensated <br />            degree presence RNases. Possible roles RNase T RNA metabolism are <br />            discussed. <br />  <br />    RNase T seven Escherichia coli exoribonu-                              MATERIALS AND METHODS <br /> cleases identified date (6, 7). The enzyme,           Bacterial strains <span id='am-62' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-63' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span>. Wild-type E. coli CA265 <br /> purified homogeneity, exists a2 dimer 50 kDa (11).       RNase-deficient derivatives CAN (BN deficient), <br /> In vitro RNase T specifically catalyzes removal           CAN20-12E (BN, II, D, I deficient), CAN20-12E/ <br /> terminal AMP residue uncharged tRNA releases                18-11 (BN, II, D, I, T deficient) described <br /> tRNA-C-C product (12). The gene encoding RNase T                previously (13). Strain MC1061 [F- araD139 A(<span id='am-16' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-20'  ></span>ara</span>-leu)7697 <br /> (rnt) recently cloned localized 36 min           Alac galU galK strA] used construction rnt <br /> the E. coli genetic map (3).                                         insertion mutant. Strain UT481 [A(lac-pro) hsdS (r- m-) <br />    Previously, rnt mutant strain retained        laCdq] routinely used transformations. pRNT14 con- <br /> 30% RNase T activity cell extracts isolated (13). In       sists 1. 1skb DNA fragment containing rnt gene <br /> studies mutant strain, deficient           cloned BamHI HindIII sites pBR328 (3). <br /> tRNA nucleotidyltransferase (cca), shown RNase           pUC4K obtained Pharmacia. pMAK700 (16) a <br /> T involved end turnover tRNA. cca mutant                gift S. Kushner (University Georgia). pMAK700, <br /> strains accumulate defective tRNA and, result,             thermosensitive DNA replication, contains <br /> dramatically reduced viability (21). However, introduction           unique HindIII, BamHI, SphI sites carries gene <br /> of rnt mutation cca mutant strains prevents             encoding chloramphenicol acetyltransferase (16). <br /> formation defective tRNA relieves slow-growth                  Materials. The RNase T substrate   tRNA-C-C-[&quot;4C]A was <br /> phenotype.                                                            prepared [14C]AMP incorporation tRNA-C-C E. <br />    Although above-mentioned studies demonstrated             coli tRNA nucleotidyltransferase (10). tRNA isolated by <br /> rnt involved end turnover, did answer the <br />                                                                       phenol extraction isopropanol fractionation described <br />                                                                       previously (9). [14C]ATP obtained ICN. Ultrogel <br /> question gene essential E. coli,         AcA54 purchased LKB. Restriction endonu- <br /> residual RNase T activity present mutant. In         cleases, mung bean nuclease, T4 DNA ligase were <br /> addition, rnt mutation isolated background         purchased New England BioLabs used accor- <br /> that lacked RNases (13), possible         dance directions manufacturer. All salts used <br /> study effect mutation isolation. For reason,       reagent grade. <br /> we constructed rnt: :kan insertion mutant using             Growth conditions. Cells grown YT media (19). <br /> gene replacement method (16) moved allele           For solid medium, 2% agar (Difco) added. YT broth and <br /> different genetic backgrounds study physiological              YT agar supplemented ampicillin (50 ,ug/ml), <br /> properties strains lacking RNase T. Our results indicate           kanamycin (50 p.g/ml), chloramphenicol (20 ,ug/ml) as <br /> that RNase T essential normal growth E. coli        needed. <br /> for recovery starvation.                                        DNA techniques. E. coli chromosomal DNA isolated <br />                                                                       described Silhavy et al. (20). Large-scale isolation of <br />                                                                       plasmid DNA method Birnboim Doly <br />                                                                       (1). Rapid plasmid screening modification of <br />                                                                       procedure Ish-Horowitz Burke (17) small <br />    *   Corresponding author.                                             volumes cell cultures. <br />                                                                   1376 <br />  VOL. 173, 1991                                                                 PHYSIOLOGICAL ROLE OF RNase T                  1377 <br />  <br />    DNA probes Southern blots labeled                noacylation assays performed described previously <br /> [oa-32P]dATP (New England Nuclear) random primer           (4) mixture amino acids E. coli aminoacyl- <br /> method Feinberg Vogelstein (15). Southern blotting         tRNA synthetases. E. coli aminoacyl-tRNA synthetases <br /> was performed described Maniatis et al. (18).               devoid tRNA nucleotidyltransferase prepared as <br />    Plasmid constructions. pRNT14, contains 1.1-kb         described Deutscher Evans (8). Protein concentra- <br /> insert carrying rnt gene (3), used construct        tions determined colorimetrically method of <br /> rnt::kan insertion mutation. The kanamycin cassette           Bradford (2). <br /> obtained digestion pUC4K (Pharmacia) Sall. The <br /> single-stranded DNA ends kanamycin DNA                                         RESULTS <br /> fragment removed mung bean nuclease. The <br /> fragment ligated pRNT14, linearized            Subcloning overexpression rnt. The rnt gene was <br /> by partial digestion HincII. pRNT14 contains             initially cloned screening cosmid library containing <br /> HinclI site rnt additional sites vector.    large fragments E. coli genome plasmid pHC79 (3). <br /> A plasmid clone, pRNT17, contained single insertion        The rnt gene plasmid resulted twofold <br /> of kan rnt gene, confirmed restriction endonu-       elevation RNase T activity (3). The rnt gene subse- <br /> clease cleavage plasmid, obtained             quently subcloned 2.5-kb fragment pHC79, pBR328, <br /> ligation.                                                         pUC18 assess level overexpression to <br />    The rnt::kan gene inserted plasmid pMAK700            determine effect elevated RNase T activity cell <br /> (16). A 2.4-kb BamHI-HindIII DNA fragment pRNT17             growth. Although copy numbers plasmids are <br /> containing rnt::kan cloned corresponding re-         thought different, levels RNase T activity cells <br /> striction sites pMAK700. The resulting plasmid, pRNT18,        harboring different plasmid subclones nearly the <br /> was used construct rnt::kan chromosomal mutation.          same. RNase T activity elevated 12.3-, 13.9-, and <br />    Construction rnt::kan chromosomal mutation. The         15.0-fold CA265 containing rnt plasmids pHC79, <br /> method used described Hamilton et al.       pUC18, pBR328, respectively. The maximum elevation <br /> (16). pRNT18 transformed strain MC1061. The              RNase T activity cells apparently tolerate <br /> resulting strain grown 30&#176;C mid-log phase,      15-fold. The fact elevated activity specif- <br /> dilutions culture plated YT agar plates          ically removed terminal AMP residue tRNAC-C- <br /> supplemented kanamycin chloramphenicol. The              [&apos;4C]-A confirmed presence rnt gene the <br /> plates incubated 44&#176;C select cells       plasmid subclones. <br /> integrated plasmid chromosome. Several cointe-          Strain CA265 harboring rnt cloned pHC79 displayed a <br /> grates selected grown cycles 300C. At         growth rate identical untransformed strain. In <br /> <span id='am-21' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-24'  ></span><span id='am-25' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> temperature, cointegrates unstable tended       contrast, considerable lag (several hours) before <br /> to recombine free plasmid, carrying        onset exponential growth CA265 contained rnt <br /> wild-type rnt::kan allele, generated      cloned high-copy-number plasmid pUC18 (data not <br /> cell (16). Strains containing plasmid resolution products         shown). Although growth curves CA265 strains <span id='am-52' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-53' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56'  ></span>car</span>- <br /> were selected screening colonies 300C culture         rying pUC18 containing rnt pHC79 containing rnt were <br /> for temperature-sensitive growth chloramphenicol plates.       dramatically different, levels RNase T activity in <br /> Plasmid DNAs 13 temperature-sensitive colonies          log-phase cultures strains (see <br /> prepared analyzed agarose gel (data shown).         above). One explanation findings high <br /> DNAs strains size pRNT18.             level RNase T activity expected be <br /> The remaining strains contained plasmid DNAs           present initially strains carrying pUC18 containing rnt <br /> smaller pRNT18, indicating wild-type rnt gene       deleterious normal growth. If so, be <br /> had replaced rnt::kan construct plasmid. Strains       strong selection cells reduced their <br /> containing smaller rnt plasmids kanamycin resis-         RNase T activity consequently grow faster. Slow growth <br /> tant chloramphenicol sensitive 440C, indicating       strains elevated RNase T activity a <br /> kan integrated chromosome. The plasmid          high level tRNA end turnover cellular levels <br /> was cured strains growth liquid cultures        tRNA nucleotidyltransferase sufficient repair all <br /> 44&#176;C absence chloramphenicol.                           defective tRNA molecules. In support explana- <br />    Analysis RNase T activity cell extracts. Cell extracts   tion, attempts transformation CCA-negative strain <br /> were prepared 30-ml cultures grown optical             rnt gene subcloned pHC79 unsuccessful, <br /> density 550 nm (OD550) 1 37&#176;C. Cells harvested      CCA-positive strains transformed normally <br /> by centrifugation, resuspended 1/5 volume sonication      (data shown). <br /> buffer (20 mM <span id='am-6' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Glycine" datatype="xsd:string"></span>glycine</span>-NaOH [pH 8.9], 0.1 mM EDTA, 1                 Southern blots wild-type rnt::kan strains. Southern <br /> mM dithiothreitol), sonicated ice 60 s. After          blots chromosomal DNAs MC1061 MC1061 <br /> centrifugation remove cell debris, 25-,u aliquots         containing rnt::kan (MC1061-rnt::kan) prepared and <br /> assayed RNase T activity 20 min reaction mixture     probed labeled 1.1-kb HindIII-BamHI DNA frag- <br /> (0.2 ml) containing 20 mM glycine-NaOH (pH 8.9), 5 mM             ment containing complete rnt gene (Fig. 1B). For blot <br /> MgCl2, 30 ,ug tRNA-C-C-[&apos;4C]A. Reactions              MC1061-rnt: :kan DNA, bands seen the <br /> stopped addition 0.3 ml 0.5% yeast tRNA 0.3      autoradiograph PstI PstI-BamHI <br /> ml 20% trichloracetic acid. The precipitate removed        digests; however, single band obtained the <br /> by centrifugation, radioactivity one-half       corresponding digests MC1061 DNA (Fig. 1A). The PstI <br /> supernatant fluid determined.                                 digest MC1061 DNA single band 7 kb (Fig. 1A, <br />   Other methods. The tRNA defective 3&apos;             lane 1), corresponding digest MC1061-rnt::kan <br /> termini various strains determined isolating tRNA       DNA bands 4.4 2.6 kb (Fig. 1A, lane 3). One band <br /> and measuring level [&apos;4C]AMP incorporation          3.2 kb obtained BamHI-PstI digest MC1061 <br /> presence E. coli tRNA nucleoltidyltransferase (9). Ami-        DNA (Fig. <span id='am-1' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-2' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>IA</span>, lane 2). In contrast, double BamHI-PstI <br />  1378     PADMANABHA AND DEUTSCHER <br />  <br />            A      1   2 3 <br />  <br />  <br />                       ; <br />                           _ ~ 9.4 <br />                              4 <br />  <br />                                    23.1 <br />                                     6.5 <br />                                  - 4.3 <br />                                    2.3 <br />                                    2.0 <br />  <br />                                  - .56 <br />                                              B <br />  <br />  <br />  <br />  <br />                                           length (kb) <br />                                                       (n <br />                                                      ra <br />  <br />  <br />                                                            Z- <br />                                                            L <br />                                                                   0L <br />  <br />  <br />  <br />  <br />                                                                  2.1 <br />                                                                             2.1 <br />  <br />  <br />  <br />  <br />                                                                             l <br />                                                                                 C <br />                                                                                 . <br />                                                                                 = m <br />                                                                                     tfi nL <br />  <br />                                                                                     0.5 <br />                                                                                              -ac <br />  <br />  <br />  <br />  <br />                                                                                               - <br />                                                                                              1.3 <br />                                                                                                    _ <br />                                                                                                         c <br />  <br />  <br />                                                                                                        1-L <br />                                                                                                        11 <br />  <br />  <br />  <br />  <br />                                                                                                              I <br />                                                                                                                  I <br />  <br />  <br />  <br />  <br />                                                                                                               :ECZ <br />  <br />  <br />                                                                                                              0.6 <br />                                                                                                                    m <br />  <br />  <br />  <br />  <br />                                                                                                                    IF <br />                                                                                                                    CC <br />  <br />                                                                                                                      LX. <br />                                                                                                                       `7 `7 <br />                                                                                                                                3.8 <br />  <br />  <br />  <br />  <br />                                                                                                                               3.8 <br />                                                                                                                                      I_e <br />  <br />  <br />  <br />  <br />                                                                                                                                       r, <br />                                                                                                                                            M(; <br />  <br />  <br />  <br />  <br />                                                                                                                                            Me <br />                                                                                                                                            rr! <br />                                                                                                                                                  i <br />                                                                                                                                                      J. BACTERIOL. <br />  <br />  <br />  <br />  <br />                                                                                                                                                      jD6 <br />  <br />  <br />  <br />  <br />                                                                                                                                                      06 <br />  <br />  <br />                                                                                                                                                      kar- <br />                                                                                                                                                            1 <br />  <br />  <br />  <br />  <br />                                                                E7Z       r nt   probe <br />                                                                L&apos;,&apos;- _chromosomal DNA outside probe regior <br />                                                                     _M kanamycin cassette <br />   FIG. 1. Southern blot chromosomal DNA MC1061-rnt::kan MC1061. (A) Lanes: 1 2, MC1061 chromosomal DNAs <br /> digested PstI PstI-BamHI, respectively; 3 4, MC1061-rnt::kan chromosomal DNAs digested PstI PstI-BamHl, <br /> respectively. The markers right lane 4 indicate migration positions lambda HindlIl size markers (in thousands). (B) <br /> Location kanamycin cassette rnt gene positions restriction sites. Note kanamycin cassette contains the <br /> two flanking PstI sites; PstI sites rnt. <br />  <br />  <br /> digest MC1061-rnt::kan DNA yielded bands 2.6                                    rnt-encoded activity. Examination effect kan <br /> and 0.6 kb (Fig. 1A, lane 4). These data agree previ-                                insertion chromosomal copy rnt gene indicated <br /> ously mapped restriction sites surrounding rnt                                 RNase T activity strain CA265 reduced about <br /> gene (3). These results, taken genetic                                  twofold comparison strain the <br /> evidence described Materials Methods, indicated                               uninterrupted gene. Thus, wild-type strain, other <br /> the kan cassette located middle rnt gene                             enzyme enzymes present ability <br /> MC1061-rnt::kan (Fig. 1B).                                                                remove terminal AMP residue tRNA-C-C-A. On <br />    RNase T activity strains carrying rnt::kcw insertion.                           hand, strain CAN20-12E, RNase T activity was <br /> The rnt::kan construct MC1061-rnt::kan trans-                                    reduced 10-fold interruption rnt gene. <br /> duced wild-type strain, CA265, strain                                   Thus, strain, remaining 3&apos; exonucleolytic <br /> CAN20-12E, deficient multiple RNase activities                                activity intact tRNA observed assay condi- <br /> (BN, II, D, I), strain CAN20-12E/18-11,                                tions encoded rnt. <br /> is deficient multiple RNase activities (BN, II, D, I,                                Previously, rnt mutant RNase-deficient strain <br /> and T). Cell extracts strains controls                             CAN20-12E isolated nitrosoguanidine mutagenesis. <br /> were prepared assayed RNase T activity (Table 1).                                 The resulting strain, CAN20-12E/18-11, 30% of <br /> The introduction intact rnt gene strain CAN20-                                 RNase T activity parent (13). The activity in <br /> 12E/18-11 plasmid pRNT14 led 15-fold elevation                               CAN20-12E containing rnt::kan (CAN20-12E-rnt::kan) was <br /> RNase T activity. When rnt gene interrupted                                threefold lower CAN20-12E/18-11, con- <br /> plasmid, pRNT17, RNase T activity overpro-                                   firming rnt mutation nitrosoguanidine <br /> duced. The insertion, therefore, completely abolished                                 mutagenesis leaky does abolish rnt-encoded <br />                                                                                           activity (Table 1). <br />                                                                                              Analysis RNase activity CAN20-12E CAN20- <br />        TABLE 1. Analysis RNase T activity rnt+ and <br />                                                                                           12E-rnt.*:kan strains gel filtration chromatography. To <br />                       rnt::kan strainsa                                                   analyze residual activity putative rmnt null mutant, <br />                                                                                           prepared cell extracts strains CAN20-12E and <br />                                                            [14C]AMP released              CAN20-12E-rnt::kan resolved AcA54 gel <br />         Strain                   Phenotype                    (nmol/mg                 filtration column. An assay fractions tRNA-C-C- <br />                                                                 protein)                  [&apos;4C1-A (Fig. 2) revealed single peak CAN20-12E <br /> CAN20-12E/18-11           Deficient RNases                     1.80                    extract corresponding size 50 kDa, known <br />                             BN, II, D, I, T                                            size RNase T. In contrast, CAN20-12E-rnt: :kan <br /> CAN20-12E/18-11 +         Deficient RNases                   27.0                      extract showed small peaks activity, 66 <br />   pRNT14                    BN, II, D, I, T                                           kDa 35 kDa, confirming RNase T is <br /> CAN20-12E/18-11 +         Deficient RNases                     1.90                    absent. The identities residual activities not <br />   pRNT17                    BN, II, D, I, T                                           known. <br /> CA265                     Wild type                               8.30 <br /> CA265-rnt::kan            Deficient RNase T                    4.90                       End turnover tRNA rwt::kan strins. tRNA was <br /> CAN20-12E                 Deficient RNases                     5.80                    isolated rnt::kan control strains analyzed <br />                             BN, II, D, I                                              acceptance AMP tRNA nucleotidyltrans- <br /> CAN20-12E-rnt::kan        Deficient RNases                     0.60                     ferase assess end turnover described previously (9). As <br />                              BN, II, D, I, T                                           reported previously (13), little terminally defective <br />   Cell extracts prepared assayed described Materials                  tRNA strains CAN CAN20-12E/18-11 (0.10 0.09 <br /> Methods. pRNT14 carries wild-type rnt gene, pRNT17 carries                     nmol (14C]AMP incorporated mg tRNA, respec- <br /> interrupted rnt gene.                                                                      tively) presence wild-type cca gene. As <br />  VOL. 173, 1991                                                                             PHYSIOLOGICAL ROLE OF RNase T                          1379 <br />  <br />                                                                           TABLE 2. Aminoacylation tRNA isolated wild-type and <br />                                                                                           RNase T-deficient strainsa <br />                                                                                   Strain                    Phenotype                  Relative <br />                                                                                                                                   aminoacylation (%) <br />                                                                          CA265                      Wild type                            100 <br />        EI <br />                                                                          CA265-rnt::kan             Deficient RNase T              91 &#177; 3.0 <br />        1f 600     - <br />  <br />                                                                          CAN20-12E                  Deficient RNases               95 &#177; 2.5 <br />                                                                                                       BN, II, D, I <br />            400    -                           CAN20-12E                  CAN20-12E-rnt::kan         Deficient RNases               85 &#177; 0.7 <br />                                                 ~~~~rnt::kan                                          BN, II, D, I, T <br />            200-                                                            The 14C-amino acid incorporated 100 ,ug tRNA was <br />                                                                          determined described Materials Methods. The percentages represent <br />               0 <br />                                                                          results experiments. <br />                   25   30      35     40     45      50     55 <br />                               Fraction <br />    FIG. 2. Gel filtration chromotagraphy cell extracts           tRNA precursors. Further work necessary evalu- <br /> CAN20-12E CAN20-12E-rnt::kan strains. The cells grown           ate possible role RNase T. <br /> to OD550 1 YT media 37&#176;C. Cells (500 ml) harvested,        Growth rates rnt::kan strains. The growth rates of <br /> washed sonication buffer, resuspended 3 ml buffer,     wild-type rnt::kan strains measured rich media <br /> and sonicated ice 3 5 min (in 1-min bursts) disruption   37&#176;C (Table 3). The presence rnt: :kan mutation had <br /> appeared complete. Cell debris removed low-speed centrif-         small, reproducible, effect growth rate the <br /> ugation 12,000 x g. The extract centrifuged high speed <br /> (100,000 x g) 3 h. About 35 mg (3 ml) extract layered     wild-type strain CA265 genetic background (30 versus 25 <br /> on AcA54 column (84 1.5 cm) eluted buffer containing        min). A pronounced effect growth observed in <br /> 20 mM Tris chloride (pH 7.4), 0.5 mM EDTA-0.1 mM phenylmeth-             RNase-deficient background strain CAN20-12E: this <br /> ylsulfonyl fluoride, 0.1 mM dithiothreitol, 10% glycerol, 0.2 M      mutant doubling time 45 min, compared 30 <br /> NaCl. Fractions 1.5 ml collected, 25-,ul aliquots       min parental strain. These results indicate RNase <br /> assayed RNase T activity described Materials Meth-         T essential normal growth E. coli that, when <br /> ods. The arrows graph indicate positions    potential compensatory RNases absent, effect on <br /> molecular mass standards (from left right): catalase (which, 232   growth quite pronounced. <br /> kDa, eluted void volume), bovine serum albumin (67 kDa),             Recovery rnt:.:kan strains starvation. CA265, <br /> ovalbumin (43 kDa), trypsin inhibitor (21 kDa). <br />                                                                          CA265-rnt: :kan, CAN20-12E, CAN20-12E-rnt: :kan <br />                                                                          grown mid-log phase YT media then <br />                                                                          starved Tris buffer (pH 7.4) 16 h. After starvation, the <br /> before, introduction cca mutation strains CAN              cells returned YT media, rate recovery <br /> CAN20-12E/18-11 led large increase terminally defec-             measured. Strains CA265, CA265-rnt::kan, CAN20- <br /> tive tRNA levels inability repair                  12E started recover rich media 2-h lag <br /> defective molecules. Defective tRNA levels                    (Fig. 3). In contrast, growth experiments was <br /> fivefold lower CAN20-12E/18-11 derivative             arrested CAN20-12E-rnt::kan culture 4 h <br /> CAN derivative, contains RNase T (0.70 3.30                    prior resumption growth. When starvation <br /> nmol [14C]AMP incorporated mg tRNA, respec-                    period lengthened 20 h, recovery of <br /> tively), confirming RNase T participates end turnover            CAN20-12E-rnt::kan observed, 6 h rich <br /> (13). Interruption rnt gene led additional two-          media (data shown). In addition, starved CAN <br /> threefold-lower defective tRNA (0.30 nmol                   20-12E-rnt::kan cells required 2 days colonies were <br /> [14C]AMP incorporated mg tRNA), compared                  visible YT plates, strains showed <br /> that strain residual RNase T activity. When              colonies 1 day incubation (data shown). These <br /> rnt-encoded activity abolished, small (about                results suggest RNase T, combination or <br /> 6%) defective tRNA present. Thus, data                RNases, required normal recovery cell <br /> indicate &gt;90% end turnover tRNA occurs               growth starvation. <br /> in E. coli RNase T small                    Additional physiological properties RNase T-deficient <br /> defective tRNA arises pathway.                                mutants. CA265, CA265-rnt::kan, CAN20-12E, CAN <br />    Aminoacylation tRNA wild-type rnt::kan <br /> strains. To determine RNase T plays additional <br /> role tRNA metabolism, isolated tRNA wild-type <br /> and rnt::kan strains tested ability accept                  TABLE 3. Doubling times wild-type RNase <br /> amino acid mixture amino acids E. coli                                          T-deficient strainsa <br /> aminoacyl-tRNA synthetases (Table 2). tRNA                             Strain                         Phenotype                  Doubling <br /> rnt::kan strains, CA265 containing rnt::kan (CA265-rnt::kan) <br /> and CAN20-12E-rnt::kan, slightly reduced capacity               CA265                           Wild type                            25 <br /> accept amino acid comparison tRNA                    CA265-rnt::kan                  Deficient RNase T                 30 <br /> wild-type CAN20-12E strains. Since strains                CAN20-12E                       Deficient RNases                  30 <br /> were wild type cca, decreased acceptor activity                                                    BN, II, D, I <br /> should incomplete 3&apos; termini (see                   CAN20-12E-rnt::kan              Deficient RNases                  45 <br /> above). Rather, lower levels aminoacylation                                                BN, II, D, I, T <br /> been unprocessed 3&apos; tRNA ends. If so, RNase T                   Overnight cultures diluted 100-fold YT media grown 37&apos;C <br /> have limited, detectable, role trimming 3&apos; ends          shaking. The OD550 culture measured 30-min intervals. <br />  1380     PADMANABHA AND DEUTSCHER                                                                                             J. BACTERIOL. <br />  <br />                                                                         end-turnover process known affect small fraction <br />                                                                         E. coli tRNAs (10), process appears futile <br />                                                                         cycle. Thus, difficult imagine absence this <br />                                                                         process affect growth. Another function RNase T <br />                                                                         may, therefore, likely. <br /> E   1074                                           CAN20-12E              One possibility function 3&apos; processing of <br />                                                                         tRNA precursor(s). This <span id='am-11' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14'  ></span><span id='am-15' property="rdfs:label" content="Idea" datatype="xsd:string"></span>idea</span> consistent the <br />                                                                         finding (Table 2) level functional tRNA slightly <br />                                                                         decreased (about 10%o) RNase T-deficient mutants under <br /> n                                            CAN20-1 2E-rnt::kan        conditions CCA sequence intact <br />                                                                         (although 10% decrease functional tRNA appear <br />                                                                         minimal, represent major defect a <br />                                                                         tRNA <span id='am-26' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> affected). This suggestion also <br />                                                                         consistent ability RNase T act synthetic <br />                                                                         tRNA precurs <br /> </body></html>