<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Radiat Res. Author manuscript; available PMC 2011 November 1. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Radiat Res. 2010 November ; 174(5): 558&#8211;565. doi:10.1667/RR2184.1. <br />  <br />  <br />  <br />                            Deficiency Bre1 Impairs Homologous Recombination Repair <br />                            Cell Cycle Checkpoint Response Radiation Damage in <br />                            Mammalian Cells <br />  <br />                            Sophia B. Chernikova, Jennifer A. Dorth, Olga V. Razorenova, John C. Game, J. Martin <br />                            Brown1 <br />                            Department Radiation Oncology, Stanford University School Medicine, Stanford, <span id='am-7' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-8' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11'  ></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span>California</span> <br />                            94305-5152 <br />  <br />                            Abstract <br />                                 The pathway involving Bre1-dependent monoubiquitination histone H2B lysine 123, leads <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 Dot1-dependent methylation histone H3 lysine 79 (H3K79me2), implicated survival <br />                                 exposure ionizing radiation <span id='am-13' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>. We depletion of <br />                                 mammalian homologs Bre1 compromises response ionizing radiation, leading increased <br />                                 radiosensitivity G2/M checkpoint defect. The deficiency Bre1a/b function also <br />                                 associated increased sensitivity crosslinking drugs defective formation Rad51 foci in <br />                                 mouse cells, suggesting defect homologous recombinational repair analogous seen in <br />                                 <span id='am-30' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span>. In budding yeast, H3K79me2 important recruitment checkpoint <br />                                 signaling protein Rad9 sites double-strand breaks (DSBs). However, mammalian cells, 53BP1 <br />                                 (the Rad9 ortholog) addition H3K79me2 recognizes different residue, H4K20me2, some <br />                                 studies argue H4K20me2 H3K79me2 preferred target 53BP1. We <br />                                 depletion Bre1b specifically reduced dimethylation H3K79 affecting <br />                                 dimethylation H4K20. Thus <span id='am-73' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-74' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-76'  ></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> suggest observed defects radiation response <br />                                 Bre1a/b-deficient cells associated reduced H3K79me2 H4K20me2. <br />  <br />  <br />                            INTRODUCTION <br />                                             It increasingly clear recent years types post-translational histone <br />                                             modification including phosphorylation, acetylation, ubiquitination methylation play <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             significant roles DNA repair mechanisms. A particular pathway studied in <br />                                             Saccharomyces involves cross-histone modifications ubiquitination histone H2B <br />                                             residue K123 affects methylation K79 residue histone H3 (1,2). <span id='am-59' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62'  ></span><span id='am-63' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> pathway is <br />                                             widely conserved eukaryotes shown influence radiation sensitivity and <br />                                             radiation-induced cell cycle checkpoints yeast (3&#8211;8). Specifically, H2BK123 is <br />                                             ubiquitinated Saccharomyces Rad6 ubiquitin conjugase conjunction Bre1 <br />                                             ubiquitin ligase (9&#8211;11), H3K79 residue uniquely methylated Dot1 <br />                                             methyltransferase (12&#8211;14). In absence H2BK123 ubiquitination, Dot1 remains able to <br />                                             monomethylate H3K79 (15), di- trimethylation residue occurs wild- <br />                                             type Saccharomyces largely abrogated. In addition, methylation H3K4, mediated <br />                                             wild-type Saccharomyces SET1 complex (COMPASS), abrogated bre1 &#916; strains <br />                                             mutants lack H2BK123 ubiquitination (1,16,17). <br />  <br />  <br />                            &#169; 2010 Radiation Research Society. All rights reproduction form reserved. <br />                            1 <br />                              Address correspondence: Department Radiation Oncology, Stanford University School Medicine, CCSR South, Room 1255, <br />                            Stanford, CA 94305-5152; mbrown@stanford.edu. <br />                             Chernikova et al.                                                                                               Page 2 <br />  <br />  <br />                                                Deleting DOT1 gene Saccharomyces confers multiple phenotypes, including reduced <br />                                                telomeric silencing, increased radiation sensitivity, abrogation radiation-induced <br />                                                G1 (but G2/M) checkpoint (4&#8211;6,14,18,19). A subtle defect G2/M detected dot1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                mutants Grenon et al. (7). Deleting BRE1 gene confers radiation sensitivity equivalent <br />                                                dot1null strains (6), indicating di- and/or trimethylation H3K79 rather <br />                                                monomethylation important wild-type radiation resistance. Epistasis analysis <br />                                                indicates recombinational repair affected absence adequate H3K79 methylation, <br />                                                double mutants involving dot1&#916; bre1&#916; recombinational repair mutant <br />                                                rad51&#916; X-ray sensitivity equivalent rad51&#916; strains (6). In contrast, <br />                                                deleting DOT1 BRE1 adds substantial sensitivity mutants rad18&#916; rad5&#916; that <br />                                                involved postreplication repair (6). Grenon et al. (7) reported Saccharomyces <br />                                                Rad9 protein binds tudor domain methylated H3K79 irradiation the <br />                                                cell&#8217;s response radiation. Abrogation binding responsible radiation <br />                                                sensitivity seen mutants dot1&#916; rad9tudor (7). Further evidence the <br />                                                importance H3K79me2 comes recent study showing Dot1 Rad9 <br />                                                DNA damage checkpoint adaptor required efficient sister chromatid recombination in <br />                                                budding yeast (20). However, <span id='am-37' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41'  ></span>Schizosaccharomyces</span> pombe, H3K79 methylated, and <br />                                                tudor domain Crb2 protein, closely related Saccharomyces Rad9, binds <br />                                                irradiation dimethylated H4K20 residues (21,22), implying significant pathway <br />                                                divergence. H4K20 methylated S. pombe Set9, SET9 deletion mutants demonstrate <br />                                                increased radiation sensitivity defects G2/M-phase checkpoint (21,23), in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Saccharomyces, H4K20 methylated (23,24). H3K79 methylated S. pombe, and <br />                                                H3K4 methylated S. pombe (25), dependent ubiquitination of <br />                                                H2BK119 (the analog H2BK123 S. cerevisiae). To knowledge data point to <br />                                                dependence H4K20 methylation ubiquitination H2BK119 S. pombe. <br />  <br />                                                The current study examined role Bre1-dependent pathway H3K79 methylation in <br />                                                mammalian DNA damage response. In human cells, conserved modification pathway <br />                                                involves ubiquitination H2B residue K120, mediated ubiquitin ligase complex <br />                                                encoded BRE1A BRE1B genes (26,27). As yeast, facilitates methylation of <br />                                                H3K79 homologous methyltransferase hDOT1L. It known Dot1L deficiency <br />                                                leads increased radiation sensitivity mouse embryonic fibroblasts 293 cells (28) <br />                                                deletion Dot1L leads embryonic lethality mice (29). With respect radiation <br />                                                response, mammalian cells include features analogous S. cerevisiae S. <br />                                                pombe. Conflicting data reported different cell systems [reviewed ref. <br />                                                (30)]. Huyen et al. (31) reported 53BP1 protein, closely related the <br />                                                Saccharomyces Rad9 S. pombe Crb2 proteins, binds methylated H3K79 residues in <br />                                                tudor domain required recruitment 53BP1 DSB sites. In addition, <br />                                                inhibition DOT1L human U2OS cells inhibited 53BP1 recruitment DSB sites (31). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                However, Botuyan et al. (32) normal binding 53BP1 DSB sites Dot1-deficient <br />                                                mouse embryonic fibroblast cells evidence instead 53BP1 binds dimethylated <br />                                                H4K20 residues dependent H4K20 methylation recruitment foci after <br />                                                irradiation. In mice, loss H4K20 methylation associated defect in <br />                                                G1/S transition G2/M (entry mitosis) checkpoint defect increased <br />                                                sensitivity radiation (33). <br />  <br />                                                In contrast studies phenotypes related down-regulation H3K79 methylation, there <br />                                                appears little direct information concerning role H2BK120 monoubiquitination in <br />                                                radiation resistance cell cycle checkpoints mammals. Given complex roles different <br />                                                histone modifications radiation responses fact H2BK120 ubiquitination has <br />                                                major impacts H3K4 methylation (1,2,16,17) addition role H3K79 methylation <br />                                                transcription (27,34,35), wished determine down-regulating H2B <br />                                                ubiquitination affect DNA repair cell cycle checkpoints. We constructed <br />  <br />  <br />                                                    Radiat Res. Author manuscript; available PMC 2011 November 1. <br />                             Chernikova et al.                                                                                            Page 3 <br />  <br />  <br />                                                shRNA knockdown cell lines involving mouse Bre1a Bre1b human BRE1A and <br />                                                BRE1B, report effects knockdowns sensitivity radiation and <br />                                                chlorambucil, formation Rad51 foci, G2/M checkpoint. H2BK120 ubiquitination is <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                thought facilitate methyltransferase access H3K79, H3K79 H4K20 are <br />                                                located close nucleosomal surface (36), possible defects <br />                                                radiation response Bre1-deficient cells result reduced methylation of <br />                                                H4K20. However, confirmed Bre1 knockdown reduces H3K79 dimethylation, <br />                                                effect H4K20, data support role H2B ubiquitination mammalian <br />                                                recombinational repair mediated H3K79 methylation analogous in <br />                                                Saccharomyces cerevisiae. <br />  <br />                               MATERIALS AND METHODS <br />                               Cell Culture Reagents <br />                                                Radiation-induced mouse fibrosarcoma (RIF-1) cells cultured Dulbecco&#8217;s modified <br />                                                Eagle medium glutamate (Gibco BRL) 15% tetracycline-free fetal bovine serum <br />                                                (Clontech). The <span id='am-1' about='obo:NCBITaxon_11646' typeof='owl:Thing obo:NCBITaxon_11632 obo:NCBITaxon_1 obo:NCBITaxon_35268 obo:NCBITaxon_10239 obo:NCBITaxon_327045'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Lentivirinae" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5'  ></span><span id='am-6' property="rdfs:label" content="Lentivirus" datatype="xsd:string"></span>lentivirus</span>-based tetracycline-inducible shRNA-expressing was <br />                                                obtained Dr. Eric Campeau, University Massachusetts (37). <br />  <br />                               Antibodies <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                The following antibodies used study: anti-H3K79me2 (Abcam), anti-H3K79me1 <br />                                                (Abcam), anti-H4K20me2 (Upstate), antipanH3 (Upstate). Polyclonal antibody recognizing <br />                                                Bre1b (Rnf40) custom Zymed Laboratories (Invitrogen, CA). The antibody was <br />                                                raised rabbits synthetic peptide (CQRVYSRGDSEAPGE) conjugated KLH under <br />                                                proprietary PolyQuick regimen, affinity purified using Sulfolink kit (Pierce, IL). <br />  <br />                               Short Hairpin RNA (shRNA) Transduction <br />                                                The shRNA sequences directed GFP (5&#8242;-GAAGCAGCACGACTTCTTC-3&#8242;), mBre1a <br />                                                (sequence &#8220;Bre1a shRNA1&#65533;? 5&#8242;-ACATCCGCATCATCCTTAA-3&#8242;, sequence &#8220;Bre1a <br />                                                shRNA2&#65533;? 5&#8242;-GTGAAGTCCTAAGGTATAA-3&#8242;), mBre1b (sequence &#8220;Bre1b shRNA1&#65533;? 5&#8242;- <br />                                                ACATATGGAGAGTGATGAA-3&#8242;, sequence &#8220;Bre1b shRNA2&#65533;? 5&#8242;- <br />                                                CAAGATAAAGTGACATCGA-3&#8242;) cloned lentiviral <span id='am-69' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-70' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>vectors</span> conferring puromycin <br />                                                hygromycin resistance (37). Lentiviruses created described ref. (37). RIF-1 cells <br />                                                transduced shRNA-encoding lentiviruses selected corresponding antibiotic <br />                                                7&#8211;10 days. To create RIF-1 cells inducible expression shRNAs, cells were <br />                                                cotransduced lentivirus-containing expression vector Tet repressor protein were <br />                                                grown 2 weeks select stably transduced heterogeneous clonal cell populations. <br />                                                shRNA expression induced treatment 2 &#181;g/ml doxycycline 3&#8211;6 days before <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                assessment target gene knockdown. <br />  <br />                               RT-PCR Analysis <br />                                                Total RNA extracts prepared using RNeasy kit (Qiagen). cDNAs synthesized <br />                                                M-MLV reverse transcriptase (Invitrogen). Both oligo-(dT) random-hexamer primers <br />                                                used obtain cDNAs. The sequences primers specific Bre1a, Bre1b GAPDH <br />                                                were: Bre1aF, 5&#8242;-GCTTCGGGAACACATTGAAAAG-3&#8242;; Bre1aR, 5&#8242;- <br />                                                CTGTGAGTAGGTCCCCCAAAC-3&#8242;; Bre1bF, 5&#8242;-ACACTCCTCATCGTCAACCG-3&#8242;; <br />                                                Bre1bR, 5&#8242;-GGGATCACATCACGGGTCAG-3&#8242;; GAPDHF, 5&#8242;- <br />                                                TGACCACAGTCCATGCCATC-3&#8242;; GAPDHR, 5&#8242;- <br />                                                GACGGACACATTGGGGGTAG-3&#8242;. <br />  <br />  <br />  <br />  <br />                                                    Radiat Res. Author manuscript; available PMC 2011 November 1. <br />                             Chernikova et al.                                                                                             Page 4 <br />  <br />  <br />                               Western Blot Analysis <br />                                                Total cellular protein extracts prepared lysing cell pellets buffer containing 50 <br />                                                mM Tris, pH 7.5, 1% SDS, 10% glycerol, 0.15M NaCl, 0.5 mM EDTA, 0.2 mM PMSF, and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                100&#215; protease inhibitor cocktail (Sigma-Aldrich). Twenty 50 &#181;g protein samples was <br />                                                separated 4&#8211;12% NuPAGE Bis-Tris polyacrylamide gels (Invitrogen) transferred to <br />                                                nitrocellulose membranes. Protein blots probed antibodies, washed incubated <br />                                                corresponding secondary antibodies conjugated alkaline phosphatase (Vector <br />                                                Laboratories). Protein bands detected using Storm scanner (Molecular Dynamics). <br />  <br />                               Radiation Drug Treatment <br />                                                After irradiation (MARK-1 cesium &#947;-ray source, J. L. Shepherd Associates, dose rate 11.49 <br />                                                Gy min) 1 h drug treatment, cells immediately plated colony formation. Cells <br />                                                plated triplicate treatment condition. After 10&#8211;14 days dishes rinsed with <br />                                                0.9% NaCl stained crystal violet, visible colonies containing 50 cells <br />                                                counted. <br />  <br />                               G2 Cell Cycle Delay <br />                                                Cells irradiated 4 Gy &#947; rays fixed FACS analysis 16 h later. <br />  <br />                               G2/M Cell Cycle Checkpoint Analysis <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Cells irradiated 3 Gy 1 h irradiation fixed FACS analysis. FITC- <br />                                                conjugated antibody phospho-H3Ser10 (Upstate) used visualization mitotic <br />                                                cells. <br />  <br />                               Immunofluorecsence <br />                                                Cells grown cover slips, irradiated 6 Gy, fixed different times <br />                                                postirradiation using 2% (w/v) paraformaldehyde PBS permeabilized using 0.1% Triton <br />                                                X-100 solution. Cells incubated anti-Rad51 antibody (H-92, Santa Cruz <br />                                                Biotechnologies, 1:200) detected secondary anti-rabbit Alexa594 antibody <br />                                                (Invitrogen, 1:400). To quantify Rad51 foci, counted cells 10 distinct bright <br />                                                foci. At 200 nuclei counted sample. <br />  <br />                               RESULTS <br />                               Depletion Bre1a Bre1b Mouse Cells Leads Increased Sensitivity Ionizing <br />                               Radiation <br />                                                To establish Bre1 homologs contribute radiation resistance mammalian cells, we <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                designed shRNA sequences uniquely targeting expression BRE1a BRE1b (Fig. 1A and <br />                                                B). BRE1-mediated monoubiquitination histone H2B facilitates methylation histone H3 <br />                                                residues K4 K79 (26, 27). As expected, RNAi knockdown of Bre1 mouse <br />                                                homologs Bre1a Bre1b reduced dimethylation histone H3K79 (Fig. 1C). The cells that <br />                                                displayed reduction methylation H3K79 showed increased sensitivity ionizing <br />                                                radiation (Fig. 1C D). The cell line expressing construct Bre1b shRNA2 failed knock <br />                                                Bre1b exhibited survival comparable control GFP shRNA cells. Attempts <br />                                                increase level knockdown, double transduction Bre1a Bre1b <br />                                                shRNA transduction greater quantities shRNA, unsuccessful. Of note, RIF-1 <br />                                                cells constitutively expressing Bre1a Bre1b shRNAs grew significantly slowly than <br />                                                mock-treated control cells expressing GFP shRNA (data shown). This observation and <br />                                                fact attempts increase knockdown proteins unsuccessful suggest that <br />                                                cells need critical level Bre1a Bre1b survive. To avoid problem insufficient <br />                                                cell numbers, created stable cell lines knockdown Bre1a and/or Bre1b could <br />  <br />                                                    Radiat Res. Author manuscript; available PMC 2011 November 1. <br />                             Chernikova et al.                                                                                                Page 5 <br />  <br />  <br />                                                induced addition doxycycline. Five days addition doxycycline, Bre1b <br />                                                protein detected Bre1b shRNA-inducible cells (Fig. 2A), Bre1b protein was <br />                                                present cells treated doxycycline GFP shRNA cells without <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                doxycycline. However, doxycycline-inducible Bre1b shRNA leaky, partial <br />                                                knockdown Bre1b occurred doxycycline present (Fig. 2A). The levels <br />                                                H3K79me2 proportional levels Bre1b protein, monomethylation of <br />                                                H3K79 affected lesser degree (Fig. 2A). Moreover, decrease levels H3K79 <br />                                                dimethylation strongly correlated increase radiation sensitivity (Fig. 2B). We did <br />                                                notice major changes cell cycle distribution caused depletion Bre1b (data shown <br />                                                Fig. 3B). Depletion Bre1b did affect levels H4K20me2 Rad51 (Fig. 2A). <br />  <br />                               Depletion BRE1A Human Cells Associated Defective G2/M Arrest <br />                                                RIF-1 cell cultures previously reported contain small fraction tetraploid cells <br />                                                (38); therefore, investigate function radiation-induced checkpoints cells Bre1 <br />                                                knockdown, chose use U2OS cells instead. As knockdown Bre1a/b <br />                                                complex RIF-1 cells, knockdown BRE1A led reduction dimethylation of <br />                                                H3K79 (Fig. 3A). FACS analysis U2OS cells exposed 4 Gy ionizing radiation and <br />                                                fixed 16 h irradiation revealed increase fraction cells G2/M content (from <br />                                                ~26% unirradiated cells ~77% cells treated 4 Gy), pointing normal function <br />                                                G2 delay checkpoints (Fig. 3B). Cells expressing BRE1A shRNA displayed delay <br />                                                G2 (G2/M content increased ~31% ~71%) (Fig. 3B). Abrogation 53BP1 been <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                reported result defect G2/M cell cycle arrest detectable exposure low doses <br />                                                (3 Gy) radiation (39). Because H3K79me2 reported binding targets the <br />                                                tudor domain 53BP1, likely reduction H3K79me2 result defective <br />                                                G2/M cell cycle arrest. To assess G2/M checkpoint BRE1A-depleted cells, irradiated <br />                                                cells 3 Gy fixed 1 h postirradiation. As shown Fig. 3C, hBRE1-deficient cells <br />                                                displayed radiation-induced G2/M checkpoint defect, fivefold cells <br />                                                entered mitosis BRE1A-depleted cells control cells expressing GFP shRNA. <br />  <br />                               Depletion Bre1b Mouse Cells Leads Defects Homologous Recombination <br />                                                A defect homologous recombination (HR) repair yeast bre1&#916; mutant reported <br />                                                previously Game et al. (6) basis radiation sensitivity epistasis analysis. <br />                                                Consistent defect HR, Bre1b knockdown cells study showed drastic <br />                                                reduction radiation-dependent formation Rad51 foci (Fig. 4A B) increased <br />                                                sensitivity crosslinking agents chlorambucil mitomycin C (Fig. 4C our <br />                                                unpublished observations) ionizing radiation (Fig. 1D 2B). The level spontaneous <br />                                                Rad51 foci significantly lower Bre1b knockdown cells (Fig. 4B), suggesting <br />                                                Bre1b-deficient cells efficiently resolve double-strand breaks occurring during <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                replication. The HR defect associated loss Bre1b RIF-1 cells manifested itself <br />                                                overall reduction delay Rad51 focus formation response radiation. <br />                                                Although frequency cells Rad51 foci similar Bre1b shRNA(&#8722;) <br />                                                cells control GFP shRNA(&#8722;/+) cells, foci formed Bre1b shRNA(&#8722;) cells were <br />                                                smaller. The levels Rad51 protein Bre1b knockdown cells affected the <br />                                                loss Bre1b (Fig. 1A), analysis gene expression array (to reported elsewhere) <br />                                                suggested HR genes affected depletion Bre1a Bre1b. Here <br />                                                deficiency Bre1a Bre1b resulted suboptimal expression the <br />                                                Brca1, Bard1 Rad51L1 genes (Fig. 4D). <br />  <br />                               DISCUSSION <br />                                                Using shRNA suppress function Bre1a/b (BRE1A/B) complex, examined the <br />                                                role proteins radiation response mammalian cells. We knockdown <br />  <br />  <br />                                                    Radiat Res. Author manuscript; available PMC 2011 November 1. <br />                             Chernikova et al.                                                                                               Page 6 <br />  <br />  <br />                                                Bre1a Bre1b mouse cells resulted moderate significant increase in <br />                                                radiation sensitivity. In addition, depletion BRE1A resulted defective G2/M checkpoint <br />                                                arrest human cells. We compromised function Bre1a/b complex was <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                associated defects homologous recombination, manifested increased sensitivity to <br />                                                crosslinking agents abnormal recruitment Rad51 protein ionizing radiation- <br />                                                induced foci. Our results indicate defects observed Bre1a/b knockdowns not <br />                                                altered methylation H4K20 associated reduced dimethylation of <br />                                                H3K79. <br />  <br />                                                Monoubiquitination H2B mediated Bre1 known facilitate methylation histone <br />                                                H3 residues K4 K79 yeast mammals (1,2,26,27). Extensive evidence points a <br />                                                significant role H3K79 methylation radiation sensitivity yeast, methylation of <br />                                                H3K4 thought impact, yeast mutants defective H3K4 are <br />                                                sensitive ionizing radiation (6). It believed H3K79 methylation plays role <br />                                                radiation response binding Rad9 (or ortholog 53BP1) tudor domain to <br />                                                dimethylated H3K79. Yeast cells mutant Rad9 tudor domain radiation sensitivity <br />                                                similar dot1 deletion mutants (and rad9 deletion mutant) (7), and <br />                                                deletion DOT1 does add sensitivity rad9 tudor domain (7) rad9 deletion <br />                                                (6) mutants. In mammalian cells, recruitment 53BP1-GFP fusion protein sites DNA <br />                                                damage abolished mutations tudor domain 53BP1 suppression of <br />                                                methylation H3K79 DOT1L knockdown cells (31). Knockdown Dot1L been <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                shown result increased radiation sensitivity mammalian cells (28). However, remains <br />                                                seen mutation tudor domain 53BP1 lead increased sensitivity <br />                                                radiation mammalian cells does Rad9 Saccharomyces (7). <br />  <br />                                                Similarly, H3K79 methylation shown participate activation G1 and <br />                                                intra-S checkpoints response DNA damage Saccharomyces (3,4,6). In studies the <br />                                                bre1&#916; strains largely showed cell cycle checkpoint defects dot1&#916;, rad9 tudor <br />                                                domain H3K79 mutants, implying checkpoint role Bre1 mediated the <br />                                                H3K79 dimethylation regulates. These studies showed obvious defect G2/M mitotic <br />                                                checkpoint cells arrested nocodazole irradiation. Note &#8220;G2/M&#65533;? checkpoint <br />                                                measured nocodazole-arrested cells distinguished G2/M mitotic entry <br />                                                checkpoint, nocodazole arrests cells prometaphase mitosis, G2/M <br />                                                checkpoint controls transition G2 M. None studies examined the <br />                                                function G2/M transition checkpoint known defective 53BP1-deficient cells <br />                                                (39). <br />  <br />                                                The controversy remains unresolved methylated residue (H3K79me2 or <br />                                                H4K20me2) preferred damage sensor 53BP1 mammalian cells [see ref. (30) for <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                review]. Our data H4K20 dimethylation affected Bre1a/b status mouse <br />                                                RIF-1 cells. Therefore, based available evidence, attribute defects radiation <br />                                                response Bre1a Bre1b knockdowns changes Bre1-dependent dimethylation of <br />                                                H3K79. Separate evidence supporting roles of H3K79me2 H4K20me2 strong <br />                                                (31,32), role modifications radiation response in <br />                                                mammalian cells, modifications probably important a <br />                                                specific situation depending cell cycle stage cell type. Thus, defect in <br />                                                G2/M checkpoint observed BRE1A-deficient cells explained suboptimal <br />                                                H2B ubiquitination-regulated transcription cell cycle control genes (our unpublished <br />                                                observations), caused abnormal recruitment 53BP1 sites double- <br />                                                strand breaks resulting impaired H3K79 dimethylation. <br />  <br />                                                A defect HR consequences Bre1a/b depletion contribute directly <br />                                                increased radiation sensitivity. HR repair shown defective yeast <br />  <br />  <br />                                                    Radiat Res. Author manuscript; available PMC 2011 November 1. <br />                             Chernikova et al.                                                                                                             Page 7 <br />  <br />  <br />                                                bre1&#916; mutant (6). In study radiation sensitivity phenotypes single multiple <br />                                                mutants involving bre1&#916;, lge1&#916;, rtf1&#916;, dot1&#916; rad51&#916; suggested gene affected <br />                                                radiation resistance RAD51-dependent mechanism mediated H3K79 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                methylation. Consistent this, recent study showed Dot1 Rad9 required <br />                                                efficient sister chromatid recombination Saccharomyces cerevisiae (20). The observation <br />                                                (7) dot1&#916; rad9 tudor domain double mutant strain shows radiation sensitivity <br />                                                equivalent single mutants provides support argument observed <br />                                                binding Rad9 H3K79me2 involved Rad51-dependent repair. In mammalian cells, <br />                                                formation Rad51 foci reported unaffected 53BP1 status (39). Therefore, <br />                                                defect Rad51 focus formation observed consequence suboptimal <br />                                                expression HR genes [(35) observations] downregulation H2B <br />                                                ubiquitination-regulated transcription. <br />  <br />                                                In summary, mammalian Bre1 confers radiation resistance important for <br />                                                homologous recombination normal function G2/M checkpoint. We found <br />                                                loss mammalian Bre1 does affect dimethylation H4K20. Thus finding a <br />                                                radiation sensitivity phenotype Bre1-deficient cells supports role H3K79me2 in <br />                                                mammalian repair. The defects observed Bre1-depleted mammalian cells reminiscent <br />                                                ones caused deletion Bre1 Saccharomyces cerevisiae, pointing an <br />                                                evolutionary conservation pathway eukaryotes. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Acknowledgments <br />                                                We indebted Dr. Eric Campeau supplying vectors used study inducible H1 <br />                                                lentiviral RNA interference system. This work supported National Institutes Health grants CA-118202 and <br />                                                CA-067166 awarded JMB. <br />  <br />  <br />                               REFERENCES <br />                                                1. Briggs SD, Xiao T, Sun ZW, Caldwell JA, Shabanowitz J, Hunt DF, Allis CD, Strahl BD. Gene <br />                                                   silencing: transhistone regulatory pathway chromatin. Nature 2002;418:498. [PubMed: 12152067] <br />                                                2. Ng HH, Xu RM, Zhang Y, Struhl K. Ubiquitination histone H2B Rad6 required efficient <br />                                                   Dot1-mediated methylation histone H3 lysine 79. J. Biol. Chem 2002;277:34655&#8211;34657. [PubMed: <br />                                                   12167634] <br />                                                3. Giannattasio M, Lazzaro F, Plevani P, Muzi-Falconi M. The DNA damage checkpoint response <br />                                                   requires histone H2B ubiquitination Rad6-Bre1 H3 methylation Dot1. J. Biol. Chem <br />                                                   2005;280:9879&#8211;9886. [PubMed: 15632126] <br />                                                4. Wysocki R, Javaheri A, Allard S, Sha F, Cote J, Kron SJ. Role Dot1-dependent histone H3 <br />                                                   methylation G1 S phase DNA damage checkpoint functions Rad9. Mol. Cell Biol <br />                                                   2005;25:8430&#8211;8443. [PubMed: 16166626] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                5. Game JC, Williamson MS, Baccari C. X-ray survival characteristics genetic analysis nine <br />                                                   Saccharomyces deletion mutants altered radiation sensitivity. Genetics 2005;169:51&#8211;63. <br />                                                   [PubMed: 15371366] <br />                                                6. Game JC, Williamson MS, Spicakova T, Brown JM. The RAD6/BRE1 histone modification pathway <br />                                                   Saccharomyces confers radiation resistance RAD51-dependent process is <br />                                                   independent RAD18. Genetics 2006;173:1951&#8211;1968. [PubMed: 16783014] <br />                                                7. Grenon M, Costelloe T, Jimeno S, O&#8217;Shaughnessy A, Fitzgerald J, Zgheib O, Degerth L, Lowndes <br />                                                   NF. Docking chromatin Saccharomyces cerevisiae Rad9 Tudor domain. Yeast <br />                                                   2007;24:105&#8211;119. [PubMed: 17243194] <br />                                                8. Game JC, Chernikova SB. The role RAD6 recombinational repair, checkpoints meiosis via <br />                                                   histone modification. DNA Repair 2009;8:470&#8211;482. [PubMed: 19230796] <br />                                                9. Robzyk K, Recht J, Osley MA. Rad6-dependent ubiquitination histone H2B yeast. Science <br />                                                   2000;287:501&#8211;504. [PubMed: 10642555] <br />  <br />  <br />  <br />  <br />                                                     Radiat Res. Author manuscript; available PMC 2011 November 1. <br />                             Chernikova et al.                                                                                                      Page 8 <br />  <br />  <br />                                                10. Hwang WW, Venkatasubrahmanyam S, Ianculescu AG, Tong A, Boone C, Madhani HD. A conserved <br />                                                    RING finger protein required histone H2B monoubiquitination cell size control. Mol. Cell <br />                                                    2003;11:261&#8211;266. [PubMed: 12535538] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                11. Wood A, Krogan NJ, Dover J, Schneider J, Heidt J, Boateng MA, Dean K, Golshani A, Zhang Y, <br />                                                    Shilatifard A. Bre1, E3 ubiquitin ligase required recruitment substrate selection Rad6 <br />                                                    promoter. Mol. Cell 2003;11:267&#8211;274. [PubMed: 12535539] <br />                                                12. Feng Q, Wang H, Ng HH, Erdjument-Bromage H, Tempst P, Struhl K, Zhang Y. Methylation H3- <br />                                                    lysine 79 mediated new <span id='am-51' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-52' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> HMTases SET domain. Curr. Biol 2002;12:1052&#8211; <br />                                                    1058. [PubMed: 12123582] <br />                                                13. Lacoste N, Utley RT, Hunter JM, Poirier GG, Cote J. Disruptor telomeric silencing-1 chromatin- <br />                                                    specific histone H3 methyltransferase. J. Biol. Chem 2002;277:30421&#8211;30424. [PubMed: 12097318] <br />                                                14. van Leeuwen F, Gafken PR, Gottschling DE. Dot1p modulates silencing yeast methylation of <br />                                                    nucleosome core. Cell 2002;109:745&#8211;756. [PubMed: 12086673] <br />                                                15. Dehe PM, Pamblanco M, Luciano P, Lebrun R, Moinier D, Sendra R, Verreault A, Tordera V, Geli <br />                                                    V. Histone H3 lysine 4 mono-methylation does require ubiquitination histone H2B. J. Mol. <br />                                                    Biol 2005;353:477&#8211;484. [PubMed: 16185711] <br />                                                16. Dover J, Schneider J, Tawiah-Boateng MA, Wood A, Dean K, Johnston M, Shilatifard A. Methylation <br />                                                    histone H3 COMPASS requires ubiquitination histone H2B Rad6. J. Biol. Chem <br />                                                    2002;277:28368&#8211;28371. [PubMed: 12070136] <br />                                                17. Sun ZW, Allis CD. Ubiquitination histone H2B regulates H3 methylation gene silencing in <br />                                                    yeast. Nature 2002;418:104&#8211;108. [PubMed: 12077605] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                18. Singer MS, Kahana A, Wolf AJ, Meisinger LL, Peterson SE, Goggin C, Mahowald M, Gottschling <br />                                                    DE. Identification high-copy disruptors telomeric silencing Saccharomyces cerevisiae. <br />                                                    Genetics 1998;150:613&#8211;632. [PubMed: 9755194] <br />                                                19. Ng HH, Feng Q, Wang H, Erdjument-Bromage H, Tempst P, Zhang Y, Struhl K. Lysine methylation <br />                                                    globular domain histone H3 Dot1 important telomeric silencing Sir protein <br />                                                    association. Genes Dev 2002;16:1518&#8211;1527. [PubMed: 12080090] <br />                                                20. Conde F, San-Segundo PA. Role Dot1 response alkylating DNA damage Saccharomyces <br />                                                    cerevisiae: regulation DNA damage tolerance error-prone polymerases Polzeta/Rev1. <br />                                                    Genetics 2008;179:1197&#8211;1210. [PubMed: 18562671] <br />                                                21. Sanders SL, Portoso M, Mata J, Bahler J, Allshire RC, Kouzarides T. Methylation histone H4 <br />                                                    lysine 20 controls recruitment Crb2 sites DNA damage. Cell 2004;119:603&#8211;614. [PubMed: <br />                                                    15550243] <br />                                                22. Greeson NT, Sengupta R, <span id='am-64' about='obo:NCBITaxon_481910' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_4210 obo:NCBITaxon_1 obo:NCBITaxon_877976 obo:NCBITaxon_102809 obo:NCBITaxon_131567 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_102804 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_4209 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_35493 obo:NCBITaxon_2759 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Arida" datatype="xsd:string"></span><span id='am-68'  ></span>Arida</span> AR, Jenuwein T, Sanders SL. Di-methyl H4 Lysine 20 targets the <br />                                                    checkpoint protein Crb2 sites DNA damage. J. Biol. Chem 2008;283:33168&#8211;33174. [PubMed: <br />                                                    18826944] <br />                                                23. Nishioka K, Rice JC, Sarma K, Erdjument-Bromage H, Werner J, Wang Y, Chuikov S, <span id='am-54' about='obo:NCBITaxon_186129' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_38127 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_239228 obo:NCBITaxon_298052 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_159975 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_30259 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33342 obo:NCBITaxon_50557'><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57'  ></span><span id='am-58' property="rdfs:label" content="Valenzuela" datatype="xsd:string"></span>Valenzuela</span> <br />                                                    P, Tempst P, Reinberg D. PR-Set7 nucleosome-specific methyltransferase modifies lysine <br />                                                    20 histone H4 associated silent chromatin. Mol. Cell 2002;9:1201&#8211;1213. [PubMed: <br />                                                    12086618] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                24. Fang J, Feng Q, Ketel CS, Wang H, Cao R, Xia L, Erdjument-Bromage H, Tempst P, Simon JA, <br />                                                    Zhang Y. Purification functional characterization SET8, nucleosomal histone H4-lysine 20- <br />                                                    specific methyltransferase. Curr. Biol 2002;12:1086&#8211;1099. [PubMed: 12121615] <br />                                                25. Zofall M, Grewal SI. HULC, histone H2B ubiquitinating complex, modulates heterochromatin <br />                                                    independent histone methylation fission yeast. J. Biol. Chem 2007;282:14065&#8211;14072. [PubMed: <br />                                                    17363370] <br />                                                26. Zhu B, Zheng Y, Pham AD, Mandal SS, Erdjument-Bromage H, Tempst P, Reinberg D. <br />                                                    Monoubiquitination human histone H2B: factors involved roles HOX gene <br />                                                    regulation. Mol. Cell 2005;20:601&#8211;611. [PubMed: 16307923] <br />                                                27. Kim J, Hake SB, Roeder RG. The human homolog yeast BRE1 functions transcriptional <br />                                                    coactivator direct activator interactions. Mol. Cell 2005;20:759&#8211;770. [PubMed: 16337599] <br />                                                28. Lin YH, Kakadia PM, Chen Y, Li YQ, Deshpande AJ, Buske C, Zhang KL, Zhang Y, Xu GL, <br />                                                    Bohlander SK. Global reduction epigenetic H3K79 methylation mark increased <br />  <br />  <br />  <br />                                                     Radiat Res. Author manuscript; available PMC 2011 November 1. <br />                             Chernikova et al.                                                                                                 Page 9 <br />  <br />  <br />                                                    chromosomal instability CALM-AF10-positive leukemias. Blood 2009;114:651&#8211;658. [PubMed: <br />                                                    19443658] <br />                                                29. Jones B, Su H, Bhat A, Lei H, Bajko J, Hevi S, Baltus GA, Kadam S, Zhai H, Chen T. The histone <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    H3K79 methyltransferase Dot1L essential mammalian development heterochromatin <br />                                                    structure. PLoS Genet 2008;4:e1000190. [PubMed: 18787701] <br />                                                30. FitzGerald JE, Grenon M, Lowndes NF. 53BP1: function mechanisms focal recruitment. <br />                                                    Biochem. Soc. Trans 2009;37:897&#8211;904. [PubMed: 19614615] <br />                                                31. Huyen Y, Zgheib O, Ditullio RA Jr, Gorgoulis VG, Zacharatos P, Petty TJ, Sheston EA, Mellert HS, <br />                                                    Stavridi ES, Halazonetis TD. Methylated lysine 79 histone H3 targets 53BP1 DNA double- <br />                                                    strand breaks. Nature 2004;432:406&#8211;411. [PubMed: 15525939] <br />                                                32. Botuyan MV, Lee J, Ward IM, Kim JE, Thompson JR, Chen J, Mer G. Structural basis the <br />                                                    methylation state-specific recognition histone H4-K20 53BP1 Crb2 DNA repair. Cell <br />                                                    2006;127:1361&#8211;1373. [PubMed: 17190600] <br />                                                33. Schotta G, Sengupta R, Kubicek S, Malin S, Kauer M, Callen E, Celeste A, Pagani M, Opravil S, <br />                                                    Jenuwein T. A chromatin-wide transition H4K20 monomethylation impairs genome integrity and <br />                                                    programmed DNA rearrangements mouse. Genes Dev 2008;22:2048&#8211;2061. [PubMed: <br />                                                    18676810] <br />                                                34. Kao CF, Hillyer C, Tsukuda T, Henry K, Berger S, Osley MA. Rad6 plays role transcriptional <br />                                                    activation ubiquitylation histone H2B. Genes Dev 2004;18:184&#8211;195. [PubMed: <br />                                                    14752010] <br />                                                35. Shema E, Tirosh I, Aylon Y, Huang J, Ye C, Moskovits N, Raver-Shapira N, Minsky N, Pirngruber <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    J, Oren M. The histone H2B-specific ubiquitin ligase RNF20/hBRE1 acts putative tumor <br />                                                    suppressor selective regulation gene expression. Genes Dev 2008;22:2664&#8211;2676. <br />                                                    [PubMed: 18832071] <br />                                                36. White CL, Suto RK, Luger K. Structure yeast nucleosome core particle reveals fundamental <br />                                   <br /> </body></html>