<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Cytometry A. Author manuscript; available PMC 2009 July 1. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Cytometry A. 2008 July ; 73(7): 637&#8211;642. doi:10.1002/cyto.a.20579. <br />  <br />  <br />  <br />                            A long-term flow cytometry assay analyze role specific <br />                            genes <span id='am-20' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-26'  ></span>Drosophila melanogaster</span> S2 cells surviving <br />                            genotoxic stress <br />                            Xia Yi1,2, Willy Lemstra1, Michel J. Vos1, Yongfeng Shang2, Harm H. Kampinga1, Tin Tin <br />                            Su3, Ody C.M. Sibon1,* <br />  <br />                            1Department Radiation Stress Cell Biology, Division Cell Biology, University Medical <br />                            Center Groningen, University Groningen, The Netherlands 2 Department Biochemistry and <br />                            Molecular Biology, Peking University Health Science Center, Beijing 100083, China 3 Department <br />                            Molecular, Cellular Developmental Biology University Colorado, Boulder, CO <br />                            80309-0347 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Abstract <br />                                  Background&#8212;Drosophila S2 cells easy manipulate culture versatile <br />                                  model high throughput screens genome-wide siRNA screens genes <br />                                  involved stress therapy resistance screening large compound libraries to <br />                                  identify cytotoxins. Clonogenic assays considered gold-standard investigate the <br />                                  cytotoxicity specific treatments compare sensitivity various cell types specific <br />                                  treatment. However, assay used Drosophila S2 cells virtually unable <br />                                  grow distinct colonies. <br />                                  Methods&#8212;We designed novel fluorescence-based flow cytometry assay study long-term <br />                                  proliferation S2 cells various conditions presence specific gene products or <br />                                  downregulation specific gene products. <br />                                  Results&#8212;Here validate assay used novel method investigate role of <br />                                  checkpoint genes grapes/Dchk1 DmChk2 cell survival responses. <br />                                  Conclusions&#8212;Our <span id='am-65' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-66' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-67'  ></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> demonstrate Grapes/Dchk1 DmChk2 required survive <br />                                  hydroxyurea. Our assay use investigate long-term effects various treatments S2 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  cells evaluate role specific proteins therein. <br />  <br />                            Keywords <br />                                  viability; sensitivity; Drosophila S2 cells; FCM; DNA damage; survival assay; RNAi technology; <br />                                  grapes/Dchk1; DmChk2 <br />  <br />  <br />                            Introduction <br />                                             The ability cells form clones certain treatment used study survival <br />                                             potential specific cells (1). <span id='am-32' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36'  ></span>This</span> so-called clonogenic assay method determine <br />                                             proliferation capacity assay determines actual ability cell repeatedly <br />                                             divide form visible colony. For clonogenic assays, cells investigation <br />  <br />  <br />                            * <br />                             Corresponding author: address: Ant. Deusinglaan 1, 9713 AV Groningen, The Netherlands, e-mail: o.c.m.<span id='am-27' about='obo:NCBITaxon_211650' typeof='owl:Thing obo:NCBITaxon_1329950 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8504 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8509 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_1329912 obo:NCBITaxon_33208 obo:NCBITaxon_1329961 obo:NCBITaxon_1329911 obo:NCBITaxon_7711 obo:NCBITaxon_1390229 obo:NCBITaxon_8570 obo:NCBITaxon_7776 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_34989'><span id='am-28' property="rdfs:label" content="Sibon" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-30'  ></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>sibon</span>@med.umcg.nl; <br />                            telephone: +31(0)50 363 2559; fax: +31(0)50 363 2913. <br />                             Yi et al.                                                                                                    Page 2 <br />  <br />  <br />                                               able form colonies solid surface (for adherent cells) fluid medium or <br />                                               substrates agar non-adherent cells. Besides clonogenic assays are <br />                                               alternative methods evaluate proliferation viability cells specific conditions or <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               specific treatments. In assays, incorporation rates radiolabeled nucleotides <br />                                               3H-thymidine used indication proliferation (2). Other well-known <br />                                               assays quantify viability, MTT assay (3) MTS assay (4), measure the <br />                                               concentration products generated converted viable cells. Using alternative <br />                                               assays possible distinguish actual cell death delayed cell death, <br />                                               temporary growth inhibition permanent growth inhibition hamper proper <br />                                               data analysis. Therefore, (although time-consuming) long-term clonogenic assay is <br />                                               considered golden standard investigate proliferation capacity dividing cells. <br />                                               Drosophila <span id='am-60' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-61'  ></span><span id='am-62' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>melanogaster</span> Schneider line 2 (S2) cells derived primary culture of <br />                                               20 24-hour-old Drosophila embryos. These cells relatively easy culture the <br />                                               cells grow room temperature controlled CO2 levels. S2 cells normally grow in <br />                                               semisuspension need trypsinization prior passaging. Another powerful <br />                                               characteristic S2 cells multiple transcripts simultaneously downregulated <br />                                               near 100 % efficiency using RNAi technology transient assays (5,6). In <br />                                               addition, S2 cells highly permeable expensive transfection carriers needed to <br />                                               perform RNAi studies. Finally, S2 cells induce interferon response interferes <br />                                               RNAi gene silencing (7). Consequently, S2 cells relatively easy manipulate and <br />                                               provide powerful model investigate pro- anti- survival functions of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               specific genes. Unfortunately, clonogenic assays used S2 cells they <br />                                               adherent grow discrete colonies. Here, present simple method <br />                                               using flow cytometry (FCM) accurately compare proliferation capacity S2 cells in <br />                                               response specific treatments presence absence specific proteins an <br />                                               extended period time. A unique feature assay continual adjustment cell <br />                                               density various samples analysis, using methods. First use &#8220;feeder cells&#65533;? <br />                                               (which differentially marked experimental cells) equalize total cell densities in <br />                                               cultures subjected different treatments <span id='am-42' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> rule indirect effects survival due <br />                                               differences cell number. Second, protocol includes adjustment cell densities <br />                                               days order extend period exponential growth allowing to <br />                                               analyze long-term effects. In addition established methods, approach useful to <br />                                               study long-term effects suspension cells well, especially cells be <br />                                               analyzed clonogenic assays. <br />  <br />                                Materials methods <br />                                Cell culture <br />                                               Drosophila Schneider&#8217;s S2 cells (S2 cells), S2 cells stably express nuclear luciferase- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               eGFP S2 cells stably express H2B-GFP cultured Schneider&#8217;s Drosophila <br />                                               medium (GIBCO, Paisley, UK) supplemented 10% heat-inactivated Fetal Bovine <br />                                               Serum (Greiner, Netherlands), 100 units/ml penicillin 100 &#956;g/ml streptomycin T25 <br />                                               flasks 25&#176;C. Nuclear luciferase-eGFP (eGFP-tagged firefly luciferase fused Large T <br />                                               Antigen nuclear localization signal) expressing cells generated according the <br />                                               protocol manufacturer (Drosophila expression system, Invitrogen). H2B-GFP <br />                                               expressing S2 cells generous gift P.H. O&#8217;Farrell (San Francisco, <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span>California</span>, USA). <br />                                               In order continue exponential cell growth, cell density kept 3&#215;105 cells/ml <br />                                               3&#215;106 cells/ml. For experimental use cells cultured 6 plates 35 <br />                                               mm dishes. <br />  <br />  <br />  <br />  <br />                                                     Cytometry A. Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                               Page 3 <br />  <br />  <br />                                RNA interference <br />                                            RNA interference performed previously described (5,8). In short, grp/Dchk1 or <br />                                            DmChk2 DNA fragment 700~800 bp length containing coding sequence grp/ <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Dchk (GenBank accession no. AF057041, sense primer 493&#8211;509 anti-sense primer <br />                                            1185&#8211;1200) DmChk2 cDNA (Gen Bank accession no. NM-165318: sense primer 666&#8211; <br />                                            683 anti-sense primer 1395&#8211;1413) amplified using PCR. Each primer used the <br />                                            PCR contained 5&#8242; T7 RNA polymerase binding site <br />                                            (GAATTAATACGACTCACTATAGGGAGA) followed sequence specific grp/ <br />                                            Dchk1 DmChk2. dsRNA generated using MEGASCRIPT T7 transcription kit <br />                                            (Ambion, Austin, TX) stored &#8722;20&#176;C. Nuclear luciferase-eGFP-expressing S2 cells, <br />                                            H2B-GFP-expressing S2 cells S2 cells diluted final concentration 1&#215;106 <br />                                            cells/ml Drosophila Serum Free Medium (GIBCO, Paisley, UK). 1 ml cells plated <br />                                            35 mm dishes 10 &#956;g dsRNA added Serum Free Medium dish. No <br />                                            additional transfection agent added, demonstrated previously no <br />                                            transfection carriers required perform successful RNAi experiments S2 cells (5,6). <br />                                            The cells incubated 1 hour 25&#176;C followed addition 2 ml complete <br />                                            Schneider&#8217;s medium. Cells incubated 96 hours downregulate Grp/Dchk1 and/or <br />                                            DmChk2 proteins. Downregulation Grp/Dchk1 and/or DmChk2 RNAi controlled <br />                                            Western Blot analysis using antibodies specific Grp/Dchk1 DmChk2 the <br />                                            course experiment (Supplementary Figure 1). As control cells (further referred as <br />                                            mock RNAi-treated cells), GFP-expressing S2 cells transfected non-relevant <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            dsRNA construct dsRNA added medium. As non-relevant dsRNA <br />                                            construct, fragment coding sequence stonewall used, stonewall gene <br />                                            required oogenesis (9)(see supplementary file). Kinetics downregulation Grp/ <br />                                            Dchk1 DmChk2 comparable nuclear luciferase-eGFP-expressing cells, H2B- <br />                                            GFP-expressing cells S2 cells. <br />  <br />                                Proliferation Assay <br />                                            GFP-expressing-S2 cells (Figure 1) transfected dsRNA (e.g. dsRNA of <br />                                            Grp/Dchk1 and/or DmChk2). After 4 days, RNAi-treated GFP-S2 cells mixed 1:1 with <br />                                            non-GFP expressing S2 cells (further referred feeder cells). All experiments shown <br />                                            manuscript performed using Nuclear luciferase-eGFP expressing cells H2B- <br />                                            GFP-expressing cells. After overnight incubation, mixed cells treated 10 mM <br />                                            HU (Sigma, St Louis, MO, USA) 6 hours left untreated. This time point is <br />                                            considered day 0 experiment. Subsequently, fraction living cells was <br />                                            calculated staining trypan blue detects dead cells (note: immediately HU <br />                                            treatment, increase observed dead cells compared untreated <br />                                            cells). For condition following protocol used. The mixed populations cells <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            (containing GFP-expressing cells feeder cells) subcultured split two <br />                                            fractions: cohort <span id='am-7' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Ia</span> cohort Ib. Cohort Ia seeded density 3 &#215; 105 living cells <br />                                            ml medium cultured 6-well plates FCM analysis day 2 performed. <br />                                            For detailed describtion FCM analysis, supplementary Figure 3. Cohort Ib seeded <br />                                            density 3 &#215; 105 living cells ml cultured T25 flask FCM analysis <br />                                            later time points (day 4, 6, 9, 11, 13). At day 2 (48 hours HU treatment), the <br />                                            percentage GFP-expressing cells cohort Ia measured using FCM analysis. Cells of <br />                                            cohort Ib split fractions (cohort IIa cohort IIb), suspended again <br />                                            3 &#215; 105 (cohort IIa) viable cells ml medium seeded 6-well plates FCM <br />                                            analysis day 4 3 &#215; 105 viable cells (cohort IIb) ml cultured T25 flask <br />                                            FCM analysis later time points (day 6, 9, 11, 13) etc. During course the <br />                                            experiment, 2 days density cells adjusted diluting mixed <br />                                            populations cells fresh medium density 3 &#215; 105 living cells ml was <br />                                            reached. The percentage viable cells calculated using trypan blue staining cell <br />  <br />  <br />                                                   Cytometry A. Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                                  Page 4 <br />  <br />  <br />                                            density corrected resulting constant narrow range viable cell density <br />                                            experiment (see supplementary file details). For every <br />                                            condition, 3 independent experiments performed data presented mean &#177; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            standard deviation mean (SDM). <br />  <br />                                FCM analysis <br />                                            Cells harvested centrifugation, washed phosphate-buffered saline (PBS), <br />                                            cell pellets resuspended 0.2&#8211;0.5 ml PBS. The percentage GFP-expressing <br />                                            cells determined using FACSCalibur Flow Cytometer (Becton Dickinson, San Jose, <br />                                            CA), using 488 nm <span id='am-45' about='obo:NCBITaxon_354775' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_40096 obo:NCBITaxon_40092 obo:NCBITaxon_33208 obo:NCBITaxon_40093 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-47'  ></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Argon" datatype="xsd:string"></span>Argon</span> <span id='am-50' about='obo:NCBITaxon_62990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_241799 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_241789 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-51'  ></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Laser" datatype="xsd:string"></span>laser</span> (15 mWatt) standard filters. 10,000 events were <br />                                            measured sample. FL1 used detect GFP fluorescence FL2 for <br />                                            autofluorescence. Data analysis WinList (version 6.0) Verity Software <br />                                            House (Topsham, ME) (see supplementary file). In typical experiment every <br />                                            condition independently performed measured triplicate. <br />  <br />                                Western blot analysis <br />                                            Western blot analysis S2 cells prepared previously described using primary rabbit <br />                                            polyclonal antibodies Grp/Dchk1 DmChk2 (5)commercial available secondary <br />                                            antibodies ECL detection reagents (Amersham Biosciences, Amersham Place, UK). &#947;- <br />                                            tubulin (Sigma T6557, St Louis, MO, USA) used loading control (see also <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            supplementary file). <br />  <br />                                Results <br />                                            Drosophila S2 cells widely used study function specific genes (6,10&#8211;12). <br />                                            Recently, demonstrated S2 cells possess functional checkpoint delays entry <br />                                            cells mitosis DNA damaged ionizing radiation (IR) DNA <br />                                            replication inhibited hydroxyurea (HU) (5). In order investigate role of <br />                                            specific Drosophila genes responses DNA damaging agents, long-term survival <br />                                            assay, applicable S2 cells, required. Firstly, investigated S2 cell survival <br />                                            investigated using clonogenic assay. Despite attempts unable to <br />                                            obtain reproducible results using clonogenic assays S2 cells did form colonies when <br />                                            grown various types agar (data shown). <br />  <br />                                            As (mammalian) tissue culture cells, S2 cells investigation be <br />                                            cultured conditions cell densities extensively fluctuate which <br />                                            cells grow exponentially. In laboratory S2 cells routinely cultured cell density <br />                                            3 &#215; 105 cells/ml 3 &#215; 106 cells/ml. S2 cell doubling time hampered when <br />                                            density lower 3 &#215; 105 cells/ml higher 3 &#215; 106 cells/ml. The cell doubling time <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            S2 cells approximately 24 hours. Therefore survival responses need investigated <br />                                            3&#8211;4 days, cells grow exponentially. The aim study design <br />                                            long-term (&gt;4 days) survival assay S2 cells provide method study function <br />                                            specific genes (such grapes/Dchk1 (further referred grp/Dchk1) DmChk2) in <br />                                            survival response various DNA damaging treatments. Because cell density influences S2 <br />                                            cell proliferation, long-term assay, cell density needs kept strict <br />                                            control allowing cells grow exponentially. <br />  <br />                                A long-term survival assay investigate role Grp/Dchk1 DNA damage response <br />                                pathways <br />                                            As alternative method clonogenic assay designed method depicted Figure <br />                                            1. In assay proliferation rate specific cell type compared the <br />                                            proliferation rate specific cell type protein downregulated (e.g. <br />  <br />  <br />                                                  Cytometry A. Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                           Page 5 <br />  <br />  <br />                                        Grp/DmChk1). This measured presence absence specific treatment (e.g. <br />                                        HU). In assay types cells mixed 1:1 ratio: GFP-expressing cells in <br />                                        specific proteins downregulated non-GFP-expressing cells serve to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        compensate cell density (and referred feeder cells). As GFP-expressing <br />                                        cells, nuclear luciferase-eGFP-expressing cells used. The percentage GFP- <br />                                        expressing cells mixture feeder cells measured FCM analysis. As a <br />                                        control GFP-expressing cells mock transfected transfected non-relevant <br />                                        dsRNA. These control cells mixed type feeder cells. To <br />                                        validate method, effect proliferation capacity Grp/Dchk1-downregulated <br />                                        cells compared survival mock-RNAi treated (= Grp/Dchk1 containing) cells <br />                                        control conditions treatment DNA synthesis inhibitor HU. Grp/ <br />                                        Dchk1 required cell cycle checkpoint regulation response HU (5). In absence <br />                                        Grp/Dchk1 presence replication inhibition, cells cell cycle <br />                                        delay, enter mitosis presence incompletely replicated DNA abnormal <br />                                        cytokineses observed (5,13). Based this, prediction Grp/Dchk1- <br />                                        downregulated cells decrease proliferative capacity HU treatment compared <br />                                        mock-RNAi treated cells. <br />  <br />                                        Firstly, efficiency RNAi treatment examined Western blot analysis 4 <br />                                        days start RNAi treatment, Grp/Dchk1 levels undetectable and <br />                                        remained undetectable 9 days (Figure 2). Because Grp/Dchk1 protein levels <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        undetectable 4 days start RENAi treatment, experiments mixed <br />                                        populations cells started time point (4 days RNAi treatment therefore <br />                                        corresponds 0 days HU treatment). Subsequently, tested Grp/ <br />                                        Dchk1-downregulation affects proliferation rate cells. Hereto percentage mock <br />                                        RNAi-treated GFP-expressing cells (within mixture feeder cells) measured <br />                                        various times compared percentage Grp/Dchk1-downregulated GFP- <br />                                        expressing cells (within mixture feeder cells). As presented Figure 3A, there <br />                                        large difference proliferation rate mock-RNAi treated GFP-expressing cells <br />                                        Grp/Dchk1-RNAi treated GFP-expressing cells (see supplementary file statistical <br />                                        analysis). Next, tested role Grp/Dchk1 surviving HU treatment. Hereto the <br />                                        percentage GFP-positive cells HU treatment measured various time points. <br />                                        Figure 3A demonstrates HU reduces proliferation larger extent Grp/Dchk1- <br />                                        downregulated cells compared mock-RNAi treated cells. In order exclude the <br />                                        observed decrease proliferation capacity Grp/Dchk1-downregulated cells HU is <br />                                        specific nuclear luciferase-eGFP-expressing cells, Grp/Dchk1 was <br />                                        downregulated H2B-GFP-expressing cells (although absolute ratios were <br />                                        different) resulted comparable outcome cells Grp/Dchk1 is <br />                                        downregulated increased sensitivity HU compared cells Grp/Dchk1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        downregulated (Figure 3B). Note regardless normal Grp/Dchk1 <br />                                        expression downregulation Grp/Dchk1, GFP-expressing cells appear a <br />                                        growth advantage feeder cells percentage fluorescent cells increased <br />                                        50% course 13 days. However, does affect outcome <br />                                        experiments reverse experiment performed Grp/Dchk1 was <br />                                        downregulated non-GFP expressing cells (and GFP-expressing cells served feeder <br />                                        cells), resulted outcome Grp/Dchk1-downregulated cells showed an <br />                                        impaired proliferation capacity HU treatment compared mock-RNAi treated cells <br />                                        (Supplementary Figure 4). These data consistent previous observations <br />                                        demonstrating grp/Dchk1 mutant larvae sensitive HU (13&#8211;15) data that <br />                                        Grp/Dchk1 required prevent mitotic catastrophe presence HU (5), <br />                                        demonstrating validity assay. <br />  <br />  <br />  <br />  <br />                                              Cytometry A. Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                                 Page 6 <br />  <br />  <br />                                DmChk2 required survive HU treatment <br />                                            After validation survival assay, tested effect downregulation another <br />                                            protein response HU. DmChk2 (or Dmnk) conserved checkpoint kinase involved in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            various aspects DNA damage responses. DmChk2 mediates p53-mediated transcription of <br />                                            DNA repair genes (16), required cell cycle checkpoint function cycle 14 of <br />                                            embryogenesis (17) involved apoptosis (16). However, previously demonstrated <br />                                            DmChk2 depleted cells normal cell cycle delay response HU Grp/ <br />                                            Dchk1 DmChk2 required G2/M checkpoint function presence of <br />                                            damaged incompletely replicated DNA (5). <br />  <br />                                            Firstly, investigated downregulation DmChk2 successful the <br />                                            course experiment demonstrated DmChk2 detectable day <br />                                            10 RNAi treatment (Figure 4). Comparable Grp/Dchk1 RNAi treatment, <br />                                            downregulation DmChk2 did change proliferation rate cells cultured <br />                                            control conditions (Figure 3A). Remarkably, HU treatment, proliferation rate <br />                                            DmChk2-downregulated cells comparable HU-treated control cells DmChk2 <br />                                            downregulated cells sensitive HU compared Grp/Dchk1-downregulated cells <br />                                            (Figure 3A). <br />  <br />                                            Finally, investigated proliferative capacity cells Grp/Dchk1 and <br />                                            DmChk2 downregulated. Double downregulation Grp/Dchk1 DmChk2 as <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            efficient downregulation single protein (Figure 5). Downregulation Grp/Dchk1 or <br />                                            DmChk2 downregulation Grp/Dchk1 DmChk2 simultaneously did alter <br />                                            cell proliferation S2 cells normal culturing conditions (Figure 3A). Double <br />                                            downregulation Grp/Dchk1 DmChk2 did result similar decrease cell <br />                                            proliferation HU treatment compared downregulation Grp/Dchk1 (Figure <br />                                            3A). Together, data demonstrate Grp/Dchk1 (or far lesser <br />                                            extent) DmChk2 required maintenance proliferation HU-treated cells and <br />                                            DmChk2 plays multiple roles DNA damage responses (16,17), these <br />                                            DmChk2-dependent responses required S2 cells recover HU treatment. <br />  <br />                                Discussion <br />                                            In addition existing methods, survival assay measure the <br />                                            proliferative capacity cells presence specific proteins downregulation of <br />                                            proteins, control culturing conditions specific treatments. One <br />                                            characteristics assay course experiment cell density is <br />                                            compensated means; 1) RNAi-treated (or mock RNAi-treated) GFP-positive cells are <br />                                            mixed feeder cells 2) course experiment, viable <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            cells condition adjusted 2 days. The adjustment cell density overcomes the <br />                                            complication low cell density hampers growth affects outcome survival <br />                                            experiments. Compensating cell density especially important cells under <br />                                            investigation need cultured following strict density protocols. When cell density not <br />                                            compensated for, theory, possible specific treatment eliminating large <br />                                            proportion viable cells remaining cells unable grow resulting <br />                                            decrease density critical concentration. Their inability proliferate not <br />                                            treatment itself low density. Thus assay cell density is <br />                                            compensated used distinguish possibilities. In <br />                                            contrast, assay described applied investigate proliferation capacity of <br />                                            cells treatments eliminate large proportions cells. <br />  <br />                                            In presented assay percentage GFP-expressing (RNAi-treated) cells the <br />                                            mixture feeder cells measured time. In case percentage GFP expressing <br />  <br />  <br />                                                  Cytometry A. Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                                             Page 7 <br />  <br />  <br />                                          cells increasing time, indicates GFP-expressing cells growth <br />                                          advantage compared feeder cells. Under normal culturing conditions (in absence <br />                                          HU) percentage GFP-expressing cells increases time, likely reflecting a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          slightly shorter doubling time GFP expressing cells. This phenomenon observed in <br />                                          control GFP-expressing cells, mock RNAi-treated, Grp/Dchk1- DmChk2- <br />                                          downregulated cells. This phenomenon observed HU treatment control GFP- <br />                                          expressing cells, mock-RNAi treated GFP-expressing cells DmChk2-downregulated <br />                                          GFP-expressing cells, demonstrating growth advantage GFP expressing cells is <br />                                          influenced HU presence absence DmChk2. However, after <br />                                          downregulation Grp/Dchk1 presence HU, percentage GFP-expressing <br />                                          cells decreasing time, indicative growth disadvantage Grp/Dchk1- <br />                                          downregulated cells. Possible explanations growth disadvantage increased cell <br />                                          death, induced cell cycle delay cell cycle arrest. The observed increased sensitivity of <br />                                          Grp/Dchk1-downregulated cells exclusive phenomenon GFP-expressing cells <br />                                          Grp/Dchk1-depleted non-GFP-expressing cells sensitive HU <br />                                          compared mock-RNAi treated DmChk2-downregulated cells (Supplementary Figure <br />                                          4). The increased sensitivity Grp/Dchk1-downregulated cells consistent previous <br />                                          data published (5,15,18). However, observation downregulation of <br />                                          DmChk2 does influence proliferation capacity HU remarkable. DmChk2 plays a <br />                                          role various cell cycle checkpoints DNA damage responses. Based this, was <br />                                          prediction DmChk2-downregulated cells hypersensitive DNA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          damaging insults. However, possible role cell checkpoint regulation been <br />                                          demonstrated specific stages embryogenesis (19,20) S2 cells (5). The <br />                                          downregulation DmChk2 did decrease proliferation capacity Grp/ <br />                                          Dchk1-downregulated cells, indicating DmChk2 S2 cells does play role <br />                                          recovering HU-induced impaired DNA integrity. <br />  <br />                                          In summary, presented assay useful tool evaluate proliferation capacities cells <br />                                          suspension provides effective method investigate function proteins <br />                                          (individual simultaneously) involved responses various treatments. Because specific <br />                                          proteins downregulated GFP-expressing cells, cells subsequently be <br />                                          sorted FACS various biochemical assays expression profile studies be <br />                                          performed analyze consequences treatments specific gene products are <br />                                          downregulated. The effect single treatments analyzed survival be <br />                                          investigated long period time, effect repeated treatments investigated <br />                                          role specific genes required survival repeated treatments. When <br />                                          proteins stably downregulated (or up-regulated), role proteins in <br />                                          survival responses analyzed extended period time. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Acknowledgments <br />                                          We thank Geert Mesander Henk Moes assistence FCM analysis. This work supported a <br />                                          UMCG Bernouilli Bursary grant (809003) X.Y, grant National Institute Health (GM66441) to <br />                                          T.T.S VIDI grant Netherlands Organization Scientific Research NWO (917-36-400) to <br />                                          O.C.M.S. <br />  <br />  <br />                                Reference List <br />                                          1. Baroffio A, Dupin E, Le Douarin NM. Clone-forming ability differentiation potential of <br />                                             migratory neural crest cells. Proc Natl Acad Sci USA. 1988; 85:5325&#8211;5329. [PubMed: 2455901] <br />                                          2. Bodycote J, Wolff S. Metabolic breakdown [3H]thymidine inability measure human <br />                                             lymphocyte proliferation incorporation radioactivity. Proc Natl Acad Sci USA. 1986; <br />                                             83:4749&#8211;4753. [PubMed: 3460068] <br />  <br />  <br />  <br />  <br />                                                 Cytometry A. Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                                   Page 8 <br />  <br />  <br />                                        3. Ferrari M, Fornasiero MC, Isetta AM. MTT colorimetric assay testing macrophage cytotoxic <br />                                           activity vitro. J Immunol Methods. 1990; 131:165&#8211;172. [PubMed: 2391427] <br />                                        4. Buttke TM, McCubrey JA, Owen TC. Use aqueous soluble tetrazolium/formazan assay to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           measure viability proliferation lymphokine-dependent cell lines. J Immunol Methods. 1993; <br />                                           157:233&#8211;240. [PubMed: 8423368] <br />                                        5. Vries HI, Uyetake L, Lemstra W, Brunsting JF, Su TT, Kampinga HH, Sibon OC. Grp/DChk1 is <br />                                           required G2-M checkpoint activation Drosophila S2 cells, Dmnk/DChk2 is <br />                                           dispensable. J Cell Sci. 2005; 118:1833&#8211;1842. [PubMed: 15860729] <br />                                        6. Goshima G, Vale RD. The roles microtubule-based motor proteins mitosis: comprehensive <br />                                           RNAi analysis Drosophila S2 cell line. J Cell Biol. 2003; 162:1003&#8211;1016. [PubMed: <br />                                           12975346] <br />                                        7. Dykxhoorn DM, Novina CD, Sharp PA. Killing messenger: short RNAs silence gene <br />                                           expression. Nat Rev Mol Cell Biol. 2003; 4:457&#8211;467. [PubMed: 12778125] <br />                                        8. Clemens JC, Worby CA, Simonson-Leff N, <span id='am-55' about='obo:NCBITaxon_1195117' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_140066 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_35567 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7033 obo:NCBITaxon_7524 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_868270 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33365 obo:NCBITaxon_33342 obo:NCBITaxon_33367 obo:NCBITaxon_50557'><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Muda" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59'  ></span>Muda</span> M, Maehama T, Hemmings BA, Dixon JE. Use <br />                                           double-stranded RNA interference Drosophila cell lines dissect signal transduction <br />                                           pathways. Proc Natl Acad Sci USA. 2000; 97:6499&#8211;6503. [PubMed: 10823906] <br />                                        9. Clark KA, McKearin DM. The Drosophila stonewall gene encodes putative transcription factor <br />                                           essential germ cell development. Development. 1996; 122:937&#8211;950. [PubMed: 8631271] <br />                                        10. Goshima G, Wollman R, Goodwin SS, Zhang N, Scholey JM, Vale RD, Stuurman N. Genes <br />                                             required mitotic spindle assembly Drosophila S2 cells. Science. 2007; 316:417&#8211;421. <br />                                             [PubMed: 17412918] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        11. Lents NH, Baldassare JJ. RNA interference takes flight: new RNAi screen reveals cell cycle <br />                                             regulators Drosophila cells. Trends Endocrinol Metab. 2006; 17:173&#8211;174. [PubMed: 16723254] <br />                                        12. Radyuk SN, Klichko VI, Spinola B, Sohal RS, Orr WC. The peroxiredoxin gene <span id='am-12' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> in <br />                                             Drosophila melanogaster. Free Radic Biol Med. 2001; 31:1090&#8211;1100. [PubMed: 11677042] <br />                                        13. Sibon OC, Stevenson VA, Theurkauf WE. DNA-replication checkpoint control Drosophila <br />                                             midblastula transition. Nature. 1997; 388:93&#8211;97. [PubMed: 9214509] <br />                                        14. Sibon OC, Laurencon A, Hawley R, Theurkauf WE. The Drosophila ATM homologue Mei-41 has <br />                                             essential checkpoint function midblastula transition. Curr Biol. 1999; 9:302&#8211;312. <br />                                             [PubMed: 10209095] <br />                                        15. Radcliffe CM, Silva EA, Campbell SD. A method assaying sensitivity Drosophila <br />                                             replication checkpoint mutants anti-<span id='am-15' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-16' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-19'  ></span>cancer</span> DNA-damaging drugs. Genome. 2002; 45:881&#8211; <br />                                             889. [PubMed: 12416620] <br />                                        16. Brodsky MH, Weinert BT, Tsang G, Rong YS, McGinnis NM, Golic KG, Rio DC, Rubin GM. <br />                                             Drosophila melanogaster MNK/Chk2 p53 regulate multiple DNA repair apoptotic <br />                                             pathways following DNA damage. Mol Cell Biol. 2004; 24:1219&#8211;1231. [PubMed: 14729967] <br />                                        17. Masrouha N, Yang L, Hijal S, Larochelle S, Suter B. The Drosophila chk2 gene loki essential for <br />                                             embryonic DNA double-strand-break checkpoints induced S phase G2. Genetics. 2003; <br />                                             163:973&#8211;982. [PubMed: 12663536] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        18. Jaklevic BR, Su TT. Relative contribution DNA repair, cell cycle checkpoints, cell death to <br />                                             survival DNA damage Drosophila larvae. Curr Biol. 2004; 14:23&#8211;32. [PubMed: <br />                                             14711410] <br />                                        19. Takada S, Kelkar A, Theurkauf WE. Drosophila checkpoint kinase 2 couples centrosome function <br />                                             spindle assembly genomic integrity. Cell. 2003; 113:87&#8211;99. [PubMed: 12679037] <br />                                        20. Xu J, Xin S, Du W. Drosophila Chk2 required DNA damage-mediated cell cycle arrest and <br />                                             apoptosis. FEBS Lett. 2001; 508:394&#8211;398. [PubMed: 11728459] <br />                                        21. Oishi I, Sugiyama S, Otani H, Yamamura H, Nishida Y, Minami Y. (1998) A novel Drosophila <br />                                             nuclear protein serine/threonine kinase expressed germline establishment. Mech <br />                                             Dev. 1998; 71:49&#8211;63. [PubMed: 9507063] <br />  <br />  <br />  <br />  <br />                                               Cytometry A. Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                         Page 9 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Figure 1. Long-term survival assay <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Schematic representation survival assay. <br />  <br />  <br />  <br />  <br />                                              Cytometry A. Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                         Page 10 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Figure 2. Downregulation Grp/Dchk1 RNAi treatment <br />                                        Grp/Dchk1 protein levels downregulated using RNAi protein levels analyzed <br />                                        using polyclonal rabbit antibody specifically binding Grp/DChk1 protein (migrating <br />                                        58kD)(5)(see Supplementary Figure 1). 4 days RNAi treatment Grp/Dchk1 <br />                                        protein levels detection levels. Therefore long-term survival assays, the <br />                                        RNAi treated cells mixed feeder cells HU added 4 days after <br />                                        RNAI treatment. 4 days RNA treatment corresponds 0 days HU <br />                                        treatment. Grp/Dchk1 remain undetectable day 13 RNAi treatment. These data <br />                                        indicate long-term survival assays Grp/Dchk1 efficiently downregulated <br />                                        RNAi treated cells untill 9 days HU treatment. &#947;-tubulin used loading <br />                                        control. &#8722; = presence mock RNAi treatment; + = presence Grp/Dchk1 <br />                                        RNAi constructs. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Cytometry A. Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                         Page 11 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Figure 3. Decreased long-term survival HU treatment Grp/Dchk1 downregulated cells <br />                                        A) Nuclear-luciferase GFP-expressing cells (NucLuc cells) mock transfected or <br />                                        transfected specific Grp/Dchk1 dsRNAi constructs, DmChk2 dsRNAi constructs or <br />                                        Grp/Dchk1 DmChk2 dsRNAi constructs 4 days start the <br />                                        experiment. The NucLuc cells mixed non-GFP-expressing cells 1:1 ratio. <br />                                        The percentage GFP-expressing cells different conditions measured time <br />                                        using FCM analysis (explained Figure 1) depicted y-<span id='am-37' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-38'  ></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="Axis" datatype="xsd:string"></span>axis</span> (day 0 4 days <br />                                        start RNAi treatment, protein levels detection). <br />                                        B) H2B GFP-expressing cells (H2B cells) mock transfected transfected Grp/ <br />                                        Dchk1 dsRNAi constructs Stonewall dsRNAi constructs (as control) 4 days before <br />                                        start experiment. The H2B GFP-expressing cells mixed non-GFP- <br />                                        expressing cells 1:1 ratio. The percentage GFP-expressing cells different <br />                                        conditions measured time using FCM analysis depicted y-axis (day <br />                                        0 4 days start RNAi treatment, protein levels detection). <br />                                        For statistical analysis supplementary file. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Cytometry A. Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                        Page 12 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Figure 4. Downregulation DmChk2 RNAi treatment <br />                                        DmChk2 protein levels downregulated using RNAi protein levels analyzed <br />                                        using polyclonal rabbit anti-Dmnk-L antibody (21) recognizing DmChk2 (54 kD). 4 days <br />                                        RNAi treatment DmChk2 protein levels detection levels. Therefore the <br />                                        long-term survival assays, RNAi treated cells mixed feeder cells HU <br />                                        added 4 days RNAi treatment. 4 days RNA treatment corresponds <br />                                        0 days HU treatment. DmChk2 remain undetectable day 10 after <br />                                        RNAi treatment. These data indicate long-term survival assays DmChk2 was <br />                                        efficiently downregulated RNAi treated cells untill 6 days HU treatment. &#947;-tubulin <br />                                        used loading control. &#8722; = presence mock RNAi treatment; + = the <br />                                        presence Grp/Dchk1 RNAi constructs. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Cytometry A. Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                         Page 13 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Figure 5. Downregulation Grp/Dchk1 DmChk2 RNAi treatment <br />                                        DmChk2 protein (54kD) levels Grp/Dchk1 (58kD) protei <br /> </body></html>