<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Towards the prediction of essential genes by integration of <br /> network topology, cellular localization and biological process <br /> information <br />  <br />  <br />  <br />                  Abstract <br />                  Background: The identification of essential genes is important for the understanding of the <br />                  minimal requirements for cellular life and for practical purposes, such as drug design. However, the <br />                  experimental techniques for essential genes discovery are labor-intensive and time-consuming. <br />                  Considering these experimental constraints, a computational approach capable of accurately <br />                  predicting essential genes would be of great value. We therefore present here a machine learning- <br />                  based computational approach relying on network topological features, cellular localization and <br />                  biological process information for prediction of essential genes. <br />                  Results: We constructed a decision tree-based <span id='am-72' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-73' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75'  ></span><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>meta</span>-classifier and trained it on datasets with <br />                  individual and grouped attributes-network topological features, cellular compartments and <br />                  biological processes-to generate various predictors of essential genes. We showed that the <br />                  predictors with better performances are those generated by datasets with integrated attributes. <br />                  Using the predictor with all attributes, i.e., network topological features, cellular compartments and <br />                  biological processes, we obtained the best predictor of essential genes that was then used to <br />                  classify yeast genes with unknown essentiality status. Finally, we generated decision trees by training <br />                  the J48 algorithm on datasets with all network topological features, cellular localization and <br />                  biological process information to discover cellular rules for essentiality. We found that the number <br />                  of protein physical interactions, the nuclear localization of proteins and the number of regulating <br />                  transcription factors are the most important factors determining gene essentiality. <br />                  Conclusion: We were able to demonstrate that network topological features, cellular localization <br />                  and biological process information are reliable predictors of essential genes. Moreover, by <br />                  constructing decision trees based on these <span id='am-82' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-85'  ></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span>, we could discover cellular rules governing <br />                  essentiality. <br />  <br />  <br />  <br />  <br /> Background                                                                      organism regardless the presence of remaining genes. <br /> Essential genes are those genes required for growth in a                        Therefore, the functions encoded by essential genes are <br /> rich medium, i.e., medium containing all nutrients                              crucial for survival and could be considered as a founda- <br /> required for growth. The deletion of only one of these                          tion of life itself [1,2]. The identification of essential genes <br /> genes is sufficient to confer a lethal phenotype on an                          is important not only for the understanding of the mini- <br />  <br />                                                                                                                                        Page 1 of 18 <br />                                                                                                                  (page number not for citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                            http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  <br /> mal requirements for cellular life, but also for practical      bility of essential genes. We found that the integration of <br /> purposes. For example, since most antibiotics target essen-     network topology, cellular localization and biological <br /> tial cellular processes, essential gene products of microbial   process information in a single predictor increased the <br /> cells are promising new targets for such drugs [3]. The pre-    predictability of essential genes in comparison with the <br /> diction and discovery of essential genes have been per-         predictor containing only network topological features. <br /> formed by experimental procedures such as single gene           Moreover, we observed that the predictability of essential <br /> knockouts [4], RNA interference [5] and conditional             genes by integration of cellular localization and biological <br /> knockouts [6], but these techniques require a large invest-     process data in a single predictor was comparable to that <br /> ment of time and resources and they are not always feasi-       of predictor containing network topological features. <br /> ble. Considering these experimental constraints, a <br /> computational approach capable of accurately predict            Finally, in addition to study the predictability of essential <br /> essential genes would be of great value.                        genes, we tried to define some general rules governing <br />                                                                 essentiality in S. cerevisiae by analyzing decision trees gen- <br /> For prediction of essential genes, some investigators have      erated by a machine learning-based technique. Using net- <br /> implemented computational approaches in which most              work topology, cellular localization and biological <br /> are based on sequence features of genes and proteins with       process information as training attributes, we discovered <br /> or without homology comparison [7,8]. With the accu-            that essentiality depends on the number of protein physi- <br /> mulation of data derived from experimental small-scale          cal interactions, the nuclear localization of proteins and <br /> studies and high-throughput techniques, however, it is          the number of regulating transcription factors. Taken <br /> now possible to construct networks of gene and proteins         together, all these findings show that the integration of <br /> interaction and then investigate whether the topological        network analysis along with cellular localization and bio- <br /> properties of these networks would be useful for predict-       logical process information is a powerful tool for both <br /> ing essential genes. Although many interaction networks         predicting biological characteristics of genes, such as <br /> have been built to date [9-12], most of studies relating        essentiality, and discovering the biological determinants <br /> essentiality with topological properties of these networks      of phenotypes. <br /> have been limited to indicate what topological properties <br /> are predictive of essentiality instead of using them as pre-    Results and Discussion <br /> dictors of essential genes [9,13]. We have previously           Integrated network of gene interactions in S. cerevisiae <br /> shown the feasibility of using network topological fea-         and calculation of topological features <br /> tures for predicting essential genes in the bacterium           For obtaining the network topological features used as <br /> <span id='am-49' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-56'  ></span><span id='am-57' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span>Escherichia coli</span> [14]. We have chosen E. coli as starting       training data for predicting essential genes, we first con- <br /> point for evaluating the prediction performance of essen-       structed an integrated network of gene interactions (INGI) <br /> tial genes by network topological features due to two rea-      of <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2'  ></span><span id='am-3' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> simultaneously containing <br /> sons: the completeness of the catalog of E. coli essential      experimentally verified protein physical interactions, met- <br /> genes [15] and the vast amount of interaction data availa-      abolic interactions and transcriptional regulation interac- <br /> ble for <span id='am-33' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-34' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>this</span> organism. In this present work, we sought to       tions (definitions for each type of interaction are detailed <br /> evaluate if network topological features can also be used       in &quot;Methods&quot;). This network is comprised by 5,667 genes <br /> as predictors of essential genes in the yeast S. cerevisiae     interacting with one another via 42,893 protein physical <br /> since most of its genes have already been classified as         interactions, 11,192 metabolic interactions and 18,721 <br /> essential or non-essential [4] and there are copious            transcriptional regulation interactions. Of 5,667 genes in <br /> amounts of available interaction data for this organism.        the network, 5,637 (99,5%) are protein-coding genes <br />                                                                 (including transposable elements), 15 (0.26%) are trans- <br /> For this purpose, we constructed a S. cerevisiae integrated     fer RNA-coding genes, 13 (0.23%) are small nucleolar <br /> network of gene interactions containing simultaneously          RNA-coding genes and 2 (0.01%) are RNA-coding genes <br /> protein physical, metabolic and transcriptional regulation      of unknown function. Regarding protein-coding genes, <br /> interactions and used the topological features of this net-     including transposable elements, our network contains <br /> work as learning attributes in a machine learning-based         96% of the total 5,884 protein-coding genes of S. cerevisiae <br /> prediction system. We tested individual and grouped net-        according to the current status of the yeast genome pro- <br /> work topological features as predictors of essential genes      vided by the <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22'  ></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span>Saccharomyces</span> Genome Database (SGD) [16]. <br /> and showed that essential genes are best predicted by inte- <br /> grating the topological features in a single predictor.         We calculated 12 different topological features for each <br /> Although the prediction performance of topological fea-         gene in the INGI, including degree centralities for each <br /> tures was shown to be acceptable, we added to this set of       type of interaction, clustering coefficient, betweenness <br /> learning attributes data on cellular localization and bio-      centralities for each type of interaction, closeness central- <br /> logical process of genes in <span id='am-46' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> to increase the predicta-     ity and identicalness. The detailed description of these <br />  <br />  <br />                                                                                                                     Page 2 of 18 <br />                                                                                               (page number not for citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                              http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  <br /> topological features and how they were calculated are            (ROC) Curve (AUC) values among all the balanced data- <br /> found in the Additional file 1 and &quot;Methods&quot;.                    sets (Figure 1 and Additional file 2), we verified that their <br />                                                                  prediction performances are not statistically different as <br /> Comparison of the classification performance among               evaluated by a nonparametric statistical method based on <br /> balanced datasets                                                the Mann-Whitney U-statistic [18] (see more details in <br /> The performance of machine learning-based approaches             &quot;Methods&quot;). Based on these results, we selected one of the <br /> is known to be affected by imbalanced data [17]. As the          balanced datasets to perform the following analyses. <br /> dataset containing yeast genes classified into essential and <br /> non-essential genes intended to be used as training data         Prediction of essential genes by network topological <br /> for our classifier is an imbalanced dataset, we used an          features <br /> undersampling scheme to generate ten balanced datasets           We started the analyzes by assessing the predictability of <br /> from the original data (see &quot;Methods&quot;). Each balanced            essential genes by each of the 12 network topological fea- <br /> dataset contains different subsets of non-essential genes as     tures (computed as described in &quot; Methods&quot;) and by all 12 <br /> a result of the sampling approach. Due to these different        network topological features integrated in a single predic- <br /> subsets of non-essential genes, therefore, we statistically      <span id='am-77' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-80'  ></span><span id='am-81' property="rdfs:label" content="Tor" datatype="xsd:string"></span>tor</span>. For this purpose, we trained our classifier on a bal- <br /> compared the prediction performance of balanced data-            anced dataset with all network topological features as <br /> sets before assessing the predictability of essential genes      training data and on a dataset containing only one of the <br /> by the different features. We trained our classifier on each     network topological features as training data (see &quot;Meth- <br /> of the balanced dataset with all available training data         ods&quot; for detailed information on construction of the bal- <br /> (network topological features and cellular localization          anced datasets). The ROC plot shown in Figure 2 indicates <br /> and biological process information) and evaluated the            that integration of all networks topological features in a <br /> prediction performance of each balanced dataset. Com-            single predictor outperforms the predictability of essential <br /> paring the Area Under the receiver operating characteristic      genes by the individual network topological features. By <br />  <br />   <br />  <br /> ROC <br /> Figurecurves <br />         1    and AUC values for classifiers trained on the ten balanced datasets with all available learning attributes <br /> ROC curves and AUC values for classifiers trained on the ten balanced datasets with all available learning <br /> attributes. Balanced datasets 1-10: datasets with all available learning attributes prepared by an undersampling scheme as <br /> described in &quot;Methods&quot;. <br />  <br />  <br />  <br /> comparing the AUC values of grouped and individual net-               topological features. We found that the combination of <br /> work topological features, we verified that the AUC value             protein physical interactions-related features with meta- <br /> of grouped network topological features (AUC = 0.773) is              bolic interactions-related features has the same perform- <br /> statistically significantly higher (P &lt; 0.002) than AUC               ance (AUC = 0.765, P = 0.302; see Additional file 2 and <br /> value of any individual network topological feature (Fig-             Figure 2) seen for the predictor containing all grouped <br /> ure 2 and Additional file 2).                                         network topological features (AUC = 0.773). Also, the <br />                                                                       combination of protein physical interactions-related fea- <br /> We then verified if different combinations of grouped net-            tures with clustering coefficient, identicalness and <br /> work topological features could show prediction perform-              betweenness and closeness centralities has the same pre- <br /> ances comparable to that of all grouped network                       diction performance (AUC = 0.763, P = 0.071; see Addi- <br />  <br />                      1.0 <br />  <br />  <br />  <br />  <br />                      0.8 <br />  <br />  <br />  <br />  <br />                                                                                           net (AUC = 0.773) <br />                      0.6 <br /> True Positive Rate <br />  <br />  <br />  <br />  <br />                                                                                           only_ppi_metabolic_features (AUC = 0.765) <br />                                                                                           only_ppi_other_features (AUC = 0.763) <br />                                                                                           only_ppi_reg_features (AUC = 0.757) <br />                                                                                           only ppi inbetppi (AUC = 0.748) <br />                                                                                           ppi (AUC = 0.747) <br />                      0.4                                                                  inbetppi (AUC = 0.701) <br />                                                                                           inbet (AUC = 0.628) <br />                                                                                           c (AUC = 0.615) <br />                                                                                           cent (AUC = 0.559) <br />                                                                                           regin (AUC = 0.547) <br />                                                                                           metin (AUC = 0.518) <br />                      0.2 <br />                                                                                           regout (AUC = 0.514) <br />                                                                                           metout (AUC = 0.513) <br />                                                                                           inbetmet (AUC = 0.512) <br />                                                                                           ident (AUC = 0.508) <br />                                                                                           inbetreg (AUC = 0.502) <br />                      0.0 <br />  <br />                            0.0   0.2                   0.4                    0.6                   0.8                     1.0 <br />                                                              False Positive Rate <br />  <br /> features <br /> ROC <br /> Figure curves <br />           2    and AUC values for the classifiers trained on balanced datasets with individual or grouped network topological <br /> ROC curves and AUC values for the classifiers trained on balanced datasets with individual or grouped net- <br /> work topological features. ROC curves and AUC values of classifiers trained on balanced dataset 9 (see Figure 1) with one <br /> or groups of network topological features as learning attributes as follows: &quot;net&quot;: all network topological features as learning <br /> attributes; &quot;ppi&quot;, &quot;inbetppi&quot;, &quot;inbet&quot;, &quot;c&quot;, &quot;cent&quot;, &quot;regin&quot;, &quot;metin&quot;, &quot;regout&quot;. &quot;metout&quot;, &quot;inbetmet&quot;, &quot;ident&quot; and &quot;inbetreg&quot;: datasets <br /> with only one of the following network topological features as learning attribute: number of protein physical interactions (ppi), <br /> betweenness centrality for the protein physical interactions (inbetppi), betweenness centrality for all types of interactions <br /> (inbet), clustering coefficient (c), closeness centrality (cent, number of regulating transcription factor (regin), number of reac- <br /> tants participating in a metabolic reaction catalyzed by the enzyme encoded by the gene (metin), number of genes regulated by <br /> the transcription factor encoded by the gene (regout), number of products generated in a metabolic reaction catalyzed by the <br /> enzyme encoded by the gene (metout), betweenness centrality for the metabolic interactions (inbetmet), number of genes with <br /> identical topological features (ident) and betweenness centrality for the transcriptional regulation interactions (inbetreg). &quot;only <br /> _ppi_metabolic_features&quot; and &quot;only_ppi_reg_features&quot;: datasets containing protein physical interactions-related features (ppi <br /> and inbetppi) and, respectively, metabolic (met, metin, metout and inbetmet) and transcriptional regulatory interactions-related <br /> features (reg, regin, regout and inbetreg). &quot;only_ppi_other_features&quot;: dataset containing protein physical interactions-related fea- <br /> tures (ppi and inbetppi) and c, ident, cent and inbet. &quot;only_ppi_inbetppi&quot;: dataset containing only the indicated network topologi- <br /> cal features as learning attributes. For more details on network topological features, see Additional file 1. <br />  <br />  <br />  <br />                                                                                                                             Page 4 of 18 <br />                                                                                                       (page number not for citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                            http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  <br /> tional file 2 and Figure 2) observed for all grouped            grating cellular localization and biological process data <br /> network topological features (AUC = 0.773). Therefore,          with network topological data, we assessed the individual <br /> smaller sets of network topological features can be used to     performance of each cellular component and each biolog- <br /> predict essential genes, thus making the calculation of all     ical process, as well as the collective performance of all <br /> topological features dispensable.                               cellular components and all biological processes on pre- <br />                                                                 dicting essential genes, in order to verify if any individual <br /> To verify if the predictive power of all grouped network        feature or grouped features related to cellular localization <br /> topological features could be improved by exclusion of          or biological process are good predictors of essential <br /> topological features with marginal AUC values, i.e., AUC        genes. <br /> values ranging from 0.500 to 0.600, we compared the pre- <br /> diction performance of all grouped network topological          Regarding cellular localization, we trained our classifier <br /> features (AUC = 0.773) with those of the combinations of        on balanced datasets with all cellular compartments as <br /> features in which one feature or a small set of features was    training data (cytoplasm, endoplasmic reticulum, mito- <br /> excluded (see the correspondent ROC curves in the Addi-         chondrion, nucleus or other localization) and on datasets <br /> tional file 3 and the pairwise comparison of predictors         containing only one of the cellular compartments as train- <br /> with the p-values of AUC differences between each pair of       ing data. We can observe in the ROC plot shown in Figure <br /> predictors in Additional file 2). We discovered that the        3 that the best predictor of essential genes seems to be the <br /> prediction performance of all grouped network topologi-         integrated set of cellular compartments. This is confirmed <br /> cal features is not improved by the removal of any individ-     by the statistical comparison of the AUC value of the inte- <br /> ual or small sets of topological features (see Additional       grated set of cellular compartments with those of individ- <br /> files 2 and 3). As expected, the exclusion of grouped fea-      ual cellular compartments: the AUC value of grouped <br /> tures related to metabolic interactions or grouped features     cellular compartments (AUC = 0.703) is significantly (P &lt; <br /> related to protein physical interactions diminishes (AUC        10-5) higher than AUC values of any individual cellular <br /> = 0.764; P = 0.002 and for metabolic interaction-related        compartment (Figure 3 and Additional file 2), although <br /> features and AUC = 0.749; P = 0.001 for protein physical        such AUC value characterizes the set of all cellular compo- <br /> interaction-related features) the prediction performance        nents as fair predictors of essential gene prediction. With <br /> of all grouped network topological features (AUC =              regard to biological processes, we trained our classifier on <br /> 0.773).                                                         balanced datasets with all biological processes as training <br />                                                                 data (cell cycle, metabolic process, signal transduction, <br /> Among all individual network topological features, the          transcription, transport or other process) and on datasets <br /> number of protein physical interactions is that one that        containing only one of the biological processes as training <br /> best predicts essential genes (AUC = 0.747). As further dis-    data. The ROC curves for biological processes (Figure 4) <br /> cussed in &quot;Cellular rules for essentiality&quot;, other investiga-   show the same behavior observed for the prediction of <br /> tors have shown that the number of physical interactions        essential genes by both network topological features and <br /> is indicative of essentiality [9,19,20]. To our knowledge,      cellular compartment: the integration of attributes in a <br /> we are the first to compare the number of protein physical      single predictor increases the predictability of essential <br /> interactions with other network topological features.           genes in comparison with predictability by individual <br /> Despite the good performance of number of protein phys-         attributes. The AUC value of the integrated set of biologi- <br /> ical interactions on predicting essential genes among           cal processes (AUC = 0.667) is statistically significantly (P <br /> other individual network topological features, the best         &lt; 0.001) higher than AUC values of any individual biolog- <br /> predictors are those integrating other groups of topologi-      ical process (Figure 4 and Additional file 2). With the AUC <br /> cal features with the number of protein physical interac-       value of 0.667, however, the set of biological processes <br /> tions. This indicates that essentiality depends more or less    can be considered a poor predictor of essential genes. <br /> on each network topological feature and, therefore, the <br /> gene location in the network seems to be important for          The moderate and poor performances of cellular localiza- <br /> determining its essentiality.                                   tion and biological processes as predictors of essential <br />                                                                 genes, respectively, suggest that essentiality, as further dis- <br /> Prediction of essential genes by cellular localization and      cussed in &quot;Cellular rules for gene essentiality&quot;, is probably <br /> biological process data                                         a result of multiple factors, reinforcing what we found by <br /> Although the prediction performance of the integrated           analyzing the prediction performance of network topo- <br /> network topological features in a single predictor can be       logical features. Therefore, we decided to evaluate the pre- <br /> considered acceptable for predicting essential genes, we        diction performance of the integration of cellular <br /> decided to check if the addition of information on cellular     localization and biological process information in a single <br /> localization and biological process as training data would      predictor. We then trained our classifier on balanced data- <br /> increase the predictability of essential genes. Before inte-    sets with all cellular compartments and biological proc- <br />  <br />  <br />  <br /> Figure <br /> ROC   curves <br />          3     and AUC values for the classifiers trained on balanced datasets with individual or grouped cellular compartments <br /> ROC curves and AUC values for the classifiers trained on balanced datasets with individual or grouped cellular <br /> compartments. ROC curves and AUC values of classifiers trained on balanced dataset 9 (see Figure 1) with one or all cellu- <br /> lar compartments as learning attributes. &quot;all compartments&quot; is the dataset with all cellular compartments as learning attributes; <br /> &quot;nucleus&quot;, &quot;cytoplasm&quot;, &quot;mitochondrion&quot;, &quot;other compartment&quot; and &quot;endoplasmic reticulum&quot; are datasets with only the respec- <br /> tive cellular compartment as learning attribute. <br />  <br />  <br />  <br /> esses as training data. Figure 5 indicates that the               features integrated in a single predictor with that by all cel- <br /> performance of integration of cellular localization and           lular compartments and all biological processes inte- <br /> biological process data on predicting essential genes is          grated in a single predictor. It is worth to mention that <br /> better than other predictors. In fact, the AUC value of pre-      although we choose the predictor containing all network <br /> dictor containing all cellular localization and biological        topological features to perform the following compari- <br /> processes data (AUC = 0.753) is statistically higher (P &lt;         sons, the sets containing protein physical interactions- <br /> 10-5) than AUC values of other predictors (see Additional         related features with metabolic interactions-related fea- <br /> file 2).                                                          tures or other features (see &quot;Prediction of essential genes <br />                                                                   by network topological features&quot; for details) also could be <br /> Prediction of essential genes by integrating network              used since their prediction performances are comparable <br /> topological features, cellular localization and biological        to that of all grouped network topological features. <br /> process information <br /> After determining the predictive power of individual and          For evaluating the integration of all data in a single predic- <br /> grouped cellular localization and biological process data,        tor and comparing it with the predictor containing only <br /> we sought to verify if integration of network topological         cellular localization and biological process information <br /> features with cellular localization and biological process        and with the predictor containing only network topologi- <br /> data in a single predictor would improve predictability of        cal features, we trained our classifier on balanced datasets <br /> essential genes. Moreover, we also sought to compare the          with all available data as training data, all cellular com- <br /> predictability of essential genes by all network topological      partments and biological processes as training data and all <br />  <br /> ROC <br /> Figure curves <br />          4    and AUC values for the classifiers trained on balanced datasets with individual or grouped biological processes <br /> ROC curves and AUC values for the classifiers trained on balanced datasets with individual or grouped biolog- <br /> ical processes. ROC curves and AUC values of classifiers trained on balanced dataset 9 (see Figure 1) with one or all biolog- <br /> ical processes as learning attributes. &quot;all processes&quot; is the dataset with all biological processes as learning attributes; &quot;metabolic <br /> process&quot;, &quot;other process&quot;, &quot;cell cycle&quot;, &quot;transcription&quot; and &quot;transport&quot; are datasets with only the respective biological process <br /> as learning attribute. <br />  <br />  <br />  <br /> network topological features, cellular components and                Regarding the comparison of the predictive power of inte- <br /> biological processes as training data. As expected, the              grated topological network features with that of integrated <br /> ROC curves in Figure 6 indicate that integration of all net-         cellular localization and biological process data, we <br /> work topological features with cellular compartments and             observed that the difference between the AUC value of <br /> biological processes information in a single predictor               predictor containing all cellular compartments and bio- <br /> increases the predictability of essential genes in compari-          logical processes information (AUC = 0.753) and the AUC <br /> son with predictors containing only network topological              value of predictor containing all network topological fea- <br /> features or cellular compartments and biological proc-               tures (AUC = 0.773) is not statistically significant (P = <br /> esses information. Indeed, comparing the AUC value of                0.269) (see Additional file 2). Considering that the func- <br /> predictor containing all network topological features and            tion of a protein is intimately linked to its cellular locali- <br /> all cellular compartments and biological processes infor-            zation [21] and that both the biological process in which <br /> mation with that of predictor containing only network                a protein is involved and the cellular localization in which <br /> topological features or cellular compartments and biolog-            a protein acts are predictable by network topological fea- <br /> ical processes information, we confirmed that predictabil-           tures [10,22], it is not surprising that the predictabilities of <br /> ity of essential genes by the integrated predictor (AUC =            essential genes by both the predictor containing all net- <br /> 0.808) is statistically significantly (P &lt; 10-4) higher than         work topological features and the predictor containing all <br /> that by others predictors (Figure 6 and Additional file 2).          cellular localization and biological process data are simi- <br />                                                                      lar. <br />  <br />  <br /> Figure <br /> ROC curves <br />          5    and AUC values for the integrated predictors with cellular localization and biological process information <br /> ROC curves and AUC values for the integrated predictors with cellular localization and biological process <br /> information. ROC curves and AUC values of classifiers trained on balanced dataset 9 (see Figure 1) with all biological proc- <br /> esses (&quot;all processes&quot;), all cellular compartments (&quot;all compartments&quot;) or all biological processes and cellular compartments <br /> (&quot;all processes and compartments&quot;) as learning attributes. <br />  <br />  <br /> Classification of yeast genes not known to be essential          To predict a gene as essential, we defined an essentiality <br /> We obtained the list of genes classified as essential and        score of 0.654 as the cutoff value, i.e., genes with essenti- <br /> non-essential used for training our classifier from Giaever      ality score above 0.654 were considered to be essential. <br /> et al. [4] (see &quot;Methods&quot;). Giaever et al. have systemati-       This cutoff value was based on the optimal threshold, <br /> cally constructed a nearly complete collection of yeast          which is the score value that leads to the maximal accu- <br /> gene-deletion mutants covering about 96% of all genes.           racy of classification, calculated by the software StAR [24] <br /> However, about 430 genes of this collection were                 for the predictor containing all features (network topolog- <br /> removed from the yeast genome after a comprehensive              ical, cellular component and biological process; see Figure <br /> reannotation process of the S. cerevisiae genome per-            6 and Additional file 2). Among the 514 genes with the <br /> formed in 2006 [23]. In addition, new genes were anno-           essentiality status not defined by Giaever et al., 44 genes <br /> tated to yeast genome as a result of this reannotation           were predicted as essential (Table 1). Analyzing these <br /> process. In order to classify these genes not analyzed by        genes, we found that 9 genes have been previously dem- <br /> Giaever et al., we used our best classifier, that is, the one    onstrated to be essential (YHR165C, YHR089C, <br /> that containing all network topological features, cellular       YHR052W, YCR042C, YDR320C-A, YHR169W, YKL138C- <br /> components and biological processes information as               A, YGL106W and YHR099W) and other 14 genes <br /> training attributes. For each gene, the predictor output the     (YGR252W, YHR027C, YOL012C, YNL147W, YGL100W, <br /> probability of classifying it as essential and non-essential,    YNL096C, YOL148C, YFL007W, YOL145C, YBR111W-A, <br /> which we called, respectively, &quot;essentiality score&quot; and          YNL055C, YHR216W, YBL071W-A and YHR039C-A) have <br /> &quot;non-essentiality score&quot;.                                        been previously demonstrated to be non-essential by <br />  <br />  <br />  <br /> ROC <br /> Figure curves <br />           6 and AUC values for the integrated predictors with available data <br /> ROC curves and AUC values for the integrated predictors with available data. ROC curves and AUC values of clas- <br /> sifiers trained on balanced dataset 9 (see Figure 1) with all network topological features, cellular compartments and biological <br /> processes (&quot;all data&quot;), all biological processes and cellular compartments (&quot;all processes and compartments&quot;) or all network <br /> topological features (&quot;network&quot;) as learning attributes. <br />  <br />  <br /> other investigators through small-scale gene deletion             hypothesize that our classifier assigned a high essentiality <br /> experiments in functional characterization studies [25-36]        score to these genes due to the following features: (i) their <br /> (Table 1). Among non-essential genes, 10 genes                    encoded proteins interact with more than 12 other pro- <br /> (YGR252W, YHR027C, YOL012C, YNL147W, YNL096C,                     teins, (ii) they are regulated by less than 4 transcription <br /> YOL148C, YOL145C, YBR111W-A, YNL055C and                          factors and (iii) their encoded proteins are located in the <br /> YHR039C-A) have been shown to impair substantially the            nucleus. These characteristics are in accordance with two <br /> growth of S. cerevisiae when they are completely deleted          cellular rules for essentiality discovered by our approach <br /> [33,36-40], whereas the 4 remaining non-essential genes           as demonstrated in the section &quot;Cellular rules for gene <br /> (YGL100W, YFL007W, YHR216W and YBL071W-A) have                    essentiality&quot;: if proteins interact with more than 7 other <br /> been shown not to affect the growth phenotype of yeast            proteins and are located in the nucleus, genes encoding <br /> when they are deleted [34,35,41,42]. Although roughly 1/          them are likely to be essential and genes regulated by <br /> 3 of the these genes predicted to be essential have been          more than 3 transcription factors tend to be non-essential. <br /> previously classified as non-essential, the complete dele- <br /> tions of most of them have been shown to severely reduce          Among the 44 genes predicted to be essential, 21 genes <br /> the fitness of organisms [33,36-40], suggesting that our          have not yet been investigated for essentiality to date <br /> predictor, even when directly contradicted by these exper-        (Table 1). One of these genes is the YER029C whose <br /> imental findings, can nonetheless identify genes impor-           encoded protein (Yer029cp) binds to other 6 proteins to <br /> tant to cellular function. Regarding the 4 non-essential          form the heteroheptameric complex that is required for <br /> genes whose deletion has been shown not to affect the             the biogenesis of the spliceosomal U1, U2, U4, and U5 <br /> growth phenotype of yeast (YFL007W and YGL100W), we               snRNPs [43]. These spliceosomal snRNPs are involved in <br />  <br />  <br />  <br />  <br /> Table 1: List of the 44 yeast genes predicted to be essential in S. cerevisiae <br />  <br />  Rank      Gene                Essentiality Score            Essentiality Status         Deletion phenotype                Reference <br />  <br />  1         YHR165C             0.940                         essential                   lethality                         [32] <br />  2         YGR252W             0.939                         non-essential               defective growth                  [33] <br />  3         YHR089C             0.937                         essential                   lethality                         [25] <br />  4         YHR052W             0.065                         essential                   lethality                         [26] <br />  5         YER029C             0.930                         not defined                 not defined                       - <br />  6         YHR027C             0.930                         non-essential               defective growth                  [37] <br />  7         YHR099W             0.929                         essential                   lethality                         [27] <br />  8         YOL012C             0.925                         non-essential               defective growth                  [33] <br />  9         YHR169W             0.921                         essential                   lethality                         [28] <br />  10        YCR042C             0.920                         essential                   lethality                         [29] <br />  11        YDR320C-A           0.897                         essential                   lethality                         [30] <br />  12        YNL147W             0.885                         non-essential               defective growth                  [33] <br />  13        YGL100W             0.866                         non-essential               not related to growth             [41] <br />  14        YNL096C             0.865                         non-essential               defective growth                  [33] <br />  15        YOL148C             0.859                         non-essential               defective growth                  [38] <br />  16        YOR145C             0.856                         essential                   lethality                         [31] <br />  17        YFL007W             0.839                         non-essential               not related to growth             [42] <br />  18        YKL138C-A           0.837                         essential                   lethality                         [30] <br />  19        YOL145C             0.824                         non-essential               defective growth                  [39] <br />  20        YBR111W-A           0.822                         non-essential               defective growth                  [40] <br />  21        YLL022C             0.816                         not defined                 not defined                       - <br />  22        YNL209W             0.816                         not defined                 not defined                       - <br />  23        YGL106W             0.813                         not defined                 not defined                       - <br />  24        YPR080W             0.813                         not defined                 not defined                       - <br />  25        YER105C             0.794                         not defined                 not defined <br />  26        YNL055C             0.783                         non-essential               defective growth                  [33] <br />  27        YOL142W             0.781                         not defined                 not defined                       - <br />  28        YAL024C             0.770                         not defined                 not defined <br />  29        YHR216W             0.768                         non-essential               defective growth                  [34] <br />  30        YHL004W             0.743                         not defined                 not defined                       - <br />  31        YHR072W-A           0.741                         not defined                 not defined                       - <br />  32        YGL190C             0.738                         not defined                 not defined                       - <br />  33        YDR079C-A           0.731                         not defined                 not defined                       - <br />  34        YNL186W             0.731                         not defined                 not defined                       - <br />  35        YJR132W             0.716                         not defined                 not defined                       - <br />  36        YDR261W-A           0.713                         non-essential               defective growth                  [33] <br />  37        YHR119W             0.696                         not defined                 not defined                       - <br />  38        YBL071W-A           0.693                         non-essential               defective growth                  [35] <br />  39        YDR261W-B           0.682                         non-essential               defective growth                  [34] <br />  40        YHR039C-A           0.680                         non-essential               defective growth                  [36] <br />  41        YHR090C             0.680                         not defined                 not defined                       - <br />  42        YER026C             0.675                         not defined                 not defined                       - <br />  43        YHR056C             0.665                         not defined                 not defined                       - <br />  44        YCL019W             0.659                         not defined                 not defined                       - <br />  <br />  <br />  <br /> splicing of nuclear pre-mRNAs [44], an essential biologi-                strated to be non-essential genes and only 5 have been <br /> cal process for cell viability, and, interestingly, all proteins         demonstrated to be essential genes. Thus, about 4% of <br /> forming the heteroheptameric complex along with                          genes with known essentiality status and predicted as <br /> Yer029cp have been demonstrated to be essential [4].                     non-essential are actually essential genes (Additional file <br /> Therefore, the presence of this gene among ones predicted                4). Providing that 38% (9 of 14; see Table 2) of the genes <br /> to be essential reinforces the fact that our predictor is able           with known essentiality status and predicted as essential <br /> to identify genes that are important to cellular function.               are actually essential genes, the predictor integrating all <br />                                                                          available features (network topological, cellular compo- <br /> Finally, regarding the remaining 470 genes predicted as                  nent and biological process; see Figure 6 and Additional <br /> non-essential, we verified that 129 of these genes have                  file 2) leads to an enrichment of actual essential genes in <br /> been previously tested for essentiality by other studies (see            the set of genes predicted as essential. This suggests that <br /> Additional file 4). Among them, 124 have been demon- <br />  <br />                                                                                                                              Page 10 of 18 <br />                                                                                                         (page number not for citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                             http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  <br /> this predictor is committed to minimize the false negative       vious studies that have demonstrated that the number of <br /> rate thus avoiding the loss of essential genes.                  protein physical interactions is indicative of essentiality <br />                                                                  [9,19,20]. Several hypotheses about the connection <br /> Cellular rules for gene essentiality                             between gene essentiality and number of protein physical <br /> Beyond the prediction capability, machine learning tech-         interactions have been proposed. Coulomb et al. [47] <br /> niques can be used for knowledge acquisition in order to         have suggested that the relationship between this network <br /> describe patterns in datasets. The machine learning algo-        feature and gene essentiality is partly due to biases in the <br /> rithms most used for knowledge acquisition are those that        interaction data that are enriched in small-scale experi- <br /> generate decision trees. Decision trees are decision sup-        ments which are partial towards essential genes. On the <br /> port tools inferred from the training data that use a graph      other hand, Zotenko et al. [48] have recently hypothesized <br /> of conditions and their possible consequences. The struc-        that the connection between gene essentiality and <br /> ture of a decision tree consists of a <span id='am-38' about='obo:NCBITaxon_1' typeof='owl:Thing'><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="all" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="root" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>root</span> node representing     number of protein physical interactions is likely due to <br /> the most important condition for discriminating classes,         the involvement of proteins encoded by essential genes in <br /> internal nodes representing additional conditions for            subnetworks of densely connected proteins with shared <br /> <span id='am-30' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> discrimination under the main condition, and leaf          biological functions that are enriched in proteins encoded <br /> nodes representing the final classification. So, one can         by essential genes. <br /> learn the conditions for classifying instances in a given <br /> class by following the path from the root node to the leaf       Following the path from root node to first leaf node <br /> node [45].                                                       through the right branch (Figure 7), we found the follow- <br />                                                                  ing rule for gene essentiality: if proteins interact with more <br /> Therefore, in order to discover the rules for gene essential-    than 7 other proteins (average of number of interactions <br /> ity in S. cerevisiae, we analyzed decision trees generated by    ranging from 6 to 12 in all decision trees) and are located <br /> training the J48 algorithm, a WEKA&apos;s implementation of           in the nucleus, genes encoding them are likely to be essen- <br /> the C4.5 algorithm [46] (for more details, see &quot;Methods&quot;),       tial. This rule can be observed in 9 of 10 decision trees <br /> on the ten balanced datasets containing all network topo-        with 128 instances per leaf and 8 of 10 decision trees with <br /> logical features, cellular components and biological proc-       64 instances per leaf (see Additional file 5). If these pro- <br /> esses as training data (the construction of balanced             teins are located in cellular compartments other than the <br /> datasets are detailed in &quot;Methods&quot;). As decision trees gen-      nucleus, essentiality of their corresponding genes depends <br /> erated from the balanced datasets could be slightly differ-      on conditions particular to each decision tree (Figure 7 <br /> ent from one another due to the undersampling scheme             and Additional file 5). The path from root node to the leaf <br /> used to balance the original set of classified genes--each       nodes through the left branch (Figure 7) drove us to dis- <br /> balanced dataset contains a different set of 1,024 non-          cover another rule for gene essentiality: if proteins interact <br /> essential genes, 1/8 of the total amount in the original         with 6 or fewer proteins and participate in a metabolic <br /> imbalanced dataset--we generated one detailed (64                process inside the nucleus, genes encoding these proteins <br /> instances per leaf) and one simplified (128 instances per        are likely to be essential. This rule can be observed in 7 of <br /> leaf) decision tree for each balanced dataset (see &quot;Meth-        10 decision trees with both 128 and 64 instances per leaf <br /> ods&quot; for details) and then we manually inspected them in         (Additional file 5). <br /> order to discover the general rules for gene essentiality. <br />                                                                  According to these rules, the ultimate condition for gene <br /> From the 20 slightly different generated decision trees, we      essentiality is the localization of proteins in the nucleus, <br /> were able to devise the general rules for gene essentiality      suggesting that this cellular component is somehow <br /> in S. cerevisiae. Figure 7 shows the decision tree that best     important for essentiality. The importance of nucleus for <br /> illustrates the general rules for gene essentiality (all deci-   essentiality has also been suggested by Seringhaus et al. [7] <br /> sion trees are available in text format in the Additional file   that have shown that nuclear localization has the strong- <br /> 5). As we can observe in Figure 7, the root node of deci-        est positive correlation with essentiality among other cel- <br /> sion tree is the number of protein physical interactions         lular components. The relationship between nucleus and <br /> (all generated decision trees exhibit this feature; see Addi-    essentiality can be explained by the fact that roughly one <br /> tional file 5); so, this attribute can be considered the most    third of nuclear proteins are encoded by essential genes <br /> important feature among all network topological features         and most of essential biological processes for cell viability <br /> and cellular localization and biological process informa-        take place within the nucleus [49]. Therefore, the partici- <br /> tion for gene essentiality. Accordingly, the predictor con-      pation of proteins in these nuclear-localized essential <br /> taining only the number of protein physical interaction as       processes, such as DNA replication, transcription and <br /> training feature is the one that best predicts (AUC =            DNA repair, should be a pivotal condition for essentiality <br /> 0.747) essential genes among all other individual features       in the rules defined by both the paths via the left and right <br /> as we can observe in Figure 2. This is in concert with pre-      branches of decision tree. It is worth to mention that, as a <br />  <br />  <br />                                                                                                                     Page 11 of 18 <br />                                                                                                (page number not for citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                                http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  <br />  <br /> Figure <br /> Decision7tree generated by training the J48 algorithm on the balanced dataset 8 with all available data <br /> Decision tree generated by training the J48 algorithm on the balanced dataset 8 with all available data. This <br /> decision tree was generated by training the J48 algorithm on the balanced dataset 8 with all available data (see &quot;Methods&quot;). The <br /> uppermost ellipse is the node root of tree that represents the most important condition for discriminating essential genes <br /> from non-essential genes. In this case, such condition is the number of protein physical interactions (ppi). The remaining ellipses <br /> are internal nodes that represent additional conditions for considering a gene as essential or non-essential. In the left branch of <br /> tree, such conditions are involvement in a metabolic process (met. proc.) and nuclear localization (nucleus). In the right branch, <br /> such conditions are nuclear localization (nucleus) and number of regulating transcription factors (regin). The rectangles are the <br /> leaf nodes that represent the final classification. Red and green rectangles depict genes that, under certain conditions (repre- <br /> sented by the root node and internal nodes), are respectively and predominantly classified as essential (E) and non-essential <br /> (N). In the round brackets inside rectangles, the number before the slash indicates the total number of genes that are actually <br /> essential or non-essential and the number after the slash indicates how many genes were incorrectly predicted. <br />  <br />  <br /> result of the annotation method we used (see more details           number of instances per leaf is set to 64 (see &quot;Methods&quot; <br /> in &quot;Methods&quot;), these nuclear-localized essential processes          for details and Additional file 5). Our finding is corrobo- <br /> are embedded in the biological process &quot;metabolic proc-             rated by Yu et al. [50] that have found that genes regulated <br /> ess&quot;, one of the conditions for essentiality along with             by &gt; 10 transcription factors are less likely to be essential <br /> nuclear localization and number of protein physical inter-          than those regulated by 2-9 transcription factors, whereas <br /> actions equal or less than 6 in the rule defined by the path        these genes are less likely to be essential than those with <br /> via the left branch of decision tree (Figure 7). In the rule        only one transcription factor. At first glimpse, the fact that <br /> defined by the path via the right branch, although essen-           essential genes tend to be regulated by a few transcription <br /> tiality is apparently not dependent on the involvement of           factors seems contradictory since one would expect that <br /> proteins in metabolic processes inside the nucleus, the             gene essentiality is correlated with a high level of tran- <br /> nuclear proteins encoded by genes classified as essential           scriptional regulation. However, most essential genes <br /> according to this rule may be actually involved in a                encode housekeeping proteins, i.e., proteins involved in <br /> nuclear metabolic process. In this case, however, the               housekeeping functions, such as rRNA metabolic process <br /> involvement in nuclear metabolic processes is over-                 and transcription initiation [48]. As housekeeping func- <br /> whelmed by the number of protein physical interactions.             tions are the most basic and important functions within <br />                                                                     cell, genes encoding housekeeping proteins are ubiqui- <br /> We discovered an additional interesting rule for gene               tously expressed and, consequently, they tend to be regu- <br /> essentiality in yeast: genes regulated by more than 3 tran-         lated by fewer transcription factors than genes encoding <br /> scription factors tend to be non-essential (Figure 7). This         non-housekeeping proteins. Therefore, this phenomenon <br /> rule can be observed in 6 of 10 decision trees with 128             is likely due to the enrichment of genes encoding house- <br /> instances per leaf and in all decision trees when the               keeping proteins in the set of essential genes. <br />  <br />  <br />  <br />  <br />                                                                                                                         Page 12 of 18 <br />                                                                                                    (page number not for citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                              http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  <br />  <br /> Conclusion                                                       Although these findings have previously been demon- <br /> The identification of essential genes has largely been an        strated by other investigators [7,9,19,20,50], it is interest- <br /> experimental effort mostly performed by time-consuming           ing to notice that we were able to obtain these same results <br /> whole-genome knockout experiments. In an effort to               by simply inspecting the decision tree generated as shown <br /> accelerate the pace of discovery of essential genes, we          in section &quot;Cellular rules for gene essentiality&quot;. So, deci- <br /> designed a machine learning-based computational                  sion trees are useful tools for extracting knowledge from <br /> approach that relies on network topological features, cel-       complex biological data. <br /> lular localization and biological process information for <br /> predicting essential genes and evaluated it in the yeast Sac-    Besides confirming previous findings, the exploration of <br /> charomyces cerevisiae.                                           decision trees can also lead to new discoveries. This can be <br />                                                                  exemplified by an additional analysis that we performed <br /> We therefore constructed an integrated network of gene           due to a referee&apos; s suggestion regarding the nuclear locali- <br /> interactions for S. cerevisiae containing protein physical,      zation of essential proteins. The referee has suggested us <br /> metabolic and transcriptional regulation interactions and        to analyze the influence of some children terms of GO <br /> computed 12 different network topological features (as           term &quot;nucleus&quot; on the nuclear localization-related gene <br /> described in Additional file 1 and &quot;Methods&quot;) that were          essentiality. For this purpose, we generated a decision tree <br /> individually and collectively evaluated for their ability to     by training the J48 algorithm on one of the ten balanced <br /> predict essential genes. We showed that the predictors           datasets (see &quot;Methods&quot; for details) with all features plus <br /> containing all 12 network topological features or different      the GO terms &quot;nucleolus&quot;, &quot;nucleoplasm&quot;, &quot;nuclear chro- <br /> combinations of protein physical interactions-related fea-       mosome&quot; and &quot;nuclear envelope&quot; and, as can be observed <br /> tures with other groups of topological features as training      in the Additional file 5, an entirely new rule can be <br /> data are reliable predictors (AUC = 0.763-0.773) of essen-       devised from the generated decision tree: the nucleolar <br /> tial genes in S. cerevisiae, thus reinforcing the fact that an   localization of proteins is the most important factor for <br /> integrated network of gene interactions can be an useful         gene essentiality. We did not mention this potential and <br /> tool for the prediction of essential genes.                      interesting rule for gene essentiality in the section &quot;Cellu- <br />                                                                  lar rules for gene essentiality&quot; since this rule per se is inter- <br /> Although the performance of predictors containing only           esting enough to deserve a more exhaustive analysis that <br /> network topological features can be considered acceptable        can be reported in a future paper. <br /> for predicting essential genes, we decided to check if the <br /> addition of cellular localization and biological process         Albeit the good prediction performance and the ability to <br /> information to these predictors would increase the pre-          discover cellular rules for essentiality, our approach suf- <br /> dictability of essential genes. In fact, we verified that the    fers from two limitations. First, it depends on existing <br /> performance of the predictor containing all network top-         Gene Ontology annotation and protein physical interac- <br /> ological features, cellular localization and biological proc-    tion data which are likely to be enriched in small-scale <br /> ess information as training data is better than those of the     experiments involving essential genes. Second, the con- <br /> predictors containing only network topological features          struction of an integrated network of gene interactions <br /> or only cellular localization and biological process infor-      requires a large amount of experimental interaction data <br /> mation. Interestingly, we also showed that the prediction        that are currently available only to a limited number of <br /> performances of the predictor containing only network            organisms. <br /> topological predictions and the predictor containing only <br /> cellular localization and biological process information         Therefore, the prediction of essential genes in newly <br /> are similar. To our knowledge, this is the first time that       sequenced organisms, for example, is impractical by our <br /> Gene Ontology terms related to cellular localization and         approach. However, the integration of our approach with <br /> biological process are shown to be useful predictors of          (i) computational-based methods for gene annotation <br /> essential genes.                                                 and (ii) computational-based methods for the construc- <br />                                                                  tion of integrated networks of predicted gene interactions <br /> In addition to prediction of essential genes, we could also      in which each type of interaction (protein physical, meta- <br /> devise some cellular rules for gene essentiality using all       bolic and transcriptional regulation interactions) can be <br /> network topological features, cellular localization and          distinguished from one another could give rise to a purely <br /> biological process information as training data for gener-       in silico network topology, cellular localization and bio- <br /> ation of decision trees (see details in section &quot;Cellular        logical process information-based methodology for pre- <br /> rules for gene essentiality&quot;). We discovered that the            diction of essential genes. Such a methodology would be <br /> number of protein physical interactions, the nuclear local-      totally independent on experimental interaction data and, <br /> ization and the number of regulating transcription factors       accordingly, unbiased in essential genes-driven experi- <br /> are important factors determining gene essentiality.             ments. <br />  <br />  <br />                                                                                                                      Page 13 of 18 <br />                                                                                                 (page number not for citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                               http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  <br /> In summary, despite the limitations discussed above, we          connected metabolites (ADP, ATP, H+, H2O, NADP+, <br /> could demonstrate that the integration of network topo-          NADPH, orthophosphate and pyrophosphate) in the <br /> logical features, cellular localization and biological proc-     original metabolic model iND750. <br /> ess information is capable to predict essential genes. In <br /> this work, we tested the predictive performance of this          The final integrated network of gene interactions (INGI) <br /> integration in S. cerevisiae, but we envisage that it might be   of yeast is the result of integration of the protein physical, <br /> useful to predict essential genes in any other organism if a     metabolic and transcriptional regulation interactions <br /> purely computational-based prediction approach, as sug-          datasets through genes common to these datasets. Before <br /> gested above, is used.                                           performing the integration, we converted all yeast gene <br />                                                                  names to their systematic names--as provided by the Sac- <br /> Methods                                                          charomyces Genome Database (SGD) Nomenclature <br /> Generation of the set of training features                       Conventions [23]--to avoid the creation of false interac- <br /> Network topological features                                     tions due to gene name ambiguity. Genes classified as <br /> In order to compute the network topological features used        dubious, i.e., genes unlikely to encode an expressed pro- <br /> as training features for predicting essential genes, we first    tein and not considered biologically significant by SGD, <br /> constructed an integrated network of gene interactions of        were removed from the final INGI. <br /> S. cerevisiae based on assumption that two genes, g1 and g2, <br /> coding respectively for proteins p1 and p2, are interacting      For each gene g in the yeast INGI, we computed twelve <br /> genes if (i) p1 and p2 interact physically (protein physical     network topological features as listed in Additional file 1. <br /> interaction), (ii) the transcription factor p1 directly regu-    Briefly, degree centrality is defined as the number of links <br /> <span id='am-25' about='obo:NCBITaxon_8186' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_1489905 obo:NCBITaxon_123366 obo:NCBITaxon_1489904 obo:NCBITaxon_1489872 obo:NCBITaxon_8184 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Lates" datatype="xsd:string"></span>lates</span> the transcription of gene g2, i.e., p1 binds to the pro-   to node (in our case, gene). We considered each type of <br /> moter region of g2 (transcriptional regulation interaction),     interaction as a distinct measure of degree as described in <br /> or (iii) the enzymes p1 and p2 share metabolites, i.e., a        Additional file 1. Clustering coefficient (c) of a node (in <br /> product generated by a reaction catalyzed by enzyme p1 is        our case, a gene) quantifies how close the node and its <br /> used as reactant by a reaction catalyzed by enzyme p2            neighbors are to being a clique, i.e., all nodes connected <br /> (metabolic interaction).                                         to all nodes. For yeast INGI, c is defined as the proportion <br />                                                                  of links between the genes within the neighborhood of g <br /> Yeast protein physical interactions data were obtained           divided by the number of links that could possibly exist <br /> from The Biological General Repository for Interaction           between them. Betweenness centrality reflects the role <br /> Datasets (BioGRID) database, a repository of literature-         played by a node (in our case, a gene) in the global net- <br /> curated protein physical and genetic interactions [51]. We       work architecture and, for the yeast INGI, is defined as the <br /> downloaded the database release 2.0.42 of July 2008 and          fraction of shortest paths between gi and gj passing <br /> removed the entries related to genetic interactions. Yeast       through g. We computed the betweenness centrality based <br /> transcriptional regulation interactions were obtained            on shortest paths via all types of interaction (inbet) as well <br /> from the Yeast Search for Transcriptional Regulators And         as based on shortest paths via each type of interaction <br /> Consensus Tracking (YEASTRACT) database, a curated               (inbetppi, inbetmet and inbetreg). Closeness centrality (cent) <br /> repository of regulatory associations between transcrip-         measures how close a node (in our case, a gene) is to all <br /> tion factors and target genes in Saccharomyces cerevisiae        others in the network and, for the yeast INGI, is defined as <br /> [52]. By using the utility &quot;Generate Matrix Regulation&quot; in       the mean shortest path between g and all other genes <br /> the YEASTRACT website, we generated and downloaded a             reachable from it. Identicalness is the number of genes <br /> regulation matrix containing only documented transcrip-          with identical network topological characteristics. All <br /> tional regulation interactions determined by direct exper-       these network topological features, except for the <br /> imental evidence.                                                betweenness centrality-related features, were calculated by <br />                                                                  a program written in a Mathematica&#174; 6.0 notebook. The <br /> Yeast metabolic interactions were extracted from the met-        betweenness centrality-related features were calculated by <br /> abolic model iND750 of Saccharomyces cerevisiae [11] by a        the <span id='am-87' about='obo:NCBITaxon_37579' typeof='owl:Thing obo:NCBITaxon_1329950 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8504 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8509 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_1329912 obo:NCBITaxon_33208 obo:NCBITaxon_1329961 obo:NCBITaxon_1329911 obo:NCBITaxon_7711 obo:NCBITaxon_8570 obo:NCBITaxon_34979 obo:NCBITaxon_7776 obo:NCBITaxon_34984 obo:NCBITaxon_32561 obo:NCBITaxon_8457'><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasAlternativeId" content="NCBITaxon:51884" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91' property="rdfs:label" content="Python" datatype="xsd:string"></span><span id='am-92'  ></span>Python</span> package NetworkX [54]. <br /> code implemented in Mathematica&#174; 6.0 (Wolfram <br /> Research, Inc.). We excluded those metabolic interactions        Cellular localization and biological process annotation of yeast genes <br /> generated by the so-called &quot;currency metabolites&quot;, abun-         We determined the cellular component in which a yeast <br /> dant molecular <span id='am-43' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-44' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> present throughout the cell most          gene product acts and the biological process in which a <br /> of the time and, therefore, unlikely to impose any con-          yeast gene is involved by using the Saccharomyces <br /> straints on the dynamics of metabolic reactions. Due to          Genome Database (SGD) Gene Ontology (GO) Slim <br /> this feature of currency metabolites, the functionality of       Mapper [55]. The SGD GO-Slim Mapper maps annota- <br /> the network would be better represented without them             tions of a group of genes to more general GO terms. <br /> [53]. We considered currency metabolites the eight most          Among GO Slim sets available at SGD, we selected cellular <br />  <br />  <br />                                                                                                                         Page 14 of 18 <br />                                                                                                    (page number not for citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                              http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  <br /> component and biological process terms from the Super             tion of essential genes. Among classifiers that we evalu- <br /> GO-Slim set, a collection of high-level GO terms. For cel-        ated, the one that provided the best performance was an <br /> lular localization annotation, genes annotated to terms           ensemble of eight decision tree algorithms using the <br /> rather than &quot;cytoplasm&quot;, &quot;endoplasmic reticulum&quot;,                 meta-classifier &quot;Vote&quot;, a WEKA&apos;s implementation of the <br /> &quot;mitochondrion&quot; and &quot;nucleus&quot; were reannotated to one             voting algorithm that combines the output predictions of <br /> of these terms or to a new term named &quot;other localiza-            each classifier by different rules [57]. We combined the <br /> tion&quot;. For biological process annotation, genes annotated         classifiers by the average rule, where the output predic- <br /> to terms rather than &quot;cell cycle&quot;, &quot;metabolic process&quot;, &quot;sig-     tions computed by the individual classifiers for each class <br /> nal transduction&quot;, &quot;transcription&quot; and &quot;transport&quot; were           are averaged and this average is used in its decision [57]. <br /> reannotated to one of these terms or to a new term named          The classifiers composing our model were: (1) REPtree <br /> &quot;other process&quot;.                                                  [56], (2) naive bayes tree [58], (3) random tree [56], (4) <br />                                                                   random forest [59], (5) J48, a WEKA&apos;s implementation of <br /> Classifier design, training and evaluation                        the C4.5 decision tree [46], with minimum number of 32 <br /> Construction of datasets for classifier training and evaluation   instances per leaf, (6) best-first decision tree with mini- <br /> We defined &quot;essential genes&quot; as those genes whose dele-           mum number of 32 instances at the terminal nodes [60], <br /> tion leads to an inviable yeast organism cultured on rich         (7) logistic model tree [61] and (8) alternating decision <br /> glucose medium. We obtained the dataset containing the            tree with 25 boost iterations [62]. In addition, we applied <br /> classification of yeast genes in essential or non-essential       the bootstrap aggregating (bagging) approach [63] to each <br /> from Giaever et al. [4]. After downloading the dataset, we        classifier. Parameters values for each classifier are pro- <br /> removed from it genes classified as dubious in SGD and            vided in the Additional file 6. <br /> converted the name of remaining genes to their systematic <br /> names as provided by the SGD Nomenclature Conven-                 Classifier training and evaluation <br /> tions [23].                                                       For each of the 10 balanced datasets, we trained our clas- <br />                                                                   sifier on half of entries and the other half was used to eval- <br /> As this dataset of classified genes is an imbalanced dataset,     uate the classifier performance, totaling 10 runs of <br /> i.e., the number of non-essential genes is much larger than       training and evaluation. For these runs, we generated a <br /> the number of essential genes, and it has been known that         receiver operating characteristic (ROC) curve and calcu- <br /> data imbalance degrades the performance of machine                lated the area under the ROC curve (AUC). The ROC curve <br /> learning algorithms [17], we built balanced datasets from         is a plot of the true positive rate versus false positive rate <br /> the original imbalanced dataset by using an undersam-             and indicates the probability of a true positive prediction <br /> pling scheme as follows: (1) first, we split the dataset of       as a function of the probability of a false positive predic- <br /> classified genes into two subsets: &quot;essential genes set&quot;,         tion for all possible threshold values [64]. AUC is a widely <br /> containing 1,024 essential gene entries, and &quot;non-essen-          used summary measure of the ROC curve and is equiva- <br /> tial genes set&quot;, containing 4,097 non-essential gene              lent to the probability that a randomly chosen negative <br /> entries; (2) second, we selected all entries from the essen-      example (in our case, a non-essential gene) will have a <br /> tial genes set (1,024 entries) and randomly selected 1,024        smaller estimated probability of belonging to the positive <br /> entries from the non-essential genes set; (3) we then cre-        class than a randomly chosen positive example (in our <br /> ated the balanced dataset containing the 2,048 selected           case, an essential gene) [65]. <br /> entries with random distribution of the essential gene and <br /> non-essential gene entries. This procedure was repeated           We used the web server version of the StAR (Statistical <br /> 10 times in order to generate 10 different balanced data-         Analysis of ROC curves) software [24] for calculating the <br /> sets containing different sets of non-essential gene entries.     true and false positive rates and the AUC values and for <br />                                                                   generating the ROC curves. The statistical comparison of <br /> To compare the predictability of essential genes by indi-         AUC values derived from the different datasets was also <br /> vidual training features with that of different groups of         performed by StAR by means of a nonparametric statisti- <br /> training features, we generated, from the balanced data-          cal method based on the Mann-Whitney U-statistic for <br /> sets, different subsets containing different combinations         comparing distributions of values from two samples [18] <br /> of training features as detailed in Additional file 2.            with a significance level (P) of 0.01. <br />  <br /> Classifier design                                                 Determination of rules for gene essentiality <br /> We used WEKA (Waikato Environment for Knowledge                   The determination of rules for gene essentiality was per- <br /> Analysis) software package, a collection of machine learn-        formed by analyzing decision trees generated through the <br /> ing algorithms for data mining tasks [56], for designing,         training of J48 algorithm on balanced datasets containing <br /> training and evaluating the classifiers applied to predic-        all training data. We used two different values of the <br />  <br />  <br />  <br />                                                                                                                      Page 15 of 18 <br />                                                                                                 (page number not for citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                                                   http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  <br /> parameter &quot;number of objects per leaf&quot; of J48 algorithm <br /> for generating two different types of decision trees: 64 for                          Additional file 5 <br /> more detailed trees and 128 for more simplified trees                                 J48 decision trees. This file contains all 10 decision trees generated by <br /> [56]. For each balanced dataset, then, we obtained two                                training the J48 algorithm on the 10 balanced datasets with all available <br />                                                                                       data as learning attributes. Decision trees are represented in text format <br /> decision trees (detailed and simplified) and manually                                 (raw output generated by WEKA). <br /> inspected all the 20 generated decision trees for determin-                           Click here for file <br /> ing the general rules for gene essentiality. The remaining                            [http://www.biomedcentral.com/content/supplementary/1471- <br /> parameters values for producing decision trees by J48                                 2105-10-290-S5.PDF] <br /> algorithm training are provided in the Additional file 6 <br /> and all decision trees are provided in text format in the                             Additional file 6 <br /> Additional file 5.                                                                    Parameters used to train the meta-classifier and J48. File containing <br />                                                                                       all parameters values used to train the meta-classifier for essential gene <br />                                                                                       prediction and all parameters values used to train the J48 algorithm to <br /> Authors&apos; contributions                                                                generate decision trees for discovery of cellular rules for essentiality. <br /> MLA obtained all interaction data, constructed the net-                               Click here for file <br /> work, designed and analyzed the classifier performance,                               [http://www.biomedcentral.com/content/supplementary/1471- <br /> pursued the biological interpretation of results and                                  2105-10-290-S6.PDF] <br /> drafted the manuscript. NL conceived, designed and <br /> directed the project and implemented the program for cal- <br /> culation of network topological features. All authors read <br /> and approved the final manuscript. <br />  <br />  individual or small sets of network topological features. File containing <br />                                                                                 <br />  ROC curves for classifiers trained on datasets whose learning attributes            <br />  were different sets of network topological features in which each set lacks     <br />  one of the topological features or a small group of 2-4 topological features.        <br />  Click here for file                                                                 <br />  [http://www.biomedcentral.com/content/supplementary/1471-                <br />  2105-10-290-S3.PDF]                                                             <br />  Additional file 4                                                                    <br />  List of the 470 yeast genes predicted to be non-essential. Tab-limited               <br />  text file containing the 470 genes classified as non-essential with their            <br />  essentiality scores, actual essentiality statuses and, if applicable, the               <br />  Pubmed references showing their essentiality statuses.                                <br />  Click here for file                                                                    <br />   <br /> </body></html>