<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, OCt. 1987, p. 4637-4645                                                                       Vol. 169, No. 10 <br /> 0021-9193/87/104637-09$02.00/0 <br /> Copyright &#169; 1987, American Society Microbiology <br />  <br />  <br />               dnaA,          Essential Host Gene, TnS Transposition <br />                                     JERRY C. P. YINt AND WILLIAM S. REZNIKOFF* <br />         Department    Biochemistry,   College Agricultural Life Sciences, University Wisconsin-Madison, <br />                                                    Madison, Wisconsin 53706 <br />                                           Received 6 February 1987/Accepted 17 July 1987 <br />  <br />             Mutations dnaA, essential gene <span id='am-32' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span>Escherichia coli</span>, decrease frequency transposition Tn5. An <br />          insertion mutation dnaA gene does affect Tn5 gene expression. Therefore, DnaA protein plays a <br />          role transposition reaction itself type cellular regulation transposition. Analysis <br />          mutation DnaA box, outside end IS50, consistent direct interaction the <br />          protein bases. IS50 transposition, utilizes end containing DnaA box, not <br />          affected dnaA mutations. Overproduction DnaA protein does increase transposition frequencies <br />          wild-type cells, transposase overproduced. <br />  <br />  <br />    Most procaryotic transposable elements encode protein,              required transposition proficiency. The DnaA protein <br /> the transposase, necessary transposition            does affect pattern TnS gene expression. There- <br /> element (27). Transposition requires cellular factors addi-           fore, postulate probably plays direct role the <br /> tion transposase. A complex small, histonelike                 transposition reaction. <br /> proteins shown necessary Mu transposi- <br /> tion vitro (10). <span id='am-19' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-20' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-21'  ></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> complex probably plays mechanistic                          MATERIALS AND METHODS <br /> role Mu transposition (10). Transposition o&amp; IS] TnJO <br /> appears require complex host proteins,                    Media, supplies, general techniques. All bacteria were <br /> integration host factor (IHF), clear             grown    LB broth M9 medium supplemented with <br /> role IHF plays ISJ transposition (14; D. Morisato,                appropriate amino acids vitamins (33). Antibiotic con- <br /> personal communication). IHF consists small pro-                  centrations used follows: kanamycin, 40 jig/ml; <br /> teins homology histone protein Hi (11, 26). IHF              tetracycline, 5 ,ig/ml; gentamicin, 5 ,ug/ml; chloramphenicol, <br /> also involved site-specific integration-excision            10 pLg/ml; ampicillin, 100 ,xg/ml. Antibiotics pur- <br /> bacteriophage k (26).                                                    chased Sigma Chemical Co. All radioactive compounds <br />    TnS compound transposon. It consists insertion            Amersham Corp. Restriction enzymes from <br /> sequences IS5OR IS50L, inverted orientation rela-              New England BioLabs, Inc., Bethesda Research Laborato- <br /> tive another, flanking region contains antibiotic          ries, Inc., Promega Biotec. Si nuclease calf intestinal <br /> resistance genes. The transposase encoded IS50R (35).              alkaline phosphatase Boehringer Mannheim <br /> When TnS transposes, outer ends compound                      Biochemicals. T4 polynucleotide kinase P-L <br /> transposon used. The bases constitute minimal                Biochemicals, Inc. The large fragment DNA polymerase I <br /> functional outer-end sequence defined (22). When               T4 DNA ligase gifts M. Cox, F. Pugh, B. <br /> the insertion sequence IS50 transposes, outer end                Schutte. The recombinant DNA experiments per- <br /> one inner end IS50 used substrates. The                    formed described previously (31). <br /> sequence requirements inner end                       Strains, bacteriophages, <span id='am-94' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span>. All bacterial <br /> examined (38).                                                           strains, bacteriophages, plasmids used described in <br />    Two host proteins implicated TnS transposi-              Table 1. In general, antibiotic selection maintained all <br /> tion. DNA gyrase shown needed                 times bacterial strains containing plasmids insertions <br /> target molecule transposition reaction supercoiled                <span id='am-1' about='obo:NCBITaxon_2387' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-2' property="rdfs:label" content="transposons" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Transposon" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>transposons</span> chromosome. Strains constructed <br /> (20). DNA polymerase I required,                  described Miller (33) noted. <br /> certain experimental conditions (39, 44). It unclear               The parental strain D47.00 (CSH26 [33]). The RNase <br /> of functions DNA polymerase I needed TnS                  H::Tn3 (18) marker introduced P1 transduction, <br /> transposition (44).                                                      selecting ampicillin resistance. The dnaA::TnJO marker <br />    The sequences outer end Tn5 necessary              moved A-mediated transduction described previ- <br /> for transposition (22) contain homology repeated se-                ously (28). Tetracycline-sensitive derivatives selected <br /> quence <span id='am-48' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-50'  ></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span>Escherichia</span> coli origin replication (45). This          technique Bochner et al. (7). <br /> sequence referred DnaA box. Purified DnaA                   The dnaA mutants monitored loss function <br /> protein, product essential host gene (17), binds            combination techniques. (i) Tetracycline-sensitive <br /> this sequence, E. coli origin outer end           derivatives selected shown incapable of <br /> of TnS (13).                                                             maintaining pSC101. (ii) A fraction II crude extract was <br />    In paper investigate requirement DnaA             prepared used measure oriC replication programmed <br /> protein TnS transposition vivo. We                 M13oriC template (13). The dependency DnaA was <br />                                                                          shown addition purified DnaA protein (kindly <br />   * <br />     Corresponding author.                                                provided R. Fuller). (iii) Southern blot analysis was <br />   t Present address: Sino-American Biotechnology Company,                performed described previously (28), the <br /> Luoyang, Henan, People&apos;s Republic China.                              plasmid pRB100 (8) used probe. <br />                                                                   4637 <br />  4638       YIN AND REZNIKOFF                                                                                             J. BACTERIOL. <br />  <br />                                       TABLE 1. Strains, phage, plasmids used study <br />  Strain, phage, plasmid                                      Genotype description                                     Source <br /> Strains <br />   D47.00                     A(pro-lac) <span id='am-14' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-15'  ></span><span id='am-16' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Ara" datatype="xsd:string"></span>ara</span>                                                                              J. Miller <br />   D47.20                     D47.00 RNase H::Tn3                                                                         This study <br />   D47.22                     D47.20 dnaA::850 TnIO                                                                       This study <br />   D47.26                     D47.22 tetracycline-sensitive derivative                                                    This study <br />   D47.30                     D47.20 ilv::Tn5                                                                             This study <br />   D47.32                     D47.30 dnaA::850 TnIO                                                                       This study <br />   D47.33                     D47.32 tetracycline-sensitive derivative                                                    This study <br />   D48.09                     D47.00 dam::Tn9 ilv::Tn5 (X84)                                                              This study <br />   D48.10                     D47.20 dam::Tn9                                                                             This study <br />   D48.15                     D48.10 dnaA::850 TnJO                                                                       This study <br />   NS294                      AlacX74 bglR ilvO rpsL att::P211                                                            A. Wright <br />   NS366                      NS294 tna::TnJO dnaA366                                                                     A. Wright <br />   DC204                      A(pro-lac) leu thy gly drm CycA supE42 Hfr (81&apos;) tna::TnIO                                  This study <br />   DC204 dnaA                 DC204 dnaA::850 TnlO                                                                 This study <br />   12.00                      Hfl-                                                                                        B. Belfort <br />   168.01                     galK (X c1857 ABam AM1 A8) ilv::TnS/pOXgen                                              This study <br />   169.01                     168.01 X c1857 N7 N53 replaces X c1857 ABam AM1 A8                                   This study <br />   B7.21                      D47.00 rpsL rpsE gyrA polA XT                                                               R. Johnson <br />   RZ211/F,lq::TnI5 (A)       D47.00 rpsL recA56/F&apos; pro lacJq::TnS (X b519 b515 cI857 nin5 S7)                            R. Johnson <br /> Phages <br />   X84                        X b519 b515 cI857                                                                           M. Syvanen <br />   X299                       X 121 ninS (no EcoRI sites)                                                                 J. Way <br />   X500                       X b221 cI857 029 P80 rex::TnS                                                               N. Kleckner <br /> ColEl-based plasmids <br />   pRZ102                     ColE1::Tn5 (25)                                                                             R. Jorgenson <br />   pRZ305                     Protein fusion p, P2 ,B-galactosidase                                             R. Johnson <br />   pRZ464                     IS50R outer (1 15) base pairs IS50L flanking tetracycline resistance (22)     R. Johnson <br />   pRZ466                     IS50R outer (1 21) base pairs IS50L flanking tetracycline resistance (22)     R. Johnson <br />   pRZ466-11A                 pRZ466 mutation dnaA box                                                      R. Johnson <br />   pRZ914                     Derivative pRZ102 fully functional contains gene codes          M. Krebs <br />                                gentamicin resistance <br /> pBR322-based plasmids <br />   pBR25                      lac promoter driving functional dnaA gene (8)                                               R. Braun <br />   pRB36                      lac promoter driving nonfunctional dnaA gene (8)                                            R. Braun <br />   pRB100                     Clone functional dnaA gene (8)                                                       R. Braun <br />   pRB101                     Clone nonfunctional dnaA gene (8)                                                      R. Braun <br />   pRZ1095                    Gentamicin-resistant derivative pRB100                                                   This study <br />   pRZ1096                    Gentamicin-resistant derivative pRB101                                                   This study <br />   pBF110                     pL-driven dnaA gene (13)                                                                    R. Fuller <br />   pBF1509                    pL-driven dnaA gene (B. Funnell, personal communication)                                    B. Funnell <br />   pRZ1073                    Protein-defective derivative IS50 outer end inner end flanking            This study <br />                                tetracycline resistance <br />   pRZ1113                    Protein-defective derivative IS50 outer ends flanking tetracycline resistance   This study <br />   pRZ986                     Derivative pBR322::TnS wild-type promoters 1S50R replaced K PR        M. Krebs <br />                                promoter. A defective element carrying gene codes gentamicin resistance <br />                                located <span id='am-68' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-72'  ></span>cis</span> transposase-coding region complemented transposition <br />                                assays. <br /> RSF1010-based plasmids <br />   RSF1010::TnS               RSF1010 TnS transposed                                                         This study <br />   pRZ1054                    Transposase-defective derivative RSF1010::TnS                                            This study <br />  <br />  <br />    Strains dam growth P1 phage strain             Wisconsin, Madison, 1981) cloned unique PstI <br /> which contained Tn9 insertion dam gene.                     site plasmid. pRZ1054 derivative RSF1010:: <br /> Transductants selected chloramphenicol resistance             Tn5 does synthesize transposase an <br /> and screened loss methylase function examining               internal deletion coding region. <br /> digests plasmid DNA MboI.                                         pRZ464 pRZ466 described previously (22). <br />    pRZ102 RSF1O1O::TnS described previ-                  pRZ466-11A derivative pRZ466 contains a <br /> ously (23, 25), pRB100, pRB101, pRB25, pRB36               single T-to-A transversion DnaA box end the <br /> (8). pBF110 pBF1509 kindly provided R. Fuller.             TnS element. It created oligonucleotide-directed <br /> pRZ1095 pRZ1096 derivatives pRB100                      mutagenesis, followed sequencing subcloning a <br /> pRB101, respectively; PstI fragment carrying gentamicin            fragment contained outer-end sequence Tn5 (R. <br /> resistance gene (S. Kagan, Ph.D. thesis, University                 Johnson, personal communication). <br />  VOL. 169, 1987                                                                     dnaA AND Tn5 TRANSPOSITION               4639 <br />  <br />   pRZ1113 derivative pBR322::ISSOR. The inner end        antibiotic-resistant exconjugants subsequent bacterial <br /> of IS50R parent plasmid replaced small         mating experiment. The mating-out transposition assay was <br /> fragment contained outer-end sequences IS50.        performed described previously (23), cells <br /> A tetracycline resistance gene exchanged           grown 34&#176;C mating mixture also <br /> DNA internal IS50. The resulting plasmid carries            incubated 34&#176;C. A 10-RI portion freshly saturated, <br /> defective element comprising outer ends IS50 flanking     washed culture male female strains mixed in <br /> a gene confers tetracycline resistance. pRZ1073         1 ml broth incubated slow shaking 6 12 h. <br /> very similar pRZ1113, original inner-end      The cells diluted plated appropriate media. A <br /> sequence IS5OR left intact. This plasmid carries     pOXgen derivative F factor used. The exconju- <br /> a defective element; case outer end inner      gants selected LB agar plates containing 200 pug of <br /> end flank gene confers tetracycline resistance.          spectinomycin ml 10 ,g gentamicin ml. Cells <br /> pRZ914 deleted derivative pRZ102 (M. Krebs, per-         containing- transposition prodiicts measured LB <br /> sonal communication). It contains functional TnS element       agar plates containing 200 jig spectinomycin ml, 50 ,g <br /> which encodes gentamicin resistance.                             nalidixic acid ml, 5 ,ug gentamicin ml, 40 jig <br />   pRZ986 derivative pBR322::TnS contains         kanamycin ml. Strain B7.21 used female <br /> functional TnS transposase-coding region X        strain matings. <br /> PR promoter control (M. Krebs, personal communication).             The mating mixtures male female cells diluted <br /> This plasmid used cis complementation experi-         plated yield approximately 50 500 colonies per <br /> ment (see Table 5).                                              plate. The variation set assays usually less <br />   Transposition assays. The lambda infection assay <span id='am-24' about='obo:NCBITaxon_1178312' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_572719 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41011 obo:NCBITaxon_41197 obo:NCBITaxon_41012 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-25' property="rdfs:label" content="Mea" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>mea</span>-          twofold. All assays performed triplicate a <br /> sures TnS transposition frequency X500, replica-      given day, numbers quoted represent averages of <br /> tion-defective, integration-defective lambda phage         sets assays. <br /> carries wild-type TnS. After absorption, cells diluted      Si nuclease mapping. RNA isolated strains <br /> LB broth allowed grow 37&#176;C equivalent       D47.20, D47.22, D48.10, D48.15 carrying pRZ102. The <br /> two cell doublings. The mixture plated LB agar       Si mapping procedure, probes, polyacrylamide gel <br /> plates containing kanamycin incubated 37&#176;C 24 h.      analysis identical described elsewhere <br /> The number transposition events measured count-        (J. C. P. Yin, M. P. Krebs, W. S. Reznikoff, J. Mol. <br /> ing number kanamycin-resistant colonies present. The      Biol., press). <br /> number drug-resistant colonies linear num- <br /> ber input phage particles. Transposition frequencies                                  RESULTS <br /> the fraction input plaque-forming particles yield <br /> kanamycin-resistant colonies. The multiplicity infection         dnaA mutations TnS transposition. Figure 1 shows the <br /> was adjusted yield 50 500 kanamycin-              features TnS known play direct role the <br /> resistant colonies plate. The standard variation    transposition process. The transposase (Pi) encoded by <br /> than factor 3, numbers quoted represent         IS50R (35). Twenty-one base pairs outer end IS50L <br /> average assays triplicate.                   ISSOR sufficient transposition Tn5 (22). <br />   The steady-state assay assay X84 infected    Within required 21 base pairs sequence similar <br /> into cells carry transposon, serves        sequence appears times E. coli origin of <br /> target transposition event. Cells grown satu-     replication (45). This sequence ISS0 oriC been <br /> ration LB broth containing antibiotic maltose (0.2%).     shown binding site DnaA protein vitro (12). <br /> Phage lysates mixing fresh plaque             To determine dnaA required transposition <br /> phage (containing ca. 106 phage) freshly saturated cells,   TnS vivo, transposition frequencies measured a <br /> buffer, molten agar. This mixture poured TYE        series E. coli strains mutations dnaA gene. For <br /> plates (33) incubated 8 16 h 37&#176;C. The lysates     dnaA mutants grow, contain extra- <br /> were collected spun, titer supernatant    genic suppressor dnaA mutation (2, 27, 28, 32). Three <br /> determined (yields typically .1010 phage). The trans-       different suppressors dnaA lethality used: the <br /> position frequency performed measuring fraction       rnh::Tn3 mutation R factor phage P2 an <br /> of plaque-forming phage acquired particular         integrative suppressor (2, 27, 28, 32). Table 1 lists the <br /> drug resistance marker transposon. This measure-          relevant strains, mutations, suppressors. Three indepen- <br /> ment performed mixing constant number plaque-        dent assays (see Materials Methods) used to <br /> forming phage (typically 107 108) 5 x 108 I2.00 cells    confirm lack functional DnaA protein appropri- <br /> and plating selective antibiotic-containing plates 60   ate strains. <br /> min growth. The frequency lysogeny                The <span id='am-84' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-88'  ></span>data</span> Table 2 effect mutations the <br /> conditions approached 40%. The number antibiotic-             dnaA gene TnS transposition. An insertion TnWO the <br /> resistant colonies shown linearly related       dnaA structural gene decreased transposition 25-fold when <br /> amount input phage.                                           compared transposition wild-type strain (lines 1 and <br />   The lambda induction assay determines frequency         3). There 13-fold decrease transposition a <br /> transposition Tn5 A prophage measured        RNase H mutant isogenic dnaA+ strain <br /> abundance A::TnS-transducing phage infectious unit        (lines 2 3). RNase H mutants extragenic suppressors <br /> after prophage induction. This assay described          dnaA mutations allow secondary replication origins <br /> previously (23), measurement A::TnS deter-        used dnaA-independent manner (28). The RNase H <br /> mined described above. This assay used      mutation itself negligible effect TnS trans- <br /> experiments test effect overproducing DnaA         position (lines 1 2). <br /> protein.                                                            The following results (Table 2) support conclusion that <br />   The mating-out assay determines frequency trans-        dnaA directly affects Tn5 transposition. First, mutations in <br /> position F factor measured occurrence       dnaA decreased transposition frequency, regardless of <br />  4640        YIN AND REZNIKOFF                                                                                                              J. BACTERIOL. <br />  <br />        S&apos; CTGACTdRTGTaC <br />  <br />                                                                                                                  .~~~~~~~~~~~~~~~~~~~~~~~~~~~S <br />                                 IS500                                   I <br />                                                                                                                             IS50R <br />                                         aTC     ACTATTCTCTGT <br />                                                                                                                                              pi <br />  <br />                                                        OUTER END        5&apos;   CTGACTCTATACAkGTAG <br />                                                        INNER END        S&apos;   CTGtCTCTT&amp;LtCA&amp;AtcTtG <br />                                                        oriC REPEATS              #1 TTATCCACA <br />                                                                                  #2   TTATGACACA <br />                                                                                  #3 TTATCC~<span id='am-78' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-79' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81'  ></span><span id='am-82' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span>AA</span> <br />                                                                                  #4 TTATCCACA <br />   FIG. 1. TnS sequence. The corresponding outer-end inner-end sequences IS50 shown compared repeat sequences <br /> found oriC region E. coli. The dnaA sequence present outer end boxed, dam methylation sequences inner <br /> end underlined. The transposase protein shown diagramatically. <br />  <br /> the site TnS transposes. Transposition TnS                        greater effect dnaA mutation infection <br /> from multicopy plasmids pRZ102 RSF1010::TnS,                            assay  steady-state assay. In experiments, <br /> which derivatives ColE1 RSF1010, affected                        dnaA mutations qualitative <br /> by mutations dnaA gene (lines 2 3; lines 4 5).                   effect TnS transposition indicated exam- <br /> When Tn5 transposed bacterial chromosome, dnaA                         ined assay, lambda induction assay (data not <br /> also affected transposition frequency (lines 6 8).                       shown). <br />   Second, effect dnaA demonstrated using                         Third, decrease transposition frequency occurred <br /> other transposition assays. The data presented lines 1 8                  suppressors dnaA mutation. The transpo- <br /> were collected using steady-state assay. In assay,                  sition assays repeated integrated R factor (lines <br /> lambda phage, serves target transposition                   11 12) lysogen phage P2 (lines 13 14) to <br /> reaction, grown lytically cells contain TnS. Lines                  suppress dnaA mutation (30, 34). In suppressor <br /> 9 10 results obtained using A donor assay                      backgrounds, TnS transposition decreased presence of <br /> which transposition infecting phage molecule                       mutation dnaA gene. <br /> the bacterial chromosome measured (24). In                             Fourth, different null alleles dnaA similar effects on <br /> steady-state assay (lines 1 8) infection assay (lines                transposition. The strain uses P2 integrative <br /> 9 10), transposition decreased dnaA mutants.                     suppressor contains amber mutation dnaA gene (40) <br /> We know biological significance                     instead TnJO insertion. Since mutation still <br />  <br />                                                    TABLE 2. dnaA mutations transposition <br />    Strain                  Relevant genotype               Suppressor        Starting location                      Transposition                 Relative <br />                                                                                   TnO                              frequency                   frequency <br /> 1. D47.00            Wild type                             None              pRZ102                              (3.9 &#177; 2.0) x 10-7                   1.0 <br /> 2. D47.20            rnh::Tn3                              RNase H           pRZ102                              (2.1 &#177; 1.3) x 10-7                  0.54 <br /> 3. D47.22            rnh::Tn3 dnaA::TnIO                   RNase H           pRZ102                              (1.5 &#177; 0.8) X 10-8                  0.04 <br /> 4. D47.20            rnh::Tn3                                  RNase H                RSF1010;:Tn5               (4.4 &#177; 0.3) x 10-7                  1.0 <br /> 5. D47.22            rnh::Tn3 dnaA::TnIO                       RNase H                RSF1010::Tn5               (1.1 &#177; 0.9) x i0-8                  0.03 <br /> 6. D47.30            rnh::Tn3                                  RNase H                ilv&apos;::TnS                  (1.2 &#177; 0.2) x 10-7                  1.0 <br /> 7. D47.32            rnh::Tn3 dnaA::TnlO                       RNase H                ilv::Tn5                   (1.5 &#177; 0.9) x 10-8                  0.13 <br /> 8. D47.33            rnh::Tn3 dnaA::TnIO Tets                  RNase H                ilv::Tn5                   (1.7 &#177; 1.5) x lo-8                  0.14 <br /> 9. D47.20            rnh::Tn3                                  RNase H                Lambda                     (1.2 &#177; 0.2) x 10-4,                 1.0 <br /> 10. D47.22           rnh::Tn3 dnaA::TnJO                       RNase H                Lambda                     (1.0 &#177; 0.1) X 10-7a               &lt;0.01 <br /> 11. DC204            Hfr (81&apos;) tna::TnIO                       R100.1                 pRZ102                     (8.4 &#177; 3.6) x 10-6                  1.0 <br /> 12. DC204-           Hfr (81&apos;) dnaA::TnJO                      R100.1                 pRZ102                     (9.2 &#177; 5.8) x 10-7                  0.11 <br /> 13. NS294            att::P211                                 P2                     pRZ102                            8.4 x 10-6&quot;                  1.0 <br /> 14. NS366            att::P211 dnaA366(Am)                     P2                     pRZ102                            2.2 x 10-8b                &lt;0.01 <br />  The lambda infection assay used measure transposition. All assays steady-state assay. <br />  b Average duplicate experiments. All values averages experiments. <br />  VOL. 169, 1987                                                                                           dnaA AND Tn5 TRANSPOSITION                        4641 <br />  <br />                TABLE 3. dnaA trans complementation                                  dnaA mutation Tn5 transposition primarily the <br />               Chromo-                      Gene                                     inactivation dnaA gene itself. <br />                                                            Transposition               Direct effect TnS. To effect dnaA <br />  Strain         somal       Plasmid&apos;                        frencyb <br />               genotype                    plasmid              frequency            mutation Tn5 transposition mediated DnaA box <br /> D47.20        dnaA         pRZ1095        dnaA+         (1.5   &#177; 0.5) x 10-6        outer end TnS, effect single point mutation <br /> D47.20        dnaA+        pRZ1096        dnaA          (1.7   &#177; 0.1) x 10-6        dnaA consensus sequence examined. One the <br /> D47.22        dnaA         pRZ1095        dnaA+         (1.1   &#177; 1.0) X 10-6        conserved bases sequence homology (46) mutated <br /> D47.22        dnaA         pRZ1096        dnaA          (1.0   &#177; 0.3) x 10-7        sequence shown Table 4. To assay transposition <br />     All cells carry <span id='am-6' about='obo:NCBITaxon_2504' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384 obo:NCBITaxon_36549'><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="RSF1010" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36501" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Plasmid RSF1010" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="IncQ plasmid RSF1010" datatype="xsd:string"></span>plasmid RSF1010</span>::Tn5, source Tn5         mutant, used compound elements contain a <br /> the transposition assay. Cells carry compatible coresident plasmid           wild-type IS50 sequence outer-end sequences of <br /> which contains cloned copy functional nonfunctional dnaA gene.            IS5OL (with mutation). These sequences flank <br />   bThe steady-state assay used measure transposition.                        selectable marker, tetracycline resistance gene. <br />                                                                                        The effect DnaA box mutation transposition is <br />                                                                                     presented Table 4. In cells containing DnaA protein, <br /> affected TnS transposition (lines 13 14), effect                         point mutation reduced transposition 50-fold (lines 1 and <br /> different dnaA null alleles transposition appears                          2). The residual transposition mutant <br /> similar.                                                                            shows background frequency dele- <br />    From results conclude mutations dnaA                            tion removed bases DnaA box <br /> decrease frequency Tn5 transposition. This effect                         (lines 2 3). In absence DnaA protein, trans- <br /> independent starting location Tn5, transposition                      position wild-type ends decreased eightfold (lines 1 and <br /> assay used, second-site suppressor used permit cell                          4), shown (Table 2). In cells lack <br /> growth, particular null allele dnaA gene.                            dnaA contain mutant DnaA box, transposition <br />    The dnaA mutation does affect Tn5 transposition                              decreased additional sixfold (lines 4 5). Thus, the <br /> indirectly affecting A absorption. This possibility ruled                     DnaA box mutation decreased transposition not <br /> out following evidence. First, dnaA                             DnaA protein present. In presence the <br /> qualitative effect TnS transposition X induction                          mutation DnaA box, additional effect <br /> assay (data shown). This assay measures transposition                           DnaA protein removed mutation (lines 2 <br /> onto induced prophage independent possible                         5). <br /> adsorption problems. Second, Tn3 TnJO transposition                               The effect DnaA box mutation (50-fold), when <br /> showed dnaA sensitivity set assays (data                          present end element, greater the <br /> not shown). Third, phage growth consistently better                          effect removing DnaA protein (8-fold). The simplest <br /> the dnaA mutant background, unlikely result                        interpretation results single-base change <br /> an absorption defect.                                                               DnaA box affects participation different <br />    Control polarity. Since dnaA gene                          components transposition machinery, DnaA pro- <br /> operon (36), inactivation structural gene                            tein additional factor factors. Presumably total <br /> either insertion mutation amber mutation result                        effect mutation DnaA box product the <br /> in polarity expression distal gene dnaN.                        effect removing DnaA protein effect the <br /> Therefore, determined effect dnaA                             additional factor factors. <br /> mutation Tn5 transposition inactivation                          dnaA IS50 transposition. Since DnaA box appears <br /> the dnaA gene itself possible polar effect distal                        outer-end sequences ISSO (Fig. 1), reasonable <br /> gene expression.                                                                    question transposition IS50 (which uses one <br />    Transposition experiments performed wild-type                          outer end inner end) requires DnaA protein. <br /> or dnaA mutant cells contained plasmid clones                         Experiments performed defective elements with <br /> a functional defective dnaA gene (8). Besides func-                          outer ends outer end inner end as <br /> tional defective dnaA gene, plasmids contained                            substrates complemented trans transposase <br /> other chromosomal genes. A cloned copy wild-type                             donor. <br /> dnaA gene reversed effect TnJO insertion                              Table 5, lines 1 3, shows eightfold <br /> chromosomal gene, cloned copy inactive dnaA                           decrease efficiency complementation element <br /> gene did (Table 3). This result shows effect                    outer ends transposition measured a <br />  <br />  <br />                                                         TABLE 4. Effect dnaA site mutation&quot; <br />  Strain               Relevant                Plasmid                          Element description                      Transposition                  Relative <br />                       genotype                                                                                           frequency                    frequency <br /> D47.20                dnaA+                pRZ466                     Wild type                                     (2.3 &#177; 1.7) x 10-7                    1.0 <br /> D47.20                dnaA+                pRZ466-11A                 Point mutation                                (4.0 &#177; 2.0) x 10-9                    0.02 <br /> D47.20                dnaA+                pRZ464                     Deletion dnaA box                      &apos;1.0 x 10-9                     &lt;0.01 <br /> D47.26                dnaA                 pRZ466                     Wild type                                     (2.8 &#177; 1.6) x 10-8                    0.12 <br /> D47.26                dnaA                 pRZ466-11A                 Point mutation                                (4.0 &#177; 2.0) x 10-9                    0.02 <br /> D47.26                dnaA   pRZ464                                   Deletion dnaA box                      &apos;1.0   x 10-9                   &lt;0.01 <br />   &apos;Wild type 5&apos;-CTGACTCTTATACACAAGTAG <br />     466-11A       CTGACTCTTAAACACAAGTAG <br />                                                It I X <br />     464          CTGACTCTTATACACCGGGAT <br />     The steady-state assay used measure transposition. &lt; denotes frequency transposition event transposition assay plate. <br />  4642        YIN AND REZNIKOFF                                                                                                                   J. BACTERIOL. <br />  <br />                                                                            TABLE 5. dnaA end usage <br />    Strain                     Relevant <br />                               genotype                          Transposase <br />                                                                donor plasmid&quot;               Target in <br />                                                                                         complementationb               Transposition <br />                                                                                                                         frequencyc                    Relative <br />                                                                                                                                                      frequency <br /> 1. D47.20                      dnaA+                           RSF1O01::Tn5          pRZ1113 (outer-outer)         (4.2 &#177; 0.8) x 10-8                    1.0 <br /> 2. D47.20                      dnaA+                           pRZ1O54               pRZ1113 (outer-outer)             &lt;4.7 x 10-9                     &lt;0.1 <br /> 3. D47.26                      dnaA                            RSFiOlO::Tn5          pRZ1i13 (outer-outer)         (5.4 &#177; 4.6) x 10-9                    0.13 <br /> 4. D47.26                      dnaA                            pRZ1054               pRZi113 (outer-outer)             &lt;2.0 x 10-9                     &lt;0.1 <br /> 5. D47.26                      dnaA+                           RSFi010::Tn5          pRZ1073 (outer-inner)         (3.4 &#177; 2.4) x 10-8                    1.0 <br /> 6. D47.20                      dnaA+                           pRZ1054               pRZ1073 (outer-inner)             s2.0 x 10-9                     &lt;0.1 <br /> 7. D47.26                      dnaA                            RSF101O:TnS           pRZ1073 (outer-inner)         (2.0 &#177; 2.0) x 10-8                    0.59 <br /> 8. D47.26                      dnaA                            pRZ1054               pRZ1073 (outer-inner)            s2.0 x 10-9                      &lt;0.1 <br />   aRSF1010::TnS encodes functional transposase, pRZ1054 does not. <br />   b The second, coresident plasmid contains element cdmolemented Trans. pkZl13 contains element outer ends flanking gene which <br /> codes tetracycline resistance. pRZ1073 contains element outer end inner end flanking gene codes tetracycline resistance. <br />   c The steady-state assay used measure transposition. s denotes frequency transposition event transposition assay plate. <br />  <br />  <br /> dnaA strain. However, element outer end                                   shown previously Pi translated Ti <br /> and inner end examined, significant                                   transcript, P2 translated T2 transcript (29). <br /> decrease transposition frequency dnaA+                                   Since DnaA box approximately 13 base pairs away <br /> and dnaA strain (lines 5 7).                                                        putative -35 region Ti promoter about <br />    dnaA Tn5 gene expression. Genetic evidence suggests                                45 base pairs away -35 region T2 promoter <br /> a possible direct interaction DnaA protein                                (29), DnaA protein affect transcription either <br /> RNA polymerase (2, 3, 40). We examined                                      promoter. <br /> possible role interaction transcription                                 Si mapping experiments RNA isolated either <br /> Tri genes. Since transposition frequency TnS                                   wild-type dnaA mutant cells showed difference the <br /> been shown function concentration                              pattern protection two- major transcripts Tn5. <br /> transposase, Pi (Yin et al., press), inhibitor, P2 (19,                        Figure 2 autoradiogram gel Si analysis. <br /> 24; submitted publication), DnaA protein                             The levels Ti T2 transcripts did appreciably <br /> stimulate expression Pi depress expression P2. It                            change mutant cells (compare lanes b). When the <br />                                                                                           dam methylation mutation added strains, there <br />                                                                                           effect dnaA mutation Ti T2 <br />                                                                                           transcripts (lanes c d), showed char- <br />                      &apos;j          b c d                                                    acteristic response tnethylation mutation (e.g., com- <br />                                                                                           pare lanes c lanes b d). <br />                                                                                              Furthermore, DnaA protein required TnS trans- <br />                                                                                           position transposase expressed a <br />                                                                                           heterologous promoter. The dependence transposition on <br />                                                                                           dnaA measured cells transposase mRNA <br />                                                                                           placed control X PR promoter (M. <br />                                                                                           Krebs, personal communication). The requirement this <br />                                                                                           element DnaA protein (Table 6, lines 3 4) was <br />                           *_;,      ........ <br />  <br />  <br />                                          :.....r           b <br />                                                                                           similar requirement wild-type element <br />                                                                                           (lines 1 2). Taken together, experiments show <br />                                                      TRT <br />                                                  ..... <br />                                                                                           dnaA does affect TnS transposition affecting <br />                                                                                           transcriptional initiation element. <br />                                                                                              Effect overproducing DnaA. Since decreasing the <br />                                                                                           DnaA protein decreases transposition, in- <br />                                   ..                                                      creasing the DnaA protein increase the <br />                                                                                           transposition frequency. Table 7 presents transposition data <br />                                                                                           various clones overproduce DnaA protein. <br />                           ........................                                        Two clones place dnaA gene control of <br />                                                                                           X PL promoter (13; R. Fuller, personal communication). <br />                                                                                           Under fully inducing conditions, pBF11O overproduces the <br />                                                                                           DnaA protein 200-fold (13). The transposition experi- <br />                                                                                           ments involved clones performed <br />                                                                                           fully inducing conditions, of <br />   FIG. 2. SI mapping transcripts. An autoradiograph 8%                           overproduction DnaA protein measured. The third <br /> polyacrylamide-7 M urea gel shown. RNAs prepared D47.20                           clone overproduces DnaA protein places dnaA <br /> (RNase H), D47.22 (RNase H dnaA), D48.10 (RNase H dam),                               gene control lactose promoter (8). Transposition <br /> D48.15 (RNase H dnaA dam) cells containing plasmid pRZiO2                                 experiments overproducer strain <br /> correspond lanes d, respectively. Two end-labeled probes <br /> were used Si reactions, hybridized                             deficient lactose repressor. Within range of <br /> j3-lactamase gene (located chromosome)                            overproduction clones exhibit exper- <br /> which detected Tn5-specific mRNA <span id='am-29' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-30' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>. The relevant Tn5                                imental conditions, did appear <span id='am-73' about='obo:NCBITaxon_1113472' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_40096 obo:NCBITaxon_40092 obo:NCBITaxon_33208 obo:NCBITaxon_40093 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-74' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-75'  ></span><span id='am-76' property="rdfs:label" content="Stimula" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>stimula</span>- <br /> transcripts shown right gel. RT denotes                             tion Tn5 transposition. Some experiments pre- <br /> readthrough transcript enters element outside.                             sented Table 7 performed dam mutants (lines 1 <br />  VOL. 169, 1987                                                                                                dnaA AND Tn5 TRANSPOSITION                  4643 <br />  <br />                               TABLE 6. Effect dnaA transposase expressed heteologous promoter <br />  <br />  Strain                   Relevant                    P <br />                                                       Plasmid                       Control                       Transposition                    Relative <br />                           genotype                                                 transposase                         frequency                     frequency <br />                                                                                     synthesis <br /> D47.20                     dnaA+                      pRZ914                       Wild type                     (1.2 &#177; 0.5) x 10-7                    1.0 <br /> D47.22                     dnaA                       pRZ914                       Wild type                     (5.0 + 1.0) x 10-9                    0.04 <br /> D47.20                     dnaA+                      pRZ986                       A PR                          (3.3 &#177; 0.3) x 10-5a                   1.0 <br /> D47.22                     dnaA                       pRZ986                       k PR                          (3.7 &#177; 0.7) x 10-6a                   0.11 <br />   a <br />       Steady-state transpositions   X299   assayed 34&#176;C. All transpositions X84 assayed 37&#176;C. <br />  <br /> to 3), transposase levels increased wild-                             fected mutations dnaA gene (data shown). <br /> type levels (Yin et al., press). Increasing the                        None elements contains DnaA box sequences. <br /> transposase did make level DnaA protein                              The data indicate mutations dnaA gene not <br /> rate limiting transposition.                                                        affect Tn5 transcriptional initiation. There effect the <br />                                                                                         levels major transcripts Ti T2, the <br />                                                                                         readthrough transcription affected. The comple- <br />                               DISCUSSION                                                mentation experiment presented Table 6 consistent <br />                                                                                         interpretation mutations dnaA effect <br />                                                                                         TnS transcriptional initiation. Therefore, requirement <br />    dnaA TnS transposition. The experiments described                             DnaA protein probably level TnS gene <br /> this paper involve use different mutations                            expression. <br /> dnaA gene (28, 40) investigate requirement                                  Role dnaA cell. The DnaA protein known to <br /> DnaA protein TnS transposition. Both TnJO insertion                                functions cell: repression transcriptional <br /> and amber mutation result substantial decrease                               initiation (8, 43; D. Smith, personal communication) a <br /> transposition. The requirement DnaA protein TnS                              role initiation DNA replication (12, 15-17). Both of <br /> transposition absolute, decrease                                functions thought involve binding protein <br /> transposition frequency 10-fold (Table 2).                                DnaA boxes, binding purified protein some <br /> Mutations transposase sequences required                               sites shown (8, 12). Is requirement DnaA <br /> the outer end TnS effects 100-fold (22, 35).                        protein TnS transposition related other <br />    Control experiments effect largely inde-                           functions dnaA known perform? <br /> pendent transposition assay, type suppressor                                 Repression. It clear Si mapping data the <br /> which allows cell growth absence DnaA protein,                            complementation experiment presented Table 6 the <br /> and donor site TnS starts. A cloned copy                              dnaA mutation does affect initiation transcription on <br /> the dnaA gene reverses effect, showing directly                            TnS. The readthrough transcript detected Si <br /> indirectly, requirement DnaA protein itself.                             mapping experiment affected dnaA muta- <br /> Analysis single point mutation consensus DnaA                               tion. Therefore, transcriptional elongation entering ele- <br /> box present outside end TnS consistent                            ment probably affected DnaA protein. Tran- <br /> interpretation DnaA protein interacts site.                          scriptional fusion experiments measure of <br /> This genetic evidence, taken results                             transcription traversing DnaA box direction also <br /> vitro binding experiment (12), suggests dnaA exerts                            dnaA effect (data shown). However, <br /> effect TnS transposition acting                              quality transcript, quantity, be <br /> site. Since transposition IS50 does require DnaA                             important factor TnS transposition. It possible that <br /> protein, protein function interacting                          RNA polymerase-DnaA protein complex formed when <br /> two DnaA box sequences located outer end Tn5.                                RNA polymerase crosses DnaA box com- <br /> An additional piece evidence supports interpre-                            plex important species TnS transposition. There is <br /> tation transposition Tn3, Tn9, TnJO unaf-                             precedence hypothesis transcriptional <br />  <br />  <br />                                                               TABLE 7. Overproduction DnaA <br />         Strain                            Relevant                            Plasmid                         Promoter/                       Transposition <br />                                          geneotype                         overproducer                         gene                            frequency <br /> 1. D48.09                                dam                                 pBF110                       X   pLjdnaA&apos;                    (3.6 &#177; 2.4) x 10-5a <br /> 2. D48.09                                dam                                 pRB25                        lac/dnaA+                       (6.6 &#177; 4.2) x 10-5a <br /> 3. D48.09                                dam                                 pRB36                        lacldnaA                        (2.8 &#177; 1.2) x 10-5a <br /> 4. 168.01                                N+ host                             None                         None                                3.1 x 10-5b <br /> 5. I68.01                                N+ host                             pBF110                       k pL/dnaA&apos;                          4.2 x 1o-5b <br /> 6. I68.01                                N+ host                             pBF1509                      k   pL/dnaA&apos;                    (2.4 &#177; 0.6) x 10-5b <br /> 7. 169.01                                N host                              pBF1509                      X   PLIdnaA&apos;                    (2.6 &#177; 1.8) x 10-5b <br /> 8. RZ211/F&apos;lq (K)                        recA                                pRB322                       None                            (2.0 &#177; 0.1) x 10-6a <br /> 9. RZ211/F&apos;Iq (K)                        recA                                pRB25                        lac/dnaA+                       (6.3 &#177; 2.7) x 10-6a <br /> 10. RZ211/F&apos;lq (K)                       recA                                pRB36                        lacldnaA                        (2.7 &#177; 0.7) x 10-6a <br />   a <br />     The lambda induction assay used measure transposition. <br />   b The mating-out assay used measure transposition. <br />  4644     YIN AND REZNIKOFF                                                                                                 J. BACTERIOL. <br />  <br /> activation model oriC replication (43). We data      involved initiation chromosomal replication (13, <br /> support refute model.                                      17). This initiation event regulated process occurs <br />    Replication. There direct evidence exten-      cell division. However, DnaA protein be- <br /> sive replication needed Tn5 transposition process     lieved synthesized constant rate the <br /> (6, 21). The elegant genetic evidence shows TnJO        cell cycle (37). Therefore form temporal regulation <br /> transposes conservative mechanism (4) does exist          exists controls timing initiation event, <br /> for TnS. The evidence does exist suggests TnS           controlling activity DnaA protein. If <br /> does transpose obligate, cointegrate structure      transposition requires active DnaA protein, transposi- <br /> as intermediate (5, 21). The cointegrate replicative       tion occur limited cell cycle. <br /> intermediate Shapiro model transposition (41). <br />    Interaction ends. It possible role DnaA <br /> protein TnS transposition bind DnaA box                               ACKNOWLEDGMENTS <br /> each outer end assist properly aligning ends          We thank M. Wickens careful reading manuscript, M. <br /> some type nucleoprotein structure. Because require-         Krebs P. Lambert discussions aid, A. Wright, R. <br /> ment DnaA protein TnS transposition              Braun, R. Johnson, T. Kogoma strains. B. Fuller very <br /> absolute, likely transposase itself   helpful initial stages work. <br /> capable bringing outer ends together,        This work supported Public Health Service grant <br /> as efficiently. If model true, IS50 transposition,    GM19670 National Institutes Health. J.C.P.Y. was <br /> which independent dnaA, rely transposase         supported Public Health Service Training Grant GM07133 <br /> itself align outer end inner end proper        National Institutes Health. <br /> structure. <br />   A related possibility DnaA protein facilitates                          LITERATURE CITED <br /> binding and/or function transposase outer end. The <br /> spatial relationship presumed transposase-              1. Abeles, A. L., K. M. Snyder, D. K. Chattoraj. 1984. P1 <br /> binding site, 5&apos;-CTGACTCTT-3&apos;, DnaA box                plasmid replication: replicon structure. J. Mol. Biol. 173: <br />                                                                        307-324. <br /> interesting, actually overlap (Fig. 1). This overlap     2. Atlung, T. 1981. Analysis seven dnaA suppressor loci in <br /> in binding sites similar relationship       Escherichia coli. ICN-UCLA Symp. Mol. Cell. Biol. 22: <br /> presumed transposase-binding site IS] IHF-                  297-314. <br /> binding site (14). It tempting speculate host    3. Bagdasarian, M. M., M. Izakowska, M. Bagdasarian. 1977. <br /> factors performing analogous roles transposi-            Suppression DnaA phenotype mutations rpoB <br /> tion process elements.                                    cistron ribonucleic acid polymerase <span id='am-55' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-56' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-61'  ></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span>Salmonella</span> typhy- <br />    Interaction dam methylation DnaA protein.                murium Escherichia coli. J. Bacteriol. 130:577-582. <br /> There evidence dam methylation           4. Bender, J., N. Kleckner. 1986. Genetic evidence TnlO <br /> DnaA protein probably interact. The DnaA protein represses             transposes non-replicative mechanism. Cell 45:801-815. <br />                                                                     5. Berg, D. E. 1983. Structural requirement IS50-mediated gene <br /> expression dam gene (D. Smith, personal communi-                transposition. Proc. Natl. Acad. Sci. USA 80:792-796. <br /> cation), recently shown         6. Berg, D. E., C. M. Berg. 1983. The prokaryotic transposable <br /> dnaA structural gene promoters activated dam meth-               element TnS. Bio/Technology 1:427-435. <br /> ylation (9). Thus, genes gene products          7. Bochner, B., H.-C. Huang, G. Schieven, B. N. Ames. 1980. <br /> form autoregulatory circuit. There juxtaposition          Positive selection loss tetracycline resistance. J. Bacte- <br /> of DnaA boxes dam methylation sites oriC region             riol. 143:926-933. <br /> (45) ori-r region phage P1 (1). Both        8. Braun, R. E., K. O&apos;Day, A. Wright. 1985. Autoregulation of <br /> origins require dnaA function. The oriC region                 DNA replication gene dnaA <span id='am-89' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-90' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia coli K-12</span>. Cell <br /> requires methylated bases function (32, 42).                        40:159-169. <br />                                                                     9. Braun, R. E., A. Wright. 1986. DNA methylation differen- <br />    Transposition IS50 sensitive dam methylation           tially enhances expression E. coli dnaA <br /> system mechanism increases usage inner               promoters vivo vitro. Mol. Gen. Genet. 202:246- <br /> end substrate undermethylated (Yin et al.,          250. <br /> press). Usage outer ends TnS transposition       10. Craigie, R., D. J. Arndt-Jovin, K. Mizuuchi. 1985. A defined <br /> is independent dam methylation (Yin et al.,           strand-transfer reaction initiation of <br /> press) stimulated dnaA. It possible        bacteriophage Mu transposition. Proc. Natl. Acad. Sci. USA <br /> forms stimulation, undermethylation acting IS50 inner            82:7570-7574. <br /> ends DnaA protein acting TnS outer ends,            11. Flamm, E. L., R. A. Weisberg. 1985. Primary structure of <br /> functionally analogous. For example, transposition                 hip gene Escherichia coli product, subunit <br /> require proper positioning ends element          integration host factor. J. Mol. Biol. 183:117-128. <br />                                                                    12. Fuller, R. S., B. E. Funnell, A. Kornberg. 1984. The dnaA <br /> together transposase. When IS50 transposes,               protein complex E. coli chromosomal replication origin <br /> transposase bound outer end inner end             (oriC) DNA sites. Cell 38:889-900. <br /> bring ends form complex. Undermethyla-        13. Fuller, R. S., A. Kornberg. 1983. Purified dnaA protein <br /> tion inner end stimulate binding               initiation replication Escherichia coli origin replica- <br /> transposase site. When TnS transposes, DnaA                tion. Proc. Natl. Acad. Sci. USA 80:5817-5821. <br /> protein bound outer end bring ends           14. Gamas, P., D. Galas, M. Chandler. 1985. DNA sequence at <br /> together facilitate performance function             end IS] required transposition. Nature (London) <br /> transposase. Thus, usage outer end inner            317:458-460. <br /> end facilitated interacting genes        15. Hansen, E. B., M. B. Yarmolinsky. 1986. Host participation <br />                                                                        plasmid maintenance: dependence dnaA replicons <br /> their products.                                                        derived P1 F. Proc. Natl. Acad. Sci. USA 83:4423- <br />    Celiular regulation TnS transposition. It formally            4427. <br /> possible DnaA protein involved regulating       16. Hasunuma, K., M. Sekiguchi. 1977. Replication plasmid <br /> timing TnS transposition. The DnaA prote <br /> </body></html>