<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                                                   JCB: Report <br />  <br />  <br />  <br />  <br />                               The NuRD chromatin&#8211;remodeling complex regulates <br />                               signaling repair DNA damage <br />                               Godelieve Smeenk,1 Wouter W. Wiegant,1 Hans Vrolijk,2 Aldo P. Solari,3 Albert Pastink,1 Haico van Attikum1 <br />                               1 <br />                               Department Toxicogenetics, 2Department Molecular Cell Biology, 3Department Medical Statistics, Leiden University Medical Center, Leiden 2300RC, Netherlands <br />  <br />  <br />  <br />  <br />                               C <br />                                        ells respond ionizing radiation (IR)&#8211;induced                              catalytic subunit (adenosine triphosphatase [ATPase]) <br />                                        DNA double-strand breaks (DSBs) orches-                                   NuRD, leads accumulation spontaneous DNA <br />                                        trating events coordinate cell cycle pro-                               damage increased IR sensitivity. MTA2 CHD4 <br />                               gression DNA repair. How cells signal repair                                  accumulate DSB-containing chromatin tracks gener- <br /> THE JOURNAL OF CELL BIOLOGY <br />  <br />  <br />  <br />  <br />                               DSBs fully understood. A genome-wide RNA                                   ated <span id='am-91' about='obo:NCBITaxon_62990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_241799 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_241789 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-92'  ></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Laser" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>laser</span> microirradiation. Directly DSBs, CHD4 <br />                               interference screen <span id='am-31' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-32' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span>Caenorhabditis elegans</span> identified                              stimulates RNF8/RNF168-dependent formation ubiq- <br />                               egr-1 factor protects worm cells IR. The                            uitin conjugates facilitate accrual RNF168 and <br />                               human homologue egr-1, MTA2 (metastasis-associated                                 BRCA1. Finally, CHD4 promotes DSB repair <br />                               protein 2), subunit nucleosome-remodeling                                 checkpoint activation response IR. Thus, the <br />                               histone deacetylation (NuRD) chromatin-remodeling                                 NuRD chromatin&#8211;remodeling complex novel regu- <br />                               complex. We knockdown MTA2 CHD4                                      lator DNA damage responses orchestrates proper <br />                               (chromo&#173;domain helicase DNA-binding protein 4),                                   signaling repair DSBs. <br />  <br />  <br />  <br />  <br />                               Introduction <br />                               The efficient accurate repair chromosomal double-strand                        arrive DSBs MDC1, directly binds &#65533;?&#65533;H2AX (Stucki <br />                               breaks (DSBs), arise exposure agents                           et al., 2005). <span id='am-53' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-54'  ></span><span id='am-55' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> allows recruitment E2 ubiq- <br />                               ionizing radiation (IR), critical maintaining genome sta-                    uitin conjugase UBC13 E3 ubiquitin ligase RNF8, <br />                               bility preventing cell death carcinogenesis. To avoid                         binds MDC1 (Huen et al., 2007; Kolas <br />                               deleterious effects DSBs, eukaryotic cells activate signaling                      et al., 2007; Mailand et al., 2007). RNF8/UBC13 promote the <br />                               cascades, called checkpoints, coordinate rapid detection                        ubiquitylation histones H2A/H2AX, leading recruit- <br />                               DNA breaks temporal arrest cell cycle progression                        ment E3 ubiquitin ligase, RNF168, associates <br />                               activation DNA repair mechanisms (Hoeijmakers, 2001;                           RNF8-ubiquitylated histones ubiquitin-interacting <br />                               Khanna Jackson, 2001).                                                            motifs (Huen et al., 2007; Kolas et al., 2007; Mailand et al., <br />                                     The cellular response DSBs predominantly co-                              2007; Doil et al., 2009; Stewart et al., 2009). RNF168 coop- <br />                               ordinated phosphatidylinositol 3-kinase&#8211;like kinases                           erates UBC13 amplify RNF8-mediated histone ubiq- <br />                               ataxia telangiectasia mutated (ATM) ataxia telangiectasia                         uitylation threshold required accumulation of <br />                               Rad3 related (ATR). ATM phosphorylates histone H2AX                               checkpoint repair proteins, including BRCA1, 53BP1, <br />                               (&#65533;?&#65533;H2AX) DSB-flanking chromatin create environment                            RAD18, PTIP, DSB-flanking chromatin compart- <br />                               allows spatiotemporal redistribution accumu-                         ment (Wang Elledge, 2007; Doil et al., 2009; Gong et al., <br />                               lation checkpoint DNA repair factors DNA breaks                             2009; Huang et al., 2009; Stewart et al., 2009). In addition to <br />                               (van Attikum Gasser, 2009). Among proteins                           ATM-driven events, DSB ends undergo extensive resec- <br />                                                                                                                     tion, leading formation replication protein A&#8211;coated <br />                               Correspondence Haico van Attikum: h.van.attikum@lumc.nl                            single-stranded DNA subsequent assembly activation <br />                               Abbreviations used paper: ATM, ataxia telangiectasia mutated; <br />                                                                                                                     &#169; 2010 Smeenk et al.&#8195; This article distributed terms Attribution&#8211; <br />                               ATR, ataxia telangiectasia Rad3 related; DDR, DNA damage response;                Noncommercial&#8211;Share Alike&#8211;No Mirror Sites license months pub- <br />                               DSB, double-strand break; IR, ionizing radiation; IRIF, IR-induced foci; NuRD,        lication date (see http://www.rupress.org/terms). After months available a <br />                               nucleosome remodeling histone deacetylation; TERT, telomerase reverse             Creative Commons License (Attribution&#8211;Noncommercial&#8211;Share Alike 3.0 Unported license, <br />                               transcriptase; WCE, cell extract.                                               described http://creativecommons.org/licenses/by-nc-sa/3.0/). <br />  <br />  <br />  <br />  <br />                               The Rockefeller University Press&#8195;&#8202;$30.00 <br />                               J. Cell Biol. Vol. 190 No. 5&#8195; 741&#8211;749 <br />                               www.jcb.org/cgi/doi/10.1083/jcb.201001048                                                                                                                                    JCB   741 <br />            ATR (Zou Elledge, 2003; Jazayeri et al., 2006; Dubrana        Zhang et al., 1999). A biochemical study indicated MTA2 <br />           et al., 2007). Finally, ATM ATR amplify signals gener-       modulates histone deacetylation activity NuRD (Zhang <br />           ated DSBs phosphorylating regulatory proteins,         et al., 1999). The chromatin-remodeling activity complex <br />           including SMC1, CHK1, CHK2, p53, p21, coordinate            resides subunit, CHD4 (chromodomain helicase <br />           cell cycle progression induce apoptosis (Khanna Jackson,      DNA-binding protein 4), identified derma- <br />           2001; Shiloh, 2003).                                                 tomyositis-specific autoantigen (Seelig et al., 1995). CHD4 a <br />                  The cross talk histone modifications (e.g., <span id='am-106' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-107' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-109'  ></span><span id='am-110' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>phos</span>-     member SNF2 <span id='am-23' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> ATPases possesses intrinsic <br />           phorylation ubiquitylation) DSB-flanking chromatin con-       ATP-dependent nucleosome-remodeling activity (Wang and <br />           trols ATM/ATR-dependent signaling repair DSBs,         Zhang, 2001). It thought NuRD represses transcription <br />           unclear achieved (van Attikum Gasser, 2009).      regulating chromatin structure (Denslow Wade, 2007). <br />           One possible mechanism ATP-dependent chromatin                Moreover, recent study showed loss NuRD <br />           remodeling performed ATPases SNF2 <span id='am-88' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-89' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="superfamily" datatype="xsd:string"></span>superfamily</span>              components results chromatin defects associated <br />           (Clapier Cairns, 2009). We previously dem-       DNA damage accumulation aging (Pegoraro et al., <br />           onstrated budding yeast chromatin-remodeling         2009). However, NuRD preserves genome stability and <br />           complexes (INO80, SWR1, SWI/SNF, RSC) recruited              regulates DDR remained unclear. <br />           DSBs, change chromatin structure distinct                  To investigate this, transfected U2OS cells siRNAs <br />           ways regulate cell cycle progression and/or DSB repair            luciferase CHD4 counted cells 2, 3, 4 d after <br />           (Morrison et al., 2004; van Attikum et al., 2004, 2007; Chai         siRNA treatment. CHD4 knockdown cells proliferated much <br />           et al., 2005; Shim et al., 2005, 2007; Tsukuda et al., 2005;         slower control cells (Fig. 1, A B). Flow cytometric analy&#173; <br />           Papamichos-Chronakis et al., 2006; Liang et al., 2007). In con-      sis cells did significant changes cell <br />           trast budding yeast, human cells, role chromatin        cycle distribution. However, morphological changes marked <br />           remodeling cellular response DSBs just begin-       sub-G1 peaks, indicative apoptosis, observed 2&#8211;4 d after <br />           ning emerge. The TIP60 SWI/SNF chromatin-remodeling           siRNA transfection (Fig. 1, B C). Consistently, levels <br />           complexes, example, implicated DNA repair           p53, phosphorylated p53 (S15p), p53 effector p21, <br />           promoting histone acetylation H2AX phosphorylation            coordinate cell cycle progression apoptosis, sig- <br />           DSBs, respectively (Murr et al., 2006; Park et al., 2006; Ikura   nificantly increased absence CHD4 (Fig. 1 D), is <br />           et al., 2007), ALC1 recently shown assist        agreement earlier study implicating role NuRD <br />           poly(ADP-ribose) polymerase&#8211;dependent chromatin remod-            apoptosis p53/p21 regulation (Luo et al., 2000). We in- <br />           eling sites DNA damage (Ahel et al., 2009; Gottschalk          vestigated apoptosis induced loss CHD4 be <br />           et al., 2009). However, unclear interplay      related spontaneous occurrence DNA lesions. Indeed, <br />           chromatin remodeling histone modifications DSBs co&#173;           CHD4 knockdown cells showed increased levels &#65533;?&#65533;H2AX as <br />           ordinates cell cycle progression DNA repair.                     early 2 d siRNA transfection (Fig. 1 D), corroborating <br />                  In study, identify nucleosome-remodeling      findings recent study (Pegoraro et al., 2009). Thus, CHD4 <br />           histone deacetylation (NuRD) complex novel regulator         depletion leads accumulation spontaneous DNA dam- <br />           DSB response human cells. NuRD prevents accumu-           age activation apoptotic p53/p21 program. We infer <br />           lation spontaneous DNA damage regu&#173;<span id='am-43' about='obo:NCBITaxon_8186' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_1489905 obo:NCBITaxon_123366 obo:NCBITaxon_1489904 obo:NCBITaxon_1489872 obo:NCBITaxon_8184 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Lates" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47'  ></span>lates</span> apoptotic            NuRD prevents genome instability apoptosis. <br />           responses p53 p21. Moreover, NuRD rapidly re- <br />           cruited DSBs, promotes RNF8/RNF168-mediated              CHD4 MTA2 protect cells the <br />           histone ubiquitylation ubiquitin-dependent accumula-         clastogenic effects IR <br />           tion RNF168 BRCA1. Consequently, loss NuRD                 EGR-1 (MTA2) protects worm cells IR (van Haaften et <br />           components causes defects DNA repair checkpoint acti-         al., 2006). To examine MTA2 protects human cells <br />           vation, rendering cells hypersensitive IR. Thus, NuRD         IR, tested depletion affects clonogenic <br />           chromatin&#8211;remodeling complex novel DNA damage response          survival VH10-SV40 cells. Loss MTA2 led increase <br />           (DDR) factor helps preserve genome stability regulating   IR sensitivity comparable observed in <br />           signaling repair DNA damage.                                  XRCC4 knockdown cells, impaired DSB repair by <br />                                                                                nonhomologous end joining (Fig. 1, E F; Grawunder et al., <br />           Results discussion                                               1998). In addition, CHD4-depleted cells in- <br />                                                                                creased IR sensitivity (Fig. 1, E G). Thus, MTA2 and <br />           CHD4 preserves genome stability                                  CHD4 protect cells effects IR, implicating role <br />           prevents apoptosis                                                   NuRD cellular response DSBs. Furthermore, MTA2 <br />           We previously performed genome-wide RNAi screen             protects worm human cells IR, sug- <br />           nematode <span id='am-38' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-41'  ></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Caenorhabditis</span> elegans identified 45 genes         gest putative role DDR conserved. <br />           protect worms IR (van Haaften et al., 2006). Among <br />           genes well-known DDR factors novel            CHD4 controls p53/p21 <span id='am-73' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-74' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-75'  ></span><span id='am-76' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>axis</span> the <br />           genes, including egr-1/lin-40. egr-1 encodes protein   IR-induced DDR <br />           homologous MTA2 (metastasis-associated protein 2),          To investigate role NuRD DDR, examined whether <br />           component human NuRD complex (Xue et al., 1998;          CHD4 depletion affects ATM/ATR-dependent phosphorylation <br />  <br />  <br />     742   JCB &#8226; VOLUME 190 &#8226; NUMBER 5 &#8226; 2010 <br />                                                                                            Figure 1.&#8195; MTA2 CHD4 depletion renders <br />                                                                                           cells sensitive IR. (A) Depletion CHD4 <br />                                                                                           reduces cell proliferation. U2OS cells were <br />                                                                                           transfected indicated siRNAs. Cells <br />                                                                                           counted 0, 2, 3, 4 d siRNA <br />                                                                                           transfection. (B) Pictures representative <br />                                                                                           areas cell dishes A shown. <br />                                                                                           Bar, 200 &#181;m. (C) FACS analysis cells <br />                                                                                           A. PI, propidium iodide. (D) CHD4 deple- <br />                                                                                           tion leads enhanced levels phosphory- <br />                                                                                           lated p53 (S15p), p53, p21, &#65533;?&#65533;H2AX. <br />                                                                                           Protein levels monitored Western blot <br />                                                                                           analysis using WCEs cells A. SMC1 a <br />                                                                                           loading control. (E) CHD4, MTA2, XRCC4 <br />                                                                                           levels monitored Western blot analy- <br />                                                                                           sis using WCEs cells F G. Tubulin is <br />                                                                                           loading control. (F) MTA2 depletion renders <br />                                                                                           cells hypersensitive IR. VH10-SV40 cells <br />                                                                                           transfected indicated siRNAs. <br />                                                                                           (G) CHD4 depletion renders cells hypersensi- <br />                                                                                           tive IR. As F, siRNAs against <br />                                                                                           CHD4 used. Graphs represent mean &#177; <br />                                                                                           SEM independent experiments. <br />  <br />  <br />  <br />  <br /> of DDR components response IR. Knockdown CHD4            phosphorylation status SMC1 CHK1, reg- <br /> did impair IR-induced ATM activation &#65533;?&#65533;H2AX for-            ulators IR-induced intra&#8211;S phase checkpoints, weak <br /> mation, led increased levels &#65533;?&#65533;H2AX unirradiated      effect cell cycle progression (Fig. 2, B C). However, loss <br /> cells, corroborating previous result (Fig. 1 D, Fig. 2 A,     CHD4 enhanced levels total p53 phosphorylated <br /> and Fig. S1). We investigated CHD4 mediates          p53 exposure IR. This accompanied increase <br /> ATM/ATR-dependent activation downstream effectors SMC1         p21 levels 24 h IR treatment (Fig. 2 A). p53 p21 <br /> (S966p), CHK1 (S317p), CHK2 (S19p), p53 (S15p), p21           play prominent role G1 checkpoint response (Khanna <br /> of DDR. We repeatedly observed small increase        Jackson, 2001; Shiloh, 2003). Accordingly, detected an <br /> phosphorylation SMC1 CHK1, CHK2          arrest CHD4-depleted cells G1 phase maintained <br /> the 30 min IR exposure (unpublished <span id='am-96' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-97' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-99'  ></span><span id='am-100' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span>). In addi-   IR exposure (Fig. 2, B C). This suggests CHD4 <br /> tion, observed small accumulation CHD4-depleted           controls p53/p21-dependent G1 checkpoint responses induced <br /> cells mid&#8211;S phase, suggesting aberration      IR. <br />  <br />  <br />                                                                   NuRD regulates DNA damage response &#8226; Smeenk et al.                    743 <br />                                                                                         NuRD rapidly accumulates within <br />                                                                                        DSB-flanking chromatin <br />                                                                                        The DSB response characterized accumulation of <br />                                                                                        checkpoint DNA repair proteins DSB-flanking chroma- <br />                                                                                        tin. To investigate NuRD plays direct role DDR, <br />                                                                                        examined recruitment subunits (CHD4, <br />                                                                                        MTA2, MBD3) sites DNA damage. We GFP- <br />                                                                                        tagged CHD4, MTA2, MBD3 accumulate microlaser- <br />                                                                                        generated DSB tracks span entire chromatin region <br />                                                                                        marked &#65533;?&#65533;H2AX (Fig. 3 A Fig. S2 A), charac- <br />                                                                                        teristic shared known DSB-associated factors (Bekker- <br />                                                                                        Jensen et al., 2006). Furthermore, rapidly recruited to <br />                                                                                        DSB tracks accumulated similar kinetics. Accumulation <br />                                                                                        detectable 30 s, reaching half-maximum 40 s and <br />                                                                                        steady-state levels 3 min (Fig. 3, B C; Fig. S2, B C). <br />                                                                                        Mailand et al. (2007) recently defined distinct kinetic groups of <br />                                                                                        proteins arrive early (e.g., MDC1 RNF8) late <br />                                                                                        (e.g., BRCA1 53BP1) DSB track. We demonstrated that <br />                                                                                        GFP-CHD4, MTA2-GFP, GFP-MBD3 arrive DSB track <br />                                                                                        early GFP-MDC1 (Fig. 3 C Fig. S2 C). Interestingly, they <br />                                                                                        reached steady-state levels accumulation significantly <br />                                                                                        faster MDC1 BRCA1. This suggest NuRD <br />                                                                                        subunits occupy DSB-containing chromatin same <br />                                                                                        level core components DSB response (Fig. 3 C and <br />                                                                                        Fig. S2 C). We conclude NuRD factors that <br />                                                                                        assemble early DSB-flanking chromatin. <br />  <br />                                                                                        CHD4 regulates DSB-associated <br />                                                                                        ubiquitylation orchestrate the <br />                                                                                        accumulation RNF168 BRCA1 <br />                                                                                        To examine NuRD modulates early events DSB <br />                                                                                        response, analyzed IR-induced foci (IRIF) formation of <br />                                                                                        &#65533;?&#65533;H2AX histone mark acts docking site for <br />                                                                                        MDC1/RNF8 DSBs (Stucki et al., 2005). However, CHD4 <br />                                                                                        knockdown did affect &#65533;?&#65533;H2AX IRIF formation (Fig. 4 A), <br />                                                                                        corroborates Western blot analysis showing proper <br />                                                                                        IR-induced &#65533;?&#65533;H2AX formation absence CHD4 (Fig. 2 A <br />                                                                                        Fig. S1). Accordingly, MDC1 RNF8 IRIF formation <br />                                                                                        affected loss CHD4 (Fig. 4, A B). <br />                                                                                               In contrast, accumulation conjugated ubiquitin <br />                                                                                        IRIF impaired CHD4-depleted cells (about twofold; <br />                                                                                        Fig. 4, C D). Consistently, observed distinct reduction in <br />                                                                                        IRIF formation RNF168 BRCA1, shown <br />                                                                                        bind DSB-associated ubiquitin moieties (Fig. 4, C D), <br />                                                                                        formation require CHD4. Indeed, found <br />                                                                                        CHD4 knockdown, like RNF8 knockdown, significantly <br />           Figure 2.&#8195; CHD4 controls IR-induced p53/p21 responses. (A) CHD4 de-          decreased level &#65533;?&#65533;H2AX mono- diubiquitylation <br />           pletion increases IR-induced p53 phosphorylation (S15p), p53, p21        IR (Fig. S3, A B; Huen et al., 2007). Because total <br />           levels. U2OS cells transfected indicated siRNAs ex- <br />                                                                                        levels endogenous MDC1, RNF8, RNF168, BRCA1 <br />           posed 6 Gy IR. WCEs prepared indicated time points and <br />           &#65533;?&#65533;H2AX, ATM S1981p, ATM, CHD4, p53 S15p, p53, p21 levels             af&#173;fected CHD4 depletion, conclude the <br />           monitored Western blot analysis. Histone H3 SMC1 loading         observed defects IRIF formation resulted reduction <br />           controls. (B) CHD4- MTA2-depleted cells accumulate G1 phase and <br />                                                                                        RNF8/RNF168-dependent ubiquitylation subsequent <br />           remain G1 arrest exposure IR. U2OS cells transfected with <br />           indicated siRNAs 72 h, exposed 6 Gy IR, 12 h later,        RNF168 BRCA1 accumulation DSBs (Fig. S1). Moreover, <br />           subjected FACS. (C) As B, cells stained BrdU.    expression dominant-negative, ATPase-dead form CHD4 <br />           PI, propidium iodide. <br />                                                                                        (GFP-CHD4 K757R) reduced RNF168 IRIF formation (Fig. S3, <br />                                                                                        C D), suggesting ATP-dependent chromatin remodeling <br />                                                                                        driven CHD4 ATPase triggers process. <br />  <br />  <br />     744   JCB &#8226; VOLUME 190 &#8226; NUMBER 5 &#8226; 2010 <br />                                                                                    change nucleosome structures inacces- <br />                                                                                   sible RNF8/RNF168 H2A-type histone targets ame- <br />                                                                                   nable ubiquitylation. <br />  <br />                                                                                   CHD4 MTA2 promote DSB repair and <br />                                                                                   G2/M checkpoint activation <br />                                                                                   RNF168 BRCA1 promote DSB repair G2/M check- <br />                                                                                   point activation response IR (Moynahan et al., 1999; Xu <br />                                                                                   et al., 2001; Xie et al., 2007; Doil et al., 2009; Stewart et al., <br />                                                                                   2009). Given CHD4 facilitates accumulation these <br />                                                                                   proteins DSB-flanking chromatin, examined role in <br />                                                                                   DSB repair G2/M checkpoint activation. <br />                                                                                          Neutral comet assays performed determine the <br />                                                                                   effect CHD4 knockdown rejoining IR-induced <br />                                                                                   DSBs VH10&#8211;telomerase reverse transcriptase (TERT) cells. <br />                                                                                   As shown Fig. 5 (A B), detected pronounced increase <br />                                                                                   level DSBs unirradiated CHD4-depleted cells. This <br />                                                                                   consistent aforementioned elevated levels total <br />                                                                                   &#65533;?&#65533;H2AX (Fig. 1 D Fig. 2 A) implies spontaneous <br />                                                                                   DSBs accumulate absence CHD4 (Fig. 1 D). Importantly, <br />                                                                                   level IR-induced DSBs remained higher 2 h CHD4 <br />                                                                                   knockdown cells returned basal levels 6 h. This suggests <br />                                                                                   CHD4 promotes proper DSB repair (Fig. 5, A B). <br />                                                                                          Next, tested effect CHD4 MTA2 depletion on <br />                                                                                   IR-induced G2/M checkpoint activation. We CHD4- <br />                                                                                   MTA2-depleted cells, like depleted BRCA1 RNF8 <br />                                                                                   (Xu et al., 2001; Huen et al., 2007; Kolas et al., 2007), failed to <br />                                                                                   fully activate G2/M checkpoint continued enter mitosis <br />                                                                                   (Fig. 5, C D), indicating CHD4 MTA2 facilitate full- <br />                                                                                   scale activation G2/M checkpoint. <br />                                                                                          In conclusion, report identification charac- <br />                                                                                   terization NuRD chromatin&#8211;remodeling complex a <br />                                                                                   novel factor involved genome surveillance. The lack CHD4 <br />                                                                                   consistently led increased levels spontaneous DNA dam- <br />                                                                                   age, phenotype associated activation of <br />                                                                                   p53 p21 responses, reduced cell proliferation, apoptosis. <br />                                                                                   We propose loss chromatin remodeling NuRD in- <br />                                                                                   duces genome-wide chromatin alterations, render chro- <br />                                                                                   matin susceptible spontaneous DNA breaks (Fig. 5 E). <br />                                                                                   Support scenario comes recent study, which <br />                                                                                   demonstrated loss NuRD components during <br />                                                                                   premature normal aging associated changes in <br />                                                                                   higher-<span id='am-78' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-79' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> chromatin structure, including loss heterochro- <br />                                                                                   matic regions, accumulation spontaneous DNA dam- <br /> Figure 3.&#8195; CHD4 MTA2 rapidly accumulate sites DNA damage. <br /> (A) CHD4 MTA2 accumulate DSB-containing regions marked by <br />                                                                                   age (Pegoraro et al., 2009). <br /> &#65533;?&#65533;H2AX. U2OS cells transiently expressing GFP-CHD4 MTA2-GFP                       We provide evidence direct role NuRD the <br /> subjected laser microirradiation. After 15 min, cells immuno&#173;             DSB response, NuRD components (CHD4, MTA2, <br /> stained &#65533;?&#65533;H2AX. (B) GFP-CHD4 MTA2-GFP, like GFP-MDC1, rapidly <br /> accumulate DSB-containing regions. U2OS cells transiently expressing <br />                                                                                   MBD3) assemble DSBs. Loss CHD4 uncoupled the <br /> GFP-CHD4, MTA2-GFP, GFP-MDC1 microirradiated A                  DSB response level RNF8/RNF168-mediated histone <br /> subjected real-time recording protein assembly damaged area.         ubiquitylation, leading defects assembly RNF168 <br /> (C) Quantitative representation results B. U2OS cells transiently ex- <br /> pressing GFP-BRCA1 included. Relative fluorescence units (RFU) are <br />                                                                                   BRCA1. This likely attenuated DSB repair activa- <br /> plotted time scale. Graphs represent mean &#177; SEM 10           tion G2/M checkpoint contributed IR sensitiv- <br /> individual cells independent experiments. Bars, 10 &#181;m.          ity observed CHD4- MTA2-depleted cells (Fig. 5 E). Thus, <br />                                                                                   NuRD novel factor preserves genome stability mod- <br />       Thus, NuRD-mediated chromatin remodeling facilitates                        ulating chromatin structure (a) genome-wide prevent sponta- <br /> RNF8/RNF168-dependent histone ubiquitylation orches-                           neous DNA damage (b) DNA breaks orchestrate a <br /> trate accumulation RNF168 BRCA1. NuRD                              proper DSB response (see Larsen et al. issue). Our work <br />  <br />  <br />                                                                                   NuRD regulates DNA damage response &#8226; Smeenk et al.                745 <br />            Figure 4.&#8195; CHD4 promotes histone ubiquitylation DSBs orchestrate accumulation RNF168 BRCA1. (A) CHD4 depletion does alter <br />           &#65533;?&#65533;H2AX, MDC1, RNF8 IRIF formation. Cells transfected indicated siRNAs, exposed 1 Gy IR, 30 and/or 60 min later, <br />           immunostained &#65533;?&#65533;H2AX, MDC1, RNF8. (B) Quantitative analysis &#65533;?&#65533;H2AX, MDC1, RNF8 IRIF formation. More 150 nuclei cells A <br />           scored time point independent experiments. (C) CHD4 depletion impairs ubiquitin, RNF168, BRCA1 IRIF formation. As A, <br />           cells immunostained conjugated ubiquitin (FK2), RNF168, BRCA1. (D) Quantitative analysis ubiquitin, RNF168, BRCA1 <br />           IRIF formation (as B). Error bars indicate mean &#177; SEM. Bars, 10 &#181;m. <br />  <br />  <br />  <br />  <br />     746   JCB &#8226; VOLUME 190 &#8226; NUMBER 5 &#8226; 2010 <br />  Figure 5.&#8195; CHD4 promotes DSB repair IR-induced activation G2/M checkpoint. (A) CHD4 depletion impairs DSB repair. VH10-TERT cells were <br /> transfected indicated siRNAs 72 h, exposed 20 Gy IR, subjected neutral comet analysis indicated time points. Representa- <br /> tive images shown. Bar, 30 &#181;m. (B) Quantification tail moments using cells A. Tail moments condition calculated minimum <br /> of 300 cells data point. Results independent experiments shown box whisker plot. The ordinate cube <span id='am-63' about='obo:NCBITaxon_1' typeof='owl:Thing'><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="all" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="root" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>root</span> scale. Data <br /> were statistically analyzed using two-way analysis variance, revealed significant delay DSB repair CHD4-depleted cells 2 h IR <br /> (P &lt; 0.0001). (C) CHD4 MTA2 depletion impairs IR-induced G2/M checkpoint activation. U2OS cells transfected indicated siRNAs, <br /> exposed 3 Gy IR, 1 h later, immunostained phosphorylated histone H3 (S10p). Mitotic indexes determined FACS. A representative <br /> experiment shown. PI, propidium iodide. Red circle indicates fraction M phase cells. (D) Graphical representation relative mitotic index values. <br /> The ratio index values irradiated unirradiated cells calculated normalized control cells, set 1. The mean &#177; <br /> SEM experiments shown. (E) Model role CHD4 maintenance genome stability. See paragraph Results <span id='am-101' about='obo:NCBITaxon_84399' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_87877 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_84398 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-103'  ></span><span id='am-104' property="rdfs:label" content="Discus" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>discus</span>- <br /> sion details. <br />  <br />  <br />                                                                                   NuRD regulates DNA damage response &#8226; Smeenk et al.                        747 <br />            provides framework future studies gain                       selected global thresholding DAPI images. Contour tracing was <br />           insight cross talk chromatin remodeling                        performed, pixels labeled unique object index that <br />                                                                                             determines nucleus pixels belong. The actual detection foci was <br />           mechanisms involved genome maintenance.                                        performed <span id='am-1' about='obo:NCBITaxon_149625' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_163739 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="rdfs:label" content="Alexa" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Alexa</span> Fluor 488 Cy3 images. First, top-hat transformation <br />                                                                                             performed reduce influence variation background staining <br />                                                                                             nuclei. On resulting image, watershed algorithm performed <br />           Materials methods                                                             determine location foci image providing unique spot <br />                                                                                             index pixels focus. By combining foci indices object indices <br />           Cell culture chemicals                                                        obtained DAPI image, foci assigned nuclei which <br />           Human U2OS, VH10-SV40&#8211;, VH10-TERT&#8211;immortalized fibroblasts                    detected, allowing calculation number foci nucleus. <br />           grown DME (Invitrogen) containing 10% FCS (Bodinco BV). siRNA <br />           plasmid transfections performed using HiPerfect (QIAGEN)             Chromatin fractionation Western blotting <br />           JetPEI (Polyplus Transfection), respectively, according manufacturer&#8217;s     Chromatin-enriched extracts prepared used immunoprecipita- <br />           instructions. The following siRNA sequences used: (luciferase), 5&#57401;-CGUA&#173;     tion described previously (Huen et al., 2007). Whole cell extracts (WCEs) <br />           CGCGGAAUACUUCGA-3&#57401;; (CHD4-1; Thermo Fisher Scientific), 5&#57401;-CCAA&#173;G&#173;                prepared cell lysis Laemmli buffer. Proteins separated Bis- <br />           GACCUGAAUGAUGA-3&#57401;; (CHD4-2; Thermo Fisher Scientific), 5&#57401;-CAAA&#173;                   Tris-HCl&#8211;buffered acrylamide gels (Invitrogen) blotted PVDF <br />           GGUGCUGCUGAUGUA-3&#57401;; (MTA2; Thermo Fisher Scientific), 5&#57401;-CAAA-                    (Millipore) nitrocellulose (GE Healthcare). Membranes incubated <br />           GUCUCUCUCCUUACAUU-3&#57401;; (RNF8; Thermo Fisher Scientific), 5&#57401;-GAGG&#173;                  primary antibody indicated Figs. 1&#8211;5 followed incubation with <br />           GCCAAUGGACAAUUA-3&#57401;; (XRCC4), 5&#57401;-AUAUGUUGGUGAA&#173;CU&#173;                             secondary antibody (Odyssey IRDye; LI-COR Biosciences). The Odyssey <br />           G&#173;AGA-3&#8217; (Sartori et al., 2007). Cells examined 48 h siRNA             imager (LI-COR Biosciences) equipped Odyssey software (version 3.0) <br />           transfection unless stated. The cDNAs human CHD4 (Thermo            used scan membranes analyze fluorescence signals. <br />           Fisher Scientific) MTA2 (Imagenes) cloned pEGFP-C1 and <br />           pEGFP-N1, respectively (Takara Bio Inc.). The point mutation K757A            FACS <br />           introduced pEGFP-C1-CHD4 site-directed mutagenesis. The pGFP-             For G2/M checkpoint analysis, cells fixed 70% ethanol and <br />           MBD3, GFP-BRCA1, GFP-MDC1 constructs provided A. Bird                 stained rabbit antibody histone H3 S10p (Millipore) followed in- <br />           (University Edinburgh, Edinburgh, Scotland, UK), O. <span id='am-48' about='obo:NCBITaxon_211650' typeof='owl:Thing obo:NCBITaxon_1329950 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8504 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8509 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_1329912 obo:NCBITaxon_33208 obo:NCBITaxon_1329961 obo:NCBITaxon_1329911 obo:NCBITaxon_7711 obo:NCBITaxon_1390229 obo:NCBITaxon_8570 obo:NCBITaxon_7776 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_34989'><span id='am-49' property="rdfs:label" content="Sibon" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51'  ></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Sibon</span> (University       cubation conjugated anti&#8211;rabbit IgG (Alexa Fluor 488). For cell cycle <br />           Groningen, Groningen, Netherlands), J. Lukas (Danish <span id='am-26' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-28'  ></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Cancer" datatype="xsd:string"></span>Cancer</span> Soci-             analysis, cells pulse labeled 10 &#181;M BrdU 1 h, fixed 70% <br />           ety, Copenhagen, Denmark) respectively.                                           ethanol, denatured 2 M HCl, stained mouse antibody BrdU <br />                                                                                             followed incubation conjugated anti&#8211;mouse IgG (Alexa Fluor 488) <br />           Generation DSBs                                                                DNA staining 0.1 mg/ml propidium iodide. Cell sorting was <br />           DSBs induced IR, delivered x-ray generator                performed flow cytometer (LSRII; BD) using FACSDiva software (ver- <br />           (200 kV; 4 mA; 1.1 Gy/min dose rate; YXLON International).                        sion 5.0.3; BD). Quantifications performed using WinMDI software <br />                                                                                             (version 2.9; J. Trotter). <br />           Cell survival assay <br />           VH10-SV40 cells transfected 48 h, trypsinized, seeded low             Neutral comet assay <br />           density, exposed IR. 7 d later, cells washed 0.9% NaCl           DSBs measured VH10-TERT cells using Comet Assay system <br />           stained methylene blue. Colonies &gt;20 cells scored.               (Trevigen) according manufacturer&#8217;s instructions. Comet tail moments <br />                                                                                             scored using Comet Score software (TriTek). <br />           Microscopy laser microirradiation <br />           Brightfield pictures taken fluorescence microscope (EVOS fl AMG;      Online supplemental material <br />           Westover Scientific) using 4&#215; Ph objective. Laser microirradiation per-   Fig. S1 shows depletion CHD4 does alter expression level of <br />           formed confocal microscope (TCS SP5/AOBS; Leica) equipped            DDR proteins IR. Fig. S2 shows MBD3-GFP rapidly accumulates <br />           environmental chamber set 37&#176;C 5% CO2. Cells grown glass-          sites laser-induced DNA damage. Fig. S3 shows CHD4 depletion <br />           dishes (MatTek) colorless medium containing 10% FCS (Invitrogen),       reduces IR-induced &#65533;?&#65533;H2AX ubiquitylation CHD4 ATPase activity is <br />           Hoechst 33342 added medium concentration 0.5 &#181;g/ml         required RNF168 IRIF formation. Online supplemental material avail- <br />           laser irradiation. 0.5 &#215; 10 &#181;m DSB-containing tracks generated        able http://www.jcb.org/cgi/content/full/jcb.201001048/DC1. <br />           diode laser (&#65533;?&#65533; = 405 nm; 30% laser power; 0.75-s irradiation time;         We thank Jiri Lukas, Dan Durocher, Adrian Bird, Ody Sibon, Mauro Modesti <br />           2,220 &#181;W) using UV-transmitting 63&#215; 1.4 NA oil immersion objective (HCX         providing reagents. We acknowledge Ron Romeijn constructing pEGFP- <br />           PL APO; Leica). Confocal images recorded laser irradi-      C1-MTA2, Antoine Morr&#233;e technical advice, Karien Wiesmeijer, Annelies <br />           ation 20- 30-s time intervals period 5 12 min              van der Laan, Roeland Dirks, Przemek Krawczyk, Jacob Aten assistance <br />           analyzed using LAS-AF software (Leica). Fluorescence intensities cor-        microlaser irradiation, Jiri Lukas sharing results publication. <br />           rected background normalized prebleach values determine re-                 This work supported Vidi grant Netherlands Organi- <br />           cruitment kinetics.                                                               zation Scientific Research (NWO). <br />           Antibodies <br />                                                                                             Submitted: 11 January 2010 <br />           IRIF Western blot analysis performed using antibodies &#65533;?&#65533;H2AX <br />           (Millipore), &#65533;?&#65533;-tubulin (Sigma-Aldrich), ATM (GeneTex), ubiquitin (FK2; Enzo       Accepted: 6 August 2010 <br />           Life Sciences, Inc.), BRCA1 (EMD), histone H3, ATM S1981p, MDC1, MTA2, <br />           CHD4, RNF8 (Abcam), p53 p21 (Santa Cruz Biotechnology, Inc.), <br />           p53 S15p (Cell Signaling Technology), SMC1 (Bethyl Laboratories). <br />                                                                                             References <br />           The antibodies RNF168 XRCC4 provided D. Durocher (Uni-             Ahel, D., Z. Horejs&#237;, N. Wiechens, S.E. Polo, E. <span id='am-6' about='obo:NCBITaxon_316718' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3646 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_235631 obo:NCBITaxon_3977 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_235892 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="rdfs:label" content="Garcia" datatype="xsd:string"></span>Garcia</span>-Wilson, I. Ahel, H. <br />           versity Toronto, Toronto, Ontario, Canada) M. Modesti (Universite                 Flynn, M. Skehel, S.C. West, S.P. Jackson, et al. 2009. Poly(ADP-ribose)- <br />           d&#8217;<span id='am-68' about='obo:NCBITaxon_8831' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8830 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_8826 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-70'  ></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="Aix" datatype="xsd:string"></span>Aix</span>-Marseille, Marseille, France), respectively.                                       dependent regulation DNA repair chromatin re&#173;modeling en- <br />                                                                                                    zyme ALC1. Science. 325:1240&#8211;1243. doi:10.1126/science.1177321 <br />           IRIF analysis                                                                     Bekker-Jensen, S., C. Lukas, R. Kitagawa, F. Melander, M.B. Kastan, J. Bartek, <br />           Cells grown glass coverslips treated indicated Figs. 1&#8211;5.              J. Lukas. 2006. Spatial organization mammalian genome sur- <br />           Subsequently, cells washed PBS, fixed 2% formaldehyde                 veillance machinery response DNA strand breaks. J. Cell Biol. <br />           0.25% Triton X-100 PBS, incubated appropriate primary                    173:195&#8211;206. doi:10.1083/jcb.200510130 <br />           secondary antibodies (Alexa Fluor 488 [Invitrogen] goat anti&#8211;rabbit IgG        Chai, B., J. Huang, B.R. Cairns, B.C. Laurent. 2005. Distinct roles RSC <br />           conjugated Cy3 [Jackson ImmunoResearch Laboratories, Inc.]) DAPI.                 Swi/Snf ATP-dependent chromatin remodelers DNA double-strand <br />           An additional extraction 0.25% Triton X-100 CSK buffer (10 mM                    break repair. Genes Dev. 19:1656&#8211;1661. doi:10.1101/gad.1273105 <br />           Hepes, 300 mM sucrose, 100 mM NaCl, 3 mM MgCl2) performed                 Clapier, C.R., B.R. Cairns. 2009. The biology chromatin remodeling com- <br />           fixation incubation primary antibodies RNF168                plexes. Annu. Rev. Biochem. 78:273&#8211;304. doi:10.1146/annurev.biochem <br />           BRCA1. Images recorded microscope (Axioplan; Carl Zeiss, Inc.)               .77.062706.153223 <br />           using 63&#215; 1.25 NA oil objective <span id='am-58' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-59' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-62'  ></span>camera</span> (AxioCam mRm; Carl Zeiss,          Denslow, S.A., P.A. Wade. 2007. The human Mi-2/NuRD complex gene <br />           Inc.) analyzed using home-made Stacks software. Nuclei detected                 regulation. Oncogene. 26:5433&#8211;5438. doi:10.1038/sj.onc.1210611 <br />  <br />  <br />     748   JCB &#8226; VOLUME 190 &#8226; NUMBER 5 &#8226; 2010 <br />  Doil, C., N. Mailand, S. Bekker-Jensen, P. Menard, D.H. Larsen, R. Pepperkok,          Pegoraro, G., N. Kubben, U. Wickert, H. G&#246;hler, K. Hoffmann, T. Misteli. <br />        J. Ellenberg, S. Panier, D. Durocher, J. Bartek, et al. 2009. RNF168                    2009. Ageing-related chromatin defects loss NURD com- <br />        binds amplifies ubiquitin conjugates damaged chromosomes                      plex. Nat. Cell Biol. 11:1261&#8211;1267. doi:10.1038/ncb1971 <br />        allow accumulation repair proteins. Cell. 136:435&#8211;446. doi:10.1016/          Sartori, A.A., C. Lukas, J. Coates, M. Mistrik, S. Fu, J. Bartek, R. Baer, J. Lukas, <br />        j.cell.2008.12.041                                                                      S.P. Jackson. 2007. Human CtIP promotes DNA end resection. <br /> Dubrana, K., H. van Attikum, F. Hediger, S.M. Gasser. 2007. The processing                 Nature. 450:509&#8211;514. doi:10.1038/nature06337 <br />       double-strand breaks binding single-strand-binding proteins            Seelig, H.P., I. Moosbrugger, H. Ehrfeld, T. Fink, M. Renz, E. Genth. 1995. <br />       RPA Rad51 modulate formation ATR-kinase foci yeast.                        The major dermatomyositis-specific Mi-2 autoantigen presumed <br />       J. Cell Sci. 120:4209&#8211;4220. doi:10.1242/jcs.018366                                       helicase involved transcriptional activation. Arthritis <span id='am-11' about='obo:NCBITaxon_3620' typeof='owl:Thing obo:NCBITaxon_1110380 obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3615 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_1110386 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Rheum" datatype="xsd:string"></span><span id='am-14'  ></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Rheum</span>. 38:1389&#8211; <br /> Gong, Z., Y.W. Cho, J.E. Kim, K. Ge, J. Chen. 2009. Accumulation Pax2                   1399. doi:10.1002/art.1780381006 <br />       transactivation domain interaction protein (PTIP) sites DNA breaks         Shiloh, Y. 2003. ATM related protein kinases: safeguarding genome integ- <br />       RNF8-dependent pathway required cell survival DNA                       rity. Nat. Rev. Cancer. 3:155&#8211;168. doi:10.1038/nrc1011 <br />       damage. J. Biol. Chem. 11:7284&#8211;7293. doi:10.1074/jbc.M809158200 <br />                                                                                        Shim, E.Y., J.L. Ma, J.H. Oum, Y. Yanez, S.E. Lee. 2005. The yeast chro- <br /> Gottschalk, A.J., G. Timinszky, S.E. Kong, J. Jin, Y. Cai, S.K. Swanson, M.P.                  matin remodeler RSC complex facilitates end joining repair DNA <br />        Washburn, L. Florens, A.G. Ladurner, J.W. Conaway, R.C. Conaway.                    double-strand breaks. Mol. Cell. Biol. 25:3934&#8211;3944. doi:10.1128/MCB <br />        2009. Poly(ADP-ribosyl)ation directs recruitment activation ATP-              .25.10.3934-3944.2005 <br />        dependent chromatin remodeler. Proc. Natl. Acad. Sci. USA. 106:13770&#8211; <br />        13774. doi:10.1073/pnas.0906920106                                              Shim, E.Y., S.J. Hong, J.H. Oum, Y. Yanez, Y. Zhang, S.E. Lee. 2007. RSC <br />                                                                                                mobilizes nucleosomes improve accessibility repair machinery to <br /> Grawunder, U., D. Zimmer, S. Fugmann, K. Schwarz, M.R. Lieber. 1998.                       damaged chromatin. Mol. Cell. Biol. 27:1602&#8211;1613. doi:10.1128/ <br />      DNA ligase IV essential V(D)J recombination DNA double-                        MCB.01956-06 <br />      strand break repair human precursor lymphocytes. Mol. Cell. 2:477&#8211; <br />                                                                                        Stewart, G.S., S. Panier, K. Townsend, A.K. Al-Hakim, N.K. Kolas, E.S. Miller, <br />      484. doi:10.1016/S1097-2765(00)80147-1 <br />                                                                                                S. Nakada, J. Ylanko, S. Olivarius, M. Mendez, et al. 2009. The RIDDLE <br /> Hoeijmakers, J.H. 2001. Genome maintenance mechanisms preventing                           syndrome protein mediates ubiquitin-dependent signaling cascade sites <br />       cancer. Nature. 411:366&#8211;374. doi:10.1038/35077232                                        DNA damage. Cell. 136:420&#8211;434. doi:10.1016/j.cell.2008.12.042 <br /> Huang, J., M.S. Huen, H. Kim, C.C. Leung, J.N. Glover, X. Yu, J. Chen.             Stucki, M., J.A. Clapperton, D. Mohammad, M.B. Yaffe, S.J. Smerdon, S.P. <br />       2009. RAD18 transmits DNA damage signalling elicit homologous re-                     Jackson. 2005. MDC1 directly binds phosphorylated histone H2AX to <br />       combination repair. Nat. Cell Biol. 11:592&#8211;603. doi:10.1038/ncb1865                      regulate cellular responses DNA double-strand breaks. Cell. 123:1213&#8211; <br /> Huen, M.S., R. Grant, I. Manke, K. Minn, X. Yu, M.B. Yaffe, J. Chen. 2007. RNF8            1226. doi:10.1016/j.cell.2005.09.038 <br />       transduces DNA-damage signal histone ubiquitylation check&#173;           Tsukuda, T., A.B. Fleming, J.A. Nickoloff, M.A. Osley. 2005. Chromatin re- <br />       point protein assembly. Cell. 131:901&#8211;914. doi:10.1016/j.cell.2007.09.041                modelling DNA double-strand break site <span id='am-16' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span> cerevi- <br /> Ikura, T., S. Tashiro, A. Kakino, H. Shima, N. Jacob, R. Amunugama, K. Yoder,                  siae. Nature. 438:379&#8211;383. doi:10.1038/nature04148 <br />         S. Izumi, I. Kuraoka, K. Tanaka, et al. 2007. DNA damage-dependent             van Attikum, H., S.M. Gasser. 2009. Crosstalk histone modifica- <br />         acetylation ubiquitination H2AX enhances chromatin dynamics.                    tions DNA damage response. Trends Cell Biol. 19:207&#8211;217. <br />         Mol. Cell. Biol. 27:7028&#8211;7040. doi:10.1128/MCB.00579-07                                doi:10.1016/j.tcb.2009.03.001 <br /> Jazayeri, A., J. Falck, C. Lukas, J. Bartek, G.C. Smith, J. Lukas, S.P. Jackson.   van Attikum, H., O. Fritsch, B. Hohn, S.M. Gasser. 2004. Recruitment the <br />        2006. ATM- cell cycle-dependent regulation ATR response                    INO80 complex H2A phosphorylation links ATP-dependent chroma- <br />        DNA double-strand breaks. Nat. Cell Biol. 8:37&#8211;45. doi:10.1038/ncb1337                  tin remodeling DNA double-strand break repair. Cell. 119:777&#8211;788. <br />                                                                                                doi:10.1016/j.cell.2004.11.033 <br /> Khanna, K.K., S.P. Jackson. 2001. DNA double-strand breaks: signal- <br />      ing, repair cancer connection. Nat. Genet. 27:247&#8211;254. doi:10             van Attikum, H., O. Fritsch, S.M. Gasser. 2007. Distinct roles SWR1 and <br />      .1038/85798                                                                               INO80 chromatin remodeling complexes chromosomal double-strand <br />                                                                                                breaks. EMBO J. 26:4113&#8211;4125. doi:10.1038/sj.emboj.7601835 <br /> Kolas, N.K., J.R. Chapman, S. Nakada, J. Ylanko, R. Chahwan, F.D. Sweeney, S. <br />        Panier, M. Mendez, J. Wildenhain, T.M. Thomson, et al. 2007. Orches&#173;            van Haaften, G., R. Romeijn, J. Pothof, W. Koole, L.H. Mullenders, A. Pastink, <br />        tration DNA-damage response RNF8 ubiquitin ligase.                        R.H. Plasterk, M. Tijsterman. 2006. Identification conserved path- <br />        Science. 318:1637&#8211;1640. doi:10.1126/science.1150034                                     ways DNA-damage response radiation protection genome-wide <br />                                                                                                RNAi. Curr. Biol. 16:1344&#8211;1350. doi:10.1016/j.cub.2006.05.047 <br /> Larsen, D.H., C. Poinsignon, T. Gudjonsson, C. Dinant, M.R. Payne, F.J. Hari, <br />        J.M.R. Danielsen, P. Menard, J.C. Sand, M. Stucki, et al. 2010. The             Wang, B., S.J. Elledge. 2007. Ubc13/Rnf8 ubiquitin ligases control foci for- <br />        chromatin-remodeling factor CHD4 coordinates signaling repair                       mation Rap80/<span id='am-81' about='obo:NCBITaxon_393334' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_82596 obo:NCBITaxon_82593 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_82592 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-82' property="rdfs:label" content="Abraxas" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Calospilos" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-85'  ></span><span id='am-86' property="oboInOwl:hasAlternativeId" content="NCBITaxon:190334" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Abraxas</span>/Brca1/Brcc36 complex response DNA <br />        DNA damage. J. Cell Biol. 190:731&#8211;740.                                            damage. Proc. Natl. Acad. Sci. USA. 104:20759&#8211;20763. doi:10.1073/pnas <br />                                                                                                .0710061104 <br /> Liang, B., J. Qiu, K. Ratnakumar, B.C. Laurent. 2007. RSC functions an <br />        early double-strand-break sensor cell&#8217;s response DNA damage.          Wang, H.B., Y. Zhang. 2001. Mi2, auto-antigen dermatomyositis, is <br />        Curr. Biol. 17:1432&#8211;1437. doi:10.1016/j.cub.2007.07.035                                 ATP-dependent nucleosome remodeling factor. Nucleic Acids Res. <br />                                                                                                29:2517&#8211;2521. doi:10.1093/nar/29.12.2517 <br /> Luo, J., F. Su, D. Chen, A. Shiloh, W. Gu. 2000. Deacetylation p53 modu- <br />         lates effect cell growth apoptosis. Nature. 408:377&#8211;381. doi:10     Xie, A., A. Hartlerode, M. Stucki, S. Odate, N. Puget, A. Kwok, G. Nagaraju, C. <br />         .1038/35042612                                                                         Yan, F.W. Alt, J. Chen, et al. 2007. Distinct roles chromatin-associated <br />                                                                                                proteins MDC1 53BP1 mammalian double-strand break repair. <br /> Mailand, N., S. Bekker-Jensen, H. Faustrup, F. Melander, J. Bartek, C. Lukas,                  Mol. Cell. 28:1045&#8211;1057. doi:10.1016/j.molcel.2007.12.005 <br />       J. Lukas. 2007. RNF8 ubiquitylates histones DNA double-strand <br />       breaks promotes assembly repair proteins. Cell. 131:887&#8211;900.              Xu, B., Kim St, M.B. Kastan. 2001. Involvement Brca1 S-phase and <br />       doi:10.1016/j.cell.2007.09.040                                                           G(2)-phase checkpoints ionizing irradiation. Mol. Cell. Biol. 21: <br />                                                                                                3445&#8211;3450. doi:10.1128/MCB.21.10.3445-3450.2001 <br /> Morrison, A.J., J. Highland, N.J. Krogan, A. Arbel-Eden, J.F. Greenblatt, J.E. <br />       Haber, X. Shen. 2004. INO80 gamma-H2AX interaction links                 Xue, Y., J. Wong, G.T. Moreno, M.K. Young, J. C&#244;t&#233;, W. Wang. 1998. <br />       ATP-dependent chromatin remodeling DNA damage repair. Cell. 119:                      NURD, novel complex ATP-dependent chromatin-remodeling <br />       767&#8211;775. doi:10.1016/j.cell.2004.11.037                                                  histone deacetylase activities. Mol. Cell. 2:851&#8211;861. doi:10.1016/ <br />                                                                                                S1097-2765(00)80299-3 <br /> Moynahan, M.E., J.W. Chiu, B.H. Koller, M. Jasin. 1999. Brca1 controls <br />      homology-directed DNA repair. Mol. Cell. 4:511&#8211;518. doi:10.1016/                  Zhang, Y., H.H. Ng, H. Erdjument-Bromage, P. Tempst, A. Bird, D. Reinberg. <br />      S1097-2765(00)80202-6                                                                     1999. Analysis NuRD subunits reveals histone deacetylase core <br />                                                                                                complex connection DNA methylation. Genes Dev. 13:1924&#8211; <br /> Murr, R., J.I. Loizou, Y.G. Yang, C. Cuenin, H. Li, Z.Q. Wang, Z. Herceg.                  1935. doi:10.1101/gad.13.15.1924 <br />       2006. Histone acetylation Trrap-Tip60 modulates loading repair <br />                                                                                        Zou, L., S.J. Elledge. 2003. Sensing DNA damage ATRIP recogni- <br />       proteins repair DNA double-strand breaks. Nat. Cell Biol. 8:91&#8211;99. <br />                                                                                             <br /> </body></html>