<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Natl. Acad. Sci. USA <br /> Vol. 95, pp. 12432&#8211;12437, October 1998 <br /> Genetics <br />  <br />  <br />  <br /> Knock-out mutants En-1 mutagenized <span id='am-6' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span><span id='am-10'  ></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span>Arabidopsis</span> <br /> thaliana population generate phenylpropanoid <br /> biosynthesis phenotypes <br /> ELLEN WISMAN, <span id='am-115' about='obo:NCBITaxon_887755' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_50483 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_50482 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_85817 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_50557 obo:NCBITaxon_6072 obo:NCBITaxon_6960'><span id='am-116'  ></span><span id='am-117' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-119' property="rdfs:label" content="Ulrike" datatype="xsd:string"></span>ULRIKE</span> HARTMANN, MARTIN SAGASSER, <span id='am-105' about='obo:NCBITaxon_304612' typeof='owl:Thing obo:NCBITaxon_9241 obo:NCBITaxon_9242 obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-106' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-107'  ></span><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="Elvira" datatype="xsd:string"></span>ELVIRA</span> BAUMANN, K LAUS PALME, K LAUS HAHLBROCK, <br /> HEINZ SAEDLER, AND BERND WEISSHAAR* <br /> Max-Planck-Institut fu <br />                      &#168;r <span id='am-1' about='obo:NCBITaxon_143324' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_81386 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_81393 obo:NCBITaxon_7776 obo:NCBITaxon_123357 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Zu" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5'  ></span>Zu</span> <br />                          &#168;chtungsforschung, Carl-von-Linne <br />                                                          &#180;-Weg 10, D-50829 Ko <br />                                                                             &#168;ln, Germany <br />  <br /> Contributed Klaus Hahlbrock, August 7, 1998 <br />  <br />  <br /> ABSTRACT          A collection 8,000 <span id='am-13' about='obo:NCBITaxon_3702' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_3701 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="thale-cress" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Arabidopsis thaliana (thale cress)" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="mouse-ear cress" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="thale cress" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Arbisopsis thaliana" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Arabidopsis thaliana" datatype="xsd:string"></span>Arabidopsis thaliana</span>                         introduced heterologous host A. thaliana, En-1 main- <br /> plants carrying 48,000 insertions maize transposable                          tains mobility (10). A heterologous transposable element was <br /> element En-1 generated. <span id='am-62' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-63'  ></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> population used                            required endogenous transposable element with <br /> for reverse genetic analyses identify insertions individual                    high activity present A. thaliana ecotype Columbia <br /> gene loci. By using PCR-based screening protocol, insertions                       (Col) chosen sequence analysis (11). The aim work <br /> were 55 genes. En-1 showed preference                                create identify loss-of-function mutations for <br /> transcribed untranscribed regions particular                            subsequent identification function mutated gene. <br /> orientation relative gene interest. In cases,                      An example phenylpropanoid metabolism demonstrates <br />                                                                                      feasibility strategy. <br /> En-1 inserted kilobases upstream down-                              Biochemical analysis phenylpropanoid biosynthetic path- <br /> stream gene. En-1 mobilized positions                           ways provided detailed information enzymes in- <br /> into respective gene cause gene disruption. Knock-out                         volved anthocyanin flavonol pigment formation (12). <br /> alleles genes involved f lavonoid biosynthesis gen-                       The corresponding genes many, all, enzymes of <br /> erated. One mutant line contained En-1 insertion                           pathway identified (13). In A. thaliana, the <br /> f lavonol synthase gene (FLS) showed drastically reduced                         transparent testa (tt) mutants define number phenylpro- <br /> levels kaempferol. Allelism tests lines containing                     panoid biosynthetic genes (14, 15). In case tt6, was <br /> En-1 insertions f lavanone 3-hydroxylase gene (F3H)                           assumed basis flavonoids detectable mutant <br /> demonstrated TRANSPARENT TESTA 6 (TT6) encodes                                  TT6 encodes flavonol synthase (FLS) (15), catalyses <br /> f lavanone 3-hydroxylase. The f3h fls null mutants com-                          conversion dihydroflavonols flavonols belongs to <br /> plete set A. thaliana lines defective early steps                   <span id='am-54' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-55' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> 2-oxoglutarate-dependent dioxygenases. Because <br /> f lavonoid pathway. These experiments demonstrate effi-                          flavonol glycosides accumulate vacuoles epidermal <br /> ciency screening method gene disruption strategy                          cells light-dependent manner, FLS gene activity ex- <br /> used assigning functions genes defined sequence.                      pected light-responsive way similar chalcone <br />                                                                                      synthase (16, 17). Sequence information&#8212;for example, from <br />                                                                                      ESTs&#8212;can used group gene enzyme the <br /> The functional identification genes cause specific                           dioxygenase family, additional biochemical information is <br /> phenotypic alterations remains major challenge genetics                         required define substrate specificity precise metabolic <br /> and molecular biology. As convenient tool, insertional                             function given dioxygenase. A cDNA encoding FLS has <br /> mutagenesis transposable elements (1) T-DNA tagging                            isolated P. hybrida (18), and, work in <br /> (2) used successfully. In forward genetic screens,                      progress, sequence putative A. thaliana FLS cDNA was <br /> DNA causing gene disruption marks respective locus                           published (19). <br /> subsequently used molecular probe isolate                              Here, application En-1-mutagenized <br /> phenotypically defined gene. Transposable elements,                          A. thaliana population link DNA sequences functions. A <br /> Mu, AcyDs EnySpm <span id='am-39' about='obo:NCBITaxon_4575' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_4479 obo:NCBITaxon_147369 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4734 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_38820 obo:NCBITaxon_147429 obo:NCBITaxon_131221 obo:NCBITaxon_147370 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Euchlaena" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Zea" datatype="xsd:string"></span><span id='am-44'  ></span>Zea</span> mays, Tam <span id='am-23' about='obo:NCBITaxon_4150' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_216780 obo:NCBITaxon_3398 obo:NCBITaxon_4143 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_156152 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="NCBITaxon:237458" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="Antirrhinum" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Antirrhinum L." datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="snapdragons" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Sairocarpus" datatype="xsd:string"></span>Antirrhinum</span>                                 FLS knock-out mutant isolated PCR-based screening. <br /> majus, dTph <span id='am-32' about='obo:NCBITaxon_4101' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_4070 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_424555 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34'  ></span><span id='am-35' property="rdfs:label" content="Petunia" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="petunia" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Petunia Juss." datatype="xsd:string"></span>Petunia</span> hybrida, applied gene                       The locus demonstrated phenotype knock- <br /> tagging native hosts, AcyDs EnySpm                        line and, subsequently, enzyme assays encode FLS. <br /> engineered function heterologous hosts (3). For reverse                        Furthermore, TT6 locus shown reverse forward <br />                                                                                      genetics encode flavanone 3-hydroxylase (F3H). These <br /> genetics, gene disruption systems based Mu (4) T-DNA <br />                                                                                      examples flavonoid biosynthesis demonstrate suc- <br /> (2) developed, PCR-based strategies                          cessful use En-1-mutagenized population PCR- <br /> used identification insertion alleles particular gene                  based screens assign gene functions unequivocally. <br /> loci (5&#8211;8). <br />    Large collections Arabidopsis thaliana expressed se-                                        MATERIALS AND METHODS <br /> quence tags (ESTs) exist, genes unknown <br /> functions, complete sequence A. thaliana                                Standard Molecular Biology Techniques Materials. <br /> genome available near future. Therefore,                        Basic molecular biology techniques applied according to <br /> the need efficient methods correlate sequence informa- <br /> tion gene function evident. To create convenient gene                      Abbreviations: Col, ecotype Columbia; EST, expressed sequence tag; <br /> disruption A. thaliana, used full-size auton-                      FLS, flavonol synthase; F3H, flavanone 3-hydroxylase; Ler, ecotype <br /> omous transposable element En-1. En-1 particular EnySpm                         Landsberg erecta; RIL, recombinant inbred line; TLC, layer <br />                                                                                      chromatography; tt, transparent testa. <br /> element isolated Z. mays WAXY locus (9). When                               <span id='am-110' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-111' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-113'  ></span><span id='am-114' property="rdfs:label" content="Data" datatype="xsd:string"></span>Data</span> deposition: The sequences reported paper been <br />                                                                                      deposited GenBank database [accession nos. U84258 (Arabi- <br /> The publication costs article defrayed page charge           dopsis thaliana ecotype Columbia FLS gene sequence), U84259 (<span id='am-57' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59'  ></span><span id='am-60' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Ara</span>- <br /> payment. This article marked &#8216;&#8216;advertisement&#8217;&#8217;           bidopsis thaliana ecotype Columbia FLS cDNA sequence), AF064064 <br />                                                                                      (Arabidopsis thaliana ecotype Columbia F3H gene sequence), and <br /> accordance 18 U.S.C. &#167;1734 solely indicate fact. <br />                                                                                      AF064065 (Arabidopsis thaliana f3h-1(tt6) allele sequence)]. <br /> &#169; 1998 The National Academy Sciences 0027-8424y98y9512432-6$2.00y0             *To reprint requests addressed. e-mail: weisshaa@ <br /> PNAS available online www.pnas.org.                                             mpiz-koeln.mpg.de. <br />  <br />                                                                              12432 <br />           Genetics: Weisshaar                                                        Proc. Natl. Acad. Sci. USA 95 (1998)      12433 <br />  <br /> Sambrook et al. (20). Radiolabeled probes derived          inaccuracy restricted number classical <br /> gel-purified DNA fragments described AtCHS UBQ            visible markers included RIL maps. <br /> (17). The Col FLS probe derived full-length FLS              Molecular Characterization Mutant Alleles. Sequences <br /> cDNA (U84259) isolated lZAPII plant cDNA                various mutants verified direct sequencing of <br /> library using probe derived EST T76883. The FLS            PCR fragments. Initially, Col F3H genomic DNA fragments <br /> transcriptional start site determined 59 rapid amplifi-       synthesized using primers designed according to <br /> cation cDNA ends described (21). The F3H transcrip-            published Ler sequences (26). For Col F3H sequence, see <br /> tional start site determined using premixed reagents          GenBank accession no. AF064064. Genomic sequences flank- <br /> (59y39 rapid amplification cDNA ends kit, Boehringer              ing En-1 insertions amplified En-1-specific <br /> Mannheim) according manufacturer&#8217;s recommenda-                primer (En205 En8130 59 39 ends, respectively; ref. <br /> tions. Sequences plasmid DNA PCR fragments               22) respective gene specific primer. <br /> determined Applied Biosystems 377 sequencers using                Cleaned Amplified Polymorphic Sequence Marker Analy- <br /> dye-terminator chemistry. Oligonucleotides purchased            sis. A. thaliana genomic DNA isolated described (27). <br /> from MWG (Ebersbach, Germany). Thin layer chromatogra-               PCR conditions, primers, polymorphic sites as <br /> phy (TLC) standards obtained Sigma Roth                follows: 50 ml total volume, &apos;20 ng genomic DNA, 20 pM of <br /> (Karlsruhe, Germany).                                                primer, 1 unit Taq Pol, buffer (Boehringer Mann- <br />    En-1 Mutagenized Population.&#8224; The En-1 mutagenized A.             heim); cycle program: 5 min, 94&#176;C; 403 (1 min, 94&#176;C; 1 min, <br /> thaliana poplulation 3,000 lines (22) extended 5,000       55&#176;C; 1 min, 72&#176;C); 10 min, 72&#176;C; FLS primers oUH5 <br /> lines total 8,000 lines carrying &apos;48,000 independent         (59-GAATCCCTAATAACGTCTCCG-39) oUH1982 (59- <br /> En-1 insertions. The new lines created similar way      TTACATATCCGCCATTGTTTCCGGC-39) F3H prim- <br /> the original 3,000 lines. First, 28 lines derived parent G11    ers oSA4 oSA21 (59-CACTTTCACCCATCCTTCAG- <br /> (23) 3 lines derived parent G69, carrying 8 16       GCTTATTTG-39); restriction polymorphisms: BssHII site <br /> En-1 insertions, selected. The 5,000 new lines consisted        present Col FLS, BclI site present in <br /> of 150 progeny 31 selected plants propagated 6            Ler F3H. <br /> additional single-seed descent generations 400               FLS Enzyme Assay. The FLS ORF, modified start and <br /> original single-seed descent lines (23) generation 12. Finally,   stop codons PCR, inserted NcoIyBamHI-digested <br /> leaf material DNA preparations seeds sampled            pQE60 (Qiagen, Hilden, Germany), resulting plasmid <br />                                                                      pQE60-FLS. <span id='am-82' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-86'  ></span><span id='am-87' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span>Escherichia coli</span> strain SG13009::pUBS520 (28) <br /> from lines. Plant genomic DNA prepared described <br />                                                                      used protein production described (29). As a <br /> (22). <br />                                                                      negative control, cells containing pQE60 included. Bac- <br />    Screening Mutant Phenotypes. To identify mutations in <br />                                                                      teria 50-ml culture resuspended 3 ml Hepes <br /> the initial population, 2,000 S6 families 20 individuals each <br />                                                                      buffer (0.2 M Hepes/KOH, pH 7.0y1 mM EDTAy200 mg/ml <br /> were analyzed soil segregation recessive mutant <br />                                                                      lysozyme) incubated ice 30 min slight <br /> phenotypes, 20 40 S6 seedlings 3,000 lines were <br />                                                                      mixing 5 min. The suspension centrifuged (13,000 3 <br /> screened MS agar (50 mgyliter ampicillin, sucrose; 16 h        g, 10 min, 4&#176;C), supernatant (crude extract) used <br /> fluorescent light day). The mutant traits scored agar         FLS activity assays. A 200-ml reaction mixture contained <br /> were germination, seedling growth, pigmentation, <span id='am-72' about='obo:NCBITaxon_1' typeof='owl:Thing'><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="all" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="root" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>root</span> length,        3,000 dpm 14C-labeled dihydrokaempferol, 10 ml crude extract, <br /> and seed size. Traits scored soil flowering time, plant      5 mM ascorbate, 50 mM FeSO4, 0.25 mM 2-oxoglutarate, and <br /> size, plant leaf shape, trichome morphology, epicuticular        50 mM HepesyKOH (pH 7.0) incubated 37&#176;C for <br /> wax, flower morphology, fertility.                               30 min (30). Flavonoids extracted twice ethyl acetate <br />    Reverse Genetic Screens. PCR-based screens per-              chromatographed cellulose plates (Merck) by <br /> formed according dimensional grid described            using chloroformyacetic acidywater (10:9:1) solvent. Unla- <br /> (22). The following primers specific respective Col          beled dihydrokaempferol, kaempferol, naringenin were <br /> allele used: oUH8 (59-GAATCCGTCGCTAAACCG-                       co-chromatographed standards visualized under <br /> GA AGAT TCG-39) oUH10 (59-GGAGGAT TAT-                           UV light. The radiolabeled substances detected by <br /> CATCTCCGGT TAGT TCCG-39) FLS, oSA4 (59-                          radioimaging (Fuji Fujix BAS1000). <br /> AGGAACTTTGACTGAGCTAGCCGGAGAGTC-39)                                  Extraction Flavonols TLC. Leaves incubated in <br /> oSA7 (59-GAGGCGAGCAAGCTCCAAATCTCTTCCC-                               2 M HCl 16 h RT subsequently 20 min 100&#176;C. <br /> 39) F3H.                                                         Flavonoid aglycons extracted isoamyl alcohol, were <br />    Other Plant Material RNA Isolation. For dark adap-            developed cellulose TLC plates (Merck) Forestal <br /> tation, 8-week-old plants transferred complete dark-         [wateryacetic acidyHCl (32%) 10:30:1] chloroformyacetic <br /> ness total 64 h. Light induction experiments           acidywater (10:9:1), visualized 366-nm UV <br /> carried irradiating dark-adapted plants 24 h         light. Standards kaempferol quercetin. <br /> continuous white light (&apos;150 microeinsteinym2zs) phy- <br /> tochamber. Immediately harvesting, leaves frozen <br /> in liquid nitrogen stored use 280&#176;C. Total                                  RESULTS <br /> RNA isolated described (21). tt6 obtained               Frequency Visible Mutations. A population 8,000 A. <br /> Nottingham Anabidopsis Stock Centre (stock number NW87).             thaliana lines containing, average, independent En-1 <br />    Mapping En-1 Insertion Sites. DNA fragments flanking           insertions (48,000 total) generated single-seed de- <br /> En-1 insertions mapped using Lister Dean (24)        scent. The number insertions line varied 1 and <br /> recombinant inbred lines (RILs) Landsberg erecta           20. In sub-population 3,000 lines, 153 mutant phenotypes <br /> (Ler) 3 Cape Verde Islands RIL map developed recently (25).          (5%) detected initial screen (Table 1). Of the <br /> Phenotypic markers used allelism tests assign         mutations, &apos;70% genetically unstable, indicating the <br /> new mutant phenotypes previously mapped mutants. Mo-              presence En-1 mutant locus. The data En-1 <br /> lecular phenotypic markers combined map           mobile A. thaliana causes broad spectrum mutant <br /> using classical map landmarks, procedure causes        phenotypes. <br />                                                                         Distribution En-1 Insertions Genome. The local- <br /> &#8224;Readers interested screening En-1 mutagenized A. thaliana    ization En-1 insertions determined using three <br /> population contact Ellen Wisman (wisman@mpiz-koeln.mpg.          different strategies. First, mutants known phenotypes <br /> de).                                                                 crossed lines containing corresponding mapped <br />  12434      Genetics: Weisshaar                                                         Proc. Natl. Acad. Sci. USA 95 (1998) <br />  <br /> Table 1.   Mutants detected 3,000 En-1-mutagenized lines          En-1 insertions recovered 55 genes (Fig. 1). In <br />                               Number          Unstable        Stable    50% cases, different alleles found. <br />                                                                         Insertions identified using three-dimensional, PCR- <br /> Morphological mutants                                                   based screening strategy confirmed individual <br />   Flower                          14              7              3      progeny selected plant. For final proof, sequence of <br />   Root                            12              1                     diagnostic PCR product determined. Of insertions <br />   Trichomes                       14                                    studied far, 37% located outside genes, 40% in <br />   Epicuticular wax                 5              1                     exons, 15% introns, 8% exonyintron border regions. <br />   Flowering time                  11              1                        The 37% insertions located close to, inside, gene <br /> Pigmentation mutants                                                    did result gene disruption. To isolate knock-out allele, <br />   Albino                          15              9              4      ability En-1 transpose linked sites exploited. <br />   Chlorina                         6              1 <br />                                                                         The success approach exemplified genes: <br />   Transparent testa               12              2 <br />                                                                         AtKC1 (potassium channel 1, U73325), AtKAT2 (voltage-gated <br />   Dark greenyred                   3              2 <br />                                                                         potassium channel, U25694), AtKCO1 (outward rectifying <br /> Reproduction mutants <br />                                                                         potassium channel, X97323), PIN homologue (AtPIN2). <br />   Lethal                          12 <br />                                                                         From respective homozygous plants, &apos;2,000 <br />   Sterile                          6              1 <br />                                                                         progeny analyzed three-dimensional, PCR-based <br />   Tetraploid                       5 <br />                                                                         screening strategy. In case, insertion located exonic <br /> Form mutants <br />                                                                         sequences recovered original insertion, located <br />   Branching                        7 <br />                                                                         1 3 kilobases distant new insertion site, had <br />   Size                            13 <br />                                                                         disappeared. We conclude En-1 powerful tool gene <br />   Leaf shape                       9 <br />   Other                            9              3              3 <br />                                                                         disruption A. thaliana. <br /> Total                            153             28             10 <br />                                                                            Identification Flavonoid Biosynthetic Mutant. Using <br />                                                                         reverse genetic screen described above, isolated a <br />   Of 153 mutant lines, 38 analyzed frequent reversion   putative knock-out mutant (line P90) gene correspond- <br /> the phenotype. A locus scored stable reversion         ing EST T76883. This EST displays 63% amino acid <br /> detected 1,000 offspring. <br />                                                                         sequence similarity functionally identified P. hybrida <br /> mutation test allelism. Second, A. thaliana sequences       FLS corresponds putative AtFLS gene (19). The <br /> to En-1 insertions isolated PCR mapped          original P90 plant homozygous insertion; 71 <br /> using RILs (loci named &#8220;B plus number&#65533;? Fig. 1). Third, map           offspring tested contained insertional allele. En-1 was <br /> positions derived genes containing En-1 insertions (see       inserted second intron gene (Fig. 2A), the <br /> below) locations known. Fig. 1 shows En-1               corresponding transcript barely detectable under <br /> insertions occurred chromosomes. The detection        light conditions inducing strong transcript accumulation the <br /> independent GL2 PIN mutants indicates high number              wild type (Fig. 2B). The homozygous mutant impaired <br /> insertions lower end chromosome I. Despite              strongly ability produce flavonol kaempferol (Fig. <br /> clusters, data indicate wide distribution insertion sites      2C). These results indicate loss-of-function mutant for <br /> over entire A. thaliana genome.                                     AtFLS obtained. <br />    Isolation Knock-Out Alleles. The En-1-mutagenized pop-               Because En-1-mutagenised population consists mul- <br /> ulation screened knock-out (null) alleles 109 loci,          ticopy lines, causal relationship observed <br />                                                                         phenotype recessive mutant particular insertion <br />                                                                         established. Phenotypic reversion wild type on <br />                                                                         excision En-1 locus provides proof. Five <br />                                                                         phenotypic revertants identified offspring (S1) <br />                                                                         original homozygous P90 plant. All heterozygous <br />                                                                         FLS locus; revertant alleles contained <br />                                                                         footprints, contained wild-type sequence (Fig. <br />                                                                         2 A). Fig. 2C shows phenotypic reversion plant 14. DNA <br />                                                                         gel-blot analysis offspring plant 14 revealed segre- <br />                                                                         gation ratio 9:13:5 (FLSyFLS: fls-1::EnyFLS: <br />                                                                         fls-1::Enyfls-1::En), similar expected 1:2:1 ratio. All <br />                                                                         analyzed S1 plants, including revertants offspring of <br />                                                                         plant 14, shown descendants originally <br />                                                                         isolated mutant plant; is, displayed P90-specific <br />                                                                         En-1 insertion pattern (data shown). These data demon- <br />                                                                         strate causal relationship presence En-1 the <br />                                                                         FLS locus flavonol-deficient phenotype line P90. <br />                                                                            Finally, activity enzyme encoded FLS locus <br />                                                                         verified biochemically. By using full-length cDNA, the <br />    FIG. 1. Distribution En-1 insertion sites A. thaliana      enzyme expressed E. coli. The reaction catalyzed FLS <br /> genome. The A. thaliana chromosomes depicted schematically.    illustrated Fig. 3A. The recombinant protein displayed <br /> Loci mapped En-1 insertions represented small squares.      FLS activity (Fig. 3B), assigning FLS function gene <br /> Each square marks independent event; open squares indicate           defined EST T76883. <br /> insertions forward genetic screens, closed squares stand      The TT6 Locus Encodes F3H. Results cross between <br /> for insertions PCR-based reverse genetic screens. Abbreviations    tt6 P90 demonstrated previous assumption that <br /> either taken classical genetic map follows: <br />                                                                         TT6 locus encodes FLS longer tenable: full <br /> AGL, agamous-like; B plus number, flanking DNA fragment; EXP, <br /> expansin; FPF, f lower promoting factor; HB, homeo box; KC,             complementation observed; is, F1 plants displayed <br /> K1(potassium)-channel; MYB, R2R3-MYB factor; PIN2, PIN <span id='am-67' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-68' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-70'  ></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>homo</span>-            wild-type phenotype terms seed color flavonol <br /> logue; SPL, squamosa promoter binding protein-like. Numbers iden-       content. In addition, analysis F2 cross between <br /> tify single genes gene family.                                     tt6 Col confirmed independent segregation tt <br />             Genetics: Weisshaar                                                                   Proc. Natl. Acad. Sci. USA 95 (1998)           12435 <br />  <br />   A <br />  <br />  <br />  <br />  <br />   B <br />  <br />  <br />  <br />  <br />       C <br />  <br />  <br />  <br />  <br />                                                                                   FIG. 3. Determination FLS enzyme activity. (A) Schematic <br />                                                                                 outline flavonoid biosynthesis. Chemical structures substrate <br />     FIG. 2. Interference En-1 insertion FLS gene expression.      (dihydroflavonol) product (flavonol) FLS reaction are <br /> (A) Partial structure fls-1::En allele. The data derived    shown. AS, anthocyanidin synthase; DFR, dihydroflavanone reduc- <br /> sequences various PCR products created En-1- FLS-specific           tase; CHS, chalcone synthase; text abbreviations other <br /> primers. Nucleotide positions given relative transcriptional start   enzyme names. (B) Product identification. The graphs represent <br /> site, determined 59 rapid amplification cDNA ends.              quantitations TLC analyses FLS activity assays using 14C-labeled <br /> Nucleotides corresponding 3-bp target site duplication under-        dihydrokaempferol (DHK) substrate; product kaempferol <br /> lined, arrow indicates orientation En-1 element. Below,      (K). Recombinant FLS expressed cells containing pQE60-FLS <br /> DNA sequences revertant alleles derived fls-1::En shown. All        (Lower) cells containing vector (Upper). Con- <br /> revertants exhibited wild-type phenotype. (B) Transcript accumulation        version rates given percent added substrate. The naringenin <br /> fls plants. Total RNA (10 mg) wild-type P90 plants subjected       detected contamination substrate dihydrokaempferol, <br /> to gel-blot analysis. Plants dark-adapted 64 h subsequently        produced naringenin. <br /> were irradiated UV-containing white light 24 h kept dark <br /> for time period. As controls, light-induced CHS transcript               In PCR-based reverse genetic screen En-1 insertions in <br /> accumulation determined wild type mutant, UBQ probe           F3H gene, plants detected produced <br /> was used demonstrate RNA integrity dark samples. (C) Flavonol             offspring tt (yellow) seeds. Additional f3h lines were <br /> levels descendants plant homozygous fls-1::En. Extracts      detected screening tt seeds (forward screens) and <br /> leaf material subjected TLC analysis using quercetin (Q)         subsequent molecular analysis F3H locus. The sequences <br /> kaempferol (K) standards Col wild-type extract control. <br /> Results shown plants 12 15. Plant 14 turned a <br />                                                                                 En-1 insertion alleles derived footprint alleles as <br /> heterozygous plant allele reverted wild type (compare          additional footprint allele depicted Fig. 4. TLC <br /> A).                                                                             analyses f3h knock-out lines detected flavonoids in <br />                                                                                 plants, consistent complete loss testa <br /> phenotype FLS-specific cleared amplified polymorphic                      pigments. Homozygous lines carrying f3h alleles did not <br /> system marker.                                                                  complement tt6 mutation test crosses, and, the <br />  12436      Genetics: Weisshaar                                                            Proc. Natl. Acad. Sci. USA 95 (1998) <br />  <br />                                                                            somatic reversion events. The unaltered quercetin level <br />                                                                            fls mutant indicate presence another, related <br />                                                                            enzyme higher affinity dihydroquercetin for <br />                                                                            dihydrokaempferol substrate. Previous low-stringency DNA <br />                                                                            gel-blot experiments provided evidence FLS-related <br />                                                                            gene A. thaliana genome (19). <br />                                                                               Seeds fls plants brown color indistinguishable from <br />                                                                            wild-type seeds, suggesting FLS required for <br />                                                                            pigment production testa. Condensed tannins and <br />                                                                            anthocyanins probably serve testa pigments (Fig. 3A). The <br />                                                                            wild type-like seed color fls plants explains this <br />                                                                            locus tt mutants (14, 32). <br />                                                                               Mapping data suggested TT6 does encode FLS. tt6 <br />                                                                            placed chromosome III (14) FLS <br />    FIG. 4. En-1-derived alleles causing F3H null phenotype           maps chromosome V (marker U1 23 centi- <br /> structure f3h-1(tt6) allele. Data derived sequences    morgans RIL map, ref. 19). TLC analyses tt6 <br /> of various PCR products created En-1- F3H-specific primers.       leaves revealed reduction FLS-derived flavonols and <br /> Nucleotide positions given relative F3H transcriptional start   dihydroflavanone reductase-derived anthocyanidins (data not <br /> site, determined 59 RACE. Nucleotides corresponding        shown). Similar reductions reported other <br /> the 3-bp target site duplication underlined, En-1 indicates        organs tt6 plants (15, 33). These data suggested that, tt6, <br /> orientation transposon, bold letters highlight differences      early enzyme flavonoid pathway (Fig. 3A) was <br /> from wild-type sequence.                                               affected. Mutants CHS (tt4) chalcone-f lavanone <br />                                                                            isomerase (CFI, tt5) established. F3H, <br /> above-mentioned F2 tt6 3 Col cross, pale brown tt6 <br />                                                                            encoding enzyme pathway, was, like tt6, <br /> phenotype co-segregated f3h locus (Table 2). Finally, <br />                                                                            mapped chromosome III (marker F3H 68 <br /> the f3h allele tt6 sequenced, single point <br />                                                                            centimorgans RIL map, ref. 26). Therefore, F3H <br /> mutation ORF detected, caused premature <br />                                                                            strong candidate gene mutated tt6. Genetic <br /> termination translation (Fig. 4). Taken together, data            complementation analysis sequencing tt6 f3h allele <br /> prove TT6 locus encodes F3H identify            yielded final proof TT6 locus encodes F3H. <br /> missing mutant early steps flavonoid biosynthesis.                 The different En-1-derived f3h alleles (Fig. 4) inde- <br />                                                                            pendent isolates respective lines displayed unre- <br />                           DISCUSSION                                       lated En-1 insertion patterns DNA gel-blot analyses (data <br />                                                                            shown). All f3h alleles caused tt phenotype very <br /> We demonstrate successful application PCR-based                   similar tt4. Such phenotype expected for <br /> screening strategy identification En-1-derived mu-              null mutation single-copy gene encoding early step in <br /> tations FLS F3H genes A. thaliana. The power              nonbranched flavonoid biosynthesis. The severe tt <br /> transposon mutagenesis autonomous elements A.                      phenotype f3h-4f f3h-5f alleles suggests the <br /> thaliana exploited forward reverse genetic                 frameshift mutations second F3H exon lead com- <br /> screens prove causal relationship mutant                    pletely nonfunctional F3H proteins point muta- <br /> genotype phenotype, confirm gene identity rever-                 tion tt6 leaky, allowing production small <br /> sion, create stable alleles, correlate DNA sequence              amounts functional enzyme. This interpretation sup- <br /> with biological function.                                                  ported results tt6 3 Col cross, not <br />    A. thaliana Mutants Flavonoid Biosynthesis. We isolated              indicate presence additional locus providing F3H <br /> an A. thaliana line carrying insertion En-1 FLS               activity. Because premature stop <span id='am-77' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-81'  ></span>codon</span> f3h&#8211;1(tt6) <br /> locus (allele fls-1::En). The mutant displays phenotype             allele causes shorter protein footprints, the <br /> proves vivo function FLS. En-1 integrated                 leakiness caused sort suppression the <br /> second intron FLS gene caused characteristic                  UAG stop codon. We observed slightly reduced level of <br /> three-bp target site duplication (10, 31). Although FLS                f3h mRNA light-induced tt6 comparison wild type <br /> ORF conserved 59 39 splice site sequences               (data shown), feature explained mRNA <br /> intron affected, FLS mRNA detect-                   destabilization premature stop codons (34). <br /> able. Thus, integration 8.3-kilobase En-1 intron               The classes allelic mutations isolated En-1 <br /> was apparently sufficient cause essentially null allele.              population complete set A. thaliana mutants first <br /> Absence mRNA caused premature termination                   phenylpropanoid metabolism. Mutants defective the <br /> of transcription defects mRNA maturation. The                early enzymes, including dihydroflavanone reductase <br /> low mRNA detected attributed sectors                   (tt3), known time (35, 36). In f3h lines, <br /> derived somatic reversion events. The detection wild             enzymatic step branch points either <br /> type-sized restriction fragment DNA gel-blot analysis               anthocyanidins flavonols blocked. As been <br /> the residual amounts kaempferol plants homozy-                 expected biochemistry flavonoid biosynthesis, this <br /> gous fls-1::En allele support assumption               mutant tt collection, leakiness of <br />                                                                            tt6 allele caused initial problems correct assign- <br /> Table 2.   Cosegregation tt6 phenotype CAPS marker           ment mutant. The fls-1::En allele provides access to <br /> for F3H                                                                    mutant specific flavonol branch pathway. This <br />                                                      Number plants      mutant important determining exact contribu- <br />    F3H CAPS             Wild type          tt            (ratio)           tion flavonols shield UV protective compounds <br />                                                                            (37, 38). <br /> ColyCol                     33             &#8211;               33 (1)             Suitability En-1 Mutant Isolation A. thaliana. Our <br /> ColyLer                     67             &#8211;               67 (2)          objective develop simple allows study <br /> LeryLer                     &#8211;             27               27 (1)          function principally gene A. thaliana. The <br /> Number plants                                                           success rate 50% recovery En-1 insertions genes <br />   (ratio)                 100 (3)        27 (1)                            analyzed expected basis calculated number <br />            Genetics: Weisshaar                                                             Proc. Natl. Acad. Sci. USA 95 (1998)            12437 <br />  <br /> of insertions population, indicating sensitivity      3.   Kunze, R., Saedler, H. &amp; Lo &#168;nnig, W.-E. (1997) Adv. Bot. Res. 27, <br /> the applied PCR screening technique sufficient. The                 331&#8211;469. <br /> population originated plants selected contain En-1          4.   Mena, M., Ambrose, B. A., Meeley, R. B., Briggs, S. P., Yanofsky, <br /> but original T-DNA. It likely                   M. F. &amp; Schmidt, R. J. (1996) Science 274, 1537&#8211;1540. <br />                                                                          5.   Koes, R., Souer, E., van Houwelingen, A., Mur, L., Spelt, C., <br /> observed clusters reflect insertion sites present in <br />                                                                               Quattrocchio, F., Wing, J., Oppedijk, B., Ahmed, S., Maes, T., et <br /> the generation, conserved transposition                al. (1995) Proc. Natl. Acad. Sci. USA 92, 8149&#8211;8153. <br /> linked sites (39). We estimate that, setup                 6.   McKinney, E. C., Aali, N., Traut, A., Feldmann, K. A., Belo- <br /> population, average 10% En-1 copies transposed                  stotsky, D. A., McDowell, J. M. &amp; Meagher, R. B. (1995) Plant <br /> generation, close agreement previously calculated                 J. 8, 613&#8211;622. <br /> excision frequency 8% (10). However, transposition                7.   Krysan, P. H., Young, J. C., Tax, F. &amp; Sussman, M. R. (1996) <br /> activity varied unpredictable manner generation                 Proc. Natl. Acad. Sci. USA 93, 8145&#8211;8150. <br /> generation. As consequence, single-seed descent                8.   van den Broeck, D., Maes, T., Sauer, M., Zethof, J., De Keuke- <br /> generations required generate new insertions, al-                     leire, P., D&#8217;Hauw, M., Van Montagu, M. &amp; Gerats, T. (1998) <br />                                                                               Plant J. 13, 121&#8211;129. <br /> though need large-scale crosses, selections, or <br />                                                                          9.   Pereira, A., Cuypers, H., Gierl, A., Schwarz-Sommer, Z. &amp; <br /> numerous transformations required two-element trans-                   Saedler, H. (1986) EMBO J. 5, 835&#8211;842. <br /> poson systems T-DNA insertion mutagenesis.                           10.   Cardon, G. H., Frey, M., Saedler, H. &amp; Gierl, A. (1993) Plant J. <br />    No obvious structural prerequisites specific En-1 inser-               3, 773&#8211;84. <br /> tions detected. En-1 inserted introns, exons,               11.   Bhatt, A. M., Lister, C., Crawford, N. &amp; Dean, C. (1998) Plant <br /> promoter regions downstream coding regions                     Cell 10, 427&#8211;434. <br /> orientations relative target gene. The exceptions           12.   Hahlbrock, K. &amp; Scheel, D. (1989) Annu. Rev. Plant Physiol. Plant <br /> from random insertions obtained                    Mol. Biol. 40, 347&#8211;369. <br /> independent f3h alleles containing En-1 insertions          13.   Weisshaar, B. &amp; Jenkins, G. I. (1998) Curr. Opin. Plant Biol. 1, <br />                                                                               251&#8211;257. <br /> site, opposite orientation. More data required <br />                                                                         14.   Koornneef, M. (1990) Arabidopsis Information Services 27, 1&#8211;4. <br /> to determine independent transpositions             15.   Shirley, B. W., Kubasek, W. L., Storz, G., Bruggemann, E., <br /> same site reversion leaving identical footprint                    Koornneef, M., Ausubel, F. M. &amp; Goodman, H. M. (1995) Plant <br /> occurred just chance result sequence-dependent                  J. 8, 659&#8211;671. <br /> insertionyexcision.                                                     16.   Feinbaum, R. L. &amp; Ausubel, F. M. (1988) Mol. Cell. Biol. 8, <br />    The moderate activity En-1 ensure unstable                1985&#8211;1992. <br /> mutants analyzed easily. On hand, En-1          17.   Hartmann, U., Valentine, W. J., Christie, J. M., Hays, J., Jenkins, <br /> mobility sufficiently high facilitate establishment              G. I. &amp; Weisshaar, B. (1998) Plant Mol. Biol. 36, 741&#8211;754. <br /> causal relationships phenotype affected             18.   Holton, T. A., Brugliera, F. &amp; Tanaka, Y. (1993) Plant J. 4, <br />                                                                               1003&#8211;1010. <br /> gene isolation wild-type revertants (germinal <br />                                                                         19.   Pelletier, M. K., Murrell, J. R. &amp; Shirley, B. W. (1997 <br /> </body></html>