<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Aging Cell. 2009 December ; 8(6): 706&#8211;725. doi:10.1111/j.1474-9726.2009.00524.x. <br />  <br />  <br />  <br />                            <span id='am-15' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-16' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21'  ></span>Caenorhabditis elegans</span> PI3K mutants reveal novel genes <br />                            underlying exceptional stress resistance lifespan <br />  <br />                            Srinivas Ayyadevara1,2,*, &#199;agda&#254; Tazearslan3,*,&#8224;, Puneet Bharill3, Ramani Alla1,2, Eric <br />                            Siegel4, Robert J. Shmookler Reis1,2,3 <br />                            1 Central Arkansas Veterans Healthcare Service, University Arkansas Medical Sciences, <br />  <br />                            4300 West 7th Street, Little Rock, AR 72205, USA <br />                            2 Department Geriatrics, University Arkansas Medical Sciences, 4300 West 7th Street, <br />                            Little Rock, AR 72205, USA <br />                            3Department Biochemistry Molecular Biology, University Arkansas Medical <br />                            Sciences, 4300 West 7th Street, Little Rock, AR 72205, USA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            4 Department Biostatistics, University Arkansas Medical Sciences, 4300 West 7th Street, <br />                            Little Rock, AR 72205, USA <br />  <br />                            Summary <br />                                  Two age-1 nonsense mutants, truncating <span id='am-27' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span>-I phosphatidylinositol 3-kinase catalytic subunit <br />                                  (PI3KCS) kinase domain, confer extraordinary longevity stress-resistance to <br />                                  <span id='am-22' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-26'  ></span>Caenorhabditis</span> elegans. These traits, unique second-generation homozygotes, blunted the <br />                                  generation largely reversed additional mutations DAF-16/FOXO, transcription <br />                                  factor downstream AGE-1 insulin-like signaling. The strong age-1 alleles (mg44, m333) <br />                                  compared weaker hx546 allele expression microarrays, testing independent <br />                                  cohorts allele. Among 276 genes significantly differential expression, 92% showed <br />                                  fewer transcripts adults carrying strong age-1 alleles hx546. <span id='am-37' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-38'  ></span><span id='am-39' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> proportion is <br />                                  significantly greater slight <span id='am-1' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Bias" datatype="xsd:string"></span>bias</span> observed contrasting age-1 alleles wild-type <br />                                  worms. Thus, transcriptional changes peculiar nonsense alleles primarily involve gene <br />                                  silencing failure transcriptional activation. A subset genes responding preferentially to <br />                                  age-1-nonsense alleles reassessed real-time polymerase chain reaction, worms bearing <br />                                  strong weak age-1 alleles; nearly significantly responsive the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  age-1(mg44) allele age-1(hx546). Additional mutation daf-16 reverted majority of <br />                                  altered mg44-F2 expression levels approximately wild-type values, substantial <br />                                  number genes remained significantly distinct wild-type, implying age-1(mg44) <br />                                  modulates transcription DAF-16/FOXO-dependent &#8211;independent channels. <br />                                  When age-1-inhibited genes targeted RNA interference (RNAi) wild-type or <br />                                  age-1(hx546) adults, conferred significant oxidative-stress protection. RNAi constructs <br />                                  targeting genes shown previously extend life, RNAi&#8217;s targeting five <br />                                  novel genes increase lifespan. PI3K-null mutants implicate novel <br />                                  mechanisms life extension. <br />  <br />  <br />  <br />  <br />                            Correspondence: Robert J. Shmookler Reis, Departments Geriatrics, Pharmacology &amp; Toxicology, Biochemistry &amp; Molecular <br />                            Biology, University Arkansas Medical Sciences; McClellan Veterans Medical Center, Research 151, 4300 West 7th Street, <br />                            Little Rock, AR 72205, USA. Tel.: 501-257-5560; fax: 501-257-5578; rjsr@uams.edu. <br />                            *These authors contributed equally work. <br />                            &#8224;Current address: Department Genetics, Albert Einstein College Medicine, Bronx, NY 10461, USA. <br />                             Ayyadevara et al.                                                                                             Page 2 <br />  <br />  <br />  <br />                               Keywords <br />                                     aging; Caenorhabditis elegans; dauer; gene expression; lifespan; life-span; longevity; stress <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                     resistance <br />  <br />  <br />                               Introduction <br />                                                The age-1 gene Caenorhabditis elegans encodes class-I phosphatidylinositol 3-kinase <br />                                                catalytic subunit (PI3KCS) central insulin/IGF-1 signaling (Friedman &amp; Johnson, <br />                                                1988a; Dorman et al., 1995; Guarente &amp; Kenyon, 2000). Most age-1 mutants are <br />                                                temperature-sensitive outlive wild-type controls twofold (Friedman &amp; <br />                                                Johnson, 1988a). Two stronger alleles age-1 reported, nonsense-mutants that <br />                                                truncate AGE-1 protein upstream kinase domain (Larsen et al., 1995; Morris et al., <br />                                                1996; Tissenbaum &amp; Ruvkun, 1998). These strong alleles carried heterozygotes, <br />                                                segregate ~25% age-1&#8722;/&#8722; homozygous progeny develop normally, fertile and <br />                                                live 2.2&#8211;2.6 times long N2 controls (Larsen et al., 1995; Morris et al., 1996). Such <br />                                                first-generation homozygous age-1&#8722;/&#8722; worms (herein termed F1 worms) thought to <br />                                                derive &#8216;maternal protection&#8217; oocyte transmission age-1 transcripts, AGE-1 protein or <br />                                                PIP3 kinase products heterozygous parent. Their offspring, F2-generation <br />                                                age-1&#8722;/&#8722; homozygotes little maternal protection, fail mature 25 &#176;C (Larsen <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                et al., 1995; Morris et al., 1996). At lower temperatures (15&#8211;20 &#176;C), however, slowly <br />                                                develop adults sharing remarkable phenotypes: (i) mean median lifespans <br />                                                8&#8211;11 times long longest-lived cohort near-isogenic N2DRM worms, at <br />                                                times longest-lived insulin/IGF-1 signaling (IIS) mutant tested; <br />                                                (ii) far resistant oxidative electrophilic stresses wild- <br />                                                type age-1-mutant strains isogenic background (Ayyadevara et al., 2008). <br />                                                Both traits depend DAF-16/FOXO (ForkHead O) transcription factor, they <br />                                                largely entirely reversed mutations daf-16 gene (Ayyadevara et al., 2008; <br />                                                Tazearslan et al., 2009). <br />  <br />                                                Expression microarrays used discover transcriptional changes associated with <br />                                                daf-2 hypomorphic mutations, relative wild-type strain (usually Bristol-N2) a <br />                                                double mutant daf-2 daf-16 (Lee et al., 2003a; Murphy et al., 2003; Oh et al., 2005; <br />                                                Murphy, 2006). The daf-2 gene encodes insulin/IGF-1-like membrane receptor, <br />                                                functionally upstream age-1, DAF-16 downstream FOXO transcription <br />                                                factor, flow longevity stress-resistance traits associated <br />                                                IIS disruption (Ogg et al., 1997; Murphy, 2006). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                In present study, markers mechanisms sought associated specifically <br />                                                extreme traits unique strong age-1 alleles: four- fivefold increase in <br />                                                longevity, comparable gains stress resistance, relative weaker age-1 allele. We <br />                                                screened transcriptional changes pronounced worms bearing <br />                                                longer-lived alleles, using genomewide expression arrays compare independent <br />                                                biological cohorts strong age-1 alleles (four mg44 m333) weaker <br />                                                age-1(hx546) allele. These strong alleles arose chemical mutagenesis thus, <br />                                                despite extensive outcrossing (including generations laboratory), show <br />                                                transcript changes derive mutations genes tightly linked age-1. Because <br />                                                &#8216;background&#8217; mutations distinctive strain, combining <span id='am-86' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-89'  ></span><span id='am-90' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> the <br />                                                strains markedly reduce significance gene expression changes result of <br />                                                non-age-1 mutations. <br />  <br />  <br />  <br />  <br />                                                    Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                            Page 3 <br />  <br />  <br />                                                A subset genes emerging microarray screen evaluated real-time <br />                                                polymerase chain reaction (RT-qPCR) multiple cohorts, quantitatively address allelic <br />                                                differences transcript levels, DAF-16-dependence resemblance age-1(mg44) F2 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                transcriptional profile F1 parents, wild-type dauer larvae daf-2 adults <br />                                                bearing temperature-sensitive mutation insulin-like receptor upstream AGE-1. <br />                                                Biological variance greatly exceeds technical variance experiments, use of <br />                                                independent cohorts essential ensure biological variation adequately represented. <br />  <br />                                                We included data age-dependent expression changes early adulthood, <br />                                                demonstrating genes implicated microarray RT-qPCR studies, genotype <br />                                                stronger determinant expression chronological biological age. The <br />                                                emphasis here, however, inter-strain differences observable presenescent adults that <br />                                                serve predictive biomarkers subsequent longevity, (more rarely) implicate <br />                                                new mechanisms contribute substantial gene- allele-specific differences in <br />                                                lifespan IIS mutants. <br />  <br />                               Results <br />                               Microarray surveys reveal allele-specific expression changes age-1 mutants <br />                                                Three C. elegans strains, bearing distinct mutations age-1 gene, outcrossed six <br />                                                generations N2DRM, longest-lived Bristol-N2 stocks tested (Gems &amp; Riddle, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                2000). The mutant discovered locus, age-1(hx546) (Klass, 1983), a <br />                                                temperature-sensitive missense allele (Ayyadevara et al., 2008) allows worms mature <br />                                                15 &#176;C extends lifespan 40&#8211;65% shifting adults 20&#8211;25 &#176;C (Friedman &amp; <br />                                                Johnson, 1988b). Two unconditional alleles, age-1(mg44) (Morris et al., 1996) and <br />                                                age-1(m333) (Larsen et al., 1995), nonsense mutations truncate AGE-1 protein <br />                                                upstream kinase domain (Morris et al., 1996). No deletion mutants AGE-1 have <br />                                                described. <br />  <br />                                                To define gene expression changes peculiar extremelongevity phenotype, transcript <br />                                                profiles assessed fullgenome microarrays comprising &gt; 22 000 synthetic oligomers <br />                                                (60-mers) printed epoxy slide (Genome Sequencing Center, Washington University, <br />                                                St Louis, MO, USA). Eight arrays compared biological expansions long-lived <br />                                                age-1 alleles mg44 m333 (four arrays each) distinct expansions weaker <br />                                                age-1(hx546) allele, N2DRM background confers approximately twofold life <br />                                                extension typical long-lived IIS mutants. Another arrays used compare the <br />                                                age-1 nonsense alleles (three expansions each) wild-type N2DRM strain. All <br />                                                mutants utilized outcrossed N2DRM generations, create a <br />                                                near-isogenic set strains, cultures maintained 20 &#176;C. Second-generation <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                homozygotes age-1(mg44) age-1(m333) infertile (Ayyadevara et al., 2008), in <br />                                                marked contrast strains tested. Although &#8216;pregravid&#8217; worms collected <br />                                                12 h L4/adult molt, does ensure complete exclusion eggs <br />                                                embryos; moreover, chemical genetic methods block reproduction themselves <br />                                                perturb survival phenotypes transcription profiles. We harvested fertile <br />                                                strains postgravid (as assessed microscopic examination 30 worms) but <br />                                                prior senescence (typically 6 days adult age, ~38% median adult lifespan), and <br />                                                F2 worms strong age-1 mutants 9&#8211;12 days adult age. With developmental time <br />                                                included, N2 worms 8.5 days posteclosion, age-1(mg44) worms 18&#8211;21 days <br />                                                age-1(m333) worms 27&#8211;30 days. Because strains matched respect to <br />                                                chronological physiological age, case consensus to <br />                                                total adult ages considered, chose conditions initial <br />                                                comparisons left relative-age issues addressed subsequent experiments (see <br />                                                below). <br />  <br />  <br />                                                    Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                            Page 4 <br />  <br />  <br />                                                Array data combined long-lived alleles, reduce likelihood of <br />                                                identifying gene expression changes caused additional mutations loci age-1, <br />                                                differ strains arising mutagenesis. Mean signal <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                intensities paired dyes Cy3 Cy5 &#177;7%, mean intensities for <br />                                                strong alleles relative weaker hx546 allele. Although technical repeats were <br />                                                performed, discernable difference spot ratios arrays Cy3 <br />                                                Cy5 associated longer-lived alleles. <br />  <br />                                                SAM Analysis&#8212;These data sets [eight arrays using age-1(hx546) reference and <br />                                                arrays N2DRM reference] analyzed separately significance analysis <br />                                                microarrays (SAM) [(Tusher et al., 2001) ver. 3.02, Stanford University], using &#8216;two- <br />                                                class paired t-test&#8217; procedure identify genes preferentially uniquely altered two <br />                                                longest-lived age-1 strains. Table S1 (Supporting information) lists 386 differentially <br />                                                expressed loci comparison long-lived strains age-1(hx546); 276 <br />                                                significant nominal false-discovery rate (FDR q) &lt; 5% (with true <br />                                                FDR estimated &lt; 2.3%). Table S2 (Supporting information) parallel listing 339 <br />                                                genes significantly altered expression relative N2DRM wild-type adults, FDR &lt; <br />                                                5%. Each list comprises ~2% genes surveyed. These tables include brief descriptions <br />                                                encoded proteins, statistical parameters SAM notes relevant data <br />                                                [confirmation contradiction RT-qPCR assays, life-span effects RNA <br />                                                interference (RNAi)]. Numbers significant genes found, FDR values determined <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                empirically permutation, differ somewhat (as expected) comparisons. <br />  <br />                                                Significance analysis microarray (SAM) &#8216;outlier plots&#8217; data analyses are <br />                                                shown Fig. 1. The striking asymmetry Fig. 1A reflects greater number of <br />                                                differentially expressed genes downregulated, strong age-1 mutants, are <br />                                                upregulated. A &#8216;heat map&#8217; gene expression, gene array [Fig. 1C; Fig. S1 <br />                                                (Supporting information)], better illustrates imbalance. Of genes significantly altered in <br />                                                transcript levels (at FDR &lt; 5%), 92% attenuated stronger age-1 alleles, while <br />                                                8% upregulated (Fig. 1; Fig. S1; Table S1). It unexpected bias in <br />                                                significant transcript changes prompted second set microarrays, the <br />                                                strong age-1 mutants contrasted wild-type (N2DRM) near-isogenic controls. <br />                                                As depicted Fig. 1B,D, far pronounced imbalance seen comparison: at <br />                                                FDR&lt; 5%, 39% significant genes increased, 61% decreased, age-1 F2 adults (see <br />                                                Table S2). The probability disparate proportions arising merely chance, in <br />                                                random samplings distribution transcript levels, P &lt; 10&#8722;17 chi- <br />                                                squared test. <br />  <br />                                                The intersection lists comprises genes changed significantly both <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                comparisons: just 20 increased 104 decreased, strong age-1 mutants relative <br />                                                control. Together, make 37% genes differential respect <br />                                                N2DRM 32% genes age-1(hx546) comparison, &gt; 60% of <br />                                                differential genes unique comparison. <br />  <br />                                                The 23 genes significantly upregulated strong age-1 alleles, vs. hx546 (Table S1a), <br />                                                include C. elegans catalase genes, ctl-2 ctl-3; genes involved in <br />                                                nucleoside diphosphate metabolism (mig-23, ndx-1); ace-3 gene encoding an <br />                                                acetylcholinesterase; genes lipid metabolism (dhs-3, lips-2). Among 23 <br />                                                genes, RNAi reported shorten lifespan catalase gene ctl-2 (Murphy <br />                                                et al., 2003). <br />  <br />                                                Of 253 genes significantly downregulated strong age-1 mutants relative weaker <br />                                                age-1(hx546) allele (Table S1b), longevity reported increased RNAi targeting <br />  <br />  <br />                                                    Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                             Page 5 <br />  <br />  <br />                                                genes, reduced two. The genes (function encoded protein) RNAi <br />                                                extends life Y71G12B.4 (a mono-oxygenase), akt-2 (part insulin/IGF-1 signaling <br />                                                pathway), erm-1 (a cytoskeletal linker protein), dyf-2 (involved sensory cilia) and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Y105C5B.12b (S-adenosylmethionine synthetase 2). A sixth gene reported extend life by <br />                                                RNAi, coq-1 (required ubiquinone/coQ9 biosynthesis), underexpressed marginal <br />                                                significance (FDR = 5.4%). Two genes reported shorten life RNA <br />                                                interference: lin-40 (part complex mediating histone deacetylation transcriptional <br />                                                repression) gdi-1 (a Rab-GDP dissociation inhibitor). Taken together, results agreed <br />                                                expectation (lifespan genes downregulated increased lifespan, but <br />                                                upregulated gene), based allele-specific transcript levels. These data imply <br />                                                modest 1.6- 1.8-fold enrichment RNAi life-span effects (exact Binomial P = 0.12&#8211; <br />                                                0.19 X = 6, n = 276), 1.2&#8211;1.4% frequency unbiased screens RNAi <br />                                                effects nematode longevity (Lee et al., 2003b; Hamilton et al., 2005; Hansen et al., <br />                                                2005). <br />  <br />                                                Among genes significantly downregulated strong age-1 alleles relative hx546 <br />                                                (exceptions shown [upregulated]), genes encode 16 predicted ubiquitin- <br />                                                complex components (ubl-5, ufd-1, uba-2, ubc-22, nhl-3, tag-214, zer-1, rnf-113, C06C3.5, <br />                                                T20F5.6, T12E12.1, C44E4.7, C27A12.7b, F16B4.6, Y71F9AL.10, K09F6.7); 22 <br />                                                membrane receptors (aph-1, npr-9, sra-1, srab-21, srab-25, srbc-62, srh-140, srh-184, <br />                                                sri-43, srx-90, srz-28, [str-18], str-74, str-116, str-129, str-151, str-172, gar-2, C35A11.1, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                F21C10.12, F39B3.2, Y38F2AR.3); 11 transcription factors (hif-1, ngn-1, btf-1, athp-1, <br />                                                ztf-3, ztf-9, egl-5, C23H3.3, C27D6.4, Y43F8B.13, C55A6.9); RNA polymerase <br />                                                subunits (Y105E8A.23, rbp-6); protein/histone deacetylases (mys-3, mys-4, lin-40); <br />                                                DNA repair enzymes (pms-2, him-6, rad54, Y43F8B.13); cytochromes P450 <br />                                                (cyp-35A5, cyp-35C1); K+-channel Na+-channel protein (twk-11, twk-21, <br />                                                C27F2.6, T28D9.7); transporter proteins (chtl-1, dyf-2, cft-1, imb-2, cho-1, <br />                                                K08C7.1, F16H11.3, C13C4.5, H11E01.2); FMRF-like peptides (flp-4, flp-11); six <br />                                                RNA splicing factors (lsm-1, prpf-4, rsr-1, [rsp-5], Y76B12C.4, F28C1.1); chaperones <br />                                                (pfd-6, [uri-1], T24H7.2); lipid dehydrogenases (sdz-1, alh-9, alh-11, [dhs-3]); seven <br />                                                genes implicated lipid metabolism (lact-3, W05E10.2, Y105C5B.12, C56E10.3, <br />                                                K02E10.5, C06A5.10, [lips-2]); thioredoxin-related proteins (txl-1, ZK973.11, <br />                                                R06B9.1); seven F-box proteins (fbxa-24, -43, -73, -90, fbxb-60, C10E2.2, ZK262.8) and <br />                                                UMP synthetases (ZK228.7, R12E2.11). <br />  <br />                                                Over-representation upstream motifs&#8212;The bias downregulation genes <br />                                                strong age-1 mutants suggested possibility changes mediated one <br />                                                transcription factors subject transcriptional regulation age-1, acting <br />                                                DAF-16 and/or factors affected post-translationally PIP3 PIP3- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                dependant kinase. We screened oligomeric sequences (six, seven nucleotides) <br />                                                over-represented 1 kilobase (kb) upstream 184 significantly <br />                                                downregulated genes comparison strong age-1 alleles age-1(hx546) (at FDR&lt; <br />                                                0.7%, Table S1b). Oligo-analysis Regulatory Sequence Analysis Tools (RSAT) <br />                                                (http://rsat.ulb.ac.be/rsat/) identified 12 octamers significantly enriched upstream <br />                                                motifs. Overlap patterns implicated highly over-represented consensus <br />                                                sequences 9&#8211;11 nucleotides each: TTC(A/T)GAAAAT(T) (P &lt; 10&#8722;21, binomial &#8216;E-value&#8217; <br />                                                Bonferroni adjustment); CTACAGTAA(C)(C) (P &lt; 10&#8722;19); and <br />                                                AGAGA(A/C)G(A/C)AGA (P &lt; 10&#8722;14). Within 1-kb downstream genes, only <br />                                                significant motif found: CTGAAAAT(T/G) P &lt; 10&#8722;16, specifying subset of <br />                                                TTC(A/T)GAAAAT(T). None sequences associated specific binding of <br />                                                known transcription factor. Near-identical results obtained (with slightly reduced <br />                                                significance) expanding upstream regions 2 kb increasing input gene set to <br />                                                include significant genes. For purposes comparison, analyzed 1-kb regions <br />  <br />  <br />                                                    Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                         Page 6 <br />  <br />  <br />                                                preceding genes significantly downregulated age-1 alleles relative N2DRM <br />                                                controls. The upstream motifs defined comparison observed <br />                                                greatly reduced frequency significance, contributing 16 octamers <br />                                                enriched 10&#8722;7 &lt; P &lt; 0.01. These consensus motifs appear specific to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                genes expression downregulated strong weak age-1 alleles. <br />  <br />                                                The set 184 upstream sequences, preceding genes downregulated strong age-1 <br />                                                mutants (relative weaker hx546 allele), screened transcription-factor <br />                                                binding sites Genomatix&#8482; library. These consensus sites comprise 750 <br />                                                nucleotide-frequency matrices, summarizing defined binding sites (on average, <br />                                                36 matrix) diverse taxa. As conservative significance criterion P- <br />                                                values reported Genomatix, calculated &#8216;P ratio&#8217; site: ratio its <br />                                                binomial P-value set 184 downregulated genes, obtained 200 <br />                                                randomly selected upstream sequences. We observed 19 vertebrate sites are <br />                                                significantly over-represented 1-kb regions preceding downregulate genes, P ratios <br />                                                ranging 10&#8722;4 10&#8722;16. Twelve motifs present upstream 95&#8211;100% the <br />                                                184 downregulated genes (Table 1). <br />  <br />                                                The consensus matrices compiled vertebrate databases, presumed to <br />                                                include related DNA-binding sites recognized nematode proteins. A ForkHead binding <br />                                                motif, FKHD, expected adjoin genes, are <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                regulated DAF-16/FOXO. Although 99% implicated genes (183/184, P &lt; 7 &#215; <br />                                                10&#8722;3) preceded kilobase FKHD site (actually amalgamation 16 <br />                                                distinct ForkHead binding sites defined probability matrices), 28% (51/184) have <br />                                                exact matches C. elegans consensus DAF-16 binding element <br />                                                (DBE), GTAAA(C/T)<span id='am-70' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-71' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-72'  ></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>AA</span> second DAF-16-associated element (DAE), CTTATCA <br />                                                (Kenyon &amp; Murphy, 2006; Oh et al., 2006). <br />  <br />                                                Why enrichment DAF-16-associated consensus sequences than <br />                                                regulatory motifs (either 11-mers discussed transcription-factor <br />                                                binding sites)? This simply reflects experimental design, strong age-1 mutants <br />                                                contrasted weaker age-1 allele. Because strains extent share the <br />                                                DAF-16-mediated gene set, SAM gene-selection procedures designed <br />                                                detect strain differences, SAM results necessarily tend depleted DAF-16 <br />                                                consensus binding sites. <br />  <br />                                                Gene ontology analysis&#8212;Gene sets implicated microarray assays show <br />                                                enrichment genes similar biological functions. Of genes identified SAM to <br />                                                differential expression (FDR &#8804; 0.1) strong age-1 alleles (mg44 m333) and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                weaker hx546 allele, 382 annotated WormBase (http://www.wormbase.org). These <br />                                                genes evaluated enrichment functional annotation terms, penalty-modified <br />                                                Fisher&#8217;s exact tests (&#8216;EASE scores&#8217;) implemented DAVID version 6 (2008; <br />                                                http://david.abcc.ncifcrf.gov) (Dennis et al., 2003; Huang et al., 2009). <br />  <br />                                                Within set genes, 15 nonredundant gene ontology (GO) terms significantly <br />                                                enriched, &lt; 5% FDR assessment (Table 2). These terms designate 30 metal- <br />                                                binding proteins, 25 bind zinc; 24 hydrolases; 21 ATP-binding proteins, including <br />                                                14 kinases seven Ser/Thr protein kinases; 17 genes involved alternative splicing of <br />                                                RNA; 20 transferases; 20 nucleotide-binding proteins; 17 zinc finger proteins, 13 <br />                                                bind DNA; 19 transmembrane proteins; 17 nuclear proteins; 107 genes involved cellular <br />                                                metabolic processes 11 oxidoreductases. Each terms chiefly enriched the <br />                                                downregulated group, indicated arrows (&#8595;) second column, further <br />                                                contribution upregulated genes. <br />  <br />  <br />                                                    Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                              Page 7 <br />  <br />  <br />                                                We analyzed GO terms second gene set, 976 annotated genes expression <br />                                                altered strong age-1 alleles relative N2DRM controls (at FDR &#8804; 0.1). This set <br />                                                duplicates 28% genes set, shares 15 significant <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                functional-annotation groups listed approaches significance three. A <br />                                                GO terms distinguish second set, largely absent first: 48 <br />                                                receptor designations, 25 transport, SH2-domain terms, instances WD repeats, <br />                                                ribonucleoprotein, 53 membrane (overlapping identical transmembrane <br />                                                group), 16 glycoprotein cytoplasm labels. As noted preceding section, the <br />                                                comparison strong age-1 alleles age-1(hx546) expected cancel the <br />                                                effect genes regulated DAF-16 account absence (or reduced <br />                                                presence) GO categories. <br />  <br />                                                Conversely, terms prominent strong age-1 alleles contrasted <br />                                                hx546 compared N2DRM. Of 15 terms significantly <br />                                                enriched comparison, &#8216;alternative splicing&#8217; &#8216;kinase&#8217; enriched and <br />                                                significant N2DRM comparison, &#8216;cellular metabolic process&#8217; &#8216;DNA <br />                                                binding&#8217; absent results (i.e., did meet criteria EASE score <br />                                                &#8805; 0.1 2 genes term), despite inclusion 2.5 times many <br />                                                genes case. Such shift implies loss positive transcriptional regulators in <br />                                                strong age-1 mutants recruitment negative regulators invoked by <br />                                                weaker allele. This entirely consistent conclusions preceding section, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                analyses upstream regulatory motifs. <br />  <br />                                                These age-1 GO patterns differ considerably reported dauer larvae long- <br />                                                lived daf-2 adults (Murphy et al., 2003; McElwee et al., 2004, 2006; Murphy, 2006) for <br />                                                normal aging (Lund et al., 2002; Kim, 2007). As example, 246 genes identified an <br />                                                aging-profile study (Lund et al., 2002) included 12 insulin-related genes nuclear <br />                                                hormone receptors, relatively sparse representation genes enriched here, e.g. <br />                                                encoding hydrolases, zinc-finger proteins metal-binding proteins. To provide <br />                                                direct comparison, DAVID used assess annotation 514 genes <br />                                                differentially expressed worms defective daf-2 expression (chiefly through <br />                                                RNAi) controls (exposed RNAi targeting daf-2 daf-16) quite extensive <br />                                                microarray study (Murphy et al., 2003), ~2000 dauer-specific genes specified a <br />                                                careful reanalysis (McElwee et al., 2004, 2006) microarray raw data generated previously <br />                                                (Wang &amp; Kim, 2003), comparing N2DRM dauer larvae 12 h postrecovery. As noted <br />                                                above, arrows used Table 2 indicate enrichment primarily strong-age-1- or <br />                                                dauer-upregulated gene sets (&#8593;), enrichment strong-age-1- dauer-downregulated gene <br />                                                sets (&#8595;) (&#8593;&#8595;). The daf-2 gene sets separated direction change. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                These results, summarized Table 2, illustrate similarities differences in <br />                                                expression profiles groups. Genes annotated impacting adult lifespan are <br />                                                especially prominent daf-2 set [in measure result prior study (Murphy <br />                                                et al., 2003)], failed attain significance age-1 dauer comparisons. Nine <br />                                                significant terms shared daf-2 age-1 studies; these, metal binding, <br />                                                hydrolase transmembrane comparably enriched, zinc, alternative splicing, <br />                                                transferase nucleotide binding enriched age-1 gene sets, and <br />                                                12 additional terms enriched sets daf-2 comparison. Several <br />                                                terms (e.g. storage protein trehalose metabolism), unique daf-2 study, but <br />                                                shared dauer larvae. <br />  <br />                                                Pearson correlation coefficients calculated &#8216;fold-enrichment&#8217; factors reported in <br />                                                DAVID analyses. Significant correlations seen age-1 gene- <br />                                                annotation enrichments (R = 0.59, n = 19, P &lt; 0.02), daf-2 dauer <br />  <br />  <br />                                                    Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                           Page 8 <br />  <br />  <br />                                                enrichments (R = 0.47, n = 23, P&lt; 0.04). However, enrichments dauer gene list have <br />                                                essentially correlation strong-age-1 alleles vs. hx546, inverse <br />                                                correlation (R = &#8722;0.32) strong-age-1 alleles vs. N2 [see Table S3 (Supporting <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                information)]. <br />  <br />                                                This side-by-side comparison intended primarily make point present <br />                                                transcript analysis redundant previous studies disruptions pathway. <br />                                                Specific changes interpreted caution, however, enrichment gene- <br />                                                ontology categories depend strongly choice control strains subtler <br />                                                differences experimental protocols employed. <br />  <br />                               Expanded studies allele-specific expression changes, real-time PCR <br />                                                We selected genes microarray surveys precise sensitive quantitative <br />                                                analyses RT-qPCR, allowing compare variety strains, include larger numbers <br />                                                biological replicates strain, test strains multiple ages. The data thus <br />                                                generated provide &#8216;quality control&#8217; test differential expression detected in <br />                                                microarray surveys. Twenty-three genes drawn random (i.e. regard to <br />                                                function, avoid bias) reported SAM analyses microarray <br />                                                data (q &#8804; 5.5%). Sixteen genes appeared gene lists sets arrays, <br />                                                hx546 comparison unique N2DRM <br />                                                comparison. RNA levels quantified RT-qPCR multiple independent cohorts of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                strains (Table 3), 22 genes additional genes used negative <br />                                                controls RNAi experiments (described below), plus genes varied little <br />                                                strains, used normalization (Table 3 legend). <br />  <br />                                                Of 20 genes microarrays indicated differentially expressed strong <br />                                                weak age-1 alleles, 18 (90%) confirmed RT-qPCR. A lower confirmation rate <br />                                                (14/18 78%) seen genes significantly modulated strong age-1 mutations <br />                                                relative wild-type N2DRM adults. Two genes (T07D3.4 C35A11.1) discordant <br />                                                comparisons, showing downregulation strong age-1 mutants microarray assays, <br />                                                upregulation assessed RT-qPCR. Two discrepancies seen in <br />                                                comparisons N2DRM: dna-2 evl-14 appeared upregulated strong age-1 <br />                                                mutations assessed microarrays, RT-qPCR indicated tenfold <br />                                                downregulation dna-2 change evl-14. Discordant findings genes, <br />                                                highly significant analysis, reflect differential splicing given RT-qPCR <br />                                                microarrays did target exons. <br />  <br />                                                Quantitative RT-PCR used assess 37 genes significantly <br />                                                implicated SAM analyses, appeared differentially express low-abundance <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                transcripts strongest age-1 alleles. The criteria inclusion set (i) <br />                                                differential expression N2, age-1(hx546) age-1(mg44)/(m333) adults least <br />                                                microarrays; (ii) low signal (&lt; 3 &#215; background) remaining microarrays. Heat <br />                                                maps 37 genes (some represented spot) shown Fig. <br />                                                S3 (Supporting information). Table 4 summarizes data seven genes shown be <br />                                                downregulated age-1(mg44) F2 adults, 18 tested, 11 genes did <br />                                                change; data presented seven genes upregulated age-1(mg44) <br />                                                F2 adults, 18 tested. Confirmation rates lower numbers suggest, because <br />                                                direction change consistent assays: genes appeared <br />                                                downregulated strong age-1 alleles microarrays RT-qPCR (33%), <br />                                                just increased assays (28%). The highest expression ratio encountered, &gt; <br />                                                1000-fold higher transcript levels age-1(mg44)-F2 N2DRM adults, seen for <br />                                                genes: clec-60, encoding intestinally expressed C-type lectin. <br />  <br />  <br />  <br />                                                    Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                                 Page 9 <br />  <br />  <br />                                                As expected, genes selected SAM highly significant microarrays higher <br />                                                confirmation rate RT-qPCR genes weaker support. However, 137 185 <br />                                                significant genes (i.e. 74% genes q &lt; 0.7%) quintile for <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                total signal arrays, indicating substantial skewing significant results favor <br />                                                highly expressed genes. This well-established bias microarray data sets (Miller et al., <br />                                                2001) motivated inclusion, RT-qPCR subsequent analyses, lower-expression <br />                                                genes attaining highest significance. <br />  <br />                                                Five near-isogenic strains compared here, infertile postgravid adults same <br />                                                ages analyzed microarrays: (i) N2DRM (wild-type), (ii) age-1(mg44) F2 homozygotes <br />                                                (very long-lived), (iii) daf-16(mu86); age-1(mg44) double mutants (largely reverted wild- <br />                                                type longevity stress resistance), (iv) age-1(hx546) (a weaker allele ~2&#215; normal <br />                                                longevity intermediate stress resistance) (v) daf-16(mu86); age-1(hx546) (fully <br />                                                reverted wild-type longevity stress resistance). Tables 3 4 provide a <br />                                                quantitative comparison transcript levels assessed RT-qPCR independent <br />                                                biological replicates strains, total 34 genes differentially <br />                                                regulated (25 up, P &lt; 0.01) long-lived age-1(mg44)-F2 adults, as <br />                                                genes (lbp-6 evl-14) altered primarily hx546. These results are <br />                                                discussed greater subsequent sections. <br />  <br />                                                DNA sequences adjacent 34 genes differentially expressed strong age-1 mutants at <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                P &lt; 0.01 searched (using RSAT&#8217;s &#8216;dna-pattern&#8217; module) known DAF-16- <br />                                                binding/associated elements DBE DAE, novel motifs discovered RSAT <br />                                                oligo-analysis (see above). Within 1 kb upstream sequence, strand, ten <br />                                                DBEs preceding genes (vs. 6.4 expected DBEs; N.S.); 18 DAEs preceding 14 genes <br />                                                (vs. 7.4 expected; exact Binomial P &lt; 3E&#8211;4); &#8216;M1&#8217; motif, TTCWGAAAAT (0.5 <br />                                                expected, N.S.); &#8216;M2&#8217; motifs, CTACAGTAA (0.4 expected, P &lt; 0.07); &#8216;M3&#8217; <br />                                                motifs, AGAGAMGMAGA (0.06 expected; P&lt; 5E&#8211;5). These upstream motif sites are <br />                                                indicated Tables 3 4. <br />  <br />                               Transcript profiles differ age-1(mg44) F2 adults long-lived worms <br />                                                Of 36 genes confirmed differ significantly age-1 allele N2DRM <br />                                                (Tables 3 4) P &lt; 0.01, 34 (94%) showed extreme shift expression mg44, <br />                                                affected hx546. These results indicate mg44 stronger <br />                                                allele regard transcriptional effects, just assessing life-history traits such <br />                                                dauer formation survival (Ayyadevara et al., 2008). <br />  <br />                                                For subset genes showing transcript modulation age-1 mutants, expanded test <br />                                                panel adding wild-type (N2DRM) dauer larvae, (F1) generation of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                age-1(mg44)&#8722;/&#8722; adults, extensively studied long-lived mutant daf-2(e1370) (Kenyon <br />                                                et al., 1993; Larsen et al., 1995). We increased number independent worm <br />                                                expansions (the biological N) 14 groups, biological variance <br />                                                gene expression &#8211; independently expanded cohorts &#8211; greatly exceeds variance <br />                                                observed technical replicates. These RT-qPCR results (Fig. 2) documented seven genes <br />                                                age-1(mg44) F2 adults differ significantly dauer larvae transcript <br />                                                abundances genes quite similar. It appears age-1 <br />                                                extreme longevity departs dauer strategies gene expression it <br />                                                resembles it. As anticipated disparity maternal protection (Ayyadevara et <br />                                                al., 2008), F2 homozygotes surpassed parental F1 generation respect transcript <br />                                                changes 11 13 genes, seven significant P &lt; 0.02 (Fig. 2D&#8211;J). <br />                                                Unexpectedly, however, F1 adults departed significantly expression phenotype <br />                                                weaker age-1(hx546) allele, resemble terms lifespan stress <br />                                                resistance, 13 genes. <br />  <br />  <br />                                                     Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                           Page 10 <br />  <br />  <br />                               DAF-16 (FOXO) mediates age-1 allele-specific changes gene expression <br />                                                Because longevity stress-resistance phenotypes age-1 mutants largely or <br />                                                entirely reversed daf-16 (m26) mutation locus encoding DAF-16, FOXO <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                transcription factor (Ayyadevara et al., 2008; Tazearslan et al., 2009), double mutants <br />                                                provide important controls proximal effectors life extension expected also <br />                                                nullified transcript level absence active DAF-16. A substantial minority of <br />                                                genes violated expectation, implying transcriptional effects transpire DAF-16- <br />                                                independent route. Of 34 genes altered expression age-1(mg44)-F2 adults, P &lt; <br />                                                0.01 RT-qPCR, 23 (68%) reverted 70&#8211;140% wild-type addition of <br />                                                daf-16(m26) mutation, 11 not. Moreover, seven genes (21%) expressed <br />                                                daf-16(m26); age-1(mg44) double mutants levels differing N2DRM controls <br />                                                (each significant P &lt; 0.01; Tables 3 4) direction age-1 mutant <br />                                                alone, indicating reversion incomplete best. Those genes dna-2, let-502, uba-2, <br />                                                lact-3, C35A11.1, C10H11.7, ZC116.2. Three genes (hsp-12.3, cyc-2.2, and <br />                                                clec-60) shifted daf-16 mutation opposite direction age-1-induced <br />                                                change. <br />  <br />                                                Incomplete negligible reversion reflect residual FOXO activity daf-16(m26) <br />                                                allele employed, allele reversed longevity phenotypes both <br />                                                age-1(hx546) age-1(mg44) worms (Ayyadevara et al., 2008). Nearly identical results <br />                                                obtained; however, double mutant constructed combine age-1(mg44) with <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                daf-16(mu86), large-deletion allele. Among 26 genes tested, (23%) showed more <br />                                                partial reversion (differing N2DRM P &#8804; 0.01). Examples shown panels <br />                                                K&#8211;M Fig. 2. These data support existence DAF-16-independent route to <br />                                                transcriptional effects downstream age-1, account small <br />                                                proportion mg44-specific changes. <br />  <br />                               Expression changes age-1(mg44) F2 adults reflect genotype age <br />                                                In experiments described far, worms approximately matched adult <br />                                                chronological age (days L4/adult molt) 6 days. All fertile strains were <br />                                                post-gravid point, ensuring egg transcripts account differential <br />                                                expression observed. Among strains differing markedly longevity, however, alignment by <br />                                                chronological age creates considerable disparity physiological age &#8211; itself <br />                                                underlie transcript-abundance changes cases (see, e.g. Helfand et al., 1995; <br />                                                Tazearslan et al., 2009). To compare contributions chronological vs. physiological <br />                                                age, selected 12 genes strongly modulated age-1(mg44) F2 adults &#8211; eight <br />                                                decreased increased. These genes reassessed wild-type N2DRM at <br />                                                adult ages age-1(mg44) F2 adults ages. The resulting time series (Fig. 3) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                demonstrate expression changes seen long-lived age-1 worms be <br />                                                interpreted secondary consequences youthful physiological state relative to <br />                                                N2DRM. In particular, expression age-1(mg44) F2 worms 8 16 days adult age <br />                                                (5 10% median adult lifespans) did overlap levels seen N2DRM <br />                                                worms 1, 3 6 days adult age (6, 18 37% adult lifespan), 12 genes <br />                                                examined (cyp-35C1, dkf-2, clec-60, sna-1, Y75B7B.1, <span id='am-64' about='obo:NCBITaxon_178515' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_6042 obo:NCBITaxon_1 obo:NCBITaxon_6040 obo:NCBITaxon_131567 obo:NCBITaxon_178475 obo:NCBITaxon_2759 obo:NCBITaxon_6049'><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Aka (de Laubenfels, 1949)" datatype="xsd:string"></span><span id='am-66'  ></span><span id='am-67' property="rdfs:label" content="Aka" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>aka</span>-1, cyc-2.2, F16H11.3 and <br />                                                nspd-2). The remaining genes shared somewhat paradoxical pattern expression, in <br />                                                N2DRM expression lowest adult day 1, rising time level observed <br />                                                day 6 (to groups normalized). Although case age-1(mg44) F2 <br />                                                adults day 8 16 showed transcript levels somewhat close day-1 wild-type <br />                                                adults, age-1 mutant worms actually trended opposite direction, expression <br />                                                declining time. Thus, gene expression patterns said scale with <br />                                                lifespan. Transcript levels appeared independent age N2DRM 12 genes (A&#8211; <br />  <br />  <br />  <br />                                                    Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                          Page 11 <br />  <br />  <br />                                                E), age-1(mg44) 4&#8211;7 genes (A, C, E, F probably I&#8211;K), genes were <br />                                                stable respect age the strains. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Functional assessment H2O2-resistance longevity RNAi knock-down <br />                                                Alterations gene expression, described above, contribute the <br />                                                exceptional survival phenotypes strong age-1 mutants incidental side-effects. <br />                                                To distinguish possibilities, bacteria retrieved Ahringer library <br />                                                (Kamath &amp; Ahringer, 2003), inducibly express double-stranded RNAs (dsRNA) <br />                                                corresponding 18 21 genes shown RT-qPCR 3.3-fold <br />                                                downregulated age-1(mg44) (each P &lt; 10&#8722;5; Tables 3 4), plus lbp-6, gene of <br />                                                lesser primarily hx546-specific effect, negative controls (with unchanged or <br />                                                increased expression age-1 mutants). We tested effects RNA interference in <br />                                                wild-type age-1(hx546) worms, survival presence hydrogen peroxide &#8211; <br />                                                resistance oxidative stress shown previously track extreme <br />                                                longevity comparisons wild-type age-1 strains (Ayyadevara et al., 2008; <br />                                                Tazearslan et al., 2009). <br />  <br />                                                RNAi knockdown, genes showing lower expression adults bearing strong age-1 alleles, <br />                                                significantly improved hydrogen-peroxide resistance N2DRM adults 11 18 <br />                                                instances (61%; P &lt; 10&#8722;3). Targeting control genes, however, produced no <br />                                                benefit relative empty-vector controls (Tables 3&#8211;5). Feeding dsRNA-expressing <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                bacteria age-1(hx546) adults conferred peroxide protection genes (60%). <br />                                                Despite numerical equivalence, benefit RNAi quite different two <br />                                                strains (see Fig. 4 Table 5). RNAi targeting 6 genes (ubl-5, F36F2.3, let-502, F53H1.3, <br />                                                cyp-35C1 lbp-6) conferred substantially peroxide protection age-1(hx546) <br />                                                adults (survival increases 1.7- threefold feeding-vector controls) N2DRM, <br />                                                suggesting effects synergistic mild age-1 deficit. Conversely, dsRNA clones <br />                                                (targeting F16H11.3, C03H5.3 T12E12.1) greater impact N2DRM, suggesting <br />                                                RNAi effects partially redundant age-1(hx546) phenotype. <br />                                                Concordance strains seen genes F36F2.3, aka-1, F53H1.3, lbp-6 <br />                                                cyp-35C1 (dsRNA&#8217;s conferred significant protection strains), dkf-2 (no <br />                                                protection strain). Two genes (ubl-5, C03H5.3) elicited small RNAi effects <br />                                                fell short significance (0.05 &lt; P &lt; 0.07) strains. <br />  <br />                                                In studies, feeding dsRNA-producing bacteria began parents tested <br />                                                worms, targeted RNAs depleted lives. In other <br />                                                experiments (not shown), similar results observed genes treatment <br />                                                commenced L4/adult molt, genes (F36F2.3, ubl-5 aka-1) less <br />                                                influential adult-only treatment, possibly incomplete inhibition. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Genes low-level expression considered (Table 4), genes are <br />                                                generally under-represented identified microarrays (Miller et al., 2001), <br />                                                particularly good candidates efficient inhibition RNAi (Fire et al., <br />                                                1998). The proportion RNAi treatments conferring peroxide resistance little <br />                                                higher group (5/7, 71%) genes implicated SAM (6/11, 55%; Table 3), <br />                                                genes comparably downregulated age-1(mg44) adults &#8211; difference not <br />                                                significant. <br />  <br />                                                Twelve peroxide-protective RNAi clones (ten conferred significant protection in <br />                                                N2DRM, primarily age-1(hx546)) tested extension N2DRM <br />                                                adult lifespan, RNAi treatment beginning parental generation (Table 6, section <br />                                                A). Five RNAi treatments improved median lifespan adults (lbp-6, F53H1.3, <br />                                                T12E12.1, F16H11.3 C03H5.3), RNAi clones targeting seven genes no <br />  <br />  <br />                                                    Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                          Page 12 <br />  <br />  <br />                                                significant effect reduced longevity. Life extension confirmed F53H1.3 lbp-6 <br />                                                sets assays (Table 6B, C), RNAi treatment began L4/ <br />                                                adult molt. Examples longevity survivals shown Fig. 4C,D. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Discussion <br />                                                The steps selecting genes downregulated age-1 mutants, narrowing group <br />                                                genes confer oxidative-stress resistance expression blocked RNAi in <br />                                                adults, produce successively greater enrichment genes RNAi extends <br />                                                lifespan. Thus, able test functional effects RNAi 16 genes confirmed be <br />                                                downregulated fourfold age-1(mg44) worms (Tables 3 4), 11 these <br />                                                (69%) conferred hydrogen-peroxide resistance wild-type worms &#8211; considerably than <br />                                                ~10% unselected RNAi clones increased resistance oxidative stressor, <br />                                                paraquat (Kim &amp; Sun, 2007) (chi-squared exact binomial P &lt; 10&#8722;8), hydrogen <br />                                                peroxide (unpublished data). We surprised high frequency RNAi, <br />                                                directed genes selected age-1-attenuated expression, resulted hydrogen peroxide <br />                                                resistance. Although false-positives possible, survival confirmed an <br />                                                independent expansion N2DRM worms. We know RNAi response itself does not <br />                                                confer protection peroxide stress, instances worms remained <br />                                                sensitive H2O2 despite effective RNAi knock-down, indicated RT-qPCR. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Twelve peroxide-protective dsRNA clones fed wild-type worms assess whether <br />                                                RNA inhibition extends lifespan. Five treatments significantly increased longevity in <br />                                                studies; targeted genes encoding uncharacterized highly conserved protein <br />                                                (F53H1.3), lipid-binding protein (lbp-6), putative E2 ubiquitin-conjugating enzyme <br />                                                (C03H5.3), predicted E3 ubiquitin ligase (T12E12.1) possible nucleoside transporter <br />                                                (F16H11.3). Two confirmed extend lifespan separate experiments, in <br />                                                dsRNA feeding began maturity. <br />  <br />                                                Among 11 RNAi treatments conferred substantial protection hydrogen <br />                                                peroxide chosen emulate under-expression age-1(mg44) F2 adults (i.e. <br />                                                excluding lbp-6 downregulated chiefly hx546), targeting rab-10 and <br />                                                cpr-1 reported previously confer extended lifespan (Murphy et al., 2003; <br />                                                Hansen et al., 2005) shown present study (Table 6 <br />                                                Fig. 4C,D). Our &#8216;rediscovery&#8217; previously known longevity-control genes of <br />                                                11 (18%) 13-fold enrichment (exact Binomial P &lt; 0.01) 1.2&#8211;1.4% frequencies <br />                                                life extension reported previous RNAi screens (Hamilton et al., 2005; Hansen et al., <br />                                                2005; Smith et al., 2007). The probability finding novel longevity genes any <br />                                                11 genes taken random survival testing (again omitting lbp-6) P &lt; 2 &#215; 10&#8722;5, whereas <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                longevity-implicated genes chance, previously reported novel, P &lt; <br />                                                4 &#215; 10&#8722;9. <br />  <br />                                                Genes tested RNAi enhancement peroxide survival included genes which <br />                                                transcripts significantly reduced (at P &lt; 0.01) weaker age-1(hx546) allele. Four <br />                                                confer significant peroxide resistance, including (rab-10) <br />                                                reported previously extend lifespan RNAi (Hansen et al., 2005) another <br />                                                (lbp-6, encoding lipid-binding protein) discovered (Fig. 4C). The probability <br />                                                finding life-extending RNAi targets genes taken random &lt; 0.003. <br />  <br />                               Conclusions <br />                                                Based observations above, conclude (i) extreme age-1 deficiency invokes <br />                                                multiple transcriptional changes, differ described previously; (ii) <br />  <br />  <br />                                                    Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                             Page 13 <br />  <br />  <br />                                                changes predominantly inhibitory predominantly (but entirely) mediated <br />                                                DAF-16/FOXO; (iii) novel upstream motifs implicated, addition 19 <br />                                                previously established transcription-factor binding sites; (iv) surprisingly large fraction of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                genes significantly downregulated age-1(mg44) F2 adults confer resistance to <br />                                                oxidative stress knocked wild-type worms; (v) genes are, turn, <br />                                                highly significantly enriched RNAi targets extend lifespan suppressed. <br />  <br />                                                We infer results extraordinary longevity stress resistance the <br />                                                age-1(mg44) strain achieved, part, concerted modulation gene- <br />                                                expression profile. This strongly implies program, evolved response recurring <br />                                                selective pressures, favors survival presence oxidative and/or electrophilic <br />                                                stresses. That program capable producing far greater longevity stress resistance <br />                                                observed previously IIS-pathway mutants, presumably functions acute <br />                                                response environmental signals, maintained life-span <br />                                                potential precludes reproduction, establishment natural <br />                                                selection. Of genes primarily exclusively inhibited activated very-long-lived <br />                                                PI3KCS-null alleles age-1, great majority operate largely entirely the <br />                                                transcriptional control DAF-16, previously implicated in <br />                                                known lifespan-modulating pathways. <br />  <br />                                                Of particular uncharacterized gene F53H1.3; lbp-6 lipid-binding protein; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                30 zinc- metal-binding proteins, including 17 zinc-finger proteins; 11 other <br />                                                transcription factors 17 RNA splicing factors; 11 lipid-metabolism genes; 21 ATP- <br />                                                binding 13 DNA-binding proteins; 11 oxidoreductases; 20 transferases; 24 <br />                                                hydrolases. We identified 16 genes encode components ubiquitin&#8211;ligase <br />                                                complexes, shown extend lifespan targeted RNAi; these <br />                                                related to, distinct from, genes implicated previously lifespan <br />                                                regulation (Ghazi et al., 2007; Li et al., 2007; Carrano et al., 2009). Such genes reflect <br />                                                expression patterns peculiar (or exaggerated in) long-lived mutant worms, is <br />                                                consistent marked enrichment potential transcription-factor-binding motifs, <br />                                                immediately upstream affected genes. At genetic mechanisms life <br />                                                extension eluded prior discovery because, fully implemented, interfere <br />                                                normal development propagation. <br />  <br />                               Experimental procedures <br />                               Strains <br />                                                Nematode strains, supplied Caenorhabditis Genetics Center (CGC, Minneapolis, MN, <br />                                                USA) derived laboratory CGC strains, maintained 20 &#176;C throughout <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                lives experimental procedures. Cohorts synchronized alkaline- <br />                                                hypochlorite lysis parents (sparing eggs/larvae), propagated 0.6% peptone NGM- <br />                                                agar plates seeded E. coli strain OP50, described (Ebert et al., 1993; Ayyadevara et <br />                                                al., 2001; Shmookler Reis et al., 2007). <br />  <br />                               Determination lifespan <br />                                                <span id='am-48' about='obo:NCBITaxon_333870' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_195229 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41087 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_71193 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-49'  ></span><span id='am-50' property="rdfs:label" content="Nematodes" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Nematodes Berthold, 1827" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Nematodes</span>, grown NGM-agar plates containing 0.6% peptone (Sulston &amp; Hodgkin, <br />                                                1988; Ebert et al., 1993; Ayyadevara et al., 2003), harvested rinsing plate <br />                                                S buffer (0.1 M NaCl, 0.05 M potassium phosphate, pH 6.0) (Sulston &amp; Hodgkin, <br />                                                1988; Ebert et al., 1993; Ayyadevara et al., 2003). Adults, recovered differential settling <br />                                                (Ayyadevara et al., 2008), resuspended alkaline hypochlorite (0.5 N NaOH, <br />                                                1.05% hypochlorite; 5 min 20 &#176;C). The recovered eggs (containing uneclosed larvae) were <br />                                                rinsed S Buffer placed fresh agar plates seeded E. coli strain OP50. Survival <br />  <br />  <br />                                                     Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                            Page 14 <br />  <br />  <br />                                                cultures established 60-mm agar plates 1-day L4/adult molt; 25&#8211;50 adults <br />                                                transferred 60-mm dishes containing NGM agar central lawn E. coli <br />                                                bacteria (strain OP50). Worms maintained 20 &#176;C live worms counted during <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                daily transfer fresh dishes, described above. Worms considered dead did <br />                                                spontaneously response touch. Those lost (stranded dish walls or <br />                                                beneath agar) killed internal hatching progeny censored midpoint of <br />                                                time interval occurred; worms inadvertently killed censored the <br />                                                time event. <br />  <br />                               Functional assessment genes peroxide survival RNAi treatment <br />                                                Adult worms (N2DRM age-1(hx546), indicated) synchronized alkaline <br />                                                hypochlorite lysis hermaphrodites; surviving embryos transferred fresh NGM- <br />                                                agar plates allowed mature. On reaching L4 stage, worms placed fresh <br />                                                NGM-agar plates seeded bacteria expressing double-stranded RNAs Ahringer <br />                                                library (Kamath &amp; Ahringer, 2003). After 48 h, transferred 24-well plates (20&#8211; <br />                                                25 worms well) containing S Medium (S buffer plus 0.5% cholesterol) 5-mM <br />                                                hydrogen peroxide (Sigma, St Louis, MO, USA) 20 &#176;C, previously described <br />                                                (Ayyadevara et al., 2008). Survival scored above, initially 1-h intervals, no <br />                                                worms remained alive. Assays performed twice strain. <br />  <br />                               Microarray analysis gene expression <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Microarrays, 60-mer oligonucleotides printed epoxy-coated slides, obtained from <br />                                                The Genome Sequencing Center Washington University (St Louis). Each array consists <br />                                                &gt; 22 000 spots representing nearly predicted confirmed genes C. elegans. <br />                                                Nematode cultures synchronized alkaline-hypochlorite lysis gravid <br />                                                hermaphrodites, followed rinsing propagation unlysed eggs embryos (see Ebert <br />                                                et al., 1993). Cultures synchronized manual selection L4 stage, full- <br />                                                sized adults separated runts time harvest, differential settling <br />                                                (Ayyadevara et al., 2008). Total RNA prepared (RNEasy; Qiagen, <span id='am-59' about='obo:NCBITaxon_195854' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_8087 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_28738 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_7898 obo:NCBITaxon_195853 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="Valencia" datatype="xsd:string"></span><span id='am-63'  ></span>Valencia</span>, CA, USA) <br />                                                fertile strains 6&#8211;8 days adult age, F2 generation strong age-1 alleles <br />                                                9&#8211;12 adult days, reverse transcribed cDNA using Superscript III (Invitrogen, <br />                                                Carlsbad, CA, USA) oligo-dT26 primer. The cDNAs, labeled indirectly Cy3 Cy5 <br />                                                fluors using Genisphere 900HS dendromer system, combined pairs (comprising <br />                                                Cy3- Cy5-labeled cDNA) hybridized microarray slides HybAid <br />                                                annealing station (The Gel Company, San Francisco, CA, USA). After careful rinsing, slides <br />                                                read using ScanArray 5000 <span id='am-76' about='obo:NCBITaxon_62990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_241799 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_241789 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-77' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79'  ></span><span id='am-80' property="rdfs:label" content="Laser" datatype="xsd:string"></span>laser</span> fluorescence scanner (GSI Lumonics, Bedford, MA, <br />                                                USA). Data pretreatment (spot recognition background subtraction) utilized GeneSpring <br />                                                (Silicon Genetics/Agilent Technologies, Santa Clara, CA, USA), followed statistical <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                analysis SAM [(Tusher et al., 2001), ver. 3.02; Stanford University]. Pathway analysis <br />                                                conducted Pathway Studio (ARIADNE, Rockville, MD, USA). Gene ontology <br />                                                analysis employed functional annotation chart clustering reports, WormBase <br />                                                annotations (http://www.wormbase.org) SAM significant-gene lists compared by <br />                                                DAVID (Dennis et al., 2003) (ver. 4, 2008; http://david.abcc.ncifcrf.gov), C. <br />                                                elegans gene set. Word patterns enriched sequences adjoining identified genes were <br />                                                assessed RSAT (J. van Helden, http://rsat.ulb.ac.be/rsat/), background defined an <br />                                                input-based 2nd <span id='am-45' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-46' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> Markov model, Genomatix software matrices for <br />                                                established consensus binding motifs (http://www.genomatix.de). <br />  <br />                               Transcript quantitation RT-qPCR <br />                                                Expression selected genes assessed real-time polymerase chain reaction after <br />                                                reverse transcription. Total RNA purified strain cohort (RNeasy; Qiagen), <br />  <br />  <br />  <br />                                                    Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                                               Page 15 <br />  <br />  <br />                                                cDNAs reverse-transcribed (SuperScript III; Invitrogen), followed RT-qPCR <br />                                                (Opticon2, MJ Research, using SYBR Green, Roche). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Statistical analyses <br />                                                False-discovery rate estimated &#8216;q-value&#8217; method Storey &amp; Tibshirani (2003) <br />                                                SAM, method Hochberg Benjamini (Hochberg &amp; Benjamini, 1990; <br />                                                Klipper- Aurbach et al., 1995) DAVID. Survivals compared log-rank tests <br />                                                (Lee, 1992) using NCSS&#8482; software. Transcript levels compared two-tailed, <br />                                                heteroscedastic t-tests (i.e. unpaired samples unknown unequal variance), <br />                                                conducted Microsoft Excel 2003. The values compared control-subtracted <br />                                                threshold cycle numbers, biological replicates (independently generated cohorts), <br />                                                addressing purely inter-strain variation relative variance biological replicates. It <br />                                                noted directly comparing values Ct, number cycles required reach <br />                                                threshold yield amplimers, equivalent working negatives log- <br />                                                transformed RNA levels (&#8722;log2 [RNA]). Actual P-values given case, prior any <br />                                                Bonferroni adjustment. <br />  <br />                               Supplementary Material <br />                                                Refer Web version PubMed Central supplementary material. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Acknowledgments <br />                                                We thank Rajani Ayyadevara expert technical assistance, David Gems Eugene Schuster (University <br />                                                College, London) kindly providing gene lists helpful suggestions. This work supported a <br />                                                grant (to R.J.S.R.) U.S. National Institutes Health, Research Career Scientist Award (R.J.S.R.) <br />                                                U.S. Department Veterans Affairs. <br />  <br />  <br />                               References <br />                                                Ayyadevara S, Ayyadevera R, Hou S, Thaden JJ, Shmookler Reis RJ. Genetic mapping quantitative <br />                                                   trait loci governing longevity Caenorhabditis elegans recombinant-inbred progeny a <br />                                                   Bergerac-BO x RC301 interstrain cross. Genetics 2001;157:655&#8211;666. [PubMed: 11156986] <br />                                                Ayyadevara S, Ayyadevera R, Vertino A, Galecki A, Thaden JJ, Shmookler Reis RJ. Genetic loci <br />                                                   modulating fitness life span Caenorhabditis elegans: categorical trait interval mapping in <br />                                                   CL2a&#215;Bergerac-BO recombinant-inbred worms. Genetics 2003;163:557&#8211;570. [PubMed: 12618395] <br />                                                Ayyadevara S, Alla R, Thaden JJ, Shmookler Reis RJ. Remarkable longevity stress resistance of <br />                                                   nematode PI3K-null mutants. Aging Cell 2008;7:13&#8211;22. [PubMed: 17996009] <br />                                                Carrano AC, Liu Z, Dillin A, Hunter T. A conserved ubiquitination pathway determines longevity in <br />                                                   response diet restriction. Nature 2009;460:396&#8211;399. [PubMed: 19553937] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Dennis G Jr, Sherman BT, Hosack DA, Yang J, Gao W, Lane HC, Lempicki RA. DAVID: database <br />                                                   annotation, visualization, integrated discovery. Genome Biol 2003;4:3. <br />                                                Dorman JB, Albinder B, Shroyer T, Kenyon C. The age-1 daf-2 genes function common <br />                                                   pathway control lifespan Caenorhabditis elegans. Genetics 1995;141:1399&#8211;1406. <br />                                                   [PubMed: 8601482] <br />                                                Ebert RH, Cherkasova VA, Dennis RA, Wu JH, Ruggles S, Perrin TE, Shmookler Reis RJ. Longevity- <br />                                                   determining genes Caenorhabditis elegans: chromosomal mapping multiple noninteractive <br />                                                   loci. Genetics 1993;135:1003&#8211;1010. [PubMed: 8307318] <br />                                                Fire A, Xu S, Montgomery MK, Kostas SA, Driver SE, Mello CC. Potent specific genetic <br />                                                   interference double-stranded RNA Caenorhabditis elegans. Nature 1998;391:806&#8211;811. <br />                                                   [PubMed: 9486653] <br />                                                Friedman DB, Johnson TE. A mutation age-1 gene Caenorhabditis elegans lengthens life and <br />                                                   reduces hermaphrodite fertility. Genetics 1988a;118:75&#8211;86. [PubMed: 8608934] <br />  <br />  <br />  <br />  <br />                                                      Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                                Page 16 <br />  <br />  <br />                                                Friedman DB, Johnson TE. Three mutants extend mean maximum life span the <br />                                                    nematode, Caenorhabditis elegans, define age-1 gene. J Gerontol 1988b;43:B102&#8211;B109. <br />                                                    [PubMed: 3385139] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Gems D, Riddle DL. Defining wild-type life span Caenorhabditis elegans. J Gerontol A Biol Sci <br />                                                    Med Sci 2000;55:B215&#8211;B219. [PubMed: 10819307] <br />                                                Ghazi A, Henis-Korenblit S, Kenyon C. Regulation Caenorhabditis elegans lifespan a <br />                                                    proteasomal E3 ligase complex. Proc Natl Acad Sci USA 2007;104:5947&#8211;5952. [PubMed: <br />                                                    17392428] <br />                                                Guarente L, Kenyon C. Genetic pathways regulate ageing model organisms. Nature <br />                                                    2000;408:255&#8211;262. [PubMed: 11089983] <br />                                                Hamilton B, Dong Y, Shindo M, Liu W, Odell I, Ruvkun G, Lee SS. A systematic RNAi screen for <br />                                                    longevity genes C. elegans. Genes Dev 2005;19:1544&#8211;1555. [PubMed: 15998808] <br />                                                Hansen M, Hsu AL, Dillin A, Kenyon C. New genes tied endocrine, metabolic, dietary <br />                                                    regulation lifespan Caenorhabditis elegans genomic RNAi screen. PLoS Genet <br />                                                    2005;1:119&#8211;128. [PubMed: 16103914] <br />                                                Helfand SL, Blake KJ, Rogina B, Stracks MD, Centurion A, Naprta B. Temporal patterns gene <br />                                                    expression antenna adult <span id='am-30' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-31'  ></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span>Drosophila melanogaster</span>. Genetics 1995;140:549&#8211;555. <br />                                                    [PubMed: 7498736] <br />                                                Hochberg Y, Benjamini Y. More powerful procedures multiple significance testing. Stat Med <br />                                                    1990;9:811&#8211;818. [PubMed: 2218183] <br />                                                Huang DW, Sherman BT, Lempicki RA. Systematic integrative analysis large gene lists using <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    DAVID bioinformatics resources. Nat Protoc 2009;4:44&#8211;57. [PubMed: 19131956] <br />                                                Kamath RS, Ahringer J. Genome-wide RNAi screening Caenorhabditis elegans. Methods <br />                                                    2003;30:313&#8211;321. [PubMed: 12828945] <br />                                                Kenyon C, Murphy CT. Enrichment regulatory motifs upstream predicted DAF-16 targets. Nat <br />                                                    Genet 2006;38:397&#8211;398. [PubMed: 16570056] <br />                                                Kenyon C, Chang J, Gensch E, Rudner A, Tabtiang R. A C. elegans mutant lives twice long as <br />                                                    wild type. Nature 1993;366:461&#8211;464. [PubMed: 8247153] <br />                                                Kim SK. Common aging pathways worms, flies, mice humans. J Exp Biol 2007;210:1607&#8211; <br />                                                    1612. [PubMed: 17449826] <br />                                                Kim Y, Sun H. Functional genomic approach identify novel genes involved regulation of <br />                                                    oxidative stress resistance animal lifespan. Aging Cell 2007;6:489&#8211;503. [PubMed: 17608836] <br />                                                Klass MR. A method isolation longevity mutants nematode Caenorhabditis elegans <br />                                                    initial results. Mech Ageing Dev 1983;22:279&#8211;286. [PubMed: 6632998] <br />                                                Klipper-Aurbach Y, Wasserman M, Braunspiegel-Weintrob N, Borstein D, Peleg S, <span id='am-54' about='obo:NCBITaxon_377306' typeof='obo:NCBITaxon_33511 owl:Thing obo:NCBITaxon_218471 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_7711 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8365 obo:NCBITaxon_7776 obo:NCBITaxon_8342 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_41666 obo:NCBITaxon_8292 obo:NCBITaxon_2759 obo:NCBITaxon_8416 obo:NCBITaxon_650418 obo:NCBITaxon_6072'><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Assa" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58'  ></span>Assa</span> S, Karp M, <br />                                                    Benjamini Y, Hochberg Y, Laron Z. Mathematical formulae prediction residual <span id='am-6' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>beta</span> <br />                                                    cell function years disease children adolescents insulin- <br />                                                    dependent diabetes mellitus. Med Hypotheses 1995;45:486&#8211;490. [PubMed: 8748093] <br />                                                Larsen PL, Albert PS, Riddle DL. Genes regulate development longevity in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    Caenorhabditis elegans. Genetics 1995;139:1567&#8211;1583. [PubMed: 7789761] <br />                                                Lee, ET. Statistical Methods Survival Data Analysis. New York: J. Wiley &amp; Sons Inc; 1992. <br />                                                Lee SS, Kennedy S, Tolonen AC, Ruvkun G. DAF-16 target genes control C. elegans life-span <br />                                                    metabolism. Science 2003a;300:644&#8211;647. [PubMed: 12690206] <br />                                                Lee SS, Lee RY, Fraser AG, Kamath RS, Ahringer J, Ruvkun G. A systematic RNAi screen identifies <br />                                                    critical role mitochondria C. elegans longevity. Nat Genet 2003b;33:40&#8211;48. [PubMed: <br />                                                    12447374] <br />                                                Li W, Gao B, Lee SM, Bennett K, Fang D. RLE-1, E3 ubiquitin ligase, regulates C. elegans aging <br />                                                    catalyzing DAF-16 polyubiquitination. Dev Cell 2007;12:235&#8211;246. [PubMed: 17276341] <br />                                                Lund J, Tedesco P, Duke K, Wang J, Kim SK, Johnson TE. Transcriptional profile aging C. <br />                                                    elegans. Curr Biol 2002;12:1566&#8211;1573. [PubMed: 12372248] <br />                                                McElwee JJ, Schuster E, Blanc E, Thomas JH, Gems D. Shared transcriptional signature in <br />                                                    Caenorhabditis elegans Dauer larvae long-lived daf-2 mutants implicates detoxification <br />                                                    longevity assurance. J Biol Chem 2004;279:44533&#8211;44543. [PubMed: 15308663] <br />  <br />  <br />  <br />                                                     Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                                  Page 17 <br />  <br />  <br />                                                McElwee JJ, Schuster E, Blanc E, Thornton J, Gems D. Diapause-associated metabolic traits reiterated <br />                                                    long-lived daf-2 mutants nematode Caenorhabditis elegans. Mech Ageing Dev <br />                                                    2006;127:922&#8211;936. [PubMed: 17216712] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Miller RA, Galecki A, Shmookler Reis RJ. Interpretation, design, analysis gene array <br />                                                    expression experiments. J Gerontol A Biol Sci Med Sci 2001;56:B52&#8211;B57. [PubMed: 11213267] <br />                                                Morris JZ, Tissenbaum HA, Ruvkun G. A phosphatidylinositol-3-OH kinase <span id='am-12' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-13' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> member <br />                                                    regulating longevity diapause Caenorhabditis elegans. Nature 1996;382:536&#8211;539. <br />                                                    [PubMed: 8700226] <br />                                                Murphy CT. The search DAF-16/FOXO transcriptional targets: approaches discoveries. Exp <br />                                                    Gerontol 2006;41:910&#8211;921. [PubMed: 16934425] <br />                                                Murphy CT, McCarroll SA, Bargmann CI, Fraser A, Kamath RS, Ahringer J, Li H, Kenyon C. Genes <br />                                                    act downstream DAF-16 influence lifespan Caenorhabditis elegans. Nature <br />                                                    2003;424:277&#8211;283. [PubMed: 12845331] <br />                                                Ogg S, Paradis S, Gottlieb S, Patterson GI, Lee L, Tissenbaum HA, Ruvkun G. The Fork head <br />                                                    transcription factor DAF-16 transduces insulin-like metabolic longevity signals C. elegans. <br />                                                    Nature 1997;389:994&#8211;999. [PubMed: 9353126] <br />                                                Oh SW, Mukhopadhyay A, Svrzikapa N, Jiang F, Davis RJ, Tissenbaum HA. JNK regulates lifespan <br />                                                    Caenorhabditis elegans modulating nuclear translocation forkhead transcription factor/ <br />                                                    DAF-16. Proc Natl Acad Sci USA 2005;102:4494&#8211;4499. [PubMed: 15767565] <br />                                                Oh SW, Mukhopadhyay A, Dixit BL, Raha T, Green MR, Tissenbaum HA. Identification direct <br />                                                    DAF-16 targets controlling longevity, metabolism diapause chromatin <br />                                                    immunoprecipitation. Nat Genet 2006;38:251&#8211;257. [PubMed: 16380712] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Shmookler Reis RJ, Kang P, Ayyadevara S. Quantitative trait loci define genes pathways <br />                                                    underlying genetic variation longevity. Exp Gerontol 2007;41:1046&#8211;1054. [PubMed: 16919411] <br />                                                Smith ED, Kennedy BK, Kaeberlein M. Genome-wide identification conserved longevity genes in <br />                                                    yeast worms. Mech Ageing Dev 2007;128:106&#8211;111. [PubMed: 17126379] <br />                                                Storey JD, Tibshirani R. Statistical significance genomewide studies. Proc Natl Acad Sci USA <br />                                                    2003;100:9440&#8211;9445. [PubMed: 12883005] <br />                                                Sulston, J.; Hodgkin, J. Methods. In: Wood, WB., editor. The Nematode Caenorhabditis elegans. Cold <br />                                                    Spring Harbor, NY: Cold Spring Harbor Laboratory Press; 1988. p. 587-606. <br />                                                Tazearslan C, Ayyadevara S, Bharill P, Shmookler Reis RJ. Positive feedback transcriptional <br />                                                    kinase suppression confers extraordinary longevity stress resistance. PLoS Genet <br />                                                    2009;5:e1000452. [PubMed: 19360094] <br />                                                Tissenbaum HA, Ruvkun G. An insulin-like signaling pathway affects longevity reproduction <br />                                                    Caenorhabditis elegans. Genetics 1998;148:703&#8211;717. [PubMed: 9504918] <br />                                                Tusher VG, Tibshirani R, Chu G. Significance analysis microarrays applied ionizing radiation <br />                                                    response. Proc Natl Acad Sci USA 2001;98:5116&#8211;5121. [PubMed: 11309499] <br />                                                Wang J, Kim SK. Global analysis dauer gene expression Caenorhabditis elegans. Development <br />                                                    2003;130:1621&#8211;1634. [PubMed: 12620986] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                           Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 1. <br />                                                Significance analysis microarray (SAM) &#8216;outlier&#8217; plots heat maps microarray <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                analyses. (A) Four Caenorhabditis elegans full-genome microarrays probed with <br />                                                cDNAs prepared age-1(mg44)-F2 adults, mixed pairwise cDNAs adults <br />                                                bearing age-1(hx546) allele (four independent cohorts strain), more <br />                                                arrays probed cDNAs age-1(m333)-F2 adults, mixed pairwise with <br />                                                age-1(hx546) cDNAs (four cohorts strain). Data combined arrays <br />                                                (treating long-lived strains equivalent) analyzed using SAM (Tusher et <br />                                                al., 2001). The SAM plot shows outliers green transcript levels lower strong <br />                                                age-1 alleles (mg44 m333) hx546, red higher strong alleles, a <br />                                                FDR &lt; 5%. (B) Three microarrays probed cDNAs age-1(mg44)-F2 <br />                                                adults, mixed pairwise cDNAs N2DRM adults bearing wild-type age-1 allele <br />                                                (three independent cohorts strain), arrays probed cDNAs <br />                                                age-1(m333)-F2 adults, mixed pairwise N2DRM cDNAs (three cohorts per <br />                                                strain). Data combined arrays (treating long-lived strains as <br />                                                equivalent) analyzed SAM above. (C) Partial heat maps represent data eight <br />  <br />  <br />                                                     Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                         Page 19 <br />  <br />  <br />                                                arrays probed cDNA age-1(hx546) adults (left), probed with <br />                                                cDNAs strong age-1 alleles mg44 m333 (center). Expression ratios shown for <br />                                                array (right) log2(mg44/hx546) (m333/hx546); red intensity indicates positive <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                log-ratios (higher expression mg44 m333), green implies negative log-ratios <br />                                                (lower expression mg44 m333). Full heat maps presented Fig. S1. (D) Partial <br />                                                heat maps represent data arrays probed cDNA N2DRM adults (left), and <br />                                                probed cDNAs age-1 alleles mg44 m333 (center). Expression <br />                                                ratios calculated array (right) log2(mg44/N2) (m333/N2); red <br />                                                intensity indicates positive log-ratios (higher expression mg44 m333), green <br />                                                implies negative log-ratios (lower expression mg44 m333). Full heat maps are <br />                                                presented Fig. S2 (Supporting information). <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                         Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 2. <br />                                                The age-1(mg44)-F2 transcript profile differs F1 adults N2 dauer larvae. <br />                                                We retested 13 age-1-regulated genes, expanded strain panel. Strains described <br />                                                legend Table 3, addition age-1(mg44) F1 (fertile first-generation <br />                                                homozygotes), daf-2(e1370), long-lived mutant insulin/IGF-1 receptor (G&#8211;M), and <br />                                                daf-16(mu86); age-1(mg44), daf-16 deletion allele, mu86, replaces m26 allele <br />                                                used Tables 3 4. Double mutants daf-16(mu86) assessed those <br />                                                genes appeared incompletely (or negligibly) reverted daf-16(m26) (F, H&#8211; <br />                                                M; indicated black striping green bars). Orange bars [at right (G&#8211;M)] represent <br />                                                N2DRM dauer larvae, induced crowding starved plates confirmed &gt; 95% <br />                                                resistance 1% sodium dodecyl sulfate samples 500 worms. Sample numbers (n = <br />                                                number independent biological preparations) follows: 8&#8211;11 groups of <br />                                                postgravid N2DRM adults, 4&#8211;5 age-1(hx546) daf-16(m26); age-1(hx546), 3 of <br />                                                age-1(mg44) F1, 10&#8211;14 age-1(mg44)-F2 adults, 5&#8211;6 daf-16(m26); age-1(mg44) <br />                                                daf-16(mu86); age-1(mg44), 3&#8211;4 N2DRM dauers, 3&#8211;4 daf-2(e1370). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                          Page 21 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 3. <br />                                                Transcriptional changes age-1(mg44) F2 adults relatively stable respect age, <br />                                                explained result physiological youthfulness. Transcript levels <br />                                                assayed real-time reverse-transcription polymerase chain reaction (RT-qPCR) as <br />                                                described legends Table 3 Fig. 2. Within histogram, mean &#177; SEM are <br />                                                shown log2 scale steady-state transcript level, comparing wild-type N2DRM adults <br />                                                ages age-1(mg44) F2 homozygotes adult ages. All transcript values were <br />                                                normalized &#946;-actin mRNA (to adjust variation RNA inputs), each <br />                                                biological group normalized mean value N2DRM day-6 adults (to correct for <br />                                                run-to-run variation Ct values). Ages measured L4/adult molt; table (inset) <br />                                                indicates percent adult lifespan represented age. To monitor longitudinal <br />                                                changes, biological preparations group assessed (fewer used Fig. 2 <br />                                                Table 3). Significance differences ascertained two-tailed Behrens-Fisher t-tests, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                appropriate samples unequal unknown variance. The transcript levels observed in <br />                                                experiment differ means seen previous experiments (e.g. Tables 3 4), <br />                                                differences significant. <br />  <br />  <br />  <br />  <br />                                                     Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                             Ayyadevara et al.                                                                                       Page 22 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 4. <br />                                                RNAi knock-down genes downregulated age-1(mg44) confers peroxide resistance and <br />                                                extend life. Caenorhabditis elegans bearing (A) age-1(hx546) (B) wild-type age-1 <br />                                                allele fed bacteria expressing indicated dsRNA. RNAi treatment (A), (B) and <br />                                                (D) commenced early larval development parental generation, the <br />                                                tested generation underwent knockdown life. Worms (C) fed dsRNA- <br />                                                expressing bacteria continuously L4/adult molt onward. Survival statistics are <br />                                                summarized Tables 5 6. <br />  <br />  <br />  <br />  <br />                                                     Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                                                                     NIH-PA Author Manuscript                                        NIH-PA Author Manuscript                  NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                        Table 1 <br />                                                                    Transcription-factor binding motifs (vertebrate consensus matrices) enriched 1-kb regions immediately preceding start sites 184 age-1- <br />                                                                    downregulated C. elegans genes. These sites defined nucleotide-frequency matrices describing vertebrate binding sites Genomatix&#8482; library. <br />                                                                    P-values calculated Genomatix, fraction surveyed genes upstream motifs relative fraction expected random-sequence <br />                                                                    DNA, allowance <span id='am-42' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> differences GC-content nucleotide biases regulatory regions upstream coding sequences. To roughly <br />                                                                    correct biases, &#8216;Random P-values&#8217; calculated 1-kb spans preceding 200 genes sampled random C. elegans genome. The final <br />                                                                                                                                                                                                                                Ayyadevara et al. <br />  <br />  <br />  <br />  <br />                                                                    column (P ratio) presents quotients obtained dividing &#8216;P-value&#8217; corresponding &#8216;Random P &#8217;. P ratios &#8804; 0.01 listed. Motif presence next <br />                                                                    &gt; 95% genes, P ratios &#8804; 10&#8722;4, shown bold. <br />  <br />                                                                     Family   Transcription factor class           Genes motifs (%)   P-value    Random P    P ratio <br />  <br />                                                                     ATBF     AT-rich binding factor               156/184 (85)            3.08E-54   7.40E-47    4.2E-08 <br />                                                                     BCL6     POZ/Zn-finger protein, B cells       141/184 (77)            2.15E-20   8.80E-18    2.4E-03 <br />                                                                     BRNF     Brn POU-domain factors               182/184 (99)            1.56E-28   2.70E-23    5.8E-06 <br />                                                                     CART     Cartilage homeoprotein               138/184 (75)            7.58E-43   3.00E-36    2.5E-07 <br />                                                                     CDXF     Caudal-related homeodomain           158/184 (86)            3.51E-44   5.40E-37    6.5E-08 <br />                                                                     CHRF     Cell-cycle-gene homol: CDE, CHR      162/184 (88)            6.45E-55   1.60E-45    4.0E-10 <br />                                                                     CLOX     Cut-like homeodomain: CLOX, CDP      184/184 (100)           6.04E-22   2.30E-17    2.6E-05 <br />                                                                     COMP     Cooperating w. myogenic proteins     115/184 (62)            9.64E-18   4.60E-15    2.1E-03 <br />                                                                     FKHD     ForkHead-domain factors (FoxO/A)     183/184 (99.5)          1.46E-19   2.30E-17    6.4E-03 <br />                                                                     GATA     GATA-binding transcription factors   182/184 (99)            7.15E-22   1.30E-16    5.5E-06 <br />                                                                     GFI1     Growth fac. independent repressor    138/184 (75)            7.01E-27   2.40E-20    2.9E-07 <br />                                                                     HNF6     Onecut homeodomain factor            178/184 (97)            4.45E-57   3.30E-41    1.3E-16 <br />                                                                     HOMF     Homeodomain binding factor           182/184 (99)            4.06E-35   7.80E-31    5.2E-05 <br />                                                                     HOXF     Homeodomain binding factor           184/184 (100)           1.88E-13   8.30E-11    2.3E-03 <br />  <br />  <br />  <br />  <br /> Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                                                                     LHXF     Lim homeodomain binding factor       176/184 (96)            4.51E-54   1.20E-42    3.7E-12 <br />                                                                     MYBL     Myb-like transcription factor        172/184 (93)            7.03E-17   7.00E-13    1.0E-04 <br />                                                                     MYT1     MyT1 C2HC Zn-finger protein          181/184 (98)            7.55E-25   1.00E-18    7.5E-07 <br />                                                                     OCTP     Oct1 binding factor (POU-specific)   142/184 (77)            1.75E-31   5.60E-27    3.1E-05 <br />                                                                     PARF     PAR/bZIP family, albumen D-box       180/184 (98)            1.69E-26   9.50E-20    1.8E-07 <br />                                                                     PAX2     PAX2 paired-domain protein           144/184 (78)            1.69E-32   4.30E-29    3.9E-04 <br />                                                                     PDX1     Pancreatic/intestinal homeoprotein   157/184 (85)            1.37E-28   3.10E-25    4.4E-04 <br />                                                                     RUSH     RING SWI/SNF helicases/ATPases    170/184 (92)            3.11E-22   3.10E-16    1.0E-06 <br />                                                                     SATB     Special AT-rich binding factor       136/184 (72)            5.36E-46   1.60E-34    3.3E-12 <br />                                                                                                                                                                                                                                Page 23 <br />                                                                     NIH-PA Author Manuscript   NIH-PA Author Manuscript                              NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                       Family   Transcription factor class          Genes motifs (%)   P-value    Random P   P ratio <br />  <br />                                                                                                       SORY     Sox/sRY sex/testis HMG-box factor   184/184 (100)           1.48E-22   7.80E-16   1.9E-07 <br />                                                                                                       TBPF     TATA-binding transcription factor   182/184 (99)            3.07E-23   4.00E-19   7.7E-05 <br />                                                                                                                                                                                                            Ayyadevara et al. <br />  <br />  <br />  <br />  <br /> Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                                                                                                                                                                                                            Page 24 <br />                                                                             NIH-PA Author Manuscript                                         NIH-PA Author Manuscript                                   NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                           Table 2 <br />                                                                    Analysis enrichment functional annotation terms. Over-representation gene ontology (GO) terms assessed 382 annotated genes <br />                                                                    differentially expressed F2 adults homozygous strong age-1 alleles (mg44 m333), relative weaker hx546 allele, 976 annotated <br />                                                                    genes differentially expressed mutants relative N2DRM wild-type allele. &#8216;Functional Analysis Chart&#8217; reports generated by <br />                                                                    DAVID, ver. 6 (http://david.abcc.ncifcrf.gov), using default conditions GO terms annotated WormBase, enrichment relative background <br />                                                                    set annotated C. elegans genes. Results shown identical analyses performed 514 annotated genes listed differentially <br />                                                                                                                                                                                                                                                                Ayyadevara et al. <br />  <br />  <br />  <br />  <br />                                                                    expressed worms treated RNAi daf-2, relative worms fed daf-2 daf-16 RNAi (Murphy et al., 2003), set ~2000 genes <br />                                                                    differentially expressed N2 dauer larvae relative postdauer worms 12 h recovery, based reanalysis (McElwee et al., 2006) primary data <br />                                                                    (Wang &amp; Kim, 2003). Bold font indicates groups Benjamini-Hochberg false-discovery rates (B-H FDR) &lt; 0.05 (see Experimental procedures). For <br />                                                                    microarray data generated present study data files provided dauer study (McElwee et al., 2006), values shown indicate <br />                                                                    genes combined up- down-regulated sets. These analyses run, however, separate gene sets, arrows table indicate <br />                                                                    enrichment occurred primarily strong-age-1- dauer-upregulated gene sets (&#8593;), primarily strong-age-1- dauer-downregulated gene sets <br />                                                                    (&#8595;), observed sets individually (&#8593;&#8595;). Blank spaces indicate enrichment indicated terms met standard criteria (EASE <br />                                                                    score &gt; 0.1 n &#8805; 2). It noted listing significant terms complete age-1 gene sets, contains most <br />                                                                    significant terms daf-2 dauer gene sets, produced larger lists annotation terms. For term listed column, <br />                                                                    however, reported results presented analyses. <br />  <br />                                                                                                                                                                                    daf-2 vs. daf-16 + daf-2 RNAi <br />                                                                     Comparison (n):                   mg44, m33 vs. hx546 (382)              mg44, m33 vs. N2DRM (976)             (514)                              Dauer vs. postdauer (2000) <br />  <br />                                                                     Term                              *   No.   P-value   Fold    B-H FDR    *    No.   P-value   Fold   B-H FDR   No.   P-value    Fold   B-H FDR    &#8224;    No.    P-value   Fold    B-H FDR <br />  <br />                                                                     Metal-binding                     &#8595;    30   3.4E-11    4.3    8.6E-09    &#8593;&#8595;    46   2.6E-11   3.1    4.4E-09    29   1.6E-08     3.4    9.1E-07   &#8593;&#8595;   111    2.1E-19     2.5   9.1E-18 <br />                                                                     Hydrolase                         &#8595;    24   2.5E-11    5.7    1.3E-08    &#8595;     32   2.0E-09   3.6    1.5E-07    29   1.5E-13     5.7    1.9E-11   &#8593;&#8595;    87    2.3E-22     3.3   1.7E-20 <br />                                                                     Zinc                              &#8595;    25   1.8E-09    4.4    3.1E-07    &#8593;&#8595;    36   2.1E-08   2.9    1.3E-06    13   4.3E-02     1.9    4.6E-01   &#8593;&#8595;    75    3.9E-09     2.1   6.4E-08 <br />                                                                     Alternative splicing              &#8595;    17   1.1E-08    6.3    1.5E-06           8   4.1E-03   3.9    1.3E-01    11   1.6E-03     3.4    3.7E-02   &#8593;&#8595;    53    8.4E-13     3.1   2.4E-11 <br />                                                                     ATP-binding                       &#8595;    21   2.7E-08    4.6    2.7E-06    &#8595;     36   6.8E-11   3.7    5.8E-09                                      &#8593;&#8595;    56    4.6E-06     1.9   5.6E-05 <br />                                                                     Transferase                       &#8595;    20   6.8E-08    4.6    5.8E-06    &#8593;&#8595;    31   2.1E-08   3.3    1.2E-06    16   2.7E-04     3.0    7.6E-03   &#8593;&#8595;    89    2.7E-22     3.2   1.7E-20 <br />  <br />  <br />  <br />  <br /> Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                                                                     Kinase                            &#8595;    14   7.1E-07    5.9    5.2E-05    &#8595;      9   4.0E-04   5.0    1.9E-02                                      &#8593;&#8595;    34    2.2E-05     2.3   2.3E-04 <br />                                                                     Zinc-finger                       &#8595;    17   5.6E-06    4.0    3.2E-04    &#8593;&#8595;    28   5.7E-07   3.0    2.9E-05                                      &#8593;     46    6.8E-04     1.7   5.5E-03 <br />                                                                     Nucleotide-binding                &#8595;    20   2.6E-06    3.6    1.7E-04    &#8593;&#8595;    40   6.7E-11   3.3    6.8E-09    12   7.3E-02     1.8    6.1E-01   &#8595;       6   1.8E-02     3.9   9.1E-02 <br />                                                                     Oxidoreductase                    &#8595;    11   2.2E-04    4.3    1.1E-02    &#8593;&#8595;    18   3.4E-05   3.3    1.3E-03    34   7.6E-25    11.1    3.9E-22   &#8593;&#8595;    95    4.2E-46     5.9   1.1 E-43 <br />                                                                     Transmembrane                     &#8595;    19   3.0E-04    2.6    1.4E-02    &#8593;&#8595;    56   9.0E-18   3.6    5.7E-14    20   1.2E-03     2.3    2.9E-02   &#8593;&#8595;   121    9.4E-23     2.6   8.0E-21 <br />                                                                     Nucleus                           &#8595;    17   4.2E-04    2.8    1.8E-02          12   6.4E-03   2.6    1.6E-01                                      &#8593;&#8595;    79    4.7E-09     2.0   7.5E-08 <br />                                                                     Cellular metabolic process        &#8595;   107   1.2E-05    1.4    2.4E-02 <br />                                                                     Serine/threonine-protein kinase   &#8595;     7   9.2E-04    6.2    3.6E-02           5   1.0E-02   5.9    2.2E-01                                            15    1.3E-02     2.1   6.8E-02 <br />                                                                     DNA-binding                       &#8595;    13   1.1E-03    3.0    4.1E-02                                                                             &#8593;     48    1.8E-04     1.8   1.7E-03 <br />                                                                                                                                                                                                                                                                Page 25 <br />                                                                             NIH-PA Author Manuscript                                    NIH-PA Author Manuscript                                   NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                                               daf-2 vs. daf-16 + daf-2 RNAi <br />                                                                    Comparison (n):               mg44, m33 vs. hx546 (382)              mg44, m33 vs. N2DRM (976)             (514)                              Dauer vs. postdauer (2000) <br />  <br />                                                                    Term                          *   No.   P-value   Fold    B-H FDR    *    No.   P-value   Fold   B-H FDR   No.   P-value    Fold   B-H FDR    &#8224;    No.    P-value   Fold    B-H FDR <br />  <br />                                                                    Receptor                                                             &#8593;&#8595;    48   4.5E-11   2.9    5.8E-09 <br />                                                                    Transport                           9   4.5E-02    2.3    6.9E-01    &#8593;     25   5.1E-06   2.9    2.4E-04 <br />                                                                                                                                                                                                                                                          Ayyadevara et al. <br />  <br />  <br />  <br />  <br />                                                                    SH2 domain                                                                  8   5.5E-05   8.1    2.0E-03 <br />                                                                    WD repeat                           4   4.8E-02    4.9    7.0E-01    &#8595;     10   6.8E-05   5.7    2.0E-03 <br />                                                                    Ribonucleoprotein                                                           6   4.2E-02   3.1    4.2E-01                                      &#8595;     20    3.4E-06     3.5   4.3E-05 <br />                                                                    Membrane                      &#8595;    14   1.6E-02    2.1    4.0E-01    &#8593;&#8595;    53   4.2E-16   3.7    1.1E-13    19   1.2E-03     2.3    2.8E-02   &#8593;&#8595;   114    4.3E-22     2.7   2.4E-20 <br />                                                                    Glycoprotein                        6   8.5E-02    2.6    7.8E-01    &#8593;&#8595;    16   1.7E-04   3.2    4.9E-03    19   5.1E-10     6.7    4.3E-08   &#8593;&#8595;    56    2.9E-17     3.7   9.8E-16 <br />                                                                    Cytoplasm                                                                   6   5.9E-04   8.6    4.9E-02    11   1.1E-04     4.8    3.5E-03   &#8595;    256    1.0E-35     2.0   4.5E-33 <br />  <br />                                                                    Determination adult life                                                                                 50   1.3E-37    10.8    1.3E-34         37    6.9E-03     1.5   2.7E-01 <br />                                                                    span <br />                                                                    CUB-like region                                                                                             21   9.0E-20    16.9    3.6E-16         16    4.1E-04     2.7   1.3E-01 <br />                                                                    Signal                                                                     12   1.9E-02   2.2    2.6E-01    28   2.5E-18     9.3    6.5E-16   &#8593;&#8595;    88    2.6E-40     5.5   4.4E-38 <br />                                                                    Iron                                                                       10   2.4E-03   3.5    5.1E-02    20   2.7E-15    12.4    4.5E-13   &#8593;&#8595;    35    3.7E-12     4.1   8.6E-11 <br />                                                                    Secreted                                                                                                    16   2.2E-12    12.7    2.2E-10   &#8593;&#8595;    41    9.8E-21     6.2   5.0E-19 <br />                                                                    Monooxygenase                                                                                               11   2.1E-09    15.6    1.5E-07   &#8593;     19    2.1E-08     5.1   3.0E-07 <br />                                                                    Protease                            5   6.3E-02    3.3    7.6E-01                                           15   3.9E-09     8.3    2.5E-07   &#8593;&#8595;    45    8.7E-18     4.7   3.4E-16 <br />                                                                    Heme                                                                                                        11   3.7E-08    11.6    1.9E-06   &#8593;     24    6.8E-10     4.8   1.2E-08 <br />                                                                    Metalloprotein                                                              7   4.2E-03   4.6    7.9E-02    10   1.9E-07    11.7    8.8E-06   &#8593;     20    9.2E-08     4.4   1.3E-06 <br />                                                                    Storage protein                                                                                              4   1.1E-05    79.2    4.4E-04 <br />                                                                    Lyase                                                                                                       14   3.5E-05     4.1    1.1E-02   &#8593;&#8595;    21    1.1E-10     6.1   2.3E-09 <br />  <br />  <br />  <br />  <br /> Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                                                                    Signal peptide                                                                                              24   3.8E-06     2.8    1.5E-02   &#8595;     70    2.6E-06     1.7   5.0E-03 <br />                                                                    Carboxylic acid metabolic                                                                                   18   6.8E-05     3.1    2.0E-02   &#8593;&#8595;    58    6.8E-07     1.9   1.4E-04 <br />                                                                    process <br />                                                                    Trehalose metabolic process                                                                                  5   5.2E-05    21.2    2.1E-02 <br />  <br />                                                                    Neuropeptide                                                                                                                                  &#8593;     16    3.5E-12    10.6   8.6E-11 <br />                                                                    Transthyretin-like                                                                                           7   1.7E-03     5.4    4.7E-01   &#8593;     24    3.6E-09     3.9   3.6E-06 <br />                                                                    Amidation                                                                                                                                     &#8593;       9   1.1E-06    10.2   1.4E-05 <br />                                                                    Transcription                 &#8595;     8   4.0E-02    2.5    6.7E-01                                                                             &#8593;     34    4.1E-03     1.7   2.7E-02 <br />                                                                    Collagen                                                                                                                                      &#8595;     76    4.5E-67    13.1   2.3E-64 <br />                                                                                                                                                                                                                                                          Page 26 <br />                                                                                NIH-PA Author Manuscript                                            NIH-PA Author Manuscript                                   NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                                                          daf-2 vs. daf-16 + daf-2 RNAi <br />                                                                        Comparison (n):                    mg44, m33 vs. hx546 (382)                mg44, m33 vs. N2DRM (976)             (514)                              Dauer vs. postdauer (2000) <br />  <br />                                                                        Term                               *   No.   P-value   Fold      B-H FDR    *    No.   P-value   Fold   B-H FDR   No.   P-value    Fold   B-H FDR    &#8224;    No.    P-value   Fold    B-H FDR <br />  <br />                                                                        Phosphate/anion transport                                                                                                                            &#8595;    103    1.3E-47     4.6   2.6E-44 <br />                                                                        Cell structure                                                                                                                                       &#8595;     71    2.3E-20     3.2   4.7E-17 <br />                                                                        Muscle expressed genes                                                                                                                               &#8595;    247    1.3E-18     1.7   2.6E-15 <br />                                                                                                                                                                                                                                                                     Ayyadevara et al. <br />  <br />  <br />  <br />  <br />                                                                        Larval development                                                                                                                                   &#8595;    137    2.1E-18     2.1   4.3E-15 <br />                                                                        Growth                                                                                                                                               &#8595;    117    2.1E-15     2.1   4.3E-12 <br />                                                                        Locomotory behavior                                                                                                                                  &#8595;    118    4.4E-14     2.0   8.9E-11 <br />                                                                        Proline-rich region                                                                                                                                  &#8595;     55    2.6E-11     2.7   5.4E-08 <br />                                                                        Physiological process                                                                                                                                &#8595;     73    2.1E-08     1.9   4.4E-05 <br />                                                                        Embryonic development                                                                                                                                &#8595;    149    6.7E-08     1.5   1.4E-04 <br />                                                                        Biosynthesis                                                                                                                                         &#8595;     90    4.8E-07     1.7   9.8E-04 <br />                                                                        RAS cluster C-variable                                                                                                                               &#8595;     41    3.4E-06     2.1   6.9E-03 <br />                                                                        Protein unknown function,                                                                                                                         &#8595;     16    7.3E-06     3.9   1.5E-02 <br />                                                                        DUF141 <br />                                                                        Mitochondrion                                                                                                                                        &#8595;     44    3.1E-06     2.1   4.5E-04 <br />                                                                        Amino acid metabolism                    8   5.2E-02    2.4      9.9E-01                                           10   1.0E-02     2.8    6.3E-01   &#8595;     29    1.3E-05     2.4   2.7E-02 <br />                                                                        Calcium                                                                                                                                              &#8593;       7   5.8E-06     3.9   6.7E-05 <br />                                                                        Lipid metabolic process                                                                                                                              &#8593;&#8595;    65    6.3E-06     1.7   8.6E-04 <br />  <br />                                                                    * <br />                                                                     Direction change age-1(mg44, m333) relative hx546 N2; <br />                                                                    &#8224; <br />                                                                     Direction change dauer larvae <br />  <br />  <br />  <br />  <br /> Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                                                                                                                                                                                                                                                                     Page 27 <br />                                                                                 NIH-PA Author Manuscript                           NIH-PA Author Manuscript                                       NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                Table 3 <br /> Quantitation functional assay gene-expression changes implicated microarray studies specific strong age-1 alleles. Twenty-two genes, <br /> initially implicated microarray analysis transcripts affected strong age-1 alleles (mg44 m333) weak allele (hx546), were <br /> tested transcript abundance real-time polymerase chain reaction (RT-qPCR) 3&#8211;8 biological replicates group. Transcript levels, measured on <br /> a log2 scale, normalized mean internal control genes did alter significantly strains (&#946;-actin, T08G5.3 and <br /> Y71D11.3). Each expressed ratio value N2DRM, identically normalized, presented log2 ratio. Of 22 genes, 21 <br />                                                                                                                                                                                                                                                          Ayyadevara et al. <br />  <br />  <br />  <br />  <br /> (95%) differ significantly transcript level age-1(mg44)-F2 adults wild-type N2DRM adults, P &lt; 0.05 (at criterion, false- <br /> positive expected), 17 (77%) differ significantly P &lt; 0.001 (i.e. 0.0025 threshold Bonferroni correction multiple <br /> assessments). In contrast, just 22 genes showed significant (P &lt; 0.05) changes weaker age-1(hx546) allele attained P &lt; 0.001. <br /> Just gene, evl-14, appeared affected stronger allele, 17 significantly affected age-1(mg44)-F2 (hx546), at <br /> P &lt; 0.01. The table reiterates genes Table 4, employed negative controls RNAi testing. Double-stranded-RNA producing <br /> bacterial clones selected Ahringer RNAi library (Kamath &amp; Ahringer, 2003) target controls, plus 10 genes strongly downregulated <br /> in age-1(mg44) F2 adults (by nine- 32-fold, P &lt; 10&#8722;6) gene declined 3.3-fold. Each RNAi clone fed N2DRM age-1(hx546) <br /> worms, followed tests survival 5-mM hydrogen peroxide. All positive RNAi results confirmed independent experiments using <br /> different nematode expansions testing. <br />  <br />                                                                                                                                                                                                                               RNAi: H2O2 resistance <br />                                                                                            Upstream              Microarray vs.     Log2 (expression) relative N2DRM                                                       in: <br />                                                                                            motifs, <br />  Gene (protein function)                                                                   &#8804;1 kb      Clone      N2DRM     hx546    age-1 (mg44)-F2   daf-16(m26); age-1(mg44)   age-1 (hx546)    daf-16(m26); age-1(hx546)   N2DRM       age-1(hx546) <br />  <br />  (predicted E3 ubiquitin ligase)&#8224;                                                          M3         T12E12.1   &#8595;         &#8595;        &#8722;5.1*****                   0.0                  &#8722;1.0                             &#8722;0.7    1.39*****   1.00 <br />  <br />  dkf-2 (protein kinase D-3)                                                                A, M1      T25E12.4             &#8595;        &#8722;4.3*****                   +0.7                  0.0                             +1.0    1.06        1.02 <br />  <br />  (homologous nucleoside transporter)&#8224;                                                   A          F16H11.3   &#8595;         &#8595;        &#8722;4.1*****                   &#8722;0.4                 &#8722;0.5                             &#8722;0.1    1.36*****   1.05 <br />  <br />  cyp-35C1 (cytochr. P450, CYP2 subfam.)                                                    A          C06B3.3    &#8595;         &#8595;        &#8722;4.1*****                   +0.4                 +1.9                             +0.8    1.15#       2.92****** <br />  <br />  F53H1.3 (unknown, highly conserved)&#8224;                                                                 F53H1.3    &#8595;         &#8595;        &#8722;4.1*****                   0.0                 &#8722;1.0#                             +1.0    1.53*****   2.51****** <br />  (ubiquitin ligase homol. Rb-BP-6)                                                      B, B, M3   F36F2.3    &#8595;         &#8595;                                    0.0                  &#8722;0.5                             &#8722;0.7 <br />  <br />  <br />  <br />  <br />             Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                                                                                                                                     &#8722;3.8*****                                                                                 1.33****    2.68****** <br />  (ATP-dep&#8217;t RNA helicase, DEAD family)                                                                R05D11.4   &#8595;         &#8595;        &#8722;3.6*****                   0.0                  &#8722;0.3                             &#8722;0.3    1.03        &#8211; <br />  <br />  ubl-5 (ubiquitin-like cytoplasmic protein)                                                           F46F11.4   &#8595;         &#8595;        &#8722;3.3*****                   +0.3                 &#8722;0.5                             +0.1    1.10        2.62****** <br />  dna-2 (replication helicase, endonuclease)                                                A          F43G6.1    &#8593;                  &#8722;3.3*****                  &#8722;1.3*                &#8722;1.6**                            &#8722;1.1    2.10****    &#8211; <br />  <br />  let-502 (myotonic dystrophy S/T kinase)                                                   A          C10H11.9   &#8595;         &#8595;        &#8722;3.2*****                  &#8722;1.0*                 &#8722;1.3                             &#8722;0.7    1.02        1.76***** <br />  pms-2 (DNA mismatch repair protein)                                                       A, B       H12C20.2   &#8595;         &#8595;        &#8722;2.8****                    &#8722;0.5                 &#8722;0.4                             &#8722;0.1    &#8211;           &#8211; <br />  <br />  (MTHFR, methylenetetrahydrofolate reductase)                                              A, A       C06A8.1    &#8595;         &#8595;        &#8722;2.0*****                   &#8722;0.4                 &#8722;1.0                             &#8722;0.3    &#8211;           &#8211; <br />  <br />  (aminoacylase ACY1, metallopeptidases)                                                    A, A, B    C10C5.4    &#8595;         &#8595;        &#8722;2.0**                      &#8722;0.1                 +0.1                             &#8722;0.1    &#8211;           &#8211; <br />                                                                                                                                                                                                                                                          Page 28 <br />                                                                                       NIH-PA Author Manuscript                            NIH-PA Author Manuscript                                       NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                                                                                                      RNAi: H2O2 resistance <br />                                                                                                  Upstream               Microarray vs.     Log2 (expression) relative N2DRM                                                       in: <br />                                                                                                  motifs, <br />      Gene (protein function)                                                                     &#8804;1 kb      Clone       N2DRM     hx546    age-1 (mg44)-F2   daf-16(m26); age-1(mg44)   age-1 (hx546)    daf-16(m26); age-1(hx546)   N2DRM      age-1(hx546) <br />  <br />      (prob. ubiquitin conjugating enzyme E2)&#8224;                                                    B          C03H5.3     &#8595;         &#8595;        &#8722;1.8*                       +0.3                 &#8722;0.3                             &#8722;0.5    1.31****   1.10 <br />  <br />      uba-2 (ubiquitin-like activating enzyme 2)                                                             W02A11.4b   &#8595;                  &#8722;1.7*                      &#8722;2.4*                 &#8722;0.4                             &#8722;0.1    &#8211;          &#8211; <br />                                                                                                                                                                                                                                                                Ayyadevara et al. <br />  <br />  <br />  <br />  <br />      ife-1 (translational initiation factor eIF-4E)                                                         F53A2.6     &#8595;         &#8595;        &#8722;1.4#                       0.0                  +0.1                             +0.1    &#8211;          &#8211; <br />  <br />      htp-2 (HORMA-domain protein)                                                                M2         Y73B6BL.2   &#8595;         &#8595;        &#8722;1.3**                      +0.2                &#8722;0.8#                               0.0   &#8211;          &#8211; <br />  <br />      cah-5 (carbonic anhydrase family)                                                           A          R173.1      &#8595;         &#8595;        &#8722;1.3#                       &#8722;0.7                 &#8722;0.3                             +0.3    &#8211;          &#8211; <br />  <br />      lact-3 (beta-lactamase domain protein)                                                      B, M3      M28.6       &#8595;         &#8595;        &#8722;1.2****                  &#8722;0.7**                 &#8722;0.6                             &#8722;0.9    &#8211;          &#8211; <br />  <br />      evl-14 (homol. yeast cohesion prot. Pds5p)                                                  A, A       H38K22.1    &#8593;         &#8593;&#8225;       &#8722;0.2                        &#8722;0.3              &#8722;2.4****                            +0.1    &#8211;          &#8211; <br />  <br />      (orthol. human disease gene, Fukutin)                                                    A          T07D3.4     &#8595;         &#8595;        +1.1**                      &#8722;0.1                 &#8722;0.5                             &#8722;0.7    &#8211;          &#8211; <br />  <br />      (probable G-protein coupled receptor)                                                                  C35A11.1    &#8595;         &#8595;        +2.4*****                  +0.8*                +0.8*                             +0.4    &#8211;          &#8211; <br />  <br />  <br />      Negative controls RNAi assays (from Table 4) <br />      ketn-1 (kettin, actin-binding titin paralog)                                                           F54E2.3                        &#8722;0.6                        &#8722;0.5                 +0.5                             &#8722;0.1    1.04       &#8211; <br />      grd-14 (hedgehog-like signaling protein)                                                               T01B10.2                       +0.8                        +0.1                 +0.4                             +0.6    0.96       &#8211; <br />      (VAP-like; phospholipid inositol metab.)                                                           C10H11.7                       +4.4*****                  +0.9*                +1.6*                             +1.3    0.98       &#8211; <br />  <br />  <br /> Significance difference N2DRM (expression), empty-vector control (RNAi), 2-tailed t-test: <br /> # <br />  P &lt; 0.05; <br /> * <br />  P &lt; 0.01; <br /> ** <br />     P &lt; 0.001; <br />  <br />  <br />  <br />  <br />                   Aging Cell. Author manuscript; available PMC 2010 December 1. <br /> *** <br />       P &lt; 1E&#8211;4; <br /> **** <br />        P &lt; 1E&#8211;5; <br /> ***** <br />         P &lt; 1E&#8211;6; <br /> ****** <br />           P &lt; 1E&#8211;10. <br />  <br /> &#8211;, assessed; N.S., significant; &#8595;, lower age-1(mg44)-F2 adults N2DRM age-1(hx546); &#8593;, higher age-1(mg44)-F2 adults N2DRM age-1(hx546); A, &#8216;DAE&#8217;, CTTATCA; B, <br /> &#8216;DBE&#8217;, GTAAAYAA; M1, TTCWGAAAAT; M2, CTACAGTAA; M3, AGAGAMGMAGA. <br /> &#8224; <br />  RNAi shown extend lifespan. <br />                                                                                                                                                                                                                                                                Page 29 <br />                                                                     NIH-PA Author Manuscript                                           NIH-PA Author Manuscript                                             NIH-PA Author Manuscript <br />                                                                     &#8225; <br />                                                                      Note RT-qPCR results calculated relative N2DRM. Thus, microarray RT-qPCR results evl-14 consistent regard age-1 alleles mg44 and/or m333 vs. hx546, because <br />                                                                     &#8722;0.2 &gt; &#8722;2.4. <br />                                                                                                                                                                                                                                                                             Ayyadevara et al. <br />  <br />  <br />  <br />  <br /> Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                                                                                                                                                                                                                                                                             Page 30 <br />                                                                     NIH-PA Author Manuscript                                             NIH-PA Author Manuscript                                            NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                                  Table 4 <br />                                                                    Quantitation functional assay expression changes genes weakly implicated microarrays. Thirty-eight genes initially suggested be <br />                                                                    affected strong age-1 alleles (mg44 m333) weak (hx546) wild-type (N2DRM) age-1 allele, based microarrays. <br />                                                                    When transcript abundances assessed real-time polymerase chain reaction (RT-qPCR), 14 38 genes (37%) differed fourfold in <br />                                                                    transcript level age-1(mg44)-F2 adults wild-type N2DRM adults, P &lt; 10&#8722;5 (well 0.0025 threshold Bonferroni <br />                                                                    correction multiple assessments). In contrast, seven 38 genes (18%) showed significant changes (P &lt; 0.01) weaker age-1(hx546) allele, <br />                                                                                                                                                                                                                                                                     Ayyadevara et al. <br />  <br />  <br />  <br />  <br />                                                                    just altered &#8805;4-fold. Two genes (T10B11.7 lbp-6) appeared affected stronger allele, 13 genes were <br />                                                                    significantly affected age-1(mg44)-F2 (hx546), P &lt; 0.01. Double-stranded RNAi clones (Kamath &amp; Ahringer, 2003) selected to <br />                                                                    target seven genes strongly downregulated age-1(mg44) F2 adults (by 4- 20-fold, P &lt; 10&#8722;5), gene <br />                                                                    (lbp-6) downregulated primarily age-1(hx546) (P &lt; 0.01), control genes unchanged upregulated age-1(mg44). <br />  <br />                                                                                                                                                Log2 (expression) relative N2DRM                                                       RNAi: H2O2 resistance <br />                                                                                                              Upstream <br />                                                                     Gene (protein function)                  motifs, &#8804;1 kb   Clone             age-1 (mg44)-F2   daf-16(m26); age-1(mg44)   age-1 (hx546)    daf-16(m26); age-1(hx546)   N2DRM       age-1(hx546) <br />  <br />                                                                     Decreased age-1(mg44) F2 adults (7 confirmed genes, 35% 20 tested) <br />                                                                     (unknown; homol. helicase,            M2              T10B11.7          &#8722;4.3*****                  &#8722;1.0*               &#8722;5.1***                            &#8722;1.0    1.05        &#8211; <br />                                                                     translation IF) <br />                                                                     cpr-1 (cys-protease, sim. cathepsin   A, A            C52E4.1           &#8722;4.3*****                  &#8722;0.7#                 &#8722;0.5                             +0.3    3.14*****   &#8211; <br />                                                                     B)&#8224;,&#8225; <br />                                                                     aka-1 (A-kinase anchor protein;                          D1022.7           &#8722;4.1****                   &#8722;1.0#                 &#8722;1.5                               0.0   1.53*****   1.50****** <br />                                                                     FYVE Zn-finger) <br />                                                                     ung-1 (uracil N-glycosylase 1)                           Y56A3A.29         &#8722;3.8*****                   &#8722;0.1                 &#8722;0.7                             &#8722;0.5    2.75***     &#8211; <br />  <br />                                                                     sna-1 (snRNP-binding protein 1)                          W02F12.6          &#8722;3.2*****                  &#8722;0.6#                &#8722;0.9*                             &#8722;0.6    1.6**       &#8211; <br />  <br />                                                                     cdc-48.1 (ATPase, proteasomal            B               C06A1.1           &#8722;2.8*****                   0.0                 &#8722;1.0*                             &#8722;0.5    1.0         &#8211; <br />                                                                     sorting) <br />                                                                     rab-10 (intestinal RAS-like GTPase)      A               T23H2.5           &#8722;2.0****                    &#8722;0.4                &#8722;0.8*                             &#8722;0.1    1.82***     &#8211; <br />                                                                     &#8225; <br />  <br />  <br />  <br />  <br /> Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                                                                     ketn-1 (kettin, actin-binding titin                      F54E2.3           &#8722;0.6                        &#8722;0.5                 +0.5                             &#8722;0.1    1.03        1.01 <br />                                                                     paralog) <br />                                                                     lbp-6 (lipocalin-like fatty acid-                        W02D3.5           &#8722;0.5                        +0.1                &#8722;1.4*                             &#8722;0.8    1.37*****   2.51***** <br />                                                                     binding protein)&#8224; <br />                                                                     grd-14 (hedgehog-like signaling                          T01B10.2          +0.8                        +0.1                 +0.4                             +0.6    1.00        &#8211; <br />                                                                     protein) <br />                                                                     Increased age-1(mg44) F2 adults (7 confirmed genes, 39% 18 tested) <br />                                                                     (homol. cruciform DNA binding                         Y75B7B.1          +2.7****                    &#8722;0.3                 +0.1                             &#8722;0.5    &#8211;           &#8211; <br />                                                                     protein) <br />                                                                     hsp-12.3 (small heat-shock protein)      B, B            F38E11.1          +3.7*****                  &#8722;0.7*                  0.0                             &#8722;0.1    &#8211;           &#8211; <br />                                                                                                                                                                                                                                                                     Page 31 <br />                                                                         NIH-PA Author Manuscript                                           NIH-PA Author Manuscript                                                 NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                Log2 (expression) relative N2DRM                                                              RNAi: H2O2 resistance <br />                                                                                                                Upstream <br />                                                                         Gene (protein function)                motifs, &#8804;1 kb   Clone           age-1 (mg44)-F2     daf-16(m26); age-1(mg44)        age-1 (hx546)    daf-16(m26); age-1(hx546)   N2DRM      age-1(hx546) <br />  <br />                                                                         (VAP-like; phospholipid, inositol      A, B            ZC196.2         +4.2*****                      &#8722;0.1                     &#8722;1.0                             &#8722;1.3    &#8211;          &#8211; <br />                                                                         metab.) <br />                                                                         (VAP-like; phospholipid, inositol                      C10H11.7        +4.4*****                      +0.9*                   +1.6*                             +1.3    0.98       N.S. <br />                                                                         metab.) <br />                                                                                                                                                                                                                                                                           Ayyadevara et al. <br />  <br />  <br />  <br />  <br />                                                                         nspd-2 (protein homologous                          K07F5.5         +6.3*****                      +0.9                    +4.9**                            +1.1    &#8211;          &#8211; <br />                                                                         prions, RNA-BP) <br />                                                                         cyc-2.2 (cytochrome C, C.    M2              ZC116.2         +6.5*****                     &#8722;2.0*                     &#8722;0.7                             &#8722;0.5    &#8211;          &#8211; <br />                                                                         elegans) <br />                                                                         clec-60 (C-type lectin, expressed   A               ZK666.6         +10.1*****                    &#8722;1.3*                     &#177;0.8                             &#8722;1.0    &#8211;          &#8211; <br />                                                                         intestine) <br />  <br />                                                                    Significance difference N2DRM (expression), empty-vector control (RNAi), 2-tailed t-test: <br />                                                                    # <br />                                                                     P&lt; 0.05; <br />                                                                    * <br />                                                                     P&lt; 0.01; <br />                                                                    ** <br />                                                                        P&lt; 0.001; <br />                                                                    *** <br />                                                                          P&lt; 1E&#8211;4; <br />                                                                    **** <br />                                                                           P&lt; 1E&#8211;5; <br />                                                                    ***** <br />                                                                            P&lt; 1E&#8211;6; <br />                                                                    ****** <br />                                                                             P &lt; 1E-10. <br />  <br />                                                                    &#8211;, assessed; N.S., significant; motifs seen 1 kb upstream gene: A, &#8216;DAE&#8217;; B, &#8216;DBE&#8217;; M2, CTACAGTAA (see Table 3 key). <br />                                                                    &#8224; <br />                                                                     Genes shown extend life inhibited RNAi. <br />  <br />  <br />  <br />  <br /> Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                                                                    &#8225; <br />                                                                     RNAi reported extend lifespan directed cpr-1 (Murphy et al., 2003) rab-10 (Hansen et al., 2005). <br />                                                                                                                                                                                                                                                                           Page 32 <br />                                                                     NIH-PA Author Manuscript                                         NIH-PA Author Manuscript                                          NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                            Table 5 <br />                                                                    Peroxide-resistance survival statistics C. elegans strains N2DRM age-1(hx546) fed dsRNA-expressing bacteria generations <br />  <br />                                                                                                Mean survival (h), &#177;SD             Mean Survival (h), &#177;SD <br />                                                                     Gene targeted RNAi              Experiment 1      % gain      Experiment 2          % gain    Significance difference vs. control* <br />  <br />                                                                     N2DRM treated generations <br />                                                                                                                                                                                                                                  Ayyadevara et al. <br />  <br />  <br />  <br />  <br />                                                                       None (vector)                          4.7 &#177; 1.0        &#8211;          4.6 &#177; 1.1              &#8211;    &#8211; <br />                                                                       F16H11.3                               6.3 &#177; 1.0       34          6.3 &#177; 0.9             37    &lt; 10&#8722;6 <br />                                                                       ubl-5                                  5.0 &#177; 0.8        6          5.2 &#177; 1.1             14    0.07 <br />                                                                       F36F2.3                                5.9 &#177; 1.0       27          6.3 &#177; 0.9             38    &lt; 10&#8722;6 <br />                                                                       aka-1                                  7.1 &#177; 0.8       53          7.0 &#177; 0.9             53    &lt; 10&#8722;7 <br />                                                                       let-502                                4.8 &#177; 1.0        3          4.6 &#177; 1.4              0    N.S. <br />                                                                       F53H1.3                                7.0 &#177; 1.1       51          7.1 &#177; 0.9             55    &lt; 10&#8722;7 <br />                                                                       C03H5.3                                5.8 &#177; 1.3       24          6.3 &#177; 1.1             37    &lt; 10&#8722;5 <br />                                                                       dkf-2                                  5.2 &#177; 0.9       12          4.8 &#177; 1.3              5    N.S. <br />                                                                       cyp-35C1                               5.2 &#177; 0.8       12          5.4 &#177; 0.9             18    &lt; 0.02 <br />                                                                       lbp-6                                  6.2 &#177; 1.1       32          6.5 &#177; 0.9             42    &lt; 10&#8722;6 <br />                                                                       ife-1                                  5.5 &#177; 0.9       18          5.7 &#177; 0.9             25    &lt; 10&#8722;4 <br />                                                                       T12E12.1                               6.4 &#177; 1.3       37          6.4 &#177; 1.2             40    &lt; 10&#8722;6 <br />                                                                     age-1(hx546) treated generations <br />                                                                       None (vector)                          6.1 &#177; 1.2        &#8211;          6.1 &#177; 1.2              &#8211;    &#8211; <br />                                                                       F16H11.3                               6.3 &#177; 1.2        3          6.5 &#177; 1.0              7    N.S. <br />                                                                       ubl-5                                 15.5 &#177; 5.0      155         16.4 &#177; 4.1            169    &lt; 10&#8722;12 <br />                                                                       F36F2.3                               15.2 &#177; 5.0      150         17.4 &#177; 3.2            185 <br />  <br />  <br />  <br />  <br /> Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                                                                                                                                                                      &lt; 10&#8722;12 <br />                                                                       aka-1                                  9.0 &#177; 2.6       48          9.3 &#177; 2.4             52    &lt; 10&#8722;10 <br />                                                                       let-502                               10.6 &#177; 4.1       74         10.8 &#177; 3.8             77    &lt; 10&#8722;8 <br />                                                                       F53H1.3                               15.2 &#177; 6.3      150         15.3 &#177; 6.1            151    &lt; 10&#8722;12 <br />                                                                       C03H5.3                                6.5 &#177; 1.3        7          6.6 &#177; 1.2              8    0.07 <br />                                                                       dkf-2                                  6.2 &#177; 0.9        1          6.2 &#177; 1.0              2    N.S. <br />                                                                       cyp-35C1                              17.4 &#177; 4.6      185         18.2 &#177; 3.4            198    &lt; 10&#8722;12 <br />                                                                       lbp-6                                 15.3 &#177; 4.0      151            N.D.                 &#8211;    &lt; 10&#8722;8 <br />                                                                       T12E12.1                               6.2 &#177; 1.1        1            N.D.                 &#8211;    N.S. <br />                                                                                                                                                                                                                                  Page 33 <br />                                                                     NIH-PA Author Manuscript   NIH-PA Author Manuscript        NIH-PA Author Manuscript <br />                                                                                                                          For group, n = 20. <br />  <br />                                                                                                                          N.S., significant. <br />                                                                                                                          * <br />                                                                                                                           Gehans-Wilcoxon log-rank test experiments 1 2 combined. <br />                                                                                                                                                                                             Ayyadevara et al. <br />  <br />  <br />  <br />  <br /> Aging Cell. Author manuscript; available PMC 2010 December 1. <br />                                                                                                                                                                                             Page 34 <br />                                                                         NIH-PA Author Manuscript                                          NIH-PA Author Manuscript                                        NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                                 Table 6 <br />                                                                     Lifespan survival statistics N2DRM worms fed dsRNA-expressing bacteria <br />  <br />                                                                        Gene targeted RNAi      n      Median survival (d)   Mean survival (d)   Standard deviation (d) [SEM]   Significance difference control* <br />  <br />                                                                        (A) RNAi treatment 2 generations <br />                                                                                                                                                                                                                                     Ayyadevara et al. <br />  <br />  <br />  <br />  <br />                                                                          None (empty vector)      32            15.4                 16.7                    2.4 [0.4]            &#8211; <br />                            <br /> </body></html>