<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Bioinformatics                                                                                                                                                                                                                                               BioMed Central <br />  <br />  <br />  <br /> Research                                                                                                                                                                                                                                                       Open Access <br /> On deducing causality metabolic networks <br /> Chiara Bodei*1, Andrea Bracciali1 Davide Chiarugi2 <br />  <br /> Address: 1Dipartimento di Informatica, Universit&#224; di <span id='am-78' about='obo:NCBITaxon_1340058' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_516942 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_6804 obo:NCBITaxon_116706 obo:NCBITaxon_197562 obo:NCBITaxon_116704 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="Pisa" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-82'  ></span>Pisa</span>, I-56127, Pisa, Italy 2Dipartimento di Scienze Matematiche e Informatiche, <br /> Universit&#224; di Siena, I-53100, Siena, Italy <br /> Email: Chiara Bodei* - chiara@di.unipi.it; Andrea Bracciali - braccia@di.unipi.it; Davide Chiarugi - chiarugi3@unisi.it <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> from Seventh International Workshop Network Tools Applications Biology (NETTAB 2007) <br /> Pisa, Italy. 12-15 June 2007 <br />  <br /> Published: 25 April 2008 <br /> BMC Bioinformatics 2008, 9(Suppl 4):S8                                                                                                                  doi:10.1186/1471-2105-9-S4-S8 <br />  <br /> &lt;supplement&gt; &lt;title&gt; &lt;p&gt;A Semantic Web Bioinformatics: Goals, Tools, Systems, Applications&lt;/p&gt; &lt;/title&gt; &lt;editor&gt;Paolo Romano, Michael Schroeder, Nicola Cannata Roberto Marangoni&lt;/editor&gt; &lt;note&gt;Research&lt;/note&gt; &lt;/supplement&gt; <br />  <br />  <br />  <br />  <br /> <span id='am-21' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-24'  ></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> article available from: http://www.biomedcentral.com/1471-2105/9/S4/S8 <br /> &#169; 2008 Bodei et al.; licensee BioMed Central Ltd. <br /> This open access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                                                                 Abstract <br />                                                                 Background: Metabolic networks present complex interconnected structure, whose <br />                                                                 understanding general non-trivial task. Several formal approaches developed to <br />                                                                 support investigation networks. One relevant problems context the <br />                                                                 comprehension causality dependencies molecules involved metabolic process. <br />                                                                 Results: We apply techniques formal methods computational logic develop an <br />                                                                 abstract qualitative model metabolic networks <span id='am-29' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> determine possible causal <br />                                                                 dependencies. Keeping mind expressiveness ease use, aimed providing: i) a <br />                                                                 minimal notation represent causality biochemical interactions, ii) automated tool <br />                                                                 allowing human experts easily vary conditions silico experiments. We exploit reading of <br />                                                                 chemical reactions terms logical implications: starting description metabolic <br />                                                                 network terms reaction rules initial conditions, chains reactions, causally depending one <br />                                                                 another, automatically deduced. Both components initial state the <br />                                                                 clauses ruling reactions easily varied new trial experiment started, according to <br />                                                                 what-if investigation strategy. Our approach aims exploiting computational logic formal <br />                                                                 modeling framework, available, naturally close human reasoning. It <br />                                                                 directly leads executable implementations support, perspective, various reasoning <br />                                                                 schemata. Indeed, abstractions supported computational counterpart, based a <br />                                                                 Prolog implementation, allows representation language closely correspondent the <br />                                                                 adopted chemical abstract notation. The proposed approach validated results <br />                                                                 regarding gene knock-out essentiality model metabolic network <span id='am-32' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45'  ></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span>Escherichia coli</span> <br />                                                                 K12, relevant coherence available wet-lab experimental <span id='am-68' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-69' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-72'  ></span>data</span>. <br />                                                                 Conclusions: Starting presented work, goal provide effective analysis toolkit, <br />                                                                 supported efficient full-fledged computational counterpart, aim fruitfully driving in <br />                                                                 vitro experiments effectively pruning non promising directions. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                Page 1 10 <br />                                                                                                                                                                                                                                          (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                   http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  <br />  <br /> Background                                                     source molecule reaction so <br /> Understanding relationships elements           forth. In words, abstract away quantities, <br /> involved biologic interaction networks,         stoichiometric proportions, kinetical thermodynamics <br /> functioning cellular metabolism, relevant problem      parameters involved production P. <br /> in Systems Biology. In words [1], &#8220;diagrams          Noticeably, abstract actual dynamics of <br /> interconnections represent sort static roadmaps,      reactions. Intuitively speaking, project reactions a <br /> what really seek know traffic patterns,      &#8220;flat&#65533;? temporal scale availability M N is <br /> such patterns emerge, control them&#65533;?.            spoiled production P, metab- <br /> Having formal description interconnections      olites caused M N. This abstrac- <br /> methodology perform software simulation              tion quantities, &#8220;infinite copies&#65533;? M N <br /> these patterns are, help orientating vitro        result available. Because abstraction, the <br /> experimentation. Under regard, causality play      reaction leading P actually represents possibility for <br /> important role finding chains reactions connect     P generated, caused, vivo, presence M <br /> the parts interest, e.g. determining      N. Indeed, model gives over-approximation of <br /> causal correlations molecules appar-        set actual pathways, possibly including some <br /> ently correlated. In general, proposed models com-      pathways actually prevented, instance, <br /> plex systems code lot information determining         lack suitable quantity reactants an <br /> possible correlations causal dependencies       inadequate temperature. The payoff abstraction <br /> straightforward computationally expensive.                  adopted terms expressiveness language and <br />                                                                effectiveness computational counterpart. Abstracting <br /> We focus metabolic networks, i.e. set       quantitative issues prevent reasoning about <br /> cellular biochemical pathways involved energy man-          dynamical features (bio)chemical objects. <br /> agement synthesis structural components.         However, makes possible consideration <br /> Biochemical pathways typically composed chains       aspects systems (typically large biochemical <br /> enzymatically catalyzed chemical reactions inter-      networks) kinetical thermodynamical param- <br /> connected complex way. Starting composi-         eters scarcely known. <br /> tion local, understood behaviour, study the <br /> overall emerging behaviour metabolic networks               Chains causal reactions be, step step, automati- <br /> appears difficult.                                             cally deduced. To aim, exploit analogy between <br />                                                                logical implications chemical reactions, interpret- <br /> We apply techniques formal methods               ing reaction M N produces P logical <br /> computational logic order develop abstract qual-      clause stating M N imply P. Our method sup- <br /> itative model metabolic networks, focussing causal-      ported computational logic counterpart, based a <br /> ity. Keeping mind expressiveness ease use,      Prolog implementation bottom-up semantics. This <br /> we aimed providing: i) minimal notation represent      allows compute set metabolites can <br /> causality biochemical interactions, ii) auto-        produced consequences given set rules and <br /> mated tool allowing human expert easily vary con-       starting set initially available metabolites. This <br /> ditions silico experiment. We introduce simple     step-wise determined repeatedly adding metabo- <br /> and skeletal notation, inspired biochemical reactions,      lites immediately caused application of <br /> to emphasize causality aspects concerned with.      rules set so-far produced metabolites. We <br /> The choice relying computational logic, &#8220;pro-      relate computational construction orig- <br /> vides straightforward intuitive representation        inal model prove convergence properties proc- <br /> human reasoning&#65533;? [2], appeared particularly suitable       ess. <br /> in multidisciplinary context work. On one <br /> hand, palatable biologists, close   Despite abstract working hypotheses adopted, our <br /> to biochemical intuition. On hand,      framework revealed sufficient provide meaningful <br /> direct computational counterpart, scien-        approximations possible chains reactions during <br /> tists build order devise needed analysis    experiments. Moreover, approach facilitates thinking <br /> tool.                                                          revising biological model itself, making easy to <br />                                                                vary components initial state the <br /> In notation, biochemical reactions given            clauses rule reactions. <br /> abstract representation: record rela- <br /> tions source target elements       The proposed approach serve sort &#8220;what-if&#65533;? <br /> single reaction, e.g. molecules M N        analysis, repeatedly exploring different scenarios, each <br /> the molecule P produce. In turn, P           derived different set working hypotheses. <br />  <br />  <br />                                                                                                                    Page 2 10 <br />                                                                                              (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                    http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  <br /> Our tool allows rapidly evaluate impact            Among formalisms developed concurrency <br /> changes hypotheses particular observable out-       theory applicable Systems Biology, recall below <br /> come. Thus, obtain interactive effective analy-       relevant purposes. Petri Nets [4] offer a <br /> sis, used differentiate promising       way graphically represent structure distributed <br /> causal relations deduced, deserve tested         systems. They successfully applied mod- <br /> vitro, instead pruned.                   elling metabolic pathways simple genetic networks <br />                                                                 (see e.g. [5,6] cite few). They model pathways <br /> We validated approach studying robust-          networks terms dynamic evolutions. <br /> ness metabolic network E. coli K12. Selected <br /> genes knocked-out disabling rules              Process calculi interactions communications <br /> regarding corresponding encoded enzymes. Results            independent agents processes. The underlying <br /> are coherent actual biological behaviour,              <span id='am-13' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-17'  ></span>idea</span> biological seen concurrent <br /> observed vitro reported [3] &#8220;Geno          systems. In particular, &#960;&#8211;calculus [7] Ambient Calcu- <br /> Base&#65533;? (http://ecoli.aist-nara.ac.jp/), database entirely      lus [8] transferred theoretical computer <br /> dedicated E. coli K-12.                                      science setting biology setting, leading suitably <br />                                                                 extended biological versions them, Bio- <br /> This paper organized follows. Next, discuss       chemical stochastic &#960;-calculus [9,10] BioAmbients <br /> related work present formal framework,             [11]. Also version CCS, RCCS [12] intro- <br /> illustrating biochemical processes represented       duced address biological issues. Other calculi have <br /> inside it. Results experiments metabolic         instead specifically defined biological modelling, <br /> network E. coli K-12 metabolic genotype follow.          &#954;-calculus [13], Brane calculi [14] <span id='am-7' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-8' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-10'  ></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Beta</span> Bind- <br /> Conclusions followed section Methods,         ers [15]. Chemistry invoked explicitly in <br /> we report techniques used.                               process algebraic context years com- <br />                                                                 ing systems biology. In [16], abstract machine based <br /> Related work comparison                                     chemical metaphor introduced: states chem- <br /> Our work included recent research trend           ical solutions floating molecules interact <br /> which exploits established theories techniques         according reaction rules. Rules specify produce <br /> from formal methods order support interpreta-         new molecules old ones. <br /> tion big raw biological data now <br /> available. By using logic, slightly diverge line    Closer approach work presented [17], <br /> of research biological modelling inspired        authors apply causal semantics &#960;-calcu- <br /> the use concurrent models. Concurrency theory offers       lus order biochemical processes. The proc- <br /> wide choice models naturally expressing causality,      ess computations obtained quite accurately <br /> is essential notions. Nevertheless, causality   capture reflect behaviour biological systems <br /> natural notion logic framework. Concurrent          causality key role enhancing precision such <br /> models focussed description dynamic           simulations. Our starting point quite similar, our <br /> behaviour systems. Considering aspects           model description biological systems given in <br /> can computationally demanding. This led        terms molecular entities reaction rules implic- <br /> choice abstract model, &#8211; men-          itly code causal relationships, possible <br /> tioned &#8211; notion time abstracted away. An        pathways. The causal semantics [17] based an <br /> abstraction similar performed posteriori       enhanced form transition systems [18], makes it <br /> on concurrent model, instance resorting static      possible capture truly concurrent aspects like causality <br /> analysis techniques, offer static approximations       interleaving setting, like process algebraic one. <br /> the dynamic behaviour. Usually, static techniques extract       There proposals, introduced same <br /> information systematically inspecting specification      aim, instance distributed transition systems in <br /> of dynamic behaviour systems. In approach            [19]. <br /> instead, want infer information directly inspect- <br /> ing set reactions, modulo abstractions dis-       It interesting observe results com- <br /> cussed above, skipping specification            parable obtained using quite efficient <br /> chemical dynamics. Resorting static techniques repre-        static technique like Control Flow Analysis (CFA) [20,21], <br /> sents typical way drastically reduce computational     applied &#960;-calculus. In settings, reaction <br /> cost study possible dynamic evolu-        M N produces P abstractly mod- <br /> tions system. The price pay amounts loss        elled synchronization process M N a <br /> precision, usually provide over-approximations       shared channel c, process P continuation. <br /> of behaviour.                                               Notice case expressiveness &#960;-cal- <br />  <br />  <br />                                                                                                                     Page 3 10 <br />                                                                                               (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                   http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  <br /> culus and, particular, name-passing         induction aim learning general rules these <br /> needed. In simplest form, CFA considers effective   abduced hypotheses. The representation language has <br /> the communications occur given              ad-hoc devised. The predictive accuracy increases <br /> shared channels, disregarding actual viability,      number training examples. This methodology <br /> for instance synchronization dynamics. In modelling         richer representation language aims at <br /> reactions way, CFA consider       addressing different <span id='am-18' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> problems different <br /> possible consumption reactants. Under regard,      experimental setting. Differently, [2], [27], are <br /> approach offer analogous result,          based quite general class languages, extended to <br /> skeletal abstract setting. Having over-approxima-       deal biological context. These language, also <br /> tion exact behaviour system, case   known action event calculi, suited the <br /> static analysis framework, means     non-monotone evolution dynamical environment, <br /> events prediction does include hap-    specifically biological networks. Abduction again <br /> pen, included, events happen, i.e.     used expressive means compute/deduce explana- <br /> are possible.                                             tions missing information dynamical nature <br />                                                                world. Although expressive our <br /> Another recent proposal shares similarities     approach, proposals deal explicit <br /> our approach based biochemical abstract      treatment dynamics systems, differently from <br /> machine BIOCHAM [22], offers formal mod-          proposal strives simplicity order address, <br /> elling environment biochemical processes, oriented         approximately, causality large metabolic net- <br /> towards qualitative aspects. It based rule-centered    works. <br /> language specifying biochemical systems. Differently <br /> from approach, BIOCHAM semantics takes con-           Finally, important refer wide usage graphs <br /> sideration dynamics systems provides tools      representation language, traditionally close biologist <br /> querying temporal properties systems using         experience. Amongst huge set papers adopting <br /> Computation Tree Logic. Refraining dealing           graphs model bio-networks, work closer those <br /> quantities, BIOCHAM offers explicit controls reactant       use arcs nodes directly represent reactions. For <br /> consumption reactions and, default, pos-     instance, approach quite corresponding logical <br /> sibilities considered. This suggests develop-      view reactions &#8211; far abstraction level con- <br /> ments approach, where, currently, case     cerned &#8211; [28]. There, starting chosen qualita- <br /> no consumption admitted. The reason choice      tive interpretation biochemistry analogous ours, the <br /> effectiveness: way, branching semantics avoided.    authors focus topology metabolic networks, <br />                                                                aiming defining representative measure network <br /> Still close approach BIOCHAM Path-       activity, Synthetic Accessibility (the number chemi- <br /> way logic (see e.g. [23,24]), rewriting logic used    cal reactions needed transform set initial metabo- <br /> for modelling biological processes. Rewrite rules     lites set output ones). This notion been <br /> local changes molecular patterns cause them.      tested predicting viability mutant strains accu- <br /> Rules concurrently applied corresponds      racy results comparable ours, differ- <br /> the actual possibility biological compartments inde-     ent perspective. <br /> pendently evolve. This offers basis silico experi- <br /> ments advanced forms symbolic analysis. We          Results discussion <br /> choose alternative approach, resorting con-     A constructive formalisation metabolic causality <br /> current semantics, accordance aims.    We clarify interpretation metabolic causal rela- <br />                                                                tions introducing notation used represent bio- <br /> Amongst computational logic tradition, largely       chemical reactions. We consider assumptions on <br /> influences proposal, interesting cite        idealised notatation relies, discussing why, <br /> recent proposals based Abductive Logic Programming.         according aims, assumptions consid- <br />                                                                ered viable working hypotheses. Such notation repre- <br /> Complexity bio-networks, understood lack com-         sents adopted formalisation biochemical <br /> plete knowledge, addressed means            reactions. Then, chosen hypotheses, a <br /> capability making assumptions provided abduction.        formal account fact metabolite, also <br /> This approach applied gene networks [25],       called reactant, caused network. In section <br /> with motivations similar ours. That proposal based       Methods, present computational counterpart <br /> on combination Abduction Induction: abduction         formalisation relate computational <br /> allows inference observable effects (see [26])       construction adopted model. <br /> and used generate hypotheses, while <br />  <br />  <br />                                                                                                                    Page 4 10 <br />                                                                                              (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                    http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  <br /> From biochemical reactions causal relations                  Moreover, let imagine overall process be <br /> Let consider biochemical reaction written clas-     described includes (some steps of) upper the <br /> sical notation:                                                 glycolytic pathway. Then, set R contain reaction <br />                                                                 rules like: <br />                   mM + nN &#8594; pP + qQ                      (1) <br />                                                                                       glc6p &#8594; fru6p                            (4) <br /> M, N, P Q <span id='am-26' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> involved m, n, p q <br /> the corresponding stoichiometric coefficients. Expression <br />                                                                                fru6p ATP &#8594; fru16p ADP                          (5) <br /> (1) indicates that, reaction occurs, certain <br /> amount M N certain P Q <br /> according stoichiometric proportions. Besides stoi-                         fru16p &#8594; gap dhap                           (6) <br /> chiometry, order exhaustively characterize chemi- <br /> cal reaction, account number                                 dhap &#8594; gap                              (7) <br /> factors related thermodynamics kinetics repre- <br /> sent propensity reaction occur rate at <br /> which reactants eventually products.                                             gap &#8594; dhap                             (8) <br />  <br /> Since interested investigating causality relation-                    gap nad &#8594; bpg13 nadh                           (9) <br /> ships only, omit description of fac- <br /> tors cited above. Then, abstract                 acronyms glc6p, fru6p, fru16p, gap, dhap, nad, <br /> representation (1) follows:                               nadh, bpg13 stand Glucose 6-phosphate, Fructose 6- <br />                                                                 phosphate, Fructose1,6-bisphosphate, Glyceraldehyde 3- <br />                       M N&#8594;P Q                            (2)    phosphate, Dihydroxyacetone phosphate, NAD+, NADH <br />                                                                 1,3 Bisphosphoglicerate, respectively. <br /> We expression reaction rule. It simply states <br /> that presence M N represents possibil-      The rules (4), (5), (6), (7) (8) together, (9) <br /> ity P Q produced caused. If applicable, i.e.   reaction catalyzed enzymes <span id='am-83' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-84' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87'  ></span>phos</span>- <br /> it known premises M N producible         phoglucose isomerase, 6-phosphofructo 1-chinase, fruc- <br /> a network, P Q producible. Informally         tose bisphosphate aldolase, triose phosphate isomerase, <br /> speaking, standard dynamical reading (2) be:         glyceraldehyde 3-phosphate dehydrogenase, respectively. <br />                                                                 Note reactant appearing right the <br />                   M N&#8594;M N P Q                            (3)    operator &#8594; appear left another <br />                                                                 rule. <br /> indeed, considering evolution network, <br /> reactants consumed step.                         We eventually decompose rules simpler monadic <br />                                                                 rules, e.g. (2) written as: <br /> The description causal relations metabolic net- <br /> work defining set metabolites initially                             M N&#8594;P                                 (10) <br /> present I set reaction rules R how <br /> new metabolites produced. Initial metabolites                                M N &#8594;Q                                (11) <br /> represented form rules premises like &#8594; <br /> P.                                                              This transformation causality preserving sense of <br />                                                                 following proposition. <br /> Example 1 <br /> Let consider experiment pathway occurring         Proposition 1 <br /> in environment providing &#945;-D Glucose, Glycerol           Let M1 &#65533;?? &#8230; &#65533;?? Mm &#8594; P1 &#65533;?? &#8230; &#65533;?? Pn applicable rule, <br /> oxygen. By using suitable acronyms like glc, gly o2,        i.e. M1, &#8230;, Mn producible, P1, &#8230;, Pn pro- <br /> respectively, initial conditions experiment      ducible. Then, simplified rules M1 &#65533;?? &#8230; &#65533;?? Mm &#8594; <br /> I include:                                                 Pi, = 1, &#8230;, n, applicable order, their <br />                                                                 application makes P1, &#8230;, Pn producible, well. <br />                            &#8594; glc <br />                                                                 The proposition holds causality behaves in <br />                            &#8594; gly                                monotone way, application subsequent <br />                                                                 rules spoil fact metabolite produci- <br />                            &#8594; o2                                 ble. <br />  <br />  <br />                                                                                                                    Page 5 10 <br />                                                                                              (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                                                              http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  <br /> The fact reactant caused network pre-                                              Note experimental evidence possible produc- <br /> cise means following <span id='am-67' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> relating                                                    tion metabolite vivo silico, suggest <br /> metabolite chain reactions                                                    need revision adopted biological model. <br /> exist, starting initial state I according set <br /> of rules R.                                                                                               The set reactants &#8220;motivated&#65533;?, sense <br />                                                                                                           illustrated, explanation starting ini- <br /> Definition 1                                                                                              tial pool given reactants specific set rules can <br /> Given set rules R, initial state I, reactant let                                          automatically determined. Amongst different and <br /> us consider following construction:                                                                   equivalent approaches used, fol- <br />                                                                                                           low logic-deductive interpretation, line the <br />              &#9127;                                     &#8594; &#8712; I                                           explanatory approach adopted. For details, sec- <br />              &#9130; <br /> ER ,I (a ) = &#9128; &#9123;&#9121; ER ,I ( a1 ) ,&#8230; , ER ,I ( )&#9124;&#9126; a1 &#8230; &#8594; &#8712; R ER ,I ( ai ) &#8800;&#8869; &#8704;i &#8712;[1..n ]   tion Methods. <br />              &#9130; &#8869;                                     otherwise <br />              &#9129; <br />  <br /> Then, ER,I(a) &#8800; &#8869; explanation R I.                                              Experiments <br />                                                                                                           We applied approach biological model <br /> The construction non-deterministic (in choice                                               based E. coli K-12 metabolic genotype proposed <br /> rule choice cases),                                               [29] [30]. This group genes represents subset <br /> there different ways cause metabolite.                                                        genome E. coli K-12 includes genes <br />                                                                                                           encoding enzymes involved energetic biosynthetic <br /> False positives false negatives: problem approximation                                           metabolism. Using formalism, represented <br /> We briefly discuss precision approach.                                                 metabolic network composed enzymes <br /> As explained above, approach intended offer                                                  encoded selected gene set metabolites <br /> over-approximation dynamic behaviour <span id='am-55' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-56' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59'  ></span>meta</span>-                                                      involved catalyzed reactions. We obtained a <br /> bolic pathways. The existence explanation gives                                                  list 600 causal rules. We performed in <br /> indications possibility production                                                silico &#8220;what-if&#65533;? experiments compared obtained <br /> metabolite, according relaxing assumptions                                                    results correspondent vitro counterpart, <br /> adopted. This means false positives, i.e.                                             excerpts reported following. The exper- <br /> is possible predict silico production metabo-                                                iments carried tool (see [31]), briefly <br /> lites produced vivo. This presence                                                   presented section Methods. <br /> expected consequence abstraction over- <br /> approximation. However, experimental results                                                          Mutually essential genes <br /> reported final section accuracy rates,                                            In silico experimental session performed gene <br /> i.e. measure approximation, comparable                                                    knock-out mimicking homologous vitro experiment <br /> obtained well-known approaches litera-                                                    presented [3]. There, authors silenced target <br /> ture.                                                                                                     genes E. coli K-12 (sucAB sucCD) encode for <br />                                                                                                           enzymes involved Krebs cycle (&#945;-ketoglutarate <br /> On contrary, adequacy adopted bio-                                                   dehydrogenase succinyl-CoA synthase, respectively). <br /> logical model, expect false negatives, i.e. metab-                                              They genes &#8220;mutually essential&#65533;? pro- <br /> olites produced vivo                                                           duction succinyl-CoA, i.e. sucAB sucCD be <br /> explanation silico, stated following claim.                                                  knocked-out individually, simultaneously in <br />                                                                                                           order achieve Succinyl-CoA production. Succinyl-CoA <br /> Claim                                                                                                     critically important metabolite involved several <br /> If explanation production certain                                                biochemical pathways leading, e.g., energy production <br /> metabolite a, producible vivo.                                                        peptidoglycan biosynthesis (via Diaminopimelate). <br />  <br /> The intuition supporting claim based follow-                                                In order simulate gene knock-out, have <br /> ing reasoning: ad absurdum metabolite actually                                                removed rules corresponding reactions cata- <br /> producible vivo, certain set biochemi-                                              lysed &#945;-ketoglutarate dehydrogenase succinyl-CoA <br /> cal reactions applied starting initial set                                            synthase. Then set starting experimental con- <br /> of metabolites. Starting reactions                                                      ditions, including initial state metabolites <br /> applying abstractions illustrated                                                       cell assumed uptake external envi- <br /> should lead explanation looking for.                                                     ronment. Checking presence succinyl-CoA the <br />                                                                                                           end computation, metabolite <br />                                                                                                           produced (i.e. correspondent fact not <br />                                                                                                           deduced) target genes (i.e. rules <br />  <br />  <br />                                                                                                                                                               Page 6 10 <br />                                                                                                                                         (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                       http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  <br /> corresponding action encoded enzymes)                [28]. This difference taken account calcu- <br /> were simultaneously turned off. This reflects actually        lating accuracy A defined below. <br /> happens vitro. <br />                                                                    Definition 4 <br /> In silico gene knock-out                                           Let TP, TN, FP, FN number true positives, true <br /> We performed silico gene knock-outs              negatives false positives, false negatives, respectively. <br /> compared results information contained             We define accuracy A as <br /> the &#8220;Geno Base&#65533;? (http://ecoli.aist-nara.ac.jp/), database <br /> entirely dedicated E. coli K-12. In database genes                   A = (TP + TN ) / (TP + TN + FP + FN ) <br /> are classified according various criteria, which <br /> their essentiality, i.e. capability causing cell death,   . <br /> when turned off. In silico knock-out experiments we <br /> tried test gene essentiality verifying       Our experiments following results: <br /> in silico knock-out mutants exhibit features typically per- <br /> taining living cells. We assumed characteris-                                         = <br />                                                                                           &#9127;    TP        102 <br /> tics reasonably include production ATP                                      &#9130; <br /> (essential cellular energetic metabolism), produc-                                &#9130;    TN   =    13 <br />                                                                                           &#9128; <br /> tion reduced coenzymes NADPH NADH                                          &#9130;    FP   =    17 <br /> production dispensable structural components,                                      &#9130;&#9129;   FN   =    0 <br /> such cell wall (murein Biosynthesis). We per- <br /> formed silico knock-out experiments sample           As reported [28], accuracy Synthetic Accessi- <br /> of 132 genes set, time removing rules cor-         bility approach presented ranges 60% to <br /> responding enzyme encoded silenced gene              74%, approach metabolic networks, i.e. <br /> and checking presence observed elements          Flux Balance Analysis [30], accuracy ranges <br /> the end computation. We interpret results         62% 86%. We 17 false positives, <br /> prediction viability viability knock-         resulting accuracy 87% specific <br /> out mutant analysis. In experimental setting             experiment. These differences probably strongly influ- <br /> (see Table 1), slightly extend notion false posi-        enced way data sets interpreted which <br /> tives negatives introduced the sec-       cases included data sets. However, obtained <br /> tion say True Positive occurs knock-          results make confident approach reasonably <br /> out mutant results viable silico vivo        accurate. <br /> counterpart, True Negative occurs silico <br /> knock-out mutant viable silico vivo,       To intuitive idea experiments, just report <br /> False Positive occurs viable silico knock-out            Table 2, examples true positive, true negative <br /> mutant vivo counterpart not, False        false positive cases (reminding false <br /> Negative occurs silico knock-out mutants         negatives). The symbols + &#8722; stand presence or <br /> viable, vivo counterpart results viable. Note         absence, respectively, observed elements in <br /> that assignment arbitrarily chosen. To eval-         silico results. <br /> uate performance method compare with <br /> similar approaches use performance measure taken              The presence false positives (in silico, mutant pre- <br /> by [28], measures number true predictions &#8211;            dicted viable, actually, vivo, not) expected in <br /> both positive (viability) negative (non viability) &#8211;        framework, consequence abstraction of <br /> the overall ones. Note definitions false posi-         over-approximation used model. This corre- <br /> tives false negatives differ proposed            sponds fact that influence on <br />                                                                    viability potentially producible silico, not <br /> Table 1: True false positives negatives according our <br />                                                                    actually produced real life. Finally, note that, the <br /> definition, depending viability knock-out mutants     experiments carried out, false negatives. <br /> silico vivo                                                 This makes confident correctness adopted <br />                                                                    biological model. <br />  CASE                      IN SILICO           IN VIVO <br />  <br />  True Positive             viable              viable <br />                                                                    Conclusions <br />  True Negative             viable          viable          In metabolic networks, metabolites produced a <br />  False Positive            viable              viable          set initial metabolites, set chemical reac- <br />  False Negative            viable          viable              tions. These reactions produce intermediate metabolites <br />                                                                    products reactants. We intro- <br />  <br />  <br />                                                                                                                           Page 7 10 <br />                                                                                                     (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                           http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  <br /> Table 2: Gene knock-out experimental results                       &#8220;emphasis hypothesis-driven research biology&#65533;? <br />                                                                    advocated [1]. <br />  Gene    ATP    NADPH      NADH      cellwall   outcome <br />  <br />  acpS    &#8722;      &#8722;          &#8722;         &#8722;          true negative      Methods <br />  lpxA    +      +          +         &#8722;          true negative      To illustrate methodology, computa- <br />  glk     +      +          +         +          true positive      tional framework represents counterpart the <br />  aceA    +      +          +         +          true positive      formalisation metabolic causality, introduced sec- <br />  prsA    +      +          +         +          false positive     tion Results Discussion. Finally, relate compu- <br />  pfk     +      +          +         +          false positive     tational construction adopted formalisation. <br />  <br />                                                                    The set reactants caused &#8211; according our <br /> duced simple skeletal notation net-        formalisation &#8211; explanation starting initial <br /> works terms molecular entities reaction rules            set reactants set rules automatically <br /> that specify interactions implicitly code pos-      determined. Amongst different equivalent <br /> sible causal dependencies reactions. This nota-            approaches choose, eventually follow a <br /> tion permits us, hand, to chemical              logic-deductive interpretation, line the <br /> reactions abstract intuitive representation           explanatory approach adopted. Technically, follows <br /> quantitative aspects abstracted away;             consists fragment Horn-based Logic Programming <br /> hand, representation straightforwardly trans-          (having finite set ground predicates) equipped <br /> lated input tool developed, paving          bottom-up semantics (see, e.g. [32]). However, for <br /> way silico experiments tool develop-            sake accessibility, theory recast terms of <br /> ment. To aim, exploited analogy           rules (i.e., clauses) reactants (i.e., predicates). In the <br /> reaction rules logical implications, allows         following, reactants directly represented, causal <br /> automatically deduce chains causally related reactions          rules model, &#65533;?? c &#8594; b say, straightforwardly <br /> by means logical-based tool. Even            translated Horn-rules, like a, c &#8594; b, according the <br /> consider dynamic evolution metabolic networks,              following definition. <br /> our model sufficient information which <br /> metabolites possibly produced and,                  Definition 2 <br /> therefore, hints possible flows reactions.       Let A finite set reactants a1,&#8230;, an, &#8712; A, then <br />  <br /> Moreover, methodology makes possible reason                                        a1 ,&#8230; , &#8594; a <br /> about model itself, allowing vary ini- <br /> tial conditions rules. It easy program          n &#8805; 0 rule. <br /> modifications evaluating impact changes the <br /> hypotheses quite immediate, tool quickly            Notice rules premises. These <br /> reacts queries (typically answer reasona-        rules used representing elements present the <br /> ble large network returns istantanelously). The             initial state I. <br /> what-if approach satisfies need simulate inves- <br /> tigate behaviour fully known metabolic net-             The set consequences given set rules (respec- <br /> works different working hypotheses. In particular         tively, semantics logic program) defined <br /> allows perform perturbative experiments,               according step-wise bottom-up process. The applica- <br /> results trivial predict. In fact, studied net-   tion rule set reactants causes new reactant <br /> work complex enough, results unfeasible priori          premises rule verified set. <br /> estimate effects produced local perturbation           Starting set initially available reactants, the <br /> the overall network. Finally, applied method-          set reactants caused obtained <br /> ology metabolic network E. coli K-12 meta-           repeatedly adding reactants immedi- <br /> bolic genotype. The silico experiments presented reflect        ately caused application rules. <br /> the vitro ones. The results obtained our <br /> method underperform analogous ones. Noticeably              Definition 3 <br /> it ground formalism provides efficient            Let R set rules A set reactants, the <br /> straightforward implementations.                                   immediate consequence operator TR(A) : 2A &#8594; 2A defined <br />                                                                    as <br /> Our ultimate goal investigations sup- <br /> porting heuristic process searching causal explana- <br />                                                                       TR ( A )    {a | a1 ,&#8230;, &#8594; &#8712; R a1 ,&#8230;, &#8712; A} &#8746; A <br /> tions metabolic phenomena, mind the <br />  <br />  <br />                                                                                                                          Page 8 10 <br />                                                                                                    (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                       http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  <br /> .                                                                 recall SLD-trees top-down semantics of <br />                                                                   Logic Programming). Considering example <br /> Moreover, TRn ( A )    TR (TRn&#8722;1 ( A)) TRo ( A ) A .          above, reactant c caused TR3 ({b}) applica- <br />                                                                   tion rule (12), turn depends rule (13), the <br /> The convergence outlined process guaranteed          production does belong initial set I = <br /> the following result, based observation           {&#8594; b}. Consequently, explanation <br /> that, trivially, &#9001;2 A , &#8838;&#9127; (finite) complete partial         c[a[b],b] c. However, important considered <br /> order(c.p.o.) T(_) continuous (keeping         convergence reactants explanations. <br /> original A guarantees monotonicity). Moreover,          It easy observe fourth step have <br /> set reactants finite &#8211; i.e. number reactants    explanation a[b[a[b]]] cycle and <br /> occurring rules R initial state I &#8211; bottom-     b. This amounts say &#8211; presence cycles &#8211; <br /> up construction guaranteed converge finite        explanations growth indefinitely, i.e. convergence <br /> number steps (then fix point indicated TRn (A)       occurs infinity. Hence, restricted to <br />                                                                   compute reactants produced, a <br /> instead TR&#8734; (A)).                                           finite process. However, following correspondence <br />                                                                   computational results potentially infi- <br /> Proposition 2 <br />                                                                   nite model explanations drawn. <br /> Let R set rules, A set reactants, m the <br /> number reactants model consider-          Theorem 1 <br /> ation. Then, n &lt;m exists that <br />                                                                   Let R set rules, a reactant, I set reactants and <br />                                                                   n     minimum natural number that <br />                       TR (TRn ( A )) = TRn ( A ). <br />                                                                   T (TRn (I )) = TRn (I ) . Then following holds <br /> Example 2 <br /> Let consider network reactants I = { &#8594; b} ini-                            &#8712; TRn ( I ) &#8660; &#8707;EI , R (a ) &#8800;&#8869; <br /> tially available consisting following simple <br /> rules R:                                                          Proof <br />                                                                   (Outline). <br />                               a, b &#8594; c                     (12) <br />                                                                   (&#8658;) By induction number steps needed firstly <br />                                                                   cause bottom-up process (the rule moti- <br />                                b&#8594;a                         (13) <br />                                                                   vated inclusion used top-most <br />                                                                   rule definition explanation a. All reac- <br />                                a&#8594;b                         (14)   tants premises rule caused in <br />                                                                   steps, then, inductive hypothesis, explanations <br />                                c&#8594;d                         (15)   exist, used construct the <br />                                                                   explanation a). <br /> Applying TR ({b}) have <br />                                                                   (&#65533;?) By induction structural complexity EI,R(a). (By <br />                   TR ({b}) =       {a, b}                         inductive hypothesis, reactants premises of <br />                   TR2 ({b}) =  {c , a, b}                         top-most rule explanation belong TRn (I ) , <br />                   TR ({b}) = {d , c , b, a} <br />                     3 <br />  <br />                                                                   TRn (I ) TRn (I ) fix point, since <br />                   TR4 ({b}) = TR3 ({b}) <br />                                                                   rule clearly applicable cause a). <br /> Coherence computational metabolic causality <br /> As standard, like relate computational           To address problem causality metabolic net- <br />                                                                   works, seen above, developed software tool based <br /> construction provided original model, i.e. our <br />                                                                   implementation standard bottom-up seman- <br /> adopted formalisation biochemical reactions, intro- <br />                                                                   tics, running SICTUS Prolog Interpreter (see <br /> duced section Results Discussion. Let start          [31]). <br /> observe keeping track rule used applying <br /> TR(_) allow reconstruct explanations (which <br />  <br />  <br />                                                                                                                          Page 9 10 <br />                                                                                                    (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                                         http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  <br />  <br /> Competing interests                                                              19.   Degano P, Montanari U: Concurrent Histories: A Basis for <br />                                                                                        Observing Distributed Systems. Journal Computer System <br /> The authors declare competing interests.                             Sciences 1987, 34(2/3):422-461. <br />                                                                                  20.   Bodei C, Degano P, Nielson F, Riis-Nielson H: Static analysis for <br /> Authors&apos; contributions                                                                 &#960;-calculus application security. Information <br />                                                                                        Computation 2001, 165:68-92. <br /> All authors equally contributed present work                          21.   Nielson F, Riis-Nielson H, Schuch-Da-<span id='am-1' about='obo:NCBITaxon_3764' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3745 obo:NCBITaxon_3398 obo:NCBITaxon_3744 obo:NCBITaxon_171638 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_1176516 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="rdfs:label" content="Rosa" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Rosa L." datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Rosa</span> D, Priami C: Static anal- <br /> and wrote paper. More precisely, CB AB taken care                              ysis systems biology. In Proceedings workshop Systeomatics <br /> of computational framework DC carried                                  &#8211; dynamic biological systems informatics Computer Science Press, Trinity <br />                                                                                        College Dublin; 2004:1-6. <br /> in silico experiments, jointly develop for-                       22.   Chabrier N, Fages F, Soliman S: The Biochemical abstract <br /> malisation metabolic causality analysis                              machine BIOCHAM. In Proceedings Computational Methods in <br />                                                                                        Systems Biology (CMSB&apos;04), Lecture Notes Computer Science 3082 <br /> the experimental results.                                                              <span id='am-60' about='obo:NCBITaxon_49669' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_50362 obo:NCBITaxon_4447 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_4667 obo:NCBITaxon_3193'><span id='am-61' property="rdfs:label" content="Paris" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Paris L." datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasAlternativeId" content="NCBITaxon:49665" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-66'  ></span>Paris</span>, France: Springer; 2005:172-191. <br />                                                                                  23.   Eker S, Knapp M, Lincoln P, Laderoute K, Talcott C: Pathway Logic: <br /> Acknowledgements                                                                       Executable Models Biological Network. In Proceedings the <br />                                                                                        Fourth International Workshop Rewriting Logic Its Applications <br /> This article published BMC Bioinformatics Volume 9 Sup-            (WRLA &apos;02), Electronic Notes Theoretical Computer Science 71 Elsevier; <br /> plement 4, 2008: A Semantic Web Bioinformatics: Goals, Tools, Sys-                 2002:144-161. <br /> tems, Applications. The contents supplement available online     24.   Talcott C, Eker S, Knapp M, Lincoln P, Laderoute K: Pathway Logic <br /> at http://www.biomedcentral.com/1471-2105/9?issue=S4.                                  Modeling Protein Functional Domains Signal Transduc- <br />                                                                                        tion. Proceedings Pacific Symposium Biocomputing 2004, <br />                                                                                        9:568-580. <br /> References                                                                       25.   Tamaddoni-Nezhad A, Chaleil R, Kakas A, Muggleton S: Application <br /> 1.    Kitano H: Systems Biology: brief overview. Science 2002,                       abductive ILP learning metabolic network inhibition <br />       295(5560):1662-1664.                                                             temporal data. Machine Learning 2006, 64(1-3):209-230. <br /> 2.    Tran N, Baral C, Nagaraj VJ, Joshi L: Knowledge-based frame-               26.   Papatheodorou I, Kakas AC, Sergot M: Inference Gene Rela- <br />       work hypothesis formation biochemical networks. Bio-                      tions Microarray Data Abduction. In Proceedings of <br />       informatics 2005, 21(Suppl 2):ii213-ii219.                                       Logic Programming Nonmonotonic Reasoning (LPNMR&apos;05), Lecture <br /> 3.    Yu B, Sung B, Lee J, Son S, Kim M, Kim S: sucAB sucCD                    Notes Computer Science 3662 Diamante, Italy; 2005:389-393. <br />       mutually essential genes <span id='am-48' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia</span> coli. FEMS Microbiology            27.   Grell S, Schaub T, Selbig J: Modelling Biological Networks by <br />       Letters 2006, 254(2):245-250.                                                    Action Languages Via Answer Set Programming. In Proceed- <br /> 4.    Reisig W: Petri Nets: introduction Springer-Verlag; 1985.                     ings International Conference Logic Programming (ICLP&apos;06) Lecture <br /> 5.    Goss P, Peccaud J: Quantitative modeling stochastic systems                   Notes Computer Science 4079 Seattle, WA, USA: Springer; <br />       molecular biology using stochastic Petri Nets. Proceed-                    2006:285-299. <br />       ings National Academy Sciences USA 1998, 12:6750-6754.           28.   Wunderlich Z, Mirny L: Using topology metabolic net- <br /> 6.    Peleg M, Yeh I, Altman R: Modeling biological processes using                    works predict viability mutant strains. Biophysical Journal <br />       Workflow Petri Nets models.                     Bioinformatics 2002,         2006, 91:2304-2311. <br />       18:825-837.                                                                29.   Kayser A, Weber J, Henkt V, Rinas U: Metabolic flux analysis of <br /> 7.    Milner R: Communicating mobile systems: &#960;-calculus Cambridge             Escherichia coli glucose-limited continuous culture. Trans- <br />       University Press; 1999.                                                          actions Computational System Biology VII 2005, 151:693-706. <br /> 8.    Cardelli L, Gordon A: Mobile Ambients. Theoretical Computer Sci-           30.   Edwards J, Palsson B: Metabolic flux balance analysis sil- <br />       ence 2000, 240:177-213.                                                          ico analisis <span id='am-73' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-74' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia Coli K-12</span> gene deletion. BMC Bioin- <br /> 9.    Priami C, Regev A, Shapiro E, Silvermann W: Application sto-                formatics 2000, 1(1):1-10. <br />       chastic name-passing calculus representation simula-                31.   A Toolkit Supporting Formal Reasoning Causality in <br />       tion molecular processes. Theoretical Computer Science 2004,                  Metabolic Networks.             . [http://www.di.unipi.it/~braccia/Tool <br />       325:141-167.                                                                     Code]. <br /> 10.   Regev A, Silvermann W, Shapiro E: Representation Simula-               32.   Lloyd J: Foundations Logic Programming Second edition edition. <br />       tion Biochemical Processes Using pi-Calculus Process                      Springer; 1987. <br />       Algebra. In Proceedings Pacific Symposium Biocomputing World <br />       Scient <br /> </body></html>