<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACrERIOLOGY, Mar. 1994, p. 1451-1459                                                                              Vol. 176, No. 5 <br /> 0021-9193/94/$04.00+0 <br /> Copyright X 1994, American Society Microbiology <br />  <br />  <br />           Characterization Cell Division Gene <span id='am-1' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-2' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-29'  ></span><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span>Bacillus subtilis</span> <br />                 That Is Required Vegetative Sporulation <br />                                 Septum Formation <br />                                         PETRA ANNE LEVIN AND RICHARD LOSICK* <br />                                          Department Cellular Developmental Biology, <br />                                         Harvard University, Cambridge, Massachusetts 02138 <br />                                          Received 4 November 1993/Accepted 22 December 1993 <br />  <br />               We report cloning characterization cell division gene, designated divIC, the <br />            gram-positive, spore-forming bacterium Bacillus subtilis. <span id='am-66' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-69'  ></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> gene previously identified basis a <br />            temperature-sensitive mutation, div-355, blocks septum formation restrictive temperatures. We show <br />            divIC gene 125-<span id='am-71' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-72'  ></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> open reading frame capable encoding protein 14.7 kDa and <br />            div-355 5-bp duplication near 3&apos; end open reading frame. We divIC an <br />            essential gene use vitro-constructed null mutation. In confirmation extension earlier results, <br />            divIC necessary vegetative sporulation septum formation, demonstrate that <br />            required activation genes expressed control sporulation transcription factors <br />             rF or. The divIC gene located 1.3 kb upstream coding sequence sporulation gene spoIIE. <br />            Between divIC spoIIE 128-codon open reading frame predicted product contains region of <br />            similarity RNA-binding domains polynucleotide phosphorylase ribosomal protein Si from <br />            <span id='am-79' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-80' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-83'  ></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span>Escherichia coli</span> putative tRNA genes methionyl-tRNA glutamyl-tRNA, gene <span id='am-76' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-77' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> being <br />            divIC orf128 tRNAMet tRNAGlu spoIIE. <br />  <br />  <br />    The spore-forming bacterium Bacillus subtilis attractive           divIB, product thought regulate frequency of <br /> organism studying problem cell division               division initiation (19). <br /> exhibits alternative pathways septum formation. During                  In addition septum initiation genes ftsA, ftsZ, and <br /> vegetative growth, bacterium divides medially, process                divIB, B. subtilis cell division genes been <br /> that strongly resembles binary fission <span id='am-95' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-98'  ></span><span id='am-99' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia</span> coli              implicated septum placement known. These minC <br /> many bacteria. On hand, spore <span id='am-107' about='obo:NCBITaxon_forma' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="forma" datatype="xsd:string"></span>forma</span>-                 minD, homologs E. coli minicell genes the <br /> tion, bacterium undergoes process polar septation             (22, 25, 45). In E. coli, products minB <br /> partitions developing cell asymmetrically un-                  locus, consists genes minC, minD, minE, <br /> equally sized cellular compartments, called mother cell                 function regulate septum position cell division. The <br /> (the large chamber) forespore. The compartments                 MinC MinD proteins act division from <br /> each contain chromosome exhibit dissimilar patterns                occurring potential septation sites, midcell and <br /> gene expression dissimilar developmental fates;                     poles. MinE relieves inhibition, the <br /> mother cell terminally differentiating cell type nur-             central site, allowing binary fission occur. Mutations <br /> tures developing spore undergoes lysis,                minC minD result production small anucleate <br /> the forespore eventually mature spore.                          minicells polar cell division. Mutations B. <br />    Several B. subtilis genes involved cell division             subtilis homologs minC minD similarly result a <br /> have identified. The important genes              minicell phenotype significantly impair sporulation <br /> B. subtilis homolog E. coli cell division geneftsZ (5). For          septum formation. No homolog E. coli minE far been <br /> E. coli, immunoelectron microscopy experiments shown                   identified B. subtilis, genes dedicated exclusively to <br /> that FtsZ protein forms ring structure probably                  sporulation septum placement formation known. <br /> serves scaffold septum formation leading edge               An additional B. subtilis gene involved septation defined <br /> the invaginating membrane (7). The FtsZ protein exhibits                  div-355, temperature-sensitive mutation blocks cell <br /> GTPase activity responsible driving poly-               division sporulation entry state of <br /> merization (13, 26, 34, 46). Genetic studies B. subtilis           genetic competence restrictive temperatures (31, 32, 40). <br /> ftsZ gene shown product required                  Previous work div-355 suggested product the <br /> vegetative sporulation septum formation (5). A null mu-                 wild-type allele gene, like FtsZ, required the <br /> tation promoter-proximal gene B. subtilis ftsZ                formation medial polar septa. To investi- <br /> operon, ftsA, lethal, shown severely               gate possibility, isolated wild-type mutant DNAs <br /> impair ability cell grow sporulate                  encompassing site div-355. We div-355 is <br /> normal physiological conditions (6). A B. subtilis division           result duplication 5-bp sequence 125-codon <br /> gene, homolog identified E. coli,               open reading frame (ORF) divIC. We show <br />                                                                             product ORF necessary formation of <br />                                                                             sporulation septum gene expression directed two <br />                                                                             sporulation sigma factors, u oF, activities are <br />   * <br />       Corresponding author.                                                 thought depend polar septum formation (5, 27). <br />                                                                      1451 <br />  1452             LEVIN AND LOSICK                                                                                                       J. BAcrERIOL. <br />  <br />                  TABLE 1. B. subtilis strains used studya                        Competent cells prepared transformed the <br />                                                                                       method described Dubnau Davidoff-Abelson (15). <br />    Strain                 Genotype description                Source            Selection antibiotic resistance LB agar con- <br />                                                                                       taining 5 ,ug chloramphenicol ml Cmr, 100 ,ug of <br /> PY78                glnA100                                 Laboratory stock          spectinomycin ml Specr, 25 jig lincomycin 1 <br /> PY79                Prototrophic                            50                        ,ug erythromycin ml macrolide-lincosamide-strepto- <br /> NIG1154b            div-355 met                         40 <br /> KS265               spoVG::Tn917QIHU265                     42                        gramin B (MLS) resistance. Selection prototrophy B. <br /> PY180               spoIIE::Tn917fQHU7                      42                        subtilis performed TSS medium (12) amino <br /> KS181               spoIIE::Tn917fQHU181                    42                        acid supplements. SP3-specialized transductions drug <br /> RL856               ctcQctc-lacZ                            Laboratory stock          selection carried described Cutting Vander <br /> RL905               SPI[spoIIUB-lacZ]                       Laboratory stock          Horn (12). <br /> PL85C               spoIIDQ7spoIID-lacZ                     This study                   All manipulations strains carrying div-355 mutation <br /> PL86d               spoIIIGAI)spoIIIG-lacZ                  This study                carried permissive temperature 30&#176;C, except <br /> PL91e               div-355                                 This study <br /> PL93f               div-355 SP3[spoIIAB-lacZ]               This study                noted otherwise. <br /> PL103C              div-355 spoIIDQspoIID-lacZ              This study                   Amylase activity assayed growing putative amyE <br /> PL104d              div-355 spoIIIGAIQspoIIIG-lacZ          This study                mutants overnight 1% starch plates staining agar <br /> PL108               amyE::divIC+                            This study                Gram&apos;s iodine solution (12). <br /> PL169               amyE::divIC spoIIE::Tn917QlHU7          This study                   Chromosomal DNA prepared described Cutting <br /> PL171               amyE::divIC divICA: spr                 This study                Vander Horn (12). <br />   All strains congenic prototrophic wild-type strain PY79,         Plasmid manipulations carried described by <br /> as noted otherwise.                                                                   Sambrook et al. (41). In addition alkaline lysis method <br />   b Congenic HA101 (37).                                                         described Sambrook et al., Xprep kit (Xgen Corp.) was <br />   c The spoIIDQspoIID-lacZ transcriptional fusion (a gift Patrick Stragier) <br /> was transferred PY79 PL91 selection Cmr. <br />                                                                                       used prepare miniprep plasmid DNA. The Geneclean kit <br />   d The spoIIIGAfIlspoIIIG-lacZ transcriptional fusion (27) transferred        (Bio 101, Inc.) used purify DNA fragments from <br /> PY79 PL91 selection Cmr.                                                   agarose. <br />   &apos;The div-355 mutation (32) introduced PY78 congression                <span id='am-102' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-103' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-104' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-106'  ></span>Electron</span> microscopy sporulating cells sampled 7 h after <br /> glnA+.                                                                                end exponential phase performed described <br />   f SPP[spoIL4B-IacZj transduced PL91 selection Cm&apos;. <br />                                                                                       previously (11). <br />                                                                                          Nucleotide sequence analysis. Nucleotide sequencing was <br />                                                                                       carried method described Sanger et al. (43) with <br />                 MATERIALS AND METHODS                                                 Sequenase kit (USB). When necessary, technique was <br />    Bacterial strains. The B. subtilis strains used study                  modified accommodate double-stranded templates. The <br /> listed Table 1. All plasmid constructions TG1                         strategy used outlined Fig. 1. Oligonucleotide <br /> (laboratory stock [41]). The strain carrying original div-355                     primers purchased New England Biolabs synthe- <br /> mutation (NIG1154) gift Yoshito Sadaie.                                    sized Applied Biosystems 380B DNA synthesizer. Se- <br />    General methods. LB medium (41) used growth                            quence assembly analysis performed Genetics <br /> of E. coli B. subtilis. Difco sporulation (DS) medium                         Computer Group (University Wisconsin) programs (14). <br /> agar used induce sporulation B. subtilis liquid                     Sequence comparisons GenBank <span id='am-110' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-111' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-114'  ></span>data</span> <br /> and plates, respectively (36).                                                     base (8). <br />  <br />  <br />  <br />                              -40- <br />             i <br />  <br />  <br />  <br />  <br />                                                                                                                     P <br />  <br />             a <br />             t)                                                                                                -6 <br />  <br />  <br />  <br />                   partial orf          1I       divIC          I        I          orf 128 <br />  <br />                                                                                                                                     I p280.1 <br />                                                                              -I pPL15 <br />  <br />                                             I                  I pPL27                                         _     100 base pairs <br />    FIG. 1. Physical map divIC region chromosome. The arrows indicate extents orientations nucleotide <br /> sequence determinations. Bars end arrows indicate sequencing began plasmid insert endpoint universal primer. <br /> All sequences obtained synthetic oligonucleotide primers. Each rectangle represents ORF, transcribed from <br /> left right. The lower portion figure identifies plasmid inserts used localize divIC ORF. <br />  VOL. 176, 1994                                                                                    B. SUBTILIS divIC GENE          1453 <br />  <br />   Phage M13 constructions. To facilitate sequencing           div-355 mutation. To isolate region DNA upstream of <br /> certain regions DNA, 1-kb EcoRI-PstI fragment              spoIIE able correct div-355 mutation, used <br /> pPL15 (see below) subcloned M13mpl8                     technique developed Banner (2). Agarose slices <br /> M13mpl9, 1.9-kb EcoRI-HindIII fragment p280.1             containing various restriction fragments plasmid p280.1 <br /> was subcloned M13mpl8. Single-stranded M13 DNA              placed lawn div-355 mutant competent cells <br /> prepared described Sambrook et al. (41).                       spread LB agar DS agar. Plates prepared were <br />    Plasmid constructions. Plasmid p280.1, containing se-         incubated overnight restrictive temperature 45&#176;C. <br /> quence extending HindIII site internal spoIIE 1.9        Only cells transformed divIC+ able to <br /> kb upstream EcoRI site (18), gift P.                grow. Cells overlaid 1.8-kb EcoRI-HindIII fragment <br /> Youngman. Plasmid pPL15 constructed subcloning            1-kb EcoRI-MspI fragment p280.1 able grow <br /> EcoRI-MspI fragment corresponding B. subtilis chromo-             45&#176;C. <br /> somal DNA p280.1 EcoRI-AccI-cut integration               To test ability 1-kb EcoRI-MspI fragment <br /> vector pSGMU2 (16). The EcoRI-HindIII fragment                  correct div-355 mutation, plasmid pPL15 (see above) <br /> pPL15 subcloned amyE vector pDG364 (12),                transformed strain NIG1154 selection Cmr. <br /> creating pPL20. The EcoRI-HindIII fragment cloned           Single-crossover events wild-type plasmid insert <br /> into pBluescript II SK - (Stratagene) make pPL24         containing full-length copy gene mutant chromo- <br /> pUC18 create pPL30. Oligonucleotides OL109 (5&apos;-CG&#163;                somal DNA result correction mutation the <br /> GATCCCGATTGACGCAAAGGGCTGTACAGAC-3&apos;)                              creation second, wild-type copy gene the <br /> OLl10 (5&apos;-AACIGCAGCCGTCTCCAGACATGAATAAG                              chromosome. Transformants screened ability to <br /> TCC-3&apos;), underlined regions refer BamHI          grow 45&#176;C. <br /> PstI sites, respectively, used amplify region              Complementation div-355 mutation. A partially dip- <br /> chromosomal DNA internal divIC. The resulting fragment            loid strain carrying second copy divIC amyE locus <br /> was subcloned pUS19, derivative pUC19          constructed described Cutting Vander Horn (12) <br /> 1.1-kb ClaI-NdeI spectinomycin cassette cloned Narl-        use plasmid pPL20 linearized BglII for <br /> NdeI sites (a gift B. Haldenwang), make plasmid              transformation PY79, selection Cmr. The result- <br /> pPL27. Plasmid pPL31 deleting 272-bp HinclI          ing transformants Amy-, linearized plasmid <br /> fragment pPL24, attaching BamHI linkers ends,            recombined locus double-crossover &quot;marker re- <br /> and recircularizing plasmid, creating truncated version     placement.&quot; One amy::divIC+ transformant, PL108, was <br /> divIC, two-thirds replaced BamHI site. The       used transform PL91 (div-355) Spo+ selection for <br /> marker replacement plasmid pPL33 cloning             Cmr. <br /> 1.1-kb EcoRV-BamHI spectinomycin cassette pJL74 (a                 Marker replacement divIC. To determine divIC <br /> gift J. Ledeaux spr gene pUS19                essential gene, transformed partially diploid strain, <br /> subcloned SmaI site pBluescript polylinker)     PL108, contains wild-type copy divIC the <br /> the BamHI site pPL31. The drug resistance cassette            amyE locus divIC locus, plasmid pPL33 had <br /> inserted orientation divIC.                           linearized ScaI, selection spectinomycin <br />    Heat kill assays. To check PL91 (div-355) sporulation         resistance. pPL33 carries divIC null mutation divICA::spr. <br /> efficiency, used modification standard assay     To ensure marker replacement occurred divIC <br /> production heat-resistant spores (36). We inoculated DS           locus amyE locus, chromosomal DNA from <br /> medium log-phase cells optical density 595 nm       individual Specr colonies used transform strain PL169, <br /> approximately 0.1 allowed cells grow 39&#176;C 7      carries amyE::divIC spoIIE::Tn917fQHU7. <br /> h end logarithmic growth. The cells           Transformants experiment screened loss <br /> serially diluted 1 x T base (12) plated       MLS resistance, indicated divICA::spr linked <br /> after 20 min incubation 80&#176;C determine proportion       spoIIE, divIC, amyE. Finally, chromosomal <br /> of heat-resistant spores percentage viable cell count.   DNA Cmr Specr MLS-sensitive strain (PL171) <br /> It necessary check heat resistance point,         used transform PY79, divIC+ haploid strain, and <br /> instead standard 48 h inoculation,          PL108. As control, transformed strains with <br /> div-355 strain lyses prolonged period restrictive         chromosomal DNA strain carrying spr insertion a <br /> temperatures.                                                        locus distinct divIC (39). Transformation growth in <br />    ,-Galactosidase assays. To test effect div-355         experiments conducted 30&#176;C. <br /> mutation expression lacZ fusions sporulation               Nucleotide sequence accession number. The sequence cov- <br /> genes, ,3-galactosidase levels sporulating cultures de-      ering region shown Fig. 2 deposited the <br /> termined previously described (28), cells         GenBank data base accession number L23497. <br /> were grown 39&#176;C. We grew cells 39&#176;C because <br /> wild-type bacteria exhibited impaired ,-galactosidase accumu-                                     RESULTS <br /> lation 42&#176;C. The div-355 mutation confers strong <br /> phenotype lower temperature.                                     Nucleotide sequence wild-type allele divIC. The <br />    Amplification single-stranded mutant DNA. To obtain            div-355 mutation originally identified temperature- <br /> mutant single-stranded DNA, asymmetric PCR carried           sensitive cell division initiation mutation (32). According the <br /> as described McCabe (29) following modifications.        nomenclature Mendelson (31), genes acting division <br /> Two nanograms chromosomal DNA (NIG1154)                   initiation -are called divI genes, earliest acting six <br /> cut BamHI used template. The ratio sense            categories cell division genes. Other categories include <br /> primer antisense primer 50:1. All product obtained        genes involved division site location (divIV) daughter <br /> from PCR run Centricon 100 microconcen-              cell separation (divV7). Previously characterized divI genes of <br /> trators (Amicon) remove excess primer subjected         B. subtilis include ftsZ (divL4) (3) divIB (4, 20). We have <br /> alkaline denaturation process (41) prior sequencing.              designated gene div-355 mutation lies divIC. <br />    Identification regions DNA capable correcting           The div-355 allele divIC previously mapped the <br />  1454           LEVIN AND LOSICK                                                                                                                                                                                                                                                                                                 J. BACTERIOL. <br />  <br />                                                                                                                                                                                                                                                                                                                                       orf <br />            E        I Y I F L A V L L G V A T Y <br />                        F           R                           O S L C K R I Y I K I L K F V I Y <br />        1   GAATTCAGGATTTACATCTTTTTAGCGGTTCTGCTGGGCGTTGCGACGTATCAGAGCCTTTGCAAACGAATCTATATAAAAATACTGAAATTCGTCATTT                                                                                                                                                                                                                100 <br />            EcoRI                              .              . <br />  <br />  <br />              L V V S V Y Q F F K K L I Q H V L F R P I V W T C G A I I W L A A <br />   101      ACCTTGTTGTTTCTGTTTATCAATTCTTCAAAAAACTCATTCAGCACGTGTTATTTCGTCCTATTGTGTGGACATGCGGAGCGATCATCTGGCTGGCGGC                                                                                                                                                                                                                200 <br />  <br />  <br />  <br />                F           L           F           K       K               T       Y               S       L               I           G       F       L       L       L       C       L       Y       K       I       V       M       V       L       C       F       P       I       R       F       I       A       K <br />   201                                                                                                                                                                                                                                                                                                                           300 <br />  <br />  <br />  <br />            Q C L K L L P V K M R L T F R R Y F E K G A G F L K K K K K L L I T <br />   301      CAATGTTTGAAACTTCTTCCTGTGAAAATGCGTCTAACTTTTAGACGTTATTTTGAAAAAGGTGCAGGATTTCTCAAAAAGAAGAAGAAACTATTGATAA                                                                                                                                                                                                                 400 <br />  <br />  <br />  <br />                    I           R           T           T           I           T           R           F           L           K           R       *                                                   M       N       F       S       R       E       R       T       I       T       E       I       Q       N       D <br />                                                                                                                                                                                                                                                                                                                                       divIC <br />   401      CCATAAGAACGACCATCACACGGTTTTTGAAGAGATGAAAGGAGAACCGTCTGGTTTGAATTTTTCCAGGGAACGAACGATAACTGAAATACAAAACGAC                                                                                                                                                                                                                 500 <br />  <br />  <br />  <br />            Y  K E Q V E R Q N Q L K K R R R K G L Y R R L T V F G A L V F L T A <br />   501      TATAAAGAACAAGTAGAACGGCAAAATCAGCTGAAGAAACGAAGACGCAAAGGGCTGTACAGACGGTTAACTGTATTCGGCGCCCTAGTATTCCTGACGG                                                                                                                                                                                                                 600 <br />                                                                                                                                                                                                                                    HincII <br />  <br />  <br />                    I           V           L           A           S           S           V           W           S                       T       S       S       L       S       A       K       E       E       K       K       E       Q       L       E       K       E       L       K       S       L       K       T <br />   601                                                                                                                                                                                                                                                                                                                           700 <br />  <br />  <br />  <br />             K Q T D L K E E I S K L K D E D Y V T E L A R R D L F M S G D G E <br />   701      AAAACAAACGGATTTAAAAGAAGAAATATCCAAATTGAAGGATGAGGATTACGTCACAGAGCTTGCCAGAAGGGACTTATTCATGTCTGGAGACGGAGAA                                                                                                                                                                                                                 800 <br />  <br />  <br />  <br />            I           I           F           N           V           E           K           K           S           K           * <br />   801      ATTATCTTCAATGTGGAGAAGAAGAGCAAGTAGCCTTGTTGACACTTAAATTTTTATTTAGGTATAATTAAGCAAACGATCTTTTTATAAGCCT^A&amp;GAG                                                                                                                                                                                                                900 <br />                                                                                                                                                        HincII <br />  <br />  <br />                                                                M           S           I           E           V           G           S       K       L       Q       G       K       I       T       G       I       T       N       F       G       A       F       V       E       L       P       G       G       S <br />                                                                                                                                                                                                                                                                                                                                       orfl28 <br />   901      fAfCAfTTTTTTTATGTCGATTGAAGTTGGCAGCAAGTTGCAAGGGAAAATTACAGGTATTACAAATTTTGGAGCATTTGTTGAATTGCCTGGAGGCTCA                                                                                                                                                                                                                 1000 <br />  <br />  <br />  <br />            T G L V H I S E V A D N Y V K D I N D H L K V G D Q V E V K V I N V <br />  1001      ACCGGTCTCGTTCACATTAGTGAGGTAGCTGATAATTATGTCAAAGACATTAACGACCACTTAAAAGTCGGCGACCAAGTTGAAGTGAAAGTCATCAACG                                                                                                                                                                                                                 1100 <br />                MspI <br />  <br />  <br />  <br />              E K D G K I G L S I K K A K D R P Q A R P R N D F R P K E S F E Q <br />  1101      TTGAAAAAGACGGGAAAATTGGTTTATCTATTAAAAAAGCTAAAGACCGTCCGCAAGCCAGACCTAGAAATGATTTCCGTCCGAAAGAATCTTTTGAACA                                                                                                                                                                                                                 1200 <br />  <br />  <br />  <br />  <br />                K           M           N           K           F           L           K           D           S           E           D       R       L       S       S       L       K       R       N       T       E       S       K       R       G       G       R       G       A       R       R       G       * <br />  1201                                                                                                                                                                                                                                                                                                                           1300 <br />  <br />  <br />   FIG. 2. Nucleotide sequence noncoding strand divIC region chromosome. The figure shows (from left right) the <br /> nucleotide sequences 3&apos; end unknown partial ORF, divIC, orfl28, methionyl-tRNA, glutamyl-tRNA, 5&apos; end spoIIE. The <br /> predicted amino acid sequences ORFs shown nucleotide sequences. All sequencing bp 1 1557 carried out <br /> on strands (see Fig. 1). Base pairs 1546 1865 previously sequenced (18, 49) (GenBank accession number M29403). Underlined (in light <br /> type) nucleotides indicate complementarity B. subtilis 16S rRNA putative Shine-Dalgarno sequences. The 5 bp duplicated in <br /> the div-355 mutant shown boldface type. The predicted extent tRNA sequences indicated heavy underlining. The anticodons <br /> of tRNAs indicated respective amino acids sequences. <br />  <br />  <br /> region dna-526 cysA (40 110) (33). We used                                                                                                                                                  gene spoIIE divIC approximately 90% <br /> DNA-mediated transformation determine relative dis-                                                                                                                                                 cotransformed upstream transposon (HU7) about <br /> tances div-355 mutation different Tn917                                                                                                                                                80% cotransformed downstream transposon (HU181). <br /> transposon insertions (HU7 HU181) sporulation                                                                                                                                                   These results indicated divIC likely lie few <br />  VOL. 176, 1994                                                                                      B. SUBTILIS divIC GENE              1455 <br />  <br />  1301   CTGGAGCAAGAAGAGGATAACTTGCTGCTTTCTATAAAATAAATGAAGCATCCGTTCATCCCGACGGATGCTTTTTTATTATCCTCATTCTCTTTAAGAG                  1400 <br />  <br />                                                                                                                               tRNAmet <br />  1401   TCAGAAAATTACAACATTAAAGTTTCTGTTGACGAAAGAGACAAACCTTTGTATTATATAACTTGTGCTTAGAAGCACAAAACAATATGGCGGTGTAGCT                  1500 <br />  <br />  <br />  <br />  1501   CAGCTGGCTAGAGCGTACGGTTCATACCCGTGAGGTCGGGGGTTCGATCCCCTCCGCCGCTACCATATTTGTTTGGCCCGTTGGTCAAGCGGTTAAGACA                  1600 <br />  <br />  <br />  <br />  1601   CCGCCCTTTCACGGCGGTAACACGGGTTCGAATCCCGTACGGTCATCCTAACAAATCGGTTTCTCTTGCAGAAGCCGATTTTTTTCATTTTTAGACAACA                  1700 <br />  <br />  <br />  <br />  1701   TTCCGGAAATTCTTTTCATAAACGAATATCAAGGCAGAAACCGTCGAAGATTTCTTTGGTATTGTTACCTTCTTTTGACAAAATCCTATCTGTGCTTTCG                  1800 <br />  <br />  <br />  <br />                                                        M E    K A E    R R    spoIIE <br />  1801   CTATAATGACAGGCAACGAATATAACAaGTGGGAGAGGAATGGAAAAAGCAGAAAGAAGA            1865 <br />  <br />  <br />                                                          FIG. 2-Continued. <br />  <br />  <br /> kilobases upstream spoIIE. In addition, used two-         First, locate site div-355 mutation cloned <br /> and three-factor crosses establish gene order spoVG ctc     DNA region upstream spoIIE, placed slices of <br /> divIC spoIIE.                                                      agarose containing separated fragments DNA from <br />    We took advantage availability cloned DNA             plasmid p280.1 lawn div-355 mutant competent cells <br /> adjacent extending upstream spoIIE (a gift P.       (see Materials Methods). We tested restriction fragment <br /> Youngman) sequence region predicted        contained region DNA EcoRI site located <br /> mapping results contain divIC (see Materials Methods).      1.8 kb upstream spoIIE coding region MspI site <br /> Nucleotide sequence analysis DNA extending          located 1 kb downstream, fragment extended approxi- <br /> EcoRI site located 1.8 kb upstream spoIIE region           mately 900 bp MspI site HindIII site internal to <br /> downstream spoIIE ORF revealed existence         spoIIE, fragment contained entire 1.9-kb <br /> partial complete ORFs tRNA-encoding genes          EcoRI-HindlIl fragment (Fig. 1). The results showed that <br /> (Fig. 1). Starting Fig. 2, partial ORF      DNA capable correcting div-355 mutation located <br /> extends nucleotide 438 found. The predicted product      1-kb stretch DNA EcoRI MspI <br /> this ORF shows obvious similarity sequence cur-          sites. <br /> rently listed GenBank data base. The complete            In addition overlay experiment, second experiment <br /> ORF, divIC, extends nucleotides      showed pPL15, plasmid containing 1-kb EcoRI-MspI <br /> 456 833 Fig. 2 capable encoding 125-amino-      DNA fragment, able correct div-355 mutation when <br /> acid protein 14.7 kDa. The predicted product ORF        integrated chromosome, presumably single-recip- <br /> shows significant sequence similarity coiled-coil proteins      rocal recombination. Finally, EcoRI-MspI DNA fragment <br /> such myosin (14) potential leucine zipper motif       able complement div-355 mutation when <br /> near COOH terminus, judged regular placement         inserted chromosome amyE locus, implying that <br /> of leucines hydrophobic residues region (1, 30)    entire coding region divIC gene regulatory <br /> (Fig. 3). In addition, product contains stretch 28        sequence needed expression contained this <br /> hydrophobic residues serve membrane-span-           region DNA. <br /> ning domain. Although amino acid sequence DivIC                The EcoRI-MspI fragment contains complete <br /> showed detectable similarity known cell division         ORF, 125-codon ORF described (Fig. 1). Nucle- <br /> protein sequences, predicted structure suggestive   otide sequencing independently amplified PCR prod- <br /> of E. coli cell division protein FtsL (17) (see Discussion).   ucts div-355 mutant (see Materials Methods) <br /> Downstream divIC ORF 128 codons (orfl28)          revealed existence 5-bp duplication (TTCAT) of <br /> extends nucleotides 914 1300 Fig. 2 capable <br /> of encoding 14.2-kDa protein. The predicted product this <br /> ORF (Orf128) shares sequence similarity RNA- <br /> binding domains polynucleotide phosphorylase riboso-           MNFSRERTIT EIQNDYKEQV ERQNQLKKRR RKGLYRRLTV FGALVFLTAI <br /> mal protein subunit Si E. coli (Fig. 4). This 128-codon <br /> ORF followed putative coding sequences methionyl-                              +   ++      _+-_ <br />  <br />  <br />  <br />  <br /> tRNA glutamyl-tRNA.                                               VLASSVWSQT SSLSAKEEKK EQLEKELKSL KTKQTDLKEE ISKLDEDYV <br />    Identification 125-codon ORF divIC. We localized <br /> the divIC gene 1-kb EcoRI-MspI fragment ways:         TELARRDLFM SGDGEIIFNV EKKSK* <br /> placement agarose gel slice containing fragment        FIG. 3. Inferred features putative DivIC protein. + - <br /> DNA div-355 competent cells, integration fragment        designate basic acidic residues, respectively. Overlining marks a <br /> into chromosome correct div-355 mutation,           putative hydrophobic membrane-spanning domain. The boxed area <br /> complementation div-355 fragment trans       indicates potential leucine zipper motif (1, 30). The asterisk indicates <br /> amyE locus.                                                         stop codon. <br />  1456     LEVIN AND LOSICKJ.AGEO. <br />                                J. BACTERIOL. <br />  <br />                     PNP      GKVTRIVDFGAFVAIGGGKEGLVHISQIADKRVEKVTDYLQMGQEVPVKVLEVDRQGR-IRLSIKEATEQSQ <br />                              11:1 I::IIIII::II:III:III::II:::                          I::I:::I:       1111:1::: <br />                     0rf128   GKITGITNFGAFVELPGGSTGLVHISEVADNYVKDINDHLKVGDQVEVKVINVEKDGK-IGLSIKKAKDRPQ <br />  <br />                     RPS1     GKVTAVDAKGATVELADGVEGYLRASEASRDRVEDATLVLSVGDEVEAKFTGVDRKNRAI SLSV-RAKDEAD <br />   FIG. 4. Comparison amino acid sequence 0rf128 RNA-binding domains E. coli polynucleotide phosphorylase (PNP) and <br /> ribosomal protein subunit Si (RPS1) (14, 38, 44). Vertical lines denote identical amino acids, colons indicate amino acid similarity. <br />  <br />  <br /> nucleotides 780 784 near 3&apos; end 125-codon ORF.          transcription spoIIIG-lacZ spoIID-lacZ significantly <br /> This duplication expected create frameshift         curtailed presence div-355 mutation (Fig. 5B and <br /> thereby extend length predicted product 7 amino         C), finding consistent <span id='am-61' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-62' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-64'  ></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>idea</span> mutant cells were <br /> acids, suggesting COOH terminus important          blocked prior sporulation septum formation. <br /> function divIC gene product.                                     Finally, direct evidence defect septum formation, <br />    As additional evidence 125-codon ORF           electron microscopy carried div-355 mutant cells <br /> divIC, used PCR generate fragment internal             collected 7 h end exponential growth DS <br /> 125-codon ORF extending nucleotides 540               medium. No septa located medial polar <br /> 796 Fig. 2. A plasmid containing insert, pPL27,      positions observed large number mutant cells <br /> integrated chromosome wild-type strain disrupt       examined electron microscopy. A typical example a <br /> the ORF (see Materials Methods). Disruption                sporulated div-355 mutant cell shown Fig. 6. <br /> 125-codon ORF generated severe phenotype the <br /> original div-355 mutation. The disruption resulted filamen- <br /> tous cells lysed plates days 250C                      3000- <br /> were unable grow 420C LB medium. In addition,                                                                         A <br /> strains carrying disruption exhibited poor growth DS <br /> agar plates, low temperature, phenotype                                2000- <br /> conferred original div-355 mutation. <br />    The product divIC gene required formation of <br /> the sporulation septum. As previously reported (31, 32), we <br /> were unable propagate div-355 mutants liquid temper-                          1000 <br /> atures 420C above. However, 390C a <br /> permissive temperature growth div-355 mutant cells <br /> in DS medium (the cells exhibited normal doubling time but <br /> eventually lysed stationary phase prolonged incu- <br /> bation). At temperature, div-355 mutant exhibited                                                                   B <br /> Spo - phenotype (characterized frequency heat-resis-                            150. <br /> tant spores 1 10&apos;), congenic <br /> wild-type strain PY79 sporulated frequency 57% (see <br /> Materials Methods). This result confirms previously                             100 <br /> reported requirement divIC sporulation, determined in <br /> experiments mutant cells grown                               50 <br /> permissive temperature resuspended sporulation me-                   PCD <br /> dium 420C (40). <br />    In previous experiments based use isopropyl- <br /> P3-D-thiogalactopyranoside-inducible promoter regulate the <br /> expression ftsZ, shown formation sporu- <br /> lation septum required transcription genes under <br /> the control uE a&apos; (5, 27). Thus, aF- a&apos;-directed <br /> gene expression considered diagnostic sporulation                        0 <br /> septum formation. As indication divIC gene <br /> is required polar septation, investigated effect the <br /> div-355 mutation transcription gene expressed prior <br /> to septation (spoIlA) genes control (TF <br /> (spolIIG) a&apos; (spoIID). To this, strains carried the <br /> div-355 mutation lacZ fusions genes were <br /> constructed. <br />    The initial pattern induction spoIIA-lacZ div-355                             0       2        4         6         8 <br /> mutant similar observed wild-type cells, by <br /> 3 h end exponential growth, level expression                                          Time (hours) <br /> began exceed wild-type cells, eventually reaching          FIG. 5. Effect div-355 mutation production n-ga- <br /> a fivefold higher level (Fig. 5A). This result significant <br />                                                                        lactosidase reporter genes, spoII-lacZ (A), spoIID-lacZ <br /> because indicates mutant cells capable             (B), spoIIIG-lacZ (C), fusion-bearing derivatives wild-type <br /> entering sporulation. We interpret overexpression               strain PY79 (El) congenic strains div-355 mutation (U). <br /> spoIIA-lacZ sign mutant cells idling         Strains cultured DS medium 390C. Samples collected <br /> preseptation stage sporulation, spoIIA-lacZ         indicated times relative end exponential growth and <br /> induced. In contrast overexpression spoIIA-lacZ,         assayed P3-galactosidase activity. <br />  VOL. 176, 1994                                                                                         B. SUBTILIS divIC GENE           1457 <br />  <br />  <br />  <br />  <br />   FIG. 6. Electron micrograph typical div-355 mutant cell collected 7 h end exponential growth DS medium 37&#176;C. Bar, 1 <br /> ,um. <br />  <br />  <br />   divIC essential gene. Both div-355 mutant           entering sporulation obligately requires formation of <br /> divIC pPL27 insertion mutant exhibited impaired growth              polar septum. Consistent interpretation, tem- <br /> were nonviable temperatures 39&#176;C below. How-              perature-sensitive sporulation mutation spoIIN279 re- <br /> ever, mutants, 80% divlC gene                  cently shown (21) allele B. subtilis homolog of <br /> remains intact. To investigate divIC essential            E. coli cell division gene ftsA. Like div-355, spoIIN279 <br /> gene, created divICA::spr (see Materials Methods),            mutation confers sporulation defect moderate fila- <br /> which two-thirds ORF deleted (from codon 39 7 bp           mentation temperatures permissive growth (21, <br /> downstream stop codon ORF; Fig. 2). We               48). <br /> constructed partially diploid strain, PL171, carries              Although predicted product divIC does share <br /> wild-type copy divIC amyE locus deletion              significant amino acid sequence similarity proteins <br /> mutation divIC locus (see Materials Methods).                currently GenBank DNA data base, bears striking <br /> Chromosomal DNA PL171 used transform                structural resemblance E. coli cell division protein FtsL <br /> 30&#176;C competent cells partially diploid strain (PL108)              (17). Both polypeptides roughly 14 kDa size and, like the <br /> containing wild-type copy divIC amyE           putative DivIC protein, FtsL stretch hydrophobic <br /> divIC loci haploid strain (PY79) containing single copy         amino acids serve membrane-spanning do- <br /> of divIC+ divIC locus Specr. Drug-resistant transfor-         main. Also, proteins hydrophobic amino acids are <br /> mants readily recovered PL108 PY79,              followed putative leucine zipper motif. It known that <br /> on basis infer divICA::spr recessive lethal          FtsL essential membrane protein required cell <br /> mutation. As control, PY79 competent cells readily               division E. coli. TnphoA analysis indicates FtsL is <br /> transformed chromosomal DNA carrying spr insertion             positioned cytoplasmic membrane way the <br /> a site distinct divIC (the site located approximately        carboxyl terminus containing leucine zipper motif the <br /> 150&#176; [39]).                                                             periplasmic space (17). It known putative <br />                                                                         leucine zipper FtsL used dimerization domain. <br />                                                                            The ftsL gene E. coli located 2-min region the <br />                          DISCUSSION                                     E. coli chromosome, immediately upstream pbpBE, which <br />                                                                         encodes penicillin-binding protein 3. A B. subtilis homolog of <br />    We shown divIC encodes 14.7-kDa polypeptide              pbpBE,pbpB, recently identified (47) approximately 1330 <br /> that required septum formation growth            chromosome, region contains genes for <br /> sporulation. In respect, similar well-character-      cell division cell wall metabolism homologous to <br /> ized division gene ftsZ, shown                genes 2-min region E. coli chromosome. Imme- <br /> necessary symmetric asymmetric cell division B.         diately upstream pbpB B. subtilis, position homol- <br /> subtilis (5, 27). Additionally, divIC shares characteristics       ogous E. coli ftsL, gene, orfA, capable of <br /> with B. subtilis cell division gene, divIB,            encoding polypeptide 13 kDa. While gene product of <br /> thought necessary controlling rate division            orfA hydrophobic domain located position <br /> initiation (19). The predicted products divIB divIC         similar FtsL DivIC (47), does share any <br /> contain putative membrane-spanning domains, mutations               significant sequence similarity FtsL, does any <br /> in genes (div-355 divIC tms-12 tsl2 divIB)           apparent leucine zipper motifs (47). Interestingly, however, the <br /> render cells temperature sensitive growth (4, 9, 19, 20).       orfA gene product does small degree amino acid <br /> However, unlike divIB, dispensable growth low           similarity DivIC (24). Although function orfA <br /> temperatures (4), divIC required viability temper-        gene known, attempts disrupt orfA unsuccessful, <br /> atures.                                                                 suggesting essential gene (47). Perhaps the <br />    A noteworthy finding present investigation         products divIC, ftsL, orfA <span id='am-58' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-59' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> small <br /> temperatures permissive growth, div-355                transmembrane proteins play critical role prokaryotic <br /> mutation substantially impairs sporulation, causing develop-            cell division. <br /> ment arrest prior onset polar septation. This re-             Intriguingly, orfA divIC bear similarity genes <br /> sult indicate divIC gene product serves                involved vesicle formation eukaryotes. The orfA gene <br /> special function sporulation different              product significantly similar alpha-SNAP, mammalian <br /> that performs growth. Alternatively, likely,         vesicle fusion protein binds cellular fusion protein <br /> this result simply indicate requirement              NSF Golgi membranes (10, 24). Likewise, DivIC similar <br /> septation stringent sporulation        product <span id='am-34' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-35' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-49'  ></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> gene BMH2. When <br /> growth. Thus, occasional septation event suffi-               present multiple copies, BMH2 capable rescuing yeast <br /> cient growth laboratory conditions, cell            mutant lacks clathrin heavy chain (23, 35) (GenBank <br />  1458        LEVIN AND LOSICK                                                                                                             J. BACTERIOL. <br />  <br /> accession number U01883). Conceivably, cell division B.                         Bacillus subtilis division initiation gene homolog has <br /> subtilis functionally related processes endocytosis                   identified organism. J. Bacteriol. 171:6835- <br /> and exocytosis eukaryotes.                                                      6839. <br />                                                                               21. Karmazyn-Campelli, C., L. Fluss, T. Leighton, P. Stragier. <br />                                                                                    1992. The spoIIN279 mutation affects FtsA protein Bacillus <br />                        ACKNOWLEDGMENTS                                             subtilis. Biochimie 74:689-694. <br />                                                                               22. Lee, S., C. W. Price. 1993. The minCD locus Bacillus subtilis <br />   We grateful Yoshito Sadaie gift strain NIG1154             lacks minE determinant provides topological specificity to <br /> to Phillip Youngman sending plasmid p280.1. We thank                   cell division. Mol. Microbiol. 7:601-610. <br /> Kirsten Price help localization divIC gene, Patrick       23. Lemmon, S. K. Personal communication. <br /> Stragier advice analyzing tRNA sequence data, Adam             24. Levin, P. A., R. Losiclk Unpublished data. <br /> Driks electron micrograph sporulating div-355 mutant             25. Levin, P. A., P. Margolis, P. Setlow, R. Losick, D. Sun. 1992. <br /> cell.                                                                              Identification Bacillus subtilis genes septum placement and <br />    This work supported grant (GM18568) NIH (to                   shape determination. J. Bacteriol. 174:6717-6728. <br /> R.L.).                                                                        26. Lutkenhaus, J. 1993. FtsZ ring bacterial cytokinesis. Mol. <br />                                                                                    Microbiol. 9:403-409. <br />                               REFERENCES                                      27. Margolis, P. 1993. Ph.D. thesis. Harvard University, Cambridge. <br />  1. Alber, T. 1992. Structure leucine zipper. Curr. Opin. Genet.       28. Margolis, P., A. Driks, R. Losick. 1993. Sporulation gene <br />     Dev. 2:205-210.                                                                spoIIB Bacillus subtilis. J. Bacteriol. 175:528-540. <br />  2. Banner, C. 1983. Ph.D. thesis. Harvard University, Cambridge.             29. McCabe, P. C. 1990. Production single-stranded DNA by <br />  3. Beall, B., M. Lowe, J. Lutkenhaus. 1988. Cloning char-                 asymmetric PCR, p. 76-83. In M. H. Innis, D. H. Gelfand, J. J. <br />     acterization Bacillus subtilis homologs Escherichia coli cell            Snisky, T. J. White (ed.), PCR protocols: guide methods <br />     division genes ftsZ ftsA. J. Bacteriol. 170:4855-4864.                     applications. Academic Press, San Diego, Calif. <br />  4. Beall, B., J. Lutkenhaus. 1989. Nucleotide sequence               30. McKnight, S. L. 1991. Molecular zippers gene regulation. Sci. <br />     insertional inactivation Bacillus subtilis gene affects cell         Am. 264:54-64. <br />     division, sporulation, temperature sensitivity. J. Bacteriol.         31. Mendelson, N. H. 1977. Cell growth division: genetic <br />     171:6821-6834.                                                                 viewpoint, p. 5-24. In D. Schlessinger (ed.), Microbiology-1977. <br />  5. Beall, B., J. Lutkenhaus. 1991. FtsZ Bacillus subtilis               American Society Microbiology, Washington, D.C. <br />     required vegetative septation asymmetric septation            32. Mendelson, N. H., R. M. Cole. 1972. Genetic regulation of <br />     sporulation. Genes Dev. 5:447-455.    <br /> </body></html>