<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Targeted metabolic labeling yeast <br /> N-glycans unnatural sugars <br /> Mark A. Breidenbacha, Jennifer E. G. Gallagherb, David S. Kingc, Brian P. Smarta, <br /> Peng Wua,2, Carolyn R. Bertozzia,b,c,d,1 <br /> a <br />  Departments Chemistry; bMolecular Cell Biology; cHoward Hughes Medical Institute, University <span id='am-6' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>California</span>, Berkeley, CA 94720; <br /> and dThe Molecular Foundry, Lawrence Berkeley National Laboratory, Berkeley, CA 94720 <br />  <br /> Edited David A. Tirrell, California Institute Technology, Pasadena, CA, approved December 23, 2009 (received review September 30, 2009) <br />  <br /> Metabolic labeling glycans synthetic sugar analogs               mass spectrometric analyses proteins displaying nonna- <br /> emerged attractive means introducing nonnatural chemi-             tural chemical functionality (5, 6). <br /> cal functionality glycoproteins. However, complexities              While existing glycan labeling methodology suitable stud- <br /> glycan biosynthesis prevent installation nonnatural moieties          ies require stochastic insertion analogs low levels, <br /> at defined, predictable locations glycoproteins high levels        technique inadequate applications specific <br /> of incorporation. Here, demonstrate conserved                    monosaccharides reliably targeted metabolic replace- <br /> N-acetyglucosamine (GlcNAc) residues chitobiose cores              ment. For example, high-efficiency predictable installation of <br /> N-glycans model organism <span id='am-17' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-18'  ></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>                 sugar analogs dramatically facilitate biophysical studies of <br /> be specifically targeted metabolic replacement unnatural              glycoprotein structure function site-specific introduction <br /> sugars. We introduced exogenous GlcNAc salvage pathway               fluorophores heavy atoms. Unfortunately, site-specific me- <br /> yeast, allowing cells metabolize GlcNAc provided supple-             tabolic labeling glycans hindered multiple factors. First, <br /> ment culture medium. We rendered yeast auxo-                 nucleotide-sugar analogs necessarily direct competition <br /> trophic production donor nucleotide-sugar uridine-                cell&#8217;s endogenous pool donor sugars, leading to <br /> diphosphate-GlcNAc (UDP-GlcNAc) deletion essential                 poor levels metabolic incorporation synthetic analog (7). <br /> gene GNA1. We demonstrate gna1&#916; strains require GlcNAc                Further, eukaryotes possess epimerase activities can <br /> supplement expression <span id='am-82' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> containing exoge-               interconvert nucleotide-sugar stereochemistries alter <br /> nous components salvage pathway, GlcNAc transporter                   final metabolic destination given monosaccharide (8, 9). <br /> NGT1 <span id='am-41' about='obo:NCBITaxon_5475' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-42'  ></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Torulopsis" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Candida" datatype="xsd:string"></span>Candida</span> albicans GlcNAc kinase NAGK <span id='am-63' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-64'  ></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Homo</span>                  Finally, glycan assembly genetically encoded is <br /> sapiens, required rescue context. Further,           inherently prone microheterogeneity, targeting ana- <br /> that cells successfully incorporate synthetic GlcNAc analogs                 logs specific positions defined <span id='am-55' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-56' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> glycoconjugates <br /> N-azidoacetyglucosamine (GlcNAz) N-(4-pentynoyl)-glucosa- <br />                                                                              untenable. <br /> mine (GlcNAl) cell-surface glycans secreted glycoproteins. <br />                                                                                 Even glycan biosynthesis template-driven, some <br />                                                                              glycans share conserved structural features result pro- <br /> To verify incorporation nonnatural sugars N-glycan core <br />                                                                              cess assembled. Asparagine-linked glycans <br /> positions, endoglycosidase H (endoH)-digested peptides a <br />                                                                              (N-glycans) excellent example; antennary regions <br /> purified secretory glycoprotein, Ygp1, analyzed mass <br />                                                                              N-glycans ultimately edited diverse compositions and <br /> spectrometry. Multiple Ygp1 N-glycosylation sites bearing GlcNAc, <br />                                                                              structures, N-glycan cores derived conserved <br /> isotopically labeled GlcNAc, GlcNAz identified; mod- <br />                                                                              tetradecasaccharide precursor featuring &#946;1,4-linked GlcNAc <br /> ifications dependent supplement added culture <br />                                                                              chitobiose unit site polypeptide attachment (10). Both <br /> medium. <span id='am-58' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-61'  ></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> enables production glycoproteins that <br />                                                                              GlcNAc monomers chitobiose unit, donated UDP- <br /> are functionalized specific chemical modifications their <br />                                                                              GlcNAc, persist lifespan N-glycan. Here, <br /> glycosylation sites.                                                         target conserved core GlcNAc residues metabolic re- <br />                                                                              placement, effectively sidestepping issue glycan structural <br /> click chemistry &#8739; GlcNAc &#8739; metabolic engineering &#8739; N-glycosylation &#8739; GNA1 <br />                                                                              heterogeneity (Fig. 1A). N-glycans uniquely suited for <br />                                                                              targeted metabolic labeling profoundly influ- <br /> T   introduction unnatural chemical moieties proteins <br />     enormous potential facilitate vivo vitro <br /> studies protein function, mechanism, structure. For exam- <br />                                                                              ence protein&#8217;s folding, trafficking, solubility, stability (11); <br />                                                                              types glycosylation, N-glycan occu- <br />                                                                              pancy mandatory function. Indeed, occupancy many <br /> ple, genetically encoded residue-specific approaches                 N-glycosylation sites sufficiently high observe core carbohy- <br /> introduction unique amino acids enabled integration               drate residues crystallographically (12). <br /> several functionalities spectroscopic probes, bioorthogonal             Targeted glycan labeling dramatically simplified <br /> chemical reporters, metal chelators, photoaffinity labels                working organism relatively well-characterized <br /> directly recombinant proteins (1, 2). As alternative              glycan biosynthesis. In study, utilize model eukaryote <br /> the insertion nonnatural amino acids polypeptide chain,            S. cerevisiae reasons including relatively simple <br /> unique chemical functionality introduced a <br /> protein&#8217;s posttranslational modifications exemplified the <br />                                                                              Author contributions: M.A.B. designed research; M.A.B., J.E.G.G., D.S.K. performed <br /> technique glycan metabolic labeling (3). In approach,                research; M.A.B., J.E.G.G., B.P.S. analyzed <span id='am-77' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-78' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-80'  ></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span>; M.A.B., J.E.G.G., C.R.B. wrote <br /> synthetic analogs natural monosaccharides introduced             paper; P.W. contributed new reagents/analytic tools. <br /> cells processed series enzymes generate            The authors declare conflict interest. <br /> activated nucleotide-sugar analogs. The sugar analogs, bearing               This article PNAS Direct Submission. <br /> unique chemical functionality, subsequently utilized              1 <br />                                                                                  To correspondence addressed. E-mail: crb@berkeley.edu. <br /> biosynthesis various cellular glycoconjugates; technique             2 <br />                                                                                  Present address: Department Biochemistry, Albert Einstein College Medicine, <br /> has vividly illustrated recent glycan imaging studies (4).               Yeshiva University, Bronx, NY 10461. <br /> Additionally, introduction synthetic sugars glycans              This article contains supporting information online www.pnas.org/cgi/content/full/ <br /> via metabolic labeling exploited affinity capture               0911247107/DCSupplemental. <br />  <br />  <br />  <br /> 3988&#8211;3993 &#8739; PNAS &#8739; March 2, 2010 &#8739; vol. 107 &#8739; no. 9                                                               www.pnas.org/cgi/doi/10.1073/pnas.0911247107 <br />                                                                                    We demonstrate synthetic, azide- alkyne-bearing <br />                                                                                GlcNAc analogs isotopically labeled GlcNAc be <br />                                                                                introduced yeast N-glycans unique salvage pathway. <br />                                                                                Azide alkyne functional groups known versatile <br />                                                                                chemical handles bioconjugation proteins, cells, living <br />                                                                                organisms (20). Here, utilize phosphine (21), alkyne (22), and <br />                                                                                azide (23) probes conjunction mass spectrometry verify <br />                                                                                GlcNAc analogs successfully metabolized integrated <br />                                                                                yeast cell surfaces secreted glycoproteins. <br />                                                                                Results Discussion <br />                                                                                Design gna1&#916; Strains. The four-step novo UDP-GlcNAc bio- <br />                                                                                synthetic pathway (Fig. 1B) responsible metabolic conver- <br />                                                                                sion fructose-6-phosphate, intermediate glycolysis, to <br />                                                                                UDP-GlcNAc. The highly conserved enzymes pathway are <br />                                                                                essential yeast viability normal growth conditions (24). <br />                                                                                An interruption UDP-GlcNAc biosynthetic pathway be <br />                                                                                rescued appropriate downstream metabolite delivered <br />                                                                                cells. For example, gfa1&#916; strains S. cerevisiae capable <br />                                                                                internalizing phosphorylating extracellular glucosamine, <br />                                                                                effectively bypassing GFA1 deletion (25). Some organisms, <br />                                                                                C. albicans, functional salvage pathways allowing <br /> Fig. 1. Targeting GlcNAc metabolic replacement. (A) The structurally       internalize phosphorylate GlcNAc (18, 19, 26), <br /> conserved GlcNAc2 Man8 core region S. cerevisiae N-glycan shown at-      S. cerevisiae lacks activities gna1&#916; strains cannot <br /> tached hypothetical integral membrane protein. N-glycan cores <span id='am-68' about='obo:NCBITaxon_9822' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9821 obo:NCBITaxon_35497 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-69'  ></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="Sus" datatype="xsd:string"></span>sus</span>-    rescued extracellular GlcNAc (Fig. 2). Because GlcNAc <br /> ceptible specific cleavage endoH PNGaseF indicated. Synthetic <br /> GlcNAc analogs bearing bioorthogonal chemical groups azides and <br />                                                                                analogs modifications C-2 acetamido substituent <br /> alkynes allow bioconjugation diverse probes cargos directly      known tolerated downstream eukaryotic biosynthetic <br /> glycans. (B) A strategy bypassing novo UDP-GlcNAc biosynthesis (black   machinery (27), chose interrupt UDP-GlcNAc production <br /> arrows) shown. An exogenous salvage pathway (blue arrows) allows extra-     N-acetyltransferase step catalyzed Gna1. Glucosamine, <br /> cellular GlcNAc analogs internalized transporter Ngt1     produced chemical enzymatic deacetylation of <br /> C. albicans. The intracellular GlcNAc (or analog) phosphorylated 6   GlcNAc, rescue gna1&#916; strains does case of <br /> position activity human GlcNAc kinase, NAGK (28). The           GFA1 disruption. To bypass GNA1 disruption, first <br /> 6-phosphorylated product subsequently converted activated <br />                                                                                necessary bestow cells functional alternative path- <br /> nucleotide-sugar mutase pyrophosphorylase activities Pcm1 <br /> and Qri1 respectively. <br />                                                                                way, illustrated Fig. 1B. We introduced expression plasmids <br />                                                                                bearing human GlcNAc kinase NAGK heterozygous <br />                                                                                GNA1/gna1&#916; strain S. cerevisiae primarily NAGK <br /> composition glycans ease yeast                previously shown tolerate GlcNAc analogs vitro <br /> genetically modified. Importantly, UDP-GlcNAc known                      (28). This enzyme capable phosphorylating position <br /> provide GlcNAc S. cerevisiae glycoconjugate types                 cytosolic GlcNAc (or analogs), delivering required <br />  <br />  <br />  <br />  <br />                                                                                                                                                             CHEMISTRY <br /> besides core chitobiose units N-glycans. UDP-GlcNAc do-                 metabolite compensate GNA1 disruption. NAGK tran- <br /> nates GlcNAc chitin, linear polymer &#946;-1,4-GlcNAc (13)                scription placed control constitutive <br /> that minor component yeast cell wall deposited              glyceraldehyde-3-phosphate dehydrogenase (GPD) promoter in <br /> as ring neck growing bud cell division.             pRS41X vector scaffolds URA3, HIS3, LEU2 selective <br /> Additionally, UDP-GlcNAc donor step GPI                markers (29). <br /> anchor biosynthesis, GlcNAc residue subsequently dea-                  To enable utilization extracellular GlcNAc, introduced <br /> cetylated (14). Unlike higher eukaryotes, S. cerevisiae does               plasmids carrying recently characterized C. albicans GlcNAc <br /> incorporate GlcNAc O-glycans (15) antennary regions                    transporter NGT1 GNA1/gna1&#916; strain (19). While <br />  <br />  <br />  <br />                                                                                                                                                             GENETICS <br /> of N-glycans (16). Furthermore, S. cerevisiae lacks epimerases                 NGT1 initially cloned pRS41X-GPD scaffolds, no <br /> that act UDP-GlcNAc (17). Thus, distribution GlcNAc                  transformants obtained plasmids containing <br /> in S. cerevisiae glycoproteome expected restricted             GPD promoters, suggesting S. cerevisiae sensitive to <br /> the chitobiose cores N-glycans.                                             NGT1 expression level. NGT1 subsequently placed the <br />    To address issue competing endogenous UDP-                       galactose-inducible GAL1 promoter remains largely re- <br /> GlcNAc pool, hypothesized genetic disruption novo                pressed glucose-containing medium. The combinations of <br /> UDP-GlcNAc biosynthesis removal essential glucos-                   NGT1 NAGK plasmids transformed GNA1/gna1&#916; are <br /> amine 6-phosphate N-acetyltransferase (GNA1) gene                        summarized Table S1. To isolate gna1&#916; haploids, subjected <br /> force cells scavenge environmental GlcNAc survive. Disrup-               heterozygous transformants sporulation. Notably, dually <br /> tion S. cerevisiae GNA1 possible viable alternate              transformed strains carrying NGT1 NAGK plasmids de- <br /> pathway provided complement loss (18). S. cerevi-              veloped tetrads. The GNA1/gna1&#916; heterozygotes sporulated <br /> siae does endogenous GlcNAc salvage pathway,                 slowly produced tetrads 7 d sporulation media; <br /> equipped strains two-component salvage pathway bor-                     supplementation GlcNAc difference. Dissections <br /> rowed organisms (Fig. 1B). Specifically, employed                resulting tetrads demonstrated high rate spore death of <br /> the recently characterized GlcNAc transporter NGT1                        25.5%   2.1% YPD &#254; GlcNAc. Only 28.4%   2.3% the <br /> <span id='am-47' about='obo:NCBITaxon_5476' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5475 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Candida stellatoidea" datatype="xsd:string"></span><span id='am-49'  ></span><span id='am-50' property="oboInOwl:hasAlternativeId" content="NCBITaxon:39399" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Candida stellatoidea type I" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:12" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Candida albicans" datatype="xsd:string"></span>Candida albicans</span> (19) human GlcNAc kinase NAGK. This                       tetrads contained viable spores half (49.8%   5%) of <br /> functional salvage pathway allows bypass lethal               tetrads viable spores contained wild-type <br /> GNA1 deletion long cells provided GlcNAc-                 GNA1. It unclear gna1&#916; segregants demonstrated a <br /> supplement growth medium. With modifications                 high rate inviability. GNA1 disruption surviving segregants <br /> place, obtained viable gna1&#916; GlcNAc auxotrophs.                             confirmed PCR (Fig. S1). Surviving gna1&#916; segregants <br />  <br /> Breidenbach et al.                                                                                 PNAS &#8739;   March 2, 2010 &#8739;   vol. 107 &#8739;   no. 9 &#8739;   3989 <br />  Fig. 2. Verification exogenous GlcNAc salvage pathway S. cerevisiae. Tenfold serial dilutions cultures GNA1 gna1&#916; carrying <br /> combinations NGT NAGK plasmids spotted solid media reveal necessity extracellular GlcNAc source NGT1 NAGK-encoding <br /> plasmids essential gna1&#916; rescue. Haploid gna1&#916; strains (rows 2, 4, 6) grow rich media supplemented GlcNAc (YPD &#254; GlcNAc) <br /> while wild-type cells showed difference growth (rows 1, 3, 5). The requirement NAGK NGT1 encoded pRS416 plasmids gna1&#916; yeast is <br /> demonstrated growth 5-<span id='am-12' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-13'  ></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Foa" datatype="xsd:string"></span>FOA</span> &#254; GlcNAc media. Cells containing pRS413-GAL-NGT1 pRS415-GPD-NAGK (rows 1-2) insensitive 5-FOA, if <br /> either NGT1 NAGK introduced pRS416 plasmid, 5-FOA prevents growth gna1&#916;, GNA1 strains (rows 3-6). <br />  <br />  <br /> demonstrated GlcNAc auxotrophy (Fig. 2) doubling                    ment. The secreted proteins GlcNAc- GlcNAz- <br /> time approximately twice wild-type haploids.                          supplemented cultures subjected Staudinger ligation <br />    To verify plasmids bearing NGT1 NAGK essen-                  (21) phosphine-FLAG (<span id='am-87' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-88' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91'  ></span>phos</span>-FLAG) chemospecifically <br /> tial GlcNAc salvage gna1&#916; rescue, negative selection <br /> scheme implemented. Haploid strains harboring either <br /> NAGK NGT1 URA3 marked plasmid (pRS416) <br /> were isolated dissected tetrads. To assess growth re- <br /> quirements various gna1&#916; haploids, strains were <br /> spotted 5-fluoroorotic acid (5-FOA) media &#254; GlcNAc <br /> (Fig. 2). Cells containing URA3 marked plasmid convert <br /> 5-FOA 5-fluorouracil grow presence this <br /> compound, enabling negative selection indivi- <br /> dual component salvage pathway (30). Because gna1&#916; cells <br /> could grow 5-FOA media NGT1 NAGK <br /> was encoded pRS416 (URA3) plasmid, genes <br /> were encoded pRS415 (LEU2) pRS413 (HIS3), respec- <br /> tively, conclude gna1&#916; yeast rescued both <br /> NAGK NGT1 gene products expressed growth <br /> medium supplemented GlcNAc. <br />    NGT1 transcription controlled GAL1 promoter <br /> which repressed glucose-containing media. Despite pre- <br /> sence glucose rich medium synthetic dropout (SD) <br /> plates, GAL1 promoter fully repressed con- <br /> firmed low levels NGT1 NAGK mRNA transcripts in <br /> cells qPCR. If cells grown nonrepressive, galactose- <br /> containing media (YPGAL &#254; GlcNAc) observed apparent <br /> toxicity Ngt1 gna1&#916; cells (Fig. 2). Wild-type cells grown <br /> on YPGAL &#254; GlcNAc selective pressure to <br /> retain NGT1 plasmid able grow on <br /> YPGAL medium (Fig. 2). These observations suggest high <br /> level NGT1 expression toxic S. cerevisiae. <br />  <br /> Incorporation Synthetic Sugars Glycoproteins. Significantly, <br /> exogenous salvage pathways serve convenient routes in- <br /> troduce nonnatural sugar analogs glycans. For example, an <br /> artificial fucose salvage pathway recently exploited install <br /> fucose analogs cell-surface polysaccharides E. coli in <br /> which novo biosynthesis donor nucleotide-sugar GDP- <br /> fucose disrupted (31). We conducted experi- <br />                                                                               Fig. 3. Introduction GlcNAc analogs cell-surface secreted glyco- <br /> ments determine GlcNAc salvage pathway gna1&#916; yeast <br />                                                                               proteins. (A) Secreted glycoproteins (5 &#956;g total protein load/lane) gna1&#916; <br /> could assimilate unnatural GlcNAc analogs secreted glyco-                cells expressed medium supplemented GlcNAc GlcNAz were <br /> proteins. First, utilized azide-bearing analog GlcNAz. We              labeled phos-FLAG analyzed &#945;-FLAG immunoblotting. PNGaseF <br /> observed cell proliferation gna1&#916; cultures administered               treatment totally removes N-glycans prevents phos-FLAG ligation (lanes <br /> with pure unnatural sugar supplement severely impaired                  2,4). Samples treated endoH (lanes 6, 8) remain reactive phos-FLAG; <br /> but cells remained viable&#8212;they continued produce                  proteins detected &#945;-FLAG blotting collectively downshifted molecu- <br /> protein readily proliferated switched GlcNAc supple-              lar weight deglycosylation. Cell-surface azidosugar-bearing glycans <br /> ment, supplemented GlcNAc/analog mix. To increase                readily detected fluorescence microscopy following chemos- <br />                                                                               pecific labeling alk-AF488; GlcNAz-supplemented gna1&#916; cells (B) show <br /> yields secreted protein, adopted strategy growing <br />                                                                               strong reactivity alk-AF488 display morphological abnormalities, <br /> cells midlog phase presence GlcNAc pellet-                observed gna1&#916; strain supplemented with <br /> ing, washing, resuspending analog-supplemented culture                 GlcNAc (C). Similarly, GlcNAl-supplemented gna1&#916; cells (D) strongly <br /> medium. Conditioned medium gna1&#916; yeast cultures                      labeled N3 -AF647, GlcNAc-supplemented cells (E) not. Nuclear <br /> collected desalted remove unmetabolized sugar supple-                  mitochondrial DNA indicated DAPI stain. <br />  <br />  <br /> 3990 &#8739;   www.pnas.org/cgi/doi/10.1073/pnas.0911247107                                                                                     Breidenbach et al. <br />  modify resident azides (Fig. 3A). The labeled glycoproteins       Characterization Ygp1 N-glycans. A single, highly glycosylated <br /> were analyzed immunoblotting &#945;-FLAG anti-             protein purified conditioned growth medium per- <br /> body. Notably, secreted proteins GlcNAz-supplemen-          form precise analysis analog location labeling ef- <br /> ted cultures showed immunoreactivity &#945;-FLAG, indicating          ficiency. YGP1 encodes secretory glycoprotein unknown <br /> azides present glycoproteins.                     function containing 15 potential sites N-glycosylation (36). <br />    To verify azide incorporation limited N-glycan cores,        YGP1 placed control GPD promoter the <br /> we subjected secreted protein fractions digestion         multicopy pRS426 vector C-terminal polyhistidine tag, <br /> either peptide:N-glycosidase F (PNGaseF) endoglycosidase           transformed gna1&#916; yeast, expressed presence of <br /> H (endoH), specifically target N-glycans (33). Consistent       GlcNAc, isotopically labeled GlcNAc, GlcNAz supplements. <br /> with N-glycan-specific labeling, PNGaseF-digested proteins lost       Ygp1 affinity purified near-homogeneity assessed by <br /> reactivity phos-FLAG (Fig. 3A). However, secreted glyco-         silver-stained SDS-PAGE (Fig. 4A). The purified glycoprotein <br /> proteins reacted phos-FLAG following endoH diges-          subjected endoH digestion remove core <br /> tion. We observed overall downshift molecular weight            GlcNAc (or GlcNAz) residues, causing significant downshift in <br /> consistent deglycosylation. These data confirm dis-     molecular weight. The identity Ygp1 confirmed immu- <br /> played azides located N-glycans glycoproteins      noblotting polyhistidine tag. Ygp1 expressed GlcNAz- <br /> the core GlcNAc residue glycans carries phos-        supplemented medium (Fig. 4A) highly reactive phos- <br /> phine-reactive azide modification.                                    FLAG fully glycosylated endoH-treated samples <br />    GlcNAc analogs displayed cell-surface glycoproteins        readily detected &#945;-FLAG blotting. <br /> detected fluorescence microscopy. In case GlcNAz- <br /> supplemented yeast, utilized alkyne-functionalized <span id='am-1' about='obo:NCBITaxon_149625' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_163739 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="rdfs:label" content="Alexa" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Alexa</span>- <br /> fluor-488 (alk-488) dye chemospecifically modify azides Cu <br /> (I)-catalyzed azide-alkyne cycloaddition (CuAAC) (22). GlcNAz- <br /> supplemented gna1&#916; yeast reactive alk-488, while <br /> GlcNAc-supplemented cells (Fig. 3B, C). Surfaces were <br /> not uniformly intense fluorescence, possibly differences <br /> in cell age. In cases, alk-488 strongly labeled bud-neck <br /> regions cells. Notably, GlcNAz-supplemented cells <br /> appeared display morphological cytokinetic defects; <br /> branched, multicell chains frequently observed cul- <br /> tures (Fig. 3B). The abnormalities observe cell shape, size, <br /> and bud-neck morphology strikingly similar observed <br /> in strains chitin synthases CHS1 CHS2 been <br /> deleted (34). In contrast, GlcNAc-supplemented gna1&#916; cells dis- <br /> played little reactivity alk-488 comparatively normal <br /> morphology (Fig. 3C). In small minority GlcNAz-supplemen- <br /> ted/alk-488 labeled cells, observed strongly fluorescent ring- <br /> like features cell surfaces (Fig. S2) consistent bud scars, <br /> the remnants primary septa seen surfaces mother <br /> cells following division. Bud scars primary repository of <br />  <br />  <br />  <br />  <br />                                                                                                                                                         CHEMISTRY <br /> chitin, chitin-specific stain calcofluor white (CW) (35) colo- <br /> calized alk-488 bud scars, possibly indicating presence <br /> of azide-bearing chitinous deposits. More frequently, however, <br /> we observed notable decrease CW fluorescence GlcNAz- <br /> supplemented cells relative supplemented GlcNAc. <br /> We speculate chitin synthases S. cerevisiae rela- <br /> tively intolerant GlcNAz analog defects chitin <br /> production lead observed morphological abnormalities and <br />  <br />  <br />  <br />                                                                                                                                                         GENETICS <br /> decreased viability. <br />    Similarly, cultures supplemented alkyne-bearing GlcNAl        Fig. 4. Analysis GlcNAz incorporation Ygp1. (A) Polyhis-tagged Ygp1 <br /> were probed azide-functionalized Alexafluor-647 (N3 -647).       overexpressed GlcNAz-supplemented SD-URA medium purified. <br /> GlcNAl-supplemented gna1&#916; yeast reactive N3 -647            Identical (1 nanogram) loads Ygp1 subjected detection silver- <br /> and displayed strong fluorescence surfaces,            stain (lanes 1&amp;2), immunoblotting &#945;-HIS (lanes 3&amp;4) &#945;-FLAG <br /> GlcNAc-supplemented cells did (Fig. 3D, E). GlcNAl-supple-        (lanes 5&amp;6). Film exposure times varied generate visibly comparable <br /> mented cells demonstrated morphological abnormalities            Western blots: lane 3, 1 hour; lane 4, 1 min, lanes 5&amp;6, 1 second. The required <br /> such increased size relative GlcNAc-supplement       differences reflect variability epitope accessibility. A molar excess of <br />                                                                       EndoHf (silverstain band **) used remove N-glycans heavily <br /> cultures. As GlcNAz, increased tendency intercellular     glycosylated Ygp1 (silverstain band *) migrates diffuse <br /> connectivity observed GlcNAl supplement; single          high-molecular weight smear (lanes 1 2). Samples probed with <br /> cells rarely observed cultures.                         &#945;-FLAG subjected chemospecific phos-FLAG ligation prior to <br />    In S. cerevisiae, UDP-GlcNAc known provide GlcNAc       blotting; immunodetection FLAG peptide indicates presence GlcNAz. <br /> for GPI anchor biosynthesis. While GlcNAc coupled phos-         (B) Ygp1 expressed culture medium supplemented either <br /> photidylinositol step GPI anchor biosynthesis,    GlcNAc 13 C6 -GlcNAc. Ygp1 treated endoH, trypsinized, sub- <br /> C-2 acetamido substituent promptly removed essential        jected ESI-FTICR MS analysis. Masses representative glycopeptide, <br />                                                                       spanning Leu 98-Arg 115 glycosylated potential <br /> deacetylase activity Gpi12 (14). While expect to <br />                                                                       sites (indicated green), shown. Relative intensities GlcNAc- <br /> find azide alkyne functionality mature GPI anchors      13 C6 -GlcNAc-modified peptides normalized other. <br /> of analog-supplemented cells, explicitly discount       (C) Ygp1 expressed culture medium supplemented GlcNAz <br /> possibility GlcNAc analogs remain unprocessed Gpi12           subjected ESI-FTICR MS analysis. Masses corresponding same <br /> activity.                                                             glycopeptide (B) shown. <br />  <br />  <br /> Breidenbach et al.                                                                           PNAS &#8739;    March 2, 2010 &#8739;    vol. 107 &#8739;   no. 9 &#8739;   3991 <br />     To verify nonnatural sugar labeling core N-glycan         hydrogen bonds acetamido group transfer- <br /> positions, endoH-treated Ygp1 samples trypsinized              ase essential activity (39), features pre- <br /> the resulting peptides examined electrospray ionization         served GlcNAz GlcNAl analogs. Importantly, data do <br /> Fourier-transform ion cyclotron resonance mass spectrometry             explicitly confirm analog incorporation Asn-distal site <br /> (ESI-FTICR MS). Tryptic peptides covering 68% Ygp1               N-glycan core. Though available biochemical data suggest <br /> sequence successfully identified mass spectra              modifications C-2 acetamido group interfere <br /> samples derived cultures supplemented GlcNAc,                 Alg13&#8725;14 OT activities, explicit confirmation of <br /> N-acetyl-&#189;13 C6  glucosamine &#240;13 C6 -GlcNAc&#222;, GlcNAz (Table S3).     analog incorporation Asn-distal site subject future <br /> Five potential N-glycosylation sites observed        interest. Qualitatively, did observe reduction the <br /> be occupied. Specifically, positions Asn 100, 106, 118, 239,        glycosylation level Ygp1 expressed GlcNAz- <br /> 286 glycosylated, tryptic peptides covering        supplemented medium; expect decreased glycosylation <br /> the potential sites Ygp1 sequence ob-         GlcNAz strongly inhibited dolichol pathway. <br /> served. The tryptic peptide spanning Leu 98-Arg 115 contained <br /> two potential N-glycosylation sites peptide masses corre-           Conclusion <br /> sponding singly- doubly glycosylated forms          High-efficiency, site-specific replacement GlcNAc unna- <br /> peptide detected, unglycosylated peptide                tural analogs chitobiose cores N-glycans allows reli- <br /> observed. Tryptic peptides spanning Val 116-Lys 124 Glu             able predictable introduction unique functionalities into <br /> 236-Lys 314 observed fully glycosylated.                     N-glycans little risk perturbing glycoprotein structure <br />    A stable isotope GlcNAc used unequivocally verify          function. We envision GlcNAc analogs convenient means <br /> that supplement added culture medium metabolically        reliable introduction fluorophores, anomalous x-ray scat- <br /> incorporated N-glycan cores secreted glycoproteins. Ygp1        terers, crosslinkers, affinity tags directly glycoprotein <br /> expressed cultures supplemented 13 C6 -GlcNAc isotope         surfaces. Larger cargos, including peptides, macromolecules, or <br /> yielded tryptic glycopeptides expected 6 Da shift          cells, tethered N-glycosylated carrier proteins <br /> occupied glycosylation site (Fig. 4B, Table S3). The labeling           uniform, predictable sites attachment. Given utility of <br /> efficiency 13 C6 -GlcNAc appears extremely high;             yeast expression platform glycoprotein biotherapeutics, <br /> GlcNAc-modified peptides detectable cultures                technique use biotechnology applications well. <br /> supplemented 13 C6 -GlcNAc. These data indicate         Although tested azide, alkyne, isotopic GlcNAc <br /> possible achieve global replacement GlcNAc unnatural         analogs study, anticipate gna1&#916; yeast able <br /> isotopes gna1&#916; strains.                                              assimilate additional nonnatural sugars N-glycans, sub- <br />    Similarly, Ygp1 expressed presence GlcNAz ana-         ject future interest. We believe targeted metabolic labeling of <br /> lyzed ESI-FTICR MS determine high levels non-          N-glycans unnatural sugars afford numerous unique <br /> natural GlcNAc analog occupy N-glycan core positions. In           applications previously possible, ultimately facilitat- <br /> case, cultures grown midlog phase GlcNAc supple-         ing studies glycoprotein structure mechanism. <br /> ment switching GlcNAz slow doubling time <br /> of gna1&#916; cells GlcNAz-supplemented medium. The resulting             Materials Methods <br /> Ygp1 glycopeptide masses observed consistent             Preparation NAGK, NGT1, YGP1 Expression <span id='am-73' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-74' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Vectors</span>. Details pertaining to <br /> mix GlcNAc GlcNAz occupying core N-glycan posi-              plasmid construction included SI Text. <br /> tions (Fig. 4C, Table S3). The mixture GlcNAc GlcNAz <br /> modifications likely results cytosolic reservoir GlcNAc       Isolation gna1&#916; Strains Viability Assays. Standard rich SD media <br />                                                                         formulations lithium acetate transformation protocols used except <br /> that persists cells resuspended GlcNAz-supplemented <br />                                                                         gna1&#916; cells supplemented 100 &#956;M GlcNAc (or GlcNAz) liquid <br /> culture medium. Unfortunately, possible draw quan-         culture. Solid medium supplemented 200 &#956;M GlcNAc. A heterozy- <br /> titative conclusions GlcNAz&#8758;GlcNAc ratio               gous GNA1/gna1&#916; strain (ATCC#4025635; MAT a&#8725;MAT &#945;his3&#916;&#8725;his3&#916;leu2&#916;&#8725; <br /> glycopeptides detected mass spectrometry potential dif-       leu2&#916;lys2&#916;&#8725; &#254; met15&#916;&#8725; &#254; ura3&#916;&#8725;ura3&#916;yf l017c::KanMX4&#8725;&#254;) trans- <br /> ferences ionization efficiency. However, likely expres-   formed GAL1-NGT1 GPD-NAGK plasmids (Table S1) sporu- <br /> sion conditions optimized thoroughly deplete cells       lated 2% potassium acetate. Haploid segregants grown overnight in <br /> of internal GlcNAc reservoirs switching non-        media (SD-URA-HIS SD-HIS-LEU) select plasmids. Cells diluted <br /> natural sugar supplement, enabling extremely high labeling         107 mL serially diluted ten-fold. Approximately 5 &#956;L spotted <br /> efficiencies.                                                           solid media. Cells allowed grow 2&#8211;4 d 30 &#176;C then <br />                                                                         photographed. <br /> Implications Promiscuity Dolichol Pathway. Collectively, <br />                                                                         Microscopy. Procedures used imaging fluorophore-labeled cells are <br /> our data indicate enzymes involved early steps yeast <br />                                                                         described SI Text. <br /> N-glycan biosynthesis tolerate subtle chemical modifications <br /> of C-2 acetamido moiety GlcNAc. Here, present strong          Western Blots. Protein electrophoresis performed Criterion tris-HCl <br /> evidence acetamido group Asn-proximal GlcNAc            4&#8211;12% gels (BioRad). Polyhistidine-tagged proteins detected with <br /> be modified correctly processed phosphoglycosyl-           &#945;-5xHis-peroxidase conjugate kit (Qiagen) specified product literature. <br /> transferase Alg7 later oligosaccharyl transferase as-        Synthesis detection reagent phos-FLAG described previously (40). <br /> sembly (OT). During oligosaccharyl transfer, believed         Fully secreted yeast glycoproteins obtained conditioned medium <br /> that C-2 acetamido group GlcNAc acts stabilize           small-scale (5 mL) cultures centrifugation remove cells. Proteins were <br /> the oxonium intermediate prior nucleophilic attack            simultaneously concentrated buffer exchanged PBS using Amicon <br /> receiving Asn residue (37); removal substantial electronic per-      10 kDa net molecular weight cutoff (NMWCO) centrifugal filters (Millipore). <br /> turbation acetamido group, Asn-distal       Total protein concentration measured colorimetric DC protein <br />                                                                         assay (BioRad). Glycoprotein samples labeled Staudinger ligation <br /> GlcNAc, effectively blocks OT activity (38). While azido al- <br />                                                                         (21) 12 h, room temperature incubation 500 &#956;M phos-FLAG. <br /> kynyl additions C-2 acetamido group obviously increase           FLAG-conjugated proteins detected &#945;-FLAG M2-peroxidase anti- <br /> size, believe modifications interfere      body (Sigma-Aldrich). Peroxidase-conjugated antibodies detected by <br /> oxonium stabilization transfer. Modifications C-2         chemiluminescence using SuperSignal West Pico substrate (Pierce). <br /> acetamido group Asn-proximal GlcNAc impact <br /> the activity Alg13&#8725;14 enzyme complex transfers         Protein Expression Purification. Procedures used protein expression, <br /> Asn-distal GlcNAc Asn-proximal GlcNAc. Specific                purification enzymatic deglycosylation described SI Text. <br />  <br />  <br /> 3992 &#8739;   www.pnas.org/cgi/doi/10.1073/pnas.0911247107                                                                                 Breidenbach et al. <br />  Chemical Synthesis. Syntheses GlcNAz GlcNAl described                          ACKNOWLEDGMENTS. We wish extend gratitude Dr. T. Starr, <br /> SI Text.                                                                                     J. Baskin, S. Hubbard, Dr. J. Seeliger, Dr. M. Boyce technical assistance, <br />                                                                                              reagents, helpful discussion. This work supported National Insti- <br /> Mass Spectrometry. Highly purified Ygp1 treated endoH (New Eng-                     tutes Health Grant GM066047 (to C.R.B.) . J.E.G.G supported NSF <br /> land Biolabs) deglycosylated products separated SDS-PAGE.                    postdoctoral fellowship DBI-0511799. <br /> Desalted, in-gel tryptic digests Ygp1 samples analyzed ESI-FTICR <br /> MS (Bruker, 9.4 T magnet). <br />  <br />  1. Link AJ, Mock ML, Tirrell DA (2003) Non-canonical amino acids protein engineering.    23. Chang PV, et al. (2009) Metabolic labeling sialic acids living animals alkynyl <br />     Curr Opin Biotechnol, 14:603&#8211;609.                                                            sugars. Angew Chem Int Ed Engl, 48:4030&#8211;4033. <br />  2. Wang L, Schultz PG (2004) Expanding genetic code. Angew Chem Int Ed Engl,            24. Milewski S, Gabriel I, Olchowy J (2006) Enzymes UDP-GlcNAc biosynthesis yeast. <br />     44:34&#8211;66.                                                                                    Yeast, 23:1&#8211;14. <br />  3. Campbell CT, Sampathkumar SG, Yarema KJ (2007) Metabolic oligosaccharide engi-           25. Dummitt B, Micka WS, Chang YH (2005) Yeast glutamine-fructose-6-phosphate <br />     neering: Perspectives, applications, future directions. Mol Biosyst, 3:187&#8211;194.          aminotransferase (Gfa1) requires methionine aminopeptidase activity proper <br />  4. Laughlin ST, Baskin JM, Amacher SL, Bertozzi CR (2008) In vivo imaging membrane-          function. J Biol Chem, 280:14356&#8211;14360. <br />     associated glycans developing zebrafish. Science, 320:664&#8211;667.                        26. Yamada-Okabe T, Sakamori Y, Mio T, Yamada-Okabe H (2001) Identification char- <br />  5. Nandi A, et al. (2006) Global identification O-GlcNAc-modified proteins. Anal Chem,       acterization genes N-acetylglucosamine kinase N-acetylglucosamine- <br />     78:452&#8211;458.                                                                                  phosphate deacetylase pathogenic fungus Candida albicans. Eur J Biochem, <br />  6. Hanson SR, et al. (2007) Tailored glycoproteomics glycan site mapping using              268:2498&#8211;2505. <br />     saccharide-selective bioorthogonal probes. J Am Chem Soc, 129:7266&#8211;7267.                 27. Saxon E, et al. (2002) Investigating cellular metabolism synthetic azidosugars with <br />  7. Luchansky SJ, Argade S, Hayes BK, Bertozzi CR (2004) Metabolic functionalization          Staudinger ligation. J Am Chem Soc, 124:14893&#8211;14902. <br />     recombinant glycoproteins. Biochemistry, 43:12358&#8211;12366.                                 28. Vocadlo DJ, Hang HC, Kim EJ, Hanover JA, Bertozzi CR (2003) A chemical approach <br />  8. Holden HM, Rayment I, Thoden JB (2003) Structure function enzymes              identifying O-GlcNAc-modified proteins cells. Proc Natl Acad Sci USA, <br />     Leloir pathway galactose metabolism. J Biol Chem, 278:43885&#8211;43888.                       100:9116&#8211;9121. <br />  9. Tanner ME (2005) The enzymes sialic acid biosynthesis. Bioorg Chem, 33:216&#8211;228.       29. Mumberg D, Muller R, Funk M (1995) Yeast vectors controlled expression of <br /> 10. Weerapana E, Imperiali B (2006) Asparagine-linked protein glycosylation: From eukar-         heterologous proteins different genetic backgrounds. Gene, 156:119&#8211;122. <br />     yotic prok <br /> </body></html>