<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Biology Cell <br /> Vol. 19, 2708 &#8211;2717, July 2008 <br />  <br />  <br /> Analysis Yeast Kinome Reveals Network of <br /> Regulated Protein Localization Filamentous Growth <br /> Nike&#168; Bharucha, Jun Ma, Craig J. Dobry, Sarah K. Lawson, Zhifen Yang, <br /> and Anuj Kumar <br />  <br /> Department Molecular, Cellular, Developmental Biology, Life Sciences Institute, University of <br /> Michigan, Ann Arbor, MI 48109-2216 <br />  <br /> Submitted December 1, 2007; Revised March 10, 2008; Accepted April 9, 2008 <br /> Monitoring Editor: Charles Boone <br />  <br />  <br />  <br /> The subcellular distribution kinases signaling proteins regulated response cellular cues; however, the <br /> extent regulation investigated gene set organism. Here, present systematic analysis <br /> of protein kinases budding yeast, screening differential localization filamentous growth. Filamentous <br /> growth important stress response involving mitogen-activated protein kinase cAMP-dependent protein kinase <br /> signaling modules, yeast cells form interconnected elongated chains. Because standard strains yeast are <br /> nonfilamentous, constructed unique set 125 kinase-yellow fluorescent protein chimeras filamentous &#9018;1278b <br /> strain study. In total, identified cytoplasmic kinases (Bcy1p, Fus3p, Ksp1p, Kss1p, Sks1p, Tpk2p) that <br /> localize predominantly nucleus filamentous growth. These kinases form interdependent, <br /> localization-based regulatory network: deletion individual kinase, loss kinase activity, disrupts nuclear <br /> translocation kinases. In particular, study highlights previously unknown function the <br /> kinase Ksp1p, indicating essentiality nuclear translocation yeast filamentous growth. Thus, local- <br /> ization Ksp1p kinases identified tightly controlled filamentous growth, representing an <br /> overlooked regulatory component stress response. <br />  <br />  <br /> INTRODUCTION                                                        phosphate starvation (O&#8217;Neill et al., 1996). Components of <br />                                                                     yeast Slt2p mitogen-activated protein kinase (MAPK) <br /> In eukaryotes, protein function regulated mech-          cell wall integrity pathway localize sites polarized <br /> anisms controlling transcription, translation, post-transla-        growth response mating factor (Mazzoni et al., 1993; <br /> tional modification, protein degradation, subcellular lo-       Buehrer Errede, 1997). Collectively, and <br /> calization. In recent years, global studies, systematic studies,    studies (Shimada et al., 2000), infer sizable <br /> or used consider majority            protein complement regulated differential local- <br /> regulatory mechanisms wide set genes pro-           ization yeast, protein kinases constituting group <br /> teins. DNA microarray technologies (DeRisi et al., 1997;            particularly subject regulation. <br /> Gasch et al., 2001) mass spectrometry-based approaches             Protein kinases play prominent role develop- <br /> (Gygi et al., 1999; Tang et al., 2005; Roth et al., 2006)      mental processes, filamentous differentiation yeast <br /> cataloged genome-wide changes transcriptional levels,            provides strong example. In certain strains yeast (e.g., <br /> protein abundance, posttranslational modifications;             &#9018;1278b), MAPK cAMP-dependent protein kinase (PKA) <br /> however, understanding regulated protein localiza-           pathways mediate stress-induced transition multicel- <br /> tion remains cursory, constructed piecemeal individual         lular, filamentous growth form (Gimeno Fink, 1994; <br /> reports given protein function regulated       Kron, 1997; <span id='am-85' about='obo:NCBITaxon_109218' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_163737 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Cullen" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasRelatedSynonym" content="Cullen Medik." datatype="xsd:string"></span><span id='am-90'  ></span>Cullen</span> Sprague, 2000; Erdman Snyder, <br /> localization.                                                       2001). Specifically, nitrogen stress growth presence <br />    Protein localization investigated intensely        short chain alcohols initiates developmental program <br /> in <span id='am-30' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-31' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34'  ></span><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span>. cerevisiae (Kumar et al., 2002; Huh et al.,       characterized formation filamentous chains cells, <br /> 2003), reports regulated protein localization           called pseudohyphae (Lorenz et al., 2000). During filamen- <br /> surfaced frequently yeast-based studies. For example,            tous growth, yeast cells delay G2/M, exhibit elongated <br /> several yeast proteins, G1 cyclins Cln2p            morphology, display altered budding pattern, remain <br /> Cln3p, regulated differential compartmentalization           physically attached cytokinesis, invade their <br /> during cell cycle progression (Edgington Futcher, 2001).        growth substrate (Gimeno et al., 1992; Kron, 1997; Madhani <br /> The transcription factor Pho4p, involved phosphate me-           Fink, 1998). These morphological genetic changes <br /> tabolism, predominantly cytoplasmic conditions          brought signaling pathways encompass- <br /> phosphate sufficiency, localizes nucleus       ing Kss1p MAPK cascade (Ste11p, Ste7p, Kss1p) and <br />                                                                     PKA (Liu et al., 1993; <span id='am-42' about='obo:NCBITaxon_9596' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-43' property="oboInOwl:hasAlternativeId" content="NCBITaxon:317028" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasAlternativeId" content="NCBITaxon:445518" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-46'  ></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="chimpanzees" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Pan" datatype="xsd:string"></span>Pan</span> et al., 2000). In yeast, PKA consists <br />                                                                     regulatory subunit Bcy1p catalytic subunit <br /> <span id='am-50' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-54'  ></span>This</span> article published online ahead print MBC Press    isoforms Tpk1p, Tpk2p, Tpk3p (Robertson Fink, <br /> (http://www.molbiolcell.org/cgi/doi/10.1091/mbc.E07&#8211;11&#8211;1199)        1998). As yeast cells undergo filamentous growth, these <br /> on April 16, 2008.                                                  kinase-based signaling modules function additional <br /> Address correspondence to: Anuj Kumar (anujk@umich.edu).            genes pathways governing cell polarity, bud site selec- <br />  <br /> 2708                                                                                &#169; 2008 The American Society Cell Biology <br />                                                                                                                       Regulated Localization Yeast Kinases <br />  <br />  <br /> tion, cell cycle progression (Chandarlapaty Errede,                         remaining yeast kinase genes (MEC1, HRK1, RIM15, TEL1, TOR1, TOR2) <br /> 1998; Madhani et al., 1999; Miled et al., 2001). Extensive                          proved difficult clone, instead chromosomally tagged at <br />                                                                                     3-ends vYFP filamentous strain Y825/6; genes HRK1, <br /> regulatory mechanisms place coordinate signaling                          RIM15, TEL1, TOR1 chromosomally tagged Y825 well. <br /> pathways filamentous growth (Gimeno Fink,                                Integrated alleles generated standard protocols using vYFP- <br /> 1994; Borneman et al., 2006), subcellular distribution                      KanMX6 cassette pBS7 (Yeast Resource Center, University Washington). <br /> of yeast kinases likely controlled regulation. <br />    To consider degree protein localization                          Live Cell Microscopy <br /> regulated eukaryotic cell growth, and, specifically,                         Cells grown overnight, diluted OD600 roughly 0.1 grown in <br /> the extent contributes filamentous                             standard medium inducing conditions required. Filamentous growth in <br /> growth response, screened protein kinases                             haploid strains induced inoculating diluted cultures standard <br /> budding yeast differential localization filamen-                         growth medium supplemented 1% butanol 4 h 30&#176;C. Diploid <br />                                                                                     filamentous strains induced follows: overnight cultures centri- <br /> tous growth. This analysis revealed kinases localized                           fuged, washed, inoculated OD600 0.1 low-nitrogen medium <br /> evenly cell vegetative growth localized                       30&#176;C 4 h observation. Osmotic stress heat stress were <br /> predominantly nucleus conditions filamen-                           induced growing diluted overnight cultures standard growth medium <br /> tous growth. Through localization-based epistasis studies,                          mid-log phase, followed 20 min growth medium containing 0.4 <br />                                                                                     M NaCl (Ferrigno et al., 1998) 20 min growth 37&#176;C (Ferrigno et al., 1998). <br /> we kinases form interdependent                            4,6-Diamidino-2-phenylindole (DAPI) added final concentration 2 <br /> network regulated protein localization&#8212;the                            &#65533;?&#65533;g/ml 30 min stain DNA, marking nucleus mitochondria. <br /> network identified eukaryote. Our results indicate a <br /> &#8220;regulatory/subordinate&#65533;? relationship kinases                          Phenotypic Assays Filamentous Growth Defects <br /> this subnetwork; using deletion mutants kinase-dead <br />                                                                                     Colony morphology deletion mutants observed streaking mid-log <br /> alleles, kinase translocation, cases,                          cultures grown YPD SLAD plates supplemented 1% (vol/vol) <br /> requires presence/activity kinase. In addition,                      butanol (Lorenz et al., 2000) incubating 30&#176;C 10 d. Invasive growth <br /> this study implicates functionally uncharacterized Ser/                         determined standard plate-washing assay Gimeno et al. (1992). <br /> Thr kinase Ksp1p filamentous growth, highlighting,                         For assay, mid-log phase cultures spotted YPD plates and <br />                                                                                     incubated 5 d 30&#176;C; surface cells washed gentle <br /> in broader terms, need consider similar regulatory                           stream water. <br /> mechanisms eukaryotes. <br />  <br /> MATERIALS AND METHODS                                                               RESULTS <br />  <br /> Yeast Strains Growth Conditions                                                 Localization Yeast Kinases Filamentous Growth <br /> The filamentous yeast strains Y825 Y825/6 derived &#9018;1278b               The yeast proteome encompasses 125 protein kinases, de- <br /> (Gimeno et al., 1992). The genotype haploid Y825 MATa ura3-52 leu2&#9004;0;         fined <span id='am-75' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> sets deposited Saccharomyces Genome <br /> the genotype diploid Y825/6 ura3-52/ura3-52 leu2&#9004;0/leu2&#9004;0. The nonfila-       Database August 2006 (www.yeastgenome.org). For <br /> mentous strain Y2269 S288c genetic background, derivative     purposes analysis, include regulatory subunit <br /> of strain BY4743 (Giaever et al., 1999). Deletion mutants constructed in <br /> strain Y825 using one-step polymerase chain reaction (PCR)-based gene          protein kinase A, Bcy1p. The yeast kinase complement, or <br /> disruption strategy (Baudin et al., 1993) G418 resistance cassette    kinome, listed Supplemental Figure SF1, with <br /> plasmid pFA6a-KanMX6 (Longtine et al., 1998). The sks1-K39R kinase-dead             relevant functions protein localization data describing <br /> allele generated site-directed mutagenesis using oligonucleotides         subcellular distribution protein nonfilamen- <br /> described Yang Bisson (1996). Kinase-dead alleles obtained or <br /> generated low-copy <span id='am-80' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-81' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span>, ksp1-K47D allele, carried       tous lab strains (e.g., S288c). To investigate subcellular <br /> on pYEPs29K47D, described Fleischmann et al. (1996).                             dynamics kinases filamentous growth, we <br />    Standard growth medium microscopy prepared using 0.17% yeast             constructed unique plasmid-based collection kinase- <br /> nitrogen base (YNB) amino acids <span id='am-65' about='obo:NCBITaxon_29189' typeof='obo:NCBITaxon_29178 owl:Thing obo:NCBITaxon_203397 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_69034 obo:NCBITaxon_29185 obo:NCBITaxon_543769 obo:NCBITaxon_2759'><span id='am-66'  ></span><span id='am-67' property="rdfs:label" content="Ammonia" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>ammonia</span>, 2% glucose,                fluorescent protein fusions. This plasmid-based approach is <br /> 5 mM ammonium sulfate (Guthrie Fink, 1991). Haploid filamentous <br /> growth induced standard medium supplemented 1% (vol/               suited localization studies nonstandard genetic <br /> vol) butanol (Lorenz et al., 2000) SLAD plates (0.17% yeast nitrogen base     backgrounds, filamentous strain &#9018;1278b; the <br /> without amino acids ammonium sulfate, 2% glucose, 50 &#65533;?&#65533;M ammonium                &#9018;1278b genetic background preferred studies fila- <br /> sulfate, essential amino acids) plus 1% (vol/vol) butanol (Gimeno et al.,      mentous growth (Gimeno et al., 1992), existing reagents <br /> 1992; Lorenz et al., 2000). Diploid filamentous growth induced liquid <br /> low-nitrogen medium (0.17% YNB amino acids ammonia, 2%                  protein localization studies available this <br /> glucose, 50 &#65533;?&#65533;M ammonium sulfate) (Gimeno et al., 1992). Invasive growth         strain. As study, designed centromeric yeast <br /> was assayed YPD medium synthetic complete (SC) &#11002;Ura indicated              shuttle vector, pDEST-vYFP, expression cloned <br /> previously (Guthrie Fink, 1991; Lorenz et al., 2000).                           genes chimeric proteins vYFP fused the <br />                                                                                     carboxy terminus target protein (Nagai et al., 2002). <br /> Construction Yeast Kinase-Fluorescent Protein Fusions <br />                                                                                     The pDEST-vYFP vector carries phage lambda att sites for <br /> In study, constructed Gateway-compatible (Invitrogen, Carlsbad, <br /> CA) yeast vector recombination-based cloning promoter-gene cas- <br />                                                                                     recombination-based cloning, accommodating gene coding <br /> settes carboxy-terminal fusions fluorescent protein (Walhout et al., 2000).   sequences 1 kb upstream promoter sequence. <br /> This Gateway vector constructed centromeric yeast shuttle              By virtue vector, genes expressed control of <br /> vector YCp50 (Rose et al., 1987). To construct pDEST-vYFP, <span id='am-60' about='obo:NCBITaxon_55714' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6580 obo:NCBITaxon_6592 obo:NCBITaxon_6544 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_6599 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_74489 obo:NCBITaxon_6072'><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62'  ></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="Venus" datatype="xsd:string"></span>Venus</span> variant        native promoters nearly endogenous levels. <br /> of yellow fluorescent protein (vYFP) (Nagai et al., 2002) amplified PCR <br /> from pBS7 (Yeast Resource Center, University Washington, Seattle, WA) <br />                                                                                        In total, cloned 119 kinase genes pDEST-vYFP and <br /> for introduction SphI-SalI site YCp50. Subsequently, YCp50          subsequently introduced kinase-vYFP fusion the <br /> vector carrying vYFP digested SphI blunt T4 DNA              filamentous strain &#9018;1278b approach outlined Fig- <br /> Polymerase (New England Biolabs, Ipswich, MA). Gateway cassette A (In-              ure 1. For kinases (MEC1, HRK1, RIM15, TEL1, TOR1, <br /> vitrogen) ligated blunt-ended vector, EcoRI used to <br /> identify orientation cassette. <br />                                                                                     TOR2), C-terminal fusions vYFP generated as <br />    In total, cloned 119 kinase genes pDEST-vYFP. Briefly, ampli-         integrated alleles &#9018;1278b (described Materials Meth- <br /> fied PCR open reading frame 1 kb upstream genomic             ods). The kinase-vYFP chimeras screened fluores- <br /> DNA introduction pDEST-vYFP; PCR primers contained phage                   cence microscopy differential localization vegeta- <br /> lambda att sites, allowing recombination att-containing destina- <br /> tion vector according protocols described Gelperin et al. (2005). On <br />                                                                                     tive filamentous growth conditions haploid and <br /> cloning, PCR product creates translational fusion 3&#11032; end       diploid strains &#9018;1278b, filamentous growth <br /> the gene vYFP, 10-<span id='am-55' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57'  ></span><span id='am-58' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> linker encoding NPAFLYKVVI. The              response differs according ploidy (Gancedo, 2001). A full <br />  <br /> Vol. 19, July 2008                                                                                                                                               2709 <br />  N. Bharucha et al. <br />  <br />  <br />                                                                          inantly nucleus filamentous growth. In each <br />                                                                          case, nuclear shift striking, &#11011;80 &#8211;95% ob- <br />                                                                          served fluorescence concentrated nucleus. The <br />                                                                          differential localization kinases evident both <br />                                                                          haploid cells (shown Figure 2) diploid cells Bcy1p, <br />                                                                          Fus3p, Kss1p, Sks1p, Tpk2p; however, Ksp1p did not <br />                                                                          localize strongly nucleus filamentous growth <br />                                                                          diploid yeast. <br />                                                                             As mentioned previously, Bcy1p, Tpk2p, Kss1p are <br />                                                                          known components filamentous growth PKA MAPK <br />                                                                          cascades, respectively. Fus3p MAPK mediating the <br />                                                                          yeast mating response (Elion et al., 1993; Choi et al., 1999), <br />                                                                          Sks1p serine/threonine kinase involved cel- <br />                                                                          lular response glucose limitation (Vagnoli Bisson, <br />                                                                          1998). The cellular function Ksp1p unknown, although <br />                                                                          overexpression known suppress mutations SRM1, <br />                                                                          nucleotide exchange factor required nucleocytoplasmic <br />                                                                          trafficking macromolecules (Fleischmann et al., 1996). <br />                                                                             To determine observed localization shifts are <br />                                                                          specific yeast filamentous growth response, further <br />                                                                          screened yeast kinases identified differen- <br />                                                                          tial localization osmotic stress (in 0.4 M NaCl) and <br />                                                                          heat stress &#9018;1278b. Bcy1p known localize differen- <br />                                                                          tially heat stress (Griffioen et al., 2001); however, the <br />                                                                          kinase-vYFP fusions distributed evenly the <br />                                                                          cytoplasm nucleus conditions tested. Thus, <br />                                                                          observed nuclear shifts represent general stress <br />                                                                          responses, likely specific conditions inducing <br />                                                                          filamentous growth. <br />  <br />  <br />                                                                          Phenotypic Analysis Nuclear-localized Kinases <br />                                                                          To clarify potential roles kinases filamen- <br />                                                                          tous growth, deleted corresponding kinase gene in <br />                                                                          haploid derivative &#9018;1278b phenotypic analysis of <br />                                                                          filamentation (Figure 3). Deletion mutants screened for <br />                                                                          colony cell morphologies, invasive growth was <br />                                                                          assayed standard plate-washing assay Gimeno <br />                                                                          et al. (1992). Filamentous growth phenotypes consistent <br />                                                                          assays. Hyperfilamentous growth evidenced <br />                                                                          extended peripheral filamentation, elongated cell mor- <br />                                                                          phology, increased invasive growth relative wild <br />                                                                          type; converse phenotypes present mutants defec- <br />                                                                          tive filamentous growth. In assays, deletion KSP1, <br />                                                                          KSS1, TPK2 resulted decreased filamentous growth, <br />                                                                          fus3&#9004; mutant hyperfilamentous. Deletion of <br />                                                                          BCY1 SKS1 did affect filamentous growth. These phe- <br />                                                                          notypic results consistent understanding of <br />                                                                          filamentous growth functions associated Bcy1p, Fus3p, <br /> Figure 1. Schematic overview construction subsequent          Kss1p, Tpk2p; however, Ksp1p implicated <br /> screening carboxy-terminal YFP fusions. Primers PCR ampli-        previously filamentous growth, and, context, the <br /> fication designed 5&#11032;-att sites resulting PCR         strong effect Ksp1p filamentous growth particularly <br /> products cloned pDEST-vYFP vector phage             noteworthy. <br /> &#65533;?&#65533;-based recombination. The resulting gene fusion carries 10-codon         The kinase shifts reported identified using pro- <br /> linker target open reading frame YFP. Subsequent         teins tagged carboxy termini vYFP; however, <br /> screening steps indicated, diagrammatic representa-        carboxy-terminal protein modifications can, cases, <br /> tion protein differentially localized nucleus condi- <br /> tions filamentous growth. <br />                                                                          disrupt function. To consider functionality ki- <br />                                                                          nase-vYFP chimeras, assessed ability Fus3p-vYFP, <br />                                                                          Ksp1p-vYFP, Kss1p-vYFP, Tpk2p-vYFP complement <br /> listing protein kinase localizations conditions           corresponding deletion mutants filamentous growth <br /> is presented Supplemental Table ST1.                                  phenotypes. In case, introduction kinase-vYFP <br />    From analysis, kinases altered                 fusion restored wild-type filamentous growth. The Bcy1p- <br /> subcellular distribution filamentous growth condi-                 vYFP chimera complemented heat-sensitive phenotype <br /> tions (Figure 2). The proteins Bcy1p, Fus3p, Ksp1p, Kss1p,               bcy1&#9004; mutant, assayed method described in <br /> Sks1p, Tpk2p distributed evenly cell                  Toda et al. (1987). The functionality Sks1p-vYFP not <br /> conditions vegetative growth, localized predom-              considered, null phenotype easily assayed. <br />  <br /> 2710                                                                                                        Molecular Biology Cell <br />                                                           Regulated Localization Yeast Kinases <br />  <br />  <br />  <br />  <br /> Figure 2. Differential localization protein ki- <br /> nases Bcy1p, Fus3p, Ksp1p, Kss1p, Sks1p, Tpk2p. <br /> Kinase-vYFP fusions visualized fluores- <br /> cence microscopy (left) vegetative growth <br /> conditions filamentous growth. Yeast <br /> cells stained DNA-binding dye DAPI <br /> (middle) visualize nucleus mitochondria. <br /> The yeast cell shape vacuoles imaged by <br /> differential interference contrast (DIC) microscopy <br /> (right). <br />  <br />  <br />  <br />  <br /> Figure 3. Phenotypic analysis kinase deletion <br /> mutants filamentous &#9018;1278b genetic back- <br /> ground. Each haploid deletion mutant assayed <br /> for surface-spread filamentation (A), cell morphol- <br /> ogy (B), invasive growth (C). Surface-spread <br /> filamentation assayed SLAD medium (see <br /> Materials Methods) supplemented 1% buta- <br /> nol. Cells colonies inoculated a <br /> small volume water DIC microscopy. Invasive <br /> growth assayed YPD medium described <br /> in Materials Methods. The &#9018;1278b background <br /> strain Y825 served wild-type control these <br /> filamentous growth assays. <br />  <br /> Vol. 19, July 2008                                                                         2711 <br />  N. Bharucha et al. <br />  <br />  <br /> Interdependent Kinase Translocation Nucleus                presence Ksp1p, Kss1p, Sks1p. The predominantly <br /> during Filamentous Growth                                         nuclear localization Ksp1p filamentous growth lost <br /> The regulated localization Bcy1p, Fus3p, Ksp1p, Kss1p,         strains deleted BCY1, FUS3, SKS1, TPK2. The deletion <br /> Sks1p, Tpk2p filamentous growth                kinase subset disrupts nuclear shift Sks1p <br /> known interplay kinases signaling pathways led         filamentous growth conditions. In contrast, localiza- <br /> us consider possibility observed nuclear trans-   tion Kss1p Tpk2p filamentous growth unaf- <br /> location kinases interdependent: pres-        fected deletion kinases. <br /> ence kinase required localization shift        Many kinases localized subgrouping exhibit recip- <br /> of another. To address explicitly, systematically ex-     rocal relationships (Figure 4C). The deletion BCY1 affects <br /> amined subcellular localization kinases       butanol-induced nuclear localization Sks1p, and, re- <br /> mutant backgrounds individually deleted       ciprocally, deletion SKS1 disrupts nuclear shift Bcy1p. <br /> other kinases. For example, subcellular distribution       We observed similar effects kinase pairs Bcy1p-Ksp1p, <br /> Bcy1p examined conditions vegetative             Fus3p-Ksp1p, Ksp1p-Sks1p. Other kinase pairs, however, <br /> filamentous growth haploid mutant backgrounds:            exhibit unidirectional &#8220;regulatory/subordinate&#65533;? relation- <br /> fus3&#9004;, ksp1&#9004;, kss1&#9004;, sks1&#9004;, tpk2&#9004;. The remaining         ship; example, presence Fus3p required the <br /> kinases examined accordingly, results           nuclear shift Sks1p, SKS1 required wild-type <br /> from study presented Figure 4.                        localization Fus3p conditions filamentous <br />    Interestingly, nuclear shift observed Bcy1p, Fus3p,    growth. Similarly, Tpk2p contributes regulated local- <br /> Ksp1p, Sks1p dependent case pres-         ization Ksp1p Sks1p, reciprocal relationships <br /> ence kinase (Figure 4, A B). Fus3p      evident. Kss1p contributes butanol-in- <br /> is distributed evenly cytoplasm nucleus un-        duced nuclear shift Bcy1p Sks1p, neither <br /> der conditions inducing filamentous growth haploid           gene product affects localization Kss1p fila- <br /> strain deleted KSP1. The nuclear shift Bcy1p requires      mentous growth. <br />  <br />  <br />  <br />  <br />                                                                                       Figure 4. Interdependent localization yeast <br />                                                                                       kinases filamentous growth. (A) Each <br />                                                                                       kinase-vYFP fusion visualized fluores- <br />                                                                                       cence microscopy filamentous growth <br />                                                                                       conditions haploid strain &#9018;1278b deleted <br />                                                                                       indicated kinase. DAPI-stained DIC <br />                                                                                       images provided. Images kinase- <br />                                                                                       vYFP fusions vegetative growth condi- <br />                                                                                       tions presented Supplemental Figure <br />                                                                                       SF2. The mixed distribution kinases <br />                                                                                       cytoplasm nucleus signifies loss <br />                                                                                       nuclear localization filamentous <br />                                                                                       growth conditions deletion indi- <br />                                                                                       cated kinase gene. Only kinases exhibiting a <br />                                                                                       loss nuclear localization shown here. (B) <br />                                                                                       Matrix kinase localizations gene deletion <br />                                                                                       backgrounds conditions filamentous <br />                                                                                       growth. The results corresponding im- <br />                                                                                       ages shown A boxed red. Images <br />                                                                                       nuclear-localized kinase-vYFP fusions unaf- <br />                                                                                       fected indicated gene deletions pre- <br />                                                                                       sented Supplemental Figure SF3. (C) Local- <br />                                                                                       ization-based regulatory relationships among <br />                                                                                       yeast kinases. Each forward pointed arrow in- <br />                                                                                       dicates given kinase required the <br />                                                                                       wild-type localization subordinate ki- <br />                                                                                       nase; example, Fus3p required the <br />                                                                                       wild-type nuclear localization Sks1p during <br />                                                                                       filamentous growth. Reciprocal relationships <br />                                                                                       kinases indicated circular <br />                                                                                       arrows left column. (D) The network of <br />                                                                                       regulated protein localization Bcy1p, <br />                                                                                       Fus3p, Ksp1p, Kss1p, Sks1p, Tpk2p il- <br />                                                                                       lustrated here; arrows drawn described <br />                                                                                       C. Double-sided arrows indicate the <br />                                                                                       localization given kinase pair recip- <br />                                                                                       rocally affected filamentous growth <br />                                                                                       conditions. <br />  <br /> 2712                                                                                                   Molecular Biology Cell <br />                                                                                                 Regulated Localization Yeast Kinases <br />  <br />  <br />    Taken collectively, kinases identified form        localization kinases dependent protein nu- <br /> part interdependent network regulated protein lo-         clear translocation filamentous growth (Figure 5). <br /> calization filamentous growth (Figure 4D). It likely     For example, localization Ksp1p Sks1p ob- <br /> that additional proteins contribute regulated localiza-     served kinase-dead fus3-K42R mutant; localization <br /> tion kinases; thus, relationships indicated        Bcy1p, Fus3p, Sks1p assessed ksp1-K47D <br /> Figure 4 represent portion potentially larger network.      mutant, on. In case, kinase activity required <br /> In particular, results highlight key regulatory roles   observed localization shift. As shown Figure 5, <br /> Kss1p Tpk2p; proteins required wild-          Ksp1p-vYFP Sks1p-vYFP evenly distributed <br /> type localization members subnetwork (i.e., Bcy1p,      nucleus cytoplasm strain bearing the <br /> Ksp1p, Sks1p), affected presence of <br /> any kinase tested here. Ksp1p plays central role in <br /> this network, affecting localization Bcy1p, Fus3p, and <br /> Sks1p, itself requiring presence Bcy1p, Fus3p, <br /> Sks1p, Tpk2p wild-type localization condi- <br /> tions filamentous growth. This particularly interesting <br /> because Ksp1p implicated previously fila- <br /> mentous growth. Downstream subordinate roles also <br /> evident subnetwork; particular, other <br /> kinases required nuclear localization Sks1p, <br /> whereas presence required wild-type lo- <br /> calization Bcy1p Ksp1p. <br />    This localization-based interdependence suggests regula- <br /> tory relationships investigated tradi- <br /> tional epistasis studies double-deletion mutants. For <br /> example, outlined above, localization Ksp1p is <br /> affected deletion BCY1, FUS3, SKS1, TPK2; thus, <br /> we deleted genes individually strain <br /> deleted KSP1 assess impact mutation on <br /> the ksp1&#9004; filamentous growth phenotype. The results from <br /> this analysis shown Supplemental Figure SF4. The <br /> ksp1&#9004;fus3&#9004; mutant exhibited exaggerated filamentous <br /> growth, assessed examination colony morphology, <br /> cell morphology, invasive growth; mimics phe- <br /> notype fus3&#9004;. As expected, ksp1&#9004;tpk2&#9004; double mutant <br /> shows filamentous growth; however, deletion BCY1 <br /> and SKS1, respectively, restores yeast filamentous growth in <br /> a ksp1&#9004; genetic background. At minimum, indicates a <br /> filamentous growth effect associated sks1&#9004;. It inter- <br /> esting ksp1&#9004; phenotype masked phenotype <br /> associated gene deletion double mu- <br /> tant. From this, Ksp1p serve upstream role fila- <br /> mentous growth pathways; however, difficult to <br /> interpret results, clear evidence <br /> that Ksp1p actually functions linear pathway the <br /> PKA modules, MAPK modules, both. <br />  <br /> Kinase Activity Is Required Nuclear Translocation <br /> of Dependent Proteins <br /> Kinases serve structural catalytic functions sig- <br /> naling pathways, phosphorylating target proteins also, <br /> in cases, acting scaffolds facilitating protein&#8211;protein <br /> interactions (Choi et al., 1994; Madhani Fink, 1997). To <br /> specifically determine role kinase phosphorylation in <br /> controlling subordinate protein translocation nucleus, <br /> we examined subcellular distribution Bcy1p, Fus3p, <br /> Ksp1p, Kss1p, Sks1p, Tpk2p kinase-dead mutants of <br /> Fus3p (fus3-K42R), Ksp1p (ksp1-K47D), Kss1p (kss1-K42R),           Figure 5. The observed nuclear translocations dependent upon <br /> Sks1p (sks1-K39R), Tpk2p (tpk2-K99R) (Fleischmann et           kinase activity Fus3p, Ksp1p, Kss1p, Sks1p, Tpk2p, <br /> al., 1996; Yang Bisson, 1996; Madhani et al., 1997; Dem-       respectively. Each indicated kinase-vYFP fusion imaged a <br /> low Fox, 2003; Zeitlinger et al., 2003); Bcy1p omitted     strain carrying kinase-dead allele listed left. For example, <br /> from study does possess kinase domain.       subcellular distribution Ksp1p Sks1p shown a <br /> As indicated Supplemental Figure SF5, kinase-dead          haploid fus3&#9004; strain carrying fus3-K42R kinase-dead allele a <br />                                                                    low-copy plasmid. DAPI-stained DIC images described <br /> mutant exhibits filamentous growth phenotypes mirroring            previously. Note mixed distribution kinase the <br /> those observed gene deletion.                                 cytoplasm nucleus loss indicated kinase activity. <br />    In analysis, individually introduced kinase-       Images kinases kinase-dead background strains <br /> dead allele haploid &#9018;1278b strain deleted           vegetative growth conditions presented Supplemental <br /> corresponding kinase gene analyzed subcellular             Figure SF6. Bar, 3 &#65533;?&#65533;m. <br />  <br /> Vol. 19, July 2008                                                                                                                   2713 <br />  N. Bharucha et al. <br />  <br />  <br /> fus3-K42R allele conditions butanol-induced fila- <br /> mentous growth. Similar results observed full <br /> panel mutants tested. Thus, kinase activity the <br /> identified proteins required nuclear translocation <br /> of dependent kinases. It noted required <br /> phosphorylation events indirect. Also, observed pro- <br /> tein localization affected kinase-dead mutant <br /> under conditions vegetative growth (Supplemental Fig- <br /> ure SF6). <br />  <br /> Nuclear Translocation Ksp1p Is Required Wild-Type <br /> Filamentous Growth <br /> As evidenced null phenotype, Ksp1p required for <br /> butanol-induced filamentous growth haploid yeast; how- <br /> ever, importance observed localization shift during <br /> filamentous growth explicitly determined from <br /> this finding. To consider point directly, gener- <br /> ated modified form Ksp1p unable translocate the <br /> nucleus. Specifically, fused nuclear export sequence <br /> (<span id='am-70' about='obo:NCBITaxon_203327' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_8220 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_8219 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489878 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_497223 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-72'  ></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="Nes" datatype="xsd:string"></span>NES</span>) 3&#11032; end KSP1-vYFP, illustrated Figure <br /> 6A. The Ksp1p-vYFP-NES chimera expressed the <br /> native KSP1 promoter low-copy destination vector <br /> described Figure 1. For analysis, NES derived <br /> from Rev protein HIV-1 (Fritz et al., 1995), has <br /> been used previously direct nucleocytoplasmic trafficking <br /> in yeast (Murphy Wente, 1996). Attachment NES <br /> prevented nuclear shift Ksp1p conditions of <br /> butanol-induced filamentous growth (Figure 6A). Pheno- <br /> typic analysis NES-tagged mutant revealed signifi- <br /> cantly reduced filamentous growth compared hap- <br /> loid ksp1&#9004; strain expressing Ksp1p-vYFP chimera: <br /> surface-spread filamentation invasive growth lost <br /> in NES-tagged strain, cell morphology was <br /> rounded elongated conditions inducing           Figure 6. Tagging Ksp1p NES abolishes filamentous <br />                                                                   growth. (A) Schematic diagram NES-tagged form Ksp1p, <br /> filamentous growth (Figure 6B). The ksp1-K47D kinase-dead         NES (encoding LQLPPLERLTLD) fused carboxy <br /> mutant defective filamentous growth (Supplemen-        terminus Ksp1p-vYFP chimera. As visualized fluorescence <br /> tal Figure SF5); therefore, nuclear translocation Ksp1p    microscopy, addition NES prevents butanol-induced nu- <br /> and kinase activity required wild-type fila-     clear translocation Ksp1p. DIC vYFP images shown. Bar, <br /> mentation, suggesting Ksp1p likely phosphorylates        3 &#65533;?&#65533;m. (B) Phenotypic analysis NES-tagged Ksp1p. For compari- <br /> or nuclear proteins filamentous growth            son, included haploid ksp1&#9004; strain carrying centromeric <br /> response.                                                         plasmid KSP1-vYFP fusion transcriptional control of <br />                                                                   native KSP1 promoter; strain exhibits wild-type filamentous <br />                                                                   growth properties. Addition NES results decreased fila- <br /> DISCUSSION                                                        mentous growth, evidenced loss surface-spread filamenta- <br />                                                                   tion, rounded cell morphology, decreased invasive growth. Bar, <br /> From numerous individual examples, clear           3 &#65533;?&#65533;m. <br /> function protein controlled mechanisms <br /> regulating subcellular distribution; however, extent to <br /> which form regulation occurs investi-        carboxy-terminal tagging typically problematic analyz- <br /> gated large scale. Accordingly, undertook       ing isoprenylated gene products geranylgeranylated <br /> systematic analysis differential protein localization      proteins (Bhattacharya et al., 1995), proteins mod- <br /> any protein set eukaryote, using yeast          ified palmitoyl farnesyl groups (Sun et al., 2004; <br /> kinome subject, identified interdependent sub-       Roth et al., 2006). Although proteins cell wall, <br /> network proteins intracellular localization           endoplasmic reticulum, peroxisomes adversely <br /> tightly regulated filamentous growth. This study,          affected carboxy-terminal modification, expect the <br /> therefore, describes underappreciated type regulatory       majority yeast proteins function normally C-terminal <br /> network, yields insight degree differential     fluorescent protein fusions (Pelham et al., 1988). Plus, the <br /> protein localization serves widespread regulatory mech-      alternative approach amino-terminal tagging poses greater <br /> anism, identifies kinase Ksp1p new fila-        potential protein mislocalization. <br /> mentous growth gene.                                                 By live cell imaging kinase-YFP chimeras, found <br />    The plasmid collection described constitutes unique     Bcy1p, Fus3p, Ksp1p, Kss1p, Sks1p, Tpk2p differentially <br /> and versatile resource yeast scientific community,     localized nucleus filamentous growth condi- <br /> complement useful set chromosomally inte-            tions. These kinases characterized varying <br /> grated green fluorescent protein-fusions presented Huh         degrees, pathway context presented in <br /> et al. (2003). Obviously, however, plasmid collection      Figure 7, summary localization data. In <br /> not limitations; carboxy-terminal fusions         cases, observed localization shifts recon- <br /> perturb functions proteins, im-        ciled easily corresponding protein functions. Specifi- <br /> portant consideration using plasmids. For example,       cally, PKA phosphorylates nuclear proteins, the <br />  <br /> 2714                                                                                                   Molecular Biology Cell <br />                                                                                             Regulated Localization Yeast Kinases <br />  <br />  <br />  <br />  <br /> Figure 7. Subcellular localization yeast <br /> kinome filamentous growth. This dia- <br /> gram summarizes localization 125 pro- <br /> tein kinases, constituting yeast kinome, <br /> under conditions filamentous growth. In <br /> total, 119 kinases shift localization dur- <br /> ing transition filamentous growth, and <br /> the localization kinases tallied the <br /> top left. For simplicity, Yck3p, localizes <br /> to vacuole, included cytoplasmic <br /> compartment. The kinases shift local- <br /> ization vegetative filamentous <br /> growth conditions boxed red, the <br /> pathway context kinase shown. <br /> Flo8p Ste12p included representa- <br /> tive targets Tpk2p Kss1p, respectively. <br /> Speculative unknown interactions in- <br /> dicated question marks. <br />  <br />  <br /> filamentous growth transcription factor Flo8p (Rupp et al.,        der conditions nitrogen stress haploid yeast by <br /> 1999), filamentous growth conditions; nuclear            growth butanol. Furthermore, endeavored to <br /> shift Bcy1p Tpk2p likely enables PKA selectively         make clear growth conditions used study <br /> phosphorylate nuclear-localized targets filamenta-          text. <br /> tion. The filamentous growth MAPK Kss1p phosphorylates                As indicated data, Ksp1p represents impor- <br /> an incompletely defined set nuclear proteins, including         tant new filamentous growth gene, deletion in- <br /> the transcriptional activator Ste12p. Kss1p does shift     hibits characteristic filamentous growth landmarks in <br /> localization response mating factor (Ma et al., 1995),   haploid yeast: cell elongation, surface-spread filamentation, <br /> this filamentous growth-induced shift constitute           invasive growth. Furthermore, localization shift of <br /> mechanism ensuring Kss1p signaling specificity.                    Ksp1p required filamentous growth, kinase <br />    In regard study, noted filamentous    activity. This strongly suggests Ksp1 phosphorylates <br /> growth induced conditions nitrogen depriva-           nuclear proteins essential step yeast <br /> tion growth presence short-chain           filamentous growth response. At present, Ksp1p no <br /> alcohols, butanol. Classically, pseudohyphal growth        confirmed targets. In vitro phosphorylation studies using <br /> refers form filamentous growth induced diploid          protein microarrays identify 187 putative substrates for <br /> yeast conditions nitrogen deprivation solid me-           Ksp1p (Ptacek et al., 2005); however, putative <br /> dium, yeast strain exhibits surface-spread        substrates belong known filamentous growth pathways. <br /> filamentation invasive growth (Gimeno et al., 1992;            The nuclear shift Ksp1p requires presence BCY1, <br /> Gancedo, 2001). Haploid strains yeast undergo invasive          TPK2, FUS3, SKS1, and, minimum, kinase activity <br /> growth rich medium, exhibit extensive sur-           Fus3p. Ksp1p established target kinases, <br /> face-spread filamentation (Roberts Fink, 1994). Growth         effect Fus3p phosphorylation indirect. <br /> in butanol used induce filamentation haploid          Interestingly, Ksp1p target Hsf1p, Pho85p, and <br /> and diploid yeast, yielding morphological properties resem-        Pcl1p; hence, predicted involvement cell cycle and <br /> bling pseudohyphal growth, liquid medium (Lorenz           yeast general stress response (Dephoure et al., 2005; <br /> et al., 2000). Compared nitrogen deprivation, butanol         Hashikawa et al., 2006). Because filamentous growth co- <br /> induction involves underlying genetic differences;         ordinated cell cycle general stress response <br /> for example, Lorenz et al. (2000) report numerous up-         machinery, Ksp1p play important role signal- <br /> stream nutrient-sensing genes required classic                 ing link processes. <br /> pseudohyphal growth required butanol-induced              The nuclear shifts Fus3p Sks1p surprising. <br /> filamentous growth. We, however, identified        Fus3p phosphorylated Ste7p translocates the <br /> these genes (GPA2 GPR1) disruption screen             nucleus response mating pheromone, <span id='am-91' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-95'  ></span>phos</span>- <br /> genes essential butanol-induced haploid filamentous             phorylates Ste12p filamentous growth transcription <br /> growth (Jin et al., 2008); thus, analysis re-      factor Tec1p. In case, phosphorylation Tec1p <br /> quired understand genetic basis induction          targets degradation, inhibiting filamentous <br /> mechanisms. To consider induction schemes,            growth pathway mating (Elion et al., 1993; Choi et al., <br /> study, assayed kinase localizations diploid yeast un-        1999; Bao et al., 2004). During filamentous growth, func- <br />  <br /> Vol. 19, July 2008                                                                                                           2715 <br />  N. Bharucha et al. <br />  <br />  <br /> tion Fus3p nucleus unclear, presum-                   REFERENCES <br /> ably phosphorylate Tec1p conditions. <br />                                                                               Bao, M. Z., Schwartz, M. A., Cantin, G. T., Yates, J. R., Madhani, H. (2004). <br /> Fus3p phosphorylated Ste7p,                  Pheromone-dependent destruction Tec1 transcription factor required <br /> phosphorylate Ste12p filamentous growth. Fus3p ki-                     MAP kinase signaling specificity yeast. Cell 119, 991&#8211;1000. <br /> nase activity required nuclear translocation                    Baudin, A., Ozier-Kalogeropoulos, O., Denouel, A., Lacroute, F., Cullin, <br /> Ksp1p Sks1p filamentation, suggesting                      C. (1993). A simple efficient method direct gene deletion Saccharo- <br /> possesses previously unappreciated roles yeast fila-                   myces cerevisiae. Nucleic Acids Res. 21, 3329 &#8211;3330. <br /> mentous growth response. The role Sks1p filamentous                     Bhattacharya, S., Chen, L., Broach, J. R., Powers, S. (1995). Ras membrane <br /> growth unclear, known functions                       targeting essential glucose signaling viability yeast. Proc. <br /> apparently distinct mediating filamentous                          Natl. Acad. Sci. USA 92, 2984 &#8211;2988. <br /> growth, deletion does affect filamentation. In               Borneman, A. R., Leigh-Bell, J. A., Yu, H., Bertone, P., Gerstein, M., and <br /> context, kinase deletion phenotypes interpreted                       Snyder, M. (2006). Target hub proteins serve master regulators devel- <br />                                                                               opment yeast. Genes Dev. 20, 435&#8211; 448. <br /> with caution, kinases known engage sig- <br /> nificant cross talk compensatory activity (Madhani                    Buehrer, B. M., Errede, B. (1997). Coordination mating cell <br />                                                                               integrity mitogen-activated protein kinase pathways Saccharomyces cerevi- <br /> Fink, 1997; McClean et al., 2007). Thus, phenotype                   siae. Mol. Cell. Biol. 17, 6517&#8211; 6525. <br /> deleted kinase masked activity another <br />                                                                               Chandarlapaty, S., Errede, B. (1998). Ash1, daughter cell-specific pro- <br /> kinase normally functioning given process.                           tein, required pseudohyphal growth <span id='am-13' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-14' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-26'  ></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>. Mol. <br />    The observed localization shifts raise interesting ques-                Cell. Biol. 18, 2884 &#8211;2891. <br /> tion regarding mechanism kinases trans-                    Choi, K., Kranz, J., Mahanty, S., Park, K., Elion, E. (1999). Characterization <br /> locate nucleus. Little reported describing                  Fus3 localization: active Fus3 localizes complexes varying size and <br /> nuclear localization signals (NLSs) proteins,                    specific activity. Mol. Biol. Cell 5, 1553&#8211;1568. <br /> sequence analysis program PSORTII (Nakai Hor-                      Choi, K., Satterberg, B., Lyons, D. M., Elion, E. (1994). Ste5 tethers <br /> ton, 1999) suggests presence putative NLS                    multiple protein kinases MAPK cascade required mating S. <br /> Ksp1p. Thus, remaining kinases likely ferried                    cerevisiae. Cell 78, 499 &#8211;512. <br /> the nucleus interaction protein. For                     Cullen, P. J., Sprague, G. F. (2000). Glucose depletion causes haploid <br /> example, conditions temperature stress S288c,                     invasive growth yeast. Proc. Natl. Acad. Sci. USA 97, 13461&#8211;13463. <br /> Bcy1p translocates nucleus cytoplasm                          Demlow, C. M., Fox, T. D. (2003). Activity mitochondrially synthesized <br /> through interaction protein Zds1p (Griffioen et                   reporter proteins lower imported proteins increased by <br /> al., 2001). Similar interactions proteins allow                lowering cAMP glucose-grown Saccharomyces cerevisiae cells. Genetics 165, <br />                                                                               961&#8211;974. <br /> for filamentous growth-induced translocation ki- <br /> nase subset reported here, obvious candidates                     Dephoure, N., Howson, R. W., Blethrow, J. D., Shokat, K. M., O&#8217;Shea, <br />                                                                               E. K. (2005). Combining chemical genetics proteomics identify protein <br /> are evident interaction data sets.                                       kinase substrates. Proc. Natl. Acad. Sci. USA 102, 17940 &#8211;17945. <br />    It tempting extrapolate findings yeast <br />                                                                               DeRisi, J. L., Iyer, V. R., Brown, P. O. (1997). Exploring metabolic and <br /> proteome whole; however, expect kinome                    genetic control gene expression genomic scale. Science 278, 680 &#8211; 686. <br /> particularly subject form regulatory control and <br />                                                                               Edgington, N. P., Futcher, B. (2001). Relationship function <br /> that overall percentage yeast proteins regulated                    location G1 cyclins S. cerevisiae. J. Cell Sci. 114, 4599 &#8211; 4611. <br /> differential localization 5% rate (6 125) <br />                                                                               Elion, E., Satterberg, B., Kranz, J. (1993). FUS3 phosphorylates multiple <br /> observed here. We expect transcription factor com-                     components mating signal transduction cascade: evidence STE12 <br /> plement yeast similarly regulated subcellular                     FAR1. Mol. Biol. Cell 4, 495&#8211;510. <br /> localization high degree, and, complement                     Erdman, S., Snyder, M. (2001). A filamentous growth response mediated <br /> study, interesting screen set yeast                yeast mating pathway. Genetics 159, 919 &#8211;928. <br /> transcription factors differential localization fila-              Ferrigno, P., Posas, F., Koepp, D., Saito, H., Silver, P. A. (1998). Regulated <br /> mentous growth.                                                               nucleo/cytoplasmic exchange HOG1 MAPK requires importin <span id='am-7' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-11'  ></span><span id='am-12' property="rdfs:label" content="Beta" datatype="xsd:string"></span>beta</span> <br />    Collectively, study presents large-scale anal-              homologs NMD5 XPO1. EMBO J. 19, 5606 &#8211;5614. <br /> ysis differential protein localization eukaryote,               Fleischmann, M., Stagljar, I., Aebi, M. (1996). Allele-specific suppression <br /> the kinase network identified defines previously over-                 Saccharomyces cerevisiae prp20 mutation overexpression nuclear <br /> looked mechanism regulatory control yeast cell                      serine/threonine protein kinase. Mol. Gen. Genet. 250, 614 &#8211; 625. <br /> growth development. Conceptually, regulated protein                       Fritz, C. C., Zapp, M. L., Green, M. R. (1995). A human nucleoporin-like <br /> localization provides level specificity pro-                protein specifically interacts HIV Rev. Nature 376, 530 &#8211;533. <br /> miscuous kinase activities, network-based structure                   Gancedo, J. M. (2001). Control pseudohyphae formation Saccharomyces <br /> of regulated localization confers greater specificity still.             cerevisiae. FEMS Microbiol. Rev. 25, 107&#8211;123. <br /> The implications findings extend un-                      Gasch, A. P., Huang, M., Metzner, S., Botstein, D., Elledge, S. J., Brown, <br /> derstanding yeast cell biology. Similar mechanisms                     P. O. (2001). Genomic expression responses DNA-damaging agents the <br /> assuredly play higher eukaryotes well, subse-                    regulatory role yeast ATR homolog Mec1p. Mol. Biol. Cell 12, 2987&#8211; <br />                                                                               3003. <br /> quent investigations higher organisms clarify the <br /> degree localization-based regulatory net-                      Gelperin, D. et al. (2005). Biochemical genetic analysis yeast <br />                                                                               proteome movable ORF collection. Genes Dev. 19, 2816 &#8211;2826. <br /> works evolutionarily conserved. <br />                                                                               Giaever, G., Shoemaker, D. D., Jones, T. W., Liang, H., Winzeler, E. A., <br />                                                                               Astromoff, A., Davis, R. W. (1999). Genomic profiling drug sensitivities <br /> ACKNOWLEDGMENTS                                                               induced haploinsufficiency. Nat. Genet. 21, 278 &#8211;283. <br /> We thank Damian Krysan (University Rochester), Robert Fuller (University   Gimeno, C. J., Fink, G. R. (1994). Induction pseudohyphal growth by <br /> of Michigan), Anthony Borneman (Yale University), Michael Snyder          overexpression PHD1, Saccharomyces cerevisiae gene related transcrip- <br /> (Yale University) providing filamentous yeast strains. We thank Hiten     tional regulators fungal development. Mol. Cell. Biol. 14, 2100 &#8211;2112. <br /> Madhani (University <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>California</span>, San Francisco) providing kinase-dead <br />                                                                               Gimeno, C. J., Ljungdahl, P. O., Styles, C. A., Fink, G. R. (1992). Unipolar <br /> alleles KSS1 FUS3, Tom Fox (Cornell University) providing the <br />                                                                               cell divisions yeast S. cerevisiae lead filamentous growth: regulation <br /> tpk2-K99R allele, Markus Aebi (ETH Zurich) providing ksp1-K47D <br />                                                                               starvation RAS. Cell 68, 1077&#8211;1090. <br /> allele. This work supported grants RSG-06-179-01-MBC the <br /> American <span id='am-37' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39'  ></span><span id='am-40' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Cancer</span> Society, DBI 0543017 National Science Foundation,    Griffioen, G., Branduardi, P., Ballarini, A., Anghileri, P., Norbeck, J., Baroni, <br /> and Basil O&#8217;Connor Award 5-FY05-1224 March Dimes (to A.K.).       M. D., Ruis, H. (2001). Nucleocytoplasmic distribution budding yeast <br />  <br />  <br /> 2716                                                                                                                       Molecular Biology Cell <br />                                                                                                                      Regulated Localization Yeast Kinases <br />  <br /> protein kinase A regulatory subunit Bcy1 requires Zds1 regulated          Nagai, T., Ibata, K., Park, E. S., Kubota, M., Mikoshiba, K., Miyawaki, A. <br /> Yak1-dependent phosphorylation targeting domain. Mol. Cell. Biol. 21,        (2002). A variant yellow fluorescent protein fast efficient matu- <br /> 511&#8211;523.                                                                            ration cell-biological applications. Nat. Biotechnol. 20, 87&#8211;90. <br /> Guthrie, C., Fink, G. (1991). Guide Yeast Genetics Molecular             Nakai, K., Horton, P. (1999). PSORT: program detecting sorting <br /> Biology, San Diego, CA: Academic Press.                                             signals proteins predicting subcellular localization. Trends Bio- <br />                                                                                     chem. Sci. 24, 34 &#8211;36. <br /> Gygi, S. P., Rist, B., Gerber, S. A., Turecek, F., Gelb, M. H., Aebersold, R. <br /> (1999). Quantitative analysis complex protein mixtures using isotope-coded       O&#8217;Neill, E. M., Kaffman, A., Jolly, E. R., O&#8217;Shea, E. K. (1996). Regulation <br /> affinity tags. Nat. Biotechnol. 17, 994 &#8211;999.                                       PHO4 nuclear localization PHO80-PHO <br /> </body></html>