<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Transcription-Associated Mutagenesis Increases Protein <br /> Sequence Diversity More Effectively Does Random <br /> Mutagenesis <span id='am-22' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-31'  ></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span>Escherichia coli</span> <br /> Hyunchul Kim1,2, Baek-Seok Lee1&#164;, Masaru Tomita1,2, Akio Kanai1,2* <br /> 1 Institute Advanced Biosciences, Keio University, Tsuruoka, Japan, 2 Systems Biology Program, Graduate School Media Governance, Keio University, Fujisawa, <br /> Japan <br />  <br />  <br />  <br />      Abstract <br />      Background: During transcription, nontranscribed DNA strand single-stranded DNA (ssDNA), form <br />      secondary structures. Unpaired bases ssDNA protected mutagens experience more <br />      mutations paired bases. These mutations called transcription-associated mutations. Transcription-associated <br />      mutagenesis increased stress depends DNA sequence. Therefore, selection significantly influence <br />      protein-coding sequences terms transcription-associated mutability transcription event stress to <br />      improve survival <span id='am-38' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-41'  ></span><span id='am-42' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span>Escherichia</span> coli. <br />  <br />      Methodology/Principal Findings: The mutability index (MI) developed Wright et al. estimate relative <br />      transcription-associated mutability bases transcription event. Using stable fold ssDNA an <br />      average length n, MI defined (the number folds base unpaired)/n6(highest &#8211;DG n folds in <br />      base unpaired), DG free energy. The MI values significant correlation mutation <span id='am-45' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-46'  ></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> <br />      stress spontaneous mutations E. coli. Protein sequence diversity preferred stress not <br />      favorable conditions. Therefore, evaluated selection pressure MI terms protein sequence diversity <br />      protein-coding sequences E. coli. The distributions MI values lower bases be <br />      substituted bases affecting amino acid sequence bases be <br />      substituted. Start codons lower distributions MI values did nonstart codons. <br />  <br />      Conclusions/Significance: Our results suggest majority protein-coding sequences evolved promote <br />      protein sequence diversity reduce gene knockout stress. Consequently, transcription-associated mutagenesis <br />      increases protein sequence diversity effectively does random mutagenesis stress. Nonrandom <br />      transcription-associated mutagenesis stress improve survival E. coli. <br />  <br />   Citation: Kim H, Lee B-S, Tomita M, Kanai A (2010) Transcription-Associated Mutagenesis Increases Protein Sequence Diversity More Effectively Does <br />   Random Mutagenesis Escherichia coli. PLoS ONE 5(5): e10567. doi:10.1371/journal.pone.0010567 <br />   Editor: Ramy K. Aziz, Cairo University, Egypt <br />   Received October 19, 2009; Accepted April 19, 2010; Published May 10, 2010 <br />   Copyright: &#223; 2010 Kim et al. <span id='am-1' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This research supported Grant-in-Aid Keio University Global Center Excellence Program Mori memorial grant by <br />   research funds Yamagata prefectural government Tsuruoka City, Japan. The funders role study design, data collection analysis, <br />   decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: akio@sfc.keio.ac.jp <br />   &#164; Current address: R&amp;D Center Bioproducts, CJ Corporation, Seoul, Korea <br />  <br />  <br />  <br /> Introduction                                                                          observed highly transcribed regions diverse <br />                                                                                       <span id='am-16' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>, Escherichia coli [9,10,11], yeast [8], humans <br />    During transcription, nontranscribed strand single                     [12,13]. The nontranscribed strand thought greater <br /> stranded, transcribed strand forms complex                         numbers mutations transcribed strand E. coli <br /> RNA polymerase nascent RNA transcript [1]. The ssDNA                          [6,9,11] humans [14,15,16]. The larger numbers mutations <br /> is vulnerable mutagens double-                          nontranscribed strand partly solely attributable <br /> stranded DNA [2] protected pairing [3]. The                      activity transcription-coupled DNA repair system, <br /> resulting mutations include single-base substitutions [2,4,5,6]                   acts transcribed strand, E. coli [6]. Transcription- <br /> insertions/deletions (indels) [4,7] called transcription-                     associated mutations considered occur regardless specific <br /> associated mutations [8] transcription-induced mutations [9].                      secondary structures [17]. Transcription-associated mutagenesis <br /> Therefore, transcription-associated mutagenesis active                      active stress [18], occurs genomic <br /> on nontranscribed strands, highly transcribed DNA regions,                     DNA plasmid DNA E. coli [17]. Therefore, transcription- <br /> and cells stress high levels mutagens active.                   associated mutagenesis considered intrinsic source of <br /> The existence significance transcription-associated muta-                      mutations [5,8,11,19]. Furthermore, transcription-associated mu- <br /> genesis widely supported. For example, increased mutations                         tations occur level affects genomic composition T7 <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                1                                  May 2010 | Volume 5 | Issue 5 | e10567 <br />                                                                                                                      Effective Nonrandom Mutations <br />  <br />  <br /> bacteriophage [20]. As transcription-associated mutations occurs               tion event, depending period base is <br /> asymmetrically transcribed nontranscribed strands [6],              unpaired [28]. Transcription-associated mutability transcrip- <br /> when transcription-associated mutations occur cell           tion event estimated methods [3,28]. Among <br /> are repaired, daughter cells different genomic                (see &#8216;&#8216;Materials Methods&#8217;&#8217; details), mutability <br /> DNA. Transcription-coupled DNA repair act selectively                   index (MI), developed Wright et al., focuses transcription- <br /> transcribed strand [21]. Most, all, mutations nontran-               associated mutagenesis stress conditions [28]. The following <br /> scribed strands expressed replication.                     description method Wright et al. [28]. Because the <br /> Therefore, transcription-associated mutations predomi-                  length ssDNA vary, length ssDNA simplified <br /> nantly exert effects transcribing cells          average value n [28]. Therefore, given base assumed to <br /> descendant cells. The mutation rate E. coli lower               belong n ssDNA. The stable fold ssDNA was <br /> base pair genome replication [22]. Therefore, E. coli cells            identified, &#8211;DG (the negative free energy) and <br /> would contain transcription-associated mutation               pairing information. Using n folds containing given base, <br /> other types mutations genomic strand mutations               MI given base defined (number folds which <br /> the genomic strand cell division transcription-               base unpaired)/n6(highest 2DG n folds the <br /> associated mutagenesis operates. In cases, unmutated                  base unpaired). After scanning average ssDNA length (n) for <br /> genomic DNA strand cell inherited                  high 2DG best match known vivo hotspot data E. coli, <br /> daughter cells. Consequently, transcription-associated mutagenesis             Wright et al. set average length ssDNA (n) 30 nt. The <br /> can considered safe way dividing cells rapidly increase            calculated MI values showed positive significant correlation with <br /> the sequence diversity generation.                                 vivo mutation data reversion assays, with <br />    Transcription-associated mutagenesis investigat-             spontaneous mutation data [28]. This result suggests MI can <br /> ed reversion assays stress [6,23]. The nonrevertants           represent relative transcription-associated mutability per <br /> reversion assays assumed nondividing cells. In                 transcription event E. coli conditions reversion <br /> reversion assays, organisms E. coli engineered divide           assays. Nontranscription-associated mutations excluded <br /> actively form large detectable viable colonies             mutation data sets used validation tests. <br /> one requisite mutations occurs. E. coli continue            Therefore, validation results imply transcription-associat- <br /> to divide slowly living debris cells [24]. For              ed mutations constitute large fraction total mutations in <br /> example, E. coli survive batch cultures addition            highly transcribed regions conditions reversion <br /> of nutrients months [25]. Therefore, possibility             assays. Conversely, invalidation MI spontaneous <br /> such nonrevertants divide times die                  mutation data attributable the <br /> excluded. If cells assumed nondividing, difficult       following causes. First, transcription-associated mutations <br /> how E. coli survive high mutation rate experienced                constitute large fraction total <br /> revertants. However, high mutation rate explained             spontaneous mutations correlation. Second, MI <br /> the E. coli cells live cell debris transcription-associated          30-nt ssDNA represent transcription-associated <br /> mutagenesis plays significant role. We possible              mutability transcription event favorable conditions. <br /> scenario. After nonrevertants plated, E. coli cells                This occur average length ssDNA set 30 <br /> continue divide slowly living debris cells,              nt screening hotspot data obtained stress [28]. <br /> perhaps following death phase, 99% cells                    Transcription-associated mutability transcription event <br /> die. When E. coli cells experience stress, transcription-                  depends secondary structures formed DNA sequence <br /> associated mutagenesis increases [18]. E. coli safely increases            [23,28,29] influenced selection [3]. Every <br /> sequence diversity daughter cells                      mutation does exert effect. In protein-coding <br /> transcription-associated mutations occur repaired                 sequences, silent mutations circumvent potentially deleterious <br /> correctly cell division. Whenever detrimental mutations                 effects increase protein sequence diversity. Protein <br /> occur combination preexisting mutations, cells            sequence diversity advantageous stress under <br /> die produce cell debris. The surviving cells gradually                     conditions spontaneous mutation. Interestingly, MI showed a <br /> accumulate mutations cell division recurs. While mutation                   positive significant correlation mutation data stress but <br /> and selection occur, transcription-associated mutations occur              conditions spontaneous mutation [28]. Transcription- <br /> at sites produce revertants. Therefore, multiple                  associated mutagenesis strongly suggested play <br /> mutations revertants interpreted result                important roles stress. For example, transcription-associated <br /> mutation selection generations,                   mutations abundant stress [18,28] highly <br /> significantly reduce estimated mutation rate. Revertants               transcribed regions [10,11]. Transcribed regions given <br /> usually counted 48 h plating [6,23], longer            stress better targets beneficial mutations that <br /> overnight day incubation typically required colonies             stress [30]. The MI validation Wright et al. [28] suggested that <br /> appear nonrevertant E. coli cells plated rich medium.              transcription-associated mutagenesis responsible large <br /> This late appearance revertants attributed slow cell              fraction total mutations. Therefore, protein-coding sequences <br /> division, relatively low mutation rate, limited numbers               evolved effectively increase protein sequence <br /> surviving cells generation. The mismatched DNA base pairing                diversity controlling transcription-associated mutability per <br /> caused mutations result cross-strand deamination vitro            transcription event stress MI values. If <br /> [26,27], possible transcription-associated mutagenesis         nonrandom MI values shaped protein-coding <br /> rescue nondividing cells.                                                      sequences, provide clear advantage survival E. <br />    The ssDNA nontranscribed strand forms secondary                      coli. In present study, analyzed effect selection MI <br /> structures [18], different stability. Therefore,               values using 4,132 protein-coding sequences E. coli K12 <br /> ssDNA sequences sustained longer time others.                   MG1655, fully sequenced E. coli strain. Our results that <br /> Consequently, individual bases nontranscribed strand                    bases higher MI values three <br /> display different transcription-associated mutability transcrip-           possible single-base substitutions base changes encoded <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          2                              May 2010 | Volume 5 | Issue 5 | e10567 <br />                                                                                                                        Effective Nonrandom Mutations <br />  <br />  <br /> amino acid single-base substitutions               missense mutations changed encoded amino acid <br /> at base changes encoded amino acid. Start codons                  (Basemissense); 3) single-base substitutions produced <br /> evolved lower MI values nonstart codons. The                      types mutations: silent mutation, missense <br /> selection pressure different different base groups individual         mutation, nonsense mutation (Baseother; Tables 1&#8211;2). <br /> proteins. Our results suggest majority protein-coding                 Generation control sequences. To analyze selection <br /> sequences evolved E. coli produce transcription-                    pressure MI terms generation protein-coding <br /> associated mutations way reduce gene knockout,                 sequence diversity, control sequences related <br /> while increasing protein sequence diversity, stress. Such                features required. Protein-coding sequences exert of <br /> nonrandom mutagenesis provide better sets mutations                   effects encoded proteins. The compositional ratios of <br /> even mutations exposed selection. Different                  Basesilent, Basemissense, Baseother affect potential <br /> selection pressures MI allow protein-coding                     generate protein sequence diversity. To comport same <br /> sequences genome different evolvability. We discuss             product protein sequence compositional ratios of <br /> the biological benefits nonrandom MI values, selection                  Basesilent, Basemissense, Baseother, generated 100 control <br /> shapes MI values, variation selection pressures              sequences protein-coding sequence shuffling positions of <br /> MI values.                                                                     synonymous codons protein-coding sequence, <br />                                                                                start codons stop codons (Figure S1). The <br /> Results                                                                        resulting control sequences GC content the <br />                                                                                corresponding protein-coding sequence, affect the <br /> Calculation MI                                                              local secondary structures [3], <span id='am-6' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>codon</span> <br />    Transcription-associated mutability transcription event                 usage. The MI values control sequences calculated as <br /> should affected local secondary structures RNA                described protein-coding sequences. <br /> transcript, functions noncoding genes depend.                     Evaluation selection pressure MI (SMI). The <br /> Therefore, noncoding genes excluded analysis. MI                protein-coding sequences control sequences have <br /> values 3,958,572 bases 4,132 protein-coding                     different average potentials produce transcription-associated <br /> sequences E. coli K12 MG1655 genome calculated                     mutagenesis transcription event [3]. By calculation, the <br /> according method described Wright et al. [28],               lacI gene, example, average MI 2.08 control <br /> negative MI values converted zero (see &#8216;&#8216;Materials            sequences average MI values 1.78 2.42. <br /> and Methods&#8217;&#8217; details). The mass distribution MI values             Transcription-associated mutability affected changes in <br /> was concentrated smaller MI values (0#MI#2) exhibited                 transcription level [10]. Therefore, compare relative MI <br /> longer tail larger MI values (8#MI, skewness = 0.85;                    values individual bases protein-coding sequences <br /> Figure 1).                                                                     control sequences, standard-normalized MI values were <br />                                                                                introduced analysis. The selection pressure MI at <br /> Evaluation selection pressures terms protein                          Basesilent, example, protein-coding sequence evaluated <br /> sequence diversity                                                             following way (Figure S2). All MI values standard <br />   Sequence properties affecting protein sequence                               normalized protein-coding sequence control <br /> diversity. To analyze selection pressure MI terms                 sequence. All resulting z-scores MI values Basesilent, <br /> the generation protein sequence diversity, protein                       example, averaged protein-coding sequence <br /> evolvability, bases protein-coding sequence,              control sequence. The average z-score Basesilent <br /> the exception comprising start stop codons,              protein-coding sequence ranked 100 <br /> divided groups. The groups based results             control sequences. The resulting rank values linearly <br /> the possible single-base substitutions base. The           transformed &#8216;&#8216;SMI&#8217;&#8217; values (21#SMI#1), indicates the <br /> groups defined follows: 1) single-base substitutions         selection pressure MI stress. As consequence, each <br /> resulted silent mutations, change encoded                  protein-coding sequence SMI value Basesilent. A <br /> amino acid (Basesilent); 2) single-base substitutions resulted       negative positive SMI value Basesilent protein-coding <br />                                                                                sequence indicated protein-coding sequence lower <br />                                                                                higher average z-score MI values Basesilent, <br />                                                                                respectively, did 50% random control sequences. For <br />                                                                                example, negative SMI value protein-coding sequence at <br />                                                                                Basesilent indicates protein-coding sequence evolved to <br />                                                                                lower transcription-associated mutability transcription <br />                                                                                event Basesilent stress. This method calculating SMI <br />                                                                                values applied Basemissense, Baseother, base <br />                                                                                groups subsequent analyses described manuscript. <br />                                                                                Basesilent exhibited lower SMI values Basemissense and <br />                                                                                Baseother (Wilcoxon signed-rank test, P,1e290 P,1e290, <br />                                                                                respectively; &#8216;&#8216;Materials Methods&#8217;&#8217; choice of <br />                                                                                statistical methods). Basesilent Basemissense exhibited <br />                                                                                distributions SMI lower higher zero, respectively <br />                                                                                (Wilcoxon rank-sum test, P,1e2100 P,1e2100, respectively; <br />                                                                                Figure 2). These results suggest protein-coding sequences have <br /> Figure 1. Distribution mutability index (MI) values of <br />                                                                                evolved increase ratio missense mutations silent <br /> bases located protein-coding sequences E. coli K12 <br /> MG1655. Closed circles indicate number bases MI value           mutations transcription-associated mutations stress. <br /> range (0.5). The vertical dotted line indicates mean value (2.54).         In short, results suggest genome sequence evolved <br /> doi:10.1371/journal.pone.0010567.g001                                          increase protein evolvability stress. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          3                              May 2010 | Volume 5 | Issue 5 | e10567 <br />                                                                                                                                  Effective Nonrandom Mutations <br />  <br />  <br />  <br />  Table 1. Mutations according position protein-coding sequence. <br />  <br />  <br />  Bases                             Single-base substitution                                        Insertion deletion <br />  <br />                                    Main results                                Main effect         Main result                       Effects <br />  <br />  Start codon                       Disruption start codon                   Gene knockout       Frameshift mutations              Gene knockout protein <br />                                                                                                    (unless multiples        truncation, some <br />                                                                                                    bases inserted             functional impairment <br />                                                                                                    deleted)                          causes frameshift <br />  Codons occurring          Only silent mutations (Basesilent)          Neutral effect <br />  start stop codons <br />                                    Only missense mutations (Basemissense)      Single amino-acid <br />                                                                                substitution <br />                                    Mixture silent, missense, nonsense   Various effects <br />                                    mutations (Baseother) <br />  <br />  doi:10.1371/journal.pone.0010567.t001 <br />  <br />  <br /> Selection pressure MI terms gene knockout                                 subtracted SMI values start codons, results <br />    Start codons sites likely involved mutation-              zero (Wilcoxon rank-sum test, P,1e220; Figure 3D). <br /> based gene knockout protein-coding sequences. To assess                     However, adjacent codon pairs tested did yield <br /> the overall distribution SMI protein-coding sequences,                    values significantly zero (Wilcoxon rank-sum test, <br /> grouped primary positions protein-coding sequences                     P.0.01; Figure 3D). Therefore, lower SMI values obtained <br /> deciles, 59 39 direction. Among 10 base groups, base               start codons just tendency 59 ends of <br /> group decile showed lowest distribution SMI                    protein-coding sequences. Start codons exhibit lower SMI values <br /> values (Wilcoxon signed-rank test division having                     ATG codons (Wilcoxon rank-sum test, <br /> second-lowest SMI, P,0.01; Figure 3A). To analyze result                   P,1e260; Figure 3E). This result excludes possibility a <br /> greater detail, decile divided 1% divisions.            fitness effect exerted mutation ATG codons. To confirm <br /> Among resulting 10 groups, base group                         protein-coding gene knockout influences effects the <br /> percentile showed lowest distribution SMI values (Wilcoxon                  selection pressure MI values, SMI values start codons <br /> rank-sum test division second-lowest SMI, P,1e210;                 compared according position nonstart ATG <br /> Figure 3B). Start codons occupy 59-most positions protein-                  codon. The presence closely following nonstart ATG codon at <br /> coding sequences. This explain base group 59-                   59 end protein-coding sequence implies single-base <br /> most position lowest SMI value groups tested.                    substitutions start codons cause deletion N- <br />                                                                                    terminal amino acid residues. These N-terminal deletions a <br /> To confirm this, bases grouped nth codon and <br />                                                                                    lower potential cause protein-coding gene knockout. The SMI <br /> the SMI compared. The start codons displayed lowest SMI <br /> values codons located 59 regions protein- <br /> coding sequences (Wilcoxon signed-rank test division of <br /> the second-lowest SMI, P,1e220; Figure 3C). The tendency to <br /> form secondary structures affected primary position <br /> on protein-coding sequence. For instance, bases 59 end <br /> are affected efficiency translation initiation bases at <br /> the 39 end Rho-independent transcription termination. The <br /> base groups percentiles showed reduction SMI <br /> values decreasing percentile (Figure 3B). Therefore, the <br /> lowest SMI values observed start codons caused by <br /> this tendency low SMI values percentiles. <br /> To exclude possibility, compared results the <br /> subtraction SMI values adjacent codons 10 <br /> codons. When SMI values second codons were <br />  <br />  <br />  Table 2. Compositions base types protein-coding                             Figure 2. SMI base groups according mutation <br />  sequences.                                                                        types generated single-base substitution. The bases were <br />                                                                                    grouped according type mutations caused single-base <br />                                                                                    substitutions. &#8216;&#8216;Basesilent&#8217;&#8217; &#8216;&#8216;Basemissense&#8217;&#8217; indicate bases all <br />  Base type                 Mean                         95% CI                     single-base substitutions bases result silent missense <br />                                                                                    mutations, respectively. &#8216;&#8216;Baseother&#8217;&#8217; indicates bases than <br />  Basesilent                16.63                        16.57&#8211;16.69                Basesilent, Basemissense, bases occur start stop codons <br />  Basemissense              56.96                        56.90&#8211;57.02                protein-coding sequences. Circles error bars indicate the <br />                                                                                    pseudomedian 95% confidence interval (CI), respectively, as <br />  Baseother                 26.42                        26.32&#8211;26.52                assessed using Wilcoxon rank-sum test. The vertical dotted line <br />  Sum                       100.0                                                   represents 0 value SMI. Calculations Wilcoxon signed-rank <br />                                                                                    test: *1 P,1e290; *2 P,1e290. <br />  doi:10.1371/journal.pone.0010567.t002                                             doi:10.1371/journal.pone.0010567.g002 <br />  <br />  <br />         PLoS ONE | www.plosone.org                                             4                                 May 2010 | Volume 5 | Issue 5 | e10567 <br />                                                                                                                     Effective Nonrandom Mutations <br />  <br />  <br />  <br />  <br /> Figure 3. SMI base groups according relative positions protein-coding sequences. (A) The relative positions <br /> were grouped deciles 0 100%. (B) The relative positions grouped percentiles 0 10%. (C) Start codons their <br /> adjacent downstream codons. The 1 Y-<span id='am-11' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Axis" datatype="xsd:string"></span>axis</span> indicates start codon. (D) Difference SMI values nth codon n+1th codon. <br /> For example, &#8216;&#8216;D1&#8211;2&#8217;&#8217; indicates difference SMI values codons (start codons) second codons. (E) Start <br /> codons following nonstart ATG codons. The 1 Y-axis indicates start codon. (F) SMI values start codons, according the <br /> positions nonstart ATG codons. $ indicates largest P value calculated base group marked $ other <br /> base groups: $1 P,0.01; $2 P,1e210; $3 P,1e220; $4 P,1e260; *1 P,1e220; *2 P,1e24; *3 P,1e24 ($1, $3: Wilcoxon signed-rank test; $2, $4, *1, *2, *3: <br /> Wilcoxon rank-sum test). Circles error bars indicate pseudomedian 95% CI, respectively, assessed using Wilcoxon rank-sum and <br /> signed-rank test. The vertical dotted lines represent 0 value SMI. <br /> doi:10.1371/journal.pone.0010567.g003 <br />  <br />  <br /> values start codons higher presence nonstart ATG           single-base substitutions (see &#8216;&#8216;Materials and <br /> codons 10 codons presence nonstart ATG           Methods&#8217;&#8217;). The MI values showed significant positive correlations <br /> codons 10 codons (Wilcoxon rank-sum test, P,1e24;            mutated bases (R2 = 0.29, P,1e2100) the <br /> Figure 3F) absence nonstart ATG codons              neighboring bases (R2 = 0.79, P,1e2100). The bases high <br /> first 10 codons (Wilcoxon rank-sum test, P,1e24; Figure 3F).                 MI values tended retain similar MI values single-base <br /> These results demonstrate selection acts reduce MI               substitutions (Figure S4). These results demonstrate MI <br /> values start codons start codon mutations cause gene                 values bases usually subject larger variation when <br /> knockout. These results suggest protein-coding sequences                mutations occur base itself mutations occur at <br /> have evolved reduce proportion gene knockouts                   neighboring bases, bases high MI values more <br /> transcription-associated mutations stress.                             tolerant variation MI values single-base <br />                                                                              substitutions bases bases low MI values. <br /> Correlation single-base <br /> substitutions                                                                Discussion <br />    A single-base substitution ssDNA changes most <br /> stable secondary structure ssDNA. Therefore, single-base            Biological benefits nonrandom distribution MI <br /> substitution base change MI value base itself            values stress <br /> and 58 neighboring bases (29 bases upstream 29 bases                 In study, demonstrated selection acts on <br /> downstream; Figure S3). The altered MI value affect                protein-coding sequences lower MI values Basesilent to <br /> positive negative selection shapes nonrandom MI. To             increase MI values Basemissense Baseother stress <br /> examine nonrandom MI distributions shaped,                     (Figure 2). Under stress, protein sequence diversity preferred and <br /> important assess mutational effect specific base          diversity beneficial. Most beneficial mutations are <br /> own MI value MI values neighboring bases.                      missense mutations indels [31]. Missense mutations <br /> Therefore, repeated 10,000 random single-base substitutions               occur Basemissense Baseother Basesilent. <br /> 100 times analyzed correlation values MI values           However, missense mutations detrimental effects. <br /> the mutated bases neighboring bases                 Therefore, stress, high MI Basemissense Baseother and <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        5                              May 2010 | Volume 5 | Issue 5 | e10567 <br />                                                                                                                     Effective Nonrandom Mutations <br />  <br />  <br /> low MI Basesilent increase chance beneficial mutations         nonsense mutations. The compositions silent mutations, <br /> at cost increasing detrimental mutations. The proportion           missense mutations, nonsense mutations Baseother were <br /> detrimental mutations indels larger proportion             26%, 60%, 14%, respectively, protein-coding <br /> of detrimental mutations single-base substitutions. There-             sequences. Because proportion missense mutations highest <br /> fore, proportion detrimental mutations indels                Baseother, MI values Baseother influenced by <br /> larger proportion detrimental mutations indels             selection similar way Basemissense to <br /> plus single-base substitutions. A protein sequence mutated            Basesilent. <br /> only indels Basesilent indels single-base <br /> substitutions Basemissense Baseother. Therefore, propor-          Variation selection pressure MI values <br /> tion detrimental mutations mutations result                  Transcription-associated mutations increase stress in <br /> change protein sequence larger Basesilent               highly transcribed regions. Individual protein-coding sequences <br /> Basemissense Baseother.                                                  repressed derepressed different stresses have <br />    These results imply protein-coding sequences evolved            different transcription levels specific stresses. Therefore, <br /> to reduce proportion detrimental mutations produced             protein-coding sequence different SMI value of <br /> increasing protein sequence diversity stress. Supporting          base groups (Table 3). This result suggests protein <br /> proposition, MI values start codons low mutations            products individual protein-coding sequences different <br /> at start codons likely cause gene knockouts (Figure 3). Gene          evolvability. Interestingly, protein-coding <br /> knockouts result detrimental mutations, example              sequences SMI values larger zero Basesilent <br /> essential genes, exert disadvantageous effects. In                 (Table 3). This suggests protein-coding <br /> prokaryotes E. coli, selective pressure exerted            sequences evolved reduce protein sequence diversity under <br /> genome length, compact genomes [32]. This                  stress. Reduced protein evolvability beneficial favorable <br /> implies preexisting genes recently contributed              conditions. Therefore, high SMI values Basesilent low <br /> fitness cell lost response selection       SMI values Basemissense Baseother attributable the <br /> pressure genome size. Consequently, data suggest                 activity transcription-associated mutagenesis favorable <br /> protein-coding sequences evolved control transcrip-              conditions. However, development index predicts <br /> tion-associated mutability transcription event increase               transcription-associated mutability transcription event under <br /> protein evolvability reduce proportion detrimental             favorable conditions subsequent analysis necessary <br /> mutations produced stress.                                             determine mechanism underlying phenomenon. <br />  <br /> Selection type shapes nonrandom MI values                        Conclusions <br /> protein-coding sequences                                                        In study, evaluated selection pressure MI, which <br />    The reduction MI values Basesilent start codons          represents relative potential transcription-associated <br /> achieved purifying selection. Detrimental mutations               mutagenesis transcription event stress. Our results <br /> passed descendant cells. We demonstrated                 suggest majority protein-coding sequences evolved <br /> mutations alter MI values neighboring bases           increase protein sequence diversity controlling transcription- <br /> of mutated base itself. Therefore, different MI values            associated mutagenesis stress transcription- <br /> start codons Basesilent caused mutations                    associated mutagenesis produces protein sequence diversity more <br /> neighboring bases. E. coli cells descendants             effectively does random mutagenesis. Therefore, transcrip- <br /> start codons Basesilent high MI values frequently             tion-associated mutagenesis confer faster protein evolvability <br /> suffer detrimental mutations mutations occur frequently           stress, improve chance survival E. coli. <br /> these sites. Increases MI values Basemissense be <br /> achieved positive selection. Positive selection implies          Materials Methods <br /> mutated bases inherited descendant cells become <br /> dominant. The new bases sites different MI                Sequence data <br /> values. However, demonstrated, new MI values                         The sequence E. coli K12 MG1655 downloaded from <br /> correlate positively old MI values mutated sites. In         NCBI ftp site (ftp://ftp.ncbi.nih.gov/genomes/Bacteria/ <br /> particular, bases high MI values tend retain high MI           Escherichia_coli_K12). The sequences 4,132 E. coli protein- <br /> values single-base substitutions (Figure S4). Therefore, high          coding genes used analysis. For protein-coding <br /> MI values Basemissense frequently retained,             sequence, 100 control sequences generated synonymous <br /> degree fluctuation avoided. The distribution SMI             codon shuffling; codon position given amino acid was <br /> values Baseother closer Basemissense       shuffled, positions start stop codons were <br /> Basesilent (Figure 2). Single-base substitutions Baseother       maintained (see Figure S1). This step repeated 20 <br /> up types mutations: silent, missense,                  amino acids. Random shuffling performed using &#8216;&#8216;random&#8217;&#8217; <br />  <br />  Table 3. Distributions SMI values. <br />  <br />  <br />                           21.0#SMI,20.5                 20.5#SMI,0                 0#SMI,0.5              0.5#SMI#1              Sum <br />  <br />  Basemissense             21.25%                        21.03%                     25.46%                 32.26%                 100.0% <br />  Baseother                19.53%                        21.93%                     24.25%                 34.29%                 100.0% <br />  Basesilent               38.53%                        25.62%                     20.57%                 15.27%                 100.0% <br />  <br />  doi:10.1371/journal.pone.0010567.t003 <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                       6                              May 2010 | Volume 5 | Issue 5 | e10567 <br />                                                                                                                    Effective Nonrandom Mutations <br />  <br />  <br /> module standard library <span id='am-55' about='obo:NCBITaxon_37579' typeof='owl:Thing obo:NCBITaxon_1329950 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8504 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8509 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_1329912 obo:NCBITaxon_33208 obo:NCBITaxon_1329961 obo:NCBITaxon_1329911 obo:NCBITaxon_7711 obo:NCBITaxon_8570 obo:NCBITaxon_34979 obo:NCBITaxon_7776 obo:NCBITaxon_34984 obo:NCBITaxon_32561 obo:NCBITaxon_8457'><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Python" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="NCBITaxon:51884" datatype="xsd:string"></span><span id='am-60'  ></span>Python</span> program, version 2.6            &#8216;number folds base unpaired&#8217; zero, <br /> (http://www.python.org).                                                     MI value zero. This contradicts assumption that <br />                                                                              bases remain unpaired longer time higher MI <br /> Sequence features                                                            values. Stable folds formed &#8211;DG most <br />    Relative base positions protein-coding sequences. The                  stable fold zero. Therefore, values converted <br /> base pairs protein-coding sequence length n base pairs        zero. <br /> were numbered 1 n. For kth base, length ratio was <br /> defined ([k&#8211;1]/[n&#8211;1])6100. For example, base start       Evaluation effect selection MI (SMI) <br /> codon base stop codon assigned positions 0             For protein-coding sequences control sequences, the <br /> and 100, respectively.                                                       MI value base standard normalized protein- <br />    nth codon. In protein-coding sequence produces                   coding sequence. To compare transcription-associated <br /> protein n amino acids, codons numbered 1                 mutability transcription event base groups, the <br /> n+1. For example, start stop codons assigned                    standard-normalized MI values base group averaged <br /> positions 1 n+1, respectively.                                           protein-coding sequence control <br />    nth ATG codon. In protein-coding sequence, start                    sequences. The rank gene sequence given base group was <br /> codon assigned position 1, ATG. The              calculated averaged values corresponding <br /> subsequent k ATG codons sequentially assigned positions                 values 100 control sequences. The rank transformed to <br /> from 2 k+1.                                                               SMI values using equation (26rank &#8211; 102)/100. As result, each <br />                                                                              protein-coding sequence value 21 1 <br /> Selection method predict transcription-associated                    (21#SMI#1) base group. <br /> mutability transcription event <br />    Two methods developed predict transcription-                 Correlation MI values mutation <br /> associated mutagenesis potentials. The MI Wright               Ten thousand single bases selected randomly protein- <br /> et al., focuses transcription-associated mutations          coding sequences randomly mutated other <br /> validated using vivo mutation data reversion assays.                 possible bases. The MI values base targeted mutation <br /> Because purpose study investigate selection           29 bases located upstream downstream this <br /> pressure transcription-associated mutability transcription            position calculated mutation. The <br /> event stress, Wright et al.&#8217;s MI adequate. Hoede et al.            neighboring bases protein-coding <br /> [3] developed index, transcription-driven mutability             sequences excluded calculation correlation. <br /> index (TDMI). To validate indices, authors used                  The MI values bases mutation were <br /> conservation data taken alignment orthologous gene               used calculate Pearson&#8217;s correlation values. These steps were <br />                                                                              repeated 100 times. The mean Pearson&#8217;s correlation values were <br /> sequences E. coli strains. The mutations generated <br />                                                                              used study. <br /> these variable sites originated stress, under <br /> favorable conditions, mixture types of <br /> conditions. The variable sites sequence alignments               Selection statistical methods <br /> the results mutations ancestral sequences. Therefore,             Wilcoxon signed-rank test Wilcoxon rank-sum test are <br /> TDMI values ancestral sequences                     nonparametric statistical methods alternatives paired <br /> compared conservation data validation process.               Student&#8217;s t test Student&#8217;s t test, respectively. Nonparametric <br /> However, TDMI values calculated sequence                statistical methods robust used normal <br /> an extant E. coli strain. Therefore, TDMI adequate           distribution assumed. To compare difference of <br /> present analysis.                                                            groups, Wilcoxon signed-rank test Wilcoxon rank-sum test <br />                                                                              used. We used paired comparisons (and Wilcoxon <br />                                                                              signed-rank test) possible. For example, Figure 3C, the <br /> Calculation MI values <br />                                                                              Wilcoxon signed-rank test used protein-coding <br />    We initially calculated MI values according method <br />                                                                              sequence 10 codons, paired comparison the <br /> described Wright et al. [28] using <span id='am-50' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-51' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53'  ></span><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>hybrid</span>-ss-min program of <br />                                                                              start codons ith codons (2#i#10) inside protein- <br /> the UNAFold package [33], local program                coding sequences possible. Conversely, Figure 3E, the <br /> DINAMelt web server, version 3.6 [34], used Wright et al. The             Wilcoxon rank-sum test used protein-coding <br /> calculation MI average ssDNA length 30 nt                  sequences did 10 ATG codons paired comparisons <br /> described here. The protein-coding sequences extended                possible protein-coding sequences. <br /> include 29-nt (that [30&#8211;1]-nt) upstream downstream                   All statistical analyses performed using R statistical <br /> sequences. Subsequences generated sliding window              package [35]. All rankings increasing <span id='am-19' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-20' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span>. <br /> of 30 nt resulting extended sequence. Consequently, <br /> each individual base belonged 30 subsequences. All subse- <br /> quences folded using hybrid-ss-min program [33] <br />                                                                              Supporting Information <br /> and DG values paired/unpaired status stable              Figure S1 Schematic representation synonymous codon <br /> folds given subsequence determined. The MI value                shuffling. (A) An imaginary sequence encoding amino acids. <br /> for base calculated using equation proposed                  The start codon stop codon excluded shuffling. The <br /> Wright et al. [28]: (number folds base unpaired)/         arrows indicate random shuffling. The positions synonymous <br /> 306(highest &#8211;DG 30 folds base unpaired). A            codons (encoding amino acid) shuffled randomly. <br /> small proportion MI values (0.4%) smaller zero              This shuffling repeated 20 amino acids. In the <br /> the factor, &#8216;number folds base unpaired&#8217;,       example sequence, codon serine, self- <br /> greater zero second factor, &#8216;highest &#8211;DG n folds         shuffled. (B) All control sequences generated sequence <br /> in base unpaired&#8217;, smaller zero. However,              (A). The control sequences include protein-coding <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        7                              May 2010 | Volume 5 | Issue 5 | e10567 <br />                                                                                                                                                Effective Nonrandom Mutations <br />  <br />  <br /> sequence synonymous-codon-shuffled sequences happen                                 stable secondary structures ssDNA sequences that <br /> include sequence protein-coding sequence.                                     contain mutated site. Each underlined sequence 30 nt. (B) <br /> Found at: doi:10.1371/journal.pone.0010567.s001 (0.01 MB                                      The sequence underlined line (A). (C) The resulting <br /> PDF)                                                                                          sequence G-to-C single-base substitution base marked <br /> Figure S2 Example calculating SMI arbitrary MI values.                                star sequence shown (B). The secondary structures <br /> (A) MI values imaginary protein-coding sequence                                (B) (C) indicate stable secondary structures these <br /> control sequences. The control sequences generated                                  sequences. In example, G-to-C mutation changes most <br /> synonymous codon shuffling. The z-score values MI                                    stable secondary structure paired/unpaired state <br /> arbitrarily assigned demonstration. Base groups START,                                2DG value bases, determine MI values. <br /> O, M, S, STOP represent start codon, Baseother,                                       Found at: doi:10.1371/journal.pone.0010567.s003 (0.54 MB TIF) <br /> Basemissense, Basesilent, stop codon, respectively. (B) The                           Figure S4 Correlation MI values single-base <br /> average z-scores MI individual base groups individual                          substitutions. The correlations MI values after <br /> sequences. For example, Basesilent protein-coding sequence                             mutation plotted. The MI values divided 10 groups <br /> has z-scores MI values 2 2.01, 2.95, 0.42 (A),                           according rank single-base <br /> its average value (0.45) written corresponding value (B).                        substitution. The diameter circle proportional the <br /> (C) The z-score average MI protein-coding sequence                             number division. <br /> ranked corresponding values control sequences.                              Found at: doi:10.1371/journal.pone.0010567.s004 (0.57 MB TIF) <br /> The rank values (1#rank#N+1) obtained linearly trans- <br /> formed SMI (2 1#SMI#1) using equation (26rank 2 2 2 <br /> N)/N, N number control sequences (which four <br />                                                                                               Acknowledgments <br /> in example 100 main text). In example,                              We thank Dr Rintaro Saito Keio University, Japan, reviewing the <br /> control sequences used clarity values                                   manuscript commenting statistical aspects. We thank Dr <br /> rounded nearest decimal places, used 100                               Kochiwa Hiromi Dr Douglas Murray Keio University, Japan, for <br /> control sequences calculated 15 decimal places                             comments manuscript. <br /> actual calculation main text. <br /> Found at: doi:10.1371/journal.pone.0010567.s002 (0.34 MB                                      Author Contributions <br /> PDF)                                                                                          Conceived designed experiments: HK. Performed experiments: <br /> Figure S3 An example effect single-base substitution                              HK. Analyzed data: HK BSL AK. Wrote paper: HK BSL MT AK. <br /> on secondary structure. (A) A si <br /> </body></html>