<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Dec. 2009, p. 7456&#8211;7465                                                                             Vol. 191, No. 24 <br /> 0021-9193/09/$12.00 doi:10.1128/JB.00963-09 <br /> Copyright &#169; 2009, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />                    An SOS-Regulated Type 2 Toxin-Antitoxin System&#17188; <br />           Larissa A. Singletary,1 Janet L. Gibson,1 Elizabeth J. Tanner,1&#8224; Gregory J. McKenzie,1&#8225; <br />                      Peter L. Lee,1&#8224; Caleb Gonzalez,1 Susan M. Rosenberg1,2,3,4* <br />   Departments Molecular Human Genetics,1 Biochemistry Molecular Biology,2 Molecular Virology Microbiology,3 <br />                   Dan L. Duncan <span id='am-52' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-56'  ></span>Cancer</span> Center,4 Baylor College Medicine, Houston, Texas 77030-3411 <br />                                               Received 22 July 2009/Accepted 1 October 2009 <br />  <br />              The <span id='am-124' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-125' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-133'  ></span><span id='am-134' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-136' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span>Escherichia coli</span> chromosome encodes seven demonstrated type 2 toxin-antitoxin (TA) systems: cassettes <br />           cotranscribed genes, encoding stable toxin protein cause cell stasis death, <br />           encoding labile antitoxin protein, regulatory protein. We demonstrate the <br />           yafNO genes constitute additional chromosomal type 2 TA upregulated SOS DNA <br />           damage response. The yafNOP genes dinB operon, dinB underlies stress-induced <br />           mutagenesis mechanisms. yafN identified putative antitoxin homology known antitoxins, <br />           implicating yafO (and/or yafP) putative toxin. Using phage-mediated cotransduction assays linkage <br />           disruption, yafN essential gene second essential yafO present. <br />           Third, yafP necessary toxin antitoxin. Fourth, DinB required, the <br />           yafNOP genes required stress-induced mutagenesis <span id='am-140' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-141' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-145'  ></span><span id='am-146' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia</span> coli Lac assay. These results <br />           imply yafN encodes antitoxin protects cells yafO-encoded toxin protein-based <br />           TA upregulated SOS response. <br />  <br />  <br />    Toxin-antitoxin (TA) systems modules bacterial ge-                 demonstrated E. coli, including relBE, mazEF, <br /> nomes cause growth arrest and/or programmed cell                    dinJ-yafQ, prlF-yhaV, yefM-yoeB, chpBI-chpBK, hipAB <br /> death cells harboring (1, 18, 19, 54). Type 1 TA                     (1, 3, 19, 20, 39, 45, 50). Of these, relBE mazEF exert <br /> systems consist RNA antitoxin protein toxin,            toxic effects controlled responses during <br /> the RNA antitoxin inhibits translation toxin mRNA.                    amino acid starvation (8, 28). Additionally, mazF kills cells <br /> Type 2 TA systems typically consist genes operon,               response DNA-damaging oxidative stresses (28). <br /> transcriptionally translationally coupled, which, usually,            One type 2 (dinJ-yafQ) (45) characterized type 1 <br /> the upstream gene encodes labile antitoxin protein                 (hok, symER, tisAB-istr1) (31, 46, 55) TA systems were <br /> downstream gene encodes stable toxin protein (Fig. 1A).                    previously reported controlled SOS response to <br /> Continuous transcription operon, continuous                  DNA damage. However, dinJ-yafQ appears SOS <br /> transcription antitoxin, ensures protection ef-              regulated. Although consensus LexA binding site was <br /> fects toxin. Interruption transcription operon              identified upstream dinJ-yafQ operon (35) LexA <br /> tips balance favor toxin antitoxin no <br />                                                                              bound site study (56) (14), <br /> longer rapidly degraded. <br />                                                                              LexA does regulate operon vivo exposure to <br />    TA systems originally identified <span id='am-204' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-205' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-206' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-207' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-208' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span> &#8220;plas- <br />                                                                              mitomycin C (14) UV light (11). The role stress re- <br /> mid-addiction modules&#65533;? (34) maintain plasmid the <br />                                                                              sponses regulating TA systems interesting problem. <br /> host cell. Failure cells inherit plasmid results rapid <br />                                                                              Here provide evidence E. coli chromosomal <br /> loss labile antitoxin, unmasking stable preexisting <br /> toxin, death cell process known postsegre-                type 2 TA SOS response control, yafNO. <br /> gational killing (13, 38).                                                      yafNOP originally identified putative TA by <br />    More recently, TA systems discovered                     homology yafN known antitoxins, implicating yafO <br /> Escherichia coli chromosome, raising question                   (and/or yafP) putative toxin(s) (19). yafNOP lies down- <br /> function (1, 19). Proposed functions chromosomal                   stream dinB dinB operon (43) (Fig. 1B), is <br /> TA systems include roles nutritional stress response (8),                 upregulated midway SOS DNA damage response <br /> protection phages (27), formation &#8220;persister&#65533;? cells                  (11). DinB central player stress-induced mutagenesis <br /> that resist antibiotics (32), selfish genetic elements (38, 54),             mechanisms E. coli (9, 17, 42), <span id='am-147' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-148' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-149'  ></span><span id='am-150' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-158' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Salmonella</span> enterica (48), <br /> and antiaddiction modules allow bacteria resist plas-                <span id='am-77' about='obo:NCBITaxon_303' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_286 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621 obo:NCBITaxon_136845'><span id='am-78' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas rugosa" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="PMID:11211254" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas incognita" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="PMID:1177780" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasAlternativeId" content="NCBITaxon:105847" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas ovalis" datatype="xsd:string"></span><span id='am-84'  ></span><span id='am-85' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas eisenbergii" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasAlternativeId" content="NCBITaxon:29437" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasAlternativeId" content="NCBITaxon:760256" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="Bacillus fluorescens putidus" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasAlternativeId" content="NCBITaxon:668614" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="Arthrobacter siderocapsulatus" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="Pseudomanas putida" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas arvilla" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Pseudomonas putida" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas striata" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasAlternativeId" content="NCBITaxon:79218" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasAlternativeId" content="NCBITaxon:691260" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="PMID:17978216" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931954" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1416793" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasRelatedSynonym" content="Bacillus putidus" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas sp. PX-A-Ps.sp2" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasAlternativeId" content="NCBITaxon:126995" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="PMID:9734035" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931964" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas convexa" datatype="xsd:string"></span>Pseudomonas putida</span> (53), <span id='am-1' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-2' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span>Bacillus subtilis</span> (52). The prox- <br /> mid addiction (54). Several type 2 chromosomal TA systems                    imity yafNOP dinB piqued genes. <br />                                                                              Previously, Brown Shaw demonstrated classical TA sys- <br />                                                                              tem dissection yafO overexpressed growth inhi- <br />   * Corresponding author: Mailing address: Department Molecular           bition occurred (5). No additional work regarding yafN <br /> and Human Genetics, Baylor College Medicine, One Baylor Plaza,            yafP conducted (5). In contrast to notion <br /> Room S809A, Mail Stop BCM225, Houston, TX 77030-3411. Phone:                 yafNOP TA published yafN deletion <br /> (713) 798-6924. Fax: (713) 798-8967. E-mail: smr@bcm.edu. <br />                                                                              strain Keio E. coli knockout collection, should <br />   &#8224; Present address: Stanford University, Palo Alto, CA 94305. <br />   &#8225; Present address: Verenium Corporation, San Diego, CA 92121.              inviable yafN antitoxin (2). Given these <br />   &#17188; <br />     Published ahead print 16 October 2009.                             conflicting results, reinvestigated yafNOP. Here, show <br />  <br />                                                                       7456 <br />  VOL. 191, 2009                                                                      AN SOS-REGULATED TOXIN-ANTITOXIN SYSTEM                                   7457 <br />  <br />                                                                                    1:4 dilution. Phage added multiplicity infection 10 cell, <br />                                                                                    allowed sit room temperature 10 minutes, diluted 10&#11003; TB, shaken <br />                                                                                    2 hours 32&#176;C, plated MacConkey-maltose plates seeded with <br />                                                                                    108 109 &#65533;?&#65533;cI 108 109 &#65533;?&#65533;cIh80 phage kill nonlysogens, red (maltose- <br />                                                                                    positive) colonies picked, purified, verified. &#65533;?&#65533;cIh80 adsorbs an <br />                                                                                    alternate receptor kills nonlysogens mutated maltose uptake <br />                                                                                    receptors, resistant wild-type &#65533;?&#65533; adsorption. <br />                                                                                       Cloning yafN pBAD24. yafN amplified strain FC36 using <br />                                                                                    primers yafN-F-KpnI (AAAAAAGGTACCATTCTGGTGTGCATTATTATG) <br />                                                                                    yafN-R-HindIII (AAAAAAAAGTTCTTATTCCTTAAAGTCATTG) with <br />                                                                                    Platinum Pfx polymerase (Invitrogen) cloned pCR-BluntII-Topo-Kanr <br />                                                                                    using Invitrogen&#8217;s Zero Blunt Topo PCR cloning kit. yafN excised the <br />                                                                                    cloning plasmid KpnI XhoI ligated pBAD24 using T4 DNA <br />                                                                                    ligase (Invitrogen), creating <span id='am-34' about='obo:NCBITaxon_2595' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384 obo:NCBITaxon_36549'><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Plasmid pLS1" datatype="xsd:string"></span>plasmid pLS1</span>. Insertion yafN pBAD24 <br />                                                                                    confirmed sequencing (SeqWright, Houston, TX). <br />                                                                                       Quantitative phage P1-mediated cotransduction assays. Phage P1-mediated <br />                                                                                    cotransductions performed based Miller protocol (44) the <br />                                                                                    following modifications. Transductants selected LBH-Tet 20 mM CIT <br />                                                                                    patched 100-colony grids LBH-Kan-CIT (and LBH-chloram- <br />                                                                                    phenicol-CIT &#9004;yafN &#9004;yafP cotransduction) screen recovery markers <br />                                                                                    interest. Plates monitored days incubation possible <br />                                                                                    slowly growing colonies. <br />                                                                                       Sequencing JW0222 yafO. The yafO gene strain JW0222 amplified <br />                                                                                    using PCR primers yafN-F-KpnI yafPExtR (AGTTTGTGGAATCAGAAA <br />                                                                                    ACG). PCR product purified Qiagen&#8217;s QIAquick PCR purification kit <br />                                                                                    sequenced using above-mentioned primers (SeqWright, Houston, TX). <br />                                                                                       Mutagenesis DNA damage sensitivity assays. Stress-induced mutagenesis <br />                                                                                    assays performed according method Harris et al. (25). For UV <br />    FIG. 1. Type 2 TA organization dinB operon.                      light kill curves, cultures grown early/mid-log phase (optical density at <br /> (A) Type 2 TA organization. Type 2 TA systems usually               600 nm, 0.1 0.3) LBH spread LBH agar plates, exposed the <br /> gene operons antitoxin (upstream) toxin (down-                    indicated UV light dose, incubated dark 37&#176;C 16 h, scored for <br /> stream) genes cotranscribed/translated. The antitoxin protein               colonies. For bleomycin mitomycin sensitivity assays, saturated LBH cultures <br /> degraded rapidly toxin, cell requires                  diluted grown mid-log phase (optical density 600 nm, 0.3 0.6) <br /> continuous transcription/translation avoid stasis/death induced              spotted fresh drug-containing plates indicated dilutions. Bleomycin <br /> the toxin. (B) The E. coli dinB operon. The yafNOP genes, encoding               dissolved 0.9% NaCl. Mitomycin C dissolved sterile H2O (final pH, <br /> putative TA system, lie downstream dinB operon. The operon               6.0), manufacturer&#8217;s recommendation. Plates incubated 16 h at <br /> is upregulated SOS DNA damage response (SOS box shown).                     32&#176;C scored CFU. <br /> yafN, encoding putative antitoxin, 297 bp; yafO, encoding                  Lambda burst size assay UV-induced lysogens. Saturated LBH cultures <br /> putative toxin, 399 bp; yafP, unknown function, 453 bp             E. coli &#65533;?&#65533; lysogens diluted K-glucose grown 1.5 &#11003; 108 cells/ml. <br /> length. Some strains used study (derivatives SMR4562                 An aliquot cells removed, lysed lysozyme chloroform, and <br /> FC40) contain additional copy dinB operon duplication              submitted plaque assay quantify phage spontaneously induced <br /> of operon F&#11032; episome. T, toxin; AT, antitoxin.                          prophage state. Cells UV irradiated (50 J/m2) induce the <br />                                                                                    lambda prophage lytic growth shaken &#11350;300 rpm 1.5 h 37&#176;C in <br />                                                                                    darkness. Lysozyme (50 mg/ml) chloroform added lyse cells and <br />                                                                                    release intracellular phage, debris pelleted, supernatants were <br /> that yafN yafO (not yafP) constitute TA                           assayed PFU. Phage titers calculated used determine burst size <br /> explore possible function cell.                                         (PFU/cell, corrected preexisting phage). <br />                                                                                       Flow cytometry cell sorting. Flow cytometry analyses fluorescence- <br />                                                                                    activated cell sorting (FACS) performed method reference 47 <br />                        MATERIALS AND METHODS                                       following modifications. Log-phase cells analyzed using BD <br />    Media, antibiotics, growth conditions. The media used included <span id='am-39' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Luria" datatype="xsd:string"></span>Luria</span>-       FACSAria cell sorter (BD Biosciences). <span id='am-189' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-190' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-191' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-192' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-193'  ></span>Data</span> shown means five <br /> Bertani-Herskowitz (LBH; 1% tryptone, 0.5% NaCl, 0.5% yeast extract, 2 &#65533;?&#65533;g/ml       experiments (one culture/experiment experiments 1 4 cultures/ <br /> thymine), tryptone broth (TB; 1% tryptone, 0.5% NaCl), BBL plates (0.5%        experiment experiment 5). At 106 nongreen 8 &#11003; 103 (typically &#11022;104) <br /> NaCl, 1% BBL Trypticase peptone solidified 1.5% agar). K-glucose consists     green cells sorted culture. Sorting purity controls performed <br /> of 2&#11003; M9 (44) following additions: 7.5% Casamino Acids, 5 mM              experiment (per method reference 47). Propidium iodide (PI) <br /> MgSO4, 0.05% NaCl, 0.2% glucose, 10 &#65533;?&#65533;g/ml vitamin B1, 3 &#65533;?&#65533;M FeCl3, 50           staining performed method reference 47. <br /> &#65533;?&#65533;M CaCl2. Plating culture broth TB following additions: 0.2% mal- <br /> tose, 5 mM MgSO4, 10 &#65533;?&#65533;g/ml thymine, 10 &#65533;?&#65533;g/ml vitamin B1. Carbon sources <br /> glucose, maltose, glycerol, lactose used 0.1% unless                                      RESULTS <br /> stated. The antibiotics used included tetracycline (Tet; 10 &#65533;?&#65533;g/ml, 3.33 &#65533;?&#65533;g/ml <br /> when sodium citrate [CIT] present), kanamycin (Kan; 30 &#65533;?&#65533;g/ml), chloram-            yafN essential presence yafO. To test whether <br /> phenicol (25 &#65533;?&#65533;g/ml), bleomycin (various concentrations, specified text),    yafNOP functional TA system, examined the <br /> mitomycin C (various concentrations, specified text), rifampin (100     putative antitoxin gene, yafN, essential gene in <br /> &#65533;?&#65533;g/ml). 5-Bromo-4-chloro-3-indolyl-&#65533;?&#65533;-D-galactoside (X-Gal; 40 &#65533;?&#65533;g/ml) also <br />                                                                                    presence functional putative toxin gene, yafO. As <br /> used. The buffers used included TM (Tris buffer &#11001; MgSO4, pH 7.5) M9 (44). <br /> Bacteria grown 32&#176;C 37&#176;C, stated experiment.                diagrammed Fig. 2A, using cotransduction assay, trans- <br />    Strains new alleles. E. coli K-12 strains plasmids used study   duced E. coli cells phage P1 grown cells <br /> are listed Table 1. New deletion alleles created study shown    carrying zae-502::Tn10 Tet resistance (Tetr) gene partially <br /> Table 2, PCR primers used construct new deletions short-         linked chromosomal &#9004;yafN::Kan deletion, conferring <br /> homology recombination methods (12) shown Table 3. Phage P1-mediated <br /> transductions performed described previously (44). To construct &#65533;?&#65533; lyso- <br />                                                                                    Kan resistance (Kanr), mutation interest. The P1 <br /> gens, stationary-phase TB cultures nonlysogens diluted 200&#11003; plating   donor cells carried yafN&#11001;-expressing plasmid allow <br /> culture broth, shaken 6 hours 32&#176;C, pelleted, resuspended TM    viability (if essential) prevent accumulation/ <br />  7458        SINGLETARY ET AL.                                                                                                                 J. BACTERIOL. <br />  <br />  <br />  <br />                                       TABLE 1. E. coli strains, plasmids, lambda phage used studya <br />       Strain, phage, <br />                                                      Relevant genotype                                               Reference(s) source <br />        plasmid <br />  <br /> Strains <br />   BT340                    DH5&#65533;?&#65533;(pCP20)                                                              7 (via CGSC) <br />   BW25113(pKD46)           pKD46                                                                    12 (via CGSC) <br />   BW25141(pKD3)            pKD3                                                                     12 (via CGSC) <br />   BW25141(pKD13)           pKD13                                                                    12 (via CGSC) <br />   CAG18436                 MG1655 zae-502::Tn10                                                     51 <br />   DH5&#65533;?&#65533;                     endA1 hsdR17 (rK&#11002; mK&#11001;) supE44 thi-1 &#65533;?&#65533;&#11002; recA1 gyrA96 relA1                21 (via W. Bridger &#20851;Edmonton, <span id='am-72' about='obo:NCBITaxon_73950' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_4055 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_24966 obo:NCBITaxon_3398 obo:NCBITaxon_169639 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_169618 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-75'  ></span><span id='am-76' property="rdfs:label" content="Alberta" datatype="xsd:string"></span>Alberta</span>, Canada&#20852;) <br />                               deoR &#9004;(lacZYA-argF)U169 &#65533;?&#65533;80dlacZ&#9004;M15 <br />   FC36                     &#9004;(lac-proAB)XIII thi <span id='am-57' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-60'  ></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>ara</span> Rifr                                            6 <br />   FC40                     &#9004;(lac-proAB)XIII thi ara Rifr &#20851;F&#11032; proAB&#11001; lacI33&#65533;?&#65533;lacZ&#20852;                    6 <br />   JW0222                   &#9004;yafN::FRTKanFRT                                                         2 <br />   MG1655                   Wild type                                                                4 <br />   N2731                    recG258::Tn10 mini-Kan                                                   36 (via R. Kolodner &#20851;San Diego, CA&#20852;) <br />   SMR4562                  Genotype FC40, independent construction                  41 <br />   SMR5833                  SMR4562(pKD46)                                                           43 <br />   SMR5889                  SMR4562 &#9004;dinB50::FRT &#20851;F&#11032; &#9004;dinB50::FRT&#20852;                                   43 <br />   SMR6068                  SMR4562 &#9004;yafN11::FRTKanFRT &#20851;F&#11032; yaf&#11001;&#20852;                                     SMR5833 &#11003; DNA &#9004;yafN::FRTKanFRT <br />                                                                                                       amplified pKD13 &#20851;from BW25141 <br />                                                                                                       (pKD13)&#20852;, location screened mating <br />   SMR6074                  SMR4562 &#20851;F&#11032; &#9004;yafO14::FRTKanFRT&#20852;                                          SMR5833 &#11003; DNA &#9004;yafO::FRTKanFRT <br />                                                                                                       amplified pKD13 &#20851;from BW25141 <br />                                                                                                       (pKD13)&#20852;, location screened mating <br />   SMR6076                  FC36 &#9004;yafO14::FRTKanFRT                                                  FC36 &#11003; P1(SMR6074) <br />   SMR6080                  SMR4562 &#9004;yafP18::FRTKanFRT &#20851;F&#11032; yaf&#11001;&#20852;                                     SMR5833 &#11003; DNA &#9004;yafP::FRTKanFRT <br />                                                                                                       amplified pKD13 &#20851;from BW25141 <br />                                                                                                       (pKD13)&#20852;, location screened mating <br />   SMR6082                  FC36 &#9004;yafP18::FRTKanFRT                                                  FC36 &#11003; P1(SMR6080) <br />   SMR6221                  FC36 &#9004;yafP20::FRT                                                        SMR6082 &#11003; pCP20 (from BT340) <br />   SMR6233                  MG1655(pKD46)                                                            MG1655 &#11003; pKD46 &#20851;from BW25113(pKD46)&#20852; <br />   SMR6353                  MG1655 &#9004;(yafN-yafP)776::FRTKanFRT                                        SMR6233 &#11003; DNA &#9004;(yafN-yafP)::FRTKanFRT <br />                                                                                                        pKD13 &#20851;from BW25141(pKD13)&#20852; <br />   SMR6669                  MG1655 &#9004;att&#65533;?&#65533;::PsulA&#65533;?&#65533;gfp-mut2                                             26, 40 <br />   SMR7491                  SMR4562 &#9004;(yafN-yafP)602 &#20851;F&#11032; &#9004;(yafN-yafP)602&#20852;                             43 <br />   SMR10285                 SMR4562 &#9004;(yafN-yafO)779::FRTKanFRT &#20851;F&#11032; yaf&#11001;&#20852;                             SMR5833 &#11003; DNA &#9004;(yafN-yafO)::FRTKanFRT <br />                                                                                                       amplified pKD13 &#20851;from BW25141 <br />                                                                                                       (pKD13)&#20852; <br />  SMR10287                  FC36 &#9004;(yafN-yafO)779::FRTKanFRT                                          FC36 &#11003; P1(SMR10285) <br />  SMR10291                  SMR4562 &#9004;yafN11::FRTKanFRT zae-502::Tn10 &#20851;F&#11032; yaf&#11001;&#20852;                       SMR6068 &#11003; P1(CAG18436) <br />  SMR10483                  FC36 &#9004;(yafN-yafO)779::FRTKanFRT zae-502::Tn10                            SMR10287 &#11003; P1(CAG18436) <br />  SMR10943                  FC36(pBAD24)                                                             FC36 &#11003; pBAD24 (from ATCC 87399) <br />  SMR10946                  DH5&#65533;?&#65533;(pLS1)                                                               DH5&#65533;?&#65533; &#11003; pLS1 (initial plasmid construction) <br />  SMR10948                  FC36(pLS1)                                                               FC36 &#11003; pLS1 (from SMR10946) <br />  SMR10953                  SMR4562 &#9004;yafP18::FRTKanFRT zae-502::Tn10 &#20851;F&#11032; yaf&#11001;&#20852;                       SMR6080 &#11003; P1(CAG18436) <br />  SMR10988                  MG1655 &#9004;att&#65533;?&#65533;::PsulA&#65533;?&#65533;gfp-mut2 &#9004;(yafN-yafP)776::FRTKanFRT                  SMR6669 &#11003; P1(SMR6353) <br />  SMR10990                  MG1655 recG258::Tn10 mini-Kan                                            MG1655 &#11003; P1(N2731) <br />  SMR10995                  SMR4562 &#9004;yafN11::FRTKanFRT &#20851;F&#11032; yaf&#11001;&#20852; &#20851;pKD46&#20852;                             SMR6068 &#11003; pKD46 &#20851;from BW25113(pKD46)&#20852; <br />  SMR11008                  SMR4562 &#9004;yafN11::FRTKanFRT &#9004;yafP19::FRTcatFRT zae-502::                  SMR11168 &#11003; P1(CAG18436) <br />                              Tn10 &#20851;F&#11032; yaf&#11001;&#20852; <br />   SMR11168                 SMR4562 &#9004;yafN11::FRTKanFRT &#9004;yafP19::FRTcatFRT &#20851;F&#11032; yaf&#11001;&#20852;                  SMR10995 &#11003; DNA &#9004;yafP::FRTcatFRT <br />                                                                                                      amplified pKD3 &#20851;from BW25141(pKD3)&#20852; <br />   SMR11228                 MG1655(&#65533;?&#65533;)                                                                MG1655 &#11003; &#65533;?&#65533;SR108 <br />   SMR11229                 MG1655(&#65533;?&#65533;) &#9004;(yafN-yafP)776::FRTKanFRT                                     SMR11228 &#11003; P1(SMR6353) <br />   SMR11231                 MG1655(&#65533;?&#65533;) recG258::Tn10 mini-Kan                                         SMR11228 &#11003; P1(N2731) <br />  <br /> Phage lambda <br />   &#65533;?&#65533;SR108                   &#65533;?&#65533; wild type                                                              F. Stahl (Oregon) <br />  <br /> Plasmids <br />   pBAD24                   Plasmid containing arabinose-inducible promoter PBAD, Ampr               23 (via ATCC 87399) <br />   pLS1                     Plasmid containing yafN control PBAD                        <span id='am-62' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-64'  ></span><span id='am-65' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> work <br />   pKD3                     cat-containing plasmid, template PCR                                 12 <br />   pKD13                    kan-containing plasmid, template PCR                                 12 <br />   pCP20                    FLP-containing plasmid, used excise drug markers flanked           12 <br />                              FRT sequences <br />   pKD46                    Red recombinase expression plasmid                                       12 <br />  a <br />      FRT, FLP recombination target; CGSC, E. coli Genetic Stock Center (Yale University); ATCC, American Type Culture Collection. <br />  VOL. 191, 2009                                                                         AN SOS-REGULATED TOXIN-ANTITOXIN SYSTEM                          7459 <br />  <br />  <br />                                                         TABLE 2. Description new deletion alleles <br />                                                                                                              Deletion coordinates (distance     Replacement <br />       Strain                        New allele                         Wild-type gene length (bp) <br />                                                                                                               gene translation start)       cassettea <br />  <br /> SMR6068                 &#9004;yafN11::FRTKanFRT                                       294                         &#11001;13   &#11001;279                    FLPable Kan <br /> SMR6074                 &#9004;yafO14::FRTKanFRT                                       399                         &#11001;13   &#11001;384                    FLPable Kan <br /> SMR6080                 &#9004;yafP18::FRTKanFRT                                       453                         &#11001;16   &#11001;438                    FLPable Kan <br /> SMR6221                 &#9004;yafP20::FRT                                             453                         &#11001;16   &#11001;438                    FRT scar <br /> SMR10285                &#9004;(yafN-yafO)779::FRTKanFRT                   See individual genes              &#11001;13   (yafN) &#11001;384 (yafO)      FLPable Kan <br /> SMR11168                &#9004;yafP19::FRTcatFRT                                       453                         &#11001;40   &#11001;416                    FLPable cat <br />   a <br />     For replacement cassette, sequence inserted DNA begins ATCC (Kan cassette) TCATA (cat cassette) ends CTACA (Kan cassette) <br /> or TACAC (cat cassette). Sequences listed 5&#11032; 3&#11032;. FRT, FLP recombination target. <br />  <br />  <br />  <br />  <br /> selection extragenic suppressor mutations al-                           We hypothesized viable, strain carry a <br /> low cell mutated essential gene grow. We initially                        spontaneous mutation inactivating yafO alternatively du- <br /> selected neutral Tetr marker screened                                plication operon copies yafN <br /> colonies Kan-replaced gene (either yafN,                          deleted. Such mutants likely responsible <br /> yafNO, yafP, yafN yafP simultaneously). The expected                           0.16% frequency cotransduction seen &#9004;yafN:: <br /> frequency cotransduction markers (Kanr                            Kan zae-502::Tn10 cotransduction (Fig. 2B). We sequenced <br /> Tetr) approximately 30% based physical distance.                          yafO gene strain JW0222 report yafO gene <br /> Cotransduction frequencies &#11011;30% imply gene                             contains single base pair deletion guanine bp 7, which, <br /> interest essential, cotransduction frequencies                         frameshift, creates early stop <span id='am-67' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-68'  ></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> (TAA) <br /> that significantly lower imply reduced viability trans-                        bp 61 399-bp yafO gene (Fig. 2D). The resultant YafO <br /> ductants received deletion, is, gene                           protein prematurely truncated likely nonfunc- <br /> deleted essential.                                                                 tional. <br />    We &#9004;yafN::Kan zae-502::Tn10 cells cotrans-                            YafP toxin antitoxin component. We <br /> duced 0.16% &#11006; 0.16% Tetr transductants (Fig. 2B).                          sought understand yafP integral function <br /> These data imply yafN essential gene. Supporting                      antitoxin toxin. If yafP integral part <br /> interpretation, loss chromosomal copy                         antitoxin, it, like yafN, essential viability in <br /> yafN transduction achieved efficiently (23% &#11006;                             presence yafO. We &#9004;yafP::Kan zae-502::Tn10 <br /> 2.7% &#9004;yafN::Kan zae-502::Tn10 cotransductants) recipient                           cotransduced efficiently 20% &#11006; 1.8% strain without <br /> cells carrying yafN-containing plasmid (pLS1) (Fig. 2B)                         additional plasmid-borne yafP gene 26% &#11006; 2% into <br /> not cells carrying plasmid vector (Fig. 2B).                        strains harboring extra yafP gene (Fig. 3A). Thus, yafP not <br /> We conclude yafN essential gene.                                           essential presence yafO YafP an <br />    If yafNO(P) genes TA system, yafN                         integral antitoxin. <br /> expected essential putative toxin, YafO,                         If yafP integral toxin, loss yafP <br /> present. We &#9004;(yafN-yafO)::Kan zae-502::Tn10                           expected allow efficient cotransduction &#9004;yafN:: <br /> cells transduced, frequency cotransduction                     Kan linked Tetr marker yafN longer <br /> markers Tetr 29% &#11006; 5% (Fig. 2C). These data                        required viability. We frequency co- <br /> yafN essential YafO functional support                       transduction double deletion &#9004;yafN::Kan &#9004;yafP::cat with <br /> interpretation yafN yafO TA system.                                    linked Tetr marker &#65533;?0.33% cells extra copy <br />    In contrast results, published yafN dele-                      yafN gene efficient cells carrying yafN <br /> tion strain, JW0222, E. coli Keio deletion collection (2).                     expression plasmid pLS1 (Fig. 3B), indicating yafN re- <br />  <br />  <br />                                                     TABLE 3. Linear replacement gene deletion primers <br />                        Linear replacement <br />         Strain                                                                                Sequencea                                             Template <br />                           primer name <br />  <br /> SMR6068                   yafNwL               tgtatattctggtgtgcattattatgagggtatcactgtatgcatcgaattATTCCGGGGATCCGTCGACC                               pKD13 <br />                           yafNwR               gctgtaagttgcaggcgaataagttttgttttgaatacccgcatccttattccttaaagtcTGTAGGCTGGAGCTGCTTC <br /> SMR6074                   yafOwL               tatgacggatgatgatttcaatgactttaaggaataaggatgcgggtattcATTCCGGGGATCCGTCGACC                               pKD13 <br />                           yafOwR               atagtttcttatttgtatgttattcataatataaattcaaaaacgcatgcgTGTAGGCTGGAGCTGCTTC <br /> SMR6080/6221              yafPwL               gcagaagcgtttcgcatgcgtttttgaatttatattatgaataacatacaaATTCCGGGGATCCGTCGACC                               pKD13 <br />                           yafPwR               ataccaggcgggcgttattttcattgcaagctggatttaatgttgcggtttTGTAGGCTGGAGCTGCTTC <br /> SMR6353                   yafNwL               tgtatattctggtgtgcattattatgagggtatcactgtatgcatcgaattATTCCGGGGATCCGTCGACC                               pKD13 <br />                           yafPwR               ataccaggcgggcgttattttcattgcaagctggatttaatgttgcggtttTGTAGGCTGGAGCTGCTTC <br /> SMR10285                  yafNwL               tgtatattctggtgtgcattattatgagggtatcactgtatgcatcgaattATTCCGGGGATCCGTCGACC                               pKD13 <br />                           yafOwR               atagtttcttatttgtatgttattcataatataaattcaaaaacgcatgcgTGTAGGCTGGAGCTGCTTC <br /> SMR11168                  yafP::CAT-L          atgaataacatacaaataagaaactatcagcctggcgatTCATATGAATATCCTCCTTAG                                          pKD3 <br />                           yafP::CAT-R          atgttgcggtttatatcgcatataaaaattagtaaacGTGTAGGCTGGAGCTGCTTC <br />   a <br />       Lowercase letters refer chromosomal sequence; uppercase letters refer template sequence. Sequences listed 5&#11032; 3&#11032;. <br />  7460      SINGLETARY ET AL.                                                                                                               J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 2. Cotransduction assay quantitative determination inviability various mutant strains. (A) The assay design. Phage P1 grown strains <br /> containing gene knockouts (Kanr) known linkage nonlethal Tetr marker, zae-502::Tn10, transduced recipient cells and <br /> selected Tet. Colonies patched Kan plates determine frequency cotransduction markers, determine gene of <br /> interest essential. If gene essential, frequency cotransduction Kanr gene deletion Tetr transductants lower that <br /> predicted distance markers. (B) yafN essential. The &#9004;yafN::Kan deletion cotransduced efficiently zae-502::Tn10 cells <br /> harboring plasmid pLS1, expresses yafN, cells carrying plasmid vector only. P1 donor: &#9004;yafN::Kan zae-502::Tn10 <br /> (SMR10291). Recipient cells: yaf&#11001; (FC36), yaf&#11001;[pBAD] (SMR10943), yaf&#11001;[pLS1] (SMR10948). Average &#11006; 1 SEM experiments. (C) yafN is <br /> essential presence yafO. &#9004;(yafN-yafO)::Kan zae-502::Tn10 cotransduction (P1 donor: SMR10483) yaf&#11001; strain FC36. Average &#11006; 1 SEM <br /> of experiments. (D) The yafO gene mutated yafN deletion strain JW0222 E. coli gene knockout Keio collection. Sequencing the <br /> JW0222 yafO gene revealed &#11002;1-bp deletion bp 7 results premature stop codon bp 61 399-bp gene. <br />  <br />  <br />  <br /> mains essential yafP deleted. To address                      formed &#9004;yafP recipient strain. The frequency cotrans- <br /> possible concern lingering toxin recipient cell                    ductants remained &#65533;?0.33% (Fig. 3B). We conclude yafN <br /> (present prior transduction &#9004;yafP) cause inviability               remains essential despite absence yafP. Therefore, YafP <br /> after cotransduction, additional cotransduction per-                    necessary toxin. <br />  VOL. 191, 2009                                                             AN SOS-REGULATED TOXIN-ANTITOXIN SYSTEM                            7461 <br />  <br />  <br />  <br />  <br />                                                                              FIG. 4. The yafNOP genes required stress-induced mu- <br />                                                                           tagenesis. Lac assay strains starved lactose days and <br />                                                                           monitored day Lac&#11001; reversion mutant colonies according to <br />    FIG. 3. YafP necessary component toxin anti-     method reference 25. (A) Representative experiment. Lac&#11001; <br /> toxin. (A) yafP essential viability   revertants 108 cells plotted time. Average &#11006; SEM five <br /> antitoxin. &#9004;yafP::Kan cotransduced efficiently zae-502::Tn10      cultures. (B) Summary mutation rates multiple experiments. <br /> (P1 donor: SMR10953) recipient cells: yaf&#11001; (FC36) yaf&#11001;[F&#11032;         Mutation rates, calculated according method reference 37, <br /> yaf&#11001;] (SMR4562). Average &#11006; SEM experiments. (B) Loss          Lac&#11001; revertants 108 cells day days 3 5. Mean &#11006; SEM <br /> yafP does allow recovery &#9004;yafN cotransductants         experiments. The yaf&#11001; strain SMR4562, &#9004;yafNOP <br /> expected yafP required toxin function. Cotransduction    strain SMR7491, &#9004;dinB strain SMR5889. <br /> P1 donor &#9004;yafN::Kan &#9004;yafP::cat zae-502::Tn10 (SMR11008) re- <br /> cipient cells: yaf&#11001; (FC36), &#9004;yafP (SMR6221), yaf&#11001;[pBAD] <br /> (SMR10943), yaf&#11001;[pLS1] (SMR10948). Average &#11006; SEM three <br /> experiments.                                                              evidence yafNOP serve protective unit against <br />                                                                           E. coli phage &#65533;?&#65533;. First, burst size &#65533;?&#65533; lytic infection not <br />                                                                           different yaf&#11001; cells infected &#9004;yafNOP <br />                                                                           cells infected; ratio yaf&#11001; &#9004;yafNOP 1.4 &#11006; 0.6 <br />    No role stress-induced mutagenesis. Because dinB,               (mean &#11006; standard error mean [SEM] experi- <br /> first gene dinB-yafNOP operon, key player stress-          ments). Next, test yafNOP genes promote <br /> induced point mutagenesis E. coli Lac (17, 42)              prevent progression &#65533;?&#65533; induction lyso- <br /> and imagine roles TA process,            genic state, transition controlled SOS response (49), <br /> tested possible involvement yafNOP stress-induced lac           spotted various dilutions &#65533;?&#65533; lysogens E. coli yafNOP&#11001; <br /> reversion. We observed yafNOP strain showed slightly             &#9004;yafNOP cells different concentrations bleo- <br /> but significantly higher stress-induced mutagenesis              mycin- mitomycin C-containing plates. By inducing SOS <br /> did yaf&#11001; strain, unlike dinB, required (Fig. 4A          response DNA-damaging drugs, simultaneously induced <br /> B). Thus, yafNOP genes required stress-induced            &#65533;?&#65533; prophage lytic, leading far severe <br /> mutagenesis. The yafNOP genes shown previously con-               killing lysogens (Fig. 6A B) nonlysogens <br /> tribute slightly generation-dependent mutagenesis (43).           (Fig. 5B C), indicating major mode killing in <br />    The SOS DNA damage response. In hopes discovering                 lysogens prophage induction. However, no <br /> possible role yafNOP SOS response, explored             difference survival &#9004;yafNOP lysogens that <br /> several assays DNA damage affects cell survival test          yaf&#11001; lysogens (Fig. 6A B). <br /> whether yafNO contribute survival loss-of-survival               SOS-induced senescence caused yafNO. Previously, <br /> phenotypes. We used different DNA damage-induc-                   Pennington Rosenberg &#11011;65% cells that <br /> ing agents including UV light, bleomycin, mitomycin C                 undergo spontaneous SOS induction unable form col- <br /> to induce SOS response. We saw significant differ-                 onies, despite fact nearly SOS-induced cells <br /> ence &#9004;yafNOP yaf&#11001; cells survival UV light               viable, determined ability exclude dye PI <br /> (Fig. 5A); bleomycin, double-strand break-inducing agent                (47), suggesting senescence-like state (47). <br /> (29) (Fig. 5B); DNA cross-linking agent mitomycin C (30)               To test senescence-like state result the <br /> (Fig. 5C).                                                                toxic (and possibly bacteriostatic) yafO, repeated ex- <br />    A previous report suggested MazEF TA                   periment using FACS sort yaf&#11001; &#9004;yafNOP reporter <br /> protects cells phage P1 infection (27). We               strains carrying chromosomal gfp gene controlled the <br />  7462     SINGLETARY ET AL.                                                                                                     J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 5. Sensitivity nonlysogens DNA-damaging agents. (A) UV light kill curve. Log-phase cells UV irradiated monitored for <br /> colony formation. Average &#11006; 1 SEM experiments shown. (B) Nonlysogen bleomycin kill plates. Log-phase cells plated onto <br /> bleomycin-containing plates monitored viability. Bleomycin creates single- double-strand breaks DNA (29). A &#9004;recG strain <br /> monitored concurrently showed sensitivity (not shown). Results shown panels B C representative experiments. (C) Nonlysogen <br /> mitomycin C kill plates. Log-phase cells plated mitomycin C-containing plates (various concentrations) monitored viability. <br /> Mitomycin C powerful DNA interstrand cross-linker (30). A &#9004;recG strain monitored concurrently showed sensitivity (not shown). For all <br /> panels, yaf&#11001; strain MG1655, &#9004;yafNOP strain SMR6353, &#9004;recG strain SMR10990. Log dilutions indicated. <br />  <br />  <br />  <br /> SOS-inducible <span id='am-209' about='obo:NCBITaxon_32641' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_9205 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_30446 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-210' property="rdfs:label" content="Sula" datatype="xsd:string"></span><span id='am-211' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-213'  ></span>sulA</span> promoter. These sorted sponta-              little difference colony-forming abilities <br /> neously SOS-induced green SOS-uninduced nongreen                     spontaneously SOS-induced green populations. We ob- <br /> subpopulations, compose &#11011;1% &#11011;99% cell                   served 38% &#11006; 12% yafNOP&#11001; SOS-induced green cells <br /> population, respectively (reference 47 study). We               formed colonies (35% &#11006; 9% green cells forming colonies <br />  VOL. 191, 2009                                                                AN SOS-REGULATED TOXIN-ANTITOXIN SYSTEM                      7463 <br />  <br />  <br />                                                                                                       DISCUSSION <br />  <br />                                                                                 We shown yafN gene dinB-yafN-yafO- <br />                                                                              yafP operon (43) essential viability presence of <br />                                                                              functional yafO gene (Fig. 2B C), providing strong evi- <br />                                                                              dence genes constitute antitoxin toxin <br />                                                                              genes, respectively, type 2 TA pair. Whereas previous in <br />                                                                              vivo demonstrations type 2 TA systems used the <br />                                                                              method separate cloning gene differentially <br />                                                                              inducible plasmids showing toxin induces cell <br />                                                                              stasis solely expressed, antitoxin also <br />                                                                              expressed (1, 50), used alternative genetic approach that <br />                                                                              allows rule possible effects specific overexpression <br />                                                                              toxin antitoxin. We conclude <br />                                                                              YafO exerts toxic effect, YafN quell effect, <br />                                                                              gene expressed normal levels native <br />                                                                              promoter, single copy normal chromosomal position. <br />                                                                                 Typically, chromosomal TA systems consist genes in <br />                                                                              operon, toxin antitoxin. Although pre- <br />                                                                              cedent TA gene element, mazEFG <br />                                                                              (22), yafP, yaf gene operon, not <br />                                                                              integral component toxin antitoxin (Fig. <br />                                                                              3A B). This conclusion does preclude possibility <br />                                                                              YafP play regulatory role YafNO TA <br />                                                                              detected, mazG does mazEF <br />                                                                              (22). <br />                                                                                 The possible function TA controlled SOS <br />                                                                              response interesting problem. We unlike dinB <br />                                                                              (42), gene operon (43), yafNOP genes not <br />                                                                              contribute stress-induced mutagenesis significantly (Fig. 4), <br />                                                                              process requires SOS-induced levels dinB, not <br />                                                                              SOS-induced levels SOS-controlled component <br />                                                                              (17). Other SOS-controlled genes recA (6, 24) ruvA <br />                                                                              ruvB (15, 25) required stress-induced Lac mutagen- <br />    FIG. 6. YafNOP does affect killing prophage induction           esis, appreciate, constitutive levels expres- <br /> DNA-damaging agents. DNA damage induces SOS response                 sion, induced levels (17). The results presented rule <br /> activates &#65533;?&#65533; lytic cycle, causing killing prophage induction, can <br /> be seen greater sensitivity DNA-damaging agents lysogens <br />                                                                              requirement YafNOP stress-induced mutagenesis <br /> (this figure) nonlysogens (Fig. 5). YafNOP does affect      constitutive expression levels. Previously, <br /> killing prophage induction. (A) Lambda lysogen bleomycin kill             YafNOP little effect spontaneous generation-depen- <br /> plates. Log-phase E. coli (&#65533;?&#65533;) lysogens plated bleomycin-           dent mutagenesis nonstressed growing cells (43). <br /> containing plates (various concentrations) monitored viability.         We did detect effects yafNOP survival cells <br /> A &#9004;recG strain monitored concurrently showed sensitivity (not shown). <br /> (B) Lambda lysogen mitomycin C kill plates. Log-phase E. coli &#65533;?&#65533;              following various SOS-inducing treatments, nonlysogens <br /> lysogens plated mitomycin C-containing plates (various con-        (Fig. 5A C) cells harboring wild-type lambda <br /> centrations) monitored viability. A &#9004;recG strain monitored           prophage, induced leading cell lysis SOS <br /> concurrently showed sensitivity (not shown). For panels, yaf&#11001;       response activated (Fig. 6A B). Finally, we <br /> strain SMR11228, &#9004;yafNOP strain SMR11229, &#9004;recG <br /> strain SMR11231. Results shown representative experi-          recapitulate previous results Pennington and <br /> ments.                                                                       Rosenberg, showing spontaneously SOS-induced <br />                                                                              green fluorescent cells (bearing SOS-controlled chromo- <br />                                                                              somal gfp reporter gene) apparently viable unable to <br /> normalized 94% &#11006; 10% nongreen cells forming colonies                   form colonies recovered FACS (47), that <br /> to control FACS-induced effects colony formation) (Fig.               yafNOP responsible senescence-like state <br /> 7). Similarly, 26% &#11006; 5% &#9004;yafNOP green cells formed col-                   (Fig. 7). <br /> onies (25% &#11006; 5% green cells forming colonies normalized                      What function SOS-controlled TA sys- <br /> by 95% &#11006; 2% nongreen cells forming colonies) (Fig. 7).                    tem? For TA systems, toxic effects ensue operon&#8217;s <br /> Also, strains&#8217; cells nearly viable, 3.6% &#11006;                expression decreased, expect possible effects of <br /> 1.4% yafNOP&#11001; green cells PI&#11001; (dead) versus 4.5% &#11006;                   YafO manifested cells recover SOS response <br /> 1.4% &#9004;yafNOP green cells PI&#11001;. Thus, numbers                  return normal DNA repair, these <br /> cells senescence-like state alive (PI&#11002;) unable             genes expressed mid-range SOS response (11). Per- <br /> to form colonies different yafNOP&#11001;                      haps YafO induces transient cell stasis recovery. This <br /> &#9004;yafNOP cells (Fig. 7).                                                      function extend cell cycle checkpoint caused by <br />  7464      SINGLETARY ET AL.                                                                                                                    J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 7. The yafNOP genes responsible senescence-like state spontaneously SOS-induced cells. Strains chromosomal <br /> gfp gene controlled SOS-inducible sulA promoter sorted SOS-induced green SOS-uninduced nongreen populations FACS <br /> and plated CFU. The decreased ability SOS-induced green cells form colonies reversed &#9004;yafNOP mutation. Percentage <br /> of live/dead cells population determined PI staining flow cytometry quantify dead cells, dye. The percentage <br /> of cells senescence-like state, excluding PI unable form colonies, different yaf&#11001; (SMR6669) &#9004;yafNOP (SMR10988) cells. <br /> Average &#11006; SEM experiments. <br />  <br />  <br />  <br />  <br /> SOS-induced expression SulA inhibitor cell division                                         ACKNOWLEDGMENTS <br /> (16). Although did effects YafNO cells                This work supported National Institutes Health grant <br /> SOS induction, rule possibility impor-              R01-GM53158. <br /> tant role SOS recovery assays                      We thank N. Fonville, R. S. Galhardo, P. J. Hastings, C. Herman, A. <br /> detected. Alternatively addition, dinB, presumably                Al Mamun, C. Shee valuable input course of <br />                                                                             work. <br /> the rest operon, upregulated slightly RpoS <br /> general stress response (33). Perhaps YafO plays role in <br />                                                                                                              REFERENCES <br /> promoting cell stasis recovery RpoS response. <br />                                                                              1. Aizenman, E., H. Engelberg-Kulka, G. Glaser. 1996. An Escherichia coli <br /> Additionally, yafNO genes transcribed                     chromosomal &#8220;addiction module&#65533;? regulated guanosine [corrected] 3&#11032;,5&#11032;- <br /> the upstream SOS-controlled (and RpoS-controlled) dinB pro-                     bispyrophosphate: model programmed bacterial cell death. Proc. Natl. <br /> moter (43), vitro evidence additional pro-                  Acad. Sci. USA 93:6059&#8211;6063. <br />                                                                              2. Baba, T., T. Ara, M. Hasegawa, Y. Takai, Y. Okumura, M. Baba, K. A. <br /> moter exist immediately upstream yafN, possibly cre-                     Datsenko, M. Tomita, B. L. Wanner, H. Mori. 2006. Construction of <br /> ating SOS (or RpoS)-independent yafNOP operon (57). If                       <span id='am-194' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-195' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-196' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-197' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-198' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span>Escherichia coli K-12</span> in-frame, single-gene knockout mutants: Keio col- <br /> so, yafNOP act outside contexts SOS (or                        lection. Mol. Syst. Biol. 2:2006.0008. <br />                                                                              3. Black, D. S., A. J. Kelly, M. J. Mardis, H. S. Moyed. 1991. Structure and <br /> RpoS) response.                                                                 organization hip, operon affects lethality inhibition of <br />    A recent report shows DinB SOS-                        peptidoglycan DNA synthesis. J. Bacteriol. 173:5732&#8211;5739. <br />                                                                              4. Blattner, F. R., G. Plunkett III, C. A. Bloch, N. T. <span id='am-199' about='obo:NCBITaxon_38948' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6547 obo:NCBITaxon_106220 obo:NCBITaxon_6546 obo:NCBITaxon_6545 obo:NCBITaxon_6544 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_301959 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-200' property="rdfs:label" content="Perna" datatype="xsd:string"></span><span id='am-201' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-202' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-203'  ></span>Perna</span>, V. Burland, M. <br /> inducible DNA polymerase, Pol V, interact directly                     Riley, J. Collado-Vides, J. D. Glasner, C. K. Rode, G. F. Mayhew, J. Gregor, <br /> <span id='am-44' about='obo:NCBITaxon_468772' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_33154 obo:NCBITaxon_43733 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_50680 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50673 obo:NCBITaxon_50557 obo:NCBITaxon_50671'><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-46'  ></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Nusa" datatype="xsd:string"></span>NusA</span> transcription antitermination factor (10). The au-                     N. W. Davis, H. A. Kirkpatrick, M. A. Goeden, D. J. Rose, B. Mau, Y. <br /> thors suggest NusA direct translesion DNA                        Shao. 1997. The complete genome sequence Escherichia coli K-12. Science <br />                                                                                 277:1453&#8211;1474. <br /> synthesis activity DinB sites active transcription. Sim-            5. Brown, J. M., K. J. Shaw. 2003. A novel <span id='am-49' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> Escherichia coli <br /> ilarly, imagine DinB affect NusA-dependent                    toxin-antitoxin gene pairs. J. Bacteriol. 185:6600&#8211;6608. <br /> transcription termination, provide                  6. Cairns, J., P. L. Foster. 1991. Adaptive reversion frameshift muta- <br />                                                                                 tion Escherichia coli. Genetics 128:695&#8211;701. <br /> level SOS control gene expression (negative positive).              7. Cherepanov, P. P., W. Wackernagel. 1995. Gene disruption Esche- <br /> Perhaps YafNO YafP functions process,                          <span id='am-179' about='obo:NCBITaxon_1430853' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_95179 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_7100 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-180'  ></span><span id='am-181' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-182' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-183' property="rdfs:label" content="Richia" datatype="xsd:string"></span>richia</span> coli: TcR KmR cassettes option Flp-catalyzed excision <br />                                                                                 antibiotic-resistance determinant. Gene 158:9&#8211;14. <br /> perhaps toxic effect YafO related transcription- <br />                                                                              8. Christensen, S. K., M. Mikkelsen, K. Pedersen, K. Gerdes. 2001. RelE, <br /> termination-specific effect.                                                    global inhibitor translation, activated nutritional stress. Proc. <br />    While manuscript prepared, group                       Natl. Acad. Sci. USA 98:14328&#8211;14333. <br />                                                                              9. Cirz, R. T., F. E. Romesberg. 2006. Induction inhibition cipro- <br /> reported YafO protein RNase (58). Their results                  floxacin resistance-conferring mutations hypermutator bacteria. Antimi- <br /> support conclusions provide mechanism toxic                   crob. Agents Chemother. 50:220&#8211;225. <br /> action YafO toxin. Important steps under-                10. Cohen, S. E., V. G. Godoy, G. C. Walker. 2009. Transcriptional modu- <br />                                                                                 lator NusA interacts translesion DNA polymerases Escherichia coli. <br /> standing yafNO TA include defining                   J. Bacteriol. 191:665&#8211;672. <br /> on targets toxin acts.                                             11. Courcelle, J., A. Khodursky, B. Peter, P. O. Brown, P. C. Hanawalt. <br />  VOL. 191, 2009                                                                            AN SOS-REGULATED TOXIN-ANTITOXIN SYSTEM                                    7465 <br />  <br />       2001. Comparative gene expression profiles following UV exposure wild-          36. Lloyd, R. G., C. Buckman. 1991. Genetic analysis recG locus of <br />       type SOS-deficient Escherichia coli. Genetics 158:41&#8211;64.                           Escherichia coli K-12 role recombination DNA repair. J. <br /> 12.   Datsenko, K. A., B. L. Wanner. 2000. One-step inactivation chromo-              Bacteriol. 173:1004&#8211;1011. <br />       somal genes Escherichia coli K-12 using PCR products. Proc. Natl. Acad.         37. Lombardo, M. J., I. Aponyi, S. M. Rosenberg. 2004. General stress <br />       Sci. USA 97:6640&#8211;6645.                                                                 response regulator RpoS adaptive mutation amplification Esche- <br /> 13.   Engelberg-Kulka, H., G. Glaser. 1999. Addiction modules pro-                   richia coli. Genetics 166:669&#8211;680. <br />       grammed cell death antideath bacterial cultures. Annu. Rev. Micro-          38. Magnuson, R. D. 2007. Hypothetical functions toxin-antitoxin systems. J. <br />       biol. 53:43&#8211;70.                                                                        Bacteriol. 189:6089&#8211;6092. <br /> 14.   Fernandez De Henestrosa, A. R., T. Ogi, S. Aoyagi, D. Chafin, J. J. Hayes, H.      39. Masuda, Y., K. Miyakawa, Y. Nishimura, E. Ohtsubo. 1993. chpA and <br />       Ohmori, R. Woodgate. 2000. Identification additional genes belonging            chpB, Escherichia coli chromosomal homologs pem locus responsible <br />       LexA regulon Escherichia coli. Mol. Microbiol. 35:1560&#8211;1572.                 stable maintenance <span id='am-174' about='obo:NCBITaxon_28399' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384 obo:NCBITaxon_36549'><span id='am-175'  ></span><span id='am-176' property="rdfs:label" content="Plasmid R100" datatype="xsd:string"></span><span id='am-177' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmid R100</span>. J. Bacteriol. 175:6850&#8211;6856. <br /> 15.   Foster, P. L., J. M. Trimarchi, R. A. Maurer. 1996. Two enzymes,          40. McCool, J. D., E. Long, J. F. Petrosino, H. A. Sandler, S. M. Rosenberg, and <br />       process recombination intermediates, opposite effects                 S. J. Sandler. 2004. Measurement SOS expression individual Esche- <br />       adaptive mutation Escherichia coli. Genetics 142:25&#8211;37.                             richia coli K-12 cells using fluorescence microscopy. Mol. Microbiol. 53: <br /> 16.   Friedberg, E. C., W. Siede, G. C. Walker. 2006. DNA repair mu-                 1343&#8211;1357. <br />       tagenesis. ASM Press, Washington, DC. <br />                                                                                          41. McKenzie, G. J., R. S. Harris, P. L. Lee, S. M. Rosenberg. 2000. The <br /> 17.   Galhardo, R. S., R. Do, M. Yamada, E. C. Friedberg, P. J. Hastings, T. <br />                                                                                              SOS response regulates adaptive mutation. Proc. Natl. Acad. Sci. USA <br />       Nohmi, S. M. Rosenberg. 2009. DinB upregulation sole role the <br />                                                                                              97:6646&#8211;6651. <br />       SOS response stress-induced mutagenesis Escherichia coli. Genetics <br />                                                                                          42. McKenzie, G. J., P. L. Lee, M. J. Lombardo, P. J. Hastings, S. M. <br />       182:55&#8211;68. <br /> 18.   Gerdes, K., S. K. Christensen, A. Lobner-Olesen. 2005. Prokaryotic                 Rosenberg. 2001. SOS mutator DNA polymerase IV functions adaptive <br />       toxin-antitoxin stress response loci. Nat. Rev. Microbiol. 3:371&#8211;382.                  mutation adaptive amplification. Mol. Cell 7:571&#8211;579. <br /> 19.   Gotfredsen, M., K. Gerdes. 1998. The Escherichia coli relBE genes              43. McKenzie, G. J., D. B. Magner, P. L. Lee, S. M. Rosenberg. 2003. The <br />       belong new toxin-antitoxin gene family. Mol. Microbiol. 29:1065&#8211;1076.             dinB operon spontaneous mutation Escherichia coli. J. Bacteriol. <br /> 20.   Grady, R., F. Hayes. 2003. Axe-Txe, broad-spectrum proteic toxin-                185:3972&#8211;3977. <br />       antitoxin specified multidrug-resistant, clinical isolate Entero-   44. Miller, J. H. 1992. A short course bacterial genetics: laboratory manual <br />       <span id='am-184' about='obo:NCBITaxon_538669' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_33381 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_33380 obo:NCBITaxon_6072 obo:NCBITaxon_7524 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_33373 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_30189 obo:NCBITaxon_33342 obo:NCBITaxon_50557'><span id='am-185'  ></span><span id='am-186' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-188' property="rdfs:label" content="Coccus" datatype="xsd:string"></span>coccus</span> faecium. Mol. Microbiol. 47:1419&#8211;1432.                                          handbook Escherichia coli related bacteria. Cold Spring Harbor <br /> 21.   Grant, S. G. N., J. Jessee, F. R. Bloom, D. Hanahan. 1990. Differential            Laboratory Press, Plainview, NY. <br />      <br /> </body></html>