<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Apr. 1990, p. 2055-2064                                                                        Vol. 172, No. 4 <br /> 0021-9193/90/042055-10$02.00/0 <br /> Copyright C 1990, American Society Microbiology <br />  <br />  <br />  Identification Characterization New <span id='am-64' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-65' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-72'  ></span><span id='am-73' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia coli</span> Gene <br />              That Is Dosage-Dependent Suppressor a <br />                        dnaK Deletion Mutation <br />                                     PIL JUNG KANG AND ELIZABETH A. CRAIG* <br />             Department Physiological Chemistry, University Wisconsin-Madison, Madison, Wisconsin 53706 <br />                                          Received 4 October 1989/Accepted 10 January 1990 <br />  <br />             We report isolation characterization previously <span id='am-120' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-121' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-125'  ></span><span id='am-126' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>unidentified</span> Eschenichia coli gene that <br />           suppresses temperature-sensitive growth filamentation dnaK deletion mutant strain. Introduction <br />           multicopy plasmid carrying wild-type gene dnaK deletion mutant strain rescued the <br />           temperature-sensitive growth dnaK deletion mutant strain 40.5&#176;C filamentation, fully 37&#176;C <br />           partially 40.5&#176;C. However, inability dnaK mutant cells support bacteriophage A growth not <br />           suppressed. <span id='am-51' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-52' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54'  ></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> gene able suppress temperature-sensitive growth grpE280 mutant strain <br />           41&#176;C. Filamentation grpE280 mutant strain suppressed 37&#176;C 41&#176;C. The dnaK <br />           suppressor gene, designated dksA, maps near mrcB gene (3.7 min E. coli chromosome). DNA <br />           sequence analysis vivo experiments showed dksA encodes 17,500-Mr polypeptide. Gene disruption <br />           experiments indicated dksA essential gene. <br />  <br />   The heat shock response highly con-              revealed synthesis DNA RNA rapidly shut <br /> served biological responses environmental changes (19,             (15) cell division blocked (10, 32) the <br /> 20, 24). Most organisms far examined, including               nonpermissive temperature. Recent isolation new dnaK <br /> bacterial, plant, animal <span id='am-61' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>, induce synthesis         allele, dnaKI I I, showed dnaKI I I unable initiate a <br /> proteins encoded HSP60, HSP70, HSP90 gene                  new round DNA replication nonpermissive temper- <br /> families response elevated temperatures. Several         ature termination round progress (28). DnaK is <br /> proteins induced heat induced variety             known involved directly indirectly the <br /> other stresses. These heat shock proteins               phosphorylation aminoacyl-tRNA synthetases (34). <br /> most highly conserved proteins known. DNA sequencing                  Furthermore, DnaK negative modulator heat shock <br /> studies shown amino acid sequence Esche-             response. Cells carrying dnaK756 mutation showed high <br /> <span id='am-90' about='obo:NCBITaxon_1430853' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_95179 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_7100 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="Richia" datatype="xsd:string"></span><span id='am-94'  ></span>richia</span> coli groEL product 54% identical yeast hsp60         level synthesis heat shock proteins 30&#176;C and <br /> protein (25). E. coli DnaK 50% identical              failed turn heat shock response 42&#176;C (31). <br /> eucaryotic hsp70 proteins (5), E. coli C62.5 heat                To study cellular function DnaK protein, have <br /> shock protein 41% identical hsp83 protein                constructed new dnaK deletion mutant strain. Phenotypic <br /> <span id='am-39' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-45'  ></span>Drosophila melanogaster</span> (4). The universality heat             analysis strain showed deletion mutant was <br /> shock response high degree conservation heat            temperature cold sensitive growth highly fila- <br /> shock proteins imply proteins fundamental,            mentous permissive temperature. As way to <br /> common roles various organisms.                                    study cellular function DnaK protein, isolated and <br />    Most eucaryotic organisms contain <span id='am-31' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> HSP70-               analyzed suppressors overcame dnaK deletion mu- <br /> related genes. For example, <span id='am-7' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-8' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-22'  ></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>              tant phenotypes. In paper, report isolation a <br /> genome contains genes related               newly identified gene, dksA, suppresses tempera- <br /> HSP70 higher eucaryotes (19). However, dnaK gene            ture-sensitive growth filamentation dnaK deletion <br /> the HSP70-like gene E. coli (5). The dnaK gene forms          mutant strain overexpressed. The dksA gene was <br /> an operon heat shock gene, dnaJ,                cloned sequenced, location E. coli <br /> structure promoter-dnaK-dnaJ.                                         chromosome determined. <br />   The dnaK gene originally identified is <br /> essential bacteriophage X growth (14, 27). Genetic                                MATERIALS AND METHODS <br /> biochemical studies suggest DnaK protein required                    Bacterial strains, phages, media. The bacterial strains <br /> for initiation X DNA replication facilitating               phages used genotypes listed Table 1. <br /> dissociation A P protein K DNA replication                    LB broth M9 minimal medium prepared de- <br /> initiation complex, allowing DnaB protein (a helicase)                 scribed Miller (23). When necessary, antibiotics were <br /> function (12). DnaK, 69,000-Mr protein, exten- <br /> sively characterized HSP70 heat shock protein. Purified                   supplemented following final concentrations: ampicil- <br /> DnaK, binds ATP tightly, weak DNA-indepen-                    lin, 100 ,ug/ml; tetracycline, 20 ,ug/ml; kanamycin, 100 <br /> dent ATPase (more generally, 5&apos;-nucleotidase) activity (6,                ,ug/ml. Amino acids supplemented 40 ,ug/ml. <br /> 37). The purified protein autophosphorylates                  <span id='am-115' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-116' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>Plasmids</span>. Plasmids obtained study briefly de- <br /> threonine residues (37). Although DnaK biochemically                   scribed Table 2, details given text. <br /> well characterized, little known cellular function.             Genetic manipulations. Linear transformation DNA into <br /> In vivo analysis temperature-sensitive dnaK mutants                    recB recC sbcB strain performed described by <br />                                                                           Winans et al. (35). Selections k lysogens P1 trans- <br />                                                                           ductions described Silhavy et al. (30), for <br />   * <br />       Corresponding author.                                               transductions dnaK deletion allele. Growth phage <br />                                                                    2055 <br />  2056       KANG AND CRAIG                                                                                                 J. BACTERIOL. <br />  <br />              TABLE 1. Bacterial phage strains                               TABLE 2. Plasmids obtained study <br />  <br />   Strain                   Genotype <br />                                                        Source or <br />                                                        reference      Plasmid                   Description                   Suppressing <br />                                                                                                                                  ability <br /> JC7623      AB1157 recB21 recC22 sbcBJS sbcC201 21                    pJK53     Originally isolated suppressor clone              + <br /> MG1655      F-                                        CGSCa           pJK72     Originally isolated suppressor clone              + <br /> DS410       F- <span id='am-46' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-47' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50'  ></span>ara</span> Azir tonA lacY minA minB xyl       C. Gross&apos;       pJK711    Originally isolated suppressor clone              + <br />               mtl thi malA strA                                       pJK530    Clal-digested pJK53, religated (large frag-       - <br /> JM109       recAl endAl gyrA96 thi hsdR17 supE44 36                               ment) <br />               relAl      A(lac-proAB)(F&apos; traD36                       pJK533    1.2-kb PvuII fragment pf pJK53 cloned <br />               proAB lacIqZAM]5)                                                   EcoRV, PvuII sites pBR322 <br /> CAG5051     HfrH, nad::TnlO thi-1 spoT supQ80         C. Gross        pJK537    1.8-kb PvuII fragment pf pJK53 cloned             + <br /> CAG5052     KL227, btu::TnlO metBi relA               C. Gross                    EcoRV, PvuII sites pBR322 <br /> CAG5053     KL208, zbc::TnlO relA                     C. Gross        pJK538    1.7-kb PvuII-EcoRV fragment pJK53               + <br /> CAG5054     KL96, trp: :TnlO thi-J relA               C. Gross                    cloned EcoRV, PvuII sites of <br /> CAG5055     KL16, zed::TnlO thi-l relA                C. Gross                     pBR322 <br /> CAG8160     KL14, leu thi-l relA rpoD+-TnlO           C. Gross        pJK539    One ClaI site pBR322 pJK538                + <br /> CAG8209     KL228, leu thi-l gal lacY supE44          C. Gross                     inactivated Klenow fragment <br />               rpoD+-TnlO                                              pJK540    pJK539 cut ClaI, inactivated by <br /> CAG18436    MG1655, zae-502::TnJO                     C. Gross                     Klenow fragment, religated <br /> CAG12025    MG1655, zad-220::TnlO                     C. Gross        pJK541    Clal-digested pJK538, religated (large <br /> CAG9253     C600, X901 F&apos; X 90, grpE280 Tetr          C. Gross                     fragment) <br /> MF6         thr-l dapD4 his-4 rpsL136 leuB6 eda       9               pJK542    176-bp PvuII-BclI deletion pJK538 <br />                thi-l lacYl Agal-1 ara-14 xyl-1S mtl-l                 pJK543    NdeI site pJK538 inactivated by <br />                tonA supE                                                           Klenow fragment <br /> MF6R        MF6, rpoB                                 9               pJK333    Same pJK533 X17-181 <br /> RX800       MF6, lpxA                                 9               pJK334    Same pJK333 opposite orientation <br /> JCR20       MF6R, firA                                9               pJK337    Same pJK537 X17-181                 + <br /> DC1         thi-1 his-4 rpsL136 leuB6 eda thr-J       9               pJK21     408-bp ClaI-PvuII fragment pJK538 <br />                lacYl lpxBI pgsA Agal-1 ara-14 xyl-                                 replaced Kanr cassette from <br />                15 mtl-l tonA supE                                                  pUC4K <br /> PK5         MG1655, grpE280 (CAG9253 x                This study      pJK15     DnaJ-expressing plasmid (dnaJ fused to <br />                MG1655, selection Tetr, screening                               dnaK promoter) <br />                X&apos;) <br /> PK9         MG1655, zae-502::TniO dksA::kan           This study <br />                (CAG18436 x PK201, selection                       compare mutant phenotypes DnaK- DnaJ- those <br />                Kanr Tet)                                              DnaJ, dnaJ deletion constructed. The dnaJ <br /> PK101       MG1655, dnaKi4 dnaJI4                                     deletion mutant strain, PK102, carries deletion the <br /> PK102       MG1655, dnaJi5 <br /> PK113       Spontaneous Ts&apos; revertant PK101                        majority dnaJ coding sequence, retaining the <br /> PK201       MG1655, dksA::kan                                         sequences encoding 23 30 amino acids of <br /> PK202       MG1655, dksA::kan dnaKI4 dnaJ14                           DnaJ (we named allele dnaJJ5). The details the <br /> A imm2&apos;                                                H. Uchidac     construction dnaK dnaJ deletion mutant strains <br />   dnaK+                                                               described elsewhere. Plasmid pJK21, which <br />   dnaJ+                                                               carries deletion 408-bp ClaI-PvuII fragment of <br /> X imm2&apos;                                               H. Uchida       pJK538, constructed replacing region the <br />   dnaK+                                                               kanamycin resistance (Kanr) cassette pUC4K (33) to <br /> X17-181                                               F. Blattnerb    make dksA deletion mutant strain. <br />   CGSC, E. coli Genetic Stock Center, Yale University, New Haven,      Construction genomic library isolation suppres- <br /> Conn. <br />   b University Wisconsin-Madison. <br />                                                                       sor clone. A multicopy genomic library constructed by <br />     University Tokyo, Tokyo, Japan.                                cloning partial Sau3AI digestion chromosomal DNA <br />                                                                       spontaneous temperature-resistant suppressor strain, <br />                                                                       PK113, unique BamHI site pBR322. This DNA <br />                                                                       used transform dnaK deletion strain, PK101, and <br /> P1 dnaK deletion strain poor; P1                  transformants selected 42&#176;C LB-ampicillin <br /> transduction efficiency extremely low. To increase            plates. Plasmid DNA isolated temperature- <br /> P1 transduction efficiency, P1 lysates prepared X           resistant clones used transform PK101 second time <br /> imm21dnaK&apos; dnaJ+ lysogen strain PK101, lysates             confirm suppression temperature-sensitive growth <br /> were treated Lambda Sorb (Promega Biotec) remove              PK101. <br /> X phages P1 lysate. Transduction efficiency                   Mapping. To map dksA gene E. coli chromo- <br /> increased 1,000-fold way.                           some, Kanr cassette pUC4K inserted one <br />    Construction dnaK, dnaJ, dksA deletion mutant           EcoRV sites pJK53. This Kanr marker intro- <br /> strains. The basic strategy construction chromo-             duced chromosome linear transformation and <br /> somal deletion construct deletion mutations dnaK,           mapped 10-min interval set Hfr strains <br /> dnaJ, dksA recombinant plasmids insert             origins distributed chromosome (ob- <br /> these mutations E. coli chromosome linear                 tained C. Gross, University Wisconsin-Madison). <br /> transformation. The dnaK deletion mutant strain, PK101,               The Kanr marker tested P1 cotransduction with <br /> carries deletion entire dnaK coding sequence             mapped TnJO insertions region described the <br /> 155 base pairs (bp) 5&apos; end dnaJ gene. Since        Results. Subsequent transduction analysis performed by <br /> of dnaJ gene deleted, resulting construction         using nearby selectable markers. Fine mapping was <br /> is DnaK- DnaJ- (we named allele dnaKJ4 dnaJJ4). To               Southern hybridization, using 1.2-kilobase-pair (kb) <br />  VOL. 172, 1990                                               CHARACTERIZATION OF A dnaK SUPPRESSOR GENE                                        2057 <br />  <br /> PvuII fragment pJK53 probe A DNA                  TABLE 3. Plating efficiency wild type (MG1655) the <br /> covering region (X17-181, obtained F. Blattner,            dnaK deletion strain (PK101) carrying dnaK suppressor <br /> University Wisconsin-Madison).                                        clone (pJK537) control plasmid (pJK333) <br />    Recombinant DNA techniques. The procedures compiled                                                                  Plating efficiencya <br /> by Maniatis et al. (22) generally used minor                   Strain <br />                                                                                                                37&#176;C                           40.50C <br /> modifications. Isolation A DNA carried using <br /> Lambda Sorb (Promega Biotec).                                      Wild type                                   99.4                           96.2 <br />    Si nuclease mapping. To identify start site dksA         PK101(pJK537)                               95.6                           91.3 <br /> RNA transcript, Si nuclease mapping previ-             PK101(pJK333)                               53.7                            0.08 <br /> ously described (13). The 5&apos;-end-labeled probe prepared         a <br />                                                                         All strains considered   100%o plating efficiency 30&#176;C. <br /> from pJK539 cleaving PvuII, followed alkaline <br /> phosphatase kinase treatment. The appropriate labeled <br /> DNA probe digested HincII purified an <br /> agarose gel. Total RNA isolated mid-log-phase           identify suppressors dnaK deletion, spontaneously <br /> culture strain MG1655 carrying pJK537 described           arising mutants PK101 able grow 42&#176;C isolated. <br /> Aiba et al. (1). For Si digestion reaction, 50 F.g total   The spontaneous reversion frequency 10-&apos;. One of <br /> RNA 100 pg labeled probe hybridized 55&#176;C,           strains, PK113, chosen analysis. <br /> and 200 U Si nuclease treated. A reaction mixture              Isolation temperature-resistant suppressor clone. When <br /> containing 30 ,ug yeast tRNA included control      began experiments designed isolate suppressor <br /> the&apos; assay. DNA fragments electrophoresed a&apos; 6%            clone, expected isolate mutant gene could <br /> polyacrylamide gel containing 7 M urea             suppress temperature-sensitive growth dnaK <br /> 32P-end-labeled HaeIII digest pBR322 DNA marker             deletion strain. However, later results indicated the <br /> fragments.                                                         isolated suppressor clone described contain&apos;ed wild- <br />    DNA sequencing. DNA sequences determined            type gene, mutant gene, suggesting overexpression <br />  dideoxy-chain termination method (29), using Sequenase            wild-type gene suppressed temperature-sensitive <br />  (U.S. Biochemical Corp.) strain JM109 host        growth dnaK deletion strain PK101. <br />  M13mpl8 M13mpl9 derivatives (36).                                 A multicopy genomic library constructed cloning <br />     Analysis plasmid-encoded polypeptides. Polypeptides         partial Sau3AI digest chromosomal DNA from <br />  encoded plasmids analyzed using minicells (11).        spontaneous temperature-resistant suppressor strain, <br />  Minicells prepared cells carrying plasmids        PK113, unique BamHI site pBR322. The temper- <br />  tested labeled L-[355]methionine (1,114 Ci/mmol; 30      ature-sensitive dnaK deletion strain, PK101, transformed <br />  ,uCi/ml, final concentration; Dupont, NEN Research Prod-          genomic library DNA, temperature-resistant <br />  ucts). Labeled protein samples analyzed sodium            transformants selected 42&#176;C LB-ampicillin plates. <br />  dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-          Plasmid DNA isolated temperature-resistant <br />  PAGE) (15% acrylamide, 0. 195% bisacrylamide); gel        transformants used transform strain PK101 a <br />  stained Coomassie blue treated 1 M              second time confirm suppression temperature- <br />  sodium salicylate. The gel dried fluorographed.       sensitive growth. After second screening, indepen- <br />     Plating efficiency. Cells grown overnight 30&#176;C          dent clones isolated. Restriction endonuclease analysis <br />  diluted 1:100 LB broth grown optical density        showed identical clones all <br />  600 nm 0.5. Appropriate dilutions LB plates          clones overlapping DNA segments. <br />  incubated overnight 30, 37, 40.5, 41&#176;C, colonies            In initial screening, isolated dnaK deletion cells <br />  counted.                                                      transformed suppressor plasmids grow at <br />     Phase-contrast microscopy. Cells fresh colonies            42&#176;C. However, additional studies showed P1 transduc- <br />  grown LB agar plates 30, 37, 40.5, 41&#176;C              tion dnaK deletion allele wild-type strain <br />  prepared glass microscope slides. A Zeiss photomicro-           carrying isolated suppressor clone resulted <span id='am-87' about='obo:NCBITaxon_forma' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-88' property="rdfs:label" content="forma" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>forma</span>- <br />   scope x40 phase objective <span id='am-1' about='obo:NCBITaxon_3863' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_163743 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Lens Mill." datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Lens" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-6'  ></span>lens</span> Kodak PanX film           tion viable colonies temperatures high as <br />  used photography.                                      40.5&#176;C. This result indicated original dnaK deletion <br />                                                                     mutant strain contained additional mutation <br />                             RESULTS                                 which, combination suppressor clone, allowed <br />                                                                     growth 42&#176;C. <br />    Isolation suppressor dnaK deletion. The dnaK              Mapping suppressor locus. To map suppressor <br />  deletion strain PK101, containing alleles dnaKJ4           locus E. coli chromosome, Kanr cassette from <br />  dnaJ14, temperature sensitive growth 40&apos;.5&#176;C. The        pUC4K inserted of&apos;the EcoRV sites pJK53. <br />  plating efficiency PK101 40.5&#176;C 0.08%       This Kanr marker introduced E. coli chromo- <br />  30&#176;C (Table 3). The dnaK deletion strain showed            linear transformation mapped 96 <br />  cold-sensitive growth 17&#176;C. When P1 transductants         6 min set Hfr strains origins are <br />  30&#176;C replated P1 transduction          distributed chromosome. To delineate the <br />  17&#176;C, colonies formed, 10 days incubation.       location suppressor, number strains carrying TnlO <br />  Cells extremely filamentous 30&#176;C            insertions region used. The Kanr marker was <br />  defective cell division. As previously mentioned,      tested P1 cotransduction mapped TnJO insertions. <br />  dnaK deletion strain PK101 actually DnaK- DnaJ-. To              One strains carrying TnJO insertion near 4 min <br />  check DnaK- phenotypes, PK101 transformed              (zae-502::TnJO) showed 80 90% cotransductiona ofstrain  the <br />  DnaJ-expressing plasmid (pJK15). The resulting trans-           tetracycline resistance (Tetr) Kanr markers; <br />  formants showed temperature-sensitive growth               carrying TnJO insertion near 3 min (zad-220::TnJO) gave <br />  filamentation. Since DnaJ-expressing plasmid          18% cotransduction. Subsequent P1 transduction analysis <br />  stably maintained dnaK deletion strain PK101,             nearby selectable markers showed 83% cotransduction <br />  PK101&apos; used studies reported here. To genetically        Kanr marker dapD locus (Fig. 1). The relative <br />  2058     KANG AND CRAIG                                                                                                     J. BACTERIOL. <br />  <br />                            minute        2                      3.5                                    5 5 <br />                                                                                                                41 <br />                            leu <br />                                                                Kan     mr         hu   d8ph          fiA            A <br />  <br />  <br />  <br />  <br />                                               &lt;.002        - <br />                                                                       ,.-   .83        - <br />                                                                       |            18 <br />                                                                   |                        .13 .04 <br />  <br />                                                                                 4 (published <span id='am-110' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-111' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-112' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-113'  ></span><span id='am-114' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span>; ref.8) <br />                                                                            .8-.9 <br />   FIG. 1. Linkage dnaK suppressor gene number reference loci 2- 4-min segment standard E. coli linkage map. <br /> The Kanr marker inserted just outside dnaK suppressor gene EcoRV site pJK53 indicated + Fig. 3 used <br /> for selectable marker. <br />  <br /> cotransduction frequencies Kanr marker            hybridized probe. This result reconfirmed <br /> other markers suggested Kanr marker located           originally isolated suppressor clones had <br /> near mrcB gene.                                                overlapping DNA segments. <br />   Comparison restriction map suppressor clone           Fine localization suppressor gene. To define <br /> with published restriction map E. coli chromosome       suppressor gene, sequential subcloning originally <br /> (17) revealed similar pattern restriction sites near      isolated suppressor clone done. Each subclone was <br /> mrcB gene. Subsequently, fine mapping South-           tested ability allow growth dnaK deletion <br /> ern hybridization analysis. The 1.2-kb PvuII fragment <span id='am-95' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-97'  ></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="Iso" datatype="xsd:string"></span>iso</span>-         strain PK101 40.5&#176;C. The temperature-sensitive growth of <br /> lated suppressor clone (pJK53) hybridized        strain PK101 suppressed plasmids pJK537 and <br /> restriction enzyme-digested DNA X transducing phage           pJK538 pJK530 pJK533 (Fig. 3). These <br /> (X17-181) covering region (Fig. 2). This result indicated     results suggested dnaK suppressor gene located <br /> that suppressor located upstream mrcB gene          1.7-kb DNA fragment PvuII site <br /> (3.7 min E. coli chromosome). The restriction frag-         EcoRV site (*s Fig. 3). <br /> ments suppressor clones (pJK72 pJK711)               Overexpression wild-type gene suppresses dnaK dele- <br />                                                                    tion. Since suppressor gene localized upstream of <br />                                                                    mrcB gene, wild-type gene cloned X <br />      EcoRI Hindj[I    Pv u ll Psji!      Hildii                    transducing phage DNA covers region. pJK337, <br />                 RV PYIU                                            carried 1.7-kb PvuII-EcoRV DNA fragment the <br />     I1      ,,,_, <br />                 ... 1 R      11.11l       -- <br />                                           .P   ,- <br />                                                               xDNA <br />                                                                    phage, constructed used transform dnaK <br />                                                                    deletion strain, PK101. PK101 transformed pJK337 <br />                        l                1 1 <br />                                                    PsC Ps <br />                                                                    formed    colonies 40.5&#176;C, did PK101 transformed with <br />                EcoFRV <br />                   Clal       }Spat EcnRsV <br />                                        PstI <br />                                                                    pJK537,    plasmid carrying original suppressor clone. <br />                                         ,mrc <br />                                                  2Kb <br />                                                                    This result implied originally isolated suppressor <br />                                                                    clone wild-type gene. Subsequent genetic experiments <br />                        probe                                       supported conclusion. To determine mutation <br /> pJK53                                                              isolated gene responsible suppression the <br /> pJ K72                                                             original temperature-resistant suppressor strain, asked <br /> pJK71 1                                                            temperature resistance PK113, spontane- <br />                                                                    ously isolated suppressor strain, cotransduced the <br />                                                                    Tetr marker present near suppressor gene. PK113 was <br />                      z_1 <br />                    o.&apos;           .7 .p <br />                                ill            P&apos;                   transduced    P1 <span id='am-105' about='obo:NCBITaxon_1288776' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6694 obo:NCBITaxon_6695 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6682 obo:NCBITaxon_115580 obo:NCBITaxon_88770 obo:NCBITaxon_6683 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-106' property="rdfs:label" content="Vir" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-109'  ></span>vir</span> grown strain CAG18436, which <br />                            -,    --c <br />                                11- <br />                               1. d&apos; <br />                                        - <br />  <br />                                            A. <br />                                                                    contained    zae-502::TnlO.   Since zae- 502::TnO deter- <br />                                                                    mined 80 90%o cotransduced Kanr marker <br />                                                                    near suppressor gene, expected suppres- <br />                                                  .*   ).)2k b <br />                                                                     sor gene actual site mutation, 70 80% of <br />                                                       4. k r, <br />                                                                     Tetr transductants temperature sensitive. How- <br />                                                  .4 <br />                                                                    ever, 95% Tetr transductants still <br />                                                                    temperature resistant. These results suggested orig- <br />                                                                     inally isolated suppressor clones carried wild-type gene and <br />                                                                    overexpression gene suppressed temperature- <br />                                                                     sensitive growth dnaK deletion strain. <br />                                                                        DNA sequence determination. The nucleotide sequence of <br />                                                                     1.7-kb PvuII-EcoRV fragment (* Fig. 3) deter- <br />                                                                     mined (Fig. 4). Northern (RNA) blot analysis using labeled <br />                                                                     single-stranded DNA probes showed transcription oc- <br />    FIG. 2. Southern hybrdization 1.2-kb PvuII fragment    curred direction PvuII EcoRV site. <br /> pJK53 X17-181, pJK72, pJK711. A 32P-labeled probe      Analysis sequence data revealed only <br /> prepared pJK53 hybridized PstI-HindIII-digested X DNA,  nonoverlapping open reading frames sense strand <br /> PstI-HindIII-digested pJK711, EcoRV-PstI-digested pJK72.        region larger 42 codons. The shorter open <br />  VOL. 172, 1990                                                                    CHARACTERIZATION OF A dnaK SUPPRESSOR GENE                                    2059 <br />  <br />                        Hpal   PuuII*         Clal                       ECORU*                 EcoRU+ <br />  <br />  <br />                           PstI                          Puu IE            Puul I <br />  <br />                                                                                                                               suppressing <br />                                              ORFI            ORF2             4           kb       &gt;                                  ability <br />  <br />                                                              __________________                          pJK 53 <br />                                                                                                               pJK533 <br />                                                                                                                  pJK537               + <br />                                                                                                                  pJK538               + <br />                                              _____________                                                       pJK54t               + <br />                                                                                                                  pJK541 <br />                                                                                                                  pJK542 <br />                                                              ___                                                 pJK543 <br />   FIG. 3. Fine localization suppressor gene. The originally isolated suppressor clone, pJK53, subcloned, mutants were <br /> constructed vitro described Materials Methods. Each clone tested suppressing ability testing ability of <br /> transformants PK101 form colonies 40.5&#176;C. pJK540 pJK543 +1 frameshift mutations. pJK542 in-frame deletion in <br /> the middle ORF2. ORF1 ORF2 open reading frames determined region analysis DNA sequencing data (Fig. 4). <br />  <br />  <br />  <br />  <br />     1   GACGAAAGAGGCTATCCTTAATGAATCAATTTCAGAACTGTCAGGCTATAGCTCGCTGAAAAGCGAAGTAAAATACGGCGCAGAACGCAGCCGTATTGAC                                                     100 <br />  <br />  <br />   101   TTTATGTTGCAGGCGGATTCGCGTCCAGACTGCTATATTAAGTGAAATCGGTTAQGTGCGGAGAACGAACAGGGATATTTTCCCGATGCGGTCACTG                                                        200 <br />                                                                     -35                            -10 <br />  <br />   201   AACGAGGTCAGAAACACTTCEaGUTTGATGAGCGTAGCGGCTGAAGGCCAGCGTGCGGTTATCTTTTTCGCCGTGCTGCATTCAGCCATTACACGGTTT                                                      300 <br />                              S.D    M S V A A E G Q R A V I F F A V L H S A I T R F <br />                               ORFP--&gt; <br />   301   TCACCCGCGCGCCACATCGATGAGAAATACGCGCAACTATTGTCAGAAGCTCAACAGAGGGGGGTAGAAATTCTGGCTTACAAAGCGGAAATTTCTGCTG                                                     400 <br />          S P A R H I D E K Y A Q L L S E A Q Q R G V E I L A Y K A E I S A <br />  <br />   401   AAGGCATGGCTCTTAAAAAATCACTGCCGGTTACATTGTAGTAAAGTAAGTAACTGGTTAATTTACATTCTGGTCGCGTGCGCAAATACGCTTTTCCTCA                                                     500 <br />         E G M A L K K S L P V T L <br />  <br />   501   CACAGTTGTCAAGTGTTACGTTTAGATAATTGCTATCCGGAAAAGCATCTGCTATTTATAGCGGCCTCATTTTTCCCCCGAACATGGGGATCGATAGTGC                                                     600 <br />  <br />  <br />   601   GTGTTAAGE&amp;aLAGCAACATGCAAGAAGGGCAAAACCGTAAAACATCGTCCCTGAGTATTCTCGCCATCGCTGGGGTGGAACCATATCAGGAGAAGCCGG                                                     700 <br />                  S.D             M   Q   E   G      Q    N    R     K     T       S   S    L   S    I    L   A   I   A    G   V   E    P   Y    Q   E   K   P <br />                      0RF2--&gt; <br />   701   GCGAAGAGTATATGAATGAAGCCCAGCTGGCGCACTTCCGTCGTATTCTGGAAGCATGGCGTAATCAACTCAGGGATGAAGTCGATCGCACCGTTACACA                                                     800 <br />         G E E Y M N E A Q L A H F R R I L E A W R N Q L R D E V D R T V T H <br />  <br />   801   TATGCAGGATGAAGCAGCCAACTTCCCGGACCCGGTAGACCGTGCAGCCCAGGAAGAAGAGTTCAGCCTCGAACTGCGTAACCGCGATCGCGAGCGTAAC                                                     900 <br />           M Q D E A A N F P D P V D R A A Q E E E F S L E L R N R D R E R N <br />  <br />   901   GTGATCAAAAAGATCGAGAAGACGCTGAAAAAAGTGGAAGACGAAGATTTCGGCTACTGCGAATCCTGCGGTGTTGAAATTGGTATTCGCCGTCTGGAAG                                                    1000 <br />          V I K K I E K T L K K V E D E D F G Y C E S C G V E I G I R R L E <br />  <br />  1001   CGCGCCCGACAGCCGATCTGTGCATCGACTGCAAAACGCTGGCTGAAATTCGCGAAAAACAGATGGCTGGCTAATTACAGCCGTTCCATCACGTTTACCA                                                    1100 <br />         A  R P T A D L C I D C K T L A E I R E K Q M A G <br />  <br />  1101   CACGGGGAAATCGTCCCGCCTTATTTTTTGTTCAAAGAGATGACAGACACACAGTATATTGGCCTGTCGCCCCTCTCTTCCGGCGAGCTTCATTTTGGCT                                                    1200 <br />  <br />  <br />  1201   CTCTGATCGCTACGCTCGGCAGCTATTTGCACGTCGCGCCCGGCAAGGTCGCTGGCTGGTACGCATAGAAGAT 1273 <br />   FIG. 4. Nucleotide sequence dksA region. The sequence sense strand amino acid sequences open reading <br /> frames shown. The sequence numbered HincII (left) EcoRV site (Fig. 3). The -10 -35 regions possible promoter <br /> are indicated. Putative Shine-Dalgamo (S.D) sequences underlined. <br />  2060     KANG AND CRAIG                                                                                                                    J. BACTERIOL. <br />  <br />                                                                              reading frame (ORF1) encode polypeptide 70 <br />                                                                              amino acids (Mr, 7,650), longer (ORF2) could <br />                                                                              encode polypeptide 151 amino acids (Mr, 17,480). The <br />                                                                              predicted ORF2 gene product contains 38% charged residues <br />                                                                              distributed length protein, net <br />                                                                              charge -6. ORF1 preceded putative ribosome- <br />                                                                              binding site, GGGAG, located 4 bp upstream initiation <br />                                                                              <span id='am-56' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-57' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-60'  ></span>codon</span>. ORF2 preceded putative ribosome- <br />                                ,r,~08e.^100i                                 binding site, GGAGA, located 6 bp upstream initiation <br />                                                                              codon. A sequence shares homology consensus <br />                                                                              E. coli promoter sequence (26) located upstream of <br />                                             600                              ORF1. This organization suggested ORF1 ORF2 <br />                                       h <br />                                                                              comprise operon. <br />                                                                                 Analysis RNA transcript start site. To estimate the <br />                                                                              size mRNA encoded ORF1 ORF2, Northern <br />                                                                              hybridization experiments done. RNAs isolated from <br />                                                                              wild-type strain MG1655 MG1655 carrying the <br />                                                                              suppressor clone, pJK537, separated electrophore- <br />                            2      3                                          sis hybridized labeled 408-bp ClaI-PvuII frag- <br />                                                                              ment pJK538. A major RNA transcript approximately <br />   FIG. 5. Analysis RNA transcripts dksA region. Total            500 nucleotides detected RNA preparations the <br /> RNAs prepared wild-type cells, wild-type cells carrying        strain suppressor clone preparations <br /> dksA clone (pJK537), cells carrying control plasmid (pJK333),          transformed strain. In addition, relatively abun- <br /> electrophoresed 1.3% agarose gel, blotted. The blot was <br /> hybridized labeled 408-bp ClaI-PvuII fragment pJK538.            dant 600-nucleotide RNA identified transformed <br /> Lanes: 1, MG1655; 2, MG1655(pJK537); 3, MG1655(pJK333).                      strain. The largest transcript hybridized probe <br />                                                                              1,000 nucleotides length (Fig. 5). <br />                                                                                 To determine 5&apos; end RNA transcript, Si nuclease <br />                                                                              mapping carried out. A 5&apos;-end-labeled probe pre- <br />                                                                              pared noncoding strand 726-bp HincII-PvuII <br />  <br />  <br />                       I-_          -      undigested probe <br />                                    -      560 nt <br />  <br />  <br />  <br />                                                              Hpol PuuII*               C1al <br />                                                                                                        Ndel <br />                                                                                                                       EcoRiJ*        ECoRV+ <br />                                                                 I <br />                                                                          I    Hincl] <br />                                                                                                          l                l <br />                                    -      255 nt                    w-                                 1       1                9 <br />  <br />  <br />  <br />  <br />                                                                 Psti <br />                                                                                   PL-             PuulI <br />                                                                                                        Bcll                   l <br />                                                                                                                             Puull <br />  <br />                                                                                                                                     1 kb <br />                                                                                                                                               M- <br />                                                                                                                                               w- <br />  <br />  <br />  <br />  <br />                                                                                            ORFI        ORF2 <br />  <br />  <br />  <br />                                    -- 1 6 0 <br />                                                                                                     160 nt <br />                                                                                                   -~255 nt <br />                                                                                        -      -    -               560 nt <br />  <br />  <br />  <br />  <br />               1 12 3 14 <br />   FIG. 6. S1 analysis dksA transcripts. Si mapping carried described text, using total cellular RNA 32P-end- <br /> labeled noncoding strand HincII-PvuII fragment. The sizes protected fragments (in nucleotides) determined running <br /> end-labeled pBR322 digested HaeIII marker. The 5&apos; end protected fragment shown restriction map this <br /> region. P, Putative promoter site. Lane 1, pBR322-HaeIII marker; lane 3, undigested probe marker. The reaction mixtures contained RNA <br /> from MG1655(pJK537) (lane 2) yeast tRNA control (lane 4). <br />  VOL. 172, 1990                                              CHARACTERIZATION OF A dnaK SUPPRESSOR GENE                                             2061 <br />                                                                        Mr. <br />  <br /> fragment (Fig. 6). The probe hybridized total RNA               68- <br />  <br /> isolated log-phase cells strain MG1655 carrying   the <br />  <br /> suppressor clone, pJK537. Three major protected fragments <br /> with approximate lengths 560, 255, 160 nucleotides                4 2- <br />  <br />  <br /> were observed (Fig. 6). Fragments lengths <br />                                                                                                                                              TET <br /> were detected using RNA isolated untrans-                                                                                       BLA <br />  <br /> formed cells (data shown). In cases, 160- <br /> nucleotide fragment abundant. The 5&apos; ends                18 <br />  <br />  <br /> these major protected fragments approximately                      1   5 <br />  <br /> mapped nucleotide positions 166, 501, 566       the <br />  <br /> sequence shown Fig. 4. Position 166 located upstream <br /> of ORF1   close consensus E. coli promoter sequence.                                       M.            B <br />  <br /> Positions 501 566, however, close proximity                      piK <br />                                                                                    537    333 <br />                                                                                                 marKe   pBR <br />                                                                                                         322 <br />                                                                                                               pK   pJ K <br />                                                                                                               537 334 <br />                                                                                                                           pBR <br />                                                                                                                           322 <br />                                                                                                                                 p1K <br />                                                                                                                                 537 <br />                                                                                                                                       p_l( <br />                                                                                                                                        334 <br /> a consensus E. coli promoter sequence. Although 160- <br /> nucleotide fragment abundant ORF2 is <br />                                                                      FIG. 7. Polypeptides encoded plasmids overexpressing <br /> completely included pJK530, pJK530 sup-          cells minicells. Proteins analyzed 15% SDS-PAGE. (A) <br /> press  temperature-sensitive phenotype. Therefore,         Coomassie blue-stained gel total cell extracts strain <br /> propose position 166 actual RNA transcription         MG1655, MG1655 carrying suppressor clone (pJK537), and <br /> start site longest RNA transcript processed    MG1655 carrying control plasmid (pJK333); (B) Coomassie blue- <br /> give smaller RNA molecules. The 1-kb RNA observed          stained gel lysed 35S-labeled minicells prepared strain DS410 <br /> analysis RNA described probably full-length       carrying parental plasmid (pBR322), suppressor clone <br /> transcript.                                                       (pJK537), control plasmid (pJK334; pJK334 same <br />    ORF2 encodes dnaK suppressor gene, dksA,        DNA fragment opposite orientation pJK333); (C) <br /> previously unidentified gene. There nonoverlapping        autoradiogram panel B. Molecular weights (in thousands) are <br />                                                                   shown left. Notation: BLA, P-lactamase; TET, tetracycline <br /> open reading frames sense strand putative dnaK <br />                                                                   resistance protein; *, putative dksA gene product. The upper band in <br /> suppressor gene region large encode       lane pJK334 C P-lactamase fusion protein initiated <br /> than 42 amino acids (Fig. 3). Neither pJK530 pJK533,          promoter activity DNA insert pJK334. <br /> which covers ORF1 ORF2 region, respectively, had <br /> suppressing activity. Si mapping DNA sequence analy- <br /> sis suggested RNA transcript starts position     growth defect dnaK deletion mutant. The dksA clone was <br /> 166 original RNA transcript region. To identify    originally selected ability suppress temperature- <br /> which open reading frame actually encodes suppressor       sensitive growth dnaK deletion mutant strain PK101. <br /> dnaK deletion, mutant clones constructed vitro        Since dnaK deletion strain extremely filamentous <br /> tested suppressing activity (Fig. 3). A mutation inactivat-   30&#176;C, dksA clone tested determine <br /> ing  ORF1    (pJK540) did affect suppressing activity,   suppress filamentous phenotype. <br /> whereas mutations inactivating ORF2 (pJK542               Cells carrying dksA clone, pJK537, showed greatly re- <br /> pJK543) lost ability suppress. pJK541, carries       duced cell length 30 37&#176;C partially reduced <br /> the entire ORF2 ORF1, showed suppressing           filamentation 40.5&#176;C (Fig. 8). Cells carrying control <br /> activity. These results indicated ORF2 encodes           plasmid, pJK333, did reduction filamentation <br /> suppressor dnaK deletion. We think pJK541 does        30&#176;C. This result indicated dksA clone <br />  <br /> suppress lacks transcription initiation site.        suppressed temperature-sensitive growth but <br /> Neither nucleotide amino acid sequence deduced           filamentous phenotype dnaK deletion strain. <br /> ORF2 showed significant homology protein sequence             Since dnaK deletion mutant strain unable to <br /> in NBRF data base DNA sequence GenBank.             support growth phage X, dksA clone tested see <br /> We designated newly identified gene dksA (dnaK          suppress phage growth defect. The <br /> suppressor).                                                      dnaK deletion mutant strain PK101, carrying dksA clone, <br />    Identification dksA gene product. To identify       pJK537, support phage X growth (data shown). <br /> dksA gene product, polypeptides encoded plasmids           This result suggested role DnaK inA growth is <br /> pJK537 pJK334 analyzed cells carrying dksA        separable host growth. <br /> gene multicopy plasmid minicells (Fig. 7). A             dksA suppresses grpE280, dnaK756, dnaJ <br />  strong band position expected 18,000-Mr            deletion mutant        phenotypes. Since            shown            both <br />  <br /> polypeptide detected MG1655 carrying pJK537           biochemically genetically DnaK GrpE <br />  total cell extracts analyzed SDS-PAGE. In contrast,      proteins interact (16), ability dksA clone to <br />  MG1655 carrying plasmid carrying control            suppress temperature-sensitive growth filamen- <br />  plasmid, pJK333, strong band position ob-         tation grpE280 mutant cells tested. The grpE280 <br />  served. The polypeptides encoded suppressor clone,        mutant strain carrying dksA clone showed 7,000- <br />  pJK537, analyzed minicells. Minicells          fold-higher plating efficiency 41&#176;C did grpE280 <br />  prepared strain DS410 carrying parental plasmid         strain carrying control plasmid. The dksA clone suppressed <br />  pBR322, suppressor clone pJK537, control plasmid,       filamentation grpE280 37&#176;C 41&#176;C (Fig. 9). <br />  pJK334. The minicells labeled L-[35S]methionine        The phage X growth defect grpE280 mutant not <br />  10 min. Minicells carrying suppressor clone, pJK537,     suppressed dksA clone (data shown). <br />  showed labeled 18,000-Mr polypeptide comigrated              Since dnaK deletion strain PK101 used study is <br />  overexpressed 18,000-Mr protein synthesized          actually DnaK- DnaJ-, dnaJ deletion mutant strain <br />  cells carrying suppressor clone. The observed          PK102 tested dnaJ deletion <br />  molecular weight consistent deduced       mutant phenotypes suppressed dnaK suppressor <br />  DNA sequence dksA gene.                                   clone. The dnaJ deletion mutant strain PK102, carrying the <br />     dksA suppresses filamentation phageA     dksA clone pJK537, temperature-resistant at <br />  2062    KANG AND CRAIG                                                                                                  J. BACTERIOL. <br />  <br />  <br />  <br />  <br /> PKIO1/ <br />  pJK537 <br />  <br />  <br />  <br />  <br /> P K 1 0 1/ <br />  pJK333 <br />  <br />  <br />  <br />  <br />                             300C                                         370C                                       40.5&#176;C <br />   FIG. 8. Suppression filamentation dnaK deletion strain dnaK suppressor clone (pJK537). pJK333 control plasmid. <br />  <br /> 44.5&#176;C, strain PK102 carrying control plasmid                recB recC sbcB strain poor, second P1 lysate was <br /> pJK333 viable temperature. The dksA clone            prepared the Kanr Tetr transductants (PK9), <br /> also suppressed cell elongation dnaJ deletion             MG1655 cells transduced. When transduc- <br /> mutant strain 44.5&#176;C (data shown).                       tants selected Kanr, 70% Tetr. A similar <br />    Since dksA multicopy suppressor originally iso-           value (77% cotransduction) obtained transduc- <br /> lated basis suppressing dnaKJ4 dnaJ14 deletion         tants selected Tetr. This result suggested that <br /> mutation, ability dksA suppress dnaK756                dksA essential gene. However, eliminate <br /> missense mutation tested. As case dnaK14           possibility presence closely linked mutation <br /> dnaJ14 deletion mutation, temperature-sensitive growth           suppress dksA loss. <br /> cell elongation caused dnaK756 mutation sup-               The dksA deletion strain PK201 grow nearly <br /> pressed dksA clone, phage A growth defect             wild-type cells rich media 30, 37, 42&#176;C. <br /> was (data shown).                                            However, grew poorly M9 minimal medium <br />    To determine dksA suppress heat               supplemented 20 amino acids vitamins tem- <br /> shock protein mutant strains, lon::TnJO, groEL140,           peratures. This poor growth overcome changing <br /> groES30 mutant strains tested. The growth phe-             carbon sources. Phage A growth normal dksA <br /> notypes strains suppressed                deletion strain. The double mutant dnaK deletion and <br /> overexpression dksA gene (data shown). There-             dksA deletion constructed. No phenotypic <br /> fore, dksA does ability suppress heat            difference observed dnaK deletion strain <br /> shock mutants specific dnaK, dnaJ,             dnaK dksA double-deletion strain (data shown). <br /> grpE mutant strains. <br />    dksA essential gene. Since dksA newly                                     DISCUSSION <br /> identified gene, constructed dksA mutant determine <br /> the effect lack dksA function. A plasmid carrying          We isolated previously unidentified E. coli gene, <br /> deletion dksA, pJK21, constructed removing             dksA, suppress temperature-sensitive growth <br /> 408-bp ClaI-PvuII fragment pJK538 replacing           dnaK deletion mutant dosage-dependent manner. <br /> the Kanr cassette. This vitro-mutated copy dksA            The filamentation dnaK deletion mutant strain also <br /> integrated E. coli chromosome linear transfor-           suppressed, defect A growth not. Although the <br /> mation strain TnJO 70 80% linked dksA              dksA clone isolated genomic library DNA from <br /> (zae-502::TnJO). Transformants selected Kanr            spontaneous temperature-resistant suppressor strain, the <br /> screened ampicillin sensitivity (Amps) confirm recom-         dksA locus original suppressor <br /> bination chromosome. P1 vir grown             mutation site, dksA clone isolated wild-type <br /> the Kanr Tetr Amps transformants, wild-type strain           strain rescued temperature-sensitive growth and <br /> MG1655 transduced Kanr Tetr. Since P1 growth           filamentation dnaK deletion strain PK101 over- <br />  VOL. 172, 1990                                                 CHARACTERIZATION OF A dnaK SUPPRESSOR GENE                         2063 <br />  <br />  <br />  <br />  <br /> PK5/                                                                       4.4 <br />  pJK537&quot;&apos;                                                                                * <br />  <br />  <br />                                  521                                             01l_;.i6Fte&gt;   i <br />                                                                                                     ll <br />  <br />  <br />  <br />  <br />                                                                                                              jlk&apos;~~f <br />  <br />  <br />  <br />  <br /> PK5/ <br />  pJK333 <br />  <br />  <br />  <br />                             300C                                          370C                                         410C <br />   FIG. 9. Suppression filamentation grpE280 mutant strain dnaK suppressor clone (pJK537). pJK333 control plasmid. <br />  <br /> expressed. Examples dosage-dependent suppression               It clear overexpression dksA suppresses the <br /> been reported previously. Kusukawa et al. (18) showed           temperature-sensitive growth filamentation dnaK <br /> overproduction DnaK GroE protein suppressed tem-               deletion mutant strain. It possible DksA bypasses <br /> perature-sensitive growth rpoH deletion mutant                DnaK protein function(s). Although dksA <br /> strain. Aldea et al. (2) cloned newly identified gene,        gene dispensable gene, DksA protein <br /> bolA, suppressed wee amber mutation                   important cell growth high temperature the <br /> overexpressed.                                                       dnaK mutant background. The dksA mRNA level the <br />    Although dnaK deletion mutant strain formed colonies          dnaK deletion mutant strain PK101 threefold lower in <br /> at 30&#176;C, original transductants grew extremely slowly            wild-type strain. In addition, dksA mRNA level the <br /> and filamentous. However, cells                spontaneously isolated suppressor strain PK113 fourfold <br /> filamentous generations growth 30&#176;C.             higher deletion strain (unpublished result). There- <br /> Suppressor mutations appear accumulate dnaK                fore, possible dnaK mutant phenotypes <br /> deletion mutant strain 30&#176;C. Bukau Walker (7)            lowered expression dksA suppres- <br /> have similar observations. Additional observations              sion phenotypes original suppressor strain is <br /> from laboratory support <span id='am-34' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-35'  ></span><span id='am-36' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>idea</span>. In initial           increased expression dksA. It also <br /> screening suppressor clone, isolated dnaK deletion         possible transcription certain genes inhibited the <br /> cells transformed suppressor plasmid             dnaK mutant strain. Mutation(s) overexpression cer- <br /> grow 42&#176;C. However, additional studies shown            tain proteins results increase transcription of <br /> P1 transduction dnaK deletion allele wild-type         RNAs bypasses function encoded <br /> strain carrying isolated suppressor clone resulted        proteins suppress dnaK mutant phenotypes. <br /> formation viable colonies temperatures high            It interesting overexpression dksA suppressed <br /> 40.5&#176;C. This failure transductants grow 42&#176;C indi-          temperature-sensitive growth dnaK deletion <br /> cated additional mutations dnaK               mutant strain, defect phage k growth. This result <br /> deletion mutant strain affected growth high tempera-         suggests functions DnaK host growth A <br /> tures. In fact, genetic evidence         growth separable. Multiple copies dksA suppress <br /> mutations, tightly linked dnaK,            phenotypes temperature-sensitive grpE <br /> originally isolated spontaneous temperature-resistant sup-           mutant, allowing growth grpE280 mutant 41&#176;C and <br /> pressor strain PK113 required temperature-resistant          suppressing filamentation caused mutation. This <br /> growth (unpublished result). It possible       suppression specific sense heat <br /> mutations sufficient suppress temperature-           shock genes, groEL, groES, Ion, affected. DnaK <br /> sensitive growth present multiple copies. This          GrpE interact vivo vitro (16). Perhaps <br /> would explain isolated wild-type gene           dksA bypasses functions carried dnaK grpE <br /> mutant genes responsible growth high temperatures.            needed growth high temperatures normal cell <br />  2064        KANG AND CRAIG                                                                                                      J. BACTERIOL. <br />  <br /> division. Recently, Ang Georgopoulos reported                     70K heat shock gene related genes <span id='am-24' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-26'  ></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span> <br /> grpE deletion mutation viable dnaK mutant                         cerevisiae. Mol. Cell. Biol. 4:1454-1459. <br /> strains grpE normally essential gene (3).              14. Georgopoulos, C. P. 1977. A new bacterial gene (groPC) which <br /> This viability suppressors dnaK mutant                     affects lambda DNA replication. Mol. Gen. Genet. 151:35-39. <br />                                                                          15. Itikawa, H., J.-I. Ryu. 1979. Isolation characterization <br /> strains, dnaK mutation itself. These results                      <br /> </body></html>