<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Apr. 1997, p. 2057&#8211;2065                                                                        Vol. 17, No. 4 <br /> 0270-7306/97/$04.0010 <br /> Copyright q 1997, American Society Microbiology <br />  <br />  <br />  <br />              <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> BUR6 Encodes DRAP1/NC2a <br />                   Homolog That Has Positive Negative <br />                          Roles Transcription In Vivo <br />                                                        GREGORY PRELICH* <br />                   Department Molecular Genetics, Albert Einstein College Medicine, Bronx, New York 10461 <br />                     Received 16 September 1996/Returned modification 24 October 1996/Accepted 16 January 1997 <br />  <br />              BUR3 BUR6 identified previously selecting mutations increase transcription an <br />           upstream activating sequence (UAS)-less promoter <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29'  ></span>Saccharomyces</span> cerevisiae. The bur3-1 bur6-1 muta- <br />           tions recessive, increase transcription suc2Duas allele, cause mutant phenotypes, <br />           suggesting Bur3p Bur6p function general repressors basal transcriptional machinery. The <br />           molecular cloning characterization BUR3 BUR6 presented here. BUR3 identical MOT1, a <br />           previously characterized essential gene encodes ATP-dependent inhibitor TATA box-binding <br />           protein. Cloning nucleotide sequence analysis reveals BUR6 encodes homolog DRAP1 (also called <br />           NC2a), mammalian repressor basal transcription. Strains contain bur6 null allele viable but <br />           grow extremely poorly, demonstrating BUR6 critical normal cell growth yeast. The Bur6p histone <br />           fold domain required function; extensive nonoverlapping set deletion alleles histone <br />           fold domain impairs BUR6 function vivo, mutations amino- carboxy-terminal tails have <br />           detectable effect. BUR6 BUR3/MOT1 different functions depending promoter context: although <br />           bur3-1 bur6-1 mutations increase transcription Duas promoters, result reduced tran- <br />           scription wild-type GAL1 GAL10 promoters. <span id='am-62' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-63' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-66'  ></span>This</span> transcriptional defect inability of <br />           GAL10 UAS function bur6-1 strains. The similar phenotypes bur6 bur3 (mot1) mutations <br />           suggest Bur6p Mot1p related, identical, functions modulating activity the <br />           general transcription machinery vivo. <br />  <br />  <br />    Basal transcription most, all, protein-encoding                 28, 36, 43, 47) indicates basal promoter elements are <br /> genes requires activity RNA polymerase II                     capable higher levels transcription vivo nor- <br /> general transcription factors (GTFs) TFIIA, TFIIB, TFIID,                    mally repressed wild-type strains. A number recent stud- <br /> TFIIE, TFIIF, TFIIH (66). Highly purified preparations                ies begun identify classes proteins that <br /> RNA polymerase II GTFs necessary                        repress GTFs distinctly different mechanisms vivo. <br /> sufficient direct promoter-specific basal transcription               One <span id='am-59' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-60' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> general transcriptional repression activity in <br /> correct initiation sites number model templates vitro.            vivo assembly DNA template nucleo- <br /> The levels transcription established basal               somes (20). Nucleosomes inhibit recognition regulatory <br /> machinery regulated positively negatively              elements UAS-binding proteins (45, 58), re- <br /> a promoter-specific manner mediated <span id='am-90' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-91'  ></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>cis</span>-acting regu-                 ducing activated transcription, nucleosomes affect <br /> latory elements. The mechanism regulatory factors                   basal transcription inhibiting function GTFs (33, 64). <br /> influence activities GTFs remains area intense              For example, assembly adenovirus major late promoter <br /> interest, transcription initiation usually major              nucleosomes represses transcription vitro the <br /> regulatory step differential gene expression per-                template incubated TFIID prior assembly (64). This <br /> forms critical roles normal cell growth develop-               result suggests nucleosomes inhibit binding TFIID to <br /> ment differentiation multicellular organisms.                         TATA box. Recent studies utilizing purified recombinant <br />    Regulators general transcription machinery                  TATA-binding protein (TBP) demonstrated nucleosomes <br /> grouped main classes. The best-characterized class               directly inhibit binding TBP, confirming extending the <br /> regulators contains site-specific DNA-binding proteins              results obtained crude TFIID fraction (24). The in- <br /> recognize cis-acting regulatory elements known upstream                   hibitory effects nucleosomes basal transcription also <br /> activating sequences (UASs) Saccharomyces cerevisiae                   demonstrated vivo; depletion nucleosomes yeast <br /> enhancers larger eukaryotes (40, 55). A second class                   results increased constitutive expression PHO5, <br /> transcriptional regulators affects general transcription ma-             CYC1, GAL1, CUP1, HIS3 basal promoters fused lacZ <br /> chinery UAS-independent manner. UAS-less promoters                      (15, 21). Furthermore, mutations genes encoding his- <br /> are transcribed RNA polymerase II GTFs vitro <br />                                                                              tones H2A H2B alter transcription promoters that <br /> yet transcribed poorly, all, vivo. The <br />                                                                              contain insertions Ty elements (Spt2 phenotype) in- <br /> identification recessive mutations increase transcription <br />                                                                              crease transcription suc2Duas promoter (9, 22). Nu- <br /> from UAS-less promoters yeast S. cerevisiae (7, 11, 13, <br />                                                                              cleosomal repression likely require factors just <br />                                                                              core histones, genetic selections yeast identi- <br />    * Mailing address: Department Molecular Genetics, Albert Ein-          fied additional factors likely inhibit transcription <br /> stein College Medicine, 1300 Morris Park Ave., Bronx, NY 10461.           chromatin effects (8, 48, 54, 62). <br /> Phone: (718) 430-2181. Fax: (718) 430-8778. E-mail: Prelich@aecom               Another class repressors directly inhibits activity of <br /> .yu.edu.                                                                     GTFs chromatin-independent manner. Proteins such <br />  <br />                                                                       2057 <br />  2058       PRELICH                                                                                                                             MOL. CELL. BIOL. <br />  <br />  <br /> as Mot1, Dr1 (also called NC2), NC1 (2, 25, 37)                                             TABLE 1. S. cerevisiae strains <br /> identified having biochemical activities inhibit func-                  Strain                                       Genotype <br /> tion TBP subunit TFIID vitro. Mot1p displaces <br /> TBP TATA box ATP-dependent manner (2),                              GY139.................MATa/a his4-912d/his4-912d lys2-128d/lys2-128d <br /> while NC1 Dr1/NC2 allow TBP bind DNA inhibit                                                   suc2Duas(-1900/-390)/suc2Duas(-1900/-390) ura3- <br />                                                                                                           52/ura3-52 trp1D63/trp1D63 LEU21/leu2D1 <br /> subsequent interactions TBP TFIIA and/or <br />                                                                                    GY215.................MATa his4-912d lys2-128d ura3-52 trp1D63 bur6-1 <br /> TFIIB (25, 32). The vivo role of TBP inhibitors                   GY218.................MATa his4-912d lys2-128d ura3-52 ade8 bur6-1 <br /> is clear, phenotype mot1 mutations con-                      GY236.................MATa his4-912d lys2-128d ura3-52 leu2D1 mot1-301 <br /> sistent proposed repressor activity, mot1 muta-                     GY315.................MATa his4-912d lys2-128d ura3-52 trp1D63 <br /> tions cause increased transcription DUAS promoters                         GY319.................MATa his4-912d lys2-128d ura3-52 ade8 <br /> vivo (13). An important issue raised studies                   GY561.................MATa his4-912d lys2-128d suc2Duas(-1900/-390) <br /> analogous activities specifically inhibit components                                           ura3-52 leu2D1 bur6-1 <br /> of basal transcription machinery exist.                                        GY562.................MATa/a his4-912d/his4-912d lys2-128d/lys2-128d <br />    We used genetic selection designed identify mu-                                              suc2Duas(-1900/-390)/suc2Duas(-1900/-390) ura3- <br />                                                                                                           52/ura3-52 trp1D63/trp1D63 LEU21/leu2D1 <br /> tations affect basal transcription S. cerevisiae (47), with <br />                                                                                                           BUR61/bur6D2::TRP1 <br /> the expectation mutations identify additional                      GY568.................MATa his4-912d lys2-128d suc2Duas(-1900/-390) <br /> chromatin-dependent chromatin-independent repressors                                                  ura3-52 trp1D63 leu2D1 bur6D2::TRP1 (CEN <br /> of basal machinery. The promoter used selection                                              URA3 BUR61) <br /> was suc2Duas(-1900/-390) allele (51), produces <br /> barely detectable levels transcripts, causing inability <br /> to grow media contain sucrose carbon source <br /> (Suc2 phenotype). A selection Suc1 suppressors                              used study. Yeast cells transformed lithium acetate <br /> suc2Duas(-1900/-390) identified mutations previously                       method (27). <br />                                                                                       <span id='am-100' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Plasmids</span>. pGP256, original BUR61 plasmid isolated a <br /> characterized SPT genes, including encode histones                      YCp50-based yeast genomic DNA library (49), contains 20,007-bp Sau3A <br /> H2A H2B. However, genes identified                         insert BamHI site YCp50. A 2,393-bp BglII fragment contains <br /> by selection designated BUR1 BUR6 (for                       BUR6 subcloned pGP256 BamHI site pRS416 create <br /> bypass UAS requirement) (47). In addition suppressing                           pGP258. To construct bur6 null allele, 820-bp EcoRI-StuI fragment TRP1 <br />                                                                                    used replace NdeI-KpnI fragment BUR6, generating the <br /> suc2Duas, bur mutations causes mutant phe-                       bur6D2::TRP1 null allele pGP267. To integrate null allele BUR6 <br /> notypes, indicating general effects                   genomic locus, 2.4-kb BglII fragment pGP267 containing bur6D2::TRP1 was <br /> that likely affect expression wild-type                      transformed diploid strain GY139 Trp1 transformants se- <br /> promoters. Further evidence BUR gene products                        lected. Integration BUR6 locus confirmed Southern blotting. The <br />                                                                                    2.4-kb BglII fragment cloned BamHI site pRS416 to <br /> important general roles transcription provided                        create pGP273. Plasmids containing hoDuas-lacZ reporter (p740), HO-lacZ <br /> ability bur mutations suppress transcriptional de-                      (p1701), cyc1Duas-lacZ reporter (pLGD2178) obtained from <br /> fects caused snf5 mutation (47). Snf5p component                     David Stillman, GAL10UAS-CYC1p-lacZ (pLGSD5) GAL4BS- <br /> the highly conserved SNF-SWI complex, required                        CYC1p-lacZ (pSV14) plasmids obtained Karen Arndt. <br />                                                                                       RNA analysis. Cells grown 1 3 107 2 3 107 cells ml, RNA <br /> transcription diversely regulated promoters vivo                        isolated described previously (6). RNA separated 1% formalde- <br /> (42, 44). Genetic biochemical evidence indicates                      hyde agarose gel. Blotting hybridization DNA probes performed as <br /> SNF-SWI complex helps TBP site-specific activators bind                        described previously (56). The probes used pDE32-1 (SPT15) an <br /> to nucleosomal DNA (12, 23, 24, 34). Based distinct                       ;1.8-kb EcoRI fragment p4812 (GAL1 GAL10). RNA cross- <br />                                                                                    linked GeneScreen (New England Nuclear) auto cross-link mode in <br /> sets unselected mutant phenotypes, including ability                     Stratalinker 1800 (Stratagene). Probes radiolabeled 32P a <br /> suppress snf5 mutations, proposed (47) bur                         random priming kit Boehringer Mannheim according manufacturer&#8217;s <br /> mutations comprise groups suppress suc2Duas dif-                       directions. <br /> ferent mechanisms. One group contains BUR1, BUR2, BUR4,                               b-Galactosidase assays. Five-milliliter yeast cultures grown density <br />                                                                                    1 3 107 2 3 107 cells ml SC-Ura medium select reporter <br /> and BUR5, group consists BUR3 BUR6.                         plasmids. Cells harvested centrifugation resuspended 250 ml <br /> Since BUR5 encodes histone H3 (47), group BUR                         breaking buffer (0.1 M Tris [pH 8.0], 20% glycerol, 1 mM dithiothreitol, 1 mM <br /> genes proposed suppress suc2Duas nucleosomal                            phenylmethylsulfonyl fluoride). Cells disrupted vortexing times 1 <br /> repression second group proposed suppress                         min 48C presence equal volume glass beads. More breaking <br />                                                                                    buffer (250 ml) added, tubes vortexed briefly, liquid <br /> suc2Duas nucleosome-independent mechanism,                            transferred 1.5-ml tube centrifuged 15 min 48C remove <br /> by directly repressing basal transcription factors. The re-                    cellular debris. The supernatants assayed b-galactosidase activity as <br /> sults presented support hypothesis: BUR3 identical                    described previously (39). Protein levels determined method of <br /> to MOT1 (13), encodes inhibitor TBP,                         Bradford (5a) bovine serum albumin protein standard. All reported <br />                                                                                    b-galactosidase levels means independent transfor- <br /> BUR6 encodes histone fold-containing protein signifi-                       mants standard errors ,20%. <br /> cant sequence similarity mammalian repressor DRAP1                             Construction bur6 deletion alleles. Deletion alleles bur6 created by <br /> (also called NC2a). Surprisingly, analysis transcriptional                      oligonucleotide-directed mutagenesis plasmid pGP258. pGP258 contains <br /> defects bur3-1 bur6-1 mutant strains suggests MOT1                     BUR6 2.4-kb BglII fragment cloned BamHI site pRS416. pGP258 <br />                                                                                    transformed dut1 ung1 mutant <span id='am-67' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-69'  ></span><span id='am-70' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span>Escherichia coli</span> strain RZ1032, and <br /> and BUR6 positive roles promoters                         single-stranded DNA prepared infection M13 helper phage <br /> vivo.                                                                              K07. The single-stranded DNA mutagenized Mut-a-gene kit from <br />                                                                                    Bio-Rad Laboratories according manufacturer&#8217;s directions. All deletion <br />                        MATERIALS AND METHODS                                       mutations verified DNA sequencing entire BUR6 open reading <br />                                                                                    frame. <br />    Media genetic methods. S. cerevisiae strains used study shown      Western analysis. Cells grown mid-log phase 10 ml dropout <br /> in Table 1. All strains used study derived FY2 (MATa ura3-52),   medium, harvested, lysed vortexing presence glass beads a <br /> a GAL21 derivative S288C (63). All media used, including rich medium            buffer contained 50 mM Tris (pH 7.5), 150 mM NaCl, 10 mM MgCl2, 1 mM <br /> (YPD), synthetic complete drop-put medium (for example, SC-Ura), minimal           EDTA, 1 mM dithiothreitol, 1 mM phenylmethylsulfonyl fluoride, 10% glycerol, <br /> medium (SD), sporulation medium, described previously (50).       200 mg aprotinin ml, 100 mg peptstatin A ml, 50 mg leupeptin <br /> YPSuc plates contained YEP, 2% sucrose, 1 mg antimycin A (Sigma)        ml. Debris cell lysates pelleted 15 min 16,000 3 g. Twenty <br /> ml. YPGal plates contained YEP, 2% galactose, 1 mg antimycin A ml.      micrograms protein extract separated sodium dodecyl <br /> Standard genetic methods mating, sporulation, tetrad analysis (50)    sulfate&#8211;15% polyacrylamide gel. Proteins transferred Immobilon P (Mil- <br />  VOL. 17, 1997                                                                       BUR6 BOTH ACTIVATES AND REPRESSES IN YEAST                              2059 <br />  <br />  <br />                                                                                      identical previously cloned genes cause similar <br />                                                                                      mutant phenotypes. The bur3-1 mutation comple- <br />                                                                                      mented plasmids contained SIN3, SIN4, GAL11, RPB1, <br />                                                                                      RPB2, previously characterized SPT genes. In <br />                                                                                      contrast, MOT1 CEN plasmid complemented pheno- <br />                                                                                      types bur3-1, including Bur2, Ts2, Gal2, slow- <br />                                                                                      growth phenotypes. To determine bur3-1 actually <br />                                                                                      mutation MOT1, complementation linkage tests were <br />                                                                                      performed. The bur3-1 mot1-1 mutations unable to <br />                                                                                      complement heterozygous diploids were <br />                                                                                      tightly linked, displaying recombination 20 four-spored <br />                                                                                      tetrads. BUR3 identical MOT1, essential gene <br />                                                                                      previously shown encode inhibitor TBP (3). The bur3-1 <br />                                                                                      allele hereinafter referred mot1-301. <br />                                                                                         Molecular cloning sequence analysis BUR6. The find- <br />                                                                                      ing BUR3 identical MOT1 prompted charac- <br />                                                                                      terization BUR6 molecular level, bur6-1 and <br />    FIG. 1. Complementation bur6-1 phenotypes BUR61 clones. Yeast               mot1-301 mutations cause similar mutant phenotypes. The <br /> strain GY561 (relevant genotype, bur6-1 suc2Duas ura3-52) transformed            BUR6 gene cloned transforming bur6-1 strain a <br /> with pRS416 (the CEN vector), pGP256 (the original CEN BUR61 plasmid), or <br /> pGP258 (a CEN BUR61 subclone; Fig. 2 insert details), transfor- <br />                                                                                      yeast genomic DNA library centromeric vector (49) by <br /> mants selected SC-Ura plates. A single transformant plasmid        screening plasmids complemented bur6-1 Gal2 <br /> was restreaked SC-Ura plate replica plated SC-Ura          phenotype. A single plasmid obtained designated <br /> (glucose), SC-Ura plus galactose plus antimycin A (galactose), SC-Ura plus       pGP256. In addition complementing Gal2 phenotype of <br /> sucrose plus antimycin A (sucrose). <br />                                                                                      bur6-1, pGP256 complemented Bur2 slow-growth <br />                                                                                      phenotypes (Fig. 1). <br />                                                                                         To identify location BUR6 plasmid insert, <br /> lipore), filter probed anti-Bur6 antibody (gift E. Gadbois <br /> and R. Young). Primary antibody washed phosphate-buffered saline            chromosomal region contained pGP256 character- <br /> plus 0.1% Tween 20, filter probed horseradish peroxi-        ized sequencing ends pGP256 insert. A compar- <br /> dase-conjugated anti-rabbit secondary antibody (Amersham). Final detection           ison nucleotide sequence sequences Gen- <br /> was enhanced chemiluminescence kit (Amersham) according          Bank database revealed pGP256 contained portions the <br /> manufacturer&#8217;s instructions. <br />                                                                                      SPT2 BEM2 open reading frames end. The <br />                                                                                      nucleotide sequence SPT2-BEM2 region chromosome <br />                            RESULTS                                                   V kindly provided F. Dietrich, revealing addition <br />    BUR3 identical MOT1. The Bur selection identified                           amino terminus BEM2 carboxy terminus of <br /> mutations genes appear comprise phenotypic                        SPT2, pGP256 contained Ty element, tRNAArg gene, <br /> classes. One class consists BUR1, BUR2, BUR4, BUR5,                           previously <span id='am-105' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-113'  ></span><span id='am-114' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-119' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span>unidentified</span> open reading frames (Fig. 2). <br /> while class consists BUR3 BUR6. Mutations                        A 2.4-kb subclone (pGP258) contained open <br /> BUR3 BUR6 similar bur                              reading frame tRNAArg gene able fully comple- <br /> mutations cause Gal2 weak Spt2 phenotypes                           ment phenotypes caused bur6-1, indicating that <br /> also unable suppress snf5D allele. Before attempting                        BUR6 contained 2.4-kb fragment (Fig. 1). To <br /> clone BUR3 standard plasmid complementation re-                            determine complementation open <br /> cessive bur3-1 allele, tested BUR3                         reading frame tRNAArg gene, deletion allele re- <br />  <br />  <br />  <br />  <br />    FIG. 2. Identification BUR6 open reading frame. Plasmid pGP256 original BUR61 complementing plasmid isolated yeast genomic CEN library. <br /> The locations open reading frames 20-kb insert shown filled arrows, approximate extents interrupted SPT2 BEM2 open reading <br /> frames indicated broken open arrows. The location tRNAArg gene shown filled box. pGP258 contains 2.4-kb subclone pGP256 encodes only <br /> the BUR6 open reading frame tRNAArg gene. pGP273 identical pGP258, BUR6 open reading frame replaced TRP1 <br /> gene. The ability plasmid complement bur6-1 depicted right. <br />  2060       PRELICH                                                                                                                                  MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 3. Similarities Bur6p selected histone fold proteins. (A) The BUR6 coding region aligned histone fold proteins gave highest <br /> similarity scores BLAST search sequences current GenBank database. The region similarity shaded protein, percentages amino <br /> acid identity similarity shown shaded histone fold domains. Amino acid residues numbered protein. The accession numbers the <br /> indicated sequences follows: BUR6, U32274; hDRAP1, U41843; <span id='am-47' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53'  ></span>Caenorhabditis elegans</span> open reading frame, Z70753; <span id='am-30' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-32'  ></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span>Schizosaccharomyces</span> pombe open <br /> reading frame, Z69795; HAP5, U19932; S. cerevisiae HTA1, P04911. (B) Alignment similar amino acids human DRAP1/NC2a Bur6p. The proposed <br /> locations helices comprise histone fold shown alignment. <br />  <br />  <br />  <br /> placed nearly open reading frame TRP1, left                        transcriptional repressor inhibits interactions between <br /> the tRNAArg gene intact, created. This bur6D2::TRP1 allele                         TBP GTFs TFIIA and/or TFIIB vitro (19, 38). <br /> was unable complement bur6-1 present                               Inhibition TBP DRAP1/NC2a occurs het- <br /> CEN plasmid, indicating short open reading frame                          erodimeric complex DRAP1/NC2a Dr1/NC2b. The <br /> responsible plasmid-borne complementation activity.                            similarity Bur6p DRAP1/NC2a highly signifi- <br /> Linkage analysis confirmed pGP258 contains authentic                              (45% identity 72% similarity 79 amino acids) <br /> BUR6 dosage-dependent suppressor bur6-1 (<span id='am-95' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-96' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-99'  ></span>data</span>                             extends slightly carboxy-terminal end the <br /> not shown).                                                                            histone fold domain. Deletion analysis (described below) dem- <br />    A BLAST comparison BUR6 nucleotide sequence                                  onstrates importance DRAP1/NC2a-homologous <br /> with sequences current GenBank database revealed                           domain Bur6p function vivo. Furthermore, recent bio- <br /> BUR6 encodes 142-amino-acid protein significant se-                             chemical studies (17, 18, 31) indicate Bur6p directly in- <br /> quence similarity histone H2A members                              teracts yeast Dr1 homolog Bur6p-yDr1 <br /> histone fold <span id='am-44' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> proteins (Fig. 3). The histone fold                        complex represses transcription vitro. Bur6p encodes a <br /> structural motif approximately 65 amino acids length                             yeast homolog DRAP1/NC2a. <br /> originally identified crystallographic analysis his-                          bur6 null phenotype. To determine effects complete <br /> tone octamer (1), domain predicted (4)                         loss BUR6 function, bur6 null allele constructed and <br /> and demonstrated (65) occur number nonhistone                               integrated genome diploid yeast strain, replacing <br /> proteins. The histone fold constitutes interface specific                       wild-type BUR6 alleles. When heterozygous <br /> and direct interactions histone fold proteins. For                         BUR61/bur6D2::TRP1 diploids sporulated tetrads <br /> example, histone fold domains responsible exten-                           dissected, resulting tetrads produced healthy col- <br /> sive direct interactions histones H2A H2B,                         onies microcolonies easily visible the <br /> histones H3 H4 (1), HAP3/CBF-A HAP5/                                   naked eye 7 10 days growth (Fig. 4). Although <br /> CBF-C (53), Drosophila TAFII42 (dTAFII42)                              bur6D2::TRP1 cells viable, remarkably poor <br /> dTAFII62 (65). The score showing highest similarity BUR6                            growth relatively high frequency reversion prohibited <br /> was DRAP1/NC2a, human protein recently identified                            analysis bur6D strains. The growth defect was <br />  VOL. 17, 1997                                                                     BUR6 BOTH ACTIVATES AND REPRESSES IN YEAST                                   2061 <br />  <br />  <br />                                                                                    93 Bur6p 66% identical 83% similar human <br />                                                                                    DRAP1 (Fig. 3). To determine redundancy the <br />                                                                                    amino-terminal tail accounts lack phenotype the <br />                                                                                    N-terminal deletions, entire N-terminal tail residues <br />                                                                                    2 40 deleted, apparent effect BUR6 <br />                                                                                    function. Western analysis (Fig. 5B) indicated non- <br />                                                                                    functional alleles produce detectable amounts mutant <br />                                                                                    proteins. In fact, nonfunctional deletion alleles generally <br />                                                                                    produced higher levels Bur6 protein functional dele- <br />                                                                                    tion derivatives, suggesting BUR6 regulated a <br />                                                                                    negative feedback loop. These results demonstrate impor- <br />                                                                                    tance BUR6 histone fold domain vivo suggest that <br />                                                                                    domain sufficient function, ami- <br />                                                                                    no- carboxy-terminal tails BUR6 appear dispens- <br />                                                                                    able vivo assays. <br />                                                                                       bur6-1 increases expression DUAS reporter genes. <br />                                                                                    Because bur6-1 increased transcription suc2Duas pro- <br />                                                                                    moter caused mutant phenotypes, proposed <br />                                                                                    (47) Bur6p function repressor general <br />    FIG. 4. BUR6 essential normal cell growth. (A) Yeast strain GY562        transcription machinery. If true, bur6-1 be <br /> (BUR6/bur6D2::TRP1) transformed URA3 vector sporulated,         expected increase expression promoters than <br /> 10 tetrads dissected. Colonies derived individual spores la-     suc2Duas, previously shown mot1-1 muta- <br /> beled d. The dissection plate photographed day 3, bur6D <br /> colonies visible aid microscope. Identical results    tion (13). To test prediction, effects bur6-1 <br /> obtained GY562 dissected absence vector. (B) The       mutation expression cyc1Duas hoDuas core <br /> strain (GY562) transformed pGP258 (CEN BUR61) prior sporula- <br /> tion, tetrads dissected photographed described panel A. <br />  <br />  <br />  <br /> directly attributable bur6D2::TRP1, poor growth and <br /> the Trp1 phenotype cosegregated precisely 100 <br /> tetrads. Furthermore, growth defect rescued by <br /> transforming heterozygous diploid strain BUR61 <br /> URA3 plasmid prior sporulation; tetrad dissection, <br /> viable healthy bur6D2::TRP1 colonies obtained (Fig. <br /> 4) Trp1 colonies Ura1, indicating the <br /> poor growth bur6D2::TRP1 strain rescued BUR61 <br /> plasmid. Continued viability dependent URA3 <br /> BUR61 plasmid, Trp1 colonies 5-fluoroorotic <br /> acid (5-<span id='am-1' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>FOA</span>) sensitive. Combined, results indicate that <br /> although BUR6 function absolutely required viability, <br /> it clearly critical normal cell growth. <br />    BUR6 deletion analysis. To determine histone <br /> fold similarity functionally significant delimit the <br /> regions BUR6 required function vivo, an <br /> extensive series nonoverlapping bur6 deletion alleles was <br /> constructed tested function vivo (Fig. 5A). BUR6 <br /> function assayed ways, first, plasmid complemen- <br /> tation slow-growth, Bur2, Gal2 phenotypes the <br /> chromosomal bur6-1 mutation second, testing phe- <br /> notypes plasmid-borne bur6 deletion alleles chro- <br /> mosomal bur6 null background. The results deletion <br /> analysis striking; different alleles 10                       FIG. 5. BUR6 deletion analysis. BUR6 deletion alleles constructed by <br /> amino acids deleted residues 51 100 com-                           oligonucleotide-directed mutagenesis deleting amino acid residues listed <br /> pletely nonfunctional BUR6 function assays;                      left. The regions deleted allele represented schematically <br /> do complement bur6-1 inviable bur6 null                           parentheses alignment wild-type BUR6 histone fold <br />                                                                                    domain. The wild-type (WT) BUR6 coding region shown top, the <br /> strain. Four alleles partial complementing activity;                    histone fold domain, extending approximately amino acids 53 118, being <br /> D41-50, residues directly amino terminal his-                      depicted hatched box. (A) CEN plasmids containing deletion alleles <br /> tone fold domain deleted, alleles that, combined,                    transformed GY561 (bur6-1) examined complementation <br /> have amino acids 101 130 deleted reduced                         bur6-1 Bur2, Gal2, slow-growth phenotypes. 111 indicates complete <br />                                                                                    complementation, 1 indicates weak complementation, 2 indicates lack of <br /> complementation bur6-1 Bur2 Gal2                              complementation. LEU2 CEN plasmids containing deletion alleles also <br /> phenotypes chromosomal bur6 null strain. In contrast,                     transformed GY568 (bur6D2::TRP1 [CEN BUR61 URA3]) streaked <br /> other deletions amino-terminal carboxy-termi-                       5-FOA plates assess plasmid phenotypes bur6D background <br /> nal tails BUR6 detectable effect BUR6 function.                      (right column). (B) Western analysis bur6 deletion alleles. CEN plasmids <br />                                                                                    containing deletion alleles shown lanes transformed into <br /> The region BUR6 absolutely essential function                       GY561 (bur6-1), extracts prepared strains probed with <br /> (amino acids 50 100) corresponds                          polyclonal anti-Bur6 antibody (a gift E. Gadbois R. Young). The position <br /> most conserved histone fold, amino acids 53                      21-kDa molecular mass marker shown right. <br />  2062       PRELICH                                                                                                                               MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 6. bur6-1 causes increased expression basal promoters. (A) Yeast strains GY218 (bur6-1) GY319 (BUR61) transformed a <br /> cyc1Duas plasmid (pLGD178). Individual transformants grown SC-Ura plus raffinose (Raff) SC-Ura plus glucose (Glu). Extracts prepared, and <br /> b-galactosidase (b-gal) activities quantitated described previously (39). Levels GY218 (bur6-1) represented filled boxes, levels GY319 <br /> (BUR61) represented hatched boxes. (B) GY218 (bur6-1) GY319 (BUR61) transformed hoDuas plasmid (M740), grown SC-Ura medium, <br /> and assayed b-galactosidase activity described panel A. (C) GY215 (bur6-1) GY315 (BUR61) transformed CYC1-lacZ plasmid (pLG669Z), <br /> grown SC-Ura plus raffinose medium, assayed b-galactosidase activity described panel A. (D) GY215 (bur6-1) GY315 (BUR61) transformed <br /> with HO-lacZ plasmid (p1701), grown SC-Ura medium, assayed b-galactosidase activity described panel A. b-Galactosidase levels means <br /> from experiments using independent transformants, standard errors (T bars) ,20%. <br />  <br />  <br />  <br /> promoters determined. Plasmids containing cyc1Duas-lacZ                         sistent Gal2 phenotype. Expression SPT15 <br /> and hoDuas-lacZ transformed BUR1 bur6-1                                gene unaffected mutation serves as <br /> strains, b-galactosidase levels quantitated. Both re-                       internal loading control experiment. The Gal2 de- <br /> porter plasmids reproducibly directed higher levels expres-                       fect likely to reduced expression Gal4 <br /> sion lacZ bur6-1 mutant strain relative                       activator, overexpression Gal4p 2mm plasmid <br /> BUR1 strain (Fig. 6A B), indicating effects                          does suppress bur6-1 mot1-301 Gal2 defect. These <br /> bur6-1 limited suc2Duas. For cyc1Duas-lacZ                            results indicate Bur6p Mot1p required, either <br /> plasmid, expression levels relatively equivalent                          directly indirectly, induction GAL1 GAL10 <br /> both repressing derepressing conditions wild-type                            mRNAs. <br /> CYC1 expression, expected absence CYC1                             To investigate cis-acting sequences responsible the <br /> UAS. Similar constitutively increased expression cyc1Duas                       Gal2 defect, series promoter fusions employed. <br /> has observed mot1-1, sin4D, gal11D strains                        BUR61 bur6-1 strains transformed plas- <br /> strains depleted histone H4 (7, 13, 21, 28). To                    mid expresses intact GAL1-lacZ promoter (pRY131) <br /> determine BUR6 effect intact CYC1                             pRY131 deletion derivatives remove most <br /> and HO promoters, plasmids express wild-type CYC1-lacZ                          (p1D7) (p1D1) Gal4p binding sites. lacZ expres- <br /> (pLG669Z) HO-lacZ (p1701) transformed                                   sion pRY131 greatly reduced bur6-1 strain, <br /> BUR61 bur6-1 strains, expression b-galactosidase                          faithfully reproducing effects observed genomic GAL1 <br /> was assayed. bur6-1 caused slight (less twofold) reduc-                       gene (Fig. 8). The p1D7 deletion removes four <br /> tion expression CYC1-lacZ caused dramatic reduc-                         Gal4p binding sites exhibits approximately fivefold lower <br /> tion expression HO-lacZ (Fig. 6C D). Combined                         expression bur6-1 strain. In contrast, deletion re- <br /> the results described below, indicates BUR6 affects                        moves Gal4p binding sites (p1D1) leaves the <br /> expression intact promoters limited                          GAL1 core promoter intact unaffected bur6-1 mutation. <br /> DUAS promoters.                                                                         To investigate Gal2 defect property the <br />    BUR6 required induction GAL1 GAL10 <br /> transcription. Among unselected phenotypes caused by <br /> bur6-1 mot1-301, Gal2 phenotype particularly <br /> interesting, suggested genes involved in <br /> galactose metabolism properly induced these <br /> mutant strains. This intriguing light previous <br /> biochemical characterization Mot1p DRAP1, it <br /> implies BUR6 MOT1 positive roles some <br /> promoters vivo negative transcriptional roles other <br /> promoters. To investigate possibility, BUR1, bur6-1, and <br /> mot1-301 strains inoculated media contained <br /> either raffinose (nonrepressing noninducing) galactose <br /> (inducing) carbon source. RNA prepared                                  FIG. 7. bur6-1 mot1-301 cause reduced levels GAL1 GAL10 <br /> these cultures time points galactose induction,                       mRNA induction. Yeast strains GY315 (BUR61), GY215 (bur6-1), GY236 <br />                                                                                      (mot1-301) grown YP plus raffinose (noninducing nonrepressing) <br /> and Northern blots used detect levels expression                      medium, harvested, resuspended YP plus galactose (inducing) liquid <br /> from GAL1 GAL10 promoters. Strong rapid in-                              medium. RNAs isolated strains 0, 1, 2, 4 h galactose <br /> duction GAL1 GAL10 observed BUR1 strains                               induction. Three micrograms total RNA separated 1% formaldehyde <br /> grown galactose medium, expected. The level induction                       agarose gel, transferred GeneScreen filter, hybridized GAL1- and <br />                                                                                      GAL10-specific probe. The filters subsequently stripped reprobed with <br /> was lower GAL1 GAL10 (Fig. 7)                               SPT15-specific probe loading control. The positions GAL1, <br /> bur6-1 mot1-301 strains time course, con-                         GAL10, SPT15 mRNAs indicated. <br />  VOL. 17, 1997                                                                   BUR6 BOTH ACTIVATES AND REPRESSES IN YEAST                                 2063 <br />  <br />  <br />  <br />  <br />    FIG. 8. A bur6 mutation reduces expression UASGAL-lacZ HO-lacZ reporter plasmids. (A) Plasmid pRY131 (GAL1-lacZ) deletion derivatives <br /> (p1D7 p1D1) transformed GY215 (bur6-1) GY315 (BUR1), expression b-galactosidase assayed 1.5 h induction galactose. The <br /> locations Gal4p binding sites UAS depicted UAS. Regions deleted p1D7 p1D1 depicted filled boxes. The figure is <br /> not drawn scale. (B) Plasmids contain UAS-less CYC1-lacZ reporter (pLG670Z), CYC1-lacZ control UASGAL1 (pLGSD5), Gal4p 17-mer <br /> binding sites (pSV14) transformed GY215 (bur6-1) GY315 (BUR1), expression b-galactosidase assayed 1.5 h galactose induction. <br /> Assays performed triplicate, units b-galactosidase expression shown right, percents error parentheses. <br />  <br />  <br /> UAS, plasmids contain intact GAL10 UAS                                The second class BUR genes, consisting BUR3 and <br /> (pLGSD5) duplicated Gal4p binding sites (pSV14) fused                        BUR6, subject report. These genes of <br /> the UAS-less CYC1p-lacZ reporter transformed                             particular proposed roles chroma- <br /> BUR61 bur6-1 strains b-galactosidase levels                           tin-independent repressors basal transcription. The identi- <br /> quantitated 1.5 h induction galactose. Both pLGSD5                        fication BUR3 MOT1 lends strong support model, <br /> and pSV14 directed high levels b-galactosidase BUR1                        MOT1 encodes ATP-dependent inhibitor TBP (3). <br /> strain galactose induction defective induc-                      The Mot1 protein specifically displaces TBP TATA <br /> tion bur6-1 strain (Fig. 8). In contrast, b-galactosidase                     box presence hydrolyzable forms ATP no <br /> expression UAS-less CYC1p-lacZ plasmid slightly                       effect site-specific DNA-binding proteins tested. <br /> increased bur6-1 strain, expected. Com-                         This biochemical characterization Mot1p consistent with <br /> bined, results indicate BUR6 affects genes                        phenotype partial loss-of-function mot1 mutations, <br /> besides SUC2, affects intact wild-type promoters,                     increase transcription DUAS promoters vivo <br /> both positive negative roles transcription vivo.                         (13). The identification BUR3 MOT1 particularly <br />                                                                                    intriguing light phenotypic similarities the <br />                              DISCUSSION                                            bur3 bur6 mutations, results suggested BUR6 <br />                                                                                    functions manner similar MOT1 vivo. <br />    The Bur selection designed identify components or <br />                                                                                       The molecular cloning BUR6 revealed region signif- <br /> regulators basal transcription machinery. The data pre- <br />                                                                                    icant amino acid similarity histone fold family pro- <br /> sented here, combined previous results, indicate that <br />                                                                                    teins. This similarity suggested Bur6p func- <br /> this approach successful. The bur mutations were <br />                                                                                    tion nucleosome-mediated repression general <br /> previously grouped phenotypic classes, im- <br />                                                                                    factors; however, certain results favor nucleosome-indepen- <br /> plication phenotypic classes mutations <br /> that suppress suc2Duas biochemically distinct mechanisms.                       dent model. First, bur6-1 mutation causes subset phe- <br /> The previously reported cloning BUR1, BUR4, BUR5                            notypes similar caused bur3-1 <br /> (47), combined results presented here, supports                      (mot1) mutation conferred histone class of <br /> hypothesis.                                                                        bur mutations. Second, similarity Bur6p his- <br />    The bur mutations appear comprise phenotypic class-                      tone H2A limited strictly histone fold domain; is <br /> es: class BUR genes includes BUR1, BUR2, BUR4,                          similarity histones outside histone fold domain, and <br /> BUR5, class consists BUR3 BUR6. Be-                         core histones highly positively charged both <br /> cause BUR5 identical HHT1, encodes histone H3,                         amino- carboxy-terminal tails, Bur6p negatively <br /> mutations class proposed suppress suc2Duas                    charged outside histone fold domain. Finally, se- <br /> by interfering nucleosome repression GTFs (47).                        quence Bur6p similar recently identified human <br /> The identification BUR4 SPT21 supports                          protein known DRAP1 (38) NC2a (19), functions <br /> classification, SPT21 role positively neg-                    repressor TBP vitro. Deletion analysis (Fig. 5) indi- <br /> atively modulating transcription large number differen-                    cates region BUR6 required vivo correlates <br /> tially regulated genes, possibly affecting levels histone                extent DRAP1/NC2a-homologous region, <br /> mRNA (14, 52). Finally, majority mutations identified                    nonessential amino-terminal carboxy-terminal re- <br /> the Bur selection genes encode histones H2A                       gions conserved distantly related organisms. Further <br /> H2B SPT genes proposed roles                      support proposal BUR6 functions DRAP1/ <br /> chromatin repression (5, 62). The specific roles BUR2,                    NC2a homolog provided yeast genome sequencing <br /> encodes novel protein (46), BUR1 (also called SGV1),                         project: yeast Dr1/NC2b homolog identified on <br /> which encodes putative protein kinase (26), remain unknown                       chromosome IV, BUR6 yeast gene similar the <br /> but currently investigation.                                             Dr1 binding partner DRAP1/NC2a. Finally, Bur6p copurifies <br />  2064     PRELICH                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br /> with yeast Dr1 column chromatography steps             The existence systems global repression transcrip- <br /> binds directly yeast Dr1 (17, 18, 31). Bur6p       tion dependent proteins Bur6p, Mot1p, <br /> displays biochemical characteristics expected DRAP1/        histones leads interesting question cell <br /> NC2a homolog.                                                        overcomes repression machinery allow transcriptional <br />    Although BUR6 initially identified selection         activation appropriate conditions. It appears the <br /> mutations affect suc2Duas promoter, effects         SNF-SWI complex plays important role overcoming nu- <br /> clearly limited SUC2 basal transcription. The           cleosomal repression, remains seen SNF- <br /> bur6-1 mutation causes increased transcription            SWI activities thought function through <br /> DUAS promoters tested (SUC2, HO, CYC1),             chromatin remodeling play role Mot1p- Bur6p- <br /> effects modest compared levels in-        dependent repression. The inability bur6 mot1 muta- <br /> duction. Similar partial increase expression observed      tions suppress snf5D suggests SNF-SWI does over- <br /> a series studies examining effects histone H4 deple-       come repression Mot1p Bur6p. Recent studies suggest <br /> tion yeast (15, 21) strains containing mutations        TFIIA play role stabilizing TBP dis- <br /> MOT1 (13), SIN4 (7, 28), SPT5 (56), SPT6 (43, 57), SPT16/            placement activity Mot1p, remains seen whether <br /> CDC68 (36), GAL11 (7). The relatively modest effects          TFIIA performs role vivo. It likely at <br /> the bur6-1 mutation fact          function activator proteins overcome <br /> single bur6 allele isolated bur6 null allele causes   effects repressors Bur6p, directly or <br /> extremely slow growth, partial loss-of-function alleles      coactivator intermediates. Detecting proposed <br /> were likely isolated. Finally, repressors,       activator-repressor interactions testing possible func- <br /> Mot1p (13), Spt proteins (62), Not proteins (11),        tional significance vivo possible organism, <br /> nucleosomes, likely repressing basal transcription   yeast, allows sophisticated genetic analysis. <br /> in bur6 mutant cells, preventing complete dere-             The combination biochemical genetic analysis pro- <br /> pression.                                                            teins, MOT1 BUR6, modulate activities of <br />    In addition increasing transcription promoters       GTFs contribute detailed understanding of <br /> have UAS deleted, bur6-1 mot1-301 muta-            intricacies transcriptional control eukaryotic cells, <br /> tions reduce activated levels transcription         including interplay proteins activate re- <br /> wild-type GAL1 GAL10 promoters. The reduced levels            press transcription. <br /> GAL1 GAL10 mRNAs simply reduced <br />                                                                                               ACKNOWLEDGMENTS <br /> activity Gal4p activator, transformation bur6-1 <br /> strain plasmid overexpresses Gal4p does revert          I thank Fred Winston, Jon Madison, Karen Arndt, Mary Ann Osley, <br /> the Gal2 phenotype (46). The activation defect likely      David Stillman plasmids strains, Fred Dietrich provid- <br /> be restricted galactose-induced genes, HO-lacZ           ing unpublished sequence BUR6 region, Ellen Gadbois, Rick <br /> reporter defective expression bur6-1 strain.       Young, Jon Madison, Fred Winston, Danny Reinberg com- <br />                                                                      municating unpublished results, Ellen Gadbois Rick Young for <br /> BUR6 required activation UAS elements, <br />                                                                      anti-BUR6 antibody, Lonny Levin, Karen Arndt, Fred Win- <br /> however, expression wild-type SPT15 CYC1 genes         ston critical readings manuscript. <br /> was unaffected bur6-1 strains. Results presented indi-          This work supported start-up funds Albert Einstein <br /> cate GAL10 activation defect inability      College Medicine National Institutes Health grant <br /> either intact GAL10 UAS duplicated Gal4p binding sites        GM52486 G.P. <br /> to function UAS, elements fused                                        REFERENCES <br /> the CYC1 promoter induces strong galactose-dependent ex-              1. Arents, G., R. W. Burlingame, B.-C. Wang, W. E. Love, E. Moudri- <br /> pression BUR1 strain bur6-1 background. The            anakis. 1991. The nucleosomal core histone octamer 3.1 &#197; resolution: a <br /> reduction GAL1 GAL10 transcript levels bur6-1              tripartite protein assembly left-handed superhelix. Proc. Natl. Acad. <br /> mot1-301 strains surprising, previously published          Sci. USA 88:10148&#8211;10152. <br />                                                                       2. Auble, D. T., S. Hahn. 1993. An ATP-dependent inhibitor TBP <br /> in vitro characterization Mot1 DRAP1/NC2a indicated               binding DNA. Genes Dev. 7:844&#8211;856. <br /> that proteins function repressors, consistent      3. Auble, D. T., K. E. Hansen, C. G. F. Mueller, W. S. Lane, J. Thorner, S. <br /> with existing genetic characterization mot1 mutations.            Hahn. 1994. Mot1, global repressor RNA polymerase II transcription, <br /> However, results presented suggest Bur6p               inhibits TBP binding DNA ATP-dependent mechanism. Genes Dev. <br />                                                                          8:1920&#8211;1934. <br /> Mot1p dual functions vivo, activators         4. Baxevanis, A. D., G. Arents, E. N. Moudrianakis, D. Landsman. 1995. A <br /> as repressors. Recent results labs consis-           variety DNA-binding multimeric proteins contain histone fold <br /> tent dual activator repressor roles Mot1p vivo           motif. Nucleic Acids Res. 23:2685&#8211;2691. <br /> (10, 35). Proteins dual activating repressing      5. Bortvin, A., F. Winston. 1996. Evidence Spt6p controls chromatin <br />                                                                          structure direct interaction histones. Science 272:1473&#8211;1476. <br /> roles uncommon. For example, mutations his-            5a.Bradford, M. M. 1976. A rapid sensitive method quantitation of <br /> tone genes, SPT21, RPD3, MCM1, SIN3 cause in-                    microgram quantities protein utilizing principle protein-dye bind- <br /> creased transcription promoters decreased tran-              ing. Anal. Biochem. 72:248&#8211;254. <br /> scription (16, 29, 30, 41, 59&#8211;61). How dual      6. Carlson, M., D. Botstein. 1982. Two differentially regulated mRNAs <br />                                                                          different 59 ends encode secreted intracellular forms yeast in- <br /> effects explained? One model Bur6p                  vertase. Cell 28:145&#8211;154. <br /> Mot1p associated different complexes,            7. Chen, S., R. W. West, Jr., S. L. Johnson, H. Gans, B. Kruger, J. Ma. <br /> one activates represses tran-            1993. TSF3, global regulatory protein silences transcription yeast <br /> scription. Such direct dual role demonstrated             GAL genes, mediates repression a2 repressor identical to <br />                                                                          SIN4. Mol. Cell. Biol. 13:831&#8211;840. <br /> the yeast Mcm1 protein number mammalian tran-                8. Chen, S., R. W. West, Jr., J. Ma, S. L. Johnson, H. Gans, G. Woldeha- <br /> scription factors. Alternatively, possible function       wariat. 1993. TSF1 TSF6, required silencing Saccharomyces cer- <br /> is direct indirect. For example, Bur6p            evisiae GAL genes, global regulatory genes. Genetics 134:701&#8211;716. <br /> have direct effects repressor phenotypically        9. Clark-Adams, C. D., D. Norris, M. A. Osley, J. S. Fassler, F. Winston. <br />                                                                          1988. Changes histone gene dosage alter transcription yeast. Genes <br /> appear activator GAL genes represses GAL-            Dev. 2:150&#8211;159. <br /> specific repressor. Future experiments required ad-       10. Collart, M. A. 1996. The NOT, SPT3, MOT1 genes functionally interact <br /> dress specific roles BUR6 repression and/or activation.        regulate transcription core promoters. Mol. Cell. Biol. 16:6668&#8211;6676. <br />  VOL. 17, 1997                                                                           BUR6 BOTH ACTIVATES AND REPRESSES IN YEAST                                    2065 <br />  <br />  <br /> 11. Collart, M. A., K. Struhl. 1993. CDC39, essential nuclear protein        40. Mitchell, P. J., R. Tjian. 1989. Transcriptional regulation mammalian <br />     negatively regulates transcription differentially affects constitutive           cells sequence-specific DNA binding proteins. Science 245:371&#8211;378. <br />     inducible HIS3 promoters. EMBO J. 12:177&#8211;186.                                    41. Natsoulis, G., C. Dollard, F. Winston, J. D. Boeke. 1991. The products <br /> 12. Cote, J., J. Quinn, J. L. Workman, C. L. Peterson. 1994. Stimulation              SPT10 SPT21 genes Saccharomyces cerevisiae increase the <br />     GAL4 derivative binding nucleosomal DNA yeast SWI/SNF com-                     amplitude transcriptional regulation large number unlinked loci. <br />     plex. Science 265:53&#8211;60.                                                                 New Biol. 3:249&#8211;259. <br /> 13. Davis, J. L., R. Kunisawa, J. Thorner. 1992. A presumptive helicase              42. Neigeborn, L., M. Carlson. 1984. Genes affecting regulation SUC2 <br />     (MOT1 gene product) affects gene expression required viability             gene expression glucose repression Saccharomyces cerevisiae. Genetics <br />     yeast Saccharomyces cerevisiae. Mol. Cell. Biol. 12:1879&#8211;1892.                       108:845&#8211;858. <br /> 14. Dollard, C., S. L. Ricupero-Hovasse, G. Natsoulis, J. D. Boeke, F.               43. Neigeborn, L., J. L. Celenza, M. Carlson. 1987. SSN20 essential <br />     Winston. 1994. SPT10 SPT21 required transcription particular              gene mutant alleles suppress defects SUC2 transcription in <br />     histone genes Saccharomyces cerevisiae. Mol. Cell. Biol. 14:5223&#8211;5228.                Saccharomyces cerevisiae. Mol. Cell. Biol. 7:672&#8211;678. <br /> 15. Durrin, L. K., R. K. Mann, M. Grunstein. 1992. Nucleosome loss                   44. Peterson, C. L., A. Dingwall, M. P. Scott. 1994. Five SWI/SNF gene <br />     activates CUP1 HIS3 promoters fully induced levels yeast                   products components large multisubunit complex required tran- <br />     Saccharomyces cerevisiae. Mol. Cell. Biol. 12:1621&#8211;1629.                                 scriptional enhancement. Proc. Natl. Acad. Sci. USA 91:2905&#8211;2908. <br /> 16. Durrin, L. K., R. K. Mann, P. S. Kayne, M. Grunstein. 1991. Yeast                45. Pina, B., U. Bruggemeier, M. Beato. 1990. Nucleosome positioning <br />     histone H4 N-terminal sequence required promoter activation vivo.              modulates accessibility regulatory proteins mouse mammary tumor <br />     Cell 65:1023&#8211;1031.                                                                       virus promoter. Cell 60:719&#8211;731. <br /> 17. Gadbois, E. L., D. M. Chao, J. C. Reese, M. R. Green, R. A. Young.               46. Prelich, G. Unpublished results. <br />     TBP-associated negative regulator NC2 global repressor class II genes        47. Prelich, G., F. Winston. 1993. Mutations suppress deletion an <br />     vivo. Proc. Natl. Acad. Sci. USA, press.                                           upstream activating sequence yeast: involvement protein kinase and <br /> 18. Goppelt, A., M. Meisterernst. 1996. Characterization basal inhib-             histone H3 repressing transcription vivo. Genetics 135:665&#8211;676. <br />     itor class II transcription NC2 Saccharomyces cerevisiae. Nucleic            48. Rine, J., I. Herskowitz. 1987. Four genes responsible position effect <br />     Acids Res. 24:4450&#8211;4455.                                                                 expression HML HMR Saccharomyces cerevisiae. Genetics <br /> 19. Goppelt, A., G. Stelzer, F. Lottspeich, M. Meisterernst. 1996. A mech-               116:9&#8211;22. <br />     anism repression class II gene transcription specific binding      49. Rose, M. D., P. Novick, J. H. Thomas, D. Botstein, G. R. Fink. 1987. A <br />     NC2 TBP-promoter complexes heteromeric histone fold domains.                      Saccharomyces cerevisiae genomic plasmid bank based centromere- <br />     EMBO J. 15:3105&#8211;3116.                                                                    containing shuttle vector. Gene 60:237&#8211;243. <br /> 20. Grunstein, M. 1990. Histone function transcription. Annu. Rev. Cell Biol.         50. Rose, M. D., F. Winston, P. Hieter. 1990. Methods yeast genetics: a <br />     6:643&#8211;678.                                                                               laboratory course manual. Cold Spring Harbor Laboratory Press, Cold <br /> 21. Han, M., M. Grunstein. 1988. Nucleosome loss activates yeast down-                   Spring Harbor, N.Y. <br />     stream promoters vivo. Cell 55:1137&#8211;1145.                                         51. Sarokin, L., M. Carlson. 1984. Upstream region required regulated <br /> 22. Hirschhorn, J., F. Winston. Personal communication.                                  expression glucose-repressible SUC2 gene Saccharomyces cerevisiae. <br /> 23. Hirschhorn, J. N., S. A. Brown, C. D. Clark, F. Winston. 1992. Evidence <br />                                                                                              Mol. Cell. Biol. 4:2750&#8211;2757. <br />     SNF2/SWI2 SNF5 activate transcription yeast altering chro- <br />                                                                                          52. Sherwood, P. W., M. A. Osley. 1991. Histone regulatory (hir) mutations <br />     matin structure. Genes Dev. 6:2288&#8211;2298. <br />                                                                                              suppress d insertion alleles Saccharomyces cerevisiae. Genetics 128:729&#8211;738. <br /> 24. Imbalzano, A. N., H. Kwon, M. R. Green, R. E. Kingston. 1994. Facil- <br />                                                                                          53. Sinha, S., I.-S. Kim, K.-Y. Sohn, B. Crombrugghe, S. Maity. 1996. <br />     itated binding TATA-binding protein nucleosomal DNA. Nature (Lon- <br />                                                                                              Three classes mutations A subunit CAAT-binding factor CBF <br />     don) 370:481&#8211;485. <br />                                                                                              delineate functional domains involved three-step assembly the <br /> 25. Inostroza, J. A., F. H. Mermelstein, I. Ha, W. S. Lane, D. Reinberg. 1992. <br />                                                                                              CBF-DNA complex. Mol. Cell. Biol. 16:328&#8211;337. <br />     Dr1, TATA-binding protein-associated phosphoprotein inhibitor of <br />                                                                                          54. Sternberg, P. W., M. J. Stern, I. Clark, I. Herskowitz. 1987. Activation <br />     class II gene transcription. Cell 70:477&#8211;489. <br />                                                                                              yeast HO gene release multiple negative controls. Cell 48: <br /> 26. Irie, K., S. Nomoto, I. Miyajima, K. Matsumoto. 1991. SGV1 encodes a <br />     CDC28/cdc2-related kinase required Ga subunit-mediated adaptive                    567&#8211;577. <br />     response pheromone S. cerevisiae. Cell 65:785&#8211;795.                             55. Struhl, K. 1995. Yeast transcriptional regulatory mechanisms. Annu. Rev. <br /> 27. Ito, H., K. Fukuda, K. Murata, A. Kimura. 1983. Transformation                    Genet. 29:651&#8211;674. <br />     intact yeast cells treated alkali cations. J. Bacteriol. 153:163&#8211;168.           56. Swanson, M. S., E. A. Malone, F. Winston. 1991. SPT5, essential gene <br /> 28. Jiang, Y. W., D. J. Stillman. 1992. Involvement SIN4 global                   important normal transcription Saccharomyces cerevisiae, encodes an <br />     transcriptional regulator chromatin structure Saccharomyces cerevi-            acidic nuclear protein carboxy-terminal repeat. Mol. Cell. Biol. 11: <br />     siae. Mol. Cell. Biol. 12:4503&#8211;4514.                                                     3009&#8211;3019. <br /> 29. Johnson, A. D. 1995. Molecular mechanisms cell-type determination              57. Swanson, M. S., F. Winston. 1992. SPT4, SPT5 SPT6 interactions: <br />     budding yeast. Curr. Opin. Genet. Dev. 5:552&#8211;558.                                        effects transcription viability Saccharomyces cerevisiae. Genetics <br /> 30. Johnson, L. M., G. Fisher-Adams, M. Grunstein. 1992. Identification               132:325&#8211;336. <br />     non-basic domain histone H4 N-terminus required repression           58. Taylor, I. C., J. L. Workman, T. J. Schuetz, R. E. Kingston. 1991. <br />     yeast silent mating loci. EMBO J. 11:2201&#8211;2209.                                      Facilitated binding GAL4 heat shock factor nucleosomal tem- <br /> 31. Kim, S., J. G. Na, M. Hampsey, D. Reinberg. 1997. The Dr1/DRAP1                      plates: differential function DNA-binding domains. Genes Dev. 5:1285&#8211; <br />     heterodimer global repressor transcription vivo. Proc. Natl. Acad.            1298. <br />     Sci. USA 94:820&#8211;825.                                                                 59. Vidal, M., R. F. Gaber. 1991. RPD3 encodes second factor required to <br /> 32. Kim, T. K., Y. Zhao, G. Hui, R. Bernstein, R. G. Roeder. 1995. TATA-                 achieve maximum positive negative transcriptional states Saccharo- <br />     binding protein residues implicated functional interplay nega-              myces cerevisiae. Mol. Cell. Biol. 11:6306&#8211;6316. <br />     tive cofactor 2 (Dr1) general factors TFIIA TFIIB. J. Biol. Chem.            60. Vidal, M., R. Strich, R. E. Esposito, R. F. Gaber. 1991. RPD1 (SIN3/ <br />     270:10976&#8211;10981.                                                                         UME4) required maximal activation repression diverse yeast <br /> 33. Knezetic, J. A., D. S. Luse. 1986. The presence nucleosomes                  genes. Mol. Cell. Biol. 11:6306&#8211;6316. <br />     DNA template prevents initiation RNA polymerase II vitro. Cell 45:             61. Wang, H., D. J. Stillman. 1993. Transcriptional repression Saccharo- <br />     95&#8211;104.                                                                                  myces cerevisiae SIN3-LexA fusion protein. Mol. Cell. Biol. 13:1805&#8211; <br /> 34. Kwon, H., A. N. Imbalzano, P. A. Khavari, R. E. Kingston, M. R. Green.               1814. <br />     1994. Nucleosome disruption enhancement activator binding                62. Winston, F. 1992. Analysis SPT genes: genetic approach analysis <br />     human SW1/SNF complex. Nature (London) 370:477&#8211;481.                                      TFIID, histones transcription factors yeast. In S. L. McKnight <br /> 35. Madison, J., F. Winston. 1997. Evidence Spt3 functionally interacts             K. R. Yamamoto (ed.), Transcriptional regulation. Cold Spring Harbor <br />     Mot1, TFIIA, TATA-binding protein confer promoter-specific                   Laboratory Press, Cold Spring Harbor, N.Y. <br />     transcriptional control Saccharomyces cerevisiae. Mol. Cell. Biol. 17:287&#8211;295.    63. Winston, F., C. Dollard, S. L. Ricupero-Hovasse. 1995. Construction of <br /> 36. Malone, E. A., C. D. Clark, A. Chiang, F. Winston. 1991. Mutations                set convenient Saccharomyces cerevisiae strains isogenic to <br />     SPT16/CDC68 suppress cis- trans-acting mutations affect promoter                S288C. Yeast 11:53&#8211;55. <br />     function Saccharomyces cerevisiae. Mol. Cell. Biol. 11:5710&#8211;5717.                 64. Workman, J. L., R. G. Roeder. 1987. Binding transcription factor <br /> 37. Meisterernst, M., R. G. Roeder. 1991. Family proteins interact               TFIID major late promoter vitro nucleosome assembly <br />     TFIID regulate promoter activity. Cell 67:557&#8211;567.                              potentiates subsequent initiation RNA polymerase <br /> </body></html>