<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Extracytoplasmic Stress Factor, sE, Is Required to <br /> Maintain Cell Envelope Integrity <span id='am-43' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-44' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-46'  ></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span>Escherichia coli</span> <br /> Jennifer D. Hayden, Sarah E. Ades* <br /> Department Biochemistry Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania, United States America <br />  <br />  <br />  <br />      Abstract <br />      Extracytoplasmic function ECF sigma factors abundant <span id='am-17' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-18' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> alternative sigma factors bacteria. <br />      Members rpoE <span id='am-1' about='obo:NCBITaxon_subclass' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-2' property="rdfs:label" content="subclass" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>subclass</span> ECF sigma factors implicated sensing stress cell envelope Gram-negative <br />      bacteria required virulence pathogens. The best-studied member <span id='am-9' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> rpoE <span id='am-59' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-63'  ></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span>Escherichia</span> <br />      coli, encoding sE protein. sE studied role combating extracytoplasmic stress, members <br />      regulon largely defined. sE required viability E. coli, studies date explain sE <br />      essential seemingly unstressed cells. In work investigate essential role sE E. coli analyzing the <br />      phenotypes associated loss sE activity isolating suppressors allow cells live absence sE. We <br />      demonstrate sE inhibited, cell envelope stress increases envelope integrity lost. Many cells lyse and <br />      develop blebs containing cytoplasmic material sides. To better understand connection between <br />      transcription sE cell envelope integrity, identified multicopy suppressors essentiality sE, ptsN and <br />      yhbW. yhbW gene unknown function, ptsN member sE regulon. Overexpression ptsN lowers the <br />      basal level multiple envelope stress responses, cytoplasmic stress response. Our results consistent <br />      model overexpression ptsN reduces stress cell envelope, promoting survival absence <br />      sE. <br />  <br />   Citation: Hayden JD, Ades SE (2008) The Extracytoplasmic Stress Factor, sE, Is Required Maintain Cell Envelope Integrity Escherichia coli. PLoS ONE 3(2): <br />   e1573. doi:10.1371/journal.pone.0001573 <br />   Editor: Steve Sandler, University Massachusetts, United States America <br />   Received December 18, 2007; Accepted January 13, 2008; Published February 6, 2008 <br />   Copyright: &#223; 2008 Hayden, Ades. <span id='am-25' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-26'  ></span><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported grant MCB-0347302 National Science Foundation. The funding agency did play role study in <br />   preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   *E-mail: ades@psu.edu <br />  <br />  <br />  <br /> Introduction                                                                         sE-dependent response E. coli, rpoE gene, encodes <br />                                                                                      sE, essential viability [7]. Despite wealth information <br />    All organisms stress responses allow sense                  role sE response cell envelope stress the <br /> respond damaging conditions altering gene expression. An                       identification sE regulon, essential role sE still <br /> additional level complexity introduced inducing                       unclear [8,9]. One suppressor DrpoE lethality, deletion the <br /> signal sensed membrane information                      ydcQ gene, identified [10]. However, function this <br /> must communicated membrane response be <br />                                                                                      gene understood clear suppresses the <br /> generated. In Gram-negative bacteria intercompartmental <br />                                                                                      essentiality rpoE. <br /> signaling required maintain cell envelope, consists <br />                                                                                         sE activated stresses interfere folding outer <br /> of inner outer membranes, periplasmic space, and <br />                                                                                      membrane proteins (OMPs) heat shock, overexpression of <br /> peptidoglycan layer [1]. <br />                                                                                      OMP genes, mutations genes encoding chaperones <br />    The cell envelope complex, dynamic compartment is <br />                                                                                      required OMP folding [11&#8211;14]. In unstressed cells, sE activity <br /> crucial survival cell. It static structure can <br /> be remodeled response environmental conditions. The                            low sE sequestered inner membrane the <br /> chemical environment cell envelope distinct                   antisigma factor RseA (Fig. 1) [15,16]. RseA single-pass inner <br /> the cytoplasm. The envelope lacks ATP, oxidizing,                      membrane protein binds sE prevents sE from <br /> subject fluctuations ionic strength passive diffusion                interacting RNA polymerase [15&#8211;17]. During envelope stress, <br /> small molecules outer membrane porins [2]. As such,                          RseA degraded response unfolded porins sequential <br /> Gram-negative bacteria possess stress responses uniquely                    action inner membrane proteases, DegS RseP, followed <br /> targeted cell envelope. These stress responses include                    cytoplasmic protease ClpXP (Fig. 1) [18&#8211;22]. A periplasmic <br /> CpxAR (Cpx), BaeRS (Bae), Rcs phosphorelays, response                        protein, RseB, binds periplasmic domain RseA and <br /> governed alternative sigma factor sE, phage shock                     enhances inhibition RseA, protecting proteolysis (Fig. 1) <br /> (PSP) response [3&#8211;6]. Each responses activated following                 [15,16,18,23]. Continual degradation RseA required to <br /> perturbation particular components envelope.                               provide cell sufficient free sE support viability, and <br />    Although stress responses important reacting                           deletion degS rseP toxic stabilization of <br /> damaging conditions, stress proteins play important                        RseA consequent sequestration sE [19,20,23]. sE also <br /> roles basic cellular physiology. This particularly true                activated independently RseA-dependent stress-signaling <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                1                               February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                       sE Cell Envelope Integrity <br />  <br />  <br />  <br />  <br /> Figure 1. The sE-dependent extracytoplasmic stress response. sE held membrane antisigma factor RseA. RseB binds the <br /> periplasmic domain RseA protects RseA proteolysis. Unfolded OMPs activate protease DegS, cleaves periplasmic domain of <br /> RseA. The partially degraded RseA substrate RseP. RseP cleaves RseA further, releasing cytoplasmic domain RseA bound sE. This <br /> remaining domain RseA degraded ClpXP freeing sE interact RNA polymerase direct transcription regulon. The major <br /> classes known genes sE regulon indicated. The alarmone ppGpp protein DksA activate sE-dependent transcription sE is <br /> released RseA, shown clarity. (IM inner membrane, OM outer membrane). <br /> doi:10.1371/journal.pone.0001573.g001 <br />  <br /> pathway cytoplasmic alarmone ppGpp, levels change               transcribe genes. Alternatively, loss of <br /> in response nutrient availability [24].                                   expression multiple non-essential genes dependent sE for <br />    As expected role stress response, sE regulon          expression lethal. However, genes sE <br /> includes genes encoding periplasmic foldases, proteases,                 regulon transcribed s70. Chromatin immoprecipitation <br /> chaperones aid OMP folding. In addition, sE transcribes              followed genomic microarray (ChIP-chip) experiments exam- <br /> an array biosynthetic enzymes involved phospho-               ining location s70 E. coli chromosome suggest that <br /> lipid, fatty acid, LPS, membrane-derived oligosaccharide                 approximately 40% sE-regulated promoters overlap s70 <br /> synthesis transport, number cell envelope                 promoters [32]. This number likely underestimation of <br /> proteins including lipoproteins, inner membrane proteins,                extent dual regulation, does include genes that <br /> envelope proteins unknown function [9,25,26]. The sE regulon              s70 promoters farther away sE promoter [32]. <br /> also includes genes encode major components                  As such, sE essential genes regulon are <br /> complexes responsible ushering LPS OMPs                   transcribed s70 misregulated lethal effect the <br /> periplasm properly assembling outer membrane                 absence sE. <br /> [27,28]. Recently sE shown transcribe small                 In work took approaches better understand the <br /> RNAs decrease expression OMPs, providing                       essential role sE. The approach characterize the <br /> mechanism decrease flux OMPs envelope                phenotypes cells sE activity inhibited gain insights <br /> times stress [29&#8211;31]. In addition envelope-associated              cellular functions impacted. We demonstrate that <br /> proteins, sE-dependent promoters upstream genes                 sE required maintain integrity cell envelope; the <br /> involved cytoplasmic processes transcription, transla-            absence sE cells developed blebs sides lysed. <br /> tion, DNA replication, DNA/RNA modification [9,25,26].                   The second approach isolate multicopy suppressors that <br />    The sE regulon contains number essential genes,              allow cells live absence sE activity. Two multicopy <br /> simplest explanation sE essential required       suppressors, ptsN yhbW, isolated suppress the <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        2                           February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                           sE Cell Envelope Integrity <br />  <br />  <br /> lethality inhibition sE activity essentiality the <br /> rpoE gene. ptsN member sE regulon, yhbW not, <br /> and gene previously associated cell envelope <br /> functions. Overexpression ptsN lowered basal activity of <br /> several envelope-sensing pathways, implying suppress <br /> DrpoE lethality reducing stress cell envelope. <br />  <br /> Results <br /> Inhibition sE activity toxic increases envelope <br /> stress <br />     To gain better understanding role particular gene <br /> in cellular physiology, informative examine the <br /> phenotypes deletion mutant lacking gene interest. <br /> Because sE encoded essential gene, possible to <br /> examine phenotype DrpoE strain. Therefore examined <br /> the effects loss sE activity inducing overexpression of <br /> its inhibitors, rseA rseB, encoded IPTG-inducible trc <br /> promoter pTrc99a plasmid, pRseAB. This method <br /> efficiently inhibits sE (Fig. 2A) preventing association with <br /> RNA polymerase [15&#8211;17]. Overexpression rseA rseB the <br /> consequent sequestration sE lethal bacterium. In <br /> liquid culture, cfu/ml began decrease 2 hours (,2.5 <br /> generations) following addition IPTG optical <br /> density stopped increasing began decrease slightly within <br /> 3 hours (,3.5 generation) (Fig. 2B [15]). On solid media, the <br /> plating efficiency presence IPTG reduced three <br /> orders magnitude compared non-inducing conditions <br /> (Table 1). To ensure observed phenotypes to <br /> inhibition sE overproduction RseA RseB, we <br /> introduced point mutation rseA gene plasmid <br /> encoding rseA rseB, changing aspartate residue position <br /> 11 RseA histidine. This mutation abrogates antisigma <br /> factor activity RseA (Fig. 2A [16]). When overproduced                Figure 2. Inhibition sE activity caused cell death. Strains <br />                                                                              SEA007 (squares, pRseAB), SEA008 (circles, pTrc99a vector control), and <br /> along RseB, RseAD11H protein reached similar steady-                SEA4044 (triangles, pRseAD11H) grown LB 30uC shaking <br /> state levels wild-type protein, properly localized         gyrotary waterbath. IPTG added OD600 ,0.1 induce <br /> inner membrane, did inhibit sE activity, did induce lysis,           overexpression rseA rseB, indicated arrow. (A) sE was <br /> and did reduce plating efficiency (Fig. 2, Table 1, <span id='am-100' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-101' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-103'  ></span><span id='am-104' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span>         inhibited overexpression rseA rseB, rseAD11H and <br /> not shown).                                                                  rseB, shown b-galactosidase activity measured sE- <br />     Because sE known respond envelope stress direct             dependent rpoHP3-lacZ reporter. Miller units (open symbols, right <span id='am-35' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-36' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39'  ></span>axis</span>) <br />                                                                              OD600 (closed symbols, left axis) plotted time point. (B) <br /> the synthesis components cell envelope, likely          Overexpression rseA rseB, rseAD11H rseB, caused a <br /> that compartment negatively impacted envelope               reduction colony forming units optical density. OD600 (closed <br /> stress increase sE activity blocked. We measured                symbols, left axis) cfu/ml (open symbols, right axis) measured <br /> activation reporter genes Cpx, Bae, Rcs envelope              growth curve. A representative experiment shown in <br /> stress responses (cpxP-lacZ activated CpxR, spy-lacZ activated         panels. <br />                                                                              doi:10.1371/journal.pone.0001573.g002 <br /> BaeR CpxR, rprA-lacZ activated RcsB, Fig. 3). b- <br /> galactosidase activity envelope stress reporters increased <br /> by 2.5 hours following inhibition sE (Fig. 4). Since activation        altered. Samples taken cultures 2.5 hours following <br /> these stress responses general increase stress          overexpression rseA rseB. This time point chosen for <br /> associated cell death, monitored activation              analysis cell envelope stress responses activated, <br /> cytoplasmic stress response mediated s32 using s32-                   cfu/ml started decrease, morphological defects <br /> regulated htpG-lacZ reporter fusion. s32 activity increased relatively <br /> little (,2-fold) following inhibition sE compared larger          Table 1. Plating efficiencies following inhibition sE <br /> increases seen envelope stress responses (3.5-fold Cpx, <br /> 8-fold Rcs, 12-fold Bae/Cpx-dependent reporters). <br /> These results suggest loss sE activity does increase stress        strain               plasmid                      plating efficiencya <br /> the cell envelope, does increase cytoplasmic stress            SEA008               pTrc99a                      1.0 6 0.3 <br /> same extent.                                                                  SEA007               pRseAB                       4.9 6 2.7 6 1023 <br />                                                                                                            D11H <br />                                                                               SEA4044              pRseA      B                 0.9 6 0.1 <br /> Loss sE activity does cause major changes the <br />                                                                               SEA4256              pRseAB+pPtsN                 0.9 6 0.2 <br /> composition membranes <br />                                                                               SEA4271              pRseAB+pYhbW                 0.6 6 0.3 <br />    Since inhibition sE increases envelope stress sE can <br /> transcribe genes localized inner outer                    a <br />                                                                                plating efficiency: cfu/ml IPTG divided cfu/ml IPTG <br /> membranes, asked envelope membranes                       doi:10.1371/journal.pone.0001573.t001 <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        3                             February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                             sE Cell Envelope Integrity <br />  <br />  <br />  <br />  <br /> Figure 3. Diagram stress regulators respective <br /> regulon members used reporters work. All reporter <br /> constructs present lacZ fusions integrated single-copy the <br /> chromosome. The rpoHP3-lacZ fusion regulated sE. The degP-lacZ <br /> fusion regulated sE CpxR. The cpxP-lacZ fusion regulated by <br /> CpxR. The spy-lacZ fusion regulated CpxR BaeR. The rprA-lacZ <br /> fusion regulated RcsB. The htpG-lacZ fusion regulated s32. sE, <br /> Cpx, Bae, Rcs monitor envelope stress, s32 monitors <br /> cytoplasmic stress. <br /> doi:10.1371/journal.pone.0001573.g003 <br />  <br />  <br /> described evident. Cell lysates fractionated by <br /> centrifugation discontinuous sucrose density gradients and <br /> several assays used detect fractions containing inner and <br /> outer membrane components. In assays results <br /> from cultures sE activity inhibited similar to <br /> those control cultures. No alterations overall protein <br /> content fractions detected Coomassie-stained SDS <br /> polyacrylamide gels (Fig. 5A). The inner outer membrane <br /> fractions separated, determined western blots <br /> probed antibodies inner membrane protein, FtsH, and <br />  <br />                                                                                  Figure 5. Fractionation strain SEA007 lysates and <br />                                                                                  inhibition sE sucrose density gradients. At <br />                                                                                  2.5 hours following inhibition sE, cells harvested lysates <br />                                                                                  separated discontinuous sucrose density gradient centrifugation. <br />                                                                                  Lysates comparable numbers cells (as determined OD600) <br />                                                                                  loaded gradients. (A) Coomassie-stained 12% SDS- <br />                                                                                  polyacrylamide gels gradient fractions control culture <br />                                                                                  (top) culture sE activity inhibited (bottom). M denotes <br />                                                                                  molecular weight markers. (B) b-NADH oxidase activity (closed symbols) <br />                                                                                  specific gravity (open symbols) fractions (A), inhibition of <br />                                                                                  sE (circles) control (squares). (C) Western blots using polyclonal <br />                                                                                  antibodies detect FtsH (inner membrane) FepA (outer <br />                                                                                  membrane) fractions (A), control (top panels) inhibition <br />                                                                                  sE (bottom panels). L denotes whole-cell lysates fractionation. <br />                                                                                  doi:10.1371/journal.pone.0001573.g005 <br />  <br /> Figure 4. Envelope stress responses activated following <br /> inhibition sE. b-galactosidase activity measured htpG- <br /> lacZ (s32, SEA4183), cpxP-lacZ (Cpx, SEA4177), rprA-lacZ (Rcs, SEA4190),         outer membrane protein, FepA (Fig. 5C). Fractions containing <br /> and spy-lacZ (Cpx Bae, SEA4187) reporters. The fold change Miller <br /> units (M.U.) rseA rseB overexpression (M.U. with <br />                                                                                  inner membrane identified using b-NADH oxidase <br /> IPTG/M.U. ITPG) shown samples taken 0 (white), 1 (grey), 2          activity assay (Fig. 5B). Thus, loss sE activity does result in <br /> (blue), 2.5 (red), 3 (black) hours addition IPTG.                   gross defects steady-state protein composition density of <br /> doi:10.1371/journal.pone.0001573.g004                                            inner outer membranes. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                            4                            February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                         sE Cell Envelope Integrity <br />  <br />  <br />  <br />  <br /> Figure 6. Cell envelope defects resulting inhibition sE. Cultures SEA007 SEA008 grown Fig. 2. Samples taken <br /> approximately 2.5 hrs. induction rseA rseB overexpression. (A&#8211;D) Images live cells shown using differential interference microscopy <br /> (DIC, column 1) fluorescence microscopy following addition FM4-64 visualize membranes (red, column 2), DAPI visualize DNA (blue, <br /> column 3), expression YFP visualize cytoplasm (green, column 4). The fluorescent micrographs overlaid column 5. (A) <br /> Images SEA008 control strain sE inhibited shown. (B&#8211;D) Images SEA007 following sE inhibition, reveal blebs that <br /> contain YFP stain DAPI (B C) contain YFP stain DAPI (D). In (D), arrow marks bleb lacking DAPI staining <br /> and arrowhead marks lysed cell retained DAPI staining, lost YFP. Scale bars 2 mm. Over 1,000 cells examined fluorescence <br /> microscopy typical micrographs represented here. Scanning <span id='am-89' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-90' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-91'  ></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>electron</span> micrographs (E) transmission electron micrographs (F) SEA007 <br /> following sE inhibition. Scale bars 1 mm. No blebs seen cells control cultures. <br /> doi:10.1371/journal.pone.0001573.g006 <br />  <br />  <br /> Morphological changes following inhibition sE                              cells culture blebs (Fig. 6) number ghosted <br />    We examined morphology cells following inhibition              cells increased cellular debris, indicative <br /> of sE. By 2 hours addition IPTG overexpress rseA              lysis. In typical experiment, approximately 100 cells were <br /> rseB (the time cfu/ml began drop), phenotypes            viewed time point. The location blebs timing <br /> evident phase contrast microscopy. Ghosted cells cells            appearance respect loss viability very <br /> blebs began appear culture. The blebs formed primarily              reproducible, suggesting phenotypes related. By 3&#8211; <br /> along lateral wall cells frequently         4 hours addition IPTG cells additional blebs, the <br /> either poles septum. Usually bleb formed              number ghosted cells cell debris increased, <br /> cell. By 2.5 hours addition IPTG, approximately 20%               cells clumped together. Similar phenotypes were <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         5                            February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                            sE Cell Envelope Integrity <br />  <br />  <br /> observed sE inhibited proteolytic stabilization                 Table 2. Cotransduction nadB::Tn10 rpoE::kan <br /> RseA depletion DegS RseP proteases (data not <br /> shown) following overexpression rseAD11H <br /> and rseB pRseAD11HB, providing evidence                  recipient               plasmida                nadB::Tn10b     rpoE::kan   c <br />  <br /> phenotypes inhibition sE overexpression of <br /> rseA rseB. In addition, phenotypes seen                 SEA001 (wild type)      _                       159             1 <br /> the cells grown glucose glycerol minimal media                      CAG43113 (sup+)         _                       184             184 <br /> supplemented amino acids indicating phenotypes                   SEA4270                 pYhbW                   48              48 <br /> were function growth rate medium composition.                      SEA4254                 pPtsN                   153             153 <br />    To probe morphology cells following inhibition            SEA4106                 pPtsNH73A               24              0 <br /> sE, fluorescent probes used detect cytoplasm, nucleic <br />                                                                                SEA4110                 pPtsNH73A/H120A         24              0 <br /> acids, membranes live cells. We visualized cytoplasm by <br /> expressing soluble fluorescent protein, YFP,                       SEA4230 (DtrkA)         _                       11              0 <br /> expression construct integrated chromosome. Membranes                   SEA4234 (DtrkA)         pPtsN                   23              23 <br /> and nucleic acids visualized respectively FM4-64                 SEA4144 (DptsP)         _                       20              0 <br /> and DAPI fluorescent dyes, added directly                    SEA4156 (DptsP)         pPtsN                   12              12 <br /> cultures. Neither dyes cytoplasmic expression YFP <br />                                                                                SEA4143 (DptsO)         _                       12              0 <br /> altered phenotypes associated inhibition sE. All the <br /> cells blebs outlined FM4-64, cells              SEA4155 (DptsO)         pPtsN                   15              15 <br /> the blebs contained YFP FM4-64 boundary,                   SEA4131 (DptsN)         _                       24              0 <br /> exception ghosted cells (Fig. 6A&#8211;D). The ghosted cells             a <br />                                                                                  &#8216;&#8216;_&#8217;&#8217; denotes plasmid <br /> outlined FM4-64, longer contained YFP (Fig. 6D,                    b <br />                                                                                  number colonies isolated P1 transduction selecting nadB::Tn10 <br /> indicated arrowhead). YFP proteolytically labile,                c <br />                                                                                 number nadB::Tn10 transductants shown rpoE::kan <br /> therefore lack YFP signal ghosted cells likely        doi:10.1371/journal.pone.0001573.t002 <br /> due release cytoplasmic contents lysis. All cells <br /> had DAPI staining, compaction nucleoids and <br /> the number nucleoids cell did differ significantly            encoded ASKA plasmid. Of approximately 10,000 <br /> control cells, suggesting loss sE activity does cause             transformants, 44 putative suppressors recovered. <br /> major defects chromosome replication segregation (Fig. 6A&#8211;                 Two classes genes isolated selection, those <br /> D). Approximately 50% cells blebs contained DAPI                  encoding genes restore sE activity longer <br /> staining bleb, addition YFP (Fig. 6B,C). We                  require sE growth. We distinguished classes <br /> hypothesize blebs contain DAPI staining (Fig. 6D,            using chromosomally encoded sE-dependent lacZ reporter <br /> indicated arrow), nucleoid simply               gene (rpoHP3-lacZ [36]) strain SEA007 detected sE activity <br /> not drawn blebs, nucleic acids              plating presence Xgal IPTG. Of 44 clones <br /> present visualized DAPI.                                           isolated selection, 26 formed blue colonies, indicating that <br />    Electron microscopy employed determine                overexpression gene plasmid restored sE activity, and <br /> any additional changes cell morphology detectable                 18 formed white colonies, indicating sE activity not <br /> light microscopy. Scanning electron micrographs revealed                   restored longer required growth. This <br /> significant alterations cells blebs ghosted cells         class contained <span id='am-105' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-106' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span> potential suppressors of <br /> (Fig. 6E). The surface cells blebs ghosted cells         requirement sE viability. For 18 plasmids, <br /> similar cells control cultures overexpres-           verified suppression linked plasmid, plating <br /> sion rseA rseB. Using transmission electron microscopy                 efficiencies rseA rseB overexpressed longer <br /> (TEM) obtained images containing sections                reduced, morphological defects significantly reduced. <br /> bleb. In images, inner outer membranes                    The selection identified plasmids permit growth the <br /> clearly traced bleb, cytoplasmic material                  presence overexpression rseA rseB. However, these <br /> present bleb (Fig. 6F). No evidence outer membrane                  strains low levels sE activity contribute to <br /> vesiculation, associated alterations sE                viability. We determined rpoE gene be <br /> activity [33,34], gross morphological defects                   deleted potential suppressor genes overexpressed <br /> observed TEM SEM images.                                            using cotransduction assay. A nadB::Tn10 allele immediately <br />                                                                               upstream rpoE::kan deletion introduced P1 transduction <br /> Isolation multicopy suppressors requirement                      wild-type strains containing 18 plasmids, and <br /> sE activity viability                                                     nadB::Tn10 transductants selected. Because genes are <br />    The phenotypic studies demonstrate sE activity                tightly linked, rpoE::kan allele cotransduced unless rpoE is <br /> depleted, cell envelope integrity compromised, resulting lysis          essential (Table 2). The rpoE deletion cotransduced strains <br /> and bleb formation. However, phenotypes readily                 carrying 2 18 plasmids (Table 2). For remainder the <br /> explained loss expression gene(s) sE regulon (see         strains, rpoE deletion cotransduced, indicating the <br /> Discussion). To gain additional information sE                   rpoE gene required viability. The multicopy <br /> essential, turned genetics isolated multicopy suppressors           suppressors rpoE essentiality yhbW ptsN (Tables 1 and <br /> that allowed E. coli live absence sE activity. Cells             2). yhbW encodes putative luciferase-like monooxygenase of <br /> containing pRseAB (strain SEA007) transformed pools                 unknown function. ptsN encodes EIIANtr, protein related the <br /> of plasmids ASKA ORF library, E. coli                  enzyme IIA components PTS carbohydrate transport systems <br /> ORF cloned control IPTG-inducible                         [37]. However, EIIANtr known associated any <br /> promoter [35]. Transformants plated presence                   known transporters E. coli Ntr phosphoryl relay is <br /> IPTG induce overexpression rseA rseB gene                   instead thought play role signaling [37&#8211;39]. ptsN a <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         6                                February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                        sE Cell Envelope Integrity <br />  <br />  <br /> member rpoE regulon [9] focused further <br /> studies. <br />  <br /> Phosphorylation EIIANtr required suppression <br />     Since EIIANtr phosphoryl relay the <br /> phosphorylation state EIIANtr important activity <br /> [40,41], determined phosphorylation EIIANtr was <br /> important suppression. EIIANtr phosphorylated His73 and <br /> possibly His120 [41]. We constructed variants, ptsNH73A and <br /> ptsNH73A/H120A, tested ability suppress <br /> the requirement sE viability cotransduction assay. <br /> The rpoE::kan allele cotransduced nadB::Tn10 <br /> (Table 2) suggesting phosphorylation EIIANtr required for <br /> survival absence sE. Overexpression EIIANtr <br /> variants detected Coomassie-stained protein gels                   Figure 7. Overexpression ptsN reduces cell envelope stress. <br /> indicating lack suppression owed instability             b-galactosidase activity rpoHP3-lacZ (sE, SEA4254), degP-lacZ <br />                                                                                (sE Cpx, SEA4181), cpxP-lacZ (Cpx, SEA4179), rprA-lacZ (Rcs, <br /> the variant proteins (data shown). <br />                                                                                SEA4199), spy-lacZ (Cpx Bae, SEA4189), htpG-lacZ (s32, <br />     The ptsP ptsO genes encode EINtr NPr, respectively,            SEA4185) fusions measured overnight cultures, without <br /> form phosphoryl relay transfers phosphate                        overexpression ptsN. Activity lacZ fusion normalized to <br /> phosphoenol pyruvate (PEP) EINtr NPr EIIANtr [37,39].                 cultures ptsN overexpression set 100%. Average <br /> Since phosphorylation EIIANtr required suppression,              values standard deviations minimum 3 experiments are <br /> asked components phosphoryl relay                shown. <br />                                                                                doi:10.1371/journal.pone.0001573.g007 <br /> also required determining overexpression ptsN could <br /> still suppress essentiality rpoE strains lacking ptsP ptsO. <br /> These genes required suppression overexpressed                 cytoplasmic stress (Fig. 3). Overexpression ptsN significantly <br /> ptsN; nadB::Tn10 rpoE::kan cotransductants obtained               lowered expression lacZ fusions regulated by <br /> strains (Table 2). Additionally, rpoE gene deleted            envelope stress response factors, effect s32- <br /> in DptsN, DptsO, DptsP strains overexpression ptsN.              dependent cytoplasmic stress reporter (Fig. 7). The result <br /> Since results point mutations ptsN indicate               indicates overexpression ptsN does lower b-galactosi- <br /> phosphorylation EIIANtr required suppression, likely           dase activity se, does dampening effect overall <br /> that paralogous proteins phosphoryl relays                    gene expression. <br /> compensate loss Ntr components phosphorylate                    ptsN overexpression reduce basal levels envelope <br /> EIIANtr absence.                                                      stress activate stress signaling pathways trigger a <br />     PtsN shown bind inhibit potassium uptake             cytoplasmic signal downregulates effectors signaling <br /> the potassium transporter TrkA [40]. K+ major cation             pathways independently cell envelope stress. If <span id='am-12' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-13' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15'  ></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>idea</span> is <br /> E. coli cytoplasm participates variety processes related           correct, ptsN lower reporter activity independently of <br /> to known functions sE including adaptation osmotic stress                envelope signal transduction pathways. In strain lacking the <br /> and maintenance turgor pressure [42&#8211;44]. In addition, K+ binds              antisigma factor RseA, sE activity high unresponsive to <br /> and regulates number intracellular enzymes including RNA                  known regulatory signals originate cell envelope, is <br /> polymerase (as potassium glutamate) [45]. Therefore, overex-                responsive levels cytoplasmic signaling factor, ppGpp <br /> pression ptsN suppresses requirement sE viability             [24]. Overexpression ptsN did lower activity rpoHP3- <br /> inhibiting TrkA, rpoE essential DtrkA strain.               lacZ reporter DrseA strain, suggesting signal transduction <br /> However, DtrkA strain used recipient                  RseA required ptsN lower sE activity (Fig. 8A). <br /> cotransduction assay, nadB::Tn10 transductants ob-                       If ptsN overexpression does fact lower basal level of <br /> tained indicating DtrkA suppressor (Table 2).                    envelope stress, decrease reporter gene expression <br /> Furthermore, trkA gene required suppression                 dependent transcriptional activators associated <br /> ptsN, cotransductants readily obtained DtrkA strain               envelope stress pathways. Interestingly, DydcQ <br /> when ptsN overexpressed (Table 2).                                         strain decrease reporter gene expression Cpx- <br />                                                                                regulated cpxR-lacZ reporter dependent response <br /> Overexpression ptsN lowers envelope stress                                  regulator CpxR [10]. We measured effects overexpression of <br />    The basal levels envelope stress responses lower          ptsN Cpx-dependent cpxP-lacZ Rcs-dependent rprA-lacZ <br /> a DydcQ strain, characterized suppressor DrpoE lethality             reporter fusions strains lacking respective transcriptional <br /> [10]. In addition, sE-dependent rpoHP3-lacZ activity low               regulators, CpxR RcsB. In WT strain, overproduction of <br /> strain containing unmapped DrpoE suppressor [46]. Consistent                ptsN lowered expression rprA-lacZ reporter 3.2-fold <br /> with results, overexpression ptsN lowered activity             compared 1.8-fold DrcsB strain (Fig. 8B) lowered <br /> sE-dependent rpoHP3-lacZ reporter wild-type rpoE+ strain                  expression cpxP-lacZ reporter 4.3-fold compared 2.2- <br /> (Fig. 7 8A). To determine effect specific              fold cpxR::Vspec strain (Fig. 8C). It reported <br /> sE, ptsN overexpression affected envelope stress                   previously cpxP-lacZ reporter expressed DcpxR <br /> responses, measured effects ptsN overexpression                   variant E. coli strain MC4100 [10,47]. However, were <br /> reporters following envelope stress responses (Fig. 3): Cpx            able reproducibly measure b-galactosidase activity this <br /> (cpxP-lacZ reporter), sE Cpx (degP-lacZ reporter), Bae Cpx             reporter cpxR::Vspec variant strain MG1655, although <br /> (spy-lacZ reporter), Rcs (rprA-lacZ reporter). A strain carrying         expression quite low (,250-fold lower WT <br /> reporter s32-dependent cytoplasmic heat shock response                 background). These results indicate decreased activity the <br /> (htpG-lacZ) included assay ptsN lowered                    reporters ptsN overexpression partially dependent <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          7                           February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                         sE Cell Envelope Integrity <br />  <br />  <br />  <br />                                                                              r <br />  <br />                                                                              fusion WT DrseA strains (SEA4254 SEA4228), (B) rprA-lacZ <br />                                                                              fusion WT DrcsB strains (SEA4199 SEA4203), (C) cpxP- <br />                                                                              lacZ fusion WT cpxR::Vspec strains (SEA 4179 SEA4251). The <br />                                                                              inset C shows cpxR::Vspec strain ptsN <br />                                                                              overexpression. The basal level expression .250-fold lower in <br />                                                                              cpxR::Vspec cells WT cells easily seen main plot. <br />                                                                              doi:10.1371/journal.pone.0001573.g008 <br />  <br />                                                                              cpxR rcsB, suggesting EIIANtr lowers envelope stress <br />                                                                              sensed pathways. <br />  <br />                                                                              Discussion <br />                                                                                 Despite extensive knowledge regulatory pathways that <br />                                                                              control sE activity, stresses activate pathways, and <br />                                                                              genes sE transcribes, essential role sE cell has <br />                                                                              remained elusive. In work demonstrate sE required <br />                                                                              maintenance cell envelope integrity. When sE activity was <br />                                                                              inhibited, envelope stress sensed stress responses <br />                                                                              increased, indicating cell envelope impacted. Many <br />                                                                              cells lysed developed blebs containing cytoplasmic <br />                                                                              material sides. Interestingly, phenotypes not <br />                                                                              accompanied major alterations composition the <br />                                                                              envelope membranes. We identified multicopy suppressors, <br />                                                                              ptsN yhbW, allowed cells live absence rpoE <br />                                                                              gene. ptsN member sE regulon [9] lowers basal <br />                                                                              level activity envelope stress responses unknown <br />                                                                              mechanism overexpressed. <br />  <br />                                                                              How does loss sE activity lead bleb formation and <br />                                                                              lysis? <br />                                                                                 The blebs form following inhibition sE resemble bulges <br />                                                                              formed E. coli treated penicillin-like antibiotics that <br />                                                                              inhibit peptidoglycan synthesis, inactive variant the <br />                                                                              peptidoglycan synthase PBP 1B overproduced [48,49]. These <br />                                                                              treatments lytic, lysis thought triggered holes <br />                                                                              formed peptidoglycan layer murein hydrolases acting in <br />                                                                              absence peptidoglycan synthesis [48,49]. The similarity the <br />                                                                              phenotypes suggests inhibition sE leads weakening of <br />                                                                              and/or defects peptidoglycan layer. These defects potentially <br />                                                                              major outcomes. Either hole forms peptidoglycan <br />                                                                              large accommodate extrusion inner <br />                                                                              membrane cytoplasm, evidenced YFP signal the <br />                                                                              blebs, cell lyses, evidenced loss YFP signal ghosted <br />                                                                              cells. The observation blebs form 20% cells is <br />                                                                              likely to inherent stochastic nature process. <br />                                                                              Many cells lyse bleb form. Further support the <br />                                                                              model inhibition sE leads defects peptidoglycan <br />                                                                              layer comes preliminary results strain lacking four <br />                                                                              enzymes involved peptidoglycan synthesis, PBPs 4, 5, 6, 7, <br />                                                                              results irregular peptidoglycan synthesis [50]. This strain <br />                                                                              develops blebs lyses quickly following inhibition sE <br />                                                                              isogenic WT strain (data shown). If peptidoglycan is <br />                                                                              affected, able reduce cell death growing cells <br />                                                                              high osmolarity medium. Unfortunately, experiment not <br />                                                                              technically feasible. High osmolarity induced sE response such <br /> Figure 8. ptsN requires known regulators reduce stress                    sE longer inhibited RseA, RseA was <br /> sensed sE, Rcs, Cpx envelope stress pathways. In                  overexpressed (data shown). <br /> each panel b-galactosidase activity measured           Even phenotypes consistent model <br /> overexpression ptsN WT strain strain lacking known <br /> regulators respective stress responses. Measurements are <br />                                                                              loss sE activity leads alterations peptidoglycan <br /> expressed percentage reporter gene activity WT strain       layer resulting bleb formation lysis, eliminate the <br /> without ptsN overexpression value noted columns.       possibility bleb formation caused different event, such <br /> Relative reporter gene activities shown (A) rpoHP3-lacZ          aberrant cell division. Future experiments examining the <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        8                            February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                        sE Cell Envelope Integrity <br />  <br />  <br /> integrity peptidoglycan layer                  suppression ptsN, DtrkA suppressor, indicating that <br /> localization cell division proteins following inhibition sE            EIIANtr additional functions, discovered, are <br /> should help distinguish possibilities.                         responsible suppression DrpoE lethality. As discussed below, <br />                                                                              results indicate novel activity likely involves regulation <br /> Relationship phenotypes known                            cell envelope stress responses. <br /> functions sE regulon members <br />    Because reported function sE direct                     Lowered envelope stress responses, characteristic sE <br /> transcription, altered regulation genes             suppressors <br /> the sE regulon ultimately lead bleb formation loss             The basal level activity multiple envelope stress responses is <br /> viability sE activity blocked. However, given known              reduced presence sE suppressors data is <br /> functions sE regulon members, point specific             available, overexpression ptsN, DydcQ [10] uncharacter- <br /> gene set genes regulon obviously associated        ized suppressor [46], suggesting lower activity envelope <br /> these phenotypes. Depletion phenotypes characterized               stress responses important suppression <br /> in essential sE-regulated genes (yaeT, yfiO, imp, lptA       mechanism. Preliminary results indicate basal levels of <br /> and lptB), encode components complexes assemble                cell envelope stress responses lower following overexpres- <br /> LPS OMPs outer membrane [28,51&#8211;54]. Depletion                   sion yhbW, multicopy suppressor isolated. In <br /> these genes causes striking defects cell envelope,            light results, model suppression the <br /> those observed following inhibition sE [28,51&#8211;55].                        suppressors strengthen cell envelope loss sE <br />    The blebbing phenotype suggests peptidoglycan cell                activity resultant impact cell envelope longer <br /> division affected loss sE. Although sE does transcribe          lethal. If model correct, suppressors lower <br /> any known murein hydrolases biosynthetic enzymes directly                 envelope stress sensed signal transduction pathways that <br /> involved peptidoglycan synthesis, does transcribe bacA,          monitor cell envelope. We effects ptsN <br /> encodes undecaprenyl pyrophosphate phosphatase [9,56].                    overexpression reporter gene expression Rcs Cpx <br /> BacA hydrolyzes phosphate moiety undecaprenyl pyro-                   pathways partially dependent respective <br /> phosphate generate undecaprenyl phosphate, lipid               stress regulators. In addition, sE activity altered ptsN <br /> carrier required synthesis peptidoglycan cell wall          overexpression strain lacking rseA, known envelope <br /> polymers [56]. However, E. coli possesses additional          sensor pathway. Although possible EIIANtr was <br /> undecaprenyl pyrophosphate phosphatase enzymes cells                     unable overcome high level activation sE DrseA <br /> lacking bacA obvious morphological phenotypes                  strain background, think unlikely based observations with <br /> [56]. A sE-dependent promoter upstream cell             regulator sE, does act RseA. sE <br /> division protein FtsZ [9], promoter multiple              activity activated reduced DrseA strain in <br /> promoters contribute complex regulation ftsZ                  response cytoplasmic regulator ppGpp [24], suggesting that <br /> expression [57], making unlikely loss sE activity           effects ptsN overexpression discernible. In <br /> have large effect non-stressed growth conditions.                    contrast, reduction activity Cpx reporter gene <br />    Since direct connection convincingly                   examined DydcQ strain dependent cpxR [10]. <br /> between sE regulon members observed phenotypes,                   However, epistasis experiments reported one <br /> think likely phenotypes caused indirect effects           reporter, known activity cell <br /> resulting loss altered regulation sE-dependent genes.          envelope stress responses reduced DydcQ background <br /> For example, sE transcribes genes required synthesis              independently respective signal transduction pathways <br /> lipid A component LPS, lpxA, lpxB, lpxD [9]. LpxA catalyzes           [10]. <br /> the step lipid A synthesis uses substrate, UDP- <br /> N-acetyl-D-glucosamine, used dedicated step            Why sE essential E. coli? <br /> peptidoglycan biosynthesis [58]. Changes lpxA lpxD                       The functions core members sE regulon conserved <br /> expression absence sE alter flux substrates           number bacterial <span id='am-40' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-41' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> [9]. However, sE essential <br /> through pathways, affecting peptidoglycan synthesis            bacteria including closely related species as <br /> integrity peptidoglycan layer.                                        <span id='am-66' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-67' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-76'  ></span><span id='am-77' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Salmonella</span> typhimurium [59]. This observation led, part, a <br />                                                                              model proposing sE essential required to <br /> Suppressors DrpoE lethality                                               transcribe particular gene set genes [10]. Instead, sE is <br />    In work genes identified overexpression               essential bacteria overreact absence and, through <br /> suppresses loss viability following inhibition sE         unknown mechanism requires ydcQ gene product, <br /> the requirement rpoE gene viability. Of known           activate unknown cell death pathway kills cells [10]. <br /> suppressors DrpoE lethality (DydcQ [10] overexpression             Our data consistent model. However, suggest that <br /> ptsN yhbW), gene known roles cell            functions core regulon members conserved <br /> ptsN. ptsN member sE regulon, suggests             bacterial species sE essential [9], significant <br /> plays role cell envelope processes [9]. Several functions          differences exist properties cell envelopes of <br /> been associated nitrogen PTS system, ptsN             different species strains species may <br /> member, including balancing nitrogen carbon metabolism,                  determine bacteria survive absence sE. <br /> suppression temperature-sensitive allele essential               For example, majority work sE, including that <br /> GTPase Era DptsN mutant, inhibition potassium                presented paper, performed K12 strains E. <br /> transporter TrkA [37,38,40]. This role ptsN held          coli, lack O-antigen LPS [60]. This variation could <br /> promise providing insights mechanism suppression             significantly alter properties bacterial cell envelope, <br /> because K+ important osmotic regulation regulate              making bacteria sensitive disruptions cell envelope <br /> RNA polymerase, activities associated sE [42,44,45].                integrity sE essential. Therefore, propose that <br /> However, demonstrated trkA required                       sE essential required transcribe genes in <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        9                           February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                       sE Cell Envelope Integrity <br />  <br />  <br /> its regulon. In absence sE genes properly                2% osmium tetroxide, 0.1% uranyl acetate acetone 290uC <br /> regulated, leading disruption cell envelope integrity lysis.        3 days. Samples slowly warmed 260uC 14 hours, <br /> Future experiments delving greater depth phenotypes            230uC 14 hours, 0uC 1 hour. Samples were <br /> cells lacking sE, roles sE regulon members unknown                  transferred acetone, brought room temperature, then <br /> function, mechanism suppression help                     infiltrated resin. Ultrathin sections obtained using a <br /> distinguish models provide new insights            Reichart-Jung Ultracut E microtome diamond knife. <br /> role important cell envelope stress response.                         Sections mounted hexagonal mesh open slot copper <br />                                                                               grids stained 2% uranyl acetate 50% ethanol for <br /> Materials Methods                                                         5 minutes followed Reynold&#8217;s lead stain 2 minutes and <br />                                                                               washed water. After drying, grids visualized a <br /> Media, strains, plasmids <br />                                                                               JEOL JEM 1200 EXII transmission electron microscope. Images <br />    Strains derivatives MG1655, unless noted, and <br />                                                                               taken digital TEITZ <span id='am-30' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-31' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34'  ></span>camera</span>. <br /> are listed Table 3. Cultures grown <span id='am-4' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="Luria" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-8'  ></span>Luria</span> Bertani (LB) <br /> broth 30uC shaking gyrotary water bath. Antibiotics <br /> were used following concentrations: ampicillin (amp)                   Fluorescence microscopy <br /> 100 mg/ml, kanamycin (kan) 30 mg/ml, tetracyline (tet) 10 mg/                     For fluorescence microscopy, location cytoplasm was <br /> ml, spectinomycin (spec) 50 mg/ml, chloramphenicol (cam)                  visualized strains SEA4287 SEA4288, express soluble <br /> 20 mg/ml. Altered alleles stress response reporters              YFP cytoplasm, periplasm. Overnight <br /> transferred appropriate recipient strains transduction            cultures grown LB/Amp subcultured media containing <br /> with P1vir according standard methods [61], exception             1 ng/ml anhydrous tetracycline, induces expression YFP. <br /> that sodium citrate used toxic cells             IPTG added cells reached OD600 0.1&#8211;0.2 and <br />                                                                               samples taken microscopy 1.5 3.5 hours later. <br /> with compromised membranes. The rseAD11H mutation was <br />                                                                               FM4-64 (Molecular Probes T-3166) 4,6-diaminidino-2-pheny- <br /> made pRseAB plasmid ptsN H73A H73A/ <br />                                                                               lindole, DAPI, added directly samples respective final <br /> H12A mutations pPtsN plasmid using QuikChange site- <br />                                                                               concentrations 5 mg/ml 1 mg/mL. Cells mounted on <br /> directed mutagenesis kit (Stratagene) verified sequencing. <br />                                                                               freshly prepared polylysine coated coverslips visualized using an <br /> The ptsN::kan, ptsO::kan, ptsP::kan, trkA::kan alleles were <br />                                                                               Olympus BX-61 epi-fluorescent microscope 1006 phase oil <br /> obtained Keio collection [62] moved P1 <br />                                                                               objective (NA 1.3), using Olympus 41028 filter cube YFP, <br /> transduction SEA001. The kan allele removed with <br />                                                                               Olympus 31000v2 filter cube DAPI, Olympus 41001 <br /> FLP recombinase method Datsenko Wanner [63] to <br />                                                                               filter cube FM4-64. Images captured Hamamatsu <br /> create strains SEA4131, SEA4143, SEA4144, SEA4230. <br />                                                                               Orca-ER camera using Slidebook 4.1 software (Compix, Inc.). <br />                                                                               Deconvolution performed using AutoQuant software <br /> b-galactosidase assays                                                        package (Media Cybernetics). Generally, 10 iterations <br />   b-galactosidase assays performed largely described [24].            necessary deconvolve images. <br /> Overnight cultures diluted OD600 0.02 grown with <br /> aeration 30uC. rseA rseB overexpression induced             Sucrose density gradient fractionation <br /> addition IPTG 1 mM cultures reached OD600                       Cultures SEA007 grown OD600 0.1 which <br /> 0.1&#8211;0.2 experiments shown Figs. 2 4. In Figs. 7 8, b-          point IPTG added 1 mM culture induce rseA and <br /> galactosidase activity determined cultures grown 12&#8211;             rseB overexpression. After 2.5 hours growth, cells were <br /> 14 hours 1mM IPTG. Experiments                          harvested centrifugation, resuspended 10 mM HEPES <br /> exponential phase cultures yielded similar results. A minimum              pH 7.8, lysed French press. Discontinuous sucrose density <br /> three independent experiments performed strain                  gradients prepared layering 860 ml 50%, 45%, 40%, <br /> and condition.                                                                35%, 30% sucrose solutions 200 ml 55% sucrose <br />                                                                               cushion. Lysates equal numbers cells treated and <br /> Electron microscopy                                                           control cultures loaded gradients, based the <br />    Cells SEA007 SEA008 used electron                         OD600 cultures time sampling. After centrifugation <br /> microscopy grown LB OD600 0.1&#8211;0.2,                  50,000 rpm 15 hours, 150 ml fractions collected from <br /> point IPTG added induce rseA rseB overexpression                gradients. The specific gravity fraction was <br /> SEA007. Cells harvested 2.5 hours later centrifugation.               determined measuring refractive index. b-NADH oxidase <br /> Samples scanning electron microscopy (SEM) subjected                 activity fraction determined previously reported <br /> to primary fixation step 6.25% glutaraldehyde Sorenson&#8217;s              [64]. Total protein content analyzed 12% SDS- <br /> Buffer 1.5 hours room temperature washed 0.1 M                 polyacrylamide gels Coomassie blue staining. Western <br /> cacodylate buffer. A secondary fixation step performed                 blotting performed polyclonal antibodies outer <br /> treating samples 2% osmium tetroxide 0.1 M                        membrane protein FepA (gift Kathleen Postle) inner <br /> cacodylate buffer, followed washing dehydration              membrane protein FtsH (gift Christophe Herman) detected <br /> in ethanol. Samples dried, sputter coated imaged                horseradish peroxidase-conjugated donkey anti-rabbit IgG <br /> JEOL JSM 5400 scanning electron microscope PGT Prism                   (GE Healthcare) using ECL Plus kit (GE Healthcare). <br /> light element detector. <br />    Samples transmission electron microscopy (TEM)                    Genetic selections <br /> prepared cryo-fixation freeze substitution. The samples                   A genetic selection carried identify genes whose <br /> were washed 0.1 M cacodylate buffer incubated 18%                   overexpression allowed cells live sE activity inhibited. <br /> glycerol (v/v) 0.1 M cacodylate 20 minutes room                     Strain SEA007 (carrying pRseAB) transformed electropo- <br /> temperature cryo-protection. The sample pelleted                   ration 5 pools plasmids ASKA plasmid library <br /> centrifugation 10 ml pellet blotted filter paper          [35], cells plated LB agar 1 mM IPTG, Amp <br /> and frozen liquid nitrogen. Freeze substitution carried            (to select pRseAB), Cam (to select library plasmids). <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        10                          February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                   sE Cell Envelope Integrity <br />  <br />  <br />  <br /> Table 3. Strains Plasmids <br />  <br />  <br /> Strain                                  Genotype                                                 Reference <br />  <br /> CAG43113                                MC1061 sup+, CAG41001                            [20] <br /> CAG45146                                MG1655 DlacX74 Wl[htpG P1-lacZ]                          [65] <br /> DH300                                   MG1655 D(argF-lac)U169; rprA142-lacZ                     [66] <br /> SEA001                                  MG1655 DlacX74 Wl[rpoHP3-lacZ]                           [67] <br /> SEA007                                  SEA001 pRseAB                                            [24] <br /> SEA008                                  SEA001 pTrc99a                                           work <br /> SEA2000                                 SEA001 nadB::Tn10 DrseA                                  [24] <br /> SEA4041                                 CAG43113 rpoE::kan                                       work <br />                                                          D11H <br /> SEA4044                                 SEA001 pRseA        B                                    work <br /> SEA4106                                 SEA001 pPtsNH73A                                         work <br /> SEA4110                                 SEA001 pPtsNH73A/H120A                                   work <br /> SEA4114                                 SEA4041 nadB::Tn10                                       work <br /> SEA4131                                 SEA001 DptsN                                             work <br /> SEA4143                                 SEA001 DptsO                                             work <br /> SEA4144                                 SEA001 DptsP                                             work <br /> SEA4155                                 SEA4143 pPtsN                                            work <br /> SEA4156                                 SEA4144 pPtsN                                            work <br /> SEA4177                                 MG1655 DlacX74 lRS88[cpxP-lacZ] pRseAB                   work, reporter [47] <br /> SEA4179                                 MG1655 DlacX74 lRS88[cpxP-lacZ] pPtsN                    work, reporter [47] <br /> SEA4181                                 MG1655 DlacX74 lRS88[degP-lacZ] pPtsN                    work, reporter [68] <br /> SEA4183                                 CAG45146 pRseAB                                          work <br /> SEA4185                                 CAG45146 pPtsN                                           work <br /> SEA4187                                 MG1655 DlacX74 lRS88[spy-lacZ] pRseAB                    work, reporter [69] <br /> SEA4189                                 MG1655 DlacX74 lRS88[spy-lacZ] pPtsN                     work, reporter [69] <br /> SEA4190                                 DH300 pRseAB                                             work <br /> SEA4199                                 DH300 pPtsN                                              work <br /> SEA4203                                 4199 rcsB::kan                                           work, rcsB::kan [66] <br /> SEA4204                                 SEA4181 nadB::Tn10 DrseA pPtsN                           work <br /> SEA4228                                 SEA2000 pPtsN                                            work <br /> SEA4230                                 SEA001 DtrkA                                             work <br /> SEA4234                                 SEA4230 pPtsN                                            work <br /> SEA4251                                 SEA4179 cpxR::Vspec                                      work, cpxR::Vspec [68] <br /> SEA4254                                 SEA001 pPtsN                                             work <br /> SEA4256                                 SEA007 pPtsN                                             work <br /> SEA4270                                 SEA001 pYhbW                                             work <br /> SEA4271                                 SEA007 pYhbW                                             work <br /> SEA4287                                 SEA007 D(xylAxylB)::tetR tetA-yfp                        work, yfp allele Goulian <br />                                                                                                  lab collection <br /> SEA4288                                 SEA008 D(xylAxylB)::tetR tetA-yfp                        work, yfp allele Goulian <br />                                                                                                  lab collection <br /> Plasmids <br /> pTrc99a                                 vector, pBR322 ori AmpR                                  Pharmacia <br /> pRseAB                                  rseA rseB pTrc99a, pLC253                 [15] <br /> pRseAD11HB                              RseAD11H pRseAB                                       work <br /> pPtsN                                   ptsN pCA24N, ASKA plasmid , CamR                      work [35] <br /> pPtsNH73A                               PtsNH73A pPtsN                                        work <br /> pPtsNH73A/H120A                         PtsNH73A/H120A pPtsN                                  work <br /> pYhbW                                   yhbW pCA24N, ASKA plasmid , Camr                      work [35] <br />  <br /> doi:10.1371/journal.pone.0001573.t003 <br />  <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                               11                February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                                                    sE Cell Envelope Integrity <br />  <br />  <br /> The ASKA plasmid library (without GFP tag) used                                    restoring sE activity transformed SEA001. The <br /> studies consists plasmids containing E. coli ORF&#8217;s cloned                              resulting strains transduced P1 lysate SEA4114. <br /> the control IPTG-inducible promoter [35]. Colonies                                   Colonies plated LB/Tet/IPTG select for <br /> grew IPTG streaked LB/IPTG/Amp/Cam                                          nadB::Tn10 overexpress putative suppressor gene on <br /> confirm growth. Plasmids isolated, library plasmids                               plasmid. Transductants streaked LB/Kan/IPTG <br /> purified transformation DH5a selection LB/Cam.                                  determine rpoE deleted. PCR used verify the <br /> The selected ASKA plasmids transformed                                      presence nadB::Tn10 rpoE::kan alleles absence of <br /> SEA007 verify conferred growth IPTG. Cells                                 rpoE gene. Deletion rpoE gene verified by <br /> also grown plates containing Xgal determine sE                                    western blotting polyclonal antibodies specific sE. In each <br /> activity restored (blue colonies) inhibited (white                             cotransduction experiment, SEA001 CAG43113 used as <br /> colonies). The selected genes encoded ASKA plasmids                                 negative positive controls, respectively. <br /> identified sequencing. <br />                                                                                                 Acknowledgments <br /> Determination essentiality rpoE <br />    The nadB::Tn10 rpoE::kan linked marker construct used                                     We thank Susan Gottesman, Tracy Raivio, Carol Gross, Mark Goulian, <br />                                                                                                 Hirotada Mori sharing strains; Mary Laubacher making strains; <br /> cotransduction assays strain contains an <br />                                                                                                 Kathleen Postle, Marta Manning, Charles Bulathsinghala for <br /> <span id='am-110' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-111' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-114' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-122'  ></span><span id='am-123' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span>unidentified</span> suppressor DrpoE (sup, CAG43113). The rpoE gene                                 assistance sucrose gradients FepA antibody. All fluorescence <br /> was deleted first, using method Datsenko Wanner [63]                                 microscopy Cytometry Facility, University Park (Huck <br /> to generate strain SEA4041. The nadB::Tn10 insertion                                       Institutes Life Sciences, Penn State University) facility is <br /> SEA2000 moved rpoE::kan strain P1                                          funded, part, grant Pennsylvania Department Health <br /> transduction, selecting resistance Kan Tet generate                               using Tobacco Settlement Funds. The Department specifically disclaims <br /> strain SEA4114. Transductants verified PCR ensure                                    responsibility analyses, interpretations, conclusions. All electron <br /> that kan allele present rpoE gene absent. The                               microscopy Electron Microscopy Facility Huck <br />                                                                                                 Institutes Life Sciences, Penn State University. <br /> essentiality rpoE determined assessing cotransduction <br /> frequency nadB::Tn10 rpoE::kan recipient strains. The <br /> markers tightly linked, nadB::Tn10 transduced                                  Author Contributions <br /> independently strains rpoE essential. Plasmids                              Conceived designed experiments: SA JH. Performed the <br /> that suppressed lethality overexpression rseA rseB                                experiments: JH. Analyzed data: SA JH. Wrote paper: SA JH. <br />  <br /> References <br />  1. Kellenberger E (1996) Structure Function Subcellular Level. In:                        factor activity RseA, RseB RseC proteins. Mol Microbiol 24: <br />     Neidhardt FC, Curtiss R III, Ingraham JL, Lin ECC, Low KB, Magasanik B,                           355&#8211;371. <br />     Reznikoff WS, Riley M, Schaechter M, Umbarger HE, eds. Escherichia coli                 17.   Campbell EA, Tupy JL, Gruber TM, Wang S, Sharp MM, et al. (2003) Crystal <br />     Salmonella: Cellular Molecular Biology. 2nd ed. Washington, DC: ASM                           structure Escherichia coli sE cytoplasmic domain anti-sigma <br />     Press. pp 17&#8211;28.                                                                                  RseA. Mol Cell 11: 1067&#8211;1078. <br />  2. Nikaido H (1994) Porins specific diffusion channels bacterial outer                  18.   Ades SE, Connolly LE, Alba BM, Gross CA (1999) The Escherichia coli sE- <br />     membranes. J Biol Chem 269: 3905&#8211;3908.                                                            dependent extracytoplasmic stress response controlled regulated <br />  3. Majdalani N, Gottesman S (2005) The Rcs phosphorelay: complex signal                            proteolysis anti-sigma factor. Genes Dev 13: 2449&#8211;2461. <br />     transduction system. Annu Rev Microbiol 59: 379&#8211;405.                                        19.   Alba BM, Leeds JA, Onufryk C, Lu CZ, Gross CA (2002) DegS YaeL <br />  4. Raivio TL (2005) Envelope stress responses Gram-negative bacterial                            participate sequentially cleavage RseA activate sE-dependent <br />     pathogenesis. Mol Microbiol 56: 1119&#8211;1128.                                                        extracytoplasmic stress response. Genes Dev 16: 2156&#8211;2168. <br />  5. Rowley G, Spector M, Kormanec J, Roberts M (2006) Pushing envelope:                     20.   Alba BM, Zhong HJ, Pelayo JC, Gross CA (2001) degS (hhoB) essential <br />     extracytoplasmic stress responses bacterial pathogens. Nat Rev Microbiol 4:                    Escherichia coli gene indispensable function provide sE activity. Mol <br />     383&#8211;394.                                                                                          Microbiol 40: 1323&#8211;1333. <br />  6. Ruiz N, Silhavy TJ (2005) Sensing external stress: watchdogs Escherichia coli        21.   Flynn JM, Levchenko I, Sauer RT, Baker TA (2004) Modulating substrate <br />     cell envelope. Curr Opin Microbiol 8: 122&#8211;126.                                                    choice: SspB adaptor delivers regulator extracytoplasmic-stress <br />  7. De Las Penas A, Connolly L, Gross CA (1997) SigmaE essential sigma                          response AAA+ protease ClpXP degradation. Genes Dev 18: <br />     factor Escherichia coli. J Bacteriol 179: 6862&#8211;6864.                                           2292&#8211;2301. <br />                                                                                                 22.   Kanehara K, Ito K, Akiyama Y (2002) YaeL (EcfE) activates sE pathway of <br />  8. Alba BM, Gross CA (2004) Regulation Escherichia coli sigma-dependent <br />                                                                                                       stress response site-2 cleavage anti-sE, RseA. Genes Dev 16: <br />     envelope stress response. Mol Microbiol 52: 613&#8211;619. <br />                                                                                                       2147&#8211;2155. <br />  9. Rhodius VA, Suh WC, Nonaka G, West J, Gross CA (2006) Conserved and <br />                                                                                                 23.   Grigorova IL, Chaba R, Zhong HJ, Alba BM, Rhodius V, et al. (2004) Fine- <br />     variable functions sE stress response related genomes. PLoS Biol 4: e2. <br />                                                                                                       tuning Escherichia coli sE envelope stress response relies multiple <br /> 10. Button JE, Silhavy TJ, Ruiz N (2007) A suppressor cell death caused                     mechanisms inhibit signal-independent proteolysis transmembrane <br />     loss sE downregulates extracytoplasmic stress responses outer membrane                     anti-sigma factor, RseA. Genes Dev 18: 2686&#8211;2697. <br />     vesicle production Escherichia coli. J Bacteriol 189: 1523&#8211;1530.                         24.   Costanzo A, Ades SE (2006) Growth phase-dependent regulation the <br /> 11. Mecsas J, Welch R, Erickson JW, Gross CA (1995) Identification                                extracytoplasmic stress factor, sE, guanosine 39,59-bispyrophosphate <br />     characterization outer membrane protein, OmpX, Escherichia coli                     (ppGpp). J Bacteriol 188: 4627&#8211;4634. <br />     homologous family outer membrane proteins including Ail Yersinia              25.   Dartigalongue C, Missiakas D, Raina S (2001) Characterization Escherichia <br />     enterocolitica. J Bacteriol 177: 799&#8211;804.                                                         coli sE regulon. J Biol Chem 276: 20866&#8211;20875. <br /> 12. Missiakas D, Betton JM, Raina S (1996) New components protein folding                 26.   Rezuchova B, Miticka H, Homerova D, Roberts M, Kormanec J (2003) New <br />     extracytoplasmic compartments Escherichia coli SurA, FkpA Skp/OmpH.                        members Escherichia coli sE regulon identified two-plasmid system. <br />     Mol Microbiol 21: 871&#8211;884.                                                                        FEMS Microbiol Lett 225: 1&#8211;7. <br /> 13. Rouviere PE, De Las Penas A, Mecsas J, Lu CZ, Rudd KE, et al. (1995) rpoE,                  27.   Ruiz N, Kahne D, Silhavy TJ (2006) Advances understanding bacterial outer- <br />     gene encoding second heat-shock sigma factor, sE, Escherichia coli.                    membrane biogenesis. Nat Rev Microbiol 4: 57&#8211;66. <br />     Embo J 14: 1032&#8211;1042.                                                                       28.   Sperandeo P, Cescutti R, <span id='am-79' about='obo:NCBITaxon_117093' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_117077 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_50704 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_50674 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_50671'><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Villa" datatype="xsd:string"></span><span id='am-82'  ></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Villa</span> R, Di Benedetto C, Candia D, et al. (2007) <br /> 14. Rouviere PE, Gross CA (1996) SurA, periplasmic protein peptidyl-prolyl                     Characterization lptA lptB, essential genes implicated lipopolysac- <br />     isomerase activity, participates assembly outer membrane porins. Genes                  charide transport outer membrane Escherichia coli. J Bacteriol 189: <br />     Dev 10: 3170&#8211;3182.                                                                                244&#8211;253. <br /> 15. De Las Penas A, Connolly L, Gross CA (1997) The sE-mediated response                     29.   Johansen J, Rasmussen <span id='am-94' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-95' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-96'  ></span><span id='am-97' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>AA</span>, Overgaard M, Valentin-Hansen P (2006) <br />     extracytoplasmic stress Escherichia coli transduced RseA RseB,                   Conserved small non-coding RNAs belong sE regulon: role in <br />     negative regulators sE. Mol Microbiol 24: 373&#8211;385.                                             down-regulation outer membrane proteins. J Mol Biol 364: 1&#8211;8. <br /> 16. Missiakas D, Mayer MP, Lemaire M, Georgopoulos C, Raina S (1997)                            30.   Thompson KM, Rhodius VA, Gottesman S (2007) sE regulates regulated <br />     Modulation Escherichia coli sE (RpoE) heat-shock transcription-                            small RNA Escherichia coli. J Bacteriol 189: 4243&#8211;4256. <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                         12                                 February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                                                      sE Cell Envelope Integrity <br />  <br />  <br /> 31. Udekwu KI, Wagner EG (2007) sE controls biogenesis antisense RNA                           viability, characteristics, implications peptidoglycan synthesis. J Bacteriol <br />     MicA. Nucleic Acids Res 35: 1279&#8211;1288.                                                            181: 3981&#8211;3993. <br /> 32. Wade JT, <span id='am-84' about='obo:NCBITaxon_1510180' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_1489923 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489922 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_30828 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-85' property="rdfs:label" content="Roa" datatype="xsd:string"></span><span id='am-86'  ></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Roa</span> DC, Grainger DC, Hurd D, Busby SJ, et al. (2006) Extensive                     51.   Braun M, Silhavy TJ (2002) Imp/OstA required cell envelope biogenesis in <br />     functional overlap sigma factors Escherichia coli. Nat Struct Mol Biol                 Escherichia coli. Mol Microbiol 45: 1289&#8211;1302. <br />     13: 806&#8211;814.                                                                                52.   Charlson ES, Werner JN, Misra R (2006) Differential effects yfgL mutation on <br /> 33. McBroom AJ, Johnson AP, Vemulapalli S, Kuehn MJ (2006) Outer membrane                             Escherichia coli outer membrane proteins lipopolysaccharide. J Bacteriol 188: <br />     vesicle production Escherichia coli independent membrane instability.                    7186&#8211;7194. <br />     J Bacteriol 188: 5385&#8211;5392.                                                                 53.   Doerrler WT, Raetz CR (2005) Loss outer membrane proteins without <br /> 34. McBroom AJ, Kuehn MJ (2007) Release outer membrane vesicles Gram-                           inhibition lipid export Escherichia coli YaeT mutant. J Biol Chem 280: <br />     negative bacteria novel envelope stress response. Mol Microbiol 63:                          27679&#8211;27687. <br />     545&#8211;558.                                                                                    54.   Malinverni JC, Werner J, Kim S, Sklar JG, Kahne D, et al. (2006) YfiO <br /> 35. Kitagawa M, <span id='am-20' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-21'  ></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Ara" datatype="xsd:string"></span>Ara</span> T, Arifuzzaman M, Ioka-Nakamichi T, Inamoto E,                                    stabilizes YaeT complex essential outer membrane protein <br />     Toyonaga H, Mori H (2005) Complete set ORF clones Escherichia coli                          assembly Escherichia coli. Mol Microbiol 61: 151&#8211;164. <br />     ASKA library (a complete set E. coli K-12 ORF archive): unique resources             55.   Wu T, McCandlish AC, Gronenberg LS, Chng SS, Silhavy TJ, et al. (2006) <br />     biological research. DNA Res 12: 291&#8211;299.                                                         Identification protein complex assembles lipopolysaccharide the <br /> 36. Mecsas J, Rouviere PE, Erickson JW, Donohue TJ, Gross CA (1993) The                               outer membrane Escherichia coli. Proc Natl Acad Sci USA 103: 11754&#8211;11759. <br />     activity sE, Escherichia coli heat-inducible sigma-factor, modulated            56.   El Ghachi M, Derbise A, Bouhss A, Mengin-Lecreulx D (2005) Identification of <br />     expression outer membrane proteins. Genes Dev 7: 2618&#8211;2628.                                    multiple genes encoding membrane proteins undecaprenyl pyrophosphate <br /> 37. Powell BS, Court DL, Inada T, Nakamura Y, Michotey V, et al. (1995) Novel                         phosphatase (UppP) activity Escherichia coli. J Biol Chem 280: 18689&#8211;18695. <br />     proteins phosphotransferase encoded rpoN operon                 57.   Flardh K, Garrido T, Vicente M (1997) Contribution individual promoters in <br />     Escherichia coli. Enzyme IIANtr affects growth organic nitrogen                        ddlB-ftsZ region transcription essential cell-division gene ftsZ in <br />     conditional lethality erats mutant. J Biol Chem 270: 4822&#8211;4839.                             Escherichia coli. Mol Microbiol 24: 927&#8211;936. <br /> 38. Commichau FM, Forchhammer K, Stulke J (2006) Regulatory links                       58.   Anderson MS, Raetz CR (1987) Biosynthesis lipid A precursors Escherichia <br />     carbon nitrogen metabolism. Curr Opin Microbiol 9: 167&#8211;172.                                   coli. A cytoplasmic acyltransferase converts UDP-N-acetylglucosamine to <br /> 39. Rabus R, Reizer J, Paulsen I, Saier MH Jr (1999) Enzyme I(Ntr) Escherichia                   UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine. J Biol Chem 262: <br />     coli. A novel enzyme phosphoenolpyruvate-dependent phosphotransferase                      5159&#8211;5169. <br />     exhibiting strict specificity phosphoryl acceptor, NPr. J Biol Chem          59.   Humphreys S, Stevenson A, Bacon A, Weinhardt AB, Roberts M (1999) The <br />     274: 26185&#8211;26191.                                                                                 alternative sigma factor, sE, critically important virulence Salmonella <br />                                                                                                       typhimurium. Infect Immun 67: 1560&#8211;1568. <br /> 40. Lee CR, Cho SH, Yoon MJ, Peterkofsky A, Seok YJ (2007) Escherichia coli <br />                                                                                                 60.   Liu D, Reeves PR (1994) Escherichia coli K12 regains O antigen. Microbiology <br />     enzyme IIANtr regulates K+ transporter TrkA. Proc Natl Acad Sci USA 104: <br />                                                                                                       140: 49&#8211;57. <br />     4124&#8211;4129. <br />                                                                                                 61.   Miller JH (1972) Experiments Molecular Genetics. Cold Spring Harbor, NY: <br /> 41. Lee CR, Koo BM, Cho SH, Kim YJ, Yoon MJ, et al. (2005) Req <br /> </body></html>