<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, July 2008, p. 4849&#8211;4858                                                                                 Vol. 190, No. 14 <br /> 0021-9193/08/$08.00&#11001;0 doi:10.1128/JB.00185-08 <br /> Copyright &#169; 2008, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />             NrdI Essentiality <span id='am-290' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-291' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-292' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Class</span> Ib Ribonucleotide Reduction in <br />                               <span id='am-51' about='obo:NCBITaxon_1301' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="PMID:10555340" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="PMID:1720654" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="PMID:8995803" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="PMID:7537076" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="PMID:14657115" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="Streptococcus" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="PMID:19620365" datatype="xsd:string"></span><span id='am-61'  ></span>Streptococcus</span> pyogenes&#17188;&#8224; <br />      Ignasi Roca,1 Eduard Torrents,1,2,3 Margareta Sahlin,2 Isidre Gibert,1 Britt-Marie Sjo <br />                                                                                               &#168;berg2* <br />      Grup Gene`tica Molecular Bacteriana, Institut Biotecnologia Biomedicina, Departament Gene`tica Microbiologia, <br />              Universitat Auto`noma Barcelona, Bellaterra, 08913 Barcelona, Spain1; Department Molecular Biology and <br />               Functional Genomics, Arrhenius Laboratories, Stockholm University, SE-10691 Stockholm, Sweden2; and <br />                      Institute Bioengineering Catalonia (IBEC), Scientific Park Barcelona, Edifici He`lix, <br />                                            Baldiri Reixac 15-21, ES-08028 Barcelona, Spain3 <br />                                               Received 6 February 2008/Accepted 14 May 2008 <br />  <br />              The <span id='am-98' about='obo:NCBITaxon_1314' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1301 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-99' property="oboInOwl:hasRelatedSynonym" content="Streptococcus scarlatinae" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="Micrococcus scarlatinae" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="Streptococcus erysipelatos" datatype="xsd:string"></span><span id='am-102' property="rdfs:label" content="Streptococcus pyogenes" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="Streptococcus hemolyticus" datatype="xsd:string"></span><span id='am-105'  ></span><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="Streptococcus pyrogenes" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Streptococcus pyogenes</span> genome harbors clusters class Ib ribonucleotide reductase genes, nrdHEF and <br />           nrdF*I*E*, second stand-alone nrdI gene, designated nrdI2. We clusters expressed <br />           simultaneously independent operons. The NrdEF enzyme functionally active vitro, NrdE*F* <br />           enzyme not. The NrdF* protein lacks highly conserved iron-liganding chains form <br />           dinuclear iron site tyrosyl radical. In vivo, hand, operons functional heterologous <br />           complementation <span id='am-361' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-362'  ></span><span id='am-363' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-364' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-365' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-366' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-367' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-368' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-369' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-370' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-371' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-372' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-373' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-374' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-375' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-376' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span>Escherichia coli</span>. The nrdF*I*E* operon requires presence nrdI* gene, nrdHEF <br />           operon gained activity cotranscription heterologous nrdI gene <span id='am-91' about='obo:NCBITaxon_1313' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1301 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="Diplococcus pneumoniae" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-95'  ></span><span id='am-96' property="rdfs:label" content="Streptococcus pneumoniae" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="Micrococcus pneumoniae" datatype="xsd:string"></span>Streptococcus pneumoniae</span>, while <br />           nrdI* nrdI2 S. pyogenes rendered active. Our results highlight essential role flavodoxin <br />           NrdI protein vivo, suggest needed reduce met-NrdF, enabling spontaneous <br />           reformation tyrosyl radical. The NrdI* flavodoxin play direct role ribonucleotide reduction by <br />           NrdF*I*E* system. We discuss possibility nrdF*I*E* operon horizontally transferred to <br />           S. pyogenes <span id='am-212' about='obo:NCBITaxon_2093' typeof='owl:Thing obo:NCBITaxon_2092 obo:NCBITaxon_1 obo:NCBITaxon_2085 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_544448 obo:NCBITaxon_31969'><span id='am-213' property="oboInOwl:hasAlternativeId" content="NCBITaxon:57371" datatype="xsd:string"></span><span id='am-214' property="oboInOwl:hasRelatedSynonym" content="Borrelomyces" datatype="xsd:string"></span><span id='am-215' property="oboInOwl:hasDbXref" content="PMID:11411711" datatype="xsd:string"></span><span id='am-216' property="oboInOwl:hasDbXref" content="PMID:15176735" datatype="xsd:string"></span><span id='am-217' property="oboInOwl:hasRelatedSynonym" content="Bovimyces" datatype="xsd:string"></span><span id='am-218' property="oboInOwl:hasRelatedSynonym" content="Asteromyces" datatype="xsd:string"></span><span id='am-219' property="oboInOwl:hasDbXref" content="PMID:11931184" datatype="xsd:string"></span><span id='am-220'  ></span><span id='am-221' property="oboInOwl:hasDbXref" content="GC_ID:4" datatype="xsd:string"></span><span id='am-222' property="oboInOwl:hasDbXref" content="PMID:8995799" datatype="xsd:string"></span><span id='am-223' property="oboInOwl:hasDbXref" content="PMID:16350067" datatype="xsd:string"></span><span id='am-224' property="oboInOwl:hasDbXref" content="PMID:10826816" datatype="xsd:string"></span><span id='am-225' property="oboInOwl:hasDbXref" content="PMID:16403858" datatype="xsd:string"></span><span id='am-226' property="oboInOwl:hasDbXref" content="PMID:11321109" datatype="xsd:string"></span><span id='am-227' property="oboInOwl:hasDbXref" content="PMID:13403276" datatype="xsd:string"></span><span id='am-228' property="oboInOwl:hasRelatedSynonym" content="Eperythrozoon" datatype="xsd:string"></span><span id='am-229' property="oboInOwl:hasDbXref" content="PMID:8863441" datatype="xsd:string"></span><span id='am-230' property="oboInOwl:hasRelatedSynonym" content="Haemobartonella" datatype="xsd:string"></span><span id='am-231' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-232' property="oboInOwl:hasAlternativeId" content="NCBITaxon:29500" datatype="xsd:string"></span><span id='am-233' property="oboInOwl:hasRelatedSynonym" content="Pleuropneumonia" datatype="xsd:string"></span><span id='am-234' property="rdfs:label" content="Mycoplasma" datatype="xsd:string"></span><span id='am-235' property="oboInOwl:hasRelatedSynonym" content="Asterococcus" datatype="xsd:string"></span>Mycoplasma</span> spp. <br />  <br />  <br />    Ribonucleotide reductases (RNRs) group enzymes                     function remains unknown (18). Typically, class Ib nrd genes <br /> that carry metabolic pathway providing living                 arranged nrdHIEF operon, examples of <br /> cells optimal amounts necessary buildings blocks                   bacteria bearing class Ib genes slightly different patterns <br /> for DNA replication DNA repair (29). Three classes                      described (43). <br /> RNRs (I, II, III) characterized date accord-                     The genome gram-positive bacterium Streptococcus <br /> ing different mechanisms radical generation, allo-                pyogenes (group A Streptococcus) harbors different clus- <br /> steric regulation, oxygen dependence quater-                 ters genes involved ribonucleotide reduction (10). One <br /> nary structural differences, common                  cluster contains genes anaerobic class III RNR and <br /> reaction mechanism based radical chemistry cata-                    cognate activase (nrdDG), surprisingly, two <br /> lytic core structure (29).                                                     clusters contains genes form unique class Ib operons <br />    Class I RNRs (which subdivided class <span id='am-7' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-8' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Ia</span>                (nrdHEF nrdF*I*E*). The S. pyogenes genome con- <br /> class Ib) composed homodimeric proteins, R1 (&#65533;?&#65533;2)                    tains second stand-alone copy nrdI gene (designated <br /> and R2 (&#65533;?&#65533;2). The larger &#65533;?&#65533; subunit contains catalytic                   nrdI2 work) chromosome. Two features <br /> allosteric sites, smaller &#65533;?&#65533; subunit harbors stable free            make class Ib operons S. pyogenes unique. First, the <br /> radical tyrosine residue diiron center bridged an <br />                                                                                operons arranged way deviates common <br /> oxo (O2&#11002;) ion. Class Ib shares class Ia subunit com- <br />                                                                                nrdHIEF operon structure, second, residues involved <br /> position conserved residues, including tyrosyl <br />                                                                                iron binding conserved &#65533;?&#65533; subunit the <br /> radical iron ligands &#65533;?&#65533; subunit active-site <br />                                                                                nrdF*I*E* operon. <br /> cysteines &#65533;?&#65533; subunit, class Ib &#65533;?&#65533; subunits lack <br />                                                                                   Although common multiple RNR classes within <br /> approximately 50 N-terminal amino acids constitute the <br />                                                                                single microorganism (41), just cases potential re- <br /> allosteric-activity site. The &#65533;?&#65533; &#65533;?&#65533; subunits encoded the <br />                                                                                dundancy, i.e., duplication nrd genes belonging the <br /> nrdA nrdB genes class Ia nrdE nrdF genes <br /> for class Ib. Class Ib operons contain additional nrd genes:               class, characterized. <span id='am-263' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-264' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-265' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-266' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-267' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-268'  ></span><span id='am-269' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-270' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-271' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-272' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-273' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-274' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-275' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-276' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-277' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-278' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-279' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br /> nrdH, encodes <span id='am-415' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-416' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-417' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-418'  ></span><span id='am-419' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>electron</span> donor glutaredoxin-like pro-                   class Ia redundancy (Rnrp1 Rnrp4), Mycobacte- <br /> tein (19), nrdI, encodes small flavoprotein                  rium tuberculosis second class Ib-like &#65533;?&#65533; subunit gene (46, <br />                                                                                48; http://rnrdb.molbio.su.se additional candi- <br />                                                                                dates). S. pyogenes bacteria (all belonging the <br />    * Corresponding author. Mailing address: Department Molecular            <span id='am-31' about='obo:NCBITaxon_1239' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="firmicutes" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Gram positive bacteria" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="Gram-positive bacteria" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="clostridial firmicutes" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="PMID:15143038" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Firmacutes" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="PMID:10555317" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="PMID:11542017" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="low GC Gram+" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Bacillus/Clostridium group" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="PMID:11034484" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="low G+C Gram-positive bacteria" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Firmicutes" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Low G+C firmicutes" datatype="xsd:string"></span><span id='am-48'  ></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Clostridium group firmicutes" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasAlternativeId" content="NCBITaxon:31968" datatype="xsd:string"></span>Firmicutes</span>) distinct class Ib RNR gene clusters. <br /> Biology &amp; Functional Genomics, Stockholm University, SE-10691                  A redundancy RNR genes great importance <br /> Stockholm, Sweden. Phone: 46-8-164150. Fax: 46-8-166488. E-mail:               different environmental conditions. In study, we <br /> britt-marie.sjoberg@molbio.su.se. <br />                                                                                analyzed vivo vitro functionality of <br />    &#8224; Supplemental material article http://jb <br /> .asm.org/.                                                                     S. pyogenes class Ib nrd genes attempt provide an <br />    &#17188; <br />      Published ahead print 23 May 2008.                                  explanation atypical redundancy. <br />  <br />                                                                         4849 <br />  4850       ROCA ET AL.                                                                                                                                J. BACTERIOL. <br />  <br />                         MATERIALS AND METHODS                                          Protein purifications. Frozen pellets disintegrated X-press (Biox AB) <br />                                                                                     extracted 50 mM Tris-HCl (pH 8), 50 mM KCl, 5 mM dithiothreitol <br />    Bacterial strains, <span id='am-435' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-436' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-437' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-438' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-439' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span>, growth conditions. <span id='am-377' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-378' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-379' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-380'  ></span><span id='am-381' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-382' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-383' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia</span> coli strains <br />                                                                                     (DTT), 1 mM phenylmethylsulfonyl fluoride (PMSF) (buffer L), which <br /> were routinely grown <span id='am-207' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-208' property="rdfs:label" content="Luria" datatype="xsd:string"></span><span id='am-209'  ></span><span id='am-210' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-211' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Luria</span>-Bertani (LB) broth 15, 30, 37&#176;C. Streptococ- <br />                                                                                     nucleic acids precipitated dropwise addition streptomycin sulfate a <br /> cus pyogenes Rosenbach M1 (ATCC 700294, SF370) grown Todd-Hewitt <br />                                                                                     final concentration 1%. The precipitate removed centrifugation, and <br /> broth (Oxoid) supplemented 0.2% yeast extract (THY) 37&#176;C in <br />                                                                                     solid ammonium sulfate added supernatant 45% saturation for <br /> Columbia sheep blood agar plates (BioMe&#180;rieux) needed. Streptococcus <br />                                                                                     NrdE NrdE* 60% saturation NrdF NrdF*. The precipitate <br /> pneumoniae (Klein) Chester R6 (ATCC BAA-255) grown Columbia <br />                                                                                     recovered centrifugation. <br /> sheep blood agar plates (BioMe&#180;rieux) 37&#176;C. Antibiotics chromogenic <br />                                                                                        The NrdF NrdF* pellets dissolved buffer L desalted <span id='am-293' about='obo:NCBITaxon_124307' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7147 obo:NCBITaxon_7203 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_92613 obo:NCBITaxon_43736 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_125599 obo:NCBITaxon_33392 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_50557 obo:NCBITaxon_6960 obo:NCBITaxon_6072'><span id='am-294' property="rdfs:label" content="Dialysis" datatype="xsd:string"></span><span id='am-295' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-296'  ></span><span id='am-297' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>dialysis</span> <br /> substrates, required, included culture media agar plates     2 liters buffer L 4 5 h. The dialyzed solution diluted with <br /> at following concentrations E. coli: 100 &#65533;?&#65533;g/ml carbenicillin, 100 &#65533;?&#65533;g/ml     buffer L 5-mg/ml protein concentration loaded DEAE-Sepharose <br /> kanamycin, 34 &#65533;?&#65533;g/ml chloramphenicol, 30 &#65533;?&#65533;g/ml 5-bromo-4-chloro-3-indo-          column (GE Healthcare) coupled fast-performance liquid chromatography <br /> lyl-&#65533;?&#65533;-D-galactopyranoside (X-Gal) (Sigma).                                          instrument (Biologic DuoFlow Systems, Bio-Rad). After washing step 140 <br />    The E. coli strain IG101 [nrdA(Ts) nrdB1 thyA thr leu thi deoC and/or deoD       ml 0.15 M KCl buffer L, elution continued 0.15 0.6 M KCl <br /> tonA lacY supE44 gyrA nrdH::Spc] constructed introducing nrdH::Spc       linear gradient buffer L total volume 300 ml. Fractions checked by <br /> allele (IG100 [<span id='am-298' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-299' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-300' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-301' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-302'  ></span>this</span> laboratory]) KK450 (31) P1 transduction.                sodium dodecyl sulfate-polyacrylamide gel electrophoresis containing NrdF <br />    Sequence analysis DNA manipulations. Genomic DNA S. pyogenes            NrdF* protein pooled concentrated ultrafiltration Centriplus <br /> was extracted described previously (11). Electroporation S. pyogenes      YM-30 instrument (Millipore) buffer L stored &#11002;80&#176;C further <br /> performed described previously (25), general DNA manipulations         purification. The NrdF proteins eluted 0.3 0.36 M KCl. The concen- <br /> were standard procedures (34). DNA sequence determination               trated enzyme solution loaded Superdex 75 10/300 GL column (GE <br /> carried using BigDye Terminator kit ABI Prism 310 sequencer          Healthcare) previously equilibrated buffer L. Proteins eluted with <br /> (Applied Biosystems) according manufacturer&#8217;s specifications. Images      buffer constant flow rate 0.4 ml/min. Fractions containing NrdF <br /> gels captured Gel Doc 2000 station Quantity One software        NrdF* pooled, concentrated, stored described above. <br /> package (Bio-Rad Laboratories).                                                        For purification NrdE NrdE* proteins, ammonium sulfate- <br />    All nrd sequences RNRdb, RNR database (http://rnrdb.molbio         precipitated pellets dissolved 50 mM Tris-HCl (pH 8), 0.75 M ammo- <br /> .su.se). The Clustal W 1.8 (40) GeneDoc 2.6.001 (28) programs used      nium sulfate, 2.5 mM DTT, 1 mM PMSF loaded HiLoad 16/10 <br /> perform global alignments.                                                          phenyl-Sepharose high-performance column (GE Healthcare) equilibrated with <br />    RNA isolation. S. pyogenes grown 10 ml THY A600          buffer. Elution performed 0.75 0 M ammonium sulfate <br /> approximately 0.4. Ten-milliliter cultures centrifuged resuspended      linear gradient 50 mM Tris-HCl (pH 8), 2.5 mM DTT, 1 mM PMSF. NrdE <br /> 200 &#65533;?&#65533;l diethyl pyrocarbonate (Sigma)-treated distilled water quick-frozen    NrdE* eluted 0.44 0.35 M 0.03 0 M ammonium sulfate, <br /> in dry ice bath. Pellets thawed ice added 2 ml FastPrep Blue       respectively. The enzyme-containing fractions pooled concentrated in <br /> tubes containing zirconite/silica beads, 500 &#65533;?&#65533;l CPRS-Blue, 500 &#65533;?&#65533;l acid    Centriplus YM-50 (Millipore) adsorbed 2 ml MonoQ anion- <br /> phenol, described manufacturer (Bio101, La Jolla, CA). Bacteria      exchange column (GE Healthcare) equilibrated buffer L. Proteins were <br /> lysed FastPrep FP120 instrument (Bio101, La Jolla, CA) setting 6      eluted 20 ml 0.05 0.5 M KCl linear gradient buffer L flow rate <br /> 11 s immediately placed ice disruption. Samples incubated      0.1 ml/min. NrdE eluted 0.21 0.28 M KCl NrdE* between <br /> 65&#176;C 10 min centrifuged 10,000 &#11003; g 5 min 4&#176;C. The upper           0.19 0.27 M KCl. Fractions 0.4 ml pooled concentrated as <br /> aqueous phase recovered extracted equal volume acid phenol       described stored &#11002;80&#176;C. <br /> heated 65&#176;C. The phases separated centrifugation described above,        Analytical methods enzyme activity assays. Iron reconstitution assays <br /> and extraction repeated acid phenol-chloroform (1:1, vol/vol)      performed anaerobically described reference 17, iron content was <br /> chloroform-isoamyl alcohol (24:1, vol/vol). The recovered aqueous phase         determined described reference 33. Protein concentration determined <br /> treated 50 U DNase I (Ambion, Inc.) 30 min 37&#176;C, 0.2        Bradford method (6). <br /> volume inactivating reagent (Ambion, Inc.) added eliminate en-           RNR activity assayed 50-&#65533;?&#65533;l mixtures containing 50 mM Tris-HCl (pH <br /> zyme. RNA concentrations determined measuring A260. Samples             8), 0.2 mM dATP effector, 20 mM magnesium acetate, 30 mM DTT. <br /> were immediately stored &#11002;80&#176;C. Reverse transcription-PCR (RT-PCR)            The reaction started addition [3H]CDP (specific activity, 10,000 to <br />                                                                                     20,000 cpm/nmol) final concentration 1 mM. Assay mixtures incu- <br /> performed described reference 43, identical aliquots processed in <br />                                                                                     bated 10 min 29&#176;C, reaction stopped addition 0.5 ml <br /> parallel addition RT <span id='am-311' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-312' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-313' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> ensure residual chromosomal <br />                                                                                     1 M ice-cold perchloric acid. The dCDP formed determined by <br /> DNA used template PCR amplification. <br />                                                                                     standard method described reference 39. One unit enzyme activity <br />    Cloning expression. The nrdF, nrdE, nrdF*, nrdE* genes am- <br />                                                                                     corresponds 1 nmol dCDP formed minute incubation. Km, Vmax, and <br /> plified PCR using primer pairs containing NdeI (upper primers) BamHI <br />                                                                                     KL (ligand binding constant) values obtained direct curve fitting as <br /> (lower primers) restriction sites S. pyogenes genomic DNA template (see <br />                                                                                     described previously (3, 7) Kaleida-Graph software (Synergy Soft- <br /> Table S1 supplemental material). The resulting fragments purified <br />                                                                                     ware). <br /> and digested NdeI BamHI. The digested nrdF, nrdE, nrdE* frag- <br />                                                                                        UV&#8211;visual-light (UV-vis) absorption spectroscopy decay tyrosyl <br /> ments inserted pET22b(&#11001;) (Novagen) cut enzymes to <br />                                                                                     radical site. Scanning spectra registered Perkin Elmer Lambda2 <br /> give p22-FSpyo, p22-ESpyo, p22-E*Spyo, respectively. The digested nrdF* <br />                                                                                     spectrophotometer 25&#176;C, 20 &#65533;?&#65533;M concentration monomer 50 <br /> fragment inserted pET24a(&#11001;) (Novagen) cut enzymes         mM Tris-HCl, pH 7.6, used. The radical decay nrdF protein was <br /> give p24-F*Spyo. The sequences of above-named constructs            studied addition hydroxyurea (HU) final concentrations ranging <br /> verified sequencing.                                                             3.5 60 mM. Spectra typically recorded 300 750 nm in <br />    E. coli BL21(DE3) (Novagen) cells transformed expression         cycles 150 s, scanning rate 240 nm/min. The endpoint spectrum, when <br /> plasmids (p22-FSpyo, p22-ESpyo, p22-E*Spyo) grown overnight LB           radical absorption 407 nm decayed, subtracted start spec- <br /> medium containing carbenicillin 30 37&#176;C. This culture used        trum radical concentration using extinction coefficient 3.25 mM&#11002;1 <br /> inoculate 5-liter flasks containing 2 liters LB medium concen-     cm&#11002;1 (30). Alternatively, radical concentration determined calculat- <br /> tration antibiotic noted (1:100). Cells grown 30&#176;C   ing dropline 402 417 nm using &#949;407 2.11 mM&#11002;1 cm&#11002;1 <br /> vigorous shaking A600 &#11011;0.5 0.6 subsequently induced 0.5 mM     equation {A407 &#11002; [A402 &#11002; (5/15) &#18528; (A402 &#11002; A417)]}/&#949;. Kinetic traces 407 <br /> IPTG (isopropyl-&#65533;?&#65533;-D-thiogalactopyranoside; final concentration) 4 hours.        nm run determine decay rate; incubations E. coli HU <br />    BL21(DE3) transformed plasmid p24-F*Spyo grown overnight LB          run 410 nm controls. Spectra run kinetic <br /> medium containing kanamycin 30&#176;C. This culture used inoculate        trace measure degree completeness reaction subtractions as <br /> 5-liter flasks containing 2 liters LB medium concentration      described above. <br /> antibiotic noted (1:100). Cells grown 30&#176;C vigorous         EPR spectroscopy. Electron paramagnetic resonance (EPR) measurements at <br /> shaking. When reached A600 &#11011;0.5 0.6, cultures chilled     X-band performed &#11350;90 K Bruker ESP300 spectrometer using <br /> 15&#176;C induced 0.3 mM IPTG (final concentration)           nitrogen flow &#65533;?60 K Bruker ESP500 using Oxford helium <br /> grown 15 17 h 15&#176;C. After induction, cells harvested centrifu-    flow cryostat. Radical concentrations determined, unsaturated con- <br /> gation pellets quickly frozen stored &#11002;80&#176;C.                          ditions, using E. coli NrdB sample known tyrosyl radical concentrations <br />  VOL. 190, 2008                                                                             CLASS Ib RNR REDUNDANCY IN S. PYOGENES                     4851 <br />  <br />  <br />  <br />  <br />   FIG. 1. (A) Schematic organization S. pyogenes class Ib nrd genes. Arrows indicate positions numbers primers used for <br /> RT-PCR amplification, black lines labeled A, B, C, D positions lengths expected transcripts. (B) Linkage RT-PCR <br /> of nrdHE (lane A), nrdEF (lane B), nrdFIE (lane C) junctions nrdI2 gene (lane D). The linkage strategy outlined panel A. <br /> MWM, molecular weight markers. <br />  <br />  <br />  <br /> as standard. The error limits radical quantitations &#11006;10%.           spans 4 kbp. The nrdI* gene encodes unknown protein <br /> Reconstituted samples EPR measurements frozen liquid               18 kDa 162 amino acids 75% similar E. coli <br /> nitrogen 6 min addition 2 Fe ions/polypeptide frozen the <br /> addition 4 Fe ions/monomer followed desalting NAP-5 column to <br />                                                                                         NrdI (19). S. pyogenes nrdE* encodes RNR large subunit <br /> remove unbound iron.                                                                    essential cysteines active site, C-terminal <br />    Heterologous complementation assay. Class Ib nrd genes amplified             cysteine pair, conserved C-terminal tyrosines (re- <br /> PCR using primers pairs containing ribosomal binding site sequence E.            quired transfer radical small subunit) as <br /> coli SacI (upper primers) PstI (lower primers) restriction sites (see <br /> Table S1 supplemental material). The resulting fragments purified <br />                                                                                         residues form allosteric-specificity site (see <br /> and digested SacI PstI inserted pBAD18Km pBAD33Cm                  Table S2 supplemental material). Finally, nrdF* gene <br /> (14) generate final constructs. p18-FI*Spyo, p18-IE*Spyo, p18-               encodes RNR small subunit lacks important <br /> FE*Spyo derived p18-FIE*Spyo enzymatic digestion.                          residues proper function enzyme. Compared to <br />    In heterologous complementation assay, constructs transformed into <br />                                                                                         E. coli NrdF protein, S. pyogenes NrdF* turned have <br /> the E. coli strain IG101 grown overnight 30&#176;C LB medium containing <br /> 50 &#65533;?&#65533;g/ml thymine plus corresponding antibiotics. The following day, serial          amino acid substitutions important residues <br /> dilutions liquid cultures plated LB agar plates containing         involved iron binding residues <br /> corresponding antibiotics plus 50 &#65533;?&#65533;g/ml thymine 0.3% L-arabinose          involved radical stability. Instead Glu115 E. coli NrdB, <br /> 0.3% D-glucose incubated 30 42&#176;C 24 48 h. Attempts <br />                                                                                         S. pyogenes NrdF* Val116, instead Glu204 Pro176, <br /> to obtain S. pyogenes chromosomal mutants insertional recombinant <br /> inactivation class Ib nrd genes unsuccessful, initially attrib-   instead Glu238 Lys210, instead Phe208 has <br /> uted essentiality genes. However, used muta-           Leu180. All residues important radical stability and <br /> genic strategies obtain chromosomal mutant previously described             transfer conserved (see Table S3 supplemental ma- <br /> nonessential covR gene (9), mutant colonies recovered extremely low <br />                                                                                         terial). <br /> frequencies (1 &#11003; 10&#11002;8); therefore, essentiality S. pyogenes class Ib nrd <br /> genes assessed direct complementation.                                     A second stand-alone nrdI gene (nrdI2) annotated else- <br />                                                                                         genome (10). S. pyogenes nrdI2 encodes protein <br />                                                                                         160 amino acids 17.7 kDa 36% similar nrdI*. <br />                                    RESULTS                                                 The redundant class Ib operons expressed simulta- <br />    S. pyogenes encodes redundant class Ib gene clusters.                            neously. To investigate class Ib RNR regions were <br /> The complete S. pyogenes Rosenbach M1 genome (10) con-                                  simultaneously transcribed assess operonic organi- <br /> tains seven genes putatively encoding class Ib RNR-related                              zation cluster, total RNA strain SF370 ex- <br /> proteins different regions genome (Fig. 1A). All                        tracted early exponential phase used perform non- <br /> 10 hitherto-completed S. pyogenes genomes encode                               quantitative RT-PCR experiments. Primer pairs selected <br /> seven class Ib genes (see RNRdb http://rnrdb.molbio                              bracket intergenic junctions nrdHE, nrdEF, and <br /> .su.se).                                                                                nrdF*I*E* (Fig. 1A). Any PCR fragment generated these <br />    The class Ib region spans 4.5 kbp comprises                           oligonucleotides derived polycistronic tran- <br /> the nrdHEF cluster, codes putative glutaredoxin-                            script. <br /> like protein (NrdH) large (NrdE) small                                  An amplicon spanning nrdHE junction detected by <br /> (NrdF) subunits holoenzyme. Both NrdE NrdF                               using primer 2 generate cDNA primers 1 2 (see <br /> proteins contained conserved residues necessary                             Table S1 supplemental material) PCR (Fig. 1B, <br /> ribonucleotide reduction (see Table S2 S3 supple-                            lane A). Similarly, PCR product bracketing junction <br /> mental material).                                                                       nrdE nrdF detected primers 3 <br />    The second Ib region comprises nrdF*I*E* cluster                             4 (Fig. 1B, lane B). Using primer 6 generate reversed <br />  4852     ROCA ET AL.                                                                                                              J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 2. (A) Line a, UV-vis spectra S. pyogenes NrdF protein. The arrow indicates tyrosyl radical. Dotted line b, HU-treated sample. <br /> Line a-b, subtraction HU-treated spectrum purified NrdF spectrum. (B) UV-vis spectra S. pyogenes NrdF* protein upon <br /> purification (dashed line) addition 2 Fe ions/monomer. <br />  <br />  <br />  <br /> transcribed cDNA template primer pair 5-6 amplify              registered EPR spectrum NrdF protein (Fig. 3), and <br /> this template, able detect amplicon span-              tyrosyl radical proved typical class Ib radical similar <br /> ning nrdF*I*E* intergenic junction (Fig. 1B, lane C). An              NrdF proteins S. enterica serovar Ty- <br /> RT-PCR second nrdI2 gene carried                phimurium, <span id='am-171' about='obo:NCBITaxon_1697' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1716 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_1653'><span id='am-172' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium ammoniagenes (Cooke and Keith 1927) Breed 1953 (Approved Lists 1980)" datatype="xsd:string"></span><span id='am-173' property="rdfs:label" content="Corynebacterium ammoniagenes" datatype="xsd:string"></span><span id='am-174' property="oboInOwl:hasRelatedSynonym" content="Bacterium ammoniagenes" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasRelatedSynonym" content="&quot;Bacterium ammoniagenes&quot; Cooke and Keith 1927" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium ammoniagenes" datatype="xsd:string"></span><span id='am-177' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasExactSynonym" content="Corynebacterium ammoniigenes" datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasRelatedSynonym" content="Corynebacterium ammoniagenes (Cooke and Keith 1927) Collins 1987" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-181'  ></span>Corynebacterium ammoniagenes</span>, M. tubercu- <br /> primers 7 8, corresponding transcript readily             losis (17, 21, 24). At 95 K, radical saturated 1 order of <br /> detected (Fig. 1B, lane D). The nrdI2 product sequenced               magnitude power does E. coli NrdB protein (4.7 <br /> to ensure correct amplification.                                      mW compared 54.0 mW [<span id='am-425' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-426' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-427' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-428'  ></span><span id='am-429' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> shown]) similarly to <br />    Control experiments conducted concurrently set               S. enterica serovar Typhimurium, C. ammoniagenes, M. <br /> of RT-PCR assays confirmed quality samples                 tuberculosis NrdF proteins (3.7 mW, 1.3 mW, 1.3 mW, <br /> guaranteed specificity component RT-PCR.               respectively) (17, 44). The S. pyogenes NrdF radical signal had <br /> (i) Our inability amplify desired product RT              hyperfine structure 10 20 K 95 K that <br /> omitted validated compulsory requirement enzyme               previously observed class Ib radicals mentioned above, <br /> for initiation cDNA synthesis ensured re-              contrast E. coli Ia signal, considerably broad- <br /> sidual DNA contaminated starting RNR preparations; (ii)               ened 95 K compared structure 10 20 K (32). The <br /> the expected amplicons detected genomic DNA                     radical saturation behavior reflects environment, case <br /> acted template, demonstrating fidelity PCR              plausibly coupling iron, geometry. The difference <br /> primers (data shown). These results indicated presence            E. coli Ia radical indicate S. pyogenes <br /> of polycistronic transcript cluster genes        NrdF radical weaker coupling iron site, perhaps <br /> their simultaneous transcription standard laboratory <br /> growth conditions. <br />    Spectroscopic characterization NrdF NrdF*. Once it <br /> was demonstrated class Ib nrd genes S. pyogenes were <br /> actively transcribed, decided check were <br /> capable forming active holoenzyme. The nrdE nrdF <br /> genes nrdHEF nrdF*I*E* operons were <br /> cloned, overproduced, purified described Materials <br /> and Methods. After different purification steps, purified <br /> proteins showed expected theoretical molecular mass. The <br /> purity samples cases 90%. Attempts to <br /> overproduce purify NrdI* NrdI2 proteins were <br /> unsuccessful overproduced proteins continu- <br /> ously recovered insoluble fraction bacterial ex- <br /> tract, regardless incubation temperature IPTG <br /> concentration used (not shown). <br />    Purified NrdF protein showed light absorption spectrum <br /> typical diiron-tyrosyl radical RNR. Charge transfer bands <br /> of &#65533;?&#65533;-oxobridged diferric center observed 325 and <br /> 370 nm, weaker bands 500 600 nm (Fig. 2A).                   FIG. 3. First-derivative EPR spectra NrdF NrdF*, indi- <br /> The tyrosyl radical showed sharp peak 407 nm. We                cated figure. <br />  VOL. 190, 2008                                                            CLASS Ib RNR REDUNDANCY IN S. PYOGENES                     4853 <br />  <br />      TABLE 1. Iron radical contents S. pyogenes NrdF        spectra (Fig. 2B, dashed line) EPR signal (Fig. 3); i.e., <br />            NrdF* iron reconstitutiona                 diiron-tyrosyl radical site. As expected, analysis the <br />                            No. Fe ions/            No. tyrosyl   purified NrdF* iron content showed 0.05 Fe ion/monomer <br /> Protein <br />                              monomer                radicals/monomer   (Table 1). The addition 2 Fe ions/monomer gave spectrum <br /> NrdF                            1.2                       0.5          radical absorption characteristics of <br /> NrdFR                           2.1                       0.5          diiron site observed NrdF protein absorp- <br /> NrdF*                           0.05                      ND           tion 325- 450-nm region, similar been <br /> NrdFR*                          0.47                      ND           observed iron ligand mutants E. coli NrdB (Fig. 2B, <br />  a <br />      R, iron reconstituted; ND, detected.                      dotted line) (&#949;350 &#11015; 1,800 M&#11002;1 cm&#11002;1) (1, 45). Iron analysis of <br />                                                                        reconstituted desalted samples gave values &#11011;0.47 Fe <br />                                                                        ion/monomer. Presumably, altered residues iron <br /> with water molecule radical closest iron,        binding center protein impair formation a <br /> as class Ib tyrosyl radicals. Addition radical        dinuclear iron center tyrosyl radical. <br /> scavenger HU reduce tyrosyl radical gave spectrum              The NrdEF enzyme enzymatically active, the <br /> shown Fig. 2A (dotted line), shown diiron      NrdE*F* enzyme not. When assayed enzymatic activity, <br /> site remained essentially unchanged. Subtraction end-           proteins activity own. When <br /> point spectrum gives typical tyrosyl radical spectrum (Fig.          mixed assayed together, NrdE NrdF proteins <br /> 2A, lowest spectrum) sharp peak 407 nm. Using             purified nrdHEF active showed pro- <br /> extinction coefficient (&#949;) 3.25 mM&#11002;1 cm&#11002;1 (30) sub-          portional dCDP formation, protein excess and <br /> tracted spectra using dropline specified Materials        increasing amounts partner. Specific activities, the <br /> and Methods, calculated, best preparation, radical        presence dATP positive allosteric effector, 169 <br /> content 0.51 monomer, good agreement 0.57           nmol/min/mg NrdF protein 45 nmol/min/mg the <br /> radical monomer determined EPR. Analysis                 NrdE protein. Activity dependent addition mag- <br /> NrdF iron content showed approximately 1.2 Fe ions/monomer             nesium DTT optimal pH 8.0 required CDP as <br /> (Table 1). It uncommon RNRs purify subop-             substrate, Km 0.34 mM (Fig. 4A). CDP reduction <br /> timal iron content and, consequently, small radical        strongly stimulated dATP effector lesser <br /> when overexpressed E. coli. Hence, added fer-           extent ATP. Optimal activity dATP obtained at <br /> rous ions protein investigate increase       nucleotide concentrations low 0.01 mM, signifi- <br /> yield diiron-radical site. Addition ferrous ions, how-       inhibition seen 2 mM dATP. When ATP <br /> ever, did increase radical yield NrdF radical-       used, concentrations 2 mM to <br /> containing samples. To assess stability tyrosyl radi-       achieve optimal activity. From Fig. 4B, apparent KL values of <br /> cal, monitored reactivity radical scavenger          0.0011 mM dATP approximately 0.3 mM ATP can <br /> HU. S. pyogenes NrdF reacted considerably lower rate (k1          calculated. No significant gain specific activity de- <br /> [first order rate constant] 0.12 min&#11002;1 60 mM HU)          tected iron-reconstituted NrdF protein, agreement <br /> E. coli NrdB (k1 0.50 min&#11002;1 15 mM HU) 25&#176;C; i.e.,           data provided EPR UV-vis spectroscopy. <br /> it order magnitude sensitive E. coli      The NrdE* NrdF* proteins activity as- <br /> enzyme reduction HU.                                             sayed together, NrdF* combined cata- <br />     On hand, purified NrdF*, obtained           lytically competent NrdE protein, agrees lack of <br /> the apoform, showed prominent features light absorption          tyrosyl radical NrdF*. One might, hand, <br />  <br />  <br />  <br />  <br />   FIG. 4. (A) Comparison CDP CTP substrates NrdF/NrdE reductase. Assay mixtures standard conditions <br /> with 1.2 &#65533;?&#65533;M NrdE 5 &#65533;?&#65533;M NrdF (E, CDP; f, CTP). (B) Effect ATP dATP CDP reduction. Incubation 1.2 &#65533;?&#65533;M NrdE and <br /> 5 &#65533;?&#65533;M NrdF, replacing standard concentration 0.2 mM dATP indicated concentrations ATP (F) dATP (f). <br />  4854     ROCA ET AL.                                                                                                                     J. BACTERIOL. <br />  <br />  <br />                                                                         formed p18-FIE*Spyo gave 1.56 &#11003; 108 CFU/ml 24 to <br />                                                                         48 h 42&#176;C plates supplemented arabinose, not <br />                                                                         glucose, indicating nrdF*I*E* cluster capable <br />                                                                         complementing temperature-sensitive phenotype (Table <br />                                                                         2). The results obtained constructs the <br />                                                                         low-copy-number plasmid pBAD33 instead pBAD18 (data <br />                                                                         shown). Neither pBAD18 pBAD33 able to <br />                                                                         support growth IG101 strain restrictive tem- <br />                                                                         perature. <br />                                                                            Our step determine genes the <br />                                                                         nrdF*I*E* operon required observed vivo <br />                                                                         complementation. A set pBAD18/pBAD33 <span id='am-397' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-398' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-399' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-400' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>vectors</span> were <br />                                                                         constructed gene nrdF*I*E* operon <br />                                                                         evaluated pairwise combinations other <br />                                                                         genes. No complementation detected the <br />                                                                         nrdF*I*E* genes missing (Table 2), suggesting that <br />   FIG. 5. Assay competitive inhibition NrdE         entire operon needed complementation. On the <br /> NrdF. Incubations carried standard conditions (i) with <br /> increasing concentrations NrdF presence 0.6 &#65533;?&#65533;M NrdE (F)    hand, complementation restored missing <br /> and 0.6 &#65533;?&#65533;M NrdE 2 &#65533;?&#65533;M NrdF* (f) (ii) increasing    gene provided trans. <br /> concentrations NrdE presence 0.6 &#65533;?&#65533;M NrdF (v) 0.6          These results stress essential role nrdI* gene in <br /> &#65533;?&#65533;M NrdF 2 &#65533;?&#65533;M NrdE* (&#338;).                                vivo activity. In view results, S. pyogenes nrdI* <br />                                                                         nrdI2 genes cloned pBAD33 cotransformed with <br />                                                                         nrdHEF operon IG101 strain. However, as <br /> expect activity combining NrdE* NrdF                  shown Table 2, S. pyogenes nrdI* nrdI2 could <br /> proteins, NrdE* catalytically im-           complement S. pyogenes nrdHEF overcome tempera- <br /> portant residues NrdF harbors tyrosyl radical              ture-sensitive phenotype. We checked nrdI <br /> catalytically competent. However, mixing NrdE*               gene closely related bacterium Streptococcus pneu- <br /> NrdF did result enzyme activity. One explanation         moniae restore complementation combined S. <br /> the lack activity case NrdE* NrdF         pyogenes nrdHEF. S. pneumoniae harbors nrdHEF operon <br /> form holoenzyme complex. To test this, performed com-              isolated nrdI2 gene lacks nrdFIE operon <br /> petition assays detect protein interactions subunits         (16). Plasmids p18-HEFSpn p33-I2Spn generated by <br /> from different systems. Fixed concentrations NrdF            cloning nrdHEF nrdI2 genes S. pneumoniae into <br /> NrdE assayed increasing amounts                  pBAD vectors. When assayed alone, p18-HEFSpn did not <br /> NrdE* NrdF*. As shown Fig. 5, concentrations             complement IG101 42&#176;C. Transformants p18- <br /> 2 &#65533;?&#65533;M NrdF* NrdE* inhibitory effect NrdF/NrdE               HEFSpn p33-I2Spn, however, restored growth the <br /> holoenzyme activity, suggesting subunits              IG101 strain 42&#176;C arabinose-LB plates (but glu- <br /> systems cross-react.                                             cose), showing S. pneumoniae nrdHEF operon can <br />    Complementation assays nrdF*I*E* functional             complement supplemented S. pneumoniae nrdI2. <br /> in vivo nrdI essential complementation. We         When combined S. pneumoniae nrdHEF operon with <br /> decided compare vitro results nrdHEF and <br /> nrdF*I*E* systems results complementation assays <br /> of vivo functionality operon. To so, used <br /> the E. coli IG101 strain, nrdA(Ts)-nrdB1/nrdH::Spc          TABLE 2. CFU/ml plating efficiencies pBAD constructs in <br />                                                                                   heterologous complementation assay <br /> double mutant unable reduce ribonucleotides aerobically at <br /> 42&#176;C unless functional RNR provided complement                                              Mean no. CFU/ <br />                                                                                                                                        Mean plating <br />                                                                              pBAD construct            ml &#11006; SD 42&#176;C with <br /> temperature-sensitive phenotype. The entire S. pyogenes                                                                              efficiency &#11006; SDa <br />                                                                                                           0.3 % arabinose <br /> nrdHEF nrdF*I*E* sequences cloned separately into <br />                                                                         p18-HIEFSal                   1.56 &#11003; 108 &#11006; 9.60 &#11003; 105        0.214 &#11006; 1.31 &#11003; 10&#11002;3 <br /> the pBAD18 vector control E. coli arabinose            p18-HEFSpyo                            &#65533;?10                     &#65533;?1 &#11003; 10&#11002;8 <br /> promoter (PBAD), on/off switch promoter repressed glu-            p18-FIE*Spyo                  1.56 &#11003; 108 &#11006; 8.10 &#11003; 105        0.214 &#11006; 1.11 &#11003; 10&#11002;3 <br />                                                                         p18-FI*Spyo                            &#65533;?10                     &#65533;?1 &#11003; 10&#11002;8 <br /> cose induced arabinose, generating plasmids p18-                 p18-FE*Spyo                            &#65533;?10                     &#65533;?1 &#11003; 10&#11002;8 <br /> HEFSpyo p18-FIE*Spyo, transformed                   p18-IE*Spyo                            &#65533;?10                     &#65533;?1 &#11003; 10&#11002;8 <br /> into E. coli IG101 strain. Complementation deter-               p18-FI*Spyo/p33-E*Spyo        1.56 &#11003; 10 &#11006; 5.69 &#11003; 105 <br />                                                                                                                8 <br />                                                                                                                                      0.214 &#11006; 7.78 &#11003; 10&#11002;4 <br />                                                                         p18-FE*Spyo/p33-I*Spyo        1.56 &#11003; 10 &#11006; 4.24 &#11003; 105 <br />                                                                                                                8 <br />                                                                                                                                      0.213 &#11006; 5.80 &#11003; 10&#11002;4 <br /> mined plating serial dilutions liquid cultures LB agar         p18-IE*Spyo/p33-F*Spyo        1.54 &#11003; 108 &#11006; 7.78 &#11003; 105        0.211 &#11006; 1.06 &#11003; 10&#11002;3 <br /> plates supplemented 0.3% L-arabinose 0.3%                p18-HEFSpyo/p33-I2Spyo                 &#65533;?10                     &#65533;?1 &#11003; 10&#11002;8 <br />                                                                         p18-HEFSpyo/p33-I*Spyo                 &#65533;?10                     &#65533;?1 &#11003; 10&#11002;8 <br /> D-glucose 30 42&#176;C. As independent positive               p18-HEFSpn/p33-I2Spn          1.55 &#11003; 108 &#11006; 9.74 &#11003; 105        0.212 &#11006; 1.33 &#11003; 10 &#11002;3 <br /> control, used <span id='am-384' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-385' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-386' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-387' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-388'  ></span><span id='am-389' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-390' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-391' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-392' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-393' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span><span id='am-394' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-395' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-396' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span>Salmonella</span> enterica serovar Typhimurium            p18-HEFSpn/p33-I2Spyo                  &#65533;?10                     &#65533;?1 &#11003; 10&#11002;8 <br />                                                                         p18-HEFSpn/p33-I*Spyo                  &#65533;?10                     &#65533;?1 &#11003; 10&#11002;8 <br /> nrdHIEF sequence cloned pBAD18 vector (plasmid                 p18-HEFSpyo/p33-I2Spn         9.66 &#11003; 10 &#11006; 3.21 &#11003; 103 <br />                                                                                                                4 <br />                                                                                                                                   0.00013 &#11006; 4.39 &#11003; 10&#11002;6 <br /> p18-HIEFSal). Unexpectedly, IG101 strain transformed                   a <br />                                                                              Plating efficiencies determined fraction CFU/ml restric- <br /> with p18-HEFSpyo able grow 42&#176;C pres-              tive conditions compared number CFU/ml nonrestrictive condi- <br /> ence arabinose glucose, IG101 trans-               tions. Values based results independent experiments. <br />  VOL. 190, 2008                                                           CLASS Ib RNR REDUNDANCY IN S. PYOGENES                   4855 <br />  <br />  <br /> either S. pyogenes nrdI* nrdI2 gene, however,          nrdHEF operon required coexpression nrdI gene, this <br /> unable detect complementation restrictive tempera-          case S. pneumoniae. To surprise, addition of <br /> ture, suggesting S. pyogenes NrdI NrdI2          nrdI* nrdI2 S. pyogenes did render nrdHEF <br /> successfully interact S. pneumoniae system. In con-          functional vivo. Moreover, vivo complementation by <br /> trast, growth 42&#176;C detected combined            nrdF*I*E* achieved genes simul- <br /> the S. pyogenes nrdHEF operon S. pneumoniae nrdI2            taneously present, indicating compulsory requirement <br /> gene, plating efficiency low (Table 2). These        nrdI* gene product. In similar way, S. pneumoniae <br /> results S. pyogenes nrdHEF operon function-         nrdHEF operon able complement coex- <br /> ality vivo, given supplemented functional   pressed nrdI gene. These results S. <br /> nrdI gene.                                                            pyogenes nrdI* functional vivo suggest S. pyogenes <br />                                                                       nrdI2 not. <br />                                                                          An nrdI gene class Ib-containing <br />                          DISCUSSION <br />                                                                       microorganism (http://rnrdb.molbio.su.se/), preliminary <br />    Genomes display multiple RNRs             assays laboratory <span id='am-401' about='obo:NCBITaxon_28901' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_590 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-402' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-403' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-404' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-405' property="oboInOwl:hasRelatedSynonym" content="Salmonella enterica ser. choleraesuis" datatype="xsd:string"></span><span id='am-406' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-407' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-408' property="oboInOwl:hasRelatedSynonym" content="Salmonella choleraesuis" datatype="xsd:string"></span><span id='am-409' property="rdfs:label" content="Salmonella enterica" datatype="xsd:string"></span><span id='am-410' property="oboInOwl:hasRelatedSynonym" content="Salmonella cholerae-suis" datatype="xsd:string"></span><span id='am-411' property="oboInOwl:hasRelatedSynonym" content="Bacillus cholerae-suis" datatype="xsd:string"></span><span id='am-412'  ></span><span id='am-413' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-414' property="oboInOwl:hasAlternativeId" content="NCBITaxon:591" datatype="xsd:string"></span>Salmonella enterica</span> serovar <br /> every group bacteria (41). The concurrence aerobic           Typhimurium class Ib RNR indicate gene required <br /> enzyme (class I) anaerobic (class III) facultative      vivo class Ib ribonucleotide reduction microor- <br /> microorganisms reflects metabolic need. In <span id='am-308' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-309' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-310' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>        ganism (I. Sala, unpublished results). The <span id='am-138' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-139' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-150' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-169'  ></span><span id='am-170' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span>Bacillus subtilis</span> nrdI <br /> combinations RNRs used envi-            gene shown essential (22, 36). Protein <br /> ronmental conditions (such classes I II, Ia Ib, II      comparison S. pyogenes NrdI* streptococcal <br /> and III), proposed enzymes          NrdI2 showed distant relatives (36% similarity). <br /> support normal cell growth support              Sequence comparison streptococcal NrdI2 pro- <br /> DNA repair growth recovery (4, 5, 8, 26, 37, 42). The             teins showed different groupings distin- <br /> genome S. pyogenes genomes contain            guished according small differences primary se- <br /> redundant RNR operons&#8212;here, different class Ib operons.           quences (Fig. 6A). The group comprised NrdI2 <br /> Moreover, genes             proteins S. pneumoniae, <span id='am-71' about='obo:NCBITaxon_1309' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1301 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Staphylococcus mutans" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1289" datatype="xsd:string"></span><span id='am-74'  ></span><span id='am-75' property="rdfs:label" content="Streptococcus mutans" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Streptococcus mutans</span>, Strep- <br /> operons (instead nrdHIEF genes classical           tococcus suis, NrdI-like protein their <br /> class Ib systems). The S. pyogenes nrdHEF cluster lacks nrdI       genomes, second group included NrdI2 proteins <br /> gene, nrdF*I*E* cluster lacks nrdH gene. A second          S. pyogenes, <span id='am-78' about='obo:NCBITaxon_1311' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1301 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Streptococcus difficile" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="Streptococcus agalactiae" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasAlternativeId" content="NCBITaxon:76757" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Streptococcus difficilis" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Streptoccocus de la mammite" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-85'  ></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasRelatedSynonym" content="Streptococcus agalactiae contagiosae" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="Streptococcus mastitidis" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="PMID:8995807" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="PMID:15774692" datatype="xsd:string"></span>Streptococcus agalactiae</span>, Streptococcus <br /> nrdI gene (nrdI2) S. pyogenes chro-         equi, posses nrdF*I*E* cluster. As shown Fig. <br /> mosome. Both Ib gene clusters transcribed simulta-         6A, clustering sequences separate groups is <br /> neously independent polycistronic operons. In addi-          mainly 9-bp (3 amino acids) indel approximately <br /> tion, Western blotting rabbit-raised nrdF antibodies          10 substituted residues differing groups. Com- <br /> demonstrated presence nrdF nrdF* products             parison B. subtilis NrdI streptococcal NrdI2 proteins <br /> within microorganism (data shown), suggesting           indicates differences affect residues <br /> both enzyme systems expressed simultaneously.                     inferred flavin mononucleotide (FMN) binding (see below), <br />    Only redundant class Ib RNRs S. pyogenes         regions NrdI2 fully conserved all <br /> is active vitro. The NrdF subunit S. pyogenes             streptococcal polypeptide chains. It remarkable the <br /> nrdHEF cluster classical class Ib protein dinuclear       streptococcal species bear nrdI2 gene the <br /> iron center stable tyrosyl radical (30). It catalytically   indel common redundant class Ib operon. <br /> competent assayed vitro presence corre-            Interestingly, nrdFIE gene cluster high levels of <br /> sponding NrdE subunit. Specific activities       similarity streptococcal nrdF*I*E* operon (87%, 57%, <br /> within range described class Ib systems M.       76% similarity NrdF*, NrdI*, NrdE*, respectively) <br /> tuberculosis (48) C. ammoniagenes (11). The nrdHEF-en-            Mycoplasma species. In addition, the <br /> coded enzyme displayed dATP insensitivity,            substitution iron binding residues described for <br /> common feature class Ib enzymes &#65533;?&#65533; subunit lacks        S. pyogenes NrdF* conserved Mycoplasma NrdFs <br /> 50 60 N-terminal residues class Ia enzymes consti-        (Fig. 6B). Mycoplasma organisms intracellular parasites <br /> tute ATP/dATP allosteric-activity site (29). Collectively         simplest prokaryotes capable self-replication <br /> these results nrdHEF behaves func-          (23). Although Mycoplasma organisms import deoxyribo- <br /> tional class Ib enzyme vitro.                                      nucleotides host, nrdFIE genes encode only <br />    The NrdF* subunit nrdF*I*E* cluster lacks           putative RNR. One possible explanation striking sim- <br /> of conserved chains normally ligate iron        ilarities nrdFIE operons Mycoplasma S. <br /> center. It low iron content lacks tyrosyl radical    pyogenes operon horizontally trans- <br /> well enzyme activity vitro mixtures NrdE*           ferred S. pyogenes compensate loss vivo <br /> subunit. Yet, NrdE* retain catalytically important       functionality streptococcal nrdHEF system, the <br /> residues. Components class Ib cluster cross-          lack functional NrdI2 protein. Further analyses needed <br /> react components cluster.                         clarify suggestion, <span id='am-303' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-304'  ></span><span id='am-305' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-306' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-307' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> usage current <br />    In vivo heterologous complementation highlights essen-         class Ib nrd genes S. pyogenes agreement that <br /> tiality nrdI. Our vivo results showed completely different    calculated rest S. pyogenes genome (2, 10, 38), <br /> picture. Both S. pyogenes gene clusters proved functional       phagic signatures mobile elements flanking <br /> in heterologous complementation assays                streptococcal nrdF*I*E* genes. Interestingly, results <br />  4856      ROCA ET AL.                                                                                                               J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 6. (A) Clustal W alignment deduced amino acid fragments nrdI gene. Ssan, Streptococcus sanguis; Spne, Streptococcus <br /> pneumoniae; Ssui, <span id='am-62' about='obo:NCBITaxon_1307' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1301 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-63' property="oboInOwl:hasDbXref" content="PMID:23245487" datatype="xsd:string"></span><span id='am-64'  ></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="PMID:5220563" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="PMID:9734065" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="PMID:9731300" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Streptococcus suis" datatype="xsd:string"></span>Streptococcus suis</span>; Smut, Streptococcus mutans; Spyo, Streptococcus pyogenes; Sequ, <span id='am-108' about='obo:NCBITaxon_1336' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1301 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_119603 obo:NCBITaxon_186826'><span id='am-109'  ></span><span id='am-110' property="rdfs:label" content="Streptococcus equi" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Streptococcus equi</span>; <span id='am-314' about='obo:NCBITaxon_441239' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_62784 obo:NCBITaxon_6994 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_70905 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_323649 obo:NCBITaxon_6993 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_33341 obo:NCBITaxon_50557 obo:NCBITaxon_6960 obo:NCBITaxon_6072'><span id='am-315' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-316' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-317'  ></span><span id='am-318' property="rdfs:label" content="Saga" datatype="xsd:string"></span>Saga</span>, Streptococcus <br /> agalactiae; Bsub, Bacillus subtilis. Shading sequence names indicates common clustering. Those residues unique group highlighted <br /> in black. Conserved residues streptococci highlighted light gray, FMN binding residues B. subtilis shown dark <br /> gray. (B) Clustal W alignment deduced amino acid fragments nrdF gene. Mpne, <span id='am-241' about='obo:NCBITaxon_2104' typeof='owl:Thing obo:NCBITaxon_2093 obo:NCBITaxon_2092 obo:NCBITaxon_1 obo:NCBITaxon_2085 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_544448 obo:NCBITaxon_31969'><span id='am-242' property="oboInOwl:hasDbXref" content="PMID:19871393" datatype="xsd:string"></span><span id='am-243'  ></span><span id='am-244' property="oboInOwl:hasRelatedSynonym" content="Filterable agent of primary atypical pneumonia" datatype="xsd:string"></span><span id='am-245' property="oboInOwl:hasRelatedSynonym" content="Schizoplasma pneumoniae" datatype="xsd:string"></span><span id='am-246' property="oboInOwl:hasDbXref" content="PMID:4867945" datatype="xsd:string"></span><span id='am-247' property="rdfs:label" content="Mycoplasma pneumoniae" datatype="xsd:string"></span><span id='am-248' property="oboInOwl:hasDbXref" content="GC_ID:4" datatype="xsd:string"></span><span id='am-249' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Mycoplasma pneumoniae</span>; Styp, Salmonella enterica <br /> serovar Typhimurium. Conserved residues highlighted light gray, conserved radical tyrosyl residue conserved iron binding <br /> residues highlighted dark gray. Substituted residues S. pyogenes, S. equi, S. agalactiae, <span id='am-250' about='obo:NCBITaxon_2107' typeof='owl:Thing obo:NCBITaxon_2093 obo:NCBITaxon_2092 obo:NCBITaxon_1 obo:NCBITaxon_2085 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_544448 obo:NCBITaxon_31969'><span id='am-251' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-252' property="oboInOwl:hasDbXref" content="PMID:16350067" datatype="xsd:string"></span><span id='am-253' property="rdfs:label" content="Mycoplasma pulmonis" datatype="xsd:string"></span><span id='am-254' property="oboInOwl:hasDbXref" content="PMID:5319406" datatype="xsd:string"></span><span id='am-255' property="oboInOwl:hasRelatedSynonym" content="Musculomyces histotropicus" datatype="xsd:string"></span><span id='am-256' property="oboInOwl:hasRelatedSynonym" content="Mycoplasma histotropicus" datatype="xsd:string"></span><span id='am-257' property="oboInOwl:hasRelatedSynonym" content="Murimyces pulmonis" datatype="xsd:string"></span><span id='am-258'  ></span><span id='am-259' property="oboInOwl:hasRelatedSynonym" content="Mycoplasma mergenhagen" datatype="xsd:string"></span><span id='am-260' property="oboInOwl:hasRelatedSynonym" content="Asterococcus pulmonis" datatype="xsd:string"></span><span id='am-261' property="oboInOwl:hasDbXref" content="PMID:14308364" datatype="xsd:string"></span><span id='am-262' property="oboInOwl:hasDbXref" content="GC_ID:4" datatype="xsd:string"></span>Mycoplasma pulmonis</span>, <span id='am-236' about='obo:NCBITaxon_2097' typeof='owl:Thing obo:NCBITaxon_2093 obo:NCBITaxon_2092 obo:NCBITaxon_1 obo:NCBITaxon_2085 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_544448 obo:NCBITaxon_31969'><span id='am-237' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-238' property="rdfs:label" content="Mycoplasma genitalium" datatype="xsd:string"></span><span id='am-239' property="oboInOwl:hasDbXref" content="GC_ID:4" datatype="xsd:string"></span><span id='am-240'  ></span>Mycoplasma genitalium</span>, and <br /> M. pneumoniae highlighted black. All sequences retrieved RNRdb http://rnrdb.molbio.su.se. <br />  <br />  <br />  <br /> also suggest nrdH dispensable nrdF*I*E* function.               role NrdI flavodoxin reduce iron cluster class <br /> This surprising nrdH gene lacking Myco-          Ib NrdF, allowing reactivation essential <br /> plasma B. subtilis, contain class Ib enzyme.         tyrosyl radical. The iron center <span id='am-128' about='obo:NCBITaxon_1392' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_86661 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-129' property="oboInOwl:hasDbXref" content="PMID:8123566" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasRelatedSynonym" content="Bacteridium anthracis" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasExactSynonym" content="anthrax bacterium" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasExactSynonym" content="anthrax" datatype="xsd:string"></span><span id='am-135'  ></span><span id='am-136' property="oboInOwl:hasRelatedSynonym" content="Bacillus cereus var. anthracis" datatype="xsd:string"></span><span id='am-137' property="rdfs:label" content="Bacillus anthracis" datatype="xsd:string"></span>Bacillus anthracis</span> NrdF (44) <br /> In E. coli, shown NrdEF reduced                  preliminary experiments efficiently reduced by <br /> glutaredoxin-1 (Grx1) absence NrdH (13), B.                incubation B. anthracis NrdI (M. Sahlin, E. Torrents, and <br /> subtilis, essential thioredoxin-1 (TrxA) proposed           B.-M. Sjo &#168;berg, unpublished data). Class Ib operons thus <br /> the electron donor class Ib RNR (15). We grx             carry specific electron donors, NrdH, provides re- <br /> homologues S. pyogenes Mycoplasma, contain              ducing power &#65533;?&#65533; subunit, NrdI, supplies elec- <br /> trx homologue, supply role nrdH.                         trons &#65533;?&#65533; subunit. <br />    NrdI member flavodoxin <span id='am-287' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-288' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-289' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span>. The three-di- <br />                                                                                Jordan et al. (19) showed activity S. enterica <br /> mensional structure NrdI protein B. subtilis is <br />                                                                             serovar Typhimurium NrdEF enzyme slightly stimu- <br /> available (Protein Data Bank accession no. 1RLJ) pro- <br />                                                                             lated addition partially purified recombinant NrdI <br /> vides evidence NrdI contains flavodoxin-like fold, and <br /> thus, NrdIs classified flavodoxin NrdI family (Pfam              protein, classical NrdEF enzymes analyzed date are <br /> accession no. PF07972). Flavodoxins small electron trans-               active vitro addition NrdI protein (17, 20, <br /> fer proteins utilize noncovalently bound FMN                  21, 44, 48). Likewise, NrdI essential vitro <br /> redox-active component (35). In E. coli, flavodoxin                     activity S. pyogenes NrdEF enzyme. In sharp contrast to <br /> NADPH:flavodoxin oxidoreductase essential class III                 this, S. pyogenes NrdE*F* mixture vitro enzyme <br /> RNR activation (27). Likewise, flavin reductase Fre (12)                activity, likely relating lack iron <br /> and YfaE ferredoxin (47) reactivate non-radical-                  binding chains NrdF*. The obvious vivo activity of <br /> containing met-&#65533;?&#65533;2 E. coli class Ia RNR. We suggest              S. pyogenes nrdF*I*E* operon suggests NrdI* play <br />  VOL. 190, 2008                                                                              CLASS Ib RNR REDUNDANCY IN S. PYOGENES                                    4857 <br />  <br />  <br /> a direct role ribonucleotide reduction activat-                             Bellido, G. Zhao, C. A. Zook, R. H. Baltz, S. R. Jaskunas, P. R. Rosteck, Jr., <br />                                                                                             P. L. Skatrud, J. I. Glass. 2001. Genome bacterium Streptococcus <br /> ing reduced met-&#65533;?&#65533;2 protein.                                                               pneumoniae strain R6. J. Bacteriol. 183:5709&#8211;5717. <br />                                                                                       17.   Huque, Y., F. Fieschi, E. Torrents, I. Gibert, R. Eliasson, P. Reichard, M. <br />                           ACKNOWLEDGMENTS                                                   Sahlin, B.-M. Sjo &#168;berg. 2000. The active form R2F protein class <br />                                                                                             Ib ribonucleotide reductase <span id='am-182' about='obo:NCBITaxon_1716' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_1653'><span id='am-183' property="oboInOwl:hasDbXref" content="PMID:19661509" datatype="xsd:string"></span><span id='am-184' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-185'  ></span><span id='am-186' property="oboInOwl:hasDbXref" content="PMID:7547284" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-188' property="rdfs:label" content="Corynebacterium" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasDbXref" content="PMID:10425754" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasDbXref" content="PMID:7547293" datatype="xsd:string"></span><span id='am-191' property="oboInOwl:hasDbXref" content="PMID:7547291" datatype="xsd:string"></span>Corynebacterium</span> ammoniagenes difer- <br />   This work supported grants BFU2004-03383 (Ministerio                            ric protein. J. Biol. Chem. 275:25365&#8211;25371. <br /> Educacio&#180;n y Ciencia, Spain), 2005SGR-00956 (Generalitat Catalu-                   18.   Jordan, A., E. Aragall, I. Gibert, J. Barbe&#180;. 1996. Promoter identification <br /> nya, Spain), 621-2004-2864 (Swedish Science Research Council).                          expression analysis Salmonella typhimurium Escherichia coli <br /> I.G. I.R. recipients grant Fundacio <br />                                                           &#180;n Maria                          nrdEF operons encoding class I ribonucleotide reductases present <br /> Francisca Roviralta. E.T. supported Ramo&#180;n y Cajal pro-                        bacteria. Mol. Microbiol. 19:777&#8211;790. <br />                                                                                       19.   Jordan, A., F. &#197;slund, E. Pontis, P. Reichard, A. Holmgren. 1997. <br /> gram Jeansson Foundations. <br />                                                                                             Characterization Escherichia coli NrdH. A glutaredoxin-like protein with <br />                                                                                             thioredoxin-like activity profile. J. Biol. Chem. 272:18044&#8211;18050. <br />                                  REFERENCES                                           20.   Jordan, A., E. Pontis, F. &#197;slund, U. Hellman, I. Gibert, P. Reichard. <br />  1. Andersson, M. E., M. Ho    &#168;gbom, A. Rinaldo-Matthis, W. Blodig, Y. Liang,              1996. The ribonucleotide reductase <span id='am-113' about='obo:NCBITaxon_1358' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1357 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-114'  ></span><span id='am-115' property="rdfs:label" content="Lactococcus lactis" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasDbXref" content="PMID:11594628" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasRelatedSynonym" content="Bacterium lacti" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="Lactococcus lactis (Lohnis) Schleifer et al. 1986" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="Streptococcus lactis" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Lactococcus lactis</span>. Character- <br />     B. O. Persson, B.-M. Sjo  &#168;berg, X. D. Su, P. Nordlund. 2004. Structural            ization NrdEF enzyme new electron transport protein. J. Biol. <br />     mutational studies carboxylate cluster iron-free ribonucleotide           Chem. 271:8779&#8211;8785. <br />     reductase R2. Biochemistry 43:7966&#8211;7972.                                          21.   Jordan, A., E. Pontis, M. <span id='am-1' about='obo:NCBITaxon_12956' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_36668 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_143999 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7434 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_34695 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_34725 obo:NCBITaxon_50557'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="leaf cutting ants" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="Atta" datatype="xsd:string"></span>Atta</span>, M. Krook, I. Gibert, J. Barbe&#180;, P. <br />  2. Beres, S. B., G. L. Sylva, K. D. Barbian, B. Lei, J. S. Hoff, N. D. Mammarella,         Reichard. 1994. A second class I ribonucleotide reductase Enterobacteri- <br />     M. Y. Liu, J. C. Smoot, S. F. Porcella, L. D. Parkins, D. S. Campbell, T. M.            aceae: characterization Salmonella typhimurium enzyme. Proc. Natl. <br />     Smith, J. K. McCormick, D. Y. Leung, P. M. Schlievert, J. M. Musser.                Acad. Sci. USA 91:12892&#8211;12896. <br />     2002. Genome sequence serotype M3 strain group A Streptococcus:           22.   Kobayashi, K., S. D. Ehrlich, A. Albertini, G. Amati, K. K. Andersen, M. <br />     phage-encoded toxins, high-virulence phenotype, clone emergence.                Arnaud, K. Asai, S. Ashikaga, S. Aymerich, P. Bessieres, F. Boland, S. C. <br />     Proc. Natl. Acad. Sci. USA 99:10078&#8211;10083.                                              Brignell, S. Bron, K. Bunai, J. Chapuis, L. C. Christiansen, A. Danchin, M. <br />  3. Birgander, P. L., S. Bug, A. Kasrayan, S.-L. Dahlroth, M. Westman, E.                   Debarbouille, E. Dervyn, E. Deuerling, K. Devine, S. K. Devine, O. Dreesen, <br />     Gordon, B.-M. Sjo   &#168;berg. 2005. Nucleotide-dependent formation cat-             J. Errington, S. Fillinger, S. J. Foster, Y. Fujita, A. Galizzi, R. Gardan, C. <br />     alytically competent dimers engineered monomeric ribonucleotide re-                Eschevins, T. Fukushima, K. Haga, C. R. Harwood, M. Hecker, D. Hosoya, <br />     ductase protein R1. J. Biol. Chem. 280:14997&#8211;15003.                                     M. F. Hullo, H. Kakeshita, D. Karamata, Y. Kasahara, F. Kawamura, K. <br />  4. Borovok, I., B. Gorovitz, M. Yanku, R. Schreiber, B. Gust, K. Chater, Y.                Koga, P. Koski, R. Kuwana, D. Imamura, M. Ishimaru, S. Ishikawa, I. Ishio, <br />     Aharonowitz, G. Cohen. 2004. Alternative oxygen-dependent oxy-                  D. Le Coq, A. Masson, C. Mauel, R. Meima, R. P. Mellado, A. Moir, S. <br />     gen-independent ribonucleotide reductases <span id='am-12' about='obo:NCBITaxon_1883' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_2062 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_85011'><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1973" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="PMID:15950131" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="PMID:1371059" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="NCBITaxon:31945" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="PMID:9731302" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="PMID:8590678" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:65506" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Streptomyces" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="PMID:9336904" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="PMID:4886644" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="PMID:8934906" datatype="xsd:string"></span><span id='am-25'  ></span><span id='am-26' property="oboInOwl:hasDbXref" content="PMID:10843062" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="PMID:16560709" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Chainia" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="PMID:11411701" datatype="xsd:string"></span>Streptomyces</span>: cross-regulation             Moriya, E. Nagakawa, H. Nanamiya, S. Nakai, P. Nygaard, M. Ogura, T. <br />     physiological role response oxygen limitation. Mol. Microbiol.                Ohanan, M. O&#8217;Reilly, M. O&#8217;Rourke, Z. Pragai, H. M. Pooley, G. Rapoport, <br />     54:1022&#8211;1035.                                                                           J. P. Rawlins, L. A. Rivas, C. Rivolta, A. Sadaie, Y. Sadaie, M. Sarvas, T. <br />  5. Borovok, I., R. Kreisberg-Zakarin, M. Yanko, R. Schreiber, M. Myslovati, F.             Sato, H. H. Saxild, E. Scanlan, W. Schumann, J. F. Seegers, J. Sekiguchi, A. <br />     &#197;slund, A. Holmgren, G. Cohen, Y. Aharonowitz. 2002. Streptomyces                   Sekowska, S. J. Seror, M. Simon, P. Stragier, R. Studer, H. Takamatsu, T. <br />     spp. contain class Ia class II ribonucleotide reductases: expression anal-          Tanaka, M. Takeuchi, H. B. Thomaides, V. Vagner, J. M. van Dijl, K. <br />     ysis genes vegetative growth. Microbiology 148:391&#8211;404.                       Watabe, A. Wipat, H. Yamamoto, M. Yamamoto, Y. Yamamoto, K. Yamane, <br />  6. Bradford, M. M. 1976. A rapid sensitive method quantitation              K. Yata, K. Yoshida, H. Yoshikawa, U. Zuber, N. Ogasawara. 2003. <br />     microgram quantities protein utilizing principle protein-dye bind-            Essential Bacillus subtilis genes. Proc. Natl. Acad. Sci. USA 100:4678&#8211;4683. <br />     ing. Anal. Biochem. 72:248&#8211;254.                                                   23.   Koonin, E. V. 2000. How genes make cell: minimal-gene-set <br />  7. Climent, I., B.-M. Sjo&#168;berg, C. Y. Huang. 1992. Site-directed mutagenesis           concept. Annu. Rev. Genomics Hum. Genet. 1:99&#8211;116. <br />     deletion carboxyl terminus Escherichia coli ribonucleotide          24.   Liu, A., S. Po <br />                                                                                                          &#168;tsch, A. Davydov, A. L. Barra, H. Rubin, A. Gra  &#168;slund. 1998. <br />     reductase protein R2. Effects catalytic activity subunit interaction.            The tyrosyl free radical recombinant ribonucleotide reductase My- <br />     Biochemistry 31:4801&#8211;4807.                                                              cobacterium tuberculosis located rigid hydrophobic pocket. Biochem- <br />  8. Dawes, S. S., D. F. Warner, L. Tsenova, J. Timm, J. D. McKinney, G. Kaplan,             istry 37:16369&#8211;16377. <br />     H. Rubin, V. Mizrahi. 2003. Ribonucleotide reduction <span id='am-192' about='obo:NCBITaxon_1763' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1762 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003'><span id='am-193' property="oboInOwl:hasDbXref" content="PMID:2275850" datatype="xsd:string"></span><span id='am-194' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasDbXref" content="PMID:7547304" datatype="xsd:string"></span><span id='am-196' property="oboInOwl:hasDbXref" content="PMID:8863452" datatype="xsd:string"></span><span id='am-197' property="oboInOwl:hasDbXref" content="PMID:16014496" datatype="xsd:string"></span><span id='am-198' property="rdfs:label" content="Mycobacterium" datatype="xsd:string"></span><span id='am-199' property="oboInOwl:hasDbXref" content="PMID:1380284" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:hasDbXref" content="PMID:1581193" datatype="xsd:string"></span><span id='am-201' property="oboInOwl:hasDbXref" content="PMID:7547284" datatype="xsd:string"></span><span id='am-202' property="oboInOwl:hasDbXref" content="PMID:1883713" datatype="xsd:string"></span><span id='am-203' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium Lehmann and Neumann 1896" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-205' property="oboInOwl:hasDbXref" content="PMID:7907223" datatype="xsd:string"></span><span id='am-206'  ></span>Mycobacterium</span>         25.   McLaughlin, R. E., J. J. Ferretti. 1995. Electrotransformation strep- <br />     tuberculosis: function expression genes encoding class Ib class II           tococci. Methods Mol. Biol. 47:185&#8211;193. <br />     ribonucleotide reductases. Infect. Immun. 71:6124&#8211;6131.                           26.   Monje-Casas, F., J. Jurado, M. J. Prieto-A  &#180; lamo, A. Holmgren, C. Pueyo. <br />  9. Federle, M. J., K. S. McIver, J. R. Scott. 1999. A response regulator          2001. Expression analysis nrdHIEF operon Escherichia coli. <br />     represses transcription virulence operons group A strepto-            Conditions trigger transcript level vivo. J. Biol. Chem. 276:18031&#8211; <br />     <span id='am-420' about='obo:NCBITaxon_538669' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_33381 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_33380 obo:NCBITaxon_6072 obo:NCBITaxon_7524 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_33373 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_30189 obo:NCBITaxon_33342 obo:NCBITaxon_50557'><span id='am-421' property="rdfs:label" content="Coccus" datatype="xsd:string"></span><span id='am-422' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-423' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-424'  ></span>coccus</span>. J. Bacteriol. 181:3649&#8211;3657.                                                    18037. <br /> 10. Ferretti, J. J., W. M. McShan, D. Ajdic, D. J. Savic, G. Savic, K. Lyon, C. <br />                                                                                       27.   Mulliez, E., D. Padovani, M. Atta, C. Alcouffe, M. Fontecave. 2001. <br />     Primeaux, S. Sezate, A. N. Suvorov, S. Kenton, H. S. Lai, S. P. Lin, Y. Qian, <br />                                                                                             Activation class III ribonucleotide reductase flavodoxin: protein <br />     H. G. Jia, F. Z. Najar, Q. Ren, H. Zhu, L. Song, J. White, X. Yuan, S. W. <br />                                                                                             radical-driven electron transfer iron-sulfur center. Biochemistry 40: <br />     Clifton, B. A. Roe, R. McLaughlin. 2001. Complete genome sequence of <br />                                                                                             3730&#8211;3736. <br />     M1 strain Streptococcus pyogenes. Proc. Natl. Acad. Sci. USA 98:4658&#8211; <br />                                                                                       28.   Nicholas, K. B., H. B. J. Nicholas. 1997. GeneDoc: tool editing and <br />     4663. <br />                                                                                             annotating multiple sequence alignments v. 2.6.001. K. B. Nicholas and <br /> 11. Fieschi, F., E. Torrents, L. Toulokhonova, A. Jordan, U. Hellman, J. Barbe&#180;, <br />                                                                                             H. B. J. Nicholas (http://www.nrbsc.org/gfx/genedoc/index.html). <br />     I. Gibert, M. Karlsson, B.-M. Sjo  &#168;berg. 1998. The manganese-containing <br />     ribonucleotide reductase Corynebacterium ammoniagenes class Ib            29.   Nordlund, P., P. Reichard. 2006. Ribonucleotide reductases. Annu. Rev. <br />     enzyme. J. Biol. Chem. 273:4329&#8211;4337.                                                   Biochem. 75:681&#8211;706. <br /> 12. Fontecave, M., R. Eliasson, P. Reichard. 1989. Enzymatic regulation        30.   Petersson, L., A. Gra &#168;slund, A. Ehrenberg, B.-M. Sjo   &#168;berg, P. Reichard. <br />     radical content small subunit Escherichia coli ribonucleotide             1980. The iron center ribonucleotide reductase Escherichia coli. <br />     reductase involving reduction redox centers. J. Biol. Chem. 264:9164&#8211;            J. Biol. Chem. 255:6706&#8211;6712. <br />     9170.                                                                             31.   Platz, A., B.-M. Sjo  &#168;berg. 1980. Construction characterization of <br /> 13. Gon, S., M. J. Faulkner, J. Beckwith. 2006. In vivo requirement                 <span id='am-430' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-431' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-432'  ></span><span id='am-433' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-434' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>hybrid</span> plasmids containing Escherichia coli nrd region. J. Bacteriol. <br />     glutaredoxins thioredoxins reduction ribonucleotide reduc-            143:561&#8211;568. <br />     tases Escherichia coli. Antioxid. Redox Signal. 8:735&#8211;742.                     32.   Sahlin, M., L. Petersson, A. Gra &#168;slund, A. Ehrenberg, B.-M. Sjo  &#168;berg, L. <br /> 14. Guzman, L. M., D. Belin, M. J. Carson, J. Beckwith. 1995. Tight regu-               Thelander. 1987. Magnetic interaction tyrosyl free radical and <br />     lation, modulation, high-level expression vectors containing ar-             antiferromagnetically coupled iron center ribonucleotide reductase. <br />     abinose PBAD promoter. J. Bacteriol. 177:4121&#8211;4130.                                     Biochemistry 26:5541&#8211;5548. <br /> 15. Ha &#168;rtig, E., A. Hartmann, M. Scha  &#168;tzle, A. M. Albertini, D. Jahn. 2006.    33.   Sahlin, M., B.-M. Sjo &#168;berg, G. Backes, T. Loehr, J. Sanders-Loehr. 1990. <br />     The Bacillus subtilis nrdEF genes, encoding class Ib ribonucleotide reduc-            Activation iron-containing B2 protein ribonucleotide reductase by <br />     tase, essential aerobic anaerobic growth. Appl. Environ. Micro-             hydrogen peroxide. Biochem. Biophys. Res. Commun. 167:813&#8211;818. <br />     biol. 72:5260&#8211;5265.                                                               34.   Sambrook, J., E. F. Fritsch, T. Maniatis. 1989. Molecular cloning: a <br /> 16. Hoskins, J., W. E. Alborn, Jr., J. Arnold, L. C. Blaszczak, S. Burgett, B. S.           laboratory manual, 2nd ed. Cold Spring Harbor Laboratory Press, Plain- <br />     DeHoff, S. T. Estrem, L. Fritz, D. J. Fu, W. Fuller, C. Geringer, R. Gilmour,           view, NY. <br />     J. S. Glass, H. Khoja, A. R. Kraft, R. E. Lagace, D. J. LeBlanc, L. N. Lee,       35.   Sancho, J. 2006. Flavodoxins: sequence, folding, binding, function be- <br />     E. J. Lefkowitz, J. Lu, P. Matsushima, S. M. McAhren, M. McHenney, K.                   yond. Cell. Mol. Life Sci. 63:855&#8211;864. <br />     McLeaster, C. W. Mundy, T. I. Nicas, F. H. Norris, M. O&#8217;Gara, R. B. Peery,        36.   Scotti, C., A. Valbuzzi, M. Perego, A. Galizzi, A. M. Albertini. 1996. The <br />     G. T. Robertson, P. Rockey, P. M. Sun, M. E. Winkler, Y. Yang, M. Young-                Bacillus subtilis genes ribonucleotide reductase similar genes <br />  4858         ROCA ET AL.                                                                                                                                   J. BACTERIOL. <br />  <br />       second class I NrdE/NrdF enzymes <span id='am-348' about='obo:NCBITaxon_543' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236'><span id='am-349' property="rdfs:label" content="Enterobacteriaceae" datatype="xsd:string"></span><span id='am-350' property="oboInOwl:hasRelatedSynonym" content="Enterobacteriaceae Rahn 1937" datatype="xsd:string"></span><span id='am-351' property="oboInOwl:hasRelatedSynonym" content="Enterobacteraceae" datatype="xsd:string"></span><span id='am-352' property="oboInOwl:hasRelatedSynonym" content="Enterobacteriaceae (ex Rahn 1937) Ewing et al. 1980, fam. nov., nom. rev." datatype="xsd:string"></span><span id='am-353' property="oboInOwl:hasRelatedSynonym" content="gamma-3 proteobacteria" datatype="xsd:string"></span><span id='am-354' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-355' property="oboInOwl:hasDbXref" content="PMID:10555334" datatype="xsd:string"></span><span id='am-356' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-357' property="oboInOwl:hasDbXref" content="PMID:10555323" datatype="xsd:string"></span><span id='am-358'  ></span><span id='am-359' property="oboInOwl:hasDbXref" content="PMID:16166704" datatype="xsd:string"></span><span id='am-360' property="oboInOwl:hasRelatedSynonym" content="Enterobacteraceae (ex Lapage 1979) Lapage 1982, fam. nov., nom. rev." datatype="xsd:string"></span>Enterobacteriaceae</span>. Microbi-              sion transcriptional analysis aerobic enzymes. J. Biol. Chem. <br />       ology 142:2995&#8211;3004.                                                                  280:16571&#8211;16578. <br /> 37.   Smalley, D., E. R. Rocha, C. J. Smith. 2002. Aerobic-type ribonucleotide    43.   Torrents, E., I. Roca, I. Gibert. 2003. Corynebacterium ammoniagenes <br />       reductase anaerobe <span id='am-319' about='obo:NCBITaxon_817' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_200643 obo:NCBITaxon_815 obo:NCBITaxon_816 obo:NCBITaxon_171549 obo:NCBITaxon_976 obo:NCBITaxon_68336'><span id='am-320' property="oboInOwl:hasRelatedSynonym" content="Ristella uncata" datatype="xsd:string"></span><span id='am-321' property="rdfs:label" content="Bacteroides fragilis" datatype="xsd:string"></span><span id='am-322' property="oboInOwl:hasRelatedSynonym" content="Fusiformis fragilis" datatype="xsd:string"></span><span id='am-323' property="oboInOwl:hasRelatedSynonym" content="Ristella incommunis" datatype="xsd:string"></span><span id='am-324' property="oboInOwl:hasRelatedSynonym" content="Sphaerophorus inaequalis" datatype="xsd:string"></span><span id='am-325' property="oboInOwl:hasRelatedSynonym" content="Sphaerophorus intermedius" datatype="xsd:string"></span><span id='am-326' property="oboInOwl:hasRelatedSynonym" content="Pseudobacterium inaequalis" datatype="xsd:string"></span><span id='am-327' property="oboInOwl:hasRelatedSynonym" content="Bacteroides uncatus" datatype="xsd:string"></span><span id='am-328' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-329' property="oboInOwl:hasRelatedSynonym" content="Pseudobacterium fragilis" datatype="xsd:string"></span><span id='am-330' property="oboInOwl:hasDbXref" content="PMID:16559622" datatype="xsd:string"></span><span id='am-331' property="oboInOwl:hasRelatedSynonym" content="Bacteroides incommunis" datatype="xsd:string"></span><span id='am-332' property="oboInOwl:hasRelatedSynonym" content="Ristella fragilis" datatype="xsd:string"></span><span id='am-333' property="oboInOwl:hasRelatedSynonym" content="Pseudobacterium incommunis" datatype="xsd:string"></span><span id='am-334' property="oboInOwl:hasRelatedSynonym" content="Bacillus fragilis" datatype="xsd:string"></span><span id='am-335' property="oboInOwl:hasAlternativeId" content="NCBITaxon:33929" datatype="xsd:string"></span><span id='am-336' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-337' property="oboInOwl:hasRelatedSynonym" content="Bacteroides fragili" datatype="xsd:string"></span><span id='am-338' property="oboInOwl:hasRelatedSynonym" content="Pseudobacterium uncatum" datatype="xsd:string"></span><span id='am-339' property="oboInOwl:hasRelatedSynonym" content="Bacteroides inaequalis" datatype="xsd:string"></span><span id='am-340'  ></span>Bacteroides fragilis</span>. J. Bacteriol. 184:895&#8211;903.            class Ib ribonucleotide reductase: transcriptional regulation atypical <br /> 38.   Smoot, J. C., K. D. Barbian, J. J. Van Gompel, L. M. Smoot, M. S. Chaussee,           genomic organization nrd cluster. Microbiology 149:1011&#8211;1020. <br />       G. L. Sylva, D. E. Sturdevant, S. M. Ricklefs, S. F. Porcella, L. D. Parkins,   44.   Torrents, E., M. Sahlin, D. Biglino, A. Gra   &#168;slund, B.-M. Sjo&#168;berg. 2005. <br />       S. B. Beres, D. S. Campbell, T. M. Smith, Q. Zhang, V. Kapur, J. A. Daly,             Efficient growth inhibition Bacillus anthracis knocking ribonu- <br />       L. G. Veasy, J. M. Musser. 2002. Genome sequence comparative                  cleotide reductase tyrosyl radical. Proc. Natl. Acad. Sci. USA 102:17946&#8211; <br />       microarray analysis serotype M18 group A Streptococcus strains associated          17951. <br />       acute rheumatic fever outbreaks. Proc. Natl. Acad. Sci. USA 99:4668&#8211;       45.   Voegtli, W. C., M. Sommerhalter, L. Saleh, J. Baldwin, J. M. Bollinger, Jr., and <br />       4673.                                                                                 A. C. Rosenzweig. 200 <br /> </body></html>