<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Cell Division                                                                                                                            BioMed Central <br />  <br />  <br />  <br /> Research                                                                                                                               Open Access <br /> Genetic analysis spindle checkpoint genes san-1, mdf-2, bub-3 <br /> and CENP-F homologues hcp-1 hcp-2 <span id='am-56' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-59'  ></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Caenorhabditis</span> <br /> elegans <br /> Vinita A Hajeri1, Anil M Stewart1, Landon L Moore2 Pamela A <span id='am-110' about='obo:NCBITaxon_458145' typeof='owl:Thing obo:NCBITaxon_243881 obo:NCBITaxon_243882 obo:NCBITaxon_94015 obo:NCBITaxon_33208 obo:NCBITaxon_94017 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_94020 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_153386 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-111'  ></span><span id='am-112' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="Padilla" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Padilla</span>*1 <br />  <br /> Address: 1Department Biological Sciences, University North Texas, Denton, TX, USA 2Department Genetics Genomics, Boston <br /> University School Medicine, Boston, MA, USA <br /> Email: Vinita A Hajeri - vinita_hajeri@unt.edu; Anil M Stewart - nms@gmail.com; Landon L Moore - lmoore@bu.edu; <br /> Pamela A Padilla* - ppadilla@unt.edu <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 4 February 2008                                                  Received: 8 November 2007 <br />                                                                             Accepted: 4 February 2008 <br /> Cell Division 2008, 3:6   doi:10.1186/1747-1028-3-6 <br /> <span id='am-69' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-73'  ></span>This</span> article available from: http://www.celldiv.com/content/3/1/6 <br /> &#169; 2008 Hajeri et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                    Abstract <br />                    Background: The spindle checkpoint delays onset anaphase sister chromatids are <br />                    aligned properly metaphase plate. To investigate role san-1, MAD3 homologue, has <br />                    <span id='am-49' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-50' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54'  ></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span>Caenorhabditis elegans</span> embryos used RNA interference (RNAi) identify genes synthetic <br />                    lethal viable san-1(ok1580) deletion mutant. <br />                    Results: The san-1(ok1580) animal low penetrating phenotypes including increased <br />                    incidence males, larvae arrest, slow growth, protruding vulva, defects vulva <br />                    morphogenesis. We viability san-1(ok1580) embryos significantly reduced when <br />                    HCP-1 (CENP-F homologue), MDF-1 (MAD-1 homologue), MDF-2 (MAD-2 homologue) BUB- <br />                    3 (predicted BUB-3 homologue) reduced RNAi. Interestingly, viability san-1(ok1580) <br />                    embryos significantly reduced paralog HCP-1, HCP-2, reduced. The phenotype <br />                    san-1(ok1580);hcp-1(RNAi) embryos includes embryonic larval lethality, abnormal organ <br />                    development, increase abnormal chromosome segregation (aberrant mitotic nuclei, <br />                    anaphase bridging). Several san-1(ok1580);hcp-1(RNAi) animals displayed abnormal <br />                    kinetochore (detected MPM-2) microtubule structure. The survival mdf-2(RNAi);hcp- <br />                    1(RNAi) embryos bub-3(RNAi);hcp-1(RNAi) embryos compromised. Finally, we <br />                    san-1(ok1580) bub-3(RNAi), hcp-1(RNAi) embryos, sensitive <span id='am-1' about='obo:NCBITaxon_1207117' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_7055 obo:NCBITaxon_7056 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_75546 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_41086 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-2'  ></span><span id='am-3' property="rdfs:label" content="Anoxia" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>anoxia</span>, <br />                    suggesting like SAN-1, BUB-3 functional role spindle checkpoint protein. <br />                    Conclusion: Together, <span id='am-115' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-116' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-117'  ></span><span id='am-118' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-119' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> suggest C. elegans embryo, HCP-1 interacts a <br />                    subset spindle checkpoint pathway. Furthermore, fact san-1(ok1580);hcp-1(RNAi) <br />                    animals severe viability defect san-1(ok1580);hcp-2(RNAi) san- <br />                    1(ok1580);hcp-2(ok1757) animals viability defect severe suggesting hcp-1 and <br />                    hcp-2 completely redundant. <br />  <br />  <br />  <br />  <br />                                                                                                                                        Page 1 18 <br />                                                                                                                  (page number citation purposes) <br />  Cell Division 2008, 3:6                                                                 http://www.celldiv.com/content/3/1/6 <br />  <br />  <br />  <br />  <br /> Background                                                      san-1/mdf-3 alleles, providing evidence met- <br /> The spindle checkpoint genes regulate chromosome segre-         aphase anaphase transition meiosis regulated by <br /> gation mitosis mutated lead variety        spindle checkpoint [19]. The spindle checkpoint is <br /> of human health concerns including <span id='am-44' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-45'  ></span><span id='am-46' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cancer</span> [1,2]. Thus,       thought function C. elegans embryonic blast- <br /> is determine genetic interactions spin-     omeres [20]. Briefly, mdf-2 san-1 (MAD3 <span id='am-81' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-82' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-85'  ></span>homo</span>- <br /> dle checkpoint genes. The spindle checkpoint genes, orig-       logue) genes required response microtubule- <br /> inally identified yeast, include Mad1, Mad2, BubR1/          depolymerizing environments anoxia-induced <br /> Mad3, Bub1, Bub2, Bub3, Mps1 [3]. With excep-           suspended animation nocodazole [21,22]. Finally, <br /> tion Bub2, genes conserved          MDF-2 SAN-1 proteins localize kineto- <br /> yeast vertebrates. In vertebrate cells addi-      chore, consistent localization spindle <br /> tional gene products involved spindle checkpoint activ-      checkpoint proteins systems [18,21]. Together, <br /> ity indicating mechanistic differences      studies indicate spindle checkpoint gene <br /> in metazoan <span id='am-30' about='obo:NCBITaxon_4751' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-31'  ></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="fungi" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="fungi" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Fungi" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="PMID:11062127" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="PMID:12684019" datatype="xsd:string"></span>fungi</span> spindle checkpoint activity. For          products function developing organisms are <br /> example, CENP-E, kinesin motor protein binds             required response survival environments that <br /> BubR1, required checkpoint establishment and,            alter microtubule structure. <br /> ZW10 ROD required checkpoint signaling in <br /> vertebrates cells [4].                                          In C. elegans embryos HCP-1, CENP-F like protein, is <br />                                                                 localized kinetochore mitosis; hcp- <br /> The spindle-checkpoint signaling pathway under-         1(RNAi) animals obvious phenotype <br /> stood discussed reviews [1,5-9].           [23]. HCP-1 HCP-2 share 42% identity (54% similar- <br /> Briefly, spindle checkpoint gene products function       ity) another, HCP-1 29% identical (45% <br /> the kinetochore inhibit onset anaphase          similar) CENP-F repeats [23,24]. However not <br /> chromatids properly attached spindle microtubules.       clear HCP-1 HCP-2 orthologues CENP- <br /> The spindle checkpoint signaled unattached kineto-        F HCP-1 HCP-2 completely redundant. Co- <br /> chores reduction spindle microtubule tension [10-       depletion HCP-1 HCP-2 RNA interference <br /> 12]. In cell culture Mad1 Mad2 localize unattached       (RNAi) results abnormal chromosome segregation and <br /> kinetochores attached kinetochores lack         embryonic lethality indicating functional role chro- <br /> tension. Bub1 BubR1/Mad3 localize kinetochores           mosome segregation [23,25]. The HCP-1/2 proteins func- <br /> that lack tension microtubule attachment. The         tion kinetochore assembly. Localization HCP-1/2 <br /> downstream target spindle checkpoint multi-       requires kinetochore proteins HCP-3 (CENP-A), HCP- <br /> protein E3 ubiquitin ligase, anaphase-promoting com-            4 (CENP-C), KNL-1 BUB-1 [23,25,26]; HCP-1 <br /> plex/cyclosome (APC/C) [13]. A complex consisting            HCP-2 required proper localization the <br /> Mad2, Cdc20, Mad3/BubR1 Bub3 inhibitor             microtubule kinetochore associated protein CLASP/ <br /> APC/C [14-16]. It thought Mad-2 BubR1/Mad3          CLS-2 [27]. Additionally, HCP-1 HCP-2 required <br /> act synergistically inhibit APC/C activity. Inactivation     spindle checkpoint function, clear <br /> of spindle checkpoint releases APC/C cyclin B,          requirement direct indirect [20]. <br /> which required onset anaphase [17]. The <br /> APC/C ubiquinates proteins, securin, targets        In report investigate functional role of <br /> them protein degradation. The destruction securin        spindle checkpoint gene san-1 using RNAi iden- <br /> leads release separase destruction         tify genes synthetic lethal san-1(ok1580). The <br /> cohesin resulting disruption sister chromatid cohe-     viability san-1(ok1580) embryos significantly <br /> sion.                                                           reduced HCP-1, MDF-2 (MAD-2 homologue) or <br />                                                                 BUB-3 (gene Y54G9A.6, predicted encode BUB-3 <br /> Much work elucidate mechanistic function       homologue) reduced RNAi. The viability mdf- <br /> the spindle checkpoint proteins conducted           2(RNAi);hcp-1(RNAi) embryos bub-3(RNAi);hcp- <br /> yeast vertebrate cell culture. However, increas-   1(RNAi) embryos compromised suggesting HCP- <br /> ing evidence spindle checkpoint required       1 interacts subset (san-1 mdf-2) spindle <br /> for specific functions Caenorhabditis elegans Dro-       checkpoint components. Phenotype analysis san- <br /> sophila melanogastor. For example, C. elegans genes      1(ok1580);hcp-1(RNAi) embryos supports <span id='am-39' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Idea" datatype="xsd:string"></span>idea</span> that <br /> mdf-1 mdf-2 (MAD1 MAD2 homologues respec-               HCP-1 functions proper alignment chromo- <br /> tively) role development germline function        somes chromosome segregation func- <br /> [18]. Specifically, embryonic lethality, gonad defects,     tions dependent spindle checkpoint <br /> a reduced brood size observed mdf-1(gk2) ani-           pathway. <br /> mals. A genetic screen suppressors weak allele of <br /> mat-3, APC subunit, identified mdf-2,mdf-1, and <br />  <br />  <br />                                                                                                                    Page 2 18 <br />                                                                                              (page number citation purposes) <br />  Cell Division 2008, 3:6                                                                     http://www.celldiv.com/content/3/1/6 <br />  <br />  <br />  <br />  <br /> Results                                                           Table 1: Phenotype analysis san-1(ok1580) hermaphrodites <br /> Phenotype analysis san-1(ok1850) <br />                                                                    Phenotype                   Wild-type         san-1(ok1580) <br /> Previously, shown san-1(RNAi) embryos are <br /> viable, sensitive conditions depolymerize              Larval arrest (%)          0.0                2.8 &#177; 1.5 <br /> microtubules (anoxia nocodazole treatment), provid-             Slow growth (%)            0.0                3.2 &#177; 1.6 <br /> ing evidence san-1 gene product role         Protruding vulva (%)       0.0                5.6 &#177; 3.9 <br /> spindle checkpoint pathway [21,22]. To investigate             Adult death (%)            4.7 &#177; 2.3          12.9 &#177; 3.2 <br /> role san-1 C. elegans obtained san-1 deletion          Eggs uterus             12.6 &#177; 2.8         15.3 &#177; 6.8 <br /> strain C. elegans Knockout Consortium. We veri-           Average brood size         312 &#177; 11.1         190 &#177; 79.5 <br />                                                                    Incidence males (%)     0.0                4.9 &#177; 2.05 <br /> fied deletion DNA sequence analysis found <br /> that san-1(ok1580) allele 992 kb deletion, which <br /> is consistent documented wormbase. The            parison wild-type animals, san-1(ok1580) animals <br /> san-1 gene contains 8 predicted exons, dele-         larvae arrest slow growth phenotype (Table 1). <br /> tion san-1(ok1580) removes exons 3&#8211;7            The san-1(ok1580) adults higher incidence death <br /> first 13 bp intron 7. Thus, san-1(ok1580) allele        resulting defects vulva morphogenesis (Table 1). <br /> minimally catalytic null predicted kinase domain         We investigated vulva defects using DIC <br /> and Mad3/Bub1 homology region predicted essen-              microscopy examine vulva region san-1(ok1580) <br /> tial binding CDC20p absent (data shown).            animals determined san-1(ok1580) animals <br />                                                                   contained tissue damage, abnormal vulva anatomy, and <br /> Others shown san-1(ok1580) hermaphrodites               increase protruding vulva phenotype (Figure <br /> grown 24&#176;C phenotypes including                 1B&#8211;D, Table 1). In comparison wild-type adults, the <br /> higher incidence males population, egg laying           average number eggs held uterus san- <br /> defects, higher death rate adult hermaphrodites          1(ok1580) animals significantly different (P = .176) <br /> due vulva bursting formation bag worms [19].        (Table 1). However, number eggs held uterus <br /> As prior work san-1(RNAi) 20&#176;C,               significantly different, ranging 8 18 wild- <br /> characterized phenotype san-1(ok1580) animals              type animals, 5 27 san-1(ok1580) animals, indi- <br /> grown 20&#176;C. Similar animals maintained 24&#176;C,             cating san-1(ok1580) animals not <br /> we san-1(ok1580) embryos viable,          laying embryos. Consistent idea, observed <br /> had low penetrant phenotypes (Table 1). In com-              18% (n = 16) san-1(ok1580) 1-day old adults <br />  <br />  <br />  <br />  <br /> Figure <br /> Abnormal 1 vulva male tail development san-1(ok1580) animal <br /> Abnormal vulva male tail development san-1(ok1580) animal. L1 larvae grown 20&#176;C three <br /> days 1-day old adults analyzed using DIC microscopy. Wild-type hermaphrodites (A), san-1(ok1580) hermaphrodites <br /> (B-D), wild-type males (E) san-1(ok1580) males (F-H) analyzed morphologic defects. White arrows point vulva <br /> defects including tissue damage (B), abnormal morphology (C), protruding vulva (D). Black arrows point defects <br /> observed male tail, including tissue damage (F), ray fan defects (G), abnormal tissue structure (H). Bar = 25 &#956;m. <br />  <br />  <br />  <br />                                                                                                                        Page 3 18 <br />                                                                                                  (page number citation purposes) <br />  Cell Division 2008, 3:6                                                                http://www.celldiv.com/content/3/1/6 <br />  <br />  <br />  <br /> had larvae hatching uterus. We analyzed prog-       assembly function (hcp-3, knl-1, knl-3, cls-2) and <br /> eny production san-1(ok1580) hermaphrodites             homologues zw10 rod (czw-1, rod-1), ana- <br /> determined average number progeny produced         lyzed RNAi gene products result severe <br /> was significantly decreased comparison wild-type         embryonic lethality, testing synthetic lethal interac- <br /> hermaphrodites (Table 1). Some san-1(ok1580)            tion san-1(ok1580) possible. We chose focus <br /> hermaphrodites died young adults produced          analysis instead genes bub-3, mdf-1, mdf-2, hcp-1, <br /> than 60 progeny, did die,    hcp-2 kbp-5. <br /> produced significantly progeny wild-type her- <br /> maphrodites (P = .001). The san-1(ok1580) animals         A reduction spindle checkpoint genes (san-1, mdf-1, <br /> have higher incidence males (Table 1). Additionally,      mdf-2, bub-3) kinetochore assembly genes (hcp-1, <br /> san-1(ok1580) males various defects tail morphol-      hcp-2) individually did result severe embryonic <br /> ogy including increase tissue damage, abnormal       lethality majority animals developed into <br /> ray morphology tissue structure (Figure 1F&#8211;H). In       adults (Figure 2A). The mdf-1(RNAi) mdf-2(RNAi) <br /> studies used san-1(ok1580) males mating indi-       animals phenotypes adults including death <br /> cating san-1(ok1580) males sterile. In        vulva bursting formation bag worms <br /> summary, results consistent defects cell     (data shown), consistent others <br /> cycle progression leading abnormal vulva development        observed [18]. The mdf-1(gk2) deletion mutant a <br /> in hermaphrodite somatic cell defects adult     significant embryo lethal phenotype [18], suggesting that <br /> male [28].                                                     mdf-1(RNAi) acts null. In comparison san- <br />                                                                1(ok1580) animals, san-1(ok1580);bub-3(RNAi) (P &lt; <br /> Genetic interactions spindle checkpoint genes      .04), san-1(ok1580);mdf-1(RNAi) (P &lt; .01) san- <br /> and hcp-1                                                      1(ok1580);hcp-1(RNAi) (P &lt; .0003) animals displayed a <br /> To analyze san-1 function C. elegans used    reduction ability embryos develop adult- <br /> RNAi identify additional genes genetically interact    hood (Figure 2B). Nearly san-1(ok1580);hcp- <br /> with san-1(ok1580). We interested determining       1(RNAi) animals died embryos young larvae. The <br /> RNAi specific gene products resulted severe       san-1(ok1580);hcp-2(RNAi) animals decrease the <br /> phenotype san-1(ok1580) background. This                ability develop adulthood, comparison san- <br /> approach similar concept synthetic lethal screen,   1(ok1580) animals (P &lt; .05), phenotype was <br /> an established technique identify genetic interactions      severe san-1(ok1580);hcp-1(RNAi) animals <br /> in double mutant phenotype distinct         (Figure 2b). The san-1(ok1580);mdf-2(RNAi) a <br /> more severe phenotype(s) individual sin-       reduced significant difference viability (P = <br /> gle mutants. The genes evaluated included        .073). We did observe viability defect san- <br /> that involved kinetochore assembly spindle         1(ok1580);kbp-5(RNAi) animals kbp-5 fur- <br /> checkpoint function severe embryonic           ther analyzed (data shown). <br /> lethality phenotypes. Thus, analyzed hcp-1, hcp-2, the <br /> putative bub-3 homologue Y54G9A.6, known spindle           Because RNAi does completely remove a <br /> checkpoint genes, mdf-1 mdf-2, predicted           genes function, used genetic deletion hcp-2 gain <br /> kinetochore protein kbp-5 [18]. The hcp-1/2 genes encode       greater understanding genetic interaction between <br /> kinetochore-localized proteins required kinetochore        hcp-2 san-1. We crossed hcp-2(ok1757) san- <br /> assembly, animals significant defect        1(ok1580) animals produce san-1(ok1580);hcp- <br /> viability HCP-1/2 simultaneously depleted        2(ok1757) animal. The hcp-2(ok1757) allele 1.281 kb <br /> by RNAi [23,25]. The gene product Y54G9A.6              deletion, removes exon 5 resulting a <br /> homology spindle checkpoint gene BUB-3 (see             frame-shift; hcp-2 8 predicted exons (data shown). <br /> Additional file 1). BLASTp analysis C. elegans Y54G9A.6     We verified deletion DNA sequence analysis. We <br /> predicted gene product suggests homology         determined 26.25% F2 offspring (n = 80) from <br /> yeast BUB3 (E value = 4e-28), <span id='am-74' about='obo:NCBITaxon_9606' typeof='owl:Thing obo:NCBITaxon_9605 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-75' property="oboInOwl:hasExactSynonym" content="man" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Homo sapiens" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasExactSynonym" content="human" datatype="xsd:string"></span><span id='am-80'  ></span>Homo sapiens</span> BUB3 (E value       san-1(ok1580) hcp-2(ok1757) cross phenotypes <br /> = 3e-27), D. malanogaster BUB3 (E value = 2e-74). The      including embryo lethality, larvae lethality, odd morphol- <br /> amino acid identity C. elegans Y54G9A.6 pre-       ogy lethargy. The number F2 progeny abnor- <br /> dicted gene product BUB3 proteins 44% H. sapi-      mal phenotypes higher expected 1/16 <br /> ens, 43% D. melanogastor 19% <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-28'  ></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span>         (6.25%) homozygous double mutant animal (san- <br /> cerevisiae, determined using ClustalW software.         1(ok1580);hcp-2(ok1757)), suggesting F2 <br /> Thus, Y54G9A.6 gene referred bub-3        progeny combination homozygous het- <br /> this report. Other gene candidates likely interact     erozygous mutations san-1(ok1580) hcp- <br /> san-1, spindle checkpoint genes (bub-1),         2(ok1757) abnormal phenotype. We geno- <br /> genes encode products essential kinetochore           typed 39 F2 animals, using single worm PCR detect the <br />  <br />  <br />                                                                                                                    Page 4 18 <br />                                                                                              (page number citation purposes) <br />  Cell Division 2008, 3:6                                                                      http://www.celldiv.com/content/3/1/6 <br />  <br />  <br />  <br />  <br /> Figure 2 lethality spindle checkpoint genes <br /> Synthetic <br /> Synthetic lethality spindle checkpoint genes. For graphs A, B, C, embryos, designated genotypes, were <br /> assayed ability hatch (white filled bars) ability develop adulthood days food (black filled <br /> bars). The san-1(ok1580);hcp-2(ok1757) animals given days develop adulthood. The data obtained four <br /> independent experiments &gt; 200 total animals assayed. Error bar standard deviation. <br />  <br />  <br />  <br />  <br />                                                                                                                          Page 5 18 <br />                                                                                                    (page number citation purposes) <br />  Cell Division 2008, 3:6                                                                     http://www.celldiv.com/content/3/1/6 <br />  <br />  <br />  <br /> san-1 hcp-2 deletions, determined             The san-1(ok1580);hcp-1(RNAi) animals develop <br /> homozygous double mutant (san-1(ok1580);hcp-                   abnormally <br /> 2(ok1757)) animals homozygous san-                 To characterize phenotype san- <br /> 1(ok1580) deletion heterozygous hcp-               1(ok1580);mdf-2(RNAi), san-1(ok1580);bub-3(RNAi) and <br /> 2(ok1757) deletion displayed embryonic larval              san-1(ok1580);hcp-1(RNAi) animals, analyzed ani- <br /> lethality phenotypes. The hcp-2(ok1757) animals                mals developmental defects. The majority animals, <br /> appeared lethargic adults, noted reduc-   did die embryos, arrested larvae an <br /> tion motility, did exhibit embryonic lethality      abnormal gonad adults (Figure 3, Table 2). First, the <br /> and majority reached adulthood days egg        san-1(ok1580);mdf-2(RNAi), san-1(ok1580);bub-3(RNAi) <br /> laying (Figure 2A). We isolated san-1(ok1580);hcp-           san-1(ok1580);hcp-1(RNAi) animals larvae arrest <br /> 2(ok1757) double mutant determined major-         slow growth phenotypes (Figure 3A). Additional <br /> ity embryos hatched, 7.3 % &#177; 2.4 lar-   defects observed san-1(ok1580);mdf-2(RNAi) or <br /> vae reached adulthood days 20&#176;C.               san-1(ok1580);bub-3(RNAi) animals included abnormal <br /> However, given days develop majority san-    cell morphology mitotic region gonad, <br /> 1(ok1580);hcp-2(ok1757) animals developed adult-            abnormal cell morphology meiotic region the <br /> hood (Figure 2b). Several san-1(ok1580);hcp-            gonad, meiotic cells button like cell morphology <br /> 2(ok1757) animals morphological defects suggesting         reminiscent apoptotic cells, abnormal oocyte struc- <br /> that double mutant severe phenotype        ture (Figure 3B). Some animals produced sperm but <br /> the single mutants (data shown). Our results indicate      spermatheca appeared abnormal structure as <br /> that phenotype observed san-1(ok1580);hcp-              determined sperm localization (Figure 3B, black <br /> 2(ok1757) identical observed san-            arrows). We observed gonad migration defects (data <br /> 1(ok1580);hcp-1(RNAi) animals.                                 shown). The majority san-1(ok1580);hcp-1(RNAi) <br />                                                                animals died arrested larvae. Abnormal gonad mor- <br /> Nearly hcp-1(RNAi);hcp-2(RNAi) embryos die          phology observed bub-3(RNAi) hcp- <br /> during embryogenesis; however, half san-           1(RNAi) animals. Some san-1(ok1580) mdf- <br /> 1(ok1580);hcp-1(RNAi) mdf-2(RNAi);hcp-1(RNAi) ani-          2(RNAi) abnormal gonad morphology (Table 2). <br /> mals survive embryogenesis die larvae (Figure 2B, <br /> 2C). Thus, embryo lethality san-1(ok1580);hcp-          The hcp-1(RNAi);hcp-2(RNAi) animals die embryos <br /> 1(RNAi) mdf-2(RNAi);hcp-1(RNAi) animals           san-1(ok1580);hcp-1(RNAi) embryos <br /> severe comparison hcp-1(RNAi);hcp-2(RNAi)            hatch later die young larvae (Figure 2). To deter- <br /> animals [23]. In comparison mdf-2(RNAi) animals,        san-1(ok1580);hcp-1(RNAi) embryos have <br /> mdf-2(RNAi);hcp-1(RNAi) animals significant              developmental defects used pharynx marker <br /> decrease ability develop survive adult-       developmental progression, analyzing expres- <br /> hood (P &lt; .011). The mdf-2(RNAi);hcp-2(ok1757) animals         sion myo-2::GFP. Briefly, myo-2 encodes muscle-type <br /> also decrease ability survive adulthood     specific myosin heavy chain isoform expressed dur- <br /> (P &lt; .025), viability defect severe     ing embryogenesis primordial pharyngeal muscle <br /> as observed mdf-2(RNAi);hcp-1(RNAi) animals            cells. Similar control embryos (Figure 4A&#8211;D), the <br /> (Figure 2C). Similar bub-3(RNAi) hcp-1(RNAi) ani-        hcp-1(RNAi) (Figure 4E&#8211;H) san-1(ok1580) (Figure <br /> mals, bub-3(RNAi);hcp-1(RNAi) animals did         4I&#8211;L) animals normal pharynx development. How- <br /> a viability defect (Figure 2C). Furthermore, bub- <br /> 3(RNAi);hcp-2(ok1757) (Figure 2C) bub-3(RNAi);hcp-         Table 2: Analysis gonad morphology <br /> 2(RNAi) (data shown) animals did viability <br />                                                                 Genotype                        Abnormal Gonad (%)               No. <br /> defects. Using RT-PCR, comparison wild-type, a <br /> greater 5 fold reduction bub-3 transcript level (P &lt; <br />                                                                 Wild-type                       0.0                              25 <br /> .029) observed bub-3(RNAi) animals, demon-          bub-3(RNAi)                     0.0                              25 <br /> strating RNAi efficiency. Together, results suggest       hcp-1(RNAi)                     0.0                              33 <br /> that san-1 genetically interacts bub-3, mdf-2, mdf-1,      mdf-2(RNAi)                     37.8                             37 <br /> hcp-1 hcp-2 C. elegans requires hcp-1     san-1(ok1580)                   10.3                             29 <br /> either checkpoint genes, san-1 mdf-2 bub-     san-1(ok1580);bub-3(RNAi)       84.8                             33 <br /> 3. Furthermore, results suggest hcp-1 hcp-2      san-1(ok1580);hcp-1(RNAi)       100.0                            9 <br />                                                                 san-1(ok1580);mdf-2(RNAi)       94.4                             36 <br /> have redundant identical functions. <br />                                                                 Animals assayed using DIC microscopy scored abnormal <br />                                                                 gonad development, included absence oocytes, abnormal <br />                                                                 gonad morphology, abnormal cell structure mitotic or <br />                                                                 meiotic region gonad. The total number animals analyzed <br />                                                                 (No.). <br />  <br />  <br />                                                                                                                        Page 6 18 <br />                                                                                                  (page number citation purposes) <br />  Cell Division 2008, 3:6                                                                 http://www.celldiv.com/content/3/1/6 <br />  <br />  <br />  <br />  <br /> Phenotype <br /> Figure 3 analysis gonad morphology san-1(ok1580);bub-3(RNAi), san-1(ok1580);hcp-1(RNAi), san-1(ok1580);mdf-2(RNAi) <br /> animals <br /> Phenotype analysis gonad morphology san-1(ok1580);bub-3(RNAi), san-1(ok1580);hcp-1(RNAi), san- <br /> 1(ok1580);mdf-2(RNAi) animals. L1 larvae grown 20&#176;C days gonad morphology assayed using <br /> DIC microscopy. Phenotypes observed include larvae arrest slow growth phenotypes (A). For san-1(ok1580);bub-3(RNAi) <br /> or san-1(ok1580);mdf-2(RNAi) animals phenotypes included abnormal cell morphology mitotic region gonad (white <br /> arrow head), abnormal cell morphology meiotic region gonad (white arrow), abnormal oocyte morphology <br /> (B, black arrow head). Animals produce sperm (B, black arrows), spermatheca appeared abnormal indi- <br /> cated sperm localization. The position vulva indicated V. Bar = 25 &#956;m. <br />  <br />  <br />  <br /> ever, san-1(ok1580);hcp-1(RNAi) embryos        hcp-1 gene products function spindle checkpoint <br /> reduction myo-2::GFP (Figure 4M&#8211;N, Q&#8211;R) abnor-            pathway. It known blastomeres wild-type <br /> mal pharynx structure (Figure 4O&#8211;P). Together,            embryos exposed oxygen deprivation (anoxia) reversi- <br /> results indicate san-1(ok1580);hcp-1(RNAi)             bly arrest cell cycle progression [21,29]. Unlike wild-type <br /> embryos developmental defects.                             embryos, san-1(RNAi) mdf-2(RNAi) embryos sen- <br />                                                                 sitive microtubule disrupting environments of <br /> Spindle checkpoint activity                                     anoxia nocodazole treatment. Specifically, san-1(RNAi) <br /> Our data suggest development san-1(ok1580)          mdf-2(RNAi) embryos exposed anoxia sig- <br /> animals depends bub-3 hcp-1,         nificantly reduced number metaphase blastomeres and <br /> clear if, like san-1,mdf-1 mdf-2 genes, bub-3   increase anaphase bridging chromosomal <br />  <br />  <br />                                                                                                                    Page 7 18 <br />                                                                                              (page number citation purposes) <br />  Cell Division 2008, 3:6                                                                         http://www.celldiv.com/content/3/1/6 <br />  <br />  <br />  <br />  <br /> The <br /> Figure <br />      san-1(ok1580);hcp-1(RNAi) <br />          4                       animals abnormal development <br /> The san-1(ok1580);hcp-1(RNAi) animals abnormal development. DIC microscopy MYO-2::GFP fusion <br /> protein, expressed cells differentiating pharynx, used assay pharynx development. Pharynx morphol- <br /> ogy, control, hcp-1(RNAi), san-1(ok1580), san-1(ok1580);hcp-1(RNAi) embryos assayed early (A, B, E, F, I, J, M, N) <br /> and late embryogenesis (C, D, G, H, K, L, O, P, Q, R). Bar = 15 &#956;m. <br />  <br />  <br /> abnormalities; supporting idea san-1 mdf-2 <br /> function spindle checkpoint genes [21,22]. Similar to <br /> what known regarding san-1(RNAi) embryos,              Table 3: Anoxia sensitivity assay spindle checkpoint activity <br /> san-1(ok1580) embryos exposed anoxia reduced <br /> survival rate (Table 3). The bub-3(RNAi) embryos exposed           Genotype        Anoxia viability a <br /> to anoxia reduced survival rate comparison <br /> to wild-type animals suggesting bub-3 functions                          Hatch (%)              Adulthood (%)              No. <br /> spindle checkpoint protein developing embryos. The <br />                                                                    Wild-type       99.2 &#177; 1.1             99.0 &#177; 1.1                 511 <br /> hcp-1(RNAi) embryos sensitive anoxia (Table <br />                                                                    san-1(ok1580)   77.3 &#177; 8.4             42.7 &#177; 21.4 (P &lt; .014)     482 <br /> 3). Additionally, did observe sensitivity anoxia       bub-3(RNAi)     90.2 &#177; 3.1             81.0 &#177; 12.0 (P &lt; .053)     481 <br /> in hcp-2(RNAi) hcp-2(ok1757) embryos (data                  hcp-1(RNAi)     92.6 &#177; 8.9             92.3 &#177; 8.5                 230 <br /> shown). These results suggest hcp-1 hcp-2 not <br /> required anoxia induced metaphase arrest            aAnimals assayed sensitivity 1 day anoxia by <br />  <br /> function spindle checkpoint protein.                          determining percentage anoxia exposed embryos able <br />                                                                    hatch develop adulthood. Four independent experiments <br />                                                                    total number embryos assayed (No.). <br />  <br />  <br />  <br />  <br />                                                                                                                            Page 8 18 <br />                                                                                                      (page number citation purposes) <br />  Cell Division 2008, 3:6                                                                http://www.celldiv.com/content/3/1/6 <br />  <br />  <br />  <br /> Analysis san-1(ok1580);hcp-1(RNAi) mitotic                  1(RNAi) embryos (Figure 5A&#8211;L), san-1(ok1580);hcp- <br /> blastomeres                                                    1(RNAi) embryos significant increase abnormal <br /> To gain greater understanding functional relation-    nuclei (44%, n = 50) (Figure 5M&#8211;T). These abnormalities <br /> ship hcp-1 san-1 gene products devel-       included abnormal mitotic structure, lagging chromo- <br /> oping embryos determined chromosome loss             somes, anaphase bridging (Figure 5M, 5Q, arrows). <br /> occurring san-1(ok1580);hcp-1(RNAi) embryos. We         These defects observed young embryos (2&#8211;4 <br /> analyzed chromosome structure mitotic blastomeres           cell embryos; 15.4% (n = 13)), indicating chro- <br /> from wild-type, hcp-1(RNAi), san-1(ok1580), san-           mosome segregation defects occur early embryo devel- <br /> 1(ok1580);hcp-1(RNAi) embryos collected, fixed             opment. <br /> stained previously described [22]. We used mAb414, <br /> which recognizes nuclear pore complexes anti-          In young wild-type embryos (less 30 blastomeres) <br /> PhosH3, recognizes phosphorlyated form            nuclear pore complexes detected mAb 414 not <br /> histone H3 distinguish mitotic blastomeres [22,29-31].      diminish mitosis [31]. In contrast, embryos with <br /> We unlike wild-type, san-1(ok1580), hcp-        greater 30 blastomeres, nuclear pore complexes <br />  <br />  <br />  <br />  <br /> Figure <br /> The san-1(ok1580);hcp-1(RNAi) <br />         5                        embryos contain blastomeres abnormal chromosomes <br /> The san-1(ok1580);hcp-1(RNAi) embryos contain blastomeres abnormal chromosomes. The <span id='am-120' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-121' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-122'  ></span><span id='am-123' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Phos</span> H3 anti- <br /> body recognizes phosphorylated form Histone H3 specific mitotic cells (prophase anaphase) mAb414 rec- <br /> ognizes nuclear pore complex. Images collected using spinning disc confocal microscope. The arrows point to <br /> abnormal mitotic nuclei. Arrowhead points metaphase blastomere display aggregates recognized mAb414 met- <br /> aphase blastomere. M denotes metaphase blastomeres. Two typical san-1(ok1580);hcp-1(RNAi) embryos shown. Images <br /> depict single focal plane. Bar = 10 &#956;m. <br />  <br />  <br />  <br />                                                                                                                   Page 9 18 <br />                                                                                             (page number citation purposes) <br />  Cell Division 2008, 3:6                                                                  http://www.celldiv.com/content/3/1/6 <br />  <br />  <br />  <br /> detected mAb 414 diminish prometaphase               abnormal nuclei aggregates nuclear pore com- <br /> anaphase. Similar wild-type embryos (Figure 5A&#8211;D),           plex suggests progression mitosis aber- <br /> with greater 30 blastomeres, nuclear pore com-         rant san-1(ok1580);hcp-1(RNAi) embryos. <br /> plexes, detected mAb414, diminished met- <br /> aphase blastomeres hcp-1(RNAi) (Figure 5E&#8211;H),            To gain greater understanding mitotic progression in <br /> san-1(ok1580) embryos (Figure 5I&#8211;L). However, san-           san-1(ok1580);hcp-1(RNAi) embryos analyzed <br /> 1(ok1580);hcp-1(RNAi) embryos greater 30              chromosome, kinetochore spindle microtubule struc- <br /> blastomeres, nuclear pore complex, detected              ture using indirect immunofluorescence. We stained <br /> mAb414, small aggregates surrounding met-           embryos MPM-2, antibody recognizes <br /> aphase plates (72%, n = 11) (Figure 5Q&#8211;T, 5S arrow              mitotic proteins localized kinetochore centro- <br /> head). This phenotype observed older         somes [32]. The kinetochore structure san-1(ok1580) <br /> embryos, younger embryos dimin-           hcp-1(RNAi) embryos similar wild-type <br /> ished nuclear pore complexes. The accumulation               embryos (Figure 6C,G,K). However, san- <br />  <br />  <br />  <br />  <br /> The <br /> Figure <br />      san-1(ok1580);hcp-1(RNAi) <br />          6                     embryos normal abnormal kinetochore centriole structure <br /> The san-1(ok1580);hcp-1(RNAi) embryos normal abnormal kinetochore centriole structure. Analy- <br /> sis metaphase blastomeres wild-type (A-D), hcp-1(RNAi) (E-H), san-1(ok1580) (I-L) san-1(ok1580);hcp-1(RNAi) embryos <br /> (M-T) assayed. The MPM-2 antibody recognizes mitotic proteins located kinetochore centrioles. Two typical <br /> san-1(ok1580);hcp-1(RNAi) metaphase blastomeres shown. The metaphase blastomere shown Q-T abnormal struc- <br /> ture. Bar = 10 &#956;m. <br />  <br />  <br />  <br />                                                                                                                    Page 10 18 <br />                                                                                               (page number citation purposes) <br />  Cell Division 2008, 3:6                                                               http://www.celldiv.com/content/3/1/6 <br />  <br />  <br />  <br /> 1(ok1580);hcp-1(RNAi) embryos contained normal           kinetochore assembly pathway [23,25]. The centro- <br /> and abnormal kinetochore structure, determined      mere structure spindle microtubules san-1(ok1580) <br /> presence absence two-line formation       hcp-1(RNAi) embryos similar wild- <br /> sister chromatids (Figure 6O, 6S). The abnormal kineto-       type embryos (Figure 7A&#8211;L). The san-1(ok1580);hcp- <br /> chore structure correlated abnormal mitotic struc-       1(RNAi) blastomeres normally formed metaphase <br /> ture.                                                         plates normal centromere microtubule structure <br />                                                               (Figure 7M&#8211;P). The san-1(ok1580);hcp-1(RNAi) blast- <br /> We used anti-HCP-3 YL1/2 analyze localization      omeres abnormal chromosome structure contained <br /> of centromeric histone HCP-3 (CENP-A) spindle         HCP-3 localized chromosomes (Figure 7U&#8211;X); in <br /> microtubules, respectively [23,26]. HCP-3    blastomeres HCP-3 associated with <br />  <br />  <br />  <br />  <br /> The <br /> Figure <br />     san-1(ok1580);hcp-1(RNAi) <br />         7                      mitotic blastomeres normal abnormal centromere spindle structure <br /> The san-1(ok1580);hcp-1(RNAi) mitotic blastomeres normal abnormal centromere spindle struc- <br /> ture. Analysis metaphase blastomeres wild-type (A-D), hcp-1(RNAi) (E-H), san-1(ok1580) (I-L) san-1(ok1580);hcp- <br /> 1(RNAi) embryos (M-X) assayed. Anti-HCP-3 recognizes centromeric histone HCP-3 YL1/2 antibody recognizes <br /> microtubules. Three typical san-1(ok1580);hcp-1(RNAi) metaphase blastomeres shown. The metaphase blastomeres shown <br /> in Q-X abnormal structure. Images collected using spinning disc confocal microscope. Arrow points HCP-3 <br /> not associated chromosomes. Bar = 2 &#956;m. <br />  <br />  <br />  <br />                                                                                                                 Page 11 18 <br />                                                                                            (page number citation purposes) <br />  Cell Division 2008, 3:6                                                                http://www.celldiv.com/content/3/1/6 <br />  <br />  <br />  <br /> the chromosomes (Figure 7Q&#8211;T). In abnormal blas-         Discussion <br /> tomeres microtubule structure abnormal (Figure         Genetic interactions san-1 <br /> 7Q&#8211;X). Embryos analyzed experiments con-             The use synthetic lethal screens genetic tool elu- <br /> tained ~20&#8211;30 blastomeres possible         cidate genes interact similar cellular functions or <br /> defects altered gene expression resulting      pathways. Thus, genetic mutant does a <br /> chromosome segregation defects.                                severe phenotype use synthetic lethal screen to <br />                                                                identify double mutants phenotype distinct or <br /> Given presence anaphase bridging, lagging chromo-       severe phenotype(s) observed indi- <br /> somes, abnormal chromosome structure observed           vidual single mutants aid understand- <br /> the san-1(ok1580);hcp-1(RNAi) animals, likely       ing function gene products. To analyze san-1 <br /> the blastomeres transitioning mitosis             function C. elegans embryos used RNAi identify <br /> abnormally. To determine occurred vivo pro-      genes genetically interact san-1(ok1580). <br /> duced san-1(ok1580);tbg-1::GFP;pie-1::GFP::H2B strain        We determined san-1(ok1580);hcp-1(RNAi), san- <br /> (PM115 strain, histone gamma-tubulin GFP-tagged            1(ok1580);bub-3(RNAi), san-1(ok1580);mdf-1(RNAi), and <br /> proteins) live cell imaging using spinning disc confo-   san-1(ok1580);mdf-2(RNAi) reduced ability sur- <br /> cal microscope. The san-1(ok1580);hcp-1(RNAi);tbg-             vive. The genetic interaction san-1, mdf-1, and <br /> 1::GFP;pie-1::GFP::H2B embryos contained blastomeres           mdf-2 expected, genes thought func- <br /> with abnormal mitotic progression. Analysis typical       tion spindle checkpoint pathway mitotic and <br /> san-1(ok1580);hcp-1(RNAi) embryo showed blast-             meiotic cells [18-21]. The predicted C. elegans bub-3 gene <br /> omere progressed anaphase normal chro-           product high homology BUB-3 systems <br /> mosome segregation, blastomere                bub-3(RNAi) embryos sensitive the <br /> abnormal chromosome segregation leading anaphase            microtubule disrupting process anoxia exposure, sup- <br /> bridging (see Additional file 2). These results dem-   porting idea Y54G9A.6 gene encodes bub- <br /> onstrated abnormal chromosome segregation                 3 spindle checkpoint gene. <br /> occurred san-1(ok1580);hcp-1(RNAi) embryos. <br />                                                                The genetic interaction hcp-1/2 spindle <br /> Thus far, shown san-1(ok1580);hcp-            checkpoint genes likely complex. The viabil- <br /> 1(RNAi) embryos abnormally segregate chromosomes               ity san-1(ok1580);hcp-1(RNAi) mdf-2(RNAi);hcp- <br /> which likely leads developmental defects lethality.     1(RNAi) animals severely compromised, suggesting <br /> The hcp-1(RNAi) embryos shown            HCP-1 interacts MDF-2 SAN-1. However, <br /> chromosome segregation issues lead increased           bub-3(RNAi);hcp-1(RNAi) animals did sig- <br /> lethality. One possible reason synthetic lethal        nificant decrease survival rate. This to <br /> interaction san-1 hcp-1 loss san-1      BUB-3 HCP-1 involved specific func- <br /> amplifies subtle hcp-1 defect. Given defect pro-      tion. The san-1(ok1580);hcp-2(ok1757) animals have <br /> gression mitosis observed mAb 414, timing         embryonic larvae lethal phenotypes. We isolated a <br /> of mitotic progression possibly perturbed hcp-1     san-1(ok1580);hcp-2(ok1757) double mutant deter- <br /> absent loss san-1 increases defect. Therefore,     mined severe phenotypes that <br /> we conducted live cell imaging compare mitotic pro-         observed single mutants; suggests hcp-2 <br /> gression hcp-1(RNAi);tbg-1::GFP;pie-1::GFP::H2B         san-1 genetically interact. However, phenotype of <br /> control animals. We analyzed mitosis AB P cell      san-1(ok1580);hcp-2(ok1757) double mutants not <br /> blastomeres 2-cell embryo. We determined time      severe observed san-1(ok1580);hcp-1(RNAi) ani- <br /> took transition nuclear envelope break            mals, suggesting HCP-1 HCP-2 com- <br /> (NEB) formation metaphase plate, metaphase        pletely redundant. HCP-1 HCP-2 54% similar <br /> anaphase onset NEB anaphase onset P cell      N- C-termini [23]. Although HCP-1 <br /> a wild-type hcp-1(RNAi) 2-cell embryo. We deter-      shares similarity human CENP-F, primarily a <br /> mined time NEB AB cell NEB           tandem repeat present HCP-1 CENP-F, HCP- <br /> the P cell. We mitotic progression hcp-      2 lacks significant similarity CENP-F [23]. Thus, is <br /> 1(RNAi) embryos analyzed 21&#176;C did              possible HCP-1 HCP-2 overlapping dis- <br /> chromosomal defects time took progress           tinct functions. It determine spe- <br /> through mitosis significantly different compar-     cific molecular interactions HCP-1, HCP-2, SAN- <br /> ison control embryos (Figure 8). Together, data         1 MDF-2. <br /> indicates hcp-1(RNAi) animals require functional <br /> spindle checkpoint normal development. <br />  <br />  <br />  <br />  <br />                                                                                                                   Page 12 18 <br />                                                                                              (page number citation purposes) <br />  Cell Division 2008, 3:6                                                                     http://www.celldiv.com/content/3/1/6 <br />  <br />  <br />  <br />  <br /> The <br /> Figure <br /> embryos <br />      hcp-1(RNAi) <br />          8       2-cell embryo does significant difference mitotic timing comparison wild-type 2-cell <br /> The hcp-1(RNAi) 2-cell embryo does significant difference mitotic timing comparison wild- <br /> type 2-cell embryos. Live cell imaging spinning disc confocal microscope used analyze mitotic progression in <br /> hcp-1(RNAi);tbg-1::GFP;pie-1::GFP::H2B control animals. The AB P cells 2-cell embryo analyzed. In P cell of <br /> a wild-type hcp-1(RNAi) 2-cell embryo time took (seconds) transition nuclear envelope break (NEB) to <br /> formation metaphase plate, formation metaphase plate anaphase onset NEB anaphase onset deter- <br /> mined. The time NEB AB cell NEB P cell determined. Each experiment conducted at <br /> 21&#176;C (n = 7 embryos). The P value determined using standard student t test. <br />  <br />  <br />  <br /> Functional analysis spindle checkpoint genes hcp-1         Others shown hcp-1(RNAi);hcp-2(RNAi) ani- <br /> in gonad development mitosis                                  mals meiotic defects, supporting idea that <br /> The majority san-1(ok1580);hcp-1(RNAi) animals             hcp-1 role meiosis [27]. The exact molecular func- <br /> severe developmental defects including embryo lar-            tion spindle checkpoint proteins HCP-1/2 in <br /> vae lethality. The viability defects observed likely      meiosis needs investigated. <br /> to abnormal chromosome segregation, turn <br /> compromise cellular structure function. The san-              The san-1(ok1580);hcp-1(RNAi) embryos severe <br /> 1(ok1580);bub-3(RNAi) san-1(ok1580);mdf-2(RNAi)               chromosome segregation defects, average 55.3% <br /> animals gonad defects. We shown               embryos die approximately 0.9% ani- <br /> that san-1(ok1580) mdf-2(RNAi) animals low               mals reach adulthood. Thus, san-1(ok1580);hcp- <br /> level gonad defects [18,19]; defects           1(RNAi) embryos, embryogenesis progress, the <br /> more severe spindle checkpoint genes reduced.          ability produce viable adults severely compromised. <br /> We observed reduction brood size san-1(ok1580)           We used myo-2::GFP analyze structure devel- <br /> hermaphrodites shown mdf-2(av14)             oping pharynx san-1(ok1580);hcp-1(RNAi) <br /> mutants reduced brood size, supporting         embryos abnormal <br /> role checkpoint genes germline function [19].         pharynx morphology observed reduction <br />  <br />  <br />                                                                                                                       Page 13 18 <br />                                                                                                  (page number citation purposes) <br />  Cell Division 2008, 3:6                                                                http://www.celldiv.com/content/3/1/6 <br />  <br />  <br />  <br /> in myo-2::GFP embryos. It known       mitosis [20]. Our data suggest hcp-1(RNAi) <br /> reduction myo-2::GFP san-1(ok1580);hcp-              blastomeres embryos analyzed 21&#176;C did take <br /> 1(RNAi);myo-2::GFP animals abnormal differen-        longer progress prometaphase metaphase, in <br /> tiation, loss myo-2::GFP DNA abnormal regula-        comparison control blastomeres, suggesting that <br /> tion myo-2::GFP. An increase genome loss      mitotic timing hcp-1(RNAi) blastomeres normal as <br /> chromosome segregation issues result        long hcp-2 gene product present. <br /> these possibilities. <br />                                                                CENP-F required chromosome alignment [34] and <br /> Closer examination chromosomes, nuclear pore            silencing CENP-F HeLa cells activates spindle <br /> proteins, centromere, kinetochore, microtubules            checkpoint [33]. The misalignment defect thought be <br /> indicates san-1(ok1580);hcp-1(RNAi) embryos           kinetochores correctly biorient rather <br /> have chromosome segregation defects leading abnor-          defect microtubule kinetochore attachment <br /> mal nuclei, anaphase bridging lagging chromosomes.         [33]. Our results support idea hcp-1(RNAi) ani- <br /> We determined nuclear pore complexes, detected        mals spindle checkpoint activated ensure <br /> by mAb414, formed small aggregates surrounding             proper chromosome segregation genome defects will <br /> metaphase plates san-1(ok1580);hcp-1(RNAi) embryos,         occur normal spindle checkpoint function; spe- <br /> indicating nuclear membrane pore complexes        cifically SAN-1 MDF-2 function required. Given <br /> not completely breaking down. This indicate           involvement spindle checkpoint tumor <br /> SAN-1 HCP-1 influences additional mitotic events        progression, fully understand genes <br /> addition chromosome segregation. The centromeric            products interact spindle checkpoint path- <br /> protein HCP-3 observed normal              way ensure genome fidelity maintenance. <br /> abnormal localization pattern san-1(ok1580);hcp- <br /> 1(RNAi) blastomeres. The abnormal localization HCP-         Conclusion <br /> 3 interpreted ways. First,   In report identified genes genetically interact <br /> to SAN-1 HCP-1 directly regulating localization     spindle checkpoint gene san-1 (mad-3 homo- <br /> HCP-3 region marks centromere,         logue). We spindle checkpoint genes, san-1 <br /> this unlikely kinetochore assembly path-       mdf-2, interact also <br /> way clearly places HCP-3 upstream HCP-1 SAN-1.          interact CENP-F homologue hcp-1. The bub-3 <br /> Alternatively, abnormal HCP-3 localization        spindle checkpoint gene does genetically interact with <br /> due chromosome segregation defects causing loss          hcp-1; suggesting hcp-1 interacts subset of <br /> chromatin marks centromere. Finally, possi-     spindle checkpoint. <br /> ble segregation defects result chromatin frag- <br /> ments HCP-3 associates with. Further studies        Detailed phenotype analysis san-1(ok1580);hcp- <br /> be required determine mechanism leading              1(RNAi) embryos demonstrates C. elegans embryos <br /> abnormal HCP-3 localization.                                   require san-1 hcp-1 proper chromosome segrega- <br />                                                                tion. Furthermore, gonad morphological defects were <br /> In san-1(ok1580);hcp-1(RNAi) embryos, kinetochore          observed san-1(ok1580);mdf-2(RNAi) san- <br /> proteins recognized MPM-2 antibody             1(ok1580);bub-3(RNAi) animals supporting a <br /> associated chromosomes chromosome            functional role spindle checkpoints meiosis. We <br /> structure aberrant. Furthermore, microtubule           provide evidence suggest hcp-1 hcp-2 gene <br /> structure altered blastomeres abnormal chro-       products overlapping distinct functions <br /> mosome structure. Together, data suggests        C. elegans. Together, data suggest C. ele- <br /> the san-1(ok1580);hcp-1(RNAi) blastomeres kineto-          gans embryo, HCP-1 functions proper alignment and <br /> chore microtubule interaction sometimes,        segregation chromosomes functions are <br /> always, compromised. Given increase anaphase            dependent elements spindle checkpoint path- <br /> bridges, lagging chromosomes abnormal mitotic              way spindle checkpoint proteins. Given <br /> nuclei, microtubule kinetochore interaction         important role spindle checkpoint kineto- <br /> san-1(ok1580);hcp-1(RNAi) animals likely sufficient     chore proteins chromosome segregation cell <br /> for proper chromosome segregation.                             cycle progression, great characterize <br />                                                                gene products developing organisms C. <br /> Others shown using mammalian cell culture (HeLa           elegans. <br /> cells) prolonged mitosis CENP-F (RNAi) <br /> cells prolonged mitosis dependent the <br /> spindle checkpoint gene BUBR1 [33]. In C. elegans hcp- <br /> 1(RNAi);hcp-2(RNAi) blastomeres longer progress <br />  <br />  <br />                                                                                                                   Page 14 18 <br />                                                                                              (page number citation purposes) <br />  Cell Division 2008, 3:6                                                               http://www.celldiv.com/content/3/1/6 <br />  <br />  <br />  <br />  <br /> Materials methods                                         placed OP50 NGM plates 20&#176;C days. <br /> Strains growth conditions                                 We quantified number animals developed to <br /> The wild-type Bristol strain (N2) mutant strains     adulthood days, animals developed to <br /> cultured NGM plates seeded E. coli (OP50)         adulthood days (slow growth) and <br /> raised 20&#176;C described (Sulston Hodgkin, 1988).      remained arrested larvae (larval arrest). To <br /> The following strains obtained Caenorhab-       assay adult death phenotype, animals grown for <br /> ditis elegans Genetics Center: RB1391(san-1(ok1580)),         5 days adulthood analyzed day for <br /> PD4790       (myo-2::GFP),    TH32     (tbg-1::GFP,    pie-   death survivorship (for wild-type san-1(ok1580), n <br /> 1::GFP::H2B), RB1492 (hcp-2(ok1757)). The                 = 210, n = 207, respectively). Hermaphrodites were <br /> Caenorhabditis elegans Gene Knockout Consortium (Okla-        moved daily fresh plates lost worms elimi- <br /> homa Medical Research Foundation) produced san-           nated analysis. The average brood size deter- <br /> 1(ok1580) hcp-2(ok1757) deletion alleles. The hcp-        mined quantifying number offspring individual <br /> 2(ok1757) mutant (strain RB1492) back-crossed          hermaphrodites produced (for wild-type san- <br /> wild-type animals times (strain PM117). The san-        1(ok1580), n = 5, n = 10, respectively). To quantify the <br /> 1(ok1580) allele crossed, using standard genetic tech-    number eggs uterus collected 1-day old <br /> niques, produce following strains: PM107 (san-         adults, placed agar pad visualized using Zeiss <br /> 1(ok1580);myo-2::GFP),      PM115      (san-1(ok1580);tbg-    compound microscope; number eggs the <br /> 1::GFP;pie-1::GFP::H2B), PM116 (san-1(ok1580);hcp-        uterus determined (for wild-type san-1(ok1580), <br /> 2(ok1757)). The san-1(ok1580) allele backcrossed          n = 16). To assay protruding vulva phenotype col- <br /> into wild-type background once. The san-1(ok1580)         lected 1-day old adults counted number adult <br /> hcp-2(ok1757) alleles genotyped single worm PCR       animals protruding vulva (for wild-type and <br /> with following primers: san-1 forward primer (CGC         san-1(ok1580), n = 210, n = 207, respectively). The p val- <br /> TTA AAG CTT GAT CAA CTT CTCG), san-1 reverse primer           ues (P) determined using standard student t test. <br /> (GCT AGT GAT TTC TCC TCC GTT TTC TCA), hcp-2 for- <br /> ward primer (ACT CTG AAG TCG GAA CAT GAA ATT),                Synthetic lethal assays <br /> and hcp-2 reverse primer (TGA AGA GCC TTC TGT GCA             RNA interference (RNAi) used decrease expres- <br /> <span id='am-104' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-105'  ></span><span id='am-106' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>AA</span>). DNA sequence analysis verify deleted region    sion specific genes. For experiments, methodol- <br /> the san-1(ok1580) hcp-2(ok1757) strains con-          ogy similar previously described (Hajeri et al). <br /> ducted using standard molecular techniques.                   Briefly, synchronous population L1 larvae (N2, san- <br />                                                               1(ok1580)) grown adulthood NGM plates sup- <br /> Analysis BUB-3 protein sequence                            plemented 200 &#956;g/ml ampicillin, 12.5 &#956;g/ml tetracy- <br /> The BUB-3 protein yeast <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-18'  ></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>     cline 1 mM IPTG, seeded bacterial strain <br /> was used conduct BLASTp C. elegans, Homo       expressing dsRNA specific mdf-2, mdf-1, bub-3, hcp-1, <br /> sapiens D. melanogastor genome identify       hcp-2 control food [35]. Several adults placed <br /> identical protein <span id='am-86' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>. The primary amino acid    secondary RNAi plate allowed lay eggs 3&#8211;4 <br /> sequences aligned using ClustalW processed           hours. The adults removed embryos were <br /> using LaserGene sequence analysis software package        grown 20&#176;C assayed embryonic lethality, larvae <br /> (DNASTAR). Clustal analysis C. elegans            lethality, morphological defects seen Nomarski <br /> Y54G9A.6 gene product (referred BUB-3) S.       microscopy using motorized Zeiss Axioskop imaged <br /> cerevisiae, H. sapiens D. melanogastor BUB-3 proteins     using Openlab 3.17. For viability experiment least <br /> was conducted determine percent identity       independent assays conducted. The P-value (P) <br /> proteins.                                                     determined using standard student t-test. In addi- <br />                                                               tion using RNAi food MRC, [35] also <br /> Phenotype analysis san-1(ok1580)                           produced RNAi food specific hcp-1 hcp-2. Briefly, <br /> For experiments animals kept 20&#176;C grown       plasmid <span id='am-95' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-96' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> feeding RNAi constructed by <br /> on E. coli (OP50). To assay high incidence males   cloning 1.1 kb Spe I &#8211; Dra I fragment hcp- <br /> (him) phenotype hermaphrodites, grown          1 cDNA LM46-3 1.6 kb Sal I fragment hcp-2 <br /> absence males, allowed lay eggs        cDNA, yk19h10 pPD129.36, allows IPTG <br /> hours percent male progeny determined          induction double stranded RNA. Respective feeding <br /> (for wild-type san-1(ok1580), n = 244, n = 325,           vectors transformed HTH115(BL23) E. coli <br /> respectively). The san-1(ok1580) animals assayed     strain fed animals, consistent previously <br /> larval arrest slow growth phenotypes (for wild-type       described RNAi methodology [35]. To conduct double <br /> and san-1(ok1580), n = 210, n = 218, respectively). To        RNAi experiments, RNAi foods plated, equal <br /> determine larval arrest slow growth observed     amounts, NGM IPTG plates. As control grew <br /> collected synchronized population L1 larvae          E. coli strain, plasmid insert, on <br />  <br />  <br />                                                                                                                 Page 15 18 <br />                                                                                            (page number citation purposes) <br />  Cell Division 2008, 3:6                                                               http://www.celldiv.com/content/3/1/6 <br />  <br />  <br />  <br /> identical NGM IPTG plates account differences       described [22,29]. Briefly, Phos H3 antibody recog- <br /> attributable vector. We conducted RT-PCR experi-        nizes phosphorylated (Ser10) form Histone H3 spe- <br /> ments analyze efficiency RNAi quantifying     cific mitotic cells (Upstate Biotechnology, Lake Placid <br /> level bub-3 transcript wild-type bub-3(RNAi) ani-    NY)[30]; mAb414 recognizes nuclear pore com- <br /> mals. Methodology previously described [36];          plex (Babco, Berkeley, CA) [37]; mAb MPM-2 recognizes <br /> conducted independent experiments           mitotic proteins located kinetochore, centrioles and <br /> each experiment performed triplicate. Primer            P granules (DAKO, Carpinteria, CA) [32]; anti-HCP-3 <br /> sequences used RT-PCR reactions bub-3 fol-       detects HCP-3 [26]; YL1/2 detects spindle micro- <br /> low:        5&apos;TGGGATCCTTTCAATCGGAAGC3&apos;                     tubules (Amersham Life Science, Little Chalfont, Bucking- <br /> 5&apos;TTATTTCGGTCTGCTCCGGA3&apos;. The P value deter-               hamshire, England). Microscopy using a <br /> mined using Mann-Whitney U Test.                           Zeiss Axioskop spinning disc confocal microscope <br />                                                                (McBain, CA). Images collected using spinning <br /> Spindle checkpoint assays                                      disc confocal microscope processed using Image J and <br /> To assay specific genes required spindle       Adobe Photoshop version 8.0. <br /> checkpoint activity exposed embryos anoxia pre- <br /> viously described [21,29]. Briefly, wild-type san-     Live imaging analyze mitosis <br /> 1(ok1580) animals grown, L1 larval stage      To analyze mitotic progression chromosomes and <br /> the 1 day old adult stage, RNAi food (vector        centriole used markers, using TH32 (tbg- <br /> insert control specific strain reduce expres-   1::GFP, pie-1::GFP::H2B) PM115 (san-1(ok1580);tbg- <br /> sion bub-3, hcp-1, hcp-2). The adults placed     1::GFP;pie-1::GFP::H2B) strain. L1 larvae grown on <br /> a fresh RNAi plate allowed lay eggs 1&#8211;2 hours.      RNAi control hcp-1 RNAi food adulthood 20&#176;C. 1- <br /> The adults removed embryos placed            day old gravid adults dissected embryos the <br /> into anoxia, 1 day, using BioBag type A environ-       appropriate stage collected mouth pipet and <br /> mental chamber. The embryos allowed recover         placed 2.5% agarose pad. The temperature the <br /> air assayed 24 hours later ability hatch    room embryos analyzed 21&#176;C. <br /> three days later ability reach adulthood. Four      Mitotic progression quantified 2-cell embryos <br /> independent experiments total 200           using spinning disc confocal microscope. To examine <br /> embryos analyzed.                                         mitotic progression, time took, seconds, tran- <br />                                                                sition onset prometaphase (as seen nuclear <br /> DIC microscopy                                                 envelope breakdown condensed chromosomes) to <br /> The L1 larvae (N2, san-1(ok1580), san-1(ok1580);myo-        formation metaphase plate, for- <br /> 2::GFP) grown adulthood appropriate RNAi            mation metaphase plate onset anaphase <br /> plates (control, bub-3, mdf-2 food). To analyze         determined. Seven embryos analyzed, using the <br /> developing pharynx embryos, san-                   Simple PCI Version 6 software program. The p-value (P) <br /> 1(ok1580);myo-2::GFP san-1(ok1580);hcp-                determined using standard student t-test. Movies, to <br /> 1(RNAi);myo-2::GFP adult animals dissected        demonstrate mitotic progression san-1(ok1580);tbg- <br /> embryos placed 2.5% agarose pads micros-         1::GFP;pie-1::GFP::H2B embryos, obtained ana- <br /> copy analysis. To analyze morphology control, mdf-      lyzing live, young embryos using spinning disc confocal <br /> 2(RNAi), hcp-1(RNAi), san-1(ok1580), san-1(ok1580);hcp-        microscope. Images collected, processed using NIH <br /> 1(RNAi),     san-1(ok1580);mdf-2(RNAi),            san-    Image, imported Quick Time display. <br /> 1(ok1580);bub-3(RNAi) animals, RNAi conducted as <br /> stated adult animals placed fresh        Competing interests <br /> RNAi plates allowed lay eggs hours. The     The author(s) declare competing inter- <br /> embryos developed days 20&#176;C DIC            ests. <br /> microscopy analysis. For experiments animals were <br /> placed 2.5% agarose pad visualized using            Authors&apos; contributions <br /> motorized Zeiss Axioskop Fluorescent microscope            VAH conducted indirect immunofluorescent assays <br /> imaged using Openlab 3.17. At independent          confocal analysis, produced analyzed genetic <br /> experiments conducted.                                    crosses, assisted experimental design data inter- <br />                                                                pretation, generated images, edited manuscript. <br /> Indirect immunofluorescent microscopy                          AMS conducted genetic crosses, assisted RNAi viabil- <br /> Wild-type san-1(ok1580) animals grown              ity assays, conducted RT-PCR reactions edited the <br /> appropriate RNAi food L1 stage adulthood. The          manuscript. LLM conducted bioinformatics analysis, con- <br /> adults collected dissected release embryos.        ceived experimental design, assisted interpretation of <br /> Embryos collected, fixed stained previously        data, provided necessary reagents study, generated <br />  <br />  <br />                                                                                                                  Page 16 18 <br />                                                                                             (page number citation purposes) <br />  Cell Division 2008, 3:6                                                                                   http://www.celldiv.com/content/3/1/6 <br />  <br />  <br />  <br /> images, edited manuscript. PAP conceived exper-                            E functions kinetochores binds cyclosome/APC. J <br />                                                                                    Cell Biol 1999, 146(5):941-954. <br /> imental design, conducted RNAi viability assays, analyzed                    5.    Amon A: The spindle checkpoint. Curr Opin Genet Dev 1999, <br /> phenotypes mutants RNAi animals, conducted                                  9(1):69-75. <br /> genetic crosses, analyzed live cell images, generated                        6.    Musacchio A, Salmon ED: The spindle-assembly checkpoint in <br />                                                                                    space time. Nat Rev Mol Cell Biol 2007, 8(5):379-393. <br /> images, wrote manuscript. All authors read                       7.    Hardwick KG: The spindle checkpoint. Trends Genet 1998, <br /> approved final manuscript.                                                     14(1):1-4. <br />                                                                              8.    Musacchio A, Hardwick KG: The spindle checkpoint: structural <br />                                                                                    insights dynamic signalling. Nat Rev Mol Cell Biol 2002, <br /> Addendum                                                                           3(10):731-741. <br /> In process revision manuscript                    9.    May KM, Hardwick KG: The spindle checkpoint. J Cell Sci 2006, <br />                                                                                    119(Pt 20):4139-4142. <br /> demonstrated genetic interaction spindle                     10.   Waters JC, Chen RH, Murray AW, Salmon ED: Localization of <br /> checkpoint genes hcp-1 C. elegans [38].                                     Mad2 kinetochores depends microtubule attachment, <br />                                                                                    tension. J Cell Biol 1998, 141(5):1181-1191. <br />                                                                              11.   Pinsky BA, Biggins S: The spindle checkpoint: tension versus <br /> Additional material                                                                attachment. Trends Cell Biol 2005, 15(9):486-493. <br />                                                                              12.   Skoufias DA, Andreassen PR, Lacroix FB, Wilson L, Margolis RL: <br />                                                                                    Mammalian mad2 bub1/bubR1 recognize distinct spin- <br />      Additional file 1                                                             dle-attachment kinetochore-tension checkpoints. Proc <br />                                                                                    Natl Acad Sci U S A 2001, 98(8):4492-4497. <br />      The Y54G9A.6 gene encodes putative BUB-3 protein. A multiple        13.   Reddy SK, Rape M, Margansky WA, Kirschner MW: Ubiquitination <br />      sequence alignment using ClustalW indicates Y54G9A.6 gene         anaphase-promoting complex drives spindle check- <br />      C. elegans encodes protein (BUB-3) homology spindle             point inactivation. Nature 2007, 446(7138):921-925. <br />      checkpoint protein BUB3. The amino acid identity C. ele-    14.   Diaz-Martinez LA, Yu H: Running treadmill: dynamic inhi- <br />      gans putative BUB-3 protein BUB3 proteins 44% H.             bition APC/C spindle checkpoint. Cell Div 2007, <br />                                                                                    2(1):23. <br />      sapiens, 43% D. melanogastor 19% S. cerevisiae. Identical <br />                                                                              15.   Fang G: Checkpoint protein BubR1 acts synergistically with <br />      amino acids black background shade.                                    Mad2 inhibit anaphase-promoting complex. Mol Biol Cell <br />      Click file                                                           2002, 13(3):755-766. <br />      [http://www.biomedcentral.com/content/supplementary/1747-               16.   Sudakin V, Chan GK, Yen TJ: Checkpoint inhibition APC/ <br />      1028-3-6-S1.tiff]                                                             C HeLa cells mediated complex BUBR1, BUB3, <br />                                                                                    CDC20, MAD2. J Cell Biol 2001, 154(5):925-936. <br />                                                                              17.   <span id='am-61' about='obo:NCBITaxon_9596' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-62' property="rdfs:label" content="Pan" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasAlternativeId" content="NCBITaxon:445518" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasAlternativeId" content="NCBITaxon:317028" datatype="xsd:string"></span><span id='am-67'  ></span><span id='am-68' property="oboInOwl:hasExactSynonym" content="chimpanzees" datatype="xsd:string"></span>Pan</span> J, Chen RH: Spindle checkpoint regulates Cdc20p stability <br />      Additional file 2                                                             Saccharomyces cerevisiae.                  Genes Dev 2004, <br />      Live cell imaging san-1(ok1580);hcp-1(RNAi);tbg-1::GFP;pie-                18(12):1439-1451. <br />      1::GFP::H2B embryos. A spinning disc confocal microscope analyze     18.   Kitagawa R, Rose AM: Components spindle-assembly <br />      chromosome segregation san-1(ok1580);hcp-1(RNAi) animals.              checkpoint essential Caenorhabditis elegans. Nat Cell <br />                                                                                    Biol 1999, 1(8):514-521. <br />      Images collected, processed using NIH Image imported      19.   Stein KK, Davis ES, Hays T, Golden A: Components Spindle <br />      Quick Time display. In embryo blastomeres         Assembly Checkpoint Regulate Anaphase-Promoting <br />      progress metaphase. One blastomeres displays normal            Complex During Meiosis Caenorhabditis elegans. Genetics <br />      mitotic progression (top metaphase blastomere) blast-         2007, 175(1):107-123. <br />      omere displays abnormal chromosome segregation (bottom metaphase        20.   Encalada SE, Willis J, Lyczak R, Bowerman B: A spindle checkpoint <br />      blastomere) leading anaphase bridging.                                     functions mitosis early Caenorhabditis elegans <br />                                                                                    embryo. Mol Biol Cell 2005, 16(3):1056-1070. <br />      Click file                                                     21.   Nystul TG, Goldmark JP, Padilla PA, Roth MB: Suspended anima- <br />      [http://www.biomedcentral.com/content/supplementary/1747-                     tion C. elegans requires spindle checkpoint. Science <br />      1028-3-6-S2.mov]                                                              2003, 302(5647):1038-1041. <br />                                                                              22.   Hajeri VA, Trejo J, Padilla PA: Characterization sub-nuclear <br />                                                                                    changes Caenorhabditis elegans embryos exposed to <br />                                                                                    brief, intermediate long-term anoxia analyze anoxia- <br />                                                                                    induced cell cycle arrest. BMC Cell Biol 2005, 6(1):47. <br />                                                                              23.   Moore LL, Morrison M, Roth MB: HCP-1, protein involved in <br /> Acknowledgements                                                                   chromosome segregation, localized centromere of <br /> We appreciate valuable input members Padilla lab. We           mitotic chromosomes Caenorhabditis elegans. J Cell Biol <br /> want thank Lon Turnbull Alex Mendenhall assistance advice.          1999, 147(3):471-480. <br /> We thank Caenorhabditis elegans Genetics Stock Center Okla-      24.   Rattner JB, Rao A, Fritzler MJ, <span id='am-99' about='obo:NCBITaxon_195854' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_8087 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_28738 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_7898 obo:NCBITaxon_195853 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-100' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-101'  ></span><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="Valencia" datatype="xsd:string"></span>Valencia</span> DW, Yen TJ: CENP-F a <br /> homa Knockout Consortium strains. This work supported                 .ca 400 kDa kinetochore protein exhibits cell-cycle <br />                                                                                    dependent localization.             Cell Motil Cytoskeleton 1993, <br /> grant National Institutes Health (NIH), National Institute          26(3):214-226. <br /> General Medical Sciences (R01GM069419) PAP NIH research            25.   Oegema K, Desai A, Rybina S, Kirkham M, Hyman AA: Functional <br /> supplemental grant promote diversity health-related research AMS.         analysis kinetochore assembly Caenorhabditis elegans. <br />                                                                                    J Cell Biol 2001, 153(6):1209-1226. <br />                                                                              26.   Buchwitz BJ, Ahmad K, Moore LL, Roth MB, Henikoff S: A histone- <br /> References                                                                         H3-like protein C. elegans. Nature 1999, 401(6753):547-548. <br /> 1.      Chi YH, Jeang KT: Aneuploidy cancer. J Cell Biochem 2007.        27.   Cheeseman IM, MacLeod I, Yates JR 3rd, Oegema K, Desai A: The <br /> 2.      Grady WM: Genomic instability colon cancer. Cancer                     CENP-F-like proteins HCP-1 HCP-2 target CLASP to <br />         Metastasis Rev 2004, 23(1-2):11-27.                                        kinetochores mediate chromosome segregation. Curr Biol <br /> 3.      Hardwick KG, Li R, Mistrot C, Chen RH, Dann P, Rudner A, Murray            2005, 15(8):771-777. <br />         AW: Lesions different spindle components activate            28.   van den Heuvel S: Cell-cycle regulation. WormBook, ed The C ele- <br />         spindle checkpoint budding yeast Saccharomyces                  gans Research Community 2005, http://www.wormbook.org:. <br />         cerevisiae. Genetics 1999, 152(2):509-518.                           29.   Padilla PA, Nystul TG, Zager RA, Johnson AC, Roth MB: Dephos- <br /> 4.      Chan GK, Jablonski SA, Sudakin V, Hittle JC, Yen TJ: Human                 phorylation Cell Cycle-regulated Proteins Correlates with <br />         BUBR1 mitotic checkpoint kinase monitors CENP- <br />  <br />  <br />  <br />                                                                                                                                      Page 17 18 <br />                                                                                                                 (page number citation purposes) <br />  Cell Division 2008, 3:6                                                                                      http://www.celldiv.com/content/3/1/6 <br />  <br />  <br />  <br />       Anoxia-induced Suspended Animation Caenorhabditis <br />       elegans. Mol Biol Cell 2002, 13(5):1473-1483. <br /> 30.   Hsu JY, Sun ZW, Li X, Reuben M, Tatchell K, Bishop DK, Grushcow <br />       JM, Brame CJ, Caldwell JA, Hunt DF, Lin R, Smith MM, Allis CD: <br />       Mitotic phosphorylation histone H3 governed Ipl1/ <br />       aurora kinase Glc7/PP1 phosphatase budding yeast <br />       <span id='am-89' about='obo:NCBITaxon_333870' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_195229 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41087 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_71193 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="Nematodes Berthold, 1827" datatype="xsd:string"></span><span id='am-91'  ></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="Nematodes" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>nematodes</span>. Cell 2000, 102(3):279-291. <br /> 31.   Lee KK, Gruenbaum Y, Spann P, Liu J, Wilson KL: C. elegans <br />       nuclear envelope proteins emerin, MAN1, lamin, nucle- <br />       oporins reveal unique timing nuclear envelope breakdown <br />       mitosis. Mol Biol Cell 2000, 11(9):3089-3099. <br /> 32.   Davis FM, Tsao TY, Fowler SK, Rao PN: Monoclonal antibodies to <br />       mitotic cells. Proc Natl Acad Sci U S A 1983, 80(10):2926-2930. <br /> 33.   Holt SV, Vergnolle MA, Hussein D, Wozniak MJ, Allan VJ, Taylor SS: <br />       Silencing Cenp-F weakens centromeric cohesion, prevents <br />       chromosome alignment activates spindle check- <br />       point. J Cell Sci 2005, 118(Pt 20):4889-4900. <br /> 34.   Yang Z, Guo J, Chen Q, Ding C, Du J, Zhu X: Silencing mitosin <br />       induces misaligned chromosomes, premature chromosome <br />       decondensation anaphase onset, mitotic cell <br />       death. Mol Cell Biol 2005, 25(10):4062-4074. <br /> 35.   Kamath RS, Fraser AG, Dong Y, Poulin G, Durbin R, Gotta M, Kana- <br />       pin A, Le Bot N, Moreno S, Sohrmann M <br /> </body></html>