<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Functional Differences Yeast Protein Disulfide Isomerases <br /> Per N&#248;rgaard, Vibeke Westphal, Christine Tachibana, Lene Als&#248;e, Bj&#248;rn Holst, Jakob R. Winther <br /> Department Yeast Genetics, Carlsberg Laboratory, DK-2500 Copenhagen Valby, Denmark <br />  <br />  <br />  <br />  <br /> Abstract. PDI1 essential gene encoding protein                         thermore, presence endogenous homologues with <br /> disulfide isomerase yeast. The <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24'  ></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span> cerevi-                       CXXC motif thioredoxin-like domain re- <br /> siae genome, however, contains nonessential                        quired suppression pdi1 deletion EUG1 (which <br /> genes homology PDI1: MPD1, MPD2, EUG1,                                contains CXXS active site motifs). <span id='am-33' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-34' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> underlines <br /> and EPS1. We investigated effects simulta-                        essentiality protein disulfide isomerase-catalyzed <br /> neous deletions genes. In cases,                          oxidation. Most mutant combinations defects in <br /> found ability PDI1 homologues restore                      carboxypeptidase Y folding glycan modifi- <br /> viability pdi1-deleted strain overexpressed                         cation. There are, however, significant effects ER- <br /> was dependent presence low endogenous lev-                          associated protein degradation various protein <br /> els homologues. This shows                        disulfide isomerase-deleted strains. <br /> that homologues functionally interchange- <br /> able. In fact, Mpd1p homologue capable                        Key words: protein disulfide isomerase &#8226; ER &#8226; protein <br /> carrying essential functions Pdi1p. Fur-                       folding &#8226; carboxypeptidase Y &#8226; <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-11'  ></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br />  <br />  <br />  <br /> Introduction <br /> An important aspect folding secretory proteins                   cycle reduced oxidized state. In lat- <br /> formation native disulfide bonds, process catalyzed                   ter state, cysteines form intramolecular disul- <br /> protein disulfide isomerase (PDI)1 (Givol et al., 1995). In                   fide bond principle enables PDI convert pair of <br /> the eukaryotic cell, endoplasmic reticulum or-                     sulfhydryl groups polypeptide substrate disul- <br /> ganelle folding secretory proteins occurs,                    fide bond. Recently, yeast protein Ero1p, been <br /> the location PDI chaperones. PDI-like                     shown essential thiol oxidation ER (Frand <br /> proteins identified domains se-                    Kaiser, 1998; Pollard et al., 1998), pathway has <br /> quence similarity thioredoxin, signal sequence,                      proposed oxidizing equivalents flow from <br /> the presence ER retention signal, (K/H)DEL. Yeast                       Ero1p Pdi1p substrate (Frand Kaiser, <br /> PDI (in study, PDI used general term                    1999). Glutathione, previously thought play <br /> a PDI-like protein, Pdi1p specifically refers                    major role oxidation ER, probably acts more <br /> protein encoded yeast gene PDI1), Pdi1p, contains                      reductant (Cuozzo Kaiser, 1999). <br /> two thioredoxin-like domains active site sequence                       In organisms, number genes encoding PDI-like <br /> motif CGHC majority eukaryotic                        proteins found. In <span id='am-49' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-50' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-64'  ></span>Escherichia coli</span>, proteins <br /> PDIs (Edman et al., 1985). The cysteines able                      thioredoxin-like domains convincingly been <br />                                                                               shown involved various aspects disulfide bond <br /> Address correspondence Jakob R. Winther, Carlsberg Laboratory, De-         formation shuffling periplasm (&#197;slund Beck- <br /> partment Yeast Genetics, Gamle Carlsberg Vej 10, DK-2500 Copenhagen <br /> Valby, Denmark. Tel.: 45 3327 5282. Fax: 45 3327 4766. E-mail: jrw@crc.dk     with, 1999). In eukaryotes, ER fulfills same <br />    Dr. Westphal&#8217;s present address The Burnham Institute, La Jolla, CA      role bacterial periplasm. The number putative <br /> 92037.                                                                        PDIs mammalian ER higher. However, spe- <br />    Dr. Tachibana&#8217;s present address University Washington, Depart-       cific function suggested limited number <br /> ment Genetics, Seattle, WA 98195-7360. <br />    Dr. Als&#248;e&#8217;s present address The Norwegian Radium Hospital, De- <br />                                                                               proteins (Ferrari S&#246;ling, 1999). One the <br /> partment Immunology, Montebello, 0310 Oslo, Norway.                        PDI-like proteins received attention is <br />    Dr. Holst&#8217;s present address Danish Veterinary Food Administra-      ERp57, shown involved ER <br /> tion, Department Food Safety Toxicology, DK-2860 S&#248;borg, Den-          quality control system. It interacts calnexin/calreticulin <br /> mark.                                                                         binding glycoprotein folding intermediates. This activity <br />    1 <br />      Abbreviations used paper: 5-<span id='am-1' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="rdfs:label" content="Foa" datatype="xsd:string"></span>FOA</span>, 5-fluoro-orotic acid; AMS, <br /> 4-acetamido-4&#11032;-maleimidylstilbene-2,2&#11032;-disulfonic acid; CPY, carboxypep-      does depend oxidoreductase properties <br /> tidase Y; ONPG, o-nitrophenyl-&#65533;?&#65533;-D-galactoside; PDI, protein disulfide         (Oliver et al., 1997). A specific function likely the <br /> isomerase.                                                                    pancreatic PDI, PDIp, known PDI-like <br />  <br />  <br />  <br /> &#63721; The Rockefeller University Press, 0021-9525/2001/02/553/10 $5.00 <br /> The Journal Cell Biology, Volume 152, Number 3, February 5, 2001 553&#8211;562 <br /> http://www.jcb.org/cgi/content/full/152/3/553                                 553 <br />                                                                                      Figure 1. Yeast contains genes encode <br />                                                                                     soluble ER proteins sequence homology to <br />                                                                                     Pdi1p. Open boxes indicate position ex- <br />                                                                                     tent thioredoxin-like domains. Active-site cys- <br />                                                                                     teines amino acid residues them <br />                                                                                     shown. The protein Eug1p &#11601;40% se- <br />                                                                                     quence identity Pdi1p contains, like <br />                                                                                     Pdi1p, regions sequence homology to <br />                                                                                     thioredoxin. MPD1 MPD2 encode proteins <br />                                                                                     smaller Pdi1p Eug1p con- <br />                                                                                     tain thioredoxin-like active site. The <br />                                                                                     protein Eps1p contains thioredoxin-like re- <br />                                                                                     gion transmembrane domain (the <br />                                                                                     closed box marked TMD). <br />  <br />  <br /> protein showing tissue--specific expression (Desilva et al.,     wild-type Pma1p transported plasma membrane, <br /> 1996). The domain organization varies differ-        allowing growth (Wang Chang, 1999). <br /> ent PDIs. For instance, stress-induced protein ERp72            In previous genetic analyses pdi1 complementation, <br /> has active thioredoxin-like proteins, compared     heterologous homologous, potential contribu- <br /> the Pdi1p (Dorner et al., 1990; Mazzarella et al.,        tions Pdi1p homologues disregarded. <br /> 1990). ERp72 genes encoding mammalian PDIs             With present work, wish determine role of <br /> are able suppress lethal effects deletion PDI1   homologues thiol oxidation and, particular, de- <br /> yeast (G&#252;nther et al., 1993; Laboissi&#232;re et al., 1995).          termine important enzymes PDI ac- <br />    In Saccharomyces cerevisiae, complete genome se-          tivity compromised. To this, constructed <br /> quence shows number putative PDIs            yeast strains simultaneously deleted of <br /> likely limited five, PDI1 gene       genes. The success approach illustrated by <br /> essential gene (Farquhar et al., 1991). Overexpression        observation mutant form Pdi1p, Pdi1pCGHS- <br /> any nonessential homologues EUG1, MPD1,                   CGHS, (this nomenclature adopted describing se- <br /> MPD2, EPS1, however, suppress           quence active sites protein; sequence <br /> lethality caused deletion PDI1 (Tachibana              NH2-terminal active site written first), which <br /> Stevens, 1992; Tachikawa et al., 1995, 1997; Wang            defective oxidation, dependent endogenous <br /> Chang, 1999). Eug1p shows overall sequence identity        homologue ability complement pdi1 deletion. <br /> &#11601;40% Pdi1p contains thioredoxin-like do-              We revealed differences functionality be- <br /> mains (Fig. 1). Unlike known PDIs, active sites       tween PDI-like proteins, MPD1 PDI1 <br /> Eug1p CXXS motif (Tachibana Stevens, 1992).           homologue that, overexpressed, able suppress a <br /> Thus, active sites Eug1p able form in-         strain deleted genes PDI1 <span id='am-30' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-31' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span>. <br /> tramolecular disulfide bonds. Therefore, Eug1p is <br /> unable carry oxidizing equivalents. MPD1 MPD2 <br /> encode proteins containing single thioredoxin-like        Materials Methods <br /> domain (Fig. 1). Apart thioredoxin domain, they <br /> share sequence homology protein           Media Materials <br /> known function. Overexpression EUG1, MPD1,                 Yeast cells grown standard YPD SC media (Sherman, 1991). <br /> MPD2 does fully complement pdi1 deletion             E. coli grown LB medium supplemented appropriate anti- <br />                                                                  biotic (Sambrook, 1989). Restriction enzymes, T4 DNA polymerase, T4 <br /> such strains reduced folding rate procarboxy-          polynucleotide kinase, T4 DNA ligase Promega. Oligonu- <br /> peptidase Y (proCPY) (Tachibana Stevens, 1992;               cleotides synthesized DNA Technology (Aarhus). Sequencing <br /> Tachikawa et al., 1995, 1997).                                   performed using Taq Dye DeoxyTM terminator cycle sequencing <br />    Recently, novel PDI-like protein named Eps1p            kit API 373A DNA Sequencer, Applied Biosystems. <br />                                                                  1,4-Dithiothreitol, 5-fluoro-orotic acid (5-FOA), o-nitrophenyl- <br /> characterized. It contains single thioredoxin-like domain <br />                                                                  &#65533;?&#65533;-D-galactoside (ONPG) Merck. N-acetyl-DL-phenylala- <br /> and localized ER membrane (Fig. 1). It        &#65533;?&#65533;-naphtyl ester calcofluor white Sigma-Aldrich. <br /> proposed involved ER quality control, deletion       4-acetamido-4&#11032;-maleimidylstilbene-2,2&#11032;-disulfonic acid (AMS) from <br /> of gene suppresses dominant-negative pma1 mutant.          Molecular Probes. [35S]Methionine DuPont. <br /> PMA1 encodes plasma membrane proton ATPase, which <br /> is essential growth. The mutant question prevents         Strains <span id='am-89' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>Plasmids</span> <br /> wild-type Pma1p transported plasma             All plasmids used study listed Table I. The yeast strains used <br /> membrane. In strain deleted eps1, mutant          study listed Table II. <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 152, 2001                    554 <br />  Table I. Plasmids Used                                                                    Table II. Yeast Strains <br />       Name                         Gene*             Marker              Source           Name             Chromosomal deletions           Plasmid            Source <br />  <br /> Plasmids used complementation&#8225;                                                                                                                      Tachibana and <br />      pCT38              PDI1                         URA3         This study              M 4129        &#8212;                                   &#8212;             Stevens, 1992 <br />      pCT58              EUG1CLHC-CIHC                TRP1         Holst et al., 1997      M 4426        eug1                                &#8212;           This study <br />      pBH1464            PDI1                         TRP1         Holst et al., 1997      M 4427        mpd1                                &#8212;           This study <br />      pBH1711            PDI1CRRC-SGHS                TRP1         Holst et al., 1997      M 4419        mpd2                                &#8212;           This study <br />      pBH1800            MPD1                         TRP1         This study              M 4360        eps1                                &#8212;           This study <br />      pBH1806            MPD2                         TRP1         This study              M 4174        pdi1                              pCT38         Tachibana and <br />      pBH1857            PDI1                         TRP1         Holst et al., 1997                                                                      Stevens, 1992 <br />      pBH1865            EUG1                         TRP1         This study              M 4741        ero1-1 pdi1                       pCT38         This study <br />      pBH1966            PDI1CGHS-CGHS                TRP1         Holst et al., 1997      M 4172        pdi1 eug1                         pCT38         This study <br />                                     Used                                              M 4496        pdi1 mpd1                         pCT38         This study <br /> Plasmids used deletion genes                                                       M 4418        pdi1 mpd2                         pCT38         This study <br />      pCT19                Deletion EUG1                        This study              M 4837        mpd1 mpd2 eug1 eps1                 &#8212;           This study <br />      pPN100               Deletion EPS1                        This study              M 4497        pdi1 eug1 mpd1                    pCT38         This study <br />      pPN330               Deletion MPD1                        This study              M 4417        pdi1 eug1 mpd2                    pCT38         This study <br />      pFA6-kanMX3          Deletion MPD2                        Wach et al., 1994       M 4508        pdi1 mpd1 mpd2                    pCT38         This study <br />                                                                                           M 4478        pdi1 eug1 mpd1 mpd2               pCT38         This study <br />                                    Promoter                                               M 4477        pdi1 eug1 mpd1 mpd2 eps1          pCT38         This study <br /> Plasmids containing promoter-lacZ fusions <br />      pPN379             MPD1 promoter                             This study              All strains isogenic W303 MAT&#65533;?&#65533; ade2-1 can1-100 ura3-1 leu2-3,112 trp1-1 <br />                                                                                           his3-11,15 (Kurjan, 1985). <br />      pPN382             MPD2 promoter                             This study <br />      pPN410             PDI1 promoter                             This study <br />      pPN575             EUG1 promoter                             This study <br />                                                                                           &#9004;pdi1 &#9004;eug1 [PDI1] strain (&#9004;pdi1 &#9004;eug1 [PDI1] denotes yeast strain de- <br /> *In cases, gene placed control PDI1 promoter.                     leted PDI1 EUG1 complemented PDI1 plasmid). <br /> &#8225; <br />  All self-replicating plasmids listed centromere based present      The EPS1 gene amplified PCR using primers oBH61 (5&#11032; <br /> a low copy number.                                                                        CGC GGA TCC GCG CAA GAA ATT CTA TCC AGG 3&#11032;) oBH62 <br />                                                                                           (5&#11032; TCC CCG CGG GGA TTT TTA TGG TAG GCG TGC 3&#11032;), the <br />                                                                                           PCR product cloned EcoRV site pBCSK&#11001; (Stratagene). <br />    Deletion PDI1 gene described Tachibana Stevens                       The SalI-XhoI EcoRV-EcoRV fragments plasmid containing the <br /> (1992). The EUG1 gene deleted plasmid pCT19, using                       EPS1 ORF inserted SalI SmaI sites, respectively, of <br /> &#8220;pop-in, pop-out&#65533;? method (Rothstein, 1991). A 4.8-kb EcoRI-SalI frag-                     YDp-L (Berben et al., 1991). The resulting plasmid, pPN100, digested <br /> ment EUG1 inserted URA3-based plasmid pRS306 (Sikor-                      SphI SalI, fragment containing LEU2 gene flanked <br /> ski Hieter, 1989). The HindIII-NruI fragment subsequently de-                     regions homologous EPS1 integrated chromosome <br /> leted, resulting plasmid pCT19, digested SnaBI,                         leading deletion base pairs 499&#8211;978 ORF EPS1. To confirm <br /> introduced yeast, integrated chromosome. Integrants                     correct integration, PCR performed using primers oPN006 (5&#11032; <br /> were spread plates containing 5-FOA, selects cells lost                AGC GGA AAT AGG TGC GAA 3&#11032;) oPN012 (5&#11032; AAC AGT ACC <br /> the URA3 gene. The desired pop-out results deletion HindIII-                  ACC GAA GTC 3&#11032;). The desired deletion EPS1 resulted PCR <br /> NruI region EUG1 locus. To confirm correct pop-in, pop-out,                    product &#11601;800 bp. <br /> PCR performed using primers oCT3 (5&#11032; CCA ACA GTA AAG                                 Of plasmids listed Table I, pBH1464, pBH1711, pBH1857, <br /> CAC TGT CC 3&#11032;) oCT5 (5&#11032; CCC CTG AAT CTA CTT CGG CG 3&#11032;).                               pBH1966, pCT58 described (Holst et al., 1997). <br /> Correct deletion EUG1 gene resulted PCR product &#11601;330 bp.                   pBH1865 constructed subcloning BamHI-NaeI fragment from <br />    The plasmid pPN196 derived pFA6-kanMX3 (Wach et al.,                           pCT40 (Tachibana Stevens, 1992) corresponding sites of <br /> 1994) exchanging kanr gene URA3 gene. Two DNA                             pBH1692 (Holst et al., 1997). pCT38 derived pRS316 contains <br /> fragments (the HincII-HincII Eco47III-PvuII fragment                         2.6-kb KpnI-XbaI fragment extending 879-bp upstream the <br /> pBH1800) sequence identity MPD1 gene inserted                       PDI1 gene 160-bp downstream PDI1. <br /> pPN196 URA3 gene, resulting plasmid pPN330.                       MPD1 MPD2 used construct plasmids complementation <br /> The fragment containing URA3 gene flanked direct repeats                       amplified PCR yeast chromosomal DNA. MPD1 ampli- <br /> and regions homologous MPD1 gene excised plasmid                      fied using oligonucleotides oBH57 (5&#11032; CGC GGA TCC GCG TCC <br /> pPN330 SspI, introduced yeast, integrated chromo-                  ACT TAA CAC AAT TAG G 3&#11032;) oBH58 (5&#11032; TCC CCG CGG GGA <br /> some. This leads deletion base pairs 101&#8211;387 MPD1 ORF.                       CTT ATT CTT ATG CCC C 3&#11032;) primers, introduced BamHI <br /> Cells lost URA3 selected plating integrants me-                   site upstream ORF SacII site downstream MPD1, respec- <br /> dium containing 5-FOA. To confirm correct deletion MPD1, PCR                       tively. After digestion BamHI SacII, PCR-amplified frag- <br /> was performed using oPN13 (5&#11032; CCT TGC GAG GCG TAT AAA 3&#11032;)                                 ment inserted corresponding sites pBH1692, resulting <br /> and oBH64 (5&#11032; CCT GAG TAC ACT TCA TTG GC 3&#11032;). The desired de-                             plasmid pBH1800. The sequence cloned ORF be <br /> letion gave PCR product &#11601;700 bp.                                                     identical published sequence, silent C T mutation at <br />    The vector pFA6-kanMX3 (Wach et al., 1994) used template                   nucleotide position &#11001;168 relative ATG <span id='am-43' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45'  ></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span>. MPD2 ampli- <br /> PCR reaction using primers oLA4 (5&#11032; CAC AGC GAG TCT AGT                               fied using oligonucleotides oBH59 (5&#11032; CGC GGA TCC GCG GCG <br /> GCA AGT ACG TCG GCA AAG TAA AAC ACA AAG GAG ATG                                           AGT CTA GTG CAA GTA CG 3&#11032;) oBH60 (5&#11032; TCC CCG CGG <br /> GGA TCC CCG GGT TAA TTA AGG CGC 3&#11032;) oLA5 (5&#11032; CGT                                      GGA CTT ATA TTG CGG CTA ACG 3&#11032;). The use oBH59 and <br /> CTG TAG TCG GTA TTC GTA AAG TAA AAG ACA GAG CGA                                           oBH60 introduced BamHI site upstream SacII site down- <br /> AGC TTA TGT TCA CTG ATA TCA TCG ATG AAT TCG AGC 3&#11032;).                                      stream, respectively, ORF. After digestion BamHI SacII, <br /> This resulted fragment containing kanr gene flanked direct                PCR-amplified fragment inserted corresponding sites <br /> repeats MPD2 fragments represented oligonucleotides. The                   pBH1692, giving pBH1806. The sequence cloned ORF found <br /> fragment integrated chromosome correct substitution                  identical published sequence. <br /> confirmed PCR using primers oLA1 (5&#11032; GGT CAA CGG GAA                                  The genes encoding Pdi1p homologues introduced plas- <br /> TGA GGG GAA 3&#11032;) oLA8 (5&#11032; CCC ATA TAA ATC AGC ATC C                                    mid shuffle procedure (Sikorski Boeke, 1991). The TRP1-containing <br /> 3&#11032;). The desired deletion MPD2 gene gave PCR product &#11601;1,480                   plasmids introduced desired strains carrying PDI1 a <br /> bp. The mating type MPD2 deleted strain changed &#65533;?&#65533;                    URA3-based plasmid. Clones lacking plasmid carrying PDI1 se- <br /> by crossing isogenic strain. This strain subsequently crossed              lected medium containing 5-FOA. <br />  <br />  <br />  <br />  <br /> N&#248;rgaard et al. Yeast PD1 Homologues                                                      555 <br />     pPN379 contains 1,500 base pairs MPD1 promoter fused lacZ.         mixed. 575 ml buffer Z/ONPG mixture added cells vor- <br /> A PCR product, using primers oPN25 (5&#11032; ACG CGC GGA                     texed 5 s. Another 575 ml buffer Z/ONPG mixture added and <br /> TCC ACA GTC TTA GGG AAG TAA CC 3&#11032;) oPN26 (5&#11032; ACG                            vortexed. After incubation 30&#11034;C, assay stopped addition of <br /> ACG GGA TCC ATT ATA TGT CAA ATT TTG TCT CTC C 3&#11032;),                          500 &#65533;?&#65533;l 1 M Na2CO3. The activity calculated described Guarente <br /> digested BamHI inserted corresponding site pFN8            (1983). Each result mean &#11006; SEM <span id='am-79' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81'  ></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> resulting in- <br /> (Nagawa Fink, 1985). The plasmid pPN382 contains 1,500 base pairs           dependent transformants measured times. <br /> of MPD2 promoter fused lacZ. A PCR product, using the <br /> primers oPN27 (5&#11032; ACG CGC GGA TCC GAA GCC TGA GTT CAT <br /> GCC CG 3&#11032;) oPN28 (5&#11032; ACG ACG GGA TCC ATC TCC TTT GTG                        Results <br /> TTT TAC 3&#11032;), digested BamHI cloned correspond- <br /> ing site pFN8. The primers oVW201 (5&#11032; CGT CAG CTG AGG ATC                    MPD1 Is Only PDI1 Homologue that, Its Own, Is <br /> CGG TAC CAC CCA GG 3&#11032;) oVW202 (5&#11032; TAG CTG ACT GGG <br /> ATC CTT CAT AAC GGG 3&#11032;) used make plasmid pPN410,                       Able Ensure Cell Viability <br /> which contains 880 base pairs PDI1 promoter fused lacZ            From combined vivo silico analysis, genes <br /> gene. The plasmid pPN575, containing 268 base pairs EUG1 pro- <br /> moter, cloning PCR product primers oPN48 (5&#11032;             identified members obvious candi- <br /> ACG CGC GGA TCC AGT CAT CCT CTG ATT AAC TTT CC 3&#11032;)                          dates PDI family: PDI1, EUG1, MPD1, MPD2, <br /> oPN49 (5&#11032; ACG ACG GGA TCC ATG GTA TTA TAT GAG CGG G                             EPS1 (Tachibana Stevens, 1992; Tachikawa et <br /> 3&#11032;). The BamHI-digested PCR product cloned BamHI-digested              al., 1995, 1997; Wang Chang, 1999). Of genes, <br /> pFN8. <br />                                                                                 PDI1 stands essential gene. All <br />                                                                                 nonessential PDI1 homologues shown be <br /> Pulse Labeling Immunoprecipitation <br />                                                                                 multicopy suppressors pdi1 deletion. Thus, these <br /> The pulse-chase immunoprecipitation experiments carried            proteins, principle, ability carry the <br /> essentially described Horazdovsky Emr (1993). Yeast strains <br />                                                                                 minimum Pdi1p function necessary cell survival, but <br /> that &#9004;prc1 PRC1 integration wild-type allele of <br /> PRC1 using plasmid pIPRC. This plasmid EcoRI shrink                present low amounts and/or not <br /> pTSY3 (Stevens et al., 1986). Digestion SacI facilitates integration       necessary catalytic potency produced at <br /> into chromosome. The transformed yeast strains tested        normal levels. <br /> PRC1&#11001; phenotype testing CPY activity using N-acetyl-DL-phe-              We interested studying yeast eu- <br /> nylalanine &#65533;?&#65533;-naphtyl ester overlay assay (Jones, 1977). Strains made <br /> &#9004;prc1::HIS3 using plasmid described Blachly-Dyson Stevens <br />                                                                                 karyotes equipped array PDI-like pro- <br /> (1987). The prc1-1 allele encoding CPY* introduced CEN-              teins. More specifically, wished uncover putative <br /> URA3 plasmid pHIT341, gift R. Hitt D. Wolf (Institut f&#252;r Bio-        synthetic phenotypes systematic disruption several <br /> chemie, Universit&#228;t Stuttgart, Stuttgart, Germany).                             PDI1 homologues cell, investigate <br />     For Western blots, cells (&#11601;4 OD600 cells sample) col- <br />                                                                                 homologue&#8217;s effectiveness suppression pdi1 <br /> lected centrifugation resuspended 5% (wt/vol) TCA. After cen- <br /> trifugation, pellet washed twice 100% acetone subse-             deletion expressed roughly levels as <br /> quently resuspended 100 &#65533;?&#65533;l lysis buffer (100 mM <span id='am-84' about='obo:NCBITaxon_258862' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_9436 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-85' property="rdfs:label" content="Mops" datatype="xsd:string"></span><span id='am-86'  ></span><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>MOPS</span>, pH 7.0, 2%          Pdi1p. To so, constructed null mutations PDI1 <br /> SDS) containing 20 mM AMS modify free thiols. Also, protease in-           EUG1, MPD1, MPD2, EPS1, combina- <br /> hibitor cocktail (Boehringer) included, according supplier&#8217;s in-     tions. The important result experiment that <br /> structions. Approximately 50 &#65533;?&#65533;l acid-washed glass beads (212&#8211;300 &#65533;?&#65533;m; <br /> Sigma-Aldrich) added sample. The samples vor- <br />                                                                                 simultaneous deletion Pdi1p homologues <br /> texed 30 min Eppendorf Mixer 5432 room temperature,            causes obvious growth defect synthetic complete <br /> incubated 30 min room temperature complete sulf-        medium normal laboratory conditions. Thus, no <br /> hydryl modification. CPY subsequently immunoprecipitated de-             synthetic phenotypes displayed long PDI1 is <br /> scribed Horazdovsky Emr (1993). Horseradish peroxidase blots <br />                                                                                 present. We placed MPD1, MPD2, EUG1, and <br /> were developed using ECL&#11001; substrate (Amersham Pharmacia Biotech), <br /> and scanned using Storm image analysis (Molecular Dynamics).             EPS1 open reading frames control PDI1 <br />                                                                                 promoter low-copy TRP1-based plasmid. These plas- <br /> Growth Phenotypes                                                               mids introduced &#9004;pdi1 strain carrying PDI1 <br />                                                                                 URA3-based plasmid. Growth 5-FOA, se- <br /> Cultures grown overnight SC &#11001; 5&#11003;Leu &#11001; 5&#11003;Ade diluted a <br /> concentration &#11601;5,000 cells/ml (5&#11003;Leu 5&#11003;Ade denotes times          lects loss PDI1-containing URA3-based plas- <br /> the normal SC concentration compound). 20 &#65533;?&#65533;l applied             mid, used measure ability gene sup- <br /> SC &#11001; 5&#11003;Leu &#11001; 5&#11003;Ade plates. The plates incubated 2 d 30&#11034;C            press PDI1 deletion. The result Eug1p, Mpd1p, <br /> before photographed. The ability yeast strains grow          Mpd2p able replace Pdi1p produced <br /> presence DTT determined spotting cells way onto <br /> freshly prepared SC &#11001; 5&#11003;Leu &#11001; 5&#11003;Ade plates buffered pH 5 con- <br /> taining 0&#8211;2.5 mM DTT. The plates subsequently incubated CO2 <br /> atmosphere minimize air oxidation DTT. After 2 d 30&#11034;C, the <br /> plates photographed. <br />  <br /> Measurement &#65533;?&#65533;-Galactosidase Activity <br /> &#65533;?&#65533;-galactosidase assays performed using protocol modified from <br /> Guarente (1983). For preparation cell suspension, overnight cul- <br /> ture diluted OD600 &#11601;0.1 total volume 25 ml 100-ml <br /> flasks. The culture grown 30&#11034;C OD600 &#11601; 0.3. An appropri- <br /> ate culture harvested centrifugation pellet was <br /> resuspended 450 &#65533;?&#65533;l buffer Z (Guarente, 1983) containing 0.01% Triton         Figure 2. Growth PDI-deficient yeast strains overexpressing <br /> X-100. The solution transferred 2-ml Eppendorf tube old <br />                                                                                 PDI1 homologues. Cell cultures diluted, applied SC &#11001; <br /> tube washed 450 &#65533;?&#65533;l buffer Z &#11001; Triton X-100. The cell <br /> suspension vortexed 10 s immediately frozen dry ice/etha-        5&#11003;Leu &#11001; 5&#11003;Ade plates incubated 30&#11034;C 2 d. The wild- <br /> nol storage overnight &#11002;80&#11034;C. For assay, cell suspension       type growth rate (defined reciprocal doubling time) was <br /> was thawed, centrifuged 10,000 g 10 min collect cells. For   &#11601;0.7 h&#11002;1 SC medium, extreme reduction growth <br /> assay, 950 &#65533;?&#65533;l buffer Z 200 &#65533;?&#65533;l ONPG (from 4 mg/ml stock)                rate seen &#9004;pdi1 [EUG1] strain (&#11601;0.3 h&#11002;1; data shown). <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 152, 2001                                   556 <br />  Table III. Rescue PDI Deficiencies Overexpression PDI Homologues <br />                                                                                                 Yeast strain <br />  <br />                                                                                                                                                                         &#9004;pdi1 <br />                                                                                                                                                      &#9004;pdi1              &#9004;eug1 <br />                                                                                                  &#9004;pdi1            &#9004;pdi1             &#9004;pdi1            &#9004;eug1              &#9004;mpd1 <br />                                             &#9004;pdi1            &#9004;pdi1             &#9004;pdi1             &#9004;eug1            &#9004;eug1             &#9004;mpd1            &#9004;mpd1              &#9004;mpd2 <br /> Introduced gene*            &#9004;pdi1           &#9004;eug1            &#9004;mpd1             &#9004;mpd2             &#9004;mpd1            &#9004;mpd2             &#9004;mpd2            &#9004;mpd2              &#9004;eps1 <br />  <br /> PDI1                        &#11001;&#11001;               &#11001;&#11001;               &#11001;&#11001;                &#11001;&#11001;                &#11001;&#11001;               &#11001;&#11001;                &#11001;&#11001;                &#11001;&#11001;                   &#11001;&#11001; <br /> EUG1                         &#11001;                &#11001;                &#11002;                 &#11002;                 &#11002;                &#11002;                 &#11002;                 &#11002;                    &#11002; <br /> MPD1                         &#11001;                &#11001;                &#11001;                 &#11001;                 &#11001;                &#11001;                 &#11001;                 &#11001;                    &#11001; <br /> MPD2                         &#11001;                &#11001;                &#11002;                 &#11001;                 &#11002;                &#11001;                 &#11002;                 &#11002;                    &#11002; <br /> PDI1CRRC-SGHS               &#11001;&#11001;               &#11001;&#11001;               &#11001;&#11001;                &#11001;&#11001;                &#11001;&#11001;               &#11001;&#11001;                &#11001;&#11001;                &#11001;&#11001;                   &#11001;&#11001; <br /> EUG1CLHC-CIHC               &#11001;&#11001;               &#11001;&#11001;               &#11001;&#11001;                &#11001;&#11001;                &#11001;&#11001;               &#11001;&#11001;                &#11001;&#11001;                &#11001;&#11001;                   &#11001;&#11001; <br /> PDI1CGHS-CGHS                &#11001;                &#11001;                &#11001;                 &#11002;                 &#11001;                &#11002;                 &#11002;                 &#11002;                    &#11002; <br /> &#11001;&#11001;, viable strain near wild-type growth. &#11001;, viable strain reduced growth rate compared wild-type strain. &#11002;, viable strain obtained. Overexpres- <br /> sion PDI1 MPD1 rescues strains, overexpression MPD2 rescues strains containing MPD1. Furthermore, rescue overexpression EUG1 requires the <br /> presence MPD1 MPD2. The gene encoding PDI-like mutant form Eug1p, EUG1CLHC-CIHC, rescues strains, gene encoding Eug1p-like form PDI, <br /> PDI1CGHS-CGHS, able rescue MPD2 deleted. <br /> *The gene control PDI1 promoter introduced low-copy number plasmid. <br />  <br />  <br /> to roughly level. However, effectiveness                                expressed 2 &#65533;?&#65533;m plasmid (Wang Chang, <br /> doing varies greatly. A &#9004;pdi1 strain complemented                                   1999). Since Eps1p specialized role ER <br /> MPD1 control PDI1 promoter (&#9004;pdi1                                            quality control, did pursue investigation this <br /> [MPD1]) showed small reduction growth rate com-                                      protein. <br /> pared wild type, &#9004;pdi1 [MPD2]                                         The importance oxidizing CXXC motif was <br /> &#9004;pdi1 [EUG1] strains showed larger reductions growth                                   shown specific EUG1 PDI1 mutants. Mutation of <br /> rates (Fig. 2). We wanted ability                                 active sites Eug1p CXXC converts Eug1p a <br /> of plasmids complement pdi1 deletion                                     Pdi1p-like enzyme judged improved ability <br /> dependent chromosomal copies PDI1 ho-                                     rescue &#9004;pdi1 strain (Holst et al., 1997). We that <br /> mologues, synthetic effects revealed.                                 overexpression corresponding gene EUG1CLHC-CIHC <br /> This accomplished shuffling various plasmids                                   able rescue simultaneous deletion PDI1, <br /> containing homologues strains containing pdi1                                  MPD1, MPD2, EUG1, EPS1 (Table III). Conversely, <br /> deletion combination deletions <span id='am-38' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40'  ></span><span id='am-41' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>homo</span>-                                       mutation active sites Pdi1p Eug1p- <br /> logues. The pattern viability array strains did                             like sequence CXXS restricted ability gene to <br /> reveal functional differences MPD1, MPD2,                                     rescue pdi1 deletion situations MPD2 was <br /> and EUG1 gene products (Table III). Deletion EUG1                                      present (Table III). This reflect higher oxidative <br /> affected viability growth rate                                  capacity Mpd2p Mpd1p. <br /> yeast strains investigated. Overexpression EUG1,                                       The protein Ero1p required ER oxidation pro- <br /> the hand, able complement pdi1 deletion                                  tein thiols (Frand Kaiser, 1998; Pollard et al., 1998). It <br /> only MPD1 MPD2 present. Differences                                      shown mutant form Mpd2p con- <br /> in functionality Mpd1p Mpd2p seen,                                     taining CQHC-to-CQHA active site mutation overex- <br /> overexpression MPD2 longer able rescue                                     pressed, mixed disulfides Ero1p trapped <br /> pdi1 deletion MPD1 deleted, overex-                                     (Frand Kaiser, 1999). We wanted test whether <br /> pression MPD1 rescued strains. Not expres-                                    Ero1p involved delivering oxidizing equivalents not <br /> sion MPD2 high-copy plasmid rescued pdi1                                      Pdi1p Mpd2p, Mpd1. To address <br /> deletion absence MPD1. The difference could,                                 question, crossed pdi1-deleted strain the <br /> principle, reflect low steady state levels Mpd2p                                  temperature-sensitive ero1-1 mutant. If redox equivalents <br /> compared Mpd1p, caused differences protein                                     passed Pdi1p, second pathway was <br /> stability. To test this, constructed plasmids containing                               used homologues overexpressed, no <br /> MPD1 MPD2, sequence encoding                                           synthetic phenotype observed. However, the <br /> COOH-terminal myc-tag followed ER retention                                        double mutant, homologues able rescue <br /> signal HDEL added. Both constructs                                     pdi1 deletion expressed control the <br /> phenocopy wild-type genes ability comple-                                 PDI1 promoter. The simplest explanation result is <br /> ment pdi1-deleted strain. A Western blot extracts                                    Ero1p able transfer oxidizing equivalents to <br /> from strains probed myc-antibodies shows                                  homologues. <br /> Mpd2p-myc present roughly level as <br /> Mpd1p-myc tagged genes expressed                                      Table IV. Expression Levels <br /> the control PDI1-promoter (data shown). This <br />                                                                                           Promoter                                                    &#65533;?&#65533;-Galactosidase activity <br /> indicates strain completely depleted other <br /> Pdi1p-like proteins, Mpd1p homologue able                                                                                                     Miller units <br /> provide Pdi1p functions required survival.                                        MPD2                                                          0.0466 &#11006; 0.0038 <br />    EPS1 able rescue pdi1-deleted strain                                 MPD1                                                           0.972 &#11006; 0.076 <br /> expressed control PDI1-promoter. It pre-                                 EUG1                                                           1.173 &#11006; 0.056 <br />                                                                                           PDI1                                                           26.28 &#11006; 2.34 <br /> viously shown, however, complement, <br />  <br />  <br />  <br /> N&#248;rgaard et al. Yeast PD1 Homologues                                                      557 <br />                                                                     central residues active site CGHC sequence fur- <br />                                                                    thermore increased DTT sensitivity (Holst et al., <br />                                                                    1997). We interested determining extent <br />                                                                    oxidizing apparatus various deletion mutants <br />                                                                    affected. A wild-type strain able grow pres- <br />                                                                    ence 5 mM DTT normal conditions. To <br />                                                                    amplify possible differences DTT sensitivity re- <br />                                                                    spect growth, used mutant PDI1CRRC-SGHS, which <br />                                                                    previously shown cause increased sensitivity <br />                                                                    DTT (Holst et al., 1997). We deletion <br />                                                                    genes encoding PDI homologues increased the <br />                                                                    DTT sensitivity strain rescued mutant gene <br />                                                                    PDI1CRRC-SGHS, clearly seen difference in <br />                                                                    growth 0.5 1.0 mM DTT (Fig. 3). <br />                                                                       The rate intracellular folding proCPY mon- <br />                                                                    itored, folded proteins allowed exit ER <br />                                                                    (Gething et al., 1986). Since proCPY converted forms <br />                                                                    different molecular mass traverses different <br />                                                                    compartments secretory pathway, rate ER <br />                                                                    exit determined (Stevens et al., 1982). Upon trans- <br />                                                                    location ER, protein receives core N-gly- <br /> Figure 3. DTT sensitivity PDI-deficient yeast strains overex-   cosyl residues, resulting 67-kD ER form proCPY <br /> pressing PDI1 homologues. Cultures diluted applied     called p1. The carbohydrate structure modified <br /> SC &#11001; 5&#11003;Leu &#11001; 5&#11003;Ade plates buffered pH 5 supple-             Golgi, giving 69-kD p2 form. Finally, when <br /> mented different concentrations DTT. To prevent oxida-     proCPY reaches vacuole, propeptide cleaved <br /> tion DTT molecular oxygen, plates freshly      off, resulting 63-kD mature, active form CPY <br /> and incubated 30&#11034;C CO2 atmosphere.                         (Stevens et al., 1982). <br />                                                                       We followed maturation proCPY CPY in <br />                                                                    strains depleted PDIs. Cells labeled [35S]me- <br /> The PDI1 Homologues Are Expressed Much Lower                  thionine 15 min chased 0, 15, 60 min. After <br /> Level PDI1                                                    immunoprecipitation CPY antibody, samples <br />                                                                    subjected SDS-PAGE analyzed using <span id='am-99' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-100' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-101'  ></span><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Phos</span>- <br /> Since PDI1 homologues able complement pdi1          phorImager. As seen Fig. 4 A, rate proCPY fold- <br /> deletion expressed control PDI1 pro-        ing strongly dependent gene used to <br /> moter, normal levels likely substantially lower    rescue pdi1 deletion. The half-time proCPY mat- <br /> than PDI1 gene. To estimate expression levels,      uration PDI1-complemented strain 5&#8211;10 min, <br /> measured &#65533;?&#65533;-galactosidase activity reporter con-           half time increased &#11601;30 min strains res- <br /> structs promoter regions genes ques-    cued overexpression MPD1. When pdi1 deletion <br /> tion fused lacZ. In wild-type yeast,      rescued overexpression EUG1, proCPY mat- <br /> &#65533;?&#65533;-galactosidase activity obtained MPD1 EUG1           uration arrested, accompanied accumulation <br /> reporters 4&#8211;5% activity PDI1-lacZ re-         p1 ER form proCPY. This effect more <br /> porter, MPD2-lacZ reporter 0.2%         pronounced strain rescued overexpression of <br /> (Table IV). This indicates homologous proteins        MPD2. Deletion genes encoding various homo- <br /> present lower concentration Pdi1p.                  logues no, little, effect rate proCPY <br />    The question posed inability           maturation long strains complemented by <br /> MPD1 complement pdi1 deletion nor-         PDI1 (Fig. 4 B), indicating Pdi1p homologues do <br /> mally low abundance gene product         normally make significant contribution proCPY <br /> is consequence reduced catalytic potency compared             folding. However, homologues recognize proCPY <br /> with Pdi1p. To test this, placed PDI1 control          substrate absence Pdi1p, maturation <br /> the weak MPD1 promoter. Interestingly, plasmid          proCPY takes place, reduced rate. <br /> able rescue strain simultaneously deleted pdi1, eug1,        As Eug1p able carry oxidation <br /> mpd1, mpd2, eps1 (data shown). This shows       reactions, oxidation efficient &#9004;pdi1 <br /> not lower expression levels PDI1 homo-             strain complemented EUG1. We tested assumption <br /> logues make incapable fulfilling PDI function.        looking redox state proCPY steady <br />                                                                    state. Extracts presence absence of <br /> The Effects Mutations PDI1 Are Enhanced                   thiol modifying agent AMS. AMS modifies free sulfhy- <br /> Strains Lacking Pdi1p Homologues                                   dryl groups only, modification causes mobility <br /> DTT sensitivity closely linked effectiveness      shift SDS-PAGE bulkiness AMS <br /> oxidizing apparatus ER (J&#228;ms&#228; et al., 1994; Frand           group. The cell extracts separated SDS-PAGE, <br /> and Kaiser, 1998). DTT penetrates ER interferes            blotted nitrocellulose membrane, probed with <br /> with formation disulfide bonds nascent proteins          CPY antibodies. As seen Fig. 5 A, proCPY wild- <br /> (Braakman et al., 1991). Mutants Pdi1p mutated           type strain modified AMS, treatment the <br />  <br />  <br />  <br />  <br /> The Journal Cell Biology, Volume 152, 2001                      558 <br />                                                                                          Figure 4. Intracellular folding and <br />                                                                                         maturation CPY followed immu- <br />                                                                                         noprecipitation pulse-labeled <br />                                                                                         cells separated SDS-PAGE. Cells <br />                                                                                         labeled 15 min chased for <br />                                                                                         0, 15, 60 min. (A) A &#9004;pdi1 strain <br />                                                                                         rescued indicated genes. (B) <br />                                                                                         Comparison &#9004;pdi1 &#9004;pdi1 <br />                                                                                         &#9004;mpd1 &#9004;mpd2 &#9004;eug1 &#9004;eps1 strain con- <br />                                                                                         taining plasmids PDI1 MPD1. <br />  <br />  <br /> cells DTT AMS modification causes              extremely sensitive calcofluor white (data <br /> large mobility shift gel. This corresponds modifi-     shown). <br /> cation cysteines fully reduced proCPY. <br /> proCPY &#9004;pdi1 [EUG1] strain partially AMS               Pdi1p Homologues Play No Significant Role in <br /> modifiable, suggesting significant portion         ER-associated Degradation <br /> proCPY completely oxidized steady state. This in-      PDI-like proteins implicated quality control <br /> dicates oxidation fact compromised EUG1        ER-associated degradation misfolded ER proteins <br /> complemented strain.                                             (Oliver et al., 1997; Gillece et al., 1999; Wang Chang, <br />                                                                  1999). In yeast, Pdi1p shown play role deg- <br /> Pdi1p Depletion Affects Protein Glycosylation                    radation ubiquitin-proteasome pathway (ERAD) <br /> In &#9004;pdi1 [EUG1], &#9004;pdi1 [MPD2], &#9004;pdi1             model substrates: unglycosylated mutant form of <br /> &#9004;eug1 &#9004;mpd1 &#9004;mpd2 &#9004;eps1 [MPD1] strains, p1               prepro-&#65533;?&#65533;-factor CPY single amino acid muta- <br /> the p2 forms proCPY fully separated (Fig. 4, A        tion, CPY* (Gillece et al., 1999). The degradation the <br /> and B). The p1 form strains appears migrate          appeared independent disulfide chemistry, <br /> slower wild type. Also, mature form CPY              involved putative peptide-binding domain of <br /> shows faster mobility wild-type strain. That       Pdi1p. CPY* degradation hand appeared to <br /> these differences mobility originate glycosylation       dependent Pdi1p activity, binding do- <br /> defects confirmed disappearance treat-          main. As suggested additional compo- <br /> ment samples endoglycosidase H (data             nents involved, wished investigate whether <br /> shown). Since p1 form migrates slower (most clearly          Pdi1p homologues played role. We that <br /> seen Fig. 5 B), defect originate ER.         homologues important efficient degrada- <br />    The compound calcofluor white interferes cell wall       tion CPY*, demonstrated fact degra- <br /> assembly, hypersensitivity compound         dation rate reduced &#9004;mpd1 &#9004;mpd2 &#9004;eug1 <br /> therefore used indicate defects cell-wall biogen-     &#9004;eps1 strain (Fig. 6 A). Surprisingly, however, substitution <br /> esis. Several mutants showing hypersensitivity cal-      PDI1 MPD1, MPD2, EUG1 signifi- <br /> cofluor white affected enzymes involved gly-       effect CPY* degradation (Fig. 6 B). To rule out <br /> cosylation apparatus (Ram et al., 1994; Lussier et al., 1997).   possible inconsistencies compared experi- <br /> Consistent observed glycosylation defects       mental procedures used Gillece et al. (1999), tested <br /> finding &#9004;pdi1 [EUG1], &#9004;pdi1 [MPD2],             CPY* degradation &#9004;pdi1 [PDI1SGHS-CGHC] strain <br /> the &#9004;pdi1 &#9004;eug1 &#9004;mpd1 &#9004;mpd2 &#9004;eps1 [MPD1] strains             used Gillece et al. (1999). Consistent find- <br />  <br />  <br />  <br />  <br /> N&#248;rgaard et al. Yeast PD1 Homologues                             559 <br />                                                                    type cells able grow CGHC-sites Pdi1p are <br />                                                                   exchanged CGHS, case gene en- <br />                                                                   coding auxiliary CXXC-containing PDI MPD2 de- <br />                                                                   leted. Similarly, EUG1 rescues pdi1-deleted strain if <br />                                                                   MPD1 MPD2 present, Eug1p mu- <br />                                                                   tant form containing CXXC active sites circumvent <br />                                                                   requirement CXXC-containing Pdi1p homo- <br />                                                                   logues. This strengthens notion context of <br />                                                                   motif lesser importance (Chivers et al., <br />                                                                   1996). It remarkable yeast evolved Pdi1p ho- <br />                                                                   mologue, Eug1p, cysteine residue each <br />                                                                   active site, rendering incapable forming internal <br />                                                                   disulfide bond required oxidation, suited for <br />                                                                   shuffling reactions. The finding steady state <br />                                                                   proCPY pool &#9004;pdi1 [EUG1] strain partially reduced <br />                                                                   (Fig. 5 A) clearly demonstrates decreased oxidative ca- <br />                                                                   pacity strain. Frand Kaiser (1999) propose that <br />                                                                   oxidizing equivalents flow Pdi1p Mpd2p pro- <br />                                                                   tein substrates, shown cross-linking experiments us- <br />                                                                   ing CXXS/A mutant forms. As &#9004;pdi1 ero1-1 strain not <br />                                                                   rescued homologues, de- <br />                                                                   pendent Ero1p pathway. A weak allele ERO1 <br />                                                                   affect ability homologues comple- <br />                                                                   ment used different pathway. This consistent <br />                                                                   model Ero1p transfers oxidizing <br />                                                                   equivalents Pdi1p Mpd2p Mpd1p. <br />                                                                      As pdi1 deletion lethal, question arises what <br />                                                                   essential cellular process processes compromised by <br /> Figure 5. Western blot showing steady state pool CPY. <br />                                                                   Pdi1p depletion, or, case, substitution Pdi1p with <br /> (A) A wild-type &#9004;pdi1 [EUG1] strain. Cells lysed in <br /> the presence absence 20 mM AMS. In lane, the <br />                                                                   homologues. We mature CPY some <br /> wild type treated 40 mM DTT lysis AMS         strains migrates differently CPY wild-type <br /> modification. (B) Western blot showing glycosylation defects   strain, indicating changed outer chain glycosylation <br /> CPY strain Mpd1p PDI source. A wild-type    (Figs. 4 5 B). Defects ER folding apparatus <br /> a &#9004;pdi1 &#9004;eug1 &#9004;mpd1 &#9004;mpd2 &#9004;eps1 [MPD1] strain differ-        rise diminished steady state levels en- <br /> ences mobility various forms CPY.                zymes involved glycosylation. Furthermore, cells <br />                                                                   hypersensitivity calcofluor white, <span id='am-94' about='obo:NCBITaxon_237249' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_30083 obo:NCBITaxon_33154 obo:NCBITaxon_35567 obo:NCBITaxon_35568 obo:NCBITaxon_33355 obo:NCBITaxon_236659 obo:NCBITaxon_33354 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_7524 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_33351 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_33349 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33345 obo:NCBITaxon_33347 obo:NCBITaxon_33342 obo:NCBITaxon_173680 obo:NCBITaxon_50557 obo:NCBITaxon_33343'><span id='am-95'  ></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="Proba" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>proba</span>- <br />                                                                   bly cell wall defects. Consistent this, mor- <br /> ings, rate CPY* degradation strain sig-        phology cells changed consequence Pdi1p <br /> nificantly reduced (data shown). Based ob-           depletion. It prominent &#9004;pdi1 [EUG1] <br /> servations, conclude homologues              strain, odd-shaped cells encountered. <br /> substitute putative role played Pdi1p deg-         There indications yeast higher <br /> radation substrate.                                       eukaryotes PDI involved quality control <br />                                                                   degradation misfolded secretory proteins. How- <br />                                                                   ever, effect deleting homologues the <br /> Discussion                                                        presence wild-type copy PDI1. Likewise, is <br /> When cysteine residues enter lumen ER          reduction rate CPY* degradation substitu- <br /> of translocated polypeptide chain, encounter             tion Pdi1p homologues Mpd1p, Mpd2p, <br /> change redox environment ultimately promotes          Eug1p (Fig. 6). It appears Pdi1p <br /> formation intrachain and/or interchain disulfide bonds.        homologues play unique, irreplaceable role in <br /> In generally accepted view, PDI involved re-        degradation CPY*. This surprising view the <br /> actions oxidative folding protein: (a) in-    previous observation substitution Pdi1p a <br /> troduction disulfide bonds polypeptide substrate       Pdi1pSGHS-CGHC mutant form results significant stabiliza- <br /> transfer disulfide bond PDI active           tion CPY* (Gillece et al., 1999). At present time, we <br /> site cysteines (i.e., oxidized state) substrate,    rationalize observations. However, possi- <br /> and (b) rearrangement incorrect disulfide bonds            ble different degradation pathway invoked under <br /> that likely form complex folding reactions.           conditions severe PDI depletion. <br />    The discovery Ero1p important oxidative fold-           Mpd1p ranks Pdi1p-like homologues, <br /> ing suggests pathway flow oxidizing equiva-          act sole PDI &#9004;pdi1 &#9004;eug1 &#9004;mpd1 <br /> lents unknown donors Ero1p, CXXC-              &#9004;mpd2 &#9004;eps1 strain and, conditions, support <br /> motif Pdi1p polypeptide substrate           normal growth. However, observation that <br /> (Frand Kaiser, 1999). The present work demon-             expression PDI1 weak MPD1 promoter suf- <br /> strated cell absolute requirement          ficient survival, conjunction multiple de- <br /> least CXXC-containing PDI. While wild-              fects seen pdi1-deleted strains rescued overex- <br />  <br />  <br /> The Journal Cell Biology, Volume 152, 2001                     560 <br />                                                                                                                             Figure 6. Degradation CPY*. <br />                                                                                                                            Cells labeled 10 min <br />                                                                                                                            chased 0, 30, 60, 90 <br />                                                                                                                            min. The results consistent <br />                                                                                                                            independent ex- <br />                                                                                                                            periments. All strains &#9004;prc1 <br />                                                                                                                            contain prc1-1 allele on <br />                                                                                                                            plasmid pHIT341. <br />  <br />  <br />  <br /> pressed homologues, demonstrates superior catalytic                            Blachly-Dyson, E., T.H. Stevens. 1987. Yeast carboxypeptidase Y be <br />                                                                                       translocated glycosylated amino-terminal signal sequence. J. <br /> potency Pdi1p. In absence Pdi1p, Mpd1p                                 Cell Biol. 104:1183&#8211;1191. <br /> and/or Mpd2p appear important contributors pro-                           Braakman, I., H. Hoover-Litty, K.R. Wagner, A. Helenius. 1991. Folding <br /> tein oxidation ER. However, MPD1 MPD2                                   influenza hemagglutinin endoplasmic reticulum. J. Cell Biol. 114: <br />                                                                                       401&#8211;411. <br /> not equally suited carry protein oxida-                           Chivers, P.T., M.C. Laboissi&#232;re, R.T. Raines. 1996. The CXXC motif: im- <br /> tion, MPD2 requires MPD1 present restore                               peratives formation native disulfide bonds cell. EMBO <br />                                                                                       (Eur. Mol. Biol. Organ.) J. 15:2659&#8211;2667. <br /> viability pdi1-deleted strain. Also, pdi1-deleted                           Cuozzo, J.W., C.A. Kaiser. 1999. Competition glutathione and <br /> strain rescued MPD2 severely affected proCPY                                 protein thiols disulphide-bond formation. Nat. Cell Biol. 1:130&#8211;135. <br /> folding. The redox potential PDI-like protein                          Desilva, M.G., J. Lu, G. Donadel, W.S. Modi, H. Xie, A.L. Notkins, M.S. <br />                                                                                       Lan. 1996. Characterization chromosomal localization new protein <br /> E. coli, DsbA, strongly dependent nature                             disulfide isomerase, PDIp, highly expressed human pancreas. DNA Cell <br /> two amino acid residues active site cysteines                             Biol. 15:9&#8211;16. <br /> (Grauschopf et al., 1995). The active site Mpd1p con-                           Dorner, A.J., L.C. Wasley, P. Raney, S. Haugejorden, M. Green, R.J. <br />                                                                                       Kaufman. 1990. The stress response Chinese hamster ovary cells. Regula- <br /> tains sequence Pdi1p, CGHC, active                              tion ERp72 protein disulfide isomerase expression secretion. J. <br /> site Mpd2p contains sequence motif CQHC. While                                 Biol. Chem. 265:22029&#8211;22034. <br />                                                                                    Edman, J.C., L. Ellis, R.W. Blacher, R.A. Roth, W.J. Rutter. 1985. Se- <br /> changing sequence CGHC did increase growth                                quence protein disulphide isomerase implications relationship <br /> rate &#9004;pdi1 [MPD2CGHC], mutation did enable                                 thioredoxin. Nature. 317:267&#8211;270. <br /> the mutant complement &#9004;pdi1 &#9004;eug1 &#9004;mpd1 &#9004;mpd2                                 Farquhar, R., N. Honey, S.J. Murant, P. Bossier, L. Schultz, D. Montgomery, <br />                                                                                       R.W. Ellis, R.B. Freedman, M.F. Tuite. 1991. Protein disulfide isomerase <br /> &#9004;eps1 strain (data shown).                                                        essential viability Saccharomyces cerevisiae. Gene. 108:81&#8211;89. <br />    In paper, addressed functionality                       Ferrari, D., H. S&#246;ling. 1999. The protein disulphide-isomerase family: un- <br /> Pdi1p homologues using approach focuses                                ravelling <span id='am-48' about='xsd:string' typeof='owl:Thing'>string</span> folds. Biochem. J. 339:1&#8211;10. <br />                                                                                    Frand, A.R., C.A. Kaiser. 1998. The ERO1 gene yeast required ox- <br /> ability homologues replace Pdi1p various                                 idation protein dithiols endoplasmic reticulum. Mol. Cell. 1:161&#8211;170. <br /> strain backgrounds. We seen clear dif-                         Frand, A.R., C.A. Kaiser. 1999. Ero1p oxidizes protein disulfide isomerase <br />                                                                                       pathway disulfide bond formation endoplasmic reticulum. <br /> ferences ability Pdi1p homologues ensure                             Mol. Cell. 4:469&#8211;477. <br /> growth Pdi1p function compromised, suggesting                              Gething, M.J., K. McCammon, J. Sambrook. 1986. Expression wild-type <br /> different vivo responsibilities. Some genes                             mutant forms influenza hemagglutinin: role folding intracel- <br />                                                                                       lular transport. Cell. 46:939&#8211;950. <br /> targets unfolded protein response (Mori et al.,                             Gillece, P., J.M. Luz, W.J. Lennarz, F.J. la Cruz, K. R&#246;misch. 1999. Ex- <br /> 1998). Differences vivo function                            port cysteine-free misfolded secretory protein endoplasmic <br /> include responding different stress conditions. Cur-                               reticulum degradation requires interaction protein disulfide <br />                                                                                       isomerase. J. Cell Biol. 147:1443&#8211;1456. <br /> rently, investigating regulation genes en-                       Givol, D., F.d. Lorenzo, R.F. Goldberger, C.B. Anfinsen. 1995. Disulfide <br /> coding homologues.                                                                interchange three-dimensional structure proteins. Proc. Natl. <br />                                                                                       Acad. Sci. USA. 53:676&#8211;684. <br /> We thank Morten C. Kielland-Brandt S&#248;ren W. Rasmussen criti-               Grauschopf, U., J.R. Winther, P. Korber, T. Zander, P. Dallinger, J.C. <br />                                                                                       Bardwell. 1995. Why DsbA oxidizing disulfide catalyst? Cell. 83: <br /> cal reading manuscript. We grateful Chris Kaiser Alison             947&#8211;955. <br /> Frand providing ero1-1 strain, Reiner Hitt Dieter Wolf         Guarente, L. 1983. Yeast promoters lacZ fusions designed study expres- <br /> providing prc1-1 plasmid.                                                         sion cloned genes yeast. Methods Enzymol. 101:181&#8211;191. <br />                                                                                    G&#252;nther, R., M. Srinivasan, S. Haugejorden, M. Green, I.M. Ehbrecht, H. <br /> Submitted: 22 May 2000                                                                Kuntzel. 1993. Functional replacement Saccharomyces cerevisiae <br /> Revised: 22 December 2000                                                             Trg1/Pdi1 protein members mammalian protein disulfide <br /> Accepted: 25 December 2000                                                            isomerase family. J. Biol. Chem. 268:7728&#8211;7732. <br />                                                                                    Holst, B., C. Tachibana, J.R. Winther. 1997. Active site mutations yeast <br />                                                                                       protein disulfide isomerase cause dithiothreitol sensitivity reduced rate <br /> References                                                                            protein folding endoplasmic reticulum. J. Cell Biol. 138:1229&#8211;1238. <br />                                                                                    Horazdovsky, B.F., S.D. Emr. 1993. The VPS16 gene product associates <br /> &#197;slund, F., J. Beckwith. 1999. The thioredoxin <span id='am-72' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="superfamily" datatype="xsd:string"></span>superfamily</span>: redundancy,           sedimentable protein complex essential vacuolar protein <br />   specificity, gray-area genomics. J. Bacteriol. 181:1375&#8211;1379.                   sorting yeast. J. Biol. Chem. 268:4953&#8211;4962. <br /> Berben, G., J. Dumont, V. Gilliquet, P.A. Bolle, F. Hilger. 1991. The YDp      J&#228;ms&#228;, E., M. Simonen, M. Makarow. 1994. Selective retention secretory <br />   plasmids: uniform set <span id='am-75' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="vectors" datatype="xsd:string"></span>vectors</span> bearing versatile gene disruption cassettes      proteins yeast endoplasmic reticulum treatment cells re- <br />   Saccharomyces cerevisiae. Yeast. 7:475&#8211;477.                                     ducing agent. Yeast. 10:355&#8211;370. <br />  <br />  <br />  <br />  <br /> N&#248;rgaard et al. Yeast PD1 Homologues                                               561 <br />  Jones, E.W. 1977. Proteinase mutants Saccharomyces cerevisiae. Genetics. 85:          grative DNA transformation yeast. Methods Enzymol. 194:281&#8211;301. <br />    23&#8211;33.                                                                             Sambrook, J., E.F. Fritsch, T. Maniatis. 1989. Molecular Cloning: A Labora- <br /> Kurjan, J. 1985. Alpha-factor structural gene mutations Saccharomyces cerevi-         tory Manual. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY. <br />    siae: effects alpha-factor production mating. Mol. Cell Biol. 5:787&#8211;796.    Sherman, F. 1991. Getting started yeast. Methods Enzymol. 194:3&#8211;21. <br /> Laboissi&#232;re, M.C.A., S.L. Sturley, R.T. Raines. 1995. The essential function      Sikorski, R.S., P. Hieter. 1989. A shuttle vectors yeast host <br />    protein-disulfide isomerase unscramble non-native disulfide bonds. J.        strains designed efficient manipulation DNA Saccharomyces cerevi- <br />    Biol. Chem. 270:28006&#8211;28009.                                                          siae. Genetics. 122:19&#8211;27. <br /> Lussier, M., A.M. White, J. Sheraton, T.d. Paolo, J. Treadwell, S.B. Southard,        Sikorski, R.S., J.D. Boeke. 1991. In vitro mutagenesis plasmid shuf- <br />    C.I. Horenstein, J. Chen-Weiner, A.F.J. Ram, J.C. Kapteyn, et al. <br /> </body></html>