<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Gill et al. Malaria Journal 2010, 9:90 <br /> http://www.malariajournal.com/content/9/1/90 <br />  <br />  <br />  <br />  <br />  RESEARCH                                                                                                                                              Open Access <br />  <br /> Structure, localization histone binding <br /> Research <br />  <br /> properties nuclear-associated nucleosome <br /> assembly protein <span id='am-4' about='obo:NCBITaxon_5820' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5794 obo:NCBITaxon_131567 obo:NCBITaxon_422676 obo:NCBITaxon_2759 obo:NCBITaxon_5819 obo:NCBITaxon_33630'><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-6'  ></span><span id='am-7' property="rdfs:label" content="Plasmodium" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Plasmodium</span> falciparum <br /> Jasmita Gill1, Anuj Kumar1, Manickam Yogavel1, Hassan Belrhali2, SK Jain3, Melanie Rug4, Monica Brown4, <br /> Alexander G Maier4,5 Amit Sharma*1 <br />  <br />  <br />  <br />   Abstract <br />   Background: Nucleosome assembly proteins (NAPs) histone chaperones crucial shuttling and <br />   incorporation histones nucleosomes. NAPs participate assembly disassembly nucleosomes thus <br />   contributing chromatin structure organization. The human malaria parasite <span id='am-42' about='obo:NCBITaxon_5833' typeof='obo:NCBITaxon_5820 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5794 obo:NCBITaxon_131567 obo:NCBITaxon_418107 obo:NCBITaxon_422676 obo:NCBITaxon_2759 obo:NCBITaxon_5819 obo:NCBITaxon_33630'><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Plasmodium (Laverania) falciparum" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Plasmodium falciparum" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47'  ></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="malaria parasite P. falciparum" datatype="xsd:string"></span>Plasmodium falciparum</span> contains two <br />   nucleosome assembly proteins termed PfNapL PfNapS. <br />   Methods: Three-dimensional crystal structure PfNapS determined analysed. Gene knockout and <br />   localization studies performed PfNapS using transfection studies. Fluorescence spectroscopy was <br />   performed identify histone-binding sites PfNapS. Extensive sequence structural comparisons done <br />   crystal structures available NAP/SET <span id='am-1' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> proteins. <br />   Results: Crystal structure PfNapS shares structural similarity previous structures NAP/SET family. Failed <br />   attempts knock-out gene PfNapS malaria parasite suggest essentiality parasite. GFP-fused PfNapS <br />   fusion protein targeting indicates cellular localization PfNapS parasite nucleus. Fluorescence spectroscopy <br />   <span id='am-136' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-137' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-138'  ></span><span id='am-139' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> suggest PfNapS interacts core histones (tetramer, octamer, H3, H4, H2A H2B) different site from <br />   interaction linker histone H1. <span id='am-59' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61'  ></span><span id='am-62' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> analysis illustrates regions PfNapS dimer possible sites for <br />   histone recognition. <br />   Conclusions: This work presents thorough analysis structural, functional regulatory attributes PfNapS <br />   P. falciparum respect previously studied histone chaperones. <br />  <br />  <br /> Background                                                                                   chaperones proteins regulate interaction of <br /> In eukaryotic cells, DNA present highly compacted                                    histones proteins DNA prevent <br /> form called chromatin. The repeating unit chromatin                                       highly basic histones forming inappropriate <br /> is nucleosome, formed histone H2A-H2B                                           aggregates [2,3]. In addition playing important role <br /> dimers histone H3-H4 tetramer                                           histone exchange nuclear processes, histone <br /> 147 bp DNA wrapped [1]. Chromatin highly                                           chaperones function nucleocytoplasmic shuttling of <br /> dynamic, characteristic vital regulating nuclear                                histones, histone storage, nucleosome assembly and <br /> processes transcription replication                                        act link chromatin remodeling factors <br /> require access DNA. Processes influence chroma-                                      histones [2,3]. Many different histone chaperones <br /> tin fluidity include post-translational modifications                                     identified, including nucleoplasmin, Asf1 <br /> histones, incorporation histone variants, histone                                     (anti-silencing function 1), HIRA proteins (Histone regu- <br /> exchange dedicated histone chaperones [2,3]. Histone                                      lator A), Spt6 (Transcription elongation factor), ACF <br />                                                                                              (ATP utilizing chromatin assembly remodelling fac- <br /> * Correspondence: amit.icgeb@gmail.com <br /> 1 Structural Computational Biology Group, International Centre Genetic               <span id='am-126' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-127' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-128'  ></span><span id='am-129' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>tor</span>), CAF1 (Chromatin assembly factor 1), NAP1 <br /> Engineering Biotechnology (ICGEB), Aruna Asaf Ali Road, New Delhi,                       (nucleosome assembly protein 1) [2,3]. <br /> 110067, India <br /> Full list author information available end article <br />  <br />                                             &#169; 2010 Gill et al; licensee BioMed Central Ltd. This Open Access article distributed terms Creative Commons At- <br />             BioMed Central tribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction any <br />                                             medium, provided original work properly cited. <br />  Gill et al. Malaria Journal 2010, 9:90                                                                          Page 2 15 <br /> http://www.malariajournal.com/content/9/1/90 <br />  <br />  <br />  <br />  <br />    Malaria common infectious diseases        fraction data collected 3.2 &#197; BM14 beamline, <br /> and remains enormous public health problem. Malaria          ESRF, Grenoble. The seleno-met crystals PfNapS <br /> is caused protozoan parasites <span id='am-107' about='obo:NCBITaxon_genus' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="genus" datatype="xsd:string"></span>genus</span> Plasmo-           belong orthorhombic space group P212121 cell <br /> dium, form disease caused        dimensions = 96.18, b = 116.84, c = 138.65 &#197; having 3 <br /> by Plasmodium falciparum [4]. Thus, important          dimers asymmetric unit (ASU). High-resolution <br /> understand fundamental biological processes P. fal-      native diffraction data 2.8 &#197; resolution collected <br /> ciparum, provide avenues identifica-          BM14 beamline, ESRF, Grenoble. These crystals also <br /> tion new protein targets development new anti-        belong orthorhombic space group P212121 differ- <br /> malarials. The human malaria parasite P. falciparum con-        ent cell dimensions = 95.79, b = 114.89, c = 139 &#197;. <br /> tains nucleosome assembly proteins, orthologs            The diffraction images processed scaled with <br /> eukaryotic NAPs, previously termed              HKL2000 suite [14]. <br /> PfNapL PfNapS [5-8]. It previously shown <br /> that PfNapL PfNapS present erythro-         Phasing, structure determination, refinement analysis <br /> cytic stages parasite [5,6]. It shown      The structure PfNapS determined using selenium- <br /> earlier PfNapS forms complexes histone           SAD technique phasing achieved utilizing a <br /> tetramer octamer predominantly localized          total 24 selenomethonines (8 dimer) ASU to <br /> the nucleus asexual sexual stages para-       3.2 &#197; resolution using SHARP [15]. Only partial initial <br /> site [5,6]. PfNapL PfNapS interact         model built side-chains using PHENIX [16] <br /> other unlike PfNapL, PfNapS able deposit his-         model subsequently rebuilt manually using <br /> tones DNA [6].                                             COOT [17]. The higher resolution structure deter- <br />    In present study, structural basis nucleosome     mined molecular replacement technique using PfNapS <br /> assembly activity P. falciparum addressed deter-      seleno-met model. This model 2.8 &#197; refined using <br /> mining analysing crystal structure smaller           CNS ([18], Table 1). Final model validated using <br /> NAP parasite called PfNapS. Data presented        PROCHECK [19]. All figures generated using Chi- <br /> the attempted gene knockout PfNapS, cellular local-      mera [20]. Least square fittings structural alignment <br /> ization histone binding properties. Attempts         carried using LSQMAN [21]. Protein localiza- <br /> generate parasites lacking PfNapS gene unsuccessful        tion predictions using LOCtree [22]. <br /> indicating likely essential nature gene para- <br /> site survival. Using GFP-fused PfNapS deletion con-             Generation <span id='am-151' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-152' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-153' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span> transgenic P. falciparum cell <br /> structs, shown PfNapS likely resident     lines <br /> the parasite nucleus. Fluorescence spectroscopy data            Transgenic parasites expressing GFP chimeras gen- <br /> indicate differential modes PfNapS interaction          erated cloning length PfNapS coding <br /> core histones versus linker histone. Finally, <span id='am-121' about='obo:NCBITaxon_652916' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6656 obo:NCBITaxon_6681 obo:NCBITaxon_6692 obo:NCBITaxon_6694 obo:NCBITaxon_1206794 obo:NCBITaxon_115581 obo:NCBITaxon_6657 obo:NCBITaxon_6682 obo:NCBITaxon_88770 obo:NCBITaxon_6683 obo:NCBITaxon_98089 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-122' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-123' property="rdfs:label" content="Thor" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-125'  ></span>thor</span>-       region truncated portions pARL vector <br /> ough comparative analysis PfNapS presented           [23], expression chimeric protein the <br /> structures NAP/SET family proteins include -           influence crt promoter. Each DNA construct was <br /> PfNapL, yeast NAP-1, yeast NAP Vps75 (vacuolar protein          confirmed sequencing. 100 ug plasmid DNA (Qiagen) <br /> sorting 75) functional domain human SET/             transfected electroporation P. falciparum 3D7 <br /> TAF-1b(&#946;)/INHAT [7-12]. These studies provide basis             parasites cultured presence 2 nM <br /> for exploration nucleosome assembly rele-       WR99210 [24]. To generate PFI0930c (PfNapS gene) <br /> vant new target development anti-malarials.              deletion construct 5&apos; 3&apos; targeting sequences were <br />                                                                 amplified using primer pairs aw771/772 aw773/ <br /> Methods                                                         774 respectively. The amplified targeting sequences (613 <br /> Protein crystallization data collection                     bp 530 bp) cloned pCC-1 vector [25] using <br /> Full length purified PfNapS protein (<span id='am-115' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-116' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-117'  ></span><span id='am-118' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span>aa</span> 1-269, molecular        SacII/SpeI 5&apos; EcoRI/AvrII 3&apos; segment, <br /> weight ~32 kDa) failed crystallize shorter        resulting pCC-1&#916;PfNapS plasmid. <br /> construct PfNapS produced containing residues              (aw771: atcccgcggATCACATTGTAATTAAGC; <br /> 29-221. PfNapS seleno-met crystals success-             Aw772: gatactagtGAAAGGCATCAAAGGATCATC; <br /> fully obtained 20&#176;C hanging drop vapour diffusion           Aw773: atcgaattcGTGCCCATGAACAAATGAAC; <br /> [13] using l &#956;l 3 mg/ml PfNapS l &#956;l 0.2 M di-           Aw774: gatcctaggCATCAAATTCTTCTAAGCC) <br /> ammonium tartrate 20% PEG 3350 (mother liquor).              The plasmid DNA proliferated <span id='am-84' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-85' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-99'  ></span>Escherichia coli</span> <br /> A single crystal soaked cryoprotectant contain-        PMC103 cells purified Qiagen Maxit kit, <br /> ing higher concentration mother liquor (30% PEG 3350         80 ug taken cytomix electroporated <br /> and 0.3 M di-ammonium tartrate) 5 min flash-            3D7 P. falciparum infected human erythrocytes (O+) <br /> frozen stream nitrogen gas 100 K. X-ray dif-      using standard procedures [24]. Parasites cultured <br />  Gill et al. Malaria Journal 2010, 9:90                                                                              Page 3 15 <br /> http://www.malariajournal.com/content/9/1/90 <br />  <br />  <br />  <br />  <br /> Table 1: Data collection refinement statistics <br />  <br />   Parameter                                    Native                           Seleno-Met <br />  <br />  <br />   Data collection <br />  <br />  <br />   Space group                                  P212121                          P212121 <br />  <br />  <br />   Cell Dimensions (&#197;)                          = 95.79, b = 114.89, c = 139   = 96.18, b = 116.84, c = 138.65 <br />  <br />  <br />   Wavelength (&#197;)                               1.0                              1.0 <br />  <br />  <br />   Anomalous scatterer                          -                                Se <br />  <br />  <br />   Resolution (&#197;)                               50.0-2.8                         50.0-3.2 <br />  <br />  <br />   Outer shell resolution (&#197;)a                  2.9-2.8                          3.31-3.2 <br />  <br />  <br />   No. unique reflections                    37166 (3101)                     24009 (1781) <br />  <br />  <br />   Redundancy                                   5.9                              5.8 <br />  <br />  <br />   Rmergeb                                      0.68 (0.295)                     0.142 (0.653) <br />  <br />  <br />   Completeness (%)                             100 (96.6)                       99.7 (69.6) <br />  <br />  <br />   I/&#963;I                                         23.5 (3.1)                       10.16 (1.2) <br />  <br />  <br />   Refinement <br />  <br />  <br />          Resolution range (&#197;)                  50-3.0 <br />  <br />  <br />          No. reflections (test set)         35024 (3501) <br />                                                0 <br />  <br />  <br />          Rfactor/Rfree (%)                     28.5/31.5 <br />  <br />  <br />          No. residues                       1006 <br />  <br />  <br />          No. water molecules                358 <br />  <br />  <br />   B factor (&#197;2) <br />  <br />  <br />   Protein atoms                                64.6 <br />  <br />  <br />   Water atoms                                  45.0 <br />  Gill et al. Malaria Journal 2010, 9:90                                                                                          Page 4 15 <br /> http://www.malariajournal.com/content/9/1/90 <br />  <br />  <br />  <br />  <br /> Table 1: Data collection refinement statistics (Continued) <br />  <br />   Stereochemistry <br />  <br />  <br />   rmsd bond length (&#197;)                                 0.01 <br />  <br />  <br />   rmsd bond angle (&#176;)                                  1.6 <br />  <br />  <br />   Ramachandran Plot <br />  <br />  <br />   Prefered regions (%)                                 66.9 <br />  <br />  <br />   Allowed regions (%)                                  33.1 <br />   Numbers      parentheses values highest resolution shell. <br />   bR <br />        merge =    | Iobs - &lt;I&gt;|/ &lt;I&gt; summed observations reflections <br />  <br />  <br /> using standard conditions parasites containing                         Protein expression, purification, fluorescent labeling and <br /> plasmid selected presence 2 nM WR99210                          ELISA-based binding studies <br /> [26]. The resulting parasite population sub-                          All histones (purchased Roche diagnostics) were <br /> jected off/on drug cycles 21 days                           dissolved autoclaved distilled water. PfNapS protein <br /> encourage loss free plasmid. After addition                     expressed purified using standard protocols <br /> WR99210, subpopulation selected 231 nM 5-                             described previously [5,6]. Site directed mutagenesis was <br /> fluorocytosine presence WR99210 (to encourage                        carried using QuickChange&#174; II Site-Directed Muta- <br /> double recombination events) analysed Southern                         genesis Kit. PfNapS labelled 6-acryloyl-2-dime- <br /> blot.                                                                          thyl-aminonaphthalene        (acrylodan)     dye      (from <br />                                                                                Invitrogen) using protocol described Prendergast <br /> SDS-PAGE western blotting                                                  et al [29] Hibbs et al [30] modifications. <br /> Parasite cultures synchronized sorbitol treatment                      Acrylodan dye extrinsic fluorophore fluores- <br /> [27] saponin pellets trophozoite parasites                         cence emission highly sensitive local microenvi- <br /> prepared. The parasite-derived proteins fraction-                         ronment bound protein. It exhibits changes <br /> ated 10% Bis-Tris precast gel (Invitrogen) using                          intensity emission wavelength reflects <br /> <span id='am-141' about='obo:NCBITaxon_258862' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_9436 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-142' property="rdfs:label" content="Mops" datatype="xsd:string"></span><span id='am-143'  ></span><span id='am-144' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>MOPS</span> buffer. Proteins electrophoretically trans-                          effective dielectric constant microenvironment <br /> ferred nitrocellulose membrane (0.22 um, Millipore)                       fluorophore. PfNapS labelled dye <br /> before western blot performed using standard proce-                        using protocol described Prendergast et al [29] and <br /> dures. Antibodies used mouse anti-GFP (1:1000,                            Hibbs et al [30]. Stock concentration dye prepared <br /> Roche) horseradish peroxidase-coupled sheep anti-                          dimethyl formamide. Purified protein incubated <br /> mouse IgG (1:2000, Chemicon, Melbourne, Australia).                            excess dye phosphate buffer (50 mM, pH = 7.5) <br /> Protein bands western blots visualized using                       overnight dark 4&#176;C. Unbound dye removed <br /> a chemiluminescent substrate (ECL, GE).                                        using Superdex&#174; 75 10/300 GL (GE healthcare). Binding of <br /> Microscopy <br />                                                                                labelled PfNapS H3 linker histone H1 stud- <br />                                                                                ied using similar protocols described previously [6]. <br /> GFP-expressing infected erythrocytes tightly syn- <br /> chronized using sorbitol studied live ambient tem-                      Steady state fluorescence measurement <br /> perature. Cells viewed Plan-Neofluar 100&#215;                          Steady state fluorescence measurement taken Per- <br /> 1.3 oil objective Zeiss Axiovert 200 M Live Cell                          kin Elmer LS50B fluorescence spectrophotometer. The <br /> Imaging Inverted Microscope equipped AxioCam                            sample titrated histones H3, H4, H2A, H2B, his- <br /> MRm <span id='am-64' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-65'  ></span><span id='am-66' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>camera</span> primarily processed AxioVision                             tone tetramer { [(H3-H4)2]} histone octamer (two <br /> 4.4 deconvolution software package. Captured images <br />                                                                                heterodimers H2A-H2B tetramer H3-H4). <br /> were processed using Photoshop ImageJ <br />                                                                                Fluorescence buffer subtracted control. Excita- <br /> software [28]. Pictures adjusted gain optimal con- <br />                                                                                tion 360 nm emission scanned from <br /> trast visualize features interest. <br />                                                                                400 550 nm. Both slits 5 nm wide open and <br />                                                                                scan speed set 100 nm minute. All fluores- <br />  Gill et al. Malaria Journal 2010, 9:90                                                                                        Page 5 15 <br /> http://www.malariajournal.com/content/9/1/90 <br />  <br />  <br />  <br />  <br /> cence emission data calculated mean               &#946;-strands &#946; sub-domain suggesting possible points <br /> replicate experiments.                                                   flexibility. <br />                                                                             The PfNapS dimer characteristic shape fold <br /> Results Discussion                                                   previously seen yNAP-1, Vps75, hSET PfNapL <br /> Structure determination PfNapS structural                         dimers (Figures 1a-b, 2a-c) [7-12]. In PfNapS, 23 residues <br /> comparisons PfNapL, hSET, yNAP-1 Vps75                          dimerization <span id='am-54' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-56'  ></span><span id='am-57' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>helix</span> &#945;2 residues &#945;7 and <br /> A construct PfNapS consisting residues 29-221 (molec-                 &#945;8 (which lie domain II) contribute dimer formation <br /> ular weight ~23 kDa) crystallized orthorhombic space                  hydrogen bonding hydrophobic interactions <br /> group P212121 solvent content 55% (the asymmet-                  (Asp32, Phe33, Ile36, Gln37, Ile40, Leu43, Asp44, Lys45, <br /> ric unit contains dimers PfNapS). Crystal struc-                Cys47, Glu50, Gln51, Ile54, Gln55, Gln57, Tyr58, Lys61, <br /> ture PfNapS determined using selenium-SAD                         Lys62, Leu65, Phe66, Lys68, Arg69, Ile72, Ile73 and <br /> (single anomalous dispersion) technique 3.2 &#197;                     His210, Pro212, Leu313, Leu217 respectively). The over- <br /> refined using native data 2.8 &#197; resolution (Table 1).                 sequence identity PfNapS yNAP-1, Vps75, <br /> Most residues PfNapS exhibited clear <span id='am-110' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-111' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-113'  ></span><span id='am-114' property="rdfs:label" content="Electron" datatype="xsd:string"></span>electron</span> density.                hSET PfNapL 20, 17, 25 18% respectively. Sim- <br /> Four C-terminal residues (aa 218-221) disordered                 ilar PfNapL NAPs malaria parasite [7], <br /> were regions 142-150 168-181. The final refined                      PfNapS evolutionary distant homologs in <br /> model PfNapS Rfactor Rfree values 28.2%                yeast human (Additional file 1). The length the <br /> 31.45% respectively (Table 1). The domain architecture                   dimerization helix &#945;2 varies proteins and <br /> and overall fold PfNapS similar PfNapL,                    PfNapS shortest helix comprising 41 resi- <br /> yNAP-1, hSET Vps75 [7-12] (Figure 1a-b, 2a-c).                       dues (Table 2). The <span id='am-69' about='obo:NCBITaxon_1' typeof='owl:Thing'><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="all" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="root" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>root</span> mean square deviation (r.m.s.d) <br /> Functional, cellular localization sub-domain proper-                 PfNapS yNAP-1, hSET, Vps75 PfNapL 1.9, <br /> ties structures summarized Table 2                  1.6, 2.0 1.9 &#197; respectively (Table 2; Figure 2a-c). <br /> and 3[6,7,31-33]. Each monomer PfNapS contains                        PfNapS exhibits highest sequence identity (25%) as <br /> domain I comprised dimerization helix &#945;2 (aa 33-73).                  structural similarity hSET. Also, r.m.s.d. <br /> The monomer contains domain II, com-                       PfNapS lowest hSET 1.6 &#197; indicating high <br /> prised &#945;-helices [&#945;3 (aa 81-85), &#945;4 (aa 99-102)] &#946;              structural identity (Table 2; Figure 2a, b). The N-terminal <br /> sub-domain containing 4 anti-parallel &#946;-strands (aa 106-                 PfNapS (aa 1-28) missing present construct <br /> 158). The 4th &#946;-strand PfNapS composed ran-                   hSET N-terminal ordered (aa 1-15) and <br /> dom coil similar hSET. Finally, &#945;-helices               forms helical structure &#945;1 (aa 2-12). The region 195-200 <br /> on opposite &#946; sub-domain [&#945;7 (aa 200-211                 domain II hSET disordered corresponding <br /> and &#945;8 (aa 214-216)] (Figure 1b). It observed                   region (aa 191-196) PfNapS weak electron density. <br /> each chains PfNapS crystallographic asym-               The helices &#945;7 &#945;8 &#946; sub-domain of <br /> metric unit differ mainly orientation                    PfNapS similar length orientation hSET. <br /> dimerization helix &#945;2 loop regions                   Interestingly, disorder present structure of <br />                                                                          PfNapS closest hSET structure (Figure 2a, b). <br />  <br /> Table 2: Localization details PfNapS, PfNapL, hSET, yNAP-1 Vps75. <br />  <br />  Protein, PDB code       Annotation              Localization              r.m.s.d C&#945;, &#197; (number C&#945;   Length dimerization helix <br />                                                                            matched)                         &#945;2 (r.m.s.d, &#197;) <br />  <br />  <br />  PfNapS, 3KYP            malaria parasite NapS   Primarily nuclear                                          41 <br />  <br />  <br />  PfNapL, 3FS3[6-8]       Malaria parasite NapL   Primarily cytoplasmic     1.94 (144)                       51 (0.9) <br />  <br />  <br />  hSET, 2E50[10,31]       Human                   Nucleocytoplasmic         1.6 (139)                        54 (0.75) <br />                          SET/TAF-I&#946;/INHAT <br />                          functional domain <br />  <br />  <br />  yNAP-1, 2AYU[9,32]      Yeast NAP-1             Nucleocytoplasmic         1.9 (112)                        51 (1.0) <br />  <br />  <br />  Vps75, 3CDM[11,12,33]   Yeast NAP               Primarily nuclear         2.0 (124)                        42 (0.9) <br />  Comparison root mean square deviation (r.m.s.d) NAP/SET monomers length dimerization helix &#945;2 PfNapS monomer. <br />  Gill et al. Malaria Journal 2010, 9:90                                                                                                    Page 6 15 <br /> http://www.malariajournal.com/content/9/1/90 <br />  <br />  <br />  <br />  <br /> Table 3: Comparison pI (isoelectic point) sequence stretches 5 crystal structures <br />  <br />    pI (isoelectric point)           Entire protein              Protein sequence             Missing N-terminal            Missing C-terminal <br />                                       sequence                     crystallised <br />  <br />  <br />           PfNapS                          4.4                           5.4                           5.2                           3.0 <br />  <br />  <br />           PfNapL                          5.0                           7.0                           4.9                           3.3 <br />  <br />  <br />            hSET                           4.2                           4.9                            -                            2.6 <br />  <br />  <br />           yNAP-1                          4.3                           4.5                           5.2                           3.5 <br />  <br />  <br />           Vps75                           4.9                           5.2                           3.4                           3.4 <br />  <br />  <br />   The overall sequence identity PfNapS yNAP-1                         168-181 PfNapS disordered, however, Vps75 an <br /> and Vps75 20 17% respectively. The differences                         ordered helical content region (aa 153-176) and <br /> with respect yNAP-1 Vps75 overall                           yNAP-1 anti-parallel &#946;-strands containing the <br /> structural orientations secondary structures (Table 2;                     nuclear localization signal (NLS). Also, comparisons with <br /> Figure 2a, c). The major difference yNAP-1                            identified NLS yNAP-1 revealed PfNapS <br /> absence &apos;accessory domain&apos; PfNapS. It                      lacks NLS case Vps75. <br /> earlier shown &apos;accessory domain&apos; absent                         A sequence alignment PfNapS homologs <br /> PfNapS, PfNapL homologs Plasmo-                          Plasmodium <span id='am-74' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-75' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> (<span id='am-36' about='obo:NCBITaxon_5855' typeof='obo:NCBITaxon_5820 owl:Thing obo:NCBITaxon_418103 obo:NCBITaxon_1 obo:NCBITaxon_5794 obo:NCBITaxon_131567 obo:NCBITaxon_422676 obo:NCBITaxon_2759 obo:NCBITaxon_5819 obo:NCBITaxon_33630'><span id='am-37' property="oboInOwl:hasExactSynonym" content="malaria parasite P. vivax" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-40'  ></span><span id='am-41' property="rdfs:label" content="Plasmodium vivax" datatype="xsd:string"></span>Plasmodium vivax</span>, Plasmo- <br /> dium species [7]. A sequence comparison PfNapS                        dium berghei, <span id='am-31' about='obo:NCBITaxon_5850' typeof='obo:NCBITaxon_5820 owl:Thing obo:NCBITaxon_418103 obo:NCBITaxon_1 obo:NCBITaxon_5794 obo:NCBITaxon_131567 obo:NCBITaxon_422676 obo:NCBITaxon_2759 obo:NCBITaxon_5819 obo:NCBITaxon_33630'><span id='am-32'  ></span><span id='am-33' property="rdfs:label" content="Plasmodium knowlesi" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Plasmodium knowlesi</span>, <span id='am-25' about='obo:NCBITaxon_5861' typeof='obo:NCBITaxon_5820 owl:Thing obo:NCBITaxon_418101 obo:NCBITaxon_1 obo:NCBITaxon_5794 obo:NCBITaxon_131567 obo:NCBITaxon_422676 obo:NCBITaxon_2759 obo:NCBITaxon_5819 obo:NCBITaxon_33630'><span id='am-26'  ></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Plasmodium yoelli" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Plasmodium yoelii" datatype="xsd:string"></span>Plasmodium yoelii</span> <br /> other apicomplexan parasites like <span id='am-14' about='obo:NCBITaxon_5810' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5794 obo:NCBITaxon_131567 obo:NCBITaxon_5809 obo:NCBITaxon_5796 obo:NCBITaxon_1280412 obo:NCBITaxon_2759 obo:NCBITaxon_75739 obo:NCBITaxon_33630 obo:NCBITaxon_423054'><span id='am-15'  ></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Toxoplasma" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Toxoplasma Nicolle and Manceaux, 1909" datatype="xsd:string"></span>Toxoplasma</span>, <span id='am-20' about='obo:NCBITaxon_5864' typeof='obo:NCBITaxon_32594 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5794 obo:NCBITaxon_131567 obo:NCBITaxon_5863 obo:NCBITaxon_422676 obo:NCBITaxon_2759 obo:NCBITaxon_33630'><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Babesia" datatype="xsd:string"></span><span id='am-24'  ></span>Babesia</span>                         <span id='am-9' about='obo:NCBITaxon_5825' typeof='obo:NCBITaxon_5820 owl:Thing obo:NCBITaxon_418101 obo:NCBITaxon_1 obo:NCBITaxon_5794 obo:NCBITaxon_131567 obo:NCBITaxon_422676 obo:NCBITaxon_2759 obo:NCBITaxon_5819 obo:NCBITaxon_33630'><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-12'  ></span><span id='am-13' property="rdfs:label" content="Plasmodium chabaudi" datatype="xsd:string"></span>Plasmodium chabaudi</span>) revealed high sequence con- <br /> and <span id='am-49' about='obo:NCBITaxon_5873' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5794 obo:NCBITaxon_131567 obo:NCBITaxon_5863 obo:NCBITaxon_27994 obo:NCBITaxon_422676 obo:NCBITaxon_2759 obo:NCBITaxon_33630'><span id='am-50'  ></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Theileria" datatype="xsd:string"></span>Theileria</span> revealed domain region absent                      servation (75-90%). Further analysis <br /> in (Additional file 2). Since &apos;accessory                         showed N-terminal exhibits lower conservation <br /> domain&apos; absent Vps75 hSET (which                        C-terminal contains acidic stretches these <br /> primarily localized nucleus), remains unclear                       proteins (Additional file 1). Dimer contributing residues <br /> what role domain plays nuclear shuttling                     dimerization helix &#945;2 domain II con- <br /> proteins, suggested yNAP-1 [7-9]. The region                       served structural homologs. Residues implicated in <br />  <br />  <br />  <br />  <br />  Figure 1 Overall structure PfNapS. (a) Domain diagram NAP/SET crystal structures determined far. Architecture PfNapS, yNAP-1, Vps75, <br />  hSET PfNapL highlighting &apos;accessory domain&apos; yNAP-1 (coloured yellow) missing remaining 4 structures (set scale). (b) <br />  Structure PfNapS dimer. Each monomer PfNapS contains domain I comprised dimerization helix &#945;2 domain II comprised &#946; sub- <br />  domain (4 anti-parallel &#946; strands) &#945;-helices. <br />  Gill et al. Malaria Journal 2010, 9:90                                                                                                       Page 7 15 <br /> http://www.malariajournal.com/content/9/1/90 <br />  <br />  <br />  <br />  <br />  Figure 2 Comparison PfNapS yNAP-1, Vps75, hSET PfNapL (a). Structure-based sequence alignment PfNapS, yNAP-1, Vps75, hSET <br />  PfNapL. Residues constituting &#945;-helices &#946;-sheets individual structures coloured magenta green respectively. Domains I and <br />  II PfNapS indicated using blue green bars respectively. Conserved hydrophobic motifs dotted underlined purple indicated as <br />  &apos;motif I&apos; &apos;motif II&apos;. The &apos;accessory domain&apos; yNAP-1 foreground coloured yellow. The predicted <span id='am-131' about='obo:NCBITaxon_203327' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_8220 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_8219 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489878 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_497223 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-132' property="rdfs:label" content="Nes" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-134'  ></span><span id='am-135' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>NES</span> NLS motifs yNAP-1 shown a <br />  shaded box coloured grey. The corresponding NES NLS motifs PfNapL shown shaded box coloured grey. Disordered/missing res- <br />  idues small caps coloured grey. Residues aligned shown italics. (b) Superposition PfNapS dimer structure (orange) <br />  hSET (PDB code: 2E50, green) PfNapL (PDB code: 3FS3, purple). (c) Superposition PfNapS dimer (coloured orange) hSET (PDB code: <br />  2E50, green) yNAP-1 (PDB code: 2AYU, purple). The &apos;accessory domain&apos; yNAP-1 absent coloured blue. <br />  <br />  <br /> PfNapL histone recognition conserved                            indicated essentiality nucleosome assembly <br /> PfNapS suggesting variable binding site histones [7].                      proteins. It shown gene ablations mouse <br /> Also, comparison residues hSET mutagene-                         Drosophila cause embryonic lethality in <br /> sis data revealed corresponding residues PfNapS                        yeast cells exhibit growth defects [35-37]. Therefore, is <br /> and homologs Plasmodium apicompl-                           surprising malaria parasite likely requires <br /> exans overall weak conservation (Additional file 1).                      gene survival. <br />                                                                                  It earlier shown PfNapS expressed dur- <br /> Gene knockout studies PfNapS localization                    ing parasite blood stages, immunofluorescence <br /> parasite                                                                       assays suggested close association PfNapS para- <br /> To assess role PfNapS vivo, disruption cor-                   site nucleus [5,6]. To address issue potential <br /> responding gene attempted genome P. falci-                         sequences PfNapS target protein <br /> <span id='am-77' about='obo:NCBITaxon_522859' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_469321 obo:NCBITaxon_37569 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_523180 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_7128 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="Parum" datatype="xsd:string"></span><span id='am-81'  ></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Parum Rothschild &amp; Jordan, 1903" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523222" datatype="xsd:string"></span>parum</span> 3D7 strain. Despite attempts, gene                           parasite nucleus (in absence clear NLS <br /> could disrupted indicating essential role                   motif ), full-length PfNapS fused green fluores- <br /> gene (Additional file 3). This observation consistent                       cence protein (GFP) gene fusions were <br /> with unsuccessful attempts disrupt NapS murine                       expressed episomally wild-type parasite. The protein <br /> malaria parasite P. berghei [34]. Due current tech-                     chimeras showed nuclear localization stages of <br /> nological limitations imposed Plasmodium sys-                           erythrocytic life-cycle determined co-localization <br /> tems, essentiality PfNapS formally                         nuclear stain DAPI (4&apos;, 6-diamidino-2-phenylin- <br /> proven; however, results provide circumstantial evi-                     dole; nuclear stain) (Figure 3a, c-f). Western blot analysis <br /> dence important role played PfNapS. Gene                         using anti-GFP antibodies confirmed expression of <br /> knockout phenotypes studied organisms                         fusion proteins (Figure 3b). All fused proteins ran the <br />  Gill et al. Malaria Journal 2010, 9:90                                                                          Page 8 15 <br /> http://www.malariajournal.com/content/9/1/90 <br />  <br />  <br />  <br />  <br /> expected sizes residues 30-90-GFP                To investigate probable histone binding <br /> resulted band 29 kDa pre-        region(s) PfNapS, surface cysteine residues <br /> dicted 34 kDa. One possible explanation rel-       PfNapS mutated independently (mutants are <br /> ative high content glutamines glutamic acids          referred C47A C154A). On plotting change <br /> this fusion protein resulting aberrant protein mobility.     fluorescence function histone concentration, it <br /> The truncated forms consisting aa1-30-GFP,             observed absence histone, chromic <br /> aa30-90-GFP aa90-269-GFP (Figure 3a) showed dif-            shift similar mutants. This analysis suggests <br /> ferent localizations live cell fluorescence microscopy:      cysteine residues experienced solvent <br /> aa90-269 showed similar nuclear localization full-     environment. Upon adding histones, blue shift was <br /> length chimera, aa1-30-GFP aa30-90-GFP              observed increase quantum yield <br /> showed fluorescence parasite cytoplasm. There-           suggesting C47A C154A play role his- <br /> fore, likely region responsible target-   tone binding. However, higher change chromic <br /> ing PfNapS nucleus contained PfNapS            shift C154A (about times) comparison C47A <br /> residues 90-269. The localization fusion proteins      (Figure 4d) possibly suggests bigger role C154A the <br /> did alter parasite life-cycle stages immu-    binding interaction. <br /> nofluorescence assays (Additional file 3). In well-           Interestingly, chromic shift increase quan- <br /> studied yeast NAP (yNAP-1), residues 290-295                    tum yield fluorescence observed titrating the <br /> (&apos;RKQRNK&apos;) experimentally identified            PfNapSdye linker histone H1 indicating change in <br /> NLS [9]. It previously shown corre-           dielectric constant microenvironment around <br /> sponding region PfNapL (aa 203-207, &apos;KKQHNK&apos;)             fluorophore (Figure 4e). These data suggest differ- <br /> disordered crystal structure, case            ent mode and/or region binding linker histone H1 to <br /> PfNapL resident parasite cytoplasm [7]             PfNapS opposed core histones. To rule the <br /> (Table 2; Figure 2a). In case nuclear associated         possibility dye affecting binding his- <br /> PfNapS, corresponding region disordered (aa        tone H1 PfNapS, interaction labelled PfNapS with <br /> 172 177-&apos;NRSDLD&apos;) [7,9] (Table 2, Figure 2a-b). There-       histone H1 analysed compared binding <br /> fore, GFP-fusion transfection data suggest PfNapS          histone H3. No significant difference observed <br /> likely contains NLS-like motif residues 90-269       binding histones PfNapS (Figure 4c). These <br /> set sets motifs, direct PfNapS nucleus       data agreement previously published work, <br /> remain discovered.                                        suggested differential binding NAPs linker <br />                                                                 histone H1 compared core histones his- <br /> PfNapS-histone interaction using fluorescence                   tone oligomers [38,39]. Interestingly, clear homolog <br /> spectroscopy                                                    histone H1 identified Plasmodium or <br /> Structural analysis reveals free surface exposed            apicomplexan organism far. However, <br /> cysteine residues (Cys47 Cys154) PfNapS dimer.           histone H1 sequences tend vary likely that <br /> The residue Cys47 present concave region     overall structure conserved. It very <br /> dimerization helix &#945;2 Cys154 present outer        unusual apicomplexans lacked linker histone H1 alto- <br /> surface domain II (Figure 4a). These cysteines          gether. The experiments canon- <br /> labelled PfNapS using molecular environment sensi-           ical histones assumption apicomplexans will <br /> tive dye 6-acryloyl-2-dimethyl-aminonaphthalene (acry-          structural homolog linker histone H1. <br /> lodan) binds covalently surface exposed cysteine <br /> residues. Post modification, tagged protein (PfNapS-        Comparison histone binding characteristics NAP/SET <br /> dye) passed gel filtration column remove         structures <br /> unbound dye. PfNapSdye used histone interac-        Mutagenesis studies performed previously hSET had <br /> tion studies. Upon titrating PfNapSdye histone             revealed important residues domain II affect the <br /> monomers histone oligomers gradual blue shift         binding hSET core histones dsDNA [10]. <br /> observed increase quantum yield            These residues mapped PfNapS dimer and <br /> (Figure 4b). This chromic shift reaches saturation level      corresponding ones conserved with <br /> and blue shift increase observed          hSET Lys163 Thr190 (Table 4). Only of <br /> upon addition histone different         12 residues affect binding hSET histones <br /> one. These data suggest embedding tagged                 (completely marginal extent) conserved in <br /> cysteines binding PfNapSdye histones. Such           PfNapS. In previous study cytoplasmically res- <br /> interaction likely causes transition dye polar        ident PfNapL, residues PfNapL mutated based <br /> (fully exposed solvent) non-polar environment (i.e.     hSET mutagenesis data shown that <br /> burial Nap-histone interface).                               effect binding PfNapL his- <br />  Gill et al. Malaria Journal 2010, 9:90                                                                                                      Page 9 15 <br /> http://www.malariajournal.com/content/9/1/90 <br />  <br />  <br />  <br />  <br />  Figure 3 Stage specific expression PfNapS-GFP chimeras (a). Deletion constructs PfNapS-GFP. (b) Western blot analysis transgenic P. fal- <br />  ciparum cell lines expressing GFP-tagged versions PfNapS antibodies GFP. PfHsp-70 expression detected loading control. Lo- <br />  calization (c) PfNapS-full length-GFP (d) aa1-30-GFP (e) aa30-90-GFP (f) aa90-269-GFP parasite merozoite, ring, trophozoite, late-trophozoite <br />  schizont stages. The column shows DIC (Differential Interference Contrast) images, followed images nuclear DAPI stain, GFP fluo- <br />  rescence, overlay nuclear GFP localization overlay images. <br />  Gill et al. Malaria Journal 2010, 9:90                                                                                                         Page 10 15 <br /> http://www.malariajournal.com/content/9/1/90 <br />  <br />  <br />  <br />  <br />  Figure 4 PfNapS-histone interaction studies using fluorescence spectroscopy (a). PfNapS dimer shown molecular surface. Domains I II <br />  coloured blue green respectively. The residues Cys47 Cys154 coloured red yellow indicated. (b) Fluorescence emission <br />  spectra PfNapS presence increasingconcentrations histones. A hypsochromic shift (blue shift) observed onbinding histones. However <br />  case histone H1 change wasobserved. Fluorescence intensity measured incubating75 micro gram PfNapS protein 5, <br />  10, 15, 20, 25, 30, 35, 40, 45, 50 micrograms ofhistones. All experiments repeated thrice. (c) Effect labeling binding PfNapS histones <br />  H3 H1. (d) Graph representing hypsochromic shift (F460/485) fluorescence PfNapS mutants function of H3 histone. (e) <br />  Graph representing differential binding linker histone H1 respect core histones. There shift fluorescence binding H1, however, <br />  core histones hypsochromic shift (F460/485) fluorescence. <br />  Gill et al. Malaria Journal 2010, 9:90                                                                        Page 11 15 <br /> http://www.malariajournal.com/content/9/1/90 <br />  <br />  <br />  <br />  <br /> Table 4: Structure-based comparison hSET mutagenesis data residues effect binding (completely or <br /> partially) histones <br />  <br />                          hSET                  PfNapL         PfNapS              Vps75                yNAP-1 <br />  <br />  <br />   1                      S162                  D192           E161                Q150                 D278 <br />  <br />  <br />   2*                     K164                  K194           K163                D152                 H281 <br />  <br />  <br />   3                      D165                  N195           N164                S153                 N282 <br />  <br />  <br />   4                      T191                  H227           E187                G181                 N314 <br />  <br />  <br />   5*                     T194                  T230           T190                R184                 D317 <br />  <br />  <br />   6                      D195 (disordered)     S231           T191                W185                 D317 <br />  <br />  <br />   7                      D202                  Y259           P198                D198                 Y342 <br />  <br />  <br />   8*                     E203                  E260           D199                S199                 S343 <br />  <br />  <br />   9*                     E206                  L263           E202                S202                 E246 <br />  <br />  <br />   10*                    K209                  K266           R205                S205                 K349 <br />  <br />  <br />   11                     D210                  E267           R206                E206                 D350 <br />  <br />  <br />   12*                    D211                  R268           E207                E207                 K351 <br />  <br />  <br />   13                     E187 (disordered)     D223           W183                K177                 E320 <br />   *conserved PfNapS <br />  <br /> tones [7]. These residues conserved            Phosphorylation status <br /> amongst PfNapL PfNapS residues              It earlier shown PfNapL PfNapS <br /> His227 Tyr259 PfNapL replaced PfNapS glu-        phosphorylated casein kinase II (CKII) [5,6]. Pro- <br /> tamine proline residues respectively (Table 5). Three     grams NetPhos NetPhosK predicted phosphoryla- <br /> residues PfNapL earlier identified          tion sites PfNapS residues Ser91, <br /> potentially contribute histone recognition, based       Thr190 Thr191 (CKII phosphorylation site) sur- <br /> Asf1-histone complex analysis [7,40,41]. Although      face exposed structure. All residues cor- <br /> these residues PfNapL conserved PfNapS,         responding serine and/or threonine residues respectively <br /> the fact alanine PfNapL cysteine      hSET PfNapL, surface exposed. <br /> (Cys154) PfNapS. Interestingly, new fluorescence data      However, site Ser91 PfNapS replaced cysteine <br /> indicates role region encompassing cysteine        alanine residues Thr190 replaced aspartate <br /> (Cys154) interaction core histones,   arginine residues yNAP-1 Vps75 respectively. <br /> linker histone H1. In summary, studies PfNapL          Thus, phosphorylation sites predicted PfNapS are <br /> and PfNapS suggest region regions            conserved PfNapL hSET yNAP-1 and <br /> domain II nucleosome assembly proteins likely        Vps75. Further, phosphorylation sites which <br /> contribute histone recognition.                            earlier identified yNAP-1 lie &apos;acces- <br />                                                               sory domain&apos; region absent PfNapS, PfNapL, <br />  Gill et al. Malaria Journal 2010, 9:90                                                                              Page 12 15 <br /> http://www.malariajournal.com/content/9/1/90 <br />  <br />  <br />  <br />  <br /> hSET Vps75. Interestingly, corresponding resi-              Yeast two-<span id='am-146' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-147' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-149' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-150'  ></span>hybrid</span> studies interacting partners of <br /> dues Ser184 PfNapS lie conserved              PfNapS suggest crucial role protein numerous <br /> hydrophobic motif (SIFEWF) conserved surface             protein networks [42]. A total 27 proteins been <br /> exposed structures possibly rep-       shown interact PfNapS [42]. However, the <br /> resent common phosphorylation site NAP/SET pro-            absence independent experimental validation each <br /> teins (Figures 2a, 5c). This conserved residue          protein-protein interactions, premature to <br /> predicted CKII site proteins            speculate biological relevance. Nonetheless, an <br /> nonetheless significance given conservation        attempt classify binding partners for <br /> in proteins.                                               PfNapS. Analysis suggests 27 proteins <br />   In summary, comparative structural analyses              predicted localized parasite nucleus <br /> show PfNapS, PfNapL, hSET, yNAP-1 Vps75                 (Additional file 4). Further, total 27 proteins are <br /> contain conserved residues dimerization helix             likely involved chromatin modelling (Additional <br /> which contribute dimer formation pro-       file 4). The crystal structures interacting proteins <br /> teins (Table 6 Figure 5a). Functional data PfNapS         unknown and, therefore, binding modes with <br /> indicate inability delete PfNapS gene suggesting      PfNapS remain unexplored. Clearly, studies pre- <br /> essentiality parasite. Transfection studies identify      sented pave way deeper dissection numer- <br /> parasite nucleus site localization PfNapS.         ous protein-proteins interactions occur PfNapS. <br /> Fluorescence data analysis highlights regions on <br /> PfNapS dimer likely contribute histone           Conclusions <br /> recognition (Figure 5b). Finally, likely region   The on-going structural, biochemical, gene knockout and <br /> for histone recognition NAPs SET domains              localization studies nucleosome assembly pro- <br /> be outer face domain II base cavity     teins P. falciparum revealed similar three- <br /> underside dimer (Figure 5b).                             dimensional structures proteins whilst having <br />                                                                  different localizations parasite. Based on <br />  <br /> Table 5: Structure-based comparison PfNapL mutagenesis data <br />  <br />                          PfNapL                PfNapS            Vps75                 yNAP-1                hSET <br />  <br />  <br />   Effect binding <br />  <br />  <br />   1                      I136                  N112              S95                   -                     F115 <br />  <br />  <br />   2                      I147                  K120              S104                  K230                  R123 <br />  <br />  <br />   No effect binding PfNapL               PfNapS            Vps75                 yNAP-1                hSET <br />  <br />  <br />   1*                     D192                  E161              Q150                  D278                  S162 <br />  <br />  <br />   2                      H227                  E187              G181                  N314                  T191 <br />  <br />  <br />   3*                     T230                  T191              R184                  D317                  T194 <br />  <br />  <br />   4                      Y259                  P198              D198                  Y342                  D202 <br />  <br />  <br />   5*                     E260                  D199              S199                  S343                  E203 <br />  <br />  <br />   6*                     K266                  R205              S205                  K349                  K209 <br />  <br />  <br />   7                      D223                  W183              K177                  E320                  Disordered <br />   *conserved PfNapS <br />  Gill et al. Malaria Journal 2010, 9:90                                                                                                   Page 13 15 <br /> http://www.malariajournal.com/content/9/1/90 <br />  <br />  <br />  <br />  <br />  Figure 5 Overall structural, functional regulatory comparisons NAP/SET proteins based 5 crystal structures [7-12]. (a). <br />  PfNapS dimerization helix &#945;2 shown molecular surface. Conserved dimer contributing residues 5 crystal structures mapped onto <br />  PfNapS. Residues chains A B coloured blue green respectively residues chain A labelled. (b) hSET shown molecular <br />  surface coloured brown. Residues hSET mutagenesis analysis coloured green corresponding residues PfNapL mutagenesis are <br />  coloured blue. The corresponding residues cysteines PfNapS fluorescence data coloured cyan indicated arrows. (c) <br />  PfNapS dimer shown molecular surface coloured sky blue conserved hydrophobic motif coloured blue. Surface exposed serine <br />  residue coloured red. <br />  <br />  <br /> extensive experimental analysis malaria parasite                     Protein Data Bank Accession Code <br /> nucleosome assembly proteins, suggested                      The coordinates deposited public data- <br /> critical pair proteins provide unique opportunity                   bank PDB code 3KYP. <br /> the exploration anti-malarial targets. <br />  <br /> Table 6: Conserved residues PfNapS, PfNapL, yNAP-1, hSET Vps75 contribute dimer formation <br />  <br />                            Conserved 5 proteins                        Conserved 4 5                   Conserved 3 5 <br />  <br />  <br />   PfNapS residues          I40, L43, K46, C47, E50, Q51, I54, Q57, Y58,       D32, F33, I36, D44, Q55, F66          Q37 <br />                            K61, K62, L65, K68, R69, I73, I73 <br />  Gill et al. Malaria Journal 2010, 9:90                                                                                                                                                                                                                                                                                                                   Page 14 15 <br /> http://www.malariajournal.com/content/9/1/90 <br />  <br />  <br />  <br />  <br /> Additional material                                                                                                                                                                                                                                                                         protein: Distinct Modes Of Protein Localization And Histone <br />                                                                                                                                                                                                                                                                                             Recognition. J Biol Chem 2009, 284:10076-10087. <br />                                                                                                                                                                                                                                                                                       8.    Yogavel M, Gill J, Sharma A: Iodide-SAD, SIR SIRAS phasing for <br />       Additional file 1 Phylogenetic tree NAPs various species                                                                                                                                                                                                                      structure solution nucleosome assembly protein. Acta Crystallogr D <br />       showing greater evolutionary distance malaria parasite NAPs                                                                                                                                                                                                                   Biol Crystallogr 2009, 65:618-612. <br />       homologs yeast man (indicated red arrows).                                                                                                                                                                                                                            9.    Park Y-J, Luger K: The structure nucleosome assembly protein 1. Proc <br />       Additional file 2 Structure-based sequence alignment PfNapS                                                                                                                                                                                                                   Natl Acad Sci USA 2006, 103:1248-1253. <br />       homologs Plasmodium apicomplexans. Residues                                                                                                                                                                                                                  10.   Muto S, Senda M, Akai Y, Sato L, Suzuki T, Nagai R, Senda T, Horikoshi M: <br />       identical conserved PfNapS homologs col-                                                                                                                                                                                                              Relationship structure SET/TAF-I&#946;/INHAT its <br />       ored red green respectively. The histone binding residues hSET                                                                                                                                                                                                               histone chaperone activity. Proc Natl Acad Sci USA 2007, 104:4285-4290. <br />       PfNapL mutagenesis experiments shown shaded box colored                                                                                                                                                                                                              11.   Tang Y, Meeth K, Jiang E, Luo C, Marmorstein R: Structure Vps75 and <br />       cyan green respectively [7,10].                                                                                                                                                                                                                                                   implications histone chaperone function. Proc Natl Acad Sci USA <br />       Additional file 3 PfNapS gene knockout analysis.                                                                                                                                                                                                                                      2008, 105:12206-12211. <br />                                                                                                                                                                                                                                                                                       12.   Berndsen CE, Tsubota T, Lindner SE, Lee S, Holton JM, Kaufman PD, Keck <br />       Additional file 4 Protein interacting partners PfNapS based upon <br />                                                                                                                                                                                                                                                                                             JL, Denu JM: Molecular functions histone acetyltransferase <br />       yeast two-hybrid data shown need experimental <br />                                                                                                                                                                                                                                                                                             chaperone complex Rtt109-Vps75. Nat Struct Mol Biol 2008:948-956. <br />       validation. Proteins marked based predicted localization - <br />                                                                                                                                                                                                                                                                                       13.   McPherson A: Current approaches macromolecular crystallization. <br />       nuclear cytoplasmic possible. <br />                                                                                                                                                                                                                                                                                             Eur J Biochem 1990, 189:1-23. <br />                                                                                                                                                                                                                                                                                       14.   Otwinowski Z, Minor W: Processing X-ray diffraction data collected in <br /> Competing interests                                                                                                                                                                                                                                                                         oscillation mode. Methods Enzymol 1997, 276:307-326. <br /> The authors declare competing interests.                                                                                                                                                                                                                            15.   Vonrhein C, Blanc E, Roversi P, Bricogne G: Automated structure solution <br />                                                                                                                                                                                                                                                                                             autoSHARP. Methods Mol Biol 2007, 364:215-230. <br /> Authors&apos; contributions                                                                                                                                                                                                                                                                16.   Adams PD, Grosse-Kunstleve RW, Hung LW, Ioerger TR, McCoy AJ, <br /> JG solved structure PfNapS AS analysed data                                                                                                                                                                                                                   Moriarty NW, Read RJ, Sacchettini JC, Sauter NK, Terwilliger TC: PHENIX: <br /> wrote manuscript. AK purified crystallized seleno-met PfNapS                                                                                                                                                                                                               building new software automated crystallographic structure <br /> performed binding studies. MY, HB, SKJ assisted work presented.                                                                                                                                                                                                                determination. Acta Cryst Sect D 2002, 58:1948-1954. <br /> MR, MB AGM performed gene KO transfection experiments. All                                                                                                                                                                                                                17.   Emsley P, Cowtan K: Coot: model-building tools molecular graphics. <br /> authors assisted manuscript preparation read approved final                                                                                                                                                                                                                Acta Cryst Sect D 2004, 60:2126-2132. <br /> manuscript.                                                                                                                                                                                                                                                                           18.   Brunger AT, Adams PD, Clore GM, Gros P, Grosse-Kunstleve RW, Kuszewski <br />                                                                                                                                                                                                                                                                                             JS, Jiang J, Nilges N, Pannu NS, Read RJ, Rice LM, Simonson T, Warren GL: <br /> Acknowledgements                                                                                                                                                                                                                                                                            Crystallography &amp; NMR system: A new software suite for <br /> JG MY supported grant &apos;PreMalStruct&apos; European Commis-                                                                                                                                                                                                                   macromolecular structure determination. Acta Cryst Sect D 1998, <br /> sion. AK, AS work supported grant Department Bio-                                                                                                                                                                                                              54:905-921. <br /> technology, Government India. The X-ray facility ICGEB, New Delhi                                                                                                                                                                                                            19.   Laskowski RA, MacArthur MW, Moss DS, Thornton JM: PROCHECK : a <br /> funded grant AS Wellcome Trust, UK. AGM supported                                                                                                                                                                                                                program check stereochemical quality protein structures. J <br /> National Health Medical Research Council, Australia. AGM ARC Austra-                                                                                                                                                                                                              Appl Crystallography 1993, 26:283-291. <br /> lian Research Fellow.                                                                                                                                                                                                                                                                 20.   Pettersen EF, Goddard TD, Huang CC, Couch GS, Greenblatt DM, Meng EC, <br />                                                                                                                                                                                                                                                                                             Ferrin TE: UCSF Chimera-a visualization exploratory research <br /> Author Details                                                                                                                                                                                                                                                                              analysis. J Comp Chem 2004, 25:1605-1612. <br /> 1Structural Computational Biology Group, International Centre Genetic                                                                                                                                                                                                         21.   Kleywegt GJ: Use non-crystallographic symmetry protein <br /> Engineering Biotechnology (ICGEB), Aruna Asaf Ali Road, New Delhi,                                                                                                                                                                                                                      structure refinement. Acta Cryst Sect D 1996, 52:842-857. <br /> 110067, India, 2European Molecular Biology Laboratory (EMBL), 6 rue Jules                                                                                                                                                                                                             22.   Nair R, Rost B: Mimicking cellular sorting improves prediction of <br /> Horowitz, BP 181, F-38042 Grenoble Cedex 9, France, 3Department                                                                                                                                                                                                                          subcellular localization. J Mol Biol 2005, 348:85-100. <br /> Biotechnology, Jamia Hamdard, New Delhi 110062, India, 4Malaria Functional                                                                                                                                                                                                            23.   Rug M, Prescott SW, Fernandez KM, Cooke BM, Cowman AF: The role of <br /> Genomics Facility, The Walter Eliza Hall Institute Medical Research, 1G                                                                                                                                                                                                              KAHRP domains knob formation cytoadherence P. <br /> Royal Parade, Melbourne 3050, Australia 5Biochemistry Department, La                                                                                                                                                                                                                    falciparum-infected human erythrocytes. Blood 2006, 108:370-378. <br /> Trobe University, Melbourne 3086, Australia                                                                                                                                                                                                                                           24.   Crabb BS, Rug M, Gilberger T, Thompson JK, Triglia T, Maier AG, Cowman <br />                                                                                                                                                                                                                                                                                             AF: Transfection human malaria parasite Plasmodium falciparum. <br /> Received: 15 January 2010 Accepted: 8 April 2010                                                                                                                                                                                                                                            In Methods Molecular Biology Volume 270. Edited by: Melville, SE. Parasite <br /> Published: 8 April 2010                                                                                                                                                                                                                                                                     Genomics Protocols. Totowa, NJ: Humana Press Inc; 2004. <br /> This <br /> &#169; 2010 <br />      article <br />      is <br /> Malaria an <br />          Gill etavailable <br />          Journal <br />            Open <br />               al;Access <br />                     2010, <br />                      licensee <br />                            article <br />                           9:90     distributed <br />                               BioMed <br />                            from:               Ltd. terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />                                   http://www.malariajournal.com/content/9/1/90 <br />                                        Central under <br />  <br />                                                                                                                                                                                                                                                                                       25.   Maier AG, Rug M, Brown MT, O&apos;Neill M, Chakravorty S, Szestak T, Chesson J, <br /> References                                                                                                                                                                                                                                                                                  Wu Y, Hughes K, Coppel R, Newbold C, Beeson J, Craig A, Crabb BS, <br /> 1. Luger K, Maeder AW, Richmond RK, Sargent DF, Richmond TJ: X-ray                                                                                                                                                                                                                          Cowman AF: Exported proteins required virulence rigidity of <br />     structure nucleosome core particle 2.8 &#197; resolution. Nature                                                                                                                                                                                                                   Plasmodium falciparum-infected human erythrocytes. Cell 2008, <br />     1997, 389:251-259.                                                                                                                                                                                                                                                                      134:48-61. <br /> 2. Krude T: Chromatin assembly DNA replication somatic cells.                                                                                                                                                                                                               26.   Fidock DA, Wellems TE: Transformation human dihydrofolate <br />     Eur J Biochem 1999, 263:1-5.                                                                                                                                                                                                                                                            reductase renders malaria parasites insensitive WR99210 does <br /> 3. Eitoku M, Sato L, Senda T, Horikoshi M: Histone chaperones: 30 years                                              <br /> </body></html>