<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Sept. 1997, p. 5146&#8211;5155                                                                                  Vol. 17, No. 9 <br /> 0270-7306/97/$04.0010 <br /> Copyright &#169; 1997, American Society Microbiology <br />  <br />  <br />  <br />            SQT1, Which Encodes Essential WD Domain Protein of <br />             <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>, Suppresses Dominant-Negative <br />                Mutations Ribosomal Protein Gene QSR1 <br />           DOMINIC P. EISINGER,&#8224; FREDERICK A. DICK, ELKE DENKE,                                 AND   BERNARD L. TRUMPOWER* <br />                      Department Biochemistry, Dartmouth Medical School, Hanover, New Hampshire 03755 <br />                          Received 14 January 1997/Returned modification 12 March 1997/Accepted 3 June 1997 <br />  <br />              QSR1 essential <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21'  ></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae gene, encodes 60S ribosomal subunit protein required <br />           joining 40S 60S subunits. Truncations QSR1 predicted encode C-terminally truncated forms of <br />           Qsr1p substitute QSR1 act dominant negative mutations, inhibiting growth yeast <br />           expressed inducible promoter. The dominant negative mutants exhibit polysome profile <br />           characterized &#8216;half-mer&#8217; polysomes, indicative subunit joining defect like seen qsr1 mutants <br />           (D. P. Eisinger, F. A. Dick, B. L. Trumpower, Mol. Cell. Biol. 17:5136&#8211;5145, 1997.) By screening high-copy <br />           yeast genomic library, isolated clones containing overlapping inserts novel gene rescues the <br />           slow-growth phenotype dominant negative qsr1 truncations. The suppressor qsr1 truncation mutants, <br />           SQT1, essential gene, encodes 47.1-kDa protein containing multiple WD repeats which <br />           interacts strongly Qsr1p yeast two-<span id='am-88' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-89'  ></span><span id='am-90' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>hybrid</span> system. SQT1 restores growth &#8220;half-mer&#65533;? polysome <br />           profile dominant negative qsr1 mutants normal, does rescue temperature-sensitive qsr1 <br />           mutants original qsr1-1 missense allele. In yeast cell lysates, Sqt1p fractionates oligomeric <br />           protein complex loosely associated ribosomes distinct known eukaryotic initiation <br />           factor complexes. Loss SQT1 function regulation inducible promoter results formation <br />           half-mer polyribosomes decreased Qsr1p levels free 60S subunits. Sqt1p appears involved <br />           late step 60S subunit assembly modification cytoplasm. <br />  <br />  <br />    The functions individual eukaryotic ribosomal proteins                 physically interact Qsr1p. SQT1 encodes cytosolic pro- <br /> are understood. We undertaken biochemical                    tein appears loosely associated ribosomes and <br /> and genetic approach elucidate function recently               involved late stage large ribosomal subunit assembly or <br /> discovered ribosomal protein, Qsr1p, Saccharomyces cerevi-                maturation. <br /> siae (13). Qsr1p eukaryotic 60S ribosomal protein encoded <br /> by essential single-copy S. cerevisiae gene QSR1 (13, 54)                                        MATERIALS AND METHODS <br /> and mammalian homolog rat ribosomal protein L10                       Construction <span id='am-93' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-94' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span>. Plasmid pDEGQ2 (CEN URA3 GALUAS::QSR1), <br /> (9).                                                                         GAL1-10 upstream activating sequence (UAS) modulating the <br />    Characterization Qsr1p temperature-sensitive qsr1               expression QSR1 minimal promoter (14), used construct three <br /> mutants revealed Qsr1p required joining 40S               C-terminal truncations QSR1-encoded protein follows. pDEGQ2 was <br />                                                                              digested HindIII remove internal 474-bp fragment QSR1 open <br /> and 60S subunits active 80S ribosomes inactive                reading frame encoding amino acids 64 221, immediately preceding stop <br /> 80S couples (14). Mutant alleles temperature                 <span id='am-38' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42'  ></span>codon</span>. Self-ligation remaining plasmid backbone generated pDEGQ64, in <br /> sensitive conditional synthesis Qsr1p 60S                 N-terminal 64 amino acids QSR1-encoded protein retained. <br /> subunits devoid Qsr1p unable join                <span id='am-33' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-37'  ></span>This</span> truncation referred qsr1-64D. Constructs pDEGQ133 and <br />                                                                              pDEGQ187 ligating HindIII-digested QSR1 PCR fragments into <br /> 40S subunits. Whereas conditional mutants 60S ribo-                  HindIII pDEGQ2 backbone. The fragment making pDEGQ133 was <br /> somal protein genes defective biogenesis 60S                   synthesized PCR primers DEQ133HIII (59-GCGTAAGCTTGACCAA <br /> subunits (11, 12, 35, 38, 39, 45, 47, 48), precludes insights          TGTCGACA [coding strand]) FDYGST59 (59-AACGGATCCAAGATGG <br /> into postassembly functions, biogenesis 60S subunits              CTAGAAGACC [noncoding strand]). The resulting construct N-termi- <br />                                                                              nal 133 amino acids encoded open reading frame referred as <br /> is slightly impaired qsr1 mutants (14). Further investi-             qsr1-133D. The PCR fragment making pDEGQ187 synthesized with <br /> gation Qsr1p likely lead better understanding             primers DEQ187HIII (59-CCTCACAAGCTTCTCTTC [coding strand]) and <br /> of late stages translation initiation ribosomal subunit            FDYGST59 (59-AACGGATCCAAGATGGCTAGAAGACC [noncoding strand]). <br /> joining, little currently known.                                 The resulting construct N-terminal 187 amino acids encoded QSR1 <br />                                                                              open reading frame referred qsr1-187D. <br />    To learn function Qsr1p, constructed                    A 1.8-kb PCR fragment encompassing SQT1 open reading frame was <br /> dominant-negative qsr1 mutants isolated novel essential                High Fidelity Taq polymerase (Boehringer Mannheim) the <br /> gene (SQT1) high-copy suppressor dominant-neg-                   primers DESQTP3 (59-CAAGGATCCCCATTCAAACCCCATC [coding strand]) <br /> ative slow-growth phenotype. The dominant-negative mutants                   DESQTP1 (59-AGAGAAGGATCCCCAAAG [noncoding strand]) which <br />                                                                              contain BamHI sites cloning. The PCR fragment 277 bp 39-flanking <br /> have defect ribosomal subunit joining rescued                sequence 258 bp 59-flanking sequence cut BamHI cloned <br /> SQT1. By directed two-hybrid analysis, Sqt1p                BamHI sites pFL36 pFL46S (5) generate pDESQT2 (SQT1 <br />                                                                              CEN LEU2) pDESQT1 (SQT1 2mm LEU2). <br />                                                                                 To make myc-tagged version SQT1-encoded protein, primers were <br />                                                                              synthesized long-inverse PCR (15) myc epitope recognized by <br />    * Corresponding author. Mailing address: Department Biochem- <br />                                                                              monoclonal antibody 9E10 (31) added C terminus Sqt1p. <br /> istry, Dartmouth Medical School, Hanover, NH 03755. Phone: (603)             Primer DSQTM1 (59-AAGCTTTTGTTCACCACCGTTTGGTACCTCGAAC <br /> 650-1621. Fax: (603) 650-1389. E-mail: Trumpower@Dartmouth.edu.              AC [noncoding strand]) primer DSQTM2 (59-ATTTCTGAAGAAGACT <br />    &#8224; Present address: Argonex Inc., c/o W. Alton Jones Cell Science          TGTGATAGATAGTCGCGTATACATA [coding strand]) add myc epitope <br /> Center, Lake Placid, NY 12946.                                               (EQKLISEEDL) preceded glycines C-terminal asparagine <br />  <br />                                                                       5146 <br />  VOL. 17, 1997                                                          SQT1 SUPPRESSES DOMINANT-NEGATIVE QSR1 MUTATIONS                                             5147 <br />  <br />  <br />                                                          TABLE 1. Yeast strains used study <br />      Strain                                                                       Genotype                                                                     Source <br />  <br /> W303-1A                    ade2-1 his3-11,15 trp1-1 leu2-3,112 ura3-1 can1-100                                                                             54 <br /> W303-1B                    ade2-1 his3-11,15 trp1-1 leu2-3,112 ura3-1 can1-100                                                                             54 <br /> HFF10                      a/a W303 markers, QSR1/qsr1D1::HIS3                                                                                               54 <br /> DEH64                      W303 markers, QSR1, 1 pDEGQ64 (GAL1-10UAS-qsr1-64D CEN URA3)                                                                    This   study <br /> DEH133                     W303 markers, QSR1, 1 pDEGQ133 (GAL1-10UAS-qsr1-133D CEN URA3)                                                                  This   study <br /> DEH187                     W303 markers, QSR1, 1 pDEGQ187 (GAL1-10UAS-qsr1-187D CEN URA3)                                                                  This   study <br /> DEH2212                    W303 markers, QSR1, 1 pDEGQ2 (GAL1-10UAS-QSR1 CEN URA3)                                                                         This   study <br /> DESQT1B                    DEH64, 1 pSQT1B (SQT1 2mm LEU2)                                                                                                 This   study <br /> DEQSR4                     DEH64, 1 pQSR-4 (QSR1 2mm LEU2)                                                                                                 This   study <br /> YPH500 X 499               a/a ura3-52 lys2-801 ade2-101 trp1-D63 his3-D200 leu2-D1                                                                          49 <br /> DPE1                       a/a YPH markers, SQT1/sqt1D1::HIS3                                                                                                This   study <br /> DPESM1                     a/a DPE1 1 pSQTMYC1 (SQT1-myc 2mm LEU2)                                                                                           This   study <br /> DPESM2-1A                  YPH markers, sqt1D1::HIS3 1 pSQTMYC2 (SQT1-myc CEN LEU2)                                                                        This   study <br /> FAD13-2B                   YPH markers, sqt1D1::HIS3 1 pFAD30 (GAL-SQT1-myc CEN URA3)                                                                      This   study <br /> EGY48                      atrp1 ura3-52 his3 leu2 LEU2::pLexAop6-Leu2                                                                                       20 <br /> Y190                       aleu2-3,112 ura3-52 trp1-901 his3-D200 ade2-101 gal4Dgal80D (URA3GAL-lacZ LYSGAL-HIS3 Cyhr)                                       22 <br />  <br />  <br />  <br /> and stop codon SQT1. pDESQT1 pDESQT2 used templates                plating density estimated 15,000 transformants 10-cm petri <br /> for addition myc tag, creating 2mm (pSQTMYC1) CEN (pSQT-                   dish. After 4 days 30&#176;C, 15 large colonies restreaked fresh plates. Total <br /> MYC2) versions, respectively, myc-tagged SQT1. Primers FDSQTP1                     yeast DNA purified transformed E. coli, purified plasmids were <br /> (59-GGATCCCCAAAGAACTTACC [coding strand]) FDSQTP2 (59-GG                          retransformed DEH64 link rescue growth inhibition caused the <br /> ATCCGCGTAATGGAACCTCAAG [noncoding strand]), contain                        galactose-induced qsr1-64D library-derived plasmid. Nine clones rescued <br /> BamHI sites, used amplify SQT1-myc plasmid pSQTMYC2, using               growth defect analyzed restriction mapping and/or partial se- <br /> the High Fidelity Taq polymerase (Boehringer Mannheim). The resulting 1.6-kb          quencing. Deletions chromosome 9 genomic clone 12 long- <br /> fragment ligated pCR2.1 TA cloning kit (Invitrogen). The            inverse PCR (15) follows. The 12.6-kb pSQT12A blunt-end <br /> 1.6-kb insert excised BamHI ligated unique BamHI site        self-ligation PCR product vector primers YEPB3 (59-GGAGGA <br /> pBM258. The resulting plasmid, contains SQT1-MYC control           TCCACAGGACGGGTGTGGTC) DESQTP1 (see above), 11-kb <br /> the GAL1-10 promoter, named pFAD30. All plasmids amplified            pSQT12B primers YEPB3 DESQTP2 (59-CTAGGATCCT <br /> in <span id='am-51' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-52' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-56'  ></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span>Escherichia coli</span> DH5a (21), DNA constructs verified sequenc-       GCTCAACTTCTTCGG [noncoding strand]). <br /> ing Applied Biosystems no. 373 DNA sequencer.                                    Polysome ribosome subunit analysis. Ribosomal subunits, 80S ribosomes, <br />    S. cerevisiae strains growth conditions. Yeast cells grown yeast       polysomes fractionated sucrose velocity gradients (14). The cells <br /> extract-peptone-dextrose (YPD), synthetic complete (SC), synthetic dropout         treated cycloheximide arrest translating ribosomes, gradi- <br /> (SD) medium relevant nutritional supplements omitted.               ents run Mg cycloheximide. To dissociate ribosomal subunits, the <br /> Where indicated, galactose substituted dextrose carbon source. A         cells collected cycloheximide treatment, Mg cycloheximide <br /> lithium acetate procedure used transform yeast cells (1). The yeast        omitted solutions. <br /> strains used listed Table 1 constructed described below. Haploid           Directed two-hybrid interaction analysis SQT1 QSR1. The yeast strain <br /> strains wild type chromosomal QSR1 harboring galactose-inducible C-           EGY48, LexA fusion plasmid pEG202, positive control LexA-GAL4 <br /> terminal qsr1 truncation mutants separately transforming             plasmid, LexAop-lacZ reporter plasmid pSH18-34 provided by <br /> centromeric plasmid pDEGQ64, pDEGQ133, pDEGQ187 HFF10,                      <span id='am-78' about='obo:NCBITaxon_1009818' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_263623 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_71526 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_7080 obo:NCBITaxon_50557 obo:NCBITaxon_115347 obo:NCBITaxon_7041'><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Roger" datatype="xsd:string"></span><span id='am-82'  ></span>Roger</span> Brent (1, 20). Strain Y190 (22) gift Steve Elledge. The GAL4 <br /> hemizygous QSR1 diploid, isolating Ura1 His2 spores generate strains           DNA binding domain vector pAS2-1 GAL4 activation domain vector <br /> DEH64, DEH133, DEH187, respectively. DEH2212 isolated                  pACT2-1 Clontech Laboratories, Inc. pEG202-QSR1, pAS2-1-QSR1, <br /> same manner full-length galactose-inducible wild-type QSR1 gene.            pACT2-QSR1 contain BamHI-BglII PCR fragment encoding full- <br /> DESQT1B derived DEH64 additional plasmid (pSQT1B)                  length QSR1 protein. The fragment generated PCR amplification of <br /> which contains SQT1 locus isolated high-copy genomic             pMY7 (54) primers HFF1 (59-CTGGATCCAGATGGCTCAAGACC) and <br /> library. DESQR4 derived DEH64 plasmid (pDEQSR4)                HFF2 (59-GAAAAGATCTTAAGCTTGAGCAGC [noncoding strand]) cre- <br /> which contains QSR1 locus isolated genomic library.              ate BamHI BglII sites, respectively, underlined primer <br />    A null allele SQT1 isogenic diploid YPH499 3 YPH500          sequences. The BamHI-BglII fragment ligated BamHI sites of <br /> targeted deletion SQT1 open reading frame (3). Two primers,          pEG202, pAS2-1, pACT2. To construct pEG202-SQT1, pAS2-1-SQT1, and <br /> 36 bp (34) SQT1 sequence flanking HIS3 gene sequence, used amplify         pACT2-SQT1, PCR fragment generated plasmid pSQT1B prim- <br /> the HIS3 gene pRS303 (49). Primer SQT1HIS1 (59-CAAGACGTTCCC                      ers DESQTP5 (59-CAAGGATCCCTCAAGAAGAGTTTATAAC) DESQ <br /> GTTGATATTGAAGGAGAGAATGACGGCCTCCTCTAGTACACTC [cod-                                     TP6 (59-CTTGGATCCTTGGTACCTCGA.ACACCA [noncoding strand]) to <br /> ing strand]) contains nucleotides 73 108 SQT1 fused 59 HIS3 se-          create BamHI sites, fragment cloned pEG202, pAS2-1, <br /> quence (shown italics). Primer SQT1HIS2 (59-TCAGTTTGGTACCTCGAA                     pACT2. The open reading frames proteins translational <br /> CACCAGAGATACACCTTCGCGCGCCTCGTTCAGAATG [noncoding                                      phase DNA binding activation domains <span id='am-74' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="vectors" datatype="xsd:string"></span>vectors</span>. The accuracy <br /> strand]) contains nucleotides 1296 1261 SQT1 fused 39 HIS3 se-           reading frames verified sequencing junctions. <br /> quence (shown italics). The amplified fragment transformed directly          Cells prepared exponentially growing cultures. To monitor inter- <br /> strain YPH499 3 YPH500 (49), His1 transformants selected                 action fusion proteins Y190, 50 mM 3-amino-1,2,4-triazole added <br /> screened colony PCR (3) generate strain DPE1. Proper integration         His dropout plates inhibit low levels leaky HIS3 expression this <br /> HIS3 gene SQT1 locus confirmed PCR genotyping                 reporter strain. Colony color b-galactosidase assays performed di- <br /> predicted unique fragments generated internal HIS3 primers exter-         rectly plates containing 5-bromo-4-chloro-3-indolyl-b-D-galactopyranoside <br /> nal SQT1 primers.                                                                     (X-Gal) (1) colony lift filter assay (Clontech). b-Galactosidase activity in <br />    Strain DPESM2-1A, plasmid pSQTMYC2 (SQT1-MYC CEN LEU2)                    solution assayed o-nitrophenyl-b-D-galactopyranoside method (Clon- <br /> contains SQT1-MYC tag allele covering chromosomal SQT1 deletion               tech) cells permeabilized freezing liquid nitrogen thawing <br /> (sqt1D1::HIS3), isolating Leu1 His1 spore derived diploid          37&#176;C. b-Galactosidase activity expressed units described Miller (37). <br /> strain DPE1 transformed pSQTMYC2. Strain DPESM1 diploid strain                   Cell fractionation immunoblot analysis proteins. Yeast cells were <br /> DPE1 plasmid pSQTMYC1 (SQT1-MYC 2mm LEU2). DPE1 trans-                       recovered 25-ml cultures centrifugation absorbance 600 nm <br /> formed pFAD30 sporulated. The spores dissected YPGal                 1.0. The cells washed 1.5 ml 20 mM Tris (pH 7.5)&#8211;50 mM <br /> plates, spore carrying sqt1D1::HIS3 allele complemented GAL1-        NaCl&#8211;0.4 M sorbitol (lysis buffer); resuspended 250 ml lysis buffer plus 10 <br /> 10-controlled SQT1-MYC identified named FAD13-2B.                             mM leupeptin, 1 mg pepstatin A ml, 1 mM diisopropyl fluorophos- <br />    Isolation high-copy suppressor dominant-negative qsr1 truncation.        phate; glass beads added 50% total volume. The cells were <br /> Strain DEH64 grown SD medium lacking uracil, transformed                broken vortexing 5 min 4&#176;C, lysates cleared centrifu- <br /> LEU2-marked YEp13 yeast genomic library (43), allowed recover YPD           gation 500 3 g 5 min 4&#176;C. A 30-ml volume supernatant kept <br /> medium 1 h 30&#176;C plating SD galactose plates lacking uracil       cell lysate. The remainder centrifuged 14,000 rpm 20 min, and <br /> leucine. The transformation efficiency 1.2 3 105 mg library plasmid,       supernatant retained crude cytosolic fraction. The pellet washed <br />  5148        EISINGER ET AL.                                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br /> by resuspended 200 ml lysis buffer recentrifuged, super- <br /> natant discarded. The crude membrane pellet resuspended 30 ml of <br /> lysis buffer. Protein concentrations determined dye-binding assay (7). <br /> Protein fractions denatured electrophoresis incubation sodium <br /> dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) sample buffer <br /> (32) 55&#176;C 15 min. <br />    Fractionation yeast cells isolate nuclei carried essentially as <br /> described Wise (57). The proteins resolved 10% SDS-PAGE gels (1), <br /> unless stated, analyzed Western blotting (13). After being <br /> electroblotted nitrocellulose, membrane blocked 5% nonfat dry <br /> milk 20 mM Tris (pH 7.5)&#8211;150 mM NaCl&#8211;0.3% Tween 20 (TTBS) least <br /> 1 h incubated 1 h primary antibody diluted 1:2,000 1% milk <br /> in TTBS. The primary antibodies used rabbit polyclonal anti-Qsr1p (13), <br /> rabbit polyclonal anti-Prt1p (10), rabbit polyclonal anti-Nop1p (52), mouse <br /> monoclonal anti-Myc (31). After blots washed TTBS, were <br /> incubated horseradish peroxidase-conjugated goat anti-rabbit antibodies at <br /> a 1:2,500 dilution horse anti-mouse antibodies 1:10,000 dilution in <br /> 1% milk TTBS 1 h, washed again, visualized Lumiglo <br /> chemiluminescent (Kirkegaard &amp; Perry, Gaithersburg, Md.) exposure <br /> to Hyperfilm-ECL (Amersham, Arlington Heights, Ill.). <br />    Gel filtration Sqt1p salt-washed ribosomes. Ribosomes purified <br /> by modification procedure used isolate translation initiation factors (10). <br /> Strain DPESM2-1A (16 liters) grown YPD absorbance 600 nm of <br /> 1.2, cells collected centrifugation. The cells washed in <br /> water 20 mM Tris (pH 7.5)&#8211;10 mM MgSO4&#8211;100 mM KCl&#8211;1 mM <br /> dithiothreitol (ribosome lysis buffer), resuspended 20 ml ribosome lysis <br /> buffer, frozen lysed liquid N2 Waring blender (1). <br />    The broken cells thawed ice, protease inhibitors added as <br /> described above. The lysate cleared sequential centrifugation 5,000 3 <br /> g 15 min 22,000 3 g 15 min Sorvall SS34 rotor, the <br /> ribosomes pelleted centrifugation 200,000 3 g 2 h Beckman <br /> Ti60 rotor. The postribosomal supernatant collected stored 270&#176;C. <br /> The ribosomal pellet resuspended 15 ml 20 mM Tris (pH 7.5)&#8211;10 mM                 FIG. 1. (A) C-terminal truncation mutations 221-amino-acid Qsr1p <br /> MgSO4&#8211;500 mM KCl&#8211;1 mM dithiothreitol (ribosome wash buffer) plus protease               constructed making in-frame deletions fused residue <br /> inhibitors centrifuged 2 h 200,000 3 g. The supernatant (ribo-         stop codon QSR1. (B) Immunoblot analysis dominant-negative qsr1 <br /> somal salt wash) dialyzed overnight 2 liters ribosome lysis buffer       truncation strains anti-Qsr1p antibodies. The strains DEH64 (GALUAS- <br /> and frozen 270&#176;C use.                                                  qsr1-64D), DEH133 (GALUAS-qsr1-133D), DEH187 (GALUAS-qsr1-187D), <br />    A 2-mg sample ribosomal salt wash post ribosomal supernatant           DEH221-(GALUAS-QSR1), W303, DESQT1B (DEH64 rescued SQT1 <br /> was loaded Superose 6 gel filtration column Pharmacia fast protein          expressed 2mm plasmid). Western blots lysates cells grown on <br /> liquid chromatography equilibrated ribosome lysis buffer. Forty             galactose dextrose shown lanes 1 6 lanes 7 12, respectively. <br /> 0.5-ml fractions collected analyzed Western blotting. The column <br /> was calibrated thyroglobulin (670 kDa), b-galactosidase (540 kDa), gamma <br /> globulin (158 kDa), ovalbumin (44 kDa), myoglobin (17 kDa), vitamin B12 <br /> (1.3 kDa). <br />    Nucleotide sequence accession number. The SQT1 nucleic acid sequence             culture, doubling times increased 1.5 3.5 h, the <br /> been assigned GenBank accession no. U75717.                                             cellular morphology appeared normal (<span id='am-83' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-84'  ></span><span id='am-85' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> shown). <br />                                                                                            In effort quantitate relative amounts Qsr1p and <br />                                    RESULTS                                              truncated Qsr1 proteins, haploid QSR1 strains carrying the <br />                                                                                         various dominant-negative alleles plasmids grown on <br />    Isolation dominant-negative QSR1 mutants. To investi-                             galactose dextrose analyzed Western blotting. The <br /> gate QSR1-encoded protein essential,                           blot Fig. 1B overexposed trun- <br /> C-terminal truncations Qsr1p constructed deleting                            cated proteins, predicted 21.6, 15.3, 7.5 kDa, be <br /> portions QSR1 (Fig. 1) placing resulting constructs                          detected induced growth galactose. Furthermore, <br /> under control GAL1-10 upstream activating se-                                endogenous Qsr1p levels remain unchanged an <br /> quence (UAS). The truncated peptides 64 (encoded                                 extra wild-type copy galactose-inducible QSR1 present. <br /> GALUAS-qsr1-64D), 133 (encoded GALUAS-qsr1-133D),                                The apparently invariant Qsr1p consistent with <br /> 187 (encoded GALUAS-qsr1-187D) residues length com-                               previous findings unassembled ribosomal proteins have <br /> pared 221 residues (encoded GALUAS-QSR1)                                   short half-lives degraded rapidly (56). Although the <br /> full-length Qsr1p. These plasmids transformed                            antibodies detect truncated forms Qsr1p, <br /> HFF10, strain hemizygous QSR1 (54), tetrads                              shortest (qsr1-64Dp) longest (qsr1-187Dp) truncated <br /> dissected sporulated diploid. None deletion                             proteins apparently present sufficient quantities exert <br /> alleles cover QSR1 chromosomal deletion,                            dominant-negative effect. The failure antibodies to <br /> full length GALUAS-QSR1 allele covered deletion strain                              detect truncated proteins antibodies <br /> when grown galactose dextrose (data shown).                           recognize C-terminal portion Qsr1p deleted <br /> Examination spores failed grow showed did                          constructs. This conserved region of <br /> not divide once, indicating nonconserved C-                               protein (15). <br /> terminal 34 residues (16) essential QSR1 function.                                 Isolation characterization SQT1. We exploited the <br />    We noticed diploids haploids harboring long-                             dominant-negative effect Qsr1p truncations growth <br /> est deletion (GALUAS-qsr1-64D) shortest deletion                                search high-copy suppressors genetic screen. Strain <br /> (GALUAS-qsr1-187D) grew slower carrying                                 DEH64 (QSR1 pDEGQ64[GALUAS-qsr1-64D CEN URA3]) <br /> full-length QSR1 spread galactose plates. The domi-                             transformed LEU2-marked high-copy genomic li- <br /> nant-negative slow growth phenotype stable                               brary, fast-growing colonies picked. Screening 105 <br /> multiple switches galactose dextrose media with-                            transformants yielded 16 large colonies. After passage through <br /> out occurrence spontaneous revertants, growth                            E. coli, library plasmids recovered fast- <br /> defect evident temperatures 16 37&#176;C. In liquid                           growing colonies suppressed slow-growth phenotype when <br />  VOL. 17, 1997                                                          SQT1 SUPPRESSES DOMINANT-NEGATIVE QSR1 MUTATIONS                                    5149 <br />  <br />  <br />  <br />  <br />    FIG. 2. Rescue dominant-negative growth defect SQT1. (A) Map of <br /> the SQT1 locus genomic clones carrying SQT1. The figure shows <br /> the open reading frames restriction map vicinity SQT1 long <br /> arm chromosome IX. The SQT1 open reading frame separated              FIG. 3. (A) The deduced amino acid sequence Sqt1p sequences <br /> divergently transcribed gene DBF8 (27) 242 bp. Clones 1B, 8, 15, 12   WD repeats underlined. The middle repeat (residues 135 183) <br /> isolated complementing slow-growth phenotype qsr1-64D mutant        conforms consensus sequence Neer et al. (44), insertion <br /> strain DEH64. Clones 12A 12B subclones clone 12. The ability    10 residues instead 3 initial VH. (B) Tetrad analysis the <br /> clones rescue slow-growth phenotype strain DEH64 indicated    hemizygous sqt1D1 strain DPE1 showing SQT1 null allele (sqt1D1) <br /> right. A restriction map given HindIII (H), EcoRI (E), BamHI (B),      segregates 2:2 lethality. <br /> XhoI (X). (B) Effect qsr1 truncations yeast growth. The strains DEH64 <br /> (GALUAS-qsr1-64D), DEH133 (GALUAS-qsr1-133D), DEH187 (GALUAS-qsr1- <br /> 187D), DEH2212 (GALUAS-QSR1), DESQT1B (DEH64 rescued SQT1 <br /> expressed 2mm plasmid), DESQT4 (DEH64 rescued QSR1 <br /> expressed 2mm plasmid).                                                    When minimal 1.8-kb fragment encompassing SQT1 <br />                                                                                   open reading frame cloned centromeric vector, it <br />                                                                                   suppressed growth defect galactose (data not <br /> retransformed strain DEH64. Restriction digestion anal-                      shown), suggesting relationship SQT1 QSR1 is <br /> ysis partial sequencing genomic inserts revealed                  specific, weaker nonspecific interaction most <br /> five plasmids carried identical overlapping                  probably require overexpression typical high-copy plas- <br /> clones QSR1 remaining carried overlapping                         mid. <br /> clones novel DNA fragment mapped region                            The SQT1 open reading frame encodes predicted protein <br /> the long arm chromosome IX (54a). The genomic                             431 amino acids molecular mass 47.1 kDa a <br /> chromosome IX clones subclones depicted Fig.                       calculated pI 4.16 (Fig. 3A). Inspection deduced <br /> 2A single open reading frame                         amino acid sequence search databases reveals that <br /> named SQT1 (suppressor QSR1 truncations) suppresses                        SQT1 encodes novel protein sequence similarity the <br /> slow-growth phenotype caused dominant-negative                             b-transducin WD repeats (44). The deduced Sqt1p sequence <br /> QSR1 truncations presence chromosomal copy                         includes two, possibly three, consensus WD re- <br /> QSR1. Examination DNA sequence surrounding                             peats (Fig. 3A). <br /> open reading frame reveals TATA-like                          Originally identified b-subunit heterotrimeric GTP- <br /> elements (consensus TATAA) region 50 120 bp up-                         binding proteins, WD-repeat motifs present <span id='am-25' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-26' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> of <br /> stream open reading frame (51) putative poly-                        conserved eukaryotic proteins common <br /> adenylation signal (55) centered position 1349.                                function known members regulatory proteins <br />    The dominant-negative effect growth GALUAS-                          enzymes (44). Searches nucleic acid and <br /> qsr1-64D GALUAS-qsr1-187D alleles rescue                           protein data banks revealed significant homologies Sqt1p <br /> GALUAS-qsr1-64D allele high-copy QSR1 SQT1                             limited proteins major WD repeats. <br /> shown Fig. 2B. SQT1 allele-specific suppressor                  The homologous proteins include known WD proteins as <br /> does rescue lethality QSR1 chromosomal dele-                       Tup1p, involved gene regulation (30); Ret1p, a <br /> tion, does suppress temperature-sensitive qsr1 mutants                     coatamer protein involved vesicular traffic (44); AAMP, <br /> or original qsr1-1 missense mutation (data shown).                        mammalian mediator heparin-sensitive cell adhesion (4). <br />  5150     EISINGER ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />    SQT1 essential gene. A hemizygous SQT1/sqt1D1:: <br /> HIS3 diploid strain (DPE1) created replacing chro- <br /> mosomal copy SQT1 HIS3 gene. Sporulation the <br /> diploid dissection tetrads yielded viable spores from <br /> each diploid, shown Fig. 3B, viable spores <br /> were His2 (data shown), indicating SQT1 essen- <br /> tial gene. Microscopic examination spores failed to <br /> grow revealed germinated failed progress <br /> beyond two- six-cell stage. The 1.8-kb SQT1 complement- <br /> ing fragment centromeric plasmid (pDESQT2) trans- <br /> formed strain DPE1 rescued lethality frequency <br /> compatible segregation centromeric plasmid when <br /> the hemizygous diploid sporulated (data shown). <br />    SQT1 rescues defect 40S 60S ribosomal-subunit join- <br /> ing caused dominant-negative qsr1-64D mutant. Since <br /> previous studies shown QSR1 essential the <br /> ability 60S subunits dock 40S subunits form 80S <br /> ribosomes (14), investigated effect dominant- <br /> negative qsr1-64D allele distribution ribosomal sub- <br /> units polysomes. Figure 4 shows polysome profiles the <br /> qsr1-64D strain repressed dextrose, induced galactose, <br /> and rescued SQT1. The qsr1-64D strain grown dextrose <br /> has polysome profile typical wild-type cells is <br /> a large 80S peak polysome peaks of <br /> the gradient. When dominant-negative qsr1-64D protein is <br /> induced galactose, 80S polysome peak sizes are <br /> dramatically reduced, half-mer polysomes seen the <br /> heavier 80S peak smaller polysomes (Fig. 4B). <br /> The polysome profile restored dextrose con- <br /> dition SQT1 present centromeric plasmid (Fig. <br /> 4C). <br />    Half-mer polysomes result 60S-40S subunit-joining <br /> defect stalled 43S preinitiation complexes an <br /> mRNA template containing translating 80S ribo- <br /> some joined 60S subunit (23, 47). A defect in <br /> the assembly 60S subunits (stoichiometric imbalance) a <br /> subunit joining defect leads half-mers. Most reported half- <br /> mers occur mutants display depleted mutated 60S <br /> ribosomal proteins defective assembling 60S <br /> subunits (11, 12, 35, 38, 39, 45, 47, 48). We previously            FIG. 4. Changes polysome profiles resulting expression the <br /> shown temperature-sensitive mutations QSR1 deple-          dominant-negative qsr1-64D allele rescue SQT1. Cells pretreated <br />                                                                       cycloheximide, lysates fractionated 5 47% sucrose gradients <br /> tion Qsr1p regulation inducible promoter             containing Mg cycloheximide. The 40S, 60S, 80S subunit peaks are <br /> results population 60S subunits lacking Qsr1p        marked absorbance profile. The arrows panel B mark location of <br /> defective subunit joining opposed deficiency 60S        half-mer polysomes. (A) DEH64, GALUAS-qsr1-64D grown dextrose; (B) <br /> subunits (14). The degree half-mer formation greater         DEH64, GALUAS-qsr1-64D grown galactose; (C) DESQT1B, DEH64, <br />                                                                       GALUAS-qsr1-64D grown galactose rescued SQT1 expressed a <br /> the temperature-sensitive strains strain expressing       2mm plasmid. <br /> the induced dominant-negative qsr1-64D allele. <br />    To investigate dominant-negative qsr1-64D mu- <br /> tation causes subunit imbalance, examined ratio of <br /> 40S 60S subunits running polysome gradients lack           binding protein pEG202 (20) GAL4 activation <br /> Mg, causes 80S ribosomes dissociate (14, 18, 46). As         domain pACT2 (17). Combinations different bait and <br /> shown Table 2, small significant increase    prey plasmids cotransformed LexA-lacZ <br /> ratio 40S 60S subunits qsr1-64D allele induced      reporter plasmid pSH18-34 EGY48 reporter strain, <br /> on galactose, ratio restored essentially    transformants identified growth selective me- <br /> dextrose condition high-copy SQT1 present. Although           dium. <br /> the subunit imbalance slight, difficult say      Interaction fusion proteins monitored by <br /> half-mers result subunit imbalance caused trun-         streaking colonies X-Gal medium performing <br /> cated Qsr1p truncated Qsr1p inhibits subunit           b-galactosidase assays. As shown Fig. 5, LexA- <br /> joining mechanisms discussed below.                     QSR1 fusion LexA-SQT1 fusion coexpressed with <br />    SQT1 QSR1 interact two-hybrid assay. To test          GAL4 activation domain alone, cells showed bas- <br /> interactions Qsr1p Sqt1p, used different           al-level b-galactosidase activity, observed when <br /> bait vectors expressing fusion proteins LexA          LexA protein expressed GAL4 acti- <br /> DNA-binding protein GAL4 DNA binding domain. To                vation domain. This basal-level activity corresponded white <br /> express second hybrid, fused open reading frames           colonies X-Gal plates demonstrates LexA <br /> encoding proteins GAL4 activation domain. The             hybrids Qsr1p Sqt1p interact GAL4 <br /> full-length Qsr1p Sqt1p fused LexA DNA                portion prey hybrid induce b-galactosidase activity <br />  VOL. 17, 1997                                                            SQT1 SUPPRESSES DOMINANT-NEGATIVE QSR1 MUTATIONS                                        5151 <br />  <br />  <br /> TABLE 2. Ratio 40S 60S subunits wild-type cells, qsr1-64D <br />        mutants, qsr1-64D mutants rescued SQT1 <br />                                               Medium            Mean 40S-to-60S <br />                Strain <br />                                             supplement           ratioa 6 SD <br />  <br /> W303-1B                                     Dextrose           0.470 6 0.04 <br /> DEH64 (GALUAS-qsr1-64D)                     Dextrose           0.419 6 0.016 <br /> DEH64 (GALUAS-qsr1-64D)                     Galactose          0.534 6 0.037 <br /> DESQT1B (GALUAS-qsr1-64D,                   Galactose          0.464 6 0.008 <br />  12mm SQT1) <br />   a <br />     Ratios 40S 60S subunits calculated determining correspond- <br /> ing areas 254-nm absorbance profiles. <br />  <br />  <br />  <br /> themselves. The basal level reporter gene activity was <br /> observed GAL4-QSR1 GAL4-SQT1 prey plas- <br /> mids coexpressed LexA protein. When the <br /> GAL4-SQT1 prey plasmid introduced cells bearing <br /> the LexA-QSR1 fusion construct, resulted b-galactosidase <br /> activity approximately 100-fold higher basal levels (Fig. <br /> 5). Coexpression LexA-SQT1 GAL4-QSR1 prey plas- <br /> mids resulted 25-fold increase b-galactosidase activity. <br /> In cases, colonies blue X-Gal medium. <br />   Similar results obtained QSR1 SQT1 were <br /> fused GAL4 DNA binding domain bait vector                                 FIG. 6. Sqt1p cytosolic protein. Protein fractions strains expressing <br /> pAS2-1 (data shown). Coexpression strain Y190 ei-                          myc-tagged Sqt1p visualized immunoblotting detection the <br /> ther QSR1 SQT1 bait fusion GAL4                         9E10 mouse monoclonal anti-myc antibody. (A) Whole-cell lysates (W), cytosol <br /> activation domain did cause expression HIS3                         (C), membrane (M) fractions indicated yeast strains. The Western <br />                                                                                      blot probed anti-myc antibodies stripped reprobed <br /> or lacZ reporter genes. When QSR1 bait fusion trans-                         anti-Qsr1p antibodies. Lanes: 1, parent strain DPE1, Sqt1p is <br /> formed yeast cells bearing SQT1 prey fusion, cells                      myc tagged; 2, strain DPESM1 (SQT1/sqt1D1) expressing Sqt1-myc protein <br /> grew minimal medium histidine formed blue                             2mm plasmid; 3, strain DPESM2-1A (sqt1D1) expressing Sqt1-myc pro- <br /> colonies b-galactosidase filter assay. Expression                      tein CEN plasmid; 4, cytosolic fraction DPESM2-1A; 5, membrane <br />                                                                                      fraction DPESM2-1A. (B) Strain DPESM2-1A fractionated obtain <br />                                                                                      nuclei, blots probed anti-myc anti-Nop1 antibodies. Lanes: 1, <br />                                                                                      whole-cell lysate; 2, postnuclear supernatant; 3, nuclear fraction. <br />  <br />  <br />  <br />  <br />                                                                                      reporter genes observed SQT1 fused the <br />                                                                                      DNA binding domain QSR1 coexpressed prey <br />                                                                                      fusion. <br />                                                                                         Sqt1p cytosolic protein affinity ribosomes. <br />                                                                                      Sqt1p myc tagged C terminus fusion myc <br />                                                                                      epitope (31) residue SQT1 open reading <br />                                                                                      frame, CEN plasmid carrying myc-tagged SQT1 was <br />                                                                                      transformed hemizygous SQT1 strain DPE1. When <br />                                                                                      diploid sporulated tetrads dissected, the <br />                                                                                      SQT1-myc allele covered SQT1 chromosomal deletion with <br />                                                                                      frequency compatible segregation centromeric plas- <br />                                                                                      mid, deletion strain covered CEN plasmid <br />                                                                                      carrying SQT1-myc grew normal doubling time liquid <br />                                                                                      culture 30&#176;C. <br />                                                                                         Western blot analysis SQT1-myc strains mouse <br />                                                                                      monoclonal anti-myc antibody reveals specific protein which <br />                                                                                      present control strain lacking myc-tagged <br />                                                                                      protein (Fig. 6A) abundant strain <br />                                                                                      expressing myc-tagged protein 2mm plasmid. The <br />                                                                                      myc-tagged protein migrates approximately 52 kDa, com- <br />                                                                                      pared predicted molecular mass 48.5 kDa. Subcellular <br />    FIG. 5. Directed two-hybrid analysis QSR1 SQT1. Proteins ex- <br />                                                                                      fractionation cytosolic membrane fractions the <br /> pressed LexA fusion vector pEG202 derivatives pEG202-QSR1           SQT1-myc strain DPESM2-1A [sqt1D1 pSQTMYC2(SQT1- <br /> and pEG202-SQT1 GAL4 activation domain fusion vector pACT2              MYC CEN LEU2)] shows Sqt1p predominantly the <br /> and derivatives pACT2-QSR1 pACT2-SQT1. LexA, full-length LexA                cytosolic fraction (Fig. 6A). The levels Qsr1p af- <br /> protein; GAD, GAL4 activation domain. Transformants assayed b- <br /> galactosidase activity, activities means 10 15 independent <br />                                                                                      fected gene dosage SQT1 different strains, and <br /> transformants, assayed triplicate. Standard errors basal activities   relative Qsr1p membrane fraction is <br /> were 30% depicted. The GAL4 activation domain LexA                 greater Sqt1p fraction (Fig. 6A). <br /> plasmid pSH17-4 used positive control showed activity 590 U.      The association Qsr1p membrane fraction con- <br />  5152      EISINGER ET AL.                                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 7. Characterization Sqt1p translational machinery. (A) Analysis polysome profiles distribution Sqt1p, Qsr1p, Prt1p strain <br /> DPESM2-1A. Cell lysates fractionated centrifugation 5 47% sucrose gradients. Immunoblots Sqt1p Prt1p aligned tracings. (B) <br /> Fractionation 0.5 M KCl ribosomal salt wash fraction gel filtration chromatography Superose 6 column. The 280-nm absorbance (A280) profile shown <br /> with location known protein standards. Immunoblots Sqt1p Prt1p aligned corresponding column fractions. <br />  <br />  <br />  <br />  <br /> sistent fact 60S subunits bind endoplasmic                   yeast eIF-3, SDS-PAGE yeast eIF-3 does protein in <br /> reticulum relatively high affinity (6, 29).                                  molecular mass range (41). We monitored distri- <br />    One possible explanation rescue subunit-join-                   bution Prt1p, subunit eIF-3 complex (19, 41, 42), in <br /> ing defect mutants 60S subunit assembly im-                     gradient fractions explore possible colocalization <br /> paired truncated Qsr1p rescued SQT1. Since 40S                      Sqt1p eIF-3 complex. The distribution Sqt1p <br /> and 60S subunits assembled nucleolus (36, 56, 58),                     does match Prt1p, concentrated the <br /> we investigated Sqt1p nuclear protein. When cell                     40S subunit (Fig. 7A). The smallest form Prt1p fraction <br /> lysates fractionated recover nuclei postnuclear                       3, agreement 600-kDa eIF-3 bound to <br /> supernatants, Sqt1p recovered supernatant fraction,                    40S subunits run 540-kDa b-galactosi- <br /> while Nop1p, nucleolar marker (52), enriched                         dase marker recovered. <br /> nuclear fraction, shown Fig. 6B.                                               To test possible association Sqt1p the <br />    Since Sqt1p cytosolic protein physically interacts                   eIF-3 complex, washed ribosomes 0.5 M KCl obtain <br /> with Qsr1p, 60S ribosomal subunit protein, analyzed                      extract enriched translational initiation factors (10, 41). <br /> distribution Sqt1p polysome gradients. As shown Fig.                     By quantitative Western blot analysis, estimated 5% of <br /> 7A, majority Sqt1p recovered 0.5 ml                   Sqt1p cell lysate recovered fraction, <br /> polysome gradient, contains free cytosolic                      remaining 95% postribosomal supernatant <br /> proteins. However, Sqt1p does migrate significantly                      (data shown). When salt-washed ribosomal extract was <br /> gradient present 80S ribosome small                        fractionated gel filtration Superose 6 column, Sqt1p <br /> polysome fractions. This unlikely nonspecific tailing,                   concentrated fraction 21, corresponding molecular <br /> since cytosolic markers hexokinase recovered solely                   mass 158 kDa significantly larger 47 <br /> in gradients (13). Furthermore, strain                    kDa predicted Sqt1p (Fig. 7B). The elution profile Sqt1p <br /> (DPESM1) expressing SQT1 high-copy plasmid                             does match Prt1, eluted position <br /> similarly analyzed, excess Sqt1p accumulated                    670-kDa marker, agreement size eIF-3 <br /> top gradient, lower fractions contained                      complex (10). We fractionated postribosomal super- <br /> same relative amounts Sqt1p observed low-copy                      natant Superose 6 column Sqt1p and <br /> strain (data shown). These properties Sqt1p consis-                    Prt1p localized fractions ribosomal salt <br /> tent loose association ribosomes.                                     wash (data shown). These results Sqt1p dis- <br />    The apparently loose association Sqt1p ribosomes                       tinct eIF-3 complex. Sqt1p behaves either <br /> resembles behavior translational initiation factors                   oligomeric associated novel complex 90 150 <br /> (24). Translational initiation factor eIF-3 stable complex                kDa. <br /> at polypeptides aggregate mass approx-                        Depletion Sqt1p results subunit-joining defect. We <br /> imately 600 kDa understood transla-                       analyzed phenotype cells depleted Sqt1p construct- <br /> tional initiation factors (for review, reference 24). eIF-3                 ing yeast strain (FAD13-2B) SQT1 controlled by <br /> is involved steps initiation pathway;              GAL1-10 promoter plasmid. The GAL-SQT1 plasmid <br /> the factors bind 40S subunit, shifting equi-                  covers SQT1 chromosomal deletion grown galac- <br /> librium 40S subunits away association 60S sub-                       tose, cells covered myc-tagged Sqt1p grow the <br /> units, thought factor leave subunit                 rate covered Sqt1p (data shown). By <br /> joining. A recent description mammalian eIF-3 subunits                     Western analysis, determined Sqt1p undetect- <br /> lists 47-kDa protein (24), molecular mass Sqt1p,                     able 8 h shift dextrose medium (data shown). The <br /> although report 47-kDa protein                     GAL-SQT1 strain pregrown dextrose 8 h deplete <br />  VOL. 17, 1997                                                          SQT1 SUPPRESSES DOMINANT-NEGATIVE QSR1 MUTATIONS                        5153 <br />  <br />  <br />                                                                                   8B Fig. 4B). Western blots 60S subunit fractions from <br />                                                                                   wild-type cells GAL-SQT1 strain indicate Qsr1p is <br />                                                                                   significantly depleted 60S fraction 16 h depletion of <br />                                                                                   Sqt1p (Fig. 8C). <br />  <br />                                                                                                            DISCUSSION <br />                                                                                      Truncations QSR1 unable substitute the <br />                                                                                   full-length gene inhibit growth expressed yeast which <br />                                                                                   contain wild-type chromosomal copy QSR1. Of three <br />                                                                                   truncations tested, encoding longest (qsr1- <br />                                                                                   187Dp) shortest (qsr1-64Dp) forms protein acted as <br />                                                                                   dominant-negative mutations, truncation predicted to <br />                                                                                   encode protein intermediate length (qsr1-133Dp) no <br />                                                                                   detectable effect growth, possibly form the <br />                                                                                   protein rapidly degraded present sufficient <br />                                                                                   amounts exert negative effect. It possible de- <br />                                                                                   termine relative amounts truncated proteins, <br />                                                                                   antibodies raised Qsr1p did detect the <br />                                                                                   truncated proteins. We suspect anti- <br />                                                                                   bodies recognize epitopes nonconserved C terminus <br />                                                                                   (16), absent truncated proteins. <br />                                                                                      The QSR1 truncations placed control a <br />                                                                                   galactose-inducible promoter, permitting controlled <br />                                                                                   expression. When dominant-negative qsr1-64D allele was <br />                                                                                   induced growth galactose, examination polysome <br />                                                                                   profile revealed ribosomal subunit-joining defect, evi- <br />                                                                                   denced presence half-mer polysomes. When qsr1- <br />                                                                                   64D allele repressed dextrose, wild-type polysome <br />                                                                                   profile restored. The half-mer phenotype qsr1-64D <br />                                                                                   allele consistent disruption wild-type Qsr1p function, <br />                                                                                   temperature-sensitive qsr1 mutants original <br />                                                                                   qsr1-1 mutant exhibit similar half-mer polysome profiles (14). <br />                                                                                      By screening yeast genomic library, isolated novel <br />                                                                                   suppressor qsr1 truncations, SQT1, rescue the <br />                                                                                   growth inhibition half-mer polysome profile caused by <br />                                                                                   qsr1-64D allele. Although SQT1 initially isolated a <br />                                                                                   high-copy suppressor, subsequent tests established expres- <br />                                                                                   sion CEN plasmid equally effective rescuing the <br />                                                                                   dominant-negative mutants, indicating slight excess of <br />                                                                                   Sqt1p addition derived chromosomal gene <br />                                                                                   sufficient reverse toxic effects truncated pro- <br />                                                                                   teins. <br />                                                                                      The screen high-copy genomic suppressors revealed <br />    FIG. 8. Loss SQT1 function results half-mer polysomes. FAD13-2B cells    QSR1 reverse dominant-negative effect the <br /> were grown galactose medium followed 8 h preculture dextrose          qsr1-64D allele. This suggests excess wild-type <br /> medium, reinoculated fresh dextrose medium. Extracts were <br /> prepared cycloheximide-arrested cells 0 h (A) 16 h (B) reinoc- <br />                                                                                   protein outcompete dominant-negative truncated pro- <br /> ulation dextrose medium analyzed 5 47% sucrose velocity gradi-     tein. In general, dominant-negative polypeptide needs be <br /> ents. Fractions containing 60S subunits pooled gradients             excess inhibitory effects substantial (25). <br /> FAD13-2B prepared (A B) wild-type control (YPH499)            Although SQT1 rescued dominant-negative qsr1 <br /> the 0- 16-h time points used prepare SDS-PAGE samples, which <br /> were probed Qsr1p L3 (C). <br />                                                                                   truncations, rescue temperature-sensitive qsr1 mu- <br />                                                                                   tants. This indicates SQT1 function bypass <br />                                                                                   suppressor temperature-sensitive alleles sub- <br />                                                                                   stitute QSR1. SQT1 rescue original <br /> Sqt1p reinoculated fresh dextrose medium. A                         qsr1-1 mutation, G194D missense mutation C terminus <br /> growth defect evident 12 h reinoculation                    protein semidominant QSR1 (54). <br /> dextrose medium, eventually cells ceased grow (data                    SQT1 appears allele-specific suppressor qsr1 <br /> not shown). Polysome profiles cells immediately rein-                    dominant-negative truncation mutations. The qsr1-1 allele is <br /> oculation indistinguishable wild-type cells                    rescued single chromosomal copy QCR6, gene <br /> (Fig. 8A). However, polysome profiles wild-                    encodes subunit 6 mitochondrial cytochrome bc1 <br /> type cells remained unchanged 16 h dextrose (data                    complex (54). It known gene protein whose <br /> shown), half-mer phenotype clearly present polysomes                      known location inner mitochondrial membrane <br /> from GAL-SQT1 cells decrease 80S                        rescue lethal missense mutation cyto- <br /> and polysome peaks (Fig. 8B). The altered polysome profile                        plasmic ribosomal protein. The failure SQT1 substitute <br /> mimics dominant-negative qsr1 truncation mutants,                     QCR6 indicates different roles SQT1 QCR6 their <br /> including slight decrease 60S abundance (compare Fig.                      relationship QSR1. <br />  5154     EISINGER ET AL.                                                                                                          MOL. CELL. BIOL. <br />  <br />  <br />    There mechanisms truncations QSR1          SQT1. Several known WD-repeat proteins participate in <br /> could exert dominant-negative effect suppressed         assembly multiprotein complexes, pro- <br /> QSR1 SQT1. One possibility truncated Qsr1p             posed common function WD proteins (44). <br /> causes reduced assembly 60S subunits, resulting stoi-         Consistent interpretation fact defect in <br /> chiometric imbalance ribosomal subunits formation           translational initiation expected manifested <br /> half-mer polysomes. This unlikely,        sooner 12 h disappearance Sqt1p (40). <br /> slight change ratio 40S 60S subunits            Interestingly, pronounced growth defects FAD13-2B <br /> qsr1-64D allele induced decrease proportion      strain Sqt1p depleted 8-h <br /> 60S subunits does sufficient account ap-          preculture starting growth curves. This indicates <br /> pearance half-mers, especially comparison previously         Sqt1p function early log-phase growth, most <br /> reported mutants grossly defective assembly        ribosomal biogenesis occurs (28). However, cytosolic loca- <br /> 60S subunits (11, 12, 35, 38, 39, 45, 47, 48). It          tion Sqt1p argues role ribosome assembly the <br /> unlikely SQT1 rescues defect assembly 60S sub-           nucleolus, suggesting important maturation <br /> units. Whereas ribosomal subunits assembled nucle-         modification 60S subunits occurs cytoplasm. A <br /> olus factors affecting assembly subunits         delay assembly Qsr1p immature 60S subunits could <br /> found localized nucleolus (36, 56, 58), Sqt1p         result degradation account slight 40S-60S <br /> excluded nucleus. Additionally, loss SQT1             subunit imbalance observed. If explanation applies, the <br /> function normal genetic background does           cytosolic location Sqt1p imply Qsr1p the <br /> appear impair assembly 60S subunits degree           small group proteins exchanges ribo- <br /> expected prolonged depletion Sqt1p.                    somes cytoplasm (12, 59). <br />    A second explanation dominant-negative effect is <br /> that truncated Qsr1p assembled 60S subunit,                                    ACKNOWLEDGMENTS <br /> where inactivates sequesters closely associ- <br />                                                                         This research supported American <span id='am-28' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-29'  ></span><span id='am-30' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Cancer</span> Society grant BE <br /> ated ribosomal proteins falls off, leaving 60S subunit <br />                                                                       204. F.A.D. supported American Heart Association predoc- <br /> incapable subunit joining. Since Sqt1p loosely         toral fellowship (NH/VT affiliate). <br /> associated ribosomes, conceivable stabilizes         We thank Alan Hinnebusch antibodies Prt1 Ed Hurt for <br /> Qsr1p 60S subunit. This hypothesis consistent          antibodies Nop1p. <br /> the observation loss SQT1 function results smaller <br /> amounts Qsr1p free 60S subunits. However,                                             REFERENCES <br /> obvious SQT1 rescue dominant-negative effect             1. Ausubel, F. M., R. E. Brent, D. D. Moore, J. A. Smith, J. G. Seidman, K. <br /> of qsr1-64D qsr1-187D mutations, correspond-            Struhl. 1996. Current protocols molecular biology. Wiley Interscience, <br />                                                                           New York, N.Y. <br /> ing truncated proteins substitute Qsr1p.                 2. Ballesta, J. P., M. Remacha. 1996. The large ribosomal subunit stalk as <br />    A possibility Sqt1p involved subunit joining       regulatory element eukaryotic translational machinery. Prog. Nu- <br /> and interacts Qsr1p 60S subunit. In case,                cleic Acids Res. Mol. Biol. 55:157&#8211;193. <br /> truncated forms Qsr1p assembled 60S           3. Baudin, A., O. Ozier-Kalogeropoulos, A. Denouel, F. Lacroute, C. Cul- <br />                                                                           lin. 1993. A simple efficient method direct gene deletion Saccha- <br /> subunit sequester Sqt1p prevent interaction             romyces cerevisiae. Nucleic Acids Res. 21:3329&#8211;3330. <br /> with Qsr1p 60S subunit. The distribution Sqt1p            4. Beckner, M. E., H. C. Krutzsch, M. L. Stracke, S. T. Williams, J. A. Gal- <br /> polysome gradients interaction Qsr1p two-             lardo, L. A. Liotta. 1995. Identification new immunoglobulin su- <br /> hybrid indicate interacts ribosomes. We               perfamily protein expressed blood vessels heparin-binding consen- <br />                                                                           <span id='am-43' about='obo:NCBITaxon_9822' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9821 obo:NCBITaxon_35497 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-44' property="rdfs:label" content="Sus" datatype="xsd:string"></span><span id='am-45'  ></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>sus</span> sequence. Cancer Res. 55:2140&#8211;2149. <br /> explored possibility Sqt1p subunit known            5. Bonneaud, N., O. Ozier-Kalogeropoulos, G. Li, M. Labouesse, L. Minivielle- <br /> translation initiation factor. Eukaryotic initiation factors          Sebastia, F. Lacroute. 1991. A family low high copy replicative, <br /> multiprotein complexes defined biochemically               integrative single-stranded S. cerevisiae/E. coli shuttle vectors. Yeast <br /> by reconstituting vitro translation. For reviews, reference        7:609&#8211;615. <br />                                                                        6. Borgese, N. 1974. Ribosomal-membrane interaction: vitro binding ri- <br /> 24 53. Five initiation factors associate 40S subunit         bosomes microsomal membranes. J. Mol. Biol. 88:559&#8211;580. <br /> in specific temporal <span id='am-48' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-49' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> and, possible exception       7. Bradford, M. M. 1976. A rapid sensitive method quantitation of <br /> eIF-2, dissociate joining 60S subunit. The peptide          microgram quantities protein utilizing principal protein-dye bind- <br /> subunits eIF-1, eIF-1A, eIF-2, eIF-4, eIF-5 known,             ing. Anal. Biochem. 72:248&#8211;254. <br />                                                                        8. Chakravarti, D., U. Maitra. 1993. Eukaryotic initiation factor 5 from <br /> and factors function initiation cycle disso-           Saccharomyces cerevisiae. J. Biol. Chem. 268:10524&#8211;10533. <br /> ciation 40S 60S subunits final 60S joining (8, 26, 33,      9. Chan, Y., J. Diaz, L. Denoroy, J. Madjar, I. G. Wool. 1996. The primary <br /> 50). It possible Sqt1p subunit eIF-3             structure rat ribosomal protein L10: relationship Jun-binding protein <br /> (24), initiation factor peptide composition          putative Wilms&#8217; tumor suppressor. Biochem. Biophys. Res. Com- <br />                                                                           mun. 225:952&#8211;956. <br /> characterized (19, 40&#8211;42). However, Sqt1p does colocalize         10. Cigan, M. A., M. Foiani, E. M. Hannig, A. G. Hinnebusch. 1991. Com- <br /> with Prt1p, known subunit eIF-3, polysome gradients            plex formation positive negative translational regulators GCN4. <br /> during size exclusion chromatography behaves            Mol. Cell. Biol. 11:3217&#8211;3228. <br /> oligomeric previously unreported protein complex.        11. Deshmukh, M., J. Stark, L. C. Yeh, J. C. Lee, J. L. Woolford. 1995. <br />                                                                           Multiple regions yeast ribosomal protein L1 important its <br /> Furthermore, inactivation eIF-3 transcriptional re-            interaction 5S rRNA assembly ribosomes. J. Biol. Chem. <br /> pression known subunit results large inactive 80S peak          270:30148&#8211;30156. <br /> (40), half-mer polysomes described here.                       12. Deshmukh, M., Y. Tsay, A. G. Paulovich, J. L. Woolford. 1993. Deple- <br />    The plausible explanation Sqt1p involved            tion Saccharomyces cerevisiae ribosomal protein L16 causes decrease in <br />                                                                           60S ribosomal subunits formation half-mer polyribosomes. Mol. Cell. <br /> assembling Qsr1p 60S subunit late assembly                Biol. 13:2835&#8211;2845. <br /> pathway, truncated Qsr1p competes full-length            13. Dick, F. A., S. Karamanou, B. L. Trumpower. QSR1, essential yeast <br /> Qsr1p assembly 60S subunit. The lack func-                gene genetic relationship subunit mitochondrial cyto- <br /> tional Qsr1p result half-mer phenotype,              chrome bc1 complex, codes 60S ribosomal subunit protein. J. Biol. <br />                                                                           Chem. 272:13372&#8211;13379. <br /> case temperature-sensitive qsr1 mutants (14). If truncated        14. Eisinger, D. P., F. A. Dick, B. L. Trumpower. 1997. Qsr1p, 60S <br /> forms Qsr1p sequester essential Sqt1p, defect               ribosomal subunit protein, required joining 40S 60S subunits. <br /> could suppressed extra expression QSR1                 Mol. Cell. Biol. 17:5136&#8211;5145. <br />  VOL. 17, 1997                                                            SQT1 SUPPRESSES DOMINANT-NEGATIVE QSR1 MUTATIONS                                         5155 <br />  <br />  <br /> 15. Eisinger, D. P., B. L. Trumpower. Long-inverse PCR generate re-              ribosomal subunits perturbed temperature-sensitive yeast mutants de- <br />     gional peptide libraries codon mutagenesis. BioTechniques, press.             fective ribosomal protein L16. Mol. Cell. Biol. 11:5681&#8211;5692. <br /> 16. Farmer, A. A., T. M. Loftus, A. A. Mills, K. Y. Sato, J. D. Neil, T. Tron, M.   40. Naranda, T., M. Kainuma, S. E. MacMillan, J. W. B. Hershey. 1997. The <br />     Yang, B. L. Trumpower, E. J. Stanbridge. 1994. Extreme evolutionary             39-kilodalton subunit eukaryotic translation initiation factor 3 essential <br />     conservation QM, novel c-Jun associated transcription factor. Hum.             complex&#8217;s integrity cell viability Saccharomyces cerevisiae. <br />     Mol. Genet. 3:723&#8211;728.                                                              Mol. Cell. Biol. 17:145&#8211;153. <br /> 17. Fields, S., O. Song. 1989. A novel genetic detect protein-        41. Naranda, T. S., E. MacMillan, J. W. B. Hershey. 1994. Purified yeast <br />     protein interactions. Nature (London) 340:245&#8211;246.                                  translational initiation factor eIF-3 RNA-binding protein complex that <br /> 18. Fioani, M., M. A. Cigan, C. J. Paddon, S. Harashima, A. G. Hinnebusch.          contains PRT1 protein. J. Biol. Chem. 269:32286&#8211;32292. <br />     1991. GCD2, translational repressor GCN4 gene, general           42. Naranda, T. S., E. MacMillan, T. F. Donahue, J. W. B. Hershey. 1996. <br />     function initiation protein synthesis Saccharomyces cerevisiae.        SUI1/p16 required activity eukaryotic translation initiation <br />     Mol. Cell. Biol. 11:3203&#8211;3216.                                             <br /> </body></html>