<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Mar. 1996, p. 960&#8211;967                                                                                 Vol. 16, No. 3 <br /> 0270-7306/96/$04.0010 <br /> Copyright q 1996, American Society Microbiology <br />  <br />  <br />  <br />       Identification Prp40, Novel Essential Yeast Splicing Factor <br />      Associated U1 Small Nuclear Ribonucleoprotein Particle <br />                                           HUNG-YING KAO           AND    PAUL G. SILICIANO* <br />        Department Biochemistry Institute Human Genetics, University Minnesota, Minneapolis, Minnesota 55455 <br />                     Received 14 September 1995/Returned modification 23 October 1995/Accepted 18 December 1995 <br />  <br />             We used suppressor genetics identify factors interact <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> U1 small <br />           nuclear RNA (snRNA). In way, isolated PRP40-1, suppressor restores growth 18&amp;C strain <br />           bearing cold-sensitive mutation U1 RNA. A gene disruption experiment shows PRP40 essential <br />           gene. To study role PRP40 splicing, created pool temperature-sensitive prp40 strains. Primer <br />           extension analysis intron-containing transcripts prp40 temperature-sensitive strains reveals splicing <br />           defect, indicating Prp40 general splicing factor. In vitro heat inactivation complementation assays <br />           demonstrate Prp40 plays direct role pre-mRNA splicing. In addition, U1 RNA coimmunoprecipitates <br />           Prp40, indicating Prp40 bound U1 small nuclear ribonucleoprotein particle vivo. <br />           Therefore, conclude PRP40 encodes novel, essential splicing component associates yeast <br />           U1 small nuclear ribonucleoprotein particle. <br />  <br />  <br />    Pre-mRNA splicing takes place multicomponent com-                    structure prominent stem-loop structures (37). The <br /> plex called spliceosome (5, 14, 15). Spliceosomes assemble               70K protein binds directly stem-loop 1 U1 RNA (17, 44, <br /> in highly ordered pathway mediated association              57), A protein binds directly stem-loop 2 (30). The <br /> of small nuclear ribonucleoprotein particles (snRNPs)                   C protein binds stem-loop 1 requires 70K <br /> one conserved sequences intron (7, 35,               protein <span id='am-58' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-59' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> associate U1 snRNP (39). <br /> 43). snRNPs stable, abundant, highly conserved parti-                   Despite limited sequence homology, S. cerevisiae U1 <br /> cles consisting small nuclear RNA molecules (snRNAs)                      RNA (568 nucleotides) fundamentally analogous human <br /> complexed sets proteins. snRNP-snRNP snRNP-                      U1 RNA. The 10 nucleotides precisely conserved <br /> intron interactions thought identify ends                  S. cerevisiae humans: sequence that <br /> intron bring catalytically active configuration              base pairs 59 splice site (49, 54). Also conserved 9 <br /> (35).                                                                        10 residues stem-loop 1, binding site 70K. Fur- <br />    Early splicing, U1 snRNP binds 59 splice site               thermore, yeast U1 RNA forms equivalent cruciform struc- <br /> (24, 33, 34, 46, 50). In <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-29'  ></span>Saccharomyces</span> cerevisiae mam-              ture places conserved 59 end, stem-loop 1, Sm- <br /> malian systems, binding occurs absence ATP                binding site configuration (23). <br /> any snRNP. Recognition 59 splice site U1                    Yeast U1 snRNPs contain core Sm proteins (53) and <br /> snRNP relies RNA-RNA base pairing U1                      homologs 70K (55) A (26) proteins. The yeast 70K <br /> RNA intron (49, 54, 58). <span id='am-45' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-46' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48'  ></span><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> recognition event                 homolog, encoded SNP1 gene, 30% overall amino <br /> requires U1 snRNP proteins (36). An intact 59 splice site                acid identity human 70K (55). We shown Snp1 <br /> and intact branch site necessary formation            binds stem-loop 1 U1 RNA (22). <br /> complex, committed splicing functional                     The MUD1 gene product sequence homology the <br /> precursor mature spliceosome (51). After U1 snRNP                     human U1A protein and, like Snp1, associates U1 <br /> binding, U2 snRNP binds branch site ATP-                    snRNP vivo. Unexpectedly, SNP1 (19) MUD1 <br /> dependent manner. Subsequently, U5 U4/U6 snRNPs                      (26) required yeast viability. Prp39, yeast U1 <br /> associate tri-snRNP form snRNP complete splicing                  snRNP-specific protein identified date, differs Snp1 <br /> complex, undergoes conformational rearrangements                       Mud1 required viability known <br /> and addition non-snRNP proteins mature                    mammalian homolog (28). Prp39 functions early spliceo- <br /> spliceosome.                                                                 assembly facilitate U1 snRNP&#8211;59 splice site inter- <br />    Metazoan U1 snRNPs contain single 164-nucleotide U1                     action (28). The yeast U1 snRNP contains additional proteins <br /> RNA complexed 11 proteins. A set common                     well: biochemical purification yeast U1 snRNPs reveals <br /> proteins, called core Sm proteins, binds near 39 end              presence U1-specific proteins (12). <br /> of U1 RNA. These proteins present U2, U5,                   To extend understanding functions U1 <br /> U4/U6 snRNPs (29) thought involved snRNP                    snRNP, employed suppressor genetics identify new <br /> stability (21) nuclear localization (32). In addition             U1 snRNP proteins. Starting cold-sensitive point muta- <br /> Sm proteins, metazoan U1 snRNPs contain U1-                        tion 59 end U1 RNA, U1-4U, isolated suppressor <br /> specific proteins called A, C, 70K.                                      mutations PRP40 gene. Here, PRP40, a <br />    Phylogenetic comparison U1 RNAs diverse <span id='am-55' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-56' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>                   novel essential gene encoding splicing factor interacts <br /> indicates U1 RNA folds conserved cruciform                   genetically physically yeast U1 snRNP. <br />  <br />  <br />    * Corresponding author. Mailing address: Department Biochem-                            MATERIALS AND METHODS <br /> istry, University Minnesota, 4-225 Millard Hall, 435 Delaware St.           Yeast strains. Strain 19SS [MATa DTRP1 DHIS3 ura3-52 lys2-801 ade2-101 <br /> SE, Minneapolis, MN 55455. Phone: (612) 625-4928. Fax: (612) 625-            snr19::LYS2 (YCp50-SNR19)] disruption chromosomal U1 RNA <br /> 2163. Electronic mail address: pauls@lenti.med.umn.edu.                      gene wild-type U1 gene carried URA3-CEN plasmid. Strain <br />  <br />                                                                        960 <br />  VOL. 16, 1996                                                                                      Prp40, A NOVEL YEAST SPLICING FACTOR                               961 <br />  <br />  <br /> 19SS-4U derived strain 19SS replacing YCp50-SNR19 pSE358-                   Genomic sequencing. Genomic DNA prepared parental strain 19SS-4U <br /> U1-4U plasmid shuffle. Strain 19SSW differs 19SS mating type             digested AccI, 1- 2-kb fragments gel purified electro- <br /> and deletion LEU2 gene. 19SSW-4U derived 19SSW plasmid                 eluted. Cycle sequencing performed according protocol Epicentre <br /> shuffle described above. The PRP40 gene disruption carried               Technologies regular dGTP, 103 PCR buffer, Taq polymerase <br /> homozygous diploid strain, TR1 (MATa/MATa ura3-52/ura3-52 trp1-289/trp1-289              (Perkin-Elmer) used. For reaction, 5 ng 32P-labeled primer 64C (5 <br /> his3-352/his3-352 lys2-801/lys2-801 ade2-101/ade2-101). A pSE360-PRP40 plas-             3 106 cpm) 3 mg genomic DNA used. Reaction mixtures were <br /> mid used transform PRP40 heterozygous strain TR1-40DD (the               denatured 928C 15 s, annealed 608C 15 s, extended 718C for <br /> as TR1 prp40::HIS3/PRP40). Stable transformants isolated                 1 min 30 cycles. <br /> sporulated, tetrads dissected create haploid plasmid-shuffling              Gap repair. Plasmid pSE360-PRP40 cut AflII SstI PRP40, <br /> strain PRP40-SS [MATa ura3-52 trp1-289 his3-352 lys2-801 ade2-101 prp40::HIS3            creating internal 457-bp deletion contains suppressor mutation <br /> (pSE360-PRP40)]. Strain PRP40-001 low-copy-number PRP40 plasmid                   C1421-T (a C-to-T mutation position 1421; diagrammed Fig. 2). The <br /> (pSE358-PRP40) strain PRP40-002 carrying high-copy-number overex-                  linearized plasmid transformed parental strain 19SS-4U. Ura1 colonies <br /> pression plasmid (pG1-PRP40) derived strain PRP40-SS plasmid                selected 2Ura plate, URA3 <span id='am-120' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-123' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span> recovered sequenc- <br /> shuffle. Temperature-sensitive strains prp40-10, prp40-12b, prp40-23            ing. <br /> also derived PRP40-001 transformation bank mutagenized PRP40                Gene disruption. Plasmid BSDBamHI-PRP40 digested StyI AgeI <br /> plasmids prior plasmid shuffling.                                                     treated Klenow fragment prior addition BamHI linkers and <br />    UV mutagenesis. Strain 19SS-4U grown mid-log phase, collected,             self ligation. A 1.8-kb BamHI fragment containing HIS3 gene inserted. <br /> resuspended sterile deionized water. A total 107 cells exposed             The resulting plasmid cut EcoRI, releasing PRP40 disruption allele <br /> 25,000 mJ yeast extract-peptone-dextrose plates, causing 65% killing. Imme-           fragment (diagrammed Fig. 2). This fragment used transformation into <br /> diately UV-irradiation, plates wrapped tinfoil prevent            homozygous strain, TR1. Stable His1 colonies selected sporulated, <br /> exposure visible light. The plates incubated 188C, viable            tetrads dissected. <br /> colonies isolated 5 days.                                                        Hydroxyamine mutagenesis. Twenty micrograms plasmid pSE358-PRP40 <br />    Deoxyoligonucleotides. Oligonucleotides synthesized Karin Musier-             incubated 1 M hydroxyamine&#8211;50 mM NaPO4 (pH 7.0)&#8211;2 mM EDTA&#8211; <br /> Forsyth University Minnesota, National Bioscience (Hopkins, Minn.)          100 mM NaCl 758C 30 min. The reaction stopped 10-min <br /> or Amitof (Boston, Mass.). The oligonucleotides used follows. The             incubation 08C followed use G-25 spin columns remove <br /> mutagenic oligonucleotides used U1 mutagenesis U13G4U, 59-GAT                   hydroxyamine. The mutagenized plasmid phenol extracted, ethanol precip- <br /> ATCTTAAGGTAAACATGAGGATTTTA-39; U13G5A, 59-GATATCTTAA                                     itated, used transform E. coli MC1066. The mutation rate determined <br /> GGTATGCATGAGGATTTTA-39; U14U5A, 59-GATATCTTAAGGTATAT                                     screening Apr colonies Trp phenotype. Approximately 1% the <br /> ATGAGGATTTTA-39; U159D, 59-CTCTGATATCTTAAATGAGGATTT                                  transformants failed grow E. coli 2Trp minimum plates. A total 15,000 <br /> TAA-39; used PRP40 mutagenesis 64(HA)3/NotI, 59-TCAGA                   E. coli transformants collected. This library transformed strain <br /> ATCAATAGCGGCCGCCGTCCAATTCCA-39. The PCR oligonucleotides                            PRP40-SS, 34,000 transformants screened temperature-sensitive <br /> 64AUG, 59-TCTGGATCCCATATGTCTATTTGGAAGGAA-39; 64C, 59-                                phenotype. <br /> TTCTGAGTGTCGATGAGGGT-39. The primer extension oligonucleotides                              Primer extension. Yeast total RNA isolated cell optical density 600 <br /> were U1, 59-CAAGGGAATGGAAACGTCAGC-39; U4, 59-AGGTATTCCAA                                 nm 0.5 1.0 previously described (10). Primer extension carried out <br /> AAATTCCCTAC-39; U5, 59-AAGCGCATAGTAAGAC-39; U6, 59-TGCTGAT                               according method Frank Guthrie (13) oligonucleotides <br /> CATCTCTGTATTG-39; RPL32, 59-GTGGACTTGTAACCTAAGGTG-39;                                described above. <br /> SAR1461, 59-TACCATGTTTGTTCCACAGA-39.                                                        In vitro splicing, heat inactivation, complementation. Splicing extracts <br />    Library construction. Genomic DNA prepared suppressor strain             prepared fractionated described Cheng Abelson (6). Splicing <br /> as previously described (45), instead CsCl gradient purification,     reactions carried according method Lin et al. (27). For heat <br /> DNA gel purified electroeluted 0.8% agarose gel. After phenol             inactivation, 4 ml extract preincubated 378C 35 min. Complemen- <br /> extraction ethanol precipitation, DNA partially digested Sau3A.         tation performed adding 1 ml 40W fraction [supernatant fraction of <br /> Inserts sizes 10 15 kb gel purified, electroeluted,            40% (NH4)2SO4 precipitation] heat-treated extracts prior 20-min splicing <br /> ligated BamHI-linearized, CEN-based URA3 plasmid, pSE360. The ligation              assay 238C. <br /> mixes transformed <span id='am-61' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-62'  ></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span>Escherichia coli</span> MC1066 generate library                   Preparation antisera (His)6-PRP40 fusion protein. The (His)6- <br /> representing genomes. To clone PRP40-1, 10 genomes screened.                    PRP40 fusion protein overexpressed purified according manu- <br />    Plasmid construction. A 3.2-kb SpeI fragment containing PRP40 sub-                facturer&#8217;s instructions (Qiagen). The column-purified protein puri- <br /> cloned Bluescript KS1 vector (Stratagene), generating plasmid BS-               fied previously described (19). Anti-(His)6-PRP40 antibodies prepared <br /> PRP40Spe. This plasmid digested PstI NsiI self ligated, yield        Berkeley Antibody Company. The rabbit serum affinity purified as <br /> plasmid BS-PRP40, contains 440 bp upstream 194 bp downstream                previously described (19), (His)6-PRP40-coupled CNBr-activated <br /> the PRP40 coding sequence addition PRP40 coding region. Derivatives            Sepharose column used. <br /> of plasmids pSE358, pSE360 (11), pRS306 (52) lack SstI site                Coimmunoprecipitation U1 snRNA. Affinity-purified anti-(His)6-PRP40 an- <br /> constructed digestion plasmids SstI, Klenow treatment, self         tiserum coupled protein A-agarose beads (GIBCO/BRL) IPP150 buffer <br /> ligation. A 2.3-kb EcoRI fragment released BS-PRP40 subcloned               (10 mM Tris-HCl [pH 8.0], 150 mM NaCl, 0.1% Nonidet P-40, 0.1% sodium <br /> into plasmid pSE358DSstI, pSE360DSstI, pSE306DSstI, generating plasmids              azide) washed times use. Twenty-five microliters antibody- <br /> pSE358-PRP40, pSE360-PRP40, pSE306-PRP40, respectively. BSDBamHI                     coupled beads incubated 5 ml splicing extract 200 ml IPP150 <br /> was constructed Bluescript KS1 (Stratagene) BamHI digestion, Klenow              buffer Nutator 258C 40 min. The supernatant saved, pellet <br /> treatment, self ligation. A PCR fragment, generated primers 64AUG                 washed times 600 ml IPP150 buffer. RNA recovered the <br /> (which creates BamHI site 3 nucleotides upstream start <span id='am-50' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-51' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53'  ></span><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span>) 64C,          immune supernatants immune pellets phenol extraction ethanol <br /> was digested BamHI AflII swapped BamHI-AflII-linearized                precipitation. The presence snRNAs immune supernatants immune <br /> pSE358-PRP40, generating plasmid pSE358-PRP40AUG. A 1.75-kb BamHI-SalI                   pellets assayed primer extension using cocktail containing oligonu- <br /> fragment containing PRP40 coding sequence pSE358-PRP40AUG                   cleotides specific U1, U2, U4, U5, U6 RNAs. <br /> inserted BamHI-SalI-linearized pG1 (47), high-expression, multicopy plas- <br /> mid, create plasmid pG1-PRP40. This 1.75-kb BamHI-SalI fragment also <br /> inserted BamHI-SalI-cut plasmid pQ30 (Qiagen) create (His)6- <br />                                                                                                                            RESULTS <br /> Prp40 fusion construct. <br />    Primer 64(HA)3/NotI used introduce NotI site Transformer mu-                 Isolation suppressors U1 RNA point mutations. Early <br /> tagenesis (Clontech) penultimate codon PRP40 plasmid pSE358-                spliceosome assembly, 59 end U1 RNA base pairs with <br /> PRP40. A 111-bp NotI fragment encoding copies hemagglutinin                 59 splice site consensus sequence. Point mutations 59 <br /> (HA) epitope, kind gift Jim Umen Christine Guthrie, inserted to <br /> generate plasmid pSE358-PRP40(HA)3. <br />                                                                                          end U1 RNA cause variety growth phenotypes <br />    Genetic manipulations. All genetic experiments including medium prepara-              ranging mild severe (Fig. 1). However, severity of <br /> tion, diploid selection, sporulation, dissection, <span id='am-90' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-91' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93'  ></span><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cis</span> trans tests, plasmid   mutant phenotype does correlate num- <br /> shuffle performed previously described (16). Yeast transformation            ber hydrogen bonds lost 59 end U1 RNA and <br /> carried method Schiestl Gietz (48). <br />    Integration pop-out. Plasmid pRS306-PRP40 linearized SstI and <br />                                                                                          59 splice site consensus sequence, suggesting other <br /> transformed parental strain 19SS-4U (diagrammed Fig. 2). Stable inte-            factors interact 59 end U1 RNA. We used <br /> grants selected 2Ura minimal medium (minimal medium lacking uracil)              suppressor genetics search factors. We started with <br /> and confirmed Southern blot analyses. The pop-out experiment per-                 yeast strain carrying C-to-U transition mutation the <br /> formed 5-fluoro-orotic acid (5-<span id='am-1' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5'  ></span>FOA</span>) plates confirmed Ura2 pheno- <br /> type Southern blot analyses. The pop-out integration strain used all <br />                                                                                          fourth nucleotide U1 RNA, U1-4U. This mutation causes <br /> phenotypic analyses prp40-1, certain genome                cold sensitivity, preventing growth 188C, does greatly <br /> contains UV-induced mutations.                                                  disturb base pairing 59 splice site. Yeast strain 19SS- <br />  962       KAO AND SILICIANO                                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />                                                                                                 TABLE 1. Allele specificity PRP40-1 suppressora <br />                                                                                                                              Growth indicated temp (8C)b: <br />                                                                                       U1 allele      PRP40 allele <br />                                                                                                                        18        25           30        33        37 <br />  <br />                                                                                        WT   c <br />                                                                                                       WT              11        11        111        111         11 <br />                                                                                                       PRP40-1         11        11        111        111         11 <br />  <br />                                                                                        4U             WT              2         1/2       11         11          1 <br />                                                                                                       PRP40-1         1         11        111        111         1 <br />  <br />                                                                                        8U             WT              2         2         1          1           1/2 <br />                                                                                                       PRP40-1         1         11        11         11          11 <br />  <br />                                                                                        4U,5A          WT              2         2         2          2           2 <br />                                                                                                       PRP40-1         1/2       1         1          11          1/2 <br />                                                                                         a <br />                                                                                           Cells grown rich media temperature indicated. Also tested <br />                                                                                       U1 alleles 4A; 4G; 5A; 5C; 8A; 8G; 3G,4U; 3G,4U,5A; 59 end D. The <br />                                                                                       growth phenotype alleles suppressed PRP40-1. <br />                                                                                         b <br />                                                                                           1/2, doubling time longer 9 h; 1, doubling time longer 4.5 h; 11, <br />                                                                                       doubling time longer 2.1 h; 111, doubling time 2.1 h; 2, inviable. <br />                                                                                         c <br />                                                                                           WT, wild type. <br />    FIG. 1. The growth phenotype U1 mutant does correlate with <br /> the number hydrogen bonds lost 59 end U1 RNA 59 <br /> splice site. Shown left base pairing yeast 59 splice site <br /> consensus sequence 59 end U1 RNA (boldface letters). The growth <br /> phenotypes selected U1 mutants number hydrogen bonds lost                  Identification suppressor genes. To clone suppres- <br /> between U1 RNA 59 splice site indicated. CS, cold sensitive.         sor gene, constructed library suppressor <br />                                                                                       strains. Genomic DNA isolated, partially digested with <br />                                                                                       Sau3A, ligated CEN-based URA3 vector. This library <br /> 4U, gene encoding yeast U1 RNA                              transformed parental strain 19SS-4U incu- <br /> disrupted (snr19::LYS2) complemented U1-4U mu-                             bated 188C screen clone(s) rescues lethality <br /> tant CEN-based TRP1 plasmid, mutagenized UV-                              188C. Ten suppressor clones recovered retested for <br /> irradiation 35% survival. After incubation 188C 5 days,                     rescue cold sensitivity. On basis restriction map- <br /> 30 viable colonies collected. We focused poten-                         ping Southern analyses, clones grouped into <br /> tial candidates grew 188C.                                               classes. Interestingly, classes able suppress <br />    To rule reversion U1-4U mutation, performed                          U1-4U mutant 188C U1-8U mutant 258C. The <br /> cis trans tests. In cis test, TRP1 plasmids isolated                     suppressor genes localized suppressor clones <br /> from suppressor strains retransformed plasmid-                           testing subclones rescue 188C. <br /> shuffling strain 19SS. This strain carries snr19::LYS2 gene                          Sequence analysis revealed suppressor <br /> disruption wild-type U1 RNA gene URA3 vector. In                           genes previously described. One suppressor <br /> the plasmid shuffle assay, cells lost URA3 plasmid                      genes, SAR1, single-copy essential gene encodes a <br /> can selected growth media containing drug 5-FOA                          polypeptide involved protein transport endoplas- <br /> (3). After selection 5-FOA plates, retransformed strains                       mic reticulum Golgi apparatus (2, 38). The SAR1 tran- <br /> still exhibit U1-4U cold sensitivity. Thus, loss cold                      script contains single small (139-nucleotide) intron an <br /> sensitivity reversion U1-4U. In addition,                       unusually short distance branch site 39 splice site <br /> presence U1-4U mutation recovered plasmids                          (18 nucleotides). <br /> confirmed sequence analysis.                                                          We believe SAR1 directly involved pre- <br />    We used trans test verify presence un-                         mRNA splicing. First, SAR1 suppressor clone contains no <br /> linked suppressor mutation. In test, fresh U1-4U plasmids                        sequence differences published sequence. Further- <br /> were transformed candidate suppressor strains. These re-                         more, wild-type SAR1 clone able suppress U1-4U <br /> transformed suppressor strains able grow 188C,                         mutant 188C. Most likely, SAR1 clone rescues U1-4U <br /> we concluded strains contained extragenic                        cold-sensitive phenotype multicopy suppression. That is, by <br /> suppressors U1-4U. When backcrossed parental                                increasing SAR1 copy number, copies, <br /> strain, suppressors showed dominant, single-gene seg-                       sufficient SAR1 gene product allow improved <br /> regation (<span id='am-115' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-116' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-119'  ></span>data</span> shown). When individual suppressor candi-                          growth. These findings indicate SAR1 gene product is <br /> dates crossed sporulated,                             rate limiting growth U1-4U cells 188C. <br /> spores suppressed U1-4U cold-sensitive phenotype, indi-                              The suppressor originally reported Yeast <br /> cating suppressors allelic other.                                Genome Sequencing Project YKL165 (41). We re- <br />    One suppressor strain analyzed determine                        named gene PRP40 role pre-mRNA processing <br /> effects mutations U1 RNA (Table 1). A total 13                         (see below). PRP40 encodes polypeptide 583 amino acids <br /> U1 mutants tested. While phenotypes U1                               predicted molecular mass 69 kDa. It compel- <br /> mutants similar wild-type suppressor strains                         ling sequence homology known protein contains no <br /> (Table 1), suppressor did rescue cold sensitivity                          RNA recognition motif (RRM). The original report (41) noted <br /> U1-8U 258C (Table 1). Intriguingly, suppressor                        weak homology S. cerevisiae Myo2 protein (20), although <br /> partially suppress U1-4U,5A double mutant, lethal                        believe homology significant (see Discussion). <br /> in wild-type background. Thus, screen isolated                           The PRP40 suppressor clone differs published se- <br /> a suppressor capable relieving growth phenotypes                            quence single nucleotide (1421; TCC3TTC), resulting in <br /> several U1 RNA point mutations.                                                       Ser-to-Phe change Prp40 protein (Fig. 2A). To show <br />  VOL. 16, 1996                                                                                Prp40, A NOVEL YEAST SPLICING FACTOR                963 <br />  <br />  <br />                                                                                      PRP40 sequence. We generated gap repair construct on <br />                                                                                      URA3 plasmid AflII-SstI fragment encompassing <br />                                                                                      C-1421 deleted (Fig. 2B). This plasmid transformed <br />                                                                                      wild-type parental strain 19SS-4U. In order the <br />                                                                                      plasmid replicate, gap repaired using the <br />                                                                                      chromosomal copy PRP40 template. Ura1 transfor- <br />                                                                                      mants isolated, URA3 plasmids recovered for <br />                                                                                      sequencing. All recovered PRP40 sequences iden- <br />                                                                                      tical published YKL165 sequence. These results unam- <br />                                                                                      biguously demonstrate nucleotide change seen the <br />                                                                                      PRP40 suppressor allele true mutation opposed a <br />                                                                                      sequence polymorphism. <br />                                                                                         To determine cloned PRP40 suppressor original <br />                                                                                      suppressor gene, integrating plasmid created (Fig. 2C). <br />                                                                                      A URA3 plasmid containing cloned PRP40 suppressor al- <br />                                                                                      lele linearized SstI PRP40 transformed <br />                                                                                      parental strain 19SS-4U. This integration resulted in <br />                                                                                      duplication PRP40 wild-type allele one <br />                                                                                      suppressor allele separated URA3 vector (Fig. 2C). All <br />                                                                                      integrants grow 188C, consistent domi- <br />                                                                                      nant phenotype suppressor strain. Pop-out experiments <br />                                                                                      carried 5-FOA plates select URA3 <br />                                                                                      marker. In experiments, recombination occurs between <br />                                                                                      PRP40 alleles, leaving single PRP40 allele the <br />                                                                                      chromosome. Depending recombination event <br />                                                                                      occurs PRP40, strains carry wild-type PRP40 <br />                                                                                      allele suppressor PRP40 allele generated. Precise <br />                                                                                      integration excision (pop-out) confirmed South- <br />                                                                                      ern analyses (data shown). As expected, recovered both <br />                                                                                      Sup1 Sup2 pop-out strains. The pop-out strain the <br />                                                                                      PRP40 suppressor phenotype backcrossed original <br />                                                                                      suppressor strain. This diploid strain sporulated, tet- <br />                                                                                      rads dissected. Seven four-spored tetrads recovered; <br />                                                                                      spores able grow 188C. These data <br />                                                                                      prove PRP40 (C1421-T) bona fide suppressor. We <br />                                                                                      allele PRP40-1. <br />                                                                                         PRP40 essential gene. We identified PRP40 a <br />                                                                                      gene genetically interacts yeast U1 RNA. To deter- <br />                                                                                      PRP40 required cell growth, prp40::HIS3 dis- <br />                                                                                      ruption allele constructed. This allele deletes 83% the <br />                                                                                      coding region replaces HIS3 gene (Fig. 2D). <br />                                                                                      This disruption allele used disrupt allele PRP40 <br />    FIG. 2. Constructs genetic analyses. (A) Restriction map PRP40. The        homozygous diploid strain TR1. The resulting PRP40/ <br /> translation start codon termination codon indicated. The asterisk de- <br /> notes suppressor mutation C1421-T. Nucleotide positions assigned ac- <br />                                                                                      prp40::HIS3 diploids isolated 2His plates, confirmed <br /> cording method Pascolo et al. (41). (B) Gap repair construct. Plasmid      Southern blot analyses, sporulated, tetrads were <br /> pSE360-PRP40 built described Materials Methods. An internal            dissected. In cases, viable spores recovered. <br /> deletion PRP40 created AflII-SstI digestion. This deletion gener-      Viability cosegregated His2 phenotype, indi- <br /> ated gap includes position suppressor mutation, 1421. Note that <br /> the scale different panel A. (C) Integration pop-out. Plasmid <br />                                                                                      cating prp40::HIS3 allele lethal. In complementary <br /> pRS306-PRP40 described Materials Methods. An SstI-linearized plas-         experiment, URA3 plasmid carrying wild-type PRP40 was <br /> mid used transformation. The arrow indicates direction transcrip-     transformed heterozygous diploid strain prior to <br /> tion, asterisk denotes suppressor mutation. The stippled hatched     sporulation. The resulting strain sporulated, tetrads <br /> boxes represent suppressor wild-type sequences, respectively. The thick <br /> and lines denote chromosomal plasmid sequences, respectively. In the <br />                                                                                      dissected. Seven tetrads spores retained <br /> example shown here, wild-type allele &#8216;&#8216;popped-out,&#8217;&#8217; leaving suppressor     URA3 plasmid isolated. Selection 5-FOA plates <br /> allele chromosome. (D) prp40 disruption allele. The prp40 disruption allele   revealed spores individual tetrad able <br /> was generated described Materials Methods.                                 grow absence wild-type PRP40. Again, viable <br />                                                                                      cells His2. Therefore, conclude PRP40 is <br />                                                                                      essential gene. <br /> that nucleotide change represents UV-induced mutation,                           The PRP40-1 suppressor affects SAR1 mRNA accumulation <br /> as opposed sequence polymorphism, genomic sequencing                              U1-4U mutant 18&amp;C. To test effect PRP40-1 on <br /> and gap repair experiments performed. Genomic DNA                               splicing U1-4U cells, examined processing the <br /> was isolated parental strain 19SS-4U digested                           SAR1 transcript U1-4U cells primer extension. One com- <br /> with AccI. Nucleotide 1421 contained 1.3-kb AccI frag-                       mon feature poorly spliced introns short distance <br /> ment. We purified AccI fragments sizes 1 2                          branch site 39 splice site, MATa1 and <br /> kb gel electrophoresis used pooled fragments                        SNR17 (40). We hypothesized SAR1 transcript con- <br /> PCR cycle sequencing. We sequence PRP40                            tains poorly spliced intron production SAR1 <br /> in 19SS-4U parental strain identical published                         mature transcript rate limiting growth U1-4U <br />  964      KAO AND SILICIANO                                                                                                                     MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 3. The suppressor allele PRP40 affects SAR1 transcript accumulation        FIG. 4. In vivo splicing phenotype prp40 mutant alleles. prp40 mutants <br /> in U1-4U mutant 188C. Parental suppressor strains containing U1-4U      grown 258C optical density 600 nm 0.4 shifted 378C for <br /> as source U1 snRNA grown 308C optical density 600     4 h. Total yeast RNA isolated, primer extension performed an <br /> nm 0.5 shifted 188C 6 h. Total yeast RNA isolated analyzed   oligonucleotide specific RPL32 exon 2. RPL32 pre-mRNA mRNA are <br /> by primer extension. Unspliced spliced SAR1 transcripts indicated       indicated arrows. Also included primer extension product U5 snRNA <br /> arrows. The primer extension U4 snRNA used internal control.          internal control. Lanes 1 2, prp40-12b; lanes 3 4, prp40-23; lanes 5 <br /> Lane 1, strain U1-4U wild-type (WT) PRP40 188C; lane 2, strain U1-4U       6, wild type (WT). Lanes 1, 3, 5, cells grown 258C; lanes 2, 4, 6, <br /> with wild-type PRP40 308C; lane 3, strain U1-4U PRP40-1 suppressor     cells shifted 378C. <br /> at 188C; lane 4, strain U1-4U PRP40-1 suppressor 308C. <br />  <br />  <br />  <br />                                                                                    mRNA degraded splicing inhibited (18, 42). On <br /> mutant 188C (1); hypothesized alleviating                   basis arguments, predict Prp40 increases <br /> SAR1 splicing deficiency, PRP40-1 allows production                        level SAR1 mature mRNA relieving U1-4U <br /> mature SAR1 transcripts rescues U1-4U cold                         splicing defect. <br /> sensitivity (2). To test hypotheses, analyzed SAR1 splic-                      PRP40 encodes general splicing factor. To determine if <br /> ing U1-4U strains PRP40-1 suppressor.                      Prp40 general splicing factor, created prp40 mutant <br /> Parental suppressor strains grown 308C early log                    library hydroxyamine mutagenesis. This library trans- <br /> phase shifted 308C (permissive temperature) 188C                        formed plasmid-shuffling strain (prp40::HIS3 pSE360- <br /> (nonpermissive temperature) 6 h. Total RNA isolated,                       PRP40). A total 34,000 yeast transformants screened <br /> and primer extension performed. We little                      plasmids bearing temperature-sensitive prp40 mutation. <br /> SAR1 mature mRNA detected U1-4U mutant                              After 5-FOA selection, 17 temperature-sensitive clones were <br /> grown 188C (Fig. 3, lane 1). We did observe accu-                        isolated retested. Among them, 378C sensitive, <br /> mulation SAR1 pre-mRNA, expected U1-4U                           308C sensitive, rest 348C sensitive. We <br /> inhibited SAR1 splicing. However, PRP40-1 suppressor                        tested vivo splicing phenotype mutant strains by <br /> strain, SAR1 mature mRNA accumulated normal levels                           primer extension analysis RPL32 transcript, has <br /> both 18 308C (Fig. 3, lanes 3 4). Thus, PRP40-1                        intron architecture different SAR1 tran- <br /> suppressor increases steady-state level SAR1 mature                         script normally efficiently spliced (8, 9). Temperature- <br /> mRNA U1-4U cells 188C.                                                       sensitive strains grown 258C early log phase and <br />    This observation consistent U1-4U mutant affect-                    shifted 378C 4 h. RNA isolated analyzed <br /> ing stability splicing SAR1 pre-mRNA. We                         primer extension. <br /> believe low steady-state level SAR1 mature mRNA                           In wild-type cells, RPL32 efficiently spliced without <br /> in U1-4U cells results inefficient splicing SAR1                       temperature shift (Fig. 4, lanes 5 6). The prp40 temper- <br /> pre-mRNA hypothesize SAR1 pre-mRNA does                               ature-sensitive mutants prp40-12b (Fig. 4, lanes 1 2) and <br /> not accumulate degraded spliced. This                   prp40-23 (Fig. 4, lanes 3 4) display phenotype expected <br /> hypothesis supported observations unspliced pre-                        strain defective pre-mRNA splicing. That is, after <br />  VOL. 16, 1996                                                                                 Prp40, A NOVEL YEAST SPLICING FACTOR                  965 <br />  <br />  <br />                                                                                       plementation experiment. First, constructed PRP40 <br />                                                                                       (HA)3 allele, contains copies HA epitope at <br />                                                                                       PRP40 carboxy terminus. This epitope recognized the <br />                                                                                       12CA5 monoclonal antibody. The PRP40(HA)3 allele retains <br />                                                                                       Prp40 function, allele fully complements prp40 gene <br />                                                                                       disruption (data shown). By Western blot (immunoblot) <br />                                                                                       analysis, Prp40(HA)3 fractionates 40P3 <br />                                                                                       fraction yeast splicing extract [pellet fraction 40% <br />                                                                                       (NH4)2SO4 precipitation; U1 RNA fractionates the <br />                                                                                       40P3 fraction]. No Prp40(HA)3 detected 40W fraction <br />                                                                                       [40% (NH4)2SO4 supernatant]. However, overexpression of <br />                                                                                       Prp40(HA)3 high-copy-number plasmid results partial <br />                                                                                       fractionation Prp40(HA)3 40W fraction. <br />                                                                                          The 40W fraction prepared strains harboring ei- <br />                                                                                       ther low-copy-number PRP40 plasmid high-copy-num- <br />                                                                                       ber PRP40 plasmid overexpressed Prp40. The 40W frac- <br />                                                                                       tion strain high-copy-number PRP40 plasmid <br />                                                                                       able restore splicing activity heat-inactivated <br />                                                                                       prp40-10 extract (Fig. 5, lane 6). However, 40W fraction <br />                                                                                       low-copy-number PRP40 does restore activity <br />                                                                                       (Fig. 5, lane 5). These results indicate wild-type Prp40 <br />                                                                                       (HA)3 complement heat-inactivated temperature-sen- <br />                                                                                       sitive Prp40. Moreover, 40W fraction able support <br />                                                                                       splicing activity (Fig. 5, lanes 7 8). In addition, the <br />                                                                                       40P3 fractions low-copy-number high-copy- <br />                                                                                       number extracts complement heat-inactivated extract, <br />                                                                                       indicating low-copy-number extract active (data <br />                                                                                       shown). These findings demonstrate PRP40 gene <br />                                                                                       product directly involved pre-mRNA splicing. <br />                                                                                          Prp40 associates U1 snRNA vivo. The isolation of <br />                                                                                       PRP40-1 allele suppresses mutations U1 RNA dem- <br />                                                                                       onstrates PRP40 interacts genetically U1 RNA. We <br />    FIG. 5. In vitro splicing, heat inactivation, complementation tem- <br /> perature-sensitive allele PRP40. Splicing extracts preincubated 378C       asked Prp40 protein physically associated the <br /> for 35 min; followed complementation splicing. Lane 1, wild-type      U1 snRNP. To answer question, constructed His- <br /> extract heat treatment; lane 2, wild-type extract heat treatment; lane   tagged allele, (His6)-Prp40, purified tagged protein. <br /> 3, prp40-10 extract heat treatment; lane 4, prp40-10 extract heat        This protein used raise rabbit polyclonal antibodies <br /> treatment; lane 5, prp40-10 extract heat treatment complemented with <br /> 40W fraction low-copy-number PRP40 extract; lane 6, prp40-10 extract        specific Prp40. We used anti-Prp40 serum preim- <br /> heat treatment complemented 40W fraction high-copy-number               mune control serum immunoprecipitate Prp40 splicing <br /> PRP40 extract; lane 7, 40W fraction low-copy-number PRP40 extract; lane          extract. We analyzed immune supernatants immune <br /> 8, 40W fraction high-copy-number PRP40 extract.                                  pellets presence U1, U2, U4, U5, U6 RNAs <br />                                                                                       primer extension (Fig. 6A B). The preimmune serum <br />                                                                                       does immunoprecipitate snRNAs (Fig. 6A, lane 3). <br /> temperature shift, level mature mRNA reduced,                          However, anti-Prp40 serum specifically immunoprecipi- <br /> a concomitant increase level pre-mRNA. Although                             tates U1 RNA (Fig. 6A, lane 5), demonstrating Prp40 is <br /> the RPL32 59 splice site differs yeast consensus (/GU                        associated U1 RNA. <br /> CAGU [the slash indicates exon-intron boundary,                                  In experiment results shown Fig. 6A, 70% <br /> nucleotides differ yeast consensus sequence                         U1 RNA recovered immune pellet. In other <br /> in boldface type]), primer extension experiments                             experiments (Fig. 6B), anti-Prp40 antiserum quantitatively <br /> ACT1 pre-mRNA yielded similar results (data shown).                          precipitated U1 RNA (lanes 5 6). We conclude that <br /> Accordingly, conclude Prp40 general splicing fac-                        U1 RNA associated Prp40 vivo. In addition, <br /> <span id='am-99' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-100'  ></span><span id='am-101' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>tor</span>.                                                                                  Prp40 coimmunoprecipitates U1 RNA just well <br />    Prp40 required splicing vitro. To prove Prp40                       300 mM NaCl (data shown), indicating Prp40 is <br /> has direct role pre-mRNA splicing, wild-type temper-                         stably associated U1 RNA. On basis data, we <br /> ature-sensitive strains grown permissive tempera-                         propose Prp40 component U1 snRNP. <br /> ture (258C) splicing extracts prepared. To heat inac- <br /> tivate prp40 mutant protein, extracts preincubated                                                 DISCUSSION <br /> either permissive temperature (238C) nonpermissive <br /> temperature (378C) 35 min. Without preincubation                              Our understanding snRNP function rests primarily the <br /> nonpermissive temperature, wild-type prp40 mu-                            discovery base pairing interactions snRNAs each <br /> tant extracts able splice vitro-transcribed ACT1                         conserved sequences intron. This <br /> pre-mRNA (Fig. 5, lanes 1 3). Upon heat treatment,                            network base pairing interactions thought culminate in <br /> splicing activity extract prp40 mutant strain                      creation ribozyme catalyzes splicing (31). In con- <br /> greatly reduced (Fig. 5, lane 4). In contrast, wild-type                          trast detailed knowledge snRNAs, roles of <br /> extract remains active (Fig. 5, lane 2).                                              snRNP proteins splicing remain largely unknown. <br />    To determine inhibition splicing activity                        Here identify Prp40, novel component yeast <br /> heat inactivation mutant protein, performed com-                          splicing machinery. Prp40 essential yeast viability for <br />  966      KAO AND SILICIANO                                                                                                                  MOL. CELL. BIOL. <br />  <br />  <br />                                                                                    (41), believe Myo2 homology significant. <br />                                                                                    BLASTP searches revealed sequences homology <br />                                                                                    scores greater Myo2 level <br />                                                                                    generally considered significant. PRP40 does stronger <br />                                                                                    match ZK1098.1, <span id='am-33' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-39'  ></span>Caenorhabditis elegans</span> open reading <br />                                                                                    frame unknown function, does appear homologous <br />                                                                                    known metazoan yeast splicing factor. <br />                                                                                       Further domain searches revealed copies loosely <br />                                                                                    conserved motif called WWP domain amino termi- <br />                                                                                    nus Prp40. The WWP domain identified protein <br />                                                                                    motif encompassing 38 amino acids various structural, <br />                                                                                    regulatory, signaling molecules S. cerevisiae, <span id='am-84' about='obo:NCBITaxon_333870' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_195229 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41087 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_71193 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasRelatedSynonym" content="Nematodes Berthold, 1827" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Nematodes" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-89'  ></span>nematodes</span>, <br />                                                                                    mammals (reviewed references 1, 4, 56). These <br />                                                                                    proteins, known common functions, include <br />                                                                                    human dystrophin, YAP65 (Yes-associated protein), yeast <br />                                                                                    Rsp5, ZK1098.1. The WWP domain contains highly <br />                                                                                    conserved aromatic residues, including tryptophans. In <br />                                                                                    addition, consensus amino acids conserved <br />                                                                                    positions (1, 4). This structural feature proposed in- <br />                                                                                    volved protein-protein interactions. In Prp40, suppressor <br />                                                                                    mutation Ser240-Phe located downstream WWP do- <br />                                                                                    main. <br />                                                                                       Is Prp40 U1 snRNP protein? Fabrizio et al. (12) have <br />                                                                                    purified yeast U1 snRNPs analyzed protein compo- <br />                                                                                    sition. Intriguingly, detected protein 69 kDa puri- <br />                                                                                    fied U1 snRNPs; size matches calculated size Prp40 <br />                                                                                    (69 kDa) closely. <br />                                                                                       What role Prp40 splicing? Both vivo in <br />                                                                                    vitro, prp40 mutants block step splicing, expected <br />                                                                                    factor functions U1 snRNP. A specific <br />                                                                                    hypothesis Prp40 function based U1 RNA <br />                                                                                    mutations suppressed PRP40-1 mutation. Both <br />                                                                                    position 4 position 8 U1 RNA known base pair <br />                                                                                    cognate nucleotides 59 splice site. At each <br />                                                                                    position, C-to-U transition viable mutation; all <br />                                                                                    mutations positions 4 8 lethal (54). Intrigu- <br />                                                                                    ingly, U1-4U U1-8U cause cold-sensitive as <br />                                                                                    temperature-sensitive growth. The temperature sensitivity of <br />                                                                                    U1-4U U1-8U result weakening base <br />                                                                                    pairing 59 splice site. However, cold sensitivity of <br />                                                                                    mutants anticipated argues favor these <br />                                                                                    nucleotides U1 having additional function inter- <br />    FIG. 6. (A) Prp40 associates specifically yeast U1 snRNA vivo. Splic-   action 59 splice site. On basis allele speci- <br /> ing extract prepared wild-type cells immunoprecipitated either <br /> preimmune serum serum specific (His)6-Prp40. RNAs extracted       ficity PRP40-1 suppressor, propose interaction <br /> the unprecipitated total (T), immune supernatant (S), immune pellet (P)        Prp40 essential function nucleotides of <br /> fractions analyzed primer extension using cocktail containing oligo-   U1 RNA. Finally, interesting vitro splicing extracts <br /> nucleotides specific U1, U2, U4, U5, U6 RNAs. The primer extension     prepared U1-4U mutants partially bypass re- <br /> products RNAs indicated arrows. Also marked (p) artifac- <br /> tual primer extension product caused U6-specific oligonucleotide. Lane 1,   quirement ATP U2 addition (25). Our demonstration <br /> total RNA immunoprecipitation; lane 2, preimmune supernatant; lane 3,      Prp40 interacts genetically U1-4U suggest a <br /> preimmune pellet; lane 4, immune supernatant; lane 5, immune pellet; lane 6,       role Prp40 U2 snRNP association. Our prp40 mutant <br /> molecular weight markers (M). (B) All U1 RNA associated Prp40.         extracts allow test exciting possibility. <br /> Splicing extract immunoprecipitated described analyzed by <br /> primer extension oligonucleotide specific U1 RNA. Note a <br /> different U1-specific oligonucleotide used panel A. Lane 1, molecular                              ACKNOWLEDGMENTS <br /> weight markers; lane 2, total RNA immunoprecipitation; lane 3, preim- <br /> mune pellet; lane 4, preimmune supernatant; lane 5, immune pellet; lane 6,           We thank Brian Rymond, Jim Umen, Christine Guthrie for <br /> immune supernatant. M, T, P, S defined panel A.                     reagents Dave LaPorte helpful discussions. We grateful to <br />                                                                                    Stew Scherer help PRP40 homology searches to <br />                                                                                    <span id='am-104' about='obo:NCBITaxon_344662' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_344711 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_42315 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-105' property="rdfs:label" content="Patricia" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasRelatedSynonym" content="Patricia Fox 1940" datatype="xsd:string"></span><span id='am-109'  ></span>Patricia</span> Hilleren Mark Murphy comments manuscript. <br />                                                                                      This work supported grant NIH (GM-44628) to <br /> the step pre-mRNA splicing vivo vitro. We                       P.G.S. <br /> show Prp40, originally identified factor sup- <br /> presses mutations 59 end U1 RNA, associates                                                    REFERENCES <br /> U1 snRNA vivo.                                                                   1. Andre, B., J. Springael. 1994. WWP, new amino acid motif present in <br />   Prp40 weak homology S. cerevisiae Myo2 protein;                           single multiple copies various proteins including dystrophin the <br />                                                                                        SH3-binding Yes-associated protein YAP65. Biochem. Biophys. Res. Com- <br /> this homology consists 31 identical amino acids 93-                            mun. 205:1201&#8211;1205. <br /> amino-acid stretch does include ATP binding do-                          2. Barlowe, C., C. d&#8217;Enfert, R. Schekman. 1993. Purification charac- <br /> main. Because extensive gaps needed make match                          terization Sar1p, small GTP-binding protein required transport <br />  VOL. 16, 1996                                                                                      Prp40, A NOVEL YEAST SPLICING FACTOR                                   967 <br />  <br />  <br />       vesicle formation endoplasmic reticulum. J. Biol. Chem. 268:873&#8211;              Birnstiel (ed.), Structure function major minor small nuclear <br />       879.                                                                                   ribonucleoprotein particles. Springer-Verlag, New York. <br />  3.   Boeke, J. D., J. Truehart, G. Natsoulis, G. R. Fink. 1987. 5-Fluoro-orotic   33.   Michaud, S., R. Reed. 1991. An ATP-independent complex commits <br />       acid selective agent yeast molecular genetics. Methods Enzymol.                pre-mRNA mammalian spliceosome assembly pathway. Genes Dev. <br />       154:164&#8211;175.                                                                           5:2534&#8211;2546. <br />  4.   Bork, P., M. Sudol. 1994. The WW domain: signaling site dystro-         34.   Michaud, S., R. Reed. 1993. A functional association 59 and <br />       phin? Trends Biochem. Sci. 19:531&#8211;533.                                                 39 splice sites established earliest prespliceosomal complex (E) in <br />  5.   Brody, E., J. Abelson. 1985. The &#8216;&#8216;spliceosome&#8217;&#8217;: yeast pre-messenger              mammals. Genes Dev. 7:1008&#8211;1020. <br />       RNA associates 40S complex splicing-dependent reaction. Science      35.   Moore, M., C. Query, P. Sharp. 1993. Splicing precursors messen- <br />       228:963&#8211;967.                                                                           ger RNAs spliceosome, p. 303&#8211;357. In R. Gesteland J. Atkins <br />  6.   Cheng, S.-C., J. Abelson. 1986. Fractionation characterization            (ed.), The RNA world. Cold Spring Harbor Laboratory Press, Cold Spring <br />       yeast mRNA splicing extract. Proc. Natl. Acad. Sci. USA 83:2387&#8211;2391.                  Harbor, N.Y. <br />  7.   Cheng, S.-C., J. Abelson. 1987. Spliceosome assembly yeast. Genes         36.   Mount, S. M., I. Pettersson, M. Hinterberger, A. Karmas, J. A. Steitz. <br />       Dev. 1:1014&#8211;1027.                                                                      1983. The U1 small nuclear RNA-protein complex selectively binds 59 <br />  8.   Dabeva, M., M. Post-Beittenmiller, J. Warner. 1986. Autogenous reg-                splice site vitro. Cell 33:509&#8211;518. <br />       ulation splicing transcript yeast ribosomal protein gene. Proc.   37.   Mount, S. M., J. A. Steitz. 1981. Sequence U1 RNA Drosophila <br />       Natl. Acad. Sci. USA 83:5854&#8211;5857.                                                     <span id='am-110' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-111' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-113'  ></span><span id='am-114' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span>melanogaster</span>: implications U1 secondary structure possible involve- <br />  9.   Dabeva, M., J. Warner. 1987. The yeast ribosomal protein L32               ment splicing. Nucleic Acids Res. 9:6351&#8211;6368. <br />       gene. J. Biol. Chem. 262:16055&#8211;16059.                                            38.   Nakano, A., M. Muramatsu. 1989. A novel GTP-binding protein, Sar1p, <br /> 10.   Domdey, H., B. Apostol, R.-J. Lin, A. Newman, E. Brody, J. Abelson.                involved transport endoplasmic reticulum Golgi appa- <br />       1984. Lariat structures vivo intermediates yeast pre-mRNA splicing.          ratus. J. Cell Biol. 109:2677&#8211;2691. <br />       Cell 39:611&#8211;621.                                                                 39.   Nelissen, R., C. Will, W. Venrooij, R. Luhrmann. 1994. The association <br /> 11.   Elledge, S., R. Davis. 1988. A <span id='am-30' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-31' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> versatile centromeric <span id='am-95' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-96' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>vectors</span>             U1-specific 70K C proteins U1 snRNPs mediated part <br />       designed use sectoring-shuffle assay Saccharomyces cerevisiae.           common U snRNP proteins. EMBO J. 17:4113&#8211;4125. <br />       Gene 70:303&#8211;312.                                                                 40.   Parker, R., R. Patterson. 1987. Architecture fungal introns: implica- <br /> 12.   Fabrizio, P., S. Esser, B. Kastner, R. Luhrmann. 1994. Isolation S.             tions spliceosome assembly, p. 451&#8211;461. In M. Inouye B. Dudock <br />       cerevisiae snRNPs: comparison U1 U4/U6.U5 human coun-                  (ed.), Molecular biology RNA: new perspectives. Academic Press, San <br />       terparts. Science 264:261&#8211;265.                                                         Diego, Calif. <br /> 13.   Frank, D., C. Guthrie. 1992. An essential splicing factor, SLU7, medi-       41.   Pascolo, S., M. Ghazvini, J. Boyer, L. Colleaux, A. Thierry, B. Dujon. <br />       ates 39 splice site choice yeast. Genes Dev. 6:2112&#8211;2124.                           1992. The sequence 9.3 kb segment located left arm yeast <br /> 14.   Frendewey, D., W. Keller. 1985. The stepwise assembly pre-mRNA                chromosome XI reveals open reading frames including CCE1 gene <br />       splicing complex requires U-snRNPs specific intron sequences. Cell                 putative products related MYO2 ribosomal protein L10. <br />       42:355&#8211;367.                                                                            Yeast 8:987&#8211;995. <br /> 15.   Grabowski, P. J., S. R. Seiler, P. A. Sharp. 1985. A multicomponent          42.   Pikielny, C., M. Rosbash. 1985. mRNA splicing efficiency yeast and <br />       complex involved splicing messenger RNA precursors. Cell                  contribution nonconserved sequences. Cell 41:119&#8211;126. <br />       42:345&#8211;353.                                                                      43.   Pikielny, C., B. C. Rymond, M. Rosbash. 1986. Electrophoresis of <br /> 16.   Guthrie, C., J. Fink. 1991. Guide yeast genetics molecular                  ribonucleoproteins reveals ordered assembly pathway yeast splicing <br />       biology. Academic Press, San Diego, Calif.                                             complexes. Nature (London) 324:341&#8211;345. <br /> 17.   Hamm, J., V. L. van Santen, R. Spritz, I. W. Mattaj. 1988. Loop I U1      44.   Query, C. C., R. C. Bently, J. D. Keene. 1989. A common RNA recog- <br />       small nuclear RNA essential RNA sequence binding                    nition motif identified defined U1 RNA binding domain 70K <br />       specific U1 small nuclear ribonucleoprotein particle proteins. Mol. Cell.              U1 snRNP protein. Cell 57:89&#8211;101. <br />       Biol. 8:4787&#8211;4791.                                                               45.   Rose, M., J. Broach. 1991. Cloning genes complementation yeast. <br /> 18.   He, F., S. Peltz, J. Donahue, M. Rosbash, A. Jacobson. 1993. Stabiliza-            Methods Enzymol. 194:195&#8211;230. <br />       tion ribosome association unspliced pre-mRNAs yeast upf1-            46.   Ruby, S. W., J. Abelson. 1988. An early hierarchic role U1 small <br />       mutant. Proc. Natl. Acad. Sci. USA 90:7034&#8211;7038.                                       nuclear ribonucleoprotein spliceosome assembly. Science 242:1028&#8211;1035. <br /> 19.   Hilleren, P. J., H.-Y. Kao, P. G. Siliciano. 1995. The amino-terminal        47.   Schena, M., D. Picard, K. Yamamoto. 1991. Vectors constitutive and <br />       domain yeast U1-70K necessary sufficient function. Mol. Cell.            inducible gene expression yeast. Methods Enzymol. 194:389&#8211;398. <br />       Biol. 15:6341&#8211;6350.                                                              48.   Schiestl, R., R. Gietz. 1989. High efficiency transformation intact yeast <br /> 20.   Johnston, G., J. Prendergast, R. Singer. 1991. The Saccharomyces                   cells using single stranded nucleic acids carrier. Curr. Genet. 16:339&#8211;346. <br />       cerevisiae MYO2 gene encodes essential myosin vectoral transport       49.   Seraphin, B., L. Kretzner, M. Rosbash. 1988. A U1 snRNA:pre-mRNA <br />       vesicles. J. Cell Biol. 113:539&#8211;551.                                                   base pairing interaction required early yeast spliceosome assembly but <br /> 21.   Jones, M., C. Guthrie. 1990. Unexpected flexibility evolutionarily           does uniquely define 59 cleavage site. EMBO J. 7:2533&#8211;2538. <br />       conserved protein-RNA interaction: genetic analysis Sm binding site.      50.   Seraphin, B., M. Rosbash. 1989. Identification functional U1 snRNA- <br />       EMBO J. 9:2555&#8211;2561.                                                                   pre-mRNA complexes committed spliceosome assembly splicing. Cell <br /> 22.   Kao, H.-Y., P. G. Siliciano. 1992. The yeast homolog U1 snRNP               59:349&#8211;358. <br />       protein 70K encoded SNP1 gene. Nucleic Acids Res. 20:4009&#8211;             51.   Seraphin, B., M. Rosbash. 1991. The yeast branchpoint sequence not <br />       4013.                                                                                  required formation stable U1 snRNA-pre-mRNA complex and <br /> 23.   Kretzner, L., A. Krol, M. Rosbash. 1990. Saccharomyces cerevisiae U1               recognized absence U2 snRNA. EMBO J. 10:1209&#8211;1216. <br />       small nuclear RNA secondary structure contains universal yeast-         52.   Sikorski, R., P. Hieter. 1989. A shuttle vectors yeast host <br />       specific domai <br /> </body></html>