<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Nov. 2008, p. 6939&#8211;6953                                                                           Vol. 28, No. 22 <br /> 0270-7306/08/$08.00&#11001;0 doi:10.1128/MCB.01115-08 <br /> Copyright &#169; 2008, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br /> The KREPA3 Zinc Finger Motifs OB-Fold Domain Are Essential <br />       RNA Editing Survival <span id='am-7' about='obo:NCBITaxon_5690' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_2759 obo:NCBITaxon_5653'><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Trypanosoma" datatype="xsd:string"></span><span id='am-10'  ></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Trypanosoma</span> brucei&#17188; <br />                             Xuemin Guo,1 Nancy Lewis Ernst,1 Kenneth D. Stuart1,2* <br />                  Seattle Biomedical Research Institute, Seattle, Washington 98109,1 Department Pathobiology, <br />                                        University Washington, Seattle, Washington 981952 <br />                         Received 15 July 2008/Returned modification 14 August 2008/Accepted 3 September 2008 <br />  <br />              Three types editosomes, identical core containing related KREPA proteins, catalyze U <br />           insertion deletion RNA editing mitochondrial mRNAs trypanosomes. Repression expression of <br />           these, KREPA3 (also known TbMP42), shows essential growth vivo editing procyclic <br />           (PF) bloodstream (BF) life cycle stages <span id='am-12' about='obo:NCBITaxon_5691' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_39700 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_5690 obo:NCBITaxon_2759 obo:NCBITaxon_5653'><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Trypanosoma brucei subgroup" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Trypanosoma brucei" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-17'  ></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Trypanosoma (Trypanozoon) brucei" datatype="xsd:string"></span>Trypanosoma brucei</span>. RNA interference knockdown results in <br />           editosome disruption altered vitro editing PFs, repression regulatable double knockout results <br />           complete loss editosomes BFs. Mutational analysis shows KREPA3 zinc fingers OB-fold <br />           domain essential growth vivo editing. Nevertheless, KREPA3 mutated zinc fingers <br />           incorporates editosomes catalyze vitro editing essential editosome integrity, although <br />           stability affected. In contrast, OB-fold domain essential editosome integrity. Overall, KREPA3, espe- <br />           cially OB-fold, functions editosome integrity, zinc fingers essential editing vivo for <br />           central catalytic steps. KREPA3 function editosome organization and/or RNA positioning. <br />  <br />  <br />    RNA editing trypanosomes creates mature mitochondrial                    proteins associated complexes have <br /> (mt) mRNAs insertion deletion uridylates (U&#8217;s)                   identified function accessory factors. <br /> as specified sequences small guide RNAs (gRNAs)                         There types editosomes distinct editing site <br /> (6, 52). It employs multiprotein complexes,                     cleavage specificities. Each distinct endonucle- <br /> which editosome sediments &#11011;20S glycerol                      ase, KREN1, KREN2 KREN3, unique <br /> gradients contains enzymes catalyze key cat-                  proteins, identical &#8220;core&#65533;? proteins (Fig. 1) (11, <br /> alytic steps pre-mRNA cleavage, U insertion deletion,                    38). The identical &#8220;core&#65533;? contains related A-type kineto- <br /> and RNA ligation (32, 50, 53). The pre-mRNA par-                       plastid RNA editing proteins (KREPA1 KREPA6), of <br /> tially duplexed gRNA 3&#11032; editing site (ES),                   C-terminal OB-fold domain, larg- <br /> which editing occurs cycles enzymatic steps. The pre-                    est (KREPA1, -A2, -A3) C2H2 zinc finger <br /> mRNA cleaved ES KREN1 endonuclease                       (ZF) motifs (39, 62). KREPA1 KREPA2 form dif- <br /> is specific deletion sites KREN2 KREN3,                     ferent heterotrimeric subcomplexes vivo vitro that <br /> are specific different insertion sites. One U&#8217;s                catalyze postcleavage steps insertion deletion edit- <br /> added 5&#11032; mRNA fragment KRET2 3&#11032; terminal                         ing, respectively (14, 15, 19, 28, 37, 47, 53). KREPA1 associates <br /> uridylyl transferase U&#8217;s removed KREX1                        KRET2 KREL2 resulting insertion subcomplex <br /> KREX2 3&#11032; exonuclease, number added removed                      adds U&#8217;s ligates products specified the <br /> U&#8217;s specified gRNA sequence. The processed 5&#11032; frag-                     gRNA. Similarly, KREPA2 associates KREX2 and <br /> ment religated 3&#11032; fragment KREL1                    KREL1 deletion subcomplex catalyzes U removal and <br /> KREL2 RNA ligase (3, 10&#8211;12, 15, 16, 27, 45, 48, 56). Multiple                  ligation specified gRNA. The binary interactions these <br /> cycles editing extend pre-mRNA/gRNA du-                       A proteins specific enzymes enhance catalytic <br /> plex, generally 5&#11032; direction, completely                 activity latter. KREPA proteins appear play roles in <br /> complementary. The editing mRNAs requires multiple                     architecture function editosomes. Loss <br /> gRNAs specifying editing 3&#11032; regions                     expression KREPA1, KREPA2, KREPA4, KREPA6 <br /> creating pre-mRNA sequence complementary                      proteins results partial complete loss editosomes (19, <br /> 5&#11032; &#8220;anchor&#65533;? sequence subsequent gRNA providing                     29, 37, 46, 54). Brecht et al. (8) reported recombinant <br /> for orderly use. Other complexes involved editing,                KREPA3 binds double-stranded RNA (dsRNA) dsDNA <br /> including MRP1/MRP2, RBP16, TbRGG1, RET1                               endo-exo-RNase activity localizes OB-fold <br /> complexes, appear preprocess traffic mRNAs <br />                                                                                domain knockdown little effect 20S editosome <br /> and gRNAs regulate editing (2, 33, 34, 42, 57). Other <br />                                                                                integrity. However, Law et al. (30) KREPA3 was <br />                                                                                important editosome stability endonuclease activity but <br />                                                                                U addition, U removal, RNA ligase activities. These <br />                                                                                findings resolved identi- <br />   * Corresponding author. Mailing address: Seattle Biomedical Re-              fication editing endonucleases U-specific exo- <br /> search Institute, 307 Westlake Avenue North, Suite 500, Seattle, WA <br /> 98109. Phone: (206) 256-7316. Fax: (206) 256-7229. E-mail: ken.stuart <br />                                                                                nucleases (10, 11, 27, 45, 56) reflect experimental <br /> @sbri.org.                                                                     differences, roles KREPA3 affect editosome <br />   &#17188; <br />     Published ahead print 15 September 2008.                             structure enzymatic activities. <br />  <br />                                                                         6939 <br />  6940     GUO ET AL.                                                                                                               MOL. CELL. BIOL. <br />  <br />  <br />                                                                       gRNA oligo(U) tail hypothesized use OB-fold <br />                                                                       domain coordinate forward progression RNA editing <br />                                                                       steps (46, 54). The OB-fold KREPA3 proposed have <br />                                                                       RNA recognition ribonucleolytic activities (8, 36). <br />                                                                          We KREPA3 essential RNA editing <br />                                                                       survival PF BF parasites. We that <br />                                                                       KREPA3 important editosome integrity knock- <br />                                                                       downs PFs generate smaller complexes U addition, <br />                                                                       U removal, ligation steps editing characteristic <br />                                                                       heterotrimeric insertion deletion editosome sub- <br />                                                                       complexes. We KREPA3 bridges subcom- <br />                                                                       plexes editosome core. The deletion insertion <br />   FIG. 1. Schematic representation 20S editosomes. Each     endonucleolytic activities reduced associated <br /> three editosomes consists common core proteins (rectangle)    &#11011;20S editosomes. We transcriptional <br /> with insertion subcomplex consisting KREPA1, KRET2,         inactivation KREPA3 BFs, appears complete <br /> KREL2 deletion subcomplex KREPA2, KREX2, and <br /> KREL1. KREPA3 core editosomes (bold). Each           RNA interference (RNAi) knockdown, results essen- <br /> editosome different endonuclease, KREN1, KREN2,              tially complete loss editosomes vitro catalytic <br /> KREN3 (N1-3) unique KREPB protein (B6-7). Only KREN1            activities. Exclusive expression KREPA3 proteins mu- <br /> editosomes contain KREX1 (X1). KREN1 deletion endonuclease            tations ZFs deletion OB-fold resulted <br /> editosomes contain KREPB8 KREX1, KREN2 insertion endo- <br />                                                                       ablation RNA editing vivo cell death. Tagged <br /> nuclease editosomes contain KREPB7, KREN3 insertion en- <br /> donuclease editosomes contain KREPB6. The KREH1 helicase           KREPA3 ZFs mutated incorporate &#11011;20S <br /> found biochemically TAP tag-purified editosomes.           editosomes, catalyze activities vitro. In con- <br />                                                                       trast, tagged KREPA3 OB-fold deleted does in- <br />                                                                       corporate editosomes rescue &#11011;20S editosomes upon <br />    C2H2-ZF proteins display diversity ZF number, position          loss wild-type (WT) KREPA3. Thus, ZFs OB-fold <br /> in protein, amino acid spacing, flanking sequence,         domain indispensable KREPA3 function vivo. How- <br /> their ligands include dsDNA, dsRNA, DNA-RNA hy-                 ever, KREPA3 responsible central catalytic steps <br /> brids, single-stranded RNA, protein (22). The binding             editing, affect steps indirectly through <br /> modes differ ligands (7, 31, 61). Binding C2H2           editosome structure. Hence, KREPA3 appears function in <br /> ZFs DNA sequence specific binding RNA                RNA binding editosome organization, coordi- <br /> shows enormous diversity sequence specific not,         nate steps editing and/or affect processivity of <br /> whereas ZFs typically function form <span id='am-38' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-42'  ></span>homo</span>- heterodimers          editing. <br /> between proteins. The ZFs KREPA1, KREPA2, and <br /> KREPA3 widely separated                                         MATERIALS AND METHODS <br /> amino acids, atypical C2H2 ZFs. Except for <br />                                                                          Plasmid constructs. The plasmid expressing tetracycline (tet)-inducible RNAi <br /> the C-terminal ZF KREPA1 (that additional                 KREPA3 constructed according report Brecht et al. (8). A 701-bp <br /> amino acids second cysteine histi-          fragment 3&#11032; end KREPA3 amplified using primers 5&#11032;- <br /> dine), ZFs share (F/T)XCX2CX3FX5&#9023;X2HX4H                     ATACTCGAGCTGCTGCTAATTCTGC-3&#11032; 5&#11032;-ATAAAGCTTTCACACCT <br /> pattern ZF proteins roles       TCAACACTG-3&#11032;, digested XhoI HindIII, inserted similarly <br />                                                                       digested pZJM (59) create pZJM-A3. The restriction sites underlined. The <br /> in DNA, RNA, protein binding. The ZFs Trypanosoma              <span id='am-66' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> used generating regulatable knockout (RKO) cell line, BF <br /> brucei KREPA1, KREPA2, KREPA3,                     KREPA3-RKO, based previous strategy (48, 60) modifications. <br /> KREPA2 ortholog <span id='am-1' about='obo:NCBITaxon_5689' typeof='owl:Thing obo:NCBITaxon_5688 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_5658 obo:NCBITaxon_33682 obo:NCBITaxon_1286322 obo:NCBITaxon_5654 obo:NCBITaxon_2759 obo:NCBITaxon_5653'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Sauroleishmania tarentolae" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Leishmania tarentolae" datatype="xsd:string"></span><span id='am-6'  ></span>Leishmania tarentolae</span>, LC-4,             Distal 5&#11032; 3&#11032; KREPA3 open reading frame (ORF) flanking sequences located <br /> proposed act protein-protein interaction substrate           outside flanking sequences used pLEW90-A3-KO2 (below) amplified <br />                                                                       T. brucei 427 genomic DNA using primers 5&#11032;-ATAGCGGCCGCGCA <br /> RNA recognition binding (25). <br />                                                                       CACGTGTGGACATTATC-3&#11032; 5&#11032;-ATAACGCGTCTCGAGAACTTATGG <br />    Different OB-fold domains little sequence similarity      GGTAGAAGACG-3&#11032; primers 5&#11032;-ATATCTAGAATTTAAATACCAAG <br /> share conserved topological architecture consisting         CGAGAAAGGGAAGA-3&#11032; 5&#11032;- ATAAGGCCTGCGGCCGCTAATGCTGT <br /> &#65533;?&#65533;-strands forming antiparallel &#65533;?&#65533;-sheets support com-       TATGGGCGCTG-3&#11032;, respectively. The PCR products digested NotI and <br /> mon binding face ligands, includes nucleic acids,   MluI XbaI StuI, respectively, inserted similarly digested <br />                                                                       pLEW13 generate construct pLEW13-A3-KO1 replacement first <br /> oligosaccharides, proteins (1, 35, 55). The loops con-      KREPA3 allele. The 5&#11032; 3&#11032; flanking sequences adjacent KREPA3 ORF <br /> necting &#65533;?&#65533;-strands vary widely length conformation      amplified genomic DNA using primers 5&#11032;-ATAGCGGCCGCTT <br /> accommodate various modes binding different ligands.        GGCGGAAGAAAGAGAACG-3&#11032; 5&#11032;-ATAACGCGTCTCGAGGGACGAG <br /> The C-terminal regions KREPA1, KREPA2, KREPA3,                 ACATGAGGTGATG-3&#11032; primers 5&#11032;-ATATCTAGAATTTAAATTGGTG <br />                                                                       CCTCAATATGATGGT-3&#11032; 5&#11032;-ATAAGGCCTGCGGCCGCTTGGGACCC <br /> KREPA6 predicted OB-fold domains based their <br />                                                                       ATCGCTATTTG-3&#11032;, respectively. The PCR products digested NotI and <br /> structural organization key types amino acids,         MluI XbaI StuI, respectively, inserted similarly digested <br /> OB-fold KREPA4 similar S1-motif (46). The            pLEW13 generate construct pLEW13-A3-KO2. The XhoI-EcoRV fragment <br /> OB-fold domains KREPA1 KREPA2 proposed               pLEW13-A3-KO2 replaced XhoI-StuI fragment from <br /> to play roles substrate RNA recognition binding        pLEW90 generate pLEW90-A3-KO2 replacement second KREPA3 <br />                                                                       allele. The full-length KREPA3 ORF amplified genomic DNA using <br /> coordination TUTase ligase, exoUase ligase          primers 5&#11032;-CCTCGAGCCACCATGAAGCGTGTTACTTCAC-3&#11032; 5&#11032;- CG <br /> enzymatic steps editing, respectively (47). Both KREPA4        GATCCTTATCACACCTTCAACACTGACC-3&#11032; cloned pLEW79 after <br /> KREPA6 display RNA-binding activity preference         digestion XhoI BamHI generate pLEW79-KREPA3. <br />  VOL. 28, 2008                                                     KREPA3 ZFs AND OB-FOLD ARE ESSENTIAL FOR RNA EDITING                                           6941 <br />  <br />    To generate KREPA3 mutants tandem affinity purification (TAP) tag      min, followed 40 cycles 95&#176;C 15 s 60&#176;C 1 min, using ABI <br /> the C terminus, full-length KREPA3 KREPA3 deletion C-             Prism 7000 thermocycler. Relative changes target amplicons determined <br /> terminal OB-fold domain amplified using primers 5&#11032;-CCTCGAGC             comparing level KREPA3-repressed cells cells which <br /> CACCATGAAGCGTGTTACTTCAC-3&#11032; 5&#11032;- CGGATCCCACCTTCAAC                                KREPA3 expressed normalization 18S rRNA &#65533;?&#65533;-tubulin using <br /> ACTGACC-3&#11032; primers 5&#11032;-CCTCGAGCCACCATGAAGCGTGTTAC                            Pfaffl method (43), PCR efficiencies calculated linear regression using <br /> TTCAC-3&#11032; 5&#11032;-CGGATCCCCAATGTGCAGCCACTTC-3&#11032;, respectively.                         LinregPCR (44). <br /> After digestion XhoI BamHI, products cloned pLEW79-             Crude mitochondria preparation glycerol gradient sedimentation. Crude <br /> MHTAP (23, 40) create pLEW79-A3-TAP pLEW79-A3&#9004;OB-TAP. Mu-                    mitochondria prepared roughly 2 &#11003; 1010 PF KREPA3-RNAi cells or <br /> tagenesis ZF cysteines alanines performed using QuikChange       3 &#11003; 109 BF KREPA3-RKO RKO KREPA3 WT-TAP ZFm2-TAP cells in <br /> II XL site-directed mutagenesis kit (Stratagene, CA). Mutation ZF      absence presence 1 &#65533;?&#65533;g tet/ml essentially described previously (18). <br /> (C53A/C56A) performed oligonucleotides 5&#11032;-CACTCCCTCCTTT                The mitochondria PFs lysed 1.5 ml mt lysis buffer (20 mM <br /> CAGGCCGGCGAAGCCGGTAAGGCTTTTCGTC-3&#11032; 5&#11032;-GACGAAAA                                  HEPES [pH 7.9], 10 mM magnesium acetate, 100 mM KCl, 1 mM EDTA) <br /> GCCTTACCGGCTTCGCCGGCCTGAAAGGAGGGAGTG-3&#11032;, mutation                                containing protease inhibitors centrifuged twice 13,000 rpm 10 min at <br /> the second ZF (C183A/C186A) performed oligonucleotides 5&#11032;-G            4&#176;C. The cleared lysates loaded 11-ml 10 30% glycerol gradients and <br /> GACAAAAAAACATTTGTCGCGACGATTGCTCAGAAAACATTTCG                                        centrifuged 38,000 rpm 12 h 4&#176;C using SW40 rotor (Beckman). <br /> G-3&#11032; 5&#11032;-CCGAAATGTTTTCTGAGCAATCGTCGCGACAAATGTTTTT                                Twenty-four 500-&#65533;?&#65533;l fractions collected bottom. The crude <br /> TTGTCC-3&#11032;, resulting plasmids named pLEW79-A3ZFm1-TAP                   mitochondria BFs lysed 600 &#65533;?&#65533;l mt lysis buffer, cleared <br /> pLEW79-A3ZFm2-TAP. To generate pLEW79-A3ZFm1&amp;2-TAP, HindIII                       lysates loaded 4.5-ml glycerol gradients centrifuged 44,000 rpm <br /> and BamHI fragment containing ZFm2 excised pLEW79-                     9 h 4&#176;C using SW55 rotor (Beckman). Twenty-four fractions were <br /> A3ZFm2-TAP inserted pLEW79-A3ZFm1-TAP digested               collected bottom. All fractions flash frozen liquid <br /> enzymes. All mutations confirmed sequencing. To generate             nitrogen stored &#11002;80&#176;C Western blotting vitro enzymatic assays. <br /> KREPA3-RKO cell lines constitutively expressed TAP-tagged WT mu-               In vitro RNA editing assays. The standard vitro RNA editing activities assay <br /> tant KREPA3 alleles, TAP-tagged KREPA3 WT mutant genes                 described previously. Briefly, 5&#11032;CL18 3&#11032;CL13pp gPCA6-2A <br /> amplified plasmids described using primers 5&#11032;-CCTCGA          RNAs used precleaved insertion editing assay specifying insertion <br /> GCCACCATGAAGCGTGTTACTTCAC-3&#11032; 5&#11032;-ATTCATGATCAGGTT                                 U&#8217;s, U5 5&#11032;CL U5 3&#11032;CL gA6[14]PC-del RNAs used for <br /> GACTTCCCCGCGGAATTC-3&#11032;, digested XhoI BclI, inserted                    precleaved deletion editing specifying removal U&#8217;s (20, 21). 5&#11032;CL18 and <br /> individually pHD1344tub (10), targets integration constitutively      U5 5&#11032;CL 5&#11032; labeled [&#65533;?&#65533;-32P]ATP T4 kinase. A6short/TAG.1 pre- <br /> expressed ectopic alleles &#65533;?&#65533;-tubulin locus.                                     mRNA D34 gRNA RNA substrates deletion site cleavage assays, <br />    Transfections repression KREPA3 expression. The T. brucei PF 29-13        A6-eES1 pre-mRNA gA6[14] gRNA RNA substrates insertion <br /> cell line (60) transfected 10 &#65533;?&#65533;g NotI-linearized pZJM-A3, stable    site cleavage assays (10, 13, 24, 49). A6short/TAG.1 A6-eES1 mRNAs were <br /> transfectants screened described previously (59). The resulting cell line   3&#11032; labeled 32P-labeled pCp T4 RNA ligase. Portions (10 &#65533;?&#65533;l) indi- <br /> was named PF KREPA3-RNAi maintained SDM-79 medium plus 10%                   vidual pooled gradient fractions TEV eluates purification immu- <br /> fetal bovine serum 15 &#65533;?&#65533;g G418/ml, 25 &#65533;?&#65533;g hygromycin/ml, 2.5 &#65533;?&#65533;g        noglobulin G (IgG) beads incubated different substrate RNAs for <br /> of phleomycin/ml. dsRNA induced adding 1 &#65533;?&#65533;g tet/ml. Growth         enzymatic reactions. The reaction products resolved 7 M urea-poly- <br /> cells monitored absence presence tet counted daily      acrylamide gels visualized Storm PhosphorImager screens (Molecular <br /> with Coulter counter. The cells maintained 1.0 &#11003; 106 2.0 &#11003;       Dynamics). <br /> 107 cells/ml.                                                                          RNA ligase adenylation assays performed follows. mt gradient frac- <br />    The BF KREPA3-RKO cell line generated previously described (48,           tions (5 &#65533;?&#65533;l) incubated 5 &#65533;?&#65533;Ci [&#65533;?&#65533;-32P]ATP 20-&#65533;?&#65533;l reaction system <br /> 60) maintained HMI-9 medium plus 10% fetal bovine serum multiple        (12.5 mM KCl, 12.5 mM HEPES [pH 7.9], 5 mM magnesium acetate, 0.25 mM <br /> selection markers (below). Briefly, wild-type BF 427 cells electroporated      dithiothreitol) 28&#176;C 5 min stopped addition 10 &#65533;?&#65533;l 3&#11003; <br /> with NotI-linearized plasmids stepwise <span id='am-43' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-44' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span>; pLEW13-A3-KO1         sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) loading <br /> and selection 2.5 &#65533;?&#65533;g G418/ml replace KREPA3 alleles,     dye. The proteins resolved SDS&#8211;10% PAGE, radiolabeled <br /> with pLEW79-KREPA3 selection 2.5 &#65533;?&#65533;g phleomycin/ml insert         proteins detected using PhosphorImager. <br /> tet-regulatable ectopic KREPA3 allele ribosomal DNA spacer region,            Coimmunoprecipitation purification TAP-tagged proteins. The mono- <br /> and finally pLEW90-A3-KO2 selection 5 &#65533;?&#65533;g hygromycin/ml          clonal antibody (MAb) specific KREPA2 (39) incubated goat <br /> addition 1 &#65533;?&#65533;g tet/ml induce ectopic KREPA3 expression replace      anti-mouse IgG-conjugated magnetic beads (Dynal) 2 h 4&#176;C rotation. <br /> second KREPA3 endogenous allele. Correct integration plasmid            The crude mt lysates uninduced induced PF KREPA3-RNAi cells were <br /> confirmed PCR. The expression ectopic KREPA3 repressed wash-           lysed IPP200 containing protease inhibitor cocktail 1% Triton X-100 <br /> ing culturing cells medium minus tet. Growth cells        (Roche). The cleared lysate incubated KREPA2 MAb coated beads <br /> monitored presence absence tet diluted daily 1.0 &#11003; 105       2 h 4&#176;C rotation. The beads washed IPP200 times, and <br /> 2.0 &#11003; 105 cells/ml.                                                                 1&#11003; SDS loading buffer added boiled 5 min. The starting lysate, the <br />    The pHD1344tub plasmids TAP-tagged WT mutant KREPA3 genes                supernatant immunoprecipitation, precipitates resolved by <br /> for constitutive expression tubulin locus linearized NotI    SDS&#8211;10% PAGE, Western blot performed specific MAbs <br /> transfected KREPA3-RKO cells independently. The resistant clones          KREPA1, KREPA2, KREPA3, KREL1 (39). <br /> selected 0.1 &#65533;?&#65533;g puromycin/ml designated RKO-KREPA3 WT-TAP,                 BF KREPA3-RKO tagged RKO-KREPA3 WT mutant cells (2 &#11003; 108) <br /> ZFm1-TAP, ZFm2-TAP, ZFm1&amp;2-TAP, &#9004;OB-TAP. Expression                      grown presence absence tet 3 days harvested lysed with <br /> tagged genes determined Western blotting.                                    IPP150 containing cocktail protease inhibitors (Roche) plus 1% Triton <br />    RNA isolation real-time RT-PCR analysis. Total RNA harvested             X-100. The whole-cell lysates incubated IgG-conjugated agarose beads <br /> from cell lines using TRIzol LS reagent (Gibco-BRL) according        2 h cleaved TEV protease 2 h 16&#176;C. The starting whole-cell <br /> manufacturer&#8217;s instructions. Real-time reverse transcription-PCR (RT-PCR)           lysates TEV eluates resolved SDS&#8211;10% PAGE assayed by <br /> was performed described previously (9). Briefly, 10 &#65533;?&#65533;g RNA treated        Western blotting MAbs KREPA1, KREPA2, KREPA3, KREL1 <br /> with DNase I using DNA-free kit (Ambion). Then, 4.5 &#65533;?&#65533;g treated          polyclonal antibodies KREPA6 (47) PGK (41), level PGK <br /> RNA reverse transcribed using random hexamers TaqMan RT                  control. Portions (10 &#65533;?&#65533;l) TEV eluates used in <br /> reagents (Applied Biosystems) generate cDNA templates real-time          vitro RNA editing assay. <br /> PCR. At time, control reaction reverse transcriptase was <br /> carried rule contamination genomic DNA. The cDNA reactions <br /> were diluted 1:7 water template PCRs diluted 1:50 <br />                                                                                                                       RESULTS <br /> for PCR high abundance internal control 18S rRNA &#65533;?&#65533;-tubulin. Each <br /> PCR included 2.5 &#65533;?&#65533;l cDNA, 10 &#65533;?&#65533;l forward reverse primers (each 0.75       KREPA3 essential growth vivo RNA editing. <br /> &#65533;?&#65533;M), 12.5 &#65533;?&#65533;l Sybr green PCR master mix (Applied Biosystems) mixed in <br /> a 96-well plate (Applied Biosystems), kind mRNA was <br />                                                                                     The effect KREPA3 repression cell growth examined <br /> analyzed triplicate. The sequences of used primers de-       PF BF stages T. brucei. Expression of <br /> scribed (9). The amplification conditions 50&#176;C 2 min 95&#176;C 10       KREPA3 repressed PFs generating RNAi cell line <br />  6942     GUO ET AL.                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 2. KREPA3 repression essential growth vivo RNA editing BFs PFs. Growth PF KREPA3-RNAi (A) BF <br /> KREPA3-RKO (B) cells KREPA3 expressed repressed. KREPA3 expression repressed addition tet PFs induce <br /> RNAi expression withdrawal tet BFs repress KREPA3 transcription. The insets Western analysis lysates PF cells <br /> at day 6 BF cells day 4 using KREPA3 MAb. The positive controls pooled 20S fractions glycerol gradient purified mt lysate from <br /> WT PF mitochondria panel A BF cell lysate panel B. (C D) The relative abundance pre-edited, edited, never-edited ND4 <br /> and COI mRNAs cells KREPA3 repressed compared cells expressed PF KREPA3-RNAi cells day 6 (C) and <br /> BF KREPA3-RKO cells day 4 (D). The relative change target amplicon determined real-time RT-PCR total RNA and <br /> normalized &#65533;?&#65533;-tubulin 18S rRNA. Note log scale 1 represents difference, &#11022;1 increase, &#65533;?1 decrease. <br />  <br />  <br />  <br /> in 29-13 PFs (PF KREPA3-RNAi) expresses dsRNA                   pressed PF BF control cells grew normal rate. <br /> portion KREPA3 mRNA control tet-inducible                 KREPA3 protein detected PFs BFs 6 4 <br /> promoter previously described (8). Expression KREPA3                days repression, respectively, (insets, Fig. 2A B) as <br /> was blocked BFs generating regulatable knockout cell              determined Western analysis KREPA3 MAb. Some <br /> line (BF KREPA3-RKO) disrupting endogenous al-                    motile PF RNAi cells seen microscope 12 days of <br /> leles homologous recombination inserting tet-regu-               RNAi induction tet, cell debris seen 6 days <br /> latable ectopic KREPA3 allele (KREPA3 Reg) ribo-                 repression BF RKO cells tet withdrawal. The <br /> somal DNA locus described Materials Methods. Cell               motile PF cells BF cells indicate complete <br /> growth inhibited dramatically KREPA3 repression                  KREPA3 repression RNAi compared conditional <br /> by tet-induced RNAi PFs (Fig. 2A) completely blocked               knockout, commonly observed, reflect <br /> in BFs withdrawal tet (Fig. 2B), nonre-               slower growth rate PFs, differences life <br />  VOL. 28, 2008                                            KREPA3 ZFs AND OB-FOLD ARE ESSENTIAL FOR RNA EDITING                           6943 <br />  <br />  <br /> cycles stages, combination thereof. Real-time PCR                KREPA3 repression, seen using cocktail four <br /> analysis total RNA cells showed mRNA                MAbs auto-adenylation KREL1 KREL2 (Fig. <br /> levels KREPA1, KREPA2, KREPA5, KREPA6                        3B). KREPA1 adenylatable KREL1 KREL2 shifted <br /> editosome proteins, related KREPA3, un-                smaller S region gradient KREPA3 repression <br /> affected KREPA3 repression RNAi induction RKO                   did KREPA2 KREL1 proteins seen Western <br /> (<span id='am-56' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-57'  ></span><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> shown), indicating repression specific         analysis (Fig. 3B). Hence, subcomplex containing KREPA1 <br /> KREPA3. Overall, dramatic repression growth               KREL2 fewer components with <br /> PFs BFs repression KREPA3 expression indicates                 KREPA2 KREL1. The peak KREL1 protein the <br /> that protein essential life cycle stages T. brucei.   greater S value region gradient seen Western <br />    The effect KREPA3 repression RNA editing vivo                blotting autoadenylation, implying adenylated <br /> was assessed PF BF stages real-time PCR. Total            primed ligate RNA. Some editosomes remain in <br /> RNA isolated nonrepressed PF KREPA3-RNAi                     &#11011;20 40S region following KREPA3 repression despite <br /> BF KREPA3-RKO cells, cells 6              extensive loss KREPA3. <span id='am-33' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37'  ></span>This</span> reflect less-than- <br /> 4 days repression KREPA3 expression, respectively,            complete elimination KREPA3 faint signal for <br /> growth inhibition manifested. The levels edited           KREPA3 detected region longer exposures <br /> pre-edited mRNAs ND9, never-edited                 and/or presence editosomes lack protein. Over- <br /> ND4 COI mRNAs, determined normalized                     all, data indicate KREPA3 required editosome <br /> the &#65533;?&#65533;-tubulin mRNA 18S rRNA internal controls. The                   integrity critical bridge interaction inser- <br /> levels ND4 COI mRNAs repressed cells relative              tion deletion subcomplexes, insertion subcom- <br /> to nonrepressed cells unaffected,              plexes failed associate deletion subcomplexes. <br /> levels pre-edited mRNAs unchanged increased                   KREPA3 repression resulted shift vitro <br /> various amounts (Fig. 2C D). In contrast, relative               precleaved U-insertion/ligation U-deletion/ligation activi- <br /> levels edited mRNAs reduced dramatically PF              ties smaller S value gradients, fraction 15-23 to <br /> and BF cells KREPA3 repression. All 2 11                fraction 11-17 (Fig. 3C D), corresponding shift of <br /> edited mRNAs tested reduced 90% (Fig. 2C                editosome proteins. KREPA3 repression did result in <br /> D). The level edited CYb mRNA detection                  obvious reduction U-insertion U-deletion activities. <br /> BFs (data shown). Most edited mRNAs re-                  However, TUTase exoUase activities somewhat <br /> duced BFs PFs, especially evident COII               affected ligated products U added (Fig. 3C) and <br /> mRNA, lower efficiency KREPA3 repres-                U&#8217;s removed (Fig. 3D) enhanced KREPA3 <br /> sion RNAi compared regulatable double knockout.                repression, possibly alterations editosome integ- <br /> The lack concomitant increase levels pre-            rity. In contrast precleaved editing activities, inser- <br /> mRNAs dramatic reduction edited RNA                    tion deletion site endonuclease activities remained &#11011;20 <br /> reflect inherent differential turnover rates RNAs.          40S reduced KREPA3 repression <br /> Thus, KREPA3 essential RNA editing vivo PFs            compared control cells (Fig. 3E F). The of <br /> and BFs. The extensive reduction edited mRNA                     remaining endonuclease activities correlates residual <br /> KREPA3 knockdown suggests structurally                  level editosomes KREPA3 level was <br /> necessary editosome integrity and/or play critical             determined phosphorimager quantitation (data shown). <br /> roles RNA editing process vivo.                                Thus, reduction endonuclease activities appears be <br />    KREPA3 repression PFs affects editosome stability              editosome disruption loss KREPA3. <br /> in vitro editing. The effect KREPA3 repression PFs              Overall, data indicate repression KREPA3 ex- <br /> editosome stability integrity examined coimmuno-              pression affected vitro editing activities this <br /> precipitation sedimentation glycerol gradients. Lysates           effect mainly attributable altered editosome structure. <br /> from PF KREPA3-RNAi cells KREPA3                        This differs conclusions Brecht et al. (8) dis- <br /> expressed repressed incubated KREPA2 MAb                  cussed below. The &#11011;20S editosomes retained PF cells upon <br /> bound IgG-coated magnetic beads. The supernatants                 KREPA3 repression competent vitro RNA ed- <br /> immunoprecipitates examined Western analysis using               iting activities insufficient support cell growth in <br /> a mixture MAbs specific KREPA1, KREPA2, KREL1,                    vivo editing. <br /> and KREPA3 (Fig. 3A). Although repression KREPA3                     Editosome loss KREPA3 repression BFs. Repres- <br /> inhibited significantly 6 days RNAi induction,        sion KREPA3 expression BF KREPA3-RKO cells re- <br /> editosome proteins evident total lysate (input). The         sulted dramatic loss editosomes vitro editing <br /> loading control shown. KREPA2 KREL1                 activities (Fig. 4) contrast partial effect observed PF <br /> immunoprecipitated lysates KREPA3 ex-                KREPA3-RNAi cells. Western analysis crude mt lysates <br /> pression repressed, KREPA1 remained                  antibody cocktail editosome proteins <br /> supernatant KREPA3 repressed. This consistent                showed proteins lost BF KREPA3-RKO <br /> with previous findings KREL1 interacts KREPA2                  cells 4 days repression KREPA3 expression (Fig. <br /> and occurs deletion subcomplex, KREX2,             4A). MAb66 used detect 3-methylcrotonoyl-coen- <br /> KREPA1, KREL2, KRET2 occur insertion subcom-                   zyme A carboxylase &#65533;?&#65533; protein (A. Panigrahi, unpublished <br /> plex (47). Glycerol gradient sedimentation cleared crude mt           data), relationship RNA editing, used <br /> lysates cells revealed shift peak             loading control. A weak signal discernible the <br /> editosome components &#11011;20S smaller S region                KREPA2 region blot KREPA3 repression BFs, <br />  6944      GUO ET AL.                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 3. Disruption editosomes effect vitro RNA editing activities KREPA3 repression RNAi PFs. (A) Western analysis of <br /> crude mt lysates PF KREPA3-RNAi cells KREPA3 expressed repressed 6 days (input) immunoprecipitation <br /> with KREPA2 MAb. The Western used mixture MAbs editosome components indicated right. Note presence KREPA2 <br /> and KREL1 precipitate. The 20S positive control described Fig. 2A. (B) Western adenylation analyses glycerol gradient fractions of <br /> crude mt lysates cells panel A. The Western blot analyses used mixture MAbs indicated, adenylation assays reveal KREL1 <br /> and KREL2. These fractions used vitro precleaved insertion (C) precleaved deletion (D) assays endonuclease assays specific for <br /> A6 insertion (E) deletion (F) sites. The RNA substrates described Materials Methods. The radiolabeled input RNAs (I), input RNA with <br /> two U&#8217;s added (&#11001;2U), U&#8217;s removed (&#11002;4U), ligated products unprocessed 5&#11032; 3&#11032; input RNAs (Lig) edited (Ed) products indicated. <br /> The position specific, gRNA-directed cleavage products indicated arrow. gRNA omitted control reactions (g-) RNase <br /> T1-digested substrate RNA (T1) used marker. Positive control reactions (&#11001;) performed using fraction &#11011;20S editosome contains <br /> peak editing activity, omitted negative control reactions (&#11002;). <br />  VOL. 28, 2008                                            KREPA3 ZFs AND OB-FOLD ARE ESSENTIAL FOR RNA EDITING                               6945 <br />  <br />  <br />  <br />  <br />    FIG. 4. KREPA3 repression BFs conditional knockout results loss editosome vitro RNA editing activities. (A) Different <br /> effect KREPA3 repression editosome BFs compared PFs. Western analysis crude mt lysates BF KREPA3-RKO (left panel) <br /> and PF KREPA3-RNAi (right panel) cells KREPA3 expressed repressed 4 6 days, respectively, using cocktail MAbs <br /> as indicated. The 20S positive control described Fig. 2A, MAb66 used loading control. (B) Effect KREPA3 repression in <br /> BF KREPA3-RKO cells editosome sedimentation. Western analysis glycerol gradient fractions crude mt lysates BF KREPA3-RKO <br /> cells KREPA3 expressed repressed 2 4 days using cocktail MAbs indicated. The 20S positive control Fig. <br /> 2B MAb66 used control. The peak editosome fractions 19 23 (&#11011;20 40S region) pooled used vitro precleaved <br /> insertion (C) precleaved deletion (D) assays endonuclease assays specific A6 insertion (E) deletion (F) sites. Assay designations <br /> are Fig. 3. <br />  <br />  <br />  <br />  <br /> but KREPA1, KREL1, KREPA3 undetectable (Fig.                      insertion deletion site endonuclease activities pooled <br /> 4A, left panel). This contrast essentially unchanged          &#11011;20 40S fractions dramatically reduced 2 days <br /> levels KREPA1, KREPA2, KREL1 proteins                          undetectable 4 days KREPA3 repression, finding <br /> KREPA3 repression RNAi PFs (Fig. 4A, right panel). It                concordant loss editosomes (Fig. 4C, D, E, F). <br /> is likely difference editosome pro-              Therefore, KREPA3 structurally necessary editosome <br /> teins remaining KREPA3 repression differ-              stability integrity BFs loss KREPA3 results in <br /> ent methods used repress KREPA3 expression dif-                  complete loss editosomes, components, vitro <br /> ferences editosomes life cycle stages.                editing activities. <br /> Glycerol gradient sedimentation analysis cleared crude mt                  ZF mutation OB-fold deletion KREPA3 lethal and <br /> lysates BF KREPA3-RKO cells KREPA3                       blocks editing vivo. The KREPA3 ZF motifs OB-fold <br /> was expressed repressed 2 4 days revealed dramatic             domain critical KREPA3 function in <br /> loss editosomes. All tested editosome components                        vivo. The effects mutating ZFs deleting the <br /> (KREPA1, KREPA2, KREL1, KREPA6) substantially de-                      OB-fold domain cell growth examined generat- <br /> creased &#11011;20 40S region 2 days KREPA3                    ing transgenic BF cells exclusively express mutant <br /> repression small amounts components seen              KREPA3 proteins. The mutations replaced cysteines with <br /> upper fractions (Fig. 4B, middle panel). No editosome proteins             alanines C2H2 ZF motifs deleted OB-fold <br /> were detected &#11011;20S 40S region 4 days                    domain diagrammed Fig. 5A. WT mutant KREPA3 <br /> KREPA3 repression small amounts KREPA1                     alleles inserted &#65533;?&#65533;-tubulin locus BF KREPA3- <br /> KREPA2 gradient fractions (Fig. 4B,                  RKO cells, constitutively expressed. These alleles <br /> lower panel). Not surprisingly, precleaved insertion (TUTase           TAP-tagged discriminate tet-regulated <br /> and ligase), precleaved deletion (ExoUase ligase),             WT KREPA3 allele, KREPA3 Reg, rRNA locus to <br />  6946     GUO ET AL.                                                                                                     MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 5. Mutation ZFs deletion OB-fold KREPA3 results cell death loss vivo RNA editing. (A) Schematic <br /> representation WT mutant TAP-tagged KREPA3. The cysteines ZFs mutated alanines resulting in <br /> ZFm1-TAP (C53A/C56A), ZFm2-TAP (C183A/C186A), ZFm1&amp;2-TAP (C53A/C56A/C183A/C186A) truncation KREPA3 <br /> before OB-fold named &#9004;OB-TAP. (B) Cell growth presence constitutively expressed WT mutant TAP-tagged KREPA3 <br /> when KREPA3 Reg expressed (E) repressed (R). Constitutively expressed WT mutant TAP-tagged KREPA3 alleles were <br /> integrated &#65533;?&#65533;-tubulin locus BF KREPA3-RKO cells, expression KREPA3 Reg controlled adding withdrawing <br /> tet. (C) Western analysis KREPA3 Reg TAP-tagged KREPA3 allele expression using KREPA3 MAb. KREPA3 Reg expression was <br /> efficiently repressed withdrawal tet 3 days. (D) The relative abundance pre-edited, edited, never-edited mRNAs cells <br /> exclusively expressing WT mutant TAP-tagged KREPA3 alleles KREPA3 Reg repressed 3 days compared cells in <br /> which expressed detected real-time RT-PCR. The relative change amplicons pre-edited edited A6, COIII, and <br /> COII never-edited ND4 normalized 18S rRNA. Note log scale 1 represents difference, &#11022;1 increase, and <br /> &#65533;?1 decrease. <br />  VOL. 28, 2008                                          KREPA3 ZFs AND OB-FOLD ARE ESSENTIAL FOR RNA EDITING                        6947 <br />  <br />  <br /> allow purification proteins. The resulting cells      formed gradient fractions using panel editosome <br /> designated RKO-KREPA3WT-TAP, RKO-KREPA3ZFm1-TAP,                      MAbs similarly treated lysates KREPA3-RKO cells in <br /> RKO-KREPA3ZFm2-TAP, RKO-KREPA3ZFm1&amp;2-TAP,                         absence KREPA3 Reg expression serving control. A <br /> RKO-KREPA3&#9004;OB-TAP. All cells grew rate             portion KREPA3 WT-TAP ZFm2-TAP proteins <br /> in presence tet, i.e., KREPA3 Reg expressed.          incorporate &#11011;20-40S complexes, with <br /> Withdrawal tet resulted cessation cell growth      KREPA1 KREL1 editosome components when <br /> 4 days KREPA3 mutants (Fig. 5B). No live cells,        KREPA3 Reg repressed, contrast virtual lack of <br /> but cell debris, observed day 6, finding           editosome proteins region control cells (Fig. <br /> similar consequence repression KREPA3 expres-            6A). Thus, tagged WT ZFm2 proteins able <br /> sion BF KREPA3-RKO cells. Growth RKO-KREPA3                     restore presence &#11011;20S editosomes. Most the <br /> WT-TAP cells inhibited extent KREPA3                 tagged WT KREPA3 sedimented &#11011;5 10S region when <br /> Reg repression, cells continued slower growth rate with-        KREPA3 Reg expressed, shifted to <br /> out ultimate death (Fig. 5B). This presumably           larger S values expression untagged KREPA3 <br /> effect TAP tag expression KREPA3 Reg              eliminated, i.e., KREPA3 Reg repressed 3 days. <br /> without tag supported cell growth normal rate (Fig.          The levels KREPA1 KREL1 lower the <br /> 2B). Western analysis KREPA3 MAb revealed               cells exclusively expressing WT-TAP protein cells <br /> the withdrawal tet essentially eliminated expression        expressing untagged KREPA3 Reg (Fig. 6A). Thus, <br /> KREPA3 Reg allele 3 days (Fig. 5C). It revealed       C-terminal tag hampered ability protein effi- <br /> the level TAP-tagged KREPA3ZFm1 ZFm1&amp;2                     ciently remain incorporated &#11011;20S editosomes, untagged <br /> proteins lower levels WT ZFm2 pro-            KREPA3 appears compete tagged version in- <br /> teins similar numbers cells. The result observed       corporation 20S editosomes. This consistent the <br /> using PAP reagent specific protein A com-         partial inhibition cell growth vivo editing RKO <br /> ponent TAP tag (data shown), indicated          WT-TAP cells absence KREPA3 Reg expression. <br /> the ZF mutation effect recognition KREPA3             Parallel studies revealed tagged KREPA3 ZFm2 pro- <br /> the MAb. Therefore, ZFs OB-fold domain               tein primarily located smaller S value region the <br /> are necessary KREPA3 function vivo, mutations         gradient WT-TAP KREPA3 Reg expressed and <br /> these motifs lethal, similar effect KREPA3 re-          did shift larger S region KREPA3 Reg expression <br /> pression BF KREPA3-RKO cells.                                      repressed (Fig. 6A). Also, KREL1 shifted smaller S <br />    Exclusive expression ZF mutation OB-fold de-        value region gradient repression KREPA3 Reg <br /> letion (&#9004;OB) KREPA3 abolished editing vivo extent         expression, case WT tagged KREPA3 <br /> similar loss KREPA3 expression, revealed real-        (Fig. 6A). Considering expression ZFm2-TAP com- <br /> time PCR. Total RNA isolated parental BF                 pletely blocked growth vivo editing, result suggests <br /> KREPA3-RKO cells BF KREPA3-RKO cells expressing               ZFm2 protein involved protein and/or RNA <br /> TAP-tagged KREPA3 WT, ZFm2 &#9004;OB mutant alleles                 interaction fashion affects editosome integrity, well <br /> KREPA3 Reg expression repression 3 days. The abun-             vivo function, different WT version. <br /> dance pre-edited edited A6, COIII, COII never-            The effect tag and/or mutations ZFs on <br /> edited ND4 mRNAs assayed real-time PCR,                association KREPA3 editosome components <br /> relative mRNAs levels normalized 18S rRNA in-           presence absence untagged WT protein ex- <br /> ternal control (Fig. 5D). The relative mRNA abundance        plored purification tagged complexes. The <br /> normalized &#65533;?&#65533;-tubulin results (data shown).       tagged complexes immunoprecipitated IgG beads <br /> Repression KREPA3 Reg expression results exclusive        KREPA3-RKO derivative cells expressing TAP- <br /> expression TAP-tagged KREPA3ZFm2 KREPA3&#9004;OB es-                  tagged KREPA3 WT ZF mutant proteins KREPA3 <br /> sentially abolished vivo editing. The relative abundance        Reg expressed repressed 3 days. The complexes were <br /> edited mRNA decreased level parental       eluted TEV protease cleavage, eluates starting <br /> BF KREPA3-RKO cells absence KREPA3 Reg expres-              whole-cell lysates analyzed Western analysis the <br /> sion, i.e., ca. 95 99%, pre-edited mRNA levels        panel editosome MAbs. The studies clearly showed the <br /> were unchanged slightly increased (Fig. 5D). Exclusive expres-     tagged WT protein did associate editosomes in <br /> sion TAP-tagged WT KREPA3 protein, somewhat              cells expressing KREPA3 Reg did KREPA3 Reg <br /> inhibited growth, resulted 30 60% inhibition vivo       expression repressed, expressed similar <br /> editing (Fig. 5D). In cases, relative abundance   levels shown Western analysis whole-cell lysates (Fig. <br /> never-edited ND4 mRNA essentially unchanged. Thus,          7A). This consistent sedimentation results Fig. <br /> data indicate ZF motifs OB-fold domain neces-        6A. Compared WT-TAP cells KREPA3 Reg re- <br /> sary KREPA3 function RNA editing vivo.                 pressed, similar amounts tested editosome components <br />    The C-terminal TAP tag ZF mutations affect KREPA3              coprecipitated tagged KREPA3 proteins or <br /> editosome association. The effect mutation second           ZFs mutated, cells expressing KREPA3 Reg <br /> ZF association KREPA3 editosome          (Fig. 6A), indicated ZF disruption little effect <br /> estimated editosome sedimentation. Crude mt lysates           KREPA3-mediated protein interactions. These results also <br /> RKO-KREPA3 WT-TAP ZFm2-TAP cells presence                   suggested association editosome WT versus the <br /> or absence KREPA3 Reg expression subjected                 ZF mutants different protein and/or RNA. Extra <br /> glycerol gradient sedimentation. Western analysis per-            bands observed TEV eluates probed the <br />  6948      GUO ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />                                                                            required protein interactions editosome, suggest- <br />                                                                            ing ZFs function RNA binding RNA pro- <br />                                                                            cessing. However, exclude possibility ZFs <br />                                                                            involved unknown protein interactions the <br />                                                                            conformation entire editosome. <br />                                                                               Analysis glycerol gradient fractions containing primarily <br />                                                                            TAP-tagged KREPA3 (Fig. 6A B) showed that <br />                                                                            KREPA3 does editing enzymatic activities. The <br />                                                                            &#11011;5 10S &#11011;20 40S gradient fractions the <br />                                                                            RKO-KREPA3 WT-TAP cells KREPA3 Reg expression <br />                                                                            repressed (Fig. 6A) pooled subjected Western <br />                                                                            analysis vitro editing assays (Fig. 6B). Western analysis <br />                                                                            showed tagged protein abundant protein <br />                                                                            detected &#11011;5 10S fractions, &#11011;20 40S <br />                                                                            fractions contained KREPA1 KREL1. Virtually none <br />                                                                            specific editing enzymatic activities detected the <br />                                                                            pooled &#11011;5 10S fractions containing WT KREPA3-TAP <br />                                                                            protein. The U-removal products &#11011;5 10S fractions are <br />                                                                            probably nonspecific cleavage nucleases <br />                                                                            detected &#11011;5 10S fractions from <br />                                                                            KREPA3-RKO cells absence KREPA3 Reg expres- <br />                                                                            sion (data shown). As expected, pooled &#11011;20 40S <br />                                                                            fractions exhibited vitro editing activities. Hence, <br />                                                                            KREPA3 does appear catalyze specific enzymatic <br />                                                                            steps editing. <br />                                                                               The effects ZF mutation vitro precleaved U-inser- <br />                                                                            tion U-deletion editing (Fig. 7B C) insertion or <br />                                                                            deletion site endonuclease cleavage (Fig. 7D E) were <br />                                                                            assayed using TEV eluates shown Fig. 7A. All the <br />                                                                            purified editing complexes enzymatic activities consistent <br />                                                                            level editosome proteins copurified TAP- <br />                                                                            tagged KREPA3 WT ZF mutants. In addition, mutation of <br />                                                                            KREPA3 ZFs direct effect activities. <br />    FIG. 6. Effect KREPA3 ZF disruption editosome sedimen-            Stronger editing activities observed WT-TAP con- <br /> tation. (A) Western analysis glycerol gradient fractions crude mt    taining complexes KREPA3 Reg repressed than <br /> lysates BF RKO-KREPA3 WT-TAP ZFm2-TAP cells which <br /> KREPA3 Reg expressed (E) repressed (R) 3 days. Western <br />                                                                            expressed, incorporation of <br /> analysis performed antibodies KREPA1, KREL1,          WT-TAP protein editosomes KREPA3 repression <br /> KREPA3. Similarly prepared lysates BF KREPA3-RKO pa-              (Fig. 6A 7A). In contrast, TEV eluates cells <br /> rental cells KREPA3 expression repressed 3 days serves     ZF mutants presence KREPA3 Reg expres- <br /> control. Compared control, exclusive expression           sion slightly decreased level U-insertion activity upon <br /> KREPA3 WT-TAP ZFm2-TAP rescued presence &#11011;20 to <br /> 40S editosomes various amounts. Also, KREPA3 WT-TAP             KREPA3 Reg repression, corresponded reduc- <br /> and ZFm2-TAP proteins displayed different distributions glyc-       tion KREPA1 protein (Fig. 7A) representing insertion <br /> erol gradients. (B) Western analysis editing assays pooled      subcomplex. Overall, editosomes rescued tagged <br /> &#11011;5 10S &#11011;20 40S gradient fractions WT-TAP               KREPA3 ZF mutants editing activities similar to <br /> KREPA3 Reg repression panel A. Western analysis performed <br /> with MAb mixture KREPA1, KREPA3, KREL1. The <br />                                                                            rescued tagged KREPA3 WT protein, indicating that <br /> editing assays designations described Fig. 3. The substantial   disruption ZFs effect enzymatic activities <br /> amount KREPA3-WT TAP protein pooled &#11011;5 10S frac-              editing vitro. Thus, KREPA3 ZFs critical for <br /> tions does catalyze editing activities.                     vitro enzymatic activities protein interactions the <br />                                                                            editosome essential survival cells editing <br />                                                                            vivo. <br /> KREPA3 MAb, protease degradation products                      KREPA3 OB-fold deletion abolishes interaction the <br /> and untagged KREA3 Reg (labeled asterisk;                editosome. The effect deletion OB-fold KREPA3 <br /> appeared KREPA3 Reg expression, size similar            editosome examined. Tagged KREPA3&#9004;OB pro- <br /> KREPA3). In addition, purification tagged complexes            tein purified lysates BF RKO-KREPA3&#9004;OB-TAP <br /> RKO WT-TAP ZFm2-TAP cells using TAP procedure                      cells KREPA3 Reg expressed repressed 3 <br /> and identification proteins liquid chromatography-tan-           days using IgG beads, protein eluted cleavage <br /> dem mass spectrometry (LC-MS/MS) showed                   TEV protease. Western analysis performed the <br /> editosome proteins copurified KREPA3 ZFm2-TAP                    whole-cell lysates (Fig. 8A) TEV eluates (Fig. 8B) <br /> similar observed WT-TAP (data shown).                 using panel editosome MAbs. KREPA1 KREPA6 <br /> Thus, data revealed ZFs KREPA3                  detected whole-cell lysates KREPA3&#9004;OB-TAP <br />  VOL. 28, 2008                                            KREPA3 ZFs AND OB-FOLD ARE ESSENTIAL FOR RNA EDITING                               6949 <br />  <br />  <br />  <br />  <br />   FIG. 7. Effect KREPA3 ZF disruption interaction editosome editing activities vitro. (A) Western analysis cell lysates <br /> and complexes copurified TAP-tagged WT ZF mutant KREPA3 proteins presence absence KREPA3 Reg. Lysates equal <br /> numbers BF KREPA3-RKO, RKO-KREPA3 WT-TAP, ZFm1-TAP, ZFm2-TAP, ZFm1&amp;2 cells KREPA3 Reg expressed (E) or <br /> repressed (R) 3 days purified IgG beads eluted TEV cleavage. The TEV eluates probed antibodies against <br /> KREPA1, KREPA2, KREPA3, KREL1, KREPA6, cell lysates probed antibodies KREPA3, KREPA6, PGK, <br /> the serving control. The 20S positive control Fig. 2B. Roughly similar amounts editosome proteins purified the <br /> TAP-tagged KREPA3 ZF mutants KREPA3 expressed not. In contrast, significant amounts editosome proteins purified <br /> with WT-TAP protein KREPA3 Reg repressed. The TEV eluates used vitro precleaved insertion (B) deletion <br /> (C) assays insertion (D) deletion (E) endonuclease cleavage assays. The activity assays designations Fig. 3. The tagged <br /> KREPA3 ZF mutant complexes contain vitro editing activities. <br />  <br />  <br />  <br />  <br /> cells following KREPA3 Reg repression TEV eluates                    analysis using MAb cocktail (Fig. 8C). KREPA3&#9004;OB-TAP <br /> whether KREPA3 Reg expressed (Fig. 8A B).                   sedimented smaller S values editosome, indicating <br /> Also, editosome components purified                   associated editosome found <br /> KREPA3&#9004;OB-TAP TEV eluates LC-MS/MS (data                            gradient. This suggested KREPA3&#9004;OB- <br /> shown). In addition, crude mt lysate cells             TAP aggregate interact proteins. Thus, ex- <br /> presence KREPA3 Reg expression sedimented glyc-                  clusive expression KREPA3&#9004;OB incapable of <br /> erol gradients, fractions subjected Western                rescuing cell death incapable recovering editosomes <br />  6950     GUO ET AL.                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 8. The KREPA3 OB-fold domain necessary editosome integrity. Western analysis purified tagged KREPA3&#9004;OB protein BF <br /> RKO-KREPA3&#9004;OB-TAP cells KREPA3 Reg expressed (E) repressed (R) 3 days using IgG beads eluted TEV cleavage. (A <br /> and B) Whole-cell lysates (A) TEV eluates (B) probed panel editing antibodies. KREPA1 KREPA6 undetectable <br /> in total cell lysates repression KREPA3 Reg expression. PGK used loading control. After purification, editosome <br /> components detected TEV eluates. (C) Western analysis gradient fractions crude mt lysates from. BF RKO-KREPA3&#9004;OB-TAP <br /> cells presence KREPA3 Reg expression. Western analysis performed panel editing MAbs. KREPA3&#9004;OB-TAP did not <br /> cosediment editosome. <br />  <br />  <br />  <br /> following repression KREPA3 Reg expression. This sug-             deletion endonuclease activities. However, KREPA3 ZF <br /> gests OB-fold, function nucleotide bind-       mutations resulted editosome destabilization subcom- <br /> ing, critical protein interactions editosome.          plexes accumulated mutation ZF2. KREPA3 that <br />                                                                      lacks OB-fold incorporated editosomes. Thus, <br />                                                                      ZFs essential vivo important editosome <br />                         DISCUSSION <br />                                                                      stability, OB-fold essential vivo critical for <br />    We KREPA3 essential RNA editing             editosome integrity. <br /> and editosome integrity (Fig. 3A B) U-              The results presented indicate while <br /> addition, U-removal, RNA ligase catalytic steps editing     KREPA3 affect activity does directly catalyze <br /> T. brucei (Fig. 3C F). RNAi-mediated knockdown                 3&#11032; TUTase, 3&#11032; exoUase, ligase steps editing since <br /> KREPA3 expression inhibited growth PFs, observed           subcomplexes devoid KREPA3 catalyze activities (30). <br /> previously (8, 30), resulted dramatic reduction      It unlikely KREPA3 editing endonuclease ac- <br /> abundance edited, pre-edited mRNAs. KREPA3                tivity itself, reports indicate recombinant <br /> knockdown resulted partial disruption 20S editosomes           KREPA3 nuclease activity (8, 36). KREN1, KREN2, and <br /> into insertion deletion subcomplexes vitro         KREN3 endonucleases shown mutation residues in <br /> U-insertion/ligation U-deletion/ligation activities     RNase III domains block editing vivo elim- <br /> can separately immunoprecipitated. Editosome endonucle-           inate endonucleolytic cleavage editosomes vitro (10, 11, <br /> ase activities reduced detected 20S edi-        56). Also, high concentrations KREN1-CBP fusion protein <br /> tosomes remained (Fig. 3E F). tet-dependent tran-           isolated insect cell expression showed vitro specific <br /> scriptional knockdown single remaining KREPA3 gene           U-deletion endonuclease activity presence of <br /> BFs resulted complete loss editosomes, loss          KREPA3 (26). In addition, endonuclease activities remain <br /> edited, unedited, mRNAs, consequently loss         following knockdown KREPA3 expression detected <br /> vitro editing activities. Exclusive expression TAP-tagged         &#11011;20 40S complexes, activity levels propor- <br /> KREPA3 mutated ZFs OB-fold                tional editosome KREPA3 levels (determined <br /> deleted lethal, i.e., did rescue cell growth inac-      phosphorimager quantitation normalized KREPA1 and <br /> tivation WT KREPA3 expression. Exclusive expression            KREPA2 [data shown]). Similarly, endonuclease activity <br /> TAP-tagged WT KREPA3 protein rescued cell growth, al-                detected 20 40S editosomes studies that <br /> though inhibited level. RNA editing somewhat in-           knock expression editosome components and <br /> hibited tagged WT cells dramatically inhibited     shift editosomes lower S values (19, 29, 48). In contrast, <br /> cells exclusively expressing mutated truncated KREPA3             mutations KREPA3 Zn fingers affect editosome structure <br /> proteins. Tagged KREPA3 proteins ZFs                and/or stability discussed below, remaining edito- <br /> mutated incorporated 20S editosomes            somes retain endonuclease editing catalytic ac- <br /> vitro U-insertion/ligation, U-deletion/ligation, insertion       tivities. Nevertheless, KREPA3 affect, essen- <br />  VOL. 28, 2008                                         KREPA3 ZFs AND OB-FOLD ARE ESSENTIAL FOR RNA EDITING                        6951 <br />  <br />  <br /> tial for, editing endonuclease activities. The &#11001;1 U insertion    critical stability insertion deletion subcom- <br /> and &#11002;3 U deletion products increased KREPA3                 plexes, respectively (14, 19, 37). Knockdown KREPA1 or <br /> repression (Fig. 3C D), implying KREPA3               KREPA2 does result dramatic loss editosomes but <br /> enhance and/or coordinate activities editing en-          shift lower S values consistent loss the <br /> zymes, similar KREPA1 KREPA2 (15, 47). These re-              insertion deletion subcomplexes. Mutation ZFs the <br /> sults, reconstitution endonuclease activity        KREPA2 (TbMP63) ortholog Leishmania (LC-4) resulted in <br /> upon supplementation KREPA3-depleted extracts re-            disruption editosome (25), implying complex stabilizing <br /> combinant KREPA3 reported Law et al. (30), suggests          role similar ZFs KREPA3. The partial KREPA3 <br /> KREPA3 enhances needed endonuclease activity ei-           knockdown PFs resulted KREPA1 insertion subcomplexes <br /> ther indirectly effect editosome structure      smaller S value KREPA2 deletion subcomplexes (Fig. <br /> direct interaction KREN endonucleases. The addition         3B). Correspondingly, KREPA1 RNAi knockdown results in <br /> of recombinant KREPA3 restored endonucle-             larger (37) subcomplexes, KREPA2 RNAi knockdowns <br /> ase editing activities KREPA3-depleted extracts assem-         results smaller (&#11011;5S) subcomplexes (19). These data together <br /> bling residual subcomplexes &#11011;20S editing complexes.         suggest deletion subcomplex stably associated <br />    KREPA3 essential editosome structure stability,        proteins insertion subcomplex, possibly with <br /> since editosomes disrupted KREPA3 RNAi knockdown              proteins KREPB4 KREPB5 and/or the <br /> in PFs lost BFs transcription inactivation. This         endonuclease subcomplex. <br /> difference likely reflects greater reduction KREPA3                The KREPA3, KREPA4, KREPA6, possibly KREPA5 <br /> mRNA protein levels latter. The quantitative PCR          proteins appear form structural core editosome that <br /> studies reveal somewhat greater knockdown KREPA3             bridges insertion deletion subcomplexes associa- <br /> the BFs PFs, KREPA3 protein faintly detectable        tion KREPA1 KREPA2 (Fig. 1) (29, 46, 47, 54). This <br /> PF BF Western blots (Fig. 3B versus 4B Fig. 4A).         core appears link endonuclease subcomplex via <br /> In addition, reduction edited mRNA generally              KREPA3 KREPB5 interaction. The editosome disruption <br /> greater BF PF knockdowns. Edited cyB mRNA            results expression KREPA3 C-terminal tag <br /> undetectable BFs, edited COII mRNA reduced                illustrates KREPA3 located inner of <br /> much BFs PFs. This probably reflects             editosome tag interfere normal <br /> developmental regulation editing little    editosome structure (Fig. 6A). The shift tagged WT <br /> of edited mRNAs BFs (5, 17). Although studies        KREPA3 higher S regions gradient repression <br /> date revealed sedimentation composition differ-           untagged KREPA3 indicates integrate edi- <br /> ences BF PF editosomes, composition             tosomes competitive disadvantage the <br /> editosome differences life cycle stages        untagged KREPA3. The TAP tag inhibited cell growth (Fig. <br /> excluded. Both ZFs important &#11011;20S editosome stabil-          5B), replacement short c-myc tag resulted cells <br /> ity. KREPA3 mutated ZFs gets incorporated edito-           grew normal rate (data shown), suggesting that <br /> somes, abundance complexes diminished            effect tag structure. Despite effects, <br /> and subcomplexes generated. Exclusive expression              tagged KREPA3 mutated ZFs integrate <br /> KREPA3 OB-fold results virtual absence         editosomes, elimination OB-fold only <br /> editosomes. This indicates proteins incorpo-       abolished integration interaction with <br /> rated editosomes degraded.                                  KREPA1, KREPA2, KREPA6 (Fig. 8A B). Indeed, <br />    The dramatic loss editosomes KREPA3 knockdown             editosome proteins LC-MS/MS asso- <br /> reported mirrors similar results knockdown KREPA4,        ciated purified TAP-tagged KREPA3&#9004;OB (data not <br /> KREPA6, KREPB4, KREPB5 (4, 29, 46, 54, 58) suggests           shown). Thus, OB-fold critical KREPA3&#8217;s interaction <br /> that network interactions proteins    editosome proteins. As reported elsewhere, <br /> is required editosome stability assembly. The loss   KREPA3, KREPA4, KREPA6, less-studied <br /> of editosome proteins knockdown       KREPA5, form network interactions involve OB- <br /> does appear to changes mRNA levels          folds (A. Schnaufer, unpublished data). Hence, illustrated <br /> proteins (58; study) instability degra-    effect expression KREPA3 OB-fold <br /> dation editosome proteins incorporated          deleted, OB-folds proteins appear function in <br /> editosomes. As reported elsewhere, yeast two-<span id='am-61' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-62'  ></span><span id='am-63' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>hybrid</span> analyses         protein-protein interaction. However, does preclude <br /> showed direct interaction KREPA3 KREPA2,                 possibility OB-fold domain KREPA3 the <br /> KREPA6, KREPB5 showed direct interaction be-            KREPA proteins affect substrate recognition, <br /> tween KREPA6 KREPA1, KREPA2, KREPA3,                         binding, translocation, suggested for <br /> KREPA4 (A. Schnaufer, unpublished data). In addition,       KREPA1 (TbMP81) KREPA2 (TbMP63) (47, 62). <br /> only detect KREPA6 copurifying &#11011;5S TAP-tagged                      Editosomes mutated KREPA3 ZFs, stable, <br /> KREPA3 subcomplexes using LC-MS/MS analysis, indicating            capable catalyzing endonuclease, 3&#11032; TUTase, 3&#11032; <br /> high-affinity interaction KREPA3 KREPA6 (data            exoUase, RNA ligase steps editing vitro. Overexpres- <br /> not shown). Previous findings showed KRET2 KREL2            sion TAP-tagged ZF mutants KREPA1, KREPA2, <br /> copurify tagged KREPA1 KREX2 KREL1 copurify             KREPA3 PFs did exhibit dominant-negative <br /> with tagged KREPA2 insertion deletion editosome sub-          effect (data shown). Nevertheless, KREPA3 ZF muta- <br /> complexes, respectively (47). KREPA1 KREPA2, like                tions block editing vivo lethal exclusively <br /> KREPA3, ZFs C-terminal OB-fold            expressed. Since KREPA3 binds RNA (8), likely possibility is <br />  6952       GUO ET AL.                                                                                                                                 MOL. CELL. BIOL. <br />  <br />  <br /> that ZFs function RNA recognition binding. C2H2                                   specific U-deletion RNA editing, REL2 RNA ligase specific for <br />                                                                                              U-insertion editing? J. Biol. Chem. 278:27570&#8211;27574. <br /> ZFs general nonspecifically bind dsRNA <span id='am-28' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-29' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32'  ></span>helix</span> back-                          17.   Hannaert, V., F. Bringaud, F. R. Opperdoes, P. A. Michels. 2003. <br /> bone specifically recognize individual bases bulge                              Evolution energy metabolism compartmentation <span id='am-19' about='obo:NCBITaxon_5653' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33682 obo:NCBITaxon_2759'><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Kinetoplastea" datatype="xsd:string"></span><span id='am-21'  ></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Protomonadida" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="kinetoplastids" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="Kinetoplastida" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="kinetoplasts" datatype="xsd:string"></span>Kinetoplastida</span>. <br /> out structurally rigid element (51). Hence, KREPA3                                  Kinetoplastid Biol. Dis. 2:11&#8211;40. <br />                                                                                        18.   Harris, M. E., D. R. Moore, S. L. Hajduk. 1990. Addition uridines to <br /> ZFs bind gRNA/pre-mRNA duplex sequence-                                         edited RNAs trypanosome mitochondria occurs independently tran- <br /> nonspecific manner conserved secondary structure.                                scription. J. Biol. Chem. 265:11368&#8211;11376. <br /> They function editing processivity. The roles                       19.   Huang, C. E., S. F. O&#8217;Hearn, B. Sollner-Webb. 2002. Assembly and <br />                                                                                              function RNA editing complex Trypanosoma brucei requires band <br /> two ZFs differ mutation ZF greater                              III protein. Mol. Cell. Biol. 22:3194&#8211;3203. <br /> effect the tagged protein editosome                               20.   Igo, R. P., Jr., S. S. Palazzo, M. L. K. Burgess, A. K. Panigrahi, K. <br />                                                                                              Stuart. 2000. Uridylate addition RNA ligation contribute speci- <br /> stability, mutation inhibited growth.                                 ficity kinteoplastid insertion RNA editing. Mol. Cell. Biol. 20:8447&#8211;8457. <br /> They function ZFs KREPA1                                 21.   Igo, R. P., Jr., D. S. Weston, N. L. Ernst, A. K. Panigrahi, R. Salavati, and <br /> KREPA2 total U1-like Zn fingers                                K. Stuart. 2002. Role uridylate-specific exoribonuclease activity in <br />                                                                                              Trypanosoma brucei RNA editing. Eukaryot. Cell 1:112&#8211;118. <br /> KREN KREPB proteins shuttle substrate RNA                              22.   Iuchi, S. 2001. Three classes C2H2 zinc finger proteins. Cell Mol. Life Sci. <br /> among editosome catalytic sites accomplish editing                                 58:625&#8211;635. <br /> multiple sites, including insertion deletion sites.                           23.   Jensen, B. C., C. T. Kifer, D. L. Brekken, A. C. Randall, Q. Wang, B. L. <br />                                                                                              Drees, M. Parsons. 2006. Characterization protein kinase CK2 from <br />                                                                                              Trypanosoma brucei. Mol. Biochem. Parasitol. 151:28&#8211;40. <br />                           ACKNOWLEDGMENTS                                              24.   Kable, M. L., S. D. Seiwert, S. Heidmann, K. Stuart. 1996. RNA editing: <br />                                                                                              mechanism gRNA-specified uridylate insertion precursor mRNA. <br />   This study supported NIH grant AI014012 K.D.S. Re-                               Science 273:1189&#8211;1195. <br /> search conducted using equipment possible Economic                     25.   Kang, X., A. M. Falick, R. E. Nelson, G. Gao, K. Rogers, R. Aphasizhev, and <br /> Development Administration-U.S. Department Commerce                               L. Simpson. 2004. Disruption zinc finger motifs Leishmania <br /> M. J. Murdock Charitable Trust.                                                              tarentolae LC-4 (&#11005;TbMP63) L-complex editing protein affects stability of <br />   We thank J. Carnes help real-time PCR, A. Schnaufer                           L-complex. J. Biol. Chem. 279:3893&#8211;3899. <br /> advice cell line construction, M. Parsons MAb66. We                    26.   Kang, X., G. Gao, K. Rogers, A. M. Falick, S. Zhou, L. Simpson. 2006. <br />                                                                                              Reconstitution full-round uridine-deletion RNA editing re- <br /> thank members Stuart laboratory extensive helpful                             combinant proteins. Proc. Natl. Acad. Sci. USA 103:13944&#8211;13949. <br /> discussion.                                                                            27.   Kang, X., K. Rogers, G. Gao, A. M. Falick, S. Zhou, L. Simpson. 2005. <br />                                                                                              Reconstitution uridine-deletion precleaved RNA editing recom- <br />                                  REFERENCES                                                  binant enzymes. Proc. Natl. Acad. Sci. USA 102:1017&#8211;1022. <br />  1. Agrawal, V., K. V. Kishan. 2003. OB-fold: growing bigger functional       28.   Law, J. A., C. E. Huang, S. F. O&#8217;Hearn, B. Sollner-Webb. 2005. In <br />     consistency. Curr. Protein Pept. Sci. 4:195&#8211;206.                                         Trypanosoma brucei RNA editing, band II enables recognition specifically at <br />  2. Aphasizhev, R., I. Aphasizheva, R. E. Nelson, L. Simpson. 2003. A                    step U insertion cycle. Mol. Cell. Biol. 25:2785&#8211;2794. <br />     100-kD complex RNA-binding proteins mitochondria Leish-             29.   Law, J. A., S. O&#8217;Hearn, B. Sollner-Webb. 2007. In Trypanosoma brucei <br />     mania tarentolae catalyzes RNA annealing interacts RNA                  RNA editing, TbMP18 (band VII) critical editosome integrity for <br />     editing components. RNA 9:62&#8211;76.                                                         insertional deletional cleavages. Mol. Cell. Biol. 27:777&#8211;787. <br />  3. Aphasizhev, R., I. Aphasizheva, L. Simpson. 2003. A tale TUTases.       30.   Law, J. A., S. F. O&#8217;Hearn, B. Sollner-Webb. 2008. Trypanosoma brucei <br />     Proc. Natl. Acad. Sci. USA 100:10617&#8211;10622.                                              RNA editing protein TbMP42 (band VI) crucial endonucleolytic <br />  4. Babbarwal, V. K., M. Fleck, N. L. Ernst, A. Schnaufer, K. D. Stuart.                 cleavages subsequent steps U-deletion U-insertion. RNA. <br />     2007. An essential role KREPB4 RNA editing structural integrity                14:1187&#8211;1200. <br />     editosome Trypanosoma brucei. RNA 13:737&#8211;744.                            31.   Lu, D., M. A. Searles, A. Klug. 2003. Crystal structure zinc-finger- <br />  5. Besteiro, S., M. P. Barrett, L. Riviere, F. Bringaud. 2005. Energy gen-              RNA complex reveals modes molecular recognition. Nature 426:96&#8211; <br />     eration insect stages Trypanosoma brucei: metabolism flux. Trends               100. <br />     Parasitol. 21:185&#8211;191.                                                             32.   Madison-Antenucci, S., J. Grams, S. L. Hajduk. 2002. Editing machines: <br />  6. Blum, B., N. Bakalara, L. Simpson. 1990. A model RNA editing                  complexities trypanosome RNA editing. Cell 108:435&#8211;438. <br />     kinetoplastid mitochondria: &#8220;guide&#65533;? RNA molecules transcribed max-            33.   Miller, M. M., K. Halbig, J. Cruz-Reyes, L. K. Read. 2006. RBP16 <br />     icircle DNA provide edited information. Cell 60:189&#8211;198.                             stimulates trypanosome RNA editing vitro early step editing <br />  7. Brayer, K. J., D. J. Segal. 2008. Keep fingers DNA: protein-             reaction. RNA 12:1292&#8211;1303. <br />     protein interactions mediated C2H2 zinc finger domains. Cell Biochem.           34.   Muller, U. F., L. Lambert, H. U. Goringer. 2001. Annealing RNA <br />     Biophys. 50:111&#8211;131.                                                                     editing substrates facilitated guide RNA-binding protein gBP21. EMBO <br />  8. Brecht, M., M. Niemann, E. Schlu    &#168;ter, U. F. Mu  &#168;ller, K. Stuart, H. U.          J. 20:1394&#8211;1404. <br />     Go&#168;ringer. 2005. TbMP42, protein component RNA editing complex            35.   Murzin, A. G. 1993. OB(oligonucleotide/oligosaccharide binding)-fold: com- <br />     African trypanosomes endo-exoribonuclease activity. Mol. Cell 17:                 mon structural functional solution non-homologous sequences. <br />     621&#8211;630.                                                                                 EMBO J. 12:861&#8211;867. <br />  9. Carnes, J., K. Stuart. 2007. Working together: RNA editing ma-             36.   Niemann, M., M. Brecht, E. Schluter, K. Weitzel, M. Zacharias, H. U. <br />     chinery Trypanosoma brucei, p. 143&#8211;164. In H. U. Go      &#168;ringer (ed.), RNA           Goringer. 2008. TbMP42 structure-sensitive ribonuclease likely <br />     editing, vol. 20. Springer Verlag, Heidelberg, Germany.                                  follows metal ion catalysis mechanism. Nucleic Acids Res. 36:4465&#8211;4473. <br /> 10. Carnes, J., J. R. Trotter, N. L. Ernst, A. G. Steinberg, K. Stuart. 2005.      37.   O&#8217;Hearn, S., C. E. Huang, M. Hemann, A. Zhelonkina, B. Sollner-Webb. <br />     An essential RNase III insertion editing endonuclease Trypanosoma bru-                2003. Trypanosoma brucei RNA editing complex: band II structurally <br />     cei. Proc. Natl. Acad. Sci. USA 102:16614&#8211;16619.                                         critical maintains band V ligase, nonessential. Mol. Cell. Biol. <br /> 11. Carnes, J., J. R. Trotter, A. Peltan, M. Fleck, K. Stuart. 2008. RNA                 23:7909&#8211;7919. <br />     editing Trypanosoma brucei requires different editosomes. Mol. Cell.      38.   Panigrahi, A. K., N. L. Ernst, G. J. Domingo, M. Fleck, R. Salavati, K. D. <br />     Biol. 28:122&#8211;130.                                                                        Stuart. 2006. Compositionally functionally distinct editosomes in <br /> 12. Cruz-Reyes, J., B. Sollner-Webb. 1996. Trypanosome U-deletional RNA                  Trypanosoma brucei. RNA 12:1038&#8211;1049. <br />     editing involves guide RNA-directed endonuclease cleavage, terminal U              39.   Panigrahi, A. K., A. Schnaufer, N. Carmean, R. P. Igo, Jr., S. P. Gygi, N. L. <br />     exonuclease, RNA ligase activities. Proc. Natl. Acad. Sci. USA 93:8901&#8211;              Ernst, S. S. Palazzo, D. S. Weston, R. Aebersold, R. Salavati, K. D. <br />     8906.                                                                                    Stuart. 2001. Four related proteins Trypanosoma brucei RNA editing <br /> 13. Cruz-Reyes, J., A. Zhelonkina, L. Rusche&#180;, B. Sollner-Webb. 2001. Try-               complex. Mol. Cell. Biol. 21:6833&#8211;6840. <br />     panosome RNA editing: simple guide RNA features enhance U deletion                 40.   Panigrahi, A. K., A. Schnaufer, N. L. Ernst, B. Wang, N. Carmean, R. <br />     100-fold. Mol. Cell. Biol. 21:884&#8211;892.                                                   Salavati, K. Stuart. 2003. Identification novel components of <br /> 14. Drozdz, M., S. S. Palazzo, R. Salavati, J. O&#8217;Rear, C. Clayton, K. Stuart.            Trypanosoma brucei editosomes. RNA 9:484&#8211;492. <br />     2002. TbMP81 required RNA editing Trypanosoma brucei. EMBO J.            41.   Parker, H. L., T. Hill, K. Alexander, N. B. Murphy, W. R. Fish, M. <br />     21:1791&#8211;1799.                                                                            Parsons. 1995. Three genes isozymes: gene conversion the <br /> 15. Ernst, N. L., B. Panicucci, R. P. Igo, Jr., A. K. Panigrahi, R. Salavati, K.         compartmentalization expression phosphoglycerate kinases of <br />     Stuart. 2003. TbMP57 3&#11032; terminal uridylyl transferase (TUTase)               Trypanosoma (<span id='am-51' about='obo:NCBITaxon_47569' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_5690 obo:NCBITaxon_2759 obo:NCBITaxon_5653'><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Nannomonas" datatype="xsd:string"></span>Nannomonas</span>) congolense. Mol. Biochem. Parasitol. 69:269&#8211; <br />     Trypanosoma brucei editosome. Mol. Cell 11:1525&#8211;1536.                                    279. <br /> 16. Gao, G., L. Simpson. 2003. Is Trypanosoma brucei REL1 RNA ligase           42.   Pelletier, M., L. K. Read. 2003. RBP16 multifunctional gene regu- <br />  VOL. 28, 2008                                                         KREPA3 ZFs AND OB-FOLD ARE ESSENTIAL FOR RNA EDITING                                           6953 <br />  <br />       latory protein involved editing stabilization specific mitochondrial          management: RNA editing trypanosomes. Trends Biochem. Sci. 30:97&#8211; <br />       mRNAs Trypanosoma brucei. RNA 9:457&#8211;468.                                             105. <br /> 43.   Pfaffl, M. W. 2001. A new mathematical model relative quantification       54.   Tarun, S. Z., Jr., A. Schnaufer, N. L. Ernst, R. Proff, J. Deng, W. Hol, and <br />       real-time RT-PCR. Nucleic Acids Res. 29:e45.                                            K. Stuart. 2008. KREPA6 RNA binding protein essential editosome <br /> 44.   Ramakers, C., J. M. Ruijter, R. H. L. Deprez, A. F. M. Moorman. 2003.               integrity survival Trypanosoma brucei. RNA 14:347&#8211;358. <br />       Assumption-free analysis quantitative real-time polymerase chain reaction      55.   Theobald, D. L., R. M. Mitton-Fry, D. S. Wuttke. 2003. Nucleic acid <br />       (PCR) data. Neurosci. Lett. 339:62&#8211;66.                                                  recognition OB-fold proteins. Annu. Rev. Biophys. Biomol. Struct. 32: <br /> 45.   Rogers, K., G. Gao, L. Simpson. 2007. Uridylate-specific 3&#11032;-5&#11032; exoribo-             115&#8211;133. <br />       nucleases involved uridylate-deletion RNA editing trypanosomatid            56.   Trotter, J. R., N. L. Ernst, J. Carnes, B. Panicucci, K. Stuart. 2005. A <br />       mitochondria. J. Biol. Chem. 282:29073&#8211;29080.                                           deletion site editing endonuclease Trypanosoma brucei. Mol. Cell 20:403&#8211; <br /> 46.   Salavati, R., N. L. Ernst, J. O&#8217;Rear, T. Gilliam, S. Tarun, Jr., K. Stuart.         412. <br />       2006. KREPA4, RNA binding protein essential editosome integrity            57.   Vanhamme, L., D. Perez-Morga, C. Marchal, D. Speijer, L. Lambert, M. <br />       survival Trypanosoma brucei. RNA 12:819&#8211;831.                                     Geuskens, S. Alexandre, N. <span id='am-46' about='obo:NCBITaxon_1113443' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_40096 obo:NCBITaxon_40092 obo:NCBITaxon_33208 obo:NCBITaxon_40093 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-48'  ></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Isma" datatype="xsd:string"></span>Isma</span>&#305;&#168;li, U. Go  &#168;ringer, R. Benne, E. Pays. <br />                                                                                               1998. Trypanosoma brucei TBRGG1, mitochondrial oligo(U)-binding pro- <br /> 47.   Schnaufer, A., N. Ernst, J. O&#8217;Rear, R. Salavati, K. Stuart. 2003. Sepa- <br />                                                                                               tein colocalizes vitro RNA editing activity. J. Biol. Chem. <br />       rate insertion deletion sub-complexes Trypanosoma brucei RNA <br />                                                                                               273:21825&#8211;21833. <br />       editing complex. Mol. Cell 12:307&#8211;319. <br />                                                                                         58.   Wang, B., N. L. Ernst, S. S. Palazzo, A. K. Panigrahi, R. Salavati, K. <br /> 48.   Schnaufer, A., A. K. Panigrahi, B. Panicucci, R. P. Igo, Jr., R. Salavati, and <br />                                                                                               Stuart. 2003. TbMP44 essential RNA editing structural integrity of <br />       K. Stuart. 2001. An RNA ligase essential RNA editing survival the <br />                                                                                               editosome Trypanosoma brucei. Eukaryot. Cell 2:578&#8211;587. <br />       bloodstream form Trypanosoma brucei. Science 291:2159&#8211;2162.                    59.   Wang, Z., J. C. Morris, M. E. Drew, P. T. Englund. 2000. Inhibition of <br /> 49.   Seiwert, S. D., S. Heidmann, K. Stuart. 1996. Direct visualization               Trypanosoma brucei gene expression RNA interference using integrat- <br />       uridylate deletion vitro suggests mechanism kinetoplastid RNA                  able vector opposing T7 promoters. J. Biol. Chem. 275:40174&#8211;40179. <br />       editing. Cell 84:831&#8211;841.                                                         60.   Wirtz, E., S. Leal, C. Ochatt, G. A. M. Cross. 1999. A tightly regulated <br /> 50.   Simpson, L., R. Aphasizhev, G. Gao, X. Kang. 2004. Mitochondrial                    inducible expression conditional gene knockouts dominant- <br />       proteins complexes Leishmania Trypanosoma involved U-                     negative genetics Trypanosoma brucei. Mol. Biochem. Parasitol. 99:89&#8211; <br />       insertion/deletion RNA editing. RNA. 10:159&#8211;170.                                        101. <br /> 51.   Stefl, R., L. Skrisovska, F. H. Allain. 2005. RNA sequence- shape-        61.   Wolfe, S. A., L. Nekludova, C. O. Pabo. 2000. DNA recognition by <br />       dependent recognition proteins ribonucleoprotein particle. EMBO               Cys2His2 zinc finger proteins. Annu. Rev. Biophys. Biomol. Struct. 29:183&#8211; <br />       Rep. 6:33&#8211;38.                                                                           212. <br /> 52.   Stuart, K., T. E. Allen, S. Heidmann, S. D. Seiwert. 1997. RNA editing        62.   Worthey, E. A., A. Schnaufer, I. S. Mian, K. Stuart, R. Salavati. 2003. <br />       kinetoplastid protozoa. Microbiol. Mol. Biol. Rev. 61:105&#8211;120.                       Comparative analysis editosome proteins trypanosomatids. Nucleic <br /> 53.   Stuart, K. D., A. Schnaufer, N. L. Ernst, A. K. Panigrahi <br /> </body></html>