<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, June 1986, p. 746-750                                                                           Vol. 166, No. 3 <br /> 0021-9193/86/060746-05$02.00/0 <br /> Copyright &#169; 1986, American Society Microbiology <br />  <br />  <br />          Deletion spf (Spot 42 RNA) Gene <span id='am-26' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-38'  ></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span>Escherichia coli</span> <br />                              GRAHAM F. HATFULL AND CATHERINE M. JOYCE* <br /> Department Molecular Biophysics Biochemistry, Yale University Medical School, New Haven, Connecticut 06510 <br />                                          Received 1 November 1985/Accepted 10 March 1986 <br />  <br />             To investigate function spot 42 RNA, small RNA <span id='am-42' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-43'  ></span><span id='am-44' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia</span> coli, constructed strain which <br />          spf, structural gene RNA, deleted. We achieved using Aatt phage k carrying DNA <br />          fragment spanning spf region precise deletion spf. By integration phage spf locus <br />          subsequent excision, able cross spf deletion bacterial chromosome. The fact <br />          deletion obtained indicated spf essential gene. We did observe major <br />          defect Aspf cells, strain background deletion caused slight growth impairment. <br />  <br />  <br />    Spot 42 RNA Escherichia coli moderately abundant                             MATERIALS AND METHODS <br /> (100 200 copies cell) RNA, 109 nucleotides long (14, <br /> 15). It encoded single gene (spf) located immediately                Materials. Restriction endonucleases obtained from <br /> downstream polA 86.6 min E. coli genetic map               New England Biolabs Boehringer Mannheim Biochemi- <br /> (2, 7, 13). The function spf gene unknown,             cals. T4 DNA ligase Collaborative Research, Inc., <br /> the abundance RNA suggested gene                    Boehringer Mannheim Biochemicals. Oligonucleotide <br /> product play important, essential, role                 linkers Collaborative Research, Inc., New <br /> vivo. The cellular distribution spot 42 RNA gives little               England Biolabs. Bacterial strains used work listed <br /> clue function; RNA associated                    Table 1. <span id='am-58' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-59' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>Plasmids</span> general techniques relevant <span id='am-16' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-19'  ></span><span id='am-20' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> <br /> membranes, ribosomes, nucleoid (15).                    work described previously (8). The <br />    Currently know ultimate gene                     plasmids derived vector pNG16, deletion <br /> product spf RNA protein. The RNA itself               derivative pBR322. <br /> structural catalytic component ribonucleoprotein                     Construction genetically marked spf deletion. This <br /> particle, like Ml RNA component RNase P (5). On                construction outlined Fig. 1. pGH78 contains 2.1- <br /> other hand, spot 42 RNA sequence features suggesting                  kilobase-pair (kb) SacI-HindIII fragment carrying end of <br /> that mRNA: ribosome-binding site followed                  polA coding region spf gene, BamHI linker <br /> by initiation <span id='am-21' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-22' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-23'  ></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>codon</span> open reading frame                 inserted AluI site 46 base pairs (bp) the <br /> coding capacity 15-residue peptide (14). The potential              termination codon polA (8). This BamHI site defined the <br /> translated product rich hydrophobic amino acids,                    upstream endpoint deletion spf. The downstream <br /> particularly leucine, contains total charged           endpoint defined using plasmid, AN3058, carrying an <br /> residues (including N- C-terminal amino acids). The               vitro-constructed deletion polA (6). This plasmid con- <br /> latter alternating pattern positively negatively           tains DNA Sau3A site position 3058 our <br /> charged groups adjacent pairs separated approxi-                  published sequence (7), HindIII site position 3822, <br /> mately turn a-<span id='am-6' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-7' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10'  ></span>helix</span>. Consideration hydro-                BamHI site position original Sau3A site. <br /> phobic nature putative peptide gene product led             As genetic marker used kanamycin resistance (Kmr) <br /> and suggest inserted mem-                   gene Tn903; plasmid pGH54 contains gene 950-bp <br /> branes, serving structural role forming channels             BamHI fragment inserted BamHI site pNG16 (8). <br /> to allow passage certain ions small molecules (7,               The 800-bp BamHI-EcoRI fragment AN3058 the <br /> 13). Sahagan Dahlberg shown expression                   950-bp BamHI fragment pGH54 purified ligated <br /> spf negatively regulated cyclic AMP (15). This             large EcoRI-BamHI fragment pGH78, a <br /> taken circumstantial evidence linking spf gene             structure Kmr gene (transcribed same <br /> product membrane structure function, cyclic                 direction polA spf) replaces spf. The Sall site the <br /> AMP negative regulator membrane                  vector DNA replaced EcoRI linker the <br /> envelope proteins (1, 10, 12) cell division (17).           resulting 3.2-kb EcoRI fragment recovered cloning <br />    The problem assigning function gene locus             EcoRI site pNG16, pGH84. The Kmr Aspf <br /> approached asking gene                   construction inserted EcoRI vector, Xgtl- <br /> (which available cloned fragment) essential               XB (4). The structure resulting phage, XGH84, was <br /> for viability E. coli. This determining                 verified Southern blot analysis, using pCJ1 probe. <br /> whether gene replaced chromosome                  A control phage, XGH86, carries Kmr gene <br /> in vitro-constructed deletion, using strategy             deletion spf, constructed analogous way <br /> described previously (8). If target gene essential         insertion Kmr fragment BamHI site of <br /> survival, obtain strain gene             pGH78. <br /> deleted. This null mutation used test                     Isolation analysis lysogens. Lysogens XGH84 E. <br /> functions gene.                                                    coli CM4722 selected 30&#176;C L-broth plates seeded <br />                                                                           Xch80del9 (11) subsequently checked X <br />   * Corresponding author.                                                 immunity Kmr. Chromosomal DNA candidate <br />                                                                     746 <br />  VOL. 166, 1986                                                                                                 DELETION OF THE spf GENE             747 <br />  <br />           TABLE 1. Bacterial strains used work                                desired transductants selected plates containing 40 <br />  Strain                   Genotype                         Source                     ,ug kanamycin ml. <br /> CM4722&apos;   F+ A(gal-bio) thi-I relAl spoTI     W. S. Kelley <br />                                                                                          Growth CJ200 CJ205 liquid medium com- <br /> CJ200     CM4722 spf&apos; Kmr                     This work                               pared using competition experiment. Each strain was <br /> CJ205     CM4722 Aspf Kmr                     This work                               mixed equal number cells parent strain, <br /> S165      AgalS165 rpsL                   Our strain                              CM4722 (Kms). After period growth, mixture was <br />                                                 collection                            plated L broth, proportion Kmr cells (CJ200 or <br /> LE392  lacY galK galT metB trpR supE supF     Our strain                              CJ205) determined. <br />          hsdR                                   collection                               The sensitivity spf&apos; Aspfstrains salt, detergents, <br /> AW330 lac gal <span id='am-11' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-15'  ></span>ara</span> xyl thr leu thi fhuA rpsL   R. Macnab                               drugs compared ways. In method, a <br /> 20SOK- galK galP                              R. Macnab                               heavy inoculum strain tested streaked out <br /> JC1552 leuB trp hisG argG metB lacY gal       B. Bachmann                             sector L-broth plate. The substance be <br />          malA xyl mtl rpsL fhuA tsx supE        (CGSC)b                               tested placed center plate. After <br /> AB2557 purF dsdA aroC ilvD lac malA xyl mtl B. Bachmann                               incubation, face-upwards, position inhibition zone <br />          rpsL fhuA tsx supE                     (CGSC) <br /> KL188 pyrD trp thyA thi galK malA xyl mtl B. Bachmann                             isogenic Aspf spf&apos; pairs compared. Alterna- <br />          rpsL                                   (CGSC)                                tively, strains plated agar substance <br />   CM4722 F+ derivative HfrH strain KS302 (16). <br />                                                                                       tested placed filter paper disk lawn of <br />  b <br />     CGSC, E. coli Genetic Stock Center, Yale University, New Haven,                   cells, described Benson DeCloux (3). Similar <br /> Conn.                                                                                 results obtained methods. <br />                                                                                          Swarming behavior motile strains examined on <br />                                                                                       tryptone plates containing 0.38% agar. <br /> lysogens analyzed Southern blots determine                                                         RESULTS <br /> structure spf locus.                                                             Our method determining particular gene is <br />    Curing lysogens. Lysogens grown overnight                               essential survival (Fig. 2) described detail <br /> 30&#176;C L broth. Cured cells selected plating                                 previously (8). Briefly, involves insertion Kmr <br /> appropriate dilutions 42&#176;C L-broth plates containing 1                          marker sequences immediately flanking the <br /> mM EDTA. The proportion survivors retained Kmr                                gene interest, using standard vitro manipulations. The <br /> was determined replica plating. To obtain independent                           genetically marked deletion cloned phage A vector <br /> estimate relative frequencies recombination                             Aatt thermoinducible (cI857). At permissive <br /> events giving rise Kmr kanamycin-sensitive (Kms)                               temperature, phage lysogenize homologous re- <br /> cells, sample lysogen culture induced                         combination cloned bacterial sequences, giving a <br /> phage lysate. The composition lysate determined                            structure integrated prophage flanked a <br /> by forming lysogens strain LE392 determining                              wild-type deleted (Kmr) copy target gene. <br /> the relative numbers Kmr Kms lysogens. (For                            Spontaneous curing lysogen regenerate <br /> details, reference 8.)                                                            wild-type configuration leave Kmr-marked dele- <br />    Hybridization probes. Southern blots probed                            tion chromosome, depending position the <br /> with nick-translated pCJ1, carries 5-kb HindIll                               recombinational crossover. Failure obtain Kmr cells <br /> fragment containing polA spf genes (9), spot                          temperature-resistant cured survivors indi- <br /> 42 RNA labeled vitro transcription described                                 cate deletion target gene lethal the <br /> previously (7).                                                                       necessary recombinational events did place a <br />    Other methods. Transduction spf Kmr E. coli                                high frequency. The possibility ad- <br /> CJ200, Aspf Kmr E. coli CJ205, carried                               dressed independent estimate fraction dele- <br /> using P1 kc following standard procedures (11). The                               tion-producing events, examining product <br />  <br />  <br />                                          500 bp <br />  <br />                                   S    Sc                            Bspf                          H.,R <br />                                                                              P.II <br />                                                                                                                        pGH78 <br />                                                                              B <br />                                                                                                                        AN3058 <br />                             R    Sc                                 KmR                                   H,R <br />                                                       za <br />                                                            B <br />                                                                     I%--H1-               B <br />                                                                                                           II <br />                                                                                                                        pGH84 <br />                             R    Sc                                 Km           IH       B                     H,R <br />                                                                                                                  ,r <br />                             I                                       . .... <br />                                                                                               -W <br />                                                                                                                       * oGH36 <br />   FIG. 1. Construction plasmids used work. For details text. The line represents DNA derived E. coli <br /> chromosome. Within region indicated polA coding sequence (hatched area), spf gene (black arrow), functional <br /> open reading frame 22-kilodalton protein (open arrow) (7). The DNA fragment carrying Kmr gene represented wavy line. <br /> DNA derived plasmid <span id='am-49' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> drawn line shown entirety. B, BamHI; H, Hindlll; R, EcoRI; S, Sall; Sc, <br /> Sacl. <br />  748     HATFULL AND JOYCE                                                                                                        J. BACTERIOL. <br />  <br /> of excision process, phage produced induction                                            XGH84 <br /> the lysogen, repeating curing addi- <br /> tional copy target gene trans remove                                            R KmR R <br /> selective pressure deletion. <br />    Phage lysogen structure. The construction XGH84, <br /> in spf gene replaced Kmr while <br /> retaining surrounding sequences, described                                 S                .                S <br /> Materials Methods section Fig. 1. The deletion <br /> endpoints carefully chosen remove completely                                               13.3 - <br /> the spf-transcribed region encroaching of <br /> the extremely close neighboring genes (7). Lysogens of <br /> XGH84 CM4722 selected, chromosomal                                                      I Integration <br /> DNA analyzed Southern blots identify in <br /> which integration taken place spflocus. Digestion <br /> with EcoRI SalI gives pair fragments (of sizes 8.2                     SR                               R KMR               S <br /> and 8.3 kb, 7.5 9.0 kb, depending position            A     -4      _                  X A            C              | <br /> the recombihational crossover) integration taken place <br />                                                                                  8.3   *                                   8.2 <br /> at spf (Fig. 3). Integration chromosome               OR <br /> results 13.3-kb fragnient diagnostic unrearranged                    S              KmR R <br /> spf locus 3.2-kb fragment derived XGHS4. We                                                            R.   *S <br /> chose independent lysogens, type A of <br />                                                                             B <br />                                                                                         9.0        b    *x-m               7.5 <br /> type B, curing experiment. <br />    Curing lysogens shows spf deleted. Cured             FIG. 3. Integration XGH84 spf locus. Two alternative <br /> cells selected overnight cultures chosen            structures, designated A B, result integration of <br /> lysogens, proportion Kmr colonies deter-              XGH84 homologous recombination spf. Symbols: *, position <br />                                                                      spf gene; mm, upstream flanking DNA carried XGH84; <br /> mined (Table 2). In case, substantial proportion           zi, downstream flanking DNA: Chroniosomal DNA external to <br /> Kmr, spf-deleted, cells obtained. For         region, homology XGH84 shown wavy line. The <br /> lysogen tested, ratio Kmr Kms cured cells              position relevant EcoRI (R) Sall (S) sites indicated, along <br /> approximately reciprocal ratio Kmr Kms                  sizes (in kilobases) fragments hybridize a <br /> transducing phage obtained induction lysogen,             pCJ1 probe. <br /> indicating selective <span id='am-1' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Bias" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5'  ></span>bias</span> spf-deleted <br /> cells. There consistent three- fourfold bias favor <br />                                                                      recombination upstream flanking region, that <br />                                                                      lysogens structure A gave tnore Kmr cells, lysogens of <br />                                                                      structure B gave Kms cells (Table 2 Fig. 3). This is <br />                                                                      probably greater length upstream region of <br />                            ., X        _                             flanking homology (1,150 bp opposed 770 bp). <br />                                                                         Southern blot analysis chromosomal DNA cured <br />                                                                      cells obtained way showed Kms cells had <br />                                                                      regained 5-kb HindIII fragment diagnostic unrear- <br />                                                                      ranged spf locus (<span id='am-53' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55'  ></span><span id='am-56' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> shown). Kmr cells, other <br />                                    Integration                       hand, new HindIII fragments 4.7 1.1 kb (Fig. <br />                                                                      4B, lanes 1 2). Moreover, DNA cells did not <br />                                                                      hybridize spot 42 RNA (Fig. 4B, lanes 5 6). This <br />                   KmR.                     x                         observation confirms hybridization data Rice and <br />                                                                      Dahlberg (13) indicated wild-type E. coli <br />                                                                      chromosome contains single locus homologous spot 42 <br />                                        Curing                        RNA. <br />                                                                         The curing experiment provided cells lacking the <br />                                                                      spf gene, serve tool examline the <br />                                                                      phenotype spf null mutation probe the <br />               x                                 KmR                  function spf gene. To sure phenotype was <br />                                                       =P__           attributable Aspf mutation, generated isogenic <br />                                   OR                                 spf&apos; Kmr strain, using control phage XGH86, which <br />                                                                      Kmr inserted removing spf (Fig. 1). The <br />                      )                           X                   strains CJ200 (spfJ Kmr) CJ205 (Aspf Kmr) identical <br />                                                                      presence CJ200 230 bp spanning spf <br />                                                                      gene (Fig. 4). Southern blot analysis DNA spf+ <br />                                                                      Kmr cells confirmed smnaller HindIII fragment was <br />    FIG. 2. Method determining gene deleted.     1.3 kb, compared 1.1 kb Aspf cells (Fig. 4B, lanes 3 <br /> Phage integration excision used cross deletion   4), fragment hybridized spot 42 RNA <br /> target gene (X) bacterial chromosome. For details the <br /> text. The tegions homology phage chromo-      (lanes 7 8). <br /> some boxed: X-i, upstream flanking DNA; =., downstream             Phenotype Aspf cells. We compared growth CJ205 <br /> flanking DNA. Chromosomal DNA external regions           (Aspf) CJ200 (spf) various solid media: L broth or <br /> homology shown wavy lines. (Note      MacConkey agar, supplemented glucose, lactose, <br /> possible integrated structures shown.)                            galactose, maltose, M9 minimal medium (11) supple- <br />  VOL. 166, 1986                                                                                      DELETION OF THE spf GENE              749 <br />  <br />  <br />                    A                        I kb <br />  <br />                                 H                                               H <br />                                                                                                            spf <br />  <br />  <br />                                 H                                  KmR ,                   H <br />                                                     4.7                   &apos;O        1.                     Aspf KmR <br />  <br />  <br />                                 H                                  KmR I,H                     H <br />                                  &apos;*                 4.7                 -O..*        1.3       w <br />                                                                                                            spf KmR <br />  <br />    B             1 2 3 4              5 6 7 8 <br />                                                                   FIG. 4. Restriction analysis cured cells. (A) The diagram <br />                                                                 compares   structure spf locus wild-type cells (spfJ), the <br />   4.7- <br />                                                                 /spf Kmr cells obtained curing XGH84 lysogens, spf&apos; <br />                                                                 Kmr cells derived XGH86. The spf gene represented the <br />                                                                 black arrow, Kmr insert represented boxed region. <br />                                                                 HindlIl sites (H) shown, sizes (in kilobases) of <br />                                                                 Hindlll fragments hybridize pCJ1 probe. (B) Southern blot <br />                                                                 analysis Aspf Kmr cells (lanes 1, 2, 5, 6) spfF Kmr cells <br />                                                                 (lanes 3, 4, 7, 8). Chromosomal DNA isolates each <br />                                                                 genotype digested HindIII. Each digest divided <br />                                                                 tracks 1% agarose gel lanes 1 4 5 to <br />                                                                 8 contain identical sets DNA samples. After transfer nitrocel- <br />   1.3-                  -..k   ~.. <br />                                                                 lulose, lanes 1 4 hybridized nick-translated pCJ1, and <br />                                                                 lanes 5 8 hybridized spot 42 RNA transcribed vitro. <br />   1.1-                                                          The sizes fragments (in kilobases) indicated. <br />  <br />  <br />  <br />  <br /> mented glucose glycerol. We difference      generations temperature, proportion CJ205 <br /> colony size morphology 37&#176;C 42&#176;C. However,        cells decreased (typically 50% start the <br /> the plates incubated 23&#176;C, CJ205 (Aspf) gave smaller    experiment 25% 10 generations). We suspect the <br /> colonies CJ200. The difference noticeable      growth difference liquid medium 42&#176;C be <br /> L-broth plates, amounted twofold difference      peculiar CM4722 background, effect was <br /> in colony diameter 2 days growth. We believe      seen S165 strain background. Moreover, effect in <br /> the slower growth 23&#176;C attributable spf deletion   liquid medium 42&#176;C accompanied observed <br /> and secondary mutation, observed          difference plating efficiency colony morphology the <br /> same result independent isolates CJ205         temperature. <br /> genotype obtained curing transduction.              We tested number properties diagnostic alter- <br /> However, effect dependent CM4722 back-           ation membrane structure function. The Aspf mutation <br /> ground. No difference growth 23&#176;C seen        did affect sensitivity cells NaCl, EDTA, sodium <br /> compared Aspf Kmr spf&apos; Kmr derivatives         dodecyl sulfate, various antibiotics (carbenicillin, tetracy- <br /> strains listed Table 1. Since CM4722 strain   cline, nalidixic acid, rifampin), 37&#176;C 23&#176;C. <br /> this group F+, transferred F factor         (Increased sensitivity compounds characteristic of <br /> CM4722 Aspf derivative strain S165. The resulting <br /> strain indistinguishable F- parent plates at <br /> 23&#176;C, ruling effect F fact <br /> </body></html>