<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                                                 Expression and purification of F-plasmid RepE and <br /> and Crystallization                             preliminary X-ray crystallographic study of its <br /> Communications <br />                                                 complex with operator DNA <br />  <br />  <br />  <br />  <br />                   The replication initiator factor RepE of the F plasmid in <span id='am-23' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-34'  ></span><span id='am-35' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span>Escherichia coli</span> is an <br />                             essential protein that stringently regulates the F-plasmid copy number. The <br />                                                 RepE protein has a dual function: its monomer functions as a replication <br />  <br />    initiator, while its dimer acts as a transcriptional repressor of the repE gene. The <br />             wild-type dimeric RepE protein was expressed as an N-terminal histidine-tagged <br />      protein, purified under native conditions with a high salt concentration and <br />  <br />                                                 crystallized in complex with the repE operator DNA using the sitting-drop <br />                                                                                    &#730; after <br />  <br />                                                 vapour-diffusion technique. The crystals diffracted to a resolution of 3.14 A <br />                         the application of dehydration and crystal annealing and belong to space group <br />                                                 P21, with unit-cell parameters a = 60.73, b = 99.32, c = 95.00 A   <br />  <br />  <br />                                                 1. Introduction <br />                                                 The mini-F plasmid, which is a derivative of the F factor in Escher- <br />                                                 ichia coli, replicates under strict control, maintaining a copy number <br />                                                 of 1&#8211;2 per host chromosome (Kline, 1985). The plasmid-encoded <br />                                                 initiator protein RepE (251 amino acids, 29 kDa) plays an important <br />                                                 role in the regulation of mini-F replication. Two distinct oligomeric <br />                                                 states, monomeric and dimeric, of RepE have been found and their <br />                                                 functions are known to be different (Ishiai et al., 1994). The RepE <br />                                                 monomer is a replication initiator that recognizes a specific 19 bp <br />                                                 sequence DNA (iteron; Masson &amp; Ray, 1986, 1988; Tokino et al., <br />                                                 1986). Within the replication origin (ori2), four directly repeated <br />                                                 iterons (DR) are present and are bound by RepE molecules, which <br />                                                 could cause local DNA melting at the 13-mer region with the assis- <br />                                                 tance of a histone-like protein HU and subsequent initiation of <br />                                                 mini-F replication (Kawasaki et al., 1996; Komori, Matsunaga et al., <br />                                                 1999). In contrast, the RepE dimer is an autogenous repressor that <br />                                                 binds to the repE promoter/operator region located just upstream of <br />                                                 the repE gene (Rokeach et al., 1985; Masson &amp; Ray, 1986, 1988; <br />                                                 Tokino et al., 1986). The operator of repE includes an inverted-repeat <br />                                                 (IR) sequence, which shares 8 bp with the iterons. The RepE dimer <br />                                                 recognizes the common 8 bp of the IR and inhibits the transcription <br />                                                 of repE itself. Since RepE exists primarily as a dimer, in <span id='am-20' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-21' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> to <br />                                                 activate RepE as an initiator, conversion from the dimer to the <br />                                                 monomer is accomplished by the molecular chaperones DnaK, DnaJ <br />                                                 and GrpE (Kawasaki et al., 1990, 1991; Wada et al., unpublished <br />                                                 results). The crystal structure of the active initiator form of RepE has <br />                                                 been reported previously (Komori, Matsunaga et al., 1999) and <br />                                                 suggested that the C-terminal domain recognizes the specific DNA <br />                                                 sequence and that the N-terminal domain interacts with the DNA <br />                                                 backbone. Based on the RepE monomer structure, a marked <br />                                                 conformational change is required for dimerization. Structural <br />                                                 information on the repressor form of RepE, however, remains <br />                                                 unknown. Because most initiator proteins generally aggregate easily, <br />                                                 wild-type and full-length initiators have not been targeted for struc- <br />                                                 tural studies. We have successfully expressed and purified wild-type <br /> # 2007 International Union of Crystallography   RepE and here we report the crystallization and preliminary X-ray <br /> All rights reserved                             diffraction analysis of RepE bound to DNA containing an IR <br />  <br /> 346       doi:10.1107/S1744309107012894                                                                              Acta Cryst. (2007). F63, 346&#8211;349 <br />                                                                                                        crystallization communications <br /> Table 1                                                                                    was added to His6-RepE solution in a 1.2-fold molar excess relative to <br /> <span id='am-46' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-47'  ></span><span id='am-48' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Data</span>-collection and processing statistics of the His6-RepE&#8211;DNA1 complex.                   the RepE dimer and the mixture was subsequently dialyzed against <br /> Values in parentheses refer to the highest resolution shell.                               crystallization buffer [100 mM potassium citrate pH 6.2, 100 mM KCl, <br /> Wavelength (A &#730;)                                                1.0000                     10 mM MgCl2, 10%(v/v) glycerol and 2 mM dithiothreitol (DTT)] at <br /> Space group                                                     P21                        277 K and concentrated to approximately 5 mg ml 1 His6-RepE <br />                       &#730; ,  ) <br /> Unit-cell parameters (A                                         a = 60.73, b = 99.32,      concentration. Before crystallization, the formation of the protein&#8211; <br />                                                                    c = 95.00,   = 108.55 <br /> Resolution range (A&#730;)                                           50.0&#8211;3.14 (3.25&#8211;3.14)      DNA complex was confirmed by native polyacrylamide gel electro- <br /> Observed reflections                                            42887                      phoresis (PAGE), in which a band corresponding to the protein&#8211; <br /> Unique reflections                                              16535 <br /> Completeness (%)                                                88.3 (73.6) <br />                                                                                            DNA complex was stained with both Coomassie Brilliant Blue (CBB) <br /> I/ (I)                                                          11.0 (3.9)                 and Mupid-Blue (Advance, Tokyo, Japan). <br /> Rsym&#8224; (%)                                                       6.7 (15.0) <br />         P P                          P P <br /> &#8224; Rsym = hkl i jIi &#240;hkl&#222;   hI&#240;hkl&#222;ij= hkl i Ii &#240;hkl&#222;, where Ii(hkl) is the ith intensity   2.3. Crystallization <br /> measurement of reflection hkl and hI(hkl)i is the average value over multiple <br /> measurements.                                                                                 The crystallization conditions for the His6-RepE&#8211;operator <br />                                                                                            complexes were screened by the sitting-drop vapour-diffusion <br /> sequence in order to elucidate the crystal structure of dimeric RepE                       method at 277 K. Initial conditions were found using a <span id='am-11' about='obo:NCBITaxon_6083' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_6073 obo:NCBITaxon_6074 obo:NCBITaxon_33208 obo:NCBITaxon_6076 obo:NCBITaxon_6080 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_37516'><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Pelmatohydra" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Hydra" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="NCBITaxon:280358" datatype="xsd:string"></span><span id='am-15'  ></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Chlorohydra" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasAlternativeId" content="NCBITaxon:6081" datatype="xsd:string"></span>Hydra</span> II Plus <br /> in complex with repE operator DNA.                                                         One (Matrix Technologies Corporation, Hudson, NH, USA) with <br />                                                                                            commercially available screening kits (Hampton Research, Aliso <br />                                                                                            Viejo, CA, USA; Sigma&#8211;Aldrich, St Louis, MO, USA). A drop was <br /> 2. Materials and methods                                                                   prepared by mixing equal volumes (0.3 ml for initial screening and <br /> 2.1. Expression and purification                                                           1.0 ml for optimization) of the His6-RepE&#8211;operator complex solution <br />                                                                                            and the reservoir solution, allowing the mixture to equilibrate against <br />    E. coli strain M15 harbouring pREP4 plasmid (Qiagen, Hilden,                            the reservoir solution (80 ml for initial screening and 100 ml for <br /> Germany) was transformed with pKV7202 plasmid carrying an                                  optimization). The presence of the complex in the crystal component <br /> N-terminal His-tagged repE gene (Matsunaga et al., 1995) and grown                         was identified by native PAGE using the washed and dissolved <br /> at 310 K in <span id='am-1' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-2'  ></span><span id='am-3' property="rdfs:label" content="Luria" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Luria</span> broth medium containing 50 mg ml 1 ampicillin and                        crystals. <br /> 25 mg ml 1 kanamycin. When the culture reached mid-log growth <br /> phase (optical density at 600 nm = 0.6), the culture was rapidly cooled <br /> on ice and protein expression was induced by the addition of                               2.4. X-ray diffraction analysis <br /> isopropyl  -d-thiogalactopyranoside (IPTG) to a final concentration <br />                                                                                               A His6-RepE&#8211;DNA1 (Fig. 1) crystal mounted in a nylon loop was <br /> of 0.1 mM. After further growth for 22 h at 293 K, the cells were                          transferred and soaked for 36 h at 277 K in a dehydration solution <br /> harvested by centrifugation at 2110g for 30 min at 277 K. All of the                       containing a 1.2-fold higher concentration of each component of the <br /> following purification steps of N-terminal His-tagged RepE (His6-                          crystallization condition. The dehydrated crystal was briefly <br /> RepE) were performed under cooled conditions at 277 K. The                                 immersed into the dehydration solution supplemented with 25&#8211; <br /> bacterial pellets were resuspended in lysis buffer [50 mM NaH2PO4                          30%(v/v) glycerol as a cryoprotectant and then frozen by flash- <br /> pH 8.0, 1 M KCl, 20%(v/v) glycerol and 10 mM imidazole] supple-                            cooling in a stream of nitrogen gas at 100 K. Prior to X-ray irradia- <br /> mented with 1 mg ml 1 lysozyme and 0.5 mM 4-(2-aminoethyl)-                                tion, we performed crystal annealing (Harp et al., 1998; Heras &amp; <br /> benzenesulfonyl fluoride (p-ABSF) and disrupted by sonication on                           Martin, 2005) by blocking the cryostream twice for 8 s each time. <br /> ice. The cell lysate was centrifuged at 27 000g for 60 min and the                         X-ray diffraction data were collected using an ADSC Quantum 315 <br /> supernatant was then filtered using a membrane filter with 0.45 mm <br />                                                                                            CCD detector on SPring-8 beamline BL41XU at a wavelength of <br /> pore size. After Ni2+&#8211;nitrilotriacetic acid (Ni&#8211;NTA) Superflow resin                       1.0000 A &#730; . A total of 180 frames were recorded with an oscillation <br /> (Qiagen) equilibrated with lysis buffer was added to the supernatant                       angle of 1.0  , an exposure time of 3.0 s per frame and a crystal- <br /> of the cell lysate, the mixture was stirred for 60 min and loaded onto                     to-detector distance of 350 mm. The diffraction intensities were <br /> an Ni&#8211;NTA Superflow column. The column was then washed with a                              integrated with the HKL-2000 (Otwinowski &amp; Minor, 1997) program <br /> stepwise gradient of a wash buffer [50 mM sodium phosphate pH 8.0, <br /> 1 M KCl and 20%(v/v) glycerol] containing 30, 50 and 100 mM <br /> imidazole. His6-RepE was eluted with a wash buffer containing <br /> 200 mM imidazole. Several cycles of purification by means of Ni&#8211; <br /> NTA immobilized metal-affinity chromatography (IMAC) were <br /> performed. For a stock solution, glycerol was added to a final <br /> concentration of 35%(v/v) and purified His6-RepE was stored in <br /> small aliquots at 193 K. <br />  <br /> 2.2. Preparation of RepE&#8211;DNA complex <br />   For RepE&#8211;DNA complex formation, oligonucleotides containing <br /> the inverted-repeat sequence in the repE operator recognized by the <br /> RepE dimer were prepared by Hokkaido System Science (Sapporo, <br /> Japan). Their sequences were designed based on the expectation that <br /> DNA duplexes form a pseudo-continuous <span id='am-6' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>helix</span> in crystal packing <br /> (Joachimiak &amp; Sigler, 1991), as shown in Fig. 1. DNA duplexes were                         Figure 1 <br />                                                                                            DNA duplexes used in complex formation and crystallization trials. The common <br /> prepared essentially as described by Komori, Sasai et al. (1999),                          8 bp sequences between iteron and operator are underlined and their directions are <br /> except that the final concentrations were 2.4 mM. Each DNA duplex                          indicated by the arrows. <br />  <br />  <br /> Acta Cryst. (2007). F63, 346&#8211;349                                                                                                      Nakamura et al.         RepE    347 <br />  crystallization communications <br /> suite. The data-collection and processing statistics are summarized in                    estimated to have a molecular weight of  100 kDa by gel-filtration <br /> Table 1.                                                                                  chromatography. In addition, it has been reported that the repE <br />                                                                                           operator is preferentially bound by the dimeric form of RepE <br />                                                                                           compared with the monomeric form (Ishiai et al., 1994). Therefore, <br /> 3. Results and discussion                                                                 we concluded that these complexes are constituted of the RepE <br />                                                                                           dimer and the DNA duplex. <br /> Wild-type RepE proteins tend to aggregate and more than 80% of                               The crystallization conditions for the four complexes were <br /> the RepE expressed in E. coli was collected as inclusion bodies. RepE                     screened by the sparse-matrix method. Although crystals of three <br /> protein has therefore previously been purified under denaturing                           complexes (complexes with DNA1, DNA2 and DNA3) were <br /> conditions using guanidine hydrochloride (Kawasaki et al., 1992;                          obtained, only one crystal (for the complex with DNA1) was suitable <br /> Kline et al., 1992; Ishiai et al., 1994; Matsunaga et al., 1997). However,                for X-ray analysis. After the presence of both His6-RepE and DNA1 <br /> in order to keep RepE proteins with their native fold for subsequent                      in the obtained crystals had been confirmed (Fig. 2), the optimization <br /> crystallization, we attempted to purify His6-RepE without dena-                           of the crystallization conditions for His6-RepE&#8211;DNA1 afforded a <br /> turing. For bacterial culture conditions, the temperature was shifted                     final condition of 100 mM HEPES&#8211;NaOH pH 7.2, 200 mM NaCl, <br /> from 310 to 293 K in order to decrease the protein-expression level                       10%(w/v) PEG 4000 and 5 mM SmCl3. Prismatic crystals grew in two <br /> during expression of the His6-RepE protein. Furthermore, during                           weeks (Fig. 3; 0.12   0.04   0.02 mm). Prior to X-ray exposure, both a <br /> RepE purification a high salt concentration of 1 M was maintained in                      dehydration treatment (Schick &amp; Jurnak, 1994; Heras &amp; Martin, <br /> the buffers to prevent protein aggregation. Owing to these modifi- <br /> cations, a significant improvement in the yield of purified RepE <br /> protein was achieved (from less than 1 mg to 25 mg of His6-RepE per <br /> litre of culture). After several cycles of purification by the IMAC <br /> method, His6-RepE attained over 90% purity as estimated by SDS&#8211; <br /> PAGE followed by staining with CBB. The result of gel-filtration <br /> chromatography for His6-RepE using a wash buffer containing <br /> 200 mM imidazole [eluted at 76 ml from Hi-Load 16/60 Superdex 200 <br /> (GE Healthcare Biosciences, Tokyo, Japan) at a flow rate of <br /> 0.3 ml min 1] indicates the presence of an  80 kDa protein but not of <br /> an  30 kDa protein, suggesting that His6-RepE forms dimers even <br /> under high-salt conditions (1 M). <br />    The complexes of His6-RepE with each of four DNA duplexes <br /> containing IR (31 or 33 bp; Fig. 1) were prepared and their <br /> complexation was confirmed by native PAGE as described above. <br /> Under normal conditions (pH 8.8), His6-RepE is unable to diffuse <br /> into a native polyacrylamide gel because the theoretical pI value is                      Figure 3 <br />                                                                                           Crystal of the His6-RepE&#8211;DNA1 complex. The scale bar is 100 mm in length. <br /> 9.5, whereas the complexes are able to move into the gel owing to the <br /> effect of the acidic oligonucleotides. As a result of staining the gels <br /> with CBB and Mupid-Blue independently to confirm the presence of <br /> His6-RepE and the DNA, respectively, specific sharp bands corre- <br /> sponding to the complexes were indeed detected at the same position <br /> in both gels (Fig. 2). No corresponding band was found in the lanes in <br /> which His6-RepE and DNA had been loaded independently onto the <br /> native gel (data not shown). The complex His6-RepE&#8211;DNA1 was <br />  <br />  <br />  <br />  <br /> Figure 2 <br /> Native PAGE of the crystals of the His6-RepE&#8211;DNA1 complex. Samples were <br /> loaded onto a 15% polyacrylamide gel (37.5:1 acrylamide:bisacrylamide) and <br /> electrophoresed. The gels were stained with either Mupid-Blue (left) or CBB               Figure 4 <br /> (right) for the detection of DNA1 and His6-RepE, respectively. The arrows labelled        Pseudo-precession image of a His6-RepE&#8211;DNA1 crystal representing the h0l zone <br /> Complex and Free DNA indicate the positions of the His6-RepE&#8211;DNA1 complex                                                                             &#730; . The picture was <br />                                                                                           of reciprocal space. The circle indicates a resolution of 4 A <br /> and free DNA1, respectively. Lane 1, solution used in crystallization; lane 2,            generated with HKLVIEW (Collaborative Computational Project, Number 4, <br /> crystallization mother liquor; lanes 3&#8211;4, washing solution; lane 5, dissolved crystals.   1994). <br />  <br />  <br /> 2005) and a process of crystal annealing were indispensable to          <br /> improve the resolution limit of X-ray diffraction from His6-RepE&#8211;         <br /> DNA1 crystals (from approximately 8 to 3 A   &#730; resolution). Diffraction      <br /> intensities were recorded to a maximum resolution of 3.14 A     &#730; using   <br /> synchrotron radiation, but the diffraction patterns were anisotropic,     <br />                                   &#730; in the weakest direction (Fig. 4).   <br /> showing diffraction to only 4.2 A                                            <br /> The diffraction symmetry and the systematic absences of the                <br /> diffraction data indicated that the His6-RepE&#8211;DNA1 crystals belong        <br /> to the monoclinic space group P21, with unit-cell parameters a = 60.73,     <br />                       &#730; ,   = 108.55  . Assuming the presence of one      <br /> b = 99.32, c = 95.00 A <br />                                                                         <br /> His6-RepE&#8211;DNA1 complex in the asymmetric unit, the VM value              <br /> (Matthews, 1968) was calculated to be 3.3 A     &#730; 3 Da 1, which corre-     <br /> sponds to a solvent content of 63%. Although diffraction data were         <br /> also collected at a wavelength of 1.6000 A    &#730; using one crystal, no       <br /> significant peaks derived from Sm atoms were found either in the             <br /> anomalous difference Patterson map or in the isomorphous differ-           <br /> ence Patterson map. Structure determination is currently in progress.        <br />   <br /> </body></html>