<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>   Depletion of the Cullin Cdc53p Induces Morphogenetic Changes in <br />                            <span id='am-56' about='obo:NCBITaxon_5475' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-57'  ></span><span id='am-58' property="rdfs:label" content="Candida" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Torulopsis" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Candida</span> albicans <br />   <br />              <span id='am-62' about='obo:NCBITaxon_5476' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5475 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-63' property="oboInOwl:hasAlternativeId" content="NCBITaxon:39399" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Candida stellatoidea" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="Candida stellatoidea type I" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:12" datatype="xsd:string"></span><span id='am-68'  ></span><span id='am-69' property="rdfs:label" content="Candida albicans" datatype="xsd:string"></span>Candida albicans</span> is an important opportunistic human fungal pathogen that can cause both mucosal and <br />           systemic infections in immunocompromised patients. Critical for the virulence of C. albicans is its ability to <br />           undergo a morphological transition from yeast to hyphal growth mode. Proper induction of filamentation is <br />           dependent on the ubiquitination pathway, which targets proteins for proteasome-mediated protein degradation <br />           or activates them for signaling events. In the present study, we evaluated the role of ubiquitination in C. <br />           albicans by impairing the function of the major ubiquitin-ligase complex SCF. <span id='am-75' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-79'  ></span>This</span> was done by depleting its <br />           backbone, the cullin Cdc53p (orf19.1674), using a tetracycline downregulatable promoter system. Cdc53p- <br />           depleted cells displayed an invasive phenotype and constitutive filamentation under conditions favoring yeast <br />           growth mode, both on solid and in liquid media. In addition, these cells exhibited an early onset of cell death, <br />           as judged from propidium iodide staining, suggesting that CDC53 is an essential gene in C. albicans. To identify <br />           Cdc53p-dependent pathways in C. albicans, a genome-wide expression analysis was carried out that revealed a <br />           total of 425 differentially expressed genes (fold change, &gt;2; P &lt; 0.05) with 192 up- and 233 downregulated <br />           genes in the CDC53-repressed mutant compared to the control strain. GO term analysis identified biological <br />           processes significantly affected by Cdc53p depletion, including amino acid starvation response, with 14 genes <br />           being targets of the transcriptional regulator Gcn4p, and reductive iron transport. These results indicate that <br />           Cdc53p enables C. albicans to adequately respond to environmental signals. <br />  <br />  <br />    Candida albicans is an important human fungal pathogen                     Hst7p, Cek1p, and the executing transcription factor Cph1p <br /> that can cause both superficial and systemic infections in im-                (reviewed in references 11 and 47). Further filamentation- <br /> munocompromised patients (57). The ability of this fungus to                  inducing pathways include the cyclic AMP-dependent protein <br /> grow in and change between different morphological forms                      kinase A pathway with its downstream effector Efg1p, the <br /> such as budding yeast (round or oval cells), true hyphae (fila-               Cph2p/Tec1p pathway, the Czf1p-dependent matrix induced <br /> mentous cells without constrictions at the sites of septation),               pathway, and the pH-responsive pathway which acts through <br /> and pseudohyphae (chains of elongated cells with constrictions                the transcriptional regulator Rim101p (reviewed in references <br /> at the sites of septation) (66) is considered critical for its                11, 40, and 74). This network, although complex, is still being <br /> virulence (12, 41). Several environmental conditions are                      refined, as can be seen by the recent discovery of the transcrip- <br /> known to trigger a switch from yeast to filamentous growth                    tional regulator, Ume6p, recently shown to regulate hypha <br /> mode, such as neutral pH, nutrient starvation, contact to solid               extension and virulence (6). Additional complexity is added by <br /> surfaces, and growth in the presence of serum at 37&#176;C, the                    negative transcriptional regulation, which is carried out by <br /> latter mimicking the bloodstream and other environments in                    heterodimeric protein complexes, consisting of Tup1p and one <br /> the host (reviewed in reference 21).                                          of the DNA-binding proteins Rfg1p or Nrg1p (reviewed in <br />    Various signal transduction pathways contribute to the reg-                references 11, 40, and 74). <br /> ulation of the yeast-filament growth switch, forming a complex                   The cytoskeleton reorganization that is necessary for serum- <br /> network that converges on a common set of hypha-specific                      induced filamentation in C. albicans is mediated by the Rho- <br /> genes (35). As in <span id='am-13' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-14' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-17'  ></span><span id='am-18' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>, a mitogen-activated               like GTPase Cdc42p and its exchange factor Cdc24p (8, 46, 70, <br /> protein kinase (MAPK) pathway is involved in filamentation in                 71). Recently, an additional Rho G-protein, Rac1p, and its <br /> C. albicans, consisting of the kinases Cst20p, probably Ste11p,               activator Dck1p have been identified in C. albicans, which are <br />                                                                               required for filamentous growth in an agar matrix (7, 29). <br />                                                                                  Interestingly, mutations in the ubiquitination pathway have <br />    * Corresponding author. Mailing address: Institute for Research in         been shown to interfere with the proper regulation of filamen- <br /> Immunology and <span id='am-70' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-71' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-73'  ></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Cancer</span>, Universite&#180; de Montre&#180;al, P.O. Box 6128,               tation in C. albicans (3, 38, 39, 60). Ubiquitination, the covalent <br /> Station Centre-Ville, Montreal, Quebec H3C 3J7, Canada. Phone:                attachment of a ubiquitin moiety (Ub) onto a target protein, is <br /> (514) 343-6746. Fax: (514) 343-6843. E-mail: martine.raymond                  carried out by an enzymatic cascade consisting of the Ub- <br /> @umontreal.ca. <br />    &#8224; Supplemental material for this article may be found at http://ec <br />                                                                               activating enzyme E1, the Ub-conjugating enzyme E2, and <br /> .asm.org/.                                                                    Ub-ligase E3, either assigning the target proteins for <span id='am-37' about='obo:NCBITaxon_4331' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4328 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_71241 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_232378 obo:NCBITaxon_3193'><span id='am-38'  ></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Protea" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>protea</span>- <br />    &#17188; <br />      Published ahead of print on 6 March 2009.                                some-mediated degradation or activating them for transcrip- <br />  <br />                                                                         756 <br />  VOL. 8, 2009                                                                                                 FUNCTION OF C. ALBICANS CDC53                           757 <br />  <br />  <br />                                                          TABLE 1. C. albicans strains used in this study <br />  Strain                                                         Genotype                                                       Parent             Source or reference <br />  <br /> CaSS1               his3::hisG/his3::hisG leu2::tetR-GAL4AD-URA3/LEU2                                                         CAI4                    58 <br /> KTY3                his3::hisG::HIS3/his3::hisG leu2::tetR-GAL4AD-URA3/LEU2                                                   CaSS1                   This   study <br /> KTY25               his3::hisG/his3::hisG leu2::tetR-GAL4AD-URA3/LEU2 cdc53&#9004;::HIS3/CDC53                                      CaSS1                   This   study <br /> KTY27               his3::hisG/his3::hisG leu2::tetR-GAL4AD-URA3/LEU2 CDC53/cdc53&#9004;::HIS3                                      CaSS1                   This   study <br /> KTY30               his3::hisG/his3::hisG leu2::tetR-GAL4AD-URA3/LEU2 cdc53&#9004;::HIS3/pTet-CDC53                                 KTY25                   This   study <br /> KTY31               his3::hisG/his3::hisG leu2::tetR-GAL4AD-URA3/LEU2 pTet-CDC53/cdc53&#9004;::HIS3                                 KTY27                   This   study <br />  <br />  <br />  <br />  <br /> tional activation or signaling events (16, 26). In S. cerevisiae,                     using a sequential PCR-generated disruption cassette that contains a HIS3 se- <br /> the ubiquitination system consists of a single E1, 11 E2s, and 42                     lectable marker flanked with CDC53 upstream and downstream homologous <br />                                                                                       sequences to replace one CDC53 allele (orf19.1674, Contig19-10123 or <br /> E3s, which can be divided into RING or HECT domain-con- <br />                                                                                       orf19.9243, Contig19-20123; chromosome 3) by homologous recombination. To <br /> taining proteins (37). Some of the E3 ligases are embedded in                         increase the frequency of homologous recombination, two successive rounds of <br /> SCF complexes, which are highly conserved in eukaryotes and                           PCR were used to extend the homologous flanking sequences from 79 bp to <br /> consist of the linker protein Skp1; a cullin, which acts as a                         &#11011;160 bp each. The primer pair used for the first round of PCR was MR1034 <br /> scaffold protein enabling spatial proximity of substrate and                          (5&#11032;-CGTATCATAACTTTCAATTTACATTATTCAATTTCATCCTCCTTTT <br />                                                                                       CCTCCTCCTCCTCCTACTCACATCAAATAATCAAGATCTTTCTGTGAC <br /> ligase; and a substrate-recognizing F-box protein (65, 75). A <br />                                                                                       TCAATTC) and MR1035 (5&#11032;-GGAAAGTAAAAGCAAGAAAAAAGATAGT <br /> well-characterized example is the S. cerevisiae SCFCdc4-E2-E3                         TTTGTAAAACTGTTTAGTCGGAGGAGGAGGAGCAGCAGCAGGAGG <br /> complex, which consists of the scaffold protein Cdc53p, the                           TATAATATGGATTTTAGTCAGTAAC), containing both a region complementary <br /> substrate-recognition F-box protein Cdc4p, the linker protein                         to the template plasmid pHIS3 (58) (plasmid-derived sequences are underlined) <br /> Skp1p, the E2-conjugating enzyme Cdc34p, and the RING-E3                              and to the CDC53 locus. The primer pair used for the second round of PCR was <br />                                                                                       MR1046 (5&#11032;-GAAACTTTTCTTTTTTACCTTCTCCCTTTTAACAAGTTATC <br /> enzyme Hrt1p (75). This complex is responsible for multiple <br />                                                                                       TTTTTTTATTTTTGTCTTGTTTTTGTTTTGTTTGATTACGTATCATAAC <br /> cellular processes, including the degradation of the cell cycle                       TTTCAATTTAC) and MR1047 (5&#11032;-CTATTTATTTGTAAGATACACCAAC <br /> inhibitor Sic1p, as well as the proper induction of amino acid                        ATACATCAACATATTTAGATTTATACTTATACTTATATATATATATAT <br /> starvation by degradation of the transcription factor Gcn4p in                        TAGGACATGGAAAGTAAAAGCAAGAAAAAAG), containing both a re- <br /> rich media (75).                                                                      gion complementary to the first set of primers (underlined nucleotides) and an <br />                                                                                       extended region of the CDC53 locus. To replace the endogenous CDC53 pro- <br />    In contrast to S. cerevisiae, not much is known about the <br />                                                                                       moter of the remaining wild-type allele with a repressible tetracycline promoter, <br /> ubiquitination system of C. albicans. Two C. albicans ubiquitin                       the heterozygous strains were transformed using a PCR-generated Tet promoter <br /> genes have been identified: UBI4 codes for a polyubiquitin                            replacement cassette containing the SAT-1 dominant selectable marker, flanked <br /> precursor which is processed to three ubiquitin units and is the                      by sequences homologous to the CDC53 locus, using the primer pair MR1063 <br /> major source of ubiquitin upon stress conditions, whereas                             (5&#11032;-GTTTTGAAGAGTGAGAGAGAGAGAGAGAAAGACTTTAAAATTT <br />                                                                                       CGCCCTTCAAATTTTTGCTCTCTCTTTTTACTTTCAACAATTGTGCAC <br /> UBI3 encodes a ribosomal protein fused to ubiquitin (59, 60). <br />                                                                                       TCCAGCGTCAAAACTAGAG) and MR1064 (5&#11032;-CACTGTGGCGGTAACA <br /> Depletion of UBI4 and RAD6 (E2), as well as CDC4 and GRR1                             CCTTGTTCTCATGAGCACCCAAAATATATTCAAGTCCTGGTTGGAT <br /> (F-box proteins), has been shown to increase filamentation,                           GAATGTCCAAGTGGCATTTAAATCACTGAATGGTGGAAGAGTAGAT <br /> indicating that ubiquitin-dependent pathways negatively regu-                         GACATTGATTAATTTAGTGTGTGTATTTG). Since flanking sequences <br /> late the yeast-to-hypha transition in C. albicans (3, 38, 39, 60).                    already consisted of at least 94 bp due to enhanced primer synthesis, only one <br />                                                                                       PCR was used for creating the promoter replacement cassette. Due to allelic <br /> We therefore assumed that impairment of the SCF ubiquitin- <br />                                                                                       variances in the CDC53 promoter region, primer MR1063 is homologous to <br /> ligase function, by depleting the cullin subunit Cdc53p, would                        nucleotides &#11002;385 to &#11002;292 of orf19.1674 and to nucleotides &#11002;447 to &#11002;354 of <br /> help characterize this complex filamentation regulon, as well as                      orf19.9243, whereas primer MR1064 is homologous to nucleotides &#11002;4 to &#11001;104 <br /> other SCF-dependent pathways in C. albicans. Obtained <span id='am-98' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-99' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-102'  ></span>data</span>                            of both orf19.1674 and orf19.9243. Plasmid pSAT1-Tet (58) was used as a <br /> indicate that impairment of SCF-dependent ubiquitination                              template, and the plasmid-derived sequences are underlined. Strain KTY3 was <br />                                                                                       created as a wild-type control for growth and transcriptional expression compar- <br /> leads to filamentation and early cell death in C. albicans and <br />                                                                                       isons by reintroduction of the HIS3 cassette back into one his3-deleted allele of <br /> suggest that proper transduction of environmental signals via                         strain CaSS1, using plasmid pHIS3. All strains were analyzed for correct inte- <br /> degradation of transcriptional factors is necessary for accurate                      gration by Southern blot analysis. <br /> induction of the amino acid starvation response, as well as the                          Transformation of C. albicans. Transformation of C. albicans strains was <span id='am-88' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-89' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-90'  ></span><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Car" datatype="xsd:string"></span>car</span>- <br /> reductive iron transport across the membrane.                                         ried out by using the improved lithium acetate method (73) with minor modifi- <br />                                                                                       cations. Briefly, strains were cultured in 100 ml of YPD to an optical density of <br />                                                                                       0.5, washed with 1&#11003; lithium acetate (LiAc) solution (100 mM lithium acetate, 10 <br />                         MATERIALS AND METHODS                                         mM Tris-HCl [pH 7.5], 1 mM EDTA [pH 8.0]), and resuspended in 500 &#9262;l of 1&#11003; <br />    Strains and culture conditions. All of the strains used in the present study are   LiAc. Portions (100 &#9262;l) of this cell suspension were mixed with 1 to 5 &#9262;g of DNA, <br /> listed in Table 1. Strains were cultured in YPD medium (2% peptone, 1% yeast          100 &#9262;g of denatured salmon sperm DNA used as carrier, and 600 &#9262;l of 40% <br /> extract, 2% glucose) at 30&#176;C, including 2% agar for solid media. To repress the       (wt/vol) polyethylene glycol 4000&#8211;1&#11003; LiAc and then incubated overnight at 30&#176;C <br /> Tet promoter of the pTet-CDC53/cdc53&#9004; strain KTY31, a YPD overnight                   on an overhead rotor. Cell suspensions were then heat shocked for 15 min at <br /> culture was diluted into fresh YPD medium containing 20 &#9262;g of doxycycline             44&#176;C, and selection for histidine prototrophy was performed on synthetic solid <br /> (Sigma)/ml (50) at an initial optical density of 0.005 or streaked for single         medium lacking histidine and uridine (0.67% nitrogen base, 2% glucose, 2% <br /> colonies onto YPD plates supplemented with 20 &#9262;g of doxycycline (stock con-           agar, and 0.2% amino acid mix without histidine and uridine). For the replace- <br /> centration of 50 mg/ml in ethanol)/ml.                                                ment of the CDC53 promoter using the SAT-1 cassette, post-heat shock cells <br />    Construction of a conditional C. albicans CDC53 mutant. Mutant construction        were regenerated for 4 h in 2 ml of YPD at 30&#176;C prior to selecting them on YPD <br /> was carried out using the GRACE method (58). For the creation of heterozygote         plates containing 200 &#9262;g of nourseothricin (clonNAT; Werner BioAgents, Jena, <br /> mutant strains, wild-type C. albicans strain CaSS1, which expresses a tetracycline-   Germany)/ml as described by Reuss et al. (56). Compared to the conditions used <br /> dependent transactivation fusion (TetR-ScGal4AD) protein, was transformed             by Roemer et al. (58), these conditions (reduction in the regeneration time, from <br />  758       TRUNK ET AL.                                                                                                                       EUKARYOT. CELL <br />  <br /> overnight to 4 h, and in the amount of nourseothricin used, from 400 to 200               similarity of 57%. A protein domain search in the Protein <br /> &#9262;g/ml) were found to be sufficient for efficient mutant selection (56).                   Knowledgebase UniProtKB (http://<span id='am-7' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-8' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-11'  ></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>beta</span>.uniprot.org/) con- <br />     Southern and Northern blot analyses. C. albicans genomic DNA was purified <br /> by using the glass-bead method as described for S. cerevisiae (61). Total RNA was <br />                                                                                           firmed a cullin domain represented by the Pfam accession <br /> extracted by using the hot phenol method (76). Southern and Northern blot                 number PF00888. Detailed analysis revealed that C. albicans <br /> analyses were performed as described previously (63). Probes used for both                orf19.1674 encodes a 752-amino-acid protein that, like S. <br /> Southern and Northern analyses were generated by PCR, gel purified, <span id='am-93' about='obo:NCBITaxon_586402' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3646 obo:NCBITaxon_1 obo:NCBITaxon_4004 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-94'  ></span><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-96' property="rdfs:label" content="Radiola" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>radiola</span>-              cerevisiae Cdc53p, contains an N-terminal region, consisting <br /> beled with [&#9251;-32P]dATP using random primers (22), and purified by using Seph- <br />                                                                                           of three cullin repeats necessary for interaction with the <br /> adex G-50 QuickSpin columns (Roche) according to the supplier&#8217;s protocol. <br /> Primer sequences are listed in Table S1 in the supplemental material. The signals         F-box linking protein Skp1p, and a C-terminal globular &#9251;/&#9252; <br /> were visualized and quantified using a Fuji Film imaging plate screen and the             domain containing the E3 ligase binding motif (79). <br /> MultiGauge v2.3 software (Fuji Film).                                                        Construction of a conditional CDC53 C. albicans mutant <br />     Microscopy. Cdc53p-depleted cells were visualized using a Zeiss Axio-Imager           strain. To investigate the role of the cullin Cdc53p in C. albicans <br /> Z1 microscope equipped with a &#11003;63 objective <span id='am-1' about='obo:NCBITaxon_3863' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_163743 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Lens" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Lens Mill." datatype="xsd:string"></span><span id='am-6'  ></span>lens</span>. For visualization of nuclei <br />                                                                                           and given the essentiality of S. cerevisiae Cdc53p (43), we con- <br /> and chitin, cells were fixed in 70% ethanol for 30 min and stained with 1 &#9262;g of <br /> DAPI (4&#11032;,6&#11032;-diamidino-2-phenylindole; Sigma) or calcofluor white (Fluorescent             structed a conditional CDC53 mutant using the GRACE tech- <br /> Brightener 28; Sigma)/ml for 20 min, respectively. Washings were carried out              nology (58) (Fig. 1). This consisted of deleting one allele of <br /> using 1&#11003; phosphate-buffered saline. Stained cells were then visualized by both            CDC53 and exchanging the endogenous CDC53 promoter of the <br /> differential interference contrast and epifluorescence microscopy using a DAPI            remaining wild-type allele with a repressible tetracycline pro- <br /> filter, and pictures were taken with an Axiocam. Wild-type and Cdc53p-depleted <br /> colonies were visualized with a Leica MZ FLIII fluorescence stereomicroscope in <br />                                                                                           moter, using a PCR-based methodology (58). For the deletion of <br /> bright and dark fields with a 0.63 objective lens using magnifications of 0.8 and         one CDC53 allele, a deletion cassette composed of the HIS3 <br /> 6.3, and pictures were taken with an Axiocam.                                             marker flanked by CDC53 upstream and downstream regions was <br />     Assessment of dead cells by propidium iodide staining. For propidium iodide           amplified and was used to transform strain CaSS1 which ex- <br /> staining, 100-&#9262;l aliquots of cell suspensions were pelleted and resuspended in <br />                                                                                           presses a chimeric tetracycline transactivator (Fig. 1A, upper <br /> 200 &#9262;l of 1&#11003; phosphate-buffered saline, and 2 &#9262;l of a 1-mg/ml propidium iodide <br /> solution (Sigma) was added. After incubation for 15 min at 30&#176;C, the cells were           panel, and see Materials and Methods). The resulting His&#11001; trans- <br /> visualized by using a Zeiss Axio-Imager Z1 microscope equipped with a &#11003;20                 formants were analyzed by Southern blotting, along with the <br /> objective by both differential interference contrast and epifluorescence micros-          CaSS1 strain, using EcoRV- and XbaI-digested genomic DNA <br /> copy using a rhodamine filter, and pictures were taken with an Axiocam. The               and a CDC53 upstream probe (Fig. 1B). In strain CaSS1, the <br /> percentage of stained cells was assessed by determination of the total cell count, <br /> as well as the amount of stained cells per picture. In the case of yeast cells, a total <br />                                                                                           probe detected a doublet band of 2.5 kb (Fig. 1B, lane 1). Ge- <br /> of 240 to 480 cells were analyzed, whereas for filaments, due to a lesser (i.e., 7 to     nome sequence analysis of strain SC5314, from which CaSS1 is <br /> 62) filament count per picture, a total of 11 to 15 pictures were analyzed per time       derived (58), showed that these two XbaI-EcoRV fragments <br /> point, totaling 306 to 509 filaments per time point. For a positive control, yeast        (2,428 bp for orf19.1674 and 2,488 bp for orf19.9243) result from <br /> cells, as well as filaments were heated to 65&#176;C for 40 min prior to treatment with <br />                                                                                           allelic variations (base deletion and/or addition) in the promoter <br /> propidium iodide. <br />     Microarray analysis. Transcription profiling and analysis were performed us-          region of CDC53. The Southern blot confirmed that one allele of <br /> ing long oligonucleotide microarrays as described elsewhere (52). Cy3- and                CDC53 was properly deleted in each of the two heterozygous <br /> Cy5-labeled cDNA probes were prepared from 25 &#9262;g of total RNA each and                    mutants analyzed (KTY25 and KTY27), as judged by the appear- <br /> hybridized on microarrays spotted with 70mer oligonucleotide probes for each of           ance of either a 1.9- or a 2.0-kb fragment (Fig. 1B, lanes 2 and 4). <br /> the 6,354 genes of Assembly 19 (http://www.candidagenome.org/) that were con- <br />                                                                                           orf19.1674 was deleted in strain KTY25 (cdc53&#9004;::HIS3/CDC53), <br /> firmed and characterized by the Candida Annotation Working Group (13). In all, <br /> four hybridizations of independently produced RNA preparations with recipro-              and orf19.9243 was deleted in strain KTY27 (CDC53/cdc53&#9004;:: <br /> cal labeling were carried out. Microarrays were scanned by using a ScanArray              HIS3), based upon the disappearance of the shorter and longer <br /> Lite microarray scanner (Packard Bioscience), and fluorescence intensities were           fragments from the doublet, respectively (Fig. 1B, lanes 2 and 4). <br /> quantified by using QuantArray. Lowess normalization and statistical analysis             A second round of transformation was carried out in KTY25 and <br /> were performed by using Genespring v7 (Agilent Technologies). Genes were <br /> considered to be differentially expressed if their average fold change in expres- <br />                                                                                           KTY27, using a PCR amplicon that consisted of the tetracycline- <br /> sion was &#11350;2 and if the average fold change was statistically significant as deter-        repressible promoter linked to the SAT1 dominant marker and <br /> mined by Student t test (P &#11349; 0.05). The complete microarray data produced in              flanking regions of the CDC53 promoter region (Fig. 1A, middle <br /> the present study are available in Table S2 in the supplemental material. It has          panel, and see Materials and Methods). Two independent Nour <br /> also been deposited in the NCBI Gene Expression Omnibus (20) and is acces- <br />                                                                                           transformants, KTY30 (cdc53&#9004;::HIS3/pTet-CDC53, derived from <br /> sible through GEO series accession number GSE13976 (http://www.ncbi.nlm.nih <br /> .gov/geo/query/acc.cgi?acc&#11005;GSE13976). Gene standard names and GO terms                    KTY25) and KTY31 (pTet-CDC53/cdc53&#9004;::HIS3, derived from <br /> used for the analysis were taken from Assembly 21 of the Candida Genome                   KTY27), were analyzed by Southern blot using the same restric- <br /> Database (http://www.candidagenome.org/), as well as S. cerevisiae orthologs/best         tion enzymes and probe as for the first round of strain construc- <br /> hits for differentially expressed C. albicans genes in Table S2 in the supplemental       tion. The disappearance of the remaining wild-type fragment of <br /> material. <br />                                                                                           the 2.5-kb doublet and the appearance of a 2.8-kb fragment con- <br />                                                                                           firmed a correct promoter replacement (Fig. 1B, lanes 3 and 5). <br />                                                                                           To evaluate CDC53 gene function independent of auxotrophy <br />                                     RESULTS <br />                                                                                           differences, the control strain KTY3 was created through recon- <br />   Identification of C. albicans CDC53. To identify the CDC53                              stitution of the HIS3 locus in CaSS1 (see Materials and Methods), <br /> gene in C. albicans, a BLAST search of the S. cerevisiae                                  and correct integration was confirmed by Southern blot analysis <br /> cullin Cdc53p (815 amino acids) against the C. albicans                                   (data not shown). <br /> genome was carried out, using BLASTP (protein/protein) at                                    To confirm conditional CDC53 expression in strains KTY30 <br /> the CandidaDB web server (http://genodb.pasteur.fr/cgi-bin                                and KTY31, both strains, as well as the control strain KTY3, <br /> /WebObjects/CandidaDB). This search revealed as closest                                   were analyzed by Northern blotting after growth in the absence <br /> homologue of the ScCdc53p protein the gene product of                                     or the presence of doxycycline, a tetracycline derivative (Fig. <br /> orf19.1674, with an overall amino acid identity of 38% and                                1C). For strains grown in the absence of doxycycline, an in- <br />  VOL. 8, 2009                                                                                   FUNCTION OF C. ALBICANS CDC53               759 <br />  <br />  <br />                                                                             crease in signal intensity of CDC53 transcripts was detected for <br />                                                                             strains KTY30 and KTY31 compared to the control strain <br />                                                                             KTY3, suggesting that the constitutively expressed tetracy- <br />                                                                             cline-repressible promoter is stronger than the endogenous <br />                                                                             CDC53 promoter. In addition, the length of the CDC53 tran- <br />                                                                             scripts derived from the Tet-repressible promoter appeared to <br />                                                                             be slightly shorter compared to those derived from the endog- <br />                                                                             enous promoter (Fig. 1C, compare lanes 2 and 3 to lane 1). <br />                                                                             This is potentially due to different transcription start sites giv- <br />                                                                             ing rise to shorter 5&#11032;-untranslated regions in the mRNA spe- <br />                                                                             cies encoded by the pTet-CDC53 allele. After 12 h of growth in <br />                                                                             YPD in the presence of 20 &#9262;g of doxycycline/ml, no CDC53 <br />                                                                             transcript could be detected in strains KTY30 and KTY31 <br />                                                                             (even after prolonged exposure [data not shown]), while the <br />                                                                             intensity of the CDC53 transcript remained unchanged in the <br />                                                                             control strain KTY3, confirming that these strains and growth <br />                                                                             conditions were suitable for the characterization of CDC53 in <br />                                                                             C. albicans. <br />                                                                                Suppression of CDC53 of C. albicans promotes an invasive <br />                                                                             phenotype. On rich medium (YPD) at 30&#176;C, the control strain <br />                                                                             KTY3, as well as the conditional CDC53 strain KTY31, typi- <br />                                                                             cally exhibited round, smooth colonies under nonrepressing <br />                                                                             conditions (data not shown). Under identical growth condi- <br />                                                                             tions but in the presence of doxycycline, strain KTY3 still grew <br />                                                                             as round soft colonies, whereas strain KTY31 displayed a <br />                                                                             deeply wrinkled surface if plated as a single streak and grew as <br />                                                                             very small single colonies, which were of a solid consistency <br />                                                                             and formed protrusions into the agar (Fig. 2A). Upon removal <br />                                                                             of both streaks and single colonies by washing, a nearly clean <br />                                                                             agar surface with rare single spots of cells was obtained for <br />                                                                             strain KTY3, whereas for strain KTY31, only the upper layer <br />                                                                             of the streaks could be washed off, leaving a complete layer of <br />                                                                             cells imprinted in the agar (Fig. 2B). This phenotype was even <br />                                                                             stronger with single colonies, since no cells whatsoever could <br />                                                                             be washed off, confirming an invasive phenotype of the <br />                                                                             Cdc53p-depleted strain. Strain KTY30 behaved as strain <br />                                                                             KTY31 (data not shown); therefore, only one strain (KTY31) <br />                                                                             was used in the subsequent experiments. <br />                                                                                Cdc53p depletion leads to filament formation under yeast <br />                                                                             growth conditions. A wrinkled colony surface and agar inva- <br />                                                                             siveness of colonies is often displayed by cells growing as fila- <br />                                                                             ments. To evaluate whether these features exhibited by <br />                                                                             Cdc53p-depleted cells under conditions favoring yeast growth <br />                                                                             mode were caused by filament formation, we determined the <br />    FIG. 1. Creation of a conditional CDC53 mutant strain. (A) For <br /> the creation of heterozygote mutant strains, one allele of CDC53 of         morphology of both control and CDC53-repressing cells during <br /> wild-type C. albicans strain CaSS1 was replaced by the HIS3 marker          exponential growth mode in rich medium at 30&#176;C by micro- <br /> (top panel). The SAT1 dominant selectable marker was used to replace        scopic analysis. Overnight cultures of strains KTY3 and <br /> the endogenous CDC53 promoter of the remaining wild-type allele by          KTY31 were diluted to an initial optical density of 0.005 into <br /> a repressible tetracycline promoter (block arrow with diagonal pat- <br /> tern) (middle panel). The two alleles of the resulting strain are de-       fresh YPD medium in the absence or the presence of doxycy- <br /> picted in the bottom panel. The probe used for Southern blot analysis       cline and were cultured at 30&#176;C. After 8 h of induction, the <br /> is indicated (filled rectangle) as well as the expected restriction frag-   growth curves of strain KTY31 under repressing conditions, <br /> ment lengths for the different alleles. E, EcoRV; X, XbaI. (B) South-       and the controls (KTY3 with or without doxycycline and <br /> ern blot analysis of the parental strain CaSS1 and the mutants KTY25 <br />                                                                             KTY31 without doxycycline) started to separate, reflected in <br /> (cdc53&#9004;::HIS3/CDC53), KTY30 (cdc53&#9004;::HIS3/pTet-CDC53), KTY27 <br /> (CDC53/cdc53&#9004;::HIS3), and KTY31 (pTet-CDC53/cdc53&#9004;::HIS3) us-               the doubling time of 1 h and 45 min for the mutant strain <br /> ing the restriction enzymes and probe shown in panel A. The different       compared to 1 h and 15 min for the controls. Furthermore, the <br /> alleles and the sizes of the corresponding restriction fragments are        optical density of the Cdc53p-depleted strain in stationary <br /> indicated at the left and the right sides of the figure, respectively.      phase reached only about two-thirds that of the controls (op- <br /> (C) Northern blot analysis of the control strain KTY3 and the mutants <br /> KTY30 and KTY31 after 12 h of growth at 37&#176;C in YPD in the absence          tical density at 600 nm &#11005; 8 compared to 12, Fig. 3A and B). To <br /> or presence of 20 &#9262;g of doxycycline/ml (Dox). rRNAs were used as            determine whether the reduced growth rate of strain KTY31 in <br /> loading controls.                                                           the presence of doxycycline was due to a change in morphol- <br />  760     TRUNK ET AL.                                                                                                        EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />   FIG. 2. Depletion of Cdc53p promotes invasive growth in C. albicans. (A) Macroscopic morphologies of streaks and single colonies of strains <br /> KTY3 and KTY31 grown on YPD for 24 h in the presence of doxycycline. (B) To confirm invasiveness for strain KTY31, both streaks and single <br /> colonies of strains KTY3 and KTY31 were washed with water and photographed in a dark field. <br />  <br />  <br />  <br /> ogy, we investigated the growth of Cdc53p-depleted cells by              ament counting after that time point. These data clearly con- <br /> microscopic analysis (Fig. 3C). After 7 to 8 h of cultivation,           firm an early onset of cell death for cells depleted in Cdc53p. <br /> when growth curves started to separate, we detected the oc-                 Genome-wide transcriptional analysis upon Cdc53p deple- <br /> currence of initial cell elongation, with subsequent filament            tion. In <span id='am-80' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> to characterize the consequences of CDC53 <br /> extension taking place well into the stationary phase. Cal-              downregulation at the molecular level and to identify potential <br /> cofluor white and DAPI staining of exponential growing cells             Cdc53p-dependent pathways in C. albicans, we analyzed the <br /> displayed one nucleus per filament, as well as septation at the          genome-wide expression profile of the mutant strain KTY31 <br /> mother bud neck and at further constrictions, two character-             compared to the control strain KTY3, using DNA microarrays. <br /> istics of pseudohyphae (66). Cells in stationary phase revealed          Four independent cell cultures were grown in YPD in the <br /> filament compartments partly missing nuclei, whereas septa               presence of doxycycline and harvested after 12 h of growth, a <br /> were located both at constriction sites and in nonconstricted            condition under which CDC53-repressed cells displayed a con- <br /> filaments, with the latter being characteristic for real hyphae          tinuous pseudohyphal growth phenotype, distinct from both <br /> (66), thus indicating a mixed population of pseudohyphae and             the initial cell elongation phase occurring after 8 h and the mix <br /> true hyphae in one filament cluster (Fig. 3C).                           of pseudohyphae and real hyphae of later time points (Fig. <br />    Depletion of the cullin Cdc53p leads to an early onset of cell        3C). To ensure that the harvested cells were in exponential <br /> death. The maldistribution of nuclei per filament as detected            growth mode, the growth of both strains KTY31 and KTY3 <br /> by DAPI and calcofluor white staining as well as the single              was monitored between 10 and 13 h. A linear regression was <br /> colony growth phenotype of Cdc53p-depleted cells suggested               performed on the log of the optical density at 600 nm values, <br /> early cell death. The growth arrest of single CDC53- repressed           returning coefficients of determination (R2) of greater than <br /> colonies after 24 h was only observed if they were not in                0.996, thus confirming exponential growth (Fig. 3B). Microar- <br /> contact with other colonies. If plated as a streak, cells kept           ray analysis was carried out with a total of four hybridizations <br /> growing as filaments, forming a deeply wrinkled surface, pos-            of independently produced RNA preparations, including re- <br /> sibly due to some Doxycycline depletion effect by neighboring            ciprocal labeling. Genes were considered differentially ex- <br /> colonies (Fig. 2). Therefore, we assessed cell viability of the          pressed if the calculated modulation was at least twofold and <br /> conditional CDC53 strain under repressing conditions (KTY31              statistically significant by t test (P &#11021; 0.5). Using these criteria, <br /> plus doxycycline) in comparison to the controls (KTY3 with or            a total of 423 genes were found to be differentially expressed <br /> without doxycycline and KTY31 without doxycycline) by pro-               with 190 upregulated and 233 downregulated genes when com- <br /> pidium iodide staining, a fluorescent compound that cannot               paring the conditional CDC53 mutant strain KTY31 to the <br /> penetrate viable cells (19). The percentage of stained and               control strain KTY3. <br /> therefore dead cells/filaments was determined every 24 h over               Differential regulation of selected genes was tested by <br /> a time-period of 5 days (Table 2). Heat-killed C. albicans yeast         Northern blot analysis (Fig. 4). In order to obtain an indepen- <br /> and hyphal cells were used as positive control, for which 100%           dent validation, a distinct set of RNA preparations than the <br /> staining was obtained. The percentage of dead filaments in               one used for the microarray hybridizations was prepared. The <br /> Cdc53p-depleted cells rose continually from 3% after 24 h to             resulting signals revealed a strong up- or downregulation of <br /> 28% after 96 h, during which time no significant cell death              the selected genes, confirming the expression data obtained by <br /> could be observed in the controls (Table 2). The assessment of           the DNA microarray experiment. Included in the Northern <br /> dead cells was abandoned after 96 h since excessive filament             blot analysis were strains KTY3 and KTY31 grown in the <br /> clustering in the Cdc53p-depleted cells hindered accurate fil-           absence of doxycycline, which displayed the same gene expres- <br />  VOL. 8, 2009                                                                                         FUNCTION OF C. ALBICANS CDC53               761 <br />  <br />  <br />  <br />  <br />    FIG. 3. Cdc53p functions as a repressor of filament development. (A) Growth curve of strains KTY3 (squares) and KTY31 (triangles) in liquid <br /> YPD. Closed shapes indicate growth in the absence of doxycycline, whereas open shapes represent growth in the presence of doxycycline. Due to <br /> superimposition of the three growth curves of KTY3 with or without doxycycline and KTY31 without doxycycline, only the closed squares of KTY3 <br /> without doxycycline can be seen. (B) Determination of exponential growth phase and growth rate of the control strain KTY3 (squares) and the <br /> cdc53 mutant strain KTY31 (triangles), both in the presence of doxycycline. The coefficients of determination (R2) were calculated using the <br /> R-square function of Excel. The arrows indicate the time point of cell harvest of both strains for microarray analysis. (C) Time course microscopic <br /> analysis of cells of strain KTY31 under Cdc53p-depleting conditions. Calcofluor white and DAPI staining of exponential grown cells display septa <br /> at mother bud necks and at further constrictions, indicating pseudohyphal growth mode. Stationary-phase cells (24 h) are more elongated, with <br /> location of septa indicating a mix of pseudohyphae and hyphae. <br />  <br />  <br />  <br /> sion levels as the control strain KTY3 in the presence of doxy-                   parental terms sulfur metabolic process, nitrogen compound <br /> cycline, indicating that this drug did not affect the expression of               metabolic process, amino acid and derivative metabolic pro- <br /> genes monitored by Northern blotting under the conditions used.                   cess, and organic acid metabolic process (Table 3, cluster I). <br />    The differentially regulated genes were further analyzed for                   Following in significance were several daughter terms of the <br /> significantly overrepresented GO terms of the ontology biolog-                    GO terms transport (ion, lipid, oligopeptide, vitamin, and basic <br /> ical process. By applying a P value of &#11021;0.01, eight GO term                       amino acid transport) and response to stimulus (response to <br /> clusters could be distinguished (I to VIII) (Table 3). The high-                  temperature stimulus and response to oxidative stress), al- <br /> est significance was obtained for the cluster containing the                      though both parental terms were not significantly overrepre- <br />                                                                                   sented (P values of 0.3 and 0.31, respectively). <br />                                                                                      Analysis of the complex cluster I revealed that all four pa- <br />    TABLE 2. Propidium iodide staining of Cdc53p-depleted cells                    rental terms include as most significant daughter term the GO <br />               (KTY31 doxycycline) and controls <br />                                                                                   term amino acid metabolic process or its daughter term sulfur <br />                                  % Cells dead or alivea at:                       amino acid metabolic process. Considering that Cdc53p is the <br />    Strain            24 h            48 h             72 h             96 h       backbone of several SCF ligase complexes, downregulation of <br />                Dead     Alive   Dead    Alive   Dead     Alive   Dead     Alive   the CDC53 gene would most likely result in a defect in ubiq- <br />                                                                                   uitination, therefore either stabilizing specific proteins (includ- <br /> KTY3             0      100      0      100       0      100       0      100 <br />                                                                                   ing transcriptional regulators) since they would be no longer <br /> KTY3 Dox         0      100      0      100       0      100       0      100 <br /> KTY31            0      100      0      100       0      100       0      100     marked for proteasomal degradation or failing to activate them <br /> KTY31 Dox        3       97      8       92      15       85      28       72     by monoubiquitination. To analyze the transcriptional re- <br />   a <br />     The numbers of dead (propidium iodide-positive) and alive (propidium          sponse of the amino acid metabolism genes to Cdc53p deple- <br /> iodide-negative) cells are given as a percentage. Dox, doxycycline.               tion, we searched the Candida Genome Database for known <br />  762     TRUNK ET AL.                                                                                                                  EUKARYOT. CELL <br />  <br />  <br />                                                                          transport genes, orf19.2006.1 and orf19.4967, are homologues <br />                                                                          of the S. cerevisiae genes COX17 and COX19, respectively, <br />                                                                          coding for two putative metallochaperones involved in the <br />                                                                          transfer of copper to the cytochrome c oxidase. The upregu- <br />                                                                          lation of these genes under CDC53-repressed conditions indi- <br />                                                                          cates an important role for Cdc53p in regulating intracellular <br />                                                                          copper transport and iron acquisition (see Discussion). <br />                                                                             Even though Cdc53p-depleted cells grew clearly as <br />                                                                          pseudohyphae at the time point of cell harvest and RNA prep- <br />                                                                          aration, neither the GO term filamentous growth nor any of its <br />                                                                          daughter terms were significantly overrepresented. However, <br />                                                                          we identified several differentially regulated genes linked to <br />                                                                          that category (Table 6). Upregulation was detected for the <br />                                                                          genes ALS2, ALS4, and UTR2, which encode adhesion mole- <br />                                                                          cules and are part of a set of genes shown to be upregulated in <br />                                                                          filamenting cells (1, 78). Differential regulation was also iden- <br />                                                                          tified for two transcription factors involved in filamentation, <br />                                                                          CPH2 and CZF1 although, surprisingly, these genes were <br />                                                                          downregulated by twofold, and no upregulation of any of the <br />                                                                          remaining known transcriptional activators of filamentation <br />                                                                          could be detected. However, we detected a strong upregulation <br />                                                                          of STE11, the potential MAPKKK in the filamentation MAPK <br />                                                                          pathway (15), whose downstream effector is the transcriptional <br />                                                                          regulator Cph1p. Interestingly, the forkhead transcription fac- <br />                                                                          <span id='am-83' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-84'  ></span><span id='am-85' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>tor</span> FKH2, as well as the pololike kinase CDC5, were down- <br />  <br />  <br />  <br />                                                                               TABLE 3. GO terms of the ontology biological process that are <br />    FIG. 4. Northern blot validation of microarray results. (A) Upregu-             significantly overrepresented by Cdc53p depletion <br /> lation of indicated genes comparing strains KTY3 and KTY31 in the <br />                                                                                                                                     No. of <br /> absence or the presence of doxycycline. ACT1 expression was used as      Cluster       GO term description               Pa       regulated <br />                                                                                                                                             Background <br /> loading control. (B) Downregulation of indicated genes comparing                                                                            frequencyc <br />                                                                                                                                    genesb <br /> strains KTY3 and KTY31 in the absence or presence of doxycycline. <br /> ACT1 expression was used as a loading control.                            I        Sulfur metabolic process        <br />                                                                                    Nitrogen compound               <br />                                                                                      metabolic process <br />                                                                                    Amino acid and derivative        <br /> transcriptional regulators of those genes (http://www                                metabolic process <br /> .candidagenome.org). Interestingly, we found that 14 of the 36                     Organic acid metabolic           <br /> genes in that category are regulated by the transcriptional                          process <br />                                                                           II       Transport                        <br /> regulator Gcn4p under amino acid starvation conditions (68),                       Ion transport                     <br /> with all 14 genes being exclusively upregulated in Cdc53p-                         Lipid transport                <br /> depleted cells (Table 4), suggesting that Cdc53p negatively                        Oligopeptide transport         <br /> regulates Gcn4p in C. albicans as in S. cerevisiae (33) and that                   Vitamin transport              <br />                                                                                    Basic amino acid transport   <br /> failed degradation of Gcn4p could be responsible for the <br />                                                                           III      Response to stimulus           <br /> amino acid starvation-like transcriptional response caused by                      Response to temp            <br /> C. albicans Cdc53p depletion (see Discussion).                                       stimulus <br />    The second most significantly overrepresented GO term                           Response to oxidative           <br /> cluster is represented by the parental term transport with five                      stress <br />                                                                           IV       Respiratory chain complex       <br /> significantly overrepresented subcategories totaling 35 genes                        IV assembly <br /> (Table 5), suggesting a major role for Cdc53p in regulating the           V        Ethanol metabolic process     <br /> homeostasis of the cell in response to its environment. Inter-            VI       Cell separation during       <br /> estingly, 13 of the 19 ion transport genes belonged to the                           cytokinesis <br />                                                                           VII      Fermentation                    <br /> subcategory metal ion transport, of which 11 were upregulated. <br />                                                                           VIII     Chromatin assembly or            <br /> Further analysis of these genes identified nine genes poten-                         disassembly <br /> tially involved in the copper-dependent reductive iron trans-               a <br />                                                                               That is, the significance of overrepresentation of listed GO terms comparing <br /> port across the membrane: the ferric reductases CFL2 and                 the transcriptional profiles of strains KTY31 to KTY3 in the presence of doxy- <br /> CFL5, the plasma membrane multicopper oxidases FET3 and                  cycline. <br />                                                                             b <br /> FET34, the high-affinity iron permeases FTR1 and FTR2, the                    That is, the number of differentially regulated genes belonging to the listed <br />                                                                          GO terms. <br /> cytosolic metallochaperone ATX1, and the copper transporters                c <br />                                                                               That is, the number of genes belonging to listed GO terms on a genome-wide <br /> CTR1 and P-type ATPase CRP1 (Table 5). Two more ion                      level. <br />  VOL. 8, 2009                                                                                               FUNCTION OF C. ALBICANS CDC53                 763 <br />  <br />  <br />   TABLE 4. Regulation by known transcription factors of genes differentially expressed upon Cdc53p depletion of the GO term amino acid <br />                                                            metabolic process <br />                No. of <br />                            Regulation of <br /> Gene         regulated                                                                                                                             Regulation <br />                             KTY31 vs                                         Gene name (open reading frame no.) <br /> group          genes                                                                                                                                 typeb <br />                               KTY3 <br />              (n &#11005; 36)a <br />  <br />   1             14             Up              ARG1 (orf19.7469), ARG3 (orf19.5610), ARO8 (orf19.2098), ARG8 (orf19.3770), ARG5,6                   Gcn4p <br />                                                  (orf19.4788), ECM42 (orf19.6500), MET15 (orf19.5645), LYS9 (orf19.7448), LYS22 <br />                                                  (orf19.4506), orf19.2092, MET6 (orf19.2551), BAT21 (orf19.797), BAT22 (orf19.6994), <br />                                                  HIS7 (orf19.5505) <br />   2              4             Up              MET1 (orf19.5811), MET3 (orf19.5025), CYS3 (orf19.6402), MAE1 (orf19.3419)                           Other <br />   3              3             Down            PUT1 (orf19.4274), CAR1 (orf19.3934), CHA1 (orf19.1996) <br />   4              9             Up              CPA1 (orf19.4630), orf19.5842, orf19.3221, ZCF33 (orf19.5992), ARG4 (orf19.6689),                    ? <br />                                                  MET14 (orf19.946), orf19.1159, LEU2 (orf19.7080), IDP1 (orf19.5211) <br />   5              6             Down            CBF1 (orf19.2876), MET28 (orf19.7046), CAR2 (orf19.5641), GAD1 (orf19.1153), PUT2 <br />                                                  (orf19.3974), ARO10 (orf19.1847) <br />   a <br />       That is, the number of differentially regulated genes belonging to the listed GO terms. <br />   b <br />       That is, the regulating factor for this group of genes. &#8220;Other&#8221; includes Hog1p, Rim101p, Ssn6p, Sfu1p, Nrg1p, Mig1p, and Cap1p. <br />  <br />  <br />  <br /> regulated by twofold; deletion of these genes results in consti-                    our GO term analysis presented above are FAD3 (orf19.4993) <br /> tutive pseudohyphal growth (9) or elongated buds (4),                               and FAD2 (orf19.118), which were upregulated upon Cdc53p <br /> respectively. The constitutive filamentous growth phenotype of                      depletion by six- and twofold, respectively. These genes encode <br /> Cdc53p-depleted cells together with the differential regulation                     fatty acid desaturases that are involved in &#9251;-linolenic acid and <br /> of several filamentation-associated genes indicates an impor-                       linoleic acid synthesis, respectively, which in turn are major <br /> tant role for Cdc53p in regulating growth morphology in C.                          membrane components (49). Furthermore, we detected strong <br /> albicans (see Discussion).                                                          downregulation of the chitinases CHT2 (orf19.3895, 6-fold) <br />    Whereas classification by GO terms is valuable for the de-                       and CHT3 (orf19.7586, 17-fold), two important factors in the <br /> termination of Cdc53p-dependent pathways, analysis of indi-                         homeostasis of chitin, a cell wall component that contributes to <br /> vidual differentially transcribed genes can also yield important                    its mechanical strength and is enriched at septa (44). The <br /> clues in the involvement of Cdc53p activity in different cellular                   differential regulation of these genes, together with transcrip- <br /> processes (see Table S2 in the supplemental material). The                          tional changes (in both directions) of several more genes, <br /> strongest upregulation in Cdc53p-depleted cells was detected                        which affect membrane composition, cell wall integrity, and <br /> for AOX2 (orf19.4773, 195-fold), an alternative oxidase that is                     cell-cell interaction (e.g., genes coding for glycosylphosphati- <br /> involved in the cyanide-resistant respiratory pathway (30). Two                     dylinositol-anchored proteins), suggests that Cdc53p is a major <br /> further differentially transcribed genes that were not part of                      regulator of fungal growth, development, and survival. <br />  <br />  <br />                      TABLE 5. Subclassification of the GO term transport of the transcriptional profile of Cdc53p-depleted cells <br />                                                     No. of                    Regulation of <br />                                                                Background <br />             GO term                    Pa         regulated                    KTY31 vs                       Gene name (open reading frame no.) <br />                                                                frequencyc <br />                                                    genesb                        KTY3 <br />  <br /> Transport                           0.3              54           687 <br /> Ion transport                       1.09E-05         19            90 <br />   Metal ion transport               1.08E-04         11            57             Up           CFL2 (orf19.1264), CTR1 (orf19.3646), FTR1 (orf19.7219), <br />                                                                                                  CRP1 (orf19.4784), orf19.2006.1, FET3 (orf19.4211), <br />                                                                                                  FET34 (orf19.1206), COX19 (orf19.4967), CFL5 <br />                                                                                                  (orf19.1930), FTR2 (orf19.7231), ATX1 (orf19.2369.1) <br />                                                       2                           Down         orf19.4940, ENA21 (orf19.5170) <br />   Other                                               4                           Up           PHO87 (orf19.2454), SSU1 (orf19.7313), SUL2 <br />                                                                                                  (orf19.2738), MEP1 (orf19.1614) <br />                                                       2                           Down         orf19.2959.1, PHO84 (orf19.1172) <br /> Lipid transport                     4.56E-04          1             16            Up           orf19.4342 <br />                                                       5                           Down         RTA2 (orf19.24), GIT2 (orf19.1978), RTA3 (orf19.23), <br />                                                                                                  GIT1 (orf19.1979), orf19.34 <br /> Oligopeptide transport              7.94E-04          5             12            Down         OPT4 (orf19.176), PTR2 (orf19.6937), IFC3 (orf19.3749), <br />                                                                                                  OPT9 (orf19.2584), OPT1 (orf19.2602) <br /> Vitamin transport                   1.21E-03          1              4            Up           TNA1 (orf19.4335) <br />                                                       2                           Down         PHO113 (orf19.2619), PHO112 (orf19.3727) <br /> Basic amino acid transport          9.08E-03          1              7            Up           CAN2 (orf19.111) <br />                                                       2                           Down         ALP1 (orf19.2337), orf19.4940 <br />   a <br />     That is, the significance of overrepresentation of the listed GO terms comparing the transcriptional profiles of strains KTY31 to KTY3 in the presence of <br /> doxycycline. <br />   b <br />     That is, the number of differentially regulated genes belonging to the listed GO terms. <br />   c <br />     That is, the number of genes belonging to the listed GO terms on a genome-wide level. <br />  764     TRUNK ET AL.                                                                                                                            EUKARYOT. CELL <br />  <br />  <br />             TABLE 6. Subclassification of the GO term filamentous growth of the transcriptional profile of Cdc53p-depleted cells <br />                                               No. of                         Regulation of <br />                                                            Background <br />          GO term                   Pa       regulated                         KTY31 vs                         Gene name (open reading frame no.) <br />                                                            frequencyc <br />                                              genesb                             KTY3 <br />  <br /> Filamentous growth                0.82          23             323 <br /> Invasive growth (sensu            0.08           1              15              Up              ALS4 (orf19.4555) <br />   <span id='am-30' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32'  ></span><span id='am-33' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span>Saccharomyces</span>) <br />                                                  2                              Down            CDC14 (orf19.4192), GAL10 (orf19.3672) <br /> Pseudohyphal growth               0.10           2              36              Up              STE11 (orf19.844), ALS4 (orf19.4555) <br />                                                  3                              Down            FKH2 (orf19.5389), ASH1 (orf19.5343), CPH2 (orf19.1187) <br /> Signal transduction during        0.43           1                8             Down            CDC5 (orf19.6010) <br />   filamentous growth <br /> Hyphal growth                     0.73           3             139              Up              ALS2 (orf19.1097), UTR2 (orf19.1671), ALS4 (orf19.4555) <br />                                                  5                              Down            FKH2 (orf19.5389), MCM1 (orf19.7025), CDC14 <br />                                                                                                   (orf19.4192), CPH2 (orf19.1187), SSK1 (orf19.5031) <br /> Other                                            4                              Up              SSU1 (orf19.7313), PHHB (orf19.2079), FGR3 <br />                                                                                                   (orf19.3845), FGR43 (orf19.4786) <br />                                                  7                              Down            CLB4 (orf19.7186), CZF1 (orf19.3127), orf19.6720, ECM4 <br />                                                                                                   (orf19.2613), GAT2 (orf19.4056), CHA1 (orf19.1996), <br />                                                                                                   FGR41 (orf19.4910) <br />  a <br />    That is, the significance of overrepresentation of listed GO terms comparing the transcriptional profiles of strains KTY31 to KTY3 in the presence of doxycycline. <br />  b <br />    That is, the number of differentially regulated genes belonging to the listed GO terms. <br />  c <br />    That is, the number of genes belonging to the listed GO terms on a genome-wide level. <br />  <br />  <br />  <br />                              DISCUSSION                                              zation of Sic1p (3). The finding that C. albicans Cdc53p is <br />                                  Cdc4                                                essential, coupled to the fact that proteins which are part of <br />    In S. cerevisiae, the SCF        complex is responsible for de- <br />                                                                                      SCF-E2 complexes are not (e.g., Cdc4p and Grr1p [3, 39]), <br /> grading the cell cycle inhibitor Sic1p (75). Mutants in one of <br />                                                                                      suggests that the essentiality of Cdc53p is potentially caused by <br /> the proteins needed for ubiquitination-dependent degradation <br />                                                                                      the combined dysfunction of several/all SCF ligases (synthetic <br /> of Sic1p, including Cdc53p, display multiple elongated buds <br />                                                                                      lethal effect), highlighting the importance of Cdc53p in C. <br /> and arrest in the G1 phase (24, 43). Our study indicates that <br />                                                                                      albicans cell physiology. However, it is also possible that <br /> repression of CDC53 in C. albicans rather results in cell elon- <br />                                                                                      Cdc53p essentiality results from its interaction with a single <br /> gation and subsequent pseudohypha formation (Fig. 3C), the <br />                                                                                      essential protein that remains to be identified. The genetic <br /> cells probably evading immediate cell cycle arrest by switching <br />                                                                                      analysis of other components of potential SCF-E2 complexes <br /> to a filamentous growth mode. This is in accordance with the <br />                                                                                      in C. albicans will help in answering this question. <br /> demonstrated ability of C. albicans to form polarized evagina- <br />                                                                                         The identification of the pathway involved in the Cdc53p- <br /> tions in all stages of the cell cycle, therefore uncoupling hyphal <br /> elongation and cell cycle morphogenesis programs (25). How-                          dependent filamentation by transcriptional profiling was lim- <br /> ever, the occurrence of a mix of pseudohyphae and real hyphae                        ited, since no general transcriptional filamentation response <br /> after 24 h of growth under CDC53 repressing conditions that                          could be detected (35, 48, 51). In fact, the transcripts of CPH2 <br /> seem to be defective in cytokinesis and septation, as can be                         and CZF1, two genes coding for transcriptional activators of <br /> observed by the missing nuclei and septa in/between part of the                      filamentous growth (14, 36), were found to be downregulated <br /> filaments (Fig. 3C), suggests that finally cell death prevails.                      in Cdc53p-depleted cells (Table 6). On the other hand, we <br /> This proposition is strengthened by the growth phenotype of                          detected a strong upregulation of STE11 encoding a MAPKKK <br /> single colonies depleted in Cdc53p, which stop growing after                         (fivefold), indicating a possible involvement of the MAPK <br /> 24 h (Fig. 2), as well as the results of the propidium iodide                        pathway in filamentation via its downstream effector Cph1p. <br /> staining, which confirmed an earlier onset of cell death for                         Although no differential regulation for the majority of Cph1p- <br /> Cdc53p-depleted cells compared to the controls (Table 2). In                         dependent genes, e.g., HWP1, HYR1, and ECE1 (35), was <br /> addition, we were unsuccessful in creating a homozygous                              identified in the CDC53 repressed strain by genome-wide tran- <br /> knockout mutant, with both alleles of CDC53 deleted, a fur-                          scriptional profiling, HWP1 could be detected as upregulated <br /> ther evidence of the essentiality of C. albicans CDC53, similar                      by Northern blot analysis using the same RNA preparations as <br /> to S. cerevisiae CDC53 (43). This growth defect of Cdc53p-                           for the microarray hybridization experiment (data not shown). <br /> depleted cells is specific for the cullin moiety of the C. albicans                  Hence, the highly stringent criteria used for the DNA microar- <br /> SCFCdc4 complex, since C. albicans homozygous cdc4&#11002;/&#11002; cells                          ray data analysis could have caused some filamentation-depen- <br /> grow as air-protruding filaments (3). The different growth re-                       dent genes to evade detection (HWP1 and HYR1 were upregu- <br /> sponse of C. albicans and S. cerevisiae to Cdc53p depletion, as                      lated by 1.2- and 1.4-fold, respectively; see Table S2 in the <br /> well as the viability of C. albicans cdc4&#11002;/&#11002; cells compared to                       supplemental material). Therefore, the involvement of this <br /> the essentiality of the ScCDC4 gene may be related to the                            pathway in the filamentation response of Cdc53p-depleted <br /> different functions of the cell cycle inhibitor Sic1p of S. cerevi-                  cells cannot be excluded. Transcriptional changes of other fac- <br /> siae and its homologue in C. albicans, Sol1p, since stabilization                    tors that are consistent with the pseudohyphal growth pheno- <br /> of Sol1p leads to only a slight delay in G1-S phase progression,                     type of the conditional CDC53 strain are the downregulation <br /> contrary to the cell cycle arrest in G1 phase caused by stabili-                     of the forkhead transcription factor FKH2 (9), as well as the <br />  VOL. 8, 2009                                                                           FUNCTION OF C. ALBICANS CDC53             765 <br />  <br />  <br /> downregulation of the pololike kinase CDC5 (4) although,             12 sulfur utilization genes (P &#11005; 4.68E&#11002;06), a daughter GO <br /> apart from downregulation of the histone machinery (Table 3,         term of the sulfur metabolic process (Table 3, cluster I), pos- <br /> group VIII), the expression profile of the cdc5 mutant does not      sibly due to changes in the availability of the as-yet-uncharac- <br /> correlate with the one obtained upon Cdc53p depletion. Con-          terized potential SCF-target CaMet28p, the homolog of <br /> sidering these results, it seems less likely that a single tran-     ScMet4p. Regarding the GO term cell wall organization, only <br /> scriptional activation pathway is responsible for the filamenta-     minor correlation between the Varelas et al. study (72) and the <br /> tion phenotype of Cdc53p-depleted cells but rather that a            present study could be detected. Furthermore, the significance <br /> combination of different effects caused by the failed degrada-       of this GO term being affected by Cdc53p depletion in C. <br /> tion and/or activation of several proteins.                          albicans (3.33E&#11002;01) was described above our cutoff level, <br />    A very clear transcriptional response to Cdc53p depletion         rendering it nonsignificant. However, we detected differential <br /> was obtained with the GO term amino acid metabolic process,          regulation of genes in C. albicans, which affect membrane <br /> similar to the transcriptional profile of amino acid-starved         composition and cell wall integrity, including glucan 1,3-&#9252;- <br /> cells. In C. albicans, the amino acid starvation response is         glucosidases (ENG1 [orf19.3066], XOG1 [orf19.2990], SCW11 <br /> directly dependent on the transcriptional regulator Gcn4p, a         [orf19.3893], and EXG2 [orf19.2952]; P value of 2.2E&#11002;03) and <br /> functional homologue of ScGcn4p (69). Whereas in S. cerevi-          the two chitinases CHT2 (orf19.3895) and CHT3 (orf19.7586), <br /> siae the amino acid starvation response is regulated mainly at       suggesting that Cdc53p depletion affects certain aspects of <br /> the translational level by the eIF2&#9251; kinase ScGcn2p (reviewed        membrane rearrangements in C. albicans. Concerning the re- <br /> in references 27 and 28), activation of the amino acid starva-       maining GO terms vitamin metabolism, sodium transport, cell- <br /> tion response in C. albicans depends mainly on the transcrip-        cell adhesion, sporulation, purine base metabolism, heavy <br /> tional regulation of the GCN4 gene (68). The activity of             metal ion homeostasis, and signal transduction of mating sig- <br /> ScGcn4p is further regulated at the posttranslational level by       nals as reported in the Varelas et al. study (72), no similar <br /> proteasomal degradation, mediated by phosphorylation by the          response could be detected in our study. Finally, the identifi- <br /> Cyclin Pcl5-dependent kinase Pho85 (64) and subsequent               cation of the following categories unique to Cdc53p depletion <br /> ubiquitination by the SCFCdc4-Cdc34p complex (45). Similarly,        in C. albicans, namely, transport, response to stimulus, respi- <br /> CaGcn4p, which is produced in rich medium, is rapidly de-            ratory chain complex IV assembly, ethanol metabolic process, <br /> graded with a half-life of less than 5 min in response to amino      cell separation during cytokinesis, fermentation, and chroma- <br /> acid availability (23, 69). Although it has been shown that          tin assembly or disassembly (Table 3), indicates that depletion <br /> degradation of CaGcn4p is also dependent on phosphorylation          and/or mutation of the cullin subunit of the SCF complex <br /> by the cyclin Pcl5-dependent kinase Pho85 (23), the ubiquitin        affects different sets of genes or pathways in C. albicans com- <br /> ligase involved in that process remains to be identified. The        pared to S. cerevisiae. <br /> transcriptome analysis presented here revealed upregulation of          Of major importance for its survival in the host and its <br /> amino acid starvation response genes under Cdc53p-depleted           pathogenicity is the ability of C. albicans to acquire iron from <br /> conditions, a finding consistent with an increase in Gcn4p           the iron-limiting host environment (55). C. albicans uses for <br /> activity. Since the fluorescence signal intensities indicate high    that purpose at least three independent high-affinity iron up- <br /> abundance of the GCN4 transcript in rich medium without a            take systems: a high-affinity iron permease, an iron-sid- <br /> change in transcript levels between the wild-type and the            erophore uptake system, and a heme uptake system (reviewed <br /> Cdc53p-depleted cells, a potential increase in Gcn4p activity        in reference 67). Interestingly, we found that the second most <br /> would appear to be mainly regulated at the protein degrada-          important GO term that was significantly affected in the tran- <br /> tion level, suggesting the involvement of a Cdc53p-containing        scriptome analysis of Cdc53p-depleted cells (transport) in- <br /> SCF complex in the degradation of Gcn4p in C. albicans.              cluded several genes involved in copper-dependent reductive <br /> Interestingly, it has been shown that amino acid starvation also     iron transport across the membrane (Table 5). These include <br /> induces a filamentation response in C. albicans (54, 69), with       the externally directed ferric reductases CFL2 and CFL5, <br /> Gcn4p stimulating pseudohyphal growth in an Efg1- and par-           which in analogy to S. cerevisiae (reviewed in reference 67) are <br /> tially Ras1p-dependent fashion, but independent of Cph1p, the        coupled to a ferrous iron transporter complex that consists of <br /> downstream effector of the MAPK pathway (69). Therefore,             a plasma membrane multicopper oxidase (FET3 or FET34) <br /> the observed filamentation of C. albicans cells depleted of          and a high-affinity iron permease (FTR1 or FTR2). Further- <br /> Cdc53p could be mediated at least <span id='am-103' about='ncbitaxon:in_part' typeof='owl:Thing'>in part</span> by nondegraded             more, activity of the multicopper ferroxidase requires efficient <br /> Gcn4p.                                                               copper uptake by the cells (2, 32, 77) involving the copper <br />    A similar Gcn4p-dependent response to mutations in the            transporter Ctr1p (42), and in analogy to S. cerevisiae, the <br /> SCF-E2 complex components CDC53 and CDC34 in S. cerevi-              metallochaperone Atx1p and the P-type ATPase Crp1p, with <br /> siae was identified in a genome-wide transcriptional analysis        the latter being a homologue of ScCcc2p (reviewed in refer- <br /> carried out by Varelas et al. (72), with the GO term amino acid      ence 62). The exclusive upregulation of all of the above-men- <br /> metabolism identified in their study, a daughter term of the         tioned genes indicates that Cdc53p negatively regulates the <br /> GO terms amino acid and derivative metabolic process and             copper-dependent reductive iron transport under iron-replete <br /> organic acid metabolic process identified in our study (Table 3,     conditions. <br /> cluster I). These authors also detected transcriptional changes         In C. albicans, the reductive iron transport system is homeo- <br /> of genes involved in sulfur utilization, which they attributed to    statically regulated in response to available iron (34). Known <br /> alterations of the transcription factor Met4p in the S. cerevisiae   regulators include Sfu1p, the homologue of the transcriptional <br /> cdc53-1 and cdc34-2 mutants (72). Likewise, downregulation of        repressor Fep1p in the filamentous <span id='am-42' about='obo:NCBITaxon_4751' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-43'  ></span><span id='am-44' property="oboInOwl:hasDbXref" content="PMID:12684019" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="fungi" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Fungi" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="PMID:11062127" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="fungi" datatype="xsd:string"></span>fungi</span> <span id='am-51' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span>Schizosaccharomyces</span> <br /> CDC53 in C. albicans resulted in differential regulation of 7 of     pombe (34, 53) and the pH-responsive transcriptional activator <br />  <br />  <br /> Rim101p (10, 17, 18). However, no general Sfu1p or Rim101p                                <br /> transcriptional response could be observed in the transcrip-                               <br /> tional profile of Cdc53p-depleted cells, rendering both factors                            <br /> unlikely to elicit the observed Cdc53p-dependent upregulation                         <br /> of the high-affinity iron transport genes. Interestingly, a novel                           <br /> transcriptional activation system for a ferric ion reductase in C.                        <br /> albicans has been recently reported that is similar to the respi-                     <br /> ratory CCAAT-binding factor regulatory system in S. cerevisiae                              <br /> (5, 31). Further studies should provide interesting information                      <br /> as to its contribution to the iron-homeostasis regulon, as well                          <br /> as its potential regulation by ubiquitination.                                        <br />   <br /> </body></html>