<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Bioinformatics                                                                                                                       BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> Mining protein networks synthetic genetic interactions <br /> Sri R Paladugu1,3, Shan Zhao2, Animesh Ray1 Alpan Raval*1,3 <br />  <br /> Address: 1Keck Graduate Institute Applied Life Sciences, 535 Watson Drive, Claremont, CA 91711, USA, 2Virtual Endoscopy Computer- <br /> Aided Diagnosis Laboratory, Department Radiology, National Institutes Health Clinical Center, Bethesda, MD 20892-1182, USA 3School <br /> of Mathematical Sciences, Claremont Graduate University, 710 N. College Avenue, Claremont, CA 91711, USA <br /> Email: Sri R Paladugu - spaladug@kgi.edu; Shan Zhao - zhaosha@mail.nih.gov; Animesh Ray - aray@kgi.edu; Alpan Raval* - araval@kgi.edu <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 9 October 2008                                                      Received: 19 March 2008 <br />                                                                                Accepted: 9 October 2008 <br /> BMC Bioinformatics 2008, 9:426   doi:10.1186/1471-2105-9-426 <br /> <span id='am-49' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-53'  ></span>This</span> article available from: http://www.biomedcentral.com/1471-2105/9/426 <br /> &#169; 2008 Paladugu et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  Abstract <br />                  Background: The local connectivity global position protein protein interaction <br />                  network known correlate functional properties, including essentiality or <br />                  dispensability. It extend observation examine network <br />                  properties proteins considered simultaneously determine joint dispensability, i.e., <br />                  propensity synthetic sick/lethal interaction. Accordingly, examine predictive power <br />                  protein interaction networks synthetic genetic interaction <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>, an <br />                  organism high confidence protein interaction networks available synthetic sick/ <br />                  lethal gene pairs extensively identified. <br />                  Results: We design support vector machine uses graph-theoretic properties two <br />                  proteins protein interaction network input features prediction synthetic sick/lethal <br />                  interactions. The trained interacting non-interacting gene pairs culled large <br />                  scale genetic screens literature-curated <span id='am-79' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81'  ></span><span id='am-82' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span>. We method capable of <br />                  predicting synthetic genetic interactions sensitivity specificity exceeding 85%. We <br />                  prediction performance reasonably robust respect errors the <br />                  protein interaction network respect changes features test datasets. Using the <br />                  prediction system, carried novel predictions synthetic sick/lethal gene pairs genome- <br />                  wide scale. These pairs appear functional properties similar that <br />                  characterize known synthetic lethal gene pairs. <br />                  Conclusion: Our analysis shows protein interaction networks used predict <br />                  synthetic lethal interactions accuracies par exceeding computational <br />                  methods use variety input features, including functional annotations. This indicates that <br />                  protein interaction networks plausibly rich sources information epistatic effects <br />                  genes. <br />  <br />  <br />  <br />  <br /> Background                                                                     emergence organization function genome <br /> Successful prediction gene function disparate data                     level, lead understanding disease mecha- <br /> sources important challenge post-genomic era.                     nisms prediction drug targets. Functional organiza- <br /> Methods illuminate new mechanisms                         tion genes dramatically revealed their <br />  <br />  <br />  <br />                                                                                                                                        Page 1 14 <br />                                                                                                                  (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:426                                              http://www.biomedcentral.com/1471-2105/9/426 <br />  <br />  <br />  <br /> positions biomolecular networks topological          lations reported earlier. Tong et.al. [20] <br /> constraints positions entail. Much work          observed synthetic genetic interactions, including <br /> been using graph properties protein interac-         synthetic lethal (SL) synthetic sick (SS, simul- <br /> tion networks (PINs) elucidate gene protein func-        taneous deletion genes causes growth retardation) <br /> tion, particularly baker&apos;s yeast <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-26'  ></span><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span>Saccharomyces</span>           interactions, prevalent genes encoding <br /> cerevisiae high quality genome-scale interaction      proteins protein complex (two more <br /> maps (or graphs) constructed. In interaction       proteins form clique) encoding <br /> graphs, nodes represent individual proteins edges           proteins complexes proteins are <br /> represent physical binding. Several previous reports      known complex. These authors also <br /> find interesting correlations network properties        addressed issue using correlation predict pro- <br /> and aspects biological function [1-10]. For example,         tein-protein interactions synthetic genetic interac- <br /> pairwise correlations protein           tions, concluded predictive value limited <br /> degree/centrality, gene essentiality, evolutionary rate     (approximately 1%) gene pairs encode pro- <br /> [1-4]. Other methods attempt uncover sets genes           teins members complex. They further <br /> implicated common function, i.e., functional mod-          number common neighbors two <br /> ules, network structure [5-9]. Yet methods           genes synthetic genetic network correlates the <br /> combine graph-theoretic data data sources,           existence protein-protein interaction cor- <br /> such expression levels [11] functional annotation      responding gene products. It understood pro- <br /> neighboring proteins interaction map [12-16],         teins encoded genes having synthetic genetic <br /> further elucidate function.                                     interaction enriched proteins lie sepa- <br />                                                                 rate pathways opposed pathway (see, for <br /> One earliest observed correlations graph-      example, [9,17]), indicating preference for <br /> theoretic property functional attribute well-      protein network position products synthetic <br /> known correlation degree (i.e., number       gene pairs. In spite studies, open question <br /> interaction partners &quot;hubness&quot;) protein yeast      information PINs for <br /> PIN essentiality corresponding protein-cod-      determining synthetic lethal interactions. The conceptual <br /> ing gene [1,17] (an essential gene produces       basis expectation biological function is <br /> lethal phenotype deleted). Correlations            ultimately defined interaction proteins with <br /> essentiality graph properties, including various      proteins, DNA/RNA, small molecules (metabo- <br /> centrality measures, reported yeast [2-4]     lites). Since protein-DNA/RNA protein-metabo- <br /> and organisms nematode worm                   lite interactions directly indirectly influenced by <br /> <span id='am-30' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32'  ></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span>Caenorhabditis elegans</span> fruit fly <span id='am-42' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-48'  ></span>Drosophila melanogaster</span>    protein-protein interactions, reasonable the <br /> [4]. The principal observation emerging stud-        structure PIN encode sufficient information <br /> ies essential proteins tend over-represented      interaction networks. However, encoding <br /> among proteins high degrees high centralities       subtle simple linear statisti- <br /> a PIN. This leads hypothesis indispensabil-     cal correlations envisaged explicit analytical models. <br /> ity given protein cellular function par- <br /> tially determined local connectivity                Arguably systematic accurate method date <br /> (quantified degree) global position (quantified      genome-wide prediction synthetic sick lethal <br /> by centrality measures) PIN [2,1].                   (SSL) interactions carried Wong et al. [21] in <br />                                                                 Saccharomyces cerevisiae using multiple input features, <br /> Synthetic lethal interactions important genetic interac-    including protein interactions, gene expression, func- <br /> tions understanding biological function poten-      tional annotation, sequence motifs. The method can <br /> tially developing novel classes drug targets [18]. Two       used streamline SSL discovery process it <br /> genes said participate synthetic lethal interac-    require experimentally test 20% of <br /> tion simultaneous deletion mutations both,        pairs discover SSL pairs 80% success rate. But <br /> in alone, cause lethality. Thus, concept syn-    method relies disparate data sources (viz., expres- <br /> thetic lethality viewed extension essenti-      sion data, subcellular localization, physical complexes) <br /> ality. Indeed, recently proposed type      readily available organisms than <br /> of genetic redundancy conferred synthetic lethality          S. cerevisiae. Furthermore, success method does <br /> leads redundancy prevalent pro-             answer question posed previous paragraph. <br /> teins highly connected highly central PINs <br /> [19]. This finding implies correlations PIN graph       Here study, S. cerevisiae, extent SSL <br /> properties synthetic lethality similar correla-      interaction genes predicted solely <br /> tions essentiality single genes. In fact, corre-   topological properties corresponding pro- <br />  <br />  <br />                                                                                                                     Page 2 14 <br />                                                                                               (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:426                                                 http://www.biomedcentral.com/1471-2105/9/426 <br />  <br />  <br />  <br /> teins PIN knowledge SSL inter-         network including r: (n - 1)(n - 2), n the <br /> actions genes question participate in. Our            number proteins PIN interest. It essentially <br /> motivation stems expectations                   measures fraction network shortest paths a <br /> described above, observation           given protein lies on. <br /> yeast protein interaction network formed literature <br /> curated interactions [22], composed ~3300 proteins             (e) Eigenvector centrality [27]: Let v denote (row or <br /> and ~12000 interactions, substantially larger         column) index particular protein adjacency <br /> size recent synthetic genetic network formed          matrix corresponding PIN. Then eigenvector cen- <br /> by literature curated high-throughput screen interac-         trality protein defined vth element the <br /> tions (~1000 genes ~7000 interactions [20,23,22]),            principal eigenvector adjacency matrix. This princi- <br /> thus potentially enabling discovery new SSL inter-         pal eigenvector normalized largest entry is <br /> actions. Strikingly, best overall accuracy       1. This centrality measure connected a <br /> for SSL prediction protein interaction data compa-        protein highly connected proteins network. <br /> rable higher combination dis- <br /> parate inputs [21], confirming predictive power          (f) Stress centrality: absolute number network <br /> of PINs SSL interactions opening possibil-         shortest paths pass protein r. <br /> ity predicting SSL network organisms <br /> where genome-scale protein interaction networks              (g) Bridging centrality [28]: The bridging centrality a <br /> been found.                                                       protein (r) defined product betweenness <br />                                                                   centrality bridging coefficient protein. <br /> Methods                                                           While betweenness centrality measures fraction of <br /> Protein Interaction Network (PIN) data                            network shortest paths given protein lies on, bridg- <br /> For computing network properties associated pro-             ing coefficient measures extent protein is <br /> tein-protein interactions yeast, used literature        lying densely connected proteins net- <br /> curated protein interactions Saccharomyces cerevisiae          work. Let dr Nr represent degree set of <br /> culled Reguly et.al [22]. This network contains 3289           neighbors protein r. Then bridging coefficient <br /> proteins 11334 interactions.                                  (BC) protein r defined as <br />  <br />                                                                                                         d r&#8722;1 <br /> PIN graph-theoretic properties used predicting SSLs <br /> For use inputs Support Vector Machine (SVM) clas-                            BC(r ) =                              . <br />                                                                                                          &#8722;1 <br /> sifier, computed following PIN graph-theoretic                                           &#8721;i&#8712;N r d i <br /> properties protein. <br />                                                                   (h) Information centrality [29]: Let A adjacency <br /> (a) Degree: number direct interactions          matrix PIN, D diagonal matrix degrees of <br /> proteins.                                                         protein, J matrix elements equal to <br />                                                                   1. Let B = D - A + J let C = B-1. This construction yields <br /> (b) Clustering coefficient [24]: fraction possible         information matrix I elements Iij = (Cii + Cjj - Cij)- <br /> interactions direct neighbors protein           1. The information centrality IC(i) protein then <br />  <br /> PIN.                                                              defined harmonic mean: <br />  <br /> (c) Closeness centrality [25]: Let d(p, q) shortest dis-                                                       &#8722;1 <br />                                                                                                &#9115;                  &#9118; <br /> tance proteins p q PIN. Then close- <br /> ness centrality protein p defined (n - 1)/&#931;q d(p, q), <br />                                                                                      IC(i) = n &#9116; <br />                                                                                                &#9116;   &#8721;     I ij&#8722;1   &#9119; <br />                                                                                                                   &#9119; <br />                                                                                                                            . <br /> where n total number proteins PIN. It                                         &#65533;?    j             &#9120; <br /> therefore measures extent protein p close         As recognized [30], measure essentially measures <br /> to proteins PIN.                                 mean lengths paths ending protein i. A similar <br />                                                                   interpretation given [31], showed infor- <br /> (d) Normalized betweenness centrality [26]: Let &#963;p,q       mation centrality identical current-flow closeness <br /> number shortest paths proteins p q            centrality. <br /> PIN, let &#963;p,q(r) number shortest paths <br /> between p q pass protein r. Then             (i) Current-flow betweenness centrality [31]: This central- <br /> betweenness centrality r defined &#931;&#963;p,q(r)/&#963;p,q,          ity measure generalization standard between- <br /> where sum taken distinct pairs p q. We        ness centrality index takes account just <br /> normalize measure lie 0 1 dividing         shortest paths paths well. Its <span id='am-78' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> is <br /> the betweenness centrality total number pairs        obtained definition betweenness centrality by <br />  <br />  <br />                                                                                                                                Page 3 14 <br />                                                                                                    (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:426                                                    http://www.biomedcentral.com/1471-2105/9/426 <br />  <br />  <br />  <br /> replacing &#963;p,q(r)/&#963;p,q &#964;p,q(r), throughput           scale genetic interaction screen [20] non- <br /> node r [31]. It related distance traversed ran-      essential genes yeast protein products not <br /> dom walk network ends particular pro-           localized mitochondria removing this <br /> tein node.                                                          dataset SSL interactions confirmed high through- <br />                                                                     experimental methods. The resulting <br /> All properties properties single nodes          number SSL pairs non-SSL pairs obtained <br /> in network. The sensitive local         literature curated high throughput methods are <br /> network structure node,             presented Table 1. It important bear mind that <br /> are sensitive global network topology. Furthermore,          inferred non-SSL pairs SSL <br /> the properties depend shortest paths           errors high throughput screen. This <br /> through network, paths well.                    explains decrease number non-SSL pairs <br />                                                                     inclusion literature curated (LC) data. For the <br /> In addition 9 single-node properties,          combined data literature curated genome wide <br /> computed set two-node properties, namely,                  screens, generated probability distributions of <br /> inverse shortest distance d(p, q) proteins p         PIN graph-theoretic property discussed above, sepa- <br /> q, number mutual neighbors proteins p q,             rately SSL non-SSL pairs. Probability distributions <br /> and <span id='am-1' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Indicator" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5'  ></span>indicator</span> variables 2Hop S-S 2Hop S-P,                  converted histograms smoothed probability <br /> which exploit fact known synthetic genetic             density functions using Gaussian smoothing imple- <br /> network contains large number triangles [21]. 2Hop             mented &apos;density&apos; function R statistical com- <br /> S-S takes value 1 genes encoding proteins          puting platform. <br /> p q share synthetic lethal partner 0 otherwise, <br /> whereas 2Hop S-P takes value 1 exists protein          Support vector machine classifier <br /> r r physical interaction protein p           We use support vector machines (SVMs) model correla- <br /> the gene corresponding protein r SSL interaction           tions PIN properties existence SSL <br /> with protein q vice versa. We computed properties (a)-           interaction. Various graph-theoretical properties (local as <br /> (d) using network analysis tool Pajek [32], (e) (f)         global) proteins PIN fed inputs <br /> using SNA package R statistical computing plat-         SVM classifier, schematically represented <br /> form [33], (g) based formula given [28], (h)          Figure 1. <br /> and (i) using implementation algorithm given <br /> in [31].                                                            The output SVM classifier score measures <br />                                                                     propensity corresponding genes partake <br /> Synthetic genetic interaction data                                  SSL interaction. High scores indicate higher propensi- <br /> Our primary data sources training testing SVM           ties SSL interactions, low scores indicate higher <br /> classifier (described below) literature curated            propensities lack SSL interaction. We that <br /> genetic interactions [22] large-scale genetic              SVMs highest overall accuracy compared to <br /> interaction screens [20,23], filtered SSL interactions.      prediction systems experimented with, <br /> From sources extracted pairs genes           including naive Bayes classifiers neural networks. In <br /> whose protein products literature                 study, used libSVM, publicly available integrated <br /> curated protein interaction network [22], resulting         software support vector classification [34]. The full <br /> dataset 4553 confirmed SSL pairs [20,23],               prediction 22 inputs lumped single vec- <br /> 7020 pairs combined literature curated SSL                <span id='am-62' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-64'  ></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="Tor" datatype="xsd:string"></span>tor</span> x: averages absolute differences 9 prop- <br /> interactions [22]. We excluded gene pairs             erties (a)-(i) protein pair, shortest distance <br /> protein products localized mitochondria &#8211;              d(p, q) proteins, number mutual <br /> was necessary initial results showed         neighbors proteins PIN, indica- <br /> difficult distinguish synthetic sick mutants yeast          tor variables 2Hop S-S 2Hop S-P. The raw output SVM <br /> mutants slow growth conferred                   scores assigned protein pairs SVM <br /> absence single (as opposed two) mitochondrial <br />                                                                     Table 1: Statistics known SSL pairs yeast <br /> protein. This resulted dataset 3962 pairs con- <br /> firmed SSL interactions [20,23], 6074 pairs                                Lethal/Sick          Non-Lethal/Non-Sick <br /> confirmed SSL interactions included pairs the <br /> literature curated genetic interactions [22]. In <span id='am-54' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-55' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span>            HTP                3962                     400,869 <br /> train classifier, required list negatives, i.e.,     HTP + LC             6074                     400,473 <br /> pairs genes confirmed partaking SSL inter- <br /> action. We constructed non-SSL pairs generating               The table lists number SSL pairs non-SSL pairs high <br />                                                                      throughput (HTP) combined (HTP + literature curated (LC)) data <br /> pairwise combinations 227 baits used large             sets. <br />  <br />  <br />                                                                                                                            Page 4 14 <br />                                                                                                      (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:426                                                http://www.biomedcentral.com/1471-2105/9/426 <br />  <br />  <br />  <br />  <br />    &apos;RAPH PROPERTIES OF A AIR OF PROTEINS <br />    0 <br /> 0  <br />                                                      /PTIMAL (YPERPLANE <br />     !VERAGE &quot;ETWEENNESS #ENTRALITIES OF PAIR <br />     0 <br /> 0  <br />     !VERAGE #LOSENESS #ENTRALITIES OF PAIR <br />     0 <br /> 0  <br />  <br />                                                    <br />     !BSOLUTE DIFFERENCE BETWEEN &quot;ETWEENNESS /UTPUT BETWEEN <br />     #ENTRALITIES OF PAIR 0 <br /> 0 AND MEASURES <br />     !BSOLUTE DIFFERENCE BETWEEN #LOSENESS <br />                                                                                                              THE PROPENSITY OF <br />     #ENTRALITIES OF PAIR 0 <br /> 0                                                                              3, INTERACTION <br />  <br />                                                    <br />       3HORTEST DISTANCE BETWEEN 0 0  <br />  <br />        -UTUAL NEIGHBHORHOOD OF 0 0  <br />  <br />                                                                                   3UPPORT 6ECTORS <br />  <br />  <br /> A <br /> Figure <br />   schematic <br />        1 diagram representing SVM classifier various graph theoretic properties fed inputs <br /> A schematic diagram representing SVM classifier various graph theoretic properties fed inputs. <br />  <br />  <br /> classifier mapped posterior probabilities          prediction accuracy (see, example, [37]) com- <br /> value ranging 0 1 measure propensity           puted non-parametric manner using trapezoidal <br /> a SSL interaction corresponding genes [35].           approximation. In similar manner, posi- <br /> The SVMs trained using radial basis function kernel        tive predictive value (precision rate) method a <br /> whose parameters optimized performing five-             function threshold &#954;, positive predictive <br /> fold cross-validation entire training set, sug-         value defined ratio correctly predicted positives <br /> gested [36]. The SVM classifier trained randomly         total number predicted positives. <br /> selected balanced subset pairs genes and <br /> without SSL interaction, i.e., training set comprised      Cross validation studies <br /> two-thirds known synthetic lethal pairs             Ten-fold cross-validation studies performance of <br /> equal number known non-SSL pairs. The test set con-            SVM classifier carried using balanced SSL <br /> sisted remaining gene pairs (both known SSL            non-SSL training data training representative <br /> non-SSL) used training SVM classifier.       datasets testing, follows. Known SSL pairs first <br />                                                                   divided randomly 10 groups. An equal number of <br /> Since SVM output score takes continuum values            non-SSL pairs randomly selected these <br /> between 0 1, set variable cutoff &#954; deciding          pairs randomly assigned 10 groups. <br /> whether pair genes partakes SSL interaction. Pairs      Thus, group contained equal number SSL and <br /> that scored &#954; predicted SSL, pairs         non-SSL pairs. Nine groups combined train- <br /> that scored &#954; predicted non-SSL. We              ing SVM classifier, subsequently tested on <br /> then measured sensitivity specificity              SSL pairs withheld group non-SSL <br /> method test sets function &#954; generated          pairs withheld group remaining <br /> ROC (Receiver Operating Characteristic) curves sensi-          data. This repeated 10 times group playing <br /> tivity vs. 1-specificity represent overall accuracy     role test group once. <br /> the method. Sensitivity defined proportion of <br /> true positives classified correctly method        Prediction novel SSL interactions <br /> and specificity defined proportion true nega-        Finally, sought identify novel SSL interactions. To <br /> tives classified correctly method. The area       end, culled data set consisting gene pairs <br /> under ROC curve, statistic commonly used assess <br />                                                                   tested synthetic genetic interactions <br />  <br />  <br />                                                                                                                       Page 5 14 <br />                                                                                                 (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:426                                                             http://www.biomedcentral.com/1471-2105/9/426 <br />  <br />  <br />  <br />  <br /> (obtained generating possible pairwise combina-                       tions obtained SVM output threshold level <br /> tions non essential genes S. cereviseae                       &#954; = 0.75 (Figure 2). <br /> removing known SSL non-SSL pairs used in <br /> the assessment cross-validation accuracy). To score                   Results discussion <br /> new set gene pairs retrained classifier                     We used thirteen different graph-theoretic properties of <br />                                                                              proteins (see Methods section) PIN putative predic- <br /> complete set known SSL pairs equal number of <br />                                                                              tors existence SSL interaction cor- <br /> randomly selected non-SSL pairs. The retrained classifier <br />                                                                              responding gene pair. This resulted maximum 22 <br /> was used evaluate propensity of &#8776;1,                  inputs SVM classifier: values single <br /> 620, 000 gene pairs newly constructed data set                  node property, value shortest dis- <br /> SSL. This prediction task repeated times,                      tance, mutual protein neighborhood, 2Hop S-S 2Hop <br /> time training different set randomly selected non-                   S-P. <br /> SSL pairs. The gene pairs scored desired cut- <br /> off runs reported putative novel SSLs                Input feature distributions SSLs non-SSLs <br />                                                                              In order assess suitability of graph-the- <br /> (the number putative novel SSLs different thresholds <br />                                                                              oretic properties distinguishing SSL pairs non-SSL <br /> is shown Figure 2). Based results high through-                   pairs obtained distributions properties <br /> put genetic analysis studies, estimated                 SSL pairs non-SSL pairs. For property, we <br /> global SSL network yeast contain &#8776;200,000 interac-                   plot distribution average property over <br /> tions [38]. As &#8776;4500 non-essential genes S.                     genes pair, absolute difference that <br /> cerevisiae, expect newly constructed data set pro-                 property genes. Most properties studied <br />                                                                              statistically significant small distributional <br />                                                               &#9115; 4500 &#9118; <br /> portionately contain &#8776;200,000 &#215; 1,620,000/ &#9116;                         &#9119;   =   differences SSL pairs non-SSL pairs (see Fig- <br />                                                               &#65533;? 2 &#9120;          ure 3). Properties display greatest distributional <br /> 31,307 true SSL interactions, turns                          differences (as measured Kolmogorov-Smirnov sta- <br /> approximately equal number novel SSL predic-                       tistic) eigenvector centrality, degree bridging cen- <br />                                                                              trality. Since shortest distance technically infinite for <br />                                                                              proteins lie different components the <br />                                  No. predicted SSL pairs                  PIN, used inverse shortest distance input to <br />                           5000    10000 15000 20000 25000 30000              SVM classifier. From Table 2 infer that, when <br />                                                                              viewed PIN, SSL pairs compared non-SSL <br />                    0.75 <br />  <br />  <br />  <br />  <br />                                                                              ones tend higher average degree, higher average <br />                                                                              closeness centrality, higher average information centrality <br />                                                                              higher number mutual neighbors. <br />                    0.80 <br />  <br />  <br />  <br />  <br />                                                                              Accuracy SSL predictions <br />                                                                              We assessed performance SVM classifier <br />  Threshold level <br />                    0.85 <br />  <br />  <br />  <br />  <br />                                                                              PIN properties used &quot;triangle- <br />                                                                              completing&quot; 2Hop properties excluded. This is <br />                                                                              2Hop properties binary inputs predic- <br />                    0.90 <br />  <br />  <br />  <br />  <br />                                                                              tion accuracy extremely sensitive choice test <br />                                                                              data set (described below). We randomly selected 2/3 of <br />                                                                              known SSL pairs equal number non-SSL <br />                                                                              pairs training SVM classifier withheld the <br />                    0.95 <br />  <br />  <br />  <br />  <br />                                                                              Table 2: Network statistics SSL vs. non-SSL pairs <br />                    1.00 <br />  <br />  <br />  <br />  <br />                                                                                   Gene pair characteristic             t statistic        P-value <br />  <br />  <br /> threshold <br /> Number <br /> Figure  2of predicted novel SSL pairs different levels                          Average Degree                    21.1908        &lt; 2.2 &#215; 10-16 <br /> Number predicted novel SSL pairs different lev-                           Average Closeness Centrality            22.9225        &lt; 2.2 &#215; 10-16 <br />                                                                                 Average Information Centrality          53.1484        &lt; 2.2 &#215; 10-16 <br /> els threshold. At threshold level 0.75, total <br />                                                                                     Mutual Neighborhood                 18.2122        &lt; 2.2 &#215; 10-16 <br /> number predictions classifier equals number of <br /> true SSL predictions data set expected contain. <br />                                                                               Difference mean values various graph properties SSL <br />                                                                               non-SSL pairs, assessed using independent sample t-test. <br />  <br />  <br />                                                                                                                                        Page 6 14 <br />                                                                                                                  (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:426                                               http://www.biomedcentral.com/1471-2105/9/426 <br />  <br />  <br />  <br />  <br /> Figure 3 <br /> Distributional differences PIN graph-theoretic properties SSL non-SSL pairs <br /> Distributional differences PIN graph-theoretic properties SSL non-SSL pairs. The distributions the <br /> average difference property proteins case SSL pairs (solid curves) non-SSL pairs (dashed <br /> curves) displayed here. Numbers plot indicate D-statistic associated Kolmogorov-Smirnov test the <br /> difference distributions corresponding P-value. <br />  <br />  <br />  <br />  <br />                                                                                                                     Page 7 14 <br />                                                                                               (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:426                                                                 http://www.biomedcentral.com/1471-2105/9/426 <br />  <br />  <br />  <br /> remaining pairs genes testing accuracy the <br />  <br />  <br />  <br />  <br />                                                                                  1.0 <br /> method. We tested method withheld data, <br /> resulting ROC curves different training sets. As <br /> is clear Figure 4, overall performance pre- <br />  <br />  <br />  <br />  <br />                                                                                  0.8 <br /> dictor, measured area ROC curve, <br /> shows slight improvement interactions litera- <br /> ture curated data included training set as <br /> opposed using interactions high throughput syn- <br />  <br />  <br />  <br />  <br />                                                                         True Positive Rate <br />                                                                                        0.6 <br /> thetic lethal screens alone, overall accuracies of <br /> 0.7960 0.7550 respectively. We note higher val- <br /> ues SVM classifier output threshold, &#954;, (correspond- <br /> ing lower false positive rate) lead higher specificity, <br />  <br />  <br />  <br />  <br />                                                                         0.4 <br /> while lower values &#954; (corresponding lower false <br /> negative rate) lead higher sensitivity. <br />                                                                                                             Without 2Hop Features <br />                                                                                                             2Hop&#8722;SS <br />  <br />  <br />  <br />  <br />                                                                                  0.2 <br /> Addition 2Hop features inputs improves                                                        With 2Hop&#8722;SP <br />                                                                                                             With 2Hop&#8722;SS 2Hop&#8722;SP <br /> performance classifier <br /> When 2Hop features included additional inputs <br /> to SVM classifier significant improvement <br />  <br />                                                                                  0.0 <br /> in performance classifier randomly cho-                                0.0    0.2        0.4           0.6         0.8           1.0 <br />                                                                                                           False Positive Rate <br /> sen test set (Figure 5), consistent previously reported <br /> increases accuracy 2Hop features included                   Figurecharacteristics <br />                                                                         Comparison <br />                                                                         2Hop      5 ROC curves addition of <br /> [21]. The best overall accuracy, measured AUC,                Comparison ROC curves addition <br /> is 90.4%, obtained SVM classifier trained                2Hop characteristics. ROC curves SVM classifiers <br />                                                                         trained combined data (literature curated high <br />                                                                         data) using 2Hop characteris- <br />                                                                         tics inputs. <br />           1.0 <br />  <br />  <br />  <br />  <br />                                                                         literature curated high throughput data using all <br />                                                                         PIN properties 2Hop features. <br />           0.8 <br />  <br />  <br />  <br />  <br />                                                                         Comparison approaches <br />                                                                         We compared results earlier study [21] where <br />                  0.6 <br />  True Positive Rate <br />  <br />  <br />  <br />  <br />                                                                         probabilistic decision trees used distinguish <br />                                                                         SSL non-SSL pairs. The accuracy our <br />                                                                         method appears higher reported [21] (the <br /> 0.4 <br />  <br />  <br />  <br />  <br />                                                                         AUC explicitly computed there), diverse <br />                                                                         genomic datasets 2Hop properties included as <br />                                                        HTP              putative determinants SSL interaction (see Additional <br />                                                        LC+HTP <br />                                                                         file 1 detailed comparison results [21]). <br />           0.2 <br />  <br />  <br />  <br />  <br />                                                                         Further, accuracy obtained method 2Hop <br />                                                                         properties excluded significantly higher that <br />                                                                         [21] 2Hop properties excluded, fur- <br />           0.0 <br />  <br />  <br />  <br />  <br />                    0.0   0.2   0.4               0.6        0.8   1.0   ther discussed (see Supplementary Informa- <br />                                 False Positive Rate <br />                                                                         tion). We note accuracies measured area the <br /> Figure <br /> ROC <br /> SSL data <br />       curves <br />         4aloneforand <br />                    SVM <br />                      combined <br />                        classifierdata <br />                                   trained high throughput            ROC curve account false positive false <br /> ROC curves SVM classifier trained high <br />                                                                         negative errors. However, account low <br /> throughput SSL data combined data. The <br /> combined data comprised high throughput litera-             prevalence SSLs pairs genes genome. <br /> ture-curated SSL interactions. In cases, inputs         We compute positive predictive accura- <br /> SVM classifier included graph theoretic PIN properties          cies SVM classifiers function threshold <br /> (but 2Hop features).                                                level (Figure 6). It interesting note addition of <br />                                                                         2Hop characteristics leads decrease predictive <br />  <br />  <br />                                                                                                                                       Page 8 14 <br />                                                                                                                 (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:426                                                                 http://www.biomedcentral.com/1471-2105/9/426 <br />  <br />  <br />  <br />  <br /> accuracy high thresholds, AUC                               icity sensitivity certain test sets (see Table 4), <br /> increases features included. The rea-                      large number SSL protein <br />                                                                                interactions genes/proteins, specificities <br /> son increase sensitivity addition <br />                                                                                sensitivities vary greatly properties test <br /> of 2Hop characteristics offset faster increase in <br />                                                                                set changed (discussed below). <br /> false positive rate. The positive predictive values (PPV) of <br /> the predictors threshold level 0.999 listed                     Robustness prediction respect choice test <br /> Table 3. Note order assess the fold-improve-                    data <br /> ment prediction ability, PPV compared                      We performed ten-fold cross validation SVM <br /> the estimated prevalence SSLs gene pairs                       classifier (Methods) inputs, than <br />                                              &#9115; 4500 &#9118; <br />                                                                                1% variation classification accuracy measured by <br /> yeast (this prevalence &#8776;200000/ &#9116;                &#9119;   = 0.0198,        area ROC curve (Table 5), confirming the <br />                                              &#65533;? 2 &#9120; <br />                                                                                robustness classification performance respect <br /> is estimated &#8773; 200,000 SSL interactions                     different choices randomly constructed test sets. <br /> &#8776;4500 non-essential genes S. cerevisiae) [38]. <br />                                                                                Next, order assess role 2Hop prop- <br /> Individual importance predictor variables                                   erties prediction task, designed test set in <br /> We investigated importance of individual                       genes SSL interactions other <br /> predictor variables training SVM classifier                     genes/proteins. Both 2Hop properties identically zero <br /> of separately (Figure 7). Classifiers trained using                       pairs test set properties therefore <br /> individual predictors perform better random classifi-                     lose predictive value set. While type test <br /> ers, classification performance                      set does reflect enhanced prevalence triangles in <br /> predictor variables used better                    SSL networks, carried procedure order to <br /> classifiers trained individual predictor variables. Of                  assess PIN properties also <br /> the PIN predictor variables studied, degree turns                    significantly lose predictive value triangles <br /> the best performing individual predictor. The known                            completed known SSL interactions test <br /> importance degree characterizing gene essentiality                       pair. Table 6 shows loss of <br /> therefore extends SSL properties gene pairs well.                     accuracy, accuracy 70% considerably larger <br /> Indeed, strong correlations synthetic lethality                        corresponding accuracy Wong et al. [21] when <br /> and node degree reported earlier [39]. The sec-                      2Hop properties included. <br /> ond best predictor information centrality, <span id='am-84' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-87'  ></span><span id='am-88' property="rdfs:label" content="hybrid" datatype="xsd:string"></span>hybrid</span> <br /> measure relates closeness centrality                         Robustness prediction respect errors the <br /> random walk based eigen-centrality, turned                       protein interaction network <br /> out significant predictor variables own.                        Since prediction method relies strongly protein <br /> Also, significant contribution information central-                     interaction data, important assess prediction <br /> ity SSL prediction indicate information prop-                      quality respect errors protein interaction data. <br /> <span id='am-67' about='obo:NCBITaxon_344170' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3646 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_24921 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="Agation" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Agation Brongn." datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72'  ></span>agation</span> biological network does favor                          Since use high confidence protein interaction data <br /> shortest paths. We tested individual impor-                        (with low false positive rate), surmised dom- <br /> tance 2Hop characteristics used singly                          inant error protein interaction network be <br /> jointly predictor variables. Since features                    attributed missing interactions. We added a <br /> assign equal scores pairs &quot;triangle comple-                   predetermined number new edges randomly the <br /> tion&quot; possible equal scores pairs                      original protein interaction network, retrained reeval- <br /> this possible, inputs lead fixed specificity                   uated SVM classifier. This task repeated, time <br /> and sensitivity values. While possible predict SSL                    adding different number random interactions (250, <br /> pairs triangle completion reasonably high specif-                      500, 750, 1000) PIN. While adding 500 <br />  <br /> Table 3: Predictive power SVM classifiers <br />  <br />                               Set input features SVM classifier                                               Positive predictive value <br />  <br />                                All graph theoretic protein characteristics                                                     0.08 <br />                          All graph theoretic protein characteristics + 2Hop SS                                                 0.50 <br />                          All graph theoretic protein characteristics + 2Hop SP                                                 0.15 <br />                    All graph theoretic protein characteristics + 2Hop SS + 2Hop SP                                             0.14 <br />  <br />  Positive predictive values classifiers trained 2Hop characteristics threshold level 0.999. <br />  <br />  <br />                                                                                                                                           Page 9 14 <br />                                                                                                                     (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:426                                                                                                        http://www.biomedcentral.com/1471-2105/9/426 <br />  <br />  <br />  <br />  <br />                                                     0.6 <br />  <br />  <br />  <br />  <br />                                                                            50000 <br />                                                     0.5 <br />  <br />  <br />  <br />                                                                   No. Of Predictions <br />                                                                            30000 <br />                                                     0.4 <br />                         Positive Predictive Value <br />  <br />  <br />  <br />  <br />                                                              10000 <br />                                                     0.3 <br />  <br />  <br />  <br />  <br />                                                                            0 <br />  <br />  <br />  <br />  <br />                                                                                   0.75        0.80       0.85     0.90         0.95     1.00 <br />                                                                                                            Threshold <br />                                                     0.2 <br />  <br />  <br />  <br />  <br />                                                                                        Without 2Hop Features <br />                                                                                        With 2Hop&#8722;SS <br />                                                                                        With 2Hop&#8722;SP <br />                                                                                        With 2Hop&#8722;SS 2Hop&#8722;SP <br />                                                     0.1 <br />                                                     0.0 <br />  <br />  <br />  <br />  <br />                                                       0.75                             0.80               0.85               0.90              0.95             1.00 <br />                                                                                                                  Threshold <br />  <br /> Figure 6 PPVs various classifiers <br /> Comparison <br /> Comparison PPVs various classifiers. Positive predictive value SVM classifiers trained combined data with <br /> and using 2Hop characteristics inputs. At threshold level 0.999, classifier trained graph theoretic <br /> protein properties 2Hop SS inputs highest positive predictive value. Addition extra indicator variable <br /> 2Hop SP input results classifier decreased positive predictive accuracy. The subplot figure shows number <br /> of predicted pairs (true positives false positives) different levels threshold. <br />  <br />  <br />  <br />  <br /> Table 4: Accuracy prediction performance using 2Hop characteristics alone <br />  <br />             2Hop Characteristic                                                                       Threshold Level (T)                       Sensitivity              Specificity <br />  <br />                     2HopSS                                                                           0.000000 &#8804; T &#8804; 0.235290                      1.000                      0.000 <br />                                                                                                      0.235290 &lt; T &#8804; 0.860423                      0.723                      0.886 <br />                                                                                                      0.860423 &lt; T &#8804; 1.000000                      0.000                      1.000 <br />                     2HopSP                                                                           0.000000 &#8804; T &#8804; 0.217713                      1.000                      0.000 <br />                                                                                                      0.217713 &lt; T &#8804; 0.905607                      0.755                      0.922 <br />                                                                                                      0.905607 &lt; T &#8804; 1.000000                      0.000                      1.000 <br />              2HopSS 2HopSP                                                                       0.000000 &#8804; T &#8804; 0.138735                      1.000                      0.000 <br />                                                                                                      0.138735 &lt; T &#8804; 0.840256                      0.868                      0.835 <br />                                                                                                      0.840256 &lt; T &#8804; 0.840260                      0.723                      0.886 <br />                                                                                                      0.840260 &lt; T &#8804; 0.840392                      0.610                      0.973 <br />                                                                                                      0.860423 &lt; T &#8804; 1.000000                      0.000                      1.000 <br />  <br />  The table lists sensitivity specificity values different threshold levels case classifiers trained using 2Hop characteristics alone. <br />  <br />  <br />                                                                                                                                                                             Page 10 14 <br />                                                                                                                                                        (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:426                                                                  http://www.biomedcentral.com/1471-2105/9/426 <br />  <br />  <br />  <br />                                                                                  Table 6: Effect exclusion gene pairs non-zero 2Hop <br />          1.0 <br />  <br />  <br />  <br />                                                                                  properties <br />  <br />                                                                                          Across 10                AUC &#8211; Before           AUC &#8211; After <br />                                                                                     cross-validation runs          exclusion              exclusion <br />          0.8 <br />  <br />  <br />  <br />  <br />                                                                                            Average                      0.797                 0.697 <br />                                                                                       Standard Deviation                0.038                 0.005 <br /> True Positive Rate <br />                0.6 <br />  <br />  <br />  <br />  <br />                                                                                   Ten fold cross-validation carried excluding <br />                                                                                   gene pairs having non-zero 2Hop properties training test <br />                                                                                   sets. In cases classifiers trained using 2Hop <br />                                                   Betweenness Centrality          characteristics. <br />                                                   Closeness Centrality <br /> 0.4 <br />  <br />  <br />  <br />  <br />                                                   Clustering Coefficient <br />                                                   Degree                         Comparison predicted SSL network known SSL <br />                                                   Eigenvector Centrality <br />                                                   Current&#8722;Flow Betweenness C     network <br />                                                   Information Centrality         Having achieved reasonably high cross-validation accu- <br />                                                   Bridge Centrality <br />          0.2 <br />  <br />  <br />  <br />  <br />                                                   Stress Centrality              racy, proceeded predicted SSL net- <br />                                                   1/(Shortest Distance)          work shared characteristics known SSL <br />                                                   Mutual Neighborhood <br />                                                   All Features                   network (see Methods construction network <br />                                                                                  comprising novel SSL predictions). Earlier studies have <br />          0.0 <br />  <br />  <br />  <br />  <br />                  0.0     0.2       0.4           0.6         0.8           1.0   indicated genes known SSL interaction tend <br />                                   False Positive Rate <br />                                                                                  share similar Gene Ontology (GO) annotation, are <br /> Figure 7 individual predictor variables <br /> Importance                                                                       enriched common upstream regulators, preferen- <br /> Importance individual predictor variables. ROC                                tially protein complex [40,41]. Since <br /> curves SVM classifiers trained literature curated                     characteristics used input features <br /> high throughput data using individual predictor variables. The                   prediction method (except, indirectly, participa- <br /> diagonal line indicates random prediction. The ROC curve                         tion protein complex), investigated <br /> for SVM classifier trained using input features                   predicted SSLs enriched these <br /> also shown figure.                                                        features different threshold levels starting 0.75 up <br />                                                                                  1.000 (See Supplementary Information list of <br />                                                                                  predictions threshold levels). As evident from <br /> random interactions (representing approximately 5%                            Table 8, predicted SSL interactions threshold <br /> the number existing protein interactions) significantly                       levels appear properties similar known SSL <br /> changes numerical value propensity SSL                            interactions. While existence common upstream <br /> interaction assigned SVM, detectable                          regulator predicted novel SSL pairs does not <br /> change ROC curves (see AUC values Table 7).                            appear statistically significant threshold level of <br /> This suggests random additional interactions contrib-                       0.85, attributed small sample size at <br /> <span id='am-89' about='obo:NCBITaxon_365374' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_86011 obo:NCBITaxon_33208 obo:NCBITaxon_1 obo:NCBITaxon_6040 obo:NCBITaxon_45124 obo:NCBITaxon_131567 obo:NCBITaxon_73655 obo:NCBITaxon_27929 obo:NCBITaxon_2759'><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Ute" datatype="xsd:string"></span><span id='am-93'  ></span>ute</span> roughly equally input features SSL non-SSL                         high thresholds. Odds ratios properties generally <br /> pairs, resulting significant change overall discrim-                    increasing trend threshold level is <br /> ination ability.                                                                 increased, showing SVM classifier preferentially <br />                                                                                  selects gene pairs having properties studied with <br /> Table 5: Area ROC curves cross validation <br /> runs <br />                                                                                  higher accuracy higher thresholds, the <br />                                                                                  properties used input features for <br />                        CrossValidation Run #                       AUC <br />                                                                                  Table 7: Robustness analysis respect addition random <br />                                Run1                                0.913         edges PIN <br />                                Run2                                0.901 <br />                                Run3                                0.916          Number random              P-value difference           AUC <br />                                Run4                                0.900          edges added PIN            propensities <br />                                Run5                                0.905 <br />                                Run6                                0.912          250                           0.7905                           0.8692 <br />                                Run7                                0.906          500                           0.1391                           0.8724 <br />                                Run8                                0.896          750                           &lt; 2.2 &#215; 10-16                    0.8646 <br />                                Run9                                0.899          1000                          &lt; 2.2 &#215; 10-16                    0.8671 <br />                                Run10                               0.908 <br />                                                                                   Paired sample t-test P-values difference SSL propensities <br />   The classifier trained using graph-theoretic properties         assigned test dataset SVM adding <br />   2Hop characteristics.                                                       random edges protein interaction network. <br />  <br />  <br />  <br />                                                                                                                                           Page 11 14 <br />                                                                                                                      (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:426                                                               http://www.biomedcentral.com/1471-2105/9/426 <br />  <br />  <br />  <br /> Table 8: Comparison predicted SSL network known SSL network <br />  <br /> Threshold Level (Number pre dictions)          Gene-pair Characteristic        S&#8745;C       Sonly     Conly       S&apos;&#8745;C&apos;        Odds          P-value <br />  <br /> Known SSL Pairs (6074)                            Cellular Component               4467     1607     193007       207466       2.988      &lt; 2.2 &#215; 10-16 <br />                                                   Molecular Function               1464     4610      38865       361608       2.955      &lt; 2.2 &#215; 10-16 <br />                                                   Biological Process               4074     2000     137827       262646       3.882      &lt; 2.2 &#215; 10-16 <br />                                                   Protein Complex                   197     5877        747       399726      17.937      &lt; 2.2 &#215; 10-16 <br />                                                   Upstream Regulator               220      5854     10327        390146      1.420         9 &#215; 10-7 <br /> 0.75 (30087)                                      Cellular Component              17967    12120     781640       799150       1.516      &lt; 2.2 &#215; 10-16 <br />                                                   Molecular Function               4115    25972     113100      1467690       2.056      &lt; 2.2 &#215; 10-16 <br />                                                   Biological Process              12966    17121     314562      1266228       3.048      &lt; 2.2 &#215; 10-16 <br />                                                   Protein Complex                   363    29724       1339      1579451      14.405      &lt; 2.2 &#215; 10-16 <br />                                                   Upstream Regulator              1052     29035      47294      1533496      1.175        3.76 &#215; 10-7 <br /> 0.80 (28440)                                      Component                       16942    11498     782665       799772      1.506       &lt; 2.2 &#215; 10-16 <br />                                                   Function                         3826    24614     113389      1469048       2.014      &lt; 2.2 &#215; 10-16 <br />                                                   Process                         12245    16195     315283      1267154       3.039      &lt; 2.2 &#215; 10-16 <br />                                                   Protein Complex                   346    28094       1356      1581081      14.360      &lt; 2.2 &#215; 10-16 <br />                                                   Upstream Regulator                986    27454      47360      1535077      1.164        3.01 &#215; 10-6 <br /> 0.850 (5149)                                      Component                        3799     1350     795808       809920      2.864       &lt; 2.2 &#215; 10-16 <br />                                                   Function                         1309     3840     115906      1489822       4.382      &lt; 2.2 &#215; 10-16 <br />                                                   Process                          3221     1928     324307      1281421       6.601      &lt; 2.2 &#215; 10-16 <br />                                                   Protein Complex                   287     4862       1415      1604313      66.927      &lt; 2.2 &#215; 10-16 <br />                                                   Upstream Regulator                173     4976      48173      1557555      1.124           0.024 <br /> 0.900 (1398)                                      Component                        1184      214     798423       811056      5.602       &lt; 2.2 &#215; 10-16 <br />                                                   Function                          508      890     116707      1492772       7.301      &lt; 2.2 &#215; 10-16 <br />                                                   Process                          1060      338     326468      1283011      12.325      &lt; 2.2 &#215; 10-16 <br />                                                   Protein Complex                   151     1247       1551      1607928     125.535      &lt; 2.2 &#215; 10-16 <br />                                                   Upstream Regulator                49      1349      48297      1561182      1.174          0.1525 <br /> 0.950 (953)                                       Component                         831      122     798776       811148      6.917       &lt; 2.2 &#215; 10-16 <br />                                                   Function                          368      585     116847      1493077       8.038      &lt; 2.2 &#215; 10-16 <br />                                                   Process                           776      177     326752      1283172      17.217      &lt; 2.2 &#215; 10-16 <br />                                                   Protein Complex                   126      827       1576      1608348     155.485      &lt; 2.2 &#215; 10-16 <br />                                                   Upstream Regulator                29      924       48317      1561607      1.014          0.4958 <br /> 0.999 (202)                                       Component                         184       18     799423       811252     10.373       &lt; 2.2 &#215; 10-16 <br />                                                   Function                           93      109     117122      1493553      10.880      &lt; 2.2 &#215; 10-16 <br />                                                   Process                           179       23     327349      1283326      30.511      &lt; 2.2 &#215; 10-16 <br />                                                   Protein Complex                    37      165       1665      1609010     216.702      &lt; 2.2 &#215; 10-16 <br />                                                   Upstream Regulator                 5      197       48341      1562334      0.820          0.7274 <br />  <br /> Association SSL interaction gene-pair characteristic, following framework [20]. S &#8745; C represents number gene pairs with <br /> both SSL interaction(S) gene-pair characteristic(C). Sonly represents number gene pairs SSL interaction only. Conly <br /> represents number non-SSL gene pairs share characteristic. S&apos; &#8745; C&apos; represents number gene pairs SSL <br /> interaction share specified characteristic. Odds represents odds ratio SSL pair sharing given characteristic non-SSL pair sharing <br /> the characteristic. The P-value represents statistical significance overlap SSL interaction specified characteristic as <br /> computed Fisher&apos;s exact test. If protein corresponding gene assigned known protein complexes gene its <br /> interacting partner considered come different protein complexes. Similarly case upstream regulators, gene does not <br /> have known upstream regulator gene interacting partner treated don&apos;t share common upstream regulator. <br /> The P-values significant case common upstream regulator fact number predicted positives share common <br /> upstream regulator small comparison total number pairs share common upstream regulator, reflected the <br /> low odds ratio seen different levels threshold. <br />  <br />  <br /> prediction. The simultaneous increase predictive                       Conclusion <br /> accuracy SSL prediction (as threshold level increased)                  Our results clearly demonstrate informative value of <br /> and enrichment participation protein                          protein interaction networks SSL genetic interactions. <br /> complex (evinced increase protein complex                           We graph-theoretic properties proteins a <br /> enrichment odds) consistent dominance                          protein interaction network serve compelling rela- <br /> &quot;within-pathway&quot; explanations genetic interactions                        tively robust determinants existence synthetic <br /> suggested Kelley Ideker [42].                                          lethality gene counterparts. When members <br />                                                                               gene pair question known SSL interactions <br />                                                                               genes, predictive power SSL interaction <br />                                                                               gene pair greatly enhanced tendency <br />  <br />  <br />                                                                                                                                         Page 12 14 <br />                                                                                                                    (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:426                                                               http://www.biomedcentral.com/1471-2105/9/426 <br />  <br />  <br />  <br /> of triangles form SSL networks (2Hop properties).                       8.    Guimer&#225; R, Amaral L: Functional cartography complex met- <br />                                                                                     abolic networks. Nature 2005, 433:895-900. <br /> However, absence known SSL interactions,                       9.    Kelley R, Ideker T: Systematic interpretation genetic inter- <br /> we shown PINs predict SSL                               actions using protein networks. Nature Biotechnology 2005, <br /> interactions significantly higher accuracy previ-                         23:561-566. <br />                                                                               10.   Deng M, Tu Z, Sun F, Chen T: Mapping Gene Ontology pro- <br /> ously found. Inclusion PIN centralities develop-                          teins based protein-protein interaction data. Bioinformatics <br /> ment <span id='am-57' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59'  ></span><span id='am-60' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>meta</span>-servers SSL prediction likely                         2004, 6(20):895-902. <br /> to useful. Further, functional                         11.   Han JD, Bertin N, Hao T, Goldberg D, Berriz G, Zhang L, Dupuy D, <br />                                                                                     Walhout A, Cusick M, Roth F, Vidal M: Evidence dynamically <br /> properties used input features method,                            organized modularity yeast protein-protein interac- <br /> method identifies gene pairs enriched partici-                         tion network. Nature 2004, 430:88-93. <br />                                                                               12.   Schwikowski B, Uetz P, Fields S: A network protein-protein <br /> pation common GO categories, protein                                 interactions yeast. Nat Biotechnol 2000, 18:1257-1261. <br /> complex, limited extent, having                         13.   Hishigaki H, Nakai K, Ono T, Tanigami A, Takagi T: Assesment of <br /> same upstream regulator. These properties there-                             prediction accuracy protein function protein-protein <br />                                                                                     interaction data. Yeast 2001, 18:523-531. <br /> fore viewed predictions method,                        14.   Vazquez A, Flammini A, Maritan A, Vespignani A: Global protein <br /> though earlier used inputs identifying SSLs                        function prediction protein-protein interaction net- <br />                                                                                     works. Nat Biotechnol 2003, 21:697-700. <br /> [21]. This shows PINs, absence qualify-                   15.   Karaoz U, Murali T, Letovsky S, Zheng Y, Ding C, Cantor C, Kasif S: <br /> ing data gene regulatory gene expression studies,                          Whole-genome annotation using evidence integration in <br /> may informative gene function normally                              functional-linkage networks. Proc Natl Acad Sci USA 2004, <br />                                                                                     101:2888-2893. <br /> envisioned.                                                                   16.   Nabieva E, Jim K, Agarwal A, Chazelle B, Singh M: Whole-proteome <br />                                                                                     prediction protein function graph-theoretic analysis of <br />                                                                                     interaction maps. Bioinformatics 2005, 21(Suppl 1):i302-i310. <br /> Authors&apos; contributions                                                        17.   Ulitsky I, Shamir R: Pathway redundancy protein essential- <br /> SP &amp; SZ performed analysis wrote paper. AlR                             ity revealed Saccharomyces cerevisiae interaction net- <br /> &amp; AnR conceived project wrote paper.                                    works. Molecular Systems Biology 2007, 3:104. <br />                                                                               18.   Parsons AB, Geyer R, Hughes T, Boone C: Yeast genomics and <br />                                                                                     proteomics drug discovery target validation. Progress <br /> Additional material                                                                 Cell Cycle Research 2003, 5:159-166. <br />                                                                               19.   Kafri R, Dahan O, Levy J, Pilpel Y: Preferential protection pro- <br />                                                                                     tein interaction network hubs yeast: Evolved functionality <br />                                                                                     genetic redundancy. Proceedings National Academy Sci- <br />      Additional file 1                                                              ences 2008, 105(4):1243-1248. <br />      Supplementary Information. This file contains analysis perform-   20.   Tong A, Lesage G, Bader G, Ding H, Xu H, Xin X, Young J, et al.: Glo- <br />      ance SSL prediction method compared method used            bal mapping yeast genetic interaction network. Science <br />      [21]. It contains novel SSLs predicted method different         2004, 303:808-813. <br />      threshold levels.                                                        21.   Wong SL, Zhang LV, Tong AH, Li Z, Goldberg DS, King OD, Lesage <br />                                                                                     G, Vidal M, Andrews B, Bussey H, Boone C, Roth FP: Combining <br />      Click file <br />                                                                                     biological networks predict genetic interactions. Proc Natl <br />      [http://www.biomedcentral.com/content/supplementary/1471-                      Acad Sci USA 2004, 101(44):15682-15687. <br />      2105-9-426-S1.pdf]                                                       22.   Reguly T, Breitkreutz A, Boucher L, Breitkreutz B, Hon G, Myers CL, <br />                                                                                     Parsons A, Friesen H, Oughtred R, Tong A, Stark C, Ho Y, Botstein <br />                           <br /> </body></html>