<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Anal Biochem. 2009 November 1; 394(1): 39&#8211;47. doi:10.1016/j.ab.2009.06.037. <br />  <br />  <br />  <br />                            A homogenous resonance energy transfer assay for <br />                            phosphopantetheinyl transferase <br />  <br />                            Timothy L. Foley Michael D. Burkart* <br />                            Department Chemistry &amp; Biochemistry; University <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span>California</span>, San Diego; 9500 Gilman Drive; <br />                            La Jolla, CA 92093-0358. telephone (85)534-5673, fax (858) 882-1174 <br />  <br />                            Abstract <br />                                 Phosphopantetheinyl transferase plays essential role activating fatty acid, polyketide and <br />                                 nonribosomal peptide biosynthetic pathways, catalyzing covalent attachment 4&#8217;- <br />                                 phosphopantetheinyl group conserved residue carrier protein domains. <span id='am-95' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-96' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-99'  ></span>This</span> enzyme has <br />                                 validated essential gene primary metabolism presents target identification <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 antibiotics new mode action. Here report development homogenous resonance <br />                                 energy transfer assay utilizing fluorescent coenzyme A derivatives surrogate peptide substrate <br />                                 serve identify inhibitors enzyme <span id='am-92' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span>. This assay lays blueprint translation <br />                                 techniques transferase enzymes accept fluorescent substrate analogues. <br />  <br />  <br />                            Keywords <br />                                 phosphopantetheinyl transferase; assay development; high throughput screen; FRET assay; <br />                                 fluorescent analogue; Coenzyme A; Acyl carrier protein; Peptidyl carrier protein; secondary <br />                                 metabolism <br />  <br />  <br />                            Introduction <br />                                               A unifying characteristic biosynthesis fatty acid, non-ribosomal peptide, and <br />                                               polyketide compounds tethering nascent polymer small carrier protein domains <br />                                               synthases thioester linkage phosphopantetheinyl (4&#8217;-PP) arm. This 4&#8217;- <br />                                               PP installed proteins post-translationally coenzyme A (CoA) 2 conserved <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               serine residue action phosphopantetheinyl transferase (PPTase) enzymes, converting <br />                                               apo- 1 holo- 4 form (Fig. 1). This modification essential synthase <br />                                               activity, ablation PPTase gene precludes natural product production [1&#8211;4], the <br />                                               case fatty acid biosynthesis, renders organism unviable [3&#8211;5]. Within bacteria, there <br />                                               exist major classes enzymes PPTase <span id='am-178' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-179' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-180' property="rdfs:label" content="superfamily" datatype="xsd:string"></span>superfamily</span>: AcpS-type Sfp- <br />                                               type [1]. Grouping designations based primary sequence, their <br />                                               canonical representatives, AcpS <span id='am-155' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-156' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-158'  ></span><span id='am-159' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-167' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-169' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia coli</span> Sfp <span id='am-25' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-26' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span>Bacillus subtilis</span>, structurally <br />                                               distinct [6; 7]. <br />  <br />                                               It demonstrated acpS essential E. coli gene [8; 9], validating a <br />                                               target inhibitor development potential treat multi-drug resistance. Indeed, a <br />  <br />  <br />                            *to correspondence addressed: mburkart@ucsd.edu. <br />                            Publisher&apos;s Disclaimer: This PDF file unedited manuscript accepted publication. As service customers <br />                            providing early version manuscript. The manuscript undergo copyediting, typesetting, review resulting <br />                            proof published final citable form. Please note production process errors discovered could <br />                            affect content, legal disclaimers apply journal pertain. <br />                             Foley Burkart                                                                                                 Page 2 <br />  <br />  <br />                                                number groups begun focused programs develop AcpS inhibitors [10&#8211;14] and <br />                                                candidates recently discussed [10&#8211;12]. In addition fatty acids, number <br />                                                compounds produced 4&#8217;-PP dependent pathways identified as <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                virulence factors, disruption biosynthesis received attention new <br />                                                angle therapeutic development [15&#8211;19]. We intrigued central role of <br />                                                phosphopantetheinylation metabolic pathways, interested studying the <br />                                                potential effects PPTase inhibitors coordinate attenuation numerous <br />                                                aspects pathogenicity. <br />  <br />                                                While AcpS-directed inhibitor development reported [10&#8211;14], work generally <br />                                                omitted screening protocols. The described method utilizes homogenous time-resolved <br />                                                fluorescence resonance energy transfer (HT-RF) means activity determination [11]. <br />                                                We replication technique capabilities limitations imposed by <br />                                                instrumentation, desired use readily available affordable biochemical <br />                                                reagents. Herein report development homogenous fluorescence resonance energy <br />                                                transfer (FRET) screen canonical PPTase representatives, AcpS Sfp. This <br />                                                method simple, requiring addition reagents reaction wells microtiter plate, <br />                                                validated process identify inhibitors enzymes. Furthermore, we <br />                                                details led successful development screen, serve blueprint <br />                                                assay design transferase enzymes accept reporter-modified substrate analogues. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Materials Methods <br />                               General <br />                                                7-dimethylamino-4-methyl-coumarin-3-maleimide (DACM) N,N,N&#8217;,N&#8217;- <br />                                                tetramethylrhodamine-5-maleimide (TAMRA) purchased Invitrogen Corporation <br />                                                (Carlsbad, CA). Coenzyme A trilithium salt purchased EMD biochemicals (San <br />                                                Diego, CA). 3&#8217;-phosphoadenosine-5&#8217;-phosphate disodium salt (PAP), 4-(2-hydroxyethyl)-1- <br />                                                piperazine ethanesulfonic acid (HEPES), dimethylsulfoxide (DMSO, Hybri-Max grade), and <br />                                                fluorescein-isothiocyanate isomer I (FITC) purchased Sigma (Saint Louis, MO). <br />                                                Concentrations fluorescently labeled reagents quantified UV-VIS spectroscopy <br />                                                Agilent 8354 diode array spectrophotometer (Agilent Technologies, Santa Clara, CA) <br />                                                using following extinction coefficients: DACM: 23,000 cm&#8722;1 M&#8722;1, FITC: 77,000 cm&#8722;1 <br />                                                M&#8722;1, TAMRA: 95,000 cm&#8722;1 M&#8722;1. <br />  <br />                               Enzyme expression &amp; Purification <br />                                                Sfp expressed purified described previously [2]. The enzyme concentrated to <br />                                                40 mg/mL, diluted equal volume 75 % glycerol, stored 100 &#181;L aliqouts at <br />                                                &#8722;80&#176;C. For routine use, stock (20 mg/mL, 765 &#181;M) diluted 1 mg/mL Sfp storage <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                buffer (50 mM NaHEPES, 120 mM NaCl, 33 % v/v glycerol) supplemented 0.1% w/v <br />                                                BSA stored &#8722;20&#176;C. Under conditions, noticeable reduction in <br />                                                enzymatic activity storage periods longer 6 months. <br />  <br />                                                E. coli AcpS expressed purified native protein pDPJ according published <br />                                                procedures [20]. The protein concentration final preparation adjusted 10 mg/mL <br />                                                addition 2X storage buffer, equal volume glycerol added 3 portions, aliquots <br />                                                stored &#8722;80&#176;C. For routine work, single tubes (200 &#181;L portions) stored &#8722;20&#176;C, with <br />                                                degradation enzymatic activity observed 1 year storage. <br />  <br />                               Synthesis assay components <br />                                                An exploratory quantity (ca. 8 mg) fluorescein-5-isothiocyanate-modified YbbR peptide <br />                                                (FITC-YbbR) 8 (sequence: Fluorescein-Ahx-DSLEFIASKLA-OH) initially purchased <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                            Page 3 <br />  <br />  <br />                                                GL Biochem (Shanghai, China). For final screen evaluation, peptide prepared <br />                                                0.2 mmol scale using automated solid phase peptide synthesizer (Applied Biosystems <br />                                                Pioneer) using standard 9-fluorenylmethyloxycarbonyl (FMOC) chemistry 2-(1H-7- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Azabenzotriazol-1-yl)-1,1,3,3-tetramethyl uronium hexafluorophosphate (HATU) activation <br />                                                (Fig. 3A) [21]. The sequence appended N-terminal N-FMOC-&#949;-aminocaproic acid <br />                                                spacer, deprotected, coupled overnight FITC. Following cleavage solid <br />                                                support, product HPLC purified yield 84 mg FITC-YbbR 8 identity verified <br />                                                ESI-MS. <br />  <br />                                                Fluorescent reporter CoA (mCoA) analogues 11 12 prepared reaction reduced <br />                                                CoA trilithium salt 2 (1 mg/mL 50 mM NaH2PO4, pH 7.4) 1.1 equivalents maleimide <br />                                                bearing probe 9 10, respectively (both dissolved 1 mg/mL methanol) (Fig. 3B). The <br />                                                reaction followed completion HPLC, determined disappearance CoA peak. <br />                                                Excess 9 10 removed extraction times ethyl acetate dichloromethane, <br />                                                respectively. The resultant aqueous phase placed vacuum (&lt; 2 mm/Hg) 2h to <br />                                                remove residual organic solvent. The purity mCoA analogues 11 12 verified be <br />                                                &gt;95% HPLC. <br />  <br />                               Kinetic evaluation fluorescent substrates <br />                                                Kinetic parameters FITC-YbbR 8 TAMRA-mCoA 12 determined HPLC. <br />                                                Reactions conducted final volume 50 &#181;L buffer containing 10 mM MgCl2, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                50 mM Na-HEPES, pH 7.6, 0.1 mg/mL BSA. Reactions initiated addition 0.5 <br />                                                &#181;M enzyme, allowed progress 15 minutes, quenched addition 50 &#181;L 50 mM <br />                                                sodium ethylenediaminetetraacetic acid, pH 8.0. The reaction mixtures separated by <br />                                                reversed phase chromatography Agilent 1100 instrument fitted diode array <br />                                                detector (Agilent Technologies, Santa Clara, CA) using OD5 C18 column (250 &#215; 4.6 mm, <br />                                                product number 9575, Burdick &amp; Jackson, Morristown, NJ). The separation performed <br />                                                following conditions: Buffer A: 10 mM ammonium acetate, Buffer B: Acetonitrile, <br />                                                flow rate: 1.5 mL/min. After injection, run initiated 2 minute isocratic flow 10% <br />                                                Buffer B, followed sample elution 10 minute linear gradient 10% 60% Buffer <br />                                                B. The column regenerated 2 minute isocratic flow 100% Buffer B, the <br />                                                column equilibrated 10% Buffer B. <br />  <br />                               Fluorescence spectroscopy <br />                                                Steady state single sample fluorescence spectra recorded QuantaMaster 2000 <br />                                                spectrofluorometer (Photon Technologies International, Princeton, NJ), using excitation and <br />                                                emission slit widths 4 nm integration times 0.1 sec. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               FRET screen conditions <br />                                                PAP parent inhibitor plates dissolving compound dry DMSO a <br />                                                concentration 10 mM serial diluting (2-fold) DMSO. The final protocol (Table <br />                                                1) follows: 2.5 &#181;L parent DMSO solutions (or DMSO negative control) were <br />                                                transferred individual wells black polystyrene 96 plate (Costar # 3694) followed <br />                                                37.5 &#181;L 1.33 X Enzyme solution [16.62 nM Sfp 66.6 nM AcpS 1.33 X PPTase <br />                                                assay buffer (66 mM Na-HEPES, 13.3 mM MgCl2, 1.33 mg/mL BSA, pH 7.6). Reactions were <br />                                                initiated addition 10 &#181;L 5X reagent solution (50 &#181;M TAMRA-mCoA 12, 25 &#181;M <br />                                                FITC-YbbR 8, 10 mM NaH2PO4, pH 7.0). The reaction monitored continuously (cycle <br />                                                time 2 minutes) 15 cycles Perkin Elmer HTS7000plus microtiter plate reader with <br />                                                excitation filter &#955; = 485 nm, emission filter &#955; = 535 nm. <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                              Page 4 <br />  <br />  <br />                               <span id='am-182' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-183' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-184' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-185'  ></span><span id='am-186' property="rdfs:label" content="Data" datatype="xsd:string"></span>Data</span> analysis <br />                                                Kinetic data processed Microsoft Excel. Assay statistics evaluated according <br />                                                standard equations [22]. Plotting nonlinear regression performed GraphPad <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Prism version 5.00 (GraphPad Inc, La Jolla, CA), IC50 curves fit four- <br />                                                parameter dose response equation using ordinary (least squares) setting. In cases, error <br />                                                bars reported error values represent standard deviation. Molecular graphics images <br />                                                (Fig. 1 &amp; Fig. 2) renditions apo-actinorhodin acyl carrier protein (PDB identifier: <br />                                                2K0Y) generated using UCSF Chimera package [23]. <br />  <br />                               Results &amp; Discussion <br />                               Design high-throughput PPTase assay <br />                                                Traditionally, PPTase bioassays conducted low throughput manner, by <br />                                                monitoring radiolabel incorporation precipitated protein mass [3H]-CoA [4] by <br />                                                HPLC separation apo- holo-states carrier protein substrate [3]. These systems <br />                                                utilize centrifugation chromatographic separations, respectively, readily <br />                                                amenable high throughput screening. In designing high throughput screen PPTase <br />                                                activity, desired allow direct monitoring reaction progress to <br />                                                eliminate probability false-positive hits arrive inhibition coupled enzyme <br />                                                systems. To accomplish this, chose exploit synthetic aspect PPTase reaction <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                conjunction fluorescent reporter-CoA (mCoA) technology developed previously in <br />                                                laboratory [24]. In system, fluorophore-appended carrier protein domains/mimics 5 <br />                                                generate FRET signal conversion thiol-blocked crypto-form 7 modification <br />                                                mCoA 6 analogs PPTase (Fig. 2). <br />  <br />                               Fluorescent probe selection substrate synthesis <br />                                                We chose make fluorescein-5-isothiocyanate (FITC) modification protein-based <br />                                                substrate, fluorophore possesses absorption emission centered visible light <br />                                                spectrum, allowing function FRET donor acceptor, depending identity <br />                                                compounds chosen mCoA 6 preparation (Fig. 2). This allow probe both <br />                                                modes FRET singly prepared pool reagent. In turning select carrier protein <br />                                                domain function acceptor substrate, initially considered E. coli fatty acid <br />                                                synthase acyl carrier protein (ACP) candidate, previous reports note ACP contains <br />                                                single tyrosine residue present C-terminus &#945;-<span id='am-87' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-90'  ></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>helix</span> 3 modification residue <br />                                                dansyl-moiety does hinder function [25;26]. We protocol other <br />                                                tyrosine-modifying techniques [27;28] provide low yields fluorescein-modified proteins <br />                                                insolubility FITC derivatives low pH reaction conditions (data not <br />                                                shown), purification labeled material insufficient needs. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Subsequently, chose investigate use residue YbbR peptide (sequence <br />                                                H-DSKLEFIASKLA-OH) identified Yin et al. undergoes modification PPTases, <br />                                                serving ACP surrogate [29]. This choice strengthened fact solid phase <br />                                                peptide synthesis (SPPS) allows access large quantities uniformly labeled material, a <br />                                                crucial requirement FRET applications, avoids potential batch-to-batch <br />                                                variability. In selecting placement <span id='am-181' about='rdfs:label' typeof='owl:Thing'>label</span>, noted YbbR isolated a <br />                                                collection N-terminal extensions consensus, suggesting site modification that <br />                                                abrogate activity. As such, chose attach FITC YbbR consensus 6- <br />                                                aminocaproic acid spacer unit sufficiently distance molecule central motif (Fig. <br />                                                3A) impart number freely rotatable bonds, ensuring random spatial orientation <br />                                                FRET-pair assembly (vide infra). <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                             Page 5 <br />  <br />  <br />                                                In selecting complimentary probes containing modest spectral overlap FITC mCoA <br />                                                6 production, sought maleimide-bearing compounds amenable organic <br />                                                extraction reaction CoA (Fig. 3B), circumvent HPLC purification; a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                characteristic make procedure easily scaleable high screening volume <br />                                                application. With mind, dimethylaminocoumarin (DACM) 9 tetramethylrhodamine <br />                                                (TAMRA) 10 chosen used prepare DACM-mCoA 11 TAMRA-mCoA 12 (Fig. <br />                                                3B) evaluated FRET donor (Fig. 4A) FRET acceptor (Fig. 4E), respectively. <br />  <br />                               HPLC analysis labeled substrates <br />                                                The labeled substrates evaluated HPLC probe effects modification the <br />                                                substrates kinetic parameters displayed enzymes. Evaluation of <br />                                                FITC-YbbR 8 AcpS Sfp, saturating concentration CoA 2 returned Km values <br />                                                86 101 &#181;M Kcat values 4.4 12.1 &#181;M, respectively. These agreement <br />                                                values previously determined unmodified substrate (Km values 200 123 <br />                                                &#181;M AcpS Sfp, respectively) [29;30]. Additionally, kinetic parameters TAMRA- <br />                                                mCoA 12 determined saturating concentration FITC-YbbR 8 gave Km values <br />                                                22 6 &#181;M AcpS Sfp, Kcat values 3.2 8.9, respectively. Again, these <br />                                                values agreement previously determined parameters [29;30] and <br />                                                demonstrate modifications substrates tolerated enzymes. <br />  <br />                               Determination F&#246;rster Radius <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                In <span id='am-103' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-104' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> assess viability donor acceptor substrates chosen, sought to <br />                                                determine theoretical Forster&#8217;s radius, R0, pairs. This value, distance where <br />                                                transfer efficiency 50%, related donor quantum yield &#1060;D, relative orientation of <br />                                                donor acceptor transition dipoles &#954;, J(&#955;), overlap integral donor emission <br />                                                FD absorption acceptor &#949;A [31]: <br />  <br />  <br />  <br />                                                                                                                                             (1) <br />  <br />  <br />  <br />                                                For &#954; value 2/3 assumed corresponds random orientation fluorophores <br />                                                space, given linked numerous freely rotatable bonds. &#951;, index of <br />                                                refraction medium, assumed 1.44 biological samples [31] NA, avagadro&#8217;s <br />                                                number, constant, leaving R0 determined &#1060;D, J(&#955;). Quantum yields these <br />                                                fluorophores previously determined [32], J(&#955;) expressed as, <br />  <br />                                                                                                                                             (2) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Simple equations written FD(&#955;) &#949;A(&#955;), thus, J(&#955;) calculated a <br />                                                summation [33]: <br />  <br />  <br />  <br />                                                                                                                                             (3) <br />  <br />  <br />  <br />                                                Absorption emission spectra recorded conjugated substrates 8, 11 and <br />                                                12 (Fig. 4B &amp; F) summation values wavelength according Eq. (3) use <br />                                                J(&#955;) values Eq. (1) gave F&#246;rster&#8217;s radii 38 &#197; 56 &#197; DACM-FITC (Fig. <br />                                                4A) TAMRA-FITC pairs (Fig. 4E), respectively. <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                              Page 6 <br />  <br />  <br />                               Photophysical evaluation phosphopantetheinylated peptide efficiency transfer <br />                               determination <br />                                                With theoretical F&#246;rster&#8217;s radii, mathematically calculated maximum distance that <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                fluorophores achieve crypto-FITC-YbbR product (exemplified 7, Fig. 2) <br />                                                summation average bond lengths, determined 42 &#197; (radius, r, 21 &#197;) if <br />                                                rigid linear extension occur. This suggests high theoretical FRET efficiency, E, <br />                                                strictly related radius separating fluorophores, r, by <br />  <br />  <br />  <br />                                                                                                                                              (4) <br />  <br />  <br />  <br />                                                rapid decay E r exceeds R0. Using R0 values determined Eq. (1) and <br />                                                calculated r 21&#197;; Eq. (4) gives theoretical E values 97 % 99 % DACM- <br />                                                FITC TAMRA-FITC pairs, respectively. However, r exists distribution the <br />                                                number freely rotatable bonds, transfer efficiency experimentally <br />                                                determined measurement donor fluorescence absence (FD) presence (FDA) of <br />                                                acceptor fitting Eq. (5) experimental data: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                                                                                              (5) <br />  <br />  <br />  <br />                                                Reactions set mCoA analogues 11 12, Sfp, FITC-YbbR 8 followed <br />                                                completion HPLC. The corresponding crypto-FITC-YbbR 7 peptides isolated <br />                                                semi-preparative HPLC. These products used prepare equimolar mixtures the <br />                                                probes varying degrees modification. Fluorescent emission spectra mixtures <br />                                                recorded (Fig. 4C &amp; G). Analysis data (Fig. 4D &amp; H) Eq. (5) yields efficiency <br />                                                transfers 0.82 0.99 DACM-FITC (Fig. 4A) TAMRA-FITC (Fig. 4E) pairs, <br />                                                respectively. <br />  <br />                               Mode observation <br />                                                Data Fig. 4 (C &amp; G) informative regard mode monitoring this <br />                                                biochemical system. FRET pairs offer flexibility assembly detected either <br />                                                quench donor emission (down arrows) sensitization acceptor fluorescence <br />                                                shorter excitation wavelengths (up arrows). Evaluation data systems shows a <br />                                                greater response quenching mode, 5-fold 10-fold reduction donor <br />                                                fluorescence conversion crypto-FITC-YbbR 7 DACM-FITC TAMRA- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                FITC pairs, respectively. Evaluation acceptor sensitized emission shorter excitation <br />                                                wavelengths showed increase 2-fold complete conversion, indicating system <br />                                                greater signal background ratio monitoring quench donor <br />                                                fluorescence. This screen function optimally substrate-consumption assay. <br />  <br />                                                Of FRET systems, TAMRA-mCoA probe 12 possessed better FRET characteristics <br />                                                (i.e. attenuation donor fluorescence complete conversion) chosen be <br />                                                carried forward screen implementation. <br />  <br />                               Determination screen conditions <br />                                                We arrived initial assay conditions based requirement end point need <br />                                                complete attenuation donor fluorescence; requiring TAMRA-mCoA 12 present in <br />                                                excess FITC-YbbR 8. With mind, probed linearity detector response with <br />                                                respect FITC-YbbR 8 concentration. This identify maximal concentration <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                              Page 7 <br />  <br />  <br />                                                obtained direct correlations relative fluorescence units and <br />                                                substrate concentration obscured inner filter effect, phenomenon all <br />                                                incedent light irradiated sample absorbed does sufficiently transition all <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                target molecules excited state. With plate reader (Perkin Elmer HTS7000) found <br />                                                delineation occur concentrations exceeding 10 &#181;M, chose hold FITC-YbbR <br />                                                8 concentration 5 &#181;M. <br />  <br />                                                We sought determine effects increasing concentration TAMRA-mCoA <br />                                                12 FITC-YbbR 8 signal. The purpose analysis acceptably high <br />                                                concentration TAMRA-mCoA 12 did impart effect FITC-YbbR 8 emission, <br />                                                displays absorption FITC-YbbR 8 excitation wavelengths. We signal <br />                                                maintained concentrations 12.5 &#181;M arrived final concentration the <br />                                                screen TAMRA-mCoA 12 10 &#181;M. <br />  <br />                               Assay optimization: Requirement BSA enzyme stability <br />                                                The screen developed initiating reactions 96 plates varied enzyme <br />                                                concentration. These experiments monitored continuously using standard fluorescein <br />                                                optics, analysis progress curves revealed reaction rate did follow linearly <br />                                                respect enzyme concentration. This identified complication time- <br />                                                dependent inactivation enzyme using Selwyn&#8217;s method [34] (data shown). A number <br />                                                reaction additives, including buffer salt composition, ionic strength, detergents were <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                screened attempt stabilize enzyme, inclusion bovine serum <br />                                                albumin (BSA) concentration 1 mg/mL (15 &#181;M) stabilized enzyme eliminated <br />                                                complication. <br />  <br />                                                With modification, stable reaction progress plots observed (Fig. 5A &amp; C) that <br />                                                displayed linear relationship rate enzyme concentration (Fig. 5B &amp; D). From <br />                                                data, determined optimal enzyme concentrations 50 12.5 nM AcpS and <br />                                                Sfp, respectively, based obtaining complete progress curve terminating approximately <br />                                                30% substrate consumption 30 minute time interval (vide infra). It noteworthy that <br />                                                inclusion BSA traditionally considered unattractive screening campaign, <br />                                                carrier provides excellent source amine thiol moieties serve repress <br />                                                inhibition enzyme nonspecific electrophilic <span id='am-100' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-102' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> present diverse compound <br />                                                libraries. <br />  <br />                               DMSO tolerance <br />                                                Since compound libraries stocked solutions DMSO, sought evaluate the <br />                                                tolerance screen organic solvent. Inclusion organic solvent reaction <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                caused immediate 10% increase RFU signal substrate mixture did not <br />                                                deteriorate observed rate reaching concentration greater 10% total volume (data <br />                                                shown). For routine screening conditions, maintained final DMSO concentration <br />                                                5% v/v. <br />  <br />                               Progress curve analysis signal statistics <br />                                                With final assay conditions hand, screening protocol developed presented in <br />                                                Table 1. Following protocol, recorded progress curves analysed data <br />                                                statistically determine appropriate time analysis. Since known inhibitors PPTases <br />                                                commercially available, enzyme-free reaction used fully inhibited <br />                                                (negative) control. Analysis data (presented Fig. 6A &amp; B) demonstrated assay <br />                                                performance relied heavily substrate consumption allowed occur, <br />                                                parameter values better consumption (Table 2). However, <br />                                                excessive substrate consumption lead erroneous determination inhibition <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                             Page 8 <br />  <br />  <br />                                                characteristics [35], progression ~20% (Table 2, boxed values) chosen balance <br />                                                conditions minimum substrate consumed assay statistics reached <br />                                                acceptable values, including Z&#8217; values 0.7, signal noise 20 (Table 2). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Subsequently, experiment repeated separate days assess robustness of <br />                                                method, data presented Table 3. The Z&#8217; factor [36] used evaluate <br />                                                suitability screening method high throughput implementation, values greater <br />                                                0.5 considered satisfactory. The assay performed well, gave Z&#8217; values 0.72 <br />                                                &#177; 0.01 0.75 &#177; 0.02 Sfp AcpS, respectively. <br />  <br />                               Screen validation inhibitor identification <br />                                                To demonstrate ability screen identify inhibitors PPTases, evaluated the <br />                                                effects product inhibition 3&#8217;-phosphoadenosine-5&#8217;-phosphate (PAP) 3 (Fig. 1), the <br />                                                nucleotide product released enzyme, utilized McAllister et al. studies the <br />                                                reaction mechanism AcpS [13]. PAP serial diluted (2-fold) concentration of <br />                                                10 mM DMSO stocked polypropylene plates, evaluated assay both <br />                                                enzymes separate days (Fig. 7), demonstrate reproducibility screen. <br />                                                The method proved yield consistent inhibition curves, IC50 values 12.3 &#177; 1.1 &#181;M and <br />                                                92 &#177; 6 &#181;M Sfp AcpS, respectively; Hill coefficients analyses &#8722;1.06 &#177; <br />                                                0.09 Sfp &#8722;0.98 &#177; 0.11 AcpS. Taken together, data support suitability of <br />                                                method determine inhibitory characteristics compounds PPTase enzymes. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Conclusions <br />                                                In summary, developed homogenous screen PPTase requires minimum <br />                                                number liquid handling steps, meets criteria set forth NIH Chemical Genomics <br />                                                Center acceptable automation [37]. Importantly, method utilizes simple reagents <br />                                                prepared traditional techniques [24;38], pure chemical entities that <br />                                                require interbatch standardization, luxury afforded complex biochemical <br />                                                reagents. It anticipated method used evaluate crossreactivity of <br />                                                currently known AcpS inhibitors, identify new inhibitor architectures Sfp AcpS, <br />                                                work currently progress. <br />  <br />                               Abbreviations used <br />                                                4&#8217;-PP, 4&#8217;-phosphopantetheine <br />                                                ACP, acyl carrier protein <br />                                                CoA, coenzyme A <br />                                                mCoA, modified-coenzyme A <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                PPTase, phosphopantetheinyl transferase <br />                                                FRET, fluorescence resonance energy transfer <br />                                                DACM, 7-dimethylamino-4-methyl-coumarin-3-maleimide <br />                                                TAMRA, tetrametylrhodamine-5-maleimide <br />                                                PAP, 3&#8217;-phosphoadeosine-5&#8217;-phosphate <br />                                                UV-VIS, ultraviolet-visible <br />                                                FITC, fluorescein-5-isothiocyanate <br />                                                HEPES, 4-(2-hydroxyethyl)-1-piperazine ethanesulfonic acid <br />                                                BSA, bovine serum albumen <br />                                                FMOC, 9-Fluorenylmethyloxycarbonyl <br />                                                HATU, 2-(1H-7-Azabenzotriazol-1-yl)-1,1,3,3-tetramethyl uronium hexafluorophosphate <br />                                                DMSO, dimethyl sulfoxide <br />                                                RFU, relative fluorescence unit <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                                          Page 9 <br />  <br />  <br />                               Acknowledgements <br />                                                This work funded NIH R01GM075797 1R03MH083266. We thank Christopher T. Walsh (Harvard <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Medical School) <span id='am-187' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-188' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-191' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span> containing Sfp AcpS expression systems, Elizabeth A. Komives guidance <br />                                                assistance peptide synthesis, Adam Yasgar Anton Simeonov (NIH Chemical Genomics Center) for <br />                                                helpful discussions. <br />  <br />  <br />                               References <br />                                                1. Lambalot RH, Gehring AM, Flugel RS, Zuber P, LaCelle M, Marahiel MA, Reid R, Khosla C, Walsh <br />                                                   CT. A new enzyme superfamily - phosphopantetheinyl transferases. Chem Biol 1996;3:923&#8211;936. <br />                                                   [PubMed: 8939709] <br />                                                2. Quadri LE, Weinreb PH, Lei M, Nakano MM, Zuber P, Walsh CT. Characterization Sfp,a Bacillus <br />                                                   subtilis phosphopantetheinyl transferase peptidyl carrier protein domains peptide synthetases. <br />                                                   Biochemistry 1998;37:1585&#8211;1595. [PubMed: 9484229] <br />                                                3. Barekzi N, Joshi S, Irwin S, Ontl T, Schweizer HP. Genetic characterization encoding the <br />                                                   multifunctional phosphopantetheinyl transferase <span id='am-106' about='obo:NCBITaxon_286' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621'><span id='am-107' property="oboInOwl:hasDbXref" content="PMID:15950132" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasRelatedSynonym" content="Liquidomonas" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="Pseudomonas" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasRelatedSynonym" content="RNA similarity group I" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasDbXref" content="PMID:9103607" datatype="xsd:string"></span><span id='am-112'  ></span><span id='am-113' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasDbXref" content="PMID:10939664" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="PMID:10758879" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasRelatedSynonym" content="Peudomonas" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasDbXref" content="PMID:23918787" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasDbXref" content="PMID:18048745" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasDbXref" content="PMID:7727274" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasRelatedSynonym" content="Loefflerella" datatype="xsd:string"></span>Pseudomonas</span> aeruginosa. Microbiology <br />                                                   2004;150:795&#8211;803. [PubMed: 15073290] <br />                                                4. Finking R, Solsbacher J, Konz D, Schobert M, Schafer A, Jahn D, Marahiel MA. Characterization of <br />                                                   new type phosphopantetheinyl transferase fatty acid siderophore synthesis Pseudomonas <br />                                                   aeruginosa. J Biol Chem 2002;277:50293&#8211;50302. [PubMed: 12381736] <br />                                                5. Lambalot RH, Walsh CT. Cloning, overproduction, characterization <span id='am-171' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-172' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-173' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-174' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-175' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-176'  ></span><span id='am-177' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span>Escherichia</span> coli holo- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   acyl carrier protein synthase. J Biol Chem 1995;270:24658&#8211;24661. [PubMed: 7559576] <br />                                                6. Parris KD, Lin L, Tam A, Mathew R, Hixon J, Stahl M, Fritz CC, Seehra J, Somers WS. Crystal <br />                                                   structures substrate binding Bacillus subtilis holo-(acyl carrier protein) synthase reveal novel <br />                                                   trimeric arrangement molecules resulting active sites. Structure Folding &amp; Design <br />                                                   2000;8:883&#8211;895. [PubMed: 10997907] <br />                                                7. Reuter K, Mofid MR, Marahiel MA, Ficner R. Crystal structure surfactin synthetase-activating <br />                                                   enzyme Sfp: prototype 4 &#8216;-phosphopantetheinyl transferase superfamily. Embo Journal <br />                                                   1999;18:6823&#8211;6831. [PubMed: 10581256] <br />                                                8. Lam HM, Tancula E, Dempsey WB, Winkler ME. Suppression Insertions Complex Pdxj <br />                                                   Operon Escherichia-Coli K-12 Ion Other Mutations. Journal Bacteriology <br />                                                   1992;174:1554&#8211;1567. [PubMed: 1537800] <br />                                                9. Takiff HE, Baker T, Copeland T, Chen SM, Court DL. Locating Essential Escherichia-Coli Genes by <br />                                                   Using Mini-Tn10 <span id='am-82' about='obo:NCBITaxon_2387' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Transposon" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="transposons" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Transposons</span> - Pdxj Operon. Journal Bacteriology 1992;174:1544&#8211;1553. <br />                                                   [PubMed: 1537799] <br />                                                10. Chu M, Mierzwa R, Xu L, Yang SW, He L, Patel M, Stafford J, Macinga D, Black T, Chan TM, <br />                                                     Gullo V. Structure elucidation Sch 538415 novel acyl carrier protein synthase inhibitor a <br />                                                     microorganism. Bioorg Med Chem Lett 2003;13:3827&#8211;3829. [PubMed: 14552789] <br />                                                11. Gilbert AM, Kirisits M, Toy P, Nunn DS, Failli A, Dushin EG, Novikova E, Petersen PJ, Joseph- <br />                                                     McCarthy D, McFadyen I, Fritz CC. Anthranilate 4H&#8211;oxazol-5-ones: novel small molecule <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     antibacterial acyl carrier protein synthase (AcpS) inhibitors. Bioorg Med Chem Lett 2004;14:37&#8211;41. <br />                                                     [PubMed: 14684293] <br />                                                12. Joseph-McCarthy D, Parris K, Huang A, Failli A, Quagliato D, Dushin EG, Novikova E, Severina <br />                                                     E, Tuckman M, Petersen PJ, Dean C, Fritz CC, Meshulam T, DeCenzo M, Dick L, McFadyen IJ, <br />                                                     Somers WS, Lovering F, Gilbert AM. Use structure-based drug design approaches obtain novel <br />                                                     anthranilic acid acyl carrier protein synthase inhibitors. J Med Chem 2005;48:7960&#8211;7969. [PubMed: <br />                                                     16335920] <br />                                                13. McAllister KA, Peery RB, Meier TI, Fischl AS, Zhao G. Biochemical molecular analyses the <br />                                                     <span id='am-7' about='obo:NCBITaxon_1301' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="PMID:19620365" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="PMID:7537076" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="PMID:10555340" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="Streptococcus" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="PMID:1720654" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="PMID:8995803" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-16'  ></span><span id='am-17' property="oboInOwl:hasDbXref" content="PMID:14657115" datatype="xsd:string"></span>Streptococcus</span> pneumoniae acyl carrier protein synthase,an enzyme essential fatty acid <br />                                                     biosynthesis. J Biol Chem 2000;275:30864&#8211;30872. [PubMed: 10903317] <br />                                                14. Payne DJ, Gwynn MN, Holmes DJ, Pompliano DL. Drugs bad bugs: confronting challenges <br />                                                     antibacterial discovery. Nat Rev Drug Discov 2007;6:29&#8211;40. [PubMed: 17159923] <br />                                                15. Neres J, Labello NP, Somu RV, Boshoff HI, Wilson DJ, Vannada J, Chen L, Barry CE, Bennett EM, <br />                                                     Aldrich CC. Inhibition siderophore biosynthesis <span id='am-58' about='obo:NCBITaxon_1763' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1762 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003'><span id='am-59' property="oboInOwl:hasDbXref" content="PMID:1380284" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="PMID:1581193" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="PMID:7547284" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium Lehmann and Neumann 1896" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="PMID:1883713" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="PMID:2275850" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="PMID:7547304" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="Mycobacterium" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="PMID:7907223" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-70'  ></span><span id='am-71' property="oboInOwl:hasDbXref" content="PMID:8863452" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="PMID:16014496" datatype="xsd:string"></span>Mycobacterium</span> tuberculosis nucleoside <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                                      Page 10 <br />  <br />  <br />                                                     bisubstrate analogues: Structure-activity relationships nucleobase domain 5 &#8216;-O-[N-(salicyl) <br />                                                     sulfamoyl]adenosine. Journal Medicinal Chemistry 2008;51:5349&#8211;5370. [PubMed: 18690677] <br />                                                16. Stirrett KL, Ferreras JA, Jayaprakash V, Sinha BN, Ren T, Quadri LEN. Small molecules with <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     structural similarities siderophores novel antimicrobials <span id='am-73' about='obo:NCBITaxon_1773' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1762 obo:NCBITaxon_1763 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_77643'><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Bacillus tuberculosis" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium tuberculosis var. hominis" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="Bacterium tuberculosis" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Mycobacterium tuberculosis" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium tuberculosis typus humanus" datatype="xsd:string"></span><span id='am-80'  ></span><span id='am-81' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Mycobacterium tuberculosis</span> <br />                                                     <span id='am-145' about='obo:NCBITaxon_632' typeof='owl:Thing obo:NCBITaxon_629 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-146' property="oboInOwl:hasRelatedSynonym" content="Bacterium pestis" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasRelatedSynonym" content="Bacillus pestis" datatype="xsd:string"></span><span id='am-149'  ></span><span id='am-150' property="oboInOwl:hasRelatedSynonym" content="Pasteurella pestis" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasRelatedSynonym" content="Pestisella pestis" datatype="xsd:string"></span><span id='am-153' property="rdfs:label" content="Yersinia pestis" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasRelatedSynonym" content="Yersinia pseudotuberculosis subsp. pestis" datatype="xsd:string"></span>Yersinia pestis</span>. Bioorganic &amp; Medicinal Chemistry Letters 2008;18:2662&#8211;2668. [PubMed: <br />                                                     18394884] <br />                                                17. Ferreras JA, Stirrett KL, Lu XQ, Ryu JS, Soll CE, Tan DS, Quadri LEN. Mycobacterial phenolic <br />                                                     glycolipid virulence factor biosynthesis: Mechanism small-molecule inhibition polyketide <br />                                                     chain initiation. Chemistry &amp; Biology 2008;15:51&#8211;61. [PubMed: 18158259] <br />                                                18. Cisar JS, Ferreras JA, Soni RK, Quadri LEN, Tan DS. Exploiting ligand conformation selective <br />                                                     inhibition non-ribosomal peptide synthetase amino acid adenylation designed macrocyclic <br />                                                     small molecules. Journal American Chemical Society 2007;1297752-+ <br />                                                19. Meier JL, Barrows-Yano T, Foley TL, Wike CL, Burkart MD. The unusual macrocycle forming <br />                                                     thioesterase mycolactone. Molecular Biosystems 2008;4:663&#8211;671. [PubMed: 18493665] <br />                                                20. Lambalot RH, Walsh CT. Holo-[acyl-carrier-protein] synthase Escherichia coli, Vitamins and <br />                                                     Coenzymes. 1997;(Pt I):254&#8211;262. <br />                                                21. Glover KJ, Martini PM, Vold RR, Komives EA. Preparation insoluble transmembrane peptides: <br />                                                     Glycophorin-A, prion (110&#8211;137)and FGFR (368&#8211;397). Analytical Biochemistry 1999;272:270&#8211;274. <br />                                                     [PubMed: 10415099] <br />                                                22. Inglese J, Johnson RL, Simeonov A, Xia MH, Zheng W, Austin CP, Auld DS. High-throughput <br />                                                     screening assays identification chemical probes. Nature Chemical Biology 2007;3:466&#8211; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     479. <br />                                                23. Pettersen EF, Goddard TD, Huang CC, Couch GS, Greenblatt DM, Meng EC, Ferrin TE. UCSF <br />                                                     chimera - A visualization exploratory research analysis. Journal Computational <br />                                                     Chemistry 2004;25:1605&#8211;1612. [PubMed: 15264254] <br />                                                24. La Clair JJ, Foley TL, Schegg TR, Regan CM, Burkart MD. Manipulation carrier proteins in <br />                                                     antibiotic biosynthesis. Chem Biol 2004;11:195&#8211;201. [PubMed: 15123281] <br />                                                25. Haas JA, Frederick MA, Fox BG. Chemical posttranslational modification Escherichia coli <br />                                                     acyl carrier protein preparation dansyl-acyl carrier proteins&#8217;. Protein Expression and <br />                                                     Purification 2000;20:274&#8211;284. [PubMed: 11049751] <br />                                                26. Blommel PG, Fox BG. Fluorescence anisotropy assay proteolysis specifically labeled fusion <br />                                                     proteins. Analytical Biochemistry 2005;336:75&#8211;86. [PubMed: 15582561] <br />                                                27. Joshi NS, Whitaker LR, Francis MB. A three-component Mannich-type reaction selective tyrosine <br />                                                     bioconjugation. Journal American Chemical Society 2004;126:15942&#8211;15943. [PubMed: <br />                                                     15584710] <br />                                                28. Schlick TL, Ding ZB, Kovacs EW, Francis MB. Dual-surface modification tobacco mosaic <br />                                                     virus. Journal American Chemical Society 2005;127:3718&#8211;3723. [PubMed: 15771505] <br />                                                29. Yin J, Straight PD, McLoughlin SM, Zhou Z, Lin AJ, Golan DE, Kelleher NL, Kolter R, Walsh CT. <br />                                                     Genetically encoded short peptide tag versatile protein labeling Sfp phosphopantetheinyl <br />                                                     transferase. Proc Natl Acad Sci U S A 2005;102:15815&#8211;15820. [PubMed: 16236721] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                30. Zhou Z, Cironi P, Lin AJ, Xu YQ, Hrvatin S, Golan DE, Silver PA, Walsh CT, Yin J. Genetically <br />                                                     encoded short peptide tags orthogonal protein labeling sfp AcpS phosphopantetheinyl <br />                                                     transferases. Acs Chemical Biology 2007;2:337&#8211;346. [PubMed: 17465518] <br />                                                31. Lakowicz, JR. Principles fluorescence spectroscopy. New York: Kluwer Academic/Plenum; 1999. <br />                                                32. Machida M, Ushijima N, Machida MI, Kanaoka Y. Fluorescent Thiol Reagents.9 N-(7- <br />                                                     Dimethylamino-4-Methyl-Coumariny)Maleimides (Dacm) - Novel Fluorescent Thiol Reagents. <br />                                                     Chemical &amp; Pharmaceutical Bulletin 1975;23:1385&#8211;1386. <br />                                                33. Campbell, ID.; Dwek, RA.; Dwek, RA. Biological spectroscopy. Menlo Park, Calif: Benjamin/ <br />                                                     Cummings Pub. Co; 1984. <br />                                                34. Selwyn MJ. A Simple Test Inactivation Enzyme During Assay. Biochimica Et Biophysica <br />                                                     Acta 1965;105:193&#8211;194. [PubMed: 4221326] <br />                                                35. Wu G, Yuan Y, Hodge CN. Determining appropriate substrate conversion enzymatic assays in <br />                                                     high-throughput screening. Journal Biomolecular Screening 2003;8:694&#8211;700. [PubMed: <br />                                                     14711395] <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                                Page 11 <br />  <br />  <br />                                                36. Zhang JH, Chung TDY, Oldenburg KR. A simple statistical parameter use evaluation and <br />                                                    validation high throughput screening assays. Journal Biomolecular Screening 1999;4:67&#8211;73. <br />                                                    [PubMed: 10838414] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                37. NCGC, Assay Guidance Criteria. 2009. <br />                                                    http://www.ncgc.nih.gov/guidance/HTS_Assay_Guidance_Criteria.html March 31 2009 <br />                                                38. Wellings DA, Atherton E. Standard Fmoc protocols. Methods Enzymology 1997;289:44&#8211;67. <br />                                                    [PubMed: 9353717] <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                       Page 12 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 1. Post-translational modification carrier proteins <br />                                                The translated apo-carrier proteins 1 coenzyme A 2 react PPTase generate 3&#8217;- <br />                                                phosphoadenosine-5&#8217;-phosphate (PAP) 3 holo-carrier proteins 4. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                              Page 13 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 2. Fluorescence resonance energy transfer assay phosphopantetheinyl transferase <br />                                                Action PPTase fluorescently labeled acceptor substrate 5, conjunction a <br />                                                fluorphore-modified coenzyme A analogue (mCoA) 6, assembles FRET pair conversion <br />                                                thiol-blocked crypto-acceptor substrate 7. The assembly FRET pair be <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                detected fluorescence microplate reader. <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                       Page 14 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 3. Synthesis assay components <br />                                                (A) Beginning L-alanine loaded polystyrene resin, standard SPPS procedures afford the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                YbbR sequence appended &#949;-aminocaproic acid residue. Reaction FITC, followed <br />                                                cleavage HPLC purification gives FITC-YbbR peptide substrate 8. (B) CoA 2 is <br />                                                reacted maleimide bearing probes 9 10 produce fluorescent mCoA derivatives <br />                                                11 12, respectively. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                            Page 15 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 4. Probe selection photophysical evaluation crypto-YbbR peptides <br />                                                DACM-mCoA 11 TAMRA-mCoA 12 probes evaluated FRET <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                characteristics FITC-YbbR 8. (A) DACM-mCoA 11 acts FRET donor FITC-YbbR <br />                                                8. (B) Excitation emission spectra probes 11 8 recorded normalized to <br />                                                determine spectral overlap 11 emission 8 excitation calculate overlap integral <br />                                                J(&#955;) [Eq. (3)] theoretical Forster&#8217;s radius R0 [Eq. (2)] system. Summation the <br />                                                normalized data gave J(&#955;) 1.60 &#215; 10&#8722;13cm4 R0 =38&#197;. (C) FRET characterization of <br />                                                crypto-DACM-FITC-YbbR peptide. Equimolar solutions 11 8 prepared with <br />                                                varying degree modification crypto-DACM-FITC-YbbR 7, fluorescent <br />                                                spectra recorded excitation 403nm. The spectra overlayed, increase 10% <br />                                                increments modification 0 100%; starting maximal emission observed at <br />                                                462nm minimum wavelength. (D) Eq. (5) fitted emission data (C) and <br />                                                gave efficiency transfer value 0.82. (E) FITC-YbbR 8 acts FRET donor to <br />                                                TAMRA-mCoA 12. (F) Excitation emission spectra probes 11 12 recorded <br />                                                normalized determine spectral overlap 11 emission 12 excitation calculate <br />                                                overlap integral J(&#955;) [Eq. (3)] theoretical Forster&#8217;s radius R0 [Eq. (2)] system. <br />                                                Summation normalized data gave J(&#955;,) 3.01 &#215; 10&#8722;13cm4 R0 =56&#197;. (G) FRET <br />                                                characterization crypto-TAMRA-YbbR peptide. Equimolar solutions 11 12 were <br />                                                prepared varying degree modification crypto-TAMRA-YbbR 7, their <br />                                                fluorescent spectra recorded excitation 475nm. The spectra overlayed, increase <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                10% increments modification 0 100%; starting maximal emission <br />                                                observed 513nm spectrum minimum value wavelength. <br />                                                (H) Eq. (5) fitted emission data 513nm (G) gave efficiency transfer <br />                                                value 0.99. <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                          Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 5. Reaction progress enzyme titration <br />                                                (A) Reaction progress plots concentrations AcpS. Rates determined the <br />                                                experiments intersect dashed threshold 32,000 RFU. (B) Reaction rates for <br />                                                progress curves A plotted AcpS concentration demonstrate linear <br />                                                response. (C) Reaction progress curves concentrations Sfp. Rates determined <br />                                                experiments intersect dashed threshold 32,000 RFU. (D) Reaction rates <br />                                                progress curves C plotted Sfp concentration demonstrate linear <br />                                                response. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                               Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 6. Determination analysis timepoint Z&#8217; <br />                                                (A) Progress curve 96 plate containing 48 reactions presense (&#65533;?) absence <br />                                                (&#9642;) 50 nM AcpS The data timepoint analysed statistically presented in <br />                                                Table 2. The 24 minute time point selected routine analysis, gave acceptable assay <br />                                                statistics linear range screen. (B) Change relative fluorescence units the <br />                                                24 minute time point (Table 2, boxed data) plotted number. The Z&#8217; value for <br />                                                data 0.72. (C) Progress curve 96 plate containing 48 reactions presense <br />                                                (&#65533;?) absence (&#9642;) 12.5 nM Sfp. The data timepoint analysed statistically and <br />                                                presented Table 2. The 16 minute time point selected routine analysis, gave <br />                                                acceptable assay statistics linear range screen. (B) Change relative <br />                                                fluorescence units 16 minute time point (Table 2, boxed data) plotted well <br />                                                number. The Z&#8217; value data 0.71. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                       Page 18 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 7. Reproducibility IC50 values 3&#8217;-phosphoadenosine-5&#8217;-phosphate <br />                                                (A) Sfp screened 12 concentrations PAP ranging 244 nM 500 &#181;M on <br />                                                separate days. The data fit four-parameter dose response curve using <br />                                                GraphPad returned IC50 values 13, 11, 13 &#181;M. (B) Replication same <br />                                                experiment A.) using AcpS enzyme. These curves returned IC50 values 86, <br />                                                 <br /> </body></html>