<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, May 1991, p. 3191-3198                                                                              Vol. 173, No. 10 <br /> 0021-9193/91/103191-08$02.00/0 <br /> Copyright C 1991, American Society Microbiology <br />  <br />  <br />   Molecular Cloning, Nucleotide Sequence, Characterization of <br />  <span id='am-1' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-2' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10'  ></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Bacillus subtilis</span> Gene Encoding DNA-Binding Protein HBsu <br />          BIRGIT MICKA,&apos; NICOLAS GROCH,2 UDO HEINEMANN,2 AND MOHAMED A. MARAHIELl* <br />  Fachbereich Chemie/Biochemie, Philipps-Universitdt Marburg, Hans-Meerwein-Strasse, D-3550 Marburg,&apos; Institut <br />             fur Kristallographie, Freie Universitat Berlin, D-1000 Berlin 33,2 Federal Republic Germany <br />                                            Received 5 October 1990/Accepted 15 March 1991 <br />  <br />               A homologous <span id='am-39' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> histonelike proteins believed wrap DNA condense the <br />            chromosome highly folded nucleoid structures identified different bacterial <span id='am-52' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-53' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>. Bacillus <br />            subtilis encodes homodimeric DNA-binding protein called HBsu. We cloned corresponding gene <br />            (hbs) 3.8-kb fragment. The gene subcloned 1-kb fragment, sequenced, characterized. It <br />            encodes 92-amino-acid protein predicted molecular mass 9,884 Da. Fortunately, analysis DNA <br />            sequence downstream 3&apos; end hbs revealed location 19 amino acid residues MtrA. <span id='am-42' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-45'  ></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> <br />            finding located hbs gene unequivocally 5&apos; end mtr operon 2040 standard genetic <br />            map B. subtilis. Northern (RNA) blot analysis primer extension studies indicated presence two <br />            distinct hbs transcripts, initiated different sites located 160 bases apart. <br />            Several attempts replace hbs gene B. subtilis chromosome cat-interrupted copy (hbs::cat) <br />            marker replacement recombination unsuccessful. In <span id='am-55' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> study hbs essential <br />            gene, constructed strain containing truncated copy gene promoter and <br />            intact copy control isopropyl-,-D-thiogalactopyranoside (IPTG)-inducible spac-1 pro- <br />            moter. In strain (BM19), normal growth depend IPTG, absence IPTG, <br />            growth severely affected. These results suggest essential role hbs gene product normal growth <br />            B. subtilis. <br />  <br />    In procaryotes, number abundant, small, basic,                  protein (49). The identity nucleoid proteins B. <br /> heat-stable proteins identified. These proteins                   subtilis (34, 39) HBsu remains unproven. <br /> wrap DNA obvious sequence specificity                      As initial step understanding physiological <br /> homologous eucaryotic histones (for reviews,                   role B. subtilis protein, cloned, sequenced, <br /> references 6, 11, 37, 41, 46). Among bacteria,                      characterized corresponding gene. The gene was <br /> primary structures highly conserved. They                     identified genomic library constructed Lambda Zap <br /> designated histonelike DNA-binding proteins (41). The                    II using radiolabeled DNA probe synthesized the <br /> best-studied abundant protein HU, present                    polymerase chain reaction (PCR). <br /> <span id='am-73' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-74' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-77'  ></span><span id='am-78' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span>Escherichia coli</span>. It exists predominantly heterodimer <br /> (HU2 HU1) encoded genes hupA hupB (25,                                     MATERIALS AND METHODS <br /> 26, 30, 32). The subunits consist 90 amino acid <br /> residues 70% identical residues. Since HU                       Bacterial strains, <span id='am-127' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-128' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-130' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span>, culture media. Bacterial <br /> protein alters structure topology bound DNA (3,                  strains listed Table 1. Plasmid pDH88 been <br /> 6, 7, 37, 38), affects cellular processes binding          described Henner (16). 2xYT NZY media as <br /> DNA. Its role DNA replication especially docu-                   SM buffer described Short et al. (43). <br /> mented (5, 10, 35, 42). Also, function accessory                      Construction oligonucleotides. The DNA sequence the <br /> protein stimulating DNA recognition proteins has <br /> been postulated (9). In addition, recent studies hupA and <br /> hupB null mutants revealed growth, lysogeny, transpo- <br /> sition, cell division affected absence HU                                   TABLE 1. Bacterial strains <br /> protein (19, 35).                                                               Strain*                                            ~~~~~~~~~~~~Source <br />                                                                                                                                        rceor <br />    Several proteins properties structures highly                                                Genotype                   reference <br /> homologous HU protein isolated                   B. subtilis <br /> other bacteria bacteriophages. Among Bacillus                        JH642       trpC2 pheAl                             J. Hoch <br /> species (B. caldolyticus, B. globigii, B. stearothermophilus,                 BM04        trpC2 pheAl hbs+ hbs::cat               This paper <br /> and B. subtilis), protein highly homologous HU, classi-                  BM19        trpC2 pheAl PhbS-hbs&apos;::cat::Pspac-hbs   This paper <br /> fied HB, isolated characterized (1, 4, 21, 27). In                                  hbs+ <br /> contrast HU, HB exists homodimeric protein (44). <br /> This protein binds single-stranded double-                      E. coli <br /> stranded DNA RNA (4, 21, 22, 48). The HB protein B.                    XL-1 Blue recAl endAl gyrA96 thi hsdRJ7 (rK-        43 <br /> subtilis, HBsu, consists 92 amino acid residues differs                            MK+) supE44 relAl X- lac [F&apos; <br /> only sequence position B. globigii HB                                     proAB lacIqZAM15 TnJO (Tetr)] <br />                                                                               JM105     endAl (lac pro) thi-J strA sbcBJ5         J. Messing <br />                                                                                           hsdR4 (F&apos; traD36 proA+B+ <br />                                                                                             lacPZAM15) <br />   * <br />       Corresponding author. <br />                                                                      3191 <br />  3192     MICKA ET AL.                                                                                                          J. BACTERIOL. <br />  <br />           A                             B                               method Krause et al. (28). Genomic DNA a <br />                                                     1 2 3               template mixed oligonucleotides prim- <br />                   1 2 3                                                 ers. PCR carried 50-,ul volume containing 1 ,ug <br />           kb                            kb                              template DNA, 200 pmol primer (100 pmol each <br />                                                                         oligonucleotide), 50 mM Tris-HCl (pH 9.0 25&#176;C), 15 mM <br />                                                                         ammonium sulfate, 7 mM magnesium chloride, 50 mM <br />          23 -                                                           potassium chloride, 1 mM (each) deoxynucleoside triphos- <br />           9A -                                                          phates, 170 ,ug bovine serum albumin ml, 1 U of <br />           6.5 -                                                         <span id='am-58' about='obo:NCBITaxon_271' typeof='owl:Thing obo:NCBITaxon_188786 obo:NCBITaxon_188787 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_68933 obo:NCBITaxon_270 obo:NCBITaxon_1297'><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="Thermus aguaticus" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="PMID:5781580" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Thermus thermophilus/T.aquaticus/T.flavus" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="Thermus aquaticus" datatype="xsd:string"></span><span id='am-65'  ></span>Thermus aquaticus</span> DNA polymerase (Anglian Biotec). The <br />                                         5.4&apos; _-                         DNA denatured 93&#176;C 1 min, annealed 50&#176;C for <br />           4.3-                          4.3       - <br />                                                                         3 min, extended 72&#176;C 3 min. Following 30 cycles <br />                                                                         amplification, 2 ptl 50-pld reaction mixture was <br />           2.3-                                                          electrophoresed 1% agarose gel. The product of <br />           2.0-                                                          amplification DNA fragment 241 bp length. <br />                                                                            Hybridization conditions PCR product. Nitrocellulose <br />                                                                         filters prehybridized 6x SSC (lx SSC 0.15 M <br />                                                                         NaCl plus 0.015 M sodium citrate)-10 mM EDTA (pH <br />                                                                         8.0)-100 p,g denatured salmon sperm DNA ml 1 h <br />                                                                         65&#176;C. The PCR probe labeled Multiprime <br />                                                                         DNA-labeling (Amersham). The probe added to <br />                                                                         prehybridization mix hybridized 5 h 65C. <br />   FIG. 1. Identification hbs gene B. subtilis. (A) Restric-   Southern blots plaque- colony-screening mem- <br /> tion analysis genomic DNA isolated B. subtilis JH642.           branes subjected washes: 2x SSC and <br /> Chromosomal DNA digested restriction enzymes          0.5% sodium dodecyl sulfate (SDS) 15 min room <br /> HindIII (lane 1), EcoRI (lane 2), PstI (lane 3). The cleaved        temperature 2x SSC 0.1% SDS 15 min at <br /> products separated visualized 0.8% agarose gel. (B)       65&#176;C. Filters exposed Kodak X-Omat film. <br /> Southern blot analysis revealed major band digestion        Construction genomic library. Genomic DNA B. <br /> HindlIl (lane 1), EcoRI (lane 2), PstI (lane 3) restriction         subtilis JH642 partially digested restriction <br /> enzymes hybridization PCR probe. The probe recog-          enzyme Sau3A. Bacteriophage Lambda Zap II used as <br /> nized fragment 4.3 kb HindIll digest        vector clone genomic DNA according instructions <br /> fragment 5.4 kb EcoRI PstI digests. <br />                                                                         supplier (Stratagene). Ten micrograms vector DNA <br />                                                                         linearized restriction enzyme XhoI. <br />                                                                            To ligate complementary ends, dCTP dTTP used <br /> oligonucleotides used PCR based amino acid             XhoI-digested vector, dATP dGTP <br /> sequence HBsu protein (49). The oligomers represent              used Sau3A-digested chromosomal DNA. <br /> distinct parts chemically synthesized hbs gene (14).             After polymerization, 1 pLg Lambda Zap II vector 0.6 <br /> Both oligonucleotides synthesized Applied Bio-               jig insert DNA packaged Gigapack II Gold <br /> systems automatic DNA synthesizer purified hydro-                packaging extract (Stratagene). The ratio nonrecombinant <br /> phobic interaction anion-exchange fast protein liquid               (blue) recombinant (white) plaques determined by <br /> chromatography. Final purification achieved poly-                plating 0.1 pu 500-pdl DNA library 200 pl XL-1 <br /> acrylamide gel electrophoresis. One oligomer, 26-oligonu-             Blue host cells NZY plates containing 10 mM isopropyl- <br /> cleotide sequence, 5&apos;-CATTAACGCCGTTGCCGAAGCC                            ,B-D-thiogalactopyranoside (IPTG) 6.25 mg X-Gal <br /> AGCG-3&apos;, matches N-terminal HBsu sequence,                   (5-bromo-4-chloro-3-indolyl-p-D-galactopyranoside) ml <br /> the oligomer, 50-mer, 5&apos;-TTTT-lAAAGCTTTGCCC                     (43). The JH642 library contained 4. 106 PFU/pug DNA <br /> GGTTTAAAGGCCGGTACCTTGCTGGCCGGAATT                                       1. 10&apos; blue background plaques. <br /> T-3&apos;, matches C-terminal amino acid sequence.                      The library amplified, phages transferred to <br />   Amplification bacterial DNA PCR. PCR used                nitrocellulose filters. Phage DNA released the <br /> amplify genomic DNA fragments B. subtilis. Chromo-                   phage head denatured standard procedures (2). <br /> somal DNA JH642 (trpC2 pheAl) prepared according                 Methods hybridization wash conditions described <br />  <br />  <br />           A     pHBS1                                                                                                    2 <br />                            .C <br />                4s <br />                                                                                                                          3I&quot; <br />                    I= <br />                                                                         1                          31 <br />                4T._c                          1                         2                          3                     3.8kb <br />           B       pHBS2 =                     &#162; <br />                      _    .                   0 <br />               --51 &apos; #&apos;AS                     ok <br />  <br />                                               1kb <br />   FIG. 2. Restriction map DNA vicinity hbs gene. (A) Restriction map 3.8-kb DNA insert plasmid pHBS1. (B) <br /> Restriction map 1.0-kb DNA insert subcloned plasmid pHBS2. The location orientation hbs coding sequence indicated. <br />  VOL. 173, 1991                                                                                                                         DNA-BINDING PROTEIN OF B. SUBTILIS                                3193 <br />  <br />                                                    -35                                                     -10                         r-P1 <br />                           1    AAATGC gATGGCTTTTTATATGTGTTACTACATACAGAAATTCTTCACTTTGTT                                                                                                       60 <br />                         61     GGACAAACATTCCTCAGAGTGCAGTTTTTCTTAAAAAGCCGTTTAATTGCCTTTCTCTTA                                                                                                 120 <br />                        121     CTTGCTCTCATTTTTTTCTGAGACAGGTTTAGAATCAGACTGAACTGTGAAGAAATGATA                                                                                                 180 <br />  <br />                                                    r-P2                m-hbs <br />                        181     ATAAACGAACTGAATGTATCCTTTTGGGAGGTGAAAGGCATGAACAAAACAGAACTT                                                                                                    240 <br />                                                                                                            SD                              M       N       K       T       E       L <br />                        241     ATCAATGCGGTTGCAGAAGCAAGCGAATTGTCTAAAAAAGACGCTACAAAAGCAGTTGAC                                                                                                 300 <br />                                I       N       A       V       A       E       A       S       E   L       S       K K             D       A       T       K       A       V       D <br />                        301     TCTGTTTTTGATACGATCTTAGATGCACTTAAAAACGGTGATAAAATCCAACTGATCGGT                                                                                                 360 <br />                                S       V       F       D       T       I       L       D       A   L K N                   G       D       K       I       Q       L       I       G <br />                        361     TTTGGTAACTTCGAGGTGCGTGAACGTTCTGCACGTAAAGGACGCAATCCTCAAACAGGT                                                                                                 420 <br />                                F       G       N       F       E       V       R       E       R   S       A       R       K       G       R&apos; N            P       Q       T       G <br />                        421     GAAGAAATCGAAATTCCAGCAAGCAAAGTACCTGCTTTCAAACCAGGTAAAGCGCTTAAA                                                                                                 480 <br />                                E       E       I       E       I       P       A       S       K   V       P       A       F       K       P       G       K       A       L       K <br />                        481     GACGCAGTTGCCGGAAAATAATTGTGAATATAGATCGTGTATGCATCTAGCTTACATACA                                                                                                 540 <br />                                D       A       V *     A       G       K                       *           * <br />                        541     CTTTATTTCTTCACAGAAAAGCCCTxCT6AGGGGTTTATATTTCAAGAGCATGGG                                                                                                      600 <br />                        601     CTTCCTGACAGGGCATTCACTTTGCTTTTAGCGGGGCATATGTGCTAGAATCGAAATTAA                                                                                                 660 <br />                                                                                                        -   mtrA <br />                        661     ATGTATTCATTGGTGGAGGACATAGAACATGAAAGAAGTTAATAAAGAGCAAATCGAACA                                                                                                 720 <br />                                                                                                        M       K       E       V       N       K       E       Q       I       E       Q <br />                        721     AGCTGTTCGTCAAATTTTAGAAGCGA                                                                                                                                   746 <br />                                    A       V       R       Q       I       L       E       A <br />    FIG. 3. DNA sequence hbs gene deduced amino acid sequence. The hbs gene 279 bp length encodes protein of <br /> 92 amino acid residues. The -10 -35 regions, putative ribosome binding site (SD), sequence potential transcription <br /> terminator underlined. Arrows indicate 5&apos; ends (P1 P2) mapped primer extension. The 5&apos; sequence mtrA gene the <br /> first 19 amino acid residues shown. <br />  <br />  <br /> above. Promising plaques      isolated agar plate                                                    tary sequence located nucleotides 111 and <br /> and transferred 500 RI     SM buffer containing 20 ,ul                                                    158. Reverse transcriptase nucleotides dATP, dGTP, <br /> chloroform (43).                                                                                                   dTTP, radiolabeled [a-32P]dCTP used extend <br />   In vivo excision pBluescript plasmid Lambda Zap II                                                       primer produce cDNA, used in <br /> vector. The Bacillus library constructed Lambda                                                        mapping 5&apos; termini mRNA hbs. <br /> Zap II use facility vivo excision cloned                                                          Strain constructions interrupt chromosomal copy of <br /> insert vector. The phagemid, pBluescript <br /> [SK(-)], excised fl helper phage Lambda <br /> Zap II incorporates signals initiation termi- <br /> nation DNA synthesis fl origin replication                                                                                                                                  12 <br /> (43). After vivo excision lambda vector, colonies                                                                                        kb <br /> that appeared plates transferred nylon filters and <br /> denatured standard procedures (2). Methods <span id='am-122' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-123' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-124' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-126'  ></span>hybrid</span>- <br /> ization wash conditions described previously <br /> (40). <br />    DNA sequencing. DNA sequencing cloned plasmid <br /> DNA denatured alkali carried chain                                                                                                                                              -23S <br /> termination reaction (United States Biochemical). The <br /> chain-terminated products separated 6% sequencing <br /> gels visualized overnight autoradiography                                                                                                                                             -16S <br /> Kodak X-Omat film. <br />    RNA preparation Northern (RNA) blot. Fifty-milliliter                                                                                       0.540- <br /> cell cultures B. subtilis collected different                                                                                           0.38 w                      &apos; <br /> times vegetative growth. Total cellular RNA was <br /> prepared procedure Penn et al. (36) modified by <br /> Igo Losick (20). RNA pellets suspended 50 ,ul of <br />  diethylpyrocarbonate-treated water. The transfer RNA                                                                                                                                          -5S <br />  gel blot performed method Igo                                                          FIG. 4. Northern blot analysis total cellular RNA (5 ,ug) B. <br />  Losick (20).                                                                                                  subtilis prepared different periods vegetative growth <br />    Primer extension. Primer extension used map 5&apos;                                                       (optical densities 600 nm 0.5 1.0) (lanes 1 2, respec- <br />  termini mRNA described reference 40. The RNA                                                             tively). With PCR probe, transcripts 0.54 0.38 kb <br />  hybridized excess single-stranded DNA                                                              identified. To right indicated locations the <br />  primer. The primer, 50-mer oligonucleotide, complemen-                                                       rRNAs. <br />                                                                   a <br /> 3194    MICKA ET AL.                                                                                                         J. BACTERIOL. <br />  <br />  <br />         A <br />                                                                                          hbs&apos; <br />                                         pDH 88                       + <br />                                                                             HindNi                    Sph I <br />                           pspac <br />                               Hind IN Sph I <br />  <br />  <br />                                                                       I <br />  <br />                                                           Phbs <br />                HindNi EcoRI                                                                                          HindM EcoRI <br />  <br />  <br />                                                                       I <br />                                           hbS&apos;                         p                        hbS <br />                       phbs                                                 spac   Ib                             1 <br />                HindlilEcoRI                        Sphl                    HindM                                     HindM EcoRI <br />                                                                         FIG. 5. (A) Construction integration pDH88hbs&apos; homol- <br />          B                                                           ogous recombination. The plasmid indicated solid lines, the <br />                           1 2                                        chromosomal DNA indicated dotted lines. The box indicates the <br />                                                                      hbs coding region, jagged ends hbs&apos; indicate truncated <br />              kb                               Kt                     copy gene. The positions promoters shown Pspac <br />                                                                      Phbs. The locations restriction sites HindIII, EcoRl, and <br />                                                                      SphI shown. The figure scale. (B) Characterization B. <br />                                                                      subtilis BM19 Southern hybridization. Chromosomal DNAs B. <br />                                                                      subtilis wild type (lane 1) BM19 (lane 2) digested with <br />                                                                      HindIII-EcoRI. The cleaved products separated 0.8% <br />                                                                      agarose gel, transferred nitrocellulose filter, hybridized a <br />                                                                      PCR probe. Arrowheads indicate locations hbs gene the <br />                                                                      wild type (2.3 kb) BM19 strain containing truncated <br />                                                                      copy (6.5 kb) Pspac-hbs fusion (1.8 kb). <br />  <br />              2.3        4W                                           promoter integrative plasmid pDH88, 191-bp <br />                                                                      fragment flanked restriction sites Hindlll SphI <br />                                                                      synthesized PCR. The HindIII-SphI PCR fragment <br />                                                                      contained natural ribosomal-binding site (SD) the <br /> the hbs gene place control spac-)             59 92 codons hbs. This truncated copy pDH88 <br /> promoter. Plasmid pHBS2 linearized hbs                digested HindlIl SphI, ligated, trans- <br /> gene using restriction enzyme HincII. The cat gene            formed competent cells E. coli JM105. The new <br /> localized 1.3-kb EcoRI fragment pZA327 (50)              construct, pDH88hbs&apos;, transformed competent <br /> isolated. To ligate blunt-ended fragments, dATP dTTP             cells B. subtilis JH642. Transformants selected on <br /> were filled EcoRI sites. After ligation, hbs::cat       2xYT plates containing 5 ,ug chloramphenicol ml and <br /> plasmid (pHBS2cat) transformed competent cells           1 mM IPTG. <br /> E. coli JM105 (40). About 2 tg linearized pHBS2cat               Antisera. Antisera HB protein B. globigii, <br /> used transform competent cells B. subtilis JH642 (15).         HBgl, raised rabbits used dilution 1:500 <br /> Transformants selected 2xYT plates containing 5 ,g           purification. Polyclonal anti-HBgl antibodies <br /> of chloramphenicol ml.                                           cross-reacted B. subtilis HBsu protein. Total pro- <br />   To place hbs gene control spac-J              tein extracts B. subtilis E. coli cells containing the <br />  VOL. 173, 1991                                                                     DNA-BINDING PROTEIN OF B. SUBTILIS                     3195 <br />  <br />        2o                                                               A                                             B <br />                                                                                5     4    3   2        1                      1   2 3   4 5 <br />  <br />        1.0 <br />                                                                                     5kl   _ <br />  <br />  <br />  <br />  <br />        0.5                                                                                        P~~~~ffM   --- <br />                                                                                                                    HtIB P-- <br />  <br />  <br />                                                                           FIG. 7. Western blot analysis HBsu protein B. subtilis. (A) <br />                                                                        Coomassie brilliant blue R 250 stain total cellular protein analyzed <br />                                                                        15% SDS-PAGE B. subtilis JH642 cells (lane 5), E. coli cells <br />                                                                        bearing recombinant plasmid pHBS1 (lane 4), vector DNA (lane <br />                                                                        3), plasmid (lane 2). (B) Proteins transferred to <br />                                                                        nitrocellulose filter incubated anti-HBgl antibodies the <br />                                                                        chromogenic substrate 5-bromo-4-chloro-3-indolyl-phosphate. Puri- <br />  0 <br />  z <br />  <br />        0.1                                                             fied HBgl protein (lane 1) used internal control. <br />  <br />  C.3 <br />                                                                        4.3 kb HindIII digest. This suggested presence of <br />        0.05X                                                           hbs gene fragments. <br />                                                                           Cloning, sequencing, transcriptional mapping the <br />                                                                        hbs gene. The efficiency constructed genomic library <br />                                                                        high, 1% PFU without <br />                                                                        inserted DNA. About 1 RI (0.002%) packaged material <br />                                                                        amplified gave rise 8_ 103 PFU/,ul. The <br />                                                                        plaques transferred nylon membranes screened <br />                                                                        PCR probe. Five plaques (L-1 L-5) identi- <br />                                                                        fied. After purification reinfection, (L-1) was <br />                                                                        subjected excision process. By using 105 phage <br />        0.01-                 I                                  I      particles L-1, ampicillin-resistant colonies ap- <br />              0      1        2         3        4      5        6      peared. After colony filter hybridization screening with <br />                                  TIME (hours)                          PCR probe, promising clone identified. A <br />                                                                        plasmid, designated pHBS1, containing 3.8-kb DNA insert <br />   FIG. 6. Effect IPTG growth B. subtilis BM19 bearing     identified. By Southern blot analysis, hbs gene was <br /> Pspac-hbs fusion. Cultures  grown 2xYT 5 F&#177;g of <br /> chloramphenicol ml presence (U) absence (a)   assigned 900-bp EcoRI-NdeI fragment 3.8-kb <br /> 1 mM IPTG.                                                             insert pHBS1 (Fig. 2A). Plasmid pHBS2, bearing hbs <br />                                                                        gene 1-kb insert, constructed elimination of <br />                                                                        2.8-kb EcoRI insert pHBS1 (Fig. 2B). <br />                                                                           By using T7 primer synthetic oligonucleotides, the <br /> recombinant plasmid pHBS1 pHBS2 analyzed                    sequence major inserted DNA pHBS2 was <br /> SDS-polyacrylamide gel electrophoresis (PAGE) (15% poly-               determined. The hbs gene 279 bp length encodes a <br /> acrylamide) described Laemmli (29). Gels stained            protein 92 amino acid residues molecular mass of <br /> with Coomassie brilliant blue R 250 transferred nitro-           9,884 Da (Fig. 3). A strong potential ribosome-binding site <br /> cellulose immunodetection analysis (Western blot)               (AGGAGGT) AG -18.8 kcal (1 cal = 4.184J) is <br /> described Towbin et al. (45).                                       located 7 bp upstream initiation <span id='am-47' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49'  ></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> AUG (33). The <br />    Nucleotide sequence accession number. Sequence <span id='am-117' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-118' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-119'  ></span><span id='am-120' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span>              hbs open reading frame ends stop codons fol- <br /> this work assigned EMBL GenBank data base                    lowed hairpin loop structure similar rho-independent <br /> accession number X52418.                                               terminators. The deduced amino acid sequence hbs <br />                                                                        gene identical purified HBsu <br />                                                                        protein (49). Also, sequence open reading <br />                              RESULTS                                   frame located downstream hbs gene determined. <br />                                                                        This sequence identical 5&apos; end <br />   Construction PCR probe hybridization               mtrA gene determined Gollnick et al. <br /> chromosomal DNA. To verify quality successfully             (12). As shown Fig. 3, determined sequence mtrA, <br /> amplified PCR probe (a 241-bp fragment chromosomal              located 3&apos; end hbs, comprises 58 nucleo- <br /> hbs gene), labeled probe hybridized chromo-               tides encode 19 amino acid residues the <br /> somal DNA blot washed stringent conditions                MtrA protein (12). The sequence determined Gollnick et <br /> indicated Materials Methods. Chromosomal DNA                    al. 5&apos; end mtrA gene overlaps sequence <br /> that cleaved restriction enzymes HindIII,            region 453 bp. This region comprises 3&apos; end of <br /> EcoRI, PstI revealed corresponding Southern blot            hbs, 5&apos; end mtrA, region 175 bp located in <br /> the presence major band digest (Fig. 1). The            between. These data revealed exact location hbs <br /> probe recognized band 5.4 kb EcoRI             gene 5&apos; end mtr operon located 2040 on <br /> PstI chromosomal digests smaller fragment               Bacillus chromosome. <br />  31%      MICKA ET AL.                                                                                                       J. BACTERIOL. <br />  <br />               1          10          20           30           40          50           60           70          80           90 <br />  <br />   HBsu <br />   HBgl <br />   HBst        ------------QT-G-----------A---NS-T--RK---V----------A------------A-----------------K---LKY <br />   HBca        --------T--- I-G-----------A-- &quot;-TZ ---K---V-------------A------------N--------------------K <br />   TF-1        -------K-I-QDT--TQVSVS-JLA-FEKITTLWTAK---V--T--L-3[KPVA-Q----F----Q-AL--AP-VGVSV--- ES--K-AE-LKYED <br />  <br />  <br />                        di                        e2                                         arm region                     63 <br />   FIG. 8. Comparison amino acid sequences HB proteins <span id='am-109' about='obo:NCBITaxon_genus' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-110' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="genus" datatype="xsd:string"></span>genus</span> Bacillus. Proteins obtained B. subtilis (HBsu), <br /> B. globigii (HBgl), B. stearothermophilus (HBst), B. caldolyticus (HBca), B. subtilis bacteriophage SPOl (TF-1). At lower part, the <br /> locations conserved a-helices potential DNA-binding arm region shown. Only amino acid residues different in <br /> B. subtilis shown proteins. <br />  <br />  <br />    In order determine level size hbs                  corresponds HBsu protein molec- <br /> transcript, Northern blot primer extension analyses                 ular mass purified HBgl protein (Fig. 7, lane 1). In <br /> were carried out. Northern blot analysis total cellular              contrast, anti-HBgl antibodies did cross-react pro- <br /> RNA prepared different stages vegetative growth                   tein extract E. coli control strain (Fig. 7, lane 2) E. <br /> (optical densities 600 nm 0.5 1.0) indicated              coli containing pBluescript (Fig. 7, lane 3). However, one <br /> presence distinct hbs transcripts constant             immunoreactive polypeptide detected protein ex- <br /> level (Fig. 4). The approximate lengths transcripts             tracts E. coli JM105 cells bearing recombinant plas- <br /> were estimated 0.54 0.38 kb. In addition,               mids pHBS1 (Fig. 7, lane 4) pHBS2 (data shown). The <br /> primer extension studies (not shown) confirmed 5&apos;             polypeptide synthesized E. coli strains carrying the <br /> ends transcripts located 160 bases apart.              recombinant plasmids molecular mass 10 kDa, <br /> Inspection DNA sequence (Fig. 3) upstream                 identical B. subtilis HBsu protein. This <br /> putative initiation site P1 -10 -35 revealed               expression independent external vector promoter; <br /> consensus sequences recognized                      plasmids, pHBS1 pHBS2, lacZ promoter <br /> vegetative form RNA polymerase (EorA) B. subtilis. In             located 3&apos; end hbs gene. <br /> contrast, -10 -35 regions upstream P2 (not <br /> underlined Fig. 3) good homology conserved                                     DISCUSSION <br /> consensus sequence recognized minor form RNA <br /> polymerase containing sigma-32 (ETC) (24). However, addi-                  We cloned, sequenced, characterized hbs <br /> tional studies needed direct evaluation              gene encoding B. subtilis histonelike protein HBsu. <br /> putative promoters.                                                     According knowledge, hbs represents identi- <br />    Interruption placing hbs gene control           fied gene histonelike protein Bacillus spe- <br /> spac-l promoter. Many attempts interrupt hbs gene                cies. The gene encodes protein 92 amino acid residues <br /> directly double-crossover recombination event using                molecular mass 9,884 Da. The deduced amino acid <br /> the linearized pHBS2cat plasmid (see Materials Meth-                sequence hbs gene identical determined from <br /> ods) resulted production chloramphenicol-                purified B. subtilis protein differs B. globigii <br /> resistant hbs-partial diploids. Southern hybridization exper-           protein (HBgl) amino acid position 40. An aspartic <br /> iments (not shown) using digested chromosomal DNA                    acid residue HBsu replaced lysine HBgl (49). This <br /> such transformants indicated hbs: :cat construct           relationship reflects immunological cross-reactivity be- <br /> integrated Campbell-like single-crossover recombina-               tween anti-HBgl antibodies HBsu protein. In con- <br /> tion event.                                                             trast, E. coli HU proteins (HU2 HU1) a <br />    Our failure interrupt hbs chromosome B.                 lower degree homology (52 57% identity) HBsu <br /> subtilis suggests essential role gene product. To            protein cross-react anti-HBgl antibodies (21). <br /> test possibility placed hbs gene             Among HB proteins different Bacillus species far <br /> control spac-J promoter (Fig. SA) tested cell            sequenced, conservation 80% (Fig. 8). How- <br /> viability relation IPTG induction. As shown Fig. 6,            ever, homology B. subtilis bacteriophage SPO1- <br /> normal cell growth strain BM19 depends highly                 encoded TF-1 protein HB proteins different Bacillus <br /> presence IPTG. In absence IPTG, cell growth                species 50% (13). TF-1 isolated infected B. <br /> strongly affected. This limited growth switched normal           subtilis cells shares 41% identical residues HBsu <br /> level addition 1 mM IPTG, derepresses                 (13). Most amino acid substitutions the <br /> expression hbs. It noted cells contain-          conserved a-<span id='am-34' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-38'  ></span>helix</span> region (Fig. 8). TF-1 binds preferentially <br /> ing Pspac-hbs fusion able grow absence            5-hydroxymethyl-uracil-containing DNA present in <br />  IPTG consequence leakiness control system.              SPOl DNA (23), divergence amino acid <br />    The integration pDH88hbs&apos; homologous recombi-                sequence Bacillus HB proteins have <br />  nation event confirmed Southern hybridization ex-               arisen selective pressure adapt novel func- <br />  periments. As shown Fig. SB, HindIII-EcoRI digest             tion. However, nucleotide sequence TF-1 <br />  wild-type DNA, hbs located single 2.3-kb frag-              gene (13) compared hbs, 57% identity was <br />  ment. In strain BM19, fragment split 6.5-kb             observed. This indicates higher level conservation the <br /> fragment carrying truncated gene hbs 1.8                   DNA level observed amino acid sequence <br />  kb-fragment containing Pspac-hbs fusion.                           level. This suggest common ancestor genes. <br />     HBsu expression E. coli. Anti-HBgl antibodies cross-             On hand, histonelike proteins lysine and <br />  reacted specifically single band 10 kDa           arginine rich devoid cysteine, tryptophan, and <br />  total B. subtilis protein extract (Fig. 7, lane 5). This band          tyrosine, generally. <br />  VOL. 173, 1991                                                                    DNA-BINDING PROTEIN OF B. SUBTILIS                     3197 <br />  <br />    The data obtained Southern hybridization                  2. Berger, S. L., G. M. Wahl. 1987. Screening colonies or <br /> sequence analysis suggest presence single copy                   plaques radioactive nucleic acid probes. Methods En- <br /> the hbs gene B. subtilis chromosome. This copy                     zymol. 152:415-423. <br /> located 5&apos; end mtrA gene maps 2040              3. Broyles, S. S., D. E. PettiJohn. 1986. Interaction the <br /> the standard genetic map B. subtilis. As determined                  <span id='am-89' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-90' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-95'  ></span>Escherichia</span> coli HU protein DNA: evidence formation <br />                                                                              nucleosome-like structures altered DNA helical pitch. J. <br /> the DNA sequence (Fig. 3), mtrA initiation codon                      Mol. Biol. 187:47-60. <br /> placed 175 bp downstream hbs stop codon. The                 4. DiJk, J., S. W. White, K. S. Wilson, K. Appelt. 1983. On the <br /> hbs gene located 900-bp EcoRI-NdeI chromosomal                       DNA binding protein II Bacillus stearothermophilus. I. <br /> fragment. In E. coli <span id='am-96' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-97' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-102' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span><span id='am-103'  ></span><span id='am-104' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span>Salmonella</span> typhimurium, genes                   Purification, studies solution, crystallization. J. Biol. <br /> were encode closely related proteins (HU2                       Chem. 258:4003-4006. <br /> HU1) form heterodimeric protein HU (17, 25, 26,                  5. Dixon, N. E., A. Kornberg. 1984. Protein HU the <br /> 30-32). The availability genes encoding HU E. coli                 enzymatic replication chromosomal origin Escherichia <br /> and S. typhimurium facilitated studies biological role             coli. Proc. Natl. Acad. Sci. USA 81:424-428. <br /> vivo. Recently, HU null mutants constructed                      6. Drlica, K., J. Rouvibre-Yaniv. 1987. Histonelike proteins of <br />                                                                              bacteria. Microbiol. Rev. 51:301-319. <br /> shown defective growth, cell division, lysogeny,             7. Durrenberger, M., M.-A. Bjornsti, T. Uetz, J. A. Hobot, E. <br /> transposition bacteriophage Mu (18, 19, 35, 47). This                     Kellenberger. 1988. Intracellular location histonelike <br /> pleiotropy suggests important role HU protein                  protein HU Escherichia coli. J. Bacteriol. 170:4757-4768. <br /> the physiology E. coli S. typhimurium.                             8. Ferrari, F. A., K. Trach, D. LeCoq, J. Spence, E. Ferrari, and <br />    In order understand vivo physiological role              J. A. Hoch. 1985. Characterization spoOA locus its <br /> HBsu protein growth development B. subtilis,                   deduced product. Proc. Natl. Acad. Sci. USA 82:2647-2651. <br /> we constructed vitro hbs::cat mutation. How-                  9. Flashner, Y., J. D. Gralla. 1988. DNA dynamic flexibility <br /> ever, unable replace hbs chromosomal copy                     protein recognition: differential stimulation bacterial <br /> by hbs::cat construct double-crossover recombina-                   histone-like protein HU. Cell 54:713-721. <br />                                                                          10. Funnell, B. E., T. A. Baker, A. Kornberg. 1987. In vitro <br /> tion event. Although transformation efficiency                  assembly prepriming complex region Escherichia <br /> low, chloramphenicol-resistant transformants                    coli chromosome. J. Biol. Chem. 262:10327-10334. <br /> analyzed Southern hybridization screen gene                11. Geider, K., H. Hoffmann-Berling. 1981. Proteins controlling <br /> interruption. All transformants tested                      helical structure DNA. Annu. Rev. Biochem. 50:233-260. <br /> partial diploids; contain intact interrupted            12. Gollnick, P., S. Ishino, M. I. Kuroda, D. J. Henner, C. <br /> copy hbs, arisen Campbell-like                        Yanofsky. 1990. The mtr locus two-gene operon required for <br /> integration mechanism.                                                       transcription attenuation trp operon Bacillus subtilis. <br />    Our failure interrupt hbs B. subtilis chromo-               Proc. Natl. Acad. Sci. USA 87:8726-8730. <br />                                                                          13. Greene, J. R., S. M. Brennan, D. J. Andrew, C. C. Thompson, <br /> some homologous recombination using hbs::cat con-                     S. H. Richards, R. L. Heinrikson, E. P. Geiduschek. 1984. <br /> struct prompted use strategy control hbs                    Sequence bacteriophage SPOl gene coding transcrip- <br /> expression vivo. We constructed strain bearing                   tion factor 1, viral homologue bacterial type Il DNA <br /> truncated hbs copy control promoter                     binding proteins. Proc. Natl. Acad. Sci. USA 81:7031-7035. <br /> and second complete copy spac promoter                       14. Groch, N., R. Quaas, U. Hahn, U. Heinemann. 1988. <br /> control (Fig. 5A). In strain, BM19, normal growth                   Chemical synthesis cloning gene coding Bacillus <br /> found depend IPTG induction, indicating essen-                    subtilis HBsu protein. Nucleosides Nucleotides 7:817-820. <br /> tial role intact HBsu protein cell growth. Cells              15. Gryczan, T. J., D. Dubnau. 1978. Construction prop- <br /> containing Pspac-hbs fusion grew threefold                     erties chimeric plasmids Bacillus subtilis. Proc. Natl. <br /> slowly cells presence 1 mM IPTG.                     Acad. Sci. USA 75:1428-1432. <br />                                                                          16. Henner, D. J. 1990. Inducible expression regulatory genes in <br /> The reduced growth absence IPTG                      Bacillus subtilis. Methods Enzymol. 185:223-228. <br /> the leakiness inducible spac promoter. Similar results            17. Higgins, N. P., D. Hillyard. 1988. Primary structure and <br /> were obtained spoOA gene fused                      mapping hupA gene Salmonella typhimurium. J. <br /> spac promoter vector (8).                                      Bacteriol. 170:5751-5758. <br />    Is hbs essential gene viability B. subtilis? The            18. Hillyard, D. R., M. Edlund, K. T. Hughes, M. Marsh, N. P. <br /> results mentioned above, including (i) failure interrupt              Higgins. 1990. Subunit-specific phenotype Salmonella typhi- <br /> hbs homologous recombination, (ii) IPTG-dependent                     murium HU mutant. J. Bacteriol. 172:5402-5407. <br /> growth strain BM19 containing hbs gene spac                 19. Huisman, O., M. Faelen, D. Girard, A. Jaffe, A. Toussaint, and <br /> promoter control, (iii) fact gene expressed              J. Rouvi*re-Yaniv. 1989. Multiple defects Escherichia coli <br /> from distinct promoters growth, suggest                   mutants lacking HU protein. J. Bacteriol. 171:3704-3712. <br />                                                                          20. Igo, M. M., R. Losick. 1986. Regulation promotor that <br /> an important essential physiological role HBsu                 utilized minor forms RNA polymerase holoenzyme B. <br /> protein B. subtilis. However, final proof                  subtilis. J. Mol. Biol. 191:615-624. <br /> HBsu essential growth development awaits addi-                21. <span id='am-112' about='obo:NCBITaxon_1088460' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_469321 obo:NCBITaxon_37569 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_523180 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_7128 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-113' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-115' property="rdfs:label" content="Imber" datatype="xsd:string"></span><span id='am-116'  ></span>Imber</span>, R., H. Bachinger, T. A. Bickle. 1982. Purification and <br /> tional detailed studies, way.                                characterization small DNA-binding protein, HB, from <br />                                                                              Bacillus globigii. Eur. J. Biochem. 122:627-632. <br />                       ACKNOWLEDGMENTS                                    22. Imber, R., M. Kimura, N. Groch, U. Heinemann. 1987. <br />   We thank L. Diederich S. Borchert assistance PCR            DNA-binding properties primary structure HB protein <br /> library construction, respectively. We thank G. Willimsky                Bacillus globigii. Eur. J. Biochem. 165:547-552. <br /> discussion J. Bittner excellent technical assistance. Also,   23. Johnson, G. G., E. P. Geiduschek. 1977. Specificity the <br /> thank D. J. Henner providing pDH88 sharing sequence              weak binding phage SPOl transcription-inhibitory <br /> data mtr operon.                                                      protein, TF-1 SPOl DNA. Biochemistry 16:1473-1495. <br />                                                                          24. Johnson, W. C., C. P. Moran, R. Losick. 1983. Two RNA <br />                            REFERENCES                                         polymerase sigma factors Bacillus subtilis discriminate <br />  1. Beck, A., J. DiJk, R. Reinhardt. 1987. Ribosomal proteins            overlapping promoters developmentally regulated <br />     DNA-binding protein II extreme thermophile                  gene. Nature (London) 302:800-804. <br />     Bacillus caldolyticus. Biol. Chem. Hoppe-Seyler 368:121-130.         25. Kano, Y., K. Osato, M. Wada, F. Imamoto. 1987. Cloning <br />  3198       MICKA ET AL.                                                                                                              J. BACTERIOL. <br />  <br />       sequencing HU-2 gene Escherichia coli. Mol. Gen.           structure circular double-stranded DNA. Cell 17:265-274. <br />       Genet. 209:408-410.                                                  39. Salti, V., F. Le Hegarat, L. Hirschbein. 1985. Isolation and <br /> 26.   Kano, Y., S. Yoshino, M. Wada, K. Yokoyama, M. Nobuhara,                 characterization small heat-stable acid-soluble DNA-binding <br />       F. Imamoto. 1985. Molecular cloning nucleotide se-               proteins Bacillus subtilis nucleoids. J. Gen. Microbiol. <br />       quence HU-1 gene Escherichia coli. Mol. Gen. Genet.            131:581-590. <br />       201:360-362.                                                         40. Sambrook, J., E. F. Fritsch, T. Maniatis. 1989. Molecular <br /> 27.   Kimura, M., K. S. Wilson. 1983. On DNA binding                   cloning: laboratory manual. Cold Spring Harbor Laboratory, <br />       protein II Bacillus stearothermophilus. lI. The amino acid          Cold Spring Harbor, N.Y. <br />       sequence relation homologous proteins       41. Schmidt, M. B. 1988. Structure function bacterial <br />       procaryotes. J. Biol. Chem. 258:4007-4011.                         chromosome. Trends Biochem. Sci. 13:131-135. <br /> 28.   Krause, M., M. A. Marahiel, H. von Dohren, H.  <br /> </body></html>