<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#211; 2009 Genetics Society America <br /> DOI: 10.1534/genetics.109.103432 <br />  <br />  <br />  <br />   Snf1 Controls Activity Adr1 Through Dephosphorylation Ser230 <br />  <br />                    Sooraj Ratnakumar, Nataly Kacherovsky, Erin Arms Elton T. Young1 <br />                              Department Biochemistry, University Washington, Seattle, Washington 98195-7350 <br />                                                        Manuscript received March 30, 2009 <br />                                                       Accepted publication April 20, 2009 <br />  <br />  <br />                                                                   ABSTRACT <br />                  The transcription factors Adr1 Cat8 act concert regulate expression numerous yeast genes <br />               diauxic shift. Their activities regulated Snf1, yeast homolog AMP-activated protein <br />               kinase higher eukaryotes. Cat8 regulated directly Snf1, Snf1 regulates Adr1 unknown. <br />               Mutations Adr1 alleviate glucose repression clustered amino acids 227 239. <span id='am-50' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> <br />               region contains consensus sequence protein kinase A, RRAS230F, Ser230 phosphorylated vitro <br />               protein kinase A Ca11 calmodulin-dependent protein kinase. Using antiphosphopeptide <br />               antibody, level Adr1 phosphorylated Ser230 highest glucose-grown cells and <br />               decreased Snf1-dependent manner glucose depleted. A nonphosphorylatable Ser230Ala <br />               mutant longer Snf1 dependent activation Adr1-dependent genes suppress Cat8 <br />               dependence genes coregulated Adr1 Cat8. Contrary expectation, protein kinase A (PKA) <br />               Ca11 calmodulin-dependent protein kinase appeared important role Ser230 phosphorylation <br />               vivo, screen 102 viable kinase deletion strains failed identify candidate kinase. We conclude <br />               Ser230 phosphorylated multiple protein kinases kinase encoded essential gene. <br />               Using Ser230Ala mutant, explain long-standing observation synergy Adr1 constitutive <br />               mutants Snf1 activation conclude dephosphorylation Ser230 Snf1-dependent pathway <br />               appears major component Adr1 regulation. <br />  <br />  <br />  <br />  <br /> T    RANSCRIPTION factors activate gene expression by <br />       recruiting coactivators promoter create a <br /> pre-initiation complex (PIC) containing RNA polII and <br />                                                                            tase Glc7, regulation event appears be <br />                                                                            targeting factor Reg1 (Sanz et al. 2000; <br />                                                                            Rubenstein et al. 2008). As expected, removing the <br /> general factors (Featherstone 2002; Fry Peterson                       negative regulatory mechanism deleting REG1 causes <br /> 2002). Signal transduction pathways regulate                         constitutive activation Snf1, measured expres- <br /> this process acetylation phosphorylation tran-                    sion glucose Snf1-dependent genes are <br /> scription factors influence ability recruit                    normally glucose repressed (Dombek et al. 1993; Tu <br /> coactivators (Kwok et al. 1994; Papoutsopoulou                         Carlson 1995; McCartney Schmidt 2001; <br /> Janknecht 2000; Zhong et al. 2002; Goel                                Orlova et al. 2008). Control Snf1 include <br /> Janknecht 2003). Adr1 yeast transcription factor                      additional mechanisms simple phosphorylation <br /> that acts concert Cat8 Oaf1/Pip2 activate                   Glc7 Reg1-dependent dephosphorylation, however, <br /> numerous genes expressed glucose                            reg1D strain, additional phosphorylation of <br /> depletion, allowing cells use nonfermentable <span id='am-89' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-90' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-93'  ></span>car</span>-                       Snf1 activation Snf1-dependent <br /> bon sources ethanol, glycerol, fatty acids                     genes achieved glucose depletion (Dombek et al. <br /> (Schuller 2003; Young et al. 2003; Santangelo 2006).                       1993; Tachibana et al. 2007; Orlova et al. 2008). <br /> The signal transduction pathway activates Adr1                    Interaction regulatory proteins changes in <br /> Cat8 mediated Snf1 (Rahner et al. 1999; Schuller                     protein structure suggested additional <br /> 2003; Charbon et al. 2004; Santangelo 2006), yeast                     levels Snf1 control (Rubenstein et al. 2008). <br /> homolog AMP-activated protein kinase widely                        Snf1 necessary activity Adr1 Cat8, <br /> regarded &#8216;&#8216;energy sensor cell&#8217;&#8217; (Hardie et al.               enhancing transcription CAT8 inactivation <br /> 1998). Snf1 activated phosphorylation Thr210                   repressor Mig1, activating Cat8 directly <br /> three upstream kinases (Hong et al. 2003). Snf1-Thr210                     phosphoryation (Hedges et al. 1995; Rahner et al. <br /> is dephosphorylated PP1-type protein phospha-                       1996, 1999; Charbon et al. 2004; reviewed Schuller <br />                                                                            2003). Snf1 required Adr1 binding (Young et al. <br />                                                                            2002) recruitment coactivators Adr1-dependent <br />   Supporting information available online http://www.genetics.org/   promoters (Biddick et al. 2008). Promoter binding by <br /> cgi/content/full/genetics.109.103432/DC1. <br />   1 <br />                                                                            Adr1 regulated phosphorylation its <br />    Corresponding author: Department Biochemistry, University of <br /> Washington, 1705 NE Pacific St., Seattle, WA 98195-7350.                   DNA-binding domain (Kacherovsky et al. 2008) and <br /> E-mail: ety@u.washington.edu                                               acetylation promoter nucleosomes (Verdone et al. <br />  <br /> Genetics 182: 735&#8211;745 ( July 2009) <br />  736                                               S. Ratnakumar et al. <br />  <br /> 2002; Tachibanaet al. 2007), known       determinant ADR1c phenotype (Denis <br /> Snf1 activates Adr1 directly indirectly      et al. 1992; Dombek Young 1997). In addition, <br /> role Adr1-dependent promoters, nucleosome           Adr1c mutations appear refute PKA <br /> modification (Lo et al. 2001).                                 candidate Ser230 kinase vivo. The <br />    Selection mutants allowed ADH2 expression        Ala229Pro Adr1c mutant shows greatly increased affin- <br /> escape glucose repression identified rare semidominant         ity PKA binding peptide phosphorylation assay, <br /> ADR1c alleles (ADR1-constitutive) (Ciriacy 1979; Denis         predict greater phosphorylation and <br /> et al. 1991). ADR1c alleles allow ADH2 expression           lower Adr1 activity. However, Ala229Pro mutant is <br /> occur presence glucose enhance dere-             constitutive Adr1c mutants <br /> pressed transcription. Surprisingly, level Adr1c        (Denis et al. 1992). Therefore, known if <br /> protein lower level wild-type Adr1         phosphorylation Ser230 affects activity Adr1 <br /> (Taylor Young 1990; Dombek Young 1997),                kinase responsible phosphorylation <br /> suggesting high transcriptional potency      vivo (Denis et al. 1992; Cook et al. 1994; Dombek and <br /> associated rapid turnover observed            Young 1997). <br /> for activators (Tansey 2001). ADR1c shows              How Adr1c achieves high transcriptional potency is <br /> strong synergism deletion Snf1 regulatory          unknown (Denis et al. 1992; Cook et al. 1994; Dombek <br /> factor REG1. In combination Adr1c, partial activa-        Young 1997). ADR1c alleles map far DNA- <br /> tion Snf1 deleting REG1 allows higher levels        binding domain Adr1-Ser230A does en- <br /> of ADH2-repressed expression observed            hanced vitro DNA-binding activity (Taylor and <br /> either mutation acting (Dombek et al. 1993). Adr1c       Young 1990). After characterizing series mutants <br /> and reg1D synergistic activating poised      amino acids 227&#8211;239, internal deletions in <br /> inactive PIC (Tachibana et al. 2007). Although            region Adr1, Cook et al. (1994) suggested this <br /> suggests Adr1c activation Snf1 reg1D act        region binds repressive factor cannot <br /> through different pathways cause constitutive expres-       interact mutants interacts blocks <br /> sion Snf1-dependent genes, precise mechanism         activation domain. Some evidence suggested <br /> the synergism unknown.                                      Adr1c mutants suppress requirement for <br />    Cloning molecular analysis ADR1c alleles             Snf1 (Ciriacy 1979; Denis 1984), imply <br /> showed mutations amino acids 227 239,            affected Snf1, contradictory to <br /> region contains consensus sequence cyclic         enhanced effect ADH2 expression seen when <br /> AMP-dependent PKA, Arg-Arg-<span id='am-78' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-79' property="rdfs:label" content="Ala" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81'  ></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Ala</span>-Ser-Phe (Denis              Adr1c combined Snf1-activating reg1D muta- <br /> Gallo 1986; Denis et al. 1992). No mutations          tion. Thus, questions Snf1 affects activity <br /> the Adr1 ORF causing constitutive ADH2 expression              Adr1 interacts Adr1c allele are <br /> were isolated, suggesting 227&#8211;239 region plays        unresolved. <br /> a unique role regulating Adr1 activity. The ADR1c              We used antibody Ser230-phosphorylated <br /> mutations suggest Adr1 post-translationally      peptide Adr1 amino acids 217&#8211;234 investigate the <br /> inhibited phosphorylation Ser230 PKA              conditions Ser230 phosphorylated <br /> consensus sequence (Denis Gallo 1986; Cherry               (Adr1-pSer230) effect Snf1 modifica- <br /> et al. 1989). Adr1 appears phosphorylated Ser230      tion. The antibody used screen specific Adr1- <br /> in vivo subtle alterations SDS&#8211;PAGE             Ser230 kinase. Phosphorylation increased by <br /> electrophoretic mobility partial sensitivity <span id='am-104' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-105' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-106'  ></span><span id='am-107' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-108' property="rdfs:label" content="Phos" datatype="xsd:string"></span>phos</span>-      enhanced PKA activity, decreased loss PKA or <br /> phatase treatment observed (Vallari et al. 1992;           CMK. These <span id='am-94' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-95' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-97'  ></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> suggest Ser230 be <br /> Dombek Young 1997). Recent studies yeast            redundantly phosphorylated vivo multiple kinases <br /> phosphorylome suggested Adr1 phosphorylation,             phosphorylated essential kinase. Using the <br /> although reported site Ser232, Ser230 (Chi         nonphosphorylatable Adr1c, probed relationship <br /> et al. 2007).                                                  Adr1-pSer230, Snf1, coregulator Cat8, <br />    In support phosphorylation Adr1 PKA,          results supporting earlier model the <br /> PKA calmodulin-dependent protein kinase (CMK)              function Ser230-containing domain. <br /> efficiently phosphorylate Ser230 vitro (Denis et al. <br /> 1991; Hook et al. 1999). Hyperactivation PKA by <br /> deletion BCY1-encoded regulatory subunit causes                       MATERIALS AND METHODS <br /> loss ADH2 expression (Cherry et al. 1989), suggest-           Yeast strains, <span id='am-99' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-100' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span>, culture conditions, enzyme <br /> ing unregulated PKA activity inhibits Adr1             assays: The strains used study listed Table 1. <br /> constitutively phosphorylating Ser230. However, replac-        Multiple deletion mutants TPK CMK genes were <br /> ing Ser230 phosphomimetic Asp gave               constructed standard genetic techniques. Epitope tag- <br />                                                                ging, gene deletion, marker swapping according to <br /> constitutively active phenotype alterations        Guldener et al. (1996), Cross (1997), Knop et al. (1999), <br /> the region, suggesting Asp does mimic          respectively. Deletions epitope tagging confirmed by <br /> phosphorylated Ser phosphorylation Ser230         PCR and, marker swapping, Western blots. Yeast strains <br />                                                 Adr1c Alleles Suppress Snf1 Deficiency                                               737 <br />  <br />                                                                TABLE 1 <br />                                                           S. cerevisiae strains <br />  <br /> Strain                                                  Genotype                                                    Source <br /> TYY201                MATa ade2 can1-100 his3-11,15 leu2-13,112 trp1-1 ura3-1                          Yeast stock center <br />   (<span id='am-83' about='obo:NCBITaxon_178515' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_6042 obo:NCBITaxon_1 obo:NCBITaxon_6040 obo:NCBITaxon_131567 obo:NCBITaxon_178475 obo:NCBITaxon_2759 obo:NCBITaxon_6049'><span id='am-84' property="rdfs:label" content="Aka" datatype="xsd:string"></span><span id='am-85'  ></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="Aka (de Laubenfels, 1949)" datatype="xsd:string"></span>aka</span> W303-1A) <br /> VBY20                 MATa adh3 ura3 his3 leu2T(pRS315-ADR1)X3 ADH2TYIp                                Voronkova et al. (2006) <br />                       ADH2/lacZ(TRP1)ADH2T YIpADH2/GFP (URA3) <br /> NKY85                 VBY20 ADR1:HATkanMX                                                              This study <br /> NKY87                 NK85 reg1DTnatMX                                                                 This study <br /> NKY91                 NK85 snf1DTnatMX                                                                 This study <br /> TYY204                W303-1A adr1D1TLEU2 ADH2TYIpADH2/lacZTTRP1                                       Young et al. (2003) <br /> TYY497                W303-1A adr1D1TLEU2 ADH2TYIpADH2/lacZTtrp1THIS3                                  Young et al. (2008) <br /> TYY498                W303-1A snf1DTURA3 adr1D1TLEU2 ADH2TYIpADH2/lacZTtrp1THIS3                       Young et al. (2008) <br /> CKY13                 W303-1A adr1DTkanMX                                                              This study <br /> CKY26                 W303-1A adr1DTnatMX snf1DTkanMX                                                  This study <br /> CKY5                  W303-1A adr1DTkanMX reg1DTnatMX                                                  This study <br /> CKY23                 W303-1A adr1DTnatMX cat8DTkanMX                                                  This study <br /> SRY60                 W303-1A reg1DTkanMX cat8DTnatMX adr1DTHYG                                        This study <br /> NKY108                NKY85 cmk1DThphmx cmk2DTnatMX cmk3DTnatMX                                        This study <br /> SGP406                MATa his3 leu2-3,112 tpk1TURA3 tpk2THIS3                                         Garrett and <br />                         tpk3TTRP1 trp1 ura3-52 yak1TLEU2                                                 Broach (1989) <br /> NKY111                NKY85 bcy1DTnatMX                                                                This study <br /> CMY323                MATa ade2-101&#176; can1 his3D200 lys2-801 trp1D1 ura3-52                             Carl Mann <br /> CMY328                MATa ade2-101&#176;adr1D2TURA3 can1 his3D200 lys2-801atrp1D1 ura3-52                  Dombek Young (1997) <br /> CMY468                MATa ade2-101&#176; his3D leu2D1 lys2-801 tpk1TURA3                                   Carl Mann <br />                         tpk2THIS3tpk3ts-1 trp1D1 ura3-52 <br /> SGP400                MATa his3 leu2-3,112 trp1 ura3-52 yak1TLEU2                                      Garrett Broach (1989) <br />  <br />  <br />  <br /> were grown complete synthetic media described                 Immunoblotting: Whole-cell extracts analyzed 3&#8211; <br /> Sherman (1991). Repressing medium contained 5% glucose;                8% 6% polyacrylamide gels (NuPage system, Invitrogen) <br /> derepressing medium contained 0.05% glucose with-              transferred PVDF membranes electrophoresis. <br /> out 3% glycerol 2% ethanol. The plasmids used                  Western blots probed a-pSer230 followed rabbit- <br /> described previously (Yu et al. 1989; Dombek Young                 specific Licor secondary Ab (l680). A second identical blot <br /> 1997) modified cases introducing epitope             probed monoclonal a-HA antibody mouse- <br /> tag C terminus Adr1 described below. For gene             specific Licor secondary antibody (l800). Probing single blot <br /> expression studies, CEN-TRP1 plasmids expressing wild-type             antibodies resulted bleed-through more <br /> ADR1 (pKD16) S230A (pKD14), R228K (pKD27), D3                abundant HA-epitope-derived signal blots were <br /> (a deletion removes Adr1 amino acids 226&#8211;233; pKD26)              single blot probed sequentially&#8212;first, a- <br /> alleles ADR1 native promoter used. To facilitate      pSer230 appropriate Licor secondary antibody and, <br /> detection Adr1-S230 phosphorylation Western blotting,            second, a-HA antibody appropriate Licor <br /> 2m plasmids expressing wild-type ADR1 (pKD17-HATkanMX                  secondary antibody. Cell extracts prepared described <br /> (TRP1) ADR1-S230A allele (pKD20-HATkanMX                       previously (Dombek Young 1997) boiling pro- <br /> (TRP1) ADH1 promoter tagged HA                    cedure addition agitation glass beads 30 sec <br /> epitope employed. Alternatively, strains addi-          boiling step (Invitrogen) (Hahn 2008). An Odyssey <br /> tional copies ADR1 integrated leu2 locus used           Infrared imaging (Licor Biosciences) used for <br /> (Sloan et al. 1999). ADR1 epitope tagged HATkanMX             quantitation. a-HA (Y-11, Santa Cruz Biochemicals), a-Adr1 <br /> in strain. The Adr1-dependent reporter UAS1-lacZ            (Dombek et al. 1993), antibodies Adr1-derived <br /> plasmid, pHDY10, containing 10 Adr1-binding sites (Yu et al.           synthetic peptide used 1:1000 dilution. The peptides, <br /> 1989). b-Galactosidase assays performed described           representing Adr1 amino acids 217&#8211;234, VKRKYLKKLTRRA <br /> Guarente (1983). ADH enzyme activity analyzed                   (pS)FSAQ, nonphosphorylated version synthe- <br /> separating proteins nondenaturing polyacrylamide gels               sized used generate affinity purify a-pSer230 <br /> and visualized in-gel chromogenic staining described             antibody Bethyl Laboratories (Montgomery, TX). Second- <br /> (Dombek Young 1997).                                               ary IR-dye-conjugated antibodies used 1:1000&#8211;1:3000 were <br />    Real-time quantitative PCR: For expression analysis, RNA            goat a-mouse <span id='am-1' about='obo:NCBITaxon_149625' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_163739 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Alexa" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5'  ></span>Alexa</span> 680 (Molecular Probes) IRdye800- <br /> was isolated hot phenol extraction (Collart Oliveiro            conjugated a-rabbit IgG (Rockland Immunochemicals). <br /> 1993) converted cDNA SuperScript III kit <br /> (Invitrogen) according manufacturer&#8217;s directions. <br /> cDNA quantified real-time quantitative PCR (RT&#8211;                                           RESULTS <br /> qPCR) (Tachibana et al. 2007) MJ Research Chromo4 <br /> system, using ABI Quantace SYBR Master Mix, according              Adr1-Ser230 phosphorylation vivo glucose <br /> the manufacturer&#8217;s instructions. Primer sequences avail-           dependent dephosphorylation Snf1 dependent: <br /> able request.                                                       Antibodies phosphorylated peptide represent- <br />  738                                                       S. Ratnakumar et al. <br />  <br />                                                                        ADH2 regulation appears unperturbed modest <br />                                                                        increase Adr1 levels (Sloan et al. 1999; Voronkova <br />                                                                        et al. 2006). <br />                                                                           Western blotting HA Adr1-pSer230 <br />                                                                        revealed pSer230 relative total <br />                                                                        Adr1 highest glucose-grown cells decreased <br />                                                                         10-fold cells grown low glucose plus <br />                                                                        ethanol (Figure 2A, wt lanes). The decrease pSer230 in <br />                                                                        absence glucose elevated Adr1 <br />                                                                        level multicopy strain decrease also <br />                                                                        observed strain single copy ADR1 (Figure <br />                                                                        2B). Although low level Adr1-pSer230 single- <br />    Figure 1.&#8212;Antiphosphopeptide antibody recognizes Adr1 <br /> phosphorylated Ser230. (A) a-pSer230 recognizes Adr1 in <br />                                                                        copy strain precluded quantification change in <br /> whole-cell extracts. Wild type (wt) S230A refer TYY497          phosphorylation, Adr1 increased 10-fold after <br /> (adr1D1) pKD17-HA pKD20-HA, expressing HA-                    22 hr growth absence glucose, level of <br /> epitope-tagged versions wild-type ADR1 ADR1-Ser230, re-          pSer230 decreased undetectable level 2 hr. <br /> spectively; &#8216;&#8216;D&#8217;&#8217; strain TYY497 vector lacking ADR1.        A strain lacking SNF1 unable activate ADH2 <br /> The protein loaded gel shown each <br /> lane. The cells grown trp synthetic medium repres- <br />                                                                        expression presumably inactive Adr1. Consistent <br /> sing conditions (5% glucose). (B) Peptide competition.                 hypothesis phosphorylation Ser230 <br /> Ten ng/ml nonphosphorylated Ser230-phosphorylated                inactivates Adr1, snf1D strain, Adr1- <br /> peptide (materials methods) representing Adr1 amino                pSer230 relative total Adr1 appeared elevated in <br /> acids 217&#8211;234 present Western blotting             presence glucose did decrease signifi- <br /> primary antibody. Cell extracts strain <br /> TYY497 carrying pKD17-HA (wt multicopy ADR1) grown re- <br />                                                                        cantly removal (Figure 2A, snf1D lanes). Since <br /> pressing conditions trp synthetic medium. The             Snf1 kinase, regulation phosphorylated <br /> protein loaded gel shown lane. A low         Ser230 indirect, example, control <br /> amount cell extract (nanograms) used allow effective         kinase phosphatase. Surprisingly, al- <br /> competition peptide.                                          dephosphorylation Ser230 appeared be <br />                                                                        Snf1 dependent, deleting REG1 inactivating GLC7 <br />                                                                        temperature-sensitive mutation activate Snf1 in <br /> ing Adr1 amino acids 217&#8211;234 (VKRKYLKKLTRRA                            repressing conditions did affect relative level of <br /> (pS)FSAQ) generated assess level Adr1                   Adr1-pSer230 repressing derepressing con- <br /> phosphorylated Ser230 vivo (designated                           ditions (Figure 2A, reg1D; unpublished data Glc7- <br /> Adr1 pSer230). ELISA assays indicated anti-                   127). The experiment repeated multiple times with <br /> phosphopeptide antibody (a-pSer230) recognized                     similar results. Snf1 control Adr1-pSer230 apparently <br /> phosphorylated peptide 50003 better non-                      requires additional levels Snf1 regulation, as <br /> phosphorylated peptide (data shown). Figure 1A                     control access Snf1 T210 phosphorylation <br /> shows a-pSer230 recognized protein size                  site, work dephosphorylation Reg1 Glc7 to <br /> Adr1 strain expressing Adr1-HA strong                    mediate Snf1 activity (Orlova et al. 2008; Rubenstein <br /> ADH1 promoter multicopy plasmid,                       et al. 2008). Alternatively, Ser230 dephosphorylation in <br /> a strain lacking Adr1. Adr1 containing Ser230Ala                     glucose prevented mechanism is <br /> mutation (hereafter referred Adr1c unless                  independent Reg1 Glc7. In summary, inactivation of <br /> ADR1c alleles specified) recognized poorly a-               Snf1 prevented decrease Ser230 phosphorylation <br /> pSer230, demonstrating specificity antibody                 normally accompanies derepression activa- <br /> for Adr1 phosphorylated Ser230. Competition                  tion Adr1-dependent genes, consistent with <br /> the phosphorylated nonphosphorylated peptides                      phosphorylation playing negative role regulating <br /> demonstrated specificity a-pSer230 (Figure 1B).                 activity Adr1. <br /> These data confirmed a-pSer230 recognized Adr1-                      The stoichiometry phosphorylation Ser230 was <br /> pSer230 cross-reacted weakly, all, Adr1                 assessed immunoprecipitation Adr1 whole-cell <br /> lacking phosphate Ser230.                                           extracts derived multicopy ADR1 strain grown <br />    A strain containing multiple integrated copies                   repressing conditions. Adr1 immunoprecipitated <br /> ADR1 employed facilitate quantitation Adr1-                  a-Adr1 a-pSer230 antisera, precipi- <br /> pSer230 different growth conditions. At                   tated supernatant fractions analyzed SDS&#8211; <br /> copy Adr1 strain tagged HA                      PAGE. Western blotting using a-Adr1 antisera, which <br /> epitope provide measure total                    recognizes phosphorylated nonphosphory- <br /> Adr1 extract. Adr1 levels multicopy strain               lated Adr1, demonstrated a- a-pSer230 antisera <br /> grown repressing conditions similar Adr1 levels              immunoprecipitated 65% 45%, respectively, the <br /> in wild-type cells grown derepressing conditions,               total Adr1 (supporting information, Figure S1). De- <br />                                             Adr1c Alleles Suppress Snf1 Deficiency                                          739 <br />  <br />  <br />  <br />                                                                                 Figure 2.&#8212;Phosphorylation Adr1-pSer230 <br />                                                                              glucose Snf1 regulated. Preparation cell <br />                                                                              extracts, electrophoresis, Western blotting <br />                                                                              carried described materials <br />                                                                              methods. (A) Multicopy ADR1 strains. Wild <br />                                                                              type (wt) NKY85; snf1D NKY91, reg1D is <br />                                                                              NKY87; R, repressing growth conditions, 5% glu- <br />                                                                              cose; derepression (DR) 6 hr 0.05% <br />                                                                              glucose&#8211;2% ethanol. Cell extract (25 mg) an- <br />                                                                              alyzed. The signal intensity measured de- <br />                                                                              scribed materials methods several <br />                                                                              biological replicates. The values shown the <br />                                                                              means 6 standard deviation (SD). (B) A sin- <br />                                                                              gle copy ADR1 strain (TYY201) analyzed a <br />                                                                              similar manner using 100 mg protein extracted <br />                                                                              repressed cells (0 hr derepression) and <br />                                                                              cells derepressed 2, 4, 22 hr in <br />                                                                              0.05% glucose&#8211;2% ethanol. <br />  <br />  <br />  <br />  <br /> phosphorylation pSer230 immunoprecipita-              essential growth carbon sources and <br /> tion contribute lower level Adr1 immuno-          expression Cat8-dependent genes FBP1 were <br /> precipitated a-pSer230 antisera compared          unaffected Adr1c. In addition, Snf1- <br /> a-Adr1 antisera, fraction Adr1     dependent transcription factors Mig1 are <br /> not phosphorylated Ser230 repressing                 involved gene expression metabolic pathways. <br /> growth conditions. The high level Adr1 precipitated             The lack Snf1 control Adr1c alleles raised a <br /> by a-pSer230 antisera indicates majority        question earlier results showing synergism be- <br /> Adr1 phosphorylated vivo repressing conditions.        tween Adr1c reg1D (Dombek et al. 1993; Tachibana <br />    Adr1c activation Snf1 independent synergism           et al. 2007), Adr1c-containing strains show <br /> with reg1D Snf1 activation Cat8: Since           additional activation ADH2 Snf1 activated in <br /> decrease Adr1-Ser230 phosphorylation did occur           glucose reg1D. Adr1c Snf1 independent, it <br /> in snf1D strain, hypothesized that, decrease       influenced reg1D. A clue mech- <br /> were important activity Adr1, Adr1c,       anism synergism came high Snf1 de- <br /> the phosphorylatable serine mutated alanine,           pendence seen Cat8-dependent FBP1 gene <br /> be insensitive snf1D. As predicted, ADH2/lacZ           (Figure 3, A B) lack synergism seen at <br /> reporter assays in-gel assays endogenous ADHII           Cat8-independent genes POX1 (data shown). <br /> activity showed Adr1c highly efficient activat-     We hypothesized additional activation ADH2, <br /> ing ADH2, absence Snf1. ADH2 de-                 seen REG1 deleted Adr1c-containing <br /> repression Adr1c reduced fourfold            strain, Snf1 activation Cat8. If hypothesis is <br /> snf1 mutant, compared complete lack             correct, reducing Cat8 effect reduce syn- <br /> expression wild-type Adr1 (Figure S2). Snf1 in-            ergism. We measured ADH2 activation independently of <br /> dependence Adr1c seen mRNA                activators using reporter lacked binding <br /> ADH2 measured directly RT&#8211;qPCR. With                sites Cat8 Adr1-binding site (UAS1) <br /> assay, Adr1-dependent genes, in-            ADH2 promoter (Yuet al. 1989). Snf1 dependence <br /> cluding POX1 SPS19, activated Snf1-          synergism reg1D determined strains <br /> independent manner Adr1c. Adr1c did change           carrying reporter wild-type Adr1 Adr1c. <br /> expression FBP1, Adr1 dependent                 The activity reporter REG1 strain wild- <br /> (Figure 3A). Other Adr1c variants Ser230 region          type Adr1 low repressing conditions and <br /> were used Snf1 independence              increased 260-fold derepressing conditions (Table 2). <br /> general effect, alleles, Adr1-D3              The Adr1c allele showed expected constitutive <br /> [D(226-223)] Adr1-R228K, activated Adr1-           activity hyperactivation. In reg1D mutant, wild- <br /> dependent genes Snf1. The CAT8-dependent                type Adr1 showed constitutive b-galactosidase activity to <br /> FBP1 gene derepressed snf1D strains Adr1c          10% 8-hr derepressed value REG1 strain. <br /> alleles (Figure 3B). ADR1c overcome Snf1          The degree constitutive activity seen when <br /> requirement growth nonfermentable carbon             Adr1c activator reg1D strain, instead the <br /> sources ethanol, glycerol, lactate, acetate,         synergistic effects seen previously assays <span id='am-60' about='obo:NCBITaxon_1178312' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_572719 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41011 obo:NCBITaxon_41197 obo:NCBITaxon_41012 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-61' property="rdfs:label" content="Mea" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63'  ></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>mea</span>- <br /> (data shown). This surprising Cat8           sured expression entire ADH2 promoter <br />  740                                                S. Ratnakumar et al. <br />  <br />  <br />  <br />                                                                               Figure 3.&#8212;Snf1 dependence Adr1c- <br />                                                                            dependent gene expression. (A) Derepressed <br />                                                                            transcript levels SNF1 snf1D, wild-type <br />                                                                            (wt) ADR1 ADR1c (ADR1-S230A). Analysis of <br />                                                                            mRNA isolated cells derepressed 0.05% <br />                                                                            glucose&#8211;3% glycerol 4 hr. RT&#8211;qPCR per- <br />                                                                            formed analyzed described materials <br />                                                                            methods. Strains TYY497 (adr1DSNF1) <br />                                                                            TYY498 (adr1Dsnf1D) carrying wild- <br />                                                                            type ADR1 (pKD16) ADR1-S230A (pKD14). <br />                                                                            The mRNA values normalized ACT1 <br />                                                                            mRNA expressed percentage of <br />                                                                            mRNA wild-type SNF1 strain wild-type <br />                                                                            Adr1 activator. The values represent the <br />                                                                            mean 6 standard deviation biological <br />                                                                            replicates assayed duplicate. (B) Relative tran- <br />                                                                            script levels snf1D various Adr1c alleles. <br />                                                                            Transcript levels quantified displayed <br />                                                                            A. The strains CKY13 (adr1D) and <br />                                                                            CKY26 (adr1D snf1D). The plasmids in <br />                                                                            section A additionally pKD26 [Adr1-D(226- <br />                                                                            233)] pKD27 (Adr1-R228K). <br />  <br />  <br />  <br /> (Dombek et al. 1993; Tachibanaet al. 2007). In reg1D        does. Expression Adr1- Cat8-dependent genes <br /> strain, wild-type Adr1c alleles showed           measured RT&#8211;qPCR RNA isogenic adr1D <br /> slight increase constitutive activity REG1          CAT8 adr1D cat8D strains wild-type Adr1 <br /> strain, possibly 2- 3-fold elevated ADR1          Adr1c plasmids grown repressing and <br /> transcription observed reg1D mutant (Dombek        derepressing conditions. <br /> et al. 1993).                                                      The mRNA levels Adr1- Cat8-dependent <br />    We additionally tested reliance Adr1c reg1D       genes derepressing conditions shown Figure <br /> synergism Cat8 deleting CAT8 adr1D reg1D            4. Table S1 data repressed de- <br /> mutant strain introducing wild-type Adr1              repressed conditions Adr1- Cat8-dependent <br /> Adr1c plasmids. ADH2 expression measured              genes, expressed manner allows com- <br /> isolating mRNA performing RT&#8211;qPCR. The results              parison mRNA levels genes. The depen- <br /> indicated Cat8 required high, glucose-         dence Cat8 markedly reduced ADH2, ACS1, <br /> resistant ADH2 expression reg1D strain              ADY2, JEN1 Adr1c activator compared <br /> either wild-type Adr1 Adr1c activator (Table         strong Cat8 dependence expression these <br /> 3). Efficient derepression Adr1c abolished       genes wild-type Adr1 activator. These <br /> the double cat8Dreg1D mutant.                                   results confirmed ADH2 reporter assays <br />    Adr1c partially suppresses Cat8 dependence genes          (Figure S3A) in-gel staining ADH activity cell <br /> activated Adr1- Cat8: In addition                extracts (Figure S3B). Genes dependent mainly Cat8, <br /> showing Cat8 required constitutivity           FBP1 ICL1, unaffected Adr1c (Figure <br /> conferred reg1D, results cat8D strains          4). In summary, Adr1c efficiently suppressed Cat8 <br /> indicated ADH2 expression mediated Adr1c             requirement numerous genes derepression is <br /> the presence Reg1 largely independent Cat8            normally codependent Adr1 Cat8. These results <br /> (Table 3: compare Adr1c Cat8 wild type Cat8D         consistent previous section that <br /> rows). This contrast strong Cat8 depen-           documented partial Snf1 independence Adr1c. <br /> dence wild-type Adr1 activator. Thus, Adr1c           Protein synthesis required regulate Adr1- <br /> acts independently of, rescues, nonactivat-        pSer230 levels: The importance pSer230 the <br /> ing ADH2 defect cat8D mutation. In previous study,       Ser230 domain demonstrated activity of <br /> we that, promoters regulated              Adr1c, low glucose coregu- <br /> Adr1 Cat8 ADH2, overexpression Adr1              lator Cat8. To address mechanism Ser230 <br /> partially compensates loss Cat8 (Biddick et al.      phosphorylation levels regulated, measured <br /> 2008), suggesting somewhat redundant functions              phosphorylation levels Adr1-Ser230 absence <br /> Adr1 Cat8 coregulated promoters. We tested               protein synthesis. The total Ser230- <br /> whether Adr1c single copy suppress re-             phosphorylated Adr1 determined Western blot- <br /> quirement Cat8 overexpressed wild-type Adr1              ting. In presence absence glucose the <br />                                                Adr1c Alleles Suppress Snf1 Deficiency                                           741 <br />  <br />                           TABLE 2                                                             TABLE 3 <br />    Expression ADR1-dependent reporter lacking                      Synergistic activation ADH2 Adr1c reg1D is <br />                     Cat8-binding sites                                                      Cat8 dependent <br />  <br />                                    b-Galactosidase activitya                   Genotypea              ADH2 mRNA level/ACT1b <br /> Genotypeb        Activatorc          Rd             DR (8 hr)      ADR1          CAT8       REG1            R             DR <br /> REG1           Adr1               1.2 (0.7)        260 (98)        Wild   type Wild type Wild type   0.0025(0.005)    2.3 (0.5) <br /> REG1           Adr1-S230A         21 (10)          450 (180)       Wild   type Wild type     D       0.54 (0.12)      1.3 (0.3) <br /> reg1D          Adr1               30 (3.8)           NMe           Wild   type     D     Wild type   0.0016 (0.001)   0.26 (0.02) <br /> reg1D          Adr1-S230A         36 (7.3)            NM           Wild   type     D         D       0.004            0.0025 <br />   R, repressing growth conditions. DR, derepressing growth         Adr1c       Wild type Wild type   0.14 (0.03)      11 (2.5) <br /> conditions.                                                        Adr1c       Wild type     D       1.4 (0.16)        5.6 (0.2) <br />   a <br />     UAS1 reporter activity (pHDY10). Data average val-        Adr1c           D         D       0.001 (0.0005)    0.13 (0.02) <br /> ues Miller units transformants stan-    Adr1c           D     Wild type   0.1 (0.02)        7.2 (0.38) <br /> dard deviations parentheses. <br />   b <br />     The strains REG1: CKY13; reg1D: CKY5;               R, repressing growth conditions. DR, derepressing growth <br /> adr1DTkanmx.                                                       conditions. <br />                                                                      a <br />   c <br />     Wild-type Adr1 carried pKD16; Adr1c pKD14              Strains used adr1D strains CKY13 (CAT8 REG1), CKY23 <br /> (S230A).                                                           (cat8D REG1), SRY60 (cat8D reg1D) plasmid pKD16 <br />   d <br />     Growth conditions described materials meth-         (ADR1 wild type) pKD14 (Adr1c). <br />                                                                      b <br /> ods.                                                                   mRNA levels measured RT&#8211;qPCR described in <br />   e <br />     NM, measured.                                              materials methods. Data mean three <br />                                                                    replicates (standard deviation). <br />  <br />  <br /> addition cycloheximide (Cyh), Adr1-pSer230              derepressing conditions, relative level Adr1- <br /> total Adr1 decreased rate, suggesting             pSer230 did decrease (compare samples lanes <br /> Ser230-phosphorylated Adr1 total Adr1 turn             b, 1Cyh). Moreover, conditions, the <br /> the rate repressing derepressing                  relative level Adr1-pSer230 did change when <br /> growth conditions absence protein synthesis              glucose added (compare samples lanes a <br /> (data shown).                                                  c, 1Cyh). Similarly, protein synthesis in- <br />    In complex experiment diagrammed Figure               hibited derepressing conditions (right panels) and <br /> 5, inhibitor added repressed (lanes a&#8211;c,         culture shifted repressing conditions, <br /> 1Cyh) derepressed cultures (samples lanes d              relative level Adr1-pSer230 did increase <br /> e, 1Cyh), samples removed various times             (compare samples lanes d e, 1Cyh). These <br /> assay total Adr1 Adr1-pSer230. Control cultures            results suggest synthesis phosphatase, kinase, <br /> lacking inhibitor sampled time. In            new Adr1 needed modulate ratio Adr1- <br /> the absence Cyh, level phosphorylation de-               pSer230 total Adr1 cells encounter different <br /> creased 3- 4.5-fold 1 1.5 hr derepression,        glucose concentrations. <br /> respectively (compare samples lanes a, b, d,               Screening Adr1-Ser230 kinase: A peptide <br /> Cyh). Adding glucose control derepressed             representing Ser230 region Adr1 efficient <br /> cultures resulted return repressed level            substrate vitro PKA (Denis et al. 1992) and <br /> phosphorylation (samples lanes c e, Cyh).                   CMK (Hook et al. 1999). Early studies suggested that <br /> However, protein synthesis inhibited repressing          Adr1 inactivated vivo phosphorylation of <br /> conditions (left panels) culture shifted         Ser230 PKA (Cherry et al. 1989), genetic analysis <br />  <br />                                                                                   Figure 4.&#8212;Adr1c suppresses cat8D-associated <br />                                                                                 defects Adr1-dependent gene expression. Cell <br />                                                                                 growth described materials methods. <br />                                                                                 RT&#8211;qPCR performed analyzed de- <br />                                                                                 scribed materials methods. Strains were <br />                                                                                 wild type (wt) CKY13 (adr1D) cat8D CKY23 <br />                                                                                 (adr1D cat8D), carrying wild-type <br />                                                                                 ADR1 (pKD16) ADR1-S230A (Adr1c, pKD14). <br />                                                                                 Derepression 4 hr 0.05% glucose&#8211;3% <br />                                                                                 glycerol. The mRNA values normalized to <br />                                                                                 ACT1 mRNA expressed percentage <br />                                                                                 mRNA level wild-type CAT8 strain <br />                                                                                 Adr1c activator. The values represent <br />                                                                                 mean 6 standard deviation bio- <br />                                                                                 logical replicates assayed duplicate. <br />  742                                                     S. Ratnakumar et al. <br />  <br />                                                                                                        Figure 5.&#8212;Stability Adr1- <br />                                                                                                     HA Adr-pSer230 ab- <br />                                                                                                     sence protein synthesis. Strain <br />                                                                                                     NKY85 grown complete <br />                                                                                                     medium containing glucose a <br />                                                                                                     final concentration 5% (R, re- <br />                                                                                                     pressing; sample lanes a&#8211;c) At <br />                                                                                                     time indicated vertical <br />                                                                                                     bars (top), portion cul- <br />                                                                                                     ture shifted derepressing <br />                                                                                                     (DR) conditions (sample lanes d <br />                                                                                                     e) centrifugation re- <br />                                                                                                     suspension low-glucose me- <br />                                                                                                     dium. In cases indicated, <br />                                                                                                     glucose added dere- <br />                                                                                                     pressed cultures times indi- <br />                                                                                                     cated vertical bars to <br />                                                                                                     reinstate repressive conditions. <br />                                                                                                     Cyh added final concen- <br /> tration 100 mg/ml times indicated vertical arrow portion culture inhibit protein synthesis. Samples were <br /> removed electrophoresis Western blotting points indicated arrows a&#8211;e. The signal intensity measured as <br /> described materials methods biological replicates. The values shown mean 6 standard deviation. <br />  <br />  <br /> was supportive model (Denis et al. 1992;                    As second method testing vivo phosphor- <br /> Dombek Young 1997). To determine                          ylation pSer230 PKA, measured relative level <br /> Ser230 phosphorylated vivo protein                     Adr1-pSer230 strain lacking Bcy1, regulatory <br /> kinases, used yeast strains lacking PKA activity               subunit PKA. The bcy1D mutation introduced into <br /> (tpk1Dtpk2Dtpk3Dyak1D tpk1Dtpk2Dtpk3-ts)                   multicopy strain used study pSer230 the <br /> CMK activity (cmk1Dcmk2Dcmk3D). The tpk-null strain                   mutants described Figures 2 5. The total amount <br /> was kept viable deleting YAK1, encoding                    Adr1, determined blotting HA epitope on <br /> protein kinase (Garrett Broach 1989). This strain                 Adr1, lower bcy1D strain wild-type <br /> showed normal glucose regulation ADH2 (data                        parent, expected, constitutive activation of <br /> not shown), consistent previous studies                   Tpk reduces expression ADR1 promoter <br /> tpk1Dtpk2Dtpk3-ts mutant (Vallari et al. 1992; Dombek                 (Dombek 1997). We observed slow rate ADH2 de- <br /> and Young 1997). The triple cmk-null strain viable                repression, measured situ staining ADH <br /> but grew slowly, ADH2 regulation normal              activity reporter assays (Figure S5), likely <br /> (data shown). We expected pSer230 absent                 lower levels Adr1. If PKA phosphor- <br /> the mutations eliminated Ser230 kinase, phos-                 ylates Ser230 vivo, expected significant <br /> phorylated Adr1 clearly detectable strains lacking             increase Adr1-pSer230 strain. However, the <br /> either PKA CMK. The relative level Adr1-pSer230                 relative level Adr1-pSer230 unaffected the <br /> compared total Adr1 measured Western                        bcy1D mutant (Figure 6B). <br /> blotting strains containing overexpressed Adr1                    We considered possibility Adr1-pSer230 <br /> was altered factor                  fully phosphorylated repressing <br /> for PKA CMK mutant strains (Figure 6A).                      conditions (Figure S1). If so, effect constitu- <br /> The 30% reduction cmk1-3D strain                      tive activation PKA bcy1D strain be <br /> consequence partial degradation Adr1                   observed derepressing conditions, Adr1- <br /> occurred strain. A similar result obtained           pSer230 decreases BCY1 parent strain (Figure 2). <br /> tpk1Dtpk2Dtpk3-ts mutant (N. Kacherovsky, unpub-                      However, loss BCY1 did increase relative level <br /> lished results).                                                      pSer230 repressing derepressing con- <br />    The presence Adr1 phosphorylated Ser230                   ditions (Figure 6B data shown). This result is <br /> the tpk-null strain overexpression. Adr1-              unlikely consequence reduced level of <br /> pSer230 detected tpk1-3DyakD strain              Adr1 bcy1D mutant level Adr1 in <br /> single-copy Adr1 immunoprecipitation. The im-                   multicopy bcy1D strain similar level Adr1 <br /> mune precipitates wild type quadruple                    single-copy ADR1 BCY1 wild-type strain. <br /> mutant analyzed SDS&#8211;PAGE Western                             In combination results PKA triple- <br /> blotting Adr1. The Licor secondary antibody re-                   mutant strains, results different approaches <br /> vealed similar Adr1 immune                        suggested PKA main kinase Adr1 <br /> precipitates (Figure S4). The low concentration Adr1-              Ser230 vivo. Adr1-Ser230 clearly phosphorylated <br /> pSer230 supernatant fraction precluded                     CMK mutant, concluded PKA <br /> detection.                                                            CMK main Adr1-Ser230 kinase. PKA CMK <br />                                               Adr1c Alleles Suppress Snf1 Deficiency                                      743 <br />  <br />                                                                   mutant lacking yeast genes encoding Cmk <span id='am-55' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-58'  ></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>homo</span>- <br />                                                                   logs minor effect Ser230 phosphorylation. <br />                                                                   A screen 102 viable yeast protein kinase mutants failed <br />                                                                   identify candidate kinase Ser230. In experi- <br />                                                                   ments test Ser230 kinase candidates, overexpression <br />                                                                   Adr1 necessary quantify level Adr1- <br />                                                                   pSer230. Although rule possibility <br />                                                                   nonspecific phosphorylation overexpressed <br />                                                                   protein masked identification important kinase, <br />                                                                   wild-type control plasmid did show <br />                                                                   excessive nonspecific phosphorylation, making this <br />                                                                   explanation unlikely. Moreover, strain expressing <br />                                                                   normal low levels Adr1, Ser230 phosphorylation <br />                                                                   appeared present level wild-type <br />                                                                   tpk-null strains. In vivo phosphorylation Ser230 <br />                                                                   involve Cmk Tpk, unregulated Tpk <br />                                                                   activity did significantly increase level Adr1- <br />                                                                   pSer230, making possibility likely. Therefore, we <br />                                                                   tentatively conclude Ser230 phos- <br />                                                                   phorylated multiple protein kinases include <br />    Figure 6.&#8212;Adr1-pSer230 phosphorylation signifi- <br />                                                                   PKA CMK specific kinase essential absent <br /> cantly altered tpkD cmkD mutants bcy1D mutant. <br /> Preparation cell extracts, electrophoresis, Western        deletion collection reason. Protein <br /> blotting carried described materials meth-         synthesis appeared necessary change relative <br /> ods legend Figure 1. (A) Wild type (wt)          level Adr1-pSer230. This interpreted to <br /> CMY323, yak1D SGP400, tpk1D tpk2D tpk3D yak1D               indicate Adr1-specific protein kinase be <br /> SGP406, cmk1D cmk2D cmk3D NKY108. All strains <br /> were transformed plasmid pNKA1, HA-tagged <br />                                                                   synthesized cells return repressing growth con- <br /> Adr1 expressed ADH1 promoter multicopy              ditions protein phosphatase synthe- <br /> TRP1 plasmid. Cells grown trp synthetic medium            sized novo cells shifted repressing to <br /> with 5% glucose (repressing conditions). Fifty micrograms         derepressing conditions. <br /> of protein extract analyzed. (B) Wild type NKY85              Our results consistent original hypothesis <br /> and bcy1D NKY111, multicopy ADR1. Pro- <br />                                                                   phosphorylation Adr1 Ser230 plays im- <br /> tein extracts (25 mg wild type, 100 mg bcy1D) analyzed as <br /> described above. The numbers parentheses indicate          portant role Adr1 function altering ability to <br /> number replicates analyzed represent repressed        regulate downstream genes glucose-dependent <br /> (complete medium 5% glucose) derepressed (2, 4,          manner (Cherry et al. 1989). Further investigations <br /> and 22 hr complete medium 0.05% glucose&#8211;2%                required address issue role Snf1 in <br /> ethanol) cultures. <br />                                                                   modulating level Ser230 phosphorylation. Re- <br />                                                                   gardless mechanism, regulating level Adr1 <br /> might partial nonspecific effects Adr1                 phosphorylated Ser230 appears major <br /> phosphorylation.                                                  function Snf1 controlling expression Adr1- <br />    Because unable demonstrate major                  dependent genes nonphosphorylatable Adr1c <br /> reduction Adr1-pSer230 mutants lacking PKA              activate Adr1-dependent genes UAS1- <br /> CMK, screened 102 viable kinase deletion mutants (as           containing reporter Snf1. In contrast, wild-type <br /> designated <span id='am-29' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-30' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-34'  ></span><span id='am-35' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span>Saccharomyces</span> Genome Database)              Adr1 unable activate genes the <br /> their ability phosphorylate Adr1-Ser230. Each mutant           absence Snf1. This consistent interpreta- <br /> was transformed high-copy plasmid overexpressing           tion pSer230 major, albeit indirect, target by <br /> wild-type Adr1 facilitate screen. Immunoprecipita-         Snf1 regulates Adr1 activity. The remaining Snf1 <br /> tion anti-Adr1 antisera, followed Western blotting        dependence Adr1c activator genes <br /> with a-pSer230, used detect phosphorylated             uniquely dependent Adr1 function of <br /> protein. No mutants defective phosphorylat-          Snf1-dependent pathways important for <br /> ing Adr1-Ser230 identified (data shown).                 activation Adr1 (Voronkova et al. 2006) could <br />                                                                   reflect requirement Snf1 dephosphor- <br />                                                                   ylation pSer98. This site adjacent DNA- <br />                        DISCUSSION <br />                                                                   binding domain Adr1, Ser98 phosphorylation <br />   Although Ser230 canonical consensus                appears play repressive role Adr1 activity by <br /> sequence PKA phosphorylated PKA vitro            reducing Adr1 promoter binding (Kacherovsky et al. <br /> (Denis et al. 1992), different methods failed              2008). Like phosphorylation Ser230, phosphoryla- <br /> confirm PKA Ser230-specific kinase. In addition,         tion Ser98 modulated Snf1 activity reduced <br />  744                                                          S. Ratnakumar et al. <br />  <br /> in derepressing growth conditions. Whether                                              LITERATURE CITED <br /> modifications causally functionally related                     Biddick, R. K., G. L. Law E. T. Young, 2008 Adr1 Cat8 me- <br /> under investigation.                                                           diate coactivator recruitment chromatin remodeling at <br />    We previously suggested Adr1c Snf1 act                          glucose-regulated genes. PLoS ONE 3: e1436. <br />                                                                           Charbon, G., K. D. Breunig, R. Wattiez, J. Vandenhaute I. <br /> ADH2 expression separate pathways (Dombek                              Noel-Georis, 2004 Key role Ser562/661 Snf1-dependent <br /> et al. 1993) basis strong synergism                      regulation Cat8p <span id='am-12' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-13' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-14'  ></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> Kluyveromy- <br />                                                                                ces lactis. Mol. Cell. Biol. 24: 4083&#8211;4091. <br /> ADR1c deletion REG1. The data presented                        Cherry, J. R., T. R. Johnson, C. Dollard, J. R. Shuster C. L. <br /> here indicate synergism Cat8. Thus                      Denis, 1989 Cyclic AMP-dependent protein kinase phosphory- <br /> primary effect Snf1 ADH2 expression strain                          <span id='am-45' about='obo:NCBITaxon_8186' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_1489905 obo:NCBITaxon_123366 obo:NCBITaxon_1489904 obo:NCBITaxon_1489872 obo:NCBITaxon_8184 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-46' property="rdfs:label" content="Lates" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-48'  ></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>lates</span> inactivates yeast transcriptional activator ADR1. Cell <br />                                                                                56: 409&#8211;419. <br /> carrying ADR1c Cat8 expression                        Chi, A., C. Huttenhower, L. Y. Geer, J. J. Coon, J. E. Syka et al., <br /> activity Snf1 dependent. The enhanced activity                          2007 Analysis phosphorylation sites proteins Saccha- <br /> Adr1c related observation activated                      romyces cerevisiae <span id='am-73' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-76'  ></span><span id='am-77' property="rdfs:label" content="Electron" datatype="xsd:string"></span>electron</span> transfer dissociation (ETD) mass <br />                                                                                spectrometry. Proc. Natl. Acad. Sci. USA 104: 2193&#8211;2198. <br /> genes normally codependent Adr1 Cat8                        Ciriacy, M., 1979 Isolation characterization <span id='am-68' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-72'  ></span>cis</span>- and <br /> absence Cat8. Eukaryotic promoters generally re-                            trans-acting regulatory elements involved synthesis glu- <br /> spond different environmental signals                           cose-repressible alcohol dehydrogenase (ADHII) Saccharomy- <br />                                                                                ces cerevisiae. Mol. Gen. Genet. 176: 427&#8211;431. <br /> action multiple transcription factors, leading                      Collart, M., S. Oliveiro (Editors), 1993 Current Protocols in <br /> combinatorial control gene expression. Many Adr1-                           Molecular Biology. Greene/Wiley Interscience, New York. <br /> dependent genes activated transcription                      Cook, W. J., D. Chase, D. C. Audino C. L. Denis, <br />                                                                                1994 Dissection ADR1 protein reveals multiple, function- <br /> factors well. For example, promoters genes                           ally redundant activation domains interspersed inhibitory <br /> encoding peroxisomal proteins enzymes b-                            regions: evidence repressor binding ADR1c region. <br /> oxidation bind Adr1 heterodimeric,                                Mol. Cell. Biol. 14: 629&#8211;640. <br />                                                                           Cross, F. R., 1997 &#8216;Marker swap&#8217; plasmids: convenient tools bud- <br /> oleate-responsive transcription factors Oaf1 Pip2                          ding yeast molecular genetics. Yeast 13: 647&#8211;653. <br /> (Simon et al. 1991, 1992; Gurvitz et al. 2000, 2001;                      Denis, C. L., 1984 Identification new genes involved the <br /> Young et al. 2003). We investigating                               regulation yeast alcohol dehydrogenase II. Genetics 108: <br />                                                                                833&#8211;844. <br /> Adr1c compensate loss Oaf1/Pip2                     Denis, C. L., 1987 The effects ADR1 CCR1 gene dosage on <br /> expression peroxisomal genes, does loss                    regulation glucose-repressible alcohol dehydrogenase <br /> Cat8 Adr1- Cat8-dependent promoters.                                    Saccharomyces cerevisiae. Mol. Gen. Genet. 208: 101&#8211;106. <br />                                                                           Denis, C. L., C. Gallo, 1986 Constitutive RNA synthesis the <br />    The finding Adr1c suppress requirement                         yeast activator ADR1 identification ADR1&#8211;5c muta- <br /> for Cat8 suggests that, spite low protein levels,                    tion: implications posttranslational control ADR1. Mol. <br />                                                                                Cell. Biol. 6: 4026&#8211;4030. <br /> Adr1c characteristics similar overexpressed wild-                  Denis, C. L., B. E. Kemp M. J. Zoller, 1991 Substrate specific- <br /> type Adr1, escapes Snf1 regulation (Denis                           ities yeast mammalian cAMP-dependent protein kinases <br /> 1987) compensates loss Cat8 coregu-                         similar identical. J. Biol. Chem. 266: 17932&#8211;17935. <br />                                                                           Denis, C. L., S. C. Fontaine, D. Chase, B. E. Kemp L. T. Bemis, <br /> lated promoters (Biddick et al. 2008). In vitro, elec-                      1992 ADR1c mutations enhance ability ADR1 activate <br /> trophoretic mobility shift assays, Adr1c does                         transcription mechanism independent effects on <br /> exceptional DNA-binding activity (Taylor Young                             cyclic AMP-dependent protein kinase phosphorylation Ser- <br />                                                                                230. Mol. Cell. Biol. 12: 1507&#8211;1514. <br /> 1990); however, able bind promoters                          Dombek, K. M., E. T. Young, 1997 Cyclic AMP-dependent pro- <br /> efficiently vivo, factors chromatin                           tein kinase inhibits ADH2 expression decreasing ex- <br /> structure stabilization coactivator interaction                         pression transcription factor gene ADR1. Mol. Cell. <br />                                                                                Biol. 17: 1450&#8211;1458. <br /> are proposed influence Adr1 binding (Tachibana                         Dombek, K. M., S. Camier E. T. Young, 1993 ADH2 expression <br /> et al. 2007; Young et al. 2008). The wide variety                           repressed REG1 independently mutations alter the <br /> constitutive Adr1 mutations, complete deletion                       phosphorylation yeast transcription factor ADR1. Mol. <br />                                                                                Cell. Biol. 13: 4391&#8211;4399. <br /> of amino acids 220&#8211;263 262&#8211;330 (Cook et al. 1994),                    Featherstone, M., 2002 Coactivators transcription initiation: <br /> suggests Ser230 phosphorylation site,                        orders. Curr. Opin. Genet. Dev. 12: 149&#8211;155. <br /> but entire region important Adr1 regu-                    Fry, C. J., C. L. Peterson, 2002 Transcription. Unlocking the <br />                                                                                gates gene expression. Science 295: 1847&#8211;1848. <br /> lation. Our results consistent model                      Garrett, S., J. Broach, 1989 Loss Ras activity Saccharo- <br /> mutations area affect protein structure alter                       myces cerevisiae suppressed disruptions new kinase <br /> interaction factors (Cook et al. 1994).                             gene, YAKI, product act downstream cAMP- <br />                                                                                dependent protein kinase. Genes Dev. 3: 1336&#8211;1348. <br /> Although Adr1-specific repressor                     Goel, A., R. Janknecht, 2003 Acetylation-mediated transcrip- <br /> spite genetic screens, findings                         tional activation ETS protein ER81 p300, P/CAF, and <br /> Adr1c suggest model drive investiga-                        HER2/Neu. Mol. Cell. Biol. 23: 6243&#8211;6254. <br />                                                                           Guarente, L., 1983 Yeast promoters lacZ fusions designed to <br /> tion novel interaction Adr1 Snf1.                           study expression cloned genes yeast. Methods Enzymol. <br />                                                                                101: 181&#8211;191. <br />   We thank members lab advice, encouragement, and <br />                                                                           Guldener, U., S. Heck, T. Fiedler, J. Beinhauer J. H. <br /> reagents; Ella Chang performing quantitative RT&#8211;               Hegemann, 1996 A new efficient gene disruption cassette for <br /> qPCR analyses; Catherine Kehl Kenneth Dombek strain                    repeated use budding yeast. Nucleic Acids Res. 24: 2519&#8211;2524. <br /> construction; Carl Mann strains; Chris Tachibana valuable     Gurvitz, A., L. Wabnegger, H. Rottensteiner, I. W. Dawes, A. <br /> comments manuscript. This work supported grant (no.            Hartig et al., 2000 Adr1p-dependent regulation oleic <br /> GM26079) National Institutes Health E.T.Y.                      acid-inducible yeast gene SPS19 encoding peroxisomal <br />                                                     Adr1c Alleles Suppress Snf1 Deficiency                                                       745 <br />  <br />     <span id='am-6' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>beta</span>-oxidation auxiliary enzyme 2,4-dienoyl-CoA reductase. Mol.       Schuller, H. J., 2003 Transcriptional control nonfermentative <br />     Cell Biol. Res. Commun. 4: 81&#8211;89.                                          metabolism yeast Saccharomyces cerevisiae. Curr. Genet. <br /> Gurvitz, A., J. K. Hiltunen, R. Erdmann, B. Hamilton, A. Hartig                43: 139&#8211;160. <br />     et al., 2001 Saccharomyces cerevisiae Adr1p governs fatty acid        Sherman, F., 1991 Getting started yeast. Methods Enzymol. <br />     beta-oxidation peroxisome proliferation regulating                  194: 3&#8211;21. <br />     POX1 PEX11. J. Biol. Chem. 276: 31825&#8211;31830.                      Simon, M., G. Adam, W. Rapatz, W. Spevak H. Ruis, 1991 The <br /> Hahn, S., 2008 Rapid yeast protein prep SDS&#8211;PAGE                       Saccharomyces cerevisiae ADR1 gene positive regulator tran- <br />     Western. http://www.fhcrc.org/science/labs/hahn/methods/                   scription genes encoding peroxisomal proteins. Mol. Cell. <br />     biochem_meth/yeast_prot_SDS.html).                                         Biol. 11: 699&#8211;704. <br /> Hardie, D. G., D. Carling M. Carlson, 1998 The AMP-activated/         Simon, M., M. Binder, G. Adam, A. Hartig H. Ruis, <br />     SNF1 protein kinase <span id='am-42' about='obo:NCBITaxon_subfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="subfamily" datatype="xsd:string"></span>subfamily</span>: Metabolic sensors eukaryotic         1992 Control peroxisome proliferation Saccharomyces <br />     cell? Annu. Rev. Biochem. 67: 821&#8211;855.                                     cerevisiae ADR1, SNF1 (CAT1, CCR1) SNF4 (CAT3). Yeast <br /> Hedges, D., M. Proft K. D. Entian, 1995 CAT8, new zinc                   8: 303&#8211;309. <br />     cluster-encoding gene necessary derepression gluconeo-         Sloan, J. S., K. M. Dombek E. T. Young, 1999 Post-translational <br />     genic enzymes yeast Saccharomyces cerevisiae. Mol. Cell. <br />                                                                                regulation Adr1 activity mediated DNA binding do- <br />     Biol. 15: 1915&#8211;1922. <br />                                                                                main. J. Biol. Chem. 274: 37575&#8211;37582. <br /> Hong, S. P., F. C. Leiper, A. Woods, D. Carling M. Carlson, <br />                                                                           Tachibana, C., R. Biddick, G. L. Law E. T. Young, 2007 A <br />     2003 Activation yeast Snf1 mammalian AMP-activated <br />     protein kinase upstream kinases. Proc. Natl. Acad. Sci. USA             poised initiation complex activated SNF1. J. Biol. Chem. <br />     100: 8839&#8211;8843.                                                            282: 37308&#8211;37315. <br /> Hook, S. S., B. E. Kemp A. R. Means, 1999 Peptide specificity de-     Tansey, W., 2001 Transcriptional activation: risky business. Genes <br />     terminants P-7 P-6 enhance catalytic efficiency Ca21/        Dev. 15: 1045&#8211;1050. <br />     calmodulin-dependent protein kinase I absence activa-       Taylor, W. E., E. T. Young, 1990 cAMP-dependent phosphor- <br />     tion loop phosphorylation. J. Biol. Chem. 274: 20215&#8211;20222.                ylation inactivation yeast transcription factor ADR1 does <br /> Kacherovsky, N., C. Tachibana, E. Amos, D. Fox, III E. T.                  affect DNA binding. Proc. Natl. Acad. Sci. USA 87: 4098&#8211; <br />     Young, 2008 Promoter binding Adr1 transcriptional ac-               4102. <br />     tivator regulated phosphorylation DNA-binding        Tu, J., M. Carlson, 1995 REG1 binds protein phosphatase <br />     region. PLoS ONE 3: e3213.                                                 type 1 regulates glucose repression Saccharomyces cerevi- <br /> Knop, M., K. Siegers, G. Pereira, W. Zachariae, B. Winsor et al.,              siae. EMBO J. 14: 5939&#8211;5946. <br />     1999 Epitope tagging yeast genes using PCR-based strategy:       Vallari, R. C., W. J. Cook, D. C. Audino, M. J. Morgan, D. E. Jensen <br />     tags improved practical routines. Yeast 15: 963&#8211;972.              et al., 1992 Glucose repression yeast ADH2 gene occurs <br /> Kwok, R. P., J. R. Lundblad, J. C. Chrivia, J. P. Richards, H. P.              multiple mechanisms, including control protein <br />     Bachinger et al., 1994 Nuclear protein CBP coactivator                synthesis transcriptional activator, ADR1. Mol. Cell. Biol. 12: <br />     transcription factor CREB. Nature 370: 223&#8211;226.                    1663&#8211;1673. <br /> Lo, W. S., L. Duggan, N. C. Emre, R. Belotserkovskya, W. S. Lane          Verdone, L., J. Wu, K. van Riper, N. Kacherovsky, M. Vogelauer <br />     et al., 2001 Snf1&#8212;a histone kinase works concert              et al., 2002 Hyperacetylation chromatin ADH2 pro- <br />     histone acetyltransferase Gcn5 regulate transcription.              moter allows Adr1 bind repressed conditions. EMBO J. <br />     Science 293: 1142&#8211;1146.                                                    21: 1101&#8211;1111. <br /> McCartney, R. R., M. C. Schmidt, 2001 Regulation Snf1 ki-          Voronkova, V., N. Kacherovsky, C. Tachibana, D. Yu E. T. <br />     nase. Activation requires phosphorylation threonine 210           Young, 2006 Snf1-dependent Snf1-independent pathways <br />     upstream kinase distinct step mediated Snf4            constitutive ADH2 expression Saccharomyces cerevisiae. <br />     subunit. J. Biol. Chem. 276: 36460&#8211;36466.                                  Genetics 172: 2123&#8211;2138. <br /> Orlova, M., L. Barrett S. Kuchin, 2008 Detection endog-            Young, E. T., N. Kacherovsky K. van Riper, 2002 Snf1 protein <br />     enous Snf1 activation state: application Saccharomyces          kinase regulates Adr1 binding chromatin transcrip- <br />     <span id='am-36' about='obo:NCBITaxon_5475' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-37'  ></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Torulopsis" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Candida" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Candida</span> <span id='am-65' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>. Yeast 25: 745&#8211;754. <br />                                                                                tional activation. J. Biol. Chem. 277: 38095&#8211;38103. <br /> Papoutsopoulou, S., R. Janknecht, 2000 Phosphorylation of <br />                                                                           Young, E. T., K. M. Dombek, C. Tachibana T. Ideker, <br />     ETS transcription factor ER81 complex coactivators <br />                                                                                2003 Multiple pathways co-regulated protein kinase <br />     CREB-binding protein p300. Mol. Cell. Biol. 20: 7300&#8211;7310. <br /> Rahner, A., A. Scholer, E. Martens, B. Gollwitzer H. J.                    Snf1 transcription factors Adr1 Cat8. J. Biol. Chem. <br />     Schuller, 1996 Dual influence yeast Cat1p (Snf1p) pro-              278: 26146&#8211;26158. <br />     tein kinase carbon source-dependent transcriptional activa-        Young, E. T., C. Tachibana, H. W. Chang, K. M. Dombek, E. M. Arms <br />     tion gluconeogenic genes regulatory gene CAT8.                   et al., 2008 Artificial recruitment mediator DNA-binding <br />     Nucleic Acids Res. 24: 2331&#8211;2337.                                          domain Adr1 overcomes glucose repression ADH2 expres- <br /> Rahner, A., M. Hiesinger H. J. Schuller, 1999 Deregulation                 sion. Mol. Cell. Biol. 28: 2509&#8211;2516. <br />     gluconeogenic structural genes variants transcrip-       Yu, J., M. S. Donoviel E. T. Young, 1989 Adjacent upstream <br />     tional activator Cat8p yeast Saccharomyces cerevisiae.              activation sequence elements synergistically regulate tran- <br />     Mol. Microbiol. 34: 146&#8211;156.                                               scription ADH2 Saccharomyces cerevisiae. Mol. Cell. <br /> Rubenstein, E. M., R. R. McCartney, C. Zhang, K. M. Shokat, M. K.              Biol. 9: 34&#8211;42. <br />     Shirra et al., 2008 Access denied: Snf1 activation loop phosphor-     Zhong, H., M. J. May, E. Jimi S. Ghosh, 2002 The phosphor- <br />     ylation controlled availability phosphorylated threo-         ylation status nuclear NF-kappa B determines association <br />     210 PP1 phosphatase. J. Biol. Chem. 283: 222&#8211;230.              CBP/p300 HDAC-1. Mol. Cell 9: 625&#8211;636. <br /> Santangelo, G. M., 2006 Glucose signaling Saccharomyces cer- <br />     evisiae. Microbiol. Mol. Biol. Rev. 70: 253&#8211;282. <br /> Sanz, P., G. R. Alms, T. A. Haystead M. Carlson, 2000 Reg- <br />     ulatory interactions Reg1-Glc7 protein phosphatase <br />     Snf1 protein kinase. Mol. Cell. Biol. 20: 1321&#8211;1328.                                            Communicating editor: M. Hampsey <br />                             Supporting Information <br />      http://www.genetics.org/cgi/content/full/genetics.109.103432/DC1 <br />  <br />  <br />  <br />  <br />     Snf1 Controls Activity Adr1 Through Dephosphorylation of <br />                                     Ser230 <br /> &#160; <br />  <br />        Sooraj Ratnakumar, Nataly Kacherovsky, Erin Arms Elton T. Young <br /> &#160; <br /> &#160; <br />            Copyright &#169; 2009 Genetics Society America <br />                        DOI:10.1534/genetics.109.103432 <br />  2 SI&#8233;                                 S. Ratnakumar et al. <br />  <br />  <br />  <br />  <br />            FIGURE S1.&#8212;Stoichiometry pSer230 phosphorylation. Wildtype (wt) NKY85 <br />         (multicopy ADR1); &#916; TYY204 (adr1&#916;1::LEU2). Whole cell extracts (WCE) made <br />         cells grown repressing conditions. 500 &#956;g protein immuneprecipitated (IP) <br />         antibody shown. After IP 1/5 pellet 1/10 supernatant was <br />         analyzed SDS-PAGE 6.5% gel. After Western blotting transferred protein <br />         identified anti-Adr1 primary Licor secondary antibodies. <br />                                                      S. Ratnakumar et al.                                                  3 SI&#8233; <br />  <br />  <br />  <br />  <br />    FIGURE S2.&#8212;SNF1-independence Adr1c synergism Adr1c reg1&#916;. A. &#223;-galactosidase activity. Reporter gene <br /> assays performed using transformants strains TYY497 (adr1&#916;SNF1) TYY498 (adr1&#916;snf1&#916;) carrying <br /> either wildtype ADR1 (pKD16) ADR1-S230A (pKD14). &#223;-galactosidase activity reported Miller units (GUARENTE <br /> 1983). The assays average standard deviation 20%. R, 5% glucose; derepressed (DR) values represent 18 <br /> hours low glucose medium. A dilution series extracts used quantify four-fold reduction ADH2 <br /> derepression snf1 mutant Adr1c, compared derepression wildtype Adr1 (data shown). B. ADH situ <br /> enzyme assays performed described Materials Methods. Derepression hours. Wildtype (wt) is <br /> CKY13 (adr1&#916;); adr1&#916;snf1&#916; CKY26, adr1&#916;reg1&#916; CKY5, carrying wildtype ADR1 (pKD16) ADR1- <br /> S230A (pKD14), respectively. ADHI constitutive fermentative isozyme alcohol dehydrogenase encoded ADH1. <br /> ADHII product ADH2 gene. Extracts prepared glucose-grown wild-type cells carrying pKD16 or <br /> pKD14 did band ADHII activity (data shown). <br />  4 SI&#8233;                                       S. Ratnakumar et al. <br />  <br />  <br />  <br />  <br />            FIGURE S3.&#8212;Adr1c suppresses cat8&#916;-associated defects Adr1-dependent gene expression. Cell <br />         growth described Materials Methods. A. &#223;-galactosidase activity. CAT8 TYY497; cat8&#916; is <br />         TYY495 carrying wild-type ADR1 (pKD16) ADR1-S230A (pKD14). &#223;-galactosidase activity is <br />         reported Miller units (GUARENTE 1983). The assays average standard deviation about <br />         20%. Derepressed (DR) values represent 20 hours low glucose medium. B. ADH situ enzyme <br />         assays performed described Materials Methods. Derepression hours. <br />         Wildtype CKY13 (adr1&#916;); cat8&#916; CKY23 (adr1&#8710; cat8&#8710;) carrying wildtype ADR1 (pKD16) <br />         ADR1-S230A (pKD14). <br />                                    S. Ratnakumar et al.                                               5 SI&#8233; <br />  <br />  <br />  <br />  <br />    FIGURE S4. &#8212;Adr1-Ser230 wildtype tpk1-3&#916; yak1&#916;. strains single copy ADR1. &#945;- <br /> Adr1 western immunoprecipitation (IP) cell extracts using anti-pSer230 antisera. IP <br /> of 500 &#956;g protein ADR1 multicopy strain (mcADR1) NK85 1 mg protein the <br /> ADR1 single copy (scADR1) strains CKY323 (wt) SGP406 (yak1&#8710; tpk1-3&#8710;). The fractions <br /> represent relative IP pellet analyzed SDSPAGE 6.5% gel. <br /> WCE: 50 &#956;g cell extract strain NK85. <br />  6 SI&#8233;                                       S. Ratnakumar et al. <br />  <br />  <br />  <br />  <br />            FIGURE S5. &#8212;ADH2 expression BCY1 bcy1&#916; strains multicopy ADR1. In gel assays ADH <br />         activity ADH2-&#946;-galactosidase activity multicopy strains NKY85 NKY111. Cells grown in <br />         YP-8% glucose derepressed times indicated YP-0.05% glucose-2% ethanol. Cell extracts were <br />         prepared analyzed described Materials Methods. Twenty &#956;g protein analyzed. <br />                                        S. Ratnakumar et al.                                  7 SI&#8233; <br />  <br />  <br />  <br />  <br />                                              TABLE S1 <br />  <br /> Gene expression levels Adr1 wildtype Adr1c activators CAT8 wildtype cat8&#8710; <br />  <br />                       repressing derepressing growth conditions <br />  <br />  <br />                                                                          Derepressed <br />                                                                          % (cat8&#8710;/CAT8) <br />  Gene     Genotype1      Activator2    mRNA     level/ACT13       DR/R   wt Adr1   Adr1c <br />                                        Repressed    Derepressed <br />                                        R            DR <br />  ACS1     CAT8           wtAdr1        0.013        3.3           254    3.3       42 <br />           cat8&#8710;          wtAdr1        0.012        0.11          9 <br />           CAT8           Adr1c         0.026        4.3           165 <br />           cat8&#8710;          Adr1c         0.021        1.8           86 <br />  <br />  <br />  ADH2     CAT8           wtAdr1        0.027        7.9           293    1.0       63 <br />           cat8&#8710;          wtAdr1        0.026        0.082         3 <br />           CAT8           Adr1c         0.14         11            79 <br />           cat8&#8710;          Adr1c         0.1          6.9           69 <br />  <br />  <br />  ADR1     CAT8           wtAdr1        0.016        0.059         4      42        88 <br />           cat8&#8710;          wtAdr1        0.011        0.025         2 <br />           CAT8           Adr1c         0.0079       0.041         5 <br />           cat8&#8710;          Adr1c         0.0096       0.036         4 <br />  <br />  <br />  ADY2     CAT8           wtAdr1        0.0063       1.3           206    1.0       20 <br />           cat8&#8710;          wtAdr1        0.0016       0.012         8 <br />           CAT8           Adr1c         0.0085       2.4           282 <br />           cat8&#8710;          Adr1c         0.0041       0.48          117 <br />  <br />  <br />  <br />  <br />  ALD4     CAT8           wtAdr1        0.048        1             21     47        190 <br />           cat8&#8710;          wtAdr1        0.055        0.47          9 <br />           CAT8           Adr1c         0.11         1.3           12 <br />           cat8&#8710;          Adr1c         0.15         2.5           17 <br />  <br />  <br />  ALP1     CAT8           wtAdr1        0.007        0.14          20     nm        nm <br />           cat8&#8710;          wtAdr1 <br />           CAT8           Adr1c         0.0085       0.15          18 <br />           cat8&#8710;          Adr1c <br />  <br />  <br />  ATO3     CAT8           wtAdr1        0.056        0.75          13     6.8       89 <br />  8 SI&#8233;                           S. Ratnakumar et al. <br />  <br />  <br />  <br />  <br />                cat8&#8710;   wtAdr1    0.066        0.051    1 <br />                CAT8    Adr1c     0.059        0.75     13 <br />                cat8&#8710;   Adr1c     0.066        0.67     10 <br />  <br />  <br />         CIT3   CAT8    wtAdr1    0.0087       0.14     16    27    78 <br />                cat8&#8710;   wtAdr1    0.00934      0.038    4 <br />                CAT8    Adr1c     0.01         0.18     18 <br />                cat8&#8710;   Adr1c     0.0073       0.14     19 <br />  <br />  <br />         CYB2   CAT8    wtAdr1    0.035        0.28     8     75    180 <br />                cat8&#8710;   wtAdr1    0.034        0.21     6 <br />                CAT8    Adr1c     0.044        0.25     6 <br />                cat8&#8710;   Adr1c     0.032        0.4      13 <br />  <br />  <br />         FBP1   CAT8    wtAdr1    0.0083       1.1      133   3.6   10 <br />                cat8&#8710;   wtAdr1    0.0034       0.04     12 <br />                CAT8    Adr1c     0.0058       0.55     95 <br />                cat8&#8710;   Adr1c     0.008        0.055    7 <br />  <br />  <br />         FDH    CAT8    wtAdr1    0.006        0.049    8     6.9   43 <br />                cat8&#8710;   wtAdr1    0.006        0.0034   1 <br />                CAT8    Adr1c     0.01         0.35     35 <br />                cat8&#8710;   Adr1c     0.006        0.15     25 <br />  <br />  <br />         FOX2   CAT8    wtAdr1    0.0085       0.063    7     33    130 <br />                cat8&#8710;   wtAdr1    0.0077       0.021    3 <br />                CAT8    Adr1c     0.0083       0.08     10 <br />                cat8&#8710;   Adr1c     0.004        0.1      25 <br />  <br />  <br />         GUT1   CAT8    wtAdr1    0.016        0.24     15    42    160 <br />                cat8&#8710;   wtAdr1    0.021        0.1      5 <br />                CAT8    Adr1c     0.015        0.25     17 <br />                cat8&#8710;   Adr1c     0.015        0.39     26 <br />  <br />  <br />         ICL1   CAT8    wtAdr1    0.037        2.5      68    3.8   3.9 <br />                cat8&#8710;   wtAdr1    0.017        0.094    6 <br />                CAT8    Adr1c     0.023        1.6      70 <br />                cat8&#8710;   Adr1c     0.0034       0.063    19 <br />  <br />  <br />         ICL2   CAT8    wtAdr1    0.014        0.14     10    16    47 <br />                cat8&#8710;   wtAdr1    0.011        0.023    2 <br />                                           S. Ratnakumar et al.                                                  9 SI&#8233; <br />  <br />  <br />  <br />  <br />               CAT8          Adr1c         0.0096       0.16           17 <br />               cat8&#8710;         Adr1c         0.0024       0.075          31 <br />  <br />  <br />     JEN1      CAT8          wtAdr1        0.0093       0.66           71          7.1        82 <br />               cat8&#8710;         wtAdr1        0.0054       0.047          9 <br />               CAT8          Adr1c         0.0088       0.82           93 <br />               cat8&#8710;         Adr1c         0.0047       0.67           143 <br />  <br />  <br />     MDH2      CAT8          wtAdr1        0.02         2.1            105         4.4        10 <br />               cat8&#8710;         wtAdr1        0.02         0.093          5 <br />               CAT8          Adr1c         0.023        1.8            78 <br />               cat8&#8710;         Adr1c         0.024        0.18           8 <br />  <br />  <br />     POT1      CAT8          wtAdr1        0.01         0.1            10          140        380 <br />               cat8&#8710;         wtAdr1        0.009        0. <br /> </body></html>