<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Apr. 1996, p. 1557&#8211;1566                                                                         Vol. 16, No. 4 <br /> 0270-7306/96/$04.0010 <br /> Copyright q 1996, American Society Microbiology <br />  <br />  <br />  <br />    Synthetic Enhancement TFIIB Defect Mutation SSU72, <br />         Essential Yeast Gene Encoding Novel Protein That <br />             Affects Transcription Start Site Selection In Vivo <br />                                            <span id='am-16' about='obo:NCBITaxon_143324' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_81386 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_81393 obo:NCBITaxon_7776 obo:NCBITaxon_123357 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-17' property="rdfs:label" content="Zu" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-19'  ></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>ZU</span>-WEN SUN&#8224;        AND     MICHAEL HAMPSEY* <br />                             Department Biochemistry Molecular Biology, Louisiana State University <br />                                            Medical Center, Shreveport, Louisiana 71130 <br />                     Received 14 November 1995/Returned modification 28 December 1995/Accepted 23 January 1996 <br />  <br />              An ssu72 mutant <span id='am-26' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> identified enhancer TFIIB defect (sua7-1) that <br />           confers cold-sensitive growth defect downstream shift transcription start site selection. The <br />           ssu72-1 allele did affect cold sensitivity but, combination sua7-1, created heat-sensitive phenotype. <br />           Moreover, start site selection ADH1 gene dramatically shifted downstream normal <br />           sites. Both effects rescued SUA7 SSU72, defining functional relationship <br />           genes. SSU72 single-copy, essential gene encoding novel protein 206 amino acids. The <br />           ssu72-1 allele result 30-bp duplication creating sequence encoding Cys-X2-Cys-X6-Cys-X2-Cys <br />           zinc binding motif near N terminus Ssu72p. Mutational analysis demonstrated N terminus of <br />           Ssu72p essential function cysteine residues normal mutant proteins critical. <br />           We discuss possibility potential zinc binding motif Ssu72p facilitates assembly tran- <br />           scription preinitiation complex effect important accurate start site selection vivo. <br />  <br />  <br />    Transcription initiation RNA polymerase II (RNAP II)                   functional transcription preinitiation complex vitro (7, 71). <br /> involves concerted action large number proteins.                 Assembly complex nucleated association TBP <br /> One <span id='am-60' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> gene specific binds promoters containing                 (or TFIID) TATA element, followed binding of <br /> specific target sequences (reviewed references 33 52).                TFIIB. TBP physically interacts TFIIB, crystal <br /> These factors stimulate rate transcription response             structure TFIIB-TBP-TATA element ternary complex was <br /> specific physiological developmental stimuli. In contrast,            recently described (45). RNAP II, association TFIIF, <br /> general transcription factors required accurate, basal-              binds complex. TFIIE TFIIH associate the <br /> level transcription absence gene-specific stimulation              complex binding RNAP II required pro- <br /> (reviewed references 6 77). With exception                 moter clearance manner dependent ATP <br /> TATA-binding protein (TBP), general factors bind                  hydrolysis (27). Although considerable evidence to <br /> to specific DNA sequences. A class factors, known                support scheme assembly preinitiation complex <br /> coactivators, mediators, adaptors, functions communica-                vitro, RNAP II appears recruited promoters vivo <br /> tion gene-specific general factors (reviewed                 holoenzyme complex includes general factors. There <br /> in reference 25). <span id='am-70' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72'  ></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> class includes TBP-associated fac-                direct support premise yeast mammalian <br /> tors that, TBP, make TFIID (26), oth-               systems, holoenzyme complexes somewhat different <br /> er adaptors, including Ada2p-Ada3p-Gcn5p complex                      compositions reported (35, 37, 47). <br /> <span id='am-43' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-44' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-48'  ></span><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae (31). Transcriptional activity                 Genetic studies using yeast S. cerevisiae provided <br /> affected chromatin structure DNA template (re-                 valuable new information regarding general transcription <br /> viewed reference 48). Both biochemical genetic exper- <br />                                                                              factors functions. For example, mutations SPT <br /> iments suggest function gene-specific factors is <br />                                                                              genes identified suppressors Ty solo d insertions <br /> to overcome repressive effects chromatin transcrip- <br />                                                                              inhibit transcription adjacent HIS4 LYS2 genes (72). <br /> tion initiation. The identities factors mecha- <br />                                                                              One genes, SPT15, encodes TBP (15, 29). Mutations <br /> nisms interact confer accurate, regulated <br /> initiation RNAP II subject intense investigation               SPT15 shift initiation d initiation site normal <br /> and critical understanding regulated cell growth                 initiation site HIS4, establishing role TBP in <br /> development.                                                                 normal transcription initiation vivo (15). Another class of <br />    The general transcription factors initially defined               genes, designated SUA, identified suppressors an <br /> biochemical studies include TBP (initially component                aberrant open reading frame located transcribed leader <br /> of TFIID complex), TFIIA, TFIIB, TFIIE, TFIIF, TFIIH,                    region CYC1 gene (50). Two genes, SUA7 and <br /> and TFIIJ (reviewed references 11 76). These factors                  SUA8 (RPB1) encode, respectively, TFIIB largest sub- <br /> assemble DNA template defined <span id='am-80' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-81' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> generate                  unit RNAP II (4, 50). Mutations gene shifted <br />                                                                              transcription start site selection downstream normal <br />                                                                              sites. These experiments demonstrated TFIIB the <br />                                                                              largest RNAP II subunit required accurate transcrip- <br />    * Corresponding author. Present address: Department Biochem- <br /> istry, Robert Wood Johnson Medical School, UMDNJ, 675 Hoes Ln., <br />                                                                              tion start site selection vivo suggested specific <br /> Piscataway, NJ 08854-5635.                                                   interactions TFIIB largest RNAP II subunit <br />    &#8224; Present address: Department Biochemistry, Robert Wood John-          affect process (4). <br /> son Medical School, University Medicine Dentistry New                 Genetic analysis transcription initiation po- <br /> Jersey, Piscataway, NJ 08854-5635.                                           tential identify transcription factors undetect- <br />  <br />                                                                       1557 <br />  1558       SUN AND HAMPSEY                                                                                                                          MOL. CELL. BIOL. <br />  <br />  <br />                                                              TABLE 1. Yeast strains used study <br />        Strain                                                                                                    Genotype <br /> T16 .............................................................................................MATa cyc1-5000 cyc7-67 leu2-3,112 ura3-52 cyh2 <br /> YDW546....................................................................................MATa cyc1-5000 cyc7-67 leu2-3,112 ura3-52 cyh2 sua7-1 <br /> YMH71-9C................................................................................MATa cyc1-5000 cyc7-67 trp5-48 his5-2 ura3-52 sua7-1 <br /> YZS19 ........................................................................................MATa cyc1-5000 cyc7-67 trp5-48 his5-2 ura3-52 sua7-1 ssu72-1 ssu73-1 <br /> YZS19/YCplac33 ......................................................................MATa cyc1-5000 cyc7-67 trp5-48 his5-2 ura3-52 sua7-1 ssu72-1 ssu73-1 [URA3] <br /> YZS19/pDW11..........................................................................MATa cyc1-5000 cyc7-67 trp5-48 his5-2 ura3-52 sua7-1 ssu72-1 ssu73-1 [URA3 SUA7] <br /> YZS19/pM385 ...........................................................................MATa cyc1-5000 cyc7-67 trp5-48 his5-2 ura3-52 sua7-1 ssu72-1 ssu73-1 [URA3 SSU72] <br /> YZS19/pM586 ...........................................................................MATa cyc1-5000 cyc7-67 trp5-48 his5-2 ura3-52 sua7-1 ssu72-1 ssu73-1 [URA3 SSU72] <br /> YZS19/pM681 ...........................................................................MATa cyc1-5000 cyc7-67 trp5-48 his5-2 ura3-52 sua7-1 ssu72-1 ssu73-1 [URA3 SSU73] <br /> YZS19-1B ..................................................................................MATa ura3-52 leu2-3,112 sua7-1 ssu72-1 <br /> YMH171 ....................................................................................MATa ura3-52 leu2-3,112 his3 trp1D <br /> YZS89A .....................................................................................MATa ura3-52 leu2-3,112 his3 trp1D ssu72::LEU2 [pM586: URA3 SSU72] <br /> LRB535/536...............................................................................MATa/MATa his3D200/his3D200 leu2-3,112/leu2-3,112 ura3-52/ura3-52 <br />  <br />  <br />  <br /> ed biochemical means. For example, SPT3 (13), SPT7                              sua7-1 segregant derived YDW546 (44). YZS19 spontaneous cs1 <br /> (21), SPT8 (14) genes encode novel factors play im-                           revertant YMH71-9C. YZS19/YCplac33 (sua7 ssu72 ssu73), YZS19/pDW11 <br />                                                                                        (SUA7 ssu72 ssu73), YZS19/pM385 YZS19/pM586 (sua7 SSU72 ssu73), and <br /> portant roles normal transcription initiation                          YZS19/pM681 (sua7 ssu72 SSU73) derivatives YZS19 carrying indi- <br /> detected biochemically. The products genes                          cated <span id='am-141' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-142' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-143' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span>. YZS19-1B sua7 ssu72 SSU73 segregant derived a <br /> functionally related TBP, Spt3p shown phys-                         backcross YZS19 YDW546. Strain YZS89A (SUA7 ssu72::LEU2 [URA3- <br /> ically interact TBP (13). Furthermore, human TAFII18                           SSU72]) used host plasmid shuffle derived from <br /> structurally similar N-terminal region Spt3p (42).                           YMH171 described below. LRB535/536 diploid strain used host for <br />                                                                                        disruption SSU72 (65). <br /> Another important example transcription factors                              All media prepared according standard procedures (59). YPD refers <br /> initially identified genetic means Srb proteins. Mu-                        rich medium consisting 1% yeast extract, 2% peptone, 2% glucose. <br /> tations SRB genes selected basis                           5-<span id='am-11' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-13'  ></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Foa" datatype="xsd:string"></span>FOA</span> medium contains 5-fluoro-orotic acid used positive selection <br /> ability suppress growth phenotypes associated trunca-                          URA31 gene (5). <br />                                                                                           Genetic methods nomenclature. Standard procedures used cross- <br /> tion heptapeptide repeat domain (CTD) located the <br />                                                                                        ing yeast strains, selecting diploids, inducing sporulation, dissecting tetrads <br /> C terminus largest subunit RNAP II (46). These                               (60). Yeast transformations lithium acetate procedure (23). <br /> factors components RNAP II holoenzyme in-                               Plasmid DNA recovered S. cerevisiae described previously (30) and <br /> clude novel proteins associated preinitiation complex                         introduced <span id='am-88' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-96' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-99'  ></span><span id='am-100' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span>Escherichia coli</span> electroporation (12). All transforma- <br /> (36, 68) kinase-cyclin pair involved <span id='am-146' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-147' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-148'  ></span><span id='am-149' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-150' property="rdfs:label" content="Phos" datatype="xsd:string"></span>phos</span>-                     tions E. coli calcium chloride procedure (40). E. coli <br />                                                                                        XL1-Blue (recA1 endA1 gyrA96 thi-1 supE44 hsdR17 (rK2 mK2) relA1 lac [F9 <br /> phorylation RNAP II CTD (39). The SPT, SUA,                                 proAB lacIq lacZDM15 Tn10 (Tetr)]) used host strain plasmids. <br /> SRB systems underscore importance genetic approaches                               Standard yeast genetic nomenclature used throughout, upper- and <br /> for identifying critical factors involved transcription initia-                     lowercase italicized symbols denoting dominant recessive alleles, respec- <br /> tion RNAP II.                                                                       tively (e.g., SSU72 ssu72-1). The symbols Ura2, Leu2, Trp2 denote <br />    We continuing use genetic methods identify factors                        failure grow media lacking Ura, Leu, Trp (2Ura, 2Leu, 2Trp <br />                                                                                        media, respectively). Cold- heat-sensitive growth phenotypes expressed <br /> that important transcription initiation RNAP II. One                        cs2 ts2, respectively, denote distinctly impaired growth (or no <br /> of approaches isolate characterize suppressors                        growth) rich medium (YPD) 16 378C. <br /> defects TFIIB (sua7) largest subunit RNAP II                              Plasmids. Restriction sites pertaining SSU72 constructs depicted Fig. <br /> (sua8) affect start site selection. The sua7-1 allele encodes                     2. The low-copy-number <span id='am-111' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-112' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-114' property="rdfs:label" content="vectors" datatype="xsd:string"></span>vectors</span> YCp50 (CEN URA3) (55), pRS314 (CEN TRP1) <br /> a TFIIB Glu-623Lys (E62K) replacement responsible                              pRS316 (CEN URA3) (62), YCplac33 (24); integrating vector <br />                                                                                        pRS305 (LEU2) (62); high-copy-number vector YEp24 (2 mm URA3) <br /> for aberrant initiation pattern cold-sensitive                          (55) described previously. The plasmids constructed follows. <br /> (cs2) growth defect (51). By selecting revertants                           (i) pM264 2.3-kb SalI-SphI fragment encompassing SUA7 high-copy- <br /> sua7-1 cs2 phenotype, uncovered genes desig-                             number vector YEp24. (ii) pM384 original SSU72 clone carrying a <br /> nated ssu71, ssu72 ssu73 (for suppressors sua7, genes 1,                        ;12.5-kb DNA insert vector YCp50. (iii) pM385 1.05-kb EcoRI- <br />                                                                                        HindIII SSU72 DNA fragment pRS316. (iv) pM471 3.8-kb HindIII- <br /> 2, 3). SSU71 identical TFG1, gene encoding                           HindIII SSU72 DNA fragment vector pRS316. (v) pM471A identical to <br /> largest subunit TFIIF (64), SSU73 identical RPB9,                         pM471, HindIII fragment inserted pRS316 the <br /> the gene encoding 14.2-kDa subunit RNAP II (66). The                            opposite orientation. (vi) pM523 derived pM539 replacement the <br /> ssu71 ssu73 suppressors functionally similar                           HindIII-BclI DNA fragment encompassing entire SSU72 open reading frame <br />                                                                                        LEU2 HindIII-BamHI DNA fragment. (vii) pM524 ssu72 DNA lacking <br /> both compensate cs2 phenotype                                sequence encoding N-terminal 23 amino acids amplified PCR <br /> restore normal initiation pattern sua7-1 background                         primers generate flanking EcoRI SpeI sites. Amplified DNA was <br /> (64, 66). Here, SSU72 gene. In contrast                         digested EcoRI SpeI ligated YCpIF17 (18). This construct <br /> ssu71 ssu73 suppressors, ssu72-1 enhancer                             expresses HA-Ssu72 protein, lacking N-terminal 23 residues Ssu72p, <br />                                                                                        GAL1 promoter. (viii) pM539 2.0-kb ApaI-BamHI SSU72 DNA <br /> sua7-1 defect, conferring synthetic heat-sensitive phenotype                         fragment pRS316. (ix) pM540 identical pM539, carries the <br /> (ts2) causing start site selection ADH1 CYC1                            ssu72-1 allele. pM540 recovered strain YZS19 gap repair using <br /> genes shift downstream normal sites. Genetic                         pM539 digested EcoRI HindIII. (x) pM550 identical to <br /> and molecular analyses demonstrated SSU72 essen-                            pM524, encodes full-length Ssu72p HA fusion protein behind <br />                                                                                        GAL1 promoter. (xi) pM552 2.7-kb XbaI-HindIII DNA fragment <br /> tial gene encoding novel protein affect assem-                        encompassing SSU72 ORF498 LEU2 integrating vector pRS305. (xii) <br /> bly transcription preinitiation complex.                                        pM586 1.05-kb EcoRI-HindIII SSU72 DNA fragment YCplac33. (xiii) <br />                                                                                        pM681 RPB9 (SSU73) gene YCplac33 vector. The 3.2-kb PstI- <br />                                                                                        BamHI fragment encompassing RPB9 transferred plasmid Ro406 (20) <br />                     MATERIALS AND METHODS                                              YCplac33. (xiv) pM699 includes entire SSU72 open reading frame was <br />                                                                                        amplified PCR oligonucleotides create flanking XmaI-NcoI sites and <br />    Yeast strains media. The yeast strains used study listed Table   inserted CUP1-glutathione S-transferase (GST) vector pCBGST-1 (41), <br /> 1. Strains T16 YDW546 described previously (50). YMH71-9C                allowing expression GST-Ssu72 fusion protein CUP1 control. (xv) <br />  VOL. 16, 1996                                                                                                           THE YEAST SSU72 GENE                     1559 <br />  <br />  <br /> pM712 1.05-kb EcoRI-KpnI DNA fragment encompassing SSU72 <br /> wild-type gene TRP1 vector pRS314. (xvi) pM740 identical pM699, <br /> except N-terminal 4 residues Ssu72p eliminated the <br /> GST-Ssu72 fusion protein restriction sites flanking SSU72 NotI and <br /> NcoI. (xvii) pM742 identical pM740, N-terminal 10 amino <br /> acids Ssu72p eliminated GST-Ssu72 fusion protein. (xviii) <br /> pDW11 SUA7 wild-type gene vector pRS316 (50). <br />    Isolation SSU72 gene. The SSU72 gene isolated YCp50 <br /> genomic library (56) complementation ts2 phenotype conferred the <br /> ssu72-1 allele strain YZS19. The relationship complementing <br /> DNA genetically defined ssu72 locus established integration of <br /> the cloned DNA genome strain YDW546 (sua7 SSU721 leu2) by <br /> homologous recombination (55). Plasmid pM552, carries 2.7-kb XbaI- <br /> HindIII DNA fragment encompassing SSU72, digested unique BamHI               FIG. 1. Conditional growth defects associated sua7, ssu72, ssu73 <br />                                                                                      mutations. Strain YZS19 (sua7 ssu72 ssu73) spontaneous cs1 derivative of <br /> site open reading frame adjacent SSU72. Linearized pM552 was <br />                                                                                      YMH71-9C (sua7 SSU72 SSU73). The strains depicted rows 2 5 were <br /> then introduced YDW546, transformants selected 2Leu me- <br />                                                                                      derived YZS19 introducing low-copy-number plasmids carrying either <br /> dium. This tags chromosomal locus cloned DNA LEU2 <br />                                                                                      SUA7, SSU72, SSU73 vector alone. Consequently, strains isogenic, <br /> marker, confirmed Southern blot analysis (<span id='am-131' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-132' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-134'  ></span><span id='am-135' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> shown). This <br />                                                                                      differing indicated alleles. Strain names follows: YMH71-9C <br /> strain crossed YZS19-1B (sua7 ssu72 leu2), resulting diploid      (sua7 SSU72 SSU73), YZS19/YCplac33 (sua7 ssu72 ssu73), YZS19/pDW11 <br /> sporulated dissected. Prior mating, SUA7 gene introduced         (SUA7 ssu72 ssu73), YZS19/pM586 (sua7 SSU72 ssu73), YZS19/pM681 <br /> YZS19-1B URA3 high-copy-number plasmid (pM264); presence             (sua7 ssu72 SSU73). These results demonstrate ssu72-1 enhancer the <br /> SUA7 wild-type gene improved sporulation germination efficiencies         sua7-1 allele, conferring severe ts2 phenotype (cf. row 5 row 1), whereas <br /> homozygous sua7 diploids. Segregants cured plasmid-borne SUA7            ssu73-1 suppressor cs2 phenotype associated sua7-1 (cf. row 4 and <br /> gene 5-FOA medium subsequently scored Leu ts phenotypes.              row 1). The phenotypes associated ssu72-1 ssu73-1 mutations are <br /> Complete cosegregation Leu1/ts1 Leu2/ts2 phenotypes confirmed             absolutely dependent sua7-1 allele, SUA7 plasmid rescues all <br /> that cloned gene SSU72.                                                       phenotypes (row 3). Columns represent cell growth serial dilutions (left to <br />    DNA sequence analysis. Plasmids pM471 pM471A linearized           right). The revertant colonies seen rows 2 5 378C indicative the <br /> SacI NotI sites multiple cloning region, nested sets dele-     high spontaneous reversion frequency ssu72-1 allele. <br /> tions generated exonuclease III S1 nuclease (Erase-a-Base Kit; <br /> Promega). Single-stranded DNA generated selected clones the <br /> VCSM13 helper phage (Stratagene Cloning Systems) presence kana- <br /> mycin, described previously (58). Both strands entire 3,785-bp DNA                                           RESULTS <br /> fragment sequenced dideoxy-terminator method M13 uni- <br /> versal primer; sequencing gaps filled SSU72-specific primers. The          Isolation ssu72 mutant. The sua7-1 allele encodes an <br /> entire open reading frame ssu72-1 allele sequenced single-           E62K replacement TFIIB confers altered start site <br /> stranded DNA derived plasmid pM540.                                             selection severe cs2 phenotype (51) (Fig. 1, row 1). Cold <br />    Construction ssu72::LEU2 null allele. Plasmid pM523 digested with <br /> ApaI BamHI, resulting ssu72::LEU2 DNA fragment introduced <br />                                                                                      sensitivity associated defects assembly mul- <br /> into diploid strain LRB535/536, selecting Leu1 transformants. Integration     tisubunit complexes (reviewed reference 73). This suggests <br /> one SSU72 loci, resulting replacement SSU72 gene           sua7-1 mutant defective assembly the <br /> LEU2 (ssu72::LEU2), confirmed Southern blot analysis (data shown).        transcription preinitiation complex. Consistent possi- <br />    Determination transcript start sites. Primer extension performed        bility, form TFIIB encoded sua7-1 functional and <br /> described previously, using total RNA ADH1-specific primer oIP-87 <br /> (50). Primer extension products resolved 6% polyacrylamide DNA se- <br />                                                                                      exhibits normal stability vivo (51). Consequently, suppres- <br /> quencing gels visualized Molecular Dynamics PhosphorImager. The           sors enhancers E62K form TFIIB include <br /> molecular size markers SUA7 DNA sequenced                factors facilitate impair complex assembly. <br /> primer oIP-133 (50).                                                                    One spontaneous cs1 revertants an <br />    Isolation ssu72-1 allele. The ssu72-1 allele cloned strain <br /> YZS19 gap repair (57). Plasmid pM539 digested EcoRI HindIII, <br />                                                                                      sua7-1 mutant (YMH71-9C) isolated. In effort to <br /> thereby deleting entire SSU72 open reading frame (see Fig. 2). The gapped        identify potential pleiotropic phenotypes, cs1 revertants <br /> vector purified following size fractionation agarose gel introduced    scored ts2. Three revertants severely <br /> into YZS19. Ura1 transformants selected screened retention       ts2, failing grow rich medium (YPD) 378C. Two of <br /> ts2 phenotype associated ssu72-1. Plasmid DNA recovered digested <br /> with EcoRI HindIII, confirming recovery ssu72 DNA. This plasmid was <br />                                                                                      strains, YZS14 YZS45, result recessive <br /> designated pM540.                                                                    mutations SSU71 (TFG1), gene encoding largest <br />    In vitro construction ssu72 alleles. Site-directed mutations SSU72     subunit TFIIF (64). Genetic analysis strain, <br /> generated synthetic oligonucleotides PCR according technique            YZS19, revealed cs1 ts2 phenotypes result <br /> described previously (43). Template DNA plasmid pM539 (SSU72) or <br /> pM540 (ssu72-1). Following amplification, DNA digested EcoRI <br />                                                                                      recessive mutations unlinked genes, designated <br /> (SSU72 site) KpnI (vector site) inserted TRP1 vector pRS314         ssu72-1 ssu73-1. The ssu73-1 mutation suppressed cs2 <br /> (62). Plasmids expressing N-terminal deletions Ssu72p (4-, 10-, 23-nucle-     phenotype conferred sua7-1 (66), ssu72-1 mu- <br /> otide deletions) constructed described above.                                tation combination sua7-1 responsible ts2 <br />    Functional analysis ssu72 alleles. A plasmid shuffle (61) set up <br /> to assess function vitro-generated ssu72 alleles. The chromosomal <br />                                                                                      phenotype (Fig. 1). <br /> SSU72 locus strain YMH171 (SUA71 SSU721) disrupted ApaI-                Isolation SSU72 gene. The ts2 phenotype strain <br /> BamHI DNA fragment encompassing ssu72::LEU2 plasmid pM523. Strain               YZS19 used clone SSU72 wild-type gene. A YCp50 <br /> viability maintained introducing plasmid pM586 (URA3 SSU72)              yeast genomic library (56) introduced YZS19, and <br /> YMH171 prior disruption. The resulting strain designated YZS89A (Table <br /> 1). Phenotypes associated vitro-constructed ssu72 alleles assessed by <br />                                                                                      transformants selected 378C. Twenty-three ts1 trans- <br /> introducing ssu72 TRP1 plasmids YZS89A subsequently curing Trp1             formants isolated total ;60,000 Ura1 transfor- <br /> transformants pM586 (URA3 SSU72) 5-FOA medium. Failure recover              mants. Plasmid DNA recovered 12 transformants <br /> 5-FOA-resistant (FOAr) transformants indicated ssu72 allele non-         reintroduced YZS19, confirming ts1 pheno- <br /> functional. FOAr strains subsequently scored growth rich medium <br /> (YPD). Plasmids pM712 (SSU72), pM699 (CUP1-GST-SSU72), pM550 <br />                                                                                      type strain conferred plasmid DNA than <br /> (GAL1-HA-SSU72) served positive controls experiments.                strain reversion. Restriction analysis identified distinct <br />    Sequence comparisons homologies. DNA protein sequence compar-             plasmids common 3.8-kb HindIII restriction fragment. <br /> isons performed BLAST algorithm (2), accessed National         One plasmids, designated pM384, chosen for <br /> Center Biotechnology Information electronic mail service. <br />    Nucleotide sequence accession number. The 3.8-kb DNA sequence encom- <br />                                                                                      analysis. <br /> passing SSU72 entered GenBank data library accession              Plasmid pM384 carries insert approximately 12.5 kb in <br /> number U20390.                                                                       vector YCp50. The gene responsible complementation <br />  1560      SUN AND HAMPSEY                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />                                                                                    We recently identified ssu73-1 suppressor allele of <br />                                                                                 RPB9 (66). Thus, genes represented YZS19 (SUA7, <br />                                                                                 SSU72, SSU73/RPB9) cloned. This enabled us <br />                                                                                 confirm genetically defined relationships these <br />                                                                                 genes introducing low-copy-number vectors carrying SUA7, <br />                                                                                 SSU72, SSU73 strain YZS19 scoring resulting <br />                                                                                 transformants growth 16 378C. The results shown <br />                                                                                 Fig. 1. Strain YZS19/pM586 (sua7 SSU72 ssu73) cs1 <br />                                                                                 ts1 (row 4), confirming ssu73-1 suppresses the <br />                                                                                 sua7-1 cs2 phenotype does confer ts2. In contrast, <br />                                                                                 YZS19/pM681 (sua7 ssu72 SSU73) exhibits cs2 phe- <br />                                                                                 notype parent strain YMH71-9C (sua7 SSU72 SSU73) <br />                                                                                 (cf. rows 1 5), confirming ssu72-1 suppressor <br />                                                                                 sua7-1. However, ts2 phenotype YZS19 retained in <br />                                                                                 presence SSU73 gene (cf. rows 2 5) is <br />                                                                                 eliminated SSU72 SUA7 (rows 3 4), confirming <br />                                                                                 dependence ts2 sua7-1 ssu72-1. These results <br />                                                                                 firmly establish ssu72-1 enhancer ssu73-1 a <br />                                                                                 suppressor TFIIB defect. <br />    FIG. 2. Restriction map SSU72 region. A partial restriction map en-      The SSU72 sequence. The entire 3.8-kb HindIII fragment <br /> compassing SSU72 gene shown. DNA fragments complement ts2       encompassing SSU72 sequenced. Three open reading <br /> phenotype associated ssu72-1 suppressor sua7-1 background are <br /> indicated black bars, open bars denote failure complement. The <br />                                                                                 frames 618, 1,482, 894 nucleotides identified (Fig. <br /> 3.8-kb HindIII-HindIII 1.05-kb EcoRI-HindIII DNA fragments vector    2). The SSU72 gene subsequently delimited 1.05-kb <br /> pRS316 (62) designated pM471 pM385, respectively. The adjacent open     EcoRI-HindIII DNA fragment (pM385) includes only <br /> reading frames encode potential proteins 494 298 amino acids (<span id='am-115' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-116' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-118' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-119'  ></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span>aa</span>)   618-nucleotide open reading frame (Fig. 3). Accordingly, <br /> unknown functions. <br />                                                                                 SSU72 encodes potential protein 206 amino acids a <br />                                                                                 calculated molecular mass 23,469 Da. Ssu72p acidic (pI 5 <br />                                                                                 4.9) enriched charged residues (27% D, E, K, R). <br /> of ts2 initially delimited common 3.8-kb HindIII                     The N terminus includes half-site C-X2-C potential metal <br /> DNA fragment (plasmid pM471) (Fig. 2). Southern blot anal-                      binding motif (see below). Two imperfect direct repeats were <br /> ysis confirmed fragment derived intact                   identified (residues 69 78 82 90), suggesting an <br /> yeast genome SSU72 single-copy gene (data                     internal element symmetry Ssu72p. These structural <br /> shown). The relationship complementing DNA                          features summarized Fig. 4. No structural features com- <br /> and genetically defined ssu72 gene established in-                   monly associated transcription factors found, al- <br /> tegration cloned DNA LEU2 vector                   clustering acidic residues C-terminal <br /> genome YDW546. This strain (sua7 leu2 SSU72::LEU2)                       half protein. <br /> crossed strain YZS19-1B (sua7 leu2 ssu72), resulting                    The Ssu72p sequence compared protein se- <br /> diploid sporulated dissected. A total 18 segregants                  quence databases using BLAST algorithm (2). Although <br /> were obtained scored. All Leu1 progeny phenotyp-                       homologous proteins found, similarity conserved <br /> ically ts1, Leu2 progeny ts2. Segregation                   region ATP-dependent RNA helicases identified. The <br /> LEU2-tagged locus opposite ssu72-1 marker (ts2) estab-                   similarity Ssu72p consensus residues within <br /> lished cloned gene SSU72.                                    region especially good (Fig. 4). Nonetheless, the <br />    Genetic relationship SUA7 SSU72. Indepen-                        Ssu72p sequence includes DEAD box ATP- <br /> dent segregation cs1 (ssu73) ts2 (ssu72) phenotypes                  binding proteins highly conserved HRIGRXXR motif <br /> from YZS19 raises question SSU72 function-                    required RNA binding ATP hydrolysis (49). Conse- <br /> ally related SUA7. As described above, SSU72 cloned                   quently, Ssu72p unlikely ATP-dependent RNA <br /> complementation ts2 phenotype. In addition, plasmid                      helicase. <br /> DNA 9 12 ts1 transformants YZS19 exhibited                            SSU72 essential gene. The SSU72 locus disrupted <br /> HindIII restriction pattern characteristic SUA7                  replacing HindIII-BclI fragment encompassing en- <br /> gene. Plasmid pDW11, carries SUA7 low-copy-                          tire SSU72 open reading frame LEU2 HindIII-BamHI <br /> number plasmid (50), subsequently transform                        DNA fragment (Fig. 2). The resulting construct used to <br /> YZS19 ts1. The ability SUA7 SSU72                       disrupt copy SSU72 homozygous leu2 diploid <br /> wild-type gene restore ts1 demonstrates ts2 phe-                    strain LRB535/536. Upon sporulation dissection, more <br /> notype associated ssu72-1 dependent sua7-1.                        viable spores recovered tetrad (Fig. <br /> Therefore, ts2 phenotype YZS19 simply conse-                    5) viable spores phenotypically Leu2. Inspection <br /> quence ssu72-1 mutation &#8216;&#8216;synthetic&#8217;&#8217; phenotype                 inviable segregants microscope revealed most <br /> requiring sua7-1 ssu72-1. This effect reminiscent                germinated, undergoing cell divisions before <br /> the ssu71-1 suppressor, similarly dependent                   cessation growth. Four-spore viability restored the <br /> sua7-1 mutation associated ts2 phenotype (64). As                   diploid strain transformed plasmid pM385 (SSU72 <br /> SSU71, extremely important result dem-                    URA3) prior dissection. When resulting segregants were <br /> onstrates genetic relationship TFIIB prod-                    replica printed 5-FOA medium, selecting plas- <br /> uct SSU72 gene. Thus, strain YZS19 contains                       mid-borne copy SSU72, segregants, which <br /> suppressor (ssu73-1) enhancer (ssu72-1)                      Leu2, grew (data shown). Therefore, SSU72 essen- <br /> mutation (sua7-1). These mutations single                   tial cell growth. <br /> strain consequence selecting phenotype (cs1)                       The ssu72-1 mutation affects transcription start site selec- <br /> and screening (ts2).                                              tion. Since sua7-1 mutation shifts transcription start site se- <br />  VOL. 16, 1996                                                                                                        THE YEAST SSU72 GENE                     1561 <br />  <br />  <br />  <br />  <br />                                                                                       FIG. 4. Structural features Ssu72p. The essential N terminus (residues 5 to <br />                                                                                    23) denoted shaded box includes sequence C-X2-C, is <br />                                                                                    duplicated ssu72-1 mutant. The imperfect repeats (Fig. 3) denoted by <br />                                                                                    arrows. The black box (residues 122 150) denotes sequence similarity known <br />                                                                                    putative ATP-dependent RNA helicases. This region aligned eIF-4A <br />                                                                                    mice; Tif1 Tif2, putative counterparts eIF-4A yeasts; <br />                                                                                    ME31B, maternally expressed Drosophila gene; Dhh1p, Ste13p, Drs1p, <br />                                                                                    putative helicases S. cerevisiae S. pombe; RHLB, putative helicase <br />                                                                                    E. coli. Sc, Mm, Dm, Sp, Ec denote S. cerevisiae, <span id='am-1' about='obo:NCBITaxon_10090' typeof='owl:Thing obo:NCBITaxon_314147 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_337687 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_862507 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_39107 obo:NCBITaxon_10088 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_10066 obo:NCBITaxon_6072 obo:NCBITaxon_33553 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9989 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasAlternativeId" content="NCBITaxon:85055" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Mus musculus" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="mice C57BL/6xCBA/CaJ hybrid" datatype="xsd:string"></span><span id='am-6'  ></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Mus muscaris" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="house mouse" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="mouse" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Mus musculus</span>, <br />                                                                                    <span id='am-63' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-64' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-69'  ></span>Drosophila melanogaster</span>, S. pombe, E. coli, respectively. Residues are <br />                                                                                    identical Ssu72p highlighted boldface type, conserved residues are <br />                                                                                    highlighted italics. Arrowheads denote residues identical the <br />                                                                                    eukaryotic proteins (excludes E. coli RHLB). <br />  <br />  <br />  <br />  <br />                                                                                    initiation ADH1 (Fig. 6, lane 4) pattern <br />                                                                                    associated sua7-1 defect (Fig. 6, lane 1). Since strain <br />                                                                                    YZS19 mutated ssu73 locus, normal initiation <br />    FIG. 3. The SSU72 sequence. The complete nucleotide sequence             pattern SSU72 transformant suggests ssu73-1 com- <br /> 1,058-bp EcoRI-HindIII DNA fragment encompassing SSU72 shown,             pensates downstream shift caused sua7-1, result <br /> with deduced amino acid sequence Ssu72p. The EcoRI site (2233)           observed previously ssu71-1 suppressor (64). <br /> located 6 nucleotides downstream termination <span id='am-75' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77'  ></span><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> ORF494 (Fig. 2).      Furthermore, ssu71-1 ssu73-1, ssu72-1, sup- <br /> The 6-bp direct repeats flank duplicated sequence ssu72-1 mutant <br /> (nucleotides 25 54), corresponding amino acid sequence duplicated   press sua7-1 cs2 phenotype. Therefore, ssu71 ssu73 <br /> (residues 9 18), region sequence similarity ATP-dependent RNA     suppressors appear functionally related and <br /> helicases (residues 122 150) underlined. The imperfect amino acid repeats   distinct ssu72-1 enhancer. <br /> (residues 69 78 82 90) indicated arrows, identical residues      Characterization ssu72-1 mutation. The ssu72-1 allele <br /> are indicated boldface type. <br />                                                                                    cloned gap repair (57), entire open reading <br />                                                                                    frame sequenced. A 30-bp duplication nucleotides 25 to <br />                                                                                    54 identified mutation (Fig. 7). The ssu72-1 <br /> lection downstream normal sites (50), asked                      duplication maintains open reading frame creates a <br /> ssu72-1 affect process. Transcription start sites                  10-amino-acid direct repeat (or partially overlapping 12-ami- <br /> were mapped ADH1 locus primer extension (Fig. 6).                        no-acid repeat) (Fig. 8). Interestingly, repeat includes the <br /> Transcription ADH1 gene normally initiates sites                     amino acid sequence C-T-V-C Cys4 zinc binding <br /> located positions 227 237 relative ATG start                         motif (C-X2-C-X6-C-X2-C) created near N terminus of <br /> codon (Fig. 6, lane 2). Consistent previous results (50), <br /> the sua7-1 mutation diminished initiation position 237 and <br /> enhanced initiation multiple sites located downstream of <br /> 227 (Fig. 6, lane 1). Strikingly, strain YZS19, initiation was <br /> nearly eliminated 237 reduced 227 dramat- <br /> ically enhanced multiple sites located downstream <br /> (Fig. 6, lane 3). This effect dependent sua7-1 <br /> and ssu72-1 mutations, normal initiation pattern was <br /> restored introducing low-copy-number plasmid carrying <br /> either SSU72 (Fig. 6, lane 4) SUA7 (lane 5) wild-type <br /> gene YZS19. A similar enhanced downstream shift was <br /> observed CYC1 (data shown), establishing ssu72 <br />                                                                                       FIG. 5. SSU72 essential gene. One copy SSU72 gene dis- <br /> effect start site selection specific ADH1 pro-                   rupted LEU2 marker diploid strain LRB535/536 (ssu72::LEU2/ <br /> moter. These results demonstrate Ssu72p involved                        SSU72 leu2/leu2). Upon sporulation dissection, viable <br /> start site selection TFIIB-dependent manner provide                       spores recovered tetrad (9 25 tetrads depicted) all <br /> further evidence functional relationship                       viable spores phenotypically Leu2. Four-spore viability restored when <br />                                                                                    dissections presence plasmid-borne SSU72 gene (data <br /> two proteins.                                                                      shown). When resulting segregants replica printed 5-FOA <br />    The plasmid-borne SSU72 gene strain YZS19/pM385                              medium, selecting plasmid-borne SSU72, segregants, of <br /> (sua7-1 ssu72-1/SSU72 ssu73-1) restored normal pattern                      Leu2, grew. These results establish SSU72 essential gene. <br />  1562        SUN AND HAMPSEY                                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br />                                                                                         strain carries SUA7 wild-type gene, allowing for <br />                                                                                         phenotypes attributed exclusively ssu72 mutations. The <br />                                                                                         results shown Table 2. Deletion N-terminal 4 <br />                                                                                         residues effect, deletion 10 23 N-terminal <br />                                                                                         residues abolished Ssu72p function. Therefore, N terminus <br />                                                                                         Ssu72p, includes duplicated sequence encoded by <br />                                                                                         ssu72-1, critical function. <br />                                                                                            We asked cysteine position 15 critical. Two <br />                                                                                         mutations, encoding serine replacement (C15S) the <br />                                                                                         encoding histidine replacement (C15H), created. <br />                                                                                         The C15S mutant inviable, C15H mutant grew <br />                                                                                         nearly wild-type strain. The inviability the <br />                                                                                         C15S mutant establishes amino acid 15 critical, the <br />                                                                                         viability C15H mutant suggests C-X2-C se- <br />                                                                                         quence involved zinc chelation. We generated <br />                                                                                         comparable replacements cysteine position 25 en- <br />                                                                                         coded ssu72-1 (Fig. 8). Despite retention normal <br />                                                                                         C-15, C25S mutant inviable, C25H re- <br />                                                                                         placement effect. These data establish cys- <br />                                                                                         teine residues located near N terminus Ssu72p and <br />                                                                                         ssu72-1-encoded protein essential function and <br />                                                                                         suggest residues involved zinc chelation. <br />  <br />  <br />  <br />  <br />    FIG. 6. Primer extension analysis ADH1 transcription start sites. Lane 1, <br /> YMH71-9C (sua7-1); lane 2, T16 (wild type); lane 3, YZS19 (sua7-1 ssu72-1 <br /> ssu73-1); lane 4, YZS19/pM385 (sua7-1 SSU72 ssu73-1); lane 5, YZS19/pDW11 <br /> (SUA7 ssu72-1 ssu73-1). Lanes A, C, G, T used molecular size <br /> markers correspond sequence ladder SUA7 DNA. Transcription <br /> initiates comparable rates positions 237 227 (A ATG start <br /> codon designated 11) wild-type strain (lane 2). The sua7-1 mutation <br /> causes downstream shift start site selection (lane 1), effect is <br /> exacerbated sua7-1 ssu72-1 ssu73-1 mutant, normal initiation <br /> sites nearly eliminated favor multiple, discrete downstream sites (lane <br /> 3). The normal initiation pattern restored presence SSU72 <br /> (lane 4) SUA7 (lane 5) gene, demonstrating downstream shift in <br /> YZS19 dependent sua7-1 ssu72-1 mutations. The normal <br /> initiation pattern observed sua7-1 SSU72 ssu73-1 strain, indicating that <br /> ssu73-1 compensates downstream shift associated sua7-1 (cf. lanes 1 <br /> and 4). This effect similar ssu71-1 suppressor (64) stark <br /> contrast effect ssu72-1. <br />  <br />  <br />  <br /> the altered form Ssu72p. This intriguing result, because <br /> this motif implicated protein-DNA protein- <br /> protein interactions (53). Moreover, factors shown <br /> previously affect start site selection S. cerevisiae (TFIIB <br /> and Rpb1p Rpb9p subunits RNAP II) contain <br /> C-X2-C-X6&#8211;17-C-X2-C/H zinc binding motifs.                                                 FIG. 7. DNA sequence ssu72-1 allele. Partial DNA sequences the <br />                                                                                         SSU72 ssu72-1 alleles shown, cases CGCAAT (base pairs <br />   We investigated importance Ssu72p N terminus                            13 18) AACAAT (base pairs 55 60) (Fig. 3). The brackets encompass the <br /> generating specific ssu72 mutations vitro assessing                        region SSU72 duplicated create ssu72-1 allele includes the <br /> phenotypes using plasmid-shuffle vivo. The host                          TCAAAC sequence flanks reiterated sequence. <br />  VOL. 16, 1996                                                                                                                              THE YEAST SSU72 GENE               1563 <br />  <br />  <br />  <br />  <br />    FIG. 8. The ssu72-1 allele encodes direct repeat near N terminus Ssu72p. The ssu72-1 allele result 30-bp tandem duplication (Fig. 7). The 6-bp <br /> direct repeats flank duplication boxed. The 10-amino-acid direct repeats resulting ssu72-1 duplication indicated arrows. The cysteine <br /> residues repeated sequence form zinc binding motif (C-X2-C-X6-C-X2-C). <br />  <br />  <br />  <br />                                         DISCUSSION                                                            basis ts2 phenotype. Although suggests that <br />                                                                                                               SSU72 unrelated SUA7, results indicate oth- <br />    Functional relationship Ssu72p TFIIB. The SSU71,                                                     erwise. First, ssu72-1 ts2 phenotype requires presence <br /> SSU72, SSU73 genes uncovered genetic selection                                                  sua7-1 allele; SUA7 ssu72-1 sua7-1 SSU72 <br /> for suppressors conditional growth defect (cs2) associated                                               single mutants ts2. This effect defined synthetic en- <br /> with TFIIB mutation (sua7-1) alters transcription                                                 hancement indicates genetic relationship the <br /> start site selection. The potential selection identify                                             genes (28). Second, transcription start site selection is <br /> genes products functionally related TFIIB                                                    shifted significantly downstream normal sites the <br /> established identity SSU71 gene. SSU71 encodes                                                  YZS19 mutant, effect rescued SUA7 <br /> a homolog mammalian RAP74 identical TFG1,                                                    SSU72 (Fig. 6, cf. lane 3 lanes 4 5). These results <br /> gene encoding largest subunit yeast TFIIF (64). The                                                    demonstrate Ssu72p affect transcription initiation a <br /> ssu71-1 suppressor compensated transcription ini-                                                TFIIB-dependent manner vivo provide strong support <br /> tiation defect associated sua7-1, restoring normal pat- <br />                                                                                                               functional relationship proteins. None- <br /> tern start site selection CYC1 gene. The SSU73 gene <br />                                                                                                               theless, unable detect direct interaction <br /> is identical RPB9, encodes 14.2-kDa subunit of <br />                                                                                                               proteins two-<span id='am-136' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-137'  ></span><span id='am-138' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-139' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>hybrid</span> or <br /> RNAP II (66). Like ssu71-1, ssu73-1 compensates cs2 <br />                                                                                                               association TFIIB GST-Ssu72p HA-Ssu72p <br /> growth defect sua7-1 (Fig. 1) restores normal pat- <br /> tern transcription start site selection ADH1 (Fig. 6, lane                                              fusion proteins (63). Therefore, know the <br /> 4). These results imply functional relationship                                                 functional relationship TFIIB Ssu72p involves a <br /> largest subunit TFIIF Rpb9p subunit RNAP II.                                                    physical interaction. <br />    The ssu72-1 allele, contrast ssu71 ssu73,                                                  The primary structure Ssu72p does offer clue its <br /> suppressor sua7-1 cs2 phenotype. Rather, ssu72-1 arose                                                 function. The significant sequence similarity several <br /> spontaneously ssu73-1 strain YZS19 identified                                                 known putative ATP-dependent RNA helicases. Although <br />                                                                                                               similarity appears significant (Fig. 4), limited to <br />                                                                                                               small region protein. The absence motifs that <br />          TABLE 2. Viability associated Ssu72p alterations                                                characteristic ATP-dependent RNA helicases implies <br />                                                                                                               Ssu72p RNA helicase. Rather, sequence <br /> Ssu72p alteration                                                                             Viabilitya <br />                                                                                                               similarity suggests Ssu72p ATP-dependent RNA he- <br />                  b <br />  N-term-D4 ...................................................................................Viable          licases share common, albeit undefined, function. <br />  N-term-D10b .................................................................................Inviable           The ssu72-1 mutation. The ssu72-1 allele result a <br />  N-term-D23b .................................................................................Inviable        30-bp duplication flanked 6-bp direct repeats (Fig. 8). Con- <br />  C15Sc ............................................................................................Inviable   sequently, ssu72-1 encodes 10-amino-acid repeat creates <br />  C15Hc ...........................................................................................Viable <br />  C25Sd ............................................................................................Inviable   C-X2-C-X6-C-X2-C zinc binding motif (Cys4) near N <br />  C25Hd ...........................................................................................Viable      terminus Ssu72p (Fig. 8). This result particularly intrigu- <br />   a <br />                                                                                                               ing. The largest subunit RNAP II contains similar zinc <br />     Strains expressing indicated Ssu72p alterations constructed plas-                             binding motif (C-X2-C-X6-C-X2-H) (1). This structure lo- <br /> mid shuffling described Materials Methods. Viability assessed by <br /> the ability Trp1 transformants YZS89A grow 5-FOA medium. FOAr                                     cated N terminal amino acid replacements encoded by <br /> colonies purified subcloning subsequently restreaked rich me-                                  sua8 alleles RPB1, exert effects start site se- <br /> dium (YPD) compare growth rates control strains expressing                                   lection similar sua7-1 (4). TFIIB includes zinc <br /> SSU721. Growth rates viable mutants wild-type control strains were <br /> comparable. <br />                                                                                                               binding motif (C-X2-C-X17-C-X2-C) near N terminus, and <br />   b <br />     The indicated alterations denote, respectively, deletion N-terminal 4,                             sua7-1, sua7-2, sua7-3 alleles, shift tran- <br /> 10, 23 residues Ssu72p. These proteins expressed plasmids                                    scription start site selection downstream normal sites, <br /> pM740 (D4) pM742 (D10) GST-Ssu72 fusion proteins plasmid                                        encode single amino acid replacements highly conserved <br /> pM524 (D23) HA-Ssu72 fusion protein. Cells expressing full-length GST- <br /> Ssu72 (pM699) HA-Ssu72 (pM550) fusion proteins viable, demonstrat- <br />                                                                                                               domain immediately adjacent motif (51). The Rpb9p <br /> ing fusion proteins functional.                                                                  subunit RNAP II includes zinc binding motifs, near <br />   c <br />     Amino acid replacements denoted single letter code. Accordingly,                               N terminus near C terminus (74). Like <br /> C15S C15H designate, respectively, cysteine3serine cysteine3histidine                                 sua7 (TFIIB), sua8 (Rpb1p), ssu72, alleles RPB9 also <br /> replacements position 15. <br />   d <br />     Amino acid replacements generated using ssu72-1 allele the <br />                                                                                                               affect start site selection. In cases, start site selection was <br /> template. Therefore, indicated replacements C-25 occur 10-                                  shifted upstream normal site attributed an <br /> residue duplication encoded ssu72-1 (Fig. 8).                                                              rpb9 null allele replacement cysteine residue within <br />  1564     SUN AND HAMPSEY                                                                                                MOL. CELL. BIOL. <br />  <br />  <br /> the N-terminal zinc binding motif (20, 32) nonsense            In addition, effects sua7 mutations start site selection <br /> mutation immediately following C-terminal zinc binding             CYC1 ADH1 genes essentially identical those <br /> motif (66). All results point role zinc binding      sua8 mutations, genetic evidence suggests TFIIB <br /> domains transcription start site selection, spe-       interacts directly largest subunit RNAP II (4). <br /> cific functions zinc binding regions remain undefined.          Clearly, RNAP II TFIIB critical determinants start <br />    The C-X2-C-X(6&#8211;25)-C-X2-C motif             site selection, presumably consequence specific inter- <br /> transcription factors, including second largest subunit         actions Rpb1p, Rpb9p, TFIIB. <br /> RNAP II (67), TFIIE (16), elongation factor TFIIS (53),               What role does Ssu72p play start site selection? It is <br /> the adenovirus E1A protein (22). A three-dimensional nuclear           important recognize data indicate that <br /> magnetic resonance structure revealed zinc binding            SSU72 necessarily plays direct role start site selection. <br /> domain TFIIS does form classical zinc finger motif.         Rather, ssu72 mutation enhances TFIIB defect; the <br /> Rather, TFIIS forms three-stranded, antiparallel b-sheet,            ssu72-1 allele does affect start site selection SUA7 <br /> designated zinc ribbon (53). It suggested motif        wild-type background (Fig. 6, lane 5). This result similar to <br /> is inaccessible (cryptic) native protein ex-        effect ssu71 suppressors. Mutations SSU71 restore <br /> posed binding TFIIS elongation complex. This          normal pattern start site selection sua7-1 back- <br /> an interesting possibility reminiscent proposal          ground (64) little effect SUA71 back- <br /> that TFIIB exists latent form N-terminal             ground (63). Among ssu71, ssu72, ssu73 alleles, only <br /> domain forms intramolecular interaction C-termi-           ssu73-1 (RPB9) influences start site selection SUA71 <br /> nal domain (54). Upon binding transcriptional activator, <br />                                                                        background (66). Therefore, findings consistent with <br /> the zinc binding motif TFIIB N terminus exposed, <br />                                                                        conclusion start site selection determined solely by <br /> providing surface interaction RNAP II-TFIIF. The <br />                                                                        interactions TFIIB RNAP II. <br /> zinc binding motif critical function, cysteine <br />                                                                           A possible role Ssu72p assembly transcription <br /> replacements eliminated ability TFIIB recruit RNAP <br /> II-TFIIF preinitiation complex (8).                           preinitiation complex. Ssu72p facilitate assembly the <br />    It noted ssu72-1-encoded protein,        transcription preinitiation complex. This <span id='am-50' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Idea" datatype="xsd:string"></span>idea</span> based the <br /> the normal Ssu72p, includes C-X2-C-X6-C-X2-C zinc bind-            following rationale. Firstly, E62K form TFIIB encoded <br /> ing motif. Nonetheless, cysteine 15 essential N terminus        sua7-1 impair assembly TFIIB preini- <br /> of Ssu72p functionally replaced histidine         tiation complex. This supported severe cs2 phenotype <br /> serine, suggesting C-X2-C sequence involved zinc        sua7-1 mutants, phenotype associated defects in <br /> chelation. There precedent possibility. The Rpb10p       assembly multisubunit complexes (reviewed reference <br /> subunit RNAP II contains similar half-site C-X2-C se-             73), observation TFIIB encoded sua7-1 is <br /> quence (75) known bind zinc (69). If N terminus          functional stable vivo (51). Consequently, suppres- <br /> of Ssu72p does bind zinc, amino acids               sors (ssu71 ssu73) enhancers (ssu72) sua7-1 <br /> cysteine constitute zinc ligands. Alternatively,         mutation include factors affect assembly the <br /> Ssu72p exist dimer creates zinc binding              preinitiation complex. Secondly, ssu71 suppressors of <br /> domain trans, proposed human immunodeficiency            sua7-1 encode altered forms largest subunit TFIIF <br /> virus Tat protein (19).                                                restore normal pattern transcription initiation the <br />    There appear discrepancy struc-           sua7-1 mutant (64). The Ssu71p subunit TFIIF homolo- <br /> ture ssu72-1-encoded protein function. The              gous mammalian RAP74. Although RAP30 subunit of <br /> ssu72-1 mutation creates, eliminates, structural         TFIIF sufficient initiate transcription (9) recruit <br /> motif (Cys4) altered protein         RNAP II preinitiation complex (17, 34, 70), RAP74 <br /> expected dominant, gain-of-function mutation. Yet              increase efficiency complex assembly (34, 70). <br /> ssu72-1 exacerbates TFIIB defect, causing start site selec-        These results suggest RAP74 function regulating <br /> tion shift downstream normal site creating       assembly preinitiation complex (10). Since Ssu71p is <br /> a synthetic ts2 phenotype. Furthermore, ssu72-1 fully reces-        homologous RAP74 (64), Ssu71p function in <br /> sive respect SSU721. This anomaly rationalized,         preinitiation complex assembly. Thirdly, Ssu72p includes a <br /> however, 10-amino-acid duplication encoded ssu72-1           half-site C-X2-C zinc binding motif critical function <br /> impairs protein stability. This case,            duplicated ssu72-1 mutant. Zinc binding domains <br /> SUA71 ssu72-1 mutants mildly ts2, albeit lesser          RNAP II proposed stabilize subunit-sub- <br /> extent sua7-1 ssu72-1 double mutants (63) (Fig. 1, rows 2         unit interactions, facilitating assembly enzyme <br /> and 3). The Cys4 zinc binding motif retain               complex (69; reviewed reference 3). Furthermore, zinc <br /> facilitate protein-protein interactions 10-res-        binding motif TFIIB necessary recruit RNAP II into <br /> idue duplication destabilizes protein. This explain          preinitiation complex (8), E62K defect (sua7-1) is <br /> why cysteine 25, component normal            located just downstream motif (51). Finally, third <br /> protein duplication encoded ssu72-1,          effector sua7-1 uncovered studies, ssu73-1, encodes an <br /> functionally critical (Table 2).                                       altered form Rpb9p. This RNAP II subunit includes two <br />    Start site selection yeasts. In yeast vitro transcription   zinc binding motifs proposed affect as- <br /> system, replacement RNAP II TFIIB S.                  sembly RNAP II complex (3). Taken together, these <br /> cerevisiae functional counterparts Schizosac-          results suggest accurate start site selection involves spe- <br /> charomyces pombe sufficient change initiation pattern        cific interactions RNAP II TFIIB these <br /> from S. cerevisiae pattern characteristic S.         interactions mediated, part, zinc binding <br /> pombe (38). On basis result, Li et al. (38) concluded      domains. The half-site zinc binding domain Ssu72p might <br /> that RNAP II TFIIB sole determinants tran-              facilitate interactions, component the <br /> scription start site selection yeasts. This premise consistent   preinitiation complex gone undetected biochemically <br /> with effects altered forms TFIIB (sua7), Rpb1p               chaperone-like protein facilitates complex assem- <br /> (sua8), Rpb9p (ssu73) start site selection (4, 20, 32, 50).     bly. <br />  VOL. 16, 1996                                                                                                           THE YEAST SSU72 GENE                       1565 <br />  <br />  <br />                           ACKNOWLEDGMENTS                                                eukaryotic transcription. Curr. Opin. Cell Biol. 6:403&#8211;409. <br />                                                                                      27. Goodrich, J. A., R. Tjian. 1994. Transcription factors IIE IIH and <br />   We grateful David Gross valuable discussions                     ATP hydrolysis direct promoter clearance RNA polymerase II. Cell 77: <br /> course work thoughtful comments manuscript.                       145&#8211;156. <br /> We thank Rolf Furter RPB9 clone.                                        28. Guarente, L. 1993. Synthetic enhancement gene interaction: genetic tool <br />   This work supported grants National Institutes                      come age. Trends Genet. 9:362&#8211;366. <br /> Health (GM-39484) American <span id='am-55' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57'  ></span><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="Cancer" datatype="xsd:string"></span>Cancer</span> Society (NP-842).                          29. Hahn, S., S. Buratowski, P. A. Sharp, L. Guarente. 1989. Isolation the <br />                                                                                          gene encoding yeast TATA binding protein TFIID: gene identical to <br />                                   REFERENCES                                             SPT15 suppressor Ty element insertions. Cell 58:1173&#8211;1181. <br />  1. Allison, L. A., M. Moyle, M. Shales, C. J. Ingles. 1985. Extensive           30. Hoffman, C., F. Winston. 1987. A ten-minute DNA preparation from <br />     homology largest subunits eukaryotic prokaryotic RNA                yeast efficiently releases autonomous plasmids transformation Esche- <br />     polymerases. Cell 42:599&#8211;610.                                                        <span id='am-121' about='obo:NCBITaxon_1430853' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_95179 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_7100 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-122' property="rdfs:label" content="Richia" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-125'  ></span>richia</span> coli. Gene 57:267&#8211;272. <br />  2. Altschul, S. F., W. Gish, W. Miller, E. W. Myers, D. J. Lipman. 1990.        31. Horiuchi, J., N. Silverman, G. A. Marcus, L. Guarente. 1995. ADA3, a <br />     Basic local alignment search tool. J. Mol. Biol. 215:403&#8211;410.                        putative transcriptional adaptor, consists separable domains in- <br />  3. Archambault, J., J. D. Friesen. 1993. Genetics eukaryotic RNA                 teracts ADA2 GCN5 trimeric complex. Mol. Cell. Biol. 15: <br />     polymerases I, II, III. Microbiol. Rev. 57:703&#8211;724.                              1203&#8211;1209. <br />  4. Berroteran, R. W., D. E. Ware, M. Hampsey. 1994. The sua8 suppressors        32. Hull, M. W., K. Mckune, N. A. Woychik. 1995. RNA polymerase II <br />     Saccharomyces cerevisiae encode replacements conserved residues                subunit RPB9 required accurate start site selection. Genes Dev. 9: <br />     largest subunit RNA polymerase II affect transcription start       481&#8211;490. <br />     site selection similarly sua7 (TFIIB) mutations. Mol. Cell. Biol. 14:226&#8211;     33. Johnson, P. F., S. L. McKnight. 1989. Eukaryotic transcriptional regu- <br />     237.                                                                                 latory proteins. Annu. Rev. Biochem. 58:799&#8211;839. <br />  5. Boeke, J. D., F. Lacroute, G. R. Fink. 1984. A positive selection        34. Killeen, M., B. Coulombe, J. Greenblatt. 1992. Recombinant TBP, <br />     mutants lacking orotidine-59-phosphate decarboxylase activity yeast:              transcription factor IIB, RAP30 sufficient promoter recognition <br />     5-fluoro-orotic acid resistance. Mol. Gen. Genet. 197:345&#8211;346.                       mammalian RNA polymerase II. J. Biol. Chem. 267:9463&#8211;9466. <br />  6. Buratowski, S. 1994. The basics basal transcription RNA polymerase II.     35. Kim, Y.-J., S. Bjorklund, Y. Li, M. H. Sayre, R. D. Kornberg. 1994. A <br />     Cell 77:1&#8211;3.                                                                         multiprotein mediator transcriptional activation interaction with <br />  7. Buratowski, S., S. Hahn, L. Guarente, P. A. Sharp. 1989. Five interme-           C-terminal repeat domain RNA polymerase II. Cell 77:599&#8211;608. <br />     diate complexes transcription initiation RNA polymerase II. Cell 56:       36. Koleske, A. J., S. Buratowski, M. Nonet, R. A. Young. 1992. A novel <br />     549&#8211;561.                                                                             transcription factor reveals functional link RNA polymerase II <br />  8. Buratowski, S., H. Zhou. 1993. Functional domains transcription               CTD TFIID. Cell 69:883&#8211;894. <br />     factor TFIIB. Proc. Natl. Acad. Sci. USA 90:5633&#8211;5637.                           37. Koleske, A. J., R. A. Young. 1994. An RNA polymerase II holoenzyme <br />  9. Chang, C., C. F. Kostrub, Z. F. Burton. 1993. RAP30/74 (transcription            responsive activators. Nature (London) 368:466&#8211;469. <br />     factor IIF) required promoter escape RNA polymerase II. J. Biol.       38. Li, Y., P. M. Flanagan, H. Tschochner, R. D. Kornberg. 1994. RNA <br />     Chem. 268:20482&#8211;20489.                                                               polymerase II initiation factor interactions transcription start site selec- <br /> 10. Chibazakura, T., S. Kitajima, M. Yonaha, Y. Yasukochi. 1994. Enhance-            tion. Science 263:805&#8211;807. <br />     ment bacterial transcription initiation vitro 74 kDa subunit     39. Liao, S. M., J. H. Zhang, D. A. Jeffrey, A. J. Koleske, C. M. Thompson, D. M. <br />     human general transcription factor IIF (RAP74). Biochim. Biophys. Acta               Chao, M. Viljoen, H. J. J. Vanvuuren, R. A. Young. 1995. A kinase-cyclin <br />     1219:592&#8211;600.                                                                        pair RNA polymerase II holoenzyme. Nature (London) 374:193&#8211;196. <br /> 11. Conaway, R. C., J. W. Conaway. 1993. General initiation factors          40. Mandel, M., A. Higa. 1970. Calcium-dependent bacteriophage DNA <br />     RNA polymerase II. Annu. Rev. Biochem. 62:161&#8211;190.                                   infection. J. Mol. Biol. 53:159&#8211;162. <br /> 12. Dower, W. J., J. F. Miller, C. W. Ragsdale. 1988. High efficiency trans-     41. McDonnell, D. P., J. W. Pike, D. J. Drutz, T. R. Butt, B. W. O&#8217;Malley. <br />     formation E. coli high voltage electroporation. Nucleic Acids Res. 16:         1989. Reconstitution vitamin D-responsive osteocalcin transcription <br />     6127&#8211;6145.                                                                           unit Saccharomyces cerevisiae. Mol. Cell. Biol. 9:3517&#8211;3523. <br /> 13. Eisenmann, D. M., K. M. Arndt, S. L. Ricupero, J. W. Rooney, F.              42. Mengus, G., M. May, X. Jacq, A. Staub, L. Tora, P. Chambon, I. <br />     Winston. 1992. SPT3 interacts TFIID allow normal transcription            Davidson. 1995. Cloning characterization hTAF(II)18, hTAF(II)20 <br />     Saccharomyces cerevisiae. Genes Dev. 6:1319&#8211;1331.                                    hTAF(II)28: subunits human transcription factor TFIID. <br /> 14. Eisenmann, D. M., C. Chapon, S. M. Roberts, C. Dollard, F. Winston.              EMBO J. 14:1520&#8211;1531. <br />     1994. The Saccharomyces cerevisiae SPT8 gene encodes acidic protein       43. Mikaelian, I., A. Sergeant. 1992. A general fast method generate <br />     functionally related SPT3 TATA-binding protein. Genetics              multiple site directed mutations. Nucleic Acids Res. 20:376&#8211;377. <br />     137:647&#8211;657.                                                                     44. Na, J. G., M. Hampsey. 1993. The <span id='am-21' about='obo:NCBITaxon_4910' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Kluyveromyces" datatype="xsd:string"></span><span id='am-24'  ></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Kluyveromyces</span> gene encoding the <br /> 15. Eisenmann, D. M., C. Dollard, F. Winston. 1989. SPT15, gene                  general transcription factor IIB: structural analysis expression Sac- <br />     encoding yeast TATA binding factor TFIID, required normal                 charomyces cerevisiae. Nucleic Acids Res. 21:3413&#8211;3417. <br />     transcription initiation vivo. Cell 58:1183&#8211;1191.                             45. Nikolov, D. B., H. Chen, E. D. Halay, A. A. Usheva, K. Hisatake, D. K. Lee, <br /> 16. Feaver, W. J., N. L. Henry, D. A. Bushnell, M. H. Sayre, J. H. Brickner, O.          R. G. Roeder, S. K. Burley. 1995. Crystal structure TFIIB-TBP- <br />     Gileadi, R. D. Kornberg. 1994. Yeast TFIIE&#8212;cloning, expression,              TATA-element ternary complex. Nature (London) 377:119&#8211;128. <br />     homology vertebrate proteins. J. Biol. Chem. 269:27549&#8211;27553.                 46. Nonet, M. L., R. A. Young. 1989. Intragenic extragenic suppressors <br /> 17. Flores, O., H. Lu, M. Killeen, J. Greenblatt, Z. F. Burton, D. Reinberg.         mutations heptapeptide repeat domain Saccharomyces cerevisiae <br />     1991. The small subunit transcription factor IIF recruits RNA polymerase          RNA polymerase II. Genetics 123:715&#8211;724. <br />     II preinitiation complex. Proc. Natl. Acad. Sci. USA 88:9999&#8211;10003.     47. Ossipow, V., J.-P. Tassan, E. A. Nigg, U. Schibler. 1995. A mammalian <br /> 18. Foreman, P. K., R. W. Davis. 1994. Cloning vectors synthesis          RNA polymerase II holoenzyme containing components required for <br />     epitope-tagged, truncated chimeric proteins Saccharomyces cerevisiae.         promoter-specific transcription initiation. Cell 83:137&#8211;146. <br />     Gene 144:63&#8211;68.                                                                  48. Paranjape, S. M., R. T. Kamakaka, J. T. Kadonaga. 1994. Role of <br /> 19. Frankel, A. D., D. S. Bredt, C. O. Pabo. 1988. Tat protein human            chromatin structure regulation transcription RNA polymerase II. <br />     immunodeficiency virus forms metal-linked dimer. Science 240:70&#8211;73.                Annu. Rev. Biochem. 63:265&#8211;297. <br /> 20. Furter-Graves, E. M., B. D. Hall, R. Furter. 1994. Role small RNA       49. Pause, A., N. Methot, N. Sonenberg. 1993. The HRIGRXXR region of <br />     pol II subunit TATA transcription start site spacing. Nucleic Acids Res.       DEAD box RNA helicase eukaryotic translation initiation factor 4A is <br />     22:4932&#8211;4936.                                                                        required RNA binding ATP hydrolysis. Mol. Cell. Biol. 13:6789&#8211; <br /> 21. Gansheroff, L. J., C. Dollard, P. Tan, F. Winston. 1995. The Saccharo-           6798. <br />     myces cerevisiae SPT7 gene encodes acidic protein important           50. Pinto, I., D. E. Ware, M. Hampsey. 1992. The yeast SUA7 gene encodes <br />     transcription vivo. Genetics 139:523&#8211;536.                                         homolog human transcription factor TFIIB required normal <br /> 22. Geisberg, J. V., W. S. Lee, A. J. Berk, R. P. Ricciardi. 1994. The zinc          start site selection vivo. Cell 68:977&#8211;988. <br />     finger region adenovirus E1A transactivating domain complexes        51. Pinto, I., W. H. Wu, J. G. Na, M. Hampsey. 1994. Characterization of <br />     TATA box binding protein. Proc. Natl. Acad. Sci. USA 91:2488&#8211;2492.               sua7 mutations defines domain TFIIB involved transcription start site <br /> 23. Gietz, D., A. St. John, R. A. Woods, R. H. Schiestl. 1992. Improved              se <br /> </body></html>