<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Volume <br /> Jovelin and <br /> 2009    10, Issue <br />             Phillips <br />                   4, Article R35                                                                                             Open Access <br /> Research <br /> Evolutionary rates centrality yeast gene regulatory <br /> network <br /> Richard Jovelin Patrick C Phillips <br /> Address: Center Ecology Evolutionary Biology, 5289 University Oregon, Eugene, OR 97403, USA. <br />  <br /> Correspondence: Richard Jovelin. Email: rjovelin@uoregon.edu <br />  <br />  <br /> Published: 9 April 2009                                                       Received: 9 October 2008 <br />                                                                               Accepted: 9 April 2009 <br /> Genome Biology 2009, 10:R35 (doi:10.1186/gb-2009-10-4-r35) <br /> The electronic version article complete be <br /> found online http://genomebiology.com/2009/10/4/R35 <br />  <br /> &#169; 2009 Jovelin Phillips; licensee BioMed Central Ltd. <br /> <span id='am-26' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28'  ></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> open access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), which <br /> permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br /> Network centrality <br /> &lt;p&gt;Yeast    transcription <br />                        evolution <br />                              factors central transcription network tend evolve quickly.&lt;/p&gt; <br />  <br />  <br />  <br />  <br />                  Abstract <br />  <br />                  Background: Transcription factors play fundamental role regulating physiological responses <br />                  developmental processes. Here examine evolution yeast transcription factors in <br />                  context structure gene regulatory network. <br />  <br />                  Results: In contrast previous results protein-protein interaction metabolic <br />                  networks, position gene transcription network affects rate of <br />                  protein evolution central transcription factors tend evolve faster. Centrality is <br />                  positively correlated expression variability, suggesting higher rate divergence <br />                  central transcription factors role controlling information flow and <br />                  result adaptation changing environmental conditions. Alternatively, central <br />                  transcription factors buffered environmental perturbations and, therefore, <br />                  subject strong purifying selection. Importantly, relationship centrality and <br />                  evolutionary rates independent expression level, expression variability gene essentiality. <br />  <br />                  Conclusions: Our analysis transcription network highlights role network structure <br />                  protein evolutionary rate. Further, effect network centrality nucleotide divergence is <br />                  different metabolic, protein-protein transcriptional networks, suggesting the <br />                  effect gene position dependant function specific network study. A better <br />                  understanding cellular networks interact needed to <br />                  fully examine impact network structure function evolution biological systems. <br />  <br />  <br />  <br /> Background                                                                     results understanding cellular [1], developmental [2] <br /> Understanding function evolution specific                    ecological [3] processes. A major challenge this <br /> gene protein requires knowledge context                     area determine various parts interact <br /> that gene operates, change single component                     <span id='am-39' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-40' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> function. With more <br /> of complex ramifications com-                  global understanding function hand, larger <br /> ponents. This system-orientated view, largely enabled                   question emerges: ways does structure the <br /> omics revolution, sparked increasing inves-                network influence evolution components that <br /> tigation biological networks yielded promising                      network? For example, yeast protein-protein interac- <br />  <br />                                                               Genome Biology 2009, 10:R35 <br />  http://genomebiology.com/2009/10/4/R35                         Genome Biology 2009,   Volume 10, Issue 4, Article R35   Jovelin Phillips R35.2 <br />  <br />  <br />  <br />  <br /> tion (PPI) metabolic networks, central highly con-          Overall, network structure does lead to <br /> nected proteins tend evolve slowly peripheral          different evolutionary dynamics depends specifi- <br /> genes [4-7]. Is global feature networks,      cally overall function network. Therefore, under- <br /> does specific function given network strong         standing relationship network structure the <br /> influence evolutionary properties? Here, address          evolution network components depend deeper <br /> these questions analyzing evolution yeast tran-       knowledge gene function. <br /> scription factors context structure tran- <br /> scriptional regulatory network. <br />                                                                     Results <br /> The premise biological systems sum        Central transcription factors tend evolve faster <br /> their parts implies systems possess emergent prop-        We obtained node statistics, specifically number regu- <br /> erties captured purely reductionist             latory inputs (in-degree, kin), number target genes (out- <br /> approach. For network, emergent property          degree, kout), betweenness, measuring centrality a <br /> topology. Comparisons entirely different types networks,      gene network, separately derived representa- <br /> including social, technological biological networks,       tions yeast transcriptional network. The dataset <br /> revealed intriguing shared topological properties,       (YTN1) [15] includes 286 transcription factors, 3,369 target <br /> overall hierarchical organization, similar node-degree distri-      genes 8,372 regulatory interactions. The second dataset <br /> butions, tendency small-world structure           (YTN2) [14] includes 157 transcription factors, 4,410 target <br /> which nodes connected interven-        genes 12,873 regulatory interactions. Only transcription <br /> ing nodes [1]. The observation metabolic PPI          factors clearly identified orthologs yeast genome <br /> networks display approximately scale-free topologies,        database (<span id='am-11' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-17'  ></span>Saccharomyces</span> Genome Database (SGD)) were <br /> few highly connected nodes majority nodes        retained analysis evolutionary rates, leading the <br /> a connections, leads proposal network struc-        retention set 256 genes YTN1 set 138 genes <br /> ture result selection, means pro-     YTN2. Because network contains 85% tran- <br /> viding mutational robustness [8]. This hypothesis remains           scription factors second, power <br /> uncertain, however, networks node connectivity         detect significant effects using network there- <br /> following power-law distribution assembled         fore focus discussion dataset. Neverthe- <br /> natural selection [9] natural selection         less, datasets yield qualitatively similar results each <br /> weak second order network properties robustness          major conclusions. <br /> [10]. Further, networks similar power-law distributions <br /> can different fine-scale architectures,           Large-scale analyses shown multiple genomic vari- <br /> functionally important [11].                                        ables effect rate protein evolution [16,17]. <br />                                                                     Among them, expression level shown correlate <br /> In study examine evolution yeast transcrip-      strongly gene&apos;s evolutionary rate [18-22], leading a <br /> tion factors ask fine differences network            wide debate importance genomic variables <br /> structure function lead different evolutionary impacts       essentiality [21-25] connectivity [7,21,24,26-28]. <br /> on elements networks. Gene regulatory networks         Therefore, examine separate effects expression, <br /> are particular allow cell modify    function network related variables rates transcrip- <br /> its physiology, cycle shape response environmental        tion factor sequence evolution turn, tease apart <br /> or developmental demands [12]. Metabolic gene regula-           independent effects using multivariate approach. <br /> tory networks different level complexity PPI <br /> networks directed explicitly model         As noted previous studies, expression level strong <br /> flow information passing nodes. Moreover,            effect transcription factor sequence evolution (Figure 1), <br /> even cellular networks characterized        highly expressed genes stronger puri- <br /> having small number highly connected nodes, net-         fying selection amino acid replacements and <br /> works differ node-degree distribution [1]. The yeast       synonymous changes, predicted translational <br /> transcription regulatory network consists mixed scale-         robustness hypothesis [20]. Further, essential transcription <br /> free exponential topology: number target            factors (those having lethal phenotype deletion-mutants <br /> genes follows power-law distribution number           [29]) tend evolve slower non-essential transcrip- <br /> of regulators exponential [13]. These structural func-       tion factors, network includes tran- <br /> tional differences result different effects evolu-    scription factors (YTN1: dN/dS: t249 = 3.62, P &lt; 0.001; <br /> tion network components. For instance, underlying            Wilcoxon two-sample P &lt; 0.0001). Similarly, corre- <br /> power-law distribution target genes distributed archi-      lation protein evolutionary rates genes&apos; essenti- <br /> tecture cause apparent independence            ality estimated growth rate deletion strains [19] <br /> connectivity retention regulatory proteins        (Figure 1). <br /> genomes [14]. <br />  <br />                                                      Genome Biology 2009, 10:R35 <br />  http://genomebiology.com/2009/10/4/R35                                      Genome Biology 2009,     Volume 10, Issue 4, Article R35        Jovelin Phillips R35.3 <br />  <br />  <br />  <br />  <br />                                                            dN                                   dS                                     dN/dS <br />                     Centrality                                         *                                                                            ** <br />                               kin                                                                                                               * <br />                                                                             **                                                                            *** <br />                             kout                                       *                                                                        * <br />                              CAI                  *                                                                               * <br />          Expression level               ***                                 ***                                               ** <br />                                                                                      * <br />  Expression variability                                                *                                                                         * <br />                  Essentiality              **                                                                           *** <br />                              GO                                                                                                    * <br />                                               *                                                                          ** <br />                                               -0.2         0       0.2              -0.2         0            0.2            -0.2       0           0.2 <br />                                                    Residuals dN-dS                              dS&apos;                                    dN/dS&apos; <br />                     Centrality                                         **                                                                           ** <br />                               kin                                  *                                                                            * <br />                                                                             ***                                                                           *** <br />                             kout                                  *                                                                              * <br />                              CAI                      *                                                                      ** <br />          Expression level                         **                          ***                                         ** <br />                                                                                      * <br />  Expression variability                                            *                                                                             * <br />                  Essentiality                 **                                                                       *** <br />                              GO                        *                                                                          * <br />                                            **                                                                           ** <br />                                               -0.2         0       0.2              -0.2         0           0.2          -0.2          0           0.2 <br />  <br /> Correlation <br /> Figure 1 expression (blue), function (green), network topology (red) related variables evolutionary rates <br /> Correlation expression (blue), function (green), network topology (red) related variables evolutionary rates. Darker colors represent <br /> results analyses YTN1, lighter colors represent results analyses YTN2. Correlations Spearman&apos;s nonparametric &#65533;?. *P &lt; 0.05, **P &lt; <br /> 0.01, ***P &lt; 0.001. <br />  <br />  <br />  <br /> To investigate impact functional constraints                      (Figure 1). Previous studies determined central <br /> sequence evolution, used number Gene Ontology                             metabolic enzymes central proteins PPI network <br /> (GO) terms [30,31] proxy gene&apos;s pleiotropic effects.                     stronger selective constraints evolve slower <br /> GO gene&apos;s properties functions assigning                          [4,6]. In contrast, transcriptional net- <br /> attributes categories &apos;cellular component&apos;, &apos;biologi-                     work, protein evolution positively correlated between- <br /> cal process&apos; &apos;molecular function&apos;. There correlation                       ness, indicating transcription factors occupy more <br /> between number GO terms essentiality (YTN1:                              central position network tend evolve faster (Figure <br /> Spearman&apos;s &#65533;? = 0.171, P = 0.007), indicating pleiotropy                        1). Similarly, contrary metabolic PPI networks, protein <br /> has direct fitness consequences. Accordingly, transcription                         sequence divergence correlates positively connectivity. <br /> factors GO terms tend evolve slowly (Fig-                         However, relationship out-degree evolution- <br /> ure 1), presumably mutations arising genes                          ary rate differs network datasets (Figure 1). <br /> larger pleiotropic effects likely deleterious and <br /> are selected against.                                                          The effect centrality protein sequence evolution <br />                                                                                     independent genomic variables <br /> Finally, position gene network, cen-                     Interpretation simple correlation patterns compli- <br /> trality, significant influence evolutionary rate                       cated fact different genetic properties corre- <br />  <br />                                                                 Genome Biology 2009, 10:R35 <br />  http://genomebiology.com/2009/10/4/R35                                 Genome Biology 2009,       Volume 10, Issue 4, Article R35      Jovelin Phillips R35.4 <br />  <br />  <br />  <br />  <br /> lated single correlation                (kout: YTN1: Spearman&apos;s &#65533;? = -0.071, P = 0.26; YTN2: &#65533;? = 0.27, <br /> two characteristics actually generated shared                 P = 0.002). However, growth rate measured under <br /> correlation causal element. To correct this,            different conditions, transcription factors numerous tar- <br /> examined relative contribution function, network                 genes YTN2 enriched essential genes [14]. <br /> expression-associated constraints transcription factor                   Nevertheless, correlation number target <br /> evolution using multivariate analysis.                                      genes protein sequence divergence fairly weak, mul- <br />                                                                             tiple regression analysis failed disentangle effect out- <br /> We used multiple regression analysis network con-                degree causal effect predictor variables <br /> nectivity network centrality separately function               (Table 1). Therefore, contrary PPI [4,8] metabolic <br /> expression-related predictor variables order estimate                 [5,6] networks, significant correlation between <br /> the contribution of elements variation evo-             connectivity essentiality, in-degree fact posi- <br /> lutionary rates transcription factors. Consistent                tively correlated protein sequence divergence. <br /> the univariate patterns, analysis reveals transcrip- <br /> tion factors having larger effects organismal fitness               Importantly, analysis shows contribution of <br /> deleted tend evolve slowly lesser fit-              network centrality protein divergence independent of <br /> ness effects (Table 1). In vein, transcription factors             expression function-related variables (Table 1). Thus, a <br /> with larger number GO terms subject stronger                    striking difference cellular networks lies influ- <br /> functional constraints tend evolve slowly (Table                ence position gene network rate of <br /> 1). These results indicate sequence divergence                 evolution. However, transcription factors cen- <br /> yeast transcription factors depends cost            tral network tend higher variability their <br /> of mutations altering protein function affecting organis-               expression level changing conditions (YTN1: Spear- <br /> mal fitness. Among genomic variables analyzed,                     man&apos;s &#65533;? = 0.178, P = 0.004), centrality correlated <br /> expression level strong effect rate synonymous              expression level (YTN1: Spearman&apos;s &#65533;? = 0.006, P = <br /> changes (Table 2)                                                           0.924) essentiality (YTN1: Spearman&apos;s &#65533;? = -0.022, P = <br />                                                                             0.735). <br /> We significant correlation in-degree and <br /> essentiality (kin: YTN1: Spearman&apos;s &#65533;? = -0.082, P = 0.2;                    The high degree correlation predictor variables has <br /> YTN2: &#65533;? = 0.033, P = 0.7), relationship                led question use multiple regression these <br /> out-degree essentiality differs datasets                types analyses [21]. We analyzed <span id='am-77' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-79'  ></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> <br />  <br /> Table 1 <br />  <br /> Multiple regression genomic variables protein evolutionary rates <br />  <br />                                                                                   dN                 dN/dS              dN/dS&apos;         Residuals dN-dS <br />  <br /> Predictor                                                                YTN1          YTN2     YTN1      YTN2     YTN1      YTN2      YTN1      YTN2 <br />  <br /> Relationships evolutionary rates predictor variables <br />     Expression level                                                     -0.105        -0.022   -0.043    0.058    -0.051    0.061     -0.038     0.050 <br />     Expression variability                                               -0.047        -0.064   -0.010    -0.015   -0.011    -0.013    -0.019    -0.024 <br />     CAI                                                                  -0.096        -0.037   0.026     -0.060   -0.088    -0.046    -0.072    -0.055 <br />     GO                                                                   -0.135*   -0.277&#8224;      -0.149* -0.311&#8225;    -0.165&#8224;   -0.313&#8225;   -0.168&#8224;   -0.307&#8225; <br />     Essentality                                                          -0.185&#8224;       -0.104   -0.229&#8225;   -0.121   -0.208&#8225;   -0.122    -0.181&#8224;   -0.121 <br />     Centrality                                                           0.162&#8224;        0.199*   0.151*    0.191*   0.164&#8224;    0.190*    0.164&#8224;    0.201* <br />  <br />  <br /> Relationships evolutionary rates seven predictor variables <br />     Expression level                                                     -0.107        -0.030   -0.044    0.047    -0.053    0.050     -0.040     0.041 <br />     Expression variability                                               -0.053        -0.084   -0.016    -0.036   -0.020    -0.035    -0.028    -0.043 <br />     CAI                                                                  -0.111        -0.059   0.010     -0.083   -0.105    -0.069    -0.091    -0.077 <br />     GO                                                                   -0.113    -0.204* -0.129* -0.244&#8224; -0.142* -0.246&#8224; -0.145* -0.237&#8224; <br />     Essentiality                                                         -0.177&#8224;       -0.039   -0.222&#8225;   -0.060   -0.199&#8224;   -0.062    -0.173&#8224;   -0.057 <br />     kin                                                                  0.139*        0.291&#8224;   0.132*    0.288&#8224;   0.148*    0.288&#8224;    0.152*    0.290&#8224; <br />     kout                                                                  0.042        -0.165   0.032     -0.148   0.037     -0.147     0.031    -0.154 <br />  <br /> Network, function expression-related variables independent effects rate protein evolution. Entries standardized regression <br /> coefficients. *P &lt; 0.05, &#8224;P &lt; 0.01, &#8225;P &lt; 0.001. <br />  <br />                                                            Genome Biology 2009, 10:R35 <br />  http://genomebiology.com/2009/10/4/R35                                     Genome Biology 2009,              Volume 10, Issue 4, Article R35           Jovelin Phillips R35.5 <br />  <br />  <br />  <br />  <br /> Table 2 <br />  <br /> Multiple regression genomic variables rates synonymous changes <br />  <br />                                                                                                                              dS                              dS&apos; <br />  <br /> Predictor                                                                                                          YTN1             YTN2           YTN1            YTN2 <br />  <br /> Relationships evolutionary rates predictor variables <br />     Expression level                                                                                               -0.216&#8224;          -0.229*        -0.218&#8224;         -0.229* <br />     Expression variability                                                                                         -0.090           -0.137         -0.091          -0.137 <br />     CAI                                                                                                            -0.086           0.050          0.046           0.005 <br />     GO                                                                                                             0.075            0.051          0.076           0.051 <br />     Essentality                                                                                                    -0.041           0.037          -0.041          0.037 <br />     Centrality                                                                                                     0.020            0.026          0.020           0.026 <br />  <br />  <br /> Relationships evolutionary rates seven predictor variables <br />     Expression level                                                                                               -0.213&#8224;          -0.228*        -0.215&#8224;         -0.229* <br />     Expression variability                                                                                         -0.083           -0.140         -0.084          -0.140 <br />     CAI                                                                                                            -0.077           0.046          0.055           0.001 <br />     GO                                                                                                             0.075            0.069          0.076           0.069 <br />     Essentiality                                                                                                   -0.039           0.053          -0.040          0.053 <br />     kin                                                                                                            -0.014           0.048          -0.014          0.048 <br />     kout                                                                                                           0.039            -0.059         0.040           -0.059 <br />  <br /> Entries standardized regression coefficients. *P &lt; 0.05, &#8224;P &lt; 0.01. <br />  <br />  <br />  <br /> Table 3 <br />  <br /> Principal component regression analysis: principal components PC1 PC4 <br />  <br />                                                                         PC1                            PC2                          PC3                        PC4 <br />  <br />                                                                YTN1           YTN2            YTN1           YTN2          YTN1           YTN2        YTN1           YTN2 <br />  <br /> Percent variance explained PC                            27             29              20            19             15             15          12            10 <br />  <br />  <br /> Effect PCs response variables <br />     dN                                                         0.118&#8224;         -0.026         -0.172&#8225;         0.166*         0.015         0.199&#8224;      -0.065         0.144 <br />     dS                                                         -0.001         -0.009         -0.122*         -0.046     -0.161&#8224;           -0.094      0.073          0.096 <br />     dS&apos;                                                        -0.014         -0.019          -0.053         -0.030        -0.108         -0.114      0.082          0.078 <br />     dN/dS                                                      0.114&#8224;         -0.028         -0.113*         0.194&#8224;         0.106         0.240&#8224;     -0.123*         0.117 <br />     dN/dS&apos;                                                     0.132&#8224;         -0.025         -0.169&#8225;         0.190&#8224;         0.064         0.248&#8224;      -0.106         0.122 <br />     Residuals dN-dS                                            0.124&#8224;         -0.024         -0.141&#8224;         0.189&#8224;         0.071         0.239&#8224;      -0.093         0.120 <br />  <br />  <br /> Contribution predictor variables PC <br />     CAI                                                        -0.095         0.228           0.535          -0.356         0.419         0.435       0.069          0.405 <br />     Expression level                                           0.008          0.289           0.515          -0.297         0.154         0.468       -0.596         -0.280 <br />     Expression variability                                     0.290          -0.105          -0.350         0.616          0.454         -0.050      0.132          -0.009 <br />     kin                                                        0.538          0.423           0.195          0.413          0.288         0.333       0.061          0.140 <br />     kout                                                       0.464          0.453           -0.047         0.052         -0.419         -0.353      -0.023         -0.157 <br />     Centrality                                                 0.623          0.564           0.176          0.312         -0.095         -0.010      0.067          0.050 <br />     Essentiality                                               -0.115         0.230           0.415          -0.241        -0.102         -0.506      0.776          0.634 <br />     GO                                                         0.011          0.314           0.287          -0.277        -0.562         -0.313      -0.109         -0.556 <br />  <br /> No single variable dominates rate protein evolution. *P &lt; 0.05,   &#8224;P   &lt; 0.01,   &#8225;P   &lt; 0.001. PC, principal component. <br />  <br />                                                              Genome Biology 2009, 10:R35 <br />  http://genomebiology.com/2009/10/4/R35                                 Genome Biology 2009,            Volume 10, Issue 4, Article R35     Jovelin Phillips R35.6 <br />  <br />  <br />  <br />  <br /> using principal component regression analysis [21]. For                           In summary, results yeast transcription network <br /> YTN1, principal component, composed                           previous work yeast metabolic PPI networks <br /> contributions network-related variables, positively                       [4-6] structure cellular networks influences <br /> correlated protein divergence second principal                       evolution proteins networks. However, the <br /> component, composed expression function-                            system-level pattern selective constraints individual <br /> related variables, correlates negatively substitution rates                  nodes differ despite networks having grossly similar <br /> (Tables 3 4). Both principal components explain similar                     topologies, relation function the <br /> amount total variance data, indicating                      nature network. <br /> single variable dominates rate protein evolution the <br /> yeast transcription factor genes. The pattern complex <br /> for YTN2 principle components tend confound                        Discussion <br /> expression network properties. For instance,                        Genomic information generated recent years only <br /> principle component YTN2 does significant                          offered new insights biological processes various levels <br /> effect evolutionary rate, presumably positive                      organization [1-3,32], enabled shift from <br /> and negative effects network, function, expression                     studying evolution single genes system-level <br /> variables counterbalancing (Tables 3 4).                      view molecular evolution integrates interactions <br />                                                                                   genes cellular context. A conse- <br /> To issues, defined new set variables                     quence new perspective recognition several <br /> composed principal components derived separately                          factors addition protein function control rate divergence <br /> the expression, network function-related variables. Mul-                      coding sequences [16,17], expression level having a <br /> tiple regression analysis composite variables shows                      strong effect [20,21]. <br /> that causal components independent effects <br /> on rate nonsynonymous changes (Table 5). Results                           A second consequence systems molecular evolution <br /> from network datasets qualitatively quantita-                     perspective yields novel insights cellular <br /> tively similar another, particular coeffi-                   networks components evolve. Previous studies have <br /> cients YTN2 tend significant                           noted metabolic enzymes high degree more <br /> reduced power.                                                                    essential low degree, rerout- <br />  <br /> Table 4 <br />  <br /> Principal component regression analysis: principal components PC5 PC8 <br />  <br />                                                                       PC5                        PC6                       PC7                      PC8 <br />  <br />                                                              YTN1           YTN2        YTN1           YTN2      YTN1            YTN2      YTN1           YTN2 <br />  <br /> Percent variance explained PC                          10             9           7              8          7              8         2              2 <br />  <br />  <br /> Effect PCs response variables <br />     dN                                                       -0.067         -0.170      -0.127         0.109     -0.144          -0.188    -0.088         -0.213 <br />     dS                                                        0.055         0.078       -0.043         0.034     -0.131          -0.249    0.011          0.074 <br />     dS&apos;                                                       0.081         0.050       0.022          0.046     -0.195*         -0.251*   -0.014         -0.073 <br />     dN/dS                                                    -0.045     -0.209*         -0.052         0.098     -0.167*         -0.106    -0.103         -0.147 <br />     dN/dS&apos;                                                   -0.081     -0.201*         -0.117         0.093     -0.114          -0.102    -0.083         -0.143 <br />     Residuals dN-dS                                          -0.089     -0.203*         -0.119         0.103     -0.108          -0.118    -0.089         -0.200 <br />  <br />  <br /> Contribution predictor variables PC <br />     CAI                                                       0.194         0.624       0.501          -0.279    -0.485          0.027     -0.022         -0.025 <br />     Expression level                                         -0.255         -0.366      0.025          0.178      0.539          0.603     0.001          -0.032 <br />     Expression variability                                    0.400         0.447       0.377          0.098      0.518          0.630     -0.010         -0.020 <br />     kin                                                       0.101         -0.060      -0.505         0.328     -0.139          -0.296    0.549          0.567 <br />     kout                                                     -0.353         -0.090      0.584          -0.691    -0.062          0.177     0.370          0.354 <br />     Centrality                                               -0.026         -0.086      -0.102         -0.023    -0.083          -0.170    -0.741         -0.738 <br />     Essentiality                                             -0.192         -0.125      -0.022         0.367      0.402          0.277     -0.049         -0.063 <br />     GO                                                        0.752         0.492       0.012          0.400      0.121          -0.119    0.098          0.047 <br />  <br /> No single variable dominates rate protein evolution. *P &lt; 0.05. PC, principal component. <br />  <br />                                                            Genome Biology 2009, 10:R35 <br />  http://genomebiology.com/2009/10/4/R35                                    Genome Biology 2009,     Volume 10, Issue 4, Article R35      Jovelin Phillips R35.7 <br />  <br />  <br />  <br />  <br /> Table 5 <br />  <br /> Results multiple regression analysis composite variables <br />  <br />                                                                       PC1-network                  PC1-expression                    PC1-function <br />  <br />                                                                    YTN1           YTN2           YTN1           YTN2           YTN1            YTN2 <br />  <br /> Percent variance explained PC1                                 69            65              48             50             54              60 <br /> dN                                                                 0.095*         0.099          -0.112*        -0.007        -0.196&#8224;          -0.246&#8224; <br /> dS                                                                 -0.016        -0.031          -0.170&#8224;        -0.032         0.065            0.098 <br /> dS&apos;                                                                -0.016        -0.032          -0.094         -0.058         0.068            0.099 <br /> dN/dS                                                              0.098*         0.112          -0.014         -0.001        -0.249&#8225;          -0.295&#8225; <br /> dN/dS&apos;                                                             0.107&#8224;         0.113          -0.083         0.008         -0.245&#8225;          -0.298&#8225; <br /> Residuals dN-dS                                                    0.106*         0.113          -0.061         0.003         -0.229&#8225;          -0.289&#8225; <br />  <br /> Each composite variable principal component expression (CAI, expression level, expression variability), network (betweenness, kin, kout) <br /> and function (GO, essentiality) related variables. *P &lt; 0.05, &#8224;P &lt; 0.01, &#8225;P &lt; 0.001. <br />  <br />  <br /> ing metabolic fluxes highly connected regions circum-                    sites involved interactions kinetic con- <br /> vents loss function mutations given node [5]. The                      straints highly used metabolites, respectively [5,7]. <br /> absence correlation connectivity essentiality <br /> observed consequence similar mecha-                       In contrast, transcription networks play fundamental roles in <br /> nism genetic robustness achieved rerouting                       regulating cell state developmental processes dur- <br /> information flow transcriptional network. This                     ing physiological adjustment changing environmental con- <br /> hypothesis suggested recent study showing                 ditions [12]. For instance, changes growth conditions lead <br /> the mean sequence divergence intermediate regula-                        <span id='am-42' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-43' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-50'  ></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span>Escherichia coli</span> regulate transcript protein levels to <br /> tors regulator target gene increases                maximize growth rate maintain stable metabolite levels, <br /> the number alternative pathways regulator-                      enzymes carbon metabolism network <br /> target gene pair [33].                                                         disrupted, stability achieved redun- <br />                                                                                dancy flux rerouting [34]. In eukaryotes yeast, <br /> We obtain qualitatively similar results analysis                   transcriptional variability (which serve <span id='am-1' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Indicator" datatype="xsd:string"></span><span id='am-5'  ></span>indicator</span> <br /> both representations transcriptional network [14,15].                   environmental sensitivity), expression level per <br /> This especially true account overall correlation              se, correlate better protein divergence [35]. <br /> structure variables network, function,                    Here, transcription factors central net- <br /> and expression classes (Table 5). Many transcription                      work tend higher variability expression level <br /> factors represented network, however,                   changing conditions. At local scale, expression variability <br /> makes easier detect significant evolutionary asso-                  regulatory motif depends network structure <br /> ciations. It clear, therefore, completeness net-                [36]. However, significant effect expres- <br /> work influence conclusions global analyses                   sion variation transcription factor evolution (Table 1). The <br /> that conducted here. Nevertheless, fact similar                       influence centrality rate protein evolution the <br /> results obtained different network datasets,                    yeast transcription factor network second- <br /> undoubtedly capture different levels network complexity,                    ary effect selection acting directly transcriptional varia- <br /> suggests results presented somewhat robust                   bility. Because central transcription factors rapid access <br /> to overall sampling issues.                                                    regions network act control the <br />                                                                                flow information network, impor- <br /> Our results yeast transcription network previous                    tant components sensory systems transduce environ- <br /> work yeast metabolic PPI networks [4-6]                        mental changes coordinate response regulatory <br /> that structure cellular networks influences selective                   network. It possible higher level amino acid <br /> constraints individual nodes, system-level                   change seen central transcription factors the <br /> constraints differ despite cellular networks having                  result historical adaptation changing environmental <br /> similar, identical, topological properties [1,13].                conditions. An alternative hypothesis central <br /> These differences ultimately nature                   transcription factors instead buffered outside <br /> networks function. Highly connected proteins                      influences subject strong purifying selec- <br /> in PPI metabolic networks subject stronger                      tion. <br /> purifying selection, presumably larger fraction of <br />  <br />  <br />  <br />                                                              Genome Biology 2009, 10:R35 <br />  http://genomebiology.com/2009/10/4/R35                         Genome Biology 2009,   Volume 10, Issue 4, Article R35   Jovelin Phillips R35.8 <br />  <br />  <br />  <br />  <br /> Although relationship centrality evolution-         generate <span id='am-31' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-32' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-33'  ></span><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>codon</span>-based DNA sequence alignments. Maximum <br /> ary rate somewhat unexpected, examination fine            likelihood estimates rates amino acid replacements <br /> scale structure networks indicates      (dN) synonymous changes (dS) computed in <br /> general property control systems. For example,          CODEML [45]. In addition, computed rate synony- <br /> highly connected proteins (hubs) yeast PPI network           mous changes corrected selection silent sites (dS&apos;) [46]. <br /> evolve slowly [4,7], intermodule hubs (those display tem-      We attempted correct correlation dN <br /> poral variation connections) divergent       dS using residuals regression dN <br /> intramodule hubs (those displaying static patterns interac-      dS analyses. <br /> tions) [37]. Similarly, directional selection recently been <br /> inferred controlling, branch-point enzymes        Essentiality defined lethal phenotype deletion <br /> five metabolic pathways converging glucose-6-phosphate           strains [29]. For quantitative measure gene&apos;s essential- <br /> in Drosophila [38]. Thus, proteins exert control       ity used growth rates measured deletion mutants [19]. <br /> flux distribution, information processing connecting          The number GO terms [30,31] used proxy gene&apos;s <br /> various protein complexes may, general, target         pleiotropic effect obtained SGD. Protein and <br /> adaptation mutations arising proteins         mRNA abundance used estimates gene <br /> the potential affect entire may, therefore,    expression studies evolutionary rates yeast [18- <br /> more exposed natural selection.                                  22,24,25,37]. We obtained protein [47] mRNA [48] <br />                                                                     abundance literature. However, sample faster <br />                                                                     evolving genes likely missing mRNA <br /> Conclusions                                                         abundance (YTN1: N = 206; YTN2: N = 108) protein <br /> The system-level pattern evolutionary rates different         abundance (YTN1: N = 195; YTN2: N = 96) datasets, leading <br /> from observed protein-protein interaction           obvious <span id='am-6' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-7' property="rdfs:label" content="Bias" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>bias</span> (YTN1: protein abundance: dN/dS: Wil- <br /> metabolic networks: central transcription factors tend           coxon two-sample P = 0.004; mRNA abundance: dN/dS: <br /> evolve faster. This suggests higher nucleotide rate        Wilcoxon two-sample P = 0.04; YTN2: protein abundance: <br /> divergence central transcription factors result         dN: Wilcoxon two-sample P = 0.03; mRNA abundance: dN/ <br /> the role proteins play controlling flow        dS: Wilcoxon two-sample P = 0.06). Nevertheless, trans- <br /> information result adaptation changing         lational robustness hypothesis suggests frequency of <br /> environmental conditions. The conclusions derived net-         translation events better indicator evolutionary rate <br /> work level analyses molecular evolution clearly vary         number proteins cell [20]. Therefore, used <br /> depending functional role played components        codon adaptation index (CAI) [49], measures syn- <br /> that network. In way shown           onymous codon usage bias correlates mRNA abun- <br /> particular function network influence inter-       dance [50], direct measures expression level, as <br /> prets impact structure protein evolution,       substitutes abundance measures. CAI com- <br /> clear begin link networks (regula-     puted [51] using reference gene set defined Carbone et <br /> tory, protein-protein, metabolic)          al. [52]. Expression level average level expression <br /> complete nature consequences network structure            198 microarrays wide range conditions [35]. <br /> the function evolution biological systems exam-       Expression variation measured coefficient varia- <br /> ined.                                                               tion defined mean standard deviation [35]. <br />  <br />                                                                     Statistical analyses performed using JMP 4.0.4 (SAS <br /> Materials methods                                               Institute Inc., Cary, NC, USA). The number GO terms and <br /> We used distinct datasets yeast transcriptional net-     kout natural-logarithmic transformed approximate a <br /> work. The dataset [15], YTN1, includes 286 transcription      normal distribution. One unit added betweenness and <br /> factors, 3,369 target genes 8,372 regulatory interactions.      kin, kout YTN2, prior natural logarithmic <br /> The second dataset [14], YTN2, includes 157 transcription           transformation null values variables. All <br /> factors, 4,410 target genes 12,873 regulatory interactions.     variables, including predictor response variables, were <br /> The networks derived largely independent              standardized mean 0 1 standard deviation unit. In <br /> genetic, biochemical ChIP-chip experiments. Node sta-           addition Spearman&apos;s rank correlations multiple <br /> tistics, including in-degree (kin), out-degree (kout)           regression analysis, performed principal component <br /> betweenness, obtained dataset using tYNA          regression analysis, using single predictor variables <br /> platform [39].                                                      defining new set principal compo- <br />                                                                     nents separately expression, network function- <br /> Protein sequences orthologous genes Saccharomyces           related variables. These composite variables obtained <br /> cerevisiae [40] S. paradoxus, closely related          principal component expression (CAI, <br /> <span id='am-36' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-37' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> [41] having genome sequenced [42], retrieved       expression level, expression variability), network (between- <br /> from SGD [43], aligned [44], subsequently used           ness, kin, kout) function (GO, essentiality) related varia- <br />  <br />                                                      Genome Biology 2009, 10:R35 <br />  http://genomebiology.com/2009/10/4/R35                                  Genome Biology 2009,    Volume 10, Issue 4, Article R35     Jovelin Phillips R35.9 <br />  <br />  <br />  <br />  <br /> bles. Principal component analyses performed                         17.   P&#225;l C, Papp B, Lercher MJ: An integrated view protein evolu- <br /> correlations.                                                                      tion. Nat Rev Genet 2006, 7:337-348. <br />                                                                              18.   P&#225;l C, Papp B, Hurst LD: Highly expressed genes yeast evolve <br />                                                                                    slowly. Genetics 2001, 158:927-931. <br />                                                                              19.   Wall DP, Hirsh AE, Fraser HB, Kumm J, Giaever G, Eisen MB, Feld- <br />                                                                                    man MW: Functional genomic analysis rates protein <br /> Abbreviations                                                                      evolution. Proc Natl Acad Sci USA 2005, 102:5483-5488. <br /> CAI: codon adaptation index; dN: rate nonsynonymous                       20.   Drummond DA, Bloom JD, Adami C, Wilke CO, Arnold FH: Why <br /> changes; dS: rate synonymous changes; dS&apos;: rate synon-                       highly expressed proteins evolve slowly. Proc Natl Acad Sci USA <br />                                                                                    2005, 102:14338-14343. <br /> ymous changes corrected selection silent sites; GO:                   21.   Drummond DA, Raval A, Wilke CO: A single determinant dom- <br /> Gene Ontology; kin: in-degree; kout: out-degree; PPI: protein-                     inates rate yeast protein evolution. Mol Biol Evol 2006, <br />                                                                                    23:327-337. <br /> protein interaction; SGD: Saccharomyces Genome Database.                     22.   P&#225;l C, Papp B, Hurst LD: Genomic function: Rate evolution <br />                                                                                    gene dispensability. Nature 2003, 421:496-497. discussion <br />                                                                                    497-498. <br />                                                                              23.   Hirsh AE, Fraser HB: Protein dispensability rate evolu- <br /> Authors&apos; contributions                                                             tion. Nature 2001, 411:1046-1049. <br /> RJ designed study collected data. RJ PCP                     24.   Plotkin JB, Fraser HB: Assessing determinants evolution- <br /> analyzed data wrote paper.                                             ary rates presence noise. Mol Biol Evol 2007, <br />                                                                                    24:1113-1121. <br />                                                                              25.   Kim SH, Yi SV: Understanding relationship sequence <br />                                                                                    functional evolution yeast proteins. Genetica 2007, <br />                                                                                    131:151-156. <br /> Acknowledgements                                                             26.   Bloom JD, Adami C: Apparent dependence protein evolu- <br /> Support provided Doctoral Dissertation Improvement Grant                  tionary rate number interactions linked biases in <br /> (NSF DEB-0710378), UO IGERT program Evolution, Development                  protein-protein interactions data sets. BMC Evol Biol 2003, 3:21. <br /> and Genomics (NSF DGE-9972830) grants National Sci-          27.   Fraser HB, Hirsh AE: Evolutionary rate depends number of <br /> ence Foundation (DEB-0441066) National Institutes Health                protein-protein interactions independently gene expres- <br /> (AG029377).                                                                        sion level. BMC Evol Biol 2004, 4:13. <br />                                                                              28.   Bloom JD, Adami C: Evolutionary rate depends number of <br />                                                                                    protein-protein interactions independently gene expres- <br />                                                                                    sion level: response. BMC Evol Biol 2004, 4:14. <br /> References                                                                   29.   Giaever G, Chu AM, Ni L, Connelly C, Riles L, Veronneau S, Dow S, <br /> 1.    Barabasi AL, Oltvai ZN: Network biology: understanding                   Lucau-Danila A, Anderson K, Andre B, Arkin AP, Astromoff A, El- <br />       cell&apos;s functional organization. Nat Rev Genet 2004, 5:101-113.               Bakkoury M, Bangham R, Benito R, Brachat S, Campanaro S, Curtiss <br /> 2.    Davidson EH, Rast JP, Oliveri P, Ransick A, Calestani C, Yuh CH,             M, Davis K, Deutschbauer A, Entian KD, Flaherty P, Foury F, Garfinkel <br />       Minokawa T, Amore G, Hinman V, Arenas-Mena C, Otim O, Brown                  DJ, Gerstein M, Gotte D, G&#252;ldener U, Hegemann JH, Hempel S, Her- <br />       CT, Livi CB, Lee PY, Revilla R, Rust AG, <span id='am-18' about='obo:NCBITaxon_9596' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-19' property="rdfs:label" content="Pan" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:317028" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23'  ></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="chimpanzees" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="NCBITaxon:445518" datatype="xsd:string"></span>Pan</span> Z, Schilstra MJ, Clarke         man Z, et al.: Functional profiling Saccharomyces cerevi- <br />       PJ, Arnone MI, Rowen L, Cameron RA, McClay DR, Hood L, Bolouri               siae genome. Nature 2002, 418:387-391. <br />       H: A genomic regulatory network development. Science               30.   Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, <br />       2002, 295:1669-1678.                                                         Davis AP, Dolinski K, Dwight SS, Eppig JT, Harris MA, Hill DP, Issel- <br /> 3.    Proulx SR, Promislow DE, Phillips PC: Network thinking ecol-              Tarver L, Kasarskis A, Lewis S, Matese JC, Richardson JE, Ringwald M, <br />       ogy evolution. Trends Ecol Evol 2005, 20:345-353.                        Rubin GM, Sherlock G: Gene ontology: tool unification <br /> 4.    Hahn MW, Kern AD: Comparative genomics centrality                     biology. The Gene Ontology Consortium. Nat Genet 2000, <br />       essentiality eukaryotic protein-interaction net-                    25:25-29. <br />       works. Mol Biol Evol 2005, 22:803-806.                                 31.   Dwight SS, Harris MA, Dolinski K, Ball CA, Binkley G, Christie KR, <br /> 5.    Vitkup D, Kharchenko P, Wagner A: Influence metabolic net-                Fisk DG, Issel-Tarver L, Schroeder M, Sherlock G, Sethuraman A, <br />       work structure function enzyme evolution. Genome                      Weng S, Botstein D, Cherry JM: Saccharomyces Genome Data- <br />       Biol 2006, 7:R39.                                                            base (SGD) provides secondary gene annotation using the <br /> 6.    Lu C, Zhang Z, Leach L, Kearsey MJ, Luo ZW: Impacts yeast                 Gene Ontology (GO). Nucleic Acids Res 2002, 30:69-72. <br />       metabolic network structure enzyme evolution. Genome                32.   Sauer U, Heinemann M, Zamboni N: Genetics. Getting closer to <br />       Biol 2007, 8:407.                                                            picture. Science 2007, 316:550-551. <br /> 7.    Fraser HB, Hirsh AE, Steinmetz LM, Scharfe C, Feldman MW: Evolu-       33.   Wagner A, Wright J: Alternative routes mutational robust- <br />       tionary rate protein interaction network. Science 2002,               ness complex regulatory networks.                Biosystems 2007, <br />       296:750-752.                                                                 88:163-172. <br /> 8.    Jeong H, Mason SP, Barabasi AL, Oltvai ZN: Lethality central-      34.   Ishii N, Nakahigashi K, Baba T, Robert M, Soga T, Kanai A, Hirasawa <br />       ity protein networks. Nature 2001, 411:41-42.                             T, Naba M, Hirai K, Hoque A, Ho PY, Kakazu Y, Sugawara K, Igarashi <br /> 9.    Wagner A: How global structure protein interaction                    S, Harada S, Masuda T, Sugiyama N, Togashi T, Hasegawa M, Takai Y, <br />       networks evolves. Proc Biol Sci 2003, 270:457-466.                           Yugi K, Arakawa K, Iwata N, <span id='am-65' about='obo:NCBITaxon_984450' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_33361 obo:NCBITaxon_33362 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_35567 obo:NCBITaxon_35568 obo:NCBITaxon_33317 obo:NCBITaxon_130551 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_7524 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_36151 obo:NCBITaxon_88770 obo:NCBITaxon_33342 obo:NCBITaxon_50557'><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-67'  ></span><span id='am-68' property="rdfs:label" content="Toya" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Toya</span> Y, Nakayama Y, Nishioka T, <br /> 10.   Proulx SR, Phillips PC: The opportunity canalization             Shimizu K, Mori H, Tomita M: Multiple high-throughput analyses <br />       evolution genetic networks. Am Nat 2005, 165:147-162.                     monitor response E. coli perturbations. Science 2007, <br /> 11.   Keller EF: Revisiting &quot;scale-free&quot; networks. Bioessays 2005,                 316:593-597. <br />       27:1060-1068.                                                          35.   Choi JK, Kim SC, Seo J, Kim S, Bhak J: Impact transcriptional <br /> 12.   Davidson EH: The Regulatory Genome Amsterdam, Boston, Heidel-                properties essentiality evolutionary rate. Genetics <br />       berg, London, New York, Oxford, <span id='am-70' about='obo:NCBITaxon_49669' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_50362 obo:NCBITaxon_4447 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_4667 obo:NCBITaxon_3193'><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72'  ></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Paris L." datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="Paris" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasAlternativeId" content="NCBITaxon:49665" datatype="xsd:string"></span>Paris</span>, San Diego, San Francisco,             2007, 175:199-206. <br />       Singapore, Sydney, Tokyo: Academic Press; 2006.                        36.   Promislow D: A regulatory network analysis phenotypic <br /> 13.   Guelzim N, Bottani S, Bourgine P, Kepes F: Topological causal            plasticity yeast. Am Nat 2005, 165:515-523. <br />       structure yeast transcriptional regulatory network.             37.   Fraser HB: Modularity evolutionary constraint pro- <br />       Nat Genet 2002, 31:60-63.                                                    teins. Nat Genet 2005, 37:351-352. <br /> 14.   Balaji S, Iyer LM, Aravind L, Babu MM: Uncovering hidden dis-        38.   Flowers JM, Sezgin E, Kumagai S, Duvernell DD, Matzkin LM, Schmidt <br />       tributed architecture scale-free transcriptional regu-                PS, Eanes WF: Adaptive evolution metabolic pathways in <br />       latory networks. J Mol Biol 2006, 360:204-212.                               Drosophila. Mol Biol Evol 2007, 24:1347-1354. <br /> 15.   Yu H, Gerstein M: Genomic analysis hierarchical struc-          39.   Yip KY, Yu H, Kim PM, Schultz M, Gerstein M: The tYNA platform <br />       ture regulatory networks. Proc Natl Acad Sci USA 2006,                    comparative interactomics: web tool managing, <br />       103:14724-14731.                                                             comparing mining multiple networks. Bioinformatics 2006, <br /> 16.   Rocha EP: The quest universals protein evolution.                 22:2968-2970. <br />       Trends Genet 2006, 22:412-416.                                         40.   Goffeau A, Barrell BG, Bussey H, Davis RW, Dujon B, Feldmann H, <br />  <br />                                                              Genome Biology 2009, 10:R35 <br />  http://genomebiology.com/2009/10/4/R35                                   Genome Biology 2009,   Volume 10, Issue 4, Article R35   Jovelin Phillips R35.10 <br />  <br />  <br />  <br />  <br />       Galibert F, Hoheisel JD, Jacq C, Johnston M, Louis EJ, Mewes HW, <br />       Murakami Y, Philippsen P, Tettelin H, Oliver SG: Life 6000 <br />       genes. Science 1996, 274:546-567. <br /> 41.   Rokas A, Williams BL, King N, Carroll SB: Genome-scale <br />       approaches resolving incongruence molecular phyloge- <br />       nies. Nature 2003, 425:798-804. <br /> 42.   Kellis M, Patterson N, Endrizzi M, Birren B, Lander ES: Sequencing <br />       comparison yeast species identify genes regu- <br />       latory elements. Nature 2003, 423:241-254. <br /> 43.   Cherry JM, Adler C, Ball C, Chervitz SA, Dwight SS, Hester ET, Jia Y, <br />       Juvik G, Roe T, Schroeder M, Weng S, Botstein D: SGD: Saccharo- <br />       myces Genome Database. Nucleic Acids Res 1998, 26:73-79. <br /> 44.   Hall TA: BioEdit: user-friendly biological sequence align- <br />       ment editor analysis program Windows 95/98/NT. <br />      <br /> </body></html>