<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Cell. Author manuscript; available PMC 2012 April 29. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Cell. 2011 April 29; 145(3): 470&#8211;482. doi:10.1016/j.cell.2011.03.037. <br />  <br />  <br />  <br />                            A High-Resolution C. elegans Essential Gene Network Based on <br />                            Phenotypic Profiling Complex Tissue <br />                            <span id='am-74' about='obo:NCBITaxon_561168' typeof='owl:Thing obo:NCBITaxon_2830 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_73026 obo:NCBITaxon_418969 obo:NCBITaxon_2759'><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-76'  ></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Rebecca" datatype="xsd:string"></span>Rebecca</span> Green, Huey-Ling Kao^,+, Anjon Audhya*,+, Swathi Arur%, Jonathan R. Mayers*, <br />                            Heidi Fridolfsson#, Monty Schulman^, Siegfried Schloissnig&#176;, Sherry Niessen&#8225;, Kimberley <br />                            Laband, Shaohe Wang, Daniel Starr#, Anthony Hyman&#176;, Tim Schedl&#167;, Arshad Desai, Fabio <br />                            Piano^, Kristin C. Gunsalus^,&#8224;, Karen Oegema&#8224; <br />                            Ludwig Institute <span id='am-30' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-31'  ></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Cancer" datatype="xsd:string"></span>Cancer</span> Research Department Cellular Molecular Medicine, <br />                            University <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="California" datatype="xsd:string"></span>California</span>, San Diego, CMM-East 3053, 9500 Gilman Drive, La Jolla, CA 92093 <br />                            *Department Biomolecular Chemistry, University Wisconsin-Madison Medical School, 1300 <br />                            University Avenue. Madison, WI 53706 <br />                            ^Center Genomics Systems Biology, Department Biology, New York University, 12 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Waverly Place, 8th Floor, New York, NY 10003 <br />                            %Department Genetics, The University Texas MD Anderson Cancer Center, 1515 <br />                            Holcombe Blvd, Unit 1010, Houston, TX 77030 <br />                            #Dept. Molecular Cellular Biology, College Biological Sciences, University California, <br />                            Davis, One Shields Avenue, Davis, CA 95616 <br />                            &#167;Department Genetics, Washington University School Medicine, Saint Louis, Missouri, <br />                            63110 Washington University, St. Louis, MO <br />                            &#176;Max Planck Institute Molecular Cell Biology Genetics, Pfotenhauerstrasse 108, 01307 <br />                            Dresden, Germany <br />                            &#8225; The Skaggs Institute Chemical Biology Department Chemical Physiology, The Center <br />                            Physiological Proteomics, The Scripps Research Institute, La Jolla, CA 92037 <br />  <br />                            SUMMARY <br />                                  High-content screening gene profiling generally limited single cells. Here, we <br />                                  explore alternative approach&#8212;profiling gene function analyzing effects gene <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  knockdowns architecture complex tissue multicellular organism. We profile 554 <br />                                  essential C. elegans genes imaging gonad architecture scoring 94 phenotypic features. To <br />                                  generate reference evaluating methods network construction, genes manually <br />                                  partitioned 102 phenotypic classes, predicting functions uncharacterized genes across <br />  <br />  <br />                            &#169; 2011 Elsevier Inc. All rights reserved. <br />                            &#8224; <br />                              Corresponding authors: Karen Oegema, Phone: (858) 534-9576, Fax: (858) 534-7750, koegema@ucsd.edu &amp; Kristin Gunsalus, <br />                            Phone: (212) 998-8236, Fax: (212) 992-4015, kcg1@nyu.edu. <br />                            +These authors contributed equally <br />                            Author Contributions: Conceived designed experiments: KO, AD, RG, <span id='am-68' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-69' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72'  ></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span>AA</span>. Performed experiments: RG, AA, SN, SA, TS, <br />                            SW, HF, DS, JM. Scored <span id='am-84' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-85'  ></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span>: AA, RG, KO. Manually partitioned genes classes: RG, KO. Reworked Phenobank to <br />                            include gonad morphology data: SS, RG, KO, AH. Developed N-Browse: HK, MS, FP, KG. Developed CSI: KO, RG, HK, MS, <br />                            FP, KG. Conceptualized performed network analysis: HK, FP, KG. Contributed reagents/materials/analysis tools: KO, RG, AA, <br />                            SN, KL, SA, TS, AD, HK, MS, KG, FP, HF, DS. <br />                            Publisher&apos;s Disclaimer: <span id='am-50' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-53'  ></span><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> PDF file unedited manuscript accepted publication. As service our <br />                            customers providing early version manuscript. The manuscript undergo copyediting, typesetting, review of <br />                            resulting proof published final citable form. Please note production process errors be <br />                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Green et al.                                                                                                  Page 2 <br />  <br />  <br />                                     diverse cellular processes. Using classification benchmark, developed robust <br />                                     computational method constructing gene networks high-content profiles based a <br />                                     network context-dependent measure ranks significance links genes. Our <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                     analysis reveals multi-parametric profiling complex tissue yields functional maps a <br />                                     resolution similar genetic interaction-based profiling unicellular eukaryotes&#8212;pinpointing <br />                                     subunits macromolecular complexes components functioning common cellular <br />                                     processes. <br />  <br />  <br />                               INTRODUCTION <br />                                               A major challenge post-genomic era translate parts lists generated genome <br />                                               sequencing maps pathways execute cellular processes. Approaches this <br />                                               combine systematic gene inhibition functional tests span continuum&#8212;from single <br />                                               readout assays complex assays interrogate broad spectrum cellular processes. <br />                                               Whereas single readout assays identify pathways impact specific process (Mathey- <br />                                               Prevot Perrimon, 2006), complex assays used construct functional networks <br />                                               collections genes diverse cellular roles. Two approaches emerged for <br />                                               distilling complex phenotypes phenotypic profiling: genetic interaction profiling and <br />                                               high-content screening. Although methodologies distinct, strategies translate the <br />                                               consequences inhibiting gene activity phenotypic profiles compared to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               generate map functional relationships genes (Boone et al., 2007; Collins et <br />                                               al., 2009; Conrad Gerlich, 2010; Piano et al., 2002; S&#246;nnichsen et al., 2005). <br />  <br />                                               Genetic interaction profiling pioneered budding yeast, using comprehensive <br />                                               deletion library non-essential genes collections hypomorphic alleles essential <br />                                               genes (Boone et al., 2007; Collins et al., 2009). Genetic interaction profiling captures the <br />                                               consequences inhibiting gene measuring effect growth rate pairwise <br />                                               inhibitions genes collection. This analysis generates <br />                                               quantitative interaction profiles gene clustered reveal functionally <br />                                               significant relationships. A genome-scale genetic interaction map recently constructed <br />                                               S. cerevisiae (Costanzo et al., 2010), maps generated subsets of <br />                                               gene implicated specific processes&#8212;such RNA processing, chromosome biology, <br />                                               proteasome function, secretory pathway (Breslow et al., 2008; Collins et al., 2007; <br />                                               Schuldiner et al., 2005; Wilmes et al., 2008). <br />  <br />                                               In metazoans, genetic interaction profiling difficult implement comprehensive <br />                                               libraries deletion/hypomorphic strains exist developing reproducible high- <br />                                               throughput methods quantify fitness formidable barrier (Gunsalus, 2008). <br />                                               Consequently, high-content screening primary method mapping functional gene <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               networks animal cells. In high-content screen, light microscopy used assess <br />                                               phenotypes arising gene inhibition RNA-mediated interference, phenotype <br />                                               captured scoring large parameter set (Conrad Gerlich, 2010). The depth the <br />                                               phenotypic profile based biological complexity assay nature and <br />                                               accuracy parameter scoring. To date, requirement high-resolution imaging has <br />                                               generally limited high-content screening single cells early embryos. <br />  <br />                                               C. elegans prototype metazoan functional mapping essential genes <br />                                               (Piano et al., 2006). C. elegans ~20,000 genes, ~2,500 essential embryo <br />                                               production viability (WormBase release WS210; Harris et al., 2010). In set of <br />                                               pioneering high-content screens, time-lapse Differential Interference Contrast (DIC) <br />                                               microscopy used film early divisions embryos following individual inhibitions <br />                                               specific subsets C. elegans genes (G&#246;nczy et al., 2000; Piano et al., 2000; Zipperlen et <br />  <br />  <br />  <br />                                                        Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                               Page 3 <br />  <br />  <br />                                           al., 2001). This extended full-genome screen generated high-content <br />                                           phenotypic profiles ~500 essential genes (S&#246;nnichsen et al., 2005). These profiles were <br />                                           combined protein-protein interaction expression profiling data create first- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           generation integrative map linked 305 essential C. elegans genes &#8220;multiple support&#65533;? <br />                                           network grouped genes modules involved specific processes including spindle <br />                                           assembly, chromosome segregation, nuclear envelope dynamics, cortical dynamics, and <br />                                           centrosome function (Gunsalus et al., 2005). Despite success studies, large <br />                                           collection essential genes profiled inhibition results sterility <br />                                           treated worm. Thus, 554 genes &#8220;sterile&#65533;? collection, control <br />                                           fundamental cellular processes membrane trafficking, translation, proteasome <br />                                           function, cortical remodeling, largely absent analysis. <br />  <br />                                           To gap analysis C. elegans essential gene set, profiled 554 sterile <br />                                           genes imaging syncytial gonad architecture high-resolution following gene knockdown <br />                                           scoring 94 phenotypic parameters. To generate reference evaluating computational <br />                                           methods network construction, genes manually partitioned 102 phenotypic <br />                                           classes, predicting functions 106 116 uncharacterized genes collection. <br />                                           Using manual classification benchmark, developed robust computational <br />                                           method constructing gene networks high-content profiles based network <br />                                           context-dependent measure ranks significance functional links genes. <br />                                           This method allowed integrate data prior high-content embryo- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           filming dataset generate network representation 818 essential C. elegans genes that <br />                                           viewed multiple levels functional resolution. <br />  <br />                               RESULTS <br />                               Phenotypic Profiling Based The Morphology Complex Tissue: C. elegans <br />                               Gonad <br />                                           Around 900 C. elegans genes required embryo production and/or early <br />                                           embryonic cell divisions (S&#246;nnichsen et al., 2005); collection includes majority of <br />                                           genes essential basic processes common cells. Because inhibition leads to <br />                                           sterility, 554 genes profiled embryo filming. Of 554 sterile <br />                                           genes, 166 unnamed, indicating prior characterization (Fig. 1A). For unnamed <br />                                           gene, determined predicted product member KOG (eukaryotic <br />                                           orthologous group; Tatusov et al., 2003) used Ensembl database determine it <br />                                           orthologs <span id='am-60' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>. Of 166 unnamed genes, 50 characterized orthologs that <br />                                           predicted function C. elegans protein (Unnamed-Group I Table S2). The <br />                                           remaining 116 members KOGs unknown function, predicted <br />                                           orthologs, multiple C. elegans paralogs (Unnamed-Group II Table S2); refer to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           116 genes &#8220;uncharacterized&#65533;? (Fig 1A). <br />  <br />                                           To profile 554 sterile genes, developed high-content assay based 3D two-color <br />                                           fluorescence confocal imaging gonad, complex tissue adult C. elegans <br />                                           hermaphrodite (Fig. 1B). The syncytial gonad contains ~1000 meiotic nuclei cup-shaped <br />                                           compartments open common cytoplasmic core. Compartments mature oocytes as <br />                                           progress distal tip proximal region gonad adjacent the <br />                                           spermatheca. Gonad maturation maintenance involves broad spectrum basic cellular <br />                                           processes (Fig. 1C), making tissue attractive substrate high-content profiling. <br />                                           Gonad architecture analyzed strain co-expressing fluorescent markers target to <br />                                           plasma membrane (GFP fusion binds PI4,5P2) chromosomes (mCherry-histone <br />                                           H2B). Hermaphrodites soaked dsRNA target gene 24 hours beginning <br />                                           late L4 larval stage, gonad achieved complement nuclei <br />  <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                           Page 4 <br />  <br />  <br />                                           (Kimble Crittenden, 2005). After 48 hours recovery, gonad architecture assessed in <br />                                           triplicate anesthetizing worms imaging gonad worm. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Binary phenotypic profiles generated scoring set 3 image stacks target <br />                                           gene 94 possible defects. The movie set gene inspected 94 <br />                                           defects (Fig. 1D; complete list examples Table S1), assigning &#8220;0&#65533;? the <br />                                           defect absent &#8220;1&#65533;? defect present 2 3 movies. All <br />                                           image stacks analyzed pair individuals, viewed scored them <br />                                           together; image stacks indexed RNA number, making analysis blind gene <br />                                           identity. In 24 cases movies consistent, experiment was <br />                                           repeated (see Fig. S1, Suppl. Experimental Procedures, Table S6 details screen <br />                                           design scoring methods). <br />  <br />                               Generation Benchmark Constructing Gene Networks Based Gonad Architecture <br />                               Phenotypes <br />                                           Initial attempts using raw parameter dataset automated clustering broadly grouped <br />                                           genes, failed partition resolution similar achieved through <br />                                           blinded manual classification experienced investigator. To develop better <br />                                           computational method, began manually partitioning genes classes generate <br />                                           reference use evaluate computational methods network construction. <br />                                           Manual partitioning placed 554 sterile genes 102 phenotypic classes (Table S2 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           contains description, sample image, gene list <span id='am-47' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span>). For organizational <br />                                           purposes, 102 classes grouped 29 broad categories (labeled A&#8211;Z, AA, AB and <br />                                           AC) contain classes sharing prominent defects. Movies class <br />                                           designations accessed Phenobank website <br />                                           (http://worm.mpi-cbg.de/phenobank_gonad). <br />  <br />                                           The manually defined phenotypic classes contained characterized genes common <br />                                           annotated molecular functions (Fig. 1E), indicating similar gonad architecture <br />                                           phenotypes reflect similar molecular functions. Manual partitioning placed 106 116 <br />                                           uncharacterized genes phenotypic classes containing characterized genes, leading to <br />                                           predictions functions. Prior using manually defined classes benchmark to <br />                                           optimize computational methods network analysis, validated functional <br />                                           predictions arising classification following 8 uncharacterized genes 5 <br />                                           classes. <br />  <br />                                           The class (E2) predicted role T09E8.1 microtubule cytoskeleton (Fig. 2A). <br />                                           Unlike class E2 genes, partial T09E8.1 inhibition did lead defects early <br />                                           embryonic cell divisions (not shown). In dividing cells, microtubule cytoskeleton is <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           organized centrosomes, gonad dominated non-centrosomal microtubule <br />                                           arrays (Zhou et al., 2009). Thus, hypothesized T09E8.1 specifically required for <br />                                           non-centrosomal microtubule array formation. Hypodermal cells, born late in <br />                                           embryogenesis, non-centrosomal microtubule arrays function nuclear <br />                                           positioning (Fridolfsson Starr, 2010). Partial T09E8.1 inhibition hypodermis led <br />                                           defects nuclear positioning (Fig. 2B), ~3 fold reduction number and <br />                                           increase length GFP-EBP-1 &#8220;comets&#65533;? mark growing microtubule ends <br />                                           (Fig. 2C). We conclude T09E8.1, noca-1 (for non-centrosomal array), <br />                                           plays important role organization non-centrosomal microtubule arrays the <br />                                           gonad hypodermis. <br />  <br />                                           The second class (G1) predicted role F54D12.5 DAF-21, hsp90-<span id='am-27' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-28' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> <br />                                           chaperone, MAPK signaling pathway (Fig. 2D). We tested predictions using a <br />                                           genetic test. Worms homozygous reduction-of-function allele mpk-1(ga111) exhibit <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                           Page 5 <br />  <br />  <br />                                           normal gonad morphology, phosphorylated MPK-1 levels, readout for <br />                                           pathway activity, reduced compared controls (Fig. 2E,F; Lee et al., 2007). Partial <br />                                           knockdown daf-21 F54D12.5, conditions did result morphological <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           phenotype control worms, led strong MAPK knockdown phenotype the <br />                                           mpk-1(ga111) background (Fig. 2E,F; Fig. S2, Table S3). RNAi daf-21 reduced <br />                                           phosphorylated MPK-1 levels control let-60 gain-of-function worms (Fig. 2E,F; <br />                                           Table S3), indicating DAF-21 acts level downstream LET-60/Ras the <br />                                           MAPK pathway (Fig. 2G). F54D12.5 RNAi did reduce phosphorylated MPK-1 levels, <br />                                           F54D12.5 contains 2 potential MPK-1 docking sites suggesting MPK-1 <br />                                           substrate (Fig. 2G). We conclude DAF-21 F54D12.5 function different points in <br />                                           MAPK signaling pathway F54D12.5, eom-1, enhancer mpk-1(ga111). <br />  <br />                                           The class (S2) predicted roles uncharacterized proteins anaphase- <br />                                           promoting complex/cyclosome (APC/C; Fig. 3A). Immuno-affinity purification one <br />                                           uncharacterized gene product, K10D2.4, C. elegans extracts followed mass <br />                                           spectrometry recovered seven APC components (Fig. 3B). The product another <br />                                           uncharacterized Class S2 gene, C09H10.7, recovered&#8212;indicating both <br />                                           K10D2.4 C09H10.7 APC/C subunits. K10D2.4 recently identified a <br />                                           metazoan-specific component APC/C (Hubner et al., 2010; Hutchins et al., 2010; <br />                                           Kops et al., 2010) gene named emb-1/apc-16. We C09H10.7, apc-17. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           The fourth class (F2) predicted role BTB-domain containing protein C08C3.4 in <br />                                           cortical remodeling/cytokinesis (Fig. 3C). A GFP fusion C08C3.4 localized the <br />                                           contractile ring tip cleavage furrow dividing embryos (Fig. 3D), embryos <br />                                           partially depleted C08C3.4 exhibited cytokinesis defects (Fig. 3E). We conclude that <br />                                           C08C3.4 required cortical remodeling gonad cytokinesis embryos and <br />                                           gene cyk-7. <br />  <br />                                           The fifth class (I2) phenotype included debris labeled plasma membrane probe, <br />                                           suggesting role membrane trafficking (Fig. 3F). We tested trafficking function by <br />                                           imaging compartment boundaries strain co-expressing mCherry labeled plasma <br />                                           membrane probe GFP fusion SNB-1, SNARE trafficked the <br />                                           endomembrane delivered plasma membrane (Fig. 3G). Compartment <br />                                           boundaries control worms SNB-1-GFP plasma membrane probe are <br />                                           yellow. Trafficking defects prevent SNB-1-GFP reaching plasma membrane, <br />                                           leading red compartment boundaries. 8 15 Class I2 genes, including F27C8.6 and <br />                                           T01B7.6, exhibited defects SNB-1 assay (Fig. 3G). We F27C8.6 T01B7.6 <br />                                           trcs-1 trcs-2, respectively, (transport cell surface). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           The follow work classes demonstrates gonad architecture sufficient <br />                                           resolution functionally classify genes broad spectrum essential cellular <br />                                           functions. It validates manual classification, establishing benchmark for <br />                                           evaluating computational methods network construction. <br />  <br />                               Development Network Context-Dependent Method Evaluate Significance of <br />                               Functional Links Between Genes Based High-Content Phenotypic Profiles <br />                                           Our initial efforts using automated clustering unable partition genes resolution <br />                                           comparable achieved experienced investigator. To circumvent this <br />                                           limitation, used manually-defined classes tool develop robust computational <br />                                           method constructing gene networks based high-content parameter profiles. We first <br />                                           compared phenotypic profiles calculating Pearson&#8217;s Correlation Coefficient <br />                                           (PCC) pair genes. The resulting network visualized using N-Browse, an <br />                                           interactive Java-based tool (Kao Gunsalus, 2008), display connections genes <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                              Page 6 <br />  <br />  <br />                                           profiles correlated PCC greater equal specified threshold <br />                                           (Fig. 4A,B; dark blue lines connecting grey gene nodes). To assess effectiveness this <br />                                           approach, circled gene clusters corresponded manually-defined classes. This <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           approach revealed optimal PCC threshold viewing functionally relevant <br />                                           connections (red outlined boxes Fig. 4B) varied substantially different network <br />                                           neighborhoods. This variability varying nature profiled phenotypes and <br />                                           extent captured parameter set, extent scored <br />                                           parameters related versus independent, fact profiles features often <br />                                           exhibit variance. Viewing entire network single PCC threshold possible <br />                                           regions threshold low view cluttered non-specific <br />                                           connections (Fig 4B, images right red boxed images), regions the <br />                                           threshold high, yielding network meaningful connections are <br />                                           absent (Fig. 4B, images left red boxed images). <br />  <br />                                           To circumvent limitations PCC-based analysis, developed measure ranks the <br />                                           significance functional links genes based network context. For pair of <br />                                           genes A B, assigned Connection Specificity Index (CSI) by: 1) calculating the <br />                                           PCC connections A B genes dataset, 2) <br />                                           counting number genes connected A B PCC &#8805; PCCAB &#956; 0.05 (i.e. level <br />                                           comparable better correlation A B, correlations PCC to <br />                                           0.05 PCCAB considered similar&#8212;this offset determined empirically); 3) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           dividing number total number genes screen (554); 4) subtracting the <br />                                           result 1.0 (Fig. 4C). The CSI equivalent fraction genes dataset whose <br />                                           profiles similar A B profile A profile B. For <br />                                           example, CSI 0.97 means similarity A B highly specific: only <br />                                           ~3% gene knockdowns profiles comparable higher similarity A or <br />                                           B. Since CSI scales uniformly functional significance entire network, <br />                                           connections similar level significance simultaneously displayed single <br />                                           CSI threshold (Fig. 4D). <br />  <br />                                           We evaluated performance CSI PCC threshold networks comparing ability <br />                                           automated clustering algorithm (MINE, Module Identification NEtworks; K. <br />                                           Rhrissorrakrai K.C. Gunsalus, press) identify gene clusters corresponding our <br />                                           manually-defined classes. This analysis revealed useful range two <br />                                           parameters, networks generated using CSI thresholds ~3-fold fewer connections than <br />                                           comparable PCC networks; noise reduction translates substantial improvement in <br />                                           ability automated algorithm identify functionally relevant gene clusters (Fig. <br />                                           4E,F; Fig. S3). Approximately half manually-defined classes containing 4 more <br />                                           genes identified automated clustering network constructed using single <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           CSI threshold 0.96, 14% identified comparable PCC network <br />                                           (Fig. 4F). When MINE allowed search networks spanning range CSI thresholds <br />                                           (0.90 0.99), clusters corresponding ~90% manually-defined classes be <br />                                           identified, compared 65% networks spanning range PCC thresholds (0.5&#8211;1). Heat <br />                                           map dendograms revealed sharply defined clusters using CSI, indicating <br />                                           CSI increases network clarity (Fig. 4G; Table S4). <br />  <br />                                           We conclude constructing networks using CSI thresholds circumvents variability in <br />                                           significance strict correlation measure different network regions arises to <br />                                           complexity high-content screening parameters. Using CSI instead PCC <br />                                           reduces connection noise allows connections similar functional significance be <br />                                           viewed entire network single threshold. <br />  <br />  <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                          Page 7 <br />  <br />  <br />                               Varying CSI Threshold Reveals Functional Modularity Gene Network Different <br />                               Levels Resolution <br />                                           The CSI-based network representation allows exploration functional modularity at <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           different levels resolution. This point illustrated region gene network <br />                                           involved protein production (Fig. 5A): relatively low threshold CSI 0.90 connects the <br />                                           entire set genes involved protein translation, mRNA splicing, protein folding a <br />                                           dense meshwork. An intermediate CSI threshold 0.93 results sparser network more <br />                                           specific connections defines smaller gene groups. At high CSI threshold 0.97, the <br />                                           chaperonins, tRNA synthetases, translation initiation/elongation factors, small ribosome <br />                                           subunits, large ribosome subunits, splicing factors resolved separate clusters. <br />                                           Thus, dialing CSI reveals functional relationships different levels of <br />                                           resolution. The automated clusters identified MINE different CSI thresholds are <br />                                           provided supplement (Table S5; note genes multiple clusters). <br />  <br />                                           We assessed resolution limit gonad architecture assay using region the <br />                                           network representing genes involved protein degradation (Fig. 5B, Fig. S4). Above the <br />                                           high CSI threshold 0.99, connections remained linked genes specific <br />                                           proteasome subcomplexes (the Lid, core &#946;-ring, core &#945;-ring 19S ATPase base; <br />                                           Fig. 5B). We conclude phenotypic profiling based complex tissue architecture, <br />                                           coupled automated construction CSI-based networks parameter profiles, is <br />                                           capable correctly assigning fine distinctions protein function. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               CSI-Enabled Integration High-Content Data Sets Generates Global View C. <br />                               elegans Essential Gene Network <br />                                           Together, gonad architecture (554 genes) time-lapse embryo filming (661 genes) <br />                                           screens provide high-content profiles 885 essential genes (330 profiled both <br />                                           screens). The CSI ideally suited integrating datasets filters low <br />                                           specificity phenotypic links, leading network combines significant <br />                                           relationships identified screens. We binarized phenotypic signatures the <br />                                           embryo timelapse screen, based scoring 45 possible defects (S&#246;nnichsen <br />                                           et al., 2005), calculated CSI gene pair, simultaneously displayed data <br />                                           sets N-Browse create integrated network. The merged network, network regions <br />                                           centered genes interest, viewed CSI threshold N-Browse (instructions <br />                                           demo video describing use N-Browse accessed at <br />                                           http://worm.mpi-cbg.de/phenobank_gonad/nbrowse). At CSI threshold 0.96, the <br />                                           integrated network 3382 high significance connections linking 818 genes (Fig. 6A&#8211;C). <br />                                           The combined view provides functional information ~90% genes required for <br />                                           embryo production embryonic divisions. Despite presence 330 genes <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           common, relationships identified datasets highly orthogonal only&#8212;23 <br />                                           total 3382 phenotypic links shared (Fig. 6A). This lack overlap explained by <br />                                           majority 330 common genes failing make high-significance connections the <br />                                           embryo time-lapse screen sterility onset (Fig. S5). Overall, CSI-based network <br />                                           integrates information acquired biologically distinct contexts gonad embryo <br />                                           provide multi-layered view function 818 genes required essential cellular <br />                                           processes multicellular organism. <br />  <br />                               DISCUSSION <br />                                           High-content screening primary method mapping functional gene networks in <br />                                           animal cells (Collins et al., 2009; Conrad Gerlich, 2010; Gunsalus, 2008). A number of <br />                                           high-content screens classified genes based cell morphology following gene <br />                                           knockdown (examples include Bakal et al., 2007; Echard et al., 2004; Eggert et al., 2004; <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                             Page 8 <br />  <br />  <br />                                           Goshima et al., 2007; Liu et al., 2009; Neumann et al., 2010; Piano et al., 2002; S&#246;nnichsen <br />                                           et al., 2005). While successful identifying genes contribute cell division and <br />                                           morphology, ability screens assess protein function breadth of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           cellular processes falls far short attained using genetic interaction profiling <br />                                           <span id='am-18' about='obo:NCBITaxon_4751' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-19'  ></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="fungi" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="PMID:11062127" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="PMID:12684019" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Fungi" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="fungi" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>fungi</span>. Screen resolution&#8212;the ability discriminate different biological <br />                                           functions&#8212;depends information content phenotypic assay; therefore, one <br />                                           approach enhance resolution time-lapse imaging (Conrad Gerlich, 2010). <br />                                           Here, explored alternative approach&#8212;analyzing effects gene knockdowns on <br />                                           complex tissue architecture single timepoint multicellular organism. We monitored <br />                                           C. elegans gonad architecture following gene knockdown scoring 94 phenotypic features. <br />                                           Our results demonstrate biological complexity gonad translates into <br />                                           phenotypes enable profiling wide spectrum essential cellular processes a <br />                                           resolution approaching genetic interaction profiling yeasts. <br />  <br />                               The Sterile Gene Collection: A Prominent Gap Functional Genomic Analysis the <br />                               C. elegans Essential Gene Set <br />                                           Sequencing C. elegans genome discovery RNAi catalyzed efforts to <br />                                           systematically catalog functions predicted genes (Piano et al., 2006). C. elegans <br />                                           ~900 essential genes required embryo production and/or events two <br />                                           embryonic divisions. A subset genes previously characterized time-lapse <br />                                           imaging early embryos (Fraser et al., 2000; G&#246;nczy et al., 2000; Piano et al., 2002; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           S&#246;nnichsen et al., 2005). However, sterility onset following RNAi major complication <br />                                           precluded in-depth characterization large number genes. Our efforts focused on <br />                                           554 sterile genes, profiled imaging gonad architecture following RNAi <br />                                           knockdowns generating parameterized description resulting phenotypes. <br />  <br />                                           Our analysis generated functional predictions 106 116 uncharacterized sterile <br />                                           genes. These predictions span variety cellular processes including membrane <br />                                           trafficking, glycosylation, fatty acid synthesis, cortical dynamics, chromosome structure/ <br />                                           segregation, RNA splicing, mitochondrial function, microtubule cytoskeleton, proteasome <br />                                           function, translation, MAPK signaling, transcription, RNA binding, metaphase- <br />                                           anaphase transition. We validated predictions 8 genes play important roles in <br />                                           microtubule cytoskeleton, MAP kinase signaling, metaphase-anaphase transition, <br />                                           cortical remodeling/cytokinesis, membrane trafficking. Although gonad used as <br />                                           &#8220;test tube&#65533;? functionally profile genes, subsequent work early later stage <br />                                           embryos indicates majority essential genes collection broadly <br />                                           important. Of 554 sterile genes, ~60% predicted homologs higher organisms, and <br />                                           50% 106 uncharacterized genes predicted homolog conserved <br />                                           domain. Thus, addition filling large gap analysis C. elegans essential <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           gene set, findings provide starting point address functions related genes in <br />                                           humans. <br />  <br />                               High-Resolution Phenotypic Profiling Based Morphology Complex Tissue <br />                                           While original motivation screen profile sterile gene set, fortuitous <br />                                           byproduct realization complex tissue, structure depends the <br />                                           dynamic coordination broad spectrum cellular processes, ideal substrate for <br />                                           phenotypic profiling. A major finding work profiles based complex tissue <br />                                           architecture acquired single time point following gene knockdown greater <br />                                           information depth profiles derived timelapse imaging divisions of <br />                                           early embryo (a single cell). For example, shown Figure 5, gonad data partitions <br />                                           set genes involved protein production 15 different classes (chaperones, <br />                                           ribosome components, tRNA synthetases, proteins involved protein folding, splicing <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                             Page 9 <br />  <br />  <br />                                           components etc.) distinctions based embryo filming <br />                                           data. This high level resolution generally holds dataset. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           We note means inferring function single point morphology assay distinct <br />                                           used embryo-filming screen. In embryo screen, gene function was <br />                                           inferred directly phenotype. For example, defects spindle assembly, chromosome <br />                                           segregation, polarity establishment led assignment classes implicated the <br />                                           corresponding processes (S&#246;nnichsen et al., 2005). By contrast, link cataloged <br />                                           phenotypic features gonad architecture assay gene function direct&#8212;we do <br />                                           know knockdown genes specific functions lead specific spectrum of <br />                                           phenotypic features. Instead, protein function inferred comparing profile of <br />                                           phenotypic features genes dataset, process conceptually analogous <br />                                           function inferred genetic interaction profiles budding yeast. <br />  <br />                                           The presence proteasome components dataset offered opportunity compare <br />                                           resolution approach genetic interaction profiling yeast. The proteasome a <br />                                           2.5-MDa macro-molecular machine contains 30 different subunits organized a <br />                                           20S core complex, composed &#945; &#946; rings, 19S regulatory particle, containing base <br />                                           lid sub-complexes (Bochtler et al., 1999). In yeast, distinct genetic interaction profiles <br />                                           obtained subunits proteasome sub-complexes using sensitive <br />                                           competitive growth assay (Breslow et al., 2008). At CSI &#8804; 0.99 (a high-stringency filter), <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           connections remained subunits specific proteasome subcomplexes <br />                                           (&#945; ring, &#946; ring, lid base; Fig. 5B). This observation indicates phenotypic profiles based <br />                                           gonad morphology provide resolution sufficient group subunits specific proteasome <br />                                           subcomplexes. <br />  <br />                                           At core dataset profiles composed parameters visually scored by <br />                                           experienced investigators acquired automated image analysis. Given the <br />                                           complexity substrate&#8212;a 3 dimensional tissue living organism variably <br />                                           positioned worm&#8212;and large spectrum knockdown phenotypes entirely <br />                                           change properties structure (gonad size, shape, position, compartment nuclei <br />                                           number morphology) automated parameter scoring exceedingly <br />                                           difficult. However, exact properties make automated analysis difficult&#8212;the varied <br />                                           dramatic effects gene knockdowns gonad architecture&#8212;are the <br />                                           properties assay profiling power. Although manual parameter scoring could <br />                                           introduce <span id='am-7' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Bias" datatype="xsd:string"></span>bias</span>, minimized performing analysis blinded gene identity <br />                                           fact individual parameters scored investigators largely <br />                                           oblivious larger patterns ultimately emerge. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               A Robust Computational Method Constructing Gene Networks From High-Content <br />                               Screening Data <br />                                           By evaluating computational methods network construction using validated manual <br />                                           classification rich phenotypic dataset, able devise robust computational <br />                                           method constructing gene networks high-content phenotypic profiles. This method <br />                                           overcomes challenges encountered network analysis high-content datasets. The <br />                                           challenge level profile correlation significant varies different <br />                                           network regions varying nature profiled phenotypes extent which <br />                                           captured parameter set. The second challenge commonly encountered <br />                                           (and informative) phenotypes generate &#8220;hairball&#65533;? connections that <br />                                           obscures meaningful functional links. At center method developed to <br />                                           overcome challenges simple metric&#8212;the CSI, network context- <br />                                           dependent measure ranks significance functional links genes. Compared <br />                                           Pearson&#8217;s Correlation Coefficient, constructing networks based CSI reduces <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                        Page 10 <br />  <br />  <br />                                           non-specific connection noise, improves network clarity, allows connections similar <br />                                           functional significance simultaneously viewed entire network single <br />                                           threshold. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Ranking connection significance allows exploration gene network different levels <br />                                           functional resolution integration high-content screening data different <br />                                           sources. We demonstrate usefulness CSI using integrate high-content <br />                                           data gonad architecture screen prior embryo-filming screen, to <br />                                           generate integrated network provides multi-layered view 818 genes C. <br />                                           elegans essential gene set. The phenotypic profiles datasets composed of <br />                                           parameters visually scored measured automated image <br />                                           analysis. However, constructing networks based phenotypic profiles faces same <br />                                           challenges, regardless parameters scored manual automated <br />                                           means. Consequently, anticipate CSI-based method described of <br />                                           equal utility analyzing integrating datasets composed parameters acquired through <br />                                           automated analysis. <br />  <br />                               EXPERIMENTAL PROCEDURES <br />                               C. elegans Strains <br />                                           Strains listed Table S7. The strains OD95, UD299, DP38, OD70, MSN142, NL2098, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           BS3623 previously described (Arur et al., 2009; Essex et al., 2009, Fridolfsson and <br />                                           Star, 2010; Kachur et al., 2008; Maduro Pilgrim, 1995; Shi et al., 2010; Sijen et al., <br />                                           2001). OD447 generated using PDS-1000/He Biolistic Particle Delivery System <br />                                           (Bio-Rad Laboratories; Praitis et al., 2001) bombard construct containing C08C3.4 <br />                                           genomic locus cloned SpeI site pIC26 (Cheeseman et al., 2004) DP38. <br />                                           OD449 generated mating OD447 OD70. <br />  <br />                               RNA Production <br />                                           Templates dsRNA production generated using primers tails containing the <br />                                           T3 T7 promoters amplify amplify 500&#8211;1000 bp region corresponding gene <br />                                           genomic DNA. When possible, oligo pairs used S&#246;nnichsen et al. (2005) were <br />                                           chosen. New oligos designed genes S&#246;nnichsen screen the <br />                                           S&#246;nnichsen oligo pairs amplified introns regions smaller larger 500&#8211;1000 bp <br />                                           (oligos listed Table S2). PCR reactions cleaned (PerfectPrep 96 kit; Eppendorf) <br />                                           used templates T3 T7 transcription reactions (Megascript T3 T7 kits, <br />                                           Ambion). Transcription reactions mixed, cleaned (MegaClear96 kit, Ambion), and <br />                                           annealed adding 3x Soaking buffer (32.7 mM Na2HPO4, 16.5 mM KH2PO4, 6.3 mM <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           NaCl, 14.1 mM NH4Cl) final concentration 1X incubating reactions 68&#176;C <br />                                           10 minutes followed 37&#176;C 30 minutes. <br />  <br />                               RNA Interference <br />                                           Larval (L4 stage) worms strain OD95 rinsed M9 soaked 5 &#956;l of <br />                                           dsRNA (supplemented 0.5 &#956;l 50/50 mixture 63 mM spermidine 1.1 % <br />                                           gelatin) 24 hrs 20 &#176;C inside humid chamber. Worms transferred NGM plates <br />                                           seeded OP50 E. coli allowed recover 20&#176;C 48 hrs prior imaging. <br />  <br />                               Live Imaging <br />                                           Six worms anesthetized fresh mixture 1 mg/ml Tricane (ethyl 3-aminobenzoate <br />                                           methanesulfonate salt) 0.1 mg/ml tetramisole hydrochloride (TMHC) dissolved in <br />                                           M9 15&#8211;30 minutes transferring agarose pad coverslip for <br />                                           imaging. Gonads imaged collecting 80 x 0.5&#956;m z-series including DIC, GFP and <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                        Page 11 <br />  <br />  <br />                                           RFP images z-plane. Microscopy performed spinning disk confocal <br />                                           mounted Nikon TE2000-E inverted microscope equipped 60x 1.4 NA <br />                                           PlanApochromat <span id='am-12' about='obo:NCBITaxon_3863' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_163743 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14'  ></span><span id='am-15' property="rdfs:label" content="Lens" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Lens Mill." datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>lens</span>, krypton-<span id='am-63' about='obo:NCBITaxon_354775' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_40096 obo:NCBITaxon_40092 obo:NCBITaxon_33208 obo:NCBITaxon_40093 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="Argon" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-67'  ></span>argon</span> 2.5 W water-cooled <span id='am-79' about='obo:NCBITaxon_62990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_241799 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_241789 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="Laser" datatype="xsd:string"></span><span id='am-83'  ></span>laser</span> (Spectra-Physics, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Mountain View, CA) Hamamatsu Orca ER CCD <span id='am-55' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-56'  ></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="Camera" datatype="xsd:string"></span>camera</span>. Acquisition parameters, <br />                                           shutters, focus controlled MetaMorph software (Molecular Devices, <br />                                           Downington, PA). <br />  <br />                               Sterility Assessment <br />                                           At time worms cohort anesthetized imaged, 3 worms <br />                                           moved fresh plate. After 24 hours 20&#176;C, number embryos L1 progeny <br />                                           plate counted. A gene &#8220;sterile&#65533;? embryos L1 larvae observed, <br />                                           &#8220;partially sterile&#65533;? &lt;50 embryos/L1 worms present, &#8220;wild-type&#65533;? &gt;50 embryos/ <br />                                           L1 worms present. After additional 24 hours, assessed knockdown <br />                                           resulted embryonic lethality. If &gt;50 L1-L2 larvae present, considered <br />                                           &#8220;viable&#65533;?, &lt;50 L1-L2 larvae present &gt;5 dead embryos, was <br />                                           considered &#8220;embryonic lethal&#65533;?. <br />  <br />                               Antibody Production <br />                                           Antibodies K10D2.4 generated using oligos <br />                                           cgcgcgggatccgctttgatgtacccattcca gcgcgcgaattctcaaggatttgcaggcatattt amplify region <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           encoding amino acids 2&#8211;81 cDNA library. The product digested Bam HI/ <br />                                           EcoRI cloned pGEX6P-1 digested enzymes (GE Healthcare Life <br />                                           Sciences). The purified GST fusion protein outsourced injection rabbits <br />                                           (Covance). Antibodies affinity purified binding columns antigen <br />                                           removal GST tag described previously (Desai et al., 2003). <br />  <br />                               Immunoprecipitation Mass Spectrometry <br />                                           K10D2.4 immunoprecipiated extracts prepared frozen worm pellets (Desai et <br />                                           al., 2003) mass spectrometry performed (Cheeseman et al., 2004) described, <br />                                           using recent version predicted C. elegans proteins (Wormprep111). <br />  <br />                               Hypodermal Cell Experiments, MAP Kinase Experiments, Membrane Trafficking Assay <br />                                           For Figure 2C, embryos produced adult hermaphrodites strain UD299 were <br />                                           imaged described (Fridolfson Starr, 2010), 20 hours injection dsRNA against <br />                                           T09E8.1 gonad. Nuclear migration scored L1 larvae using DIC optics as <br />                                           described (Starr et al., 2001). The attenuated RNAi, gonad dissection, immunofluorescence, <br />                                           gonad analysis Figure 2E F (Arur et al., 2009) GFP-SNB-1 trafficking <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           assay (Shi et al., 2010) performed described. <br />  <br />                               Phenobank N-Browse <br />                                           Movies accessed Phenobank website <br />                                           (http://worm.mpi-cbg.de/phenobank_gonad) RNAiDB (http://www.rnai.org). <br />                                           Written instructions demo video explaining search Phenobank at <br />                                           http://worm.mpi-cbg.de/phenobank_gonad/project. Written instructions demo video <br />                                           describing visualize gene network N-Browse2 at <br />                                           http://worm.mpi-cbg.de/phenobank_gonad/nbrowse. As described instructions, gonad <br />                                           embryo data viewed N-Browse2 entering URL <br />                                           http://gnetbrowse.org/gonad.jnlp browser providing password <br />                                           &#8220;phenotypes&#65533;? prompt. Additional information N-Browse, including system <br />                                           requirements tutorials, http://www.gnetbrowse.org. <br />  <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                                       Page 12 <br />  <br />  <br />                               Automated clustering using MINE <br />                                           MINE (Module Identification Networks) graph clustering algorithm shows <br />                                           comparable better performance compared similar algorithms identifying <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           functional modules dense interaction networks (K. Rhrissorrakrai K.C. Gunsalus, in <br />                                           press). MINE, similar MCODE (Bader Hogue, 2003) uses modified <br />                                           weighting scheme takes account network modularity, available Cytoscape <br />                                           plug-in http://www.cytoscape.org Perl package (upon request authors). <br />  <br />                               Supplementary Material <br />                                           Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                           This work supported grants American Cancer Society (PF-06-254-01-CCG, RG), Helen Hay <br />                                           Whitney Foundation (to AA), NIH (R01 GM074207 KO, R01 HD046236, KCG FP; R01 <br />                                           GM085503, KCG; R01 GM085150 TS; R01 GM088151 AA) funding Ludwig Institute for <br />                                           Cancer Research KO AD. We thank Kahn Rhrissorrakrai MINE algorithm, Michael Volkmer for <br />                                           movie formatting, <span id='am-42' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-43' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-45'  ></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Caenorhabditis</span> Genetics Center, funded NIH National Center Research <br />                                           Resources, strains. <br />  <br />  <br />                               References <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Arur S, Ohmachi M, Nayak S, Hayes M, Miranda A, Hay A, Golden A, Schedl T. Multiple ERK <br />                                             substrates execute single biological processes <span id='am-35' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Caenorhabditis elegans</span> germ-line development. <br />                                             Proc Natl Acad Sci U S A. 2009; 106:4776&#8211;4781. [PubMed: 19264959] <br />                                           Bader GD, Hogue CW. An automated method finding molecular complexes large protein <br />                                             interaction networks. BMC Bioinformatics. 2003; 4:2. [PubMed: 12525261] <br />                                           Bakal C, Aach J, Church G, Perrimon N. Quantitative morphological signatures define local signaling <br />                                             networks regulating cell morphology. Science. 2007; 316:1753&#8211;1756. [PubMed: 17588932] <br />                                           Bochtler M, Ditzel L, Groll M, Hartmann C, Huber R. The proteasome. Annu Rev Biophys Biomol <br />                                             Struct. 1999; 28:295&#8211;317. [PubMed: 10410804] <br />                                           Boone C, Bussey H, Andrews BJ. Exploring genetic interactions networks yeast. Nat Rev <br />                                             Genet. 2007; 8:437&#8211;449. [PubMed: 17510664] <br />                                           Boxem M, Maliga Z, Klitgord N, Li N, Lemmens I, Mana M, Lichtervelde L, Mul JD, van Peut <br />                                             D, Devos M, et al. A protein domain-based interactome network C. elegans early <br />                                             embryogenesis. Cell. 2008; 134:534&#8211;545. [PubMed: 18692475] <br />                                           Breslow DK, Cameron DM, Collins SR, Schuldiner M, Stewart-Ornstein J, Newman HW, Braun S, <br />                                             Madhani HD, Krogan NJ, Weissman JS. A comprehensive strategy enabling high-resolution <br />                                             functional analysis yeast genome. Nat Methods. 2008; 5:711&#8211;718. [PubMed: 18622397] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Cheeseman IM, Niessen S, Anderson S, Hyndman F, Yates JR 3rd, Oegema K, Desai A. A conserved <br />                                             protein network controls assembly outer kinetochore ability sustain tension. Genes <br />                                             Dev. 2004; 18:2255&#8211;2268. [PubMed: 15371340] <br />                                           Collins SR, Miller KM, Maas NL, Roguev A, Fillingham J, Chu CS, Schuldiner M, Gebbia M, Recht <br />                                             J, Shales M, et al. Functional dissection protein complexes involved yeast chromosome <br />                                             biology using genetic interaction map. Nature. 2007; 446:806&#8211;810. [PubMed: 17314980] <br />                                           Collins SR, Weissman JS, Krogan NJ. From information knowledge: new technologies defining <br />                                               gene function. Nat Methods. 2009; 6:721&#8211;723. [PubMed: 19953683] <br />                                           Conrad C, Gerlich DW. Automated microscopy high-content RNAi screening. J Cell Biol. 2010; <br />                                               188:453&#8211;461. [PubMed: 20176920] <br />                                           Costanzo M, Baryshnikova A, Bellay J, Kim Y, Spear ED, Sevier CS, Ding H, Koh JL, Toufighi K, <br />                                               Mostafavi S, et al. The genetic landscape cell. Science. 2010; 327:425&#8211;431. [PubMed: <br />                                               20093466] <br />  <br />  <br />  <br />  <br />                                                     Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                                  Page 13 <br />  <br />  <br />                                           Desai A, Rybina S, Muller-Reichert T, Shevchenko A, Hyman A, Oegema K. KNL-1 directs assembly <br />                                               microtubule-binding interface kinetochore C. elegans. Genes Dev. 2003; 17:2421&#8211; <br />                                               2435. [PubMed: 14522947] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Echard A, Hickson GR, Foley E, O&#8217;Farrell PH. Terminal cytokinesis events uncovered RNAi <br />                                               screen. Curr Biol. 2004; 14:1685&#8211;1693. [PubMed: 15380073] <br />                                           Eggert US, Kiger AA, Richter C, Perlman ZE, Perrimon N, Mitchison TJ, Field CM. Parallel chemical <br />                                               genetic genome-wide RNAi screens identify cytokinesis inhibitors targets. PLoS Biol. <br />                                               2004; 2:e379. [PubMed: 15547975] <br />                                           Essex A, Dammermann A, Lewellyn L, Oegema K, Desai A. Systematic analysis Caenorhabditis <br />                                               elegans reveals spindle checkpoint composed largely independent branches. Mol. <br />                                               Biol. Cell. 2009; 20:1252&#8211;1267. <br />                                           Fraser AG, Kamath RS, Zipperlen P, Martinez-Campos M, Sohrmann M, Ahringer J. Functional <br />                                               genomic analysis C. elegans chromosome I systematic RNA interference. Nature. 2000; <br />                                               408:325&#8211;330. [PubMed: 11099033] <br />                                           G&#246;nczy P, Echeverri C, Oegema K, Coulson A, Jones SJ, Copley RR, Duperon J, Oegema J, Brehm <br />                                               M, Cassin E, et al. Functional genomic analysis cell division C. elegans using RNAi genes <br />                                               chromosome III. Nature. 2000; 408:331&#8211;336. [PubMed: 11099034] <br />                                           Fridolfsson HN, Starr DA. Kinesin-1 dynein nuclear envelope mediate bidirectional <br />                                               migrations nuclei. J Cell Biol. 2010; 191:115&#8211;128. [PubMed: 20921138] <br />                                           Goshima G, Wollman R, Goodwin SS, Zhang N, Scholey JM, Vale RD, Stuurman N. Genes required <br />                                               mitotic spindle assembly Drosophila S2 cells. Science. 2007; 316:417&#8211;421. [PubMed: <br />                                               17412918] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Gunsalus KC. A Caenorhabditis elegans genetic-interaction map wiggles view. J Biol. 2008; 7:8. <br />                                               [PubMed: 18341704] <br />                                           Gunsalus KC, Ge H, Schetter AJ, Goldberg DS, Han JD, Hao T, Berriz GF, Bertin N, Huang J, <br />                                               Chuang LS, et al. Predictive models molecular machines involved Caenorhabditis elegans <br />                                               early embryogenesis. Nature. 2005; 436:861&#8211;865. [PubMed: 16094371] <br />                                           Harris TW, Antoshechkin I, Bieri T, Blasiar D, Chan J, Chen WJ, De La Cruz N, Davis P, Duesbury <br />                                               M, Fang R, et al. WormBase: comprehensive resource nematode research. Nucleic Acids <br />                                               Res. 2010; 38:D463&#8211;467. [PubMed: 19910365] <br />                                           Hubner NC, Bird AW, Cox J, Splettstoesser B, Bandilla P, Poser I, Hyman A, Mann M. Quantitative <br />                                               proteomics combined BAC TransgeneOmics reveals vivo protein interactions. J Cell Biol. <br />                                               2010; 189:739&#8211;754. [PubMed: 20479470] <br />                                           Hutchins JR, Toyoda Y, Hegemann B, Poser I, Heriche JK, Sykora MM, Augsburg M, Hudecz O, <br />                                               Buschhorn BA, Bulkescher J, et al. Systematic Analysis Human Protein Complexes Identifies <br />                                               Chromosome Segregation Proteins. Science. 2010; 328:593&#8211;599. [PubMed: 20360068] <br />                                           Kachur TM, Audhya A, Pilgrim DB. UNC-45 required NMY-2 contractile function early <br />                                               embryonic polarity establishment germline cellularization C. elegans. Dev. Biol. 2008; <br />                                               314:287&#8211;299. <br />                                           Kimble, J.; Crittenden, SL. WormBook. Germline proliferation control. The C. elegans <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Research Community, WormBook. August 15. 2005 http://www.wormbook.org <br />                                           Kops GJ, Voet MV, Manak MS, van Osch MH, Naini SM, Brear A, McLeod IX, Hentschel DM, Yates <br />                                               JR 3rd, van den Heuvel S, et al. APC16 conserved subunit anaphase-promoting <br />                                               complex/cyclosome. J Cell Sci. 2010; 123:1623&#8211;1633. [PubMed: 20392738] <br />                                           Lee MH, Ohmachi M, Arur S, Nayak S, Francis R, Church D, Lambie E, Schedl T. Multiple functions <br />                                               dynamic activation MPK-1 extracellular signal-regulated kinase signaling Caenorhabditis <br />                                               elegans germline development. Genetics. 2007; 177:2039&#8211;2062. [PubMed: 18073423] <br />                                           Li S, Armstrong CM, Bertin N, Ge H, Milstein S, Boxem M, Vidalain PO, Han JD, Chesneau A, Hao <br />                                               T, et al. A map interactome network metazoan C. elegans. Science. 2004; 303:540&#8211; <br />                                               543. [PubMed: 14704431] <br />                                           Liu T, Sims D, Baum B. Parallel RNAi screens different cell lines identify generic cell <br />                                               type-specific regulators actin organization cell morphology. Genome Biol. 2009; 10:R26. <br />                                               [PubMed: 19265526] <br />  <br />  <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                                Page 14 <br />  <br />  <br />                                           Maduro M, Pilgrim D. Identification cloning unc-119, gene expressed Caenorhabditis <br />                                               elegans nervous system. Genetics. 1995; 141:977&#8211;988. [PubMed: 8582641] <br />                                           Mathey-Prevot B, Perrimon N. Drosophila genome-wide RNAi screens: delivering the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               promise? Cold Spring Harb Symp Quant Biol. 2006; 71:141&#8211;148. [PubMed: 17381290] <br />                                           Neumann B, Walter T, Heriche JK, Bulkescher J, Erfle H, Conrad C, Rogers P, Poser I, Held M, <br />                                               Liebel U, et al. Phenotypic profiling human genome time-lapse microscopy reveals cell <br />                                               division genes. Nature. 2010; 464:721&#8211;727. [PubMed: 20360735] <br />                                           Oegema, K.; Hyman, AA. WormBook. Cell division. The C. elegans Research Community, <br />                                               WormBook. January 19. 2006 http://www.wormbook.org <br />                                           Piano, F.; Gunsalus, KC.; Hill, DE.; Vidal, M. WormBook. 2006. C. elegans network biology: a <br />                                               beginning; p. 1-20. <br />                                           Piano F, Schetter AJ, Mangone M, Stein L, Kemphues KJ. RNAi analysis genes expressed the <br />                                               ovary Caenorhabditis elegans. Curr Biol. 2000; 10:1619&#8211;1622. [PubMed: 11137018] <br />                                           Piano F, Schetter AJ, Morton DG, Gunsalus KC, Reinke V, Kim SK, Kemphues KJ. Gene clustering <br />                                               based RNAi phenotypes ovary-enriched genes C. elegans. Curr Biol. 2002; 12:1959&#8211; <br />                                               1964. [PubMed: 12445391] <br />                                           Praitis V, Casey E, Collar D, Austin J. Creation low-copy integrated transgenic lines in <br />                                               Caenorhabditis elegans. Genetics. 2001; 157:1217&#8211;1226. [PubMed: 11238406] <br />                                           Rhrissorrakrai K, Gunsalus KC. MINE: Module Identification NEtworks. BMC Bioinformatics. in <br />                                               press. <br />                                           Schuldiner M, Collins SR, Thompson NJ, Denic V, Bhamidipati A, Punna T, Ihmels J, Andrews B, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Boone C, Greenblatt JF, et al. Exploration function organization yeast early <br />                                               secretory pathway epistatic miniarray profile. Cell. 2005; 123:507&#8211;519. [PubMed: <br />                                               16269340] <br />                                           Shi A, Chen CC, Banerjee R, Glodowski D, Audhya A, Rongo C, Grant BD. EHBP-1 functions with <br />                                               RAB-10 endocytic recycling Caenorhabditis elegans. Mol. Biol. Cell. 2010; 21:2930&#8211; <br />                                               2943. <br />                                           Sijen T, Fleenor J, Simmer F, Thijssen KL, Parrish S, Timmons L, Plasterk RH, Fire A. On role of <br />                                               RNA amplification dsRNA-triggered gene silencing. Cell. 2001; 107:465&#8211;476. [PubMed: <br />                                               11719187] <br />                                           Simonis N, Rual JF, Carvunis AR, Tasan M, Lemmens I, Hirozane-Kishikawa T, Hao T, Sahalie JM, <br />                                               Venkatesan K, Gebreab F, et al. Empirically controlled mapping Caenorhabditis elegans <br />                                               protein-protein interactome network. Nat Methods. 2009; 6:47&#8211;54. [PubMed: 19123269] <br />                                           S&#246;nnichsen B, Koski LB, Walsh A, Marschall P, Neumann B, Brehm M, Alleaume AM, Artelt J, <br />                                               Bettencourt P, Cassin E, et al. Full-genome RNAi profiling early embryogenesis in <br />                                               Caenorhabditis elegans. Nature. 2005; 434:462&#8211;469. [PubMed: 15791247] <br />                                           Starr DA, Hermann GJ, Malone CJ, Fixsen W, Priess JR, Horvitz HR, Han M. unc-83 encodes novel <br />                                               component nuclear envelope essential proper nuclear migration. Development. <br />                                               2001; 128:5039&#8211;5050. [PubMed: 11748140] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Tatusov RL, Fedorova ND, Jackson JD, Jacobs AR, Kiryutin B, Koonin EV, Krylov DM, Mazumder <br />                                               R, Mekhedov SL, Nikolskaya AN, et al. The COG database: updated version includes <br />                                               eukaryotes. BMC Bioinformatics. 2003; 4:41. [PubMed: 12969510] <br />                                           Wilmes GM, Bergkessel M, Bandyopadhyay S, Shales M, Braberg H, Cagney G, Collins SR, <br />                                               Whitworth GB, Kress TL, Weissman JS, et al. A genetic interaction map RNA-processing <br />                                               factors reveals links Sem1/Dss1-containing complexes mRNA export splicing. <br />                                               Mol Cell. 2008; 32:735&#8211;746. [PubMed: 19061648] <br />                                           Zhou K, Rolls MM, Hall DH, Malone CJ, Hanna-Rose W. A ZYG-12-dynein interaction nuclear <br />                                               envelope defines cytoskeletal architecture C. elegans gonad. J Cell Biol. 2009; 186:229&#8211; <br />                                               241. [PubMed: 19635841] <br />                                           Zipperlen P, Fraser AG, Kamath RS, Martinez-Campos M, Ahringer J. Roles 147 embryonic lethal <br />                                               genes C. elegans chromosome I identified RNA interference video microscopy. EMBO <br />                                               J. 2001; 20:3984&#8211;3992. [PubMed: 11483502] <br />  <br />  <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                             Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 1. Phenotypic profiling using high-content assay based architecture complex <br />                                           tissue <br />                                           A. Breakdown 554 genes sterile collection. B. Screen flow chart. C. The C. <br />                                           elegans gonad complex dynamic tissue architecture depends broad spectrum <br />                                           interacting cellular processes (blue text schematic). Spinning disc confocal <br />                                           microscopy used collect 80 plane two-color z-series gonad. The imaged <br />                                           region indicated (red dashed box) sample central z-section control gonad is <br />                                           shown. D. Six sample defects illustrated pairing numbered boxed regions the <br />                                           control gonad C corresponding regions gonads indicated defects. <br />                                           E. Central plane images gene knockdowns 3 phenotypic classes. Knockdowns <br />                                           Class B2 (left column), contains chaperonin complex subunits, led rounded <br />                                           compartments nuclei fell compartments (yellow arrowheads). <br />                                           Knockdowns Class F3 (middle column), contains genes implicated Rho GTPase <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                        Page 16 <br />  <br />  <br />                                           signaling, led &#8220;tubulated&#65533;? gonads clustered nuclei. Knockdowns Class C2 (right <br />                                           column), contains cell cycle regulators, led gonads compartments/ <br />                                           oocytes. Bars, 10 &#956;m. See Figure S1, Table S1 Table S2. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                         Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 2. Validation manual classification I <br />                                           Class E2: A. List genes Class E2. Schematic central plane images illustrate the <br />                                           class phenotype. B. Nuclei dorsal cords T09E8.1(RNAi) (yellow arrowheads), <br />                                           control, L1 worms (insets 2.4X), reflecting hypodermal cell nuclear migration <br />                                           defect. Table shows quantification nuclear migration defect. The effects of <br />                                           simultaneous inhibition dynein-regulatory proteins BICD-1 NUD-2 quantified <br />                                           dataset Fridolfsson et al. (2010) shown comparison. C. The effect of <br />                                           T09E8.1 knockdown microtubule arrays hypodermis (schematic) monitored <br />                                           timelapse imaging EB1-GFP fusion (Fridolfsson Starr, 2010). Bar graphs show <br />                                           mean length (left) number (right) EB1-GFP comets. Error bars SE. <br />                                           Class G1: D. Class G1 contains 3 characterized genes implicated MAPK signaling <br />                                           (gray), 1 characterized gene previously implicated MAPK signaling (orange), 1 <br />                                           uncharacterized gene (purple). Schematic central plane images illustrate class <br />                                           phenotype. E. Partial RNAi daf-21, GFP control, performed presence or <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                      Page 18 <br />  <br />  <br />                                           absence weak mpk-1 loss-of-function mutation (ga111). Gonads fixed stained <br />                                           chromosomes (DAPI, green), plasma membrane (SYN-4 PTC-1, red), and <br />                                           activated MPK-1 (Phospho-MPK-1, right panels). All analysis performed the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           rrf-1(pk1417) background RNAi effective gonad, the <br />                                           surrounding somatic cells. F. Quantification phenotypes resulting partial RNAi of <br />                                           daf-21 F54D12.5 mpk-1(ga111) background (see Table S3). * Percentages not <br />                                           include 21% germlines showed severe MPK-1 &#8220;null&#65533;? phenotype. G. Schematic <br />                                           places DAF-21 F54D12.5 MAPK signaling pathway. Error bars SE. Bars, <br />                                           10 &#956;m. See Figure S2, Table S2 Table S3 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                            Page 19 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 3. Validation manual classification II <br />                                           Class S2: A. List genes Class S2. Schematic central plane images gonads <br />                                           (left) adjacent embryos (right) illustrate class phenotype. B. List relevant proteins <br />                                           identified mass spectrometry immuno-affinity purification K10D2.4 worm <br />                                           extracts, percent coverage. Proteins encoded uncharacterized Class S2 <br />                                           genes (purple) listed, APC components Class S2 (gray) additional <br />                                           APC components screen (black). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Class F2: C. List genes Class F2. Schematic central plane images illustrate the <br />                                           class phenotype. D. Embryo co-expressing GFP-C08C3.4 (green) mCherry tagged <br />                                           plasma membrane probe (red) cell division. E. Central plane images of <br />                                           control C08C3.4(RNAi) embryos expressing mCherry-histone H2B (green) GFP <br />                                           plasma membrane probe (red). Multiple nuclei cell C08C3.4(RNAi) embryo <br />                                           cytokinesis failure. <br />                                           Class I2: F. Schematic central plane images illustrate Class I2 phenotype, which <br />                                           includes punctate debris containing plasma membrane probe (arrow schematic). G. <br />                                           Schematic trafficking assay. List Class I2 genes defects SNB-1-GFP <br />                                           trafficking assay. Images assay 1 characterized 2 uncharacterized Class I2 <br />                                           genes. Bars 10&#956;m A-F 5 &#956;m G. See Table S2. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                            Page 20 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 4. Constructing gene networks using Connection Specificity Index (CSI) instead the <br />                                           PCC reduces connection noise allows connections similar functional significance be <br />                                           viewed entire network uniform threshold <br />                                           A. Flowchart steps used construct gene networks B. B. Gene networks were <br />                                           constructed displaying connections (dark blue lines) genes knockdown <br />                                           profiles correlated PCC &#8805; specified thresholds (0.46, top; 0.60, middle; <br />                                           0.70, bottom). Each column shows network region, labeled based primary function <br />                                           genes region. To compare computational network manually-defined <br />                                           phenotypic classes, gene groups manually-assigned classes circled labeled. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           The optimal PCC threshold significant connections displayed non- <br />                                           specific connections filtered (red boxes) different different network <br />                                           regions. C. Method used calculate CSI. D. Gene network showing regions in <br />                                           B constructed displaying connections (light blue lines) CSI &#8805; 0.97. E. Bar graph <br />                                           showing number connections gene networks constructed using PCC CSI <br />                                           thresholds 0.55 0.96, respectively. These thresholds chosen the <br />                                           highest thresholds retain genes network; genes drop the <br />                                           network longer make connections PCC/CSI exceeds the <br />                                           specified threshold. F. Bar graph showing percent 49 manually-defined <br />                                           phenotypic classes containing 4 genes identified automated clustering <br />                                           algorithm (MINE) networks constructed using PCC CSI thresholds 0.55 0.96, <br />                                           respectively. G. Heatmap dendograms sterile gene set constructed based PCC <br />                                           CSI. See Figure S3 Table S4. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                              Page 21 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 5. Varying CSI threshold reveals functional modularity gene network at <br />                                           different levels resolution <br />                                           A. The region gene network involved protein production shown using three <br />                                           different CSI thresholds filter displaye <br /> </body></html>