<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                    Drosophila poly suggests novel <br />                                                     role Elongator complex in <br />                                                     insulin receptor &#8211; target of <br /> rsob.royalsocietypublishing.org <br />                                                     rapamycin signalling <br />                                                     Ekin Bolukbasi&#8224;, Sharron Vass, Neville Cobbe&#8225;, Bryce Nelson}, <br /> Research <br />                                                     Victor Simossis k , Donald R. Dunbar Margarete M. S. Heck <br /> Cite article: Bolukbasi E, Vass S, Cobbe <br /> N, Nelson B, Simossis V, Dunbar DR, Heck            University Edinburgh, Queen&#8217;s Medical Research Institute, University/BHF Centre for <br /> MMS. 2011 Drosophila poly suggests novel          Cardiovascular Science, 47 Little France Crescent, Edinburgh EH16 4TJ, UK <br /> role Elongator complex insulin <br /> receptor &#8211; target rapamycin signalling. <br /> Open Biol 2: 110031.                                1. Summary <br /> http://dx.doi.org/10.1098/rsob.110031               Multi-<span id='am-129' about='obo:NCBITaxon_131567' typeof='owl:Thing obo:NCBITaxon_1'><span id='am-130' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-131' property="rdfs:label" content="cellular organisms" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasRelatedSynonym" content="biota" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cellular organisms</span> need successfully link cell growth metabolism <br />                                                     environmental cues development. Insulin receptor &#8211;target rapamy- <br /> Received: 23 November 2011                          cin (InR&#8211;<span id='am-146' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-147'  ></span><span id='am-148' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-150' property="rdfs:label" content="Tor" datatype="xsd:string"></span>TOR</span>) signalling highly conserved pathway mediates link. <br />                                                     Herein, poly, essential gene Drosophila mediates InR &#8211;TOR <br /> Accepted: 21 December 2011 <br />                                                     signalling. Loss poly results lethality instar larval stage, but <br />                                                     stage extreme larval longevity. Analysis Drosophila demon- <br /> Subject Area: <br />                                                     strates Poly InR interact poly mutants overall <br /> cellular biology/developmental biology/             decrease InR&#8211;TOR signalling, evidenced decreased phosphorylation <br /> genetics/molecular biology                          Akt, S6K 4E-BP. Metabolism altered poly mutants, revealed by <br />                                                     microarray expression analysis decreased triglyceride : protein ratio in <br /> Keywords:                                           mutant animals. Intriguingly, cellular distribution Poly dependent <br /> cell growth, nucleus, signalling, insulin           insulin stimulation Drosophila human cells, moving the <br />                                                     nucleus insulin treatment, consistent role InR&#8211;TOR signalling. <br /> Author correspondence:                          Together, <span id='am-161' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-162'  ></span><span id='am-163' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-164' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> reveal Poly novel, conserved (from flies to <br />                                                     humans) mediator InR signalling promotes increase cell growth <br /> Margarete M. S. Heck <br />                                                     metabolism. Furthermore, homology small subunits Elongator <br /> e-mail: margarete.heck@ed.ac.uk                     demonstrates novel, unexpected role complex insulin signalling. <br />  <br />  <br /> Present addresses: <br /> &#8224; <br />   Institute Healthy Ageing, University College <br />                                                     2. Introduction <br /> London, The Darwin Building, Gower Street,          Multi-cellular organisms evolved mechanisms link cellular metabolism <br /> London WCIE 6BT, UK.                                growth external environmental cues, nutrient growth factor <br />                                                     levels, <span id='am-95' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-96' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> survive adapt fluctuations availability these <br /> &#8225; <br />  School Biological Sciences, University       factors. The InR&#8211;TOR pathway key regulators cellular energy <br /> Liverpool, The Biosciences Building, Crown          homeostasis growth [1], signalling pathway evolutionarily con- <br /> Street, Liverpool L69 7ZB, UK.                      served <span id='am-171' about='obo:NCBITaxon_33208' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-172' property="oboInOwl:hasRelatedSynonym" content="animals" datatype="xsd:string"></span><span id='am-173' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-174' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasExactSynonym" content="metazoans" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasExactSynonym" content="multicellular animals" datatype="xsd:string"></span><span id='am-177'  ></span><span id='am-178' property="oboInOwl:hasRelatedSynonym" content="Animalia" datatype="xsd:string"></span><span id='am-179' property="rdfs:label" content="Metazoa" datatype="xsd:string"></span>metazoa</span> [1,2]. Thus, studies carried using Drosophila a <br /> } <br />  Donnelly Center Cellular Biomolecular      model played major role expanding understanding of <br /> Research, University Toronto, 160 College        mechanisms, downstream consequences, signalling <span id='am-85' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87'  ></span><span id='am-88' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>this</span> <br /> Street, Toronto, Ontario, Canada, M5S 3E1.          pathway [3&#8211;5]. In humans, aberrations InR&#8211;TOR signalling lead various <br />                                                     metabolic syndromes, including diabetes obesity, develop- <br /> k <br />  Institute Molecular Oncology, Biomedical        ment various types cancers [6]. <br /> Sciences Research Center &#8216;Alexander Fleming&#8217;,           A cascade phosphorylation events mediates signalling InR&#8211; <br /> 16672 Varkiza, Greece.                              TOR pathway. The binding insulin InR leads phosphorylation <br />                                                     insulin receptor substrate (IRS) InR. IRS acts recruitment site <br /> Electronic supplementary material available      phosphatidylinositol 3-kinase (PI3K), catalyses conversion of <br /> at http://dx.doi.org/10.1098/rsob.110031            phosphatidylinositol (4,5)-bisphosphate (PIP2) phosphatidylinositol <br />  <br />  <br />                                                      &amp; 2012 The Authors. Published Royal Society terms Creative Commons Attribution <br />                                                      License http://creativecommons.org/licenses/by/3.0/, permits unrestricted use, provided original <br />                                                      author source credited. <br />  (3,4,5)-trisphosphate (PIP3) cell membrane. PIP3 turn      verified expression overlapping CG8790 gene              2 <br /> recruits PDK1 Akt membrane, PDK1 <span id='am-180' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-181' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-182' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-183' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-184'  ></span>phos</span>-              affected P-element insertion poly05137 allele <br />  <br />  <br />  <br />  <br />                                                                                                                                             rsob.royalsocietypublishing.org <br /> phorylates activates Akt. Phosphorylated Akt signals             (figure 1b). Two independent experiments additionally corro- <br /> inhibit tuberous sclerosis complex (TSC, Tsc1 &#8211;Tsc2) [7 &#8211;        borate lesion CG9829 gene responsible the <br /> 9]. When TSC inhibited, small GTPase Rheb             mutant phenotype: (i) excision P-element following gen- <br /> active [10]. This activates TOR, integrating TOR           etic exposure transposase completely reverted mutant <br /> the insulin signalling process.                                      phenotype, (ii) successful rescue poly05137 lethality <br />     TOR component different complexes: TORC1         achieved using hs-Gal4 driver direct expression <br /> TORC2. Activation TORC1 various downstream effects            UAS-poly transgene larval development. <br /> contributing increase cell growth proliferation. For        Mutation poly results pleiotropic effects manifesting a <br /> example, TORC1 directly phosphorylates S6K 4E-BP, result-        particularly striking phenotype. poly mutants appear progress <br /> ing increase ribosome biogenesis m7G cap-               normally embryogenesis, larval development pro- <br />  <br />  <br />  <br />  <br />                                                                                                                                             Open Biol 2: 110031 <br /> dependent translation [1]. In addition, TORC1 activation inhibits    ceeds slowly. While normal generation time is <br /> autophagy [11]. A negative feedback loop signals IRS     10 days 258C, poly mutant larvae exhibit extreme instar <br /> through S6K, ensuring attenuation TOR signalling cer-     longevity&#8212;up 21 days&#8212;before dying pupation <br /> tain level [12]. The TORC2 complex phosphorylates activates      (figure 1d). When homozygous mutant larvae examined, <br /> Akt kinase [13], resulting phosphorylation forkhead-   morphology tissues appeared abnormal: brain, <br /> like transcription factor FoxO. Phosphorylated FoxO excluded      ring gland, salivary glands imaginal discs reduced in <br /> from nucleus, precluding transcription FoxO target        size compared heterozygous siblings wild-type ani- <br /> genes [14&#8211;16].                                                       mals, suggesting cell growth and/or proliferation defects <br />     A critical consequence activation InR &#8211;TOR sig-        (figure 1e). Mutant larval neuroblasts characterized by <br /> nalling inhibition autophagy: cellular response       abnormally shaped nuclei, mitotic figures, evident even <br /> starvation components cytoplasm                  20 day old mutant larvae, normal appearance. These <br /> engulfed small double-membrane-enclosed vesicles. The             lobulated nuclei resembled nuclei mammalian poly- <br /> contents vesicles degraded autophagic            morphonuclear leucocytes, suggesting poly <br /> machinery, breakdown products serve nutrient           (figure 1f ). During lengthened instar larval phase, <br /> source cell nutrients available         melanotic masses appeared haemolymph poly <br /> the environment. Alterations autophagy            mutants, increasing size number time (figure 1g). <br /> in <span id='am-65' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-66' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-68'  ></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>cancer</span> neurodegenerative diseases [17,18].                    A second hypomorphic allele poly recently reported <br />     Herein, report identification Poly novel          result abnormal nurse-cell chromosome dispersal and <br /> mediator InR &#8211;TOR signalling pathway. poly              oocyte polarity Drosophila germline, possibly owing a <br /> essential gene Drosophila mutated P-element         proposed interaction mRNP complex involved similar <br /> transposon mutagenesis screen [19]. Crucially, gene pro-         processes [20]. <br /> duct conserved higher eukaryotes, including humans,                Expression analysis poly revealed high levels mRNA <br /> showing homology ELP6 subunit Elongator                protein 4 h embryogenesis, suggestive of <br /> complex. Loss poly function results lethality late      maternal loading mRNA possibly protein into <br /> larval stage, extreme larval longevity. Many          oocytes (figure 1h,i). While level Poly decreased <br /> intriguing phenotypic features observed larvae lacking        remainder embryogenesis, level still <br /> poly, including abnormal nuclear morphology neuroblasts           higher later stages development. Overall, poly <br /> and development large melanotic masses instar        expressed development (figure 1h,i). <br /> larvae. We combined genetic, biochemical bioinfor- <br /> matic approaches functionally characterize poly. Our data <br /> reveal poly novel mediator InR&#8211;TOR signalling <br />                                                                      3.2. Poly conserved higher eukaryotes and <br /> and loss poly results downregulation number               resembles Elp6 subunit Elongator <br /> of components InR &#8211;TOR pathway. We pro- <br /> pose wild-type Poly protein positive regulator <br />                                                                           complex <br /> of cell growth metabolism.                                       The Drosophila poly gene encodes 251-amino-acid-long <br />                                                                      protein, lacking motifs suggestive specific function. <br />                                                                      Sequence similarity searches revealed Poly conserved <br />                                                                      higher eukaryotes and, importantly, human <br /> 3. Results                                                           homologue (figure 2a). The apparent orthologue Poly in <br />                                                                      <span id='am-33' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-34' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> Elp6 protein, 6-sub- <br /> 3.1. Characterization poly mutant phenotype                   unit Elongator complex that, association RNA <br /> The poly mutation isolated P-element mutagenesis            polymerase II holoenzyme, responsible transcriptional <br /> screen aimed generate large collection single           elongation [21,22]. The Elongator holocomplex conserved <br /> P-element-induced mutations Drosophila [19]. The                  composition yeast humans [23,24], acetylation <br /> P-element insertion led lethal poly05137 allele      activity contributed Elp3 catalytic subunit [25]. Intri- <br /> mapped single intron CG9829 gene, localizing           guingly, acetylation directed different substrates, <br /> to 87E7-8 chromosome (figure 1a). The poly05137         depending cell complex (e.g. tubulin <br /> insertion led absence poly mRNA assessed              cytoplasm, histone nucleus) [26,27]. Elonga- <br /> Northern blotting (not shown), reverse transcriptase &#8211;               tor additionally linked translational control <br /> polymerase chain reaction (RT-PCR; figure 1b) Poly               tRNA modification cytoplasm. To date, only <br /> protein revealed immunoblotting (figure 1c). RT-PCR            Elp3 subunit studied Drosophila, where <br />                                                                                                                                                                                          poly/TM6B <br />             (a)                                                                                        (b)                                                     (c)                                                                      3 <br />  <br />  <br />  <br />  <br />                                                                                                                           wild-type <br />  <br />  <br />                                                                                                                                           poly/poly <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            wild-type <br />  <br />  <br />                                                                                                                                                                                                      poly/poly <br />                                                                                                                                                                                                                     Dmel2 <br />                                                     P{PZ} poly 05137 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                       rsob.royalsocietypublishing.org <br />                                                                                         poly <br />  <br />                                                                                    CG9829 <br />                                                                                                              poly <br />  <br />                                                                                    1 kb                                                                        poly <br />                                                                                                        CG8790 <br />                   CG8790 <br />                                                                                                                                                                actin <br />                                                                                                              rp49 <br />  <br />             (d)          day      1                 2                 3       4                    6                                                           10 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                       Open Biol 2: 110031 <br />                      wild-type embryo               1*                2*           3*                                    pupation                                                      eclosion <br />                    poly/TM6B embryo                     1*            2*           3*                                    pupation                                                      eclosion <br />                       poly/poly embryo                  1*             2*           3*          melanotic masses visible                                                                            21 days <br />  <br />  <br />             (e) <br />  <br />  <br />  <br />  <br />                                                                             wild-type                                                                                   poly <br />  <br />             (f)                                                                                                          (g) <br />                        wild&#8211;type                                           poly <br />  <br />  <br />  <br />  <br />             (h)                                                                                    (i) <br />                                                                                                                               emb 4-20 <br />                                                                                                                emb 0-4 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                           adult-m <br />                                                2nd IL <br />  <br />  <br />  <br />  <br />                                                                                                                                                       2nd IL <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                             adult-f <br />                                                              3rd IL <br />  <br />  <br />  <br />  <br />                                                                                                                                                                3rd IL <br />                                       1st IL <br />  <br />  <br />  <br />  <br />                                                                                                                                          1st IL <br />                                                                                         adult <br />                                emb <br />  <br />  <br />  <br />  <br />                                                                        EP <br />  <br />                                                                               LP <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            EP <br />  <br />                                                                                                                                                                                          LP <br />  <br />  <br />  <br />  <br />                     poly <br />                                                                                                        Poly <br />  <br />  <br />                     rp49                                                                             actin <br />  <br /> Figure 1. Characterization poly gene mutant phenotype. (a) Gene map showing P-element insertion intron poly. (b) RT-PCR wild- <br /> type homozygous poly instar larvae demonstrating absence Poly poly mutant animals. The level CG8790 unaffected poly mutants. RP49 <br /> serves control. (c) Immunoblotting extracts wild-type, heterozygous homozygous poly instar larvae demonstrating absence Poly poly <br /> mutant animals. Poly present Dmel2-cultured Drosophila cells. (d ) Developmental timing wild-type, heterozygous homozygous poly animals. <br /> (e) Third instar larval brain imaginal discs dissected wild-type poly animals. Note reduced brain size absence imaginal discs the <br /> homozygous poly mutant. (f ) DAPI staining wild-type poly larval neuroblasts. Note abnormally shaped nuclei homozygous poly mutant. <br /> (g) Homozygous poly larvae develop melanotic masses larval instar, increase number time. (h,i) Expression analysis poly mRNA and <br /> protein development. Northern immunoblotting wild-type extracts specific developmental stages showing Poly mRNA protein expression <br /> levels. emb, embryo (0&#8211; 4, 4 &#8211; 20 &#188; age hours); IL, instar larvae (1st, 2nd 3rd); EP, early pupae; LP, late pupae; adult-m, adult males; adult-f, adult females. <br />  <br /> the mutant phenotype recently reported                                                    (and identified flies humans). Consistent <br /> remarkably similar described [28].                                                        previous identification Elp4 Elp6 proteins as <br />     Elp6 budding yeast Elongator subcomplex                                             paralogues [29], resulting phylogeny implies of <br /> that contains Elp4 Elp5 proteins [21,22],                                             different Elongator subunits probably arose from <br /> share sufficient homology aligned                                                 common gene duplication events eukaryotic evolution <br />                         (a) Hs                                                                                                                                                                                 4 <br />                              Dm <br />                              At <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                             rsob.royalsocietypublishing.org <br />                              Sc <br />  <br />                              Hs <br />                              Dm <br />                              At <br />                              Sc <br />  <br />                              Hs <br />                              Dm <br />                              At <br />                              Sc <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                             Open Biol 2: 110031 <br />                              Hs                                                                                                                                                              conserved <br />                              Dm <br />                              At <br />                              Sc <br />                                                                                                                                                                                                  variable <br />  <br />  <br />  <br />                        (b) <br />                                                                                                                                                      Elp 4                                    s <br />                                                                                          Dicty <br />                                                                                                                                                                                             ce <br />  <br />  <br />  <br />  <br />                                                                                                                                    <span id='am-57' about='obo:NCBITaxon_5782' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_142796 obo:NCBITaxon_33083 obo:NCBITaxon_2759 obo:NCBITaxon_554915'><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="Dictyostelium" datatype="xsd:string"></span><span id='am-60'  ></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Dictyostelium</span> <br />                                                                       p6                                                                                                                 y <br />                                                                    /El                                                                                                                om <br />  <br />  <br />  <br />  <br />                                                                                                                                                                    is <br />                                                                                                                                                                                   ar <br />                                                                                             osteli <br />  <br />  <br />                                                            y <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  ps <br />                                                                                                                                                                               ch <br />                                                          ol                                                        h <br />  <br />  <br />  <br />  <br />                                                                                                                                                             o <br />                                                                                                                                                                              c <br />                                                                                                                 zebrafis <br />  <br />  <br />  <br />  <br />                                                                                                                                                          id <br />                                                                                                                                                                           sa <br />  <br />  <br />  <br />  <br />                                                                                                                                                      rice <br />                                 Sch <br />                                                                                      rice <br />  <br />  <br />  <br />  <br />                                                                                                                                                       ab <br />                                                                                                                                                                         zo <br />                                                                                                   um <br />  <br />  <br />  <br />                                      izo                                                                                                                                                           ces <br />                                                                                                                                                                      hi <br />                                                     P <br />  <br />  <br />  <br />  <br />                                                                                                                                                     Ar <br />                                         sac <br />                                              cha <br />                                                                Ar <br />                                                                    ab                                                                                              Sc             har    omy <br />                                                     rom                  ido <br />                                                                                psi                                                                                        Sacc <br />                                                                                                                            m <br />                                                          yce                      s                                    hu                                                               habditis <br />                                                                s                                                                                                          Caenor <br />                              Caenorha                                                                                                                           Drosoph <br />                                            bditis                                                                                                                          ila <br />                                                                                                                                                                zeb <br />                                                                                                                                                                    rafi <br />                                                                                                                                                           hum           sh <br />                                                                                                                                                            an <br />  <br />                                             yces <br />                                    arom <br />                              Sacch                                       hil <br />                                                                               a <br />                                                                    op <br />                                                            D   ros <br />                                                                                                                                                            human <br />  <br />                                                                 rice                                                                      zeb <br />                                                                                                                                                     rafi <br />                                                                                                                                                           sh <br />                                                                           s <br />                                                                       psi <br />                                                                    ido <br />                                                            ab <br />                                                          Ar <br />                                                                                                          stelium <br />  <br />  <br />  <br />  <br />                                                                          es <br />                                                                  m  yc <br />                                                               ro <br />                                                                                                        Dictyo <br />  <br />  <br />  <br />  <br />                                                           ha <br />                                                       cc <br />                                                                                      s <br />                                                                                   yce <br />  <br />  <br />  <br />  <br />                                                      a <br />                                                iz os                                                                                                                         Ca <br />                                                                                                                                                                                  en <br />                                                                             rom <br />  <br />  <br />  <br />  <br />                                               h <br />                                            Sc                                                                                                                                      or <br />                                                                                                                                            Dros <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                        ha <br />                                                                          cha <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                          bd <br />                                                                                                   E lp 5                                                                                         <span id='am-166' about='obo:NCBITaxon_106778' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_980193 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-167' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-168'  ></span><span id='am-169' property="rdfs:label" content="Iti" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>iti</span> <br />                                                                                                                                                                                                     s <br />                                                                                                                                                    ophi <br />                                                                      Sac <br />  <br />  <br />  <br />  <br />                                                                                                                                                     la <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                    0.5 <br /> Figure 2. Alignment phylogenetic analysis Poly. (a) Alignment showing Poly homologues human (Hs; NP_001026873), <span id='am-70' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-74'  ></span><span id='am-75' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span>Drosophila melanogaster</span> (Dm; <br /> AAF40432), <span id='am-16' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-17' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span><span id='am-18'  ></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Arabidopsis</span> thaliana (At; NP_567351) <span id='am-50' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-51' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae (Sc; NP_014043), shading identical similar amino acids rendered by <br /> the BOXSHADE program. The coloured band underneath section alignment indicates sequence conservation according PAM250 matrix, averaged <br /> over sliding window 10 amino acids. The location Walker B motif (ordinarily associated ATP binding) indicated blue box, Walker <br /> A (P-loop) motif required ATP hydrolysis similarly conserved. (b) Phylogenetic tree showing relationship Poly, identified Drosophila <br /> orthologue Elp6, homologous sequences. The scale bar indicates distance according WAG substitution matrix. <br />  <br />  <br /> (figure 2b; electronic supplementary material, figure S1).                                                                 interacting physically Poly, immunoprecipitation with <br /> Thus, Poly evolutionarily conserved protein exhibits                                                            antibody generated recombinant Poly per- <br /> homology yeast Elp6 Elongator subcomplex component.                                                                 formed using 0 &#8211;5 h Drosophila embryo extracts (figure 3a,b). <br />                                                                                                                            Poly immunoprecipitated immune serum, <br />                                                                                                                            wild-type extracts. Samples analysed <br /> 3.3. Identification protein interactors Poly                                                                         using tandem mass spectrometry [30,31]. Immunoprecipi- <br /> As poly gene appeared &#8216;pioneer&#8217;, adopted                                                                 tates pre-immune serum served control. Among <br /> unbiased approach determine pathways processes                                                                  Poly-binding proteins identified significant <br /> in Poly involved. In order identify proteins                                                             scores, InR scored highest represented numerous <br />                                                                                                                    a-poly <br />                                                                                       (b)                                                                                5 <br />  <br />  <br />  <br />  <br />                                                            a-poly <br />                                          (a) <br />  <br />  <br />  <br />  <br />                                                                                                          pre-I <br />                                                    pre-I <br />  <br />  <br />  <br />  <br />                                                                                                                                                                        rsob.royalsocietypublishing.org <br />                                                                                             97 <br />                                          97 <br />                                                                                                              6             1 <br />                                                                                             64               7             2 <br />                                          64                                                                  8             3 <br />                                                                                             51 <br />                                          51                                HC <br />                                                                                             39 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                        Open Biol 2: 110031 <br />                                          39 <br />                                                                                                              9             4 <br />                                                                                             28 <br />                                          28                                poly                              10            5 <br />                                                                                             19 <br />                                          19                                                 14 <br />  <br />  <br />                        (c)                                      (d)                              (e) <br />                                                                                                                                    poly <br />                                  wt       InR      Akt1                    wt         InR <br />  <br />                          Poly                                       poly <br />  <br />  <br />                        &#181;-tub                                        rp49 <br />                                                                                                        InR <br />  <br /> Figure 3. Poly insulin receptor physically interact. (a) An antibody generated recombinant Poly immunoprecipitates Poly wild-type Drosophila third <br /> instar larval extracts. HC represents heavy chain recognized secondary antibody detection. Poly migrates predicted size 28 kDa. (b) Poly <br /> was immunoprecipitated 0 &#8211; 5 h wild-type embryos using Poly antibodies compared identical sample immunoprecipitated pre-immune <br /> serum. Bands numbered 1 &#8211; 5 immunoprecipitate lane corresponding bands pre-immune lane (numbered 6 &#8211; 10) excised the <br /> Coomassie-stained gel analysed mass spectrometry. (c) Immunoblotting using Poly antibody carried wild-type, InR05545 Akt104226 larval <br /> extracts. (d ) RT-PCR performed using primers specific poly rp49 RNA extracted wild-type InR05545 larvae. (e) Melanotic masses (arrows) <br /> observed InR05545 poly mutant larvae. <br />  <br />  <br /> peptides distinct bands (table 1). This result suggests                        3.5. Poly mutation results decreased signalling <br /> that Poly InR coexist complex, <br /> possibly playing role InR signalling pathway.                                     activity downstream insulin receptor <br />                                                                                       Based results presented far, hypothesized that <br />                                                                                       genetic interactions poly com- <br /> 3.4. Poly decreased insulin receptor mutants                                    ponents InR signalling pathway. Consequences of <br /> Considering physical interaction Poly InR,                             interactions revealed state down- <br /> assessed level Poly various InR signalling mutants.                         stream effectors InR pathway. <br /> Strikingly, level Poly dramatically decreased                               We over-expressed poly adult fly eyes, using a <br /> InR05545 mutant larvae, level Poly Akt1                             GMR-Gal4 construct drive expression UAS-poly <br /> mutant larvae unaffected (Akt1 situated downstream                             transgene [33]. At 278C, led rough eye (disorganized <br /> of InR; figure 3c). The level poly mRNA unchanged                              ommatidia) phenotype (figure 4a). Disruption InR&#8211; TOR <br /> in InR mutant, suggesting difference protein                          signalling mutation dAkt1 dS6K led to <br /> level instability decreased protein syn-                           striking suppression poly-induced rough eye pheno- <br /> thesis Poly absence InR (figure 3d).                                     type (figure 4b,c), suggesting intact InR &#8211;TOR <br />     Further evidence connection Poly                                signalling cascade required Poly exert effects. <br /> the InR provided phenotypic analysis InR05545                               These experiments critically demonstrate cell auton- <br /> mutant larvae (figure 3e). While allele reported                             omous importance Poly insulin signalling. <br /> previously embryonic lethal [32], noticed                                 We hypothesized dAkt dS6K mutants act as <br /> small number homozygous InR05545 animals reached                               suppressors poly over-expression phenotype, the <br /> third instar larval stage. Strikingly, phenotype                               activity kinases altered poly mutant ani- <br /> these InR mutant larvae resembled observed poly                               mals. We probed early instar larval extracts <br /> larvae increased larval lifespan development                              antibodies recognize specifically phos- <br /> melanotic masses (figure 3e). These observations demonstrate                          phorylated (active) forms dAkt dS6K. Indeed, <br /> that loss InR leads decrease Poly InR05545                        phosphorylation dAkt dS6K decreased <br /> mutants exhibit phenotypic features similar poly                          mutation poly (figure 4d,e). These data reveal that <br /> mutant larvae.                                                                        activity dAkt dS6K kinases decreased <br />                          (a)                 +                  GMR-Gal4&gt;UAS-pol                   (d)                          wt     poly                                    6 <br />                                             +                          +                             64 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                             rsob.royalsocietypublishing.org <br />                                                                                                                                                     p-dAkt <br />                                                                                                                       51 <br />                                                                                                                       64 <br />                                                                                                                                                     dAkt <br />  <br />  <br />                                                                                                                       51                            &#181;-tub <br />  <br />  <br />                                                                                                   (e)                           wt        poly <br />  <br />  <br />  <br />  <br />                                                                                                                                                                             Open Biol 2: 110031 <br />                         (b) GMR-Gal4&gt;UAS-poly                 GMR-Gal4&gt;UAS-poly                                64                                   p-dS6K <br />                                           TM3                             Akt1                                 51 <br />  <br />                                                                                                                       51                            &#181;-tub <br />  <br />  <br />                                                                                                   (f)                                       ** <br />                                                                                                                        0.8 <br />  <br />  <br />  <br />  <br />                                                                                                    4E-BP expression <br />                                                                                                                        0.6 <br />  <br />                                                                                                                        0.4 <br />  <br />                                                                                                                        0.2 <br />                         (c) GMR-Gal4&gt;UAS-poly                 GMR-Gal4&gt;UAS-poly <br />                                                                                                                            0 <br />                                           TM3                             S6K                                                        wt           poly <br />  <br />  <br />                                                                                                   (g)                           wt         poly <br />  <br />                                                                                                                       19                                 p-4E-BP <br />  <br />  <br />                                                                                                                       51                                 &#181;-tub <br />  <br /> Figure 4. Phosphorylation levels Akt S6K decreased poly larvae. (a &#8211; c) Images showing Drosophila compound eyes. (a) A rough eye phenotype is <br /> caused UAS-poly overexpression control GMR-Gal4 driver. (b,c) The rough eye phenotype caused UAS-poly overexpression control of <br /> GMR-Gal4 driver suppressed Akt104226 S6K07084 mutations. TM3 represents balancer chromosome carried control sibling progeny. (d,e) <br /> Immunoblotting instar larval extracts using phospho-specific antibodies dAkt dS6K reveals decreased phosphorylation kinases poly larval <br /> extracts. (f ) d4E-BP transcript levels assessed real-time qPCR RNA isolated wild-type poly mutant larvae. d4E-BP levels normalized to <br /> Actin5C levels. The error bars derive reactions performed biological triplicate samples. The double asterisk represents significant difference ( p , 0.01) 4E- <br /> BP expression levels wild-type poly larvae. (g) Immunoblotting instar larval extracts showing decreased phosphorylation d4E-BP poly <br /> compared wild-type larvae. <br />  <br />  <br /> Table 1. Mass spectrometry identi&#65533;?es insulin receptor physical interactor Poly. Analysis pre-immune immunoprecipitation samples tandem <br /> mass spectrometry identi&#65533;?ed interacting proteins speci&#65533;?cally immune-precipitate sample. Insulin receptor identi&#65533;?ed bands 2 3, the <br /> highest scores. &#8216;exp score&#8217; quanti&#65533;?es log scale (base 10) expectation hit achieved chance, calculated using program X!TANDEM. <br />  <br />   band                    exp score                     unique/total peptides                             protein <br />  <br />   1                       23                            1/1                                               ATP-dependent RNA helicase (Rm62) <br />   2                       213                           4/7                                               insulin-like receptor <br />   3                       213                           4/7                                               insulin-like receptor <br />   4                       25.3                          3/3                                               cadherin <br />   5                       22.5                          2/2                                               transient receptor potential cation channel <span id='am-62' about='obo:NCBITaxon_subfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="subfamily" datatype="xsd:string"></span>subfamily</span> A (TrpA1) <br />  <br />  <br /> in poly mutant animals. In addition, level p-Akt                                      One crucial downstream effector TOR 4E-BP, the <br /> at tissue level, examined immunofluorescence                                   translation initiation factor eIF-4E-binding protein. Phos- <br /> whole mount instar larval brains, additionally corro-                              phorylation 4E-BP TOR leads dissociation from <br /> borates decrease (electronic supplementary material,                                m7G-cap-binding protein eIF-4E, allowing acti- <br /> figure S2).                                                                              vation cap-dependent translation consequent <br />  positive effects cell growth [34]. Disruption InR&#8211; TOR       detect mitosis apoptosis, respectively (figure 5c). Mitotic      7 <br /> signalling causes inhibition cap-dependent translation       figures, rare latest stages, normal in <br />  <br />  <br />  <br />  <br />                                                                                                                                       rsob.royalsocietypublishing.org <br /> a decrease 4E-BP phosphorylation results binding         appearance (data shown). As generation mosaic <br /> to eIF-4E [14]. Disruption signalling causes increased     imaginal discs allows side-by-side comparison wild- <br /> transcription 4E-BP owing increased FoxO activity. Con-      type versus mutant cells, generated poly loss-of-function <br /> sistent decrease InR &#8211;TOR signalling, level       clones imaginal discs. Staining mosaic discs revealed <br /> d4E-BP transcript threefold greater poly             increased levels cleaved caspase-3 poly mutant clones, <br /> mutant larvae compared control animals (figure 4f ).          discernible lack &#223;-galactosidase <br /> In addition, immunoblotting revealed d4E-BP               (figure 5d ). Cleaved caspase-3 detectable the <br /> hypo-phosphorylated poly mutant extracts compared               adjacent wild-type cells. <br /> with wild-type extracts (figure 4g). Taken together,             Thus, independent lines evidence demonstrate <br /> results consistent decrease cap-dependent            loss poly leads activation autophagy coupled <br />  <br />  <br />  <br />  <br />                                                                                                                                       Open Biol 2: 110031 <br /> translation loss Poly function.                            increase apoptotic cell death. <br />  <br />  <br /> 3.6. Autophagy constitutively active fat body            3.8. Metabolism affected poly mutant larvae <br />      poly larvae                                                Drosophila frequently used study metabolic regulation as <br />                                                                    fruitflies share majority metabolic functions ver- <br /> A critical consequence activation InR &#8211;TOR signal- <br />                                                                    tebrates [42]. The larval fat body main organ for <br /> ling inhibition autophagy. Drosophila undergoes <br />                                                                    regulation energy homeostasis excess energy stored <br /> developmentally programmed autophagy defined times <br />                                                                    form glycogen triglycerides (TAGs; lipids). Acti- <br /> to facilitate tissue remodelling metamorphosis <br />                                                                    vation signalling InR pathway promotes both <br /> [35 &#8211;38]. On hand, starvation-induced autophagy <br />                                                                    anabolic metabolism storage nutrients as <br /> (in response nutrient deprivation) takes place specifi- <br />                                                                    TAGs [43]. This important development, break- <br /> cally larval fat body [11]. As larval fat body <br />                                                                    larval fat body consequent release energy <br /> serves similar function vertebrate liver acting <br />                                                                    facilitates Drosophila metamorphosis. <br /> as nutrient storage organ, commonly used to <br />                                                                        In order identify genes differentially <br /> examine autophagy starvation-induced and <br />                                                                    expressed poly mutant larvae, carried microarray <br /> developmentally regulated autophagy [11,39,40]. While the <br />                                                                    analysis. Among list 106 genes downregulated poly <br /> fat body fed wild-type larvae does normally exhibit <br />                                                                    mutants, functional enrichment group analysis identified a <br /> autophagy, autophagy evident short period of <br />                                                                    strong enrichment gene ontology (GO) terms belonging <br /> amino acid starvation. <br />                                                                    metabolic processes, suggesting metabolism might <br />     TOR directly inhibits starvation-induced autophagy, <br />                                                                    affected poly mutants (table 2). We investi- <br /> while components InR signalling (such InR Akt, <br />                                                                    gated decrease InR &#8211;TOR activity also <br /> acting upstream TOR) behave negative regulators <br />                                                                    manifest metabolic level storage TAGs in <br /> of autophagy. We hypothesized poly acts InR signal- <br />                                                                    poly mutants. TAG levels normalized total protein <br /> ling, state autophagy visualized Lysotracker <br />                                                                    accurate quantification lipids unit mass in <br /> staining altered poly mutant fat body compared <br />                                                                    wild-type poly mutant larval extracts. Consistent a <br /> with wild-type fat body. As anticipated, Lysotracker <br />                                                                    decrease InR&#8211;TOR signalling, TAG : protein <br /> puncta observed fed wild-type early instar fat <br />                                                                    ratio half detected wild-type larvae (figure 6a). <br /> body (figure 5a, fed). However, Lysotracker puncta became <br />                                                                        Lipid storage droplet-2 protein (Lsd-2), protector against <br /> apparent following 4 h amino acid starvation, owing to <br />                                                                    lipolysis Drosophila perilipin homologue, localized <br /> the activation autophagy (figure 5a, starved). Strikingly, <br />                                                                    outer membrane lipid droplets, main storage <br /> Lysotracker puncta abundant fed poly fat <br />                                                                    organelle TAGs [44,45]. We level Lsd-2 <br /> bodies, demonstrating autophagy constitutively <br />                                                                    poly larval extracts decreased relative wild-type <br /> active fat body poly mutants (figure 5b, compare <br />                                                                    fat body, consistent decreased TAG : protein ratio <br /> fed starved). <br />                                                                    observed poly mutants (figure 6b). <br />                                                                        Together, reduction TAG storage Lsd-2 protein <br /> 3.7. Poly loss function leads increase                 poly mutant larvae suggest decrease anabolic <br />                                                                    metabolism, consistent with&#8212;and expected consequence <br />      apoptotic cell death                                          of&#8212;diminished InR&#8211; TOR signalling. <br /> Autophagy apoptosis closely related, increased <br /> levels autophagy lead apoptosis [39,41]. For <br /> example, increased autophagy resulting clonal <br />                                                                    3.9. The Poly protein relocalizes following insulin <br /> over-expression Atg1 wing disc resulted elevated           stimulation Drosophila haemocytes human <br /> apoptosis, evidenced appearance cleaved <br /> caspase-3 clones [39]. <br />                                                                         cultured cells <br />     Because loss poly led constitutive activation auto-   Given physical interaction Poly InR func- <br /> phagy fat body, investigated loss         tional links Poly InR&#8211; TOR signalling, were <br /> poly resulted increased apoptosis. Indeed, cell death      interested determine level and/or distribution <br /> increased dramatically instar larval neuroblasts       Poly cell dependent InR activity. <br /> poly larvae aged. This readily seen larval neuro-             Strikingly, staining Poly noticeably increased fol- <br /> blasts stained pS10-histone H3 TUNEL               lowing insulin stimulation larval haemocytes (cells the <br />                     (a)                             wild-type                            (b)                            poly                                                     8 <br />  <br />                                       fed                             starved                           fed                          starved <br />  <br />  <br />  <br />  <br />                                                                                                                                                                               rsob.royalsocietypublishing.org <br />                  Lysotracker <br />                    DNA Lysotracker <br />  <br />  <br />  <br />  <br />                                                                                                                                                                               Open Biol 2: 110031 <br />                                         (c) 8 <br />                                             7 <br />                                             6 <br />                                             5 <br />                                         % cells <br />  <br />  <br />  <br />  <br />                                             4 <br />                                             3 <br />                                             2 <br />                                             1 <br />                                             0 <br />                                                   6-wt 6-het      6       8     10    12     14        16     18      20      22 <br />                                                                                   age (days) <br />  <br />                               (d)                                     b-gal                      caspase-3                          DAPI <br />  <br />  <br />  <br />  <br /> Figure 5. poly mutations leads constitutive autophagy increased cell death. (a,b) Lysotracker (an acidic component-specific fluorescent dye) used monitor <br /> the state autophagy early instar larval fat body. Autophagy indicated punctate Lysotracker appearance. Live fat body tissues stained with <br /> Lysotracker (red) Hoechst 33342 <span id='am-139' about='rdfs:label' typeof='owl:Thing'>label</span> DNA (blue). (a) In wild-type larval fat body, autophagy evident. As indicated Lysotracker positive bright spots, <br /> autophagy active 4 h amino acid starvation 20 cent sucrose solution. (b) Lysotracker puncta detected fed poly fat body, starved poly <br /> fat body. (a,b) Scale bar, 50 mm. (c) Quantitation mitosis apoptosis larval neuroblasts development. While mitotic activity decreases, apoptosis increases <br /> during extended larval stage poly larva. Blue bars, p H3; red bars, TUNEL. (d) Images showing imaginal disc dissected instar larva genotype <br /> eyFLP; FRT82Bb-Gal/FRT82Bpoly05137. Immunostaining cleaved caspase-3 (red), b-galactosidase (green) DNA (blue) performed. poly mutant clones are <br /> indicated lack b-Gal staining. Scale bar, 30 mm. <br />  <br />  <br />  <br /> innate immune system). Prior isolation haemocytes,                                  insulin stimulation (figure 7a). Nearly, 60 cent cells <br /> larvae starved 3 h 20 cent sucrose. Haemo-                                showed relocalization Poly 60 min insulin stimu- <br /> cytes stimulated culture 200 nM insulin                                 lation, level increasing 100 cent 90 min. <br /> (electronic supplementary material, figure S3). This increased                                This rapid change HsPoly behaviour dependent <br /> staining Poly appears rapid response detect-                          TOR signalling. Incubation HeLa cells rapamycin <br /> able following 15 min insulin stimulation, levels                             overnight starvation period prevented increase <br /> of Poly remain elevated 75 min.                                                     HsPoly staining following insulin stimulation (figure 7b). <br />     A change appearance Poly following insulin                                  In rapamycin-treated cells, HsPoly remained evenly dis- <br /> stimulation conserved human cells. Overnight serum-                                tributed cell following 90 min of <br /> starved HeLa cells stimulated insulin                                 insulin treatment. <br /> 90 min. As Drosophila haemocytes, staining HsPoly                                      Taken together, observations demonstrate the <br /> was significantly stronger following insulin stimulation,                                 increase nuclear relocalization HsPoly following insu- <br /> appearing maximal 90 min. It additionally appeared                                lin treatment occurs TOR-dependent manner. Overall, <br /> this increase HsPoly concentrated near                                   results identify Poly novel component InR&#8211; TOR <br /> nucleus, relocalization evident 30 min                                 signalling conserved flies humans. <br />                                          (a)                1.2                        (b)                                                                                 9 <br />                                                                                                      wt       poly <br />  <br />  <br />  <br />  <br />                                     triglyceride:protein <br />                                                            1.0 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                         rsob.royalsocietypublishing.org <br />                                                                                       51 <br />  <br />  <br />  <br />  <br />                                          normalized <br />                                                            0.8 <br />                                                                        **                                                     Lsd-2 <br />                                                            0.6 <br />                                                                                       39 <br />                                                            0.4 <br />                                                            0.2                                                                &#181;-tub <br />                                                                                       51 <br />                                                             0 <br />                                                                  wt    poly <br /> Figure 6. poly mutant larvae characterized reduced levels TAGs Lsd-2. (a) TAG levels assessed wild-type poly mutant instar larvae. <br /> The total TAG level normalized total protein level. Error bars derive standard deviation independent experiments. The unpaired two- <br />  <br />  <br />  <br />  <br />                                                                                                                                                                         Open Biol 2: 110031 <br /> tailed t-test used compare data wild-type poly larvae. The double asterisk represents significant difference ( p , 0.01) TAG : protein ratio <br /> between control poly mutant larvae. (b) Immunoblotting carried wild-type poly mutant larval extracts antibody specifically recognizing <br /> Lsd2 protein. <br />  <br />  <br />  <br /> Table 2. Microarray analysis revealed metabolic functions genes downregulated poly mutants. DAVID (the Database for <br /> Annotation, Visualization Integrated Discovery) functional enrichment group analysis list differentially expressed genes identi&#65533;?ed enrichment in <br /> gene ontology (GO) terms belonging metabolic processes genes downregulated poly mutants. The data analysed Limma (see <br /> &#167;5.10) biological replicates. Genes selected functional/pathway analysis adjusted (corrected multiple testing) p-value was <br /> less 0.05. Log (base 2) fold changes given (log2FC). &#8216;rep&#8217; indicates reported gene function, &#8216;pred&#8217; indicates predicted gene function according to <br /> the gene-speci&#65533;?c FlyBase report. <br />  <br />   GO TERM                                                                      affected gene (log2FC)                                   fold-enrichment <br />  <br />   GO:0006739 NADP metabolic process                                            FBgn0004057 // CG12529 (21.42)                                   81.9 <br />                                                                                                                         rep <br />                                                                                glucose-6-phosphate dehydrogenase <br />                                                                                FBgn0023477 // CG2827 (21.27) <br />                                                                                                rep <br />                                                                                transaldolase <br />                                                                                FBgn0004654 // CG3724 (22.15) <br />                                                                                                                   rep <br />                                                                                phosphogluconate dehydrogenase <br />                                                                                FBgn0030239 // CG17333 (21.34) <br />                                                                                6-phosphogluconolactonase pred <br />                                                                                FBgn0037607 // CG8036 (21.39) <br />                                                                                transketolase pred <br />   GO:0006769 nicotinamide metabolic process                                    &#65533;?ve genes                                                   64.4 <br />   GO:0019362 pyridine nucleotide metabolic process                             &#65533;?ve genes                                                   60.1 <br />   GO:0006006 glucose metabolic process                                         &#65533;?ve genes,                                              16.4 <br />                                                                                FBgn0002569 // CG8694 (20.89) <br />                                                                                maltase A2 rep <br />  <br />                                                                                        growth. Phosphorylation Akt substrates contributes a <br /> 4. Discussion                                                                          range cellular processes including cell growth, proliferation <br /> Poly conserved protein plays novel essential role                       survival [46]. Dysregulation Akt involved various <br /> InR signalling and, crucially, promotes effects TOR                         diseases, including type-2 diabetes cancer [47,48]. S6K <br /> kinase cell growth metabolism.                                                  kinase downstream effectors InR&#8211; TOR <br />                                                                                        signalling, subject phosphorylation activation <br />                                                                                        TOR. Activation S6K leads increase translation <br /> 4.1. The activity InR signalling decreased                                    phosphorylation activation ribosomal <br />                                                                                        protein S6 [1]. <br />      poly larvae                                                                           Poly acts upstream of kinases InR <br /> Our data reveal essential involvement Poly InR sig-                          signalling pathway, levels phosphorylated (active) <br /> nalling, important pathway linking nutritional status                            dAkt dS6K kinases decreased poly larvae. That <br /> metabolism cell growth [3]. The kinases, Akt                               Poly acts activation dAkt dS6K also <br /> S6K, key effectors InR &#8211;TOR signalling. The Akt                                 supported genetic data revealed suppression a <br /> kinase located upstream TSC1&#8211;TSC2 complex                                 poly-induced rough eye phenotype mutations dAkt <br /> this pathway critical role promoting cell                                 dS6K. These kinases act genetic <br />                                     (a)                               poly                  DNA                    actin                                                 10 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                         rsob.royalsocietypublishing.org <br />                                     0 <br />                                     30 min <br />                                     60 min <br />  <br />  <br />  <br />  <br />                                                                                                                                                                         Open Biol 2: 110031 <br />                                     90 min <br />  <br />  <br />  <br />  <br />                                    (b) + rapamycin <br />                                     0 <br />                                     30 min <br />                                     60 min <br />                                     90 min <br />  <br />  <br />  <br />  <br /> Figure 7. Insulin stimulation HeLa cells leads increase HsPoly staining, accumulated nuclear area. (a) HeLa cells serum-starved overnight and <br /> then stimulated 100 nM insulin 30, 60 90 min prior staining HsPoly (red), DNA (blue) F-actin (green). Staining HsPoly indicated peri- <br /> nuclear accumulation 30 min, reaching maximum following 90 min insulin stimulation. (b) The relocalization HsPoly inhibited overnight <br /> incubation cells 20 nM rapamycin prior insulin treatment. Scale bars, 50 mm. <br />  <br /> suppressors situated downstream poly sig-                            Together data predict decrease InR&#8211;TOR signalling <br /> nalling pathway, resulting overall decrease InR &#8211;TOR                          absence Poly. As read-out TOR signalling, we <br /> signalling absence Poly.                                                     examined autophagy, multi-step, catabolic process used for <br />     The biochemical data presented revealed phys-                             nutrient recycling development starvation. TOR <br /> ical interaction Poly InR. Intriguingly, level                          activity responsible inhibiting autophagy cell <br /> Poly reduced InR mutant larval extracts indepen-                                growth-promoting conditions. Our finding autophagy is <br /> dently transcription, suggesting fraction Poly                            constitutively active fat body poly mutant larvae in <br /> may unstable subject degradation absence                              accord observation reduced InR&#8211;TOR signalling, <br /> of InR. Indeed, numerous examples instabil-                          consistent inhibition cell growth and/or prolifer- <br /> ity partner protein complex absent [49&#8211;51].                             ation poly mutant animals. Furthermore, increase in <br /> Whether Poly exists discrete complex InR,                                apoptotic cell death poly mutant animals clones generated <br /> how complex(es) regulated response                                   imaginal discs result increased levels autophagy <br /> insulin stimulation, currently investigation. Such                           mutation poly. <br /> an interaction transient highly dynamic, <br /> as Poly appears accumulate nucleus following <br /> insulin stimulation. Considering insulin-induced, TOR- <br />                                                                                        4.2. Metabolism disrupted poly mutants <br /> dependent nuclear enrichment Poly HeLa cells,                              A major function InR&#8211; TOR pathway modulation <br /> likely insulin-mediated signalling Poly conserved                          metabolism. For example, 4E-BP mutant animals show <br /> from flies humans.                                                                  increased sensitivity starvation [16]. Interestingly, was <br />  previously shown loss tumour suppressor                 If Poly role transcription, phylogenetic       11 <br /> PTEN (responsible dephosphorylation PIP3)            data suggest, explain binding transmem- <br />  <br />  <br />  <br />  <br />                                                                                                                                            rsob.royalsocietypublishing.org <br /> Drosophila nurse cells results accumulation activated    brane receptor? One explanation dynamic changes in <br /> Akt cytoplasm. This activated Akt drives formation      cellular localization Poly, occurring TOR-depen- <br /> of enlarged lipid droplets increase           dent manner. Interestingly, InR phosphorylation IRS-1 <br /> expression Drosophila Lsd-2 [52]. Lsd-2 mutants             IRS-2 (two human IRS homologues) only <br /> characterized decreased TAG levels [45]. Interestingly,         leads activation PI3K signalling, associated <br /> autophagy lipid metabolism shown inter-         IRS-1 translocation nucleus, activates <br /> linked processes, suggested decrease autophagy          transcription various genes [59 &#8211;61]. Given Poly also <br /> resulting increase lipid storage cell [53]. Con-   interacts InR, moves nucleus following insu- <br /> sistent decreased InR&#8211; TOR signalling absence          lin stimulation, possible changes localization <br /> of poly, Lsd-2 TAG : protein ratio reduced       Poly ultimately response InR signalling. <br />  <br />  <br />  <br />  <br />                                                                                                                                            Open Biol 2: 110031 <br /> in poly mutant animals. Therefore, propose Poly            Immunofluorescence Drosophila haemocytes and <br /> affects metabolism interaction InR&#8211;TOR path-      HeLa cells demonstrated level and/or distribution <br /> way, acting positive regulator anabolic metabolism.        Poly significantly affected following insulin stimulation. <br />                                                                    Importantly, HeLa cells, Poly relocalization nucleus <br /> 4.3. Comparison poly mutants                       occurred rapamycin-sensitive manner. Thus, speculate <br />                                                                    stimulatory effects Poly cell growth metab- <br />      Elongator                                                     olism exerted effects transcription. However, <br /> Numerous aspects poly phenotype observed          analysis required assess Poly participates <br /> in mutations, including InR TOR. An              Elongator complex and, so, myriad <br /> extreme larval longevity, remarkable aspects       functions currently ascribed Elongator. <br /> of poly mutant phenotype, observed Tor             Future research address detailed nature the <br /> [54] InR mutants (this study). The appearance melano-       interaction Poly InR. Does Poly interact with <br /> tic masses frequently seen mutations aberrant           InR absence presence insulin? How dynamic <br /> immune responses and/or haematopoietic defects [55&#8211;57].            interaction? And level post-translational state <br /> In poly instar larvae, melanotic masses appearing         Poly modified insulin treatment? <br /> multiple different locations larval body likely           In light data presented herein, propose Poly <br /> to result alteration immune response         novel mediator InR&#8211;TOR signalling regulation of <br /> and/or hyper-proliferation blood cells, haemocytes.        cell metabolism growth Drosophila (figure 8). We suggest <br /> The observation InR mutant larvae reach          physical interaction Poly InR followed <br /> the instar stage develop melanotic masses high-         translocation Poly nucleus (upon insulin stimu- <br /> lights phenotypic similarities poly InR loss-of-        lation), expression key metabolic genes affected, <br /> function mutations, corroborates functional          contributing promotion cell growth metab- <br /> relationship poly InR.                                 olism. While detailed nature regulation Poly&#8217;s <br />     Given apparent similarity Poly yeast Elp6,     interaction InR remain elucidated, highly <br /> therefore intriguing recently described mutant phe-       likely that, given conservation Poly, crucial interaction <br /> notype Elongator Elp3 catalytic subunit Drosophila      function hold true human cells. <br /> resembles poly [28]. These observations suggestive <br /> of role Elongator holocomplex insulin signalling&#8212; <br /> through histone and/or tubulin acetylation&#8212;or transla-        5. Material methods <br /> tional control tRNA modification. In addition, <br /> as human mutations Elp3 linked familial            5.1. Drosophila culture <br /> dysautonomia [58], likely future analysis of <br />                                                                    All fly stocks maintained 258C standard medium <br /> Elongator function model organisms significant <br />                                                                    unless stated. Fly stocks used study were: <br /> preclinical value. <br />                                                                    Canton S (wild-type), poly05137/TM6B. The following fly <br />                                                                    lines obtained Bloomington Stock Center: <br /> 4.4. Model Poly action                                         InR05545/TM3, Akt104226/TM3, S6K07084/TM3, Cg-Gal4. <br /> We shown Poly binds InR (a transmembrane <br /> receptor) modulates activity various downstream <br /> proteins. While lack clearly discernible functional <br />                                                                    5.2. Phylogenetic analysis <br /> motifs hampered prediction Poly&#8217;s molecular function,         Sequences homologous Drosophila Poly identified by <br /> extensive database searches phylogenetic analyses ident-       multiple iterative searches using PSI-BLAST program [62] <br /> ified Poly member Elp6 subfamily Elongator          HHpred interactive server [63]. Alignments between <br /> proteins, distant homology various members            corresponding sequences generated using M-COFFEE <br /> of RecA/Rad51/DCM1 <span id='am-126' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-127' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-128' property="rdfs:label" content="superfamily" datatype="xsd:string"></span>superfamily</span> (such KaiC               [64] manually adjusted based predicted secondary <br /> RadB); observations described distinct study        structure according ALI2D [65]. Identical similar <br /> from laboratory. Both KaiC Elp6 shown            amino acids representative subset aligned sequences <br /> to significantly affect gene expression, pointing degree      shaded using BOXSHADE server (http://www.ch. <br /> of functional conservation. Thus, suggest involvement        embnet.org/software/BOX_form.html). Phylogenetic trees <br /> for Poly transcription ( relocated      calculated using TREE-PUZZLE [66], MRBAYES [67] and <br /> nucleus following insulin stimulation).                            PROML program PHYLIP Package [68] clustering <br />                          (a)                                                                 (b)                                                                                           12 <br />                                                         insulin                                                               insulin <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                           rsob.royalsocietypublishing.org <br />                                                     InR                                                                  InR <br />  <br />                                                                                                              Poly <br />                                    Poly                            Akt                                                               Akt <br />  <br />  <br />                                          positive                           TOR                                                             TOR <br />                                         regulation <br />  <br />                                           lsd2              S6K                                                  lsd2       S6K <br />                                                                                                    cytosol <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                           Open Biol 2: 110031 <br />                                                       autophagy               4E-BP                                  autophagy                4E-BP <br />  <br />  <br />  <br />                                            Poly          transcription                                                    transcription <br />  <br />                                               metabolic genes                                                            disrupted <br />  <br />                                                     wild-type                                                           loss poly <br /> Figure 8. Model function Poly InR&#8211;TOR signalling pathway. The interaction Poly (a) insulin receptor allows fulfil role positive regulator of <br /> InR&#8211;TOR signalling, overall effect increasing phosphorylation and, hence, activity numerous positive regulators InR&#8211;TOR pathway, Akt, S6K <br /> and 4E-BP, inhibiting negative regulators cell growth autophagy. Furthermore, Poly aids promote anabolic metabolism resulting increase TAG and <br /> Lsd-2 levels. (b) The absence Poly leads decrease activation positive regulators cell growth Akt S6K kinases. The 4E-BP protein hypo- <br /> phosphorylated, consistent overall downregulation InR&#8211;TOR signalling. Furthermore, autophagy&#8212;a negative effector InR/TOR signalling&#8212;becomes constitutively <br /> active. Anabolic metabolism downregulated indicated decrease TAG Lsd-2 levels. <br />  <br /> of related sequences smaller groups using SPLITSTREE4 [69].                                  MS). A high-performance liquid chromatography (HPLC) <br /> Branch lengths calculated application WAG                                          instrument (Dionex, UK) interfaced LTQ ion <br /> substitution matrix [70] using TREE-PUZZLE, modelling rate                                        trap mass spectrometer (ThermoFinnigan, CA). Chromato- <br /> heterogeneity according gamma distribution 16                                           graphy buffer solutions (buffer A: 5% acetonitrile 0.1% <br /> rate categories, bootstrap confidence intervals ( provided                                    formic acid; buffer B: 80% acetonitrile 0.1% formic <br /> in electronic supplementary material, figure S1) esti-                                       acid) used deliver 60 min gradient (35 min 45% <br /> mated using SEQBOOT program PHYLIP Package [68].                                       buffer B, 10 min 90%, hold 10 min, 3 min 5% hold <br />                                                                                                   15 min). A flow rate 2 ml min21 used electro- <br /> 5.3. Immunoprecipitation                                                                          spray source. One scan followed 10 MS/MS <br />                                                                                                   events, duty cycle programmed enforce dynamic <br /> Fifty microlitres 0&#8211; 5 h wild-type embryos (raised 188C)                                    exclusion 2 min. In-house proteomics pipeline software <br /> were homogenized 1 ml cold lysis buffer (100 mM NaCl,                                       (PROLINE) used process data. Spectra searched <br /> 10 mM EDTA, 50 mM Tris &#8211;HCl pH 7.6, 0.1% Triton-X100,                                             using Sequest algorithm [72] SwissProt.2007.04.19 <br /> 10 mg ml21 chymostatin, leupeptin, antipain pep-                                      database restricted Drosophila <span id='am-156' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-157' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-159' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span><span id='am-160'  ></span>melanogaster</span> entries. Proteins <br /> statin, 50 mg ml21 PMSF) briefly sonicated. Eight                                         (i) peptide prophet probability score greater 0.99 <br /> microlitres rabbit pre-immune anti-poly serum coupled                                       [73] (ii) identified minimum different peptide <br /> to 40 ml Dynal beads (Invitrogen) added pre-                                           spectra automatically accepted, spectra the <br /> cleared embryo lysates overnight. Ten cent                                            minority proteins identified single spectra were <br /> sample subjected immunoblot analysis verify suc-                                        manually checked quality. <br /> cessful immunoprecipitation Poly, remaining <br /> 90 cent sample resolved precast 4&#8211;12 cent <br /> Bis-Tris polyacrylamide gels (Novex) stained                                             5.5. Larval manipulations <br /> Colloidal Coomassie Blue (Invitrogen). Comparable molecu-                                         Prior dissections protein extractions, 10&#8211;15 second instar <br /> lar weight regions excised lane                                        larvae transferred fresh vial food supplemented <br /> ( pre-immune- immune-precipitations) mass spec-                                           fresh yeast paste. Manipulations carried when <br /> trometry analysis performed (Dr Gerard Cagney, Dublin).                                           larvae reached early instar stage. For starvation exper- <br />                                                                                                   iments, early instar larvae starved 20 cent <br /> 5.4. Mass spectrometry                                                                            sucrose solution 3&#8211;4 h prior dissection. <br /> The proteins slices sodium dodecyl sulphate poly- <br /> acrylamide gel electrophoresis (SDS &#8211;PAGE) gels                                              5.6. RNA extraction quantitative reverse <br /> digested in-gel trypsin method Shevchenko <br /> et al. [71]. The resulting peptide mixtures resuspended <br />                                                                                                        transcriptase &#8211;polymerase chain reaction <br /> in 1 cent formic acid analysed nano-electrospray                                       Larvae transferred RNase-free Eppendorf tubes <br /> liquid chromatography mass spectrometry (nano-LC MS/                                              (Ambion). RNA extracted Qiagen RNeasy <br />  Plus Kit (Qiagen Hilden, Germany) according manufac-            (Molecular Probes), 1 mM Hoechst 33342 (Molecular              13 <br /> turer&#8217;s instructions. cDNA synthesis carried            Probes) PBS. Fat bodies mounted phosphate- <br />  <br />  <br />  <br />  <br />                                                                                                                                        rsob.royalsocietypublishing.org <br /> using SuperScript III (Invitrogen) following manufac-      buffered saline (PBS) glass slide visualized using <br /> turer&#8217;s instructions. Real-time quantitative RT-PCR analysis       Olympus AX-70 Provis epifluorescence microscope and <br /> was performed LightCycler (Roche) using          Hamamatsu Orca II charge coupled device (CCD) <span id='am-90' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-91' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-92'  ></span><span id='am-93' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>camera</span>. <br /> universal probe library. The reactions set following       Images captured using SMARTCAPTURE 3 processed <br /> manufacturer&#8217;s recommendation LightCycler                 using Adobe PHOTOSHOP. <br /> master mix kit. The relative cDNA ratio calculated with <br /> Lightcycler software 480. Actin5C used control to <br /> normalize equal loading template cDNA. <br />                                                                    5.10. Microarray processing analysis <br />                                                                    Ten second instar wild-type Canton S poly larvae <br />                                                                    transferred vials supplemented fresh yeast paste. <br /> 5.7. Preparation larval protein extracts for <br />  <br />  <br />  <br />  <br />                                                                                                                                        Open Biol 2: 110031 <br />                                                                    Early instar larvae transferred TRIzol 24 h later <br />      immunoblotting                                                RNA extraction. All microarray processing Flychip <br />                                                                    team University Cambridge (http://www.flychip.org. <br /> Fifteen wandering instar larvae appro- <br />                                                                    uk/). RNA wild-type Canton S poly early instar <br /> priate genotype placed 1.5 ml tubes rinsed three <br />                                                                    larvae extracted (medium scale) using TRIzol (http:// <br /> times Ephrussi&#8211;Beadle Ringer&#8217;s solution (130 mM <br />                                                                    www.flychip.org.uk/protocols/gene_expression/standard_ <br /> NaCl, 4.7 mM KCl, 1.9 mM CaCl2, 10 mM HEPES pH <br />                                                                    extraction.php). RNA reverse transcribed labelled using <br /> 6.9). Next, 300ml cold lysis buffer (Ephrussi&#8211;Beadle Ringer&#8217;s <br />                                                                    Klenow labelling: 5 mg total RNA reverse transcribed to <br /> solution 10 mM EDTA, 10 mM DTT, 1 mg ml21 each <br />                                                                    cDNA (anchored oligo (dT)23 (Sigma), Superscript III (Invitro- <br /> of chymostatin, leupeptin, antipain pepstatin [CLAP, <br />                                                                    gen)) second strand synthesis performed (Second <br /> Sigma], 1 mM phenylmethanesulphonyl fluoride [PMSF, <br />                                                                    strand buffer (Invitrogen), DNA Polymerase I (Invitrogen), <br /> Sigma], 1 unit Aprotinin [Calbiochem] added to <br />                                                                    RNaseH (New England Biolabs), <span id='am-98' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-99' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-110'  ></span><span id='am-111' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia coli</span> DNA ligase <br /> the tubes. The larvae homogenized using motor- <br />                                                                    (GE Healthcare)) obtain double-stranded DNA. Random <br /> ized hand pestle starting lowest speed gently <br />                                                                    primers annealed 500 ng denatured DNA template <br /> increasing speed maximum approximately <br />                                                                    extended Klenow fragment using Bioprime <br /> 2 min. Then, 150 ml hot (708C) 3X SDS-PAGE Sample <br />                                                                    DNA Labeling System (Invitrogen), fluorescent dyes <br /> Buffer containing 10 mM DTT added homogenate <br />                                                                    Cy3-dCTP Cy5-dCTP (GE Healthcare) incorpora- <br /> and tube placed 1008C 10 min. Particulate matter <br />                                                                    ted (http://www.flychip.org.uk/protocols/gene_expression/ <br /> was pelleted 13 000 r.p.m. 2 min, supernatant <br />                                                                    klenow_v2.php). Hybridization FL002 microarrays: hybridiz- <br /> transferred fresh tube. Samples stored 2208C <br />                                                                    ation amino-modified long oligonucleotide microarrays <br /> until required. <br />                                                                    using Genomic Solutions hybridization station Bioso- <br />                                                                    lutions hybridization buffer (http://www.flychip.org.uk/ <br /> 5.8. Sodium dodecyl sulphate polyacrylamide gel                    protocols/gene_expression/hyb_oligoMWG.php). Scanning <br />                                                                    Genepix 4000B dual <span id='am-151' about='obo:NCBITaxon_62990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_241799 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_241789 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-152' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-154' property="rdfs:label" content="Laser" datatype="xsd:string"></span><span id='am-155'  ></span>laser</span> scanner 5 mM <br />      electrophoresis immunoblotting                            pixel resolution (http://www.flychip.org.uk/protocols/gene_ <br /> Protein samples resolved 170 V SDS-PAGE              expression/scanning2.php). Spot finding quantitation <br /> precast 4&#8211;12% Bis-Tris polyacrylamide gels (Novex)             DAPPLE package. Raw data normalized by <br /> transferred nitrocellulose membranes Trans-Blot          vsn method BIOCONDUCTOR (http://www.bioconductor. <br /> apparatus (Biorad). Membranes blocked TBSTw (Tris-         org/packages/2.8/bioc/html/vsn.html) generate log <br /> buffered saline (150 mM NaCl, 20 mM Tris pH 7.5) &#254; 0.05%           (base 2) fold changes average signals. Differential expres- <br /> Tween 20) 5% (w/v) semi-skimmed milk 1 h room           sion tested using Limma, BIOCONDUCTOR (http:// <br /> temperature incubated 1 h primary antibody       www.bioconductor.org/packages/2.8/bioc/html/limma.html). <br /> in TBSTw. After washing times 5 min TBSTw,      GO KEGG biological pathway enrichment differen- <br /> membranes incubated appropriate horseradish per-        tially expressed gene set assessed using DAVID <br /> oxidase-linked secondary antibody (1 : 10 000) 1 h TBSTw    functional annotation bioinformatics microarray analysis <br /> at room temperature. Finally, membranes washed         tool (http://david.abcc.ncifcrf.gov/). Microarray data were <br /> above Tris-buffered saline plus 0.2% Triton X immune-       deposited gene expression omnibus (GEO) the <br /> complexes detected enhanced chemiluminescence (ECL;             accession number GSE32637 (http://www.ncbi.nlm.nih.gov/ <br /> Amersham Biosciences).                                             geo/query/acc.cgi?acc=GSE32637). <br />      Primary antibodies dilutions used immunoblotting <br /> experiments follows: DmPoly 504 (1 : 1000), anti-Isd2 <br /> (1 : 2000), p-Akt antibody (1 : 1000), p-S6K antibody (1 : 1000) <br />                                                                    5.11. Triglyceride assay larvae <br /> and p-4E-BP antibody (1 : 200). The anti-Isd2 antibody       TAG assay larvae carried described Gronke <br /> generous gift Ronald Ku&#168;hnlein (MPI, Go&#168;ttingen),       et al. [75]. Briefly, larvae corresponding gen- <br /> the purchased Cell Signaling Technology.      otype collected 500 ml 0.05 cent Tween 20 and <br />                                                                    homogenized using Polytron apparatus, followed treat- <br />                                                                    ment 708C 5 min. Samples centrifuged 1 min at <br /> 5.9. Lysotracker staining                                          3500 r.p.m. 350 ml supernatant transferred to <br /> Lysotracker staining performed described [74]. Fat          new vial centrifuged 3 min 2500 r.p.m. Six hun- <br /> body fed starved larvae dissected PBS,          dred microlitres Thermo Infinity Trig solution (Thermo <br /> incubated 2 min 100 nM LysoTracker Red DND-99               <span id='am-134' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-135'  ></span><span id='am-136' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-137' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Electron</span>, 981786) added 75 ml isolated supernatant <br />  and absorbance 540 nm measured following incu-                                     5.14. Immunofluorescence HeLa cells                                                 14 <br /> bation 30 min. Similarly, protein content determined <br />                                                                                               HeLa cells cultured coverslips six-well plates over- <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                      rsob.royalsocietypublishing.org <br /> by adding 10 ml isolated supernatant Bradford reagent <br /> (Sigma, B6916) reading absorbance 595 nm. TAG                                      night. Cells rinsed PBS 3 min followed 3 min <br /> levels normalized corresponding protein levels.                                       fixation PBS containing 4 cent paraformaldehyde <br />                                                                                               (diluted 16% ampoules). The fixative removed by <br />                                                                                               2 min wash PBS. Cells permeabilized 5 min <br /> 5.12. Antibody staining haemocytes                                                         incubation PBS &#254; 0.5% Triton X-100 followed 1 h <br />                                                                                               block PBS containing 3 cent BSA. Subsequently, cells <br /> Third instar larvae bled multispot microscope slides                                  washed PBS &#254; 0.1% Triton X-100 5 min. Primary <br /> (Hendley-Essex) using pair forceps 25-gauge                                        antibody diluted PBS &#254; 0.3% BSA &#254; 0.1% Triton <br /> needle 20 ml PBS. Cells left settle slide                            X-100 incubated cells 1 h room temperature. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                      Open Biol 2: 110031 <br /> 1 h room temperature humidified chamber allow                                      Cells washed twice 5 min followed 1 h second- <br /> adherence slide. Cells fixed 20 ml 3.7                                ary antibody incubation diluted PBS &#254; 0.3% BSA &#254; 0.1% <br /> cent paraformaldehyde PBS 5 min. Cells washed                                     Triton X-100. Subsequently, 45 min washes performed <br /> with PBS 5 min, followed 5 min permeabilization                                   PBS &#254; 0.1% Triton X-100. DAPI included the <br /> PBS &#254; 0.1% Triton X-100. Following additional 5 min wash                                   penultimate wash 0.1 mg ml21 concentration. <br /> in PBS, blocking performed incubating cells PBS &#254;                                       Primary rabbit anti-HsPoly serum used immuno- <br /> 3% BSA 1 h. Cells incubated primary anti-                                  fluorescence experiments 1 : 1000 dilution. Texas-Red <br /> body diluted PBS &#254; 3% BSA overnight 48C. The                                            conjugated goat anti-rabbit (1 : 500) secondary antibody <br /> following day cells washed PBS times incu-                                  <span id='am-11' about='obo:NCBITaxon_149625' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_163739 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-12' property="rdfs:label" content="Alexa" datatype="xsd:string"></span><span id='am-13'  ></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Alexa</span> Fluor-488-conjugated phalloidin (1 : 500) were <br /> bated room temperature secondary antibody diluted                                     purchased Molecular Probes. <br /> in PBS &#254; 3% BSA. Following secondary antibody, incu- <br /> bation cells washed twice PBS 5 min. Cells were <br /> incubated DAPI diluted PBS (1 : 5000 dilution)                                    6. Acknowledgements <br /> 5 min followed final 5 min wash PBS. Coverslips were <br /> mounted mowiol slide.                                                      The authors like express gratitude Dr <br />      Primary rabbit anti-DmPoly serum used immuno-                                     Gerard Cagney (Conway Institute, University College <br /> fluorescence experiments 1 : 1000 dilution. Alexa Fluor-488                              Dublin) invaluable assistance mass spectrometry <br /> anti-rabbit (1 : 500) Alexa Fluor-594 conjugated phalloidin                               experiments. We thank Kasper Pedersen the <br /> (1 : 500) purchased Molecular Probes.                                               Conway Institute Mass Spectrometry Resource exper- <br />                                                                                               imental support, Andreas Stefani bioinformatics <br />                                                                                               support. Emanuela Giacometti, Josefin Fernius, Shubha <br />                                                                                               Gururaja Rao, Swati Naidu Mei Xuan Lye valuable <br /> 5.13. HeLa cell culture                                                                       contributions various stages project. We thank <br /> HeLa cells maintained Dulbecco&#8217;s modified Eagle&#8217;s                                     William Earnshaw Paul Hartley critical feedback on <br /> medium (DMEM; Sigma) supplemented 10 cent foetal                                     manuscript. E.B., S.V., N.C. M.M.S.H. sup- <br /> bovine serum (FBS; Gibco). Cells grown 80&#8211;90% conflu-                                 ported The Wellcome Trust. B.N. supported the <br /> ence overnight serum withdrawal. For insulin stimulation,                              Natural Sciences Engineering Research Council of <br /> serum-starved cells treated 100 nM insulin (Sigma).                                 Canada, V.S. MSc student University of <br /> Cells treated 20 nM rapamycin (Cell Signaling                                       Edinburgh. D.R.D. supported British Heart Foun- <br /> Technology) overnight serum starvation.                                             dation Centre Research Excellence Award. <br />  <br />  <br />  <br /> References <br /> 1.   Wullschleger S, Loewith R, Hall MN. 2006                 5.   Puig O et al. 2003 Control cell number          9.    Potter CJ, Huang H, Xu T. 2001 Drosophila Tsc1 <br />      TOR signaling growth metabolism.                       Drosophila FOXO: downstream feedback                    functions Tsc2 antagonize insulin signaling <br />      Cell 124, 471&#8211;484. (doi:10.1016/j.cell.2006.                  regulation insulin receptor pathway.                 regulating cell growth, cell proliferation, and <br />      01.016)                                                       Genes Dev. 17, 2006&#8211;2020. (doi:10.1101/gad.                 organ size. Cell 105, 357&#8211; 368. (doi:10.1016/ <br /> 2.   Grewal SS. 2009 Insulin/TOR signaling growth               1098703)                                                    S0092-8674(01)00333-6) <br />      homeostasis: view fly world.              6.   Manning BD. 2004 Balancing Akt S6K:              10.   Long X et al. 2005 Rheb binds regulates the <br />      Int. J. Biochem. Cell Biol. 41, 1006&#8211; 1010. (doi:10.          implications metabolic diseases                mTOR kinase. Curr. Biol. 15, 702&#8211;713. (doi:10. <br />      1016/j.biocel.2008.10.010)                                    tumorigenesis. J. Cell Biol. 167, 399&#8211;403.                  1016/j.cub.2005.02.053) <br /> 3.   Britton JS et al. 2002 Drosophila&#8217;s insulin/PI3-kinase        (doi:10.1083/jcb.200408161)                           11.   Scott RC, Schuldiner O, Neufeld TP. 2004 Role and <br />      pathway coordinates cellular metabolism             7.   Cai SL et al. 2006 Activity TSC2 inhibited         regulation starvation-induced autophagy the <br />      nutritional conditions. Dev. Cell 2, 239&#8211;249.                 AKT-mediated phosphorylation membrane                   Drosophila fat body. Dev. Cell. 7, 167&#8211; 178. <br />      (doi:10.1016/S1534-5807(02)00117-X)                           partitioning. J. Cell Biol. 173, 279 &#8211;289. (doi:10.         (doi:10.1016/j.devcel.2004.07.009) <br /> 4.   Hennig KM, Colombani J, Neufeld TP. 2006 TOR                  1083/jcb.200507119)                                   12.   Radimerski T et al. 2002 dS6K-regulated cell growth <br />      coordinates bulk targeted endocytosis         8.   Gao X, <span id='am-77' about='obo:NCBITaxon_9596' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-78' property="rdfs:label" content="Pan" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasExactSynonym" content="chimpanzees" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81'  ></span><span id='am-82' property="oboInOwl:hasAlternativeId" content="NCBITaxon:317028" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasAlternativeId" content="NCBITaxon:445518" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Pan</span> D. 2001 TSC1 TSC2 tumor                      dPKB/dPI(3)K-independent, requires dPDK1. <br />      Drosophila melanogaster fat body regulate cell             suppressors antagonize insulin signaling cell            Nat. Cell. Biol. 4, 251&#8211;255. (doi:10.1038/ncb763) <br />      growth. J. Cell Biol. 173, 963&#8211;974. (doi:10.1083/             growth. Genes Dev. 15, 1383&#8211;1392. (doi:10.1101/       13.   Sarbassov DD et al. 2008 Phosphorylation and <br />      jcb.200511140)                                                gad.901101)                                                 regulation Akt/PKB rictor-mTOR complex. <br />        Science 307, 1098 &#8211;1101. (doi:10.1126/science.               Acids Res. 30, 3643&#8211;3652. (doi:10.1093/nar/                  Dev. 120, 1071&#8211;1081. (doi:10.1016/S0925-                    15 <br />       1106148)                                                     gkf498)                                                      4773(03)00158-8) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                             rsob.royalsocietypublishing.org <br /> 14.   Marr MT et al. 2007 IRES-mediated functional           30.   Figeys D, McBroom LD, Moran MF. 2001 Mass              46.   Manning BD, Cantley LC. 2007 AKT/PKB signaling: <br />       coupling transcription translation amplifies          spectrometry study protein&#8211;protein                navigating downstream. Cell 129, 1261&#8211;1274. <br />       insulin receptor feedback. Genes Dev. 21,                    interactions. Methods 24, 230&#8211; 239. (doi:10.1006/            (doi:10.1016/j.cell.2007.06.009) <br />       175&#8211;183. (doi:10.1101/gad.1506407)                           meth.2001.1184)                                        47.   Bellacosa A et al. 2005 Activation AKT kinases in <br /> 15.   Puig O, Tjian R. 2005 Transcriptional feedback         31.   Shiio Y et al. 2002 Quantitative proteomic analysis          cancer: implications therapeutic targeting. Adv. <br />       control insulin receptor dFOXO/FOXO1. Genes            Myc oncoprotein function. Embo J. 21, 5088&#8211;               Cancer Res. 94, 29 &#8211;86. (doi:10.1016/S0065- <br />       Dev. 19, 2435&#8211;2446. (doi:10.1101/gad.1340505)                5096. (doi:10.1093/emboj/cdf525)                             230X(05)94002-5) <br /> 16.   Teleman <span id='am-140' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-141'  ></span><span id='am-142' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-143' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>AA</span>, Chen YW, Cohen SM. 2005 4E-BP              32.   Fernandez R et al. 1995 The Drosophila insulin         48.   Engelman JA, Luo J, Cantley LC. 2006 The evolution <br />       functions metabolic brake used stress             receptor homolog: gene essential embryonic             phosphatidylinositol 3-kinases regulators of <br />       conditions normal growth. Genes               development encodes receptor isoforms               growth metabolism. Nat. Rev. Genet. 7, <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                             Open Biol 2: 110031 <br />       Dev. 19, 1844&#8211;1848. (doi:10.1101/gad.341505)                 different signaling potential. Embo J. 14,                   606&#8211;619. (doi:10.1038/nrg1879) <br /> 17.   Mizushima N et al. 2008 Autophagy fights disease             3373 &#8211;3384.                                            49.   Ciapponi L et al. 2004 The Drosophila Mre11/Rad50 <br />       cellular self-digestion. Nature 451,           33.   Brand AH, Perrimon N. 1993 Targeted gene                     complex required prevent telomeric <br />       1069&#8211;1075. (doi:10.1038/nature06639)                         expression means altering cell fates             fusion chromosome breakage. Curr. Biol. 14, <br /> 18.   Levine B, Kroemer G. 2008 Autophagy                   generating dominant phenotypes. Development                  1360&#8211; 1366. (doi:10.1016/j.cub.2004.07.019) <br />       pathogenesis disease. Cell 132, 27&#8211; 42. (doi:10.          118, 401 &#8211;415.                                         50.   Savvidou E et al. 2005 Drosophila CAP-D2 required <br />       1016/j.cell.2007.12.018)                               34.   Hay N, Sonenberg N. 2004 Upstream                        condensin complex stability resolution of <br /> 19.   Karpen GH, Spradling AC. 1992 Analysis                    downstream mTOR. Genes Dev. 18, 1926&#8211;1945.                sister chromatids. J. Cell. Sci. 118, 2529&#8211;2543. <br />       subtelomeric heterochromatin Drosophila               (doi:10.1101/gad.1212704)                                    (doi:10.1242/jcs.02392) <br />       minichromosome Dp1187 single P element              35.   Butterworth FM, Forrest EC. 1984 Ultrastructure        51.   Vass S et al. 2003 Depletion Drad21/Scc1 Drosophila <br />       insertional mutagenesis. Genetics 132, 737 &#8211;753.             preparative phase cell death                cells leads instability cohesin complex and <br /> 20.   Klusza S, Deng WM. 2010 poly required nurse-          larval fat body Drosophila melanogaster. Tissue           disruption mitotic progression. Curr. Biol. 13, <br />       cell chromosome dispersal oocyte polarity             Cell.     16,      237&#8211;250.      (doi:10.1016/0040-          208&#8211;218. (doi:10.1016/S0960-9822(03)00047-2) <br />       Drosophila. Fly (Austin) 4, 128&#8211;136.                         8166(84)90047-8)                                       52.   Vereshchagina N, Wilson C. 2006 Cytoplasmic <br /> 21.   Krogan NJ, Greenblatt JF. 2001 Characterization        36.   Butterworth FM, Emerson L, Rasch EM. 1988                    activated protein kinase Akt regulates lipid-droplet <br />       six-subunit holo-elongator complex required             Maturation degeneration fat body           accumulation Drosophila nurse cells. Development <br />       regulated expression group genes             Drosophila larva <span id='am-121' about='obo:NCBITaxon_96459' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_71510 obo:NCBITaxon_71511 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_977774 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_6072'><span id='am-122' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-123'  ></span><span id='am-124' property="rdfs:label" content="Pupa" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>pupa</span> revealed                     133, 4731&#8211;4735. (doi:10.1242/dev.02659) <br />       Saccharomyces cerevisiae. Mol. Cell Biol.                 morphometric analysis. Tissue Cell 20, 255&#8211;268.        53.   Singh R et al. 2009 Autophagy regulates lipid <br />       21, 8203&#8211;8212. (doi:10.1128/MCB.21.23.8203-                  (doi:10.1016/0040-8166(88)90047-X)                           metabolism. Nature 458, 1131&#8211;1135. (doi:10. <br />       8212.2001)                                             37.   Juhasz G et al. 2003 The Drosophila homolog               1038/nature07976) <br /> 22.   Winkler GS et al. 2001 RNA polymerase II elongator           Aut1 essential autophagy development.       54.   Zhang H et al. 2000 Regulation cellular <br />       holoenzyme composed discrete                       FEBS Lett. 543, 154 &#8211;158. (doi:10.1016/S0014-                growth Drosophila target rapamycin dTOR. <br />       subcomplexes. J. Biol. Chem. 276, 32 743&#8211;                    5793(03)00431-9)                                             Genes Dev. 14, 2712&#8211;2724. (doi:10.1101/gad. <br />       32 749. (doi:10.1074/jbc.M105303200)                   38.   Rusten TE et al. 2004 Programmed autophagy                835000) <br /> 23.   Hawkes NA et al. 2002 Purification                       Drosophila fat body induced ecdysone         55.   Braun A, Hoffmann JA, Meister M. 1998 Analysis of <br />       characterization human elongator complex.             regulation PI3K pathway. Dev. Cell            Drosophila host defense domino mutant <br />       J. Biol. Chem. 277, 3047&#8211; 3052. (doi:10.1074/jbc.            7, 179&#8211; 192. (doi:10.1016/j.devcel.2004.07.005)              larvae, devoid hemocytes. Proc. Natl <br />       M110445200)                                            39.   Scott RC, Juhasz G, Neufeld TP. 2007 Direct                  Acad. Sci. USA. 95, 14 337 &#8211;14 342. (doi:10.1073/ <br /> 24.   Nelissen H et al. 2005 The elongata mutants                  induction autophagy Atg1 inhibits cell growth          pnas.95.24.14337) <br />       identify functional Elongator complex plants            induces apoptotic cell death. Curr. Biol. 17,      56.   Dearolf CR. 1998 Fruit fly &#8216;leukemia&#8217;. Biochim. <br />       role cell proliferation organ               1 &#8211;11. (doi:10.1016/j.cub.2006.10.053)                       Biophys. Acta 1377, M13&#8211;M23. <br />       growth. Proc. Natl Acad. Sci. USA. 102, 7754 &#8211;         40.   Juhasz G et al. 2007 Gene expression profiling         57.   Minakhina S, Steward R. 2006 Melanotic mutants in <br />       7759. (doi:10.1073/pnas.0502600102)                          identifies FKBP39 inhibitor autophagy            Drosophila: pathways phenotypes. Genetics <br /> 25.   Winkler GS et al. 2002 Elongator histone H3             larval Drosophila fat body. Cell Death Differ. 14,           174, 253&#8211;263. (doi:10.1534/genetics.106.061978) <br />       H4 acetyltransferase important normal                1181 &#8211;1190. (doi:10.1038/sj.cdd.4402123)               58.   Nguyen L et al. 2010 Elongator&#8212;an emerging role <br />       histone acetylation levels vivo. Proc. Natl Acad.   41.   Berry DL, Baehrecke EH. 2007 Growth arrest               neurological disorders. Trends Mol. Med. 16, <br />       Sci. USA 99, 3517&#8211; 3522. (doi:10.1073/pnas.                  autophagy required salivary gland cell               1&#8211;6. (doi:10.1016/j.molmed.2009.11.002) <br />       022042899)                                                   degradation Drosophila. Cell 131, 1137&#8211;1148.        59.   Prisco M et al. 2002 Nuclear translocation insulin <br /> 26.   Svejstrup JQ. 2007 Elongator complex:               (doi:10.1016/j.cell.2007.10.048)                             receptor substrate-1 <span id='am-1' about='obo:NCBITaxon_10633' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_151341 obo:NCBITaxon_35237 obo:NCBITaxon_10239 obo:NCBITaxon_10624'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Polyomavirus maccacae" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="rhesus polyomavirus" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Rhesus macaque polyomavirus" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Polyomavirus maccacae 1" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Simian virus 40" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasBroadSynonym" content="SV40" datatype="xsd:string"></span>simian virus 40</span> T <br />       roles does play? Curr. Opin. Cell. Biol. 19,        42.   Baker KD, Thummel CS. 2007 Diabetic larvae               antigen activated type 1 insulin-like <br />       331&#8211;336. (doi:10.1016/j.ceb.2007.04.005)                     obese flies-emerging studies metabolism                growth factor receptor. J. Biol. Chem. 277, <br /> 27.   Versees W, De Groeve S, Van Lijsebettens M. 2010             Drosophila. Cell Metab. 6, 257 &#8211;266. (doi:10.1016/           32 078 &#8211;32 085. (doi:10.1074/jbc.M204658200) <br />       Elongator, conserved multitasking complex? Mol.            j.cmet.2007.09.002)                                    60.   Sun H et al. 2003 Insulin-like growth factor I <br />       Microbiol. 76, 1065&#8211;1069. (doi:10.1111/j.1365-         43.   Saltiel AR, Kahn CR. 2001 Insulin signalling         receptor signaling nuclear translocation of <br />       2958.2010.07162.x)                                           regulation glucose lipid metabolism. Nature           insulin receptor substrates 1 2. Mol. <br /> 28.   Walker J et al. 2011 Role Elongator Subunit Elp3          414, 799 &#8211;806. (doi:10.1038/414799a)                         Endocrinol. 17, 472 &#8211;486. (doi:10.1210/me. <br />       Drosophila melanogaster larval development      44.   Gronke S et al. 2007 Dual lipolytic control body          2002-0276) <br />       immunity. Genetics 187, 1067 &#8211;1075. (doi:10.1534/            fat storage mobilization Drosophila. PLoS       61.   Wu A, Chen J, Baserga R. 2008 Nuclear insulin <br />       genetics.110.123893)                                         Biol. 5, e137. (doi:10.1371/journal.pbio.0050137)            receptor substrate-1 activates promoters cell cycle <br /> 29.   Ponting CP. 2002 Novel domains orthologues      45.   Teixeira L et al. 2003 Drosophila Perilipin/ADRP             progression genes. Oncogene 27, 397&#8211; 403. (doi:10. <br />       eukaryotic transcription elongation factors. Nucleic         homologue Lsd2 regulates lipid metabolism. Mech.             1038/sj.onc.1210636) <br />  62. Altschul SF et al. 1997 Gapped BLAST PSI-           67. Ronquist F, Huelsenbeck JP. 2003 MrBayes 3:                  gels. Anal. Chem. 68, 850&#8211;858. (doi:10.1021/          16 <br />     BLAST: new generation protein database search          Bayesian phylogenetic inference mixed                  ac950914h) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                    rsob.royalsocietypublishing.org <br />     programs. Nucleic Acids Res. 25, 3389&#8211;3402.                 models. Bioinformatics 19, 1572&#8211; 1574. (doi:10.        72.   Yates III JR et al. 1995 Method correlate <br />     (doi:10.1093/nar/25.17.3389)                                1093/bioinformatics/btg180)                                  tandem mass spectra modified peptides to <br /> 63. So&#168;ding J, Biegert A, Lupas AN. 2005 The HHpred         68. Felsenstein J. 2005 PHYLIP (Phylogeny Inference              amino acid sequences protein database. <br />     interactive server protein homology detection           Package) version 3.6. Distributed author.             Anal. Ch <br /> </body></html>