<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Mar. 1987, p. 1180-1192                                                                     Vol. 7, No. 3 <br /> 0270-7306/87/031180-13$02.00/0 <br /> Copyright &#169; 1987, American Society Microbiology <br />  <br />  <br />   RAD4 Gene <span id='am-7' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>: Molecular Cloning and <br />      Partial Characterization Gene That Is Inactivated in <br />                           <span id='am-66' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-67' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span>Escherichia coli</span> <br />          REINHARD FLEER, CHARLES M. NICOLET,t GLENN A. PURE,t AND ERROL C. FRIEDBERG* <br />               Department Pathology, Stanford University Medical Center, Stanford, <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5'  ></span><span id='am-6' property="rdfs:label" content="California" datatype="xsd:string"></span>California</span> 94305 <br />                                         Received 11 August 1986/Accepted 24 November 1986 <br />  <br />             In contrast <span id='am-24' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-30'  ></span>Saccharomyces</span> cerevisiae RAD genes involved nucleotide excision repair DNA, the <br />          RAD4 gene isolated screening yeast genomic library recombinant <span id='am-113' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-114' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-116' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span> which <br />          complement UV sensitivity rad4 mutants (Pure et al., J. Mol. Biol. 183:31-42, 1985). We therefore <br />          attempted walk RAD4 neighboring SPT2 gene obtained integrating derivative plasmid <br />          isolated Roeder et al. (Mol. Cell. Biol. 5:1543-1553, 1985) contains 4-kilobase fragment yeast <br />          DNA including mutant allele SPT2. When integrated different rad4 mutant strains, <span id='am-56' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-57'  ></span><span id='am-58' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>this</span> <br />          plasmid (pR169) complements UV sensitivity frequency -10%. However, centromeric plasmid <br />          containing rescued sequences include flanking yeast DNA longer complements phenotype rad4 <br />          mutants. Complementing activity restored vivo repair defined gap centromeric plasmid. <br />          The repaired plasmid fully complements UV sensitivity rad4 mutants tested isolated directly <br />          yeast cells, plasmid propagated <span id='am-82' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-87'  ></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span>Escherichia</span> coli complementing activity lost. We <br />          mapped physical location RAD4 gene insertional mutagenesis transcript mapping. <br />          The gene -2.3 kilobases size located immediately upstream SPT2 gene. Both genes are <br />          transcribed direction. RAD4 essential gene, increased transcription gene is <br />          observed cells exposed DNA-damaging agent 4-nitroquinoline-1-oxide. The site inactivation RAD4 <br />          particular plasmid propagated E. coli localized 100-base-pair region gene disruption gap <br />          repair experiments. In addition, identified approximate locations chromosomal rad4-2, <br />          rad4-3, rad44 mutations. <br />  <br />    The excision base damage, causes conforma-                     Prakash S. Prakash, personal communication). A re- <br /> tional distortion secondary structure DNA (such               combinant plasmid (pNF4000) partially restores UV <br /> that produced UV irradiation bulky chemicals),               resistance particular mutant strain designated rad4-3 <br /> complex biochemical process generally referred nucle-                (29). However, rad4-3 allele contains ochre mutation, <br /> otide excision repair (8). In yeast Saccharomyces cere-                partial complementation observed result of <br /> visiae 10 genes involved repair process (9,           overexpression normal yeast tRNAGiA gene present in <br /> 11). Mutations genes render cells abnormally sensi-               multicopy plasmid pNF4000 (29). <br /> tive DNA damage UV radiation defective (to                          The RAD4 gene mapped chromosome V S. <br /> various degrees) ability incise UV-irradiated DNA              cerevisiae genetic analysis tightly linked the <br /> and excise pyrimidine dimers vivo (9, 11). At             SPT2 locus (formerly SPM2) (36). A mutant allele the <br /> genes (RADI, RAD2, RAD3, RAD4, RADIO) abso-                        gene (spt2-1) -3 kilobases (kb) of <br /> lutely required excision repair UV-irradiated DNA                upstream flanking sequences isolated Roeder et al. <br /> yeast, strains mutations genes totally <br /> defective pyrimidine dimer excision detect-                 recombinant plasmid called pR140 (32). In study we <br /> able incision DNA incubation UV irradiation                provide evidence integrating derivative plas- <br /> (9, 30, 35).                                                               mid designated pR169 (G. S. Roeder, personal communica- <br />    Previous studies laboratory resulted                   tion) contains entire RAD4 gene. Integration (by homol- <br /> isolation recombinant plasmids containing yeast                     ogous recombination) plasmid genome of <br /> RADI, RAD2, RAD3, RADIO genes (22, 24, 34, 38).                        various UV-sensitive rad4 mutant strains generated UV- <br /> These isolated screening yeast genomic library               resistant derivatives significant frequency. However, <br /> autonomously replicating multicopy plasmids comple-                  appropriate sequences integrant strain <br /> ment UV sensitivity selected rad mutants. Similar                   rescued subcloned centromeric plasmid, the <br /> results reported (12-14, 28, 39)                   complementing activity lost. The ability complement <br /> include recent isolation RAD7 gene (27). However,               UV sensitivity rad4 mutants tested restored <br /> extensive screening yeast libraries failed            repair vivo gap generated putative RAD4 <br /> yield plasmids containing functional RAD4 gene (29; L.                   gene centromeric plasmid. When gap-repaired <br />                                                                            plasmid isolated directly yeast propagated in <br />                                                                            various Escherichia coli strains, however, plasmids re- <br />   * <br />     Corresponding author.                                                  covered lost complementing activity. We therefore <br />   t Present address: 32/93 Chewings Street, Scullin A.C.T. 2614,           conclude plasmids containing functional RAD4 gene <br /> Australia. <br />    t Present address: Department Physiological Chemistry, Uni-          propagated E. coli suffering inactivation <br /> versity Wisconsin, Madison, WI 53706.                                   gene. <br />                                                                     1180 <br />  VOL. 7, 1987                                                                            R       CENEVIS. <br />                                                                                                        CEREVISIAE               1181 <br />  <br />                  TABLE 1. S. cerevisiae strains                      YCp5O BgllI site BglII PvuI sites, <br /> Strain         Relevant genotype                  Source          respectively, inactivated filling termini E. <br />  mutant                                                              coli DNA polymerase I (Klenow fragment) religating the <br /> YNN218      RAD his3A200 ade2-101          R. Davis, Stanford      blunt-ended plasmids. Tailoring BglII site results in <br />               ura3-52 lys2-801                 University            generation new ClaI site does interfere with <br /> SX46a       RAD his3-532 ade2 ura3-52      Our laboratory stocks   capacity high-frequency transformation endowed by <br />               trpl-289                                               ARS function (4). Tailoring PvuI site yields bacterial <br /> S288C       RAD gal2                       Our laboratory stocks   colonies Amps Tetr. A derivative YCpS0 was <br /> radl-J      radl-J his3-532 ade2 ura3-     E. Yang, labora-   used gap repair experiments. This plasmid (YCpS1) <br />               52 trpl-289                      tory                  contains -0.8-kb deletion Tetr gene YCp5O, <br /> rad2-1      rad2-1 his3-532 ade2 ura3-     L. Naumovski,      removes BamHI, SphI, SalI sites leaves <br />               52 trpl-289                      laboratory            EcoRl HindIll sites intact. Transformation yeast <br /> rad3-1      rad3-1 his3-532 ade2 ura3-     L. Naumovski, this <br />               52 trpl-289                      laboratory            strains plasmid DNA carried treating cells <br /> rad4-2      rad4-2 his3-532 ade2 ura3-52   G. Pure, labora-   lithium acetate promote DNA uptake (17). Transfor- <br />               trpl-289                         tory                  mation E. coli carried previously described <br /> rad4-3      rad4-3 his3-532 ade2 ura3-     G. Pure, labora-   (22). <br />               52 trpl-289                      tory                     Plasmids generally extracted E. coli a <br /> rad44       rad44 his3-532 ade2 ura3-52    G. Pure, labora-   modification (19) alkaline lysis procedure Bimboim <br />                                                tory                  Doly (2). For rapid screening plasmid DNA, lysates <br /> rad4-10     rad4-10 his3A 200 ade2-101     R. Fleer, study    prepared procedure Holmes Quigley (15) <br />               lys2-801                                               subjected electrophoresis 0.7% agarose gels <br /> rad4-11     rad4-11 his3-532 ade2 trpl-    R. Fleer, study <br />               289                                                    prior purification. A scaled-up version alka- <br />                                                                      line lysis procedure (19) used isolate chlorampheni- <br />                                                                      col-amplified plasmid DNA followed purification <br />                                                                      cesium chloride-ethidium bromide gradients. Yeast <br />                                                                      genomic DNA prepared spheroplasts described <br />                                                                      previously (34), ethanol-precipitated DNA was <br />    We mapped physical location RAD4 gene             additionally treated 50 ,ug DNase-free RNase ml <br /> and immediately flanks STP2 gene           37&#176;C 30 min. Cesium chloride-ethidium bromide gra- <br /> both genes transcribed direction. The site        dient centrifugation plasmid DNA, restriction enzyme <br /> inactivation RAD4 particular plasmid propagated E.        digestion, gel electrophoresis, size fractionation DNA <br /> coli localized 100-base-pair (bp) region        fragments sucrose gradients, general recombi- <br /> the coding sequence gene disruption vivo repair         nant DNA methods followed standard protocols (5, 19). 32p <br /> of plasmids containing defined gaps. Finally,           labeling DNA carried using mixed random <br /> unlike RAD2 (18, 31) enhanced expression RAD4 gene            primers prime DNA synthesis method Feinberg <br /> is induced exposure cells DNA-damaging           Vogelstein (7). <br /> agent 4-nitroquinoline-1-oxide unlike RAD3 (13, 23)            Isolation purified plasmid DNA yeast cells. Cells <br /> it essential gene.                                         grown early stationary phase minimal me- <br />                                                                      dium. Cells washed, suspended 1 M sorbitol, and <br />                 MATERIALS AND METHODS                                converted spheroplasts 300 ,ug zymolyase ml in <br />                                                                      presence 25 mM EDTA, 50 mM phosphate, 1 p.l of <br />   Yeast bacterial strains culture conditions. Strains        P-mercaptoethanol ml. After washing 1 M sorbitol, <br /> for transformation plasmids containing URA3 gene            spheroplasts (from 250-ml culture) suspended 2.5 <br /> as selectable marker originally constructed            ml 1 M sorbitol, equal volume solution 50 <br /> laboratory crossing rad mutants strain SX46a,                  mM glucose-25 mM Tris hydrochloride (pH 8.0)-10 mM <br /> sporulating diploids, isolating appropriate haploid          EDTA added. Subsequent steps described for <br /> strains. The genotypes yeast strains           large-scale alkaline lysis E. coli preparations (19), except <br /> used listed Table 1.                                          resulting potassium acetate supematants pre- <br />    Yeasts bacteria grown previously described            cipitated 23&#176;C 15 min adding 14 ml isopropanol. <br /> (38). Yeast strains transformed plasmids grown          The dissolved precipitates treated 50 ,ug RNase <br /> synthetic medium lacking uracil maintain selection            ml 37&#176;C 20 min 150 ,ug proteinase K (in <br /> plasmid-bearing cells. YPD medium (1% yeast extract, 2%              0.5 M NaCl, 0.5% Sarkosyl, 25 mM EDTA) ml 60&#176;C <br /> Bacto peptone [Difco], 2% glucose) used nonselec-            1 h. After centrifugation 5 min Eppendorf <br /> tive growth measurement cell viability UV           table-top centrifuge, supernatants ethanol precipitated <br /> irradiation. For selection E. coli transformants cells       -70&#176;C 10 min, pellets dissolved and <br /> grown L broth supplemented ampicillin (50 ,ug/ml)         purified cesium chloride-ethidium bromide gradients. Cen- <br /> tetracycline (15 ,ug/ml) appropriate. For isolation            trifugation Ti5O fixed-angle rotor 42,000 rpm for <br /> plasmid DNA E. coli cells grown L broth (19)              48 h, followed rebanding plasmid-containing fractions <br /> which chlorampenicol (200 p.g/ml) added absor-             VTi65 vertical rotor 55,000 rpm 12 16 h. Yeast <br /> bance 600 nm 0.55. Incubation continued 20 h.          2 ,um plasmid DNA accounted 90% the <br />    Cloning <span id='am-89' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-90' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-91' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> transformation procedures. The shut-          resulting plasmid preparations, served localize the <br /> tle vectors YIpS (for integrating plasmids) YCpSO (for           plasmid band UV light. The yield single-copy <br /> centromeric plasmids) carrying yeast URA3 gene              centromeric plasmid -600-800 ng DNA liter of <br /> kindly provided R. W. Davis, Stanford University. The             stationary-phase culture. Assuming single copy the <br /> integrating plasmid pR169 gift G. S. Roeder, Yale         14-kb plasmid cell (-1.5 x 10-11 ,ug DNA), liter of <br /> University. Plasmids YCp52 YCp53 derivatives              yeast culture grown minimal medium (density, -8 x 107 <br />  1182    FLEER ET AL.MOL.MOL. CELL. BIoL. <br />                                    BIOL. <br />  <br /> cells ml) contains -1.2 ,ug plasmid DNA. Based         ing blunt-ended XbaI site generate plasmid pNF420A <br /> these calculations recovery centromeric plasmid DNA       (Fig. 1). Linear fragments (HindIII-XmnI, 4.5 kb; HindIII- <br /> from yeast cells -50 70%.                                 PvuI, 4.6 kb; HindIII-HpaI, 4.85 kb) used transforn <br />    Measurement UV survival yeast strains. Quantitative     strain SX46a. <br /> survival transformed yeast strains UV irradiation          Transcript mapping. RNA isolated Rad+ <br /> was measured described previously (22). For rapid             yeast strain S288C previously described (25) used <br /> screening UV-irradiated transformants small amounts        map RAD2 transcriptional start site modification <br /> single yeast coloties individually suspended 0.5 ml      (6) method Berk Sharp (1). A 1.65-kb BglII- <br /> of sterile water, 5-,ul samples applied YPD agar     XhoI fragment including 5&apos; end RAD4 gene was <br /> plates regular pattern. The plates (containing similarly    end labeled BglII site hybridized 4 h against <br /> treated samples Rad+ strain SX46a Rad- rad4         100 ,ug total RNA 10 pu 90% recrystallized formam- <br /> mutant positive negative controls, respectively),    ide. The optimal temperature hybridization (43&#176;C) and <br /> irradiated 40 J/m2 incubated 30&#176;C 36 h. The        optimal S1 nuclease concentration (1,500 U/ml) were <br /> growth UV-sensitive (Rad-) mutants completely abol-        determined empirically. <br /> ished conditions, repair-proficient <br /> (Rad+) cells visible confluent colonies. A semiquan-                                RESULTS <br /> titative survival test performed similar manner, <br /> except cells different strains applied agar       Complementation UV sensitivity rad4 mutants with <br /> plates horizontal rows 5-,u samples given suspen-     plasmid pR169. Plasmid pR169 integrating plasmid <br /> sion, vertically aligned spots irradiated 0, 10,   containing 4.0-kb yeast chromosomal insert includes <br /> 20 40 J UV light m2 respectively.                      spt2-1 allele near end (Fig. 1). The region between <br />    Gap repair yeast. Repair yeast centromeric plas-     leftmost HindIll site BglII site identified Fig. <br /> mids defined gaps carried described           1 sequenced Roeder et al. (32), identified three <br /> Orr-Weaver et al. (26) modified Naumovski et al.          complete incomplete open reading frames (ORFs) <br /> (21). A gap PvuI BamHI sites present      (Fig. 1). One complete ORFs contains coding <br /> insert pNF402 (Fig. 1) generated complete diges-       region spt2-1 gene. The small overlap- <br /> tion BamHI partial digestion PvuI, followed        ping unassigned ORfs, 257 155 amino acids length. <br /> by isolation appropriate DNA fragment agarose gel      The incomplete ORF present end sequenced <br /> electrophoresis. Stable Ura+ transformants screened         fragment unassigned (32). DNA-RNA hybridiza- <br /> for UV resistance, presence centromeric plasmids      tion analysis revealed transcripts encoded. se- <br /> was confirmed demonstrating cosegregation UV resist-       quences pR169. One (-1.4 kb length) the <br /> ance Ura+ phenotype growth nonselective        spt2-1 transcript (32). The transcript -2.6 kb size, <br /> medium. Gaps plasmids generated unique      position pR169 mapped (M. Smith, <br /> restriction sites, cells transformed complete      personal communication). A detailed restriction map the <br /> digests prior purification DNA fragments.             region including SPT2 locus -6 kb 5&apos;-flanking <br />    Gene transplacement. We constructed series insertion     DNA shown Fig. 1. <br /> mutants using one-step gene transplacement method            In view tight genetic linkage SPT2 and <br /> of Rothstein (33). Mutations generated inserting     RAD4 (36), considered possibility the <br /> yeast URA3 gene (which served selectable marker)       ORFs pR169 represent RAD4 <br /> into selected restriction sites various plasmids. Linear      coding sequence. To investigate integrated plasmid <br /> fragments containing URA3 disruptions                  pR169 rad4 mutant strains (rad4-2, rad4-3, <br /> transplaced genome selected Rad+ strains.            rad44); -10% stable rad4-2 rad4-3 transform- <br />    (i) Disruption BamHI. Plasmid pNF402GR-EC1, de-          ants generated untargeted integration smaller <br /> rivative pNF402 (Fig. 1), cut unique BamHI site      number transformants generated targeted integra- <br /> in insert ligated 2.5-kb BamHI fragment con-        tion showed wild-type levels UV resistance (Table 2). <br /> taining URA3 gene generate plasmid pNF404 (Fig. 1).       However, rad44 integrants showed enhanced <br /> Linear fragments (EcoRI, 9.1 kb; XmnI, 5.1 kb; HpaI-XmnI,        UV resistance (Table 2). We initially thought incom- <br /> 4.75 kb) isolated 0.7% agarose gels used        plete ORF pR169 likely candidate gene <br /> transform strain SX46a (Rad+ ura3-52).                           encoding -2.6-kb transcript described hence <br />    (ii) Disruption BglII. A 4.6-kb HindlIl fragment      expected 5&apos; end located -1 kb upstream <br /> pNF402GR-EC1 cloned plasmid YIpS generate            BamHI site defines right end insert in <br /> plasmid pNF411 (Fig. 1). The 2.5-kb BamHI fragment <span id='am-93' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-94' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-97'  ></span>car</span>-          plasmid (Fig. 1). Thus, interpretation these <br /> rying URA3 gene (see above) cloned unique       integration experiments incomplete ORF may <br /> BglIj site plasmid generate plasmid pNF412 (Fig.      represent truncated RAD4 allele. Depending sites of <br /> 1), linear 4.3-kb HpaI-XhoI fragment containing URA3       homologous recombination plasmid chromo- <br /> inserted BglII site used transform strain SX46a.   somal sequences relative location rad4-2 and <br />    (iii) Disruption MstII. Plasmid pNF416 (Fig. 1) contains   rad4-3 mutations, integration reconstitute a <br /> a 1.lTkb SmaI fragment carrying URA3 gene cloned        complete wild-type copy RAD4 (Fig. 2A). The failure to <br /> the blunt-ended uniique MqtII site pNF411. A 3.9-kb           observe UV-resistant rad44 integrants expected the <br /> SphI-XmnI linear fragment pNF416 (including -530 bp         rad44 mutation located region chromosomal <br /> of pBR322 DNA end) used transform strains          gene represented pR169, complete wild-type <br /> SX46a YNN218 (Rad+ ura3-52).                                 gene reconstituted conditions (Fig. <br />    (iv) lpisruption XbaI SPT2 gene. Plasmid pNF411     2A). <br /> was deleted 1.7-kb BglII-XhoI fragment, remov-         Plasmid rescue integrant strain yields noncom- <br /> ing XbaI sites. A 2.2-kb HindIII-BamHI frag-        plementing derivatives. The results described sug- <br /> ment containing URA3 gene cloned remain-        gested strategy rescuing plasmid containing entire <br />  VOL. 7, 1987                                                                                                                 RAD4 GENE OF S. CEREVISIAE   1183 <br />  <br />                                          0                        2                       4                              6        kb <br />                                         H                                                          H               H H <br />                                         H                                                          H               H H <br />                                         H                     H           H       H                H           H   H H       H <br />                                         t        H            H       HH H    H   $                            H <br />                                          ~       41           41       V~.-I~   A                  A                          0 <br />  <br />  <br />                                                             .1I           f [I                                 111,1 <br />                                              H <br />                                                  I                    ~~I <br />                                                                       H <br />                                                                          I <br />                                                                               H <br />                                                                                           I-IH IH <br />                                                                                           H <br />  <br />  <br />  <br />  <br />                                          H                                G                                                       Vector DNA: <br />                                          H X                  M       VG                   B <br />                        pRl 69                                                                                                           YIp5 <br />                                               4--      4-                 4~~---------- <br />                                              SPT2      ORFs                   ORF <br />                                        ER                             V                    B                                 E <br />                                         11                                                    l                                         YCp50 <br />                        pNF402 <br />                                                  (P)                  VG                  (B) <br />                        pNF417                                            II                                                              YCp53 <br />                                                                                                                                        (ABgIl,APvul) <br />                                                  (P)                  VG X                X            (H) <br />                        pNF418                     L                   I II                I             I                                YCp53 <br />                                                                           B       U1RA3           2 .5 kbB                             (ABgIl,APvul) <br />                                        El <br />                                         II <br />                        pNF404                                                                                                           YCp51 <br />                                                                                                                                        (ABamHI) <br />                                          H X                  x           G                B       OR <br />                                                                                                        I <br />                        pNF411                                                                                                           YIp5 <br />                                                         B         UIR&quot;        2 . 5 kb B <br />  <br />                                          a <br />                                                         I{- <br />                        pNF412                                                                                                            YIp5 <br />                                                      UPA3 1. 1 kb <br />                                                     Sm           Sm <br />                                   Sp                                                       B               H <br />  <br />  <br />                        pNF416 46                                                                                                         YIp5 <br />                                (H) URA3 2. 2 kb (B)                      Np <br />  <br />                                                                    NJV G)X X B(O)H <br />                         pNF420A \\x\\                                                                                                    YIp5 <br />                                                                               I-A1.7 kb-I <br />    FIG. 1. Physical maps recombinant inserts various plasmids used study. The line shows detailed restriction map of <br /> a 6.6-kb region includes SPT2 gene upstream regions. The sequenced region pR169 extends HindIII site BgllI <br /> site identified bar (H-G). The ORFs identified region shown open boxes relative restriction maps. The <br /> horizontal arrows represent direction translation ORFs inferred deduced amino acid sequences. The dashed horizontal <br /> arrow indicates extension incomplete ORF, based assumption ORF encodes unassigned -2.6-kb transcript <br /> detected using fragments pR169 hybridization probes (M. Smith, personal communication). The regions various plasmid inserts <br /> deleted gap repair linear transplacement genome identified open hatched boxes respectively. Plasmid vectors are <br /> indicated right case. The following abbreviations used: B, BamHI; E, EcoRI; G, BglII; H, HindIlI; Hp, HpaI; K, KpnI; <br /> M, MstII; N, XmnI; 0, XhoI; P, PstI; S, Sall; Sm, SmaI; Sp, SphI; V, PvuI; X, XbaI. Restriction sites parentheses destroyed by <br /> blunt-end ligation. <br />  <br /> RAD4 gene. After integration putative truncated                                     -15-kb fragment containing entire pR169 yeast insert <br /> RAD4 gene genome rad4-2 rad4-3 mutant,                                    -6 kb yeast chromosomal DNA right <br /> UV-resistant integrants contain complete copy                                  region essentially YIp vector sequences <br /> the wild-type gene truncated copy containing                                     (Fig. 2A). Since vector sequences include pBR322- <br /> rad4-2 rad4-3 mutation (Fig. 2A). Restriction mapping                                derived origin replication Ampr gene, <br /> genomic DNA UV-resistant rad4-2 integrant                                       intramolecular ligation SalI fragments vitro followed by <br /> carried Southern hybridization analysis pR169                               transformation E. coli yield plasmids containing <br /> a probe indicated digestion Sall yield                             complete RAD4 gene (Fig. 2A). <br />  1184       FLEER ET AL.                                                                                                                                                MOL. CELL. BIOL. <br />  <br />       TABLE 2. Integration plasmids pR169 pNF400                                            TABLE 3. Transformation plasmids pNF401, pNF402GR and <br />                       various rad4 mutantsa                                                                     pNF402GR-EC1 various rad4 mutantsa <br />                      UV-resistant integrants plasmid (%):                                                           UV-resistant transformants plasmid (%): <br />                                                                                                        MIutant <br />                             pR169                                       pNF400                                             pNF401                  pNF402GR                    pNF402GR-EC1 <br /> Mutant <br />    Mutant <br />               Untargeted                Targeted           Untargeted <br />                                                                                 ~~~~~~~~~~~~Targeted <br />                                                                                        to <br />                                                                                                        rad4-2             0 (0/50) <br />                                                                                                                           0 (0/50) <br />                                                                                                                                                    100 (18/18) <br />                                                                                                                                                     95 (21/22) <br />                                                                                                                                                                                     0 (0/75) <br />                                                                                                                                                                                     0 (0/75) <br />                          Bglll                                                                      rad4-3 <br />                     BamH(                                                                              rad4-4             0 (0/50)                 100 (3/3)                        0 (0/75) <br /> rad4-2         12 (7/60) 2 (1/50) 0 (0/169) 0 (0/21)                                                     The numbers parentheses represent UV-resistant transformants a <br /> rad4-3         10 (6/63) 2 (1/50) 0 (0/116) 0 (0/21)                                                   function total number Ura+ clones screened. See text details. <br /> rad44           0 (0/92)                0 (0/70)            3 (3/114)            5 (6/114) <br />   a <br />     The numbers parentheses represent UV-resistant integrants a <br /> function total number Ura+ clones screened. See text details.                        plasmid (pNF401) contains 6.6-kb yeast insert which <br />                                                                                                        includes 2.6 kb rescued sequence contiguous insert <br />                                                                                                        pR169. Plasmid pNF401 did complement UV <br />    Using strategy isolated single Ampr E. coli                                               sensitivity of rad4 mutant strains (Table 3). <br /> transformant containing plasmid expected size                                               Integration plasmid pR169 rad4-2 rescue the <br /> restriction pattern, designated pNF400 (Fig. 2A). To estab-                                            complete RAD4 gene pNF400 predicated the <br /> lish presence entire RAD4 gene pNF400                                                 hypothesis plasmid missing 5&apos; end of <br /> integrated plasmid rad4 mutant strains.                                            RAD4 (Fig. 2A). However, considered alternative <br /> Cohtrary results obtained pR169, tJV-                                                   possibility pR169 contains complete mutant copy <br /> resistant colonies identified untargeted                                          RAD4, case integration rad4-2 strain <br /> targeted integration pNF400 rad4-2 rad4-3,                                             generate complete genes (Fig. 2B). If integrative <br /> only small fraction rad44 integrants UV                                                  recombination occurred chromosomal plas- <br /> resistant (Table 2). We constructed autonomously                                               mid-borne mutations (crossover 2 Fig. 2B), two <br /> replicating single-copy derivative plasmid pNF400. This                                             integrant genes wild type would <br />  <br />                     SPT2    ORF          RAD4    ?                                                                        SPT2        RAD     ? <br />   A_                                                                                                    B <br />                   HX            M     VG&#174;B                 E     S                                                       HX           M <br />                                                                                                                                           *VG~ <br />                                                                                                                                             *q B             E     S <br />  <br />  <br />  <br />  <br />                                 It  vector      HX        M VG           B          E     S             H X        M VG          BS <br />                                                                                                                                       It  vector   HX        OVG           B    &#174;      E   S <br />   H X        M VG          BS <br />  <br />               truncated gene                                   complete gene <br />                                                                                                                  c <br />                                                                                                                  complete g. <br />                                                                                                                       l. gene                              l <br />                                                                                                                                                          complete ge <br />                                                                                                                                                                   gene <br />  <br />                                                   15 kb Sall fragment                                                                                15 kb Sall fragment <br />  <br />  <br />                                                                It <br />                                                 H X        M VG          B          E      S                                                       HX         MVGM         B           E       S <br />  <br />  <br />  <br />  <br />                                                                         pF400                                                                                          pNF400 <br />  <br />  <br />  <br />    FIG. 2. Cloning chromosomal DNA sequences contiguous yeast insert plasmid pR169. Panels A B diagrammatically <br /> illustrate alternative models location RAD4 gene relative SPn2 locus chromosome insert pR169. The <br /> model panel A assutnes plasmid pR169 contains truncated RAD4 allele, model shown panel B based the <br /> hypothesis entire RAD4 gene present plasmid, mutated. Homologous chromosomal plasmid sequences are <br /> identified boxed regions RAD4 alleles hatched. The circled M represents hypothetical locations sites mutation(s) <br /> in chromosomal plasmid RAD4 alleles (or both). Integration plasmid occur homologous recombination of <br /> the mutations (numbered crossovers), yielding UV-resistant UV-sensitive integrants. However, products homologous <br /> recombination shown regenerate wild-type RAD4 allele result UV-resistant integrants (crossover no. 2 each <br /> case). Digestion chromosomal integrant DNA Sall subsequent intramolecular ligation generates plasmid pNF400. Note that <br /> inversion relative positions chromosomal plasmid-borne mutations B generate UV-resistant integrant containing <br /> the wild-type RAD4 allele 15-kb Sall fragment, mutant allele shown figure. In case, plasmid pNF400 would <br /> rescue wild-type RAD4 gene, inactivated subsequent propagation E. coli. The abbreviations restriction <br /> sites shown Fig. 1. <br />  VOL. 7, 1987                                                                              RAD4 GENE OF S. CEREVISIAE                  1185 <br />  <br />                                                                        mutation(s) pNF400 centromeric deriv- <br />                                                                        ative pNF402. <br />                                                                           After transformation gapped centromeric plas- <br />                                                                        mid, -10% Ura+ transformants showed wild-type <br />                          PvuI         amIC <br />                                                     UVs                levels UV resistance. By growing independent <br />                                                                        transformants nonselective conditions, observed <br />                                                                        strict cosegregation UV resistance Ura+ pheno- <br />                                                                        types case, indicating UV resistance deter- <br />                          (pNF42                                        minant plasmid borne. These results demonstrate success- <br />                                                                        ful repair gapped centromeric plasmid pNF402 and <br />                                                                        support contention plasmid contains muta- <br />                                 7      GAP-                            tion(s) present equivalent region rad44 <br />                                        REPAIR                          allele. These results rad44 mutation is <br />                                                                        located outside PvuI-BamHI interval yeast <br />                                                                        genome (Fig. 3). The observation -10% the <br />                                                                        stable rad44 transformants UV resistant indicates that <br />                                                                        efficiency gap repair rad44 strain limited. <br />                                                                        This result rescue chromosomal rad44 <br />                                                        UVR             mutation extension PvuI-BamHI gap by <br />                                                                        exonucleolytic degradation vivo (Fig. 3). Alternatively, or <br />  <br />                            pNr                                                    I <br />  <br />    FIG. 3. Reconstitution wild-type RAD4 gene vivo gap                                                N- <br /> repair mutant rad4 plasmid allele, using rad4 mutant                                                    &quot;IN <br /> recipient strain. Chromosomal plasmid RAD4 alleles shown                                                            N&apos; <br /> as hatched open boxes, respectively, circled Ms <br /> representing hypothetical locations mutations alleles. <br /> The recipient UV-sensitive (UVs) strain transformed a <br /> linearized plasmid (pNF402) containing deletion removes <br /> the plasmid mutation(s). The double-strand gap repaired gene <br /> conversion homologous chromosomal sequences, Ura+ <br /> transformants screened UV resistance. UV-resistant (UVR) <br /> transformants arise repaired gap does include                             \ <br /> chromosomal mutation. Hence, vivo extension gap          z     IV <br /> limit efficiency productive gap repair. The efficiency gap   0 <br /> repair limited plasmid-bome mutation situated <br /> outside close gap. In event, productive repair will <br /> occur instances vivo extension gap <br /> removes plasmid-borne mutation.                                    IL <br />                                                                                                     U <br />  <br />  <br />  <br />                                                                        o 10 <br />                                                                        z <br /> contain mutations (Fig. 2B). Rescue mutant allele <br /> could account results observed plasmids                   5; <br /> pNF400 (Table 2) pNF401 (Table 3).                                                                      U\ <br />   Gap repair rescued sequences. To test <span id='am-36' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-38'  ></span><span id='am-39' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>idea</span>           ct) <br /> the putative RAD4 genes plasmids pR169, pNF400, and <br /> pNF401 represent mutated alleles, replaced portion                   io-4 <br /> the cloned gene transfer homologous sequences from <br /> the yeast genome using modification (21) gap <br /> repair technique Orr-Weaver et al. (26) (Fig. 3). A <br /> centromeric derivative plasmid pNF401 containing a <br /> unique BamHI site yeast insert constructed to <br /> yield plasmid pNF402 (Fig. 1), 1.6-kb region be-                     10-5 <br /> tween BamHI PvuI sites plasmid deleted.                                         20        40                       60 <br /> We carried gap repair pNF402 rad4 mutant                                      UV FLUENCE (J /m2) <br /> UV-sensitive cells screened transformation to <br /> UV resistance (Fig. 3). The detection repaired plasmids               FIG. 4. Survival transformed yeast strains various UV <br /> containing functional RAD4 gene obviously requires              doses. Cells transformed relevant plasmids grown in <br /> chromosomal sequences homologous deleted plas-                minimal broth, harvested centrifugation, spread mini- <br />                                                                        mal agar plates appropriate dilutions 0.07 M phosphate buffer. <br /> mid sequences free mutations. To end used           Plates exposed UV irradiation indicated incubated <br /> rad44 strain, demonstrated ability generate               5 days 30&#176;C colonies quantitated. Duplicate <br /> some UV-resistant transformants integration               symbols represent independent experiments. Symbols: 0, RAD4 <br /> plasmid pNF400 strain (Table 2) suggested           (YCp5O); O, rad44 (YCp5O); 0, rad44 (pNF400GR); *, rad44 <br /> rad44 mutation located site distinct putative         (pNF400GR-EC). <br />  1186     FLEER ET AL. <br />  <br />  <br />  <br />  <br />          a <br />          b <br />          c <br />                   I <br />  <br />                   H <br />                   H <br />                   H <br />                   V <br />                   .I4 <br />                   _ <br />                       4 <br />                           SPT2 <br />  <br />                            H <br />                            . <br />  <br />  <br />  <br />                           L&#177;z <br />                            X <br />                                    -   -4 <br />                                             pR1 69 <br />                                             s <br />                                          ORFs <br />                                               H <br />                                               H <br />                                               4J <br />  <br />  <br />  <br />                                        -JKVK,Ctkqqm <br />                                                    I. 4 <br />                                                      a <br />                                                      c <br />                                                           .1 <br />  <br />  <br />  <br />  <br />                                                            \:l\K\\\\\I I II <br />                                                                I <br />  <br />  <br />                                                                U <br />                                                                II <br />                                                                II <br />                                                                    ~ <br />                                                                    H H H <br />                                                                    &gt; <br />                                                                    of <br />                                                                         pNF402 <br />                                                                         - <br />  <br />  <br />  <br />  <br />                                                                         :U: <br />                                                                               4 <br />                                                                                   O0 <br />                                                                                   ORF <br />                                                                                   H <br />                                                                                   - <br />  <br />  <br />                                                                                   to <br />                                                                                         H <br />                                                                                         .4 <br />                                                                                         ff <br />                                                                                         co <br />                                                                                         CJ <br />                                                                                              1 <br />  <br />                                                                                              H <br />                                                                                              m <br />                                                                                              9 <br />  <br />                                                                                              m <br />  <br />  <br />  <br />  <br />                                                                                          BamHJ1 <br />  <br />  <br />                                                                                              I <br />                                                                                                   H <br />                                                                                                   0 <br />  <br />  <br />                                                                                                   x m <br />                                                                                                       H <br />                                                                                                       H <br />                                                                                                       H <br />                                                                                                       XI <br />  <br />  <br />                                                                                                       I, II <br />                                                                                                            H <br />  <br />                                                                                                            X <br />  <br />  <br />                                                                                                            i <br />                                                                                                            I <br />                                                                                                                        -A <br />                                                                                                                        I <br />  <br />  <br />  <br />  <br />                                                                                                                        H <br />                                                                                                                        0 <br />                                                                                                                        U <br />                                                                                                                         C) <br />                                                                                                                         I <br />                                                                                                                                        UV-sensitive <br />                                                                                                                                         disruption <br />                                                                                                                                          mutants: <br />  <br />  <br />  <br />                                                                                                                                        70% (34/50) <br />                                                                                                                                         3% (3/92) <br />                                                                                                                                         0 (0/200) <br />                                                                                                                                                     MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                                           BglII <br />          d                                              I                                                                              95% (48/50) <br />                                                      III!!!! <br />                                             mst I I: <br />                                                                : <br />           e- <br />           Af      I                            I <br />                                                                                                                                        95% (89/93) <br />          f <br />                       ,   XbaI-,,.01 <br />                                I <br />                                                                                                                                          n      (n/inni <br />   FIG. 5. Mapping physical location RAD4 gene gene disruption. The various linear fragments (each containing URA3 gene <br /> [shaded triangles] inserted restriction sites shown) used omega transformation shown lines f derived from <br /> the plasmid inserts shown Fig. 1. The upper figure shows extent yeast inserts plasmids pR169 pNF402 the <br /> arrows indicate ORFs described Fig. 1. A partial restriction map region genome provided. The open box connotes the <br /> location SP12 gene. The hatched box shows location RAD4 gene based results gene disruptions transcript <br /> mapping (see Fig. 8). The percentage UV-sensitive disruption mutants obtained type transplacing fragment shown the <br /> right case. Numbers parentheses indicate number UV-sensitive transformants function total number clones <br /> screened. <br />  <br />  <br /> additionally, mutation(s) present pNF402                                                 apparent identity pNF402GR, plasmid <br /> located close outside PvuI-BamHI interval;                                            isolates complemented UV sensitivity rad4-2, rad4-3, <br /> successful repair require gap extension vivo.                                                   rad44 mutants (Table 3). <br />    Plasmids complement rad4 mutants prop-                                                Based results conclude autonomously <br /> agated E. coli. The gap-repaired centromeric plasmid                                                replicating plasmids containing functional yeast RAD4 <br /> which complements rad44 designated pNF402GR. We                                                     gene inactivated E. coli. Inactivation result from <br /> isolated plasmid directly yeast transformed                                              mutation(s) specific sequence recognized E. coli, in <br /> rad4-2, rad4-3, rad44 mutants                                                case independently cloned plasmids ex- <br /> propagation plasmid E. coli. When plasmid                                            pected contain mutation(s). Alternatively, for <br /> not propagated E. coli essentially Ura+ trans-                                           given plasmid, site mutation(s) generated E. <br /> formants UV resistant (Table 3 Fig. 4). Once again,                                           coli differ independently cloned plasmids. To avoid <br /> in cases tested phenotypes UV resistance Ura+                                           possible complications arising situation we <br /> cosegregated. However, centromeric plasmids                                                used stable derivatives single plasmid (pNF402GR-EC1) <br /> isolated E. coli HB101 able support phenotypic                                             exclusive source transforming DNA further <br /> complementation rad4 mutants tested, regardless                                              experiments. <br /> the genetic background E. coli strain. Details                                             Localization RAD4 gene. The inability propagate <br /> analysis single-copy multicopy autonomously repli-                                              plasmids functional RAD4 gene E. coli precluded <br /> cating plasmids isolated various E. coli strains gap                                        standard subcloning strategies designed localize gene <br /> repair yeast presented (R. Fleer, W.                                              6.6-kb yeast insert plasmid pNF402. We therefore <br /> Siede, E. C. Friedberg, manuscript preparation).                                                constructed series linear yeast DNA fragments from <br /> Eight independent transformants E. coli HB101 contained                                             plasmid pNF402GR-EC1, contains URA3 <br /> plasmids detectable deletions rearrangements (as                                            gene inserted selected restriction site. Integration of <br /> determined restriction analysis) compared                                                 linear fragments yeast genome occurs by <br /> plasmid pNF402GR isolated directly yeast. The                                              double-crossover events regions sequence homology, <br /> plasmids stably propagated E. coli HB101                                             resulting exchange plasmid-derived linear fragments <br /> designated collectively pNF402GR-EC. Despite                                                  containing URA3 insertions, homologous regions of <br />  VOL. 7, 1987                                                                                                                            RAD4 GENE OF S. CEREVISIAE              1187 <br />  <br />                                 EcoRI <br />                                                           XmnI                     BglIl                                 In initial experiments, Rad+ ura3-52 strain trans- <br />                                II               I                                                                   formed linear 5.1-kb XmnI fragment plasmid <br />                                                                                 A          B                          pNF404 (Fig. 1) carrying URA3 disruption BamHI <br />                                     UVrUV                     UVrUVs            WT     UVrUVs <br />                                                                                                                       site shown Fig. 5. When Ura+ transformants exam- <br />                                WT                   WT <br />                                                                                                                       ined response UV irradiation, great majority <br />                                                                                                                       UV resistant (Fig. 5, line b). We confirmed that <br />                        10-                                                                                            transplacement occurred near SPT2 locus by <br />                                                                                                                       Southern hybridization analysis UV-resistant <br />                                                                                                                       UV-sensitive Ura+ transformants (Fig. 6). These <br />                                                                                                                       transplacements disruption the <br />                        5 <br />                                                                                                                       BamHI site does affect UV resistance Rad+ <br />                        4-                                                                                             strain, conclude site RAD4 <br />                                                                                                                       gene. The small fraction (3%) UV-sensitive transformants <br />                        3-                                                                                             population Ura+ transformants containing a <br />                                                                                                                       disruption BamHI site (Fig. 5, line b) presumably <br />                                                                                                                       arose recombinational events include site(s) of <br />                        2-                                                                                             inactivation mutant rad4 allele plasmid pNF402GR- <br />                                                                                                                       ECL. The low frequency events suggests this <br />                                                                                                                       site(s) located near end transplacing DNA and <br />                                                                                                                       included small fraction crossovers. <br />                         I1- <br />                                                                                                                       In support suggestion, transplacement larger <br />                                                                                                                       (9.1-kb) EcoRI fragment plasmid pNF404 yielded a <br />                                             PROBE <br />                                      I                                                                              larger fraction UV-sensitive Ura+ transformants (Fig. 5, <br />                                                                                                                       line a), transplacement slightly smaller <br />                E           G        N            NG                  N       N E               G <br />                                                                                                                       (4.75-kb) HpaI-XmnI fragment resulted UV-sensitive <br />              Iws       S <br />                                     IH                           B <br />                                                                                                    7 <br />                                                                                                    S <br />                                                                                                         - <br />                                                                                                                       transformants (Fig. 5, line c). Furthermore, observation <br />                                                                                                                       transplacement linear fragment terminates at <br />                I.1--                      12.2                           ----      I                          EcoRI   HpaI site (Fig. 5, line c) results detectable <br />                                      1    2.4       1     -2.7       X                                        XmnI    UV-sensitive transformants, terminates at <br />                                     5.2                                    6.2       _       }              Bgll    XmnI site just 350 bp left does (Fig. 5, line b), <br />            L <br />                                                                                                                       indicates sites mutation plasmid <br />                E           G        N            N        G      B N         N E               G                      pNF402GR-EC1 located restriction <br />            -_xv1                                    1         .II                                      I&apos;             sites. <br />                        S            H <br />                                                                                                                          The demonstration disruption BamHI site does <br />                                                                                                    s <br />  B                                                      URA3 (2.5 Kb)                                                 affect function RAD4 gene suggests that <br />                I__                         14.7               1IIITIEIIITIII                                  EcoRI   plasmid pR169 contains entire gene, site <br />                                                                                                               XmnI    defines end yeast insert carried plasmid <br />      h.. <br />                            1         &#167;.2                  1      11111111111111 8.7                         BgIll   (Fig. 5). To localize RAD4 gene carried out <br />   FIG. 6. Southern hybridization analysis chromosomal yeast                                                        disruptions BglII MstII sites shown Fig. 5. The <br /> DNA gene disruption BamHI site near SPT2 locus.                                                        situated incomplete ORF, MstII site <br /> UV-resistant UV-sensitive disruption mutants generated                                                       small overlapping ORFs identified Roeder et <br /> by transforming Rad+ ura3-52 strain linear DNA fragment                                                      al. (32) derivative plasmid pR169 (Fig. 5). To <br /> carrying URA3 disruption BamHI site defines                                                        distinguish possible mechanisms RAD4 <br /> boundary yeast insert plasmid pR169 (Fig. 5, line b).                                                       inactivation discussed above, control transplacements were <br /> Chromosomal DNA cleaved EcoRI, XmnI, BglII,                                                               carried similar fragments sites of <br /> electrophoresed 0.7% agarose gel, transferred nylon                                                         disruption known outside RAD4 gene. <br /> membrane, hybridized radiolabeled 4-kb HindIII-BamHI                                                       Transplacement HpaI-XhoI fragment URA3 <br /> fragment pR169 (probe). The figure shows restric-                                                    disruption BglII generated 95% UV-sensitive transform- <br /> tion maps relevant region genomic DNA sizes of <br /> the restriction fragments expected hybridize probe                                                  ants (Fig. 5, line d). However, demonstrated above, a <br /> untransformed cells (A) transformed cells (B). In case                                                       control transplacement slightly larger HpaI-XmnI <br /> DNA fragments expected sizes observed (top), regard-                                                      fragment containing URA3 insert BamHI (situated <br /> less UV phenotype. Restriction sites described                                                   outside RAD4) (Fig. 5, line c) resulted UV-sensitive <br /> legend Fig. 1.                                                                                                     transformants. Similarly, disruption MstII site re- <br />                                                                                                                       sulted 95% UV-sensitive transformants (Fig. 5, line e), <br /> yeast genomic DNA. This process referred omega                                                               control disruption XbaI site SP72 gene <br /> transformation (33) provides mechanism generating                                                           (using identical linear DNA fragment) yielded <br /> URA3 disruptions selected sites near chromo-                                                             exclusively UV-resistant colonies (Fig. 5, line f). These <br /> somal RAD4 allele. Disruption RAD4 gene Rad+                                                                results UV-sensitive transformants obtained <br /> cells result UV-sensitive Ura+ transformants.                                                               insertional mutagenesis BglII MstII sites arose <br /> However, analysis complicated fact                                                            result gene disruption inclusion of <br /> transplacing DNA fragments derived plasmid                                                                       sites inactivation plasmid-borne RAD4 gene. Hence <br /> pNF402GR-EC1 carry mutations unknown                                                                   conclude RAD4 gene extends the <br /> locations. Hence, special care taken distinguish                                                               MstII site interval BglII BamHI sites <br /> transplacement events inactivated RAD4 gene                                                              (Fig. 5). <br /> URA3 disruption inactivated gene                                                                  Genetic evidence plasmid pNF400 derivatives <br /> introduction plasmid-borne mutation(s).                                                                        contain RAD4 gene. To confirm plasmid pNF400 and <br />  1188     FLEER ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                            0       10    20      30    40                                0     10    20     30    40 <br />                                UV DOSE         m2l                                               UV DOSE (J/m2) <br />   FIG. 7. A UV-sensitive transplacement derivative SX46a (Rad+) (rad4-11) contains mutation(s) allelic rad4-3. Equal <br /> samples various cell suspensions (vertical rows) ordered horizontally agar plates, plates exposed UV doses <br /> shown (see text details). Plate a: 1, SX46a (Rad+); 2, (rad4-11): 3, rad4-IJIradl-l diploid; 4, rad4-JJIrad2-1 diploid; 5, <br /> rad4-JlIrad3-1 diploid; 6, rad4-lJIrad4-3 diploid. Plate b: 1 2, a; 3, radl-1; 4, rad2-1; 5, rad3-1; 6, rad4-3. <br />  <br /> its derivatives contain RAD4 gene nonspe-                 unlike RAD3 gene (13, 23), RAD4 essential the <br /> cific suppressor UV sensitivity rad mutants, dem-                  viability yeast cells absence DNA damage. <br /> onstrated mutants generated gene disruption                     Transcriptional analysis RAD4 gene. Based the <br /> the genetic determinant UV-sensitive phenotype be-                   localization RAD4 gene described above, likely <br /> haves like rad4 allele. As indicated (Fig. 5, line d),              3&apos; end small ORF closest SPT2 gene in <br /> transplacement 4.3-kb HpaI-XhoI fragment containing                  pR169 represents 3&apos; end RAD4-coding region (Fig. <br /> URA3 disruption BglII site RAD4 renders 95%                    5). To identify 5&apos; end gene mapped RAD4 <br /> the transformants UV sensitive. When mutants                   transcriptional start site hybridizing total yeast RNA <br /> (designated rad4-11; Rad- Ura+) mated radl-1,                        1.65-kb BglII-XhoI DNA probe end labeled the <br /> rad2-1, rad3-1 rad4-3 strains, complementation                   BglII site (Fig. 8). S1 nuclease digestion revealed protected <br /> UV-sensitive phenotype observed diploids                  fragment -790 nucleotides long, placing transcrip- <br /> cases (Fig. 7), demonstrating allelism                 tional start site near SphI site shown Fig. 8. This site <br /> between rad4-11 rad4-3 mutations. We mated                     550 bp downstream BamHI site defines one <br /> the UV-sensitive mutant generated disruption                      end insert pR169 outside RAD4 <br /> MstII site RAD4 gene (rad4-10) (Fig. 5, line e),              gene (Fig. 5). These results provide confirmation that <br /> rad4-2 strain. The resulting UV-sensitive diploid                       entire RAD4 gene present plasmid pR169. <br /> sporulated subjected conventional tetrad analysis. All                  Further transcriptional analysis showed increase in <br /> complete tetrads examined showed linkage genetic                     RAD4 mRNA treatment cells DNA- <br /> determinants Rad- phenotype, expected                    damaging agent 4-nitroquinoline-1-oxide (Fig. 8). On the <br /> if URA3 disruption MstII site inactivated                    contrary, RAD4 messenger actually decreased <br /> RAD4 gene (Table 4). Three unlinked auxotrophic markers                     30 min treatment showed partial recovery 60 <br /> segregated 2+:2-, demonstrating normal meiotic segrega-                     min levels near observed control cells (Fig. 8). A <br /> tion tetrads (Table 4).                                              similar response treatment 4-nitroquinoline-1-oxide <br />    Haploid cells containing disruptions RAD4 gene                 observed examination transcripts URA3 <br /> BglII MstII growth characteristics indistinguishable                gene (Fig. 8). Thus, unlike RAD2 gene S. cerevisiae <br /> from Rad+ strains. We conclude that,                     (18, 31), RAD4 inducible DNA-damaging agents. <br />                                                                                Mapping mutation(s) plasmid pNF402GR-EC1 and <br />                                                                             various chromosomal alleles. The gene disruption experi- <br /> TABLE 4. Genetic analysis UV-sensitive disruption mutanta              ments described localized mutation(s) in <br /> Phenotype            No. marker segregation (ratio):                   pNF402GR-EC1 350-bp interval XmnI <br />  selected      4+:0-        3+:1-        2+:2-        1+:3-         0+:4- <br />                                                                             HpaI sites (Fig. 5). To localize mutation(s) more <br />                                                                             precisely, generated defined gaps plasmid by <br /> Rad+              0            0            0          0             11     restriction enzyme digestion allow gap repair homol- <br /> Trp+              0            1           10          0              0     ogous chromosomal sequences transformation the <br /> Lys+              0            0           11          0              0     gapped plasmid various rad4 mutant strains. The fre- <br /> Ura+              0            0           10          1              0     quency Ura+ Rad+ colonies recovered rad44 <br />   YNN218 disrupted rad4-10 crossed rad4-2 mutant.           transformants -25% gap created between <br />  VOL. 7, 1987                                                                               RAD4 GENE OF S. CEREVISIAE               1189 <br />  <br />                         0      30 60             0    30 60              degradation vivo (21, 26) result rescue the <br />                                                                          chromosomal rad44 mutation. To investigate possibility <br />                                                                          used mutant strains rad44 recipients. No <br />                                                                          UV-resistant colonies recovered rad4-2 and <br />        848         -_                                                    rad4-3 mutants transformation plasmids contain- <br />                              M_         p <br />                                                                          ing gaps PvuI BglII sites the <br />                                   - <br />                                                                          PvuI XbaI sites (Fig. 9A, lines e-h). These results <br />                                                                          demonstrate rad4-2 rad4-3 mutations map <br />                                                                          near 350-bp PvuI-BgIII interval. The availability of <br />        622                                                               new mutation RAD4 gene (rad4-10) known <br />                                                                          location 450 bp left PvuI site (generated by <br />                                                                          disruption RAD4 gene MstII site; Fig. 5, line e), <br />                                                                          afforded opportunity test proximity <br />                                                                          rad44 mutation PvuI site limits efficiency of <br />                                                                          productive repair PvuI-BglII gap. We observed that <br />                                                                          60% Ura+ rad4-10 transformants UV resistant <br />                    RAD4                              URA3                (Fig. 9A, line i). This represents significant increase over <br />                                                                          25% UV-resistant transformants recovered rad44 <br />                                                                          supports view rad44 mutation situated <br />                                       PROBE                              MstII PvuI sites apparently located <br />       5&apos;   *                                               3X            closer the yeast chromosome. <br />                             790 bp                                          As suggested above, additional factor limit <br />                1 <br />                                                                          efficiency productive gap repair possibility that <br />                                                                          gap PvuI BglII sites (Fig. 9A) not <br />                G        X         K         Sp   XB O                    delete site inactivation plasmid pNF402GR-EC1 <br />                                                                          vivo gap expansion PvuI site <br />                                                                          necessary. The absence unique restriction sites <br />                                                                          PvuI MstII sites precluded controlled <br />                                                                          leftward extension PvuI-BglII gap test hypo- <br />    FIG. 8. Si nuclease mapping 5&apos; end RAD4 tran-           thesis. We carried omega transforma- <br /> script. RNA isolated Rad+ strain hybridized DNA        tions linear fragments containing URA3 disruptions at <br /> probe (5&apos; end labeled Bgll site), treated S1 nuclease,   defined sites outside cloned RAD4 gene. As indicated <br /> electrophoresed described text. The figure shows           above, transplacement linear 4.5-kb HindIII-XmnI frag- <br /> protected bands observed treatment cells with <br /> the DNA damaging agent 4-nitroquinoline-1-oxide. (The width       ment containing 3&apos; region RAD4 disrupts flanking <br /> band 0 min suggests doublet consisting        SPT2 gene, does inactivate RAD4 gene (Fig. 9B, <br /> protected fragments -790 -780 bp size.) Posttreatment times       line a). This demonstrates absence inactivating muta- <br /> were 0, 30, 60 min indicated. As control, right panel       tions left XmnI site. However, right <br /> shows protected fragments URA3 transcript obtained           end fragment extended 100 bp include the <br /> similar fashion. The arrows left indicate size markers    PvuI site (Fig. 9B, line b), 350 bp include HpaI <br /> RAD4 transcript mapping. A schematic representation pro-          site (Fig. 9B, line c), progressively increasing numbers of <br /> tected fragment (solid bar) relative physical map RAD4     UV-sensitive transformants generated. Similarly, as <br /> locus shown bottom. The right end arrow represents      shown above, omega transformation carried out <br /> the mapped transcriptional start sites RAD4 gene. The <br /> restriction sites shown described legend Fig. 1.        fragment containing 5&apos; region RAD4 (extending <br />                                                                          PvuI site rightward XmnI site outside the <br />                                                                          gene), results URA3 disruption BamHI, no <br />                                                                          UV-sensitive transformants observed (Fig. 9B, line d). <br /> the PvuI XbaI sites (1.6 kb) (Fig. 9A, line a)            However, extension left end fragment 100 bp <br /> the PvuI BglII sites (0.35 kb) (Fig. 9A, line b). However,           include XmnI site RAD4 gene (Fig. 9B, line e) or <br /> when deletion BglII XbaI sites                   2.4 kb include EcoRI site vector (Fig. 9B, line <br /> (immediately right PvuI-BglIll deletion), 2%          f) resulted progressively increasing frequency UV- <br /> of Ura+ transformants UV resistant (Fig. 9A, line               sensitive transformants. The results transplace- <br /> c). These results localize site inactivation region          ments obtained gap repair plasmids <br /> with approximate limits defined PvuI BglII sites.             demonstrate mutations generated the <br /> Parenthetically, noted repair              propagation pNF402GR1 E. coli HB101 localized to <br /> gap PvuI BgIII sites carried              100-bp interval defined XmnI PvuI sites <br /> plasmid (pNF417) (Fig. 1) containing yeast insert equiva-              highlighted Fig. 9B. <br /> lent present pR169 (except exclusion of <br /> most spt2-1 allele) (Fig. 9A, line d), frequency of <br /> UV-resistant Ura+ transformants essentially                                         DISCUSSION <br /> as just described repair plasmid (pNF418) (Fig. <br /> 1) containing additional upstream sequences (Fig. 9A, line                 Previous attempts isolate RAD4 gene screening <br /> b), providing independent evidence entire                  different yeast genomic libraries recombinant plasmids <br /> RAD4 gene present plasmid pR169.                                   complement UV sensitivity rad4 mutants were <br />    One explanation limited recovery UV-resistant              unsuccessful. The experiments reported the <br /> transformants repair gapped plasmids strain                  isolation functional RAD4 gene alternative strat- <br /> rad44 expansion defined gaps exonucleolytic                egy provide explanation failure isolate the <br />  1190     FLEER ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />              A        I <br />                                 I <br />                                        pR169 -H <br />                                         pNF417 --I <br />                                 I       pNF418                                  I <br />                       SPT2                    RAD4                                   recipient              UV-resistant <br />                                                    H                H <br />                                                                                       strain:                gap-repair <br />                           H              ~H            Xe HHm       xH          H <br />                      5) <br />                       4 <br />                             H   H         H    H <br />                                                          d <br />                                                                 H <br />                                                                 4 <br />                                                                         m   H t <br />                                                                             O                              transformants: <br />                           m xP            X x            m      x       m   x to <br />  <br />  <br />  <br />  <br />               a <br />                               II <br />                                I dAzl                                                                        25% (6/25) <br />               b                                                                                              25% (6/25) <br />                                                                                         rad44                 2% (1/50) <br />               c <br />               d                                                                                              20% (5/25) <br />               e <br />                                                                                     }rad4-2                   0 (0/25) <br />               f                                          .: .M                                                0 (0/25) <br />               g                                                                     }   rad4-3                0 (0/25) <br />               h                                         .7 -1 ..                                              0 (0/25) <br />                                                                                     )rad4-10                 60% (29/50) <br />  <br />  <br />                                                                                       recipient             UV-sensitive <br />                                                                                        strain:               disruption <br />                                                                                                               mutants: <br />  <br />                                                                                                              0 (0/200) <br />              b                                                                                                 5% (9/200) <br />               c <br />                                                                                         RAD4 <br />                                                                                                               15% (10/70) <br />               d                                                                                                0 (0/200) <br />               e                                                                                                3% (3/92) <br />               f                                                                                               70% (34/50) <br />   FIG. 9. Mapping chromosomal plasmid mutations RAD4 gap repair (A) gene transplacement (B). In case, the <br /> physical map SPT2 RAD4 loci relevant flanking sequences shown. The hatched box represents location of <br /> the RAD4 gene established gene disruption transcript mapping (see text details). (A) Lines indicate yeast inserts <br /> in centromeric plasmids pNF417 (line d) pNF418 (all others). Double-stranded gaps created restriction digestion vitro are <br /> represented dotted lines. (B) Lines e indicate linear DNA fragments used gene transplacement assay described text. <br /> The shaded triangles represent insertions fragments carrying URA3 gene restriction sites indicated (see text details). <br /> The hatched vertical zone XmnI PvuI sites denotes localization site(s) inactivation plasmid pNF402GR-EC1. <br />  VOL. 7, 1987                                                                           RAD4 GENE OF S. CEREVISIAE                 1191 <br />  <br /> functional gene libraries propagated E. coli. It       knowledge RAD4 gene represents documented <br /> known previous studies SPT2 RAD4 genes           example inability propagate yeast gene E. coli <br /> are tightly linked genetically (36). We demonstrated        HB101 suggests presence yeast genes in <br /> physical linkage loci showing integration      recombinant libraries unrecognized similar rea- <br /> a 4-kb yeast fragment (containing mutant allele SPT2)         sons. <br /> into rad4 mutant strains generates UV-resistant integrants,           We provided genetic evidence complement- <br /> albeit low frequency. Rescue relevant genomic              ing activity present gap-repaired plasmids is <br /> sequences integrants yielded plasmid           lost propagation E. coli RAD4 gene, and <br /> with inactivated RAD4 gene. A functional gene               physically mapped gene position immedi- <br /> restored deleting (presumably) mutated sequences            ately upstream SPT2 gene. This localization demon- <br /> repairing deletions gene transfer normal homol-        strates entire gene present plasmid pR169, <br /> ogous chromosomal sequences vivo. Direct transforma-            originally isolated G. S. Roeder (personal communica- <br /> tion gap-repaired plasmid rad4 mutants            tion). Hence, tight genetic linkage SPT2 and <br /> restores UV resistance wild-type levels. However,         RAD4 genes (36) confirmed physical analysis <br /> complementing activity lost plasmid           presented here. Our initial difficulty recognizing that <br /> propagated E. coli.                                             plasmid pR169 contains entire RAD4 gene stems from <br />    Inactivation cloned RAD4 gene occur         inability detect Rad+ transformants integration of <br /> or unique sites gene, reflecting presence       plasmid rad44 mutant. This presumably reflects <br /> structural features consistently modified E. coli.     close proximity chromosomal rad44 muta- <br /> Several examples phenomenon noted           tion site mutations rad4 allele in <br /> other eucaryotic genes DNA sequences           plasmid pR169, resulting low frequency of <br /> be propagated certain E. coli strains (3, 20, 37). In       recombinational events them. <br /> hands, transformation plasmids carrying functional               The presence inactivating mutations rad4 allele in <br /> RAD4 gene different recA recBC sbcB mutants did           pR169 altered translational reading frame in <br /> not prevent inactivation gene (Fleer et al., prepa-      region sequenced Beard Smith (personal commu- <br /> ration). A pathway recombination E. coli            nication), leading erroneous identification of <br /> independent recA, recBC, sbcB genes involves        multiple ORFs (32). By shifting reading frame two <br /> recE gene, encodes exonuclease VIII regulated         positions -450 bp apart, single continuous ORF be <br /> by sbcA gene (16). We tested recE mutants         generated, 3&apos; end located 280 bp upstream of <br /> tolerance RAD4 gene E. coli. However,              beginning SPT2 coding region. This ORF is <br /> interesting note site-specific deletions sequences 3&apos;   interrupted BglII site defines end the <br /> to adult P globin gene <span id='am-41' about='obo:NCBITaxon_8355' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8342 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8360 obo:NCBITaxon_2759 obo:NCBITaxon_262014 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8353 obo:NCBITaxon_7776 obo:NCBITaxon_8352 obo:NCBITaxon_30319 obo:NCBITaxon_41666 obo:NCBITaxon_8292'><span id='am-42' property="oboInOwl:hasExactSynonym" content="platanna" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="clawed frog" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="common platanna" datatype="xsd:string"></span><span id='am-45'  ></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Bufo laevis" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Xenopus laevis" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="African clawed frog" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Xenopus leavis" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Xenopus laevis (Daudin, 1802)" datatype="xsd:string"></span>Xenopus laevis</span> occurs range     sequenced region pR169. We contend ORF <br /> of E. coli hosts including recA recBC sbcB recBC          contains coding region RAD4 (Fig. 5). The stop <span id='am-61' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-62'  ></span><span id='am-63' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> <br /> sbcA recE strains (10).                                            defines 3&apos; end ORF separated the <br />    An alternative explanation inability propagate     RAD4 transcriptional start site distance -2.3 kb, a <br /> functional RAD4 gene E. coli Rad4 protein toxic      value agrees unassigned 2.6-kb <br /> to organism, plasmid copy number-                transcript detected pR169 hybridization probes (M. <br /> dependent manner. We observed                Smith, personal communication). Our estimated size the <br /> plasmids isolated E. coli HB101 (designated               RAD4 gene (-2.3 kb) indicates encode protein <br /> pNF402GR-EC <span id='am-53' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span>) stable propagation E.         Mr -88,000. <br /> coli arid restriction analysis evidence exten-          Haploid cells containing disruptions RAD4 gene at <br /> sive deletion rearrangement (Fleer et al., preparation).     different sites normal growth characteristics, and <br /> If plasmids RAD4 gene randomly inacti-           Southern analysis demonstrates single copy of <br /> vated point mutations small deletions,         RAD4 present yeast genome. We conclude <br /> anticipate different plasmids containing functional         that, unlike RAD3, RAD4 essential viability of <br /> RAD4 gene suffer mutational inactivation indepen-         haploid yeast cells absence DNA damage. In <br /> dent sites passaged E. coli. Evidence        addition, RAD4 mRNA increased cells exposed a <br /> support notion comes observation             DNA-damaging agent known enhance transcriptional <br /> integration plasmid pR169 rescued derivative         activity RAD2 gene. <br /> pNF400 various rad4 mutants yielded different results            The molecular cloning RAD4 gene completes our <br /> (Table 2). This suggests pNF400 rescued         search known RAD genes absolutely required for <br /> a UV-resistant rad4-2 integrant contained            nucleotide excision repair yeast S. cerevisiae. If the <br /> wild-type RAD4 allele inactivated propa-          inability propagate plasmids functional RAD4 gene <br /> gation E. coli, yielding stable derivative rad4        E. coli reflects toxic effects native Rad4 protein, <br /> mutation site(s) different plasmid pR169.        controlled reversible inactivation gene should <br /> Since plasmid pR169 yielded UV-resistant integrants             facilitate propagation mutant gene E. coli and <br /> after transformation rad44, plasmid pNF400 yielded          expression wild-type gene yeast. Such studies in <br /> no UV-resistant integrants transformation rad4-2      progress. <br /> and rad4-3 strains, site(s) inactivation pR169 is <br /> apparently close rad44 mutation,                                 ACKNOWLEDGMENTS <br /> pNF400 closer rad4-2 rad4-3 mutations. We <br /> have localized site RAD4 inactivation particular         We particularly grateful G. Shirleen Roeder gift of <br /> plasmid designated pNF402GR-EC1 100-bp region          plasmid pR169 helpful discussions Mike Smith <br /> middle gene. Detailed examination lesion(s)       Caroline Beard communicating unpublished <span id='am-108' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-109' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-112'  ></span>data</span>. We thank <br /> this plasmid await sequencing region              Jane Cooper, Louie Naumovski, David Kalainov, Bill Weiss, <span id='am-103' about='obo:NCBITaxon_1191644' typeof='obo:NCBITaxon_34667 obo:NCBITaxon_79514 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71528 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_294789 obo:NCBITaxon_197563 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-104' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-105'  ></span><span id='am-106' property="rdfs:label" content="Linda" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Linda</span> <br /> comparison sequence wild-type gene. To         Couto, Paul Saxon, <span id='am-98' about='obo:NCBITaxon_1009818' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_263623 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_71526 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_7080 obo:NCBITaxon_50557 obo:NCBITaxon_115347 obo:NCBITaxon_7041'><span id='am-99' property="rdfs:label" content="Roger" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-102'  ></span>Roger</span> Schultz, Wolfram Siede helpful <br />  1192       FLEER ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br /> comments suggestions Jean Oberlindacher typing            19. Maniatis, T., E. F. Fritsch, J. Sambrook. 1982, Molecular <br /> manuscript.                                                                   cloning: laboratory manual. Cold Spring Harbor Laboratory, <br />   These studies supported Public Health Service research               Cold Spring Harbor, N.Y. <br /> grant CA 12428 National Institutes Health. R.F.       20. Nader, W. F., T. D. Edlind, A. Huettermann, H. W. Sauer. <br /> recipient postdoctoral fellowship Deutscher                     1985. Cloning <span id='am-31' about='obo:NCBITaxon_5790' typeof='obo:NCBITaxon_5789 owl:Thing obo:NCBITaxon_1115744 obo:NCBITaxon_33680 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_142796 obo:NCBITaxon_137627 obo:NCBITaxon_2759 obo:NCBITaxon_554915'><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Physarum" datatype="xsd:string"></span><span id='am-35'  ></span>Physarum</span> actin sequences exonuclease- <br /> Akademischer Austauschdienst, G.A.P. supported                   deficient bacterial host. Proc. Natl. Acad. Sci. USA <br /> Public Health Service postdoctoral fellowship Fogarty                82:2698-2702. <br /> Foundation.                                                               21. Naumovski, L., G. Chu, P. Berg, E. C. Friedberg. 1985. <br />                                                                               RAD3 gene Saccharomyces cerevisiae: nucleotide sequence <br />                         LITERATURE CITED                                      wild-type mutant alleles, transcript mapping, aspects <br />  1. Berk, A., P. Sharp. 1977. Size mapping early                   gene regulation. Mol. Cell. Biol. 5:17-26. <br />     adenovirus mRNAs gel electrophoresis S1 endonuclease-           22. Naumovski, L., E. C. Fri <br /> </body></html>