<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>        Report <br />  <br /> Cell Cycle 10:10, 1690-1698; May 15, 2011; &#169; 2011 Landes Bioscience <br />  <br />  <br />  <br />                     Inviability DNA2 deletion mutant <br />                    DNA damage checkpoint <br />                    Martin E. Budd,1 Igor A. Antoshechkin,2 Clara Reis,1,3 Barbara J. Wold2 Judith L. Campbell1,2,* <br />                                  1 <br />                                   Division Chemistry; 2Division Biology; <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5'  ></span><span id='am-6' property="rdfs:label" content="California" datatype="xsd:string"></span>California</span> Institute Technology; Pasadena, CA USA; <br />                                        3 <br />                                          Telomere Genome Stability Lab; Instituto Gulbenkian Ciencia; Oeiras, Portugal <br />  <br />  <br />                     Key words: yeast, RAD27, RAD9, RAD53, Okazaki fragment processing, DNA replication, exo1 <br />  <br />                  Abbreviations: OF, Okazaki fragment; OFP, Okazaki fragment processing; DSB, double-strand break <br />  <br />  <br />            Dna2 dual polarity exo/endonuclease, 5&apos; 3&apos; DNA helicase involved Okazaki Fragment Processing (OFP) <br />            Double-Strand Break (DSB) Repair. In yeast, DNA2 essential gene, expected DNA replication protein. <br />            Suppression lethality dna2&#916; mutants occur mechanisms: overexpression RAD27scFEN1, <br />            encoding 5&apos; 3&apos; exo/endo nuclease processes Okazaki fragments (OFs) ligation, deletion PIF1, 5&apos; 3&apos; <br />            helicase involved mitochondrial recombination, telomerase inhibition OFP. Mapping novel, spontaneously <br />            arising suppressor dna2&#916; reveals mutation rad9 double mutation rad9 mrc1 suppress the <br />            lethality dna2&#916; mutants. Interaction dna2&#916; DNA damage checkpoint mutations provides insight why <br />            dna2&#916; lethal rad27&#916; not, evidence shows Rad27ScFEN1 processes Okazaki fragments, <br />            Dna2 processes subset. <br />  <br />  <br />  <br />  <br />                             Introduction                                                Rad9 Mrc1 &#8220;mediators&#65533;? transduce signal <br />                                                                                     Mec1/Tel1 Rad53, Chk1 Dun1 kinases.13-15 <br /> Chromosomal DNA subjected constant damage                            RAD9 is, fact, original defining gene DNA dam- <br /> from exogenous agents, ionizing radiation, ultravio-                        age checkpoint response.16 Rad9 scaffold protein binds <br /> let radiation base-damaging drugs, damage                         forkhead domain Rad53 phosphorylation Mec1.17 <br /> arising faulty DNA replication spontaneous base loss.                      The Rad9/Rad53 interaction results Rad53 kinase activation <br /> Unrepaired misrepaired damage lead cell lethality                      Rad53 dimerization subsequent transphosphoryla- <br /> genome instability, translocations aneuploidy. The                       tion.14,18 Activated Rad53 delays G2 /M transition inhibiting <br /> DNA damage checkpoint involves cascade protein kinases                         degradation cohesin inhibiting spindle elongation.19,20 <br /> that arrests slows cell cycle, allowing time repair                  Activated Rad53 stabilizes stalled replication forks.21-23 rad9&#916; <br /> the damage. Additional functions checkpoint include sta-                     mutants sensitive IR, UV prematurely enter mitosis <br /> bilizing DNA replication fork stimulating DNA repair.                       damaged DNA.24 Mrc1 non-essential DNA replication <br /> Yeast (<span id='am-7' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>) Mec1 Tel1 <span id='am-104' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-105' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108'  ></span>phos</span>-                        protein binds Mcm2, replicative helicase, Cdc45, a <br /> phoinositol kinase-related protein kinases initiate DNA                    subunit replicative helicase DNA polymerase &#949; (pol &#949;), <br /> damage response response single-strand double-strand                      leading strand polymerase.25,26 Mrc1 couples DNA unwind- <br /> DNA breaks.1,2 Two downstream kinases, Rad53 Chk1,                          ing replicative synthesis, mrc1&#916; mutants, replication <br /> activated Mec1/Tel1 kinases DNA damage,                           fork rate reduced.13,27,28 After DNA damage Mrc1 phosphory- <br /> Mec1 primary upstream kinase.3 Activated Rad53                          lated Mec1, primarily N terminus, allowing Mrc1 to <br /> positively regulates additional kinase, Dun1.4 Dun1 regulates                    activate Rad53 undefined mechanism.29 The mechanism <br /> levels ribonucleotide reductase arrests cells having DNA                     activation Rad53 Mrc1 presumably differs Rad9 <br /> damage late anaphase.4 Additional activities Rad53                      activation Rad53, Mrc1 travels replication <br /> DNA damage include stimulation DNA repair phosphory-                          fork.30 However, test Mrc1 phosphorylation Mec1 <br /> lation Rad55, inhibition firing late origins, induction                 required Rad53 activation, Osborn Elledge mutated all <br /> transcription DNA repair genes phosphorylation                        potential Mec1/Tel1 phosphorylation sites Mrc1, creating <br /> nuclease Exo1 apparent inhibition.5-7 Exo1 5&apos; 3&apos;                    mutant mrc1-AQ. Activation Rad53 HU treatment is <br /> nuclease involved mismatch base repair, double-strand break                      abolished rad9&#916; mrc1-AQ mutant, rad9&#916; mrc1&#8209;AQ <br /> (DSB) repair 5&apos; 3&apos; degradation uncapped telomeres.8-12                    mutant sensitive low doses HU. Unlike mrc1&#916; mutant, <br />  <br />  *Correspondence to: Judith L. Campbell; Email: jcampbel@caltech.edu <br />  Submitted: 03/28/11; Accepted: 03/29/11 <br />  DOI: 10.4161/cc.10.10.15643 <br />  <br />  <br /> 1690	                                                                      Cell Cycle	                                                 Volume 10 Issue 10 <br />        Report                                                                                                                             report <br />  <br />  <br />  <br />                                                                               DNA replication, DNA repair, histone modification cel- <br />                                                                               lular stress response. However, noticed dif- <br />                                                                               ferences. In particular, dna2 mutants synthetically lethal <br />                                                                               DNA damage checkpoint mutants mec1, rad53, rad9, rad24, <br />                                                                               rad17, ddc1, mec2, rad27 mutants synthetically lethal <br />                                                                               checkpoint genes. The DNA damage checkpoint <br />                                                                               mutants synthetically lethal dna2 mrc1&#916;, tof1&#916; <br />                                                                               csm3&#916;; however, replication pro- <br />                                                                               gression complex important DNA synthesis.25 Therefore, <br />                                                                               synthetic lethality presumably results role MRC1, <br />                                                                               TOF1 CSM3 DNA replication. <br />                                                                                  Previously, suppressors dna2&#916; affected components of <br />                                                                               OFP system. In work, report surprising observa- <br />                                                                               tion spontaneously arising suppressor dna2&#916; mutants that, <br />                                                                               instead, disrupts DNA damage checkpoint inactiva- <br />                                                                               tion Rad9 &#8220;mediator.&#65533;? We propose DNA repair <br />                                                                               functions Dna2 contribute essential function, the <br />                                                                               absence repair function, cell cycle inhibition the <br />                                                                               checkpoint leads permanent arrest survival, it <br />                                                                               does rad27&#916; cells, repair proficient. <br />  <br />                                                                                                           Results <br />  <br />                                                                               Identification extragenic suppressor lethal- <br />                                                                               ity dna2&#916;. dna2&#916;::kanMX strains inviable; however, <br />                                                                               inviability suppressed deletion PIF1. When a <br />                                                                               DNA2/&#8202;dna2&#916;::kanMX PIF1/pif1&#916;::HIS3 diploid sporulated <br />  Figure 1. Mutation RAD9 suppresses lethality dna2&#916;. Tetrads        dissected, dna2&#916;::kanMX spores segregate with <br />  dissection MB221 DNA2/dna2&#916;::kanMX RAD9/rad9-320 illus- <br />  trate presence mutation suppressing lethality dna2&#916;. The <br />                                                                               pif1&#916;::HIS3. During dissection DNA2/dna2&#916;::kanMX <br />  dna2&#916; colonies circled.                                                  PIF1/pif1&#916;::HIS3 diploid, however, noticed kanMX <br />                                                                               (kanomycin resistant) his3 (did grow histidine minus <br />                                                                               plates) colony, suggesting dna2&#916; PIF1. <span id='am-58' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-59'  ></span><span id='am-60' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> indicated <br /> the mrc1-AQ mutant does exhibit slow DNA replication                    dna2&#916; mutant suppressed mutation other <br /> phenotype specifically checkpoint-defective.30                         pif1&#916;::HIS3. To check suppressor (supX ) represented <br />     Dna2 5&apos; 3&apos; helicase 5&apos; 3&apos; 3&apos; 5&apos; exo/endonucle-      genetic suppressor nonspecific suppressor (e.g., suppression <br /> ase single-strand annealing strand exchange activity.31-36           resulting overproduction heat shock proteins), viable <br /> dna2-1 mutants defective DNA replication accumu-                   dna2&#916; PIF1 supX strain crossed BY4741-derived strain <br /> late short DNA fragments restrictive temperature,37,38             sporulated. Thirteen tetrads dissected (Fig. 1). If the <br /> dna2&#916; mutants inviable. Rad27scFEN1 (Flap Endonuclease)                   dna2&#916; strain suppressed suppressor, expected num- <br /> is 5&apos; 3&apos; exo/endonuclease processes majority             ber dna2&#916; strains recovered 13. In keeping this, <br /> Okazaki fragments.39-41 Dna2 assists Rad27scFEN1 RNA primer                28 DNA2 spores 10 dna2&#916;::KanMX spores recovered. <br /> removal coordinated Rad27scFEN1, Dna2, RPA interac-                 The recovery dna2&#916; strains 20 25% dissected <br /> tion subset Okazaki fragments.42-46 rad27&#916; mutants                tetrads suggests mutation segregating cross <br /> viable exhibit high frequency mutation repeated              suppressing lethality dna2&#916;. The dna2&#916; spores in <br /> DNA sequences resulting delayed OFP.47 The dna2-1                        cross temperature-sensitive ionizing radiation (IR)- <br /> mutants increased frequency mutations,                sensitive (not shown). The IR sensitivity dna2&#916; supX strain <br /> and therefore, high mutational rate rad27&#916; mutants sug-                significantly different IR resistance dna2&#916; pif1&#916; <br /> gests Rad27scFEN1 processes majority Okazaki fragments.            strain.52 Therefore, supx likely pif1&#916;. <br /> This raises obvious question: OFP essential,                  To identify supX gene, DNA prepared segregant <br /> dna2&#916; mutants inviable rad27&#916; mutants viable?                             (MB220) cross BY4741(WT) x dna2&#916;, genome <br />     Comparison genetic interaction network DNA2                 sequenced compared strain BY4741 (see Materials <br /> RAD27 provided clue answer.48-51 dna2 mutants                   Methods). The complete list mutations MB220 be <br /> synthetic lethality approximately 46 genes, two-               Supplemental Table 1. Three genes have <br /> dimensional clustering revealed DNA2 RAD27                  C-terminal deletions. Among these, logical candidate <br /> share greatest number interactions, consistent func-              suppressor dna2&#916; RAD9. No mutations in <br /> tion pathway. This overlapping set mutants affect              PIF1 gene. <br />  <br />  <br /> www.landesbioscience.com	                                             Cell Cycle	                                                           1691 <br />                                                                                         lowest concentration galactose results detectable <br />                                                                                        induction GAL1 promoter, level induction <br />                                                                                        1/100 induction of&#160;2%.56 Concentrations <br />                                                                                        0.05% 0.2% galactose resulted minor inhibition <br />                                                                                        growth rad9&#916;&#160; DNA2. This level expression was <br />                                                                                        sufficient completely suppress IR sensitivity the <br />                                                                                        strain, (not shown). We&#160;then used induc- <br />                                                                                        tion conditions dna2&#916; rad9-320 strain carrying the <br />                                                                                        GAL1-RAD9 plasmid. On glucose containing plates, the <br />                                                                                        RAD9 plasmid resulted detectable growth inhibi- <br />                                                                                        tion. On 0.05% 0.2% galactose, RAD9 plasmid, <br />                                                                                        control <span id='am-99' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-100' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span>, completely blocked growth <br />                                                                                        dna2&#916; rad9-320 strain. These results that <br />                                                                                        expression RAD9 causes inviability strain dna2&#916; <br />                                                                                         rad9-320. We conclude DNA damage check- <br />                                                                                         point deleterious beneficial dna2&#916; <br />    Figure 2. Expression RAD9 dna2&#916; rad9-320 strain results lethality. <br />    (A) Contol showing RAD9 expression DNA2 rad9&#916; lethal. Strain <br />                                                                                         mutant. <br />    MB228-DNA2::9MYC rad9&#916;::HIS3 transformed plasmids pSEY-                     rad9&#916; suppresses dna2&#916; lethality, rad9&#916; <br />    18GAL BG1805-Rad9. Cells carrying plasmids incubated                 mrc1-AQ suppresses efficiently. The dna2&#916; <br />    medium lacking uracil containing 2% glucose, 0.05% galactose 2% raffinose         rad9-320 strain viable grew slowly than <br />    0.2%&#160;galactose 2% raffinose. (B) Expression RAD9 dna2&#916; rad9-320 lethal.     wild type. We reasoned con- <br />    Strain MB220 dna2&#916; rad9-320 trainsformed plasmids pSEY18GAL, <br />    BG1805-Rad9 pSEY18GAL-DNA2 incubated 2% glucose, 0.05%&#160;ga- <br />                                                                                         tinued induction S-phase checkpoint MRC1, <br />    lactose 2% raffinose, 0.2% galactose 2% raffinose.                                   mutation checkpoint function MRC1 <br />                                                                                         enhance growth dna2&#916; rad9-320. We <br />                                                                                         tested combined effect rad9&#916; mrc1-AQ double <br />      RAD9 encodes 1,309 amino acid protein, suppres- mutation temperature sensitivity dna2-1 strains. The <br /> sor mutation consisted stop <span id='am-63' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-64'  ></span><span id='am-65' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>codon</span> amino acid&#160;320. Rad9 suppressive effect rad9&#916;, mrc1-AQ rad9 mrc1-AQ mutants <br /> possesses domains required activity. Amino dna2-1 growth 23&#176;C, 28&#176;C 34&#176;C shown Figure 3. <br /> acids 1&#8211;231 required activate Chk1; residues 390&#8211;458 dna2-1 strains cease growth 28&#176;C; dna2-1 rad9&#916; dna2-1 <br /> residues 593&#8211;620 phosphorylated Mec1 contain mrc1-AQ cease growth 34&#176;C. Thus, rad9&#916; suppresses <br /> Rad53 interaction domain; residues 754&#8211;947 contain Tudor dna2&#916; lethality, does suppress dna2-1 temperature sensi- <br /> domain, binds methylated histones, residues 1,027&#8211; tivity. However, dna2-1 rad9&#916; mrc1-AQ capable growth at <br /> 1,310 contain BRCT repeats bind phosphorylated 34&#176;C. Thus, combined rad9&#916; mrc1-AQ mutation strong <br /> histones.14,53-55 The BRCT repeats required Rad9 dimer- suppressor temperature-sensitive phenotype dna2-1 <br /> ization, necessary step Rad53 activation. The 1&#8211;320 amino strains, similar pif1&#916; suppression dna2-1 strains.52 We <br /> acid fragment rad9 suppressor possesses region tested ability mrc1-AQ combined rad9&#916; an <br /> required activation Chk1 missing Rad53 interac- additional suppressor dna2&#916; lethality. Five independent <br /> tion domain, Tudor domain BRCT domain.14 Since WT/dna2&#916;::natR rad9&#916;::kanMX mrc1-AQ::HIS3 diploids were <br /> the chromatin binding region, dimerization region Mec1 sporulated dissected average 12 13 tetrads per <br /> phosphorylation sites missing, mutant pre- diploid. The <span id='am-94' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-95'  ></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> crosses illustrated Figure <br /> sumably completely defective activation Rad53 Chk1. 4. The results 62 total diploids scored follows: (the <br /> The rad9-320 mutation partially accounts IR sensitivity number following genotype indicates number spores <br /> of strain, dna2&#916; rad9-320 strain significantly identified): WT, 22; rad9&#916;, 32; mrc1-AQ, 29; rad9&#916; mrc1-AQ, <br /> more sensitive irradiation rad9&#916; strain (Budd M 27; dna2&#916; rad9&#916;. 5; dna2&#916; mrc1-AQ, 1; dna2&#916; rad9&#916; mrc1-AQ, <br /> Campbell JL, unpublished data). We designate suppres- 17; dna2&#916;, 0. Assuming 100% viability, 31 spores from <br /> sive allele (supX&#160;above) rad9-320, indicative site genotype recovered (62 tetrads x 4 spores &#247; 8&#160;gen- <br /> mutation.                                                                        otypes = 31). From data, point viable <br />      RAD9 reverses suppression dna2&#916; rad9&#8209;320. To dna2&#916; rad9&#916; spores serve independent control, showing <br /> confirm rad9-320 mutation suppressing dna2&#916;, complete deletion RAD9 suppresses dna2&#916; inviability, <br /> we asked expression RAD9 restore lethality consistent rad9-320 likely functioning Rad9-null in <br /> dna2&#916; rad9-320. RAD9 cloned yeast plasmid con- suppression. Second, 23 dna2&#916; strains recovered combi- <br /> trol GAL1 promoter introduced dna2&#916; rad9-320. nation checkpoint mutants, rad9 and/ <br /> Since overexpression RAD9 itself cause slow growth, mrc1-AQ, recovered dna2&#916; com- <br /> first determined induction conditions lethal bined checkpoint mutant. All dna2&#916; rad9&#916;, dna2&#916; <br /> rad9&#916; DNA2 strain (Fig. 2A). On 2% glucose plates, plasmid mrc1-AQ dna2&#916; rad9&#916; mrc1-AQ strains form colonies <br /> had effect growth strain. 0.05% galactose restreaking 23&#176;C 36&#176;C. mrc1-AQ barely <br />  <br />  <br /> 1692	                                                                Cell Cycle	                                                   Volume 10 Issue 10 <br />                                                                                        date, N-terminal domain remaining are <br />                                                                                       Chk1 activation binding replication initiator <br />                                                                                       protein Dbp11.53 Indeed, independent experiments, <br />                                                                                       verified rad9&#916; suppressed dna2&#916; (Fig.&#160;4), and <br />                                                                                       expression RAD9 reversed suppression by <br />                                                                                       rad9-320 (Fig. 2). None additional mutations <br />                                                                                       identified DNA sequencing original suppres- <br />                                                                                       sor strain appears candidate suppressor muta- <br />                                                                                       tion. DNA sequencing did detect mutations in <br />                                                                                       PIF1 gene, previously identified as <br />                                                                                       dna2&#916; suppressor. Interestingly, rad9&#916; mrc1AQ <br />                                                                                       double mutation resulted efficient suppression <br />                                                                                       mutation alone. We conclude inef- <br />                                                                                       ficiently processed DNA intermediates arising to <br />                                                                                       absence Dna2 activate Rad53 <br />                                                                                       Chk1 cell cycle checkpoint functions dna2&#916; <br />                                                                                       rad9&#916; mrc1AQ mutants. In way, terminal arrest <br />                                                                                       avoided, allowing division complete <br />                                                                                       absence Dna2. <br />                                                                                           Previously, specific endogenous suppressors dna2 <br />                                                                                       lethality attributed increasing efficiency <br />                                                                                       OFP. This mediated overexpression of <br />                                                                                       Rad27ScFEN1, directly involved OFP, of <br />                                                                                       BLM, WRN, Exo1 Mph1, provide auxiliary <br />                                                                                       functions stimulate nuclease activities Dna2 <br />                                                                                       Rad27scFEN1.57-61 In <span id='am-31' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33'  ></span><span id='am-34' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Schizosaccharomyces</span> pombe, <br />                                                                                       overexpression FEN1, pol &#948; DNA ligase suppress a <br />                                                                                       dna2 helicase mutant.62 A related phenomenon involves <br />                                                                                       suppressing production irreparable toxic struc- <br />                                                                                       tures long 5&apos; flaps Pif1 helicase. pif1 muta- <br />                                                                                       tion allows dna2-1 strains grow 37&#176;C.52,57 Pif1 a <br />                                                                                       5&apos; 3&apos; helicase unwinds forked DNA substrates, <br />                                                                                       RNA/DNA duplexes removes telomerase bound <br />                                                                                       telomeres.63,64 A dna2&#916; pif1&#916; strain viable but <br />   Figure 3. rad9&#916; mrc1AQ double mutation suppresses temperature-sensitive         temperature sensitive; however, dna2&#916; pif1&#916; pol32&#916; <br />   growth dna2-1 mutants. Mutants dna2-1, dna2-1 rad9&#916;, dna2-1 mrc1AQ, dna2-1       strain does grow 37&#176;C.52 Thus, Pol32 additional <br />   rad9&#916; mrc1AQ, rad9&#916; mrc1AQ incubated plates 23&#176;C, 28&#176;C 34&#176;C.         suppressor dna2 mutants. Pol32 stimulatory <br />                                                                                       subunit pol &#948;.65 The suppression dna2&#916; pif1&#916; <br /> suppressed dna2&#916;, suppressor recovered. rad9&#916;                        suppression pol32&#916; suggests long <br /> is efficient suppressor mrc1-AQ, rad9&#916; mrc1-AQ flaps created OPF pol &#948;, Pol32 Pif1 poten- <br /> provided efficient suppression, recovered tially lethal substrates unless processed Dna2 (Fig. 5). <br /> at higher frequency, dna2&#916; rad9&#916; mrc1-AQ colo- Additional suppressors dna2 mutants involved OFP. <br /> nies necessarily larger dna2&#916; rad9&#916; colonies E1A overexpression identified suppressors <br /> (Fig. 4, compare squares circles).                                      Dna2.66 E1A encoded human adenovirus, activates <br />                                                                             viral transcription promote cell cycle progression. <span id='am-79' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-80' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-82'  ></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>TOR</span> <br />                             Discussion                                      overexpression suppresses dna2 helicase mutants.67 The <br />                                                                             mechanisms involved understood. <br /> We identified rad9&#916; novel suppressor lethality              Our current work identifies new compensatory pathway <br /> dna2&#916; using genome sequencing spontaneous suppres- propose involves desensitizing dna2 mutants replica- <br /> sor. Suppression complete absence Dna2 surprising, tion stress encountered absence. In addition newly <br /> because rad9&#916; suppresses growth defect subset documented suppression lethality deletion DNA2 by <br /> dna2 point mutants does suppress inability dna2-1 deletion RAD9, rad9&#916; mec1 mutants observed <br /> mutants grow 34&#176;C. The rad9-320 mutation identified suppress growth defects subset dna2 point mutants <br /> this work gives rise stop codon amino acid 320, synthetic lethality dna2-2 ctf4&#916; mutant.67,68 rad9 <br /> expected create functionally null allele. Even fragment mrc1 mutants desensitize DNA replication mutants <br /> survives nonsense-mediated decay, functions associated, replication damage well. cdc9 rad9 mrc1 mutants progress <br />  <br />  <br /> www.landesbioscience.com	                                         Cell Cycle	                                                              1693 <br />  further DNA synthesis cdc9 rad9, progress fur- <br /> ther cdc9 mutants 36&#176;C.69 The lethality orc2-1 and <br /> orc1-4 mutants restrictive temperature suppressed by <br /> rad9&#916; mutation.70 This general interest, desensitiz- <br /> ing cells DNA damage DNA replication hallmark <br /> of <span id='am-48' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-49'  ></span><span id='am-50' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cancer</span> cells cause genome instability. The most <br /> widespread desensitizing mutation p53, common mutation in <br /> human cancer. <br />     We propose cell cycle checkpoint activation absence <br /> of Dna2 probably results persistant 5&apos; flaps created com- <br /> bined pol&#948;/Pif1 DNA strand displacement synthesis during <br /> Okazaki fragment processing (see Fig. 5 details). Supporting <br /> this mode checkpoint activation, Rad53 highly phosphory- <br /> lated dna2-1 strains, phosphorylation Mec1-dependent, <br /> while Rad53 remains largely unphosphorylated dna2-1 pif1&#916; <br /> strains.52 Furthermore, calculate mechanism could <br /> easily generate sufficient single-stranded DNA activate the <br /> DNA damage response. It estimated 104 <br /> bp single-stranded DNA required activate Rad53 an <br /> HO-induced DSB, based rate 5&apos; 3&apos; resection the <br /> time appearance phosphorylated Rad53.71,72 Yeast contains <br /> about 2.3 x 107 nt, therefore, 2 x 105 Okazaki frag- <br /> ments, 125 nt length, synthesized dur- <br /> ing S phase.73 If pol&#948;/Pif1 strand displacement causes 1% <br /> of flaps 30 nt longer, 2 x 103 flaps <br /> greater 30 nt long 1.2 x&#160;105 nt single-stranded DNA,            Figure 4. rad9&#916; mrc1AQ double mutation suppresses lethality of <br /> is expected generated S&#160;phase dna2&#916; mutant,              dna2&#916; better rad9&#916; alone. Tetrads dissection MB227 <br />                                                                            DNA2/dna2&#916;::natR MRC1/mrc1AQ::HIS3 RAD9/rad9&#916;::kanMX. The dna2&#916; <br /> more necessary activate checkpoint. After check- <br />                                                                            rad9&#916; colonies enclosed square, dna2&#916; rad9&#916; mrc1AQ <br /> point activation, cells normally repair DNA damage, deac-              colonies enclosed circle. <br /> tivate checkpoint resume cell division. dna2-1 strains <br /> repair DNA damage (long RPA bound 5&apos; flaps) inefficiently <br /> since Rad53 constitutively activated dna2-1 strains divide        RAD9 RAD53 inhibit processes dependent Exo1 <br /> very slowly, 4 hour generation time 23&#176;C. dna2&#916; strains             nucleolytic functions. Cdc13 essential telomere capping <br /> would unable repair unprocessed Okazaki fragments              protein. After Cdc13 inactivation, extensive 5&apos; 3&apos; degrada- <br /> during G2 division delay; Rad9/Mrc1-dependent G2            tion occurs telomere end, continuing far 15 kb from <br /> division delay contributes cell death recovery.           end.80,81 The degradation mainly Exo1-dependent.10 In the <br />     In addition failing induce cell cycle arrest,          absence Rad9, 5&apos; 3&apos; degradation proceeds significantly fur- <br /> mechanism probably involved suppression            ther faster, suggesting Rad9/Rad53 inhibits Exo1.82,83 Other <br /> dna2&#916; lethality rad9 rad9 mrc1AQ. Dna2 recently               evidence Exo1 inhibited Rad53-dependent step the <br /> been shown involved resection DSBs yeast.9,74-76          observation rad53 mutants, replication forks stalled by <br /> Although DSB break repair function Dna2 missing,           MMS HU exhibit long regions single-stranded DNA on <br /> viability dna2&#916; rad9-320 suggests absence,        nascent lagging strand, appearance single-stranded <br /> in presence, Rad9, lethal endogenous DSBs channeled           DNA dependent Exo1. In exo1&#916; rad53 mutants, no <br /> into alternative repair pathway. The Exo1-dependent DSB               single-stranded DNA, exo1&#916; suppresses MMS sensitivity <br /> repair pathway good candidate, given previously documented          rad53 mutant.84 In RAD53 cells, stalled forks remain fully <br /> genetic interactions overlaps function Dna2           duplex collapse, explained Exo1 is <br /> Exo1.52 Exo1 5&apos; 3&apos; nuclease homology Rad27scFEN1         inhibited.85 In rad9&#916; rad9 mrc1 mutants, Rad53 deac- <br /> and involved mismatch base repair, resection ends        tivated, putative Rad53-dependent inhibition Exo1 is <br /> DSBs decapped telomeres function OFP                 relieved. In addition, Rad9 H3K79 histone methyltrans- <br /> either Rad27&#8202;scFEN1 Dna2 defective.9,10,77-79 Overproduction       ferase Dot1 inhibit 5&apos; 3&apos; degradation DSBs telomeres.86 <br /> of Exo1 partially suppresses growth defect dna2-1 cells           A rad9&#916; mutant relieve inhibition Exo1, <br /> at 30&#176;C suppresses growth defect rad27&#916; mutants               rad9&#916; mrc1AQ relieve remainder inhibition, <br /> at 37&#176;C.50,78 dna2-1, dna2-2, dna2&#916; pif1&#916; rad27&#916; mutants             allowing Exo1 function OFP/DSB repair backup repair <br /> are synthetically lethal exo1&#916; mutants.50,52,78 In processing       pathway absence Dna2 (Fig. 5). <br /> of DSBs, Exo1 Dna2 provide resection func-                 The question remains rad27&#916; mutants viable and <br /> tion.9,74-76 Several recent lines evidence suggest activation    dna2&#916; mutants inviable. We previously proposed <br />  <br />  <br /> 1694	                                                            Cell Cycle	                                                  Volume 10 Issue 10 <br />   Figure 5. Checkpoint activation replication errors absence Dna2. In vitro, DNA strand displacement synthesis pol &#948; presence <br />  Rad27scFEN1 protein primer adjacent upstream fragment results appearance cleaved flaps. Although cleaved flaps are <br />  short, 8&#8211;12 nt, small subset 30 nt.45,89 Pol &#948; strand displacement stimulated Pif1, significant increase 30 bp cleaved product <br />  observed presence Pif1, 30 nt cleaved product 1&#8211;2% total cleaved products.90,91 The 30 nt flap synthesized by <br />  Pol&#948;/Pif1 refractory cleavage Rad27scFEN1 presence RPA, result lengthening flap.91,92 As shown <br />  (A), normal cells containing Dna2, RPA bound flap bound Dna2, cleaves long flap, releasing RPA allowing Rad27scFEN1 to <br />  cleave flap, resulting ligatable nick.43-45,91,93 In absence Dna2, (B), RPA-bound flap, however, persists bind Mec1/&#8202;Ddc2 <br />  kinase. The 5&apos; flap-bound Mec1/Ddc2 recruits phosphorylates Rad9, binds methylated histone H3-K79 Tudor domain. <br />  Mec1/&#8202;Ddc2 phosphorylate histone H2A, providing additional site Rad9 binding chromatin. Chromatin bound Rad9 recruits <br />  Rad53, resulting dimerization, autophosphorylation Rad53 kinase activation, allowing Rad53 activate DNA damage response cell <br />  cycle arrest. (Similarly, MCM helicase pol &#949; target Mrc1 Mec1-bound 5&apos; flaps). After checkpoint activation, cells normally repair DNA <br />  damage, deactivate checkpoint resume cell division. dna2&#916; strains unable repair unprocessed Okazaki fragments the <br />  G2 division delay; Rad9/Mrc1-dependent G2 division delay contributes cell death recovery. <br />  <br />  <br />  <br /> that synthetic lethality rad27&#916; recombinational                      remaining OFs inefficiently processed Dna2, Exo1 and <br /> repair mutants viability dna2 mutants combina-                     RnaseH. The synthetic lethalities suggest Rad27ScFEN1 is <br /> tion recombinational repair mutants suggests                       required DSB repair dedicated DNA replication, <br /> that DSBs occur frequently rad27&#916;, that, there-                      rad27&#916; mutant requires use G2 division delay to <br /> fore, Rad27ScFEN1 processes Okazaki fragments. The                   repair damage, allowing survival (albeit high frequency <br />  <br />  <br /> www.landesbioscience.com	                                                Cell Cycle	                                                                      1695 <br />   Table 1. Strains <br />         Y238                                                     MATa pep4-3 trp4-580 ura3-52 leu2-3,112 <br />     BY4742                                                          MAT&#945; his3&#916;1 leu2&#916;0 lys2&#916;0 ura3&#916;0 <br />     BY4741                                                          MATa his3&#916;1 leu2&#916;0 lys2&#916;0 ura3&#916;0 <br />     MB220                                              MATa dna2&#916;::kanMX rad9-320 his3&#916;1 leu2&#916;0 met15&#916;0 ura3&#916;0 <br />     MB221                                                                       4742/MB220 <br />     MB222                                         MATa dna2&#916;::natR his3&#916;1 leu2&#916;0 met15&#916;0 ura3&#916;0 trp1&#916; (pSEY18-DNA2) <br />    580-10D                                               MATa trp1-1 ura3-1 his3-3-11,15 leu2-3,112 ade2-1 can1-100 <br />     MB223                                  MATa rad9&#916;::kanMX mrc1AQ::HIS3 trp1-1 ura3-1 his3-3-11,15 leu2-3,112 ade2-1 can1-100 <br />     MB224                                     MATa dna2-1 rad9&#916;::kanMX trp1-1 ura3-1 his3-3-11,15 leu2-3,112 ade2-1 can1-100 <br />     MB225                                        MATa dna2-1 mrc1AQ trp1-1 ura3-1 his3-3-11,15 leu2-3,112 ade2-1 can1-100 <br />     MB226                                  MAT&#945; dna2-1rad9&#916;::kanMX mrc1AQ trp1-1 ura3-1 his3-11,15 leu2-3,112 ade2-1 can1-100 <br />                     MATa/MAT&#945; DNA2/dna2&#916;::natR MRC1/ mrc1AQ::HIS3 RAD9/ rad9&#8710;::kanMX his3-11,15/his3&#8710;1 leu2-3,112/leu2&#916;0 ura3-1/ura2&#916;0 trp1-1/trp1&#8710; <br />     MB227 <br />                                                           ADE2/ade2-1 CAN/can1-100 MET15/met15&#916;0 <br />         Y438                                        MATa rad9&#916;::HIS3 trp1-1 ura3-1 his3-11,15 leu2-3,112 ade2-1 can1-100 <br />     MB228                                  MATa DNA2::9MYC::LEU2 rad9&#916;::HIS3 trp1-1 ura3-1 his3-11,15 leu2-3,112 ade2-1 can1-100 <br />  <br />  <br /> of mutation). Hence, rad27 rad9 lethal.48,51,87 With dna2&#916;,                 GFF format (downloaded SGD ftp://ftp.yeastgenome. <br /> instead, fewer OFP failures fewer rep-               org/yeast/data_download/chromosomal_feature/saccharomy- <br /> lication-induced DSBs, G2 division delay lethal Dna2                ces_cerevisiae.gff) loaded MySQL database, the <br /> is required efficient repair. Potential backup repair pathways             output TruePoly module VAAL containing differ- <br /> are suppressed presence RAD9, deletion results               ences MB220 BJ4741 compared it. For <br /> in dna2 survival.                                                              polymorphism, recorded MB220, BJ4741 reference <br />                                                                                genome sequences flanking mutation, overlapping gene(s), <br />                      Materials Methods                                     gene description, associated GO terms available func- <br />                                                                                tional annotations, nature mutation (i.e., stop codon, non- <br /> Strains plasmids. The strains listed Table 1. The plas-             synonymous substitution, synonymous substitution) amino <br /> mid BG1805-GAL1-Rad9 (2 &#956; ori, URA3) obtained                       acid change produced it. Our analysis88 identified 476 SNPs <br /> strain Y258 Open Biosystems. The plasmid BG1805&#8209;Rad9                      21 small indels. 347 did fall genes <br /> was purified Y258, transformed Eschericia coli              (115) resulted nucleotide substitutions did affect <br /> purified. Standard genetic techniques used tetrad analy-              amino acid sequence (232). One forty-five muta- <br /> sis genetic analysis. The chromosomal DNA MB220                       tions produced single amino acid change, deleted and <br /> was isolated using Qiagen Genomic-tip P/20. The kanMX                      amino acids (without frame change), produced stop <br /> disruption cassette confers resistance G418/geneticin,                  codons. The complete list identified polymorphisms be <br /> natR cassette confers resistance Nourseothricin.                            Supplemental Table 1 (MB220_vs._BJ4741_polymor- <br />     DNA sequencing mutation detection. Libraries pre-                 phisms_all). Mutations producing stop codons, genomic <br /> pared genomic DNA isolated BJ4741 deriva-                    coordinates affected genes are: polymorphism 19, chr4 <br /> tive strain MB220 carrying dna2&#916; suppressor, following                 902499, RAD9 (DNA damage-dependent checkpoint protein); <br /> standard Illumina protocol. The libraries sequenced                polymorphism 432, chr13 74415, YML099W-A (dubious open <br /> Illumina Genome Analyzer II using single read sequencing pro-                reading frame unlikely encode protein); polymorphism 471, <br /> tocol producing 15.4 20.2 million 64 nt reads passing qual-                chr15 286230, TAT2 (high-affinity tryptophan tyrosine <br /> ity filtering, respectively. The sequence output corresponds                permease). <br /> 80.2x coverage S. cerevisiae genome BJ4741 105.2x <br /> coverage MB220. To identify polymorphisms MB220,                                                    Acknowledgments <br /> each strain compared reference S. cerevisiae genome                 This work supported Public Health Service, NIGMS <br /> (version S288C, downloaded <span id='am-24' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-29'  ></span><span id='am-30' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span>Saccharomyces</span> Genome                       078666 Army Research Office (ARO) 09-1-0041. <br /> Database, www.yeastgenome.org) using the <br /> VAAL software package, detect single nucleotide                                                        Note <br /> polymorphisms (SNPs) deletion/insertion events (indels).88                 Supplemental materials at: <br /> To identify genes affected mutations, genome annotations                www.landesbioscience.com/journals/cc/article/15643 <br />  <br />  <br />  <br />  <br /> 1696	                                                                  Cell Cycle	                                                 Volume 10 Issue 10 <br />                                                                   21.	 Tercero JA, Diffley JF. Regulation DNA replication       40.	 Johnson RE, Gopala KK, Prakash L, Prakash S. <br />                        References                                     fork progression damaged DNA Mec1/                Requirement yeast RTH1 5&apos; 3&apos; exonuclease <br /> 1.	 Mallory JC, Petes TD. Protein kinase activity                  Rad53 checkpoint. Nature 2001; 412:553-7.                        stability simple repetitive DNA. Science <br />      Tel1p Mec1p, Saccharomyces cerevisiae proteins      22.	 Lopes M, Cotta-Ramusino C, Pellicioloi A, Liberi G,              1995; 269:238-40. <br />      related human ATM protein kinase. Proc Natl               Plevani P, Muzi-Falconi M, et al. The DNA replication       41.	 Chen C, Kolodner RD. Gross chromosomal rear- <br />      Acad Sci USA 2000; 97:13749-54.                                  checkpoint response stabilizes stalled replication forks.        rangements Saccharomyces cerevisiae replication and <br /> 2.	 Sanchez Y, Desany BA, Jones WJ, LiuQ, Wang B,                     Nature 2001; 412:557-61.                                         recombination defective mutants. Nat Genet 1999; <br />      Elledge SJ. Regulation RAD53 ATM-like             23.	 Lucca C, Vanoli F, Cotta-Ramusino C, Pellicioli A,               23:81-5. <br />      kinases MEC1 TEL1 yeast cell cycle checkpoint             Liberi G, Haber J, et al. Checkpoint-mediated control       42.	 Budd ME, Campbell JL. A yeast replicative helicase, <br />      pathways. Science 1996; 271:314-5.                               replisome-fork association signalling response         Dna2 helicase, interacts yeast FEN-1 nuclease in <br /> 3.	 Sanchez Y, Bachant J, Wang H, Hu F, Liu D, Tetzlaff               replication pausing. Oncogene 2004; 23:1206-13.               carrying essential function. Mol Cell Biol 1997; <br />      M, et al. Control DNA damage checkpoint              24.	 Aboussekhra AVJ, Morrison DE, la Torre-Ruiz                   17:2136-42. <br />      chk1 rad53 protein kinases distinct               MA, Cernakova L, Fabre F, Lowndes NF. A novel               43.	 Bae KH, Kim HS, Bae SH, Kang HY, Brill S, Seo YS. <br />      mechanisms. Science 1999; 286:1166-71.                           role budding yeast RAD9 checkpoint gene                  Bimodal interaction replication-protein A and <br /> 4.	 Zhou Z, Elledge SJ. DUNI encodes protein kinase                 DNA damage-dependent transcription. EMBO J                    Dna2 critical Dna2 function vivo in <br />      controls DNA damage response yeast. Cell             1996; 15:3912-22.                                                vitro. Nucl Acids Res 2003; 31:3006-15. <br />      1993; 75:1119-27.                                           25.	 Gambus A, Jones RC, Sanchez-Diaz A, Kanemaki M,             44.	 Bae SH, Bae KH, Kim JA, Seo YS. RPA governs <br /> 5.	 Herzberg K, Bashkirov VI, Rolfsmeier M, Haghnazari                van Deursen F, Edmondson RD, et al. GINS maintains               endonuclease switching processing Okazaki <br />      E, McDonald WH, Anderson S, et al. Phosphorylation               association Cdc45 MCM replisome progres-              fragments eukaryotes. Nature 2001; 412:456-61. <br />      Rad55 serines 2, 8 14 required efficient        sion complexes eukaryotic DNA replication forks.         45.	 Ayyagari R, Gomes XV, Gordenin DA, Burgers PMJ. <br />      homologous recombination recovery stalled              Nat Cell Biol 2006; 8:358-66.                                    Okazaki fragment maturation yeast. I. Distribution <br />      replication forks. Mol Cell Biol 2006; 26:8396-409.         26.	 Lou H, Komata M, Katou Y, Guan Z, Reis CC, Budd                  functions Fen1 Dna2. J Biol Chem <br /> 6.	 Santocanale C, Diffley JF. A Mec1- Rad53-                     M, et al. Mrc1 DNA polymerase epsilon function               2003; 278:1618-25. <br />      dependent checkpoint controls late-firing origins             linking DNA replication S phase         46.	 Kao HI, Veeraraghavan J, Polaczek P, Campbell JL, <br />      DNA replication. Nature 1998; 395:615-8.                         checkpoint. Mol Cell 2008; 32:106-17.                            Bambara RA. On roles Saccharomyces cerevisiae <br /> 7.	 Morin I, Ngo HP, Greenall A, Zubko M, Morrice N,             27.	 Katou Y, Kanoh Y, Bando M, Noguchi H, Tanaka H,                  Dna2p FEN1 Okazaki fragment processing. J <br />      Lydall D. Checkpoint-dependent phosphorylation                Ashikari T, et al. S-phase checkpoint proteins Tof1              Biol Chem 2004; 279:15014-24. <br />      Exo1 modulates DNA damage response. EMBO                     Mrc1 form stable replication-pausing complex.         47.	 Tishkoff DX, Filosi N, Gaida GM, Kolodner RD. A <br />      Journal 2008; 27:2400-10.                                        Nature 2003; 424:1078-83.                                        novel mutation avoidance mechanism dependent on <br /> 8.	 Genschel J, Bazemore LR, Modrich P. Human exo-               28.	 Szyjka SJ, Viggiani CJ, Aparicio OM. Mrc1 required            Saccharomyces cerevisiae RAD27 distinct DNA <br />      nuclease I required 5&apos; 3&apos; mismatch repair. J          normal progression replication forks           mismatch repair. Cell 1997; 88:253-63. <br />      Biol Chem 2002; 277:13302-11.                                    chromatin S. cerevisiae. Mol Cell 2005; 19:691-7.        48.	 Tong AHY, Evangelista M, Parsons AB, Xu H, Bader <br /> 9.	 Mimitou EP, Symington LS. Sae2, Exo1 Sgs1                29.	 Naylor MLLJ, Osborn AJ, Elledge SJ. Mrc1 phos-                   GD, Page N, et al. Systematic genetic analysis with <br />      collaborate DNA double-strand break processing.               phorylation response DNA replication stress             ordered arrays yeast deletion mutants. Science 2001; <br />      Nature 2008; 455:770-4.                                          required Mec1 accumulation stalled fork.              294:2364-8. <br /> 10.	 Maringele L, Lydall D. EXO1-dependent single-strand-             PNAS 2009; 106:12765-70.                                    49.	 Tong AHY, Lesage G, Bader GD, Ding H, Xu H, Xin <br />      ed DNA telomeres activates subsets DNA damage         30.	 Osborn AJ, Elledge SJ. Mrc1 replication fork com-           X, et al. Global mapping yeast genetic interaction <br />      spindle checkpoint pathways budding yeast                 ponent phosphorylation response DNA                  network. Science 2004; 303:808-13. <br />      yku70Delta mutants. Genes Dev 2002; 16:1919-33.                  replication stress activates Rad53. Genes Dev 2003;         50.	 Budd ME, Tong AH, Polaczek P, Peng X, Boone C, <br /> 11.	 Bertuch <span id='am-68' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-72'  ></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>AA</span>, Lundblad V. EXO1 contributes telo-                17:1755-67.                                                      Campbell JL. A Network Multi-Tasking Proteins at <br />      mere maintenance telomerase-proficient          31.	 Budd ME, Campbell JL. Identification new yeast              DNA Replication Fork Preserves Genome Stability. <br />      telomerase-deficient Saccharomyces cerevisiae. Genetics          DNA polymerase DNA polymerase &#946; <span id='am-45' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-46' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span>                    PLoS Genet 2005; 1:634-50. <br />      2004; 166:1651-9.                                                evidence supporting role meiosis. J Biol Chem      51.	 Loeillet S, Palancade B, Cartron M, Thierry A, Fichard <br /> 12.	 Schurer KA, Rudolph C, Ulrich HD, Kramer W. Yeast                1995.                                                            GF, Dujon B, et al. Genetic network interactions <br />      MPH1 gene functions error-free DNA damage             32.	 Budd ME, Choe WC, Campbell JL. The nuclease                      replication repair nuclear pore deficiencies <br />      bypass pathway requires genes homologous               activity yeast Dna2 protein, related             yeast. DNA Repair 2005; 4:459-68. <br />      recombination, postreplicative repair.              RecB-like nucleases, essential vivo. J Biol    52.	 Budd ME, Reis CC, Smith S, Myung K, Campbell <br />      Genetics 2004; 166:1673-86.                                      Chem 2000; 275:16518-29.                                         JL. Evidence suggesting Pif1 helicase functions in <br /> 13.	 Alcasabas AA, Osborn AJ, Bachant J, Hu F, Werler            33.	 Bae SH, Choi E, Lee K, Park J, Lee S, Seo Y. Dna2                DNA replication Dna2 helicase/nuclease and <br />      PJH, Boussett K, et al. Mrc1 transduces signals               Saccharomyces cerevisiae possesses single-stranded          DNA polymerase <span id='am-89' about='obo:NCBITaxon_998453' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_50638 obo:NCBITaxon_7438 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7434 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_34725 obo:NCBITaxon_50557'><span id='am-90'  ></span><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Delta" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>delta</span>. Mol Cell Biol 2006; 26:2490- <br />      DNA replication stress activiate Rad53. Nat Cell              DNA-specific endonuclease activity able act           500. <br />      Biol 2001; 3:958-65.                                             double-stranded DNA presence ATP. J Biol       53.	 Blankley RTLD. A domain Rad9 specifically <br /> 14.	 Schwartz MF, Duong JK, Sun Z, Morrow JS, Pradhan                 Chem 1998; 273:26880-90.                                         required activation Chk1 budding yeast. J Cell <br />      D, Stern DF. Rad9 phosphorylation sites couple Rad53        34.	 Bae SH, Seo YS. Characterization enzymatic                Sci 2004; 117:601-8. <br />      Saccharomyces cerevisiae DNA damage check-                properties yeast Dna2 helicase/endonuclease          54.	 Lancelot NCG, Couprie J, Duband-Goulet I, Alpha- <br />      point. Mol Cell 2002; 9:1055-65.                                 suggests new model Okazaki fragment processing.            Bazin B, Quemeneur E, Ma E, et al. The checkpoint <br /> 15.	 Vialard JE, Gilbert CS, Green CM, Lowndes NF. The                J Biol Chem 2000; 275:38022-31.                                  Saccharomyces cerevisiae Rad9 protein contains tan- <br />      budding yeast Rad9 checkpoint protein subjected          35.	 Budd ME, Choe WC, Campbell JL. DNA2 encodes                      dem tufor domain recognizes DNA. Nucleic Acids <br />      Mec1/Tel1-dependent hyperphosphorylation                  DNA helicase essential replication eukaryotic           Res 2007; 35:5898-912. <br />      interacts Rad53 DNA damage. EMBO J                    chromosomes. J Biol Chem 1995; 270:26766-9.                 55.	 Soulier JLN. The BRCT domain S. cerevisiae <br />      1998; 17:5679-88.                                           36.	 Masuda-<span id='am-74' about='obo:NCBITaxon_66037' typeof='owl:Thing obo:NCBITaxon_147366 obo:NCBITaxon_78536 obo:NCBITaxon_359160 obo:NCBITaxon_1 obo:NCBITaxon_4479 obo:NCBITaxon_131567 obo:NCBITaxon_1149709 obo:NCBITaxon_4447 obo:NCBITaxon_4734 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_38820 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-75' property="rdfs:label" content="Sasa" datatype="xsd:string"></span><span id='am-76'  ></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Sasa</span> T, Polaczek P, Campbell JL. Single strand            checkpoint protein Rad9 mediates Rad9-Rad9 inter- <br /> 16.	 Weinert TA, Hartwell TA. The RAD9 gene controls              annealing ATP-independent strand exchange                    action DNA damage. Curr Biol 1999; 9:551-4. <br />      cell cycle response DNA damage Saccharomyces               activities yeast human DNA2: possible role           56.	 Li JWSVW, Schultz LD, George HA, Herber WK, <br />      cerevisiae. Science 1988; 241:317-22.                            Okazaki fragment maturation. J Biol Chem 2006;                Chae HJ, Bentley WE, Rao G. Green Fluorescent <br /> 17.	 Sun ZHJ, Fay DS, Stern DF. Rad53 FHA domain asso-                281:38555-64.                                                    Protein Saccharomyces cerevisiae: real-time studies of <br />      ciated phosporylated Rad9 DNA damage            37.	 Kuo C, Nuang H, Campbell JL. Isolation yeast                  GAL1 Promoter. Biotechnol Bioeng 2000; 70:187- <br />      checkpoint. Science 1998; 281:272-4.                             DNA replication mutants permeabilized cells. Proc             96. <br /> 18.	 Gilbert CS, Green CM, Lowndes NF. Budding yeast                  Natl Acad Sci USA 1983; 80:6465-9.                          57.	 Budd ME, Campbell JL. A yeast replicative helicase, <br />      Rad9 ATP-dependent Rad53 activating machine.          38.	 Budd ME, Campbell JL. A yeast gene required                  Dna2 helicase, interacts yeast FEN-1 nuclease in <br />      Mol Cell 2001; 8:129-36.                                         DNA replication encodes protein homol-                    carrying essential function. Mol Cell Biol 1997; <br /> 19.	 Surana U, Amon A, Dowzer C, McGrew J, Byers B,                   ogy DNA helicases. Proc Natl Acad Sci USA 1995;               17:2136-42. <br />      Nasmyth K. Destruction CDC28/CLB mitotic                  92:7642-6.                                                  58.	 Imamura O, Campbell JL. The human Bloom syn- <br />      kinase required metaphase anaphase        39.	 Waga S, Bauer G, Stillman B. Reconstitution com-              drome gene suppresses DNA replication repair <br />      transition budding yeast. EMBO J 1993; 12:1969-78.            plete SV40 DNA replication purified replication             defects yeast dna2 mutants. Proc Natl Acad Sci USA <br /> 20.	 Agarwal RTZ, Yu H, Cohen-Fix O. Two distinct path-               factors. J Biol Chem 1994; 269:10923-34.                         2003; 100:8193-8. <br />      ways inhibiting pds1 ubiquitination response to <br />      DNA damage. J Biol Chem 2003; 278:45027-33. <br />  <br />  <br />  <br />  <br /> www.landesbioscience.com	                                                               Cell Cycle	                                                                                      1697 <br />  59.	 Sharma S, Sommers JA, Wu L, Bohr VA, Hickson               72.	 Vaze MB, Pellicioli A, Lee SE, <span id='am-53' about='obo:NCBITaxon_717364' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_82596 obo:NCBITaxon_82593 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_82592 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-55'  ></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Ira" datatype="xsd:string"></span>Ira</span> G, Liberi G, Arbel-       84.	 Segurado M, Diffley JF. Separate roles DNA <br />      ID, Brosh RM Jr. Stimulation flap endonuclease-1             Eden A, et al. Recovery checkpoint-mediated                  damage checkpoint protein kinases stabilizing DNA <br />      Bloom&#8217;s syndrome protein. J Biol Chem 2004;              arrest repair double-strand break requires Srs2        replication forks. Genes Dev 2008; 22:1816-27. <br />      279:9847-56.                                                    helicase. Mol Cell 2002; 10:373-85.                          85.	 Cotta-Ramusino C, Fachinetti D, Lucca C, Doksani Y, <br /> 60.	 Budd ME, Tong AH, Polaczek P, Peng X, Boone C,             73.	 Bielinsky AK, Gerbi SA. Chromosomal ARS1                    Lopes M, Sogo J, et al. Exo1 processes stalled replica- <br />      Campbell JL. A network multi-tasking proteins             single leading strand start site. Mol Cell 1999; 3:477-86.        tion forks counteracts fork reversal checkpoint- <br />      DNA replication fork preserves genome stability.       74.	 Zhu Z, Chung WH, Shi EY, Lee SE, Ira G. Sgs1                      defective cells. Mol Cell 2005; 17:153-9. <br />      PLoS Genet 2005; 1:61.                                          helicase nucleases Dna2 Exo1 resect DNA          86.	 Lazzaro FSV, <span id='am-84' about='obo:NCBITaxon_479585' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_479584 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216281 obo:NCBITaxon_6072 obo:NCBITaxon_216275'><span id='am-85' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Granata" datatype="xsd:string"></span><span id='am-88'  ></span>Granata</span> M, Pellicioli A, Vaze M, Haber <br /> 61.	 Kang YH, Kang MJ, Kim JH, Lee CH, Cho IT,                       double-Strand break ends. Cell 2008; 134:981-94.                  JE, Plevani P, et al. Histone metyltransferase Dot1 <br />      Hurwitz J, et al. The MPH1 gene Saccharomyces           75.	 Budd ME, Cox Ls, Campbell JL. Coordination                     Rad9 inhibit single-stranded DNA accumulation <br />      cerevisiae functions Okazaki fragment processing. J          Nucleases Helicases DNA Replication                DSBs uncapped telomeres. EMBO J 2008; <br />      Biol Chem 2009; 284:10376-86.                                   Double-strand Break Repair. In: Cox LS, ed. Molecular             27:1502-12. <br /> 62.	 Kang JY, Choi E, Bae SH, Lee KH, Gim BS, Kim                    Themes DNA Replication. London: RSC Publishing            87.	 Garvik B, Carson M, Hartwell L. Single-stranded <br />      HD, et al. Genetic analyses Schizosaccharomyces              2009.                                                             DNA arising telomeres cdc13 mutants may <br />      pombe dna2+ reveal Dna2 plays essential role       76.	 Cejka P, Cannavo E, Polaczek P, Masuda-Sasa T,                    constitute specific signal RAD9 checkpoint <br />      Okazaki fragment metabolism. Genetics 2000;                  Pokharel S, Campbell JL, et al. DNA end resection              [published erratum appears Mol Cell Biol 1996; <br />      155:1055-67.                                                    Dna2-Sgs1-RPA stimulation Top3-Rmi1                16:457]. Mol Cell Biol 1995; 15:6128-38. <br /> 63.	 Boule JB, Vega LR, Zakian VA. The yeast Pif1p heli-             Mre11-Rad50-Xrs2. Nature 2010; 467:112-6.                    88.	 Nusbaum C, Ohsumi TK, Gomez J, Aquadro J, Victor <br />      case removes telomerase telomeric DNA. Nature         77.	 Fiorentini P, Huang KN, Tishkoff DX, Kolodner RD,                 TC, Warren RM, et al. Sensitive, specific polymor- <br />      2005; 438:57-61.                                                Symington LS. Exonuclease I Sacchromyces cerevisiae            phism discovery bacteria using massively parallel <br /> 64.	 Boule JB. The yeast Pif1 DNA helicase preferentially            functions mitotic recombination vivo vitro.          sequencing. Nat Methods 2009; 6:67-9. <br />      unwinds RNA DNA substrates. Nucleic Acids Res                   Mol Cell Biol 1997; 17:2764-73.                              89.	 Rossi ML, Bambara RA. Reconstituted Okazaki frag- <br />      2007; 35:5809-18.                                          78.	 Tishkoff DX, Boerger AL, Bertrand P, Filosi N, Gaida              ment processing indicates pathways primer <br /> 65.	 Johansson E, Garg P, Burgers PMJ. The Pol32 subunit             GM, Kane MF, et al. Identification characteriza-              removal. J Biol Chem 2006; 281:26051-61. <br />      DNA polymerase &#948; contains separable domains                  tion Saccharomyces cerevisiae EXO1, gene encoding       90.	 Rossi ML, Pike JE, Wang W, Burgers PM, Campbell <br />      processive replication proliferating cell nucle-        exonuclease interacts MSH2. Proc Natl                JL, Bambara R. Pif1 helicase directs eukaryotic Okazaki <br />      ar antigen (PCNA) binding. J Biol Chem 2004;                    Acad Sci USA 1997; 94:7487-92.                                    fragments two-nuclease cleavage pathway <br />      279:1907-15.                                               79.	 Bertuch AA, Lundblad V. EXO1 contributes telo-                 primer removal. J Biol Chem 2008; 283:27483-93. <br /> 66.	 Zieler HA WM, Berg P. Suppression Mutations               mere maintenance telomerase-proficient           91.	 Pike JE, Burgers PM, Campbell JL, Bambara RA. <br />      Two Saccharomyces cerevisiae Genes Adenovirus            telomerase-deficient Saccharomyces cerevisiae. G <br /> </body></html>