<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Crystal Structure Small GTPase <br /> Arl6/BBS3 <span id='am-9' about='obo:NCBITaxon_5690' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_2759 obo:NCBITaxon_5653'><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11'  ></span><span id='am-12' property="rdfs:label" content="Trypanosoma" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Trypanosoma</span> brucei <br />  <br />  <br /> Glyn R. Hemsworth,1 Helen P. Price,2 Deborah F. Smith,2 Keith S. Wilson1* <br /> 1 <br /> Structural Biology Laboratory, Department Chemistry, University York, Heslington, York, United <span id='am-52' about='obo:NCBITaxon_kingdom' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-53' property="rdfs:label" content="kingdom" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Kingdom</span> <br /> 2 <br /> Centre Immunology Infection, Department Biology, University York, Heslington, York, United Kingdom <br />  <br />  <br /> Received 24 September 2012; Revised 12 November 2012; Accepted 12 November 2012 <br /> DOI: 10.1002/pro.2198 <br /> Published online 26 November 2012 proteinscience.org <br />  <br />  <br />  <br /> Abstract: Arl6/BBS3 small GTPase, mutations implicated human ciliopathy <br /> Bardet&#8211;Biedl Syndrome (BBS). Arl6 proposed facilitate recruitment large protein <br /> complex known BBSome base primary cilium, mediating specific trafficking of <br /> molecules important sensory organelle. Orthologues Arl6 BBSome core subunits <br /> have identified genomes trypanosomes. Flagellum function motility crucial <br /> to survival <span id='am-14' about='obo:NCBITaxon_5691' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_39700 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_5690 obo:NCBITaxon_2759 obo:NCBITaxon_5653'><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Trypanosoma (Trypanozoon) brucei" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Trypanosoma brucei" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18'  ></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Trypanosoma brucei subgroup" datatype="xsd:string"></span>Trypanosoma brucei</span>, causative agent human African sleeping sickness, in <br /> the human bloodstream stage lifecycle function BBSome proteins in <br /> trypanosomes warrants study. RNAi knockdown T. brucei Arl6 (TbArl6) recently been <br /> shown result shortening trypanosome flagellum. Here present crystal structure <br /> of TbArl6 bound non-hydrolysable GTP analog GppNp 2.0 A&#730; resolution highlight <br /> important differences trypanosomal human proteins. Analysis TbArl6 active <br /> site confirms lacks key glutamine activates nucleophile GTP hydrolysis in <br /> other small GTPases. Furthermore, trypanosomal proteins significantly shorter their <br /> N-termini suggesting different method membrane insertion compared humans. Finally, <br /> analysis sequence conservation suggests surface patches important for <br /> protein&#8211;protein interactions. Our structural analysis provides basis future biochemical <br /> characterisation important <span id='am-6' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-7' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> small GTPases. <br /> Keywords: Trypanosoma brucei; Arl6; BBS3; BBSome; flagellum <br />  <br />  <br />  <br /> Introduction                                                    flagellum crucial survival tsetse fly <br /> The protozoan parasite Trypanosoma brucei causes                vector provides motility enable parasite to <br /> human African sleeping sickness 10,000 new                 migrate midgut salivary glands where <br /> cases annually resulting massive morbidity               flagellum used attach parasite epi- <br /> mortality Africa.1 In host bloodstream, T.           thelium.6&#8211;8 These key roles led proposal <br /> brucei flagellum required cell division2&#8211;4           perturbation flagellar function viable <br /> for rapid variant surface glycoprotein recycling,             avenue development new drug treatments.9 <br /> critical immune evasion strategy parasite sur-                   Bardet&#8211;Biedl Syndrome (BBS) human <br /> vival.5 To avoid opsonisation human antibodies,              genetic disorder symptoms include retinal <br /> the trypanosome flagellum beats direct surface               degeneration, obesity, renal abnormalities, mental <br /> bound immune complexes flagellar                     retardation, polydactyly, hypogenitalism.10 <br /> pocket rapidly endocytosed, hiding               Mutations 16 genes, ciliates and <br /> the parasite human immune system. The                  flagellates, shown result BBS, in <br />                                                                 observed defects caused dys- <br />                                                                 function primary cilium, non-motile thin <br /> *Correspondence to: Keith S. Wilson, Structural Biology         protuberance cell types acts as <br /> Laboratory, Department Chemistry, University York, York   signalling sensory organelle.11,12 Seven of <br /> YO10 5DD, UK. E-mail: keith.wilson@york.ac.uk                   genes (BBS1, 2, 4, 5, 7, 8, 9) encode <br />  <br />  <br /> 196   PROTEIN SCIENCE 2013 VOL 22:196&#8212;203                                    Published Wiley-Blackwell. V <br />                                                                                                            C 2012 The Protein Society <br />  Figure 1. (A) The structure TbArl6 coloured secondary structure. The Switch I, Interswitch Switch II regions are <br /> coloured magenta, red blue respectively. The bound nucleotide analog shown ball stick representation coloured <br /> by atom type Mg2&#254; ion orange sphere. (B) Human Arl6 (pink) superposed TbArl6 (green). The b-bulge <br /> present TbArl6 largest difference protein structures. The structures Figures 1&#8211;4 produced <br /> with CCP4mg.58 <br />  <br />  <br /> proteins form large complex known                 IFT.27 In T. brucei, knockdown Arl6 expression by <br /> BBSome, protein unknown func-                RNAi resulted significant shortening flagel- <br /> tion, BBIP10.13,14 The BBSome required traf-             lum loss motility.28 Genetic <br /> ficking proteins primary cilium              deletion Arl6 achieved, suggesting that <br /> shown BBS2 BBS4 knockout mice,                     essential parasite viability. Furthermore, Price <br /> have defects trafficking specific G-protein           et al. 28 showed protein N-myristoylated, <br /> coupled receptors.15,16 Recent <span id='am-61' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-62'  ></span><span id='am-63' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> Caeno-               interacts tubulin co-localises flagellar <br /> <span id='am-21' about='obo:NCBITaxon_6244' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55887 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-22' property="rdfs:label" content="Rhabditis" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-25'  ></span>rhabditis</span> elegans model suggests BBSome                pocket BBS1 protein overexpressed, <br /> interacts directly intraflagellar transport (IFT) <br />                                                                 suggesting functional association T. brucei <br /> components regulates IFT particle assembly.17 <br />                                                                 Arl6 BBSome humans. <br /> How BBSome does remains uncertain, the <br />                                                                       Here present structure T. brucei Arl6 <br /> current proposal forms coat complex <br /> for generation vesicles transport proteins         (TbArl6) step structure function <br /> base cilium.18 Examination kinetoplastid              studies BBSome parasites which <br /> genomes, including Trypanosoma,                provide excellent models study cilium and <br /> revealed presence conserved orthologues               flagellum function eukaryotic cells. <br /> the BBSome subunits listed above. Given conse- <br /> quences BBSome mutations humans,                Results <br /> the flagellum plays crucial role sur- <br /> vival T. brucei host, analysing struc-        Structure TbArl6 <br /> ture function BBSome para-                The 2.0 A &#730; resolution structure shows typical small <br /> sites fundamental importance               GTPase single subunit asymmetric <br /> elucidate roles complex.                            unit [Fig. 1(a) Table I], consisting central <br />      BBS3/Arl6 member ADP-ribosylation              6-stranded b-sheet (residues 7&#8211;13, 43&#8211;50, 53&#8211;60, 79&#8211; <br /> factor-like (Arl) family small GTPases. Members              85, 120&#8211;125, 153&#8211;159) surrounded a-heli- <br /> of Arf Arl families widely involved              ces (residues 20&#8211;27, 92&#8211;103, 106&#8211;109, 135&#8211;142, 144&#8211; <br /> regulating vesicle trafficking,19 microtubule dynam-            148, 166&#8211;180). The similarity GTPases <br /> ics,20 endosome-lysosome fusion,21 ciliogenesis.22          allows <span id='am-60' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> switch I (residues 29&#8211; <br /> They achieve various functions interacting             43), switch II (residues 59&#8211;72), interswitch <br /> with different protein partners, dependent                 regions (residues 44&#8211;58) (Fig. 1), 310 <span id='am-26' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-27'  ></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>helix</span> <br /> their nucleotide bound state. While Arl6               forming switch II region. There no <br /> member core BBSome complex, mutations                 <span id='am-42' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-43'  ></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>electron</span> density residues (32&#8211;35) the <br /> the gene cause BBS humans.23,24 These muta-              switch I region, indicating conformation flex- <br /> tions result C-terminal truncations            ible. The central b-sheet consists largely parallel <br /> protein changes highly conserved residues             strands apart antiparal- <br /> required interaction guanine nucleo-               lel forms interswitch region, seen <br /> tide.25 Arl6 GTP bound state shown              members Arf Arl family.29 Five <br /> to assist recruitment BBSome               (15 including His-tag) residues are <br /> base cilium.18 Furthermore mouse model,             missing N- C-termini respectively, pre- <br /> Arl6 deletion resulted BBS-like symptoms to-                 sumed reflect intrinsic flexibility. <br /> gether Arl6-specific phenotypes26 whilst Cae-                There clear electron density GppNp <br /> norhabditis elegans Arl6 shown undergo                   non-hydrolysable substrate analog cleft the <br />  <br />  <br /> Hemsworth et al.                                                                         PROTEIN SCIENCE VOL 22:196&#8212;203   197 <br />  Table I. Data Processing Refinement Statistics               ligand octahedral coordination sphere being <br /> Data processing                                                  completed water molecules. <br /> Space group                                    P3221 <br /> a, b, c (A &#730;)                             70.9, 70.9, 80.9       Crystal contacts <br /> a, b, c (&#730;)                              90.0, 90.0, 120.0 <br />                                                                  Analysis structure using PISA30 indicated <br /> Radiation source                           Diamond IO4 <br /> Wavelength (A  &#730;)                              0.980             TbArl6 form dimer crystallo- <br /> Resolution range (A &#730;)                61.42&#8211;2.00 (2.11&#8211;2.00)     graphic two-fold buried surface area 4460 <br /> No. observed reflections              168740 (26079)          A&#730; 2 [Fig. 3(a)]. The interface involves 26 residues <br /> No. unique reflections                  15861 (2373)          H-bonds salt bridges. Size Exclu- <br /> Completeness (%)                            96.7 (100.0) <br />                                                                  sion Chromatography Multi-Angle <span id='am-55' about='obo:NCBITaxon_62990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_241799 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_241789 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-56'  ></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Laser" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Laser</span> Light <br /> Multiplicity                                 10.6 (11.0) <br /> &lt;I/r(I)&gt;                                     15.0 (3.0)          Scattering (SEC-MALLS) used analyse <br /> Rmerge                                     0.119 (0.839)         TbArl6 forms dimer solution. Samples <br /> Rp.i.m.                                    0.038 (0.264)         pre-incubated 1 mM GppNp 1 mM GDP <br /> Refinement                                                       applied SEC column. In presence <br /> Resolution range (A &#730;)                61.42&#8211;2.00 (2.05&#8211;2.00) <br />                                                                  GppNp, major <span id='am-36' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> molecular mass of <br /> No. reflections                     15009/796 (1043/58) <br />    (working/free sets)                                           21.4 6 1.3 kDa [Fig. 3(b)], GDP 21.7 6 <br /> Number Atoms                                1439              1.5 kDa [Fig. 3(c)]. So, expected mass 21803 <br /> Final Rwork                                0.183 (0.229)         Da, protein monomer. A minor second peak <br /> Final Rfree                                0.248 (0.235)         detected samples molecular mass of <br /> rmsd bond lengths (A &#730;)                     0.010 <br />                                                                  53.1 6 2.7 kDa presence GppNp 64.3 6 <br /> rmsd bond angles (  )                        1.46 <br /> Overall average B factor (A &#730; 2)                31.2             3.9 kDa GDP. We attribute species be <br /> Ramachandran plot analysis                                       minor contaminant purification which <br /> Most favoured regions (%)                      99.4              visible SDS-PAGE. <br /> Additionally allowed regions (%)                0.6 <br /> Disallowed regions (%)                          0.0 <br />                                                                  Conserved regions TbArl6 <br /> Values outer shell given parentheses.              We constructed model look sequence conser- <br />                                                                  vation protein surface aim identi- <br />                                                                  fying possible surfaces protein partners <br /> surface protein (Fig. 2). <span id='am-31' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-32'  ></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> pocket highly           interact (Fig. 4). The residues disordered <br /> conserved small GTPases interac-                 switch I region ordered human <br /> tions protein ligand              homologue loop added model <br /> other structures. Fifteen direct hydrogen bonds              conformation. Missing chains were <br /> formed protein GppNp                    added likely rotamers missing <br /> specificity GTP provided prototypical (N/             N- C-termini omitted reli- <br /> T)KXD motif. The c-phosphate forms hydrogen                    able way predicting positions. <br /> bond main chain amide Gly62 switch                  A BLAST search used identify Arl6 <br /> II region, interaction important induc-            sequences species Uniprot were <br /> ing active conformation protein. A single             aligned TbArl6 using T-COFFEE.31 To <br /> Mg2&#254; ion coordinated Oc&#8217;s Thr21                 prevent conservation skewed GTPases <br /> Thr40 switch I region protein             functions, proteins annotated as <br /> oxygen atoms b c phosphates                  Arl6 included. The resulting alignment was <br />  <br />  <br />  <br />  <br /> Figure 2. Stereo diagram electron density active site cleft contoured 1r. The residues form cleft <br /> are shown sticks coloured atom type direct hydrogen bonds ligand shown black dashed lines. Residues <br /> which form switch I II regions magenta blue carbon atoms respectively. Ala63, residue, Gln <br /> in small GTPases important hydrolysis GTP, labeled. <br />  <br />  <br />  <br /> 198    PROTEINSCIENCE.ORG                                                                           Structure T. brucei Arl6/BBS3 <br />                                                                     surface formed terminal strands the <br />                                                                    central b-sheet contributed adjacent a- <br />                                                                    helices [Fig. 4(c,d)]. <br />  <br />                                                                    Discussion <br />                                                                    Overall structure <br />                                                                    Comparisons using secondary structure matching34 <br />                                                                    structure similar of <br />                                                                    human Arl6 (pdb code 2h57, RMSD 1.14 A      &#730; over <br />                                                                    163 Ca&#8217;s corresponding sequence identity of <br />                                                                    47%).35 The largest difference presence a <br />                                                                    b-bulge TbArl6 residue insertion <br />                                                                    residues 110 115 [Fig. 1(b)]. Hits also <br />                                                                    obtained Arf Arl structures their <br />                                                                    activated state GTP non-hydrolysable ligand <br />                                                                    bound. TbArl6 active conformation <br />                                                                    primed interacting protein partners. <br />  <br />                                                                    Crystal contacts <br />                                                                    Analysis using PISA30 indicated TbArl6 might <br />                                                                    form dimer [Fig. 3(a)]. Some GTPases known <br />                                                                    dimerise &#8212; Arf1 dimerisation, example, im- <br />                                                                    portant inducing membrane curvature allow <br />                                                                    formation COPI vesicles.36,37 Other small <br />                                                                    GTPases use dimerisation regulate activity, <br />                                                                    dimerisation providing residues aid GTP <br />                                                                    hydrolysis.38 However, side-by-side topology for <br />                                                                    dimer suggested <span id='am-66' about='obo:NCBITaxon_1340058' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_516942 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_6804 obo:NCBITaxon_116706 obo:NCBITaxon_197562 obo:NCBITaxon_116704 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-69'  ></span><span id='am-70' property="rdfs:label" content="Pisa" datatype="xsd:string"></span>PISA</span> indicative this <br />                                                                    interaction playing regulatory role. SEC-MALLS <br />                                                                    analysis conclusively showed TbArl6 mono- <br />                                                                    mer solution presence GppNp or <br />                                                                    GDP conclude probable bio- <br />                                                                    logical unit. Nevertheless, dimerisation be <br />                                                                    completely ruled conditions, with <br />                                                                    Arf1 dimerisation observed presence of <br />                                                                    membranes. The potential dimer crystal may <br /> Figure 3. (A) Structure dimer suggested PISA          hint possibility similar phenomenon for <br /> an oligomeric assembly. (B) (C) SEC-MALLS data          TbArl6. <br /> presence GppNp GDP respectively indicating that <br /> TbArl6 5 mg/mL monomer solution. The solid,             The nucleotide binding pocket <br /> dashed dotted curves represent Rayleigh ratio, UV          The TbArl6 active site typical small GTPases <br /> light absorption, differential refractive index respectively   significant difference. There highly <br /> of solution emerging SEC column. Open and <br />                                                                    conserved glutamine residue (Gln73 human <br /> closed circles represent molecular mass species <br />                                                                    enzyme) active site small GTPases, <br /> across selected peaks calculated MALLS data. <br />                                                                    proposed responsible polarising stabilis- <br /> uploaded ConSurf server generate                     ing nucleophilic water.39,40 While Gln substi- <br /> sequence conservation scores protein                        tuted His organisms, mutation <span id='am-47' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49'  ></span><span id='am-50' property="rdfs:label" content="Ala" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Ala</span> is <br /> structure.32                                                       routinely used produce GTP locked form the <br />      The use Arl6-annotated sequences only, led              enzyme greatly reduced ability hydro- <br /> there fewer sequences optimal                lyse GTP.41,42 In contrast, natural residue this <br /> analysis, resulted lot residues having               position TbArl6 alanine (Ala63), noted by <br /> insufficient data scoring [Fig. 4(a,b)]. Neverthe-             Price et al.28 [Fig. 4(a)]. Since obvious <br /> less, results GTP binding pocket                 residue substitute role played by <br /> is highly conserved [Fig. 4(b)], adjacent                 Gln activating nucleophile (Fig. 2), TbArl6 <br /> water filled cavity volume 142 A   &#730; 3 33 [Fig.          likely reduced ability hydrolyse GTP. <br /> 4(b)]. In addition, smaller clus-               In small GTPases Ras-related pro- <br /> ters conserved residues regions                   teins, catalytic Gln replaced threonine <br />  <br />  <br /> Hemsworth et al.                                                                         PROTEIN SCIENCE VOL 22:196&#8212;203   199 <br />  Figure 4. Consurf analysis TbArl6. (A) Sequence alignment coloured Consurf score according colour key shown. <br /> The secondary structure TbArl6 shown alignment Ala63 indicated red star (Figure <br /> generated using ALINE59). (B&#8211;D) The sequence conservation TbArl6 surface. The GTPase pocket adjacent <br /> small pocket highly conserved (B) surfaces end central b-sheet (C D) show <br /> some sequence conservation. <br />  <br />  <br /> which interacts GTPase Activating Protein                investigate hypothesis further, especially one <br /> (GAP) partner.43 This GAP provides Asn &#8216;&#8216;thumb&#8217;&#8217;             surfaces PISA suggests might <br /> which acts catalytic residue inducing                represent dimer interface. <br /> GTPase activity.44,45 TbArl6 require                   The sequence alignment reveals significant dif- <br /> the action yet-to-be-identified GAP, possibility         ferences N-termini trypanosomal pro- <br /> which warrants investigation.                           teins compared counterparts higher <br />                                                                 eukaryotes [Fig. 4(a)]. The N-terminal portion of <br /> Conserved regionsin TbArl6                                      Arf/Arl proteins typically form amphipathic helix <br /> We constructed model allow analysis                 tucked away protein surface and <br /> sequence conservation protein surface                 runs antiparallel C-terminal helix GDP <br /> identify surfaces Arl6 use interact               bound extends away protein insert <br /> with protein partners. The region             anchor protein membranes when <br /> sequence conservation GTP                 GTP bound.48 Arfs N-myristoylated to <br /> binding site. Interestingly, adjacent GTP bind-          aid interaction membranes. The <br /> ing cleft conserved small water-filled pocket,             N-terminal helix truncated crystallisa- <br /> also noted previous Arf/Arl structures           tion Arf/Arl proteins, example human Arl6.35 <br /> proposals function.46,47                              While crystallised full-length protein, we <br />      There conserved surfaces             model residue onwards, reflect- <br /> end central b-sheet contributed             ing flexibility residues suggesting that <br /> by adjacent a-helices. Price et al.28 showed           shorter N-terminus TbArl6 does form a <br /> TbArl6 interacts tubulin suggested             helix. The alignment Figure 4(a) shows the <br /> links tubulin cargo BBSome pro-              Arl6s trypanosomes 10 residues shorter <br /> posed vesicle trafficking function. These surfaces              N-termini lack amphipathic <br /> may interact tubulin BBSome bridging               helix predicted proteins higher eukar- <br /> between them, clearly work required              yotes [Fig. 4(a)]. Possible sites N-myristoylation <br />  <br />  <br /> 200   PROTEINSCIENCE.ORG                                                                         Structure T. brucei Arl6/BBS3 <br />  can predicted using N-terminal sequence           final concentration 8 mM. Crystallisation trials <br /> motif, G-[EDRKHPFYW]-x(2)-[STAGCNDEF]-[P].49             set using Mosquito robot (TTP Labtech). <br /> The T. brucei protein known N-myristoylated     Crystals obtained 0.2 M CsCl, 2.2 M (NH4)2SO4 <br /> in vivo28 sequence analysis suggests Trypa-     sufficient quality data collection. <br /> nosoma vivax Arl6 highly likely N-myr- <br /> istoylated. While prediction strong     X-ray data collection, processing structure <br /> other trypanosome species, conceivable        determination <br /> they N-myristoylated Arl6 proteins.50 None      Crystals cryo-cooled 100K plunging <br /> of Arl6s extended N-termini      directly liquid nitrogen addition of <br /> candidates N-myristoylation suggesting          cryo-protectant. Diffraction images collected at <br /> these proteins use different method membrane        Diamond Light Source, station I04, wave- <br /> attachment localisation compared trypanoso-       length 0.979 A &#730; indexed using XDS.51 Subse- <br /> mal Arl6s.                                               quent data processing performed using the <br />                                                          CCP4 software package.52 The structure deter- <br /> Conclusions                                              mined molecular replacement using MrBUMP53 <br /> The structure T. brucei Arl6 complex      used human Arl6 structure (PDB code <br /> non-hydrolysable GTP analog GppNp reveals typi-        2h57) search model. A round autobuilding <br /> cal Arl structure significant differences       performed BUCCANEER54 assign the <br /> compared orthologues organisms             sequence. Subsequent refinement manual model <br /> including humans. The recent work Price           building performed using REFMAC555 and <br /> co-workers28 suggests TbArl6,       COOT56 respectively. The quality model was <br /> BBSome, functions relating flagellum,        monitored rebuilding refinement <br /> an organelle key importance trypanosome        using MolProbity.57 Data processing structure <br /> life cycle. This clearly area warrants fur-   refinement statistics Table I. <br /> ther investigation structure presented           The X-ray data coordinates de- <br /> will help guide future biochemical investigations     posited PDB accession code 4bas. <br /> this important small GTPase. <br />                                                          Size exclusion chromatography Multi-Angle <br /> Materials Methods                                    Laser Light Scattering (SEC-MALLS) analysis <br />                                                          Around 50 lL samples TbArl6 1 5 mg/mL <br /> TbArl6 expression purification                       loaded BioSep-SEC-S 3000 column (Phe- <br /> N-terminally His6 tagged TbArl6 expressed            nomenex) equilibrated 50 mM HEPES pH 7.0, 5 <br /> from pET-YSBLLIC vector described Price        mM MgCl2, 250 mM NaCl, 1 mM DTT presence <br /> et al.28 Cells resuspended 5 volumes          1 mM GppNp 1 mM GDP. Light scattering data <br /> Buffer A (50 mM HEPES pH 7.0, 0.5 M NaCl, 5 mM           recorded Dawn Heleos II 18-angle light scat- <br /> MgCl2, 1 mM DTT, 30 mM imidazole) lysed           tering detector in-line OptilabrEX refractive <br /> sonication. Cell debris removed centrifuga-       index monitor (Wyatt Technology). Data analysed <br /> tion 18,000 rpm Sorval SS-34 rotor 30        using ASTRA software fitted Zimm <br /> minutes. The supernatant removed applied         model estimated dn/dc value 0.186 mL/g. <br /> to 5 mL His-trap column equilibrated Buffer A. <br /> After washing 4 column volumes (CVs)             Acknowledgments <br /> Buffer A, gradient applied 0 100 %         We thank Diamond Light Source beamline IO4 pro- <br /> buffer B (Buffer A &#254; 300 mM Imidazole) 20           vision X-ray data collection facilities. Funding was <br /> CVs collecting 1.6 mL fractions. Peak fractions con-     provided Welcome Trust Programme Grant 077503 <br /> taining TbArl6 pooled concentrated          DFS. <br /> size exclusion chromatography. The protein was <br /> applied 26/60 Superdex 75 column had <br /> been equilibrated SEC buffer (10 mM HEPES pH          REFERENCES <br /> 7.0, 250 mM NaCl, 5 mM MgCl2, 1 mM DTT). After            1. Simarro PP, Cecchi G, Paone M, Franco JR, Diarra A, <br /> a void volume 80 mL, 4 mL fractions col-             Ruiz JA, Fevre EM, Courtin F, Mattioli RC, Jannin JG <br />                                                              (2010) The Atlas human African trypanosomiasis: a <br /> lected. Peak fractions containing pure TbArl6 were <br />                                                              contribution global mapping neglected tropical <br /> combined concentrated 92 mg/mL deter-              diseases. Int J Health Geogr 9:57. <br /> mined A280 using extinction coefficient      2. Branche C, Kohl L, Toutirais G, Buisson J, Cosson J, <br /> 15470/M/cm molecular mass 21803.8.                  Bastin P (2006) Conserved specific functions of <br />                                                              axoneme components trypanosome motility. J cell sci <br />                                                              119:3443&#8211;3455. <br /> TbArl6:GppNp complex crystallisation                      3. Broadhead R, Dawe HR, Farr H, Griffiths S, Hart SR, <br /> A stock 200 mM guanosine 50 -[b,c-imido]triphos-          Portman N, Shaw MK, Ginger ML, Gaskell SJ, <br /> phate (GppNp) added 92 mg/mL TbArl6           McKean PG, Gull K (2006) Flagellar motility is <br />  <br />  <br />  <br /> Hemsworth et al.                                                                PROTEIN SCIENCE VOL 22:196&#8212;203   201 <br />        required viability bloodstream trypano-             membranes stabilizes protein transport Caeno- <br />       some. Nature 440:224&#8211;227.                                          rhabditis elegans. J Cell Biol 188:953&#8211;969. <br />  4.   Ralston KS, Hill KL (2006) Trypanin, component          23.   Chiang AP, Nishimura D, Searby C, Elbedour K, Carmi <br />       flagellar Dynein regulatory complex, essential           R, Ferguson AL, Secrist J, Braun T, Casavant T, Stone <br />       bloodstream form African trypanosomes. PLoS Pathog                 EM, Sheffield VC (2004) Comparative genomic analysis <br />       2:e101.                                                            identifies ADP-ribosylation factor-like gene the <br />  5.   Engstler M, Pfohl T, Herminghaus S, Boshart M, Wie-                cause Bardet-Biedl syndrome (BBS3). Am J Hum <br />       gertjes G, Heddergott N, Overath P (2007) Hydrody-                 Genet 75:475&#8211;484. <br />       namic flow-mediated protein sorting cell surface      24.   Fan Y, Esmail MA, Ansley SJ, Blacque OE, Boroevich <br />       trypanosomes. Cell 131:505&#8211;515.                                 K, Ross AJ, Moore SJ, Badano JL, May-Simera H, <br />  6.   Vickerman K (1969) On surface coat flagellar               Compton DS, Green JS, Lewis RA, van Haelst MM, <br />       adhesion trypanosomes. J cell sci 5:163&#8211;193.                    Parfrey PS, Baillie DL, Beales PL, Katsanis N, David- <br />  7.   Vickerman K (1985) Developmental cycles biology                son WS, Leroux MR (2004) Mutations member of <br />       pathogenic trypanosomes. Br Med Bull 41:105&#8211;114.                Ras <span id='am-39' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="superfamily" datatype="xsd:string"></span>superfamily</span> small GTP-binding proteins <br />  8.   Hill KL (2003) Biology mechanism trypanosome                causes Bardet-Biedl syndrome. Nat Genet 36:989&#8211;993. <br />       cell motility. Eukaryot Cell 2:200&#8211;208.                      25.   Kobayashi T, Hori Y, Ueda N, Kajiho H, Muraoka S, <br />  9.   Ginger ML, Portman N, McKean PG (2008) Swimming                    Shima F, Kataoka T, Kontani K, Katada T (2009) Bio- <br />       protists: perception, motility flagellum assem-           chemical characterization missense mutations the <br />       bly. Nat Rev Microbiol 6:838&#8211;850.                                  Arf/Arl-family small GTPase Arl6 causing Bardet-Biedl <br /> 10.   Fliegauf M, Benzing T, Omran H (2007) When cilia                syndrome. Biochem Biophys Res Commun 381: <br />       bad: cilia defects ciliopathies. Nat Rev Mol cell biol         439&#8211;442. <br />       8:880&#8211;893.                                                   26.   Zhang Q, Nishimura D, Seo S, Vogel T, Morgan DA, <br /> 11.   Iqbal H, Wahedi HM, Hafeez FY, Mir A (2010) Molecu-                Searby C, Bugge K, Stone EM, Rahmouni K, Sheffield <br />       lar variants Bardet-Biedl Syndrome. J Cell Biol                 VC (2011) Bardet-Biedl syndrome 3 (Bbs3) knockout <br />       Genet 1:1&#8211;11.                                                      mouse model reveals common BBS-associated pheno- <br /> 12.   Forsythe E, Beales PL (2012) Bardet-Biedl syndrome.                types Bbs3 unique phenotypes. Proc Natl Acad Sci <br />       Eur J Hum Genet.                                                   USA 108(51):20678&#8211;20683. <br /> 13.   Nachury MV, Loktev AV, Zhang Q, Westlake CJ, <span id='am-1' about='obo:NCBITaxon_256812' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3646 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_629717 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Pera" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5'  ></span>Pera</span>-           27.   Ou G, Blacque OE, Snow JJ, Leroux MR, Scholey JM <br />       nen J, Merdes A, Slusarski DC, Scheller RH, Bazan JF,              (2005) Functional coordination intraflagellar trans- <br />       Sheffield VC, Jackson PK (2007) A core complex                  port motors. Nature 436:583&#8211;587. <br />       BBS proteins cooperates GTPase Rab8 pro-         28.   Price HP, Hodgkinson MR, Wright MH, Tate EW, <br />       mote ciliary membrane biogenesis. Cell 129:1201&#8211;1213.              Smith BA, Carrington M, Stark M, Smith DF (2012) A <br /> 14.   Loktev AV, Zhang Q, Beck JS, Searby CC, Scheetz TE,                role vesicle-associated tubulin binding protein <br />       Bazan JF, Slusarski DC, Sheffield VC, Jackson PK,                  ARL6 (BBS3) flagellum extension Trypanosoma <br />       Nachury MV (2008) A BBSome subunit links ciliogene-                brucei. Biochim Biophys Acta 1823:1178&#8211;1191. <br />       sis, microtubule stability, acetylation. Dev Cell 15:    29.   Wittinghofer A, Vetter IR (2011) Structure-function <br />       854&#8211;865.                                                           relationships G domain, canonical switch motif. <br /> 15.   Berbari NF, Lewis JS, Bishop GA, Askwith CC, Myky-                 Annu Rev Biochem 80:943&#8211;971. <br />       tyn K (2008) Bardet-Biedl syndrome proteins              30.   Krissinel E, Henrick K (2007) Inference macromolec- <br />       required localization G protein-coupled recep-          ular assemblies crystalline state. J mol biol 372: <br />       tors primary cilia. Proc Natl Acad Sci U S A 105:               774&#8211;797. <br />       4242&#8211;4246.                                                   31.   Di Tommaso P, Moretti S, Xenarios I, Orobitg M, Mon- <br /> 16.   Domire JS, Green JA, Lee KG, Johnson AD, Askwith                   tanyola A, Chang JM, Taly JF, Notredame C (2011) T- <br />       CC, Mykytyn K (2011) Dopamine receptor 1 localizes              Coffee: web server multiple sequence align- <br />       neuronal cilia dynamic process requires              ment protein RNA sequences using structural <br />       Bardet-Biedl syndrome proteins. Cell Mol Life Sci 68:              information homology extension. Nucl Acids Res <br />       2951&#8211;2960.                                                         39:W13&#8211;W17. <br /> 17.   Wei Q, Zhang Y, Li Y, Zhang Q, Ling K, Hu J (2012)           32.   Ashkenazy H, Erez E, Martz E, Pupko T, Ben-Tal N <br />       The BBSome controls IFT assembly turnaround                 (2010) ConSurf 2010: calculating evolutionary conser- <br />    <br /> </body></html>