<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> An in silico approach combined with in vivo <br /> experiments enables the identification of a new <br /> protein whose overexpression can compensate <br /> for specific respiratory defects in <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-26'  ></span><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span> <br /> cerevisiae <br />  <br />   Abstract <br />   Background: The mitochondrial inner membrane contains five large complexes that are essential for oxidative <br />   phosphorylation. Although the structure and the catalytic mechanisms of the respiratory complexes have been <br />   progressively established, their biogenesis is far from being fully understood. Very few complex III assembly factors <br />   have been identified so far. It is probable that more factors are needed for the assembly of a functional complex, <br />   but that the genetic approaches used to date have not been able to identify them. We have developed a systems <br />   biology approach to identify new factors controlling complex III biogenesis. <br />   Results: We collected all the physical protein-protein interactions (PPI) involving the core subunits, the <br />   supernumerary subunits and the assembly factors of complex III and used Cytoscape 2.6.3 and its plugins to <br />   construct a network. It was then divided into overlapping and highly interconnected sub-graphs with clusterONE. <br />   One sub-graph contained the core and the supernumerary subunits of complex III, it also contained some subunits <br />   of complex IV and proteins participating in the assembly of complex IV. <span id='am-33' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-37'  ></span>This</span> sub-graph was then split with another <br />   algorithm into two sub-graphs. The subtraction of these two sub-graphs from the previous sub-graph allowed us <br />   to identify a protein of unknown function Usb1p/Ylr132p that interacts with the complex III subunits Qcr2p and <br />   Cor1p. We then used genetic and cell biology approaches to investigate the function of Usb1p. Preliminary results <br />   indicated that Usb1p is an essential protein with a dual localization in the nucleus and in the mitochondria, and <br />   that the over-expression of this protein can compensate for defects in the biogenesis of the respiratory complexes. <br />   Conclusions: Our systems biology approach has highlighted the multiple associations between subunits and <br />   assembly factors of complexes III and IV during their biogenesis. In addition, this approach has allowed the <br />   identification of a new factor, Usb1p, involved in the biogenesis of respiratory complexes, which could not have <br />   been found using classical genetic screens looking for respiratory deficient mutants. Thus, this systems biology <br />   approach appears to be a fruitful new way to study the biogenesis of mitochondrial multi-subunit complexes. <br />  <br />  <br /> Background                                                     which enable systematic pair wise PPI identification and <br /> The oxidative phosphorylation system (OXPHOS) is               affinity-purification followed by mass spectrometry (AP/ <br /> located in the inner membrane of mitochondria and is           MS) (for examples see [25-27]). The integration of all <br /> responsible for the production of ATP, the energy cur-         these <span id='am-46' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-50'  ></span>data</span> generates a network represented by a graph <br /> rency of the cell. It is composed of five large complexes,     where every node corresponds to a protein and every <br /> the complexes of the respiratory chain (complexes I to IV)     edge to an interaction. This large network is too complex <br /> and the ATP synthase, or complex V. The model organism         to be used directly, so specific tools are needed in <span id='am-43' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> <br /> <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> lacks complex I, but instead has      to extract the relevant information from this network. <br /> three single subunit NADH dehydrogenases. The other            Many algorithms and methods of protein network analy- <br /> OXPHOS complexes II-V are similar to the mammalian             sis (clustering and assessment of data quality) have been <br /> complexes. Complex II is composed of four subunits of          developed and tested on the various datasets available for <br /> nuclear origin while the complexes III, IV and V are com-      the yeast S. cerevisiae. <br /> posed of subunits of dual genetic origin (10, 11 and 19           Broh&#233;e and Van Helden [28] tested different algo- <br /> subunits respectively in S. cerevisiae). A few core subunits   rithms for clustering PPI networks. Starting from all the <br /> are encoded by the mitochondrial genome, translated on         PPI data of S. cerevisiae they showed that, they could <br /> mitochondrial ribosomes and immediately inserted into          find all the 220 protein complexes presently documen- <br /> the membrane, while most subunits are encoded by               ted in the MIPS database [29]. Amongst these algo- <br /> nuclear genes, translated in the cytoplasm and imported        rithms, some enable the extraction of independent <br /> into mitochondria via the TOM/TIM machineries (for a           classes (every protein is a part of a single module), while <br /> review see [1]). The assembly of the subunits appears to       others enable the extraction of overlapping classes (a <br /> be coordinated (for examples see [2-5]) by site-specific       protein can be a part of several modules). Each of these <br /> translation mechanisms, both within and at the periphery       algorithms identifies sub-networks having a specific <br /> of the mitochondria and numerous assembly factors that         topology: weakly interconnected clusters [30] or strongly <br /> are extrinsic to the complexes have been identified. In        interconnected clusters [31]. Within the latter group, <br /> addition, supra-complex associations, either between dif-      MCODE [32] and ClusterONE [33] identify large-sized <br /> ferent mitochondrial complexes, or with the TIM import         clusters. This is particularly interesting for our purpose, <br /> machinery have been reported [6-10]. However, the              because biochemical analyses suggest that there are few <br /> assembly of the complexes, especially of complex III, and      assembly modules for complex III and that these mod- <br /> the interactions between the complexes during their            ules contain numerous strongly interconnected proteins. <br /> assembly is poorly understood.                                    In this report, we used a systems biology approach that <br />    Complex III occupies a central position in the respira-     consists of the integration of data from different types of <br /> tory chain, between the entry points of the electrons          experiments, but mainly from large-scale studies. Using <br /> (NADH dehydrogenases and complex II) and their exit,           the APID [34] and BioGRID databases [20], we collected <br /> complex IV. The crystallographic structure of the enzyme       all the physical interactions concerning the core subunits, <br /> from several sources has been determined [11-14], and a        the supernumerary subunits and the assembly factors of <br /> modular assembly scheme has been proposed ([15] and            complex III. After the repeated partitioning of the initial <br /> references within). Until now, only six assembly factors       network into sub-graphs we identified a group of pro- <br /> specific for complex III have been reported: Cyt2p, Bcs1p,     teins having few interactions with the main complex III <br /> Cbp3p, Cbp4p, Mzm1p (or Aim8p) and Bca1p [16-19],              sub-graph. Cell biology and genetic methods were then <br /> while more than 25 have been identified for the complex        used to investigate the function of one of the proteins <br /> IV (for review see [5]), an enzyme of similar complexity,      identified by our in silico approach. <br /> suggesting that many of the factors involved in complex <br /> III biogenesis remain to be discovered.                        Results and Discussion <br />    In order to identify proteins strongly connected with       Network modelling <br /> complex III and to look for proteins participating in its      In order to find new proteins involved in complex III <br /> assembly process, a systems biology approach could be          biogenesis, we used the Cytoscape facilities [35] to com- <br /> used. Many high-throughput datasets coming from                bine all the physical interactions concerning complex III <br /> genetic and proteomic studies concerning the yeast S.          (core proteins, supernumerary proteins and assembly <br /> cerevisiae are accessible via public databases [20]. The       factors, see Table 1) from the APID [34] and BioGRID <br /> interactome represents all the physical interactions           databases [20]. <br /> between proteins of the cell. Two main techniques have           The computational approach leading to the complete <br /> been used to produce high quality and complementary            PPI network (see section methods) gave the following <br /> data: yeast two-<span id='am-51' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>hybrid</span> studies (for examples see [21-24]),     results: <br />  Glatigny et al. BMC Systems Biology 2011, 5:173                                                                                                     Page 3 of 12 <br /> http://www.biomedcentral.com/1752-0509/5/173 <br />  <br />  <br />  <br />  <br /> Table 1 List of the proteins of complex III (input list) <br /> Complex III                                                 Ordered Locus Name                            UniProt ID                        SGD Name <br /> Catalytic core                                                     YEL024W                                  P08067                              RIP1 <br />                                                                    YOR065W                                  P07143                              CYT1 <br />                                                                      Q0105                                  P00163                              COB <br /> Supernumerary subunits                                              YBL045C                                 P07256                             COR1 <br />                                                                    YPR191W                                  P07257                             QCR2 <br />                                                                     YFR033C                                 P00127                             QCR6 <br />                                                                    YDR529C                                  P00128                             QCR7 <br />                                                                    YJL166W                                  P08525                             QCR8 <br />                                                                    YGR183C                                  P22289                             QCR9 <br />                                                                   YHR001W-A                                 P37299                             QCR10 <br /> Assembly factors                                                    YKL087C                                 Q00873                              CYT2 <br />                                                                    YDR375C                                  P32839                              BCS1 <br />                                                                    YPL215W                                  P21560                              CBP3 <br />                                                                    YGR174C                                  P37267                              CBP4 <br />                                                                    YLR077W                                  Q08023                             BCA1 <br />                                                                    YDR493W                                  Q03429                             MZM1 <br /> Complex III is defined by three groups of proteins: catalytic core and supernumerary subunits that are part of the structure, and other proteins involved in the <br /> assembly of the complex. UniProt ID was used to retrieve data in APID database whereas in BioGRID (or SGD) it was the SGD Name. <br />  <br />  <br />   1) From the APID database, we obtained a list of 521                             In this way the network was sub-divided in 26 sub-graphs <br /> unique PPI involving 125 proteins (list 1 of Figure 1).                            (clusters 1 to 26), of which six have p-values under 0.05 <br />   2) From the BioGRID database, 123 unique PPI involving                           and good quality scores (Table 2). Note that the clustering <br /> 91 different proteins were collected (list 2 of Figure 1).                         method is robust, since the same sub-graphs are found <br /> Some of these interactions are new and allowed us to                               when starting the analysis from only the core and super- <br /> enrich those obtained from APID.                                                   numerary proteins. <br />   3) Among these new interactions, some concern ele-                                  Cluster 1 is composed of 175 PPI involving 37 proteins; <br /> ven new proteins of which some have numerous direct                                cluster 2 is composed of 157 PPI involving 34 proteins <br /> PPI. For example, there are 2645 physical interactions                             and cluster 4 of 144 PPI involving 33 proteins (Additional <br /> with Nab2p and 1444 with Ubi4p, proteins that are                                  file 1 Figure S1). All are built around non-mitochondrial <br /> independent of complex III. We will discuss later the                              proteins having many interactions independent of com- <br /> consequences of these PPI on the network.                                          plex III. For example Nab2p is a nuclear polyAdenylated <br />   4) The final list, obtained by merging the three PPI lists                       RNA-binding protein, Ubi4p is a ubiquitin, Ssa2p is a <br /> and eliminating redundancy, was composed of 719 unique                             stress-seventy <span id='am-30' about='obo:NCBITaxon_subfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="subfamily" datatype="xsd:string"></span>subfamily</span> A protein, Rpn10p a non- <br /> PPI involving 136 proteins. 80 proteins are common to                              ATPase subunit of the regulatory particle of the 26S pro- <br /> both databases, 45 are from APID and 11 from BioGRID.                              teasome, Tef1p a translational elongation factor and <br /> Eliminating redundancy facilitates the identification of                           Ssb1p is a ribosome-associated molecular chaperone. <br /> links between proteins not often studied and proteins                              Clusters 1, 2 and 4 thus contain PPI that are not specific <br /> known to be involved in complexes.                                                 for complex III. <br />   5) The resulting network was too large to yield any                                 Cluster 5 is composed of 101 PPI involving 23 proteins <br /> interesting information. It was necessary to divide it into                        and cluster 6 is composed of 63 PPI involving 16 proteins <br /> highly connected sub-graphs that represent functional                              (Additional file 2 Figure S2). The majority of these pro- <br /> modules, or protein sub-complexes that we could con-                               teins are translocases involved in the import of proteins <br /> sider as assembly intermediates.                                                   into mitochondria, but there are also a few proteins <br />                                                                                    involved in complex III: Bcs1p, Cobp and Cbp2p, a pro- <br /> Network clustering                                                                 tein of complex IV: Cox7p, and Atp1p and Atp2p, two <br /> To identify new proteins associated with the assembly                              proteins of the F1 part of complex V. These clusters are <br /> intermediates of complex III, we looked for sub-groups of                          clearly not specific for complex III as they include only a <br /> strongly interconnected proteins. First we looked for over-                        few proteins of the input list. <br /> lapping sub-graphs to take into account the fact that one                             Cluster 3 (Figure 2) is composed of 193 PPI involving <br /> protein can be associated with several sub-complexes.                              36 proteins, including all the complex III subunits. When <br />  Glatigny et al. BMC Systems Biology 2011, 5:173                                                                     Page 4 of 12 <br /> http://www.biomedcentral.com/1752-0509/5/173 <br />  <br />  <br />  <br />  <br />           List of proteins involved in complex III in S. cerevisiae <br />  <br />                             APID database                                 BioGRID database <br />  <br />               list 1       (521 PPI, 125 Proteins)                            list 2 (123 PPI,        91 Proteins) <br />  <br />  <br />  <br />                                    ID unification (SGD Standard Name) <br />  <br />  Comparison of list 1 and 2                                        11 Proteins of list 2 and not in list 1 (N) <br />  <br />  <br />         BioGRID: proteins in the functional enrivonment of N                                               PPI list 3 <br />                                                                                          (359 PPI, 104 Proteins) <br />  <br />        Merging of lists 1, 2 and 3 and elimination of redundancy <br />                            (719 PPI, 136 Proteins) <br />   (80 proteins common to APID and BioGRID, 45 from APID, 11 from BioGRID) <br />  <br />                  Network modeling and clustering in Cytoscape <br />                                              (ClusterONE and MCODE) <br />  <br />  <br />                      Clusters of strongly interconnected proteins <br />  <br />                            Identification of new proteins involved <br />                                   in complex III biogenesis <br />  Figure 1 Computational workflow. An overview of the network modelling and clustering. <br />  <br />  <br /> it was split again into overlapping sub-graphs, only one              (CoA1p, Mss51p, Shy1p, Rrg10p/CoA3p). The fact that <br /> sub-graph (cluster 3.1) was extracted, with 185 PPI invol-            this network contains proteins from complexes III and <br /> ving 30 proteins. Its quality is equal to 0.97. Most of pro-          IV, although our input list was exclusively composed of <br /> teins of this sub-graph are members of complex III (core              proteins of complex III is probably related to the exis- <br /> and supernumerary subunits) and complex IV (Cox1p,                    tence of III-IV super-complexes, which have been clearly <br /> Cox2p, Cox3p, Cox4p, Cox5Ap, Cox6p, Cox14p,                           identified in blue native gels of mitochondrial extracts <br /> Cox15p) or participate in the assembly of complex IV                  [6,7]. Our results show that many associations link these <br />  Glatigny et al. BMC Systems Biology 2011, 5:173                                                                         Page 5 of 12 <br /> http://www.biomedcentral.com/1752-0509/5/173 <br />  <br />  <br />  <br />  <br /> Table 2 Characteristics of the complex III PPI sub-                     result obtained in silico led us to study the function of <br /> networks obtained with ClusterONE                                       Usb1p using genetic and cell biology methods. <br />  cluster    nodes      edges      density     quality     p-value <br />     1         37        175         0.25          0.476   2.25 10-7     Usb1p is located in both the nucleus and the <br />     2         34        157         0.25          0.444   1.01 10  -5   mitochondria <br />     3         36        193         0.29          0.556   1.35 10-4     The in silico analysis has shown that Usb1p belongs to <br />                                                                         the cluster 3.1 that contains all the complex III subunits, <br />     4         33        144         0.25          0.414   1.00 10-3 <br />                                                                         six out of the 11 subunits of complex IV, as well as six <br />     5         23        101         0.36          0.419   2.00 10-3 <br />                                                                         assembly factors of complex IV. In order to determine if <br />     6         16         63        0.450          0.432    0.011 <br />                                                                         Usb1p could be involved in the biogenesis of the mito- <br />    3.1        30        185        0.409          0.967    0.000        chondrial respiratory complexes III and/or IV, we first <br />                                                                         analyzed its sub-cellular localization. <br />                                                                            Large-scale analysis of the localization of GFP-tagged <br /> complexes. Furthermore, several assembly factors of                     S. cerevisiae proteins suggested that Usb1p exhibits a <br /> complex IV identified here interact with several subunits               multiple location in the nucleus, the cytoplasm and the <br /> of complex III: Cor1p, Qcr2p, Cyt1p, suggesting that                    mitochondria [36]. To verify this localization, Usb1p was <br /> some factors may play a role in the assembly of both                    tagged at its C-terminus with GFP. The genetic con- <br /> complexes III and IV.                                                   struction was integrated at the USB1 chromosomal <br />    As it was no longer possible to split the cluster 3.1,               locus in a strain carrying a CFP-tagged Nic96 protein <br /> which is still large, into overlapping sub-graphs it was                that revealed the pores of the nuclear membrane [37]. <br /> divided into independent clusters: cluster 3.1.1 is com-                The USB1-GFP gene is therefore expressed under the <br /> posed of 81 PPI involving 13 proteins belonging to com-                 control of its own promoter sequence in order to avoid <br /> plex IV, three proteins of complex III (Cor1p, Qcr2p and                mislocalization due to overexpression when the con- <br /> Cyt1p) and Aac1p, a minor isoform of the ADP/ATP                        struction is carried on a plasmid. Cells expressing the <br /> mitochondrial transporter (Figure 2). Cluster 3.1.2 is                  Nic96-CFP and the Usb1-GFP fusion proteins were <br /> composed of 27 PPI involving 8 proteins; most of them                   examined by fluorescence microscopy using the appro- <br /> are members of complex III (Cobp, Qcr6p, Qcr7p,                         priate filters. Mitotracker was used to stain the mito- <br /> Qcr8p, Qcr9p, Qcr10p, Rip1p) and one protein Pet9p/                     chondrial network. As shown in Figure 3 andAdditional <br /> Aac2, the major isoform of the ADP/ATP transporter                      file 4 (Figure S3), Usb1-GFP was located within the <br /> (Figure 2). Thus the cluster 3.1 can be separated into two              nucleus (3A) whose membrane was also stained by the <br /> parts corresponding broadly to complexes III (see cluster               Nic96-CFP (3C) as well as in fluorescent tubules also <br /> 3.1.2) and IV (see cluster 3.1.1). However, a few proteins              stained with the Mitotracker probe (3B). Thus the <br /> are linked with the proteins of the other complex rather                Usb1-GFP protein has a dual localization in the nucleus <br /> than with the proteins of their own complex. For exam-                  and mitochondria. <br /> ple, Cor1p and Qcr2p are in the sub-graph of complex <br /> IV rather than in the sub-graph of complex III as                       Overexpression of USB1 compensates specific respiratory <br /> they have more interactions with proteins of complex IV                 defects <br /> and Aac1p than with the other subunits of complex III.                  Systematic deletion analysis suggested that the gene could <br /> This strengthens the hypothesis of the existence of com-                be essential [26]. Indeed, we were not able to obtain any <br /> mon factors involved in the assembly process of both                    viable haploid cell carrying a deleted allele of the USB1 <br /> complexes.                                                              gene from a heterozygous diploid strain (see Methods). To <br />    If we subtract the two sub-clusters (3.1.1 and 3.1.2)                further investigate its role, within mitochondria, we have <br /> from cluster 3.1, nine proteins are left, of which five                 analyzed the effect of the overexpression of the USB1 gene <br /> (Usb1p, Smk1p, Yju2p, Pcl9p, Rbs1p) are connected to                    on a set of respiratory deficient mutants. We have cloned <br /> Cor1p and Qcr2p. They form a sub-graph, which is not                    USB1 on a multicopy plasmid and transformed either a <br /> interconnected enough to be detected by MCODE.                          wild type strain or a collection of respiratory deficient <br />    Among these proteins Usb1p is the only one whose                     mutants that carry mutations in various factors essential <br /> function is unknown. The association of these three pro-                for the biogenesis of the respiratory complexes. Table 3 <br /> teins (Cor1p, Qcr2p and Usb1p) corresponds to a complex                 and Figure 4A show that the overexpression of USB1 com- <br /> isolated by tandem affinity purification by Gavin et al. [25].          pensates for the respiratory deficiency of an oxa1-E65G- <br /> Thus Usb1p is connected with complex III, and more par-                 F229S mutant and the deletion of MFT2, but not a bcs1 <br /> ticularly to the Cor1p [25,26] and Qcr2p sub-units [25,27]              mutant, or the deletions of OXA1, BCS1 or RMD9. The <br /> (see also Additional file 3 Table S1: the list of all the pro-          same level of compensation was observed with the oxa1 <br /> teins interacting with Usb1p present in our network). This              and mtf2 mutants. We have previously shown that the <br />  Glatigny et al. BMC Systems Biology 2011, 5:173                                                                                      Page 6 of 12 <br /> http://www.biomedcentral.com/1752-0509/5/173 <br />  <br />  <br />  <br />  <br />  <br />  <br />  Figure 2 Complex III sub-graph 3 and its sub-graphs. Images are taken from Cytoscape. Cluster 3 comes from the partition of the whole PPI <br />  network of complex III with ClusterONE. Cluster 3.1 comes from the partition of cluster 3 with ClusterONE. Cluster 3.1.1 and 3.2.2 come from the <br />  partition of cluster 3.1 with MCODE. Complex III proteins are represented by grey circles; complex IV proteins by grey rounded squares; complex <br />  IV assembly factors by grey octagons; Usb1p by a white lozenge and other proteins by white circles. <br />  Glatigny et al. BMC Systems Biology 2011, 5:173                                                                                                    Page 7 of 12 <br /> http://www.biomedcentral.com/1752-0509/5/173 <br />  <br />  <br />  <br />  <br />                        Usb1-GFP                                     Mitochondria                                       Nucleus <br />                                                                     (mitotracker)                                    (Nic96-CFP) <br />  <br />  <br />  <br />  <br />   Figure 3 Dual localization of Usb1p. Cells expressing the Usb1-GFP and Nic96-CFP fusion proteins were grown in complete glucose medium <br />   with a five times excess of adenine at 28&#176;C, washed once and incubated with Mitotracker red CMX ROS (MT, Molecular Probes) for 10 min. After <br />   two additional washes, cells were observed by fluorescence microscopy with the appropriate filters to visualize GFP, CFP or Mitotracker. <br />  <br />  <br /> respiratory deficiency of the oxa1-E65G-F229S mutant is                            to choose which gene/protein is the best candidate for <br /> correlated to a strong decrease of cytochrome b of com-                            further analysis in a complicated system like OXPHOS, <br /> plex III and cytochromes aa3 of complex IV [38]. Cyto-                             which is composed of several complexes. Thus, an inte- <br /> chrome spectra of cells grown on respiratory medium                                grative method can help to orient subsequent research <br /> show that the overexpression of USB1 in the oxa1 mutant                            and can also highlight specific interactions between the <br /> restored the assembly of cytochromes b and aa3 (Figure                             complexes. <br /> 4B). Thus the overexpression of USB1 specifically com-                               Our approach revealed multiple associations between <br /> pensates for some respiratory deficiencies.                                        subunits of complexes III and IV and interactions with <br />                                                                                    the ADP/ATP transporters. Cluster 3.1 identifies connec- <br /> Conclusions                                                                        tions between complex III subunits, complex IV subunits <br /> Although several computational approaches combined                                 and assembly factors, suggesting some interaction during <br /> with experimental methods (for examples [39,40]) have                              the assembly process of these two complexes (see [41] <br /> recently established a comprehensive network for the                               and references therein for review on complex IV assem- <br /> mitochondrial proteome, our analysis is the first systems                          bly factors). For example, our results show that the <br /> biology study strictly devoted to the OXPHOS complexes                             assembly factor Shy1p is connected with all the proteins <br /> in yeast. As there are huge amounts of interaction data                            of the cluster 3.1.1, i.e. with the complex III subunits, <br /> from different sources (genetic, proteomics) it is not easy                        Cor1p, Qcr2p, Cyt1p and some subunits and assembly <br />  <br />  <br /> Table 3 Overexpression of Ylr132 compensates for specific respiratory deficiencies <br />    Genotype            Transformation with the empty vector                  Transformation with the wt gene                 Transformation with YLR132 <br />       oxa1                                   -                                                 ++                                            + <br />       &#916;oxa1                                  -                                                 ++                                            - <br />       bcs1                                   -                                                 ++                                            - <br />       &#916;bcs1                                  -                                                 ++                                            - <br />       &#916;mtf2                                  -                                                 ++                                            + <br />      &#916;rmd9                                   -                                                 ++                                            - <br /> Respiratory-deficient mutants affecting various steps in the biogenesis of the respiratory complexes were transformed with three different high-copy <span id='am-56' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span>: (1) <br /> the empty control vector, (2) the plasmid carrying the corresponding wild type gene (OXA1 for lines 1 and 2; BCS1 for lines 2 and 3; MTF2 for line 5 and RMD9 <br /> for line 6), (3) the plasmid carrying the wild type USB1 gene. The transformants were selected on minimal glucose medium lacking uracil to select for the <br /> plasmids and their growth were tested on non fermentable medium containing 2% glycerol and 2% ethanol at 28 or 36&#176;C. +: respiratory growth; - no growth. <br /> See also Figure 4A. The oxa1 mutant carries a double amino-acid substitution E65G-F229S in the Oxa1 protein [38]. The bcs1 mutant carries a single amino-acid <br /> substitution F342C in Bcs1p [53]. The other mutants carry gene deletions as indicated [38,44,53]. <br />  Glatigny et al. BMC Systems Biology 2011, 5:173                                                                                     Page 8 of 12 <br /> http://www.biomedcentral.com/1752-0509/5/173 <br />  <br />  <br />  <br />  <br />                      A Transformation of the                   B             Cytochrome spectra <br />                                 oxa1 mutant                                             b <br />                                                                  A <br />                                                                                    c c1 <br />                           OXA1 vector USB1 <br />  <br />  <br />                                                                                                           a+aa3 <br />                           Fermentable medium                                                                  OXA1 <br />  <br />                                                                0.1                                                USB1 <br />  <br />                                                                                   550        600   Onm <br />                            Respiratory medium                         Growth on respiratory medium <br />  Figure 4 Overexpression of USB1 compensates for a respiratory deficiency. Panel A: The thermosensitive mutant oxa1-E65GF229S [38] was <br />  transformed with high-copy plasmids carrying the wild type OXA1 gene or the USB1 gene, or the empty control vector (vector). The <br />  transformants were spotted onto fermentable (glucose) and respiratory (glycerol/ethanol) medium. The plates were incubated for five days at 36&#176; <br />  C. Panel B: Cytochrome absorption spectra of the oxa1 cells transformed with the high copy plasmids carrying OXA1 or YLR132, grown on <br />  respiratory medium for 2 or 5 days at 36&#176;C. Cytochrome c1 and b are part of respiratory complex III while cytochromes a+aa3 are part of <br />  complex IV. <br />  <br />  <br /> factors of complex IV. The minor ADP/ATP transporter,                     translation [38-43]. The overexpression of USB1 cannot <br /> Aac1p is also part of the same cluster. Cluster 3.1.2 is                  compensate for the absence of Rmd9p but can restore <br /> composed of core (Rip1p, Cobp) and supernumerary pro-                     respiratory growth to a deletion of MTF2 (NAM1) that <br /> teins (except Cor1p, Qcr2p), as well as Pet9p/Aac2p, the                  affects the processing/stability of two mitochondrial <br /> major ADP/ATP transporter.                                                mRNAs, encoding two subunits of complex V [44]. Mtf2p <br />    Although some subunits of the complex II (Sdh3p and                    was also shown to interact with the mitochondrial RNA <br /> Sdh4p) are present in the PPI network we did not retrieve                 polymerase [45]. Finally, Usb1p was detected in a systema- <br /> them in the clusters, suggesting that the interactions                    tic screen for RNA-binding proteins in yeast [46]. Because <br /> between complex III and complex II are weak. Cyc1p is                     of the dual location of Usb1p, in the nucleus and mito- <br /> also present in the PPI network but there is no strong con-               chondria, it is tempting to suggest that Usb1p might con- <br /> nection with cytochrome c. However, cytochrome c is a                     trol the processing or stability of nuclear and <br /> mobile carrier, shuttling electrons between complexes III                 mitochondrial RNAs. As the mitochondrial respiratory <br /> and IV and the interactions are probably too dynamic to                   function is not required for yeast viability, the essential <br /> be detected by our analysis that only detects strongly                    function of Usb1p would be due to its eventual role in sta- <br /> interconnected proteins. No connections have been found                   bility/processing of nuclear RNA(s). <br /> between complex III and the NADH dehydrogenases sug-                         Thus, the sub-graph analysis that we have developed has <br /> gesting that there is no tight contact between them.                      highlighted the strong connections between subunits and <br />    Finally, the approach identified a protein Usb1p that had              assembly factors of the complexes III and IV and has led <br /> probably escaped previous genetic screens because it is                   to the identification of a new factor acting in the biogen- <br /> essential for cell survival. This protein has a dual localiza-            esis of respiratory complexes in yeast. <br /> tion, in the nucleus and mitochondria. Overexpression of <br /> USB1 compensates for the respiratory deficiency of an                     Methods <br /> oxa1 mutant. Oxa1p mediates the co-translational inser-                   Databases <br /> tion of mitochondrially-encoded subunits of the respiratory               To construct our network we collected data from different <br /> complexes within the inner membrane (for review see                       sources. Information about protein-protein interactions <br /> [42]). We have previously shown that an increase in the                   (PPI) were from APID (Agile Protein Interaction DataAna- <br /> steady state level of mitochondrial mRNAs encoding                        lyzer) which integrates and unifies PPI from different pub- <br /> respiratory complex subunits can also compensate for the                  lic repositories [34]. This database includes data from high <br /> respiratory deficiency of this oxa1 mutant via the overex-                throughput proteomic techniques as well as small-scale <br /> pression of RMD9. Rmd9p controls the processing/stability                 experimental data; data quality is critical as it indicates the <br /> of all the mitochondrial mRNAs encoding respiratory sub-                  reliability of a given interaction. For S. cerevisiae, 5902 <br /> units but might also control the initiation of mitochondrial              proteins (from a total of about 6510) and 105728 separate <br />  Glatigny et al. BMC Systems Biology 2011, 5:173                                                                     Page 9 of 12 <br /> http://www.biomedcentral.com/1752-0509/5/173 <br />  <br />  <br />  <br />  <br /> interactions were listed in the last release (February 2009).      3) To be consistent with the APID results we had to <br /> APID is appropriate for our purpose because it qualifies        look for the functional environment of the new proteins <br /> the functional environment of a given protein. Looking at       that were present in BioGRID (list 2) and absent in APID <br /> the neighbours of one protein expands the resulting net-        (list 1). We compared output lists, list 1 from APID and <br /> work and gives a more reliable map thus allowing us to          list 2 from BioGRID, to find proteins that are absent in <br /> find proteins that could be involved in the same biological     APID list. We then added all physical interactions between <br /> process. Unfortunately, the APID database is not updated.       these &#8220;new&#8221; proteins and the proteins of list 1 and absent <br /> So to collect the data of recent experiments we used the        from the input list. In this way we obtained the list 3. <br /> BioGRID database (The Biological General Repository of             4) We then merged the lists 1, 2 and 3, and PPI that <br /> Interaction Datasets), release 3.1.72 (December 25 th ,         were identified by several different experiments were <br /> 2010), which is updated monthly and archives genetic and        taken into account only once to give a list of unique PPI. <br /> protein interaction data from model organisms and                  5) This final list was imported into Cytoscape 2.6.3 [35]. <br /> humans including S. cerevisiae. For the budding yeast the       The resulting network is un-weighted, since each PPI was <br /> complete coverage of the literature is maintained and Bio-      kept only once. As it is too large to yield any interesting <br /> GRID provides interactions to the Saccharomyces Gen-            information it is necessary to divide it into connected sub- <br /> ome Database (SGD). When the data were retrieved,               networks that might represent functional modules or pro- <br /> 163188 non-redundant physical and genetic interactions          tein sub-complexes. <br /> were listed in the base, corresponding to 6049 unique pro-         The network is un-weighted, i.e. no score is assigned to <br /> teins [20]. We eliminated the genetic interactions because      the edges. When a protein is very well studied, lots of <br /> not all the genes of complex III proteins were submitted        experiments describe its partners and one interaction <br /> to the high throughput experiments like the Synthetic           could be identified several times. As the number of occur- <br /> Genetic Array analysis [47], thus the inclusion of the          rences of an interaction is a criterion of reliability, it could <br /> genetic interactions would lead to a lack of balance in the     be advantageous to attribute a higher weight to the edges <br /> network. For example there are no interactions for COR1         that were the more frequent. However, the weighting <br /> in the Synthetic Genetic Array analysis as it was not           would introduce an important <span id='am-1' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Bias" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5'  ></span>bias</span>, as it would favour the <br /> included, but there are over one hundred interactions for       most studied proteins. <br /> QCR2. <br />    Complementary information was from the SGD [48].             Network clustering <br />                                                                 To find new proteins associated with assembly intermedi- <br /> Network modelling in Cytoscape                                  ates of complex III requires the identification of sub- <br /> Figure 1 shows the computational workflow from the              complexes or assembly modules. In a PPI network, sub- <br /> structural and assembly proteins of complex III to the          graphs of highly interconnected proteins can be consid- <br /> identification of new proteins implicated in complex III        ered as protein complexes or functional modules. Since <br /> biogenesis. The input list of the proteins already known        the same protein can interact with several sub-complexes <br /> to be involved in complex III is given in Table 1 (core         (this is the case of the assembly factors or chaperons) <br /> subunits, supernumerary subunits and assembly factors).         they must have the possibility of overlapping. <br /> A five-step approach was necessary to build the com-               The partition of the resulting PPI network into overlap- <br /> plete PPI network:                                              ping sub-graphs of highly interconnected nodes was per- <br />    1) By using the APID2NET plugin of Cytoscape [49]            formed with the plugin ClusterONE (Clustering with <br /> and the UniProtID of the proteins listed in Table 1             Overlapping Neighbourhood Expansion) [33] of Cytos- <br /> (input list), we collected the PPI list: list 1. It corre-      cape. This algorithm works by &#8220;growing&#8221; dense regions <br /> sponds to the direct PPI and to interactions between            out of small seeds guided by a quality function. The qual- <br /> two proteins having one direct interaction with a pro-          ity of a group is evaluated by the number of internal edges <br /> tein of the input list. Then, we collected all the physical     divided by the number of edges involving nodes of the <br /> interactions involving proteins of the same input list          group. Starting from a single seed (in our case: every <br /> from the BioGRID database. We obtained an output list,          node) the algorithm extends the group step by step with <br /> list 2, composed of PPI involving proteins named by             new edges if they increase the quality of a group. An edge <br /> their standard SGD name.                                        can be removed when its removal increases the quality of <br />    2) To use all the interactions of both APID and Bio-         its group. The process stops when it is not possible to <br /> GRID, we need to fuse the lists but before merging them         increase the quality of the groups by adding or removing <br /> it is necessary to unify the names. APID and BioGRID do         another edge. Finally, sub-graphs smaller than 3 or having <br /> not use the same identifiers (UniProt names and SGD             a density (number of edges within the cluster divided by <br /> names respectively). We used the SGD names, as they are         the number of theoretically possible edges), less than 0.25 <br /> the standard names used by the yeast community.                 and a p-value under than 0.05, are discarded and those <br />  Glatigny et al. BMC Systems Biology 2011, 5:173                                                                               Page 10 of 12 <br /> http://www.biomedcentral.com/1752-0509/5/173 <br />  <br />  <br />  <br />  <br /> overlapping significantly (overlap threshold equal 0.8) are   was homozygous for USB1-GFP and heterozygous nic96- <br /> merged to form a larger sub-graph; we chose a density of      CFP/NIC96. Fluorescence microscopy was performed <br /> 0.25 to get several overlapping sub-graphs having good        using a Zeiss Axioplan 2 microscope linked to a Cool <br /> quality. Indeed, we suppose that several assembly inter-      Snap <span id='am-38' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-39'  ></span><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Camera" datatype="xsd:string"></span>camera</span> (Princeton Instruments). <br /> mediates having proteins in common are required to            Cytochrome absorption spectra <br /> make complex III. The p-value of one cluster is computed      Cytochrome absorption spectra of whole cells were <br /> a posteriori with the one-sided Mann-Whitney U test per-      recorded at liquid nitrogen temperature after reduction by <br /> formed on the number of intra edges and external bound-       dithionite using a Cary 400 spectrophotometer (Varian, <br /> ary edges. An intra edge is between a cluster node and a      San Fernando, CA). Absorption maxima for the bands of <br /> node within the cluster. An external boundary edge is         cytochrome c, c1, b and a+aa3 are expected at 546, 552, <br /> between a cluster node and a node outside the cluster.        558 and 602 nm, respectively. <br />   When a large cluster cannot be further split into over- <br /> lapping sub-graphs of highly interconnected nodes, it can     Additional material <br /> be split into independent sub-graphs of highly intercon- <br /> nected nodes. The partition into independent sub-graphs           Additional file 1: Figure S1 - Sub-networks 1, 2 and 4 obtained by <br /> was made with the plugin MCODE (Molecular Complex                 partition of the whole complex III PPI network with ClusterONE. <br />                                                                   Image taken from Cytoscape. <br /> Detection) [32] of Cytoscape. This algorithm detects den- <br />                                                                   Additional file 2: Figure S2 - Sub-networks 5, 6 obtained by <br /> sely connected regions. First it assigns a weight to each         partition of the whole complex III PPI network with ClusterONE. <br /> node, corresponding to its local neighbourhood density.           Image taken from Cytoscape. <br /> Then, it recursively moves outward, including in the clus-        Additional file 3: Table S1 - List of Usb1p interactors present in the <br /> ter the nodes whose weight is above a given threshold.            network [54]. <br /> We used MCODE plugin with its default parameters.                 Additional file 4: Figure S3 - Localization of Usb1p. Cells expressing <br />                                                                   Usb1-GFP were grown in complete medium and stained with <br />                                                                   Mitotracker red CMX ROS. The figure shows the Usb1-GFP and <br /> Yeast genetics and biochemical analysis                           Mitotracker fluorescent images colored green and red respectively, and <br /> Strains, media, genetic methods, plasmids                         the merged image showing the co-localization of the Mitotracker and <br />                                                                   part of the Usb1-GFP signal. <br /> CW252 (mat alpha, ade2, ura3, his3, trp1, leu2) is the <br /> wild type yeast strain. Fermentable media contain 2% glu- <br /> cose while respiratory medium contains 2% glycerol and <br /> 2% ethanol. Yeast cells were transformed by the one-step    <br /> procedure [50]. The high copy vector pFL44 carries the        <br /> replication origin of the yeast 2 micron plasmid and the      <br /> URA3 marker. YepNB6 contains the wild type OXA1 <br />                                                            <br /> gene and YepML3 the wild type USB1 gene; both are         <br />                                                              <br /> cloned in the pFL44 vector.                             <br /> USB1 disruption and epitope tagging of Usb1p                   <br /> The USB1 gene was inactivated in the diploid strain        <br /> W303, using the PCR method described by Wach et al.          <br /> (1994) [51]: the entire ORF was replaced by the KanR    <br /> marker gene that confers resistance to the drug G418.         <br /> The heterozygous USB1/&#916;usb1::KanR diploids were              <br /> sporulated and micro-dissected. Only G418 sensitive <br />                                                             <br /> spores germinated confirming that the USB1 gene is            <br /> essential. C-terminal epitope tagged proteins were con- <br /> structed as described: Usb1-GFP using the plasmid              <br /> pFA6-5GA super bright that uses a G418 resistance mar-      <br /> ker [52] and Nic96-CFP using pBS4, which carries a            <br /> hygromycin B resistance marker (The Yeast Resource              <br /> Center, University of Washington). The strain expressing       <br /> Usb1-GFP shows growth indistinguishable from the wild          <br /> type on both fermentable and respiratory medium, show-           <br /> ing that the tagged protein is fully functional. The Nic96-     <br /> CFP strain shows a weak growth defect; because of this            <br /> the microscopic observations were made in a strain that        <br /> </body></html>