<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Interplay Protein L-Isoaspartyl <br /> Methyltransferase Activity Insulin-Like Signaling to <br /> Extend Lifespan <span id='am-30' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Caenorhabditis elegans</span> <br /> Shilpi Khare1*, Carole L. Linster1,3*, Steven G. Clarke1,2* <br /> 1 Department Chemistry Biochemistry, University <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-6'  ></span>California</span>-Los Angeles, Los Angeles, California, United States America, 2 Molecular Biology Institute, <br /> University California-Los Angeles, Los Angeles, California, United States America, 3 Laboratory Physiological Chemistry, Duve Institute Universite&#180; catholique <br /> de Louvain, Brussels, Belgium <br />  <br />  <br />  <br />      Abstract <br />      The protein L-isoaspartyl-O-methyltransferase functions initiate repair isomerized aspartyl asparaginyl residues <br />      spontaneously accumulate age variety organisms. <span id='am-37' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Caenorhabditis</span> elegans <span id='am-85' about='obo:NCBITaxon_333870' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_195229 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41087 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_71193 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Nematodes" datatype="xsd:string"></span><span id='am-88'  ></span><span id='am-89' property="oboInOwl:hasRelatedSynonym" content="Nematodes Berthold, 1827" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>nematodes</span> lacking pcm-1 <br />      gene encoding enzyme display normal lifespan phenotype standard laboratory growth conditions. <br />      However, significant defects development, egg laying, dauer survival, autophagy observed pcm-1 <br />      mutant nematodes deprived food exposed oxidative stress. Interestingly, overexpression repair <br />      enzyme Drosophila C. elegans extends adult lifespan thermal stress. In work, the <br />      involvement insulin/insulin-like growth factor-1 signaling (IIS) pathway PCM-1-dependent lifespan extension C. <br />      elegans. We demonstrate reducing levels DAF-16 downstream transcriptional effector IIS pathway by <br />      RNA interference reduces lifespan extension resulting PCM-1 overexpression. Using quantitative real-time PCR <br />      analysis, up-regulation DAF-16-dependent stress response genes PCM-1 overexpressor animals <br />      compared wild-type pcm-1 mutant nematodes mild thermal stress conditions. Additionally, similar other <br />      long-lived C. elegans mutants IIS pathway, including daf-2 age-1 mutants, PCM-1 overexpressor adult animals <br />      display increased resistance severe thermal stress, pcm-1 mutant animals survive long these <br />      conditions. Although observe higher accumulation damaged proteins pcm-1 mutant nematodes, basal level <br />      isoaspartyl residues detected wild-type animals reduced PCM-1 overexpression. Our results support a <br />      signaling role protein L-isoaspartyl methyltransferase lifespan extension involves IIS pathway, that <br />      independent function overall protein repair. <br />  <br />   Citation: Khare S, Linster CL, Clarke SG (2011) The Interplay Protein L-Isoaspartyl Methyltransferase Activity Insulin-Like Signaling Extend Lifespan <br />   Caenorhabditis elegans. PLoS ONE 6(6): e20850. doi:10.1371/journal.pone.0020850 <br />   Editor: Matt Kaeberlein, University Washington, United States America <br />   Received January 18, 2011; Accepted May 11, 2011; Published June 13, 2011 <br />   Copyright: &#223; 2011 Khare et al. <span id='am-52' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56'  ></span>This</span> open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported NIH Grant GM026020 AG032303 Ellison Medical Foundation Senior Scholar Aging Award. CLL supported <br />   Fonds le Recherche Scientifique (FNRS) European Union Seventh Framework Programme (FP7/2007&#8211;2013) grant agreement 276814. <br />   The funders role study design, <span id='am-101' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-104'  ></span><span id='am-105' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: clarke@mbi.ucla.edu (SGC); skhare@chem.ucla.edu (SK); carole.linster@uclouvain.be (CLL) <br />  <br />  <br />  <br /> Introduction                                                                              phenotype daf-2 pathway mutants abolished upon <br />                                                                                           additional mutation DAF-16 transcription factor [5,7,8]. <br />    A master regulator aging nematode Caenorhabditis elegans                        Another protein implicated C. elegans longevity protein <br /> is insulin/insulin-like growth factor-1 signaling (IIS) pathway.                      L-isoaspartyl (D-aspartyl)-O-methyltransferase (PCM-1), prod- <br /> The IIS pathway conserved organisms, including                                uct pcm-1 gene [9]. Known primarily protein repair <br /> <span id='am-45' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-46'  ></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span>Drosophila melanogaster</span> mammals [1]. Activation DAF-2                          role higher organisms, highly conserved enzyme specifi- <br /> insulin-like receptor C. elegans triggers successive phosphor-                     cally recognizes damaged aspartyl asparaginyl residues that <br /> ylation activation kinases including AGE-1 (<span id='am-42' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-43' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Class</span>                          accumulate age [10]. Following spontaneous deamidation <br /> 1A phosphatidylinositol-3-kinase) AKT (protein kinase B);                         and/or isomerization residues, protein backbones become <br /> activated AKT kinase subsequently phosphorylate FoxO                              kinked. The repair enzyme methylates a-carboxyl group of <br /> transcription factor DAF-16, excluding nucleus                            isoaspartyl residues, facilitating &#8216;&#8216;unkinking&#8217;&#8217; back- <br /> thereby rendering inactive [2,3]. However, kinase                            bone conversion isomerized residues normal <br /> cascade phosphorylation events deactivated, DAF-16                              L-aspartyl residues [10]. E. coli lacking isoaspartyl methyl- <br /> transcription factor enters nucleus, leading enhanced                          transferase gene display increased susceptibility various environ- <br /> expression genes involved longevity, dauer formation,                           mental stresses [11]. Knockout mice corresponding Pcmt1 <br /> stress resistance [2,3]. C. elegans mutants lacking functional                          gene accumulate altered proteins, growth retardation, and <br /> insulin-like receptor PI3-kinase (daf-2 age-1 mutants, res-                        subject premature death fatal seizures average <br /> pectively), display lifespan extensions spanning two-fold ten-fold                     age 42 days [12]. In plants, absence repair methyltrans- <br /> and enhanced stress resistance [4&#8211;6]. The long-lived                                 ferase shown involved decreased seed longevity <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                    1                                   June 2011 | Volume 6 | Issue 6 | e20850 <br />                                                                                                      Protein Repair Insulin-Like Signaling Aging <br />  <br />  <br /> and germination vigor <span id='am-7' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span>Arabidopsis</span> thaliana [13]. On contrary,         endogenous promoter pcm-1 mutant background. Three <br /> overexpression methyltransferase E. coli results in-            seven-fold higher isoaspartyl methyltransferase activities were <br /> creased survival heat shock [14] oxidative stress [15]             measured transgenic PL51 worm extracts compared wild- <br /> and extended lifespan thermal stress D. <span id='am-96' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-97'  ></span><span id='am-98' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>melanogaster</span> [16]           type N2 extracts prepared animals grown liquid culture <br /> and C. elegans [17].                                                         [26]. Strain PL54 contains similar plasmid (also pcm-1 <br />     Further characterization repair enzyme revealed              mutant background) PL51 strain encodes a <br /> interplay PCMT major intracellular signaling path-               PCM-1 protein mutated residues enzyme&#8217;s <br /> ways, including IIS pathway mitogen-activated protein            AdoMet cofactor binding site (G88V G90V). Extracts prepared <br /> kinase (MAPK) pathway. For example, Pcmt1 knockout mice                      PL54 nematodes did demonstrate methyltransferase <br /> showed activation IIS signaling pathway evidenced               activity [26]. The pcm-1 (qa201) mutant lacks exons 2&#8211;5 pcm- <br /> increased phosphorylation kinases including AKT, glycogen                 1 gene [24] alternative exon 1 partially overlapping <br /> synthase kinase-3 <span id='am-24' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-26'  ></span><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>beta</span> (GSK-3b), PDK-1 [18]. In relation              C10F3.4/mcp-1 gene [26]. This allele backcrossed times <br /> the MAPK pathway, PCMT1-knockdown human cells stimulated                     N2 background [17] obtain pcm-1 (qa201) strain, <br /> with EGF (epidermal growth factor) showed increased phosphor-                totally lacks methyltransferase activity. Protein L- <br /> ylation MAPK pathway components, including Raf-1, MEK,                    isoaspartyl O-methyltransferase activity measured in <br /> and ERK1/2 [19,20]. Additionally, Doyle colleagues                 strains used study values provided in <br /> that absence PCMT1 mammalian lymph node cells                      Table 1. The integrated DAF-16::GFP roller strain TJ356 was <br /> causes T-cell hyperproliferation increased phosphorylation            used test daf-16 RNAi efficacy monitor subcellular DAF- <br /> MAPK pathway components MEK1/2, ERK1/2, RSK1                          16 localization various conditions. This strain kindly <br /> response CD3 receptor stimulation [21]. In contrast, enhanced             given Dr. Brian Head (UCLA Department Biological <br /> PCMT1 expression induced valproic acid lithium               Chemistry). In absence daf-16 RNAi, confirmed expres- <br /> accompanied inactivation GSK-3b astrocytoma                     sion GFP-tagged DAF-16 protein cell types <br /> neuroblastoma cells [22,23]. These findings illustrate that,              (including hypodermis, neurons, intestine, gonad) except <br /> addition protein repair, PCMT1 directly indirectly            pharynx TJ356 strain [28]. With daf-16 RNAi, we <br /> involved regulation IIS pathway and/or MAPK                observed loss fluorescence daf-16 expressing regions <br /> pathway mammalian systems.                                                neurons. The efficiency pcm-1 RNAi tested with <br />     As opposed mice, effects deficiency protein L-            xtEx104 (UZ122) strain expressing PCM-1::GFP fusion <br /> isoaspartyl-O-methyltransferase resulting mutations              protein [25]. In absence pcm-1 RNAi, confirmed PCM-1 <br /> pcm-1 gene C. elegans apparent certain stress           expression (body wall, reproductive tissues, neurons) as <br /> conditions. In control conditions, pcm-1 mutants display similar           initially reported Gomez et al. [25] observed decrease <br /> lifespan brood size compared wild-type (N2) animals              fluorescence (except neurons) following pcm-1 RNAi <br /> [24]. Under starvation conditions, pcm-1 mutants display defects          treatment UZ122 strain. All strains kept 20uC or <br /> dauer formation survival L1 larvae survival [17,25].          25uC nematode growth medium plates streaked E. coli <br /> During dauer formation, pcm-1 mutants display reduced levels              OP50 (NGM + OP50) [27]. <br /> autophagy, indicative enhanced insulin/<span id='am-91' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-92' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-95'  ></span>TOR</span> signaling the <br /> absence PCM-1 [25]. Furthermore, oxidative stress con-              Lifespan analysis RNAi <br /> ditions, recently shown pcm-1 mutants display defects              The protocol used complete lifespan studies adapted <br /> in egg laying delayed larval development [26]. An                 Sutphin Kaeberlein [29]. First, eggs strains <br /> additional mutation daf-2 gene pcm-1 mutants restored          (N2, PL51, PL54) collected following hypochlorite <br /> a normal dauer lifespan [17] recovered stress defects larval          treatment gravid adults transferred NGM + OP50 <br /> development egg laying [26], indicating epistatic rela-               plates. Following approximately 48 h incubation 20uC, L4 <br /> tionship genes IIS pathway gene encoding <br /> the isoaspartyl methyltransferase. <br />     Given normal adult lifespan pcm-1 mutant C. elegans,        Table 1. PCM-1 activity measurements various C. elegans <br /> a surprise PCM-1 overexpression extends adult life-              strains. <br /> span two-fold compared wild-type nematodes mild <br /> thermal stress [17]. As previously shown lifespan <br />                                                                               Strain              Methyl Groups Transferred (pmol/min/mg protein) <br /> extension long-lived IIS pathway mutants DAF-16-depen- <br /> dent, work focused determining prolonged               N2                  0.6060.04 <br /> lifespan PCM-1 overexpressor animals involves tran-               pcm-1(qa201)        0.0160.03 <br /> scriptional activity DAF-16. Furthermore, importance the <br />                                                                               PL51                1.9760.26 <br /> protein repair function PCM-1 context been <br />                                                                               PL54                20.1060.01 <br /> addressed. <br />                                                                               Nematode protein extracts prepared liquid culture analyzed for <br /> Methods                                                                       PCM-1 methyltransferase activity described Kagan et al. (1997). Protein <br />                                                                               extracts incubated methyl-accepting peptide substrate KASA- <br /> Worm strains procedures                                                   isoD-LAKY (100 mM) 20 mM [14C]-AdoMet 30uC 2 h. Control <br />                                                                               incubations conducted KASA-isoD-LAKY not <br />    Standard procedures used maintain C. elegans strains               added reaction. Protein L-Isoaspartyl O-methyltransferase activity levels <br /> adapted Sulston Hodgkin [27]. The N2 strain                      given methyl groups transferred correction the <br /> obtained Caenorhabditis Genetics Center (St. Paul, MN).              background levels measured absence KASA-isoD-LAKY various <br /> Two transgenic strains (PL51 PL54) pcm-1                   C. elegans strains used lifespan analyses, thermotolerance assays, RT- <br />                                                                               qPCR analyses, isoaspartyl quantifications performed study. The <br /> (qa201) mutant strain used experimental procedures               results shown means (6 SD) separate activity measurements <br /> [17]. Strain PL51 non-integrated strain created              conducted set lysates. <br /> injection plasmid expressing pcm-1 gene                    doi:10.1371/journal.pone.0020850.t001 <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        2                                 June 2011 | Volume 6 | Issue 6 | e20850 <br />                                                                                                     Protein Repair Insulin-Like Signaling Aging <br />  <br />  <br /> larvae strain transferred NGM plates containing              eggs collected starved overnight M9 media obtain <br /> isopropyl b-D-1-thiogalactopyranoside 1 mM (pcm-1 RNAi)                  synchronized L1 larvae. The L1 larvae transferred to <br /> 2 mM (daf-16 RNAi) (IPTG; Anatrace, Inc. #I-1003), 0.025 mg/                   NGM + OP50 plates incubated 25uC 48 h until <br /> ml carbenicillin sodium salt (CB; Sigma #C-1389), 45 mM 5-                 nematodes reached day 1 adulthood. Day 1 adults were <br /> fluorodeoxyuridine (FUDR; Sigma #46875) streaked                      collected M9 media (n = 200, triplicate samples strain) <br /> RNAi-expressing bacteria (NGM + IPTG + CB + FUDR +                             stored 280uC. For total RNA extraction, animals were <br /> RNAi). Following transfer (day 0), lifespan plates incubated              homogenized 1 mL TRI ReagentH (Molecular Research Cen- <br /> at 25uC final scoring survival completed (survival           ter, #TR-118) 1% b-mercaptoethanol (BioUltra, Sigma, <br /> scored day animals dead). Animals              #63689) using Polytron PT2000 homogenizer 7 mm <br /> considered dead signs viability (motility, pharyngeal               generator. Phase separation RNA DNA proteins was <br /> pumping, response platinum wire prodding) detec.                   completed using bromochloropropane (Molecular Research Cen- <br /> Lifespan experiments repeated times. Averages mean               ter, #BP-151). RNA DNAse-treated purified <br /> and maximum survival determined examined sta-                     using RNeasy Fibrous Tissue Mini Kit (Qiagen; Cat #74704). <br /> tistical significance using ANOVA test significance.                   Following quantitation RNA measuring absorbance at <br />    The RNAi bacteria (either scrambled control daf-16 (I-5m24))             260 nm using NanoDrop spectrophotometer (the yield was <br /> were obtained Ahringer library [30] kindly provided                   generally 4 mg RNA 200 animals) verification of <br /> by Dr. Brian Head UCLA Department Biological                         absence protein contamination 260/280 nm ratio deter- <br /> Chemistry. The pcm-1 RNAi feeding clone, derived C.                   mination (between 1.8 2.0), sample run 1% TAE <br /> elegans ORFeome Library v1.1 [31], obtained Open                      agarose gel check RNA degradation. RNA integrity was <br /> Biosystems (Huntsville, AL). Several clones derived              noted observation sharp discrete 26S 18S <br /> original bacterial stock; clones #2 #3 used study              ribosomal bands absence leading smears, genomic <br /> are lacking exon 3 exon 2 pcm-1 (C10F3.5a) open              DNA, extra bands well. DNA-free RNA (2 mg a <br /> reading frame, respectively. The identity RNAi clones               total volume 40 ml) converted complementary DNA <br /> verified sequencing using sjj_R13H8.1 primers (listed                using oligo (dT) primers two-step RT-PCR procedure with <br /> www.wormbase.org sjj_R13H8.1_f: AGTACAGCAATTCC-                             heat denaturation (RETROscript Kit, Ambion #AM1710) <br /> CAAATGAAA sjj_R13H8.1_b: ATTGGATTTCGAAG-                                   according manufacturer&#8217;s instructions. <br /> fAAGTGGAT) daf-16 gene pL4440-dest-RNAi                           Quantitative real-time PCR qPCR primers. Ampli- <br /> universal primers (For: GTTTTCCCAGTCACGACGTT, Rev:                             fication reactions carried total volume 20 ml con- <br /> TGGATAACCGTATTACCGCC) pcm-1 gene. For life-                            taining SYBR Premix Ex Taq reagent (Clontech Laboratories, <br /> span experiments C. elegans, individual RNAi bacterial colonies             Inc., Takara Product #TAK RR041A), 0.3 mM gene-specific <br /> were grown LB + 50 mg/mL ampicillin 7 h 37uC; bacteria               forward reverse primers, cDNA corresponding 100 ng <br /> were seeded NGM + IPTG + CB + FUDR plates                        input RNA reverse transcriptase reaction. The reaction <br /> allowed grow 72 h L4 larvae transfer.                            conditions Opticon 2 (MJ Research) were: 95uC for <br />                                                                                5 min, followed 40 cycles 95uC 10 s, 60uC 20 s, 72uC <br /> Thermotolerance assays                                                         20 s. The fluorescence measured cycle 80uC. <br />     The protocol developed assay adapted Lithgow             Melting curves performed PCR assess pre- <br /> et al. [32], day 1 adult animals tested 37uC instead       sence unique final product, primer pair the <br /> of 35uC. L4 larvae strain transferred (20&#8211;30 animals            product reaction run agarose gel further <br /> per trial) NGM + OP50 plates placed 20uC overnight.             verify specificity confirm expected band size. <br /> The morning, plates transferred 37uC and, in-                 The gene expression data presented -fold change <br /> ffdicated times, nematodes scored viability (motility, pharyn-        mRNA transcript abundance transgenic mutant worm <br /> geal pumping, response platinum wire prodding). Animals                 strains, normalized endogenous reference genes (cdc-42 and <br /> were considered dead signs viability detectable. The           ama-1), relative wild-type N2 strain. Cdc-42 encodes the <br /> assay repeated triplicate significance survival time             RHO GTPase ama-1 encodes RNA polymerase II; genes <br /> points determined using ANOVA test significance.                   previously validated relatively reliable reference genes to <br />                                                                                use quantitative gene expression studies C. elegans [33]. The <br /> Microscopy Assays                                                              expression results similar normalized cdc-42 or <br />    To monitor subcellular localization DAF-16 va-                 ama-1 individually. Normalized -fold changes (and statistical sig- <br /> rious experimental conditions, TJ356 strain used. For              nificance) calculated using REST 2009 software (Qiagen). <br /> microscopy assays 25uC, progeny RNAi-fed TJ356 L4                       Quantitative real-time PCR primers newly designed for <br /> larvae grown 25uC adulthood control                  C. elegans genes ama-1 (forward primer: 59-CGGTCAGA- <br /> pcm-1 RNAi conditions scored DAF-16 localization                  AAGGCTATCGAG-39; reverse primer: 59-CCAACCTCCTG- <br /> as previously Henderson Johnson [28]. For                      ACGATTGAT-39), pcm-1 (forward primer: 59-TGTATGGCA- <br /> microscopy assays 37uC, progeny RNAi-fed TJ356 L4                    ATGATGGTTGG-39; reverse primer: 59-ACGACCGTCTCC- <br /> larvae grown manner, animals                 CTCAATAA-39), hsp-12.6 (forward primer: 59-GTGATG- <br /> kept 20uC. At day 1 adulthood, control RNAi-treated              GCTGACGAAGGAAC-39; reverse primer: 59-GGGAGGAA- <br /> pcm-1 RNAi-treated nematodes incubated 37uC                    GTTATGGGCTTC-39). Previously published primer pairs were <br /> hours DAF-16 subcellular localization determined [28].                 used C. elegans genes cdc-42 [33], sod-3, daf-16 [34], mtl-1, <br />                                                                                dod-3, F21F3.3, dod-22, dod-24 [35]. <br /> Reverse transcription quantitative real-time PCR <br />    RNA extraction reverse transcription. For                      Quantification protein substrates L-isoaspartyl <br /> stains tested (N2, PL51, PL54, pcm-1 (qa201)), gravid adults              methyltransferase C. elegans extracts <br /> treated 3 min final concentration 1.9 % sodium                       Nematodes strain (N2, PL51, pcm-1 (qa201)) were <br /> hypochlorite 0.3 M NaOH. After washes M9 media,                  grown 20uC 25uC 5 days liquid cultures <br />  <br />  <br />        PLoS ONE | www.plosone.org                                          3                             June 2011 | Volume 6 | Issue 6 | e20850 <br />                                                                                                    Protein Repair Insulin-Like Signaling Aging <br />  <br />  <br /> (S-media supplemented concentrated E. coli OP50) and <br /> harvested washes sucrose flotation previously described <br /> [24]. Preparation C. elegans cytosol determination pro- <br /> tein concentration previously [24]. To <br /> quantify level isoaspartyl methyltransferase substrates the <br /> various protein extracts, base-labile methyl ester groups were <br /> assayed extracts method modified Kagan <br /> et al. [24]. Specifically, nematode cytosolic extracts (125 mg pro- <br /> tein) incubated 50 ml reaction mixture following <br /> composition: 0.1 M Bis-Tris (pH 6.4) buffer, 0.67 mM S-adenosyl- <br /> L-[methyl-3H]methionine (Perkin Elmer, 78 Ci/mmol), 2.9 mg <br /> human recombinant L-isoaspartyl methyltransferase (this enzyme <br /> preparation specific activity 19.5 nmol min21 mg21 when <br /> the peptide KASA-isoD-LAKY used substrate). The <br /> reaction mixtures incubated 2 h 30uC were <br /> quenched addition 50 ml 2X Laemmli sample buffer <br /> containing 10% b-mercaptoethanol followed heating 70uC <br /> for 8 min. Approximately 40 mg polypeptides the <br /> above reaction mixtures separated gel electropho- <br /> resis using NuPAGEH Novex 4&#8211;12% SDS-PAGE gels (pH = 7) <br /> (Catalog # NP0335) NuPAGEH MES SDS Running Buffer <br /> (Catalog #NP002) Invitrogen. The level base-labile [3H]- <br /> methyl esters assayed 3 mm gel slices vapor diffusion <br /> method described Kagan et al. [24], 150 ml 1 N <br /> sodium hydroxide added gel slices scintillation <br /> vials incubated 38uC 24 hours counting. <br /> Methyl-accepting proteins quantified bio- <br /> logical replicate extracts worm strain tested. <br />  <br /> Results <br /> PCM-1 overexpression extends lifespan a <br /> DAF-16-dependent manner <br />     It previously shown PCM-1 overexpression <br /> extends lifespan adult nematodes mild thermal stress <br /> [17]. Nematodes expressing PCM-1 protein mutated <br /> AdoMet-binding site pcm-1 null background (PL54 strain) had <br /> a lifespan similar wild-type (N2) animals the <br /> same conditions indicating lifespan extension the <br /> overexpressor (PL51) strain dependent methyltransfer- <br /> ase activity PCM-1 [17]. We confirm lifespan <br /> extension observed PL51 strain dependent PCM-1 <br /> overexpression, lifespan PL51 adult nematodes fed with <br /> two different clones pcm-1 RNAi comparable wild- <br /> type nematodes treated control RNAi (Figure S1). The <br /> lifespan wild-type nematodes significantly affected by <br /> pcm-1 RNAi. <br />     To test PCM-1 overexpression effect a <br /> modulation IIS system, tested dependence this <br /> effect presence DAF-16, critical downstream tran- <br /> scription factor IIS. Specifically, grew N2, PL51, and <br /> PL54 strains bacteria expressing scrambled control RNAi or <br /> daf-16 RNAi used FUDR prevent egg laying cross <br /> generation sampling [29,36]. We confirmed PCM-1 overex- <br /> pressing animals display average increase 38% 51% <br /> in median maximum lifespan, respectively, compared wild-             Figure 1. PCM-1 overexpression extends lifespan daf-16- <br /> type animals, change survival observed            dependent manner mild thermal stress (256C). Lifespan <br /> wild-type animals nematodes expressing catalytically in-              analyses (three trials) completed 25uC nematode strains <br /> active PCM-1 enzyme (Fig. 1, Table 2). In strains tested,           (N2 wild-type (top panel), wild-type PCM-1 overexpressor strain (PL51) <br /> lifespan values obtained study presence FUDR              (middle panel), mutant PCM-1 overexpressor strain (PL54) (bottom panel)) <br />                                                                             grown conditions: control RNAi (solid lines) daf-16 RNAi <br /> were slightly reduced compared obtained previously          (dotted lines). L4 larvae transferred (Day 0) NGM plates streaked <br /> absence FUDR Banfield et al. [17], lifespan              RNAi bacteria survival scored day all <br /> patterns observed control RNAi bacteria (Fig. 1, Table 2).          nematodes dead. Statistical analysis data given Table 2. <br /> All lifespan extensions (including 50%, 25% 10% survival            doi:10.1371/journal.pone.0020850.g001 <br />  <br />  <br />       PLoS ONE | www.plosone.org                                        4                                June 2011 | Volume 6 | Issue 6 | e20850 <br />                                                                                                                  Protein Repair Insulin-Like Signaling Aging <br />  <br />  <br />  <br />  Table 2. PCM-1 overexpression extends C. elegans lifespan DAF-16-dependent manner mild thermal stress (25uC). <br />  <br />  <br />  Strain                 n             50% Survival (Hours)            25% Survival (Hours)            10% Survival (Hours)            Maximum Lifespan (Hours) <br />  <br />  N2                     152           2.360.9                         3.260.8                         4.261.1                         5.060.8 <br />  pcm-1 (qa201)          136           1.760.6(p = 0.04)**             2.860.6 (p = 0.003)**           3.361.0 (p = 0.01)**            4.061.0 <br />  PL51                   142           3.760.6                         4.560.5 (p = 0.04)*             5.260.6                         6.060.8 <br />  PL54                   123           2.761.0                         3.361.1                         3.961.2                         4.761.2 <br />  <br />  survival data Figure 1 determined using one-way ANOVA test. One asterisk (*, significance compared N2 control RNAi) asterisks <br />  (**, significance compared PL51 control RNAi) denote statistical significance indicated p-value. The total number adult nematodes (n) scored survival in <br />  replicates indicated. Values represent day (6 standard deviation) specified survival strain observed. <br />  doi:10.1371/journal.pone.0020850.t002 <br />  <br />  <br /> values) measured overexpressor strain statistically                       previous studies [28] (Figure S3). A similar nuclear localization <br /> significant compared wild-type (N2) animals (Table 2).                         observed DAF-16 pcm-1 RNAi incubation 37uC <br />    We methyltransferase-induced increase survival                        (Figure S3). Thus, PCM-1 deficiency does affect DAF-16 <br /> was greatly limited RNAi knockdown daf-16. Although                          localization thermal stress. This indicates decreased <br /> small lifespan decreases observed wild-type PL54                           resistance pcm-1 mutant acute thermal stress we <br /> strains daf-16 RNAi feeding compared control RNAi,                        observe mediated DAF-16. Further work be <br /> much pronounced reduction survival measured                         needed determine PCM-1 overexpression, unlike <br /> PL51 overexpressor strain conditions (Fig. 1 Table 2).                    PCM-1 deficiency, affects DAF-16 localization. <br /> In PL51 strain, median maximum lifespan extensions over <br /> wild-type values reduced 38% 51% control                              PCM-1 overexpression activates DAF-16-dependent gene <br /> RNAi 10 22% daf-16 RNAi. These results                             expression <br /> that lifespan extension observed PCM-1 overexpressing                              Since results lifespan analyses mild thermal <br /> animals partially dependent daf-16 expression levels.                   stress showed extended lifespan PCM-1 overexpressing <br />                                                                                        animals dependent daf-16 expression, wanted to <br /> PCM-1 overexpression enhances resistance severe                                     determine PCM-1 overexpression modulates transcrip- <br /> thermal stress                                                                         tion DAF-16 target genes previously linked nematode <br />    In addition displaying extended adult lifespans, daf-2 pathway                   longevity. cDNA synthesized total RNA extracted from <br /> mutants resistance variety stressors, including                      synchronized day 1 adults (grown 25uC) used <span id='am-57' about='obo:NCBITaxon_1178312' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_572719 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41011 obo:NCBITaxon_41197 obo:NCBITaxon_41012 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59'  ></span><span id='am-60' property="rdfs:label" content="Mea" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>mea</span>- <br /> severe thermal stress [32]. In testing PL51 PCM-1 over-                            sure expression levels pcm-1, daf-16, genes previously shown <br /> expressor strain resistance heat stress, day 1                    up-regulated DAF-16 (sod-3, hsp-12.6, dod-3, F21F3.3, and <br /> adults displayed increased thermotolerance 37uC compared                         mtl-1) [35,38], genes previously shown down- <br /> wild-type (N2) animals, significantly 25% survival time                    regulated DAF-16 (dod-22 dod-24) [35,38] real-time <br /> point (Fig. 2, Table 3). PL51 adult animals displayed 25% survi- <br /> val 4.560.5 h compared 3.260.8 h N2 animals. The <br /> enhanced thermotolerance appeared partially depen- <br /> dent functional methyltransferase cofactor-binding domain, <br /> as PL54 animals did significantly increased survival at <br /> any noted time point. Furthermore, pcm-1 mutant animals dis- <br /> played significant decrease thermotolerance compared <br /> to PL51 animals 50%, 25%, 10% survival time points <br /> (Table 3). <br />    Preliminary experiments using daf-16 RNAi indicates the <br /> increased resistance acute thermal stress PL51 over- <br /> expressor strain is, unlike lifespan extension 25uC, DAF-16 <br /> independent (Figure S2). To investigate DAF-16 dependency <br /> under thermal stress further, examined effect PCM-1 <br /> deficiency subcellular localization transcription <br /> factor 25uC 37uC. TJ356 nematodes (which express GFP- <br /> tagged DAF-16 protein) fed bacteria expressing scrambled <br /> control RNAi pcm-1 RNAi. Following incubation 25uC for <br />                                                                                        Figure 2. PCM-1 overexpression enhances resistance severe <br /> 48 hours 37uC hours, cellular localization                        heat stress (376C). Survival assays (three trials) completed for <br /> DAF-16 observed (Figure S3). In experiments 25uC,                           nematode strains (N2 wild-type, pcm-1 mutant (qa201), PCM-1 <br /> DAF-16::GFP gave diffuse signal indicating presence                       overexpressor strain (PL51), mutant PCM-1 overexpressor strain (PL54)). <br /> cytosol nucleus, concurring results shown                       Shown representative survival curves trial. L4 larvae were <br /> recently Kawli colleagues (37). No difference DAF-16                         transferred NGM + OP50 plates allowed grow 20uC <br />                                                                                        overnight. The day, animals transferred 37uC survival <br /> subcellular localization observed control RNAi- or <br />                                                                                        scored hours hours hour <br /> pcm-1 RNAi-fed animals. This agreement absence                       nematodes dead. Statistical analysis these <br /> a lifespan phenotype pcm-1 mutant 25uC. On control                           data given Table 3 replicates experiment. <br /> RNAi 37uC, DAF-16 expression nuclear, confirming                         doi:10.1371/journal.pone.0020850.g002 <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                 5                                   June 2011 | Volume 6 | Issue 6 | e20850 <br />                                                                                                                    Protein Repair Insulin-Like Signaling Aging <br />  <br />  <br />  <br />  Table 3. PCM-1 overexpression enhances resistance severe thermal stress (37uC) C. elegans. <br />  <br />  <br />  Strain                n             50% Survival (Hours)            25% Survival (Hours)               10% Survival (Hours)              Maximum Lifespan (Hours) <br />  <br />  N2                    152           2.360.9                         3.260.8                            4.261.1                           5.060.8 <br />  pcm-1 (qa201)         136           1.760.6 (p = 0.04)**            2.860.6 (p = 0.003)**              3.361.0 (p = 0.01)**              4.061.0 <br />  PL51                  142           3.760.6                         4.560.5 (p = 0.04)*                5.260.6                           6.060.8 <br />  PL54                  123           2.761.0                         3.361.1                            3.961.2                           4.761.2 <br />  <br />  Significance survival data Figure 2 determined using one-way ANOVA test. One asterisk (*, significance compared N2) asterisks <br />  (**, significance compared PL51) denote statistical significance indicated p-value. The total number adult nematodes (n) scored survival replicates <br />  indicated. Values represent hour (6 standard deviation) specified survival strain observed. <br />  doi:10.1371/journal.pone.0020850.t003 <br />  <br />  <br /> qPCR. Transcript levels experimental genes                                  observed PL51 PL54 strains. None five <br /> normalized transcript levels reference genes (cdc-42                         &#8216;&#8216;DAF-16 up-regulated&#8217;&#8217; genes significantly up-regulated <br /> and ama-1) cDNA samples included study. Cdc-                          versus wild-type mutant pcm-1 strain those <br /> 42 ama-1 previously shown relatively stably                                 genes (F21F3.3 mtl-1) actually significantly down- <br /> expressed various larval stages daf-2 pathway mutants [33]                          regulated strain (Fig. 3B, Table 4). As &#8216;&#8216;DAF- <br /> and genes considered reliable reference genes                         16 down-regulated&#8217;&#8217; genes tested, down-regulation also <br /> for expression studies context insulin-like signaling                         observed long-lived PCM-1 overexpressor dod- <br /> C. elegans.                                                                                22 dod-24 genes (about 2-fold decrease versus wild-type for <br />    As expected, pcm-1 transcript levels overexpressor strain                        genes, Fig. 3C, Table 4). On opposite hand, increases <br /> PL51 significantly higher wild-type strain (30-fold                       decreases expression measured genes <br /> increase versus wild-type; Fig. 3A). Pcm-1 transcript levels                          mutant overexpressor PL54 strain. In mutant pcm-1 <br /> reduced 500-fold versus wild-type pcm-1 deletion                           strain, dod-22 dod-24 expression down-regulated up- <br /> strain qa201 (Fig. 3A, Table 4). Surprisingly, mutant                               regulated versus wild-type, respectively (Fig. 3C, Table 4). We <br /> overexpressor strain PL54, pcm-1 transcript levels increased                      note regulation mRNA levels genes <br /> above wild-type levels PL51 strain. However, PCR                                 examined involve transcription factors addition to <br /> amplification exon 3 (i.e. region contains point                      DAF-16; changes expression pcm-1 affect daf-16- <br /> mutations decrease AdoMet binding) PL54 cDNA                                 independent pathways. <br /> subsequent sequencing confirmed strain expresses pcm-1 <br /> transcript encoding expected mutant methyltransferase. The                             PCM-1 overexpression lead decreased protein <br /> reason low expression levels mutated transcript                         damage C. elegans <br /> compared non-mutated PL51 strain remains                                    Accumulation damaged (isoaspartyl-containing) proteins has <br /> presently unclear. It noted <span id='am-106' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-107' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-110' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> used                         clearly demonstrated wide range molecular weights <br /> creating transgenic PL51 PL54 strains (in pcm-1 mutant                           various tissues isoaspartyl methyltransferase-deficient <br /> background) contained C10F3.4/mcp-1 gene,                                   mice [12]. To better characterize protein repair function of <br /> overlaps pcm-1 gene C. elegans genome antiparallel                           PCM-1 C. elegans, quantified level substrate proteins <br /> orientation [17,26]. We C10F3.4/mcp-1 gene                               L-isoaspartyl methyltransferase extracts prepared from <br /> overexpressed similar extents PL51 PL54 strains                         wild-type, PL51, pcm-1 (qa201) mutant strains. Therefore, we <br /> (data shown). This indicates difference pcm-1                              labeled worm protein extracts (prepared liquid cultures <br /> transcript levels strains results effect                    containing mixed larval adult population grown 5 days at <br /> mutations present pcm-1 transgene PL54 strain (but                       20uC) [3H]AdoMet recombinant human isoaspartyl <br /> the PL51 strain), large difference copy number                       methyltransferase separated polypeptides SDS <br /> of transgene DNA extrachromosomal array present                          polyacrylamide gel electrophoresis pH 7 analysis [3H]- <br /> strains. Despite lower pcm-1 transcript levels compared PL51                        methyl ester-containing proteins individual gel slices de- <br /> strain, PL54 strain remains important control strain                        scribed Methods section. <br /> exclude phenotypes observed PL51 strain                                In independent experiments, significantly higher <br /> C10F3.4/mcp-1 overexpression. The gene encodes GDP-                           levels base-labile [3H]-methyl ester groups pcm-1 (qa201) <br /> D-glucose phosphorylase proposed function                              mutant extracts compared wild-type extracts prepared from <br /> sanitizing nucleotide sugar pool [39]. No significant changes                       nematodes grown 20uC (Figs. 4 S4) proteins of <br /> daf-16 transcript levels observed transgenic                         molecular weights ranging 25 200 kDa. However, <br /> mutant strains compared wild-type levels (Fig. 3A, Table 4).                            reasons presently remain unclear, expe- <br />    The changes transcript levels measured DAF-16 target                             riments type did observe significant an <br /> genes correlated PCM-1 activity levels. Among &#8216;&#8216;DAF-16                            increase protein damage pcm-1 mutant (Figure S5, top <br /> up-regulated&#8217;&#8217; genes, genes tested up-regulated (4-                       panel). For assays performed extracts derived from <br /> 20-fold increases vs. wild-type) PCM-1 overexpressing animals,                          nematodes grown 25uC, did observe significantly higher <br /> the increase statistically significant sod-3, hsp-12.6,                      isoaspartyl levels pcm-1 mutant protein extracts wild- <br /> dod-3 (Fig. 3B, Table 4). Much pronounced increases                                type extracts (Figure S5, panel). Further work necessary <br /> transcript level versus wild-type level observed                            understand reason underlying variability results, <br /> &#8216;&#8216;DAF-16 up-regulated&#8217;&#8217; genes mutant overexpressor PL54                             indicate pcm-1 mutant animals display tendency to <br /> strain, hsp-12.6 gene, similar increases                          accumulate higher protein isoaspartyl levels. As pcm-1 mutant <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                   6                                   June 2011 | Volume 6 | Issue 6 | e20850 <br />                                                                                                       Protein Repair Insulin-Like Signaling Aging <br />  <br />  <br />                                                                                 C. elegans present lifespan phenotype 20uC or <br />                                                                                 25uC, results indicate isoaspartyl damage not <br />                                                                                 affect lifespan nematodes. In labeling experiments we <br />                                                                                 generally observed major peak radioactive methyl group <br />                                                                                 incorporation 21 kDa, level incorporation <br />                                                                                 independent PCM-1 activity. <br />                                                                                    We measured isoaspartyl levels protein extracts derived <br />                                                                                 PCM-1 overexpressing nematodes 20uC 25. Our <br />                                                                                 results showed decrease base-labile [3H]-methyl ester groups <br />                                                                                 basal wild-type levels PCM-1 overexpressing (PL51) <br />                                                                                 extracts derived nematodes grown 20uC 25uC. <br />                                                                                 On contrary, equal variably increased amounts damaged <br />                                                                                 proteins PL51 animals versus wild-type animals observed <br />                                                                                 (data shown). This indicates several-fold increase in <br />                                                                                 isoaspartyl methyltransferase activity wild-type activity levels <br />                                                                                 does decrease isoaspartyl levels accumulating in <br />                                                                                 wild-type animals suggests lifespan extension <br />                                                                                 displayed PCM-1 overexpressing animals does result from <br />                                                                                 decrease overall protein damage (in agreement above <br />                                                                                 conclusion increased protein damage does affect lifespan). <br />  <br />                                                                                 Discussion <br />                                                                                     We PCM-1 overexpression extends nematode <br />                                                                                 lifespan mild thermal stress (25uC) DAF-16-dependent <br />                                                                                 manner. This finding agreement previous results indi- <br />                                                                                 cating PCM-1 activity adult nematodes dispensable <br />                                                                                 control conditions, critically important under <br />                                                                                 various stress conditions specific larval stages [17,25,26]. Simil- <br />                                                                                 arly, preliminary experiments carried preparation this <br />                                                                                 study, lifespan extension wild-type observed PCM-1 <br />                                                                                 overexpressing nematodes grown 20uC. <br />                                                                                     By employing daf-16 RNAi, able reduc- <br />                                                                                 tion daf-16 expression partially reduced lifespan extension <br />                                                                                 displayed PL51 overexpressor strain 25uC. Daf-16 RNAi <br />                                                                                 decreased lifespan mutant overexpressor animals, but <br />                                                                                 reduction pronounced mimicked one <br />                                                                                 observed wild-type animals. A small negative effect daf-16 <br />                                                                                 RNAi lifespan N2 animals reported previously <br />                                                                                 [40,41]. Our results indicate PCM-1 isoaspartyl <br />                                                                                 methyltransferase extends lifespan 25uC partially by <br />                                                                                 directly indirectly downregulating IIS pathway thereby <br />                                                                                 promoting nuclear translocation DAF-16 concomitant up- <br />                                                                                 regulation target genes promoting survival mild thermal <br />                                                                                 stress (Figure 5). <br />                                                                                     The partial nature reduction PCM-1-dependent <br />                                                                                 lifespan extension daf-16 RNAi suggests PCM-1 also <br />                                                                                 act independently daf-16 increase lifespan. Possible targets <br />                                                                                 include transcription factor SKN-1 [42,43]. SKN-1 functions <br />                                                                                 parallel DAF-16 promote longevity direct <br />                                                                                 target IIS; response oxidative xenobiotic stress, SKN-1 <br />                                                                                 induces set stress response genes reduced IIS leads to <br />                                                                                 nuclear accumulation transcriptional activation SKN-1 in <br /> Figure 3. PCM-1 modulates expression levels DAF-16 target                    intestine [42]. Alternatively, complete suppression the <br /> genes 256C. Fold changes expression various genes              lifespan extension require daf-16 down-regulation the <br /> nematode strains PL51 (PCM-1 overexpressor), PL54 (mutant PCM-1 <br /> overexpressor), pcm-1 (qa201) mutant (pcm-1 null) versus wild-type          neurons. It shown RNAi does penetrate neurons <br /> (N2) shown. Total RNA extracted animals grown 25uC.             effectively C. elegans animals [44,45]; confirmed this <br /> Relative transcript levels versus N2 shown pcm-1 daf-16 genes       result finding daf-16 RNAi treatment does affect <br /> (panel A), DAF-16 up-regulated genes (panel B), DAF-16 down-                DAF-16 expression neurons. Although DAF-16 activation <br /> regulated genes (panel C). Ama-1 cdc-42 used endogenous             occurs neuronal intestinal cells, shown that <br /> reference genes normalization transcript levels of the <br /> genes involved expression study. Normalized -fold changes and <br />                                                                                 DAF-16 activity specifically intestine increases lifespan <br /> statistical significance determined using Qiagen REST 2009             daf-2;daf-16 double mutants substantially [46,47]. However, it <br /> program (*p#0.05, **p#0.001, ***p#0.0001) shown Table IV.                 shown lifespan extension age-1 mutants <br /> doi:10.1371/journal.pone.0020850.g003                                           requires neuronal intestinal DAF-16 activity [47,48]. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                           7                               June 2011 | Volume 6 | Issue 6 | e20850 <br />                                            Table 4. PCM-1 modulates expression DAF-16 target genes mild thermal stress (25uC). <br />  <br />  <br />                                                                                                 Fold Change       Fold Change         Fold Change       Fold Change        Average Fold      Standard      95% Confidence      P-value          P-value (Versus <br />                                           Strain                                  Gene          Versus N2         Versus N2           Versus N2         Versus N2          Change            Deviation     Interval            (Versus N2)      pcm-1 (qa201)) <br />  <br />                                                                                                 Trial #1          Trial #2            Trial #3          Trial #4 <br />                                           PL51 overexpressor strain               pcm-1         28.0              22.0                41.1                                 30.4              9.8           9.6 &#8211; 98.5          0.03* <br />                                                                                   daf-16        0.9               0.5                 3.7                                  1.7               1.8           0.02 &#8211; 135.6        0.9 <br />  <br />  <br />  <br />  <br /> PLoS ONE | www.plosone.org <br />                                           Daf-16 up-regulated Genes               sod-3         27.0              14.4                17.7                                 19.7              6.5           6.5 &#8211; 65.3          0.0001*** <br />                                                                                   hsp-12.6      5.0               2.6                 5.7                                  4.4               1.6           1.7 &#8211; 11.2          0.03* <br />                                                                                   dod-3         26.0              2.2                 3.9               16.0               12.0              11.2          1.1 &#8211; 35.9          0.001** <br />                                                                                   F21F3.3       55.7              3.2                 2.1               11.0               18.0              25.4          0.5 &#8211; 69.9          0.06 <br />                                                                                   mtl-1         24.3              1.9                 3.4               11.3               10.2              10.2          0.9 &#8211; 28.4          0.07 <br />                                           Daf-16 down-regulated Genes             dod-22        0.2               0.7                                                      0.4               0.4 <br />                                                                                   dod-24        0.4               0.8                                                      0.6               0.3 <br />                                           PL54 mutant overexpressor strain        pcm-1         0.9               1.0                 0.5                                  0.8               0.3           0.42 &#8211; 1.2          0.5              0.09 <br />                                                                                   daf-16        0.8               0.6                 1.8                                  1.1               0.6           0.02 &#8211; 163.3        0.8              0.9 <br />                                           Daf-16 up-regulated Genes               sod-3         3.3               2.8                 1.0                                  2.4               1.2           0.9 &#8211; 4.5           0.1              0.05 <br />  <br />  <br />  <br />  <br /> 8 <br />                                                                                   hsp-12.6      4.6               9.4                 1.5                                  5.2               4.0           1.6 &#8211; 10.8          0.04*            0.3 <br />                                                                                   dod-3         2.5               0.3                 1.7               5.7                2.6               2.3           0.3 &#8211; 5.9           0.3              0.1 <br />                                                                                   F21F3.3       9.2               1.0                 4.4               12.1               6.7               4.9           0.6 &#8211; 19.4          0.06             0.004* <br />                                                                                   mtl-1         2.0               0.3                 1.0               9.2                3.1               4.1           0.20 &#8211; 8.4          0.6              0.03* <br />                                           Daf-16 down-regulated Genes             dod-22        8.3               5.3                                                      6.8               2.1 <br />                                                                                   dod-24        5.5               6.9                                                      6.2               1.0 <br />                                           pcm-1 (qa201) mutant strain             pcm-1         0.001             0.006               0.000                                0.002             0.003         0.000 &#8211; 0.006       0.03* <br />                                                                                   daf-16        0.04              1                   14.6                                 5.2               8.1           0.04 &#8211; 29.7         0.9 <br />                                           Daf-16 up-regulated Genes               sod-3         0.6               3.3                 0.7                                  1.5               1.5           0.3 &#8211; 3.1           0.9 <br />                                                                                   hsp-12.6      1.3               7.3                 1.2                                  3.3               3.5           0.8 &#8211; 6.9           0.1 <br />                                                                                   dod-3         0.3               0.2                 0.8                                  0.4               0.3           0.2 &#8211; 1.1           0.2 <br />                                                                                   F21F3.3       0.04              0.1                 0.03                                 0.1               0.0           0.008 &#8211; 0.2         0.0001*** <br />                                                                                   mtl-1         0.2               0.1                 0.3                                  0.2               0.1           0.06 &#8211; 0.6          0.0001*** <br />                                           Daf-16 down-regulated Genes             dod-22        0.01              0.04                0.04                                 0.0               0.0           0.006 &#8211; 0.2         0.0001*** <br />                                                                                   dod-24        1.3               11.3                3.7                                  5.4               5.2           0.9 &#8211; 11.1          0.2 <br />  <br />                                           Normalized relative transcript levels indicated genes worm strains compared wild-type (N2) animals determined growth 25uC. Ama-1 cdc-42 used endogenous reference genes for <br />                                           normalization. Fold changes statistical parameters calculated using Qiagen REST 2009 program (*p#0.05, **p#0.001, ***p#0.0001). <br />                                           doi:10.1371/journal.pone.0020850.t004 <br />                                                                                                                                                                                                                                                                   Protein Repair Insulin-Like Signaling Aging <br />  <br />  <br />  <br />  <br /> June 2011 | Volume 6 | Issue 6 | e20850 <br />                                                                                                   Protein Repair Insulin-Like Signaling Aging <br />  <br />  <br />  <br />  <br /> Figure 4. Quantification methyl-accepting protein substrates C. elegans extracts recombinant human L-isoaspartyl <br /> methyltransferase 206C. Protein extracts N2 pcm-1 (qa201) nematode strains prepared mixed stage nematode <br /> population liquid culture 5 days 20uC incubated [3H]AdoMet recombinant human L-isoaspartyl <br /> methyltransferase described Methods section. Base-labile methyl esters quantified wild-type (N2 - hrPCM, dashed line open <br /> rectangles; N2 + hrPCM, solid line closed rectangles) pcm-1 (qa201) mutant (pcm-1 (qa201) - hrPCM, dashed line open triangles; pcm-1 <br /> (qa201) + hrPCM, solid line closed triangles) strains polypeptide fractionation SDS polyacrylamide gel electrophoresis. The migration <br /> positions molecular weight markers kDa indicated arrows (Biorad SDS-PAGE Standards Catalog #161-0317). Base-labile [3H]-methyl <br /> ester groups present peak 22 (not visible figure) N2 + hrPCM pcm-1 (qa201) + hrPCM 7285 cpm 3683 cpm, respectively. Similar <br /> results obtained independent experiments; representative experiment shown Figure S4. <br /> doi:10.1371/journal.pone.0020850.g004 <br />  <br /> Thus, possible small lifespan extension observe           rule effects PCM-1 DAF-16-independent pathways. <br /> with PCM-1 overexpression daf-16 RNAi treated adult                  Although bulk evidence points linkage PCM-1 <br /> nematodes result DAF-16 activation neurons. It           activity activity DAF-16, possible PCM-1 can <br /> known PCM-1 expressed primarily neurons, body                affect DAF-16 target genes transcription factors. <br /> wall, reproductive tissues C. elegans [25]. We speculate           Surprisingly, mutant overexpressor strain PL54 did not <br /> that complete suppression daf-16-dependent lifespan                overexpress pcm-1 extent wild-type overexpressor <br /> extension triggered PCM-1 overexpression                   strain PL51. We confirmed presence expected point <br /> achieved neuronal RNAi-sensitive worm strains.                           mutations pcm-1 transcript PL54 strain, con- <br />    We observed increase median survival severe             sequently believe mutant overexpressor strain expresses a <br /> thermal stress (37uC) overexpressor strain dependent         non-functional PCM-1 protein lower levels, activity assays <br /> on functional L-isoaspartyl methyltransferase. Preliminary                confirm PCM-1 activity strain. It was <br /> examination enhanced thermal stress resistance                interesting observe &#8216;&#8216;DAF-16 up-regulated&#8217;&#8217; <br /> PL51 strain indicates increased thermotolerance observed           genes significantly up-regulated compared wild-type <br /> is daf-16-independent (Figure S2). In agreement this,              animals PL54 mutant overexpressor strain. Except hsp- <br /> preliminary microscopy results pcm-1 RNAi does                12.6, approximate 5-fold induction observed in <br /> suppress nuclear localization DAF-16 heat shock 37uC           PL54 PL51 strains, levels induction other <br /> (Figure S3). Thus, increased survival PL51 nematodes              DAF-16 up-regulated genes were, however, lower PL54 <br /> 37uC involve DAF-16-independent mechanism.                          mutant overexpressor strain reached PL51 wild- <br />    Real-time qPCR analyses showed PCM-1-dependent induc-                  type overexpressor strain. These effects attributed the <br /> tion DAF-16 target genes sod-3, F21F3.3, mtl-1, dod-3         regulation hsp-12.6 HSF-1 transcription factor (50) by <br /> 25uC. The 20-fold induction sod-3 transcription                  effects C10F3.4/mcp-1 gene overlaps pcm-1 gene in <br /> approximately 10-fold 20-fold induction dod-3, F21F3.3,           antiparallel orientation overexpressed same <br /> and mtl-1 genes wild-type levels PCM-1 overexpressor            extent PL51 PL54 transgenic strains. <br /> strain 25uC solidifies hypothesis PCM-1 down-                Concerning &#8216;&#8216;DAF-16 down-regulated&#8217;&#8217; genes analyzed in <br /> regulating IIS signaling extend nematode longevity mild            expression study (dod-22 dod-24), results obtained in <br /> thermal stress. We note mtl-1 expression regulated         PCM-1 transgenic strains (about 2-fold down-regulation the <br /> by ELT-2 transcription factor [49]. At point,            PL51 strain opposed 6- 7-fold up-regulation in <br />  <br />  <br />        PLoS ONE | www.plosone.org                                       9                              June 2011 | Volume 6 | Issue 6 | e20850 <br />                                                                                                     Protein Repair Insulin-Like Signaling Aging <br />  <br />  <br />  <br />  <br /> Figure 5. The protein L-isoaspartyl O-methyltransferase (PCM-1) opposes insulin-like signaling extend C. elegans lifespan. Upon <br /> binding insulin-like ligand DAF-2 receptor, AGE-1 kinase activated phosphorylates membrane-bound phosphatidylinositol-4,5- <br /> bisphosphate (PIP2 ), leading formation phosphatidylinositiol-3,4,5-trisphosphate (PIP3) recruitment kinases PDK-1 AKT-1/AKT-2/ <br /> SGK-1 complex membrane. PDK-1 subsequently phosphorylates activates AKT/SGK complex, leading phosphorylation and <br /> inactivation FoxO transcription factor DAF-16. In absence phosphorylation, DAF-16 enters nucleus induce transcription genes <br /> implicated stress resistance longevity. Our results indicate overexpression protein repair enzyme PCM-1 leads down-regulation of <br /> the insulin-like signaling pathway subsequent activation DAF-16, leading increased transcription longevity genes. Our results show <br /> that, absence PCM-1 activity leads increased accumulation protein damage, overexpression decrease level of <br /> isoaspartyl-containing proteins wild-type levels. This suggests PCM-1 control lifespan C. elegans regulatory signaling <br /> function role bulk protein repair. <br /> doi:10.1371/journal.pone.0020850.g005 <br />  <br /> the PL54 strain) support hypothesis PCM-1 mediated              regulated p38/MAPK pathway independently DAF-16 <br /> downregulation IIS. However, dod-24 gene               [53]. Both studies partial/restrictive dependence on <br /> up-regulated similar extent PL54 mutant overexpressor             DAF-16, explaining presence original screen [38]. <br /> strain pcm-1 deletion strain, puzzling result            Given recent observations regulation dod-22 and <br /> the low expression measured dod-22 strain. It          dod-24 expression, effects PCM-1 expression levels of <br /> might noted status dod-22 dod-24 genes            genes not, partially mediated DAF-16. <br /> as &#8216;&#8216;downstream DAF-16&#8217;&#8217; targets presently controversial.               Further work needed clarify point. <br /> Initial work Murphy colleagues [38] identified dod-22                 To determine increase PCM-1 activity leads <br /> dod-24 genes acting downstream DAF-16 regulate                       increased protein repair C. elegans, measured protein <br /> longevity C. elegans. These genes shown down-              isoaspartyl levels wild-type, pcm-1 mutant, PL51 extracts by <br /> regulated long-lived TGF-b mutant adult animals                   using human recombinant isoaspartyl methyltransferase. Our <br /> regulation dependent DAF-16 [51]. Additionally, dod-               results increased isoaspartyl content pcm-1 mutant <br /> 22 dod-24 genes used control down-regulated DAF-16                extracts compared wild-type extracts prepared nematodes <br /> target genes recent study HCF-1, negative regulator              grown liquid culture, suggesting defect repair of <br /> DAF-16 [35]. Other recent studies [52,53] suggest, however,              isoaspartyl residues absence methyltransferase. In the <br /> regulation dod-22 dod-24 expression largely independent             PCM-1 overexpressing (PL51) extracts, apparent <br /> of DAF-16. Dod-24 expression shown negatively                       decrease isoaspartyl damage compared wild-type levels. <br /> regulated nuclear hormone receptor DAF-12 signaling                However, PL51 non-integrated transgenic strain, an <br /> from somatic reproductive tissues independently                    observed transmission 30%. Thus, estimate the <br /> DAF-16 [48] dod-22 expression shown                        liquid cultures prepared extracts contained <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        10                              June 2011 | Volume 6 | Issue 6 | e20850 <br />                                                                                                                  Protein Repair Insulin-Like Signaling Aging <br />  <br />  <br /> about 30% PCM-1 overexpressing transgenic animals 70%                              magnification. The panels diffuse, unlocal- <br /> non-transgenic pcm-1 mutant animals. Hence, transgenic                          ized DAF-16 expression animals fed control RNAi (n = 22), pcm-1 <br /> PCM-1 overexpressing animals decreased levels isoaspartyl                       RNAi clone #2 (n = 25), pcm-1 RNAi clone #3 (n = 19) at <br /> damage, change masked increased levels                       25uC. The panels nuclear DAF-16 expression in <br /> of damage 70% non-transgenic pcm-1 mutant nematodes.                            animals fed control RNAi (n = 8), pcm-1 RNAi clone #2 (n = 12), <br />    Taken together, results suggest increased accumula-                      pcm-1 RNAi clone #3 (n = 13) following incubation 37uC. <br /> tion protein damage (as observed mutant pcm-1 animals) does                      (PDF) <br /> not affect lifespan (no difference lifespan wild-type and <br /> pcm-1 mutant animals) and, conversely, PCM-1 overexpres-                          Figure S4 Quantification protein isoaspartyl levels in <br /> sion extend lifespan decreasing overall content                    additional C. elegans extracts 206C. Protein extracts were <br /> damaged proteins cell. Rather, PCM-1 act                               prepared liquid nematode cultures grown 20uC and <br /> signaling protein directly indirectly downregulate IIS                       analyzed isoaspartyl content described Figure 4. Data <br /> pathway turn expression stress resistance genes                           additional biological replicate lysates shown panel <br /> activation DAF-16 (Figure 5). One possible mechanism                       A (replicate 1) panel B (replicate 2). For panel A, base-labile <br /> a posttranslational regulatory carboxyl methylation inter-                       [3H]-methyl ester groups present peak 22 (not visible figure) <br /> mediate IIS protein phosphatase PCM-1. The                           N2+ hrPCM pcm-1 (qa201) + hrPCM 14,514 cpm and <br /> future identification potential regulatory targets PCM-1                    12,862 cpm, respectively. For panel C, base-labile [3H]-methyl <br /> will help elucidate mechanism observed interaction                          ester groups present peak 26 (not visible figure) N2+ <br /> between isoaspartyl methyltransferase, IIS, lifespan                       hrPCM pcm-1 (qa201) + hrPCM 247 cpm 380 cpm, <br /> control nematodes higher organisms.                                   respectively. A active human recombinant PCM used in <br />                                                                                        replicate 2 compared hrPCM used experiments <br /> Supporting Information                                                                 shown Figure 4 panel A Figure, same <br />                                                                                        increase base-labile [3H]-methyl ester groups labeled pcm-1 <br /> Figure S1 Pcm-1 RNAi reduces lifespan extension                                 (qa201) mutant extracts observed. <br /> PCM-1 overexpressing adult nematodes 256C. Lifespan                                 (TIF) <br /> analyses (one trial) completed 25uC (with FUDR) two <br /> nematode strains (N2 wild-type (top panel) wild-type PCM-1                         Figure S5 Quantification protein isoaspartyl levels in <br /> overexpressor strain (PL51) (bottom panel)) grown                             additional C. elegans extracts prepared nematodes <br /> conditions: control RNAi (solid lines) pcm-1 RNAi (clones                          grown 206C (top panel) 256C (bottom panel). <br /> #2 #3, dotted lines). L4 larvae transferred (Day 0)                        Protein extracts prepared liquid nematode cultures and <br /> NGM plates streaked RNAi bacteria survival scored                         analyzed isoaspartyl content described Figure 4. The <br /> every day nematodes dead. Wild-type (N2)                          averages data additional biological replicate lysates <br /> animals scored survival follows: control (n = 74)                      (prepared nematodes grown 20uC) shown top <br /> pcm-1 RNAi conditions (n = 77 pcm-1 RNAi clone #2; n = 76                          panel. Base-labile [3H]-methyl ester groups present peak 23 (not <br /> for pcm-1 RNAi clone #3). PL51 animals scored survival                        visible figure) N2+ hrPCM pcm-1 (qa201) + hrPCM are <br /> as follows: control (n = 55) pcm-1 RNAi conditions (n = 70                     7,851 cpm 19,384 cpm, respectively. In panel, <br /> pcm-1 RNAi clone #2; n = 70 pcm-1 RNAi clone #3).                                  data biological replicate lysates (prepared from <br /> (TIF)                                                                                  nematodes grown 25uC) strains shown. Base-labile <br />                                                                                        [3H]-methyl ester groups present peak 24 (not visible figure) <br /> Figure S2 Daf-16 RNAi does significantly affect                                    N2+ hrPCM pcm-1 (qa201)+ hrPCM 4,858 cpm and <br /> resistance severe heat stress (376C) PCM-1-                                      8,017 cpm, respectively. <br /> overexpressing animals. Survival assays (one trial)                               (TIF) <br /> completed nematode strains (N2 wild-type PCM-1 <br /> overexpressor strain (PL51)) control RNAi daf-16 RNAi <br /> conditions. L4 larvae transferred NGM + RNAi plates                        Acknowledgments <br /> were allowed grow 20uC overnight. The day, animals                          We grateful Dr. Brian Head, Dr. Qinfang Shen, Dr. Alex van <br /> were transferred 37uC survival scored hours                       der Bliek UCLA Department Biological Chemistry for <br /> for hours hour                           generously providing Ahringer library RNAi strains, strain TJ356 for <br /> nematodes dead. Wild-type (N2) animals scored survival                        experimentation, use dissection confocal microscopes. <br /> were follows: control (n = 27) daf-16 RNAi conditions                           Additionally, like thank Dr. Sabeeha Merchant the <br />                                                                                        UCLA Department Chemistry Biochemistry, use Opticon <br /> (n = 26). PL51 animals scored survival follows: control <br />                                                                                        machine software quantitative RT-PCR analysis. Finally, would <br /> (n = 15) daf-16 RNAi conditions (n = 20).                                          like thank colleagues, Dr. Jonathan Lowenson Dr. <br /> (TIF)                                                                                  Jonathan Wanagat, helpful advice, assistance statistics and <br /> Figure S3 Monitoring DAF-16 localization pcm-1                                    scientific guidance study. <br /> RNAi mild (256C) severe (376C) thermal <br /> stress. TJ356 nematode animals fed bacteria expressing                            Author Contributions <br /> either control (scramble) pcm-1 RNAi described                            Conceived designed experiments: SK CLL. Performed the <br /> Methods sections. DAF-16 localization monitored using                              experiments: SK CLL. Analyzed data: SK CLL SGC. Wrote the <br /> Nomarski differential interference contrast microscopy 200-fold                     paper: SK CLL SGC. <br />  <br />  <br /> References <br /> 1. Engelman JA, Luo J, Cantley LC (2006) The evolution phosphatidylinositol          2. Taguchi A, White MF (2008) Insulin-like signaling, nutrient homeostasis, life <br />    3-kinases regulators growth metabolism. Nature Reviews Genetics 7:            span. Annu Rev Physiol 70: 191&#8211;212. <br />    606&#8211;619.                                                                             3. Kenyon CJ (2010) The genetics aging. Nature 464: 504&#8211;512. <br />  <br />  <br />  <br />       PLoS ONE | www.plosone.org                                                  11                                   June 2011 | Volume 6 | Issue 6 | e20850 <br />                                                                                                                              Protein Repair Insulin-Like Signaling Aging <br />  <br />  <br />  4. Friedman DB, Johnson TE (1988) A mutation age-1 gene Caenorhabditis                  28. Henderson ST, Johnson TE (2001) Daf-16 integrates developmental and <br />     elegans lengthens life reduces hermaphrodite fertility. Genetics 118: 75&#8211;86.                   environmental inputs mediate aging nematode Caenorhabditis elegans. <br />  5. Kenyon C, Chang J, Gensch E, Rudner A, Tabtiang R (1993) A C. elegans                              Current Biology 11: 1975&#8211;1980. <br />     mutant lives twice long wild type. Nature 366: 461&#8211;646.                             29. Sutphin GL, Kaeberlein M (2009) Measuring Caenorhabditis elegans life span on <br />  6. Ayyadevara S, Alla R, Thaden JJ, Shmookler-Reis RJ (2008) Remarkable                               solid media. J Vis Exp 27 pii: 1152. doi: 10.3791/1152. <br />     longevity stress resistance nematode PI3K-null mutants. Aging Cell 7:                   30. Kamath RS, Fraser AG, Dong Y, Poulin G, Durbin R, et al. (2003) Systematic <br />     13&#8211;22.                                                                                             functional analysis Caenorhabditis elegans genome using RNAi. Nature 421: <br />  7. Dorman JB, Albinder B, Shroyer T, Kenyon C (1995) The age-1 daf-2                              231&#8211;237. <br />     genes function common pathway control lifespan Caenorhabditis                   31. Rual JF, Ceron J, Koreth J, Hao T, Nicot AS, et al. (2004) Toward improving <br />     elegans. Genetics 141: 1399&#8211;1406.                                                                  Caenorhabditis elegans phenome mapping ORFeome-based RNAi library. <br />  8. Ogg S, Paradis S, Gottleib S, Patterson GI, Lee L, et al. (1997) The Fork head                     Genome Res 14: 2162&#8211;2168. <br />     transcription factor DAF-16 transduces insulin-like metabolic longevity                    32. Lithgow G, White TM, Melov S, Johnson TE (1995) Thermotolerance and <br />     signals C. elegans. Nature 389: 994&#8211;999.                                                        extended life-span conferred single-gene mutations induced thermal <br />  9. Kagan RM, Clarke S (1995) Protein L-isoaspartyl methyltransferase                         stress. Proc Natl Acad Sci USA 92: 7540&#8211;7544. <br />     nematode Caenorhabditis elegans: genomic structure substrate specificity.                  33. Hoogewijs D, Houthoofd K, Matthijssens F, Vandesompele J, Vanfleteren JR <br />     Biochemistry 34: 10794&#8211;10806.                                                                      (2008) Selection validation set reliable reference genes for <br />                                                                                                        quantitative sod gene expression analysis C. elegans. BMC Mol Biol 22: 9. <br /> 10. Clarke S (2003) Aging war chemical biochemical processes: <br />                                                                                                    34. Zhang L, Jie G, Zhang J, Zhao B (2009) Significant longevity-extending effects of <br />     protein methylation recognition age-damaged proteins repair. <br />                                                                                                        EGCG Caenorhabditis elegans stress. Free Radic Biol Med 46: <br />     Ageing Res Rev 2: 263&#8211;285. <br />                                                                                                        414&#8211;21. <br /> 11. Visick JE, Cai H, Clarke S (1998) The L-isoaspartyl protein repair <br />                                                                                                    35. Li J, Ebata A, Dong Y, Rizki G, Iwata T, et al. (2008) Caenorhabditis elegans HCF- <br />     methyltransferase enhances survival ageing <span id='am-62' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-63'  ></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span>Escherichia coli</span> subjected                        1 functions longevity maintenance DAF-16 regulator. PLOS Biol 6: e233. <br />     secondary environmental stress. J Bacteriol 180: 2623&#8211;2629.                                    36. Mitchell DH, Stiles JW, Santelli J, Sanadi DR (1979) Synchronous growth and <br /> 12. Kim E, Lowenson JD, MacLaren DC, Clarke S, Young SG (1997) Deficiency                           aging Caenorhabditis elegans presence fluorodeoxyuridine. J Gerontol <br />     protein-repair enzyme results accumulation altered proteins,                           34: 28&#8211;36. <br />     retardation growth, fatal seizures mice. Proc Natl Acad Sci USA 94:                  37. Kawli T, Wu C, Tan M (2010) Systemic cell instrinsic roles Gqa sig- <br />     6132&#8211;6137.                                                                                         naling regulation innate immunity, oxidative stress, longevity in <br /> 13. Oge&#180; L, Bourdais G, Bove J, Collet B, Godin B, et al. (2008) Protein repair L-                     Caenorhabditis elegans. PNAS 107: 13788&#8211;13793. <br />     isoaspartyl methyltransferase1 involved seed longevity germina-                 38. Murphy CT, McCarroll SA, Bargmann CI, Fraser A, Kamath RS, et al. (2003) <br />     tion vigor Arabidopsis. Plant Cell 20: 3022&#8211;3037.                                               Genes act downstream DAF-16 influence lifespan Caenorhabditis <br /> 14. Kindrachuk J, Parent J, Davies GF, Dinsmore M, Attah-Poku S, et al. (2003)                         elegans. Nature 424: 277&#8211;283. <br />     Overexpression L-isoaspartate O-methyltransferase <span id='am-78' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-79' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-84'  ></span>Escherichia</span> coli increases             39. Adler LN, Gomez TA, Clarke SG, Linster CL (2011) A novel GDP-D-glucose <br />     heat shock survival mechanism independent methyltransferase activity.                      phosphorylase involved quality control nucleoside diphosphate sugar <br />     J Biol Chem 278: 50880&#8211;50886.                                                                      pool Caenorhabditis elegans mammals. J Biol Chem 286: press. <br /> 15. Verma P, Singh A, Kaur H, Majee M (2010) Protein L-isoaspartyl                                 40. Oh SW, Mukhopadhyay A, Svrzikapa N, Jiang F, Davis RJ, et al. (2005) JNK <br />     methyltransferase1 (CaPIMT1) chickpea mitigates oxidative stress-induced                      regulates lifespan Caenorhabditis elegans modulating nuclear translocation of <br />     growth inhibition Escherichia coli. Planta 231: 329&#8211;336.                                        forkhead transcription factor/DAF-16. Proc Natl Acad Sci USA 102: <br /> 16. Chavous DA, Jackson FR, O&#8217;Connor CM (2001) Extension Drosophila                             4494&#8211;4499. <br />     lifespan overexpression protein repair methyltransferase. Proc Natl Acad               41. Raices M, Maruyama H, Dillin A, Karlseder J (2005) Uncoupling longevity <br />     Sci USA 98: 14814&#8211;14818.                                                                           telomere length C. elegans. PLOS Genet 1: e30. <br /> 17. Banfield KL, Gomez TA, Lee W, Clarke S, Larsen PL (2008) Protein-repair                    42. Tullet JM, Hertweck M, An JH, Baker J, Hwang JY, et al. (2008) Direct <br />     hormone-signaling pathways specify dauer adult longevity dauer                             inhibition longevity-promoting factor SKN-1 insulin-like signaling C. <br />     development Caenorhabditis elegans. J Gerontol A Biol Sci Med Sci 63: 798&#8211;808.                  elegans. Cell 132: 1025&#8211;1038. <br /> 18. Farrar C, Houser CR, Clarke S (2005) Activation PI3K/Akt signaling                      43. Wang J, Robida-Stubbs S, Tullet JM, Rual JF, Vidal M, et al. (2010) RNAi <br />     pathway increased levels insulin receptor protein repair-deficient mice.                 screening implicates SKN-1-dependent transcriptional response stress <br />     Aging Cell 4: 1&#8211;12.                                                                                resistance longevity deriving translational inhibition. PLoS Genetics <br /> 19. Kosugi S, Furuchi T, Katane M, Sekine M, Shirasawa T, et al. (2008)                                6: e1001048. <br />     Suppression protein L-isoaspartyl (D-aspartyl) methyltransferase results                 44. Kamath RS, Martinez-Campos M, Zipperlen P, Fraser AG, Ahringer J (2001) <br />     hyperactivation EGF-stimulated MEK-ERK. Biochem Biophys Res Commun                              Effectiveness specific RNA-mediated interference ingested double- <br />     371: 22&#8211;27.                                                                                        stranded RNA Caenorhabditis elegans. Genome Biol 2, Available http:// <br /> 20. Furuchi T, Sakurako K, Katane M, Sekine M, Homma H (2010) The role                              genomebiology.com/2000/2/1/research/0002. Accessed 2000 December 20. <br />     protein L-isoaspartyl/D-aspartyl O-methyltransferase (PIMT) intracellular                   45. Timmons L, Court DL, Fire A (2001) Ingestion bacterially expressed dsRNAs <br />     signal transduction. Chem Biodivers 7: 1337&#8211;1348.                                                  produce specific potent genetic interference Caenorhabditis elegans. <br />                                                                                                         <br /> </body></html>