<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                        INVESTIGATION <br />  <br />  <br />  <br />  <br /> Coupling Between Noise Plasticity E. coli <br /> Gajinder Pal Singh1 <br /> School Biotechnology, KIIT University, Bhubaneswar 751024, Odisha, India <br />  <br />  <br />  <br /> ABSTRACT Expression levels genes vary different environmental conditions                                           KEYWORDS <br /> (&#8220;plasticity&#65533;?) genetically identical cells constant environment (&#8220;noise&#65533;?). Intriguingly,                           gene expression <br /> these measures gene expression variability correlate positively yeast. <span id='am-9' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-10' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> coupling                          noise <br /> was particularly strong genes speci&#65533;?c promoter architecture (TATA box high                                    expression <br /> nucleosome occupancy) weak genes high noise detrimental (e.g., essential genes),                                plasticity <br /> suggesting noise&#8211;plasticity coupling evolvable trait yeast constrain evolution gene                           expression <br /> expression promoter usage. Recently, similar genome-wide <span id='am-47' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49'  ></span><span id='am-50' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> noise plasticity                                   responsiveness <br /> available <span id='am-19' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-33'  ></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span>Escherichia coli</span>, providing opportunity study noise&#8211;plasticity correlation mech-                          gene expression <br /> anism prokaryote, follows fundamentally different mode transcription regulation eu-                              regulation <br /> karyote yeast. Using data, I signi&#65533;?cant positive correlation noise plasticity <br /> in E. coli. Furthermore, coupling highly in&#64258;uenced following: level expression; essentiality <br /> and dosage sensitivity genes; regulation speci&#65533;?c nucleoid-associated proteins, transcription factors, <br /> and sigma factors; involvement stress response. Many features analogous found <br /> to in&#64258;uence noise&#8211;plasticity coupling yeast. These results generality noise&#8211;plasticity <br /> coupling phylogenetically distant organisms suggest mechanism similar. <br />  <br />  <br />  <br />  <br /> Expression levels genes differ genetically identical cells           transcription regulation (Lehner 2010). Further, noise&#8211;plasticity cou- <br /> same environment. This variation &#8220;noise&#65533;? differs genes,                   pling lower genes detrimental &#65533;?tness, e.g., <br /> with genes noisier (Newman et al. 2006;                       essential dosage-sensitive genes (genes overexpression or <br /> Taniguchi et al. 2010; Silander et al. 2012). Cells respond                  underexpression inhibits growth) (Lehner 2010). These genes may <br /> different environmental conditions changes gene expression                     need different responsiveness conditions cannot <br /> levels, genes showing higher expression responsiveness                  high noise, lead inappropriate expres- <br /> &#8220;plasticity&#65533;? (Tirosh et al. 2006; Choi Kim 2009).                    sion levels cells. <br /> These measures expression variation cor-                       It proposed coupling noise plasticity <br /> relate yeast (Landry et al. 2007; Lehner 2008, 2010; Choi Kim                 general feature biological systems (Lehner Kaneko <br /> 2009), suggesting noise plasticity mechanisti-                  2011). In bacteria eukaryotes, mRNA production been <br /> cally coupled (Tirosh et al. 2009). It genes TATA                shown occur bursts, continuous manner, which <br /> box promoters high promoter nucleosome occupancy                       linked expression noise (Golding et al. 2005; Kaern et al. <br /> stronger coupling, suggesting association particular mode              2005; So et al. 2011), suggesting source transcriptional <br />                                                                                      noise similar bacteria eukaryotes. Thus, noise&#8211; <br /> Copyright &#169; 2013 Singh                                                               plasticity coupling observed yeast present bacteria. <br /> doi: 10.1534/g3.113.008540                                                           Recently, promoter-mediated noise measured majority <br /> Manuscript received August 20, 2013; accepted publication September 23, <br />                                                                                      (60%, 1522 promoters) <span id='am-35' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-36' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-39'  ></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span>Escherichia</span> coli promoters (Silander et al. <br /> 2013; published Early Online October 11, 2013. <br /> This open-access article distributed terms Creative           2012) using promoter library strain carries low-copy <br /> Commons Attribution Unported License (http://creativecommons.org/licenses/           number (three &#65533;?ve copies cell) plasmid E. coli promoter <br /> by/3.0/), permits unrestricted use, distribution, reproduction      region inserted upstream gene GFP. The expression variation <br /> medium, provided original work properly cited.                                gene single cells measured &#64258;ow cytometry. <br /> Supporting information available online http://www.g3journal.org/lookup/ <br /> suppl/doi:10.1534/g3.113.008540/-/DC1. <br />                                                                                      Given strong correlation mean expression level ex- <br /> 1 <br />  Corresponding author: School Biotechnology, KIIT University, Campus-XI, <span id='am-57' about='obo:NCBITaxon_320278' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_7114 obo:NCBITaxon_151208 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-60'  ></span><span id='am-61' property="rdfs:label" content="Patia" datatype="xsd:string"></span>Patia</span>,   pression variation, expression variation gene normal- <br />  Bhubaneswar 751024, Odisha, India. E-mail: gajinder.pal.singh@gmail.com             ized mean, referred &#8220;noise.&#65533;? These strains <br />  <br />  <br />  <br />                                                                                                                    Volume 3   | December 2013      |   2115 <br />  allowed authors ask promoter architecture in&#64258;uences          Essentiality dosage-sensitive stress- <br /> noise&#8211;plasticity coupling E. coli, independent features like chro-    responsive genes <br /> mosomal context, mRNA half-life, translation, protein half-life.        Gene essentiality growth data taken Baba et al. (2006). <br /> Interestingly, noise&#8211;plasticity coupling observed E. coli        I de&#65533;?ned slow-growth genes deletion leads the <br /> (Silander et al. 2012). The authors suggested re&#64258;ect          slowest 10% growth rate (excluding essential genes). Overexpression <br /> a fundamentally different mode transcription regulation          toxicity data taken Kitagawa et al. (2005) dosage- <br /> E. coli yeast, e.g., lack TATA box nucleosomes            sensitive genes de&#65533;?ned Singh Dash (2013). Genes be- <br /> E. coli. Although nucleosomes E. coli, genome        longing GO <span id='am-1' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-2' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> &#8220;response stress&#65533;? taken ECOCYC <br /> nevertheless highly compacted nucleoid-associated proteins               (Keseler et al. 2013). <br /> (NAPs) (Dillon Dorman 2010), in&#64258;uence noise <br /> (Silander et al. 2012) noise&#8211;plasticity coupling. I performed    RESULTS <br /> detailed analysis E. coli observed signi&#65533;?cant coupling. Impor- <br /> tantly, noise&#8211;plasticity coupling highly in&#64258;uenced mode          Weak signi&#65533;?cant correlation noise <br /> and extent transcription. The coupling observed essential     plasticity E. coli <br /> and slow-growth genes, high stress-responsive genes.          Noise&#8211;plasticity relationship investigated previously in <br /> These results similar yeast, suggesting           E. coli, correlation observed (Silander et al. 2012). <br /> noise&#8211;plasticity coupling common organisms mech-          Authors previous study took expression data unpub- <br /> anism similar.                                                  lished resource. These data consist un-normalized expression data <br />                                                                             different platforms laboratories. Thus, possible that <br /> MATERIALS AND METHODS                                                       quality expression data reason lack correlation <br />                                                                             observed authors. To test hypothesis, I took well-established <br /> Gene expression plasticity                                                  resource expression levels uniformly normalized and <br /> Gene expression data taken Many Microbe Microarrays           contain single-channel Affymetrix microarray data (Faith et al. <br /> Database (Faith et al. 2008), contains uniformly normalized           2008). For gene, I took SD mRNA expression levels across <br /> mRNA expression levels conditions. All data database         different conditions. The SD correlates signi&#65533;?cantly mean expres- <br /> are single-channel Affymetrix microarray only. For gene, I        sion levels (Pearson r = 0.40). To assess expression variation in- <br /> measured Standard Deviation (SD) mRNA expression levels              dependent mean expression level, I took residuals regression <br /> across different conditions (466 conditions; Build 6). The SD corre-        SD mean measure expression plasticity. I found <br /> lated signi&#65533;?cantly mean expression (Pearson r = 0.40). Thus,        weak signi&#65533;?cant correlation noise plasticity (Spearman <br /> assess expression variation independent mean expression, I took      rho = 0.14; P = 8E28; n =1456). As alternate measure expres- <br /> residuals regression SD mean measure expres-       sion plasticity, distance SD gene the <br /> sion plasticity. This measure expression plasticity did         running median SD calculated (see Materials Methods <br /> signi&#65533;?cant correlation mean expression level (Spearman P =             section). This measure indicated signi&#65533;?cant correlation between <br /> 0.3). As alternate measure expression plasticity, I calculated        noise plasticity (Spearman rho = 0.17; P = 1E29). Thus, noise&#8211; <br /> the distance SD gene running median            plasticity coupling exists E. coli, albeit weaker yeast (when <br /> a window size 200 (Supporting Information, Figure S1). The results       Spearman correlation = 0.3 observed) (Lehner 2010). <br /> remained essentially similar using alternate measure expression         Next, I investigated level expression in&#64258;uence <br /> plasticity (Table S1).                                                      noise&#8211;plasticity coupling. Genes placed &#65533;?ve equally sized <br />                                                                             nonoverlapping bins according expression level, correla- <br /> Gene expression noise                                                       tion noise plasticity calculated bin. Genes <br /> The promoter-mediated noise data E. coli taken                high mRNA expression levels showed high noise&#8211;plasticity cou- <br /> Silander et al. (2012). The authors study used E. coli pro-      pling (genes 20% expression Spearman rho = 0.23; <br /> moter library (Zaslaver et al. 2006), strain carries low-   P = 7E25; n = 291), lowly expressed genes did show <br /> copy number plasmid (three &#65533;?ve copies cell) E. coli          signi&#65533;?cant coupling (Figure 1). <br /> promoter region inserted upstream gene fast-folding GFP. <br /> Single-cell &#64258;uorescence measurements reliably performed            Noise&#8211;plasticity coupling disfavored essential, <br /> for 1522 1822 strains library. Most mRNA se-           slow-growth, dosage-sensitive genes <br /> quence identical construct, allowing expression noise      In yeast, observed noise&#8211;plasticity coupling much <br /> measured different promoter sequences independent             lower essential genes, genes deletion causes slow growth, <br /> many factors likely affect protein expression noise,      dosage-sensitive genes, stronger genes (Lehner <br /> such chromosomal context, mRNA half-life, translation, pro-          2010). The noise essential, slow-growth, dosage-sensitive genes <br /> tein half-life. There strong dependence expression variation       low coupled plasticity, lead to <br /> on mean expression level; thus, noise de&#65533;?ned deviation           inappropriate (very low high) levels proteins cells, <br /> from smoothed spline running median expression level vs. co-          detrimental &#65533;?tness. Consistent this, essential <br /> ef&#65533;?cient variation expression. This metric decouples expression       genes low noise yeast (Newman et al. 2006; Lehner 2008) and <br /> variation mean expression level (Silander et al. 2012).                E. coli (Silander et al. 2012). <br />                                                                                 I noise&#8211;plasticity coupling essential (P = 0.87) and <br /> Transcription-regulatory data                                               slow-growth genes (P = 0.19), coupling lower dosage- <br /> Transcription-regulatory data taken RegulonDB (Gama-              sensitive genes (Spearman rho = 0.1; P = 0.01; n = 562), the <br /> Castro et al. 2011). All factors analyzed RegulonDB,          genes coupling stronger (Spearman rho = 0.22; P = <br /> regulate 30 targets genes.                                          9E28; n =574) (Figure 2). Even highly expressed genes (which <br />  <br />  <br />  <br /> 2116 |     G. P. Singh <br />  Figure 1 Noise&#8211;plasticity coupling genes different expression levels. (A) Spearman correlation coef&#65533;?cients noise plasticity <br /> are shown genes divided &#65533;?ve equally populated nonoverlapping bins genes according expression levels (e.g., extreme left bin <br /> has 20% lowest expressing genes).  Bins signi&#65533;?cant correlation (P , 0.01). (B) Noise plasticity values plotted highest <br /> expression bin. Correlation P shown inset. <br />  <br />  <br /> show stronger overall coupling), essential, slow-growth, dosage-      0.006, respectively); therefore, lack correlation these <br /> sensitive genes signi&#65533;?cant coupling (P . 0.7),        genes attributable different expression levels. Although <br /> highly expressed nonessential, nonslow-growth, nondosage-         dosage-sensitive genes expressed signi&#65533;?cantly lower levels, re- <br /> sensitive genes increases coupling (Spearman rho =        moving lowest expressing dosage-sensitive genes (such the <br /> 0.46; P = 4E27; n = 112).                                                 mean expression dosage-sensitive genes signi&#65533;?cantly different <br />    Essential slow-growth genes expressed signi&#65533;?cantly          rest) did change noise&#8211;plasticity correlation (Table <br /> higher levels rest genes (two-tailed t test: P = 3E218    S2), indicating lower correlation dosage-sensitive genes is <br />  <br />  <br />  <br />  <br />                                                                                                    Figure 2 Noise&#8211;plasticity coupling genes <br />                                                                                                    different growth phenotypes. The corre- <br />                                                                                                    lation gene expression noise and <br />                                                                                                    gene expression plasticity shown essen- <br />                                                                                                    tial genes (A), genes deletion causes <br />                                                                                                    slow growth (B), genes overexpression <br />                                                                                                    toxic (C), nonessential, nonslow- <br />                                                                                                    growth, nondosage-sensitive genes (D). <br />                                                                                                    Spearman rank correlation coef&#65533;?cients (rho) <br />                                                                                                    P shown inset. <br />  <br />  <br />  <br />  <br />                                                   Volume 3 December 2013 |           Coupling Between Noise Plasticity E. coli | 2117 <br />  not attributable lower expression level. In summary,        &#8220;response stress,&#65533;? known regulated sigma <br /> results consistent model indicating noise&#8211;plasticity     38. I high coupling genes (Spearman rho = 0.34; P = <br /> coupling evolvable trait natural selection uncouple noise    6E25; n = 132). Conversely, sigma 38&#8211;regulated genes high <br /> and plasticity detrimental (Lehner 2010).                       coupling excluding stress-response genes (Spearman rho = <br />                                                                            0.40; P = 4E24; n = 77). <br /> Mode transcriptional regulation in&#64258;uences                                   There association noise levels noise&#8211;plasticity <br /> noise&#8211;plasticity coupling                                                  coupling genes regulated different regulators, e.g., genes regu- <br /> In yeast, observed genes particular promoter            lated H-NS IHF signi&#65533;?cantly noisy (Silander et al. <br /> architecture, speci&#65533;?cally TATA box high nucleosome          2012), H-NS shows high noise&#8211;plasticity coupling, suggesting <br /> occupancy, stronger noise&#8211;plasticity coupling (Lehner            mode transcription regulation, noise levels of <br /> 2010). TATA box&#8211;containing promoters frequently regulated           transcription itself, determines noise&#8211;plasticity coupling. Genes regu- <br /> the <span id='am-14' about='obo:NCBITaxon_441239' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_62784 obo:NCBITaxon_6994 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_70905 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_323649 obo:NCBITaxon_6993 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_33341 obo:NCBITaxon_50557 obo:NCBITaxon_6960 obo:NCBITaxon_6072'><span id='am-15' property="rdfs:label" content="Saga" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-18'  ></span>SAGA</span> complex associated stress response (Blake            lated H-NS, sigma 70, sigma 38 expressed signi&#65533;?- <br /> et al. 2006). Although equivalent TATA box promoters          cantly higher levels rest genes (two-tailed t test: P . <br /> in E. coli, global regulators (sigma factors, global        0.12); therefore, high coupling genes attributable to <br /> transcription factors) transcription mechanism differ-        higher expression levels. Genes regulated CRP, Fis, stress- <br /> ent in&#64258;uence noise&#8211;plasticity coupling.            responsive genes expressed higher levels; therefore, I calculated <br /> Similarly, nucleosomes E. coli, Nu-     noise&#8211;plasticity correlation removing highly expressed <br /> cleoid Associated Proteins (NAPs) H-NS (histone-like nucleoid      genes regulated regulators (thus eliminating difference <br /> structuring protein), Fis (factor inversion stimulation), IHF      expression level). For CRP, stress-responsive genes, stress- <br /> (integration host factor) involved compaction bacterial     responsive genes excluding sigma 38&#8211;regulated genes, correlation <br /> chromosomes in&#64258;uence gene expression (Dillon Dorman           remains high, Fis correlation nonsigni&#65533;?cant <br /> 2010). Therefore, I tested different modes transcription        (Table S2), suggesting high coupling independent expression <br /> regulation in&#64258;uence noise&#8211;plasticity coupling.                             level CRP-regulated genes stress-responsive genes, for <br />     Genes regulated different regulators marked difference      Fis-regulated genes. In summary, results highly analogous to <br /> noise&#8211;plasticity coupling (Table 1), H-NS&#8211;regulated genes show-       yeast, different mode transcription regulation <br /> ing highest coupling (rho = 0.65; P = 4E26; n = 42) (Figure 3A)        associated difference noise&#8211;plasticity coupling. <br /> and genes regulated NAP, IHF, showing cou- <br /> pling (P = 0.43). Similarly, genes regulated transcription factor       DISCUSSION <br /> CRP, sigma factor 38, sigma factor 70 high noise&#8211;plasticity       Noise gene expression levels responsiveness (plasticity) of <br /> coupling (Figure 3), rest global transcription      gene expression environmental changes fundamental aspects of <br /> sigma factors signi&#65533;?cant coupling (Table 1). Sigma 38       genes. Intriguingly, measures gene expression variation <br /> the stress-response sigma factor E. coli. To test high noise&#8211;   correlate yeast (Landry et al. 2007; Lehner 2008, <br /> plasticity coupling general feature stress-responsive genes, I     2010; Choi Kim 2009), mechanism coupling <br /> calculated noise&#8211;plasticity correlation genes GO class          remains unclear. Recently, genome-wide promoter-mediated noise <br />  <br />  <br />  <br /> n Table 1 Coupling expression noise expression plasticity different classes genes E. coli <br />   Class                                                                    Spearman Correlation Coef&#65533;?cient (rho)              P         Genes <br /> All                                                                                          0.14                        8.08E208        1456 <br /> Highly expressed                                                                             0.23                        7.27E205         291 <br /> Essential                                                                                   20.01                           0.94           99 <br /> Slow-growth                                                                                  0.07                           0.20          312 <br /> Dosage-sensitive                                                                             0.10                           0.01          562 <br /> Not essential, slow-growth, dosage-sensitive                                              0.22                        8.53E208         574 <br /> Highly expressed essential, slow-growth, dosage-sensitive                         0.46                        4.20E207         112 <br /> NAPs <br />   H-NS                                                                                        0.65                       6.56E206           42 <br />   Fis                                                                                         0.35                          0.01            53 <br />   IHF                                                                                         0.12                          0.43            47 <br /> Transcription factors <br />   CRP                                                                                        0.31                        1.19E204         147 <br />   <span id='am-42' about='obo:NCBITaxon_44596' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_106221 obo:NCBITaxon_6545 obo:NCBITaxon_6544 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_6552 obo:NCBITaxon_6553 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-43'  ></span><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Arca" datatype="xsd:string"></span>ArcA</span>                                                                                       0.25                           0.09           47 <br />   FNR                                                                                       20.01                           0.93           74 <br />   FUR                                                                                        0.31                           0.09           31 <br /> Sigma factors <br />   Sigma 70                                                                                   0.20                        4.28E207         634 <br />   Sigma 38                                                                                   0.41                        2.49E205         100 <br />   Sigma 24                                                                                   0.10                           0.24          153 <br />   Sigma 28                                                                                   0.30                           0.03           51 <br />   Sigma 54                                                                                  20.01                           0.97           30 <br />   Sigma 32                                                                                  20.11                           0.25          120 <br /> Stress response                                                                              0.38                        9.43E207         155 <br />  <br />  <br />  <br /> 2118 |     G. P. Singh <br />                                                                                                            Figure 3 Noise&#8211;plasticity coupling genes <br />                                                                                                           different transcription regulation mecha- <br />                                                                                                           nisms. The correlation gene expres- <br />                                                                                                           sion noise gene expression plasticity is <br />                                                                                                           shown genes expression regu- <br />                                                                                                           lated H-NS (A), sigma 38 (B), CRP (C), <br />                                                                                                           sigma 70 (D). Spearman rank correlation <br />                                                                                                           coef&#65533;?cients (rho) P shown inset. <br />  <br />  <br />  <br />  <br /> data E. coli available (Silander et al. 2012). However,             genes (which higher coupling), essential, slow-growth, and <br /> the authors did &#65533;?nd signi&#65533;?cant correlation noise               dosage-sensitive genes signi&#65533;?cant noise&#8211;plasticity correlation, <br /> plasticity (Silander et al. 2012). I revisited question using well-       rest highly expressed genes higher correlation of <br /> established normalized gene expression data (Faith et al. 2008)         0.46. These observations consistent noise&#8211;plasticity cou- <br /> calculate expression plasticity overall correlation be-         pling evolvable trait (Lehner 2010) E. coli. <br /> tween noise plasticity weak highly signi&#65533;?cant (rho = 0.14;              Although high coupling detrimental dosage-sensitive <br /> P = 8E28).                                                                     essential genes, advantageous subset <br />     I observed high noise&#8211;plasticity coupling highly expressed             genes. This true genes involved stress response, <br /> genes, lowly expressed genes (Figure 1). Noise gene             genes responsive allow physiological <br /> expression consists components, intrinsic noise extrinsic           adaptation allow cell-to-cell variability, allowing cells <br /> noise (Elowitz et al. 2002). Intrinsic noise attributable discrete   survive harsh conditions. Genes regulated stress-responsive <br /> nature biochemical process gene expression dom-           sigma factor 38 genes GO class &#8220;stress response&#65533;? high <br /> inant source noise lowly expressed genes, extrinsic             noise&#8211;plasticity coupling (Table 1). This analogous TATA box <br /> noise attributable &#64258;uctuations levels global factors      genes yeast, commonly associated stress response <br /> regulate gene expression, RNA polymerase, dominant            (Blake et al. 2006) high noise&#8211;plasticity coupling (Lehner <br /> source noise highly expressed genes (Newman et al. 2006;                2010). <br /> Taniguchi et al. 2010). Different environmental conditions likely              Given known in&#64258;uence transcription mechanism noise&#8211; <br /> to change levels global factors changing growth rate cell        plasticity coupling yeast (Lehner 2010), I asked also <br /> size (Klumpp Hwa 2008); therefore, I speculate high noise&#8211;            true E. coli. In E. coli, global transcription regulators control the <br /> plasticity coupling highly expressed genes attributable           majority regulated genes (Browning Busby 2004). These in- <br /> their high sensitivity extrinsic noise. Interestingly, expression level     clude NAPs, compact bacterial chromosomes in&#64258;uence <br /> was in&#64258;uence noise&#8211;plasticity coupling yeast (Lehner           access RNA polymerase transcription factors (Dillon <br /> 2010), re&#64258;ect difference regulation global factors         Dorman 2010). Genes regulated different regulators show <br /> in E. coli yeast.                                                          marked differences noise&#8211;plasticity coupling (Table 1), e.g., al- <br />     Noise&#8211;plasticity coupling signi&#65533;?cant essential             H-NS shows high coupling, IHF does not. H-NS acts as <br /> slow-growth genes weaker dosage-sensitive genes (Table             global transcriptional repressor important forming topo- <br /> 1). Although overall correlation small,            logically independent DNA microdomains (Hardy Cozzarelli <br /> enough evolution uncouple noise plasticity genes           2005). Similarly, genes regulated transcription factor CRP show <br /> it detrimental. Even considering highly expressed                high coupling, genes regulated global transcription <br />  <br />  <br />  <br />                                                        Volume 3 December 2013 |           Coupling Between Noise Plasticity E. coli | 2119 <br />  factors signi&#65533;?cant coupling (Table 1). It pro-                 <span id='am-52' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span>Escherichia coli K-12</span> integrated genetic sensory response units <br /> posed CRP act like NAP (Grainger et al. 2005). Con-                   (Gensor Units). Nucleic Acids Res. 39: D98&#8211;D105. <br /> sistent this, genes repressed CRP higher coupling             Golding, I., J. Paulsson, S. M. Zawilski, E. C. Cox, 2005 Real-time <br /> (rho = 0.46; P = 0.002; n = 42) genes activated CRP (rho =                  kinetics gene activity individual bacteria. Cell 123: 1025&#8211;1036. <br />                                                                                 Grainger, D. C., D. Hurd, M. Harrison, J. Holdstock, S. J. Busby, <br /> 0.27; P = 0.003; n = 113). High nucleosome promoter occupancy has <br />                                                                                     2005 Studies distribution Escherichia coli cAMP-receptor <br /> been associated high noise&#8211;plasticity coupling yeast. Our                   protein RNA polymerase E. coli chromosome. Proc. Natl. <br /> results that, analogous yeast, binding speci&#65533;?c NAPs                   Acad. Sci. USA 102: 17693&#8211;17698. <br /> promoters associated high noise&#8211;plasticity coupling E. coli.         Hardy, C. D., N. R. Cozzarelli, 2005 A genetic selection supercoiling <br />     Transcription regulation prokaryotes eukaryotes                 mutants Escherichia coli reveals proteins implicated chromosome <br /> been proposed follow fundamentally different logic (Struhl 1999),                structure. Mol. Microbiol. 57: 1636&#8211;1652. <br /> and difference noise&#8211;plasticity coupling yeast E. coli           Kaern, M., T. C. Elston, W. J. Blake, J. J. Collins, 2005 Stochasticity in <br /> was proposed re&#64258;ect difference (Silander et al. 2012). Our                  gene expression: theories phenotypes. Nat. Rev. Genet. 6: 451&#8211; <br /> results regarding in&#64258;uence essentiality promoter architec-               464. <br /> ture noise&#8211;plasticity coupling suggest transcription regulation         Keseler, I. M., A. Mackie, M. Peralta-Gil, A. Santos-Zavaleta, S. Gama-Castro <br />                                                                                     et al., 2013 EcoCyc: fusing model organism databases systems <br /> between eukaryotes prokaryotes similar than <br />                                                                                     biology. Nucleic Acids Res. 41: D605&#8211;D612. <br /> commonly appreciated. Indeed, mRNA production shown <br />                                                                                 Kitagawa, M., T. <span id='am-4' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-7'  ></span><span id='am-8' property="rdfs:label" content="Ara" datatype="xsd:string"></span>Ara</span>, M. Arifuzzaman, T. Ioka-Nakamichi, E. Inamoto et al., <br /> to occur bursts bacteria eukaryotes (Golding et al. 2005;              2005 Complete set ORF clones Escherichia coli ASKA library (a <br /> Kaern et al. 2005; So et al. 2011). The burst-like transcription                 complete set E. coli K-12 ORF archive): unique resources biological <br /> TATA box&#8211;contacting promoters associated high noise                   research. DNA Res. 12: 291&#8211;299. <br /> in gene expression eukaryotes (Blake et al. 2006),           Klumpp, S., T. Hwa, 2008 Growth-rate-dependent partitioning RNA <br /> high noise&#8211;plasticity coupling. Thus, burst-like transcription                polymerases bacteria. Proc. Natl. Acad. Sci. USA 105: 20245&#8211;20250. <br /> lead noise&#8211;plasticity coupling bacteria eukaryotes. The fact          Landry, C. R., B. Lemos, S. A. Rifkin, W. J. Dickinson, D. L. Hartl, <br /> that noise&#8211;plasticity coupling present yeast E. coli (and,             2007 Genetic properties in&#64258;uencing evolvability gene expression. <br /> hence, elephants) (Friedmann 2004) suggests com-                 Science 317: 118&#8211;121. <br /> mon feature gene expression organisms.                                Lehner, B., 2008 Selection minimise noise living systems im- <br />                                                                                     plications evolution gene expression. Mol. Syst. Biol. 4: 170. <br />                                                                                 Lehner, B., 2010 Con&#64258;ict noise plasticity yeast. PLoS Genet. <br /> ACKNOWLEDGMENTS                                                                     6: e1001185. <br /> I thank Dr. Debasis Dash, Dr. Beena Pillai, Dr. Kausik Chakraborty,             Lehner, B., K. Kaneko, 2011 Fluctuation response biology. Cell. <br /> Meenakshi Anurag, Dhirendra Kumar, Dr. Amit Kumar Yadav,                            Mol. Life Sci. 68: 1005&#8211;1010. <br /> Ankita Narang, Rishi Das Roy, Abhilash Gangadharan                Newman, J. R., S. Ghaemmaghami, J. Ihmels, D. K. Breslow, M. Noble et al., <br /> critical comments manuscript.                                                2006 Single-cell proteomic analysis S. cerevisiae reveals archi- <br />                                                                                     tecture biological noise. Nature 441: 840&#8211;846. <br /> LITERATURE CITED                                                                Silander, O. K., N. Nikolic, A. Zaslaver, A. Bren, I. Kikoin et al., 2012 A <br /> Baba, T., T. Ara, M. Hasegawa, Y. Takai, Y. Okumura et al.,                         genome-wide analysis promoter-mediated phenotypic noise in <br />     2006 Construction Escherichia coli K-12 in-frame, single-gene                Escherichia coli. PLoS Genet. 8: e1002443. <br />     knockout mutants: Keio collection. Mol. Syst. Biol. 2: 2006.0008.       Singh, G. P., D. Dash, 2013 Electrostatic Mis-Interactions Cause <br /> Blake, W. J., G. Balazsi, M. A. Kohanski, F. J. Isaacs, K. F. Murphy et al.,        Overexpression Toxicity Proteins E. coli. PLoS ONE 8: e64893. <br />     2006 Phenotypic consequen <br /> </body></html>