<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Second-Site Suppression RNase E Essentiality Mutation the <br /> deaD RNA Helicase <span id='am-48' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58'  ></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span>Escherichia coli</span> <br /> Masaru Tamura,a Johan A. Kers,a Stanley N. Cohena,b <br /> Departments Geneticsa Medicine,b School Medicine, Stanford University, Stanford, <span id='am-14' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-15'  ></span><span id='am-16' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="California" datatype="xsd:string"></span>California</span>, USA <br />  <br /> <span id='am-64' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-67'  ></span><span id='am-68' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span>Escherichia</span> coli cells normally require RNase E activity propagate form colonies. Using random Tn10 insertion mutagen- <br /> esis, screened second-site suppressor mutations restore colony-forming ability (CFA) E. coli cells lacking RNase E <br /> function mutations separate chromosomal loci phenotype. Restoration CFA mutations in <br /> two genes identified observed nutrient-poor medium, effects mutation ATP-dependent <br /> RNA helicase DeaD medium independent. Suppression rne mutant phenotype inactivation deaD partial, as <br /> rne deaD doubly mutant bacteria greatly prolonged generation time grew filamentous chains liquid medium. <br /> Moreover, CFA restoration deaD inactivation requires normal expression endogenous rng gene dou- <br /> bly mutant rne deaD cells. Second-site suppression deaD mutation attributable specifically ablation helicase <br /> activity DeaD reversed adventitious expression RhlE RNase R, unwind double-stranded <br /> RNA. Our results suggest previously unsuspected role RNA secondary structure determinant RNase E essentiality. <br />  <br />  <br />  <br /> R    Nase E discovered initially Escherichia coli endori- <br />      bonuclease processes 9S rRNA (21). More 3 decades <br /> of subsequent studies shown RNase E key role <br />                                                                                       published attempt identify second-site suppressors RNase E <br />                                                                                       essentiality. Using Tn10 insertion-based screen generate ran- <br />                                                                                       dom chromosomal mutations &#9004;rne strain reversibly <br /> in processing structural RNAs (5S rRNA [21], 16S                         complemented plasmid expressing N-Rne (34), identified <br /> rRNA [30, 51], tRNA [44]), global degradation                           multiple chromosomal sites inactivation restored CFA to <br /> mRNA (7, 27, 38, 47), control plasmid DNA replication                       rne deletion mutant bacteria N-Rne-expressing plasmid <br /> (31), maturation/processing variety small cata-                    inactivated lost. These experiments described below. <br /> lytic RNAs (for example, ssrA RNA [32] M1 RNA subunit <br /> of RNase P [33]). The ribonucleolytic activity RNase E resides                  MATERIALS AND METHODS <br /> its N-terminal half, N-Rne (34), C-terminal half                          Strains <span id='am-104' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-105' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-106' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span>. The strains used study listed Table 1 <br /> (CTH) includes docking sites proteins form multicom-                       (see Fig. S1 supplemental material). MT2009 obtained in- <br /> ponent complex termed &#8220;degradosome&#65533;? (35, 43). DNA se-                             troduction pBAD-NRNE expressing N-Rne, tagged hexahisti- <br /> quence analysis predicted orthologs E. coli RNase E                       dine tag C terminus control BAD promoter, <br />                                                                                       strain N3433 rne(pRNG3) (27). Selection incoming plasmid <br /> exist dozens evolutionarily disparate bacterial <span id='am-44' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-45' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> <br />                                                                                       growing transformants containing plasmids (pBAD-NRNE and <br /> (28, 45).                                                                             pRNG3) 60 generations presence kanamycin 0.1% <span id='am-34' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>ara</span>- <br />     In E. coli, actions RNase E essential colony-form-                 binose IPTG (isopropyl-&#65533;?&#65533;-D-thiogalactopyranoside) resulted in <br /> ing ability (CFA) (2), transfer liquid cultures rne(Ts)                     displacement pRNG3 pBAD-NRNE. MT2000 &#9004;rng constructed <br /> mutant bacteria RNase E-inactivating temperature prevents                       transducing P1 (&#9004;rne::Cm) strain KSL1004 (27) harboring pBAD- <br /> normal cell division results formation elongated                        RNE. To obtain MT3001, &#9004;deaD774::Km derived JW5531-1 <br /> filamentous structures containing multiple connected cell bodies                      (CGSC11447) (5) introduced N3433 &#9004;rne(pLAC-RNE2) by <br /> (22). Similarly, rne null mutant E. coli cells complemented trans                  transduction, rne plasmid replaced pSC109. <br /> by full-length RNase E N-terminal catalytic region soon                         ASKA library plasmids (26) used construct ampicillin (Ap)- <br />                                                                                       resistant plasmids (Table 1) complementation Tn10-mutated <br /> cease divide synthesis complementing enzyme is <br />                                                                                       strains. The bla fragment amplified PCR using primers bla 5 <br /> turned (48). However, CFA restored rne dele-                       prime (AgeI) (5=-ATCGACCGGTGAACGTCGTGCTGACGCTTC-3=) <br /> tion mutants overproduction RNase G (13, 16, 27, 48),                        bla 3 prime (AgeI) (5=-ATCGACCGGTCAGAGCAAGAGATTACG <br /> RNase E paralog (30, 51) structurally resembles amino-                       CGC-3=), pACYC177 (11) DNA template. PCR products were <br /> terminal catalytic region RNase E (34). Such complemented                          digested AgeI, gel purified, used ligation reactions. Aska-de- <br /> cells grow slowly rne&#11001; bacteria, &#11350;90% decrease                       aD(Ap) constructed ligating BanII-HindIII fragment from <br /> in CFU optical density unit (CFU/ml/OD unit) (28, 48),                        Aska-deaD(Cm), containing deaD, larger BanII-HindIII frag- <br /> retain filamentation phenotype characteristic RNase E-de- <br /> ficient mutants liquid culture (48). Whereas filamentation <br /> of RNase G-overproducing &#9004;rne mutants reversed overex-                             Received 8 December 2011 Accepted 27 January 2012 <br /> pression FtsZ, remedies imbalanced FtsZ/FtsA pro-                           Published ahead print 10 February 2012 <br /> tein ratio, cells remain defective rate propagation                     Address correspondence Stanley N. Cohen, sncohen@stanford.edu. <br /> (48).                                                                                    Supplemental material article http://jb.asm.org/. <br />     Previous efforts restore CFA rne deletion mutant bacteria                      Copyright &#169; 2012, American Society Microbiology. All Rights Reserved. <br /> by screening plasmid library expressing E. coli chromosomal                            doi:10.1128/JB.06652-11 <br /> genes detected gene rne, no <br />  <br />  <br /> 0021-9193/12/$12.00    Journal Bacteriology   p. 1919 &#8211;1926                                                                                              jb.asm.org   1919 <br />  Tamura et al. <br />  <br />  <br />  <br /> TABLE 1 Bacterial strains plasmids used study <br /> E. coli strain plasmid                          Descriptiona                                                                         Reference source <br /> Strains <br />    N3433                                           lacZ43(Fs) LAM&#11002; relA1 spoT1 thi-1                                                    44 <br />    MT2000                                          Same N3433, rne::cat(pBAD-RNE)                                                27, 48 <br />    MT2009                                          Same N3433, rne::cat(pBAD-NRNE)                                               <span id='am-39' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> study <br />    MT2000 &#9004;rng                                     Same MT2000, &#9004;rng                                                             This study <br />    MT2000(deaD::Tn10)                              Same MT2000, deaD::Tn10                                                       This study <br />    MT2009(ydfV::Tn10)                              Same MT2009, ydfV::Tn10                                                       This study <br />    MT2000(ihfA-btuC::Tn10)                         Same MT2000, ihfA-btuC::Tn10                                                  This study <br />    MT3001                                          Same N3433, rne::cat &#9004;deaD774::Km(pSC109)                                     This study <br />  <br /> Plasmids <br />   pBAD-RNE                                         pSC101 ori Kmr; rne PBAD                                                       27, 48 <br />   pBAD-NRNE                                        pSC101 ori Kmr; N-rne PBAD                                                     29 <br />   pLAC-RNE2                                        pSC101 ori Apr; rne PlacUV5                                                    27 <br />   pACYC-duet-CsdAE174A                             P15A ori Cmr; helicase KO mutant deaD(DAAD) PT7-lac                            4 <br />   pCA24N(Ap)                                       ColE1 ori Apr                                                                        This study <br />   Aska-deaD(Ap)                                    ColE1 ori Apr; wild-type deaD(DEAD) PT5-lac                                    This study <br />   Aska-DAAD(Ap)                                    ColE1 ori Apr; helicase KO mutant deaD(DAAD) PT5-lac                           This study <br />   Aska-ydfV(Ap)                                    ColE1 ori Apr; ydfV PT5-lac                                                    This study <br />   Aska-relB(Ap)                                    ColE1 ori Apr; relB PT5-lac                                                    This study <br />   Aska-btuC(Ap)                                    ColE1 ori Apr; btuC PT5-lac                                                    This study <br />   Aska-btuE(Ap)                                    ColE1 ori Apr; btuE PT5-lac                                                    This study <br />   Aska-btuD(Ap)                                    ColE1 ori Apr; btuD PT5-lac                                                    This study <br />   Aska-rhlE(Ap)                                    ColE1 ori Apr; rhlE PT5-lac                                                    This study <br />   Aska-rnr(Ap)                                     ColE1 ori Apr; rnr PT5-lac                                                     This study <br /> a <br />     KO, knockout. <br />  <br />  <br />  <br />  <br /> ment pCA24N(Ap). pCA24N(Ap) constructed inserting              showed similar colony forming efficiencies ara(&#11001;) ara(&#11002;) <br /> bla fragment AgeI site pCA24N(Cm). Aska-DAAD(Ap)               plates identified. <br /> constructed ligating BglII-DraIII fragment pACYC-duet-                 To confirm restoration CFA resulted specifically from <br /> CsdAE174A (4), containing coding region DeaD(DAAD) open             Tn10 insertions, used P1 transduction (36) transfer Tn10 inser- <br /> reading frame (ORF), larger BglII-DraIII fragment Aska-           tion loci MT2000 MT2009 clones sub- <br /> deaD(Ap). pACYC-duet-csdA pACYC-duet-CsdAE174A plasmids                    jected selection. Colonies appearing ara(&#11002;) plates restreaked <br /> were kindly provided Sangita Phadtare.                                      freshly prepared M9-pyruvate plates containing lacking 0.1% ar- <br />     All results described obtained using derivatives   abinose incubated 37&#176;C 7 days. We identified clones from <br /> original strain (i.e., N3433) used identify RNase E essentiality    ara(&#11001;) plates produced colonies ara(&#11002;) plates cultured <br /> (21). Second-site suppression mutations identified con-          M9-pyruvate-ara(&#11001;) medium 37&#176;C prepare glycerol stocks. <br /> firmed transduction mutations E. coli strain MG1655 (<span id='am-89' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-90' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-91'  ></span><span id='am-92' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span>       Clones producing approximately numbers colonies ara(&#11001;) <br /> not shown).                                                                    ara(&#11002;) plates designated showing 100% restoration CFA <br />     Media culture conditions. <span id='am-20' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Luria" datatype="xsd:string"></span><span id='am-23'  ></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Luria</span>-Bertani (LB) medium M9             subjected arbitrary PCR analysis identify chromosomal <br /> minimal medium (37) containing 0.1 mM CaCl2, 1 mM MgSO4, 5                 location Tn10 insertion. <br /> &#65533;?&#65533;g/ml thiamine hydrochloride indicated carbon sources                 Arbitrary PCR DNA sequencing. Arbitrary PCR as <br /> used. Detailed conditions culture described figure legend.     described previously (10, 39). In round, primer unique left <br /> Antibiotics used indicated, following concentrations:        end (with IS10 LEFT) Tn10 elements (Tn10L1 [5=-CAACTTAAATGT <br /> ampicillin, 50 &#65533;?&#65533;g/ml; chloramphenicol, 10 &#65533;?&#65533;g/ml; kanamycin, 20 &#65533;?&#65533;g/ml;          GAAAGTGGGTCT-3=]) arbitrary primers (ARB1 [5=-GGCCAC <br /> tetracycline, 5 &#65533;?&#65533;g/ml; streptomycin, 50 &#65533;?&#65533;g/ml.                             GCGTCGACTAGTACNNNNNNNNNNGATAT-3=] ARB6 [5=-GG <br />     Tn10 random mutagenesis. MT2009 cells grown LB medium              CCACGCGTCGACTAGTACNNNNNNNNNNACGCC-3=]) used <br /> containing 0.1% arabinose 37&#176;C optical density 600 nm              50-&#65533;?&#65533;l PCR mixtures. The second round PCR amplification per- <br /> (OD600) approximately 1.5. Cells centrifuged room tempera-          formed similarly, 2.5 &#65533;?&#65533;l first-round PCR product was <br /> ture (RT) 5,000 rpm 5 min resuspended LB medium con-             used source DNA template primers ARB2 (5=-GG <br /> taining 10 mM MgSO4 100 &#65533;?&#65533;M IPTG final OD600 5.0. A cell            CCACGCGTCGACTAGTAC-3=) Tn10L2 (5=-TGGTAACTTCACG <br /> lysate containing approximately 108 PFU/ml &#65533;?&#65533;1098 (52) added,           GTAACCAAGAT-3=). The ARB2 sequence identical 5= end the <br /> cultures incubated 37&#176;C 15 min shaking. Arabinose          ARB1 primer, sequence Tn10L2 identical left end of <br /> was added final concentration 0.1% maintain production           Tn10, adjacent end tetR gene near junction the <br /> RNase E, cells cultured 37&#176;C 90 min shaking (250          transposon chromosome. The predominant PCR products were <br /> rpm). The cells collected centrifugation, resuspended M9       purified 1.2% agarose gel sent DNA sequencing Se- <br /> medium, plated M9 solid medium containing 0.2% sodium pyru-             quetech (Mountain View, CA). <br /> vate (M9-pyruvate), 100 &#65533;?&#65533;M IPTG, chloramphenicol, kanamycin,                   Immunoblotting. Immunoblotting performed described pre- <br /> tetracycline, 0.1% arabinose [ara(&#11001;)] arabinose         viously (48). To detect RNase G protein, rabbit polyclonal antibody <br /> [ara(&#11002;)]. Plates incubated 37&#176;C aerobically 9 days. Colonies       directed polypeptide NNEDHRRRVLHSLEQALSK (anti- <br /> were replated M9-pyruvate medium described above, clones          RNG 352) KVQIEPLYNQEQFDVVMM (anti-RNG C-term) used <br />  <br />  <br /> 1920     jb.asm.org                                                                                                                    Journal Bacteriology <br />                                                                                                                 Second-Site Suppression RNase E Essentiality <br />  <br />  <br />  <br />  <br />                                                                                FIG 2 Effect Tn10 insertion CFA &#9004;rne bacteria. MT2000(pBAD- <br /> FIG 1 Procedure screening Tn10-restored &#9004;rne E. coli. Tn10 intro-   RNE) containing deaD::Tn10 (B) btuC-ihfA::Tn10 (D) and <br /> duced randomly strain MT2009 &#65533;?&#65533; phage. Tn10 insertion P1 trans-     MT2009(pBAD-NRNE) containing Tn10 insertion (A) (as negative <br /> duced new MT2009 MT2000 strain. The location Tn10 inser-      control) ydfV::Tn10 (C) grown spread M9-pyruvate medium <br /> tion identified arbitrary PCR subsequent DNA sequencing analysis.   plates containing lacking 0.1% arabinose. The plates incubated aero- <br />                                                                                bically 37&#176;C, photos taken. <br />  <br />  <br /> at 1:5,000 dilution. Anti-rabbit IgG&#8211; horseradish peroxidase (HRP) (1 <br /> mg/ml; Promega) diluted 1 5,000 used secondary antibody. <br />                                                                                ara(&#11001;) ara(&#11002;) plates isolated (Fig. 2, sites of <br /> To detect histidine-tagged protein, <span id='am-79' about='obo:NCBITaxon_1310384' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_83136 obo:NCBITaxon_1 obo:NCBITaxon_83139 obo:NCBITaxon_131567 obo:NCBITaxon_6854 obo:NCBITaxon_426515 obo:NCBITaxon_6843 obo:NCBITaxon_426501 obo:NCBITaxon_33154 obo:NCBITaxon_6933 obo:NCBITaxon_6656 obo:NCBITaxon_70332 obo:NCBITaxon_6946 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_70320 obo:NCBITaxon_6947 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-80' property="rdfs:label" content="Tetra" datatype="xsd:string"></span><span id='am-81'  ></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>tetra</span>-His mouse monoclonal anti-         Tn10 insertions determined DNA sequencing. Eight Tn10 <br /> body (0.2 mg/ml; Qiagen) used 1:1,000 dilution. Goat anti-mouse       insertion site loci different genes associated the <br /> IgG&#8211;HRP (200 &#65533;?&#65533;g/0.5 ml) 1:5,000 dilution used secondary        acquisition CFA &#9004;rne bacteria. These insertions shown <br /> antibody. Reprobing performed incubating nitrocellulose             nucleotide numbers Table 2, deaD (five independent <br /> membrane 100 ml reprobing reagent (0.1 M <span id='am-25' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-30'  ></span>beta</span>-mercaptoethanol,           insertions ORF), ydfV (one insertion ORF), and <br /> 62.5 mM Tris-HCl [pH 6.8], 2% SDS) 50&#176;C 30 min, shaking            ihfA-btuC (two independent insertions ihfA btuC). <br /> every 5 min.                                                                   However, N-rne-complemented &#9004;rne cells grew nor- <br />                                                                                mally (28), colony-forming &#9004;rne Tn10 insertion mutant <br /> RESULTS AND DISCUSSION                                                         clones showed prominent filamentation cells examined micro- <br /> Tn10 insertions E. coli genes confer CFA               scopically dramatically reduced rate doubling liq- <br /> on rne deletion mutants. To determine RNase E es-                  uid culture (Table 3): &#11350;4 days culture M9-pyruvate medium <br /> sentiality phenotype suppressed mutations second                required colonies attain normal size (approxi- <br /> site E. coli chromosome, used bacteriophage &#65533;?&#65533; Tn10                 mately 1 mm diameter) absence arabinose. Clones <br /> construct (&#65533;?&#65533;1098) (52) randomly mutagenize rne deletion                  containing Tn10 insertion ydfV ihfA-btuC produced mu- <br /> strain complemented trans plasmid-borne gene producing                 coid colonies. <br /> N-Rne arabinose (ara)-controlled promoter (Table 1;                        The possibility propagation &#9004;rne Tn10 insertion <br /> also Fig. S1 supplemental material) selected clones             mutant clones described resulted leakiness of <br /> that formed colonies N-Rne complementation termi-                     pBAD-controlled N-rne cells eliminated experi- <br /> nated plating M9 medium lacking arabinose (Fig. 1). Since                ments showing removal pBAD-NRNE plasmid dis- <br /> LB reported unable support growth                placement pSC109 (14) effect CFA colony <br /> E. coli mutants, including containing alterations glyco-              size increased arabinose replacement. Loss of <br /> lytic pathway genes (24; unpublished data), defined nutri- <br /> ent-poor medium (i.e., M9) used screen. Pyruvate was <br /> chosen stable carbon source (24); choice proved                TABLE 2 Locations Tn10 insertions <br /> fortuitous, later certain second-site sup-             Mutated gene          Clone       Location Tn10 insertion (nucleotides) <br /> pressor mutations identified did restore CFA 0.2% <br />                                                                                deaD::Tn10            2           Between 98A 99G (deaD ORF) <br /> glucose used sole carbon source. The basis carbon <br />                                                                                                      4           Between 705A 706C (deaD ORF) <br /> source effects second-site suppression elucidated.                                   5           Between 310G 311T (deaD ORF) <br />     Plates lacking arabinose contained approximately 100 colonies                                    11          Between 1218G 1219C (deaD ORF) <br /> after incubation 37&#176;C 9 days, 5,000 colonies                                    36          Between 107G 108T (deaD ORF) <br /> were observed ara(&#11001;) plates. Colonies showing similar colony <br /> forming efficiencies ara(&#11001;) ara(&#11002;) plates iden-               ydfV::Tn10                        Between 37G 38C (ydfV ORF) <br /> tified, role Tn10 insertion CFA phenotype was <br /> confirmed described Materials Methods). Nine trans-                  ihfA-btuC::Tn10       6           Between 29G 30G (after TAA ihfA) <br />                                                                                                      20          Between 20C 21C (after TAA ihfA) <br /> ductants producing approximately numbers colonies <br />  <br />  <br /> April 2012 Volume 194 Number 8                                                                                                              jb.asm.org 1921 <br />  Tamura et al. <br />  <br />  <br />  <br /> TABLE 3 Doubling times strains inserted Tn10a                                   insertions reversed CFA phenotype, confirming existence <br />                                      Doubling time (min)                                multiple second-site suppressors RNase E essentiality under <br />                                                                                         growth conditions tested (Fig. 3). For ydfV, located <br />                                      M9 medium                  LB medium <br />                                                                                         upstream relBEF operon (6), reversal effect the <br />                                      0.1%                       0.1%                    Tn10 insertion accomplished overexpression down- <br /> Strain                               ara          ara(&#11002;)        ara          ara(&#11002;)     stream relB gene overexpression ydfV itself (Fig. <br /> MT2000                               63           Lethal        33           Lethal     3B). For ihfA-btuC, Tn10 insertion located pro- <br /> Clone 2 (deaD::Tn10)                 73           391           35           152        moter region btuCED operon (17), reversal accom- <br /> Clone 18 (ydfV::Tn10)                73           373           ND           Lethal <br />                                                                                         plished overexpression of btuCED operon genes <br /> Clone 20 (ihfA-btuC::Tn10)           73           339           ND           Lethal <br /> a <br />                                                                                         (btuC, btuE, btuD) (Fig. 3C), implying Tn10 insertion <br />   Cells cultured presence arabinose grown mid- late log phase at <br /> 37&#176;C LB M9-pyruvate medium supplemented 0.1% arabinose antibiotics <br />                                                                                         promoter region eliminated functions three <br /> and diluted OD600 value 0.1 15 ml medium. The OD600    genes. During analysis Tn10 insertion strains, ob- <br /> measured 30 60 min. For ara(&#11002;) cultures, pBAD-RNE pBAD-NRNE             served &#9004;rne deaD::Tn10 strain formed colonies <br /> plasmid &#9004;rne strains carrying Tn10 insertions replaced pSC109 during <br />                                                                                         equal efficiencies M9 LB media (Fig. 4). These <br /> growth LB (for deaD::Tn10 insertion strain) M9-pyruvate (for ydfV::Tn10 and <br /> ihfA-btuC::Tn10 insertion strains) medium absence arabinose. Cultures    observations prompted focus report role of <br /> diluted OD600 value 0.1 20 ml medium. The OD600             deaD (also known csdA [25]) second-site suppressor of <br /> measured 1 2 h. OD600 values used doubling time calculations ranged   RNase E essentiality. LB medium used investigate this <br /> 0.1 0.6. <br />                                                                                         role. <br />                                                                                             Overexpression relB, btuC, btuE, btuD gene reversed <br />                                                                                         ability deaD::Tn10 cells form colonies (Fig. 5A B), <br /> the originally complementing rne gene confirmed PCR <br />                                                                                         conversely, overexpression deaD reversed CFA pheno- <br /> analysis, absence detectable RNase E protein con- <br />                                                                                         type ihfA-btuC::Tn10 insertion (Fig. 5C). However, path- <br /> firmed Western blotting (data shown). Together, these <br />                                                                                         way analysis using <span id='am-47' about='xsd:string' typeof='owl:Thing'>STRING</span> (ver. 9.0; http://string-db.org/) and <br /> results argue Tn10 inactivation of genes iden- <br /> tified partially suppressed need RNase E accomplish                      EcoCyc (http://ecocyc.org/) showed known commonality in <br /> cell growth.                                                                            metabolic pathways known affected various sec- <br />     Overexpression genes loci affected Tn10                      ond-site suppressor genes identified, basis their <br />  <br />  <br />  <br />  <br /> FIG 3 Confirmation role identified genes restoration CFA. (A C) Clones &#9004;rne strain [MT2000(pBAD-RNE) containing deaD::Tn10 or <br /> btuC-ihfA::Tn10 MT2009(pBAD-NRNE) containing ydfV::Tn10] transformed plasmids expressing genes identified putative second-site sup- <br /> pressors RNase E essentiality (deaD ydfV) or, indicated, expressing neighboring genes (relB, btuC, btuE, btuD). Cultures spread on <br /> M9-pyruvate plates containing lacking 0.1% arabinose. IPTG added indicated. Plates scanned incubation 2 7 days ara(&#11001;) plates [7 <br /> days MT2000(deaD::Tn10) 2 days others] 4 8 days ara(&#11002;) plates [7 days MT2000(deaD::Tn10), 4 days MT2000(ihfA-btuC::Tn10), <br /> 5 days MT2000(ydfV::Tn10) plus Aska-ydfV(Ap), 8 days MT2009(ydfV::Tn10) plus Aska-relB(Ap)]. (D) Summary results obtained these <br /> complementation experiments. <br />  <br />  <br />  <br /> 1922     jb.asm.org                                                                                                                      Journal Bacteriology <br />                                                                                                                     Second-Site Suppression RNase E Essentiality <br />  <br />  <br />  <br />  <br /> FIG 4 Effect LB medium restoration CFA &#9004;rne strains. Cultures of <br /> MT2000(deaD::Tn10) (A), MT2009(ydfV::Tn10) (B), MT2000(ihfA- <br /> btuC::Tn10) (C) spread LB plates containing lacking 0.1% arabi- <br /> nose. Plates scanned incubation 1 day 0.1% ara-containing      FIG 6 Effect endogenous RNase G restoration CFA. (A) Effect deaD <br /> plates 4 days ara(&#11002;) plates.                                              mutation RNase G protein abundance. Liquid cultures MT2000 and <br />                                                                                   MT3001 (N3433 &#9004;rne &#9004;deaD774::Km) grown 37&#176;C presence of <br />                                                                                   0.1% arabinose harvested OD600 0.6, immunoblotting per- <br />                                                                                   formed described Materials Methods. Membranes probed anti- <br /> functional interaction context RNase E essentiality                 RNase G antibody stripped reprobed anti-S1 antibody. RNase G <br /> phenotype known.                                                           protein abundance normalized according abundance ribosomal pro- <br />    Restoration CFA Tn10 insertion requires function                     tein S1, ratio shown image. (B) Effect Tn10 insertion on <br />                                                                                   CFA rne rng deletion mutant strains. MT2000 &#9004;rng containing deaD::Tn10 <br /> of endogenous RNase G. Since overproduction RNase G                         grown spread ara(&#11001;) ara(&#11002;) M9-pyruvate plates. Plates were <br /> known circumvent need RNase E CFA (13, 16, 27,                     scanned incubation 24 days 37&#176;C. <br /> 48), determined loss function deaD in- <br /> creased production RNase G. Arguing possibility <br /> were immunoblotting results DNA sequencing data showing                       function provided presence normal chromosomal copy <br /> no change RNase G protein abundance (Fig. 6A). Additionally,                   rng required deaD mutation suppress effect of <br /> DNA sequencing bacteria showing restoration CFA revealed                    deletion rne, efforts circumvent RNase E essentiality rne <br /> no mutation rng ORF (data shown). Nevertheless,                   rng doubly mutant bacteria successful (Fig. 6B). The data <br />  <br />  <br />  <br />  <br /> FIG 5 Reversal second-site suppression Tn10-inserted genes absence RNase E. (A) Cultures MT2000(deaD::Tn10) carrying pCA24N(Ap), Aska- <br /> btuC(Ap), Aska-btuE(Ap), Aska-btuD(Ap) spread LB plates containing 50 &#65533;?&#65533;M IPTG containing lacking 0.1% arabinose. Plates containing 0.1% arabinose <br /> were scanned incubation 1 day pCA24N(Ap) 2 days Aska-btuC(Ap), Aska-btuE(Ap), Aska-btuD(Ap), plates lacking arabinose scanned <br /> after 4 days. (B) Cultures MT2000(deaD::Tn10) carrying pCA24N(Ap) Aska-relB(Ap) spread LB plates containing 100 &#65533;?&#65533;M IPTG containing lacking <br /> 0.1% arabinose. Plates containing 0.1% arabinose scanned incubation 1 day, plates lacking arabinose scanned 4 days. (C) Cultures of <br /> MT2000(ihfA-btuC::Tn10) carrying Aska-deaD(Ap) Aska-DAAD(Ap) grown spread M9-pyruvate plates lacking IPTG containing lacking 0.1% <br /> arabinose. Plates containing 0.1% arabinose scanned incubation 2 days, plates lacking arabinose scanned 9 days. <br />  <br />  <br />  <br /> April 2012 Volume 194 Number 8                                                                                                                   jb.asm.org 1923 <br />  Tamura et al. <br />  <br />  <br />  <br />  <br /> FIG 7 Effect RNA helicase activity DeaD CFA &#9004;rne bacteria. (A) Cultures MT2000(deaD::Tn10) Aska-deaD(Ap) plasmid or <br /> Aska-DAAD(Ap) plasmid spread LB plates containing lacking 0.1% arabinose. Plates scanned incubation 1 day (0.1% ara LB) 3 <br /> days [ara(&#11002;) LB plates]. (B) Cultures MT2000(deaD::Tn10) carrying Aska-deaD(Ap) Aska-DAAD(Ap) grown mid-log phase 37&#176;C single <br /> colony presence 0.1% arabinose (no IPTG), diluted fresh medium lacking arabinose IPTG, cultured 37&#176;C additional 3 h, harvested <br /> at OD600 0.6. Harvested pellets dissolved 60 &#65533;?&#65533;l sample buffer, 10 &#65533;?&#65533;l suspension loaded 7% Tris-acetate gel. <br /> Immunoblotting performed described Materials Methods. Membranes probed anti-His antibody stripped reprobed using anti-S1 <br /> antibody. (C) Cultures MT2000(deaD::Tn10) carrying pCA24N(Ap), Aska-rhlE(Ap), Aska-rnr(Ap) grown spread LB plates containing or <br /> lacking 0.1% arabinose. Fifty micromolar IPTG added plate induce expression Aska-rnr(Ap). Plates scanned incubation 1 day [0.1% <br /> ara plates pCA24N(Ap) Aska-rhlE(Ap)], 2 days [0.1% ara plates Aska-rnr(Ap)], 4 days [all ara(&#11002;) plates]. <br />  <br />  <br />  <br /> below model presented later report suggest               RNase R, contains separate domains showing exoribo- <br /> function be.                                                                 nuclease helicase activities, enables cold shock growth deaD <br />     Elimination helicase activity DeaD sufficient sec-              mutant cells (3, 4). Such overexpression prevented second- <br /> ond-site suppression RNase E essentiality. The deaD gene en-                  site suppression RNase E essentiality 37&#176;C (Fig. 7D). <br /> codes ATP-dependent E. coli RNA helicases:                      Proposed mechanism second-site suppression rne mu- <br /> DbpA, SrmB, RhlB, RhlE, DeaD itself (8, 49, 50). Four               tation mutation deaD. Our results identified multiple <br /> proteins (DbpA, SrmB, RhlE, DeaD) known im-                      genes actions prevent colony formation cells lacking <br /> portant roles ribosomal assembly maturation (46). These                   RNase E. However, insertional inactivation deaD ORF <br /> RNA helicases reported function translation                 resulted medium-independent restoration CFA ab- <br /> and mRNA decay (15, 23), them, RhlB DeaD,                    sence rne, suggesting gene acts mechanism distinct <br /> been bind C-terminal half RNase E                      actions identified genes. Second-site sup- <br /> normal cell growth shift cultures low temperature                pression RNase E essentiality phenotype mutation of <br /> (i.e., cold shock conditions) (35, 42, 43). Normally, DeaD pres-              deaD reversed restoration deaD function overex- <br /> ent small quantities cells cultured 37&#176;C induced                 pression RhlE helicase, RNase R, can <br /> significantly cold shock (25). Knockout deaD little                unwind double-stranded regions RNA using domain is <br /> effect cell growth 37&#176;C retards growth 15&#176;C (25).                   physically separate exoribonucleolytic domain (3); more- <br />     The fact helicase function DeaD specifically             over, restoration CFA deaD overexpression shown to <br /> prevents CFA deaD&#11001; rne null mutant bacteria shown                      require helicase domain protein. Together, find- <br /> introducing deaD::Tn10 insertion mutant cells plasmid pro-                ings, indicate novel functional relationship between <br /> ducing DeaD variant protein containing missense mutation                  RNase E DeaD known ability DeaD partici- <br /> the helicase domain [Aska-DAAD(Ap)] (Fig. 7). Whereas expres-                    pate cold shock degradosome, suggest previously unsus- <br /> sion wild-type deaD [Aska-deaD(Ap)] reversed CFA ob-                      pected role RNA secondary structure determinant of <br /> served deaD::Tn10 strain, helicase mutant [Aska-DA-                   RNase E-dependent cell growth E. coli. <br /> AD(Ap)] produced reversal (Fig. 7A B),                           In principle, ability mutation deaD restore CFA to <br /> expression level mutant DeaD protein similar               &#9004;rne bacteria occur production RNA when <br /> the wild-type DeaD protein (Fig. 7C). While deaD helicase                 unwound DeaD toxic unless cleaved RNase E. In a <br /> normally required E. coli propagation cold shock con-                  case, requirement endogenous RNase G production to <br /> ditions (4, 25), overexpression related RNA helicase RhlE              enable CFA rne deaD double mutants result synthetic <br />  <br />  <br /> 1924   jb.asm.org                                                                                                                        Journal Bacteriology <br />                                                                                                                         Second-Site Suppression RNase E Essentiality <br />  <br />  <br />  <br /> lethality mutations genes. However, likely                        defective Rho-dependent transcription termination. Mol. Microbiol. <br /> model involves ribosome biogenesis. RNase E required mat-                         82:1330 &#8211;1348. <br />                                                                                     2.   Apirion D. 1978. Isolation, genetic mapping characterization of <br /> uration 16S rRNA (30, 51), DeaD role                             mutation Escherichia coli affects processing ribonucleic <br /> production 50S rRNA subunits (12). A misassembled 50S rRNA                            acid. Genetics 90:659 &#8211; 671. <br /> that sediments 40S produced large amounts                       3.   Awano N, et al. 2010. Escherichia coli RNase R dual activities, helicase <br /> low temperature (12) observed early-log-                            RNase. J. Bacteriol. 192:1344 &#8211;1352. <br /> phase growth 37&#176;C (41). The helicase activity DeaD pro-                   4.   Awano N, et al. 2007. Complementation analysis cold-sensitive <br />                                                                                          phenotype Escherichia coli csdA deletion strain. J. Bacteriol. 189: <br /> posed enable structural rearrangement pre-50S particle,                       5808 &#8211;5815. <br /> which includes immature 30S particle containing precursor                    5.   Baba T, et al. 2006. Construction <span id='am-94' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-95' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-96' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia coli K-12</span> in-frame, <br /> 16S rRNA (12). We suggest bacteria lacking DeaD, suffi-                          single-gene knockout mutants: Keio collection. Mol. Syst. Biol. <br /> cient 40S particles formed processed mature                              2:2006.0008. <br />                                                                                     6.   Bech FW, Jorgensen ST, Diderichsen B, Karlstrom OH. 1985. Sequence <br /> rRNA. If model correct, RNase E essentiality me- <br />                                                                                          relB transcription unit Escherichia coli identification the <br /> diated level ribosome production, notwith-                    relB gene. EMBO J. 4:1059 &#8211;1066. <br /> standing evidence cleavage 9S RNA, identified                     7.   Bernstein JA, Khodursky AB, Lin PH, Lin-Chao S, Cohen SN. 2002. <br /> substrate RNase E (21), virtually normal cells defective                     Global analysis mRNA decay abundance Escherichia coli sin- <br /> RNase E activity (40). The extraordinarily low rate growth                         gle-gene resolution using two-color fluorescent DNA microarrays. Proc. <br />                                                                                          Natl. Acad. Sci. U. S. A. 99:9697&#8211;9702. <br /> observed doubly mutant &#9004;rne deaD::Tn10 strain suggests                      8.   Bizebard T, Ferlenghi I, Iost I, Dreyfus M. 2004. Studies E. coli <br /> that rRNA processing alternative pathway postulate                        DEAD-box helicases point unwinding mechanism different from <br /> occurs inefficiently. Moreover, finding second-site sup-                        model DNA helicases. Biochemistry 43:7857&#8211;7866. <br /> pression rne deletion mutation deaD does occur                  9.   Brack C. 1979. <span id='am-74' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-75' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-76'  ></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Electron</span> microscopic analysis transcription: mapping of <br /> absence RNase G argues endogenous level                          initiation sites direction transcription. Proc. Natl. Acad. Sci. U. S. A. <br />                                                                                          76:3164 &#8211;3168. <br /> RNase G, itself implicated maturation 16S                     10.   Caetano-Anolles G. 1993. Amplifying DNA arbitrary oligonucleo- <br /> rRNA (30, 51), required hypothesized alternative path-                        tide primers. PCR Methods Appl. 3:85&#8211;94. <br /> way ribosomal biogenesis. Consistent notion, rng                   11.   Chang AC, Cohen SN. 1978. Construction characterization am- <br /> not normally required CFA cells producing rne essen-                       plifiable multicopy DNA cloning vehicles derived P15A cryptic <br />                                                                                          miniplasmid. J. Bacteriol. 134:1141&#8211;1156. <br /> tial second-site suppression mutation deaD.                              12.   Charollais J, Dreyfus M, Iost I. 2004. CsdA, cold-shock RNA helicase <br />     Garrey Mackie recently reported synthetic lethality                        Escherichia coli, involved biogenesis 50S ribosomal sub- <br /> rne mutation affecting ability RNase E sense nature                     unit. Nucleic Acids Res. 32:2751&#8211;2759. <br /> the 5= end substrates rne mutation deletes CTH                  13.   Chung DH, Min Z, Wang BC, Kushner SR. 2010. Single amino acid <br /> of enzyme (20). Similarly, Anupama et al. deletion                        changes predicted RNase H domain Escherichia coli RNase G lead <br />                                                                                          complementation RNase E deletion mutants. RNA 16:1371&#8211;1385. <br /> of C-terminal RNase E synthetically lethal E. coli mutations               14.   Cohen SN, Chang AC, Boyer HW, Helling RB. 1973. Construction of <br /> in chromosomal rppH gene (1). Anupama et al. discovered                              biologically functional bacterial plasmids vitro. Proc. Natl. Acad. Sci. <br /> that mutation rho nusG restored viability &#9004;rpph &#9004;CTH                            U. S. A. 70:3240 &#8211;3244. <br /> doubly mutant bacteria and, additionally, lethality suppres-                  15.   Cordin O, Banroques J, Tanner NK, Linder P. 2006. The DEAD-box <br />                                                                                          protein <span id='am-31' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> RNA helicases. Gene 367:17&#8211;37. <br /> sion dependent ability strain digest RNA-DNA                   16.   <span id='am-84' about='obo:NCBITaxon_581821' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_581380 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_37573 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_268499 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Deana" datatype="xsd:string"></span><span id='am-88'  ></span>Deana</span> A, Belasco JG. 2004. The function RNase G Escherichia coli is <br /> duplexes. They consequently proposed increased abun-                             constrained amino carboxyl termini. Mol. Microbiol. 51:1205&#8211; <br /> dance R loops (i.e., RNA-DNA hybrids) formed tran-                             1217. <br /> scription (9; review, reference 18), ordinarily                 17.   Veaux LC, Clevenson DS, Bradbeer C, Kadner RJ. 1986. Identifica- <br /> detrimental growth rho&#11001; nusG&#11001; bacteria, contributes                        tion btuCED polypeptides evidence role vitamin B12 <br />                                                                                          transport Escherichia coli. J. Bacteriol. 167:920 &#8211;927. <br /> to rescue synthetic lethality digestion <span id='am-99' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-100' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-101'  ></span><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="hybrid" datatype="xsd:string"></span>hybrid</span>               18.   Drolet M. 2006. Growth inhibition mediated excess negative super- <br /> duplexes RNase H provides alternative means RNA deg-                            coiling: interplay transcription elongation, R-loop formation <br /> radation. Whereas reported evidence E. coli                      DNA topology. Mol. Microbiol. 59:723&#8211;730. <br /> DeaD protein unwind RNA-DNA hybrids, eukaryotic                            19.   Du MX, et al. 2002. Comparative characterization DEAD-box <br />                                                                                          RNA helicases <span id='am-71' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-72' property="rdfs:label" content="superfamily" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>superfamily</span> II: human translation-initiation factor 4A <br /> DEAD-box family protein eIF4A act RNA-DNA duplexes                                <span id='am-1' about='obo:NCBITaxon_11103' typeof='owl:Thing obo:NCBITaxon_11102 obo:NCBITaxon_1 obo:NCBITaxon_10239 obo:NCBITaxon_11050 obo:NCBITaxon_439488 obo:NCBITaxon_35278'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="hepatitis C virus HCV" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Hepatitis C" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="human hepatitis C virus" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasBroadSynonym" content="HCV" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="human hepatitis virus HCV" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Hepatitis C virus" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="human hepatitis virus C HCV" datatype="xsd:string"></span><span id='am-11'  ></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="post-transfusion hepatitis non A non B virus" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="human hepatitis C virus HCV" datatype="xsd:string"></span>hepatitis C virus</span> non-structural protein 3 (NS3) helicase. Biochem. J. <br /> as double-stranded RNA (19). If DeaD acts similarly                        363:147&#8211;155. <br /> hybrid duplexes, reduction helicase activity contrib-                20.   Garrey SM, Mackie GA. 2011. Roles 5=-phosphate sensor domain <br /> <span id='am-109' about='obo:NCBITaxon_365374' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_86011 obo:NCBITaxon_33208 obo:NCBITaxon_1 obo:NCBITaxon_6040 obo:NCBITaxon_45124 obo:NCBITaxon_131567 obo:NCBITaxon_73655 obo:NCBITaxon_27929 obo:NCBITaxon_2759'><span id='am-110'  ></span><span id='am-111' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="Ute" datatype="xsd:string"></span>ute</span> ability deaD mutations suppress loss CFA                      RNase E. Mol. Microbiol. 80:1613&#8211;1624. <br /> rne null mutant bacteria.                                                          21.   Ghora BK, Apirion D. 1978. Structural analysis vitro processing to <br />                          <br /> </body></html>