<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Addition <span id='am-28' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33'  ></span><span id='am-34' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span>Escherichia coli</span> K-12 Growth Observation Gene <br /> Essentiality <span id='am-70' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-73'  ></span><span id='am-74' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Data</span> EcoCyc Database <br /> Amanda Mackie,a Suzanne Paley,b Ingrid M. Keseler,b Alexander Shearer,b Ian T. Paulsen,a Peter D. Karpb <br /> Department Chemistry Biomolecular Science, Macquarie University, Sydney, Australiaa; Bioinformatics Research Group, SRI International, Menlo Park, <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-6'  ></span>California</span>, <br /> USAb <br />  <br /> The sets compounds support growth organism defined presence transporters metabolic path- <br /> ways convert nutrient sources cellular components energy growth. A collection known nutrient sources can <br /> therefore serve impetus investigating new metabolic pathways transporters reference computa- <br /> tional modeling known metabolic pathways. To establish collection <span id='am-44' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-45' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-47'  ></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia</span> coli K-12, integrated data <br /> on growth nongrowth E. coli K-12 obtained published observations using variety individual media from <br /> high-throughput phenotype microarrays EcoCyc database. The assembled collection revealed substantial number of <br /> discrepancies high-throughput data sets, investigated possible using low-throughput growth assays <br /> on soft agar liquid culture. We integrated data sets describing 16,119 observations growth single-gene <br /> knockout mutants E. coli K-12 EcoCyc, relevant antimicrobial drug design, provide clues regarding roles <br /> of genes unknown function, useful validating metabolic models. To make information easily accessible to <br /> EcoCyc users, developed software capturing, querying, visualizing cellular growth assays gene essentiality data. <br />  <br />  <br />  <br /> T    diversity chemical environments support organ- <br />      ismal growth fundamental corpus scientific knowledge. <br /> However, despite decades research large amounts accu- <br />                                                                                       throughput phenotype microarray (PM) data (2); (iv) PM <br />                                                                                       data generated group. PMs measure cellular respiration as <br />                                                                                       proxy growth (henceforth, refer growth simplicity) <br /> mulated knowledge, compendium set nutrients Esch-                        sets 96-well plates, containing a <br /> erichia coli K-12 able utilize exists. Such compendium                 different combination nutrients. We integrated into <br /> only basic serve reference com-                   EcoCyc data sets growth single-gene <br /> putational metabolic modeling, requires comprehensive                         knockout mutants E. coli K-12 published in <br /> set reference growth data sets evaluating accuracy                      recent years. <br /> growth predictions.                                                                       We developed software tools capturing, querying, vi- <br />     Also validating computational metabolic models                    sualizing cellular growth assays, corresponding nutrients and <br /> are experimental data describing essentiality organism&#8217;s                    growth conditions, gene essentiality data. These tools have <br /> genes, metabolic models predict phenotypes                         integrated Pathway Tools software (3), en- <br /> gene knockouts. An essential gene gene indispensable                  abling use conjunction 2,000 <br /> support life specific set growth conditions. Further,                      genomes contained BioCyc database collection (BioCyc <br /> gene essentiality data useful predicting antibiotic targets               .org) organism databases created Pathway Tools <br /> pathogenic bacteria, guiding design minimal genomes,                       users. <br /> and providing clues regarding roles genes unknown <br /> function.                                                                             MATERIALS AND METHODS <br />     A large number high-throughput data sets&#8212;from gene ex-                         Bacterial strains. E. coli K-12 MG1655 obtained Yale <br /> pression metabolomics phenotypic measurements&#8212;are be-                           Coli Genetic Stock Center (CGSC; strain 7740) American <br /> coming available variety organisms. To maximize                        Type Culture Collection (ATCC; strain 700926). <br />                                                                                           Laboratory evaluation individual growth media. To evaluate <br /> value, data sets captured integrated a <br />                                                                                       growth soft agar, following protocol used based modifica- <br /> database published website, tools querying                    tion earlier <span id='am-51' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-52' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-56'  ></span><span id='am-57' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span>Salmonella</span> nutrient evaluation method (4). Bacterial <br /> and analysis ensure data maximally available                   cells grown overnight LB medium 37&#176;C. The overnight culture <br /> scientific community. However, integration requires                         washed times phosphate-buffered saline (PBS), resus- <br /> simply collecting multiple data sets common repos-                      pended PBS, diluted 10:1 cooling liquefied 0.6% agar solution <br /> itory. For data types, integration include identifying <br /> and resolving conflicts data sets possible in <br /> <span id='am-25' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> extract knowledge possible noisy data.                          Received 10 October 2013 Accepted 15 December 2013 <br />     We integrated following growth data single collec-                        Published ahead print 20 December 2013 <br /> tion available EcoCyc database (1) supplemental                        Address correspondence Peter D. Karp, pkarp@ai.sri.com. <br /> files: (i) collection observations growth nongrowth                    Supplemental material article http://dx.doi.org/10.1128 <br /> laboratory strains E. coli K-12 variety media                        /JB.01209-13. <br /> been obtained low-throughput methods                                Copyright &#169; 2014, American Society Microbiology. All Rights Reserved. <br /> previously published literature; (ii) low-throughput growth                       doi:10.1128/JB.01209-13 <br /> data generated group; (iii) previously published high- <br />  <br />  <br /> 982   jb.asm.org                                       Journal Bacteriology   p. 982&#8211;988                                          March 2014 Volume 196 Number 5 <br />                                                                                                             Addition Growth Essentiality Data EcoCyc <br />  <br />  <br />  <br /> before plated appropriate base plate. Fully cooled        analyzed using Omnilog PM software response calls growth <br /> plates used subsequent nutrient testing. Base plates        (G), growth (NG), low growth (LG) based param- <br /> made minimal salts medium (1.9 mM potassium sulfate, 25.8 mM              eters maximum height (the 10th percentile highest value all <br /> dipotassium phosphate, 11.8 mM monopotassium phosphate, 0.13 mM                values time points) area (the sum Omnilog values over <br /> magnesium sulfate heptahydrate) supplemented 1.5% agar ei-            time points). No calls 13 nitrogen source wells, 10 <br /> ther 0.5% succinate (for nitrogen source testing) 2.5 mM NH4Cl (for         sulfur source wells, phosphate sources, which <br /> carbon source testing). Each nutrient tested plates (containing     showed ambiguous responses compared negative-control well. <br /> either succinate NH4Cl) placing 2 20 mg nutrient        Data set S6 supplemental material includes raw data PM <br /> center plate. The plates incubated 37&#176;C, growth        experiments conducted course work. &#8220;Mackie12&#65533;? refers <br /> evaluated visually 24, 48, 72 h. A plate scored positive     experiments conducted 2012, data currently avail- <br /> growth bacterial lawn &#11350;1 cm2 present           able EcoCyc website. The results later experiments (dated 2013 <br /> scoring checks. All growth conditions tested duplicate.                File S6 supplemental material) conducted compare Biolog PM <br />     To evaluate growth liquid culture, strains grown overnight      responses different stock cultures E. coli K-12 MG1655 in- <br /> M9 minimal media 0.2% (vol/vol) glycerol. The overnight culture           cluded future EcoCyc release. <br /> was washed times modified M9 minimal media containing                 After PM data sets added EcoCyc, reviewed the <br /> carbon nitrogen [48 mM sodium phosphate dibasic, 22 mM monopo-              growth observations difference observations between <br /> tassium phosphate, 8.5 mM sodium chloride, 2 mM magnesium sulfate,             PM data sets existed, resolved conflict significant <br /> 0.1 mM calcium chloride, 0.01 mM Fe(II) sulfate] transfer new      evidence regarding actual growth outcome found. The factors <br /> M9 minimal medium starting optical density 600 nm 0.05.           considered included number available observations growth ver- <br /> Carbon sources tested M9 minimal medium final concentra-          <span id='am-20' about='obo:NCBITaxon_9822' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9821 obo:NCBITaxon_35497 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22'  ></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Sus" datatype="xsd:string"></span>sus</span> growth, low-throughput observations in <br /> tion 0.2% (wt/vol), nitrogen sources tested modified M9       literature given growth condition (in cases were), and <br /> minimal medium final concentration 0.2% (wt/vol). The carbon           minority data set(s) generally judged reliable <br /> source assays 0.2% sodium succinate. Cultures         based total pairwise conflicts data sets. When we <br /> incubated 37&#176;C 48 h, growth rates reported means &#11006;       did resolve conflict, <span id='am-69' about='rdfs:comment' typeof='owl:Thing'>comment</span> usually created EcoCyc to <br /> standard deviations replicates. The appropriate positive neg-    summarize rationale, literature citations recorded appro- <br /> ative controls included experiments. For nitrogen source as-       priate. For example, aerobic cell PM1:F2, tests growth on <br /> says, positive control ammonium chloride; carbon source            citric acid sole carbon source, resolved favor growth <br /> assays, positive control sodium succinate.                             (5), aerobic cell PM1:G7, tests growth acetoacetic <br />     Biolog PMs. Biolog phenotype microarrays (PMs) commercially          acid sole carbon source, resolved favor growth (6) (mouse <br /> available (Biolog, Inc., Hayward, CA) high-throughput           cells online EcoCyc version Fig. 1 http://ecocyc.org <br />                                                                                /ECOLI/NEW-IMAGE?object&#11005;Growth-Media explanation for <br /> global analysis microbial respiration phenotypes (2). PMs use color- <br />                                                                                resolution). <br /> imetric reaction (reduction tetrazolium dye NADH) sensitive <br /> <span id='am-7' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Indicator" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-11'  ></span>indicator</span> microbial respiration response presence individ-     RESULTS <br /> ual nutrients chemicals. PM plates 1 4, contained 190 sole <br /> carbon sources, 95 sole nitrogen sources, 59 sole phosphate sources,       Table 1 summarizes sources nutrient utilization data pres- <br /> 35 sole sulfur sources, used analysis. E. coli K-12 MG1655    ent EcoCyc version 17.5 released October/Novem- <br /> pregrown nutrient agar used inoculate plates according      ber 2013. The results low-throughput growth experiments <br /> to Biolog instructions. The data collected analyzed using         reported appear future EcoCyc release. <br /> OmniLogH PM system, records color change 15 min                Table 1 includes total 1,860 growth observations 552 <br /> each 96-well assay plates. All incubations performed       growth conditions. The 552 growth conditions include 456 aero- <br /> 37&#176;C 48 h.                                                                bic conditions 96 anaerobic conditions include 384 PM <br />     To assess effect pregrowth conditions carbon-source utiliza-     media (including 5 negative controls) 72 media. For <br /> tion, bacterial strain pregrown types nutrient agar        example, growth assessed set nutrients, under <br /> plates&#8212;LB agar, R2A agar, BUG-S (Biolog universal growth agar         aerobic anaerobic conditions, count situation as <br /> 5% sheep blood)&#8212; inoculation PM plates 1 2. For the <br />                                                                                growth conditions. <br /> nitrogen, phosphate, sulfur utilization plates (PM3 PM4), bacte- <br /> ria pregrown LB agar and, instance, inoculated accord- <br />                                                                                    The growth data integrated literature in- <br /> ing manufacturer&#8217;s instructions. The user-supplied carbon source        cluded following commonly used media: AB, LB medium en- <br /> in assays 20 mM sodium succinate 2 &#65533;?&#65533;M ferric citrate.            riched, LB Lennox, 3-(N-morpholino)propanesulfonic acid <br /> Under conditions, positive response recorded negative-      (<span id='am-75' about='obo:NCBITaxon_258862' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_9436 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Mops" datatype="xsd:string"></span><span id='am-79'  ></span>MOPS</span>), M9, M56, M63. These media listed in <br /> control nitrogen assays sulfur assays. Thereafter,   table EcoCyc.org All Growth Media site (http: <br /> all bacterial pregrowth assays nutrient-limited      //biocyc.org/ECOLI/NEW-IMAGE?object&#11005;Growth-Media). <br /> R2A agar and, discussion manufacturer, concentration            Integration PM data sets. We integrated aerobic <br /> of cell inoculum decreased factor 10. The use R2A agar      phenotype microarray (PM) data sets produced <br /> for bacterial pregrowth preferred method minimizing re-         independently different laboratories refer as <br /> sponse seen negative-control nitrogen sulfur assays    Bochner12 (B. Bochner, unpublished data), AbuOun09 (7), <br /> been reported previously (2).                                                  Baumler11 (8), Yoon12 (9), Mackie12. These data sets con- <br />     Data integration arbitration. Respiration data obtained from <br />                                                                                sisted measurements PM plates designated PM1 (95 <br /> PMs used proxy growth. Growth observations re- <br /> corded Pathway Tools growth, growth, low (borderline) <br />                                                                                carbon sources), PM2 (95 carbon sources), PM3 (95 nitrogen <br /> growth. No attempt unify criteria used distinguish          sources), PM4 (59 phosphorus plus 35 sulfur sources), except <br /> between states imported PM data original            Baumler11, covers PM1 only. <br /> growth/respiration curve data available data             For data set, compared result the <br /> sets; rather, used determinations specified providers      result data sets turn. <br /> each data set. For PM experiments (Mackie12), kinetic data        Table 2 summarizes degree pairwise agreement dis- <br />  <br />  <br /> March 2014 Volume 196 Number 5                                                                                                               jb.asm.org 983 <br />  Mackie et al. <br />  <br />  <br />  <br />  <br /> FIG 1 Table showing current display integrated growth observations EcoCyc PM1 carbon sources aerobic conditions. Colors indicate <br /> consensus conflicts (see color legend top). The colored bar cell depicts individual observations, e.g., cell H5 (D-psicose) shows two <br /> observations growth, observations growth, observation low growth. Mousing bar BioCyc.org version diagram <br /> provides literature reference growth observations. <br />  <br />  <br />  <br /> agreement data sets. The percent pairwise agreement                    sets shows significantly pairwise conflicts the <br /> is number matches expressed percentage total                    data sets does data set. Further, 15 16 <br /> number comparisons (for example, 86% pairwise com-                       cases data sets recorded growth observa- <br /> parisons AbuOun09 data sets carbon                   tion (meaning Y [growth] N [no growth], ignoring low growth) <br /> source PM plates agreement). Data set S1 supple-                    data set recorded opposite observation, outlier <br /> mental material lists pairwise comparisons individ-                  Baumler11. If remove Baumler11 data set com- <br /> ual data sets. Table 2 combines pairwise comparison results                    puting &#8220;all combined&#65533;? agreement score Table 2, percent <br /> into single number reflects percentage agreement                     agreement carbon sources increases 82 87%; similarly, <br /> each data set data sets. We pairwise                Y/N conflict percentage carbon sources Table 2 drops <br /> agreement data sets individual growth observa-                     6 2%. <br /> tions ranged 72 87%. When conflicts exist, distinguish                      Note PM3 (nitrogen source) plate shows significantly <br /> between cases data set records growth                     disagreement plates. The different data <br /> other shows growth (we &#8220;Y/N&#65533;? conflicts)                      sets similar degrees disagreement. After systematic in- <br /> cases data set records low borderline growth                   vestigation differences data sets consid- <br /> the shows growth growth. For 379 growth                     eration low-throughput E. coli growth data literature, <br /> conditions surveyed PM data sets, conflict existed be-                    resolve 73 differences (24 Y/N). As result, <br /> tween pair data sets 119 wells, 57             consensus growth calls exist 333 379 conditions <br /> wells Y/N conflicts. <br />                                                                                    surveyed. Unresolved differences prevent growth calls re- <br />     The Baumler11 data set outlier PM data <br />                                                                                    maining 45 (of 34 represent Y/N differences). Table 3 sum- <br />                                                                                    marizes number conflicts resolved the <br />                                                                                    nutrient types aerobic conditions, number of <br /> TABLE 1 Numbers aerobic anaerobic growth observations <br /> described here, grouped data source                                             conflicts remaining case. Table 4 lists compounds that <br />                                                                                    involved remaining conflicts. <br />                                                         No. growth <br />                                                         observations <br />                                                                                        Baumler11 Bochner12 include measurements under <br />                                                                                    anaerobic observations, PM1 only. Their data sets agreed for <br /> Type data                                            Aerobic     Anaerobic      45% wells, Y/N conflict rate 49%. We did not <br /> Low-throughput data literature                 23          0              attempt resolve conflicts. <br /> Low-throughput data generated group              49          0                  Low-throughput growth experiments. After integration of <br /> High-throughput PM data literature             1,244       191            PM results EcoCyc, ability 33 compounds sup- <br /> High-throughput PM data generated group          353         0 <br />                                                                                    port growth E. coli K-12 sole carbon nitrogen source <br />  <br />  <br /> 984   jb.asm.org                                                                                                                            Journal Bacteriology <br />                                                                                                                                    Addition Growth Essentiality Data EcoCyc <br />  <br />  <br />  <br /> TABLE 2 Concordance aerobic datasetsa <br />                         % Agreement                                                                          % Y/N conflicts <br /> Data set                Carbon           Nitrogen           Phosphorus            Sulfur         All         Carbon           Nitrogen           Phosphorus            Sulfur          All <br /> AbuOun09                86               75                 89                    84             84          5                11                 5                     1               4 <br /> Baumler11               72               NA                 NA                    NA             72          21               NA                 NA                    NA              21 <br /> Bochner12               80               75                 91                    81             81          5                13                 3                     1               6 <br /> Mackie12                87               80                 93                    91             87          4                8                  4                     0               5 <br /> Yoon12                  86               75                 92                    75             84          4                3                  2                     1               3 <br />  <br /> All combined            83               76                 91                    82             83          6                9                  3                     1               6 <br /> a <br />   The left half table shows percent pairwise agreement different datasets sources carbon, nitrogen, phosphorus, sulfur, respectively. The right half the <br /> table shows percent pairwise Y/N (growth/no-growth) conflicts data. NA, applicable. <br />  <br />  <br />  <br /> remained unresolved (Table 4). We tested 19 compounds                                     The Baba06 (11) knockouts using lambda Red <br /> in minimal salts soft agar M9 minimal medium de-                                   recombinase system, replacing target genes in-frame kana- <br /> termine support aerobic growth sole ni-                                mycin resistance genes. It later subset these <br /> trogen source (Table 5), tested compounds deter-                                 mutants contained partial duplications were <br /> mine support aerobic growth sole carbon                                cross-contaminated (12). We combined data pub- <br /> source (Table 6). <span id='am-15' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-16' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-18'  ></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> information included conflict                                lications EcoCyc. A total 324 genes unable dis- <br /> resolutions future release EcoCyc.                                                     rupted predicted essential growth rich media <br />     Integration gene essentiality data. Table 7 lists gene                              37&#176;C. The 3,844 strains containing knockouts genes were <br /> essentiality data sets present version 17.5 EcoCyc.                             nonessential rich media (henceforth referred as <br /> These data sets include total 16,119 growth observations                              Keio collection) tested growth glucose <br /> knockout strains E. coli. Note numbers listed                                minimal medium (11). <br /> reflect number observations incorporated EcoCyc                                    Joyce et al. (13) profiled Keio collection growth on <br /> not necessarily numbers reported original studies. Dis-                             minimal medium glycerol sole source carbon and <br /> crepancies numbers caused subsequent                                   energy. A total 113 genes nonessential rich me- <br /> corrections data (which reflected numbers                              dium identified essential growth glycerol. Joyce et al. <br /> below), gene merges, deletions, etc.                                                           combined observations Baba et al. <br />     The Gerdes03 (10) study used genetic footprinting technique                              regarding conditional essentiality mutants grown on <br /> with Tn5-based transposon reported assessment                                  glucose-supplemented minimal media able identify a <br /> ca. 87% E. coli open reading frames essentiality. Although                              conserved conditionally essential core 94 genes required <br /> that study identified 620 genes essential aerobic growth                             E. coli K-12 grow minimal nutritional supplementa- <br /> rich media, actual experimental result transposon                               tion essential growth rich conditions. <br /> insertions identified genes. We did consider                                    Feist et al. (14) used experimental data regarding condi- <br /> that sufficient evidence essentiality incorporated                           tional gene essentiality earlier studies (11, 13) compared <br /> into EcoCyc observations 3,082 genes trans- <br /> poson insertions identified. Because transposon in- <br /> sertion result null mutation, genes identified non-                            TABLE 4 Nutrients generate conflicting responses phenotype <br /> essential data set fact required growth.                                 microarrays <br />                                                                                                Source                     Nutrients <br />                                                                                                Carbon                     L-Glutamic acid, Tween 20, D-psicose, <br /> TABLE 3 Summary PM data integration conflict <br /> resolutiona                                                                                                                 glucuronamide, &#65533;?&#65533;-methyl-D-glucoside, pectin, <br />                                                                                                                             &#65533;?&#65533;-keto-valeric acid, L-alaninamide <br />                               No. wells                                                     Nitrogen                   L-Cysteine, L-glutamic acid, L-lysine, L-methionine, <br />  <br /> Parameter status           Carbon          Nitrogen        Phosphorus           Sulfur                                   L-phenylalanine, L-threonine, D-asparagine, <br />                                                                                                                             D-lysine, D-valine, L-homoserine, putrescine, <br /> No. wells                  190             95              59                   35 <br />                                                                                                                             agmatine, D-mannosamine, N-acetyl- <br /> No growth                     99              38              5                    4 <br />                                                                                                                             D-mannosamine, adenine, guanosine, uracil, <br /> Low growth                    2               0               0                    0 <br />                                                                                                                             uridine, xanthine, xanthosine, uric acid, alloxan, <br /> Growth                        80              32              50                   23 <br />                                                                                                                             DL-&#65533;?&#65533;-amino-N-butyric acid &#65533;?&#65533;-amino-N-butyric <br /> Remaining conflicts           8 (9)b          25              4                    8 <br />                                                                                                                             acid, DL-&#65533;?&#65533;-amino-caprylic acid <br /> Resolved conflicts            52              10              6                    5 <br />                                                                                                Phosphorus                 Cytidine 3=,5=-cyclic mono-P, phosphono acetic acid, <br /> a <br />   Row 1 indicates total numbers wells nutrient type. Rows 2 4 indicate                                   2-aminoethyl phosphonic acid, methylene <br /> the numbers wells nutrient type showing different growth levels                                       diphosphonic acid <br /> growth assessed. Row 5 indicates number wells growth state <br />                                                                                                Sulfur                     L-Cysteic acid, cysteamine, S-methyl-L-cysteine, <br /> cannot assessed conflicting observations arbitrated. Row 6 <br />                                                                                                                             DL-ethionine, thiourea, 1-thio-6-D-glucose, <br /> gives number wells originally contained conflicts resolved our <br />                                                                                                                             DL-lipoamide, taurocholic acid <br /> arbitration procedure. The sum rows 2 5 equals row 1. <br /> b                                                                                              a <br />   The conflicting PM result recorded dulcitol (galactitol) genetic variation       Nutrients boldface growth/no-growth (as opposed growth/low-growth or <br /> between stock cultures E. coli K-12 MG1655.                                                 low-growth/no-growth) conflicts. <br />  <br />  <br />  <br /> March 2014 Volume 196 Number 5                                                                                                                                           jb.asm.org 985 <br />  Mackie et al. <br />  <br />  <br />  <br /> TABLE 5 Growth E. coli K-12 MG1655 minimal salts agar M9                 TABLE 7 Gene essentiality datasets present EcoCyca <br /> minimal medium sole nitrogen sourcesa                                                                                         No. knockouts <br />                                              Growth in: <br />                                                                                                                                    No                      Data <br />                                              Minimal                 M9 minimal   Data set      Medium                             growth     Growth       unavailable <br /> Nitrogen source                              salts agar              medium       Gerdes03      LB medium enriched,                           3,082        1,419 <br /> L-Cysteine                                   Y                       Y                            aerobic <br /> L-Glutamic      acid                         Y                       Y            Baba06        LB Lennox, medium,                 324        3,844        333 <br /> L-Lysine                                     Y                       Y                            aerobic <br /> Putrescine                                   Y                       Y            Baba06        MOPS &#11001; 0.4% glucose,               15         3,828        658 <br /> N-Acetyl-D-mannosamine                       Y                       Y                            aerobic <br /> Adenine                                      Y                       Y            Feist07       MOPS &#11001; 0.4% glucose,               107        996          3,398 <br /> Guanosine                                    Y                       NT                           aerobic <br /> Uric acid                                    Y                       N            Patrick07     M9 &#11001; 0.4% glucose, aerobic         106                     4,395 <br /> DL-&#65533;?&#65533;-Amino-N-butyric acid                    Y                       N            Joyce06       M9 &#11001; 1% glycerol, aerobic          113        3,704        684 <br /> L-Threonine                                  Y                       Y            a <br />                                                                                     For data set, list growth conditions gene <br /> L-Methionine                                 N                       Y            essentiality determined, number knockouts exhibiting growth, <br /> L-Phenylalanine                              N                       Y            growth, indeterminate level growth indicated. MOPS, <br /> D-Valine                                     N                       N            morpholinepropanesulfonic acid. <br /> Agmatine                                     N                       Y <br /> Uracil                                       N                       Y <br /> Uridine                                      N                       Y <br /> Xanthine                                     N                       N            chemical compounds comprise single growth medium. The <br /> &#65533;?&#65533;-Amino-N-butyric acid                       N                       Y            new schema <span id='am-12' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> &#8220;Growth-Observations&#65533;? defines more <br /> Alloxan                                      N                       N            measurements growth nongrowth specified <br /> a <br />   Growth curves M9 minimal medium provided Fig. S4 S5        growth media. This class specifies experimental condi- <br /> supplemental material. Y, yes; N, no; NT, tested.                             tions temperature, aerobicity, gene knockouts <br />                                                                                   strain studied. <br />                                                                                       Growth media growth observations entered <br /> them computationally predicted essential genes                    using interactive editing windows Pathway Tools. In addi- <br /> genome-scale metabolic reconstruction E. coli K-12. This data                  tion, growth observations imported data files, as <br /> set (which includes genes used model) included               importing PM data. Once data sets imported, the <br /> in EcoCyc facilitate benchmarking computational predic-                     user inspect compare growth data ways through <br /> tions essentiality EcoCyc model predictions                 Pathway Tools-based web site EcoCyc.org. The web site <br /> the model Feist et al.(14). Based ability Keio col-              command &#8220;Search &#161; Growth Media&#65533;? enables searching for <br /> lection single-gene knockout mutants grow rich versus de-                   growth media according chemical composition growth <br /> fined media, Baba et al. Joyce et al. (11, 13) identified set            outcome. In addition, user growth medium by <br /> conditionally essential genes. Patrick et al. (15) tested subset             navigating page listing growth observations recorded for <br /> of Keio mutants ability form colonies M9-glucose                  organism (http://biocyc.org/ECOLI/NEW-IMAGE?object <br /> agar.                                                                             &#11005;Growth-Media), contains series diagrams, including <br />     Growth essentiality data Pathway Tools soft-                   shown Fig. 1. Clicking PM image page <br /> ware. We extended Pathway Tools software (3) handle                   user page describing composition the <br /> several aspects growth observation data gene essentiality                  corresponding growth medium (see Fig. S1 supplemental <br /> data. A pathway/genome database (PGDB) EcoCyc                         material). The PGDB pages chemical compounds include links <br /> now capture growth observations wild-type mutant strains                    growth media contain compound. <br /> and capture descriptions media growth                         The growth-medium page (see Fig. S1 supplemental ma- <br /> assayed. The new schema class &#8220;Growth-Media&#65533;? lists set                     terial) lists composition growth medium forms: as <br />                                                                                   recipe substances used create medium (e.g., <br />                                                                                   ions listed salts) listing of constituents of <br /> TABLE 6 Growth E. coli K-12 MG1655 minimal salts agar M9                 medium (e.g., ion listed once, concentration <br /> minimal medium sole carbon sourcesa                                          summed multiple constituents necessary). When growth <br />                                          Growth in:                               data available, growth medium page lists wild- <br />                                          Minimal                     M9 minimal <br />                                                                                   type E. coli grows medium, growth status the <br /> Carbon source                            salts agar                  medium       single-gene knockouts listed available. <br />                                                                                       EcoCyc gene pages include table listing growth status of <br /> L-Glutamate                              N                           N <br /> &#65533;?&#65533;-Keto-valeric acid                      N                           N <br />                                                                                   knockouts gene growth media data are <br /> Tween 20                                 Y/N                         N            available (see Fig. S2 supplemental material). From page <br /> Pectin                                   N                           N            listing growth media, possible generate heatmap com- <br /> &#65533;?&#65533;-Methyl-D-glucoside                     N                           N            paring gene essentiality growth different nutrient sources. <br /> L-Alaninamide                            N                           N            Currently, EcoCyc, gene essentiality data available for <br /> a <br />   Growth curves M9 minimal medium provided Fig. S4 S5        growth glucose, glycerol, small number sub- <br /> supplemental material. N, no; Y/N, ambiguous.                                     strates. As PM data gene knockouts available, the <br />  <br />  <br /> 986   jb.asm.org                                                                                                                                Journal Bacteriology <br />                                                                                                          Addition Growth Essentiality Data EcoCyc <br />  <br />  <br />  <br /> power tool apparent. Figure S3           preclude possibility genetic variation stock <br /> supplemental material shows example heatmap data               cultures contribute variation PM data. <br /> currently available EcoCyc.                                                 We tested different pregrowth agars used the <br />                                                                            experimenters (LB, R2A, BUG-S) compared response <br /> DISCUSSION                                                                 PM plates 1 2 (see Fig. S5 supplemental material). <br /> Differences PM data sets. The addition high-                 Despite difference colony appearance agar plates, <br /> throughput Biolog PM data sets EcoCyc highlighted number              difference seen kinetic curves, thus <br /> of apparent conflicts data regarding E. coli K-12 MG1655&#8217;s          conclude variation PM response carbon sources <br /> ability use various compounds sole source carbon,             data sets collated present study to <br /> nitrogen, phosphate, sulfur. This integration exercise suc-         differences pregrowth conditions. Another potential contribu- <br /> cessful identifying conflicting information resolv-         <span id='am-64' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-65' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-66'  ></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>tor</span> disagreement PM data sets different labora- <br /> ing conflicts based low-throughput growth data            tories use different criteria distinguish growth, no <br /> from literature experimental results. It           growth, low growth. We unable investigate factor <br /> interesting briefly discuss possible reasons disparity       original growth curve data available of <br /> growth observation PM data highlights consider-             data sets. The existence international repository PM <br /> ations arise attempting unify data various               data included growth curves facilitate data reanal- <br /> sources.                                                                   ysis. <br />      One clear difficulty integrating data sets con-            Growth tests minimal salts agar M9 minimal medium <br /> founding effect result slight variations experi-          identified 15 compounds support growth sup- <br /> mental method used laboratory. Potential reasons dis-          plied sole nitrogen source despite generating conflicting re- <br /> agreement PM data sets added EcoCyc include (i)               sults PM data sets (Table 5). Varying incubation <br /> genetic differences E. coli K-12 MG1655 stock cultures, (ii)        time likely confounding factor nitrogen source PM assays. <br /> variation pregrowth conditions bacteria (LB, R2A,         Long lag times particularly noticeable response nitrogen <br /> BUG-S agar), (iii) variation length incubation times used        compounds (see Fig. S4 supplemental material, PM3) and <br /> for assay (26 48 h), (iv) variability choice cutoff   contribute higher conflict rate observed nitro- <br /> values assigning positive/negative results respiration      gen source plate. For example, nitrogen source PM assays, 6 <br /> PM assays.                                                                 compounds lag time &#11022;20 h, 10 additional com- <br />      Genetic variation stock cultures E. coli K-12              pounds lag time &#11022;24 h (see Data set S6 supplemen- <br /> MG1655 documented literature (16, 17) likely           tal material). This suggests important incubate Biolog <br /> have implications physiological experiments. It predicted,          nitrogen source plates 48 h recommended the <br /> for example, frameshift gatC present stock       manufacturer. Within nitrogen compounds tested, there <br /> cultures E. coli K-12 MG1655 result nonfunctional             variation detecting growth minimal <br /> product inability transport D-galactitol (dulcitol),      salts agar compared M9 minimal medium. For example, <br /> is likely differences PM data compound         growth E. coli K-12 using &#65533;?&#65533;-amino-N-butyrate uric acid as <br /> genetic variation stock cultures. We able identify        sole nitrogen source detected soft agar in <br /> different stock cultures E. coli K-12 MG1655 used          liquid culture, growth L-methionine, L-phenylalanine, <br /> generate data sets added EcoCyc: came central              agmatine, uracil, uridine, amino-N-butyrate sole nitrogen <br /> repositories (ATCC CGSC), laboratory-held                sources detected liquid minimal media soft <br /> strain. We tested stock cultures (ATCC 700926             agar. The agar used plate media contain impurities, and <br /> CGSC7740) PM plates 1 4 compared responses                 contact solid surface neighboring cells result in <br /> (see Fig. S6 supplemental material). Overall, little      differential regulation enzymes. <br /> variation response majo <br /> </body></html>