<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Li Liu BMC Systems Biology 2012, 6(Suppl 1):S11 <br /> http://www.biomedcentral.com/1752-0509/6/S1/S11 <br />  <br />  <br />  <br />  <br />  RESEARCH                                                                                                                                      Open Access <br />  <br /> Characterizing criticality proteins systems <br /> dynamics: <span id='am-27' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-28' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-33'  ></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span>Escherichia coli</span> central carbon <br /> metabolism working example <br /> Ru-Dong Li1, Lei Liu1,2* <br /> From The 5th IEEE International Conference Computational Systems Biology (ISB 2011) <br /> Zhuhai, China. 02-04 September 2011 <br />  <br />  <br />   Abstract <br />   Background: Systems biology calls studying system-level properties genes proteins their <br />   individual chemical/biological properties, regarding bio-molecules components. By characterizing <br />   critical components classifying accordingly, study underlying <br />   complex mechanisms, facilitating researches drug target selection, metabolic engineering, complex disease, etc. <br />   Up date, studies aiming goal confined topology-based flux-analysis approaches. <br />   However, proteins tertiary structures specific functions, especially metabolic systems. Thus topological <br />   properties connectivity, path length, etc., good surrogates protein properties. Also, manner <br />   individual sensitivity analysis flux-analysis approaches reveal simultaneous impacts on <br />   collateral components overall impact system, lacking system-level perspective. <br />   Results: In present work, developed method directly assess protein system-level properties based on <br />   dynamics silico knockouts, regarding conceptual term &#8220;criticality&#65533;?. Applying method E. <br />   coli central carbon metabolic system, multiple enzymes including phosphoglycerate kinase, enolase, <br />   transketolase-b, etc., critical roles terms states dynamical stability. In <br />   contrast, set enzymes including glucose-6-phosphate isomerise, pyruvate kinase, phosphoglucomutase, <br />   etc., exerted little influences deleted. The finding consistent experimental characterization of <br />   metabolic essentiality studies E. coli gene essentiality functions. We enzymes <br />   affect distant metabolites enzymes greater close neighbour asymmetry system-level <br />   properties enzymes catalyzing alternative pathways rise local flux compensation. <br />   Conclusions: Our method creates different angle evaluating protein criticality biological the <br />   conventional methodologies. Moreover, method leads consistent results experimental references, <br />   showing efficiency studying protein system-level properties. Besides working metabolic systems, the <br />   application method extended kinds bio-systems reveal constitutive/functional <br />   properties building blocks. <br />  <br />  <br /> Background                                                                           components. By exploring interplay structure in <br /> Systems biology focuses studying properties bio-                               system, evaluate critical building <br /> molecules like genes proteins level,                               block different parts vary properties [1,2]. <br /> especially constitutive/functional roles                             Based knowledge, understand a <br />                                                                                      formed, system-level function is <br /> * Correspondence: leiliu@sibs.ac.cn                                                  achieved modified according to <br /> 1 <br />  Key Laboratory Systems Biology, Shanghai Institutes Biological               needs, enhancing researches drug target selection, <br /> Sciences, Chinese Academy Sciences, Shanghai, China                               metabolic engineering, complex diseases, [3,4]. E. <br /> Full list author information available end article <br />  <br />                                         &#169; 2012 Li Liu; licensee BioMed Central Ltd. <span id='am-9' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-11'  ></span><span id='am-12' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> Open Access article distributed terms Creative Commons <br />                                         Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction in <br />                                         medium, provided original work properly cited. <br />  Li Liu BMC Systems Biology 2012, 6(Suppl 1):S11                                                           Page 2 14 <br /> http://www.biomedcentral.com/1752-0509/6/S1/S11 <br />  <br />  <br />  <br />  <br /> coli best-studied organism, knowledge accu-       fluctuations restricted limited area spread <br /> mulated biological hierarchies, e.g. genetic   broader range; large ampli- <br /> regulation, genomic information, metabolism, [5-7].       tudes were. Moreover, investigated dynamical <br /> The central carbon metabolism contains glycolysis         stability residual sys- <br /> pentose phosphate pathways principal parts (Addi-          tem maintained lost metabolic robustness after <br /> tional file 1). It common conservative        removing enzyme (Figure 1). From computa- <br /> pathway prokaryotes, close resemblance          tions, characterized criticality proteins our <br /> eukaryotes [5,7,8].                                           results consistent published experiments. <br />   Up date, multiple genome-scale networks        Furthermore, method create new viewpoint <br /> built organism regard pathway           protein system-level property characterization, which <br /> reveal essentiality genes proteins [9,10]. However,    differs conventional methodologies more <br /> most studies based network topology         comprehensive analyzing complex systems. <br /> flux analysis. In topology-based approaches, system-level <br /> properties defined connectivity molecule      Results <br /> or shortest path lengths, [11,12]. Such properties        We computed criticality enzymes sys- <br /> usually poor consistency experimental charac-       tem, observed categorized two <br /> terizations, especially protein level. For example,    classes: critical properties with <br /> multiple studies suggest proteins large connec-     uncritical properties. An enzyme characterized cri- <br /> tivity protein-protein interaction networks        tical deletion caused large influences system <br /> essential. Also, enzymes associated large           states qualitative changes dynamical <br /> number accompanies exert little influence          stability. <br /> cell growth [6,13,14]. We think possible reason is <br /> that mere topology does encode specific biochem-          System state fluctuation <br /> ical/biological functions proteins, topology-      We simulated obtain metabolite <br /> based approaches purely regard bio-molecules           kinetics flux distributions normal conditions. <br /> vertices abstract graph. While flux-analysis         Next, carried silico enzyme knockouts by <br /> approaches (e.g. flux balance analysis - FBA; metabolic       modifying corresponding parameters re-simulat- <br /> control analysis - MCA), calculate extent         ing system. Following <span id='am-75' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> previous stu- <br /> how perturbation parameter influences         dies, regarded concentrations primary system <br /> specific, pre-defined objective. Although         state [6,16]. We calculated state deviations modi- <br /> individual sensitivity analyses quantitative <span id='am-19' about='obo:NCBITaxon_1178312' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_572719 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41011 obo:NCBITaxon_41197 obo:NCBITaxon_41012 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Mea" datatype="xsd:string"></span>mea</span>-      fied computed fluctuation amplitude of <br /> sure component&#8217;s control functional pathway,        metabolite enzyme&#8217;s removal. Here we <br /> they reveal simultaneous impacts          encoded vector f. Second, assessed the <br /> parts the overall system. In          impact area calculating distances metabolites <br /> words, pre-defined objective interest,      removed enzyme encoded a <br /> will know perturbation triggers collateral influ-    vector d. This allowed influence <br /> ences parts impli-         limited radius propagated distant <br /> cates overall system. The lack system-level         parts system. In short, used vector pair U = <br /> perspective possibly gives rise false positive results,    (d,f) represent state fluctuation, could <br /> because simulated perturbation favouring objective       quantify overall impact measurement formula <br /> may actually carried know       (see section &#8220;Methods&#65533;? details). All results were <br /> has lethal impacts collateral crucial components       summarized Additional file 2. <br /> in system.                                                   We discovered enzymes exert (upon <br />   Under consideration, developed new                deletion) large influences kinetics many <br /> method characterize protein criticality based           metabolites, i.e. caused large state fluctuations <br /> kinetic systems, accurately reflect          deleted. For example, transketolase-b (TKb), an <br /> behaviours explicit context biophysical/       enzyme catalyzing coupling branch glycolysis <br /> biochemical basis [5,15]. Because E. coli central carbon      pentose phosphate pathways, relatively <br /> metabolism comprehensive              large influences metabolites central <br /> kinetic <span id='am-76' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-77'  ></span><span id='am-78' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span>, used model. The com-        carbon metabolism, especially glucose, sedoheptu- <br /> ponents (bio-molecules) enzymes,              lose-7-phosphate, erythrose-4-phosphate (Figure <br /> defined criticality component silico       2A). Phosphoglycerate kinase (PGK), enzyme cata- <br /> knockout. We explored deletion enzyme           lyzing conversion 1,3-diphosphoglycerate <br /> influenced state, i.e. state               3-phosphoglycerate linear branch the <br />  Li Liu BMC Systems Biology 2012, 6(Suppl 1):S11                                                                                          Page 3 14 <br /> http://www.biomedcentral.com/1752-0509/6/S1/S11 <br />  <br />  <br />  <br />  <br />  Figure 1 Schematic illustration protein criticality characterization method. System components exert different impacts when <br />  perturbed. Color gradient corresponds amplitudes, color representing zero effect. (A) Circumstance impact a <br />  limited area near epicenter (i.e. perturbed component, marked &#8220;1&#65533;?). (B) Circumstance impact spreads network, <br />  distant spots severely affected (marked &#8220;2&#65533;?). (C) A stable equilibrium (Xeq) attracts neighboring trajectories. (D) An unstable <br />  equilibrium repels trajectories. Stable equilibriums collapse perturbations critical components. Components whose <br />  perturbations exert large impacts, cause qualitative changes dynamics (upon perturbation) regarded critical. <br />  <br />  <br /> glycolysis pathway, exerted greater impacts                            acid (TCA) cycle, polysaccharide synth- <br /> these metabolites ones                         esis, external pathway connected central <span id='am-65' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-66' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68'  ></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>car</span>- <br /> system glucose-6-phosphate, fructose-6-<span id='am-81' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-82' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-83'  ></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>phos</span>-                           bon metabolism (Figure 2B). Similarly, enzymes at <br /> phate, glyceraldehydes-3-phosphate, ribulose-5-phos-                           locations enolase (ENO), glyceralde- <br /> phate, etc. Moreover, exert large impact                         hydes-3-phosphate dehydrogenase (GAPDH), ribose-5- <br /> on oxaloacetate, intermediate tricarboxylic                          phosphate isomerase (R5PI), aldolase (ALDO), <br />  <br />  <br />  <br />  <br />  Figure 2 System state fluctuations caused enzyme deletions. The impacts enzyme deletions states shown. In all <br />  subfigures, x-<span id='am-14' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-15' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18'  ></span>axis</span>: state variable (metabolite) indexes (denoted &#8220;Metab&#65533;?, Additional file 6 details); y-axis: distance state variable from <br />  deleted enzyme (&quot;Dist&#65533;?); z-axis: impact amplitude (&quot;Amp&#65533;?). (A - D) The influences TKb, PGK, TIS, PGI deletions state, <br />  respectively. The figures exemplify enzyme affect distant metabolites greater closest neighbours. <br />  Li Liu BMC Systems Biology 2012, 6(Suppl 1):S11                                                          Page 4 14 <br /> http://www.biomedcentral.com/1752-0509/6/S1/S11 <br />  <br />  <br />  <br />  <br /> transaldolase (TA), etc., exhibited large impacts    similar pattern results presented (Addi- <br /> system states (Additional file 2). The overall influences    tional file 3). <br /> of PGK, ENO, GAPDH superior of <br /> ALDO, R5PI, TKb TA, especially distance &#8805;         System dynamical stability <br /> 4 5 (Figure 2, Additional file 2). This indicated        We original asymptotically <br /> that enzymes like PGK, ENO GAPDH                   stable equilibrium point Xeq large range ordinary <br /> impact distant areas strongly exert          intracellular concentrations parameter/state space, <br /> persistent impact respect structure.          trajectories wide neighborhood <br /> Noteworthy, triosephosphate isomerase (TIS),           tending (Figure 3). This gives rise metabolic <br /> catalyzed inter-conversions glyceraldehydes-3-       robustness, slight perturbations initial values do <br /> phosphate isomer dihydroxyacetonephosphate,          cause large changes states [17,18]. As is <br /> would regarded peripheral component           accord Lyapunov stability, could <br /> system traditional topology-based flux-analysis       characterize enzyme&#8217;s criticality examining the <br /> approaches uni-directional rate      bifurcations X eq respect deletion the <br /> limiting steps. However, computation results             enzyme. Such bifurcations included: (1) deleting <br /> showed deletion resulted large impact       enzyme residual equili- <br /> (Figure 2C). The difference prediction        brium; (2) residual equilibrium(s), <br /> our method assumed influence exerted          far location deviated Xeq; (3) whether <br /> enzyme depend location (network             stability property changed (i.e. changes in <br /> topology), determined parametric prop-       neighborhood orbit structure). Equilibrium(s) was <br /> erties (kinetic parameters).                                 computed dynamical simulation optimization <br />   Meanwhile,           methods. When located, deviation Xeq <br /> group enzymes, contrast mentioned             calculated neighborhood orbit structure was <br /> above, having little influences states. For   described rules topological conjugacy (see the <br /> example, multiple enzymes glucose-6-phosphate        &#8220;Methods&#65533;? section details). As multiple enzyme dele- <br /> isomerase (PGI), pyruvate kinase (PK), phosphogluco-         tions generate topologically identical orbit struc- <br /> mutase (PGM), etc., resulted small state        tures, showed typical cases examples here. <br /> fluctuations deleted. The amplitudes slight,       See Additional file 4 complete catalogue all <br /> and influences limited areas       results. <br /> the amplitudes negligible distance &#8805; 2 (Fig-        After silico knockout TKb, residual system <br /> ure 2D, Additional file 2). Hence, system-level prop-    large qualitative changes dynamics. It <br /> erties far state fluctuation considered,    exhibited equilibrium far away Xeq differ- <br /> the enzymes greater         ent stability property (Figure 4A-4B). It unstable <br /> latter group.                                                equilibrium trajectory representing sedoheptu- <br />   More interestingly, severely        lose-7-phosphate kinetics divergent two <br /> influenced metabolites did concentrate     dimensions representing ribulose-5-phosphate xylu- <br /> close neighborhood perturbed enzyme. For exam-        lose-5-phosphate forming limit cycle certain <br /> ple, largest impacts TKb deletion dis-    initial values held. By setting different initial values on <br /> tances 2 3 distance 1 (Figure       2-dimensional plane limit cycle investi- <br /> 2A). Likewise, largest impacts PGK deletion           gating trajectory dynamics, seen limit <br /> occurred distances 3 4 dis-    cycle unstable one. Trajectories plane <br /> tance 1 (Figure 2B). Similar patterns seen      inside range converged equilibrium&#8217;s projec- <br /> from results enzymes like ENO, R5PI,            tion plane; trajectories outside range <br /> ALDO, GAPDH, TA PGI (Figure 2, Additional file           spread quickly dimensions (Figure 4B). <br /> 2). This suggested contrast intuition    Likewise, deleting TA caused equilibrium to <br /> perturbation cause largest changes neigh-       relocate similar distance similar proper- <br /> borhood, distant effects occur leverage     ties case TKb. It unstable <br /> of dynamics.                                          dimension divergent another <br />   We examined impacts enzyme knockouts           dimensions forming unstable limit cycle. What <br /> from enzyme-centric view method. With           differs TKb divergent dimension <br /> each enzyme representing reaction using fluxes      6-phosphogluconate cycling dimensions <br /> system states, computed flux change amplitudes        xylulose-5-phosphate sedoheptulose-7-phos- <br /> impact radiuses enzyme-centric network         phate. R5PI knockout equilibrium shift a <br /> same way stated previously. The results showed             long distance reversed stability. For the <br />  Li Liu BMC Systems Biology 2012, 6(Suppl 1):S11                                                                                        Page 5 14 <br /> http://www.biomedcentral.com/1752-0509/6/S1/S11 <br />  <br />  <br />  <br />  <br />  Figure 3 The asymptotically stable equilibrium point. The stable equilibrium (Xeq) illustrated trajectories phase orbits 18D <br />  space 10% perturbations. The red line: original curve respect experimental initial value [5]; yellow/blue lines: curves <br />  initial value -10%/+10% perturbation experimental one; triangular spots: projections Xeq corresponding <br />  dimensions. (A) Asymptotical stability shown trajectories. Each subplot represents dimension 18D space, i.e. kinetics a <br />  metabolite. All trajectories eventually consistently converge Xeq (projection corresponding dimension) 10% <br />  perturbation initial value. The x-axis: time (s); y-axis: concentration (mM). (B) Asymptotical stability shown phase orbits. Stability is <br />  clearly illustrated presentation. We randomly chose 2 state variables (metabolites #7 #8 plot) form phase orbit in <br />  2D subspace. Arrows denote directions orbits eventually consistently converge Xeq (projection 2D subspace, <br />  marked triangular spot). For 2D subspaces, orbit profiles same. <br />  Li Liu BMC Systems Biology 2012, 6(Suppl 1):S11                                                                                          Page 6 14 <br /> http://www.biomedcentral.com/1752-0509/6/S1/S11 <br />  <br />  <br />  <br />  <br />  Figure 4 Equilibrium deviations orbit structure changes caused enzyme deletions. Impacts enzyme deletions dynamical <br />  stability shown terms equilibrium deviations orbit structure changes. The curves drawn principle topology <br />  conjugacy. They qualitative dynamics real-value trajectories. Lines different colors represent different curves <br />  initiated different values (init 1-3). In subfigure, 3 dimensions represent metabolites differ original kinetics. (A) <br />  The original equilibrium (Xeq), denoted (0,0,0). All trajectories converge it. (B) The unstable equilibrium deleting TKb. The dimension <br />  sed7p divergent, (ribu5p, xyl5p) form unstable limit cycle. The 3 colored orbits initiated different values lead convergence, limit cycle <br />  divergence 2D plane (ribu5p, xyl5p). (A) (B) separately drawn <span id='am-24' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-25' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> limit cycle clearly. Orbits centred at <br />  (0,0,0) achieve better visual effect. (C) Deleting R5PI causes long-distance equilibrium deviation alters stability. The source <br />  target ends grey arrow mark Xeq re-established equilibrium (Xd) respectively, distance (in metric unit state <br />  space) marked box. Initiated identical values, Xeq attracts orbits Xd repels orbits dimensions fdp 3pg. (D) <br />  Deleting TIS causes equilibrium deviate larger distance. Legends same. (E) Deleting PGK makes no <br />  equilibrium. The deviation denoted infinite (&#8734;). (F) Deleting PGI does cause obvious changes dynamics. The re-established <br />  equilibrium asymptotically stable near Xeq. See Additional file 6 abbreviations metabolite names. <br />  <br />  <br /> neighborhood orbits, divergent dimensions                             deletion, residual run- <br /> fructose-1,6-biphosphate 3-phosphoglycerate (Figure                        ning own, approach extreme posi- <br /> 4C).                                                                           tion regular state space special <br />   Deleting TIS ALDO caused re-                                dynamics. In words, residual could <br /> establish equilibrium extreme distance                          maintain regular operating functional- <br /> the ordinary range (Figure 4D). This indicated                            ity, deletions ALDO R5PI both <br />  Li Liu BMC Systems Biology 2012, 6(Suppl 1):S11                                                            Page 7 14 <br /> http://www.biomedcentral.com/1752-0509/6/S1/S11 <br />  <br />  <br />  <br />  <br /> regarded having large qualitative influence       close normal [16]. Since <span id='am-70' about='obo:NCBITaxon_997520' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_95175 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-71'  ></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="Moma" datatype="xsd:string"></span>MOMA</span> accepted a <br /> dynamics.                                                      rationale, adopted flux simulation enzyme <br />   Moreover, equilibrium         deletions, formulating computation optimiza- <br /> deleting enzymes PGK, ENO GAPDH (Figure             tion problem solving numerically (see section <br /> 4E). This meant original equilibrium              &#8220;Methods&#65533;? details). We flux-sums of <br /> destroyed residual re-establish       essential metabolites reduced than <br /> another one. This residual         50% deleting enzymes predicted as <br /> upon removal of enzymes            critical (Figure 5A-5C, Additional file 5), dele- <br /> so ill-suited trajectories did exhibit nor-   tions result 50% reduction in <br /> mality well-imposed biological kinetic systems,          cell growth according Kim et al. On side, <br /> which trajectories tended stabilize near           deleting (predicted) uncritical enzymes did <br /> regions state space. This indicated       cause flux-sums drop 50% (Figure <br /> residual system, operating own, effec-     5D-5E, Additional file 5), relatively mild <br /> tively maintain functionality. Hence, deletions     effect cell growth. This indicated predicted <br /> PGK, ENO GAPDH regarded having                critical enzymes weight functional <br /> larger qualitative influences dynamics com-          essentiality uncritical enzymes, sup- <br /> pared previously mentioned enzymes.                   ported characterizations criticality. <br />   In contrast above, enzymes like PGI, PK             We compared results E. coli gene <br /> PGM showed different property. After deleting          essentiality studies Keio collection, the <br /> anyone them, residual equili-       genetic footprinting study Profiling E. coli <br /> brium locating near Xeq. Moreover, equili-        Chromosome (PEC) database, results sup- <br /> brium asymptotically stable,                 ported experimental characterizations. <br /> dimensions converging (Figure 4F). Therefore, PGI,       For example, &#8220;critical&#65533;? enzymes PGK GAPDH <br /> PK PGM knockout did qualitatively change            encoded genes pgk gapA respectively. And <br /> system dynamics. Hence, system-level properties         2 genes characterized essential studies <br /> far dynamical stability considered, enzymes like        Keio collection genetic footprinting <br /> PGK, ALDO, TKb, etc. critical enzymes           [14,19]. ENO encoded gene eno gene is <br /> like PGI, PK, PGM. Based above,            essential, according Keio collection the <br /> see <span id='am-6' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> enzymes exemplified PGK,              PEC database [19,20]. Moreover, gene fbaA, which <br /> ENO, TKb, ALDO, TIS, R5PI, GAPDH, TA                  encodes ALDO, characterized essential the <br /> critical properties terms impact          Keio collection, genetic footprinting PEC data- <br /> states dynamical stability. And class         base [14,19,20]. Furthermore, &#8220;uncritical&#65533;? enzymes <br /> enzymes exemplified PGI, PK, PGM oppo-              PGI, PK PGM encoded genes pgi, pykF and <br /> site properties. Therefore, class charac-       pgm respectively, 3 genes characterized <br /> terized &#8220;critical&#65533;? &#8220;uncritical&#65533;?.         nonessential Keio collection, genetic foot- <br />                                                                printing PEC database [14,19,20]. Such compari- <br /> Comparison experimental characterizations                 sons showed predictions consistent with <br /> We compared characterizations system-level              experimental results (Table 1). In addition, ribulose-5- <br /> properties characterizations essentiality     phosphate epimerase (Ru5P) encoded gene rpe. <br /> basis multiple (previous) validated studies. Kim et         However, gene characterized essential by <br /> al.&#8217;s work E. coli metabolism defined set essen-       genetic footprinting nonessential Keio collec- <br /> tial metabolites demonstrated flux-sum         tion PEC database. Given Ru5P critical to <br /> of essential metabolite reduced 50%,           central carbon metabolic revealed our <br /> the cell growth rate decrease 50%           method verified flux-sums essential <span id='am-50' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-51'  ></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>meta</span>- <br /> correspondingly [6]. There 12 metabolites         bolites (earlier context), propose viewpoint <br /> our working model examined flux-sums           criticality gene rpe essential. <br /> utilizing simulation power kinetic model with <br /> respect perturbations (i.e. enzyme deletion). A naive       Discussion <br /> method modifying corresponding enzymatic               Studying system-level properties bio-molecules is <br /> parameter zero leaving rest        essential systems biology [1,2]. But studies are <br /> they originally were. However, theory Minimiza-         based network topology working <br /> tion Metabolic Adjustment (MOMA) suggested             protein level, flux analysis lacks <br /> when severe perturbation occurred,                level perspective [13,14,21]. To overcome <br /> adjusted itself extent state        drawbacks, propose method criticality <br />  Li Liu BMC Systems Biology 2012, 6(Suppl 1):S11                                                                                                    Page 8 14 <br /> http://www.biomedcentral.com/1752-0509/6/S1/S11 <br />  <br />  <br />  <br />  <br />   Figure 5 The flux-sums essential metabolites enzyme deletions. The flux-sum values 12 essential metabolites <br />   enzyme knockout shown. The x-axis: metabolites; y-axis: flux-sum values. The blue bars: flux-sum values the <br />   original system; red bars: flux-sum values enzyme knockout. (A - C) Flux-sums essential metabolites the <br />   knockouts TKb, PGK TIS respectively. All metabolites suffer devastating flux reductions. (D - E) Flux-sums essential metabolites upon <br />   knockouts PGI PGM respectively. All metabolites&#8217; flux-sums sustained high level compared original values. The <br />   observations support conclusion enzymes critical ones. See Additional file 6 abbreviations of <br />   metabolites. <br />  <br />  <br />  <br />  <br /> Table 1 Comparison multiple E. coli gene essentiality studies <br /> Protein            Gene            Criticality           Flux-sum             Keio collection             Genetic footprinting               PEC database <br /> PGK                pgk             Critical              &#8730;                    &#8730;                           &#8730; <br /> GAPDH              gapA            Critical              &#8730;                    &#8730;                           &#8730; <br /> ENO                eno             Critical              &#8730;                    &#8730;                                                              &#8730; <br /> ALDO               fbaA            Critical              &#8730;                    &#8730;                           &#8730;                                  &#8730; <br /> PGI                pgi             Uncritical            &#8730;                    &#8730;                           &#8730;                                  &#8730; <br /> PK                 pykF            Uncritical            &#8730;                    &#8730;                           &#8730;                                  &#8730; <br /> PGM                pgm             Uncritical            &#8730;                    &#8730;                           &#8730;                                  &#8730; <br /> Ru5P               rpe             Critical              &#8730;                                                &#8730; <br /> G1PAT              glgC            Uncritical            &#8730;                    &#8730;                           &#8730;                                  &#8730; <br /> Our characterizations protein criticality (Col 3) compared multiple gene essentiality studies (Col 5-7) validations flux-sum (Col 4). <br /> Note: &#8220;&#8730;&#65533;? means evidence supports characterization criticality <br />  Li Liu BMC Systems Biology 2012, 6(Suppl 1):S11                                                             Page 9 14 <br /> http://www.biomedcentral.com/1752-0509/6/S1/S11 <br />  <br />  <br />  <br />  <br /> characterization basis kinetic modeling. In        encoding TIS) attenuated cell growth 70%. <br /> kinetic system, interaction expressed            And method appropriately revealed TIS could <br /> kinetic rate equation. How component influences          exert large impacts deleted, of <br /> system determined position              designs (mentioned above). Hence metho- <br /> kinetic parameters. Position equivalent topological      dologically, method creates different angle from <br /> property, kinetic parameters encode specific bio-        topology-based methods viewed an <br /> chemical/biological functions. Both kinds information       improvement conventional flux-analysis approaches. <br /> are integrated modeling revealed dynamical              After silico deleting protein, residual is <br /> simulation [15,22]. According typical formulism         actually virtual structure. We assume struc- <br /> of biochemical systems, kinetic rate equations con-        ture encodes important information the <br /> stitute deterministic complex           mutant maintain functionality would <br /> dynamics viewed &#8220;driving force&#65533;?         dynamically behave/evolve provided stills operates <br /> of [23]. Thus theoretically, criticality        own. The residual fails maintain func- <br /> characterization proposed method study        tionality kinetics <span id='am-55' about='obo:NCBITaxon_376232' typeof='obo:NCBITaxon_34667 obo:NCBITaxon_79514 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71528 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_192380 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-56'  ></span><span id='am-57' property="rdfs:label" content="Goes" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>goes</span> regular ranges (e. <br /> structural factors built &#8220;driving force&#65533;?         g. occurring negative values, soaring extreme <br /> system.                                                        values exceeding regular intracellular molecular concen- <br />   Differing topology-based methods, method            trations), dynamics trapped mode where <br /> characterizes system-level properties quantitative      stable equilibrium sabotaged, stable equilibrium <br /> basis. But unlike conventional sensitivity analysis,    gives rise robustness essential prerequisite <br /> employ network structure information calculating        valid mathematical formulations living cell <br /> the distances deleted spot affected enti-      dynamics [17,24-26]. Either case indicates deleting <br /> ties computing fluctuations. Moreover,             protein makes ill-suited cannot <br /> unlike conventional flux-analysis approaches, explore       run own. <br /> the stability retrieve dynamics struc-          By applying method E. coli central carbon <br /> ture. Incorporating network/dynamics structure             metabolism, deleting enzymes PGK, <br /> information allows reveal simultaneous/collat-       GAPDH, etc. causes ill-sui- <br /> eral influences overall impact system.          ted structure state values soaring levels <br /> Another major difference sensitivity analysis      normal range trajectories highly <br /> that use silico deletions instead mild perturba-      divergent state space (Figure 2B and <br /> tions (e.g. 5% 10%, flux-analysis approaches        4B). Likewise, deleting enzymes TKb, ALDO, <br /> do). Because well-casted biological network usually          etc. causes relatively large impacts system <br /> has parametric properties favouring robustness          kinetics qualitative dynamics (Figure 2A 4A). <br /> dynamics, critical components tolerate mild           On contrary, knocking enzymes PGI, <br /> perturbations (i.e. parameters exhibiting Lyapunov         PK, etc. exerts small influences (Figure 2D 4E). <br /> stability). Therefore, individual sensitivity analysis   We enzymes mediate large influences on <br /> fails identify critical spots, inability    distant metabolites enzymes. For instance, TKb, PGK, <br /> reveal simultaneous influences worsens situation.          PGI, etc. exert largest impacts entities of <br /> That develop &#8220;criticality characterization&#65533;?.     distances 1 (Figure 2A, 6A-6B Additional <br /> In silico deletion equivalent investigating      file 2, 3). This bio-systems complex <br /> system component forcefully assumed         structures consisting branches, alternative pathways <br /> to absent, eliminating parametric properties sta-       loops, various kinetic parameters differing <br /> ted earlier. Furthermore, method&#8217;s capacity             orders magnitudes [6,24]. Such structure acts a <br /> revealing simultaneous/overall impacts           special leverage, determining special ways interactions <br /> level enables distinguish real critical spots       influence propagations. Only kinetic modeling can <br /> uncritical ones effectively. In addition, utilizing       reveal knowledge, analyses us <br /> kinetic model analytical basis superiority    clues selecting potential regulatory targets for <br /> the stoichiometric flux-balance modeling traditional        use drug development, metabolic engineering, etc. <br /> flux-analysis methods, enabling appropriately               By utilizing power kinetic model approaching <br /> explore behaviours real-time scale [15].         real-time events, simulated fluxes enzyme dele- <br /> For example, traditional topology-based flux-         tions compared results previously validated <br /> analysis approaches regard TIS peripheral      study metabolic essentiality [6]. The comparison shows <br /> highly connected uni-directional       characterization criticality supported by <br /> rate limiting reactions. However, experimen-        functional essentiality. Interestingly, discovered that <br /> tal studies showing knocking tpiA (i.e. gene      asymmetry criticalities building blocks give <br />  Li Liu BMC Systems Biology 2012, 6(Suppl 1):S11                                                                                     Page 10 14 <br /> http://www.biomedcentral.com/1752-0509/6/S1/S11 <br />  <br />  <br />  <br />  <br />  Figure 6 Remote impact local flux compensation. A component affect distant entities greater closest <br />  neighbours, illustrated TKb. Moreover, asymmetry criticalities components result local flux compensation, is <br />  illustrated alternative paths TKb, TA PGI. (A) An explicit demonstration Figure 2A biological network overlay. It is <br />  organized metabolite network TKb highlighted blue metabolites arranged green circles representing distance levels. The <br />  amplitude proportional color gradient (upper-right corner, &#8220;red-yellow&#65533;? corresponds &#8220;strong-slight&#65533;?). (B) The enzyme-centric view of <br />  (A). Subfigure (A) (B) mainly distributions distances amplitudes network&#8217;s view, exact vertices&#8217; labels are <br />  important here. To pictures high resolution, refer Additional file 7 8. (C) Metabolites ribu5p, sed7p, rib5p e4p the <br />  pentose-phosphate pathway increased fluxes PGI knockout asymmetry system-levels properties PGI, TKb TA. The <br />  enzymes mutually form alternative paths associating essential metabolite f6p. <br />  <br />  <br /> rise local flux compensation. For instance, multiple                      mainly mediated alternative pathways [6]. When a <br /> metabolites (e.g. ribulose-5-phosphate, sedoheptulose-7-                     critical component deleted, leaving inferior components <br /> phosphate, etc.) pentose phosphate pathway                       backup rely on, work efficiently. <br /> increased flux-sums PGI knockout (Figure 5D                        On contrary, deleting PGI leaves alternative <br /> 6C). The cutoff PGI induces alternative path-                     pathways superior properties &#8220;ON&#65533;? state <br /> ways generating essential metabolite fructose-6-                     works, fluxes/states effi- <br /> phosphate, TKb TA, operate greater volume.                       ciently compensated. This gives hint criticality <br /> Thus fluxes relative reactions compensated,                      characterization help bio-system modifications such <br /> resulting local amplified fluxes. This likely result              metabolic engineering. We delete system <br /> accordance MOMA mechanism [17]. Although                              components inferior properties, leaving alternative <br /> MOMA compensate fluxes/states                             pathways superior properties work. And pheno- <br /> degrees, results effects caused dele-                   types local areas relating alternative path- <br /> tions critical components TKb, TA, PGK, etc.                      ways compensated leverage system <br /> cannot smoothed compensations (Figure 5A-                         structure MOMA mechanism. Therefore, compre- <br /> 5B, Additional file 5). This compensations                   hensive methods exploring system-level properties can <br />  Li Liu BMC Systems Biology 2012, 6(Suppl 1):S11                                                            Page 11 14 <br /> http://www.biomedcentral.com/1752-0509/6/S1/S11 <br />  <br />  <br />  <br />  <br /> help make use bio-complexity engineering,      useful revealing general principles governing <br /> as knowledge discovery.                                      complex bio-systems. Naturally, complicated bifurcation <br />    It noteworthy functionally important compo-          dynamics harder analyze; <span id='am-1' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-2' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5'  ></span>idea</span> our <br /> nents necessarily critical, studies suggest     method applied coexisting dynami- <br /> the important reaction function,          cal characteristics bifurcation associated bio- <br /> likely backup pathway [6,13]. For example,        logical implications [28]. By integrating knowledge and <br /> PK connects fundamental chemical compounds             using theoretical generic forms models [15,30], <br /> it regarded uncritical level,       kinetic modeling eventually feasible more <br /> there alternative paths (e.g. phosphotransferase        organisms. Hence, instead traditional approaches, <br /> system - PTS, bacteria glycolysis; Additional file 1)        propose complex systems studied casting <br /> that prevent large impacts kinetics/              network kinetic equations computing the <br /> dynamics. This exemplifies bio-system components           system-level properties respect kinetics/ <br /> have dichotomy. They &#8220;importance&#65533;? biochemical           dynamics (criticality). Overall, method provide <br /> molecules, &#8220;criticality&#65533;?       new viewpoint revealing constitutive/functional <br /> as constitutive building blocks. Actually, method           properties building blocks biological system. <br /> does aim &#8220;functionally important&#65533;? mole- <br /> cules, &#8220;critical&#65533;? system, i.e. components      Conclusions <br /> that absent, severely aber-     Our method creates new angle traditional topol- <br /> rant. Since criticality enzyme depends        ogy-based methodologies evaluating system-level prop- <br /> factors (e.g. kinetic parameters, substrates inhibiting/acti-   erties bio-molecules. Moreover, proposed method <br /> vating reactions, degree effects, etc.),       viewed improvement conventional <br /> assignments boundaries modeling              flux-analysis approaches FBA MCA. In addi- <br /> affect prediction results. As enzymes located        tion, method leads results consistent with <br /> boundary incomplete interplay structure,         experimental references, showing efficient in <br /> above factors occur properly kinetic equa-       characterizing protein criticality studying biological <br /> tions. Therefore, accurate criticality characterization      systems. Furthermore, method&#8217;s application be <br /> facilitated appropriate inclusiveness model-       extended types bio-systems (e.g. transcriptional <br /> ing. For example, glucose-1-phosphate adenyltransferase         networks signaling networks) reveal constitu- <br /> (G1PAT) connects external polysaccharide               tive/functional properties building blocks. <br /> synthesis pathway, interactions large-capa- <br /> city reactions outside pathways.         Methods <br /> Thus boundary assigned it, validity      Kinetic modeling <br /> the results enhanced (Table 1). Furthermore, funda-         We utilized existing kinetic data E. coli central carbon <br /> mental, common conserved pathways cho-              metabolism adopted previous modeling frame- <br /> sen comparison genome-scale gene essentiality          work working platform [5]. The kinetic model <br /> studies regard global cellular functionality. For       consists 30 metabolites (including external metabo- <br /> instance, bacteria central carbon metabolism        lites biosynthesis products) 30 biochemical <br /> an appropriate example [5,7,8], various predictions        reactions (24 enzymes 6 lumped reactions standing <br /> of protein criticality consistent global gene     transport/biosynthetic processes relating external <br /> essentiality characterizations [13,14,19].                      pathways; Additional file 1). The model re- <br />    Although used metabolic working           casted enzyme-centric network, adding a <br /> model, application method confined         directed connection enzyme A B A&#8217;s <br /> metabolic systems. For instance, model gene              products B&#8217;s substrate. We explicitly the <br /> transcription dynamics deriving gene transcription           interactions enzymes enzyme-centric <br /> rate power-law formulism, Hill equation,        view (Additional file 1). <br /> equations chemical kinetic actions [27-29]. Or           All kinetic rate equations formulated according <br /> describe ligand-receptor protein-protein binding            biochemical mechanisms [5]. Most were <br /> actions mass action law build models           casted uni-/bi-substrate Michaelis-Menten formu- <br /> signaling networks [22]. We obtain          lism. The kinetics metabolite expressed by <br /> exact parameters fitting modeled solutions assay         ordinary differential equation (ODE; Eqn (1)). <br /> measurements analyzing generic behavioral <br /> potential system, e.g. parameter ranges             dX <br />                                                                       = A &#183; R(X, P) + B(X, P)                            (1) <br /> the exhibits certain dynamics                  dt <br /> change parameters. Such qualitative predictions are <br />  Li Liu BMC Systems Biology 2012, 6(Suppl 1):S11                                                             Page 12 14 <br /> http://www.biomedcentral.com/1752-0509/6/S1/S11 <br />  <br />  <br />  <br />  <br />   Here vector X denoted state P denoted             Dynamical stability <br /> kinetic parameters. R function vector collocated         Normal bio-systems subjected robustness they <br /> by rate equations, A stoichiometric            structurally consist abundant alternative pathways <br /> matrix. B term standing extra reactions (e.g.      feedback loops [6,17,24]. Thus valid formulations of <br /> transport, metabolite utilization cellular growth, etc).   bio-systems usually stable equilibrium, attracting <br /> Most parameters published studies            neighborhood trajectories allowing slight changes to <br /> the rest estimated using experimental con-        tolerated disturbing normality [5,25,26]. The <br /> ditions, steady-state reaction rates concentrations        trajectories tend area adequately large <br /> reported previous studies [5,31,32]. For complete           ranges time parameter spaces has <br /> descriptions metabolites, reactions, forms kinetic       equilibrium. And did not, trajectories spread out <br /> rate equations ODEs, Additional file 6.                dimensions traversing orders mag- <br />                                                                nitudes. To locate equilibrium, utilized state <br /> Dynamical simulation state fluctuation                     end time point simulation initial guess <br /> By substituting initial value, typical Cauchy pro-     used trust-region method solve problem <br /> blem formed numerical integration curves          [26,34]. By carefully refining numerical approach, <br /> computed Eqn (1). We used Gear method               equilibrium computed distances from <br /> computation alleviate stiffness problem        original Xeq calculated Euclid norm. <br /> ODEs [33]. With initial value normal experimen-            We defined dynamical stability following con- <br /> tal conditions [5], obtained kinetic states      cept Lyapunov stability, explicit physi- <br /> system X0, i.e. time-courses metabolite concentrations      cal/chemical context suitable describing <br /> under normal conditions. After deleting enzyme,          metabolic robustness [25,26]. The stability equilibrium <br /> computed kinetics residual Xe         determined eigenvalues Jacobian matrix <br /> how deviated original state. Thus influ-       evaluated equilibrium (Eqn (4)). If eigenvalues <br /> ence deleted enzyme assessed. Assuming         negative real parts, equilibrium asymptotically <br /> solution X organized matrix column           stable; positive real part, equili- <br /> represented kinetics metabolite, calcu-      brium unstable; Jacobian matrix pair of <br /> late amplitude metabolite k&#8217;s state fluctuation      purely imaginary conjugate eigenvalues, limit cycle is <br />                                                                likely bifurcate equilibrium. <br />     fk = ||X&#8226;k <br />             e <br />                &#8722; X&#8226;k <br />                   0 <br />                      ||2 /||X&#8226;k <br />                              0 <br />                                 ||2                      (2)                                 <br />                                                                             &#8706; A &#183; R(X, P) <br />   We calculate distances metabolites             JXeq =                                              (4) <br />                                                                                   &#8706;X <br /> the deleted enzyme structure metabolite-cen-                                            X=Xeq <br /> tric network. Metabolites directly associating the <br /> enzyme assigned distance 1; metabolites            The Hartman-Grobman Theorem Center Manifold <br /> directly associating enzyme associating           Theorem prove Jacobian matrix evaluated an <br /> with 1st distance level metabolites direct        equilibrium 2 conjugate purely imaginary eigenvalues, <br /> single reaction assigned distance 2, on.      Ns eigenvalues negative real parts Nu eigenvalues <br /> We combined distances amplitudes             positive real parts, trajectories Eqn (1) near the <br /> which ranges influences occurred strong           equilibrium topologically equivalent Eqn <br /> were. We computed flux distributions           (5). Here b kinetic parameters s +1 <br /> residual based metabolites concentrations        according system. In words, orbit struc- <br /> and rate equations. Thus observe flux         ture (near equilibrium) Eqn (5) topologically con- <br /> distributions deviated original            jugate Eqn (1). Because Eqn (5) much <br /> assessed way Eqn (2). The dis-             simpler, investigate instead complex <br /> tances effects directly counted           Eqn (1). In way, explicitly drew orbit structure <br /> enzyme-centric network. Furthermore, com-             Eqn (5) near equilibrium know quali- <br /> bine amplitude distance data single mea-        tative dynamics Eqn (1) accordingly. <br /> surement assessing overall impact,               &#9127; dy                                 <br />                                                                   &#9130; <br />                                                                   &#9130; <br />                                                                        1 <br />                                                                          = &#946;y1 &#8722; y2 + &#963; y1 y12 + y22 <br /> metabolite-centric network enzyme-centric network             &#9130; <br />                                                                   &#9130; <br />                                                                   &#9130; <br />                                                                   &#9130;  dt <br />                                                                                                        <br />                                                                   &#9130; dy2 <br />                                                                   &#9130; <br /> (Eqn (3)).                                                        &#9128;      = y1 + &#946;y2 + &#963; y2 y12 + y22 <br />                                                                      dt                                                   (5) <br />                                                                   &#9130; <br />                                                                   &#9130;           dyNs <br />                                                                   &#9130; <br />                                                                   &#9130;                  = &#8722;yNs <br />     M(d, f ) =     fk dnk , n &#8712; N+                                &#9130; <br />                                                                   &#9130;            dt <br />                                                       (3)         &#9130; <br />                                                                   &#9130;            dy Nu <br />                                                                   &#9129;                   = yNu <br />                     k <br />                                                                                 dt <br />  Li Liu BMC Systems Biology 2012, 6(Suppl 1):S11                                                                                        Page 13 14 <br /> http://www.biomedcentral.com/1752-0509/6/S1/S11 <br />  <br />  <br />  <br />  <br />   If bifurcation caused silico deletion (para-                    circles arrows indicate interactions. For detailed information, <br /> meter modification) yields multiple equilibriums,                          &#8220;Methods&#65533;? section Additional file 6. <br /> impact dynamical stability regarded large any-                     Additional file 2: Impacts states metabolite-centric <br /> one equilibriums exhibit qualitative difference                         view. The file format *.pdf, plot showing the <br />                                                                                impacts enzymes deletions metabolite kinetics. The metabolite <br /> from Xeq dynamical properties.                                              indexes, impact distances state uation amplitudes form 3 <br />                                                                                dimensions. Legends Figure 2. <br /> MOMA flux-sum                                                              Additional file 3: Impacts states enzyme-centric <br /> MOMA suggested metabolic systems sub-                                view. The file format *.pdf, plot showing the <br />                                                                                impacts enzymes deletions kinetic fluxes. The reaction/enzyme <br /> jected biological robustness. When perturbed,                            indexes, impact distances flux fluctuation amplitudes form 3 <br /> could adjust itself state relatively                        dimensions. Legends Additional file 2. <br /> close original state. We formulate pro-                       Additional file 4: The catalogue results dynamical stability <br /> cess optimization problem                                             analysis. The file categorizes enzymes generate topologically <br />                                                                                equivalent orbit structures, deleted. Enzymes same <br />                                 <br />     min S(P&#956; ) =  X(P&#956; ) &#8722; X0  2 <br />                                                                                category exert similar impacts qualitative dynamics system. <br />                                                                                The file format MS Word electronic table (*.doc). <br />          &#9127; <br />          &#9130; dX                                                                  Additional file 5: The flux-sum validations critical uncritical <br />          &#9130; <br />          &#9130; dt = A &#183; R(X, P&#956; ) <br />          &#9130; <br />          &#9130;                                                                     enzymes. The file format *.pdf, plot showing the <br />          &#9128;                                           (6)                       flux-sums essential metabolites deletion an <br />     s.t.        0&#8804;X                                                            enzyme. Deletions presented enzymes shown. The metabolite <br />          &#9130; <br />          &#9130;                                                                     symbols flux-sum values form lateral vertical dimensions. <br />          &#9130; <br />          &#9130; <br />                   <br />                 0 &#8804; P&#956; <br />          &#9130; <br />          &#9129;                                                                     Legends Figure 5. <br />             X(t0 = 0) = C0                                                     Additional file 6: Description modeling. The file the <br />                                                                                supplementary text modeling. It format *.doc and <br />   Here P&#956; parameter set relevant enzy-                        includes detailed descriptions metabolites, enzymes, kinetic rate <br />                                                                                equations ODEs. <br /> matic parameters deleted, X0 original state and <br />                                                                                Additional file 7: High resolution images Figure 6A 6B. The <br /> C0 initial value. Some states close to <br />                                                                                files (in format *.png) high resolution versions of <br /> X0 feasible space solved genetic                      Figure 6A 6B, respectively. <br /> algorithm, heuristic numerical approach <span id='am-60' about='obo:NCBITaxon_28690' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_28683 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_8906 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="Alle" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-64'  ></span>alle</span>-                       Additional file 8: High resolution images Figure 6A 6B. The <br /> viate computation difficulty large variable space                        files (in format *.png) high resolution versions of <br />                                                                                Figure 6A 6B, respectively. <br /> some extent. <br />   We adopted definition essential metabolite and <br /> flux-sum Kim et al.&#8217;s work E. coli metabolism [6]. <br /> The 12 essential metabolites occurred central carbon                       Acknowledgements <br />                                                                               We thank Yi-Xue Li valuable advices dynamics analysis. <br /> metabolism shown Figure 5. Here flux-sum <br />                                                                               This article published BMC Systems Biology Volume 6 <br /> of metabolite k defined                                                Supplement 1, 2012: Selected articles The 5th IEEE International <br />                                                                               Conference Systems Biology (ISB 2011). The contents the <br />      k =     Aki &#183; Ri (X, P)                          (7) <br />                                                                               supplement available online http://www.biomedcentral.com/ <br />                                                                               bmcsystbiol/supplements/6/S1. <br />            i&#8712; k <br />                                                                               Author details <br />   &#937; k index set reactions producing                          1 <br />                                                                                Key Laboratory Systems Biology, Shanghai Institutes Biological <br /> metabolite k.                                                                 Sciences, Chinese Academy Sciences, Shanghai, China. 2Shanghai Center <br />                                                                               Bioinformatics Technology, Shanghai, China. <br />   After MOMA computation, obtained (or <br /> more) set parameters states. Using rate                         Authors&#8217; contributions <br /> equations, simulated fluxes calculate flux-                        Conceiving designing research: RDL LL. Data acquisition and <br />                                                                               analysis: RDL. Drafting manuscript: RDL LL. <br /> sums according Eqn (7). <br />                                                                               Competing interests <br /> Additional material                                                           The authors declare competing interests. <br />  <br />                                                                               Published: 16 July 2012 <br />  Additional file 1: The metabolite-centric enzyme-centric views <br />  E. coli central carbon metabolic network. The file the <br />                                                                               References <br />  format *.png, figures included showing metabolite-centric <br />                                                                               1. Hood L: Systems biology: Integrating technology, biology and <br />  (Page 1) enzyme-centric metabolic (Page 2) networks. In the <br />                                                                                   computation. Mech Ageing Dev 2003, 124:9-16. <br />  metabolite-centric view, enzymes/reactions abbreviated symbols <br />                                                                               2. Ideker T, Galitski T, Hood L: A new approach decoding life: Systems <br />  denoted rectangles; metabolites abbreviations; inhibitors/ <br />                                                                                   biology. Annu Rev Genomics Hum Genet 2001, 2:343-372. <br />  activators drawn circles reactions. Synth1 lumped <br />                                                                               3. Bailey JE: Toward science metabolic engineering. Science 1991, <br />  reaction synthesizing chorismate murine; Synth2 lumped <br />                                                                                   252:1668-1675. <br />  reaction synthesizing isoleucine, alanine, ketoisovalerate, and <br />                                                                               4. Li H, Zhan M: Systematic intervention transcription identifying <br />  diaminopimelate [5]. In enzyme-centric view, enzymes denoted <br />                                                                                   network response disease cellular phenotypes. Bioinformatics <br />                                                                                   2006, 22:96-102. <br />  Li Liu BMC Systems Biology 2012, 6(Suppl 1):S11                                                                                                    Page 14 14 <br /> http://www.biomedcentral.com/1752-0509/6/S1/S11 <br />  <br />  <br />  <br />  <br /> 5.    Chassagnole C, Noisommit-Rizzi N, Schmid JW, Mauch K, Reuss M: Dynamic           31. Bhattacharya M, Fuhrman L, Ingram A, Nickerson KW, Conway T: Single-run <br />       modeling central carbon metabolism <span id='am-43' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-44' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-48'  ></span><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia</span> coli. Biotechnol            separation detection multiple metabolic intermediates anion- <br />       Bioeng 2002, 79:53-73.                                                               exchange high-performance liquid chromatography application to <br /> 6.    Kim PJ, Lee DY, Kim TY, Lee KH, Jeong H, Lee SY, Park S: Metabolite                  cell pool extracts prepared Escherichia coli. Anal Biochem 1995, <br />       essentiality elucidates robustness Escherichia coli metabolism. Proc              232:98-106. <br />       Natl Acad Sci USA 2007, 104:13638-13642.                                         32. Buziol S, Bashir I, Baumeister A, Claa&#223;en W, Noisommit-Rizzi N, Mailinger W, <br /> 7.    Neidhardt FC, Curtiss R, Ingraham JL, Lin ECC, Low KB, Magasanik B,                  Reuss M: New bioreactor-coupled rapid stopped-flow sampling <br />       Reznikoff W, Riley M, Umbar <br /> </body></html>