<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Identification Novel Human Damage Response <br /> Proteins Targeted Yeast Orthology <br /> J. Peter Svensson1&#164;a, <span id='am-51' about='obo:NCBITaxon_561168' typeof='owl:Thing obo:NCBITaxon_2830 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_73026 obo:NCBITaxon_418969 obo:NCBITaxon_2759'><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Rebecca" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-55'  ></span>Rebecca</span> C. Fry1,2&#164;b, Emma Wang1, Luis A. Somoza1, Leona D. Samson1,2,3* <br /> 1 Biological Engineering Department, Center Environmental Health Sciences, Massachusetts Institute Technology, Cambridge, Massachusetts, United States of <br /> America, 2 Computation Systems Biology, Massachusetts Institute Technology, Cambridge, Massachusetts, United States America, 3 Department Biology, <br /> Massachusetts Institute Technology, Cambridge, Massachusetts, United States America <br />  <br />  <br />  <br />      Abstract <br />      Studies <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14'  ></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> proteins influence cellular survival exposure DNA damaging <br />      agents. We hypothesized human orthologs S. cerevisiae proteins required cellular survival <br />      treatment DNA damaging agents. For purpose, human homologs S. cerevisiae proteins identified and <br />      mapped human protein-protein interaction network. The resulting human network highly modular series <br />      selection rules implemented identify 45 candidates human toxicity-modulating proteins. The corresponding <br />      transcripts targeted RNA interference human cells. The cell lines depleted target expression were <br />      challenged DNA damaging agents: alkylating agents MMS 4-NQO, oxidizing agent t-BuOOH. A <br />      comparison survival revealed majority (74%) proteins conferred sensitivity resistance. The <br />      identified human toxicity-modulating proteins represent variety biological functions: autophagy, chromatin <br />      modifications, RNA protein metabolism, telomere maintenance. Further studies revealed MMS-induced <br />      autophagy increase survival cells treated DNA damaging agents. In summary, damage recovery <br />      proteins humans identified homology S. cerevisiae pathways are <br />      represented toxicity modulators. <br />  <br />   Citation: Svensson JP, Fry RC, Wang E, Somoza LA, Samson LD (2012) Identification Novel Human Damage Response Proteins Targeted Yeast <br />   Orthology. PLoS ONE 7(5): e37368. doi:10.1371/journal.pone.0037368 <br />   Editor: <span id='am-56' about='obo:NCBITaxon_153389' typeof='owl:Thing obo:NCBITaxon_243881 obo:NCBITaxon_243882 obo:NCBITaxon_94015 obo:NCBITaxon_33208 obo:NCBITaxon_94017 obo:NCBITaxon_33213 obo:NCBITaxon_94020 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_33154 obo:NCBITaxon_153385 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58'  ></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Beata" datatype="xsd:string"></span>Beata</span> G. Vertessy, Institute Enzymology Hungarian Academy Science, Hungary <br />   Received January 2, 2012; Accepted April 21, 2012; Published May 16, 2012 <br />   Copyright: &#223; 2012 Svensson et al. <span id='am-33' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported grants National Institute Environmental Health Sciences (NIEHS) (ES02109) National <span id='am-25' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Cancer" datatype="xsd:string"></span>Cancer</span> Institutes (NCI) <br />   (CA55042 CA112967). Dr. Svensson supported post-doctoral fellowship Swedish Research Council. The funders role study design, <br />   <span id='am-61' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-62'  ></span><span id='am-63' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: lsamson@mit.edu <br />   &#164;a Current address: Department Biosciences Nutrition, Karolinska Institutet, Huddinge, Sweden <br />   &#164;b Current address: Department Environmental Sciences Engineering, Gillings School Global Public Health, The University North Carolina, Chapel Hill, <br />   North Carolina, United States America <br />  <br />  <br />  <br /> Introduction                                                                          large DNA base adducts formed metabolically activated 4- <br />                                                                                       NQO stall transcription replication, does 3- <br />    Sensing, signaling repair DNA damage requires                          methyladenine (3MeA), contrast, 4-NQO induced lesions <br /> proteins [1] depletion proteins affect                    necessarily toxic 3MeA [10]. Many 4-NQO <br /> cellular survival DNA damage. DNA damaging agents,                         induced lesions require nucleotide excision repair resolved <br /> both endogenous exogenous sources, constantly challenge                           [9]. Also, process activation, 4-NQO metabolism <br /> genome integrity, causing mutations, permanent cell cycle arrest                      generates reactive oxygen <span id='am-38' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>, causing oxidative damage to <br /> and cell death. The endpoints exploited                         cellular components. Another pro-oxidant oxidizing agent <br /> therapeutic purposes. For example, common <span id='am-30' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-31' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> cancer                           tert-butyl hydroperoxide (t-BuOOH), effects on <br /> chemotherapy agents DNA damaging agents act                               cell metabolism [11]. <br /> alkylation, represented drugs Temozolomide                                 All mentioned DNA damaging agents shown <br /> Carmustine (1,3-bis(2-chloroethyl)-1-nitrosourea, BCNU) [2,3].                        modulate expression genes, cells lacking wide <br /> Other alkylating agents include extensively studied model                         variety proteins aberrant responses DNA damage <br /> agents methyl methanesulfonate (MMS) 4-nitroquinoline-N-                          [12,13,14,15,16,17,18,19,20,21,22,23,24,25]. Indeed, recent ge- <br /> oxide (4-NQO) used explore DNA damage                           nome-wide siRNA screens human cells revealed many <br /> responses cells organisms (reviewed [4]). The simple SN2                    unexpected pathways involved maintaining genome stability <br /> alkylating agent MMS attacks DNA, forming products                               [19,22,23]. In budding yeast, extensive studies deletion mutants <br /> include 7-methylguanine highly toxic 3-methyladenine                          revealed approximately 30% genes affect recovery <br /> [5]. These lesions efficiently removed DNA glycosylases                     damage alkylating agents. Previous studies our <br /> like AAG/MPG mammals initiate base excision pathway                         group determined yeast survival libraries gene deletion <br /> [6]. Damage induced bulky alkylating agent 4-NQO                               mutants exposure DNA damaging agents (MMS, 4- <br /> requires complex arsenal repair capacities [7,8,9]. The                     NQO, t-BuOOH UV). Distinct toxicity profiles were <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                 1                                   May 2012 | Volume 7 | Issue 5 | e37368 <br />                                                                                                             Novel Human Damage Recovery Proteins <br />  <br />  <br /> identified agent, surprisingly, gene deletion           deletion strains [13], 28 strains sensitive the <br /> strains sensitive agents. Similar screens toxicity-       tested DNA damaging agents. Twelve 28 proteins had <br /> modulating proteins conducted MMS Drosophila                 human homologs included screen; proteins <br /> [26] ionizing radiation C. elegans [27], showing                   display heterogeneity cellular functions (Table S2). Additional <br /> comparable results. The toxicity-modulating proteins represent                targets selected large interconnected sub-network. <br /> a variety biological functions biochemical pathways. Apart             Proteins established roles DNA repair cell cycle <br /> from proteins involved stress signaling, cell cycle control, DNA           control excluded, ribosomal proteins. Preference was <br /> repair cell death, functions transcription, vesicle               given proteins human homolog specific yeast <br /> transport, protein RNA metabolism, telomere mainte-                   protein, proteins protein-protein interactions. <br /> nance affect recovery exposure DNA damaging                     In addition, genes expressed reasonable levels in <br /> agents. However, direct role processes damage                 human cells measured previous study [29]. We gave <br /> recovery remains largely unknown.                                             preference proteins highly represented <br />    In study, aimed identify novel pathways needed              categories yeast, transcription, chromatin remodeling, <br /> human cells recover exposure DNA damaging agents.                  vesicle transport protein/mRNA degradation. In light the <br /> We hypothesized yeast data combined human                   involvement telomere maintenance toxicity-modu- <br /> protein interactome used pinpoint human proteins                  lating yeast proteins, proteins shelterin complex were <br /> needed recovery, identifying novel damage response                   selected shelterin complex specific telomere <br /> pathways humans. Based results S. cerevisiae [13],          maintenance mammalian cells. A brief summary 45 <br /> used computational techniques identify human protein                       selected proteins, including description functions, GO terms <br /> candidates toxicity modulation. 45 human proteins, spanning                yeast homologs sensitivity, Table S2. <br /> the human pathways identified toxicity-modulating S. <br /> cerevisiae, tested role recovery human cells         Efficient reduction mRNA levels 293T cells <br /> after damage. We 74% proteins tested modulated                 RNA interference used deplete transcript levels the <br /> the survival human cells.                                                  selected targets human cells. Stable clonal cell lines created <br />                                                                               lentiviral infection shRNAs targeting mRNA of <br /> Results                                                                       selected genes. We used adherent embryonic kidney cell lines <br />                                                                               293T parental cell line cells readily stably <br /> Identification human homologs toxicity-modulating                       express foreign DNA. For 35 gene targets achieved reasonable <br /> yeast proteins                                                                knock-down effect (,60% residual mRNA level compared to <br />    The aim study determine human                   controls) 293T background (Figure 2A). <br /> homologs toxicity-modulating proteins yeast, spanning wide <br /> range cellular functions, play roles damage response           The large majority selected homologs toxicity- <br /> of human cells. First, human-yeast protein homologs                      modulating human cells <br /> identified based amino acid sequence similarity. Toxicity-                    To test deficiency targeted proteins resulted in <br /> modulating proteins selected [13]. Two public                       altered sensitivity DNA damaging agents, cells reduced <br /> databases, Ensembl Inparanoid, used identify 1,368                levels target mRNA exposed different <br /> homologs 4,733 proteins represented S. cerevisiae gene          damaging agents equitoxic doses: alkylating agents MMS <br /> deletion library. Of these, 646 human proteins identified             4-NQO oxidizing agent t-BuOOH. The cell lines <br /> homologous yeast proteins toxicity-modulating properties              compared control experiments performed <br /> for DNA damaging agents used yeast            contemporaneously. The variation days minimal, <br /> study. The homologs projected human protein-                      determined repeated survival data control cell line <br /> protein interaction network previously described [28]. Surprising-            expressing shRNA construct targeting sequence present <br /> ly, 44% nodes, representing 284 proteins, connected            human genome (data shown). To account off-target <br /> one large connected component (p,1610216, permutation test),                  effects, non-silenced cell lines expressing shRNA construct <br /> indicating proteins involved disparate               tested survival treatment damaging agents. <br /> functions, large proportion connected protein-               These non-silenced cell lines expressed target shRNA against <br /> protein interactions (Figure 1, interactive version http://             RNASEH2A, TBL1XR1, AP3D1 non-silencing clone of <br /> www.bionut.ki.se/users/pesv/MIT/fig1.html). In large con-                 ATP6V1F significant effect target gene expression <br /> nected component interactome, numerous biological                      (.60% residual levels target RNA, Figure S1A). The sensitivity <br /> categories represented, including DNA repair, stress signaling,           range cell lines range cells expressing <br /> vesicle transport, chromatin modification, plus lipid, protein            shRNA targeting sequence present human genome <br /> RNA metabolism. The network highly modular                 set detection limits screen (Figure S1B&#8211;C, grey <br /> functional categories represented yeast represented            black lines). The survival data cell lines confirmed <br /> the human network, exception telomere maintenance                 targeted gene silencing summarized heatmap (Figure 2B). <br /> (Tables S1a S1b). Telomere stability maintained non-                XPA-deficient cells included positive control. XPA is <br /> homologous proteins S. cerevisiae mammals                DNA repair protein know important repair UV- <br /> a group telomere-specific human proteins queried                      induced lesions [30]. Here lack XPA lead to <br /> separately.                                                                   specific reduction survival treatment UV- <br />                                                                               mimetic 4-NQO. In summary, 34 targets not <br /> Selection putative toxicity-modulating human                               previously associated DNA damage response, we <br /> proteins                                                                      obtained significant reproducible results regarding effect <br />    To reduce number targets human homologs                sensitivity DNA damaging agents. These data show <br /> of toxicity-modulating yeast proteins, set selection rules           reduced transcript levels 14 34 proteins (41%) <br /> implemented. In previous study yeast proteins library          conferred high (.25% different WT) moderate (20&#8211;25% <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         2                               May 2012 | Volume 7 | Issue 5 | e37368 <br />                                                                                                         Novel Human Damage Recovery Proteins <br />  <br />  <br />  <br />  <br /> Figure 1. Human interaction network shows high connectivity putative human toxicity-modulating proteins homologous to <br /> toxicity-modulating proteins yeast. The largest connected component human interactome selected yeast orthologs being <br /> required damage recovery treatment MMS, 4NQO, t-BuOOH UV [13]. The circles represent: red &#8211; proteins toxicity-modulating <br /> yeast homologs targeted silencing study; grey &#8211; proteins toxicity-modulating yeast homologs targeted study; blue &#8211; <br /> proteins non-toxicity-modulating yeast homologs targeted study; green &#8211;proteins specific mammalian telomere maintenance <br /> targeted study. An interactive version figure available http://www.bionut.ki.se/users/pesv/MIT/fig1.html. <br /> doi:10.1371/journal.pone.0037368.g001 <br />  <br />  <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       3                             May 2012 | Volume 7 | Issue 5 | e37368 <br />                                                                                                          Novel Human Damage Recovery Proteins <br />  <br />  <br /> different WT) sensitivity DNA damaging agents cells.            autophagy needed cells recover MMS exposure. <br /> Surprisingly, target proteins selected based                Significant sensitization cells observed treatment <br /> sensitivity yeast deletion mutants, 11 (32%) human cell          4-NQO t-BuOOH, pronounced compared <br /> lines showed high moderate resistance DNA damaging                   MMS (Figure S2). This finding indicate general requirement <br /> agents. In total, 19 34 proteins (56%) showed high (.25%            autophagy cellular treatment DNA damaging agents <br /> different WT) toxicity-modulation (see Material                   rescue exposed cells. <br /> Methods section details). Six additional proteins showed                  In attempt study dynamics autophagy induction, <br /> moderate (20&#8211;25% different WT) toxicity-modulation                    cells transfected GFP-tagged LC3 followed during <br /> (FBXL2, POT1, PEX10, HDAC6, PRPS2, LSM1), bringing                     6 hours. LC3 accumulates autophagosomes [32] these <br /> the total percentage toxicity-modulating proteins                GFP-labelled autophagosomes visualized puncta using <br /> selection 74%.                                                          fluorescent microscope. To study autophagic flux, we <br />                                                                            followed progression autophagy incubation the <br /> A random selection human proteins contains low                        cells Lysotracker, stain acidic compartments such <br /> proportion toxicity-modulating proteins                                 lysosomes autolysosomes. Cells contained .5 visible <br />    Given 74% targeted proteins caused toxicity-              puncta scored autophagic cells (Figure 3C). A subset the <br />                                                                            LC3-positive autophagosomes (stained green) fuse lysosomes <br /> modulating phenotype, sought estimate be <br />                                                                            (red) make autolysosomes (yellow), untreated and <br /> found random chance. We hypothesized human deficiency <br />                                                                            treated cells. This observation consistent previous studies of <br /> of homologs yeast proteins did modulate toxicity in <br />                                                                            293 cells [33,34]. Low frequencies autophagic cells found <br /> yeast likewise result sensitivity changes DNA <br />                                                                            control cultures 3MeA. Cells treated <br /> damaging agents human cells. To test hypothesis, we <br />                                                                            BA1 inhibit late steps autophagy trap cells <br /> identified proteins evidence toxicity-modulation in <br />                                                                            induced completed autophagy, known <br /> yeast [13]. Out 724 yeast-human homologs non-toxicity- <br />                                                                            autophagy-inducer rapamycin. Both treatments induced 3&#8211;4-fold <br /> modulators, 200 genes expressed human cell lines [29]. <br />                                                                            higher levels autophagic cells. MMS treatment induced <br /> Five proteins selected completely random: <br />                                                                            autophagy extent (Figure 3D). Further analysis of <br /> SLC25A5, AP3D1, ADIPOR1, URM1, RASA1. For four <br />                                                                            dose-dependencies autophagy induction revealed robust <br /> proteins, reduced mRNA levels (,60%) achieved. For cell <br />                                                                            dose time response MMS. After pretreatment cells <br /> lines lacking proteins, survival treatment the <br />                                                                            inhibitor BA1, percentage cells induced autophagy <br /> DNA damaging agents determined. A deficiency one <br />                                                                            did increase subsequent MMS exposure <br /> of proteins, URM1, resulted altered sensitivity             (Figure 3E, F). <br /> phenotype. URM1 recently described affect cellular <br />                                                                               We sought elucidate autophagy-related role the <br /> recovery starvation oxidative stress [31]. Despite the <br />                                                                            proteins ATP6V1D ZFYVE20. Cells depleted ATP6V1D <br /> small number cell lines tested here, conclude 74% of <br />                                                                            ZFYVE20 sensitive MMS. Autophagy inhibitor <br /> toxicity-modulating proteins screen appears different         3MeA sensitized depleted cell lines low loses of <br /> from random sampling proteins (borderline significance,             MMS (Figure 4A). The study LC3 puncta disadvantaged by <br /> p = 0.08 (Fischer&#8217;s exact test)). Therefore, selection yeast-       fact cells expressed GFP level and <br /> human homologs advantageous discovering new mam-                  uniform cytoplasmic background GFP. <br /> malian toxicity-modulating proteins, able          However, transfection LC3-GFP construct, cells <br /> predict direction toxicity-modulation, i.e. relative            clearly defined puncta GFP background be <br /> sensitivity resistance.                                                 scored (Figure 4B). As consequence background GFP- <br />                                                                            levels, percentages identifiable autophagy-positive cells were <br /> Requirement autophagy survive MMS-induced                      lower cells (Figure 4C). The cell lines depleted of <br /> damage                                                                     ATP6V1D ZFYVE20 sensitive MMS, and <br />    The toxicity-modulation results (Figure 2B) revealed cells         cell line able significantly induce autophagy over <br /> lacking vesicle proteins ZFYVE20, ATP6V1D VPS16                    background levels following MMS treatment. Reduced levels of <br /> became sensitive MMS, suggesting involvement early            ATP6V1D lead accumulation autophagosomes, suggest- <br /> late endosomal pathways damage recovery. The late endosomal             ing ATP6V1D involved late steps MMS-induced <br /> vesicle transport intersects autophagic pathway,           autophagy, clearance autophagosomes. However, <br /> set characterize role autophagy damage DNA          reduced levels ZFYVE20 resulted lower percentage of <br /> damaging agents. During autophagic process, cellular                   autophagic cells treated cultures, suggesting ZFYVE20 <br /> components engulfed autophagosomes LC3-molecules               involved early steps MMS-induced autophagy, as <br /> on surface. These autophagosomes fused acidic                 formation autophagosomes. <br /> lysosomes form autolysosomes engulfed components <br /> are broken possibly recycled (Figure 3A). To determine            Damage sensitivity chromatin remodeling <br /> the significance autophagy DNA damage, studied                Many targets screen affect structural status of <br /> effect inhibiting autophagy wild-type 293T cells. A chemical         chromatin. These include histone modifiers histone <br /> inhibitor early steps autophagy (3-methyladenine, 3MeA)          ubiquitin ligase (RNF20) components complexes changing <br /> and inhibitor late steps (Bafilomycin A1, BA1) used         acetylation status histones (TADA2A, SIN3A HDAC6). <br /> (Figure 3A). BA1 inhibits autophagic completion leads               TOP3A creates transient single stranded DNA breaks will <br /> accumulation late autophagic vesicles. Wild-type cells             alter topology chromatin, POLR2D subunit of <br /> incubated presence autophagy inhibitor hours              RNA polymerase II. Reduced cellular levels proteins lead <br /> prior hour treatment damaging agent. The               increased (SIN3A, RNF20, TOP3A, POLR2D) decreased <br /> survival MMS severely diminished reduced                  (TADA2A, HDAC6) survival 4-NQO exposure compared to <br /> autophagy mediated inhibitors (Figure 3B), indicating         survival control cells. The bulky lesions induced 4-NQO <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       4                              May 2012 | Volume 7 | Issue 5 | e37368 <br />                                                                                                            Novel Human Damage Recovery Proteins <br />  <br />  <br />  <br />  <br /> Figure 2. The majority selected proteins modulate recovery damage compounds MMS, 4-NQO t- <br /> BuOOH. A) RNA levels shRNA targeted genes 293T cells measured qRT-PCR compared cells infected non-silencing control <br /> shRNA. B) Survival cells depleted target proteins exposed DNA damaging agents revealed heatmap. The color represents <br /> sensitivity damaging agent compared cell lines non-silenced targets. ++ indicate high resistance. + low resistance, 2 high <br /> sensitivity, 2 low sensitivity. C) Knock-down human homologs non-toxicity modulating proteins yeast, measured qRT-PCR. D) Survival of <br /> cells depleted human homologs non-toxicity modulating proteins yeast. Colors symbols B. <br /> doi:10.1371/journal.pone.0037368.g002 <br />  <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        5                              May 2012 | Volume 7 | Issue 5 | e37368 <br />                                                                                                          Novel Human Damage Recovery Proteins <br />  <br />  <br />  <br />  <br /> Figure 3. Response MMS relies autophagy. A) Model induction inhibition autophagy. 3-methyladenine (3MeA) inhibits the <br /> formation autophagosomes bafiloycin A1 inhibits acidification lysosomes leading accumulation autophagosomes. LC3 is <br /> a marker autophagosomes, tagged GFP. B) Inhibition autophagy decreases survival, 3MeA BA1. C) Autophagy as <br /> seen formation LC3-GFP-puncta showing autophagosomes treated cells (top panel). A subset LC3-GFP-puncta co-stain (white <br /> arrows) acidic vesicles labeled Lysotracker Red (white red arrows) (bottom panel). D) Significant induction autophagy MMS <br /> treatment (1.2 mM). E&#8211;F) MMS induces autophagy dose time dependent manner, accumulation autophagosomes BA1 not <br /> affected MMS. <br /> doi:10.1371/journal.pone.0037368.g003 <br />  <br /> [7,8] presumably cause conformational changes chromatin,                 fact yeast telomere maintenance proteins an <br /> possibly explaining requirement chromatin modifiers              active role DNA repair. RNA degradation represented in <br /> survival.                                                                   study LSM1, EXOSC10, CDC40 NCBP2. Depletion <br />                                                                             proteins led 4NQO resistance (CDC40 LSM1) or <br /> Other proteins functions                                                visible phenotype (EXOSC10 NCBP2). The rest the <br />    The telomere specific proteins tested study (TERF1,              targeted proteins form mosaic different known unknown <br /> TERF2, ACD POT1) resulted cellular sensitivity              functions. Interestingly, signaling protein CTDNEP1 (the <br /> to MMS depleted, arguing enrichment                   homolog yeast Nem1, YHR004C) [35] proteins <br /> &#8216;telomere maintenance&#8217; term yeast screen caused            lead cellular sensitivity t-BuOOH depleted, <br /> need maintain intact telomeres survival, addition          relative WT. XPA tested protein clearly <br />  <br />  <br />        PLoS ONE | www.plosone.org                                       6                              May 2012 | Volume 7 | Issue 5 | e37368 <br />                                                                                                         Novel Human Damage Recovery Proteins <br />  <br />  <br />  <br />  <br /> Figure 4. Autophagic response MMS modulated ATP6V1D ZFYVE20. A) Inhibition autophagy sensitizes cells that <br /> have depleted ZFYVE20 APT6V1D MMS. B) The formation autophagosomes MMS treatment visible background of <br /> cytoplasmic GFP. C) MMS induces autophagy cells dependent ZFYVE20. The statistical significance difference each <br /> condition untreated control indicated asterisks (* p,0.05, ** p,0.01). <br /> doi:10.1371/journal.pone.0037368.g004 <br />  <br /> defined role DNA repair. It selected positive control       involved mRNA processing, chromatin binding Charcot <br /> it known cells lacking nucleotide excision repair              Marie Tooth-disease [19,22,23]. Here, alternative to <br /> component XPA sensitive 4-NQO [36]. We confirmed               genome-wide RNAi screens, implemented focused <br /> finding, cell line deficit XPA 4-NQO          approach advantage previous results model <br /> sensitive screen.                                                  organisms, S. cerevisiae, extensive knowledge protein- <br />                                                                            protein interactions interactomes. <br /> Discussion                                                                    Among human homologs toxicity-modulating yeast <br />                                                                            proteins, 25 34 proteins toxicity- <br />    &#8216;DNA damaging agents&#8217; cause damage numerous cellular                 modulating human cells. The DNA damaging agents, <br /> molecules damage DNA. In response treatment             MMS, 4-NQO t-BuOOH, revealed distinct toxicity profiles <br /> with agents, cells modulate expression levels genes        proteins specifically conferring resistance sensitivity at <br /> several different pathways. The results study that <br />                                                                            agents. Deficiency the <br /> many proteins pathways needed recovery specific <br />                                                                            proteins, largely uncharacterized suppressor actin mutations <br /> types damage. Previously, results genome-wide RNAi <br />                                                                            1-like SACM1L [37], modulated toxicity damaging <br /> screens suggested involvement vast repertoire of <br />                                                                            agents. SACM1L phosphatase, regulating Golgi morphology <br /> DNA damage recovery proteins. Among proteins <br />  <br />  <br />       PLoS ONE | www.plosone.org                                       7                              May 2012 | Volume 7 | Issue 5 | e37368 <br />                                                                                                           Novel Human Damage Recovery Proteins <br />  <br />  <br /> [38,39]; interestingly, deletion yeast homolog protein        modulator study. Interestingly, component involved the <br /> caused cellular sensitivity damaging agents tested        deubiquitination H2B (TADAD2A) reverse toxicity- <br /> yeast screen [13].                                                           modulation compared H2B ubiquitin ligase (RNF20). Other <br />    One striking result study identified          studies shown depletion RNF20 inhibits G1 <br /> proteins that, mRNA levels reduced, cause relative            arrest apoptosis, stimulates tumor advancement; its <br /> resistance DNA damaging agents, proteins              promotor hypermethylated tumors [56,57]. Further, <br /> were selected based corresponding yeast deletion strains              RNF20 ser-522 identified ATM/ATR phosphor- <br /> being sensitive agents. This observation particularly             ylation substrate exposure ionizing radiation [57]. <br /> noteworthy exposure 4-NQO members histone                Depletion human RNF20 paralog RNF40 recently <br /> modifier complexes, RNF20, SIN3A, CDC40, topoisomerase                       shown stimulate cell growth cell migration [58]. RNF20/ <br /> TOP3A RNA polymerase subunit POLR2D. Possibly,                   RNF40-mediated ubiquitination H2B prerequisite RNA <br /> a reflection differential maintenance chromatin structure           PolII transcription, possibly explaining observed similarity in <br /> mammals versus S. cerevisiae. While discrepancy puzzling,          toxicity-modulation RNF20 RNA PolII subunit D <br /> has shown previously organism,                (POLR2D) (Figure 2). <br /> cells different origin display distinctive, opposite, <br /> phenotypes exposed damaging agents [40,41,42,43].             Conclusions <br /> One dramatic example mouse ES cells deficient              Based yeast orthology conserved network structures, we <br /> Aag glycosylase MMS sensitive, relative WT, myeloid bone              identified human proteins necessary recovery after <br /> marrow cells retinal rods cones deficient                cellular damage, components autophagy and <br /> enzyme extremely MMS resistant [43,44].                                  chromatin modifiers. Clearly, functional relationships between <br />    Vesicle transporters unexpected toxicity-                  yeast human homologs complex lack some <br /> modulators yeast gene deletion screen. The endosomal                  proteins conferred sensitivity yeast cells human cells <br /> vesicle transport, especially late endosomal/lysosomal trans-            resulted resistance compared WT counterparts. <br /> port, used degradation biomaterial, process                 Nevertheless, studying machinery surrounds core <br /> intersects pathway autophagy. This group proteins          DNA repair proteins, obtain better understanding way <br /> highly conserved yeast humans. Interestingly,                cells respond genotoxic insults. Most identified toxicity- <br /> classical autophagy proteins (ATG1-ATG31) over-                     modulating proteins linked DNA repair, cell <br /> represented sensitive yeast deletion strains, suggesting           cycle arrest cell death highlight vast array proteins <br /> an alternative autophagy-like path taken treatment                involved damage recovery exposure DNA <br /> DNA damaging agents. Processing autophagosomes/lysosomes                  damaging agents. <br /> and proteasome ways clear cell proteins other <br /> biomaterial. The autophagosomes generated                    Materials Methods <br /> cytoplasm derived trans-Golgi, cells <br /> are exposed topoisomerase II inhibitor etoposide [45]. The            Bioinformatic analysis <br /> previously described role autophagy DNA damage response                   Genome-wide yeast sensitivity data [13] downloaded from <br /> has usually linked cellular death program,            http://genomicphenotyping.mit.edu/source2.html.          Human <br /> genotoxic agents shown induce autophagic cell death             orthologs yeast proteins identified Ensembl <br /> [46,47,48,49]. In contrast, study suggests cells escape cell       Inparanoid. Orthologues S. cerevisiae genes in <br /> death induction autophagy, autophagy                   human, mouse, yeast, obtained Ensembl49 (http:// <br /> reduced, cells sensitive MMS. This supported               ensembl.org/) [59]. A merged human interactome A. Garrow, <br /> other recent studies that damage lead non-             Y. Adeleye G. Warner [28] combines human interactions <br /> lethal autophagy [45,50]. The study autophagy important           reported IntAct, DIP, BIND HPRD, addition papers <br /> from clinical viewpoint autophagy shown                [60,61]. The interactome queried using Cytoscape 2.6 <br /> suppress tumorigenesis [51], clearing cells contain          (http://cytoscape.org). <br /> protein aggregates formed Huntinton&#8217;s disease                  Human expression data used [29]. Genes with <br /> [52].                                                                        microarray expression values .100 considered expressed. <br />    Here, confirmed telomere-specific proteins are <br /> needed cells recover treatment DNA damaging agents.             Cell culture <br /> Telomere proteins yeast involved DNA damage                      293T cells (ICLC catalog code: HTL04001, [62]) their <br /> repair, shown specific loss telomere                derivatives cultured Dulbecco&#8217;s minimal essential media <br /> maintenance, reducing protein levels members                (Invitrogen) complemented 10% fetal bovine serum, L- <br /> shelterin complex, results sensitivity alkylating damage. We           glutamine, 1% penicillin, streptomycin. shRNAs expressed in <br /> have identified new toxicity-modulating proteins involved            lentiviral plasmid (pGIPZ) purchased Open <br /> chromatin modification. Previously, known                Biosystems. Three clones analyzed mRNA levels, <br /> Ada2-homolog, TADA2B mammalian paralog TADA2A,                       clone lowest residual mRNA concentration was <br /> needed cellular response UV irradiation. This adaptor              subsequently used. Identity shRNAs sequences qRT- <br /> protein STAGA (homologous <span id='am-46' about='obo:NCBITaxon_441239' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_62784 obo:NCBITaxon_6994 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_70905 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_323649 obo:NCBITaxon_6993 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_33341 obo:NCBITaxon_50557 obo:NCBITaxon_6960 obo:NCBITaxon_6072'><span id='am-47' property="rdfs:label" content="Saga" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49'  ></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>SAGA</span> yeast)                   PCR primers (Eurofin) Table S3. Knockdown <br /> histone acetylation complex required transcription             cells compared 293T cells expressing non-targeting <br /> p53 responsive elements UV [53,54]. TADA2A                shRNA (#RHS4346). Virus generated 293T cells using <br /> hand component similar histone acetylation complex               packaging <span id='am-66' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span> psPAX2, pMD2.G (Addgene plasmid 12260 <br /> PCAF, activity recently implicated p53 pathway              12259). Parental cell lines infected virus stable <br /> [55]. The Ada2 homologs H2B                          clones selected using Puromycin (Invivogen). A proteins were <br /> deubiquitination complexes. Ubiquitination H2B performed               targeted multiple shRNA constructs. After shRNA infection, <br /> by ubiquitin ligase RNF20, identified toxicity-                genes residual levels 30%, 30&#8211;60% <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        8                             May 2012 | Volume 7 | Issue 5 | e37368 <br />                                                                                                                            Novel Human Damage Recovery Proteins <br />  <br />  <br /> residual levels. For targets, reduction mRNA levels                          formaldehyde, nuclei stained Prolong Gold with <br /> could detected target plasmids in-                              DAPI (Invitrogen). <br /> corporated parental cells, determined co- <br /> expression GFP. Catalog numbers primer sequences                             Supporting Information <br /> available Table S3. <br />                                                                                         Figure S1 Survival cell lines significant RNA <br />                                                                                         reduction. A) mRNA levels target transcripts not <br /> Colony forming assay <br />                                                                                         significantly reduced (ns &#8211; non-silencing clone). B&#8211;C) Survival <br />    50&#8211;5,000 cells seeded 6-well plates 16 hours later <br />                                                                                         curves control cell line expressing non-silencing shRNA <br /> cells washed PBS exposed duplicates 4-NQO, <br />                                                                                         (black), cell lines non-significant reduction levels the <br /> tBuOH, MMS (Sigma) serum-free media. After hour, drug- <br />                                                                                         targeted RNA (grey), additional cell lines reduced levels <br /> containing media replaced complete media incubated <br />                                                                                         TMLHE (red), TADA2A (blue) TERF2 (green) after <br /> for 6 days. Colonies washed cold PBS, dried overnight, <br />                                                                                         treatment B) 4-NQO C) MMS. <br /> fixed stained 0.25% Methylene blue ethanol and <br />                                                                                         (TIF) <br /> counted. <br />                                                                                         Figure S2    The survival exposed WT cells di- <br /> Calculation toxicity-modulation                                                      minished inhibition autophagy. The cells were <br />    For protein called toxicity-modulating, survival                  exposed A) 4-NQO, B) tBuOOH. <br /> corresponding cell line significantly different                      (TIF) <br /> non-silencing control cells (p,0.05, t-test)                     Table S1 GO terms enriched networks toxicity <br /> 20% sensitive resistant (+/2 0.26 log2-space)                       modulating proteins. Enrichment human cells (S1a) is <br /> of control cells &#8216;noise region&#8217; (the region created                   contrasted yeast cells (S1b). <br /> boundaries cell lines targeted knock-down) at                         (PDF) <br /> least dose-point. For protein confer &#8216;high sensitivity&#8217;/ <br />                                                                                         Table S2 Summary human potential toxicity- <br /> &#8216;high resistance&#8217;, cellular survival exceed 25% (+/2 0.32 <br />                                                                                         modulating proteins. The summary includes described <br /> in log2-space) &#8211; &#8211; dose-point. <br />                                                                                         function cell yeast homologs, with <br />                                                                                         toxicity-modulation summary yeast human cells. <br /> Autophagy detection                                                                     (PDF) <br />    The plasmid EGFP-LC3 purchased Addgene (plasmid <br /> 11546) [63], transfected 293T cells. Autophagy                             Table S3 shRNA constructs qRT-PCR primers. <br /> inhibited addition 0.1 uM Bafilomycin A1 (B-1080 LC                          (PDF) <br /> Laboratories, Woburn, MA) 10 mM 3-Methyladenine (from <br /> Sigma-Aldrich, Louisville) hours prior                         Author Contributions <br /> treatment DNA damaging agent. For LysoTracker Red                              Conceived designed experiments: JPS RCF LDS. Performed the <br /> staining, cells treated 50 nM LysoTracker Red                             experiments: JPS EW LAS. Analyzed data: JPS EW LAS. Wrote the <br /> DND-99 (Invitrogen) 37uC 30 min. Cells fixed 3.7%                        paper: JPS LDS. <br />  <br /> References <br /> 1.  Friedberg EC, Walker GC, Siede W, Wood RD, Schultz RA, et al. (2006) DNA            12. Begley TJ, Rosenbach AS, Ideker T, Samson LD (2002) Damage recovery <br />     repair mutagenesis: ASM Press, Washington, D. C p.                                  pathways <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-24'  ></span>Saccharomyces</span> cerevisiae revealed genomic phenotyping and <br /> 2. Stupp R, Mason WP, van den Bent MJ, Weller M, Fisher B, et al. (2005)                    interactome mapping. Molecular Cancer Research 1: 103&#8211;112. <br />     Radiotherapy plus concomitant adjuvant temozolomide glioblastoma.           13. Begley TJ, Rosenbach AS, Ideker T, Samson LD (2004) Hot spots for <br />     New England Journal Medicine 352: 987&#8211;996.                                           modulating toxicity identified genomic phenotyping localization <br /> 3. Walker MD, Green SB, Byar DP, Alexander E, Batzdorf U, et al. (1980)                     mapping. Molecular Cell 16: 117&#8211;125. <br />     Randomized Comparisons Radiotherapy Nitrosoureas Treatment           14. Bennett CB, Lewis LK, Karthikeyan G, Lobachev KS, Jin YH, et al. (2001) <br />     Malignant Glioma Surgery. New England Journal Medicine 303:                 Genes required ionizing radiation resistance yeast. Nature Genetics 29: <br />     1323&#8211;1329.                                                                              426&#8211;434. <br /> 4. Wei QY, Li CY, Wang LE (2009) DNA repair phenotype cancer                        15. Chang M, Bellaoui M, Boone C, Brown GW (2002) A genome-wide screen for <br />     susceptibility-A mini review. International Journal Cancer 124: 999&#8211;1007.            methyl methanesulfonate-sensitive mutants reveals genes required S phase <br /> 5. Beranek DT (1990) Distribution Methyl Ethyl Adducts Following                     progression presence DNA damage. Proceedings National <br />     Alkylation Monofunctional Alkylating-Agents. Mutation Research 231:                Academy Sciences United States America 99: 16934&#8211;16939. <br />     11&#8211;30.                                                                              16. Giaever G, Chu AM, Ni L, Connelly C, Riles L, et al. (2002) Functional <br /> 6. Scha&#168;rer OD, Campbell AJ (2010) Mechanisms Base Excision Repair                   profiling Saccharomyces cerevisiae genome. Nature 418: 387&#8211;391. <br />     Nucleotide Excision Repair. In: Geacintov NE, Broyde S, eds. The Chemical           17. Hanway D, Chin JK, Xia G, Oshiro G, Winzeler EA, et al. (2002) Previously <br />     Biology DNA Damage. Weinheim, Germany: Wiley-VCH Verlag GmbH &amp;                       uncharacterized genes UV- MMS-induced DNA damage response in <br />     Co. KGaA.                                                                               yeast. Proceedings National Academy Sciences United States of <br /> 7. Galieguezouitina S, Bailleul B, Ginot YM, Perly B, Vigny P, et al. (1986) N2-            America 99: 10605&#8211;10610. <br />     Guanyl N6-Adenyl Arylation Chicken Erythrocyte DNA Ultimate           18. Hillenmeyer ME, Fung E, Wildenhain J, Pierce SE, Hoon S, et al. (2008) The <br />     Carcinogen 4-Nitroquinoline 1-Oxide. Cancer Research 46: 1858&#8211;1863.                  chemical genomic portrait yeast: Uncovering phenotype genes. <br /> 8. Galieguezouitina S, Bailleul B, Loucheuxlefebvre MH (1985) Adducts                  Science 320: 362&#8211;365. <br />     Invivo Action Carcinogen 4-Hydroxyaminoquinoline 1-Oxide Rats         19. Hurov KE, Cotta-Ramusino C, Elledge SJ (2010) A genetic screen identifies the <br />     Invitro Reaction 4-Acetoxyaminoquinoline 1-Oxide DNA                   Triple T complex required DNA damage signaling ATM ATR <br />     Polynucleotides. Cancer Research 45: 520&#8211;525.                                           stability. Genes &amp; Development 24: 1939&#8211;1950. <br /> 9. Wade MH, Chu EHY (1979) Effects DNA Damaging Agents Cultured                   20. Jelinsky SA, Estep P, Church GM, Samson LD (2000) Regulatory networks <br />     Fibroblasts Derived Patients Cockayne Syndrome. Mutation                      revealed transcriptional profiling damaged Saccharomyces cerevisiae cells: <br />     Research 59: 49&#8211;60.                                                                     Rpn4 links base excision repair proteasomes. Molecular Cellular <br /> 10. Suwaki N, Child ES, Elphick LM, Mann DJ (2010) Dose-dependent changes                Biology 20: 8157&#8211;8167. <br />     cyclin D1 response 4-nitroquinoline 1-oxide-induced DNA damage.               21. Jelinsky SA, Samson LD (1999) Global response Saccharomyces cerevisiae to <br />     Archives Biochemistry Biophysics 497: 55&#8211;61.                                     alkylating agent. Proceedings National Academy Sciences the <br /> 11. Drahota Z, Krivakova P, Cervinkova Z, Kmonickova E, Lotkova H, et al. (2005)            United States America 96: 1486&#8211;1491. <br />     Tert-butyl hydroperoxide selectively inhibits mitochondrial respiratory-chain       22. Lovejoy CA, Xu X, Bansbach CE, Glick GG, Zhao RX, et al. (2009) Functional <br />     enzymes isolated rat hepatocytes. Physiological Research 54: 67&#8211;72.                  genomic screens identify CINP genome maintenance protein. Proceedings <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                  9                                    May 2012 | Volume 7 | Issue 5 | e37368 <br />                                                                                                                                     Novel Human Damage Recovery Proteins <br />  <br />  <br />       National Academy Sciences United States America 106:                       mice. Proceedings National Academy Sciences United States <br />       19304&#8211;19309.                                                                                   America 106: 888&#8211;893. <br /> 23.   Paulsen RD, Soni DV, Wollman R, Hahn AT, Yee MC, et al. (2009) A                         44.   Samson LD, Roth RB (2002) 3-methyladenine DNA glycosylase-deficient Aag <br />       Genome-wide siRNA Screen Reveals Diverse Cellular Processes Pathways                       null mice display unexpected bone marrow alkylation resistance. Cancer <br />       Mediate Genome Stability. Molecular Cell 35: 228&#8211;239.                                     Research 62: 656&#8211;660. <br /> 24.   Ross-Macdonald P, Coelho PSR, Roemer T, Agarwal S, Kumar A, et al. (1999)                45.   Nishida Y, Arakawa S, Fujitani K, Yamaguchi H, Mizuta T, et al. (2009) <br />       Large-scale analysis yeast genome transposon tagging gene                        Discovery Atg5/Atg7-independent alternative macroautophagy. Nature 461: <br />       disruption. Nature 402: 413&#8211;418.                                                               654&#8211;U699. <br /> 25.   Svensson JP, Quiros Pesudo L, Fry RC, Adeleye YA, Carmichael P, et al. (2011)            46.   Kanzawa T, Germano IM, Komata T, Ito H, Kondo Y, et al. (2004) Role of <br />       Genomic phenotyping essential non-essential yeast genome detects                    autophagy temozolomide-induced cytotoxicity malignant glioma cells. Cell <br />       novel pathways alkylation resistance. BMC Systems Biology In press.                        Death Differentiation 11: 448&#8211;457. <br /> 26.   Ravi D, Wiles AM, Bhavani S, Ruan JH, Leder P, et al. (2009) A Network                47.   Kanzawa T, Zhang L, Xiao LC, Germano IM, Kondo Y, et al. (2005) Arsenic <br />       Conserved Damage Survival Pathways Revealed Genomic RNAi Screen.                          trioxide induces autophagic cell death malignant glioma cells upregulation <br />       Plos Genetics 5: -.                                                                            mitochondrial cell death protein BNIP3. Oncogene 24: 980&#8211;991. <br /> 27.   van Haaften G, Romeijn R, Pothof J, Koole W, Mullenders LHF, et al. (2006)               48.   Lee SJ, Cho KS, Koh JY (2009) Oxidative Injury Triggers Autophagy in <br />       Identification conserved pathways DNA-damage response radiation                      Astrocytes: The Role Endogenous Zinc. <span id='am-41' about='obo:NCBITaxon_90461' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_241786 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-42'  ></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="Glia" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Glia</span> 57: 1351&#8211;1361. <br />       protection genome-wide RNAi. Current Biology 16: 1344&#8211;1350.                           49.   Paglin S, Hollister T, Delohery T, Hackett N, McMahill M, et al. (2001) A novel <br /> 28.   Garrow A, Adeleye Y, Warner G Available: http://cytoscape.org/cgi-bin/                         response cancer cells radiation involves autophagy formation acidic <br />       moin.cgi/Data_Sets.                                                                            vesicles. Cancer Research 61: 439&#8211;444. <br /> 29.   Fry RC, Svensson JP, Valiathan C, Wang E, Hogan BJ, et al. (2008) Genomic                50.   Ding ZB, Hui B, Shi Y <br /> </body></html>