<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Silva et al. BMC Structural Biology 2011, 11:40 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                             Open Access <br />  <br /> Central domain deletions affect SAXS solution <br /> structure function Yeast Hsp40 proteins <br /> Sis1 Ydj1 <br /> Julio C Silva1,2,6&#8224;, Julio C Borges3&#8224;, Douglas M Cyr4, Carlos HI Ramos5 Iris L Torriani1,2* <br />  <br />  <br />   Abstract <br />   Background: Ydj1 Sis1 structurally functionally distinct Hsp40 proteins yeast cytosol. Sis1 an <br />   essential gene ydj1 gene essential growth elevated temperatures complement sis1 <br />   gene deletion. Truncated polypeptides capable complementing sis1 gene deletion comprise J-domain of <br />   Sis1 Ydj1 connected G/F region Sis1 (but Ydj1). Sis1 mutants G/F deleted <br />   G/M maintained capable complementing sis1 gene deletion. <br />   Results: To investigate relevance central domains structure function Ydj1 Sis1 we <br />   prepared Sis1 constructs deleting specific domains. The mutants decreased affinity heated luciferase but <br />   equally capable stimulating ATPase activity Hsp70. Detailed low resolution structures obtained and <br />   overall flexibility Hsp40 mutants assessed using SAXS methods. Deletion G/M or <br />   G/M plus CTDI domains little impact quaternary structure Sis1 analyzed SAXS technique. <br />   However, deletion ZFLR-CTDI changed relative position J-domains Ydj1 way they <br />   ended resembling Sis1. The results revealed G/F G/M regions flexible <br />   domains. All model structures exhibit common clamp-like conformation. <br />   Conclusions: Our results suggest central domains, previously appointed important features for <br />   substrate binding, relevant keeping J-domains specific relative positions. The clamp-like <br />   architecture observed favorable interactions Hsp40 Hsp70. <br />  <br />  <br /> Background                                                                          families Heat Shock Proteins (HSPs), <span id='am-13' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-14' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> acts <br /> Molecular chaperones proteins involved                              assist protein folding different way. <br /> assisting folding assembly newly synthesized                               An important chaperone family 40-kDa Heat <br /> proteins recognizing non-native substrate proteins pre-                             shock protein (Hsp40). Chaperones Hsp40/ <br /> dominantly exposed hydrophobic residues [1].                              DnaJ family play important roles cells working <br /> However, conditions successful folding                               molecular chaperone Hsp70 promote <br /> vivo favorable. The cellular environment                             protein folding, assembly, translocation degradation <br /> is crowded protein denaturation aggrega-                               [2-5]. Hsp40 proteins interact hydrophobic <br /> tion major problems. Thus, need                            side-chains non-native polypeptides preventing aggre- <br /> chaperones protect cells elevated tem-                               gation [6,7]. Hsp40 form transient complexes <br /> perature cellular stress situations, achieve                            Hsp70 presenting non-native polypeptides sub- <br /> successful folding proteins vivo. There                           sequent protein folding [8-10]. The members the <br />                                                                                     Hsp40 family typically contain J-domain, regu- <br />                                                                                     <span id='am-19' about='obo:NCBITaxon_8186' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_1489905 obo:NCBITaxon_123366 obo:NCBITaxon_1489904 obo:NCBITaxon_1489872 obo:NCBITaxon_8184 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Lates" datatype="xsd:string"></span><span id='am-23'  ></span>lates</span> ATP-dependent binding peptides Hsp70 <br /> * Correspondence: torriani@ifi.unicamp.br <br />                                                                                     [4,11,12]. <br /> &#8224; Contributed equally                                                                 Members Hsp40 family act molecular cha- <br /> 1 <br />  Department Condensed Matter Physics, &#8220;Gleb Wataghin&#65533;? Physics Institute,         perones bind deliver non-native proteins to <br /> State University Campinas (UNICAMP), Campinas, SP 13083-859, Brazil <br /> Full list author information available end article <br />                                                                                     Hsp70 divided main groups, from <br />  <br />                                        &#169; 2011 Silva et al; licensee BioMed Central Ltd. <span id='am-24' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-25' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-28'  ></span>This</span> Open Access article distributed terms Creative Commons <br />                                        Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction in <br />                                        medium, provided original work properly cited. <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                                      Page 2 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br /> which two: Type I Type II studied.                 High-resolution structural studies fragments of <br /> The types functionally equivalent [13-15]             Sis1 Type II Hsp40s indicated pro- <br /> and exhibit major differences chaperone activity [16].         teins function homodimers clamp-like <br /> In types J-domains connected central          architecture use shallow groove located the <br /> and C-terminal domains G/F-rich (<span id='am-1' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Glycine</span>/pheny-             surface monomers bind non-native proteins <br /> lalanine-rich) linker (Figure 1).                                 [20,21]. In addition, G/F-rich regions Ydj1 and <br />   Type I Hsp40s, <span id='am-42' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-43' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-54'  ></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span>Escherichia coli</span> DnaJ, yeast             Sis1 lie adjacent putative polypeptide binding <br /> Ydj1 human Hdj2, contain zinc-finger-like linker            domains specify functions these <br /> region (ZFLR) G/F domain C-ter-               Hsp40s [22-24]. These structure/function studies <br /> minus (Figure 1B), Type II Hsp40 proteins             hydrodynamic analysis showed both <br /> yeast Sis1 human Hdj1 contain G/M-rich (Gly-                types Hsp40 proteins form dimers solution <br /> cine/Methionine-rich) domain linker region (Fig-           [6,18,19]. Dimer formation plays critical role in <br /> ure 1A) [14-17]. Both types substrate-binding site         Hsp40s chaperone activity disruption the <br /> located C-terminal domain, divided           dimerization motifs results severe defects cha- <br /> subdomains I (CTDI) II (CTDII) (Figure 1).                    perone functions [20,25]. <br />   The reason Type I Type II Hsp40s exhibit                  Previous results showed human yeast Type I <br /> differences chaperone activity unknown. Some bio-           Type II Hsp40s, distinct quaternary structure <br /> chemical structural studies provided             [18,19]. These results raised hypothesis cen- <br /> some insight question, suggesting              tral domains controlled quaternary structure both <br /> answer resides structural differences [15,18,19].        types Hsp40s, chimeric mutants, which <br /> Hence, need comprehensive study                central domains Ydj1 (ZFLR) Sis1 (G/M) <br /> structure considerable importance.                          switched, properties exchanged. A <br />  <br />  <br />  <br />  <br />  Figure 1 Domain organization Hsp40s mutants. (A) Schematic representation Sis1 domains known high-resolution <br />  structures. (B) Schematic representation Ydj1 domains high-resolution structures. <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                                               Page 3 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br /> chimeric Ydj1 ZFLR switched                   G/M region deleted, Sis1_&#916;121- <br /> the G/M Sis1 proved functionally struc-                 257, G/M CTDI been <br /> turally similar Type II Sis1. Correspondingly, chi-               deleted (Figure 1A). Ydj1 yeast Type I Hsp40 that <br /> meric Sis1 G/M switched                   contains 409 residues arranged highly con- <br /> ZFLR Ydj1 proved functionally structu-                  served a-helical N-terminal J-domain, disordered mid- <br /> rally similar Type I Ydj1 [15,19].                                  dle region (a glycine/phenylalanine (G/F)), Zinc finger <br />   To increase knowledge role central                 domain (ZFLR Cys-rich domain) C-term- <br /> domains structure/function relationship                      inal sub-domains (CTDI CTDII) shown Figure <br /> defined biophysical functional features Sis1                1B. Again, understand role regions in <br /> and Ydj1 mutants deleted central domains                     protein structure, studied Ydj1 deletion <br /> were specific Type I Type II Hsp40s. These stu-                 mutant named Ydj1_&#916; 106-255 , the <br /> dies entailed functional analysis mutated Hsp40s                  Zinc finger-like region (ZFLR) CTDI been <br /> coupled biophysical investigation quaternary               deleted (Figure 1B). The mutants purified no <br /> structure dynamic light scattering, analytical ultra-               apparent contamination (&gt;95% pure; Figure 2A) and <br /> centrifugation small-angle X-ray scattering (SAXS).                maintained 4&#176;C avoid degradation. The folded con- <br /> A refined solution structure Sis1 obtained                 formation proteins investigated circular <br /> using improved SAXS <span id='am-70' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-71' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74'  ></span>data</span>. The outcome stu-                    dichroism (CD) spectroscopy (Figure 2B). As previously <br /> dies showed deletion G/M G/M                 shown wild-type proteins Sis1 Ydj1 [19], the <br /> plus CTDI domain regions minor impact                       mutants CD spectra folded proteins with <br /> overall quaternary structure Sis1. Consequently,                minima 208 220 nm positive peak <br /> results suggest central domains important                 bellow 200 nm (Figure 2B). The shapes spectra <br /> for substrate binding maintenance J-domains                 indicated large unfolded portion present. <br /> in specific relative positions.                                  This good agreement results hydro- <br />                                                                        dynamic measurements (see below). <br /> Results Discussion <br /> Protein samples                                                        Hsp40 function <br /> Sis1 yeast member Type II Hsp40s family                Hsp40s act binding unfolded partially unfolded <br /> contains 352 residues arranged highly conserved a-                protein (client protein) delivering Hsp70 con- <br /> helical N-terminal J-domain, disordered middle region                comitantly stimulating Hsp70 ATPase activity. ATP <br /> (divided glycine/phenylalanine (G/F) glycine/                 hydrolysis Hsp70 crucial step protein folding <br /> methionine (G/M) rich regions) C-terminal                      assisted chaperone. The effect deletions <br /> sub-domains (CTDI CTDII) shown Figure 1A.                    function Hsp40s assayed testing <br /> In <span id='am-39' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> understand role central regions              ability bind client protein <span id='am-65' about='obo:NCBITaxon_1113472' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_40096 obo:NCBITaxon_40092 obo:NCBITaxon_33208 obo:NCBITaxon_40093 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-66' property="rdfs:label" content="Stimula" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-69'  ></span>stimula</span>- <br /> the structure function Type II Hsp40s, Sis1                 tory effect ATPase activity Hsp70 (Figure 3). <br /> deletion mutants produced: Sis1_&#916; 124-174 ,                  First, ability bind heated luciferase (a client <br />  <br />  <br />  <br />  <br />  Figure 2 Results protein expression: (A) SDS-PAGE 12% showing recombinant mutant proteins &gt; 95% pure. Molecular <br />  masses standard proteins shown left. (B) CD spectra recombinant proteins (10-40 &#956;M) measured 195 260 nm 25 <br />  mM Tris-HCl (pH 7.5) containing 500 mM NaCl shown molar residual ellipticity ([&#920;]). <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                                                       Page 4 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br />  Figure 3 Functional tests: (A) Hsp40s ability bind client proteins 40%. (B) Hsp40s ability stimulate ATPase activity Hsp70 <br />  90% compared Sis1. (See text details). <br />  <br />  <br /> protein) tested Sis1 binding set stan-                  s 0 20,w induced external factors (pH changes, salt <br /> dard (100%). Compared Sis1, efficacy                             strength, ligands temperature) related conforma- <br /> Sis1_&#916;124-174, 60%, Sis1_&#916;121-257                    tional changes [26]. The values s 0 20,w D 0 20,w , <br /> and Ydj1_&#916;106-255 40% (Figure 3A). Sec-                      obtained dynamic light scattering (DLS) experi- <br /> ond, mutants assayed regarding ability                   ments, shown Table 1. The M values obtained <br /> stimulate ATP hydrolysis Ssa1 (Hsp70) Sis1                      c(M) distribution s/D ratio <br /> stimulatory effect set standard (100%). Sis1_&#916;124-                  (Equation 4), weight average factor &#402;/&#402;0 also <br /> 174 Ydj1_&#916;106-255 effect similar Sis1,                  shown (Table 1). Our results suggested Sis1_&#916;124- <br /> inside error, effect Sis1_&#916;121-257                   174 , Sis1_&#916; 121-257 Ydj1_&#916; 106-255 dimers in <br /> about 90% (Figure 3B). For comparison, perfor-                         solution asymmetric elongated <br /> mance Ydj1 experiments similar                      shape shown previously Hsp40 proteins [18,19]. <br /> of Sis1, experimental error (data shown). The <br /> results deletions decreased                   SAXS results <br /> 50% ability bind client proteins,                      To explore impact selective central domains dele- <br /> have effect ability J-domain interact-                 tion Hsp40s quaternary structure, SAXS analysis was <br /> ing Hsp70. These results suggested J-                        performed. The corrected normalized experimental <br /> domains maintained proper conformation                           SAXS curves proteins Sis1, Sis1_&#916; 124-174 , <br /> functionality.                                                             Sis1_&#916;121-257, Ydj1_&#916;106-255 q range 0.01 &lt; q <br />                                                                            &lt; 0.25&#197;-1 displayed Figure 5(A), the <br /> Hydrodynamics                                                              respective regularization fitting (solid lines) resulting <br /> Sis1 Ydj1 dimers solution used                      p(r) function calculation using GNOM. The <br /> analytical ultracentrifugation (AUC) investigate                    corresponding p(r) functions, shown Figure 5(B), indi- <br /> oligomeric status deleted mutants. We performed                     cated slightly elongated shape proteins in <br /> AUC sedimentation velocity (SV) experiments fitted                     solution, confirming AUC results. The Kratky plots <br /> the data using SedFit supplied continuous sedimen-                    presented Figure 5(C) showed proteins are <br /> tation distribution c(S) (Figure 4A). From maximum                     quite flexible solution, gave clue about <br /> of peaks c(S) curve, apparent s,                  difficulty crystallizing them. An inspection the <br /> corrected standard conditions (s 20,w ) plotted                     Porod plots Figure 5(D) shows curves not <br /> against protein concentration (Figure 4B). The extrapo-                    contain significant plateau region indicating well <br /> lation s20,w 0 mg/mL gave s020,w intrin-                 defined particle volumes, plots Sis1 and <br /> sic property protein contains information                       Sis1_&#916;124-174 present fairly flat region q4-range <br /> about molecular mass (M) asymmetry                        0.0012&lt;q4&lt;0.0025&#197;-4, suggesting compact con- <br /> of molecule. Normally, variation value                     formation proteins. The complete <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                                                    Page 5 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br />                                                                                    deviation Porod regime leads predict more <br />                                                                                    flexible structures Sis1_&#916;121-257, Ydj1_&#916;106- <br />                                                                                    255 proteins. The degree flexibility individual <br />                                                                                    protein confirmed plots. The inset in <br />                                                                                    Figure 5(A) displays ln(I(q)) vs. q2 plot the <br />                                                                                    validity region Guinier approximation (qR g&lt;1) <br />                                                                                    corresponding linear regression for <br />                                                                                    protein. The linearity plots confirmed the <br />                                                                                    monodispersity samples. The maximum dimen- <br />                                                                                    sion (Dmax) values, radii gyration (Rg) obtained by <br />                                                                                    Guinier approximation p(r) functions, as <br />                                                                                    calculated estimated M plus oligo- <br />                                                                                    merization states proteins presented in <br />                                                                                    Table 2. The Rg values obtained approaches <br />                                                                                    close agreement. The reliable Rg value for <br />                                                                                    molecule obtained p(r) function, <br />                                                                                    derived complete experimental curve. As <br />                                                                                    described Material Methods, bovine serum <br />                                                                                    albumin (BSA) solution used reference sample <br />                                                                                    M estimation using SAXS data. The results <br />                                                                                    indicated proteins exist dimeric state in <br />                                                                                    solution, values obtained approximately <br />                                                                                    twice values calculated primary sequence <br />                                                                                    (Table 2). <br />  <br />                                                                                    Ab initio rigid body model calculations based on <br />                                                                                    SAXS data <br />                                                                                    Two ab-initio computational routines used cal- <br />                                                                                    culate low resolution models molecular <br />                                                                                    envelopes proteins Sis1, Sis1_&#916;124-174, Sis1_&#916;121- <br />   Figure 4 Results analytical ultracentrifugation (AUC) <br />                                                                                    257 , Ydj1_&#916; 106-255 proteins. The Dummy Atoms <br />   sedimentation velocity (SV) experiments. (A) The Figure shows <br />   experiments using 1,000 &#956;g/mL protein concentration 25                     (DA) Dummy Residues (DR) models derived <br />   mmol/L Tris-HCl (pH 7.5) containing 500 mmol/L NaCl. (B) The                     X-ray scattering data introducing 2-point <br />   software Sednterp used correct apparent s s20,w.               symmetry constraint calculations. This assump- <br />   Proteins s020,w values calculated plot s20,w versus             tion based solution scattering data, which <br />   protein concentration shown Table 1. <br />                                                                                    indicated molecules study dimers <br />                                                                                    solution. The crystallographic structure identified <br />  <br />  <br /> Table 1 Summary hydrodynamic data obtained DLS AUC experiments. <br />       Hydrodynamic properties                                                                  Protein <br />                                                      Sis1                   Sis1_&#916;124-174                Sis1_&#916;121-257       Ydj1_&#916;106-255 <br />           Predicted M (kDa)*                         75.2                        66.1                        46.8                56.1 <br />          Experimental M (kDa)                      64 &#177; 3&#8364;                 58 &#177; 2# 58 &#177; 1&#8364;           45 &#177; 2# 43 &#177; 1&#8364;        54 &#177; 1# 55 &#177; 2&#8364; <br />                 s020,w (S)#                       3.5 &#177; 0.1%                   3.4 &#177; 0.1                   3.0 &#177; 0.1           3.6 &#177; 0.1 <br />                   ssph (S)                            6.5                         5.9                         4.7                 5.2 <br />           D20,w (10-7cm2/seg)                     5.1 &#177; 0.2                    5.4 &#177; 0.1                   6.3 &#177; 0.1           5.9 &#177; 0.2 <br />            Dsph (10-7cm2/seg)                         7.7                         8.0                         9.0                8.45 <br />                   &#402;/&#402;0#                           1.6 &#177; 0.1                    1.6 &#177; 0.1                   1.5 &#177; 0.1           1.4 &#177; 0.1 <br /> * M predicted amino acids sequence proteins dimers. <br /> # <br />   data SV experiments fitted SedFit software (see Material methods details); <br /> &#8364; <br />   M calculated s/D ratio showed Equation 4; <br /> % <br />   data [19]. <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                                                              Page 6 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br />   Figure 5 SAXS results proteins Sis1, Sis1_&#916;124-174, Sis1_&#916;121-257, Ydj1_&#916;106-255. (A) Scattering intensity curves the <br />   proteins. The inset shows Guinier region showing linear regression fit experimental data proteins. (B) Corresponding p(r) <br />   functions. (C) Kratky plots showing different degrees flexibility protein. (D) Porod plots proteins. A small plateau region in <br />   curves Sis1 Sis1_&#916;124-174 indicate defined molecular volume macromolecules. <br />  <br />  <br /> the Sis1 peptide binding fragment homodimer                            generally chosen application program <br /> the crystal [20] information modeled                         (qmax~ 8/Rg). The q-range range (0.01&lt;q&lt;0. 25 &#197;-1) <br /> dimer structure Ydj1 peptide binding fragment                           used DR rigid body modeling routines <br /> [25] taken account calculations. The q-                       (see Methods). Keeping mind SAXS ab initio <br /> range used DAMMIN model calculation                             modeling routines produce unique solution, <br />  <br />  <br /> Table 2 Overall parameters derived SAXS results proteins Sis1, Sis1_&#916;124-174, Sis1_&#916;121-257 and <br /> Ydj1_&#916;106-255 <br />                                                                                              Protein <br />        Structural properties                      Sis1                   Sis1_&#916;124-174                 Sis1_&#916;121-257                 Ydj1_&#916;106-255 <br />              C (mg/mL)                           7.1, 3.1                   6.8, 4.2                      5.2, 3.9                      5.5, 2.75 <br />            Rg (&#197;) (Guinier)                      42 &#177; 1                     42 &#177; 1                        35 &#177; 1                         33 &#177; 1 <br />           Rg (&#197;) (from p(r))                   43.3 &#177; 0.5                 42.6 &#177; 0.3                     35.2 &#177; 0.3                    34.2 &#177; 0.3 <br />               Dmax (&#197;)                             160                        140                           125                           110 <br />            M (kDa) (SAXS)                         ~71                        ~69                            ~42                           ~47 <br />          Predicted M (kDa)*                       75.2                       66.1                           46.8                          56.1 <br />        Oligomerization state                     Dimer                      Dimer                          Dimer                         Dimer <br /> * M predicted amino acids sequence proteins dimers. <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                                          Page 7 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br /> ten independent runs performed                        identification position individual domains <br /> calculation.                                                        RB models. <br />    The resolution DA DR models does                     The ab initio rigid body models calculated for <br /> permit unambiguous determination spatial                  length Sis1 protein deleted-domain <br /> positions secondary structure elements,                 mutants presented Figure 6. The DA (filter aver- <br /> portray overall recurring extended shape                 aged), DR RB models proteins Sis1, Sis1_&#916;124- <br /> most frequent conformations adopted mole-                  174 , Sis1_&#916; 121-257 , shown panels A, B C <br /> cules solution. A spherical start volume used                respectively. In case DR models, averaging <br /> for DA modeling calculation order minimize                does add substantially results. So, mod- <br /> the generation specific direction p2                 els displayed presenting lowest normal- <br /> symmetry introduced based fact mole-                ized spatial discrepancy (NSD) values, also <br /> cules dimers. Also, dealing                 showed best agreement DA model. NSDs <br /> flexible molecules, important check DR             tend zero nearly similar objects they <br /> and Rigid Body approaches (non initial-volume depen-                exceed 1, objects systematically differ one <br /> dent) giving similar results. Consequently, rigid              (as explained Methods section). The <br /> body (RB) calculations performed attempt                 NSDs calculated considered reasonably good <br /> to obtain information position J-domain               DA DR modeling approaches account <br /> and domains available crystallographic data              flexibility conformational changes these <br /> to restore protein structures based SAXS data.               molecules. The itemized NSD values DA, DR <br /> In cases, tried model molecular envelope              RB models Sis1, Sis1_&#916;124-174, Sis1_&#916;121-257, are <br /> of protein, performing calculations 2-point                listed Table 3. <br /> symmetry constraints proteins dimeric                   The filtered average ab initio DA model, the <br /> structure explained above. Also, previously men-              DR model lowest NSD value rigid <br /> tioned, dimerization sites proteins                 body model Ydj1_&#916;106-255 protein presented <br /> known priori X-ray crystallographic                    Figure 7. These results obtained following the <br /> structure information used impose                   procedures previously described Sis1 full <br /> dimerization contact conditions RB calculations.             length protein deleted domain mutants. The <br /> The position linker C-terminal                   NSD values corresponding model calculations <br /> domains J-domain protein known                  (Table 3) values considered <br /> from amino acid sequences. Ten runs per-                   reasonably good different approach. The DR <br /> formed set calculations. The multiple runs              model lowest NSD value presents remarkably <br /> gave coincident results allowed                      good resemblance DA rigid body models, <br />  <br />  <br />  <br />  <br />  Figure 6 Results ab initio (DA DR) model calculations rigid body models displayed length <br />  Sis1 protein deleted mutants. (A) Sis1, (B) Sis1_&#916;124-174, (C) Sis1_&#916;121-257. <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                                        Page 8 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br /> Table 3 Normalized Spatial Discrepancy (NSD) range obtained pairwise comparison SAXS models each <br /> group calculations <br />                                                                                    Protein <br />         Modelling method                     Sis1                  Sis1_&#916;124-174             Sis1_&#916;121-257       Ydj1_&#916;106-255 <br />           Dummy atoms                     1.00 - 1.22               1.17 - 1.41               0.75 - 1.01         0.88 - 1.21 <br />          Dummy residues                   1.41 - 1.97               1.84 - 2.68               1.47 - 1.76         1.38 - 1.56 <br />       Ab initio rigid body            1.95 - 2.87               1.95 - 2.51               1.54 - 1.85         1.87 - 2.11 <br />  <br />  <br /> and considered excellent description                   activity, did decrease polypeptide binding activity. <br /> Ydj1_&#916;106-255 protein conformation solution.                         Surprisingly, deletion G/F region did change <br />                                                                         quaternary structure overall flexibility Hsp40s. <br /> Conclusions                                                             However, deleting ZFLR plus CTDI domain in <br /> In previous studies, Ydj1 Sis1                     Type I Ydj1 altered conformation J-domains from <br /> homodimers, structures differed context                 lying long <span id='am-34' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>axis</span> molecule orien- <br /> the orientation J-domain relation long-                tated crosswise direction Type II Sis1. Thus, <br /> axis respective proteins [18,19]. These differences              orientation J-domains Type I Hsp40 con- <br /> in structure correlate functional differences exhib-               trolled ZFLR-CTDI, consistent with <br /> ited Ydj1 Sis1, G/F G/M regions                   results domain swap experiments [15,19]. When <br /> these proteins proposed impact J-domain orien-                  domains missing, deleted mutants <br /> tation [15]. To test hypothesis carried                     studied Type II Hsp40s, J-domains <br /> hydrodynamic low-resolution structural studies                   oriented crosswise direction. Our results <br /> Sis1 Ydj1 deletion mutants. The deletion G/F                 point interactions ZFLR-CTDI and <br /> region Sis1 Ydj1 little impact func-                J-domains important factor determining the <br /> tion J-domain regulating Hsp70 ATPase                         unique quaternary structure Type I Hsp40s. <br />                                                                            One interesting feature Type I Type II Hsp40s <br />                                                                         clear data sub-types of <br />                                                                         Hsp40s highly flexible. The SAXS intensity data <br />                                                                         obtained wider range reciprocal space the <br />                                                                         present experiments allowed comparative analysis of <br />                                                                         flexible nature proteins means the <br />                                                                         Kratky Porod plots. These results showed the <br />                                                                         Type I Ydj1 Type II Sis1 Hsp40 proteins highly <br />                                                                         flexible solution confirm dimerization the <br />                                                                         C-terminal. Besides, flexibility observed con- <br />                                                                         structs studied, deleting flexible region (G/ <br />                                                                         M). So, flexible regions exist protein <br />                                                                         addition G/M. Prior studies presented <br />                                                                         assumed G/F G/M domains <br />                                                                         flexible regions Hsp40s enabled <br />                                                                         J-domain multiple paths gain access Hsp70 <br />                                                                         ATPase domain. However, Kratky Porod analysis of <br />                                                                         SAXS data Sis1 Ydj1 deletion mutants that <br />                                                                         remain highly flexible absence G/F <br />                                                                         region. Hsp40s bind deliver proteins range <br />                                                                         nascent monomeric polypeptides amyloid-like <br />                                                                         aggregates Hsp70. Thus, overall flexibility Type <br />                                                                         I Type II Hsp40s detected important per- <br />                                                                         mit delivery proteins different sizes assembly <br />                                                                         states Hsp70 polypeptide binding site allow <br />  Figure 7 Results ab initio (DA DR) model                    simultaneous interaction J-domains with <br />  calculations rigid body model obtained                 Hsp70s ATPase domain [5]. These results in <br />  Ydj1_&#916;106-255 protein, using high resolution data available        agreement crystallographic structural studies <br />  domains. <br />                                                                         reported Hu et al. [27], observed that <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                             Page 9 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br /> the CTDI human Hsp40 possess significant flex-         favorable interactions Hsp40 with <br /> ibility. In work, authors proposed &#8220;anchor-      Hsp70. The models built proteins to <br /> ing docking&#65533;? model Hsp40                 agree anchoring docking model, proposed <br /> flexibility CTDI important Hsp40         Qian et al. [10], describing Hsp40 facilitates the <br /> regulate size cleft interaction non-   delivery non-native polypeptides Hsp70. <br /> native polypeptides transfer Hsp70. <br />   Functions G/F G/M regions Type         Methods <br /> II Hsp40 regulation Hsp70 ATPase activity and/or        Protein expression purification <br /> substrate binding clear. Yet, important    The recombinant protein Sis1 Ydj1 expressed <br /> question Sis1 fragment containing just J-       purified chromatographic steps previously <br /> domain G/F region capable rescuing lethal-      described [15,19,21]. Two Sis1 mutants prepared <br /> ity sis1&#916; strain [22]. In addition, amino acid resi-   DNA constructions expressed <span id='am-58' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-59'  ></span><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span>Escherichia</span> <br /> dues G/M region appear play role               coli BL21(DE3)pLys: Sis1_&#916; 124-174 (pET11aSIS1_&#916; 124- <br /> specification Sis1 function [28]. Studies forms    174), deleting residues 124 174, Sis1_&#916;121- <br /> Sis1 G/M domain deleted            257 (pET11aSIS1_&#916;121-257), deleting residues 121 to <br /> loss G/M impairs ability Sis1 bind dena-     257. Additionally, DNA construction Ydj1 <br /> tured luciferase, effect regulation         mutant (pET11aYdj1_&#916; 106-255 ) deleted residues <br /> Hsp70 ATPase activity. The G/M region lies adjacent        106 255, prepared expressed E. <br /> the hydrophobic grove CTDI implicated polypep-        coli BL21(DE3) strain. Cells grown 37&#176;C to <br /> tide binding site methionine hydrophobic       optical density 600 nm 0.7. The temperature <br /> chain, help build proper binding site.       reduced 30&#176;C protein expression was <br /> Thus, G/M region helps specify Type II Hsp40 func-        induced 0.4 mM isopropyl thio-b-D-galactoside <br /> tion assisting substrate binding.                       (IPTG) 4 hours. Thereupon, cells har- <br />   Since effects function likely related   vested centrifugation 10 min 2,600 &#215; g. <br /> changes structure Hsp40s, important         The pellet ressuspended 50 mM Tris-HCl (pH <br /> objective work obtain low resolution mod-      8.0), 500 mM KCl 1 mM EDTA (15 mL/L LB <br /> els mutants study using latest computa-      medium). The cells lysed adding 30 &#956;g/mL of <br /> tional methods available spatial representation    lysozyme (Sigma) 5 U DNAse (GIBCO BRL), <br /> these molecules solution using small angle scattering      kept 30 min ice bath, disrupted soni- <br /> data. To date, crystallographic structures       cation centrifuged (30 min 26,000 &#215; g). <br /> isolated domains quaternary structure                 The purification proteins performed pre- <br /> Hsp40s mainly known result SAXS data          viously described [16,29]. Summarily, Sis1, Sis1_&#916;124-174 <br /> these proteins solution, combined hydro-        Sis1_&#916;121-257 submitted cationic chroma- <br /> dynamic techniques. Previous structural studies          tography Macro-prep (TM) High S Support resin <br /> also proved Type I Ydj1 Type II Sis1 dis-       (BioRad) using &#196;KTA FPLC device (Pharmacia Bio- <br /> tinct functions quaternary structures [15,19]. In    tech). The resin equilibrated 20 mM Tris-HCl <br /> work, estimation molecular masses SAXS            buffer (pH 7.5) 20 mM NaCl. Ydj1 Ydj1_&#916;106- <br /> AUC data indicated deletion mutants solu-         255 submitted anionic chromatography a <br /> tion dimerized C-terminal, just like full-length   Macro-prep(R) High Q Support resin (BioRad) using an <br /> protein. Thus, clear C-terminal region      &#196;KTA FPLC device (Pharmacia Biotech). The proteins <br /> very important dimerization Type I            eluted NaCl gradient, dialyzed overnight against <br /> Type II Hsp40s.                                               buffer 20 mM Phosphate (pH 7.5), purified <br />   Using different low-resolution modeling meth-         chromatography CHTTM Ceramic Hydroxyapa- <br /> ods, obtained molecular envelopes protein.        tite Type II resin (BioRad) &#196;KTA FPLC (Pharma- <br /> The rigid body modeling method          cia Biotech). The target proteins eluted a <br /> appropriate flexible proteins like Hsp40 elu-    phosphate gradient. Ydj1_&#916;106-255 purified <br /> cidated domain arrangement important          size exclusion chromatography Superdex <br /> understand possible functions deleted domain           200pg using &#196;KTA FPLC (Pharmacia Biotech) pre- <br /> mutants cell. Interestingly, deleting        viously equilibrated 25 mM Tris-HCl buffer (pH <br /> flexible domains Sis1 Ydj1, constructs kept        7.5) 500 mM NaCl. The efficacy purification <br /> their flexibility maintained clamp-like architec-     checked 12% SDS-PAGE. Unless stated other- <br /> ture full-length protein J-domains point-     wise, proteins diluted buffer 25 mM Tris- <br /> ing outwards opposite directions. This architecture        HCl (pH 7.5) containing 500 mM NaCl. <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                             Page 10 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br /> Circular dichroism                                            applying equation 1 frictional ratio (&#402;/&#402; 0 ) is <br /> Circular dichroism (CD) measurements performed           used indicate particle asymmetry compared to <br /> using Jasco J-810 spectropolarimeter coupled Pel-      globular protein M giving information the <br /> tier-type System PFD 425S temperature control         shape proteins [26,31]. <br /> optimized best performance previously described <br /> [30]. The proteins re-suspended buffer 25 mM          Analytical Ultracentrifugation <br /> Tris-HCl (pH 7.5) containing 500 mM NaCl. Proteins con-       Analytical ultracentrifugation (AUC) experiments were <br /> centration ranged 10 40 &#956;M spectra       performed Beckman Optima XL-A analytical ultra- <br /> collected scan rate 50 nm/min spectral band-   centrifuge. Sedimentation velocity (SV) experiments <br /> width 1 nm using 0.2 mm path length cell.            carried concentrations ranging 150 to <br />                                                               1,000 &#956;g/mL. The SV experiments performed at <br /> Chaperone activity                                            20&#176;C, using 30,000 rpm (AN-60Ti rotor) Sis1_&#916;121- <br /> Hsp40s activities tested ability bind        257 Ydj1_&#916;106-255, 25,000 rpm Sis1_&#916;124-174. <br /> heated denaturated luciferase stimulate Hsp70          The SedFit software (Version 9.4) used deconvo- <br /> Ssa1 ATPase activity previously described [15,21].         lute sedimentation diffusion data order to <br /> The ability bind heated luciferase (a client protein)      obtain continuous sedimentation distribution c(S) <br /> was tested Sis1 binding set standard           weight average value frictional ratio. The &#402;/&#402;0 <br /> (100%). With respect Hsp40s ability stimulate       value used parameter regularization <br /> ATPase activity Hsp70, mutants assayed            function used estimate molecular mass <br /> regarding ability stimulate ATP hydrolysis        c(M) plots [32,33]. The apparent sedimentation <br /> Ssa1 (Hsp70) Sis1 stimulatory effect set       coefficients (s) obtained maximum peak <br /> standard (100%).                                              values c(S) curves. The standard sedimentation <br />                                                               coefficients (s20,w) protein concentration were <br /> Dynamic Light Scattering                                      estimated avoid interferences caused viscosity and <br /> The experimental diffusion coefficient (D) obtained       density increment [26,31]. The Sednterp software http:// <br /> by dynamic light scattering (DLS) using DynaPro-MS/         www.jphilo.mailway.com/download.htm used to <br /> X device (Protein Solutions). The experiments per-       estimate important hydrodynamic parameters: (1) the <br /> formed 20&#176;C, proteins concentration ranged        partial specific volume (V bar ) Sis1 (0.7263 mL/g), <br /> 0.5 2.0 mg/mL. The D value corrected stan-          Sis1_&#916;124-174 (0.7312 mL/g), Sis1_&#916;121-257 (0.7284 mL/g) <br /> dard conditions (D20,w ) extrapolated 0 mg/mL          Ydj1_&#916;106-255 (0.7331 mL/g) amino acid <br /> concentration (D020,w) order avoid effects viscos-   sequence; (2) ssph Dsph globular protein of <br /> ity temperature. D related frictional coeffi-   molecular mass M buffer viscosity (h = 1.0605 <br /> cient (&#402;) following equation:                          x10-2 poise) density (r = 1.01938 g/mL); (3) to <br />                                                               correct apparent value s s 20,w . The standard <br />           RT <br />    D=                                                   (1)   sedimentation coefficient extrapolated 0 mg/mL (s020, <br />           NA f <br />                                                               w) calculated linear regression values s20, <br />  <br />   T absolute temperature, R gas con-      w versus protein concentration. The molecular mass <br />  <br /> stant NA Avogadro&#8217;s number.                        values obtained ratio sedimentation <br />   For protein known Stokes radius (Rs) visc-       diffusion coefficient using following equation. <br /> osity (h), &#402; obtained applying Stokes equa-                    sRT <br /> tion:                                                            M=                                                  (4) <br />                                                                       D(1 &#8722; Vbar &#65533;?) <br />     f = 6&#960; &#951;Rs                                          (2) <br />   For comparison, frictional coefficient spheri-    Small-Angle X-ray scattering experiments <br /> cal particle (&#402;0) calculated using predicted       SAXS experiments performed D02A-SAXS2 <br /> Stokes radius (R0) smooth compact spherical         beamline Laborat&#243;rio Nacional Luz S&#237;ncrotron <br /> protein molecular mass M:                                  (LNLS, Campinas-SP, Brazil). The X-ray scattering data <br />                         1/3                                   recorded using two-dimensional position-sensi- <br />               3MVbar                                          tive MARCCD detector. The measurements per- <br />    R0 =                                                 (3) <br />               4&#960; NA                                           formed monochromatic X-ray beam (wavelength <br />                                                               l = 1.488 &#197;) sample-to-detector distance of <br />  Vbar partial specific volume, &#402;0 used     1374.4 mm, corresponding scattering vector <br /> obtain maximum diffusion coefficient (D sph )             range 0.01 &lt; q &lt; 0.25&#197;-1, q magnitude <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                            Page 11 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br /> of q-vector defined (2&#952; scattering angle).    qualitative information degree flexibility <br /> The samples placed 1-mm path length cell          compactness [38,39,41-43]. <br /> with mica windows [34]. The scattering patterns were <br /> recorded different sample concentrations         Ab initio modeling based SAXS data <br /> each sample: 7.1 3.1 mg/mL Sis1, 6.8 4.2        Ab initio calculations based SAXS data per- <br /> mg/mL Sis1_&#916; 124-174 , 5.2 3.9 mg/mL            formed obtain low-resolution models confor- <br /> Sis1_&#916;121-257, 5.5 2.75 mg/mL Ydj1_&#916;106-255.    mation following proteins: Sis1, Sis1_&#916; 124-174 , <br /> All samples measured buffer 25 mM Tris-         Sis1_&#916;121-257, Ydj1_&#916;106-255. Two different ab initio <br /> HCl (pH 7.5) containing 500 mM NaCl. Three succes-          approaches applied using dummy atoms (DA) <br /> sive frames 300s recorded sample      dummy residues (DR) modeling methods. Given the <br /> and frames buffer. The scattering          existence certain flexibility proteins those <br /> curves individually corrected detector response    types calculations necessary identify the <br /> and scaled incident beam intensity sam-      existence common structural features the <br /> ples absorption. The corrected buffer scattering curve      different models. The dummy atoms modeling method <br /> was subtracted corresponding sample scatter-       provided bead model calculated intensity fitted <br /> ing. The resulting curves normalized respec-    experimental SAXS curve (see Figure S1 Addi- <br /> tive concentrations carefully inspected check    tional file 1). This DA approach implemented using <br /> possible radiation-induced damage concentration         program DAMMIN [44] q-range (qmax &lt;8/Rg) <br /> effects, effects observed. A 5.6 mg/      ends slightly different protein. <br /> mL bovine serum albumin (BSA, 66 kDa) solution             Since unique solution obtained cal- <br /> used standard sample determine molecular        culation, independent calculations per- <br /> mass proteins. The molecular mass estimated      formed. Thereupon, models pair wise compared <br /> by comparison extrapolated value intensity    averaged using programs DAMAVER <br /> at origin value, I(0), samples scattering data   suite [45] SUPCOMB [46]. The program aligns <br /> with reference solution Bovine Serum       models represented ensembles points mini- <br /> Albumin (BSA) described Orthaber et al. [35]      mizing dissimilarity measure called Normalized Spatial <br /> Mylonas et al. [36].                                        Discrepancy (NSD). Generally, NSD values tend zero <br />                                                             increasingly similar objects; significantly <br /> SAXS data analysis                                          exceed 1, objects systematically differ one <br /> Determination radius gyration (Rg) per-       [46]. Subsequently, new calculations per- <br /> formed using Guinier approximation:                     formed protein using DR approach. The <br />                                                             dummy residues modeling method provides further <br />    I(q) = I(0) exp(&#8722;q2 R2g /3)                        (5)   insights possible dimensional conformation <br />                                                             proteins deleted mutants solution. <br />   valid q-range qRg&lt;1 [37-39]. The linear-     This implemented using program GASBOR [47] <br /> ity scattering curves validity region con-    using range q values. Again, indepen- <br /> firmed monodispersity samples allowed            dent calculations performed NSD values <br /> further analysis. Moreover, Rg values evaluated   evaluated. However, average DR represen- <br /> from pair distance distribution function p(r) calcu-    tations does substantially improve quality the <br /> lated scattering intensity data means    models flexibility molecules. So, quan- <br /> Indirect Fourier Transform package GNOM [40]. The p         titative analysis NSD values obtained sev- <br /> (r) function provided maximum dimension            eral models performed order obtain the <br /> Dmax molecule, p (r &#8805; Dmax) = 0 [38,39].     appropriate molecular conformation (i.e., model <br /> Both Guinier approximation, calculated p(r)     having lowest average NSD value), the <br /> function provided values forward scattering         low resolution structure protein. <br /> intensity I(0). These I(0) values used esti- <br /> mation molecular mass. The confirmation       SAXS-based Modeling multidomain arrangement <br /> monodispersity dimerization Sis1, Sis1_&#916;124-     proteins studied <br /> 174 , Sis1_&#916; 121-257 , Ydj1_&#916; 106-255 proteins      The topology protein deleted-domain <br /> inferred molecular mass obtained         mutant Sis1 examined applying rigid body <br /> proteins. The flexibility molecules estimated    modeling method SAXS data. This approach <br /> from scattered intensity analyzing behavior      employs simulated annealing protocol opti- <br /> the Kratky curves (q2I(q) versus q) [38,39] Porod       mal positions orientations high-resolution struc- <br /> plots (q4I(q) versus q4). These representations provided    tures known regions protein. At same <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                                                                Page 12 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br /> time, conformation obtained unknown                                 hydrodynamics spectroscopic experiments interpreted them <br />                                                                                 CHIR. DMC produced deletion mutants performed <br /> regions (flexible linker attached appropriate resi-                      functional experiments. All authors organized results, wrote the <br /> dues domains) calculated finding best fit                        manuscript, read approved final version manuscript. <br /> to experimental scattering data. The rigid body cal- <br />                                                                                 Received: 17 June 2011 Accepted: 19 October 2011 <br /> culations implemented program BUNCH                                 Published: 19 October 2011 <br /> [48] using q-range. In order construct the <br /> model mutants Sis1, used high resolu-                          References <br /> tion structure Sis1 C-terminal peptide-binding                           1. Fink AL: Chaperone-mediated protein folding. Physiol Rev 1999, <br />                                                                                     79:425-449. <br /> domain Sis1 J-domain, Pro-                            2. Hartl FU: Molecular chaperones cellular protein folding. Nature 1996, <br /> tein Data Bank http://www.rcsb.org identified                                381:571-580. <br /> codes 1C3G 2O37 respectively. To compose                                3. Hartl FU, Hayer-Hartl M: Protein folding - Molecular chaperones the <br />                                                                                     cytosol: nascent chain folded protein. Science 2002, <br /> model Ydj1 deleted domain mutant, used                               295:1852-1858. <br /> high resolution structure Ydj1 dimerization                              4. Bukau B, Horwich AL: The Hsp70 Hsp60 chaperone machines. Cell <br /> domain (PDB code 1XAO) J-domain                                   1998, 92:351-366. <br />                                                                                 5. Summers DW, Douglas PM, Ramos CHI, Cyr DM: Polypeptide transfer from <br /> the Sis1 structure (PDB code 2O37). Several indepen-                                Hsp40 Hsp70 molecular chaperones. Trends Biochem Sci 2009, <br /> dent calculations performed protein. We                               34:230-233. <br /> applied 2-point symmetry constraint model cal-                         6. Langer T, Lu C, Echols H, Flanagan J, Hayer MK, Hartl FU: Successive Action <br />                                                                                     Dnak, Dnaj Groel Along Pathway Chaperone-Mediated <br /> culations, using priori knowledge                            Protein Folding. Nature 1992, 356:683-689. <br /> were dimers solution. The NSD values evaluated                          7. Schmid D, Baici A, Gehring H, Christen P: Kinetics Molecular Chaperone <br /> in order select typical model pro-                             Action. Science 1994, 263:971-973. <br />                                                                                 8. Misselwitz B, Staeck O, Rapoport TA: J proteins catalytically activate hsp70 <br /> tein. The fits models (Figure S1) corre-                            molecules trap wide range peptide sequences. Mol Cell 1998, <br /> sponding chi values (Table S1)                                      2:593-603. <br /> additional files 1 2 respectively.                                          9. Laufen T, Mayer MP, Beisel C, Klostermeier D, Mogk A, Reinstein J, et al: <br />                                                                                     Mechanism regulation Hsp70 chaperones DnaJ cochaperones. <br />                                                                                     Proc Natl Acad Sci USA 1999, 96:5452-5457. <br /> Additional material                                                             10. Qian XG, Hou WB, Li ZG, Sha BD: Direct interactions molecular <br />                                                                                     chaperones heat-shock protein (Hsp) 70 Hsp40: yeast Hsp70 Ssa1 <br />                                                                                     binds extreme C-terminal region yeast Hsp40 Sis1. Biochem J <br />     Additional file 1: Figure S1. Model fits proteins studied in <br />                                                                                     2002, 361:27-34. <br />     work. <br />                                                                                 11. Cyr DM, Langer T, Douglas MG: Dnaj-Like Proteins - Molecular <br />     Additional file 2: Table S2. Chi-values (<span id='am-29' about='obo:NCBITaxon_1' typeof='owl:Thing'><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="root" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="all" datatype="xsd:string"></span>root</span> mean square c2)                Chaperones Specific Regulators Hsp70. Trends Biochem Sci 1994, <br />     obtained fit theoretical scattering calculated         19:176-181. <br />     models experimental intensity curve.                                 12. Wall D, Zylicz M, Georgopoulos C: The Nh2-Terminal 108 Amino-Acids of <br />                                                                                     Escherichia-Coli Dnaj Protein Stimulate Atpase Activity Dnak <br />                                                                                     Are Sufficient Lambda-Replication. J Biol Chem 1994, <br />                                                                                     269:5446-5451. <br /> Acknowledgements Funding                                                    13. Luke MM, Sutton A, Arndt KT: Characterization Sis1, A <span id='am-6' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span>Saccharomyces</span>- <br /> This research supported Brazilian Synchrotron Light Laboratory           Cerevisiae Homolog Bacterial Dnaj Proteins. J Cell Biol 1991, <br /> (LNLS)/Ministry Science Technology (MCT) State S&#227;o Paulo          114:623-638. <br /> Research Foundation (FAPESP). Research laboratory CHIR             14. Caplan AJ, Douglas MG: Characterization Ydj1 - A Yeast Homolog of <br /> supported grants FAPESP, MCT/National Research Council                  Bacterial Dnaj Protein. J Cell Biol 1991, 114:609-621. <br /> (MCT/CNPq), NIH-R03TW007437 funded Fogarty International             15. Fan CY, Lee S, Ren HY, Cyr DM: Exchangeable chaperone modules <br /> Center (FIRCA-NIH). J.C. Silva thanks FAPESP CNPq financial support         contribute specification type I type II Hsp40 cellular function. <br /> and FIRCA-NIH paying travelling expenses training. J.C. Borges          Mol Biol Cell 2004, 15:761-773. <br /> thanks FIRCA-NIH fellowship, FAPESP (2007/05001-4) financial support    16. Lu Z, Cyr DM: Protein folding activity Hsp70 modified differentially <br /> and CNPq Research Fellowship. ILT CNPq (1B) Research Fellow,           Hsp40 co-chaperones Sis1 Ydj1. J Biol Chem 1998, <br /> CHIR CNPq (1C) Research Fellow. DMC supported NIH RO1-                      273:27824-27830. <br /> GM067785.                                                                       17. Banecki B, Liberek K, Wall D, Wawrzynow A, Georgopoulos C, Bertoli E, et al: <br />                                                                                     Structure-function analysis zinc finger region DnaJ <br /> Author details                                                                      molecular chaperone. J Biol Chem 1996, 271:14840-14848. <br /> 1 <br />  Department Condensed Matter Physics, &#8220;Gleb Wataghin&#65533;? Physics Institute,     18. Borges JC, Fischer H, Craievich AF, Ramos CHI: Low resolution structural <br /> State University Campinas (UNICAMP), Campinas, SP 13083-859, Brazil.             study human HSP40 chaperones solution - DjA1 from <br /> 2 <br />  Brazilian Biosciences National Laboratory, Center Research Energy       <span id='am-16' about='obo:NCBITaxon_subfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-17' property="rdfs:label" content="subfamily" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>subfamily</span> A DjB4 subfamily B different quaternary <br /> Materials (CNPEM), Campinas, SP 13083-970, Brazil. 3Institute Chemistry       structures. J Biol Chem 2005, 280:13671-13681. <br /> S&#227;o Carlos, University S&#227;o Paulo, S&#227;o Carlos, SP 13.560-970, Brazil.         19. Ramos CHI, Oliveira CLP, Fan CY, Torriani IL, Cyr DM: Conserved Central <br /> 4 <br />  Department Cell Developmental Biology, University North                  Domains Control Quaternary Structure Type I Type II Hsp40 <br /> Carolina, Chapel Hill, NC 27599, USA. 5Department Organic Chemistry,             Molecular Chaperones. J Mol Biol 2008, 383:155-166. <br /> Institute Chemistry, University Campinas UNICAMP, SP 13083-970, Brazil.   20. Sha BD, Lee S, Cyr DM: The crystal structure peptide-binding <br /> 6 <br />  European Synchrotron Radiation Facility, Grenoble, France.                         fragment yeast Hsp40 protein Sis1. Structure 2000, 8:799-807. <br />                                                                                 21. Lee S, Fan CY, Younger JM, Ren HY, Cyr DM: Identification essential <br /> Authors&#8217; contributions                                                              residues Type II Hsp40 Sis1 function polypept <br /> </body></html>