<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>     1995 Oxford University Press                                          Nucleic Acids Research, 1995, Vol. 23, No. 17 3493-3500 <br />  <br />  <br /> HYS2, essential gene required DNA replication <br /> in <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br /> Katsunori Sugimoto*, Yoshitaka Sakamoto, Osamu Takahashi Kunihiro <br /> Matsumoto <br /> Department Molecular Biology, Faculty Science, Nagoya University, Chikusa-ku, Nagoya 464-01, Japan <br /> Received May 12, 1995; Revised Accepted August 8, 1995                                                    DDBJ accession no. D50324 <br />  <br />  <br />  <br /> ABSTRACT                                                                mechanism prevents mitosis completion DNA <br />                                                                         replication established (reviewed refs 1,2,7,8). <br /> To investigate cell cycle regulation S G2 phase                  DNA replication fundamental maintenance growth <br /> in <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19'  ></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae, isolated mu-                       eukaryotic cells. Saccharomyces cerevisiae provides an <br /> tants displaying supersensitivity hydroxyurea (HU),                  excellent model identifying components the <br /> a chemical inhibits DNA replication. Such mu-                      eukaryotic DNA replication machinery distinctive <br /> tants, named hydroxyurea sensitive                        growth pattern budding yeast allows cell-division-cycle <br /> (hys), defined linkage groups character-                    (Cdc) phenotypes easily distinguished defects other <br /> ized hys2 mutation study. The hys2-1 mutant                 cellular processes (9). For example, studies S.cerevisiae cdc <br /> displays temperature sensitive growth constella-                  mutants resulted identification DNA ligase (CDC9) <br /> tion phenotypes indicating defective DNA metab-                      (10,11), DNA polymerase (CDCJ7) (12,13) DNA <br /> olism. At restrictive temperature, hys2-1 cells                     polymerase 6 (CDC2) (14,15). Thus, determination the <br /> arrest large budded cells single nucleus               nature defect cdc mutants led better <br /> neck bud short spindle. The hys2-1 mutant                  understanding DNA replication mitosis. <br /> exhibits increased rates chromosome loss                            Enoch et al. (16) developed screen fission yeast <br /> recombination. Additionally, hys2-1 appears accu-                    mutants defective coupling mitosis completion DNA <br /> mulate incompletely replicated DNA                          replication. These mutants isolated hydroxyurea <br /> detected pulse field electrophoresis assay. Finally,               (HU) sensitive mutants initiate mitosis DNA repli- <br /> deletion RAD9 hys2-1 strain decreases                       cation blocked HU. Eight different genes (radi, rad3, radl 7, <br /> percentage arrested cells, suggesting intact                 husi, hus2, hus3, hus4 hus5) identified, are <br /> RAD9-checkpoint required cell cycle arrest                required arrest mitosis response inhibition DNA <br /> hys2-1 cells. HYS2 encodes 55 kDa protein                     replication DNA damage. We searched mutants that <br /> essential viability temperatures. Taken to-                  supersensitive HU S.cerevisiae identify genes playing <br /> gether, <span id='am-36' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-37'  ></span><span id='am-38' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> suggest Hys2 plays role                    role DNA replication itself checkpoint responds <br /> DNA replication.                                                        inhibition DNA replication. We isolated different <br />                                                                         hys mutations (hysi, hys2, hys3 hys4). <br /> INTRODUCTION                                                               We report characterization HYS2. The hys2 mutant <br /> The cell division cycle eukaryotic cells requires high degree   HU sensitive simultaneously confers temperature sensitive <br /> coordinated control. The ability <span id='am-33' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-34' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> pathways     growth. The hys2 mutation causes cells arrest single <br /> govern carry steps cell division, including DNA              large bud. The HYS2 gene essential growth encodes <br /> replication mitosis, fundamental importance       55 kDa protein. The hys2 mutant exhibits increased levels of <br /> eukaryotes. Incomplete DNA replication DNA damage prevents           mitotic chromosome loss recombination, suggesting that <br /> the subsequent mitosis (1,2). In fission yeast, dependency      Hys2 plays role DNA replication. <br /> checkpoint control shown involve p34Cdc2 kinase. <br /> The signal generated active replication ultimately impinges       MATERIALS AND METHODS <br /> the phosphorylation state Tyrl5 p34Cdc2 kinase. <span id='am-28' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-31'  ></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> signal       Strains, media general methods <br /> transduction pathway required prevent mitosis events in <br /> S phase G2 perturbed (3,4). In budding yeast, tyrosine          Saccharomyces cerevisiae strains used study listed in <br /> phosphorylation p34Cdc28 kinase required arrest mitosis    Table 1. Standard genetic techniques yeast culture media <br /> under circumstances (5,6). Thus,         described previously (17,18). Media used maintain <br /> another way arresting cell cycle tyrosine phospho-       selection TRPI URA3 <span id='am-41' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-42' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> synthetic complete <br /> rylation p34Cdc28. A number genes involved               media containing 0.5% casamino acid appropriate <br /> checkpoint control yeasts identified,    supplements. <br />  <br />  * <br />      To correspondence addressed <br />   3494 Nucleic Acids Research, 1995, Vol. 23, No. 17 <br />  <br /> Table 1. List strains used study                              gene. After sporulation tetrad dissection ofdiploid, viability <br />                                                                          spores assessed. Only half spores viable, and <br />  Strain           Genotype                                               expected, HU temperatr sensitive, well <br />  KSC688          MATa ade2 ural leu2 lys2                                Leu-. <br />  KSH106          MATa adel his2 trpl ura3 leu2 <br />                                                                          Plasmid construction <br />  KSH141          MATa ade2 his3 trpl ura3 <br />  KSH163          MATa adel his3 trpl ura3 leu2                           The Sall-HindIl fragment p12 subcloned YCplac22 <br />                                                                          (22) YCplac33 (22) creating YCp22-HYS2 and <br />  KSH171          MATa adel his2 trpl ura3 leu2                           YCp33-HYS2, respectively. This fragment cloned into <br />  KSH172          Isogenic KSH106 MATa/MATa                 pSP72 (Promega) cleaved Sall HindHI creating <br />  KSH387          MATa hys3-1 adel his2 trpi ura3 leu2                    pSP-63HS. The nucleotide sequence Sail HindIH <br />  KSH389          MATa hys4-1 adel his2 trpl ura3 leu2 <br />                                                                          fragment determined strands. For construction of <br />                                                                          YCpG-HYS2, 1.5 kb fragment containing HYS2 coding <br />  KSH404          MATa hysl-J adel his3 trpl ura3 leu2                    sequence obtained polymerase chain reaction using the <br />  KSH541          MATa hys2-1 adel his2 trpl ura3 leu2                    primer GCTCTAGAGGCAATGGACGCATTGT, corresponding <br />  KSH542          MATa hys2-1 adel his2 his3 trpl ura3 leu2 <br />                                                                          nucleotide residues -4 +13 HYS2 gene primer <br />                                                                          GCGTCGACTATTTAAATGTCAATCT, corresponding nu- <br />  KSH543          MATa ade2 his3 trpl ura3                                cleotide residues +1451 +1567 HYS2 gene (DDBJ <br />                  MATa adel his2 trpl ura3 leu2                           accession no. D50324). XbaI Sall sites included the <br />  KSH544          MA Ta hvs2-1 ade2 his3 trpl ura3                        oligonucleotides underlined. The fragment cleaved with <br />                  MATa hys2-1 adel his2 trpi ura3 leu2                    XbaI Sall cloned YCpG33 (17), YCplac33 <br />  KSH687          MATa hys2-1 rad9A::LEU2 adel his2 his3 trpl ura3 leu2 <br />                                                                          derivative containing GALl promoter, creating YCpG-HYS2. <br />                                                                          The plasmid YCpG-HYS2 complemented disruption HYS2, <br />  KSH689          MATa hys2-1 ade2 his3 trpl ura3                         cells grown YEPD medium. <br />  KSH700          Isogenic KSH172 HYS2/hys2A::LEU2 <br />                                                                          Disruption HYS2 <br />   DNA manipulated standard procedures (19). Sequence              For construction hys2A::LEU2, plasmid pDhu2L was <br /> analysis performed dideoxy chain termination tech-            created replacing internal BamHI fragment HYS2 with <br /> nique using Sequenase (USB) according manufacturer&apos;s              LEU2 gene. The XbaI-PvuII fragment containing <br /> instruction. Yeast tranformation performed alkali             hys2A::LEU2 integrated diploid strain KSH172, <br /> cation method (20). One step gene replacement described           creating strain KSH700. After sporulation, tetrad analysis was <br /> by Rothstein (21).                                                       used determine phenotype caused deletion. <br />  <br /> Mutant isolation                                                         UV radiation DNA damaging agent sensitivities <br /> A wild type strain (KSH106) mutagenized 2% ethyl                UV radiation 254 nm delivered use Stratagene <br /> methanesulfonate (EMS) 10% survival. Mutagenized cells           Stratalinker. Cells grown exponentially 30&#176;C plated on <br /> grown single colonies YEPD medium 30&#176;C. Colonies                YEPD irradiated UV according manufacture&apos;s <br /> were replica plated YEPD YEPD containing 10 mg/ml HU.             manual. After incubation 30&#176;C 3 days, number of <br /> After 3 4 day incubation 30&#176;C, 19 colonies grew poorly        colonies counted. <br /> in presence HU. Among mutants accumulated                  Methyl methanesulfonate (MMS) sensitivity assay de- <br /> &gt;80% large budded cells 4 h shift HU containing           scribed previously (23). Cells grown exponentially YEPD <br /> medium.                                                                  medium 30&#176;C incubated 0-0.8% MMS 40 <br />                                                                          min. The incubation terminated adding sodium thiosulfate <br /> Cloning HYS2 gene                                                 final concentration 5%. Aliquots plated YEPD, <br />                                                                          followed incubation 30&#176;C. After 3 days incubation, the <br /> Strain KSH542 (hys2-1 ura3) transformed genomic               colony number counted. <br /> library constructed YCp5O 30&#176;C replica plated on <br /> YEPD medium containing 10 mg/ml HU. A single plasmid (p12)               Viability assays <br /> was identified complement HU temperature <br /> sensitive growth hys2-1 mutation.                                 The method Weinert Hartwell (24) used determine <br />   To establish complementing plasmid p12 carries            hys2-1 rad9A::LEU2 mutant (KSH687) loses <br /> HYS2 gene, crossed hys2-1 mutant hys2A. :LEU2            viability incubated restrictive temperature. Cells -were <br /> (see below) strain carrying plasmid p12. The plasmid p12             grown YEPD medium 250Cto mid log phase were <br /> cured resulting diploid. Because diploid heterozy-      shifted restrictive temperature (37&#176;C). Aliquots the <br /> gous hys2-1 mutation, disruption wild type gene         culture removed intervals, sonicated plated YEPD <br /> should unmask HU sensitive temperature sensitive                 plates. After incubation 25&#176;C 2 days, cell viability was <br /> phenotypes cloned fragment exclusively HYS2 coding             determined microscopy; individual microcolonies plate <br /> sequence. The diploid HU temperature sensitive, confrm-          scored inviable (&lt;16 cell bodies) viable (&gt;16 cell <br /> ing mutation resides gene disrupted LEU2      bodies). <br />                                                                                   Nucleic Acids Research, 1995, Vol. 23, No. 17 3495 <br />  <br /> Immunofluorescence microscopic analysis                                blot analysis sensitive probes derived large <br />                                                                        chromosomes used (28). <br /> Cells processed fluorescence indirect immunofluores- <br /> cence microscopy described previously (25). Cells fixed        Cell synchronization RNA analysis <br /> and stained DNA 25 jg/ml 4,6-diamidino-2-phenylindole <br /> (DAPI). Microtubule structures observed following for-            Synchronization a-factor mating pheromone Northern <br /> maldehyde fixation using anti-tubulin monoclonal antibody          blot analysis performed described previously (29). DNA <br /> TAT-1 FITC-conjugated goat anti-mouse antibody                probes were: CLNJ2, 0.9 kb XhoI-HindIII fragment the <br /> described (26).                                                        CLN2 gene (30); ACT], 1 kb XhoI-HindHI fragment derived <br />                                                                        pYS91 containing ACT1 cDNA (unpublished); HYS2, <br />                                                                        1 kb BamHI fragment YCp33-HYS2. <br /> DNA flow cytometry analysis <br /> Flow cytometry DNA quantitation determined described            RESULTS <br /> previously (26). Cells (1 x 106-1 x 107) collected, washed <br /> once 1 ml water resuspended 0.3 ml 0.2 M Tris-HCl    Isolation hys mutants <br /> (pH 7.5). Ethanol added final concentration 70%    The culture wild type cells presence HU (10 mg/ml) <br /> vigorous agitation cells stored -20&#176;C overnight. After     temporarily accumulates cells large buds eventually <br /> resuspension 0.3 ml 0.2 M Tris-HCl (pH 7.5), cells          recovers HU arrest. We mutagenized wild type cells <br /> sonicated briefly preboiled RNase A (Sigma) added         (strain KSH106) ethyl methanesulfonate (EMS) screened <br /> final concentration 1 mg/ml. Following 3 h incubation 37&#176;C,    colonies sensitive HU replica-plating cells onto <br /> propidium iodine (Sigma) added (final concentration, 5 jig/ml)     plates HU identifying clones failed form <br /> and resulting stained cell suspensions analyzed using       colonies HU. Because HU blocks DNA synthesis, mutations <br /> Becton-Dickinson FACScan.                                              defective DNA metabolism expected suffer severe <br />                                                                        growth retardation presence HU. Also incubated <br /> Analysis chromosome IH missegregation                               HU, mutations deficient ability monitor impaired <br />                                                                        DNA synthesis allow cells enter mitosis with <br /> Quantitative measurement chromosome III loss recom-             defective chromosomes, resulting lethality. Of -10 000 EMS- <br /> bination performed essentially previously described (27).      mutagenized cells screened, 19 clones showed HU sensitive <br /> Strains tested allowed grow 30&#176;C YEPD              phenotype. Among HU sensitive mutants, that <br /> medium. Mating initiated mixing -1 x 106 cells            accumulated 80% large budded cells 4 h shift HU <br /> tested 1 x 106 MATa haploid tester (KSC688) final            media identified microscopic observation. In crosses a <br /> volume 200 gl. After 4 h 30&#176;C mating mixture plated      wild type strain, mutant recessive HU sensitivity and <br /> on medium selecting prototrophic triploids. Wild type MATat        segregated single gene mutation. Complementation analysis <br /> haploid KSH171 used control determine mating               showed mutants formed different complementation groups, <br /> efficiency. To ensure rate MATa homozygosis             calling hysl-hys4 (Table 2). In paper, present <br /> chromosome Ill missegregation underestimated          data characterizing hys2 mutation. <br /> mating defect associated hys2-1, hys2-1 haploid strain             The HU-teated cultures hys2-1 mutant subjected to <br /> KSH541 tested. This revealed mating deficiency             cytological examination. Cells stained DNA specific <br /> associated hys2-1 30&#176;C. The combined rate MATa              fluorescence dye DAPI analysis cell nuclear morphol- <br /> homozygosis plus chromosome III loss determined           ogy. More 90% large budded hys2-1 cells, like wild type <br /> total number cells mated. The rate chromosome mI loss       cells, contained single nucleus near neck short <br /> was calculated subtracting rate mitotic recombination        spindle (data shown). In addition HU sensitive phenotype, <br /> determined analysis Leu+ maters combined rate.          hys2-1 mutation cells conferred sensitivity MMS and <br /> The assay performed twice using independent colonies.              temperature sensitive growth defect (Table 2). To test hys2 <br />                                                                        growth defect 37&#176;C HU sensitivity caused single <br />                                                                        mutation, analyzed meiotic products diploid strain <br /> Pulse field gel analysis                                               heterozygous hys2-1. Temperature sensitive growth phenotype <br />                                                                        cosegregated HU sensitive phenotype tetrads, <br /> Cultures grown early log phase 25&#176;C                indicating hys2-1 mutation determines phenotypes. <br /> incubated 37&#176;C 4 h. For control samples, cultures hys2-1 <br /> cells incubated a-factor (10 ,ug/ml) 2.5 h, HU (10       The hys2-1 mutation defective DNA replication <br /> mg/ml) 4 h nocodazole (20 jg/ml) 4 h 25&#176;C. Yeast <br /> chromosomal DNA samples prepared previously de-                To examine tenninal morphology strains carrying the <br /> scribed (28). Pulse field gel electrophoresis carried 1 %   hys2-1 mutation, growth-arrested populations analyzed <br /> agarose (LE, FMC) pulsaphor electrophoresis unit (Pharma-         microscopically (Fig. 1). When exponentially growing cultures of <br /> cia). Electrophoresis performed 24 h switching          hys2-1 mutant HYS2+ cells grown 25&#176;C shifted to <br /> time of70 s 170 V 0.5 x Tris-borate-EDTA (TBE). The DNA          37&#176;C 4 h, 80% hys2-1 cells arrested large buds, <br /> was transferred nylon membranes probed                 compared 19% HYS2+. Thus, hys2-1 exhibits a <br /> 32P-labeled TRPJ fragment (the 1.4 kb EcoRI YRp7                  marked elevation proportion large budded cells the <br /> plasmid). This probe chromosome IV chosen          restrictive temperature. <br />   3496 Nucleic Acids Research, 1995, Vol. 23, No. 17 <br />  <br />              Table 2. Properties hys mutants <br />  <br />               Genotyp&amp;&apos;              Growthb                                                MMS sensitivityc    UV sensitivityc <br />                                      300C              370C                 10 mg/ml HU     0.4%                90 J/m2                120 J/m2 <br />               HYS                    +                 +                   +                16%                 30%                   7% <br />               hysl                   +                 +                   -               0.09%                0.2%                  NT <br />               hys2                   +                 -                   -               0.06%                17%                   5% <br />               hys3                   +                 +                   -               NT                   NT                    NT <br />               hys4                   +                 +                   -               NT                   NT                    NT <br />              aStrains used KSH106 (HYS), KSH404 (hysl), KSH542 (hys2), KSH387 (hys3) KSH389 (hys4). <br />             bGrowth cells 30 37&#176;C tested streaking cells YEPD plates incubating corresponding temperature 2 days. <br />             Cells tested growth YEPD plates containing 10 mg/ml HU 300C 4 days. +, growth; -, growth. <br />             CDNA damage sensitivities determined treating log-phase cultures cells 0.4% MMS 40 min irradiating 90 or <br />             120 J/m2 UV light (254 nm) percent survival measured 3 days incubation 30&#176;C. NT, tested. <br />  <br />                                                                                      Mutations genes involved DNA replication, such <br /> a                                                                  0               encoding DNA polymerase (CDCJ 7), DNA polymer- <br />                      0                                                             ase 6 (CDC2) DNA ligase (CDC9), exhibit significantly <br />                                                                                    elevated frequencies genetic recombination chromo- <br />                      IC,) (a....1. <br />                   -,IV             .:~                                             loss (32). This feature distinguishes mitotic <br />                   fd3 <br />                    (&apos;)4-&quot;                                   .7 <br />                                                                                    defects elevated chromosome loss. We assayed the <br />                                                                                    recombination chromosome loss frequencies using standard <br />                                                                                    heterozygous markers chromosome III (see Materials and <br />                                                                                    Methods) (Table 3). Diploids homozygous hys2-1 <br />                                                                                    mutation grown permissive temperature. Frequencies <br />                                                                                    recombination chromosome loss normalized a <br />                                                                                    HYS2+1HYS2+ strain. The hys2-J/hys2-1 strain exhibited signifi- <br />                                                                                    increases frequency recombination (16-fold <br />                                                                                    increase) chromosome loss (20-fold increase) the <br />                                                                                    permissive temperature (Table 3). <br />  <br />                                                                                    Table 3. Rates recombination chromosome III missegregation the <br /> b                                                                                  hys2-1 mutanta <br />        DAPI <br />                                                                                     Genotype           Recombination rate         Chromosome missegregation rate <br />       TubLI lii                                                                     HYS2/HYS2          8.6 x 10-5 (1)             9.8 x 10-5 (1) <br />                                                                                     hys2-1/hys2-1      1.4 x 10-3 (16)            2.0 x 10-3 (20) <br />                                                                                    aThe rates mating competent diploids arose mitotic recombination <br /> Figure 1. Cell cycle arrest phenotype conferred hys2-1 mutation. (a)        versus chromosome III missegregation HYS2/HYS2 (KSH543) and <br /> DAPI stained hys2-1 cells. A logarithmically growing culture KSH542             hys2-1/hys2-1 (KSH544) diploids analyzed 30&#176;C. Mating competent di- <br /> (hys2-1) shifted 25 37&#176;C 4 h. Cells fixed ethanol     ploids scored protrophic triploids incubation leu2 mating <br /> examined phase contrast microscopy (upper panel) DAPI staining (lower       tester (KSC688). The fraction diploids mated MATa ho- <br /> panel). (b) Nuclear microtubule structure hys2-1 cells. Cells KSH542 <br /> (hys2-1) grown logarytmiically 250C shifted 370C 4 h.      mozygosis mitotic recombination distinguished had <br /> Samples fixed formaldehyde stained DAPI (upper panel)         missegregated chromosome III, replica plating plates lacking leucine. Rates <br /> anti-tubulin antibodies (lower panel) visualize nuclei spindles.            calculated numbers described Materials Methods. <br />                                                                                    Numbers parentheses normalized wild type (HYS2/HYS2) back- <br />                                                                                    ground. <br />  <br />   After 4 h incubation 37&#176;C, 83% large budded cells                     To determine extent DNA replication hys2-1 <br /> the hys2-1 culture single undivided nucleus                mutant cells undergo arrest, examined DNA <br /> neck mother daughter cells (Fig. la). Following                    content hys2-1 cells restrictive temperature. We <br /> a shift 370C, samples hys2-1 cells prepared                         incubated exponentially growing hys2-1 HYS2+ strains a <br /> anti-tubulin immunofluorescence microscopy (Fig. lb). The                          permissive (25&#176;C) restrictive temperature (370C), stained the <br /> hys2-1 cells short mitotic spindle, consisting brightly                 cells propidium iodide determined DNA content of <br /> stained bar nuclear microtubules faintly staining                     cells FACS analysis. A high percentage hys2-1 cells <br /> cytoplasmic microtubules. Thus, phenotype hys2-1 mu-                        arrest 2N content DNA 37&#176;C, indicating the <br /> tants restrictive temperature similar observed                majority DNA synthesis completed restrictive <br /> cdc mutants defects DNA replication mitosis (31).                       temperature (Fig. 2). The informative result FACS <br />                                                                                                 Nucleic Acids Research, 1995, Vol. 23, No. 17 3497 <br />  <br />  <br />                                                                              hys2                         hys2             rad9 <br />                                          IL <br />                     4hr <br />                                                                                                                                     see <br />  <br />  <br />  <br />  <br />                                                     rA Pr-L <br />                                                           - , A-,M <br />                                                               ,      , <br />                                                     600     800 <br />  <br />  <br />  <br />  <br />                           0 <br />  <br />  <br /> Figure 2. DNA flow cytometry analysis hys2-1 hys2-1 rad9 mutant cells. Strains KSH106 (HYS2), KSH542 (hys2-1) KSH687 (hys2-1 rad9A::LEU2) were <br /> grown  early log phase YEPD 25&#176;C. After shifting 37&#176;C, samples cells taken 2 h intervals fixed, stained propidium iodine examined <br /> by flow cytometry. Control experiment hys2-1 cells treating 20 tg/ml nocodazole 25&#176;C 4 h subsequently shifting 37&#176;C 4 h. <br /> This experiment confirned hys2-1 cells arrest 2N content DNA 370C. <br />  <br />  <br />  <br /> analysis hys2-1 mutation does disrupt bulk of <br /> DNA replication restrictive temperature. The hys2-1 <br /> temperature sensitive mutation leaky leaky DNA                                                     00 <br /> synthesis mutants (e.g. cdc17) (33) arrest G2/M <br /> content DNA. It noted culture hys2-1 mutant <br />                                                                                                        e m C5 X Z <br /> accumulated cells late S G2/M phase DNA content at <br /> 25&#176;C. The hys2 mutant appears defect S-G2/M phase                                          __         _j&amp;Om      -   well <br /> execution permissive temperature.                                                                                          chromosome IV <br />  <br />    Although bulk DNA replicated hys2-1 mutant, <br /> there small amounts unreplicated DNA sufflcient to <br /> cause arrest cell cycle. To determine DNA is <br /> completely replicated hys2-1 cells, chromosomes                           Figure 3. Chromosome abnormalities hys2-1 mutant. Exponentially <br /> arrested cells analyzed pulse field gel electrophoresis                     growing cultures hys2-1 (KSH542) HYS2 (KSH106) cells 25&#176;C were <br /> (Fig. 3). In assay fully replicated DNA enters gel;                   shifted 37&#176;C incubated 4 h YEPD. As controls, hys2-1 cells were <br /> incompletely replicated DNA (e.g. replication loops single-                      incubated 25&#176;C YEPD containing 10 jg/ml a-factor 2.5 h, 10 mg/ml <br />                                                                                     HU 4 h 20 jg/ml nocodazole 4 h. The chromosomes sample <br /> stranded regions) causes DNA retarded wells (28). DNA                      separated pulse field gel electrophoresis, blotted probed a <br /> was prepared hys2-1 strain incubated                  DNA fragment ofthe TRPI gene resides chromosome IV. The row <br /> restrictive temperature 4 h. As controls, hys2-1 strain                 bands indicates samples loaded (well) contains residual <br /> treated a-factor mating pheromone (blocked GI phase),                       material able migrate gel. <br /> HU (blocked S phase) nocodazole (blocked M phase) at <br /> the permissive temperature DNA isolation. As shown <br /> previously (28), DNA isolated HU-treated culture                          Therefore, likely explanation phenotype the <br /> mostly remained wells electrophoresis,                    hys2-1 mutant defective DNA replication, but <br /> majority DNA mating pheromone nocodazole                            somewhat leaky, DNA replication, complete, <br /> treated cultures migrated gel. Compared DNA                          result accumulation DNA level approaching 2N. <br /> isolated wild type cells, DNA prepared hys2-1 cells                    If hys2-1 mutants undergo cell cycle arrest DNA <br /> at restrictive temperature entered gel greatly reduced                lesions, arrest mediated product RAD9 <br /> efficiency (Fig. 3). Thus, appears DNA incom-                      gene (24,34). To test role RAD9 checkpoint cell cycle <br /> pletely replicated hys2-1 cells restrictive temperature.                 arrest hys2-1 mutant, constructed hys2-1 rad9&amp;:^LEU2 <br />   3498 Nucleic Acids Research, 1995, Vol. 23, No. 17 <br />  <br />                                         b                                         sponding HYS2 gene. This sequence predicts protein 487 <br />                                                                                     amino acids, assuming tmnslation begins ATG. A <br />                                                                                     search sequence banks failed idenfify any <br />                                                                                    known proteins significant sequence similarity Hys2. The <br />                                                                                    nucleotide sequence 3&apos;-region HYS2 gene identical <br />                                                                                    upstream sequence SUI2 gene (36), indicating that <br />                                                                                    genes reside adjacent other. This finding consistent <br />                                                                                    result chromosom blotting ofHYS2, SUI2 <br />                                                                                    gene located near centromere chromosome X. Any genes <br />                                            &quot;-4                                     genetically mapped close SUI2, correspond <br />  ,- <br />                                         a2                                      HYS2 gene. <br />                                           u <br />                                                                                       To determine HYS2 essential viability S.cerevisiae, we <br />                                                                                    created disruption HYS2 open reading frame. A LEU2 <br />                                                                                    fragment replaced 1 kb BamHI fiagment inmtemal the <br />                                                                                    HYS2 coding region XbaI-PvuH fragment containing the <br />                                                                                    LEU2 insert used transform diploid strain KSH172. <br />                   2        4                                                       Southern blot analysis Leut transfomiants showed of <br />                                                             2         4 <br />                   Time (hr)                                 Time (hr)              copies ofthe HYS2 gene disrupted. When diploid <br />                                                                                    (KSH700) sporulated, tetrads segregated 2:2 viable:non-vi- <br />                                                                                    able. All viable spores Leun, indicating the <br />  Figure 4. Viability hys2-1 rad9 double mutant restrictive           wild type copy HYS2 gene. This result consistent the <br />  temperature. Strains KSH542 (hys2-1), KSH687 (hys2-J rad9As:LEU2)           HYS2 gene product essential growth. Microscopic <br />  wild type strain KSH106 grown exponentially 25&#176;C shifted         examiaton inviable spores indicated genninated <br />  37&#176;C. Cell viability determined shifting described Materials <br />  Methods percent viability intervals shown (a). Samples     form microcolonies ftree 11 cell bodies. The <br />  analyzed cell number (b). Symbols: hys2-1(0), hys2-1 rad9 &amp;A:LEU2        HYS21hys2&amp;:LEU2 diploid (KSH700) transformed a <br />  (U), wild type (0).                                                               plasmid YCp33-HYS2 bearing HYS2 URA3 genes after <br />                                                                                    sporulation, tetrad analysis caried In ascus, to <br />                                                                                    spores viable Leu+ spore clone canying the <br />                                                                                    hys2&amp;:LEU2 mutation Urat. Even cultring Leut <br /> double mutant strain. Cultures growig exponentially 25&#176;C                        Ura+ spore clones generations non-selective YEPD <br /> were shifted 37&#176;C cell cycle properties examined.                  medium, Leut Urar segregants selected. Thus, HYS2 <br /> Whereas hys2-1 RAD9+ strain exhibited G2/M arrest                         gene appears essential cell growth. <br /> incubated 370C 4 h, introduction rad9&amp;:LEU2                             The nucleotide sequence HYS2 5&apos;-noncoding region <br /> mutation hys2-1 reduced level arrest G2/M (Fig. 2).                   revealed single MIul site located 107 bp upstam the <br /> Thus, rad9 mutation appears largely alleviate cell cycle                tranlational initiation site HYS2. The sequence &apos;ACGCGT&apos; or <br /> arrest hys2-1 mutant. If failure arrest caused            closely related sequence, called MluI cell cycle box (MCB) <br /> removal checkpoint, expect observe                             element, required coordinate expression group of <br /> concomitant loss viability dividing double mutant strain                 DNA replication genes including CDCJ7 (DNA polymerase a), <br /> (24,35). Indeed, viability cells examined                   CDC2 (DNA polymerase 8), POL2 (DNA polymerase e), CDC9 <br /> same experiment, hys2-1 rad9&amp; :LEU2 strains exhibited rapid                      (DNA ligase) CDC8 (thymidylate kinase), beginning late in <br /> loss viability failure cease cell division 37WC (Fig.               Gi extending S phase (reviewed ref. 37). We analyzed <br /> 4). We conclude cell cycle arrest hys2-1 shift                HYS2 gene tanscript Northern blotting using methods <br /> the restrictive temperature requires intact RAD9 gene.                          described Materials Methods. We identified single ranscript <br />                                                                                    -1.7 kb exponentally growing cells (data shown). To <br />                                                                                    detemine whetier expression ofHYS2 coordinately reulated <br /> Cloning    HYS2 gene                                                        tughout cell cycle, HYS2 mRNA levels monitred a <br /> We cloned HYS2 gene complementation HU                               culture synchronized G1 phase tment a-factor mating <br /> sensitive phenotpe hys2-1 mutant. After transforming                        pheromone, released arrest incubation fresh medium <br /> genomic library constructed YCp5O strain KSH542                            sampled intervals 20 mi. As shown Figure 5, <br /> (hys2-1 ura3), screened tasformants able grow                         accumulation ofHYS2 mRNA oscillated peaking arwnd time of <br /> YEPD containing HU. A single plasmid-linked transfonnant                       bud egence, corresponds late Gl early S. However, <br /> identified transfonnant contained plasmid (p12)                   cell cycle regulation modest (activated 2- 3-fold) tie <br /> 10 kb insert. Analysis subclones p12 insert                          triscnpts present cell cycle. <br /> demonstrated 2.5 kb HindLl-SalI fragment complem- <br /> ented HU sensitivity temperature sensitive growth (data                   DISCUSSION <br /> not shown). We HYS2 gene maps chromosome <br /> X using cloned gene probe blot yeast chromosomes                    We screened mutants sensitive HU identified <br /> separated pulse field gel electrophoresis (data shown). The                 distinct HYS genes. The hys mutations expected fall into <br /> nucleotide sequence 2.5 kb Hind-Sall DNA fragment                        classes. The <span id='am-25' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-26' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> consists mutatons defective in <br /> both sans detenined (DDBJ accession no. D50324). A                             coupling mitosis completion DNA replication. In the <br /> single open-reading firme witiin region, corre-                     presence HU, enter mitosis incomplete chromo- <br />                                                                                                               Nucleic Acids Research, 1995, Vol. 23, No. 17 3499 <br />  <br />                                                                                                   adjacent SUI2. Although molecular function HYS2 yet <br />                     MI                                                                            unknown, observations suggest Hys2 likely be <br />                                                                                                   involved DNA replication. At restrictive temperature, the <br />                                                                                                   hys2-J mutation causes accumulation cells large bud <br />              7,J                                                                                  single nucleus neck bud. The <br />                    4 <br />                                                                                                   average DNA content population hys2-1 cells shifted <br />                                                                                                   G2 value. These phenotypes indicate while <br />                                                                                                   bulk DNA replication completed hys2-J cells the <br />                                                                                                   restrictive temperature, mitosis blocked. The hys2-J mutation <br />                                                                                                   causes elevated levels chromosome loss genetic <br />                    8 <br />                                                                                                   recombination causes increase probability death cells <br />                              I)~~~~~~~~~~~~~~~~~~~~~~~~~~                                         lacking functional RAD9 checkpoint gene. <br />                        I() <br />                                                                                                      The hys2-J mutation causes cells arrest DNA repli- <br />                                                                                                   cation essentially complete level detection <br />                                                                                                   provided flow cytometry. The hys2-J mutant arrested the <br />                                                                                                   G2 phase restrictive temperature, hys2-J rad9 <br />                                                                                                   double mutant failed arrest died rapidly. This consistent <br />                              1   1)   411   61)     81)    1(1(1   12(    140 160 181) 21)0 220   previous observation RAD9 checkpoint is <br />                                                           Timie (min)                             required late S/G2 phase arrest mutants defective in <br />                                                                                                   DNA replication (24). It suggested DNA lesions <br />        HYS2                                   UbN                        e.d           N          activate RAD9 checkpoint arrest cell division G2 prior <br />                                                                                                   mitosis. DNA damage evident hys2-J cells level of <br />        CLN2                           4. 4. <br />                                                                           -a          -         elevated mitotic recombination (32) inability chromo- <br />        ACT]                  ,*Sp           *       au ae                          tpa@ <br />                                                                                                   somes migrate pulse field gel electrophoresis be <br />                                                                                                   associated DNA damage. Therefore, possible that <br />                                                                                                   DNA lesions formed restrictive temperature hys2 <br /> Figure 5. Northern blot analysis HYS2 transcription. Three micrograms                       mutant activate RAD9 checkpoint arrest G2. Finally, it <br /> total RNA prepared samples cells taken 20 min intervals                          noted cell cycle arrest cdc8 mutants the <br /> a-factorrelease separated 1% formaldehyde-agarose gels analyzed                       restrictive temperature HU treated cells does require <br /> by Northern blot analysis using HYS2 probe. The cell cycle synchronization                    RA4D9 gene (24,42,43). The CDC8 gene encodes thymidylate <br /> was confirmed probing CLN2. The levels CLN2 transcript <br /> fluctuate cell cycle, peaking G1/S boundary (29,44). The                   kinase required dTTP synthesis HU potent inhibitor <br /> blot hybridized ACT] loading control (bottom panel). Samples                          ribonucleotide reductase necessary deoxyribonucleotides <br /> were analyzed cell-cycle synchrony (top panel). Unbudded cells                       production. Thus, limitation precursors DNA synthesis <br /> (U) cells small buds (0) quantitated. The intensity HYS2                     does appear result RAD9 checkpoint activation. This may <br /> ACT] bands quantitated densitometric tracing film an <br /> appropriate exposure levels HYS2 (O) CLN2 (A) mRNAs                               suggest HYS2 plays role step DNA replication <br /> normalized relative toACTI transcript levels (middle panel). The peak value                    production precursors DNA synthesis. <br /> transcript levels given value 100 graphical purpose.                                    Most genes involved DNA replication periodically <br />                                                                                                   expressed cell cycle, peaking GI/S boundary. <br />                                                                                                   This periodic expression dependent MCB element <br />                                                                                                   located promoter regions (37). The HYS2 gene copy <br />                                                                                                   MCB element appropriate distance upstream the <br /> somal duplication, resulting lethality. Enoch et al. (16)                                 translation start site. However, cell cycle periodicity HYS2 <br /> used screen fission yeast identify genes involved                                   transcription weak (varied 2- 3-fold) the <br /> checkpoint control prevents mitosis occurring S                                   transcript present cell cycle. Furthermore, cells <br /> phase completed. DNA sequence complementation analyses                                     expressing HYS2 constitutively GALI promoter did <br /> revealed HYS] identical SPK1/MEC21RAD531SADJ,                                        progress cell cycle observable perturba- <br /> gene known involved S phase checkpoint <br />                                                                                                   tion (data shown). These results suggest transcrip- <br /> (38-41). Thus, hysi-I mutation assigned class.                                tional control cell cycle critical roles HYS2. <br /> The second class consists mutations defective DNA <br /> replication itself. Because HU blocks DNA synthesis, mutations                                    Additionally, results imply Hys2 protein is <br /> defective DNA metabolism predicted cause severe                                    present cell cycle roles DNA <br /> growth retardation presence HU. In contrast                                metabolism DNA replication, DNA damage <br /> class, mutations advance cell cycle                                       repair. In fact, hys2-J mutation conferred sensitivity the <br /> presence HU. We shown evidence indicating                                     DNA damage-inducing agent MMS. <br /> hys2 mutation assigned second class. The screen                                         In summary, characterization hys2 mutant demonstrated that <br /> unlikely saturated, identified allele                                 screening hys mutants useful better understanding cell <br /> HYS gene.                                                                                          cycle regulation S G2 phases. In addition extensive <br />    A single recessive mutation HYS2 responsible HU                                 studies HYSJ/SPKJIMEC21RAD531SADJ HYS2, charac- <br /> sensitive temperature sensitive growth phenotypes. We                                    terization hys3 hys4 mutants understanding <br /> shown HYS2 essential gene located chromosome X                                      replication control cell cycle. <br />  3500 Nucleic Acids Research, 1995, Vol. 23, No. 17 <br />  <br /> ACKNOWLEDGMENTS                                                                  18 Sherman, F., G. R. Fink J. B. Hicks. (1986) Laboratory Course <br />                                                                                     Manualfor Methods Yeast Genetics. Cold Spring Harbor Laboratory, <br />                                                                                     Cold Spring Harbor, NY <br /> We thank Ted Weinert sending materials communicating                     19 Sambrook, J., E. F. Fritsch T. Maniatis. (1989) Molecular Cloning: A <br /> unpublished results, Steve Elledge communicating SADI                       Laboratory Manual, Second edition. Cold Spring Harbor Laboratory, Cold <br /> data publication, Hiroyulk Araki Stuart MacNeill                     Spring Harbor, NY <br /> helpful discussion Charlie Brenner, Joe Horecka Rosama-                  20 Ito, H., Y Fukuda, K. Murata A. Kimura. (1983) J. Bacteriol., 153, <br />                                                                                      163-168. <br /> na Ruggieri critical reading manuscnpt. This work                 21 Rothstein, R. (1983) Methods Enzymol., 101, 202-211. <br /> supported Inamori foundation (K.S.) Grants-in-Aid                    22 Gietz, R. D. A. Sugino. (1988) Gene, 74, 527-534. <br /> the Ministry Education, Science Culture Japan (K.S.                23 Blank, A., B. Kim L. A. Loeb. (1994) Proc. Natl. Acad. Sci. USA, 91, <br /> K.M.).                                                                              9047-9051. <br />                                                                                  24 Weinert, T. A. L. H. Hartwell. (1993) Genetics, 134, 63-80. <br />                                                                                  25 Hagan, I. M. J. S. Hyams. (1988) J. Cell Sci., 89, 343-357. <br />                                                                                  26 Hisamoto, N., K. Sugimoto K. Matsumoto. (1994) Mol. Cell. Biol., 14, <br />                                                                                     3158-3165. <br /> REFERENCES                                                                       27 Gerring, S. L., F. Spencer P. Hieter. (1990) EMBO J., 11, 87-96. <br />                                                                                  28 Hennessy, K. M., A. Lee, E. Chen D. Botstein. (1991) Genes Dev., 5, <br />   1 Hartwell, L. H. T. A. Weinert. (1989) Science, 246, 629-634.                958-969. <br />  2 Munray, A. W. (1992) Nature, 359,599-604.                                     29 Wittenberg, C., K. Sugimoto S. I. Reed. (1990) Cell, 62, 225-237. <br />  3 Enoch, T. P. Nurse. (1990) Cell, 60, 665-673.                             30 Hadwiger, J. A., C. Wittenberg, M. A. Barros Lopes, H. E. Richardson <br />  4 Lundgren, K., N. Walworth, R. Booher., M. Dembski, M. Kirschner D.           S. I. Reed. (1989) Proc. Natl. Acad. Sci. USA, 86, 6255-6259. <br />     Beach. (1991) Cell, 64, 1111-1122.                                           31 Hartwell, L. H. (1976) J. Mol. Biol., 104, 803-817. <br />  5 Amon, A., U. Surana, I. Muroff K. Nasmyth. (1992) Nature, 355,            32 Hartwell, L. H. D. Smith. (1985) Genetics, 110, 381-395. <br />     368-371.             <br /> </body></html>