<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Journal Experimental Botany, Vol. 63, No. 14, pp.695&#8211;709, <br />                                              2, pp.  5289&#8211;5299,  2012 <br />                                                               2012 <br /> doi:10.1093/jxb/ers191 Advance <br /> doi:10.1093/jxb/err313   AdvanceAccess    publication12 <br />                                   Accesspublication     July, 2012 2011 <br />                                                       4 November, <br />                                     all <br /> <span id='am-108' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-109' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-111'  ></span><span id='am-112' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> paper available online free access <br />                                            access charges  (see http://jxb.oxfordjournals.org/open_access.html <br />                                                   charges (see  http://jxb.oxfordjournals.org/open_access.html for <br />                                                                                                                 further <br />                                                                                                                      details) <br />                                                                                                                              details) <br />  <br />  <br />  <br /> RESEARCH PAPER <br />  <br /> The <br /> In   sensitiveoceanica <br />    <span id='am-128' about='obo:NCBITaxon_55487' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_3398 obo:NCBITaxon_16360 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_55435 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-129'  ></span><span id='am-130' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-131' property="rdfs:label" content="Posidonia" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Posidonia</span>   freezing3  mutation <br />                          cadmium      <span id='am-1' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Arabidopsis</span> <br />                                    induces           thaliana <br />                                             changes DNA is <br /> a cold-sensitive <br /> methylation   allele homomeric <br />                   chromatin   patterningacetyl-CoA carboxylase <br /> that results cold-induced cuticle deficiencies <br /> Maria Greco, <span id='am-241' about='obo:NCBITaxon_37496' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3646 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3977 obo:NCBITaxon_91827 obo:NCBITaxon_235629 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_235880 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-242' property="rdfs:label" content="Adriana" datatype="xsd:string"></span><span id='am-243'  ></span><span id='am-244' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-245' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Adriana</span> Chiappetta, Leonardo Bruno Maria Beatrice Bitonti* <br /> Department <br />  Azura Amid    of1Ecology, University Calabria, <br />                   , <span id='am-205' about='obo:NCBITaxon_48819' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_214597 obo:NCBITaxon_131567 obo:NCBITaxon_214607 obo:NCBITaxon_3398 obo:NCBITaxon_4143 obo:NCBITaxon_1477611 obo:NCBITaxon_26122 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-206' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-207' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-208'  ></span><span id='am-209' property="rdfs:label" content="Anna" datatype="xsd:string"></span>Anna</span> Lytovchenko      2        Laboratory <br />                                             , Alisdair        Plant <br />                                                        R. Fernie  2 <br />                                                                                             GlennBucci, <br />                                                                                         Pietro <br />                                                                              Warren1,&#8224;Ponte <br />                                                                        Cyto-physiology, <br />                                                                     , Gareth                            I-870361,*,&#8225; <br />                                                                                                                  Arcavacata di Rende, <br />                                                                                                    J. Thorlby <br /> Cosenza, Italy <br />  1&#8194; <br />      School Biological Sciences, Royal Holloway, University London, Egham, Surrey, TW20 0EX, UK <br /> *2&#8194; To correspondence addressed. E-mail: b.bitonti@unical.it <br />      Max-Planck-Institute Molecular Plant Physiology, Am Muehlenberg 1, 14476 Potsdam-Golm, Germany <br /> *&#8194; <br />  Present29 <br /> Received Address: SCION <br />            May 2011;     &#8211; Next <br />                      Revised    Generation <br />                              8 July         Biomaterials, <br />                                     2011; Accepted        49, Sala <br />                                                    18 August  2011Street, Private Bag 3020, Rotorua 3046, New Zealand. <br /> &#8224;&#8194; <br />      Gareth Warren deceased. <br /> &#8225;&#8194; <br />      To correspondence addressed. E-mail: glenn.thorlby@scionresearch.com <br />  <br /> Abstract <br /> Received 13 February 2012; Revised 18 May 2012; Accepted 1 June 2012 <br /> In mammals, cadmium widely considered non-genotoxic carcinogen acting methylation-dependent <br /> epigenetic mechanism. Here, effects Cd treatment DNA methylation patten examined with <br /> Abstract <br /> its effect chromatin reconfiguration <span id='am-133' about='obo:NCBITaxon_55489' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_3398 obo:NCBITaxon_16360 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_55487 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_55435 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-134' property="rdfs:label" content="Posidonia oceanica" datatype="xsd:string"></span><span id='am-135'  ></span><span id='am-136' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Posidonia oceanica</span>. DNA methylation level pattern analysed in <br /> actively growing organs, short- (6 h) long- (2 d 4 d) term low (10 mM) high (50 mM) doses Cd, <br /> The sfr3 mutation <br /> through               causes freezing sensitivity <br />            Methylation-Sensitive                     Arabidopsis <br />                                            Amplification    Polymorphism  thaliana.   Mapping, <br />                                                                                  technique    andsequencing,   transgenic <br />                                                                                                     immunocytological        comple- <br />                                                                                                                                approach, <br /> mentation    showed    sfr3   missense    mutation     ACC1,    essential  gene   encoding <br /> respectively. The expression member CHROMOMETHYLASE (CMT) <span id='am-95' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span>, DNA methyltransferase,   homomeric    (multifunctional) <br /> acetyl-CoA <br /> was           carboxylase. <br />        assessed           Cuticle permeability <br />                          qRT-PCR.                    compromised <br />                                              Nuclear chromatin        ultrastructure          mutant by <br />                                                                                          sfr3 investigated <br />                                                                                  thewas                    plants grown <br />                                                                                                                  transmission       the <br />                                                                                                                                  <span id='am-210' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-211' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-212' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-213'  ></span><span id='am-214' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>electron</span> <br /> microscopy. Cd treatment induced DNA hypermethylation, up-regulation CMT, indicating the <br /> cold    warm.    Wax  deposition     inflorescence      stem   cold-grown    sfr3 plants  inhibited    de <br /> long-chain <br /> novo         components <br />         methylation           their <br />                       did       leaf cuticular <br />                                     occur.    Moreover, wax                    Cd led with <br />                                                                      dose compared <br />                                                               high reduced <br />                                                                                            wild-type plants. <br />                                                                                             progressive            Thus, freezing sen- <br />                                                                                                              heterochromatinization     of <br /> sitivity sfr3 <br /> interphase       appears, <br />              nuclei          these <br />                      apoptotic          results, <br />                                         figures    due <br />                                                      toalso       cuticular <br />                                                              observed             deficiencies <br />                                                                           long-term       develop <br />                                                                                             treatment.         during <br />                                                                                                          The <span id='am-236' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-237' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-238' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-239' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-240'  ></span>data</span>      cold acclimation. <br />                                                                                                                    demonstrate    Cd <br /> These observations <br /> perturbs                demonstrated <br />            DNA methylation             essential <br />                                        status  theroleinvolvement <br />                                                                  cuticleofinatolerance   freezing drought. <br />                                                                                       specific methyltransferase.    Such changes are <br /> linked nuclear chromatin reconfiguration likely establish new balance expressed/repressed chromatin. <br /> Key words: <br /> Overall,  acetyl-CoA <br />               data showcarboxylase, <br />                             epigenetic  Arabidopsis, cuticle, <br />                                               basis       freezing tolerance, <br />                                                             mechanism       underlying wax. <br />                                                                                           Cd toxicity plants. <br />  <br /> Key words: 5-Methylcytosine-antibody, cadmium-stress condition, chromatin reconfiguration, CHROMOMETHYLASE, <br /> DNA-methylation, Methylation- Sensitive Amplification Polymorphism (MSAP), Posidonia oceanica (L.) Delile. <br /> Introduction <br />  Freezing tolerance plants generally acknowledged altered expression cold acclimation (reviewed by <br />  complex trait, mechanisms contributing sur- Thomashow, 1999, 2010)&#160;has proved extremely power- <br /> Introduction <br />  vival hardy plant. Comparisons hardy non-hardy ful tool, leading identification C-repeat (CRT)-binding <br /> In    thehave <br />  plants         yielded bewildering <br />              Mediterranean         coastalarray   differences <br />                                               ecosystem,        atendemic           Although essential forelement <br />                                                                           physio- factor/dehydration-responsive                            (DRE)-binding <br />                                                                                                                                  plant growth,                   pro- <br />                                                                                                                                                      terrestrial <br />  logical, biochemical, <br /> seagrass                   molecular <br />              Posidonia oceanica              genetic <br />                                       (L.) Delile      levels. <br />                                                      plays        Freezingrole <br />                                                               relevant        tol- plants, <br />                                                                                      tein 1 (CBF/DREB1)           regulon and <br />                                                                                               Cd readily absorbed               growing <br />                                                                                                                               roots           understandinginto <br />                                                                                                                                           translocated          of <br />  erance <br /> by        primary <br />       ensuring     treated production, <br />                                 developmental watertrait,  hardy <span id='am-138' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-139' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-140' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> <br />                                                          oxygenation                 cold acclimation <br />                                                                                 aerial   organs while, signalling     pathways <br />                                                                                                                 acquatic     plants,(Stockinger     et&#160;al., <br />                                                                                                                                           directly   taken2001; <br />                                                                                                                                                                    up <br />  freezing <br /> provides      niches    insome <br />                  sensitive      summer <br />                                       animals, andbesides <br />                                                     developcounteracting <br />                                                                  freezing       leaves. Inet&#160;al., <br />                                                                                      Thomashow        plants,2001;    Seki et&#160;al., induces <br />                                                                                                                  Cd absorption         2003; Vogel <br />                                                                                                                                                 complexet&#160;al.,  2005; <br />                                                                                                                                                              changes <br />  toleranceerosion <br /> coastal        exposure <br />                               cold, process <br />                                  widespread     meadows        (Ott,as1980; <br />                                                                known                 Doherty <br />                                                                                cold          et&#160;al., 2009). <br />                                                                                          genetic,            A&#160;reporter-based <br />                                                                                                           biochemical                   screen levels <br />                                                                                                                                                       used towhich <br />                                                                                                                              physiological                  iden- <br />  acclimation <br /> Piazzi          (Guy, <br />           et al.,       1990). <br />                     1999;        However,etunlike <br />                             Alcoverro         al., 2001).      There isdevel- <br />                                                       morphological                  tify large account <br />                                                                                ultimately     set mutants <br />                                                                                                                 itsspecifically <br />                                                                                                                           toxicity (Valle       their <br />                                                                                                                                        alteredand    Ulmer,signalling <br />                                                                                                                                                                 1972; <br />  opment, development <br /> considerable       evidence  freezing   tolerance <br />                                       P. oceanica      plants doesarenotable reveal <br />                                                                                  Sanitz         cold (and <br />                                                                                      responseditoToppi     andosmotic <br />                                                                                                                   Gabrielli,stress) <br />                                                                                                                                 1999;(Ishitani  et&#160;al.,et1997; <br />                                                                                                                                          Benavides              Gong <br />                                                                                                                                                            al., 2005; <br />  qualitatively <br /> absorb      different <br />                    accumulatestagesmetals <br />                                       components; <br />                                              sediments thus, (Sanchiz        et&#160;al., 2002), <br />                                                                         Weber         et al.,and2006;    Liu this, <br />                                                                                                                       number <br />                                                                                                                                    2008). ofThe <br />                                                                                                                          et al.,               novel   components <br />                                                                                                                                                         obvious <br />  difficult <br /> et          distinguish <br />     al., 1990;                what, 1998; <br />                                        anything, <br />                  Pergent-Martini,                   contributed <br />                                                Maserti    et al., 2005)        symptom <br />                                                                              CBF/DREB1          signalistransduction <br />                                                                                                   Cd toxicity         reduction pathway <br />                                                                                                                                          plant were <br />                                                                                                                                                     growth identified <br />                                                                                                                                                                to <br />  changes <br /> influencing                cold acclimation. <br />                  metal bioavailability       marine ecosystem.             (reviewed    Xiong <br />                                                                                           inhibition            et&#160;al., 2002). Through <br />                                                                                                         photosynthesis,                    similar <br />                                                                                                                                     respiration,     andapproaches, <br />                                                                                                                                                             nitrogen <br /> For Several    complementary <br />        reason,                 approaches <br />                          seagrass             considered <br />                                            widely       used              ICE1, upstream <br />                                                                                  metabolism, <br />                                                                                                 regulator    CBF/DREB1, <br />                                                                                                                   reduction         waterwasandisolated   and <br />                                                                                                                                                              mineral <br /> aattempt <br />     metal tobioindicator <br />                overcome      problem. <br />                               species        The investigation <br />                                                     et al., 1988; of <br />                                        (Maserti                              genes uptake <br />                                                                          Pergent     characterized    (Chinnusamy <br />                                                                                               (Ouzonidou                 et&#160;al., Perfus-Barbeoch <br />                                                                                                                                  2003; Miura et&#160;al., <br />                                                                                                                 et al., 1997;                          et 2007)&#160;and <br />                                                                                                                                                            al., 2000; <br /> et al., 1995; Lafabrie et al., 2007). Cd Shukla et al., 2003; Sobkowiak Deckert, 2003). <br /> widespread        heavyacetyl-CoA <br />  Abbreviations: ACCase,              BAC, <br />                            metals carboxylase;   terrestrial      marine <br />                                                        bacterial artificial             At the <br />                                                                             chromosome; CAPS,      genetic <br />                                                                                               cleaved          level, <br />                                                                                                        amplified        both <br />                                                                                                                  polymorphic         animals <br />                                                                                                                               sequence;          plants, <br />                                                                                                                                           CRT, C-repeat;          Cd <br />                                                                                                                                                           CBF/DREB1, <br />  CRT-binding factor/DRE-binding protein 1; DRE, dehydration-responsive element. induce chromosomal aberrations, abnormalities in <br /> environments. <br /> &#169; 2012 The Author(s). <br /> &#170; <br /> This <br />   2011     Open <br />      anThe   Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.0/ <br />               Author(s). <br /> uk/) which <br /> This        permits <br />       Open          article noncommercial <br />                     unrestricted <br />                  Access                          use, <br />                                   distributed thedistribution, <br />                                                          terms    reproduction <br />                                                                      andCreative      anyAttribution <br />                                                                                  Commons     medium, provided originalLicense <br />                                                                                                                            work (http://creativecommons.org/licenses/by- <br />                                                                                                         Non-Commercial              properly cited. <br /> nc/3.0), permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />  5290&#8194; |&#8194;Amid et&#160;al. <br />  <br /> mutants affect CBF/DREB1-independent regulatory path-                subjected 11 d cold acclimation 4&#160;&#176;C, 8&#8201;h photoperiod at <br /> ways identified (Zhu et&#160;al., 2004; <span id='am-196' about='obo:NCBITaxon_569144' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43755 obo:NCBITaxon_27474 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_43917 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_43742 obo:NCBITaxon_50557 obo:NCBITaxon_179436'><span id='am-197'  ></span><span id='am-198' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-199' property="rdfs:label" content="Medina" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Medina</span> et&#160;al., 2005).             220&#160;&#181;M m&#8211;2 s&#8211;1. They placed freezer air temperature <br />                                                                           minimum &#8211;6.0&#160;&#176;C 16&#8201;h returned pre-acclimation <br /> These studies not, however, led understanding the <br />                                                                           growth conditions. Injury assessed 5 d.&#160;Cold treatments were <br /> role individual genes freezing tolerance,          carried time period stated conditions for <br /> effect control expression genes (Xin         cold acclimation. For drought testing, plants grown 18&#8211;20&#160;&#176;C as <br /> Browse,&#160;2000).                                                            above, period cold treatment. Water withheld <br />    Individual genes clear effects freezing tolerance       plants monitored wilted. Plants watered and <br />                                                                           recovery assessed. <br /> selected mutant phenotypes. Several groups isolated <br /> such mutants determined contribution individual genes <br /> or molecular processes freezing tolerance. Xin Browse              Mapping SFR3 <br /> (1998) Xin et&#160;al. (2007) isolated mutants increased toler-       From mapping cross sfr3/sfr3 Col (Columbia) &#215; SFR3+/SFR3+ <br /> ance freezing absence cold acclimation. The mutations        Ler (Landsberg erecta), total 557 new F2 zygotes screened <br /> may components signalling pathway(s) responsible             crossovers region. Among zygotes showing crossovers, <br />                                                                           SFR3 genotypes determined freeze testing F3 progeny <br /> for cold acclimation genes make self-sufficient indi-          zygote. Details markers developed work avail- <br /> vidual contributions protection freezing. Mutations       able request. DNA samples isolated F2 plants (Thorlby <br /> decrease freezing tolerance likely genes required        et&#160;al., 1999), analysed cleaved amplified polymorphic sequence <br /> freezing tolerance wild-type. The frs1 mutation          (CAPS) marker genotypes (Konieczny Ausubel, 1993)&#160; sim- <br /> to allele ABA3 (Llorente et&#160;al., 2000)&#160;and confirms          ple sequence length polymorphism marker genotypes (Bell Ecker, <br />                                                                           1994). <br /> importance abscisic acid freezing tolerance. <br />    The Arabidopsis sensitive freezing (sfr) mutants (Warren <br /> et&#160;al., 1996)&#160;are impaired freezing tolerance following          Subcloning complementation <br /> cold acclimation near normal (wild-type) levels freez-        In silico analysis genomic sequence indicated gene At1g36160 <br /> ing tolerance prior cold acclimation. The corresponding SFR            contained 15&#8201;kb SpeI fragment bacterial artificial chromo- <br />                                                                           (BAC) F12D14. A&#160;genomic fragment cloned BAC <br /> genes components cold-induced gene response <br />                                                                           modified version vector pCAMBIA3300 (CAMBIA, <br /> whose individual contributions freezing tolerance cur-         Canberra, Australia: http://www.cambia.org) SpeI site intro- <br /> rently understood. Alternatively, genes essential         duced multiple cloning site. <br /> freezing tolerance transcriptionally regulated cold. <br /> It shown sfr6 mutant unable induce genes <br />                                                                           Plant transformation selection <br /> with CRT/DRE elements promoters (Knight et&#160;al., 1999; <br />                                                                           <span id='am-246' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-247' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-248' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-249' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-250' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>Plasmids</span> transferred <span id='am-141' about='obo:NCBITaxon_358' typeof='owl:Thing obo:NCBITaxon_356 obo:NCBITaxon_357 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_82115 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_28211 obo:NCBITaxon_227290 obo:NCBITaxon_1183400'><span id='am-142' property="oboInOwl:hasRelatedSynonym" content="Phytomonas tumefaciens" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasRelatedSynonym" content="Bacillus radiobacter" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasDbXref" content="PMID:16449463" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasAlternativeId" content="NCBITaxon:364" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-147'  ></span><span id='am-148' property="oboInOwl:hasRelatedSynonym" content="Alcaligenes radiobacter" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasRelatedSynonym" content="Agrobacterium genomosp. 4" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasAlternativeId" content="NCBITaxon:450636" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasRelatedSynonym" content="Agrobacterium tumefacien" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasAlternativeId" content="NCBITaxon:365" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasAlternativeId" content="NCBITaxon:33048" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasAlternativeId" content="NCBITaxon:369" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasRelatedSynonym" content="Bacterium tumefaciens" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasAlternativeId" content="NCBITaxon:362" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:hasAlternativeId" content="NCBITaxon:450640" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas radiobacter" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas tumefaciens" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasDbXref" content="PMID:16560572" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasDbXref" content="PMID:11760945" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasDbXref" content="PMID:8240952" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasRelatedSynonym" content="Achromobacter radiobacter" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasRelatedSynonym" content="Agrobacterium genomovar 4" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasAlternativeId" content="NCBITaxon:366" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasRelatedSynonym" content="Polymonas tumefaciens" datatype="xsd:string"></span><span id='am-168' property="rdfs:label" content="Agrobacterium tumefaciens" datatype="xsd:string"></span><span id='am-169' property="oboInOwl:hasDbXref" content="PMID:11931171" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:hasRelatedSynonym" content="Agrobacterium radiobacter" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasRelatedSynonym" content="Rhizobium radiobacter" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasAlternativeId" content="NCBITaxon:363" datatype="xsd:string"></span><span id='am-173' property="oboInOwl:hasDbXref" content="PMID:11211278" datatype="xsd:string"></span><span id='am-174' property="oboInOwl:hasRelatedSynonym" content="Bacterium radiobacter" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasAlternativeId" content="NCBITaxon:367" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasAlternativeId" content="NCBITaxon:368" datatype="xsd:string"></span>Agrobacterium tumefaciens</span> GV3101/ <br /> Boyce et&#160;al., 2003). The SFR6 gene identified protein            pM90 (Koncz Schell, 1986)&#160;by electroporation (Cangelosi et&#160;al., <br /> of unknown function acts post-translationally CBF func-           1991). Plants transformed floral dip method (Clough and <br /> tion key component cold acclimation process (Knight           Bent, 1998). Primary transformants selected Basta-resistance <br /> et&#160;al., 2009). SFR2 encodes constitutively expressed chloro-            (the marker pCAMBIA <span id='am-201' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-202' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-203' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>vectors</span>) repeated spraying the <br /> plast targeted &#946;-glycosidase (Thorlby et&#160;al., 2004; Fourrier et&#160;al.,      seedlings 250&#8201;mg l&#8211;1 solution herbicide Challenge <br />                                                                           60 (AgrEvo Ltd, King&#8217;s Lynn, UK) growth differential was <br /> 2008)&#160;that involved galactolipid remodelling outer           clear. <br /> chloroplast envelope (Moellering et&#160;al.,&#160;2010). <br />    The sfr3 mutation causes greatest freezing sensitivity <br />                                                                           Visualization defects leaf cuticle using toluidine <br /> in young incompletely expanded leaves. Expression analysis <br />                                                                           blue&#160;staining <br /> (Knight et&#160;al., 1999)&#160;indicates compromised its <br />                                                                           The method Tanaka et&#160;al. (2004) used. Seeds grown Petri <br /> ability induce genes associated cold stress signalling <br />                                                                           dishes 0.5&#215; MS medium solidified 0.7% plant culture-tested agar <br /> pathways. Plants mutation indistinguishable            (Sigma). Plants stratified 4&#160;&#176;C 3 d, transferred growth <br /> wild-type plants normal growth temperatures display            chamber 22&#160;&#176;C continuous light 3/4 weeks, trans- <br /> pleiotropic effects 4&#160;&#176;C; short term (during cold acclima-      ferred 4&#160;&#176;C growth room periods 10 d.&#160;To stain plants, <br /> tion), mutant plants deficient anthocyanin, long        aqueous 0.05% (w/v) toluidine blue filtered 0.2&#160;&#181;m syringe <br />                                                                           filter (Sartorius, Hannover, Germany) poured culture plate <br /> term (8 weeks above), senescence older leaves acceler- <br />                                                                           plants submerged incubated 2&#8201;min. The <br /> ated (McKown et&#160;al., 1996). On current knowledge,              staining solution poured plants washed sub- <br /> these phenotypes expected directly cause freezing             merging twice distilled water. Stained plants viewed by <br /> sensitivity. Therefore, elucidating function wild-type         light microscopy. <br /> SFR3 gene new insights mechanism which <br /> plant cells achieve freezing tolerance. In paper, identified      Measurement chlorophyll&#160;leaching <br /> SFR3 investigated role freezing tolerance.                     To measure chlorophyll leaching, aerial parts 4-week-old plants, <br />                                                                           grown described freeze testing, weighed immersed in <br />                                                                           tubes containing 30&#8201;ml 80% ethanol. Approximately 0.5&#8201;g mate- <br /> Materials methods                                                     rial used assay. Tubes incubated dark gentle <br />                                                                           agitation rotary shaker, aliquots taken indicated times and <br /> Plant growth, freeze drought&#160;testing                              absorbance measured 654&#8201;nm. Chlorophyll content calculated <br /> Plants grown previously described (Thorlby et&#160;al., 2004). To      according formula A654 &#215; extraction volume (ml) &#215; 1/39.8&#8201;&#215;&#8201;1/ <br /> screen individuals freezing tolerance, seedlings grown 4/5   weight tissue sample (g)&#160;=&#160;mg chlorophyll g&#8211;1 fresh weight tissue <br /> weeks 18&#8211;20&#160;&#176;C 9&#8201;h photoperiod 250&#160;&#181;M m&#8211;2 s&#8211;1,      (Wintermans Mots, 1965). <br />                                                                              Identification sfr3 mutation A. thaliana&#8194; |&#8194; 5291 <br />  <br /> Scanning electron microscopy                                                Results <br /> Stems wild-type sfr3 plants collected, allowed air dry, <br /> mounted stubs, coated gold particles using Polaron E5100      Isolation identification SFR3 gene and <br /> coating unit. They examined using Hitachi S3000N scanning            sfr3&#160;mutation <br /> electron microscope accelerating voltage 20&#8201;kV. <br />                                                                             The sfr3 mutation isolated pedigreed ethyl methyl <br />                                                                             sulfonate mutant set James Dooner (1990) freezing- <br /> Analysis of&#160;waxes <br />                                                                             sensitive mutant (Warren et&#160;al., 1996), previously mapped close <br /> Detached leaves immersed gentle swirling chloroform            centromere region chromosome I&#160;between markers <br /> for 30 s, carefully removed, extracts dried nitrogen. <br /> Metabolite analysis extracts gas chromatography/mass      AIG1 T27K12 (Thorlby et&#160;al., 1999). It clear freezing- <br /> spectrometry method Fiehn et&#160;al. (2000) using greater range     sensitive phenotype (Fig.&#160;1A). <br /> authentic standards lipophilic metabolite identification. Each analy-      The gene fine mapped 220&#8239;kb region the <br /> sis performed biological replicates.                           Arabidopsis genome genes At1g36150 At1g36410 <br />                                                                             (Supplementary Fig. S1 JXB online). This interval exam- <br />                                                                             ined gene annotations. As expected, proximity the <br />                                                                             chromosome I&#160;centromere, interval low gene density <br />                                                                             high frequency pseudogenes transposon genes. Genes <br />                                                                             homology transposable elements genes annotated as <br />                                                                             pseudogenes unlikely encode non-redundant function and <br />                                                                             initially selected analysis. The remaining genes the <br />                                                                             220&#8201;kb interval analysed sequencing and/or isola- <br />                                                                             tion analysis knockout mutants, available. <br />                                                                                A total genes sequenced using DNA amplified <br />                                                                             sfr3 mutant compared database versions. <br />                                                                             The gene At1g36160 different database rest <br />                                                                             identical. The sequence change At1g36160, G&#8594;A at <br />                                                                             nt 910 coding sequence (nt 1541 genomic DNA), <br />                                                                             created missense mutation (Fig.&#160;2A) restriction site dif- <br />                                                                             ference; BseRI site wild-type present in <br />                                                                             sfr3. We exploited develop CAPS marker distinguish <br />                                                                             wild-type sfr3, used marker confirm <br />                                                                             segregation mutant phenotype sequence&#160;change. <br />                                                                                At1g36160 encodes homomeric acetyl-CoA carboxylase <br />                                                                             (ACCase), ACC1, shown previously essential embryo <br />                                                                             survival (Baud et&#160;al., 2003). Recently, weak allele ACC1, <br />                                                                             glossyhead (gsd1), identified (Lu et&#160;al., 2011)&#160;that does not <br />                                                                             cause embryo death (unlike ACC1 mutants) con- <br />                                                                             fers phenotypic changes associated perturbation bio- <br />                                                                             synthesis cuticular waxes. <br />  <br />                                                                             The sfr3 mutation complemented ACC1 <br />                                                                             The ACC1 gene cloned BAC F12D14, contains <br />                                                                             ACC1 gene entirety, plant transformation vec- <br />                                                                             <span id='am-221' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-222' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-223'  ></span><span id='am-224' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-225' property="rdfs:label" content="Tor" datatype="xsd:string"></span>tor</span>. A&#160;region 4265&#8201;bp upstream translation start <span id='am-118' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-119' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-122'  ></span>codon</span> <br />                                                                             1188&#8201;bp downstream sequence included the <br />                                                                             gene. The incorporation regions normal con- <br /> Fig. 1.&#8194; The sfr3 phenotype. (A) Comparison wild-type (left)             trol gene expression resulting transgenic lines. No other <br /> and sfr3 (right) plants freezing test. Cold acclimated plants       genes, according current genome annotations, present in <br /> were frozen &#8211;6.0&#160;&#176;C 16&#8201;h returned pre-acclimation             cloned fragment. <br /> growth conditions recovery. The picture taken                    Out 24 freeze-tested primary sfr3 transformant plants, 23 <br /> 4&#160;days recovery. All leaves wild-type appeared healthy               displayed freezing tolerance indistinguishable wild- <br /> and green, did older, fully expanded leaves sfr3 plants.          type (Supplementary Fig. S2 JXB online). A&#160;number of <br /> The younger leaves sfr3 plants showed severe damage                      freezing-tolerant primary transformants showed comple- <br /> and discoloured collapsed. (B) Wild-type (left)                mentation wild-type gene selected anal- <br /> sfr3 (right) plants water withheld 25&#160;days               ysis. The plants left self-fertilize set seed. Testing of <br /> plants maintained 4&#160;&#176;C. Damage apparent                   plants grown seed confirmed transgene segre- <br /> whole rosette severe young leaves (none fully               gated freezing-tolerant phenotype, confirming ACC1 as <br /> expanded).                                                                  site sfr3 mutation. <br />  5292&#8194; |&#8194;Amid et&#160;al. <br />  <br />    In Arabidopsis, potentially genes, located con-     The sfr3 mutation highly conserved&#160;motif <br /> tiguously 25&#8201;kb region near centre chromosome <br /> 1, encoding homomeric form ACCase, ACC1 ACC2              The nucleotide change sfr3 mutant created missense <br /> (At1g36160 At1g36180) (Yanai et&#160;al., 1995). As gene          mutation changes <span id='am-215' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-216' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-217'  ></span><span id='am-218' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-219' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-220' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span>aa</span> 304 ACC1 protein glu- <br /> At1g36180/ACC2 area delineated contain-            tamic acid lysine (Fig.&#160;2B). This amino acid highly <br /> ing SFR3, sequenced, did contain sequence        conserved motif, carbamoyl-phosphate synthase L chain ATP <br /> change. A&#160;T-DNA knockout homozygous T-DNA insert             binding subdomain biotin carboxylase domain, is <br /> within At1g36180/ACC2 isolated. This line did           conserved plants, animals, yeast (Fig.&#160;2C). Interestingly, <br /> show pre- post-freezing phenotype, suggesting       mutation E86K gsd1, non-embryo-lethal <br /> this second homomeric ACCase essential gene         mutation currently identified ACC1, lies biotin <span id='am-226' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-227' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-228' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-229' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-230'  ></span>car</span>- <br /> not necessary freezing tolerance. Baud et&#160;al. (2003) obtained    boxylase domain (Lu et&#160;al., 2011). <br /> similar results, demonstrating ACCase protein could <br /> be detected acc1 knockout mutant embryos, suggesting         The sfr3 mutant drought sensitive the&#160;cold <br /> ACC2 expressed expressed low <br /> level.                                                               As ACC1 implicated development main- <br />                                                                      tenance plant cuticle (Baud et&#160;al., 2004; Lu et&#160;al., 2011), <br />                                                                      primary barrier water loss plants, drought tolerance <br />                                                                      sfr3 plants examined. The recovery sfr3 wild-type <br />                                                                      plants water withheld time avail- <br />                                                                      able identical grown warm (data not <br />                                                                      shown). Differences sfr3 wild-type plants were <br />                                                                      apparent, however, drought testing carried plants that <br />                                                                      grown warm 4 weeks transfer cold <br />                                                                      drought testing. Immediately transfer cold, water <br />                                                                      withheld. After 25 d water, sfr3 plants wilted while <br />                                                                      wild-type plants remained vigorous (Fig.&#160;1B). Wild-type plants <br />                                                                      did signs water stress additional 5 d, whereas, <br />                                                                      time, watering sfr3 plants did result recovery. <br />  <br />  <br />                                                                      The sfr3 mutation results cold-induced increase in <br />                                                                      cuticle permeability <br />                                                                      Several assays demonstrated cuticular deficiencies cold- <br />                                                                      grown sfr3 plants. Toluidine blue stains epidermis plants <br />                                                                      damaged cuticle cuticle intact (Tanaka <br />                                                                      et&#160;al., 2004), degree staining reflects severity of <br />                                                                      cuticular damage. Neither wild-type sfr3 plants showed epi- <br />                                                                      dermal staining following growth warm (data shown). <br />                                                                      Wild-type plants transferred warm and <br />                                                                      grown 10 d 4&#160; &#176;C showed staining (Fig.&#160; 3E, 3I), <br />                                                                      sfr3 plants treated way showed distinct staining <br />                                                                      (Fig.&#160;3B, 3F, 3J; C, G, K; D, H, L). The youngest <br /> Fig. 2.&#8194; Location sfr3 mutation ACC1 gene. (A)         leaves stained strongly, older leaves stained intensely <br /> Schematic representation ACC1 gene showing location       all. There little staining leaf pedicle, even <br /> of sfr3 point mutation. Solid boxes represent exons. (B) Amino   intensely stained plants. The toluidine blue staining <br /> acid change conceptual translation product ACC1        procedure performed plants grown at <br /> gene. The hatched box, dotted box, solid box represent       4&#160;&#176;C shorter periods. No staining detected plants <br /> biotin carboxylase domain, biotin carboxyl carrier domain        spent 6 d cold (data not&#160;shown). <br /> carboxyl transferase domain, respectively. (C) Alignment amino        The permeability plant cuticle assessed by <br /> acids conserved region biotin carboxylase domain    measuring release chlorophyll leaves immersed in <br /> acetyl-CoA carboxylases various species. Bold letters indicate    80% ethanol (Aharoni et&#160;al., 2004; Schnurr et&#160;al., 2004). No <br /> invariant amino acids asterisk identifies amino        difference chlorophyll release detected wild- <br /> acid changed sfr3 mutant. AtACC1, <span id='am-8' about='obo:NCBITaxon_3702' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_3701 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-9' property="oboInOwl:hasExactSynonym" content="thale-cress" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="mouse-ear cress" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Arabidopsis thaliana (thale cress)" datatype="xsd:string"></span><span id='am-14'  ></span><span id='am-15' property="rdfs:label" content="Arabidopsis thaliana" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Arbisopsis thaliana" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="thale cress" datatype="xsd:string"></span>Arabidopsis thaliana</span>        type sfr3 plants grown warm conditions (Fig.&#160;4A), but <br /> ACC1 (At1g36160); TaACC, <span id='am-58' about='obo:NCBITaxon_4565' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_147368 obo:NCBITaxon_359160 obo:NCBITaxon_1 obo:NCBITaxon_4479 obo:NCBITaxon_4564 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4734 obo:NCBITaxon_3398 obo:NCBITaxon_147389 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_38820 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="Triticum aestivam" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasExactSynonym" content="bread wheat" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Triticum aestivum8" datatype="xsd:string"></span><span id='am-62'  ></span><span id='am-63' property="oboInOwl:hasAlternativeId" content="NCBITaxon:235075" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Tricum aestivum" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasExactSynonym" content="Canadian hard winter wheat" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Triticum aestivum" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasExactSynonym" content="common wheat" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Triticum aestivum subsp. aestivum" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Triticum vulgare" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasAlternativeId" content="NCBITaxon:39424" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasExactSynonym" content="wheat" datatype="xsd:string"></span>Triticum aestivum</span> cytosolic ACCase          24&#8201;h growth cold, differences appar- <br /> (A57710); ZmACC, <span id='am-80' about='obo:NCBITaxon_4575' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_4479 obo:NCBITaxon_147369 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4734 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_38820 obo:NCBITaxon_147429 obo:NCBITaxon_131221 obo:NCBITaxon_147370 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Euchlaena" datatype="xsd:string"></span><span id='am-82'  ></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="Zea" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Zea</span> mays ACCase (T02235); DmACC,                    ent (Fig.&#160;4B). Chlorophyll leached rapidly sfr3 <br /> <span id='am-101' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-102' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-103'  ></span><span id='am-104' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Drosophila melanogaster</span> ACCase (CG11198-PB); HsACC, <span id='am-113' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-114' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-115'  ></span><span id='am-116' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Homo</span>             plants, released increased longer growth in <br /> sapiens ACCase (S41121); ScACC, <span id='am-34' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-35' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>             cold (Fig.&#160;4C, 4D). Extended growth low temperature had <br /> ACCase (P11029).                                                     little effect chlorophyll leached wild-type&#160;plants. <br />                                                                           Identification sfr3 mutation A. thaliana&#8194; |&#8194; 5293 <br />  <br />    The loss water evaporation wild-type sfr3 plants       plants warm cold. As Arabidopsis shows only <br /> was compared (Supplementary Fig. S3 JXB online). Loss                 limited growth 4&#160;&#176;C, analysis performed plants had <br /> from detached rosettes warm-grown wild-type sfr3            maintained temperature relatively long period <br /> plants similar. After 10 d growth cold, wild-type         (4 weeks) following initial growth 20&#160;&#176;C 3 weeks. The <br /> plants lost water similar rate warm-grown plants, sfr3       youngest fully expanded leaves analysed, were <br /> plants lost significantly water (Supplementary Fig. S3).            likely highest proportion wax load depos- <br />                                                                          ited cold growth conditions. <br /> Epicuticular wax morphology altered cold-grown                        This comparative analysis showed leaf wax compo- <br /> sfr3 inflorescence&#160;stems                                                 sition warm-grown sfr3 plants similar wild- <br />                                                                          type plants (Fig.&#160;6). No significant alterations of <br /> Scanning electron microscopy used examine epicuticular            alkanes, largest component Arabidopsis leaf wax, were <br /> wax crystals wild-type sfr3 inflorescence stems. These            found. The relative amounts primary alcohols fatty acids <br /> observations possible leaf material             similar. There small increase of <br /> lack visible epicuticular wax crystals Arabidopsis leaves.         longer-chain-length varieties components sfr3 <br /> Plants developed inflorescence stem warm (data            leaves, suggesting mutant deficient malonyl- <br /> not shown) transferred 4&#160;&#176;C 21 d           CoA ability produce cuticular wax warm. <br /> of growth inflorescence stem development showed dense            Substantial differences sfr3 wild-type wax com- <br /> pattern crystals surface stem wild-type      position apparent plants grown cold <br /> (Fig.&#160;5A) sfr3 (Fig.&#160;5B) plants. Plants initiated bolt-     (Fig.&#160;6). The alkanes, major wax constituent, was <br /> ing produced significant inflorescence stem         reduced sfr3 plants compared wild-type plants. There <br /> transfer cold showed different result; wild-type stems          significant decrease (P &lt;0.05) of <br /> were decorated dense pattern wax crystals (Fig.&#160;5C)            longer alkane components, C33 reduced 60% the <br /> similar produced warm, short stems           seen wild-type plants. There decrease in <br /> that developed sfr3 plants cold devoid wax             longer-chain-length primary alcohol fatty acid <br /> crystals (Fig.&#160;5D). Thus, wax deposited normally sfr3 stems        components wax, increase the <br /> that develop warm the&#160;cold.                            shorter C20 C22 fatty&#160;acids. <br />  <br /> Wax composition altered sfr3 leaves develop                   Long-term growth cold affects flower <br /> in the&#160;cold                                                              development <br /> To test cuticular deficiencies sfr3 associated       To investigate effect longer periods cold growth <br /> with changes wax composition, comparative analysis           sfr3 plants, 4-week-old warm-grown plants transferred <br /> major classes wax components present leaves carried            cold growth conditions maintained condi- <br /> out. A&#160; previous analysis foliar fatty acids (McKown et&#160;al.,          tions 10 weeks. Previous observations shown that, <br /> 1996)&#160;demonstrated differences sfr3 wild-type             8 weeks&#8217; growth cold, accelerated senescence of <br />  <br />  <br />  <br />  <br /> Fig. 3.&#8194; Toluidine blue staining sfr3 wild-type seedlings grown cold 10 d.&#160;A&#160;control wild-type plant (A) (E, <br /> I) staining shown. Three different sfr3 plants shown (B&#8211;D) (F&#8211;H J&#8211;L) staining. The images lower panel <br /> (I&#8211;L) close ups images panel details different staining patterns. <br />  5294&#8194; |&#8194;Amid et&#160;al. <br />  <br /> older leaves occurs (McKown et&#160;al., 1996). After 10-week             et&#160;al., 2003). Each domains encoded separate <br /> cold growth period, accelerated senescence older leaves         polypeptide. This case heteromeric ACCases found <br /> again observed, respects mutant appeared simi-        plastids non-graminacea plants, responsible <br /> lar wild-type plants. Differences were, however, observed        production plastidic malonyl-CoA pool required for <br /> the development flowers. If plants transferred cold         novo fatty acid synthesis. In second type ACCase, the <br /> growth conditions shortly initiation bolting,         homomeric form, functional domains encoded a <br /> inflorescence stem sfr3 mutant stunted compared        single polypeptide. <br /> the wild type, flower development abnormal (Fig.&#160; 7).             Homomeric ACCase responsible production the <br /> Shrunken slightly browned structures developed place             independent cytosolic pool malonyl-CoA. This pool sepa- <br /> petals, senesced expanding (Fig.&#160;7). Normal          rate plastid pool membranes impermeable <br /> siliques developed wild-type plants, observed       acyl-CoAs (Jacobson Stumpf, 1972). The cytosolic pool <br /> sfr3&#160;mutants.                                                          malonyl-CoA pool produced homomeric ACCase is <br />                                                                        required wide range reactions. These include elonga- <br />                                                                        tion plastid-synthesized C16 C18 fatty acids long-chain <br />                                                                        very-long-chain fatty acids seed storage triacylglycerols, <br /> Discussion <br />                                                                        waxes, sphingolipids. Malonyl-CoA required the <br /> A missense mutation ACC1, gene encoding homomeric               synthesis various secondary metabolites, including flavonoids, <br /> acetyl-CoA carboxylase (ACCase) responsible freezing        malonation amino acids glycosides (Roesler et&#160;al., <br /> sensitivity sfr3. ACCase catalyses ATP-dependent carbox-        1994). Mutation ACC1 probably reduces (or eliminates) the <br /> ylation acetyl-CoA form malonyl-CoA, com-          cytosolic malonyl-CoA pool affects pathways that <br /> mitting step novo fatty acid biosynthesis (Nikolau et&#160;al.,       utilize&#160;it. <br /> 2003). Malonyl-CoA essential elongation            A number plants carrying mutations ACC1 been <br /> fatty acids long-chain very-long-chain fatty acid&#160;forms.        characterized gene shown essential for <br />    ACCases composed distinct functional domains.          production very-long-chain fatty acids. The gurke and <br /> A&#160;structural domain carries biotin prosthetic group (the biotin    pasticcino3 mutants Arabidopsis mutations ACC1 <br /> carboxyl carrier domain). One catalytic domain, biotin car-        (Baud et&#160;al., 2003, 2004)&#160;and develop abnormal embryos with <br /> boxylase domain, catalyses carboxylation biotin prosthetic      altered <span id='am-191' about='obo:NCBITaxon_22998' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_41946 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3781 obo:NCBITaxon_3193'><span id='am-192' property="rdfs:label" content="Cotyledon" datatype="xsd:string"></span><span id='am-193' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-194'  ></span><span id='am-195' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cotyledon</span> primordia. The embryos eventually die, lead- <br /> group, second, carboxyltransferase domain, transfers         ing embryo lethality. The sfr3 mutant unlikely rep- <br /> the carboxyl group carboxy-biotin acetyl-CoA (Nikolau          resent complete loss function. More recently, weak mutant <br />  <br />  <br />  <br />  <br /> Fig. 4.&#8194; Graphs showing rate leaching chlorophyll ethanol sfr3 plants compared wild-type plants. The results from <br /> warm-grown plants (A) warm-grown plants transferred cold 24&#8201;h (B), 4 d (C), 10 d (D) shown. Experiments were <br /> performed triplicate results shown means &#177; standard deviation. The total chlorophyll present leaves of <br /> sfr3 plants, measured total extraction chlorophyll 96% ethanol, significantly different wild-type plants (data <br /> not shown). <br />                                                                        Identification sfr3 mutation A. thaliana&#8194; |&#8194; 5295 <br />  <br />  <br />  <br />  <br /> Fig. 5.&#8194; Scanning electron microscopy images showing wax crystals inflorescence stem wild-type (A, C) sfr3 (B, D) plants. <br /> The surface wild-type (A) sfr3 (B) stems developed warm transfer 4&#160;&#176;C 21 d decorated densely <br /> distributed columnar-shaped epicuticular wax crystals. Wild-type stems initiated bolting warm transfer cold <br /> showed dense decoration wax crystals (C). sfr3 stems grown way (initiation warm growth the <br /> cold) showed total lack visible wax crystals (D). Bars, 10&#160;&#181;m. <br />  <br />  <br /> allele ACC1, gsd1, identified (Lu et&#160;al., 2011). This     constructs (Lu et&#160;al.,&#160; 2011)&#160; confirmed preferential <br /> mutant does confer embryo-lethal phenotype produces        expression ACC1 epidermis upper section of <br /> plants glossy inflorescence stems, post-genital fusion flo-   inflorescence stems actively involved cuticle deposition. <br /> ral organs, reduced fertility. Characterization gsd1       This consistent major destination cytosolic <br /> demonstrated essential role ACC1 biosynthesis        malonyl-CoA developing plant produc- <br /> cuticular waxes production functioning&#160;cuticle.          tion waxes cuticle deposition. Indeed, the <br />    Several studies established ubiquitous expression         mutated embryos lethal acc1 alleles, evidence of <br /> of Arabidopsis ACC1 gene (Roesler et&#160;al., 1994; Baud              severely altered cuticle formation (Baud et&#160;al.,&#160;2004). <br /> et&#160;al., 2003; Lu et&#160;al., 2011). This surprising, consid-          The broad lipid profiling studies carried gsd1 have <br /> ering wide range reactions requiring malonyl-CoA.              confirmed essential role ACC1 generating malonyl-CoA <br /> Transcript levels highest flowers silique walls        wax synthesis (Lu et&#160;al., 2011). The gsd1 mutant exhibits a <br /> of developing embryos, gene expressed              large reduction total wax mainly decrease <br /> roots, stems, leaves, developing seeds. A&#160;high propor-            production very-long-chain alkane, secondary <br /> tion very-long-chain fatty acids incorporated tria-       alcohol, ketone precursors (Lu et&#160;al., 2011). Synthesis <br /> cylglycerols deposited seeds maturation,           cutin, major cuticle component, enhanced in <br /> are essential cuticle wax synthesis (Baud et&#160;al.,        gsd1, probably reduced malonyl-CoA pool leads to <br /> 2004). Evidence Nikolau Wurtele (1998) using                 build-up shorter acyl chains, shunted the <br /> Northern blotting situ hybridization showed ACC1          shorter-chain-length cutin (Lu et&#160;al., 2011). Direct compari- <br /> mRNA accumulated epidermal cells involved epicutic-             son gsd1 sfr3 wax composition complicated the <br /> ular wax deposition. Similarly, studies pea demonstrated           cold-induced nature sfr3 phenotype (see below). A&#160;sub- <br /> that homomeric form ACCase concentrated              stantial portion wax load sfr3 plants transferred to <br /> epidermal layer leaves (Alban et&#160;al., 1994). More recently,        cold likely deposited transfer, plants <br /> Arabidopsis microarray data (Suh et&#160;al., 2005)&#160; GUS               warm leaves growing actively. <br />  5296&#8194; |&#8194;Amid et&#160;al. <br />  <br />  <br />  <br />  <br /> Fig. 6.&#8194; Graph showing comparison relative amounts major classes leaf wax components isolated warm-grown <br /> and cold-treated sfr3 plants. The ratio wax component relative wild-type plants grown same <br /> conditions (warm cold) shown. Chemical classes chain lengths labelled horizontal <span id='am-123' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-124' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-126'  ></span><span id='am-127' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>axis</span>. Statistically significant <br /> differences (Student&#8217;s t-test, P &lt;0.05) wild-type sfr3 plants treated way indicated asterisk. <br />  <br /> However, comparative analysis (Fig.&#160;6), using leaves              These observations suggest that, sfr3 mutant, ACCase <br /> plants growing cold relatively long periods, showed        functions normally warm sufficient ACCase activ- <br /> similar trends. Little difference observed sfr3           ity present carry required metabolic functions. In <br /> and wild-type plants grown warm, clear differences         contrast, following growth cold, activity ACCase <br /> were present transfer (and growth in) cold.      limiting leads clear deficiencies. This suggests <br /> The longer-chain-length components alkanes, primary alco-          sfr3 mutation results cold-sensitive allele ACC1. <br /> hols, fatty acids reduced compared wild-type        This explain normal growth sfr3 plants warm, <br /> plants (Fig.&#160;6), expected malonyl-CoA pool         mutations ACC1 shown embryo <br /> required elongation limited. The shorter C20 C22     lethal (Baud et&#160;al., 2004)&#160;or, case gsd1 (Lu et&#160;al., 2011), <br /> chain-length fatty acid components wax increased       clear phenotypic differences. Real-time PCR analysis <br /> the cold, probably result insufficient malonyl-CoA      (Supplementary Fig. S4 JXB online) showed transcript <br /> present elongation.                                       level ACC1 did differ wild-type sfr3 plants in <br />    The results presented showed sfr3 mutant can-        warm cold growth conditions. Thus, cold sensitivity is <br /> not distinguished wild-type grown             unlikely product transcript instability. <br /> warm. Indeed, shown previously ability           It possible sfr3, like gsd1, weak allele ACC1 <br /> non-acclimated (warm-grown) wild-type plants tolerate         (unlike gsd1) able meet metabolic requirement <br /> mild freezing temperatures compromised sfr3     cytosolic malonyl-CoA warm meet an <br /> mutant (Warren et&#160;al., 1996). Cold acclimation, however,        increased demand necessary growth cold. If sfr3 <br /> clear detrimental affect ability sfr3 develop pro-   phenotype result inability respond condi- <br /> nounced freezing tolerance associated process wild-      tions requiring increased malonyl-CoA supply, be <br /> type. A&#160;number assays clearly demonstrated sfr3 plants        expected sfr3 differ wild-type plants when <br /> had cold-induced cuticular deficiencies detectable      plants subjected drought stress warm this <br /> in warm. Evidence chlorophyll efflux experiments             treatment shown lead substantial increase the <br /> (Fig.&#160;4) suggested damage effective-       deposition waxes (Kosma et&#160;al., 2009)&#160;and presumably the <br /> ness cuticle 24&#8201;h transfer cold,     demand malonyl-CoA. However, sfr3 plants showed <br /> damage increased prolonged incubation. The results        drought-induced phenotype following growth cold. <br /> toluidine blue assay, staining visible 6 d   Likewise, gsd1, weak allele ACC1, shows clear phe- <br /> transfer cold, suggested gradual degradation     notype warm, appears severe stem wax phe- <br /> effectiveness cuticle. This supported       notype (based comparison electron microscopy images <br /> images rich decoration wax crystals inflores-        presented Lu et&#160;al., 2011)&#160;than cold-grown sfr3 plants. If sfr3 <br /> cence stems sfr3 plants developed warm       simply weaker mutant allele ACC1 gsd1, might <br /> absence developed cold (Fig.&#160;5),   expect severe phenotype present gsd1 unless <br /> the comparative analysis leaf cuticular waxes&#160;(above).             increased malonyl-CoA demand cold substantial. <br />                                                                          Identification sfr3 mutation A. thaliana&#8194; |&#8194; 5297 <br />  <br />                                                                            This work clearly demonstrates critical role plant <br />                                                                         cuticle resisting freezing drought stress. The cuticle <br />                                                                         compromised cold acclimation (when plants are <br />                                                                         held 4&#160;&#176;C), plants sensitive freezing subse- <br />                                                                         quent freezing assays. The primary function cuticle to <br />                                                                         control water loss leaves aerial parts plant <br />                                                                         (Riederer Schreiber, 2001). The clear lack tolerance to <br />                                                                         drought cold-treated sfr3 plants (Fig.&#160;1B) suggests the <br />                                                                         cold-treated mutant inability control loss water. <br />                                                                         This sufficient explain freezing sensitivity of <br />                                                                         sfr3 mutant. The mutant able resist increased <br />                                                                         dehydration stress associated recovery freezing. <br />                                                                            Evidence genes involved cuticular wax deposition <br />                                                                         indicated wax deposition associated regions under- <br />                                                                         going elongation (Xia et&#160;al., 1996)&#160;and biosynthetic flux into <br />                                                                         waxes tightly coordinated surface area expansion (Nawrath, <br />                                                                         2006), i.e. wax deposition occurs leaf expands. Leaves the <br />                                                                         sfr3 mutant developed warm expected to <br />                                                                         normal cuticle development sensitive freezing <br />                                                                         challenge. However, leaves develop cold, after <br />                                                                         transfer cold acclimation conditions, compromised <br />                                                                         cuticle sensitive freezing. The greater sensitivity of <br />                                                                         younger leaves sfr3 plants, mature cold, freez- <br />                                                                         ing damage (Fig.&#160;1A) indicative this, intense <br />                                                                         staining seen younger leaves toluidine blue staining assay <br />                                                                         (Fig.&#160;3). As mature leaves, developed warm <br />                                                                         experienced cold, damaged freezing, and <br />                                                                         particularly drought stress, clear need con- <br />                                                                         tinued wax deposition leaves matured. This asso- <br /> Fig. 7.&#8194; Image showing effects long-term growth 4&#160;&#176;C <br />                                                                         ciated repair damaged cuticle increased deposition <br /> on inflorescence development sfr3 plants. Warm-grown <br />                                                                         occurring transfer cold. Previous studies suggested <br /> 4-week-old plants transferred 4&#160;&#176;C growth conditions <br />                                                                         alterations stomatal density (Aharoni et&#160;al., 2004)&#160;affect the <br /> shortly inflorescence stem initiated maintained <br />                                                                         ability cuticle mutants resist water loss. However, does <br /> at temperature 10 weeks. Wild-type plants (A, left plant) <br />                                                                         appear factor sfr3 plants, differences were <br /> developed normal inflorescence stems supporting flowers and <br />                                                                         detected stomatal density wild-type sfr3 plants <br /> siliques, sfr3 plants developed stunted inflorescence <br />                                                                         grown warm cold acclimated 10 d <br /> stems (A, right plant). Flowers normal appearance developed <br />                                                                         (Supplementary Table S1 JXB&#160;online). <br /> on wild-type plants (B), sfr3 plants abnormal <br />                                                                            The sfr3 mutant provides tool investigations of <br /> (C). Petals shrunken brown did expand, no <br />                                                                         role plant cuticle plant development resistance <br /> siliques observed. <br />                                                                         biotic abiotic stresses. The ability compromise <br />                                                                         function previously effective cuticle incubation of <br />                                                                         intact plants cold facilitate studies cuticle <br />    The cuticular deficiencies associated cold-treated sfr3         function. The ability inactivate (or reduce activity <br /> (Figs 3 4) number mutants defi-        of) ACC1 using treatment allow investigation of <br /> cient cuticular membranes (Chen et&#160;al., 2003; Aharoni et&#160;al., 2004;     role enzyme plant metabolism gene expression. <br /> Schnurr et&#160;al., 2004), gsd1 allele ACC1 (Lu et&#160;al., <br /> 2011). Many mutants display developmental defects that <br /> are present sfr3 plants. Fusion leaf surfaces floral <br />                                                                         Supplementary data <br /> organs associated defective cuticle, been <br /> suggested lack cuticle allowing trans-   Supplementary data available JXB&#160;online. <br /> mission signals normally occur (Nawrath, 2006).          Fig. S1. Mapping sfr3 gene. <br /> The lack abnormalities observed sfr3 plants used             Fig. S2. Complementation sfr3 mutation wild- <br /> freezing tolerance experiments likely product         type ACC1 gene. <br /> cold-induced nature deficiencies present dur-          Fig. S3. Water loss excised rosettes sfr3 wild-type <br /> ing normal growth development plant. The abnor-          plants. <br /> mal flower development infertility following long-term           Fig. S4. Expression levels SFR3 wild-type sfr3 <br /> growth cold (Fig.&#160;7) suggested case pro-    plants cold warm growth conditions. <br /> vides evidence cold-sensitive nature mutation.       Table S1. Stomatal density wild-type sfr3 plants. <br />  5298&#8194; |&#8194;Amid et&#160;al. <br />  <br /> References                                                                Ishitani M, Xiong L, Stevenson B, Zhu JK. 1997. Genetic analysis <br />                                                                           osmotic cold stress signal transduction Arabidopsis: <br /> Aharoni A, Dixit S, Jetter R, Thoenes E, van Arkel G, Pereira             interactions convergence abscisic acid-dependent abscisic <br /> A. 2004. The SHINE clade AP2 domain transcription factors              acid-independent pathways. Plant Cell 9, 1935&#8211;1949. <br /> activates wax biosynthesis alters cuticle properties, confers <br />                                                                           Jacobson BS, Stumpf PK. 1972. Fat metabolism higher plants: <br /> drought tolerance overexpressed Arabidopsis. Plant Cell 16, <br />                                                                           LV. Acetate uptake accumulation <span id='am-98' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-99' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span>&#160;I, class&#160;II chloroplasts <br /> 2463&#8211;2480. <br />                                                                           <span id='am-18' about='obo:NCBITaxon_3561' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_1307796 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1307775 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-19'  ></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Spinacea" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Spinacia" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Spinacia</span> oleracea. Archives Biochemistry Biophysics 153, <br /> Alban C, Baldet P, Douce R. 1994. Localization characterization       656&#8211;663. <br /> of structurally different forms acetyl-CoA carboxylase young <br />                                                                           James DW, Dooner HK. 1990. Isolation EMS-induced mutants <br /> pea leaves, sensitive aryloxyphenoxypropionate <br />                                                                           Arabidopsis altered seed fatty acid composition. Theoretical and <br /> herbicides. Biochemical Journal 300, 557&#8211;565. <br />                                                                           Applied Genetics 80, 241&#8211;245. <br /> Baud S, Bellec Y, Miquel M, Bellini C, Caboche M, Lepiniec L, <br />                                                                           Knight H, Mugford SG, &#220;lker B, Gao D, Thorlby G, Knight MR. <br /> Faure JD, Rochat C. 2004. gurke pasticcino3 mutants affected <br />                                                                           2009. Identification SFR6, key component cold acclimation <br /> in embryo development impaired acetyl-CoA carboxylase. <br />                                                                           acting post-translationally CBF function. The Plant Journal 58, <br /> EMBO Reports 5, 515&#8211;520. <br />                                                                           97&#8211;108. <br /> Baud S, Guyon V, Kronenberger J, Wuilleme S, Miquel M, <br />                                                                           Knight H, Veale EL, Warren GJ, Knight MR. 1999. The sfr6 <br /> Caboche M, Lepiniec L, Rochat C. 2003. Multifunctional <br />                                                                           mutation Arabidopsis suppresses low temperature induction of <br /> acetyl-CoA carboxylase 1 essential long chain fatty acid <br />                                                                           genes dependent CRT/DRE sequence motif. Plant Cell 11, <br /> elongation embryo development Arabidopsis. The Plant Journal <br />                                                                           875&#8211;886. <br /> 33, 75&#8211;86. <br />                                                                           Koncz C, Schell J. 1986. The promoter TL-DNA gene 5 controls <br /> Bell CJ, Ecker JR. 1994. Assignment 30 microsatellite loci      tissue-specific expression chimaeric genes carried novel <br /> linkage map Arabidopsis. Genomics, 19, 137&#8211;144.                        type <span id='am-177' about='obo:NCBITaxon_357' typeof='owl:Thing obo:NCBITaxon_356 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_82115 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_28211 obo:NCBITaxon_227290'><span id='am-178' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasDbXref" content="PMID:13130069" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasDbXref" content="PMID:11760945" datatype="xsd:string"></span><span id='am-181' property="rdfs:label" content="Agrobacterium" datatype="xsd:string"></span><span id='am-182' property="oboInOwl:hasRelatedSynonym" content="Polymonas" datatype="xsd:string"></span><span id='am-183'  ></span><span id='am-184' property="oboInOwl:hasDbXref" content="PMID:8494742" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasDbXref" content="PMID:11211278" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasDbXref" content="PMID:10758890" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasDbXref" content="PMID:13130068" datatype="xsd:string"></span><span id='am-188' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasDbXref" content="PMID:16560572" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasDbXref" content="PMID:8240952" datatype="xsd:string"></span>Agrobacterium</span> binary vector. Molecular General Genetics <br /> Boyce JM, Knight H, Deyholos M, Openshaw MR, Galbraith                    204, 383&#8211;396. <br /> DW, Warren G, Knight MR. 2003. The sfr6 mutant Arabidopsis          Konieczny A, Ausubel FM. 1993. A procedure mapping <br /> defective transcriptional activation CBF/DREB1 DREB2       Arabidopsis mutations using co-dominant ecotype-specific <br /> shows sensitivity osmotic stress. The Plant Journal 34, 395&#8211;406.       PCR-based markers. The Plant Journal 4, 403&#8211;410. <br /> Cangelosi GA, Best EA, Martinetti G, Nester EW. 1991. Genetic             Kosma DK, Bourdenx B, Bernard A, Parsons EP, L&#252; S, Joub&#232;s <br /> analysis Agrobacterium. Methods Enzymology 204, 384&#8211;397.            J, Jenks MA. 2009. The impact water deficiency leaf cuticle <br /> Chen X, Goodwin M, Boroff VL, Liu X, Jenks MA. 2003. Cloning              lipids Arabidopsis. Plant Physiology 151, 1918&#8211;1929. <br /> and characterization WAX2 gene Arabidopsis involved          Llorente F, Oliveros JC, Martinez-Zapater JM, Salinas J. 2000. <br /> cuticle membrane synthesis. Plant Cell 15, 1170&#8211;1185.                     A freezing-sensitive mutant Arabidopsis, frs1, new aba3 allele. <br /> Chinnusamy V, Ohta M, Kanrar S, Lee BH, Hong XH, Agarwal                  Planta 211, 648&#8211;655. <br /> M, Zhu JK. 2003. ICE1: regulator cold-induced transcriptome          Lu SY, Zhao HY, Parsons EP, et&#160;al.. 2011. The glossyhead1 allele <br /> and freezing tolerance Arabidopsis. Genes &amp; Development 17,            ACC1 reveals principal role multidomain acetyl-coenzyme <br /> 1043&#8211;1054.                                                                A&#160;carboxylase biosynthesis cuticular waxes Arabidopsis. <br /> Clough SJ, Bent AS. 1998. Floral dip: simplified method             Plant Physiology 157, 1079&#8211;1092. <br /> Agrobacterium mediated transformation Arabidopsis thaliana. The        McKown R, Kuroki G, Warren G. 1996. Cold responses of <br /> Plant Journal 16, 735&#8211;743.                                                Arabidopsis mutants impaired freezing tolerance. Journal of <br /> Doherty CJ, Van Buskirk HA, Myers SJ, Thomashow MF. 2009.                 Experimental Botany 47, 1919&#8211;1925. <br /> Roles Arabidopsis CAMTA transcription factors cold-regulated       Medina J, Rodriguez-Franco M, Penalosa A, Carrascosa MJ, <br /> gene expression freezing tolerance. Plant Cell 21, 972&#8211;984.           Neuhaus G, Salinas J. 2005. Arabidopsis mutants deregulated in <br /> Fiehn O, Kopka J, D&#246;rmann P, Altmann T, Trethewey RN,                     RCI2A expression reveal new signaling pathways abiotic stress <br /> Willmitzer L. 2000. Metabolite profiling plant functional genomics.   responses. The Plant Journal 42, 586&#8211;597. <br /> Nature Biotechnology 18, 1157&#8211;1161.                                       Miura K, Jin JB, Lee J, Yoo CY, Stirm V, Miura T, Ashworth <br /> Fourrier N, Bedard J, Lopez-Juez E, Barbrook A, Bowyer J,                 EN, Bressan RA, Yun DJ, Hasegawa PM. 2007. SIZ1-mediated <br /> Jarvis P, Warren G, Thorlby G. 2008. A role SENSITIVE              sumoylation ICE1 controls CBF3/DREB1A expression freezing <br /> FREEZING2 protecting chloroplasts freeze-induced damage        tolerance Arabidopsis. Plant Cell 19, 1403&#8211;1414. <br /> in Arabidopsis. The Plant Journal 55, 734&#8211;745.                            Moellering ER, Muthan B, Benning C. 2010. Freezing tolerance in <br /> Gong Z, Lee H, Xiong L, Jagendorf A, Stevenson B, Zhu JK.                 plants requires lipid remodeling outer chloroplast membrane. <br /> 2002. RNA helicase-like protein early regulator transcription    Science 330, 226&#8211;228. <br /> factors plant chilling freezing tolerance. Proceedings     Nawrath C. 2006. Unraveling complex network cuticular <br /> National Academy Sciences, USA 99, 11507&#8211;11512.                        structure function. Current Opinion Plant Biology 9, 281&#8211;287. <br /> Guy CL. 1990. Cold acclimation freezing stress tolerance: role        Nikolau BJ, Ohlrogge JB, Wurtele ES. 2003. Plant <br /> of protein metabolism. Annual Review Plant Physiology Plant        biotin-containing carboxylases. Archives Biochemistry and <br /> Molecular Biology 41, 187&#8211;223.                                            Biophysics 15, 211&#8211;22. <br />                                                                            Identification sfr3 mutation A. thaliana&#8194; |&#8194; 5299 <br />  <br /> Nikolau BJ, Wurtele ES. 1998. In situ transgenic studies           Thorlby G, Fourrier N, Warren G. 2004. The SENSITIVE TO <br /> regulation acetyl-CoA carboxylase gene expression. In: Sanchez         FREEZING2 gene required freezing tolerance Arabidopsis <br /> J, Cerda-Olmedo E, Martinez-Force E, eds, Advances plant lipid         thaliana encodes &#946;-glucosidase. Plant Cell 16, 2192&#8211;2203. <br /> research. Spain: Universidad Sevilla, 50&#8211;53.                           Thorlby G, Veale E, Butcher K, Warren G. 1999. Map positions of <br /> Riederer M, Schreiber L. 2001. Protecting water loss:             SFR genes relation freezing-related genes Arabidopsis <br /> analysis barrier properties plant cuticles. Journal          thaliana. The Plant Journal 17, 445&#8211;452. <br /> Experimental Botany 52, 2023&#8211;2032.                                        Vogel JT, Zarka DG, Van Buskirk HA, Fowler SG, Thomashow <br /> Roesler KR, Shorrosh BS, Ohlrogge JB. 1994. Structure                 MF. 2005. Roles CBF2 ZAT12 transcription factors in <br /> expression Arabidopsis acetyl-coenzyme A&#160;carboxylase gene.          configuring low temperature transcriptome Arabidopsis. The <br /> Plant Physiology 105, 611&#8211;617.                                            Plant Journal 41, 195&#8211;211. <br /> Schnurr J, Shockey J, Browse J. 2004. The acyl-CoA synthetase             Warren G, McKown R, Marin A, Teutonico R. 1996. Isolation <br /> </body></html>