<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                           An Array of <span id='am-172' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-173' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-174' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-176' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-177' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-181'  ></span><span id='am-182' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-183' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-184' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span>Escherichia Coli</span> Clones Over-Expressing Essential <br />                            Proteins: <br />                            A New Strategy of Identifying Cellular Targets of Potent Antibacterial Compounds <br />  <br />                            Abstract <br />                                 With the advancement of high throughput screening, it has become easier and faster to discover hit <br />                                 compounds that inhibit proliferation of bacterial cells. However, development in technologies used <br />                                 to identify cellular targets of potent antibacterial inhibitors has lagged behind. Here we describe a <br />                                 novel strategy of target identification for antibacterial inhibitors using an array of <span id='am-188' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-189' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-190'  ></span><span id='am-191' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-192' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-193' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-194' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span>Escherichia</span> coli <br />                                 clones each over-expressing one essential protein. In a proof-of-concept study, eight essential <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 genes were cloned into pLex5BA vector under the control of an inducible promoter. Over- <br />                                 expression of target proteins was confirmed. For two clones, one over-expressing FabI and the <br />                                 other over-expressing MurA enzymes, the host cells became 17-fold and 139-fold more resistant <br />                                 to the specific inhibitors triclosan and phosphomycin, respectively, while the susceptibility of <br />                                 other clones towards these inhibitors remained unchanged after induction of gene expression. <br />                                 Target identification via target protein over-expression was demonstrated using both mixed clone <br />                                 and individual clone assay formats. <br />  <br />  <br />                            Keywords <br />                                 Escherichia coli; Essential genes; Drug targets; Antibacterial compounds; Over-expression; <br />                                 Triclosan; Phosphomycin <br />  <br />  <br />                            Introduction <br />                                              Over the past 20 years, emergence and dissemination of bacterial resistance to existing <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              antibiotics has increased at an alarming rate [1-3]. Especially troublesome is the emergence <br />                                              of diverse pathogens that became multi-drug resistant [4-7], posing a serious threat to global <br />                                              public health. During <span id='am-147' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-148'  ></span><span id='am-149' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-151' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> same period, few antibiotics of New Chemical Entities (NCEs) <br />                                              have emerged [8], reflecting both the challenges of identifying NCEs and a reduced <br />                                              commitment of antibiotic drug discovery by pharmaceutical companies. One factor <br />                                              contributing to the dismal productivity in this therapeutic area has been the lack of a <br />                                              comprehensive knowledge of bacterial essential genes. Since the first sequencing of a <br />                                              complete microbial genome, <span id='am-84' about='obo:NCBITaxon_2093' typeof='owl:Thing obo:NCBITaxon_2092 obo:NCBITaxon_1 obo:NCBITaxon_2085 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_544448 obo:NCBITaxon_31969'><span id='am-85'  ></span><span id='am-86' property="oboInOwl:hasRelatedSynonym" content="Bovimyces" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="PMID:11931184" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="Mycoplasma" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="GC_ID:4" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="Haemobartonella" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="PMID:15176735" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasDbXref" content="PMID:8995799" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="PMID:8863441" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasRelatedSynonym" content="Pleuropneumonia" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="PMID:13403276" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasDbXref" content="PMID:11411711" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="PMID:10826816" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasRelatedSynonym" content="Eperythrozoon" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasAlternativeId" content="NCBITaxon:29500" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasAlternativeId" content="NCBITaxon:57371" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="PMID:11321109" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasRelatedSynonym" content="Borrelomyces" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasDbXref" content="PMID:16350067" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="PMID:16403858" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="Asterococcus" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasRelatedSynonym" content="Asteromyces" datatype="xsd:string"></span>Mycoplasma</span> genitalium [9], a decade ago, more than 350 <br />                                              bacterial genomes have been completely sequenced, including Escherichia coli [10], <br />                                              <span id='am-46' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-47' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-61'  ></span><span id='am-62' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span>Bacillus subtilis</span> [11] and many important human pathogens. The rapid advances in <br />                                              microbial genomics could potentially lead to renewed efforts and technological <br />  <br />  <br />                            *Corresponding author. Fax: +1 323 343 6451. E-mail address: hxu3@calstatela.edu (H. Howard Xu). <br />                            Publisher&apos;s Disclaimer: This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our <br />                            customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of <br />                            the resulting proof before it is published in its final citable form. Please note that during the production process errors may be <br />                            discovered which could affect the content, and all legal disclaimers that apply to the journal pertain. <br />                             Xu et al.                                                                                                  Page 2 <br />  <br />  <br />                                            breakthroughs in antibiotic drug discovery to meet the urgent medical need of new <br />                                            antibiotics [12]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Currently available antibiotics target approximately 20 of the estimated 200 essential gene <br />                                            products in bacteria [13]. The recognition of the utilities of untapped potential antibiotic <br />                                            targets has fostered various innovative approaches for bacterial essential gene identification, <br />                                            facilitating the development of a new discipline&#8212;bacterial functional genomics. <br />                                            Comparative genomics has been combined with conditional expression to identify essential <br />                                            genes in E. coli [14,15]. Promoter replacement techniques have been developed to assess the <br />                                            essentiality of <span id='am-1' about='obo:NCBITaxon_1279' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1385 obo:NCBITaxon_90964 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-2' property="oboInOwl:hasDbXref" content="PMID:12656157" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="PMID:10319495" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="PMID:10758876" datatype="xsd:string"></span><span id='am-5'  ></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="PMID:10319469" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Aurococcus" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="PMID:10425778" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="PMID:17220435" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="PMID:9734063" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Staphylococcus" datatype="xsd:string"></span>Staphylococcus</span> aureus genes [16,17]. Massive transposon mutagenesis has <br />                                            been applied to identify essential genes on genome-scale in M. genitalium [18], <br />                                            <span id='am-155' about='obo:NCBITaxon_727' typeof='owl:Thing obo:NCBITaxon_135625 obo:NCBITaxon_712 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_724'><span id='am-156' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium influenzae" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasRelatedSynonym" content="Influenza-bacillus" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:hasRelatedSynonym" content="Coccobacillus pfeifferi" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasRelatedSynonym" content="Haemophilus meningitidis" datatype="xsd:string"></span><span id='am-161' property="rdfs:label" content="Haemophilus influenzae" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasRelatedSynonym" content="Bacterium influenzae" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-164'  ></span>Haemophilus influenzae</span> [19,20], and E. coli [21]. Gene disruption methodologies have been <br />                                            employed to systematically identify essential genes in <span id='am-28' about='obo:NCBITaxon_1301' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-29' property="oboInOwl:hasDbXref" content="PMID:1720654" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="PMID:14657115" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Streptococcus" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="PMID:8995803" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="PMID:10555340" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="PMID:7537076" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="oboInOwl:hasDbXref" content="PMID:19620365" datatype="xsd:string"></span>Streptococcus</span> pneumoniae [22] and <br />                                            B. subtilis [23]. Finally, regulated antisense expression [24,25] has been utilized to identify a <br />                                            comprehensive set of essential genes in S. aureus. Conjoint analysis of several <br />                                            computational models and experimental strategies has led to the proposition that, from a <br />                                            standpoint of cellular processes necessary to maintain cellular life, a minimal bacterial <br />                                            essential set contains 206 genes [13]. Bacterial essential genes thus far identified have been <br />                                            increasingly used in either cell-based or biochemical drug screens [26-28]. <br />  <br />                                            Historically, the most productive paradigm for antibacterial lead discovery involved a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            straightforward initial screen for the inhibition of bacterial growth. This was followed by <br />                                            laborious experimental attempts to identify drug targets and establish mechanisms of action <br />                                            for those discovered antibacterial compounds. Once and if the target of the hit compound <br />                                            was identified, antibacterial potency and selectivity were then optimized through the <br />                                            synthesis and testing of structural analogs of the initial hit compound. With increased <br />                                            exploitation of chemical space (in which distance between points approximates compound <br />                                            similarity), it is still simple, fast and straightforward to identify inhibitors of bacterial <br />                                            growth. What remains difficult is to experimentally link the inhibitor to the cellular target. <br />                                            There is therefore a need for development of fast and comprehensive assay systems to <br />                                            identify cellular targets of potent antibacterial inhibitors. Li and colleagues [29] recently <br />                                            reported a strategy of identifying targets of antibacterial inhibitors via suppression of <br />                                            inhibition by multicopy plasmid clones of an E. coli random genomic library. While this <br />                                            approach does not require large scale cloning efforts, it would be difficult to judge the <br />                                            comprehensiveness of the essential gene coverage in the clones of the genomic library <br />                                            without detailed validation experiments. In this report, we describe a novel strategy to <br />                                            identify the target of an inhibitor via a comprehensive collection (array) of E. coli clones <br />                                            each engineered to over-express one essential protein. This strategy is based on the concept <br />                                            that elevated levels of the drug target protein confer resistance because higher <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            concentrations of a specific inhibitor are required to bind to the excess target in <span id='am-152' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-153' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-154' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> to <br />                                            inhibit bacterial growth [30]. In this proof-of-concept report, we confirmed elevated <br />                                            inhibitor-specific resistance of E. coli strains over-expressing specific essential targets and <br />                                            demonstrated identification of cellular targets for inhibitors using either a culture of mixed <br />                                            clones or parallel growth of individual clones each over-expressing one particular essential <br />                                            gene. If this strategy is applicable to a large number of essential genes in E. coli, it would <br />                                            become a very useful assay tool to identify cellular targets of inhibitors of unknown <br />                                            mechanisms of action. <br />  <br />                               Materials and Methods <br />                               Bacterial strains, plasmid and materials <br />                                            Purchased from ATCC (Manassas, VA), E. coli K-12 strain MG1655 (Genotype: F- lambda- <br />                                            ilvG- rfb-50 rph-1) was used as the source of chromosomal DNA with known genomic <br />  <br />  <br />                                        Biochem Biophys Res Commun. Author manuscript; available in PMC 2011 September 19. <br />                             Xu et al.                                                                                                Page 3 <br />  <br />  <br />                                            sequence [10] from which essential genes were PCR amplified and cloned. E. coli K-12 <br />                                            DH5&#945; was obtained from Invitrogen (Genotype: F-&#934;80&#916;lacZ&#916;M15 &#916;(lacZYA-&#945;&#961;&#947;F) U169 <br />                                            recA1 endA1 hsdR17 (rk-, mk+) phoA supE44 &#955;-thi-1 gyrA96 relA1). Expression vector <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            pLex5BA [31] was used to clone and express essential genes under control of the inducer <br />                                            isopropyl &#946;-D-thiogalactopyranoside (IPTG). Restriction enzymes and T4 DNA ligase were <br />                                            purchased from Invitrogen (Carlsbad, CA). Antibiotics and inhibitors (ampicillin, <br />                                            phosphomycin and triclosan), IPTG and dithiothreitol (DTT) were purchased from Sigma- <br />                                            Aldrich (St. Louis, MO). Phenylmethylsulfonyl fluoride (PMSF) was obtained from Pierce <br />                                            (Rockford, IL). Difco <span id='am-79' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-80' property="rdfs:label" content="Luria" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-83'  ></span>Luria</span>-Bertani (LB) broth powder was purchased from Fisher <br />                                            Scientific (Tustin, CA). E. coli cells were grown in LB broth or agar media under <br />                                            appropriate antibiotic selection when necessary (100 &#956;g/ml of ampicillin). <br />  <br />                               Gene cloning <br />                                            E. coli essential genes were PCR amplified using E. coli K-12 MG1655 genomic DNA as <br />                                            the template and cloned into the plasmid pLex5BA [31]. Essential genes were selected from <br />                                            the PEC (Profiling of E. coli Chromosome) database. PCR primers were designed to include <br />                                            the complete open reading frame (ORF) of each gene and its putative Shine-Delgarno <br />                                            sequence in the amplified product. Appropriate restriction endonuclease recognition sites <br />                                            were designed into PCR primers to facilitate the cloning process. Genes were directionally <br />                                            cloned downstream of an IPTG-inducible promoter present on the plasmid [31]. <br />                                            Recombinant plasmid clones were initially validated via restriction digest and agarose gel <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            electrophoresis. One clone for each gene was confirmed via DNA sequencing using an ABI <br />                                            Big-Dye terminator Kit 3.1 and the ABI 3100 Avant Genetic Analyzer (ABI, Foster City, <br />                                            CA). <br />  <br />                               Over-expression of cloned essential proteins in E. coli <br />                                            Based on a procedure from [32] with modifications, cell-free protein extracts were obtained <br />                                            from E. coli cells of various clones under appropriate inducing conditions. Briefly, an E. coli <br />                                            strain harboring a plasmid with an inserted essential gene was streaked on an LB agar plate <br />                                            with 100 &#956;g/ml of ampicillin and grown overnight at 37 &#176;C. An Erlenmeyer flask containing <br />                                            50 ml LB medium (with 100 &#956;g/ml ampicillin) was then inoculated with 4 colonies from the <br />                                            plate and grown at 37 &#176;C until the culture reached an OD600 of 0.5. Expression of the <br />                                            essential gene was induced by the addition of IPTG to a final concentration of 1 mM and <br />                                            continued growth of the culture overnight (approximately 17 h) at 37 &#176;C. At either 4 h after <br />                                            induction or after overnight induction ten milliliters of the culture were harvested and <br />                                            resuspended in 0.455 ml of phosphate buffered saline (PBS: 137 mM NaCl; 2.7 mM KCl; 10 <br />                                            mM Na2HPO4; 2 mM KH2PO4). Lysozyme was added to a final concentration of 1 mg/ml <br />                                            and the suspension was incubated on ice for 60 min. One milliliter of 0.2% Triton X-100 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            was added, the suspension was vigorously mixed, and DNase and RNase enzymes were each <br />                                            added to a final concentration of 5 &#956;g/ml. The mixture was gently mixed for 20 min at 4 &#176;C <br />                                            by rocking, followed by centrifugation at 3,500 &#215; g for 30 min at 4 &#176;C. The supernatant was <br />                                            collected and placed into a fresh tube and DTT and PMSF were added to a final <br />                                            concentration of 1 mM. Glycerol was then to a final concentration of 10%. The cell-free <br />                                            protein extract was either stored at -20 &#176;C or subjected to polyacrylamide gel electrophoresis <br />                                            analysis using precast protein gels from Pierce (Rockford, IL). Following electrophoresis the <br />                                            protein gels were stained with Bio-Safe Coomassie blue from Bio-Rad (Hercules, CA). <br />  <br />                               Determination of inducer concentrations on growth of E. coli clones <br />                                            A series of IPTG dilutions were combined, in wells of a Corning Costar flat-bottom 96-well <br />                                            microplate, with LB broth plus ampicillin inoculated with cells of E. coli expression clones <br />                                            (to an initial inoculation OD600 of 0.001). The plate was sealed with clear thermal sealing <br />                                            film (E&amp; K Scientific, Campbell, CA) and growth of each well at 37&#176;C was monitored <br />  <br />                                        Biochem Biophys Res Commun. Author manuscript; available in PMC 2011 September 19. <br />                             Xu et al.                                                                                                    Page 4 <br />  <br />  <br />                                             kinetically by Tecan Genios plate reader equipped with an incubation chamber (Tecan, <br />                                             Durham, NC). Plates were read every 30 min for approximately 18 h with low speed shaking <br />                                             prior to every read. E. coli cell growth curves and optical density comparison charts were <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             generated by GraphPad Prizm software (San Diego, CA). <br />  <br />                               Drug dose response and resistance testing <br />                                             Individual E. coli clones carrying <span id='am-200' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-201' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-202' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-203' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span> containing essential genes were grown in LB <br />                                             medium with 100 &#956;g/ml ampicillin for maintenance of the plasmid, in the presence of serial <br />                                             dilutions of specific inhibitors available, either with or without the inducer IPTG (at final <br />                                             concentration of 1 mM). The growth experiments were performed as described above using <br />                                             Corning Costar flat-bottom 96-well microplates and Tecan Genios plate reader. Inhibitor <br />                                             dose response curves were generated by GraphPad Prizm software. <br />  <br />                               Target identification using mixed clone assay format <br />                                             Eight E. coli over-expressing clones were mixed and grown in concentrations of either <br />                                             triclosan or phosphomycin at levels that inhibit the parental strain under inducing or <br />                                             noninducing conditions. Cells grown from two treatment conditions (8 &#956;g/ml triclosan and 2 <br />                                             &#956;g/ml phosphomycin) in the presence of 1 mM IPTG were diluted and plated on selectable <br />                                             media to obtain isolated colonies. Plasmid DNAs were isolated and restriction digest was <br />                                             performed to determine the identities of insert DNAs. Sequence confirmation of the plasmid <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             inserts was carried out as described above. <br />  <br />                               Target identification using individual clone assay format <br />                                             To test if cellular targets of antibacterial inhibitors can also be identified using individually <br />                                             grown clones, over-expressing clones were arrayed and grown in separate microplate rows <br />                                             under various conditions. Some wells of the microplate contained triclosan (4 &#956;g/ml), others <br />                                             contained phosphomycin (2 &#956;g/ml). Some wells contained no antibacterial inhibitors to <br />                                             serve as innoculum control. Clones were grown either with or without IPTG. Cell growth in <br />                                             the microplate was monitored in Tecan Genios plate reader as describe above. <br />  <br />                               Results <br />                               Gene cloning <br />                                             PCR-amplified DNA fragments for eight E. coli essential genes (Table 1) were cloned into <br />                                             pLex5BA and transformed into E. coli DH5&#945; host cells. The length and identity of insert <br />                                             DNA from recombinant plasmids were confirmed via restriction digest analysis (Fig. 1) and <br />                                             DNA sequencing (<span id='am-195' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-196' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-197' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-198'  ></span><span id='am-199' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> not shown). These eight essential genes were chosen to demonstrate <br />                                             a proof of concept for our strategy of target identification via target protein over-expression <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             because they encode a variety of essential target proteins involved in several distinct cellular <br />                                             functions or pathways. MurA and MurG are enzymes involved in bacterial peptidoglycan <br />                                             (cell wall) biosynthesis and are critical for maintaining cellular integrity. FabD and FabI are <br />                                             essential enzymes involved in fatty acid biosynthesis. Tryptophanyl tRNA synthetase <br />                                             (TrpS), peptide chain release factor 2 (PrfB), 30S ribosomal protein S12 and 50S ribosomal <br />                                             protein L10 are involved in different facets of protein synthesis. Conveniently, triclosan and <br />                                             phosphomycin are known antibacterial inhibitors specific for FabI and MurA enzymes, <br />                                             respectively, and are available commercially to test our strategy. <br />  <br />                               Protein over-expression <br />                                             To confirm over-expression of cloned essential proteins, clones were grown to stationary <br />                                             phase and induction was initiated by adding IPTG to a final concentration of 1 mM. Cell <br />                                             culture samples were taken at either 4 h or 17 h (overnight) after induction. Examples of <br />  <br />  <br />                                         Biochem Biophys Res Commun. Author manuscript; available in PMC 2011 September 19. <br />                             Xu et al.                                                                                                Page 5 <br />  <br />  <br />                                            protein over-expression are shown in Figure 2. Results indicated that for FabI, MurA and <br />                                            TrpS clones, there was significant over-expression of essential proteins at 4 h after induction <br />                                            and that continued induction beyond 4 h did not appear to further increase the protein <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            expression. Specifically, there were protein bands of similar intensity present in both 4 hr <br />                                            and overnight post-induction samples of a MurA clone that match the calculated molecular <br />                                            weight (44.8 kD) of MurA protein (Fig. 2). Similarly, cells of a TrpS clone produced an <br />                                            obviously over-expressed polypeptide band that corresponds to TrpS enzyme&#8217;s calculated <br />                                            molecular weight (37.4kD) (Fig. 2). In contrast, over-expression of a FabI clone resulted in a <br />                                            protein band corresponding to a molecular weight of around 33 kDalton (Fig. 2), which is <br />                                            inconsistent with its calculated molecular weight of 28 kD. This type of discrepancies <br />                                            between calculated molecular weight of a polypeptide and electrophoretic outcome is not <br />                                            unusual. Interestingly, Bergler and coworkers over-expressed the same protein (FabI, used <br />                                            to be called EnvM) from E. coli and had obtained a similar result [33]. This led us to believe <br />                                            that the observed over-exppressed protein from FabI clone belonged to FabI enzyme. Of <br />                                            eight essential gene clones tested, all except PrfB clone demonstrated detectable over- <br />                                            expression by protein gel electrophoresis of cell-free extracts (data not shown). <br />  <br />                               Effects of induction on cell growth <br />                                            To determine whether or not the IPTG inducer of the over-expression of these essential <br />                                            proteins caused adverse effects on growth of these clones (since robust growth from each <br />                                            clone was critical for target identification assays), individual clones were grown in the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            presence of a series of IPTG concentrations between 0.01 mM to 10 mM. Our results <br />                                            indicated that inducer concentrations as high as 10 mM (ten times as high as used in our <br />                                            experiments) did not cause significant growth defects on TrpS, RplJ, FabI and MurA clones <br />                                            (Fig. 3). Similar results were obtained with FabD, RpsL, PrfB, MurG and vector clones (data <br />                                            not shown) <br />  <br />                               Drug dose response and resistance testing <br />                                            To confirm the concept that target over-expression increases resistance to specific inhibitors, <br />                                            a FabI clone was grown in various concentrations of triclosan (the specific inhibitor of the <br />                                            FabI enzyme) in the LB medium with ampicillin in the presence and absence of 1 mM <br />                                            IPTG. Triclosan dose response curves for the FabI clone exhibited a significant shift in the <br />                                            presence of the inducer as compared to that in the absence of an inducer (Fig. 4). The IC50 <br />                                            values increased from 584 ng/ml to 9,927 ng/ml, a 17-fold increase in resistance, when over- <br />                                            expression of FabI was induced. In contrast, as expected since the vector contains no insert, <br />                                            the IC50 values for the vector clone did not change at all (Fig. 4). When a MurA clone was <br />                                            grown in various concentrations of phosphomycin, the specific inhibitor of the MurA <br />                                            enzyme, the increase in resistance was even more dramatic (a 139-fold change in IC50 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            values as shown in Fig. 5). Interestingly, the elevated resistance conferred by induction of <br />                                            the target expression did not require the presence of ampicillin in the medium (Fig. 6). <br />                                            Without ampicillin in the LB medium, cells of MurA clone grew fairly well in the presence <br />                                            of 8 &#956;g/ml of phosphomycin under the condition of induction (Fig. 6). In the absence of <br />                                            IPTG, cells failed to grow in 8 &#956;g/ml of phosphomycin. Similar growth profiles were <br />                                            observed for the FabI clone in the absence of ampicillin (data not shown). <br />  <br />                                            To determine that the increased resistance was specific for target and inhibitor pairs <br />                                            involved, IC50 values were obtained from dose response curves for each E. coli clone treated <br />                                            with triclosan and phosphomycin both in the presence and absence of an inducer. The ratio <br />                                            of IC50 induced vs IC50 non-induced was used as a measure of changes in a clone&#8217;s <br />                                            susceptibility toward a particular inhibitor. Results indicated that elevated resistance to <br />                                            triclosan was observed only in the FabI clone, but not in any of the other seven over- <br />                                            expression clones (Fig. 7). Similarly, elevated phosphomycin resistance was only evident in <br />  <br />  <br />                                        Biochem Biophys Res Commun. Author manuscript; available in PMC 2011 September 19. <br />                             Xu et al.                                                                                                 Page 6 <br />  <br />  <br />                                            the MurA clone but not in the other seven clones (Fig. 7). Even over-expression of enzymes <br />                                            (FabD and MurG) in the same respective pathways did not cause increases in resistance to <br />                                            triclosan and phosphomycin, respectively (Fig. 7). These results indicate that the inhibitor <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            resistance as a result of target protein over-expression was highly specific. <br />  <br />                               Target identification using mixed clone assay <br />                                            The specificity nature of the enhanced resistance due to target over-expression became the <br />                                            scientific basis for our strategy of identifying cellular targets using a collection of E. coli <br />                                            clones each over-expressing a particular target protein. When eight over-expressing clones <br />                                            were mixed and subjected to treatment by inhibitory (to the parental strain) concentrations of <br />                                            triclosan and phosphomycin, respectively, cell growth was observed in the presence of 1 <br />                                            mM IPTG in both treatments over a range of inhibitor concentrations (data not shown). We <br />                                            suspected that the cell growth observed was due to the enhanced resistance of the one clone <br />                                            that was over-expressing a specific target protein. To test this, cultures with growth from the <br />                                            wells containing 2 &#956;g/ml of phosphomycin and 8 &#956;g/ml of triclosan were serially diluted and <br />                                            plated out on LB agar plates with 100 &#956;g/ml of ampicillin. Isolated colonies from each <br />                                            inhibitor treatment were grown and plasmids were isolated to perform restriction digest <br />                                            analysis. Restriction analysis results indicated that 100% of the colonies from phosphomycin <br />                                            treatment contained murA gene inserts and 100% of the colonies from triclosan treatment <br />                                            contained fabI gene inserts (Fig. 8a and 8b). Subsequent plasmid DNA sequencing further <br />                                            confirmed that the growth was indeed the result of the clones over-producing the specific <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            target proteins to which the specific inhibitors binds (data not shown). <br />  <br />                               Target identification using individual clone assay <br />                                            To test whether or not treatment by an inhibitor of individually grown over-expression <br />                                            clones also could identify the target of the inhibitor, eight over-expression clones were <br />                                            individually grown in the presence or absence of inhibitors triclosan (4 &#956;g/ml) and <br />                                            phosphomycin (2 &#956;g/ml), with or without 1 mM IPTG (Fig. 9). As can be seen from Figure <br />                                            9, for those wells without inhibitors, cells grew normally with or without inducer as <br />                                            expected (used as inoculum control). Microplate wells G5 and G6, the only two wells <br />                                            containing the FabI clone with both triclosan and IPTG present, produced robust growth <br />                                            similar to the wells of the inoculum control (Fig. 9). For wells treated with phosphomycin, <br />                                            only two wells (F1 and F2) from the MurA clone row containing both phosphomycin and <br />                                            the inducer grew normally. <br />  <br />                               Discussion <br />                                            Recognition of the mechanism of antibiotic resistance due to drug target over-expression <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            was first reported decades ago [34]. This line of research has received little attention over <br />                                            the years, probably because the earlier examples were laboratory phenomena with no <br />                                            clinical relevance [34,35]. More recently, however, it has been reported that target over- <br />                                            expression probably accounts for acquired resistance to isoniazid and ethambutol in clinical <br />                                            isolates of mycobacteria [36,37]. In these studies, over-expression of targets through <br />                                            promoter mutations appeared to be responsible for resistance in drug resistant clinical <br />                                            isolates. In a laboratory setting, bacterial target proteins have been over-expressed either to <br />                                            study the function of the enzymes or to examine the interaction of inhibitors and target <br />                                            proteins. Over-expression of prolipoprotein signal peptidase (gene product of lsp) was <br />                                            shown to confer its host cells a high level of resistance to globomycin, an inhibitor of <br />                                            lipoprotein signal peptidase [38]. Turnowsky and colleagues [39] demonstrated that over- <br />                                            expression of EnvM (FabI) protein from the plasmid-borne gene conferred resistance to <br />                                            diazaborine, a specific inhibitor for FabI. Tsay and coworkers [40] screened a <br />                                            thiolactomycin sensitive E. coli strain against an E. coli genomic library for recombinant <br />  <br />  <br />                                        Biochem Biophys Res Commun. Author manuscript; available in PMC 2011 September 19. <br />                             Xu et al.                                                                                                  Page 7 <br />  <br />  <br />                                            plasmids that conferred resistance to thiolactomycin, an inhibitor for FabB protein (&#946;- <br />                                            ketoacyl-acyl carrier protein synthase I). They isolated nine plasmids, all of which contain a <br />                                            functional fabB gene. Over-expression of FabB protein appeared to lead to increased <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            resistance to thiolactomycin in E. coli [40]. Apfel and colleagues [41] demonstrated that <br />                                            over-expression of peptide deformylase in a cell reduced its susceptibility to inhibitors <br />                                            specific for the essential target, peptide deformylase. <br />  <br />                                            Recently, there have been several reports of using target over-expression to confirm targets <br />                                            of drugs or inhibitors in a number of microorganisms. Over-expression of farnesyl <br />                                            diphosphate synthase (FDP synthase) in social <span id='am-137' about='obo:NCBITaxon_5774' typeof='owl:Thing obo:NCBITaxon_555370 obo:NCBITaxon_1 obo:NCBITaxon_180229 obo:NCBITaxon_131567 obo:NCBITaxon_35218 obo:NCBITaxon_555369 obo:NCBITaxon_2759 obo:NCBITaxon_554915'><span id='am-138' property="rdfs:label" content="Amoeba" datatype="xsd:string"></span><span id='am-139'  ></span><span id='am-140' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>amoeba</span> <span id='am-142' about='obo:NCBITaxon_5782' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_142796 obo:NCBITaxon_33083 obo:NCBITaxon_2759 obo:NCBITaxon_554915'><span id='am-143' property="rdfs:label" content="Dictyostelium" datatype="xsd:string"></span><span id='am-144'  ></span><span id='am-145' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Dictyostelium</span> resulted in resistance <br />                                            to aminobisphosphate inhibitors [42]. Butcher and coworkers [43] constructed a pool of <br />                                            <span id='am-113' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-114' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-122'  ></span><span id='am-123' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> strains each over-expressing a different protein. These strains <br />                                            were grown in the presence of rapamycin and the growth was monitored by microarray. <br />                                            Candidate target proteins of rapamycin were identified [43]. While the approaches by these <br />                                            groups [42,43] are valuable and robust, they are limited only for use in identification of <br />                                            targets of antifungal compounds. In a recent report using a bacterial system (E. coli), <br />                                            multicopy suppressors were identified for antibacterial compounds as a method of <br />                                            determining the targets of the inhibitors and the mechanisms of resistance [29]. In this study <br />                                            [29], an E. coli random genomic library was screened for clones which could suppress <br />                                            antibacterial activities of inhibitors to a hyperpermeable strain of E. coli. As a result, the <br />                                            majority of suppressors were found to encode the multidrug efflux pump AcrB and a target <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            of two lead compounds was identified as dihydrofolate reductase (DHFR), encoded by the <br />                                            folA gene [29]. This approach of screening genomic libraries for suppressor clones is a <br />                                            significant method of identifying targets of antibacterial inhibitors. However, due to the <br />                                            random nature of genomic libraries and of cloned inserts, it is difficult to determine if an <br />                                            essential gene is structurally intact or functionally expressed in a plasmid clone without <br />                                            extensive validation experiments. <br />  <br />                                            Here we report on a first deliberate effort to clone individual essential genes from E. coli for <br />                                            the sole purpose of constructing an array of E. coli over-expression clones to identify <br />                                            cellular targets of potent antibacterial inhibitors generated from high throughput screening <br />                                            campaigns. As a proof-of-concept, we cloned eight essential genes in E. coli, observed <br />                                            essential protein over-expression in majority of the clones (seven out of eight) and <br />                                            confirmed elevated resistance of clones to their specific antibacterial inhibitors. We <br />                                            demonstrated proof of concept in identification of targets of two known inhibitors. Our <br />                                            results suggest that the approach is very likely a generic one and can provide a viable tool <br />                                            for identification of cellular targets of potent inhibitors and for studies of drug-target <br />                                            interactions. Additionally, with each essential gene clone sequence validated and over- <br />                                            expression demonstrated, we have the choices of two complimentary and corroborating <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            assay formats for the identification of targets of potent inhibitors: the mixed clone method <br />                                            and the individual clone method. Both assay formats have their own advantages. The mixed <br />                                            clone method streamlines the number of growth vessels (microplate wells) used and may <br />                                            amplify even slightest fitness advantages of certain clones in a fitness test environment. The <br />                                            individually arrayed method is advantageous in that the well positions of every clone are <br />                                            known in advance, which enables a direct linkage of growth of a particular well to a target <br />                                            without genetic profiling of clones. In addition, both methods are amenable to robotic <br />                                            adaptation to a high throughput screening setting involving a large number of over- <br />                                            expressing clones and multiple potent antibacterial compounds. Once completed, this array <br />                                            of over-expression E. coli clones and the associated assays would facilitate discovery of <br />                                            novel antibiotics. <br />  <br />  <br />  <br />  <br />                                        Biochem Biophys Res Commun. Author manuscript; available in PMC 2011 September 19. <br />  <br />  <br />                                            Figure 1. <br />                                            Agarose gel stained with ethidium bromide after electrophoresis. Lane 1, DNA ladder; <br />                                            Lanes 2-10, various plasmids digested with EcoR I and BamH I: lane 2, vector pLex5BA; <br />                                            lane 3, pLex5BA-fabD; lane 4, pLex5BA-fabI; lane 5, pLex5BA-murA; lane 6, pLex5BA- <br />                                            murG; lane 7, pLex5BA-prfB; lane 8, pLex5BA-rplJ; lane 9, pLex5BA-rpsL; lane 10, <br />                                            pLex5BA-trpS. Note, due to restriction sites and Shine-Delgarno sequences designed into <br />                                            PCR primers, the sizes of inserts shown in the gel are slightly bigger than the ORF sizes <br />                                            listed in Table 1. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Biochem Biophys Res Commun. Author manuscript; available in PMC 2011 September 19. <br />                             Xu et al.                                                                                                Page 12 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 2. <br />                                            Acrylamid gel electrophoresis separation of protein extracts after induction of essential <br />                                            protein expression by IPTG. Protein extracts from cell cultures of FabI clone (lanes 1-3), <br />                                            MurA clone (4-6) and TrpS clone (7-9) before adding inducer (1 mM IPTG) (lanes 1, 4, 7), <br />                                            4 h after induction (lanes 2, 5, 8) and 17 h after induction (lanes 3, 6, 9). Lane 10, protein <br />                                            ladder. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Biochem Biophys Res Commun. Author manuscript; available in PMC 2011 September 19. <br />                             Xu et al.                                                                                             Page 13 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 3. <br />                                            Effect of inducer concentrations on the growth of four E. coli clones (TrpS, RplJ, FabI and <br />                                            MurA). Cell growth in the wells of a microplate was monitored kinetically using a plate <br />                                            reader. For each concentration of inducer for every clone, three replicate wells were used. <br />                                            Late log-phase (10 h) plate reading values were used to generate the bar graph. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Biochem Biophys Res Commun. Author manuscript; available in PMC 2011 September 19. <br />                             Xu et al.                                                                                              Page 14 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 4. <br />                                            Increased resistance to triclosan via over-expression of inhibitor-specific essential gene <br />                                            product, FabI. The FabI clone was grown in presence of serial dilutions of triclosan either <br />                                            with or without 1 mM IPTG. A vector control clone containing vector alone served as a <br />                                            control. For each condition (in respect to inhibitor and inducer), three replicate wells were <br />                                            used. Late log-phase (10 h) plate read values were used to generate the dose response graph. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Biochem Biophys Res Commun. Author manuscript; available in PMC 2011 September 19. <br />                             Xu et al.                                                                                           Page 15 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 5. <br />                                            Increased resistance to phosphomycin via over-expression of inhibitor-specific essential <br />                                            gene product MurA. The MurA clone was grown in presence of serial dilutions of <br />                                            phosphomycin either with or without 1 mM IPTG. For each condition (in respect to inhibitor <br />                                            and inducer), three replicate wells were used. Late log-phase (10 h) plate read values were <br />                                            used to generate the dose response graph. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Biochem Biophys Res Commun. Author manuscript; available in PMC 2011 September 19. <br />                             Xu et al.                                                                                           Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 6. <br />                                            Growth profiles for cells of MurA clone in the absence of ampicillin. Cells were grown in <br />                                            LB media without ampicillin in wells of a microplate in the presence or absence of IPTG <br />                                            and/or phosphomycin. Growth was monitored kinetically in a plate reader at 37 &#176;C for 17.5 <br />                                            h, with optical density reading every 0.5 h. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Biochem Biophys Res Commun. Author manuscript; available in PMC 2011 September 19. <br />                             Xu et al.                                                                                             Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 7. <br />                                            Specificity of elevated resistance conferred by target protein over-expression. Each of the <br />                                            eight clones plus the vector clone was treated with a series of concentrations of either <br />                                            triclosan or phosphomycin in the presence or absence of the inducer (1 mM IPTG). Dose <br />                                            response curves for each clone were plotted for either induced or noninduced conditions. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            IC50 values were obtained. The ratio of IC50 values (IC50 induced vs IC50 noninduced) was <br />                                            used as a measure of fold increase in resistance. <br />  <br />  <br />  <br />  <br />                                        Biochem Biophys Res Commun. Author manuscript; available in PMC 2011 September 19. <br />                             Xu et al.                                                                                              Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 8. <br />                                            Restriction digest analysis of isolated plasmids from colonies obtained from mixed strain <br />                                            treatment with either triclosan (4 &#956;g/ml) (A) or phosphomycin (2 &#956;g/ml) (B). A, fabI gene <br />                                            contains an internal Pst I site. EcoR I/Pst I double digest of pLex5BA (which also contains a <br />                                            Pst I site in the polylinker) containing cloned fabI gene would produce two small fragments <br />                                            (620 and 231 bps) in additional to the vector band. B, murA gene contains an internal Hind <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            III site. EcoR I/Hind III double digest of pLex5BA (which also contains a Hind III site in the <br />                                            polylinker) containing cloned murA gene would produce two fragments (841 and 465 bps) in <br />                                            addition to the vector band. <br />  <br />  <br />  <br />  <br />                                        Biochem Biophys Res Commun. Author manuscript; available in PMC 2011 September 19. <br />                             Xu et al.                                                                                             Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 9. <br />                                            A kinetic growth image of a microplate-based target identification assay. The wells of each <br />                                            row contained one individual clone (marked on the right). Columns 1 to 4 contained 2 &#956;g/ml <br />                                            of phosphomycin. Columns 5 to 8 contained 8 &#956;g/ml of triclosan. Columns 9 to 12 contained <br />                                            no inhibitor to serve as cell growth control. For each inhibitor treatment, half of the wells <br />                                            also contained 1 mM IPTG while the other half did not (shown at the bottom). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Biochem Biophys Res Commun. Author manuscript; available in PMC 2011 September 19. <br />                             Xu et al.                                                                                                             Page 20 <br />  <br />  <br />  <br />  <br />                                                                                       Table 1 <br />                            E. coli essential genes cloned. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Gene Name   Gene Product (Target)                                                     Gene Size (base   E. coli Clone Name <br />                                                                                                                   pairs) <br />  <br />                             fabD        Malonyl CoA-ACP transacylase                                              930               FabD clone <br />  <br />                             fabI        Enol-acyl carrier protein reductase                                       789               FabI clone <br />  <br />                             murA        UDP-N-acetylglucosamine enolpyruvyl transferase                           1260              MurA clone <br />  <br />                             murG        UDP-N-acetylglucosamine--N-acetylmuramyl-(pentapeptide) pyrophosphoryl-   1068              MurG clone <br />                                         undecaprenol N-acetylglucosamine transferase <br />  <br />                             prfB        Peptide chain release factor 2                                            1023              PrfB clone <br />  <br />                             rplJ        50S ribosomal protein L10                                                 498               RplJ clone <br />  <br />                             rpsL        30S ribosomal protein S12                                                 375               RpsL clone <br />                             trpS        Tryptophanyl tRNA synthetase                                              1005              TrpS clone <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Biochem Biophys Res Commun. Author manuscript; available in PMC 2011 September 19. <br />   <br /> </body></html>