<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> ADP Ribosylation Factor Is an Essential Protein in <span id='am-24' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-25' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-30'  ></span>Saccharomyces</span> <br />             cerevisiae and Is Encoded by Two Genes <br />  <br />             ADP ribosylation factor (ARF) is a ubiquitous 21-kDa GTP-binding protein in eucaryotes. ARF was first <br />          identified in animal cells as the protein factor required for the efficient ADP-ribosylation of the mammalian G <br />          protein G. by cholera toxin in vitro. A gene (ARFI) encoding a protein homologous to mammalian ARF was <br />          recently cloned from <span id='am-7' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> (Sewell and Kahn, Proc. Natl. Acad. Sci. USA, 85:4620-4624, <br />          1988). We have found a second gene encoding ARF in S. cerevisiae, ARF2. The two ARF genes are within 28 <br />          centimorgans of each other on chromosome IV, and the proteins encoded by them are 96% identical. <br />          Disruption of ARFI causes slow growth, cold sensitivity, and sensitivity to normally sublethal concentrations <br />          of fluoride ion in the medium. Disruption of ARF2 causes no detectable phenotype. Disruption of both genes <br />          is lethal; thus, ARF is essential for mitotic growth. The ARF1 and ARF2 proteins are functionally homologous, <br />          and the phenotypic differences between mutations in the two genes can be accounted for by the level of <br />          expression; ARFI produces approximately 90% of total ARF. Among revertants of the fluoride sensitivity of <br />          an arfi null mutation were ARFI-ARF2 fusion genes created by a gene conversion event in which the deleted <br />          ARFI sequences were repaired by recombination with ARF2. <br />  <br />    ADP ribosylation factor (ARF) was originally identified as          protein (39). <span id='am-31' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-32' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-35'  ></span>This</span> level of sequence homology between ARF <br /> the protein cofactor required for efficient in vitro ADP               proteins from very divergent organisms suggests that ARF <br /> ribosylation of the stimulatory regulatory component of                has a fundamental role in cellular physiology. We are inter- <br /> adenylate cyclase, Gas, by cholera toxin (17, 36). ARF is              ested in using yeast genetics to determine this role. Studies <br /> itself a 21-kDa GTP-binding protein and is active in the               in yeasts on the function of other low-molecular-weight, <br /> cholera toxin reaction only in the GTP-liganded state (18).            GTP-binding proteins, e.g. RAS], RAS2 (45), YPTI (38), and <br /> For recent reviews on the role of ARF in the cholera toxin             SEC4 (32), have allowed the initial description of mutant <br /> reaction, see reference 16 and R. Kahn, Methods Enzymol.,              phenotypes as well as the identification of functionally <br /> in press. Based on the in vitro ADP-ribosylation assay (36),           related gene products. A role for the ARF protein in the <br /> ARF has been purified from a number of tissues, including              protein secretory pathway has recently been described, <br /> bovine brain, in which ARF represents as much as 1 to 2% of            deduced from the phenotype of yeast ARF mutants (43). <br /> total cell protein.                                                      In this paper we report the presence of a second ARF gene <br />    ARF protein is encoded by at least two genes in both cows           in Saccharomyces cerevisiae that encodes a protein func- <br /> (26, 39) and humans (3). The bovine ARFI gene was used as              tionally homologous to that encoded by ARFI and demon- <br /> a probe to isolate a homologous yeast gene, ARFI (39), and             strate that at least one ARF gene is required for viability. <br /> antibodies against a conserved region of the ARF protein               These results are the first evidence that ARF has an essential <br /> have been used to detect ARF in many organisms (19),                   function in normal cell growth. <br /> suggesting that ARF is a ubiquitous protein in eucaryotes. In <br /> all cases, the antibodies specifically recognize proteins with                           MATERIALS AND METHODS <br /> an apparent molecular weight of 20,000 to 24,000. In several <br />                                                                              Strains and <span id='am-57' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span>. The yeast strains and plasmids used <br /> samples, including purified bovine brain ARF, the anti-ARF-                are listed in Table 1. Strains bearing disruptions of the ARF <br /> reactive <span id='am-36' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-37' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> ran on sodium dodecyl sulfate (SDS) gels as               genes were constructed by direct replacement of the wild- <br /> a closely spaced doublet (17). Although this doublet might be              type chromosomal loci with disrupted loci constructed in <br /> due to differences in the migration of ARF proteins encoded                vitro (31). ARFI was disrupted by replacing the BglII-XbaI <br /> by different genes, it could also be due to covalent modifi-               fragment of ARF1 (see Fig. 2) with either the BamHI-SpeI <br /> cation of a population of the ARF proteins (e.g., myristyla-               fragment of the yeast plasmid YEp24, making arfi:: URA3 <br /> tion [191). No functional significance has been ascribed to the            (pRB1292), or, after making the ends flush with T4 polymer- <br /> different forms of ARF.                                                    ase, with the EcoRV-HincII fragment of the yeast plasmid <br />    The yeast gene ARFI encodes a protein highly homolo-                    YEp6, making arf7::HIS3 (pRB1293). ARF2 was disrupted <br /> gous (74% amino acid sequence identity) to the bovine ARF                  by replacing the XbaI-XbaI fragment of ARF2 with the <br />                                                                            NheI-SpeI fragment of YEp24, making arJ2:: URA3 <br />                                                                            (pRB1295). The GALI-ARFJ plasmids were constructed by <br />   * <br />     Corresponding author.                                                  cloning an NlaIV to HaeIII fragment of ARF1 into the <br />   t Present address: Department of Biochemistry and Biophysics,            pUC19 HincIl site. This plasmid was then cut with BamHI <br /> University of <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>California</span>, San Francisco, CA 94143.                         and HindIII and the ARFI fragment was ligated into the <br />   t Present address: Department of Genetics, Stanford University, <br /> Stanford, CA 94305.                                                        BamHI and HindIII sites of pBM272 (15) to give pRB1301. <br />   &#167; Present address: Department of Biology, The Johns Hopkins              The AatII-HindIII fragment of pRB1301 was inserted into <br /> University, Baltimore, MD 21218.                                           YCp402 (21) to give pRB1302. <br />                                                                     6690 <br />  VOL. 10, 1990                                                     ADP RIBOSYLATION FACTOR IN S. CEREVISIAE                     6691 <br />  <br />                                                       TABLE 1. Strain list <br />   Strain or                                                             Genotype <br />   plasmid <br /> Strain <br />   DBY4974            MATot his3-A200 leu2-3,112 Iys2-801 ura3-52 Gal&apos; <br />   DBY4975            MATa ade2-101 his3-A200 leu2-3,112 lys2-801 ura3-52 Gal&apos; <br />   DBY5305            MATa his3-A200 leu2-3,112 lys2-801 ura3-52 arfl::URA3 <br />   DBY5306            Ura- fluoride-resistant revertant of 5305 <br />   DBY5307            MATa his3-A200 leu2-3,112 lys2-801 ura3-52 arfl::URA3 <br />   DBY5308            Ura- fluoride-resistant revertant of 5307 <br />   DBY5403            MATa his3-A200 leu2-3,112 lys2-801 ura3-52 arfl::his3 arJ2::ura3 (pRB1302) <br />   DBY5406            MATa his3-A200 leu2-3,112 lys2-801 ura3-52 (pRB1301) <br />   DBY5457            MATa/MATa ade2-1OJ/ADE2 his3-A2001his3-A200 leu2-3,11211eu2-3,112 ura3-521ura3-52 arfi:: URA31ARFI <br />   DBY5458            MATa/MATTa ade2-101/ADE2 his3-A2001his3-A200 leu2-3,1121leu2-3,112 lys2-801/LYS2 trpl-l/trpl-l ura3-521ura3-52 <br />                        arf7::HIS31ARFI <br />   DBY5459            MATa/MATTa ade2-JO1/ADE2 his3-A2001his3-A200 leu2-3,1121leu2-3,112 lys2-801/LYS2 trpl-l/trpl-l ura3-52/ura3-52 <br />                        arfl::HIS31ARFJ arJ2::URA31ARF2 <br /> Plasmid <br />   pRB1292            arfl:: URA3 <br />   pRB1293            arfl::HIS3 <br />   pRB1295            arf2:: URA3 <br />   pRB1297            ARFI in YCp5O <br />   pRB1301            gall::ARFI URA3 CEN <br />   pRB1302            gall::ARFI LEU2 CEN <br />   pRB1306            ARF2 in YCp50 <br />   pRB1344            ARFI EcoRI-BgIII fragment <br />   pRB1345            Plasmid recovered from DBY5308 <br />   pRB1346            Plasmid recovered from DBY5306 <br />  <br />  <br />   Media, genetic techniques, and transformation. Media for         ments in which relative amounts of the ARF1 and ARF2 <br /> yeast growth and sporulation were as described by Sherman          proteins were estimated was tested with known amounts of <br /> et al. (40), except for 5-fluoro-orotic acid plates, described     bacterially expressed ARF1. <br /> by Boeke et al. (5), and YEPGal, which is identical to YEPD          DNA hybridizations and sequencing. The ARF2 gene was <br /> except that 2% galactose is substituted for the glucose. Yeast     cloned by hybridization of a radiolabeled ARFI probe to a <br /> growth, mating, sporulation, and tetrad analysis were per-         yeast genomic DNA library. Some 100,000 colonies of a <br /> formed as described by Sherman et al. (40). Growth on              YCp5O-based yeast DNA library (29) were probed with the <br /> plates was scored by spotting suspensions of cells in sterile      EcoRI-PstI fragment of ARF labeled to 106 cpm/ml by nick <br /> water onto plates with a 32- or 48-point inoculator. Yeast         translation (27). Hybridization was at 37&#176;C in 6x SSC (lx <br /> cells were transformed with DNA by the lithium acetate             SSC is 0.15 M NaCl plus 0.015 M sodium citrate)-5x <br /> method of Ito et al. (14), modified as described before (42).      Denhardt solution-0.5% SDS-50% formamide-50 mM KPi- <br /> Transformants were plated on synthetic complete (SC) me-           100 ,ug of carrier DNA per ml. The filters were washed at <br /> dium lacking the appropriate nutrient to select cells carrying     370C in 0.5x SSC-0.1% SDS. Twenty-six positive clones <br /> the transformation marker.                                         were analyzed by restriction digestion and DNA hybridiza- <br />    Preparation of antibodies and immunodetection of ARF. A         tion. Of these, 24 were ARF and the remaining 2 were <br /> peptide corresponding to amino acid residues 159 to 173 of         ARF2. Both strands of the NarI fragment containing the <br /> ARF and ARF2 was purchased from Peptide Technologies,              entire coding region of ARF2 were sequenced by the dide- <br /> Washington, D.C. This peptide contains the sole cysteine in        oxy-chain termination method (33). <br /> the ARF sequence, allowing conjugation to carrier keyhole             Total yeast genomic DNA was prepared as described <br /> limpet hemocyanin as described by Green et al. (12). Rabbits       previously (9). Restricted DNA was transferred from agar- <br /> were immunized with multiple intradermal injections of             ose gels as described by Southern (41). Hybridizations with <br /> conjugated peptide in complete Freund adjuvant and were            32P-labeled probes were performed under conditions de- <br /> boosted after 2 weeks as described by Mumby et al. (24).           scribed by Wahl et al. (46); washes were carried out in 2x <br /> The serum from rabbit R-16 was used in all experiments.            SSPE (lx SSPE is 0.18 M NaCl, 10 mM sodium phosphate, <br />    Total yeast protein extracts were prepared by harvesting        and 1 mM EDTA [pH 7.7])-0.1% SDS at 650C. 32P-labeled <br /> approximately 2 x 107 cells by centrifugation, resuspending        probes were prepared by random hexamer labeling of re- <br /> in 20 uil of Laemmli sample buffer, and immediately boiling        striction fragments in low-melting-point agarose (11). <br /> for 3 min. Glass beads were then added and the samples                Cloning of afl::URA3 intragenic revertant loci. The ARFE <br /> were vortexed at high speed for 2 min. Fifty microliters of        loci from two independent Ura- arf7::URA3 revertants <br /> sample buffer was added, the supernatant fraction was              (DBY5306 and DBY5308) were cloned by gene eviction (28) <br /> removed and boiled again, and an aliquot was loaded on an          as follows. Plasmid pRB1344 was made by inserting the <br /> SDS-polyacrylamide (15%) gel. Proteins were electrophoret-         EcoRI-BgII fragment of ARF) in the yeast-integrating plas- <br /> ically transferred to a nitrocellulose membrane, and ARF           mid YIp5. This plasmid was then cut with NheI and trans- <br /> was detected by incubation of the membrane with anti-ARF           formed into the two Ura- revertants, selecting for the URA3 <br /> primary serum followed by &apos;25I-labeled protein A (6). That         marker of the plasmid. DNA was then isolated from two <br /> antibody R-16 was functioning in the linear range in experi-       transformants for each of the two revertants. This DNA was <br />  6692     STEARNS ET AL.                                                                                                   MOL. CELL. BIOL. <br />  <br />                   1 GCCTGCAGGT ACGCCCCTTT ATTTGATCAG GAAGCCGTAT TGATTATCTA ATAGGGCCTA <br />                  61 GTTATCCTAA TTGTGGGGAG TCGAGCAGTA CGGCTCTGAT GTTTTTCGAA CGAAGATAAG <br />                121 GAGTTGACAT ACAAAGTCAA CAGAAGTTCT TCTTGTTAGC GTCTCTGTGC TCAATATCTC <br />                181 TCTTTTTTTC TTTAAGTAGT AATTACTAAC ATCAGCCAAC CAATAGAGAT AAAAAAAAAA <br />                241 GGAATTAAGA TTTCATAGAG AAAAGATGGG TCTATACGCT TCTAAGTTAT TCAGCAATCT <br />                  1                            M G    L Y A     S K L F      S N L <br />                                                        (F) <br />                301 TTTTGGCAAC AAAGAGATGC GTATACTTAT GGTTGGTCTA GATGGTGCCG GTAAGACCAC <br />                 13 F G N      K E M R      I L M     V G L     D G A G      K T T <br />  <br />                361 CGTTTTGTAC AAGTTGAAGT TGGGCGAAGT TATCACTACC ATTCCAACCA TTGGTTTCAA <br />                 33 V L Y      K L K L      G E V     I T T     I P T I      G F N <br />  <br />                421 CGTTGAGACT GTCCAATATA AGAACATTTC CTTCACTGTC TGGGACGTCG GTGGACAAGA <br />                 53 V E T      V Q Y K      N I S     F T V     W D V G      G Q D <br />  <br />                481 CAGGATTAGA TCTTTATGGA GACACTACTA CAGAAACACC GAAGGTGTTA TTTTTGTCAT <br />                 73 R I R      S L W R      H Y Y     R N T     E G V I      F V I <br />                                                                                                                          (V) <br />                541 CGATTCCAAC GATAGATCGC GTATTGGTGA AGCCAGAGAA GTCATGCAGA GAATGCTGAA <br />                 93 D S N      D R S R      I G E     A R E     V M Q R      M L N <br />  <br />                601 TGAAGATGAA TTGAGAAATG CTGTCTGGTT AGTCTTCGCT AACAAACAAG ATTTGCCAGA <br />                113 E D E      L R N A      V W L     V F A     N K Q D      L P E <br />                                           (A) <br />                661 AGCCATGTCT GCTGCTGAAA TCACCGAAAA ATTAGGTTTA CATTCTATTA GAAACCGTCC <br />                133 A M S      A A E I      T E K     L G L     H S I R      N R P <br />  <br />                721 ATGGTTTATC CAGTCTACTT GTGCAACCTC GGGTGAAGGT                              CTGTACGAAG GTCTGGAGTG <br />                153 W F I       Q S T C     A T S     G E G                                  L Y E G      L E W <br />                                   (A) <br />                781 GTTAAGCAAC AACTTGAAGA ATCAATCCTA ATCTAAATCT                              GTATAGAACG TTTAGTCATG <br />                173 L S N       N L K N     Q S <br />                               (S)         (S) (T) <br />                841 CGCACCTTGT GTGTTTTGTT TCTAGATTGT TTTATTTTTA                              TGATTGTTGA AGATATAAAC <br />                 901 CACTGTATAG TTGTATAAGA TAGGATAATG ATGGTGCACT GAAAATAAAC TTACTAGCTC <br />                 961 TTTAATATTG CAACGGCTTG TAACGGGCGA CTCTAGAGGA TCCCCGGGTA CC <br />   FIG. 1. DNA sequence of the ARF2 gene and the predicted protein sequence. The differences between the predicted ARFI and ARF2 <br /> proteins are indicated in parentheses below the ARF2 sequence. All other residues, as well as predicted total protein length, are identical to <br /> ARFI. The GenBank accession number for the ARF2 sequence is M 35158. <br />  <br />  <br /> cut with SphI, diluted, ligated, and used to transform Esch-             at the ARF1 locus (see below). The ARF+ phenotype in <br /> erichia coli, selecting for ampicillin resistance. In all cases          these intragenic revertants was linked to the ARF1 locus, <br /> the recovered plasmids (pRB1346 from DBY5306 and                         and the revertants were phenotypically Ura-. These results <br /> pRB1345 from DBY5308) had the expected structure. The                    suggested that the reversion event might be gene conversion <br /> EcoRI-BgIII fragment of each plasmid was cloned into                     of the arfi:: URA3 mutation by homologous sequences in the <br /> M13tgl3O and sequenced by the dideoxy-chain termination                  yeast genome, possibly a second ARF gene, resulting in loss <br /> method (33).                                                             of the URA3 insertion concomitant with gain of ARF func- <br />                                                                          tion. To isolate this homologous gene, we used the ARFI <br />                          RESULTS                                         gene as a probe of yeast genomic DNA. A labeled DNA <br />                                                                          fragment containing the entire ARFI coding sequence was <br />   Two genes encode ARF in S. cerevisiae. The yeast gene                  used to screen a yeast genomic DNA library in plasmid <br /> ARFI was cloned by virtue of its homology to the bovine                  YCp5O (29). Of 26 clones isolated in this way, 24 were found <br /> ARF gene and encodes a protein that is 74% identical to                  to contain inserts from the ARF1 locus; 2 appeared to be <br /> mammalian ARF (39). Two lines of evidence indicated the                  different from ARFI but related to each other. The sequence <br /> presence of a second ARF gene in yeast cells. First, DNA                 of this ARFI-homologous locus was determined and found <br /> hybridization experiments showed a second locus that hy-                 to contain an open reading frame that is 91% identical to <br /> bridized with an ARFI probe under stringent hybridization                ARFJ at the nucleotide level and will be called ARF2 (Fig. <br /> conditions (not shown). Second, in characterizing null mu-               1). There is no significant homology between the two genes <br /> tations in ARFI, we found revertants of an arfi::URA3                    outside of the coding sequences. It is worth noting that the <br /> deletion-insertion mutation that appeared to have occurred               homology between the ARF1 and ARF2 genes is such that <br />  VOL. 10, 1990                                                         ADP RIBOSYLATION FACTOR IN S. CEREVISIAE                   6693 <br />  <br />          TABLE 2. Genetic mapping of ARFI and ARF2a                                                        IV <br />   Interval             PD:              NPD:                     iT <br /> arfl-arJ2               51                0                      65 <br /> arfl-cdc2               12                1                      17 <br /> arf2-cdc2               41                0                      16                           ARF1 <br /> arf)-cdc9               44                0                      21 <br /> arf2-cdc9               16                0                       7                           cdc36    - <br />  <br />  <br /> arfl-cdc36              36                0                      12                            cdc9 <br /> arJ2-cdc36              16                0                       6                            ste7 &lt; <br /> cdc2-cdc36              11                0                      11 <br /> cdc9-cdc36              22                0                       1 <br />  a PD, Parental ditype; NPD, nonparental ditype;   Tr, tetrad.                                ARF2 <br /> certain restriction sites within the coding sequences are                                      cdc2 <br /> conserved, possibly explaining the failure to identify ARF2 <br /> in the experiments in which ARFI was originally cloned (39). <br />    The predicted ARF2 protein is identical in length to the <br /> ARF1 protein and 96% identical in protein sequence, dif- <br /> fering at only 7 of 181 residues (Fig. 1). Most of these seven <br /> differences are conservative amino acid changes, and three <br /> of the seven are at the C terminus of the protein, a region <br /> relatively divergent among other low-molecular-weight, <br /> GTP-binding proteins. The GTP-binding consensus regions <br /> and the putative myristylation site present in ARF are                  FIG. 2. Genetic map location of ARF and ARF2 on chromo- <br /> conserved in ARF2.                                                    some IV. Three-point crosses were used to establish the <span id='am-39' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> of <br />    Genetic mapping of ARF and ARF2. A DNA blot of yeast               ARF, cdc9, and ARF2; ARF, cdc9, and cdc36; and ARF2, cdc2, <br /> chromosomes separated by CHEF electrophoresis was                     and cdc36. <br /> probed with a fragment of the ARFI gene to determine the <br /> chromosomal locations of the two ARF genes. The condi-                In this strain the two arf mutations are in trans: the <br /> tions used allowed hybridization of the probe to both ARFE            ar]fl::HIS3 mutation in one copy of chromosome IV in the <br /> and ARF2. The only hybridization signal observed in these             diploid, arJf2::URA3 in the other. DBY5459 was sporulated <br /> experiments was to chromosome IV, indicating that both                and tetrads were dissected. Of 50 tetrads scored, 24 had four <br /> genes are on the same chromosome. Strains bearing geneti-             viable spores, with the arfl::HIS3 and arJ2:: URA3 muta- <br /> cally marked ARF loci were then crossed to each other and             tions in the parental configuration, whereas 26 had three <br /> to strains bearing chromosome IV genetic markers. Segre-              viable spores, with the markers in the tetratype configura- <br /> gants of the appropriate genotype from these crosses were             tion. In all cases the missing spore in the tetratype tetrads <br /> then crossed to each other to order linked genes in three-            was the inferred double mutant, indicating that the arf] arJ2 <br /> point crosses. The tetrad <span id='am-52' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54'  ></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> presented in Table 2 show that         double null mutant is inviable. This ratio of parental to <br /> the two ARF genes are separated by approximately 28                   tetratype tetrads is consistent with the genetic distance <br /> centimorgans on the left arm of chromosome IV, on either              between ARF and ARF2 determined in mapping crosses. A <br /> side of the ste7-cdc9-cdc36 gene cluster. ARF2 is centromere          strain in which the two disruption mutations were in the <span id='am-42' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-43' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-44'  ></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>cis</span> <br /> proximal, between cdc2 and cdc36, and ARF is centromere               configuration also produced parental and tetratype tetrads in <br /> distal. This arrangement is depicted in Fig. 2. The two ARF           the expected ratios (data not shown); in this case the <br /> genes occupy map positions distinct from all other mapped             parental tetrads had two viable spores. A parental ditype <br /> yeast genes (23) and by that criterion represent newly                tetrad from DBY5459 was subjected to DNA blotting anal- <br /> identified genes.                                                     ysis, with probes against both ARF and ARF2 (Fig. 4). <br />    At least one functional ARF gene is required for viability.        Each disruption mutation segregated 2+:2-, and in each of <br /> As a first step in determining the function of the two ARF            the four spores one of the ARF genes was wild type whereas <br /> genes, we constructed deletion-insertion mutations in each            the other was disrupted. <br /> of the genes. In each case a large portion of the coding                The double-disruption experiment shows that a spore <br /> sequence was replaced with a yeast marker gene (Fig. 3).              lacking a functional ARF gene is unable to form a colony, <br /> Two arfl disruption constructions (arf7:: URA3 and                    suggesting that ARF is required for growth. We sought to <br /> arfl::HIS3), differing only in the inserted marker, were              demonstrate this requirement for ARF independent of spore <br /> transformed separately into wild-type diploid strain                  germination. A strain was constructed in which the only <br /> DBY1830 such that they replaced the chromosomal ARFE                  complete ARF gene was under control of the inducible GAL) <br /> locus (31). The resulting strains (DBY5457 and DBY5458)               promoter. The level of transcription from this promoter is <br /> were sporulated and tetrads were dissected. All four spores           high when galactose is the carbon source and very low when <br /> were viable for both the ar)fl:: URA3 and arfl::HIS3 con-             glucose is the carbon source; thus, growth should be carbon <br /> structions, and the disruption markers segregated 2+:2- in a          source dependent if ARF was required for mitotic growth. <br /> total of 39 tetrads examined. DNA hybridization experi-               The yeast centromere plasmid pRB1302 contains the ARFE <br /> ments showed that the Ura+ and His&apos; segregants lacked a               coding sequence and 3&apos; noncoding region fused to the GAL) <br /> wild-type copy of the ARF locus (see below).                          promoter, as well as the LEU2 gene. pRB1302 was trans- <br />    The arf2:: URA3 disruption construction was transformed            formed into a diploid strain that is phenotypically Gal&apos; and <br /> into a diploid ARFlarfl::HIS3 strain, such that it replaced           has the ARF genotype ARF)/arfl: :HIS3 ARF21arJ2:: URA3. <br /> one of the wild-type ARF2 loci, resulting in strain DBY5459.          A transformant was sporulated and tetrads were dissected <br />  <br />   FIG. 3. Restriction maps of ARFI and ARF2 and disruption constructions. The insert fragments (containing URA3 and HIS3) are not to <br /> scale; their sizes are indicated separately. <br />  <br />  <br />                                                                        on  rich medium with galactose as the carbon source (YEP- <br />                                                                        Gal). These conditions should allow the growth of a haploid <br />                                                                        segregant lacking both chromosomal copies of ARF, but <br />                                   a      b c d                         carrying the plasmid with the galactose-inducible ARFI <br />                                                                        gene. A segregant that was phenotypically His&apos;, Ura+, and <br />                                                                        Leu+ (DBY5403) was checked by DNA hybridization to <br />                                                                        confirm that the plasmid-borne ARFI gene was the only <br />                                                                        complete ARF gene (data not shown). The ability of this <br />                          ARF2-         *                               strain to grow on different carbon sources was tested. <br />                                                                        Growth on galactose medium appeared normal, though <br />                                                                        slower than that of wild type. Growth on glucose medium <br />                                                                        was completely inhibited (Fig. 5). We examined the growth <br />                                                                        of this strain in liquid medium and found that, when shifted <br />                                                                        from YEPGal to YEPD medium, growth continued for 7 h <br />                                                                        and then leveled off, presumably reflecting depletion of ARF <br />                                                                        protein. No morphological changes were observed in cells <br />                          ARF1-4 0                                      from these cultures. Most cells in the DBY5403 culture were <br />                                                                        inviable 8 h after the shift to glucose; wild-type cells contin- <br />                                                                        ued to grow exponentially under the same conditions. We <br />                  arfi ::HIS3-         so                               found that incorporating 0.2% glucose into the 2% galactose <br />                                                                        medium improved the growth rate of strain DBY5403. Pre- <br />                                                                        sumably this improvement is due to a reduction in the level <br />                                                                        of expression from the GAL] promoter, as overexpression of <br />                                                                        the ARF1 protein from this promoter in wild-type cells is <br />                                                                        deleterious (see below). <br />                                                                          Phenotypes of arf mutants. (i) Growth defect. Dissection of <br />                                                                        tetrads derived from diploids bearing arf7 and arJ2 null <br />                                                                        mutations revealed an obvious growth defect in the arfi null <br />                                                                        segregants. Spore colonies of strains carrying the arfl dis- <br />                 arf2::URA3-         IW                                 ruption marker were much smaller than their ARFI siblings. <br />                                                                        In contrast, the arJ2 null spore colonies were indistinguish- <br />                                                                        able in size from wild-type spore colonies. The growth rates <br />   FIG. 4. DNA blot of arf disruption strains. Genomic DNA from         of wild-type, arfi null, and arJ2 null strains in liquid medium <br /> strains of a parental ditype tetrad of DBY5459 was cut with PstI and   confirmed these observations; wild-type and arJ2 null strains <br /> probed with a mixture of ARFI and ARF2 probes.                         had a generation time of approximately 85 min at 30&#176;C, <br />  VOL. 10, 1990                                                        ADP RIBOSYLATION FACTOR IN S. CEREVISIAE                   6695 <br />  <br />  <br />                                                    Glucose                             Galactose <br />  <br />  <br />                    DBY5403                7 <br />  <br />                    DBY5406 <br />  <br />  <br />   FIG. 5. Phenotype of ARFI depletion and overexpression. DBY5403 arfl::HIS3 arf2::URA3 (pRB1302) and DBY5406 ARFI ARF2 <br /> (pRB1302) were inoculated on SC-ura medium containing the indicated carbon source and incubated for 3 days at 30&#176;C. <br />  <br /> whereas the arfl null strain had a generation time of 125 min.       wells in plates seeded with lawns of either wild-type or arfi <br /> The arfl null phenotypic defect was strikingly accentuated           null mutant cells. The zones of inhibition caused by the <br /> by incubation at lower temperatures. At 14&#176;C, wild-type and          EDTA were dependent only on the concentration of EDTA <br /> arJ2 null strains grow with a generation time of 8 h, whereas        used and not on the genotype of the cells in the lawn. Thus, <br /> the arfi null strain doubled once in 17 h but did not double         lowering the extracellular magnesium concentration with <br /> again during a further 72 h of incubation. Microscopic               EDTA does not mimic the effect of fluoride. Chelation of <br /> examination of the arfl null cells after 18 h at 14&#176;C did not        aluminum or formation of aluminum fluoride complexes over <br /> reveal any cell cycle arrest phenotype. Examination of the           hours or days is also a possibility for the effect of age on the <br /> morphologies of the nucleus with the DNA-specific dye                fluoride plates. <br /> 4&apos;,6-diamidino-2-phenylindole and the microtubule cytoskel-             (iii) Overexpression lethality. Low-copy-number (CEN) <br /> eton by antitubulin immunofluorescence did not reveal any            and high-copy-number (2,um) plasmids that contained either <br /> significant structural defects at either 30 or 14&#176;C (not shown).     the ARFI or the ARF2 coding sequence were constructed <br />    (ii) Sensitivity to fluoride ion. It has been observed that the   and used to transform a wild-type strain. Although ARFI <br /> trimeric GTP-binding proteins can be activated in vitro by a         and ARF2 centromere plasmids had no effect, the 2,im- <br /> combination of aluminum and fluoride ions, although no               based plasmids severely inhibited the growth rate of wild <br /> such activation has been reported for monomeric GTP-                 type. In particular, the ARFI-2,um plasmid (pRB1300) <br /> binding proteins, including ARF. Interestingly, we found             yielded transformants at a much lower frequency than the <br /> that arfl null mutants were sensitive to what are normally           vector without an insert, and the transformants that did grow <br /> sublethal amounts of potassium fluoride in the medium.               into colonies grew very slowly. The ARF2-2,um plasmid <br /> Wild-type haploid yeasts are typically resistant to approxi-         (pRB1307) had a similar, though not as severe, effect. A <br /> mately 60 mM KF in YEPD, whereas arfi null mutants were              similar result was obtained with a centromere plasmid <br /> inhibited for growth by as little as 15 mM. The same result          (pRB1301) carrying the GALI-ARFJ construction described <br /> was obtained with sodium fluoride, while potassium chloride          above. A wild-type strain bearing this plasmid grew normally <br /> and sodium chloride had no effect. We found that 40 mM F-            on medium containing glucose but was unable to grow when <br /> was a reliable concentration for easily distinguishing arfi          galactose was the sole carbon source, and the GAL] pro- <br /> null mutants from wild type. No cell cycle-specific arrest           moter is maximally induced (Fig. 5). The growth of a control <br /> could be detected either in a wild-type culture treated with         strain bearing the GAL vector without an insert was not <br /> 100 mM KF or an arfl null mutant culture treated with 40             affected by the change in carbon source. <br /> mM KF. Addition of AlCl3 in the range of 0 to 10 mM to                  Antibodies against the ARF gene products. Antibodies <br /> plates containing various KF concentrations had no effect            against yeast ARF were generated by immunizing rabbits <br /> (other than an increasing amount of precipitate at higher            with a synthetic peptide coupled to keyhole limpet hemocy- <br /> concentrations). If aluminum ions are required for this              anin. The sequence used for immunization is identical in <br /> effect, however, it is likely that a sufficient amount is already    ARFI and ARF2; thus, the antibodies generated should <br /> present in the media. arJ2 null mutants were indistinguish-          recognize both proteins equally well. The polyclonal antise- <br /> able from wild type in terms of sensitivity to fluoride ions.        rum derived from rabbit R-16 used in these experiments <br />    YEPD plates containing 40 mM KF or NaF lost the ability           recognized a -21-kDa protein in whole yeast extract, as well <br /> to support growth of even wild-type strains about 5 to 7 days        as one other protein of higher molecular weight. Peptide <br /> after preparation. The ability to support the growth of wild         competition experiments showed that only the 21-kDa signal <br /> type, but not arfl null strains, could be restored by adding         is diminished by peptide. ARF protein is expressed at a level <br /> Mg2+ to the plate prior to use (typically 0.1 ml of 1 M MgCl2        of 0.03 to 0.1% of total cell protein as determined by <br /> per 35-ml plate). The probable cause for this effect is the          quantitation of immunoblots with purified recombinant <br /> complexing and precipitation of Mg2+ ions by F-. This                ARFI as the standard. The R-16 polyclonal antiserum was <br /> raises the possibility that the arfl null defect is actually a       used to examine the relative contributions of ARFI and <br /> sensitivity to reduced Mg2+ levels in the media. To test this        ARF2 to the total amount of ARF in the cell. Although ARFI <br /> possibility, the effect of EDTA on growth was examined.              and ARF2 are of identical length and nearly identical se- <br /> EDTA solutions of different concentrations were placed in            quence, they have slightly different mobilities under appro- <br />  6696     STEARNS ET AL.                                                                                              MOL. CELL. BIOL. <br />  <br />                                                                           Suppression of the fluoride supersensitivity of arfi null <br />                            ARFi ARF2                                    mutants. The strong fluoride supersensitivity of arfi null <br />                                                                         mutants provided a genetic selection for increased ARF <br />                                                                         function. Spontaneous revertants able to grow on plates <br />                          &apos;ARFl arf2                                     containing 40 mM KF were selected from the arfl::URA3 <br />                                                                         mutants DBY5305 and DBY5307. Such revertants arose at a <br />                                                                         frequency of about 1 in 107 cells plated. In all of the <br />                            arfl1 ARF2                                   revertants isolated, the slow growth and cold sensitivity <br />                                                                         phenotypes of the arfl:: URA3 mutant were also suppressed. <br />                                                                         Surprisingly, 29 of 102 independent pseudorevertants <span id='am-47' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-48' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51'  ></span>iso</span>- <br />                                                 yARF2                   lated had become uracil auxotrophs, phenotypically losing <br />                                                                         the URA3 marker inserted into ARF. When three of these <br />                                                                         Ura- revertants were backcrossed to a wild-type strain, only <br />                            arflt arf2-        p ARF1                    parental ditype asci were recovered; all spores showed <br />                                                                         wild-type resistance to fluoride. This indicated that the <br />    FIG. 6. ARF protein levels in wild-type and mutant strains.          mutation causing suppression was tightly linked to ARF or <br /> Whole protein extracts from strains of the indicated genotypes were     was in the ARF gene itself. In addition, the suppression was <br /> subjected to electrophoresis, blotted, and probed with anti-ARF         dominant; when Ura- pseudorevertants were crossed to an <br /> antibody as described in Materials and Methods. The antibody used       arfi:: URA3 strain, the resulting diploids were fluoride resis- <br /> is directed against a peptide epitope of ARF that is identical in the   tant. The simplest explanation of these results is that the <br /> ARFI and ARF2 proteins.                                                 disrupted arfl gene was repaired by recombination with <br />                                                                         ARF2 sequences. This seemed unlikely, however, as the <br />                                                                         arfi:: URA3 construction deleted sequences past the 3&apos; end <br /> priate electrophoretic conditions. The three viable spores              of the ARFJ coding sequence and thus lacked any homology <br /> from a tetratype tetrad of DBY5459 were examined by                     with the ARF2 locus at the 3&apos; end. <br /> immunoblotting (Fig. 6). The arfl: :HIS3 spore clearly lacked              To determine the event that occurred in the Ura- rever- <br /> the major band of ARF reactivity, but retained a minor band             tants, genomic DNA from wild type, arfl:: URA3, and three <br /> that migrates slightly faster than ARF. This minor band                 Ura- revertants was subjected to DNA blot analysis. The <br /> reproducibly appeared to be missing in the arj2:: URA3                  probe in these experiments was the EcoRI-PstI fragment of <br /> strain, although its proximity to the major band made this              ARF containing the entire coding sequence that, under the <br /> difficult to resolve. The minor band was overproduced                   hybridization conditions used, hybridizes to both ARFI and <br /> severalfold in a strain carrying ARF2 on a centromere                   ARF2. An internal BgIH site present at a homologous <br /> plasmid (presumably present in more than single copy,                   location in both ARF and ARF2 (Fig. 3) was destroyed in <br /> although not tested), demonstrating that the minor band is              the arfl:: URA3 allele by ligation with a BamHI end of a <br /> the product of the ARF2 gene. Assuming that the R-16                    fragment carrying URA3. The BamHIIBglll junction formed <br /> antibody recognizes both forms of ARF equally, the minor                cannot be cut by either enzyme. The pattern of hybridization <br /> band was estimated to constitute approximately 10% of the               confirmed that the ARF BglII site was missing in the <br /> total ARF protein. There did not appear to be an increase in            arfi:: URA3 strain but surprisingly reappeared in the rever- <br /> the expression of the ARF2 protein in arfi disruption mu-               tants (not shown). Hybridization of the ARF probe to a <br /> tants. These findings have been confirmed with ARF- and                 number of genomic digests revealed that the restored BglII <br /> ARF2-specific antibodies (R. Kahn, unpublished observa-                 site in the revertants was derived from homologous DNA at <br /> tion).                                                                  the ARF2 locus. The resulting genes must then encode <br />    The ARF and ARF2 proteins are functionally homologous.               fusion proteins with an amino-terminal portion derived from <br /> The near identity of the ARF and ARF2 proteins suggested                ARF and the remainder derived from ARF2. In all digests, <br /> that they might perform identical functions and that the                the ARF2 gene remained unchanged in the revertants. These <br /> genetic difference between the two genes could be accounted             results suggested that the repair of arfi:: URA3 and the loss <br /> for by the lower level of expression from the ARF2 gene. In             of the URA3 marker were accomplished by a gene conver- <br /> this case, simply increasing the copy number of the ARF2                sion event: a nonreciprocal transfer of information from <br /> gene should suppress the phenotypic defects of an arfi null             ARF2 to arfl:: URA3. <br /> mutant. This hypothesis was tested by transforming arfl null               There must be two crossover points that define the extent <br /> mutant strains with plasmids bearing the ARF2 gene. First, a            of the replacement of arfi:: URA3 DNA with ARF2 DNA, <br /> yeast-integrating plasmid bearing ARF2 was integrated at the            one on each side of the nonhomologous URA3 insert. DNA <br /> chromosomal ARF2 locus of an arf) null strain by homolo-                blot analysis revealed that the converted arfi loci in strains <br /> gous recombination, resulting in a duplication of ARF2.                 DBY5306 and DBY5308 do not have the XbaI site located <br /> Transformants in which only a single copy of the plasmid                near the 5&apos; end of the ARF2 reading frame (Fig. 3), while <br /> had integrated were identified by DNA blotting. Second, a               both have the BglII site from ARF2 (see above). The site of <br /> yeast centromere plasmid bearing ARF2 (pRB1306) was                     the 5&apos; crossover then must be in the 151-bp region between <br /> introduced into an arfl null strain; centromere plasmids are            these two restriction sites. The site of the 3&apos; crossover could <br /> typically maintained at 1 to 2 copies per cell (8). In both             not be determined but must lie further than an XbaI site <br /> cases, the increased ARF2 gene dosage was sufficient to                 located 52 bp from the 3&apos; end of the ARF2 reading frame as <br /> completely suppress the three arfi null phenotypes of slow              this site was acquired in the revertant loci. <br /> growth, cold sensitivity, and fluoride supersensitivity. Thus,             To determine directly the structure of the revertants, the <br /> when the ARF2 copy number is as little as doubled, there is             arfi loci from two revertant strains were cloned by integrat- <br /> sufficient ARF2 protein to perform all ARF functions in the             ing a plasmid carrying ARF homology immediately adjacent <br /> cell, indicating that the ARFE and ARF2 proteins are func-              to the converted arfl, isolating genomic DNA from transfor- <br /> tional homologs.                                                        mants, and cutting and ligating the DNA so that the con- <br />  <br />    FIG. 7. Structure of two ARFJ/ARF2 fusions. The genomic DNA at the ARFI locus of two intragenic arfl:: URA3 revertants (DBY5306 <br /> and DBY5308) that had undergone a recombination event between arfl::URA3 and ARF2 was cloned and sequenced. The locations of the <br /> 5&apos; crossover events in the two revertants are shown as X&apos;s between the ARF2 and arf7:: URA3 sequences (the DBY5306 site is the more 5&apos; <br /> of the two). The boxed bases in the ARF2 sequence shown are those that differ from ARF1. <br />  <br /> verted loci would be recovered in E. coli on the plasmid (see        copies of ARF2 and are phenotypically wild type still have <br /> Materials and Methods). The DNA sequence of these loci               less ARF protein than an ARF ARF2 cell (Fig. 6). An <br /> revealed that, as predicted, they are ARFJ/ARF2 fusion               alternative explanation of these results is that the ARF2 <br /> genes, with the crossover points in the expected region (Fig.        protein is more active in some way than ARFI protein and is <br /> 7). The actual site of the crossover could only be approxi-          able to perform all necessary functions at lower protein <br /> mated due to the high degree of sequence identity between            levels. Second, the function of ARF2 is obscure, as arJ2 null <br /> ARFI and ARF2 in this region. For strain DBY5306, the                mutants are phenotypically indistinguishable from wild type, <br /> crossover point could be narrowed to a region 39 to 61 bp 5&apos;         and the ARFI and ARF2 proteins appear to be functionally <br /> of the BglII site, and for strain DBY5308, it could be               homologous. Among the known examples in yeast cells of <br /> narrowed to a region 6 to 22 bp 5&apos; of the BglII site (Fig. 7).       duplicated genes are those in which the two genes are <br />                                                                      regulated differently, such as cytochrome c (20), and those in <br />                         DISCUSSION                                   which the proteins are directed to different cellular compart- <br />    In this paper we demonstrate that ARF is encoded by two           ments, such as citrate synthase (30). Although we presently <br /> genes in S. cerevisiae, that at least one of these genes must        lack information on the regulation of the ARF genes and the <br /> be functional for viability, and that the ARF proteins en-           localization of the ARF proteins in S. cerevisiae, the arfi <br /> coded by these two genes are functionally homologous.                suppression results indicate that any difference in regulation <br /> These results are the first evidence of an essential role for        or localization can be overcome by simple duplication of the <br /> ARF in cellular physiology. We have also begun to analyze            ARF2 gene. The duplicated yeast ax-tubulin genes, TUB] and <br /> the phenotypes of arf mutations, gaining information about           TUB3, closely resemble the ARF genes in a number of <br /> the function of ARF not available from the previous studies          respects. The two a-tubulin proteins are functionally homol- <br /> on the in vitro properties of this protein.                          ogous, but disruption of TUB], which makes the majority of <br />    The ARF and ARF2 genes encode proteins that are 96%               a-tubulin, is lethal, whereas disruption of TUB3 has only <br /> identical, yet mutations in the two genes resulted in different      minor phenotypic consequences (35). Strikingly, TUB) and <br /> phenotypes. It is likely that this genetic difference is due to      TUB3, like ARF and ARF2, are genetically linked to each <br /> a difference in the level of expression of the two ARF               other, separated by approximately 38 centimorgans on chro- <br /> proteins. We found that approximately 90% of total ARF               mosome XIII (35). <br /> protein is produced by the ARF) gene, and the remainder is              The two ARF genes are located on chromosome IV, about <br /> produced by ARF2; a single extra copy of ARF2 could                  28 centimorgans apart, and can recombine with each other. <br /> suppress the phenotypic defects caused by deletion of                This is clearly demonstrated by the recovery of ARFI/ARF2 <br /> ARFI. This raises a number of points concerning the two              fusion proteins created by in vivo recombination in strains <br /> ARF genes. First, there is likely to be excess ARF in yeast          selected for suppression of the arpf:: URA3 null phenotype. <br /> cells grown under standard laboratory conditions, as arf7            The two independent recombinants that were sequenced <br /> arf2 double null mutant strains that have plasmid-borne              shared common features. The recombination events were <br />  6698     STEARNS ET AL.                                                                                                MOL. CELL. BIOL. <br />  <br /> nonreciprocal; the ARF2 locus remained unchanged, while             micromolar range. Often, exogenous Al3&quot; need not be added <br /> the fusion protein was created at the arfi locus. The 5&apos; ends       as it is a common contaminant of most aqueous solutions <br /> of the recombination events (relative to the coding strand of       (44). In contrast to trimeric G proteins (13), tests conducted <br /> ARFI) were in a region of DNA homology between the                  with purified bovine ARF show no effect of fluoride on ARF <br /> ARFJ and ARF2 coding sequences, the exact location dif-             activity or intrinsic fluorescence, using conditions in which <br /> fering in the two recombinants. Third, the 3&apos; ends of the           activating guanine nucleotides dramatically increased each <br /> recombination events were not determined but must have              (17). Thus, a direct effect of a fluoride complex with ARF <br /> been beyond the end of the ARFI coding sequence. This 3&apos;            appears unlikely. A plausible model for the fluoride sensitiv- <br /> crossover is interesting because there is no detectable ho-         ity of arf mutants is that fluoride activates a previously <br /> mology in the sequenced 3&apos; untranslated regions of the two          uncharacterized trimeric G protein in S. cerevisiae that is <br /> ARF genes. It is possible that the 3&apos; crossover occurred in         distinct from those containing GPAI and GPA2 as a sub- <br /> repeated DNA, such as Ty, d, or a elements (4).                     units. If ARF normally acted to inhibit this G protein, then in <br />    What is the nature of the essential function of ARF in S.        the absence of ARF, fluoride might overactivate it. Alterna- <br /> cerevisiae? In separate experiments (44) we have found that         tively, millimolar concentrations of fluoride are also known <br /> ARF is involved in protein secretion in yeast cells and that        to affect protein phosphatases, phosphorylases, ATPases, <br /> ARF protein is localized to the Golgi of mammalian cells.           and free-metal concentration (34). The last is interesting in <br /> The phenotypes that we report here for arfi arJ2 double null        light of the ability of magnesium to restore growth of <br /> mutants are consistent with this; there is no specific cell         wild-type cells on agar plates containing fluoride and the <br /> cycle defect associated with loss of ARF function and cells         reported effects in yeast cells of yptl mutations on Ca2&quot; <br /> depleted of ARF by repression of a GALl-ARFI construc-              uptake and rescue from growth arrest by increased extracel- <br /> tion arrest growth at all points in the cell cycle. The             lular calcium (37). YPTI is also a 21-kDa GTP-binding <br /> phenotypes of the arfi null mutant, slow growth, cold               protein with subcellular distribution and apparent role in <br /> sensitivity, and fluoride sensitivity, are genetically useful but   protein secretion (38) similar to those recently described for <br /> not revealing of function, even in light of the role of ARF in      ARF (43). Ultimately, knowledge of the basis of the in- <br /> secretion (see below).                                              creased fluoride sensitivity of arfi null mutants may provide <br />    As ARF was originally identified as a cofactor in the            insight into the function of ARF in the secretory pathway. <br /> covalent modification of the G protein G,S in animal cells, it <br /> is perhaps most interesting to note what phenotypes were                                 <br /> not observed in the yeast arf mutants. There are two known <br /> yeast G,, proteins; GPAI is involved in mating pheromone               <br /> signal transduction (10, 22), whereas the function of GPA2 is      <br /> unknown, and mutations in GPA2 result in no discernible        <br /> phenotype (25). If ARF were required for GPAI function,           <br /> then sterility or constitutive activation of the mating path-    <br /> way might be expected, but neither of these phenotypes was         <br /> observed in either arfi null mutants or under conditions of      <br /> ARF depletion (see Results; also, unpublished observation).       <br /> A caveat of this interpretation is that the known secretion <br /> defect of arf mutants might prevent the observation of the                          <br /> morphological changes associated with activation of the         <br /> mating pathway. If ARF were required solely for GPA2            <br /> function, then ARF might be expected to be dispensable, but        <br /> we have shown that ARF is essential. We have not ruled out      <br /> the possibility that ARF is involved with GPA2 function,          <br /> only that this cannot be the sole function of ARF, nor have           <br /> we addressed whether the in vitro GOS ADP-ribosylation         <br /> assay is actually relevant to in vivo function. Further,             <br /> cholera toxin has been reported to have no effect on yeast            <br /> adenylate cyclase and no protein acceptor of ADP-ribose has             <br /> been identified in yeast extracts (7). Our results do clearly      <br /> demonstrate that the level of ARF protein is critical for          <br /> proper function. Either underexpression or overexpression             <br /> of ARF protein results in impaired growth or complete          <br /> cessation of growth, depending on the degree of alteration.        <br />    The fluoride sensitivity of arf mutants is intriguing. Al-        <br /> though a model for the mechanism of activation of the a                 <br /> subunits of the trimeric G proteins by fluoride complexes has       <br /> been proposed (2), there is no evidence that fluoride or                <br /> aluminum fluoride complexes bind to or activate any of the          <br /> low-molecular-weight GTP-binding proteins. For the trimer-         <br /> ic G proteins, it is believed that AlF4- is the active species      <br /> for the activating effect (44), resulting from the resemblance       <br /> of this compound to the gamma phosphate of GTP. Thus,                   <br /> liganded GDP plus A1F4- is the equivalent of liganded GTP            <br /> (1). Whereas F- concentrations in the millimolar range are               <br /> required for the in vitro effects, Al3&quot; is only required in the         <br /> </body></html>