<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BioMog: A Computational Framework De Novo <br /> Generation Modification Essential Biomass <br /> Components <br /> Christopher J. Tervo, Jennifer L. Reed* <br /> Department Chemical Biological Engineering, University Wisconsin-Madison, Madison, Wisconsin, United States America <br />  <br />  <br />  <br />      Abstract <br />      The success genome-scale metabolic modeling contingent model&#8217;s ability accurately predict growth and <br />      metabolic behaviors. To date, little focus directed developing systematic methods proposing, <br />      modifying interrogating organism&#8217;s biomass requirements used constraint-based models. To address <span id='am-28' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-29'  ></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>this</span> <br />      gap, biomass modification generation (BioMog) framework created used generate lists biomass <br />      components novo, modify predefined biomass component lists, models <span id='am-95' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-96' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-99'  ></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-108' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span>Escherichia coli</span> (iJO1366) and <br />      <span id='am-48' about='obo:NCBITaxon_70863' typeof='owl:Thing obo:NCBITaxon_267890 obo:NCBITaxon_1 obo:NCBITaxon_22 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135622'><span id='am-49'  ></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Shewanella oneidensis Venkateswaran et al. 1999" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasAlternativeId" content="NCBITaxon:321336" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="PMID:10319494" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Shewanella oneidensis" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Shewanella oneidensis</span> (iSO783) high-throughput growth phenotype fitness datasets. BioMog&#8217;s novo biomass <br />      component lists included, implicitly explicitly, seventy percent components included predefined <br />      biomass equations, resulting novo biomass equations outperformed predefined biomass equations at <br />      qualitatively predicting mutant growth phenotypes percent. Additionally, BioMog procedure quantify <br />      experiments support refute particular metabolite&#8217;s essentiality cell, facilitates determination of <br />      inconsistent experiments inaccurate reaction and/or gene reaction associations. To interrogate metabolite <br />      essentiality, BioMog framework includes experiment generation algorithm allows design experiments <br />      test metabolite essential. Using BioMog, correct experimental results relating essentiality thyA <br />      gene E. coli, perform knockout experiments supporting essentiality protoheme. With capabilities, <br />      BioMog valuable resource analyzing growth phenotyping <span id='am-134' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-135' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-136' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-138'  ></span>data</span> component model developer&#8217;s toolbox. <br />  <br />   Citation: Tervo CJ, Reed JL (2013) BioMog: A Computational Framework De Novo Generation Modification Essential Biomass Components. PLoS <br />   ONE 8(12): e81322. doi:10.1371/journal.pone.0081322 <br />   Editor: Julio Vera, University Erlangen-Nuremberg, Germany <br />   Received May 21, 2013; Accepted October 11, 2013; Published December 5, 2013 <br />   Copyright: &#223; 2013 Tervo, Reed. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: CT supported NHGRI training grant Genomic Sciences Training Program (T32HG002760). Additional support research was <br />   provided grant NSF (NSF 1053712). The funders role study design, data collection analysis, decision publish, preparation the <br />   manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: reed@engr.wisc.edu <br />  <br />  <br />  <br /> Background                                                                            predict growth (false negative prediction). Another approach, <br />                                                                                       GrowMatch [3] groups model-data inconsistencies false <br />    The accuracy constraint-based model predictions (e.g.,                       negatives (model predicted growth, experimental growth) and <br /> growth phenotypes) application models                              false positives (model predicted growth, experimental growth) <br /> metabolic engineering (e.g., strain design biofuel production)                    adds removes metabolic reactions, respectively, to <br /> are contingent biologically accurate biomass equation.                           reconcile individual model-data inconsistencies. A newer ap- <br /> Despite this, numerous computational methods exist                           proach, MIRAGE [7], makes use metabolic flux functional <br /> curate genome-scale models interrogating, hypothesizing                        genomics data identify reactions missing a <br /> refining organism&#8217;s reaction, gene-reaction and/or transcrip-                      model network. None methods, however, used to <br /> tional regulatory networks [1,2,3] comparatively little                      automatically test refine microorganism&#8217;s modeled biomass <br /> done <span id='am-129' about='obo:NCBITaxon_150701' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6694 obo:NCBITaxon_1206794 obo:NCBITaxon_115581 obo:NCBITaxon_6657 obo:NCBITaxon_6682 obo:NCBITaxon_88770 obo:NCBITaxon_6683 obo:NCBITaxon_29966 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-130' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-131' property="rdfs:label" content="Automate" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-133'  ></span>automate</span> generation modification                                (while GrowMatch [3] does include biomass reformulation step <br /> organism&#8217;s biomass requirements. To niche,                          correct false positives manual inspection). <br /> developed novel biomass modification generation                               Moreover, scaling issues inherent design, of <br /> (BioMog) framework means determine, novo, biomass                          methods correct single mispredictions time making <br /> components consistent large numbers (containing                         sequential modifications model considering how <br /> 1000 s unique experiments) high-throughput growth                               modifications impact overall predictive accuracy of <br /> phenotype datasets, becomingly increasingly facile                      model. Such sequential model modifications result in <br /> inexpensive create [4,5].                                                          suboptimal model refinements and, thus, proposed modifi- <br />    To date, numerous methods created                                cation requires subsequent evaluation entire global <br /> available refine constraint-based models using growth pheno-                       dataset ensure overall improved model performance. In <br /> typing data (see [2] recent review). SMILEY [6] works add                      addition, alternate modifications able reconcile the <br /> missing reactions metabolic network correct discrepancies                     model-data inconsistency existing approaches <br /> where organism grows experimentally model does                        identify alternate solutions best agree <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                        1                           December 2013 | Volume 8 | Issue 12 | e81322 <br />                                                                                                    Algorithm Identifying Biomass Components <br />  <br />  <br /> with entire experimental dataset. Thus, developed           Results Discussion <br /> BioMog (Biomass Modification Generation) framework that <br /> can, scalable, parallelizable efficient manner, consider          The BioMog framework uses concept blocked metabolites <br /> experimental results simultaneously set(s) biomass             growth phenotype data determine potential biomass <br /> components best matches experimental growth phenotypes.                components yield fatal non-fatal knockout predictions <br /> Consequently, BioMog proposes biomass requirements best                growth growth experimental results, respectively. Once <br /> match existing information avoiding substantial               include exclude metabolite lists generated all <br /> computational costs associated existing mixed-integer linear           experiments, improved biomass computationally <br /> programming (MILP) refinement approaches. Moreover, BioMog                  proposed novo based existing biomass equation <br /> can used complement existing tools suggesting types          appropriate algorithm. Based new biomass any <br /> of model adjustments improve model predictions.                          relevant statistics teased metabolite lists, BioMog propose <br />                                                                             additional growth phenotype experiments designed to <br />    Building concepts blocked metabolites [8] (i.e., <br />                                                                             generate data specific essentiality particular metabolite <br /> metabolites produced consumed an <br />                                                                             cellular growth. Such experiments complete BioMog cycle <br /> organism), BioMog determines metabolites net production <br />                                                                             performed iteratively improve quantity and <br /> blocked (i.e., metabolites generated) a <br />                                                                             quality evidence supporting essentiality given <br /> particular mutant wild type. We note that <br />                                                                             metabolite. This entire process summarized Figure 1. <br /> flux net production blocked metabolites occur <br /> (e.g., metabolite cycle) net production of <br /> the metabolite possible. For simplicity, refer          The BioMog Framework: An Illustrative Example <br /> metabolites blocked metabolites remainder paper.             For illustrative purposes, demonstrate BioMog <br /> Depending experimental growth phenotype mutant                process small reaction network (shown Figure 2), where <br /> in dataset, set blocked metabolites assigned       media contains metabolites A G (Figure 2A). The <br />                                                                             predefined biomass equation removed network and <br /> of list types, assuming correct network reconstruction: <br />                                                                             sinks added metabolite network. Under these <br /> 1) Exclude Metabolite List: growth observed,                 conditions, wild type network (Figure 2B) able produce <br />    blocked metabolite list included BioMog             metabolites Iex I, blocked the <br />    biomass result false negative growth prediction.               choice media components. Iex I removed <br />    Metabolites list provide negative evidence         consideration possible biomass components their <br />    inclusion biomass.                                                    inclusion suggest wild type incapable of <br />                                                                             growth. For knockout mutant evaluated experimentally, list <br /> 2) Include Metabolite List: growth observed, any <br />                                                                             blocked metabolites (excluding wild type, i.e., <br />    discovered blocked metabolites, added biomass <br />                                                                             Iex I) generated stored include/exclude metabolite <br />    equation, result model recapitulating particular <br />                                                                             lists based knockout phenotype (i.e., lethal non-lethal <br />    experimental result. Metabolites list viable             knockout). Once include/exclude metabolite lists been <br />    candidates biomass components.                                       completed, BioMog propose novo biomass modify the <br />    Once lists created, simple integer program          predefined biomass using simple integer program (IP) that <br /> (IP) constructed propose new biomass novo              agreement model growth predictions growth <br />                                                                             phenotype data maximized (Figure 2C). Depending the <br /> modifying predefined biomass equation components the <br />                                                                             network structure, process result new consumption <br /> computational agreement experimental observations is <br />                                                                             blocked metabolites (i.e., metabolite consuming <br /> maximized. It important note essential biomass <br />                                                                             reaction exists) removal metabolites the <br /> components proposed BioMog useful qualitative <br />                                                                             predefined biomass, naturally behaves outlet many <br /> predictions (i.e., account relative <br />                                                                             essential metabolites. <br /> abundance component and, such, appropriate to <br /> use quantitative predictions pathway fluxes yields without <br /> further experimental biomass composition measurements). Once                Application BioMog <span id='am-111' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-112' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-114' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-115'  ></span><span id='am-116' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span>Escherichia</span> coli <span id='am-118' about='obo:NCBITaxon_22' typeof='owl:Thing obo:NCBITaxon_267890 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135622'><span id='am-119' property="rdfs:label" content="Shewanella" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-121'  ></span><span id='am-122' property="oboInOwl:hasRelatedSynonym" content="Shewanella MacDonell and Colwell 1986" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Shewanella</span> <br /> this set new biomass components proposed, additional            oneidensis Metabolism <br /> experiments designed interrogate essentiality               In <span id='am-45' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-46' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> demonstrate efficacy utility BioMog <br /> given metabolite using methods similar recently proposed             framework, BioMog framework applied models of <br /> FOCAL[9], designs experiments test accuracy                 Escherichia coli (with ,4200 growth phenotype experiments) and <br /> reaction gene-reaction relationships constraint-based                Shewanella oneidensis (with ,1500 fitness experiments) metabolism, <br /> models.                                                                     resulting proposed biomass components generated de <br />    Below, application BioMog genome-scale             novo modifying preexisting biomass equations. BioMog&#8217;s de <br /> models Escherichia coli (iJO1366) <span id='am-33' about='obo:NCBITaxon_211586' typeof='owl:Thing obo:NCBITaxon_267890 obo:NCBITaxon_1 obo:NCBITaxon_70863 obo:NCBITaxon_22 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135622'><span id='am-34' property="rdfs:label" content="Shewanella oneidensis MR-1" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Shewanella oneidensis ATCC 700550" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Shewanella sp. MR-1" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="Shewanella oneidensis str. MR-1" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="Shewanella oneidensis strain MR-1" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasAlternativeId" content="NCBITaxon:60479" datatype="xsd:string"></span>Shewanella oneidensis MR-1</span>        novo proposed biomass components summarized Table 1 <br /> (iSO783) using growth phenotype fitness data                complete details predefined biomass modifications are <br /> organisms [5,10,11] propose novo modified biomass                 provided supplementary materials (Tables S1 S2 in <br /> requirements. We demonstrate new biomass equations                 File S1). In proposing biomasses, metabolites most <br /> outperform qualitative growth predictions predefined           commonly associated biomass amino nucleic <br />                                                                             acids, various currency metabolites, weighted such <br /> counterparts, providing potential insights inconsistent <br />                                                                             selected preferentially alternative <br /> experiments model structures. We subsequently designed and <br />                                                                             metabolites available (see methods details). <br /> conducted additional experiments interrogate essentiality of <br />                                                                                The proposed novo biomass requirements consistently <br /> metabolites existing phenotype data provides no <br />                                                                             outperformed predefined counterparts predicting the <br /> insight provides conflicting results. <br />                                                                             qualitative outcomes growth phenotype experiments (Figure 3). <br />                                                                             Despite improvements growth phenotype predictions, still <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              2                       December 2013 | Volume 8 | Issue 12 | e81322 <br />                                                                                                         Algorithm Identifying Biomass Components <br />  <br />  <br />  <br />  <br /> Figure 1. A flow diagram summarizing BioMog framework. Initially, mutants evaluated computationally determine metabolites <br /> that blocked mutant strains wild type strain. While strictly necessary, mutants substantially deleterious impacts on <br /> the network (.100 blocked metabolites) considered uninformative filtered improve quality proposed biomass components. <br /> Blocked metabolites assigned include/exclude metabolite list based experimentally observed growth phenotype. Once these <br /> lists created, BioMog use information propose novo biomass components modify existing biomass equation. <br /> Additional experiments designed run BioMog informational gaps exist current dataset the <br /> cycle repeated. <br /> doi:10.1371/journal.pone.0081322.g001 <br />  <br /> be seen room improvement, particularly                   properly predict media condition, 123 were <br /> mutants did grow (NG) experimentally. Of 139 E. coli              improperly predicted biomasses, 15 incorrectly <br /> NG mutants novo predefined biomass equations failed             predicted solely predefined biomass just (Db1098 &#8211; <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 3                        December 2013 | Volume 8 | Issue 12 | e81322 <br />                                                                                                     Algorithm Identifying Biomass Components <br />  <br />  <br />  <br />  <br /> Figure 2. Application BioMog illustrative example. (A) For existing model set biomass requirements (metabolites F H), <br /> BioMog capable (depending quality quantity data) generating, novo, organism&#8217;s biomass requirements modifying a <br /> predefined biomass equation. This accomplished removing initial biomass equation network adding sinks metabolite <br /> (not shown). Blocked metabolites determined wild type mutant strains particular experimental condition (B). This process is <br /> repeated growth phenotype experiments data exist. The set blocked metabolites used propose new biomass <br /> equation modify existing (C). Based include/exclude metabolite lists generated example, original biomass equation <br /> composed substrates F H modified adding metabolite C removing F. Since novo biomass relies solely experimental <br /> evidence, important data exist test essentiality different metabolites desires accurate complete <br /> understanding biomass requirements. Here, metabolite H absent proposed novo biomass supporting refuting <br /> evidence existed experimental dataset justify inclusion. <br /> doi:10.1371/journal.pone.0081322.g002 <br />  <br /> a mutant defective deoxythymidine triphosphate, dttp, biosyn-            wild type network (implying mutant result a <br /> thesis) incorrectly predicted BioMog&#8217;s proposed                 growth phenotype given existing metabolic network). These <br /> biomass. For 123 commonly mispredicted mutants, just              mispredicted mutants experimental errors, <br /> 90% did blocked metabolites observed              incomplete erroneous GPR associations, enzyme redundancy <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              4                       December 2013 | Volume 8 | Issue 12 | e81322 <br />                                                                                                                 Algorithm Identifying Biomass Components <br />  <br />  <br />  <br />  Table 1. BioMog proposed novo biomass equations E. coli S. oneidensis. <br />  <br />  <br />  E. coli Biomass                                                                                          S. oneidensis Biomass <br />  <br />  Unique                                     Contains Alternatives                                         Unique                   Contains Alternatives <br />  <br />  2,3-dihydroxybenzoylserine; 5-             2omhmbl; [2Fe-1S] desulfurated iron-sulfur cluster; 4c2me; L- L-Arginine; gdptp; L-    Reduced glutathione; DNA; L- <br />  Formyltetrahydrofolate; L-Glutamate 5-     Arginine; adenosine thiamine triphosphate; Biotin; cardiolipin Histidine; O-Phospho-   Leucine; L-Methionine; 2-Oxo-3- <br />  phosphate; L-Histidine; L-Isoleucine;      (tetraoctadec-11-enoyl, n-C18:1); Coenzyme A; Flavin adenine L-homoserine              hydroxy-4-phosphobutanoate; L- <br />  KDO(2)-lipid IV(A); protoheme; L-Proline   dinucleotide oxidized; L-Leucine; L-Lysine; Murein5p3p_p;                              Phenylalanine; L-Proline; O-Phospho- <br />                                             nadp; pe181; L-Phenylalanine; O-Phospho-L-serine;                                      L-serine; L-Tryptophan; L-Tyrosine <br />                                             Pyridoxamine; Siroheme; Selenite; L-Tryptophan; uagmda <br />  <br />  Unique metabolites compounds alternative maximized objective function. Conversely, metabolites which <br />  possessed alternatives replaced compound adversely impacting objective function. Abbreviations used follows: 2omhmbl, <br />  2-octaprenyl-3-methyl-5-hydroxy-6-methoxy-1,4-benzoquinol, 4c2me, 4-(cytidine 59-diphospho)-2-C-methyl-D-erythritol, murein5p3p_p, linked disacharide <br />  pentapeptide tripeptide murein units (uncrosslinked, middle chain), nadp, nicotinamide adenine dinucleotide phosphate, pe181, phosphatidylethanolamine <br />  (dioctadec-11-enoyl, n-C18:1), uagmda, undecaprenyl-diphospho-N-acetylmuramoyl-L-alanyl-D-glutamyl-meso-2,6-diaminopimeloyl-D-alanyl-D-alanine, gdptp, <br />  guanosine 39-diphosphate 59-triphosphate. <br />  doi:10.1371/journal.pone.0081322.t001 <br />  <br />  <br /> due isozymes, metabolic redundancy presence                         blocked lethal knockout (i.e., high frequency in <br /> alternate metabolic pathways used experimen-                        include lists) likely added biomass equation. <br /> tally, incomplete pathways filled using methods like                  Frequencies alone, however, provide incomplete picture of <br /> GapFill [8]. For example, folA (encoding dihydrofolate reductase)                    selection evidenced numerous modifications the <br /> in E. coli repeatedly reported essential gene                         predefined biomass equation S. oneidensis (metabolites listed in <br /> [10,12,13]; however, present GPR association reaction                   red, Figure 4B). To completely understand BioMog decisions, it <br /> suggests isozyme, folM, compensate loss folA                  necessary know metabolite&#8217;s frequency include/ <br /> resulting model having new blocked metabolites                      exclude lists frequencies brethren (i.e., other <br /> erroneous growth phenotype prediction. Another experimentally                        blocked metabolites co-appear include/exclude <br /> essential gene, guaB (encoding IMP dehydrogenase) converts imp                       lists). Assuming correct metabolic network, blocked <br /> (inosine 5-phosphate) xmp (xanthoine 5-phosphate)                            metabolite appears include list metabolites, is <br /> guanine biosynthesis; however, model route                        possible selection metabolites be <br /> make xmp imp (imp R inosine R hypoxanthine R xanthine                           preferable order obtain accurate biomass equation <br /> R xmp) new blocked metabolites                                fewest number components. The metabolite, <br /> DguaBmutant wildtype. While structural issues                        protein_son_aerobic (representing average protein molecule <br /> model (e.g., GPR associations alternative pathways)                          cell) iSO783 example replacing the <br /> cause discrepancies model predictions data,                         metabolite predefined biomass brethren <br /> such discrepancies problems                              metabolites biomass improves agreement experimental <br /> experimental data itself. For example, sucC, (encoding succinyl-                     observations. In viable mutant DSO2085, protein_son_aerobic <br /> CoA synthetase)was classified experimentally essential; however,                  sole blocked metabolite (beyond observed wild <br /> other reports indicate non-essential [12,13] agreement                 type network) providing evidence exclusion the <br /> with model predictions. The remaining 10%                                 biomass. Moreover, majority growth mutants, <br /> mispredicted NG mutants did blocked metabolites,                        protein_son_aerobic blocked various <br /> inclusion blocked metabolites biomass                       amino acids precursors protein production. As a <br /> contradicted results experiments. For example, number                     consequence, BioMog determines composing biomass <br /> of mutants (Db0722 [sdhD], Db0588 [fepC], Db0590 [fepD], Db0592                      using essential amino acids protein_son_aerobic will <br /> [fepB], Db1252 [tonB] Db3919 [tpiA]) Fe-enterobactin                         improve model&#8217;s qualitative growth predictions. <br /> (feenter) blocked metabolite; however, excluded                       Having evaluated decision making process proposing de <br /> BioMog&#8217;s proposed biomass requirements numerous                               novo modified biomasses, compared novo BioMog <br /> conflicting experimental results regarding metabolite&#8217;s essen-                   biomass components predefined model biomass components <br /> tiality (e.g., Db0584 [fepA], viable mutant feenter                (Figure 5). A detailed biomass component comparison is <br /> only blocked metabolite). Consequently, improve qualitative                   provided Tables S3 S4 File S1. As seen E. <br /> predictions experiments require including                         coli, 70% predefined biomass components captured <br /> additional regulatory information, repeating conflicting                        BioMog&#8217;s proposed biomass, alternative downstream <br /> experiments, modifying reaction network gene-protein-                      metabolites considered. When proposing E. coli S. oneidensis <br /> reaction (GPR) associations.                                                         biomass requirements, BioMog select metabo- <br />    The BioMog framework provides information                          lites related predefined biomass are <br /> many experiments support/refute essentiality given                          located farther downstream. While upstream predefined metabo- <br /> metabolite. This extracted frequency                              lites explicitly included BioMog proposals, they <br /> appearance given metabolite include/exclude                              implicitly essential media conditions examined when <br /> metabolite lists. These metabolite list frequencies select                       downstream metabolites included novo biomass. <br /> predefined biomass components (E. coli metabolites                           For E. coli analysis, BioMog biomass components <br /> experimental evidence excluded improve                        upstream predefined biomass components. <br /> visualization) provided Figure 4. Unsurprisingly, items                  For S. oneidensis, novo Biomog biomass components were <br /> more experimental evidence indicating metabolite                      downstream predefined components unique de <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                       5                          December 2013 | Volume 8 | Issue 12 | e81322 <br />                                                                                                     Algorithm Identifying Biomass Components <br />  <br />  <br />  <br />  <br /> Figure 3. Percent growth phenotype agreements BioMog De Novo biomass predefined biomass. Unsurprisingly, the <br /> BioMog biomass equation consistently outperforms curated biomass entire dataset. Numbers reported given bar indicate the <br /> total number experiments category. <br /> doi:10.1371/journal.pone.0081322.g003 <br />  <br /> novo biomass, exception amino acids              frequency appearance include/exclude metabolite lists <br /> included biomass precursors                     (Figure 6), algorithm did sufficient evidence to <br /> protein_son_aerobic. For E. coli S. oneidensis, seven           include amino acids biomass components though <br /> proposed metabolites respectively unique novo                 analytical data indicates amino acids (in addition other <br /> biomass (i.e., did alternatives               components) present biomass [11]. To address this <br /> directly included upstream downstream metabolites              shortcoming, treated analytically measured biomass <br /> in predefined biomass). The unique elements BioMog                components additional experimental data points BioMog <br /> biomasses (seven components E. coli thirteen S.                  fit. By doing this, capture supporting evidence these <br /> oneidensis, including amino acid precursors) critical       analytical measurements provide, allowing BioMog to <br /> improving growth phenotype predictions                 determine phenotypic data contradicts inclusion a <br /> the novo biomass modification predictions (Figure 5B). It          metabolite biomass. De novo modified biomass <br /> important note metabolites unique predefined              requirement proposals S. oneidensis using additional <br /> biomass absent novo biomass              information displayed Table 2 Table 3. As be <br /> no data support inclusion data suggests           seen, proposals similar using just growth <br /> metabolites essential growth.                                    phenotypes (Tables 1 S2); however, amino acids <br />                                                                              additional analytically measured currency metabolites <br /> Inclusion Analytical Biomass Measurements BioMog                       (e.g., nadh amp) added. Additionally, list <br />     While BioMog designed make proposals based growth               items removed predefined biomass largely same, <br /> phenotype data, account additional                     extracellular crosslinked peptidoglycan (peptx_e Figure 4B) <br /> experimental data analytically measured biomass compo-               retained based experimental measurements, even <br /> sitions. Such data especially useful metabolites            appeared equal numbers include exclude lists <br /> creating informative mutant experiment prohibitive               indicating growth phenotypes inconclusive regarding its <br /> due time complexity (see Experimental Design                     essentiality. <br /> section below). This biomass composition data used to <br /> improve BioMog proposals, amino acids S.                Experimental Design <br /> oneidensis. It known organisms generally require              In addition, proposing modifying biomasses, BioMog is <br /> twenty amino acids growth; however, based just                    capable proposing experiments interrogate essentiality of <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               6                      December 2013 | Volume 8 | Issue 12 | e81322 <br />                                                                                                         Algorithm Identifying Biomass Components <br />  <br />  <br />  <br />  <br /> Figure 4. Frequency appearance include/exclude metabolite lists. Frequency select predefined biomass metabolites include <br /> (red bars, corresponding experiments mutants grow) exclude (blue bars, corresponding experiments mutants grow) <br /> metabolite lists, difference (green bars) E. coli (A) S. oneidensis (B). For E. coli, 29 72 predefined <br /> components (40%) direct supporting refuting experimental evidence (i.e., red blue bars zero). A positive difference between <br /> the include exclude frequency indicates potential improvement agreement including metabolite biomass component. Note that <br /> a positive difference does ensure metabolite&#8217;s inclusion lead maximal agreement score. Selection biomass components from <br /> multiple blocked metabolites associated given experiment potentially replicate experimental phenotypes minimizing <br /> overall disagreements (see text details). Red labeled metabolites indicate recommended removal biomass modification <br /> algorithm. Metabolite abbreviations match used metabolic models. <br /> doi:10.1371/journal.pone.0081322.g004 <br />  <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 7                        December 2013 | Volume 8 | Issue 12 | e81322 <br />                                                                                                Algorithm Identifying Biomass Components <br />  <br />  <br />  <br />  <br /> Figure 5. Comparison predefined biomass BioMog&#8217;s novo (top) modified biomass (bottom). Numbers bars indicate <br /> the number metabolites category. iJO1366 iSO783 genome-scale models used E. coli S. oneidensis, respectively. <br /> doi:10.1371/journal.pone.0081322.g005 <br />  <br /> any given metabolite (see File S1 File S2 methods                inorganic compounds, deletion genes essential to <br /> and implementation respectively). Various experimental                   metabolite production generated E. coli predefined <br /> proposals involving use minimal media lacking essential           biomass components BioMog indicated little no <br />  <br />  <br /> PLOS ONE | www.plosone.org                                           8                      December 2013 | Volume 8 | Issue 12 | e81322 <br />                                                                                                                 Algorithm Identifying Biomass Components <br />  <br />  <br />  <br />  <br /> Figure 6. Frequency Amino Acid appearance include/exclude metabolite lists S. oneidensis. While amino acids were <br /> not included original novo biomass proposals BioMog (Table 1), growth phenotypes preclude addition shown here. <br /> Their exclusion BioMog result insufficient evidence inclusion based soley growth phenotype experiments performed. <br /> Metabolites green selected BioMog essential biomass components. Metabolites blue evidence to <br /> support inclusion omitted redundant metabolites matching experimental results (e.g., isoleucine, <br /> leucine valine biosynthesis pathways share enzymes). Metabolites colored red growth phenotype evidence support <br /> their inclusion exclusion biomass. <br /> doi:10.1371/journal.pone.0081322.g006 <br />  <br />  <br />  <br />  Table 2. BioMog Proposed De Novo Biomass S. oneidensis using Growth Phenotype Experimentally Measured Biomass. <br />  <br />  <br />  Amino Acids                                                         Present Predefined Biomass              No Equivalent Predefined Biomass <br />  <br />  L-Alanine                      L-Leucine                            Putrescine                                 O-Phospho-L-homoserine <br />  L-Arginine                     L-Lysine                             nad                                        O-Phospho-L-serine <br />  L-Asparagine                   L-Methionine                         nadh                                       gdptp <br />  L-Aspartate                    L-Phenylalanine                      nadp                                       Reduced Glutathione <br />  L-Cystine                      L-Proline                            nadph                                      2-Oxo-3-hydroxy-4-phosphobutanoate <br />  L-Glutamine                    L-Serine                             dna_son <br />  L-Glutamate                    L-Threonine                          Glycogen <br />  <span id='am-23' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-24'  ></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Glycine</span>                        L-Tryptophan                         peptx_e <br />  L-Histidine                    L-Tyrosine                           rna_son <br />  L-Isoleucine                   L-Valine                             amp <br />  <br />  When given additional information biomass components measured experimentally, BioMog able propose physiologically accurate components, such <br />  including amino acids. The abbreviation peptx_e stands extracellular cross-linked peptidoglycan, dna_son rna_son stand average DNA RNA <br />  macromolecules. Other abbreviations match Table 1. <br />  doi:10.1371/journal.pone.0081322.t002 <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                        9                         December 2013 | Volume 8 | Issue 12 | e81322 <br />                                                                                                                     Algorithm Identifying Biomass Components <br />  <br />  <br />  <br />  Table 3. BioMog Modified Predefined Biomass S. oneidensis using Growth Phenotype Experimentally Measured Biomass. <br />  <br />  <br />  Added                                                                      Unmodified        Removed <br />  <br />  L-Alanine         L-Leucine             gdptp                              amp               1,2-Diacyl-sn-glycerol 3-phosphate           Phosphatidylglycerol <br />  L-Arginine        L-Lysine              Reduced Glutathione                dna_son           1,2-Diacylglycerol                           protein_son_aerobic <br />  L-Asparagine      L-Methionine          2-Oxo-3-hydroxy-4-                 Glycogen          5-Methyltetrahydrofolate                     Spermidine <br />                                          phosphobutanoate <br />  L-Aspartate       L-Phenylalanine       O-Phospho-L-homoserine             nad               Acetyl-CoA                                   Succinyl-CoA <br />  L-Cystine         L-Proline             O-Phospho-L-serine                 nadh              acyl-glycerophosphoethanolamine              UDP-glucose <br />  L-Glutamine       L-Serine                                                 nadp              acyl-glycerophosphoglycerol <br />  L-Glutamate       L-Threonine                                              nadph             Coenzyme-A <br />  Glycine           L-Tryptophan                                             peptx_e           Flavin adenine dinucleotide <br />  L-Histidine       L-Tyrosine                                               Putrescine        lipopolysaccharide <br />  L-Isoleucine      L-Valine                                                 rna_son           Phosphatidylethanolamine <br />  <br />  While BioMog able capture amino acid essentiality, propose removal certain metabolites predefined biomass equation based the <br />  present model structure results growth phenotype experiments. These removals indicate metabolites non-essential may <br />  instead indicate experimental network structural issue. Abbreviations match Table 1 2 addition protein_son_aerobic represents an <br />  average protein macromolecule. <br />  doi:10.1371/journal.pone.0081322.t003 <br />  <br />  <br /> experimental evidence existed experimental evidence                              Since quality BioMog solutions good the <br /> conflicting (see supplementary dataset S1). Experiments                                  experimental data used generate them, important a <br /> testing essentiality protoheme (pheme) deoxythy-                          large, diverse accurate experimental dataset. This especially <br /> midine triphosphate (dttp) conducted. The                                true generating biomass components novo, the <br /> experiment designed test essentiality pheme                        method include metabolites biomass is <br /> the double deletion hemF hemN. We unable                                  experimental evidence support (or refute) inclusion as <br /> construct viable double knockout mutant &#8211; implying double                          seen S. oneidensis. Presently, numerous methods <br /> knockout lethal LB medium &#8211; indicating metabolite                         used generate high-throughput growth fitness <br /> likely essential. The second experiment, designed test                        data construct higher order genetic knockouts as <br /> essentiality dttp, performed Orth et al.[10],                      TagModule [17], TRMR [18], MAGE [19]. Using such <br /> that growth phenotyping DthyA mutant glucose M9                           methods, conjunction BioMog experiment generation <br /> medium. However, positive growth result contradicted findings                        tool, ensure broad evaluation organism&#8217;s essential <br /> of publications [12,14,15] led removal dttp                          metabolites. Nonetheless, organisms robust metabolic <br /> from BioMog&#8217;s biomass. Since BioMog&#8217;s decision exclude dttp                           networks, higher order knockouts needed detect <br /> from biomass solely DthyA experiments,                             essential metabolites. With E. coli iJO1366 network, for <br /> decided repeat results based supporting literature.                        example, 50% metabolites (after filtering mutants <br /> Thus, growth phenotype experiments performed evaluating                             resulting large numbers blocked metabolites) direct <br /> the DthyA mutant&#8217;s fitness LB medium supplemented                                supporting refuting evidence essentiality when <br /> thymine, glucose M9 minimal media, glucose M9 media                                  considering experiments involving single gene deletions. <br /> supplemented thymine. Our results (see Figures S1&#8211;S3                             Perhaps critically, 40% components E. coli <br /> File S1 growth curves) suggest DthyA thymine                               predefined biomass explicit evidence inclusion in <br /> auxotroph, incapable growth glucose M9 media,                                   biomass based single knockout datasets considered here <br /> agreement findings Weiss[15]. Based experi-                             [10]. Thus, properly interrogate accuracy organism&#8217;s <br /> ments, seen BioMog framework                                 biomass composition critical targeted higher order <br /> valuable tool, discovering new metabolite require-                          mutants created. <br /> ments growth, identifying potentially inaccurate                           From mathematical perspective, BioMog works the <br /> experimental results large dataset.                                             best biomass requirements current model structure <br />                                                                                          (reactions GPR associations) provided experimental <br /> Conclusions                                                                              results. As such, proposed novo modified biomass <br />                                                                                          components represent needs organism assuming the <br />    The BioMog framework provides facile, fast scalable                             model experiments accurate portrayal the <br /> approach task generating biomass related hypotheses.                           organism&#8217;s genetic metabolic capabilities. If known biomass <br /> Such approach used identify organism specific                               components proposed BioMog possible that <br /> biomass components improve model&#8217;s predictive power                                model experiments incorrect. This readily <br /> compared using predefined biomass equation. Alternatively,                          apparent BioMog suggests removal previously <br /> future efforts focus using growth phenotype data                             reported essential metabolites dttp, Cu2+ Ca2+ for <br /> conjunction 13C multiomics data                                E. coli protein_son_aerobic S. oneidensis. In instances, it <br /> improve refine essential biomass components. Since                             immediately interpreted metabolites are <br /> approaches require biomass defined priori,                           inessential growth (especially compounds been <br /> should possible combine metabolic flux analysis (MFA) [16]                         measured experimentally component biomass) but <br /> and BioMog.                                                                              instead result considered red flag, indicating <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                         10                          December 2013 | Volume 8 | Issue 12 | e81322 <br />                                                                                                    Algorithm Identifying Biomass Components <br />  <br />  <br /> additional experiments needed verify accuracy                respectively. The DzupTand DyrbG mutants ones <br /> validity model structure experiments. In               contain Ca2+ Cu2+ blocked metabolites both <br /> aspect, BioMog behaves hypothesis generation model                 mutants viable, suggesting incomplete picture of <br /> interrogation tool easy traceability (as exclude metabolite        copper calcium transport cell membrane the <br /> lists allows facile determination experiments conflict         non-essentiality metabolites. While been <br /> with metabolites classified essential).                            difficult pick entire dataset, using BioMog&#8217;s blocked <br />     Because BioMog depends model structure order               metabolite frequencies makes finding areas metabolism needing <br /> make biomass proposals, used conjunction               model improvement easier. Consequently, experimenters <br /> other model refinement GrowMatch [3], SMILEY [6]                confident quality defined biomass composition, <br /> other knowledge gap filling algorithms refining metabolic               BioMog valuable component model curator&#8217;s toolbox <br /> models experimental results conflict known biomass                supplements existing approaches [2]. <br /> components. By using tools conjunction,                  Accurate knowledge organism&#8217;s essential metabolites is <br /> possible add remove reactions GPR associations,          critical model&#8217;s predictive utility. Constraint-based analyses <br /> a particular gene knockout resulting phenotype                  rely model&#8217;s biomass equation determine a <br /> consistent known biomass components. For example,                  particular genetic modification fatal. Failure accurately <br /> biotin synthesis pathway incomplete latest S. oneidensis         capture aspect cell costly terms time and <br /> genome-scale model enzyme producing precursor             money models used experimental design, as <br /> pimelyl-ACP recently discovered [20]. Consequently,             strain design metabolic engineering [21]. Such metabolite <br /> would impossible BioMog predict essentiality           essentiality information valuable development <br /> of biotin; however, biotin precursors S-Adenosyl-L-              new antimicrobial agents facilitating identification of <br /> methionine predicted alternative biomass component.            novel drug targets using computational methods [22,23,24,25]. <br /> Any network structural issues cause metabolite net             For application particular, finding unique essential <br /> production blocked wildtype strain prevent BioMog&#8217;s             metabolites valuable selective targeted <br /> ability propose metabolites biomass. There            treatment specific pathogenic <span id='am-42' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>. By generating more <br /> number net production blocked metabolites wildtype (134           representative organism specific biomass components, Bio- <br /> and 109 E. coli S. oneidensis, respectively)            Mog potential improve performance algorithms <br /> biomass candidates inability synthesize precursors          use constraint-based metabolic models. <br /> due various attached cofactors (e.g., ACP trna) which <br /> there synthesis pathway included model. We should <br />                                                                             Methods <br /> note determining net production blocked metabolites <br /> sinks added metabolites network, metabolites             The biomass modification generation (BioMog) framework <br /> will blocked precursors produced (an            allows identification required biomass components from <br /> inability degrade by-products cause metabolite          high-throughput growth fitness experiments. The framework <br /> net production blocked BioMog). If net                     enables scientists systematically make metabolic additions or <br /> production blocked metabolites considered biomass              removals predefined biomass create new biomass <br /> components, model need gap filled               equations novo. Subsequently, experimentalists use the <br /> metabolite produced using methods like                   BioMog framework recommend additional wet lab experiments <br /> SMILEY GrowMatch adding artificial source reactions               interrogate essentiality particular metabolites which <br /> (e.g., using reaction notation, RACP) precursors. Nonetheless,          data lacking contradictory. This process accomplished <br /> while various structural issues prevent BioMog                     series linear, integer mixed integer programs (LP, <br /> proposing essential end product used biomass,            IP MILP respectively), described in following <br /> capture essential upstream precursors produced           sections. <br /> network. As mentioned above, biotin net production <br /> blocked metabolite S. oneidensis, S-Adenosyl-L-methionine,             Genome-Scale Models <br /> biotin precursor, alternative proposed biomass                   BioMog applied genome-scale models Escherichia coli <br /> candidates. Thus, process model refinement automated <br />                                                                             (iJO1366) Shewanella oneidensis MR-1 (iSO783). While iJO1366 <br /> biomass proposals iterative model is <br />                                                                             evaluated is, thirty-five reactions added iSO783 to <br /> refined, BioMog used modify biomass unexpected <br />                                                                             reflect additional knowledge regarding organism&#8217;s metabolic <br /> results investigated refined using existing tools. <br />                                                                             capabilities. These modifications summarized supplemen- <br />     While valuable hypothesis experimental generation <br />                                                                             tary dataset S2). <br /> tool, BioMog value identifying experiments with <br /> erroneous contradictory results. Researchers use <br /> blocked metabolite lists assess experiments support             Finding Blocked Metabolites <br /> refute essentiality particular biomass component,           BioMog functions determining metabolites be <br /> which experiments appear inconsistent experi-             produced removed the wild type strain, <br /> mental model inaccuracies. Moreover, BioMog biomass                      particular mutant. Such metabolites referred as <br /> modification proposals point potential structural           blocked metabolites. The discovery blocked metabolites was <br /> flaws existing genome-scale model&#8217;s GPR reaction                   accomplished using following MILP similar used <br /> networks. For example, recommendation removing calcium                  GapFind algorithm [8]: <br /> and copper E. coli biomass appears direct <br />                                                                                                                 X <br /> consequence incomplete depiction transport systems                                          max             zi                  &#240;1&#222; <br /> for inorganic compounds. The iJO1366 model includes                                           vi ,xi ,zi <br />                                                                                                                    i[M <br /> single calcium copper transport reactions cytoplasm <br /> that associated zupT (b3040) yrbG (b3196), <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             11                       December 2013 | Volume 8 | Issue 12 | e81322 <br />                                                                                                      Algorithm Identifying Biomass Components <br />  <br />  <br />                                                                               phenotyping data, following integer program (IP) be <br />                      X                                                        solved propose, novo, optimal biomass: <br />                              Sij :vj {xi ~0, Vi[M                 &#240;2&#222; <br />                      i[R\m                                                                                       X              X            X <br />                                                                                         max Agreement ~                  ne z         ge z          wi    &#240;7&#222; <br />                                                                                                                  e[NG           e[G          i[EM <br />  <br />                              vj &#402;0, Vj[R\UL                       &#240;3&#222; <br />  <br />                                                                                                   ge &#402;1{wi , V(i,e)[Ex(i,e)                               &#240;8&#222; <br />                              vj &#167;0, Vj[R\LL                       &#240;4&#222; <br />                                                                                                             X <br />                                                                                                     ne &#402;               wi , Ve[NG                         &#240;9&#222; <br />                                                                                                            i[In(i,e) <br />                         xi &#167;zi   vmin , Vi[M                      &#240;5&#222; <br />  <br />  <br />                                                                                                         ne ,ge ,wi [f0,1g                                &#240;10&#222; <br />                   Network Specific Constraints                    &#240;6&#222; <br />  <br />                                                                                  Where ne ge binary variables indicating agreement with <br />    Where zi binary variable indicating metabolite i <br />                                                                               growth growth experiments respectively, wi binary <br /> can produced, vj free variable magnitude flux <br />                                                                               variable indicating metabolite biomass component, <br /> through reaction j, xi acts sink metabolite used <br />                                                                               In(i,e) Ex(i,e) mappings blocked metabolite an <br /> as <span id='am-18' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Indicator" datatype="xsd:string"></span><span id='am-22'  ></span>indicator</span> metabolite production. M set all <br />                                                                               experimental include exclude metabolite lists, <br /> metabolites, R set reactions, m set predefined <br />                                                                               respectively, EM optional set containing metabolites that <br /> biomass components (if exist), UL set reactions with <br /> non-zero upper limits LL set reactions non-zero            experimentally measured microbe&#8217;s biomass. <br /> lower limits. The parameter vmin sets minimum production                    For analyses EM left set, exception <br /> threshold set 0.0005. Network specific constraints refer           S. oneidensis analysis using analytical biomass measurements. <br /> to constraints particular given genome-scale                While purposes paper, possible to <br /> model. For example, constraints force reaction fluxes zero            weight individual experiments Eq. 7 based one&#8217;s confidence <br /> by default, fix P/O ratios set oxidase ratios fixed         particular result (e.g., independent repetitions particular <br /> value. Blocked metabolites zi 0                experiment). Once maximal experimental agreement is <br /> determined mutant wild type strains used                   determined, biomass equation best agrees experimental <br /> experiments. A set difference performed               results generated. When, multiple possible solutions exist, <br /> and remaining blocked metabolites considered potential                negative weights added select common metabolic <br /> biomass equation candidates inclusion exclusion.                       components preferentially including new objective <br />    Note formulation written mass               (Eq. 11) (e.g., amino acids) positive weights used find <br /> actively removed sinks (Eq. 2). As                      minimal number required biomass components (in case all <br /> consequence, MIP unique feature finding                        metabolites positive weight). <br /> metabolites associated metabolic dead ends, as,                                                             X                X <br /> metabolites participating cycles depleted                          min WeightedObj ~                     wi {0:9           wi     &#240;11&#222; <br /> the violating conservation mass. For example,                                                      i[M\W             i[W <br /> the S. oneidensis mutant DSO4249, succoa blocked metabolite <br /> even flux metabolite possible running the <br />                                                                                                          X               X            X <br /> citric acid cycle. This means blocked metabolites                             FixObj ~           ne z           ge z          wi          &#240;12&#222; <br /> may implicitly essential growth, removed                                         e[NG             e[G          i[EM <br /> the biomass equation. <br />  <br /> Validation Blocked Metabolites Generation                                                  FixObj ~ Agreement                                  &#240;13&#222; <br /> Blocked Metabolite Lists <br />     Once blocked metabolites determined all <br /> experiments, mutants associated .100 blocked metabolites                    Here, W set weighted metabolites preferential <br /> are filtered consideration deemed uninformative              inclusion biomass, FixObj, objective score obtained <br /> as filtered mutants deletion essential           running LP using Eq. 7&#8211;10, WeightedObj new <br /> early reaction (e.g., glucose transport). The set blocked                  weighted objective function. While shown, Eq. 8&#8211;10 should <br /> metabolites given mutant given experiment number                 included Eq. 11&#8211;13 weighted LP in <br /> (e) organized include (In), mutant does grow,              order problem solved correctly. <br /> exclude (Ex), mutant does grow, metabolite lists. NG set <br /> of experiments mutants did grow G set                 Modify Biomass Components <br /> experiments mutants grew.                                                   In addition generating biomass equation novo, also <br />                                                                               possible begin existing biomass equation modify it <br /> Generate De Novo Biomass Components                                           include likely biomass components remove compo- <br />   Once complete include exclude metabolite lists                nents fail match experimental results. This allows to <br /> created based blocked metabolite lists growth                      include components supporting growth phenotype <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               12                       December 2013 | Volume 8 | Issue 12 | e81322 <br />                                                                                                           Algorithm Identifying Biomass Components <br />  <br />  <br /> data exists, demonstrated essential using <br /> alternative methods:                                                                                               X <br />                                                                                                            max               Sinki              &#240;20&#222; <br />                                                                                                                  i[Biomass <br /> max Obj ~ <br />         X      X      X       X      X                             &#240;14&#222; <br /> 100   (   ne z   ge z   wi ){   wi {   0:5   yi                                                X <br />                                                                                                       Sij :(vj,for {vj,rev ) {xi ~0, Vi[M       &#240;21&#222; <br />         e[NG       e[G        i[EM           i[M        i[M <br />                                                                                                 j[R <br />  <br />  <br />  <br />                    ge &#402;1{wi {yi , V(i,e)[Ex(i,e)                   &#240;15&#222; <br />                                                                                                          xi &#167;Sinki , Vi[Biomass                 &#240;22&#222; <br />  <br />                             X <br />                     ne &#402;               wi {yi , Ve[NG              &#240;16&#222; <br />                                                                                                            xi ~0, Vi6 [Biomass                  &#240;23&#222; <br />                            i[In(i,e) <br />  <br />  <br />  <br />  <br />                              wi ~ 1, Vi[m                          &#240;17&#222;                               vj,for &#402;0, Vj[KillForwardRxn              &#240;24&#222; <br />  <br />  <br />  <br />                             yi ~ 0, Vi[M\m                         &#240;18&#222;                           vj,rev &#402;0, Vj[KillBackwardRxn                 &#240;25&#222; <br />  <br />  <br />  <br />                            ne ,ge ,wi ,yi [ f0,1g                  &#240;19&#222;                                       Sinki [f0,1g                      &#240;26&#222; <br />  <br />  <br />   Here yi binary variable removes biomass components                  Here Biomass set novo biomass components Sinki is <br /> experiments disagree inclusion, wi binary variable           binary variable indicating flux sink xi. For <br /> adds biomass components experiments support inclusion,                 metabolite contained novo biomass, the <br /> and m set metabolites predefined biomass equation.             metabolite individually removed (i.e., all <br />                                                                                 associated reaction fluxes consume metabolite fixed <br />                                                                                 zero), MIP given Eq. 20&#8211;26 solved. If cell is <br /> Precursors Alternative Biomass Components <br />                                                                                 viable metabolite present (i.e., cell only <br />    For purpose paper, alternative metabolites any <br />                                                                                 produce biomass components metabolite present), <br /> compound replace proposed biomass compound <br />                                                                                 compound declared essential precursor. This <br /> without negatively impacting objective function (Eq. 7). To <br />                                                                                 process repeated using predefined biomass determine if <br /> determine alternative metabolites, wi, proposed biomass <br />                                                                                 novo components upstream. In LP, <br /> components fixed 1 single metabolite, <br />                                                                                 KillForwardRxn KillBackwardRxn remove consuming reactions <br /> k, alternative metabolite sought. For this <br />                                                                                 associated targeted metabolite forward reverse <br /> single metabolite, wk set 0. The novo minimal biomass LP <br />                                                                                 directions respectively, well, forward (reverse) reaction <br /> is resolved (Eq. 8&#8211;13) using variant objective <br />                                                                                 upper (lower) limit equal zero. <br /> (Eq. 11) metabolites preferentially weighted (i.e., <br /> the set W Eq. 11). If solution exists, newly <br /> added metabolite alternative metabolite, k. The newly                 Addition Dead-End Metabolites Biomass <br /> discovered alternative metabolite&#8217;s indicator value subsequently                Due addition sinks metabolites finding <br /> fixed 0, process repeated problem                   blocked metabolites BioMog framework (Eq. 1&#8211;6), is <br /> infeasible possible alternatives metabolite            possible biomass equation generated biomass <br /> exhausted.                                                                      components proposed using Eq. 8&#8211;13 result growth <br />    Precursor metabolites defined upstream compounds                 predictions cases model supposed predict <br /> are essential production biomass metabolite. As such,              growth (i.e., ge equals 1). This occurs by-product a <br /> including particular metabolite biomass implicitly makes               proposed biomass component way consumed in <br /> that metabolite&#8217;s precursors essential. Thus, metabolites                   network experiment (e). The presence sinks these <br /> within predefined biomass precursors                by-products Eq. 2 allows biomass components be <br /> the novo biomass components considered implicitly                     produced evaluating blocked metabolites. As result these <br /> included novo biomass conditions tested. Since                dead-end by-products need way consumed the <br /> precursor lists depend nutritional environment cell,            model, adding additional consuming <br /> robust media modeled included possible conditions              reactions model including biomass equation <br /> tested experimentally (glucose, succinate lactate E. coli           well. For example, order correct erroneous predictions due <br /> lactate, pyruvate, ammonium nitrate S. oneidensis). To                  dead end by-products, metabolites (S-adenosyl-L- <br /> determine precursors, reaction fluxes, vj, decoupled              homocysteine, glycolaldehyde oxidized glutathione) were <br /> forward reverse directions vj = vj,for2vj,rev vj,for        added iJO1366 BioMog proposed novo biomass behave <br /> and vj,rev positive variables. Then, following                     sinks by-products. To determine dead end by- <br /> formulation used:                                                           products, following formulation used: <br />  <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 13                         December 2013 | Volume 8 | Issue 12 | e81322 <br />                                                                                                                   Algorithm Identifying Biomass Components <br />  <br />  <br />                                                                                        lactact aerobic). Additionally, essential genes Keio <br />                                        X                                               collection [12,26] included growth mutants under <br />                                  min         xi                          &#240;27&#222; <br />                                        i[M <br />                                                                                        glucose aerobic conditions. For Shewanella, fitness score less <br />                                                                                        21 indicated growth greater 21 indicated <br />                                                                                        growth. From dataset, tested conditions (lactate <br />                         X                                                              aerobic, pyruvate aerobic, lactate anaerobic nitrate <span id='am-124' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-125' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-127' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-128'  ></span>electron</span> <br />                               Sij :vj {xi ~0, Vi[M                       &#240;28&#222; <br />                                                                                        acceptor). <br />                         j[R <br />  <br />                                                                                        Strains <br />                                                                                           All E. coli strains used experiments derived the <br />                               vj &#402;vupper <br />                                    j     ,   Vj[R                        &#240;29&#222; <br />                                                                                        Keio collection [12]. The following single knockout mutants were <br />                                                                                        used: thyA::kan, hemF::kan, hemN::kan parental <br />                                                                                        strain, E. coli K-12 BW25113. Two double mutants (DhemN::kan <br />                            {vj &#402;vlower <br />                                  j     , Vj[R                            &#240;30&#222;          DhemF, DhemF::kan DhemN) attempted first <br />                                                                                        removing kan gene single knockouts using FLP <br />                                                                                        recombinase [27] moving second mutation the <br />                                  vBiomass wE                             &#240;31&#222;          single knockout using P1 transduction [28]. However, no <br />                                                                                        kanamycin resistant double knockouts found. P1 phage <br />                                                                                        efficacy recipient strain sensitivity verified using P1 <br />                    Network Specific Constraints                          &#240;32&#222;          transduction original mutation kanomycin <br />                                                                                        sensitive strains (i.e., infecting P1 phage DhemN::kan <br />                                                                                        strains kanomycin sensitive DhemN strain DhemF::kan into <br />    Where vBiomass flux newly proposed novo                       DhemF) finding robust growth kanomycin plates following <br /> biomass equations E small number (e.g., 0.001). Network                       P1 transduction. <br /> specific constraints (Eq. 32) covers additional constraints <br /> specific network (e.g., P/O ratios iJO1366). To avoid                        Growth Phenotype Plate Experiments <br /> erroneous results predefined biomass behaving sink,                       All strains grown triplicate unless noted. All <br /> the predefined biomass equation removed stoichio-                         strains pre-cultured approximately 24 hours 90% (V/ <br /> metric matrix subsequent analysis involving BioMog                         V) M9 minimal medium supplemented 2 g/L glucose and <br /> biomass equation. All non-zero xi, considered dead end                       10% (V/V) LB 125 mg/mL thymine. Cells washed twice <br /> by-products required biomass components. These                            M9 minimal media containing carbon source remove <br /> fixed addition metabolite biomass equation                      residual glucose LB precultures. Cells then <br /> adding consuming reactions network. This process                            resuspended different media, M9 minimal media 2 g/L <br /> repeated experiments model supposed                           glucose M9 minimal media 2 g/L glucose 125 mg/mL <br /> predict growth (ge = 1). Additionally, cuts added                            thymine, starting OD600 measurement ,0.05 <br /> determine alternative solutions given experiment fixing                     cultured 3 days Tecan plate reader 37uC. <br /> a proposed sink 0. <br />                                                                                        Supporting Information <br /> FBA Testing <br />    To test qualitative performance novo biomass                          Dataset S1 <br /> requirements, FBA solved described Feist et al. [13]                      (XLSX) <br /> for growth non-growth phenotype experiment                          Dataset S2 <br /> data collected using predefined biomass, as,                    (XLSX) <br /> novo variant. Since BioMog makes qualitative proposals, an <br /> arbitrary stoichiometric value 20.0002 assigned                         File S1 <br /> essential metabolite proposed. To prevent predefined biomass                       (DOCX) <br /> from behaving sink, core wild type biomasses                         File S2 <br /> were removed iJO1366 model evaluating de <br />                                                                                        (ZIP) <br /> novo biomass components. <br />                                                                                        Author Contributions <br /> Description Growth Phenotype Datasets <br />    Growth phenotyping datasets E. coli S. oneidensis                      Conceived designed experiments: CJT JLR. Performed the <br /> taken publications Orth et al. [10] Deutschbauer et                        experiments: CJT. Analyzed data: CJT JLR. Contributed reagents/ <br /> al. [5] respectively. For E. coli, growth growth                            materials/analysis tools: CJT. Wrote paper: CJT JLR. Developed the <br />                                                                                        algorithms processing scripts, performed simulations, and <br /> determinations used reported paper four <br />                                                                                        composed figures: CJT. <br /> conditions (glucose aerobic, glucose anaerobic, succinate aerobic <br />  <br /> References <br /> 1. Barua D, Kim J, Reed JL (2010) An Automated Phenotype-Driven Approach                3. Kumar VS, Maranas CD (2009) GrowMatch: An Automated Method for <br />    (GeneForce) Refining Metabolic Regulatory Models. PLoS Comput Biol              Reconciling In Silico In Vivo Growth Predictions. PLoS Comput Biol 5: <br />    6: e1000970.                                                                            e1000308. <br /> 2. Orth JD, Palsson B&#216; (2010) Systematizing generation missing metabolic         4. Oh J, Fung E, Price MN, Dehal PS, Davis RW, et al. (2010) A universal <br />    knowledge. Biotechnology Bioengineering 107: 403&#8211;412.                               TagModule collection parallel genetic analysis microorganisms. Nucleic <br />                                                                                            Acids Research 38: e146. <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                        14                          December 2013 | Volume 8 | Issue 12 | e81322 <br />                                                                                                                      Algorithm Identifying Biomass Components <br />  <br />  <br />  5. Deutschbauer A, Price MN, Wetmore KM, Shao W, Baumohl JK, et al. (2011)              18. Warner JR, Reeder PJ, Karimpour-Fard A, Woodruff LBA, Gill RT (2010) <br />     Evidence-Based Annotation Gene Function Shewanella oneidensis MR-1                 Rapid profiling microbial genome using mixtures barcoded oligonucle- <br />     Using Genome-Wide Fitness Profiling 121 Conditions. PLoS Genet 7:                 otides. Nat Biotech 28: 856&#8211;862. <br />     e1002385.                                                                            19. Wang HH, Isaacs FJ, Carr PA, Sun ZZ, Xu G, et al. (2009) Programming cells <br />  6. Reed JL, Patel TR, Chen KH, Joyce AR, Applebee MK, et al. (2006) Systems                 multiplex genome engineering accelerated evolution. Nature 460: 894&#8211; <br />     approach refining genome annotation. Proceedings National Academy              898. <br />     Sciences 103: 17480&#8211;17484.                                                        20. Agarwal V, Lin S, Lukk T, Nair SK, Cronan JE (2012) Structure enzyme- <br />  7. Vitkin E, Shlomi T (2012) MIRAGE: functional genomics-based approach               acyl carrier protein (ACP) substrate gatekeeper complex required biotin <br />     metabolic network model reconstruction application <span id='am-1' about='obo:NCBITaxon_1117' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Cyanophyceae" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Oxygenic photosynthetic bacteria" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Oxyphotobacteria" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="Cyanobacteria" datatype="xsd:string"></span><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="PMID:11542017" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="PMID:9382701" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="cyanophytes" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="PMID:15388760" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="blue-green bacteria" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="cyanobacteria" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Cyanophyta" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="blue-green algae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Cyanophycota" datatype="xsd:string"></span>cyanobacteria</span>              synthesis. Proceedings National Academy Sciences 109: 17406&#8211;17411. <br />     networks. Genome Biology 13: R111.                                                   21. Copeland WB, Bartley BA, Chandran D, Galdzicki M, Kim KH, et al. (2012) <br />  8. Kumar VS, Dasika M, Maranas C (2007) Optimization based automated                        Computational tools metabolic engineering. Metabolic Engineering 14: 270&#8211; <br />     curation metabolic reconstructions. BMC Bioinformatics 8: 212.                        280. <br />  9. Tervo C, Reed J (2012) FOCAL: experimental design tool systematizing          22. Sigurdsson G, Fleming RMT, Heinken A, Thiele I (2012) A Systems Biology <br />     metabolic discoveries model development. Genome Biology 13: R116. <br />                                                                                              Approach Drug Targets <span id='am-56' about='obo:NCBITaxon_286' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621'><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasRelatedSynonym" content="Peudomonas" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="Liquidomonas" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="PMID:9103607" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Pseudomonas" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="PMID:10939664" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="PMID:23918787" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="PMID:10758879" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-66'  ></span><span id='am-67' property="oboInOwl:hasDbXref" content="PMID:18048745" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="PMID:15950132" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="PMID:7727274" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="RNA similarity group I" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Loefflerella" datatype="xsd:string"></span>Pseudomonas</span> aeruginosa Biofilm. PLoS ONE 7: <br /> 10. Orth JD, Conrad TM, Na J, Lerman JA, Nam H, et al. (2011) A comprehensive <br />                                                                                              e34337. <br />     genome-scale reconstruction Escherichia coli metabolism &#8212; 2011. Mol Syst <br />                                                                                          23. Lee D-S, Burd H, Liu J, Almaas E, Wiest O, et al. (2009) Comparative Genome- <br />     Biol 7. <br />                                 <br /> </body></html>