<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>High-Resolution Profiling Stationary-Phase Survival <br /> Reveals Yeast Longevity Factors Their Genetic <br /> Interactions <br /> Erika Garay, <span id='am-12' about='obo:NCBITaxon_338213' typeof='owl:Thing obo:NCBITaxon_177244 obo:NCBITaxon_177236 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_116702 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6682 obo:NCBITaxon_88770 obo:NCBITaxon_6683 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14'  ></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Sergio" datatype="xsd:string"></span>Sergio</span> E. Campos, Jorge Gonza&#180;lez la Cruz, Ana P. Gaspar, Adrian Jinich, <br /> Alexander DeLuna* <br /> Laboratorio Nacional Geno&#180;mica para la Biodiversidad, Centro Investigacio&#180;n y Estudios Avanzados del IPN, Irapuato, Guanajuato, Mexico <br />  <br />  <br />  <br />      Abstract <br />      Lifespan influenced large number conserved proteins gene-regulatory pathways. Here, introduce a <br />      strategy systematically finding longevity factors <span id='am-27' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-38'  ></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> scoring genetic interactions <br />      (epistasis) factors. Specifically, developed automated competition-based assay chronological <br />      lifespan, defined stationary-phase survival yeast populations, used phenotype 5,600 single- double- <br />      gene knockouts unprecedented quantitative resolution. We 14% viable yeast mutant strains were <br />      affected stationary-phase survival; extent true-positive chronological lifespan factors estimated by <br />      accounting effects culture aeration adaptive regrowth. We lifespan extension dietary restriction <br />      depends Swr1 histone-exchange complex functional link autophagy lipid-homeostasis <br />      factor Arv1 impact cellular lifespan. Importantly, genetic interaction network based aging <br />      phenotypes, successfully recapitulated core-autophagy machinery confirmed role human tumor <br />      suppressor PTEN homologue yeast lifespan phosphatidylinositol phosphate metabolism. Our quantitative analysis of <br />      longevity factors genetic interactions provides insights gene-network interactions aging cells. <br />  <br />   Citation: Garay E, Campos SE, Gonza&#180;lez la Cruz J, Gaspar AP, Jinich A, et al. (2014) High-Resolution Profiling Stationary-Phase Survival Reveals Yeast <br />   Longevity Factors Their Genetic Interactions. PLoS Genet 10(2): e1004168. doi:10.1371/journal.pgen.1004168 <br />   Editor: Stuart K. Kim, Stanford University Medical Center, United States America <br />   Received May 2, 2013; Accepted December 23, 2013; Published February 27, 2014 <br />   Copyright: &#223; 2014 Garay et al. <span id='am-86' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-90'  ></span>This</span> open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work funded Consejo Nacional Ciencia y Tecnolog&#305;&#180;a (CONACYT CB-164889), University <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-6'  ></span>California</span> Institute Mexico the <br />   United States (UC-MEXUS CN-11-559), pilot grant Northwestern University Physical Sciences-Oncology Center. The funders role study <br />   design, <span id='am-127' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-128' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-130' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-131'  ></span>data</span> collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: adeluna@langebio.cinvestav.mx <br />  <br />  <br />  <br />  <br /> Introduction                                                                            time stationary phase estimated monitoring the <br />                                                                                         outgrowth individual-strain cultures parallel [19,20] or <br />     Lifespan complex trait clear genetic underpinnings                        large pools gene-deletion strains coupled microarray [21,22] <br /> [1]. Mutations insulin/IGF-1 receptor gene daf-2 increase                        deep-sequencing analysis [23]. Such large-scale approaches <br /> the lifespan <span id='am-59' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-60'  ></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Caenorhabditis elegans</span> [2] <span id='am-71' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-76'  ></span><span id='am-77' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span>Drosophila melanogaster</span>                  recently opened door genome-wide screens for <br /> [3], mice lacking insulin receptor adipose tissue live                     lifespan factors [19,21&#8211;23]. With catalogs hand, next <br /> longer [4]. Studies budding yeast <span id='am-44' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-47'  ></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces</span>                          great challenge different genetic factors are <br /> cerevisiae pivotal discover conserved longevity                       integrated determine lifespan yeast. <br /> mechanisms [5&#8211;7]. Aging dividing cells modeled                               Quantitative genetic interaction (GI) analysis great potential <br /> yeast replicative lifespan, defined number daughter                    shed light genetic architecture lifespan phenotype <br /> cells produced death [8]. In addition, chronological                             mechanisms underlie aging process. By depicting <br /> lifespan (CLS), duration yeast survival stationary phase,                     ways individual mutations affect other&#8217;s <br /> provides model aging non-dividing quiescent cells [9,10].                       phenotypic consequences, GIs inform genes act in <br /> Evidence date indicates forms yeast aging                           common independent pathways [24]. A number break- <br /> modulated <span id='am-111' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-112' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-114'  ></span><span id='am-115' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Tor</span>/Sch9 [11,12] Ras/cAMP/PKA [13]                                 studies based large-scale GI analysis cell- <br /> pathways. Likewise, downstream activity proteins                            proliferation phenotypes (mainly colony growth) provided <br /> roles autophagy [14], redox balance [15,16], mitochon-                           important insights genetic organization living cell <br /> drial function [17,18] influence lifespan yeast mammals.                        [25&#8211;31]. Multi-conditional multi-phenotype GI maps have <br /> Moreover, dietary restriction delays replicative chro-                         helped elucidate cellular pathways responses to <br /> nological aging yeast; non-genetic intervention                        stimuli [32,33] and, notably, GI networks retain conserved <br /> linked down-regulation Tor Sch9 response                            features overall levels pathway crosstalk large <br /> nutrients [12,19].                                                                      evolutionary distances [34]. Yet, systematic double-mutant anal- <br />     Current phenotyping strategies allow characterization                        yses lifespan phenotypes ensuing GI networks still <br /> yeast CLS high-throughput manner. Survival function                        missing. <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                             1                         February 2014 | Volume 10 | Issue 2 | e1004168 <br />                                                                                                     Systematic Genetic Analysis Yeast Longevity <br />  <br />  <br />  <br />   Author Summary                                                            highly correlated (Pearson&#8217;s r = 0.88; Figure S1). Dye-swap <br />                                                                             experiments 235 deletion strains independently <br />   The budding yeast Saccharomyces cerevisiae emerged                    tagged RFP CFP fluorescent proteins showed a <br />   important model genetic analysis aging,                  good correlation (Pearson&#8217;s r = 0.79; Figure S1). In addition, we <br />   insights gained process yeast cells                     compared results competition-assay alternative <br />   enhance understanding aging organisms,                    high-throughput approach based outgrowing individu- <br />   including humans. Even yeast, knowledge                     ally-cultured strains different age intervals [20] (Figure S2). <br />   number identity genes determine lifespan                  Competition-based monitoring CLS resulted considerably <br />   limited lacking general picture              variability measured phenotypes experiments in <br />   different genetic aging factors work together. Here, using                strains individually aged outgrown, underscoring <br />   innovative sensitive technique, characterized                  effectiveness experimental approach terms precision. <br />   stationary-phase survival yeast single- double-                   Given strain-strain interactions obscure results of <br />   gene knockout mutants screen genes control <br />                                                                             CLS screen based competitive survival, evaluated the <br />   chronological aging score genetic interactions <br />                                                                             effects using different reference strains. The relative lifespans of <br />   genes. Our results showed important <br />   fraction (14%) genome contributes regulation                set 24 knockouts consistent experiments in <br />   lifespan, including genes pathways                    different combinations strains used (p,0.05, <br />   previously associated phenotype. We                     Spearman&#8217;s correlation), indicating survival ranks were <br />   constructed genetic interaction map recapitulated                 typically insensitive choice reference strain (Figure S3). <br />   lifespan-determining pathways highlighted genetic                        Taken together, results indicate strategy possesses <br />   associations autophagy machinery                      accuracy robustness enable large-scale characteriza- <br />   phosphatidylinositol-phosphate lipid-homeostasis                      tion CLS high resolution. <br />   pathways. Our study provides accurate <br />   catalogue yeast aging genes, picture                 A high-resolution genome-wide screen lifespan <br />   gene-network wiring aging cells.                                       factors <br />                                                                                To screen genome lifespan-determining genes and <br />                                                                             pathways, used CLS method characterize 3,878 <br />    In study, sought investigate quantitative manner         nonessential-gene knockouts yeast deletion collection <br /> the genes gene-network wiring underlies yeast CLS. By              [37] (data set Table S1). The distribution lifespan effects <br /> using novel profiling tool based competitive survival           peaked near neutrality (L = 1), long tail short-lived <br /> yeast populations stationary phase, generated high-                 knockouts smaller substantial proportion long-lived <br /> resolution CLS catalogue 3,878 single-knockout mutations             strains (Figure 2A). Given high-throughput screen was <br /> constructed systematic lifespan GI map derived          performed low aeration conventional <br /> 1,800 double-knockout phenotypes. In doing so, revealed novel            aeration conditions (see Materials Methods), characterized <br /> genetic aging factors shed light functional associations             subset mutants high aeration confirmed was <br /> within lifespan-determining pathways.                           general agreement long- short-lived behaviors <br />                                                                             regardless aeration (Figure S4). <br /> Results                                                                        In genetic studies, mutations increase maximal lifespan <br />                                                                             likely informative aging processes than <br /> A sensitive high-throughput strategy profile yeast                       mutations reduce life span [38], deleterious <br /> chronological lifespan                                                      phenotypes arise variety reasons necessarily <br />     To gain deeper knowledge genes shape lifespan              related aging. Indeed, observed short-lived knockouts <br /> phenotype and&#8212;more importantly&#8212;on genes work                      likely affected rate exponential growth <br /> together, developed sensitive parallel technique quanti-           strains wild-type CLS (Figure S5). Still, note at <br /> fying CLS yeast single- double-gene deletions (Figure 1).         64% short-lived knockouts identified genome- <br /> We used fluorescent-protein labeling track apparent death            wide screen specific CLS, is, deletion affected <br /> rates mutant wild-type (WT) reference populations                 stationary-phase survival rate growth (Figure S5). <br /> stationary phase. Specifically, mixed population suspended          Furthermore, overlap mutants affected stationary- <br /> stationary phase automatically sampled regular age                   phase survival exponential-growth rate did increase <br /> intervals inoculating fresh medium, competition                 substantially subset mutant strains grown non- <br /> assays used estimate abundance viable mutants                fermentatively ethanol-glycerol medium (Figure S6), which <br /> relative WT time sampling. An apparent relative            indicates respiratory integrity overlap stationary- <br /> lifespan, L, mutant strain defined change           phase survival largely taken account describing the <br /> relative viability function time stationary phase.               mutant&#8217;s fermento-respiratory growth capacity standard <br /> Importantly, accounted differences growth rates            glucose medium. <br /> outgrowth (see model Materials Methods). By using fully-                We important fraction non-essential yeast <br /> prototrophic strains screens, avoided strong effects       genome mediates CLS. Overall, 6.8% single-gene knockouts <br /> amino acid auxotrophy stationary-phase survival [23,35]. In              long-lived phenotype, 7.2% lived significantly than <br /> addition, buffered aging medium identify lifespan             wild type showing major growth defects (5% FDR; see <br /> effects genes interactions independently acetic acid        Figure 2a inset, genes Table S2). To quantitatively the <br /> toxicity [36].                                                              cellular processes protein complexes underlie CLS in <br />     We evaluated performance CLS method               yeast, identified functional groups single deletions a <br /> different ways. First, estimated technical biological            skewed distribution lifespan effects (Figure 2B). Mutant strains <br /> replicability L measurements. We observed results           compromised mitochondrial function, novo purine biosyn- <br /> of independent large-scale (720 deletion strains) experiments           thesis, vacuole function, autophagy typically resulted in <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                    2                      February 2014 | Volume 10 | Issue 2 | e1004168 <br />                                                                                                           Systematic Genetic Analysis Yeast Longevity <br />  <br />  <br />  <br />  <br /> Figure 1. A novel strategy enabling quantitative genetic analysis yeast CLS. (A) Schematic showing RFP-labeled mutant, xD, lives <br /> shorter CFP-labeled WT reference. Strains mixed equal amounts grown saturation 96-deep-well plates, mixed <br /> population ages. (B) At regular time intervals, outgrowth cultures automatically inoculated aging stationary-phase plate fresh <br /> medium monitored growth. The relative initial fraction viable cells (magenta braces) approximated change relative <br /> fluorescence signal interpolated fixed time point (cyan vertical line) exponential outgrowth. (C) The change ratio viable cells function <br /> of age used obtain apparent relative survival coefficient, s, direct estimate lifespan xD relative WT. <br /> doi:10.1371/journal.pgen.1004168.g001 <br />  <br />  <br /> reduced lifespan phenotypes. This confirmed main findings                number chromatin-modification proteins resulted extended <br /> recent genome-wide screens yeast CLS factors [21&#8211;23]. We                    lifespan (Figure 2B). Such novel chronological aging factors <br /> also note genes involved regulation replicative cell             included Swr1-complex proteins Arp6, Swc3, Swr1. <br /> aging enriched CLS effects.                                             The <span id='am-106' about='obo:NCBITaxon_441239' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_62784 obo:NCBITaxon_6994 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_70905 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_323649 obo:NCBITaxon_6993 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_33341 obo:NCBITaxon_50557 obo:NCBITaxon_6960 obo:NCBITaxon_6072'><span id='am-107'  ></span><span id='am-108' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-110' property="rdfs:label" content="Saga" datatype="xsd:string"></span>SAGA</span>-complex subunits Chd1 Sgf11 also, our <br />     An inspection list short- long-lived strains (Table           knowledge, previously unrecognized pro-aging proteins, which <br /> S2) indicated able confirm number                     add proteins SAGA-like complex histone- <br /> previously-reported CLS genes. For instance, inactivation                modifying factors underlying yeast lifespan [44]. On other <br /> Leu3 transcription factor involved amino acid biosynthesis                   hand, disruption Rpd3L histone-deacetylation complex <br /> increased CLS, expected [39]. Lifespan extension                    subunits Cti6, Dep1, Rxt2, Rxt3, Ume1 resulted short <br /> observed ald6D [40]. In example, depletion                   lifespan, altering cell&#8217;s growth capacity. Likewise, <br /> apoptotic proteins Fis1 Aim14 extended yeast CLS,                        Sdc1, Spp1, Swd1 subunits Set1/COMPASS <br /> consonance studies showing chronological aging induces                histone-methylation complex novel yeast CLS factors (see <br /> apoptosis yeast [41,42]. This case                       Table S2). We identified number novel lifespan <br /> mitochondrial ubiquinone oxidoreductase: disruption Ndi1                     phenotypes associated genes roles sterol and <br /> decreases ROS production elongates CLS yeast [43].                   sphingolipid metabolism homeostasis (Figure 2B; Table <br /> Moreover, strains impaired Msn2, Glc8, Glc3 reduced                  S2). Depletion ergosterol-biosynthetic genes Erg5, Erg6, <br /> lifespan, agreement studies showing decreased              Hmg1 extended CLS, disruption fatty-acid <br /> Tor activity activates stress-response transcription factors                elongase Sur1&#8212;involved sphingolipid biosynthesis&#8212;reduced <br /> genes glycogen accumulation [19].                                           CLS affecting growth. The ARV1-encoded protein, <br />     Importantly, screen allowed uncover biological                    mediates sterol shingolipid homeostasis [45&#8211;48], was <br /> processes protein complexes previously                    novel lifespan factor identified genome-wide CLS <br /> associated CLS phenotype. Noteworthy, depletion                     screen. <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                        3                        February 2014 | Volume 10 | Issue 2 | e1004168 <br />                                                                                                          Systematic Genetic Analysis Yeast Longevity <br />  <br />  <br />  <br />  <br /> Figure 2. High-resolution genome-wide screening yeast CLS. (A) Cumulative frequencies relative lifespan viable knockout <br /> strains (black circles, n = 3,878) replicates WT (gray crosses, n = 414). Colored symbols correspond selected knockouts (right panel). Inset <br /> shows fraction significant short- (magenta) long-lived phenotypes (cyan) (5% FDR). Short-CLS phenotypes reduced growth capacity <br /> (G,0.95) filtered-out subsequent analyses (see Figure S5). (B) CLS-specific gene groups pathways; plot shows median relative <br /> lifespan (L~ ) genes annotated GO term, compared case median lifespan entire data set. Labels the <br /> vertical <span id='am-101' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-105'  ></span>axis</span> representative enriched GO categories (p,0.05, Wilcoxon rank sum test). The radius circle indicates number of <br /> measured gene knockouts GO group, minimum (de novo IMP biosynthesis) maximum 39 (nucleolus); colors were <br /> arbitrarily assigned similar functional categories. Data set includes 3,286 strains marked defect growth (G.0.95). <br /> doi:10.1371/journal.pgen.1004168.g002 <br />  <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                        4                       February 2014 | Volume 10 | Issue 2 | e1004168 <br />                                                                                                   Systematic Genetic Analysis Yeast Longevity <br />  <br />  <br />  <br /> Confirming chronological lifespan phenotypes                           A functional link Arv1 autophagy underlies <br /> estimating extents adaptive regrowth                                    yeast lifespan <br />    To validate results competition-based                   To gain deeper knowledge lifespan role Arv1, a <br /> genome-wide screen, assayed 14 longest-lived seven           protein involved lipid homeostasis, asked novel <br /> of shortest-lived knockout strains using standard CLS              CLS factor modulates yeast lifespan indepen- <br /> method non-buffered medium [49] (see Materials                    dently autophagy, conserved modulator lifespan <br /> Methods). Knockout strains Drps4b, Drps24a, Drps10b, Drps18b,              yeast multicellular eukaryotes [14,19,52] intimately <br /> Dleu3, Drcy1, Dald6 confirmed long-lived (Figure 3A&#8211;           linked lipid metabolism [53]. We generated collection of <br /> G), Dlsc1, Dglk1, Dlsc2, Dstf1, Dubp14 confirmed         double knockouts combining arv1D 26 deletions core- <br /> short-lived (Figure 3H&#8211;L). The CLS strains deleted ADE5,7,          autophagy CVT-pathway genes. Strikingly, relative lifespan <br /> AQR1, CYB5, ERG6, NEW1, RRP8, SPP1, YAK1, YPL062W                      double knockouts significantly increased when <br /> was distinguishable WT retested (data                compared expected independent combination of <br /> shown). Overall, seven 14 long-lived strains        deleterious lifespan phenotypes of corresponding <br /> seven short-lived knockouts confirmed using standard              single knockouts (Figure 5A). This result suggested anti- <br /> CLS method.                                                                aging effect autophagy depends extent action of <br />    Subpopulations aging stationary phase cultures undergo           Arv1. <br /> adaptive regrowth, phenomenon non-dividing cells                    To confirm observed genetic association Arv1 the <br /> acquire mutations allow re-enter cell cycle           autophagy machinery, monitored different reporters of <br /> take advantage nutrients released dead cells [50]. If        autophagic activity wild-type Arv1-disrupted cells. We <br /> rate adaptive regrowth given mutant strain different           arv1D cells accumulate Atg8&#8212;a marker of <br /> than wild-type, mutant register artificially high        autophagic bodies&#8212;in vacuole, partially resembling the <br /> or low abundance ratio function age. We                  molecular phenotype atg1D cells autophagy activity <br /> estimated extent adaptive regrowth gave rise false-        completely blocked (Figure 5B). In addition, examined the <br /> positive lifespan factors screen (Figure S7). To so,          degradation Pgk1-GFP, low-turnover rate cytosolic fusion <br /> focused nearly mutant strains scored short- long-lived        protein rapidly degraded non-selective autophagy upon <br /> (505 511 knockouts; Table S2) monitored               nitrogen starvation [54]. Also agreement genetic <br /> chronological aging water prevent occurrence adaptive         interactions scored autophagy genes ARV1, found <br /> regrowth [49]. We observed phenotypes              autophagy-directed appearance processed GFP protein <br /> aging conditions consistent other; distributions        clearly diminished arv1D cells (Figure 5C). Taken together, <br /> of lifespan effects water markedly different               results showed autophagic activity compromised in <br /> mutants scored short-lived long-lived buffered medium             arv1D deletion strains confirmed novel connection Arv1 <br /> (p,10279, Wilcoxon rank sum test). In words, strains            autophagy pathway, anticipated means genetic- <br /> aged fast slow buffered medium typically did water          interaction analysis. <br /> well. For instance, CLS longest-lived and <br /> one shortest-lived strains buffered medium              Deriving genetic interactions systematically from <br /> significantly different WT CLS water (see Table 1).            quantitative lifespan phenotypes <br /> Overall, able confirm 61% short-lived 48% long-                The combination high throughput sensitivity our <br /> lived phenotypes, sets minimal fractions true-               phenotyping approach provides accurate catalogue of <br /> positive lifespan factors original genome-wide screen            genes pathways underlying CLS, lends itself the <br /> terms accounting adaptive regrowth.                                 systematic construction lifespan-based quantitative genetic <br />                                                                            interaction (GI) maps. By describing ways mutations <br /> The Swr1 complex novel aging factor mediates                     affect other&#8217;s phenotypes, GI analyses yeast very <br /> lifespan extension dietary restriction                                  successful elucidating genetic pathways [27&#8211;31,55]. Since cell- <br />    Swr1 multisubunit chromatin-remodeling complex                proliferation phenotypes typically used generate maps, <br /> contains SWR1-encoded Swi2/Snf2-related ATPase                  given genes promote growth necessarily <br /> required incorporation histone variant H2A.Z           important survival, expected CLS-based GI data <br /> chromatin [51]. To investigate mechanisms             uncover novel pathways, particular relevance aging. <br /> activity Swr1 complex promotes chronological aging,              Indeed, noted 64% genetic factors identified our <br /> quantified lifespan effects Swr1 mutants glutamine            genome-wide screen specific stationary-phase survival, <br /> (nitrogen-rich) GABA (nitrogen-poor) conditions (Figure 4). By          is, deletion affected CLS rate exponential <br /> doing so, asked lifespan extension dietary                   colony growth (Figure S5). <br /> restriction depends reduced activity novel aging               For systematic study lifespan-epistatic interactions we <br /> factor. Interestingly, Swr1-complex gene knockouts lived               focused autophagy pathway, drives degradation <br /> relatively longer WT glutamine,                  recycling unnecessary dysfunctional cellular components <br /> nitrogen-limited condition, exception arp1D          [56&#8211;58]. While autophagy consistently shown to <br /> a significant effect GABA glutamine (Figure 4). As           modulate CLS, way autophagy genes interact <br /> control, analyzed effects Tor1 Rim15,                 phenotypically with genes related <br /> known key players pathway extends longevity           processes systematically analyzed. We surveyed <br /> nutrient limitation [13]. As expected, tor1D rim15D strains        interactions core-autophagy genes, with <br /> showed, respectively, increased decreased relative longevity           signaling accessory factors (39 genes possible <br /> preferentially nitrogen-rich conditions. These results suggest       combinations). The single knockout genes examined had <br /> that lifespan extension dietary restriction depends,        small significant deleterious effect CLS, the <br /> part, action Swr1 histone-exchange complex.                  measured CLS phenotypes marker-swap experiments <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                   5                     February 2014 | Volume 10 | Issue 2 | e1004168 <br />                                                                                                       Systematic Genetic Analysis Yeast Longevity <br />  <br />  <br />  <br />  <br /> Figure 3. Longevity phenotypes confirmed using standard CLS method. Survival curves non-buffered SC medium WT <br /> (black), (A&#8211;G) long-lived (cyan), (H&#8211;L) short-lived knockout strains (magenta). Only strains different WT shown; for <br /> comparison, CLS WT shown panel (gray). Each strain assayed independent experiments; data points average <br /> and error bars indicate SEM. One-hundred percent survival corresponds colony-forming units measured day inoculation (age <br /> zero). <br /> doi:10.1371/journal.pgen.1004168.g003 <br />  <br />  <br /> showed good correlation (Figure S8). Importantly, verify            knockouts combined multiplicatively, suggesting genes <br /> CLS specificity genes tested, confirmed exponential           act independently (Figure 6A, left panel). <br /> growth capacity single knockouts remained largely                   Departing neutral expectation, GIs classified as <br /> unaffected (Figure S9).                                                      negative (aggravating) positive (alleviating) [59]: Negative <br />   For gene pair, lifespan epistatic interactions defined           epistasis scored double mutant exhibited a <br /> as significant deviations null multiplicative expectation             significantly shorter CLS phenotype expect from <br /> based CLS phenotype single knockouts.                      CLS single knockouts (Figure 6A, center panel). <br /> Neutrality defined lifespan phenotypes single            Such aggravating phenotype expected deletion pairs <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                     6                       February 2014 | Volume 10 | Issue 2 | e1004168 <br />                                                                                                                      Systematic Genetic Analysis Yeast Longevity <br />  <br />  <br />  <br />  Table 1. Chronological-lifespan effects strains aged SDC medium water. <br />  <br />  <br />                                    SDC buffered medium                                                 Water <br />                                                                                        b <br />  Strain                            Relative lifespan, L                        p-value                 Relative lifespan, La                        p-valueb <br />  <br />  WT                                1                                                                   1 <br />  Long-lived strains <br />  rps4bD                            1.03360.003                                 ,1023                   1.00960.010                                  0.105 <br />  rps24aD                           1.03560.007                                 ,1023                   1.01460.009                                  0.045 <br />  rps10bD                           1.04260.004                                 ,1023                   1.02060009                                   0.006 <br />  rps18bD                           1.03560.003                                 ,1023                   1.01460.014                                  0.043 <br />  leu3D                             1.01260.004                                 0.003                   1.01560.006                                  0.032 <br />  rcy1D                             1.06260.006                                 ,1023                   1.05160.005                                  ,1023 <br />  ald6D                             1.10960.005                                 ,1023                   1.06360.005                                  ,1023 <br />  Short-lived strains <br />  lsc1D                             0.96460.003                                 ,1023                   0.97560.009                                  ,1023 <br />                                                                                     23 <br />  glk1D                             0.95760.003                                 ,10                     0.99960.015                                  0.444 <br />  lsc2D                             0.96460.002                                 ,1023                   0.98460.006                                  0.022 <br />  stf1D                             0.95960.003                                 ,1023                   0.97360.007                                  ,1023 <br />                                                                                     23 <br />  ubp14D                            0.96360.004                                 ,10                     0.97160.010                                  ,1023 <br />  <br />  a <br />   Relative lifespan obtained survival coefficient (1+s), expressed change viability day relative WT reference, 6 error linear fit. <br />  b <br />   p-value Z-score 414 65 WT replicates SDC medium water, respectively. <br />  doi:10.1371/journal.pgen.1004168.t001 <br />  <br />  <br /> of genes act lifespan pathways partially compensate                     pathway (Atg19, Atg23, Atg27) clustered with <br /> each other&#8217;s loss. Conversely, positive epistatic interaction,                  autophagy-accessory proteins apart core-ATG <br /> double mutant exhibited extended CLS relative neutral                          module (CVT module, Figure 7A). <br /> expectation (Figure 6A, right panel). In general, phenotypic                           We used gene-clustering information gene- <br /> response takes place deleted genes                        interaction significance generate lifespan-epistasis network <br /> linear lifespan-determining pathway protein complex.                                  (Figure 7B). We observed high enrichment positive epistatic <br />    Overall, scored significant GI 201 721 gene pairs                     interactions genes core-ATG module, again <br /> tested (3.2% negative 24.7% positive, 95% CI, data                            consistent fact core-autophagy proteins act single <br /> Table S3). In agreement common assumption                             functional unit (p,1024, Fisher&#8217;s exact test). Furthermore, <br /> gene pairs interact, lifespan-epistasis                         directionality assigned positive epistatic interactions when <br /> spectrum peaked close neutrality, skewed positive                           genes larger single-knockout effect its <br /> epistasis (gray histogram Figure 6B). Many positive                            interaction partner [30]. For instance, scored positive <br /> interactions data set corresponded gene pairs                       interactions targeted ATG18; single-deletion gene had <br /> core-autophagy machinery (purple histogram outline Figure 6B).                        smaller quantitative effect CLS compared typical <br /> For set, scored 55.8% significant positive interactions                      core-autophagy gene. This observation consistent fact <br /> no negatively interacting gene pair. The distribution epistatic                       Atg18 required reaching autophagic activity <br /> effects remained largely unaffected interactions defined                       autophagy proteins localize pre-autophagosome in <br /> from additive multiplicative neutral expectation                          absence Atg18, absence core- <br /> model (Figure S10).                                                                      autophagy proteins [60]. <br />    We observed distribution epistasis gene                           Our lifespan-epistasis map recapitulated group proteins <br /> pairs code proteins interact physically higher                       related phosphatidylinositol phosphate metabolism regula- <br /> median e value gene pairs protein products                         tion (Figure 7B), including Atg21, Tax4, Snx4, Hsv2, Tep1. <br /> not interact (Figure 6C). In summary, observed prevalence                         The association Tep1 PIP module provided genetic <br /> positive epistasis data set consistent fact                      evidence putative role phosphoinositide phosphatase. In <br /> that autophagy proteins operate concert non-essential                       <span id='am-54' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-55' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-56'  ></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cancer</span> cells, TEP1 homologue PTEN controls autophagy by <br /> functional unit promotes CLS.                                                       downregulating PI3KclassI/PKB pathway [61]. The syntaxin- <br />                                                                                          like t-SNARE Tlg2 protein PIP module. <br /> A lifespan-epistasis map autophagy                                                   Recent evidence suggests vesicle-tethering proteins some <br />   Based correlation quantitative epistatic-interaction                     SNARE proteins required autophagosome biogenesis and <br /> pattern derived CLS phenotypes gene,                                     normal anterograde transport Atg9 [62]. This result provided <br /> constructed high-density GI map autophagy (Figure 7A).                             genetic evidence role Tlg2 SNARE protein <br /> Using purely phenotypic information, unsupervised map                               essential autophagy suggested functional <br /> recapitulated known pathways complexes shed light                             association needed normal lifespan. <br /> novel gene associations. With exceptions (Atg5, Atg12,                              Three signaling factors (Pho80, Snf1, Reg1) grouped <br /> Vps30), core autophagy proteins grouped (core-                         lifespan-epistasis map (Figure 7A,B). For PHO80, <br /> ATG module, Figure 7A). Proteins major role CVT                            positive epistatic interactions directed signaling <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                                 7                          February 2014 | Volume 10 | Issue 2 | e1004168 <br />                                                                                                       Systematic Genetic Analysis Yeast Longevity <br />  <br />  <br />  <br />  <br /> Figure 4. Relative lifespan extension inactivation Swr1 partially abolished dietary restriction. Bars relative <br /> lifespan effects (L21) mutant strains aged competition WT glutamine (rich nitrogen, blue bars) GABA (poor nitrogen, orange <br /> bars) aging medium. Error bars indicate standard error independent experiments. Box plot (inset) shows absolute half life <br /> measurements individual WT cultures aged glutamine GABA (n = 28). <br /> doi:10.1371/journal.pgen.1004168.g004 <br />  <br />  <br /> gene autophagy genes, agreement regulatory role            Swr1-complex knockouts different nitrogen conditions. <br /> Pho80-Pho85 upstream autophagy [63] suggesting              In doing so, provided evidence suggesting lifespan <br /> cyclin regulates downstream pathways important            extension dietary restriction depends activity the <br /> yeast CLS. As opposed PHO80, SNF1 interacted negatively              Swr1 complex. <br /> the core-ATG PIP modules, reflect Snf1                       We showed Arv1&#8212;an endoplasmic reticulum protein <br /> independently promotes autophagy inositol biosynthesis                   conserved yeast mammals&#8212;and autophagy promote <br /> [64,65].                                                                     chronological longevity concert. Such genetic association of <br />   Taken together, results quantitative lifespan-             ARV1 confirmed monitoring different autophagy reporters <br /> epistasis maps represent powerful systematic strategy establish         Arv1-depleted cells. This result adds growing body of <br /> functional associations aging pathways.                   evidence suggesting autophagy intimately connected lipid <br />                                                                              metabolism: While autophagy regulates lipid metabolism [53], <br /> Discussion                                                                   sphingolipids ceramides play important roles autophagy <br />                                                                              itself, level regulation membrane constituents <br />    We used novel functional genomics platform generate             autophagic vesicles [67,68]. In absence Arv1, yeast cells <br /> a high-resolution compendium genes affect yeast station-             up-regulate unfolded protein response [69] display a <br /> ary-phase survival construct systematic GI map              variety lipid-related phenotypes include aberrant sterol and <br /> based lifespan phenotypes.                                                sphingolipid metabolism loss plasma-membrane asymmetry <br />    Our genome-wide screen indicated substantial fraction              [45&#8211;48]. In humans, Arv1 required normal cholesterol and <br /> of non-essential yeast genome (14%) regulates stationary-                bile acid homeostasis [70]. In regard finding autophagy <br /> phase survival independently general growth defects. Among                activity depends Arv1, speculate protein be <br /> novel CLS factors detected screen different                      specifically required proper targeting Trs85 pre- <br /> proteins Swr1 complex, required ATP-                 autophagosomal structure, recently shown Arv1 <br /> dependent recruitment histone variant Htz1&#8212;the yeast                      involved targeting tail-anchored proteins the <br /> homologue mammalian H2A.Z&#8212;and promotes expression                         endoplasmic reticulum [48]. Alternatively, general lipid- <br /> near silent heterochromatin [51]. It known chromatin                 homeostasis role Arv1 essential correct activity <br /> structure dynamics change aging process [66].                 autophagy machinery. In case, results indicate that <br /> To explore specific role Swr1 complex                  functional link Arv1 autophagy impact on <br /> a regulator CLS yeast, characterized number                    cellular lifespan. <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                     8                       February 2014 | Volume 10 | Issue 2 | e1004168 <br />                                                                                                       Systematic Genetic Analysis Yeast Longevity <br />  <br />  <br />  <br />  <br /> Figure 5. Functional link autophagy Arv1 underlies CLS. (A) Genetic interaction analysis ARV1. The relative lifespan 26 <br /> arv1D, xD double knockouts plotted expected multiplicative combination corresponding pairs single knockouts. Dotted lines <br /> at horizontal vertical axes effect arv1D single knockout. Green circles indicate significant deviations neutral <br /> expectation (.95% CI), shown green edges connecting ARV1 interacting gene (right). (B) Micrographs WT gene-knockout <br /> strains expressing GFP-Atg8 fusion protein basal conditions (SD, left) 7 h nitrogen starvation (SD-N, right). Vacuoles were <br /> stained FM4-64; DIC images position yeast cells. (C) Western blot analysis nitrogen-starved strains expressing fusion protein <br /> Pgk1-GFP: WT cells depleted Arv1, core-autophagy Atg1 protein, accessory Atg21 protein. Time zero corresponds mid-log <br /> cultures transfer SD-N medium cells grew 2, 5, 7 h lysis. <br /> doi:10.1371/journal.pgen.1004168.g005 <br />  <br />  <br />    Our systematic lifespan-epistasis network successfully recapitu-          CVT pathway. A specific role ATG5 ATG12 has <br /> lated functional associations autophagic machinery                reported mitophagy lifespan regulation human <br /> related biological processes. While core-autophagy genes                umbilical vein endothelial cells: Mitochondrial damage induces <br /> clustered expected, exceptions included ATG5             upregulation genes LC3B/ATG8, in <br /> ATG12, grouped genes selective autophagy                  turn increases replicative cellular lifespan [71]. Furthermore, <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                     9                       February 2014 | Volume 10 | Issue 2 | e1004168 <br />                                                                                                             Systematic Genetic Analysis Yeast Longevity <br />  <br />  <br />  <br />  <br /> Figure 6. Lifespan-epistatic interactions recapitulate functional physical associations. (A) Plots measured relative lifespan <br /> of representative gene pairs (single double knockouts) neutral (left panel), negative (center panel), positive epistasis (right panel). <br /> Dashed lines indicate multiplicative expectation case, is, LxD ? LyD. Error bars indicate 95% CI. (B) Spectrum epistasis measured <br /> for gene pairs (gray histograms, n = 721) core-autophagy gene pairs (purple histogram outline, n = 75); (C) Comparative cumulative <br /> distributions e gene pairs coding proteins interact (orange line, n = 31) interact physically (black line, n = 690); p-value is <br /> from Wilcoxon rank-sum test. <br /> doi:10.1371/journal.pgen.1004168.g006 <br />  <br /> our epistasis network established Tep1 Tlg2 work                      aging paradigm generate aging networks, shown for <br /> concert phosphatidylinositol-phosphate metabolism                        autophagy. Our pure phenotypic characterizations proved pow- <br /> pathway modulate lifespan. While shown                        erful establishing functional relationships genes <br /> Tep1 replicative lifespan factor yeast [72], data                 remained hidden large-scale genetic analysis. Despite <br /> indicated Tep1 activity promotes CLS, line PTEN-                     fact epistasis data set limited scale priori <br /> overexpressing mice increased longevity [73]. The CLS                        enriched genes share functional relationships, we <br /> phenotype yeast Tep1 association phosphati-                   anticipate strategy described open door <br /> dylinositol proteins provide simple model elucidate               spectrum lifespan-epistatic interactions a <br /> molecular mechanisms underlying role mammalian                         comprehensive unbiased manner. <br /> homologue PTEN embryonic survival tumor suppression                           With growing number lifespan-related genes, systems view <br /> [74].                                                                             aging needed grant deeper mechanistic <br />     Compared previous genome-wide screens 6% [19]                     understanding cellular longevity [7,76]. Our quantitative anal- <br /> or 31% [21] longevity phenotypes confirmed                           ysis longevity factors genetic interactions yeast an <br /> retested, approach performed modestly better correctly                     important step end. <br /> predicting 50% (seven 14) knockout strains extended <br /> lifespan. There number possible reasons               Materials Methods <br /> mutants retested standard CLS assay successfully <br /> confirmed. Our relative lifespan measurements based                      Strains media <br /> simple deterministic model assumes constant exponential                         Fluorescent SGA starter strains YEG01-CFP YEG01-RFP <br /> death outgrowth rates population ages. For instance,                   (MATa PDC1-XFP-CaURA3MX4 can1D::STE2pr-SpHIS5 lyp1D <br /> cell heterogeneity stationary-phase cultures [75]                      his3D1 ura3D0 LEU2) generated direct PCR-based <br /> possibly taken account. Moreover, results control                     tagging S288C-derivative SGA starter strain Y8205 [77] <br /> screens suggest differences aeration effects                    pBS10 Cerulean-hphMX4 (CFP) pBS35 mCherry- <br /> adaptive regrowth explain false positives                   hphMX4 (RFP) cassettes (The Yeast Resource Center) PDC1 <br /> obtained. Importantly, set mutants confirmed,                 locus subsequent direct replacement hphMX4 cassette <br /> our approach readily used leverage chronological                    CaURA3MX4; strong constitutive expression fluorescent <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                         10                        February 2014 | Volume 10 | Issue 2 | e1004168 <br />                                                                                                           Systematic Genetic Analysis Yeast Longevity <br />  <br />  <br />  <br />  <br /> Figure 7. Quantitative lifespan-epistasis analysis reveals functional associations pathways underlying CLS. (A) <br /> Hierarchical-cluster analysis lifespan-epistasis data. Each column/row represents interaction pattern specific gene; color scale indicates <br /> the epistasis interaction strength, e. Dendrogram clusters highlighted colored bars (right). (B) Network representation gene clusters their <br /> interactions. Node color depicts gene&#8217;s known biological function. Edges significant negative (red) positive (green) epistasis (95% CI). <br /> Arrows indicate cases significant CLS-differences single knockouts, directionality largest smallest single <br /> knockout effect. <br /> doi:10.1371/journal.pgen.1004168.g007 <br />  <br />  <br /> proteins exponential growth achieved carboxyl-                     ura3D0 leu2D0 met15D0). The GFP-ATG8 fusion generated <br /> terminal fusion Pdc1 protein. Deletion strains              PCR-based tagging natMX4-CUP1pr-yeGFP cassette <br /> yeast deletion collection (Open Biosystems) S288C-                        pYM-N4; PGK1-GFP fusion obtained GFP- <br /> derivative BY4741 background (MATa xxxD::kanMX4 his3D1                           tagged yeast strain collection (Invitrogen). Genes were <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                        11                       February 2014 | Volume 10 | Issue 2 | e1004168 <br />                                                                                                        Systematic Genetic Analysis Yeast Longevity <br />  <br />  <br /> knocked-out GFP fusion strains direct gene replacement                Infinite M1000), 96-channel pipetting head, orbital plate <br /> with kanMX4 cassette.                                                     shaker, robotic manipulator arm, contained isolated <br />    The following growth media used: (1) Aging medium:                    environmental room. Mixed stationary-phase cultures were <br /> synthetic complete (SC) medium 2% glucose, 0.2% amino                    maintained constant temperature (30uC) relative humidity <br /> acid supplement mix defined (Cold Spring Harbor Laboratory                 (70%) shaking; cultures fully resuspended 24 h <br /> Manual 2005), buffered pH 6.0 17.9 mM citric acid                 automated pipetting. <br /> and 64.2 mM dibasic sodium phosphate. When noted, 25 mM                          Five days inoculation (age zero, previously defined [50]) <br /> glutamine 25 mM gamma-aminobutyric acid (GABA) instead                     2&#8211;3 days 23 days point, stationary- <br /> of ammonium sulfate 0.07% amino acid supplemented mix                  phase cultures transferred liquid-handling deck for <br /> were used. (2) Low-fluorescence medium (YNB-lf) [78]. (3)                     outgrowth: 5 ml aliquots inoculated 150 ml fresh low- <br /> Nitrogen-starvation medium (SD-N): 0.17% yeast nitrogen base                  fluorescence medium 96-well plates. Such outgrowth cultures <br /> without amino acids ammonium sulfate 2% glucose.                 kept 30uC shaking monitored following <br /> (4) Yeast-extract peptone 2% dextrose (YPD) 2% ethanol,               procedure: Every 3 h (15 h, data points), outgrowth cultures <br /> 2% glycerol (YPEG) medium.                                                    vigorously shaken transferred plate reader to <br />                                                                               collect data fluorescence channels (CFPraw: Ex 433 nm/ <br /> Generation yeast libraries                                                 5 nm Em 475 nm/5 nm; RFPraw: Ex 587 nm/5 nm Em <br />    Collections prototrophic yeast strains generated                610 nm/5 nm) optical density 600 m (OD600). Background <br /> synthetic genetic array (SGA) methodology [77]. Colony arrays                 fluorescence signal function OD600 routinely collected <br /> were transferred manually 384-head pin tool (V&amp;P                       RFP-only (CFPbg) CFP-only (RFPbg) control cultures. <br /> Scientific, VP384F); antibiotic concentrations used selection             Fluorescent signal outgrowth culture defined as: <br /> were 200 mg/ml G418 (Invitrogen) 100 mg/ml clonNAT                        RFP = RFPraw&#8211;RFPbg CFP = CFPraw&#8211;CFPbg. <br /> (Werner BioAgents). <br />    To generate collection RFP-tagged strains used              Data analysis <br /> genome-wide CLS screen, YEG01-RFP starter strain                         The log ratio RFP CFP signal (lnR/C) obtained <br /> mated array 4,844 viable single-deletion strains, followed           single value outgrowth culture age T was <br /> by diploid selection, sporulation, rounds haploid                calculated interpolating lnR/C fixed point time, t* (see <br /> selection (HIS+ MATa mating type, URA+ fluorescence                   Model below). For experiments, t* fixed 10 h. <br /> marker, G418+ knockout selection). The resulting                      Importantly, results insensitive choice t* for <br /> prototrophic strains MATa PDC1-RFP-CaURA3MX4                             outgrowth data collected late exponential growth (8&#8211;12 h <br /> can1D::STE2pr-SpHIS5 lyp1D ura3D0 his3D1 LEU2 MET15                           post inoculation, data shown). An apparent survival coefficient, <br /> xxxD::kanMX4.                                                                 s, standard error, ss, obtained slope the <br />    For double-knockout collection autophagy-related genes,             linear fit lnR/Ct* T (robust regression, Matlab). From an <br /> 41 query strains generated PCR-based gene replacement                 empirically determined dynamic range (Figure S11), defined <br /> of YEG01-RFP background using natMX4 module. This                     conservative low lnR/Ct* thresholds 23.5 2.5, <br /> query collection mated array 44 deletion strains.                respectively. Only fits data points were <br /> Double-knockout haploid strains selected described above,             considered obtain value s. Hence, mutant strains with <br /> plus clonNAT+ second knockout. The query collection                   low fluorescence signal strains grew slow co-culture <br /> included neutral-insertion hoD controls, his3D                 (less 5% tagged deletion collection) excluded from <br /> controls present array collection. All combinations          analyses. <br /> knockout strain neutral insertion (up replicates) define           All screens included large number WT samples scattered <br /> the &#8216;&#8216;single knockouts&#8217;&#8217;, combination neutral                strain arrays (typically .300 wells); used to <br /> markers define &#8216;&#8216;WT strain&#8217;&#8217;. Likewise, CFP-labeled                   account survival differences WT CFP- <br /> reference strain obtained YEG01-CFP starter strain               labeled reference strain potential batch effects. By <span id='am-126' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>, <br /> by procedure combining neutral markers. All                   survival coefficient RFP-labeled WT strain, swt , equals <br /> query array strains PCR-verified correct marker                  zero. Hence, data normalized subtracting median value <br /> insertion disruption WT allele (data shown).                   WT control samples, ~swt , raw s values batch (see <br />    To obtain ARV1 double knockouts, arv1D::natMX4 hoD                     example Figure S12). The mutant&#8217;s apparent relative lifespan <br /> neutral-insertion query strains (YEG01-RFP background)                   expressed decrease mutant population relative to <br /> mated triplicate array 29 autophagy-gene deletion                 WT, is, Lx ~1zsx . <br /> strains, including arv1D::kanMX4 his3D::kanMX4 <br /> neutral-insertion. Double knockouts selected described                Model outgrowth stationary phase populations <br /> above.                                                                        co-culture <br />                                                                                  In experimental setup, outgrowth stationary phase is <br /> Automated CLS assay: Experimental setup, robotic                              used estimate relative numbers viable cells an <br /> integration, data acquisition                                             experiment. In practice, quantity obtained several <br />    Saturated cultures RFP-labeled mutant (single double                 hours inoculation fresh medium. In follows, show <br /> knockout) CFP-labeled reference strains grown SC medium                 measuring ratios exponentially-growing popula- <br /> were mixed 1:1 ratio final volume 150 ml 96-well              tions fixed time-point hours, t*, different ages the <br /> plates (Corning 3585) pinned manual replicator (V&amp;P                culture stationary phase, T, decouples potential differences in <br /> Scientific, VP 407) 700 ml fresh aging medium deep-well            growth rates differences quantity interest, is, the <br /> plates (Nunc 260251) disposable plastic covers. All subsequent           rates death. In words, decrease relative abundance <br /> steps carried automated robotic station (Tecan                 slow-growing strain remain constant not <br /> Freedom EVO200) integrates plate carrousel (Liconic                    affect estimation change relative viability (slope), <br /> STX110), high-performance multilabel plate reader (Tecan                    provided relative abundance measured each <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                     12                       February 2014 | Volume 10 | Issue 2 | e1004168 <br />                                                                                                              Systematic Genetic Analysis Yeast Longevity <br />  <br />  <br /> outgrowth iteration time inoculation fresh              Equation 2 relationship death rate the <br /> medium.                                                                                                      ln(2) <br />                                                                                half life population, l~       , obtaining estimate of <br />    Let Nx Nwt sizes populations, mutant (x)                                            r <br /> a wild-type reference (wt), growing exponentially co-culture             mutant&#8217;s half life: <br /> rates gx gwt: <br />                                                                                                                           ln(2) <br />                                                                                                                  lx ~ <br />                          Nx (t)~Nx,t0 e{gx t                                                                            ln(2) <br />                                                                                                                               {s <br />                                                                                                                          lwt <br />                                                                                lwt measured half life RFP-labeled WT strain. <br />                         Nwt (t)~Nwt,t0 e{gwt t <br />  <br /> Therefore, relative number viable cells outgrowth <br />                                                                                Genome-wide screen determination lifespan <br /> curve inoculated age T (measured days) stationary phase               factors <br /> and measured fixed time point inoculation (t*, measured                A total 4,340 mutant strains yeast deletion <br /> in hours), given by:                                                        collection successfully tagged; 4,050 reached OD600 and <br />                                                                                RFP signal arbitrary threshold. This collection was <br />                                    !                                           rearranged 47 96-well plates included 414 WT-RFP <br />              Nx         <br />                             Nx,t0                                              strains controls 48 additional wells fresh medium or <br />          ln       &#240;T,t &#222;~ln          &#240;T &#222;z&#240;gx {gwt &#222;t              &#240;1&#222; <br />              Nwt            Nwt,t0                                             individual cultures WT-RFP CFP-reference account <br />                                                                                background signal. The RFP-labeled deletion collection was <br /> What measure interest&#8212;the relative number                  screened WT CFP-reference strain half-life of <br /> viable cells age T&#8212;plus constant term depends                   21.7 days. After data acquisition analysis, relative survival <br /> potential difference growth rates ignored.                 coefficient (corrected s) successfully calculated 3,878 <br /> This convenient decoupling possible                        strains (data Table S1). <br /> measurement taken, instance, fixed optical density                   Significantly short- long-lived strains scored by <br /> of culture. We note that, experimentally, population sizes            assigning Z-score (Z) knockout obtained L <br /> large (.106) data collected exponential growth              value standard deviation 414 L measurements the <br /> (see above).                                                                   WT (normally distributed, p = 0.48, x2 test). Two-tailed p-values <br />    To model aging populations, consider simplest model               obtained Z estimate false discovery rate (FDR) for <br /> which mutant (x) reference (wt) populations co-culture die           multiple hypothesis testing; significant phenotypes assigned <br /> constant exponential rates [79], rx rwt, resulting population           using q,0.05 cutoff. <br /> sizes Nx Nwt decay exponentially function age, T: <br />                                                                                Functional enrichment analysis <br />                                                                                   The median relative lifespan (L  ~ ) genes Gene <br />                           Nx ~Nx,T0 e{rx T <br />                                                                                Ontology (GO) term compared median relative lifespan <br />                                                                                                        ~ %1). GO terms L <br />                                                                                entire data set (L                       ~ value was <br />                                                                                significantly different genome shown in <br />                          Nwt ~Nwt,T0 e{rwt T                                   Figure 2B (p,0.05, Wilcoxon rank sum test). GO annotations <br />                                                                                downloaded Yeast Genome Database (April 2013); <br /> To measure relative number viable cells population, <br />                                                                                GO terms 60 genes <br /> we logarithm population ratios function age            genes successful L measurement relative growth at <br /> and obtain equation line, slope equal                    G.0.9 considered analysis. <br /> difference death rates populations: <br />  <br />                                     !                                          Double knockout screen, definition epistasis, and <br />                  Nx          Nx,T0                                             hierarchical clustering <br />              ln      &#240;T &#222;~ln          z&#240;rwt {rx &#222;T                 &#240;2&#222; <br />                 Nwt          Nwt,T0                                               The final collection 1,845 double-gene knockouts (41 query <br />                                                                                strains X 45 array strains) included single-knockout <br />                                                                                replicates gene marker-swap replicates per <br />                                                                                double knockout (see correlation Figure S8). Such indepen- <br />                             [s~rwt {rx <br />                                                                                dently-measured replicate values L6s combined obtain <br />                                                                                                                                        Xn Li :si {2 <br />                                                       Nx                       error-weighted average relative lifespan, L       ~                                , <br /> where s apparent survival coefficient ln         &#240;T &#222;                                                                       i~1 s {2 <br />                                                       Nwt                                                 s&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;                           i <br />  <br /> quantity obtained experimentally estimating number                                                   1 <br />                                                                                standard error sL ~ Pn                {2 <br />                                                                                                                                  . <br /> of viable cells different ages culture (Equation 1). Since                                            i~1 si <br /> the death rate mutant, rx, positive, death rate              For double deletion genes X Y, epistasis was <br /> reference strain, rwt, sets upper bound value s s#rwt.        defined multiplicative neutral expectation as: e~Lxy { <br /> We note that, experimentally, small number (16                   (Lx :Ly ), Lx, Ly, Lxy relative lifespan Dx <br /> 3,878) CLS measurements exceeded theoretical upper                     Dy single knockouts Dxy double knockout, respec- <br /> bound, presumably higher error estimate                                                                        q&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259; <br /> extreme positive s values (l = NaN; Table S1).                                 tively. Epistasis error estimated se ~ s2Lx zs2Ly zs2Lxy . <br />    Where needed, estimated absolute apparent lifespan                    Negative positive interactions defined 95% <br /> value mutant relative s value combining                    confidence interval. <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                      13                           February 2014 | Volume 10 | Issue 2 | e1004168 <br />                                                                                                     Systematic Genetic Analysis Yeast Longevity <br />  <br />  <br />    The use additive neutral expectation results <span id='am-22' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Bias" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-26'  ></span>bias</span>          et al. (2008, J. Gerontol. 63:113) adapted automated cell-assay <br /> positive epistasis (Figure S10), suggested epistasis        station. For outgrowth culture given age, viability was <br /> derived cell growth based phenotypes [59]. We use            defined OD600 nm reached fixed point time (10 hrs) <br /> the conservative multiplicative model epistasis analyses.        normalized age zero (five days inoculation); half <br />    Genes sharing similar epistatic-interaction patterns clus-          life strain defined linear regression of <br /> tered hierarchically (complete linkage) based correlation         ln(OD600 nm) function age (days). For competition-based <br /> raw e values (Spearman&#8217;s rank correlation used account           assays, outgrowth monitored relative fluorescence as <br /> strong outliers sides distribution e). Significant        described absolute half life obtained s (see <br /> interactions hierarchical clusters used generate             Materials Methods). Starting days inoculation, <br /> epistasis network, visualized Cytoscape 2.8 [80].            stationary-phase cultures monitored 27 days (individual <br />                                                                             cultures) 21 days (competition cultures). The dashed line <br /> Standard chronological lifespan assay                                       indicates median half life WT (21.7 days). <br />    For validation experiments using standard CLS method                 (PDF) <br /> [49], overnight cultures non-buffered SC (starting            Figure S3     Relative survival ranks insensitive choice of <br /> isolated colonies) diluted 0.1 OD units 25 mL fresh            reference strain. (A) Heat map illustrates relative survival <br /> non-buffered SC medium incubated 30uC shaking                coefficients, s, 24 RFP-labeled deletion strains aged in <br /> 220 rpm 125 mL flasks covered aluminum foil. Viability              competition 16 CFP-labeled strains span s scale. In <br /> was measured plating aging cells YPD-agar plates                heat map, query reference strains ordered short- <br /> monitoring colony-forming units starting day 4                   lived (bottom left) long-lived (top right); extreme negative and <br /> inoculation, considered initial survival (100%).        positive s values concentrated left bottom <br />                                                                             right. (B) Cluster analysis Spearman&#8217;s correlation coeffi- <br /> Fluorescence microscopy                                                     cients (color bar) s values obtained CFP-labeled <br />    Yeast cells expressing CUP1pr-GFP-ATG8 construct                reference strain (treatment) RFP-labeled strains <br /> grown mid-log phase (OD600&gt;1) SC medium. Expression                (observations). <br /> the fusion protein induced 10 mM CuSO4 2 h,                    (PDF) <br /> followed FM4-64 labeling final concentration 10 mg/ml <br />                                                                             Figure S4 Culture aeration minor effect CLS estimates. <br /> for 20 min. Labeled cells incubated 90 min YPD medium <br />                                                                             (A) CLS phenotypes low aeration highly correlated. <br /> to allow dye uptake starved 7 hours SD-N <br />                                                                             Thirty-two mutant strains aged competition WT <br /> medium induce autophagy. Images taken using a <br />                                                                             outgrowth monitored relative fluorescence. Low <br /> fluorescence microscope (Leica DM600 B) digital <span id='am-91' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-92' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-95'  ></span>camera</span> <br />                                                                             aeration (horizontal axis) corresponds 700 ml cultures deep- <br /> (Leica DFC420 C). <br />                                                                             plates, described (see Materials Methods), high <br />                                                                             aeration (vertical axis) corresponds 8 ml cultures aged test <br /> Western blotting                                                            tubes shaken 250 rpm outgrown microtiter plates; r the <br />    The autophagic degradation Pgk1 monitored                      Pearson&#8217;s linear correlation coefficient. (B) Semi-automated CLS <br /> immunoblotting Pgk1-GFP protein fusion described                  profiling high aeration confirms CLS phenotypes low aera- <br /> [54]. Cells grown overnight 30uC stationary phase                tion. Plot shows cumulative distributions CLS phenotypes at <br /> (OD600.4) SC medium. 20 OD600 units cells harvested              high aeration mutant strains defined neutral, short lived, or <br /> and starved nitrogen SD-N medium. At time point                 long lived automated low aeration conditions used in <br /> after starvation, 2 OD600 units cells lysed 1.85M              study. Cultures aged deep-well plates shaken at <br /> NaOH, 7.5% b-mercaptoethanol). Protein extracts separated              900 rpm Titramax vibratory shaker) Median L short- <br /> on SDS-polyacrylamide gels, transferred PVDF membrane,                 long-lived strains significantly different neutral <br /> and probed anti-GFP (Roche Applied Science 11814460001)                mutant strains (p,0.05, Wilcoxon rank sum test). <br /> and secondary antibody (Goat Anti-Mouse IgG coupled HRP,                 (PDF) <br /> 31430 Thermo Scientific); reactive band detected by <br /> chemiluminescence (GE Healthcare Life Sciences, RPN2209).                   Figure S5 Comparison growth-rate phenotypes allows <br />                                                                             identification CLS-specific genes. (A) Short-lived mutant strains <br /> Supporting Information                                                      likely affected growth phenotype. Plot <br />                                                                             compares distribution growth-rate phenotypes neutral <br /> Figure S1 Competition-based assays provide replicable CLS                   (black dots) short-lived mutant strains; p p-value the <br /> data. (A) Scatter plot comparing relative lifespan data (L)        Wilcoxon rank sum test. (B) Venn diagrams numbers of <br /> independent measurements 720 deletion strains; r                  mutant strains CLS growth-defect phenotypes; overlap <br /> Pearson&#8217;s linear correlation coefficient. (B) Scatter plot comparing        indicates mutants affected CLS (short long lived) and <br /> measurements obtained biological-replicates. WT                growth rate. CLS phenotypes study (Table S1); G is <br /> 235 deletion strains (different shown panel a)           relative growth rate, expressed average value data <br /> independently tagged RFP CFP mutant&#8217;s relative              obtained Breslow et al. (2008, Nat. Methods 5(8):711) and <br /> lifespan measured dye-swap experiments.                              Costanzo et al. (2010, Science 327(5964):425). A mild/strong growth <br /> (PDF)                                                                       defect (top) defined G,0.95, strong growth defect <br />                                                                             (bottom) G,0.9. <br /> Figure S2     Competition provides increased quantitative resolu- <br />                                                                             (PDF) <br /> tion CLS measurements. The absolute half-life WT six <br /> gene-deletion strains measured replicate (n = 24)                 Figure S6 Overlap CLS growth phenotypes increases only <br /> monitoring outgrowth individual (gray box plots)           small extent non-fermentative growth conditions. Scatter <br /> of competing populations (this study, yellow box plots). Outgrowth          plots comparing relative lifespan (L) aging medium growth rate <br /> of individual cultures monitored method Murakami,             (G) (A) fermentative (dextrose) (B) non-fermentative (ethanol+ <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                   14                      February 2014 | Volume 10 | Issue 2 | e1004168 <br />                                                                                                                 Systematic Genetic Analysis Yeast Longevity <br />  <br />  <br /> glycerol) yeast-extract peptone medium random subset                        pairs, non-core (top) core-autophagy gene sets <br /> knockout strains (n = 257). Red circles indicate strains                        (bottom). <br /> growth defects (G,0.9). Labels indicate gene mutations                          (PDF) <br /> overlapping CLS growth phenotypes non- <br />                                                                                      Figure S11 Relative-fluorescence dynamic range. Scatter plots <br /> fermentative conditions (YPEG). (C) Pie charts fraction <br />                                                                                      observed (lnR/Ct*) fluorescent signal versus expected <br /> of growth-defect phenotypes fermentative (YPD, left)                        (lnR/C) signal (defined dilutions RFP- CFP-labeled WT <br /> non-fermentative (YPEG) conditions tested mutant strains                     cells). Panels (A) (B) experiments different <br /> with short- long-lived CLS phenotype. Strains growth                       dilution rates populations. Values represent mean of <br /> in 150 ml YPD YPEG medium 96-well plates shaken                          independent experimental replicates standard <br /> 900 rpm. Absolute growth rates (gr) obtained                           deviation (error bars). Red dotted lines depict conservative upper <br /> linear fit ln(OD600 nm) function time; growth relative                 lower lnR/Ct* thresholds employed select per-day lnR/ <br /> the WT G = grx/grwt.                                                              Ct* measured values genome wide CLS assay. Expected <br /> (PDF)                                                                                fluorescent signal equal defined dilutions RFP-marked WT <br /> Figure S7 Our experimental approach readily adapted                        strain CFP-marker WT strain signal detected from <br /> measure lifespan independently potential regrowth. Cumulative                     dilutions fixed time point post inoculation, t* correspond <br /> frequencies relative lifespan measured water mutant                        observed fluorescent signal. <br /> strains classified short-lived (cross signs, n = 262) long-lived               (PDF) <br /> (plus signs, n = 243). Plot shows strains confirmed                     Figure S12 Data normalization. Representative scatter plot of <br /> water short-lived (magenta circles) long-lived strains (cyan                  raw survival coefficients, s data obtained CLS measurements <br /> circles), based distribution 57 WT-control replicates                      single double-knockout mutants arranged 44 batches <br /> (95% CI). Strains diluted 800 ml buffered SC medium                     (two batches 96-well plate), (A) (B) plate- <br /> in semi-deep plates transferred water days                    mean normalization (corrected s). Normalization by <br /> inoculation. Every days, cultures washed twice water                   subtracting mean value WT-control samples batch <br /> by microplate centrifugation. Cultures monitored relative                   raw s values batch. Blue dots represent WT <br /> survival 21 days relative lifespan calculated                      replicates, light-blue dots single deletions, gray dots are <br /> described (see Materials Methods).                                               double-deletion strains. <br /> (PDF)                                                                                (PDF) <br /> Figure S8 Characterization single double knockouts                        Table S1 Genome-wide CLS screen, complete data set. Relative <br />                                                                      x) <br /> epistasis analysis. (A). Plot shows average relative lifespan (L                 growth rate, G, average data Breslow et al. (2008, Nat. <br /> of single autophagy mutants mutant&#8217;s L           x value               Methods 5(8):711) Costanzo et al. (2010, Science 327(5964):425). <br /> (black circles) derived independent single                 (XLS) <br /> mutant replicates; error bars standard error. Single- <br />                                                                                      Table S2 Lists long-lived short-lived single knockouts. A <br /> knockout strains horizontal axis ranked largest to <br />             x . (B) Scatter plot compares L data marker-                    filter G.0.95 applied short-lived strains. No growth <br /> smallest L <br />                                                                                      filter used long-lived strains. Lwater = 1 indicates the <br /> swap replicates available double-gene knockout; r the <br />                                                                                      longevity phenotype confirmed water. <br /> Pearson&#8217;s linear correlation coefficient. <br />                                                                                      (XLSX) <br /> (PDF) <br />                                                                                      Table S3 Lifespan-epistasis data autophagy genes. Interac- <br /> Figure S9 Most single mutants used epistasis analysis                        tions indicate positive negative epistasis 95% CI. <br /> unaffected exponential-growth rates. (A) Strains                       (XLS) <br /> grown 150 ml YPD medium 96-well plates shaken at <br /> 900 rpm. Absolute growth rates (gr) obtained linear <br />                                                                                      Acknowledgments <br /> fit ln(OD600 nm) function time exponential phase. <br /> (B) Histogram shows growth rates single knockouts                        We thank Cei Abreu-Goodger, Tobias Bollenbach, Javier D&#305;&#180;az-Mej&#305;&#180;a, <br /> relative WT, calculated G = grx/grWT.                                      <span id='am-96' about='obo:NCBITaxon_1326982' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_6073 obo:NCBITaxon_86626 obo:NCBITaxon_33208 obo:NCBITaxon_1 obo:NCBITaxon_6103 obo:NCBITaxon_131567 obo:NCBITaxon_6101 obo:NCBITaxon_6102 obo:NCBITaxon_2759 obo:NCBITaxon_163699 obo:NCBITaxon_6072'><span id='am-97' property="rdfs:label" content="Alicia" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-99'  ></span><span id='am-100' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Alicia</span> Kowaltowski, Eugenio Mancera, Assen Roguev, Noam Shoresh <br /> (PDF)                                                                                critical reading manuscript members DeLuna Lab <br />                                                                                      useful discussions. We grateful <span id='am-17' about='obo:NCBITaxon_146270' typeof='owl:Thing obo:NCBITaxon_6514 obo:NCBITaxon_33208 obo:NCBITaxon_216331 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_146272 obo:NCBITaxon_33154 obo:NCBITaxon_680346 obo:NCBITaxon_1206795 obo:NCBITaxon_70849 obo:NCBITaxon_216305 obo:NCBITaxon_216307 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_6072'><span id='am-18'  ></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Doris" datatype="xsd:string"></span>Doris</span> Damian Mariana <br /> Figure S10      Comparing spectrum epistasis derived                     Go&#180;mez-Schiavon help data analysis statistics, <span id='am-132' about='obo:NCBITaxon_179363' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_1489907 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489904 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_8157 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-133' property="rdfs:label" content="Selene" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-135'  ></span><span id='am-136' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Selene</span> Herrera <br /> different neutral expectations. Epistasis defined (A)                     survival curves, Don Isaac, Emiliano Izquierdo, Shai Kaplan <br /> multiplicative neutral expectation, emultiplicative ~Lxy {(Lx :Ly ),              robotic automation. <br /> (B) additive neutral expectation, eadditive ~Lxy {Lx {Ly z1. In <br /> each case, spectrum epistasis shown interactions                       Author Contributions <br /> among non-core-autophagy (blue histograms) core-autophagy                        Conceived designed experiments: EG AD. Performed the <br /> genes (purple histogram outline, n = 75). Pie charts                        experiments: EG SEC JGdlC APG. Analyzed data: EG AJ AD. Wrote <br /> fraction significant negative (e,0) positive (e.0) interaction                paper: EG AD. <br />  <br /> References <br />  1. Kenyon CJ (2010) The genetics ageing. Nature 464: 504&#8211;512.                     5. Bishop NA, Guarente L (2007) Genetic links diet lifespan: shared <br />  2. Kenyon C, Chang J, Gensch E, Rudner A, Tabtiang R (1993) A C. elegans                mechanisms yeast humans. Nat Rev Genet 8: 835&#8211;844. <br />     mutant lives twice long wild type. Nature 366: 461&#8211;464.                6. Kaeberlein M (2010) Lessons longevity budding yeast. Nature 464: <br />  3. Tatar M, Kopelman A, Epstein D, Tu M-P, Yin C-M, et al. (2001) A mutant              513&#8211;519. <br />     drosophila insulin receptor homolog extends life-span impairs            7. Longo VD, Shadel GS, Kaeberlein M, Kennedy B (2012) Replicative and <br />     neuroendocrine function. Science 292: 107&#8211;110.                                       Chronological Aging Saccharomyces cerevisiae. Cell Metabolism 16: 18&#8211;31. <br />  4. Blu&#168;her M, Kahn BB, Kahn CR (2003) Extended longevity mice lacking         8. Mortimer RK, Johnston JR (1959) Life span individual yeast cells. Nature <br />     insulin receptor adipose tissue. Science 299: 572&#8211;574.                            183: 1751&#8211;1752. <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                            15                         February 2014 | Volume 10 | Issue 2 | e1004168 <br />                                                                                                                               Systematic Genetic Analysis Yeast Longevity <br />  <br />  <br />  9. Longo VD, Gralla EB, Valentine JS (1996) Superoxide dismutase activity                   39. Alvers AL, Fishwick LK, Wood MS, Hu D, Chung HS, et al. (2009) Autophagy <br />     essential stationary phase survival Saccharomyces cerevisiae. Journal                 amino acid homeostasis required chronological longevity in <br />     Biological Chemistry 271: 12275&#8211;12280.                                                          Saccharomyces cerevisiae. Aging Cell 8: 353&#8211;369. <br /> 10. MacLean M, Harris N, <span id='am-7' about='obo:NCBITaxon_13215' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_16736 obo:NCBITaxon_131567 obo:NCBITaxon_16739 obo:NCBITaxon_3398 obo:NCBITaxon_232347 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Piper" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11'  ></span>Piper</span> PW (2001) Chronological lifespan stationary                   40. Onodera J, Ohsumi Y (2004) Ald6p preferred target autophagy yeast, <br />     phase yeast cells; model investigating factors influence               Saccharomyces cerevisiae. J Biol Chem 279: 16071&#8211;16076. <br />     ageing postmitotic tissues higher organisms. Yeast 18: 499&#8211;509.                       41. Herker E, Jungwirth H, Lehmann KA, Maldener C, Frohlich KU, et al. (2004) <br /> 11. Fabrizio P, Pozza F, Pletcher SD, Gendron CM, Longo VD (2001) Regulation                        Chronological aging leads apoptosis yeast. J Cell Biol 164: 501&#8211;507. <br />     Longevity Stress Resistance Sch9 Yeast. Science 292: 288&#8211;290.                  42. Fabrizio P, Longo VD (2008) Chronological aging-induced apoptosis yeast. <br /> 12. Kaeberlein M, Powers RW, Steffen KK, Westman EA, Hu D, et al. (2005)                            Biochim Biophys Acta 1783: 1280&#8211;1285. <br />     Regulation Yeast Replicative Life Span TOR Sch9 Response                    43. Li W, Sun L, Liang Q, Wang J, Mo W, et al. (2006) Yeast AMID homologue <br />     Nutrients. Science 310: 1193&#8211;1196.                                                              Ndi1p displays respiration-restricted apoptotic activity involved in <br /> 13. Wei M, Fabrizio P, Hu J, Ge H, Cheng C, et al. (2008) Life Span Extension                    chronological aging. Mol Biol Cell 17: 1802&#8211;1811. <br />     Calorie Restriction Depends Rim15 Transcription Factors Downstream                   44. Jazwinski SM (2005) The retrograde response links metabolism stress <br />     Ras/PKA, Tor, Sch9. PLoS Genet 4: e13.                                                   responses, chromatin-dependent gene activation, genome stability yeast <br /> 14. Cuervo AM (2008) Autophagy aging: keeping old broom working.                           aging. Gene 354: 22&#8211;27. <br />     Trends Genetics 24: 604&#8211;612.                                                             45. Tinkelenberg AH, Liu Y, Alcantara F, Khan S, Guo Z, et al. (2000) Mutations in <br /> 15. Jakubowski W, Bilinski T, Bartosz G (2000) Oxidative stress aging                     yeast ARV1 alter intracellular sterol distribution complemented by <br />     stationary cultures yeast Saccharomyces cerevisiae. Free Radical Biology                 human ARV1. J Biol Chem 275: 40667&#8211;40670. <br />     Medicine 28: 659&#8211;664.                                                                   46. Swain E, Stukey J, McDonough V, Germann M, Liu Y, et al. (2002) Yeast cells <br /> 16. Barros MH, Bandy B, Tahara EB, Kowaltowski AJ (2004) Higher Respiratory                         lacking ARV1 gene harbor defects sphingolipid metabolism. Comple- <br />     Activity Decreases Mitochondrial Reactive Oxygen Release Increases Life                     mentation human ARV1. J Biol Chem 277: 36152&#8211;36160. <br />     Span Saccharomyces cerevisiae. Journal Biological Chemistry 279: 49883&#8211;               47. Beh CT, Rine J (2004) A role yeast oxysterol-binding protein homologs in <br />     49888.                                                                                          endocytosis maintenance intracellular sterol-lipid distribution. J Cell <br /> 17. Jazwinski SM (2005) Yeast longevity aging&#8211;the mitochondrial connection.                     Sci 117: 2983&#8211;2996. <br />     Mech Ageing Dev 126: 243&#8211;248.                                                               48. Georgiev AG, Johansen J, Ramanathan VD, Sere YY, Beh CT, et al. (2013) <br /> 18. Merz S, Westermann B (2009) Genome-wide deletion mutant analysis reveals                        Arv1 Regulates PM ER Membrane Structure Homeostasis But is <br />     genes required respiratory growth, mitochondrial genome maintenance                     Dispensable Intracellular Sterol Transport. Traffic 14: 912&#8211;921. <br />     mitochondrial protein synthesis Saccharomyces cerevisiae. Genome Biol 10:                49. Longo V, Shadel G, Kaeberlein M, Kennedy B (2012) Replicative and <br />     R95.                                                                                            Chronological Aging Saccharomyces cerevisiae. Cell metabolism 16: 18. <br /> 19. Powers RW, 3rd, Kaeberlein M, Caldwell SD, Kennedy BK, Fields S (2006)                      50. Fabrizio P, Longo V (2003) The chronological life span Saccharomyces <br />     Extension chronological life span yeast decreased TOR pathway                          cerevisiae. Aging Cell 2: 73&#8211;81. <br />     signaling. Genes Dev 20: 174&#8211;184.                                                           51. Krogan NJ, Keogh M-C, Datta N, Sawa C, Ryan OW, et al. (2003) A Snf2 <br /> 20. Murakami C, Burtner C, Kennedy B, Kaeberlein M (2008) A method high-                        <span id='am-51' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-52' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Family</span> ATPase Complex Required Recruitment Histone H2A Variant <br />     throughput quantitative analysis yeast chronological life span. The Journals              Htz1. Molecular Cell 12: 1565&#8211;1576. <br />     Gerontology Series A: Biological Sciences Medical Sciences 63: 113.                     52. Kamada Y, Sekito T, Ohsumi Y, Thomas G, Sabatini DM, et al. (2004) <br /> 21. Matecic M, Smith DL, <span id='am-78' about='obo:NCBITaxon_9596' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-79' property="rdfs:label" content="Pan" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasAlternativeId" content="NCBITaxon:317028" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasAlternativeId" content="NCBITaxon:445518" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasExactSynonym" content="chimpanzees" datatype="xsd:string"></span><span id='am-85'  ></span>Pan</span> X, Maqani N, Bekiranov S, et al. (2010) A                              Autophagy Yeast: A TOR-Mediated Response Nutrient Starvation. TOR <br />                                                                                                     Target Ramamycin: Springer Berlin Heidelberg. pp. 73&#8211;84. <br />     microarray-based genetic screen yeast chronological aging factors. PLoS <br />     Genet 6: e1000921.                                                                          53. Singh R, Kaushik S, Wang Y, Xiang Y, Novak I, et al. (2009) Autophagy <br />                                                                                                     regulates lipid metabolism. Nature 458: 1131&#8211;1135. <br /> 22. Fabrizio P, Hoon S, Shamalnasab M, Galbani A, Wei M, et al. (2010) Genome- <br />     wide screen Saccharomyces cerevisiae identifies vacuolar protein sorting,                54. Welter E, Thumm M, Krick R (2010) Quantification nonselective bulk <br />                                                                                                     autophagy S. cerevisiae using Pgk1-GFP. Autophagy 6: 794&#8211;797. <br />     autophagy, biosynthetic, tRNA methylation genes involved life span <br />     regulation. PLoS Genet 6: e1001024.                                                         55. Pan X, Ye P, Yuan DS, Wang X, Bader JS, et al. (2006) A DNA integrity <br />                                                                                                     network yeast Sacchammyces cerevisiae. Cell 124: 1069&#8211;1081. <br /> 23. Gresham D, Boer VM, Caudy A, Ziv N, Brandt NJ, et al. (2011) System-level <br />                                                                                                 56. Klionsky DJ, Emr SD (2000) Autophagy regulated pathway cellular <br />     analysis genes functions affecting survival nutrient starvation in <br />                                                                                                     degradation. Science 290: 1717&#8211;1721. <br />     Saccharomyces cerevisiae. Genetics 187: 299&#8211;317. <br />                                                                                                 57. <span id='am-121' about='obo:NCBITaxon_996740' typeof='obo:NCBITaxon_33511 owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_33208 obo:NCBITaxon_1312402 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_147099 obo:NCBITaxon_131567 obo:NCBITaxon_31265 obo:NCBITaxon_309553 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-122' property="rdfs:label" content="Kuma" datatype="xsd:string"></span><span id='am-123'  ></span><span id='am-124' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Kuma</span> A, Mizushima N (2010) Physiological role autophagy intracellular <br /> 24. Phillips P (2008) Epistasis&#8212;the essential role gene interactions structure <br />                                                                                                     recycling system: emphasis nutrient metabolism. Semin Cell Dev Biol <br />     evolution genetic systems. Nat Rev Genet 9: 855&#8211;867. <br />                                                                                                     21: 683&#8211;690. <br /> 25. Tong A, Lesage G, Bader G, Ding H, Xu H, et al. (2004) Global mapping the <br />                                                                                                 58. Levine B, Klionsky DJ (2004) Development self-digestion: molecular <br />     yeast genetic interaction network. Science 303: 808. <br />                                                                                                     mechanisms biological functions autophagy. Developmental cell 6: 463. <br /> 26. Pan X, Yuan DS, Xiang D, Wang X, Sookhai-Mahadeo S, et al. (2004) A robust <br />                                                                                                 59. Mani R, St Onge RP, Hartman JL, Giaever G, Roth FP (2008) Defining genetic <br />     toolkit functional profiling yeast genome. Molecular Cell 16: 487. <br />                                                                                                     interaction. Proc Natl Acad Sci USA 105: 3461&#8211;3466. <br /> 27. Segre` D, DeLuna A, Church G, Kishony R (2005) Modular epistasis yeast <br />                                                                                                 60. Suzuki K, Kubota Y, Sekito T, Ohsumi Y (2007) Hierarchy Atg proteins in <br />     metabolism. Nature Genetics 37: 77&#8211;83. <br />                                                                                                     pre-autophagosomal structure organization. Genes Cells 12: 209&#8211;218. <br /> 28. Schuldiner M, Collins S, Thompson N, Denic V, Bhamidipati A, et al. (2005) <br />                                                                                                 61. Arico S, Petiot A, Bauvy C, Dubbelhuis PF, Meijer AJ, et al. (2001) The tumor <br />     Exploration function organization yeast early secretory                       suppressor PTEN positively regulates macroautophagy inhibiting the <br />     pathway epistatic miniarray profile. Cell 123: 507&#8211;519.                              phosphatidylinositol 3-kinase/protein kinase B pathway. Journal Biological <br /> 29. Collins SR, Miller KM, Maas NL, Roguev A, Fillingham J, et al. (2007)                           Chemistry 276: 35243&#8211;35246. <br />     Functional dissection protein complexes involved yeast chromosome                     62. Nair U, Jotwani A, Geng J, Gammoh N, Richerson D, et al. (2011) SNARE <br />     biology using genetic interaction map. Nature 446: 806&#8211;810.                                   proteins required macroautophagy. Cell 146: 290&#8211;302. <br /> 30. St Onge R <br /> </body></html>