<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 27 November 2013                                              Nucleic Acids Research, 2014, Vol. 42, No. 4 2405&#8211;2414 <br />                                                                                                                 doi:10.1093/nar/gkt1200 <br />  <br />  <br /> Genetic robustness functional evolution <br /> of gene duplicates <br /> Germa&#180;n Plata1,2 Dennis Vitkup1,3,* <br /> 1 <br />  Department Systems Biology, Center Computational Biology Bioinformatics, Columbia University, <br /> New York City, NY 10032, USA, 2Integrated Program Cellular, Molecular, Structural, Genetic Studies, <br /> Columbia University, New York City, NY 10032, USA 3Department Biomedical Informatics, Columbia <br /> University, New York City, NY 10032, USA <br />  <br /> Received September 5, 2013; Revised November 3, 2013; Accepted November 4, 2013 <br />  <br />  <br />  <br /> ABSTRACT                                                                         duplicates (1,6). Gene duplications frequent evolu- <br />                                                                                  tion range size small-scale (SSD) whole- <br /> Gene duplications major source evolution-                               genome events (WGD) (7,8). While  90% cases <br /> ary innovations. Understanding functional diver-                             duplicate eventually lost evolution (6), duplicated <br /> gence duplicates role genetic                                    genes remain genome can, partially, <br /> robustness important challenge biology.                                 backup other&#8217;s functions. Importantly, functional <br /> Previously, analyses genetic robustness                                  compensation duplicates plays signi&#65533;?cant role buf- <br /> primarily focused duplicates essentiality                                 fering deleterious human mutations (9). <br /> epistasis laboratory conditions. In                                 Genetic robustness gene duplicates inherently <br /> study, use quantitative <span id='am-90' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-94'  ></span>data</span> sets                                  tied functional divergence. Duplicates acquire <br /> understand compensatory interactions                                     distinct molecular functions (MFs) naturally unable to <br /> <span id='am-13' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> duplicates                                     compensate another. In addition, MF is <br />                                                                                  conserved, incomplete compensation duplicates is <br /> likely relevant natural biological popula- <br />                                                                                  possible owing different expression patterns dosage <br /> tions. We that, owing high functional                              effects. Gene duplications major source new <br /> load, close duplicates unlikely provide sub-                              genes (10) conceptual models duplicates&#8217; <br /> stantial backup context large natural popu-                            evolution proposed (11,12). In neofunctio- <br /> lations. Interestingly, duplicates diverge                               nalization model duplicate gains new functions, i.e. <br /> each other, overall functional load reduced.                            functions associated ancestral gene, while <br /> At intermediate divergence distances quantita-                               duplicate retains ancestral functions <br /> tive decrease fitness removal dupli-                            (10,13,14). In contrast, subfunctionalization model <br /> cate smaller. At distances, yeast                                  duplicates indispensable retained in <br /> duplicates display balanced functional loads                                evolution partitioning ancestral gene functions <br /> and transcriptional control signifi-                               (15,16). Both models imply eventual loss the <br />                                                                                  ability duplicates fully substitute other. It is <br /> cantly complex. As yeast duplicates diverge <br />                                                                                  likely signi&#65533;?cant fraction duplicates &#65533;?xed <br /> beyond 70% sequence identity, ability                                   retained genomes owing selective advantages, <br /> compensate similar                                     dosage effects condition-speci&#65533;?c expression <br /> that random pairs singletons.                                              patterns, present moment duplication <br />                                                                                  (17,18). In cases &#65533;?xation selective advantage, <br />                                                                                  compensation duplicates unlikely. <br /> INTRODUCTION                                                                        Even compensation duplicates is <br /> Survival biological systems crucially depends robust-                      expected long term, ability duplicates <br /> ness harmful genetic mutations, i.e. genetic robustness,                      buffer deleterious mutations paralogs has <br /> and changes environmental conditions (1&#8211;3). Two                            demonstrated independent observa- <br /> distinct mechanisms genetic robustness                              tions. These include lower expected fraction of <br /> previously discussed. First, alternative signaling <span id='am-75' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-76'  ></span><span id='am-77' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="Meta" datatype="xsd:string"></span>meta</span>-                     essential genes close duplicates (1), paucity of <br /> bolic pathways provide important mechanism                                pairwise epistatic interactions involving duplicated genes <br /> rerouting molecular networks (4,5). Second,                            (19), excess aggravating genetic interactions <br /> major role genetic robustness attributed gene                           paralogs (20,21). The contribution duplicates <br />  <br />  <br /> *To correspondence addressed. Tel: +1 212 851 5152; Fax: +1 212 851 5149; Email: dv2121@columbia.edu <br />  <br /> &#223; The Author(s) 2013. Published Oxford University Press. <br /> <span id='am-50' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54'  ></span>This</span> Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/3.0/), permits non-commercial re-use, distribution, reproduction medium, provided original work properly cited. For commercial <br /> re-use, contact journals.permissions@oup.com <br />  2406 Nucleic Acids Research, 2014, Vol. 42, No. 4 <br />  <br />  <br /> to robustness primarily considered context      DeLuna et al. (34) Costanzo et al. (35). Gene <br /> of qualitative quantitative growth phenotypes      essentiality data obtained study Giaever <br /> nutrient rich small number laboratory condi-         et al. (36). <br /> tions (1,22,23). Although popular experiments,            To functionally characterize duplicated genes, Gene <br /> conditions unlikely approximate natural           Ontology (GO) (37) annotations collected from <br /> &#8216;milieu&#8217; living systems, constantly bombarded      SGD Enzyme Commission (EC) annotations from <br /> by diverse array environmental stresses stimuli.       Comprehensive Yeast Genome Database (CYGD) <br /> Perhaps importantly, strong com-        (38). Transcription factor binding motifs used our <br /> pensatory interaction pair duplicates, evo-     work compiled Kafri et al. (39) high- <br /> lutionary relevant decrease &#65533;?tness persist&#8212;due     con&#65533;?dence predictions Kellis et al. (28). We used <br /> to incomplete buffering&#8212;after damaging mutation         protein localization data Huh et al. (40), <span id='am-60' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62'  ></span><span id='am-63' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Codon</span> <br /> one duplicates (24). In context long-term         Adaptation Index (CAI) calculations based data <br /> evolution, difference mu-         set Lu et al. (41) annotation protein <br /> tations leading lethal phenotype mutations           complexes CYGD. <br /> associated &#65533;?tness decrease substantially larger <br /> than inverse effective population size (25,26). <br /> Given typical population sizes free-living microbial    RESULTS <br /> <span id='am-65' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> large (&gt;106&#8211;108) (27), small &#65533;?tness <br />                                                                 Hillenmeyer et al. (33) quanti&#65533;?ed growth phenotypes of <br /> decrease effectively lethal organisms. <br />                                                                 single-gene yeast deletion strains large collection of <br /> Consequently, quantitative analyses growth pheno- <br />                                                                 environmental conditions. The assembled data set <br /> types, preferably multiple environmental conditions, <br />                                                                 contains  5.5 million phenotypes heterozygous and <br /> are necessary understand extent compen- <br />                                                                 homozygous mutants  400 conditions. The sampled <br /> sation duplicates plays important role in <br />                                                                 conditions represent 27 different environmental stresses <br /> natural biological populations. Here perform an <br />                                                                 hundreds perturbations diverse chemical com- <br /> analysis context natural popula- <br />                                                                 pounds. Environmental stresses comprised different <br /> tions, genetic buffering mediated duplicates likely to <br />                                                                 growth media, media lacking speci&#65533;?c vitamins amino <br /> be rare and, surprisingly, monotonic function of <br />                                                                 acids, different pH temperature regimes. <br /> duplicates&#8217; divergence. <br />                                                                 This comprehensive collection phenotypes allowed us <br />                                                                 investigate diversi&#65533;?cation duplicates&#8217; <br />                                                                 functions contribution genetic robustness in <br /> MATERIALS AND METHODS                                           multiple conditions. <br /> Gene protein sequences <span id='am-35' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae,           We &#65533;?rst investigated average number sensi- <br /> <span id='am-85' about='obo:NCBITaxon_27291' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-86'  ></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="Saccharomyces paradoxus" datatype="xsd:string"></span>Saccharomyces paradoxus</span>, <span id='am-30' about='obo:NCBITaxon_4931' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Saccharomyces bayanus" datatype="xsd:string"></span><span id='am-34'  ></span>Saccharomyces bayanus</span>,                 tive conditions, i.e. conditions signi&#65533;?cant growth <br /> Saccharomyces      castelli,    Saccharomyces       mikatae,    decrease deletion duplicate, depends on <br /> <span id='am-68' about='obo:NCBITaxon_114524' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-69'  ></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasAlternativeId" content="NCBITaxon:89941" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Saccharomyces kudriavzevii" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces sp. IFO 1802" datatype="xsd:string"></span>Saccharomyces kudriavzevii</span> Saccharomyces kluyveri           sequence divergence (Ka) duplicated genes <br /> were obtained Saccharomyces Genome                     (Figure 1A B). We considered fraction different <br /> Database (SGD; http://downloads.yeastgenome.org/)               conditions growth phenotype quantitative <br /> and study Kellis et al. (28). Pairs gene duplicates   measure compensation capacity duplicates at <br /> were identi&#65533;?ed sequence homology proteins            various divergence distances. For close duplicates the <br /> within genome using BLASTP (29). Only duplicates           average number sensitive conditions signi&#65533;?cantly <br /> that bidirectional best hits aligned       different random pair yeast singletons <br /> &gt;80% open reading frame&#8217;s sequence length          (Figure 1B, horizontal line). Importantly, result does <br /> considered analysis (30). Following previous studies     imply random gene pairs close duplicates are <br /> (1), excluded ribosomal genes analysis owing        equivalent terms similarity MF. As we <br /> to high expression, dominant impact growth         demonstrate below, observed pattern likely a <br /> strong codon adaptation <span id='am-1' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-2' property="rdfs:label" content="Bias" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>bias</span>. Evolutionary distances            higher overall functional load close duplicates. Here <br /> between duplicated genes estimated using               article use term &#8216;functional <br /> method Yang Nielsen (31) implemented              load&#8217; gene characterize average &#65533;?tness de- <br /> PAML package (32); use methods,            crease&#8212;across considered conditions&#8212;due gene <br /> maximum likelihood, estimate Ka Ks did sig-          deletion; note that, based de&#65533;?nition above, the <br /> ni&#65533;?cantly change observed patterns (Supplementary           functional load measure total number of <br /> Figure S1A).                                                    MFs gene has, re&#64258;ects gene&#8217;s overall &#65533;?tness <br />    We used data obtained Hillenmeyer et al. (33)      contribution. <br /> measure &#65533;?tness contribution duplicates               Interestingly, number sensitive conditions initially <br /> multiple environmental conditions chemical perturb-         drops duplicates diverge, decreasing 30% the <br /> ations. Using P-value cutoff 0.01, obtained         distances corresponding Ka &amp; 0.1 (Ks &amp; 1, see <br /> number experimental conditions growth            Supplementary Figure S2A B). As duplicates <br /> defect observed single gene deletion              diverge further, average number sensitive conditions <br /> mutant. We analyzed quantitative growth measure-           increases again, reaching average random pair <br /> ments double single deletion yeast strains obtained     yeast singletons Ka &amp; 0.25. The trend shown in <br />                                                                                                                                                              Nucleic Acids Research, 2014, Vol. 42, No. 4 2407 <br />  <br />  <br />    A <br />    Fraction conditions significant     0.30                                                        B <br />                                                                                                                                                           0.12 <br />  <br />  <br />  <br />  <br />                                                                                                              Fraction conditions significant <br />                                                  0.25 <br />                                                                                                                                                           0.11 <br />                                                  0.20 <br />               growth decrease <br />  <br />  <br />  <br />  <br />                                                                                                                          growth decrease <br />                                                                                                                                                           0.10 <br />                                                  0.15 <br />                                                                                                                                                           0.09 <br />                                                  0.10 <br />                                                                                                                                                           0.08 <br />  <br />                                                  0.05                                                                                                     0.07 <br />                                                                                                                                                                   p=0.01 <br />                                                  0.00                                                                                                     0.06 <br />  <br />                                                          0.0   0.1   0.2     0.3    0.4   0.5    0.6                                                             0.0   0.1   0.2    0.3   0.4    0.5   0.6 <br />                                                                              Ka                                                                                                     Ka <br />  <br />  <br />           C 0.25                                                                                             D                                                                                         SSD <br />                                                                                                                                                          0.14 <br />                                                                                                                                                                                                        WGD <br />  <br />  <br />                                                                                                              Fraction conditions significant <br />             Fraction essential duplicates <br />  <br />  <br />  <br />  <br />                                                0.20 <br />                                                                                                                                                          0.12 <br />  <br />                                                0.15                                                                      growth decrease <br />                                                                                                                                                          0.10 <br />  <br />                                                0.10 <br />                                                                                                                                                          0.08 <br />                                                0.05 <br />                                                                                                                                                          0.06 <br />                                                0.00 <br />                                                                                                                                                          0.04 <br />                                                         0.0    0.2     0.4         0.6     0.8         1.0                                                       0.0   0.1   0. 2   0.3   0. 4   0.5   0.6 <br />                                                                            Ka <br />                                                                                                                                                                                     Ka <br /> Figure 1. Compensation patterns yeast duplicates function evolutionary divergence, Ka, number nonsynonymous <br /> substitutions site. (A) Scatterplot fraction sensitive conditions, i.e. conditions detectable growth phenotypes resulting from <br /> duplicate gene deletion, versus Ka. Each dot &#65533;?gure represents pair yeast duplicates. (B) The average fraction sensitive conditions <br /> per duplicate pair. The P-value calculated using Mann&#8211;Whitney U test. The horizontal lines A B indicate average fraction of <br /> sensitive conditions random pair yeast singletons. (C) The average fraction essential duplicates, i.e. duplicates lethal phenotype on <br /> deletion, function Ka. The horizontal line indicates fraction essential yeast singletons. Gene essentiality data obtained the <br /> Saccharomyces Genome Deletion Project (36). (D) Fraction conditions signi&#65533;?cant growth decrease deletion yeast duplicates arising <br /> from small-scale (SSD) whole-genome duplications (WGD). The duplicates classi&#65533;?ed SSD WGD based study Kellis et al. <br /> (8) The horizontal line shows average fraction sensitive conditions random pair yeast singletons. In &#65533;?gures, error bars represent the <br /> standard error mean (SEM). <br />  <br />  <br />  <br /> Figure 1B sensitive P-value cutoff used                                                     compensation heterozygous homozygous <br /> determine signi&#65533;?cance growth decrease observed                                                    mutants similar multiple-drug resistance genes, <br /> in mutant strains (Supplementary Figure S3). A similar                                                       de&#65533;?ned Hillenmeyer et al. (33), considered <br /> trend observed average growth decrease                                                      (Supplementary Figure S4C). The trends remain <br /> (functional load), measured log ratios                                                          similar environmental perturbations are <br /> Z-scores tested conditions (Supplementary                                                         analyzed homozygous experiments (Supplementary <br /> Figure S4A B). Bin-free analyses data                                                             Figure S4D). We checked observed compen- <br /> (Supplementary Figures S1B C S2B)                                                               sation patterns closest duplicates signi&#65533;?- <br /> revealed smaller &#65533;?tness cost loss duplicates                                                 cantly in&#64258;uenced additional, i.e. diverged, <br /> at intermediate distances (Ka &amp; 0.1).                                                                        paralogs (Supplementary Figure S4E). This lack signi&#65533;?- <br />    Because actively growing wild-type yeast popula-                                                     compensation diverged duplicates results an <br /> tions diploid (42), mainly focused analysis                                                    approximately linear relationship number of <br /> heterozygous mutant strains. The patterns functional                                                      sensitive conditions yeast protein <span id='am-42' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-43' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> the <br />  2408 Nucleic Acids Research, 2014, Vol. 42, No. 4 <br />  <br />  <br /> family size (Supplementary Figure S5). Finally,              pairs decreases function sequence divergence <br /> observed compensation patterns affected              (Figure 2E F) [see (21)]. <br /> removal gene pairs high CAI (Supplementary                Besides smaller overall functional load, possible <br /> Figure S6A), suggesting observed trend           duplicates intermediate distances other <br /> be explained expression-based constraints rate         properties favor genetic robustness. To explore this <br /> of duplicate sequence evolution (Ka) (43) high expres-        possibility, duplicate pair, looked gene <br /> sion levels certain duplicates.                               largest gene smallest number of <br />    It interesting compare ability duplicates     sensitive conditions (Figure 3A). Notably, dupli- <br /> buffer mutations leading detectable growth                cate conditions (Figure 3A, sensitive <br /> decrease given &#65533;?tness threshold (Figure 1B)             duplicate) follows average trend duplicates <br /> and role protecting no-growth pheno-        (Figure 1B), duplicate fewer conditions (Figure <br /> type, i.e. likelihood observe essential genes dupli-   3A, sensitive duplicate) shows steady gain the <br /> cate pairs. In Figure 1C, using data study           number conditions function Ka. Consequently, <br /> Giaever et al. (36), fraction essential du-       functional load close duplicates, measured the <br /> plicates function divergence. In agreement         number sensitive conditions, different, differ- <br /> with previous studies (1,22,23)                ence signi&#65533;?cantly smaller genes diverge <br /> fraction essential genes remains low approximately        (Figure 3B. Pearson&#8217;s r = 0.64, P = 7   10 4, also <br /> constant close duplicates, increases substantially       Supplementary Figure S2C). Close duplicates the <br /> only divergence distances corresponding Ka &gt; 0.4.          larger number sensitive conditions higher <br /> Notably, pattern qualitatively different        evolutionary constraint, evaluated normalized ratio <br /> compensation quantitative growth phenotypes                  nonsynonymous synonymous substitutions nucleo- <br /> (Figure 1B), demonstrating aforementioned impact             tide site, Ka/Ks (Wilcoxon Signed Rank test P = 7   10 3, <br /> of using quantitative phenotypes assess evolution-        Figure 2C). This result agrees previous reports of <br /> arily relevant consequences mutations. Also contrast       asymmetric evolution duplicates context co-ex- <br /> to patterns obtained studies based essential genes         pression, genetic interaction protein&#8211;protein interaction <br /> (22), observed similar compensation pro&#65533;?les gene          networks (19,44,45). The observed asymmetry func- <br /> pairs originating small-scale genome-wide dupli-        tional load close duplicates make buffering <br /> cations (Figure 1D). Because WGD duplicates         dif&#65533;?cult. For example, sensitive duplicate is <br />                                                                  expressed speci&#65533;?c environmental conditions. <br /> same age, result suggests ability duplicates <br />                                                                     To explore mechanism observed <br /> to buffer other&#8217;s function multiple conditions <br />                                                                  backup patterns, analyzed functional diversi&#65533;?cation <br /> depends strongly sequence divergence than <br />                                                                  yeast duplicates function sequence diver- <br /> on time duplication. <br />                                                                  gence (Ka). First, genes encoding metabolic enzymes <br />    It likely observed decrease number of <br />                                                                  calculated fraction gene pairs conserved EC <br /> sensitive conditions intermediate divergence distances <br />                                                                  numbers (Figure 4A); conservation EC numbers <br /> (Ka &amp; 0.1) decrease functional load           indicates corresponding proteins catalyze identical <br /> carried distances union duplicate             biochemical reactions. Second, calculated fraction <br /> genes. To explore possibility, considered quan-      shared GO terms describing protein MF dupli- <br /> titative &#65533;?tness data DeLuna et al. (34) syn-        cates (Figure 4B). Both measures showed MF of <br /> thetic genetic array (SGA) data Costanzo et al. (35).       yeast duplicates typically starts substantially diverge <br /> In studies, authors performed quantitative             Ka &gt; 0.4. The timing divergence ap- <br /> growth measurements yeast strains individual             proximately coincides signi&#65533;?cant increase the <br /> and simultaneous deletions duplicates. Using              fraction essential duplicates (Figure 1C). On other <br /> single deletion phenotypes DeLuna et al.                hand, signi&#65533;?cant changes number quantitative <br /> (Figure 2A) Costanzo et al. studies (Figure 2B),          growth phenotypes observed MF dupli- <br /> observed &#65533;?tness pro&#65533;?les similar obtained based        cates usually conserved. <br /> on data Hillenmeyer et al. (Figure 1B)                A complementary analysis transcription factor <br /> function Ka, smaller phenotypic effects inter-        binding sites suggests gene regulation plays import- <br /> mediate distances. Interestingly, overall functional         ant role establishing observed compensation <br /> load duplicate pairs, measured phenotype            patterns. It previously demonstrated duplicated <br /> double deletions, substantially decreases            yeast genes have, average, higher number <span id='am-80' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-81' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-84'  ></span>cis</span>-regu- <br /> their divergence (Figure 2C D). This result suggests         latory motifs singleton genes (46). Using compre- <br /> that close duplicates likely similar      hensive data set  150 known predicted DNA <br /> functions, higher functional load makes complete           binding motifs yeast (28,39), average <br /> compensation likely. Because overall functional         number different motifs regulating duplicate pair in- <br /> load duplicates remains approximately constant            creases signi&#65533;?cantly Ka &amp; 0.1 (Figure 4D, dashed line, <br /> Ka &gt; 0.15, higher fraction detectable growth pheno-       Mann&#8211;Whitney U test, P = 0.06). At divergence <br /> types distances likely decreased ability    distance, average number different motifs dupli- <br /> for functional compensation duplicates diverge.               cate pair twice number motifs pair <br /> Compensation duplicates quanti&#65533;?ed                 yeast singletons (Figure 4D, dashed horizontal line). <br /> presence aggravating interactions duplicate           The number regulatory motifs increases the <br />                                                                                                                        Nucleic Acids Research, 2014, Vol. 42, No. 4 2409 <br />  <br />  <br />               A                                  0.7 <br />                                                                 p=0.01 <br />                                                                                  B                                  0.20 <br />                                                                                                                                       p=0.08 <br />  <br />  <br />               Fraction single deletions <br />  <br />  <br />  <br />  <br />                                                                                  Fraction single deletions <br />                                                  0.6 <br />  <br />  <br />                growth phenotype <br />  <br />  <br />  <br />  <br />                                                                                   growth phenotype <br />                                                                                                                     0.15 <br />                                                  0.5 <br />  <br />                                                  0.4 <br />                                                                                                                     0.10 <br />                                                  0.3 <br />  <br />                                                  0.2 <br />                                                                                                                     0.05 <br />                                                  0.1 <br />  <br />                                                  0.0                                                                0.00 <br />                                                     0.0   0.1        0.2   0.3                                          0.0     0.1            0.2   0.3 <br />                                                                     Ka                                                                     Ka <br />               C                                                                  D                                   1.1 <br />                                                  1.0 <br />               Fraction double deletions <br />  <br />  <br />  <br />  <br />                                                                                  Fraction double deletions <br />                                                                                                                      1.0 <br />                growth phenotype <br />  <br />  <br />  <br />  <br />                                                                                   growth phenotype <br />                                                  0.8                                                                 0.9 <br />  <br />                                                                                                                      0.8 <br />                                                  0.6 <br />                                                                                                                      0.7 <br />                                                  0.4                                                                 0.6 <br />  <br />                                                  0.2 <br />                                                                                                                      0.5 <br />  <br />                                                                                                                      0.4 <br />                                                     0.0   0.1        0.2   0.3                                          0.0     0.1            0.2   0.3 <br />                                                                     Ka                                                                     Ka <br />               E                                                                  F <br />                                                                                  Fraction buffering duplicates <br />               Fraction buffering duplicates <br />  <br />  <br />  <br />  <br />                                                  1.0                                                                1.0 <br />  <br />                                                  0.8                                                                0.8 <br />  <br />                                                  0.6 <br />                                                                                                                     0.6 <br />  <br />                                                  0.4 <br />                                                                                                                     0.4 <br />                                                  0.2 <br />                                                                                                                     0.2 <br />                                                  0.0 <br />                                                     0.0   0.1        0.2   0.3                                        0.0      0.1         0.2       0.3 <br />                                                                     Ka                                                                    Ka <br /> Figure 2. Growth phenotypes individual simultaneous deletion duplicates function sequence divergence (Ka). The results in <br /> the &#65533;?rst column (A, C, E) based competition experiments DeLuna et al. (34), second column (B, D, F) synthetic <br /> genetic arrays (SGA) Costanzo et al. (35). (A, B) Fractions single duplicate deletions signi&#65533;?cant growth decrease. (C, D) Fractions of <br /> simultaneous (double) duplicate deletions signi&#65533;?cant growth decrease. Due different measurement sensitivities studies, different <br /> cutoffs used determine signi&#65533;?cant growth decrease: 1% DeLuna et al. (A, C) 10% Costanzo et al. (B, D); presented results <br /> are sensitive exact cutoff values (see Supplementary Figure S7). P-values obtained using Fisher&#8217;s exact test. (E, F) Fraction of <br /> paralogs signi&#65533;?cant negative epistatic interaction studies DeLuna et al. Costanzo et al., respectively. In &#65533;?gures error bars <br /> represent SEM. <br />  <br />  <br />  <br /> duplicate highest duplicate                            shared motifs [Supplementary Figure S9A, (48)], <br /> smallest number sensitive conditions (Supplementary                           overlap GO terms describing biological processes <br /> Figure S8A B). The increase complexity du-                         (Figure 4C) overlap cellular localization <br /> plicates regulation Ka &amp; 0.1 con&#65533;?rmed sig-                       observed &#64258;uorescence-tagging experiments (40) <br /> ni&#65533;?cant increase (Mann&#8211;Whitney U test, P = 1   10 3)                          decrease (Supplementary Figure S8B). Such pattern <br /> these distances number transcription factor                            suggests increase regulatory complexity <br /> mutants (47) affecting duplicate gene expression (Figure                         allows duplicates specialize different biological <br /> 4D, solid line).                                                                 processes preserving common MFs. The <br />    While total number DNA motifs regulating du-                           ability duplicates partially diverged regulatory <br /> plicates initially increases divergence, fraction                    regions compensate expression <br />  2410 Nucleic Acids Research, 2014, Vol. 42, No. 4 <br />  <br />  <br />              A                                           0.10                                                                       B <br />                                                                                               More sensitive duplicate <br />              Fraction conditions significant <br />  <br />  <br />  <br />  <br />                                                                                                                                     Relative difference number of <br />                                                                                               Less sensitive duplicate                                                     1.0 <br />                                                          0.09 <br />  <br />                                                          0.08                                                                                                              0.8 <br />  <br />  <br />  <br />  <br />                                                                                                                                             sensitive conditions <br />                          growth decrease <br />  <br />  <br />                                                          0.07 <br />                                                                                                                                                                            0.6 <br />                                                          0.06 <br />                                                                                                                                                                            0.4 <br />                                                          0.05 <br />  <br />                                                          0.04                            p=2E-4                                                                            0.2 <br />  <br />                                                          0.03 <br />                                                                                                                                                                            0.0 <br />                                                          0.02 <br />                                                                      0.0            0.1        0.2          0.3   0.4   0.5   0.6                                                0.00   0.02   0.04   0.06   0.08   0.10 <br />                                                                                                       Ka                                                                                          Ka <br />              C                                                                      p=0.007 <br />                                                          0.07 <br />  <br />                                                          0.06 <br />  <br />                                                          0.05 <br />  <br />                                                          0.04 <br />                                              Ka/Ks <br />  <br />  <br />  <br />  <br />                                                          0.03 <br />  <br />                                                          0.02 <br />  <br />                                                          0.01 <br />  <br />                                                          0.00 <br />                                                                                te                      te <br />                                                                        <span id='am-95' about='obo:NCBITaxon_179264' typeof='owl:Thing obo:NCBITaxon_1329950 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8504 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8509 obo:NCBITaxon_2759 obo:NCBITaxon_42424 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_1329912 obo:NCBITaxon_33208 obo:NCBITaxon_1329961 obo:NCBITaxon_1329911 obo:NCBITaxon_7711 obo:NCBITaxon_8511 obo:NCBITaxon_7776 obo:NCBITaxon_8515 obo:NCBITaxon_32561 obo:NCBITaxon_8457'><span id='am-96' property="rdfs:label" content="Plica" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-98'  ></span><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>plica</span>                    pli <br />                                                                                                      ca <br />                                                                 e   du                    e   du <br />                                                            itiv                       itiv <br />                                                     e    ns                    e    ns <br />                                            r     es                        sss <br />                    Mo                                                Le <br /> Figure 3. Differences number sensitive conditions duplicates. (A) The average fraction sensitive conditions duplicates <br /> with higher lower number sensitive conditions pair; Ka values represent sequence divergence duplicates. The P-value for <br /> the Mann&#8211;Whitney U test. (B) The relative difference number sensitive conditions duplicates function initial <br /> divergence; Ka values represent sequence divergence duplicates. The relative difference calculated absolute difference the <br /> number sensitive conditions duplicates normalized total number sensitive conditions pair (Spearman&#8217;s r = 0.60, <br /> P = 2   10 3; Pearson&#8217;s r = 0.64, P = 7   10 4). (C) The average Ka/Ks ratio paralogs largest (more sensitive) smallest <br /> (less sensitive) number conditions signi&#65533;?cant growth decrease. Ka/Ks ratios calculated relative orthologous sequences S. bayanus. <br /> Only duplicates Ka &lt; 0.15 considered. The P-value Wilcoxon signed rank test. <br />  <br />  <br />  <br />  <br /> changes intact gene previously described                                                                             Finally, patterns diversi&#65533;?cation functional <br /> Kafri et al. (39,49). Also, recent study DeLuna                                                                          compensation described correlate the <br /> et al. (50) showed deletion duplicate, ex-                                                                       process duplicate loss evolution. We investigated the <br /> pression changes remaining paralog need-                                                                       retention yeast duplicates using complete genomic <br /> based, i.e. happen primarily corresponding                                                                        sequences seven species: S. cerevisiae, S. paradoxus, <br /> function required. Such regulatory backup circuits                                                                           S. bayanus, S. castelli, S. mikatae, S. kudriavzevii and <br /> should, cases, enable functional compen-                                                                       S. kluyveri. We calculated number remaining dupli- <br /> sation homologs different expression                                                                               cates function sequence divergence (Figure 5, <br /> patterns wild type. Notably, based data                                                                          Supplementary Figure S10A correspond- <br /> recent study Springer et al. (51), measured                                                                          ing relationships individual yeast species). This analysis <br /> expression changes yeast genes                                                                               suggests relatively brief initial period high dupli- <br /> genomic copies deleted diploid cells, observed                                                                        cate loss (6) followed long evolutionary period <br /> a signi&#65533;?cant dosage response genes forming                                                                             (Ka &gt; 0.1) average loss rate decreases <br /> recently duplicated pairs (Ka &lt; 0.15, Figure 4E). This                                                                          &gt;10-fold (red Figure 5). Interestingly, loss rate sig- <br /> suggests genes close duplicates respon-                                                                      ni&#65533;?cantly decreases approximately divergence <br /> sive dosage effects.                                                                                                         distance duplicates similar terms <br />                                                                                                                                                                                        Nucleic Acids Research, 2014, Vol. 42, No. 4 2411 <br />  <br />  <br />                                                                                                                                       B 1.0 <br />                    A 1.0 <br />  <br />  <br />  <br />  <br />                                                                                                                                  Fraction shared GO molecular functions <br />                  Fraction pairs sharing EC numbers <br />                                                                0.8                                                                                                           0.9 <br />  <br />  <br />                                                                0.6                                                                                                           0.8 <br />  <br />                                                                0.4 <br />                                                                                                                                                                              0.7 <br />  <br />                                                                0.2 <br />                                                                                                                                                                              0.6 <br />  <br />                                                                0.0 <br />                                                                                                                                                                              0.5 <br />                                                                  0.0         0.2    0.4         0.6         0.8    1.0                                                          0.0       0.2         0.4          0.6         0.8        1.0 <br />                    C 0.8                                                                  Ka                                                                                                                Ka <br />                   Fraction shared GO biological processes <br />  <br />  <br />  <br />  <br />                                                                                                                          D                                                                 TF binding motifs <br />                                                                0.7                                                                                                                         TF mutations affecting expression <br />                                                                                                                                                                                        p=0.06 <br />  <br />                                                                0.6                                                                                                                    p=0.001 <br />  <br />                                                                                                                                                                               10 <br />                                                                0.5 <br />                                                                                                                          Number unique TF associations <br />  <br />  <br />  <br />                                                                                                                                                                                9 <br />  <br />                                                                0.4                                                                                                             8 <br />                                                                                                                                 duplicate pair <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                7 <br />                                                                0.3 <br />                                                                                                                                                                                6 <br />                                                                  0.0         0.2    0.4        0.6          0.8    1.0 <br />                                                                                                                                                                                5 <br />                                                                                           Ka <br />                                                                                                                                                                                4 <br />             E                                                                                          p=0.003 <br />                                                                0.25 <br />                                                                                                                                                                                3 <br />            Expression responsiveness <br />  <br />  <br />  <br />  <br />                                                                0.20                                                                                                            2 <br />                                                                                                                                                                                 0.0             0.1         0.2          0.3         &gt;   0.4 <br />                                                                0.15                                                                                                                                           Ka <br />  <br />  <br />                                                                0.10 <br />  <br />                                                                0.05 <br />  <br />                                                                0.00 <br />  <br />                                                            -0.05 <br />                                                                        0.0    0.1   0.2    0.3        0.4    0.5   0.6 <br />                                                                                            Ka <br /> Figure 4. Diversi&#65533;?cation duplicates function regulation. (A) Fraction metabolic duplicates sharing EC numbers; conservation of <br /> EC numbers indicates catalysis identical biochemical reactions. (B) Fraction GO MF terms shared duplicates. (C) Fraction GO <br /> Biological Process (BP) terms shared duplicates. In panels B C considered GO terms distance the <br /> corresponding GO <span id='am-55' about='obo:NCBITaxon_1' typeof='owl:Thing'><span id='am-56' property="rdfs:label" content="root" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="all" datatype="xsd:string"></span>root</span> hierarchy term. (D) Dashed line, average number different transcription factor binding motifs duplicate pair. <br /> Transcription factor (TF) binding motifs compiled studies Kafri et al. (39) Kellis et al. (28). Solid line, average number of <br /> transcription factor deletions S. cerevisiae signi&#65533;?cantly affect expression duplicate genes. The data obtained study Hu <br /> et al. (47). For comparison average number motifs TF mutants affecting expression random pairs yeast singletons <br /> (horizontal dashed solid lines); P-values calculated using Mann&#8211;Whitney U test. (E) The average dosage compensation (respon- <br /> siveness) duplicates function sequence divergence (Ka). The data average expression responsiveness obtained work of <br /> Springer et al. (51). In study, responsiveness measured diploid yeast strains Log2 ratio (perturbed versus normal) expression <br /> changes remaining gene copy following deletion equivalent gene copy sister chromosome. The P-value calculated using the <br /> Mann&#8211;Whitney U test. In &#65533;?gures error bars represent SEM. <br />  2412 Nucleic Acids Research, 2014, Vol. 42, No. 4 <br />  <br />  <br />                                        100 <br />                                                                        Ka &lt; 0.1         et al. (35), duplicates annotated redundant not <br />                                         80 <br />                                                                                         signi&#65533;?cantly different yeast duplicates <br />       Number remaining duplicates <br />  <br />                                                                        Ka &gt; 0.1 <br />        averaged 7 yeast species <br />  <br />                                        60                                               (Mann&#8211;Whitney U, P = 0.13 0.35, respectively, <br />                                                                                         Supplementary Figure S11). This demonstrates that, <br />                                        40 <br />                                                                                         yeast duplicates func- <br />                                                                                         tional overlap laboratory conditions, com- <br />                                                                                         pensation properties probably signi&#65533;?cantly less <br />                                        20 <br />                                                                                         important large natural populations ability <br />                                                                                         purifying selection ef&#65533;?ciently prune mutations <br />                                                                                         causing small &#65533;?tness decrease. <br />                                        10 <br />                                                                                            It likely different factors contribute the <br />                                         8 <br />                                                                                         relative paucity functional compensation between <br />                                         6                                               paralogs small divergence distances. A signi&#65533;?cant <br />                                          0.0   0.1   0.2   0.3   0.4     0.5      0.6   fraction duplications likely &#65533;?xed owing to <br />                                                                                         dosage effects (17), functional compensation <br />                                                            Ka <br />                                                                                         duplicates context natural popula- <br /> Figure 5. The average number duplicates retained genomes                   tions unlikely. For example, lack signi&#65533;?cant com- <br /> yeast species function duplicates divergence Ka, number                 pensation histone pairs, HTA1-HTA2 and <br /> of nonsynonymous substitutions site. The number remaining <br /> duplicates averaged genomes seven yeast species: S.                     HHT1-HHT2, likely consequence role <br /> cerevisiae, S. paradoxus, S. bayanus, S. castelli, S. mikatae, S.                       maintaining proper histone levels yeast cells. Gene <br /> kudriavzevii S. kluyveri. See Supplementary Figure S10                     dosage explain inability duplicates to <br /> the number remaining duplicates individual species                    backup other, unlikely explan- <br /> the number remaining duplicates function Ks. The rate of <br /> duplicate loss evolution &gt;10 times lower distances <br />                                                                                         ation. We showed duplicate pairs a <br /> corresponding Ka &gt; 0.1 compared distances Ka &lt; 0.1.                      high CAI (Supplementary Figure S6A) pairs forming <br /> In &#65533;?gure, error bars represent SEM.                                             known protein complexes (Supplementary Figure S6B) are <br />                                                                                         removed analysis, patterns functional com- <br />                                                                                         pensation remain similar. Notably, genes high CAI <br /> of functional load (Figure 2B) regu-                               associated higher frequencies of <br /> latory complexity signi&#65533;?cantly increases (Figure 5D). It                             interlocus gene conversion (IGC) (53,54). While IGC <br /> likely duplicates surviving initial loss stage                             slow rate duplicates sequence divergence <br /> develop independent functionalities preserved                                   (55), analyses based WGDs evidence of <br /> for long times genomes yeast species.                                         IGC [using data recently reported Casola et al. (56)] <br />                                                                                         revealed essentially identical compensation patterns <br />                                                                                         (Supplementary Figure S12). <br /> DISCUSSION                                                                                 Close duplicates likely compensate for <br />                                                                                         probably owing aforementioned dichot- <br /> In present study, analyzed genetic robustness                                omy functional loads (Figure 3A B). Many <br /> to duplicates context quantitative growth pheno-                              close duplicates classi&#65533;?ed, based activity <br /> types sensitivities gene deletions multiple envir-                            breadth expression, major minor func- <br /> onmental conditions. Such robustness important                                   tional isoforms. For example, glyceraldehyde-3-<span id='am-100' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-101' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-102' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-103'  ></span><span id='am-104' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>phos</span>- <br /> understanding buffering deleterious mutations                                 phate dehydrogenase TDH1 active various stress <br /> large natural biological populations. Our results demon-                                conditions, isoenzyme TDH2 used primarily <br /> strate that, contrary commonly held view, close gene                                 exponential growth (57). Similarly, ubiquitin <br /> duplicates unlikely provide high level backup                            conjugating enzyme UBC4 expressed exponen- <br /> the context large natural populations. Consequently,                              tial growth, duplicate UBC5 active sta- <br /> is unlikely duplicates &#65533;?xed natural popu-                              tionary phase (58). The difference functional load <br /> lations speci&#65533;?cally selection robustness.                                    close yeast duplicates consistent the <br />    Our analysis suggests duplicate redundancies                               asymmetric partition functions, interactions gene <br /> described genomics databases, frequently observed                                expression, observed close duplicates or- <br /> in laboratory experiments, considered                                    ganisms, example, <span id='am-6' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span><span id='am-10'  ></span><span id='am-11' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Arabidopsis</span> Human (45,59,60). <br /> caution, respect functions                                    This suggests duplicate-dependent compensation in <br /> natural biological populations. To investigate point                               context natural populations limited in <br /> further, analyzed set, compiled Kafri et al. (52),                              species well. <br /> of 112 yeast duplicates reported partially                                  Our analysis suggests typical lifecycle gene <br /> redundant research publications. These duplicates                                    duplicates yeast consists distinct evolutionary <br /> have described redundant based func-                                   stages (11,12). In &#65533;?rst stage (at duplicate distances cor- <br /> tional overlap compensatory interactions observed                                   responding Ka &lt; 0.05), duplicates tend high <br /> in small-scale experimental studies. Interestingly, based                               overall functional loads signi&#65533;?cant asymmetry the <br /> on number conditions quantitative growth                                    number sensitive conditions; factors make <br /> phenotypes study Hillenmeyer et al. (33),                                   complete compensation unlikely. The high functional load <br /> and quantitative growth measurements Costanzo                                    close duplicates suggests adaptive selection plays <br />                                                                                   Nucleic Acids Research, 2014, Vol. 42, No. 4 2413 <br />  <br />  <br /> an important role &#65533;?xation. In second stage                 10. Ohno,S. (1970) Evolution Gene Duplication. Springer-Verlag, <br /> (0.05 &lt; Ka &lt; 0.25), duplicates diverge further,                    Berlin, New York. <br />                                                                         11. Conant,G.C. Wolfe,K.H. (2008) Turning hobby job: <br /> overall functional load usually decreases. This                         duplicated genes &#65533;?nd new functions. Nat. Rev. Genet., 9, <br /> happen, example, relaxation environmen-                   938&#8211;950. <br /> tal conditions, facilitated original duplicate                12. Innan,H. Kondrashov,F. (2010) The evolution gene <br /> &#65533;?xation. The vast majority duplicates, likely                        duplications: classifying distinguishing models. Nat. <br />                                                                             Rev. Genet., 11, 97&#8211;108. <br /> paralogs relatively smaller functional loads                       13. Nadeau,J.H. Sankoff,D. (1997) Comparable rates gene <br /> (Figure 3C), lost stage (Figure 5). Gene pairs                  loss functional divergence genome duplications early in <br /> that survive period high duplicate loss display                 vertebrate evolution. Genetics, 147, 1259&#8211;1266. <br /> balanced functional loads complex regulation;                 14. Sidow,A. (1996) Genome duplications evolution early <br /> gene pairs usually retained long evolutionary times                 vertebrates. Curr. Opin. Genet. Dev., 6, 715&#8211;722. <br />                                                                         15. Force,A., Lynch,M., Pickett,F.B., Amores,A., Yan,Y.L. and <br /> in yeast genomes (Figure 5). Surviving duplicates                       Postlethwait,J. (1999) Preservation duplicate genes by <br /> provide partial compensation intermediate di-                   complementary, degenerative mutations. Genetics, 151, 1531&#8211;1545. <br /> vergence distances serve important source                16. Lynch,M. Force,A. (2000) The probability duplicate gene <br /> of new protein functions. In stage (Ka &gt; 0.3                   preservation subfunctionalization. Genetics, 154, 459&#8211;473. <br />                                                                         17. Kondrashov,F.A., Rogozin,I.B., Wolf,Y.I. Koonin,E.V. <br />  70% sequence identity), lifecycle duplicates                     (2002) Selection evolution gene duplications. Genome <br /> completed functional roles diverge,                    Biol., 3, RESEARCH0008. <br /> quantitative compensation properties indistin-                   18. Bergthorsson,U., Andersson,D.I. Roth,J.R. (2007) Ohno&#8217;s <br /> guishable random pairs yeast singletons.                   dilemma: evolution new genes continuous selection. <br />                                                                             Proc. Natl Acad. Sci. USA, 104, 17004&#8211;17009. <br />                                                                         19. VanderSluis,B., Bellay,J., Musso,G., Costanzo,M., Papp,B., <br />                                                                             Vizeacoumar,F.J., Baryshnikova,A., Andrews,B., Boone,C. and <br /> SUPPLEMENTARY DATA                                                          Myers,C.L. (2010) Genetic interactions reveal evolutionary <br /> Supplementary Data available NAR Online.                             trajectories duplicate genes. Mol. Syst. Biol., 6, 429. <br />                                                                         20. Ihmels,J., Collins,S.R., Schuldiner,M., Krogan,N.J. and <br />                                                                             Weissman,J.S. (2007) Backup redundancy: genetic <br />                                                                             interactions reveal cost duplicate gene loss. Mol. Syst. <br /> ACKNOWLEDGEMENTS                                                            Biol., 3, 86. <br />                                                                         21. Li,J., Yuan,Z. Zhang,Z. (2010) The cellular robustness <br /> We thank Fyodor Kondrashov (Centre Genomic                              genetic redundancy budding yeast. PLoS Genet., 6, <br /> Regulation) members Vitup laboratory                         e1001187. <br /> illuminating discussions regarding manuscript.                      22. Guan,Y., Dunham,M.J. Troyanskaya,O.G. (2007) Functional <br />                                                                             analysis gene duplications Saccharomyces cerevisiae. <br />                                                                             Genetics, 175, 933&#8211;943. <br /> FUNDING                                                                 23. Conant,G.C. Wagner,A. (2004) Duplicate genes and <br />                                                                             robustness transient gene knock-downs <span id='am-45' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-46'  ></span><span id='am-47' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Caenorhabditis</span> <br /> National Institutes Health (NIH) [GM079759]                      elegans. Proc. Biol. Sci., 271, 89&#8211;96. <br /> National    Centers        Biomedical  Computing                    24. Thatcher,J.W., Shaw,J.M. Dickinson,W.J. (1998) Marginal <br />                                                                             &#65533;?tness contributions nonessential genes yeast. Proc. Natl <br /> [U54CA121852]. Funding open access charge: NIH                          Acad. Sci. USA, 95, 253&#8211;257. <br /> [GM079759].                                                             25. Gillespie,J.H. (1998) Population Genetics, A Concise Guide. Johns <br />                                                                             Hopkins Univ. Press, Baltimore. <br /> Con&#64258;ict statement. None declared.                           26. Hartl,D. Clark,A. (1997) Principles Population Genetics, <br />                                                                             3 edn. Sinauer Associates, Sunderland. <br />                                                                         27. Lynch,M. (2006) Streamlining simpli&#65533;?cation microbial <br /> REFERENCES                                                                  genome architecture. Annu. Rev. Microbiol., 60, 327&#8211;349. <br />                                                                         28. Kellis,M., Patterson,N., Endrizzi,M., Birren,B. Lander,E.S. <br />  1. Gu,Z., Steinmetz,L.M., Gu,X., Scharfe,C., Davis,R.W.                (2003) Sequencing comparison yeast species identify <br />     Li,W.H. (2003) Role duplicate genes genetic robustness            genes regulatory elements. Nature, 423, 241&#8211;254. <br />     null mutations. Nature, 421, 63&#8211;66.                         29. Altschul,S.F., Madden,T.L., Schaffer,A.A., Zhang,J., Zhang,Z., <br />  2. Stelling,J., Sauer,U., Szallasi,Z., Doyle,F.J. III Doyle,J.         Miller,W. Lipman,D.J. (1997) Gapped BLAST PSI- <br />     (2004) Robustness cellular functions. Cell, 118, 675&#8211;685.            BLAST: new generation protein database search programs. <br />  3. Wagner,A. (2005) Robustness Evolvability Living Systems.         Nucleic Acids Res., 25, 3389&#8211;3402. <br />     Princeton University Press, Princeton.                              30. Gu,Z., Cavalcanti,A., Chen,F.C., Bouman,P. Li,W.H. (2002) <br />  4. Wagner,A. (2000) Robustness mutations genetic                Extent gene duplication genomes Drosophila, <br />     networks yeast. Nat. Genet., 24, 355&#8211;361.                            nematode, yeast. Mol. Biol. Evol., 19, 256&#8211;262. <br />  5. Papp,B., Pal,C. Hurst,L.D. (2004) Metabolic network             31. Yang,Z. Nielsen,R. (2000) Estimating synonymous and <br />     analysis causes evolution enzyme dispensability        nonsynonymous substitution rates realistic evolutionary <br />     yeast. Nature, 429, 661&#8211;664.                                            models. Mol. Biol. Evol., 17, 32&#8211;43. <br />  6. Lynch,M. Conery,J.S. (2000) The evolutio <br /> </body></html>