<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Feb. 1991, p. 872-885                                                                       Vol. 11, No. 2 <br /> 0270-7306/91/020872-14$02.00/0 <br />  <br />  <br />  <br />       Identification Structure Four Yeast Genes (SL Y) That Are <br />                Able To Suppress Functional Loss YPTI, <br />                       Member RAS <span id='am-85' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="superfamily" datatype="xsd:string"></span>Superfamily</span> <br />            CHRISTIANE DASCHER, RAINER OSSIG, DIETER GALLWITZ,* AND HANS DIETER SCHMITT <br />                   Department Molecular Genetics, Max Planck Institute Biophysical Chemistry, <br />                                 P.   0.   Box 2841, D-3400   Gottingen,   Federal Republic Germany <br />                                            Received 4 September 1990/Accepted 20 November 1990 <br />  <br />               In <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>, GTP-binding Yptl protein (Yptlp) essential endoplasmic reticulum- <br />            to-Golgi protein transport. By exploiting GALIO-YPTI fusion regulate YPTI expression, multicopy <br />            suppressors, SLY2, SLY12, SLY41, single-copy suppressor, SLYI-20, allowed YPTi-independent <br />            growth isolated. Wild-type Slylp hydrophilic, essential cell viability, differs Slyl-20p by <br />            single amino acid. SLY2 SLY12 encode proteins hydrophobic tails similar synaptobrevins, integral <br />            membrane proteins synaptic vesicles higher eucaryotes. Sly4lp hydrophobic exhibits sequence <br />            similarities chloroplast phosphate translocator. SLY12 SLY41 essential gene. The SLY2 <br />            null mutant cold heat sensitive. The SLY gene products comprise elements protein transport <br />            machinery. <br />  <br />    A large increasing number genes encoding                     Whereas yeast Ras proteins signal amplifiers, several <br /> small GTP-binding proteins identified                   ras-related proteins likely mode of <br /> lower higher eucaryotes. They collectively constitute                 action. Through analysis conditional null mutants <br /> the ras superfamily, best known members                  SEC4 (26, 57), YPTI (4, 21, 60, 63), SARI (47), ARFI <br /> the ras proto-oncogenes (for review, reference 6). ras              (65), genes encoding GTP-binding proteins have <br /> and ras-related proteins evolutionary highly conserved                shown involved protein transport the <br /> and share similar structural biochemical properties (for              endoplasmic reticulum (ER) plasma membrane. Mech- <br /> reviews, references 17 23). The conformational                    anistically, proteins thought cycle a <br /> change occurring transition GDP-               membrane-bound soluble form (43) direct the <br /> GTP-bound form characteristic feature allows              vesicular transport different cellular compartments, <br /> regulation proteins gives regulatory              i.e., ER Golgi complex, separate <br /> function (42, 50). When proteins complexed                 Golgi cisternae, Golgi apparatus plasma <br /> GTP, assume active conformation allows inter-                membrane (4, 11, 63, 78). At restrictive temperature, <br /> action specific targets associated activation                 sec4 mutants blocked late secretory pathway and <br /> specific cellular reactions. The subsequent hydrolysis                 accumulate small vesicles (49, 57). The YPTI gene product, <br /> protein-bound GTP, brought intrinsic                 structurally highly related Sec4p (21), contrast, <br /> GTPase activity accelerated interacting GTPase-                    participates transport process ER Golgi <br /> activating proteins (25, 33, 75, 77), ensures action          compartment possibly intra-Golgi vesicle move- <br /> the regulator temporally limited allows regenera-              ments. Yptlp-depleted cells conditional yptl <br /> tion activation-competent complex.                                  mutants nonpermissive temperature characterized <br />    A challenging problem characterize normal cel-               massive accumulation ER partially glycosylated <br /> lular function members ras superfamily              invertase (60, 63). The secretion defect ob- <br /> proteins. Examining functions proteins bio-               served cell-free transport systems yptl mutant cells <br /> logical systems accessible classical genetics               blocking vitro protein transport anti-Yptlp <br /> most feasible approach determining various roles. <br /> <span id='am-46' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-47'  ></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> applies unicellular yeasts, 10             antibodies (3, 4). Evidence similar function the <br /> small GTP-binding proteins, essential                   mammalian yptl protein (27), named <br /> functions, identified. For example, defects              rabl (74), comes immunofluorescence subceilular <br /> RAS2 gene budding yeast <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-22'  ></span><span id='am-23' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae                   fractionation studies showing yptlp primarily local- <br /> interfere normal cyclic AMP metabolism. In fact,                     ized Golgi membranes (Sla, 63) finding that <br /> evidence RAS2 gene influencing activity                    mouse yptlp functionally replace yeast counter- <br /> adenylyl cyclase obtained observing suppression              (28). <br /> of ras2 defect bcyl mutants (73) lacking functional                   In search proteins interact Yptlp fulfill a <br /> regulatory subunit cyclic AMP-dependent protein kinase                 related function, use genetic molecular <br /> (41). Further analysis suppressors led identification           genetic methods combination makes yeast system <br /> of genes acting downstream upstream RAS2                    exceptionally attractive. The basic strategy switch off <br /> (14, 15, 55, 69).                                                         expression essential YPTI gene control of <br />                                                                           repressible GALIO promoter (61) identify genes <br />                                                                           able suppress loss YPTI function. In way, a <br />                                                                           mutant gene, SL YJ-20 (for suppressor loss YPTJ <br />   * <br />       Corresponding author.                                               function), protein product carries single amino acid <br />                                                                    872 <br />  VOL . 1 l, 1991                                                  SLY GENES AS SUPPRESSORS OF YPTI IN YEAST CELLS                      873 <br />  <br />             TABLE 1. S. cerevisiae strains used study              selective YEPD plates. The EMS treatment did not <br />                                                              Source     reduce viability cells strong effect on <br />    Strain                       Genotyperernc <br />                                           reference                        number colonies. All colonies, however, appeared at <br />                                                                            time size; therefore, the <br /> GYW8-2A          MATa leu2 his3 GALlO-YPTI-LEU2             61             colonies replica plated conveniently 4 days of <br /> HLR3             MATa leu2 his3 GALlO-YPTI-HIS3             60             incubation 25&#176;C. <br /> GFUII-2B         MATa Ieu2 his3 GALlO-YPTI-HIS3             This work <br /> GFUII-1B         MATa ura3 his3 GAL1O-YPTI-HIS3             This work         For genetic analysis, YPTJ-independent mutants iden- <br /> LSY20-1A         MATa leu2 his3 GALlO-YPTI-LEU2             This work      tified YPTI deletion (yptl-) crossed strain <br />                    SLYl-20                                                 GYW8-2A, strain carrying GALIO-YPTI fusion (pheno- <br /> DAH430H          MATaIMATa YPTl/yptl::HIS3 leu2/            60             type Glu-) complementary amino acid requirements. All <br />                    Ieu2 his3/his3                                          diploids obtained Glu+. Since diploids were <br /> DAH430           MATa/MATa YPTIIYPTI leu2/Ieu2              60             homozygous GALIO- YPTI fusion, Glu+ pheno- <br />                    his3/his3                                               type diploids suggests suppressor mutations <br /> INTla            Same GFUII-2B plasmid              This work      dominant. About half spores derived these <br />                    YRp5L-SLY1/5 integrated the <br />                   SLYI locus                                               diploids Glu+. Although spore viability was <br />   This transformant phenotypically Glu- [presumably SLYJ(wt)-YRp5L- <br />                                                                            low cases, result suggests mutations one <br /> SLYl(wt)]. <br />                                                                            particular gene sufficient make cells YPTI inde- <br />                                                                            pendent. Since suppressor mutations dominant, <br />                                                                            allelism mutations different isolates to <br /> substitution compared wild-type Slylp,                  tested crossing spores derived different clones <br /> multicopy suppressor genes, SLY2, SLY12, SLY41,                        subsequent tetrad analysis. This test allelism was <br /> were isolated characterized. Cells overexpressing wild-                performed using isolates showed good spore <br /> type SLY2, SLY12, SLY41, SLY], cells                       viability spores derived second screen <br /> producing Slyl-20 mutant protein viable                YPTI-independent mutants. None different com- <br /> disruption YPTI gene. The sequences Sly2p,                       binations tested gave rise Glu- spores, suggesting all <br /> Slyl2p, Sly4lp suggest integral membrane                 clones mutated gene (SLYJ). <br /> proteins. Moreover, protein products SLY2                      <span id='am-140' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-141' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-144' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>Plasmids</span> nucleic acid techniques. DNA manipulations, <br /> SLYJ2 multicopy suppressor genes structural features                  E. coli transformation, restriction enzyme diges- <br /> in common synaptobrevin, synaptic vesicle mem-                      tions, ligations, performed described Sam- <br /> brane protein highly conserved higher eucaryotes                brook et al. (58). Preparation yeast DNA RNA and <br /> (67).                                                                      Northern (RNA) blotting described previously (36, 37). <br />                                                                               Plasmid YCp5O (52) used low-copy-number yeast <br />                                                                            vector. The high-copy-number library constructed by <br />                  MATERIALS AND METHODS                                     using 2,um-based vector YEp13 (13). Other high-copy- <br />    Genetic techniques. <span id='am-62' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-63' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-77'  ></span>Escherichia coli</span> RR1 JM101                 number <span id='am-95' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>vectors</span> used YEp511, derived from <br /> used cloning experiments expression lacZ                    YEp51 (12) deleting GALIO sequences this <br /> fusion genes. S. cerevisiae strains used study                 vector, YEp24 (16), carries URA3 marker. <br /> listed Table 1. Crosses, sporulation diploids,                   These plasmids used subcloning fragments <br /> dissection tetrads described Sherman et                 derived original isolates YEp13-SLY2, YEp13- <br /> al. (64). Rich medium (YEP) contained 1% yeast extract                 SLY12, YEp13-SLY41, YCp50-SLY1/5(wt) localize <br /> 2% peptone (Oxoid), supplemented 8% glucose (YEPD)                    complementing activity check the <br /> for repression GALIO promoter                  SLYJ(wt) allele acts suppressor overexpressed. <br /> YPTI gene strains HLR3, GFUII-1B, GFUII-2B,                            The GALIO-YPTI fusion gene HIS3 <br /> GYW8-2A mutants transformants derived                      marker gene deleted mutants transformation <br /> these strains. To induce expression YPTI gene                5.4-kb HindIII fragment derived plasmid <br /> these strains, YEP medium supplemented 2%                         pAYL-2 (60). This plasmid contains parts TUB2 and <br /> galactose (YEPGal) instead 8% glucose. Transformants                    ACT] genes, flank YPTI gene chromosome VI. <br /> and diploids selected minimal medium containing                    The NarI-BamHI fragment containing YPTI gene was <br /> 0.67% yeast nitrogen base amino acids (Difco)                  replaced vector NarI-BgIII fragment containing <br /> either 2% glucose (SD) 2% galactose (SGal). These media                 LEU2 gene. The transformants checked the <br /> were supplemented meet auxotrophic requirements                  absence YPTJ sequences Southern blot analysis <br /> particular strains (64). Yeast cells competent               (<span id='am-130' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-131' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-132'  ></span><span id='am-133' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> shown). <br /> transformed lithium acetate technique described                  To check suppressor activity SLY2 SLY12 <br /> Ito et al. (30).                                                           genes low-copy-number plasmid, 3.0-kb-long BamHI <br />    Since growth GALI0- YPTI strains stops 12             fragment containing SLY2 gene 3.0-kb HindIlI <br /> h glucose medium (61), transformants derived                 fragment carrying SL Y12 gene subcloned into <br /> strains replica plated YEPD plates second time              YCp5O (52). The resulting plasmids transformed into <br /> after 24 h. This allowed determination Glu+ Glu-                ura3 strain GFUII-1B. To achieve moderate overexpres- <br /> phenotypes 2 days incubation YEPD plates.                      sion SLY41 gene, 1.9-kb SpeI fragment containing <br />    For mutant isolation, strain HLR3 grown YEPGal                   2iim origin replication deleted plasmid <br /> medium washed 50 mM KPO4 buffer, pH 6.5. The                      YEp24-SLY41, carries SLY41 gene 3.0-kb <br /> cells treated 1% ethyl methanesulfonate (EMS)                ClaI fragment. The resulting plasmid cut with <br /> 1 h 25&#176;C buffer. After washing YEPD                    NcoI direct integration URA3 locus XbaI <br /> medium, cells incubated medium 25&#176;C                    direct integration SLY41 locus. The linearized <br /> mutation fixation. After 4 h incubation, cells spread              plasmids mixed transformed GFUII-1B. <br />  874    DASCHER ET AL.                                                                                           MOL. CELL. BIOL. <br />  <br /> Transformants selected SGal plates, colonies           suppress turning YPTJ expression when <br /> were replica plated YEPD plates.                              subcloned multicopy vector YEp511. <br />    Yeast DNA libraries constructed using DNA             One-step gene disruption carried according to <br /> a yeast strain deletion YPT1 gene carrying       Rothstein (53). A 2.2-kb HpaI fragment 3.0-kb BgIIl <br /> the dominant SLY1-20 allele. This DNA partially di-            fragment containing LEU2 gene, 2.2-kb EcoRV-HincII <br /> gested endonuclease Sau3A TaqI. The endonu-            fragment containing HIS3 gene, 1.2-kb HindlIl <br /> clease digestion stopped adding EDTA reac-           fragment containing URA3 gene inserted the <br /> tion mixture different time intervals. The DNA run       coding regions SL Y1, SL Y12, SL Y2, SL Y41 genes, <br /> on preparative agarose gel, region containing DNA        respectively. These fragments derived vectors <br /> fragments longer 4 kb cut gel. DNA         YEp13 (13), pBR322-HIS3 (66), YEp24 (16) inserted <br /> purified gel slice according method Tautz       unique restriction sites particular SLY genes <br /> and Renz (70) ligated vectors YEp13               present pSPT18 subclones: HpaI site SLY), the <br /> YCp5O. YEp13 YCp5O DNA cut BamHI                  BglII site SLY12, XbaI site SLY41, NcoI <br /> ClaI, respectively, dephosphorylated prevent religa-        site SLY2. Then 57 96 nucleotides removed from <br /> tion. The BamHI-Sau3A ligation ClaI-TaqI ligation          coding region SLY2 gene BAL 31 nuclease. <br /> yielded 40,000 4,000 E. coli clones. About 90%        Where necessary, ends DNA fragments the <br /> the plasmids contained insert, determined checking        vector DNA blunt using Klenow fragment <br /> for tetracycline-sensitive clones (the insert disrupt       E. coli DNA polymerase I. Fragments containing the <br /> the tet gene).                                                     disrupted SLY genes cut plasmids using <br />    Three different clones containing SLY) mutant gene,         endonucleases SpeI (SLY)), NcoI-HindIII (SLY12), Hin- <br /> YCp50-SLY1/2, -SLY1/5, -SLY1/6, derived              dIII-NheI (SLY41), SmaI-Sall (SLY2). These fragments <br /> the TaqI library. The LEU2 marker gene inserted           purified agarose gels used transform strain <br /> the unique Sall site smallest isolates, YCp5O-     DAH430 (SLY), -2, -12) DUR3 (SLY41) (see Fig. 4, <br /> SLY1/5,     2.2-kb XhoI-SalI fragment     excised   10, 11, 12). In case SLY2, nucleotides were <br /> from YEp13 (13). The resulting centromere plasmid,                 left derived pSPT18 polylinker to <br /> YCp5LU-SLY1/5, transformed leu211eu2 strain           BamHI sites boundaries yeast DNA insert. <br /> IDAH430H check ability plasmid complement        However, small mismatch end fragments <br /> the YPTI deletion. YCp5LU-SLY1/5 cut XhoI              did interfere gene disruption, effects <br /> delete URA3 gene CEN4 sequences           SL Y2 disruption fully complemented by <br /> this plasmid, creating YRp5L-SLY1/5. As ARS)               plasmids carrying insert 1.5-kb BamHI-EcoRV <br /> plasmid, vector integrate genome       fragment contains SL Y2 sequences se- <br /> replicate autonomously. Hence, fast- slow-growing        quences neighboring reading frames. Moreover, <br /> transformants obtained plasmid cut           Southern blot analysis showed BamHI sites present <br /> the unique PvuII site cloned SLY) sequence used         end yeast DNA sequence present the <br /> for yeast transformation. The GAL10- YPTI strain trans-        transformants (data shown). Transformants sporu- <br /> formed linearized plasmid, transformants        lated, tetrad analysis performed YEPD plates. <br /> selected minimal plates lacking leucine containing          The integration DNA fragments different SLY <br /> galactose sole carbon source. Both fast-        loci verified Southern blot analysis shown Fig. <br /> slow-growing clones Glu+ Glu-. We             4, 10, 11, 12. <br /> assume Glu- clones SLY)(wt) allele           Sequence comparison secondary structure predictions. <br /> chromosome used template repair              The search proteins sequence similarities the <br /> linearized plasmid DNA. DNA small Glu- colony               products SLY), SL Y2, SL Y12 genes screening <br /> wns used E. coli transformation reisolate plasmid           MIPSX protein data base (Martinsrieder Institut fur <br /> YRp5L-SLY1/5 containing wild-type allele SLY)           Proteinsequenzen; release 16 31435 sequences) did not <br /> geqe (YRp5L-SLY1/5wt). A fast-growing stable Glu- trans-           detect entry data base yielded optimized <br /> formant (INT1) mated SLY1-20 mutant (LSY20-             score 82 use FASTA algorithm <br /> 1A) prove integration SLY) locus tetrad           (word size = 2) Pearson Lipman (51). Two proteins <br /> analysis.                                                          identified gave optimized score &gt;100 <br />    For DNA sequence analysis, DNA fragments sub-              compared primary sequence putative Sly4lp. <br /> clones yptl --complementing activity inserted       Two algorithms, RDF2 (51) algorithm written by <br /> the vector pSPT18 (Pharmacia). Supercoiled plasmid DNA             Argos (2), used test significance simi- <br /> was purified agarose gels according Tautz Renz         larities. The score sequences aligned Fig. 8B 5.4 <br /> (70) sequenced chain termination method (18, 59),       (region 1 3) 6.5 (region 2) standard deviations above <br /> using SP6 T7 primers synthetic primers           mean score randomly permutated sequences. <br /> synthesized according sequences established. In               Nucleotide sequence accession numbers. The sequences of <br /> case, strands sequenced.                                 SLY), SLY2, SLY12, SLY41 genes been <br />    Southern transfers Nytran-NY13N membrane (Schle-             assigned accession numbers X54323, X54236, X54237, <br /> icher &amp; Schull) performed using standard proce-            X54238, respectively, EMBL data base. <br /> dores. Hybridization carried 65&#176;C 1 M NaCl- <br /> 1Q% dextran sulfate-1% sodium dodecyl sulfate 100 mg                                    RESULTS <br /> of calf <span id='am-99' about='obo:NCBITaxon_49990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_216706 obo:NCBITaxon_216718 obo:NCBITaxon_3398 obo:NCBITaxon_4136 obo:NCBITaxon_4143 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-100' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-102'  ></span><span id='am-103' property="rdfs:label" content="Thymus" datatype="xsd:string"></span>thymus</span> DNA ml carrier. The same <br /> conditions used yeast DNA library (vector                 Isolation YPTI-independent mutants. The YPTJ gene in <br /> YCp5O) screened clones containing SLYl(wt)             S. cerevisiae encodes protein essential function <br /> gene 0.8-kb Bgll fragment mutant sequence          (61, 62). YPTI-independent yeast mutants isolated by <br /> as probe. One positive clone obtained,             using GAL1O-YPT1 fusion. Placing regulatable GAL1O <br /> named YCp5O-SLY1(wt). The cloned sequence able             promoter YPT1 coding region makes cells <br />  VOL . 1 l, 1991                                                SLY GENES AS SUPPRESSORS OF YPTI IN YEAST CELLS                     875 <br />  <br />                                                                         chromosome VI (61), yptl- transformants His- <br />                                                                         Leu+ correct integration linear DNA frag- <br />                                                                         ment YPTI locus. In His-/Leu+ <br />                                                                         transformants derived revertant, absence of <br />                                                                         YPTI coding sequences verified Southern analysis <br />                                                                         (data shown). <br />                                                                            Five YPTI-independent mutants, crossed a <br />                                                                         wild-type haploid, good spore viability. The genetic <br />                                                                         analysis isolates showed (i) mutations one <br />                                                                         particular gene sufficient suppress YPTI defect, <br />                                                                         (ii) mutations linked YPTI gene, (iii) all <br />                                                                         mutations gene closely linked <br />                                                                         genes, (iv) mutations behaved dominant suppres- <br />                                                                         sors YPTJ deficiency. Four mutants derived a <br />                                                                         second screening YPTJ-independent growth also <br />                                                                         mutated closely linked gene. <br />                                                                         This gene named SLYJ (for suppressor loss YPTI <br />                                                                         function). In accordance standard S. cerevisiae nomen- <br />                                                                         clature, dominant mutant alleles designated SLYJ-20 <br />                                                                         example, refer wild-type gene as <br />                                                                         SL YJ(wt). <br />                                                                            Cloning SLYI-20, mutant gene acting single-copy <br />                                                                         suppressor. The dominance SL YJ suppressor muta- <br />                                                                         tions allowed easily clone gene. As outlined Fig. 1, <br />                                                                         fragmented chromosomal DNA SL YJ-20 mutant <br />                                                                         lacking YPTI gene inserted yeast single- <br />                                                                         copy vector YCp5O (52), carries URA3 marker, <br />                           tetrad analysis                               ARSI (yeast replication origin), CEN4 (yeast cen- <br />                                                                         tromere) sequences (step 3 Fig. 1). The plasmid collection <br />    FIG. 1. Scheme isolation cloning suppressor           used transform ura3 strain GAL1O-YPT1 <br /> SLYJ-20. (1) Mutagenized HLR3 cells spread selective <br /> YEPD plates. Mutant colonies appearing 4 days incubation       fusion gene chromosome VI. All Ura+ colonies selected <br /> at 25&#176;C replica plated different media identify clones     galactose medium replica stamped glucose- <br /> with indicated phenotypes. (2) YPTI     replaced LEU2    containing plates (step 4 Fig. 1). Transformants containing <br /> gene Ts- clones (Glu+ 25&#176;C 37&#176;C). (3) A DNA library    plasmid dominant SLY]-20 allele able to <br /> with fragments genetically analyzed yptl - strain, inserted   survive shutdown YPTI transcription presence <br /> the single-copy vector YCp5O, (4) used transform GAL10-        glucose. As expected, isolated Glu+ colonies <br /> YPTI strain identify Glu+ Ts- colonies. (5) The recombinant          temperature sensitive glucose- galactose- <br /> plasmid isolated (6) transformed DAH430H <br /> diploid.                                                                containing media. From Glu+ transformants, three <br />                                                                         different plasmids overlapping DNA inserts <span id='am-125' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-126' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-129'  ></span>iso</span>- <br />                                                                         lated (step 5 Fig. 1). The smallest plasmids <br />                                                                         (YCp5O-SLY1/5) contained 3.1-kb insert 1,998-bp- <br /> galactose dependent sensitive presence glucose            long open reading frame (Fig. 2). Deletions starting from <br /> in growth medium (61). With use selective            insert ranging open reading <br /> with EMS-mutagenized cells, however, mutations                   frame destroyed suppressor activity. This result sug- <br /> regulatory GAL4 GAL80 gene products allow                  gested 1,998-bp open reading frame encodes the <br /> mutants grow glucose-containing medium. With              yptl --complementing activity. <br /> mind, isolated temperature-sensitive revertants (step 1               Evidence cloned sequence representing the <br /> Fig. 1), assuming YPTI-specific extragenic suppressor            SL YI-20 mutant gene obtained different ways. <br /> might unable rescue YPTJ-deficient cells temper-           First, cloned sequence suppress the <br /> atures. In fact, high expression GAL4 gene suppresses            turning GAL J0- YPTI fusion deletion of <br /> the shutdown GAL10- YPT1 expression 25                   YPTI gene. To prove this, diploid strain DAH430H <br /> 37&#176;C (see below). Moreover, efficient selection condi-           YPTI gene replaced HIS3 marker gene (60) <br /> tional mutations regulatory GAL genes requires primary               transformed plasmid YCp5LU-SLY1/5. This cen- <br /> mutations GAL80 gene (40).                                       tromere plasmid derived original SLYJ-20- <br />   To ensure conditional phenotype revertants            containing isolate YCp5O-SLY1/5 insertion LEU2 <br /> isolated specific YPTI deficiency, ana-              gene additional marker. Leucine-prototrophic transfor- <br /> lyzed clones Ts- glucose-containing               mants selected sporulated. Eleven tetrads dis- <br /> not galactose-containing medium (Fig. 1). Most                sected gave rise 22 large His- colonies. The His- pheno- <br /> revertants isolated survive deletion              type indicated carried YPTI allele. In addition <br /> YPTI coding region, indicating YPTI               wild-type spores, 12 somewhat smaller His+/Leu+ <br /> independent. To delete YPTI gene, revertants               colonies grew spores derived plasmid- <br /> transformed linear DNA fragment containing                   containing diploids. The His+/Leu+ phenotype suggested <br /> LEU2 marker gene place YPTI coding region (step               required growth suppressor gene present on <br /> 2 Fig. 1). As original strain HLR3 carries HIS3              plasmid. The segregation pattern (12 His&apos; 10 non- <br /> marker gene near 3&apos; end GAL10- YPTI fusion gene              growing spores 11 tetrads) consistent the <br />  876        DASCHER ET AL.                                                                                                                                          MOL. CELL. BIOL. <br />  <br />  <br />        A                                                                                                  B <br />                                                                                                      bp    1561                       1596                         1611 <br />       CCATTGGACGTGCTCGCAATGGTGGAGGACGAGTTGGATGACGCTCAGGATTCTGACGAA                        -517                  ATC TAT GGC TTA ACT   MA A       GGC AAA TTA CAA GGA          SL YJ-20 <br />       ACGAGATATGTTGGTGTGGACTTGAACATAGGTGATTCTTGCGACCCTACCATCGAGGGT                        -457 <br />                                                                                           -397 <br />                                                                                                                     Y   G    L   T    0            K    L   0    Q <br />  <br />                                                                                                                 ATC TAT GGC TTA ACT   MAA        GGC AAA TTA CAA   GGA        SL rY (w t) <br />                                                                                           -337 <br />       CAGTAATAAAAAATAATAATAATAATAATAATAACAiGAACACGAATACTAGTAGTAATA                        -277                      Y   G    L   T    m            K    L   0    0 <br />  <br />       GTAGTCAGATACAAAAiTTCTATGTAACATATTTTTTCCAGTCACTATCATCATCATCAT                        -217         527 <br />                                                                                                       Ca                              6532                             6537 <br />       ATTTTGTATATATACATTTATTATATAAAATATCATACCAATTTTTGCCAAAGAACACCG                        -157 <br />       TTTCCCTCTTCGCGTTGACCACAGTAiTTGCAGAAACGACAAAAAAAAACAAAAAAG                            -97 <br />       AAGCACCATTCTAACTCAGCCTTTAATAACCTGACATGATAACTTATAATAGTCATTGCC                         -37 <br />                                                                                                           C <br />       AGTTGCTAAGTATCTTTGACCAAAAATCACAACATCATGGCTGTGGAGGAAATTGCGTCC                          24       &amp;a   106               Kvlo E          Ly       YM            ~ <br />                                                          N             Z   X    A S        S       <span id='am-104' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-105' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-106'  ></span><span id='am-107' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span>aa</span>   220   3GPNIT                        s <br />       CGCAAGGACATTAGTTTAAGAGATATGCAAATCTCTGCCATATTAAAAATGCTGTTTTTA                          84       aa   436                         I LP3 y <br />        R   1   D             LRDNQ <br />                               X                  X   8   A   X   L K         L    r   L     28       aa   476           IN              ag <br />                                                                                                                                  V Y _.NM <br />       AATAAGGATTTGAACJACAACGACAACATCACCACCATCACAGACGATATTTTTAACCAG                         144 <br />        W X     D   L     E N N D             I TT        I T         D   X   r N Q          48 <br />                                                                                                     FIG. 2. Nucleotide deduced amino acid sequences the <br />       CAGGAGATCATCTGGAAGGTGTTGATACTGGACATCAAGAGCACTGCTACCATATCTTCT                         204 <br />        Q Z     I   XI        K V     L   X   L   D   X       8   TT A T      x    8 8       68    SL Yl-20 gene. (A) Nucleotide sequence complete insert the <br />       GTTCTCAGAGTCAATGACCTGCTGAAAGCTGGTATCACCGTTCATTCCTTGATTAAACAA                         264    original isolate, YCp5O-SLY1/5. Bold numbers refer amino acids <br />        V L R V N D L L X a x T v Z 8 L X X Q                                              66 <br />                                                                                                   starting methionine open reading frame. A, <br />                                                                                            324 <br />        D       SS <br />                        ... <br />                         L        DVPAI               Yr      V    -P T <br />                                                                  E8               -- N     106    Potential N-glycosylation site. (B) Comparison SLYI-20 and <br />       ATTGACATTATAGTCAATGACTTGAAAAGCGACAAGTATTCTGAATTTTACATCAACTTT                         384    SLYl(wt) sequences <span id='am-51' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53'  ></span><span id='am-54' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> 532, showing single amino <br />        I   D   I    I V      S D     L   K S D E         Y   8   Z F     Y   xz       r    128    acid (aa) substitution SLYJ-20. (C) Repeated structure Slylp, <br />       ACCTCATCTCTGCCCiGAAACCTCCTGGAAGATTTGGCTCAACAAGTCTCCATCACGGGT                         444 <br />                                                                                                   identified using REPEAT program University of <br />        T 8 8 L P R N L L Z D L Q Q V 8 I T                                             148 <br />       AAATCTGATAAAATCAAACAAGTTTACGACCAATACCTGGACTTCATTGTAACTGAACCG                         504    Wisconsin Genetics Computer Group package. Conserved and <br />        K S D       K    I    X   Q   V   Y   D   Q   Y   L   D   F   I   V   T    Z P      166    identical amino acid residues dark background; identical <br />       GAACTGTTCTCACTAGAAATTTCAAiCGCATACTTGACATTGAATGACCCGAAAACCACA <br />        Z                                             L   T   L N D       P <br />                                                                                            564 <br />                                                                                            186    amino acids boldface. <br />            L       S    L    Z   I   S W A Y                                 E T T <br />       GAAGAGGAAATCACGGGCTTATGCGCCAACATCGCGGACGGTTTATTCAACACTGTCTTA                         624 <br />        Z Z Z I T L C             I A D G L r N T V L                                   208 <br />                                                                                            684 <br />        T I N 8        P I II R A A X G P Z I I                                       228 <br />       GAGAAACTGGGTACAAAATTACGTGATTTCGTCATCAATACCAATTCCTCCTCCACTTCC                         744 <br />                                                                                                   suppressor centromere plasmid segregates inde- <br />        Z   R   LQ T X L R D r V I N T N 8 8 8 T 8                                          248    pendently YPTI locus. <br />       ACTTTGCAGGGAAATGATTCTCTGGAAAGAGGAGTGTTGATTATTTTGGATAGAAACATA <br />        T L Q Q 1 D 8 L X R Q v L I I L D R                       I <br />                                                                                            804 <br />                                                                                            268       The final proof correct clone suppressor <br />       GATTTTGCCTCCATGTTTTCACATTCGTGGATTTATCAGTGCATGGTCTTTGATATTTTC                         864    gene obtained came ability cloned DNA <br />                                     II Y Q C N V F D I V <br />        D T aA SN T         8V  8                                                           288 <br />                                                                                                   direct plasmid integration chromosomal SL YJ <br />       AAATTATCAAGAAACACCGTTACTATTCCACTTGAAAGTAAGGAAAACGGAACTGACAAT                         924 <br />         E L    8S R          T V T       I P     L   Z3 8        Z   N   T <br />                                                                          A <br />                                                                                   D   N <br />                                                                                       . <br />                                                                                            308    locus. For purpose, plasmid YCp5LU-SLY1/5 was <br />                    X         L A         X X Y       D       Z P N D              F <br />                                                                                            984 <br />                                                                                            328 <br />                                                                                                   modified deleting CEN4 URA3 sequences create <br />                                                                                                   YRp5L-SLY1/5. This plasmid cleaved unique <br />        T   T A          P            T                   I                   F <br />  <br />  <br />                                                                                           1044 <br />        N               S9    L   P   r       Z   A   A   Z       v   S   A   A    L   N    348    PvuII site cloned sequence direct integration to <br />        T Y   IR Z     A Z I T R         XT     vV    N I 8 D <br />                                                                                           1104 <br />                                                                                            368    homologous chromosomal site Ieu2 strain carrying <br />       TTGGACCCAAACTCGAATAATGATACGGTTCAAAT$CAGGAAGTCGTAAAGAAACTGCCA                        1164    GAL10- YPTJ fusion gene (strain HLR3). As revealed by <br />        L   D   P   N    8        N   D   T   V   QI  1Q Z        V   V       X    L   P    368 <br />                                                                                           1224 <br />                                                                                                   sequence analysis (see below), PvuII site close the <br />        9   L T 9 R N T I D T 2 N N I T a L L                                           408    SLYJ-20 mutation mutation easily lost <br />       TCACAATTGGAAAGTiAAAGTCTTGATACTTTCTTTGAAGTTGAACAAGACCCTGGAAGC <br />        8   Q L Z        8    R   8   L   D   T   *   *   Z   V   Z   Q D     P    G   8 <br />                                                                                           1284 <br />                                                                                            428 <br />                                                                                                   repair double-strand break (68). In fact, several <br />       ACCAAAACAAGATCTAGGTTTTTGGATATACTAAAiGATGGTAAGACAAACAATCTTGAG                        1344 <br />                                                                                                   Leu+ transformants obtained did grow on <br />        T X T R          8    R V     L   D I     L   K   D   aG E T               L   Z    448    glucose, showing contained SL YJ(wt) <br />       GACAAACTGAGATCGTTTATTGTTC&#177;TTACTTGACATCAACGACAGGCTTACCAAAGGAT <br />        D   R   L   R    8    Y   I   V   L   Y   L   T   8   T   T     L   P    R   D <br />                                                                                           1404 <br />                                                                                            466    allele. To test linkage LEU2 sequence SL Y (wt) <br />       TTTGTCCAAAATGTGGAAAACTATTTCAAGGAAAACGATTACGATATCAATGCTTTAAAA <br />                                        N D Y D I N A L E <br />                                                                                           1464    allele, transformants (INT1) crossed a <br />        rV Q      NV zaS     Y TF X <br />       TATGTTTATAAGCTAiGAGAGTTCA+GCAATTATCAAATATGTCGTTACAAAATAiAATCT <br />                                                                                            488 <br />                                                                                           1524 <br />                                                                                                   SLYI-20 leu2 strain (LSL20-1A) subjected tetrad <br />        Y   V   Y    E L      R   Z   r N Q       L   8   N N 8       L   Q   N    X   8    506    analysis. All 46 spores derived (10 complete tetrads <br />        L Z D G 8 D 8 r R P 8 N L T L 8 x Y <br />                                                                                           1584 <br />                                                                                            526 <br />                                                                                                   2 viable spores) SLYJ(wt) and <br />       GGCTTiACTAAAGGCAAATTACAAGGAGGTGTAGGAAGTTTAATATCAGGTATTAAGAAG                        1644    Leu+ SLYI-20 Leu-. This finding clear evidence <br />        L T X O R L Q G v G 8 L I 8 ra I R x                                            548    site integration closely linked the <br />       TTACTGCCTGAAAAGAAAACCATCCCGATAACAAATGTTGTTGATGCGATAATGGkCCCT                        1704 <br />                                                                                                   SLYJ locus, suggests cloned sequence the <br />        L   L    3Z X E  TIS P X T N V V D A x N D P                                        568 <br />       TTAAACAGCTCTCAAAAAAATTTGGiGACTACTGACAGTTACCTATACATCGACCCAAAA <br />            U                 E N L <br />                                                                                           1764    SLY] -20 allele. <br />        L       8   8    Q      ZSTT D 8 Y L Y X D P X                                      566 <br />       ATTACAGGGGTTCCCATACCAGAAAGCCAAAAAGACAATCTTATAATAAGTCACTAGTG                         1824 <br />                                                                                                      Cloning SLYI(wt) allele. The repair double- <br />        x T R 8 Z ST R           P X R Q S8 TY          8 L V                             608    strand break PvuII-cleaved plasmid YRp5L-SLY1/5 was <br />       TTTGTAGTTGGTGGCGGTAACTACCTAGAATATCAAAACCTCCAAGAATGGGCACATTCC <br />         V V V G N Y L Z Y Q N L Q 3Z                   S 8 <br />                                                                                           1884 <br />                                                                                            626 <br />                                                                                                   used clone SLYJ(wt). This plasmid inte- <br />                                                                                           1944    grate genome replicate autonomously yeast <br />        Q L     S N P E X V N Y G 8 T AI T T P A X                   648                           cells result presence ARS1 sequences. As <br />       TTCTTAAATGAGATTiCTCGCCTTGGCGCAAGTAAiAGCAGCAACAACGATGCATAATTG 2004                           repair intermediates resolved plasmid inte- <br />         T LC    Z I 8 R L AC    A 8      8 8 N T DA        *      666 <br />       GCATCTAGCCCCAATGACGATACACTAATATATATATATATATATATATATATATATCTG                        2064    gration genome, free YRp5L-SLY1/5 plasmids are <br />                                                                                           2124    released. This type transformant grows slowly minimal <br />       GGTCCCATTTTGATACAATGTTTTTGGTTTAGTGAiTTGCGTAATATTTATTTTTATTTA                        2184    medium lacking leucine, ARSI-containing plasmids are <br />       TGAAATGTGCTTTAGGCTTTCTTCACTGTTACAAATAATTTGCGGAAGTTTCTAAiATCT                        2244    poorly transmitted daughter cells (44). Transformants of <br />       TTGTTGACCTTTTGGCATCCAAAAACAGCAATTTGGCTTCATTAACGTCATTAACAACAA <br />       TTTCCTTACGATTGCiTGTTTGTGCiAGGATACCACATGGTGCCAJATAATTGTAAjGCGT <br />                                                                                           2304 <br />                                                                                           2364 <br />                                                                                                   Glu- phenotype assumed contain plasmids carrying <br />       AACGTAACGAAGTTTCTGTACCCATCGTGGCTAAAAGGTCCAATGCGCTACTTTC$ACTT                        2424 <br />                                                                                                   SL Y (wt) gene. The SL Y (wt) gene contained YRp5L- <br />       GCAATCTTTCAACGG$AGCTCTTCTCTCTATAATGGTACGGATCTCGTCCTTGTCATATG                        24 84 <br />                                                                                                   SLY1/5wt slow-growing Glu- transformant was <br />       GTAATGTACGAACAAfTAACAATGTATCGAT                                                     2515    sequenced differ nucleotide from <br />                                                                                                   sequence SL Y -20 mutant gene (G instead A in <br />  VOL. 1 l, 1991                                                        SLY GENES AS SUPPRESSORS OF YPTI IN YEAST CELLS                                                               877 <br />  <br />  <br />          A                           B          N       w <br />                                                                  &gt;q               A <br />                                                                                                                                                                              I kb <br />                         )p. <br />                         VW                      &gt;JCl)   .&gt;ICl)   Cl <br />                                                                                                       Spel                     -        Spel <br />  <br />  <br />          3.36-                       3.36- <br />  <br />                                                                                                                      )............-...i <br />                                                                                                                        _s               . - <br />                                                                                                                                        .......... d11       ........ <br />                                                                                                                            sx,yz <br />  <br />                                      1.65- <br />  <br />          1.65 - <br />                                      1.44 _ <br />                                      1.31 - <br />                                                                                   EcoRl <br />                                                                                                                      I. <br />                                                                                           Pouoh                                                                      Pvull <br />          1.31 - <br />                                                                                   EcoRt                                                EcoRl <br />                                                                                           PvuhI                  oi <br />                                                                                                                 P, <br />                                      0.61   - <br />  <br />  <br />  <br />  <br />                                                                                                                 Be IX-Iragment <br />  <br />                                      0.31-                                        B <br />                                                                                                                                                             0 <br />                                                                                                                                                             w: <br />                                                                                                                                0.                           LU <br />   FIG. 3. Detection specific transcripts SL Y], SL Y2, SLY12, <br /> and SL Y41. Total cellular RNA glyoxylated, 10 ,ug lane                                              I                                 II <br />                                                                                                                                          b                   b <br /> separated 1.5% agarose gels, mRNAs hybridized with <br /> 32P-labeled DNA fragments. To identify SLY] (A) SLY12                                     8.45- <br /> SL Y41 (B) transcripts, DNA fragments used are <br /> indicated probes Southern blot analysis shown Fig. 4, 11,                               5.69 <br /> and 12, respectively. The SLY2-specific probe AatII frag-                                  4.82- <br /> ment ranging 186 bp 5&apos; translation start 4 bp 3&apos;                            3.68-          410                   aw               4m m <br /> stop codon. Sizes marker fragments (in kilobases) shown on <br /> the left.                                                                        C <br />  <br />  <br />  <br />  <br /> the position codon 532 1,998-bp open reading <br /> frame). <br />    The wild-type SLYJ gene isolated subge-                     FIG. 4. Disruption SLYI gene. (A) The SLYI gene was <br /> nomic library constructed HindIll fragments (7 9 kb)                 interrupted inserting 2.2-kb HpaI fragment carrying LEU2 <br /> of wild-type yeast DNA vector YCp5O. The gene                        gene selectable marker HpaI site SLYI coding <br /> identified cloned HindIII fragment 8 kb                 region. The linear 4.3-kb SpeI fragment carrying disrupted SLY] <br /> hybridization 32P-labeled probe SLYJ-20 mutant                   gene used transform diploid Leu- strain. Transformants <br /> gene. The recombinant plasmid YCp5O-SLY1(wt) contained                       wild-type SLYJ gene chromosome disrupted <br /> the entire 1,998-bp open reading frame flanking se-                      SL YI gene identified Southern blot analysis <br /> quences. Sequence analysis showed single base                       used sporulation tetrad dissection. The 0.8-kb BgIII <br /> difference codon 532 (compared SL YI-20)                      fragment used Southern blot hybridization. (B) Southem blot <br />                                                                              analysis diploid strain disrupted SLY] allele. Chromo- <br /> SLYJ(wt) derived YRp5L-SLY1/5wt.                                        somal DNAs untransformed Leu- diploid strain (lanes <br />    It noted SLYJ(wt) overexpressed                     (lanes a) Leu+ transformant (lanes b) digested with <br /> multicopy vector unable suppress loss YPTJ                     PvuII EcoRI probed 32P-labeled 0.8-kb BgIII <br /> function, SLYJ-20 plasmid integrated                  fragment. The appearance 7.2-kb PvuII fragment 4.5-kb <br /> into genome rescue cells lacking Yptlp.                            EcoRI fragment (lanes b) indicates correct integration disrup- <br />    Sequences SLYl(wt) SLYI -20 genes protein                tion SL YI allele. The sizes marker fragments (in kilobases) <br /> products. As mentioned above, dideoxy sequencing                      shown left. (C) Tetrad analysis slyl -ISL Yl + diploid <br /> 3.1-kb DNA insert contained YCp5O-SLY1/5 revealed                       strain analyzed panel B, showing (SLYI ) spores <br />                                                                              form colonies. <br /> long open reading frame 1,998 bases (Fig. 2A). Deletions <br /> within cloned sequence extending reading frame <br /> eliminated yptl--complementing activity. Assuming the <br /> first AUG reading frame translation initiation                tein 14 potential N-glycosylation sites (marked Fig. <br /> codon, SL YJ-20 mutant gene encode protein                    2A) internal repeats 40 amino acids (Fig. <br /> 666 amino acids. Sequence comparison wild-type                    2C). Neither repeats region Slylp <br /> mutant genes disclosed nucleotide difference (G-                    exhibits significant homology proteins listed available <br /> to-A transition) result substitution lysine              data banks. <br /> for glutamic acid position 532 Slyl-20 mutant                        SLY) encodes essential function. When BgIIH fragment <br /> protein (Fig. 2B).                                                           ranging nucleotides 258 1294 SLY) protein- <br />    Slylp hydrophilic protein total 25% charged                coding region used hybridization probe RNA <br /> amino acids evenly distributed entire length               blot analysis, unique 2.2-kb RNA detected growing <br /> of protein. There hydrophobic regions long                        yeast cells (Fig. 3A). To determine gene coded <br /> enough predict membrane-spanning domain. The pro-                       essential function, chromosomal SLYJ allele was <br />  878        DASCHER ET AL.                                                                                                                                                        MOL. CELL. BIOL. <br />  <br />                GGATCCCCTTTACTTCTTCATTTAGCGATGATACCTTCAAACAAAGACACCATTT                   -646   medium subjected restriction analysis. Among 11 <br />       AAAGAGGCCAATGATTTGCTTGAAGAACAACTCTTTAAGATCGCTCACTGTCAAAATCAT                       -586 <br />       ACGTATTGAAAAATGTTCATCCATCCGAATCCAGGGATTTTACCTACCAAATCTGCAGGA                       -526 <br />                                                                                                 different plasmids, 7 contain distinguishable <br />       ACTGTTCATCAACAATACCCTTTGCGACACCACCCAACACCAAAAACAAGCGTCAAATGA                       -466 <br />                                                                                                 yptl--suppressing sequences designated SLY2, <br />                                                                                          -406 <br />                                                                                                 SLY3, SLY6, SLYJ2, SLYJ3, SLY41, SLY44. SLY44 <br />       TGCTCAATTAGAGGACGACTCTAACTACGTTCTTAATATCAGTAACCTACAAGACCATTT                       -346   identical GAL4 gene (38) by <br />       ACCAAATCTGGACCTGCGGGAGAGTTTCCCAACTTTCTTCGATATAAGACAGAAGTTTGT                       -286   restriction enzyme mapping. The isolation GAL4 the <br />       CGTGGTTTGAGGTCTAATTTTCAACCAAACGTGTTGTACTAGCATATATAATTAAATAAA                       -226   suppressor selection procedure chosen surprising, <br />                                                                                          -166   constitutive expression GALJ-GALJO-regulated genes <br />       CAATAAATCCTCAACGCTTTTTGTTTTCATTTTTCGGTCTTTTTCTACCATAGAAGATAA                       -106   known result overexpression transcriptional <br />       ACAAGTAGAGCCCATGTAATATAATATATCTACAACCTATTATTACCCAAAACCAACAAA                        -46   activator (32). SL Y441GAL4 proved the <br />       AACCCTGACAGTGACACCCCGTTACACACTCACAATTAAGTAGGGATGATAAAGTCAACA <br />                                                                      x   x   X   8   T <br />                                                                                            15 <br />                                                                                             5 <br />                                                                                                 seven multicopy suppressors allowing cellular growth at <br />                                                                                            75 <br />                                                                                                 37&#176;C glucose-containing medium (YPTI gene switched <br />        L   I     Y   R   K   D   0   L   P   L   C   T   S   V   D   H   Z   N   D   P     25   off). Transformants overexpressing SL Y2, SL Y3, SL Y6, <br />       TCATTATTTGAACAAAAGCAAAAGGTGAAAATCGTCGTTTCCAGATTGACACCACAGTCT                        135   SL YJ2, SL Y3, SL Y41 Ts- medium, similar <br />        8 L F Z Q X Q X V X I V V 8 R L T P Q 8                                             45 <br />       GCCACGGAGGCTACTTTGGAAAGTGGCTCCTTTGAGATCCATTATTTGAAGAAATCCATG                        195   SLYJ-20 mutant. <br />        A T Z A T L Z 8 8 Jr               I  H Y L X KX 8                                65      A stringent test YPTJ independence newly <br />       GTGTACTACTTCGTCATTTGCGAATCTGGATACCCAAGAAACTTAGCATTCTCATACCTT                        255   discovered suppressors performed analyzing their <br />        v Y S r V C          S8 Y P R          L A r S Y L                              85 <br />       AACGATATAGCGCAAGAATTTGAACACTCATTTGCTAATGAGTACCCCAAGCCTACCGTA                        315   suppressing capacity YPTI deletion strain. The diploid <br />        N D I A Q ZK F Z           8 F A H Z Y P X P T V                                   105   strain DAH430H (YPTJIyptJ::HIS3) transformed with <br />       AGACCATACCAGTTTGTAAACTTTGATAACTTTCTACAGATGACAAAAAAGTCATACAGT                        375 <br />        R P Y Q r V x F D N T L Q N T X X S Y 8                                            125   recombinant plasmids containing different multicopy <br />       GATAAGAAAGTTCAGGACAATTTGGACCAACTAAACCAAGAGCTTGTAGGTGTCAAGCAA                        435   suppressors, Leu+ transformants sporulated, tet- <br />        D X X V Q D N L D Q L N Q Z L V G V X Q <br />       ATCATGTCCAAGAACATCGAAGACCTACTTTACAGGGGAGATTCTCTCGATAAAATGAGT <br />                                                                                           145 <br />                                                                                           495 <br />                                                                                                 rads dissected. Since YPTI deletion lethal, His&apos; <br />        I   N     8   X   N   I   Z   D   L   L   Y   R   C   D   8   L   D   X   N   8    165   colonies generated spores untransformed <br />       GACATGAGTTCTTCTTTGAAAGAAACATCCAAAAGGTACAGAAAGTCCGCGCAAAAGATC <br />        D     8 8      L K8 <br />                          K ZST 8 K R Y R              A Q      I X 8             x <br />                                                                                           555 <br />                                                                                           185 <br />                                                                                                 DAH430H. In contrast, small number slowly growing <br />                                                                                           615 <br />                                                                                                 His+/Leu+ colonies (one 10 tetrads) grew of <br />        N T D L L I 8 Q Y A P I V I V A J r r v                                            205   spores derived SL Y2-, SLYJ2-, SL Y41-transformed <br />       TTTCTCTTCTGGTGGATCTTCCTCAAATAGACGTCACCCCATCATCCCACATCACAAATC <br />         V L V W W I * L X * <br />                                                                                           675 <br />                                                                                           214 <br />                                                                                                 diploids rich medium. Their His+/Leu+ phenotype indi- <br />       CCGATCAATTTGGTCCAAGTGTATAACATAGTCTAGTTCAGTATTAGTATAGTATGGCAA                        735 <br />                                                                                                 cated contained YPTI deletion were <br />       TTTATTTGACATAATTACGTATTTTTCAGGCAAGCTGGCAGGTGACCCGCCGCACCTGCA                        795 <br />       CTTTTTAGGGAGTCtTTTCAAGGATATCATCACTATATATCGAAGGAAAAGAGGCTTTCC                        855 <br />       AACTCTTTTGGATGATCGAATTGCCATCACTACTCTATTAACTGTCTGTGTATTAACTAA                        915       CCCCATGGTGTTTACTCTGTGATAGGAACCAAAGATCCCGTAATTATCGCTTTGTAATTG                                 -386 <br />       ATAATCACATAGTATTAAGTTATAATTCCTGGAAAAAAAAAGAAAGCGAAACGACAAAAT                        975       TATTTCCAGTTATTTCTGTACATTCTAGGTTACTGTTCAGGGATATGTTTCACTCTTATC                                 -326 <br />                                                                                          1035       GATGTGGCGTGCAAAAAATTTGAAGGGGCGACATATCAGGGAAAGTATTAGTGAT TTTT                                 -266 <br />                                                                                          1095       ATAATGTTGCACTATCTCAGATGAAATTTATGCATGCTAATTGGCGGATACTACTCTACC                                 -206 <br />                                                                                          1155                                                                                                    -146 <br />       GAAGACCCTCTCAAGATGGCCGCTCCACTCCAGTTTTATATAATTGTGAAAACGAATATT                       1215       AAGTATCCAGTAGTTGTTAAAAATGGAGTTTTTTCAACTCGCGCCTTTTGCCAAGGAATG                                  -86 <br />       CTTGCATCAACGGAiTTCAGGAATTAAAGGAGATiACTTCCAATGATATTTACTACTGGG                       1275       GTAGAAAACAAACAGTAGTTCACTTGAGACTGTGAATGTGTTAGAAAATCGTGATTGAAG                                  -26 <br />       GTTTGTCAGTGATAAAGCAG                                                               1295       AGCCTCACAACACTAGGCTACACAGATGAGTTCAAG,A.TAGAAATTATTAGACTATTC                                    35 <br />                                                                                                          TT                                     8    8       R                                   .  4 <br />   FIG. 5. Nucleotide deduced amino acid sequences                                        TAAAGTTAGAGTAGCGCTTTTTGTTAAACA TCTACCA TAACACCTCGA TA TTGGGAGTTA                               95 <br /> SLY2 gene.                                                                                          TTGACCACGAGTGAACTTCGTTTaCTAACTATAGCGTGTGTATAiATTTGCAGGGGGA                                    155 <br />                                                                                                                                                    F A c                                          a <br />                                                                                                                                                                                      .~AGO <br />                                                                                                     AACGCTTATCAACGTGATACTGGTAGAACACAGTTATTCGGACCGGCTGATGGATCAAAT                                  215 <br />                                                                                                      N A Y         Q R D T O R T Q L                             r     P A     D   C   8   N     28 <br />                                                                                                     AGTCTCGATGACAATGTATCATCAGCGCTAGGGAGCACAGATAAATTAGACTACTCCCAA                                  275 <br /> disrupted insertion LEU2 marker gene                                              S    L    D   D    N   V   8   8   A   L 0          8       T   D   K   L   D   Y   8   Q     48 <br />  <br /> HpaI site codon 390. A linear 4.3-kb SpeI fragment                                          AGTACTTTGGCATCTCTTGAATCTCAAAGTGAGGAACAGATGGGAGCTATGGGTCAGAGA                                  335 <br />                                                                                                      ST L A 8               L   Z S Q 8 Z                Z       Q H GC H              Q   R     68 <br /> part interrupted SLYJ coding region used                                              ATAAAAGCACTCAAGTCATTATCGTTGAAGATGGGTGATGAGATTAGAGGCAGCAATCAA <br />                                                                                                          I K A L K <br />                                                                                                                                                                                                   395 <br />                                                                                                                             8   L   8   L K H                  D   Z   I   R GC    8       Q       88 <br /> transform homozygous Leu- diploid strain replace one <br />                                                                                                     ACTATTGACCAGCTTGGTGATACTTTCCATAACACTTCTGTAAAACTCAAAAGGACTTTT                                  455 <br /> chromosomal SLY] gene disrupted allele (Fig. 4A).                                           T I D Q L C D T r N N TS V X L K R T r                                                       108 <br /> Southern analysis Leu+ transformants confirmed                                           GGAAACATGATGGAGATGGCCAGAAGATCTGGGATCAGTATAAAAACATGGTTAATAATA <br />                                                                                                           NH           HZ <br />                                                                                                                                                                                                   515 <br />                                                                                                      G                      M   AR      R   S       CG           8   I X T U L I             I    128 <br /> the integration LEU2 marker SL YJ locus (Fig.                                         TTTTTTATGGTAGGCGTGCTATTTTTTTGGGTATGGATTACATAAATATTTGCATATTGT                                  575 <br /> 4B). Each 10 tetrads derived transformant                                             rF r     v G V L r r          V 11 I T                                                       142 <br /> gave rise viable spores (Fig. 4C),                                          ACATTAAACCTAAGTACAAAATAAGAGCGACGTTTGTTAGATGATTGTAGATATTGATAC                                  635 <br /> Leu-. Microscopic inspection spores carrying                                           GTGAAATAAAGGATAGTTATGATAGCATCATAAGiTATCAGCATTCTTTTATTTTTATAA                                  695 <br /> the disrupted SLY] allele showed unable                                           CTACTTAAGCGTTTAATCTTGATTTTTTTTTTTCTCCATATTTCCTAGGATACCGACAGC                                  755 <br /> divide, indicating intact SLYJ gene essential                                        GTTCCTTTTCGCCCTATAGTGTTTCCTTACTTTGTATCTTCTGTAAATAGATGGACCTTT                                  815 <br />  <br /> cell viability.                                                                                     GAGTATTGCGATAAACAAAACAATAACAATTAAGTAACCGGGCATGCGTTCCCGTGGATT                                  875 <br />  <br />    Isolation multicopy suppressors YPTI deletion. Multi-                                      ATGTCTAGGAAACTGAGGACATTATGGAATGGAGAATTTATCAATCTCCCCTTCAACTTA                                  935 <br />                                                                                                     GAGCTGTATAGAGCTTTTGATGGAATATTTAAGGACACGAATGCiTCTCTCAGCAATTGA                                  995 <br /> copy suppressors essential genes exhibit similar                                         GGGTTGTTAGAAGTCAAGGAGTTGCCAAAATTATCTTTATTGTAGTAGTAATTATTTGAT                                 1055 <br /> function act pathway gene able                                       TTATCAATGATAATCACATCACCAATTTTATATTCTCCAGTAATACCATATTTTCTAAAA                                 1115 <br /> to suppress. In searching suppressors loss YPTI                                       AAATTCTTCTCCTTTGAAATATCTAAATATCCTAATAACAATTTTTTTTGATCCATATGG                                 1175 <br /> function, genomic library partially Sau3A-digested                                           GGAATCTCTTCACGCATTTTATCAiCAATTTTGTTAGCCGGAGCCTTTTTTTCCTTTATT                                 1235 <br /> DNA SLYJ-20 yptl - strain multicopy vector                                             TTCTTGATCCCGTCTGCTTTCCTTGCCCTTCTCTCATTAATTTCTTCGAAGTTATTTTCC                                 1295 <br /> YEp13 (13) constructed used transform haploid                                          ATCCGAATATGTTCATACTCCCAACTTGCGAGGAGCAAGTTTTTAAT                                              1342 <br /> strain chromosomal YPTJ gene GALIO                                                FIG. 6. Nucleotide deduced amino acid sequences the <br /> promoter control. Recombinant plasmids retrieved                                           SLY12 gene. The splicing consensus sequences intron are <br /> from transformants growing glucose-containing                                        underlined. <br />  VOL. 11, 1991                                                             SLY GENES AS SUPPRESSORS OF YPTI IN YEAST CELLS                        879 <br />  <br />  <br />        A                                                                         dependent recombinant plasmids carrying LEU2 <br />                                                                                  marker gene. The small number His+/Leu+ colonies <br />         Sly2p                    MIKSTLIYREDGLPLCT-SVONEND                       explained unequal transmission plasmids <br />        Slyl2p MSSRFAGGNRYORDTGRTOLFGPRDGSNSLOONVS <br />                                                                                  different spores meiosis assumes a <br />         Sly2p PSLFE-OKQKVKIVVSRLTPQSATERTLESGSFEI                                certain number vectors containing suppressor genes is <br />        Slyl2p SALGSTDKLD-       YSQS TLSLESOSEE                                  required allow growth yptl- spores. <br />                                                                                     To confirm SLY2, SLY12, SLY41 multicopy <br />         Sly2p HYLKKSMVYYFVICESGYPRNLRFSYLNDIROEFE                                suppressors, cloned sequences harboring suppress- <br />        Slyl2p                                                                    ing activity cloned single-copy vector <br />         Sly2pHSFRNEYPKPTVRPYOFVNFDNFLOMTKKSYSDKK                                 YCp5O (SL Y2 SL Y12) transform GALI O- YPTI <br />        SIyl2p                                                                    haploid strain duplicated genome (SLY41) through <br />                                                                                  integration recombinant vector lacking 2,um origin of <br />         SIy2P VODNLOQLNELVGVKOIMSKNIEOLLYRGOSLOK                                 replication (for details, Materials Methods). None of <br />        Slyl2p    OMGRMGURIKRLKSLSLSMGDEI-RG -NOT                                 Leu+ transformants survived shutdown YPTI <br />        Sly2p MSDMSSSLKETS-KRYRKSAQKINFO                          LISOY <br />                                                                                  transcription glucose medium. This result proved the <br />       Slyl2p IDOLGDTFHNTSVKLKRTFGNMMEMARRSGISIKT                                 genes multicopy suppressors. SL Y3, SL Y6, <br />                                                                                  SLY13, unable suppress deletion of <br />        Sly2p PIVIVRFFFVFLFWVIF                                                    YPTI, analyzed further. <br />       Slyl2p WLIIFFMVGV-LFFVWIT                                                     SLY2 SLY12 encode related proteins similar structure <br />                                                                                  synaptobrevin. Deletion analysis cloned DNA <br />       B                                                                          fragments contained YEp13-SLY2, YEp13-SLY12, and <br />                                                                                  YEp13-SLY41 established suppressor activities <br />       aa 125   8 D K K V 0 0 N L DO L NO E L V 0 v K 0 1 M S                     confined short regions, 1.5-kb BamHI- <br />       aa     .             Q M G R M GO R I K A L. K S L S L                   EcoRV fragment (SLY2), 1.8-kb Ncol-HindIll fragment <br />       aa 43    f K K L 4.      T 0 AK V 0 E v V G I M R V N V E <br />       ea 28      N R R L QQT         0 V vE V:D I M R v N V 0 <br />                                                                                  (SLY12), 1.6-kb HindIII-Sau3A fragment (SLY41). <br />                                                                                  These longer fragments subjected sequence <br />                K NI E         L L Y R CO    S L DKMS O M S 8 S                   analysis. In case SL Y2, longest open reading frame <br />                KM G 0E            I RO      S NOT I DOL GO T                     indicated protein 214 amino acids (Fig. 5). This pre- <br />                KVL E R 0 0 K L S E L OE     R A O O L E G AS O <br />                K V L E R DO K L S E L OD    R AR L o R G A S o <br />                                                                                  dicted reading frame correlates 0.9-kb SL Y2 <br />                                                                                  mRNA (Fig. 3B) allows 200 nucleotides of <br />                L K ET S      K R Y R KSA c K   I N F O. L L                      untranslated sequence. <br />                FHNTS       v KLKRTF G NM       ME MA R RSGIs                        The nucleotide sequence SLY12 (Fig. 6) revealed the <br />                F   EQ0 3 G KLK R K c wW R      N M K .......                     presence 131-bp intron interrupting fourth codon. <br />                F   E T 8 R FKLKRKYWW K         N L K    .. <br />                                                                                  The intron predicted highly conserved yeast <br />                0 YR    P Iv IV R F F F V F L F W W I FL K    *     Sly2p         splice site sequences, GTATGT 5&apos; TAG 3&apos; <br />                I K T   W L I I F F MV G v L F F w V W I T    *     Slyl2p        splice junction, branchpoint TACTAAC sequence <br />                MM I    I L G VI A V V L L I I V L V S Vw P   5     SB, Dro.      near 3&apos; splice site (22, 36, 37). The predicted 5&apos; splice site <br />                MM I    I L G VI C R I I L I I I I V . YFIS   S     SB,bov.       sequence SLY12 intron, GTATGA, deviates one <br />                                                                                  nucleotide typical GTATGT sequence, but <br />                                                                                  yeast gene, S1O-2 (39), the <br />       C                                               Bovine                     GTATGA donor sequence. As typical 5&apos; <br />                        Sly2p                          Synaptobrevin              splice site sequence NcoI restriction site <br />                                                                                  (position -442; Fig. 6) introns typically interrupt S. <br />                                                                                  cerevisiae protein-coding regions codons <br />                                                                                  (22), predicted intron size location correct. <br />                                                                                  According DNA sequence, SL Y12 gene encodes a <br />                                                                                  protein 142 amino acids. This size compatible the <br />                                                                                  length SLY12 mRNA, RNA blot analysis <br />                                                                                  shown 0.6 kb long (Fig. 3B). <br />                                                                                     An alignment Sly2p Slyl2p sequences shown in <br />                                                                                  Fig. 7A. At C termini, proteins contain highly <br />                                                                                  hydrophobic stretch 20 amino acids rich in <br />                                                                                  aromatic amino acids, followed polar or <br />                                                                                  charged residue. Slyl2p shares sequence similarities <br />                                                                                  Sly2p hydrophobic C terminus. Between <br />  <br />                          F                               R                       Slyl2p corresponding regions synaptobrevins from <br />                                                                                  Drosophila head (Dro.) bovine brain (bov.) (66). Only those <br />   FIG. 7. Sequence comparison yeast Sly2p Slyl2p                      amino acid (aa) residues highlighted conserved or <br /> synaptobrevins higher eucaryotes. (A) Comparison amino                   identical (bold) Sly protein the <br /> acid sequences Sly2p Slyl2p. Identities (bold) favored                synaptobrevins. The hydrophobic C-terminal regions boxed. (C) <br /> substitutions highlighted shaded boxes. The C-terminal                    Helical wheel analysis portions Sly2p bovine synaptobre- <br /> hydrophobic regions boxed. The regular spacing tyrosine               vin. In case, consecutive segments displayed (see <br /> phenylalanine residues central region Sly2p indicated              numbers schematic <span id='am-28' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-29' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32'  ></span>helix</span>). The idealized helix is <br /> by large bold letters. (B) Alignment c-terminal regions Sly2p              drawn according conventions given Landschulz et al. (35). <br />  880                   DASCHER ET AL.                                                                                                                                                     MOL. CELL. BIOL. <br />  <br />                                                                                                                                             regions, 74-residue-long gap at <br />                                                                                                                                             position 62 Slyl2p. The domain, present Sly2p, is <br />                                                                                                                                             unusual having tyrosyl phenylalanyl resi- <br />        A                                                                                                                                    dues eighth position. <br />                                                                                                                                                Notably, primary structures Sly2p Slyl2p <br />                                                                                                                                             resemble VAMP-1 <span id='am-36' about='obo:NCBITaxon_7786' typeof='owl:Thing obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_7711 obo:NCBITaxon_30480 obo:NCBITaxon_117888 obo:NCBITaxon_131567 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_33154 obo:NCBITaxon_7776 obo:NCBITaxon_7777 obo:NCBITaxon_7778 obo:NCBITaxon_117893 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-37'  ></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Torpedo" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Torpedo</span> electric lobe (76) <br />   GCTTACGATATTTAGAGAGTAAGTGGATTGACGATAAAAGTGTTTAATAAGGAGCQATTC                                                                              synaptobrevin bovine brain Drosophila head <br />   GTTTTTATAAAGGAGAAACAATACCGGCCTAACATGATTCAAACGCAAAGTACAGCGATC <br />                                                                                       S! Q       T Q      2S       A       S                (67). These proteins known integral membrane <br />   AAICGACGTAATTCTGTTCATAAAAACCTCTTTGATCCATCACTGTATCAAATACCAGAA                                                                              proteins synaptic vesicles. The hydrophobic <span id='am-115' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-116' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-117'  ></span><span id='am-118' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>car</span>- <br />                                    S                Xx E          L I         D       S 8        L T      Q    I *         Z <br />       x       R       R N                  V <br />                                                                                                                                             boxy-terminal domain serves membrane anchor, with <br />   CCACCACGGGGGTTCCAACACCAAAAAAAGAATACAGTAAGGAGACATTCAGTAATCAA <br />     1    1aa1 Q EQ xx *                *3 x1   S 13        Q                                                                                rest protein exposed cytoplasm (67, 76). The <br />                                                                                                                                             highest degree conservation proteins from <br />                                                                                                                                             different <span id='am-56' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> region 65 amino acids <br />   ATACAAGGGTACTTACCTGAAGTTGACCTAAGAATAACCATTATTTGTTCTATATGGTAC <br />    I Q T L P 2 V D L A I I I X C S X W T                                                                                                  precedes membrane-spanning domain (67). This <br />   GTCACGTCATCTiTTTCAAGTAACCTATCTAAiGCCATTTTAAGAACTTTCAACCATCCC                                                                              region flanked group basic amino acids, can <br />    V     S!X            N L L     R KA XA LI R IS r       r <br />   ATAGCTCTCACAUATTGCAATTTCTTGTTAGTGCTGTTCTATGTGTTGGATTTGCTTCC <br />                                                                                                                                             readily aligned corresponding regions Sly2p <br />    x AL S z L Q r L V S AV L C V r s                                                                                                    Slyl2p. If 52 amino acid residues aligned as <br />   ATCGTAAATTTATTCCGACTACCTCGTTTGAAGCATACGAAGTTTTCAAAGGCACTCAAT <br />       X       V E          L       r       R L       P    R       L A                 T1 x       r   8         A L E <br />                                                                                                                                             shown Fig. 7B, 30.8 32.6% identity, respectively, is <br />                                                                                                                                             observed Sly2p synaptobrevin either <br />       8       1       1    D       G       x    L    I    r       x   L       D         E      1   3    8    3   I       L                bovine brain Drosophila cells. The alignment 47 amino <br />   CACAAGTTTTTAGTTCCCTCAAAGCTGGTTTTGATGACCACTTTCCCTATGGGAATATTC <br />    9 x r L V P 8 S L V L X T T r P X C x J <br />                                                                                                                                             acids region Slyl2p synaptobre- <br />   CAATTTATTGGTCATATCACATCGCACAAGGCGGTATCTATGATACCAGTATCACTAGTG <br />                                                                                                                                             vins shows 21.3% identical residues. These domains may <br />    Q P I X I T s X s A V 8 X I P v s L V                                                                                                  maintained ability form similar secondary struc- <br />                                                                                                                                             tures. Secondary structure predictions (24) strongly suggest <br />       CGTTATTACAATTCCATGACTTATTATACCTTGTTACTTTTAATTTTTGGCGTTATGACT                                                                          large parts domains exist ao-helical <br />        R r s 1I s N s s s T L L L L I T C V N T                                                                                             conformation. Helical wheel analysis (Fig. 7C) these <br />       ACTTGCTGGTCAACACATGGCAGTAAGAGGGCTTCAGATAACAAAAGCGGATCTTCATTA <br />        T c W s T 3 G 8 A R s D N A S C S S L <br />                                                                                                                                             domains reveals nonrandom distribution hydrophobic as <br />       ATCGGGTTGCTTTTTGCCTTTATTTCCATGATAATATTTGTAGCACAGAATATTTTTGCA                                                                          charged polar amino acids putative <br />        I C L L r r I s K I s T v Q                   X s ra                                                                             helices. <br />       AAGAATATTTTAACCATCAGAAGGAAGGTAGGAATACTGCCGTCTTCTTCTACGGATGAC <br />           K E I L                  T       I R       R A          V C         S       L   P S S           8 T D            D                   SLY41 encodes protein related phosphate translo- <br />                                                                                                                                             cator chloroplasts. Sequence analysis 1.6-kb <br />        V      T 8          R       Z       C    Q P 8             L   D       A       T   R J        S    P    L   Q V                      HindIII-Sau3A fragment YEpl3-SLY41 containing the <br />       GATAAGATTACCATATTGTTCTACTCGTCCTGCATTGGGTTTTCTTTAACCCTATTACCT                                                                          suppressor YPTJ deletion disclosed long open reading <br />        D      I I          T           I   L    r    Y     8      8   C I C               J      8   L T L         L       P <br />       TTTTTAACCGGCGAATTAATGCATGGCGGTAGCGTTATCAACGATTTAACGCTAGAAACA                                                                          frame hydrophobic protein 452 amino acids <br />        J L T C Z L M E&#162; C            8 V XI      D L T L Z T <br />                                                                                                                                             (Fig. 8A). The predicted size protein consistent <br />       GTAGCCCTTGTAGCGATTCATGGAATAGCCCAtTTTTTCCAAGCAATGCTTGCTTTCCAG                                                                          1.6-kb transcript identified Northern blots a <br />        V A L V          K G !s A       1 1 Q Aa M L A I Q <br />       TTGATCGGTTTACTATCTTCCATTAATTATTCGGTAGCAAACATCATGAAGAGGATTGTT                                                                          32P-labeled fragment deduced SL Y41 coding region <br />        L I C L L 8 8 I! N            8 V A E I         K R I Vv <br />                                                                                                                                             (Fig. 3B). <br />        V          I    8   V           A   L     T    E       3   T       K       L   E      I   T     Q V F           C V           429       Four different regions Sly4lp hydrophobicity val- <br />       ATCTTGACAATTGCCGGATTGTACGGTTATGACAAATGGGGGCTTTCCAAAAAAGATGGA                                                                   1347   ues 1.6 determined according Kyte and <br />           1       L T          I             C L T C T D KX                                 C L        8 K           D       C      449 <br />                                                                                                                                             Doolittle (34), using window 19 amino acids. Thus, <br />       CGTCAGGCATAAiATGAAGTAATACTATTGAGTACAAACCATCATTCAAGGTAGGACACT <br />        R Q A* <br />                                                                                                                                      1407 <br />                                                                                                                                       452   Sly4lp contain transmembrane domains. If <br />       TAGTACTATATCAAATATATAATCAAAAACCCTCGAAGGTCTTTTTTATCTTATT                                                                        1467   protein forms channel embedded the <br />       TACTGCTACTACCTCTGTTGCTAGCATTACTTCAAGATCC                                                                                       1507   membrane, contain amphiphilic helices. A struc- <br />                                                                                                                                             ture prediction analysis uses algorithm plotting <br />               B            9*                                                                                                         130 <br />                                                                                                                                             hydrophobicity putative helices (31) <br />                                                                                                                                             predicts Sly4lp form membrane- <br />                                                                                                                                             spanning helices, hydrophobic seven <br />               Sly4lp IS AfLRT NH I <br />       Ph.-Transi. NWK                             N <br />                                                 Y YN <br />                                                          R <br />                                                      PY YF <br />                                                                                       E <br />                                                                                       V HL <br />                                                                                            L              RIL <br />                                                                                                           A <br />                                                                                                               Y <br />                                                                                                                                FR <br />                                                                                                                                S W          amphiphilic (data shown). Corroborating assump- <br />                            119                                                                                                        iS1   tions, channel-forming phosphate translocator spinach <br />                            1Ss                                                                                                        217   chloroplasts (20) protein detected comput- <br />            Sly41p E_F 0 F                                             H       HKl|VlM L v l_sl <br />                                                                                                                                             er-assisted alignment exhibits significant similarity with <br />        Ph.-Transl.  CHA <br />                             69201 <br />                            1s <br />                                                                               NVIFIRA*F T                                            El     primary sequence Sly4lp. Two different algorithms, <br />                            386  41 s                                                                                                        RDF2 (51) written Argos (2), regions <br />            S1y41p        L_lNY <br />        Ph.-Transl. ST TER lTH H <br />                                MFJLJG                                                                    F~F <br />                                                                                                               VVl                           high similarity (standard deviations 6); these <br />                            333                                                                                                 362          regions indicated hydropathy profiles presented in <br />                                                                                                                                              Fig. 9. A comparison sequences shown Fig. 8B. <br />    FIG. 8. Nucleotide sequence SL Y41 gene comparison                                                                             Most remarkably, Lys-353 Arg-354, thought <br /> of Sly4lp phosphate translocator spinach chloroplasts.                                                                          involved substrate binding phosphate trans- <br /> (A) Nucleotide sequence 1.6-kb HindIII-Sau3A fragment                                                                           locator (79), correspond Lys-406 Arg-407 in <br /> carries suppressor function. (B) Comparison regions of <br /> Sly4lp phosphate translocator spinach chloroplasts                                                                              Sly4lp (Fig. 8B). These basic amino acids located at <br /> (20) display highest degree homology (regions 1, 2, 3                                                                       equivalent positions similar amphipathic helices a <br /> indicated Fig. 9).                                                                                                                       distance 45 50 amino acids C terminus. <br />                                                                                                                                                SLY12 SLY2 SLY41 essential gene. Since the <br />                                                                                                                                              SL Y genes isolated complementation defective <br />                                                                                                                                              YPTI gene, protein products function same <br />  VOL. 11, 1991                                                   SLY GENES AS SUPPRESSORS OF YPTJ IN YEAST CELLS                                                       881 <br />  <br />           Sly4lp <br />           0        1 00 <br />            1,,,1,,,I,,,1,,,I,1II,, <br />                                   200      300 <br />                                                    II <br />                                                         400 <br />                                                         II                     A <br />       3- <br />                                                                                               BamHI                                  BamHi <br />  <br />                 &apos;AA <br />            11V - rWT <br />               W                                Iv <br />             Vt <br />            *3i <br />  <br />                                                v <br />                                            i <br />  <br />  <br />  <br />  <br />                                                                                                             5-i rz <br />       3                            - <br />  <br />       0I4-I W.Av          y   v         WqAk <br />                                            U&apos; -W,&apos; <br />                                                    \A [0                      EcaRl   BamHl                                                       EcoRl       BamHI <br />  <br />      -31 <br />                                                                                           EcoRI BamHI                    Ec RI           BamolH <br />           0        1 00           200      300          400 <br />            Phosphate-Tranalocator <br />   FIG. 9. Hydropathy profile Sly4lp phosphate translo- <br /> cator spinach chloroplasts. Hydrophobicity Sly4lp the <br /> phosphate translocator determined according Kyte                    B                                                     2 <br /> Doolittle (34) window 11. The relative positions                                              I                                    I <br /> regions displaying significant homologies, determined according                                      1      2         A       B         C       D <br /> to Argos (2), indicated. <br />                                                                                       5.69 - <br />                                                                                                                                                   m <br /> pathway GTP-binding protein. In attempt                             3.68 - <br /> prove this, mutants constructed disruptions the <br /> multicopy suppressors. Different marker genes, HIS3,                                  2.32 - _                           -                 m <br /> LEU2, URA3, inserted coding regions                              1.93 - <br /> SLY2, SLY12, SLY41, respectively (Fig. 10 12). <br /> Linear DNA fragments containing disrupted genes                     FIG. 10. Disruption SL Y2 gene. The HIS3 gene a <br /> used transform diploids homozygous his3, leu2,              selectable marker inserted SLY2 coding region to <br /> or ura3 defects. In case, representative transfor-               disrupt SLY2 gene described Materials Methods. A <br /> mants shown Southern blot analysis                    linear 5.2-kb SmaI-SalI fragment carrying disrupted SLY2 gene <br /> disrupted wild-type allele subjected                  used transform diploid his3/his3 strain (DAH430). (A) The <br /> sporulation. Tetrad analysis showed insertion                 1.5-kb BamHI-EcoRV fragment used Southern analysis. Rele- <br /> URA3 gene middle SL Y41 coding region                     vant restriction sites indicated. (B) Southern blot analysis to <br />                                                                           demonstrate correct integration disrupted gene one <br /> (codon 208) effect spore viability. Of 32 tetrads               chromosome. Chromosomal DNAs untransformed His- <br /> dissected, 28 gave rise spores formed colonies            diploid strain (lane 1), His&apos; transformant (lane 2), from <br /> equal size. The severe truncation Sly4lp               spores derived His&apos; transformant (lanes 2A 2D) <br /> effect cell viability, suggests, does neces-             digested EcoRI probed 32P-labeled 1.5-kb <br /> sarily prove, protein essential function.                BamHI-EcoRV fragment. The hybridizing 4.3-kb fragment lanes <br />    Different results obtained SL Y2 SL Y12               2, 2B, 2D indicates correct integration disruption one <br /> inactivated. No viable spores carrying LEU2 marker                    SLY allele. The sizes marker fragments (in kilobases) shown <br /> inserted codon 117 SLY12 gene obtained.                  left. <br /> None 37 tetrads dissected produced viable <br /> spores, 53 viable spores Leu-; i.e., they <br /> contained wild-type SL Y12 allele. Microscopic inspec-                explained fact diploid strain used trans- <br /> tion inviable spores showed 25%                 formation absolutely isogenic contained a <br /> had germinated stopped dividing               modifier gene determining phenotype some <br /> cell divisions. Thus, SLY12 gene encode function                sly2: :HIS3 haploids. To prove this, Cs-ls- spore <br /> that essential yeast vegetative growth. For disrup-            (sly2::HIS3) crossed wild-type spore (SL Y2) having <br /> tion SL Y2, 57 96 bp NcoI site codon 64              Ts- background, inferred Ts&apos; phenotype the <br /> were deleted BAL 31 digestion insertion                  Cs- spores tetrad. The wild-type strain <br /> HIS3 marker protein-coding region. As 5&apos;                     used cross derived tetrad containing two <br /> deletion endpoints differed base pair, truncated                   Cs- spores Ts-. Assuming modifier <br /> proteins formed N-terminal 55 amino acids               gene segregates 2:2 fashion, strain carry the <br /> of Sly2p fused 17 9 amino acids encoded               allele causing heat sensitivity SLY2-disrupted cells. The <br /> inserted DNA. In cases, 80% tetrads                     resulting diploid gave rise sly2::HIS3 spores that, without <br /> derived different transformants having wild-type                 exception, Cs- Ts-. <br /> and disrupted SLY2 allele produced viable spores. <br /> After 2 days 25&#176;C, clear 2:2 segregation                                               DISCUSSION <br /> size colonies. The smaller colonies His&apos; <br /> and carried SLY2 disruption. They                   We isolated genes able suppress deletion <br /> strictly cold sensitive (Cs-), half                YPTJ, essential yeast gene encoding small GTP- <br /> thermosensitive (Ts-), judged inability grow               binding protein. Three genes identified multicopy <br /> on plates 15 37&#176;C, respectively. The heterogeneity                 suppressors (SLY2, SLY12, SLY41), SLYI-20 <br /> with respect Ts- phenotype sly2- cells                 mutant allele exerts suppressing activity present in <br />  882       DASCHER ET AL.                                                                                                                       MOL. CELL. BIOL. <br />  <br />                                                                                                                                                        0.5Skb <br />       A                                                                        A <br />                   NccI                                                                             Hindlh                             Nhe1 <br />  <br />                                        f.....    ~ <br />                                       {L- <br />                                            sL Yrt <br />  <br />  <br />  <br />           Clal <br />            ---   E! <br />                                                             Clal <br />  <br />  <br />  <br />  <br />                                                                                                                    Scal <br />                                                                                                                        I                       Scal <br />                                                                                        cap <br />                              Clal <br />                                       II <br />                                                                                              .:         ..;                     .&quot;, <br />                                                                                                                                     Scal <br />  <br />                                     StIy        <span id='am-41' about='obo:NCBITaxon_717364' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_82596 obo:NCBITaxon_82593 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_82592 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-42' property="rdfs:label" content="Ira" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45'  ></span>Ira</span> gment <br />  <br />  <br />       B                             b              c                                                      Ha clT/ScaI-fragmrenf <br />  <br />                                                             -      3.68        B                                          b              c <br />                                                      -_                                           2.32- <br />                                                                                                   1.93- <br />                                                                    1.93 <br />                                                                                                                             _              _ <br />                                                                                                   1.37 _ <br />                                                             -      1.37                           1.26- <br />   FIG. 11. Disruption SLY12 gene. The SLY12 gene                 FIG. 12. Disruption SLY41 gene. The SLY41 gene was <br /> interrupted insertion 3.0-kb BglII fragment carrying          interrupted insertion 1.2-kb HindlIl fragment carrying the <br /> LEU2- gene selectable marker BglII site SLY12        URA3 gene selectable marker XbaI site SLY41 <br /> coding region. The linear 4.8-kb HindIII-NcoI fragment carrying       coding region. The linear 2.7-kb HindIlI-NheI fragment used to <br /> disrupted SL Y12 gene used transform diploid Leu- strain.        transform diploid Ura- strain. (A) Relevant restriction sites and <br /> Transformants wild-type SL Y12 gene chromosome              1.4-kb HaeIII-ScaI fragment used Southern blots. (B) <br /> and disrupted SL Y12 gene identified Southern      Southern blot analysis. Chromosomal DNAs untrans- <br /> blot analysis. (A) Relevant restriction sites 0.83-kb StyI        formed Ura- diploid strain (lane a) Ura+ transfor- <br /> fragment used hybridization probe Southern blot analysis. (B)      mants (lanes b c) digested ScaI. The appearance of <br /> Southern blot analysis. Chromosomal DNAs untrans-                additional hybridizing 1.6- 1.3-kb fragments indicates <br /> formed Leu- diploid strain (lane a) Leu+ transfor-           correct integration disruption SL Y41 allele. The sizes of <br /> mants (lanes b c) digested ClaI HindIII             marker fragments (in kilobases) shown left. <br /> probed 32P-labeled 0.83-kb StyI fragment. The appearance <br /> of additional hybridizing 2.8-kb fragment indicates correct <br /> integration disruption SLY12 allele. The sizes marker <br /> fragments (in kilobases) shown right.                          GTP-binding protein. In particular, sequence motifs typical <br />                                                                           GTP-binding proteins absent different Sly <br />                                                                           proteins. Sly2p, Slyl2p, Sly4lp integral mem- <br /> only copy. From published restriction map cloned               brane proteins ER, Golgi, vesicular structures, <br /> DNA fragment carrying BET] gene, protein                        cellular compartments involved functioning Yptlp <br /> product acts ER-to-Golgi transport (48),                  (4, 60, 63). Evidence membrane localization been <br /> possible BET] SLY12 identical genes.                         obtained Sly2p (49a). <br />   The selection genes suppressing deletion instead                 How rationalize overexpression gene <br /> point mutations essential gene certain advantages.              complements deletion essential gene encoding a <br /> It precludes isolation intragenic suppressors                  protein totally unrelated primary structure? This kind of <br /> suppressors simply act altering expression             genetic complementation signal transmis- <br /> mutant gene processing, stability, translational                sion pathways, defect essential protein can <br /> efficiency transcription product. Both suppression             suppressed overexpression downstream element. <br /> of deletion suppression overproduction                   For example, functional adenylyl cyclase dispensable in <br /> indicative suppressor gene replaces                  yeast cells overexpressing different cyclic AMP-dependent <br /> deleted gene pathway induces pathway                protein kinases (72). In contrast, proteins like Yptlp and <br /> that bypasses need deleted gene. Other investi-               Sec4p thought act unidirectional vesicle transport <br /> gators shown yeast cells, suppression               (26, 63), transport cycle regulated the <br /> deletion (15) multicopy suppression (for review,               switch GDP- GTP-bound form. As first <br /> see reference 7) genes functionally                pointed Boume (11), role proteins might <br /> related defective gene. Interestingly, yeast genes            analogous played elongation factor EF-Tu, <br /> encoding small GTP-binding proteins, SEC4 (57), SARI (47),                GTP-binding protein increases translational accuracy <br /> RSRJ (7), CDC42 (7), identified multicopy                (71). According hypothesis, Yptlp GTP-bound <br /> suppressors protein products structurally unre-                 state bind protein vesicle surface and, <br /> lated genes suppress. Similarly, func-               simultaneously, docking protein target mem- <br /> tional relatedness totally different primary structures           brane. GTP hydrolysis release GDP-bound pro- <br /> now observed case Yptlp SLY gene                        tein follow vesicle docking correct target <br /> products compensate complete loss                 membrane. <br />    V <br /> VOL.        1LY GENt <br />      1 l, 1991                                                            AS SUPPRESSORS OF YPTI IN YEAST CELLS                        883 <br />  <br />    Speculating line, envisage spe-        ertheless, defect transport Ca2&quot; specific <br /> cific vesicle surface target membrane proteins used          membrane compartments increase phosphate <br /> vesicle docking usually limiting, vesicle attach-       release compartments comparable <br /> ment acceptor membrane inefficient         effects yptl defects. In contrast pmrl mutations, <br /> process absence GTP-binding protein. Under            however, suppress cold-sensitive yptl-l <br /> conditions excess of membrane compo-          mutant (54), Sly4lp overproduction suppress dele- <br /> nents, efficiency vesicle docking increased         tion YPTI. Moreover, suppression Sly4lp overpro- <br /> such rate protein transport sufficient      duction specific yptl defects, yptl- <br /> allow cell viability absence Yptlp. This       suppressors improve growth certain sec <br /> mechanism multicopy suppressors SLY2               mutants restrictive temperatures (49a). Unfortu- <br /> SLY12 act. It possible SLY2 SLY12 genes             nately, sly41 - cells viable particular <br /> code vesicle-specific surface docking proteins,        phenotype tested different growth conditions. <br /> they contain domains degree hydrophobicity        Therefore, analysis mutants fail determine the <br /> sufficient span membrane. As noted above, Sly2p            normal function gene. <br /> Slyl2p able assume secondary structure similar            SL YJ, SL Y12, SL Y2 essential cell <br /> to VAMP-1/synaptobrevin proteins higher            viability confer conditional lethality. This characteristic <br /> eucaryotes region preceding C-terminal hydrophobic         makes possible analyze mode their <br /> transmembrane region. If, indeed, parts domains            yptl--complementing activity determine the <br /> exist a-helical conformation, hydrophobic residues       normal cellular functions genes. <br /> always limited ridge face helix <br /> (Fig. 7C). This structure enable proteins form                              ACKNOWLEDGMENTS <br /> dimers composed identical different subunits (35). The <br /> a-helical regions form long flexible coiled-coil              We indebted F. Pfeiffer (Munich) protein sequence <br /> structures expose N-terminal domain pro-           comparisons, U. I. Flugge (Wurzburg) communicating unpub- <br />                                                                     lished results, R. Merkl help programs. We <br /> tein (possibly true receptor domain) distance        thank Hannegret Frahm Heike Behr expert technical assis- <br /> the surface membrane.                                        tance, Hans-Peter Geithe synthesizing oligonucleotides, and <br />    The single-copy suppressor SL YJ-20 encode protein         Ingrid Balshusemann invaluable secretarial help. <br /> acting downstream Yptlp. In contrast multicopy              This work supported grant Fonds der <br /> suppressors, SL YJ gene carry specific point             Chemischen Industrie D.G. <br /> mutation <span id='am-59' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> complement loss Yptlp. Overex- <br /> pression wild-type SL YJ allele equivalent                                       REFERENCES <br /> effect. Therefore, protein encoded SL YI-20 mu-           1. Achstetter, T., A. Franzusoff, C. Field, R. Schekman. 1988. <br /> tant gene exist specific conformation                SEC7 encodes unusual, high molecular weight protein re- <br /> wild-type protein adopt conformational                   quired membrane traffic yeast Golgi apparatus. J. <br /> change induced, complex Yptlp. A                Biol. Chem. 263:11711-11717. <br />                                                                      2.   Argos, P. 1987. A sensitive procedure compare amino acid <br /> trivial explanation activation SLYJ-20               sequences. J. Mol. Biol. 193:385-3%. <br /> parallel pathway substituting Yptlp-requiring one.           3.   Bacon, R. A., A. Salminen, H. Ruohola, P. Novick, S. <br /> There evidence, however, Slylp itself required             Ferro-Novick. 1989. The GTP-binding protein Yptl required <br /> the ER-to-Golgi transport (49a). Slylp does exhibit                 transport vitro: Golgi apparatus defective yptl <br /> significant hydrophobic character. This unusual,             mutants. J. Cell Biol. 109:1015-1022. <br /> most cloned sequenced SEC genes yeast cells,           4.   Baker, D., L. Wuestehube, R. Schekman, D. Botstein, N. <br /> SEC2 (45), SEC7 (1), SEC14 (5), SEC15 (56), SEC18 (19),                   Segev. 1990. GTP-binding Yptl protein Ca2+ function <br /> SEC23 (29), SEC53 (8), code cytosolic factors                independently cell-free protein transport reaction. Proc. <br /> most likely transiently associated membranes                Natl. Acad. Sci. USA 87:355-359. <br /> through interaction proteins. Only       5.   Bankaitis, V. A., D. E. Malehorn, S. D. Emr, R. Green. <br />                                                                           1989. The Saccharomyces cerevisiae SEC14 gene encodes a <br /> characterized SEC genes, SECJJ (10), SEC12 (46),                      cytosolic factor required transport secretory <br /> SEC59 (9), known code integral membrane pro-                   proteins yeast Golgi complex. J. Cell Biol. 108:1271- <br /> teins. Yptlp Sec4p, hand, bound                 1281. <br /> membranes C-terminally located cysteine res-           6. Barbacid, M. 1987. ras genes. Annu. Rev. Biochem. 56:779- <br /> idues, case Yptlp shown                 827. <br /> modified lipidation (43, 78).                                     7. Bender, A., J. R. Pringle. 1989. Multicopy suppression of <br />    The mechanism SL Y41 gene product act               cdc24 budding defect yeast CDC42 newly <br /> is certainly different suppressors. The          identified genes including ras-related gene RSRJ. Proc. <br />                                                                         Natl. Acad. Sci. USA 86:9976-9980. <br /> sequence similarity Sly4lp phosphate transloca-          8. Bernstein, M., W. Hoffmann, G. Ammerer, R. Schekman. <br /> <span id='am-110' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-111' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-114'  ></span>tor</span> spinach pea chloroplasts (20, 79) suggest             1985. Characterization gene product (SecS3p) required for <br /> that SLY41 codes channel protein trans-               protein assembly yeast endoplasmic reticulum. J. Cell <br /> locate ions metabolites intracellular membranes. It           Biol. 101:2374-2382. <br /> is known particular yptl defects suppressed              9. Bernstein, M., F. Kepes, R. Schekman. 1989. SEC59 en- <br /> either adding Ca2&quot; growth medium (60)                   codes membrane protein required core glycosylation in <br /> defects Ca2&quot; ATPase (Pmrlp) sequester                  Saccharomyces cerevisiae. Mol. Cell. Biol. 9:1191-1199. <br />                                                                     10. Bohni, P. C., R. J. Deshaies, R. Schekman. 1988. SECII is <br /> cytosolic Ca&quot; storage compartment           com- <br />                                                                         required signal peptide processing yeast cell growth. J. <br /> partment secretory pathway (54). This suppression                Cell Biol. 106:1035-1042. <br /> seems nonspecific, cell-free transport           11. Bourne, H. R. 1988. Do GTPases direct membrane traffic in <br /> showed Ca2&quot; addition restore protein trans-              secretion? Cell 53:669-671. <br /> port extracts yptl mutants (3) Yptlp-          12. Broach, J. R., Y. Y. Li, L.-C. Wu, M. Jayaram. 1983. <br /> requiring step precedes calcium-dependent (4). Nev-             Vectors high level, inducible expression cloned genes in <br />  884     DASCHER ET AL.                                                                                                       MOL. CELL. BIOL. <br />  <br />      yeast, p. 83-117. In M. Inouye (ed.), Experimental manipula-          33. Kikuchi, A., T. Sasaki, S. Araki, H. Yutaka, T. Yoshimi. <br />      tions gene expression. Academic Press, Inc., New York.                   1989. Purification characterization bovine brain cyto- <br /> 13. Broach, J. R., J. N. Strathern, J. B. Hicks. 1979. Transfor-             sols GTPase-activating proteins specific smg p21, a <br />      mation yeast: development <span id='am-135' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-136' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-138'  ></span><span id='am-139' property="rdfs:label" content="hybrid" datatype="xsd:string"></span>hybrid</span> cloning vector                 GTP-binding protein having effector region c-ras <br />      isolation CANI gene. Gene 8:121-133.                                 p2ls. J. Biol. Chem. 264:9133-9136. <br /> 14. Broek, D., T. Toda, T. Michaeli, L. Levin, C. Birchmeier, M.           34.   Kyte, J., R. F. Doolittle. 1982. A simple method for <br />      Zoller, S. Powers, M. Wigler. 1987. The S. cerevisiae                   displaying hydropathic character protein. J. Mol. Biol. <br />      CDC25 gene product regulates RAS/adenylate cyclase path-                157:105-132. <br />      way. Cell 48:789-799.                                                 35.   Landschulz, W. H., P. R. Johnson, S. L. McKnight. 1988. <br /> 15. Cannon, J. F., J. B. Gibbs, K. Tatchell. 1986. Suppressors               The leucine zipper: hypothetical structure common new <br />      RAS2 mutation Saccharomyces cerevisiae. Genetics                  <span id='am-33' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-34' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> DNA binding proteins. Science 240:1759-1764. <br />      113:247-264.                                                          36.   Langford, C. J., D. Gallwitz. 1983. Evidence intron- <br /> 16. Carlson, M., D. Botstein. 1982. Two differentially regulated             contained sequence required splicing yeast RNA poly- <br />      mRNAs different 5&apos; ends encode secreted intracellular              merase II transcripts. Cell 33:519-527. <br />      forms yeast invertase. Cell 28:145-154.                            37.   Langford, C. J., F.-J. Klinz, C. Donath, D. Gallwitz. 1984. <br /> 17. Chardin, P. 1988. The ras superfamily proteins. Biochimie                    Point mutations identify conserved, intron-contained TAC <br />      70:865-868.                                                                 TAAC box essential splicing signal sequence yeast. Cell <br /> 18. Chen, E. Y., P. H. Seeburg. 1985. Supercoil sequencing:                36:645-653. <br />      fast simple method sequencing plasmid DNA. DNA                38.   Laughton, A., R. F. Gesteland. 1984. Primary structure of <br />      4:165-170.                                                                  Saccharomyces cerevisiae GAL4 gene. Mol. Cell. Biol. <br /> 19. Eakle, K. A., M. Bernstein, S. D. Emr. 1988. Characteriza-               4:260-267. <br />      tion component yeast secretion machinery: identifi-       39.   Leer, R. J., M. M. C. van Raamsdonk-Duin, C. M. T. Molenaar, <br />      cation SEC18 gene product. Mol. Cell. Biol. 8:4098-4109.             H. M. A. Witsenboer, W. H. Mager, R. J. Planta. 1985. <br /> 20. Flugge, U. I., K. Fischer, A. Gross, W. Sebald, F. Lottspeich,             Yeast contains functional genes ribosomal protein S10. <br />      C. Eckerskorn. 1989. The triose phosphate-3-phosphoglyc-              Nucleic Acids Res. 13:5027-5039. <br />      erate-phosphate translocator spinach chloroplasts: nucle-        40. Matsumoto, K., A. Toh-e, Y. Oshima. 1978. Genetic control <br />      otide sequence full-length cDNA clone import           galactokinase synthesis Saccharomyces cerevisiae: evi- <br />      vitro synthesized precursor protein chloroplasts. EMBO J.            dence constitutive expression positive regulatory gene <br />      8:39-46.                                                                  gal4. J. Bacteriol. 134:446-457. <br /> 21. Gallwitz, D., C. Donath, C. <span id='am-88' about='obo:NCBITaxon_283033' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_641308 obo:NCBITaxon_117571 obo:NCBITaxon_8165 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489922 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_8111 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_8112 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-89' property="rdfs:label" content="Sander" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="Stizostedion" datatype="xsd:string"></span><span id='am-92'  ></span><span id='am-93' property="oboInOwl:hasAlternativeId" content="NCBITaxon:54345" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Sander</span>. 1983. A yeast gene             41. Matsumoto, K., I. Uno, Y. Oshima, T. Ishikawa. 1982. <br />      encoding protein homologous human c-haslbas proto-               Isolation characterization yeast mutants deficient in <br />      oncogene product. Nature (London) 306:704-707.                            adenylate cyclase cyclic AMP-dependent protein kinase. <br /> 22. Gallwitz, D., H. Halfter, P. Mertins. 1987. Splicing                Proc. Natl. Acad. Sci. USA 79:2355-2359. <br />      mRNA precursors yeast, p. 27-40. In J. R. Kinghorn (ed.),          42. Milburn, M. V., L. Tong, A. M. deVos, A. Brunger, Z. Yamai- <br />      Gene structure eukaryotic microbes. IRL Press, Oxford.                 tumi, S. Nishimura, S.-H. Kim. 1990. Molecular switch for <br /> 23. Gallwitz, D., H. Haubruck, C. Molenaar, R. Prange, M. Puzicha,             signal transduction: structural differences active and <br />      H. D. Schmitt, C. Vorgias, P. Wagner. 1989. Structural            inactive forms protooncogenic ras proteins. Science 247:939- <br />      functional analysis ypt proteins, <span id='am-25' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> ras-related                945. <br />      nucleotide-binding proteins eukaryotic cells, p. 257-264. In       43. Molenaar, C. M. T., R. Prange, D. Gallwitz. 1988. A <br />      L. Bosch, B. Kraal, A. Parmeggiani (ed.), The guanine-                carboxyl-terminal cysteine residue required palmitic acid <br />      nucleotide binding proteins. Plenum Publishing Corp., New                 binding biological activity ras-related yeast YPTI <br />      York.                                                                     protein. EMBO J. 7:971-976. <br /> 24. Garnier, J., D. J. Osguthorpe, B. Robson. 1978. Analysis        44. Murray, A. W., J. W. Szostak. 1983. Pedigree analysis of <br />      accuracy implications simple methods predicting            plasmid segregation yeast. Cell 34:%1-970. <br />      secondary structure globular proteins. J. Mol. Biol.           45. Nair, J., H. Muller, M. Peterson, P. Novick. 1990. Sec2 <br />      120:97-120.                                                               protein contains coiled-coil domain essential vesicular <br /> 25. Garrett, M. D., A. H. Self, C. von Oers, A. Hall. 1989.                  transport dispensable carboxy terminal domain. J. Cell <br />      Identification distinct cytoplasmic targets ras/R-ras rho       Biol. 110:1897-1909. <br />      regulatory proteins. J. Biol. Chem. 264:10-13.                        46. Nakano, A., D. <span id='am-120' about='obo:NCBITaxon_307617' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_279648 obo:NCBITaxon_131567 obo:NCBITaxon_105389 obo:NCBITaxon_279647 obo:NCBITaxon_6341 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_105391 obo:NCBITaxon_6340 obo:NCBITaxon_6072'><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-123' property="rdfs:label" content="Brada" datatype="xsd:string"></span><span id='am-124'  ></span>Brada</span>, R. Schekman. 1988. A membrane <br /> 26. Goud, B., A. Salminen, N. C. Walworth, P. J. Novick. 1988.             glycoprotein, Secl2p, required protein transport the <br />      A GTP-binding protein required secretion rapidly associates           endoplasmic reticulum Golgi apparatus yeast. J. Cell <br />      secretory vesicles plasma membrane yeast. Cell            Biol. 107:851-863. <br />      53:753-768.                                                           47. Nakano, A., M. Muramatsu. 1989. A novel GTP-binding <br /> 27. Haubruck, H., C. Disela, P. Wagner, D. Gallwitz. 1987. The             protein, Sarlp, involved transport endoplasmic <br />      ras-related ypt protein ubiquitous eukaryotic protein:              reticulum Golgi apparatus. J. Cell Biol. 109:2677-2691. <br />      isolation sequence analysis mouse cDNA clones highly           48. Newman, A. P., J. Shim, S. Ferro-Novick. 1990. BET], <br />      homologous yeast YPTI gene. EMBO J. 6:4049-4053.                   BOSI, SEC22 members group interacting yeast <br /> 28. Haubruck, H., R. Prange, C. Vorgias, D. Gallwitz. 1989.                genes required transport endoplasmic reticulum <br />      The ras-related mouse yptl protein functionally replace           Golgi complex. Mol. Cell. Biol. 10:3405-3414. <br />       YPTJ gene product yeast. EMBO J. 8:1427-1432.                     49. Novick, P., S. Ferro, R. Schekman. 1981. Order events in <br /> 29. Hicke, L., R. Schekman. 1989. Yeast Sec23p acts                 yeast secretory pathway. Cell 25:461-469. <br />      cytoplasm promote protein transport endoplasmic           49a.Ossig, R., C. Dascher, H. H. Trepte, H. D. Schmitt, D. <br />      reticulum Golgi complex vivo vitro. EMBO J.              Gallwitz. Unpublished data. <br />      8:1677-1684.                                                          50. Pai, E. F., W. Kabsch, U. Krengel, K. C. Holmes, J. John, and <br /> 30. Ito, H., Y. Fukuda, K. Murata, A. Kimura. 1983. Transfor-              A. Wittinghofer. 1989. Structure guanine-nucleotide- <br />       mation intact yeast cells treated alkali cations. J.             binding domain Ha-ras oncogene product p21 the <br />       Bacteriol. 153:163-168.                                                  triphosphate conformation. Nature (London) 341:209-214. <br />  31. Jahnig, F. 1990. Structure predictions membrane proteins       51. Pearson, W. R., D. J. Lipman. 1988. Improved tools for <br />       bad. Trends Biochem. Sci. 15:93-95.                             biological sequence comparison. Proc. Natl. Acad. Sci. USA <br /> 32. Johnston, S. A., J. E. Hopper. 1982. Isolation yeast            85:2444-2448. <br />       regulatory gene GAL4 analysis dosage effects       51a.Puzicha, M., D. Gallwitz. Unpublished data. <br />       galactose/melibiose regulon. Proc. Natl. Acad. Sci. USA 79:          52. Rose, M. D., P. Novick, J. H. Thomas, D. Botstein, G. R. <br />       6971-6975.                                                               Fink. 1987. A Saccharomyces cerevisiae genomic plasmid bank <br />  VOL. 11, 1991                                                     SLY GENES AS SUPPRESSORS OF YPTI IN YEAST CELLS                          885 <br />  <br />       based centromere containing shuttle vector. Gene 60:237-             Res. 13:8587-8601. <br />       243.                                                                  67. Sudhof, T. C., M. Baumert, M. S. Perin, R. Jahn. 1989. A <br /> 53.   Rothstein, R. J. 1983. One-step gene disruption yeast. Meth-           synaptic vesicle membrane protein conserved mammals <br />       ods Enzymol. 101:202-211.                                                 Drosophila. Neuron 2:1475-1481. <br /> 54.   Rudolph, H. K., A. Antebi, G. Fink, C. M. Buckley, T. E.              68. Szostak, J. W., T. L. Orr-Weaver, R. J. Rothstein, F. W. <br />       Dorman, J. LeVitre, L. S. Davidow, J. Mao, D. T. Moir.                Stahl. 1983. The double-strand-break repair model recombi- <br />       1989. The yeast secretory pathway perturbed mutations            nation. Cell 33:25-35. <br />       PMRJ, member Ca2&apos; ATPase family. Cell 58:133-145.              69. Tanaka, K., K. Matsumoto, A. Toh-e. 1989. IRA]: an <br /> 55.   Ruggieri, R., K. Tanaka, M. Nakafuku, Y. Kaziro, A. Toh-e,            inhibitory regulator RAS/cyclic AMP pathway Saccha- <br />       K. Matsumoto. 1989. MSI, negative regulator RAS-                 romyces cerevisiae. Mol. Cell. Biol. 9:757-768. <br />       cAMP pathway Saccharomyces cerevisiae. Proc. Natl. Acad.           70. Tautz, D., M. Renz. 1982. An optimized freeze-squeeze <br />       Sci. USA 86:8778-8782.                                                    method recovery DNA fragments agarose gels. <br /> 56.   Salminen, A., P. Novick. 1989. The SeclS protein responds             Anal. Biochem. 132:14-19. <br />       function GTP binding protein, Sec4, control          71. Thompson, R. C. 1988. EFTu provides internal kinetic <br />       vesicular traffic yeast. J. Cell Biol. 109:1023-1036.                  standard translation accuracy. Trends Biochem. Sci. 13:91- <br /> 57.   Salminen, A., P. J. Novick. 1987. A ras-like protein               93. <br />       required post-Golgi event yeast secretion. Cell 49:527- <br />       538.                                                                  72. Toda, T., S. Cameron, P. Sass,  <br /> </body></html>