<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Nutrient Dependence RNase E Essentiality <span id='am-65' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-70'  ></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span>Escherichia coli</span> <br /> Masaru Tamura,a Christopher J. Moore,a Stanley N. Cohena,b <br /> Departments Geneticsa Medicine,b School Medicine, Stanford University, Stanford, <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>California</span>, USA <br />  <br /> <span id='am-81' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-82'  ></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span>Escherichia</span> coli cells normally require RNase E activity form colonies (colony-forming ability [CFA]). The CFA-defective phe- <br /> notype cells lacking RNase E partly reversed overexpression related endoribonuclease RNase G mutation of <br /> the gene encoding RNA helicase DeaD. We carbon source utilization rne deaD doubly mutant bacteria dif- <br /> fers rne&#11545; cells cells mutated deaD loss rne function bacteria limits <br /> conversion glycolytic pathway product phosphoenolpyruvate tricarboxylic acid (TCA) cycle intermediate oxalo- <br /> acetic acid. We mechanism underlying effect reduced production enzyme phosphoenolpyruvate <span id='am-98' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-99' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-101'  ></span><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>car</span>- <br /> boxylase (PPC) adventitious overexpression PPC, facilitates phosphoenolpyruvate utilization connects <br /> the glycolytic pathway TCA cycle, restored CFA rne deaD mutant bacteria cultured carbon sources otherwise <br /> were unable sustain growth. We bacteria producing full-length RNase E, allows formation degra- <br /> dosomes, nutritional requirements different cells supplied N-terminal catalytic region RNase <br /> E mitigation RNase E deficiency overexpression related RNase, RNase G, affected carbon source. Our <br /> results reveal previously unsuspected effects RNase E deficiency degradosome formation nutrient utilization E. coli <br /> cells. <br />  <br />  <br />  <br />  <br /> R    Nase E initially discovered Escherichia coli endori- <br />      bonuclease processes 9S rRNA (1) later shown to <br /> have multifunctional role processing various other <br />                                                                                       ditions: colony formation observed cells grown <br />                                                                                       <span id='am-44' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Luria" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48'  ></span>Luria</span>-Bertani (LB) medium grown on <br />                                                                                       M9 medium containing glucose sole carbon source. <span id='am-54' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-55' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57'  ></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> <br /> structural RNAs (1, 2, 3, 4), global degradation mRNA (5, 6,                   finding prompted investigate nutritional factors affecting the <br /> 7, 8), control plasmid DNA replication (9), matu-                      growth &#9004;rne bacteria, ability cells containing muta- <br /> ration/processing variety small catalytic RNAs (9, 10).                       tions deaD gene (26), encodes RNA helicase (28), <br /> Commonly, cleavage targeted RNA RNase E fol-                         propagate enabled absence comple- <br /> lowed digestion RNase E-generated fragments ri-                        menting RNase. The results report combina- <br /> bonucleases tightly coupled events (11). DNA sequence analysis                     tion glycolytic pathway sugar plus tricarboxylic acid (TCA) <br /> has predicted orthologs E. coli RNase E exist dozens                    cycle sugar needed propagation &#9004;rne deaD doubly mutant <br /> of evolutionarily disparate bacterial <span id='am-59' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-60' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> (12, 13). The ribonu-                   bacteria, requirement observed cells mu- <br /> cleolytic activity RNase E resides N-terminal half, N-Rne                   tated deaD alone. They indicate, shown Western <br /> (14), C-terminal half (CTH) includes binding sites                    blotting confirmed genetically, RNase E-deficient cells <br /> several proteins form multicomponent complex termed                        defective production enzyme phosphoenolpyruvate <br /> &#8220;degradosome&#65533;? (15, 16). Although degradosome formation                         carboxylase (PPC), converts phosphoenolpyruvate ox- <br /> essential E. coli growth (14, 17, 18), deletion CTH                    aloacetic acid connects glycolytic pathway TCA <br /> been reduce rate decay RNase E substrates                     cycle, establishing previously unsuspected role PPC RNase <br /> (19, 20).                                                                             E essentiality. <br />     In E. coli, actions RNase E essential colony-form- <br />                                                                                       MATERIALS AND METHODS <br /> ing ability (CFA) (21), transfer liquid cultures rne(Ts) <br /> mutant bacteria nonpermissive temperature prevents normal                        Strains <span id='am-116' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-117' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-120' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span>. The strains used study listed Table 1. <br />                                                                                       MT4000 constructed method described Lee et al. (6). <br /> cell division results formation elongated filamentous <br />                                                                                       MT4001, MT4002, MT4003, MT2105, MT2005 constructed by <br /> structures containing connected cell bodies (22). Multiple studies                    transformation MT4000 pNRNE4, MT4000 pRNG2S, MT4000 <br /> aimed elucidating basis RNase E essentiality                      pRNG2S&#9004;H, CM2100 pRNG2S&#9004;H, MT2000 pRNG2S&#9004;H, <br /> carried (6, 23, 24, 25, 26). While CFA restored rne                     respectively. MT2113 obtained P1 transduction deaD::Tn10 into <br /> deletion mutants overproduction mutation RNase G (6,                         CM2100, colonies selected tetracycline (Tc) plates containing <br /> 24, 25, 27), RNase E paralog (2, 4) structurally resembles <br /> the amino-terminal catalytic region RNase E (14), cells <br /> grow slowly rne&#11001; bacteria retain filamentation                         Received 29 August 2012 Accepted 17 December 2012 <br /> phenotype characteristic RNase E-deficient mutants liquid                          Published ahead print 28 December 2012 <br /> culture (25). The filamentation RNase G-overproducing &#9004;rne                            Address correspondence Stanley N. Cohen, sncohen@stanford.edu. <br /> mutants reversed overexpression FtsZ; however, FtsZ-                            Supplemental material article http://dx.doi.org/10.1128 <br /> overproducing cells remain defective rate division (25).                     /JB.01558-12. <br />     During recently reported investigations insertion mutations                       Copyright &#169; 2013, American Society Microbiology. All Rights Reserved. <br /> that restore CFA E. coli cells lacking RNase E (26), observed                      doi:10.1128/JB.01558-12 <br /> that survival &#9004;rne cells dependent culture con- <br />  <br />  <br /> March 2013 Volume 195 Number 6                                            Journal Bacteriology   p. 1133&#8211;1141                                           jb.asm.org       1133 <br />  Tamura et al. <br />  <br />  <br />  <br /> TABLE 1 Bacterial strains plasmids <br /> Strain plasmid           Description                                                                       Reference(s) source <br /> Strains <br />    MG1655                   ilvG rfb-50 rph-1 fnr-267 eut                                                     E. coli Genetic Stock Center (CGSC 6300) <br />    MG165D                   Same MG1655 carrying deaD::Tn10 (clone 2)                                  This study <br />    NCM3416                  Same MG1655 carrying rph&#11001; zib-207::Tn10                                40 <br />    MT4000                   Same NCM3416 carrying rne::cat pBAD-RNE                                This study <br />    MT4001                   Same MT4000 carrying pNRNE4                                                This study <br />    MT4002                   Same MT4000 carrying pRNG2S                                                This study <br />    MT4003                   Same MT4000 carrying pRNG2S&#9004;H                                              This study <br />    MT2000                   lacZ43(Fs) LAM&#11002; relA1 spoT1 thi-1 rne::cat; carries pBAD-RNE                      25 <br />    MT2005                   Same MT2000 carrying pRNG2S&#9004;H                                              This study <br />    MT2013                   Same MT2000 carrying deaD::Tn10 (clone 2)                                  26 <br />    CM2100                   Same MG1655 carrying rne::cat pBAD-RNE                                 This study <br />    MT2103                   Same MG1655 carrying rne::cat pLAC-RNE2                                This study <br />    MT2105                   Same CM2100 carrying pRNG2S&#9004;H                                              This study <br />    MT2113                   Same CM2100 carrying deaD::Tn10 (clone 2)                                  This study <br />    MT2114                   Same MT2113 carrying ASKA-ppc                                              This study <br />    MT2115                   Same MT2113 carrying pACYC177                                              This study <br />    MT2116                   Same MT2113 carrying pMT177Z                                               This study <br />    MT2117                   Same MT2113 carrying pCA24N(Ap)                                            This study <br />  <br /> Plasmids <br />   pBAD-RNE                  pSC101 ori Kmr, rne control PBAD                                         6, 25 <br />   pLAC-RNE2                 pSC101 ori Apr, rne control lacUV5 promoter                              41 <br />   pNRNE4                    P15A ori Apr, His-tagged N-rne control lacUV5 promoter                   25 <br />   pRNG2                     P15A ori Apr, His-tagged rng (also N-terminally extended)                   Gift Kangseok Lee <br />                               control lacUV5 promoter <br />   pRNG2S                    P15A ori Apr, His-tagged rng control lacUV5 promoter                     Gift Kangseok Lee <br />   pRNG2S&#9004;H                  P15A ori Apr, natural short-form rng control lacUV5 promoter             Gift Kangseok Lee <br />   pCA24N(Ap)                ColE1 ori Apr                                                                     26 <br />   ASKA-ppc                  ColE1 ori Apr, ppc control PT5-lac                                       This study <br />   pACYC177                  P15A ori Kmr Apr                                                                  42 <br />   pMT177Z                   P15A ori Apr, ftsZ control natural promoter                              25 <br />  <br />  <br />  <br />  <br /> 0.1% arabinose (0.1% <span id='am-49' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Ara</span>) induce adventitious production RNase E      CTAGTGAATTCAGTGGTGGTGGTGGTGGTGCATCATTACGACGT <br /> in &#9004;rne strain (6, 26). P1(deaD::Tn10) prepared MT2013      GCAAACT-3=). The pRNG2S&#9004;H, pRNG2S, pRNG2 plasmids were <br /> described previously (26). MT2103 constructed replacing pBAD-        kindly provided Kangseok Lee (C. J. Moore, H. Go, K. Lee, S. N. <br /> RNE CM2100 pLAC-RNE2 method described Lee et al.          Cohen, unpublished <span id='am-111' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-112'  ></span><span id='am-113' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-114' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span>). Construction ASKA plasmid Ap <br /> (6). To test effects PPC overexpression, introduced      resistance gene described previously (26). <br /> modified ASKA library plasmid (29) expressing PPC (i.e., ASKA-ppc)           Media culture conditions. LB medium M9 minimal medium <br /> an vector [pCA24N(Ap)] MT2113 strain, generating         (30) containing 0.1 mM CaCl2, 1 mM MgSO4, appropriate carbon <br /> MT2114 MT2117 strains, respectively.                                    sources following concentrations used: D-(&#11001;)-glucose, 0.2%; <br />     pRNG2S&#9004;H, expresses natural form RNase G (i.e.,         D-mannitol, 0.2%; D-(&#11002;)-fructose, 0.2%; glycerol, 0.5%; sodium pyru- <br /> N-terminal C-terminal extension), constructed ligating ClaI-      vate, 0.2%; sodium acetate, 0.2%; succinic acid, 0.5%; L-(&#11002;)-malic acid, <br /> and EcoRI-digested PCR products encoding C-terminal region           0.5%; fumaric acid, 0.5%; Casamino Acids (CAA) (see BD Bionutri- <br /> RNase G lacking hexahistidine tag pRNG2S using restric-     ents Technical Manual), 0.05%. Where indicated, LB components the <br /> tion enzyme sites. The PCR primers used rng-948F (5=-TGTCGGTC          following concentrations used: yeast extract (Bacto Yeast Extract; <br /> ATCGCAATCT-3=) 3=-rng-no-His (5=-AGACTAGTGAATTCACAT                     BD), 5 g/liter; tryptone (Bacto Tryptone; BD), 10 g/liter; sodium <br /> CATTACGACGTCAAACTG-3=). pRNG2S, expresses C-terminal                  chloride (Invitrogen), 10 g/liter. Where indicated, antibiotics used at <br /> His-tagged RNase G lacking N-terminal extension, constructed        following concentrations: ampicillin, 50 &#65533;?&#65533;g/ml; chloramphenicol, 10 <br /> by ligating NdeI- NsiI-digested PCR product encoding             &#65533;?&#65533;g/ml; kanamycin, 20 &#65533;?&#65533;g/ml; tetracycline, 5 &#65533;?&#65533;g/ml; streptomycin 50 <br /> N-terminal region RNase G containing natural N-terminal se-          &#65533;?&#65533;g/ml. For experiments, agar plates freshly prepared day <br /> quence (ATGACGGCTGAA. . .) pRNG2 using restriction            used (never stored 4&#176;C overnight bench), bacteria were <br /> enzyme sites. The PCR primers used 5=-rng-nat (5=-ATACATATG            streaked plates containing 0.1% L-arabinose glycerol stock <br /> ACGGCTGAATTGTTAGTA-3=) rng-595R (5=-AGACGCGTTTCA                        (containing 40% glycerol) incubated 37&#176;C overnight, colonies <br /> GATAAG-3=). pRNG2, expresses C-terminal His-tagged RNase G            picked inoculations. <br /> lacking N-terminal extension, constructed ligating NotI-         Plating experiments. For plating experiments, cells freshly <br /> SpeI-digested PCR DNA encoding region rng containing              grown single colony (never grown overnight) mid-log phase <br /> extended N-terminal sequence (ATGAGAAAAGGGATAAAC)                      (optical densities 600 nm [OD600] ranged 0.3 0.6) 37&#176;C LB <br /> pNRNE4 (25) using restriction enzyme sites. The PCR primers        medium containing indicated supplements antibiotics. Cells were <br /> used 5=-rng-2 (5=-GGATCCGCGGCCGCTTTAAGAAGGAGATAT                       diluted 10&#11002;4 10&#11002;5, 100 &#65533;?&#65533;l diluted culture spread LB or <br /> ACATATGAGAAAAGGGATAAAC-3=) 3=-rng-His (5=-GTCTAGA                       M9 plates containing carbon sources, supplements, antibiotics. The <br />  <br />  <br /> 1134   jb.asm.org                                                                                                                 Journal Bacteriology <br />                                                                                                                                            RNase E Essentiality E. coli <br />  <br />  <br />  <br />  <br /> FIG 1 Effects individual LB components CFA &#9004;rne deaD::Tn10 strain. (A) A culture MT2113 LB containing 0.1% arabinose, induces <br /> adventitious expression plasmid-borne rne, spread LB containing lacking 0.1% arabinose, LB containing additional components M9-glucose <br /> but lacking inducer, LB containing 0.2% glucose arabinose. (B) A culture MT2113 spread M9-glucose lacking additive or <br /> supplemented yeast extract, tryptic digest casein (labeled tryptone), NaCl, absence arabinose. (C) Identification amino acid <br /> requirements CFA &#9004;rne deaD::Tn10 strain cultured M9-glucose. A culture MT2113 spread M9-glucose containing variety amino acid <br /> mixtures, absence arabinose. A mixture glycolytic pathway-derived amino acids, i.e., Ser, Cys, Gly, Trp, Tyr, Phr, <span id='am-88' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-89'  ></span><span id='am-90' property="rdfs:label" content="Ala" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Ala</span>, Val, Leu, Ile, His, defined <br /> as &#8220;glycolytic a.a.,&#65533;? mixture TCA cycle-derived amino acids, i.e., Lys, Met, Thr, Asn, Asp, Glu, Gln, Pro, Arg, His, defined &#8220;TCA a.a.&#65533;? The <br /> concentration amino acid 160 &#65533;?&#65533;M. Plates scanned incubation 2 days (A B) 4 days (C) 37&#176;C. <br />  <br />  <br />  <br /> plates incubated 37&#176;C 2 7 days depending experiment            VersaDoc model 1000 imaging (Bio-Rad), corrected loading <br /> and scanned, colony number determined. Different nutrient                    based S1 protein control difference total <br /> combinations resulted different rates colony growth, in-             protein loaded MT2114 strains. Values then <br /> dicated, incubation period extended 2 days enable                normalized relative RNase E PPC band CM2100 cells. <br /> colonies better visualized. Incubation conditions experi- <br /> ment described figure legends. The absence detectable colony           RESULTS AND DISCUSSION <br /> formation 7 days incubation defined CFA-negative                   RNase E required glycolysis E. coli. We recently reported <br /> (CFA&#11002;) phenotype. Colony-forming efficiency (CFE) determined                  Tn10 insertion mutation deaD gene, encodes <br /> dividing colony number observed indicated growth condi-                RNA helicase, enables colony formation E. coli N3433 cells <br /> tion colony number observed condition                  lacking RNase E function (26). During experiments, ob- <br /> arabinose-inducible RNase E gene expressed (26).                                 served ability affected carbon source included <br />     Western blotting. Cultures CM2100, MT2113, MT2114,                         medium. While N3433 E. coli strain RNase E <br /> MT2117 cells grown overnight 37&#176;C LB plates containing 0.1% <br />                                                                                      discovered widely employed study <br /> arabinose appropriate antibiotics. Randomly selected colonies <br /> from plate suspended LB diluted OD600 0.001             RNase E essentiality, strain contains multiple mutations <br /> fresh LB containing appropriate selective antibiotics,           potentially affect nutritional requirements [i.e., lacZ43(Fs), <br /> 0.1% arabinose (as indicated) 100 &#65533;?&#65533;M isopropyl-&#65533;?&#65533;-D-thiogalac-               relA1, spoT1, thi-1] consequently affect ability even <br /> topyranoside (IPTG) (for strains MT2114 MT2117). The cultures                    rne&#11001; cells form colonies efficiently carbon sources (see <br /> were grown approximate OD600 0.8, minimum 6 h,              Fig. S1 supplemental material); chose further <br /> then harvested prepared loading polyacrylamide gel. The               investigate nutritional effects loss RNase E function E. <br /> cells diluted OD600 0.01 &#65533;?&#65533;l water SDS loading               coli strain MG1655, prototrophic loci, con- <br /> buffer boiled. Lysate CM2100, MT2113, MT2117 cells               firming &#9004;rne deaD derivative MG1655 (i.e., strain <br /> an OD600 0.24 cultured inducing agents indicated loaded               MT2113) forms colonies LB 100% colony-forming effi- <br /> alongside lysate MT2114 cells OD600 0.1, proteins <br />                                                                                      ciency relative cells producing RNase E (Fig. 1A). <br /> were separated Criterion XL 3 8% Tris-acetate polyacrylamide gel <br /> (Bio-Rad). The samples transferred 0.1-&#65533;?&#65533;m nitrocellulose mem-                  As seen Fig. 1, absence arabinose induce <br /> brane probed anti-PPC (Agrisera rabbit polyclonal), anti-RNase              plasmid-borne RNase E, &#9004;rne deaD MT2113 strain formed <br /> E (mouse monoclonal), anti-S1 (rabbit polyclonal) antibodies,            colonies LB medium M9 medium containing glu- <br /> <span id='am-62' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-63' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span>, intervening membrane stripping. The numbers provided rep-                cose mixture amino acids carbon sources; however, no <br /> resent densitometric measurements band determined                  MT2113 colonies observed yeast extract mixture <br />  <br />  <br /> March 2013 Volume 195 Number 6                                                                                                                        jb.asm.org 1135 <br />  Tamura et al. <br />  <br />  <br />  <br />  <br /> FIG 2 Effect carbon source combination CFA &#9004;rne deaD::Tn10 strain. A culture MT2113 LB spread carbon sources indicated. The <br /> presence absence 0.1% arabinose indicated. Plates scanned incubation 37&#176;C 4 days. <br />  <br />  <br />  <br />  <br /> of amino acids omitted (Fig. 1B). In contrast, absence             cultured M9-glucose medium. No restoration CFA occurred <br /> any amino acid supplement, strain MG165D, MG1655                     absence deaD mutation (data shown), is <br /> derivative containing intact rne gene mutated Tn10 in-                 consistent previously reported ability mutation of <br /> sertion deaD, formed colonies M9 medium containing                       deaD suppress need RNase E E. coli cells (26). To- <br /> any series tested sugars carbon source (see Fig. S2          gether, findings suggest PPC production neces- <br /> supplemental material), arguing failure MT2113                    sary CFA cells lacking RNase E function, sufficient, <br /> form colonies media lacking supplemental amino acids                  CFA-promoting function mediated mutation <br /> supplied glucose sole carbon source results de-                 deaD distinct PPC-related function. Additionally, <br /> fective RNase E function. Testing additives individual amino               adventitious expression PPC (Fig. 4) did result CFA on <br /> acids (Fig. 1C) indicated amino acids specifically synthesized              M9-malate medium, supporting view upregulation of <br /> through glycolytic pathway did enable CFA MT2113                      PPC does restore normality &#9004;rne deaD mutant cells. <br /> supplied glucose, addition amino acids generated                 During experiments, noted faint &#8220;shad- <br /> through TCA cycle did. Moreover, addition malate, <br /> fumarate, succinate, TCA cycle components and <br /> can generate TCA cycle-dependent amino acids, enabled CFA <br /> on M9-glucose (Fig. 2). Together, results indicate the <br /> pathway TCA cycle-dependent amino acid biosynthesis in- <br /> tact &#9004;rne deaD::Tn10 mutant strain suggest defect <br /> in enzymatic connection glycolytic pathway and <br /> TCA cycle mutant bacteria prevents products glycolysis <br /> from entering TCA cycle. <br />     Functional interactions RNase E PPC. The gly- <br /> colytic pathway TCA cycle connected enzyme <span id='am-121' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-122' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-123'  ></span><span id='am-124' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>phos</span>- <br /> phoenolpyruvate carboxylase (PPC), converts phospho- <br /> enolpyruvate oxaloacetic acid (31). To test hypothesis that <br /> deficiency PPC expression occurs &#9004;rne deaD bacteria, we <br /> used Western blotting examine effects RNase E expres-                  FIG 3 Effect RNase E expression PPC expression &#9004;rne strain, &#9004;rne <br /> sion expression PPC encoded endogenous                   deaD::Tn10 strain, &#9004;rne deaD::Tn10 strain complemented ASKA- <br /> gene plasmid-borne PPC gene (Fig. 3). As seen, expression               ppc vector. Cultures CM2100, MT2113, MT2117, MT2114 <br />                                                                                  grown 0.1% arabinose induce deplete full-length <br /> of PPC protein highly dependent expression RNase E.                 RNase E pBAD-RNE plasmid (MT2114 MT2117 addition- <br /> When full-length RNase E depleted undetectable levels                  ally grown 100 &#65533;?&#65533;M IPTG), separated 3 8% Tris-acetate gel, trans- <br /> growth absence arabinose, PPC protein reduced                  ferred nitrocellulose membrane, probed appropriate anti- <br /> by 79 91%. To determine overexpression PPC res-                    bodies presence RNase E, PPC, S1 proteins. Due high level <br />                                                                                  PPC overexpression ASKA-ppc MT2114 induced 100 &#65533;?&#65533;M <br /> cues CFA bacteria fail produce RNase E, tested                 IPTG, smaller culture loaded lane other <br /> ability MT2114 strain form colonies M9-glucose. As                  strains (as described Materials Methods). Protein expression deter- <br /> seen Fig. 4, overexpression PPC restored CFA bacteria                   mined described Materials Methods. <br />  <br />  <br />  <br /> 1136   jb.asm.org                                                                                                                        Journal Bacteriology <br />                                                                                                                                        RNase E Essentiality E. coli <br />  <br />  <br />  <br />                                                                                   shown). We observed overproduction PPC protein <br />                                                                                   ASKA-ppc plasmid increased RNase E deficiency, <br />                                                                                   contrast observed cells expressing PPC the <br />                                                                                   endogenous gene. As ASKA-ppc plasmid expresses open <br />                                                                                   reading frame PPC gene lacks 5= 3= untranslated <br />                                                                                   regions, result suggests RNase E production a <br />                                                                                   dual role PPC synthesis affecting mRNA synthesis and <br />                                                                                   translation. <br />                                                                                       Interestingly, introduction FtsZ-producing plasmid <br />                                                                                   (pMT177Z), previously shown reverse filamen- <br />                                                                                   tation RNase E-deficient cells (25) (see Fig. S3 supple- <br />                                                                                   mental material), &#9004;rne deaD::Tn10 bacteria enabled CFA on <br />                                                                                   M9-malate M9-glucose absence amino acids <br />                                                                                   synthesis dependent TCA cycle (Fig. 5). As the <br />                                                                                   genes expressed ftsQAZ operon known con- <br />                                                                                   tribute glycolysis gluconeogenesis, effect FtsZ the <br />                                                                                   nutritional requirements RNase E-deficient cells suggests that <br />                                                                                   additional step glycolytic gluconeogenetic pathway <br />                                                                                   (32, 33) impaired RNase E-deficient bacteria. <br /> FIG 4 Effect PPC complementation CFA &#9004;rne deaD::Tn10 strain                Role degradosome carbon source utilization E. <br /> cultured M9-glucose M9-malate. Cultures MT2114 spread on <br /> M9-glucose 0.5% malate, M9-glucose, M9-malate without <br />                                                                                   coli. The C-terminal half RNase E functions binding site for <br /> 100 &#65533;?&#65533;M IPTG absence arabinose. Plates scanned incuba-        multiple proteins (15); glycolytic pathway <br /> tion 37&#176;C 7 days.                                                          enzyme enolase, converts 2-phosphoglycerate phos- <br />                                                                                   phoenolpyruvate (34), cellular substrate PPC. <br />                                                                                   Interaction RNase E enolase normally required <br /> ows&#65533;? colonies appeared following plating bacteria                propagation E. coli, N-terminal catalytic region of <br /> on M9-malate plates (see Fig. 5), presumed minute                      RNase E (N-Rne), lacks enolase binding site (35), is <br /> colonies did increase size incubation                   sufficient colony formation rne mutant bacteria (14). As <br /> upon replating did result colony formation (data                       mannitol glucose upstream glycolytic pathway <br />  <br />  <br />  <br />  <br /> FIG 5 Effect FtsZ complementation CFA &#9004;rne deaD::Tn10 strain M9-glucose M9-malate. Cultures MT2115 MT2116 spread M9 <br /> plates containing glucose and/or malate, indicated, containing lacking 0.1% arabinose. Plates scanned incubation 37&#176;C 4 days or, where <br /> indicated, 2 additional days incubation. <br />  <br />  <br />  <br /> March 2013 Volume 195 Number 6                                                                                                                    jb.asm.org 1137 <br />  Tamura et al. <br />  <br />  <br />  <br />  <br /> FIG 6 (A B) Growth wild-type E. coli (A) full-length RNase E-complemented &#9004;rne strain (B) indicated carbon sources. Cultures NCM3416 <br /> or MT2103 spread LB M9 plates indicated (10 &#65533;?&#65533;M IPTG MT2103). (C) Effect RNase E degradosome formation growth E. coli on <br /> different carbon sources. Cultures MT4001 spread LB M9 plates containing lacking 0.1% arabinose indicated, 100 &#65533;?&#65533;M IPTG added <br /> as indicated. Plates scanned incubation 37&#176;C 2 days and, indicated, 2 additional days incubation. <br />  <br />  <br />  <br /> step requires enolase (i.e., conversion mannitol pyru-             (pNRNE4)] did form colonies carbon sources <br /> vate) combinations glucose plus pyruvate mannitol plus              utilized effectively full-length RNase E (i.e., fumarate <br /> pyruvate did restore CFA RNase-E deficient cells (Fig. 2),             succinate), colony growth certain carbon <br /> the sugar-related defect CFA RNase E deficiency nec-             sources occurred slowly observed &#9004;rne <br /> essarily independent enolase.                                              cells complemented trans plasmid expressing full-length <br />     As PPC production impaired RNase E-deficient bacteria               RNase E (Fig. 6C Table 2). Thus, degradosome <span id='am-103' about='obo:NCBITaxon_forma' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-104' property="rdfs:label" content="forma" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>forma</span>- <br /> and component RNase E-based degradosome generates                    tion necessary utilization glucose E. coli (Fig. 6C), <br /> the cellular substrate PPC (i.e., phosphoenolpyruvate), hy-            absence scaffold region enzyme affected utiliza- <br /> pothesized presence scaffold region RNase E              tion sugars. It noted overproduction of <br /> affect carbon source utilization E. coli cells. To test           full-length RNase E &#9004;rne strain (MT2103) did enable <br /> notion, wild-type bacteria deaD&#11001; rne deletion mutant strain             formation colonies M9 medium containing 0.2% lactose, <br /> complemented N-Rne full-length Rne tested               wild-type NCM3416 bacteria able form colonies <br /> their ability grow M9 plates supplemented specific sug-            (data shown). The basis effect currently known. <br /> ars. As shown Fig. 6, wild-type bacteria (Fig. 6A)                  Complementation &#9004;rne bacteria different forms of <br /> &#9004;rne strain carrying plasmid expressing full-length Rne (Fig. 6B)           RNase G results differential sugar utilization. Lee et al. re- <br /> formed normal-size colonies tested carbon sources               ported overproduction His-tagged RNase G restores CFA <br /> acetate 2 days culture M9 plates. However, &#9004;rne bacte-            &#9004;rne bacteria LB plates LB liquid medium (6). <br /> ria carrying plasmid expressing N-Rne [i.e., MT4001                         Subsequent investigations shown complementation of <br />  <br />  <br /> TABLE 2 Summary plating experiments <br />                                 Resulta plating on: <br />                                                    M9 with: <br /> Strain                          LB medium          Glucose    Glycerol          Pyruvate       Acetate              Malate      Fumarate         Succinate <br /> NCM3416                         ND                 &#338;          &#338;                 &#338;              o                    &#338;           &#338;                &#338; <br /> MT4001(pNRNE4)                  &#338;                  &#338;          o                 o              &#11003;                    o           &#11003;                &#11003; <br /> MT4002(pRNG2S)                  &#338;                  &#338;          46% CFE, slow     o              37% CFE, slow        &#338;           &#338;                &#338; <br /> MT2105(pRNG2S&#9004;H)                &#338;                  &#11003;          ND                ND             ND                   &#11003;           ND               ND <br /> MT2113 (deaD::Tn10)             o                  &#11003;          ND                o              ND                   &#11003;           ND               ND <br /> a <br />     &#338;, 100% CFE; o, 100% CFE slow growth; &#11003;, growth. <br />  <br />  <br />  <br /> 1138     jb.asm.org                                                                                                                  Journal Bacteriology <br />                                                                                                                                     RNase E Essentiality E. coli <br />  <br />  <br />  <br />  <br /> FIG 7 Growth natural-form RNase G-complemented &#9004;rne strain LB variety carbon sources. (A B) Cultures MT2105 (A) MT2005 (B) <br /> were spread LB M9 plates containing lacking 0.1% arabinose (as indicated), 1 mM IPTG added indicated. Plates scanned incubation <br /> at 37&#176;C 2 days (A) 4 days (B). (C) Growth &#9004;rne strain expressing His-tagged RNase G. Cultures MT4002 spread LB M9 plates containing <br /> or lacking 0.1% arabinose indicated carbon sources, 1 mM IPTG added indicated. Plates scanned incubation 37&#176;C 2 days and, <br /> where indicated, 2 additional days incubation. <br />  <br />  <br />  <br /> rne-defective mutants RNase G facilitated unnatural ex-                (37% CFE) (Fig. 7C). Complementation natural form of <br /> tensions termini RNase G protein, conse-               RNase G observed, LB, N3433-derived <br /> quently concluded RNase G function constrained                   &#9004;rne strain (Fig. 7B), noted differs MG1655 <br /> termini enzyme (24). In instances, mutations                derivatives carbon sources needed CFA, suggesting that <br /> catalytic region RNase G reported facilitate com-               strain differences account disparate results reported in <br /> plementation rne mutants (27). Given information                    investigations complementation RNase E deficiency by <br /> the results described above, interested learning             RNase G overexpression (6, 24, 27, 36, 37). <br /> whether &#9004;rne cells complemented viability overproduction                      Proposed mechanism role glycolytic pathway <br /> of natural versus elongated forms RNase G different                 TCA cycle RNase E essentiality. The results reported here <br /> nutritional requirements.                                                       indicate absence RNase E E. coli viable a <br />     As shown Fig. 7, overproduction natural form                   second-site Tn10 insertion mutation deaD (26) results defec- <br /> RNase G restored CFA introduced MG1655-derived                     tive production PPC, links glycolysis TCA cycle <br /> &#9004;rne strain plated LB medium failed enable CFA                   (Fig. 8) (31). We conclude deficiency enzyme results <br /> cells plated M9 containing tested sugar              inability cells use glycolytic pathway carbon sources for <br /> combination glucose malate (Fig. 7A). In contrast, His-                  synthesis amino acids generated TCA cycle. Conse- <br /> tagged RNase G enabled CFA 100% efficiency, LB                 quently, rne deaD mutant cells failed grow solely on <br /> but M9 medium containing glucose, pyruvate, malate,                     glucose glycolytic pathway intermediates, formed <br /> fumarate, succinate, indicating nutritional require-                colonies minimal medium containing sugars the <br /> ments complementation resemble bacteria com-                  cellular connection glycolytic pathway TCA cycle <br /> plemented full-length RNase E (Fig. 7C). However, relative                enhanced adventitious overexpression cloned ppc gene <br /> cells complemented plasmid-encoded RNase E, CFA occurred                     TCA cycle amino acids added adventitiously. How- <br /> in &#9004;rne cells reduced efficiency His-tagged RNase G com-               ever, defective PPC synthesis nutritional conse- <br /> plementation growth glycerol (46% CFE) acetate                     quence RNase E deficiency, cells overexpressing PPC re- <br />  <br />  <br /> March 2013 Volume 195 Number 6                                                                                                                 jb.asm.org 1139 <br />  Tamura et al. <br />  <br />  <br />  <br />                                                                                  3. Ray A, Apirion D. 1980. Cloning gene ribonuclease E, RNA <br />                                                                                     processing enzyme. Gene 12:87&#8211;94. <br />                                                                                  4. Wachi M, Umitsuki G, Shimizu M, Takada A, Nagai K. 1999. Esche- <br />                                                                                     <span id='am-106' about='obo:NCBITaxon_1430853' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_95179 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_7100 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-107' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-109'  ></span><span id='am-110' property="rdfs:label" content="Richia" datatype="xsd:string"></span>richia</span> coli cafA gene encodes novel RNase, designated RNase G, in- <br />                                                                                     volved processing 5= end 16S rRNA. Biochem. Biophys. Res. <br />                                                                                     Commun. 259:483&#8211; 488. <br />                                                                                  5. Bernstein JA, Khodursky AB, Lin PH, Lin-Chao S, Cohen SN. 2002. <br />                                                                                     Global analysis mRNA decay abundance Escherichia coli sin- <br />                                                                                     gle-gene resolution using two-color fluorescent DNA microarrays. Proc. <br />                                                                                     Natl. Acad. Sci. U. S. A. 99:9697&#8211;9702. <br />                                                                                  6. Lee K, Bernstein JA, Cohen SN. 2002. RNase G complementation rne <br />                                                                                     null mutation identifies functional interrelationships RNase E in <br />                                                                                     Escherichia coli. Mol. Microbiol. 43:1445&#8211;1456. <br />                                                                                  7. Ono M, Kuwano M. 1979. A conditional lethal mutation Escherichia <br />                                                                                     coli strain longer chemical lifetime messenger RNA. J. Mol. Biol. <br />                                                                                     129:343&#8211;357. <br />                                                                                  8. Stead MB, Marshburn S, Mohanty BK, <span id='am-93' about='obo:NCBITaxon_69650' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_57612 obo:NCBITaxon_69649 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-94' property="rdfs:label" content="Mitra" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-97'  ></span>Mitra</span> J, Castillo LP, Ray D, van <br />                                                                                     Bakel H, Hughes TR, Kushner SR. 2011. Analysis Escherichia coli <br /> FIG 8 Proposed model RNase E effects glycolytic pathway TCA          RNase E RNase III activity vivo using tiling microarrays. Nucleic <br /> cycle. PEP, phosphoenolpyruvate.                                                    Acids Res. 39:3188 &#8211;3203. <br />                                                                                  9. Lin-Chao S, Cohen SN. 1991. The rate processing degradation of <br />                                                                                     antisense RNAI regulates replication ColE1-type plasmids vivo. <br /> mained unable form colonies supplied certain                           Cell 65:1233&#8211;1242. <br />                                                                                 10. Lundberg U, Altman S. 1995. Processing precursor catalytic <br /> sugars. Moreover, combination glucose malate did                    RNA subunit RNase P Escherichia coli. RNA 1:327&#8211;334. <br /> not support CFA &#9004;rne cells lack additional mutation               11. Alifano P, Rivellini F, Piscitelli C, Arraiano CM, Bruni CB, Carlomagno <br /> deaD. Thus, RNase E essentiality attributable single gene               MS. 1994. Ribonuclease E provides substrates ribonuclease P-depen- <br /> or pathway.                                                                         dent processing polycistronic mRNA. Genes Dev. 8:3021&#8211;3031. <br />     Whereas model proposed explains bulk                 12. Lee K, Cohen SN. 2003. A <span id='am-7' about='obo:NCBITaxon_1883' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_2062 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_85011'><span id='am-8' property="oboInOwl:hasDbXref" content="PMID:11411701" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="PMID:10843062" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="PMID:16560709" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="PMID:15950131" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasAlternativeId" content="NCBITaxon:31945" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="PMID:1371059" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="PMID:4886644" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Streptomyces" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="PMID:9336904" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Chainia" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="PMID:8590678" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="PMID:9731302" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="PMID:8934906" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1973" datatype="xsd:string"></span><span id='am-23'  ></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="NCBITaxon:65506" datatype="xsd:string"></span>Streptomyces</span> coelicolor functional orthologue of <br />                                                                                     Escherichia coli RNase E shows shuffling catalytic PNPase-binding <br /> findings observed, findings remain unexplained. (i)                    domains. Mol. Microbiol. 48:349 &#8211;360. <br /> A Keio collection E. coli K-12 BW25113 strain deleted ppc                    13. Schein A, Sheffy-Levin S, Glaser F, Schuster G. 2008. The RNase E/G- <br /> (JW3928-1) (38), expected proposed model (see                      type endoribonuclease higher plants located chloroplast and <br /> reference 39), did form colonies M9 medium containing                        cleaves RNA similarly E. coli enzyme. RNA 14:1057&#8211;1068. <br /> either glucose pyruvate sole carbon source. However, rne                14. McDowall KJ, Cohen SN. 1996. The N-terminal domain rne gene <br />                                                                                     product RNase E activity non-overlapping arginine- <br /> deaD doubly mutant bacteria form colonies M9-pyruvate,                       rich RNA-binding site. J. Mol. Biol. 255:349 &#8211;355. <br /> as reported previously (26). This finding consistent             15. Miczak A, Kaberdin VR, Wei CL, Lin-Chao S. 1996. Proteins associated <br /> conclusion inactivation PPC does fully explain                    RNase E multicomponent ribonucleolytic complex. Proc. Natl. <br /> the modified metabolism observed E. coli absence                       Acad. Sci. U. S. A. 93:3865&#8211;3869. <br /> RNase E. (ii) M9 minimal medium containing CAA mixture                     16. Py B, Higgins CF, Krisch HM, Carpousis AJ. 1996. A DEAD-box RNA <br />                                                                                     helicase Escherichia coli RNA degradosome. Nature 381:169 &#8211;172. <br /> the sole carbon source did confer CFA rne deaD mutant                    17. Lopez PJ, Marchand I, Joyce SA, Dreyfus M. 1999. The C-terminal half <br /> cells, contrary expected. As combination pyru-                   RNase E, organizes Escherichia coli degradosome, participates <br /> vate (a glycolytic pathway intermediate) malate (a TCA cycle                    mRNA degradation rRNA processing vivo. Mol. Microbiol. <br /> intermediate) supported CFA (as shown Fig. 2), glycolytic                    33:188 &#8211;199. <br /> pathway (and gluconeogenetic pathway) steps upstream                       18. Ow MC, Liu Q, Kushner SR. 2000. Analysis mRNA decay rRNA <br />                                                                                     processing Escherichia coli absence RNase E-based degrado- <br /> pyruvate principle active cultured rne deaD                     assembly.  <br /> </body></html>