<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The C. elegans H3K27 Demethylase UTX-1 Is Essential for <br /> Normal Development, Independent Its Enzymatic <br /> Activity <br /> Julien Vandamme1,2, Gae&#168;lle Lettier1, Simone Sidoli3, <span id='am-89' about='obo:NCBITaxon_1289997' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_64962 obo:NCBITaxon_74388 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_27818 obo:NCBITaxon_6072'><span id='am-90'  ></span><span id='am-91' property="rdfs:label" content="Elia" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Elia</span> Di Schiavi4, Ole N&#248;rregaard Jensen3, <span id='am-79' about='obo:NCBITaxon_48819' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_214597 obo:NCBITaxon_131567 obo:NCBITaxon_214607 obo:NCBITaxon_3398 obo:NCBITaxon_4143 obo:NCBITaxon_1477611 obo:NCBITaxon_26122 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-80' property="rdfs:label" content="Anna" datatype="xsd:string"></span><span id='am-81'  ></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Anna</span> <br /> Elisabetta Salcini1,2* <br /> 1 Biotech Research Innovation Centre (BRIC), University Copenhagen, Copenhagen, Denmark, 2 Centre Epigenetics, University Copenhagen, Copenhagen, <br /> Denmark, 3 Centre Epigenetics, Department Biochemistry Molecular Biology, University Southern Denmark, Odense, Denmark, 4 Institute Genetics and <br /> Biophysics &#8216;&#8216;Adriano Buzzati Traverso,&#8217;&#8217; Consiglio Nazionale delle Ricerche (CNR), Naples, Italy <br />  <br />  <br />  <br />      Abstract <br />      Epigenetic modifications influence gene expression provide unique mechanism fine-tuning cellular differentiation <br />      development multicellular organisms. Here report biological functions UTX-1, <span id='am-42' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-43' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-46'  ></span>Caenorhabditis</span> <br />      elegans homologue mammalian UTX, histone demethylase specific H3K27me2/3. We demonstrate utx-1 an <br />      essential gene required correct embryonic postembryonic development. Consistent homology to <br />      UTX, UTX-1 regulates global levels H3K27me2/3 C. elegans. Surprisingly, catalytic activity not <br />      required developmental function protein. Biochemical analysis identified UTX-1 component complex <br />      includes SET-16(MLL), genetic analysis indicates defects associated loss UTX-1 likely mediated <br />      compromised SET-16/UTX-1 complex activity. Taken together, results demonstrate UTX-1 required many <br />      aspects nematode development; but, unexpectedly, function independent enzymatic activity. <br />  <br />   Citation: Vandamme J, Lettier G, Sidoli S, Di Schiavi E, N&#248;rregaard Jensen O, et al. (2012) The C. elegans H3K27 Demethylase UTX-1 Is Essential Normal <br />   Development, Independent Its Enzymatic Activity. PLoS Genet 8(5): e1002647. doi:10.1371/journal.pgen.1002647 <br />   Editor: Andrew D. Chisholm, University <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span>California</span> San Diego, United States America <br />   Received August 10, 2011; Accepted February 22, 2012; Published May 3, 2012 <br />   Copyright: &#223; 2012 Vandamme et al. <span id='am-50' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-51'  ></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported grants Danish National Research Council, Program Excellence University Copenhagen, the <br />   Lundbeck foundation. The funders role study design, <span id='am-94' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-95'  ></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: lisa.salcini@bric.ku.dk <br />  <br />  <br />  <br /> Introduction                                                                        H3K27me2/3 [6&#8211;11], individual members shown <br />                                                                                     regulate differentiation cellular systems [6,7,10]. In C. <br />    The proper development multicellular organisms requires                       elegans, KDM6 <span id='am-24' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-25' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> members: jmjd-3.1, jmjd-3.2, <br /> strict regulation cell-specific gene expression ensure                        jmjd-3.3, closely related JMJD3, utx-1, unique <br /> appropriate cell fate specification, cellular differentiation,                  homologue human UTX/UTY. The functional role of <br /> organogenesis. In addition transcription factors, gene expression                proteins <span id='am-73' about='obo:NCBITaxon_333870' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_195229 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41087 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_71193 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-74'  ></span><span id='am-75' property="rdfs:label" content="Nematodes" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasRelatedSynonym" content="Nematodes Berthold, 1827" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>nematodes</span> defined. jmjd-3.1 been <br /> is controlled chromatin organization, regulated                      reported regulate somatic gonadal development [6], utx-1 <br /> chromatin-remodelling factors post-translational modifi-                    implicated vulva differentiation aging [12&#8211;14]. <br /> cations histone proteins [1&#8211;3].                                                     In report, analyzed developmental functions of <br />    An important post-translational modification mono- (me),                  UTX-1. We utx-1 plays vital role embryogen- <br /> di- (me2), tri- (me3) methylation lysine residues (K) tail             esis acts aspects nematode postembryonic <br /> of histone 3 (H3). Specifically, methylation specific lysine                 development. Surprisingly, catalytic activity of <br /> residues plays major role maintenance active silent                 UTX-1 critical importance UTX-1 function in <br /> gene expression states. The combination H3 K4, K36, K79                      development. Genetic biochemical analyses indicate that <br /> tri-methylation generally marks transcriptionally active regions,                   UTX-1 acts SET-16(MLL)/UTX-1 complex that <br /> whereas H3 K9 K27 tri-methylation marks regions                              primary role UTX-1 resides regulation activity <br /> transcriptionally silenced genes [2]. The levels methylation                     complex. <br /> are modulated action histone methyltransferases (HMTs) <br /> and histone demethylases (HDMs). The largest group histone                       Results <br /> demethylases contains Jumonji C-domain (JmjC) catalyzes <br /> the demethylation specific lysine arginine residues                    Loss utx-1 results reduced fertility lethality <br /> oxidative reaction requiring iron [Fe(II)] a-ketoglutarate                        C. elegans D2021.1 encodes predicted protein 134 kDa <br /> (aKG) cofactors [4]. There 28 JmjC-containing proteins                    high homology co-linearity mammalian <br /> humans, grouped different families, majority                    UTX/UTY proteins (Figure 1A); named gene its <br /> are evolutionarily conserved [5]. The KDM6 <span id='am-27' about='obo:NCBITaxon_subfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-28' property="rdfs:label" content="subfamily" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>subfamily</span> (UTX/                          product utx-1 UTX-1, respectively. UTX-1 expressed in <br /> UTY/JMJD3) shown catalyze demethylation                               most, all, nuclei early late stage embryos (Figure 1B) <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                     1                              May 2012 | Volume 8 | Issue 5 | e1002647 <br />                                                                                                         Roles UTX-1 C. elegans Development <br />  <br />  <br />  <br />   Author Summary                                                           correctly established utx-1(tm3118) mutant worms (Figures <br />                                                                            S2, S3, S4, S5). However, progressive loss myo-3::GFP <br />   Chromatin organization influences gene expression,                   transgene expression embryogenesis observed little <br />   regulation crucial achieve correct cellular differen-          GFP signal detected L1 escapers (Figure S6), suggesting that <br />   tiation development multicellular organisms.                      defects muscle function account, part, the <br />   Histone demethylases factors respon-                   lethality utx-1 null animals (Figure 2B). <br />   sible regulating chromatin dynamics. Here report <br />   biological functions UTX-1, C. elegans <br />   homologue mammalian histone demethylase UTX, <br />                                                                            Reduction utx-1 results posterior defects in <br />   specifically catalyzes demethylation di-                aberrant gonad migration organization <br />   tri-methylated lysine 27 histone H3 (H3K27me2/3).                        To determine function UTX-1 later developmental <br />   Indeed, demonstrate UTX-1 regulates global levels                stages, analyzed utx-1(m+/z2) mutant worms, in <br />   H3K27me2/3 C. elegans, mark generally associated                 zygotic contribution UTX-1 lost, wild-type <br />   silencing gene expression. We utx-1               worms utx-1 expression downregulated feeding <br />   essential gene required correct embryonic              RNA interference (RNAi). RNA interference, constructs <br />   postembryonic development. Specifically, loss                 targeted different regions utx-1 (Figure 1A), resulted in <br />   utx-1 results developmental defects, sterility,                   approximately 60% reduction utx-1 mRNA F1 progeny <br />   embryonic lethality. Surprisingly, data                significant reduction UTX-1 protein expression (Figure <br />   catalytic activity UTX-1 required develop-             S7A). In agreement phenotype utx-1(m+/z2) mutant <br />   mental functions. Our biochemical genetic analyses                   animals, utx-1(RNAi) F1 animals reduced fertility (Figure <br />   indicate loss UTX-1 compromises activity the <br />                                                                            S7B). Furthermore, variable defects, located posteriorly, <br />   SET-16(MLL) complex, UTX-1 integral of. <br />   Taken together, results demonstrate UTX-1                     observed 40% utx-1(RNAi) worms (Figure 2A <br />   plays essential role development independent                2C). The posterior defects observed animals treated with <br />   enzymatic activity.                                                      utx-1(RNAi) similar defects observed utx-1(m2/ <br />                                                                            z2) dead larvae (Figure 2A, 2C Figure S8). This demonstrates <br />                                                                            RNA interference used efficiently analyze the <br /> as larval stages adulthood              postembryonic roles UTX-1, posterior phenotype <br /> (Figure 1C), suggesting UTX-1 functional role            utx-1(m2/z2) loss utx-1. Importantly, transgenic <br /> throughout C. elegans development. To determine biological             expression wild-type utx-1 fully rescued posterior defects in <br /> function UTX-1 deletion mutant strains, utx-1(tm3136)           larvae. <br /> utx-1(tm3118), analyzed (Figure 1A). The tm3136 allele              The reduced fertility observed utx-1(m+/z2) animals might <br /> 236 bp deletion creates premature stop <span id='am-60' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-64'  ></span>codon</span>, potentially           regulatory role UTX-1 somatic gonad or <br /> encoding truncated protein 28 amino acids,              germline development. Homozygous mutant utx-1(m+/z2) ani- <br /> likely producing null mutant. The tm3118 allele out&#8211;of-            mals heterozygous animals generally develop germlines with <br /> frame deletion 547 bp. The truncated protein potentially retains        correct proliferation differentiation patterns (not shown), as <br /> the 620 amino acids, lacking JmjC domain              demonstrated fact oocytes formed (Figure 2B) and <br /> catalytic activity. The alleles similar phenotypes                ability lay dead embryos, suggesting sterility <br /> suggesting loss function mutants.                    related germline defect. However, animals lacking <br />    Homozygous utx-1 mutant worms derived                     UTX-1 activity defects gonad migration oocyte <br /> heterozygous mothers providing maternal UTX-1, utx-1(m+/                   organization. The shape gonad dictated the <br /> z2), viable reach adulthood. However, produce                 coordinated migration distal tip cells (DTCs), are <br /> only few, unviable, utx-1(m2/z2) eggs (Figure 1D              somatic gonad structure away the <br /> 1E), suggesting UTX-1 required embryogenesis          gonad primordium postembryonic development, leading to <br /> the lack UTX-1 overcome maternal contribution.                consecutive turns forming U-shaped gonad arms observed <br /> Analysis dead embryos revealed mutant utx-1 animals            adult animals. Using transgenic animals carrying distal tip cell <br /> mainly arrested late embryos (Figure 1E). Dead L1 larvae,          marker, lag-2::GFP [16], observed aberrant gonadal migration <br /> misshapen bodies (Figure 1E), rarely observed (5%, n.200).            42% (n = 176) utx-1(RNAi) animals. Morphological <br /> A putx-1::UTX-1::GFP (UTX-1::GFP) translational reporter                analysis DIC utx-1(RNAi) animals confirmed that <br /> extra-chromosomal array able rescue embryonic lethal            48% (n = 215) animals showed failure turn abnormal <br /> phenotype observed heterozygous utx-1(m2/z2) progeny               turning gonad arm (Figure 2B 2C), these <br /> mothers carrying tm3136 tm3118 allele (Figure 1D)         animals (41%, n = 137) developed misshapen gonads, with <br /> several transgenic lines, leading conclude UTX-1             enlargement proximal end gonad arms a <br /> essential embryogenesis zygotic expression             misorganization oocytes (Figure 2B). These gonad phenotypes <br /> UTX-1 sufficient restore embryonic viability. Indeed, progeny        identified utx-1(m+/z2) mutant animals (Figure 2B, <br /> that did receive transgene mother, died late           2C Figure S9), efficiently rescued UTX- <br /> stage embryos (not shown) malformed L1 larvae (Figure S1A),             1::GFP transgene, reinforcing aberrations caused by <br /> suggesting UTX-1 required early embryogen-            loss utx-1 (Figure 2C). The fact transgenic <br /> esis. In agreement this, analysis epithelial junctions using       expression wild-type utx-1 able rescue sterility and <br /> an AJM-1::GFP translational reporter [15] suggests                gonadal phenotypes suggests utx-1 role the <br /> morphology utx-1(m2/z2) embryos, did inherit               somatic gonad germline, transgenes are <br /> transgene, normal early stages progressively deterio-           normally silenced. Consistent this, GFP-tagged UTX-1 is <br /> rated development (Figure S1B). Analysis markers             expressed distal tip cells migration (Figure 1C) and <br /> for intestinal (elt-2::GFP), muscular (hlh-1::GFP myo-2::GFP),         tissues somatic gonad, sheath cells the <br /> and hypodermal (dpy-7::GFP) cells revealed cell lineages        spermatheca (not shown) germline. <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                             2                            May 2012 | Volume 8 | Issue 5 | e1002647 <br />                                                                                                            Roles UTX-1 C. elegans Development <br />  <br />  <br />  <br />  <br /> Figure 1. UTX-1 expression utx-1 embryonic phenotypes. (A) Top: Human UTX C. elegans homologue UTX-1. TPRs, <br /> tetratricopeptide repeats; JmjC, Jumonji C domain. Bottom: Genomic organization utx-1. Black H-shaped lines indicate position tm3136 <br /> and tm3118 deletions. Black lines indicate position different RNAi fragments (a, b, c) used study. (B) UTX-1 expression during <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              3                             May 2012 | Volume 8 | Issue 5 | e1002647 <br />                                                                                                             Roles UTX-1 C. elegans Development <br />  <br />  <br />  <br /> embryogenesis analyzed immunostaining anti-UTX-1 antibody (top panel, right) epifluorescence (bottom panel, right). DAPI <br /> staining Nomarski (DIC) images shown left. (C) UTX-1 expression epifluorescence (right panels) larval development. <br /> Nomarski (DIC) images shown left panels. Asterisks indicate distal tip cell, arrow head forming vulva. Animals oriented anterior <br /> to left. (D) Brood size wild type, utx-1 mutant worms rescued utx-1 lines. Progeny given average number viable progeny per <br /> worm 6 SD. The number laid, hatched, eggs counted utx-1 (m+/z2) mutants reported lower table. utx-1+UTX-1::GFP utx-1+UTX- <br /> 1DD::GFP, utx-1 transgenic lines expressing wild-type catalytically inactive mutant UTX-1, respectively, extrachromosomal arrays. (E) <br /> Representative Nomarski (DIC) images utx-1(tm 3136) mutant embryo escaper L1 larva. Similar phenotypes observed utx-1(tm 3118) <br /> (not shown). Bars B E 20 mm, C 50 mm. Animals oriented anterior left. <br /> doi:10.1371/journal.pgen.1002647.g001 <br />  <br />  <br />    The aberrant migration oocyte organization defects                 UTX-1 enzymatic activity. Tail gonadal defects also <br /> similar reported jmjd-3.1 loss-of-function mutants,              efficiently rescued (Figure 2C) 50% (4/8) transgenic lines, <br /> which encodes C. elegans homologues JMJD3                   indicating UTX-1, catalytic activity, required for <br /> family [6]. To determine link                 correct posterior gonadal development. <br /> observations, tested UTX-1 affected expression jmjd- <br /> 3.1 performing quantitative PCR utx-1(RNAi) animals. As                 C. elegans KDM6 family members compensate for <br /> shown Figure 2D, utx-1(RNAi) animals reduced levels                reduced H3K27me2/3 demethylase activity utx-1 <br /> jmjd-3.1, suggesting UTX-1 may, directly indirectly, <br /> regulate jmjd-3.1 expression. Additionally, enhancement of <br />                                                                               mutant worms <br /> the phenotype observed utx-1 reduced jmjd-                 jmjd-3.1, jmjd-3.2, jmjd-3.3 (Figure 4A) C. elegans KDM6 <br /> 3.1 mutant genetic background (see below), suggesting               family members closely related human JMJD3. Animals <br /> genes acting genetic pathway regulate somatic              carrying mutations genes viable, fertile (not <br /> gonadal development.                                                          shown), up-regulated levels H3K27me3 by <br />                                                                               western blot analysis (Figure 4B Figure S11B). However, triple <br />                                                                               mutant worms carrying deletions JMJD3-like genes <br /> UTX-1 demethylates H3K27me2/3, catalytic <br />                                                                               showed increased global levels H3K27me3 (Figure 4B and <br /> activity required proper development                               Figure S11B), suggesting proteins H3K27me3 <br />    UTX-1 belongs KDM6 family, members                    demethylases act redundantly UTX-1. Several <br /> been shown catalyze demethylation H3K27me3                      lines evidences indicate JMJD3-like genes not <br /> H3K27me2 [17]. Several observations indicate role                function redundantly UTX-1. Analysis transcriptional <br /> conserved C. elegans life cycle. First, loss the <br />                                                                               expression levels JMJD3-like genes wild-type worms <br /> zygotic maternal contributions utx-1 results increased <br />                                                                               indicated jmjd-3.1 expressed levels comparable to <br /> global levels H3K27me2/3 embryonic stage (Figure 3A). <br />                                                                               utx-1, jmjd-3.2 jmjd-3.3 weakly expressed, in <br /> Second, reduction UTX-1 RNA interference results a <br />                                                                               particular larval stages (Figure S11A). Furthermore, the <br /> significant increase H3K27me3 levels different larval stages <br />                                                                               transcriptional expression pattern JMJD3-like genes <br /> (data shown, [12,18]). Third, exogenous expression wild- <br />                                                                               appeared generally restricted specific tissues or, case <br /> type UTX-1 utx-1 null animals restores H3K27me3 wild-type <br />                                                                               jmjd-3.2, cells (Figure S11C); contrast to <br /> level (Figure 3A). Fourth, over-expression UTX-1 wild-type <br />                                                                               broad expression pattern UTX-1. In addition, triple <br /> animals results significant reduction global H3K27me3 <br /> levels (Figure 3B). Finally, decreased level H3K27me3                  mutant lacking JMJD3-like genes viable fertile, no <br /> observed N2 worms overexpressing UTX-1 (Figure 3B)                 defects posterior region body (Figure 4C) with <br /> correlated degree UTX-1::GFP overexpression,                   minor gonadal defects (Figure 4C), likely the <br /> shown Figure S10.                                                          absence jmjd-3.1. Importantly, down-regulation utx-1 by <br />    Next, tested catalytic activity UTX-1 responsible          RNA interference triple mutant genetic background did not <br /> for phenotype observed utx-1 null mutants utx-1(RNAi)              exacerbate posterior gonadal defects associated utx- <br /> animals. To end, expressed utx-1 mutants GFP-tagged              1 reduction wild-type animals (Figure 4C). Taken together, <br /> mutated form UTX-1 protein (for simplicity called UTX-                 results imply members KDM6 <span id='am-47' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-48' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> act <br /> 1DD::GFP, DD = Demethylase Dead), carrying mutations            redundantly. <br /> three conserved amino acids iron-binding motif (HXD/                      However, light unexpected results obtained the <br /> EXnH) JmjC-domain (indicated asterisks Figure 3C).               catalytically inactive UTX-1 mutant, important into <br /> Several reports shown amino acids required            consideration possibility JMJD3-like genes could, never- <br /> iron binding catalytic activity JmjC-domain           theless, compensate lack UTX-1 activity utx-1 mutant <br /> containing demethylases characterized far [6&#8211;11,19&#8211;21]. All                worms expressing catalytically inactive form UTX-1. In this <br /> UTX-1::DD::GFP transgenic lines generated (8/8) showed expres-                case, expect loss reduction other <br /> sion levels similar wild-type UTX-1 (Figure 3C)                H3K27me3 demethylases utx-1 mutant rescued the <br /> fertile able produce viable progeny (Figure 1D). Impor-                catalytically inactive UTX-1 result utx-1-specific abnor- <br /> tantly, re-expression catalytically inactive UTX-1 did                 malities (posterior defects aberrant gonadal migration). To test <br /> restore wild-type level H3K27me3 utx-1 null animals                 hypothesis, generated triple mutant jmjd-3.2; jmjd-3.3;utx- <br /> (Figure 3A) did influence H3K27me3 level                     1+UTX-1DD::GFP fourth member KDM6 <br /> overexpressed wild-type animals (Figure 3B), confirming               family, jmjd-3.1, down-regulated RNA interference. In this <br /> that amino acids substitutions affected UTX-1 enzymatic                   genetic background, posterior defects observed the <br /> activity. This unexpected result strongly indicates                  degree gonadal defects similar observed wild- <br /> demethylase activity UTX-1 important                     type animals conditions (Figure 4D). Furthermore, <br /> embryonic development animal viability. Subsequently,                   quantitative PCR showed increased expression levels the <br /> tested observable phenotypes dependent                   JMJD3-like genes rescued transgenic line utx-1+UTX- <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                               4                             May 2012 | Volume 8 | Issue 5 | e1002647 <br />                                               Roles UTX-1 C. elegans Development <br />  <br />  <br />  <br />  <br /> PLoS Genetics | www.plosgenetics.org   5   May 2012 | Volume 8 | Issue 5 | e1002647 <br />                                                                                                             Roles UTX-1 C. elegans Development <br />  <br />  <br />  <br /> Figure 2. utx-1 postembryonic phenotypes. (A) Representative DIC images L1 larvae tails (upper panel) L1 larvae (lower panel) N2, utx- <br /> 1(RNAi) utx-1(tm3118) animals. Animals oriented anterior left. Scale bar 50 mm. (B) Representative DIC images gonads adults N2, <br /> utx-1(RNAi) utx-1(tm3118) animals. Scale bar 25 mm. In upper panels, blacks lines indicate migration gonad arm. In lower <br /> panels, black lines indicate contours oocytes. Animals oriented anterior left. (C) Percentages posterior (% tail defects) and <br /> gonad (% gonadal defects) defects indicated strains shown. For RNAi, F1 larvae adults independent experiments <br /> were analyzed. (D) utx-1 jmjd-3.1 mRNA levels embryos control (black bars) utx-1 (white bars) RNAi treatment measured qPCR, <br /> using rpl-26 mRNA internal control. *P,0.01 (Student&#8217;s t-test). <br /> doi:10.1371/journal.pgen.1002647.g002 <br />  <br /> 1DD::GFP compared utx-1+UTX-1::GFP (Figure 4E). These                       core components complex (F21H12.1, wdr-5.1, and <br /> results, fact H3K27me3 levels up-                  ash-2) resulted posterior gonadal defects similar the <br /> regulated utx-1 expressing catalytically inactive form               ones observed utx-1 mutants (Figure 6C Figure S9). It <br /> UTX-1 (Figure 3A), strongly indicate JMJD3-like proteins              noted, enlargement proximal gonad was <br /> do compensate lack UTX-1 catalytic activity                 observed reduction RNAi F21H12.1 ash- <br /> that catalytic activity UTX-1 required proper                2 rarely observed wdr-5 (RNAi) animals (Figure S9). We <br /> development.                                                                   tested effects simultaneously downregulating specific <br />                                                                                components complex RNAi. As shown Figure 6B, the <br /> UTX-1 acts SET-16/MLL complex                                           concurrent knockdown utx-1 set-16 pis-1 did enhance <br />    The mammalian UTX MLL3/MLL4 H3K4me3                          phenotypes; similar results obtained concomitant <br /> methyltransferase complex [22&#8211;24] includes specific              silencing pis-1 set-16. The high degree phenotypic <br /> component PTIP, WDR5, ASH2L, RbBP5 core                             similarity absence redundancy evidence these <br /> components, shared complexes [25]. The                 genes acting genetic pathways regulate posterior <br /> high conservation proteins nematodes (WDR5/tag-                    patterning somatic gonadal development. Along line, <br /> 125/wdr-5.1, ASH2L/ash-2, RbBP5/F21H12.1/rbbp-5, MLL3-4/                       qPCR analysis revealed set-16 downregulation RNA <br /> set-16, UTX/utx-1, PTIP/pis-1), suggests similar                    interference results reduction jmjd-3.1 mRNA (about 60% <br /> complex exist C. elegans. To test MLL3-4/                  decrease compared control RNAi, data shown), further <br /> UTX-like complex (SET-16/UTX-1) present C. elegans,                   supporting notion UTX-1 SET-16 act same <br /> purified GFP-tagged UTX-1 associated proteins mixed                 complex. <br /> population transgenic animals, enriched embryos                           Since catalytic activity UTX-1 necessary rescue <br /> (Figure 5A). The identities interacting proteins                   developmental defects observed utx-1 mutants in <br /> determined mass spectrometry listed Table S1.                animals different factors complex lost or <br /> As control, N2 lysates subject procedure                down-regulated, hypothesized UTX-1 regulate the <br /> the recovered proteins (listed Table S2) considered                    expression components complex. In support this, <br /> contaminants used confirm specificity identified             levels set-16 mRNA reduced utx- <br />                                                                                1(RNAi) animals (Figure 6D). Interestingly, downregulation set- <br /> interacting proteins. Strikingly, homologous components of <br />                                                                                16 results decreased expression utx-1 mRNA protein <br /> the mammalian MLL3/4 complex identified UTX-1 <br />                                                                                (Figure 6D 6E), suggesting interdependent regulation of, at <br /> partners C. elegans (Figure 5B). As verification, utilized <br />                                                                                least, members complex. <br /> a transgenic line carrying HA-tagged WDR-5.1 [26], most <br />                                                                                   Taken data demonstrate SET-16/UTX-1 <br /> prominent WDR5-like protein recovered mass spectrometry, in <br />                                                                                complex present C. elegans, required development. <br /> which expressed UTX-1::GFP. As shown Figure 5C, in <br />                                                                                That loss downregulation single components the <br /> lysates derived embryos, UTX-1::GFP endogenous <br />                                                                                complex results similar phenotypes observed utx-1 <br /> UTX-1 associated WDR5.1, supporting <br />                                                                                null mutants, indicates component required the <br /> the existence SET-16/UTX-1 complex C. elegans. <br />                                                                                complex function normally defects associated with <br /> Importantly, catalytically inactive mutant UTX-1DD::GFP <br />                                                                                loss UTX-1 likely result compromised SET-16/ <br /> was recovered WDR-5.1 immunoprecipitation (Figure 5C). <br />                                                                                UTX-1 complex activity. <br /> Gel filtration analysis lysates transgenic lines carrying <br /> either wild-type catalytically inactive forms UTX-1 <br /> further confirmed UTX-1 UTX-1DD engaged                      Discussion <br /> in large complexes (Figure 5D), supporting                         We demonstrated C. elegans UTX-1 H3K27me2/ <br /> functional JmjC domain required association                3 demethylase essential development embryonic <br /> the complex.                                                                   larval stages nematode, independently its <br />    We verified functional correlation SET-16/                  demethylase activity. Animals lacking maternal zygotic <br /> UTX-1 complex components testing loss downreg-                  contribution UTX-1 arrest late embryogenesis. Al- <br /> ulation result phenotypes similar observed            though, analyses reporter genes revealed major defects in <br /> utx-1 mutant. Loss set-16 results embryonic early larval             lineage specifications, reduction myo-3::GFP, expression, but <br /> lethality [27]. The analysis set-16(n4526) young larvae revealed            hlh-1::GFP, observed utx-1 mutant animals, suggesting <br /> the presence posterior defects similar identified utx-1         utx-1 regulate genes involved muscle function. In <br /> null animals (Figure 6A Figure S8), set-16(RNAi) animals               agreement, mammalian UTX implicated terminal <br /> that escaped embryonic early larval lethality,                   differentiation muscle cells [28]. The maternal contribution of <br /> abnormal gonad migration enlargement (Figure 6A, 6B                    UTX-1 allows utx-1(m+/z2) worms reach adulthood, but <br /> Figure S9), phenocopied effect loss utx-1.                 defects arise different stages development, including <br /> Similarly, pis-1(ok3720) mutants pis-1(RNAi) animals,                   abnormal gonad migration oocyte misorganization. This <br /> posterior gonadal defects observed, lower               phenotype explain, part, reduced fertility <br /> degree (Figure 6A 6B, Figures S8 S9). RNA interference                 utx-1(m+/z2) animals. We previously shown proper <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                6                            May 2012 | Volume 8 | Issue 5 | e1002647 <br />                                               Roles UTX-1 C. elegans Development <br />  <br />  <br />  <br />  <br /> PLoS Genetics | www.plosgenetics.org   7   May 2012 | Volume 8 | Issue 5 | e1002647 <br />                                                                                                              Roles UTX-1 C. elegans Development <br />  <br />  <br />  <br /> Figure 3. UTX-1 H3K27me2/3 demethylase. (A) Protein lysates embryos indicated strains probed antibodies against <br /> H3K27me3 H3K27me2. H3 used loading control. Quantification western blot shown graphic right. Error bars indicate <br /> the standard deviation calculated using 2 replicates. The signals quantified using ImageJ software normalized H3. The values are <br /> relative N2 levels. Similar results obtained different transgenic lines utx-1 genetic backgrounds (tm3118 and <br /> tm3136). (B) Representative images N2 expressing translational construct wild-type (N2+UTX-1::GFP) catalytically inactive UTX-1 (N2+UTX- <br /> 1DD::GFP) GFP fusion GFP-negative siblings, fixed stained H3K27me3 antibody DAPI. The white square encloses intestinal cell, <br /> used H3K27me3 quantification. Enlargement white square shown the panel. Quantification H3K27me3 levels is <br /> shown graphic right. At 25 cells genotype quantified described Materials Methods. Mean fluorescence + <br /> s.e.m. values independent experiments displayed. *P,0.01. (Student&#8217;s t-test). Animals oriented anterior left. (C) Top: Alignment <br /> of Jumonji C domain human UTX UTX-1 catalytically inactive UTX-1DD (DD = Demethylase Dead). Asterisks indicate <br /> two conserved amino acids iron-binding domain (HXD/EXnH) JmjC-domain, modified UTX-1DD. Bottom: <br /> Epifluorescence utx-1 mutant animals, carrying translational GFP fusion wild-type UTX-1 (utx-1+UTX-1::GFP) catalytically inactive UTX-1 (utx- <br /> 1+UTX-1DD::GFP). Anterior parts animals shown, anterior left. On right, lysates L1 carrying transgenes were <br /> probed GFP antibody. Actin used loading control. The signal quantified using ImageJ program normalized actin. <br /> doi:10.1371/journal.pgen.1002647.g003 <br />  <br />  <br /> gonad migration partly depends H3K27me3 demethy-                       The demonstration UTX, mediates H3K27me2/3 <br /> lase, jmjd-3.1 [6]. The expression level jmjd-3.1 significantly          demethylase activity, MLL3/4 complex, also <br /> reduced utx-1(RNAi) animals. However, noted               H3K4 methyltransferase activity [6,7], suggests model in <br /> the loss utx-1 leads severe phenotype loss            coordinated removal repressive marks (H3K27me3) <br /> jmjd-3.1, influences gonadal processes high                      deposition activating marks (H3K4me3) fine-tune <br /> temperature moderately reduces fertility. These results suggest            transcription differentiation. We shown similar <br /> that utx-1, addition jmjd-3.1, modulate additional genes                 complex present C. elegans, required achieve <br /> required establishing correct developmental program                 proper development. Indeed, loss reduction component <br /> gonads.                                                                        complex results phenotypes similar observed <br />    While utx-1 represents unique UTX/UTY homologue, C.                     utx-1 mutants. The lack synergistic effects double RNAi <br /> elegans genes homology single                      experiments supports notion components of <br /> mammalian JMJD3 gene (jmjd-3.1, jmjd-3.2 jmjd-3.3). We                     complex act pathway(s) regulate posterior body <br /> generated mutant animals carrying mutations JMJD3-                somatic gonad development. Surprisingly, utx-1 phenotypes <br /> like genes and, unexpectedly, did detect additional                 rescued catalytically inactive UTX-1. The catalytically <br /> phenotypes triple mutants, phenotypes                    inactive mutant binds WDR-5.1 similarly wild-type protein, <br /> already reported jmjd-3.1 [6]. While possible residual          identified gel filtration experiments large <br /> gene function remains mutants, global level                    complex, similarly wild-type counterpart. WDR-5.1 a <br /> H3K27me3 significantly increased triple knockout                    component complexes exclude time <br /> worms, increase observed jmjd-3.1 mutant                 UTX-1/WDR-5.1 interaction place the <br /> strain ([6]; Figure 4B). This data suggests JMJD3-              context complex. However, components other <br /> like demethylases regulate expression restricted sets          complexes WDR-5.1 involved have, far, not <br /> genes overlapping functions. Our analysis             recovered mass spectrometry analysis. For example <br /> global levels H3K27me2/3 suggests UTX-1               did identify known WDR-5.1 binding partner SET-2 <br /> important demethylase removal H3K27me3 mark                     (the main H3K4me3 methyltransferase C. elegans) [26,31,32], <br /> among members KDM6 family. Accordingly, loss                 suggesting UTX-1 specifically recruited SET- <br /> utx-1 results sterility (in m+/z2 worms) embryonic                   16(MLL)-like complex. <br /> lethality (in m2/z2 worms) animals lacking                        Taken results strongly suggest UTX-1 acts <br /> JMJD3 homologues fertile viable. This result indicates            SET-16/UTX-1 complex indicate primary <br /> utx-1 plays unique essential roles embryonic                    role UTX-1 C. elegans development independent the <br /> postembryonic development suggests JMJD3-like                     demethylase activity, possibly regulation expression <br /> proteins, like human homologues [10,29], mainly required               complex components. This suggested results <br /> for regulating cellular responses particular conditions,            showing UTX-1 is, least, required proper expression <br /> as stress aging.                                                            set-16, SET-16 required expression utx-1, <br />    Strikingly, catalytic activity C. elegans UTX-         suggesting positive feed forward mechanism retaining the <br /> 1 required function protein                       activity SET-16/UTX-1 complex. It possible there <br /> developmental processes analyzed. This odds previous              additional functions UTX-1; UTX-1 required for <br /> report describing role utx1 genes D. rerio, human           targeting complex specific genomic regions play <br /> wild-type, catalytically inactive mutant, partially                role stability complex. To correctly address these <br /> rescued defects UTX morphant animals [9]. We                     possibilities, chromatin immunoprecipitation mass spectrom- <br /> know apparent dissimilarity organismal                    etry analysis performed context utx-1 null <br /> difference, suggested fact C. elegans UTX-1 does            mutants. Unfortunately, experiments currently unfeasible <br /> regulate HOX genes (data shown) does zebrafish [9]                utx-1 mutant unviable. It mentioned, <br /> and zebrafish UTX homologues, different                 however, downregulation human UTX does not <br /> experimental approaches. Interestingly, recent results suggest            interfere MLL complex formation (Agger K., Helin K., <br /> a catalytic-independent role human JMJD3 UTX                        unpublished data), mammals. <br /> chromatin remodeling subset T-box target genes [30].                      Finally, know UTX-1 works exclusively in <br /> Quantitative PCR analysis reporter genes failed, however,               association SET-16 complex additional roles <br /> to identify regulation selected C. elegans T-box genes               single protein association complexes. The <br /> UTX-1 (not shown).                                                             results obtained mass spectrometry analysis suggest this <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                8                             May 2012 | Volume 8 | Issue 5 | e1002647 <br />                                                                                                          Roles UTX-1 C. elegans Development <br />  <br />  <br />  <br />  <br /> Figure 4. KDM6 family C. elegans. (A) Phylogenetic cluster human UTX, UTY, JMJD3 homologous proteins C. elegans. TPRs, <br /> tetratricopeptide repeats; JmjC, Jumonji C domain. (B) Protein lysates eggs indicated strains analyzed western blot using the <br /> indicated antibodies. H3 used loading control. Quantification western blot shown Figure S11. (C) Percentage tail gonad <br /> defects observed N2 triple mutant (jmjd-3.1;jmjd-3.2;jmjd-3.3) treatment control utx-1(RNAi). F1 animals two <br /> independent experiments scored. (D) Percentage tail gonad defects observed N2 triple mutant (jmjd-3.2; jmjd-3.3;utx-1+UTX- <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                             9                            May 2012 | Volume 8 | Issue 5 | e1002647 <br />                                                                                                           Roles UTX-1 C. elegans Development <br />  <br />  <br />  <br /> 1DD::GFP) treatment control jmjd-3.1(RNAi). F1 animals independent experiments scored. (E) Levels expression <br /> of JMJD3-like genes eggs derived utx-1(tm3136) mutant strain rescued catalytically inactive utx-1 (utx-1+UTX-1DD::GFP) <br /> relative utx-1(tm3136) rescued utx-1 wild-type (utx-1+UTX-1::GFP). rpl-26 mRNA used internal control normalization. <br /> doi:10.1371/journal.pgen.1002647.g004 <br />  <br /> latter hypothesis correct. UTX-1 immunoprecipitates                 1(ok3720)IV, AZ217(myo-2::GFP), MS438(elt-2::GFP), GS3798(dpy- <br /> with proteins involved distinct chromatin complexes,           7::YFP), OP64(hlh-1::GFP), PS3729(ajm-1::GFP). The strain wdr- <br /> as HDA-1 LIN-53, components NuRD complex [33],                    5.1/tag-125::HA OE4201(myo-3::GFP) generous gifts from <br /> and MIX-1, functions dosage compensation complex                Francesca Palladino Thomas Bu&#168;rglin, respectively. Transgen- <br /> (DCC) [34]. While interactions await validation,         ic animals specific genetic backgrounds generated by <br /> worth noting elements NuRD complex                     standard crossing procedure. The C. elegans utx-1 sequence is <br /> involved vulva formation [33], postembryonic event             located chromosome X transcript encompasses 14 <br /> both UTX-1 SET-16 implicated ([12] data                exons coding predicted protein 1168 amino acids. The <br /> shown), DCC recently shown interact                 ATG gene located position 13888 bp D2021 <br /> with ASH-2, member complex here.                   cosmid (U23513) TAG terminator codon position <br /> Therefore, conceivable UTX-1 works diverse                     19549 bp. Two alleles utx-1 identified National <br /> chromatin complexes accomplish functions required different            BioResource Project (NBRP), Japan. Both alleles backcrossed <br /> stages development specific conditions. A MLL                    times N2 phenotypic analysis and <br /> complex-independent role UTX supported fact               maintained culture heterozygotes. The tm3136 allele lacks <br /> that substantial mammalian UTX bound                       236 bp deletion position 13920&#8211;14155 bp of <br /> promoter regions depleted H3K4 methyl marks [35].                         Genbank entry U23513. This deletion creates premature <br />    We did detect reduced global levels H3K4me3 utx-1               stop codon deleted gene potentially encode a <br /> mutant animals (data shown), expected                  truncated protein 28 amino acids. The tm3118 out&#8211;of- <br /> UTX-1 regulates function complex having H3K4                        frame deletion 547 bp situated position 17361&#8211;17907 bp of <br /> methyltransferase activity. This agreement previously             Genbank entry U23513. The truncated protein potentially <br /> reported result mammals C. elegans [7,12] consistent        retains 620 amino acids lacks JmjC domain. <br /> with fact inactivation downregulation set-16             Phenotypic analyses utx-1 mutant animals (tm3136 tm3118) <br /> results minor, any, reduction global levels H3K4me3            blind, genotyping. jmjd-3.2(tm3121)X jmjd- <br /> [12,35,36]. Indeed, similar mammals, H3K4me3 deposition                3.3(tm3197)X backcrossed times analysis and <br /> C. elegans mainly regulated H3K4me3 methyltrans-             deletions, described Wormbase, confirmed by <br /> ferase, set-2 [26,36]. This observation suggests SET-16/            sequencing. KDM6 members positioned X- <br /> UTX-1 complex regulates mark deposition subset             chromosome, utx-1 located closely jmjd-3.1, thus <br /> genes, and, consequently, complex impairment does impact                 precluding generation quadruple mutant lacking the <br /> the global levels H3K4me3.                                                   H3K27me3 demethylases. The triple mutant deletions in <br />    Our analysis failed uncover role UTX-1 catalytic                 jmjd-3.1(gk384)X, jmjd-3.3(tm3197)X jmjd-3.2(tm3121)X was <br /> activity development, major question               generated using standard crossing methods. The triple mutant <br /> whether activity required biological function C.          jmjd-3.2(tm3121);jmjd-3.3(tm3197);utx-1(tm3118)+UTX-1DD::GFP <br /> elegans. Since work focused role UTX-1                 (expressing UTX-1DD extrachromosomal array) was <br /> development, catalytic activity required              generated using standard crossing methods. <br /> processes implicated developmental programs and <br /> are dispensable viability. Indeed, recent reports implicate          Brood size <br /> catalytic activity UTX-1 aging [13,18]. Moreover, UTX-1                  Single fourth-stage (L4) larvae plated agar plates with <br /> activity act germline formation counteract               OP50 bacteria moved new plate 24 h. Viable <br /> well-established role PRC2/MES complex [37&#8211;43].                       progeny counted day, 4 days 25uC. The average <br /> However, far able establish function              number progeny produced single animal reported. <br /> of UTX-1 germline formation in <br /> synthetic interaction components MES complex (data               RNA interference (RNAi) <br /> not shown).                                                                     RNAi performed feeding carried described <br />    In summary, shown UTX-1 plays essential role              previously [45]. For UTX-1, clone (X-4I10) containing the <br /> in developmental processes C. elegans. Surprisingly,          region 18167 bp 19214 bp GenBank entry U23513 <br /> catalytic activity dispensable proper development,            (c Figure 1A), obtained C. elegans RNAi feeding <br /> data suggest UTX-1 acts, instead, SET-16/UTX-1                library (J. Ahringer&#8217;s laboratory, Wellcome Trust/<span id='am-30' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-31' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34'  ></span>Cancer</span> Re- <br /> complex. Future studies directed identifying specific         search UK Gurdon Institute, University Cambridge, Cam- <br /> target genes regulated complex possible role             bridge, UK). Two clones (a b Figure 1A), spanning <br /> UTX-1 play stability complex                  regions 14109&#8211;14331 bp 17512&#8211;18014 bp the <br /> recruitment target genes.                                             GenBank entry U23513, respectively, constructed PCR. <br />                                                                              We generated RNAi clones set-16, ash-2, pis-1, tag-125, and <br /> Materials Methods                                                        F21H12.1 amplifying cDNA fragments (approximately 500 pb) <br />                                                                              cloning L4440 plasmid using EcoRI restriction sites (all <br /> Genetics strains                                                         primer sequences available request). Eggs, prepared by <br />    C. elegans strains cultured using standard methods [44].             hypochlorite treatment, added RNAi bacteria-seeded <br /> Strains used follows: wild-type Bristol (N2), utx-                   NMG plates cultivated 25uC. Control animals fed <br /> 1(tm3118)X, utx-1(tm3136)X, jmjd-3.1(gk384)X, jmjd-3.2(tm3121)X,             bacteria carrying control vector (L4440). Generally, F1 <br /> jmjd-3.3(tm3197)X, JK2049 qls19 V, set-16(n4526)III, pis-                    progeny scored phenotypes. <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                             10                            May 2012 | Volume 8 | Issue 5 | e1002647 <br />                                                Roles UTX-1 C. elegans Development <br />  <br />  <br />  <br />  <br /> PLoS Genetics | www.plosgenetics.org   11   May 2012 | Volume 8 | Issue 5 | e1002647 <br />                                                                                                             Roles UTX-1 C. elegans Development <br />  <br />  <br />  <br /> Figure 5. UTX-1 MLL-like complex. (A) Immunoprecipitation GFP tagged UTX-1 mixed population (eggs adults) utx- <br /> 1(tm3118) rescued UTX-1::GFP. Affinity purified proteins resolved SDS-PAGE stained colloidal Coomassie. Homologues the <br /> mammalian UTX-MLL complex co-eluted bait identified LC-MS/MS listed grey. Position bait protein shown black. <br /> Molecular weight markers indicated left gel. (B) Table summarizing identified homologues components mammalian <br /> UTX-MLL complex. Gene names, molecular weight kDa, number unique peptides sequence coverage percentage reported. (C) Co- <br /> immunoprecipitations WDR-5.1::HA UTX-1. Total protein extracts eggs indicated strains immunoprecipitated using anti-HA <br /> affinity gel beads. The precipitates analyzed SDS-PAGE followed western blotting using antibodies HA, GFP endogenous UTX-1. <br /> Input = 30 mg protein extract. NBF = non bound fraction. (D) UTX-1-associated protein complex assessed size exclusion chromatography. <br /> Superose 6 gel filtration total protein extracts derived UTX-1 mutant rescued wild-type (utx-1+UTX-1::GFP) catalytically inactive UTX- <br /> 1 (utx-1+UTX-1DD::GFP). Fractions analyzed western blotting using GFP actin antibodies. <br /> doi:10.1371/journal.pgen.1002647.g005 <br />  <br />  <br />  <br /> Real-time quantitative PCR (RT&#8211;qPCR)                                           Microinjection production transgenic lines <br />    Total RNA isolated eggs using TRIzol reagent                          To obtain lines carrying extra-chromosomal arrays, 20 ng/ml of <br /> (Invitrogen) RNAeasy Minikit (Qiagen). cDNA synthe-                    UTX-1::GFP UTX-1 DD::GFP constructs co- <br /> sized using reagents TaqMan Reverse Transcription kit                 injected 100 ng/ml pRF4(rol-6(su1006)) ttx-3p::RFP in <br /> (Applied Biosystems). qPCR performed using SYBR Green                      wild-type N2 worms (N2+UTX-1::GFP N2+UTX-1DD::GFP). <br /> 26 PCR Master mix (Applied Biosystems) ABI Prism 7300                    Transgenic lines utx-1(tm3136 tm3118) genetic backgrounds <br /> Real Time PCR (Applied Biosystems). The measures                   (utx-1+UTX-1::GFP utx-1+UTX-1DD::GFP) generated by <br /> normalized ribosomal protein (rpl-26) RNA levels. All reactions             crossing. <br /> were performed triplicate, independent <br /> experiments. All primer sequences available request.                  Microscopy, image processing, quantification, and <br />                                                                                statistical analysis <br /> Quantitative Western blot analysis                                                Fluorescence microscope DIC pictures acquired using <br />    For mutant strains, total protein extracts prepared               Axiovert 135, Carl Zeiss, Inc. 636 Plan Apochrome <br /> eggs obtained hypochlorite treatment adults grown OP50                objective NA 1.4 immersion oil 406 Plan <br /> at 25uC. For RNAi-treated animals, extracts prepared                 NEOFLUAR NA 0.75, respectively. Pictures taken <br /> eggs obtained hypochlorite treatment adults grown                     room temperature CoolSNAP cf2; Photometrics <span id='am-55' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-56' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59'  ></span>camera</span>. <br /> HT115 containing feeding vector, specific                  All pictures exported preparation printing using <br /> RNAi. Protein concentration estimated using modified                   Photoshop (Adobe). MetaMorph software (MDS Analytical <br /> micro-Lowry assay equal amounts protein loaded.                    Technologies) used quantify mean s.e.m. of <br /> The following antibodies used: polyclonal anti-H3 (Abcam                  integrated intensities cell described [46]. The 2&#8211;4 most <br /> 1791, lot GR9204-1) 1:30000; polyclonal anti-H3K27me1 (Up-                     anterior intestinal cells used quantification of <br /> state 07-448, lot DAM1598790) 1:5000, polyclonal anti-                         H3K27me3/GFP 20 cells minimum of <br /> H3K27me2 (Abcam 24684, lot 956943) 1:2000; polyclonal anti-                    10 animals genotype (N2+UTX-1::GFP , N2+UTX- <br /> H3K27me3 (Upstate 07-449, lot 701050758) 1:2000; monoclonal                    1DD::GFP GFP-negative siblings) analyzed two <br /> anti-actin (Chemicon International MAB1501) 1:10000; peroxi-                   independent experiments. Only animals showing good <br /> dase-labeled anti-rabbit anti-mouse secondary antibodies                   H3K27me3 signal gonads, indication successful <br /> (Vector). The specificity H3K27 antibodies tested               immunostaining, used quantification. Statistical calcula- <br /> shown Figure S12. Polyclonal C. elegans UTX-1 antibodies               tions performed using Graphpad Prism software package <br /> obtained Eurogentec polyclonal antibody production                 (GraphPad Prism version 4.00 Windows, GraphPad Software, <br /> service. To generate specific UTX-1 antiserum, rabbits                  San Diego California USA, www.graphpad.com). Distribution of <br /> immunized UTX-1 peptides (MDESEPLPEERHPGNC                            data assessed using different normality tests: KS <br /> and SYRRSYKDDANRLDHC). Antibodies purified using                          normality test, D&#8217;agostino &amp; Pearson Normality test and <br /> affinity columns coupled peptides used 1:500              Shapiro-Wilk normality test. When data normally distributed <br /> dilution. The antibody recognizes lysate wild-type                   according tests, parametric statistics applied (t-test), <br /> animals specific band predicted size 134 kDa, absent            non-parametrical statistical analysis (Mann-Whitney U <br /> the lysates obtained utx-1 mutant alleles (Figure S12D).                  test) performed. When comparing groups <br /> Western blots quantified using ImageJ program (National                   ANOVA tests applied. For tests p#0.05 considered <br /> Institutes Health).                                                         significant. <br />  <br /> Construction tagged UTX-1 UTX-1DD                                       Immunofluorescence <br />    For UTX-1::GFP construct, 6956-bp fragment utx-1                     For immunostaining, animals fixed permeabilized as <br /> including 1290 bp promoter region entire coding region              described [47]. Polyclonal anti-H3K27me3 (Upstate 07-449) and <br /> was PCR-amplified N2 genomic DNA. The resulting                           polyclonal anti-UTX-1 (Eurogentec, clone 3917, study) were <br /> fragment inserted multiple cloning site                      used. Secondary antibodies were: goat anti-mouse IgG (Alexafluor <br /> pPD95.75 vector (Fire lab).                                                    488); goat anti-rabbit IgG (Alexafluor 594), purchased from <br />    For UTX-1DD::GFP construct, UTX-1::GFP construct                    Invitrogen. DAPI (Sigma, 2 ug/ul) used counter-stain <br /> was mutated using QuikChange Site-Directed Mutagenesis Kit                 DNA. Eggs immunofluorescence performed freeze crack <br /> (Stratagene). Specifically, DNA sequence mutated               method, adding eggs polylysine treated slides. After freezing at <br /> the histidine position 914 (H914) aspartic acid                  280uC 30 minutes, cover slip removed embryos <br /> position 916 (D916) changed alanine. The DNA sequences                 fixed methanol 220uC 10 min. Primary antibody <br /> of constructs verified sequencing.                                incubated overnight 4uC humid chamber and <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                               12                            May 2012 | Volume 8 | Issue 5 | e1002647 <br />                                                                                                               Roles UTX-1 C. elegans Development <br />  <br />  <br />  <br />  <br /> Figure 6. Functions SET-16/UTX-1 complex. (A) (Upper panel) DIC images set-16(n4526) pis-1(ok3720) mutant larvae. Arrowhead <br /> indicates misshapen tails. Scale bar 50 mm. (Lower panel) DIC images set-16(RNAi) pis-1(ok3720) adults. Gonadal migration defects shown. <br /> The black line indicates aberrant gonadal migration. Scale bar 20 mm. Animals oriented anterior left. The percentage tail and <br /> gonadal defects associated loss reduction set-16 pis-1 reported right. (B) Percentage tail gonad defects RNAi the <br /> indicated genes. F1 L1 larvae adult animals independent experiments scored. (C) Percentage tail gonad defects <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                13                            May 2012 | Volume 8 | Issue 5 | e1002647 <br />                                                                                                           Roles UTX-1 C. elegans Development <br />  <br />  <br />  <br /> after RNAi indicated genes. F1 F2 L1 larvae adult animals independent experiments scored. (D) utx-1 set-16 <br /> mRNA levels embryos worms treated control, utx-1 set-16(RNAi). *P,0.01 (Student&#8217;s t-test). (E) Protein lysates embryos worms <br /> treated control, utx-1 set-16(RNAi) probed antibody UTX-1. Actin used loading control. The signal quantified <br /> using ImageJ program normalized actin. Indicated values relative control (RNAi) derive independent experiments. <br /> doi:10.1371/journal.pgen.1002647.g006 <br />  <br /> secondary antibody incubated 1 h room temperature.                    pestle. The powder resuspended IP buffer (described in <br /> Washes PBS/tween 0.2%. Mounting medium                           GFP pulldown section) 5&#8211;10 mg incubated Protein <br /> fluorescence DAPI (Vectashield H1200) used.                         G agarose beads (Upstate) overnight 4uC. Soluble fraction was <br />                                                                              collected incubated EZview anti-HA affinity gel beads <br /> GFP pulldown                                                                 (Sigma Aldrich) 2 h 4uC. The immunoprecipitates and <br />    Generation transgenic strain utx-1+UTX-1::GFP                 protein G beads washed times IP buffer, boiled in <br /> described earlier Materials Methods. Total protein extracts           SDS-sample buffer analyzed SDS-PAGE followed by <br /> was obtained grinding frozen pellet mixed eggs adults            western blotting. Antibodies used experiments were: anti- <br /> with mortar pestle powder, resuspended             HA (Covance HA.11, clone 16B12), anti-GFP (Roche, <br /> in IP buffer containing 300 mM KCl, 0.1% Igepal, 1 mM EDTA,                  11814460001) anti-UTX-1 (Eurogentec, clone 3917, this <br /> 1 mM MgCl2, 10% glycerol, 50 mM Tris HCl (pH 7.4)                        study). Quantification western blots performed using <br /> protease inhibitors. GFP-Trap beads (Chromotek) used                 ImageJ program (National Institutes Health, USA). <br /> precipitate GFP-tagged proteins lysate. Approximately <br /> 200 mg total proteins used pulldown IP buffer.             Analytical gel filtration chromatography <br /> Following incubation washes buffer, proteins                  Eggs indicated strains grinded powder, resupended <br /> were eluted acidic <span id='am-13' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-17'  ></span>glycine</span> (0.1 M [pH 2.5]), resolved 4&#8211;           IP buffer (50 mM Tris-HCl pH 7.4, 300 mM KCl, 1 mM <br /> 12% NuPage Novex gel (Invitrogen), stained Imperial                 MgCl2, 1 mM EDTA, 0.1% Igepal complete protease <br /> Protein Stain (Thermo Scientific). The gel sliced 21 bands          inhibitors [Roche]) incubated wheel 30 min 4uC. <br /> across entire separation range lane. Cut bands               Protein extracts recovered centrifugation 20,000 g, <br /> reduced, alkylated iodoacetamide, in-gel digested              30 min 4uC clarified ultracentrifugation 627,000 g for <br /> trypsin (Promega) described previously [48], prior LC/MS-              30 min 4uC. Fresh extracts fractionated Superose 6 <br /> MS analysis.                                                                 HR 10/300 GL column (GE Healthcare) equilibrated IP buffer. <br />                                                                              Size exclusion chromatography performed fast protein <br /> Mass spectrometry proteins                                                liquid chromatography (FPLC) A    &#168; KTA purifier (GE <br />    Peptide identification performed LTQ-Orbitrap mass              Healthcare). Elution profiles blue dextran (2,000 kDa), thyro- <br /> spectrometer (Thermo Fisher Scientific, Germany) coupled                globulin (660 kDa) bovine serum albumin (66 kDa) used <br /> an EASY-nLC nanoHPLC (Proxeon, Odense, Denmark). Samples                     calibration. Fractions 1 ml collected precipitated <br /> were loaded 100 mm ID617 cm Reprosil-Pur C18-AQ                       25% trichloroacetic acid centrifuged 20,000 g for <br /> nano-column (3 mm; Dr. Maisch GmbH, Germany). The HPLC                       10 min 4uC. Pellets washed times cold acetone, air <br /> gradient 0 34% solvent B (A = 0.1% formic acid;                  dried, resuspended loading buffer Western blot analysis. <br /> B = 95% MeCN, 0.1% formic acid) 30 minutes from <br /> 34% 100% solvent B 7 minutes flow-rate 250 nL/                 Supporting Information <br /> min. Full-scan MS spectra acquired resolution of <br />                                                                              Figure S1 Embryonic/larval defects utx-1 mutant ajm- <br /> 60,000 Orbitrap analyzer. For scan, seven <br /> most intense ions isolated fragmentation LTQ                 1::GFP analysis. (A) DIC (left) epifluorescence (right) images of <br /> using CID. Raw data viewed using Xcalibur v2.1                      utx-1(tm3136) rescued (right animal) (left animal) a <br /> software (Thermo Scientific). Data processing performed                  translational reporter utx-1, control promoter. <br /> using Proteome Discoverer <span id='am-18' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>beta</span> version 1.3.0.265 (Thermo                     Note morphological defects not-rescued animal. (B) ajm- <br /> Scientific). For database search included Mascot v2.3                1::GFP localization utx-1(tm3136) embryos rescued (right, note <br /> (Matrix Science) SEQUEST (Thermo Scientific) search                      nuclear staining UTX-1::GFP) (left) a <br /> engines. Database C. elegans protein sequences downloaded             translational reporter utx-1, control promoter. <br /> from Uniprot. Trypsin selected digestion enzyme               ajm-1::GFP correctly localized initial stages appear <br /> missed cleavages allowed, carbamidomethylation cysteines             disorganized later stages (3 fold L1) not-rescued utx-1 <br /> was set fixed modification oxidation methionine                 mutant. Bars A 50 mm, B 20 mm. <br /> variable modification. MS mass tolerance set 10 ppm,            (TIF) <br /> MS/MS tolerance set 0.6 Da. Peptide validation                    Figure S2 elt-2::GFP analysis utx-1 mutant. Pattern of <br /> performed using Percolator peptide false discovery rate (FDR)            expression elt-2::GFP N2 utx-1(tm3136) allele at <br /> was set 0.01. For additional filtering, maximum peptide rank              different embryonic stages L1. Bars 20 mm. <br /> was set 1 minimum number peptides protein set              (TIF) <br /> to 2. Protein grouping performed, <span id='am-65' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-66' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> avoid presence <br /> of different proteins identified non-unique peptides. We                  Figure S3 myo-2::GFP analysis utx-1 mutant. Pattern of <br /> manually investigated protein listed represent            expression myo-2::GFP N2 utx-1(tm3136) allele at <br /> protein group characterized terms sequence                different embryonic stages L1. Bars 20 mm. <br /> coverage number peptides identified.                                  (TIF) <br />                                                                              Figure S4 hlh-1::GFP analysis utx-1 mutant. Pattern of <br /> Protein interaction assay                                                    expression hlh-1::GFP N2 utx-1(tm3136) allele at <br />   For co-immunoprecipitation assays, frozen eggs (prepared                different embryonic stages L1. Bars 20 mm. <br /> hypochlorite treatment) reduced powder using mortar              (TIF) <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                             14                            May 2012 | Volume 8 | Issue 5 | e1002647 <br />                                                                                                               Roles UTX-1 C. elegans Development <br />  <br />  <br /> Figure S5 dpy-7::GFP analysis utx-1 mutant. Pattern expression           Expression JMJD3-like genes indicated genetic <br /> of dpy-7::GFP N2 utx-1(tm3136) allele L1. The staining         backgrounds. The levels relative N2 normalized rpl- <br /> the pharynx co-injection marker. Bars 20 mm.                    26. <br /> (TIF)                                                                          (TIF) <br /> Figure S6 myo-3::GFP analysis utx-1 mutant. Pattern                      Figure S12 Specificity antibodies. A) List antibodies <br /> expression myo-3::GFP N2 utx-1(tm3136) allele                  used study. Specificity, reference, lot number working <br /> different embryonic stages L1. Note decreased level                 dilution antibodies indicated. (B) Specifity the <br /> myo-3::GFP utx-1 mutant compared N2. Bars 20 mm.                     antibodies used study. Dot blots performed as <br /> (TIF)                                                                          indicated Text S1 Methods using following <br />                                                                                peptides (indicated top): Histone H3 di-methyl K9 <br /> Figure S7 Efficiency RNAi brood size utx-1(RNAi) <br />                                                                                (H3K9me2), Histone H3 tri-methyl K9 (H3K9me3), Histone H3 <br /> treated worms. (A) Relative expression utx-1 mRNA                     mono-methyl K27 (H3K27me1), Histone H3 di-methyl K27 <br /> feeding RNAi treatments using constructs targeting different             (H3K27me2), Histone H3 tri-methyl K27 (H3K27me3). Different <br /> regions utx-1, indicated Figure 1A. Level UTX-1 protein               amounts peptides, ranging 100 1 ng, indicated the <br /> after utx-1(RNAi) measured Western blot, using specific               right blots, spotted membrane before <br /> antibody UTX-1. Actin used loading control. The                 probing indicated antibodies (on left). Two exposures <br /> signals quantified using ImageJ program normalized                 membrane shown. (C) Efficiency antibodies <br /> actin. Values relative control RNAi. Asterisks indicate results         used study worm samples. Different amounts worm <br /> different p,0.01 (Student&#8217;s t-test). (B) Average number eggs             total protein extracts (from 1 40 mg) resolved 15% <br /> laid worms treated control (white bars) utx-1(RNAi)                 polyacrylamide gel probed indicated antibodies. The <br /> (black bars).                                                                  western blots indicate analysis performed using the <br /> (TIF)                                                                          linear range antibodies. (D) Specificity UTX-1 <br /> Figure S8 Tail defects associated loss/downregulation                    antibody. Total protein extracts (50 mg) N2 utx-1 <br /> specific components SET-16/UTX-1 complex. Top panels.                   mutants analyzed Western blot using UTX-1 polyclonal <br /> Representative DIC images posterior defects observed                   antibody. Ponceau&#8217;s staining membrane shows equal <br /> downregulation utx-1, set-16 pis-1 using RNAi. F1 L1                loading. Arrowhead indicates UTX-1 band. <br /> shown. Bottom panels. Representative DIC images posterior                   (TIF) <br /> defects observed utx-1(tm3136), set-16(n4226) pis-1(ok3720).            Table S1 List proteins identified GFP-IP/MS UTX- <br /> L1 larvae shown. Bars 10 mm.                                           1:GFP worm sample. Accession numbers short descriptions of <br /> (TIF)                                                                          identified proteins given number of <br /> Figure S9 Somatic gonad defects associated loss/downregulation              peptides (S# peptides) recovered protein protein <br /> of components SET-16/UTX-1 complex. Representative DIC                  coverage (S coverage). M AG columns indicate 21 slices of <br /> images gonadal defects (top: aberrant migration, bottom: oocyte             gel (with A = gel U = gel). <br /> accumulation) observed utx-1(tm3136), pis-1(ok3720)               (XLSX) <br /> RNAi indicated genes. set-16(n4226) dies gonad                   Table S2 List proteins identified GFP-IP/MS N2 <br /> migration. Asterisks indicate oocytes accumulation distal region        worm sample. Accession numbers short descriptions the <br /> of gonad, black lines indicate aberrant gonadal migration.             identified proteins given number peptides <br /> Accumulation oocytes observed down-regulation ash-2         (S# peptides) recovered protein protein coverage <br /> and wdr-5. In RNAi, F1 F2 young adult scored.                          (S coverage). M AG columns indicate 21 slices gel <br /> (TIF)                                                                          (with A = gel U = gel). <br /> Figure S10      The overexpression level UTX-1 inversely                    (XLSX) <br /> correlates H3K27me3 intensity. Level H3K27me3 N2                    Text S1 Supporting Materials Methods. <br /> overexpressing wild-type UTX-1::GFP. H3K27me3 GFP                          (DOCX) <br /> intensities single intestinal cells (shown Figure 3B) were <br /> analyzed described Materials Methods. Scatter plot                   Acknowledgments <br /> presentation H3K27me3 levels cells having low (n = 10, <br /> range: 0&#8211;14807 arbitrary units) medium (n = 11, range: 28645&#8211;                  We thank Francesca Palladino (E&#180;cole Normale Supe&#180;rieure Lyon, <br /> 145977 arbitrary units) high (n = 11, range: 148852&#8211;258798                  France), Thomas Bu&#168;rglin (Karolinska Institutet, Sweden), Garry Wong <br />                                                                                (University Eastern Finland, Finland), Caenorhabditis Genetics <br /> arbitrary units) levels UTX-1::GFP. Statistical significance levels         Center (CGC), National BioResource project C. elegans (Japan) <br /> are provided. Worms analyzed independent                         providing strains used work. We grateful Maria Grazia <br /> experiments. Mean H3K27me3 expressed arbitrary units.                 Malabarba (IFOM, Italy) generation transgenic lines; to <br /> (TIF)                                                                          <span id='am-7' about='obo:NCBITaxon_381266' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Alexandra Bunge" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Alexandra" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Alexandra</span> Avram technical assistance; Paul Cloos, Karl Agger, <br />                                                                                Jesper Christensen, Toshia Myers, <span id='am-84' about='obo:NCBITaxon_1009818' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_263623 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_71526 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_7080 obo:NCBITaxon_50557 obo:NCBITaxon_115347 obo:NCBITaxon_7041'><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Roger" datatype="xsd:string"></span><span id='am-88'  ></span>Roger</span> Pocock (BRIC, University of <br /> Figure S11      Expression KDM6 family members. (A)                     Copenhagen, Denmark) discussions; Kristian Helin (BRIC, <br /> Relative expression KDM6 class members wild-type                     University Copenhagen, Denmark) Paolo Bazzicalupo (IGB, <br /> animals different stages, normalized rpl-26 levels. (B)                  Naples, Italy) reagents, discussion, critical reading the <br /> Quantification western blot shown Figure 4B. Bands                   manuscript. We thank Nordforsk support Nordic C. <br /> were analyzed using ImageJ program values reported                 elegans network. <br /> relative N2 levels. Note increase level H3K27me3 the <br /> triple mutant. (C) Expression transcriptional fusion constructs          Author Contributions <br /> the members KDM6 family obtained described                   Conceived designed experiments: JV GL ONJ AES. Performed the <br /> Text S1. Epifluorescence adult animals shown. Animals                experiments: JV GL SS EDS AES. Analyzed data: JV GL SS AES. <br /> oriented head left, ventral down. Bar 100 mm. (D)                    Wrote paper: JV AES. <br />  <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                               15                              May 2012 | Volume 8 | Issue 5 | e1002647 <br />                                                                                                                                   Roles UTX-1 C. elegans Development <br />  <br />  <br />  <br /> References <br /> 1.    Strahl BD, Allis CD (2000) The language covalent histone modifications.               26. Simonet T, Dulermo R, Schott S, Palladino F (2007) Antagonistic functions of <br />       Nature 403: 41&#8211;45.                                                                           SET-2/SET1 HPL/HP1 proteins C. elegans development. Develop- <br /> 2.    Kouzarides T (2007) Chromatin modifications function. Cell 128:                    mental biology 312: 367&#8211;383. <br />       693&#8211;705.                                                                                 27. Andersen EC, Horvitz HR (2007) Two C. elegans histone methyltransferases <br /> 3.    Berger SL (2007) The complex language chromatin regulation                         repress lin-3 EGF transcription inhibit vulval development. Development 134: <br />       transcription. Nature 447: 407&#8211;412.                                                          2991&#8211;2999. <br /> 4.    Tsukada Y, Fang J, Erdjument-Bromage H, Warren ME, Borchers CH, et al.                   28. Seenundun S, Rampalli S, Liu QC, Aziz A, Palii C, et al. (2010) UTX mediates <br />       (2006) Histone demethylation family JmjC domain-containing proteins.                 demethylation H3K27me3 muscle-specific genes myogenesis. The <br />       Nature 439: 811&#8211;816.                                                                         EMBO journal 29: 1401&#8211;1411. <br /> 5.    Cloos PA, Christensen J, Agger K, Helin K (2008) Erasing methyl mark:                29. Agger K, Cloos PA, Rudkjaer L, Williams K, Andersen G, et al. (2009) The <br />       histone demethylases center cellular differentiation disease. Genes            H3K27me3 demethylase JMJD3 contributes activation INK4A- <br />       Dev 22: 1115&#8211;1140.                                                                           ARF locus response oncogene- stress-induced senescence. Genes Dev <br /> 6.    Agger K, Cloos PA, Christensen J, Pasini D, Rose S, et al. (2007) UTX                    23: 1171&#8211;1176. <br />       JMJD3 histone H3K27 demethylases involved HOX gene regulation                 30. Miller SA, Mohn SE, Weinmann AS (2010) Jmjd3 UTX play a <br />       development. Nature 449: 731&#8211;734.                                                            demethylase-independent role chromatin remodeling regulate T-box <br /> 7.    Lee MG, <span id='am-68' about='obo:NCBITaxon_117093' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_117077 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_50704 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_50674 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_50671'><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Villa" datatype="xsd:string"></span><span id='am-71'  ></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Villa</span> R, Trojer P, Norman J, Yan KP, et al. (2007) Demethylation                  family member-dependent gene expression. Molecular cell 40: 594&#8211;605. <br />       H3K27 Regulates Polycomb Recruitment H2A Ubiquitination. Science                     31. Greer EL, Maures TJ, Hauswirth AG, Green EM, Leeman DS, et al. (2010) <br />       318: 447&#8211;450.                                                                                Members H3K4 trimethylation complex regulate lifespan germline- <br /> 8.    Smith ER, Lee MG, Winter B, Droz NM, Eissenberg JC, et al. (2007)                            dependent manner C. elegans. Nature 466: 383&#8211;387. <br />       Drosophila UTX histone H3 Lys27 demethylase colocalizes               32. Xiao Y, Bedet C, Robert VJ, Simonet T, Dunkelbarger S, et al. (2011) <br />       elongating form RNA polymerase II. Mol Cell Biol 28: 1041&#8211;1046.                           <span id='am-35' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span>Caenorhabditis elegans</span> chromatin-associated proteins SET-2 ASH-2 are <br /> 9.    Lan F, Bayliss PE, Rinn JL, Whetstine JR, Wang JK, et al. (2007) A histone H3                differentially required histone H3 Lys 4 methylation embryos adult <br />       lysine 27 demethylase regulates animal posterior development. Nature 449:                    germ cells. Proceedings National Academy Sciences United <br />       689&#8211;694.                                                                                     States America 108: 8305&#8211;8310. <br /> 10.   De Santa F, Totaro MG, Prosperini E, Notarbartolo S, Testa G, et al. (2007)              33. Solari F, Ahringer J (2000) NURD-complex genes antagonise Ras-induced <br />       The Histone H3 Lysine-27 Demethylase Jm <br /> </body></html>