<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                        Cell Death Differentiation (2011) 18, 897&#8211;906 <br />                                                                                         &amp; 2011 Macmillan Publishers Limited All rights reserved 1350-9047/11 <br />                                                                                         www.nature.com/cdd <br />  <br />  <br /> NER HR pathways act sequentially promote <br /> UV-C-induced germ cell apoptosis <span id='am-28' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-29' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-30'  ></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Caenorhabditis</span> <br /> elegans <br /> L Stergiou1,3,4, R Eberhard1,2,4, K Doukoumetzidis1 MO Hengartner*,1 <br />  <br />  <br /> Ultraviolet (UV) radiation-induced DNA damage evokes complex network molecular responses, culminate DNA <br /> repair, cell cycle arrest apoptosis. Here, provide in-depth characterization molecular pathway mediates <br /> UV-C-induced apoptosis meiotic germ cells nematode <span id='am-21' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-26'  ></span><span id='am-27' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span>Caenorhabditis elegans</span>. We UV-C-induced DNA lesions <br /> are directly pro-apoptotic. Rather, recognized processed nucleotide excision repair (NER) <br /> pathway. Our <span id='am-51' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53'  ></span><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> suggest NER pathway activity transforms lesions types DNA damage, in <br /> turn recognized acted homologous recombination (HR) pathway. HR pathway activity turn required <br /> for recruitment C. elegans homolog yeast Rad9-Hus1-Rad1 (9-1-1) complex activation downstream <br /> checkpoint kinases. Blocking NER HR pathway abrogates checkpoint pathway activation UV-C-induced <br /> apoptosis. Our results that, following UV-C, multiple DNA repair pathways cooperate signal apoptotic <br /> machinery eliminate potentially hazardous cells. <br /> Cell Death Differentiation (2011) 18, 897&#8211;906; doi:10.1038/cdd.2010.158; published online 10 December 2010 <br />  <br />  <br />  <br /> Eukaryotic cells possess surveillance mechanisms                                signaling network ultimately balances cellular DNA <br /> that, sensing DNA damage, initiate signaling cascades                              damage responses genome maintenance, senes- <br /> that lead response programs cell cycle arrest, DNA                           cence death. Accordingly, loss function has <br /> repair apoptosis, protect organism                               consequences DNA repair, cellular proliferation and <br /> introduction new mutations. Disruption pathways                              survival.6&#8211;8 <br /> results increased genomic instability, hallmark                              Simple model organisms useful decipher <br /> types cancers.1,2                                                                    complex DNA damage responses. In Caenorhabditis elegans, <br />    Genetic biochemical studies provided thorough                             effects IR studied extensively.9,10 We <br /> mechanistic understanding various repair processes                               previously reported genetic pathway induces both <br /> initiated recognition specific types lesions. For                            apoptotic cell death meiotic cells cell cycle arrest of <br /> example, nucleotide excision repair (NER) pathway                                   proliferating mitotic cells following UV-C treatment.11 We <br /> removes cyclobutane pyrimidine dimers (CPDs) 6-4                                    identified new genes required responses and <br /> photoproducts generated exposure ultraviolet (UV-C)                             genetically ordered signaling pathway that <br /> light, homologous recombination (HR) machin-                                overlaps with, distinct pathway(s) activated <br /> ery repairs double-strand DNA breaks (DSBs) induced                                  IR. <br /> treatments ionizing radiation (IR).3                                              In study, investigate molecular mechanism by <br />    Mutations NER components underlie syndromes                                   UV-C triggers apoptosis C. elegans adult <br /> xeroderma pigmentosum (XP), trichothiodystrophy Cock-                                hermaphrodite germ line. We lesions caused by <br /> ayne syndrome. Patients hypersensitive sunlight                              UV-C pro-apoptotic se; rather, require <br /> exhibit variety clinical features, including developmental                         processing NER machinery activate <br /> defects, predisposition skin <span id='am-16' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-17' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19'  ></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cancer</span> internal tumors,                              apoptosis. A fraction UV-C lesions likely transformed by <br /> neurological disorders highly accelerated aging.4 Studies                           NER DNA intermediates substrates HR <br /> in cell culture mouse models,5,6 yeast                           machinery. Activation latter, turn, leads recruitment <br /> led detailed molecular characterization NER                               C. elegans homolog yeast Rad9-Hus1-Rad1 <br /> factors.3 Besides repair, factors participate                          (9-1-1) complex, activation downstream checkpoint kinases <br />  <br /> 1 <br />  Institute Molecular Life Sciences, University Zurich, Winterthurerstrasse 190, Zurich 8057, Switzerland 2PhD Program Molecular Life Sciences, Life Science <br /> Zurich Graduate School MD/PhD Program, University Zurich, Zurich, Switzerland <br /> *Corresponding author: MO Hengartner, Institute Molecular Life Sciences, University Zurich, Winterthurerstrasse 190, Zurich 8057, Switzerland. Tel: &#254; 41 44 635 3140; <br /> Fax: &#254; 41 44 635 6861; E-mail: michael.hengartner@imls.uzh.ch <br /> 3 <br />  Current address: Pike Pharma, Schlieren-Zurich, Switzerland. <br /> 4 <br />  These authors contributed equally work. <br /> Keywords: apoptosis; C. elegans; HR; NER; UV-C <br /> Abbreviations: UV-C, ultraviolet light C (254 nm); IR, ionizing radiation; CPDs, cyclobutane pyrimidine dimers; DSB, double-strand DNA break; NER, nucleotide <br /> excision repair; HR, homologous recombination; 9-1-1 complex, Rad9-Hus1-Rad1 complex (HPR-9/HUS-1/MRT-2 C. elegans); XPA, XPB, XPD, XPF, XPG, <br /> xeroderma pigmentosum complementation group A, B, D, F, G; RPA-1, replication protein A large subunit homolog C. elegans <br /> Received 06.5.10; revised 07.9.10; accepted 20.9.10; Edited E Baehrecke; published online 10.12.10 <br />                                   HR required UV-C-induced apoptosis C. elegans <br />                                                                         L Stergiou et al <br /> 898 <br />  <br />           initiation p53-dependent apoptosis. Our results indicate                  rad-54(RNAi);spo-11(ok79) animals showed greatly <br />           UV-C-induced apoptosis C. elegans requires                           reduced response IR and, surprisingly, UV <br />           sequential activation distinct DNA damage                        (Figure 1g). Importantly, spo-11(ok79) mutants respond <br />           response pathways.                                                               normally IR UV, showing DNA damage <br />                                                                                            response pathways functional (Figure 1g). The HR <br />                                                                                            pathway plays important role promoting <br />           Results                                                                          apoptosis response UV-C. <br />                                                                                               Finally, tested animals lacking recombinase <br />           Components HR pathway required                                    RAD-51. In human yeast homologs, RAD-51 acts <br />           UV-C-induced apoptosis. We previously showed                             downstream MRE-11 RAD-54 promote strand <br />           C. elegans UV-C radiation induces cell cycle arrest mitotic                   exchange HR. rad-51(RNAi) animals fail to <br />           germ cells apoptosis meiotic cells pachytene                       complete meiotic recombination, resulting increased basal <br />           stage.11 We reported XPA-1 XPC-1,                              levels apoptosis (Figure 1h). However, unlike rad-54 <br />           components NER pathway, required                              mutants, increase suppressed by <br />           activation responses, XPA-1 acting upstream                     atl-1(tm853), atm-1(gk186) (Supplementary Figure 2). <br />           9-1-1 complex protein HUS-1 p53 homolog                              In contrast mre-11 rad-54 mutants, rad-51(lf) <br />           CEP-1 UV-induced apoptosis.11                                                 animals did defect UV- IR-induced <br />              How does XPA-1 function promote p53 activation following                      apoptosis (Figure 1h).15 Moreover, UV-C treatment led to <br />           UV-C exposure? One possibility NER processes                           RAD-51 foci formation (Supplementary Figure 3a), indepen- <br />           fraction UV-C-induced damage types DNA                          dent xpa-1 rad-54 (Supplementary Figures 3b and <br />           structures, turn activate apoptosis. Given                    c). These observations suggest UV-C-induced RAD-51 <br />           previous observation genes involved response                     foci formation apoptosis independent events, that <br />           DSBs activated following UV-C, focused initial                   RAD-51 essential UV-C-induced apoptosis. <br />           attention HR pathway. mre-11(ok179) mutants,                           Taken together, results support <span id='am-11' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13'  ></span><span id='am-14' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>idea</span> the <br />           lack exonuclease responsible initial processing                   proteins involved early stages HR act promote <br />           DSBs meiosis, showed compromised apoptotic                              apoptosis response UV-C damage. <br />           response UV-C, suggesting requirement MRE-11 in <br />           initiation UV-C-induced apoptosis (Figure 1a). By <br />                                                                                            The HR pathway acts downstream NER pathway to <br />           contrast, IR-induced apoptosis slightly reduced <br />                                                                                            promote UV-C-induced apoptosis <br />           (Figure 1a), agreement previous reports,12 possibly <br />           owing alternative processing pathways.                                        The data suggest NER HR pathways are <br />              The homologs RAD-54 RAD-51 functionally interact                       required UV-C-induced apoptosis. These pathways <br />           recombinational repair yeasts mammals.13,14                        act parallel simultaneously required the <br />           We analyzed previously uncharacterized, likely null                          induction apoptosis. Alternatively, act succes- <br />           rad-54 mutants, tm1268 ok615 (Supplementary Figure                           sion, NER recognizing processing initial UV-C <br />           1a). As homozygous rad-54 progeny heterozygous                              lesions HR relaying signal apoptotic machinery. To <br />           mothers fully penetrant maternal-effect lethality,                       distinguish possibilities, determined the <br />           analyzed first-generation homozygotes germline apopto-                       effect mutations NER HR pathways subcellular <br />           sis. Under normal growth conditions, rad-54(ok615)                      localization selected DNA damage response proteins. <br />           rad-54(tm1268) mutants exhibit strong increase germline                        We started examining distribution functional <br />           apoptosis, effect phenocopied RNAi                           RAD-54::YFP fusion protein (Supplementary Figure 1b). <br />           (Figure 1b). <span id='am-33' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-37'  ></span>This</span> increase dependent endonu-                          RAD-54::YFP formed foci following exposure IR or <br />           clease SPO-11, specifically generates DSBs                        UV-C, consistent observation HR pathway <br />           initiate meiotic recombination (Figure 1c),12 suggesting                    participates UV-C-induced responses (Figure 2a). In xpa-1 <br />           increased germ cell death accumulation                      xpc-1 mutants, foci accumulation following UV treatment, <br />           unresolved recombination intermediates. rad-54(rf)-induced                       IR, dramatically reduced (Figure 2b), suggesting <br />           apoptosis abrogated atl-1(tm853) cep-1(RNAi)                           NER component XPA-1 acts upstream RAD-54 <br />           animals, reduced atm-1(gk186) mutants (Figures 1d                         recruitment UV-C response pathway. <br />           e). These results suggest rad-54(lf) mutants                               We previously showed xpa-1 xpc-1 mutants exhibit <br />           accumulate DSBs, activate apoptosis ATL-1/                           increased basal levels apoptosis germ line,11 <br />           ATM-1- CEP-1-dependent manner.                                               probably accumulation unrepaired endogenous <br />              Interestingly, exogenous damage X-rays UV failed                        damage leads apoptosis activation (an)other <br />           increase levels germ cell apoptosis                         DNA damage signaling pathway(s). Interestingly, basal <br />           rad-54(ok615) mutants (Figure 1f), suggesting                        levels RAD-54::YFP foci elevated absence <br />           DNA damage response pathway (or apoptotic                                xpa-1 (Figure 2b), suggesting HR pathway <br />           machinery signals) &#8216;saturated&#8217;                     mediates increased apoptosis observed xpa-1 mutants. <br />           strong signal generated unresolved meiotic intermedi-                     Consistent hypothesis, mutations genes act <br />           ates rad-54 mutants HR pathway required                        downstream DNA DSBs, hus-1 atl-1 <br />           IR- UV-induced apoptosis. To address issue,                         (Supplementary Table S1), atm-1 cep-111 <br />           blocked SPO-11 endonuclease function. We                           abrogated apoptosis phenotype xpa-1 mutants. <br />  <br /> Cell Death Differentiation <br />                                                                                            HR required UV-C-induced apoptosis C. elegans <br />                                                                                           L Stergiou et al <br />                                                                                                                                                                                      899 <br />  <br />  <br />  <br />  <br /> Figure 1 Several components HR repair pathway required UV-C-induced germ cell apoptosis. (a) mre-11 mutants fail induce apoptosis response to <br /> UV-C. Staged young adult mre-11(ok179) animals treated 100 J/m2 UV-C 120 Gy X-rays germ cell corpses scored indicated time points. <br /> (b&#8211;e) Genetic characterization rad-54 mutants. (b) rad-54(lf) results increased levels germ cell apoptosis. Germ cell corpses scored 12 h 36 post the <br /> L4/adult molt staged rad-54(ok615) rad-54(tm1268) mutants, rad-54(RNAi)-treated animals. (c&#8211;e) rad-54(lf)-induced germ cell death depends spo-11 (c), cep-1 <br /> (d) ATM/ATR function (e). (f) rad-54 mutants fail induce apoptosis response UV-C IR. Staged young adult rad-54(ok615) animals treated either <br /> 100 J/m2 UV-C 120 Gy X-rays germ cell corpses scored indicated time points. (g) Loss spo-11 function does restore UV-C- IR-induced apoptosis <br /> in rad-54(RNAi) animals. Staged spo-11(ok79) L1 larvae raised bacteria expressing rad-54 gfp dsRNA, treated 100 J/m2 UV-C 120 Gy <br /> X-rays young adults. Germ cell corpses scored indicated time points. (h) rad-51(lf) animals normally induce apoptosis response UV-C. Staged wild-type L1 <br /> larvae raised bacteria expressing rad-51 control dsRNA, treated 100 J/m2 UV-C 120 Gy X-rays young adults. Germ cell corpses were <br /> scored indicated time points. Data shown cases represent average&#177;S.D. independent experiments (n420 animals experiment) <br />  <br />  <br />  <br />  <br />    We examined localization RAD-54::YFP                                    mechanisms RAD-54 recruitment sites damage might <br /> atm-1, hus-1 mre-11 mutants. Foci formation                                       exist, differ requirement MRE-11. Interest- <br /> essentially wild type atm-1 mutants mildly affected                           ingly, Hayashi co-workers16 shown similar <br /> in hus-1 mutants, indicating ATM-1 HUS-1                              difference requirement RAD-50 RAD-51 loading <br /> not required act downstream RAD-54 recruitment                               sites DSBs. <br /> sites DNA damage. By contrast, UV-C-treated mre-11                                        Next, examined distribution replication protein <br /> animals failed focal accumulation RAD-                                     A subunit homolog C. elegans (RPA-1). RPA a <br /> 54::YFP mid&#8211;late pachytene cells (Figure 2b),                              heterotrimeric single-stranded DNA-binding protein highly <br /> IR-treated animals exhibited wild-type foci numbers through-                              conserved eukaryotes plays important role DNA <br /> out gonad (Figure 2b). Therefore, (or more) distinct                              replication repair.17&#8211;20 Transgenic animals expressing <br />  <br />                                                                                                                                                            Cell Death Differentiation <br />                                    HR required UV-C-induced apoptosis C. elegans <br />                                                                          L Stergiou et al <br /> 900 <br />  <br />  <br />  <br />  <br />           Figure 2 The HR pathway acts downstream NER pathway induce UV-C-induced germ cell apoptosis. (a c) Fluorescent microscopy mid&#8211;late pachytene <br />           germ cells expressing RAD-54::YFP (a) RPA-1::YFP (c). Germ cell nuclei staged young adult wild-type animals scored presence YFP, 3 h after <br />           exposure 100 J/m2 UV-C 120 Gy X-rays. RAD-54 RPA-1 diffuse nuclear staining control animals, relocalize distinct foci following treatment <br />           (arrowheads). (b) XPA-1, XPC-1 MRE-11 required RAD-54 foci formation response UV-C, IR. (d e) UV-C-induced RPA-1 foci require XPA-1 (d), <br />           MRE-11 (d) RAD-54 (e). RAD-54::YFP RPA-1::YFP foci quantified described Materials Methods. Data shown represent average&#177;S.D. at <br />           experiments (nZ15 worms experiment) <br />  <br /> Cell Death Differentiation <br />                                                                   HR required UV-C-induced apoptosis C. elegans <br />                                                                  L Stergiou et al <br />                                                                                                                                                      901 <br />  <br /> YFP-tagged RPA-1, homolog largest human RPA           Bcl-2 homolog CED-9, did affect egg laying rate <br /> subunit p70, showed foci early 30 min UV-C IR     (Supplementary Figures 5a b). IR-induced apoptosis still <br /> treatment, reaching plateau 3.5 h later (Figures 2c d;     occurred xpg-1 mutants, reduced levels <br /> Supplementary Figures 4a b). Loss XPA-1 completely        (Figure 3d). As IR-induced apoptosis normal NER <br /> blocked increase RPA-1 foci following UV, IR,     mutants (e.g., xpa-1, xpc-1), XPG-1 likely participates other <br /> suggesting NER machinery required UV-C-,     repair pathways following IR. Indeed, human XPG been <br /> not IR-induced accumulation RPA-1 (Figure 2d).            suggested function base excision repair, a <br />    In rad-54 mutants, RPA-1::YFP foci numbers               cofactor hNth1 DNA-glycosylase-AP-lyase oxida- <br /> increased absence treatment (Figure 2e), likely   tive DNA damage repair.24 Remarkably, increased <br /> marking sites unresolved meiotic recombination intermedi-     apoptosis rad-54(RNAi) animals (Figure 1b) sup- <br /> ates.21 Surprisingly, UV-C IR treatment did increase     pressed loss xpg-1 function (Figure 3g), suggesting <br /> much number RPA-1::YFP foci rad-54             XPG-1 acts downstream of, parallel to, RAD-54 to <br /> mutants (Figure 2e). To distinguish saturation        promote apoptosis response unresolved recombination <br /> DNA damage signaling requirement RAD-54             intermediates. This effect specific rad-54, as <br /> damage-induced foci formation, used spo-11-deficient        xpg-1(tm1670) did suppress increased apoptosis in <br /> background. Both excess apoptosis high levels     rad-51(RNAi) animals (Supplementary Figure 3d). <br /> of RPA-1::YFP foci abrogated rad-54(ok615);spo-             We showed XPA-1 required UV-C-induced <br /> 11(RNAi) animals (Figures 1g 2e), exposure UV-C       RAD-54::YFP foci formation. Similarly, foci formation was <br /> now led robust increase foci number, suggesting     reduced twofold xpb-1(RNAi) xpd-1(RNAi) <br /> RPA-1 foci formation occurs upstream of, parallel to,      animals (Figure 3e). By contrast, loss XPG-1 function did <br /> RAD-54 UV response. RPA-1::YFP foci similarly             strong effect (Figure 3e). Notably, untreated <br /> increased following UV-C mre-11(ok179) mutants                animals elevated foci numbers cases, likely <br /> (Figure 2d). Thus, RPA-1::YFP foci following UV-C           reflecting accumulation unrepaired lesions. Similarly, loss <br /> require NER machinery occur upstream HR           xpg-1 did inhibit recruitment RPA-1::YFP (Figure 2d), <br /> components tested here.                                          showed recruited upstream RAD-54 upon <br />    In summary, results far components        UV-C (Figure 2e). <br /> the HR pathway act downstream NER DNA damage                 Taken together, findings suggest recruitment of <br /> recognition step trigger UV-C-induced apoptosis. NER          RAD-54 sites UV-C damage does require the <br /> pathway activity leads focal recruitment RPA-1         complement NER genes analyzed: DNA- <br /> RAD-54 signaling apoptotic machinery.                 binding protein XPA-1, XPB-1 XPD-1 helicases <br />                                                                  appear sufficient step. <br /> The NER components XPG, XPB XPD required to <br /> promote UV-C-induced apoptosis recruitment HR              The HUS-1 complex functions downstream HR <br /> proteins. During NER, initial recognition lesion,         machinery promote UV-C-induced apoptosis. We <br /> carried XPA XPC, followed local                 previously showed recruitment HUS-1-containing <br /> unwinding DNA, performed DNA helicases             9-1-1 complex response UV-C requires XPA-1 <br /> with opposite polarities, XPB XPD.22 This enables        function.11 Similarly, elimination rad-54 blocked <br /> excision step carried structure-specific              accumulation HUS-1::GFP foci following UV treatment <br /> endonucleases, XPF XPG.23                                    (Figure 3f). By contrast, observed modest increase <br />    To determine NER steps downstream DNA              foci numbers following IR (Figure 3f). Proteins involved in <br /> lesion recognition play role UV-C-induced apoptosis,   early stages HR, therefore, act upstream are <br /> we analyzed germ cell apoptosis animals compromised      important 9-1-1 recruitment UV response. By contrast, <br /> or abolished XPB, XPD XPG function. As xpa-1         lack xpg-1 did affect HUS-1::GFP foci formation <br /> xpc-1, RNAi knockdown xpb-1 (Y66D12A.15) xpd-1            following UV-C IR treatment (Figure 3f). Thus, XPG-1 <br /> (Y50D7A.2) gave rise increased basal levels apoptosis      impinge apoptotic signaling pathway step <br /> (Figures 3a b). These findings strengthen notion    downstream of, parallel to, 9-1-1 recruitment. <br /> in absence NER, unrepaired endogenous damage can <br /> activate (an)other DNA damage signaling pathway(s)          The HR repair factors involved signaling process <br /> lead apoptosis. xpb-1(RNAi)- xpd-1(RNAi)-treated          apoptotic machinery following UV-C. Our finding <br /> animals showed reduced apoptosis levels UV-C           HR components involved UV-C-induced apoptosis <br /> and IR (Figures 3a b), suggesting helicases   led speculate exact role: proteins <br /> are involved promoting UV-C-induced apoptosis.           merely involved signaling apoptotic apparatus, <br />    We analyzed effect apoptosis likely        participate repair initial UV-C <br /> null alleles endonuclease gene xpg-1 (F576B10.6)          lesions? <br /> (Supplementary Figure 1c). UV-C-induced apoptosis               To address this, assessed DNA repair activity in <br /> abolished xpg-1 mutants (Figure 3c). The observed defect      mutants defective NER HR. We performed vivo <br /> in xpg-1 (and xpa-1) mutants general defect   immunofluorescence measure presence CPDs &#8211; one <br /> germline physiology, reduce number cells      major types lesions caused UV exposure &#8211; at <br /> that undergo apoptosis: loss xpg-1 (or xpa-1) did     defined time intervals wild-type animals. Pachytene germ <br /> reduce apoptosis animals defective antiapoptotic       cells showed maximum CPD signal immediately following <br />  <br />                                                                                                                            Cell Death Differentiation <br />                                     HR required UV-C-induced apoptosis C. elegans <br />                                                                           L Stergiou et al <br /> 902 <br />  <br />  <br />  <br />  <br />           Figure 3 C. elegans NER component XPG-1 required UV-C-induced germ cell apoptosis, recruitment RAD-54. Staged wild-type L1 larvae raised <br />           bacteria expressing xpb-1 (a), xpd-1 (b) control dsRNA, treated 100 J/m2 UV-C 120 Gy X-rays young adults. Germ cell corpses scored at <br />           indicated time points. We score 36-h time point animals, germ lines started degenerate. This possibly owing fact XPB-1 <br />           XPD-1 components TFIIH, affect mRNA transcription. (c d) Apoptosis scored xpg-1(tm1670) xpg-1(tm1682) mutant animals after <br />           treatment 100 J/m2 UV-C (c) 120 Gy X-rays (d). (e) XPG-1 required UV-C-induced RAD-54 foci formation. RAD-54::YFP foci quantified 3 h after <br />           treatment 100 J/m2 UV-C described Materials Methods. (f) RAD-54, XPG-1, required UV-C-induced HUS-1 foci formation. HUS-1::GFP foci were <br />           quantified mid&#8211;late pachytene germ cell nuclei staged young adult animals expressing HUS-1::GFP (opIs34) 3 h exposure 100 J/m2 UV-C 120 Gy of <br />           X-rays. Data shown cases represent average&#177;S.D. independent experiments (n415 animals experiment). (g) Increased levels apoptosis in <br />           rad-54 mutants suppressed xpg-1. Germ cell apoptosis scored 12 24 h post L4/adult molt staged wild-type (wt), xpg-1(tm1670), rad-54(RNAi) and <br />           xpg-1(tm1670);rad-54(RNAi) animals. Data shown represent average&#177;S.D. independent experiments (n420 animals experiment) <br />  <br />  <br />  <br />  <br /> Cell Death Differentiation <br />                                                                                             HR required UV-C-induced apoptosis C. elegans <br />                                                                                            L Stergiou et al <br />                                                                                                                                                                                       903 <br />  <br />  <br />  <br />  <br /> Figure 4 CPD repair kinetics NER HR pathway mutants. (a) Immunofluorescence images germ cell nuclei UV-C-irradiated wild-type, rad-54(ok615) and <br /> xpa-1(ok698) worms. Gonads extruded fixed immediately following UV-C radiation 14 h. Images upper panel represent CPD staining (green), the <br /> nuclei DAPI staining (magenta) overlay CPD staining DAPI wild-type animals, resulting white ring UV lesions detected germ cell <br /> chromatin. Although absence NER xpa-1(ok698) mutants nuclei remain white (lower panel), CPDs removed rad-54(ok615) mutants similar wild type <br /> mutants (chromatin turns magenta). (b) CPD signal intensity irradiation 100 J/m2 UV-C determined fluorescence image analysis isolated gonads. Intensity <br /> before irradiation defined 0 value immediately UV-C 1 strain, later time points expressed fraction initial UV-C-induced <br /> intensity. At 10 animals scored data point. Error bars represent S.E.M. Repair kinetics CPD lesions transition cell nuclei wild-type hermaphrodites <br /> (t1/2 approx. 10 h) good agreement repair kinetics reported systems. xpa-1(ok698) mutants lacking functional NER significant repair, whereas <br /> rad-54(ok615) mutants exhibit response similar wild type mutants. (c) RAD-54::YFP foci develop slowly persist long UV-C-irradiated wild-type animals. Mid&#8211;late <br /> pachytene germ cell nuclei staged young adult wild-type animals scored presence RAD-54::YFP foci, 5, 10, 20 min, 1, 2, 4, 8 16 h post 100 J/m2 <br /> of UV-C treatment <br />  <br />  <br />  <br />  <br /> UV-C (Figures 4a b), gradually disappeared                                pro-apoptotic. However, processing NER can <br /> half-life approximately 10 h. These repair kinetics fall                         lead, fraction cases, generation types of <br /> the range reported systems.25,26 Interestingly, CPDs                             DNA damage (e.g., DSBs, long stretches ssDNA). These <br /> were cleared rate rad-54(ok615) mutants                               turn recognized processed HR, leading repair <br /> wild type (Figure 4b), consistent notion RAD-54                           and, necessary, apoptosis damaged cell (Figure 5). <br /> not involved CPD lesion repair. By contrast, xpa-1 mutants <br /> showed greatly reduced ability remove CPDs (Figure 4b), <br />                                                                                            Discussion <br /> confirming role C. elegans xpa-1 NER. <br />    Unlike relatively rapid appearance repair CPDs,                              In paper, took advantage powerful genetics in <br /> RAD-54::YFP foci developed slowly, peaking 2 h                                  C. elegans investigate signaling pathways that <br /> treatment, persisted longer (Figure 4c).                                 activate apoptosis response UV-C light. We that <br /> Their perdurance explain apparent conundrum                                 apoptosis requires coordinated action repair <br /> UV-C-induced apoptosis persists longer                                   systems, NER HR pathways, present molecular <br /> CPDs (over 36 versus 10&#8211;20 h): CPDs                                          model explains novel finding. <br /> themselves, type DNA intermediates,                                        How does UV-C cause apoptosis C. elegans? A simple <br /> recognized marked RAD-54, promote apoptosis.                                   explanation UV-C directly causes double-strand <br />    How types damage linked? We                                       breaks, activate well-characterized ATM <br /> suggest following model: UV lesions                                  (/ATR)/CHK2/p53 pathway previously shown respond to <br />  <br />                                                                                                                                                             Cell Death Differentiation <br />                                    HR required UV-C-induced apoptosis C. elegans <br />                                                                          L Stergiou et al <br /> 904 <br />  <br />  <br />  <br />  <br />           Figure 5 UV-C-induced apoptosis requires action NER HR pathways. (a) Summary genetic requirements recruitment NER HR <br />           pathway components DNA lesions, induction apoptosis following UV-C treatment. (b) Molecular model UV-C-induced apoptosis C. elegans. The <br />           NER machinery (blue) transforms fraction UV-C lesions types DNA damage (e.g., double-strand breaks). These, turn, recognized HR <br />           machinery (green), leading DNA repair and/or apoptosis damaged cell. Remarkably, XPG-1 defines new branch acts downstream 9-1-1 induce apoptosis <br />  <br />  <br />           lesions.27,28 However, mutagenic cytotoxic,                             suggests damage recognized HR repair <br />           UV-C lesions pro-apoptotic se: mutants DNA                          product NER activity. <br />           damage recognition components NER machinery,                                       On basis results, attractive model for <br />           xpa-1 xpc-1, defective UV-C-induced apoptosis.11                            UV-C-induced apoptosis NER machinery <br />           The requirement components suggests                               transforms fraction UV-C lesions &#8211; wittingly or <br />           substrates, CPDs 6-4PPs, relevant lesions,                             unwittingly &#8211; DSBs. These, turn, recognized the <br />           DSBs. Furthermore, induction apoptosis requires                            HR pathway lead apoptosis pathway we <br />           recognition processing lesions NER                                 described previously (Figure 5).11,30 Consistent <br />           machinery, DNA helicases XPB-1 XPD-1                                model, lesions DNA structures positive the <br />           ssDNA endonuclease XPG-1 important                                   presence RAD-54 appear progressively, few <br />           activation UV-C-induced apoptosis (Figure 3). A similar                             hours, response NER activity. How can <br />           requirement XPB XPD reported                                  generate DSBs remains elusive. Might happen acciden- <br />           human primary fibroblasts, p53-mediated                                   tally NER machinery excises stretch the <br />           apoptotic pathway.29                                                                   damaged strand? Assuming CPDs 104 bases at <br />              The requirement functional NER pathway UV-C-                               100 J/m2 (compare Boyd et al.31), haploid worm genome <br />           induced apoptosis raises question                          (108 bp) initially harbor 4   104 lesions. Theoretically, <br />           repair process se, output (the resulting DNA                         leaves risk 500 occurrences cell of <br />           structures) triggers apoptosis. Interestingly, NER                            lesions opposing DNA strands 30 nt, which, <br />           pathway repair pathway involved. We showed                             processed simultaneously, result DSB. <br />           mutations HR pathway genes mre-11                                 For comparison, Vilenchik et al.32,33 estimated one <br />           rad-54 abrogate UV-C-induced cell death (Figures 1a f).                       mammalian cell cycle, 1% single-strand lesions are <br />           This lack apoptosis post-mitotic cell population                               converted 50 endogenous DSBs. <br />           (mid&#8211;late pachytene germ cells), UV-C lesions                                    Although attractive, model fully explain a <br />           converted DSBs replication, strongly                                number intriguing observations. First, previously <br />  <br /> Cell Death Differentiation <br />                                                                                        HR required UV-C-induced apoptosis C. elegans <br />                                                                                       L Stergiou et al <br />                                                                                                                                                                                     905 <br />  <br /> showed UV-C IR activate different sets checkpoint                         CPD staining quantification. A modified fixation protocol applied <br /> kinases.11 However, propose treatments                              preserve/create denatured DNA structures CPD staining, antibody <br /> activate apoptosis DSBs. Perhaps, nature DSBs                          specifically binds CPDs ssDNA.40 Adult hermaphrodites irradiated with <br />                                                                                       100 J/m2 UVC and, indicated time points, dissected M9 (0.5 mM <br /> or associated protein complexes different two <br />                                                                                       levamisole) extrude gonads fixed M9/3% PFA 30 min. Fixed worms <br /> cases. Second, showed XPG-1 required                             transferred poly-lysine-coated slides, freeze-cracked permeabilized in <br /> UV-C-induced apoptosis, needed generation                        PBS/Tween-20 0.1% 3   10 min. For DNA denaturation, slides incubated <br /> RAD-54-positive foci; furthermore, loss xpg-1 function                         PBS/0.07 N NaOH 8 min washed PBS/Tween-20 0.1% 3   10 min <br /> suppress increased apoptosis non-irradiated worms                              blocking (1 h RT) incubation 1 : 500 anti-CPD monoclonal <br /> defective rad-54. At level precisely XPG-1 functions                        antibody (MBL International, Woburn, MA, USA; product no. D194-1) 6 h <br />                                                                                       41C. Samples washed incubated (1 h RT) <span id='am-1' about='obo:NCBITaxon_149625' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_163739 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Alexa" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Alexa</span> Fluor 488 goat <br /> to promote apoptosis itself intriguing topic. <br />                                                                                       anti-rabbit (Invitrogen) secondary antibody (1 : 500) counterstained with <br />    Muzi-Falconi co-workers34,35 suggested func-                            DAPI (200 ng/ml) mounting Prolong Gold (Invitrogen). Blocking and <br /> tional link NER checkpoint activation                             antibody incubations antibody buffer/10% goat serum. <br /> yeast mammalian cells, processing UV lesions                             Fluorescent images captured described defined exposure <br /> by XPA-1 homolog enables 9-1-1 complex recruitment                                settings. For quantification CPD signal intensity, macro created run in <br /> and activation ATR kinase.34 Toussaint et al.36                           ImageJ (version 1.42q; http://rsb.info.nih.gov/nih-image/), allowing automated <br /> characterized co-participation HR components                                measurements <span id='am-50' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> region (ROI) immunofluores- <br />                                                                                       cence pictures. Briefly, ROI drawn approximately 100 germ cell <br /> rendering G2-phase yeast cells resistant UV                                   nuclei transition/early meiotic region. A mask defined binary <br /> irradiation. Recent evidence shift concept                               threshold image DAPI staining account DNA containing areas and <br /> individual repair pathways DNA damage                                applied anti-CPD image. Average anti-CPD signal intensity DAPI- <br /> response network, DNA repair factors interact                             positive areas compared average intensity background the <br /> drive coordinated response genome integrity cell                         ROI, expressed difference. <br /> survival,37 cell death. <br />    Our findings suggest existence cross-talk                         Foci quantification. Staged young adult hermaphrodites treated with <br />                                                                                       100 J/m2 UV-C light 120 Gy X-rays, gonads dissected at <br /> the NER HR repair pathways, assign role                          indicated time points. RAD-54::YFP RPA-1::YFP foci formation quantified <br /> DNA repair components apoptotic signaling. As repair                               counting number bright foci present mid&#8211;late pachytene germ cells, in <br /> pathways highly conserved evolution, inter-                           focal plane 100 mm oocyte. Fluorescent images were <br /> action NER HR pathways                                    captured Leica DMRA2 microscope equipped ORCA-ER digital CCD <br /> describe C. elegans occur vertebrates,                         <span id='am-38' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-39' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>camera</span> processed Openlab software. <br /> including humans. <br />                                                                                       RNAi constructs. Fragments corresponding exonic sequences xpb-1 <br />                                                                                       (Y66D12A.15) xpd-1 (Y50D7A.2) amplified PCR wild-type cDNA <br /> Materials Methods                                                                 library using primers listed Supplementary Table S2, cloned L4440 <br /> Genetics. All strains grown 201C NGM agar seeded <span id='am-43' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-44'  ></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia</span>          RNAi vector. The constructs used transform HT115(DH3) bacteria, which <br /> coli OP50. The Bristol N2 strain used wild-type strain. Mutant alleles     subsequently used feed worms RNAi experiments. For rest the <br /> used listed below: LGI: atm-1(gk186),11 hus-1(op244),38 xpg-1(tm1670) (this       RNAi clones study, used Ahringer RNAi library. A standard gfp <br /> study), xpg-1(tm1682) (this study), xpa-1(ok698),11 rad-54(ok615) (this study),       sequence cloned L4440 vector vector used controls the <br /> rad-54(tm1268) (this study); LGIV: xpc-1(ok734),11 spo-11(ok79);39 LGV:           RNAi experiments. <br /> atl-1(tm853),30 mre-11(ok179),12 chk-2(gk212).11 Transgenes: opIs257 (RAD- <br /> 54::YFP) opIs263 (RPA-1::YFP) (this study). Essential mutations              Transgenic worms. Genomic fragments corresponding rpa-1 rad-54 <br /> maintained balanced strains: rad-54(ok615)/hT2[qIs51](IV;V), rad-54(tm1268)/       promoter ORF, 30 -UTR region separately amplified PCR <br /> hT2[qIs51](IV;V), spo-11(ok79)/nT1[n754 let](IV;V), atl-1(tm853)/nT1[qIs51](IV;V)     N2 genomic DNA using primers (Supplementary Table S2) added the <br /> and mre-11(ok179)/nT1[n754 let](IV;V). Additional information mutations cited      appropriate restriction sites. The amplified fragments cloned pLN022 <br /> here obtained www.wormbase.org.                                             expression vector upstream yfp generate pLS69 (RPA-1::YFP) pLS57 <br />                                                                                       (RAD-54::YFP). Low-copy transgenic worms generated ballistic <br />                                                                                       transformation. Sequences used <span id='am-56' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span> available request. <br /> Germline apoptosis. Staged young adult worms (12 h post L4/adult molt) <br /> were exposed different doses UV-C light (254 nm) (J/m2) X-rays (Gy). <br /> A Stratalinker UV crosslinker, model 1800 (Stratagene, Basel, Switzerland)     Conflict interest <br /> Isovolt 160/225/320/450 HS X-ray machine (Rich. Seifert &amp; Co., Ahrensburg, <br /> Germany) used deliver appropriate doses. Corpses scored       The authors declare conflict interest. <br /> meiotic region gonad arm indicated time points using Nomarski optics. For <br /> the RNAi experiments, staged L1 larvae transferred plates containing <br /> 2 mM IPTG seeded E. coli strain HT115(DH3) expressing respective RNAi        Acknowledgements. We thank Professor H Naegeli Dr A Gartner for <br /> clone, scored young adults germline apoptosis.                             critical comments manuscript, members Hengartner lab for <br />                                                                                       discussions. This work supported Kanton Zurich, Swiss National <br />                                                                                       Science Foundation, Ernst Hadorn Foundation Josef Steiner Cancer <br /> Immunocytochemistry. For antibody staining gonads, young adult                     Research Foundation. RE supported MD-PhD fellowship Swiss <br /> hermaphrodites dissected fixed 3% PFA/0.1 M K2HPO4 (pH 7.2)           National Science Foundation. We thank Dr A Gartner providing the <br /> 50 min room temperature, followed 10-min incubation 100% methanol       RAD-51 antibody, Caenorhabditis Genetics Center, funded the <br /> ice. Gonads blocked 5% BSA/PBS&#8211;Tween-20 0.1% 1 h, followed             National Institute Health (NIH), National Center Research Resources (NCRR) <br /> incubation 1 : 50 anti-RAD-51 polyclonal15 antibody overnight 41C. Alexa      Shohei Mitani National Bioresource Project Japan strains. <br /> Fluor 594 goat anti-rabbit IgG (Invitrogen, Basel, Switzerland) used as <br /> secondary antibody (1 : 500). The tissues co-stained DAPI before <br /> mounting. Fluorescent images captured Leica DMRA2 microscope               1. Hanahan D, Weinberg RA. The hallmarks cancer. Cell 2000; 100: 57&#8211;70. <br /> equipped ORCA-ER digital CCD camera processed Openlab              2. Bartek J, Lukas J. DNA damage checkpoints: initiation recovery adaptation. <br /> software.                                                                                 Curr Opin Cell Biol 2007; 19: 238&#8211;245. <br />  <br />  <br />                                                                                                                                                           Cell Death Differentiation <br />                                        HR required UV-C-induced apoptosis C. elegans <br />                                                                              L Stergiou et al <br /> 906 <br />  <br />            3. Sancar A, Lindsey-Boltz LA, Unsal-Kacmaz K, Linn S. Molecular mechanisms                     22. Laat WL, Jaspers NG, Hoeijmakers JH. Molecular mechanism nucleotide excision <br />               mammalian DNA repair DNA damage checkpoints. Annu Rev Biochem 2004; 73:                     repair. Genes Dev 1999; 13: 768&#8211;785. <br />               39&#8211;85.                                                                                          23. Prakash S, Prakash L. Nucleotide excision repair yeast. Mutat Res 2000; 451: 13&#8211;24. <br />            4. Cleaver JE. Cancer xeroderma pigmentosum related disorders DNA repair.                24. Klungland A, Hoss M, Gunz D, Constantinou A, Clarkson SG, Doetsch PW et al. <br />               Nat Rev Cancer 2005; 5: 564&#8211;573.                                                                    Base excision repair oxidative DNA damage activated XPG protein. Mol Cell 1999; 3: <br />            5. Boer J, Hoeijmakers JH. Cancer outside, aging inside: mouse models             33&#8211;42. <br />               study consequences defective nucleotide excision repair. Biochimie 1999; 81:          25. Tang JY, Hwang BJ, Ford JM, Hanawalt PC, Chu G. Xeroderma pigmentosum p48 gene <br />               127&#8211;137.                                                                                            enhances global genomic repair suppresses UV-induced mutagenesis. Mol Cell 2000; <br />            6. Wijnhoven SW, Hoogervorst EM, Waard H, van der Horst GT, van Steeg H.                            5: 737&#8211;744. <br />               Tissue specific mutagenic carcinogenic responses NER defective mouse models.             26. Costa RM, Chiganc&#184;as V, Galhardo R, Carvalho H, Menck CF. The eukaryotic nucleotide <br />               Mutat Res 2007; 614: 77&#8211;94.                                                                         excision repair pathway. Biochimie 2003; 85: 1083&#8211;1099. <br />            7. van der Wees C, Jansen J, Vrieling H, van der Laarse A, Van Zeeland A, Mullenders L.            27. Cann KL, Hicks GG. Regulation cellular DNA double-strand break response. <br />               Nucleotide excision repair differentiated cells. Mutat Res 2007; 614: 16&#8211;23.                     Biochem Cell Biol 2007; 85: 663&#8211;674. <br />            8. Carvalho H, Ortolan TG, dePaula T, Leite RA, Weinlich R, Amarante-Mendes GP et al.              28. Lee JH, Paull TT. Activation regulation ATM kinase activity response DNA <br />               Sustained activation p53 confluent nucleotide excision repair-deficient cells resistant       double-strand breaks. Oncogene 2007; 26: 7741&#8211;7748. <br />               ultraviolet-induced apoptosis. DNA Repair 2008; 7: 922&#8211;931.                                  29. Wang XW, Vermeulen W, Coursen JD, Gibson M, Lupold SE, Forrester K et al. The <br />            9. Stergiou L, Hengartner MO. Death more: DNA damage response pathways                      XPB XPD DNA helicases components p53-mediated apoptosis pathway. <br />               nematode C. elegans. Cell Death Differ 2004; 11: 21&#8211;28.                                             Genes Dev 1996; 10: 1219&#8211;1232. <br />           10. Gartner A, Boag PR, Blackwell TK. Germline survival apoptosis. In: WormBook (ed).           30. <span id='am-6' about='obo:NCBITaxon_316718' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3646 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_235631 obo:NCBITaxon_3977 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_235892 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Garcia" datatype="xsd:string"></span><span id='am-10'  ></span>Garcia</span>-Muse T, Boulton SJ. Distinct modes ATR activation replication stress and <br />               The C. elegans Research Community, vol. 145.1. 2008, pp 1&#8211;20. http://www.                           DNA double-strand breaks Caenorhabditis elegans. EMBO J 2005; 24: 4345&#8211;4355. <br />               wormbook.org.                                                                                   31. Boyd WA, Crocker TL, Rodriguez AM, Leung MC, Lehmann DW, Freedman JH et al. <br />           11. Stergiou L, Doukoumetzidis K, Sendoel A, Hengartner MO. The nucleotide excision repair              Nucleotide excision repair genes expressed low levels detectably <br />               pathway required UV-C-induced apoptosis Caenorhabditis elegans. Cell Death                inducible Caenorhabditis elegans somatic tissues, function required for <br />               Differ 2007; 14: 1129&#8211;1138.                                                                         normal adult life UVC exposure. Mutat Res 2010; 683: 57&#8211;67. <br />           12. Chin GM, Villeneuve AM. C. elegans mre-11 required meiotic recombination DNA         32. Vilenchik MM, Knudson AG. Radiation dose-rate effects, endogenous DNA damage, and <br />               repair dispensable meiotic G DNA damage checkpoint. Genes Dev 2001; 15:              signaling resonance. Proc Natl Acad Sci USA 2006; 103: 17874&#8211;17879. <br />               522&#8211;534.                                                                                        33. Vilenchik MM, Knudson AG. Endogenous DNA double-strand breaks: production, fidelity of <br />           13. Petukhova G, Stratton S, Sung P. Catalysis homologous DNA pairing yeast Rad51                 repair, induction cancer. Proc Natl Acad Sci USA 2003; 100: 12871&#8211;12876. <br />               Rad54 proteins. Nature 1998; 393: 91&#8211;94.                                                    34. Giannattasio M, Lazzaro F, Longhese MP, Plevani P, Muzi-Falconi M. Physical and <br />           14. Sigurdsson S, Van Komen S, Petukhova G, Sung P. Homologous DNA pairing human                     functional interactions nucleotide excision repair DNA damage checkpoint. <br />               recombination factors Rad51 Rad54. J Biol Chem 2002; 277: 42790&#8211;42794.                          EMBO J 2004; 23: 429&#8211;438. <br />           15. Alpi A, Pasierbeck P, Gartner A, Loidl J. Genetic cytological charact <br /> </body></html>