<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Chin et al. Journal Biomedical Science 2013, 20:97 <br /> http://www.jbiomedsci.com/content/20/1/97 <br />  <br />  <br />  <br />  <br />  RESEARCH                                                                                                                                       Open Access <br />  <br /> Dissection <span id='am-35' about='obo:NCBITaxon_5475' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-36' property="rdfs:label" content="Candida" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Torulopsis" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Candida</span> albicans Cdc4 protein <br /> reveals involvement domains in <br /> morphogenesis cell flocculation <br /> Chuen Chin1, Wei-Chung Lai2, Tai-Lin Lee3, Tzu-Ling Tseng2 Jia-Ching Shieh2,4* <br />  <br />  <br />   Abstract <br />   Background: CDC4, encodes F-box protein member Skp1-Cdc53/Cul1-F-box (SCF) <br />   ubiquitin E3 ligase, initially identified budding yeast <span id='am-11' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-13'  ></span><span id='am-14' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> essential gene for <br />   progression G1-S transition cell cycle. Although <span id='am-41' about='obo:NCBITaxon_5476' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5475 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:12" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Candida stellatoidea type I" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Candida albicans" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasAlternativeId" content="NCBITaxon:39399" datatype="xsd:string"></span><span id='am-47'  ></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Candida stellatoidea" datatype="xsd:string"></span>Candida albicans</span> CDC4 (CaCDC4) release the <br />   mitotic defect caused loss CDC4 S. cerevisiae, CaCDC4 nonessential suppresses filamentation. <br />   Results: To elucidate function CaCDC4, C. albicans strain, CaCDC4 allele deleted the <br />   repressible C. albicans MET3 promoter (CaMET3p) control, introducing cassettes <br />   capable doxycycline (Dox)-induced expression various C. albicans Cdc4 (CaCdc4) domains. Cells each <br />   strain express specific CaCdc4 domain Dox-induced, CaMET3-CaCDC4 repressed conditions. Cells <br />   expressing domains F-box WD40-repeat exhibited filamentation flocculation similarly to <br />   lacking CaCDC4 expression, indicating functional essentiality F-box WD40-repeat. Notably, cells <br />   expressing N-terminal 85-amino acid truncated CaCdc4 partially reverse filament-to-yeast weaken the <br />   ability flocculate compared expressing full-length CaCdc4, suggesting N-terminal 85-amino acid <br />   CaCdc4 regulates morphogenesis flocculation. <br />   Conclusions: The F-box WD40-repeat CaCdc4 essential inhibiting yeast-to-filament transition and <br />   flocculation. The N-terminal region (1&#8211;85) CaCdc4 positive role function, lost impairs <br />   ability flocculate reverse filamentous growth C. albicans. <br />   Keywords: Candida albicans, CDC4 domains, Morphogenesis, Flocculation <br />  <br />  <br /> Background                                                                              Recently, F-box protein encoded C. albicans CDC4 <br /> Candida albicans natural diploid complete                             (CaCDC4) shown play role filamentous <br /> sexual cycle exists yeast, pseudohyphal, hyphal                           development [6,7]. Cdc4, originally identified bud- <br /> cells [1]. It capable morphological switch induced                        ding yeast <span id='am-28' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31'  ></span><span id='am-32' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae, encodes ubiquitin E3 <br /> environmental stimuli [2], essentially cAMP-mediated                             ligases, belongs member Skp1-Cdc53/ <br /> and MAPK signaling pathways [3]. Importantly, ability                            Cul1-F-box (SCF) complex. <span id='am-54' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-55' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58'  ></span>This</span> complex known to <br /> to alter morphology cell types associated                              play role ubiquitin-proteasome dependent degrad- <br /> virulence humans [4]. Many cell cycle regulators includ-                          ation regulatory proteins eukaryotes [8]. A specific <br /> ing cyclins known control morphogenesis                               SCF complex designated associated F-box protein. <br /> C. albicans [5].                                                                     This protein variable interacting domains of <br />                                                                                      F-box Skp1 WD40-repeat (or LRR) specific <br />                                                                                      substrates [9], Cdc4 named SCFCdc4. To <br /> * Correspondence: jcs@csmu.edu.tw <br />                                                                                      progress G1-S transition S. cerevisiae, <br /> 2 <br />  Department Biomedical Sciences, Chung Shan Medical University, No.               SCFCdc4 required degrade Sic1 [10] Far1 [11], <br /> 110, Sec. 1, Jianguo N. Road, Taichung City 40201, Taiwan                            cyclin-dependent kinase inhibitors. Therefore, <br /> 4 <br />  Department Medical Research, Chung Shan Medical University Hospital, <br /> Taichung City, Taiwan <br />                                                                                      S. cerevisiae CDC4 (ScCDC4) essential S. cerevisiae. <br /> Full list author information available end article <br />  <br />                                        &#169; 2013 Chin et al.; licensee BioMed Central Ltd. This Open Access article distributed terms Creative <br />                                        Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, and <br />                                        reproduction medium, provided original work properly cited. The Creative Commons Public Domain Dedication <br />                                        waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies <span id='am-68' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-69' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-70'  ></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> available article, unless otherwise <br />                                        stated. <br />  Chin et al. Journal Biomedical Science 2013, 20:97                                                     Page 2 11 <br /> http://www.jbiomedsci.com/content/20/1/97 <br />  <br />  <br />  <br />  <br />    Although CaCdc4 structural homolog S. cerevi-     50 &#956;g/ml uridine 1 mg/ml 5-fluoroorotic acid (5- <br /> siae Cdc4 (ScCdc4) capable rescuing mi-        <span id='am-6' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="rdfs:label" content="Foa" datatype="xsd:string"></span>FOA</span>, MD Bio). To repress CaCDC4 expression that <br /> totic defect caused loss ScCDC4 S. cerevisiae   controlled CaMET3p, strains grown SD <br /> [7], functions CaCdc4 ScCdc4 dissimilar       medium plates 2.5 mM Met/Cys, has <br /> as null Cacdc4 mutant viable depletion     shown optimally switch expression the <br /> CaCdc4 causes accumulation Sol1 (Sic1 like)       CaMET3p-driven downstream gene [16]. To induce <br /> hyphal development initiation cell cycle      gene expression Tet-on system, 40 &#956;g/ml Dox <br /> arrest [6]. This verifies CaCDC4 nonessential    (Sigma) added YEPD SD media. <br /> suppresses filamentation suggests controlling <br /> the degradation Sol1 C. albicans CaCdc4 im-      Plasmid DNA manipulation <br /> portant inhibition filamentation. Therefore,    Plasmid DNA extracted routinely E. coli cul- <br /> C. albicans Sol1 likely substrate SCFCaCdc4,   tures using Gene-SpinTM MiniPrep purification Kit-V2 <br /> can demonstrated reduction Sol1            (PRO TECH, Taipei, Taiwan) instructions pro- <br /> CaCdc4 overexpressed [6], dir-     vided manufacturer. E. coli transformed with <br /> ect evidence support hypothesis. Additionally,   plasmid DNA using CaCl2. The DNA cassettes were <br /> filamentous properties mutants Cacdc4 null        introduced C. albicans lithium acetate <br /> Cacdc4 sol1 double null comparable. This refutes        method described previously [17]. <br /> the <span id='am-49' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-50' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-53'  ></span>idea</span> Sol1 sole target CaCdc4. Indeed, <br /> with affinity-purification approach, isolated     Construction C. albicans strains <br /> at novel CaCdc4-associated proteins [12]      Initially, strain repressed CaCDC4 expression was <br /> are potential substrates CaCdc4.                          made. A mini-Ura-blaster cassette, flanked 60-bp <br />    To elucidate role CaCDC4 medi-     sequences homologous CaCDC4, PCR-amplified <br /> ation characteristic F-box protein SCF ubi-     using template plasmid pDDB57 long primers <br /> quitin E3 ligase C. albicans, sought dissect   CaCDC4-URA3-F CaCDC4-URA3-R (Table 1). <br /> the CaCdc4 domains associated filamentation. In         BWP17 transformed integration cassette <br /> this study, C. albicans strain deleted    CaCDC4 locus generate Ura+ strain <br /> CaCDC4 allele repressed CaMET3              JSCA0018. The plasmid pFA-HIS1-MET3p-CaCDC4, <br /> promoter (CaMET3p) using methionine cysteine             partial CaCDC4 coding sequence N-terminal <br /> (Met/Cys). We used strain introduce <span id='am-73' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-74' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span>         CaCdc4 (1&#8211;563), linearized BspEI used <br /> capable inducing expression various CaCdc4 do-         transform JSCA0018 generate His+ JSCA0021 <br /> mains doxycycline (Dox). We observed roles       (Figure 1A; Table 1). Cells JSCA0021 plated with <br /> F-box WD40-repeat CaCdc4 function            5-FOA induce recombination copies of <br /> possible role N-terminal 85-amino acid <span id='am-63' about='obo:NCBITaxon_64446' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_442183 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_127263 obo:NCBITaxon_33415 obo:NCBITaxon_42282 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="Morpho" datatype="xsd:string"></span><span id='am-67'  ></span>morpho</span>-    dpl200 flanking mini-Ura-blaster loss of <br /> genesis. We showed C. albicans cells lacked   CaURA3 generate JSCA0022. <br /> CaCdc4 triggered flocculation. Moreover,         To allow expression cassettes encoding assorted <br /> N-terminal 85-amino acid CaCdc4 required in-       CaCdc4 domains C. albicans, Tet-on plasmid, <br /> hibition filamentation flocculation.             pTET25M [18], derived pTET25 [19] for <br />                                                              inducing gene expression Dox, developed. <br /> Methods                                                      To regulate CaCDC4 expression Tet-on system, <br /> Strains growth conditions                                coding sequence CaCDC4 PCR-amplified <br /> E. coli strain DH5&#945; used routine manipula-       using plasmid CaCDC4-SBTA bearing CaCDC4 (Lai <br /> tion plasmids. They grown 37&#176;C LB          WC, unpublished results), primers CaCDC4-SalI and <br /> broth medium [13] plates containing 1.5% agar          CaCDC4-BglII (Table 2), Pfu polymerase (5 U/&#956;l, <br /> (Difco, BD Biosciences), 50 &#956;g/ml ampicillin         MD bio), digested SalI BglII cloning into <br /> 30 &#956;g/ml kanamycin. All C. albicans strains (Table 1)        pTET25M, pTET25M-CaCDC4 gener- <br /> were derived auxotrophic strain BWP17 (arg4/arg4        ated. Moreover, CaCDC4-6HF, encodes 6&#215;histi- <br /> his1/his1 ura3/ura3) [14]. They grown 30&#176;C        dine FLAG (6HF) tags C-terminal CaCdc4, <br /> either yeast extract-peptone-dextrose (YEPD) supple-      PCR-amplified primers CaCDC4-6HF SalI and <br /> mented minimal synthetic defined (SD) medium            CaCDC4-6HF BglII (Table 2), followed digestion with <br /> 2% glucose 2% agar [15]. While Ura+          SalI BglII cloning pTET25M obtain <br /> prototrophs selected SD agar plates uri-     pTET25M-CaCDC4-6HF. <br /> dine, His+ prototrophs selected SD plates with-        To define function distinct CaCdc4 domains <br /> out histidine. Selection loss C. albicans     (Figure 2A), different CaCDC4 portions used to <br /> URA3 (CaURA3) marker performed plates            replace length CaCDC4 coding sequence on <br />  Chin et al. Journal Biomedical Science 2013, 20:97                                                                               Page 3 11 <br /> http://www.jbiomedsci.com/content/20/1/97 <br />  <br />  <br />  <br />  <br /> Table 1 Candida albicans strains used study <br /> Systemic strain       Parental strain       Name relevant genotype      Genotype <br /> BWP17                                                   CaCDC4 +/+                     ura3::imm434/ura3:imm434 his1::hisG / his1::hisG <br />                                                                                        arg4::hisG/arg4::hisG <br /> JSCA0018                          BWP17                 CaCDC4 +/U3&#8722;                   CaCDC4/cdc4::CaURA3-dpl200 <br /> JSCA0021                          JSCA0018              CaCDC4 M3/U3&#8722;                  Cacdc4::URA3-dpl200/PMET3-CaCDC4:HIS1 <br /> JSCA0022                          JSCA0021              CaCDC4 M3/&#8722;                    Cacdc4::dpl200/PMET3-CaCDC4:HIS1 <br /> JSCA0023                          JSCA0022              CaCDC4 M3/&#8722; &#8739; Tet-CaCDC4       Cacdc4::dpl200/PMET3-CaCDC4:HIS1 &#8739; CaADH1/adh1::PTET- <br />                                                                                        CaCDC4:CaURA3 <br /> JSCA0024                          JSCA0022              CaCDC4 M3/&#8722; &#8739; Tet-CaCDC4-6HF   Cacdc4::dpl200/PMET3-CaCDC4:HIS1 &#8739; CaADH1/Caadh1::PTET- <br />                                                                                        CaCDC4-6HF:CaURA3 <br /> JSCA0025                          JSCA0022              CaCDC4 M3/&#8722; &#8739; Tet-&#916;N-6HF       Cacdc4::dpl200/PMET3-CaCDC4:HIS1 &#8739; CaADH1/Caadh1::PTET <br />                                                                                        -CaCDC4(85&#8211;768)-6HF:CaURA3 <br /> JSCA0026                          JSCA0022              CaCDC4 M3/&#8722; &#8739; Tet-F-box-6HF    Cacdc4::dpl200/PMET3-CaCDC4:HIS1 &#8739; CaADH1/Caadh1::PTET <br />                                                                                        -CaCDC4(241&#8211;392)-6HF:CaURA3 <br /> JSCA0027                          JSCA0022              CaCDC4 M3/&#8722; &#8739; Tet-WD40-6HF     Cacdc4::dpl200/PMET3-CaCDC4:HIS1 &#8739; CaADH1/adh1::PTET <br />                                                                                        -CaCDC4(393&#8211;768)-6HF:CaURA3 <br /> JSCA0030                          JSCA0022              CaCDC4 M3/&#8722; &#8739; Tet-&#916;NF-6HF      Cacdc4::dpl200/PMET3-CaCDC4:HIS1 &#8739;CaADH1/Caadh1::PTET <br />                                                                                        -CaCDC4(85&#8211;392)-6HF:CaURA3 <br />  <br />  <br />  <br /> pTET25M-CaCDC4-6HF. By using primer sets                               CaCDC4_Probe_F CaCDC4_Probe_R CaCDC4 <br /> listed Table 2, following constructs made:                     locus CaADH1 Probe_F CaADH1 probe_R for <br /> pTET25M-&#916;NCaCDC4-6HF (with primers CaCDC4 &#916;N                               ADH1 locus (Table 2) using DIG Easy Hyb (Roche). To <br /> AatII CaCDC4 &#916;N XhoI), encodes N-                            reveal structure gene locus, DIG Luminescent <br /> terminal truncated CaCdc4; pTET25M-F-6HF (with                             Detection Kit (Roche) used hybridization, and <br /> primers CaCDC4 F-box AatII CaCDC4 F-box                                luminescent images blot captured the <br /> XhoI), encodes F-box domain flanking                        imaging analysis (ImageQuant LAS4000 mini, <br /> regions; pTET25M-WD40-6HF (with primers CaCDC4                             GE Healthcare Life Sciences). <br /> WD40 AatII CaCDC4 &#916;N XhoI), encodes <br /> eight copies WD40-repeat; pTET25M-&#916;NF-6HF                           Protein extraction Western blot analysis <br /> (with primers CaCDC4 &#916;N AatII CaCDC4 F-box                             Cultured cells collected, total protein from <br /> XhoI), encodes truncated N-terminal CaCdc4                       sample extracted described previously [20]. <br /> the F-box domain. All inserts constructs                       The proteins resolved 10% SDS-PAGE and <br /> released AatII XhoI replace full-length                    transferred PVDF membranes (PerkinElmer, Boston, <br /> CaCDC4 pTET25M-CaCDC4-6HF. Consequently,                                USA). Proteins membranes probed with <br /> plasmids bearing CaCDC4 segments flanked                             polyclonal antibody FLAG (Sigma) 1:2000 dilution <br /> with common C. albicans ADH1 (CaADH1) sites                           detected using SuperSignal West Pico Chemilu- <br /> digested SacII KpnI,                            minescent Substrate Kit (PIERCE). These recorded <br /> transformed C. albicans integration                        Luminescent Image Analyzer (FUJIFILM LAS- <br /> CaADH1 locus. All strains verified colony                          1000) analyzed ImageGauge 3.46 L Process v <br /> PCR specific primers subjecting Southern                    1.96 (FUJIFILM). <br /> blotting analysis. <br />                                                                            Flocculation assay low-speed centrifugation <br /> Southern blotting analysis                                                 The cells strains streaked YPD agar plate for <br /> Genomic DNA C. albicans strains isolated                      3 days colonies picked inoculated SD <br /> by MasterPure&#8482; Yeast DNA Purification Kit (Epi-                        medium required supplements 48 hrs. Next, the <br /> centre&#174;, Illumina company) according manu-                       cultures diluted fresh SD medium 0.1 of <br /> facture&#8217;s instruction. Southern blotting performed                     initial OD600 required supplements. To simultan- <br /> with aid Rapid Downward Transfer System                         eously repress expression CaMET3p-driven <br /> (TurboBlotter&#8482;, Whatman) using 10 &#956;g restriction                    CaCDC4 induce expression various CaCDC4 <br /> enzyme-digested genomic DNA. The DNA blot                           segments encoding series CaCdc4 domains, 2.5 mM <br /> was hybridized probe amplified PCR                           Met/Cys 40 &#956;g/ml Dox added SD <br /> DIG probe synthesis kit (Roche) primers                           medium. After 48 hrs, cultures spun for <br />  Chin et al. Journal Biomedical Science 2013, 20:97                                                                                                 Page 4 11 <br /> http://www.jbiomedsci.com/content/20/1/97 <br />  <br />  <br />  <br />  <br />   A <br />                                                               pDDB57 <br />                                                                                                                    pFA-HIS1- <br />                                                                                                                  MET3p-CaCDC4 <br />                                                                CaURA3 <br />  <br />                                                                dpl200                                                                    BspEI <br />                                                                   PCR <br />  <br />                                                     mini Ura-blaster deletion                                      Transformation <br />                                                   cassette <br />                                                           Transformation <br />                                                                                                                  Amp        HIS1      MET3 p <br />  <br />                                                                CaCDC4 <br />                 BWP17 (CaCDC4 +/+) <br />                                                                 CaCDC4 <br />                                                             Recombination <br />  <br />                 JSCA0018 <br />                 (CaCDC4 M3/U3 ) <br />                                                                                                                       Recombination <br />  <br />                                   Amp         HIS1        MET3 p <br />                                                                                                                 Amp         HIS1      MET3 p <br />  <br />                 JSCA0022 <br />                 (CaCDC4 M3/ )                                                                              JSCA0021 <br />                                                                                           5-FOA <br />                                                                                                            (CaCDC4 M3/U3 ) <br />  <br />  <br />   B                                Probe <br />       NdeI                NdeI                                                               NdeI <br />                                                          CaCDC4 <br />                                                               14 kb <br />                      NdeI     Probe           NdeI                                           NdeI     kb <br />                                                                                                       14                                          CaCDC4 <br />                                  4.5 kb <br />                                               Probe                                                   10                                          Cacdc4::dpl200 <br />                                     NdeI                                                     NdeI <br />                                                                                                        8                                          CaCDC4 <br />                                                                          13.5 kb                       6                                          Cacdc4::PMET3 - <br />                                                                                                                                                   CDC4:HIS1 <br />       NdeI                NdeI     Probe                                           NdeI      NdeI      5 <br />                                                          CaCDC4                                        4                                         Cacdc4::URA3- <br />                                                      8.5 kb <br />                                                                                                                                                  dpl200 <br />                                                                                                        3 <br />                           partial CaCDC4 <br />          NdeI     Probe                    NdeI                                           NdeI <br />                                   Amp             HIS1     MET3 p <br />  <br />                      7.4 kb <br />  <br />  Figure 1 Construction C. albicans strain repressibly expressing CaCDC4. (A) Strain construction (detailed Methods). The first <br />  CaCDC4 allele BWP17 deleted mini-Ura-blaster obtain JSCA0018. Plasmid pFA-HIS1-MET3p-CaCDC4 containing partial CaCDC4 <br />  coding sequence linearized unique site introducing strain JSCA0018 generate JSCA0021. 5-FOA used counter-select <br />  CaURA3 removal obtain JSCA0022 re-introducing Tet-on plasmid CaURA3 marker. (B) Verification constructed strains Southern blotting <br />  analysis. Organization CaCDC4 locus respect NdeI sites shown. The relative positions probe used predicted NdeI-digested pat- <br />  tern CaCDC4 locus indicated. Two NdeI-fragments 14 kb 8.5 kb, specific CaCDC4, detected genomic DNA BWP17 <br />  digested NdeI; NdeI-fragments 8.5 kb 4.5 kb, specific CaCDC4 Cacdc4::URA3-dpl200, respectively, detected JCSA0018; two <br />  NdeI-fragments 4.5 kb 7.4 kb specific Cacdc4::URA3-dpl200 Cacdc4::PMET3 -CDC4:HIS1, respectively, detected JSCA0021; two <br />  NdeI-fragments 13.5 kb 7.4 kb specific Cacdc4::dpl200 Cacdc4::PMET3 -CDC4:HIS1, respectively, detected JSCA0022. A non-specific <br />  NdeI-fragment indicated &#8220;*&#65533;? detected strains tested. <br />  <br />  <br />  <br /> 1 minute 500 rpm, suspensions cultures                                  [22]. Since FLO11-associated flocculation dependent on <br /> were sampled determine optical density OD600.                                 presence Ca2+, adopted alternative floccula- <br /> Three independent assays conducted sam-                                     tion assay rate flocculation initiated by <br /> ple assayed duplication. A paired Student t test                              Ca2+ optical density assessed short <br /> p &lt; 0.05 considered significance.                                                     time-frame [23]. Briefly, initiate flocculation, aliquot <br />                                                                                           800 &#956;l deflocculated cell suspension transferred <br /> Ca2+-initiated flocculation assay                                                         1-ml cuvette, followed addition 200 &#956;l of <br /> The FLO-encoded flocculins known essential                                  100 mM CaCl2. The cuvette mixed robustly pipet- <br /> flocculation S. cerevisiae [21]. Functional homologues                                 ting absorbance (OD600) assessed instantly at <br /> of FLO genes C. albicans. In particular,                               30-s intervals 5 minutes using spectrophotometer <br /> the important S. cerevisiae gene FLO11 responsible                                    (DU800, Beckman Coulter, Inc.). All assays con- <br /> flocculation C. albicans functional counterpart ALS1                                  ducted triplicate. <br />  Chin et al. Journal Biomedical Science 2013, 20:97                                                                              Page 5 11 <br /> http://www.jbiomedsci.com/content/20/1/97 <br />  <br />  <br />  <br />  <br /> Table 2 Oligonucleotides used study                                       Tet-on cassettes CaURA3 selectable marker <br /> Name                        Sequencea                                             mini-Ura-blaster JSCA0021, 5-FOA used as <br /> CaCDC4 XhoI F               GAACTCGAGATGGATAAGAAATCAAAG                           counter-selection agent remove CaURA3 from <br /> CaCDC4 XhoI R               GAACTCGAGCTGTAAAAGTGGTTGACT <br />                                                                                   JSCA0021, JSCA0022 obtained (Figure 1A; <br />                                                                                   Table 1). The strains PCR-confirmed specific <br /> CaCDC4 SalI                 TAGCGTCGACATGGATAAGAAATCAAAGC <br />                                                                                   primers subjecting Southern blotting analysis. <br /> CaCDC4 BglII                TCGAGATCTTCACTGTAAAAGTGGTTGAC                         The CaCDC4 locus BWP17 strain detect two <br /> CaURA3-dpl200 BamHI AATGGATCCCCAGATATTGAAGGTAAAAGG                                NdeI-digested fragments size 14 kb 8.5 kb, re- <br /> CaURA3-dpl200 XhoI          ATTCTCGAGCTAGAAGGACCACCTTTGAT                         spectively (Figure 1B). The size shifting NdeI-fragment <br /> TET25M KpnI                 CAAGGTACCGAACCATCGTGAGTGTAA                           flanking CaCDC4 14 kb 4.5 kb demonstrated that <br /> TET25M BamHI                GAAGGATCCCGACATTTTATGATGGAA <br />                                                                                   CaCDC4 allele integrated mini-Ura- <br />                                                                                   blaster cassette strain JSCA0018 (Figure 1B). The size <br /> CaCDC4-6HF SalI             GCGTGTCGACGTCATGGATAAGAAATCAAAGCTA <br />               b <br />                                                                                   shifting NdeI-fragment flanking CaCDC4 8.5 kb <br /> CaCDC4-6HF BglII            TCGAGATCTttatttatcatcatcatctttataatcACCACC            7.4 kb demonstrated CaCDC4 allele inte- <br />                             gtggtggtggtggtggtgCTCGAG CGGCCGCTGTAA <br />                             AAGTGGTTGACTGAAATC                                    grated MET3-diven CaCDC4 plasmid strain <br /> CaCDC4 &#916;N AatII             AATAGACGTCCTTATGCCCTCATGTGACGAC <br />                                                                                   JSCA0021 (Figure 1B). Strain JSCA0021 further <br />                                                                                   popped mini-Ura-blaster cassette obtain strain <br /> CaCDC4 &#916;N XhoI              ATCCTCGAGCTGTAAAAGTGGTTGACTGA <br />                                                                                   JSCA0022 size shifting NdeI-fragment <br /> CaCDC4 F-box AatII          AAGCGACGTCATGAGCAATGAACCTACT                          flanking CaCDC4 4.5 kb 13.5 kb (Figure 1B). <br /> CaCDC4 F-box XhoI           GCCACTCGAGCCACCTATTGACAATTAT                          These results indicate strains constructed ex- <br /> CaCDC4 WD40 AatII           GCTAGACGTCATGGATCCAAAGTTCAAAC                         pected organizations genome. <br /> CaCDC4-URA3-F               ATGGATAAGAAATCAAAGCTATTCAAATATCCTTT <br />                             GAGCGAGGAGACGGCTAAATTTGAGGTTTTCCCA                    Phenotypic verification C. albicans strains capable of <br />                             GTCACGACGTT                                           conditionally repressing expression CaCDC4 <br /> CaCDC4-URA3-R               TCACTGTAAAAGTGGTTGACTGAAATCTAGAATCT                   It shown Ura&#8212; auxotrophic mutants are <br />                             CAATAAACGTTTCACCTTCATCTTCTGTGGAATTGT <br />                             GAGCGGATA <br />                                                                                   avirulent [25] virulence-associated features can <br />                                                                                   influenced level CaURA3 gene expression <br /> CaADH1_probe_F              GGAGTATTGGCATTGTTGGG <br />                                                                                   [26]. To assess presence CaURA3 having effect on <br /> CaADH1_probe_R              AAGCTTGCTTGCATGACGAG                                  yeast-to-filament transition, yeast-to-filament transi- <br /> CaCDC4_probe_F              GGTTTCCAACACTTTCCCAG                                  tions strain JSCA0021 JSCA0022 com- <br /> CaCDC4_probe_R              CACTACTAGTTGGTTGCTGT                                  pared, cells strains assessed CaMET3p <br /> a <br />  Restriction enzyme sites italics.                                         repressed de-repressed conditions. Cells strains <br /> b <br />   Sequences complementary encoding 6&#215;His FLAG lower           SD plates Met/Cys grew circular colonies <br /> case letters. The italics used restriction enzymes note &#8220;a&#65533;?. <br /> The underline new replaced lower case letters.                            smooth surfaces (Figure 2). By contrast, cells plates <br />                                                                                   Met/Cys formed irregular colonies filaments <br /> Results                                                                           (Figure 2). Under microscope, strains exhibited <br /> Constructing C. albicans strain capable conditionally                        equivalent filamentous forms, suggesting comparable <br /> repressing expression CaCDC4                                               ability deplete CaCDC4 expression inability of <br /> To establish C. albicans strains capable expressing                            CaURA3 interfering yeast-to-filament transition in <br /> CaCDC4 domains solely controlled Tet                              C. albicans. Subsequently, JSCA0022 used paren- <br /> promoter directly C. albicans, BWP17, alleles                        tal strain introduce Tet-on cassettes (with CaURA3 <br /> of CaCDC4 deleted, constructed accommodate                                 marker) encoded assorted CaCdc4 domains. <br /> Tet-on plasmid cassettes capable expressing assorted <br /> CaCdc4 domains induced Dox. The allele                                Establishment Tet-on cassettes capable expressing <br /> CaCDC4 deleted BWP17 mini-Ura-blaster [24]                              assorted CaCDC4 domains C. albicans reveals both <br /> to generate JSCA0018 strain (Figure 1A; Table 1). This                        F-box WD40-repeat required CaCdc4 <br /> strain used delete second CaCDC4 allele ob-                         function <br /> tain Cacdc4 null mutant. However, Cacdc4 null mutant                            The filamentous development JSCA0022 under <br /> cells growing filamentous form toughened cell                             CaMET3p-CaCDC4 repressed conditions, Met/Cys <br /> walls obstructed transformation.                                                  Tet-on system, allows study function <br />   To overcome problem, strain JSCA0021                                   CaCdc4 domains. A set Tet-on cassettes <br /> (Figure 1A; Table 1) created CaCDC4 al-                          (obtained pTET25M-CaCDC4-6HF, pTET25M- <br /> lele deleted CaMET3 control                              &#916;N-6HF, pTET25M-F-box-6HF, pTET25M-WD40-6HF, <br /> was Met/Cys repressible. To allow introduction                             pTET25M-&#916;NF-6HF) encoded the <br />  Chin et al. Journal Biomedical Science 2013, 20:97                                                                                  Page 6 11 <br /> http://www.jbiomedsci.com/content/20/1/97 <br />  <br />  <br />  <br />  <br />                                           Met/Cys                                            +Met/Cys <br />                        JSCA0021                  JSCA0022                   JSCA0021                   JSCA0022 <br />  <br />  <br />  <br />  <br />  Figure 2 Morphological analysis constructed CaCDC4 repressible strains. Cells strains JSCA0021 JSCA0022 grown SD <br />  medium plates (+) (&#8722;) Met/Cys. Colonies photographed MEIJI stereoscopic microscope EMZ5 40&#215; magnification <br />  (top panel). Cells liquid culture visualized recorded Nikon 50i microscope 400&#215; magnification (bottom panel). Bars represent <br />  10 &#956;m. <br />  <br />  <br />  <br /> assorted domains CaCdc4 (Figure 3A) used                            expression similar strains expressing the <br /> to transform JSCA0022 (which contained CaMET3p-                           domains, BWP17 JSCA0021 parental <br /> repressible CaCDC4) Ura+ integration                           strain (Lai WC, unpublished results). Therefore, even <br /> CaADH1 locus (Figure 3B). The correctness                            strains expressed domains un- <br /> strains confirmed yeast colony PCR specific                     expected size, unique negative con- <br /> primers Southern blotting analysis. The CaADH1                       trol JSCA0022. We concluded Tet-on system <br /> locus strain JSCA0022 detect SpeI-digested                     functions JSCA0022 CaCdc4 be <br /> fragment size 3.3 kb (Figure 3C). The CaADH1                        undergoing undefined modifications. <br /> locus strains JSCA0023 JSCA0024 detected                           To determine function assorted CaCdc4 <br /> increased SpeI-digested fragment 9.4 kb                       domains, JSCA0022-based strains capable repres- <br /> integration Tet-on cassettes pTET25M-                          sing CaCDC4 inducing expression assorted <br /> CaCDC4 pTET25M-CaCDC4-6HF (Figure 3C). The                               CaCdc4 domains grown SD medium or <br /> CaADH1 locus strains showed expected                        Met/Cys presence absence of <br /> alteration size according size different                       Dox. Cells strains SD medium Met/ <br /> CaCDC4 domains (Figure 3C). These results confirmed                         Cys grew yeast presence absence Dox <br /> the correctness strains.                                             (Figure 4B). By contrast, cells strains medium <br />   The JSCA0022 strain, expressed non-tagged                       Met/Cys grew filaments (Figure 4B). As ex- <br /> and repressible CaCdc4, used negative control.                     pected, cells JSCA0023 JSCA0024 growing on <br /> The sample obtained JSCA0022 contained                             medium Met/Cys Dox expressed the <br /> prominent proteins approximately 55 kDa 72 kDa                       full-length CaCdc4 tag grew yeast. <br /> (Figure 4A) presumably result cross-                        Disregarding full-length CaCdc4, cells all <br /> reactivity anti-FLAG antibody. Those proteins                    strains, JSCA0025 expressing assorted domains, <br /> were used internal control. The F-box WD40-                       grew filaments (Figure 4B). Under Met/Cys and <br /> repeat proteins strains JSCA0026 JSCA0027                          Dox conditions, cells JSCA0025 expressing the <br /> migrated expected positions approximately                       N-terminal 85-amino acid truncated CaCdc4 seemed <br /> 19 kDa 43 kDa (Figure 4A), respectively. However,                       ability suppress filamentation not <br /> the full-length CaCdc4 N-terminus truncated                         complete yeast form (Figure 4B). This in <br /> CaCdc4 (&#916;N) strains JSCA0024 JSCA0025 ex-                          consistent previous observation which, <br /> hibited signals positions corresponding 100 kDa                       comparing cells capable expressing full- <br /> and 100 kDa (Figure 4A), respectively, opposed                      length CaCdc4 CaMET3p repressible control, <br /> to 86 kDa 77 kDa, respectively. Three distinctive                       cells expressing N-terminal 85-amino acid <br /> signals (Figure 4A) observed strain JSCA0030                       truncated CaCdc4 lagged reaching exponen- <br /> expressing &#916;NF CaCdc4, matched                          tial stage (Additional file 1: Figure S1) converted to <br /> the expected size 34 kDa; however, signal                     filamentous form earlier (Additional file 2: Figure S2) in <br /> lowest position meaningful. These patterns                      repressed condition. <br />  Chin et al. Journal Biomedical Science 2013, 20:97                                                                                           Page 7 11 <br /> http://www.jbiomedsci.com/content/20/1/97 <br />  <br />  <br />  <br />  <br />                                                                                                          Predicted       Strain     Predicted Tet- <br />   A                                                                                                     Protein size   (JSCA00)      cassette <br />                                                                         F-box         WD40-repeat          (kDa)                      size (kb) <br />  <br />            CaCdc4        1-768                                                                               86           24 (23)        9.4 <br />  <br />  <br />                N       85-768                                                                                77           25             9.15 <br />  <br />                                                                                                              19           26             7.55 <br />            F-box      241-392 <br />  <br />            WD40       393-768                                                                                43           27             8.23 <br />  <br />  <br />                NF      85-392                                                                                34           30             8.02 <br />  <br />  <br />  <br />  <br />   B <br />                     pTET25M-CaCDC4-6HF <br />   SpeI SacII                                             XhoI          AatII    KpnI SpeI <br />         CaADH1 rtTA   ACT1T    URA3     ACT1T   FLAG   6xHis    CaCDC4 tetO CaADH1 <br />  <br />                                                                                             C                 JSCA00 <br />                                                                                             kb      21 22 23 24 25 26          27 30 <br />                                       CaADH1                                                14 <br />                                                                                             10                                             Caadh1::PTET - <br />                                       CaADH1 <br />                                                                                              8                                             CDC4:URA3 <br />                               Probe                                                          6 <br />                       SpeI                                      SpeI <br />                                                                                              5 <br />                                       CaADH1                                                 4                                             CaADH1 <br />                                       3.3 kb                                                 3 <br />   SpeI Probe                                                                         SpeI <br />         CaADH1 rtTA   ACT1T    URA3     ACT1T   FLAG   6xHis    CaCDC4 tetO CaADH1 <br />  <br />  <br />                                          9.4 kb <br />  Figure 3 Construction C. albicans strains Dox-inducing expression assorted CaCDC4 domains. (A) Schematic representation <br />  CaCdc4 domains expressed Tet-on system. The strains derived predicted Tet-on cassette size shown <br />  (B) Generation Tet-on cassettes expressing assorted CaCdc4 domains. Different portions CaCDC4 PCR-generated primer sets <br />  (Table 2) containing common AatII XhoI sites replacing full-length CaCDC4 pTET25M-CaCDC4-6HF described Methods. By <br />  digestion SacII KpnI, cassette used transform C. albicans strain JSCA0022 integration CaADH1 locus. (C) Verification <br />  Tet-on cassettes integrated CaADH1 locus Southern blotting analysis. Organization CaADH1 locus respect SpeI sites is <br />  shown Figure 3B. The relative positions probe used predicted SpeI-digested pattern CaADHI locus indicated Figure 3B. <br />  One SpeI-fragment 3.3 kb specific CaADH1 locus detected genomic DNA strain JSCA0022 derivatives digested SpeI. <br />  The correctness integration cassette CaADH1 locus various strains confirmed alteration SpeI-fragment size of <br />  3.3 kb 9.4 kb (Figure 3B) various sizes indicated Figure 3A. <br />  <br />  <br />  <br /> C. albicans CDC4 negatively regulating cell flocculation                               sampled determination optical density. Cells <br /> Significant differences ability strains form                           resisted flocculation remain suspension upon <br /> suspensions (to resist flocculation) observed. The ex-                            centrifugation. Under CaMET3p de-repressed condi- <br /> tent flocculation strains observed                                  tion presence absence Dox, strains <br /> resuspending cells cuvettes, remained                                exhibited similar degree suspension. However, under <br /> for 30 seconds. When cells grown Met/                                   CaMET3p repressed condition, JSCA0026, JSCA0027, <br /> Cys Dox conditions, JSCA0023                                   JSCA0030 displayed flocculation similar JSCA0022 <br /> JSCA0024 somewhat easier maintain suspen-                                 regardless presence absence Dox (Figure 5A). <br /> sion. To exclude possibility result                             While cells strains JSCA0023, JSCA0024 maintained <br /> increases cell density, cells strains initially                       suspension, JSCA0025 filament- <br /> grown saturation, cultures subsequently                                ous cells, showed comparable extent flocculation to <br /> diluted initial optical density grown expo-                            JSCA0022 CaMET3p repressed Tet-on in- <br /> nentially similar optical density. The extent floccula-                          duced conditions (Figure 5). <br /> tion strains observed spinning cells                                 To solidify observations, alternative floccula- <br /> for 1 minute 500 rpm. The suspended cells                                      tion assay flocculation initiated addition of <br />  Chin et al. Journal Biomedical Science 2013, 20:97                                                                                    Page 8 11 <br /> http://www.jbiomedsci.com/content/20/1/97 <br />  <br />  <br />  <br />  <br />                      A                                               B                        Met/Cys         +Met/Cys <br />                                 CaCDC4 M3/ <br />                                                                                                   +Dox              +Dox <br />                                                                                                                             JSCA00 <br />  <br />                                                                                  ve                                           22 <br />  <br />  <br />                                                          KDa                  Tet- <br />                                                                                                                               23 <br />                                                                               CaCDC4 <br />                                                           100                 Tet- <br />                                                            72                 CaCDC4-                                         24 <br />                                                                               6HF <br />  <br />  <br />  <br />  <br />                                                                  CaCDC4 M3/ <br />                                                            55 <br />  <br />                                                            40                 Tet-&#916;N- <br />                                                                                                                               25 <br />                                                                               6HF <br />                                                            33 <br />                                                                               Tet-F-box- <br />                                                                                                                               26 <br />                                                            24                 6HF <br />  <br />                                                                               Tet-WD40- <br />                        22      23 24 25 26      27 30   JSCA00                                                                27 <br />                                                                               6HF <br />                                  +Met/Cys <br />                                   +Dox                                        Tet-&#916;NF- <br />                                                                                                                               30 <br />                                                                               6HF <br />  <br />  Figure 4 Morphological analysis C. albicans strains capable Dox-inducing expression assorted CaCDC4 domains. Cells were <br />  grown initially SD medium Met/Cys saturation diluted initial concentration. Cells grown exponentially <br />  SD absence 2.5 mM Met/Cys, 50 &#956;g/ml Dox (&#8722;Met/Cys + Dox &#8722;Met/Cys), presence 2.5 mM Met/Cys, <br />  50 &#956;g/ml Dox (+Met/Cys + Dox + Met/Cys). (A) The Dox-inducibly expressing assorted CaCdc4 protein domains under <br />  CaMET3-CaCDC4 repressed conditions verified Western blotting polyclonal antibody FLAG. The non-specific signals 72 <br />  55 kDa, 55 40 kDa served loading controls. (B) The images visualized recorded Nikon 50i <br />  microscope 400&#215; magnification. The arrow white indicates filamentous cells. Bars represent 10 &#956;m. The designations strains the <br />  Table 1. <br />  <br />  <br />  <br />  <br /> Ca2+ culture medium depleted Ca2+                                   unpublished results). We confirmed JSCA0022 <br /> beforehand used [23]. Only cells JSCA0023                                  strain repressed conditions equivalent a <br /> JSCA0024 remained resistance flocculation                               strain completely lost CaCDC4 function. Hence, <br /> time-frame 5-minute assay compared                                   introduction Tet-on cassettes JCSA0022 <br /> the rest strains (Figure 6), consistent                            strain, strains capable expressing indi- <br /> the results shown Figure 5. However, strains                                  vidual CaCdc4 domains presence Met/Cys and <br /> JSCA0025 JSCA0027 exhibited greater ability re-                                Dox functional comparisons. <br /> sist flocculation JCSA 0026 JSCA0030                                   To verify ability Tet-on cassettes C. albicans, <br /> when considering differences OD600 ini-                               cassettes encoding various CaCdc4 domains <br /> tial end points.                                                               transformed BWP17 JSCA0021 before <br />                                                                                       introducing JSCA0022 CaADH1 locus. <br /> Discussion                                                                            Individual CaCdc4 domains relevant strains were <br /> In study, aimed dissect function CaCdc4                             detectable, suggesting Tet-on func- <br /> domains introducing Tet-on cassettes                                 tions C. albicans. However, cells expressing <br /> that encoded variety CaCdc4 domains                                     F-box WD40 repeat detected as <br /> C. albicans mutant Cacdc4 null. However,                                       expected sizes, expressing full-length <br /> Cacdc4 null mutant filamentous form                                  CaCdc4, N-terminus truncated CaCdc4 (&#916;N), and <br /> be easily used introduce Tet-on cassettes; there-                              &#916;NF CaCdc4 detected positions <br /> fore, constructed JSCA0022 strain,                                       higher anticipated (Figure 4A). In particular, the <br /> CaURA3 released strain JSCA0021,                                     sample strain JSCA0030 expressing &#916;NF <br /> CaCDC4 expression repressible. Under repressed                                    detected signals (Figure 4A), which <br /> conditions, JSCA0022 strain showed similar fila-                                  greater predicted sizes. These results <br /> mentous morphology (Figure 2) previous                                  suggest N-terminal CaCdc4 residue 85 <br /> reports cells CaCDC4 repressed strain [6,7]                            241 (Figure 3A) undergoing post-translational <br /> cacdc4 null mutant [6] (Tseng TL, Hsu WH, Shieh JC,                               modification Tet-on-induced expression, <br />  Chin et al. Journal Biomedical Science 2013, 20:97                                                                              Page 9 11 <br /> http://www.jbiomedsci.com/content/20/1/97 <br />  <br />  <br />  <br />  <br />                      A <br />  <br />  <br />  <br />  <br />                                JSCA00 21                22        23     24          25       26        27      30 <br />  <br />  <br />  <br />  <br />                                                                          CaCDC4 M3/ <br />  <br />                      B <br />                                1 <br />                                                              **         ** <br />                               0.8 <br />                               0.6 <br />                       OD600 <br />  <br />  <br />  <br />  <br />                               0.4 <br />                               0.2 <br />                                0 <br />  <br />                                                                    Strain JSCA00 <br />  Figure 5 Analysis cell flocculation low-speed centrifugation. Cultures indicated strains grown SD medium histidine, <br />  arginine, uridine 2 days diluting SD medium initial OD600 = 0.1 addition 2.5 mM Met/Cys repress the <br />  expression CaMET3p-driven CaCDC4 40 &#956;g/ml Dox induce expression CaCdc4 domains tested 48 hrs OD600 &#8776; 1.6. Cultures <br />  photographed centrifugation. (A). A representative cultures. Upper panel: two-day culture. Bottom panel: cultures <br />  spun 500 rpm 1 minute. (B). Quantitative results. Data represented means standard deviation three <br />  independent experiments, sample duplication. The data JSCA0022 compared strains. **: P &lt; 0.01. <br />  The designations strains Table 1. <br />  <br />  <br />  <br />  <br /> although functional significance unknown. Interest-                adh1 null mutant reintegrated functional copy of <br /> ingly, region residue 85 241 CaCdc4                    CaADH1 CaADH1 locus appears similar <br /> contains abundant serine threonine residues,                      biofilm formation isogenic wild-type strain. <br /> majority homologous S. cerevisiae Cdc4                    In addition, disruption CaADH1 consequence <br /> [7]. This implies possible phosphorylations modi-                morphology alteration C. albicans [27] (Lai WC, <br /> fications residues specific C. albicans.              unpublished results). Therefore, possible effect of <br /> However, genuine nature residues remains                     Tet-on cassette flocculation filamentation by <br /> to determined, functional significance                    integration, disruption copy CaADH1 locus <br /> this N-terminal CaCdc4 requires study.                            excluded. <br />    With regards integration CaADH1 locus                            Under Met/Cys Dox conditions, cells express- <br /> the Tet-on cassette, known C. albicans adh1                    ing F-box, WD40 repeat, &#916;NF CaCdc4 exhib- <br /> homozygous null mutant gains ability form bio-                     ited filamentous forms similar JSCA0022, <br /> film vitro vivo [27], suggesting possible                CaCDC4 repressed, compared ex- <br /> role CaADH1 flocculation. However, heterozy-                    pressing full-length CaCdc4 tag <br /> gous CaADH1 null mutant homozygous                         (JSCA0023 JSCA0024), exhibited yeast forms <br />  Chin et al. Journal Biomedical Science 2013, 20:97                                                                                 Page 10 11 <br /> http://www.jbiomedsci.com/content/20/1/97 <br />  <br />  <br />  <br />  <br />                                                                       acid truncated CaCdc4 unable totally overturn <br />                         1.45 <br />                                                                       filamentous-to-yeast cells, suggesting N-terminal <br />                                                                21     85-amino acid required activity CaCDC4 <br />                                                                       function C. albicans inhibit filamentation. However, <br />                          1.4 <br />    Cell culture OD600 <br />  <br />  <br />  <br />  <br />                                                                22     flocculation tightly associated filamentation, <br />                                                                23     expect extent flocculation JCSA0025 <br />                                                                       (&#916;N 6HF) greater JSCA0022 less <br />                         1.35 <br />                                                                24     JSCA0023 JSCA0024 presence <br />                                                                       Met/Cys Dox. This revealed low <br />                                                                25 <br />                          1.3 <br />                                                                       speed-centrifugation method Ca2+-initiation <br />                                                                26     assay. Importantly, JSCA0025 JSCA0027 express- <br />                                                                       ing CaCdc4 lacking N-terminal 85-amino acid (Figure 3A) <br />                                                                27 <br />                         1.25                                          exhibits similar extent flocculation. Moreover, JSCA0025 <br />                                                                30     expressing CaCdc4 lacking N-terminal 85-amino acid <br />                                                                       partially suppress filamentation JSCA0027 <br />                          1.2                                          expressing CaCdc4 lacking N-terminal 85-amino <br />                                                                       acid F-box flanking regions completely lose <br />                                                                       ability inhibit filamentation (Figure 3A and <br />                                Time (sec) <br />                                                                       Figure 4B). These results imply N-terminal 85-amino <br />  Figure 6 Analysis Ca2+ initiated cell flocculation. The strains <br />                                                                       acid CaCdc4 role inhibition cell flocculation <br />  grown described Figure 5 saturation OD600 &#8776; 1.4. <br />  The cultures harvested washed twice deflocculation     C. albicans F-box flanking region in <br />  buffer, followed initiation flocculation described   addition N-terminal 85-amino acid CaCdc4 <br />  Methods. The assays conducted triplicate. The   associated proper control morpho- <br />  strain shows numbers.                              genesis flocculation. <br />  <br /> (Figure 4B). These results suggest WD40                 Conclusions <br /> repeat F-box essential suppress yeast-                 Therefore, conclude F-box WD40-repeat <br /> to-filament transition. Cells strain JSCA0025 ex-                important suppressing yeast-to-filament transition <br /> pressing &#916;N CaCdc4, grown                    flocculation N-terminal region (1&#8211;85) <br /> presence Met/Cys Dox, partially able                 positive role CaCDC4 function, lost which <br /> to reverse filamentous cells yeast cells, suggesting               impairs reverse filament-to-yeast reduces abil- <br /> that N-terminal 85-amino acid CaCdc4 plays                   ity flocculate C. albicans. Moreover, function of <br /> role yeast-to-filament transition C. albicans.              CaCdc4 suppressing flocculation related to <br /> The role N-terminal 85-amino acid CaCdc4                cell-cell adhesion [21] implies role CaCDC4 bio- <br /> growth observed previously, cells express-               film formation [28] investigation. <br /> ing N-terminal 85-amino acid truncated CaCdc4 lagged <br /> slightly proliferation exponential stage <br /> (Additional file 1: Figure S1), repression                 Additional files <br /> expression N-terminal 85-amino acid truncated <br /> CaCdc4 resulted prominently lagging                       Additional file 1: Figure S1. N-terminal 85-amino acid CaCdc4 is <br />                                                                        required normal growth C. albicans. Strains: BWP17, heterozygous <br /> growth, presumably morphological                  null mutant CaCDC4 +/&#8722;, M3CaCDC4 +/&#8722; carrying CaMET3-full-length <br /> alteration cells filaments advance (Additional                CaCDC4, M3NTCaCDC4 +/&#8722; carrying CaMET3-partial CaCDC4 (capable <br /> file 2: Figure S2) delays proliferation compared               expressing N-terminal 85-amino acid truncated CaCdc4). Cells of <br />                                                                        strains grown initially SD medium Met/Cys to <br /> to yeast cells. Since N-terminal 85-amino                 saturation diluted initial concentration. Cells were <br /> acid CaCdc4 unique compared                       grown 12 hrs SD 2.5 mM Met/Cys (&#8722;Met/Cys <br /> S. cerevisiae Cdc4 [7], finding reveals role                  + Met/Cys) 2-hr interval cells sampled to <br />                                                                        determine optical density 595 nm (O.D. 595) growth <br /> N-terminal 85-amino acid CaCdc4 morphogen-                       curves plotted. <br /> esis, unknown previously.                                     Additional file 2: Figure S2. N-terminal 85-amino acid CaCdc4 is <br />    Importantly, cells JSCA0022-based strains exhib-             required suppression yeast-to-filament transition C. albicans. Cells <br /> ited flocculation medium Met/Cys, strains              strains grown initially SD medium Met/Cys to <br />                                                                        saturation diluted initial concentration. Cells were <br /> JSCA0023 (CaCDC4) JSCA0024 (CaCDC4-6HF)                            grown 8 hrs SD 2.5 mM Met/Cys (&#8722;Met/Cys <br /> exhibited flocculation adding Dox simultaneously               + Met/Cys). The images visualized recorded Nikon <br /> (Figure 5). Unlike cells JSCA0023 JSCA0024,                     50i microscope 400&#215; magnification. Bars represent 10 &#956;m. The <br />                                                                        designations strains Additional file 1: Figure S1. <br /> those JSCA0025 expressing N-terminal 85-amino <br />  Chin et al. Journal Biomedical Science 2013, 20:97                                                                                            Page 11 11 <br /> http://www.jbiomedsci.com/content/20/1/97 <br />  <br />  <br />  <br />  <br /> Abbreviations                                                                         presence novel proteins involved morphogenesis. Biochem Biophys <br /> SCF: Skp1-Cdc53/Cul1-F-box; CaCDC4: Candida albicans CDC4; CaMET3p:                   Res Commun 2010, 395:152&#8211;157. <br /> C. albicans MET3 promoter; Dox: Doxycycline; CaCdc4: C. albicans Cdc4;          13.   Miller J: Experiments Molecular Genetics. New York: Cold Spring Harbor <br /> ScCDC4: S. cerevisiae CDC4; ScCdc4: S. cerevisiae Cdc4; Met/Cys: Methionine           Laboratory Press, Cold Spring Harbor; 1972. <br /> and cysteine; YEPD: Yeast extract-peptone-dextrose; SD: Synthetic defined;      14.   Wilson RB, Davis D, Mitchell AP: Rapid hypothesis testing Candida <br /> CaURA3: C. albicans URA3; CaADH1: C. albicans ADH1.                                   albicans gene disruption short homology regions. <br />                                                                                       J Bacteriol 1999, 181:1868&#8211;1874. <br /> Competing interests                                                             15.   Sherman F, Fink GR, Hick J: Methods Yeast Genetics. New York: Cold Spring <br /> The authors declare competing interests.                            Harbor Laboratory Press, Cold Spring Harbor; 1986. <br />                                                                                 16.   Care RS, Trevethick J, Binley KM, Sudbery PE: The MET3 promoter: new <br /> Authors&#8217; contributions                                                                tool Candida albicans molecular genetics. Mol Microbiol 1999, <br /> CC, WCL, JCS, TLL conceived designed experiments. CC, WCL,                34:792&#8211;798. <br /> TLT performed experiments. CC, WCL, JCS, TLT analyzed data.         17.   Walther A, Wendland J: An improved transformation protocol the <br /> WCL, TLL, TLT contributed reagents materials. JCS wrote paper.            human fungal pathogen Candida albicans. Curr Genet 2003, 42:339&#8211;343. <br /> All authors read approved final manuscript.                             18.   Lai WC, Tseng TL, Jian T, Lee TL, Cheng CW, Shieh JC: Construction of <br />                                                                                       Candida albicans Tet-on tagging <span id='am-59' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="vectors" datatype="xsd:string"></span>vectors</span> Ura-blaster cassette. <br />                                                                                       Yeast 2011, 28(3):253&#8211;263. <br /> Acknowledgements <br />                                                                                 19.   Park YN, Morschhauser J: Tetracycline-inducible gene expression and <br /> The authors thank Dr. A. Mitchell (Columbia University) C. albicans <br />                                                                                       gene deletion Candida albicans. Eukaryot Cell 2005, 4:1328&#8211;1342. <br /> strain BWP17 plasmid pDDB57, Dr. J. Wendland pFA-HIS1-MET3p <br />                                                                                 20.   Shieh JC, Cheng YC, Su MC, Moore M, Choo Y, Klug A: Tailor-made <br /> (Friedrich-Schiller-University, Jena, Germany), Dr. J. Morschh&#228;user for <br />                                                                                       zinc-finger transcription factors activate FLO11 gene expression with <br /> pTET25 (Universit&#228;t W&#252;rzburg University, W&#252;rzburg, Germany). We gratefully <br />                                                                                       phenotypic consequences yeast Saccharomyces cerevisiae. <br /> acknowledge financial support work provided grant a <br />                                                                                       PLoS ONE 2007, 2:e746. <br /> collaborative agreement Chung Shan Medical University <br />                                                                                 21.   Verstrepen KJ, Klis FM: Flocculation, adhesion biofilm formation in <br /> (Taichung, Taiwan) Antai Tian-Sheng Memorial Hospital (Pingdung, <br />                                                                                       yeasts. Mol Microbiol 2006, 60:5&#8211;15. <br /> Taiwan) (CSMU-TSMH-097-005) grants National Health <br />                                                                                 22.   Fu Y, Ibrahim AS, Sheppard DC, Chen YC, French SW, Cutler JE, Filler SG, <br /> Research Institutes (NHRI-EX100-9808SI) National Science Council <br />                                                                                       Edwards JE Jr: Candida albicans Als1p: adhesin downstream <br /> (NSC 101-2629-B-040-001-MY3 ) Taiwan, R.O.C. <br />                                                                                       effector EFG1 filamentation pathway. Mol Microbiol 2002, 44:61&#8211;72. <br />                                                                                 23.   Bayly JC, Douglas LM, Pretorius IS, Bauer FF, Dranginis AM: Characteristics <br /> Author details <br /> 1                                                                                     Flo11-dependent flocculation Saccharomyces cerevisiae. <br />  Division Infectious Disease, Department Internal Medicine, Antai <br />                                                                                       FEMS Yeast Res 2005, 5:1151&#8211;1156. <br /> Medical Care Cooperation Antai Tian-Sheng Memorial Hospital, Pingtung, <br />                                                                                 24.   Wilson RB, Davis D, Enloe BM, Mitchell AP: A recyclable Candida albicans <br /> Taiwan. 2Department Biomedical Sciences, Chung Shan Medical University, <br />                                                                                       URA3 cassette PCR product-directed gene disruptions. Yeast 2000, <br /> No. 110, Sec. 1, Jianguo N. Road, Taichung City 40201, Taiwan. 3Department <br />                                                                                       16:65&#8211;70. <br /> of Molecular Biotechnology, Da-Yeh University, Changhua County, Taiwan. <br /> 4                                                                               25.   Kirsch DR, Whitney RR: Pathogenicity Candida albicans auxotrophic <br />  Department Medical Research, Chung Shan Medical University Hospital, <br />                                                                                       mutants experimental infections. Infect Immun 1991, 59:3297&#8211;3300. <br /> Taichung City, Taiwan. <br />                                                                                 26.   Bain JM, Stubberfield C, Gow NA: Ura-status-dependent adhesion of <br />                                                                                       Candida albicans mutants. FEMS Microbiol Lett 2001, 204:323&#8211;328. <br /> Received: 3 October 2013 Accepted: 15 December 2013 <br />                                                                                 27.   Mukherjee PK, Mohamed S, Chandra J, Kuhn D, Liu S, Antar OS, Munyon R, <br /> Published: 20 December 2013 <br />                                                                                       Mitchell AP, <span id='am-1' about='obo:NCBITaxon_491294' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_33361 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_35567 obo:NCBITaxon_35568 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_7524 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_36152 obo:NCBITaxon_36151 obo:NCBITaxon_88770 obo:NCBITaxon_33342 obo:NCBITaxon_50557'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="rdfs:label" content="Andes" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Andes</span> D, Chance MR, et al: Alcohol dehydrogenase restricts <br />                                                                                       ability pathogen Candida albicans form biofilm on <br /> References                                                                            catheter surfaces ethanol-based mechanism. Infect Immun <br /> 1. Whiteway M, Bachewich C: Morphogenesis Candida albicans (*).                    2006, 74:3804&#8211;3816. <br />     Annu Rev Microbiol 2007, 61:529&#8211;553.                                        28.   Finkel JS, Mitchell AP: Genetic control Candida albicans biofilm <br /> 2. Biswas S, Van Dijck P, Datta A: Environmental sensing signal                   development. Nat Rev Microbiol 2011, 9:109&#8211;118. <br />     transduction pathways regulating morphopathogenic determinants of <br />     Candida albicans. Microbiol Mol Biol Rev 2007, 71:348&#8211;376. <br />                                                                                   doi:10.1186/1423-0127-20-97 <br /> 3. Liu H: Transcriptional control dimorphism Candida albicans. <br />                                                                                   Cite article as: Chin et al.: Dissection Candida albicans Cdc4 <br />     Curr Opin Microbiol 2001, 4:728&#8211;735.                                          protein reveals involvement domains morphogenesis cell <br /> 4. Lo HJ, Kohler JR, DiDomenico B, Loebenberg D, Cacciapuoti A, Fink GR:          flocculation. Journal Biomedical Science 2013 20:97. <br />     Nonfilamentous C. albicans mutants avirulent. Cell 1997, 90:939&#8211;949. <br /> 5. Berman J: Morphogenesis cell cycle progression Candida albicans. <br />     Curr Opin Microbiol 2006, 9:595&#8211;601. <br /> 6. Atir-Lande A, Gildor T, Kornitzer D: Role SCFCDC4 ubiquitin ligase <br />     Candida albicans morphogenesis. Mol Biol Cell 2005, 16:2772&#8211;2785. <br /> 7. Shieh JC, White A, Cheng YC, Rosamond J: Identification functional <br />     characterization Candida albicans CDC4. J Biomed Sci 2005, <br />     12(6):913&#8211;924. <br /> 8. Hochstrasser M: Protein degrada <br /> </body></html>