<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                        Citation: Cell Death Disease (2013) 4, e900; doi:10.1038/cddis.2013.432 <br /> OPEN                                                                                    &amp; 2013 Macmillan Publishers Limited All rights reserved 2041-4889/13 <br />                                                                                         www.nature.com/cddis <br />  <br />  <br /> Gene miles-apart required formation otic <br /> vesicle hair cells zebrafish <br /> Z-y Hu1,2, Q-y Zhang1,2, W Qin1, J-w Tong1, Q Zhao1, Y Han1, J Meng1 J-p Zhang*1 <br />  <br />  <br /> Hearing loss burden physical mental health worldwide. Aberrant development damage hearing organs <br /> are recognized causes hearing loss, molecular mechanisms underlining pathological processes remain <br /> elusive. Investigation new molecular mechanisms involved proliferation, differentiation, migration maintenance of <br /> neuromast primordium hair cells contribute better understanding hearing loss pathology. <span id='am-37' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-38' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> knowledge will <br /> enable development protective agents mechanism study drug ototoxicity. In study, demonstrate the <br /> zebrafish gene miles-apart, homolog sphingosine-1-phosphate receptor 2 (s1pr2) mammals, important role the <br /> development otic vesicle, neuromasts survival hair cells. Whole-mount situ hybridization embryos showed that <br /> miles-apart expression occurred mainly encephalic region somites 24 h.p.f. (hour post fertilization), the <br /> midbrain/hindbrain boundary, brainstem pre-neuromast lateral line 48 h.p.f. strict spatiotemporal regulation. <br /> Both up- downregulation miles-apart led abnormal otoliths semicircular canals, excess hair cells and <br /> neuromasts, disarranged depositions lateral lines. Miles-apart (Mil) dysregulation caused abnormal <br /> expression hearing-associated genes, including hmx2, fgf3, fgf8a, foxi1, otop1, pax2.1 tmieb zebrafish <br /> organogenesis. Moreover, larvae miles-apart gene knockdown significantly upregulated proapoptotic gene zBax2 and <br /> downregulated prosurvival gene zMcl1b; contrast, level zBax2 decreased zMcl1b enhanced miles-apart <br /> overexpression. Collectively, Mil activity linked organization number decision hair cells neuromast, to <br /> deposition neuromasts formation otic vesicle zebrafish organogenesis. At larva stage, Mil upstream <br /> regulator bcl-2 gene <span id='am-6' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-7' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> role protection hair cells apoptosis promoting expression prosurvival gene <br /> zMcl1b suppressing proapoptotic gene zBax2. <br /> Cell Death Disease (2013) 4, e900; doi:10.1038/cddis.2013.432; published online 31 October 2013 <br /> Subject Category: Neuroscience <br />  <br />  <br />  <br />  <br /> Hearing loss impairs personal physical mental                              S1P ligand human G protein-coupled receptors <br /> health increases social economic burden.                                 (GPCRs), called S1P1, S1P2, S1P3, S1P4 S1P5. <br /> Hearing damage caused wide range factors,                                Dissimilarly, mice possess receptors: S1p1, S1p2 <br /> which include defects hair cells inner ear structures,                       S1p3. Though coupled G-proteins, the <br /> the defect molecular targets remain elusive. Thus, identifica-                          S1P receptors demonstrate functional redundancy. <br /> tion new molecular mechanisms associated develop-                               Interactions receptors mediated tyrosine <br /> mental processes hearing organs migration                             kinase receptors.4 Transduction S1P signaling occurs <br /> neuromast primordium, formation inner ear, differentiation                           combining Edg family receptors.5&#8211;7 Although <br /> and maintenance hair cells, hair cell apoptosis,                          reports S1P receptor function relate cardiovas- <br /> will contribute better understanding hearing loss                               cular, nervous immune systems,8&#8211;11 some <br /> pathology. This knowledge developing facilitate                                evidence S1P receptors related function of <br /> development new medicines prevent ototoxicity                                 cochlea vestibular apparatus. Herr et al.2 S1p1, <br /> protect hearing.                                                                        S1p2 S1p3 expressed mouse cochlea, that <br />    Sphingosine-1-phosphate (S1P) lipid-active mediator                             S1p2 levels higher S1p1 S1p3, and <br /> and signal molecule, addition component                            upregulated developmental process. S1p2- <br /> cell membrane. It involved regulating multiple physiolo-                          knockout mice normal cochlear vestibular develop- <br /> gical functions, including cell movement, migration, growth                             ment initially, rapid degeneration structures led to <br /> and survival, vasculogenesis, nervous immune                                 deep deafness 22 days age regressive lost of <br /> function, reproduction, lymphocyte transfer audition.1&#8211;3                            vestibular function age growth. S1p2/S1p3 double <br />  <br /> 1 <br />  Institute Medicinal Biotechnology, Chinese Academy Medical Sciences Peking Union Medical College, Beijing, China <br /> *Corresponding author: J-p Zhang, Institute Medicinal Biotechnology, Chinese Academy Medical Sciences Peking Union Medical College, No. 1, TianTan XiLi, <br /> Dong-cheng District, Beijing 100050, China. Tel: 086 10 63186645; Fax: 86 10 63017302; E-mail: zjp5577@126.com <br /> 2 <br />  Co-first authors. <br /> Keywords: miles-apart; hair cells; neuromast; otoliths; hearing-associated genes; apoptosis <br /> Abbreviations: S1P, Sphingosine-1-phosphate; WT, wild-type; d.p.f., day post fertilization; h.p.f., hour post fertilization; MOmil, morpholino oligo; Mil, miles-apart; o1, <br /> orbital 1; ml1, middle lateral line 1; ml2, middle lateral line 2; o2, orbital 2; io4, infraorbital 4 <br /> Received 03.6.13; revised 30.9.13; accepted 30.9.13; Edited A Stephanou <br />                                                   Mil otic vesicle hair cells zebrafish <br />                                                                                      Z-y Hu et al <br /> 2 <br />  <br />           knockout caused increased perinatal lethality decreased                                      In study, explored functions zebrafish <br />           litter size. As S1p2-knockout mice, significant                                         miles-apart hair cell neuromast development and <br />           progressive degeneration auditory vestibular organs                                     survival report association miles-apart <br />           occurred S1p2/S1p3 double knockout mice age,                                      activity development neuromasts, hair cells and <br />           culminating complete hearing loss.1&#8211;2 These results                                         otic vesicles, expression auditory- <br />           indicate S1p receptors important roles maintain-                                  related genes bcl-2 gene family. We infer Mil may <br />           ing hair cell functions mice.                                                               important regulator formation mainte- <br />               In zebrafish, mutation S1P2 ortholog, miles-apart (Mil),                            nance hearing organs zebrafish. Our findings will <br />           resulted cardia bifida phenotype. This defect cardiac                                  contribute better understanding processes <br />           development failed migration heart precursor                                      hearing development facilitate research oto- <br />           cells midline, revealing important function                                   protective agents. <br />           S1P2 receptor directing cardio-primordium loca- <br />           lization cardiac tube fusion zebrafish. Loss Mil                                     Results <br />           function caused edematous pericardial sac epithe- <br />           lium-like blistering tail.5,8,10 However, reports                          Expression pattern miles-apart zebrafish <br />           zebrafish role Mil development auditory                                organogenesis. Whole-mount situ hybridization was <br />           vestibular organs, hair cells neuromasts.                                      used detect miles-apart spatiotemporal expression pattern <br />           Zebrafish hair cells similar mammalian counterparts                               zebrafish embryonic development. The miles-apart <br />           morphology function, reside inside otic                                    expression mainly aggregated encephalic region, <br />           vesicles neuromasts lateral line                               particularly midbrain/hindbrain boundary the <br />           body surface. These characteristics make zebrafish good                                      ventral hindbrain; weak expression observed in <br />           model study effects drug toxicity hearing                                  somites notochord 24 h.p.f. (Figures 1a, a0 a00 ). <br />           protection regeneration hair cells.1&#8211;2 In zebrafish                             At pharyngula stage (48 h.p.f.), miles-apart transcrip- <br />           embryonic stage otic capsule contains maculae:                                    tion focused midbrain/hindbrain boundary, midbrain, <br />           anterior utriculus vestibular function, posterior                                ventral hindbrain area, floor plate neural tube <br />           sense macula (saccule) role hearing. Each macula                                   vasculature, particularly appeared regular and <br />           covered otolith, attached kinocilium hair                               discrete pattern positions future neuromasts on <br />           cells. Semicircular canals begin form 48 h.p.f. (hour post                               lateral lines (Figures 1b, b0 b00 ). However, the <br />           fertilization). The number hair cells maculae lateral                                hatching period (72 h.p.f.), miles-apart mRNA not <br />           lines increases rapidly 3 7 days post fertilization (d.p.f.),                          detected lateral lines posterior floor plate but <br />           7 d.p.f. key structures emerged (the lagena,                               highly expressed midbrain/hindbrain bound- <br />           major auditory macula, does form days later).                              ary, brainstem floor plate (Figure 1c). These <br />           Neuromasts lateral line body surface                                    results suggest miles-apart expression pattern is <br />           similar structure function sensory maculae                                correlated development nervous system, including <br />           inner ear.12 These functions mainly dependent                                       brain, central nervous tube neural sensory <br />           physiological activity hair cells.                                                      zebrafish. <br />  <br />  <br />  <br />  <br />           Figure 1 Expression pattern miles-apart organogenesis zebrafish. Hybridization situ performed miles-apart antisense RNA probe (a), (b) and <br />           (c); miles-apart sense RNA probe used negative control probe (d). The ventral parts removed embryos primary presentation heads the <br />           trunks. The bodies showed 1 d.p.f. (a), 2 d.p.f. (b) 3 d.p.f. (c); corresponding heads showed (a0 ) (b0 ), trunks showed (a00 ) <br />           (b00 ). Black arrows indicate scheduled neuromast positions post lateral lines (b00 ). Black half brackets signed mid/hindbrain boundary (a0 ) (b0 ). In all <br />           images, head left, tail right; dorsal upside, ventral bottom. Scale bars represent 310 mm (a) (b), 330 mm (c), 380 mm <br />           (d), 160 mm (a0 ), (b0 ), 130 mm (a00 ), 180 mm (b00 ) <br />  <br /> Cell Death Disease <br />                                                                                            Mil otic vesicle hair cells zebrafish <br />                                                                                           Z-y Hu et al <br />                                                                                                                                                                                      3 <br />  <br /> Dysregulation miles-apart expression disturbs devel-                                   canal (psc), otolith abnormality 72 h.p.f. <br /> opment hair cells otic vesicles. Using indica-                                 embryos (11/15) (Figure 2b). Correspondingly, capped <br /> tion high miles-apart expression occurred ventral                             mRNA Mil injected embryos stage <br /> forebrain midbrain, midbrain/hindbrain boundary, brain-                               miles-apart overexpression. In embryos, utricle <br /> stem lateral line neuromasts 48 h.p.f. WT                                      appeared shrunk smaller, saccular otolith was <br /> embryos, hypothesized Mil functions                                        deformed divided 48 h.p.f. (10/10) (Figure 2a). <br /> associated formation neuromasts hair cells                                At 72 h.p.f., middle junction semicircular canals <br /> beside function development central nervous                                 deficient, semicircular canals totally absent <br /> system. We investigated Mil function development                                (7/11) (Figure 2b). These <span id='am-50' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-51' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53'  ></span><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> suggest miles-apart gene <br /> otic vesicle zebrafish embryo using down- upregula-                                dysregulation considerably result mis-patterning the <br /> tion miles-apart expression. An antisense morpholino oligo                             otic vesicle abnormal structure otoliths. <br /> (MOmil) binding miles-apart translation initiation site                               Hair cell staining YO-PRO-1 used examine the <br /> was used inhibition miles-apart translation injection                           impact miles-apart formation deposition hair <br /> into embryos one- four-cell stage. The morphant                                 cells neuromasts lateral lines surface of <br /> embryos displayed aberrant otic vesicle otoliths                                 zebrafish head body. The results 6 d.p.f. <br /> utricles non-oval, disfigure bigger saccule;                              morphants posterior lateral line neuromasts <br /> and contrast, saccule morphous presented small, deficient                              absent shrunken (Figure 3b); anterior <br /> and non-spherical structures 48 h.p.f. (9/10). The size ratio                          lateral line number hair cells neuromast orbital 1 (o1), <br /> of saccule utricle changed significantly compared                             middle lateral line 1 (ml1), middle lateral line 1 (ml2), orbital 2 <br /> WT ratio (Figure 2a). The semicircular canals defect                                  (o2) infraorbital 4 (io4), resided otic vesicles <br /> structure gap breakage developed                                     around, counted hair cells were <br /> anterior semicircular canal (psc) posterior semicircular                              obviously diminished neuromast o1 o2 (Figures 3g j), <br />  <br />  <br />                                                  Change otolith size <br />                                     3 <br />                Sacculus/Utricle <br />  <br />  <br />  <br />  <br />                                   2.5 <br />                                     2 <br />                                   1.5 <br />                                     1                  ** <br />                                   0.5 <br />                                     0 <br />                                           WT           MO      misMO          mRNA <br />                                         (n=12)       (n=10)    (n=11)         (n=10) <br />                                                                                                                     WT    MOmil               misMO           mRNA <br />  <br />  <br />  <br />       WT <br />                                                                                               Othlith <br />  <br />  <br />  <br />  <br />   MOmil <br />  <br />                                                                                                                     WT   MOmil               misMO         mRNA <br />                                                                                               Semicircular canals <br />  <br />  <br />  <br />  <br />   misMO <br />  <br />  <br />  <br />  <br />   mRNA <br />  <br />  <br />  <br /> Figure 2 Developmental defects otoliths semicircular canals caused miles-apart dysregulation. Embryos injected 40 mM MOmil 40 ng/ul mil-mRNA <br /> at one-to four-cell stage observed 2 d.p.f. (a) 3 d.p.f. (b). Compared WT embryos, abnormal otolith shapes emerged morphants 2 d.p.f., such <br /> as enlarged non-oval utricles (black arrows), shrink sacculus (orange arrows) MOmil injection (9/10). Although misMO control group otoliths present shape <br /> (12/13) similar WT (10/10). Embryos treated miles-apart mRNA injection exhibit deformed, divided syncytium-like sacculus (red arrows) smaller utricles <br /> (black arrowhead) miles-apart mRNA injection (11/12). A histogram indicates variation otolith sizes caused miles-apart gene knockdown overexpression using <br /> otolith area scanning, area ratio sacculus utricle used sign. The means S.D.&#8217;s derived specified otic vesicles. <br /> **P-values o0.01. At 3 d.p.f. embryos (b) panel shows WT embryos developed normal semicircular canals, deficient semicircular canals appeared at <br /> the posterior morphants (11/15), deformation missing occurred middle semicircular canals mil-mRNA injection group (7/11). Few of <br /> the misMO group defect (3/13). Blue arrows deficient semicircular canals. In addition, enlarged non-oval utricles (black arrows) shrink sacculus (orange <br /> arrows) MOmil injection (11/15) abnormal sacculuses miles-apart mRNA injection (5/11, red arrows) seen 3 d.p.f. The images left lateral view <br />  <br />                                                                                                                                                                  Cell Death Disease <br />                                                    Mil otic vesicle hair cells zebrafish <br />                                                                                       Z-y Hu et al <br /> 4 <br />  <br />  <br />  <br />  <br />                                                                                                                                                               WT <br />  <br />  <br />  <br />  <br />                                                                                                                                                               MO <br />  <br />  <br />  <br />  <br />                                                                                                                                                     misMO <br />  <br />  <br />  <br />  <br />                                                                                                                                                         mRNA <br />  <br />                                                                                                                    Number PLL neuromasts <br />                                                                                                          18 <br />                                                                                                                                        * <br />                                                                                                          16 <br />                                                                                     Neuromast counting <br />  <br />  <br />  <br />  <br />                                                                                                          14 <br />                                                                                                          12 <br />                                                                                                          10 <br />                                                                                                           8 <br />                                                                                                           6 <br />                                                                                                           4 <br />                                                                                                           2 <br />                                                                                                           0 <br />                                                                                                               WT        MO         misMO         mRNA <br />  <br />  <br />                                                   O1         ML1 <br />  <br />                                                         O2 <br />                                                IO4                              ML2 <br />  <br />  <br />  <br />  <br />                                                                                20                                                                   WT <br />                                                                                                                                                     MO <br />                                                              Hair cell count <br />  <br />  <br />  <br />  <br />                                                                                                                                                     misMO <br />                                                                                15                                                  *                mRNA <br />                                                                                10                                  ** <br />                                                                                5 <br />                                                                                0 <br />                                                                                                          o1        o2        ml1           ml2          io4 <br />                                                                                                                           Neuromast <br />           Figure 3 Formation hair cells neuromasts disturbed miles-apart up- downregulation. All images shot 6 d.p.f. (a) A WT larva shows normal pattern <br />           neuromasts head posterior lateral line. (b) A morphant exhibits small deficient neuromasts neuromast-missing positions (yellow rings) lateral line. <br />           (c) A control larva injected misMO shows neuromast distribution similar WT. (d) miles-apart mRNA injection caused extra neuromasts developed posterior lateral line <br />           (a red arrow red cycle). (e) A histogram shows number change PLL neuromasts. The means s.d.&#8217;s derived larvae. (f) Morphology normal <br />           neuromasts well-organized hair cell clusters otic vesicle WT larva. (g) Fewer, disarranged deficient hair cells neuromasts otic vesicle a <br />           morphant. (h) A larva injected misMO shows hair cell clusters similar WT. (i) Scattered excess hair cells neuromasts occurred mRNA-injected larvae. <br />           Names neuromasts otic vesicle labeled f o1 orbital 1, ml1 middle lateral line 1, ml2 middle lateral line 2, o2 orbital 2 io4 infraorbital 4. <br />           Scale bars represent 310 mm (a&#8211;d). Hair cells stained dye YO-PRO-1 (a&#8211;d), (f&#8211;i). (j) A histogram indicates variation hair cell counts miles-apart gene <br />           knockdown overexpression. The means S.D&#8217;s derived larvae. *P-values o0.05, **P-values o0.01 <br />  <br />  <br />           opposed ordered hair cell aggregation WT                                                           (Figures 3d, e, I j). Statistical analysis presents a <br />           group mismatched MO (misMO) control group                                                                significant difference neuromast numbers between <br />           (Figures 3f h). However, miles-apart expression                                                            morphants mRNA-injected larvae between <br />           upregulated mRNA injection, excess neuromasts                                                             WT larvae injection groups, respectively <br />           posterior lateral line ectopic fluffy hair cells                                                 (Figure 3e). In addition, compared WT group hair <br />           neuromasts otic vesicle observed                                                               cell number o1 o2 morphants ml1 mil <br />  <br /> Cell Death Disease <br />                                                                       Mil otic vesicle hair cells zebrafish <br />                                                                      Z-y Hu et al <br />                                                                                                                                                 5 <br />  <br /> overexpressed larvae changed significantly (Figure 3j).         approach animals vertebrates. The <br /> These data suggest Mil functions probably             development inner ear lateral line involve in <br /> involved controlling number organization          spatiotemporal patterning hearing-associated gene expres- <br /> hair cells neuromast, controlling number        sion, expansion otic vesicle signaling cascade, cell <br /> and localization neuromasts lateral lines.                 migration interaction neighboring tissues.13&#8211;14 <br />                                                                      Previous studies knockout mice showed loss of <br /> Dysregulation miles-apart affects expression                   S1p2 disturbed mouse hearing, concluding S1p2 <br /> hearing-associated genes zebrafish embryo. In <span id='am-47' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span>               important physiological role survival functional <br /> to explore miles-apart expression correlated         maintenance hair cells mice.1&#8211;2 However, role the <br /> genes involved developing hair cells              zebrafish S1p2 ortholog, Mil, hair cell function remains be <br /> neuromasts, examined expression levels hearing-                investigated. In study, observed miles-apart expres- <br /> related genes hmx2, fgf3, fgf8a, foxi1, otop1, pax2.1        sion encephalic region including hindbrain the <br /> and tmieb, 48 h.p.f. embryos. This time point corresponds         otic capsule resides, typical neuromast preset <br /> to stage otic vesicle developing            positions lateral lines 48 h.p.f. WT embryos. This is <br /> pro-neuromast primordium begins migrate. The results              likelihood predict future neuromast deposition the <br /> show transcription genes reduced                lateral lines. Between 48 72 h.p.f., miles-apart expression <br /> a concentration-dependent manner MOmil-injected                   disappeared embryonic trunk, suggesting miles- <br /> embryos, hmx2, fgf3, foxi1 pax2.1               apart subjected rigorous spatiotemporal regulation the <br /> reduced statistical significance (Figures 4a c). As           lateral line system. Using MOmil-mediated gene knockdown <br /> injection mock, misMO group gene levels slightly        miles-apart capped-mRNA overexpression, role Mil <br /> went down, hmx2 tmieb showed descent                   organization maintenance hair cells, neuromasts and <br /> significantly (Figures 4b d). Conversely, gene             otic vesicle organ identified. Further, explored <br /> levels elevated mRNA-injected embryos            potential miles-apart targeting genes known be <br /> elevation fgf3, foxi1 tmieb statistical significance   associated development function otic <br /> (Figures 4e f). So, suppose genes           vesicle hair cells. Seven hearing-related genes were <br /> effectors Mil activity development functions       coincidentally regulated miles-apart level <br /> hearing organs.                                                      variation. They upregulated, Mil over- <br />                                                                      expressed reduced miles-apart knocked down. <br /> Survival hair cells associated miles-apart            Thus, infer Mil important upstream regulator <br /> function suppressing zBax2 promoting zMcl1b.                  seven genes. In view literatures, fgf3 fgf8 are <br /> Respecting deficiency hair cells otic vesicles            required inner ear hindbrain patterning, induction of <br /> resulted MOmil injecton, asked miles-apart           otic placode migration PLL primordia.14&#8211;17 foxi1 the <br /> knockdown prompted hair cell death mechanism               earliest marker otic precursor cells otic placode <br /> was involved in? Survival larva hair cells detected       formation,18 pax2.1 essential induction <br /> using apoptosis staining kit (Invitrogen, Carlsbad, CA,           functional hearing cells inner ear lateral <br /> USA) dyes. Hair cells neuromast ml1 o2         lines.19&#8211;20 Notably, pax2.1 specifically expressed hair <br /> were counted analyzed statistics. The results            cells differentiation, maintained high <br /> that compared WT hair cells, number live hair cells       levels mature hair cells, expressed support <br /> 6 d.p.f. morphants decreased dimly stained                   cells.19 hmx2 involved formation mechanosensory <br /> Hoechst33324, apoptotic hair cells condensed                neuromasts,21 otop1 identified required for <br /> chromatin increased stained brightly Hoechst33324              otolith initiation development otolith size and <br /> and green fluorescence YO-PRO-1 dye, dead hair            shape.16,22 And tmieb identified involved the <br /> cells grew primarily red fluorescence          functioning auditory vestibular mechanoelectrical <br /> green fluorescence concentration-dependent manner               transduction.23 These data support inference study <br /> (Figure 5). Meanwhile, apoptosis/prosurvival genes           miles-apart gene activity likely linked hearing organ <br /> including bcl-2 family caspase family detected              development probably mediated hearing- <br /> using qPCR. Significantly, zMcl1b, prosurvival gene            associated genes. <br /> downregulated proapoptotic gene, zBax2,                       Previous research showed Mil responsible for <br /> upregulated morphants. In contrast, significantly too,        directing migration fusion cardiac precursor cells the <br /> zMcl1b increased zBax2 decreased larva           heart field subsequent differentiation functional <br /> mil-mRNA injection (Figure 6). So, suppose miles-            organ cardiovascular development.5,8,24 In present <br /> apart knockdown sensitized hair cell                 study, Mil function partly involved in <br /> apoptosis enhancing zBax2 expression suppressing              migration development hair cell precursors pro- <br /> zMcl1b expression. One Mil functions probably involved         neuromast primodia ALL PLL (Figure 3). The functional <br /> in hair cell survival.                                               concordance Mil divergent organs suggests basic <br />                                                                      common role Mil directing precursor cell migrating and <br />                                                                      positioning zebrafish organogenesis. <br /> Discussion <br />                                                                         Our results knockdown miles-apart gene <br /> The inner ear lateral line systems important organs      increased number apoptotic hair cells dead hair <br /> hearing sensing fluid flow pressure waves associated          cells, decreased hair cell survival caused otic vesicle <br />  <br />                                                                                                                             Cell Death Disease <br />                                Mil otic vesicle hair cells zebrafish <br />                                                                   Z-y Hu et al <br /> 6 <br />  <br />                                                                                MOmil(&#956;M)                                                        WT          MisMO <br />                                      WT                      10                   20                    40 <br />                                                                                                                                hmx2 <br />                           hmx2 <br />                                                                                                                                  fgf3 <br />                           tmieb <br />                                                                                                                                fgf8a <br />                           fgf8a <br />                                                                                                                                 foxi1 <br />                             fgf3 <br />  <br />                                                                                                                                otop1 <br />                            foxi1 <br />  <br />                          pax2.1                                                                                               pax2.1 <br />  <br />                           otop1 <br />                                                                                                                                tmieb <br />                          &#946;-actin <br />                                                                                                                               &#946;-actin <br />  <br />                                                                                        Relative mRNA level <br />                                    1.2                                                                                                                 WT <br />                                                                                                                                                        10 <br />                                    1.0                                                                                                                 20 <br />                                                                                                                                                        40 <br />                                    0.8 <br />  <br />                                    0.6 <br />  <br />                                    0.4                                                                                *                 * <br />                                                                                                    * ** <br />                                    0.2            ** <br />                                    0.0 <br />                                                hmx2               tmieb          fgf8a          fgf3          foxi1              pax2.1          otop1 <br />                                                                                        MOmil injection ( M) <br />  <br />                                                                                        Relative mRNA level <br />                                                                                                                                             WT <br />                                            1                                                                                                Mis 40uM <br />                                          0.9 <br />                                          0.8 <br />                                          0.7           * <br />                                          0.6 <br />                                          0.5 <br />                                          0.4 <br />                                          0.3 <br />                                          0.2 <br />                                                                                                                                                  * <br />                                          0.1 <br />                                            0 <br />                                                 hmx2                fgf3         fgf8a         foxil1        otop1            pax2.1          tmieb <br />                                                                                    misMO injection (40  M) <br />  <br />                                                                                                         mRNA (ng/ml) <br />                                                                           WT              10                 20                   40 <br />  <br />                                                       hmx2 <br />  <br />                                                       tmieb <br />  <br />                                                        fgf8a <br />  <br />                                                            fgf3 <br />  <br />                                                        foxi1 <br />  <br />                                                   pax2.1 <br />  <br />                                                       otop1 <br />  <br />                                                   &#946;-actin <br />  <br />  <br />                                                                                            Relative mRNA <br />                                      1.0                                                                                                        WT <br />                                      0.9                                                                                                        10 <br />                                                                                                                                                 20 <br />                                      0.8                                                                                  *                     40 <br />                                      0.7 <br />                                      0.6                                   * <br />                                      0.5 <br />                                      0.4 <br />                                      0.3                                                                * <br />                                      0.2 <br />                                      0.1 <br />                                      0.0 <br />                                                hmx2                tmieb          fgf8a          fgf3             foxi1           pax2.1          otop1 <br />                                                                                        mRNA injection (ng/ml) <br />  <br />  <br />  <br />  <br /> Cell Death Disease <br />                                                                                          Mil otic vesicle hair cells zebrafish <br />                                                                                         Z-y Hu et al <br />                                                                                                                                                                                        7 <br />  <br /> structure damaged. These phenomena similar                               Zebrafish maintenance embryo collection. Zebrafish (<span id='am-28' about='obo:NCBITaxon_7954' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:487617" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Danio" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32'  ></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Brachydanio" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Celestichthys" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasAlternativeId" content="NCBITaxon:46777" datatype="xsd:string"></span>Danio</span> <br /> cochlear vestibular degeneration S1p2-knockout                                   rerio) WT Tuebingen strain originally obtained College Life <br /> mice.2 Moreover, significant results proapoptotic                              Sciences, Beijing University. The fish raised 28.5&#177;1 1C 14/10 h light&#8211; <br />                                                                                         dark cycle.27 Embryos obtained natural mating; synchronous embryos at <br /> gene zBax2 prosurvival gene zMcl1b tightly <br />                                                                                         appropriate stage collected recorded h.p.f. d.p.f.28 Embryos <br /> regulated Mil indicate Mil upstream                                   fixed 4% paraformaldehyde phosphate-buffered saline situ <br /> coregulator zBax2 zMcl1b gene expression. As                                 hybridization immersed Trizol reagent (Invitrogen) mRNA isolation. <br /> know, zMcl zBax2 interactions regulate survival/                                Pigmentation embryos past 24 h.p.f. prevented incubating in <br /> apoptosis pathway zebrafish. Kratz et al.25 reported                            phenylthiourea (PTU) (Sigma-Aldrich, Saint Louis, MO, USA) fixed. <br /> zMcl-1a zMcl1b function gatekeepers against <br />                                                                                         Synthesis MOs miles-apart capped-mRNA. Antisense <br /> apoptosis induction stimulation extrinsic pathway. <br />                                                                                         morpholino oligonucleotides MOmil misMO designed bought <br /> Knockdown zMcl-1a zMcl-1b decreases viability                                 Genetools (Philomath, OR, USA; http://www.gene-tools.com). MOmil was <br /> zebrafish embryos sensitizes zebrafish embryos                                   used inhibition miles-apart translation binding miles-apart initiation <br /> apoptosis induction. zBax2 homologous mammalian                                  sites, sequence 50 -CCGCAAACAGACGGCAAGTAGTCAT-30 . The control <br /> Bax sequence synteny, functional conservation                                misMO&#8217; designed negative injection control bases were <br /> with human Bak. Coincidentally, human healthy cells, Bak                             missense labeled underline sequence 50 -CCCCAAACACACCGCAAC- <br /> keeps silent proapoptotic role associating Mcl-1                              TACTCAT-30 . Both MOs redissolved stock solution 1.0 mmol/ml in <br />                                                                                         ddH2O. <br /> and Bcl-x(L); releasing Bak activate apoptosis                                      The full-length miles-apart cDNA sequence cloned 24 h.p.f. WT <br /> process.26 On basis literatures current                            embryos using Reverse Transcriptase polymerase chain reaction (RT-PCR) and <br /> results, propose model Mil mechanism underlining                                primers designed according zebrafish miles-apart sequence logged in <br /> the development survival hair cells neuromast                             GenBank (NM_001159970), identified sequencing (Sangon Biotech, <br /> the lateral lines zebrafish. During embryonic organo-                            Shanghai, China). This cDNA used template miles-apart capped mRNA <br /> genesis stage, Mil position signal guides migration                            synthesis using Ambion mMESSAGE mMACHINE T3 kit (Applied Biosystems, <br />                                                                                         Foster City, CA, USA), product tested quality yield by <br /> and deposition pro-neuromast primodia correct posi- <br />                                                                                         electrophoresis. <br /> tions ALL PLL; Mil knockdown led deficient <br /> deposition signals reduced neuromasts lateral                                Microinjection. For hair cell staining, 1 nl 20 mM MOmil 20 ng/ml miles- <br /> lines, just Mil mutant failed direct cardiomyo-                           apart capped mRNA separately injected WT embryos one- four-cell <br /> cytes migrate midline.5 Meanwhile, Mil                                   stage; larvae 6 d.p.f. used counting hair cells the <br /> participates activating expression hearing genes                              neuromasts o1, o2, ml1, ml2 io4 counting neuromasts post <br /> that subsequently induce development otic vesicles                               lateral lines. For otic capsule observation living larvae counting hair cells, <br />                                                                                         1 nl 20 mM MOmil 20 ng/ml miles-apart capped mRNA injected into <br /> and differentiation pro-neuromast primodia neuro-                               embryos stage. For apoptosis staining hair cells, 1 nl MOmil at <br /> mast hair cells. Since 2 days post hatching, zebrafish                              40 mM 500 mM, misMO concentrations, respectively <br /> larvae begin live independently behavioral                                 injected one- four-cell embryos; larvae collected 6 d.p.f. <br /> capacity including swim, hiding looking food,                             For RT-PCR analysis hearing-related genes, 1 nl 10, 20 40 mM MOmil, and <br /> means functional balance hearing emerged.                                     10, 20 40 ng/ml miles-apart capped mRNA injected embryonic <br /> Execution functions guaranteed                                    stage. The embryos incubated collected 48 h.p.f. For qPCR analysis <br />                                                                                         zBax2 zMcl1b, 1 nl 40 mM MOmil 50 ng/ml miles-apart capped mRNA <br /> normal hair cells normal lateral lines. Mil, probably                           separately injected embryonic stage. The embryos were <br /> as upstream regulator bcl-2 gene family, protects hair                            incubated condition above. <br /> cells apoptosis upregulating expression of <br /> prosurvival gene zMcl1b downregulating proapoptotic                                 Observation otic vesicles. The morphous structure otic vesicles <br /> gene zBax2. Knockdown miles-apart gene produce                                   WT, MOmil- miles-apart mRNA-injected zebrafish embryos 48 and <br /> excess zBAX2 deficient zMCL1B, leads plenty                                72 h.p.f. observed differential interference contrast IX71 microscope <br />                                                                                         (Olympus, Tokyo, Japan) recorded 500D digital <span id='am-42' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-43' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-46'  ></span>camera</span> (Canon, Tokyo, <br /> of free zBAX2 sensitizes hair cells apoptosis. <br />                                                                                         Japan). Each group 10 larvae otolith counting histogram. <br />    This study provides evidence miles-apart gene <br /> function involved development otic vesicles                        Whole-mount situ hybridization. For vitro synthesis RNA <br /> lateral line hair cells neuromasts zebrafish,                        probe, miles-apart gene sequence inserted pBluescript KS (Agilent, <br /> change Mil level regulate expression bcl-2 family                         Santa Clara, CA, USA) used template miles-apart sense and <br /> gene zBax2 Mcl1B.                                                                   antisense RNA probe synthesis using DIG RNA labeling kit (Roche Applied <br />                                                                                         Sciences, Basel, Switzerland). For determination WT miles-apart expression <br />                                                                                         patterns, embryos collected 24, 48 72 h.p.f. Embryos treated with <br /> Materials Methods                                                                   MOmil mRNA injections, WT controls, collected 48 h.p.f. <br /> Ethics statement. This study carried strict accordance              Embryos allowed develop past 24 h.p.f. pretreated PTU suppress <br /> recommendations Regulation Management Laboratory Animals           pigmentation fixation 4% paraformaldehyde overnight 4 1C. Embryos <br /> the Ministry Science Technology China. The protocol approved           treated DNase hybridization remove pseudo-positive <br /> the Committee Ethics Animal Experiments Institute Medicinal         interference caused genomic DNA. Subsequent steps whole-mount situ <br /> Biotechnology, Chinese Academy Medical Sciences (IMBF20060302).                      hybridization procedure followed standard protocol.29 <br />  <br />  <br /> Figure 4 Expression hearing-associated genes down- upregulated miles-apart zebrafish larvae. Transcription hearing-associated genes hmx2, fgf3, <br /> fgf8a, foxi1, otop1, pax2.1 tmieb examined RT-PCR 48 h.p.f. embryos, reduced concentration-dependent manner morphants <br /> (a c) elevated miles-apart mRNA-injected embryos (e f). The upper panels representative RT-PCR electrophoresis results MOmil (a) miles-apart <br /> mRNA-injected groups (e), respectively, corresponding histograms density-scanning triplicate results RT-PCR (c f). The mock control group was <br /> injected misMO showed (b) (d). *P-values o0.05, **P-values o0.01 <br />  <br />                                                                                                                                                                    Cell Death Disease <br />                                                Mil otic vesicle hair cells zebrafish <br />                                                                                   Z-y Hu et al <br /> 8 <br />  <br />                                                      WT                                  MO40               MO500                misMO40      misMO500 <br />  <br />  <br />                                  Hoechest <br />                                    33342 <br />  <br />  <br />  <br />  <br />                                  YO-PRO-I <br />  <br />  <br />  <br />  <br />                                  Propidium <br />                                     iodide <br />  <br />  <br />  <br />  <br />                                                                                      Comparison hair cell survival ML1 neuromasts <br />                                                                                 30 <br />                                                           cells neuromast <br />                                                             Number hair <br />  <br />  <br />  <br />  <br />                                                                                 25 <br />                                                                                 20 <br />                                                                                 15 <br />                                                                                 10                                 * <br />                                                                                                                * <br />                                                                                  5 <br />                                                                                  0 <br />                                                                                       h <br />  <br />  <br />  <br />  <br />                                                                                                     h <br />  <br />  <br />  <br />  <br />                                                                                                                h <br />  <br />  <br />  <br />  <br />                                                                                                                              h <br />  <br />  <br />  <br />  <br />                                                                                                                                        h <br />                                                                                 pt <br />  <br />  <br />  <br />  <br />                                                                                               pt <br />  <br />  <br />  <br />  <br />                                                                                                          pt <br />  <br />  <br />  <br />  <br />                                                                                                                        pt <br />  <br />  <br />  <br />  <br />                                                                                                                                  pt <br />                                                                                 ap tal <br />  <br />  <br />  <br />  <br />                                                                                               ap tal <br />  <br />  <br />  <br />  <br />                                                                                                          ap tal <br />  <br />  <br />  <br />  <br />                                                                                                                        ap tal <br />  <br />  <br />  <br />  <br />                                                                                                                                  ap tal <br />                                                                                    at <br />  <br />  <br />  <br />  <br />                                                                                                  at <br />  <br />  <br />  <br />  <br />                                                                                                             at <br />  <br />  <br />  <br />  <br />                                                                                                                           at <br />  <br />  <br />  <br />  <br />                                                                                                                                     at <br />                                                                                   o <br />  <br />  <br />  <br />  <br />                                                                                                 o <br />  <br />  <br />  <br />  <br />                                                                                                            o <br />  <br />  <br />  <br />  <br />                                                                                                                          o <br />  <br />  <br />  <br />  <br />                                                                                                                                    o <br />                                                                                  to <br />  <br />  <br />  <br />  <br />                                                                                                to <br />  <br />  <br />  <br />  <br />                                                                                                           to <br />  <br />  <br />  <br />  <br />                                                                                                                         to <br />  <br />  <br />  <br />  <br />                                                                                                                                   to <br />                                                                                        WT       MO40        MO500       misMO40 misMO500 <br />  <br />                                                      WT                                     MO40             MO500               misMO40      misMO500 <br />  <br />  <br />  <br />                                  Hoechest <br />                                    33342 <br />  <br />  <br />  <br />  <br />                                  YO-PRO-I <br />  <br />  <br />  <br />  <br />                                  Propidium <br />                                     iodide <br />  <br />  <br />  <br />  <br />                                                                                      Comparison hair cell survival O2 neuromasts <br />                                                           cells neuromast <br />  <br />  <br />  <br />  <br />                                                                                 30 <br />                                                             Number hair <br />  <br />  <br />  <br />  <br />                                                                                 25 <br />                                                                                 20 <br />                                                                                 15 <br />                                                                                                                ** * <br />                                                                                 10 <br />                                                                                  5 <br />                                                                                  0 <br />                                                                                        h <br />  <br />  <br />  <br />  <br />                                                                                                     h <br />  <br />  <br />  <br />  <br />                                                                                                                h <br />  <br />  <br />  <br />  <br />                                                                                                                              h <br />  <br />  <br />  <br />  <br />                                                                                                                                        h <br />                                                                                  pt <br />  <br />  <br />  <br />  <br />                                                                                               pt <br />  <br />  <br />  <br />  <br />                                                                                                          pt <br />  <br />  <br />  <br />  <br />                                                                                                                        pt <br />  <br />  <br />  <br />  <br />                                                                                                                                  pt <br />                                                                                  ap tal <br />  <br />  <br />  <br />  <br />                                                                                               ap tal <br />  <br />  <br />  <br />  <br />                                                                                                          ap tal <br />  <br />  <br />  <br />  <br />                                                                                                                        ap tal <br />  <br />  <br />  <br />  <br />                                                                                                                                  ap tal <br />                                                                                     at <br />  <br />  <br />  <br />  <br />                                                                                                  at <br />  <br />  <br />  <br />  <br />                                                                                                             at <br />  <br />  <br />  <br />  <br />                                                                                                                           at <br />  <br />  <br />  <br />  <br />                                                                                                                                     at <br />                                                                                     o <br />  <br />  <br />  <br />  <br />                                                                                                  o <br />  <br />  <br />  <br />  <br />                                                                                                             o <br />  <br />  <br />  <br />  <br />                                                                                                                           o <br />  <br />  <br />  <br />  <br />                                                                                                                                     o <br />                                                                                   to <br />  <br />  <br />  <br />  <br />                                                                                                to <br />  <br />  <br />  <br />  <br />                                                                                                           to <br />  <br />  <br />  <br />  <br />                                                                                                                         to <br />  <br />  <br />  <br />  <br />                                                                                                                                   to <br />  <br />  <br />  <br />  <br />                                                                                       WT        MO40       MO500       misMO40    misMO500 <br />  <br />           Figure 5 Apoptosis hair cells prompted mil knockdown. Zebrafish larvae 6 d.p.f. stained dye mixed solution 30 min 28 1C, following <br />           kit instruction. (a) Images neuromast ml1 larvae injected MOmil 40 500 mM, misMO 40 500 mM, separately, compared WT ml1 neuromast. <br />           (b) A corresponding histogram Figure 5a. (c) Images neuromast O2 treated the neuromast ml1 MOmil misMO, separately. (d) A corresponding <br />           histogram Figure 5c. *P-values o0.05, **P-values o0.01 <br />  <br /> Cell Death Disease <br />                                                                                                              Mil otic vesicle hair cells zebrafish <br />                                                                                                             Z-y Hu et al <br />                                                                                                                                                                                                                  9 <br />  <br />                                       bax2                                               bax2               counted shot. Then, dyed hair cells classified based kit <br />                        700                                                    120                           instruction: &#8216;Blue-fluorescent Hoechst33324 brightly stains condensed <br />  Relative ratio (%) <br />  <br />  <br />  <br />  <br />                                                          Relative ratio (%) <br />                                        ** <br />                        600                                                    100 <br />                        500                                                                                  chromatin apoptotic cells dimly stains normal chromatin live <br />                                                                                80 <br />                        400 <br />                                                                                60                           cells; Green-fluorescent YO-PRO-1 dye enter apoptotic cells intact plasma <br />                        300                                                                                  membrane, red-fluorescent propidium iodide (PI) cannot. Thus, after <br />                        200                                                     40 <br />                        100                                                     20                           staining YO-PRO-1 dye PI, apoptotic cells green fluorescence and <br />                                                                                                  ** <br />                          0                                                      0                           dead cells primarily red fluorescence green fluorescence&#8217; <br />                              WT       MO       misMO                                WT          mRNA <br />                                                                                                             (Invitrogen, Catalog no. V23201).32 <br />                                       mcl1b                                              mcl1b <br />                        120 <br />                                                                               350                           RT-PCR qPCR. Embryos larvae selected stages collected into <br />   Relative ratio (%) <br />  <br />  <br />  <br />  <br />                                                          Relative ratio (%) <br />                        100                                                                            ** <br />                                                                               300                           Trizol Reagent. Total RNA extracted following Trizol Reagent RNA <br />                         80                                                    250 <br />                         60                                                    200                           extraction kit manual. First-strand cDNAs synthesized reverse <br />                         40 <br />                                                                               150                           transcription using M-MLV RTase cDNA Synthesis Kit (Takara Bio, Shiga, <br />                                                                               100                           Japan). For RT-PCR amplification, 0.5 lL cDNA used reaction <br />                         20                                                     50 <br />                          0 <br />                                         ** <br />                                                                                 0                           different primer pairs (primer sequences listed Table 1). b-Actin was <br />                              WT        MO      misMO                                WT           mRNA       amplified template loading control. PCR parameters 94 1C 60 s, 55 1C <br />                                                                                                             50 s, 72 1C 60 s, 30 cycles hearing-related genes, 26 cycles miles-apart <br /> Figure 6 Mil involved regulation proapoptotic/prosurvival gene <br />                                                                                                             25 cycles b-actin. A final incubation 72 1C 10 min included to <br /> expression. The larvae 6 d.p.f. (for MO misMO groups) 4 d.p.f. (for <br />                                                                                                             ensure complete synthesis. RT-PCR products subjected agarose gel <br /> mRNA group) collected examine transcription level zBax2 zMcl1b by <br />                                                                                                             electrophoresis. For statistical analysis density, results normalized to <br /> qPCR. The proapoptotic gene Bax2 mRNA notably elevated MOmil injection <br />                                                                                                             b-actin WT. For detection apoptosis gene zBax2 prosurvival <br /> (a) reduced mil-mRNA injection (b) compared WT misMO injection. <br />                                                                                                             gene zMcl1b, larvae 6 d.p.f. (for MO misMO injection groups) at <br /> Oppositely, prosurvival gene Mcl1b significantly downregulated morphants <br />                                                                                                             4 d.p.f. (for mRNA injection group) collected Trizol reagent; different <br /> (c) upregulated larvae mil-mRNA injection (d) **P-values o0.01 <br />                                                                                                             collecting time chosen based preliminary experiments. Transcription <br />                                                                                                             levels zBax2 zMcl1b examined qPCR (Agilent Technologies <br /> Table 1 Primer sequences PCR PCR sizes                                                              Stratagene Mx3000P, Santa Clara, CA, USA) parameters 95 1C 5 min, <br />                                                                                                             95 1C 30 s, 55 1C 30 s, 72 1C 60 s, 40 cycles. Then ct values analyzed <br /> Gene                                     Primer sequence                                    PCR size   statistics calculation. <br />                                                                                                    (bp) <br />  <br /> mil                               F: 50 -GCTTTGTCCGCCTCCGTTTTC-30                                     314   Statistical analysis. Statistical analyses data (means&#177;S.D.) were <br />                                   R: 50 -GCTGGTGCGCACAATGAGATAGAT-30                                        performed using One way ANOVA tests P-values, 0.05 assigned as <br /> hmx2                              F: 5&#8217;-GAGGACAGCGGAAGCAAG-3&#8217;                                         663   significant. Data bars histograms represent means S.D.&#8217;s are <br />                                   R: 5&#8217;-GGGCATCCCAACCAAAGT-3&#8217; <br /> fgf3                              F: 5&#8217;-TCCGAGTTTGGAGGAATC-3&#8217;                                         730   derived triplicates. <br />                                   R: 5&#8217;-TTAAATGTCAGCCCTTCTGT-3&#8217; <br /> fgf8a                             F: 5&#8217;-ACGGTTGAGTTATCTATTCC-3&#8217;                                       594 <br />                                   R: 5&#8217;-GTGCGTTTAGTCCGTCTG-3&#8217;                                               Conflict Interest <br /> foxi1                             F: 5&#8217;-CGGCTGGCAGAACTCTAT-3&#8217;                                         551   The authors declare conflict interest. <br />                                   R: 5&#8217;-TGTTGACGCTGAAATGGT-3&#8217; <br /> otop1                             F: 5&#8217;-CGAGATGGGATACGCAAAC-3&#8217;                                        592 <br />                                   R: 5&#8217;-CAGCAGCGAGTAGGTCAGG-3&#8217; <br /> pax2.1                            F: 5&#8217;-GTCAGGCAAAGAATCGTG-30                                         621   Acknowledgements. We thank Mr Weixian Wang supporting in <br />                                   R: 5&#8217;-CAGGTGCTTCCGTAAACT-30                                               microinjection, thank professor Bo Zhang (Beijing University) provided zebrafish <br /> tmieb                             F: 50 -GTAACGGCGGTGTTGATC-30                                        350   TU stain seedling. This study supported The National Natural Science <br />                                   R: 50 -GCTGAGGTTTCCACGAGA-30                                              Foundation China (No. 30772681 81173043), Chinese National Key <br /> Bax2                              F: 5&#8217;-ATAAGCAACAGCCAGGAC-3&#8217;                                         442 <br />                                   R: 5&#8217;-CACCAGTGAAGGCAAACA-3&#8217;                                               Technology R&amp;D Program (2012BAK25B01) Presidential Foundation for <br /> Mcl1b                             F: 5&#8217;-CCAGAACTGAAAGCGCATAA-3&#8217;                                       514   governmental public welfare scientific research institutions (No. IMBF 201101). <br />                                   R: 5&#8217;-ACTCCACAAAGCCATCCC-3&#8217; <br /> b-Actin                           F: 5&#8217;-AATCCCAAAGCCAACAGA-3&#8217;                                         492 <br />                                   R: 5&#8217;-GATACCGCAAGATTCCATAC-3&#8217;                                              1. MacLennan AJ, Benner SJ, Andringa A, Chaves AH, Rosing JL, Vesey R et al. The S1P2 <br />                                                                                                                 sphingosine 1-phosphate receptor essential auditory vestibular function. Hear <br />                                                                                                                 Res 2006; 220: 38&#8211;48. <br />                                                                                                              2. Herr DR, Grillet N, Schwander M, Rivera R, Mu&#168;ller U, Chun J. Sphingosine 1-phosphate <br /> Hair cell staining counting apoptosis/death detection.                                                  (S1P) signaling required maintenance hair cells mainly activation S1P2. <br /> For hair cell counting, zebrafish larvae 6 d.p.f. anesthetized 0.016%                                J Neurosci 2007; 27: 1474&#8211;1478. <br /> Tricaine (Sigma-Aldrich) incubated fluorescent dye YO-PRO-1                                       3. Liu GW, Burns S, Huang GH, Boyd K, Proia RL, Flavell RA et al. S1P1 receptor overrides <br /> (Invitrogen) final concentration 1 mmol/l used staining hair cell                                 regulatory T cell-mediated immune suppression Akt-mTOR. Nat Immunol 2009; <br />                                                                                                                 10: 769&#8211;777. <br /> nuclei30 neuromasts, o1, ml1, ml2, o2 io431 resided the <br />                                                                                                              4. Alewijnse AE, Peters SL, Michel MC. Cardiovascular effects sphingosine-1-phosphate <br /> surface otic vesicle around. The larvae visualized using IX51                               sphingomyelin metabolites. Br J Pharmacol 2004; 143: 666&#8211;684. <br /> inverted phase contrast fluorescence microscope (Olympus). Each group                               5. Kupperman E, An S, Osborne N, Waldron S, Stainier DY. A sphingosine-1-phosphate <br /> larvae staining hair-cell counting. Photographs taken Canon                                 receptor regulates cell migration vertebrate heart development. Nature 2000; 406: <br /> 500D Camera.                                                                                                    192&#8211;195. <br />    For apoptosis/death detection hair cells, apoptosis staining kit                                6. Yelon D. Cardiac patterning morphogenesis zebrafish. Dev Dyn 2001; 222: <br /> (Chromatin Condensation/Membrane Permeability/Dead Cell Apoptosis Kit                                      552&#8211;563. <br /> Hoechst 33342/YO-PRO-1 PI Flow Cytometry. Catalog no. V23201;                                        7. Oskouian B, <span id='am-1' about='obo:NCBITaxon_141610' typeof='owl:Thing obo:NCBITaxon_4056 obo:NCBITaxon_78536 obo:NCBITaxon_4055 obo:NCBITaxon_1498440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_167487 obo:NCBITaxon_91827 obo:NCBITaxon_91888 obo:NCBITaxon_167499 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="rdfs:label" content="Saba" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saba</span> JD. Death taxis: non-mammalian models tell about <br /> Invitrogen) used inspect miles-apart gene knockdown cause                                    sphingosine-1-phosphate. Semin Cell Dev Biol 2004; 15: 529&#8211;540. <br />                                                                                                              8. Matsui T, Raya A, Callol-Massot C, Kawakami Y, Oishi I, Rodriguez-Esteban C et al. <br /> hair cell apoptosis. The dye mix solution dyes prepared fish <br />                                                                                                                 miles-apart-mediated regulation cell&#8211;fibronectin interaction myocardial migration in <br /> water following kit instruction. The living larvae 6 d.p.f. soaked                        zebrafish. Nat Clin Pract Cardiovasc Med 2007; 4: S77&#8211;S82. <br /> mixed dye solution wrapped tin foil avoid light 28 1C 30 min,                     9. Kimura A, Ohmori T, Kashiwakura Y, Ohkawa R, Madoiwa S, Mimuro J et al. Antagonism <br /> anesthetized observed fluorescence inverted microscope. The                                      sphingosine 1-phosphate receptor-2 enhances migration neural progenitor cells <br /> neuromasts, o1, ml1, ml2, o2 io4 checked hair cells ml1 o2                                  area brain. Stroke 2008; 39: 3411&#8211;3417. <br />  <br />  <br />                                                                                                                                                                                              Cell Death Disease <br />                                                      Mil otic vesicle hair cells zebrafish <br />                                                                                         Z-y Hu et al <br /> 10 <br />  <br />           10. Trifilieff A, Baur F, Fozard JR. Role sphingosine-1-phosphate (S1P)                 23. Gleason MR, Nagiel A, Jamet S, Vologodskaia M, Lo&#180;pez-Schier H, Hudspeth AJ. <br />               S1P(2) receptor allergen-induced, mast cell-dependent contraction rat lung                   The Transmembrane inner ear (Tmie) protein essential normal hearing balance in <br />               parenchymal strips. Naunyn Schmiedebergs Arch Pharmacol 2009; 380:                                 zebrafish. PNAS 2009; 106: 21347&#8211;21352. <br />               303&#8211;309.                                                                                       24. Skoura A, Hla T.. Regulation vascular physiology pathology S1P2 receptor <br />           11. Sensken SC, Gra&#168;ler MH. Sphingolipids medical impact. Dtsch Med Wochenschr               subtype. Cardiovasc Res 2009; 82: 221&#8211;228. <br />               2009; 134: 259&#8211;263.                                                                            25. Kratz E, Eimon PM, Mukhyala K, Stern H, Zha J, Strasser A et al. Functional characterization <br />           12. Shen YC, Jeyabalan AK, Wu KL, Hunker KL, Kohraman DC, Thompson DL.                                 Bcl-2 gene family zebrafish. Cell Death Differ 2006; 13: 1631&#8211;1640. <br />               The transmembrane inner ear (tmie) gene contributes vestibular lateral                  26. Willis SN, Chen L, Dewson G, Wei A, Naik E, Fletcher JI et al. Proapoptotic Bak is <br />               line development function zebrafish (<span id='am-9' about='obo:NCBITaxon_7955' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7954 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Danio rerio (Hamilton, 1822)" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasAlternativeId" content="NCBITaxon:37966" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="leopard danio" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Cyprinus rerio" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Danio rerio" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Brachidanio rerio" datatype="xsd:string"></span><span id='am-17'  ></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Danio rerio frankei" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Brachydanio rerio frankei" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="zebra danio" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Brachydanio rerio" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="zebrafish" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Danio frankei" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Cyprinus rerio Hamilton, 1822" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="zebra fish" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="NCBITaxon:27702" datatype="xsd:string"></span>Danio rerio</span>). Dev Dyn 2008; 237:                   sequestered Mcl-1 Bcl-xL, Bcl-2, displaced BH3-only proteins. <br />               941&#8211;952.                                                                                           Genes Dev 2005; 19: 1294&#8211;1305. <br />           13. Rinkwitz S, Bober E, Baker R. Development vertebrate inner ear. Ann N Y Acad Sci        27. Westerfield M. THE ZEBRAFISH BOOK: A guide laboratory use zebrafish <br />               2001; 942: 1&#8211;14.                                                                                   (Danio rerio). University Oregon Press: Eugene, OR, USA, 20004th edn linehttp:// <br />           14. Whitfield TT, Riley BB, Chiang MY, Phillips B. D <br /> </body></html>