<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                 INVESTIGATION <br />  <br />  <br />  <br />  <br />                                 The Jaw Worm: GTPase-activating Protein <br />                                          EAT-17 Regulates Grinder Formation in <br />                                                         <span id='am-39' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span>Caenorhabditis elegans</span> <br />                                                           Sarah Straud,*,1,2 Inhwan Lee,&#8224;,1 Bomi Song,* Leon Avery,&#8225; Young-Jai You&#8224;,3 <br />                                      *Department Molecular Biology, University Texas Southwestern Medical Center, Dallas, Texas 75390-9148, <br />                          &#8224;Department Biochemistry Molecular Biology, <span id='am-11' about='obo:NCBITaxon_183605' typeof='owl:Thing obo:NCBITaxon_1329950 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8504 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8509 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_1329912 obo:NCBITaxon_33208 obo:NCBITaxon_1329961 obo:NCBITaxon_1329911 obo:NCBITaxon_7711 obo:NCBITaxon_8570 obo:NCBITaxon_169862 obo:NCBITaxon_7776 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_8578 obo:NCBITaxon_34989'><span id='am-12'  ></span><span id='am-13' property="rdfs:label" content="Virginia" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Virginia</span> Commonwealth University, Richmond, Virginia 23298-0614, and <br />                                       &#8225;Department Physiology Biophysics, Virginia Commonwealth University, Richmond, Virginia 23298-0551 <br />  <br />  <br />  <br />  <br /> ABSTRACT Constitutive transport cellular materials essential cell survival. Although multiple small GTPase Rab proteins are <br /> required process, regulators Rabs known. Here report EAT-17, novel GTPase-activating protein (GAP), <br /> regulates RAB-6.2 function grinder formation <span id='am-46' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-47' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49'  ></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Caenorhabditis</span> elegans. We identi&#65533;?ed EAT-17 novel RabGAP interacts with <br /> RAB-6.2, protein presumably regulates vesicle traf&#65533;?cking Golgi, endoplasmic reticulum, plasma membrane to <br /> form functional grinder. EAT-17 canonical GAP domain critical function. RNA interference 25 con&#65533;?rmed <br /> and/or predicted RABs C. elegans shows RNAi rab-6.2 produces phenotype identical eat-17. A directed yeast two- <br /> <span id='am-167' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-168'  ></span><span id='am-169' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>hybrid</span> screen using EAT-17 bait 25 RAB proteins prey identi&#65533;?es RAB-6.2 interacting partner EAT-17, <br /> con&#65533;?rming RAB-6.2 speci&#65533;?c substrate EAT-17. Additionally, deletion mutants rab-6.2 grinder defects identical to <br /> those eat-17 loss-of-function mutants, RAB-6.2 EAT-17 expressed terminal bulb pharynx the <br /> grinder located. Collectively, results suggest EAT-17 speci&#65533;?c GTPase-activating protein RAB-6.2. Based the <br /> conserved function Rab6 vesicular transport, propose EAT-17 regulates turnover rate RAB-6.2 activity cargo <br /> traf&#65533;?cking grinder formation. <br />  <br />  <br />  <br /> C   ELLS constitutively transport newly synthesized proteins, <br />     lipids, molecules periphery through <br /> vesicle traf&#65533;?cking. As process conserved yeast to <br />                                                                                        et al. 1992; Martinez et al. 1994, 1997; Girod et al. 1999; <br />                                                                                        Opdam et al. 2000; Del Nery et al. 2006). For fast turnover, <br />                                                                                        Rabs require guanine nucleotide exchange factors (Rab GEFs) <br /> humans, essential function survival cells.                        activation GTPase-activating proteins (Rab GAPs) to <br /> Rab GTPases small GTP binding proteins regulate                               turn activity (Grosshans et al. 2006). While role Rab6 <br /> the transport vesicles different compartments                            membrane traf&#65533;?cking established, GAPs or <br /> the cell (Zerial McBride 2001; Jordens et al. 2005;                                GEFs Rab6 identi&#65533;?ed; physiologi- <br /> Grosshans et al. 2006). Rab6&#8217;s localized Golgi mem-                             cal importance largely unknown. The physiological roles of <br /> branes mark target anterograde cargos                                 RabGAPs addressed recently &#64258;ies (Houalla et al. <br /> Golgi post-Golgi compartments (such plasma                                   2010; Uytterhoeven et al. 2011) worms (Chotard et al. <br /> membrane) retrograde cargos early/recycling en-                               2010). In cases, removing function RabGAPs <br /> dosomes Golgi endoplasmic reticulum (Jasmin                                 produced phenotypes identical removing the <br />                                                                                        relevant Rabs, showing fundamental roles RabGAPs in <br />                                                                                        modulating Rab functions. Currently, identi&#65533;?ed GAP <br /> Copyright &#169; 2013 Genetics Society America <br /> doi: 10.1534/genetics.113.152538 <br />                                                                                        Rab6 GAPCenA humans, associated with <br /> Manuscript received April 25, 2013; accepted publication June 12, 2013             centrosomes regulates Golgi dynamics dividing cells <br /> Supporting information available online http://www.genetics.org/lookup/suppl/    (Cuif et al. 1999). Nonetheless, misregulation GAPs for <br /> doi:10.1534/genetics.113.152538/-/DC1. <br /> 1 <br />   These authors contributed equally work.                                      small G proteins, Ras, directly relates diseases <br /> 2 <br />   Present address: Department Biological Physical Sciences, Saint Augustine&#8217;s   <span id='am-34' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="rdfs:label" content="Cancer" datatype="xsd:string"></span>cancer</span>, implicating essential roles GAPs con- <br />   University, Raleigh, NC 27610-2298. <br /> 3 <br />  Corresponding author: P. O. Box 980614, 1220 E. Broad St., MMRB Bldg, Room <br />                                                                                        trolling kinetics G protein activity (Tanabe et al. 2006; <br />  2046, Richmond, VA 23298. E-mail: yjyou@vcu.edu                                       Durkin et al. 2007; Pamonsinlapatham et al. 2009). <br />  <br />  <br />  <br />                                                                                                      Genetics, Vol. 195, 115&#8211;125 September 2013     115 <br />      The Caenorhabditis elegans grinder complex structure     SNP mapping <br /> required proper grinding food (bacteria) is <br />                                                                   CB4856 males crossed DA773 hermaphrodites. <br /> passed intestine. When food abundant, worms feed <br />                                                                   unc-93; lin-15 eat-17 sup-10/CB4856(+) worms <span id='am-152' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-153' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-154' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-156'  ></span>iso</span>- <br /> at average rate 200 pumps minute (ppm) based on <br />                                                                   lated F2 generation ability produce 25% <br /> counts pharyngeal pumping grinding motions (Avery <br />                                                                   Muv non-Unc self-progeny near absence non-Muv <br /> and You 2012; Raizen et al. 2012). <span id='am-74' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-78'  ></span>This</span> high frequency of <br />                                                                   non-Unc self-progeny. Sup non-Muv recombinants iso- <br /> grinding worm&#8217;s life suggests transport of <br />                                                                   lated F3 generation. These scored eat-17 <br /> grinder components build functional grinder essential <br />                                                                   feeding defect, progeny analyzed PCR and <br /> for worms&#8217; rapid growth normal development. Consid- <br />                                                                   subsequent restriction digestion SNPs located between <br /> ering location structure C. elegans pharynx, <br />                                                                   lin-15 sup-10. SNPs examined primers used for <br /> grinder components likely produced terminal bulb <br />                                                                   ampli&#65533;?cation shown Supporting Information, <br /> muscles pm6 pm7 traf&#65533;?cked apical surfaces <br />                                                                   Table S1. <br /> (Albertson Thomson 1976). The exact components of <br />                                                                       Deletion mutants ser-1(ok345), hda-4(ok518), and <br /> the grinder molecular mechanisms regulating trans- <br />                                                                   T27A8.2(ok570) mapped scoring F4 progeny for <br /> port, however, remain unknown. <br />                                                                   deletion detected PCR. Primers used detecting the <br />     Here report C. elegans gene eat-17 encodes Rab <br />                                                                   deletions shown Table S2. <br /> GTPase-activating protein (Rab GAP) speci&#65533;?c RAB-6.2, <br /> a Rab6 homolog C. elegans. eat-17 loss-of-function         Analysis eat-17 gene structure identi&#65533;?cation <br /> rab-6.2 deletion mutants similar phenotypes,          splice variants <br /> disorganized, poorly formed grinders. RNAi rab-6.2        To determine intron exon structure eat-17, pri- <br /> produces similar defects. A directed yeast two-hybrid screen      mers designed based Gene&#65533;?nder predictions shown <br /> using EAT-17 bait identi&#65533;?ed RAB-6.2 interacting         WormBase amplify partial full-length transcripts by <br /> partner. Furthermore, transgenes mutations con-       RT&#8211;PCR. First-strand poly(A)+ cDNA derived mixed <br /> served catalytic arginine residue required GAP activ-     stage N2 hermaphrodites used template reac- <br /> ity EAT-17 failed rescue grinder defects injected      tions. Trizol (Invitrogen) used isolate RNA the <br /> into eat-17 mutants. Based results con-          First-Strand cDNA Synthesis kit (Roche) used gener- <br /> served function Rab6 vesicle transport, propose     ate cDNA. Primers used amplify PCR fragments shown <br /> RAB-6.2 EAT-17 cooperate transport grinder          Table S3. All PCR reactions yielded single product, and <br /> components apical surfaces terminal bulb            exons &#65533;?rst appeared correct. To assess <br /> muscles form functional grinder.                             number abundance different splice variants, full- <br />                                                                   length eat-17 cDNAs cloned pGEM-T Easy <br /> Materials Methods                                             vector (Promega) sequenced. <br /> Worm culture                                                      SL1 trans-splicing <br /> Worms cultured handled described (Sulston             To identify &#65533;?rst exon eat-17, forward primer recog- <br /> and Hodgkin 1988) following modi&#65533;?cations. First,         nizing 22 nucleotide-SL1-splice leader sequence (59- <br /> they routinely grown NGMSR plates (Avery 1993).           ggtttaattacccaagtttgag -39) nested reverse primers <br /> NGMSR differs NGM containing 200 pg/ml strepto-           recognizing sequences exon 2 [1050: 59-tgttcagctgctc <br /> mycin sulfate, 10 pg/ml nystatin, 2% agar instead          catcttg-39(outside primer)/1051: 59-cgacttcatttacgcatactg-39 <br /> 1.7%. (Streptomycin nystatin reduce bacterial fun-        (inside primer)] used. Products puri&#65533;?ed (Qiaquick <br /> gal contamination. The higher agar concentration delays       Gel Extraction kit), cloned pGEM-T Easy <span id='am-136' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-137' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> se- <br /> burrowing worms agar.) In addition, worms     quenced. The resulting sequences BLASTed (blastn) <br /> were maintained 20  <span id='am-102' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-103' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-110'  ></span><span id='am-111' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span>Escherichia coli</span> HB101 unless in-       C. elegans ESTs genomic DNA using WormBase. <br /> dicated otherwise. The wild-type strains Bristol N2 and <br />                                                                   Cosmid rescue <br /> Hawaiian CB4856. eat-17(ad707) rab-6.2(ok2254) were <br /> used study outcrossed twice        Cosmids used study obtained Alan Coulson <br /> N2. Other strains used study DA2120 adEx2120          (Wellcome Trust, UK). T24D11 F01G12 isolated <br /> [eat-17p::GFP rol-6(sd)], DA773 unc-93(e1500sd); lin-15(n309)     using Qiagen Plasmid Mini kit co-injected into <br /> eat-17(ad707) sup-10(n183), DA1814 ser-1(ok345), RB758            eat-17 mutants (50 ng/ml). <span id='am-172' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-173' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-174' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-175' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Plasmids</span> pPD118.20 myo-3:: <br /> hda-4(ok518), RB787 T27A8.2(ok570), DA707 eat-17(ad707),          GFP (from Andrew Fire, Stanford University) pRAK3 <br /> NL2099 rrf-3(pk1426), DA1983 eat-17(ad707); adEx1983              rol-6(d) used co-injection markers concentration <br /> [F01G12 T24D11 rol-6(d)], DA1982 rrf-3(pk1426); eat-17            10 ng/ml. Worms fed <span id='am-94' about='obo:NCBITaxon_283' typeof='owl:Thing obo:NCBITaxon_80840 obo:NCBITaxon_28216 obo:NCBITaxon_80864 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2'><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="Comamonas" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasExactSynonym" content="Comimonas" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasRelatedSynonym" content="Comomonas" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="&quot;Comamonas&quot; Davis and Park 1962" datatype="xsd:string"></span><span id='am-101'  ></span>Comamonas</span> injection to <br /> (ad707), DA2035 adEx2035[rab-6.2p::GFP rol-6(d)], DA2033          allow good growth (Avery Shtonda 2003). Transgenic <br /> eat-5(ad1402); adEx2033[RAB-6.2::GFP unc-122::RFP eat-5           worms fed DA837 enhance feeding defect. <br /> (+)], YJ89 rab-6.2(ok2254), MT993 lin-10(e1439); him-5               Growth rates transgenic nontransgenic progeny <br /> (e1467ts).                                                        compared assess rescue. Eggs transgenic mothers <br />  <br />  <br />  <br /> 116    S. Straud et al. <br />  were placed individually plates seeded DA837            &#183;100 objective <span id='am-16' about='obo:NCBITaxon_3863' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_163743 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Lens Mill." datatype="xsd:string"></span><span id='am-18'  ></span><span id='am-19' property="rdfs:label" content="Lens" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>lens</span>. Images acquired using Zeiss <br /> and checked 12 hr stage worms. Growth           Axiovision software. <br /> rate de&#65533;?ned inverse time required worms <br />                                                                   RNAi <br /> to reach adulthood produce progeny. Worms scored <br /> as adults laid egg.                      RNAi performed described minor modi&#65533;?cations <br />                                                                   (Kamath Ahringer 2003). cDNA sequences eat-17 and <br /> Generating eat-17 rescue construct <br />                                                                   25 C. elegans Rabs ampli&#65533;?ed PCR using primers <br /> Rescuing fragments containing full-length eat-17 cDNA         shown Table S5. The HiScribe RNAi Transcription kit <br /> were generated overlap extension PCR (Ho et al. 1989).         (New England Biolabs) used generate dsRNAs. <br /> In &#65533;?rst round PCR reactions, 59 39 eat-17 cDNA fragments <br />                                                                   GFP fusions EAT-17 RAB-6.2 <br /> were ampli&#65533;?ed using primers: 59-ttgtcaccgccgatggcagccactg <br /> cagcgctac&#8211;39 59-tagggatgttgaagagtaattggacctagtggctatcc        eat-17p::GFP: A 5.6-kb eat-17 promoter ampli&#65533;?ed from <br /> gacagtt&#8211;39. The resulting PCR fragments puri&#65533;?ed          N2 genomic DNA using following primers: 59-taggt <br /> subjected second round PCR using overlap exten-       tacggtagttggtacg-39 59-gaaaagttcttctcctttactcatcggcggt <br /> sion method fuse eat-17 promoter unc-54 39-UTR         gacaattgg-39. In addition 59 regulatory sequence, this <br /> to end. The primers used amplify promoter region      DNA contains &#65533;?rst codons eat-17. GFP ampli- <br /> are: 59-taggttacggtagttggtacg&#8211;39 59-gtagcgctgcagtggctgc       &#65533;?ed pPD95.75 plasmid DNA (gift A. Fire, Stanford <br /> catcggcggtgacaa&#8211;39. The primers used amplify 500 bp unc-54     University) using primers: 59-ccaattgtcaccgccgatgagtaaagga <br /> 39 UTR 59-aactgtcggatagccactaggtccaattactcttcaacatcccta&#8211;39    gaagaacttttc-39 59-tttggtatattgggaatggtattctg-39. DNAs were <br /> and 59-tttggtatattgggaatgtattctg&#8211;39. The resulting PCR prod-      fused overlap extension PCR using following <br /> ucts puri&#65533;?ed injected eat-17 worms (25 ng/ml).      nested primers: 59-acggtagtgttttatcagtagtg-39 59-caaacc <br /> The &#65533;?nal injected product contained 5.6 kb sequence            caaaccttcttccgatc-39. <br /> located directly upstream eat-17 transcriptional start        The resulting product puri&#65533;?ed injected N2 <br /> site. A 2.5-kb full-length eat-17 cDNA construct fused        adults pRAK3 rol-6(d) co-injection marker. The <br /> to &#8220;promoter region&#65533;? heterologous unc-54 39-         established transgenic lines observed using Zeiss <br /> UTR added promote stability. This construct lacked         Axio A2 Imager &#183;630 &#183;1000 magni&#65533;?cation. <br /> intron elements, including large 4-kb &#65533;?rst intron. let-       Images acquired using Zeiss Axiovision software. <br /> 858::GFP plasmid DNA (18 ng/ml) used co-injection <br /> marker.                                                           rab-6.2p::GFP: A 2.8-kb rab-6.2 promoter fragment was <br />    Two transgenic lines isolated examined. F2            ampli&#65533;?ed N2 genomic DNA using following pri- <br /> transgenic animals identi&#65533;?ed using SZX12GFP dis-         mers: 59-aatcgcacagcaggcctcc-39 59-gtgaaaagttcttctcctt <br /> secting scope (Olympus). Because transgenic embryos          tactcggattaccaaagtccgacat-39. In addition 59 regulatory <br /> die, animals reached adulthood scored          sequence, DNA fragment contains &#65533;?rst seven codons <br /> the grinder phenotype. For transgenic lines, 100%         rab-6.2. GFP ampli&#65533;?ed plasmid pPD95.75 using <br /> the adult transgenic animals rescued defects          following primers: 59-atgtcggactttggtaatccgagtaaaggagaa <br /> grinder formation. Grinder morphology observed using          gaacttttcac-39 59-tttggtatattgggaatggtattctg-39. Products <br /> a Zeiss Axio A2 Imager &#183;630 &#183;1000 magni&#65533;?ca-          fused overlap extension PCR using following <br /> tion. Images acquired using Zeiss Axiovision software.       primers: 59-tttgccgaacggaagagcc-39 59-caaacccaaaccttcttcc <br />                                                                   gatc-39. The &#65533;?nal product puri&#65533;?ed injected N2 <br /> Assay EAT-17 GAP activity <br />                                                                   adults rol-6(d) co-injection marker. GFP expression <br /> To determine functional importance GAP activity        observed described above. <br /> of EAT-17, rescuing fragments containing catalytically <br />                                                                   Yeast two-hybrid assay <br /> inactive R116/119K R116/119A mutations gener- <br /> ated using overlap extension method above.        To generate &#65533;?rst-strand cDNA, RNA isolated a <br /> Primers used generate R116/119K mutation               well-fed, mixed stage N2 population using Trizol. Poly(A)+ <br /> shown Table S4. The resulting PCR products gel            &#65533;?rst-strand cDNA prepared using First-Strand cDNA <br /> puri&#65533;?ed injected eat-17 worms concentration         Synthesis kit RT&#8211;PCR (Roche). <br /> of 25 ng/ml. let-858::GFP plasmid DNA used co-              Inserts used preys ampli&#65533;?ed PCR using the <br /> injection marker concentration 100 ng/ml. Wild-type,      Expand Long Template PCR kit (Roche). Primers amplify <br /> R116/119K, R116/119A injections performed             preys shown Table S6. PCR products cloned <br /> parallel. To score rescue grinder defects, F1 gravid   pGEM-T Easy vector sequenced. Constructs with <br /> adult progeny injected mothers picked 4% agar          wild-type sequence subcloned pACT2 vector (gift <br /> pads containing 10 mM sodium azide (Sigma, St. Louis).            R. Lin, University Texas Southwestern Medical Center <br /> Defects grinder formation recorded           Dallas), frame GAL4 activation domain. <br /> presence absence GFP expression determined.                Six truncated versions eat-17 used baits after <br /> Worms observed using Zeiss Axio A2 Imager             ampli&#65533;?cation, sequencing cloning pVJL11 vector <br />  <br />  <br />  <br />                                                                                             EAT-17 RabGAP RAB-6.2   117 <br />                                                                                                     Figure 1 eat-17 mutants show <br />                                                                                                    defects grinder formation. (A) <br />                                                                                                    DIC image wild-type pharynx. <br />                                                                                                    The C. elegans grinder com- <br />                                                                                                    posed plates arranged <br />                                                                                                    approximate triradial symme- <br />                                                                                                    try. Two fully formed plates are <br />                                                                                                    visible lateral sections (arrow). <br />                                                                                                    (B) DIC image eat-17(ad707) <br />                                                                                                    mutant pharynx. Most ma- <br />                                                                                                    terial comprising grinder plates <br />                                                                                                    missing eat-17 mutants. (C) <br />                                                                                                    The intestine wild-type worms <br />                                                                                                    fed mCherry-expressing E. coli <br />                                                                                                    shows smooth &#64258;uorescence due <br />                                                                                                    &#64258;uorescence released from <br />                                                                                                    ground bacteria. (D) The intestine <br />                                                                                                    eat-17 mutants many <br />                                                                                                    unground bacteria indicated by <br />                                                                                                    arrow. (E F) Correspond- <br />                                                                                                    ing DIC images wild type (E) <br />                                                                                                    eat-17 mutant (F), respec- <br />                                                                                                    tively. In F, arrow indicates <br />                                                                                                    DIC image unground bacteria. <br />                                                                                                    For images, anterior shown <br />                                                                                                    left. Bars, 10 mm. <br />  <br />  <br />  <br /> (gift M. Cobb, University Texas Southwestern Medical     bacteria slightly asynchronous terminal bulb con- <br /> Center Dallas), frame LexA DNA binding do-        tractions feeding (Avery 1993). The striking <br /> main. Primers shown Table S7. Mutant versions      phenotype, however, defect grinders; wild-type <br /> baits generated using vitro mutagenesis kit (Invi-    grinders consist highly refractile cuticle ridges be <br /> trogen). Primers containing mutations shown Table S4.     observed DIC optics (Figure 1A, arrow). These ridges are <br />    To test interactions, combinations baits preys     completely disorganized eat-17 mutants (Figure 1B). In <br /> were transformed yeast strain L40 (Hama et al. 1999).   wild-type worms, grinder composed separate <br /> X-gal &#65533;?lter assays quantitative ortho-nitrophenyl-b-D-       plates arranged approximate triradial symmetry <br /> galactopyranoside (ONPG) assays performed de-            (Albertson Thomson 1976; von Lieven 2003). These <br /> scribed (Yeast Protocols Handbook, Clontech). An average         plates secreted apical surfaces pharyngeal <br /> of colonies tested each. For growth assays,          muscle cells pm6 pm7 terminal bulb (Albertson <br /> colonies streaked plates lacking histidine         Thomson 1976). During feeding, plates rotate <br /> supplemented 5 mM 3-amino-1,2,4-triazole (3-AT).            another, grinding material comes be- <br /> Miller units calculated OD420/t &#183; V &#183; OD600.            tween them. Due defects grinder, eat-17 mutants <br />                                                                  grind bacteria ef&#65533;?ciently. As result, unground bac- <br /> Chitin staining <br />                                                                  teria pass intestine (Figure 1, C&#8211;F) contributing to <br /> Chitin stained described (Ruvkun Finney 2005).        Eat phenotype. <br /> Fluorescein-conjugated chitin-binding probe (P5211S, NEB) <br /> was used 1:100 dilution.                                    eat-17 encodes GTPase activating protein with <br />                                                                  coiled-coil domains C terminus <br /> Grinder function assay <br />                                                                  Previous work placed eat-17 chromosome X, between <br /> Worms fed mCherry-expressing HB101 described             lin-15 sup-10 (Avery 1993). We mapped eat-17 <br /> (You et al. 2008). Images acquired using Zeiss Axiovi-      47-kb region SNP markers, pkP6096 and <br /> sion software.                                                   snp-F38E9.2 (Figure 2A), using multipoint SNP mapping <br />                                                                  strategy (Wicks et al. 2001). Three genes predicted in <br /> Results                                                          interval (Figure 2B): F01G12.1 encodes putative copper <br />                                                                  transporter, T24D11.1 (currently designated tbc-4) pre- <br /> eat-17 mutants Eat defects <br />                                                                  dicted encode Rab GAP containing TBC (Tre2/Bub/ <br /> in grinder formation <br />                                                                  Cdc16 homology) domain&#8212;a canonical GAP domain&#8212;and <br /> eat-17 mutants isolated genetic screen           F01G12.6 predicted encode protein several <br /> defects feeding behavior. They defective trapping      coiled-coil domains. Interestingly, T24D11.1 F01G12.6 <br />  <br />  <br />  <br /> 118    S. Straud et al. <br />                                                                                                        Figure 2 eat-17 gene structure. <br />                                                                                                       (A) Region C. elegans genome <br />                                                                                                       eat-17 maps. eat-17 was <br />                                                                                                       mapped 47 kb two <br />                                                                                                       SNP markers pkP6096 snp- <br />                                                                                                       F38E9.2 using multipoint SNP <br />                                                                                                       mapping strategy. (B) Genes pre- <br />                                                                                                       dicted located inter- <br />                                                                                                       val gene structure of <br />                                                                                                       eat-17. Red arrows indicate the <br />                                                                                                       exons targeted amplify and <br />                                                                                                       validate eat-17 transcripts. (C) <br />                                                                                                       Exon usages possible <br />                                                                                                       splice forms deduced SL1 <br />                                                                                                       RT&#8211;PCR analysis. The red <br />                                                                                                       box indicates exon 4B. The two <br />                                                                                                       different splice forms eat-17a <br />                                                                                                       eat-17b encode proteins <br />                                                                                                       differing amino acids. <br />                                                                                                       (D) Predicted protein structure <br />                                                                                                       EAT-17 location of <br />                                                                                                       mutation ad707 allele. <br />                                                                                                       An N-terminal Rab GAP domain <br />                                                                                                       followed C-terminal <br />                                                                                                       predicted coiled-coil domains. <br />                                                                                                       The arrow indicates position <br />                                                                                                       mutated amino acid in <br />                                                                                                       ad707. <br />  <br />  <br />  <br />  <br /> share homology human protein Evi5 (Figure S1),             predicted WormBase (www.wormbase.org) failed. <br /> a Rab11 GAP, mutation associated neuroblas-           Next identi&#65533;?ed individual intron/exon boundaries and <br /> toma (Liao et al. 1997; Dabbeekeh et al. 2007). Because             &#65533;?rst exon incorrectly predicted. To iden- <br /> T24D11.1 F01G12.6 transcribed direc-            tify 59 end transcript, used SL1 trans-splicing <br /> tion (Figure 2B) unable retrieve             analysis. In C. elegans,  60% transcripts either <br /> cDNAs coding individual genes RT&#8211;PCR, hy-                SL1 SL2 trans-spliced 59 ends (Blumenthal <br /> pothesized predicted genes comprised single        1995; Conrad et al. 1995). The results &#65533;?rst exon <br /> gene. RT&#8211;PCR using primers predicted exon          eat-17 located  4 kb upstream exon 2 (indicated * <br /> of T24D11.1 &#65533;?fth predicted exon F01G12.6 iden-           Figure 2B). We learned transcript contains <br /> ti&#65533;?ed single transcript (indicated arrows Figure 2B),      40 bp 59-UTR. Using primer targeting newly iden- <br /> con&#65533;?rming T24D11.1 F01G12.6 parts               ti&#65533;?ed &#65533;?rst exon targeting predicted exon of <br /> gene. Furthermore, injecting cosmids, T24D11             eat-17, able amplify single 2.5-kb fragment. To <br /> F01G12, completely cover region, rescued          con&#65533;?rm isolated full-length transcript, we <br /> growth, feeding, grinder defects eat-17 mutants. Inject-     generated rescuing transgene 2.5-kb sequence <br /> ing did rescue phenotype (<span id='am-162' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-163' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-165' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-166'  ></span>data</span>             attached  5.6-kb region sequence upstream of <br /> shown).                                                             predicted eat-17 transcriptional start site (eat-17 pro- <br />     To determine gene structure eat-17, &#65533;?rst tried        moter) 59 end 500-bp unc-54 39-UTR. When <br /> to isolate full-length transcripts intron/exon structure   construct injected eat-17 mutants, found <br />  <br />  <br />  <br />                                                                                               EAT-17 RabGAP RAB-6.2       119 <br />                                                                                                                         Figure 3 EAT-17 interacts with <br />                                                                                                                        RAB-6.2. (A) eat-17 rescue. Ex- <br />                                                                                                                        pressing wild-type eat-17 cDNA <br />                                                                                                                        construct mutant worms res- <br />                                                                                                                        cues defects grinder formation. <br />                                                                                                                        Grinder plates normal size <br />                                                                                                                        (yellow arrow) no <br />                                                                                                                        unground bacteria intes- <br />                                                                                                                        tine (red arrow). (B) Fluorescence <br />                                                                                                                        image A. Smooth &#64258;uorescence <br />                                                                                                                        intestine indicates bac- <br />                                                                                                                        teria ground (red arrow). (C) <br />                                                                                                                        rab-6.2(ok2254) mutants show <br />                                                                                                                        grinder defect. Grinder for- <br />                                                                                                                        med size small. (D) <br />                                                                                                                        eat-17(ad707) grown 15  con- <br />                                                                                                                        tainslarge refractile body the <br />                                                                                                                        terminal bulb (arrow). (E) Yeast <br />                                                                                                                        two-hybrid screen using EAT-17 <br />                                                                                                                        bait identi&#65533;?ed RAB-6.2 the <br />                                                                                                                        interacting Rab protein. (F) A <br />                                                                                                                        constitutively GTP bound form of <br />                                                                                                                        RAB-6.1 enhances binding of <br />                                                                                                                        EAT-17 RAB-6.1. On other <br />                                                                                                                        hand, constitutively GDP bound <br />                                                                                                                        form RAB-6.2 (6.2T) abolishes <br />                                                                                                                        binding EAT-17 RAB-6.2. <br />                                                                                                                        The EAT-17 construct used was <br />                                                                                                                        truncated protein (amino acids <br /> 1&#8211;460) containing &#65533;?rst coiled coil domain. The truncated protein binds RAB-6.2 ef&#65533;?ciently, suggesting &#65533;?rst coiled-coil domain suf&#65533;?cient <br /> for binding. Abbreviations follows: 6.1, RAB-6.1 full-length wild type 6.1Q, RAB-6.1 Q70L mutation; 6.1T, RAB-6.1 T27N mutation; <br /> 6.2T, RAB-6.2 T27N mutation; 6.2, RAB-6.2 full-length wild type 6.2Q, RAB-6.2 Q69L mutation. *P , 0.05; NS, statistically signi&#65533;?cant by <br /> Student t-test. Miller units calculated OD420/t &#183; V &#183; OD600. Bars, 10 mm. <br />  <br />  <br /> that 100% surviving F2 transgenic progeny rescued                      GAP) domain coiled-coil domains (CC1&#8211;3, Figure <br /> for defects grinders (Figure 3A).                                     2D). The Rab GAP domain implicates EAT-17 vesicle traf- <br />    To determine exact splicing pattern(s) eat-17,                    &#65533;?cking, role consistent grinder defects eat-17 <br /> cloned ampli&#65533;?ed PCR fragments vectors se-                         mutants. To identify speci&#65533;?c mutation eat-17(ad707), <br /> quenced inserts (see Materials Methods). We named                      sequenced predicted eat-17 coding region (based on <br /> the possible splice products EAT-17 a, b, c, d (Figure                     RT&#8211;PCR results) C-to-T base-pair transition <br /> 2C). EAT-17a intron/exon boundaries consistent                        nucleotide 607 exon 5 ad707 mutant allele, <br /> WormBase/GeneFinder predictions. EAT-17b uses alter-                        creating stop <span id='am-84' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-85'  ></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>codon</span> amino acid 203 (Figure 2D). This <br /> native splice acceptor site exon 3/4 boundary (exon                     mutation generate protein truncated Rab- <br /> 4A, labeled red, Figure 2, B C). For these,                     GAP domain, likely rendering nonfunctional. <br /> predicted exon 5 T24D11.1 truncated 180 nucleo-                         To determine EAT-17 contains GAP activity, we <br /> tides spliced directly predicted exon 2 F01G12.6                  mutated catalytic arginine residue (R116) either <br /> WormBase. Neither splice form contained predicted &#65533;?rst                     lysine (K) alanine (A) abolish GAP activity and <br /> exon F01G12.6. The proteins encoded EAT-17a                          expressed mutated constructs eat-17 mutants (see <br /> EAT-17b differ 3 amino acids. When designed                         Materials Methods). A total 21% F1&#8217;s carrying <br /> a set primers amplify sequences exon 1                       wild-type copy eat-17 rescued defects in <br /> exon 6 (corresponding predicted &#65533;?rst exon F01G12.6),                 grinder formation (Figure 3, A B), 4.1 and <br /> we isolate transcripts included exon 6. We named                    1.6% mutants expressing R116K R116A <br /> the predicted transcripts include exon 6 EAT-17c                      mutations rescued (Table 1). Collectively, data <br /> EAT-17d (Figure 2C). GeneFinder predictions suggested                        catalytic activity EAT-17 critical its <br /> splicing pattern create stop codon junc-                   function EAT-17 GAP protein. <br /> tion exon 5 exon 6. This transcript en- <br />                                                                                EAT-17 RAB-6.2 interact regulate <br /> code truncated protein lacking C terminus. It <br />                                                                                grinder formation <br /> seems likely b splice forms encoding full- <br /> length proteins functional ones.                                       To identify substrates GAP activity EAT-17, we <br />    Analysis predicted protein sequence revealed                     performed RNA interference 25 29 predicted <br /> conserved domains EAT-17: Rab GTPase activating (Rab                      Rab genes C. elegans genome (Audhya et al. 2007) <br />  <br />  <br />  <br /> 120     S. Straud et al. <br />  Table 1 GAP activity required EAT-17 function                             Table 2 RNAi rab-6.2 phenocopies eat-17 mutants <br />  <br />                        No. rescued        Total no. examined           % rescue       dsRNA injected                                 Phenotype <br /> Wild-type eat-17             16                      76                    21         None                             Wild type <br />   construct                                                                           rab-1                            Embryonic lethal <br /> R116K                          2                     49                    4.1        rab-2                            Slight growth delay <br /> R116A                          1                     64                    1.6        rab-3                            Wild type <br /> eat-17 mutants injected DNA constructs encoding wild-type one <br />                                                                                       rab-5                            Embryonic lethal <br /> of mutant versions (R116K R116A) EAT-17 protein (see Materials      rab-6.1                          Wild type <br /> Methods). F1 transgenic progeny blindly assayed rescue grinder defects.   rab-6.2                          Growth delay abnormal grinder <br /> GFP used identify transgenic progeny afterward.                                rab-7                            Wild type <br />                                                                                       rab-8                            Loopy movement, exaggerated body bends <br />                                                                                       rab-10                           NDa <br /> and looked grinder phenotype similar eat-17                          rab-11.1                         Embryonic lethal <br />                                                                                       rab-11.2                         Arrested L1s <br /> mutants. RNAi performed worms mutant rrf-3, <br />                                                                                       rab-14                           Some growth delay, Dpyb <br /> a hypersensitive background RNAi (Simmer et al. 2002).                            rab-18                           Growth delay, infrequent pumping but <br /> Among 25 Rab genes tested, RNAi rab-6.2                                                                 normal grinder <br /> produced phenotype similar eat-17 (Table 2),                             rab-19                           Wild type <br /> rrf-3 mutants obvious defects grinder formation                            rab-21                           Sick, starved, strong Dpy <br />                                                                                       rab-27                           Wild type <br /> (data shown). When examined rab-6.2 deletion <br />                                                                                       rab-28                           Wild type <br /> mutants, defects size organization                              rab-30                           Wild type <br /> grinder plates identical eat-17 mutants                              rab-33                           Slightly Uncc, wild type <br /> (Figures 1B 3C), con&#65533;?rming RNAi results.                                      rab-35                           Wild type <br />     To examine EAT-17 RAB-6.2 directly bind                            rab-37                           Loopy movement, wild type <br />                                                                                       rab-39                           Wild type <br /> each binding speci&#65533;?c, per- <br />                                                                                       4R79.2                           Loopy movement, Unc, wild type <br /> formed directed yeast two-hybrid assays EAT-17                                   K02E10.1                         ND <br /> and 25 RAB proteins. Among RABs,                                  F11A5.4                          Wild type <br /> only RAB-6.2 interacted EAT-17 (Figure 3E, data                              F11A5.3                          Wild type <br /> shown). To examine guanylate nucleotide binding                                C56E6.2                          Loopy movement, Unc <br /> status RAB-6.2 critical, introduce types muta-                        Among 25 Rabs tested, RNAi rab-6.2 shows identical grinder phenotype to <br />                                                                                       eat-17. Full-length dsRNAs Rab listed injected rrf-3 RNAi hyper- <br /> tions RAB-6.2: (1) glutamine (Q69) leucine (L) to <br />                                                                                       sensitive mutants. Worms grown 15  enhance phenotype. <br /> produce constitutively GTP bound form (2) threonine                             a <br />                                                                                         ND, Not determined. <br />                                                                                       b <br /> (T27) asparagine (N) produce constitutively GDP                                 Dpy, dumpy phenotype, worms shorter wild type. <br />                                                                                       c <br />                                                                                         Unc, uncoordinated movement phenotype, worms abnormally. <br /> bound form (Martinez et al. 1994, 1997). The interaction <br /> between EAT-17 RAB-6.2 completely abolished <br /> when T27N mutation introduced (Figure 3E, 6.2T),                              mammalian RabGAP, interacts Rab6 coiled- <br /> but enhanced Q69L mutation                                coil domain (Cuif et al. 1999). When tested trun- <br /> (Figure 3C, 6.2Q), compared wild-type RAB-6.2 (Figure                           cated EAT-17 constructs, EAT-17 (<span id='am-146' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-147' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-149'  ></span><span id='am-150' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>aa</span> 1&#8211;316), EAT-17 (aa <br /> 3F, 6.2). This suggests reaction reached maximum                             1&#8211;364), EAT-17 (aa 1&#8211;405), did interact with <br /> binding wild-type RAB-6.2 en-                              RAB-6.2, showing GAP domain suf&#65533;?- <br /> hance binding conditions                            cient interaction (data shown). A construct of <br /> performed assay.                                                                  EAT-17 containing &#65533;?rst coiled-coil domain [EAT-17 (aa <br />     RAB-6.1, Rab6 C. elegans genome,                            1&#8211;460)], however, shows signi&#65533;?cant interaction RAB-6.2 <br /> shares 81% amino acid sequence identity RAB-6.2. Be-                             (Figure 3E). Collectively, results GAP <br /> cause high similarity Rab6&#8217;s,                               domain EAT-17 mediates catalytic activity and <br /> tested EAT-17 bind RAB-6.1 well. Al-                              &#65533;?rst coiled-coil domain mediates interaction with <br /> though did detect binding wild-type                                RAB-6.2. Presumably, coiled-coil domains impor- <br /> RAB-6.1 EAT-17 assay, RAB-6.1(Q70L) (equiva-                               tant provide EAT-17 speci&#65533;?city RAB-6.2. <br /> lent Q69L RAB-6.2) showed enhanced binding EAT-                              Altogether, data suggest conserved structural <br /> 17, suggesting EAT-17 binds RAB-6&#8217;s                               mechanism interaction Rab6&#8217;s Rab <br /> conserved guanylate nucleotide binding site. Taken together,                          GAP proteins coiled-coil domain (Cuif et al. <br /> these data strongly support hypothesis RAB-6.2                            1999). <br /> a speci&#65533;?c substrate EAT-17 EAT-17 promotes <br />                                                                                       EAT-17 RAB-6.2 co-expressed terminal bulb <br /> hydrolysis GTP binding active (GTP bound) form <br />                                                                                       muscle, site grinder secretion <br /> of RAB-6. <br />     Our studies identi&#65533;?ed EAT-17 GAP protein                          To determine EAT-17 RAB-6.2 function to- <br /> coiled-coil domains. Coiled-coil domains generally in-                            gether terminal bulb, examined expression <br /> volved protein&#8211;protein interactions, GAPCenA,                                patterns transgenic lines carry promoters each <br />  <br />  <br />  <br />                                                                                                                          EAT-17 RabGAP RAB-6.2             121 <br />                                                                                                     Figure 4 EAT-17 expressed in <br />                                                                                                    pharyngeal muscle, intestine, and <br />                                                                                                    vulva. (A B) Corresponding <br />                                                                                                    GFP DIC images EAT-17:: <br />                                                                                                    GFP. (C D) Corresponding GFP <br />                                                                                                    DIC images RAB-6.2::GFP. <br />                                                                                                    Both EAT-17 RAB-6.2 are <br />                                                                                                    highly expressed pharyn- <br />                                                                                                    geal muscle. RAB-6-2 also <br />                                                                                                    highly expressed neurons. (E <br />                                                                                                    F) Corresponding GFP and <br />                                                                                                    DIC images EAT-17::GFP in <br />                                                                                                    pharynx (red arrows), intes- <br />                                                                                                    tine (white arrows), vulva <br />                                                                                                    (yellow arrows). For Figure 4, A&#8211;D, <br />                                                                                                    bars, 10 mm. For Figure 4, E F, <br />                                                                                                    bars, 30 mm. <br />  <br />  <br />  <br />  <br /> gene fused GFP (see Materials Methods). Both            bodies share similar refractile characteristics that <br /> EAT-17 RAB-6.2 strongly expressed terminal        grinder. These results suggest materials traf- <br /> bulb muscles secrete components grinder          &#65533;?cked RAB-6.2 terminal bulb compo- <br /> (Figure 4, A&#8211;D, Materials Methods). Our construct        nents grinder refractile bodies eat-17 <br /> also shows EAT-17 expression outside pharynx,        components, accumulating muscle be- <br /> as intestine vulva (Figure 4, E F).                   cause traf&#65533;?cking defect. <br />    This tissue expression pattern EAT-17 overlaps largely <br /> with previously reported RAB-6.2 (Zhang et al.          Discussion <br /> 2012), supporting inference EAT-17 speci&#65533;?c <br /> GAP regulate RAB-6.2 activity tissues.               In study mapped characterized eat-17, new <br />                                                                  GTPase-activating protein gene C. elegans. Using dif- <br /> Chitin deposition super&#65533;?cially intact eat-17 <br />                                                                  ferent approaches, RNA interference yeast hybrid, <br /> and rab-6.2 mutants <br />                                                                  showed EAT-17 interacts RAB-6.2, conserved <br /> Chitin provides mechanical support exoskeletons       small G protein regulating membrane traf&#65533;?cking Golgi <br /> invertebrate animals. In yeast, proper chitin deposition      (Del Nery et al. 2006; Girod et al. 1999). The fact both <br /> essential strictly regulated controlling traf&#65533;?cking   approaches identify RAB-6.2 substrate EAT-17 <br /> of synthase, chs3p (Schorr et al. 2001). Because chitin   25 tested Rabs strongly suggests EAT-17 speci&#65533;?c <br /> one components grinder synthase         GAP RAB-6.2. Both eat-17 rab-6.2 mutants show <br /> regulated traf&#65533;?cking, stained grinders eat-17       identical defects grinders, the feeding <br /> and rab-6.2 mutants chitin examine chitin deposi-      organ crushes food, which, therefore, essential <br /> tion misregulated mutants. Despite global        rapid growth normal development. A new grinder <br /> disorganization grinder, overall chitin deposition    replaces old molt, implying the <br /> was indistinguishable eat-17, rab-6.2, wild-type       grinder components needed transported every <br /> (Figure 5, A&#8211;F). However, grown low tempera-           molt build new one. Our observations mutation in <br /> ture, eat-17 mutants occasionally large refractory bod-     rab-6.2, functions cargo traf&#65533;?cking, mutation <br /> ies terminal bulbs secrete components     GAP protein EAT-17 striking phenotypes <br /> grinder (Figure 3D). Interestingly, appearance,      grinder, suggest precise regulation traf&#65533;?cking <br />  <br />  <br />  <br /> 122    S. Straud et al. <br />                                                                                                    Figure 5 Chitin deposition the <br />                                                                                                   grinders eat-17 rab-6.2 <br />                                                                                                   mutants normal. (A B) <br />                                                                                                   Chitin staining grinder from <br />                                                                                                   wild type corresponding DIC <br />                                                                                                   image. (C D) From eat-17 <br />                                                                                                   mutants corresponding DIC <br />                                                                                                   image. (E F) From rab-6.2 <br />                                                                                                   mutants corresponding DIC <br />                                                                                                   image. Arrowheads indicate the <br />                                                                                                   position grinder. Bars, 10 mm. <br />  <br />  <br />  <br />  <br /> of grinder components critical build functional         worms super&#65533;?cially normal, rab-6.2(RNAi) worms <br /> grinder.                                                        defects grinder. Knockout genes, <br />    Although know identities cargos      however, produces gross defects molting growth. The <br /> RAB-6.2 targeted vesicles, eat-17 mutants         fact EAT-17 does interact RAB-6.1 regulate <br /> grown 15  refractory bodies terminal bulbs.      grinder formation similarity eat-17 phenotype to <br /> The bodies appear similar (i.e., highly refractile)      rab-6.2 single mutant rab-6.2(RNAi) consistent <br /> grinder. This suggests bodies grinder com-    proposed EAT-17 speci&#65533;?city RAB-6.2. <br /> ponents accumulating muscle slow traf-           Recently Zhang et al. (2012) identi&#65533;?ed LIN-10 RAB- <br /> &#65533;?cking low temperature. In fact, Chotard et al. (2010)       6.2 effector regulate glutamate receptor recycling C. <br /> showed mutant tbc-2, encodes          elegans neurons. However, did grinder defect <br /> RabGAP C. elegans, accumulates enormous RAB-7-               lin-10 mutants (data shown). In addition, could <br /> positive late endosome intestine containing refractile   detect EAT-17 expression neurons LIN-10 <br /> materials. These results strongly suggest refractile   interacts RAB-6.2. Although does exclude the <br /> bodies common phenotype endosomal traf&#65533;?cking        possibility LIN-10 functions effector RAB-6.2 <br /> mutants EAT-17 regulates endosomal traf&#65533;?cking.         EAT-17 pathway places, LIN-10 does obvi- <br />    Small G proteins substrates GAPs fast       ously function grinder formation. This suggests LIN-10 <br /> turnover. The importance GAPs demonstrated        affects RAB-6.2 function probably speci&#65533;?cally neurons, <br /> examples oncogene Evi5, homologous         supporting notion differential expression speci&#65533;?c <br /> to EAT-17 (Liao et al. 1997; Dabbeekeh et al. 2007). Only       GEFs GAPs different places different times <br /> one GAP Rab6 identi&#65533;?ed far, GAPCenA,      provides spatial temporal speci&#65533;?city G proteins <br /> which interacts Rab6 coiled-coil domain.    (Segev Kahn 2012). <br /> The requirement coiled-coil domain interaction          That mutations Rab GAPs cause an <br /> appears unique Rab6 GAPs. Studies shown      identical phenotype demonstrates critical precise <br /> that <span id='am-51' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-52' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> Rab6 effector, Golgins, interacts Rab6&#8217;s   kinetics Rab provided GAPs are. Furthermore, <br /> through coiled-coil domains direct vesicle capture     implies identify Rab interactors as <br /> and sorting Golgi. It possible Rab6&#8217;s bind   GAPs looking mutants exhibiting phenotypes <br /> to GAPs effectors similar mechanism         similar Rab mutants. Among 20 TBC GAP domain- <br /> mediated coiled-coil domains (Sinka et al. 2008).            containing proteins, characterized <br />    Our discovery EAT-17 speci&#65533;?city RAB-       (Mukhopadhyay et al. 2007; Chotard et al. 2010). Our &#65533;?nd- <br /> 6.2 strongly suggest speci&#65533;?c GAPs      ing provides insight useful discovering <br /> discovered RAB-6.1, Rab6 C. elegans,      GAPs Rabs. In addition, using Eat phe- <br /> sharing 81% amino acid sequence identity RAB-6.2.          notype caused grinder defect eat-17 mutants, we <br /> rab-6.1 rab-6.2 partially redundant; rab-6.1(RNAi)      identify interactors RAB-6.2 effectors <br />  <br />  <br />  <br />                                                                                            EAT-17 RabGAP RAB-6.2         123 <br />  through genetic screens isolate suppressors enhancers              Ho, S. N., H. D. Hunt, R. M. Horton, J. K. Pullen, L. R. Pease, <br /> of eat-17 mutants.                                                          1989 Site-directed mutagenesis overlap extension using the <br />                                                                             polymerase chain reaction. Gene 77: 51&#8211;59. <br />                                                                          Houalla, T., L. Shi, D. J. van Meyel, Y. Rao, 2010 Rab-mediated <br /> Acknowledgments                                                             vesicular transport required neuronal positioning de- <br />                                                                             veloping Drosophila visual system. Mol. Brain 3: 19. <br /> Several strains used studies provided               Jasmin, B. J., B. Goud, G. Camus, J. Cartaud, 1992 The low <br /> Caenorhabditis Genetics Center, funded National                 molecular weight guanosine triphosphate-binding protein <br /> Institutes Health (NIH) Of&#65533;?ce Research Infrastructure                 Rab6p associates distinct post-Golgi vesicles <span id='am-54' about='obo:NCBITaxon_7786' typeof='owl:Thing obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_7711 obo:NCBITaxon_30480 obo:NCBITaxon_117888 obo:NCBITaxon_131567 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_33154 obo:NCBITaxon_7776 obo:NCBITaxon_7777 obo:NCBITaxon_7778 obo:NCBITaxon_117893 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Torpedo" datatype="xsd:string"></span><span id='am-57'  ></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Torpedo</span> <br />                                                                             marmorata electrocytes. Neuroscience 49: 849&#8211;855. <br /> Programs (P40 OD010440). We thank Melanie Cobb and <br />                                                                          Jordens, I., M. Marsman, C. Kuijl, J. Neefjes, 2005 Rab pro- <br /> Rueyling Lin DNA constructs reagents David                      teins, connecting transport vesicle fusion. Traf&#65533;?c 6: 1070&#8211; <br /> Raizen, Jim McKay, Scott Cameron, Pamela Marshall                       1077. <br /> for invaluable discussions. This work supported NIH               Kamath, R. S., J. Ahringer, 2003 Genome-wide RNAi screen- <br /> grant HL46154 (L.A.) 09SDG2150070 Ameri-                       ing Caenorhabditis elegans. Methods 30: 313&#8211;321. <br />                                                                          Liao, X., Y. Du, H. C. Morse, 3rd, N. A. Jenkins, N. G. Copeland, <br /> can Heart Association (Y.-J.Y.). <br />                                                                             1997 Proviral integrations Evi5 locus disrupt novel 90 <br />                                                                             kDa protein homology Tre2 oncogene cell-cycle <br />                                                                             regulatory proteins. Oncogene 14: 1023&#8211;1029. <br /> Literature Cited                                                         Martinez, O., A. Schmidt, J. Salamero, B. Ho&#64258;ack, M. <span id='am-131' about='obo:NCBITaxon_1510180' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_1489923 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489922 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_30828 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-132' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-134' property="rdfs:label" content="Roa" datatype="xsd:string"></span><span id='am-135'  ></span>Roa</span> et al., <br />                                                                             1994 The small GTP-binding protein rab6 functions intra- <br /> Albertson, D. G., J. N. Thomson, 1976 The pharynx Cae-               Golgi transport. J. Cell Biol. 127: 1575&#8211;1588. <br />    norhabditis elegans. Philos. Trans. R. Soc. Lond. B Biol. Sci. 275:   Martinez, O., C. Antony, G. Pehau-Arnaudet, E. G. Berger, J. Salamero <br />    299&#8211;325.                                                                 et al., 1997 GTP-bound forms rab6 induce redistribution of <br /> Audhya, A., A. Desai, K. Oegema, 2007 A role Rab5                Golgi proteins endoplasmic reticulum. Proc. Natl. Acad. <br />    structuring endoplasmic reticulum. J. Cell Biol. 178: 43&#8211;56.         Sci. USA 94: 1828&#8211;1833. <br /> Avery, L., 1993 The genetics feeding Caenorhabditis elegans.       Mukhopadhyay, A., X. <span id='am-66' about='obo:NCBITaxon_9596' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-67' property="oboInOwl:hasExactSynonym" content="chimpanzees" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasAlternativeId" content="NCBITaxon:317028" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="Pan" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasAlternativeId" content="NCBITaxon:445518" datatype="xsd:string"></span><span id='am-72'  ></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Pan</span>, D. G. Lambright, H. A. Tissenbaum, <br />    Genetics 133: 897&#8211;917.                                                   2007 An endocytic pathway target tubby regulation <br /> Avery, L., B. B. Shtonda, 2003 Food transport C. ele-            fat storage. EMBO Rep. 8: 931&#8211;938. <br />    gans pharynx. J. Exp. Biol. 206: 2441&#8211;2457.                           Opdam, F. J., A. Echard, H. J. Croes, J. A. van den Hurk, R. A. van de <br /> Avery, L. You, Y.J. C. elegans feeding (May 21, 2012),                  Vorstenbosch et al., 2000 The small GTPase Rab6B, novel <br />    WormBook, ed. The C. elegans Research Community, WormBook,               Rab6 <span id='am-31' about='obo:NCBITaxon_subfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="subfamily" datatype="xsd:string"></span>subfamily</span> member, cell-type speci&#65533;?cally expressed and <br />    doi/10.1895/wormbook.1.150.1, http://www.wormbook.org.                   localised Golgi apparatus. J. Cell Sci. 113(Pt 15): 2725&#8211; <br /> Blumenthal, T., 1995 Trans-splicing polycistronic transcrip-            2735. <br />    tion Caenorhabditis elegans. Trends Genet. 11: 132&#8211;136.            Pamonsinlapatham, P., R. Hadj-Slimane, Y. Lepelletier, B. Allain, M. <br /> Chotard, L., A. K. Mishra, M. A. Sylvain, S. Tuck, D. G. Lambright          Toccafondi et al., 2009 P120-Ras GTPase activating protein <br />    et al., 2010 TBC-2 regulates RAB-5/RAB-7-mediated endoso-                (RasGAP): multi-interacting protein downstream signaling. <br />    mal traf&#65533;?cking Caenorhabditis elegans. Mol. Biol. Cell 21:            Biochimie 91: 320&#8211;328. <br />    2285&#8211;2296.                                                            Raizen, D., B. M. Song, N. Trojanowski, Y. J. You, 2012 Wormbook, <br /> Conrad, R., K. Lea, T. Blumenthal, 1995 SL1 trans-splicing              Methods measuring pharyngeal behaviors. Available http:// <br />    speci&#65533;?ed AU-rich synthetic RNA inserted 59 end              www.ncbi.nlm.nih.gov/pubmed/23255345. <br />    Caenorhabditis elegans pre-mRNA. RNA 1: 164&#8211;170.                      Ruvkun, G., M. Finney, 2005 Antibody staining formaldehyde- <br /> Cuif, M. H., F. Possmayer, H. Zander, N. Bordes, F. Jollivet et al.,        &#65533;?xed animals. Available http://www.wormatlas.org/ <br />    1999 Characterization GAPCenA, GTPase activating pro-               EMmethods/Antibodystaining.htm Wormatlas, edited <br />    tein Rab6, associates centrosome.             D. Hall. WormBook. <br />    EMBO J. 18: 1772&#8211;1782.                                                Schorr, M., A. Then, S. Tahirovic, N. Hug, P. Mayinger, <br /> Dabbeekeh, J. T., S. L. Faitar, C. P. Dufresne, J. K. Cowell,           2001 The phosphoinositide phosphatase Sac1p controls traf- <br />    2007 The EVI5 TBC domain provides GTPase-activating                  &#65533;?cking yeast Chs3p chitin synthase. Curr. Biol. 11: 1421&#8211; <br />    protein motif RAB11. Oncogene 26: 2804&#8211;2808.                         1426. <br /> Del Nery, E., S. Miserey-Lenkei, T. Falguieres, C. Nizak, L. Johannes    Segev, N., R. A. Kahn, 2012 Kinetic cell-based analyses of <br />    et al., 2006 Rab6A Rab6A&#8217; GTPases play non-overlapping               GTPase regulators. Cell. Logist. 2: 138&#8211;139. <br />    roles membrane traf&#65533;?cking. Traf&#65533;?c 7: 394&#8211;407.                      Simmer, F., M. Tijsterman, S. Parrish, S. P. Koushika, M. L. Nonet <br /> Durkin, M. E., B. Z. Yuan, X. Zhou, D. B. Zimonjic, D. R. Lowy et al.,      et al., 2002 Loss putative RNA-directed RNA polymerase <br />    2007 DLC-1:a Rho GTPase-activating protein tumour sup-               RRF-3 makes C. elegans hypersensitive RNAi. Curr. Biol. 12: <br />    pressor. J. Cell. Mol. Med. 11: 1185&#8211;1207.                               1317&#8211;1319. <br /> Girod, A., B. Storrie, J. C. Simpson, L. Johannes, B. Goud et al.,       Sinka, R., A. K. Gillingham, V. Kondylis, S. Munro, 2008 Golgi <br />    1999 Evidence COP-I-independent transport route               coiled-coil proteins contain multiple binding sites Rab <span id='am-28' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-29' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> <br />    Golgi complex endoplasmic reticulum. Nat. Cell Biol.          G proteins. J. Cell Biol. 183: 607&#8211;615. <br />    1: 423&#8211;430.                                                           Sulston, J. E., J. G. Hodgkin, 1988 Methods, pp. 587&#8211;606 <br /> Grosshans, B. L., D. Ortiz, P. Novick, 2006 Rabs              The Nematode Caenorhabditis elegans, edited W. B. <br />    effectors: achieving speci&#65533;?city membrane traf&#65533;?c. Proc. Natl.          Wood Cold Spring Harbor Laboratory Press, Cold Spring <br />    Acad. Sci. USA 103: 11821&#8211;11827.                                         Harbor, NY. <br /> Hama, H., G. G. Tall, B. F. Horazdovsky, 1999 Vps9p               Tanabe, K., S. Kon, W. Natsume, T. Torii, T. Watanabe et al., <br />    guanine nucleotide exchange factor involved vesicle-                2006 Involvement novel ADP-ribosylation factor GTPase- <br />    mediated vacuolar protein transport. J. Biol. Chem. 274: 15284&#8211;          activating protein, SMAP, membrane traf&#65533;?cking: implications <br />    15291.                                                                   cancer cell biology. Cancer Sci. 97: 801&#8211;806. <br />  <br />  <br />  <br /> 124     S. Straud et al. <br />  Uytterhoeven, V., S. Kuenen, J. Kasprowicz, K. Miskiewicz, P.   You, Y. J., J. Kim, D. M. Raizen, L. Avery, 2008 Insulin, cGMP, <br />    Verstreken, 2011 Loss skywalker reveals synaptic endo-           TGF-<span id='am-22' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-25'  ></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Beta" datatype="xsd:string"></span>beta</span> signals regulate food intake quiescence in <br />    somes sorting stations synaptic vesicle proteins. Cell       C. elegans: model satiety. Cell Metab. 7: 249&#8211;257. <br />    145: 117&#8211;132.                                                    Zerial, M., H. McBride, 2001 Rab proteins membrane or- <br /> von Lieven, A. F., 2003 Functional morphology evolutionary         ganizers. Nat. Rev. Mol. Cell Biol. 2: 107&#8211;117. <br />    origin three-part pharynx <span id='am-125' about='obo:NCBITaxon_333870' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_195229 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41087 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_71193 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-126'  ></span><span id='am-127' property="oboInOwl:hasRelatedSynonym" content="Nematodes Berthold, 1827" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-130' property="rdfs:label" content="Nematodes" datatype="xsd:string"></span>nematodes</span>. Zoology (Jena)    Zhang, D., N. R. Isack, D. R. Glodowski, J. Liu, C. C. Chen et al., <br />    106: 183&#8211;201.                                                       2012 RAB-6.2 retromer regulate glutamate receptor <br /> Wicks, S. R., R. T. Yeh, W. R. Gish, R. H. Waterston, R. H.        recycling retrograde pathway. J. Cell Biol. 196: <br />    Plasterk, 2001 Rapid gene mapping Caenorhabditis elegans         85&#8211;101. <br />    using high density polymorphism map. Nat. Genet. 28: 160&#8211; <br />    164.                                                                                         Communicating editor: D. I. Greenstein <br />  <br />  <br />  <br />  <br />                                                                                                 EAT-17 RabGAP RAB-6.2     125 <br />              GENETICS <br />                                                               Supporting Information <br />            http://www.genetics.org/lookup/suppl/doi:10.1534/genetics.113.152538/-/DC1 <br />  <br />  <br />  <br />  <br /> The Jaw Worm: GTPase-activating Protein <br />         EAT-17 Regulates Grinder Formation in <br />                        Caenorhabditis elegans <br />                  Sarah Straud, Inhwan Lee, Bomi Song, Leon Avery, Young-Jai You <br />  <br />  <br />  <br />  <br />                                                Copyright &#169; 2013 Genetics Society America <br />                                                                DOI: 10.1534/genetics.113.152538 <br />    Dm CG11727-PA    -----------------------------------------------MTLTTTTTASSAE   13 <br />   Dm CG11727-PB    -----------------------------------------------MTLTTTTTASSAE   13 <br />   EAT-17a          -----------------------------------------------MAATAALRRSSDY   13 <br />   EAT-17b          -----------------------------------------------MAATAALRRSSDY   13 <br />   Hs Evi5/NB4S     MVTNKMTAAFRNPSGKQVATDKVAEKLSSTLSWVKNTVSHTVSQMASQVASPSTSLHTTS   60 <br />   Mm Evi5          MVTTKMTAAFRNPNRRQVATDKVAEKLSSTLSWVKNTVSHTVSQMASQVASPSASLHTTS   60 <br />                                                                    . :.:    : <br />  <br />    Dm CG11727-PA   SQAKMDVKGGALPGEENLPTSEMDLLAKLEAANKLIESDAKSLNSLHSTHSRKNSDTSQI   73 <br />    Dm CG11727-PB   SQAKMDVKGGALPGEENLPTSEMDLLAKLEAANKLIESDAKSLNSLHSTHSRKNSDTSQI   73 <br />    EAT-17a         GSADTECGEPCETG-APVSLNEVDLLAKMEQLNKSNEEDSRSVASKKTGSS--ESRKGAR   70 <br />    EAT-17b         GSADTECGEPCETG-APVSLNEVDLLAKMEQLNKSNEEDSRSVASKKTGSS--ESRKGAR   70 <br />    Hs Evi5/NB4S    SSTTLSTPALSPSSPSQLSPDDLELLAKLEEQNRLLETDSKSLRSVNGSRRNSGSSLVSS   120 <br />    Mm Evi5         SSTTLSTPTQSPSSPSKLSPDDLELLAKLEEQNRLIETDSKSLRSVNGSRRNSGSSLVSS   120 <br />                    ..: .     . ..   :. .:::****:* *: * *::*: * :         * <br />  <br />    Dm CG11727-PA   SLTSSGNSVAEEDIWTTWATILNDWEGALKRKNPCVSELVRRGIPHHFRAIVWQQLSGAS   133 <br />    Dm CG11727-PB   SLTSSGNSVAEEDIWTTWATILNDWEGALKRKNPCVSELVRRGIPHHFRAIVWQQLSGAS   133 <br />    EAT-17a         EHSPEEDE---EDLWSVWGELILNWEIEVKKRPNYIKDLVKRGIPQHFRMIAWQNLSNAS   127 <br />    EAT-17b         EHSPEEDEYFQEDLWSVWGELILNWEIEVKKRPNYIKDLVKRGIPQHFRMIAWQNLSNAS   130 <br />    Hs Evi5/NB4S    SSASSNLSHLEEDSWILWGRIVNEWEDVRKKKEKQVKELVHKGIPHHFRAIVWQLLCSAQ   180 <br />    Mm Evi5         SSASSNLSHLEEDSWILWGRIVNEWDDVRKKKEKQVKELVRKGIPHHFRAIVWQLLCNAQ   180 <br />                    . :.. .    ** * *. :: :*:    *::   :.:**::***:*** *.** *..*. <br />  <br />    Dm CG11727-PA   DGDKK-QYAEYIKATSACEKVIRRDIARTYPEVEFFKEKDGPGQEALFNVIKAYSLHDRE   192 <br />    Dm CG11727-PB   DGDKK-QYAEYIKATSACEKVIRRDIARTYPEVEFFKEKDGPGQEALFNVIKAYSLHDRE   192 <br />    EAT-17a         VSSVHDLYSDYMRQSSVYEKVIQRDIPRTYPELDFFKDGER-GQSLLFNVIKAYSVHDKE   186 <br />    EAT-17b         VSSVHDLYSDYMRQSSVYEKVIQRDIPRTYPELDFFKDGER-GQSLLFNVIKAYSVHDKE   189 <br />    Hs Evi5/NB4S    SMPIKDQYSELLKMTSPCEKLIRRDIARTYPEHNFFKEKDSLGQEVLFNVMKAYSLVDRE   240 <br />    Mm Evi5         SMTIKDQYSELLKMTSPCEKLIRRDIARTYPEHNFFKEKDSLGQEVLFNVMKAYSLVDRE   240 <br />                        : *:: :: :* **:*:***.***** :***: : **. ****:****: *:* <br />  <br />    Dm CG11727-PA   VGYCQGSGFIVGLLLMQMPEEEAFAVLVQIMQQHRMRHMFKPSMSELGLCMYQLENLVQE   252 <br />    Dm CG11727-PB   VGYCQGSGFIVGLLLMQMPEEEAFAVLVQIMQQHRMRHMFKPSMSELGLCMYQLENLVQE   252 <br />    EAT-17a         VGYCQGSAFIVGLLLLQMPEEEAFAVLVSLMENYRLRELYKPTMTDLGLCMFQLECLVQD   246 <br />    EAT-17b         VGYCQGSAFIVGLLLLQMPEEEAFAVLVSLMENYRLRELYKPTMTDLGLCMFQLECLVQD   249 <br />    Hs Evi5/NB4S    VGYCQGSAFIVGLLLMQMPEEEAFCVFVKLMQDYRLRELFKPSMAELGLCMYQFECMIQE   300 <br />    Mm Evi5         LVTVRAVLSSLDCCCMQMPEEEAFCVFVKLMQDYRLRELFKPSMAELGLCMYQFECMIQE   300 <br />                    :   :.    :.   :********.*:*.:*:::*:*.::**:*::*****:*:* ::*: <br />  <br />    Dm CG11727-PA   QIPDMHIHFQQQGFQTTMYASSWFLTLYTTTLNVNLSCRIMDVFLSEGMEFIFKVALALL   312 <br />    Dm CG11727-PB   QIPDMHIHFQQQGFQTTMYASSWFLTLYTTTLNVNLSCRIMDVFLSEGMEFIFKVALALL   312 <br />    EAT-17a         QMPDLYTHFNNMGFDTSMYASSWFLTLFTTTMPLDIANRIMDCFLVEGMDFIFCISIAIL   306 <br />    EAT-17b         QMPDLYTHFNNMGFDTSMYASSWFLTLFTTTMPLDIANRIMDCFLVEGMDFIFCISIAIL   309 <br />    Hs Evi5/NB4S    HLPELFVHFQSQSFHTSMYASSWFLTIFLTTFPLPVATRIFDIFMSEGLEIVFRVGLALL   360 <br />    Mm Evi5         YLPELFVHFQSQSFHTSMYASSWFLTIFLTTFPLPIATRIFDIFMSEGLEIVFRVGLALL   360 <br />                     :*::. **:. .*.*:*********:: **: : :: **:* *: **::::* :.:*:* <br />  <br />    Dm CG11727-PA   LTGKDTLLCLDMEAMLKFFQKELPGRVEADVEGFFNLAYSIKLNTKRMKKMEKEYQDLKK   372 <br />    Dm CG11727-PB   LTGKDTLLCLDMEAMLKFFQKELPGRVEADVEGFFNLAYSIKLNTKRMKKMEKEYQDLKK   372 <br />    EAT-17a         QQARIELLRLDMEGMLKYFQREVRERYEFDADLLFTVANQVQLNAKRMKRLEKDYLTKRT   366 <br />    EAT-17b         QQARIELLRLDMEGMLKYFQREVRERYEFDADLLFTVANQVQLNAKRMKRLEKDYLTKRT   369 <br />    Hs Evi5/NB4S    QMNQAELMQLDMEGMLQHFQKVIPHQFDGVPDKLIQAAYQVKYNSKKMKKLEKEYTTIKT   420 <br />    Mm Evi5         QMNQAELMQLDMEGMLQHFQKVIPHQFDGGPEKLIQSAYQVKYNSKKMKKLEKEYTTIKT   420 <br />                       : *: ****.**:.**: : : :     : :: * .:: *:*:**::**:*    :. <br />  <br />    Dm CG11727-PA   KEQEEMAELRRLRRENCLLKQRNELLEAESAELADRLVRGQVSRAEEEETSYAIQTELMQ   432 <br />    Dm CG11727-PB   KEQEEMAELRRLRRENCLLKQRNELLEAESAELADRLVRGQVSRAEEEETSYAIQTELMQ   432 <br />    EAT-17a         KEQEEAVELRRLRTENRLLRQRIDYLEAESSALADRLVKGQVNLAQEAENYINIAHELNK   426 <br />    EAT-17b         KEQEEAVELRRLRTENRLLRQRIDYLEAESSALADRLVKGQVNLAQEAENYINIAHELNK   429 <br />    Hs Evi5/NB4S    KEMEEQVEIKRLRTENRLLKQRIETLEKH-----------KCSSNYNEDFVLQLEKELVQ   469 <br />    Mm Evi5         KEMEEQGEIKRLRTENRLLKQRIETLEKH-----------KCSSTYNEDFVLQLEKELVQ   469 <br />                    ** ** *::*** ** **:** : ** .            : .   : :    : ** : <br />  <br />    Dm CG11727-PA   LRRSYLEVSHQLENANEEVRGLSLRLQENNNSRQSSIDELCMKEEALKQRDEMVSCLLEE   492 <br />    Dm CG11727-PB   LRRSYLEVSHQLENANEEVRGLSLRLQENNNSRQSSIDELCMKEEALKQRDEMVSCLLEE   492 <br />    EAT-17a         LRDMNSDVHRKLEGAYETIRELSSARRDNIMDTGTQVDDTSMIEHIHS--------LQQE   478 <br />    EAT-17b         LRDMNSDVHRKLEGAYETIRELSSARRDNIMDTGTQVDDTSMIEHIHS--------LQQE   481 <br />    Hs NB4S/Evi5    ARLSEAESQCALKEMQDKV--LDIEKRNNSLPDENNIAR-----------------LQEE   510 <br />    Mm Evi5         ARLSEAESQCALKEMQDKV--LDIEKKNNSFPDENNIAR-----------------LQEE   510 <br />                     *    :    *:   : : *.    ::*     ..:                   * :* <br />  <br />  <br />  <br />  <br /> 2 SI                                        S. Straud et al. <br />   Dm CG11727-PA            LVKVRQGLAESEDQIRNLKAKVEELEEDKKTLRETTP--------DNSVAHLQDELIASK                      544 <br />  Dm CG11727-PB            LVKVRQGLAESEDQIRNLKAKVEELEEDKKTLRETTP--------DNSVAHLQDELIASK                      544 <br />  EAT-17a                  LIEAHTRQADSENTLRDAKLRVSELEMANKRLLENEP--------SEDVAGLQEELISVK                      530 <br />  EAT-17b                  LIEAHTRQADSENTLRDAKLRVSELEMANKRLLENEP--------SEDVAGLQEELISVK                      533 <br />  Hs NB4S/Evi5             LIAVKLREAEAIMGLKELRQQVKDLEEHWQRHLARTTGRWKDPPKKNAMNELQDELMTIR                      570 <br />  Mm Evi5                  LIAVKLREAEAIMGLKELRQQVRTLEEHWQRHLARTSGRWKDPPKKNAVNELQDELMSIR                      570 <br />                           *: .:   *::   ::: : :* **    :      .        .: : **:**:: : <br />  <br />  Dm CG11727-PA            LREAEASLSLKDLKQRVQELSSQWQRQLAE----------NQRSESERTTNAVDSTP---                      591 <br />  Dm CG11727-PB            LREAEASLSLKDLKQRVQELSSQWQRQLAE----------NQRSESERTTNAVDSTP---                      591 <br />  EAT-17a                  MREAESSLALKEMRQRLAELEQHWAKYVHVRAFDPSSASIEKESTSEAHSTQQQPSPPLT                      590 <br />  EAT-17b                  MREAESSLALKEMRQRLAELEQHWAKYVHVRAFDPSSASIEKESTSEAHSTQQQPSPPLT                      593 <br />  Hs NB4S/Evi5             LREAETQAEIREIKQRMMEMETQNQINSNH----------LRRAEQEVISLQEKVQYLSA                      620 <br />  Mm Evi5                  LREAETQAEIREMKQRMMEMETQNQINSNQ----------LRRAEQEVNSLQEKVCSLSV                      620 <br />                           :****:. :::::**: *:. :                   :.: .* :    . <br />  <br />  Dm CG11727-PA            ------KKLLTNFFD-SSKSSE---HTQKLEEELMTTRIREMETLTELKELRLKVMELET                      641 <br />  Dm CG11727-PB            ------KKLLTNFFD-SSKSSE---HTQKLEEELMTTRIREMETLTELKELRLKVMELET                      641 <br />  EAT-17a                  SARARLAKITASLIGGSTEETDNCISVRELEDQLMGVRIKEADTLAELKEMRQKVMELET                      650 <br />  EAT-17b                  SARARLAKITASLIGGSTEETDNCISVRELEDQLMGVRIKEADTLAELKEMRQKVMELET                      653 <br />  Hs NB4S/Evi5             QN----KGLLTQLSEAKRKQAE---IECKNKEEVMAVRLREADSIAAVAELRQHIAELEI                      673 <br />  Mm Evi5                  KN----KGLLAQLSEAKRRQAE---IECKNKEEVMAVRLREADSIAAVAELQQHIAELKI                      673 <br />                                   : :.:   . ..::      : ::::* .*::* :::: : *:: :: **: <br />  <br />  Dm CG11727-PA            QVQVSTNQLRRQDEEHKKLKEELEMAVTREKDMSNKAREQQHRYSDLESRMKDELMNVKI                      701 <br />  Dm CG11727-PB            QVQVSTNQLRRQDEEHKKLKEELEMAVTREKDMSNKAREQQHRYSDLESRMKDELMNVKI                      701 <br />  EAT-17a                  QNHVCTNQLKRQDEEMKRVREDSEVLVKKRKELEDQLKDEKEKLDNKESEFNEGRINDRL                      710 <br />  EAT-17b                  QNHVCTNQLKRQDEEMKRVREDSEVLVKKRKELEDQLKDEKEKLDNKESEFNEGRINDRL                      713 <br />  Hs NB4S/Evi5             QKEEGKLQGQLN-------KSDSNQYIG---ELKDQIAELNHELRCLKGQKG---FSGQP                      720 <br />  Mm Evi5                  QKEEGKLQGQLN-------RSDSNQYIR---ELKDQIAELTHELRCLKGQRD---FSSRP                      720 <br />                           * . . * : :        :.: : :     ::.:: : ..      :..     :. : <br />  <br />  Dm CG11727-PA            KFTEQSQTVAELKQEISRLETKNSEMLAEGELR----ANLDDSDKVR-------------                      744 <br />  Dm CG11727-PB            KFTEQSQTVAELKQEISRLETKNSEMLAEGELR----ANLDDSDKVR-------------                      744 <br />  EAT-17a                  KYSEAMQTIQDLQSSISQLELKKAEKWTQNQLRGSSVCDLDEESNSHGSICSNVDHLSLA                      770 <br />  EAT-17b                  KYSEAMQTIQDLQSSISQLELKKAEKWTQNQLRGSSVCDLDEESNSHGSICSNVDHLSLA                      773 <br />  Hs NB4S/Evi5             PFDG-IHIVNHLIGDDESFHSSDEDFIDNSLQETGVGFPLHGKSGSMS------------                      767 <br />  Mm Evi5                  PFDG-IHIVSHLIGDDELFHSSDEDFIDSSLQESAIGFPLHRKSGPMS------------                      767 <br />                            :    : : .* . . :. .. :    .. .       *. .. <br />  <br />  Dm CG11727-PA            --DLQDRLADMKAELTALKSRGKFPGAKLRSSSIQSIESTEIDFNDLNMVRRGSTELST-                      801 <br />  Dm CG11727-PB            --DLQDRLADMKAEYPTP---------------ITSPDTEPWKWIS--------------                      773 <br />  EAT-17a                  SDEMNALLADMTVRIPTLDDLAE------EGSATETDELRPKELNDGNDTTDSGVQLSDS                      824 <br />  EAT-17b                  SDEMNALLADMTVRIPTLDDLAE------EGSATETDELRPKELNDGNDTTDSGVQLSDS                      827 <br />  Hs NB4S/Evi5             ---LDPAVADGSESETEDSVLET----------RESNQVVQKERPPRRRESYSTTV----                      810 <br />  Mm Evi5                  ---LNPALADGSESEAEDGMLGP----------QESDPEAPQKQPPQR-ESYSTTV----                      809 <br />                              :: :** .    .                   :      . <br />  <br />  Dm CG11727-PA            - Dm <br />  CG1172               - <br />  EAT-17a                  H 825 <br />  EAT-17b                  H 828 <br />  Hs NB4S/Evi5             - <br />  <br />  <br /> Figure S1 Sequence alignments EAT-17 Evi5 homologs <br /> EAT-17 shows greatest sequence similarity EVI5/NB4S proteins implicated human neuroblastoma and <br /> mouse tumorigenesis. EAT-17 contains types conserved domains. A putative Rab GAP domain (aa107-316) (shown in <br /> yellow) located N-terminal half protein. Three predicted coiled-coil domains located C terminal half of <br /> the protein (aa 364-405, aa 475-552, aa 628-736 coiled-coil (CC) domains 1, 2, 3, respectively) (shown blue). <br /> Shown red indicated arrow invariant arginine residue crucial GAP function. All sequences obtained <br /> through GenBank. Accession numbers follows: NM-005665.3 (human Evi5/NB4S), NM-007964.1 (mouse Evi5), NM- <br /> 167285.1 (Drosophila CG11727-PA), NM-132488.2 (Drosophila CG11727-PB). Dm=<span id='am-59' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-65'  ></span>Drosophila melanogaster</span>, Hs= <span id='am-79' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-80'  ></span><span id='am-81' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="Homo" datatype="xsd:string"></span>Homo</span> <br /> sapiens, Mm= <span id='am-1' about='obo:NCBITaxon_10090' typeof='owl:Thing obo:NCBITaxon_314147 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_337687 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_862507 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_39107 obo:NCBITaxon_10088 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_10066 obo:NCBITaxon_6072 obo:NCBITaxon_33553 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9989 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-2' property="oboInOwl:hasAlternativeId" content="NCBITaxon:85055" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Mus musculus" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="mice C57BL/6xCBA/CaJ hybrid" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="house mouse" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="mouse" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Mus muscaris" datatype="xsd:string"></span>Mus musculus</span>. EAT-17a EAT-17b spliced form EAT-17 C. elegans. <br />  <br />  <br />  <br />  <br />                                                              S. Straud et al.                                                         3 SI <br />  Table S1 Primers used SNP mapping <br />  <br /> SNP                 Primer Number       Forward Primer (5&#8217;- 3&#8217;)         Reverse Primer (5&#8217;- 3&#8217;) <br />                     (For/Rev) <br /> pkP6169             742/743             cctcaggatttaccagtgacac          ttagtcttgcgcccctagag <br /> pkP6093             744/745             tagatatcgtggaacccc              cctgggaatccgttttctcc <br /> pkP6096             746/747             gattgaacatagctcacagc            tttcgatcgttttggacgcc <br /> pkP6171             748/749             cgatgcggtttcctagcttac           attgcccatttcaagccc <br /> pkP6170             785/786             cgctgtcacaatctctaaaatg          aaaccctccccactttgttgtc <br /> snp-T21F2           889/890             ttgacgcgctcttctactga            ttggggtggttgtttgatct <br /> snp-C06G1           891/892             gagcataccgtttggcagat            atggctcagcactctcgaat <br /> snp-F59C12          893/894             gaaaaagtggccaaaggtga            aaaaactgacaagcggcaat <br /> snp-T25D1           895/896             catttcagaaaccatgaacctg          ttgaagaactactcctctggca <br /> snp-F38E9           897/898             ctcacgctgacctctttcct            tttgcatctttggagaattgg <br /> snp-T24D11          899/900             gcgggaatgtgcactaaaat            gcgtgtatatttggcagcaa <br /> snp-W09B12          901/902             cctcaggatttaccagtgacac          ttagtcttgcgcccctagag <br /> snp-C11G6           903/904             tgctggccagactacaaaaa            ctcgaaaagtcctatcagcca <br /> snp-F23A7           905/906             tttaaagttcccagctgtgct           gaaagggattctgccacaaa <br />  <br />  <br />  <br />  <br /> 4 SI                                                 S. Straud et al. <br />  Table S2 Deletion mapping primers <br /> Deletion strain       Primer        Primers outside deletion               Primers inside deletion <br /> DA1814                forward       5&#8217;- aagcatctttgagcgcattt &#8211;3&#8217;           5&#8217;- catagcgagtgtttggagca &#8211;3&#8217; <br /> ser-1(ok345)          Reverse       5&#8217;- aatttcaggggtgtggacat &#8211;3&#8217;           5&#8217;- aatcatttttgaaaccgaccc &#8211;3&#8217; <br /> RB758                 forward       791: 5&#8217;- tcacagctcaccaaagatcg &#8211;3&#8217;      793: 5&#8217;- ttgccaacaggagtaaaggg &#8211;3&#8217; <br /> hda-4(ok518)          Reverse       792: 5&#8217;- gttgttgctgctgcatttgt &#8211;3&#8217;      794: 5&#8217;- ccaatgagtgcctggaattt &#8211;3&#8217; <br /> RB787                 forward       787: 5&#8217;- atcgaatacatccgtccagc &#8211;3&#8217;      789: 5&#8217;- ggcaacataccatttccacc &#8211;3&#8217; <br /> T27A8.2(ok570)        Reverse       788: 5&#8217;- tcttgacccagaaacgaacc &#8211;3&#8217;      790: 5&#8217;- tgacccagaaacgtacccat &#8211;3&#8217; <br />  <br />  <br />  <br />  <br />                                                         S. Straud et al.                                       5 SI <br />  Table S3 Primers used gene structure analysis <br /> Gene                     Forward Primer (5&#8217;- 3&#8217;)          Reverse Primer (5&#8217;- 3&#8217;)           Amplified Region <br /> T24D11.1                 atgaatgtcctcccactctttttc         tgcaatcattcgaaagtgctg             5&#8217; R116/119 codon <br /> T24D11.1                 atgaatgtcctcccactctttttc         acggggaatgtcgcgttgaat             5&#8217; the <br />                                                                                             D152/155 codon <br /> T24D11.1                 cagcactttcgaatgattgca            ctagtggctatccgacagtt              3&#8217; R116/119 codon <br /> T24D11.1                 attcaacgcgacattccccgt            ctagtggctatccgacagtt              3&#8217; D152/155 codon <br /> T24D11.1                 atgaatgtcctcccactctttttc         gagacggagaagctcgatg               3&#8217; Exons 1-5 <br /> T24D11.1                 aatgtggaatgagagaggcg             gagacggagaagctcgatg               Exons 1-5 <br /> T24D11.1                 gatccgcagacaccgagtg              gagacggagaagctcgatg               Exons 2-5 <br /> T24D11.1                 gaagtgttgcctccaagaag             gagacggagaagctcgatg               Exons 3-5 <br /> T24D11.1                 cgtgtggggagagcttattc             gagacggagaagctcgatg               Exons 4-5 <br /> T24D11.1                 aggcattcgcgttcaaactc             gagacggagaagctcgatg               Exons 1*-5 <br /> T24D11.1/F01G12.6        gaagtcaagaagcgtcccaa             tcgttgaactcgctctcctt              Exon 3 T24D11.1 to <br />                                                                                             exon 7 F01G12.6 <br /> T24D11.1/F01G12.6        ccgtctcgatatggaaggaa             tcgttgaactcgctctcctt              Exon 5 T24D11.1 to <br />                                                                                             exon 7 F01G12.6 <br /> T24D11.1/F01G12.6        gacgtcagcctcatggaaaact           gccaatgtgtcagcttcctt              Exon 5 T24D11.1 to <br />                                                                                             exon 5 F01G12.6 <br /> T24D11.1/F01G12.6        aggcattcgcgttcaaactc             ctagtggctatccgacagtt              Exon 1** T24D11.1 last <br /> (eat-17 cDNA)                                                        <br /> </body></html>