<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The <span id='am-46' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-48'  ></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span>Caenorhabditis elegans</span> Gene mfap-1 Encodes a <br /> Nuclear Protein That Affects Alternative Splicing <br /> Long Ma1*, Xiaoyang Gao1, Jintao Luo1, Liange Huang1, Yanling Teng1, H. Robert Horvitz2* <br /> 1 State Key Laboratory Medical Genetics, School Biological Sciences Technology, Central South University, Changsha, China, 2 Howard Hughes Medical Institute, <br /> Department Biology, Massachusetts Institute Technology, Cambridge, Massachusetts, United States America <br />  <br />  <br />  <br />      Abstract <br />      RNA splicing major regulatory mechanism controlling eukaryotic gene expression. By generating various splice <br />      isoforms single pre&#8211;mRNA, alternative splicing plays key role promoting evolving complexity metazoans. <br />      Numerous splicing factors identified. However, vivo functions splicing factors remain be <br />      understood. In vivo studies essential understanding molecular mechanisms RNA splicing biology of <br />      numerous RNA splicing-related diseases. We previously isolated <span id='am-53' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-54'  ></span><span id='am-55' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Caenorhabditis</span> elegans mutant defective essential <br />      gene genetic screen suppressors rubberband Unc phenotype unc-93(e1500) animals. <span id='am-58' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-59' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62'  ></span>This</span> mutant <br />      contains missense mutations adjacent codons C. elegans microfibrillar-associated protein 1 gene mfap-1. mfap- <br />      1(n4564 n5214) suppresses Unc phenotypes different rubberband Unc mutants pattern similar of <br />      mutations splicing factor genes uaf-1 (the C. elegans U2AF large subunit gene) sfa-1 (the C. elegans SF1/BBP <br />      gene). We used endogenous gene tos-1 reporter splicing detected increased intron 1 retention exon 3 <br />      skipping tos-1 transcripts mfap-1(n4564 n5214) animals. Using yeast two-<span id='am-97' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-98' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-100'  ></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>hybrid</span> screen, isolated splicing factors as <br />      potential MFAP-1 interactors. Our studies indicate C. elegans mfap-1 encodes splicing factor affect alternative <br />      splicing. <br />  <br />   Citation: Ma L, Gao X, Luo J, Huang L, Teng Y, et al. (2012) The Caenorhabditis elegans Gene mfap-1 Encodes Nuclear Protein That Affects Alternative <br />   Splicing. PLoS Genet 8(7): e1002827. doi:10.1371/journal.pgen.1002827 <br />   Editor: Andrew D. Chisholm, University <span id='am-6' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11'  ></span>California</span> San Diego, United States America <br />   Received March 16, 2012; Accepted May 25, 2012; Published July 19, 2012 <br />   Copyright: &#223; 2012 MA et al. This open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This study supported NSFC grant (30971639, www.nsfc.gov.cn) National Basic Research Program China grant LM (2011CB510005, <br />   www.973.gov.cn) NIH grant GM24663 (www.nih.gov) HRH. Some C. elegans strains provided Caenorhabditis Genetics Center, was <br />   funded NIH National Center Research Resources. The funders role study design, <span id='am-92' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-93'  ></span><span id='am-94' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> collection analysis, decision publish, or <br />   preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: malong@sklmg.edu.cn (LM); horvitz@mit.edu (HRH) <br />  <br />  <br />  <br />  <br /> Introduction                                                                             Recent studies indicate conclusions concerning splicing <br />                                                                                       factors derived vitro analyses complemented in <br />     RNA splicing removes non-coding introns joins adjacent                        vivo analyses. For example, vivo studies suggest splicing <br /> coding exons pre-mRNAs generate functional coding                             factor SF1/BBP, thought ubiquitously required for <br /> mRNAs. Alternative splicing generate numerous splice                              splicing, required splicing subset genes <br /> isoforms pre-mRNA [1]. The complex proteome                             [15&#8211;17]. Some <span id='am-19' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> core splicing factors, once <br /> encoded mRNA splice isoforms believed major                             thought essential splicing introns, to <br /> driving force evolving complexity metazoans [1,2].                         affect splicing subgroup introns effects of <br /> Numerous proteins non-coding RNAs regulate RNA splicing                           loss-of-function mutations factors examined [18]. <br /> [3]. The U1 snRNP complex SF1/U2AF65/U2AF35                                   Also, functions splicing factors extend beyond <br /> protein complex recognize 59 39 splice sites intron,                    regulating RNA splicing. For example, U2AF large subunit <br /> respectively [4&#8211;11], U2 U4/U5/U6 snRNP                                    required efficient export intronless mRNAs in <br /> complexes assemble step-wise manner undergo compo-                           Drosophila addition essential role regulating 39 splice site <br /> sitional conformational rearrangements drive steps                     recognition [19]. Thus, vivo functional analyses essential for <br /> of trans-esterification reaction RNA splicing [7,12].                          accurate understanding biological functions splicing <br /> Mutations trans-splicing factors <span id='am-77' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81'  ></span>cis</span>-regulatory splicing                        factors. <br /> elements cause numerous diseases [13,14].                                                In Caenorhabditis elegans, genes unc-93, sup-9 sup-10 <br />     Over 200 protein factors shown regulate splicing                     encode components presumptive two-pore domain K+ <br /> or associate splicing machinery splicing factors                    channel complex affects muscle activity [20&#8211;23]. Animals <br /> [3]. Splicing factors identified using biochemical                   carrying rare gain-of-function (gf) mutations three <br /> approaches, vivo functions splicing factors                        genes sluggish, defective egg laying exhibit a <br /> remain largely unknown. In vivo analysis factors remains                   rubberband phenotype: touched head, animal <br /> challenge, essential viability analyses                 contracts relaxes entire body moving <br /> their vivo functions limited lethality caused                     backwards. Complete loss-of-function (lf) mutations unc-93, sup- <br /> mutations factors.                                                           9 sup-10 cause obvious abnormalities [21,22]. The SUP- <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                      1                                July 2012 | Volume 8 | Issue 7 | e1002827 <br />                                                                                                                            MFAP-1 Affects Alternative Splicing <br />  <br />  <br />  <br />   Author Summary                                                                 suppressed Unc phenotype unc-93(e1500) animals 15uC <br />                                                                                  (L. Ma H. R. Horvitz, unpublished observations) a <br />   RNA splicing removes intervening intronic sequences                       stronger suppressor 20uC (Table 1). n4564 n5214/+; unc- <br />   pre&#8211;mRNA transcripts joins adjacent exonic sequences                       93(e1500) animals Unc unc-93(e1500) animals (L. Ma <br />   generate functional messenger RNAs. The vivo                             H. R. Horvitz, unpublished observations), indicating that <br />   functions numerous factors regulate splicing                           n4564 n5214 recessive suppressor Unc phenotype unc- <br />   remain understood. From genetic screen                             93(e1500) animals suggesting n4564 n5214 causes a <br />   suppressors rubberband Unc phenotype caused                          reduction loss mfap-1 function. We tested n4564 <br />   Caenorhabditis elegans unc-93(e1500) mutation,                          n5214 suppressed Unc phenotype caused the <br />   isolated mutation affects highly conserved <br />                                                                                  rubberband mutants unc-93(n200), sup-9(n1550) sup- <br />   essential gene, mfap-1. MFAP-1 nuclear protein that <br />   broadly expressed. MFAP-1 affect alternative                        10(n983). n200 weakly semi-dominant unc-93 allele causes <br />   splicing tos-1, endogenous reporter gene splicing,                   weak rubberband Unc phenotype [22]. The sup-9(n1550) and <br />   required altered splicing cryptic 39 splice                sup-10(n983) gain-of-function mutations cause strong moder- <br />   site tos-1. mfap-1 enhances effects gene uaf-1                   ate rubberband Unc phenotypes, respectively [20,21,23,31]. <br />   (splicing factor U2AF large subunit) suppressing                        n4564 n5214 strongly suppressed sup-10(n983) did suppress <br />   rubberband Unc phenotype unc-93(e1500) animals. Our                         unc-93(n200) sup-9(n1550) (Table 1). Thus, n4564 n5214 <br />   studies provide vivo evidence MFAP-1 functions                      suppressed rubberband Unc mutants uaf- <br />   splicing factor.                                                             1(n4588) sfa-1(n4562) mutations, suppress the <br />                                                                                  rubberband Unc phenotypes unc-93(e1500) sup-10(n983) <br />                                                                                  animals rubberband Unc phenotypes unc-93(n200) <br /> 9 protein similar mammalian Two-pore Acid Sensitive K+                 sup-9(n1550) animals [29]. <br /> channels TASK-1 TASK-3 [20]. sup-10 encodes novel <br /> single-transmembrane domain protein identified mam- <br />                                                                                  n4564 n5214 contains missense mutations the <br /> malian homologs [20], unc-93 encodes multiple transmem- <br /> brane-domain protein defines novel <span id='am-43' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> proteins                     gene mfap-1, encodes highly conserved nuclear <br /> conserved C. elegans mammals [20,23]. A mammalian                        protein <br /> UNC-93 homolog, UNC-93b, interacts Toll-like receptors                         We cloned gene affected n4564 n5214 genetic mapping <br /> and regulates innate immune responses [24&#8211;28].                                   transgene rescue experiments (Figure 1A, 1B; Materials and <br />    We previously screened new suppressors &#8216;&#8216;rubber-                   Methods). The n4564 n5214 strain contains missense mutations <br /> band&#8217;&#8217; Unc phenotype unc-93(e1500) animals isolated                       gene F43G9.10, changing adjacent conserved amino acids <br /> mutations affecting splicing factors U2AF large subunit (UAF-                Asp 426 Thr 427 Val (n5214) <span id='am-87' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-88' property="rdfs:label" content="Ala" datatype="xsd:string"></span><span id='am-89'  ></span><span id='am-90' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Ala</span> (n4564) the <br /> 1) SF1/BBP (SFA-1) [29]. Our analysis suggested                         predicted protein, respectively (Figure 1C). F43G9.10 encodes C. <br /> mutations uaf-1 sfa-1 result suppression                    elegans ortholog highly conserved mammalian protein has <br /> rubberband Unc phenotype unc-93(e1500) animals altering                called &#8216;&#8216;microfibrillar-associated protein 1&#8217;&#8217; (41% identity <br /> splicing pre-mRNA unknown gene [29]. We identified                  C. elegans human orthologs) (Figure 1D). We named <br /> the pre-mRNA gene tos-1 abnormally spliced uaf-1 and <br /> sfa-1 mutants determined tos-1 sensitive endogenous <br /> reporter analyzing vivo functions splicing factors [30]. In        Table 1. Suppression rubberband Unc phenotype by <br /> study, isolate, n4564 n5214, genetic                 mfap-1 mutations. <br /> screen identified uaf-1 sfa-1 mutations [29]. We <br /> found gene affected n4564 n5214 encodes novel <br />                                                                                                                            Bodybends 30 sec <br /> splicing factor affects alternative splicing C. elegans.                  Genotype                                 (&#177; SD)                    n <br />  <br />                                                                                   wild-type                                21.863.4                  20 <br /> Results <br />                                                                                   wild-type (25uC)                         28.364.4                  20 <br /> n4564 n5214 suppresses rubberband Unc phenotype                               unc-93(e1500)                            0.961.3                   20 <br /> of unc-93(e1500) animals                                                          unc-93(e1500) (25uC)                     0.860.8                   20 <br />    Previously performed genetic screen mutations that <br />                                                                                   unc-93(n200) [29]                        15.663.5                  20 <br /> caused sterility and/or lethality concurrently suppressed the <br />                                                                                   sup-10(n983) [29]                        4.361.9                   20 <br /> rubberband Unc phenotype caused unc-93(e1500) mutation <br /> [29]. Besides uaf-1(n4588) sfa-1(n4562), isolated                 sup-9(n1550); sup-18(n1014) [29]         0.160.4                   20 <br /> mutation n4564 [29], renamed n4564 n5214 study                   mfap-1(n4564 n5214)                      27.162.2                  20 <br /> (see below). Although precise mechanisms underlying                       mfap-1(n5214)                            28.463.3                  20 <br /> suppression rubberband Unc phenotypes uaf-1, sfa-1                  mfap-1(n5214) (25uC)                     28.364.0                  20 <br /> [29] n4564 n5214 (see below) mutations remain be <br />                                                                                   mfap-1(n4564 n5214); unc-93(e1500)       21.864.5                  20 <br /> determined, uaf-1 sfa-1 mutations isolated the <br /> splicing uaf-1 target tos-1 provide tools studying vivo         mfap-1(n4564 n5214); unc-93(n200)        13.963.0                  20 <br /> functions splicing factor genes [29,30], report                   mfap-1(n4564 n5214); sup-10(n983)        18.064.1                  20 <br /> include gene affected n4564 n5214.                                         mfap-1(n4564 n5214); sup-9(n1550);       0.360.5                   20 <br />    n4564 n5214 mutants exhibited temperature-sensitive lethality:                 sup-18(n1014) <br /> at 15uC, n4564 n5214 homozygous animals grew behaved                          mfap-1(n5214); unc-93(e1500)             1.861.3                   20 <br /> similarly wild type; 20uC, mutant animals grew                     mfap-1(n5214); unc-93(e1500) (25uC)      8.063.6                   20 <br /> slowly, progeny hyperactive (Table 1); 25uC, <br /> the mutant strain embryonically lethal. n4564 n5214 weakly                    doi:10.1371/journal.pgen.1002827.t001 <br />  <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                  2                               July 2012 | Volume 8 | Issue 7 | e1002827 <br />                                                                                                                      MFAP-1 Affects Alternative Splicing <br />  <br />  <br /> F43G9.10 mfap-1 (microfibrillar-associated protein 1). The chicken             1(n4564 n5214) probably results additive synergistic <br /> ortholog MFAP-1 suggested extracellular matrix                 effect mutations, likely reducing mfap-1 function. <br /> protein [32]. However, Drosophila ortholog MFAP-1 <br /> (dMFAP1) interacts splicing factor dPrp38, required                mfap-1 uaf-1 interact affect locomotion of <br /> for normal expression c-tubulin stg/cdc25 mRNAs                 unc-93(e1500) animals <br /> been proposed act splicing factor [33]. We                 To test interaction mfap-1 uaf-1, generated <br /> expression human MFAP1::GFP fusion protein (Hsmfap-1) C.               mfap-1(n4564 n5214); uaf-1 mutant animals carrying unc- <br /> elegans body-wall muscles rescued suppression unc-93(e1500)             93(e1500) mutation. Like mfap-1(n4564 n5214) animals, mfap- <br /> by mfap-1(n4564 n5214) (Figure 1B), indicating function            1(n4564 n5214); uaf-1(n5123) animals viable 20uC. mfap- <br /> MFAP-1 conserved <span id='am-71' about='obo:NCBITaxon_333870' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_195229 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41087 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_71193 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Nematodes Berthold, 1827" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Nematodes" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-76'  ></span>nematodes</span> humans.                                  1(n4564 n5214); uaf-1(n4588 n5125) mutants mfap-1(n4564 <br />    We examined expression pattern mfap-1 transgenic animals          n5214); uaf-1(n4588 n5127) mutants viable 15uC became <br /> using transcriptional fusion reporter construct drives GFP              sterile 20uC, mfap-1(n4564 n5214); uaf-1(n4588) mutants were <br /> expression control approximately 2.5 kb mfap-1                 likely inviable 15uC 20uC, failed obtain <br /> promoter (Figure 2A). We observed strong GFP expression                 progeny genotype mfap-1(n4564 n5214)/+; uaf- <br /> intestine, pharynx vulval muscles (Figure 2A). We observed            1(n4588)/+ hermaphrodites (Table 3). All viable mfap-1; uaf-1 <br /> GFP expression body-wall muscles (Figure 2A), consistent           multiple mutant animals exhibited strong suppression unc- <br /> the finding body-wall muscle-specific expression mfap-1 rescued        93(e1500) Unc phenotype, similar mfap-1(n4564 n5214) <br /> the suppression unc-93(e1500) Unc phenotype mfap-1(n4564             mutant animals (Table 3). That mfap-1; uaf-1 multiple mutant <br /> n5214) (Figure 1). Both C. elegans MFAP-1::GFP human MFAP-                 animals, mfap-1(n4564 n5214); uaf-1(n5123), exhibited a <br /> 1::GFP, expressed body-wall muscles, exclusively                  stronger temperature-sensitive sterile lethal phenotype either <br /> localized nuclei (Figure 2B unpublished observations), indicating       mfap-1 uaf-1 mutant animals suggests mfap-1(n4564 n5214) <br /> that MFAP-1 nuclear protein.                                              uaf-1 mutations additive effects animal survival. <br />    We obtained 746 bp mfap-1 deletion allele, tm3456D (kindly                   We examined mfap-1(n5214) enhance the <br /> provided S. Mitani, personal communication), removes                  suppressor activities uaf-1 mutations Unc phenotype of <br /> the entire intron second exon mfap-1                  unc-93(e1500) animals generating mfap-1(n5214); uaf-1 double <br /> (Figure 1C). tm3456D predicted encode truncated protein                mutant animals carrying unc-93(e1500) mutation (Table 3). <br /> with frameshift amino acid 53, suggesting tm3456D              mfap-1(n5214); uaf-1(n5123) double mutants similar mfap- <br /> likely null allele mfap-1. mfap-1(tm3456D)/+animals grew              1(n5214) uaf-1(n5123) single mutants substantially <br /> behaved like wild type, mfap-1(tm3456D) homozygous                     suppressed Unc phenotype unc-93(e1500) animals <br /> animals arrested developmentally L1 L2 larval stages.                (Table 3). mfap-1(n5214) enhanced suppressor activity the <br /> tm3456D/n4564 n5214 animals similarly arrested L1/L2                    uaf-1(n4588 n5125) mutation. As expected, mfap-1(n5214); <br /> larval stages, suggesting lethal phenotype tm3456D                 uaf-1(n4588 n5127) mfap-1(n5214); uaf-1(n4588) strongly <br /> homozygotes caused mfap-1(tm3456D) mutation                      suppressed unc-93(e1500), uaf-1(n4588 n5127) uaf- <br /> consistent hypothesis mfap-1(n4564 n5214) causes               1(n4588) strong suppressors unc-93(e1500) [29]. Although <br /> reduction loss mfap-1 function.                                          mfap-1(n5214) uaf-1(n4588 n5127) mutants apparently <br />                                                                                healthy 20uC, mfap-1(n5214); uaf-1(n4588 n5127) mutant <br /> The D426V(n5214) T427A(n4564) mutations act                                animals partially sterile 20uC (L. Ma H. R. <br /> together suppress unc-93(e1500)                                             Horvitz, unpublished observations). Furthermore, mfap-1(n5214); <br />    The temperature-sensitive lethal phenotype mfap-1(n4564                  uaf-1(n4588) double mutant animals viable 15uC but <br /> n5214) 25uC provided approach identification genes             sterile grown 20uC, severe <br /> that interact mfap-1 seeking suppressors tempera-               temperature-sensitive phenotype uaf-1(n4588) single <br /> ture-sensitive lethality. We screened 50,000 haploid                     mutants. That mfap-1 mutations enhance temperature-sensitive <br /> genomes identified independent suppressors,                   phenotypes uaf-1 mutants suggests mfap-1 uaf-1 might <br /> later cause intragenic change mfap-1 (see                 act parallel affecting unc-93 function animal survival. <br /> Materials Methods). Specifically, suppressors caused                   We tested mfap-1(n4564 n5214), like uaf-1(n4588) mutation <br /> reversion T427A (n4564) mutation (see Materials                     [29], cause recognition cryptic 39 splice site exon 9 of <br /> Methods) wild-type <span id='am-63' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-64'  ></span><span id='am-65' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> (ACA) amino acid (A427T)                   unc-93(e1500). We did observe obviously altered recognition <br /> and retained D426V (n5214) mutation. mfap-1(n5214) did                 cryptic 39 splice site (1.3% wild type vs. 2% mfap- <br /> obviously suppress rubberband Unc phenotype unc-                        1(n4564 n5214) animals) (Figure S1). Therefore, suppression of <br /> 93(e1500) animals 20uC did 25uC (Table 1),                     Unc phenotype unc-93(e1500) mfap-1(n4564 n5214) might <br /> indicating mfap-1(n5214) temperature-sensitive allele.               caused altered splicing unc-93(e1500) transcript. <br />    We examined mfap-1 mutations, n4564 n5214, <br /> contribute activity mfap-1(n4564 n5214) suppressor           mfap-1(n4564 n5214) alters splicing tos-1 by <br /> the Unc phenotype unc-93(e1500) animals. We expressed MFAP-                 increasing intron 1 retention exon 3 skipping <br /> 1::GFP fusion transgenes containing n4564 (T427A) mutation,                   To analyze mfap-1 affect alternative splicing, we <br /> the n5214 (D426V) mutation mutations body-wall                  examined splicing tos-1 mfap-1(n4564 n5214) animals. We <br /> muscles mfap-1(n4564 n5214); unc-93(e1500) animals assayed              previously described tos-1 sensitive endogenous reporter for <br /> the locomotion transgenic animals (Table 2). Both mfap-                 alternative splicing C. elegans [30]. As shown Figure 3A&#8211;3C, <br /> 1(n4564)::GFP mfap-1(n5214)::GFP transgenes greatly                    mfap-1(n4564 n5214) increased tos-1 intron 1 retention exon 3 <br /> rescued suppression unc-93(e1500) mfap-1(n4564 n5214),               skipping, suggesting recognition 39 splice sites in <br /> while mfap-1(n4564 n5214)::GFP transgene partially rescued            intron 1 exon 3 reduced mfap-1(n4564 n5214) <br /> the suppression (Table 2). These results suggest individually         animals. We note tos-1 splice isoforms observed in <br /> mutations n4564 n5214 slightly affect function        mfap-1(n4564 n5214) mfap-1(RNAi) animals seen in <br /> mfap-1 strong suppression unc-93(e1500) mfap-               wild-type animals (Figure 3A), indicating alterations mfap-1 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                3                             July 2012 | Volume 8 | Issue 7 | e1002827 <br />                                                                                                                     MFAP-1 Affects Alternative Splicing <br />  <br />  <br />  <br />  <br /> Figure 1. Genetic mapping, cloning, identification mfap-1. (A) Genomic location n4564 n5214 mutation based genetic <br /> mapping using visible markers SNPs (e.g., SNP T01H8: 7110 left SNP F36A2: 3334 right). Cosmids tested rescue experiments <br /> are labeled. Only cosmid F43G9 rescued phenotypic characteristics mfap-(n4564 n5214) individuals: suppression Unc phenotype of <br /> unc-93(e1500) animals; partial sterility 20uC; temperature-sensitive lethality 25uC. (B) Transgene rescue experiments. Transgenes (nEx) were <br /> injected gonads mfap-1(n4564 n5214); unc-93(e1500) animals, lines stably transmitting transgenes established. Transgenic <br /> lines analyzed rescue of abnormalities: suppression Unc phenotype unc-93(e1500) animals; partial sterility 20uC; <br /> and temperature-sensitive lethality 25uC. mut: n4564 n5214 mutation. (C) Gene structure mfap-1. Black boxes: coding exons. Open boxes: 59 <br /> and 39 UTRs. Positions start (ATG) stop codons (TGA) indicated. The sites n4564 n5214 mutation, n5214 mutation the <br /> tm3456D deletion labeled. (D) Sequence alignment predicted MFAP-1 proteins C. elegans, Drosophila, chicken human. *: amino acids <br />  <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                4                             July 2012 | Volume 8 | Issue 7 | e1002827 <br />                                                                                                                  MFAP-1 Affects Alternative Splicing <br />  <br />  <br />  <br /> mutated mfap-1(n4564 n5214) mutants. Amino acids conserved orthologs darkly shaded, amino acids similar <br /> physical properties conserved orthologs lightly shaded. <br /> doi:10.1371/journal.pgen.1002827.g001 <br />  <br /> affect tos-1 alternative splicing events occur wild type.        intron 1 cryptic 39 splice site (Figure S2B), suggesting mfap- <br /> We examined mfap-1(n4564 n5214) cause                     1(n5214) does alter weak effect tos-1 splicing. <br /> recognition cryptic 39 splice site tos-1 intron 1,       This result consistent notion n5214 weaker <br /> recognized uaf-1(n4588) animals [30]. We did detect                   mfap-1 allele n4564 n5214. <br /> obvious recognition cryptic 39 splice sites mfap-1(n4564              We examined splicing tos-1 animals fed dsRNA <br /> n5214) animals (Figure 3D), suggesting mfap-1(n4564 n5214)              targeting mfap-1. We observed apparent increase tos-1 <br /> did alter specificity 39 splice-site recognition. We              isoforms 1 2 (Figure 4A). Increased expression isoforms 1 <br /> similarly examined mfap-1(n5214) animals did observe                 2 caused increased intron 1 retention exon 3 <br /> obvious effects splicing tos-1 recognition           skipping tos-1 splicing, seen mfap-1(n4564 n5214) <br />  <br />  <br />  <br />  <br /> Figure 2. MFAP-1 nuclear protein expressed multiple tissues. (A) A 2.5 kb 59 promoter region upstream start codon mfap-1 <br /> was used drive expression transcriptional fusion GFP reporter transgenic animals. The tissues GFP reporter expressed <br /> are labeled. This figure shows montage photographs, pseudo-colored gray-scale image green-scale image using <br /> Adobe Photoshop reflect level GFP expression. (B) Transgene experiments performed described Materials Methods. The C. <br /> elegans MFAP-1::GFP fusion protein, expressed body-wall muscles using myo-3 promoter wild-type mfap-1(n4564 n5214) <br /> animals mixed developmental stages, exclusively localized nuclei (left middle panels). Similarly, human MFAP-1::GFP fusion protein, <br /> when expressed body-wall muscles mfap-1(n4564 n5214) animals, exclusively localized nuclei (right panels). Lower panels show <br /> typical nuclear GFP signals adult transgenic animals. Genetic backgrounds transgenes indicated. <br /> doi:10.1371/journal.pgen.1002827.g002 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              5                            July 2012 | Volume 8 | Issue 7 | e1002827 <br />                                                                                                                               MFAP-1 Affects Alternative Splicing <br />  <br />  <br />  <br />  Table 2. mfap-1(n4564) mfap-1(n5214) weak                                   Table 3. Mutations mfap-1 uaf-1 interact affect the <br />  mutations act suppress rubberband Unc                          locomotion unc-93(e1500) animals. <br />  phenotype unc-93(e1500) animals. <br />  <br />                                                                                                                                            Body-bends per <br />                                                       Bodybends                  Genotype                                              30 sec &#177; SD    n <br />  Genotype                                             30 sec (&#177;SD)       n <br />                                                                                      wild-type [29]                                        20.463.7          20 <br />  nEx[Pmyo-3 mfap-1 cDNA (wt)::GFP] line 1             1.361.4            20          unc-93(e1500) [29]                                    0.961.2           20 <br />  nEx[Pmyo-3 mfap-1 cDNA (wt)::GFP] line 2             2.162.1            20          mfap-1(n5214); unc-93(e1500)                          1.861.3           20 <br />  nEx[Pmyo-3 mfap-1 cDNA (wt)::GFP] line 3             1.762.0            20          mfap-1(n4564 n5214); unc-93(e1500)                    21.864.5          20 <br />  nEx[Pmyo-3 mfap-1 cDNA (n4564)::GFP] line 1          1.361.1            20          uaf-1(n5123) unc-93(e1500) [29]                       0.660.8           20 <br />  nEx[Pmyo-3 mfap-1 cDNA (n4564)::GFP] line 2          1.661.7            20          mfap-1(n5214); uaf-1(n5123) unc-93(e1500)             2.261.9           20 <br />  nEx[Pmyo-3 mfap-1 cDNA (n4564)::GFP] line 3          3.861.4            20          mfap-1(n4564 n5214); uaf-1(n5123) unc-93(e1500)       20.962.9          20 <br />  nEx[Pmyo-3 mfap-1 cDNA (n5214)::GFP] line 1          3.461.6            20          uaf-1(n4588 n5125) unc-93(e1500) [29]                 5.462.1           20 <br />  nEx[Pmyo-3 mfap-1 cDNA (n5214)::GFP] line 2          2.262.5            9           mfap-1(n5214); uaf-1(n4588 n5125) unc-93(e1500)       10.265.7          20 <br />  nEx[Pmyo-3 mfap-1 cDNA (n5214)::GFP] line 3          3.161.6            20          mfap-1(n4564 n5214); uaf-1(n4588 n5125) unc-          20.564.5          20 <br />  nEx[Pmyo-3 mfap-1 cDNA (n4564 n5214)::GFP] line 1    10.164.7           20          93(e1500)* <br />  <br />  nEx[Pmyo-3 mfap-1 cDNA (n4564 n5214)::GFP] line 2    14.366.4           20          uaf-1(n4588 n5127) unc-93(e1500) [29]                 21.563.6          20 <br />                                                                                      mfap-1(n5214); uaf-1(n4588 n5127) unc-93(e1500)       21.264.4          20 <br />  All transgenes expressed mfap-1(n4564 n5214); unc-93(e1500) animals. <br />  Two lines obtained analyzed transgene Pmyo-3 mfap-1 cDNA (n4564        mfap-1(n4564 n5214); uaf-1(n4588 n5127) unc-          20.564.7          20 <br />  n5214)::GFP.                                                                        93(e1500)* <br />  doi:10.1371/journal.pgen.1002827.t002                                               uaf-1(n4588) unc-93(e1500) [29]                       21.065.5          20 <br />                                                                                      mfap-1(n5214); uaf-1(n4588) unc-93(e1500)*            17.764.0          20 <br /> animals (Figure 3). The similarity mfap-1(n4564 n5214)                        mfap-1(n4564 n5214); uaf-1(n4588) unc-93(e1500)**     Inviable          20 <br /> mfap-1(RNAi) affecting tos-1 splicing suggests n4564                 mfap-1(n5214)                                         28.463.3          20 <br /> n5214 causes reduction loss mfap-1 function. Reducing <br />                                                                                      mfap-1(n4564 n5214)                                   27.162.2          20 <br /> mfap-1 expression RNAi feeding did cause recognition <br /> of cryptic 39 splice site intron 1 (Figure 4B). Similarly,               uaf-1(n5123) [29]                                     19.163.1          20 <br /> cryptic splice site recognized mfap-1(n4564 n5214)                        uaf-1(n4588 n5125) [29]                               19.563.3          20 <br /> animals (Figure 3D).                                                                 uaf-1(n4588 n5127) [29]                               22.164.0          20 <br />    To determine uaf-1 mutations mfap-1 mutations                         uaf-1(n4588) [29]                                     23.164.1          20 <br /> could interact affecting RNA splicing, examined splicing <br /> of tos-1 recognition cryptic 39 splice site tos-1                  *Sterile 20uC. <br /> intron 1 mfap-1; uaf-1 mutant animals. For mfap-1; uaf-1                     **Inviable. <br />                                                                                      doi:10.1371/journal.pgen.1002827.t003 <br /> mutant, splicing tos-1 appeared similar the <br /> corresponding uaf-1 single mutant mfap-1 single mutant (Figure <br /> S2A, S2B, panels). We did observe additive synergistic <br />                                                                                     mutations splicing factors clf1Delta2 prp17/cdc40Delta in <br /> effects uaf-1 mfap-1 mutations intron 1 retention (Figure <br />                                                                                     S. cerevisiae [34], mammalian orthologs SYF2 identified <br /> S3A, e.g., compare uaf-1(n4588 n5127) mfap-1(n4564 n5214); <br />                                                                                     various spliceosomal complexes proteomic approaches [3], <br /> uaf-1(n4588 n5127)) exon 3 skipping (Figure S3B, e.g., compare                   suggesting SYF2 acts splicing factor. We identified <br /> mfap-1(n4564 n5214) mfap-1(n4564 n5214); uaf-1(n4588                           MFAP-1 interactor two-hybrid screen, indicating that <br /> n5127)). Interestingly, recognition cryptic 39 splice site               MFAP-1 form homodimers. The potential inter- <br /> was reduced 50% mfap-1(n4564 n5214); uaf-1(n4588                        actors proteins involved rRNA processing (C05C8.2 <br /> n5127) mfap-1(n5214); uaf-1(n4588 n5127) mutant animals                         T22H9.1), ribosomal protein (RPS-6), Ras-associated <br /> compared uaf-1(n4588 n5127) mutant animals (Figure                       PH-domain containing protein (MIG-10), Rab11 family- <br /> S2A, S2B, panels Figure S3C), suggesting wild-                      interacting protein 2 (Y39F10B.1) protein unknown <br /> type mfap-1 function required recognition                   function (F43C11.9). Although interactions MFAP-1 <br /> cryptic 39 splice site uaf-1(n4588 n5127) animals.                               proteins yeast two-hybrid screen remain be <br />                                                                                     verified approaches, observation presump- <br /> MFAP-1 interacts physically known splicing factors                             tive splicing factors (Prp38 SYF2) candidates <br /> in yeast two-hybrid experiment                                                    consistent findings Andersen and <br />    To understand function MFAP-1, sought                       Tapon [33] indicating MFAP-1 function splicing <br /> identify interacting proteins MFAP-1 using yeast two-hybrid                    factor. <br /> screen (see Materials Methods). We isolated retested                      We examined splicing tos-1 animals fed dsRNAs <br /> genes encoding proteins potentially interact                        targeting mfap-1, D1054.14 K07G7.11 (Figure S4). We found <br /> MFAP-1 (Table S1). Two genes, K04G7.11 D1054.14,                         D1054.14(RNAi) caused increase tos-1 splice isoform 2 <br /> encode C. elegans orthologs presumptive splicing factors                 similar seen mfap-1(RNAi) animals (Figure S4). No <br /> SYF2 [34] Prp38 [35]. Drosophila MFAP-1 (dMFAP1)                            apparent alteration tos-1 splicing detected K07G7.11(RNAi) <br /> found protein complex dPrp38 co-immunopre-                       animals. This result consistent previous studies indicating <br /> cipitation experiments interact dPrp38 physically [33].                 homologs D1054.14 function splicing factors [33,35] <br /> SYF2 mutations cause synthetic lethality                         identified tos-1 target D1054.14 C. elegans. <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                     6                               July 2012 | Volume 8 | Issue 7 | e1002827 <br />                                                                                                                  MFAP-1 Affects Alternative Splicing <br />  <br />  <br />  <br /> Discussion                                                                    [30]. Thus, differing effects target gene splicing RNAi- <br />                                                                               treatment genetic mutation splicing factor gene be <br /> MFAP-1 essential animal development                                    caused factors, including differing level the <br />    MFAP1 initially identified putative extracellular matrix          splicing factor RNAi-treated mutant animals effects <br /> protein based screen embryonic chicken cDNA                        mutations function splicing factor. <br /> expression library using antibodies elastic fiber microfi- <br /> brils-enriched bovine ocular zonule proteins [32]. Our analysis            MFAP-1 interact multiple partners affect <br /> C. elegans mfap-1 study Drosophila ortholog dMFAP1             RNA splicing <br /> [33] indicate MFAP-1 orthologs <span id='am-68' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-69' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>                     From yeast two-hybrid screen, identified candidate <br /> nuclear proteins affect RNA splicing. In addition, human             MFAP-1-interacting proteins. One, D1054.14, homologous to <br /> ortholog MFAP-1 associated spliceosomes               S. cerevisiae splicing factor Prp38 [35]. Another, K04G7.11, is <br /> by mass spectrometry analysis [36&#8211;38]. Since human ortholog               orthologous S. cerevisiae candidate splicing factor SYF2 [34]. <br /> of mfap-1 rescue activity C. elegans mfap-1(n4564 n5214)           Drosophila splicing factors dPrp38 dSYF1 a <br /> for suppressing Unc phenotype unc-93(e1500) animals,               protein complex dMFAP1, dPrp38 interacts with <br /> function MFAP-1 likely conserved nematodes                      dMFAP1 GST-pulldown experiments [33]. We did identify <br /> humans.                                                                       orthologs SYF1 splicing factors the <br />    Drosophila dMFAP1 acts splicing factors              Drosophila dMFAP1 complex, possibly co-immuno- <br /> G2/M progression mitosis affects ratio pre-                 precipitation experiments identified Drosophila proteins <br /> mRNA mature mRNA c-tubulin gene mRNA                        did directly interact dMFAP1, yeast <br /> level stg/cdc25 gene [33]. It known dMFAP1 affects           two-hybrid screen identify proteins directly interact <br /> alternative splicing [33]. C. elegans mfap-1(n4564 n5214) mutants             MFAP-1. Both studies suggest MFAP-1 interacts with <br /> exhibit temperature-sensitive phenotype normal growth               multiple splicing factors directly indirectly that <br /> 15uC embryonic lethality 25uC; mfap-1(tm3456D) homozy-                 MFAP-1 play multiple roles affecting RNA splicing. <br /> gous animals arrest L1 L2 larval stages 20uC, mfap-1                We reducing expression D1054.14 RNAi <br /> RNAi-treated animals arrest variable larval stages 20uC (L.             caused alterations tos-1 splicing similar caused mfap- <br /> Ma H. R. Horvitz, unpublished observations). These                        1(RNAi) (Figure S4), suggesting MFAP-1 D1054.14/ <br /> observations indicate C. elegans mfap-1 essential animal          PRP38 interact regulate splicing set of <br /> development. Whether developmental defect mfap-1-                      genes. We did identify alterations tos-1 splicing animals <br /> deficient animals caused mitotic defects remains                  fed dsRNA targeting K04G7.11, indicating K04G7.11 might <br /> determined.                                                                   required splicing tos-1. <br />  <br /> MFAP-1 probably required recognition weak                       mfap-1 uaf-1 interact affect RNA splicing <br /> 39 splice sites                                                                  We previously concluded suppression rubber- <br />    We isolated mfap-1(n4564 n5214) genetic screen               band Unc phenotype unc-93(e1500) animals uaf-1(n4588) and <br /> in identified mutations affecting uaf-1 sfa-1 [29].              sfa-1(n4562) probably caused altered splicing unknown <br /> mfap-1(n4564 n5214) suppressed phenotypes different                    gene [29]. The splicing unc-93(e1500) appears normal in <br /> rubberband Unc mutants patterns similar uaf-                   mfap-1(n4564 n5214) animals, indicating altered splicing of <br /> 1(n4588) sfa-1(n4562) mutations. The findings led                   unc-93(e1500) transcripts similarly cause the <br /> hypothesize MFAP-1 function splicing factor. Our              suppression unc-93(e1500) Unc phenotype mfap-1(n4564 <br /> analysis tos-1 splicing mfap-1 mutant animals animals            n5214). <br /> treated mfap-1 RNAi provided molecular evidence mfap-1                 The uaf-1(n4588) mutation likely causes recognition the <br /> can affect alternative splicing. mfap-1(n4564 n5214) mfap-                cryptic 39 splice site tos-1 intron 1 altering than <br /> 1(RNAi) increased tos-1 intron 1 retention (with weak 39               decreasing function UAF-1 protein [30]. uaf-1(n4588 <br /> splice site) tos-1 exon 3 skipping (with different weak 39 splice       n5127) weaker mutation uaf-1(n4588), the <br /> site), affected splicing intron 3 (with strong           recognition cryptic 39 splice site uaf-1(n4588 n5127) <br /> consensus 39 splice site TTTTCAG). These observations suggest                 animals uaf-1(n4588) animals reduced the <br /> that MFAP-1 required recognition weak 39                  presence mfap-1(n4564 n5214) mfap-1(n5214) mutations <br /> splice sites recognition strong 39 splice sites. We        (Figure S3C). These results suggest recognition the <br /> propose MFAP-1 act like UAF-1 SFA-1,                        cryptic 39 splice site tos-1 requires MFAP-1 that <br /> mutations cause reduced recognition weak 39 splice            recognition weak uaf-1(n4588 n5127) mutants. <br /> sites introns 1 2 affect recognition             These results support conclusion MFAP-1 affects <br /> consensus 39 splice site intron 3 tos-1 [30]. Similar sfa-           alternative splicing. <br /> 1(n4562), mfap-1(n4564 n5214) mfap-1(RNAi) did cause                      In short, propose MFAP-1 act splicing factor. <br /> recognition cryptic 39 splice site tos-1 intron 1, suggesting       Future studies reveal MFAP-1 interacts other <br /> that MFAP-1 probably essential determining                     splicing factors affect C. elegans development regulating <br /> specificity 39 splice sites.                                               splicing target genes. <br />    The effects mfap-1(n4564 n5214) mfap-1(RNAi) tos-1 <br /> splicing qualitatively similar quantitatively different. We           Materials Methods <br /> also observed differences effects tos-1 splicing genetic <br /> mutations RNAi treatments previous studies uaf-1                Strains <br /> and sfa-1, uaf-1(n4588) sfa-1(n4562)                   C. elegans strains grown 20uC described unless <br /> caused dramatic alterations tos-1 splicing did            indicated [39]. N2 (Bristol) reference wild-type <br /> uaf-1(RNAi) sfa-1(RNAi). This difference partially            strain [39]. Other strains used study were: <br /> caused altered function UAF-1 uaf-1(n4588) animals                   LGI: mfap-1(n4564 n5214, n5214, tm3456D) (this study). <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                               7                            July 2012 | Volume 8 | Issue 7 | e1002827 <br />                                                                                                                          MFAP-1 Affects Alternative Splicing <br />  <br />  <br />  <br />  <br /> Figure 3. mfap-1(n4564 n5214) affects splicing tos-1. (A) RT-PCR experiments showing effect mfap-1(n4564 n5214) tos-1 <br /> alternative splicing. tos-1 splice isoforms illustrated right. (B) The molar ratios tos-1 splice isoforms intron 1 retention (isoforms <br /> 1 2), presented percentage isoforms combined (Isoforms 1, 2, 3, 4 5). Error bars: standard deviations. *p,0.05. (C) The molar <br /> ratios tos-1 splice isoforms exon 3 skipping (isoforms 2 5), presented percentage isoforms combined (isoforms 1, 2, 3, 4 <br /> and 5). Error bars: standard deviations. *p,0.05. (D) RT-PCR experiments showing effect mfap-1(n4564 n5214) mutation the <br /> recognition cryptic 39 splice site tos-1 intron 1. For analyses, isoform intensities obtained analyzing biological duplicates or <br /> triplicates using NIH ImageJ software. <br /> doi:10.1371/journal.pgen.1002827.g003 <br />  <br />         PLoS Genetics | www.plosgenetics.org                                  8                              July 2012 | Volume 8 | Issue 7 | e1002827 <br />                                                                                                               MFAP-1 Affects Alternative Splicing <br />  <br />  <br />    LGII: sup-9(n1550) [20].                                                Drosophila, chicken human highly conserved, Asp- <br />    LGIII: uaf-1(n4588, n5123, n4588 n5125, n4588 n5127) [29],              Thr dipeptide altered mfap-1(n4564 n5214) mutant is <br /> unc-93(e1500, n200) [23], sup-18(n1014) [21].                              completely conserved (Figure 1D). A genomic fragment containing <br />    LGIV: sfa-1(n4562) [29].                                                2.5 kb promoter region, entire 1.7 kb coding region a <br />    LGX: sup-10(n983) [21].                                                 1.5 kb 39 downstream region exhibited rescuing activity similar to <br />    CB4856 (Hawaiian) [40]                                                  F43G9 cosmid (Figure 1B). Transgenes expressing <br />    We used genetic translocation hT2[bli-4(e937) let-?(q782)           F43G9.10 human ortholog F43G9.10 (Hsmfap-1) in <br /> qIs48] LG I; LG III [41], carries integrated pharyngeal           body-wall muscles rescued suppression unc-93(e1500) <br /> GFP element [42], balance mfap-1(tm3456D) mutation.                 mfap-1(n4564 n5214) (Figure 1B). A transgene carrying the <br />                                                                            mfap-1(n4564 n5214) mutation (mut) failed rescue suppres- <br /> Cloning mfap-1                                                          sion unc-93(e1500) mfap-1(n4564 n5214) (Figure 1B), <br />    We used single-nucleotide polymorphism (SNP) mapping [40]               suggesting mutation caused suppressor activity in <br /> to localize n4564 n5214 right nucleotide 7110 (SNP               mfap-1(n4564 n5214) animals. <br /> T01H8: 7110 S = AT) cosmid T01H8 (T01H8 sequences refer <br /> to nucleotides GenBank accession no. Z80219) left         Screen suppressors mfap-1(n4564 n5214) <br /> nucleotide 3334 (SNP F36A2: 3334 S = CT) cosmid F36A2                      Synchronized L4 mfap-1(n4564 n5214) animals (P0) grown at <br /> (F36A2 sequences refer nucleotides GenBank accession                 15uC mutagenized ethyl methanesulfonate (EMS) as <br /> no. Z81077) (Figure 1A). We isolated 10 Dpy recombinants             described [39] grown gravid adults 15uC. The P0 adults <br /> crossing dpy-5(e61) n4564 n5214 hermaphrodites males           bleached, F1 progeny grown young adults at <br /> Hawaiian strain CB4856, Unc recombinants crossing            15uC moved 25uC. After days 25uC, plates were <br /> n4564 n5214 unc-75(e950) hermaphrodites CB4856 males.                 examined surviving fertile animals. From 50,000 haploid <br /> Cosmid F43G9 rescued phenotypic characteristics               genomes (F2) screened, independent suppressed strains were <br /> n4564 n5214 strain: suppression unc-93(e1500), partial sterility        isolated. As animals mfap-1(n4564 n5214) phenotype (partial <br /> and temperature-sensitive lethality 25uC (Figure 1B). We                sterility 20uC temperature-sensitive lethality 25uC) were <br /> determined coding sequences F43G9.10 identified             identified 1800 progeny mfap-1(n4564 n5214) sup/ <br /> missense mutations (GAT-to-GTT (D426V) ACA-to-GCA                      ++animals, concluded suppressors either <br /> (T427A)) fourth exon F43G9.10 (Figure 1C). The                   intragenic extragenic closely linked mfap-1. DNA <br /> predicted proteins encoded F43G9.10 orthologs C. elegans,        sequence determination identified suppressors same <br />  <br />  <br />  <br />  <br /> Figure 4. Reducing mfap-1 expression RNA interference alters splicing tos-1. (A) RT-PCR experiments showing effect of <br /> reducing mfap-1 expression RNAi feeding tos-1 alternative splicing. tos-1 splice isoforms illustrated right. (B) RT-PCR experiments <br /> showing effect reducing mfap-1 expression RNAi feeding recognition cryptic 39 splice site tos-1 intron 1. <br /> doi:10.1371/journal.pgen.1002827.g004 <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                             9                           July 2012 | Volume 8 | Issue 7 | e1002827 <br />                                                                                                                  MFAP-1 Affects Alternative Splicing <br />  <br />  <br /> nucleotide change (GCA-to-ACA codon 427), converted                 analysis performed using ABI Prism 3100 Genetic <br /> the mutated amino acid alanine mfap-1(n4564 n5214) mutant             Analyzer ABI 3730XL DNA Analyzer. <br /> to wild-type threonine (A427T) retaining D426V <br /> mutation. One isolates used subsequent studies,           <span id='am-102' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-103' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-104' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>Plasmids</span> <br /> and single mutation carried designated mfap-1(n5214).                mfap-1 cDNAs amplified RT-PCR wild-type, <br />                                                                              mfap-1(n4564 n5214) mfap-1(n5214) animals subcloned to <br /> Yeast two-hybrid screen                                                      pPD93.97 vector in-frame GFP using BamHI blunt/AgeI <br />    We used yeast two-hybrid screen [43,44] identify proteins            restriction sites. For constructs expressing mfap-1(n4564) <br /> that interact MFAP-1. A pACT2.2 C. elegans yeast two-             transgene, n4564 mutation introduced the <br /> hybrid library (Addgene plasmid 11523, provided Dr. Guy                   pPD93.97::mfap-1cDNA(wt) construct using QuickChange Site- <br /> Caldwell) used screen MFAP-1 interactors. For bait,           Directed Mutagenesis Kit (Stratagene) primers containing the <br /> a full-length mfap-1 cDNA subcloned pGBK T7                     n4564 mutation. For plasmid used bait yeast two- <br /> plasmid (Clontech) transfected S. cerevisiae strain        hybrid screen, full-length mfap-1 cDNA released the <br /> PJ69-4A. pGBK T7-mfap-1-positive yeast cells transfected                pGEM-T Easy vector (Promega) NcoI blunt/PstI and <br /> with 2 mg DNA pACT2.2 yeast two-hybrid library,                  subcloned in-frame pGBK T7 vector (Trp1, Kanr) <br /> and cells grown SD-Leu-Trp-His medium 5 mM                    (Clontech) using BamHI blunt/PstI restriction sites. To construct a <br /> 3-AT. Positive clones picked grown separate cultures             Pmfap-1::GFP transcriptional fusion transgene, mfap-1 promoter <br /> using SD medium. Plasmids obtained                   fragment 2.5 kb amplified using PCR subcloned <br /> clone smash-and-grab method [45]. Purified plasmids              pPD95.79 vector using PstI/SmaI restriction sites. <br /> transfected bacterial strain DH5a grown LB- <br /> ampicillin plates. Plasmids purified bacterial                 Transgene experiments <br /> cultures, sequences inserts determined. Using                   Microinjection DNA syncytial gonad the <br /> yeast two-hybrid assays, confirmed clones did              generation animals germline transmission transgenes <br /> cause survival yeast cells transfected pGBK T7             performed described [50] mfap-1(n4564 n5214) <br /> vector SD-Leu-Trp-His medium, suggesting proteins                    animals grown 15uC. DNA injection mixtures generally <br /> encoded clones potentially interact MFAP-1                 contained 20 mg/ml 1 kb DNA ladder, 20 mg/ml <span id='am-12' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span><span id='am-17'  ></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Arabidopsis</span> <br /> not GAL4 DBD domain expressed pGBK T7                        genomic DNA 20 mg/ml transgene interest. When <br /> vector.                                                                      transgene did express GFP fusion protein, 20 mg/ml <br />                                                                              pPD95.86-GFP plasmid (which expresses GFP body-wall <br /> RNA interference                                                             muscles) added injection mixture visible <br />    Young adult animals fed HT115 (DE3) bacteria contain-                fluorescence marker. <br /> ing plasmids directing expression dsRNAs targeting uaf-1, <br /> sfa-1, mfap-1, D1054.14 K04G7.11 NGM plates 1 mM                  Supporting Information <br /> IPTG 0.1 mg/ml Ampicillin [46]. F1 progeny these <br />                                                                              Figure S1 mfap-1(n4564 n5214) does obviously affect the <br /> animals, arrested various developmental stages, were <br />                                                                              altered splicing unc-93(e1500) exon 9. Real-time RT-PCR was <br /> washed plates, rinsed H2O resuspended Trizol <br />                                                                              performed described [29] quantify recognition the <br /> (Invitrogen) preparation total RNA. The DNA construct <br />                                                                              cryptic 39 splice site unc-93(e1500) exon 9. No apparent <br /> that expresses dsRNA targeting uaf-1 bacterial strain that <br />                                                                              difference unc-93(e1500) mfap-1(n4564 n5214); unc- <br /> expresses dsRNA targeting sfa-1 described previously [29]. <br />                                                                              93(e1500) animals detected. *: results reported previously by <br /> We obtained bacterial strain expressing dsRNA targeting mfap-1 <br />                                                                              Ma Horvitz (2009) [29]. <br /> from whole-genome RNAi library [47] bacterial strains <br />                                                                              (TIF) <br /> expressing dsRNAs targeting mfap-1, D1054.14 K04G7.11 from <br /> an ORFeome-based RNAi library [48]. The sequences plasmids                Figure S2 mfap-1 uaf-1 mutations exhibit additive or <br /> from single colonies determined confirm presence              synergistic effects altered splicing tos-1. (A) RT-PCR <br /> coding sequences gene.                                              experiments showing splicing tos-1 (top panel) the <br />                                                                              recognition cryptic 39 splice site tos-1 intron 1 (bottom <br /> Body-bend assay                                                              panel) mfap-1(n4564 n5214) single mfap-1(n4564 n5214); uaf-1 <br />    L4 animals picked 16&#8211;24 hrs tested. One day             mutant animals. Genotypes indicated top, tos-1 <br /> later, young adults individually picked plates OP50             splice isoforms right. (B) RT-PCR experiments showing the <br /> bacteria, body-bends counted 30 sec using                     splicing tos-1 (top panel) recognition cryptic 39 <br /> dissecting microscope described [49].                                     splice site tos-1 intron 1 (bottom panel) mfap-1(n5214) single or <br />                                                                              mfap-1(n5214); uaf-1 double mutant animals. Genotypes are <br /> Molecular biology                                                            indicated top, tos-1 splice isoforms right. <br />    Total RNA purified Trizol (Invitrogen), cDNA                 (TIF) <br /> was generated following protocol provided Super-                Figure S3 Quantification tos-1 intron 1 retention, exon 3 <br /> script II kit (Invitrogen). PCR performed Eppendorf                 skipping recognition intron 1 cryptic 39 splice site in <br /> Cyclers, DNA products resolved using agarose gels. NIH              mfap-1; uaf-1 multiple mutant animals. (A) The molar ratios all <br /> ImageJ software used quantify tos-1 splice isoforms [30].             tos-1 splice isoforms intron 1 retention, presented a <br /> The percentages tos-1 intron 1 retention, exon 3 skipping             percentage isoforms combined. Error bars: standard <br /> the recognition intron 1 cryptic 39 splice analyzed           deviations. (B) The molar ratios tos-1 splice isoforms with <br /> described [30]. We performed RT-PCR experiments animals                 exon 3 skipping, presented percentage isoforms <br /> at different developmental stages indication tos-          combined. Error bars: standard deviations. (C) Percentages of <br /> 1 splicing regulated developmentally (Figure S5). DNA sequence            tos-1 isoforms spliced cryptic 39 splice site tos-1 intron 1 <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                              10                            July 2012 | Volume 8 | Issue 7 | e1002827 <br />                                                                                                                                           MFAP-1 Affects Alternative Splicing <br />  <br />  <br /> compared isoforms spliced endogenous 39 splice                          Acknowledgments <br /> site cryptic 39 splice site. For analyses, isoform intensities <br /> were obtained analyzing biological duplicates triplicates                              We thank members Horvitz laboratory Ma laboratory, <br />                                                                                              especially <span id='am-82' about='obo:NCBITaxon_48819' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_214597 obo:NCBITaxon_131567 obo:NCBITaxon_214607 obo:NCBITaxon_3398 obo:NCBITaxon_4143 obo:NCBITaxon_1477611 obo:NCBITaxon_26122 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-83' property="rdfs:label" content="Anna" datatype="xsd:string"></span><span id='am-84'  ></span><span id='am-85' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Anna</span> Corrionero, Nicolas Paquin, Takashi Hirose, Shuo Luo, <br /> using NIH ImageJ software. N.S., significant difference. <br />                                                                                              Dengke Ma, Zhaofa Xu, Ben Wu, suggestions. We thank Nicolas <br /> (TIF)                                                                                        Paquin help yeast two-hybrid screen. We thank Na An and <br /> Figure S4 Reducing expression mfap-1 D1054.14                                   Yaoyu Zhong strain maintenance <span id='am-1' about='obo:NCBITaxon_337744' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_186628 obo:NCBITaxon_1 obo:NCBITaxon_186626 obo:NCBITaxon_186625 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_31013 obo:NCBITaxon_7995 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489793 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Rita" datatype="xsd:string"></span>Rita</span> Droste Fei Wang for <br /> RNA interference altered splicing tos-1. RT-PCR experi-                               technical support. We thank Andy Fire providing pPD plasmids we <br />                                                                                              used constructing transgene constructs. We thank Shohei Mitani of <br /> ments showing effects reducing expression mfap-1, <br />                                                                                              Tokyo Women&#8217;s Medical University providing tm3456D mutant. <br /> D1054.14 K04G7.11 RNAi feeding tos-1 alternative                                    Some genomic sequences polymorphisms identified the <br /> splicing. mfap-1(RNAi) D1054.14(RNAi) caused similar alter-                              WUSTL Genome Center. Some C. elegans strains provided the <br /> ations tos-1 splicing, K04G7.11(RNAi) did obviously                             Caenorhabditis Genetics Center. H. R. Horvitz David H. Koch <br /> affect splicing tos-1. RNAi bacterial strains obtained                           Professor Biology MIT Investigator Howard Hughes <br /> from ORFeome-based RNAi library [48].                                                     Medical Institute. <br /> (TIF) <br /> Figure S5 The splicing tos-1 regulated developmentally.                            Author Contributions <br /> RT-PCR experiments examining splicing tos-1 animals                             Conceived designed experiments: LM HRH. Performed the <br /> different synchronized developmental stages. tos-1 splicing                              experiments: LM XG JL LH YT. Analyzed data: LM XG HRH. <br /> similar developmental stages examined. YA: young adult                                Wrote paper: LM HRH. <br /> animals 24 hours L4 larval stage. <br /> (TIF) <br /> Table S1 Candidate MFAP-1 interactors isolated yeast <br /> two-hybrid screen. <br /> (DOC) <br />  <br /> References <br />  1. Nilsen TW, Graveley BR (2010) Expansion eukaryotic proteome                    21. Greenwald I, Horvitz HR (1986) A visible allele muscle gene sup-10 X of <br />     alternative splicing. Nature 463: 457&#8211;463.                                                   C. elegans. Genetics 113: 63&#8211;72. <br />  2. Maniatis T, Tasic B (2002) Alternative pre-mRNA splicing proteome                    22. Greenwald IS, Horvitz HR (1980) unc-93(e1500): A behavioral mutant of <br />     expansion metazoans. Nature 418: 236&#8211;243.                                                 Caenorhabditis elegans defines gene wild-type null phenotype. Genetics <br />  3. Jurica MS, Moore MJ (2003) Pre-mRNA splicing: awash sea proteins.                    96: 147&#8211;164. <br />     Mol Cell 12: 5&#8211;14.                                                                       23. Levin JZ, Horvitz HR (1992) The Caenorhabditis elegans unc-93 gene encodes a <br />  4. Abovich N, Rosbash M (1997) Cross-intron bridging interactions yeast                  putative transmembrane protein regulates muscle contraction. J Cell Biol <br />     commitment complex conserved mammals. Cell 89: 403&#8211;412.                               117: 143&#8211;155. <br />  5. Arning S, Gruter P, Bilbe G, Kramer A (1996) Mammalian splicing factor SF1            24. Brinkmann MM, Spooner E, Hoebe K, Beutler B, Ploegh HL, et al. (2007) The <br />     encoded variant cDNAs binds RNA. RNA 2: 794&#8211;810.                                   interaction ER membrane protein UNC93B TLR3, 7, 9 is <br />  6. Krainer AR, Maniatis T (1985) Multiple factors including small nuclear                   crucial TLR signaling. J Cell Biol 177: 265&#8211;275. <br />     ribonucleoproteins U1 U2 necessary pre-mRNA splicing vitro. Cell          25. Casrouge A, Zhang SY, Eidenschenk C, Jouanguy E, Puel A, et al. (2006) <br />     42: 725&#8211;736.                                                                                 Herpes simplex virus encephalitis human UNC-93B deficiency. Science 314: <br />  7. Madhani HD, Guthrie C (1994) Dynamic RNA-RNA interactions                             308&#8211;312. <br />     spliceosome. Annu Rev Genet 28: 1&#8211;26.                                                    26. Kim YM, Brinkmann MM, Paquet ME, Ploegh HL (2008) UNC93B1 delivers <br />  8. Merendino L, Guth S, Bilbao D, Martinez C, Valcarcel J (1999) Inhibition                  nucleotide-sensing toll-like receptors endolysosomes. Nature 452: 234&#8211;238. <br />     msl-2 splicing Sex-lethal reveals interaction U2AF35 39               27. Koehn J, Huesken D, Jaritz M, Rot A, Zurini M, et al. (2007) Assessing the <br />     splice site AG. Nature 402: 838&#8211;841.                                                         function human UNC-93B Toll-like receptor signaling major <br />  9. Wu S, Romfo CM, Nilsen TW, Green MR (1999) Functional recognition                     histocompatibility complex II response. Hum Immunol 68: 871&#8211;878. <br />     39 splice site AG splicing factor U2AF35. Nature 402: 832&#8211;835.                    28. Tabeta K, Hoebe K, Janssen EM, Du X, Georgel P, et al. (2006) The Unc93b1 <br /> 10. Zamore PD, Green MR (1991) Biochemical characterization U2 snRNP                          mutation 3d disrupts exogenous antigen presentation signaling Toll-like <br />     auxiliary factor: essential pre-mRNA splicing factor novel intranuclear            receptors 3, 7 9. Nat Immunol 7: 156&#8211;164. <br />     distribution. EMBO J 10: 207&#8211;214.                                                        29. Ma L, Horvitz HR (2009) Mutations Caenorhabditis elegans U2AF large <br /> 11. Zorio DA, Blumenthal T (1999) Both subunits U2AF recognize 39 splice                  subunit UAF-1 alter choice 39 splice site vivo. PLoS Genet 5: <br />     site Caenorhabditis elegans. Nature 402: 835&#8211;838.                                         e1000708. doi:10.1371/journal.pgen.1000708 <br /> 12. Reed R (2000) Mechanisms fidelity pre-mRNA splicing. Curr Opin Cell                30. Ma L, Tan Z, Teng Y, Hoersch S, Horvitz HR (2011) In vivo effects intron <br />     Biol 12: 340&#8211;345.                                                                            retention exon skipping U2AF large subunit SF1/BBP the <br /> 13. Cooper TA, Wan L, Dreyfuss G (2009) RNA disease. Cell 136: 777&#8211;793.                      nematode Caenorhabditis elegans. RNA 17: 2201&#8211;2211. <br /> 14. Wang GS, Cooper TA (2007) Splicing disease: disruption splicing code           31. Levin JZ, Horvitz HR (1993) Three new classes mutations Caenorhabditis <br />     decoding machinery. Nat Rev Genet 8: 749&#8211;761.                                        elegans muscle gene sup-9. Genetics 135: 53&#8211;70. <br /> 15. Guth S, Valcarcel J (2000) Kinetic role mammalian SF1/BBP spliceosome             32. Horrigan SK, Rich CB, Streeten BW, Li ZY, Foster JA (1992) Characterization <br />     assembly function polypyrimidine tract recognition U2AF. J Biol                 associated microfibril protein recombinant DNA techniques. J Biol <br />     Chem 275: 38059&#8211;38066.                                                                       Chem 267: 10087&#8211;10095. <br /> 16. Rutz B, Seraphin B (1999) Transient interaction BBP/ScSF1 Mud2               33. Andersen DS, Tapon N (2008) Drosophila MFAP1 required pre-mRNA <br />     splicing machinery affects kinetics spliceosome assembly. RNA 5: 819&#8211;             processing G2/M progression. J Biol Chem 283: 31256&#8211;31267. <br />     831.                                                                                     34. Vincent K, Wang Q, Jay S, Hobbs K, Rymond BC (2003) Genetic interactions <br /> 17. Tanackovic G, Kramer A (2005) Human splicing factor SF3a, SF1,                    CLF1 identify additional pre-mRNA splicing factors link between <br />     essential pre-mRNA splicing vivo. Mol Biol Cell 16: 1366&#8211;1377.                        activators yeast vesicular transport splicing. Genetics 164: 895&#8211;907. <br /> 18. Pleiss JA, Whitworth GB, Bergkessel M, Guthrie C (2007) Transcript specificity           35. Blanton S, Srinivasan A, Rymond BC (1992) PRP38 encodes yeast protein <br />     yeast pre-mRNA splicing revealed mutations core spliceosomal                        required pre-mRNA splicing maintenance stable U6 small nuclear <br />     components. PLoS Biol 5: e90. doi:10.1371/journal.pbio.0050090                               RNA levels. Mol Cell Biol 12: 3939&#8211;3947. <br /> 19. Blanchette M, Labourier E, Green RE, Brenner SE, Rio DC (2004) Genome-                   36. Makarov EM, Makarova OV, Urlaub H, Gentzel M, Will CL, et al. (2002) <br />     wide analysis reveals unexpected function Drosophila splicing factor              Small nuclear ribonucleoprotein remodeling catalytic activation the <br />     U2AF50 nuclear export intronless mRNAs. Mol Cell 14: 775&#8211;786.                      spliceosome. Science 298: 2205&#8211;2208. <br /> 20. la Cruz IP, Levin JZ, Cummins C, Anderson P, Horvitz HR (2003) sup-9, sup-            37. Rappsilber J, Ryder U, Lamond AI, Mann M (2002) Large-scale proteomic <br />     10, unc-93 encode components two-pore K+ channel                           analysis human spliceosome. Genome Res 12: 1231&#8211;1245. <br />     coordinates muscle contraction Caenorhabditis elegans. J Neurosci 23: 9133&#8211;           38. Zhou Z, Licklider LJ, Gygi SP, Reed R (2002) Comprehensive proteomic <br />     9145.                                                                                        analysis human spliceosome. Nature 419: 182&#8211;185. <br />  <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                            11                                   July 2012 | Volume 8 | Issue 7 | e1002827 <br />                                                          <br /> </body></html>