<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                     Database, Vol. 2013, Article ID bat059, doi:10.1093/database/bat059 <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br />  <br />  <br />  <br /> Database update <br /> Version 6 consensus yeast metabolic <br /> network refines biochemical coverage <br /> and improves model performance <br /> Benjamin D. Heavner1, Kieran Smallbone2, Nathan D. Price1 Larry P. Walker3,* <br /> 1 <br />   Institute Systems Biology, 401 Terry Ave N., Seattle, WA 98109, USA, 2Manchester Centre Integrative Systems Biology, University of <br /> Manchester, Manchester, M1 7DN, UK 3Biomass Conversion Laboratory, Department Biological Environmental Engineering, <br /> Cornell University, Ithaca, NY 14853, USA <br />  <br /> *Corresponding author: Email: lpw1@cornell.edu <br />  <br /> Submitted 1 March 2013; Revised 14 May 2013; Accepted 12 July 2013 <br />  <br /> Citation details: Heavner,B.D., Smallbone,K., Price,N.D., et al. Version 6 consensus yeast metabolic network refines biochemical coverage and <br /> improves model performance. Database (2013) Vol. 2013: article ID bat059; doi:10.1093/database/bat059. <br />  <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br /> Updates maintain state-of-the art reconstruction yeast metabolic network essential reflect under- <br /> standing yeast metabolism functional organization, eliminate inaccuracies identified earlier iterations, to <br /> improve predictive accuracy continue expand novel subsystems extend comprehensiveness the <br /> model. Here, present version 6 consensus yeast metabolic network (Yeast 6) update community <br /> effort computationally reconstruct genome-scale metabolic network <span id='am-1' about='obo:NCBITaxon_559292' typeof='owl:Thing obo:NCBITaxon_4932 obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Saccharomyces cerevisiae S288c" datatype="xsd:string"></span>Saccharomyces cerevisiae S288c</span>. Yeast 6 <br /> comprises 1458 metabolites participating 1888 reactions, annotated 900 yeast genes encoding the <br /> catalyzing enzymes. Compared Yeast 5, Yeast 6 demonstrates improved sensitivity, specificity positive nega- <br /> tive predictive values predicting gene essentiality glucose-limited aerobic conditions analyzed flux balance <br /> analysis. Additionally, Yeast 6 improves accuracy predicting likelihood mutation cause auxotrophy. <br /> The network reconstruction available Systems Biology Markup Language (SBML) file enriched Minimium <br /> Information Requested Annotation Biochemical Models (MIRIAM)-compliant annotations. Small- macromol- <br /> ecules network referenced authoritative databases Uniprot ChEBI. Molecules reactions also <br /> annotated appropriate publications contain supporting evidence. Yeast 6 freely available http://yeast.sf.net/ <br /> as separate SBML files: model using SBML level 3 Flux Balance Constraint package, model compatible the <br /> MATLAB&#213; COBRA Toolbox backward compatibility reconstruction containing reactions is <br /> experimental evidence (without non-biological reactions necessary simulating growth). <br /> Database URL: http://yeast.sf.net/ <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br />  <br />  <br />  <br />                                                                                                                                                   network, Yeast 6. The differences Yeast 5 and <br /> Introduction                                                                                                                                      Yeast 6 described fully detailed the <br /> In 2007, community effort integrate previously pub-                                                                                          supplementary <span id='am-43' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-44' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-45'  ></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> attached publication. <span id='am-29' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-33'  ></span>This</span> <br /> lished genome-scale reconstructions yeast metabolic                                                                                        update maintains emphasis standards compliance, <br /> network (1, 2) produced &#8216;consensus&#8217; representation                                                                                           unambiguous metabolite naming computer-readable <br /> yeast metabolism (3), subsequently                                                                                                 annotations available structured document <br /> updated iterative collaborative curation mul-                                                                                          format. Additionally, developed MATLAB&#213; scripts <br /> tiple research groups (4, 5). Here, introduce version 6                                                                                        demonstrate approach comparing Yeast 5 and <br /> of consensus reconstruction yeast metabolic                                                                                            Yeast 6 using flux balance analysis (FBA) methods, <br />  <br /> ............................................................................................................................................................................................................................................................................................. <br /> &#223; The Author(s) 2013. Published Oxford University Press. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/ <br /> licenses/by/3.0/), permits unrestricted reuse, distribution, reproduction medium, provided original work properly <br /> cited.                                                                                                                       Page 1 5 <br />                                                                                                  (page number citation purposes) <br />  Database update                                                                                                                      Database, Vol. 2013, Article ID bat059, doi:10.1093/database/bat059 <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br />  <br /> leveraging COnstraint-Based Reconstruction                                                                                             compatible MATLAB&#213; COBRA toolbox (6) back- <br /> Analysis (COBRA) Toolbox (6). These scripts                                                                                              ward compatibility GENRE containing reactions <br /> included supplementary data.                                                                                                                   experimental evidence. <br />    To emphasize distinction established <br /> biochemistry included metabolic genome-scale net- <br /> work reconstruction (GENRE) (7) additional model- <br />                                                                                                                                                   Results <br /> ing assumptions required analysis simulation                                                                                        Overview network characteristics <br /> genome-scale model (GEM) (7), Yeast 6                                                                                                Yeast 6 resulted effort improve predictive <br /> available http://yeast.sf.net/ separate Systems                                                                                       accuracy Yeast 5 manual curation, particu- <br /> Biology Markup Language (SBML) files: GEM using                                                                                             lar focus removing information sup- <br /> SBML level 3 Flux Balance Constraint package, GEM                                                                                               ported published literature adding metabolic <br />                                                                                                                                                   pathway information recently discovered. <br />                                                                                                                                                   As result effort, Yeast 6 contains fewer metabolites <br /> Table 1. Summary statistics Yeast 5 Yeast 6 <br />                                                                                                                                                   reactions Yeast 5 (Table 1), accurate in <br /> Summary statistics                                                                    Yeast 5                             Yeast 6                 predictions gene essentiality (Table 2) auxotroph- <br />                                                                                                                                                   inducing mutations (Table 3). Ninety-seven 1868 re- <br /> Genes                                                                                    918                                900                   actions shared Yeast 5 Yeast 6 different <br /> Metabolites                                                                           1655                                1458                    constraints, reflecting refinements reaction reversibility <br /> Reactions                                                                             2110                                1888                    yeast metabolic network. <br /> Reactions PMID references                                                        37.6%                               40.4% <br />                                                                                                                                                   Novel features Yeast 6 <br /> Because additional quality curation, Yeast 6 fewer blocked <br />                                                                                                                                                   Metabolites, reactions genes differ Yeast 5 <br /> reactions greater percentage reactions annotated with <br /> literature evidence; contains fewer genes, metabolites                                                                                Yeast 6. Detailed lists common distinct <br /> reactions Yeast 5. Neither reconstruction includes open read-                                                                                metabolites, reactions, genes, constraints, auxotrophs and <br /> ing frames annotated &#8216;dubious&#8217; <span id='am-22' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-28'  ></span>Saccharomyces</span> Genome                                                                                     knockout predictions included supplementary data, <br /> Database (7).                                                                                                                                     code used generate comparisons. The <br />  <br />  <br />  <br />              Table 2. Comparing gene essentiality predictions Yeast 5 Yeast 6 <br />  <br />  <br />                  Knockout Simulation Results                                                              Yeast 5                    Yeast 6                                              Observed viable                          Observed inviable <br />                                                                                                                                                                                              mutants                                   mutants <br />                                                                                                            683                         672 <br />                                                                                                                                                                   Simulated viable <br />  <br />  <br />  <br />  <br />                  True Positive                                                                            (74%)                      (75%)                                           24                              13         34                                        2 <br />                                                                                                                                                                      mutants <br />  <br />  <br />  <br />  <br />                                                                                                            108                         135 <br />                                                                                                                                                                                                  659                                                 80 <br />                  True Negative                                                                            (12%)                      (15%) <br />                                                                                                            114                         82 <br />                  False Positive                                                                           (12%)                       (9%)                                                  True Positive                                False Positive <br />                                                                                                             13                         11 <br />                                                                                                                                                                   inviable mutants <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                       2                                                                                 27 <br />                  False Negative                                                                            (1%)                       (1%) <br />                                                                                                                                                                       Simulated <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                   11                                               108 <br />                  Sensitivitya                                                                              98.1%                      98.4% <br />                  Specificityb                                                                              48.7%                      62.2% <br />                                                                                                                                                                                            False Negative                                True Negative <br />                                                                            c <br />                  Positive Predictive Value                                                                 85.7%                      89.1%                                          Yeast 5            Yeast 6                   Both networks <br />                                                                                d <br />                  Negative Predictive Value                                                                 89.3%                      92.5% <br />                                                                                                e <br />                  Matthews Correlation Coefficient                                                            0.59                        0.70 <br />              a <br />                TP/(TP + FN) <br />              b <br />                TN/(TN + FP) <br />              c <br />               TP/(TP + FP) <br />              d <br />                TN/(TN + FN) <br />              e              TP TN FP FN <br />                p&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259;&#64259; <br />                                                                   &#64259; (8) <br />                    &#240;TP&#254;FP &#222;&#240;TP&#254;FN&#222;&#240;TN&#254;FP &#222;&#240;TN&#254;FN&#222; <br />              Yeast 6 fewer false-positive predictions true-negative predictions gene essentiality, leading improve- <br />              ments sensitivity, specificity, positive predictive value, negative predictive value Matthews correlation coefficient (see <br />              Discussion information use metric). <br />  <br />  <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br /> Page 2 5 <br />  Database, Vol. 2013, Article ID bat059, doi:10.1093/database/bat059                                                                                                                                                                Database update <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br />  <br /> Table 3. Comparing auxotroph mutant predictions Yeast 5                                                                                        biomass produced following deletion in- <br /> and Yeast 6 <br />                                                                                                                                                   essential genes), 27 new true negative predictions (i.e. FBA <br /> Simulation Results                                                                                 Yeast 5                Yeast 6                 predicts biomass produced essential <br />                                                                                                                                                   genes deleted), 1 new gene predicted induce auxo- <br /> Auxotroph-inducing genes                                                                                92                     93                 trophy following mutation 7 newly correct predictions <br /> Correct auxotrophy predictions                                                                          57                     64                 auxotrophy phenotypes. Yeast 6 introduces new pre- <br /> Mutant incorrectly predicted viable                                                               32                     26                 dictions differ laboratory observations, new <br />  minimal medium                                                                                                                                false-positives (growth predicted despite deletion an <br /> Mutant incorrectly predicted inviable                                                               3                       3               essential gene), new auxotrophic mutant predicted in- <br />  supplemented medium <br />                                                                                                                                                   correctly incapable growth supplemented media <br /> Yeast 6 genes deletion reported                                                                                     new mutants incorrectly predicted viable in <br /> to cause auxotrophy correct predictions                                                                                      minimal media despite reported auxtrophy. We note that <br /> auxotrophy Yeast 5.                                                                                                                          differences model prediction observation <br />                                                                                                                                                   arise regulatory constraints outside the <br />                                                                                                                                                   scope metabolic reconstruction, informative <br /> supplementary data include specific rationale reac-                                                                                      opportunities continued network curation. <br /> tion deletions additions, detailed descriptions <br /> of function gene removed added. In sum-                                                                                            Yeast 6 aerobic anaerobic growth <br /> mary, Yeast 5 203 metabolites Yeast 6,                                                                                        Unlike Yeast 5, Yeast 6 does include separate biomass <br /> whereas Yeast 6 introduces 6 new metabolites. The smaller                                                                                         definitions simulating aerobic anaerobic growth. <br /> number metabolites Yeast 6 arises removal                                                                                          Yeast 6 correctly predicts yeast grow min- <br /> of metabolites involved dead-end reactions anno-                                                                                         imal media strict anaerobic conditions [Saccharomyces <br /> tated modeling reaction (i.e. included Yeast 5                                                                                     cerevisiae requires supplementation unsaturated <br /> without biochemical genomic evidence). No dead-end                                                                                             fatty acids sterols (9, 10)]. Simulating anaerobic <br /> reactions annotated gene removed.                                                                                            growth Yeast 6 requires simulated medium be <br /> Yeast 5 242 reactions Yeast 6,                                                                                        supplemented sterols [i.e. exchange reactions for <br /> Yeast 6 20 reactions Yeast 5. As detailed                                                                                     episterol, ergosterol, fecosterol, lanosterol, zymosterol and <br /> in supplementary data, reactions removed Yeast 6 in-                                                                                           ergosta-5,7,22,24(28)-tetraen-3beta-ol nonzero <br /> clude incorrect cofactor specificity, invol-                                                                                     lower bounds]. <br /> ving protein modification, supported upon <br /> literature review dead-end modeling reactions. Yeast                                                                                          Data annotation standards <br /> 5 31 genes Yeast 6, Yeast 6                                                                                           The network reconstruction provided SBML (11) file <br /> 13 genes Yeast 5. Of 31 genes removed,                                                                                        enriched MIRIAM-compliant (12) annotations. Small- <br /> 21 encode proteins involved protein modification                                                                                            macromolecules referenced community-stand- <br /> Glycosylphosphatidylinositol (GPI)-anchor assembly (pro-                                                                                          ard databases Uniprot (13) ChEBI (14). <br /> cesses strictly metabolic), 4 annotated                                                                                          Molecules reactions annotated appropri- <br /> with &#8216;putative&#8217; function, 1 removed unclear                                                                                        ate publications contain supporting evidence. Thus, <br /> cofactor specificity, 2 annotated poorly characterized reac-                                                                                      network presented computational framework <br /> tions involved ergosterol biosynthesis, 1 encodes                                                                                          adheres community standards entirely trace- <br /> flippase, require finer level membrane                                                                                           able. To facilitate comparison reconstructions and <br /> compartmentalization currently exists model,                                                                                          models, Yeast 6 metabolite reaction identifiers con- <br /> and metabolic function remaining 2                                                                                             sistent Yeast 5 identifiers (e.g. reaction &#8216;r_0123&#8217; in <br /> well established.                                                                                                                                 Yeast 5 reaction &#8216;r_0123&#8217; Yeast 6, and <br />                                                                                                                                                   metabolite &#8216;s_0042&#8217; Yeast 5 metabolite as <br /> Essentiality auxotrophy predictions Yeast 6                                                                                              &#8216;s_0042&#8217; Yeast 6). <br /> These changes reconstructed metabolic network,                                                                                                Yeast 6 follows modeling conventions Yeast <br /> combined incorporation suggested changes                                                                                           5. We used SBML specification encoding reaction <br /> the biomass <span id='am-42' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> (8), rise altered FBA predic-                                                                                      metabolite annotation COBRA <br /> tions gene essentiality glucose-limited aerobic condi-                                                                                      Toolbox-specific convention using custom &#8216;Notes&#8217; <br /> tions altered predictions auxotrophy induced                                                                                            field. Our sign convention exchange reactions that <br /> through gene deletion. When compared list es-                                                                                           positive flux values represent compounds produced in <br /> sential genes compiled (this list included sup-                                                                                      FBA simulation, negative flux values represent com- <br /> plementary data testYeast.m file), Yeast 6 13 new true                                                                                        pounds consumed, include biomass specific <br /> positive predictions gene essentiality (i.e. FBA predicts                                                                                      <span id='am-34' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> model. <br />  <br />  <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br />                                                                                                                                                                                                                                                       Page 3 5 <br />  Database update                                                                                                                      Database, Vol. 2013, Article ID bat059, doi:10.1093/database/bat059 <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br />  <br /> Assessing metabolic models                                                                                                                           Additional limitations arise appropriately lim- <br /> Because gene picked random likely                                                                                              ited scope metabolic network reconstruction. <br /> inessential essential growth (15), overall accuracy                                                                                      Condition-dependent constraints arise various <br /> is good metric assessing model predictive ability                                                                                       regulatory mechanisms included Yeast 6. Thus, <br /> (16). This statistical issue previously recognized,                                                                                      pathways affected transcriptional regu- <br /> leading use &#8216;geometric mean accuracy&#8217;                                                                                          latory events glucose repression incor- <br /> metric evaluating metabolic network models (2).                                                                                               rectly predicted carry fluxes FBA (e.g. the <br /> Because geometric mean ignores positive predictive                                                                                            absence additional constraints, malate cycle <br /> value (also called precision) (17), report values                                                                                       mitochondrial malate dehydrogenase cyto- <br /> the contingency matrix (Table 2), summarize pre-                                                                                          plasmic malate dehydrogenase). Integration regulatory <br /> dictive ability model Matthews correlation                                                                                        metabolic networks remains area active re- <br /> coefficient (18), metric robust range                                                                                       search (21, 22). <br /> prevalence values incorporates positive negative                                                                                             Like metabolic network models, reactions in <br /> predictive values.                                                                                                                                Yeast 6 carry flux FBA simulation. Yeast 6 738 <br />    We include testYeast.m script supplementary                                                                                             blocked reactions (39%), similar portion blocked reac- <br /> data facilitate evaluation model. This script                                                                                          tions Yeast 5 (38%). Blocked reactions indicate know- <br /> compares model phenotype predictions lists ver-                                                                                        ledge limitations (such reactions leading dead-end <br /> ified yeast open reading frames, genes consider                                                                                           metabolites metabolic fate origin unknown, <br /> essential genes cause auxotrophy deletion.                                                                                          reactions involved unconnected portions metabol- <br /> We compiled lists Yeast Deletion Project (15)                                                                                      ism, form unconnected subgraphs network). <br /> and information Saccharomyces Genome                                                                                                  Like FBA predictions differ observation, the <br /> Database (19).                                                                                                                                    number blocked reactions affected condition- <br />                                                                                                                                                   specific constraints, particularly constraints reaction re- <br />                                                                                                                                                   versibility. We observed relaxing reversibility <br /> Discussion                                                                                                                                        reactions involving nucleotide cofactors reduces the <br />                                                                                                                                                   number blocked reactions. <br /> Yeast 6 current state-of-the-art reconstruction the <br />                                                                                                                                                      Unlike Yeast 5, prediction anaerobic ethanol pro- <br /> S. cerevisiae metabolic network. It eliminates inferred <br />                                                                                                                                                   duction FBA simulation requires manual restriction the <br /> reactions evidence, adds new reac- <br />                                                                                                                                                   reaction catalyzed ATP synthase. The requirement for <br /> tions based recent evidence results improved pre- <br />                                                                                                                                                   condition-dependent constraint arise the <br /> dictions experimental data. It maintains distinction <br />                                                                                                                                                   lack regulatory constraints described above, from <br /> between GENRE GEM, emphasizing traceable <br />                                                                                                                                                   need physicochemical capacity constraints allow- <br /> annotation included information, differentiates be- <br />                                                                                                                                                   able flux. Additionally, reflect other, presently <br /> tween established biochemistry hypotheses may <br />                                                                                                                                                   uncharacterized limitations. We note integrat- <br /> be generated automated techniques gap-filling <br />                                                                                                                                                   ing regulatory constraints, detailed reconstruction cofac- <br /> algorithms (20). It useful addition con- <br />                                                                                                                                                   <span id='am-37' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-38' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-41'  ></span>tor</span> proton balancing remains area active <br /> sensus resource large community researchers <br />                                                                                                                                                   research constraint-based modeling community <br /> who use yeast metabolic model guide experimental <br />                                                                                                                                                   (23); redox conditions clearly critical importance to <br /> and modeling efforts. <br />                                                                                                                                                   function ATP synthase. <br />  <br /> Limitations                                                                                                                                       An invitation participate community effort <br /> Improving reconstruction yeast metabolic net-                                                                                          reconstruct yeast metabolic network <br /> work remains ongoing project. In addition                                                                                               Computational reconstruction modeling yeast <br /> model predictions differ experimental observa-                                                                                          metabolism ongoing project, invite additional <br /> tions described (i.e. false-positive, false-negative                                                                                        community participation effort. Suggestions for <br /> and incorrect auxotroph predictions), remains sub-                                                                                          improving yeast consensus reconstruction derived <br /> stantial opportunity improve reconstruction                                                                                             models submitted network.reconstruc <br /> lipid metabolism. This point evident                                                                                             tion@manchester.ac.uk. Metabolites enzymes should <br /> fact unsaturated fatty acids currently                                                                                               unambiguously identified, using existing model data- <br /> required simulating anaerobic growth,                                                                                                base (ChEBI UniProt) identifiers. New reactions should <br /> arises continued use generic lipid species                                                                                         supplied primary evidence mechanism <br /> [i.e. compounds using generic residual (-R) groups,                                                                                        catalysis, PubMed identifiers. Reactions without <br /> than precise stoichiometrically balanced definitions                                                                                           evidence clear reasons proposed <br /> fatty acid moieties].                                                                                                                             addition. <br />  <br />  <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br /> Page 4 5 <br />  Database, Vol. 2013, Article ID bat059, doi:10.1093/database/bat059                                                                                                                                                                Database update <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br />  <br />                                                                                                                                                      6. Schellenberger,J., Que,R., Fleming,R.M.T. et al. (2011) Quantitative <br /> Supplementary data                                                                                                                                      prediction cellular metabolism constraint-based models: the <br />                                                                                                                                                         COBRA Toolbox v2.0. Nat. Protoc., 6, 1290&#8211;1307. <br /> Supplementary data available Database Online. <br />                                                                                                                                                      7. Price,N.D., Reed,J.L. Palsson,B.O. (2004) Genome-scale models <br />                                                                                                                                                         microbial cells: evaluating consequences constraints. <br />                                                                                                                                                         Nat. Rev. Microbiol., 2, 886&#8211;897. <br /> Acknowledgments                                                                                                                                      8. Zomorrodi,A.R. Maranas,C.D. (2010) Improving iMM 904 S. <br /> The authors gratefully acknowledge Hnin Aung Kalliopi                                                                                               cerevisiae metabolic model using essentiality synthetic lethality <br />                                                                                                                                                         data. BMC Syst. Biol., 4, 178. <br /> Trachana, Julie Bletz, James Eddy, John Earls and <br />                                                                                                                                                      9. Andreasen,A.A. Stier,T.J. (1954) Anaerobic nutrition of <br /> other members Price group constructive <br />                                                                                                                                                         <span id='am-5' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>. II. Unsaturated fatty acid requirement <br /> comments manuscript stimulating discussion                                                                                                growth defined medium. J. Cell. Physiol., 43, 271&#8211;281. <br /> metrics evaluating model predictive accuracy.                                                                                                  10. Andreasen,A.A. Stier,T.J.B. (1953) Anaerobic nutrition of <br />                                                                                                                                                        Saccharomyces cerevisiae. I. Ergosterol requirement growth in <br />                                                                                                                                                        defined medium. J. Cell. Physiol., 41, 23&#8211;36. <br /> Funding                                                                                                                                            11. Hucka,M., Finney,A., Sauro,H.M. et al. (2003) The systems biology <br />                                                                                                                                                        markup language (SBML): medium representation ex- <br /> Support provided work Cornell Biofuels                                                                                              change biochemical network models. Bioinformatics, 19, <br /> Research Laboratory U. S. Department                                                                                                             524&#8211;531. <br /> Transportation, Federal Grant (#DTOS59-07-G-00052),                                                                                            12. Nove`re,N.L., Finney,A., Hucka,M. et al. (2005) Minimum informa- <br /> the U.S. Department Agriculture, Award (#2010-38502-                                                                                                tion requested annotation biochemical models <br />                                                                                                                                                        (MIRIAM). Nat. Biotechnol., 23, 1509&#8211;1515. <br /> 21900). Support provided work Institute <br />                                                                                                                                                    13. The UniProt Consortium. (2012) Update activities the <br /> for Systems Biology Center Systems Biology/2P50 <br />                                                                                                                                                        Universal Protein Resource (UniProt) 2013. Nucleic Acids Res., <br /> GM076547, Grand Duchy Luxembourg Systems                                                                                                        41, D43&#8211;D47. <br /> Medicine Consortium, Camille-Dreyfus Teacher                                                                                               14. Hastings,J., De Matos,P., Dekker,A. et al. (2012) The ChEBI reference <br /> Scholar Program (NDP) (to B.D.H.). Support provided                                                                                                database ontology biologically relevant chemistry: enhance- <br /> by EU FP7 (KBBE) grant (289434) &#8216;BioPreDyn: New                                                                                                    ments 2013. Nucleic Acids Res., 41, D456&#8211;D463. <br /> Bioinformatics Methods Tools Data-Driven                                                                                                   15. Winzeler,E.A., Shoemaker,D.D., Astromoff,A. et al. (1999) <br /> Predictive Dynamic Modelling Biotechnological                                                                                                       Functional characterization S. cerevisiae genome gene <br />                                                                                                                                                        deletion parallel analysis. Science, 285, 901&#8211;906. <br /> Applications&#8217; (to K.S.). Funding open access charge: <br />                                                                                                                                                    16. Alberg,A.J., Park,J.W., Hager,B.W. et al. (2004) The use &#8216;&#8216;overall <br /> U. S. Department Transportation, Federal Grant <br />                                                                                                                                                        accuracy&#8217;&#8217; evaluate validity screening diagnostic tests. <br /> (#DTOS59-07-G-00052), U.S. Department                                                                                                       J. Gen. Intern. Med., 19, 460&#8211;465. <br /> Agriculture, Award (#2010-38502-21900)                                                                                                             17. Kubat,M., Holte,R.C. Matwin,S. (1998) Machine learning for <br />                                                                                                                                                        detection oil spills satellite radar images. Mach. Learn., <br /> Conflict interest. None declared.                                                                                                                   30, 195&#8211;215. <br />                                                                                                                                                    18. Baldi,P., Brunak,S., Chauvin,Y. et al. (2000) Assessing accuracy of <br />                                                                                                                                                        prediction algorithms classification: overview. Bioinformatics, <br />  <br /> References                                                                                                                                             16, 412&#8211;424. <br />                                                                                                                                                    19. Cherry,J.M., Hong,E.L., Amundsen,C. et al. (2012) Saccharomyces <br />   1. Mo,M.L., Palsson,B.&#216;. Herrga&#730;rd,M.J. (2009) Connecting extra-                                                                                 Genome Database: genomics resource budding yeast. <br />      cellular metabolomic measurements intracellular flux states                                                                                 Nucleic Acids Res., 40, D700&#8211;D705. <br />      yeast. BMC Syst. Biol., 3, 37.                                                                                                                20. Brooks,J.P., Burns,W.P., Fong,S.S. et al. (2012) Gap detection for <br />   2. Kuepfer,L., Sauer,U. Blank,L.M. (2005) Metabolic functions                                                                                 genome-scale constraint-based models. Adv. Bioinformatics, 2012, <br />      duplicate genes Saccharomyces cerevisiae. Genome Res., 15,                                                                                     1&#8211;10. <br />      1421&#8211;1430.                                                                                                                                    21. Blazier,A.S. Papin,J.A. (2012) Integration expression data in <br />   3. Herrgard,M.J., Swainston,N., Dobson,P. et a <br /> </body></html>