<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />           The Fission Yeast dis3+ Gene Encodes a 11O-kDa Essential <br />                      Protein Implicated in Mitotic Control <br />  <br />               The fission yeast mutant dis3-54 is defective in mitosis and fails in chromosome disjunction. Its phenotype is <br />            similar to that of dis2-11, a mutant with a mutation in the type 1 protein phosphatase gene. We cloned the dis3+ <br />            gene by transformation. Nucleotide sequencing predicts a coding region of 970 amino acids interrupted by a <br />            164-bp intron at the 65th <span id='am-60' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63'  ></span><span id='am-64' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span>. The predicted dis3+ protein shares a weak but significant similarity with the <br />            budding yeast SSDI or SRKI gene product, the gene for which is a suppressor for the absence of a protein <br />            phosphatase SIT4 gene or the BCYI regulatory subunit of cyclic AMP-dependent protein kinase. Anti-dis3 <br />            antibodies recognized the 110-kDa dis3+ gene product, which is part of a 250- to 350-kDa oligomer and is <br />            enriched in the nucleus. The cellular localization of the dis3+ protein is reminiscent of that of the dis2+ protein, <br />            but these two proteins do not form a complex. A type 1 protein phosphatase activity in the dis3-54 mutant <br />            extracts is apparently not affected. The dis3+ gene is essential for growth; gene disruptant cells do not <br />            germinate and fail in cell division. Increased dis3+ gene dosage reverses the Ts&apos; phenotype of a cdc25 wee) <br />            strain, as does increased type 1 protein phosphatase gene dosage. Double mutant dis3 dis2 is lethal even at the <br />            permissive temperature, suggesting that the dis2+ and dis3+ genes may be functionally overlapped. The role <br />            of the dis3+ gene product in mitosis is unknown, but <span id='am-55' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57'  ></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> gene product may be directly or indirectly involved <br />            in the regulation of mitosis. <br />  <br />   Protein phosphorylation and dephosphorylation           are   im-          inant (19, 24). There are also two PP2A genes (ppal + and <br /> portant posttranslational modifications essential for a variety              ppa2+) in fission yeast. Again, single-gene disruptants were <br /> of biological systems. In mitotic cell cycle control, cdc2+                  viable but the double-gene disruptant was not. The semi- <br /> protein kinase plays a pivotal role as a component of                        dominant cs PP2A mutant was found to display a phenotype <br /> maturation/M-phase promoting factor (21); its phosphoryla-                   quite distinct from that of the PP1 mutant (19). <br /> tion state changes during the cell cycle, and its activity is                   Two other classes of dis mutants (disl and dis3) which <br /> regulated by dephosphorylation (9). In contrast to the iden-                 show a defective phenotype similar to that of the dis2-11 <br /> tification of an ever-increasing number of protein kinases                   mutant are known (23). All of them were identified as cs <br /> (15), protein dephosphorylation seems to be carried out by a                 mutants in which, at the restrictive temperature (20&#176;C), <br /> limited number of protein phosphatases (PPases) (6); in                      mitotic chromosome disjunction is blocked. These cs mu- <br /> mammals, there are generally four serine- and threonine-                     tants enter mitosis at 20&#176;C with normal timing but lose <br /> specific PPases, that is, types 1 (PP1), 2A (PP2A), 2B                       viability during mitosis. There is extensive phenotypic sim- <br /> (PP2B), and 2C (PP2C). An apparent paradox is how PPases                     ilarity among the different dis mutants. All of the mutants are <br /> can counteract the diversity of protein kinases. One possi-                  caffeine hypersensitive. They lose artificially constructed <br /> bility is that there may be many more PPases awaiting                        minichromosomes at high rates. Despite these similarities, <br /> discovery. Alternatively, the diversity of PPases may be                     however, genetic crossing indicates that they represent three <br /> based on the presence of specific modulators, inhibitors, and                different genetic loci. cs dis2-11 cells indeed lack PP1 en- <br /> activators (16, 25, 30, 35).                                                 zyme activity at 20&#176;C (19). However, no attempt has been <br />    The essential roles of PPases in the mitotic cell division                made to assay PPase activity in disi and dis3 mutant <br /> cycle have only recently been established. Mutations in the                  extracts. <br /> PP1 catalytic subunit were identified in the filamentous                        We addressed the question of whether the disl + and dis3+ <br /> fungus <span id='am-44' about='obo:NCBITaxon_5052' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5042 obo:NCBITaxon_33154 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Aspergillus" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasAlternativeId" content="NCBITaxon:37503" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-49'  ></span>Aspergillus</span> nidulans (BimG mutant [8]) and the                        genes are   functionally related to the dis2+ gene. Do they <br /> fission yeast <span id='am-30' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-31' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-34'  ></span>Schizosaccharomyces</span> pombe (dis2 mutant                         encode other phosphatases or modulators, inhibitors, acti- <br /> [24]). Failure in chromosome disjunction was the common                      vators, or substrates of phosphatase? Alternatively, is the <br /> phenotype of fission yeast, filamentous fungus, and fruit fly                dis phenotype similarity coincidental, not necessarily re- <br /> PP1 mutants (2, 8, 24). In S. pombe, there are two PP1                       flecting a functional similarity of the gene products? The fact <br /> catalytic subunit genes, namely, dis2+ and sds2l1. Results                   that a high dis2+ gene dosage extragenically complements <br /> of gene disruption experiments showed that the single-gene                   disl mutants (24) favors the possibility that the disl + gene <br /> disruptants (Adis2 and Asds2J) were viable whereas the                       product is implicated in protein dephosphorylation. How- <br /> double-gene disruptant (Adis2 Asds2J) was not, indicating                    ever, the nature of the dis1+ gene product is unknown. On <br /> that these two genes share an essential function (24). The                   the other hand, no functional link except the phenotypic <br /> cold-sensitive (cs) dis2-11 mutation appeared to be semidom-                 similarity is known between the dis3+ and dis2+ genes. In <br />                                                                              the present study, we have investigated the gene function of <br />                                                                              dis3+ and examined whether the dis3+ gene product is <br />   * <br />       Corresponding author.                                                  implicated in protein dephosphorylation. To this end, we <br />                                                                       5839 <br />  5840       KINOSHITA ET AL.                                                                                                                             MOL. CELL. BIOL. <br />  <br />        TABLE 1. Synthetic lethality of dis3 and dis2 mutants                         a <br />                                                                                                                                     A    IS3+ <br />           Crossa                 Colony <br />                                 formation              Phenotype <br />                                                       double     of <br />                                                              mutants <br />                                                                                            1kb <br />                                                                                       H            H <br />                                                                                                        ageB                   tix        Bg <br />                                                                                                                                                 Y .i8 <br />                                                                                                                                                    <span id='am-1' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-2' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>ia</span>       Bg <br />                                                                                                                                                                  +   pCDI1 12 <br /> dis3-54 x dis2-11                   -         Nonviable (no germination)                                                                                   -pCD 112-15 <br /> dis3-54 x disl-288                  -         Nonviable (two to three                                                                                      -pCDI 12-19 <br />                                                 divisions)                                                                                                 -PCD 125 <br /> dis3-54 x dis2::ura4+               -         Nonviable (germination but                                                                                ----pCD 126 <br />                                                 no division) <br />                                                                                                                                                                  -pCD 122 <br /> dis3-54 x sds2l::LEU2               cs        Viable (cs) <br />                                                                                                                                                                  + pCD131 <br />   a The genotypes of the strains used were as follows: dis3-54, h+ leul his2                                                                                     + pCD 1 74 <br /> dis3-54 or h- leul dis3-54; dis2-11, h- leul dis2-11; dis2::ura4+, h- leul ura4                                                                                  -pCD 163 <br /> dis2::ura4+; and sds2l::LEU2, h+ leul his2 sds2l::LEU2. The gene-disrupted <br /> strains dis2::ura4+ and sds2l::LEU2 (24) are viable; cell lethality is caused <br /> only when both the dis2 and the sds2l genes are disrupted (24). <br />                                                                                       b <br /> isolated the dis3+ gene, examined by gene disruption                                             dls3+                                   nda3 * <br />                                                                                                                                                 -..&gt;                  Ikb <br /> whether it is essential, and identified and characterized the                             I~~~~~~I~~~~~ <br />                                                                                       HH   Ea Ha                        H E     H EH                    H <br /> dis3+ gene product with anti-dis3 antibodies. We also inves-                           ..a <br />  <br />  <br /> tigated the dis3 mutant with regard to PPase activity, cell <br /> cycle arrest, and the relationship to cell cycle-controlling                                                  pCD1 12 <br /> genes.                                                                                                                                  pDB(NDA3) <br />                                                                                     FIG. 1. (a) Subcloning of the genomic DNA sequence that com- <br />                                                                                   plements the cs dis3 mutation (+ indicates complementation). H, <br />                MATERIALS AND METHODS                                              HindIII; Ba, BamHI; Bg, BglII; T, TthlllI; X, XhoI. The box <br />    Strains, media, and chemicals. Haploid S. pombe strains                        indicates the coding region. (b) Linkage between the dis3+ and <br /> were used. The genotypes of the strains used for gene                             nda3+ genes. <span id='am-93' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>Plasmids</span> pCD112 and pDB(NDA3) contain the over- <br /> disruption are indicated in Table 1. Rich (YPD), minimal                          lapping region. H, Hindlll; E, EcoRI. <br /> (EMM2), nitrogen source-deprived (EMM2-N), and sporu- <br /> lation (SPA) media were used (11). TEG buffer contained 50 <br /> mM Tris HCl (pH 7.5), 1 mM EDTA, 10% glycerol, 1 mM                               itor the cellular DNA contents. The procedures used were <br /> 2-mercaptoethanol, and 1 mM phenylmethylsulfonyl fluo-                            similar to those described previously (7). Cells (1 x 107 to 5 <br /> ride.                                                                             X 107) were collected, washed twice with 1 ml of distilled <br />    Transformation, integration, and gene disruption. The lith-                    water, and resuspended in 3 ml of distilled water. Ethanol (7 <br /> ium acetate method (17) was used for transformation of S.                         ml) was added with vigorous agitation, and the cells were <br /> pombe cells. Integration of cloned sequences into the chro-                       stored at 4&#176;C for more than 12 h. After one wash with 5 ml of <br /> mosome was performed by homologous recombination (27).                            50 mM sodium citrate (pH 7.0) and resuspension in 1 ml of <br /> For disruption of the dis3+ gene, plasmid pCD140 was                              the same buffer, RNase A (Sigma; preboiled for 15 min) was <br /> constructed, linearly cleaved, and used for transformation.                       added to a final concentration of 0.5 mg/ml. Following 2 h of <br /> Stable transformants were analyzed by tetrad analysis and                         incubation at 37&#176;C, propidium iodide (Sigma) was added <br /> Southern hybridization.                                                           (final concentration, 12.5 ,ug/ml) and the resulting stained <br />   Blotting and nucleotide sequence determination. Standard                        cell suspensions were analyzed. <br /> procedures for Southern and Northern (RNA) blotting (20)                             GenBank-EMBL accession number. The GenBank-EMBL <br /> were used. Immunoblotting was performed by electro-                               accession number for the sequence of the dis3+ gene is <br /> phoretically transferring proteins to nitrocellulose after so-                    M74094. <br /> dium dodecyl sulfate-polyacrylamide gel electrophoresis <br /> (SDS-PAGE) (34). Nucleotide sequences were determined                                                                   RESULTS <br /> by the dideoxy method with pUC plasmids (38). <br />   Construction of the fusion protein and preparation of anti-                       Isolation of the dis3+ gene by complementation. A fission <br /> serum. Plasmid pAR3039 (31) containing the T7 promoter                            yeast genomic DNA fragment (pCD112; Fig. la) that com- <br /> was used for expression of the dis3+ coding region in                             plemented cs dis3-54 was isolated. This fragment also com- <br /> bacterial strain BL21. The 2.2-kb BglII fragment containing                       plemented the caffeine hypersensitivity of the mutant. Sub- <br /> a part of the dis3+ gene was ligated to the plasmid, and the                      sequent subcloning indicated that a 3.3-kb fragment was the <br /> resulting plasmid, pCD141, caused the synthesis of a large                        minimal requirement for the complementation of cs dis3-54 <br /> quantity of a 70-kDa protein in bacterial cells induced by                        (pCD174; Fig. la). Multicopy plasmid pCD112 failed to <br /> isopropyl-p-D-thiogalactopyranoside. This 70-kDa protein                          complement other dis gene mutants, namely, disl-288 and <br /> was purified by previously described procedures (26), and                         dis2-11. Genomic Southern hybridization (50&#176;C, 2 x SSC [lx <br /> 200 ,ug of polypeptide was injected into rabbits every 2                          SSC is 0.15 M NaCl plus 0.015 M sodium citrate]) blots <br /> weeks for 10 weeks.                                                               probed with the dis3+ coding fragment in pCD174 showed <br />   Immunofluorescence microscopy and FACScan analysis.                             only an intense self-hybridizing band (<span id='am-88' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-92'  ></span>data</span> not shown), <br /> Previously described procedures for S. pombe immunofluo-                          suggesting that the dis3+ gene is unique in the genome. <br /> rescence microscopy were used (10, 18). The cell wall was                         Attempts to clone a budding yeast homolog by hybridization <br /> digested with 0.2 mg of Zymolyase per ml and partially                            were not successful. <br /> purified Novo Biolabs mutanase. The second antibody was a                           The 1.4-kb HindIII fragment in pCD112 was integrated <br /> rhodamine-conjugated goat anti-rabbit immunoglobulin G                            into the chromosome by homologous recombination with <br /> antibody. A Becton-Dickinson FACScan was used to mon-                             chromosome integration vector pYC20 (5). Tetrad analysis <br />  VOL . 1 l, 1991                                                                           FISSION YEAST dis3+ GENE            5841 <br />  <br /> showed that the integrated locus was tightly linked to the         phenotype. However, arrested cells proved difficult to dis- <br /> dis3 locus (parental ditype:nonparental ditype:tetratype, 44:      tinguish from plasmid-harboring dividing cells by fluores- <br /> 0:0). Therefore, the isolated sequence should be derived           cence microscopy with 4&apos;,6-diamidino-2-phenylindole <br /> from the dis3+ gene.                                               (DAPI) stain. We did not see cells with a mitotic arrest <br />    A previous genetic analysis (23) showed that the dis3 locus     phenotype reminiscent of that of the cs dis3-54 mutants (data <br /> is very close to the nda3 locus (map distance, -1 centimo-         not shown). The cell cycle phenotype of cs dis3-54 might be <br /> rgan). Indeed, we found that the sequence in pCD112                allele specific. <br /> overlapped that in pDB(NDA3), which carried the nda3+                 Identification of the dis3+ gene product. To identify the <br /> gene (14) (Fig. lb). The distance between dis3+ and nda3+ is       dis3+ gene product, we raised rabbit antiserum against the <br /> 5 kb, consistent with the genetic distance. Three-factor           dis3+ protein as described in Materials and Methods. The <br /> crosses showed that the dis3 locus is centromere distal            antiserum was affinity purified with the fusion polypeptide. <br /> relative to the nda3 locus (data not shown).                       S. pombe protein extracts were run in SDS-PAGE and <br />    The nucleotide sequence of dis3&apos; predicts a 110-kDa pro-        immunoblotted. A protein band with a molecular mass of 110 <br /> tein. The nucleotide sequence of pCD174 was determined by          kDa was obtained; this band was not seen with preimmune <br /> the dideoxy method (Fig. 2). It predicts a hypothetical dis3+      serum (data not shown). Introduction of the high-copy- <br /> polypeptide with a 164-bp putative intron (the consensus           number dis3+ plasmid (pCD174) into S. pombe wild-type <br /> sequences are underlined) at the 65th codon, a total of 970        cells failed to increase significantly the level of this 110-kDa <br /> amino acids, and a predicted molecular mass of 110 kDa (pl,        polypeptide (data not shown), so the experiment described <br /> 6.5). Northern hybridization blots of S. pombe poly(A)+            below was required to obtain definitive evidence to assign <br /> mRNA probed with the coding fragment showed a 2.8-kb               the 110-kDa polypeptide as the dis3+ gene product. <br /> band (data not shown), consistent with the size of the                Eight truncated dis3+ genes (172A to 172H) were made by <br /> predicted polypeptide.                                             a series of deletions from the NH2 or COOH termini (Fig. <br />    A computer search of the EMBL database did not reveal           5a). 172A lacks the first 74-amino-acid portion, whereas <br /> any homology to known protein sequences. However, as               172F terminates 147 residues before the wild-type termina- <br /> shown in Fig. 3, weak but significant homology to the              tion codon. These truncated genes were ligated with the <br /> <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-15'  ></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> SRKI gene (36) and SSDJ gene              alcohol dehydrogenase gene promoter sequence (28), and <br /> (32) was found. There are 23.5% identical and 37% similar          the resulting plasmids were introduced into cs dis3-54 mu- <br /> conserved amino acids in 523 amino acids. The highest              tants. Immunoblots of some of these strains (Fig. Sb) showed <br /> similarity is found in the COOH domains of the dis3+ and           polypeptide bands with the molecular masses expected for <br /> SRK1 or SSD1 proteins (Fig. 3); 22 residues are identical          these truncated genes. The predicted mass of the 172A gene <br /> (69% identity) in the 32-amino-acid stretch from positions         product was consistent with the presence of a putative <br /> 796 to 827 of the dis3+ protein. The SSDI gene can suppress        intron. (Polypeptides encoded by 172G and 172H and con- <br /> the lethality attributable to the deletion of an essential PPase   sisting of the NH2 domain of the dis3+ protein were not <br /> gene, SIT4 (1, 32). The SRKI gene (36) was identified as a         detected because the anti-dis3 antibodies were raised against <br /> suppressor of a mutation in BCYJ, the gene for the regula-         the COOH domain). These results proved that the bands <br /> tory subunit of cyclic AMP-dependent protein kinase (29, 33,       detected by anti-dis3 antibodies represented the dis3+ gene <br /> 37). The SRKI and SSDI genes were found to be identical.           product in S. pombe extracts. None of these truncated <br /> The SRK1 or SSD1 protein is 280 amino acids larger than the        genes, except for 172A, was able to complement cs dis3-54 <br /> dis3+ protein.                                                     mutants (although the colonies formed with 172A were <br />    To examine whether dis3+ and SRKI have identical                small). Hence, the NH2-terminal 74 amino acids appear not <br /> functions, we introduced a plasmid carrying the SRKI gene          to be essential. <br /> (pW31, a gift from K. Tatchell) into cs dis3 mutant cells by          The dis3+ gene product is enriched in the nucleus. Immu- <br /> transformation and tested it for complementation. However,         noblotting of salt-extracted fractions indicated that the dis3+ <br /> no Cs&apos; transformant was obtained. This result, together            protein was soluble in TEG buffer (see Materials and Meth- <br /> with the low sequence similarity and size difference, sug-         ods) containing 0 to 0.1 M NaCl. Most of the dis3+ protein <br /> gested that the dis3+ and SRKI or SSDI genes are not true          was found in the supernatant fraction of glass bead-broken <br /> homologs. Because the dis3+ gene seems to contain an               extracts after centrifugation at 50,000 x g for 20 min. We <br /> intron which is not spliced in budding yeast cells, transfor-      performed Percoll gradient centrifugation of cellular ho- <br /> mation of an srkl or an ssdl mutant by the dis3+ gene has not      mogenates; this process fractionates the nuclei from the <br /> been done.                                                         cytosol (11, 13). The dis3+ protein was present in both the <br />    The dis3+ gene is essential for growth. One-step gene           higher cytosol and the lower nuclear fractions (data not <br /> disruption was carried out as shown in Fig. 4. Tetrad              shown). <br /> analysis indicated that fewer than two spores were viable,            Immunofluorescence microscopy of growing wild-type <br /> and all the viable spores were Leu-. Therefore, a dis3             cells with affinity-purified anti-dis3 antibodies revealed in- <br /> disruption is lethal; dis3+ is an essential gene. Genomic          tense nuclear immunofluorescence (Fig. 6a). Additional <br /> Southern hybridization of heterozygous diploids (+IAdis3;          granular fluorescence was also found in the cytoplasm. The <br /> Fig. 4) and a wild-type diploid (+/+) produced hybridizing         dis3+ protein exists in both the nucleus and the cytoplasm <br /> bands of the expected sizes.                                       but is enriched in the nucleus. A comparison of DAPI- <br />    None of the dis3::LEU2-disrupted spores germinated on           stained DNA and anti-dis3 antibody immunofluorescence <br /> YPD plates, indicating that the dis3+ gene is essential for        showed that chromosomes were weakly stained, whereas <br /> germination. The dis3-disrupted heterozygous diploid cells         immunofluorescence in the nonchromosomal domain was <br /> were transformed by plasmid pCD131 and sporulated. The             intense (Fig. 6a). These results were clearly demonstrated in <br /> resulting plasmid-carrying spores were germinated, grown in        the dividing nuclei. The level of nuclear immunofluorescence <br /> minimal EMM2 medium, and transferred to complete me-               did not appear to be altered during the cell cycle. This <br /> dium at a restrictive temperature for 20 h. Cells which have       anti-dis3 antibody immunofluorescence pattern was similar <br /> lost the plasmid should stop dividing and reveal the defective     to that obtained with anti-dis2 antibodies (24). <br />  5842          KINOSHITA ET AL.                                                                                                                                       MOL. CELL. BIOL. <br />  <br />  <br />                                                                                                                           Tthlll I <br />                                                                                                                           GACCCAGTCTGTTCTCGGTGTGTGAAATATGAGAAGTTTTT -121 <br />         ATCCTGCGTTAAAATAATAACGTGTAGTGTTGTGTTGTGAATGCCGTAAGTGATAGTAGCTTTTCCAGGGTCTAAAACTTTTCTAGTAAAGCAATCTTCAGTGTTCTGAAGATTTACAGG                                                     -1 <br />                                  StuI <br />         ATGAGCACTGTTTCAGGATTGAAGAGGCCTCAAAGCTCCGAGAAGAATCACAGAGATAGAGTATTTGTTCGTGCAACCAGGGGGAAGGTTCAAMAAGTTGTTCGTGAGCAATATTTACGA                                                    120 <br />         M S T V S G L K R P Q S S e K N H R D R V F V R A T R G K V Q K V V R E Q Y L R                                                                                              40 <br />         AATGATATACCCTGTCAAAGTCGGGCTTGCCCTCTTTGCCGTTCTAAACTTCCAAAGGACTCGAGGGGAAATGJTAWTATTAAAGGAATCCCACATGAGTAATTTGCATTGGTCTGGAA                                                     240 <br />         N D        I       P C Q        S       R A C P L C R S K L P K D S R G N V                                                                                                  65 <br />         TTAGTTGGTATAATTTTCTGTGAAGTTAATTTCTGAGAGCTATTGAAACTAATTCATTT ACTATTCAATTTCA CAmmm-&quot;---I---&quot;--.-UTTAATGTGAAGCAAAATTTAATTAACZCZ=GAwaTGT 360 <br />                    164bp i ntron                                                                                                            L 66 <br />         TAGAGCCTATTTTATCAGAGAAGCCCATGTTCTTGGAGAAATTTGGGCATCATTATTTAATCCCTGATTCCAATATATTTTACCATTGTATTGATGCCTTAGAACATCCAAATAATTTTT 4840 <br />           E P I L S E K P O) F L E K F G H H Y L I P D S N I F Y H C I D A L E H P N N F F 106 <br />        TCGATGTCATTATTCTGCAAMCGGTTTTTTCAGAAATTTCTTCAAAATCTATTCCTCTATACAATAGAATGAAMCGCCTTTGCCAAGAGAAAACTAAAAGATTTACCCCTTTTAGTAATG                                                     600 <br />          D     V       I    I   L Q         T V F        S   E   I   S   S    K     S   I   P   L Y N   R N   K R L C Q E             K   T   K R F T P F S N E 146 <br />        AATTTTTTGTTGATACGTTCGTTGAAAGATTGGATGATGAATCTGCTAATGACCGTAATGATCGTGCAATTCGCAACGCGGCTAGTTGGTTTGCCAGTCATTTGGCCAGCTTAGGTATTA                                                     720 <br />           F    F V D            T F V E R                L D D       E S A N D R N D R A I R N A A S                              W   F A S        H   L A S     L G     I    K     186 <br />                                       NruI <br />        AGATTGTTTTATTAMCCGATGATCGCGAAAATGCTCGCTTGGCTGCTGAACAAGGTATTCAAGTCTCTACATTGAAAGACTACGTTCAGTACCTGCCCGACTCTGAAATTTTACTGGATA                                                     840 <br />          I     V       L    L   T D D            R   E   N A R       L A A E Q G                I   Q V S T L K D Y V Q Y L P D S E I L L D                                  (0     226 <br />  <br />        TGGTGTCGGCCATTGCGGACGCCATTGCTTCTAAAGAGCAGGTAGAGTCTGGAACAAAAMCGTATATGAACTACATTGGTCTATGTCTAGGCTGTTAGCTTGTATAAAAMATGGGGAAG 9960 <br />          V S A I A D A I A S                                 K E Q V E              S G T K N V Y E L H W                 S M S       R L     L A C        I   K N G     E V 266 <br />        TGCATAAAGGACTTATCAATATTAGTACTTATAATTATTTAGAAGGTTCTGTTGTTGTTCCTGGTTACAACAAGCCCGTGCTAGTTTCAGGAAGAGAAAATTTAAATCGTGCTGTGCAAG 1080 <br />          H K G L I N I S T Y N Y L E G S V V V P G Y N K P V L V S G R E N L N R A V Q G 306 <br />                                                                                                                                             PatI <br />        GTGATATCGTTTGCATTCAMTATTACCTCAAGATCAATGGAAAACGGAAGCTGAAGAGATAGCTGATGATGATGAAGACGTTGTTGTGAGTACTGCAGCAGAACCCGATTCGGCCAGAA 12100 <br />          D     I       V C      I   Q       I    L   P   Q D Q W K T E A E E                        I A D   D D   E   D V V V S           T A A K          P D   S   A   R I 346 <br />        TAAATGACTTGGAACTGATTACGAAAAGAAATGCTCATCCAACCGCTAAGGTGGTTGGTATTTTAAAAAGGAATTGGCGCCCATACGTTGGCCATGTTGATAATGCTACTATTGCCCAGT 13120 <br />          N D L E L I T K R N A H P T A K V V G I L K R N W R P Y V G H V D N A T I A Q S 386 <br />        CAAAAGGCGGTTCTCAGCAAACAGTTTTGCTAACACCTATGGATAGGCGAGTACCTAAGATTCGATTTCGACAAGGCAAGCGCCTAGACTTGTAGGTCGTCGTATTGTAGTAGCTATAG                                                    14140 <br />          K G G S Q Q T V L L T P                                     G <br />                                                 D R R V P K I R F R T R Q A P R L V G R R I V V A I D <br />                                                      Bg1II <br />                                                                                                                                                                                    426 <br />        ACTTATGGGATGCTTCTAGTCGTTATCCAGAAGGTCATTTTGTTCGAGATCTTGGAGAAATGGAAACCAAGGAAGCTGAAACTGAAGCTCTTCTTCTTGAATATGATGTTCAGCACCGTC                                                   15160 <br />          L W   D A SS fR Y P E G H F            V  R D L G E 0 E T K E A E T E A L L L K Y D V Q H R P                                                                             466 <br />        CCTTTCCCAAGGCTGTTCTTGATTGCCTTCCCGAGGAAGGTCATAATTGGAAGGTTCCTGCTGATAAAACACATCCACTTTGGAAAAATAGGAAAGATTTTAGAGATAAATTAATCTGTA 1680 <br />          F P           K A V L D C                   L   P   E   E   G   H    N W       K V P A D       K T H     P   L W     K N     R K D F R D K L I C S 506 <br />                                                                              M1UI <br />        G TATTGATC C TCCTGGTTGTCAGGATAT TGATGATGCATTGCACGCGTG TGTC CTTCC TAATGGTAATTAC GAAGTTGGAGTTCATATCGCCGATGTTAC TCACTTTGTAAAG CCAAATA 18100 <br />           I D P P G C Q D I D D A L H A C V L P N G N Y E V G V H I A D V T H F V K P N T 546 <br />        CCTCAATGGATAGCGAGGCAGCGTCCCGTGGAACGACAGTTTATCTGGTAGACAAGCGTATAGATATGTTGCCTATGTTGCTTGGCACAGATTTGTGCTCTCTACGACCATACGTTGAAC 11920 <br />          S 0 D S E A A S R G T T V Y L V D K R I D M L P                                                          N   L   L   G   T   D   L C      S   L   R P Y V E R 586 <br />        GTTTTGCCTTTTCATGCATTTGGGAAATGGATGAGAACGCCAATATAATTAAGGTTCATTTTACCAAATCTGTAATAGCTTCTAAAGAAGCATTTAGTTACGCAGATGCACAAGCTCGAA 2040 <br />          F A F S C I W E M D E N A N I I K V H F T K S V I A S K E A F S Y A D A 0 A R I 626 <br />  <br />        TTGATGATCAAAAAATGCAAGATCCGTTAACTCAAGGAATGCGTGTTTTGTTAAAGCTTTCAAAMATACTGAAACAAAAAGAATGGATGAAGGCGCTTTGAATCTGGCCAGCCCCGAAG 22160 <br />          D D Q             K M Q            D   P    L   T   Q G M R V L L K L S K                      I   L K Q     KfR     M D E G A L N L A S P E V 666 <br />        TTCGTATTCAGACTGATAACGAAACTTCAGATCCCATGGATGTTGAAATTAAGCAGCTTTTGGAAACCAATTCTTTGGTTGAAGAATTTATGTTGCTGGCTAATATATCTGTCGCACAAA 22280 <br />          R I Q T D N E T S D P M D V E I K Q L L K T N S L V E E F M L L A N I S V A Q K 706 <br />        AAATATACGATGCATTTCCGCAAACTGCGGTGCTTCGTCGACATGCTGCTCCACCATTAACAAACTTTGATTCGCTTCAAGATATTTTACGAGTATGTAAAGGCATGCACCTAAAATGTG 2400 <br />          I Y D A F P Q T A V L R R H A A P P L T N F D S L Q D I L R V C K G M H L K C D 746 <br />        ATACATCAAAATCTTTAGCTAAATCTTTGGACGAATGCGTGGATCCCAAAGAGCCATATTTTAATACTTATTACGAATTCTCACCACAAGATGCATGCTTTCAGCAGAATATTTTTGCT 22520 <br />          T S K S L A K S L D E C V D P K K                                                  P Y F N T L L R I <br />                                                                                        L T T R C M L S A E Y F C S 786 <br />                                                                                                                       NheI <br />        CTGGTACATTTGCACCACCCGACTTTCGGCACTATGGCTTAGCTTCTCCAATCTATACTCATTTTACATCCCCGATTCGACGATACGCTGATGTATTGGCTCATCGCCAGCTAGCAGCCG 22640 <br />          G T F A P P D F R H Y G L A S P I Y T H F T S P I R R Y A D V L A H R Q L A A A 826 <br />        CAATTGATTATGAAACGATAAATCCCAGTTTGTCTGACAAATCACGCTTAATC.GAAATTTGTAACGGTATTAACTATCGGCATCGCATGGCTCAAATGGCTGGACGAGCCAGTATAGAAT 22760 <br />          I D Y E T I N P S                               L S D K S R L I E                      I C N G I N Y R H R M A 0 M A G R A S                                I e Y 866 <br />        ATTATGTTG GACAAGCCTTGAAGGGTG G TG TTG CTGAAGAAGACGCTTATGTTATTAAAGTG TTTAAAAACGGATTTGTTGTATTTATC GC TCGTTTTGGTTTAGAAGGAATTGTATATA 2280 <br />          Y V G Q A L K G G V A E E D A Y V I K V F K N G F V V F I A R F G L k G I V Y T 906 <br />        CAAsAAGCTTATCCTCGGTTCTCGAACCCAATGTTGAGTATGTAGAGGACGAATACAGCTAAATATTGAAATTCGTGATCAACCTAAACCGCAAACCGTGCAAATACAAATGTTTCAGC 33000 <br />          K S L S S V L E P N V E Y V E D E Y K L N I e I R D Q P K P Q T V Q I Q M F Q Q 946 <br />        AAGTGCGTGTTCGTGTAACGACGGTACGTGATGAGCACTCTGGAAAACAAAAAGTCCAGATTACTTGGTGTATTAATTAATTGGAATGAAGGCATTTGAATAATTTTACTTTCCTTTG 3120 <br />          V R V R V T T V R D E H S G K Q K V Q I T L V Y                                                                       970 <br />        GGTAAAATAGTCCTGGGAATAAAATTTATTAGAAAATAAAATTTGTTGTATTGTGAATATTCTAAGCTT                                                                                                      3177 <br />   FIG. 2. Nucleotide sequence of the dis3+ gene and predicted amino acid sequence. The putative intron consensus sequences are <br /> underlined. The restriction sites used for construction of the truncated genes are indicated, and the circled Ms (methionines) represent the <br /> putative initiation codons of the truncated genes. <br />  VOL. 11, 1991                                                                                                                 FISSION YEAST dis3+ GENE               5843 <br />  <br />  <br />      dis3+ <br />                   I <br />  <br />                   I <br />                                  In              m <br />                                                     _             1970   a&amp;                       a <br />    SSD1I                                                                             1250aa <br />     SRK1I <br />     I dia3+ 289 NKPVLVSGRENLNRAVQDIVCIQILPQO <br />          SSD1/ 383 DADIYICGSKDRNRALEGDLVAVELLVVD <br />  <br />     II <br />           SRK1 <br />                   332 DVVVSTAAEPDSARINDLELITltRNA <br />                                                                                                                                     2CL. <br />                   456 NNFLSSPSSSDSLSKDDLSVRRKRSS <br />  <br />     III           390 SQQTVLLTPMDRRVPKIRFRTROAPRLVGRRIVVAIDL8DASSRYPEGHFVRDLGEMETK <br />                   587 TDKKVPLIAIPTELAPKDF-VENADKYSEKLFVASIKRVPITSLHPrGILVSKLGDIHDP                    4             5 C        30 -* s &gt; <br />                                                                                                     &lt; ~ ~~     ~    ~74 ~ ~ ~~~) )p&lt; <br />                   450 EAETEALLLEYDVQHRPFPKAVLDCLPEEGHNWKVPADKTHPLWMOKDFRDKLICSIDP <br />                   646 DTEIDSILRDNNFLSNEYLDQ--KNPQKEKPSFQPLPLTAESL-EYRRNFTDTEYNOIIFA <br />                   510 PGCQD-IDD-ALHACVLPNGNYEVGVHIADVTHFVKPNTS8DSEAASRGTTVYLVDKRID <br />                   703 ISELG98VSEFALHVRNNGNGTLELGCHWDVTSHIEEGSSVDRRARKRSSAVFPQKLVN <br />                   568 MLPMLLGTDLCSLRPYVERFAFSCIWE0MENANIIKVHF-TKSVIASKEAFSYADAQARI                b            Hindlil                  Clal              KpnI <br />                   763 LLPQSFNDEL-SLAPGKESATLSVVYTLDSSTLRIKSTWVGESTISPSNILSLEQLDEKL <br />                                                                                                                         C&apos;,                    C&apos;,)          C&apos;,cn <br />                   627 DDQKMODPLTQGK8RVLLKLSKILKQKRMDEGALNLASPEVRIQTDNETSDPMDVEIKQLL <br />                   822 ST----GSPTSYLSTVQEIARSFYARRINDPEATLLPTLSLLESLDDEKVKVDLNILDRT                                                                    + <br />  <br />                   687 ETNSLVEEFMLLANISVAOKIYDAFPQTAVLRRHAAPPLTNFDSLQDILRVCKGMH LKCD <br />                   878 LGFVVINEIKRKVNSTVAERKIYTKLGDLALLRRQMQPIATKM6SFRKKIQNF-GYNFDTN <br />                                                                                                            +            +           +      +          +      + <br />  <br />                  747 TSKSLAKSLDECVDPKEPY97TLLRILTTRCOLSAEYFCSGTFAPPDFRHYGLASPIYTH <br />                                                                                                                                       .._1 <br />                                                                                                                                     4.A <br />                                                                                                                    ..         ..h               - <br />  <br />  <br />  <br />                   937 TADELIKGV---LKIKDDDVRVGIZILLFKTMPRARYFIAGKVDPDQYGHYALN0LPIYTH <br />                   807 FTSPIRRYADVLAHRQLAAAIDYETINPSLSDKSRLIEICN---GINYRHR0AQ8AGRAS <br />                   994   FTAP9RRYADHWHRQLKAVIHDTPYTEDMEALKITSEYCNFKKDCAYQAEQ^AIHLLLC <br />                                                                                                     12- <br />                  864 IEYYVGQALKGGVAEEDAYVIKVFIKNGFVVFIARFGLEGIVYTKSLSSV        912 <br />                                                                                                     9.0- <br />                  1054 KT INDMGNTTGQLLT88AT-VLQVYKSSFDVFIPEFGIEKRVHGDQLPLI     1101 <br />                                                                                                     6.6- <br />   FIG. 3. Amino acid (<span id='am-82' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-85'  ></span><span id='am-86' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>aa</span>) sequence homology between the fission <br /> yeast dis3+ and the budding yeast SRKI or SSDI gene products.                                       4.4- <br /> Similar domains I, II, and III in the proteins are schematically                                     kb <br /> shown. Identical and conserved amino acids are indicated by double <br /> and single dots, respectively. <br />                                                                                                 FIG. 4. Disruption of the dis3 gene. (a) Construction of the <br />                                                                                               disrupted dis3::LEU2 gene. The 4.4-kb HindIll fragment was in- <br />    An immunofluorescence micrograph of nuc2 mutant cells                                      serted at the BglII site with the S. cerevisiae LEU2 gene. Linear <br /> revealing condensed chromosomes (11, 12) is shown in Fig.                                     DNA containing the disrupted dis3 gene (dis3::LEU2) was used for <br /> 6b. Punctate staining of chromosomes with anti-dis3 anti-                                     transformation of S. pombe leul- diploid cells (strain CM3/CM7; <br />                                                                                               h-lh+ +/his2 leullleul ura4/ura4 ade6-M2101ade6-M216), and the <br /> bodies was observed in addition to nonchromosomal stain-                                      resulting heterozygous diploids containing the integrated dis3:: <br /> ing.                                                                                          LEU2 were sporulated by nitrogen starvation. The expected lengths <br />    The dis3+ gene product exists in an oligomeric form. In gel                                of restriction fragments with or without the gene disruption are <br /> filtration of wild-type S. pombe extracts, the dis3+ polypep-                                 shown in kilobases. The 4.4-kb HindIII fragment was used as a <br /> tide eluted in the 250- to 350-kDa range (Fig. 7a, middle                                     probe. (b) Genomic Southern hybridization of the wild type (+/+) <br /> panel). This result indicates that the dis3+ protein does not                                 and two Leu+ integrants (+I/Adis3) probed with a 1.1-kb BglII- <br /> exist in a monomeric form in its native state. The elution                                    BamHI fragment. Genomic DNAs were digested with HindIII, ClaI, <br />                                                                                               or KpnI and run in an agarose gel. The sizes of fragments are <br /> characteristics of the dis3+ protein partly match those of the                                indicated in kilobases. <br /> dis2+ protein (Fig. 7a, lower panel), as determined by <br /> immunoblotting with anti-dis2 antibodies. PP1, assayed with <br /> 32P-labeled phosphorylase as a substrate in the presence of <br /> 10 nM okadaic acid, which inhibits PP2A but not PP1, eluted                                     Double mutants involving dis3, disi, and dis2 are nonviable. <br /> in the -80-kDa range (Fig. 7a, upper panel). Therefore, the                                   Because the phenotype of dis3-54 is similar to that of dis2-11 <br /> dis3+ protein is probably not a component of PP1, at least                                    and disl-288, we tried to construct double mutants by <br /> when phosphorylase is used as a substrate.                                                    crossing. If two recombined genes share a similar, indispens- <br />    Consistently, immunoprecipitates obtained with anti-dis2                                   able function, then a double mutation may cause cell death, <br /> antibodies (indicated by anti dis2 ppt in Fig. 7b) contained                                  even at the permissive temperature. Tetrads resulting from <br /> dis2+ protein, which was detected by immunoblotting with                                      the crosses listed in Table 1 were dissected. <br /> anti-dis2 antibodies (left side of lower panel), but did not                                    Colony-forming double mutants were not obtained from <br /> contain dis3+ protein (left side of upper panel). Similarly,                                  these crosses, except for dis3cs sds2Jnu&quot;, which could grow <br /> immunoprecipitates obtained with anti-dis3 antibodies (indi-                                  at the permissive temperature. Double-mutant spores de- <br /> cated by anti dis3 ppt in Fig. 7b) contained dis3+ protein                                    rived from the other three crosses, namely, dis3cs dis2cs, <br /> (upper panel) but not dis2+ protein (lower panel). Thus,                                      dis3cs dislcs, and dis3cs dis2nul, could not form colonies. <br /> complex formation between the dis2+ and dis3+ proteins is                                     Similarly, crossing of cs disi and cs dis2 did not yield a <br /> unlikely.                                                                                     viable double mutant. Thus, the synthetic lethality obtained <br />  5844           KINOSHITA ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />        a 3                                                                             a                                           b <br />         Tthl11 1         StulI       Nrul   Psti Bgl1ll Mlul       Nhel <br />     pCD              M <br />      174                                                                       + <br />  <br />  <br />        172A                 - <br />  <br />  <br />            B <br />            C <br />                                                     v <br />            C <br />            E <br />            F E-- <br />            G C--- <br />                    I.r <br />            HZ                        -__-- <br />                                                                                         FIG. 6. Localization of the dis3+ gene product by immunofluo- <br />                                                                                       rescence microscopy. S. pombe cells were fixed and prepared for <br />                                                                                       immunofluorescence microscopy with anti-dis3 antibodies. (a) Wild- <br />                                                                                       type cells. Left, DAPI stain for chromosomal DNA. Right, anti-dis3 <br />                                                                                       antibody stain. (b) ts nuc2 mutant cells containing mitotic condensed <br />                                                                                       chromosomes. Top, DAPI stain. Bottom, anti-dis3 antibody stain. <br />                                                                                       Bars, 2 jim. <br />                                  &gt;   172        A       B      C    E <br />                                                                           -   dis3o <br />                                                                                       Sepharose chromatography as described previously (19). <br />                66 -                                                                   Two peaks, A and B, were obtained for phosphorylase <br />                                                                                       phosphatase activity (Fig. 8). Peaks A and B correspond to <br />                45-                                                                    PP1 and PP2A activities, respectively (19). In dis2 and ppa2 <br />                kd                                                                     mutant extracts, these PP1 and PP2A peaks, respectively, <br />                                                                                       were greatly reduced (19). In dis3 mutant extracts prepared <br />    FIG. 5. Construction and expression of truncated dis3+ genes.                      at the restrictive temperature (Fig. 8a), PP1 activity was not <br /> (a) Plasmids containing different parts of the dis3+ gene were ligated                reduced. In dis3 mutant cells carrying a plasmid with the <br /> to the S. pombe ADH promoter (indicated by the box [28]) and                          dis3+ gene, a similar elution profile was obtained (Fig. 8b). <br /> transformed into cs dis3 mutant cells. M, putative initiation codons.                 Thus, although the phenotype of dis3 mutants is similar to <br /> Complementation is indicated on the right. The wild-type gene and,                    that of dis2 mutants, the levels of PP1 activity in dis3 mutant <br /> to a lesser degree, a truncated gene (172A) lacking 74 NH2-terminal                   extracts monitored as phosphorylase phosphatase were not <br /> residues could complement cs dis3. (b) Immunoblotting of wild-type                    affected. <br /> and transformant extracts with anti-dis3 antibodies. Polypeptides of                     The dis3 and dis2 mutants can transit G1/S. We addressed <br /> expected molecular masses were detected.                                              the question of whether the cs dis3-54 and dis2-11 mutants <br />                                                                                       can transit the G1/S boundary at a restrictive temperature <br />                                                                                       (20&#176;C). This point had not been established previously, as <br /> by combining these three dis mutations indicates functional                           the duration of the G1 phase is negligible in exponentially <br /> similarity among the dis+ genes. The viability of dis3cs                              growing fission yeast cells. To this end, cells growing at a <br /> sds2Jnull at the permissive temperature may be explained by                           permissive temperature (33&#176;C) were temporarily starved in <br /> the fact that the sds2J&apos; gene encodes a minor PP1 (19, 24).                           nitrogen-deficient medium (EMM2-N), in which cells enter a <br />    High-dosage expression of dis3+ reverses the weel cdc25                            G1-like phase (22); cells resume growth upon readdition of a <br /> phenotype. Other similarities between the dis2+ and dis3+                             nitrogen source. <br /> genes come from their interrelationship with the cell cycle                              dis3-54, dis2-11, and wild-type cells were grown in com- <br /> regulator genes. The bwsl+ gene (identical to dis2+) was                              plete medium EMM2 at 33&#176;C, transferred to nitrogen source- <br /> cloned by its ability to reverse the Ts&apos; phenotype of the                             deficient medium EMM2-N for 7 h, and returned to complete <br /> temperature-sensitive (ts) double mutant cdc25-22 wee1-50                             medium EMM2 again at either the restrictive or the permis- <br /> (4). cdc25 is a mitosis inducer, while weel+ is a mitosis                             sive temperature. Cellular DNA contents were analyzed <br /> inhibitor (21). To examine whether the high-dosage dis3+                              with a Becton-Dickinson FACScan (see Materials and Meth- <br /> gene has the same ability, we used a multicopy plasmid or a                           ods). All the cultures clearly showed a high accumulation of <br /> high-expression plasmid with the fission yeast ADH pro-                               cells with lc DNA content at 7 h after deprivation of the <br /> moter (28) and the dis3+ gene for transformation. The                                 nitrogen source (Fig. 9a). These cells appeared to move <br /> ADH-dis3+ gene could reverse the phenotype of cdc25 weel                              across the G1/S phases after readdition of the nitrogen <br /> (Table 2). The multicopy plasmid carrying the sds2l+ gene                             source at 20 as well as 33&#176;C. Therefore, dis3-54 and dis2-11 <br /> coding for a PP1 gene also could reverse the phenotype.                               cells are not defective in transit of the G1/S boundary. <br /> However, the multicopy plasmid carrying the ppa2+ gene (a                             Consistent with this <span id='am-50' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Idea" datatype="xsd:string"></span>idea</span>, cells with lc DNA content were <br /> PPA2 gene [19]) could not.                                                            not observed in a dis3-54 culture grown asynchronously at <br />    PPase activity in dis3 mutant extracts. PP1 activity in dis3                       the permissive temperature and transferred to nonpermis- <br /> mutant extracts was assayed with 32P-labeled phosphorylase                            sive conditions for 10 h (Fig. 9b). If dis3-54 cells were <br /> as the substrate. Cell extracts were fractionated by DEAE-                            blocked at two points, G1/S and G2/M, two peaks corre- <br />  VOL. 11, 1991                                                                                                         FISSION YEAST dis3+ GENE                5845 <br />  <br />  <br />    a-02   - <br />                                                                                                   a <br />     I                                                                                                                               B <br />                                                                                                  c <br />                                                                                                                       A <br />                        700               300     200           80kd                              0 <br />                            v              .         V          v                                 EQ. <br />                                                                                                 1- <br />  <br />  <br />  <br />  <br />                                                                              anti dls3 <br />                                                                                                  C) <br />                                                                                                  Cu <br />                                                                               anti dis2 <br />  <br />  <br />     b                          N <br />                                .4 <br />                                I <br />                                     4w <br />  <br />                                     R <br />                                                 M <br />                                                 .1 a <br />                                                 i <br />                                                     O.- <br />  <br />  <br />  <br />  <br />               1s3&apos; <br />                                                           anti dis3 wst                          0.2 <br />  <br />               cM2    2mp <br />  <br />                                                           anti dis2 wst <br />    FIG. 7. (a) Gel filtration chromatography of dis3+ and dis2+ <br /> proteins. Wild-type cell extracts were prepared and fractionated by <br /> gel filtration chromatography with Superose 6 (Pharmacia). Each <br /> fraction was assayed by immunoblotting with anti-dis3 and anti-dis2 <br /> antibodies. The dis3+ protein was eluted in the range of 250 to 350 <br /> kDa, and the dis2+ protein was eluted in the range of 80 to 200 kDa. <br /> A PP1 was assayed with 32P-labeled phosphorylase as the substrate <br /> in the presence of 10 nM okadaic acid. PP2A was inhibited by 10 nM <br /> okadaic acid. (b) Immunoblotting of immunoprecipitates by anti-                                FIG. 8. Assay of PPase activity in dis3 mutant extracts with <br /> dis3 and anti-dis2 antibodies. First, the wild-type cell extracts were                       32P-labeled phosphorylase as the substrate. The mutant extracts <br /> immunoprecipitated by anti-dis3 or anti-dis2 antibodies. Then, the                           were fractionated with DEAE-Sepharose, and two activity peaks, A <br /> immunoprecipitates were run in SDS-PAGE and immunoblotted by                                 and B (eluting at 0.17 and 0.24 M NaCl), corresponding to PP1 and <br /> anti-dis3 or anti-dis2 antibodies (indicated by anti dis3 wst and anti                       PP2A, respectively, were obtained (19). The extracts used were <br /> dis2 wst, respectively). The immunoprecipitates obtained with anti-                          obtained from dis3 mutant cells incubated at the restrictive temper- <br /> dis2 antibodies did not contain dis3+ protein, while those obtained                          ature (20&apos;C) for 10 h (a) and from dis3 mutant cells carrying plasmids <br /> with anti-dis3 antibodies did not contain dis2+ protein.                                     with the wild-type dis3+ gene and grown at 20&apos;C (b). <br />  <br />  <br /> sponding to different DNA contents should have been                                          lethal phenotype of cs dis2 and dis3 mutants is allele depen- <br /> present. On the basis of these results, we conclude that                                     dent. <br /> dis3-54 and dis2-11 mutants are blocked only in mitosis. <br /> These results, however, do not exclude the possibility that <br /> the dis2+ and dis3+ genes function in G1/S, because the                                                                DISCUSSION <br />                                                                                                 In this paper, we report the identification and characteri- <br /> TABLE 2. Reversal of the cdc25 weel double-mutant phenotype <br />                                                                                              zation of the fission yeast dis3+ gene product. The cs dis3-54 <br />         by the dis3+ gene with the ADH promoter&apos;                                             mutant appears to enter mitosis but fails to exit from it. The <br />                                                                                              phenotype is reminiscent of that of dis2-11, a mutation in the <br />   Plasmid                                      Gene                       Colony formation   catalytic subunit gene of PP1. Gene cloning by transforma- <br /> Multicopy                                 None                                  Ts&apos;          tion and nucleotide sequence determination demonstrated <br /> Multicopy                                 dis2+                                 Ts-          that the dis3+ gene encodes a 970-amino-acid protein. The <br /> ADH                                       dis3+                                 Ts-          coding region is interrupted at the 65th position with a 164-bp <br /> Multicopy                                 sds21                                 Ts-          putative intron. The gene truncation indicates that the NH2- <br /> Multicopy                                 ppa2+                                 Ts+          terminal 74 amino acids are dispensable but that the COOH- <br /> Multicopy                                 sds22+                                Ts&apos;          terminal 147 amino acids are essential for complementation <br /> Multicopy                                 sds23+                                Ts&apos;          of cs dis3-54. By an immunochemical method, the 110-kDa <br />   &apos; S. <br />        pombe double mutant cdc25 weel is able to grow, as the ts cdc25                       polypeptide was identified as the dis3+ gene product, which <br /> mutation is suppressed by wteel-SO. Multicopy plasmid pDB248&apos; (3) carrying                   existed as salt-extractable oligomers. Immunofluorescence <br /> one of the genes indicated was used for transformation of the double mutant,                 microscopy showed that the dis3+ protein is enriched in the <br /> and the growth property of transformants was examined by plating at 26 and                   nucleus. Judging from the intensity of the immunoblot band <br /> 36&#176;C. For the dis3+ gene, a plasmid containing the S. pombe ADH promoter                     for the adjusted amount of dis3 + protein made in <span id='am-75' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-77'  ></span><span id='am-78' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span>Escherichia</span> <br /> was constructed and ligated with the dis3+ gene. The ADH-dis3+ plasmid in <br /> addition to the multicopy dis2+ and sds21+ plasmids reversed the phenotype                   coli, the amount of dis3+ protein made in S. pombe cells is <br /> of the double mutant.                                                                        approximately 5 x 103 to 10 x 103 copies per cell, represent- <br />  <br />   FIG. 9. Transit of the G1/S phases at a restrictive temperature by dis3-54 and dis2-11 mutants. (a) Wild-type, dis3-54, and dis2-11 cells were <br /> grown at 33&#176;C in complete EMM2 medium, transferred to EMM2-N medium (lacking a nitrogen source) for 7 h at 33&#176;C, returned to complete <br /> EMM2 medium, and incubated for 12 h at 20 or 33&#176;C. Cells were stained with propidium iodide and treated with RNase, and their DNA <br /> contents were analyzed with a Becton-Dickinson FACScan. The two peaks present in the culture incubated in EMM2-N for 7 h represent cells <br /> with lc (G1) and 2c DNA contents. Cells with lc DNA content disappeared at 20 as well as 33&#176;C 12 h after the readdition of the nitrogen <br /> source, indicating that both wild-type and mutant cells can move across the GQ/S phases at the restrictive temperature (20&#176;C). (b) <br /> Exponentially growing dis3-54 and wild-type cultures were transferred to 20&#176;C. Samples were taken after 4 and 10 h and analyzed with the <br /> FACScan. All cells analyzed at 10 h contained 2c DNA content. <br />  <br />  <br /> ing medium abundance. The biochemical activity of the                                    cdc25 weel, as can the other two PP1 genes. We recently <br /> dis3+ protein is not known.                                                              found that the ability of dis2+ gene overexpression to <br />    The results of the gene disruption experiments showed                                 reverse the phenotype of cdc25 weel resides in a small <br /> that the dis3+ gene is essential for cell division and spore                             domain near the COOH terminus of the dis2+ protein (37a). <br /> germination, consistent with the isolation of a recessive cs                             The full-length PP1 protein is not needed, so the dis3+ <br /> mutation. The disruption phenotype, however, was appar-                                  protein may directly or indirectly interact with PP1 to affect <br /> ently different from that of cs dis3-54. Haploid gene disrup-                            cdc25 weel regulation of cdc2+ kinase activation. Fourth, <br /> tant cells carrying a plasmid with the dis3+ gene grew but                               the predicted amino acid sequence of dis3+ is similar to that <br /> were arrested upon the loss of the plasmid in the rich                                   of the budding yeast SSDI or SRKI gene product (32), which <br /> medium. The characteristic mitosis phenotype of dis3-54                                  can suppress the lethality attributable to the absence of <br /> was not found in these plasmid-deficient cells. Therefore, the                           PPase gene SIT4 (1). We recently isolated in fission yeast a <br /> dis3-54 mutant phenotype at the restrictive temperature may                              gene (designated ppxl ) which is similar to SIT4 and will <br /> be allele specific and may be produced by a partially active                             examine the functional relationship between dis3+ and <br /> mutant protein. The dis3+ gene product is perhaps required                               ppxl+. The phenotypic similarity to dis2-11, the synthetic <br /> not only in mitosis but also in other stages of the cell cycle.                          lethality attributable to dis2-11, and the high-dosage effect <br /> Although cs dis3-54 mutant cells can transit the G1/S bound-                             upon weel and/or cdc25 mutations suggested that the dis2+ <br /> ary at the restrictive temperature, it is still possible that the                        and dis3+ gene products are related. The sequence similarity <br /> complete loss of the dis3+ protein may cause arrest at the G1                            to budding yeast SSDI or SRKI indicated the relationship to <br /> phase. This point should be examined by generating more                                  protein dephosphorylation and phosphorylation. <br /> mutations in the dis3+ gene.                                                                However, we have also obtained results which do not <br />    Several lines of evidence favor the hypothesis that dis3+                             support the hypothesis. First, the level of PP1 activity <br /> gene function is involved in the regulation of protein dephos-                           monitored as phosphorylase phosphatase is not reduced in <br /> phorylation during mitosis. First, the cell cycle phenotype of                           dis3-54 mutant extracts, whereas it is hardly detected in <br /> the dis3-54 mutant is similar to that of cs dis2-11. The cellular                        dis2-11 mutant extracts. However, dis3-54 may affect a <br /> localization of the dis3+ protein is reminiscent of that of the                          phosphatase having different protein substrates. Second, the <br /> dis2+ protein. Second, dis2cs dis3cs or dis2nu&quot; dis3cs double                            dis3+ gene product is not complexed with the dis2+ gene <br /> mutants are not viable, even at the permissive temperature,                              product. Third, although dis3+ and SRKI or SSDI may be <br /> consistent with the notion that these two genes may have an                              functionally related, the dis3+ gene is essential, but the <br /> overlapping function. Third, the dis3+ gene under the con-                               SRKI or SSDI gene is nonessential. The amino acid se- <br /> trol of the ADH promoter can reverse the Ts&apos; phenotype of                                quence identity between dis3+ and SRKI or SSDI is low, so <br />  <br />  <br /> their relationship remains to be determined. The dis3 muta-         <br /> tion blocks mitosis, but the SSDI gene is implicated in the                <br /> G1/S transition (32). Therefore, despite the apparent pheno-            <br /> typic similarity of dis3-54 to dis2-11, the defect in dis3-54 may        <br /> not be related to protein dephosphorylation. Only future work        <br /> will elucidate the gene function at the molecular level.                    <br />   <br /> </body></html>