<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, OCt. 1992, p. 4314-4326                                                                   Vol. 12, No. 10 <br /> 0270-7306/92/104314-13$02.00/0 <br /> Copyright &#169; 1992, American Society Microbiology <br />  <br />  <br />         RPC53 Encodes Subunit <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-10'  ></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> RNA <br />             Polymerase C (III) Whose Inactivation Leads a <br />                         Predominantly G1 Arrest <br />           CARL MANN,* JEAN-YVES MICOUIN, NUCHANARD CHIANNILKULCHAI, ISABELLE TREICH, <br />                                  JEAN-MARIE BUHLER, AND ANDRE SENTENAC <br />                 Service Biochimie et Genetique Moleculaire, Batiment 142, Centre d&apos;Etudes Saclay, <br />                                              F-91191 Gif-sur-Yvette, France <br />                               Received 4 May 1992/Returned modification 12 June 1992/Accepted 30 June 1992 <br />  <br />              RPC53 shown essential gene encoding C53 subunit specifically associated yeast RNA <br />            polymerase C (M). Temperature-sensitive rpc53 mutants generated showed rapid inhibition tRNA <br />            synthesis transfer restrictive temperature. Unexpectedly, rpc53 mutants preferentially arrested <br />            cell division G1 phase large, round, unbudded cells. The RPC53 DNA sequence predicted code <br />            hydrophiic M,-46,916 protein enriched charged amino acid residues. The carboxy-terminal 136 amino <br />            acids C53 significantly similar (25% identical amino acid residues) region human BN51 <br />            protein. The BN51 cDNA originally isolated ability complement temperature-sensitive hamster cell <br />            mutant undergoes G1 cell division arrest, true rpc53 mutants. <br />  <br />    The eukaryotic RNA polymerases complex enzymes                                     MATERIALS AND METHODS <br /> composed multiple, distinct subunits. <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-25'  ></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span> <br /> cerevisiae RNA polymerase C activity associated                    Strains media. The yeast strains used study are <br /> complex 13 different polypeptides ranging 10             described Table 1. CMY356 haploid meiotic segregant <br /> to 160 kDa (4, 11, 13, 49, 56). A subset yeast RNA                 obtained sporulation CMY242 transformed the <br /> polymerase C subunits homologous eubacterial                    plasmid pEMBLYc32-RPC53. CMY396 derived by <br /> RNA polymerase core enzyme. C160 C128 homolo- <br />                                                                           growing log-phase CMY356 cells yeast extract-peptone- <br />                                                                           dextrose (2%) (YPD) presence 4 ,ug acriflavine <br /> gous <span id='am-54' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-66'  ></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia coli</span> 1 13&apos; subunits, respectively             ml generations. A [rhoo] segregant identi- <br /> (1, 24). Two molecules subunit E. coli              fied inability use glycerol carbon source by <br /> RNA polymerase. AC40 AC19 domain                          absence mitochondrial DNA determined fluores- <br /> similar functionally important domain subunit               cent staining 4&apos;,6&apos;-diamidino-2-phenylindole (DAPI) (see <br /> (8). It proposed copy AC40                  Fig. 7E). CMY397 mitotic segregant CMY396 which <br /> AC19 RNA polymerases A C functionally homol-                    pEMBLYc32-RPC53 lost. <br /> ogous ao homodimer E. coli RNA polymerase. A                   Yeasts cultivated YPD synthetic complete <br /> homodimer B44.5 subunit likely represent               medium (50). <br /> homolog B enzyme (29, 30). The largest subunits                   DNA sequencing RPC53. The sequence (see Fig. 2) of <br /> along al homologs probably perform                   2,414-bp RPC53 HindIII-NmI fragment (Fig. 1C) was <br /> same functions polymerase assembly basic catal-                determined subcloning portions fragment in <br /> ysis transcription homologs well-studied            M13 Bluescript (Stratagene) <span id='am-77' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>vectors</span> orien- <br /> eubacterial enzyme. In contrast core subunits,              tations. The DNA sequence fragments deter- <br /> a function assigned remaining                mined dideoxynucleotide method modified T7 <br /> small subunits. Five subunits (ABC27, ABC23,                     DNA polymerase (Sequenase; United States Biochemical <br /> ABC14.5, ABC10a, ABC10,B) shared                      Corp.). As described Results, unexpectedly ob- <br /> three nuclear polymerases probably contribute               served 2.4-kb RPC53 HindIII-NruI fragment cloned <br /> common eukaryotic (and possibly archaebacterial) core en-                 AB320 yeast strain (38) complement an <br /> zyme (4, 56). The small subunits specifically associated             rpc53::HIS3-2 lethal disruption cloned pUN20 <br /> with RNA polymerase C (C82, C53, C34, C31)                       (TRPI SUP11 ARSI CEN4) vector (10), cloned <br /> destined perform functions specific transcription           YCp5O (URA3 ARS1 CEN4) (46). We eventually ascribed <br /> polymerase. The C82 (6), C34 (52), C31 (37) proteins                  result inhibitory plasmid context effect YCp5O <br /> have shown necessary yeast cell viability              expression RPC53. However, pUN20 carries <br /> and synthesis tRNA RNA polymerase C. In                suppressor tRNA gene, YCp5O does not, we <br /> report, C53 performs essential cellular              initially wanted exclude possibility RPC53 <br /> function required tRNA synthesis. Furthermore,                     gene obtained AB320 strain contained nonsense <br /> report sequence similarity C53 BN51                     mutation. We cloned directly corresponding 2.4-kb <br /> protein encode human homolog C53.                         HindIII-NruI RPC53 fragment strain LL20 (39) the <br />                                                                           vector pEMBLYc32 (37). The RPC53 DNA sequences from <br />                                                                           strains LL20 AB320 determined. Shown in <br />                                                                           Fig. 2 DNA sequence strain LL20. The <br />                                                                           sequence RPC53 gene strain AB320 did contain <br />   * <br />       Corresponding author.                                               nonsense mutations, did number sequence <br />                                                                    4314 <br />                                    J <br /> VOL. 12, 1992                                                         YEAST RNA POLYMERASE C53, HUMAN BN51, AND G1 ARREST                               4315 <br />  <br />                                                                       TABLE 1. Yeast strains used <br />   Strain                                                                                                      Genotype <br /> CMY214 .....               a/a trpl Al/trpl Al ura3-52/ura3-52 his3A200Ihis3A200 lys2-801/lys2-801 ade2-101/ade2-101 canl/+ <br /> CMY242 .....               CMY214 rpcS3::HIS3-1/+ <br /> CMY274 .....               trplAl his3A200 lys2-801 ade2-101 canl RPC53::pEMBLYi32(URA3) <br /> CMY292 .....               trplAl his3 A200 lys2-801 ade2-101 canl rpcS3::HIS3-l/pUN20-RPC53 <br /> CMY356 .....               trplAl his3 A200 lys2-801 ade2-101 canl rpcS3:JHIS3-l/pEMBLYc32-RPC53 <br /> CMY370 ......              CMY214 rpcS3:HIS3-2/+ <br /> CMY381 .....               trplAl his3 A200 lys2-801 ade2-101 canl rpcS3:1HIS3-2/pEMBLYc32-rpcS3-ts28 <br /> CMY383 .....               trplAl his3 A200 lys2-801 ade2-101 canl rpc53::HIS3-2/pUN20-RPC53 <br /> CMY384 .....               trplAl his3 A200 lys2-801 ade2-101 canl rpcS3::HIS3-2/pEMBLYc32-RPC53 <br /> CMY323a ..... (RY262) rpbl-l ura3-52 <br /> CMY388b ......a cdc9 his7 leu2 canl ura3 hom3 sap3 <br /> CMY396 .....               trplAl his3A200 lys2-801 ade2-101 canl rpcS3::HIS3-1 [rho&#176;]/pEMBLYc32-RPC53 <br /> CMY397 .....               trplAl his3A200 lys2-801 ade2-101 canl rpc53::HIS3-1 [rho&apos;] <br /> SC12 .....                 leu2Al trplAl his3A200 lys2-801 ade2-101 canl rpc53::HIS3-2IpEMBLYc32-RPCS3 <br />  A gift Rick Young. <br />   b   A gift Lee Hartwell. <br />  <br />  <br /> differences relative RPC53 sequence LL20.                                      ARS fragmentation vector transformants carried chro- <br /> Five substitutions <span id='am-41' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-44'  ></span><span id='am-45' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> positions hav-                                    mosome 4 derivative 200- 250-kb deletion, whereas <br /> ing effect predicted C53 amino acid sequence:                                        opposite orientation, CEN vector transformants <br /> C303T, A570G, C633T, A885G, A984G (the numbering                                           displayed chromosome fragment 200 250 kb. Southern <br /> scheme refers shown Fig. 2). Two substitutions                                      blotting analysis gel showed HO gene <br /> were leading conservative amino acid changes                                       hybridized 200- 250-kb chromosome fragment. <br /> the predicted C53 proteins: G417C leading substitution                                  These results indicated RPC53 gene 200 <br /> E139D, C833T leading substitution A276V. Two                                        250 kb left end chromosome 4. After sequenc- <br /> more consequential sequence differences found:                                          ing RPC53 gene, noticed presence rare NotI <br /> deletion 6 bp, 793 798, leading deletion                                     sequence coding region. When NotI-SfiI map <br /> amino acids (GlyLeu, 255 256), inversion 4 bp,                                   S. cerevisiae genome available (32), we <br /> 858 861, leading change residues, LysArg                                   attempted identify RPC53 NotI site. There three <br /> (286 287) AsnAla. All variations represent                                        NotI sites 250 kb left arm chromo- <br /> sequence polymorphisms effect function                                       4. We probed membrane containing yeast chromo- <br /> the C53 protein RPCS3 genes LL20 AB320                                         somal DNA separately digested NotI Sfil sepa- <br /> are equally capable complementing rpc53::HIS3-2                                         rated orthogonal-field-alternation gel electrophoresis to <br /> mutation cloned pUN20 pEMBLYc32 vectors.                                            localize RPC53 NotI site (see Fig. 4A). <span id='am-36' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-38'  ></span><span id='am-39' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> blot a <br />    Physical mapping RPCS3. We initially mapped                                          generous gift Phil James, digested yeast chromo- <br /> RPCS3 gene left arm chromosome 4                                              somal DNA situ blocks agarose (24). The hybridiza- <br /> chromosome fragmentation method (62). An RPCS3 frag-                                           tion pattern obtained probing RPC53 PvuII- <br /> ment cloned orientations ARS fragmen-                                       XbaI 1-kb fragment spanned NotI site conclusively <br /> tation vector YCF3 CEN fragmentation vector                                            identified RPC53 NotI site sites <br /> YCF4. After linearization <span id='am-99' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-100' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span> junction                                      left end chromosome. This places RPC53 <br /> between RPCS3 vector sequences, stable transfor-                                       gene 210 kb chromosome 4 left telomere, good <br /> mants diploid yeast strain obtained. Orthogonal-                                     agreement estimation chromosome frag- <br /> field-alternation gel electrophoresis (OFAGE) chromo-                                       mentation method. <br /> somes transformants showed orientation,                                         Genetic mapping RPC53. With knowledge that <br />  <br />                                                                          X gtll-RPC53 <br />                                   A)                                     IacZ             p- <br />  <br />  <br />                                                                                 R                             R       R <br />  <br />                                          B&apos; R             Sn   RH P       VRG G <br />                                                                                 I        X INr                R   G <br />                                                                                                                       I <br />                                                                                                                       SnRv                  B&apos; <br />                                                                                                  L~~~~                I <br />                                                                                                                                1 kb <br />                                                                                                                           i_            I <br />  <br />  <br />                                                 H   PNc          ATG V          RG   G    NIh N Ns X     Nr <br />                                                                                                                                0.5 kb <br />                                   r- I          I <br />                                                 I   I a <br />                                                                   r         .        .           1,I ---I1                          I <br />                                                                            So S      Not           TAA <br />    FIG. 1. Restriction maps cloned RPCS3 fragments. (A) Agtll-RPC53 clone obtained immunoscreening anti-C53 immunoglob- <br /> ulins (45). The solid portion represents RPCS3 coding sequences fused carboxy terminus p-galactosidase. (B) Genomic fragment <br /> containing complete RPC53 gene obtained screening YRp7-Sau3A yeast genomic DNA bank (38) EcoRI DNA insert of <br /> Agtll-RPC53. (C) The 2,414-bp RPCS3 HindIII-NnI fragment DNA sequence determined (see Fig. 2). B&apos;, BamHI-Sau3A <br /> junction; G, BglII; H, HindIII; N, NaeI; Nc, NcoI; Nh, NMeI; Ns, NsiI; P, PstI; R, EcoRI; Rv, EcoRV; S, Sall; Sn, SnaBI; Ss, SstI; V, <br /> PvuII; X, XbaI. <br />  4316     MANN ET AL.                                                                                                                                                                                                                                                                                                                                                                            MOL. CELL. BIOL. <br />  <br />            aagcttccttttctatttccttagcagcttcgtcatcatctaaatcatcagccccataatattctccttttgtggacccccaggcattctcgttgtctaa -670 <br />            cataccattctcttcatcttcctcaggaagctggtcagtttctaagtttctaccaaatattttcttataggctgcagcaccatctagttcctcttcctct -570 <br />  <br />            tcttcttcctcatcttctct&apos;ttcatctattgactcatcatcttcatccatggctaagacttcttcctcatcttcgtcttctaacagagaatggtcatcat                                                                                                                                                                                                                                                                                            -470 <br />  <br />            ctttattactgtcgccaaaggtggattgtccaagtaaaaccttttcccttttagaagcaaaatcatcaacttcgtttaaaccatacggattaatctcatc -370 <br />                                     *                                   *                                       *                                       *.&lt;M                                    *                       RPG                                                                                                     RPC160 <br />            cccaacttcagaggtcttagttctgtttgagcctttgcgtacCATattattctatj= <br />                                                                  ggA A tatgttttctcttcctccaaaactttttg= -270 <br />                                     *           PAC                                         PAC                 * <br />           AaaagcatcatgtagatgaQcag&amp;t_                                                                                    cttcattattgaattttttttcattg cagaagcg cagtttgaaaatttttcatccgccctgcaaa -170 <br />  <br />            gtccaattttttgatcagcagaagataatgaccagagaattatggtattcaagagaacaattcaaaagaggacagctagaaagatattgaggtattt&amp;ata                                                                                                                                                                                                                                                                                            -70 <br />  <br />           Atttgctagactaaccgaaagtcgaagcgtaatcttgttttttattaagcgggatag cattcattaagtATGAGCAGTAATAAAGGAAATGGGCGCTTGC                                                                                                                                                                                                                                                                                              31 <br />                                                                                                                                                                                                                                                                                 M           S           S           N           K           G           N           G           R           L           P <br />  <br />            CATCATTAAAAGATTCCTCCTCCAATGGAGGAGGATCTGCCAAGCCCTCATTAAAGTTTAAACCAAAAGCAGTTGCAAGAAAGTCCAAGGAAAAAAGAGA 131 <br />              S L K D S S S N G G G S A K                  S L K F K P K A V A R K S K E K R E <br />  <br />            AGCAGCTGCGTCCAAAGTAAAGCTAGAGGAGGAATCTAAGAGAGGTAATGACAAGAAACATTTCAATAACAAGAATAAAAGAGTAACCGGCGCTGGCGGC 231 <br />             A       A       A           S       K       V               K           L       E               E       E               S           K           R       G           N               D           K           K           H           F       N               N           K           N           K           R           V           T           G           A           G           G <br />  <br />            CAGCAAAGGCGAATGGCCAAATACTTAAATAACACACACGTTATCTCTAGCGGTCCATTGGCGGCTGGGAACTTTGTAAGTGAGAAGGGTGATTTGAGAA 331 <br />            Q Q R R M A K Y L N N T H V I S S G P L A A G N F V S E K G D L R R <br />  <br />            GAGGATTCATCAAATCAGAAGGAAGCGGGTCATCTCTTGTGCAAAAGGGCCTAGAAACTATTGACAATGGTGCTGAGAGCTCTGAGAATGAGGCAGAAGA 431 <br />                 G       F       I           K       S           E           G           S           G           S           S           L           V           Q           K           G           L           E           T           I           D           N           G           A           E           S           S           E           N           E           A           E           D <br />  <br />            CGATGATAATGAAGGTGTAGCGTCCAAATCTAAGAAGAAGTTCAATATGGGAAAAGAATTCGAGGCACGCAATCTCATAGAGGACGAAGATGACGGCGAA 531 <br />             D       D       N           E       G           V           A           S           K           S           K           K           K           F           N           M           G           K           E           F           E           A           R           N           L           I           E           D           E           D           D           G           E <br />  <br />            AGTGAAAAGAGCAGTGACGTCGACATGGATGACGAAGAATGGAGATCTAAACGGATTGAACAGTTATTCCCTGTGAGACCTGTCCGCGTAAGACACGAAG 631 <br />            S E K S S D V D M D D E E W R S K R I E Q L F P V R P V R V R H E D <br />  <br />            ACGTTGAAACTGTCAAAAGAGAAATACAAGAAGCTCTTTCAGAAAAGCCAACTCGTGAACCAACCCCCAGTGTGAAGACAGAGCCTGTCGGAACCGGATT 731 <br />              V E T V K R E I Q E A L S E K P T R E P T P S V K T E P V G T G L <br />  <br />            ACAATCTTATTTGGAAGAAAGAGAAAGGCAAGTCAATGAGAAACTGGCAGATCTTGGACTTGGACTTGAAAAGGAGTTTCAATCGGTTGATGGGAAAGAA 831 <br />             Q S Y L E E R E R Q V N E K L A D L G L G L E K E F Q S V D G K E <br />  <br />            GCGGCCGCTGAGTTGGAATTATTAAAACGTGATCATCAGCATATATTACGAAAACTAAAGAAAATGAATAATAAACCAGAAAGATTCATGGTATTCCAGT 931 <br />            A A A E L E L L K R D H Q H I L R K L K K M N N K P E R F M V F Q L <br />  <br />            TACCTACTAGGTTACCAGCTTTTGAAAGACCCGCTGTGAAAGAAGAAAAAGAAGACATGGAAACCCAGGCTAGCGACCCTTCAAAGAAGAAGAAGAATAT 1031 <br />                 P       T       R           L       P           A           F           E           R           P           A           V           K           E           E           K           E           D           M           E           T           Q           A           S           D           P           S           K           K           K           K           N           I <br />  <br />            TAAAAAGAAGGACACGAAGGATGCTTTGTCTACTAGAGAACTTGCCGGCAAGGTTGGGTCTATACGGGTTCACAAATCTGGAAAACTTTCCGTGAAAATT 1131 <br />             K K K D T K D A L S T R E L A G K V G S I R V H K S G K L S V K I <br />  <br />            GGAAATGTGGTGATGGATATTGGCAAAGGTGCCGAAACCACATTTTTACATGATGTTATAGCATTAAGTATCGCTGATGATGCATCCTCAGCGGAACTTC 1231 <br />            G        N       V       V           M       D           I           G           K           G           A           E           T           T           F           L           H           D           V           I           A           L           S           I           A           D           D           A           S           S           A           E           L           L <br />  <br />            TAGGCCGTGTGGACGGTAA&apos;AATAGTAGTCACACCTCAAATCTAAtcgc <br />              G R V D G K I V V T P Q               * <br />                                                              actcgc                                                                                                                                                                     atctgtcgagtatataaatgaatatacacagtcataaatacttcta 1331 <br />  <br />            gaacaaattacactaattaagatgcttagattcccattcaaaaggtactattgacgtcttgtcttctacaatttctcatcctctttgaacattggtaatg 1431 <br />            ttattctagttattattacccagtttcggaactttaggggagccacactttaaacaaaaagaagcgccataaccaccgtgaagtaatgagtatctcactt                                                                                                                                                                                                                                                                                             1531 <br />  <br />            tgataagaagacatttaagaacagcctgaaatatcaaaatcacggaattattaggttatggagagagatgaataccagttacccaactctcatgggaaga 1631 <br />            atactttcttatcg <br />    FIG. 2. Sequence 2,414-bp RPC53 HindIII-NruI (Fig. 1C) DNA fragment predicted C53 amino acid sequence. This sequence <br /> has assigned EMBL <span id='am-94' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-96'  ></span><span id='am-97' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Data</span> Library accession number X63501. The asterisks -87 -89 major sites transcription <br /> initiation, asterisk 1446 major site transcription termination determined S1 nuclease mapping (35). The underlined <br /> sequence KPSLK (amino acids 25 29) represents possible nuclear localization signal (16). An RPG box [consensus sequence <br /> R(C/A)AYCCRYNCAYY (61)] -314, PAC boxes [consensus sequence TG(A/C)GATGAG (8)] -258 -246, 16-bp sequence <br /> (-290) similar sequence -45 RPC160 promoter (1), TATAA sequences -272 -73 RPC53 <br /> promoter region. These sequences contribute transcriptional regulation RPC53 gene. Note divergently oriented open <br /> reading frame encoding 148 amino acids observed starting -325 (indicated &lt;M initiating codon figure). <br /> The predicted protein sequence open reading frame composed 50% acidic amino acid residues. <br />  <br />  <br /> RPC53 200 kb left end chromosome 4,                                                                                                                                                           plasmid. An 800-bp EcoRI-XbaI RPC53 fragment cloned <br /> we sought genetic linkage mutations                                                                                                                                                              pEMBLYi32 (2), resulting plasmid cut <br /> region. The RPC53 chromosomal locus marked                                                                                                                                                                 RPC53 sequences BglII target integration to <br /> URA3 targeted integration pEMBLYi32-RPC53                                                                                                                                                                   homologous site lithium acetate transformation (20) <br />  VOL. 12, 1992                                        YEAST RNA POLYMERASE C53, HUMAN BN51, AND G1 ARREST                         4317 <br />  <br /> of CMY215. The integration pEMBLYi32(URA3)                  tained retransformation CMY383 tester strain <br /> RPC53 locus confirmed Southern blot analysis           purified mutant plasmid. <br /> transformant CMY274 used meiotic mapping experi-                  Continuous radioactive labeling protein RNA. <br /> ments. CMY274 (RPC53::URA3) crossed rpbl                       CMY397 rpc53::HIS3-1 [rho&apos; isogenic wild-type <br /> (CMY323) cdc9 (CMY388) temperature-sensitive mutant               CMY396 rpc53::HIS3-1 [rho ]/pEMBLYc32-RPC53 cells <br /> strains. After sporulation resulting diploid strains, tetrad   grown 23&#176;C synthetic complete medium contain- <br /> analysis showed RPC53::URA3 lie 1.5 centimorgans              ing 10 ,ug uracil ml optical density 660 nm of <br /> rpbl 4 centimorgans cdc9 (see Fig. 4B).                      0.1. [3H]uracil (50 Ci/mmol) [35S]methionine (500 Ci/ <br />    Disruption RPC53. A pBR322-rpc53::HIS3-1 plasmid                mmol) added cultures concentration of <br /> was created subcloning 2.4-kb HindIII-NruI               1 0.5 p,Ci/ml, respectively. The cultures transferred <br /> RPC53 fragment pBR322 substituting internal           38&#176;C 1 h addition radioactive precursors. <br /> 207-bp BglII RPC53 fragment 1.7-kb BamHI HIS3                Incorporation radioactivity macromolecules (see Fig. <br /> fragment (54). Digestion plasmid NcoI NruI           6D) followed trichloroacetic acid precipitation: 4 ml <br /> liberates 3.6-kb rpc53::HIS3-1 fragment (see Fig. 5)           cold 5% trichloroacetic acid added 1 ml culture, <br /> was used gene disruption (47) copy RPC53            precipitate collected filtration glass fiber <br /> gene diploid yeast strain CMY214.                              filters extensive washing 5% trichloroacetic <br />    An essentially complete deletion RPC53 gene             acid. The filters washed ethanol, dried, and <br /> made subcloning 2.4-kb HindIII-NruI RPC53                placed liquid scintillation fluid simultaneous count- <br /> fragment YCp5O (46). After digestion plasmid          ing tritium 35S. <br /> PvuII, short treatment nuclease Bal 31 performed              The increase cell number optical density as <br /> to delete RPC53 coding sequences. This followed                percentage unbudded cells monitored parallel <br />                                                                       unlabeled cultures (see Fig. 6A, B, C). <br /> digestion XbaI, filling-in protruding ends          Labeling analysis small, stable RNAs. CMY396, <br /> Klenow DNA polymerase, recircularization plas-             CMY397, CMY381, CMY384 cells grown 23&#176;C <br /> mid T4 DNA ligase presence BamHI linker                synthetic complete medium containing 5 ,ug uracil per <br /> oligonucleotides. After recovery E. coli, plasmid            ml optical density (660 nm) 0.4. For labeling, 20 <br /> found containing deletion approximately 60 bp         ml cells transferred 37 38&#176;C 30 <br /> RPC53 coding sequence BamHI linker site              min, 3 h, 6 h (the temperature time incubation for <br /> the deletion. The size deletion estimated           different strains indicated Fig. 9). RNA was <br /> gel migration restriction fragments. The 1.7-kb BamHI              labeled adding culture [3H]uracil (50 Ci/mmol) to <br /> HIS3 fragment inserted plasmid yield          concentration 15 ,uCi/ml 1 h. Labeled cells were <br /> YCp5O-rpc53::HIS3-2 construction (see Fig. 5). Digestion           centrifuged resuspended 5 ml extraction buffer (100 <br /> this plasmid NcoI NruI liberates 2.6-kb                    mM Tris-HCl [pH 7.5], 250 mM NaCl, 5 mM EDTA, 0.5% <br /> rpc53::HIS3-2 fragment used delete             sodium dodecyl sulfate [SDS]) followed immediately the <br /> chromosomal RPC53 genes CMY214 diploid yeast                   addition equal volume phenol. Small RNAs were <br /> strain.                                                               extracted vigorous agitation emulsion 65&#176;C 1 <br />    In vitro mutagenesis RPC53. The pEMBLYc32-RPC53                 h. RNA aqueous phase precipitated the <br /> plasmid converted single-stranded form superin-             addition 2 volumes ethanol. The RNA pellet next <br /> fecting E. coli JM107 cells containing plasmid          resolubilized 0.1 ml diethyl pyrocarbonate-treated <br /> R408 Fl helper bacteriophage (48). A gapped heteroduplex              water, radioactivity incorporated determined <br /> (42) formed denaturation double-stranded            yield RNA estimated sample A260 (an <br /> pEMBLYc32 vector (digested sites used subclon-             A260 1 taken represent 40 ,ug RNA ml). RNA <br /> ing RPC53 fragment) presence single-stranded            (20 p,g) sample denatured 90% formamide <br /> pEMBLYc32-RPC53 DNA; followed period                 loading buffer electrophoresis 6% polyacryl- <br /> renaturation. The resulting heteroduplex molecules com-           amide-7 M urea-i x TBE (Tris-borate-EDTA) gel sepa- <br /> posed double-stranded vector sequences               rate small RNAs. The gel briefly stained 0.5 <br /> RPC53 fragment remaining single stranded. Treatment                mg ethidium bromide ml visualize RNA and <br /> these heteroduplexes sodium bisulfite specifically re-           photographed fixed 7% acetic acid, treated <br /> stricts GC-to-AT transition mutations RPC53                Amplify (Amersham) fluorography, dried. Flu- <br /> fragment. Gapped heteroduplexes incubated 4 M               orography -70&#176;C preflashed Kodak XAR <br /> sodium bisulfite 37&#176;C 10, 20, 40 min,               film. <br /> reactions stopped DNA purified previously                Flow cytometiry yeast cells. For sample be <br /> described (33). The mutagenized heteroduplexes              analyzed, approximately 107 cells fixed 70% ethanol <br /> used transform directly E. coli ung strain BD1528 (42)         1 h. Cells washed 0.2 M <br /> to repair single-stranded gap fix C-to-U               Tris-HCl (pH 7.5) resuspended 0.1 ml same <br /> transition mutations. Conditional rpc53 mutants             buffer containing 100 ,ug RNase A. RNA digested by <br /> screened plasmid-shuffle test. Mutagenized DNA               incubating 37&#176;C 2 h. Cells washed in <br /> banks used transform yeast strain CMY383                  phosphate-buffered saline (PBS) resuspended 0.1 ml <br /> rpc53::HIS3-2/pUN20-RPC53, conditional mutants                    50 ,ug propidium iodide ml PBS 20 min. After <br /> (cold heat sensitive) sought transformants            washed PBS, cells resuspended 5 ,ug of <br /> unable grow restrictive temperature counters-         propidium iodide ml PBS concentration about <br /> electing pUN20(SUP11)-RPC53 plasmid replica plat-              5 x 106 cells ml. Cells lightly sonicated and <br /> ing transformants YPD plus 2.5 M ethylene glycol plates            observed fluorescence microscopy ensure specific <br /> (10). Mutations shown plasmid linked recov-             nuclear staining quantifying fluorescent emission with <br /> ering pEMBLYc32-RPC53 E. coli showing                 OrthoDiagnostics Systems 2150 fluorescence-activated <br /> the conditional mutant phenotype uniformly ob-               cell sorter (FACS). <br />  4318     MANN ET AL.                                                                                                                        MOL. CELL. BIOL. <br />  <br />    Expression functional testing BN51 protein yeast      R        Acidic Basic Residues C53 <br /> cells. Two galactose-inducible BN51 genes con-                              I   I   I <br />                                                                                               100 <br />                                                                                                   I   a <br />                                                                                                                 200 <br />                                                                                                                       I   I      I    I    I   I     I   a <br />                                                                                                                                                                  400 <br />                                                                                                                                                                   I    a <br /> structed, entire coding sequence         A    I                I 11I1 I d I III I II I III III A <br />                                                                                          AI <br /> that portion sequence encoding 167 amino           B         I               I1                0I 0                     I10 4I0 <br />                                                                                                                                              1 <br /> acids BN51 protein (starting internal methi- <br />                                                                                               100               200               300                            400 <br /> onine position 228 amino acid sequence [22]). These <br /> BN51 sequences placed control                B <br /> GALIO promoter Fusionator multicopy plasmid (a                                                                                                                      2 <br /> generous gift Stephen Johnson, Department Genet-                                                                                                                    1 <br />                                                                                                                                                                            10 <br /> ics, University Washington). The Fusionator 2 ,um-                                                                                                                 -1 <br /> based plasmid containing LEU2 selectable marker (26).                                                                                                                  -2 <br /> Oligonucleotides used amplify polymerase chain                                                                                                                  -3 <br />                                                                                                                                                                            -4 <br /> reaction BN51-coding sequences SalI-BamHI DNA <br /> fragments starting human BN51 cDNA clone (19). <br /> The following primers used amplification       c <br /> entire BN51-coding region: HBN3M = TTCCGTCGACCAT <br /> GTCGGAAGGAAACGC (5&apos; oligonucleotide) HBN1V <br />                                                                         0 <br /> =  GGGGGGATCCTTACCGGTG&apos;TTGTGATCC (3&apos; oligo-                            -1 <br /> nucleotide). A SalI-BamHI fragment encoding 167               -2 <br /> BN51 amino acids amplified using primers                    -3 <br />                                                                        -4- <br /> HBN2M (GGGGGTCGACCATGAAGGCTCCTCCCAAA <br /> GC [5&apos; oligonucleotide]) 3&apos; oligonucleotide <br /> (HBN1V) above. Cloning SalI-BamHI fragments             FIG. 3. (A) Graphic display positions acidic (A) and <br /> into sites Fusionator polylinker placed       basic (B) residues C53 protein sequence. The bars in <br /> transcription coding sequences control      A (acidic) row represent glutamate residues, lower bars <br /> the GAL10 promoter. Polymerase chain reaction-derived             represent aspartate residues. In B (basic) row, bars <br /> clones plasmids introduced            represent arginine, intermediate bars represent lysine, the <br /> the yeast strain SC12 lithium acetate transformation           small bars represent histidine residues. (B C) Kyte-Doolittle <br /> (20). SC12 [rpc53::HIS3-2IpEMBLYc32(URA3)-RPC53]              hydropathy plots C53 BN51 proteins, respectively. <br />                                                                   Positive values represent hydrophobic regions, negative values <br /> used test ability BN51 proteins complement       represent hydrophilic regions, window 11 amino acids <br /> a complete deletion RPC53 gene. SC12 transformants         protein. Notice overall similarity profiles the <br /> expressing BN51 BN51 carboxy-terminal sequences           proteins. All graphics figure executed the <br /> galactose-containing medium tested ability      DNA Strider program (34). <br /> grow absence pEMBLYc32(URA3)-RPC53 <br /> plasmid. Cells unable lose plasmid sensitive the <br /> drug 5-fluoro-orotic acid (3). <br />    EMBL data library submission. The RPC53 DNA se-                (Fig. 2). This reading frame fused lacZ <br /> quence (Fig. 2) assigned accession number            original Xgtll clone determined sequencing the <br /> X63501.                                                           fusion site phage DNA. The difference the <br />                                                                   predicted molecular weight C53 observed by <br />                           RESULTS                                 SDS-polyacrylamide gel electrophoresis inherent <br />                                                                   property sequence posttranslational <br />    Cloning sequencing RPC53 gene. Two Xgt11            modifications yeast cells. Both factors been <br /> clones potentially encoding RPC53 gene         contribute anomalous electrophoretic migration <br /> obtained screening expression library antibodies       RNA polymerase subunits (56). No TACTAAC <br /> prepared C53 subunit (45). Temperature-induc-         splicing signal (64) DNA sequence, we <br /> ible lysogens clones produced fusion protein      assume gene does contain intron. Northern <br /> 30 kDa larger 0-galactosidase reacted         (RNA) blot nuclease S1 protection analysis (35) indi- <br /> anti-C53 anti-,-galactosidase antibodies. Extracts         cated RPC53 expressed mRNA 1,500 <br /> bacteria producing fusion protein able bind          nucleotides initiation termination sites shown in <br /> antibodies anti-C53 immunoglobulin preparation           Fig. 2. <br /> that capable specifically inhibiting RNA polymerase          The deduced amino acid sequence C53 subunit is <br /> C transcription vitro (35), indicating        hydrophilic length, excepting carboxy- <br /> likely isolated C53-coding sequences. Restriction enzyme          terminal 50-amino-acid region shown hydropathy <br /> analysis showed clones contained yeast DNA         plot Fig. 3B. This hydrophilicity accounted a <br /> insert common EcoRI site ligated 3&apos; end      striking distribution charged residues (Fig. 3A). An ami- <br /> lacZ gene Xgtll (Fig. 1A). Since fusion protein            no-terminal 125-amino-acid domain highly basic is <br /> analysis indicated portion RPC53 coding        followed alternating clusters acidic basic residues. <br /> sequence isolated, complete gene sought          These structural features suggest C53 subunit lies on <br /> from YRp7 yeast genomic DNA bank (38) colony                 surface RNA polymerase C extended conforma- <br /> hybridization Xgt1l yeast DNA insert. A 7.7-kb           tion. This suggestion consistent extreme suscep- <br /> genomic DNA fragment obtained contained              tibility C53 subunit proteolytic degradation (19, 49) <br /> complete gene (Fig. 1B). The DNA sequence 2,414-bp         relative ease dissociated <br /> HindIII-NnuI fragment shown Fig. 1C determined.            polymerase (5, 63). C53 calculated isoelectric <br /> An open reading frame 1,272 nucleotides corresponding       point 9.2. <br /> a protein 424 amino acids (predicted Mr, 46916)         Sequence motifs potentially involved nuclear transport <br />  VOL. 12, 1992                                                               YEAST RNA POLYMERASE C53, HUMAN BN51, AND G1 ARREST                                  4319 <br />  <br />           A Chrs Noti Sfi1                                                                             participate transport C53, asso- <br />         chr4- &lt;                      B                                                                 ciated RNA polymerase C subunits, nucleus. <br />                                       Genetic Interval PD NPD T SM <br />                                                                                                           Putative regulatory signals seen sequences <br />                                                                                                        flanking RPCS3 coding sequence (Fig. 2). The 5&apos; non- <br />                                       RPC531URA31-rpb 1 66             0          2   1.5 <br />                                       (CMY274&apos;CMY323)                                                  coding region contains candidate TATA box sequences, <br />         620_ J <br />          kb       . -s <br />                   _.4 <br />                          _ <br />                                       RPC531URA3]-cdc9           58    0          5   4 <br />                                                                                                        16-bp sequence similar sequence the <br />                                       (CMY274 CMY388 <br />                                                    8                                                   RPC160 promoter, RPG box sequence, tandem <br />                              *:., <br />                                       cdc9-rpb   I               37     0         4   5                repetition PAC box. The RPG box upstream <br />                                                                                                        activating sequence genes (61), including those <br />        40kb-                                                                                           encoding proteins involved protein stable RNA syn- <br />                                                                                                        thesis. The RPG sequence bound GRF1/RAPITUF <br />               C                                                                                        protein (61). The PAC box sequence the <br />                              BA      CDC9- -:: RPCbS-           :-t RPB I                              promoter region 11 genes examined date encoding <br />                                                      Not]                                              subunits specific RNA polymerases A (I) C (III) with <br />                                                                                                        exception RPCJ60 (8). <br />    FIG. 4. Physical genetic mapping RPC53. (A) S. cerevi-                                          Physical genetic mapping RPC53 gene. The <br /> siae chromosomal DNA, (Chrs lane) digestion                                      RPC53 gene estimated lie 200 250 kb <br /> NotI SfiI (as indicated), separated pulsed-field gel                                        left telomere chromosome 4 S. cerevisiae by <br /> electrophoresis, DNA transferred nylon                                           chromosome fragmentation method mapping (see <br /> membrane probed radioactively labeled 1-kb PvuII-XbaI                                       Materials Methods). The presence rare NotI site in <br /> RPC53 fragment (Fig. 1C). This fragment contains sequences                                          RPCS3 gene allowed precise determination its <br /> either NotI site RPCS3 gene. The size                                  physical map position. RPC53 DNA hybridized NotI <br /> the hybridizing fragments identifies NotI site six <br /> NotI sites left end chromosome 4 (32). (B) Table                                           bands 40 620 kb Sfi band 635 kb <br /> summarizing tetrad analysis indicating genetic linkage                                   (Fig. 4A). This pattern identifies NotI site RPC53 as <br /> URA3 plasmid integrated RPCS3 chromosomal locus                                         NotI sites left telomere chromo- <br /> cdc9 rpbl mutations. Shown number tetrads                                       4 Link Olson NotI-SfiI map yeast <br /> case markers segregated parental ditypes (PD),                                     genome (32) places 210 kb left end the <br /> nonparental ditypes (NPD), tetratypes (T). The genetic distance                                     chromosome. Genetic mapping URA3 insertion the <br /> in centimorgans (cM) calculated according equation cM                                       RPC53 locus showed RPC53 locus tightly linked to <br /> = 50(T + 6 NPD)/(PD + NPD + T). (C) Diagram summarizing                                            CDC9 RPB1 (Fig. 4B). <br /> physical genetic mapping RPC53 gene.                                                          RPC53 encodes essential gene. Two different disruptions <br />                                                                                                        RPC53 gene constructed determine whether <br />                                                                                                        gene essential. In rpc53::HIS3-2 disruption, but <br /> were observed C53. The sequence KPSLK (amino acids                                                  approximately 60 nucleotides RPC53 coding sequence <br /> 25 29) form KPXXK, serves <span id='am-33' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span>                                              substituted HIS3 DNA fragment (Fig. 5A). A <br /> of nuclear targeting signal S. cerevisiae (16). Some                                         diploid strain heterozygous rpc53::HIS3-2 substitu- <br /> basic clusters protein sequences resembling                                           tion gave rise sporulation tetrads containing more <br /> other nuclear targeting signals (9, 12). These sequences                                           viable spores invariably auxotrophic for <br />  <br />  <br />                                      R                      H <br />                                                                             770 <br />                                                                                             ATG 130 <br />                                                                                                  A v <br />                                                                                                         440 <br />                                                                                                               G AA_ <br />                                                                                                                    207 <br />                                                                                                                           G <br />                                                                                                                                495 <br />                                                                                                                                      TAA X   300 <br />                                                                                                                                                    N <br />                                                                                                                            1                       I <br />                                            RPC53 <br />                                                             II <br />  <br />                                                             ___I_ <br />                                                                                                 160            H <br />                                     rpc53::HIS3-2                        ____HIS3I <br />                                                                       _I__                                                                  II <br />  <br />  <br />  <br />  <br />                                                                                           ATG                 GB          GB         TAAI <br />                                     rpc53::HIS3-1           L                                                 H           H <br />                                                                                                                    HIS3 <br />  <br />  <br />  <br />                                                             B                                     E <br />  <br />  <br />  <br />    FIG. 5. Effect disrupting RPC53 gene. (A) Shown different RPC53 disruption constructions. The rpcS3::HIS3-2 disruption <br /> deletes 60 bp RPC53 substitutes 1.7-kb BamHI HIS3 fragment (54). The rpc53::HIS3-1 disruption replaces the <br /> internal 207-bp RPCS3 BglII fragment 1.7-kb BamHI HIS3 fragment. Restriction sites defined legend Fig. 1. The <br /> orientation HIS3 gene indicated. (B) Tetrad dissection asci obtained sporulation CMY370 (a/at rpcS3::HIS3-21+). No more <br /> than viable spores obtained, auxotrophic histidine. The absence viable rpcS3::HIS+ spore colonies <br /> suggests deletion RPCS3 creates recessive lethal mutation. (C) Tetrad dissection asci obtained sporulation CMY242 (a/ax <br /> rpcS3::HIS3-1I+). Ten percent spores gave rise colonies growing considerably slowly sister-spore colonies <br /> after 10 days growth 23&#176;C. The slow-growing cells proved genotype rpcS3: :HIS3-1 [rho-] temperature sensitive <br /> for growth. <br />  4320     MANN ET AL.                                                                                           MOL. CELL. BIOL. <br />  <br /> histidine (Fig. 5B). Microscopic examination rpc53::       having achieved cell volumes larger normal (Fig. <br /> HIS3-2 spores showed germinated under-          7). <br /> gone cell division arresting unbudded cells.           We generated rpc53 point mutants determine a <br /> The absence His&apos; haploid spores suggested             G1 arrest general property associated inactivation of <br /> RPC53 deletion lethal. This possibility tested          RPC53 subunit specific pheno- <br /> cloning HindIII-NruI fragment containing RPC53            type rpc53::HIS3-1 [rho&apos;] disruption. The RPC53 gene <br /> gene (Fig. 1C) centromere vectors pUN20 (TRP1            pEMBLYc32 mutagenized vitro sodium <br /> SUP11) YCp50 (URA3) separately introducing           bisulfite, conditional mutants screened after <br /> each plasmid rpc53::HIS3-21+ diploid strain. After       plasmid shuffling strain CMY383 (rpc53::HIS3-2 [rho+]/ <br /> sporulation, haploid rpc53::HIS3-2IpUN20-RPC53 segre-             pUN20-RPC53) (see Materials Methods). One cold- <br /> gants readily obtained. These cells incapable        sensitive heat-sensitive rpc53 [rho&apos;] mutants were <br /> losing RPC53 plasmid determined plasmid-              obtained. In case, mutant cells arrested their <br /> sectoring assay based colony color         restrictive temperature cell divisions with <br /> inability grow YPD containing 2.5 M ethylene glycol,        65 85% unbudded cells. Thus, preferential arrest the <br /> which kills cells carrying SUPJ1 gene pUN20 (10).          G1 phase general consequence Rpc53 <br /> Thus, RPC53 essential gene. Unexpectedly,        inhibition. <br /> same RPC53 fragment complemented rpc53::HIS3-2              Synthesis tRNA inhibited rpc53 mutants. Incorpo- <br /> substitution present pUN20 unable            ration [3H]uracil total RNA (Fig. 6D) greatly <br /> when present YCp50. To investigate further,            affected transfer temperature-sensitive strain <br /> transferred HindIII-NruI RPC53 fragment YCp50            CMY397 (rpc53::HIS3-1 [rhoo]) 38&#176;C. As this <br /> to centromeric vector pEMBLYc32(URA3) (37). The               synthesis represented large rRNAs these <br /> YCp5O-RPC53 pEMBLYc32-RPC53 plasmids in-                 continuous labeling conditions, concluded that <br /> troduced strain CMY383 (rpc53::HIS3-2IpUN20-                 RNA polymerase A (I) functional rpc53::HIS3-1 <br /> RPCS3), transformants tested ability        [rhoo] strain. The C53 protein specifically associated with <br /> lose resident pUN20-RPCS3. By plasmid-shuffle            RNA polymerase C, anti-C53 antibodies inhibit in <br /> test, pEMBLYc32-RPC53 capable complementing                vitro transcription tRNA gene (19). We examined <br /> the rpcS3::HIS3-2 mutation, YCp5O-RPCS3             RNA polymerase C transcription affected in <br /> not. The 2.4-kb HindIII-NruI fragment, containing 760 bp          rpc53 mutants labeling RNA 60 min [3H]uracil <br /> upstream 360 bp downstream RPCS3 coding                extracting electrophoretically separating the <br /> sequence, appears contain promoter termina-        small RNAs polyacrylamide gels. Transfer RNA synthe- <br /> tion signals necessary RPCS3 expression (Fig. 2). Thus,       sis severely inhibited CMY397 (rpc53::HIS3-1 [rhoo]) <br /> we believe RPCS3 expression inhibited          cells 6 h incubation 38&#176;C comparison the <br /> unknown manner presented context          CMY396 (RPC53+ [rhoo]) congenic control strain (Fig. 9A <br /> was inserted YCp50. Plasmid context effects gene            B). In contrast, 5S RNA, transcribed RNA <br /> expression observed yeast TRPI         polymerase C, polymerase A-derived 5.8S RNA <br /> HIS3 genes (53).                                                  little affected rpc53::HIS3-1 strain over <br />    A disruption RPC53 gene constructed         period. <br /> replacing 207-bp BglII fragment encoding amino acids 193            We examined synthesis small RNAs the <br /> to 259 HIS3 DNA. Sporulation diploids heterozygous        strain CMY381 (rpc53::HIS3-2 [rho+])IpEMBLYc32-rpc53- <br /> for rpc53::HIS3-1 disruption occasionally gave rise       ts28. A specific inhibition tRNA synthesis seen this <br /> slow-growing His&apos; colonies germination 23&#176;C (Fig.        strain 30 min transfer 37&#176;C (Fig. 9C). Thus, <br /> 5C). Southern blot analysis confirmed His&apos; cells         temperature-sensitive inactivation RPC53 caused either <br /> contained disrupted form RPC53 gene (data         insertional disruption point mutations leads an <br /> not shown). Surprisingly, rpc53::HIS3-1 haploids       inhibition tRNA gene transcription RNA polymerase <br /> always [rho-] (lacking functional mitochondrial genome)         C. A similar preferential inhibition tRNA compared with <br /> and temperature sensitive growth. A detailed anal-      5S RNA synthesis observed yeast RNA <br /> ysis [rho-I phenotype rpc53::HIS3-1 disruption      polymerase C mutants examined date. These include <br /> will described separate communication (5). Below       mutants 5 13 subunits compose C <br /> describe cell division arrest associated temper-     enzyme (see Discussion). <br /> ature-sensitive phenotype.                                           Sequence similarity yeast RPC53 human <br />    rpc53 mutants preferentially arrest G1 phase. When          BN5J gene products. One protein significant similarity <br /> CMY397 (rpc53::HIS3-1 [rhoo]) cells growing 23&#176;C          C53 searching EMBL data library <br /> transferred 38&#176;C, cell number increased threefold     (release 29, December 1991) TFASTA program (40, <br /> before division arrest (Fig. 6A). In contrast, optical        41). The carboxy-terminal 136 amino acids human <br /> density culture (Fig. 6C) incorporation [3H]        BN51 protein (22) yeast C53 share 25% identity 78% <br /> uracil total RNA [35S]methionine total protein      similarity conservative substitutions dictated the <br /> increased steadily 13 h incubation (Fig. 6D).    PAM250 matrix (40) allowed. The FASTA similarity <br /> A congenic control strain, CMY396 rpc53::HIS3-1 [rho0]!           score 166 carboxy-terminal region two <br /> pEMBLYc32-RPC53, grew divided continuously                 proteins statistically significant; match 8 standard <br /> 38&#176;C. Most mutant cells arrested division         deviations mean comparison C53 <br /> large, unbudded cells single nucleus (Fig. 6B 7).      shuffled BN51 sequences RDF2 program (40, 41). <br /> The morphology cells suggested              Weaker similarities seen amino-terminal regions <br /> arrested G1 phase cell cycle, analysis       proteins (Fig. 10). Furthermore, overall size <br /> flow cytometry confirmed cells G1 content      (395 amino acids BN51, 424 amino acids C53) and <br /> DNA (Fig. 8). The mutant cells retained good viability (85%)      overall hydrophilicity (compare Fig. 3B C) two <br /> over 12 h restrictive temperature despite   proteins similar. A direct test functional homology <br />  VOL. 12, 1992                                                  YEAST RNA POLYMERASE C53, HUMAN BN51, AND G1 ARREST                               4321 <br />  <br />  <br />  <br />  <br />                              44                 //-m           ut                  X         |                               Ur-wt <br />  <br />                          o                                                             o <br />                                                                                        100                                   Met-mut <br />                     co <br />                                                                                                                              Ur-mut <br />  <br />  <br />                              0                                                         ~~~~~~~~~~~~~~~~~0 <br />                               0    2   4   6    8    10   12   14   16   18   20             0   2   4   6     8   10   12   14   16   18   20 <br />                                                Hours 38&#176;C                                                 Hours 38&#176;C <br />   FIG. 6. Characterization temperature-sensitive phenotype associated rpc53::HIS3-1 disruption mutation. CMY397 <br /> rpc53::HIS3-1 [rho&apos;] (designated mut figure) isogenic CMY396 rpc53::HIS3-1 [rho0]/pEMBLYc32-RPC53 (designated wt) cells <br /> were grown synthetic complete medium 23&#176;C early log phase transferred 38&#176;C times indicated figure. <br /> (A) Relative increase cell number. Cultures wild-type mutant cells shifted 38&#176;C initial value 2 x 106 cells ml. <br /> (B) Percentage unbudded cells. The initial accumulation unbudded cells hours transfer 38&#176;C transient <br /> heat shock inhibition cell cycle G1 phase (28). Note wild-type mutant cells recovered heat shock effect <br /> but mutant cells rearrested unbudded cells end division. (C) Relative increase optical density (OD) <br /> of cultures. Mutant wild-type cell cultures transferred 38&#176;C A6. 0.15. (D) Continuous labeling total cellular protein <br /> and RNA. [35S]methionine [3H]uracil added cell cultures 1 h transfer 38&#176;C. Incorporation radioactive precursors was <br /> followed trichloroacetic acid (TCA) precipitation. Note mutant cells continued accumulate protein RNA arrest <br /> period. <br />  <br /> between proteins attempted placing BN51                             C53 mutant proteins temperature sensitive their <br /> human cDNAs control yeast GAL1O pro-                              function stability associate other <br /> moter multicopy plasmid (see Materials Methods).                          RNA polymerase C subunits. Mutants uniquely <br /> No complementation lethal rpc53::HIS3-2 disruption                          temperature sensitive assembly RNA polymerase <br /> was observed entire BN51-coding sequence                           C, rpc4O-ts mutants (33), exhibit decreased levels <br /> or portion encoding 167 amino acids                           transcription generations growth at <br /> protein (the region exhibiting similarity C53                        restrictive temperature. The genetic data con- <br /> and corresponding functionally essential portion                         sistent antibody inhibition studies suggesting a <br /> C53 [5]) expressed GAL promoter. However,                            direct role C53 subunit tRNA synthesis. Analysis <br /> preliminary results indicated BN51 protein                           vitro transcription RNA polymerase C purified from <br /> does accumulate yeast strains, work                       C53 mutants permit precise delineation its <br /> will necessary definitive conclusion drawn                      role transcriptional process (5). <br /> regarding capacity BN51 functionally replace C53.                          A preferential inhibition tRNA compared 5S RNA <br />                                                                                    synthesis vivo property RNA polymerase C <br />                                   DISCUSSION                                       mutants examined date. Aside C53 (Fig. 9), this <br />                                                                                    includes mutants C160 (15), C82 (6), C34 (52), C31 <br />   RPC53 encodes RNA polymerase C-specific subunit                               (37) subunits. In C31 subunit, nonsense mutation <br /> required tRNA gene transcription. A role C53 RNA                        weakly suppressed tRNA inserting homologous <br /> polymerase C function previously exper-                          amino acid led specific inhibition tRNA synthesis. <br /> iments anti-C53 antibodies seen inhibit                           Thus, simple reduction the wild-type <br /> tRNA gene transcription vitro (19). In report,                          RNA polymerase C preferentially affects tRNA compared <br /> show functional C53 protein required yeast cell                      5S RNA synthesis. On hand, of <br /> viability inactivation C53 temperature-sensitive                       rpc160 rpc53 mutant phenotypes an <br /> mutants rapidly leads inhibition tRNA gene tran-                          enhanced thermolability mutant polymerase. There <br /> scription vivo. The rapidity inhibition                           likely possibilities explain heightened <br /> rpc53::HIS3-1 rpc53-ts28 mutants (Fig. 9) indicates                       resistance 5S RNA synthesis compared tRNA syn- <br />  4322     MANN ET AL.                                                                                                       MOL. CELL. BIOL. <br />  <br />  <br />        R) 230                                                                B) 370- 6h <br />  <br />  <br />  <br />  <br />        C) 370- 12 h                                                          D) 370- 24 h <br />  <br />  <br />  <br />  <br />        E) DFPI; 370- 10 h                                                    F) 380- 24 h <br />  <br />  <br />  <br />  <br />   FIG. 7. Morphology CMY397 rpc53::HIS3-1 temperature-arrested cells. (A D) Log-phase CMY397 cells YPD 23&#176;C (A 0 = 0.17; <br /> 2 x 106 cells ml; 47% unbudded cells) transferred 37&#176;C periods indicated figure. At time point, aliquot of <br /> cells fixed 3.7% formaldehyde examined differential interference contrast microscopy. (E) DAPI (4&apos;,6&apos;-diamidino-2- <br /> phenylindole) staining CMY397 DNA cells incubated 10 h 37&#176;C. Cells fixed 70% ethanol stained with <br /> 0.5 ,ug DAPI ml. The arrested cells exhibited single nucleus. No punctate cytoplasmic staining seen [rho&apos;] cells lack <br /> mitochondrial DNA. (F) Terminal morphology CMY397 cells 24 h incubation YPD 38&#176;C. The mutant cells arrest division <br /> and lyse rapidly 38&#176;C 37&#176;C. All cells figure shown magnification (approximately x880). <br />  1. <br /> E <br /> z <br /> S <br /> = <br />      VOL. 12, 1992 <br />  <br />  <br />  <br />  <br />       7 <br />      X=, <br />  <br />  <br />  <br />         -I <br />              A. <br />              A <br />  <br />  <br />  <br />  <br />               B <br />  <br />  <br />  <br />               w <br />  <br />  <br />  <br />  <br />              1flS. <br />                   Gl <br />                   56% <br />  <br />  <br />  <br />  <br />                       ... .... .... <br />  <br />  <br />  <br />  <br />                      Gl <br />                       75% <br />  <br />  <br />  <br />                      .^ <br />  <br />  <br />  <br />  <br />                      40 <br />                                       Mut 23- 54%1UB <br />  <br />  <br />  <br />  <br />                                       - <br />                                         G2 <br />                                         44% <br />  <br />  <br />  <br />  <br />                                       mut 38 90% UB <br />  <br />  <br />  <br />  <br />                                       T,$ <br />  <br />                              Fluorescence <br />                                             G2 <br />                                             c <br />                                                -lOh <br />  <br />  <br />  <br />  <br />                                             ~~25 <br />                                                    1 22 17i <br />  <br />  <br />  <br />  <br />                                                    120 <br />                                                               i <br />  <br />  <br />  <br />  <br />                                                               160 <br />                                                                   0 <br />  <br />  <br />  <br />  <br />         FIG. 8. Flow cytometry quantify DNA content yeast <br />      cells. CMY397 rpc53::HIS3-1 [rho&apos;] cells (mut figure) YPD <br />      23&#176;C (A) 10 h incubation 38&#176;C (B) fixed 70% <br />      ethanol stained propidium iodide FACS analysis. Also <br />      shown CMY396 rpc53::HIS3-1 [rho0]/pEMBLYc32-RPC53 con- <br />      trol cells (wt figure) log-phase growth 38&#176;C YPD. (C) <br />      Plot number cells (vertical <span id='am-46' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-47' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50'  ></span>axis</span>) emitting given arbitrary <br />      unit fluorescence (horizontal axis) proportion DNA <br />      content cell. The peaks labeled G1 G2 correspond to <br />      peaks haploid diploid content DNA. S-phase cells are <br />      poorly resolved G1 G2 peaks analysis, no <br />      attempt define subpopulation cells. It is <br />      clear CMY397 mutants arrest cell division <br />      preferentially G1 content DNA incubation 38&#176;C. No <br />      accumulation G1 cells seen CMY396 RPC53+ cells <br />      grown 38&#176;C. We consistently observed slightly lower percentage <br />      Gl-phase cells, determined FACS analysis, unbud- <br />      ded (UB) cells, seen light microscopy, CMY397 tempera- <br />                                                                         17j <br />  <br />  <br />  <br />  <br />                                                                       280 <br />  <br />  <br />  <br />  <br />      ture-arrested mutants. This mean fraction unbud- <br />                                                                               -    C <br />  <br />  <br />  <br />  <br />                                                                                   II.. <br />                                                                                          Gl <br />                                                                                          46% <br />  <br />  <br />  <br />  <br />                                                                                          40 <br />                                                                                                        YEAST RNA POLYMERASE C53, HUMAN BN51, AND G1 ARREST <br />  <br />  <br />  <br />  <br />                                                                                                  Wt 38c 36%UB <br />  <br />  <br />  <br />  <br />                                                                                                  54% <br />  <br />  <br />  <br />  <br />                                                                                                   es <br />  <br />  <br />  <br />  <br />                                                                                               Fluorescence <br />  <br />                                                                                                                    genes <br />                                                                                                                              23&apos; 3h <br />  <br />  <br />  <br />  <br />                                                                                                                              _ <br />                                                                                                                                  R <br />  <br />  <br />  <br />                                                                                                                                  S <br />                                                                                                                                      RPC53+ <br />                                                                                                                                       389 389 <br />  <br />  <br />  <br />  <br />                                                                                                                                        ._ <br />                                                                                                                                             6h <br />                                                                                                                                             w     w@ . . =-S1B <br />                                                                                                                                                  io <br />  <br />                                                                                                                                                 --_ <br />                                                                                                                                                       B <br />                                                                                                                                                           rpc53::HIS3-1 <br />                                                                                                                                                    23&apos; 3..23&apos; 39&apos; <br />                                                                                                                                                           23&apos; <br />  <br />  <br />                                                                                                                                                           , <br />  <br />  <br />  <br />  <br />                                                                                                                                                           am <br />                                                                                                                                                                 38&#176; 38&apos; <br />                                                                                                                                                                 3h <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 * <br />                                                                                                                                                                  U <br />  <br />  <br />  <br />  <br />                                                                                                                    CMY396 rpc53::HIS3-1/pEMBLYc-RPC53 (A) CMY397 <br />                                                                                                                                                                      6h <br />  <br />                                                                                                                                                                       &apos; <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      -m <br />                                                                                                                                                                      __ <br />                                                                                                                                                                           c <br />  <br />  <br />  <br />  <br />                                                                                                                       FIG. 9. Synthesis small RNA molecules rpc53 mutants. <br />                                                                                                                    rpc53::HIS3-1 (B) cells grown synthetic complete medium <br />                                                                                                                    labeled [3H]uracil 60 min 23 38&#176;C 3 6 h of <br />                                                                                                                    incubation. Small RNAs extracted, 20 p1g RNA from <br />                                                                                                                    sample analyzed electrophoresis 6% polyacryl- <br />                                                                                                                    amide gel fluorography. (C) Small RNA synthesis was <br />                                                                                                                    examined CMY rpcS3-ts28 point mutants [3H]uracil <br />                                                                                                                    labeling 60 min 23&#176;C 30 min incubation 37&#176;C. In <br />                                                                                                                    instances, specific inhibition tRNA accumulation was <br />                                                                                                                    observed rpcS3 mutants restrictive temperature. <br />  <br />  <br />                                                                                                                    preferentially transcribed relative nucleolar 5S RNA <br />                                                                                                                           (59). It interesting determine <br />                                                                                                                    preference remained conditions RNA <br />                                                                                                                    polymerase C limitation. This model predicts tRNA <br />                                                                                                                    5S RNA synthesis equally affected RNA <br />                                                                                                                    polymerase C mutations eukaryotes 5S RNA <br />                                                                                                                    genes localized nucleolus. <br />                                                                                                                       Preferential G1 arrest rpc53 mutants. An unexpected <br />                                                                                                                    phenotype rpc53 mutants rapid prefer- <br />                                                                                                                                                                               RPC53. ts28 <br />  <br />  <br />  <br />  <br />                                                                                                                    ential arrest G1 phase cell cycle large, round, <br />                                                                                                                    unbudded cells (Fig. 6 8). The arrested cells maintain high <br />                                                                                                                    levels protein synthesis viability restrictive <br />                                                                                                                    temperature exhibit shmoo morphology <br />                                                                                                                    typical cdc28 mutants arrested G1 phase Start <br />                                                                                                                    (18). Rather, phenotype closely resembles certain <br />                                                                                                                    translational machinery mutants (prtl/cdc63; translation <br />                                                                                                                    initiation factor [17], mesl; methionyl-tRNA synthetase [57, <br />                                                                                                                                                                                , <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                &quot; <br />                                                                                                                                                                                    370      370 <br />  <br />  <br />                                                                                                                                                                                              I <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                          ,,,M. <br />                                                                                                                                                                                                   25619$8 <br />                                                                                                                                                                                                   5.8S <br />                                                                                                                                                                                                   5.9 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                    ItRNs <br />                                                                                                                                                                                                             4323 <br />  <br />  <br />  <br />  <br />      ded cells arrest S/G2 content DNA,                                             58]) G1 arrest associated partial <br />      experimental error classification budded versus unbudded                                            inhibition protein synthesis. It believed, having <br />      G1 versus G2 cells.                                                                                        demonstrated, partial inhibition protein syn- <br />                                                                                                                    thesis lead specific arrest G1 preferentially <br />                                                                                                                    affecting accumulation unstable polypeptides (such as <br />                                                                                                                    G1 cyclins) accumulation necessary the <br />      thesis conditions RNA polymerase C limitation.                                                       passage Start G1 (44). Similarly, G1 arrest the <br />      Transcription 5S RNA genes requires protein TFIIIA                                                     rpc53 mutants result partial inhibition of <br />      factors TFIIIB TFIIIC, tRNA                                                        translation secondary consequence inhibition of <br />      gene transcription requires TFIIIB TFIIIC                                                         stable RNA transcription RNA polymerase C. An alter- <br />      TFIIIA (11, 13). Perhaps RNA polymerase C higher                                                        native hypothesis C53 subunit required the <br />      affinity thermostability associated TFIIIA,                                                     transcription RNA polymerase C nonabundant RNA <br />      -B, -C compared TFIIIB -C alone. Under                                                      <span id='am-51' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> required passage G1 phase. <br />      hypothesis, expected purified compo-                                                    Based drug inhibition studies, previously proposed <br />      nents, greater affinity thermoresistance                                                       transcription RNA species necessary <br />      5S RNA compared tRNA gene transcription                                                        performance Start yeast cells (27, 60). The U6 <br />      vitro. A second possibility suggested exceptional                                                small nuclear RNA nuclear RNase P RNA (RPR1) <br />      localization 5S RNA genes nucleolus S.                                                       genes genes apart tRNA 5S <br />      cerevisiae (43). RNA polymerase C higher                                                           genes presently known transcribed RNA <br />      affinity 5S RNA genes nucleolus                                                        polymerase C S. cerevisiae (31, 36). The transcription of <br />      stable nucleoplasm. However,                                                     genes inhibited rpc160-41 mutant (31, 36). <br />      transfer marked 5S RNA gene autonomously                                                           Interestingly, reexamination rpcl6O-41 mutant has <br />      replicating plasmid yeast cells showed plasmid-                                                   shown accumulates large, round, unbudded cells at <br />      borne 5S RNA genes (presumably nucleoplasmic)                                                            37&#176;C rpc53 mutants (55). A particular tRNA, the <br />  4324          MANN ET AL.                                                                                                                 MOL. CELL. BIOL. <br />  <br />   R                                                                                    hamster cells, tRNA 5S RNA syntheses <br />                                                                                        (products transcription RNA polymerase C) inhibited <br />                                                                                        15 h restrictive temperature, the <br />                                                                                        synthesis 5.8S RNA. This RNA derived by <br />                                                                                        nucleolytic cleavage 45S rRNA precursor transcribed <br />           z                                                                            RNA polymerase A (I). It remains possible RNA <br />                                                                                        polymerase C inhibition primary defect BN51 <br />                                                                                        mutant secondarily maturation 35S RNA <br />                                                                                        affected depletion 5S RNA RNAs <br />                                                                                        necessary rRNA processing. Indeed, observed that <br />                                                                                        accumulation stable RNA inhibited 12 h of <br />                                                                                        incubation restrictive temperature rpcS3:: <br />                                            C53                                         HIS3-1 mutant (data shown), tRNA synthe- <br />   B                                                                                    sis preferentially inhibited 6 h incubation (Fig. <br />        25.0%    identity 136      <span id='am-88' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-91'  ></span><span id='am-92' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>aa</span>    overlap; init:     102, opt:    166         9). Finally, unable demonstrate complementation <br />                  300          310             320           330           340    350   rpcS3 mutants expression BN51 gene yeast <br />        C53:    LKKMN-NKPERFMVFQLPTRLPAFER-PAVKEEKEDMETQASDP--SKKKKNIKKKDTKD <br />                                                                                        cells. Likewise, expression RPC53 gene mutant <br />        BN51   :LRELSLTKEEELLFLQLPDTLPGQPPTQDIKPIKTEVQGEDGQWLIKQEKDREA}LAEN             BN51 hamster cells unable complement temper- <br />                        260          270             280       290         300    310 <br />  <br />                         360          370             380          390      400 <br />                                                                                        ature-sensitive defect (21). Although yeast mammalian <br />                ALSTRELA-GKVGSIRVHKSGKLSVKIGNVVMIGKGAETTFLHDVIALSIADD-ASSAE <br />                : .   :. X.: : .. .:::.. <br />                                    .   .   .: . :... <br />                                                   . . .. <br />                                                        : :: .. .. ... X.X <br />                                                                                        homologs interchangeable, always <br />                ACTLADLTEGQVGKLLIRKSGRVQLLLGKVTLDVTMGTACSFLQELVSVGLGDSRTGEMT <br />                        320          330             340       350         360    370 <br />                                                                                        case. For example, human fly TFIID unable <br />                                                                                        supply essential function homologous S. cerevi- <br />           410          420 <br />                LLGRVDGKIVVTPQI                                                         siae protein (7, 14). Functional incapacity due <br />               VLGHVKHKLVCSPDFE <br />                                                                                        small number amino acid differences. The substitution <br />                        380                                                             single amino acid human fibrillarin sequence <br />   FIG. 10. Sequence similarity yeast C53                           exists yeast fibrillarin sufficient correct a <br /> human BN51 proteins. (A) PLFASTA analysis showing regions                           temperature-sensitive defect ability human <br /> similarity proteins. The numbers                          protein complement yeast mutant (25). Polypeptides <br /> diagonal line indicate optimized FASTA score different                     approximately molecular weight determined the <br /> local regions similarity. (B) Shown FASTA alignment (ktup                    BN51 protein associated highly <br /> = 1) carboxy-terminal regions C53 BN51 proteins. <br />  <br /> Evolutionarily conservative amino acid substitutions (PAM250 <br />                                                                                        purified preparations RNA polymerase C mouse <br /> scores zero greater) indicated single dot, contrast <br />                                                                                        plasmacytoma (51) human KB (23) cells. It remains be <br /> to double dots identical amino acids. The Xs indicate                      seen correspond BN51 protein. <br /> boundaries region highest local similarity between <br /> the C53 BN51 proteins. These analyses performed                                            ACKNOWLEDGMENTS <br /> FASTA program package (40, 41). <br />                                                                                          A study C.M. Burroughs- <br />                                                                                        Wellcome postdoctoral fellow Life Sciences Research Foun- <br />                                                                                        dation. This work supported research funding grant SCI- <br /> U6 RPR1 RNAs, as-yet-<span id='am-104' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-105' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-111'  ></span><span id='am-112' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-114' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span>unidentified</span> RNA species                                 CT91-0702 European Economic Community. <br />                                                                                          C.M. thanks Lee Hartwell hospitality. We thank Michael <br /> transcribed C enzyme preferentially required                             Ittmann collaboration Pierre Thuriaux initially <br /> in G1 phase.                                                                       bringing attention BN51 similarity, thank them <br />    Homology yeast C53 mammalian BN51                                   Ben Hall critical reading manuscript. <br /> proteins. A search protein sequence data <br /> banks C53 sequence revealed significant degree                                                        REFERENCES <br /> sequence similarity protein, human BN51                               1. Allison, L. A., M. Moyle, M. Shales, C. J. Ingles. 1985. <br /> protein (21, 22). These proteins 25% amino acid iden-                                 Extensive homology largest subunits eukaryotic <br /> tity 78% similarity (allowing conservative amino acid                               prokaryotic RNA polymerases. Cell 42:599-610. <br /> substitutions PAM250 [40] score zero greater)                              2. Baldari, C., G. Cesareni. 1985. Plasmids pEMBLY: new <br /> the 136 carboxy-terminal amino acids proteins                                   single-stranded shuttle vectors recovery analysis of <br /> (Fig. 10). Several points argue sequence similarities                             yeast DNA sequences. Gene 35:27-32. <br /> between proteins represent functional homology.                               3. Boeke, J. D., F. Lacroute, G. R. Fink. 1984. A positive <br /> The proteins similar size hydrophilicity,                               selection mutants lacking orotidine-5&apos;-phosphate decarbox- <br />                                                                                            ylase activity yeast: 5-fluoro-orotic acid resistance. Mol. Gen. <br /> both localized nucleus. The carboxy-terminal                                    Genet. 197:345-346. <br /> region proteins highest level sequence                            4. Caries, C., I. Treich, F. Bouet, M. Riva, A. Sentenac. 1991. <br /> similarity, region corresponds function-                              Two additional common subunits, ABC10a ABC10p, are <br /> ally essential portion C53 subunit (5). The BN51                                    shared yeast RNA polymerases. J. Biol. Chem. 266:24092- <br /> protein originally identified ability cDNA                            24096. <br /> complement temperature-sensitive G1-phase arrest                               5. Chiannilkulchai, N., A. Moenne, A. Sentenac, C. Mann. <br /> BHK (baby hamster kidney) cell mutant. Similarly, rpcS3                                    Biochemical genetic dissection S. cerevisiae RNA <br /> mutations yeast cells lead preferential G1 arrest.                                 polymerase C53 subunit analysis mitochondri- <br />                                                                                            ally mis-sorted mutant construct. J. Biol. Chem., press. <br /> Finally, tRNA synthesis thermolabile vivo                             6. Chiannilkulchai, N., R. Stalder, M. Riva, C. Caries, M. Werner, <br /> vitro temperature-sensitive rpc53 (Fig. 9) (5)                                 A. Sentenac. RPC82 encodes highly conserved third- <br /> BN51 cell mutants (21). However, possible                                 largest subunit RNA polymerase C (III) Saccharomyces <br /> conclude BN51 protein mammalian homolog                                    cerevisiae. Mol. Cell. Biol. 12:4433-4440. <br /> of C53 subunit. In temperature-sensitive BN51 mutant                                7. Cormack, B. P., M. Strubin, A. S. Ponticelli, K. Struhl. <br />  VOL. 12, 1992                                           YEAST RNA POLYMERASE C53, HUMAN BN51, AND G1 ARREST                               4325 <br />  <br />       1991. Functional differences yeast human TFIID              tion apparatus. Mol. Cell. Biol. 9:5387-5394. <br />       localized highly conserved region. Cell 65:341-348.      30. Kolodziej, P., R. A. Young. 1991. Mutations three <br />  8.   Dequard-Chablat, M., M. Riva, C. Caries, A. Sentenac.               largest subunits yeast RNA polymerase II affect enzyme <br />       1991. RPC19, gene subunit common yeast RNA                 assembly. Mol. Cell. Biol. 11:4669-4678. <br />       polymerase A (I) C (III). J. Biol. Chem. 266:15300-15307.       31. Lee, J.-Y., C. F. Evans, D. R. Engelke. 1991. Expression of <br />  9.   Dingwall, C., R. A. Laskey. 1991. Nuclear targeting se-             RNase P RNA Saccharomyces cerevisiae controlled an <br />       quences-a consensus? Trends Biochem. Sci. 16:478-481.                   unusual RNA polymerase III promoter. Proc. Natl. Acad. Sci. <br /> 10.   Elledge, S. J., R. W. Davis. 1988. A <span id='am-30' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-31' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> versatile            USA 88:6986-6990. <br />       centromeric vectors designed use sectoring-shuffle       32. Link, A. J., M. V. Olson. 1991. Physical map the <br />       mutagenesis assay Saccharomyces cerevisiae. Gene 70:303-             Saccharomyces cerevisiae genome 110-kilobase resolution. <br />       312.                                                                    Genetics 127:681-698. <br /> 11.   Gabrielsen, 0. S., A. Sentenac. 1991. RNA polymerase C          33. Mann, C., J.-M. Buhler, I. Treich, A. Sentenac. 1987. <br />       (III) transcription factors. Trends Biochem. Sci. 16:412-       RPC40, unique gene subunit shared yeast RNA <br />       416.                                                                    polymerases A C. Cell 48:627-637. <br /> 12.   <span id='am-1' about='obo:NCBITaxon_316718' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3646 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_235631 obo:NCBITaxon_3977 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_235892 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Garcia" datatype="xsd:string"></span>Garcia</span>-Bustos, J., J. Heitman, M. N. Hall. 1991. Nuclear        34. Marck, C. 1988. &quot;DNA strider&quot;: &quot;C&quot; program fast <br />       protein localization. Biochim. Biophys. Acta 1071:83-101.               analysis DNA protein sequences Apple Macintosh <br /> 13.   Geiduschek, P. E., G. P. Tocchini-Valentini. 1988. Tran-            family computers. Nucleic Acids Res. 16:1829-1836. <br />       scription RNA polymerase III. Annu. Rev. Biochem. 57:            35. Micouin, J.-Y. 1988. Clonage et etude du gene RPC53, codant <br />       879-914.                                                                pour la sous-unite &quot;C53&quot; l&apos;ARN polym6rase C la levure <br /> 14.   Gill, G., R. Tjian. 1991. A highly conserved domain              Saccharomyces cerevisiae. Ph.D. thesis. Universite <span id='am-81' about='obo:NCBITaxon_49669' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_50362 obo:NCBITaxon_4447 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_4667 obo:NCBITaxon_3193'><span id='am-82' property="rdfs:label" content="Paris" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-84'  ></span><span id='am-85' property="oboInOwl:hasRelatedSynonym" content="Paris L." datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasAlternativeId" content="NCBITaxon:49665" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Paris</span>- <br />       TFIID displays species specificity vivo. Cell 65:333-340.            Sud, Centre d&apos;Orsay, France. <br /> 15.   Gudenus, R., S. Mariotte, A. Moenne, A. Ruet, S. Memet, J. M.       36. Moenne, A., S. Camier, G. Anderson, F. Margottin, J. Beggs, <br />       Buhler, A. Sentenac, P. Thuriaux. 1988. Conditional mu-             A. Sentenac. 1990. The U6 gene Saccharomyces cerevi- <br />       tants RPC160, gene encoding largest subunit RNA           siae transcribed RNA polymerase C (III) vivo in <br />       polymerase C Saccharomyces cerevisiae. Genetics 119:517-             vitro. EMBO J. 9:271-277. <br />       526.                                                                37. Mosrin, C., M. Riva, M. Beltrame, E. Cassar, A. Sentenac, and <br /> 16.   Hall, M. N., L. Hereford, I. Herskowitz. 1984. Targeting         P. Thuriaux. 1990. The RPC31 gene Saccharomyces cerevi- <br />       P-galactosidase nucleus yeast. Cell 36:1057-1065.             siae encodes subunit RNA polymerase C (III) acidic <br /> 17.   Hanic-Joyce, P. J., G. C. Johnston, R. A. Singer. 1987.             tail. Mol. Cell. Biol. 10:4737-4743. <br />       Regulated arrest cell proliferation mediated yeast prtl       38. Nasmyth, K. A., S. I. Reed. 1980. Isolation genes by <br />       mutations. Exp. Cell Res. 172:134-145.                                  complementation yeast: molecular cloning cell-cycle <br /> 18.   Hartwell, L. H., R. K. Mortimer, J. Culotti, M. Culotti.            gene. Proc. Natl. Acad. Sci. USA 77:2119-2123. <br />       1973. Genetic control cell division cycle yeast. V.       39. Orr-Weaver, T. L., J. W. Szostak, R. J. Rothstein. 1981. <br />       Genetic analysis cdc mutants. Genetics 74:267-286.                   Yeast transformation: model study recom- <br /> 19.   Huet, J., M. Riva, A. Sentenac, P. Fromageot. 1985. Yeast           bination. Proc. Natl. Acad. Sci. USA 78:6354-6358. <br />       RNA polymerase C subunits. Specific antibodies           40. Pearson, W. R. 1990. Rapid sensitive sequence comparison <br />       structural functional probes. J. Biol. Chem. 260:15304-             FASTP FASTA. Methods Enzymol. 183:63-98. <br />       15310.                                                              41. Pearson, W. R., D. J. Lipman. 1988. Improved tools for <br /> 20.   Ito, H., Y. Fukuda, K. Murata, A. Kimura. 1983. Transfor-           biological sequence comparison. Proc. Natl. Acad. Sci. USA <br />       mation intact yeast cells treated alkali cations. J.            85:2444-2448. <br />       Bacteriol. 153:163-168.                                             42. Peden, K. W. C., D. Nathans. 1982. Local mutagenesis <br /> 21.   Ittmann, M., J. Ali, A. Greco, C. Basilico. The gene                deletion loops DNA heteroduplexes. Proc. Natl. <br />       complementing temperature-sensitive cell cycle mutant              Acad. Sci. USA 79:7214-7217. <br />       BHK cells human homolog yeast RPC53 gene,         43. Philippsen, P., M. Thomas, R. A. Kramer, R. W. Davis. <br />       encodes subunit RNA polymerase III. Submitted            1978. Unique arrangement coding sequence 5S, 5.8S, 18S <br />       publication.                                                            25S ribosomal RNA Saccharomyces cerevisiae deter- <br /> 22. Ittmann, M., A. Greco, C. Basilico. 1987. Isolation            mined R-loop hybridization analysis. J. Mol. Biol. <br />     human gene complements temperature-sensitive cell                  123:387-404. <br />     cycle mutation BHK cells. Mol. Cell. Biol. 7:3386-3393.            44. Reed, S. I. 1991. Gl-specific cyclins: search S-phase- <br /> 23. Jaehning, J. A., P. S. Woods, R. G. Roeder. 1977. Purifica-           promoting factor. Trends Genet. 3:95-99. <br />     tion, properties, subunit structure deoxyribonucleic           45. Riva, M., S. Memet, J.-Y. Micouin, J. Huet, I. Treich, J. Dassa, <br />     acid-dependent ribonucleic acid polymerase III uninfected            R. Young, J.-M. Buhler, A. Sentenac, P. Fromageot. 1986. <br />     adenovirus 2-infected KB cells. J. Biol. Chem. 252:8762-              Isolation structural genes yeast RNA polymerases by <br />     8771.                                                                     immunological screening. Proc. Natl. Acad. Sci. USA 83:1554- <br /> 24. James, P., S. Whelen, B. D. Hall. 1991. The RETI gene              1558. <br />     yeast encodes second-largest subunit RNA polymerase            46. Rose, M. D., P. Novick, J. H. Thomas, D. Botstein, G. R. <br />     III. Structural analysis wild-type retl-l mutant               Fink 1987. A Saccharomyces cerevisiae genomic plasmid bank <br />     alleles. J. Biol. Chem. 266:5616-5624.                                    based centromere-containing shuttle vector. Gene 60:237- <br /> 25. Jansen, R. P., E. C. Hurt, H. Kern, H. Lehtonen, M. Carmo-                243. <br />     Fonseca, B. Lapeyre, D. Tollervey. 1991. Evolutionary             47. Rothstein, R. J. 1983. One-step gene disruption yeast. Meth- <br />     conservation human nucleolar protein fibrillarin           ods Enzymol. 101:202-211. <br />     functional expression yeast. J. Cell. Biol. 113:715-729.           48. Russel, M., S. Kidd, M. R. Kelley. 1986. An improved <br /> 26. Johnson, S. L. 1991. Structure function analysis yeast         filamentous helper phage generating single-stranded plasmid <br />     CDC4 gene product. Ph.D. thesis. University Washington,                DNA. Gene 45:333-338. <br />     Seattle.                                                              49. Sentenac, A. 1985. Eukaryotic RNA polymerases. Crit. Rev. <br /> 27. Johnston, G. C., R. A. Singer. 1978. RNA synthesis                Biochem. 18:31-90. <br />     control cell division yeast S. cerevisiae. Cell 14:951-     50. Sherman, F. 1991. Getting started yeast. Methods En- <br />     958.                                                                      zymol. 194:3-21. <br /> 28. Johnston, G. C., R. A. Singer. 1980. Ribosomal precursor          51. Sklar, V. E. F., R. G. Roeder. 1976. Purification subunit <br />     RNA metabolism cell division yeast Saccharomyces               structure DNA-dependent RNA polymerase III the <br />     cerevisiae. Mol. Gen. Genet. 178:357-360.                                 mouse plasmacytoma, MOPC 315. J. Biol. Chem. 251:1064- <br /> 29. Kolodziej, P., R. A. Young. 1989. RNA polymerase II                   1073. <br />     subunit RPB3 essential component mRNA transcrip-         52. Stettler, S., S. Mariotte, M. Riva, A. Sentenac, P. Thuriaux. <br />  4326       MANN ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br />       RPC34 encodes essential specific subunit yeast RNA        59. Van Ryk, D. I., Y. Lee, R. N. Nazar. 1990. Efficient <br />       polymerase C (III). J. Biol. Chem., press.                           expression utilization mutant 5S rRNA Saccharomy- <br /> 53.   Struhl, K. 1981. Position effects Saccharomyces cerevisiae. J.       ces cerevisiae. J. Biol. Chem. 265:8377-8381. <br />       Mol. Biol. 152:569-575.                                             60. Veinot-Drebot, L. M., R. A. Singer, G. C. Johnston. 1989. <br /> 54.   Struhl, K. 1985. Nucleotide sequence transcriptional map-           rRNA transcription initiation decreased inhibitors the <br />       ping yeast pet56-his3-dedl gene region. Nucleic Acids            yeast cell cycle control step &quot;start.&quot; J. Biol. Chem. 264:19528- <br />       Res. 13:8587-8601.                                                      19534. <br /> 55.   Thuriaux, P. Unpublished da <br /> </body></html>