<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 30 July 2013                                                Nucleic Acids Research, 2013, Vol. 41, No. 19 9033&#8211;9048 <br />                                                                                                                  doi:10.1093/nar/gkt654 <br />  <br />  <br /> High-resolution <span id='am-163' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> <span id='am-73' about='obo:NCBITaxon_666' typeof='owl:Thing obo:NCBITaxon_641 obo:NCBITaxon_662 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135623'><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Liquidivibrio cholerae" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Vibrio cholera" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="Microspira comma" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Vibrio choleae" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="PMID:9272984" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Spirillum cholerae-asiaticae" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Bacillus cholerae" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Kommabacillus" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Bacillus cholerae-asiaticae" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasRelatedSynonym" content="Bacillo virgola del Koch" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasRelatedSynonym" content="Pacinia cholerae-asiaticae" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasRelatedSynonym" content="Vibrio cholerae-asiaticae" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="Vibrio cholerae" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="Spirillum cholerae" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="Vibrio comma" datatype="xsd:string"></span>Vibrio cholerae</span> <br /> essential gene set hidden Markov model&#8211;based <br /> analyses transposon-insertion sequencing <span id='am-164' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-165' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-166'  ></span><span id='am-167' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-168' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> <br /> Michael C. Chao1,2,3, Justin R. Pritchard4, Yanjia J. Zhang4, Eric J. Rubin1,2,4, <br /> Jonathan Livny5, Brigid M. Davis1,2,3 Matthew K. Waldor1,2,3,* <br /> 1 <br />  Division Infectious Disease, Brigham &amp; Women&#8217;s Hospital, Boston, MA 02115, USA, 2Department of <br /> Microbiology Immunobiology, Harvard Medical School, Boston, MA 02115, USA, 3Howard Hughes Medical <br /> Institute, Boston, MA 02115, USA, 4Department Immunology Infectious Diseases, Harvard School of <br /> Public Health, Boston, MA 02115, USA 5Genome Sequencing Analysis Program, Broad Institute, <br /> Cambridge, MA 02142, USA <br />  <br /> Received June 13, 2013; Revised July 1, 2013; Accepted July 2, 2013 <br />  <br />  <br />  <br /> ABSTRACT                                                                        INTRODUCTION <br /> The coupling high-density transposon mutagen-                                The coupling high-density transposon mutagenesis with <br /> esis high-throughput DNA sequencing (trans-                                  high-throughput DNA sequencing (referred as <br /> poson-insertion sequencing) enables simultaneous                                &#8216;transposon-insertion sequencing&#8217;) given microbiolo- <br /> and genome-wide assessment contributions                                 gists potent new tool&#8212;the capacity simultaneously <br />                                                                                 comprehensively assess genome-wide fashion <br /> of individual loci bacterial growth survival. <br />                                                                                 contributions individual loci growth. There <br /> We refined analysis transposon-insertion                                different methods transposon-insertion <br /> sequencing data normalizing effect                                sequencing, using different <span id='am-34' about='obo:NCBITaxon_2387' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Transposon" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="transposons" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>transposons</span> methods for <br /> DNA replication sequencing output using                                library preparation {e.g. TnSeq, InSeq, TraDIS HITS <br /> hidden Markov model (HMM)-based filter exploit                               [reviewed (1,2)]}. However, transposon-insertion <br /> heretofore unappreciated information inherent                            sequencing studies start construction high- <br /> transposon-insertion sequencing data sets. The                                  density transposon library, principle contain <br /> HMM smooth variations read abundance                                     mutations disrupting nonessential locus the <br /> and reduce effects read noise,                                genome. Then, next-generation DNA sequencing used <br /> well permit fine scale mapping independ-                             map transposon insertion sites en masse. Quantitative <br /> ent genomic annotation enable classification                             analyses frequency mutations each <br />                                                                                 genetic locus detected enables discrimination between <br /> of loci functional categories (e.g. essen- <br />                                                                                 loci required dispensable growth the <br /> tial, domain essential &#8216;sick&#8217;). We generated                               conditions interest. Mutations absent es- <br /> high-resolution map genomic loci (encompassing                               sential loci underrepresented loci promote, but <br /> both intra- intergenic sequences)                                  required for, survival proliferation. <br /> required beneficial vitro growth                                  Since independent introduction 2009 dif- <br /> cholera pathogen, <span id='am-92' about='obo:NCBITaxon_662' typeof='owl:Thing obo:NCBITaxon_641 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135623'><span id='am-93' property="oboInOwl:hasDbXref" content="PMID:7520733" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="PMID:17978204" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="PMID:1371064" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasAlternativeId" content="NCBITaxon:705" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="Vibrio" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasRelatedSynonym" content="Listonella" datatype="xsd:string"></span><span id='am-100'  ></span><span id='am-101' property="oboInOwl:hasDbXref" content="PMID:8590667" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="PMID:4935323" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasRelatedSynonym" content="Microspira" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="Pacinia" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="PMID:21296930" datatype="xsd:string"></span>Vibrio</span> cholerae. <span id='am-58' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-59' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61'  ></span><span id='am-62' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> work un-                                ferent groups (3&#8211;6), transposon-insertion sequencing <br /> covered new metabolic physiologic require-                                  approaches applied study diverse bac- <br /> ments V. cholerae survival, combining                                teria variety growth conditions [see (1,2)]. <br /> transposon-insertion sequencing transcri-                                   However, past current methods analysis of <br /> ptomic data sets, identified novel                              transposon-insertion sequencing data ad- <br /> noncoding RNA <span id='am-63' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> contribute                                        vantage information stored large data <br />                                                                                 sets. For example, studies yield genome- <br /> V. cholerae growth. Our findings suggest that <br />                                                                                 wide information, analyses characterize <br /> HMM-based approaches enhance extraction                                 annotated open reading frames (ORFs) omit <br /> biological meaning transposon-insertion                                    consideration untranslated loci <span id='am-143' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-144'  ></span><span id='am-145' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-146' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cis</span>-acting sites <br /> sequencing genomic data.                                                        noncoding RNAs (ncRNAs) (1). Moreover, no <br />  <br />  <br />  <br /> *To correspondence addressed. Tel: +1 617 525 4646; Fax: +1 617 525 4660; Email: MWALDOR@research.bwh.harvard.edu <br />  <br /> &#223; The Author(s) 2013. Published Oxford University Press. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which <br /> permits unrestricted reuse, distribution, reproduction medium, provided original work properly cited. <br />  9034 Nucleic Acids Research, 2013, Vol. 41, No. 19 <br />  <br />  <br /> transposon-insertion sequencing study reported fully          optimal V. cholerae growth. Collectively, study <br /> exploited local context information inherent high-         provides highly re&#65533;?ned curation essential V. <br /> density transposon libraries. In general, insertion mutants       cholerae gene set, illustrates utility HMM- <br /> within genetic locus high probability          based analyses bring transposon-insertion <br /> causing similar phenotypes. However, tech-         sequencing data sets. <br /> nical biological sources noise, including library <br /> undersaturation, ampli&#65533;?cation <span id='am-39' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Bias" datatype="xsd:string"></span>bias</span>/jackpots stochas- <br /> tic variation replicates, number reads         MATERIALS AND METHODS <br /> insertion mutants adjacent sites vary widely. Such         Strains, media culture conditions <br /> noise obscure biologically meaningful <br /> patterns data, especially reads          All strains grown LB Miller (1% NaCl) unless <br /> locus averaged together.                                      noted. Antibiotic concentrations used were <br />    In work, exploit hidden Markov model                 200 mg/ml streptomycin (Sm), 50 mg/ml kanamycin (Km) <br /> (HMM) improve previous analyses lessening im-           100 mg/ml ampicillin (Amp). Wild-type V. cholerae <br /> pact intrinsic variability (noise) transposon-insertion     C6706 <span id='am-107' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-108' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-111'  ></span><span id='am-112' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia coli</span> SM10 lambda pir carrying <br /> sequencing data generating robust consist-           Himar1 suicide transposon vector pSC189 (9) were <br /> ent biological signals. HMMs widely used             grown 37  C LB+Sm LB+Amp, respectively. <br /> &#65533;?nd meaningful patterns bioinformatics contexts,          Individual transposon mutants ordered V. <br /> including DNA alignment, protein fold prediction              cholerae transposon library (10) propagated in <br /> motif searching (7), analyses trans-           LB+Sm+Km 37  C, overnight, unless otherwise <br /> poson-insertion sequencing data. The HMM calculates               stated. Where indicated, transposon mutant strains <br /> the likely sequence nonobserved &#8216;hidden states&#8217;           grown M9 media+0.2% glucose+Sm+Km. <br /> given series observed &#8216;emissions&#8217;. Speci&#65533;?cally,         Ribo&#64258;avin (Sigma) supplemented indicated. <br /> adapted mariner-based transposon-insertion seque- <br /> ncing data, number reads insertion site            Transposon mutant library construction <br /> (emissions) enables discovery hidden identity           Three independent transposon libraries created in <br /> each TA site (i.e. site &#8216;essential&#8217;, &#8216;neu-   V. cholerae C6706 conjugation. Brie&#64258;y, 1.6 ml of <br /> tral&#8217; &#8216;sick&#8217; locus). The HMM approach creates stat-          overnight culture SM10 lambda pir E. coli carrying a <br /> istical framework incorporates local context            suicide transposon vector, pSC189 (9), mixed with <br /> predictions, hidden identity TA site          1.6 ml overnight culture V. cholerae C6706. <br /> is inferred reads              Cells pelleted, media removed, washed with <br /> previous TA site. This approach relatively insensitive         LB resuspended 800 ml LB. Hundred micro- <br /> to read count noise confound previous analysis           liters aliquots resuspended cells spotted onto <br /> methods, creates genomic pro&#65533;?le (i.e. regions      0.45 mm HA &#65533;?lters (Millipore) LB agar plate. The <br /> are essential growth display similar pattern      mating plate incubated 37  C 2 h, the <br /> reads) used functionally separate genomic          &#65533;?lters single tube 7.5 ml LB the <br /> loci different biological categories.                        cells resuspended vortexing. The resuspended cells were <br />    Here, present high-resolution transposon-insertion        equally plated 245   245 mm2 (Corning) <br /> sequencing-based study contribution genetic         LB+Sm+Km agar plates colonies grown 30  C <br /> loci growth rich medium Vibrio cholerae,            24 h. <br /> Gram-negative etiologic agent diarrheal disease               Colonies library plates scraped, resuspended <br /> cholera. We categorized genes intergenic loci          LB pooled &#65533;?nal volume  15 ml. A 3 ml <br /> the genome based contribution growth              aliquot cells used genomic DNA extraction using <br /> and survival organism vitro, using HMM-              Wizard Genomic DNA extraction kit (Promega), using <br /> based secondary analysis enhanced ability             10  volumes reagents lysis. At the <br /> discern essential gene signals noise. Because trans-         &#65533;?nal DNA precipitation step, isopropanol added and <br /> poson-insertion sequencing methods constrained by <br />                                                                   precipitated genomic DNA spooled <span id='am-158' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-159' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-161' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-162'  ></span>iso</span>- <br /> sequencing capacity, use &#8216;essential&#8217; study to <br />                                                                   propanol solution using blunted glass Pasteur pipette. <br /> refer loci absolutely necessary survival as <br />                                                                   The DNA pellet successively washed 70 and <br /> well loci interruption compromises growth to <br />                                                                   100% ethanol, spooling wash new <br /> the point represented sequenced <br />                                                                   tube. After &#65533;?nal wash, excess ethanol pressed out <br /> library. Comparison V. cholerae essential gene set to <br />                                                                   DNA pipette air dried RT for <br /> that <span id='am-123' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-124' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-126' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-127'  ></span><span id='am-128' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span>Escherichia</span> coli identi&#65533;?ed differences these <br />                                                                   30 min. Dry DNA pellets &#65533;?nally resuspended 1.5&#8211; <br /> related organisms critical metabolic physiologic <br />                                                                   2.5 ml nuclease-free water 65  C 1 h, with <br /> processes. Moreover, HMM allowed identify a <br />                                                                   frequent mixing. <br /> new <span id='am-55' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> &#8216;sick&#8217; genes inactivation impaired <br /> growth. We identi&#65533;?ed cis-acting noncoding essential <br />                                                                   Transposon junction ampli&#65533;?cation, sequencing <br /> loci, including putative 50 untranslated regions <br />                                                                   mapping <br /> (UTRs) essential genes. Finally, comparing our <br /> data set transcriptome data (8), identi&#65533;?ed candi-         In general, transposon junctions prepared am- <br /> date trans acting ncRNA species required             pli&#65533;?ed according previously published method (11). <br />                                                                        Nucleic Acids Research, 2013, Vol. 41, No. 19 9035 <br />  <br />  <br /> However, digested genomic DNA using                     distribution enabled calculate boundary on <br /> Fragmentase (NEB) opposed mechanical                     left peak encompasses same <br /> shearing&#8212;speci&#65533;?cally, 25 mg genomic DNA              number genes. Using boundary, used Excel to <br /> replicate library treated Fragmentase (&#65533;?ve reac-      calculate fraction ( 0.34) 1% of <br /> tions worth enzyme 250 ml solution 1&#8211;1.25 h     neutral genes found. Five genes <br /> 37  C). After fragmentation, DNA end repaired              cutoff (the Hit Distribution Hit Dist List) <br /> (Quick Blunt kit, NEB), adaptors ligated (sequences         underrepresented transposon insertion used <br /> Supplementary Table S15) PCR ampli&#65533;?cation con-             compare downstream analysis pipelines [i.e. <br /> ducted previously described. The &#65533;?nal PCR ampli&#65533;?ed          sliding window (SW) HMM data sets]. <br /> products gel puri&#65533;?ed 2% agarose gel isolate <br /> 200&#8211;500 bp fragments, subjected qPCR using              SW analysis <br /> primers Illumina P5 P7 hybridization           Mapped reads run custom scripts of <br /> sequence (Supplementary Table S15) estimate input           SW pipeline; scripts (in Supplementary Methods) and <br /> and ensure equal multiplexing downstream sequencing.        example input &#65533;?les Supplementary Materials <br /> Equimolar DNA fragments library                 (11). Brie&#64258;y, TA-site count X, null distributions <br /> combined sequenced MiSeq V2.0                  generated data set totaling number <br /> platform 65 cycles using 50-cycle cartridge              reads X randomly sampled sites 10 000 times. Then, <br /> (Illumina).                                                    number reads monitored 10 bp intervals <br />    Libraries 1&#8211;3 following number total reads:      genome overlapping 200 bp windows, and <br /> 812 415; 1 857 563; 1 510 158. Reads trimmed          numbers compared null distribution. <br /> transposon adaptor sequences using CutAdapt (12),          For window, P-value calculated rank of <br /> and mapped V. cholerae N16961 genome          test mean divided 10 000. The false discovery rate <br /> three allowable mismatches using Bowtie aligner (13).      (FDR) window calculated using the <br /> Note V. cholerae El <span id='am-153' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-154'  ></span><span id='am-155' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-157' property="rdfs:label" content="Tor" datatype="xsd:string"></span>Tor</span> clinical isolates, like C6706     Benajmini&#8211;Hochberg test (17). De&#65533;?nition signi&#65533;?cantly <br /> and N16961, differ  50&#8211;250 single nucleotide           underrepresented windows data set (P &lt; 0.01 and <br /> polymorphisms (SNPs) entire genome (14).            FDR &lt; 0.05) required presence seven TA <br /> Duplicated loci mapped absent         sites reads. All windows locus were <br /> the study&#8212;these loci, loci absent V. cholerae    &#65533;?ltered following criteria: loci containing <br /> C6706 compared V. cholerae N16961 (personal com-          signi&#65533;?cantly underrepresented windows (P &lt; 0.01, <br /> munication) listed Supplementary Table S7. For          FDR &lt; 0.05) assigned essential, loci contain- <br /> libraries 1&#8211;3, number total mapped reads           ing nonsigni&#65533;?cant windows assigned neutral. <br /> 375 775, 588 918 494 757 ( 30&#8211;45% mapping ef&#65533;?-             Loci contain signi&#65533;?cant nonsigni&#65533;?- <br /> ciency), hit 31, 33 35% TA dinucleotides      window assigned domain essential. Loci that <br /> in genome, respectively. After mapping, reads      contained windows TA sites capable of <br /> each TA site tallied, assigned known gene      reaching statistical signi&#65533;?cance (less seven TA sites) <br /> intergenic region visualized DNAplotter         unable evaluated. Intergenic loci de&#65533;?ned <br /> Artemis (15). Finally, library contained similar   sequence lies entirely end a <br /> percentage TA sites hit normalized 375 000 total     previous gene start current one. All <br /> mapped reads combined single data set               intergenic regions 50 locus number the <br /> covering  1.1 million reads genome (and insertions      forward strand [e.g. IG_VC0001 comprises nucleotides <br /> at 56% TA sites) using R (16).                          1&#8211;324 (the bp coding sequence VC0001)]. <br />                                                                Analyses performed using <span id='am-174' about='obo:NCBITaxon_37579' typeof='owl:Thing obo:NCBITaxon_1329950 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8504 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8509 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_1329912 obo:NCBITaxon_33208 obo:NCBITaxon_1329961 obo:NCBITaxon_1329911 obo:NCBITaxon_7711 obo:NCBITaxon_8570 obo:NCBITaxon_34979 obo:NCBITaxon_7776 obo:NCBITaxon_34984 obo:NCBITaxon_32561 obo:NCBITaxon_8457'><span id='am-175'  ></span><span id='am-176' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-177' property="rdfs:label" content="Python" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasAlternativeId" content="NCBITaxon:51884" datatype="xsd:string"></span>Python</span> 2.7.3 with <br /> Positional correction replication hit                  NumPy 1.7.1 SciPy 0.12.0 add-on packages. <br /> distribution analysis <br /> To normalize chromosomal replication biases             Hidden Markov modeling functional categorization <br /> read counts, chrI chrII divided 10 kb            loci <br /> nonoverlapping windows starting windows centered          To adapt HMM framework mariner-based <br /> around origins. Using Microsoft Excel functions,       transposon-insertion sequencing data, fol- <br /> number reads window (after removing in-         lowing analogy: hidden states &#8216;essential&#8217;, &#8216;neutral&#8217; <br /> sertion sites reads reduce overscaling     &#8216;sick&#8217;. Emissions read counts given TA site. <br /> to essential loci) normalized based factor           To adapt continuous read count data discrete <br /> calculated average read depth              modeling framework, binned data based the <br /> window divided average depth chromosome.        distribution read counts TA sites. The bin <br />    After normalization, fraction TA sites containing    size chosen smallest number bins that <br /> a transposon insertion calculated genes. The       allowed separate true-positive sick genes (visualized <br /> genes sorted bins based fraction    Artemis) true-positive essential genes. For this <br /> TA sites hit transposon, distribution      analysis, bin size seven reads were <br /> plotted. We visually estimated distribution        discretized bin. Following discretization, we <br /> random insertions neutral genes peak centered         trained HMM blocks SW calls the <br /> at 0.75. Counting number genes right half     entire genome&#8212;TA sites underrepresented windows <br />  9036 Nucleic Acids Research, 2013, Vol. 41, No. 19 <br />  <br />  <br /> were assigned essential, TA sites nonsigni&#65533;?cant      grown 37  C Bioscreen C optical density <br /> windows assigned neutral. To incorporate sick            reader (Growth Curves USA) OD600 measurements <br /> genes, manually annotated small set 20 candidate       taken 15-min intervals 12 h. Additionally, candidate <br /> sick genes (Supplementary Table S8) training set          sick transposon mutants diluted 1:1 lacZ  wild- <br /> (these genes chosen appeared consistently        type V. cholerae grown 30  C, overnight. While the <br /> have lower read depth compared likely neutral loci).        mutants ordered V. cholerae transposon library <br /> The initial estimates emission transition prob-       constructed lacZ  background (10), trans- <br /> ability matrices derived using Baum&#8211;Welch algo-         poson used study carried lacZ reporter gene, <br /> rithm implemented Matlab R2011b (Mathworks) (code             making insertion mutants functionally lacZ+ and <br /> in Supplementary Methods). Following initial training,       blue X-gal media. Inoculum overnight <br /> the sequence hidden states uncovered implement-        cultures plated LB+Sm200+60 mg/ml Xgal <br /> ing Viterbi method. Furthermore, exact               resultant mutant (blue) wild-type (white) <br /> training sets produce biases, iteratively re&#65533;?ned          colonies counted. Competitive indices were <br /> the hidden states using sequence &#65533;?rst iteration    calculated ratios mutant wild-type colony- <br /> to rederive transition emission matrices        forming units (CFUs). <br /> iteration Viterbi parse. We did the <br /> emission transition matrices equivalent subse-       ribE <br /> quent iterations. Iteration necessary reduce bias   Tn cells grown ordered transposon library, <br /> a given training set. Different training sets differ-   pelleted, washed phosphate buffered saline streaked <br /> ent answers &#65533;?rst implementation Viterbi algo-      agar plates containing ribo&#64258;avin <br /> rithm, practice, iteration converges set   [LB+Sm+Km (LB)], M9+glucose+Sm+Km+ <br /> predictions, training sets vary.                     5 mM ribo&#64258;avin (M9+B2) [M9+glucose+ <br />    After HMM ran convergence (i.e. algorithm          Sm+Km (M9)]. Cells grown 48 h 37  C to <br /> no longer changed assignments looped           visualize colonies. Additionally, washed cells also <br /> data), set secondary &#65533;?lters (Supplementary Methods)         diluted 1:1000 LB, M9 M9+B2 the <br /> were used functionally separate loci appropri-       Bioscreen C OD600 measurements taken 15-min <br /> ate categories. Essential genes de&#65533;?ned       intervals 24 h. <br /> central 80% coding sequence contained TA sites clas- <br /> si&#65533;?ed HMM essential. Essential intergenic regions          Identi&#65533;?cation ncRNAs important optimal <br /> had essential TA sites entire intergenic space.      growth vitro <br /> Neutral genes intergenic regions de&#65533;?ned              Annotated known ncRNA species (18), putative transcrip- <br /> regions containing neutral TA sites. Regions clas-      tional units identi&#65533;?ed comparative genomics- <br /> si&#65533;?ed essential neutral loci investigated po-      based searches (19) small intergenic RNAs found <br /> tential domain essential sick regions. To classed      analyses RNA-Seq data sets (8) overlaid <br /> domain essential, genes/loci contain nonessen-       SW results. Then, P-values overlapping <br /> tial TA sites contiguous stretch &#65533;?ve es-       windows &#65533;?t ncRNA averaged <br /> sential TA sites, covers 15&#8211;300 bp 90% loci      together, ncRNAs containing window(s) stat- <br /> in genome. Regions classi&#65533;?ed sick          istically underrepresented reads (P &lt; 0.05) were <br /> had majority &#8216;sick&#8217; TA sites entire locus.       identi&#65533;?ed candidate species required optimal <br /> Regions did meet criteria domain essential       growth. <br /> sick categories placed neutral category. To <br /> calculate con&#65533;?dence HMM assignment, we <br /> assessed posterior probabilities (likelihood)      RESULTS AND DISCUSSION <br /> TA site categorized essential, neutral sick. The     Generation high-density transposon library in <br /> data calculated using modi&#65533;?ed version              V. cholerae <br /> &#8216;hmmdecode&#8217; function Matlab (code available in <br /> the Supplementary Methods), average                      We used well-characterized Himar1 mariner-based <br /> probabilities loci presented Supplementary        transposon vector pSC189 (9) create high-density <br /> Table S14 (raw data Supplementary Sample              transposon insertion library V. cholerae. Himar1 <br /> input &#65533;?le folder). The inputs &#8216;hmmdecode&#8217;                inserts TA dinucleotides additional sequence <br /> function transition emission probability            constraints (20,21), rendering extremely useful for <br /> matrices readcount emissions &#65533;?nal iteration         high-density mutagenesis organism as <br /> of HMMconvergence script.                                    V. cholerae, T+A 52%. The V. cholerae <br />                                                                  genome consists unequally sized chromosomes, <br />                                                                  chrI ( 3 Mb) chrII ( 1.1 Mb) (22). Comparative <br /> Growth analysis candidate sick ribo&#64258;avin <br />                                                                  analyses chromosomes V. cholerae <br /> synthesis mutants <br />                                                                  N16961 (a sequenced seventh pandemic El Tor strain) <br /> A transposon mutant candidate sick genes ribE           revealed chrI 2.75-fold TA dinucleotides <br /> (vc2268) isolated ordered V. cholerae trans-        chrII ( 140 000 versus  50 000), difference that <br /> poson library (10) grown LB+Sm+Km.                        mirrors 2.76-fold difference size. In addition, the <br /> Candidate sick genes diluted 1:1000 LB+Sm                distances adjacent TA sites two <br />                                                                       Nucleic Acids Research, 2013, Vol. 41, No. 19 9037 <br />  <br />  <br /> chromosomes extremely similar,                transposon mutant library. To circumvent this <br /> annotated genes intergenic sequences (Supple-          issue, normalized read depth chromosome <br /> mentary Figure S1A). Thus, mariner transposition           position-dependent correction, pre- <br /> likely genome, chromo-         viously applied transposon-insertion sequencing studies <br /> somes analyzed using pipeline.                (26). Brie&#64258;y, chromosome divided 10 kb <br />    We generated independent transposon libraries,       wide windows reads window scaled <br /> which contained 600 000 transposon         average reads entire chromosome. The <br /> mutants (on average,  3-fold saturation TA sites),      normalized data [Figure 2A (inner track), 2C] enabled <br /> the seventh pandemic El Tor V. cholerae clinical isolate      direct comparison transposon insertion frequency in <br /> C6706 (10). The sites Himar1 insertion library     loci, regardless genomic position. <br /> were determined next-generation sequencing             The pooled library contained insertions  56% the <br /> MiSeq platform subsequently mapped               TA dinucleotides genome. This translates an <br /> closely related V. cholerae N16961 genome. El Tor             average insertion  15&#8211;16 transposons locus the <br /> V. cholerae clinical isolates (such N16961 C6706)      library (Supplementary Figure S1B) average 11 <br /> are thought differ  50&#8211;250 SNPs         8 reads TA position chrI chrII, respect- <br /> genome (13), did expect systematic dif&#65533;?culties     ively (Supplementary Figure S1C). TA sites lacking inser- <br /> in mapping C6706 reads N16961 reference genome.        tions eluded transposition chance or <br /> The number insertions TA site tallied,     located regions tolerate insertions <br /> then normalized library contained equal        essential growth LB plates. To <br /> numbers mapped reads. Although precise sites        distinguish possibilities estimate the <br /> insertion varied expected libraries, in-    true saturation library, assessed distribution <br /> sertion sites assessed annotated locus          insertion frequencies gene. As expected for <br /> ( 6000 combined genic intergenic loci), compos-       subsaturating insertions &#8216;neutral&#8217; genes (i.e. genes <br /> ition libraries appeared extremely         inactivation impact bacterial growth), we <br /> similar. The Pearson correlation coef&#65533;?cient (R)       observed large symmetric peak centered genes with <br /> libraries 0.94 0.95, coef&#65533;?cients         insertion fraction 0.70&#8211;0.75, suggesting genes <br /> of determination (R2 value) 0.88 0.89             (and likely intergenic loci) average underwent random <br /> (Figure 1A). The high degree concordance           transposon insertion  75% available TA sites. <br /> the biological replicates suggests transposition   Importantly, estimating area peak, we <br /> occurred similarly samples subsequent        set 1 percentile cutoff curve  34% of <br /> ampli&#65533;?cation did introduce distortion data       TA sites gene disrupted (Figure 1B, dotted line). <br /> set. Furthermore, sequencing protocol pre-      Notably, cutoff yields 502 genes (hereafter referred <br /> viously accurately maintain composition       &#8216;Hit Dist&#8217; gene set) fewer transposon inser- <br /> each library (11). Therefore, pooled mapped reads      tions expected chance (P &lt; 0.01). Most of <br /> together single high-density transposon insertion      &#8216;under-inserted&#8217; genes (Supplementary Table S1) are <br /> data set downstream analysis.                             likely loci essential survival are <br />                                                               required optimal growth&#8212;i.e. transposon insertions <br /> Distribution transposon insertions                         genes render cells &#8216;sick&#8217; likely <br />                                                               represented transposon library. However, add- <br /> We plotted position read depth trans-         itional analysis tools (discussed below) required dis- <br /> poson insertion chromosomes evaluate          tinguish possibilities. <br /> if genomic &#8216;hotspots&#8217; transposition. <br /> Although transposon insertions mapped both <br />                                                               SW hidden Markov analysis <br /> chromosomes (Figure 2A), noticeable pos- <br /> itional bias read depth; chromosome, reads        Several different analytical approaches, their <br /> were abundant replication origin region,      bene&#65533;?ts drawbacks, used identify <br /> gradually decreased minimum level near         genes required growth using transposon-insertion <br /> chromosome terminus (Figure 2A B). This skew           sequencing data (3,11,27,28). Our data set, a <br /> likely relative abundance origin-proximal       high number transposition sites relatively few <br /> chromosomal regions result ongoing initiation       mapped reads insertion locus average <br /> of bidirectional DNA replication sites (23). An      (Figure 1C), permits evaluation genomic insertion <br /> elevated number chrI insertion sites, relative chrII    sites high resolution, requires analysis method- <br /> sites, observed (Figure 2B), consistent    ology does depend large differences read <br /> fact chrII replication initiates cell cycle,    depth essential neutral loci. The recent SW <br /> whereas multiple chrI initiation events occur (24,25).    approach developed Zhang et al. (11), the <br /> Thus, positional skew read depth appears re&#64258;ect     frequency insertions sequential overlapping <br /> the known replication-based skew chromosome                windows compared random null distribution <br /> abundance.                                                    based data set, allowed evaluate whether <br />    While positional bias informative             loci underrepresented relatively low read <br /> V. cholerae chromosome replication dynamics, interferes    depth library. Furthermore, approach enables <br /> with comparison relative abundance             analysis variably sized genomic windows, allowing <br />  9038 Nucleic Acids Research, 2013, Vol. 41, No. 19 <br />  <br />  <br />  <br />  <br /> Figure 1. Reproducibility distribution Himar1 transposon insertions V. cholerae. (A) The total reads transposon insertions each <br /> intergenic genic region ( 6000 loci) summed library. Pairwise comparisons read abundance locus plotted the <br /> three libraries used determine coef&#65533;?cient determination (R2 value) pair. (B) The fraction TA sites containing a <br /> transposon insertion calculated gene pooled transposon-insertion sequencing data set, number genes associated with <br /> each fraction plotted. Using distribution presumed neutral genes (right peak), 1% signi&#65533;?cance cutoff calculated 0.34 (dotted <br /> line). Genes left line signi&#65533;?cantly fewer transposon insertions expected neutral insertion events (P &lt; 0.01). (C) After <br /> positional correction, total reads transposon insertions gene data set averaged total TA sites gene. The <br /> number genes associated average frequency plotted. <br />  <br />  <br />  <br /> characterization small loci, including intergenic regions                   secondary &#65533;?lter used P-values window <br /> and subdomains ORFs, analyses                            categorize V. cholerae genes func- <br /> explicitly address (28). De&#65533;?ning essential domains                          tional categories: (i) essential growth, (ii) containing a <br /> valuable evaluating effect truncation point                     domain essential growth (iii) neutral. With SW <br /> mutations locus, allowing separate cat-                    analysis, 247 V. cholerae genes classi&#65533;?ed essential <br /> egorization subregions multi-domain genes                        growth (having windows signi&#65533;?cantly fewer <br /> their contribution growth. Alternative approaches                      reads core 80% coding sequence gene); 332 <br /> average reads entire locus mask                              genes putative essential domains (i.e. genes both <br /> presence domains tolerate insertion.                            signi&#65533;?cantly underrepresented windows neutral <br />    In SW pipeline, genomic regions data                        windows); 2759 genes neutral growth (did not <br /> randomly sampled create null distribution expected                     contain underrepresented windows) signi&#65533;?cance <br /> reads given window size. Then, overlapping windows                       412 genes able determined <br /> were scanned data compared null                        (Supplementary Table S1). Additionally, 136 genes in <br /> distribution assess given window contained                        V. cholerae N16961 duplicated absent in <br /> statistically fewer reads expected chance alone. In                    V. cholerae C6706 (Supplementary Table S7) thus <br /> principle, optimum SW small                             included analyses study. <br /> resolve protein domains noncoding regions,                               Although SW approach statistically robust, the <br /> while retaining suf&#65533;?cient number TA dinucleotides                         secondary &#65533;?ltering SW results appears overly stringent <br /> to reach statistical signi&#65533;?cance. We applied SW algo-                      assigning essential genes permissive for <br /> rithm (11) minor modi&#65533;?cations (see &#8216;Materials                         domain essential candidates. For example, essen- <br /> Methods&#8217; section) data using 200 bp window.                           tial ORFs de&#65533;?ned genes underrepresented <br /> Then, windows given locus compiled,                          windows central 80% coding sequence this <br />                                                                                    Nucleic Acids Research, 2013, Vol. 41, No. 19 9039 <br />  <br />  <br />  <br />  <br /> Figure 2. Normalization transposon-insertion sequencing read abundance correct positional bias. (A) Reads mapped V. <br /> cholerae chromosomes; amplitude peak represents read abundance speci&#65533;?c insertion site. Annotated genes encoded on <br /> the+strand shown red, genes &#8211; strand green (outer track). Raw mapped reads plotted outer gold track, <br /> while position-normalized read abundance shown inner gold track. (B) Raw mapped reads plotted according genomic <br /> positions chromosomes I II. Origin proximal sequences located ends graphs, termini located near the <br /> midpoints. Each spot represents unique transposon insertion site. (C) Read abundance following normalization using moving average smoothing <br /> method plotted function chromosomal position. <br />  <br />  <br />  <br /> approach, known essential gene dnaA erroneously                   long stretches essential dispensable sequence <br /> assigned domain essential category single                 (Figure 3B), SW &#65533;?lter does assess charac- <br /> window containing read mapped gene                         teristic, makes approach susceptible false- <br /> (Figure 3A). True domain essential genes contain                   negative classi&#65533;?cations essential genes. Another <br />  9040 Nucleic Acids Research, 2013, Vol. 41, No. 19 <br />  <br />  <br />  <br />  <br /> Figure 3. Examples read abundance essential, domain essential sick genes chrII origin replication (oriCII). Read abundance and <br /> sites transposon insertions shown (A) essential, (B) domain essential, (C) sick loci (D) oriCII. Red bars represent number of <br /> reads insertions facing forward the+strand green bars re&#64258;ect insertions complementary strand. Black bars represent potential <br /> transposon insertion sites (TA dinucleotides) indicated regions. In (B), previously mapped E. coli RneE domains, including RNase catalytic <br /> domain, arginine-rich ribosomal binding site (ARBBS) scaffolding protein interaction domain aligned VC2030 overlaid on <br /> the transposon-insertion sequencing data. In (D), transposon insertions mapped oriCII. The schematic spans noncoding vca0001 (rctA), <br /> the intergenic region IG_vca0002 rctB (vca0002), encodes initiator chrII replication. Genetic analyses revealed chrII <br /> origin includes regions dispensable (blue) essential (orange) DNA replication (29,30). Functional assignments study [NE = neutral <br /> locus, D = locus essential domain (subregion)] shown bottom. <br />  <br />  <br />  <br /> limitation published SW approach inability                  read counts, used HMM predict re&#65533;?ne <br /> categorize genes essential required               essentiality TA site genome. The logic <br /> for optimal growth. Insertions &#8216;sick&#8217; genes impair                  bene&#65533;?t approach illustrated by <br /> growth, therefore, lead fewer reads relative                       reexamining dnaA (Figure 3A). With HMM-based <br /> neutral loci library (Figure 3C). In SW                           analysis, probability having stretch essential <br /> pipeline, V. cholerae genes ultimately                        TA dinucleotides followed single neutral TA site con- <br /> validated &#8216;sick&#8217; (see below) classi&#65533;?ed                            taining read returning stretch of <br /> nonessential contain reads gene                      essential TA dinucleotides highly unlikely; indeed, the <br /> (Supplementary Table S1).                                                    HMM discounts single read assigns dnaA the <br />    To circumvent limitations, used HMM                         essential gene category (Supplementary Table S2). <br /> re&#65533;?ne SW predictions. HMM incorporates local                            In fashion, HMM able use local <br /> TA context insertion acknowledging es-                     TA context make calls possible the <br /> sentiality given site consideration                  SW pipeline. The SW algorithm unable categorize <br /> the number mapped reads adjacent insertion site                     294 unique genes fewer seven TA dinucleotides <br /> because transposon mutants genomic region                        possessed potential insertion sites to <br /> likely behave similarly. Speci&#65533;?cally, HMM uses                         make statistical determination underrepresentation. <br /> series observed &#8216;emissions&#8217; (the number reads                       However, HMM assign functional categoriza- <br /> each TA site) calculate probability nonobserved                    tion genes (Supplementary Table S6) taking <br /> hidden states (i.e. site &#8216;essential&#8217;,              advantage information provided adjacent loci. <br /> &#8216;neutral&#8217; &#8216;sick&#8217; region). The HMM incorporates                        Because genes did bene&#65533;?t having prelim- <br /> read count previous TA site predicted                         inary essentiality assignment SW method, these <br /> state evaluating likelihood insertion                 gene calls statistically robust (Supplementary <br /> site belongs essential, neutral sick category given                Table S14) genes containing transposon inser- <br /> its read count. However, HMM requires training set                     tions presented separate down- <br /> to estimate likelihood TA site essential,                     stream analyses. Finally, HMM trained <br /> neutral sick region producing speci&#65533;?c read count.                       small set candidate genes (Supplementary <br /> Thus, trained emission hidden state                               Table S8) impaired growth detect regions that <br /> probabilities HMM using preliminary SW es-                       contain reduced transposon insertions lower <br /> sentiality assignments TA site associated                 read depth truly neutral regions, i.e. regions <br />                                                                         Nucleic Acids Research, 2013, Vol. 41, No. 19 9041 <br />  <br />  <br /> corresponding &#8216;sick&#8217; genes. For example, case      V. cholerae homologs essential promote <br /> ubiA (Figure 3C), SW algorithm classi&#65533;?ed           optimal growth V. cholerae (Supplementary Table S9). <br /> nonessential (Supplementary Table S1), HMM                  Collectively, good correspondence our <br /> recognized TA sites gene fewer reads       HMM results separate essential gene studies, sug- <br /> than true neutral genes. Thus, addition HMM           gesting HMM-based analysis suf&#65533;?ciently sensi- <br /> appears generate accurate assignments essen-        tive identify V. cholerae&#8217;s essential genes. <br /> tial genes SW pipeline capable          The HMM method generated noticeably smaller <br /> of categorizing small sick genes.                           list domain essential candidates compared SW <br />                                                                 approach (91 versus 332), expect given <br /> Comparisons HMM-based classi&#65533;?cation                 SW &#65533;?lter categorizes domain essential any <br /> of essential genes analyses                           gene essential nonessential <br />                                                                 window. This means essential genes a <br /> The HMM-based analysis data set yielded 343 es-          single nonsigni&#65533;?cantly underrepresented window, as <br /> sential, 91 domain essential, 128 sick 2894 neutral         dnaA (Figure 3A), misclassi&#65533;?ed domain essen- <br /> genes good con&#65533;?dence probabilities (Figure 4A          tial. In fact, 88 (27%) genes SW domain <br /> Supplementary Tables S2&#8211;S5, S14). It includes (343         essential list classed essential HMM <br /> versus 246) number V. cholerae genes classed        (Figure 4D), 10 88 mutants that <br /> as essential using SW pipeline (Figure 4A),       map central 80% gene Cameron <br /> given hypothesis SW approach              ordered      transposon     library.    In    addition    to <br /> prone false negatives, HMM results likely          underestimating essential genes, SW domain essential <br /> more accurate list true essential genes. This supported   list likely include genes fact <br /> by independent analyses. First, nearly (342      neutral. Because library 75% saturation, is <br /> 343) HMM essential genes overlap Hit Dist       likely single underrepresented windows present <br /> list, identi&#65533;?ed 502 genes having fewer transposon      neutral genes low detectable frequency. <br /> insertions expected chance (Supplementary         Consistent <span id='am-50' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-51' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>idea</span>, 97 (29%) the <br /> Table S1, Figure 4A). Furthermore, 78% HMM               SW domain essential loci classi&#65533;?ed neutral by <br /> essential genes essential (having rep-      HMM, 69 genes (21%) shared the <br /> resentative transposon mutant) previously de&#65533;?ned           approaches (Figure 4D). <br /> ordered V. cholerae transposon library Cameron                  Although set HMM-assigned domain essential <br /> et al. (10) (Figure 4B). Additionally, 16% HMM           genes likely includes false positives, analysis is <br /> essential genes transposon mutants ordered          concordant previous analyses individual <br /> library &#65533;?rst 10% genes&#8217; coding       loci. For example, HMM identi&#65533;?ed essential dispens- <br /> regions, consistent possibility [also suggested     able regions VC2030, encodes ribonuclease <br /> Cameron et al. (10)] disruptions termini es-     E, essential enzyme required 5S rRNA processing <br /> sential genes compromise gene                 (32) mRNA turnover (33). The highly homologous <br /> function. In contrast, Cameron library contains cen-        E. coli RneE domains: (i) ribonuclease cata- <br /> trally located mutations (within core 80% coding            lytic domain; (ii) arginine-rich RNA binding domain <br /> region) 6% essential gene candidates            (ARRBS) (iii) C-terminal scaffolding region (34). <br /> (Figure 4B). This discordance represent false posi-         Both transposon insertions VC2030 (Figure 3B) <br /> tives HMM assignments, con&#65533;?dence              genetic analyses rneE E. coli (34,35) indicate <br /> values genes appear good (Supplementary               N-terminal catalytic domain essential, <br /> Table S14). Interestingly, HMM essential genes list         ARRBS scaffolding domains dispens- <br /> contained far fewer 789 essential genes identi&#65533;?ed      able cell growth. Similarly, HMM assigned rctB, <br /> by Cameron et al. Given relatively small transposon       encodes initiator chrII replication, the <br /> library, suggested  350 essential candi-     domain essential class. Transposon insertions were <br /> dates lacked transposon insertions owing chance       tolerated C-terminal 373 nucleotides rctB <br /> than truly essential. Our data support this,              (Figure 3D), consistent genetic analyses have <br /> indicating 44% 789 genes apparent         revealed deletions 400 bp 30 end <br /> effect V. cholerae growth (Figure 4C).                       rctB abolish RctB&#8217;s ability initiate chrII rep- <br />    We conducted comparative analyses pre-           lication (29,36). Thus, HMM facilitated identi&#65533;?- <br /> dicted 343 essential genes 287 known essential      cation essential genes tolerate insertions at <br /> genes related gamma-proteobacterium E. coli [as          sites reduced misclassi&#65533;?cation neutral loci as <br /> identi&#65533;?ed based individual in-frame deletion studies         domain essential comparison SW method. <br /> (31)]. Homology searches revealed extensive concordance         However, optimization HMM predictions is <br /> between E. coli data set (Supplementary         possible reduce false-positive assignments; for <br /> Table S9)&#8212;homologs 87% E. coli essential genes        example, robust identi&#65533;?cation domain essential loci <br /> were categorized essential V. cholerae;  2.5%     favored generation saturated trans- <br /> the E. coli genes did apparent V. cholerae          poson libraries, lower chance containing <br /> homolog, remaining 10% essential            underrepresented windows chance. Additionally, as <br /> E. coli genes V. cholerae homologs sick         domain essential candidates validated, likely size <br /> gene list. In total,  90% E. coli essential genes   essential domain better estimated allow <br />  9042 Nucleic Acids Research, 2013, Vol. 41, No. 19 <br />  <br />  <br />  <br />  <br /> Figure 4. Comparative analyses V. cholerae gene classi&#65533;?cations. (A) Overlap V. cholerae essential genes predicted using various analytical <br /> approaches. The overlap essential genes predicted using HMM SW analyses (SW) shown, genes Hit Dist list, which <br /> have signi&#65533;?cantly low (P &lt; 0.01) fraction TA sites disrupted. (B) The 343 V. cholerae genes classi&#65533;?ed essential based HMM analyses were <br /> compared assignments Cameron et al. Genes representative transposon mutants ordered transposon library marked <br /> as &#8216;Essential&#8217;, genes contained transposon insertion separated groups: insertion 10% ends of <br /> the genes (&#8216;N/C terminal&#8217;), insertions central 80% gene&#8217;s coding sequence (&#8216;Core insertion&#8217;). (C) HMM-based gene <br /> classi&#65533;?cations shown 789 putative essential genes de&#65533;?ned Cameron et al (10). The frequency duplicated loci genes in <br /> the V. cholerae C6706 genome (duplicate/absent), classi&#65533;?ed HMM, shown. (D) HMM classi&#65533;?cation 332 genes <br /> identi&#65533;?ed domain essential using SW analysis. <br />  <br />  <br />  <br /> more accurate &#65533;?ltering genes regions containing                      optimal growth V. cholerae LB (Supplementary <br /> signi&#65533;?cantly fewer insertions.                                               Table S4). Interestingly, list contains 72 genes that <br />                                                                              overlap Hit Dist list (Supplementary Table S1), <br />                                                                              indicating sick genes had <br /> Identifying genes contribute optimum growth <br />                                                                              fewer templates (reads) library, a <br /> with HMM <br />                                                                              low frequency TA sites disrupted. In principle, the <br /> The HMM approach facilitates identi&#65533;?cation genes                     fraction TA sites disrupted gene vary <br /> required optimal growth permits detection                     &#8216;sick&#8217; neutral loci. However, relatively <br /> of regions consistently represented fewer reads                      low read depth data (average  10 reads per <br /> expected growth-neutral genes. Such &#8216;sick&#8217; genes                     transposon mutant; Supplementary Figure S1C) likely <br /> not readily distinguished growth neutral loci using                     results insertions sick loci undetected; <br /> the SW pipeline, missed studies treat insertions                 library greater read depth, expect the <br /> as binary events regardless reads (28), require                    Hit Dist list contain sick genes. <br /> outgrowth step &#65533;?tness-based studies (2). Our                           To validate HMM assignments sick genes, we <br /> HMM results indicate 128 genes required                         compared growth insertion mutants the <br />                                                                                         Nucleic Acids Research, 2013, Vol. 41, No. 19 9043 <br />  <br />  <br />  <br />  <br /> Figure 5. In vitro growth mutants transposon insertions sick genes. (A) Culture density (OD 600) wild-type V. cholerae mutants <br /> with transposon insertions &#8216;sick&#8217; gene candidates plotted function time. (B) Competitive indices vitro competition assays between <br /> wild-type V. cholerae mutants insertions candidate &#8216;sick&#8217; genes shown. Competitive indices calculated ratios mutant <br /> and wild-type CFUs overnight co-culture LB. *P &lt; 0.001. <br />  <br />  <br />  <br /> candidate genes ordered V. cholerae transposon                       essentiality&#8212;of tyrosyl tRNA synthetase paralogs <br /> library (10) growth wild-type cells. Mutants                     vc0465 vc0631, vc0631 appears essential. <br /> sick genes contained lowest level mapped reads                    Likewise, genes encoding glycerol 3 phosphate <br /> (average 1&#8211;2 reads/insertion; Supplementary Table S4) ex-                     acyltransferases (vc0093 vc2024), vc2024 is <br /> hibited reduced growth rate vitro &#65533;?ve genes                   required growth (Table 1). The nonessential paralogs <br /> tested (Figure 5A), con&#65533;?rming HMM analysis                        pseudogenes, diverged function dif- <br /> transposon-insertion sequencing data accurately                           ferentially expressed. <br /> identify mutations result growth defects.                                After removing paralogous loci, remain 14 <br /> Insertion mutants sick genes higher average                           nonessential genes V. cholerae considered es- <br /> mapped reads exhibited degree                              sential E. coli (Supplementary Table S9). In in- <br /> growth impairment. For example, insertion mutants                          stances, differences organisms be <br /> vc0309 vc0753, average reads 5.6                         explained conditional essentiality. ftsE ftsX on <br /> 4.7, respectively, severely attenuated growth                         E. coli essential list, E. coli strains inactivating <br /> (Figure 5A). Likewise, vc0237 vc1773 mutants                              mutations genes grown presence of <br /> (average 3 5.4 reads TA insertion)                       high salt (1% NaCl) (37). Most likely, V. cholerae ftsE and <br /> &#65533;?t wild-type cells in vitro competition experi-                    ftsX assigned nonessential category we <br /> ment (Figure 5B). However, vc0246, vc0259 vc0831                          isolated insertion library LB medium containing <br /> mutants (average reads TA insertion 5, 6.9                         1% NaCl. However, cases, differences essenti- <br /> 9.5, respectively) grew comparably wild-type                         ality E. coli V. cholerae re&#64258;ect interesting <br /> strain (Figure 5B). Thus, HMM calls candidate                          differences organisms&#8217; physiologies (Table 1). <br /> sick genes appear accurate insertions con-                    For example, &#65533;?ve essential E. coli genes ribABCDE are <br /> taining lower average reads, false-positive assign-                     required synthesis ribo&#64258;avin (vitamin B2) [reviewed <br /> ments likely genes higher average                           (38)], ribCDE homologs dispensable in <br /> reads. Because provided HMM relatively                          V. cholerae (ribA ribB paralogs genome). <br /> small uncharacterized training set candidate sick                      This observation suggests V. cholerae, like <br /> genes, possible improve accuracy                   <span id='am-1' about='obo:NCBITaxon_1578' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_33958 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-2'  ></span><span id='am-3' property="rdfs:label" content="Lactobacillus" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="PMID:21724957" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="PMID:19244449" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="PMID:19060088" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="PMID:18048724" datatype="xsd:string"></span>Lactobacillus</span> (39) Bacillus species (40), able <br /> HMM detecting sick genes training algorithm                        use ribo&#64258;avin present environment (e.g. LB <br /> with larger set biologically validated candidates.                       media), relying solely novo synthesis. In <br />                                                                               support hypothesis, ribE::Tn <br /> Differentially essential genes V. cholerae E. coli                     (vc2268::Tn) strain does grow M9 media, but <br /> It initially surprising &#65533;?nd 25 E. coli                     does grow M9 supplemented ribo&#64258;avin <br /> essential genes V. cholerae homologs neutral                        (Supplementary Figure S2). Thus, V. cholerae <br /> in study (Supplementary Table S9). However, 11                         lacks recognizable ribo&#64258;avin transporters, likely that <br /> these genes appear duplicated V. cholerae                        organism, unlike E. coli, encodes ribo&#64258;avin uptake <br /> genome cases likely functionally                        mechanism substitute novo synthesis. In <br /> redundant (Table 1). For example,                               addition ribCDE, homologs mukB, mukE and <br /> nonessential paralogs peptide deformylase (vc0046                          mukF, required chromosome compaction <br /> and vca0150), aspartate semialdehyde dehydro-                      E. coli (41), essential V. cholerae <br /> genase (vc2107 vc2036) V. cholerae. Intriguingly,                      (Table 1). We readily detected V. cholerae transposon <br /> two additional paralogous pairs exhibited differential                        insertions sites, mutants present in <br />  9044 Nucleic Acids Research, 2013, Vol. 41, No. 19 <br />  <br />  <br /> Table 1. Genes functions selected loci differentially essential V. cholerae E. coli <br />  <br /> Essential homolog         Neutral homolog               Homolog          Essential In         Annotated function <br /> V. cholerae                  V. cholerae                      E. coli             E. coli? <br />  <br /> vc0631                       vc0465                           tyrS                Yes                  Tyrosyl tRNA synthetase <br /> vc2024                       vc0093                           plsB                Yes                  Glycerol 3 phosphate acyltransferase <br />                              vc0743, vca0693                  secD                Yes                  General secretion pathway translocase <br />  <br />                              vc2269, vc1263                   ribA                Yes                  GTP cylclohydrolase subunit II <br />  <br />                              vc2269, vca1060                  ribB                Yes                  3,4 dihydroxy-2-butanone 4-phoshate synthase <br />  <br />                              vc2107, vc2036                   asd                 Yes                  Aspartate semialdehyde dehydrogenase <br />  <br />                              vc0058, vc0586, vca0274                          Yes                  Carbonic anhydrase <br />  <br />  <br />                              vc0046, vca0150                  def                 Yes                  Peptide deformylase <br />  <br />                              vc2268                           ribE                Yes                  Ribo&#64258;avin synthase <span id='am-44' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-49'  ></span>beta</span> subunit <br />                              vc2270                           ribC                Yes                  Ribo&#64258;avin synthase alpha subunit <br />                              vc2271                           ribD                Yes                  Ribo&#64258;avin-speci&#65533;?c deaminase <br />                              vc1714-vc1716                    mukBEF              Yes                  Condensin subunits BEF <br /> vc2626                                                        dam                 No                   Adenine Methyltransferase <br /> vc2763-vc2767                                                 atpCDGAH            No                   F1 ATP synthase (e,b,g,a,d) subunits <br /> vc2768-vc2770                                                 atpFEB              No                   F0 ATP synthase (B,C,A) subunits <br />  <br />  <br />  <br />  <br /> the ordered transposon insertion library (10), raising                    remodeling hypothetical proteins (Figure 6A, <br /> possibility V. cholerae encodes machinery outside                    Supplementary Table S10). The &#65533;?rst classes ex- <br /> mukBEF locus carry genome compaction.                            pected represent conserved multicomponent <br />    Of 343 V. cholerae genes classi&#65533;?ed HMM                              pathways required fundamental metabolic struc- <br /> analysis essential (at growth LB), 56                     tural functions bacteria, genes have <br /> be disrupted lack homologs E. coli (Supplementary                       predominated studies essential genes organ- <br /> Table S2). Some genes falsely                          isms (27,51,52). To better understand role se- <br /> classed essential HMM owing small size;                        quences initially annotated hypothetical essential genes <br /> however, list likely re&#64258;ect true differences                  (53), constitute 13% (45 genes) essential genes, <br /> in bacterial processes organisms,                         performed sequence protein structure homology <br /> does include previously described differentially essential                    searches. Notably, analyses medium high <br /> loci, including dam (vc2626), parA2 (vca1114) parB2                       con&#65533;?dence (e.g. PHYRE con&#65533;?dence values &gt; 50%) func- <br /> (vca1115) (Table 1) (42&#8211;44). The list includes                         tional predictions 33 genes (Figure 6B, <br /> number antitoxin genes putative toxin/antitoxin                       Supplementary Table S11). While substantial number <br /> addiction loci, including vca0360, vca0477, vca0486                       predicted involved metabolic and <br /> vca0488 (Supplementary Table S2). Such genes                              nucleic acid catabolism processes, enrichment <br /> presumed essential associated active                          phage toxin-associated genes (15 versus 1% of <br /> toxins, activity of V. cholerae&#8217;s                             total essential genes), essential antitoxins <br /> numerous toxin/antitoxin pairs previously                        described earlier (Supplementary Table S11). The preva- <br /> assessed. Genes newly reported essential                             lence phage/toxin-related genes essential <br /> V. cholerae include seven genes F0/F1 ATP                          hypotheticals likely re&#64258;ects diversity relatively <br /> synthase locus (vc2763-vc2770). ATP synthase essential                     understudied nature, centrality, horizontally <br /> in variety bacteria (45&#8211;48), E. coli                    acquired elements bacterial growth survival. <br /> (49), survive absence aerobic res- <br /> piration machinery fermentation (50). Vibrio cholerae                      Intergenic noncoding regions required for <br /> may metabolic requirements fermentation                         V. cholerae growth <br /> are optimally available LB. Thus, comparison                       Most transposon-insertion sequencing studies reported to <br /> of essential genes organisms uncover distinct                     date restricted analyses annotated open <br /> ecological physiological requirements species.                   reading frames, exception studies <br />                                                                               (11,27,52,54). However, SW HMM methods <br /> Predicted functions essential V. cholerae genes <br />                                                                               enable comprehensive analysis entire genome, <br /> Most 343 V. cholerae essential genes identi&#65533;?ed                      useful de&#65533;?ning unannotated <br /> fall major categories [based predicted gene                      regions important growth, cis acting <br /> ontology (GO) categorization annotated N16961                          sites ncRNAs. We used HMM-based pipeline to <br /> genome]: translation, metabolism, cell wall/envelope                          functionally categorize intergenic region the <br />                                                                                         Nucleic Acids Research, 2013, Vol. 41, No. 19 9045 <br />  <br />  <br />  <br />  <br /> Figure 6. Essential gene processes V. cholerae. (A) V. cholerae genes predicted essential based HMM analysis categorized their <br /> GO terms broader biological processes highlight essential activities V. cholerae. (B) Essential hypothetical proteins (A) subjected <br /> to homology searches protein fold threading evaluate additional functional assignments genes. These analyses <br /> allowed preliminary categorization biological roles 32 46 essential hypothetical proteins general biological categories. <br />  <br />  <br />  <br /> V. cholerae genome (Supplementary Table S12 S13),                         processing pre-tRNA RNA species (58,59). <br /> and identi&#65533;?ed 200 intergenic regions                           The HMM classi&#65533;?ed sick domain essential <br /> appear essential domain essential (Supplementary                     intergenic regions encode V. cholerae <br /> Table S12). The HMM-based assignments consistent                         ncRNAs [(8) unpublished]. In subsequent analyses, <br /> with experimental data. For example, closely mirror                      assessed extent correspondence experi- <br /> previous genetic studies de&#65533;?ning essential sequences                   mentally identi&#65533;?ed ncRNA coordinates regions found <br /> the chrI chrII origins replication (oriCI oriCII,                  SW analysis underrepresented insertion <br /> respectively) (30,55). The precision transposon-in-                    sites. This approach enabled identi&#65533;?cation eight <br /> sertion sequencing data illustrated reads                  intergenic transcribed loci associated signi&#65533;?cantly <br /> intergenic region includes oriCII (IG_vca0002). The 50                   fewer sequencing reads (P &lt; 0.05), likely be <br /> end intergenic sequence, known dis-                    required optimal growth vitro (Table 2). These <br /> pensable replication, contains Himar1 insertions,                         ncRNAs encode previously mentioned RNase P M1, <br /> whereas 30 sequence, known essential                       S15 Alpha RBS ncRNAs, &#65533;?ve novel ncRNA <br /> (30), contains insertions (Figure 3D). In fact,                   species functions remain uncharacterized. <br /> insertion intergenic region occurred position                  Interestingly, two, B2 M9_15_chrII_36, have <br /> 753, TA dinucleotide DnaA box                       signi&#65533;?cantly different expression vitro vivo <br /> essential replication. The HMM essential region                      conditions [(8) unpublished]. Additional combined <br /> includes sequence upstream rctB, essential                    analyses transposon-insertion sequencing trans- <br /> replication (30). Similarly, 41% essential domain                   criptomic data cells grown diverse environ- <br /> essential intergenic regions lie immediately upstream                      mental conditions powerful approach for <br /> predicted essential domain essential genes, consistent                     identifying regulatory ncRNAs mediate cell survival <br /> with expectation promoters 50 UTRs                          stress conditions. <br /> of essential genes required growth <br /> (Supplementary Table S12). In fact, transcribed 50 <br /> UTR regulatory sequences governing expression essen-                       CONCLUSIONS <br /> tial ribosomal proteins [S15 (56) Alpha RBS (57)]                         Our high-resolution analysis essentiality genic <br /> were de&#65533;?ned domain essential (Table 2),                          intergenic loci V. cholerae genome provides a <br /> analyses shown similar essential ribosomal leader se-                    wealth data considerable value future <br /> quences organisms (52).                                              studies important human pathogen. For example, <br />    Underrepresented intergenic sites                             knowledge essential genes pathways inform <br /> associated essential genes represent trans-                          screens new drugs treat cholera facilitate an <br /> acting ncRNA loci required optimal growth. While                          enhanced understanding speci&#65533;?c metabolic and <br /> there known essential ncRNAs, HMM algo-                           physiologic growth requirements organism. In <br /> rithm correctly identi&#65533;?ed RNase P ncRNA M1 es-                         addition, &#65533;?ne scale mapping allowed identi&#65533;?cation of <br /> sential (Table 2). M1 ribozyme forms catalytic                  &#8216;subdomains&#8217; genes intergenic regions that <br /> core RNase P, essential enzyme required                             required optimal growth. These regions included <br />  9046 Nucleic Acids Research, 2013, Vol. 41, No. 19 <br />  <br />  <br /> Table 2. A selected list intergenic regions signi&#65533;?cantly underrepresented transposon insertion <br />  <br /> Intergenic locus       Annotation                Category         Functional description <br />  <br /> IG_1                   Origin                    E                Chromosome I origin <br /> IG_vca0002             Origin                    D                Chromosome II origin: essential region 380 nucleotides that <br />                                                                     contain DnaA RctB boxes <br /> IG_vc2410              Rnase P                   E                M1 ribozyme compoent Rnase P, essential tRNA rRNA processing <br /> IG_vc2574              Alpha_RBS                 E                Ribosome binding site containing pseudoknot required expression 4 essen- <br />                                                                     tial ribosome components <br /> IG_vc0647              Rab_I_chrI_7              S                ncRNA discovered transcriptomics; function unknown <br /> IG_vc0431              C-CF                      D                ncRNA discovered transcriptomics; function unknown <br /> IG_vc0143              B2                        D                ncRNA discovered transcriptomics; expression signi&#65533;?cantly <br />                                                                     upregulated mouse rabbit infection <br /> IG_vc0646              S15                       D                Ribosome binding site containing pseudoknot required expression es- <br />                                                                     sential ribosomal gene rpsO <br /> IG_vc2473              RabM9_I_chrI_27           S                ncRNA discovered transcriptomics; signi&#65533;?cantly upregulated during <br />                                                                     rabbit infection <br /> IG_vca0933             M9_I5_chrII_36            D                ncRNA discovered transcriptomics; signi&#65533;?cantly downregulated during <br />                                                                     rabbit infection, upregulated mice <br />  <br /> Each intergenic region classi&#65533;?ed E = essential region; D = domain essential (contains subdomain essential); S = &#8216;sick&#8217; (nonessential, <br /> but required optimal growth vitro). <br />  <br />  <br />  <br />  <br /> not protein domains, promoters UTR se-                          ACKNOWLEDGEMENTS <br /> quences well. Furthermore, overlay transcriptomic                      We thank Bill Robins information regarding the <br /> data high-density transposon insertion data                    V. cholerae C6706 genome, members of <br /> set facilitated identi&#65533;?cation small ncRNA species                    Waldor lab discussion, especially Yoshi Yamaichi, <br /> appear regulate growth V. cholerae.                                    Andrea Mo&#168;ell Stavroula Hatzios editing. <br />    The approach developed analysis high- <br /> density transposon mutagenesis data broadly <br /> applicable useful future transposon-insertion <br /> sequencing studies. We propose factors                          FUNDING <br /> should routinely considered analyzing trans-                         Howard Hughes Medical Institute National <br /> poson-insertion sequencing data. First, correction                       Institutes Health [AI R37-42347 M.K.W., F32 <br /> DNA replication&#8212;especially data sets different                       GM108355-01 M.C.C.]; National Institute of <br /> growth conditions compared&#8212;allows direct com-                        Allergy Infectious Diseases, National Institutes of <br /> parisons data sets bias rate repli-                   Health, Department Health Human Services <br /> cation experiment. Second, genomically unbiased                      [HHSN272200900018C J.L.] (in with <br /> analysis pipelines, SW approach, permit                     Federal funds). Funding open access charge: Howard <br /> characterization requirement unannotated loci                     Hughes Medical Institute. <br /> (both outside coding regions) bacterial <br /> growth. Finally, importantly, incorpor-                     Con&#64258;ict statement. None declared. <br /> ation HMM-based &#65533;?lter enhance biological <br /> signal intrinsic noise high-density transposon <br /> data sets HHM takes advantage fea-                          REFERENCES <br /> ture inherent transposon-insertion sequencing data                      1. Barquist,L., Boinett,C.J. Cain,A.K. (2013) Approaches to <br /> sets&#8212;local context information&#8212;that ex-                              querying bacterial genomes transposon-insertion sequencing. <br /> ploited previous analyses. Additionally,                             RNA Biol., 10, 1161&#8211;1169. <br /> modi&#65533;?cations, HMM approach useful                          2. van Opijnen,T. Camilli,A. (2013) Transposon insertion <br />                                                                                   sequencing: new tool systems-level analysis of <br /> nonmariner&#8211;based transposon insertion sequence tech-                              microorganisms. Nat. Rev. Microbiol, 11, 435&#8211;442. <br /> niques TraDIS (6). Moreover, HMM-based                                 3. van Opijnen,T., Bodi,K.L. Camilli,A. (2009) Tn-seq: high- <br /> approaches adaptable analyses condi-                             throughput parallel sequencing &#65533;?tness genetic interaction <br /> tional essentiality, data multiple conditions                          studies microorganisms. Nat. Methods, 6, 767&#8211;772. <br />                                                                                4. Gawronski,J.D., Wong,S.M., Giannoukos,G., Ward,D.V. and <br /> and genetic backgrounds compared. Thus, antici-                            Akerley,B.J. (2009) Tracking insertion mutants libraries <br /> pate study &#65533;?rst applications                         deep sequencing genome-wide screen <span id='am-66' about='obo:NCBITaxon_724' typeof='owl:Thing obo:NCBITaxon_135625 obo:NCBITaxon_712 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236'><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="Haemophilus" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Haemophilus Winslow et al. 1917" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="PMID:1736960" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-72'  ></span>Haemophilus</span> <br /> for HMM-based analyses transposon-insertion                                    genes required lung. Proc. Natl Acad. Sci. USA, 106, <br /> sequencing data.                                                                  16422&#8211;16427. <br />                                                                                5. Goodman,A.L., McNulty,N.P., Zhao,Y., Leip,D., <span id='am-148' about='obo:NCBITaxon_69650' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_57612 obo:NCBITaxon_69649 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-149' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-150' property="rdfs:label" content="Mitra" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-152'  ></span>Mitra</span>,R.D., <br />                                                                                   Lozupone,C.A., Knight,R. Gordon,J.I. (2009) Identifying <br />                                                                                   genetic determinants needed establish human gut symbiont in <br /> SUPPLEMENTARY DATA                                                                habitat. Cell Host Microbe., 6, 279&#8211;289. <br />                                                                                6. Langridge,G.C., Phan,M.D., Turner,D.J., Perkins,T.T., Parts,L., <br /> Supplementary Data available NAR Online.                                   Haase,J., Charles,I., Maskell,D.J., Peters,S.E., Dougan,G. et al. <br />                                                                                    Nucleic Acids Research, 2013, Vol. 41, No. 19 9047 <br />  <br />  <br />     (2009) Simultaneous assay <span id='am-130' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-131' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-137'  ></span><span id='am-138' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-142' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span>Salmonella</span> Typhi gene using            essentiality based sequencing transposon insertion libraries. <br />     million transposon mutants. Genome Res, 19, 2308&#8211;2316.                Bioinformatics, 29, 695&#8211;703. <br />  7. Amitai,M. (1998) Hidden models biopolymers. Science, 282,          29. Koch,B., Ma,X. Lobner-Olesen,A. (2012) rctB mutations that <br />     1436&#8211;1437.                                                                increase copy number Vibrio cholerae oriCII Escherichia <br />  8. Mandlik,A., Livny,J., Robins,W.P., Ritchie,J.M., Mekalanos,J.J.           coli. Plasmid, 68, 159&#8211;169. <br />     Waldor,M.K. (2011) RNA-Seq-based monitoring infection-         30. Venkova-Canova,T., Srivastava,P. Chattoraj,D.K. (2006) <br />     linked changes Vibrio cholerae gene expression. Cell Host              Transcriptional inactivation regulatory site replication of <br />     Microbe., 10, 165&#8211;174.                                                    Vibrio cholerae chromosome II. Proc. Natl Acad. Sci. USA, 103, <br />  9. Chiang,S.L. Rubin,E.J. (2002) Construction mariner-              12051&#8211;12056. <br />     based transposon epitope-tagging genomic targeting.           31. Kato,J. Hashimoto,M. (2007) Construction consecutive <br />     Gene, 296, 179&#8211;185.                                                       deletions Escherichia coli chromosome. Mol. Syst. Biol., 3, <br /> 10. Cameron,D.E., Urbach,J.M. Mekalanos,J.J. (2008) A de&#65533;?ned              132. <br />     transposon mutant library use identifying motility genes   32. Ehretsmann,C.P., Carpousis,A.J. Krisch,H.M. (1992) <br />     Vibrio cholerae. Proc. Natl Acad. Sci. USA, 105, 8736&#8211;8741.            Speci&#65533;?city Escherichia coli endoribonuclease RNase E: vivo <br /> 11. Zhang,Y.J., Ioerger,T.R., Huttenhower,C., Long,J.E.,                      vitro analysis mutants bacteriophage T4 mRNA <br />     Sassetti,C.M., Sacchettini,J.C. Rubin,E.J. (2012) Global              processing site. Genes Dev, 6, 149&#8211;159. <br />     assessment genomic regions required growth                  33. Mudd,E.A., Krisch,H.M. Higgins,C.F. (1990) RNase E, an <br />     <span id='am-10' about='obo:NCBITaxon_1763' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1762 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003'><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="PMID:1883713" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="PMID:7547284" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="PMID:1581193" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="PMID:2275850" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Mycobacterium" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="PMID:7547304" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19'  ></span><span id='am-20' property="oboInOwl:hasDbXref" content="PMID:1380284" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="PMID:8863452" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium Lehmann and Neumann 1896" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="PMID:16014496" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="PMID:7907223" datatype="xsd:string"></span>Mycobacterium</span> tuberculosis. PLoS Pathog, 8, e1002946.                     endoribonuclease, general role chemical decay of <br /> 12. Martin,M. (2011) Cutadapt removes adapter sequences high-            Escherichia coli mRNA: evidence rne ams same <br />     throughput sequencing reads. EMBnet J., 17, 3.                            genetic locus. Mol. Microbiol., 4, 2127&#8211;2135. <br /> 13. Langmead,B. Salzberg,S.L. (2012) Fast gapped-read                 34. Ow,M.C., Liu,Q. Kushner,S.R. (2000) Analysis mRNA <br />     alignment Bowtie 2. Nat. Methods, 9, 357&#8211;359.                        decay rRNA processing Escherichia coli absence of <br /> 14. Mutreja,A., Kim,D.W., Thomson,N.R., Connor,T.R., Lee,J.H.,                RNase E-based degradosome assembly. Mol. Microbiol., 38, <br />     Kariuki,S., Croucher,N.J., Choi,S.Y., Harris,S.R., Lebens,M.              854&#8211;866. <br />     et al. (2011) Evidence waves global transmission    35. Claverie-Martin,F., Diaz-Torres,M.R., Yancey,S.D. and <br />     seventh cholera pandemic. Nature, 477, 462&#8211;465.                       Kushner,S.R. (1989) Cloning altered mRNA stability (ams) <br /> 15. Carver,T., Thomson,N., Bleasby,A., Berriman,M. Parkhill,J.            gene <span id='am-169' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-170' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-171' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-173' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span>Escherichia coli K-12</span>. J. Bacteriol., 171, 5479&#8211;5486. <br />     (2009) DNAPlotter: circular linear interactive genome             36. Yamaichi,Y., Gerding,M.A., Davis,B.M. Waldor,M.K. (2011) <br />     visualization. Bioinformatics, 25, 119&#8211;120.                               Regulatory cross-talk links Vibrio cholerae chromosome II <br /> 16. R Development Core Team. (2008) R: A language                         replication segregation. PLoS Genet, 7, e1002189. <br />     environment statistical computing. R Foundation Statistical   37. Reddy,M. (2007) Role FtsEX cell division Escherichia <br />     Computing, Vienna, Austria.                                               coli: viability ftsEX mutants dependent functional SufI or <br /> 17. Benjamini,Y. Hochb <br /> </body></html>