<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Research <br />  <br /> Coding exons function tissue-specific enhancers <br /> of nearby genes <br /> Ramon Y. Birnbaum,1,2 E. Josephine Clowney,3,4 Orly Agamy,5 Mee J. Kim,1,2 <br /> Jingjing Zhao,1,2,6 Takayuki Yamanaka,1,2 Zachary Pappalardo,1,2 Shoa L. Clarke,7 <br /> Aaron M. Wenger,8 Loan Nguyen,1,2 Fiorella Gurrieri,9 David B. Everman,10 <br /> Charles E. Schwartz,10,11 Ohad S. Birk,5 Gill Bejerano,8,12 Stavros Lomvardas,3 <br /> and Nadav Ahituv1,2,13 <br /> 1 <br />  Department Bioengineering Therapeutic Sciences, 2Institute Human Genetics, 3Department Anatomy, 4Program in <br /> Biomedical Sciences, University <span id='am-6' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>California</span>, San Francisco, California 94143, USA; 5The Morris Kahn Laboratory Human Genetics, <br /> NIBN, Ben-Gurion University, Beer-Sheva 84105, Israel; 6Key Laboratory Advanced Control Optimization Chemical <br /> Processes Ministry Education, East China University Science Technology, Shanghai 200237, China; 7Department of <br /> Genetics, 8Department Computer Science, Stanford University, Stanford, California 94305-5329, USA; 9Istituto di Genetica Medica, <br /> Universita` Cattolica S. Cuore, Rome 00168, Italy; 10JC Self Research Institute, Greenwood Genetic Center, Greenwood, South Carolina <br /> 29646, USA; 11Department Genetics Biochemistry, Clemson University, Clemson, South Carolina 29634, USA; 12Department of <br /> Developmental Biology, Stanford University, Stanford, California 94305-5329, USA <br />  <br />  <br />        Enhancers essential gene regulatory elements alteration lead morphological differences <span id='am-33' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>, <br />        developmental abnormalities, human disease. Current strategies identify enhancers focus primarily noncoding se- <br />        quences tend exclude protein coding sequences. Here, analyzed 25 available ChIP-seq <span id='am-58' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-60'  ></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> sets identify enhancers <br />        unbiased manner (H3K4me1, H3K27ac, EP300) peaks overlap exons. We that, average, 7% ChIP- <br />        seq peaks overlap coding exons (after excluding peaks overlap exons). By using mouse zebrafish enhancer <br />        assays, demonstrate exonic enhancer (eExons) candidates function enhancers neighboring <br />        genes exonic sequence necessary enhancer activity. Using ChIP, 3C, DNA FISH, one <br />        exonic limb enhancers, Dync1i1 exon 15, active enhancer marks physically interacts Dlx5/6 promoter regions <br />        900 kb away. In addition, removal chromosomal abnormalities humans cause split hand foot malformation <br />        1 (SHFM1), disorder associated DLX5/6. These results demonstrate DNA sequences dual function, operating <br />        coding exons tissue enhancers nearby gene(s) tissue, suggesting phenotypes resulting coding <br />        mutations caused protein alteration disrupting regulation gene. <br />        [Supplemental material available article.] <br />  <br /> Precise temporal, spatial, quantitative regulation gene ex-                2012). In addition, recent study scanning synonymous <br /> pression essential proper development. <span id='am-23' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27'  ></span>This</span> tight transcrip-               constraint protein coding regions (Lin et al. 2011) an <br /> tional regulation mediated DNA sequences called en-                 overlap eExons (Lampe et al. 2008; Tumpel <br /> hancers, regulate gene promoters. By use comparative                     et al. 2008) synonymous constraint elements. Here, ana- <br /> genomics chromatin immunoprecipitation followed next-                       lyzed 25 available ChIP-seq data sets enhancer marks <br /> generation sequencing (ChIP-seq), candidate enhancer sequences                    (H3K4me1, H3K27ac, EP300, known p300) their <br /> can identified relatively high-throughput manner                      overlap coding exons. Following analysis, wanted <br /> (Heintzman Ren 2009; Visel et al. 2009b). These sequences                 specifically determine eExons regulate their <br /> then assayed enhancer activity using various vitro               neighboring genes gene reside in. This of <br /> vivo assays (Woolfe et al. 2005; Pennacchio et al. 2006; Heintzman                phenotypic implications coding <br /> et al. 2009). However, majority experiments remove                   mutations nearby genes. For purpose, ana- <br /> coding sequences analyses assumption                    lyzed specific EP300 ChIP-seq data set mouse embryonic <br /> they function enhancers, protein coding                    day (E) 11.5 limb tissue (Visel et al. 2009a), ability to <br /> role.                                                                             identify active enhancers high accuracy (88%) tissue <br />       Previous exonic enhancers (eExons) reported                    specificity (80%) vivo. At E11.5, mouse limb development <br /> vertebrates (Neznanov et al. 1997; Lampe et al. 2008; Tumpel et al.               progresses axes: proximal-distal (P-D), anterior-poste- <br /> 2008; Dong et al. 2010; Eichenlaub Ettwiller 2011; Ritter et al.              rior (A-P), dorsal-ventral (D-V). Specific signaling centers the <br />                                                                                   limb bud create gradients feedback loops determine these <br />                                                                                   axes (Gilbert 2000; Nissim Tabin 2004; Zeller et al. 2009), and <br /> 13 <br />    Corresponding author.                                                          alteration lead morphological differences. In this <br /> E-mail nadav.ahituv@ucsf.edu. <br /> Article published online print. Article, supplemental material, pub-   study, focused identifying limb eExons involved de- <br /> lication date http://www.genome.org/cgi/doi/10.1101/gr.133546.111.         velopment P-D A-P axes. <br />  <br />  <br /> 22:1059&#8211;1068 &#211; 2012, Published Cold Spring Harbor Laboratory Press; ISSN 1088-9051/12; www.genome.org               Genome Research            1059 <br />                                                                                                                           www.genome.org <br />  Birnbaum et al. <br />  <br />  <br />       The apical ectodermal ridge (AER) signaling center            15, using chromatin conformation capture (3C) DNA fluo- <br /> keeps underlying mesenchyme proliferative state al-                rescent situ hybridization (FISH) showed physically in- <br /> lows limb grow, governing P-D <span id='am-28' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-29' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-30'  ></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Axis" datatype="xsd:string"></span>axis</span>. In devel-               teracts Dlx5/6 promoter regions developing mouse <br /> oping mouse limb bud, distal-less homeobox 5 6 (Dlx5/6)                 limb. Mutation analysis individuals SHFM1 indicated that <br /> genes expressed AER (Fig. 1D9,E9), Dlx5                  chromosomal aberrations encompassing enhancer be <br /> expressed anterior mesenchyme (Fig. 1E9). Disruption                  causes SHFM1 phenotype. Combined, these <br /> both Dlx5 Dlx6 mice leads split hand foot mal-                  findings demonstrate DNA sequence function a <br /> formation (SHFM) phenocopy (Robledo et al. 2002). In humans,                    coding exon tissue enhancer different tissue <br /> chromosomal aberrations DLX5/6 region,                  suggest need cautious assigning coding mu- <br /> not encompass coding sequences DLX5/6, cause SHFM1                       tation phenotype protein function. <br /> (MIM 183600) associated incomplete penetrance <br /> with intellectual disability, craniofacial malformations and <br /> deafness (Elliott Evans 2006). Other <span id='am-17' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span>                   Results <br /> a DLX5 missense mutation (Shamseldin et al. 2012), cod- <br /> ing mutation gene individuals                  Exon overlap analyses enhancer-associated ChIP-seq data sets <br /> SHFM1, suggesting disruption DLX5/6 gene regulatory ele-                    To determine genome-wide prevalence enhancer-associated <br /> ments lead SHFM1 phenotype.                                              ChIP-seq peaks overlap coding exons, analyzed 25 available <br />       The A-P axis controlled signaling center called zone              ChIP-seq data sets enhancer marks (H3K4me1, H3K27ac, and <br /> of polarizing activity, located posterior mesenchyme                     EP300) various human cell lines mouse E11.5 tissues <br /> defined expression Sonic Hedgehog (SHH). Twist1,                     (Supplemental Table 1; Methods). Since enhancer marks <br /> transcription factor inhibits Shh expression anterior                   identify potential promoters, looked overlap <br /> limb bud antagonizing HAND2, Shh-positive regulator (Firulli                   coding exons excluding exon. In anal- <br /> et al. 2005). Homozygous Twist1-null mice limb bud devel-                      yses described study, coding exons defined only <br /> opmental defects (Chen Behringer 1995), heterozygous                        exons exons. Analysis individual his- <br /> mice develop polydactyly (Bourgeois et al. 1998) ectopic                   tone marks, H3K4me1 H3K27ac, showed 7% 10% all <br /> Shh expression (O&#8217;Rourke et al. 2002). In humans, mutations                      peaks overlap coding exons, respectively (Supplemental Table 1). It is <br /> TWIST1 lead various syndromes, majority encom-                      worth noting average peak size H3K4me1 and <br /> pass various forms limb malformations (MIM*601622).                              H3K27ac ChIP-seq data sets 2441 3107 bp, average <br />       Here, examining various enhancer-associated ChIP-seq data                  size peaks overlapping coding exons 3476 4195 bp for <br /> sets, characterized general prevalence peaks overlap                 mark. Analysis average exon size human genome <br /> exons. We chose seven limb EP300 ChIP-seq exonic se-                           shows ;280 bp (see Methods). Compared aver- <br /> quences functionally tested enhancer activity using                    age peak size histone marks overlap exons in <br /> a mouse transgenic enhancer assay. Four seven tested exonic                  particular, quite possible functional entity peak <br /> sequences shown functional limb enhancers                         does constitute exon percentages an <br /> mouse. Further analysis enhancers, Dync1i1 eExon                    overestimate. Therefore, analyzed different EP300 ChIP-seq <br />                                                                                                                 data sets various human cell lines <br />                                                                                                                 shorter peak sizes, average <br />                                                                                                                 426 bp. In data sets, <br />                                                                                                                 average, 4% the <br />                                                                                                                 peaks overlapped coding exons (Sup- <br />                                                                                                                 plemental Table 1). To better indi- <br />                                                                                                                 cation sequences func- <br />                                                                                                                 tional enhancers, examined overlap <br />                                                                                                                 exonic EP300 ChIP-seq peaks <br />                                                                                                                 H3K4me1 H3K27ac peaks. We <br />                                                                                                                 used ChIP-seq data different <br />                                                                                                                 cell lines, GM12878 K562, where <br />                                                                                                                 enhancer marks avail- <br />                                                                                                                 able. We 8% 5% the <br /> Figure 1. eExons DYNC1I1 HDAC9 characterized using mouse transgenic enhancer as-                   ChIP-seq peaks en- <br /> say. A schematic DYNC1I1-DLX5/6 (A) HDAC9-TWIST1 (F ) genomic regions. Black boxes               hancer marks overlapped coding exons <br /> orange ovals represent coding exons positive eExons, respectively. The black arrows point the <br /> genes thought regulated eExons. (B&#8211;C9) Mouse enhancer assays DYNC1I1 eExon             GM12878 K562 cells, respectively <br /> 15 17 embryonic day 11.5 (E11.5). (B,B9) DYNC1I1 eExon 15 shows apical ectodermal ridge (AER)            (Supplemental Table 1). We screened <br /> and limb mesenchyme enhancer activity (red arrows), (C,C9) DYNC1I1eExon 17 shows anterior limb              coding sequences en- <br /> bud mesenchyme enhancer activity (red arrow). (D,E ) Mouse whole-mount situ hybridization Dlx6            hancer marks overlap with <br /> and Dlx5. (D9,E9) Dlx6 Dlx5 limb expression pattern similar DYNC1I1 eExon 15 enhancer ac- <br /> tivity. In addition, Dlx5 expressed anterior limb bud depicted red arrow (E9), similar  recently published study scanned <br /> DYNC1I1 eExon 17 enhancer activity (C9). (G&#8211;H9) Mouse enhancer assays HDAC9 eExons 18 19.                genome synonymous constraint <br /> (G,G9) HDAC9 eExon 18 shows anterior limb bud enhancer activity (red arrows), (H,H9) HDAC9                  protein coding regions (Lin et al. 2011). <br /> eExon 19 shows posterior limb bud (red arrows) branchial arch enhancer activity E11.5 mice. (I )         We 9% coding exons all <br /> Mouse whole-mount situ hybridization Twist1 E11.5. (I9) Twist1 limb expression pattern similar <br />                                                                                                                 ChIP-seq enhancer marks overlapped <br /> to HDAC9 eExon 18 anterior limb bud enhancer activity (G9) marked red arrow HDAC9 eExon <br /> 19 posterior limb bud enhancer activity (H9). For B, C, G, H, numbers right corner        synonymous constraint exons for <br /> indicate number embryos showing limb expression pattern/total LacZ stained embryos.                 GM12878 K562 cell lines (Sup- <br />  <br />  <br /> 1060      Genome Research <br />              www.genome.org <br />                                                                                                       Exonic enhancers nearby genes <br />  <br />  <br /> plemental Table 2). In summary, average, 7%       cost-efficient, carried deletion series analyses using this <br /> peaks 25 enhancer-associated ChIP-seq data sets ana-       assay. We characterized functional mouse limb <br /> lyzed overlapped coding exons removing exons.                 enhancers positive fin enhancer expression zebrafish. <br />                                                                           The limb enhancers (Supplemental Table 3) cloned from <br /> Analysis EP300 limb ChIP-seq data set eExon                      human genomic DNA zebrafish enhancer assay vector, con- <br /> candidate selection                                                       taining E1b minimal promoter followed green fluorescent <br />                                                                           protein (GFP) reporter gene (Li et al. 2009). These <span id='am-44' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="vectors" datatype="xsd:string"></span>vectors</span> were <br /> Given eExons previously discovered regulate <br />                                                                           microinjected one-cell-stage zebrafish embryos the <br /> the gene reside (Neznanov et al. 1997; Lampe et al. 2008; <br />                                                                           Tol2 transposase facilitate genomic integration. GFP expression <br /> Tumpel et al. 2008; Ritter et al. 2012), explicitly set to <br />                                                                           monitored 48 72 h post-fertilization (hpf), time <br /> search coding eExons autoregulate could <br />                                                                           points pectoral fin observed. Two four <br /> regulate nearby genes. This the <br />                                                                           functional mouse limb enhancers, DYNC1I1 eExon 15 HDAC9 <br /> phenotypic consequences coding mutations on <br />                                                                           eExon 19, functional fin enhancers zebrafish <br /> their nearby genes. In <span id='am-36' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-37' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> this, needed tissue-specific <br />                                                                           (Supplemental Table 4). At 72 hpf, DYNC1I1 eExon 15 drove GFP <br /> ChIP-seq enhancer data set. We focused analysis EP300 <br />                                                                           expression pectoral fin, caudal fin, somitic muscles (Sup- <br /> ChIP-seq data sets shown predict functional en- <br />                                                                           plemental Fig. 1B), HDAC9 eExon 19 exhibited enhancer ac- <br /> hancers different mouse E11.5 tissues (forebrain, mid- <br />                                                                           tivity pectoral fin branchial arch (Supplemental Fig. 2C). <br /> brain, limb) high accuracy tissue specificity (Visel et al. <br />                                                                                 In order determine actual exonic sequences are <br /> 2009a). In data set, observed 4% EP300 ChIP-seq <br />                                                                           necessary enhancer activity, used fin enhancers, <br /> peaks tissues overlap coding exons excluding <br />                                                                           DYNC1I1 eExon 15 HDAC9 eExon 19, deletion series anal- <br /> the exon (Supplemental Table 1). These lower percentages could <br />                                                                           yses. DYNC1I1 eExon 15 divided segments&#8212;59 in- <br /> be experimental differences cell line versus tissue. For <br />                                                                           tron, exon, 39 intron (Fig. 2A)&#8212;and HDAC9 eExon 19 di- <br /> our functional assays, focused limb EP300 ChIP-seq <br />                                                                           vided following segments: 59 distal intron, 59 proximal <br /> data set. We scanned data set exonic sequences reside in <br />                                                                           intron, exon (Fig. 2C). We cases exon <br /> genes known expressed limb located in <br />                                                                           59 intron sequence adjacent exon lower enhancer <br /> the vicinity (up 1 Mb side) known limb-associated genes <br />                                                                           activity themselves, combined, enhancer activity <br /> (see Methods). From 252 limb EP300 ChIP-seq peaks overlaid <br />                                                                           substantially increased comparable previously <br /> exons, 152 sequences overlapped coding exons 134 in <br />                                                                           injected longer version DYNC1I1 eExon 15 HDAC9 eExon 19, <br /> a gene expressed limb (Supplemental Table 3). <br />                                                                           respectively (Fig. 2B,D). These results demonstrate exonic <br /> Out 134 sequences, 90 limb expressed <br />                                                                           sequences necessary sufficient enhancer activity. <br /> gene 1 Mb away gene. We chose seven <br /> exons near important limb developmental genes (C14orf49 exon <br /> 16 [near DICER1], CDC14B exon 13 [near PTCH1], DYNC1I1 exon               Limb genes associated eExons enhancer function <br /> 15 exon 17 [near DLX5/6], HDAC9 exon 18 exon 19 [near <br />                                                                           In order identify limb expressed genes regu- <br /> TWIST1], STX18 exons 4&#8211;5 [near MSX1]) subsequent mouse <br />                                                                           lated characterized eExons, analyzed RNA expres- <br /> enhancer assays (Supplemental Table 3). <br />                                                                           sion nearby genes carried synteny block analysis (Ahituv <br />                                                                           et al. 2005). Whole-mount situ hybridization neighboring <br /> Mouse enhancer assays                                                     genes Dlx5/6 similar limb expression patterns <br /> To test exonic sequences function enhancers,          DYNC1I1 eExons 15 and17 (Fig. 1B&#8211;E9; Supplemental Figs. 1,3), <br /> tested seven sequences enhancer activity mice. The       Twist1 limb expression pattern similar HDAC9 <br /> human sequences cloned Hsp68-LacZ vector               eExons 18 19 (Fig. 1G&#8211;-I9; Supplemental Figs. 2,3). We also <br /> contains heat shock protein 68 minimal promoter followed              extracted RNA E11.5 mouse limbs adult mouse cortex and <br /> by LacZ reporter gene (Kothary et al. 1988). Transgenic mice       heart performed quantitative PCR (qPCR) validate tissue <br /> generated, embryos harvested E11.5 stained            specific expression genes. Dlx5, Dlx6, Twist1 were <br /> LacZ. We seven sequences showed limb           expressed E11.5 limbs (Supplemental Fig. 3F,G). However, <br /> enhancer activity mice. DYNC1I1 eExon 15 drove specific LacZ           Dync1i1 Hdac9 detected mouse E11.5 limbs but <br /> expression limb mesenchyme AER (Fig. 1B,B9; Supple-            expressed mouse adult cortex heart, respectively (Sup- <br /> mental Fig. 1A), DYNC1I1 eExon 17 drove specific LacZ expres-         plemental Fig. 3F,G). In addition, examination genomic <br /> sion anterior limb mesenchyme (Fig. 1C,C9; Supplemental Fig.       location DYNC1I1-DLX5/6 HDAC9-TWIST1 various <br /> 1C). HDAC9 eExon 18 showed enhancer activity anterior limb         vertebrate genomes shows remain adjacent each <br /> bud (Fig. 1G,G9; Supplemental Fig. 2A) HDAC9 eExon 19          human fish (Supplemental Fig. 4). Based previous <br /> posterior limb bud (Fig. 1H,H9; Supplemental Fig. 2B).                    analysis (Ahituv et al. 2005), human&#8211;mouse&#8211;chicken DYNC1I1- <br />                                                                           DLX5/6 block 1.37 Mb size HDAC9-TWIST1 2.52 Mb, <br />                                                                           1.02 Mb average length (N50) human&#8211;mouse&#8211; <br /> The exonic sequence necessary enhancer activity                    chicken synteny block study. These results suggest that <br /> The sequences tested mouse enhancer assays                eExons important DLX5/6 TWIST1 regulation. <br /> intronic regions ChIP-seq peak overlapping the <br /> intron (Supplemental Table 3). We wanted assess whether <br /> the exonic sequence necessary enhancer activity. Since hu-         Dync1i1 eExon 15 marked limb enhancer <br /> man limb zebrafish fin development considered highly              chromatin signature <br /> comparable molecular level (Hall 2007; Iovine 2007;                To examine dual role DNA sequences, chose DYNC1I1 <br /> Mercader 2007) zebrafish enhancer assays rapid          eExon 15 functional analysis. We analyzed eExon <br />  <br />  <br />                                                                                                                Genome Research            1061 <br />                                                                                                                   www.genome.org <br />  Birnbaum et al. <br />  <br />  <br />  <br />  <br /> Figure 2. Segmental analysis DYNC1I1 eExon 15 HDAC9 eExon 19 enhancer function zebrafish. (A) DYNC1I1 eExon 15 divided three <br /> overlapping segments: 59 intron, exon, 39 intron. (Below) The UCSC Genome Browser (http://genome.ucsc.edu) conservation track shows 59 <br /> intron exon conserved human fish. (B) Zebrafish enhancer assay results different DYNC1I1 eExon 15 segments. While 59 <br /> intron exon enhancer activity fins somitic muscles, combination gives comparable enhancer expression 608-bp <br /> originally injected fragment DYNC1I1 eExon 15. The 39 intron segment did enhancer activity. (C ) HDAC9 eExon 19 divided three <br /> overlapping segments: distal 59 intron, proximal 59 intron, exon. (Below) The UCSC Genome Browser conservation track shows proximal 59 <br /> intron exon conserved human fish. (D) Zebrafish enhancer assay results different HDAC9 eExon 19 segments. While the <br /> proximal 59 intron exon enhancer activity pectoral fin branchial arches, combination gave comparable enhancer <br /> expression previously injected 1098-bp HDAC9 eExon 19 sequence. The distal 59 intron segment did enhancer activity. Enhancer function <br /> is plotted percentage GFP expression/total live embryos. Each segments injected 100 zebrafish embryos. <br />  <br />  <br /> for histone modification signatures limb development. We               heart stage (Fig. 1D,E). We observed increased <br /> carried ChIP followed qPCR (ChIP-qPCR) Dync1i1 eExon                interaction frequency Dync1i1 eExon 15 Dlx5/6 <br /> 15 enhancer (H3K4me1, H3K27ac), promoter (H3K4me3),                   promoters limb tissue compared heart, indicating <br /> transcribed gene (H3K36me3) chromatin signatures (Hon et al.                  physical interaction limb (Fig. 4B). These <br /> 2009). We mouse E11.5 limb, Dync1i1 eExon 15                results suggest Dync1i1 eExon 15 functions enhancer in <br /> is marked H3K4me1 H3K27ac (Fig. 3B,C)                       AER enhancer&#8211;promoter DNA looping. <br /> H3K4me3 H3K36me3 (Fig. 3D,E). In contrast, Dync1i1 exon 6                        To analyze chromosomal conformation the <br /> was marked H3K4me1 H3K27ac (Fig. 3B,C) limb,                 Dlx5/6 locus limb development, performed DNA FISH <br /> and Dlx5/6 coding exons marked H3K36me3 (Fig. 3E).                    using Dlx5/6 Dync1i1 eExon 15 probes mouse E11.5 limb <br /> Thus, chromatin status correlates proposed limb en-              buds heart. After capturing images fluorescent signals, <br /> hancer activity Dync1i1 eExon 15.                                          physical distance Dync1i1 eExon 15 Dlx5/6 <br />                                                                               coding region calculated (Fig. 4C&#8211;J). Frequency distribution pat- <br />                                                                               terns physical distance Dync1i1 eExon 15 Dlx5/6 <br /> 3C DNA FISH Dync1i1 eExon 15 physically                         AER compared heart measured (Fig. 4K,L). In the <br /> interacts promoter regions DLX5/6                                 AER, 35% Dync1i1 eExon 15 signals close proximity to <br /> To determine Dync1i1 eExon 15 physically interacts               Dlx5/6 signals (&lt;0.2 mm) (Fig. 4K), mean distance 0.32 6 <br /> the Dlx5/6 promoter regions, carried 3C mouse E11.5                 0.06 mm. In contrast, frequency colocalized signals heart <br /> heart limb (AER enriched; Methods) tissues. The mouse                 greatly reduced (12%; P &lt; 0.01, t-test) (Fig. 4L), overall <br /> heart tissue served negative control, Dlx5/6 expressed        frequency separated signals higher compared AER <br />  <br />  <br /> 1062      Genome Research <br />              www.genome.org <br />                                                                                                             Exonic enhancers nearby genes <br />  <br />  <br />  <br />  <br /> Figure 3. Histone modification signatures Dync1i1 eExon 15 mouse E11.5 limb bud. (A) Schematic representation Dync1i1-Dlx5/6 <br /> locus, showing relative positions primer sets used ChIP-qPCR analyses: Dync1i1 exon 6 eExon 15; Dlx6 promoter (pro.) exon 2; Dlx5 exon <br /> 2 promoter (pro.). (B) ChIP&#8211;qPCR analyses H3K4me1, enhancer histone mark. (C ) ChIP-qPCR analyses H3K27ac, active enhancer histone <br /> mark. (D) ChIP-qPCR analyses H3K4me3, promoter histone mark. (E ) ChIP-qPCR analyses H3K36me3, transcribed gene histone mark. (X-axis) <br /> Primer pairs; (y-axis) percentage input recovery. (Error bars) SE technical replicates representative experiment. <br />  <br />  <br /> (P &lt; 0.01, t-test) (Fig. 4L), mean distance 0.47 6 0.3 mm.           Discussion <br /> These results Dync1i1 eExon 15 close proximity with <br />                                                                                Studies aimed discovering gene regulatory elements usually <br /> Dlx5/6 promoter regions developing AER E11.5, supporting <br />                                                                                concentrate noncoding DNA sequences potential candidates <br /> its proposed role enhancer limb development. <br />                                                                                ignore coding sequences. However, studies shown <br />                                                                                protein coding sequences additional encrypted in- <br /> Human chromosomal aberrations encompassing DYNC1I1                             formation sequence (Chamary et al. 2006; Itzkovitz and <br /> eExons 15 17 associated SHFM1                                     Alon 2007; Lin et al. 2011). Here, analyzing ChIP-seq data sets <br /> To test alterations DYNC1I1 eExon 15 17                   enhancer marks various cell lines tissues, found <br /> be associated limb phenotype, analyzed available in-                 average 7% peaks overlap coding exons ex- <br /> dividuals previously reported cases SHFM1 (Fig. 5). We                cluding exon. With ;1.6% human mouse <br /> mapped family (GK) SHFM1 46,XY,t(7;20)(q22;p13)              genomes encoding protein, eExons overrepresented <br /> translocation (Fig. 5). In addition, mapped inversion break-            ChIP-seq enhancer-associated data sets. To test whether <br /> points previously published SHFM1 family (Tackels-Horne et al.            exons enriched ChIP-seq enhancer data sets, generated <br /> 2001) chr 7: 96,219,611 109,486,136 (K6200                    random data set mappable sequences used <br /> family) (Everman et al. 2005; Everman et al. 2006). In addition,            whole-genome sequencing alignment UCSC <br /> referred recently reported SHFM1 cases: individual              Genome Browser (http://<span id='am-53' about='obo:NCBITaxon_997520' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_95175 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-55'  ></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Moma" datatype="xsd:string"></span>moma</span>.ki.au.dk/genome-mirror/cgi-bin/ <br /> SHFM1 novo pericentric inversion chromosome 7:                 hgTrackUi?hgsid=148&amp;c=chrX&amp;g=wgEncodeMapability) contain- <br /> 46, XY, inv(7) (p22q21.3), breakpoint mapped chr 7:                ing identical number peaks EP300 ChIP-seq data sets <br /> 95.53&#8211;95.72 Mb (van Silfhout et al. 2009), individual              GM12787 K562 (51,260 17,883 peaks, respectively) and <br /> with split foot phenotype 880-kb microdeletion                 tested peaks overlap exons compared ChIP-seq <br /> 95.39&#8211;96.27Mb (Fig. 5; Kouwenhoven et al. 2010). It worth not-              data sets. We significantly higher percentage peaks <br /> ing AER enhancer named BS1 recently identified 300                 overlapping coding exons (after excluding exon) in <br /> kb centromeric DLX5/6 (Kouwenhoven et al. 2010). However,                EP300 ChIP-seq data sets GM12787 (P &lt; 0.014; Fisher <br /> least individuals SHFM1 described                  exact test) K562 (P &lt; 8.44 3 10 8; Fisher exact test) cell lines. In <br /> chromosomal aberrations include BS1 (Fig. 5), suggest-             addition, using random sampling approach, randomly sampled <br /> ing additional limb enhancers, DYNC1I1 eExon 15               1000 peaks (from mappable sequences) having equal distri- <br /> 17, lead SHFM1. All chromosomal abnormalities                  bution EP300 ChIP-seq data sets 1000 times and <br /> described overlap DYNC1I1 eExon 15 17 suggest                    significantly higher fraction peaks overlapped coding <br /> that removal disrupt transcriptional regulation             exons ChIP-seq data sets versus random samples (P &lt; 2.2 3 <br /> DLX5/6 causes human limb malformations.             10 16). Combined, assays suggest overrepresentation of <br />  <br />  <br />                                                                                                                       Genome Research             1063 <br />                                                                                                                          www.genome.org <br />  Birnbaum et al. <br />  <br />  <br />                                                                                                           ;57% exons overlapping enhancer- <br />                                                                                                           associated ChIP-seq peaks bona fide <br />                                                                                                           enhancers functional assays <br />                                                                                                           needed order determine this. <br />                                                                                                                 It worth noting Dlx5/6 and <br />                                                                                                           Twist1 expression detected whole- <br />                                                                                                           mount situ hybridization restricted <br />                                                                                                           AER A-P domains the <br />                                                                                                           developing mouse limb, respectively. <br />                                                                                                           However, mouse limb enhancer ex- <br />                                                                                                           pression pattern DYNC1I1 eExon 15 <br />                                                                                                           extends limb bud mesenchyme <br />                                                                                                           HDAC9 eExon 18 extends into <br />                                                                                                           posterior limb bud mesenchyme. <br />                                                                                                           These expression pattern discrepancies <br />                                                                                                           inability whole- <br />                                                                                                           mount situ assay detect low RNA <br />                                                                                                           expression levels versus robust <br />                                                                                                           staining seen LacZ enhancer as- <br />                                                                                                           says. One example ability to <br />                                                                                                           study role expression myo- <br />                                                                                                           cyte enhancer factor 2C (MEF2C) gene in <br />                                                                                                           neural crest enhancer func- <br />                                                                                                           tion, previously confounded <br />                                                                                                           observed in <br />                                                                                                           situ hybridization (Agarwal et al. 2011). <br />                                                                                                           Alternatively, limb enhancers could <br />                                                                                                           potentially regulating limb-as- <br />                                                                                                           sociated genes. However, results for <br />                                                                                                           DYNC1I1 eExon 15 demonstrate physi- <br />                                                                                                           cal interaction eExon and <br />                                                                                                           Dlx5/6 promoter regions, suggesting <br />                                                                                                           regulates Dlx5/6 expression the <br /> Figure 4. 3C DNA-FISH physical interaction Dync1i1 eExon 15 Dlx5/6                 limb. Another possibility discrep- <br /> promoter regions mouse E11.5 limb. (A) Schematic Dync1i1-Dlx5/6 locus, showing          ancy expression patterns as- <br /> relative location primers used 3C BAC probes used DNA-FISH. (B) Chromatin          sociated &#8216;&#8216;artificial&#8217;&#8217; nature the <br /> looping events detected using 3C Dync1i1 eExon 15 (orange oval) promoters          transgenic enhancer assay, could <br /> Dync1i1-Dlx5/6 locus. The closest HindIII restriction sites (RS1 RS2) promoter used to <br /> analyze interaction frequencies Dync1i1 eExon 15 (anchoring point). In limb, interaction <br />                                                                                                           lead different results use of <br /> frequencies Dync1i1eExon 15 Dlx6 Dlx5 promoter regions significantly higher          minimal promoter instead pro- <br /> compared heart negative control (more 10- 15-fold, respectively). No significant in-      moter regulated gene, site of <br /> teraction differences Dync1i1eExon 15 Dync1i1 promoter, closest            transgene integration, variation in <br /> tested site anchoring point, control regions (;900 kb away Dync1i1-Dlx5/6 <br />                                                                                                           transgene copy number trans- <br /> locus) limb versus heart tissues. (Error bars) SE average independent PCR reactions. <br /> (C&#8211;L) DNA-FISH results BAC probe RP23-430G21, covers Dync1i1eExon 15 region (red),         genic animals, and/or complications. <br /> and BAC probe RP23-77O3, covers Dlx5/6 gene regions (green). (C ) E11.5 limb section             Two characterized limb en- <br /> the dotted line highlighting AER, depicted p63 staining nucleus. (D) BAC probes      hancers, DYNC1I1 eExon 15 17, reside <br /> DAPI staining E11.5 limbs. (Squares) Magnified regions E F highlight colocalization    coding exons DYNC1I1, subunit <br /> of Dync1i1eExon 15 Dlx5/6 signals. (G) E11.5 heart section shows p63 staining cytoplasm. <br /> (H) BAC probes DAPI staining E11.5 heart. (Squares) Magnified regions I J         cytoplasmic Dynein 1 motor pro- <br /> a separation Dync1i1eExon 15 Dlx5/6 signals. The white scale bars represent 5 mm length. (K,L)     tein complex expressed the <br /> Calculated frequencies 0.2 mm distance interval mouse E11.5 AER (K) heart (L) tis-       limb development (Supplemen- <br /> sues. (Black columns) Fraction colocalized signals (0&#8211;0.2 mm). The number (n) loci observed      tal Fig. 3A,F; Crackower et al. 1999). Both <br /> this experiment indicates significant difference frequencies colocalized signals in <br /> the AER heart tissues (**P &lt; 0.01; Student&#8217;s t-test).                                                 DYNC1I1 eExon 15 17 present <br />                                                                                                           DYNC1I1 splice isoforms <br />                                                                                                           (RefSeq: NM_004411.4, NM_001135557.1, <br /> coding exons ChIP-seq data sets. However, worth noting             NM_001135556.1), suggesting alternative splicing does not <br /> the technical variability ChIP-seq assay differences            occur exons. DYNC1I1 eExon15 marked en- <br /> antibodies, cross-linking, pull down, sequencing depth,               hancer chromatin signature physically interacts Dlx5/ <br /> along sequence mappability taken account                    6 promoter regions specifically limb. Both eExons encode <br /> these analyses.                                                                  protein domains important cargo binding specificity of <br />       Using mouse transgenic enhancer assay seven mouse                    Dynein (Kardon Vale 2009). Cytoplasmic Dynein 1 in- <br /> E11.5 limb EP300 ChIP-seq peaks, eExons                    volved neuronal migration brain development inter- <br /> functional limb enhancers regulate neighboring                   acting Dynactin platelet-activating factor acetylhydrolase <br /> genes. The observed 57% (4/7) success rate does imply                   1b (LIS1). Defects neuronal migration lead brain malfor- <br />  <br />  <br /> 1064      Genome Research <br />              www.genome.org <br />                                                                                                              Exonic enhancers nearby genes <br />  <br />  <br />                                                                                                             expression nearby gene/s another <br />                                                                                                             tissue novel. It raises possibility <br />                                                                                                             mutations certain gene, even <br />                                                                                                             synonymous ones, potentially af- <br />                                                                                                             fect regulation nearby gene. <br />                                                                                                             Therefore, careful analysis tissue- <br />                                                                                                             specific expression function gene <br />                                                                                                             required order determine <br />                                                                                                             phenotype truly caused by <br />                                                                                                             mutation coding sequence. <br />  <br />  <br />                                                                                                             Methods <br /> Figure 5. Chromosomal abnormalities chromosome 7q21-23 associated SHFM1. A sche-                    Computational ChIP-seq data set <br /> matic representation genomic positions breakpoints chromosomal rearrangements in <br /> individuals SHFM1 mapped human genome assembly 18 (hg18) compared location <br />                                                                                                             analyses <br /> of DYNC1I1 eExon 15 17. An 880-kb microdeletion individual split foot pheno-              We identified exonic sequences the <br /> type 95.39&#8211;96.27 Mb (Kouwenhoven et al. 2010). In GK family,                    human hg18 mouse mm9 genome <br /> 46,XY,t(7;20)(q22;p13) translocation breakpoints mapped chr 7: 96.2&#8211;96.47 Mb. In K6200 family,      assemblies using UCSC knownGene <br /> the chromosomal inversion breakpoints mapped chr 7: 96,219,611 109,486,136. The breakpoint <br /> coordinates 7:46, XY, inv(7) (p22q21.3) SHFM1 pervasive developmental disorder-not           track (http://genome.ucsc.edu). We down- <br /> otherwise specified (PDD-NOS) chr 7: 95.53&#8211;95.72 Mb (van Silfhout et al. 2009). All  loaded exonic sequences, including <br /> these chromosomal abnormalities overlap DYNC1I1 eExon 15 17 (orange ovals). Two          59 UTR 39 UTR, using txStrat and <br /> chromosomal aberrations overlap BS1 AER enhancer (white oval). (Lightning bolts)           txEnd filter field. All exon sequence sizes <br /> Translocation inversion breakpoints; (diamonds) deletion.                                              divided number exons to <br />                                                                                                            calculate average exon size. We down- <br />                                                                                                            loaded coding exon sequences using the <br /> mations lissencephaly, subcortical laminar heterotopias,            cdsStart cdsEnd filter field. The 22 ChIP-seq data sets human <br /> pervasive developmental disorder-not specified (PDD-                  cell lines obtained Ernst et al. (2011), Myers et al. (2011), <br /> NOS) (Kato Dobyns 2003). Interestingly, individual                  Rosenbloom et al. (2012) downloaded UCSC <br /> PDD-NOS SHFM1 inversion DYNC1I1 region (chr 7:                Genome Browser, EP300 ChIP-seq data sets mouse <br /> 95.53&#8211;95.72 Mb) (Fig. 5), breakpoint finely                  E11.5 tissues obtained Visel et al. (2009a) down- <br /> characterized (van Silfhout et al. 2009). Further analysis             loaded Gene Expression Omnibus (http://www.ncbi.nlm. <br /> required order establish PDD-NOS SHFM                     nih.gov/geo) (Supplemental Table 1 includes links down- <br /> phenotypes individual disruption            loaded data). In order unify results, human sequences with <br /> the DYNC1I1 gene characterized eExons.                                  hg19 coordinates converted hg18 using UCSC Genome <br />                                                                                 Browser LiftOver tool. A ChIP-seq peak considered overlap <br />        Two characterized limb enhancers, HDAC9 eExon 18 <br />                                                                                 exon 1 bp exonic sequence overlapped. BED files of <br /> and 19, reside coding exons HDAC9, member the <br />                                                                                 ChIP-seq peaks overlap exons various data sets <br /> histone acetyltransferase <span id='am-20' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> II family. HDAC9 eExon 19 also <br />                                                                                 obtained http://bts.ucsf.edu/ahituv/resources.html. First <br /> been shown exonic remnant zebrafish speculated                  exons splice isoforms gene determined the <br /> to <span id='am-39' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-40' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cis</span>-regulatory function (Dong et al. 2010). Both HDAC9                exonStarts exonEnds field UCSC knownGene track. To <br /> eExons 18 eExon 19 appear 3/9 HDAC9 spice isoforms                       identify limb expressed genes, used available mouse RNA situ <br /> (RefSeq: NM_178423.1, NM_058176.2, NM_178425.2). HDAC9                          data Mouse Genome Informatics (MGI) gene expression <br /> expression shown selective compared                 data query form (http://www.informatics.jax.org/javawi2/servlet/ <br /> other HDAC family members (de Ruijter et al. 2003). Our RNA                     WIFetch?page=expressionQF) defined limb expressing gene as <br /> analysis whole-mount situ hybridization results                having RNA situ expression data TS19 (E11.0&#8211;12.25) <br /> Hdac9 expressed mouse limb E11.5 (Supplemental                 TS20 (E11.5&#8211;13.0). <br /> Fig. 3D,G). Hdac9-null mice generated deletion exons 4 5 <br /> are fertile survive normal life span develop cardiac hy- <br />                                                                                 Transgenic enhancer assays <br /> pertrophy age response pressure overload (Zhang <br /> et al. 2002). Interestingly, despite Hdac9 expressed           By use primers designed amplify EP300 ChIP-seq peaks <br /> limb E11.5, Hdac9 homozygous knockout mice develop poly-                     overlap exons (Supplemental Table 5), carried poly- <br />                                                                                 merase chain reaction (PCR) human genomic DNA (Qiagen). <br /> dactyly hindlimbs partial penetrance (Morrison and <br />                                                                                 Primers designed 500 bp additional sequence <br /> D&#8217;Mello 2008), similar polydactyly phenotype Twist1 <br />                                                                                 flanking EP300 peak. Previous experiments shown be <br /> heterozygous knockout mice (Bourgeois et al. 1998). Although <br />                                                                                 reliable method obtaining positive enhancer activity using <br /> Hdac9 eExons 18 19 removed Hdac9-null mice,                evolutionary conserved regions (Pennacchio et al. 2006) EP300 <br /> the regulation Twist1 potential Twist1 en-                ChIP-seq peaks (Visel et al. 2009a). For mouse enhancer assays, <br /> hancers disrupted leading polydactyly phenotype.                PCR products human genomic regions cloned vector <br />        The ability eExons enhance expression nearby          containing Hsp68 minimal promoter followed LacZ re- <br /> genes, gene reside in, suggest mechanisms           porter gene (Pennacchio et al. 2006) sequence verified. Transgenic <br /> such involved epigenetic regulation high-order chro-            mice generated UCSF transgenic facility Cyagen <br /> matin organization control function tissue. To              Biosciences using standard procedures (Nagy et al. 2002). Embryos <br /> our knowledge, functional demonstration DNA sequences                  harvested E11.5 stained LacZ expression previously <br /> can act protein coding sequence tissue regulate             described (Pennacchio et al. 2006). For zebrafish enhancer assays, <br />  <br />  <br />                                                                                                                       Genome Research           1065 <br />                                                                                                                          www.genome.org <br />  Birnbaum et al. <br />  <br />  <br /> the human PCR products cloned E1b-GFP-Tol2          CHORI BACPAC resource center covering SHFM1 minimal <br /> enhancer assay vector containing E1b minimal promoter followed      region (RP23-430G21, RP24-73K21, RP23-336P10, RP23-389M11, <br /> by GFP (Li et al. 2009). They injected following standard         RP24-343G1, RP24-270A16). <br /> procedures (Nusslein-Volhard Dahm 2002; Westerfield 2007) <br /> into 100 embryos construct Tol2 mRNA           DNA florescent situ hybridization <br /> (Kawakami 2005) facilitate genomic integration. GFP expression <br /> was observed annotated 48 72 hpf. An enhancer           DNA florescent situ hybridization (FISH) carried pre- <br /> considered positive 60% GFP expressing fish showed           viously described (Lomvardas et al. 2006). BAC clones RP23-77O3 <br /> a consistent expression pattern. All animal work approved       Dlx5/6 RP23-430G21 Dync1i1 obtained the <br /> the UCSF Institutional Animal Care Use Committee.                  CHORI BACPAC resource center. Probes labeled Digoxi- <br />                                                                        genin-11-dUTP Biotin-16-dUTP Nick Translation (Roche). <br />                                                                        Limb heart tissues (E11.5) embedded fixation, and <br /> Whole-mount situ hybridization                                      10 mM cryosections collected Superfrost Plus slides (Fisher). <br /> Mouse E11.5 embryos fixed 4% paraformaldehyde. Clones          After drying, sections fixed 4% PFA 5 min 4&#176;C. DNA <br />                                                                        fragmented incubation 0.1 M HCl 5 min room <br /> containing mouse Dync1i1 (MMM1013-9202215, Open Biosystems), <br /> Dlx5 (Depew et al. 1999), Dlx6 (OMM5895-99863403 Open Bio-             temperature, slides treated RNase A 1 h 37&#176;C. <br /> systems), Hdac9 (EMM1032-601163 EMM1002-6974502, Open              Slides dried ethanol series, denatured solution of <br /> Biosystems), Twist1 (Chen Behringer 1995) used         75% formamide 2xSSC 5 min 85&#176;C, rinsed immediately in <br />                                                                        ice-cold 2xSSC, dried 4&#176;C ethanol series. Pre-dena- <br /> templates digoxygenin-labeled probes. Mouse whole-mount in <br /> situ hybridizations performed according standard procedures    tured, CotI-annealed probes applied overnight. The probe was <br /> (Hargrave et al. 2006).                                                washed times 15 min 55% formamide, 0.1% NP-40 in <br />                                                                        2xSSC 42&#176;C. Probes detected using Dylight 488 anti-digoxi- <br />                                                                        genin Dylight 549 anti-biotin (Jackson Immunoresearch). <br /> RNA expression analysis                                                Antibody washes carried solution PBS containing <br />                                                                        0.1% Triton-x-100 8% formamide room temperature. All <br /> Mouse E11.5 limb AER enriched tissues (limb buds the <br />                                                                        images obtained using confocal fluorescence microscopy <br /> AER region carefully dissected), adult mouse heart and <br />                                                                        (Nikon C1 Spectral). FISH signals recorded separate <br /> cortex tissues dissected. Total RNA isolated using RNeasy <br />                                                                        RGB channels. The image stacks reconstructed using the <br /> (Qiagen) according manufacturer&#8217;s protocol. qPCR was <br />                                                                        Volocity program (PerkinElmer), shortest distance between <br /> performed using SsoFast EvaGreen Supermix (Biorad) run on <br />                                                                        gravity centers Dlx5/6 Dync1i1 signals calculated. <br /> the Eppendorf Mastercycler ep realplex 2 thermal cycler. Samples <br /> were tested duplicates. Specificity absence primer dimers <br /> was controlled denaturation curves. b-Actin (Actb) mRNA         Subjects chromosomal breakpoint mapping <br /> used normalization. Primer sequences used amplification        The GK family consisted male ectrodactyly, <br /> are listed Supplemental Table 5.                                    micrognathia, elongated neck, bilateral <span id='am-1' about='obo:NCBITaxon_311122' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_171578 obo:NCBITaxon_40040 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_33208 obo:NCBITaxon_311087 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-2' property="rdfs:label" content="Microtia" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5'  ></span>microtia</span> with <br />                                                                        neurosensory deafness female offspring died before <br />                                                                        birth ectrodactyly, micrognathia, bilateral microtia. <br /> ChIP followed qPCR <br />                                                                        Karyotypes father offspring demonstrated reciprocal <br /> ChIP following standard techniques (Nelson et al. 2006) per-       balanced chromosomal translocation 46,XY,t(7;20)(q22;p13) that <br /> formed mouse E11.5 AER-enriched tissue. For ChIP, 100&#8211;         GK&#8217;s healthy mother. By use FISH, following <br /> 500 mg chromatin used. For immunoprecipitation, used         standard techniques (Trask 1991), BACs (RP11-94N7, <br /> 2 mg H3K4me1 (ab8895, Abcam), H3K4me3 (ab8580, Abcam),              RP11-78B12), breakpoint coordinates chromosome 7 were <br /> H3K27ac (ab4729, Abcam), H3K36me3 (ab9050; Abcam) an-              mapped 96.2 96.47 Mb. The K6200 family had <br /> tibodies. qPCR carried using SsoFast EvaGreen Supermix         autosomal dominant SHFM variable sensorineural hearing loss <br /> (Biorad) run Eppendorf Mastercycler ep realplex 2           previously reported (Tackels-Horne et al. 2001). Subsequent <br /> thermal cycler. ChIP-qPCR signals standardized input           studies family pulse field gel electrophoresis FISH <br /> chromatin (percentage input). Primer sequences used am-         identified chromosome inversion breakpoints SHFM1 <br /> plification listed Supplemental Table 5.                        critical region (Everman et al. 2005; Everman et al. 2006). Southern <br />                                                                        blot analysis inverse PCR previously described (Vervoort et al. <br />                                                                        2002) used identify inversion breakpoints (D.B. <br /> 3C assay <br />                                                                        Everman, C.T. Morgan, M.E. Laughridge, T. Moss, S. Ladd, B. <br /> 3C performed following standard procedures (Dostie             DuPont, D. Toms, A. Dobson, K.D. Clarkson, F. Gurrieri, et al., <br /> Dekker 2007). Mouse E11.5 heart AER enriched tissues          unpubl.). The inversion family balanced, minimal <br /> dissected 30 embryos, cross-linked 1% formaldehyde,          changes normal sequence breakpoint segregated <br /> and processed single cell preparations. Cells lysed     SHFM/hearing loss phenotype. <br /> purify nuclei digested HindIII (1200 units) restriction <br /> enzyme (New England Biolabs). Cross-linked fragments li- <br /> gated 2000 units T4 DNA ligase (New England Biolabs) 3     Acknowledgments <br /> d 4&#176;C. The samples reverse cross-linked, purified DNA      We thank members Ahituv laboratory helpful comments <br /> was amplified whole-genome amplification (WGA2, Sigma-              manuscript. We thank Juhee Jeong John L.R. <br /> Aldrich). Product detection triplicate qPCR, de-     Rubenstein reagents. This research supported NICHD <br /> scribed ChIP, averaged primer pair (Sup-        grant no. R01HD059862. N.A. G.B. supported by <br /> plemental Table 5). Each data point corrected PCR <span id='am-12' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-13'  ></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Bias" datatype="xsd:string"></span>bias</span>   NHGRI grant number R01HG005058, N.A. supported <br /> by dividing average PCR signals average signal     NIGMS award number GM61390. M.J.K. supported part <br /> of internal control template. Data AER heart          NIH Training Grant T32 GM007175 Amgen Research <br /> normalized BAC library containing seven BACs obtained        Excellence Bioengineering Therapeutic Sciences Fellow- <br />  <br />  <br /> 1066       Genome Research <br />             www.genome.org <br />                                                                                                              Exonic enhancers nearby genes <br />  <br />  <br /> ship. O.A. O.S.B. supported Morris Kahn family                Iovine MK. 2007. Conserved mechanisms regulate outgrowth zebrafish <br /> foundation. D.B.E C.E.S. supported grant                    fins. Nat Chem Biol 3: 613&#8211;618. <br />                                                                               Itzkovitz S, Alon U. 2007. The genetic code nearly optimal allowing <br /> from South Carolina Department Disabilities Special                    additional information protein-coding sequences. Genome Res <br /> Needs, Genetic Endowment South Carolina, previous                    17: 405&#8211;412. <br /> grant (no. 8510) Shriners Hospitals Children. The content            Kardon JR, Vale RD. 2009. Regulators cytoplasmic dynein motor. Nat <br /> is solely responsibility authors does necessarily              Rev Mol Cell Biol 10: 854&#8211;865. <br />                                                                               Kato M, Dobyns WB. 2003. Lissencephaly molecular basis of <br /> represent official views NIH, NICHD, NHGRI, NIDCR,                  neuronal migration. Hum Mol Genet 12: R89&#8211;R96. <br /> the NIGMS.                                                                    Kawakami K. 2005. Transposon tools methods zebrafish. Dev Dyn <br />                                                                                   234: 244&#8211;254. <br />                                                                               Kothary R, Clapoff S, Brown A, Campbell R, Peterson A, Rossant J. 1988. A <br />                                                                                   transgene containing lacZ inserted dystonia locus expressed <br /> References                                                                        neural tube. Nature 335: 435&#8211;437. <br />                                                                               Kouwenhoven EN, van Heeringen SJ, Tena JJ, Oti M, Dutilh BE, Alonso ME, <br /> Agarwal P, Verzi MP, Nguyen T, Hu J, Ehlers ML, McCulley DJ, Xu SM, Dodou         la Calle-Mustienes E, Smeenk L, Rinne T, Parsaulian L, et al. 2010. <br />     E, Anderson JP, Wei ML, et al. 2011. The MADS box transcription factor        Genome-wide profiling p63 DNA-binding sites identifies element <br />     MEF2C regulates melanocyte development direct                        regulates gene expression limb development 7q21 <br />     transcriptional target partner SOX10. Development 138: 2555&#8211;           SHFM1 locus. PLoS Genet 6: e1001065. doi: 10.1371/journal. <br />     2565.                                                                         pgen.1001065. <br /> Ahituv N, Prabhakar S, Poulin F, Rubin EM, Couronne O. 2005. Mapping cis-     Lampe X, Samad OA, Guiguen A, Matis C, Remacle S, Picard JJ, Rijli FM, <br />     regulatory domains human genome using multi-species                    Rezsohazy R. 2008. An ultraconserved Hox-Pbx responsive element <br />     conservation synteny. Hum Mol Genet 14: 3057&#8211;3063.                         resides coding sequence Hoxa2 active rhombomere 4. <br /> Bourgeois P, Bolcato-Bellemin AL, Danse JM, Bloch-Zupan A, Yoshiba K,             Nucleic Acids Res 36: 3214&#8211;3225. <br />     Stoetzel C, Perrin-Schmitt F. 1998. The variable expressivity         Li Q, Ritter D, Yang N, Dong Z, Li H, Chuang JH, Guo S. 2009. A systematic <br />     incomplete penetrance twist-null heterozygous mouse phenotype          approach identify functional motifs vertebrate developmental <br />     resemble human Saethre-Chotzen syndrome. Hum Mol Genet 7:            enhancers. Dev Biol 337: 484&#8211;495. <br />     945&#8211;957.                                                                  Lin MF, Kheradpour P, Washietl S, Parker BJ, Pedersen JS, Kellis M. 2011. <br /> Chamary JV, Parmley JL, Hurst LD. 2006. Hearing silence: Non-neutral              Locating protein-coding sequences selection additional, <br />     evolution synonymous sites mammals. Nat Rev Genet 7: 98&#8211;108.            overlapping functions 29 mammalian genomes. Genome Res 21: <br /> Chen ZF, Behringer RR. 1995. twist required head mesenchyme             1916&#8211;1928. <br />     cranial neural tube morphogenesis. Genes Dev 9: 686&#8211;699.                  Lomvardas S, <span id='am-48' about='obo:NCBITaxon_457831' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6544 obo:NCBITaxon_52914 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_6599 obo:NCBITaxon_61346 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_278205 obo:NCBITaxon_6072'><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-50'  ></span><span id='am-51' property="rdfs:label" content="Barnea" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Barnea</span> G, Pisapia DJ, Mendelsohn M, Kirkland J, Axel R. 2006. <br /> Crackower MA, Sinasac DS, Xia J, Motoyama J, Prochazka M, Rommens JM,             Interchromosomal interactions olfactory receptor choice. Cell 126: <br />     Scherer SW, Tsui LC. 1999. Cloning characterization                403&#8211;413. <br />     cytoplasmic dynein intermediate chain genes mouse human.           Mercader N. 2007. Early steps paired fin development zebrafish <br />     Genomics 55: 257&#8211;267.                                                         compared tetrapod limb development. Dev Growth Differ 49: 421&#8211; <br /> Depew MJ, Liu JK, Long JE, Presley R, Meneses JJ, Pedersen RA, Rubenstein         437. <br />     JL. 1999. Dlx5 regulates regional development branchial arches     Morrison BE, D&#8217;Mello SR. 2008. Polydactyly mice lacking HDAC9/HDRP. <br />     sensory capsules. Development 126: 3831&#8211;3846.                             Exp Biol Med (Maywood) 233: 980&#8211;988. <br /> de Ruijter AJ, van Gennip AH, Caron HN, Kemp S, van Kuilenburg AB. 2003.      Myers RM, Stamatoyannopoulos J, Snyder M, Dunham I, Hardison RC, <br />     Histone deacetylases (HDACs): Characterization classical HDAC          Bernstein BE, Gingeras TR, Kent WJ, Birney E, Wold B, et al. 2011. A <br />     family. Biochem J 370: 737&#8211;749.                                               user&#8217;s guide encyclopedia DNA elements (ENCODE). PLoS Biol <br /> Dong X, Navratilova P, Fredman D, Drivenes O, Becker TS, Lenhard B. 2010.         9: e1001046. doi: 10.1371/journal.pbio.1001046. <br />     Exonic remnants whole-genome duplication reveal cis-regulatory         Nagy A, Gertsenstein M, Vintersten K, Behringer R. 2002. Manipulating the <br />     function coding exons. Nucleic Acids Res 38: 1071&#8211;1085.                    mouse embryo: A laboratory manual. Cold Spring Harbor Laboratory Press, <br /> Dostie J, Dekker J. 2007. Mapping networks physical interactions       Cold Spring Harbor, NY. <br />     genomic elements using 5C technology. Nat Protoc 2: 988&#8211;1002.             Nelson JD, Denisenko O, Bomsztyk K. 2006. Protocol fast chromatin <br /> Eichenlaub MP, Ettwiller L. 2011. De novo genesis enhancers                 immunoprecipitation (ChIP) method. Nat Protoc 1: 179&#8211;185. <br />     vertebrates. PLoS Biol 9: e1001188. doi: 10.1371/journal.pbio.1001188.    Neznanov N, Umezawa A, Oshima RG. 1997. A regulatory element within <br /> Elliott AM, Evans JA. 2006. Genotype-phenotype correlations mapped             coding exon modulates keratin 18 gene expression transgenic mice. <br />     split hand foot malformation (SHFM) patients. Am J Med Genet A 140:           J Biol Chem 272: 27549&#8211;27557. <br />     1419&#8211;1427.                                                                Nissim S, Tabin C. 2004. Development limbs. In Inborn errors of <br /> Ernst J, Kheradpour P, Mikkelsen TS, Shoresh N, Ward LD, Epstein CB,              development (ed. C.E.R. Erickson T. Wynshaw-Boris), pp. 148&#8211;167. <br />     Zhang X, Wang L, Issner R, Coyne M, et al. 2011. Mapping                  Oxford University Press, New York. <br />     analysis chromatin state dynamics human cell types. Nature     Nusslein-Volhard C, Dahm R. 2002. Zebrafish. Oxford University Press, <br />     473: 43&#8211;49.                                                                   Oxford. <br /> Everman D, Morgan C, Clarkson K, Gurrieri F, McAuliffe F, Chitayat D,         O&#8217;Rourke MP, Soo K, Behringer RR, Hui CC, Tam PP. 2002. Twist plays an <br />     Stevenson R, Schwartz C. 2005. Submicroscopic rearrangements                  essential role FGF SHH signal transduction mouse limb <br />     involving SHFM1 locus chromosome 7q21-22 associated           development. Dev Biol 248: 143&#8211;156. <br />     split-hand/foot malformation sensorineural hearing loss. Proc         Pennacchio LA, Ahituv N, Moses AM, Prabhakar S, Nobrega MA, Shoukry M, <br />     Greenwood Genet Cent 24: 137.                                                 Minovitsky S, Dubchak I, Holt A, Lewis KD, et al. 2006. In vivo enhancer <br /> Everman D, Morgan C, Stevenson R, Schwartz C. 2006. Chromosome                    analysis human conserved non-coding sequences. Nature 444: 499&#8211;502. <br />     rearrangements: An emerging theme causation split-hand/foot     Ritter DI, Dong Z, Guo S, Chuang JH. 2012. Transcriptional enhancers in <br />     malformation. Proc Greenwood Genet Cent 25: 138&#8211;139.                          protein-coding exons vertebrate developmental genes. PLoS One. doi: <br /> Firulli BA, Krawchuk D, Centonze VE, Vargesson N, Virshup DM, Conway              10.1371/journal.pone.0035202. <br />     SJ, Cserjesi P, Laufer E, Firulli AB. 2005. Altered Twist1 Hand2      Robledo RF, Rajan L, Li X, Lufkin T. 2002. The Dlx5 Dlx6 homeobox <br />     dimerization associated Saethre-Chotzen syndrome limb             genes essential craniofacial, axial, appendicular skeletal <br />     abnormalities. Nat Genet 37: 373&#8211;381.                                         development. Genes Dev 16: 1089&#8211;1101. <br /> Gilbert SF. 2000. Developmental Biology, 6th ed. Sinauer Associates,          Rosenbloom KR, Dreszer TR, Long JC, Malladi VS, Sloan CA, Raney BJ, Cline <br />     Sunderland, MA.                                                               MS, Karolchik D, Barb <br /> </body></html>