<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  Genome-Wide Transcriptional Dependence on Conserved Regions of Mot <br />  <br />  <br />              TATA binding protein (TBP) plays a central role in transcription complex assembly and is regulated by a <br />           variety of transcription factors, including Mot1. Mot1 is an essential protein in <span id='am-11' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> that <br />           exerts both negative and positive effects on transcription via interactions with TBP. It contains two conserved <br />           regions important for TBP interactions, another conserved region that hydrolyzes ATP to remove TBP from <br />           DNA, and a fourth conserved region with unknown function. Whether these regions contribute equally to <br />           transcriptional regulation genome-wide is unknown. Here, we employ a transient-replacement assay using <br />           deletion derivatives in the conserved regions of Mot1 to investigate their contributions to gene regulation <br />           throughout the S. cerevisiae genome. These four regions of Mot1 are essential for growth and are generally <br />           required for all Mot1-regulated genes. Loss of the ATPase region, but not other conserved regions, caused TBP <br />           to redistribute away from a subset of Mot1-inhibited genes, leading to decreased expression of those genes. A <br />           corresponding increase in TBP occupancy and expression occurred at another set of genes that are normally <br />           Mot1 independent. The <span id='am-73' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-74' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77'  ></span>data</span> suggest that Mot1 uses ATP hydrolysis to redistribute accessible TBP away from <br />           intrinsically preferred sites to other sites of intrinsically low preference. <br />  <br />  <br />    From <span id='am-28' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-30'  ></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae to human, the TATA binding                       ulated removal of TBP, which Mot1 is well suited to do. In <span id='am-40' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> <br /> protein (TBP) provides an indispensable role in nearly all                        context, Mot1 is a negative regulator. <br /> RNA polymerase I, II, and III transcription events (29). TBP                         Since TBP binds to the minor groove of DNA, which has <br /> is the central component of a complex regulatory network                          limited sequence specificity, TBP has relative high affinity for <br /> governing transcription complex assembly (33). Consequently,                      nonspecific DNA (14). If bound inappropriately, this might <br /> TBP is subjected to an extraordinary level of regulation by                       lead to aberrant or nonproductive assembly of the transcrip- <br /> numerous transcription factors, one of which is Mot1 (6). Mot1                    tion machinery. Biochemical experiments have demonstrated <br /> is a conserved Snf2/Swi2-related ATPase (21) that regulates                       that Mot1 can remove nonspecifically bound TBP (41), per- <br /> the dynamics of TBP-promoter interactions by removing TBP                         haps acting as a chaperone allowing TBP to rebind in a pro- <br /> from DNA using the energy of ATP hydrolysis (6, 45, 47). The                      ductive mode. For example, at the URA1 gene, Mot1 can <br /> first 800 amino-terminal residues of Mot1 are both necessary                      promote transcription by removing a nonproductive TBP <br /> and sufficient for TBP binding (2, 7). The Snf2-related ATPase                    bound in the reverse orientation (46). In this context, Mot1 <br /> domain resides within the last 600 carboxy-terminal residues                      operates as a positive regulator. <br /> (7). Genome-wide expression studies using temperature-sensi-                         The mechanism by which Mot1 acts on TBP is well defined <br /> tive mot1 alleles indicate that Mot1 regulates between 3 and                      biochemically, and this provides a basis for interpreting less <br /> 15% of the yeast genome, some negatively and others posi-                         defined in vivo experiments. Because Mot1 is essential for <br /> tively (4, 17, 24).                                                               growth in S. cerevisiae (19), in vivo functional analysis of im- <br />    In vitro biochemical experiments have largely defined mech-                    portant regions of Mot1 is not straightforward in that loss of <br /> anisms by which Mot1 regulates TBP-DNA interactions. Mot1                         function is lethal. Thus, an investigation into the genome-wide <br /> can bind and stabilize TBP-DNA interactions, but in the pres-                     functions of essential proteins like Mot1 is hampered by the <br /> ence of ATP, it dissociates TBP from DNA and, to some                             practical limitation that mutations that eliminate function <br /> extent, Mot1 from TBP, thereby recycling both (2, 7, 12, 26).                     cause cell death. Temperature-sensitive mutations might <span id='am-62' about='obo:NCBITaxon_28690' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_28683 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_8906 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-63' property="rdfs:label" content="Alle" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65'  ></span><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>alle</span>- <br /> This reaction is important for two reasons. First, dynamic as-                    viate this problem to some extent, but they are difficult to <br /> sembly and disassembly of the transcription machinery impart                      target to specific regions of the protein and often vary in <br /> precise control over gene expression. Therefore, regulated re-                    severity. To circumvent this limitation, we utilized a transient- <br /> cruitment of TBP to promoters must be accompanied by reg-                         replacement strategy (34) to investigate the contributions of <br />                                                                                   conserved Mot1 domains to transcription and TBP recruitment <br />                                                                                   genome-wide. Our study revealed that Mot1-regulated tran- <br />    * Corresponding author. Mailing address: Department of Biochem-                scription is dependent on nearly all conserved regions of Mot1. <br /> istry and Molecular Biology, Penn State University, 456 N. Frear Lab- <br /> oratory, University Park, PA 16802. Phone: (814) 863-8252. Fax: (814)             Strikingly, transcriptional dependence for a subset of genes is <br /> 863-8595. E-mail: bfp2@psu.edu.                                                   specifically altered when the Mot1 ATPase domain is deleted. <br />    &#8224; Present address: Gene Regulation and Chromosome Biology Lab-                 Genome-wide location analysis of TBP in a strain that lacks the <br /> oratory, National <span id='am-35' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-36' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-38'  ></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Cancer</span> Institute, National Institutes of Health, Fred-          Mot1 ATPase region corroborates the expression-profiling ex- <br /> erick, MD 21702-1201. <br />    &#8225; Present address: Schering-Plough Research Institute, Union, NJ <br />                                                                                   periments, suggesting a direct effect on TBP. Furthermore, <br /> 07083.                                                                            coimmunoprecipitation of TBP and the Mot1 ATPase deletion <br />    &#17188; <br />      Published ahead of print on 28 March 2011.                                   mutant demonstrates that the two directly interact. These find- <br />  <br />                                                                            2253 <br />  2254         VENTERS ET AL.                                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />                                                        TABLE 1. MOT1 yeast strains used in this study <br />  Strain              Promoter            MOT1 allele              Deletion (<span id='am-67' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72'  ></span>aa</span>)a                      Plasmid                           MAT                Reference <br />  <br /> BY4743                   MOT1           WT                                                                                            Diploid             9 <br /> yLAC1                    GAL1           WT                                                pSH47                                       Diploid             34 <br /> yjdi408                  GAL1           FHT-WT1                                           pMOT221      (mot1-42)   LEU2               Alpha               This   study <br /> yjdi410                  GAL1           FHT-TBP1                   05&#8211;80                  pMOT221      (mot1-42)   LEU2               Alpha               This   study <br /> yjdi412                  GAL1           FHT-TBP2                  307&#8211;432                 pMOT221      (mot1-42)   LEU2               Alpha               This   study <br /> yjdi414                  GAL1           FHT-UK1                  1090&#8211;1259                pMOT221      (mot1-42)   LEU2               Alpha               This   study <br /> yjdi416                  GAL1           FHT-ATP1                 1403&#8211;1867                pMOT221      (mot1-42)   LEU2               Alpha               This   study <br /> yjdi418                  GAL1           Null                    MOT1::kanMX               pMOT221      (mot1-42)   LEU2               Alpha               This   study <br /> yjdi420                  GAL1           WT                                                pMOT221      (mot1-42)   LEU2               Alpha               This   study <br />   a <br />       aa, amino acids. <br />  <br />  <br />  <br />  <br /> ings reveal that Mot1-regulated genes are generally regulated                       deoxynucleoside triphosphates (dNTPs), and 0.2 mM each primer were used per <br /> by all parts of Mot1 and that a TBP-binding portion of Mot1                         50-ml reaction for 32 cycles. <br />                                                                                        Cell growth assays. MATa haploid FHT-Mot1 mutant strains carrying <br /> can alter the selectivity of TBP for promoters.                                     pMOT221 (mot1-42 LEU2) were grown at 25&#176;C in YPR (yeast-peptone-3% <br />                                                                                     raffinose) to mid-log phase. Cells (A600 &#11005; 0.5) were removed, and 2.5 &#9262;l of <br />                           MATERIALS AND METHODS                                     10-fold serial dilutions was spotted on three sets of YPD (2% dextrose) and <br />                                                                                     YPG (2% galactose) plates and incubated at 25&#176;C, 30&#176;C, or 37&#176;C. Photo- <br />    <span id='am-83' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Plasmids</span>. pCALF-T(PGK) (36) was converted to pCALF-FHT-T(PGK) 2.2                graphs were taken after 48 h. <br /> by inserting a 66-bp HIS-TEV oligonucleotide into the NdeI site. pUG6-FHT-P            Microarray analysis. Microarrays were performed essentially as described <br /> (4,170 bp) was made by PCR amplifying 259 bp containing the FHT (Flu-His-           previously (13, 32). Briefly, cultures were grown to an A600 of &#11011;0.6, induced with <br /> TEV) sequence from the pCALFHT-T(PGK) 2.2 plasmid. The PCR product was              2% galactose in YPR medium for 60 min at 25&#176;C, and shifted to 37&#176;C for 45 min <br /> digested with SalI, and 161 bp was ligated into the SalI-digested pUG6 plasmid      to inactivate the temperature-sensitive copy of Mot1 encoded by the mot1-42 <br /> (4,009 bp) so that the orientation was FHT-loxP-kanMX-loxP.                         allele. Cells were harvested by centrifugation at room temperature, washed in <br />    FHT-Mot1 mutant strains. A list of the yeast strains used in this study is       RNase-free DEPC (diethyl pyrocarbonate)-treated double-distilled H2O, and <br /> provided in Table 1. S. cerevisiae strain BY4743 (9) (Invitrogen) was used as the   frozen in liquid nitrogen. <br /> parental strain. Initially, the strain was transformed with pSH47 (URA3) (25)          Total RNA was isolated as described previously (32), and poly(A) tail mRNA <br /> encoding galactose-inducible Cre recombinase. Oligonucleotides (70-mer) were        was purified using oligo(dT) cellulose (Ambion) according to the manufacturer&#8217;s <br /> used to PCR amplify 1,991 bp of pFA6a-His3MX6-PGAL1 containing the HIS3 <br />                                                                                     instructions. Reverse transcription, labeling with fluorescent dyes (Cy3 and Cy5; <br /> gene and GAL1 promoter (38). The PCR product was transformed into BY4743 <br />                                                                                     Amersham), hybridization, and scanning were all performed as described previ- <br /> using a high-efficiency lithium acetate method (25) to replace 550 bp of the <br />                                                                                     ously (13, 32). Four micrograms of mRNA was used for hybridizations instead of <br /> endogenous MOT1 promoter with the GAL1 promoter, creating strain yLAC1. <br />                                                                                     the conventional 2 &#9262;g. Slides were treated with Dye Saver2 (Genisphere) ac- <br /> HIS&#11001; homologous-recombination transformants were selected on complete syn- <br />                                                                                     cording to the manufacturer&#8217;s instructions to preserve signal intensity. Data sets <br /> thetic medium lacking histidine and uracil (CSM-HIS-URA medium) and veri- <br />                                                                                     were mode normalized by using R software (Bioconductor) to mode-center <br /> fied by colony PCR. <br />                                                                                     replicates (dye swaps) (data are available at http://atlas.bx.psu.edu/). <br />    Regions of MOT1 were deleted by replacing coding sequences with an FHT <br />                                                                                        Genes were filtered by several criteria to minimize false positives. (i) Genes <br /> tag. The FHT tag encodes three hemagglutinin (HA) (Flu) repeats, a decahis- <br />                                                                                     were eliminated if their signals on the array were greater than 25% saturated. (ii) <br /> tidine (H), sequence, and the TEV protease sequence (T). The kanamycin <br />                                                                                     The mean foreground signal minus the median background signal had to be <br /> resistance region of pUG6-FHT-p was PCR amplified with 68-mer oligonucle- <br />                                                                                     greater than the standard deviation of background signal. (iii) Quality data were <br /> otides with 50-bp homology to distinct regions of MOT1. The PCR products were <br />                                                                                     needed from both replicates of the dye swap. (iv) The directional change of the <br /> transformed into yLAC1 and selected on CSM-HIS-URA (dextrose) plates con- <br />                                                                                     mutant&#8217;s signal (relative to the reference) had to be equivalent in the replicates. <br /> taining 500 mg/ml G418 (Invitrogen). The kanamycin resistance cassette flanked <br /> by loxP sites was removed by induction of Cre recombinase with 2% galactose for     False-discovery rates (FDRs) were determined using a modified version of a <br /> 4 h, leaving the FHT tag coding sequence upstream of the mutation in MOT1.          method described previously (35). The false-discovery rate is reported as a <br /> Kanamycin-sensitive colonies were identified by replica plating them on media       percentage of the number of expression changes above and below a given thresh- <br /> containing and lacking G418. Additionally, mutations were verified by colony        old (&#11006;0.59; log2 scale) in the homotypic control (yjdi420) compared to each <br /> PCR with primers specific to each mutation.                                         FHT-Mot1 mutant expression experiment (yjdi410 to -416). The FDRs are 4.3%, <br />    Haploids. Kanamycin-sensitive FHT-Mot1 strains (Table 1) were plated on          2.0%, 2.4%, 3.0%, and 2.9% for TBP1, TBP2, UK1, ATP1, and Null, respec- <br /> CSM-HIS plus 5-fluoroorotic acid (5-<span id='am-1' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>FOA</span>) to select cells that had lost pSH47        tively. <br /> and verified by replica plating on CSM-HIS and CSM-HIS-URA. The strains                K-means clustering was performed using Cluster (22) on 515 genes that con- <br /> were then transformed with pMR13 (MOT1 wild type [WT] and URA3), and                tained data in 80% of the experiments and that had a change of at least 1.5-fold <br /> transformants were selected on CSM-HIS-URA medium. The strains were                 (log2 ratio &#11005; 0.59) in one of the mutants. K was chosen to equal 6 clusters (K &#11005; <br /> plated on presporulation medium (1% yeast extract, 2% peptone, and 10%              6). Clusters 5 and 6 were merged because they were visually indistinguishable. <br /> dextrose) for 2 days at 30&#176;C. Cells were cultured in sporulation medium (0.3%       Clustering information was visualized using Treeview (22). <br /> potassium acetate, 0.02% raffinose) for 3 days at 30&#176;C. Two hundred microliters        ChIP-chip. ChIP-chip was performed essentially as described previously (48), <br /> of the culture was pelleted; resuspended in 1.2 M sorbitol, 10 mM Tris, pH 7.4;     with minor changes noted below. Briefly, cultures were grown to an A600 of &#11011;0.6, <br /> and treated with 20 units of (1 mg/ml) zymolyase (MP Biomedicals) at room           induced with 2% galactose in YPR medium for 60 min at 25&#176;C, and shifted to <br /> temperature for 20 min. Tetrads were dissected according to standard yeast          37&#176;C for 45 min to inactivate the temperature-sensitive copy of Mot1 encoded by <br /> techniques on YPD (yeast-peptone-dextrose) plates. Spores were replica plated       the mot1-42 allele. The cells were then fixed by adding formaldehyde to a final <br /> onto CSM-HIS-URA medium to select for the HIS3 gene (and therefore the              concentration of 1% for 2 h at 25&#176;C (instead of a typical 15-min cross-link time) <br /> GAL1 promoter). Mating types of the mot1 strains were confirmed with MATa           and quenched for 5 min with <span id='am-6' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="rdfs:label" content="Glycine" datatype="xsd:string"></span>glycine</span>. The cultures were diluted 2-fold with the <br /> and MATa sex tester strains. MATa leu&#11002; HIS&#11001; LYS&#11001; tetrads were selected. The         same volume of temperature-adjusted distilled water just prior to addition of <br /> strains were then transformed with pMOT221 (mot1-42 LEU2) or pAV20                  formaldehyde to achieve a medium temperature of 25&#176;C. The harvested cells <br /> (MOT1 WT LEU2) and selected on CSM-LEU medium. Cells that lost pMR13                were lysed with glass beads, and the chromatin pellet was washed and sonicated. <br /> (MOT1 WT URA3) were selected by plating them on CSM-LEU plus 5-FOA.                 Sheared chromatin was immunoprecipitated with IgG-Sepharose. This ChIP- <br />    Colony PCR. For colony PCR, 1&#11003; 25 mM MgCl2 buffer (Gene Choice), 2.5 U           enriched DNA was amplified by ligation-mediated PCR (LMPCR) as described <br /> Taq polymerase (Gene Choice), 0.0002 U Pfu polymerase (Stratagene), 0.4 mM          elsewhere (27), and 100- to 250-bp LMPCR-amplified fragments were gel puri- <br />  VOL. 31, 2011                                                                          IN VIVO STRUCTURE-FUNCTION STUDY OF Mot1                  2255 <br />  <br /> fied according to the manufacturer&#8217;s protocol (Qiagen) and subsequently <span id='am-78' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-79' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-81'  ></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>hybrid</span>-       2B). The untagged Mot1 (WT) and null controls were not <br /> ized to low-density tiled spotted microarrays containing &#11022;21,000 oligonucleotide      detected because both lack the FHT tag and thus are not <br /> probes as described previously (48). Briefly, each yeast gene is interrogated with <br /> a set of at least 3 oligonucleotide probes, which survey the relative occupancy <br />                                                                                       recognized by the HA antibody. All mutants except UK1 were <br /> levels for a given transcription factor at the &#11002;250 and &#11002;60 sites relative to the     expressed at levels roughly equivalent to that of the wild-type <br /> translational start site and the downstream portion of the open reading frame         Mot1 containing an FHT tag at the N terminus (WT1). UK1 <br /> (ORF). Data were filtered and analyzed as previously described (49) (data are         was expressed at about 50% of the WT1 level. Importantly, the <br /> available at http://atlas.bx.psu.edu/). <br />                                                                                       expression levels of the mutants were not diminished after <br />    Coimmunoprecipitation and immunoblotting. Briefly, FHT-Mot1 mutant <br /> strains were grown as described above for ChIP-chip, except the cells were not        inactivation of the mot1-42 allele at 37&#176;C for 45 min. <br /> cross-linked with formaldehyde. The harvested cells were then pelleted and flash         Four conserved regions of Mot1 contribute essential func- <br /> frozen in liquid nitrogen. The cells were lysed with glass beads in NP-S buffer (10   tions to Mot1. Haploid strains carrying a galactose-inducible <br /> mM Tris-Cl, pH 7.5, 0.5 mM Spermidine, 0.075% Igepal (Sigma), 50 mM NaCl,             chromosomal copy of one of the mot1 deletion mutants and a <br /> 5 mM MgCl2, 1 mM CaCl2). Chromatin pellets were washed in NP-S buffer, the <br /> chromatin was micrococcal nuclease digested (15 units) in a volume of 300 &#9262;l for <br />                                                                                       plasmid-borne temperature-sensitive mot1-42 (16) were tested <br /> 20 min, and then the chromatin was solubilized by washing the spun pellet with        for the ability of the mot1 domain deletion mutants to support <br /> FA lysis buffer (50 mM HEPES, pH 7.5, 150 mM NaCl, 1% Triton X-100, 0.1%              viability. The mot1-42 supporting cell viability was inactivated <br /> Na deoxycholate, 2 mM EDTA). This native nuclear extract was immunopre-               at 37&#176;C and replaced with the domain deletion mutants by <br /> cipitated with anti-TBP rabbit polyclonal antibody serum, and the eluate was <br />                                                                                       adding galactose to the medium. At 25&#176;C and 30&#176;C, mot1-42 <br /> assayed by Western blot analysis for interacting FHT-Mot1 mutants. <br />    For Western blotting, FHT-Mot1 mutant coimmunoprecipitation eluates were           remained functional, allowing cell growth, as expected. In the <br /> electrophoresed in 6% SDS-PAGE and transferred to a polyvinylidene difluoride         presence or absence of the galactose-induced domain deletion <br /> (PVDF) membrane (Pall Gelman Laboratory) in Western transfer buffer for 60            mutants (Fig. 3, 25&#176;C and 30&#176;C), viability was unaffected, <br /> min at 1.5 A. FHT-Mot1 mutants were detected with 1:3,300 anti-HA (HA.11;             indicating that the domain deletion mutants did not have a <br /> Babco) and 1:5,000 anti-mouse&#8211;horseradish peroxidase (HRP) antibodies (Am- <br /> ersham) and exposed to Hyperfilm (Amersham) with enhanced chemilumines- <br />                                                                                       dominant-negative effect on cell growth. At 37&#176;C, wild-type <br /> cence (ECL) (Amersham).                                                               Mot1 (WT and WT1) and all mutants failed to support <br />    Comparisons with public microarray data. The relationships to the top and          viability in dextrose medium, where these proteins are not <br /> bottom 10th percentiles of the expression and ChIP-chip data were calculated in       expressed, verifying the temperature-sensitive nature of the <br /> Excel with the data downloaded from the referenced laboratory or journal&#8217;s web- <br />                                                                                       mot1-42 allele (16). As positive controls, both the untagged <br /> sites. The percent rank of the distribution was calculated with the PERCENTRANK <br /> function. Next, the numbers of genes that appear in the top 10% (&#11022;0.9 in              (WT) and tagged (WT1) inducible Mot1 supported growth <br /> PERCENTRANK) or the bottom 10% (&#11021;0.1 in PERCENTRANK) and also ap-                     in galactose medium at 37&#176;C. However, none of the Mot1 <br /> pear in each cluster were calculated. The CHITEST function of Excel was then used     domain deletion mutants support growth, indicating that <br /> to calculate P values from the observed and expected values.                          each of the four conserved domains is essential in S. cerevi- <br />                                                                                       siae. These findings are consistent with related studies pre- <br />                                                                                       sented elsewhere (1, 7, 16). <br />                                   RESULTS <br />                                                                                          Mot1-regulated genes display strong transcriptional depen- <br />    Replacement strategy for essential Mot1 functions. Mot1 is                         dence on all regions of Mot1. Inasmuch as Mot1 regulates <br /> demarcated by several regions that are highly conserved from                          genes both negatively and positively, we tested whether the <br /> yeast to human (12, 16). To test their in vivo importance, we                         four conserved regions of Mot1 make distinct gene-specific <br /> targeted four conserved regions for deletion, creating mutants                        contributions to gene expression on a genome-wide scale. Ex- <br /> named TBP1, TBP2, UK1, and ATP1 (Fig. 1A to C). The                                   pression of the Mot1 mutants was induced with galactose for <br /> names reflect the associated functional regions (e.g., the TBP1                       60 min, and then the resident functional mot1-42 allele was <br /> deletion removes one of two TBP interaction domains, UK1 is                           shut down by an abrupt temperature shift from 25&#176;C to 37&#176;C <br /> unknown, and ATP1 removes the ATPase domain). Deletion                                for 45 min (Fig. 2A). During this time, we expect heat shock- <br /> of each region of MOT1 was achieved by homologous recom-                              regulated genes to change in expression within 15 min of the <br /> bination using a PCR-amplified cassette containing kanMX                              temperature shift and then return to near-normal expression <br /> flanked by loxP Cre recombination sites (34). The cassette also                       levels by 45 min (11, 23). <br /> contained coding sequences that allowed the deleted region to                            To place any changes in gene expression into the appropri- <br /> be replaced by an FHT epitope tag that encodes a triple-HA                            ate context, each expression profile conducted on a mutant was <br /> tag, a decahistidine tag, and a TEV protease cleavage site.                           also conducted in parallel using a galactose-inducible untagged <br /> After selection for recombinants on G418 plates and subse-                            wild-type MOT1 allele (WT; yjdi420) (Table 1). Thus, if a <br /> quent excision of kanMX with the Cre recombinase, the de-                             mutant is as functional as wild-type Mot1, then no changes in <br /> leted region was replaced with the FHT tag and a single loxP                          gene expression are expected. Changes in gene expression <br /> site, both of which maintained an open reading frame through                          were mode centered, meaning that the most frequent binned <br /> the replaced region. Cell viability was maintained in a resident                      ratio (mutant/WT) corresponded to no change in gene expres- <br /> mot1-42 temperature-sensitive allele, which allowed subse-                            sion. This centering is valid, since most genes are not appre- <br /> quent temperature inactivation of the mot1-42 allele at 37&#176;C                          ciably regulated by Mot1 (4, 17, 24). Log2-transformed changes <br /> (Fig. 2A). The location of each deletion mutation was verified                        in gene expression are presented as a cluster plot (Fig. 4A), <br /> by PCR across the deletion borders, with the appropriate-size                         where red and green denote increased and decreased expres- <br /> products detected (not shown). Expression of the deletion                             sion, respectively. Black denotes no change. Each row corre- <br /> mutants was placed under the control of the GAL1 promoter.                            sponds to a protein-coding gene, and each column corresponds <br /> Immunoblot analysis demonstrated the presence of an appro-                            to the expression profile for a Mot1 mutant. The 515 genes that <br /> priate-size band that reacted with anti-HA antibodies and was                         met a specified cutoff (Fig. 4A) for changes in expression in at <br /> present only after the addition of galactose to the cells (Fig.                       least one set of experiments are shown. The data were clus- <br />  2256     VENTERS ET AL.                                                                                                  MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 1. Mutagenesis of conserved regions in the MOT1 gene. (A) Schematic of the MOT1 gene showing the approximate locations of four <br /> conserved regions. (B) Schematic of the constructed Mot1 mutants. The null control represents WT1 in which the kanMX selection gene was not <br /> removed. (C) Conservation of the four regions of Mot1 targeted for mutagenesis in this study. S. cerevisiae (Sc) and <span id='am-45' about='obo:NCBITaxon_9606' typeof='owl:Thing obo:NCBITaxon_9605 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-46' property="rdfs:label" content="Homo sapiens" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="man" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="human" datatype="xsd:string"></span><span id='am-50'  ></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Homo sapiens</span> (Hs) Mot1 <br /> protein sequences were aligned using the BLAST algorithm (3). <br />  <br />  <br /> tered by K-means (22) into five clusters, representing the max-        expression were very modest compared to the untagged refer- <br /> imum number of visually nonredundant clusters. The <span id='am-52' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-53' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-56'  ></span>homo</span>-               ence (log2 average &#11005; 0.033, or 1.02-fold &#11006; 0.12-fold change), <br /> typic control expression profile, reflecting comparisons               indicating that the FHT tag had little or no effect on expression <br /> between two independent biological replicates of galactose-            genome-wide. In particular, the standard deviation in gene <br /> induced wild-type Mot1, produced no appreciable changes in             expression was not significantly different than that observed <br /> expression (Fig. 4A, WT), as expected.                                 with the homotypic control. <br />    The homotypic control also provided a measure of intrinsic             Changes in gene expression of the galactose-induced null <br /> variability in the data. The mean log2 ratio was 0.045 (1.03- &#11006;        mutant (empty cassette in the mot1-42 strain) at 37&#176;C provided <br /> 0.12-fold [standard deviation] change). To ensure that the             a measure of the maximal level of expected change in expres- <br /> FHT tag was not perturbing expression, an FHT-tagged wild-             sion and correlated well with changes in gene expression re- <br /> type Mot1 control (Fig. 4A, WT1) was tested. Changes in gene           ported for the mot1-1 or mot1-14 allele (4, 17) (Fig. 4B). Thus, <br />  VOL. 31, 2011                                                                 IN VIVO STRUCTURE-FUNCTION STUDY OF Mot1                          2257 <br />  <br />  <br />  <br />  <br />   FIG. 2. Transient-replacement strategy. (A) The transient-replacement approach simultaneously expresses the mot1 test allele from an <br /> inducible galactose-driven promoter while inactivating the temperature-sensitive (ts) version expressed from the mot1-42 allele supporting viability. <br /> Consequently, at the time of harvest, the Mot1 mutants expressed from the galactose promoter have replaced the ts copy of Mot1. (B) Galactose <br /> induction of Mot1 derivatives. An immunoblot probed with monoclonal anti-HA antibodies is shown, revealing the FHT epitope-tagged Mot1 <br /> derivatives. Cells were taken just prior to addition of 2% galactose to raffinose media (&#11002;), after 2% galactose induction for 60 min at 25&#176;C (&#11001;), <br /> or after galactose induction and another 45 min of incubation at 37&#176;C (&#11001;) to inactivate the endogenous mot1-42 allele (not detectable in this assay). <br /> The uniformity of the background bands just above the 121- and 54-kDa markers confirms approximately equal loading and transfer of samples. <br />  <br />  <br />  <br /> this transient-replacement system appears to provide an ade-                 less, and TFIID dominated. In addition, the ribosomal protein <br /> quate reflection of Mot1 dependency.                                         genes dominate this group. The positive contribution of Mot1 <br />    From these expression-profiling experiments, we find genes                to cluster 5 expression is in line with reports proposing that <br /> that are negatively regulated by Mot1 (clusters 1 to 3 in Fig.               Mot1 positively regulates transcription by redistributing TBP <br /> 4A) and those that are positively regulated by Mot1 (cluster 5)              throughout the genome (15, 41) and/or by dismantling tran- <br /> each require the TBP1, TBP2, and UK1 conserved domains of                    scriptionally inactive TBP (18, 46). Taken together, Mot1 typ- <br /> Mot1 in that changes in gene expression were similar to that of              ically regulates 6% of all TFIID-dominated genes in a positive <br /> the null mutant (Fig. 4A, compare columns 3 to 6). Thus, in                  manner, whereas 28% of all <span id='am-57' about='obo:NCBITaxon_441239' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_62784 obo:NCBITaxon_6994 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_70905 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_323649 obo:NCBITaxon_6993 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_33341 obo:NCBITaxon_50557 obo:NCBITaxon_6960 obo:NCBITaxon_6072'><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59'  ></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Saga" datatype="xsd:string"></span>SAGA</span>-dominated genes tend to <br /> general, Mot1 uses the same conserved domains to positively                  be negatively controlled by Mot1. <br /> and negatively regulate transcription. Genes in cluster 1 are                   Deletion of the Mot1 ATPase domain causes ectopic gene <br /> characterized as being stress induced, TATA containing,                      expression. The Mot1 ATP1 mutant, corresponding to a dele- <br /> SAGA dominated, and negatively regulated by a wide range of                  tion of the ATPase domain, had mixed behavior. At most <br /> TBP and chromatin regulators (Table 2). Cluster 2 genes have                 analyzed genes, reflected by clusters 2 and 5 in Fig. 4A, nega- <br /> the same characteristics as those of cluster 1, except that they             tive and positive regulation by Mot1 required the ATPase <br /> are not stress induced and are not inhibited by histones. Clus-              domain as much as it required the other conserved regions. <br /> ter 3 appears to be a mixture of cluster 1 and 2 genes. Genes                Cluster 1 expression displayed less dependence on the ATPase <br /> in cluster 5 are characterized as being stress repressed, TATA-              domain of Mot1. The difference between clusters 1 and 2 may <br />  <br />  <br />  <br />  <br />    FIG. 3. Effects of Mot1 mutants on cell growth. Strains carrying pMOT221 (mot1-42 LEU2) and mot1 mutant derivative alleles were grown at <br /> 25&#176;C in YPR (3% raffinose) to mid-log phase. Then, 0.5 A600 unit of cells were removed, and 2.5 &#9262;l of 10-fold serial dilutions were spotted on three <br /> sets of YPD (2% dextrose) and YPG (2% galactose) plates and incubated at 25&#176;C, 30&#176;C, or 37&#176;C. The mot1 mutant strains are indicated at the <br /> left of the plate images. The photographs were taken after 48 h. <br />  2258      VENTERS ET AL.                                                                                                  MOL. CELL. BIOL. <br />  <br />  <br />                                                                            tions (except ATP1) had little effect on transcription (Fig. 4A, <br />                                                                            columns 3 to 6 in cluster 4). The ATP1 mutant caused an <br />                                                                            increase in transcription. Since the primary TBP binding re- <br />                                                                            gions of this protein are intact, conceivably the ATP1 mutant <br />                                                                            might promote DNA binding of TBP at these genes but is <br />                                                                            unable to dissociate TBP without the ATPase domain. In this <br />                                                                            case, the binding of the Mot1 ATP1 mutant to TBP would <br />                                                                            seem not to completely interfere with subsequent transcription <br />                                                                            complex assembly. A plausible rationale for this lies in our <br />                                                                            observation that cluster 4 genes tend to be repressed by the <br />                                                                            SSN6-TUP1 complex (P &#11005; 10&#11002;39) (Table 2). In this context, <br />                                                                            any enhancement of TBP binding by Mot1(ATP1), even in a <br />                                                                            weakened state, would provide some increase in expression. <br />                                                                            One implication is that stabilization of TBP binding at SSN6- <br />                                                                            TUP1-repressed genes circumvents to some extent SSN6- <br />                                                                            TUP1 repression. <br />                                                                               Cluster 3 genes are generally inhibited by wild-type Mot1 <br />                                                                            but showed decreased expression when the ATPase domain <br />                                                                            was removed (Fig. 4A, column 2, cluster 3), paradoxically sug- <br />                                                                            gesting that the Mot1 ATPase region plays an apparently pos- <br />                                                                            itive role at these genes while Mot1 as a whole plays a negative <br />                                                                            role. Two alternative explanations might account for the ap- <br />                                                                            parent paradox. First, as in cluster 4, the ATP1 mutant might <br />                                                                            stabilize TBP binding at the promoter of cluster 3 genes. How- <br />                                                                            ever, this TBP-Mot1(ATP1) mutant complex seems to inter- <br />    FIG. 4. Genome-wide transcription profiling of Mot1 mutants.            fere with productive transcription complex assembly at these <br /> (A) Cluster analysis of Mot1 mutant expression profiles. Strains har- <br /> boring each of the indicated Mot1 mutants were harvested according <br />                                                                            cluster 3 genes. Alternatively, the ATP1 mutant might promote <br /> to the transient-replacement strategy (Fig. 2A). mRNA was isolated         nonspecific binding of TBP to the genome, which would reduce <br /> and cohybridized, along with an untagged wild-type reference (WT), to      the amount of TBP that could be recruited to cluster 3 genes, <br /> spotted full-length PCR-generated ORF microarrays. Changes in gene         resulting in decreased expression of cluster 3 genes. This would <br /> expression (log2 scale) were clustered using Cluster software and vi- <br /> sualized with Treeview (22). Membership required quality data in six <br />                                                                            be consistent with the notion that Mot1 normally removes TBP <br /> of the seven experiments and a log2 absolute value of 0.59 (1.5-fold       from nontargeted regions of the genome (41). We explore <br /> change) in at least one experiment and data in six of the seven clus-      these possibilities below. <br /> tered experiments. A total of 515 ORFs met these criteria, with FDRs          Mot1(ATP1) directly interacts with TBP. Potential interpre- <br /> across all experiments of &#11349;4.3%. Rows were clustered by K means into <br />                                                                            tations of the Mot1(ATP1) expression profile are predicated <br /> five clusters (n &#11005; 120, 120, 21, 49, and 209). The rows represent <br /> individual genes, and fold changes in gene expression are reflected in     upon the Mot1(ATP1) mutant maintaining the ability to inter- <br /> the color intensity. Columns representing an average of at least two       act with TBP. Thus, to test whether Mot1(ATP1) directly in- <br /> dye-swapped experiments were clustered hierarchically. The dendro-         teracts with TBP in vivo, we performed coimmunoprecipitation <br /> gram relating them is depicted above the column labels. The table <br />                                                                            assays for TBP in the FHT-Mot1 mutant strains. Growth and <br /> below the cluster plot provides the median log2 ratio in each cluster in <br /> each experiment. (B) Moving-average comparison of genome-wide              Mot1 replacement were performed as in the expression stud- <br /> changes in expression for mot1-42 (null) with mot1-1 (4) and mot1-14       ies. Importantly, to maintain native protein-protein interac- <br /> (17). Fold changes in gene expression (log2 scale) for mot1-42 were        tions, the cells were not cross-linked throughout the coimmu- <br /> sorted. The average changes in expression for mot1-42 in each sliding      noprecipitation procedure. TBP was immunoprecipitated from <br /> 200-gene window was plotted against an equivalent moving average <br /> obtained from mot1-1 and mot1-14 strains.                                  digested, soluble chromatin derived from Mot1 mutant cells. <br />                                                                            The ability of Mot1 mutants to interact with TBP on chromatin <br />                                                                            was revealed by immunoblot analysis (Fig. 5A). The positive <br /> be subtle, such as a greater rate-limiting dependency on TBP               control, the tagged (WT1) inducible Mot1, showed an interac- <br /> binding versus ATP hydrolysis. Alternatively, since cluster 1              tion with TBP (Fig. 5A, bottom). As in Fig. 2B, the untagged <br /> genes were generally induced and then partially repressed by               Mot1 (WT) and null controls were not detected, because both <br /> the temperature regime used here, it is possible that while                lack the FHT tag and thus are not recognized by the HA <br /> Mot1 binding is needed for this shutdown, the Mot1 ATPase                  antibody. Among the Mot1 mutant derivatives, the <br /> activity may be partially dispensable. In other words, Mot1                Mot1(ATP1) mutant showed the strongest interaction in vivo <br /> binding in the absence of ATPase activity is sufficient to elicit          with TBP, while the other mutants (TBP1, TBP2, and UK1) <br /> some repression. This would be consistent with current models              displayed either weak or no interaction. Therefore, the inter- <br /> of Mot1 function in which simply binding to TBP would be                   action between Mot1(ATP1) and TBP revealed by their coim- <br /> sufficient to preclude binding to certain general transcription            munoprecipitation supports one assertion of the interpretation <br /> factors.                                                                   that expression changes in clusters 3 and 4 may be a direct <br />    Cluster 4 genes appeared to be largely Mot1 independent,                result of the Mot1(ATP1) mutant altering the DNA-binding <br /> since expression of the null or any of the Mot1 domain dele-               status of TBP at these genes. <br />  VOL. 31, 2011                                                                              IN VIVO STRUCTURE-FUNCTION STUDY OF Mot1                                    2259 <br />  <br />  <br />                                    TABLE 2. Relationship of Mot1 expression profiles with published microarray data <br />                                                                                                                          P value for clustera: <br />        Category                            Protein/characteristic                    % rank                                                                      Reference <br />                                                                                                          1          2             3          4        5 <br />  <br /> Group/property                   TATA box                                                               52         26            10           9       &#8211;              8 <br />                                  SAGA dominated                                                         73         33            13          &#8211;        &#8211;              32 <br />                                  TATA box and SAGA dominated                                            85         26            18          &#8211;        &#8211;              32 <br />                                  ESRb upregulated                                                       69          &#8211;             7          &#8211;        &#8211;              23 <br />                                  ESR downregulated                                                       &#8211;          &#8211;             &#8211;          &#8211;        52             23 <br />                                  Rap1 ChIP and ribosomal protein                                         &#8211;          &#8211;             &#8211;          &#8211;       101             37 <br />                                  High transcription frequency                                            &#8211;          &#8211;             &#8211;          &#8211;        40             30 <br />  <br /> Expression <br />   Stress response                Heat shock                                        Top 10               68           6             7         &#8211;        &#8211;              51 <br />                                  Salt stress                                       Top 10               35          &#8211;             &#8211;          &#8211;        &#8211;              11 <br />                                  Alkali stress                                     Top 10               36         12             &#8211;          &#8211;        &#8211;              11 <br />                                  Diauxic shift                                     Top 10               29          &#8211;             &#8211;          &#8211;        &#8211;              20 <br />                                  Oxidative stress                                  Top 10               25          &#8211;             &#8211;          &#8211;        &#8211;              11 <br />                                  Amino acid starvation                             Top 10               23         11             &#8211;          &#8211;        &#8211;              28 <br />   Regulatory factors             mot1-14                                           Top 10               59         45            19         &#8211;         &#8211;              17 <br />                                  mot1-14                                           Bottom 10             &#8211;          &#8211;             &#8211;          &#8211;        81             17 <br />                                  mot1-1                                            Top 10               33         19             &#8211;         &#8211;         &#8211;              4 <br />                                  mot1-1                                            Bottom 10             &#8211;          &#8211;             &#8211;          &#8211;        14             4 <br />                                  bur6-1 (NC2)                                      Top 10               39         17            17          &#8211;        &#8211;              10 <br />                                  tup1&#9004; (SSN6-TUP1)                                 Top 10               14          &#8211;              7        39        &#8211;              31 <br />                                  TBP K145E (Mot1 binding)                          Top 10               24         35             &#8211;          &#8211;        &#8211;              33 <br />                                  TBP F182V (NC2 binding)                           Top 10               76         30             &#8211;          &#8211;        &#8211;              33 <br />   Nucleosome                     H3&#9004;1-28                                           Top 10                 8         &#8211;              6          8       &#8211;              42 <br />                                  H4&#9004;2-26                                           Top 10               31          &#8211;             &#8211;          &#8211;        &#8211;              43 <br />                                  htz1&#9004;                                             Top 10               26          &#8211;             &#8211;          &#8211;        &#8211;              40 <br />   a <br />     The &#11002;log10(P value) for overlap between membership in each of the five Mot1 expression clusters and the top/bottom 10th percentile expression change or <br /> group/property membership from published genomic data sets. The P value was calculated using the chi-square test and returns the probability that the overlap between <br /> the two data sets occurs by chance. Thus, the most statistically significant relationships have the largest &#11002;log10(P value). For clarity &#11002;log10 (P value) values of less than <br /> or equal to 5 were replaced with dashes. <br />   b <br />     ESR, environmental stress response. <br />  <br />  <br />  <br />  <br />    Deletion of the Mot1 ATPase domain causes TBP redistri-                               increased in the Mot1(ATP1) mutant at cluster 4 genes. Be- <br /> bution in the genome. To further understand how the                                      cause cluster 3 genes normally have more TBP than cluster 4 <br /> Mot1(ATP1) mutant might affect TBP recruitment to promot-                                genes (Fig. 5C), they have more to lose if TBP is distributed <br /> ers, we used genome-wide location analysis (ChIP-chip) to                                nonspecifically throughout the genome in the Mot1(ATP1) <br /> monitor the changes in TBP, Mot1, and TFIID (Taf1 and Taf4                               mutant. <br /> subunits) occupancy at every yeast gene in the Mot1(ATP1)                                   One interpretation of the positive activity of Mot1 on ex- <br /> mutant. Growth and Mot1 replacement were performed as in                                 pression at cluster 5 genes is that it removes an inactive form <br /> the expression studies. Each factor was immunoprecipitated                               of TBP, allowing productive binding of TFIID. Accordingly a <br /> from sheared soluble chromatin derived from formaldehyde                                 Mot1(ATP1) mutant might &#8220;lock down&#8221; TBP at cluster 5 <br /> cross-linked wild-type (WT1) and Mot1(ATP1) mutant cells.                                genes, thereby preventing TFIID from binding. To test this <br /> Bound DNA was differentially labeled and cohybridized to                                 hypothesis, we conducted ChIP-chip on the Taf1 and Taf4 <br /> microarrays containing all intergenic regions. Median log2                               subunits of TFIID, comparing its occupancy at cluster 5 genes <br /> changes in TBP and Mot1 occupancy for each of the clusters                               in the wild type versus a Mot1(ATP1) mutant. Consistent with <br /> defined in Fig. 4A are plotted in Fig. 5B and compared to                                this hypothesis, Taf1 and Taf4 occupancy levels decreased at <br /> changes in gene expression for the Mot1(ATP1) and null mu-                               cluster 5 genes in the Mot1(ATP1) mutant, whereas TBP levels <br /> tants.                                                                                   remained largely unchanged (Fig. 5D). A constant level of TBP <br />    TBP and Mot1 occupancy changes at cluster 3 and 4 genes                               is consistent with the hypothesis, in that one type of TBP (TAF <br /> mirrored the expression output for these genes in the ATP1                               free) replaces another type of TBP (TAF bound; TFIID). <br /> mutant (a decrease at cluster 3 and an increase at cluster 4), <br /> suggesting that the expression change is a direct result of the                                                             DISCUSSION <br /> Mot1(ATP1) mutant altering the DNA-binding status of TBP <br /> at these genes. The loss of TBP at cluster 3 genes suggests that                            The binding of TBP to DNA is generally considered to be a <br /> when Mot1 lacks its ATPase domain, the corresponding loss in                             primary nucleating event in transcription complex assembly. <br /> transcription is not due to stabilization of an inactive form of                         TBP-DNA binding is therefore subjected to substantial posi- <br /> TBP at these promoters. Instead, the results are more consis-                            tive and negative regulation. TBP not only binds to the TATA <br /> tent with the loss of TBP possibly being due to stabilization of                         box located in promoters, it also binds to TATA-less promoter <br /> TBP bound to other sites in the genome (e.g., cluster 4 genes)                           regions, and it binds to nonspecific DNA with fairly high af- <br /> by the Mot1(ATP1) mutant. Indeed, TBP and Mot1 occupancy                                 finity. Since nonspecific DNA binding by TBP can nevertheless <br />  2260     VENTERS ET AL.                                                                                                               MOL. CELL. BIOL. <br />  <br />  <br />                                                                          under nonstress conditions (11, 23). These promoters tend to <br />                                                                          rely more on a free form of TBP that does not involve TFIID. <br />                                                                          This form of TBP may be directed to the appropriate promot- <br />                                                                          ers via the SAGA complex, and since it is not TFIID, it may be <br />                                                                          more accessible to Mot1. Indeed, the abundance of accessible <br />                                                                          TBP at such promoters attracts Mot1, where it downregulates <br />                                                                          expression (cluster 1 in Fig. 4A). A moderately high level of <br />                                                                          expression is achieved through this balance of positive SAGA- <br />                                                                          TBP action (among other factors) and negative Mot1-TBP <br />                                                                          action (among other factors). Indeed, by using multidimen- <br />                                                                          sional chromatography coupled to mass spectrometry (5), a <br />                                                                          recent study found that Mot1 interacts with a variety of acti- <br />                                                                          vators and transcriptional coregulators, such as Msn2, Hsf1, <br />                                                                          and RSC. <br />                                                                             Since Mot1 has the ability to bind both TBP and DNA (44), <br />                                                                          it potentially can stabilize TBP-DNA interactions. However, <br />                                                                          this is not realized in general because Mot1 uses the energy of <br />                                                                          ATP hydrolysis to remove TBP from DNA. A form of Mot1 <br />                                                                          that lacks the ATPase domain provides a window into how <br />                                                                          potential stabilization of TBP-DNA interactions through Mot1 <br />                                                                          affects the distribution of TBP genome-wide. If the <br />                                                                          Mot1(ATP1) mutant were to increase TBP-DNA stability in an <br />                                                                          undirected way, then genomic loci that normally lack TBP <br />    FIG. 5. TBP coimmunoprecipitation (CoIP) and location profiling <br />                                                                          should see an increase in TBP in the mutant. Those loci that <br /> of TBP, Mot1, and TFIID subunits in an ATPase-defective Mot1             normally have higher TBP levels should suffer a decrease in <br /> strain. (A) TBP was immunoprecipitated from chromatin extracts and       occupancy as TBP is sequestered at the vast number of non- <br /> then probed by Western blotting for both TBP (top) and FHT-Mot1          specific loci. Indeed we identified a set of genes (cluster 4) that <br /> (anti-HA antibodies) (bottom). (B) ChIP-chip was performed in an         had lower levels of TBP and whose levels of TBP, Mot1, and <br /> ATPase-defective Mot1 mutant (ATP1) and wild-type Mot1 strain <br /> (WT1) background. The median log2 changes in occupancy for each of       transcription increased in the Mot1(ATP1) mutant. We iden- <br /> the five clusters in Fig. 4A were calculated. For comparison, the me-    tified another set of genes (cluster 3) that had higher levels of <br /> dian log2 expression changes for the null and ATP1 expression profiles   TBP, which decreased in the Mot1(ATP1) mutant. <br /> from Fig. 4A are also shown. (C) The median TBP occupancy at 25&#176;C           Taken together, our findings suggest that the four conserved <br /> from a previous study (50) for each cluster is shown as a histogram. <br /> Log2 values are relative to nonpromoter intergenic regions (&#11022;1,800), <br />                                                                          regions of Mot1 are essential for viability and required for <br /> which are regions located between two convergently transcribed genes.    proper regulation of most Mot1-regulated genes. Loss of the <br /> (D) ChIP-chip was performed on two subunits of TFIID, Taf1 and           ATPase domain, however, imparts some unexpected regula- <br /> Taf4. Shown is a histogram of the median log2 changes in occupancy       tion on certain genes. Although only a fraction of all yeast <br /> for each of the five clusters in Fig. 4A for Taf1 and Taf4 occupancy     genes are overtly regulated by Mot1, essentially all yeast genes <br /> changes in an ATPase-defective Mot1 strain relative to wild-type Mot1 <br /> (TBP is also shown for comparison).                                      require TBP and thus are potential targets for Mot1. Genes <br />                                                                          that are insensitive to loss of Mot1 likely reflect those that are <br />                                                                          relatively quiescent and thus lack TBP or those that involve <br />                                                                          TFIID, which is a Mot1-insensitive form of TBP. Allowing <br /> nucleate transcription (14), robust mechanisms exist in vivo to          TBP to redistribute genome-wide in a potentially more non- <br /> prevent promiscuous assembly.                                            specific manner results in a net gain of TBP at TBP-deficient <br />    Mot1 plays an important role in removing TBP from inap-               genes and a net loss at TBP-enriched genes. Thus, one conse- <br /> propriate genomic sites (18, 41), which would free up TBP                quence of Mot1 using ATP hydrolysis to remove TBP from <br /> and/or the underlying DNA to engage in productive interac-               DNA may be an increase in promoter selectivity. <br /> tions (46). In this way, Mot1 would play a positive role at <br /> promoters that are rate limited either by the availability of TBP                                   <br /> or by dissociation of an inactive TBP-promoter complex. Such             <br /> an arrangement might predominate at ribosomal protein                   <br /> genes, where Mot1 plays a positive role (cluster 5 in Fig. 4A).         <br /> Conceivably, a Mot1-inaccessible form of TBP (i.e., TFIID) <br />                                                                    <br /> binds to ribosomal promoters (12, 39). Nonproductive binding <br /> of a TFIID-independent form of TBP might antagonize TFIID                                                  <br /> recruitment or any other stage in transcription initiation. Re-      <br /> moval of this nonproductive TBP would therefore positively                   <br /> impact transcription at these genes. Indeed, we find that sta-              <br /> bilizing a TBP-Mot1 promoter interaction at Mot1-upregu-             <br /> lated genes has the effect of displacing TFIID.                             <br />    Stress-induced promoters often are highly transcribed even            <br /> </body></html>