<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 24 September 2010                                                Nucleic Acids Research, 2011, Vol. 39, No. 3 795&#8211;807 <br />                                                                                                                  doi:10.1093/nar/gkq784 <br />  <br />  <br /> Investigating predictability essential genes <br /> across distantly related organisms using an <br /> integrative approach <br /> Jingyuan Deng1,2, Lei Deng1,3, Shengchang Su4, Minlu Zhang5, Xiaodong Lin6, Lan Wei7, <br /> Ali A. Minai3, Daniel J. Hassett4 Long J. Lu1,2,5,8,* <br /> 1 <br />  Division Biomedical Informatics, Cincinnati Children&#8217;s Hospital Research Foundation, Cincinnati, OH 45229, <br /> 2 <br />  Department Biomedical Engineering, 3Department Electrical Computer Engineering, 4Department of <br /> Molecular Genetics, Biochemistry Microbiology, 5Department Computer Science, University Cincinnati, <br /> Cincinnati, OH 45229, 6Department Management Science Information Systems, Rutgers University, <br /> Piscataway, NJ 08854, 7School Medicine, Yale University, New Haven, CT 06511 8Department of <br /> Environmental Health, University Cincinnati, Cincinnati, OH 45229, USA <br />  <br /> Received May 28, 2010; Revised August 15, 2010; Accepted August 18, 2010 <br />  <br />  <br />  <br /> ABSTRACT <br />                                                                                 robust portable existing approaches, sig- <br /> Rapid accurate identification new essential                              nificantly extending ability predict essential <br /> genes under-studied microorganisms signifi-                             genes orthologs. <br /> cantly improve understanding cell <br /> works ability re-engineer microorgan- <br /> isms. However, predicting essential genes                                INTRODUCTION <br /> distantly related organisms remains challenge.                                The recent success &#8216;synthetic bacteria&#8217; rekindled <br /> Here, present machine learning-based integra-                              people&#8217;s enthusiasm using systems synthetic <br /> tive approach reliably transfers essential gene                            biology serve needs humanity re-engineering <br /> annotations distantly related bacteria. We                              microorganisms (1,2). Essential genes important in <br /> focused bacterial <span id='am-124' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-125' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>                                     bioengineering efforts functional micro- <br />                                                                                 organism contain minimal set essential genes <br /> well-characterized essential genes, tested the <br />                                                                                 required survival carrying desired <br /> transferability pairs them.                                 functions. Studying gene essentiality important in <br /> For pair, trained classifier learn                               basic science crucial step under- <br /> traits associated essential genes                                   standing complex relationship genotype and <br /> organism, applied make predictions                             phenotype (3). To date, genomic-scale knockout experi- <br /> other. The predictions evaluated                                   ments dozen microorganisms shown that <br /> examining agreements known essential                               different organisms share limited number essen- <br /> genes           target    organism.    Ten-fold                          tial genes. Therefore, rapid accurate identi&#65533;?cation of <br /> cross-validation organism yielded AUC                               essential genes under-studied organism, especially <br /> scores 0.86 0.93. Cross-organism pre-                               unique organism, poses signi&#65533;?cant chal- <br /> dictions yielded AUC scores 0.69 0.89.                              lenge post-genomic era. The lack ability <br />                                                                                 prevented fully comprehending cell <br /> The transferability likely affected growth con- <br />                                                                                 works ef&#65533;?ciently re-engineering microorganisms <br /> ditions, quality training <span id='am-227' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-228'  ></span><span id='am-229' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-230' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-231' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> set evo-                          energy, bioremediation, pharmaceutical and <br /> lutionary distance. We report                             biodefence applications (4&#8211;6). <br /> that gene essentiality reliably predicted                                   Experimental identi&#65533;?cation essential genes ac- <br /> using features trained tested distantly                                complished targeted mutagenesis, speci&#65533;?c <br /> related organism. Our approach proves                                      genes identi&#65533;?ed prior genetic manipulations and <br />  <br />  <br />  <br /> *To correspondence addressed. Tel: +1 513 636 8720; Fax: +1 513 636 2056; Email: long.lu@cchmc.org <br />  <br /> The authors wish known that, opinion, &#65533;?rst authors regarded joint First Authors. <br />  <br /> &#223; The Author(s) 2010. Published Oxford University Press. <br /> <span id='am-109' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-110' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-112' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-113'  ></span>This</span> Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/2.5), permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />  796 Nucleic Acids Research, 2011, Vol. 39, No. 3 <br />  <br />  <br /> con&#65533;?rmatory studies based experimental data,                         relevant features essential genes makes predictions <br /> or random mutagenesis, target genes                          using weighted combination hallmark features. <br /> identi&#65533;?ed experimental disruptions (7).                           By taking advantage near-complete mapping <br /> While targeted mutagenesis produces reliable                             essential genes <span id='am-72' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80'  ></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br /> results higher accuracy, random mutagenesis                              (13&#8211;14) E. coli (15&#8211;17), collection genomic <br /> appears cost-effective. Nonetheless, genomic-                      features identi&#65533;?ed correlate gene essen- <br /> scale systematic screening lethal gene disruptions                     tiality (Table 1). These features broadly classi&#65533;?ed <br /> either approach formidable undertaking. Taken                            categories: intrinsic context-dependent <br /> together, universal consensus arguably multiple                          features. Intrinsic features determined the <br /> laboratories required ensure results                       genomic sequences, e.g. GC content, depend <br /> accurate, requiring consortium labs                           external conditions. Context-dependent features <br /> years research.                                                            derived organism&#8217;s genome and <br />    To circumvent expense dif&#65533;?culty                           vary different conditions, e.g. &#64258;uctuations in <br /> screens, researchers attempting identify essential                         mRNA expression. The dependency essentiality on <br /> genes under-studied organisms rely                        categories features suggests multiple aspects <br /> homology mapping help elucidate essential genes.                           biology unite make gene essential. Therefore, <br /> However, method limitations. First,                          taking account similarity genomic se- <br /> homology mapping limited conserved orthologs                        quences provides limited perspective highly <br /> between species, correspond small                            complex property. <br /> portion target bacterial genome (8). For example,                           A number machine learning systems de- <br /> <span id='am-179' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-180' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-181'  ></span><span id='am-182' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-183' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-184' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-188' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-191' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-192' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-193' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-194' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia coli</span> <span id='am-140' about='obo:NCBITaxon_286' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621'><span id='am-141' property="oboInOwl:hasDbXref" content="PMID:9103607" datatype="xsd:string"></span><span id='am-142' property="rdfs:label" content="Pseudomonas" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasDbXref" content="PMID:15950132" datatype="xsd:string"></span><span id='am-144'  ></span><span id='am-145' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasRelatedSynonym" content="Loefflerella" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasRelatedSynonym" content="Peudomonas" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasDbXref" content="PMID:7727274" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasDbXref" content="PMID:10939664" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasRelatedSynonym" content="RNA similarity group I" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasDbXref" content="PMID:23918787" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasDbXref" content="PMID:10758879" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasDbXref" content="PMID:18048745" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasRelatedSynonym" content="Liquidomonas" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Pseudomonas</span> aeruginosa share                             veloped integrate subset features cross- <br /> &lt;35% genes orthologs. In addition,                       validation essential genes S. cerevisiae E. coli, <br /> essential genes tend conserved, conserved genes                     showing various degrees prediction accuracy (AUC: <br /> often essential. For example, quarter                      0.7&#8211;0.82; PPV: 0.34&#8211;0.68) (18&#8211;21). However, these <br /> the highly conserved bacterial genes essential                      studies address applicability methods <br /> tested experimentally model bacteria (9&#8211;12). Finally,                      novel genomes. Such applicability important, because <br /> beyond testability homology mapping, differences                       useful predictor perform outside organism <br /> in genetic regulation protein modi&#65533;?cation, genetic re-                     trained. Seringhaus et al. (22) recently <br /> dundancy divergence cellular pathways processes                      developed method train classi&#65533;?ers S. cerevisiae <br /> between organisms great bearing                              essential genes applied predict in <br /> relative essentiality. For example, alaS gene encodes                    S. mikatae. However, limited knowledge of <br /> alanyl-tRNA synthetase essential E. coli                            essential genes S. mikatae, impossible system- <br /> non-essential P. aeruginosa. It non-essential                           atically assess performance classi&#65533;?ers in <br /> in P. aeruginosa probably paralog                            transferring gene essentiality. More importantly, when <br /> (PA2106) carries function.                                  predicting essential genes unstudied organism, a <br /> Considering limitations, group researchers                     closely related well-studied organism classi- <br /> reached disappointing conclusion &#8216;gene conser-                         &#65533;?ers trained unavailable. Therefore, it <br /> vation bacteria does necessarily indicate                      useful machine learning-based <br /> essentiality organism extrapolated others&#8217;                   approach proven applicable distantly <br /> (12).                                                                         related species. <br />    Thus, study, developed machine learning-                         In study, re-examined question of <br /> based integrative approach alternative transfer                      gene essentiality accurately transferred <br /> gene essentiality annotations organisms. In                           organisms machine learning approach. We <br /> contrast homology mapping relies strictly                      &#65533;?rst focused bacterial species: E. coli and <br /> similarity nucleotide sequences, approach identi&#65533;?es                   <span id='am-130' about='obo:NCBITaxon_469' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_468 obo:NCBITaxon_1236 obo:NCBITaxon_72274'><span id='am-131' property="oboInOwl:hasDbXref" content="PMID:7520730" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasDbXref" content="PMID:10028249" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasDbXref" content="PMID:9226915" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasDbXref" content="PMID:8934907" datatype="xsd:string"></span><span id='am-136'  ></span><span id='am-137' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasDbXref" content="PMID:9336926" datatype="xsd:string"></span><span id='am-139' property="rdfs:label" content="Acinetobacter" datatype="xsd:string"></span>Acinetobacter</span> baylyi ADP1. Both bacteria have <br />  <br />  <br />  <br /> Table 1. Features correlated gene essentiality S. cerevisiae E. coli <br />  <br /> References                         Genomic features <br />  <br /> Jeong et al. (41)                  (i) Fluctuation mRNA expression; (ii) Protein functions; (iii) Connectivity protein&#8211;protein interaction <br />                                       (PPI) network <br /> Chen Xu (18)                   (i) Evolutionary rate; (ii) Duplication rate; (iii) Gene expression correlation network; (iv) Connectivity PPI <br />                                       network <br /> Saha Heber (20)                (i) Phylogenetic conservation; (ii) Degree paralogy; (iii) Number PPIs <br /> Seringhaus et al. (22)             14 intrinsic features, as: GC content; length protein; hydrophobicity; <span id='am-114' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-115' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-116' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-118'  ></span>codon</span> adaptation index; predicted <br />                                       subcellular localization compartments, etc <br /> Gustafson et al. (21)              (i) Codon usage; (ii) Paralogs; (iii) Size localization; (iv) Protin interaction network degree; (v) Phyletic re- <br />                                       tention measure; (vi) Recombination rate; (vii) Strand <span id='am-67' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-69'  ></span><span id='am-70' property="rdfs:label" content="Bias" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>bias</span>; (viii) Regulatory complexity, etc <br />                                                                             Nucleic Acids Research, 2011, Vol. 39, No. 3 797 <br />  <br />  <br /> well-characterized essential genes targeted mutagenesis.      Homology mapping COG groups <br /> For pair, developed classi&#65533;?er learn traits <br />                                                                  We implemented alternative approach homology <br /> of essential genes organism predict              mapping using COG groups (30). The COG groups <br /> those other. We systematically evaluated         contain list genes organisms same <br /> predictions examining overlaps known             distinct phylogenetic lineage. We used COG <br /> essential genes target organism, compared         groups map essential genes organism <br /> performance homology mapping. To examine                other. Thus, gene EC essential, then <br /> applicability approach, tested transfer-      PA genes COG group assumed be <br /> ability pairs bacteria: E. coli              essential. <br /> P. aeruginosa PAO1, E. coli <span id='am-34' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-35' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-61'  ></span><span id='am-62' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span>Bacillus subtilis</span>.              COG groups unavailable genes AB. <br />                                                                  Therefore, performed homology mapping between <br />                                                                  EC AB based sequence identity groups. We <br /> MATERIALS AND METHODS                                            clustered AB proteins groups based their <br />                                                                  sequence identities other. Each protein <br /> Data sources <br />                                                                  sequence AB searched protein sequences <br /> E. coli K-12 (EC) sequence data downloaded             AB Blastp E-value cutoff 1e&#8211;5. Proteins <br /> Comprehensive Microbial Resource (CMR) database               considered sequence identity group every <br /> http://cmr.jcvi.org/. It contains 4289 protein sequences         pair proteins (i) sequence similarity satis&#65533;?es the <br /> in total (23). The essential genes E. coli K-12          E-value cutoff mutually (ii) sequence identities &gt;35%. <br /> downloaded PEC database (17). The Kato data             Similarly, protein sequence identity groups extracted <br /> set contains 302 essential genes gene deletion              EC. If protein organism essential, all <br /> experiments.                                                     proteins organism sequence <br />    P. aeruginosa PAO1 (PA) sequence data down-              identity group essential protein predicted RBH <br /> loaded                http://www.pseudomonas.com/           considered essential. <br /> (Pseudomonas_aeruginosa_PAO1.faa, revision 17 July <br /> 2009). PA essential genes downloaded (24).             Intrinsic context-dependent genomic features <br /> The Jacobs dataset contains 678 essential genes             To create training dataset classi&#65533;?er, features are <br /> transponson mutagenesis PAO1.                                 extracted available ORF organism <br />    A. baylyi ADP1 (AB) sequences collected         annotated known essentiality values the <br /> Magnifying Genomes database (http://www.genoscope.               essential gene data sets. Our study considered main <br /> cns.fr/). Out total 3308 genes, 499 essential        types features: (i) intrinsic gene&#8217;s sequence <br /> genes (25).                                                 (e.g. GC content, protein length); (ii) derived from <br />    B. subtilis (BS) sequence data downloaded           genomic sequence (e.g. localization signals codon <br /> Microbial Genome Database (http://mbgd.genome.ad                 adaptation measures) (iii) experimental functional <br /> .jp/). This data set contains 4117 ORFs. The essential           genomics data (e.g. gene-expression microarray data) <br /> gene list downloaded (26) consists               (Supplementary Table S1). <br /> three data sets: (i) 150 essential genes determined              (i-a) Genomic sequence properties. Evolutionary selec- <br /> Kobayashi&#8217;s mutantgenesis experiments; (ii) 42 known             tion genes detected base compos- <br /> essential genes previous studies; (iii) 79 essential        ition genome, e.g. GC content, overall amino <br /> genes homology mapping bacteria,             acid composition, e.g. hydrophobicity (31). Essential <br /> which encode proteins involved ribosome synthesis.         genes evolutionarily <br />    Gene expression data bacteria down-             conserved non-essential genes bacteria the <br /> loaded NCBI GEO (27), ArrayExpress (28),            negative selection essential genes stringent <br /> as Gasch et al. (29).                                       non-essentials (32). Therefore, correlating the <br />                                                                  base amino acid composition essentiality may <br /> Homology mapping reciprocal best hit                          reveal evolutionary constraints predictive of <br />                                                                  essential genes. We used CodonW (http://bioweb.pasteur <br /> We developed reciprocal best hit (RBH) method               .fr/) calculate following properties associated with <br /> identify orthologs EC PA: We                     genomic sequences: Kyte Doolittle&#8217;s grand average of <br /> &#65533;?rst queried ORFi PA known ORFs                hydropathicity (GRAVY) (33), protein length (amino <br /> in EC Blastp, E-value threshold 10 5,           acids), GC content measures codon usage: ef- <br /> yield set hits {W}. Then, queried                  fective Nc (34&#8211;35) CAI (36). <br /> hit lowest E-value {W} (ORFj)                   (ii-a) Predicted subcellular localization. Subcellular lo- <br /> all ORFs PA yield set hits {Y}. A pair              calization potentially predictive gene essentiality <br /> of proteins (ORFi, ORFj) considered putative                 gene products carry speci&#65533;?c functions <br /> orthologs ORFiis hit {Y} lowest               con&#65533;?ned certain subcellular compartments. <br /> E-value, meet strict                        For example, proteins involved essential functions <br /> criteria: (i) LengthShorterProtein =LengthLongerProtein  80%;    information storage processing DNA repli- <br /> (ii) LengthAlignedRegion =LengthShorterProtein   80% ensure   cation, recombination mRNA synthesis locate in <br /> suf&#65533;?cient coverage aligned regions.                           nucleus Eukaryotes cytoplasm Prokaryotes (37). <br />  798 Nucleic Acids Research, 2011, Vol. 39, No. 3 <br />  <br />  <br /> In contrast, membrane proteins function trans-            narrower range, expression of <br /> porters participate metabolic related processes. This       non-essential genes &#64258;uctuates widely (41). This is <br /> also explains protein transmembrane <span id='am-96' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-97' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-99'  ></span><span id='am-100' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>helix</span>          potential existence feedback mechanisms <br /> (PredHel) likely non-essential                     stabilize expression level essential genes. <br /> (Supplementary Figure S1). We used PA-SUB Server              It observed wide &#64258;uctuation expres- <br /> v2.5 obtain features (38). Gram-negative bacteria        sion level essential genes cause death an <br /> (EC, PA AB) &#65533;?ve predicted localizations: inner           organism (41). The variance gene calculated <br /> membrane, extracellular, cytoplasm, periplasm outer           gene expression pro&#65533;?les measure the <br /> membrane. Gram-positive bacteria (BS) pre-             &#64258;uctuation gene expression. <br /> dicted localizations: extracellular, cytoplasm plasma             (iii-b) Topology gene co-expression network. <br /> membrane.                                                         Previous research shown protein&#8211;protein inter- <br />    (ii-b) Transmembrane helices ORF. The                 action (PPI) network, hubs highly connected proteins, <br /> putative transmembrane helices calculated                 likely essential evolve slowly (42&#8211;44). It <br /> TMHMM Web server v2.0 (39,40).                                    shown interacting proteins tend be <br />    (ii-c) Phylogenetic pro&#65533;?le gene. Essential genes      co-expressed involved same <br /> more evolutionarily conserved non-essential genes            pathway (45,46). From gene expression microarray data, a <br /> (14,20&#8211;21). This essential genes              gene-expression cooperativity graph constructed as <br /> likely involved basic cellular processes,             Gg (D) = (Vg, Eg), vertex set Vg &#188; fdi jdi 2 Dg <br /> negative selection acting essential genes strin-      edge set Eg &#188; f&#240;di ,dj &#222;jdi ,dj 2 Dg 6&#188; j and <br /> gent non-essentials (32). We used RBH                jrij j   0.7. Each vertex represents gene edge <br /> method search orthologs multiple complete                   represents gene pair gene expression pro&#65533;?les cor- <br /> genomes gene target organism (PA, EC,             relation coef&#65533;?cient jrij j &gt;0.7. This cutoff value jrij j is <br /> AB BS). The number genomes                       determined based previous work (46). The hubs <br /> orthologous hits used measure evolutionary            (nodes high degrees) bottlenecks (nodes with <br /> conservation gene. Such conservation shown          high betweenness shortest paths occurrence) have <br /> to correlate dispensability gene (18).         correlations gene essentiality <br />    (ii-d) Paralogy. Duplicated genes organism           (42). The network statistics calculated using tYNA <br /> often referred paralogs. Paralogs typically          (http://tyna.gersteinlab.org/). <br /> similar function arose gene duplication <br /> events species. It expected essential     Feature evaluation selections <br /> genes fewer number paralogs than <br /> non-essential genes essential gene&#8217;s function          We used criteria described &#8216;Results&#8217; section <br /> is indispensable likely compensated         select suitable features. <br /> paralogs (14,20). An all-against-all FASTA search                To measure predictive power different features, <br /> conducted set ORFs target                 performed Na&#305;&#168; ve Bayes analysis ranked all <br /> organism (PA, EC, AB BS) identify paralogs             features according coverage length log-odds <br /> with E-value threshold 10 20.                               ratio (Supplementary Figure S1). The longer overall <br />    (ii-e) Domain enrichment. In contrast evolution-        coverage length is, greater contribution cor- <br /> ary conservation gene (ii-c), domain enrichment           responding feature target <span id='am-101' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-102' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span>, i.e. gene essen- <br /> re&#64258;ects conservation local sequences           tiality. Since interested predicting essential genes <br /> the entire gene. For individual domain, collected         non-essential genes, features positive <br /> its occurrence organism (PA, EC, AB BS)               coverage length considered useful features. <br /> using Pfam database (http://pfam.sanger.ac.uk).                  The log-odds ratio calculated feature <br /> Then estimated domain enrichment score accord-             compared features nomograms. <br /> ing ratio occurrence frequencies essential      Comparing span attribute <span id='am-119' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-120'  ></span><span id='am-121' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-122' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>axis</span> nomograms <br /> gene sets total genes target organism:             easily identi&#65533;?es important attributes. The effects of <br /> DES &#188; &#240;ness =Ness &#222;=&#240;ness =Ness+nnon ess =Nnon ess &#222;, ness   attribute value clearly represented a <br /> and nnon ess represent domain&#8217;s occurrence frequency            nomogram, making easy spot direction magni- <br /> in essential non-essential data set, respectively.        tude in&#64258;uence. Attribute axis aligned to <br /> Ness Nnon ess represent size essential         zero-point in&#64258;uence (prior probability), allows <br /> non-essential dataset, respectively. For round            straightforward comparison contributions <br /> training-testing, re-computed DES scores based             different values attributes (Supplementary <br /> on available data testing data. For example,           Figure S1). <br /> in EC ! PA, given domain PA, DES                         For feature, ranked features according to <br /> computed calculating ratio essential                 positive in&#64258;uence gene essentiality. Those with <br /> non-essential genes EC genes encode           high positive in&#64258;uence monotonic relationship with <br /> domain. Therefore, did use information regard-         essentiality candidate features. Speci&#65533;?cally, in <br /> ing PA essential genes, ensuring correct training           feature Category (A), kept CBI, CAI, Fop, Nc, <br /> testing.                                                          L_aa, Aromo removed GC, C3s, A3s, T3s, Gravy <br />    (iii-a) Fluctuation gene-expression. The mRNA ex-           G3s. In Category (B), kept DES, PHYS, <br /> pression levels essential genes vary, average,        Cytoplasm, Extracellular, PA Inner Membrane and <br />                                                                                         Nucleic Acids Research, 2011, Vol. 39, No. 3 799 <br />  <br />  <br /> removed Periplasm, Outer Membrane, ExpAA, First60                            organisms, chose perform analysis pair of <br /> and PredHel. In Category (C), kept CEB, CEH                           relatively distantly related organisms: E. coli (EC) A. <br /> FLU.                                                                         baylyi (AB). The reasons select organism pair are: <br />   Next, considered prior biological information to <br /> remove feature redundancy. For example, CBI                                (i) Essential genes well-characterized organ- <br /> Pearson correlation coef&#65533;?cients 0.92 0.99                              isms. Large-scale gene-knockout experiments have <br /> CAI Fop, respectively (Supplementary Figure S2),                               identi&#65533;?ed 302 (or 7%) essential genes E. coli <br /> because features derived                                    K-12 total 4289 genes (17). In <br /> codon usage gene share similar biological                                 A. baylyi ADP1, total 3308 genes, <br /> meanings. Therefore, removed CBI Fop                                   499 (or 15%) essential genes identi&#65533;?ed <br /> Category (A) resulted 13 candidate features                           large-scale gene-knockout experiments (25). <br /> (Table 2).                                                                         Both mutagenesis experiments performed <br />                                                                                    aerobic conditions, on <br /> Training testing sets preparation                                              standard laboratory rich (LB) media latter <br /> The training data included attribute values                           minimal medium supplemented succinate. <br /> feature class assignments. Each gene assigned                     (ii) Both AB EC g-<span id='am-1' about='obo:NCBITaxon_1224' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-2' property="oboInOwl:hasExactSynonym" content="purple bacteria" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="purple photosynthetic bacteria" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="Proteobacteria" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="proteobacteria" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="purple non-sulfur bacteria" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="purple photosynthetic bacteria and relatives" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="PMID:11542017" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="PMID:16280474" datatype="xsd:string"></span><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="purple bacteria and relatives" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="PMID:11321122" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="PMID:11837318" datatype="xsd:string"></span>proteobacteria</span> taxonomy; <br /> a Boolean value regarding essentiality (1&#8212;essential;                           however, closely related&#8212;the time <br /> 0&#8212;non-essential). The feature values Boolean                             frame divergence estimated 50&#8211;200 <br /> real depending type individual features                                  myrs (47&#8211;48) (Supplementary Figure S3). Because <br /> (Supplementary Table S1). The training data                                   approach useful proven ap- <br /> divided 10 equal parts. Nine-folds used                               plicable distantly related species, pair of <br /> train classi&#65533;?ers remaining fold used                           species provides excellent testing ground for <br /> for testing. The control training set generated                             examining accuracy coverage our <br /> randomly assigning essential labels E. coli genes.                          approach. <br /> The number random &#8216;essential genes&#8217;                              If hypothesis true, rationalize the <br /> number true essential genes used training                      genomic features trained tested using EC essential <br /> and testing frame. <br />                                                                              genes able produce reliable predictions of <br /> Classi&#65533;?er design                                                             essential genes AB, vice versa. To provide an <br />                                                                              objective assessment accuracy coverage the <br /> We used classi&#65533;?ers train test model:                         predictions, known annotations gene essentiality in <br /> (i) Na&#305;&#168; ve Bayes classi&#65533;?er; (ii) logistical regression                    target organism used evaluation <br /> model; (iii) C4.5 decision tree; (iv) CN2 rule. Each                   stage. <br /> classi&#65533;?er scheme independently generates separate                             We &#65533;?rst used reciprocal best hit (RBH) method to <br /> probability score gene essentiality. The performances                     compare genomes organisms (see <br /> of classi&#65533;?ers different complementary. The                     &#8216;Materials Methods&#8217; section) (18). Between EC and <br /> best performance obtained combining outputs                       AB, 1198 orthologs. This represents 28 36% <br /> of diverse classi&#65533;?ers using unweighted average                      EC AB genomes, respectively (Figure 1). <br /> approach. All classi&#65533;?ers implemented using                             We examined overlaps two <br /> Orange software package (http://www.ailab.si/orange/).                       essential gene datasets based identifying orthologs <br />                                                                              (Figure 1). There 195 essential genes common <br /> RESULTS                                                                      EC AB essential datasets, making up <br />                                                                              65 39% essential gene sets, respectively. <br /> Comparing genomes essential genes E. coli                         It clear pathogens substantial portion <br /> and A. baylyi                                                                unique essential genes, consistent previous report <br /> In <span id='am-127' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-128' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-129' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> test hypothesis essential gene annota-                  bacterial species share limited number common <br /> tions transferred distantly related                           essential genes (12). <br />  <br />  <br />  <br /> Table 2. Thirteen features selected 10-fold cross-validation EC <br />  <br />                                       Intrinsic features                                             Context-dependent features <br />                                                                                                      (From functional genomics experiment) <br /> Sequence based                              Sequence derived <br />  <br /> Codon bias index (CBI)                      Domain enrichment score (DES)                            Fluctuation gene expression (FLU) <br /> Hydrophobicity score (Nc)                   Phylogenetic score (PHYS)                                Co-expression network bottlenecks (CEB) <br /> Length Amino Acid (L_aa)                 Subcellular localization: cytoplasm (Cyto)               Co-expression network hubs (CEH) <br /> Aromaticity (Aromo)                         Subcellular localization: extracellular (Extra) <br />                                             Paralogy (PA) <br />                                             Subcellular localization: inner membrane (Inner) <br />  800 Nucleic Acids Research, 2011, Vol. 39, No. 3 <br />  <br />  <br />                                                                         gene essentiality EC relatively weak <br />                                                                         correlations (Table 2 Figure 2). <br />                                                                         Interestingly, features represent different aspects <br />                                                                         sequence function. These diverse aspects the <br />                                                                         correlated features suggest gene essentiality likely <br />                                                                         determined solely genomic sequence gene, <br />                                                                         multiple aspects biology. Among 13 features, <br />                                                                         strongest turns DES (domain enrichment in <br />                                                                         essential genes), considered by <br />                                                                         previous studies. The strongest features are <br />                                                                         CBI, Nc, PHYS L_aa, consistent previous <br />                                                                         studies (18,21&#8211;22) (Table 1). <br />  <br /> Figure 1. Comparison genomes essential genes EC AB.       Cross-validations classi&#65533;?er using E. coli <br /> The square represents 4289 EC total genes; rectangle represents     essential gene set <br /> 3308 AB total genes. The overlap represents 1198 orthologs <br /> determined RBH method. The rectangle dashed border          The 13 selected features (Table 2) used input <br /> represents total 302 EC essential genes. The rectangle         variables classi&#65533;?ers: Na&#305;&#168; ve Bayes, logistical regres- <br /> diagonal brick shades represents total 499 AB essential genes. <br /> The rectangle dashed border diagonal brick          sion, decision tree CN2 rule. The input classi- <br /> shades represents common essential genes species. The       &#65533;?ers contained features gene class <br /> area rectangle approximately proportional number      labels used training data. Each classi&#65533;?er <br /> of genes represents.                                                 scheme independently generated probability score of <br />                                                                         gene essentiality. The best performance obtained by <br />                                                                         combining output probability scores diverse <br /> Selecting suitable features predicting gene essentiality            classi&#65533;?ers using unweighted approach was <br /> It increasingly apparent genomic se-                   used &#65533;?nal prediction. <br /> quences represent aspect complex genetic                   The 10-fold cross-validation result shown ROC <br /> relationships evolved diverse selection                 curve indicated that, level 1% FPR, classi&#65533;?er <br /> pressures (49); therefore, necessary consider                achieved 45% TPR (Figure 3A). The area curve <br /> variety features, including intrinsic                       (AUC) score classi&#65533;?er 0.93 positive pre- <br /> context-dependent features. We used criteria                   dictive value (PPV precision) 0.70 probabil- <br /> select suitable features.                                      ity threshold set 0.5. Our classi&#65533;?er represented a <br />    First, features easily obtained avail-             signi&#65533;?cant improvement classi&#65533;?er integrates <br /> able microorganisms. Intrinsic features                  sequence features S. cerevisiae (22) (AUC = 0.70). <br /> easily extracted long microorganism com-                  Our results outperformed recent study E. coli by <br /> pletely sequenced genome. For context-dependent                         considerable margin (21) (AUC = 0.70). <br /> features, considered gene expression pro&#65533;?les                   Because imbalanced training dataset (essen- <br /> measured microarray experiments. We did include                  tial:non-essential = 1:13), avoid making excessive false <br /> functional annotations, pathway protein&#8211;protein inter-               positive predictions, slightly higher cost assigned <br /> actions (PPIs) method                     false positives. This equivalent raising the <br /> unavailable under-studied organisms.                                 probability threshold predictions yields <br />    Second, features high predictive power               fewer false positives. At probability threshold set at <br /> of gene essentiality. To quantify predictive power               0.75, precision predictions increased 14% to <br /> each feature, performed Na&#305;&#168; ve Bayes analysis                 0.80 (108/135) (Figure 3A). <br /> ranked features according coverage length                    The control training set generated randomly as- <br /> log-odds ratio (Supplementary Figure S1).                               signing essential labels E. coli genes (50). The same <br />    Third, features minimize biological redun-                number random &#8216;essential genes&#8217; assigned the <br /> dancy. Biologically redundant features derived                original training testing sets. The performance the <br /> from similar source high correlations                   classi&#65533;?ers random set signi&#65533;?cantly lower than <br /> each other. For example, CBI Pearson correlation                    using real training set (Supplementary Figure S4). <br /> coef&#65533;?cients 0.92 0.99 CAI Fop, respectively             This suggested method learning the <br /> (Supplementary Figure S2), features               features characteristic gene essentiality. <br /> derived codon usage gene similar <br />                                                                         Predicting AB essential genes integrating intrinsic and <br /> biological meanings. Including redundant features <br />                                                                         context-dependent features <br /> will create problems types classi&#65533;?ers, <br /> e.g. Na&#305;&#168; ve Bayes classi&#65533;?er, increase complexity          After 10-fold cross-validation known essential <br /> without necessarily increasing inferential predictive           genes EC, applied classi&#65533;?er predict AB essen- <br /> power classi&#65533;?er (Supplementary Table S2).                        tial genes, denoted EC ! AB. <br />    Using criteria, total 28 charac-                   A. baylyi Gram-negative bacterium commonly <br /> teristic features considered (Supplementary                     aquatic soil environments. It belongs to <br /> Table S1), identi&#65533;?ed 13 potentially associated               class g-proteobacteria EC (Supplementary <br />                                                                                         Nucleic Acids Research, 2011, Vol. 39, No. 3 801 <br />  <br />  <br />  <br />  <br /> Figure 2. The Nomogram visualization 13 selected features. Each feature corresponding line indicating relationship a <br /> feature value predictive contribution assessed Na&#305;&#168; ve Bayes analysis. The number line value feature value <br /> corresponds point score above. The longer line is, predictive power feature prediction. <br />  <br />  <br /> Figure S3). A set 499 AB essential genes                        highest scores AB, 172 true essential genes. <br /> identi&#65533;?ed targeted mutagenesis (25). Because                         The prediction accuracy excellent considering a <br /> targeted mutagenesis performed minimal media,                        random selection 212 AB genes contain only <br /> many genes involved biosynthesis essential com-                   32 essential genes. <br /> pounds essential required                        We performed reciprocal prediction EC essen- <br /> under rich media. This difference media results                     tial genes using AB essential gene data set, denoted as <br /> substantially higher percentage essential genes AB                    AB ! EC. The prediction yielded ROC curve an <br /> (16%) compared EC (7%).                                                  AUC score 0.89 PPV 0.43 (Figure 3C D). <br />    When applying classi&#65533;?er organisms, dis-                   We speculated lower precision the <br /> tributions raw feature compared &#65533;?rst                        AB data set contained  100 genes associated biosyn- <br /> ensure comparable organisms. Between                       thesis function (e.g. amino acids, cofactors) are <br /> a pair organisms, feature suf&#65533;?ciently                    needed survival minimal media (25). <br /> different follow completely different distribu-                   Inclusion training set genes essential <br /> tions normalized range                             minimal media led classi&#65533;?er learn <br /> (Supplementary Figure S5). We quanti&#65533;?ed similarity                      characteristics unique genes, resulting a <br /> of distributions overlapping area. Two distri-                 poorer classi&#65533;?cation &#8216;true&#8217; essential genes. To test <br /> butions overlapping area &gt;0.5 considered                     re&#65533;?ned AB training set lead to <br /> similar. As result, subset 10 features (Supplementary                increased precision prediction, removed 82 genes <br /> Table S2A) similar distribution EC AB                    associated biosynthesis function AB essen- <br /> was compiled 3308 AB ORFs,                          tial gene set. The re&#65533;?ned data set achieved substantially <br /> classi&#65533;?er trained EC applied dataset.                   better precision (PPV = 0.53) predicting EC essential <br /> The accuracy evaluated examining agreement                       genes, obvious 10% predictions <br /> with assignments gene knockout experiments                     (Supplementary Figure S6). <br /> in AB. <br />    At level 1% FPR, result indicated                    Prediction essential genes E. coli and <br /> classi&#65533;?er achieved 28% TPR (Figure 3B). The AUC                           P. aeruginosa <br /> score 0.80 PPV 0.81 threshold 0.5.                      To transferability essential genes not <br /> That is, 212 predictions received                        limited EC AB, important extend the <br />  802 Nucleic Acids Research, 2011, Vol. 39, No. 3 <br />  <br />  <br />  <br />  <br /> Figure 3. ROC curves plot TPR versus FPR different thresholds classi&#65533;?er probability output. (A) (B): EC ! AB; (C) (D): AB ! <br /> EC. (A) Ten-fold cross-validations EC essential gene data set. (B) Predictions AB essential genes. The classi&#65533;?er trained EC dataset <br /> and evaluated AB essential genes. (C) Ten-fold cross-validations AB essential gene data set. (D) Predictions EC essential genes. The <br /> classi&#65533;?er trained AB data set evaluated EC essential genes. <br />  <br />  <br />  <br /> analysis pairs organisms. We conducted                     learning features EC, denoted EC ! PA, <br /> predictions EC P. aeruginosa PAO1 (PA).                         generated ROC curve AUC score 0.69 <br />    PA ubiquitous opportunistic pathogen capable                    PPV = 0.57 (Supplementary Figure S8A B). The <br /> of causing chronic infection lungs cystic &#65533;?brosis                 reciprocal PA 678 ! EC prediction showed similar <br /> patients. It Gram-negative belongs class                 pattern decreased accuracy (AUC = 0.79 and <br /> of g-proteobacteria EC AB (Supplementary                             PPV = 0.41). <br /> Figure S3). A set 678 PA essential genes, 12%                         The relatively lower accuracy transferring essentiality <br /> its total genes, identi&#65533;?ed transposon mutagen-                  EC PA likely lower quality (i.e. <br /> esis (24). Due random nature transposon inser-                    errors) associated essential gene set <br /> tion events, results transposon mutagenesis                    produced transposon mutagenesis (52). A subset of <br /> contain systematic bias. For example, essential genes                   335 genes, consensus PAO1 another <br /> determined transposon mutagenesis contain dispro-                      Pseudomonas species PA14 essential gene sets (51), is <br /> portionately higher percentage short proteins                    believed accurate. We trained classi- <br /> shorter proteins likely missed trans-                     &#65533;?er 335 PA essential genes predicted in <br /> posons (24,51). Comparison orthologs essential                       EC, denoted PA 335 ! EC. Training re&#65533;?ned data <br /> genes EC PA shown (Supplementary                          set achieved improved performance PA 678 ! EC <br /> Figure S7A).                                                                (AUC = 0.82 PPV = 0.47). <br />    Using feature selection strategy we <br /> employed predict essential genes EC AB,                      Prediction essential genes E. coli and <br /> we identi&#65533;?ed set features EC (Supplementary                    B. subtilis <br /> Table S2B). Note different used                    To explore limit transferability, at- <br /> in EC ! AB (Supplementary Table S2A). We used                          tempted predict essential genes B. subtilis (BS). <br /> the method predict essential genes PA                         Unlike EC, PA AB, BS Gram-positive bacteria. <br />                                                                                     Nucleic Acids Research, 2011, Vol. 39, No. 3 803 <br />  <br />  <br /> The evolutionary distance EC BS substan-                 EC!BS likely lower quality the <br /> tially farther species: estimated              dataset greater evolutionary distance we <br /> around 3000 myrs (Supplementary Figure S3). Among                   mentioned previous sections. <br /> 271 essential genes listed (26), included 192 were <br /> determined experimental techniques disregarded                   Integrative genomics signi&#65533;?cantly improves accuracy <br /> 79 genes predicted homology mapping                        coverage compared homology mapping <br /> from bacteria, E. coli. A comparison                    In order illustrate substantial improvement in <br /> orthologs essential genes EC BS                      coverage method, &#65533;?rst used homology <br /> shown (Supplementary Figure S7B).                                    mapping transfer essential gene annotations EC <br />    Using strategy described previous sections,                AB. Among 302 known essential genes EC, 234 <br /> applied methods transfer essential gene annotations              genes directly mapped AB genes using an <br /> from EC BS, denoted EC ! BS, compared                        RBH approach. Therefore, corresponding 234 <br /> predictions available known essential gene                     orthologous genes AB predicted RBH be <br /> dataset BS. The prediction BS generated ROC                     essential. Among 234 predictions, 195 true es- <br /> curve AUC score 0.80 PPV 0.54                            sential determined AB essential gene dataset <br /> (Supplementary Figures S8C D). Similarly, recip-                (Figure 5). Please note, 234 orthologs the <br /> rocal BS ! EC prediction yielded ROC curve                    maximal number predictions homology mapping can <br /> AUC score 0.86 PPV 0.48.                                         make, given de&#65533;?nition orthologs. <br />    The results suggest despite long evolutionary                  We selected appropriate cutoffs our <br /> distance BS EC, common charac-                    method number predictions the <br /> teristics underlying EC BS essential genes represented              number essential genes target organism, i.e. 499 <br /> by features recognized machine                 AB. Compared homology mapping, the <br /> learning approach.                                                      195 genes correctly predicted homology mapping, our <br />    Although AUC score ROC curve provides                     approach predicted 189 (97%) true essential. On <br /> useful estimation predictive accuracy models,             hand, approach predicted 77 unique predic- <br /> it weights false positive false-negative errors                 tions homology mapping. <br /> equally (53). Because ratio essential genes                  We used following examples illustrate <br /> genome different organisms, e.g. 4% BS                      discrepancies method homology <br /> and 16% AB, evaluating transferability                      mapping (Table 3). For example, ACIAD0822 was <br /> from EC target organisms,                    determined essential targeted mutagenesis <br /> directly compare AUC scores. In addition, pre-                 (25) prediction. This gene annotated <br /> dicting essential genes unstudied organism, precision                   function       aspartyl/glutamyl-tRNA <br /> of prediction (PPV), genes predicted                 amidotransferase ortholog EC (54). Its closest <br /> essential essential, useful.                   homolog EC b1394 involved fatty acid metabolic <br /> Therefore, plotted precision predictions                  process (GO:0006631), different of <br /> from EC target organisms (Figure 4). The                   ACIAD0822. In addition, b1394 non-essential gene. <br /> result clearly indicated EC!AB achieved                        In case, homology mapping unable predict <br /> highest precision, lower precisions EC!PA                  ACIAD0822 essential. In contrast, integrated <br />                                                                         effect strong features (PHYS, PA, CAI and <br />  <br />  <br />  <br />  <br /> Figure 4. Precision predictions EC target organisms.   Figure 5. The integrative approach signi&#65533;?cantly extends coverage <br /> The Precision versus Rank plot pairs bacteria: EC!AB   homology mapping. IG stands integrative approach. RBH <br /> (Gray solid), EC!PA (Gray dashed) EC!BS (Black solid). The          stands reciprocal best hit approach. For IG method, the <br /> little cross curve represents precision PPV      cutoffs set the number essential genes each <br /> corresponding probability threshold set 0.5.                         organism, i.e. (PA: 678, AB: 499, BS: 192). <br />  804 Nucleic Acids Research, 2011, Vol. 39, No. 3 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                          Non&#8211;ess (&#8211;142) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                           The number parenthesis indicates normalized log-odds ratio (points). The larger number indicates higher correlation essentiality. Shaded features determine the <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              Cyto) enabled method correctly assign gene as <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              essential. In example, ACIAD2634 was <br />  <br />                                                                     Prediction <br />  <br />                                                                                           Ess (15) <br />  <br />                                                                                                                                                                                                                                                                          Ess (81) <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              determined essential targeted mutagenesis <br />  <br />  <br />  <br />  <br />                                                                                                                                                                0.17 (&#8211;25) 0.32 (&#8211;15) 0.21 (3) 0.61 (&#8211;15) 0.08 (&#8211;10) 0.24 (&#8211;29) 0.31 (&#8211;30) 0.44 (&#8211;5) 0.05 (&#8211;3) 0.22 (&#8211;10) (&#8211;139) <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              (25) prediction. Its closest homology EC is <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              b2499, non-essential gene function. In <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              case, RBH incorrectly predicted non-essential. <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                               1 (&#8211;20) <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              However, combined in&#64258;uence DES, PHYS, Nc <br />                                                                                           0 (&#8211;3) <br />  <br />                                                                                                                                                                                                                                                               0 (&#8211;3) <br />                                                                     Inner <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              CAI allowed method successfully override <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              incorrect assignment RBH. These examples <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              highlighted power integrative approach. <br />                                                                                           0.13 (&#8211;7) 0.37 (&#8211;5) 0.24 (&#8211;28) 1.00 (20) 0 (&#8211;4) <br />  <br />                                                                                                                                                                0.74 (58) 1.00 (45) 0 (10) 0.39 (20) 0.09 (&#8211;9) 0.40 (4) 0.19 (&#8211;15) 0 (&#8211;25) 0 (&#8211;4) <br />  <br />  <br />                                                                                                                                                                0.43 (3) 0.67 (22) 1 (&#8211;40) 0.56 (&#8211;10) 0.05 (&#8211;12) 0.25 (&#8211;28) 0.24 (&#8211;28) 0 (&#8211;25) 0 (&#8211;4) <br />                                                                     Extra <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 On hand, ACIAD2640 determined an <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              essential gene targeted mutagenesis (25) the <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              RBH approach, method incorrectly predicted it <br />                                                                     Cyto <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              non-essential. The main reason failed predict as <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              essential gene paralog AB, resulted in <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              strongly unfavorable PA score. Its predicted subcellular <br />                                                                     Aromo <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              localization cemented incorrect assignment. <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 Our predictions PA BS suggested similar <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              conclusion (Figure 5). The RBH method sometimes <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              considered stringent mapping orthologs (55), <br />                                                                     CAI <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              forward BLAST yields paralogous best <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              hit, regardless reciprocal BLAST corrects <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              error recovering actual ortholog, pairs will <br />                                                                     L_aa <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              excluded. Therefore, performed homology <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              mapping using COG groups. Using alternative <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              method homology mapping did change our <br />                                                                                           0.49 (4) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              conclusions (Supplementary Figure S9). <br />                                                                     Nc <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 A simple way increase coverage homology <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              mapping include homologs orthologs <br />                                                                                           0.67 (22) 0 (10) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              choosing relaxed E-value cutoff. By gradually <br />                                                                     PA <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              loosening E-value thresholds, compared our <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              approach RBH method ROC curves <br />                                                                     PHYS <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              (Supplementary Figure S10). It clear our <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              method dominated RBH method entire range <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              ROC curves. <br />                                                                     DES <br />  <br />                                                                                           NA <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              DISCUSSIONS <br />                                                                                                                                               membrane-speci&#65533;?c <br />                                                                                           acyl-coA hydratase <br />  <br />  <br />                                                                                                                                               aminoimidazole <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              By taking advantage abundant genomic sequences <br />                                                                                                                                             Phosphoribosyl <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              functional genomics data available bacterial <br />                                                                                                                                               transporter <br />                                                                                                                                               lipoprotein <br />                                                                                                                                               synthetase <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              species, EC, PA, AB BS, developed a <br />                                                                                                                                               subunit <br />                                                                     Function <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              machine learning-based approach predicts essential <br />                                                                                                                                             Outer <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              genes integrating features potentially associated with <br />            Table 3. Examples correct incorrect predictions <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              gene essentiality Prokaryotes. Although essential gene <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              data sets available genomes (56), <br />                                                                     EC gene ID <br />  <br />  <br />                                                                                             (Non-ess) <br />  <br />                                                                                             (Non-ess) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              determined transposon mutagenesis <br />                                                                                                                                                                                              (Ess) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              results contain systematic biases (24,52). To <br />                                                                                           b1394 <br />  <br />                                                                                           b2499 <br />  <br />  <br />                                                                                                                                                                                            b1117 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              strike balance comprehensiveness validity <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              analysis, chose include bacterial <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              species (EC, AB BS) essential gene were <br />                                                                                                                                                                                                                                                                                           prediction outcome case. <br />                                                                                                        aminoimidazole <br />                                                                                                        glutamyl-tRNA <br />  <br />  <br />  <br />                                                                                           ACIAD 2640 ABC <span id='am-202' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-203' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-204' property="rdfs:label" content="superfamily" datatype="xsd:string"></span>superfamily</span> <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              determined targeted mutagenesis, considered <br />                                                                                           ACIAD 2634 Phosphoribosyl <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              highest quality, (PA) essential genes <br />                                                                                                        synthetase <br />  <br />                                                                                                        atp_bind <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              determined transposon mutagenesis inde- <br />                                                                                           ACIAD 0822 Aspartyl/ <br />                                                                     AB Gene ID Function <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              pendent groups. <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 Our 10-fold cross-validations organisms showed <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              AUC scores  0.9, suggesting gene essentiality, albeit <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              complex property highly predictable learning the <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              characteristics underlying gene essentiality. We believe this <br />                                                                                                                                                                                                                                                                           Average <br />                                                                                            (E ss) <br />                                                                                            (Ess) <br />  <br />  <br />  <br />  <br />                                                                                            (Ess) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              best cross-validation result organism to <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              date predicting essential genes. We attributed this <br />                                                                             Nucleic Acids Research, 2011, Vol. 39, No. 3 805 <br />  <br />  <br /> signi&#65533;?cant improvement previous studies                  corrected studying role feature plays <br /> incorporating intrinsic context-dependent               determining essentiality adjusting relative <br /> features. In particular, discovered domain enrichment,        weights. <br /> which considered previous studies,           Our study signi&#65533;?cant &#65533;?rst report <br /> strongest feature.                                               gene essentiality reliably transferred between <br />    Our results protein domain enrichment          distantly related organisms using machine learning- <br /> stronger feature predicting gene essentiality            based approach. When using method transfer <br /> orthologs (PHYS) suggests gene essentiality likely       essentiality distantly related organisms, the <br /> preserved function protein domains             accuracy predicting essential genes affected by <br /> domain combinations instead conservation          following factors: <br /> of entire genes. This unexpected reasonable              First, essential gene data set classi&#65533;?ers <br /> because numerous examples literature             trained high quality. Errors the <br /> that supports postulation. For example, DNA poly-           training dataset signi&#65533;?cantly reduce accuracy of <br /> merase III subunits t g domain III (PF12169) (57)            predictions, observed PA ! EC. <br /> only single copy EC, PA BS. The host gene              Second, essentiality transferred the <br /> of domain, dnaX, essential species;         highly similar growth conditions. Gene essential- <br /> however, sequence identity low. To             ity likely contextual property (49). Organisms are <br /> further support modularity essential genes,           likely use different sets essential genes different <br /> previous studies discovered              conditions. Predicting essential genes different con- <br /> genes essential, domains              ditions training set likely result in <br /> required essential function. For example, E. coli        decreased predictive accuracy, observed EC <br /> ftsK (b0890) essential gene consisting              AB. However, recent study E. coli conditional <br /> domains: N-terminal (amino acid 1&#8211;780) C-terminal            essential genes showed &lt;20% total essential <br /> (amino acid 780&#8211;1329) domains. Only N-terminal               genes different glycolysis glucose metab- <br /> domain gene required role cell             olisms (59). Therefore, algorithm useful in <br /> division viability (58).                                     capturing majority essential genes target <br />    Another reason DES predictive orthologs          organism growth conditions different, <br /> is protein domains transferable or- <br />                                                                  best performance achieved the <br /> ganisms orthologs. For example, data showed <br />                                                                  highly similar growth conditions. <br /> that EC PA share &lt;35% genes orthologs, <br />                                                                     Third, evolutionary distance play an <br /> but share 70% domains. A list domains <br />                                                                  important role accuracy predictions. It is <br /> that highest lowest DES given in <br />                                                                  encouraging classi&#65533;?er transfer gene es- <br /> Supplementary Table S4. Identifying exploring such <br />                                                                  sentiality Gram-negative Gram-positive <br /> domains actually responsible carrying the <br /> essential function, &#8216;essential domains&#8217;, greatly         bacteria, accuracy transferring lower <br /> improve understanding mechanistic basis            gram-negative bacteria. An interesting <br /> gene essentiality.                                               future direction investigate what <br />    Other dominant features DES intrin-        extent method applicable. For example, to <br /> sic features strongest context-dependent feature       extent essential genes transferred between <br /> CEB ranks seventh 13 features. This suggests           Prokaryotes Eukaryotes? <br /> that gene&#8217;s essentiality primarily determined bio-      Fourth, prediction depends availability <br /> chemical functions, functions es-        features similar distribution organisms. <br /> sential in&#64258;uenced bacteria&#8217;s growth           To useful prediction, features a <br /> conditions. For instance, rewiring gene regula-       similar distribution organisms order allow <br /> tory networks different conditions alter           accurate training testing (Supplementary Figure S5). <br /> degree gene&#8217;s essentiality organism. Further            The comparison method homology <br /> investigations in&#64258;uence context-dependent        mapping highlights limitations homology <br /> features shed light conditional gene essentiality.    mapping. Homology mapping useful closely <br />    Most errors current method in&#64258;uenced        related organisms, S. cerevisiae S. mikatae <br /> by strong features. For example, false positive predictions      (22). However, divergent organisms, severely <br /> often high Phylogenetic score (PHYS). Although              limited number conserved orthologs. In contrast, <br /> essential genes tend evolutionarily conserved         approach does limitation the <br /> than non-essential genes, quarter highly      prediction based features computed <br /> conserved bacterial genes essential tested             genes; therefore, easily explore gene space <br /> experimentally model bacteria (9&#8211;12). Therefore,              homology mapping inapplicable of. <br /> assigning excessive weight gene conservation             Another advantage approach can <br /> inevitably create false positive errors. On hand,      incorporate organism-speci&#65533;?c context-dependent <br /> false negative errors, i.e. essential genes incor-     features, e.g. gene expression number paralogs <br /> rectly predicted non-essential, result       target organism, potentially identify one <br /> combined effect features failing override        organism essential genes orthologs non-essential <br /> decisions strong features. These errors           organisms, e.g. ACIAD2634 Table 3. <br />  806 Nucleic Acids Research, 2011, Vol. 39, No. 3 <br />  <br />  <br />    Our method easily extended predict essential                    alcohol dehydrogenase genes. Appl. Microbiol. Biotechnol., 85, <br /> genes unstudied organism. The genomic sequences                     651&#8211;657. <br />                                                                           7. Pucci,M.J. (2006) Use genomics select antibacterial targets. <br /> the genes functional genomics data microarray                       Biochem. Pharmacol., 71, 1066&#8211;1072. <br /> gene-expression analysis available                        8. Bruccoleri,R.E., Dougherty,T.J. Davison,D.B. (1998) <br /> organism whole-genome mutagenesis experi-                         Concordance analysis microbial genomes. Nucleic Acids Res., <br /> ments carried out. As result, essentiality predic-                 26, 4482&#8211;4486. <br />                                                                           9. Arigoni,F., Talabot,F., Peitsch,M., Edgerton,M.D., Meldrum,E., <br /> tion prior costly whole-genome                            Allet,E., Fish,R., Jamotte,T., Curchod,M.L. Loferer,H. (1998) <br /> screening using mutagenesis experiments.                                     A genome-based approach identi&#65533;?cation essential <br />    In summary, integrating features available                      bacterial genes. Nat. Biotechnol., 16, 851&#8211;856. <br /> genes, method provides valuable alternative pre-               10. Freiberg,C., Wieland,B., Spaltmann,F., Ehlert,K., Brotz,H. and <br /> dicting essential genes orthologs. The application                    Labischinski,H. (2001) Identi&#65533;?cation novel essential <span id='am-195' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-196' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-197'  ></span><span id='am-198' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-199' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-201' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia</span> <br />                                                                              coli genes conserved pathogenic bacteria. J. Mol. <br /> of approach bacterial species tremendous po-                      Microbiol. Biotechnol., 3, 483&#8211;489. <br /> tential signi&#65533;?cantly improve ability re-engineer               11. Song,J.H., Ko,K.S., Lee,J.Y., Baek,J.Y., Oh,W.S., Yoon,H.S., <br /> microorganisms respond emergency                       Jeong,J.Y. Chun,J. (2005) Identi&#65533;?cation essential genes in <br /> situations, bioterrorist attack bioremediation                    <span id='am-16' about='obo:NCBITaxon_1301' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-17' property="oboInOwl:hasDbXref" content="PMID:7537076" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Streptococcus" datatype="xsd:string"></span><span id='am-19'  ></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="PMID:10555340" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="PMID:19620365" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="PMID:14657115" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="PMID:8995803" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="PMID:1720654" datatype="xsd:string"></span>Streptococcus</span> pneumoniae allelic replacement mutagenesis. <br /> of oil-spilled Gulf regions. Although research                       Mol. Cells, 19, 365&#8211;374. <br />                                                                          12. Zalacain,M., Biswas,S., Ingraham,K.A., Ambrad,J., Bryant,A., <br /> performed Prokaryotes, highest quality                          Chalker,A.F., Iordanescu,S., Fan,J., Fan,F., Lunsford,R.D. et al. <br /> essential gene datasets available, conclusions                       (2003) A global approach identify novel broad-spectrum <br /> drawn study expected valid                       antibacterial targets proteins unknown function. <br /> other domains life.                                                       J. Mol. Microbiol. Biotechnol., 6, 109&#8211;126. <br />                                                                          13. Winzeler,E.A., Shoemaker,D.D., Astromoff,A., Liang,H., <br />                                                                              Anderson,K., Andre,B., Bangham,R., Benito,R., Boeke,J.D., <br />                                                                              Bussey,H. et al. (1999) Functional characterization S. <br /> SUPPLEMENTARY DATA                                                           cerevisiae genome gene deletion parallel analysis. Science, <br /> Supplementary Data available NAR Online.                              285, 901&#8211;906. <br />                                                                          14. Giaever,G., Chu,A.M., Ni,L., Connelly,C., Riles,L., Veronneau,S., <br />                                                                              Dow,S., Lucau-Danila,A., Anderson,K., Andre,B. et al. (2002) <br />                                                                              Functional pro&#65533;?ling <span id='am-89' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-95'  ></span>Saccharomyces</span> cerevisiae genome. <br /> ACKNOWLEDGEMENTS                                                             Nature, 418, 387&#8211;391. <br /> The authors like thank anonymous review-                  15. Gerdes,S.Y., Scholle,M.D., Campbell,J.W., Balazsi,G., Ravasz,E., <br />                                                                              Daugherty,M.D., Somera,A.L., Kyrpides,N.C., Anderson,I., <br /> ers valuable suggestions. L.D., J.D. L.J.L.                          Gelfand,M.S. et al. (2003) Experimental determination system <br /> designed research, J.D. L.D. implemented                         level analysis essential genes Escherichia coli MG1655. <br /> research. A.A.M. L.W. offered critical suggestions,                      J. Bacteriol., 185, 5673&#8211;5684. <br /> L.D., J.D., L.J.L., M.Z., X.L., S.S. D.J.H.                          16. Baba,T., <span id='am-104' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-105' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-107'  ></span><span id='am-108' property="rdfs:label" content="Ara" datatype="xsd:string"></span>Ara</span>,T., Hasegawa,M., Takai,Y., Okumura,Y., Baba,M., <br /> contributed writing revising article.                             Datsenko,K.A., Tomita,M., Wanner,B.L. Mori,H. (2006) <br />                                                                              Construction <span id='am-232' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-233' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-234' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-235' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-236' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span>Escherichia coli K-12</span> in-frame, single-gene <br />                                                                              knockout mutants: Keio collection. Mol. Syst. Biol., 2, 2006 <br />                                                                              0008. <br /> FUNDING                                                                  17. Kato,J. Hashimoto,M. (2007) Construction consecutive <br /> Cincinnati Children&#8217;s Hospital Medical Center (CCHMC)                        deletions Escherichia coli chromosome. Mol. Syst. Biol., 3, <br />                                                                              132. <br /> Trustee Grant awarded (to L.J.L.); National Institutes                18. Chen,Y. Xu,D. (2005) Understanding protein dispensability <br /> Health grant GM69845 (to D.J.H.). Funding open                           machine-learning analysis high-throughput data. <br /> access charge: CCHMC Trustee Grant.                                          Bioinformatics, 21, 575&#8211;581. <br />                                                                          19. Jeong,H., Mason,S.P., Barabasi,A.L. Oltvai,Z.N. (2001) <br /> Con&#64258;ict statement. None declared.                                Lethality centrality protein networks. Nature, 411, 41&#8211;42. <br />                                                                          20. Saha,S. Heber,S. (2006) In silico prediction yeast deletion <br />                                                                              phenotypes. Genet. Mol. Res., 5, 224&#8211;232. <br /> REFERENCES                                                               21. Gustafson,A.M., Snitkin,E.S., Parker,S.C., DeLisi,C. Kasif,S. <br />                                                                              (2006) Towards identi&#65533;?cation essential genes using targeted <br />  1. Gibson,D.G., Glass,J.I., Lartigue,C., Noskov,V.N., Chuang,R.Y.,          genome sequencing comparative analysis. BMC Genomics, 7, <br />     Algire,M.A., Benders,G.A., Montague,M.G., Ma,L.,                         265. <br />     Moodie,M.M. et al. (2010) Creation bacterial cell controlled    22. Seringhaus,M., Paccanaro,A., Borneman,A., Snyder,M. and <br />     chemically synthesized genome. Science, 329, 52&#8211;56.                 Gerstein,M. (2006) Predicting essential genes fungal genomes. <br />  2. Pennisi,E. (2010) Genomics. Synthetic genome brings new life          Genome Res., 16, 1126&#8211;1135. <br />     bacterium. Science, 328, 958&#8211;959.                                    23. Hashimoto,M., Ichimura,T., Mizoguchi,H., Tanaka,K., <br />  3. Dowell,R.D., Ryan,O., Jansen,A., Cheung,D., Agarwala,S.,                 Fujimitsu,K., Keyamura,K., Ote,T., Yamakawa,T., Yamazaki,Y., <br />     Danford,T., Bernstein,D.A., Rolfe,P.A., Heisler,L.E., Chin,B.            Mori,H. et al. (2005) Cell size nucleoid organization of <br />     et al. (2010) Genotype phenotype: complex problem. Science,         engineered Escherichia coli cells reduced genome. <br />     328, 469.                                                                Mol. Microbiol., 55, 137&#8211;149. <br />  4. Haselbeck,R., Wall,D., Jiang,B., Ketela,T., Zyskind,J., Bussey,H.,   24. Jacobs,M.A., Alwood,A., Thaipisuttikul,I., Spencer,D., <br />     Foulkes,J.G. Roemer,T. (2002) Comprehensive essential gene           Haugen,E., Ernst,S., Will,O., Kaul,R., Raymond,C., Levy,R. <br />     identi&#65533;?cation platform novel anti-infective drug                et al. (2003) Comprehensive transposon mutant library of <br />     discovery. Curr. Pharm. Des., 8, 1155&#8211;1172.                              <span id='am-156' about='obo:NCBITaxon_287' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_286 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_136841 obo:NCBITaxon_135621'><span id='am-157' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224290" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931956" datatype="xsd:string"></span><span id='am-159'  ></span><span id='am-160' property="oboInOwl:hasRelatedSynonym" content="Bacillus aeruginosus" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931958" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931957" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1437770" datatype="xsd:string"></span><span id='am-164' property="rdfs:label" content="Pseudomonas aeruginosa" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasRelatedSynonym" content="Bacterium pyocyaneum" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas polycolor" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasRelatedSynonym" content="Peudomonas aeruginosa" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1437769" datatype="xsd:string"></span><span id='am-169' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931955" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasRelatedSynonym" content="Micrococcus pyocyaneus" datatype="xsd:string"></span><span id='am-173' property="oboInOwl:hasAlternativeId" content="NCBITaxon:932477" datatype="xsd:string"></span><span id='am-174' property="oboInOwl:hasRelatedSynonym" content="Bacillus pyocyaneus" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasRelatedSynonym" content="Bacterium aeruginosum" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas pyocyanea" datatype="xsd:string"></span><span id='am-177' property="oboInOwl:hasRelatedSynonym" content="probable synonym or variety: &quot;Pseudomonas polycolor&quot; Clara 1930" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1437768" datatype="xsd:string"></span>Pseudomonas aeruginosa</span>. Proc. Natl Acad. Sci. USA, 100, <br />  5. Fraser,C.M. (2004) A genomics-based approach biodefence               14339&#8211;14344. <br />     preparedness. Nat. Rev. Genet., 5, 23&#8211;33.                            25. Berardinis,V., Vallenet,D., Castelli,V., Besnard,M., Pinet,A., <br />  6. Atsumi,S., Wu,T.Y., Eckl,E.M., Hawkins,S.D., Buelter,T.              Cruaud,C., Samair,S., Lechaplais,C., Gyapay,G., Richez,C. et al. <br />     Liao,J.C. (2010) Engineering isobutanol biosynthetic pathway         (2008) A complete collection single-gene deletion mutants of <br />     Escherichia coli comparison aldehyde reductase/           <span id='am-216' about='obo:NCBITaxon_202950' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_468 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_469'><span id='am-217' property="rdfs:label" content="Acinetobacter baylyi" datatype="xsd:string"></span><span id='am-218'  ></span><span id='am-219' property="oboInOwl:hasDbXref" content="PMID:12892111" datatype="xsd:string"></span><span id='am-220' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-221' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Acinetobacter baylyi</span> ADP1. Mol. Syst. Biol., 4, 174. <br />                                                                                       Nucleic Acids Research, 2011, Vol. 39, No. 3 807 <br />  <br />  <br /> 26. Kobayashi,K., Ehrlich,S.D., Albertini,A., Amati,G.,                   43. Kim,P.M., Lu,L.J., Xia,Y. Gerstein,M.B. (2006) Relating <br />     Andersen,K.K., Arnaud,M., Asai,K., Ashikaga,S., Aymerich,S.,              three-dimensional structures protein networks provides <br />     Bessieres,P. et al. (2003) Essential Bacillus subtilis genes. Proc.       evolutionary insights. Science, 314, 1938&#8211;1941. <br />     Natl Acad. Sci. USA, 100, 4678&#8211;4683.                                  44. Zhang,M., Deng,J., Fang,C., Zhang,X. Lu,L.J. (2010) <br /> 27. Barrett,T., Troup,D.B., Wilhite,S.E., Ledoux,P., Rudnev,D.,               Biomolecular network analysis applications. In Knowledge- <br />     Evangelista,C., Kim,I.F., Soboleva,A., Tomashevsky,M.                 Based Bioinformatics: From Analysis Interpretation, Vol. 11, <br />     Edgar,R. (2007) NCBI GEO: mining tens millions                      pp. 253&#8211;288. <br />     expression pro&#65533;?les&#8211;database tools update. Nucleic Acids Res.,     45. Jansen,R., Greenbaum,D. Gerstein,M. (2002) Relating <br />     35, D760&#8211;D765.                                                            whole-genome expression data protein-protein interactions. <br /> 28. Parkinson,H., Kapushesky,M., Shojatalab,M.,                               Genome Res., 12, 37&#8211;46. <br />     Abeygunawardena,N., Coulson,R., Farne,A., Holloway,E.,                46. Lu,L.J., Xia,Y., Paccanaro,A., Yu,H. Gerstein,M. (2005) <br />     Kolesnykov,N., Lilja,P., Lukk,M. et al. (2007) ArrayExpress&#8211;a             Assessing limits genomic data integration predicting <br />     public database microarray experiments gene expression             protein networks. Genome Res., 15, 945&#8211;953. <br />     pro&#65533;?les. Nucleic Acids Res., 35, D747&#8211;D750.                           47. Brown,J.R. Volker,C. (2004) Phylogeny of <br /> 29. Gasch,A.P., Spellman,P.T., Kao,C.M., Carmel-Harel,O.,                     gamma-proteobacteria: resolution branch universal <br />     Eisen,M.B., Storz,G., Botstein,D. Brown,P.O. (2000)                   tree? Bioessays, 26, 463&#8211;468. <br />     Genomic expression programs response yeast cells         48. Lerat,E., Daubin,V., Ochman,H. Moran,N.A. (2005) <br />     environmental changes. Mol. Biol. Cell., 11, 4241&#8211;4257.                   Evolutionary origins genomic repertoires bacteria. <br /> 30. Tatusov,R.L., Galperin,M.Y., Natale,D.A. Koonin,E.V.                  PLoS Biol., 3, e130. <br />     (2000) The COG database: tool genome-scale analysis          49. D&#8217;<span id='am-222' about='obo:NCBITaxon_1289997' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_64962 obo:NCBITaxon_74388 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_27818 obo:NCBITaxon_6072'><span id='am-223' property="rdfs:label" content="Elia" datatype="xsd:string"></span><span id='am-224' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-225' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-226'  ></span>Elia</span>,M.A., Pereira,M.P. Brown,E.D. (2009) Are essential <br />     protein functions evolution. Nucleic Acids Res., 28, 33&#8211;36.           genes really essential? Trends Microbiol., 17, 433&#8211;438. <br /> 31. Knight,R.D., Freeland,S.J. Landweber,L.F. (2001) A simple         50. Edgington,E.S. (1980) Randomization tests. M. Dekker, <br />     model based mutation selection explains trends codon            New York. <br />     amino-acid usage GC composition             51. Liberati,N.T., Urbach,J.M., Miyata,S., Lee,D.G., Drenkard,E., <br />     genomes. Genome Biol, 2, RESEARCH0010.                                    Wu,G., Villanueva,J., Wei,T. Ausubel,F.M. (2006) An <br /> 32. Jordan,I.K., Rogozin,I.B., Wolf,Y.I. Koonin,E.V. (2002)               ordered, nonredundant library Pseudomonas aeruginosa strain <br />     Essential genes evolutionarily conserved                PA14 transposon insertion mutants. Proc. Natl Acad. Sci. USA, <br />     nonessential genes bacteria. Genome Res, 12, 962&#8211;968.                  103, 2833&#8211;2838. <br /> 33. Kyte,J. Doolittle,R.F. (1982) A simple method displaying      52. Gerdes,S., Edwards,R., Kubal,M., Fonstein,M., Stevens,R. and <br />     hydropathic character protein. J Mol. Biol., 157,                Osterman,A. (2006) Essential genes metabolic maps. Curr. <br />     105&#8211;132.                                                                  Opin. Biotechnol., 17, 448&#8211;456. <br /> 34. Wright,F. (1990) The &#8216;effective number codons&#8217; used gene.     53. Lobo,J.M., Jimenez-Valverde,A. Real,R. (2008) AUC: a <br />     Gene, 87, 23&#8211;29.                                                          misleading measure performance predictive distribution <br /> 35. Fuglsang,A. (2004) The &#8216;effective number codons&#8217; revisited.            models. Global Ecol. Biogeogr., 17, 145&#8211;151. <br />     Biochem. Biophys. Res. Commun., 317, 957&#8211;964.                         54. Barbe,V., Vallenet,D., Fonknechten,N., Kreimeyer,A., Oztas,S., <br /> 36. Sharp,P.M. Li,W.H. (1987) The codon adaptation index&#8211;a                Labarre,L., Cruveiller,S., Robert,C., Duprat,S., Wincker,P. et al. <br />     measure directional synonymous codon usage bias,               (2004) Unique features revealed genome sequence of <br />     potential applications. Nucleic Acids Res., 15, 1281&#8211;1295.                <span id='am-205' about='obo:NCBITaxon_62977' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_468 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_469'><span id='am-206' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-207'  ></span><span id='am-208' property="oboInOwl:hasAlternativeId" content="NCBITaxon:927641" datatype="xsd:string"></span><span id='am-209' property="oboInOwl:hasDbXref" content="PMID:15514110" datatype="xsd:string"></span><span id='am-210' property="oboInOwl:hasRelatedSynonym" content="Acinetobacter calcoaceticus ADP1" datatype="xsd:string"></span><span id='am-211' property="oboInOwl:hasExactSynonym" content="Acinetobacter baylyi strain ADP1" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-213' property="oboInOwl:hasExactSynonym" content="Acinetobacter baylyi str. ADP1" datatype="xsd:string"></span><span id='am-214' property="oboInOwl:hasRelatedSynonym" content="Acinetobacter baylyi ADP1" datatype="xsd:string"></span><span id='am-215' property="rdfs:label" content="Acinetobacter sp. ADP1" datatype="xsd:string"></span>Acinetobacter sp. ADP1</span>, versatile naturally transformation <br /> 37. Zhang,C.T. Zhang,R. (2008) Gene essentiality analysis based           competent bacterium. Nucleic Acids Res., 32, 5766&#8211;5779. <br />     DEG, database essential genes. Methods Mol. Biol., 416,       55. Wall,D.P., Fraser,H.B. Hirsh,A.E. (2003) Detecting putative <br />     391&#8211;400.                                                                  orthologs. Bioinformatics, 19, 1710&#8211;1711. <br /> 38. Lu,Z., Szafron,D., Greiner,R., Lu,P., Wishart,D.S., Poulin,B.,        56. Hannay,K., Marcotte,E.M. Vogel,C. (2008) Buffering gene <br />     Anvik,J., Macdonell,C. Eisner,R. (2004) Predicting subcellular        duplicates: analysis molecular correlates evolutionary <br />     localization proteins using machine-learned classi&#65533;?ers.                conservation. BMC Genomics, 9, 609. <br />     Bioinformatics, 20, 547&#8211;556.                                          57. Jergic,S., Ozawa,K., Williams,N.K., Su,X.C., Scott,D.D., <br /> 39. Sonnhammer,E.L., von Heijne,G. Krogh,A. (1998) A                      Hamdan,S.M., Crowther,J.A., Otting,G. Dixon,N.E. (2007) <br />     hidden Markov model predicting transmembrane helices               The unstructured C-terminus tau subunit Escherichia <br />     protein sequences. Proc. Int. Conf. Intell. Syst. Mol. Bio <br /> </body></html>