<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Billerbeck Panke Microbial Cell Factories 2012, 11:110 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />  TECHNICAL NOTES                                                                                                                               Open Access <br />  <br /> A genetic replacement selection-based <br /> engineering essential proteins <br /> Sonja Billerbeck Sven Panke* <br />  <br />  <br />   Abstract <br />   Background: Essential genes represent core biological functions required viability. Molecular <br />   understanding essentiality design synthetic cellular systems includes engineering essential <br />   proteins. An impediment effort lack growth-based selection systems suitable directed evolution <br />   approaches. <br />   Results: We established simple strategy genetic replacement essential gene (library of) variant(s) <br />   transformation. <br />   The validated using different essential genes plasmid combinations reproducibly <br />   shows transformation efficiencies <span id='am-58' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-59' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> 107 transformants microgram DNA identifiable <br />   false positives. <span id='am-45' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-47'  ></span><span id='am-48' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> allowed reliable recovery functional variants 105-fold excess of <br />   non-functional variants. This outperformed selection conventional bleach-out strains orders of <br />   magnitude, recombination functional non-functional variants interfered reliable recovery <br />   recA negative strains. <br />   Conclusions: We propose selection extremely suitable evaluating large libraries of <br />   engineered essential proteins resulting reliable isolation functional variants clean strain background <br />   readily used vivo applications expression purification use in vitro studies. <br />  <br />  <br /> Background                                                                           proteins complicated knock-outs cause le- <br /> About percent E. coli genes essential                           thality. This means phenotypes engineered proteins <br /> cell [1]. Essential genes particular scientific                      easily evaluated vivo suitable clean strain <br /> as encode proteins required important biological                            backgrounds available. <br /> functions, building minimal core cellular                                In decade approaches investi- <br /> viability tends conserved <span id='am-55' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-56' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>.                                gated identify essential genes study func- <br /> Knowledge essential genes protein pro-                               tion vivo conditional elimination protein from <br /> ducts important drug design [2,3], biotechnological                           cell. This achieved triggering interference of <br /> applications [4], minimal genome approaches [5-8] and,                            synthesis target protein transcrip- <br /> general, crucial understanding engineering                               tional translational level [10-15]. However, these <br /> basic cellular functions required life [9]. While                            &#8220;bleach-out&#65533;? methods rely conditional protein elimin- <br /> construction Keio-collection, collection single                          ation elimination target gene itself and <br /> gene knock-outs E. coli [1], enormously facilitated                           retain wild-type copy essential gene the <br /> systematic investigation physiology E. coli                        cell. This sets limitations utility systems as <br /> as protein strain-engineering approaches,                            ready-to-use selection systems directed evolution <br /> restricted non-essential genes protein pro-                             experiments recombination library members with <br /> ducts. Engineering approaches involving essential genes                              chromosomal wild-type gene mutations sys- <br />                                                                                      tem regulating expression wild-type protein can <br />                                                                                      lead selection false positive variants. This par- <br /> * Correspondence: sven.panke@bsse.ethz.ch <br /> ETH Z&#252;rich, Department Biosystems Science Engineering (D-BSSE), <br />                                                                                      ticularly true using library small <br /> Mattenstrasse 26, 4058 Basel, Switzerland                                            fraction variants expected functional. In this <br />                                        &#169; 2012 Billerbeck Panke; licensee BioMed Central Ltd. This Open Access article distributed terms the <br />                                        Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, <br />                                        distribution, reproduction medium, provided original work properly cited. <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                      Page 2 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br /> case recombination events preferentially selected     contains 18 bp I-SceI cleavage site allows <br /> functional library members, results selec-      conditional elimination pKOCOMP the <br /> tion effort turning laborious screen bona fide      presence I-SceI. I-SceI expressed helper <br /> functional library members.                                    plasmid pI-SceI control rhamnose- <br />    Besides evaluation large libraries, desire       inducible RhaSR/PrhaBAD-system [18]. A variant this <br /> during protein engineering essential genes re-        helper plasmid, pParaI-SceI, carries arabinose- <br /> place wild-type gene single engineered heter-      inducible regulatory AraC/ParaBAD [19] instead of <br /> ologous variant vivo functional studies      rhamnose-based system. After testing various set- <br /> construction specialized strains used       ups, promoters chosen exhibited high <br /> purify mutant protein free wild-type protein.           expression level presence rhamnose arabin- <br /> Phage P1-mediated transduction chromosomal                ose efficiently switched presence <br /> knock-out strain expressing variant essen-     glucose catabolite repression [19,20]. It should <br /> tial gene plasmid current            noted specific set-up complementation <br /> method choice achieve genetic replacement          vector flexible long carries I-SceI site, <br /> essential target gene variant (e.g [16]). The knock-      easily introduced PCR, compatible <br /> out created complementing                pKD46, required &#955; red-mediated <br /> chromosomal gene loss plasmid-encoded version          knock-out target gene cured growth <br /> the essential gene.                                            42&#176;C. The completed subse- <br />    Although P1-transduction widely used, protocol       quent transformation pI-SceI pParaI-SceI. <br /> is time-consuming restricted variants a <br /> time efficiencies successful transductions low,      Validation characterization using the <br /> often requiring empirical testing proper phage         essential gene product adenylate kinase (Adk) <br /> concentration followed re-plating PCR-screening         The target gene chosen validation and <br /> for correct genotypes. Therefore, aim de-        characterization adk, encoding E. coli&#8217;s <br /> velop general genetic set-up turns working        Adk, essential gene product required biosyn- <br /> essential genes engineering gene pro-         thesis purine ribonucleotides regulation of <br /> ducts straight-forward approach facile work-      intracellular nucleotide availability [1,21,22]. For comple- <br /> ing non-essential genes. Here, present simple        mentation, adk control Ptac inserted into <br /> transformation-based system. Establishment sys-         pKOCOMP, giving rise pKOCOMP-adk. Upon induc- <br /> tem begins chromosomal replacement             tion pKOCOMP-adk IPTG, chromosomal <br /> essential gene PCR-derived selection          copy adk replaced PCR-generated kanamycin <br /> marker [17], conjunction complementary               resistance cassette [17]. The genotype resulting <br /> vector-encoded version target. A central element        strain E. coli adk::kan [pKOCOMP-adk] confirmed <br /> of method complementation vector <span id='am-88' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-92'  ></span>car</span>-          PCR using primers binding chromosomal regions <br /> ries I-SceI nuclease recognition site       up- downstream adk-locus (Additional file 1: <br /> rapidly conditionally eliminated presence        Table S1 Additional file 2 Figure S1). To complete <br /> an I-SceI nuclease-expressing helper plasmid. During           selection system, strain adk::kan [pKOCOMP-adk] <br /> elimination complementation vector, cells           transformed helper plasmid pI-SceI. The result- <br /> made electrocompetent transformed vector-           ing strain adk::kan [pKOCOMP-adk; pI-SceI] grown <br /> encoded library specific variant essential         presence glucose maintenance repress <br /> gene. Thus, actual step gene exchange reduced        I-SceI nuclease production. However, cells were <br /> to single transformation plus preparation suitable      transferred glucose-free LB medium, I-SceI produc- <br /> knock-out strain.                                              tion apparently efficiently induced addition <br />                                                                rhamnose, colony-forming units recov- <br /> Results                                                        ered induction-8period 180 minutes <br /> Overview replacement                             (Figure 2A). Importantly, copy adk <br /> The selection flexibly assembled vari-        gene, cells remained viable generations, <br /> ous user-defined ways, discussed sim-       presumably mRNA protein depleted. <br /> plest version first. It relies <span id='am-84' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-85' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span>: knock-out   This expected linear DNA rapidly degraded by <br /> and complementation vector pKOCOMP helper              cellular exonucleases half-life range of <br /> plasmid pI-SceI (Figure 1A Table 1). Plasmid pKO-          minutes [23] proteins relatively stable <br /> COMP derivative pKD46 [17] encodes             half-lives range hours [24]. <br /> arabinose-inducible &#955; red recombination (genes            During intermediate period cells elec- <br /> &#946;, &#947; exo) essential gene interest. It    trocompetent washing H2O glycerol and <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                                          Page 3 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />  Figure 1 Overview transformation-based replacement essential genes. A: <span id='am-103' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-104' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-106' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Plasmids</span> required establishment selection <br />  B: General procedure: The essential target gene adk replaced antibiotic resistance cassette pKOCOMP-adk complements for <br />  chromosomal loss. Plasmid pKOCOMP conditionally eliminated co-expression I-SceI nuclease helper plasmid pI-SceI. During <br />  elimination, cells electrocompetent transformed variant library investigation evaluate functionality to <br />  select functional library members complement loss pKOCOMP. <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                                  Page 4 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br /> Table 1 Bacterial strains plasmids used study <br /> Bacterial strain plasmid               Characteristics                                                             Source <br /> Strains <br /> W3110                                     F- &#955;- rph-1 INV(rrnD, rrnE)                                                 [33], internal <br />                                                                                                                       strain collection <br /> DH10B                                     F- endA1 recA1 galE15 galK16 nupG rpsL &#916;lacX74                              [34], internal <br />                                           &#934;80lacZ&#916;M15 araD139 &#916;(<span id='am-40' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-44'  ></span>ara</span>,leu)7697 mcrA &#916;(mrr-hsdRMS-mcrBC) &#955;-              strain collection <br /> W3110 adk::kan                            Chromosomal adk replaced KmR cassette.                                 This work <br />                                           The strain viable adk deletion complemented. <br /> W3110 groE::kan                           Chromosomal groS groL replaced KmR cassette.                       This work <br />                                           The strain viable groE deletion complemented. <br /> W3110                                     Chromosomal secB gpsA replaced KmR cassette.                       This work <br /> secB-gpsA::kan                            The strain viable gpsA deletion complemented. <br /> SB&#916;recA                                   lacIq rrnB3 &#916;lacZ4787 hsdR514 &#916; (araBAD) 567 &#916; (rhaBAD)                     This work <br />                                           568 rph-1 groE::ParaBAD-groE-KmR recA::FRT <br /> Plasmids <br /> pKD46                                     ori pSC101ts, ApR. Encodes &#955; red recombination genes                        [1] <br />                                           &#947;, &#946; exo control arabinose-responsive promoter ParaBAD. <br /> pKOCOMP-adk                               pKOCOMP-derived vector adk control                         This work <br />                                           IPTG-responsive tac promoter <br /> pSEVA432                                  ori pBBR1, SpecR, multiple cloning site                                     provided by <br />                                                                                                                       Victor Lorenzo <br /> pCOMP-adk                                 pSEVA432 encoding adk control IPTG-responsive               This work <br />                                           tac promoter, contains I-SceI cleavage site <br /> pCOMP-ESL                                 pSEVA432 encoding groS groL control                           This work <br />                                           IPTG-responsive tac promoter, contains I-SceI cleavage site <br /> pCOMP-secB-gpsA                           pSEVA432 encoding natural secB-gpsA transcriptional unit,           This work <br />                                           contains I-SceI cleavage site <br /> pSEVA132                                  ori pBBR1, ApR, multiple cloning site                                       provided by <br />                                                                                                                       Victor Lorenzo <br /> pSEVA132-adk                              pSEVA132 encoding adk control natural promoter             This work <br /> pSEVA132-adkStop                          pSEVA132-adk internal stop <span id='am-50' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53'  ></span><span id='am-54' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span>                                    This work <br /> pSEVA132-adkwatermark                     pSEVA132-adk peptide insertion position P140                  This work <br /> pSEVA132-groE                             pSEVA132 coding groS groL control                    This work <br />                                           natural promoter <br /> pSEVA132-groEStop                         pSEVA132-groE internal stop codon                                   This work <br />                  watermark <br /> pSEVA132-groE                             pSEVA132-groE peptide insertion site I301                     This work <br /> pSEVA132-secBgpsA                         pSEVA132 encoding secB gpsA control                  This work <br />                                           natural promoter <br /> pSEVA132- secBgpsAStop                    pSEVA132- secBgpsA internal stop codon                              This work <br /> pSEVA671                                  ori p15A, GmR, multiple cloning site                                        provided by <br />                                                                                                                       Victor Lorenzo <br /> pI-SceI                                   pSEVA671, I-SceI nuclease control rhamnose                This work <br />                                           inducible promoter PRha response regulators <br />                                           RhaS RhaR, derived rhammnose metabolizing <br />                                           transcriptional unit E. coli. <br /> pParaI-SceI                               pI-SceI PrhaBAD regulators RhaS RhaR exchanged                 This work <br />                                           arabinose promoter ParaBAD regulator araC <br />  <br />  <br /> transformed various test plasmids (Table 1) order                   transformation efficiencies. Transformation efficiencies <br /> to determine transformation efficiencies fre-                       107 &#956;g-1 DNA routinely achieved. The system <br /> quency false positive variants. Vector pSEVA132-adk                      intensively characterized regarding frequency of <br /> encoding wild-type adenylate kinase control                       false positive variants arise recom- <br /> of natural promoter used determination                       bination library plasmid residual linearized <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                                          Page 5 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />  Figure 2 Characterization pKOCOMP-adk/pI-SceI pCOMP-adk/pParaI-SceI systems. A: Elimination pKOCOMP-adk from <br />  adk::kan rhamnose-induced expression I-SceI nuclease helper plasmid pI-SceI. Red arrow: Time point harvest competent <br />  cell preparation. The inlet gives better resolution elimination dynamics. The chromosomal adk-replacement established using <br />  pKOCOMP-adk. B: Elimination pCOMP-adk adk::kan arabinose-induced expression I-SceI nuclease helper plasmid <br />  pParaI-SceI. The chromosomal adk-replacement established using pKD46 pCOMP-adk. <br />  <br />  <br />  <br />  <br /> pKOCOMP-adk uninduced subpopula-                             transformed cell (Table 2). This demonstrates the <br /> tion cells maintained pKOCOMP (Table 2).                          importance careful management plasmid <br /> Recombination-based false positives tested                        elimination step. <br /> transformation pSEVA132-adkstop, harboring stop <br /> codon adk gene repaired recom-                    Flexibility replacement system <br /> bination allowing growth corresponding carrier                     To verify selection set al- <br /> cell. False positives incomplete induction                    ternative combinations, constructed pCOMP-adk. <br /> examined transformation vector                          This vector based pBBR1 ori expected <br /> pSEVA132. In cases false positive colonies                   copy number 10-20 cell [25], carrying Ptac <br /> detected I-SceI expression induced pKO-                      promoter-controlled adk gene, lac-repressor LacI, <br /> COMP eliminated competent cell prepar-                         I-SceI cleavage site. While previous comple- <br /> ation (Table 2), corresponding frequency                     mentation vector pKOCOMP-adk used temperature- <br /> than 2 x 10-4 false positives transformed cell. How-                  sensitive pSC101 ori 2-3 copies cell when <br /> ever, detectable frequency recombination                   grown 37&#176;C [26], increased copy number of <br /> events pKOCOMP eliminated                             pCOMP-adk allowed examination plas- <br /> transformation test plasmids tried                    mid elimination sufficiently efficient higher intra- <br /> to eliminate pKOCOMP solely growth non-                         cellular plasmid concentration. We used pCOMP-adk in <br /> permissive temperature. In case, I-SceI nucle-                 combination pKD46 replace adk kanamycin <br /> ase expression induced transformation                   cassette. After curing cells pKD46 growth 43&#176;C, <br /> plating arabinose-supplemented agar plates,                  resulting strain adk::kan [pCOMP-adk] trans- <br /> both plasmids simultaneously present                      formed pParaI-SceI used gene replacement <br /> cells short period time. Here il-                       described using arabinose induce I-SceI pro- <br /> legitimate events frequency 3 x 10-3                      duction. Despite higher copy number pCOMP, the <br />  <br />  <br /> Table 2 Transformation efficiencies frequency false positive variants Adk-specific selection systems <br />                   Strain                          I-SceI induction                            Number transformants <br />                                                 competent cell preparation                      (frequency CFU transformant) <br />                                                                                    pSEVA132-adk         pSEVA132-adkstop           pSEVA132 <br /> W3110 adk::kan [pKOCOMP-adk, pI-SceI]                        No                     20 200 &#177; 2900        66 &#177; 21 (~3x10-3)       28 &#177; 13 (~1x10-3) <br />                                                              Yes                    21 000 &#177; 6000           0 (&lt;2x10-4)            0 (&lt;2 x10-4) <br />                                                                                                                           -3 <br /> W3110 adk::kan [pCOMP-adk, pParaI-SceI]                      No                     32 100 &#177; 3700         61 &#177; 17 (2x10 )        35 &#177; 11 (1x10-3) <br />                                                              Yes                    28 000 &#177; 2200           0 (&lt;2x10-4)             0 (&lt;2x10-4) <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                     Page 6 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br /> plasmid rapidly eliminated cells       pCOMP-plasmid carrying adk (Figure 3). After that, <br /> the presence arabinose (Figure 2B). Probably       systems routinely showed transformation efficien- <br /> the faster on-set possibly higher expression         cies 106-107 colonies &#956;g-1 DNA transformed <br /> levels arabinose responsive promoter ParaBAD          positive control vectors pSEVA132-groE or <br /> [19,20], elimination-dynamics pCOMP/pParaI-SceI       pSEVA132-secBgpsA. No false positive variants be <br /> system faster pKOCOMP/pI-SceI         detected transformation test plasmids <br /> system. Although elimination <span id='am-93' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-96'  ></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> sys-        pSEVA132-groEstop pSEVA132-secBgpsAstop, con- <br /> tems difficult compare differences          structed analogy pSEVA132-adkstop before. Import- <br /> copy number promoters, results indicate       antly, general feature cells stay <br /> the described approach set multiple ways,       viable - judged doubling time comparison <br /> making easy adapt plasmid strategies spe-       non-induced control culture did produce <br /> cific purposes. Transformation efficiencies           I-SceI - generations, depending the <br /> pCOMP/pParaI-SceI (~107 colonies &#956;g-1 DNA)            target gene, loss complementing plasmid. <br /> were comparable pKOCOMP/pI-SceI                This important characteristic cells <br /> (Table 1) identify false positive           competent transformation library <br /> transformants.                                               variant essential gene period the <br />                                                              complementing plasmid lost no <br /> Generality                                     longer contribute recombination. <br /> To confirm utility limited <br /> to adk easily extended essential       Performance selection <br /> genes, constructed vivo selection systems    To test regarding performance reliably <br /> essential gene products: chaperonin GroEL        identifying functional variants large library without <br /> co-chaperonin GroES (encoded groE operon con-         contamination false positive variants challenged the <br /> taining genes groL groS), glycerol-3-            GroEL- Adk-specific selection mock li- <br /> phosphate dehydrogenase (GpsA encoded gpsA). For          braries various excesses non-functional variants. As <br /> establishment GroEL-specific intro-         conventional bleach-out available for <br /> duced I-SceI cleavage site vector pSEVA431-      GroEL (the E. coli MGM100 strain [10]), compared <br /> groE PCR giving rise vector pCOMP-groE. Plasmid        performance replacement developed here <br /> pSEVA431-groE encodes groE-operon control          bleach-out system. In strain MGM100, groE pro- <br /> of IPTG-inducible Ptac promoter. It harbors     moter replaced araBAD promoter the <br /> lacI gene, spectinomycin resistance cassette repli-    regulatory gene araC. The strain main- <br /> cates pBBR1 ori. Plasmid pCOMP-groE used          tained presence arabinose GroEL production <br /> in combination pKD46 replace chromosomal         chromosomal locus fully repressed the <br /> groE-operon kanamycin resistance cassette (Add-         presence glucose. This way, functional variants be <br /> itional file 2 Figure S1). The resulting strain groE::kan    selected library presence glucose. However, <br /> [pCOMP-groE] cured pKD46 43&#176;C           copy wild-type gene present selec- <br /> transformed helper plasmid pParaI-SceI.             tion, recombination lead selection false positive <br />    The GpsA-specific constructed cloning       variants, dominate stringent selection <br /> the natural gpsA transcriptional unit (consisting genes   conditions. To improve strain regarding expected <br /> secB gpsA control natural promoter)       recombination frequency deleted recA leading the <br /> into vector pSEVA431 using primers encoding I-        strain SB&#916;recA (Additional file 2: Figure S1). <br /> SceI restriction site, resulting vector pCOMP-               The created mock libraries contained decreasing ratios <br /> secBgpsA. Chromosomally encoded secB gpsA           functional non-functional gene variants 1:102 <br /> then replaced kanamycin resistance cassette using       1:105 (functional:non-functional) mimicking libraries <br /> pKD46 (Additional file 2 Figure S1). To complete         decreasing number variants expected be <br /> set-up, strain secBgpsA::kan [pCOMP-secBgpsA]            functional large pool non-functional variants, <br /> cured pKD46 transformed helper plasmid           typical e.g. libraries created error-prone <br /> pParaI-SceI. Both systems characterized regarding       PCR different error rates. As non-functional var- <br /> elimination dynamics complementing plasmids           iants used pSEVA132-groEstop pSEVA132-adkstop. <br /> pCOMP-groE pCOMP-secBgpsA I-SceI induc-            As functional variants used pSEVA132-groEwatermark <br /> tion, regarding transformation efficiencies    pSEVA132-adkwatermark. These plasmids encode a <br /> electrocompetent cells prepared pCOMP-                GroEL Adk-variant in-frame insertion a <br /> elimination frequency false positive variants. Both   short peptide previously identified permissive sites. <br /> pCOMP-type plasmids lost comparable rate        Both variants fully functional identified <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                                                          Page 7 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />     Figure 3 Characterization pCOMP-groE/pParaI-SceI pCOMP-secBgpsA/pParaI-SceI system. Elimination (A) pCOMP-groE <br />     groE::kan (B) pCOMP-secBgpsA secBgpsA::kan presence arabinose helper plasmid pParaI-SceI. Elimination induced <br />     arabinose-induced expression I-SceI nuclease helper plasmid pParaI-SceI. Red arrow: Time point harvest competent cell <br />     preparation. <br />  <br />  <br /> by PCR using watermark-specific primers. To compare                                functional non-functional variants. <br /> the GroEL-specific selection GroEL                                 After plating incubation overnight LB-agar plates <br /> bleach-out SB&#916;recA, electrocompetent W3110                                  supplemented arabinose, corresponding anti- <br /> groE::kan [pCOMP-groE, pParaI-SceI] cells electro-                              biotics, and, case W3110 groE::kan [pCOMP- <br /> competent SB&#916;recA cells transformed dif-                             groE, pParaI-SceI], glucose, colonies scored a <br /> ferent mock libraries separate experiments. As                                  subset colonies genotyped watermark- <br /> positive negative controls, cells transformed                             specific primers (Table 3). <br />  <br />  <br /> Table 3 Recovery functional GroEL-variants Adk-variants mock libraries increasing excess of <br /> non-functional variants using established selection comparison conventional bleach-out system <br /> Strain                         Library (functional: non-functional)        Number colonies         Number genotyped           Number detected <br />                                                                               transformationa                    variants                false positive variants <br /> W3110 groE::kan                            Only functional                           ~ 27 000                           20                           0 <br /> [pCOMP-groE, pParaI &#8211;SceI] <br />                                                  1:102                                  231                             20                           0 <br />                                                  1:103                                   33                             20                           0 <br />                                                  1:104                                   4                              4                            0 <br />                                                  1:105                                   1a                             1                            0 <br />                                          Only non-functional                             0                              0                            0 <br /> SB&#916;recA                                    Only functional                           ~ 25 000                           20                           0 <br />                                                  1:102                                  253                             20                           0 <br />                                                  1:103                                   25                             20                           1 <br />                                                  1:104                                   7                              7                            5 <br />                                                  1:105                                   6                              6                            5 <br />                                          Only non-functional                             7                              7                            7 <br /> W3110 adk::kan                             Only functional                           ~ 46 000                           20                           0 <br /> [pCOMP-adk, pParaI-SceI] <br />                                                  1:102                                  459                             20                           0 <br />                                                  1:103                                   36                             20                           0 <br />                                                  1:104                                   3                              3                            0 <br />                                                  1:105                                   1                              1                            0 <br />                                          Only non-functional                             0                              0                            0 <br /> a            9 <br />  Typically, 10 cells used transformation, cases allowed isolate positive colony high-stringency case. In these <br /> cases, number transformed cells increased 3*109 cells. <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                        Page 8 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />    In GroEL-specific system, numbers colonies         transformation-competent. Efficient plasmid elimination <br /> correlated numbers expected trans-            essential prerequisite high performance of <br /> formation efficiencies decreasing number              evaluation large libraries under <br /> functional variants library. Even          stringent conditions elimination wild-type <br /> stringent library (functional variants frequency         gene cells prior introduction variants pre- <br /> 10-5) functional variants carrying watermark            vents recombination-based background growth. The <br /> were recovered transformation ~ 81,000 cells. No        major shortcoming conventionally used bleach-out <br /> colony detected transformation approxi-             systems recombination silenced <br /> mately number cells non-                   wild-type gene selection. Even recA strain, <br /> functional variants. In contrast using SB&#916;recA          RecA-independent recombination homologous <br /> the experiment, false positive variants                regions occurs frequencies 10-3-10-4 [27]. <br /> detected transformation negative control          As soon number expected functional variants <br /> (transformation non-functional variants)          library drops 10-3 selection primarily <br /> a frequency ~ 2 x 10-4. In addition, identified false      recombination events instead functional library <br /> positive variants genotyping subset col-          members. This clearly seen selection ex- <br /> onies selected different mock            periment using GroEL bleach-out strain SB&#916;recA <br /> libraries (Table 3). Selection stringency positively corre-      mock libraries decreasing ratio functional <br /> lated false positive rate, requiring laborious     variants. Even strain deficient recA, <br /> orthogonal assays differentiate true false posi-         frequently isolated false positive variants. <br /> tives. In directed evolution experiment re-           Besides recombination-free, advantage of <br /> sult requirement intensive post-screening          presented set-up bleach-out systems that <br /> selected variants true functional library members.           selected variants directly expressed clean strain- <br /> Plasmids false positive variants,          background circumventing laborious post-transformation <br /> had isolated transformation non-            work P1-transductions. This allows subsequent <br /> functional variants, analyzed sequencing.        purification engineered proteins vitro <br /> For stop codon reverted              characterization user-defined vivo applications <br /> the wild-type codon, probably recombination               running risk wild-type contamination. <br /> with chromosomal groL copy. The analyzed                 We elimination com- <br /> variant stop codon groL indicating          plementing vector knock-out strain, cells re- <br /> that ability grow arisen muta-           main viable generations - depending on <br /> tion araBAD promoter preventing repression.          gene - electrocompetent cells prepared after <br /> This phenomenon described strain             elimination yield 107 transformants &#956;g-1 DNA. <br /> MGM100 [16].                                                       As set-up essential target genes are <br />    The experiment performed Adk-               expressed inducible promoter competent <br /> specific selection using mock libraries decreas-     cell preparation, appropriate bleach-out times be <br /> ing ratios functional Adk-variants non-functional          adjusted individual gene products tuning in- <br /> Adk-variants. Also set-up did identify       ducer concentration. <br /> false positive variant transformation           The validated different essential E. <br /> most stringent library reliably identify functional   coli proteins: Adk, glyceraldehyde-3-phosphate dehydro- <br /> variant containing watermark transformation            genase chaperonin GroEL. Transformation efficien- <br /> of ~ 138,000 cells (Table 3).                                    cies absence detectable recombination events <br />                                                                  proved independent essential target gene. <br /> Discussion <br /> In study present facile efficient set-up a <br /> (within tested boundaries) background-free selection         Conclusions <br /> system functional, engineered, essential proteins. It     In current work present straightforward, <br /> based conditional elimination complemen-             transformation-based enables genetic <br /> tary plasmid-based copy essential gene knock-         replacement wild-type essential gene a <br /> out strain order replace essential gene genes       library variant. It directly results isola- <br /> from library (for example).                                    tion functional variants clean strain background <br />   We I-SceI nuclease-based cleavage proves          considerably reduced effort. <br /> be suitable strategy fast efficient elimin-           Furthermore, substantially facilitates working and <br /> ation complementing plasmid knock-out                engineering essential genes protein products <br /> strain occur strain            making experimentally easy, fast scalable task. <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                  Page 9 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />   Finally, possible adapt intro-    operon. groS amplified W3110 PCR with <br /> duced replacement strategy hosts &#8211; like e.g.        primers groS-fw groS-rv groL PCR- <br /> yeast, <span id='am-7' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-15'  ></span><span id='am-16' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span>Bacillus subtilis</span> Clustridium spec. -    amplified primers groL-fw groL-rv se- <br /> homologous recombination-based knock-out strategies          quentially cloned KpnI HindIII sites of <br /> are available [28-30].                                       pACT3. Plasmid pSEVA132-groE constructed by <br />                                                              amplification natural groE operon E. coli <br /> Materials methods                                        genomic DNA using primers groE-forward groE- <br /> Chemical enzymes                                         reverse cloning unique sites XmaI and <br /> Restriction enzymes ligase obtained New        XbaI. Plasmid pCOMP-secBgpsA constructed clon- <br /> England Biolabs (Ipswich, MA, USA) used according        ing natural secB-gpsA transcriptional unit, amplified <br /> to manufacturers&#8217; instructions. Chemicals purchased     primers secBgpsA-forward gpsA_I-SceI-reverse, <br /> in highest purity available Sigma-Aldrich, Fluka    pSEVA431 restriction sites XmaI XbaI. <br /> (Buchs, Switzerland) Roth (Lauterbourg, France). Tryp-    Plasmids pSEVA132-adkstop, pSEVA132-adkwatermark , <br /> ton yeast extract BD Bioscience (Basel,        pSEVA132-groEstop,       pSEVA132-groEwatermark       and <br />                                                                                  stop <br /> Switzerland). Oligonucleotides Sanger-sequencing         pSEVA132-secBgpsA        constructed amplifica- <br /> service purchased Microsynth (Balgach,             tion re-ligation pSEVA132-adk, pSEVA132-groE <br /> Switzerland).                                                pSEVA132-secBgpsA using primers adk-stop-fw/ <br />                                                              adk-stop-rv, adk-watermark-fw/adk-watermark-rv, groE- <br /> Strains plasmids                                         watermark-fw/groE-watermark-rv, groE-stop-fw/groE- <br /> E. coli DH10B used general cloning procedures.       stop-rv secBgpsA-stop-fw/secBgpsA-stop-rv. <br /> E. coli W3110 used chassis chromo-          Helper plasmids pI-SceI pParaI-SceI derivatives <br /> somal deletions (see Table 1 overview strains      pSEVA671 (ori p15A, GmR). The gene I-SceI nu- <br /> used study). SB&#916;recA derivative strain       clease amplified plasmid pSTKST [32] using <br /> MGM100 [10]. It constructed BW25113 recA::          primers I-SceI-fw I-SceI-rv cloned into <br /> FRT [1] P1-phage transduction using lysate         pSEVA671 PacI EcoRI restriction sites. The <br /> MGM100 selecting KmR. The final clone            RhaR-RhaS/PrhaBAD regulatory amplified <br /> confirmed PCR analysis groE recA::FRT          E. coli genomic DNA using primers Rha-forward <br /> locus inability grow glucose          Rha-reverse cloned I-SceI restric- <br /> GroEL-bleach-out. Plasmid pKOCOMP-adk deriva-           tion sites NsiI SpeI. The AraC/ParaBAD regulatory <br /> tive pKD46 [17] constructed cloning      amplified E. coli genomic DNA using <br /> the adk gene multiple cloning site expression    primers ParaBAD-fw ParaBAD-rv used ex- <br /> vector pACT3 [31] restriction sites BamHI Hin-       change RhaR-RhaS/Prha using sites NsiI SpeI. <br /> dIII using primers pKOCOMP-adk-fw pKOCOMP- <br /> adk-rv (see Additional file 1 Table S1 primer            Preparation competent cells transformation <br /> sequences). The resulting vector pACT-adk used        E. coli adk::kan [pKOCOMP-adk, pI-SceI] [pCOMP- <br /> template amplify Ptac promoter-controlled adk         adk, pParaI-SceI] grown overnight LB liquid <br /> gene primers pACT-forward pACT-reverse, en-         broth supplemented 50 &#956;g mL-1 kanamycin, 10 &#956;g <br /> coding I-SceI recognition site, cloned        mL-1 gentamicin, 100 &#956;M IPTG 0.5% (wt/vol) glu- <br /> pKD46 unique NcoI-site. For construction          cose (to efficiently repress I-SceI production) 30&#176;C or <br /> pCOMP-adk Ptac promoter-controlled adk gene          37&#176;C. Cells pelleted, washed LB and <br /> amplified primers pACT-SceI-Spe pACT-Pac            diluted 1:100 fresh LB broth, supplemented the <br /> and PCR product cloned unique SpeI          antibiotics glucose. Cells <br /> and PacI sites pSEVA432 (ori pBBR1, SpecR resist-         grown 37&#176;C. At OD600 0.2, 10 mM rham- <br /> ance). Plasmid pSEVA132-adk derivative               nose 0.2% arabinose (wt/vol) added induce I- <br /> pSEVA132 (ori pBBR1, ApR) encodes adk controlled         SceI nuclease production. When reaching OD600 of <br /> by natural promoter fused C-terminal            0.4-0.5, cells chilled ice 30 min, harvested <br /> 6xHis-tag. It amplified genomic E. coli DNA         washed twice chilled water 10% <br /> using primers adk-forward adk-reverse cloned         glycerol described [26]. For transformation the <br /> into pSEVA132 restriction sites XmaI SacI. Plas-     test plasmids mock libraries, 50 &#956;L cells (OD600 <br /> mid pCOMP-groE constructed amplification           100) mixed 1.5 ng DNA, exposed to <br /> the Ptac-controlled groE-operon pACT-ESL using          electrical pulse 1.3 kV recovered 1 mL LB <br /> primers pACT-SceI-Spe pACT-Pac cloned           broth supplemented 10 mM rhamnose 0.2% ara- <br /> the unique SpeI PacI sites pSEVA431. pACT-ESL         binose 1 h 37&#176;C. Selection overnight at <br /> is pACT3 derived encodes Ptac controlled groE        37&#176;C LB agar plates containing 50 &#956;g mL-1 <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                                            Page 10 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br /> kanamycin, 10 &#956;g mL-1 gentamicin, 100 &#956;g mL-1 ampi-                  Authors&#8217; contributions <br /> cillin 10 mM rhamnose 0.2% arabinose. The                     SB performed experiments, SB SP designed experimental <br />                                                                      approach wrote manuscript, SP supervised research. All authors <br /> GroEL-specific selection based W3110 groE::                read approved final manuscript. <br /> kan [pCOMP-groE, pParaI-SceI] GpsA-specific <br /> selection based W3110 secBgpsA::kan                        Acknowledgment <br /> [pCOMP-secBgpsA, pParaI-SceI] treated                  This work funded EU FP6 projects Eurobiosyn NANOMOT <br /> way. For preparation competent SB&#916;recA, cells                ESF project Nanocell. The authors indebted Tania Roberts for <br />                                                                      proofreading manuscript. <br /> grown overnight LB supplemented 50 &#956;g mL-1 <br /> kanamycin 0.2% arabinose. Cells washed twice                Received: 13 June 2012 Accepted: 8 August 2012 <br /> with water diluted 1:100 fresh LB medium. When                Published: 16 August 2012 <br /> cells reached OD600 0.1 0.5% glucose added to <br /> repress chromosomal GroEL production. Cellular GroEL                 References <br /> was bleached generations cells har-              1. Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko KA, <br />                                                                          Tomita M, Wanner BL, Mori H: Construction <span id='am-61' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-62' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-72'  ></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span>Escherichia coli</span> K-12 <br /> vested competent cell preparation (at OD600 0.4).                 in-frame, single-gene knockout mutants: Keio collection. Mol Syst Biol <br />                                                                          2006, 2:2006&#8211;0008. <br />                                                                      2. Rosamond J, Allsop A: Harnessing power genome the <br />                                                                          search new antibiotics. Science 2000, 287:1973&#8211;1976. <br /> Determination plasmid loss                                        3. Haselbeck R, Wall D, Jiang B, Ketela T, Zyskind J, Bussey H, Foulkes JG, <br /> Loss pKOCOMP-adk pCOMP-adk, pCOMP-groE                             Roemer T: Comprehensive essential gene identification platform for <br /> and pCOMP-secBgpsA determined follows: The                        novel anti-infective drug discovery. Curr Pharm Des 2002, 8:1155&#8211;1172. <br />                                                                      4. Johnson DBF, Xu JF, Shen ZX, Takimoto JK, Schultz MD, Schmitz RJ, Xiang Z, <br /> corresponding knock-out strains adk::kan, groE::kan                  Ecker JR, Briggs SP, Wang L: RF1 knockout allows ribosomal incorporation <br /> secBgpsA::kan containing helper plasmid pI-SceI                       unnatural amino acids multiple sites. Nat Chem Biol 2011, 7:779&#8211;786. <br /> pParaI-SceI grown LB liquid broth supplemented               5. Gil R, Silva FJ, Pereto J, <span id='am-1' about='obo:NCBITaxon_490011' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_233875 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_4305 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Moya Griseb." datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="rdfs:label" content="Moya" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Moya</span> A: Determination core minimal <br />                                                                          bacterial gene set. Microbiol Mol Biol Rev 2004, 68:518&#8211;537. <br /> with 50 &#956;g mL-1 kanamycin 10 &#956;g mL-1 gentamicin                  6. Posfai G, Plunkett G 3rd, Feher T, Frisch D, Keil GM, Umenhoffer K, <br /> until exponential growth reached. Next, I-SceI nu-                   Kolisnychenko V, Stahl B, Sharma SS, Arruda M, et al: Emergent properties <br /> clease production induced 10 mM rhamnose                        reduced-genome <span id='am-77' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-78' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-81'  ></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia</span> coli. Science 2006, 312:1044&#8211;1046. <br />                                                                      7. Trinh CT, Unrean P, Srienc F: Minimal Escherichia coli cell most <br /> or 0.2% arabinose. After induction aliquots taken                   efficient production ethanol hexoses pentoses. Appl Environ <br /> after 0, 10, 20, 40, 60, 120 180 min normalized                  Microbiol 2008, 74:3634&#8211;3643. <br /> to OD600. Serial dilutions subsequently plated               8. Lee JH, Sung BH, Kim MS, Blattner FR, Yoon BH, Kim JH, Kim SC: Metabolic <br />                                                                          engineering reduced-genome strain Escherichia coli for <br /> LB agar supplemented 0.5% glucose. The day                     L-threonine production. Microb Cell Fact 2009, 8:2. <br /> the number colony forming units (CFU) counted.                9. Danchin A: Natural selection immortality. Biogerontology 2009, 10:503&#8211;516. <br /> The number colonies resulting aliquots                 10. McLennan N, Masters M: GroE vital cell-wall synthesis. Nature 1998, <br />                                                                          392:139&#8211;139. <br /> had taken control culture grown ab-               11. Herring CD: Introduction conditional lethal amber mutations in <br /> sence rhamnose arabinose set 100% CFU.                     Escherichia coli. Methods Mol Biol 2008, 416:323&#8211;334. <br />                                                                      12. Jin Y, Watt RM, Danchin A, Huang JD: Use riboswitch-controlled <br />                                                                          conditional hypomorphic mutation uncover role essential csrA <br />                                                                          gene bacterial autoaggregation. J Biol Chem 2009, 284:28738&#8211;28745. <br /> Chromosomal knock-out essential genes                             13. Ji Y, Zhang B, Van SF, Horn, Warren P, Woodnutt G, Burnham MK, <br /> Knock-outs &#955; red-based recombination                        Rosenberg M: Identification critical staphylococcal genes using <br />                                                                          conditional phenotypes generated antisense RNA. Science 2001, <br /> with PCR-encoded kanamycin resistance cassette as <br />                                                                          293:2266&#8211;2269. <br /> described earlier [17]. The kanamycin-cassettes                 14. Tang YC, Chang HC, Chakraborty K, Hartl FU, Hayer-Hartl M: Essential role of <br /> generated pKD13 template primers adk-                      chaperonin folding compartment vivo. EMBO J 2008, 27:1458&#8211;1468. <br />                                                                      15. Herring CD, Blattner FR: Conditional lethal amber mutations essential <br /> H1 adk-H2, groE-H1 groE-H2 secBgpsA-H1 <br />                                                                          Escherichia coli genes. J Bacteriol 2004, 186:2673&#8211;2681. <br /> and secBgpsA-H2. Knock-outs PCR-verified                   16. van der Vies SM, Lund PA: Determination chaperonin activity vivo. <br /> primers P1-P6 indicated Additional file 2 Figure S1                Methods Mol Biol 2000, 140:75&#8211;96. <br />                                                                      17. Datsenko KA, Wanner BL: One-step inactivation chromosomal genes in <br /> and Additional file 1 Table S1. <br />                                                                          <span id='am-98' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-99' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia coli K-12</span> using PCR products. Proc Natl Acad Sci U S A 2000, <br />                                                                          97:6640&#8211;6645. <br />                                                                      18. Tobin JF, Schleif RF: Transcription rha operon psr promoter. J Mol <br /> Additional files                                                         Biol 1990, 211:1&#8211;4. <br />                                                                      19. Guzman LM, Belin D, Carson MJ, Beckwith J: Tight regulation, modulation, <br />                                                                          high-level expression vectors containing arabinose PBAD <br />  Additional files file 1: Table S1. Primers used study.          promoter. J Bacteriol 1995, 177:4121&#8211;4130. <br />  Additional files file 2: Figure S1. PCR verification knock   20. Egan SM, Schleif RF: A regulatory cascade induction rhaBAD. <br />  strains.                                                                J Mol Biol 1993, 234:87&#8211;98. <br />                                                <br /> </body></html>