<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                           ORIGINAL RESEARCH ARTICLE <br />                                                                                                                                 published: 25 May 2012 <br />                                                                                                                          doi: 10.3389/fmicb.2012.00183 <br />  <br />  <br />  <br /> Genomic biochemical analysis diaminopimelate <br /> and lysine biosynthesis pathway <span id='am-149' about='obo:NCBITaxon_2735' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_203557 obo:NCBITaxon_74201 obo:NCBITaxon_51290 obo:NCBITaxon_48461 obo:NCBITaxon_203494'><span id='am-150' property="rdfs:label" content="Verrucomicrobium" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-153'  ></span><span id='am-154' property="oboInOwl:hasRelatedSynonym" content="Verrucosimicrobium" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasDbXref" content="PMID:15653928" datatype="xsd:string"></span>Verrucomicrobium</span> <br /> spinosum: identi&#65533;?cation partial characterization of <br /> L,L-diaminopimelate aminotransferase and <br /> UDP-N-acetylmuramoylalanyl-D-glutamyl-2,6-meso- <br /> diaminopimelate ligase <br /> <span id='am-311' about='obo:NCBITaxon_4420' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4410 obo:NCBITaxon_3398 obo:NCBITaxon_261007 obo:NCBITaxon_33090 obo:NCBITaxon_131221 obo:NCBITaxon_232365 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-312' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-313' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-314'  ></span><span id='am-315' property="rdfs:label" content="Victoria" datatype="xsd:string"></span>Victoria</span> R. Nachar 1 , Francisco C. Savka 1 , Sean E. McGroty 1 , Katherine A. Donovan 2 , Rachel A. North 2 , <br /> Renwick C. J. Dobson 2,3 , Larry J. Buckley 1 Andr&#233; O. Hudson 1 * <br /> 1 <br />   The Thomas H. Gosnell School Life Sciences, Rochester Institute Technology, Rochester, NY, USA <br /> 2 <br />   Biomolecular Interaction Centre, School Biological Sciences, University Canterbury, Christchurch, New Zealand <br /> 3 <br />   Department Biochemistry Molecular Biology, Bio21 Molecular Science Biotechnology Institute, The University Melbourne, Parkville, VIC, Australia <br />  <br />  <br />  <br /> Edited by:                                The Gram-negative bacterium <span id='am-144' about='obo:NCBITaxon_2736' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_203557 obo:NCBITaxon_74201 obo:NCBITaxon_2735 obo:NCBITaxon_51290 obo:NCBITaxon_48461 obo:NCBITaxon_203494'><span id='am-145'  ></span><span id='am-146' property="rdfs:label" content="Verrucomicrobium spinosum" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Verrucomicrobium spinosum</span> attracted recent <br /> Naomi L. Ward, University of <br />                                           years following sequencing annotation genome. Comparative genomic <br /> Wyoming, USA <br />                                           analysis V. spinosum using diaminopimelate/lysine metabolic genes <span id='am-435' about='obo:NCBITaxon_810' typeof='obo:NCBITaxon_204428 owl:Thing obo:NCBITaxon_204429 obo:NCBITaxon_809 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_1113537 obo:NCBITaxon_51290 obo:NCBITaxon_51291'><span id='am-436' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-437' property="oboInOwl:hasDbXref" content="PMID:21048221" datatype="xsd:string"></span><span id='am-438' property="oboInOwl:hasDbXref" content="PMID:11211265" datatype="xsd:string"></span><span id='am-439'  ></span><span id='am-440' property="oboInOwl:hasDbXref" content="PMID:9103632" datatype="xsd:string"></span><span id='am-441' property="rdfs:label" content="Chlamydia" datatype="xsd:string"></span><span id='am-442' property="oboInOwl:hasDbXref" content="PMID:10319462" datatype="xsd:string"></span><span id='am-443' property="oboInOwl:hasRelatedSynonym" content="Rickettsiaformis" datatype="xsd:string"></span><span id='am-444' property="oboInOwl:hasDbXref" content="PMID:21048222" datatype="xsd:string"></span><span id='am-445' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-446' property="oboInOwl:hasRelatedSynonym" content="Miyagawanella" datatype="xsd:string"></span><span id='am-447' property="oboInOwl:hasDbXref" content="PMID:9103637" datatype="xsd:string"></span><span id='am-448' property="oboInOwl:hasRelatedSynonym" content="Bedsonia" datatype="xsd:string"></span><span id='am-449' property="oboInOwl:hasRelatedSynonym" content="Rakeia" datatype="xsd:string"></span>Chlamydia</span> tra- <br /> Reviewed by: <br /> Uli Stingl, King Abdullah University   chomatis suggests V. spinosum employs L,L-diaminopimelate aminotransferase <br /> Science Technology, Saudi Arabia      (DapL) pathway diaminopimelate/lysine biosynthesis. The open reading frame corre- <br /> Evelyn Toh, Indiana University            sponding putative dapL ortholog cloned recombinant enzyme was <br /> Bloomington, USA <br />                                           shown possess L,L-diaminopimelate aminotransferase activity vitro. In vivo analysis <br /> Yufeng Wang, University Texas at <br /> San Antonio, USA                          using functional complementation con&#65533;?rmed dapL ortholog able func- <br /> *Correspondence:                          tionally complement E. coli mutant confers auxotrophy diaminopimelate and <br /> Andr&#233; O. Hudson, The Thomas H.            lysine. In addition role lysine biosynthesis, intermediate diaminopimelate an <br /> Gosnell School Life Sciences,          integral role peptidoglycan biosynthesis. To end, UDP-N-acetylmuramoylalanyl- <br /> Rochester Institute Technology, 85 <br />                                           D-glutamyl-2,6-meso-diaminopimelate ligase ortholog identi&#65533;?ed, cloned, was <br /> Lomb Memorial Drive, Rochester NY <br /> 14623-5603, USA.                          shown possess meso-diaminopimelate ligase activity vivo. The L,L-diaminopimelate <br /> e-mail: aohsbi@rit.edu                    aminotransferase pathway experimentally con&#65533;?rmed bacteria, some <br />                                           deemed pathogenic animals. Since animals, particularly humans, lack <br />                                           genetic machinery synthesis diaminopimelate/lysine novo, enzymes <br />                                           involved pathway attractive targets development antibiotics. Whether dapL <br />                                           essential gene bacteria currently known. V. spinosum excellent candi- <br />                                           date investigate essentiality dapL, bacterium employs DapL pathway <br />                                           lysine cell wall biosynthesis, non-pathogenic humans, facile grow, can <br />                                           genetically manipulated. <br />                                           Keywords: lysine, diaminopimelate, L,L-diaminopimelate aminotransferase, peptidoglycan, functional complemen- <br />                                           tation <br />  <br />  <br /> INTRODUCTION                                                                      diaminopimelate/lysine novo bacterium pro- <br /> Verrucomicrobium spinosum rod-shaped heterotrophic Gram-                     totrophic lysine. Lysine synthesized using general <br /> negative bacterium fresh water soil envi-                    pathways evolutionarily divergent respect the <br /> ronments characterized non-motile contains                    intermediates used. One pathway utilizes intermediate &#945;- <br /> appendages known prosthecae. V. spinosum important                       amino adipic acid, derived metabolism of <br /> the biotechnology medical sectors closely related                    2-ketoglutarate product citric acid cycle. The &#945;-amino <br /> to Chlamydia (Wagner Horn, 2006). A recent study con-                         adipic acid pathway primarily used <span id='am-316' about='obo:NCBITaxon_4751' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-317' property="oboInOwl:hasRelatedSynonym" content="fungi" datatype="xsd:string"></span><span id='am-318'  ></span><span id='am-319' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-320' property="oboInOwl:hasDbXref" content="PMID:12684019" datatype="xsd:string"></span><span id='am-321' property="oboInOwl:hasDbXref" content="PMID:11062127" datatype="xsd:string"></span><span id='am-322' property="rdfs:label" content="Fungi" datatype="xsd:string"></span><span id='am-323' property="oboInOwl:hasExactSynonym" content="fungi" datatype="xsd:string"></span><span id='am-324' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>fungi</span> narrowly <br /> &#65533;?rmed organism contains genetic components                    distributed <span id='am-367' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-368' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-369' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> belonging domain <span id='am-164' about='obo:NCBITaxon_2157' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567'><span id='am-165' property="oboInOwl:hasDbXref" content="PMID:11321083" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasRelatedSynonym" content="Prokaryotae" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasRelatedSynonym" content="Procaryotae" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasDbXref" content="PMID:2112744" datatype="xsd:string"></span><span id='am-169' property="oboInOwl:hasRelatedSynonym" content="prokaryote" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:hasDbXref" content="PMID:11542149" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasDbXref" content="PMID:11411719" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasDbXref" content="PMID:10425796" datatype="xsd:string"></span><span id='am-173' property="oboInOwl:hasDbXref" content="PMID:10939677" datatype="xsd:string"></span><span id='am-174' property="oboInOwl:hasRelatedSynonym" content="prokaryotes" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasRelatedSynonym" content="Monera" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasDbXref" content="PMID:270744" datatype="xsd:string"></span><span id='am-177' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasDbXref" content="PMID:10490293" datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasRelatedSynonym" content="Mendosicutes" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasDbXref" content="PMID:11541975" datatype="xsd:string"></span><span id='am-181' property="oboInOwl:hasRelatedSynonym" content="Archaebacteria" datatype="xsd:string"></span><span id='am-182' property="oboInOwl:hasDbXref" content="PMID:11211268" datatype="xsd:string"></span><span id='am-183' property="oboInOwl:hasDbXref" content="PMID:11760965" datatype="xsd:string"></span><span id='am-184' property="oboInOwl:hasDbXref" content="PMID:8590690" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasDbXref" content="PMID:10939673" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasDbXref" content="PMID:9336922" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasDbXref" content="PMID:11542064" datatype="xsd:string"></span><span id='am-188' property="oboInOwl:hasDbXref" content="PMID:11321113" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasDbXref" content="PMID:11540071" datatype="xsd:string"></span><span id='am-191'  ></span><span id='am-192' property="oboInOwl:hasDbXref" content="PMID:12054223" datatype="xsd:string"></span><span id='am-193' property="oboInOwl:hasDbXref" content="PMID:10425795" datatype="xsd:string"></span><span id='am-194' property="oboInOwl:hasDbXref" content="PMID:8123559" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasDbXref" content="PMID:10939651" datatype="xsd:string"></span><span id='am-196' property="rdfs:label" content="Archaea" datatype="xsd:string"></span><span id='am-197' property="oboInOwl:hasDbXref" content="PMID:10843050" datatype="xsd:string"></span><span id='am-198' property="oboInOwl:hasRelatedSynonym" content="Metabacteria" datatype="xsd:string"></span><span id='am-199' property="oboInOwl:hasDbXref" content="PMID:9103655" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:hasDbXref" content="PMID:10425797" datatype="xsd:string"></span><span id='am-201' property="oboInOwl:hasRelatedSynonym" content="Prokaryota" datatype="xsd:string"></span><span id='am-202' property="oboInOwl:hasRelatedSynonym" content="archaea" datatype="xsd:string"></span>archaea</span> <br /> encode Type III Secretion System able kill Drosophila                (Nishida et al., 1999; Velasco et al., 2002). The pathway <br /> <span id='am-568' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-569' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span><span id='am-570' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-571' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-572'  ></span>melanogaster</span> <span id='am-328' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-329' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-330' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-331' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-332' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-333'  ></span><span id='am-334' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span>Caenorhabditis elegans</span>, model invertebrate                   employs intermediate diaminopimelate, derived from <br /> hosts (Sait et al., 2011).                                                        oxaloacetate. The diaminopimelate pathway bac- <br />    Genomic analysis V. spinousm organ-                           teria photosynthetic organisms including <span id='am-44' about='obo:NCBITaxon_1117' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Oxyphotobacteria" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Cyanophyceae" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Cyanophycota" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Cyanophyta" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Oxygenic photosynthetic bacteria" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Cyanobacteria" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="PMID:15388760" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="blue-green algae" datatype="xsd:string"></span><span id='am-55'  ></span><span id='am-56' property="oboInOwl:hasDbXref" content="PMID:9382701" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="cyanophytes" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasExactSynonym" content="blue-green bacteria" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="cyanobacteria" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="PMID:11542017" datatype="xsd:string"></span>cyanobacteria</span>, algae, <br /> ism contains genes necessary synthesis                         plants. <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                           May 2012 | Volume 3 | Article 183 | 1 <br />  Nachar et al.                                                                         Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />     Four variants diaminopimelate/lysine pathway                   L,L-diaminopimelate tetrahydrodipicolinate transam- <br /> been discovered far (Figure 1): acyl pathways,                      ination reaction using glutamate amino donor and <br /> which use succinyl acetyl intermediates; meso-                   tetrahydrodipicolinate amino acceptor single reac- <br /> diaminopimelate dehydrogenase pathway; recently dis-                   tion bypassing 2,3,4,5-tetrahydropyridine-2,6-dicarboxylate <br /> covered L,L-diaminopimelate aminotransferasae (DapL) pathway                   -acyl-transferase, acyl-diaminopimelate aminotransferase, acyl- <br /> (Hudson et al., 2006, 2008; McCoy et al., 2006). The biosyn-                   diaminopimelate deacylase steps present succinylated or <br /> thesis lysine aspartate diaminopimelate/lysine                 acetylated diaminopimelate/lysine pathways (Table 1, Figure 1). <br /> pathway divided main events. The &#65533;?rst event                  The ultimate event diaminopimelate variant path- <br /> is synthesis tetrahydrodipicolinate aspartate. <span id='am-348' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-349'  ></span><span id='am-350' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-351' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-352' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span>             ways de&#65533;?ned enzyme meso-diaminopimelate decarboxy- <br /> a general feature diaminopimelate pathway variants                 lase catalyzes decarboxylation meso-diaminopimelate <br /> and carried enzymes, aspartate kinase, aspartate                 synthesize lysine. This decarboxylation step shared four <br /> semialdehyde dehydrogenase, dihydrodipicolinate synthase,                  variants (Table 1, Figure 1). <br /> dihydrodipicolinate reductase, respectively (Table 1). The sec-                    Since animals contain machinery synthesize <br /> ond event constitutes conversion tetrahydrodipicolinate                 diaminopimelate lysine, used bacteria pep- <br /> to penultimate intermediate meso-diaminopimelate. The syn-                 tidoglycan biosynthesis, enzymes involved bacterial <br /> thetic steps tetrahydrodipicolinate meso-diaminopimelate               diaminopimelate/lysine biosynthesis pathways to <br /> de&#65533;?ne uniqueness diaminopimelate variant pathways.                  scienti&#65533;?c community. meso-Diaminopimelate serves one <br /> In acyl pathways, enzymes required. These reactions               cross linking amino acids cell wall Gram- <br /> are carried enzymes 2,3,4,5-tetrahydropyridine-2,6-                 negative bacteria, lysine role Gram- <br /> dicarboxylate N -acyl-transferase, acyl-diaminopimelate amino-                 positive bacteria (Hutton et al., 2007). Inhibition enzymes <br /> transferase, acyl-diaminopimelate deacylase, diaminopime-                  diaminopimelate/lysine biosynthesis pathways have <br /> late epimerase, respectively (Table 1, Figure 1). In dehy-                 detrimental effect bacterial growth perspectives: <br /> drogenase pathway, tetrahydrodipicolinate converted meso-                diaminopimelate necessary cell wall synthesis, cell lysis <br /> diaminopimelate enzyme meso-diaminopimelate dehydro-                    occur result osmotic pressure lack pep- <br /> genase step circumventing 2,3,4,5-tetrahydropyridine-               tidoglycan (Cox, 1996; Baizman et al., 2000); addition, the <br /> 2,6-dicarboxylate-acyl-transferase, acyl-diaminopimelate amino-                inhibition enzymes pathway prevent protein syn- <br /> transferase, acyl-diaminopimelate deacylase, diaminopime-                  thesis, lysine 20 common proteogenic amino <br /> late epimerase enzymatic reactions (Table 1, Figure 1). The L,L-               acids. meso-Diaminopimelate incorporated cross <br /> diaminopimelate aminotransferase (DapL) pathway synthesizes                    linking amino acids cell wall Gram-negative bacteria <br />  <br />  <br />  <br />  <br />   FIGURE 1 | Diaminopimelate/lysine biosynthesis pathways. The            pathways, (B) meso-diaminopimelate dehydrogenase pathway, (C) the <br />   variants diaminopimelate/lysine biosynthetic pathways starting   L,L-diaminopimelate aminotransferase (DapL) pathway. The abbreviations of <br />   tetrahydrodipicolinate denoted (A) succinylated acetylated     enzymes enzyme reaction de&#65533;?nitions listed Table 1. <br />  <br />  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                        May 2012 | Volume 3 | Article 183 | 2 <br />  Nachar et al.                                                                                Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br /> Table 1 | List diaminopimelate/lysine biosynthesis genes V. spinosum DSM 4136T . <br />  <br /> Locus tag                   Gene symbol          Gene                         EC #             Reaction <span id='am-581' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> <br />  <br /> VspiD_010100025585          lysC                 Aspartate kinase                  2.7.2.4          ATP+aspartate &#8596; ADP+4-phospho-aspartate <br /> VspiD_010100030480          asd                  Aspartate semialdehyde            1.2.1.11         aspartate 4-semialdehyde + orthophosphate + NADP+ &#8596; 4- <br />                                                  dehydrogenase                                      phospho-L-aspartate + NADPH + H+ <br /> VspiD_010100011370          dapA                 Dihydrodipicolinate               4.2.1.52         Aspartate 4-semialdehyde + pyruvate &#8596; L-2,3- <br />                                                  synthase                                           dihydrodipicolinate + 2H2 O <br /> VspiD_010100011365          dapB                 Dihydrodipicolinate               1.3.1.26         2,3,4,5-Tetrahydrodipicolinate + NADP + &#8596; L-2,3- <br />                                                  reductase                                          dihydrodipicolinate + NADPH + H+ <br /> -                           dapD                 2,3,4,5-tetrahydropyridine-       2.3.1.117        Acyl-CoA + 2,3,4,5-tetrahydrodipicolinate + H2 O &#8596; CoA + N- <br />                                                  2,6-dicarboxylate                                  acyl-2-L-amino-6-oxoheptanedioate <br />                                                  Nacyl-transferase <br /> -                           dapC                 Acyl-diaminopimelate              2.6.1.17         N-acyl-L,L-2,6-diaminoheptanedioate + 2-oxoglutarate&#8596;N- <br />                                                  aminotransferase                                   acyl-2-L-amino-6-oxoheptanedioate + L-glutamate <br /> -                           dapE                 Acyl-diaminopimelate              3.5.1.18         N-acyl-L,L-2,6-diaminoheptanedioate + H2 O &#8596; acyl + L,L- <br />                                                  deacylase                                          2,6-diaminoheptanedioate <br /> -                           ddh                  Diaminopimelate                   1.4.1.16         meso-2,6-Diaminoheptanedioate + NADP+ + H2O &#8596; L-2- <br />                                                  dehydrogenase                                      amino-6-oxoheptanedioate + NH3 + NADPH <br /> VspiD_010100012510          dapL                 L,L-diaminopimelate               2.6.1.83         L,L-2,6-diaminopimelate + 2-oxoglutarate &#8596; 2,3,4,5- <br />                                                  aminotransferase                                   tetrahydrodipicolinate + L-glutamate + H2 O <br /> VspiD_010100011375          dapF                 Diaminopimelate                   5.1.1.7          L,L-2,6-diaminopimelate&#8596;meso-2,6-diaminopimelate <br />                                                  epimerase <br /> VspiD_010100003650          lysA                 Diaminopimelate                   4.1.1.20         meso-2,6-Diaminopimelate&#8594;lysine + CO2 <br />                                                  decarboxylase <br />  <br /> The list generated using genomic information deposited IMG database (http://img.jgi.doe.gov/cgi-bin/w/main.cgi). <br />  <br />  <br />  <br /> by enzyme UDP-N -acetylmuramoylalanyl-d-glutamyl-2,6-                          diaminopimelate/lysine biosynthesis. In addition, identify and <br /> meso-diaminopimelate ligase (MurE) (E.C. 6.3.2.15),                       characterize MurE V. spinosum, suggesting bacterium <br /> encoded murE gene.                                                          incorporates meso-diaminopimelate peptidoglycan a <br />     The DapL pathway identi&#65533;?ed numerous bacteria                       cross linking amino acid. <br /> and experimentally con&#65533;?rmed limited number bac- <br /> teria, deemed pathogenic (McCoy et al., 2006;                    RESULTS <br /> Hudson et al., 2008; Liu et al., 2010). The three-dimensional crys-                IDENTIFICATION OF PUTATIVE dapL AND murE GENES IN V. SPINOSUM <br /> tal structure DapL <span id='am-276' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-277' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span><span id='am-278' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-279' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-280' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span><span id='am-281'  ></span><span id='am-282' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span>Arabidopsis</span> thaliana solved                    The dapL ortholog V. spinosum identi&#65533;?ed using the <br /> (Watanabe et al., 2007, 2008) <span id='am-479' about='obo:NCBITaxon_813' typeof='obo:NCBITaxon_204428 owl:Thing obo:NCBITaxon_810 obo:NCBITaxon_204429 obo:NCBITaxon_809 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_1113537 obo:NCBITaxon_51290 obo:NCBITaxon_51291'><span id='am-480' property="oboInOwl:hasDbXref" content="PMID:10192388" datatype="xsd:string"></span><span id='am-481' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-482' property="oboInOwl:hasDbXref" content="PMID:8102247" datatype="xsd:string"></span><span id='am-483' property="oboInOwl:hasRelatedSynonym" content="Rickettsia trachomatis" datatype="xsd:string"></span><span id='am-484' property="oboInOwl:hasDbXref" content="PMID:10319462" datatype="xsd:string"></span><span id='am-485' property="rdfs:label" content="Chlamydia trachomatis" datatype="xsd:string"></span><span id='am-486' property="oboInOwl:hasDbXref" content="PMID:8347519" datatype="xsd:string"></span><span id='am-487' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-488' property="oboInOwl:hasRelatedSynonym" content="Chlamydozoon trachomatis" datatype="xsd:string"></span><span id='am-489'  ></span><span id='am-490' property="oboInOwl:hasRelatedSynonym" content="Rickettsia trachomae" datatype="xsd:string"></span>Chlamydia trachomatis</span> (Watan-                   DapL protein C. trachomatis Ct 390 (NP_219900) the <br /> abe et al., 2011) algae <span id='am-293' about='obo:NCBITaxon_3052' typeof='owl:Thing obo:NCBITaxon_33090 obo:NCBITaxon_3051 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3042 obo:NCBITaxon_3041 obo:NCBITaxon_2759 obo:NCBITaxon_3166'><span id='am-294' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-295' property="rdfs:label" content="Chlamydomonas" datatype="xsd:string"></span><span id='am-296' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-297'  ></span>Chlamydomonas</span> reinhardtii                     query BlastP algorithm Integrated Microbial <br /> (Dobson et al., 2011; Hudson et al., 2011a). Inhibitors <span id='am-343' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-344' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-345' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-346' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-347'  ></span>Ara</span>-               Genomes (IMG) database1 . This search identi&#65533;?ed putative L,L- <br /> bidopsis DapL ortholog reported using vitro studies                   diaminopimelate aminotransferase annotated locus tag <br /> (Fan et al., 2010). It noted inhibitors                    VspiD_010100012510 (ZP_02927470) 38% amino acid <br /> the Arabidopsis DapL found, known                     identity C. trachomatis DapL. The murE ortholog iden- <br /> compounds inhibit aminotransferases.                                         ti&#65533;?ed using C. trachomatis MurE (NP_219774) query. This <br />     The identi&#65533;?cation dapL murE genes, addition                       search resulted identi&#65533;?cation murE ortholog V. <br /> the easy culturability genetic manipulation V. spinosum,                    spinosum annotated locus tag VspiD_010100019130, which <br /> makes bacterium excellent model organism experi-                        37% identical C. trachomatis MurE. <br /> ments addressing essentiality dapL murE respect <br /> to lysine peptidoglycan biosynthesis (Domman et al., 2011).                    CRUDE SOLUBLE PROTEIN EXTRACT FROM V. SPINOSUM CONTAINS <br /> Most bacterial species contain DapL pathway dif&#65533;?-                     DapL ACTIVITY <br /> cult work with, slow growing, dif&#65533;?cult                 If V. spinosum utilizes DapL pathway diaminopime- <br /> culture, anaerobic, pathogenic humans.                                      late/lysine biosynthesis, suspected genomic analysis, a <br />     Here present &#65533;?rst genomic biochemical analysis                      crude soluble protein extract possess DapL activity. The <br /> of diaminopimelate/lysine biosynthesis pathway V. spin-                   ortho-aminobenzaldehyde assay employed using crude sol- <br /> osum. We identify genes necessary diaminopimelate/lysine                   uble protein extract V. spinosum. Using L,L-diaminopimelate <br /> biosynthesis genome V. spinosum demonstrate that <br /> the bacterium uses recently discovered DapL pathway                        1 http://www.jgi.doe.gov/ <br />  <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                            May 2012 | Volume 3 | Article 183 | 3 <br />  Nachar et al.                                                                        Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br /> as amino donor 2-ketoglutarate amino accep-                     consistent genomic analysis regarding absence a <br /> <span id='am-527' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-528' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-529'  ></span><span id='am-530' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-531' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>tor</span>, assay measures production dihydroquinazolium                   diaminopimelate dehydrogenase ortholog genome V. <br /> which product interaction tetrahydrodipicol-             spinosum. <br /> inate ortho-aminobenzaldehyde. DapL activity detected <br /> from crude soluble extract showing rate                    THE GENE ANNOTATED BY THE LOCUS TAG VspiD_010100012510 <br /> reaction proportional V. spinosum pro-                     (VsdapL) ENCODES AN AUTHENTIC L,L-DIAMINOPIMELATE <br /> tein extract added assay (Figure 2). Enzymatic activity                 AMINOTRANSFERASE <br /> was observed amino donor, amino accep-                         The putative VsdapL gene cloned recombinant <br /> tor, omitted assay. In addition, activity                  enzyme puri&#65533;?ed homogeneity using af&#65533;?nity chromatogra- <br /> observed protein extract heated boiling water                  phy (Figure 3). The ortho-aminobenzaldehyde assay used to <br /> for 5 min.                                                                     determine putative VsDapL L,L-diaminopimelate <br />                                                                                aminotransferase activity. The results analysis that <br /> DapL IS THE ONLY ROUTE FOR DIAMINOPIMELATE/LYSINE                              open reading frame annotated locus tag encodes an <br /> BIOSYNTHESIS IN V. SPINOSUM                                                    authentic L,L-diaminopimelate aminotransferase enzyme. With <br /> The genome V. spinosum searched catalog genes                    L,L-diaminopimelate amino donor 2-ketoglutarate as <br /> necessary novo anabolism lysine aspartate.                  amino acceptor, speci&#65533;?c activity VsDapL 4.1 &#177; 0.25 <br /> Since DapL activity detected soluble extract,                    &#916;A 440 min&#8722;1 mg&#8722;1 . Unlike plant algae enzyme, VsDapL <br /> wanted know DapL pathway sole route                     possesses transamination activity diamine donors <br /> toward lysine biosynthesis. There examples litera-                  structurally similar L,L-diaminopimelate, including the <br /> ture multiple pathways lysine biosynthesis bacteria;                 racemic isomer meso-diaminopimelate (Table 2), albeit rela- <br /> for example, genomes <span id='am-450' about='obo:NCBITaxon_817' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_200643 obo:NCBITaxon_815 obo:NCBITaxon_816 obo:NCBITaxon_171549 obo:NCBITaxon_976 obo:NCBITaxon_68336'><span id='am-451' property="oboInOwl:hasRelatedSynonym" content="Ristella incommunis" datatype="xsd:string"></span><span id='am-452'  ></span><span id='am-453' property="oboInOwl:hasRelatedSynonym" content="Bacteroides inaequalis" datatype="xsd:string"></span><span id='am-454' property="oboInOwl:hasRelatedSynonym" content="Bacteroides incommunis" datatype="xsd:string"></span><span id='am-455' property="oboInOwl:hasRelatedSynonym" content="Bacteroides fragili" datatype="xsd:string"></span><span id='am-456' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-457' property="oboInOwl:hasRelatedSynonym" content="Fusiformis fragilis" datatype="xsd:string"></span><span id='am-458' property="oboInOwl:hasDbXref" content="PMID:16559622" datatype="xsd:string"></span><span id='am-459' property="rdfs:label" content="Bacteroides fragilis" datatype="xsd:string"></span><span id='am-460' property="oboInOwl:hasRelatedSynonym" content="Bacillus fragilis" datatype="xsd:string"></span><span id='am-461' property="oboInOwl:hasRelatedSynonym" content="Pseudobacterium incommunis" datatype="xsd:string"></span><span id='am-462' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-463' property="oboInOwl:hasAlternativeId" content="NCBITaxon:33929" datatype="xsd:string"></span><span id='am-464' property="oboInOwl:hasRelatedSynonym" content="Bacteroides uncatus" datatype="xsd:string"></span><span id='am-465' property="oboInOwl:hasRelatedSynonym" content="Pseudobacterium uncatum" datatype="xsd:string"></span><span id='am-466' property="oboInOwl:hasRelatedSynonym" content="Sphaerophorus intermedius" datatype="xsd:string"></span><span id='am-467' property="oboInOwl:hasRelatedSynonym" content="Pseudobacterium fragilis" datatype="xsd:string"></span><span id='am-468' property="oboInOwl:hasRelatedSynonym" content="Ristella uncata" datatype="xsd:string"></span><span id='am-469' property="oboInOwl:hasRelatedSynonym" content="Ristella fragilis" datatype="xsd:string"></span><span id='am-470' property="oboInOwl:hasRelatedSynonym" content="Pseudobacterium inaequalis" datatype="xsd:string"></span><span id='am-471' property="oboInOwl:hasRelatedSynonym" content="Sphaerophorus inaequalis" datatype="xsd:string"></span>Bacteroides fragilis</span> <span id='am-89' about='obo:NCBITaxon_1485' typeof='owl:Thing obo:NCBITaxon_186802 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_186801 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_31979'><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Clostridium" datatype="xsd:string"></span><span id='am-93'  ></span>Clostridium</span>               tively low rates. The enzyme able use oxoacids <br /> thermocellum contain DapL meso-                     amino acceptors addition 2-ketoglutarate (Table 2); again <br /> diaminopimelate dehydrogenase pathways (Hudson et al., 2011b).                 relative rates low. <br /> Also, <span id='am-94' about='obo:NCBITaxon_1718' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1716 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_1653'><span id='am-95' property="oboInOwl:hasAlternativeId" content="NCBITaxon:37715" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasRelatedSynonym" content="Micrococcus maripuniceus" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium seonmiso" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasRelatedSynonym" content="&apos;Corynebacterium lactofermentum&apos;" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium glutamigenes" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium divaricatum" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153272" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium thiogenitalis" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153275" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasRelatedSynonym" content="Corynebacterium lilium" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium saccharolyticum" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153278" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasAlternativeId" content="NCBITaxon:92707" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="PMID:1713055" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153277" datatype="xsd:string"></span><span id='am-111'  ></span><span id='am-112' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153234" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium divaricatum Su and Yamada 1960 (Approved Lists 1980)" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasDbXref" content="PMID:10220171" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasRelatedSynonym" content="Corynebacterium lilium Lee and Good 1963 (Approved Lists 1980)" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasRelatedSynonym" content="&quot;Micrococcus glutamicus&quot; Kinoshita et al. 1958" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasAlternativeId" content="NCBITaxon:70020" datatype="xsd:string"></span><span id='am-118' property="rdfs:label" content="Corynebacterium glutamicum" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium chang-fua" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153273" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium lactofermentum" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasDbXref" content="PMID:11988527" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasRelatedSynonym" content="Corymebacterium glutamicum" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153271" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153274" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasRelatedSynonym" content="Cornyebacterium lactofermentum" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasRelatedSynonym" content="Corynebacterium lactofermentum" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasRelatedSynonym" content="&apos;Brevibacterium lactofermentum&apos;" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="Micrococcus glutamicus" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasRelatedSynonym" content="Brevibacterium taipei" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasAlternativeId" content="NCBITaxon:153276" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasRelatedSynonym" content="Corynebacterium glutamicum (Kinoshita et al. 1958) Abe et al. 1967" datatype="xsd:string"></span>Corynebacterium glutamicum</span> employs acyl pathway in <br /> conjunction Ddh pathway (Schrumpf et al., 1991). A                    VsdapL IS ABLE TO FUNCTIONALLY COMPLEMENT THE E. COLI <br /> search V. spinosum genome suggests DapL path-                  &#916;dapD/dapE (AOH1) MUTANT <br /> way route diaminopimelate/lysine biosynthesis,                 The E. coli strain AOH1 harbors deletion dapD <br /> since orthologs acyl dehydrogenase genes                  gene mutation dapE renders enzyme non- <br /> be identi&#65533;?ed genome (Table 1; Figure 1). Even                    functional (Hudson et al., 2006). As such, mutant unable <br /> the search did identify diaminopimelate dehydrogenase                    synthesize meso-diaminopimelate lysine cell wall <br /> ortholog, tested dehydrogenase activity using                  biosynthesis. As result, cells lyse osmotic pressure <br /> protein extract used detect DapL activity. This test               lack meso-diaminopimelate cross linking <br /> was using 1.0 mg total protein different buffer                 amino acid cell wall deemed auxotrophic for <br /> systems, &#65533;?nding activity. The recombinant diaminopime-                      diaminopimelate lysine. For complementation analysis, E. <br /> late dehydrogenase Clostridium thermocellum used                      coli AOH1 transformed vector (pBAD33), <br /> as positive control (Hudson et al., 2011b). This result                  DapL expression <span id='am-507' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-508' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-509' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-510' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> (pBAD33 + dapL) from <br />                                                                                plant <span id='am-283' about='obo:NCBITaxon_3702' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_3701 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-284'  ></span><span id='am-285' property="rdfs:label" content="Arabidopsis thaliana" datatype="xsd:string"></span><span id='am-286' property="oboInOwl:hasRelatedSynonym" content="Arbisopsis thaliana" datatype="xsd:string"></span><span id='am-287' property="oboInOwl:hasRelatedSynonym" content="Arabidopsis thaliana (thale cress)" datatype="xsd:string"></span><span id='am-288' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-289' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-290' property="oboInOwl:hasExactSynonym" content="mouse-ear cress" datatype="xsd:string"></span><span id='am-291' property="oboInOwl:hasExactSynonym" content="thale cress" datatype="xsd:string"></span><span id='am-292' property="oboInOwl:hasExactSynonym" content="thale-cress" datatype="xsd:string"></span>Arabidopsis thaliana</span> (AtdapL; Hudson et al., 2006), <br />                                                                                alga C. reinhardtii (CrdapL; Dobson et al., 2011) the <br />                                                                                <span id='am-370' about='obo:NCBITaxon_400048' typeof='owl:Thing obo:NCBITaxon_49928 obo:NCBITaxon_1 obo:NCBITaxon_2323 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-371' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-372'  ></span><span id='am-373' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-374' property="rdfs:label" content="bacterium V" datatype="xsd:string"></span>bacterium V</span>. spinosum (VsdapL). While E. coli AOH1 able <br />                                                                                grow media supplemented L,L-diaminopimelate, <br />                                                                                auxotrophic mutant expressing authentic orthologous <br />  <br />  <br />  <br />  <br />                                                                                 FIGURE 3 | Recombinant expression purification VsDapL using <br />  FIGURE 2 | DapL activity crude protein extract V. spinosum.          affinity chromatography. Lane (1) Protein Marker (kDa), Lane (2) 10 &#956;g of <br />  The graph shows relationship protein reaction       soluble protein uninduced cells, Lane (3) 10 &#956;g soluble proteins <br />  rate. The DapL ortho-aminobenzaldehyde assay described Section       induced cells, Lane (4) 1 &#956;g puri&#65533;?ed VsDapL. The proteins were <br />  &#8220;Materials Methods.&#65533;? The assay triplicates protein    resolved 10% (w/v) acrylamide gel stained Coomassie <br />  concentration.                                                                 blue visualization. <br />  <br />  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                       May 2012 | Volume 3 | Article 183 | 4 <br />  Nachar et al.                                                                    Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br /> DapLs able grow L,L-diaminopimelate-free media                  VsmurE IS ABLE TO FUNCTIONALLY COMPLEMENT THE E. COLI murE <br /> (Figure 4). This analysis demonstrates recombinant               MUTANT <br /> enzymes able convert tetrahydrodipicolinate L,L-                meso-Diaminopimelate serves penultimate pre- <br /> diaminopimelate, bypassing acyl enzymatic reactions present         cursor lysine biosynthesis, incorporated as <br /> in E. coli acyl pathway, facilitate synthesis meso-         cross linking amino acids cell wall Gram- <br /> diaminopimelate lysine cell wall biosynthesis (Figures 1          negative bacteria. This facilitated enzyme UDP-N - <br /> and 5).                                                                   acetylmuramoylalanyl-D-glutamyl-2,6-meso-diaminopimelate lig- <br />                                                                           ase (MurE; Figure 5). Since V. spinosum classi&#65533;?ed Gram- <br /> VsDapL BELONGS TO THE DAPL1 FORM OF L,L-DIAMINOPIMELATE                   negative bacterium, possess MurE ortholog (Schlesner <br /> AMINOTRANSFERASES                                                         et al., 2006). The VsmurE ortholog identi&#65533;?ed open <br /> Diaminopimelate aminotransferases approximately 400 amino             reading frame cloned expression vector test via <br /> acids length members pyridoxal-5  phosphate             functional complementation gene encodes authen- <br /> dependent <span id='am-325' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-326' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-327' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> <span id='am-340' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-341' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-342' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> I/II transaminases. A recent study              tic MurE enzyme. The E. coli mutant TKL-11 harbors muta- <br /> revealed diverged forms DapL enzymes;               tion murE gene results temperature sensitive <br /> DapL1 DapL2 (Hudson et al., 2008). The forms                   growth phenotype mutant able grow per- <br /> DapLs share approximately 30% homology amino acid                  missive temperature 30&#730;C, 42&#730;C (Lugtenberg and <br /> level. The DapL ortholog C. trachomatis evolutionar-              van Schijndel-van Dam, 1972). The mutant transformed with <br /> ily related DapL1 enzymes (Hudson et al., 2008). Since             vector (pBAD33) vector expressing puta- <br /> homology V. spinosum DapL 38% C. tra-               tive MurE (pBAD33 + VsmurE). Using replica-plating, results <br /> chomatis ortholog, form V. spinosum DapL               analysis demonstrate permissive temperature <br /> apparent. Using phylogenetic analysis, DapL ortholog             30&#730;C, mutant harboring vector control vec- <br /> V. spinosum clusters enzymes                 tor expressing recombinant enzyme able grow. However, <br /> experimentally con&#65533;?rmed DapL1 orthologs patho-                exposed non-permissive temperature 42&#730;C, the <br /> genic organisms (Hudson et al., 2008, 2011b). The analysis shows          mutant expressing MurE ortholog able grow (Figure 7A). <br /> two distinct clades, DapL1, DapL2. The aspartate aminotrans-              This result corroborated assessing bacterial growth a <br /> ferase <span id='am-491' about='obo:NCBITaxon_633' typeof='owl:Thing obo:NCBITaxon_629 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-492' property="oboInOwl:hasRelatedSynonym" content="Bacillus pseudotuberkulosis" datatype="xsd:string"></span><span id='am-493' property="oboInOwl:hasRelatedSynonym" content="Bacterium pseudotuberculosis" datatype="xsd:string"></span><span id='am-494' property="oboInOwl:hasRelatedSynonym" content="Pasteurella pseudotuberculosis" datatype="xsd:string"></span><span id='am-495' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-496' property="rdfs:label" content="Yersinia pseudotuberculosis" datatype="xsd:string"></span><span id='am-497' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-498'  ></span><span id='am-499' property="oboInOwl:hasRelatedSynonym" content="Shigella pseudotuberculosis" datatype="xsd:string"></span>Yersinia pseudotuberculosis</span>, classi&#65533;?ed          period 10 h. At 30&#730;C mutant harboring vector and <br /> as class I/II transaminase used outgroup (Figure 6).           murE expression vector grew expected. However, the <br /> Four analyses performed (ML, MP, ME, NJ)                cultures switched non-permissive temperature 42&#730;C <br /> moderate strong support DapL2 clade (57&#8211;100%)                  5 h 30&#730;C, mutant harboring murE expres- <br /> for DapL1 clade (56&#8211;99%). The variable maximum                   sion vector continued grow. The optical density vector <br /> likelihood bootstrap support alignment dependent alter-           culture declined attributed rapid lysis the <br /> native alignments techniques (Clustal vs. MUSCLE) exhibiting              cell lack proper peptidoglycan synthesis (Figure 7B). <br /> higher ML bootstrap values DapL1 DapL2 clades                 The assessment crude soluble protein extracts com- <br /> respectively. Muscle-aligned sequences generally exhibit higher           plementation experiment using SDS-PAGE analysis con&#65533;?rmed the <br /> bootstrap support DapL1 DapL2 clades                    expression recombinant MurE (59 kDa) TKL-11 cells <br /> Clustal.                                                                  harboring expression vector (pBAD33 + VsmurE) grown at <br />  <br />  <br />  <br />  <br />   FIGURE 4 | Functional complementation E. coli &#916;dapD/dapE (AOH1) mutant. The AOH1 mutant harboring <span id='am-600' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-601' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-602' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-603' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-604' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> pBAD33, pBAD33 + AtdapL, <br />   pBAD33 + CrdapL, pBAD33 +VsdapL replica-plated LB medium supplemented 0.2% (w/v) arabinose 50 &#956;g/mL <br />   L,L-diaminopimelate. For presentation purposes colony vector recombinant plasmids represented diagram. <br />  <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                May 2012 | Volume 3 | Article 183 | 5 <br />  Nachar et al.                                                                         Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />   FIGURE 5 | Schematic representation bacterial peptidoglycan           muramoylalanine-D-glutamate ligase), MurE (UDP-N-acetylmuramoyl- <br />   synthesis using m-DAP cross linking amino acids. The           L-alanyl-D-glutamate 2,6-meso-diaminopimelate ligase), MurF <br />   cytosolic, membrane, periplasmic steps shown. The abbreviations   (UDP-N-acetylmuramoyl-tripeptide-D-alanyl-d-alanine ligase), Ddl <br />   enzymes follows: MurA (UDP-N-acetylglucosamine enolpyruvyl        (D-alanine-D-alanine ligase B), MraY (phospho-N-acetylmuramoyl- <br />   transferase), MurB (UDP-N-acetylenolpyruvoylglucosaminereductase), MurC      pentapeptide-transferase), MurG (pyrophosphoryl-undecaprenol- <br />   (UDP-N-acetylmuramate-L-alanine ligase), MurD (UDP-N-acetyl-                 N-acetylglucosamine transferase), PBPs (penicillin binding proteins). <br />  <br />  <br />  <br /> 42&#730;C present extract TKL-11 harboring              (residues red bold), putative active site residues well <br /> vector control (pBAD33) grown 30&#730;C (Figure 7C).                        conserved DapL enzymes. In addition, loop <br />                                                                                regions form active site cleft generally con- <br /> THE AMINO ACIDS THAT CONSTITUTE THE ACTIVE SITE ARE                            served. In particular, loops A C conserved, although <br /> CONSERVED IN VsDapL                                                            loop B shows variability. From previous structural studies <br /> To examine closely VsDapL variant amino acid                   (Watanabe et al., 2007, 2011; Dobson et al., 2011), known <br /> level, aligned amino acid sequence DapL                loop B disordered role catalysis clear. <br /> orthologs functionally structurally characterized             However, noting increased substrate promiscuity displayed by <br /> Arabidopsis thaliana (Watanabe et al., 2007), C. reinhardtii (Dob-             VsDapL, possible loop B involved substrate <br /> son et al., 2011), C. trachomatis (Watanabe et al., 2011;                  binding step. <br /> Figure 8). The VsDapL sequence similarity                         A homology model VsDapL generated using the <br /> CrDapL At DapL sequences, like Ct DapL sequence,                   Swiss-Model Protein Modeling Server (Figure 9). The tertiary <br /> has truncated N-terminal region. The longer N-terminal regions               structure DapL monomer annotated hav- <br /> for eukaryotic sequences attributed subcellular localiza-            ing large small domain, includes arm <br /> tion sequences, used A. thaliana C. reinhardtii target               region (Figure 9A). Most aminotransferases approximately <br /> the enzyme chloroplast, lysine biosynthesis known              100 kDa mass, representing homodimeric quaternary struc- <br /> to occur (Mills Wilson, 1978). As seen Figure 8                  ture. Assuming functional unit VsDapL enzyme is <br />  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                          May 2012 | Volume 3 | Article 183 | 6 <br />  Nachar et al.                                                                        Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />  FIGURE 6 | Phylogenetic relationship DapL amino acid sequences.           Branch lengths proportional divergence; scale tree <br />  The VsDapL ortholog annotated black circle. Minimum evolution      indicates percent divergence branch lengths. The tree rooted <br />  (ME) topology shown maximum likelihood (ML), maximum                    single outgroup aspartate aminotransferase (AspC) from <br />  parsimony (MP), minimum evolution (ME), neighbor-joining (NJ)            Y. pseudotuberculosis. Figure drawing produced using FigTree (ver. 1.6.1; <br />  bootstrap values DapL1 DapL2 clades (from left right).   Rambaut, 2009). <br />  <br />  <br />  <br /> a dimer, required proper orientation conserved            essentiality dapL mutagenic experiments using transposon <br /> active site residues active site, VsDapL probably              homologous recombination. Moreover, organism is <br /> two active sites (Figure 9B). Closer examination active site           aerobic, relatively easy culture non-pathogenic humans, <br /> (Figure 9C) highlights residues conserved             unlike bacteria contain DapL pathway, Lep- <br /> multiple sequence alignment (Figure 8) key loop regions               tospira interrogans, Bacteriodes fragilis, C. trachomatis (McCoy <br /> that line active site. Importantly, model predicts loop          et al., 2006; Hudson et al., 2008). <br /> B positioned entrance active site cleft, ideally placed          Recently, laboratories demonstrated orthologous <br /> to interact substrates entering active site.                         dapL model plant Arabidopsis thaliana essen- <br />                                                                               tial growth development (Dobson et al., 2011). However, <br /> DISCUSSION                                                                    known true bacteria exclusively <br /> The discovery new biosynthetic route diaminopimelate                  utilize DapL pathway diaminopimelate/lysine synthesis. It <br /> and lysine plants, algae, bacteria provides fresh target         known aminotransferase capable inter- <br /> the discovery novel herbicides, algaecides, antibiotics. The           converting tetrahydrodipicolinate L,L-diaminopimelate exists <br /> amino acids constitute active site DapL conserved             bacteria solely contain DapL pathway. Thus, suit- <br /> in V. spinosum ortholog based protein alignment mod-               able dapL bacterial mutant auxotrophic diaminopime- <br /> eling. Therefore, inhibitors Ct DapL At DapL likely             late/lysine important step forward especially since <br /> inhibit VsDapL. To end, V. spinosum suitable bacterial              examples literature regarding substrate promis- <br /> system development vivo assays discovery             cuity aminotransferases involved amino acid metabolism. <br /> compounds able inhibit DapL facilitate antibiotic              For instance, aspartate aminotransferase, tyrosine amino- <br /> development.                                                                  transferase, branched-chain amino acid aminotransferase <br />     Since V. spinosum genetically altered using exogenous              shown possess overlapping activities E. coli (Gu <br /> DNA, V. spinosum excellent candidate determining                et al., 1998). In addition, recent study demonstrated three <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                       May 2012 | Volume 3 | Article 183 | 7 <br />  Nachar et al.                                                                           Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />                                                                                  supplemented diaminopimelate medium observe <br />                                                                                  growth phenotype. If inhibitory compound speci&#65533;?c for <br />                                                                                  DapL, expect effect the <br />                                                                                  mutant. However, compound effect on <br />                                                                                  wild-type parental strain used create dapL mutant. The <br />                                                                                  inhibition DapL prevent m-DAP production would <br />                                                                                  lead cell death caused lack proper protein cell wall <br />                                                                                  biosynthesis. <br />                                                                                     Here present elucidation biosynthetic pathway <br />                                                                                  diaminopimelate/lysine V. spinosum using genomic, bio- <br />                                                                                  chemical, structural approaches. The enzymes involved the <br />                                                                                  bacterial diaminopimelate/lysine biosynthetic pathways pre- <br />                                                                                  sumed targets antibiotic development. The identi&#65533;?cation and <br />                                                                                  characterization enzymes provides valuable information <br />                                                                                  pertaining experiments aiming elucidate essentiality of <br />                                                                                  genes involved bacterial diaminopimelate/lysine biosyn- <br />                                                                                  thesis pathways. The DapL pathway sole route toward <br />                                                                                  diaminopimelate/lysine V. spinosum. In addition, bacterium <br />                                                                                  genetically modi&#65533;?ed, organism relatively easy cul- <br />                                                                                  ture non-pathogenic animals. These criteria make V. spin- <br />                                                                                  osum excellent bacterial model elucidating essentiality <br />                                                                                  genes involved diaminopimelate/lysine metabolism. <br />  <br />                                                                                  MATERIALS AND METHODS <br />                                                                                  V. SPINOSUM GROWTH CONDITIONS <br />                                                                                  The plasmids strains used study listed Table 3. The <br />                                                                                  V. spinosum DSM 4136T organism cultured R2A medium <br />                                                                                  26&#730;C. <br />  <br />                                                                                  MULTIPLE-PROTEIN SEQUENCE ALIGNMENT, PHYLOGENETIC TREE <br />                                                                                  CONSTRUCTION, AND HOMOLOGY MODEL OF VsDapL <br />                                                                                  A DapL protein sequence alignment generated using the <br />   FIGURE 7 | Functional complementation E.coli murE mutant.               ClustalW server2 . Phylogenetic trees constructed MUS- <br />   (A) The E. coli mutant TKL-11 harboring pBAD33 pBAD33 +VsmurE. The <br />                                                                                  CLE aligned amino acid sequences using maximum likelihood <br />   bacteria grown OD 0.1 600 nm serially diluted to <br />   10&#8722;1 , 10&#8722;2 , 10&#8722;3 using 0.85% (w/v) saline. The bacteria             (ML), maximum parsimony (MP) minimum evolution (ME), <br />   replica-plated LB medium supplemented 0.2% (w/v) arabinose. The        neighbor-joining (NJ) techniques MEGA 5.05 (Tamura <br />   growth assessed 30 42&#730;C 24 h. Only colony        et al., 2011). ML searches performed using likelihood- <br />   vector recombinant plasmid used create diagram.            based WAG (Whelan Goldman, 2001) model amino acid <br />   (B) Growth curve E. coli TKL-11 harboring vector <br />                                                                                  character evolution, gamma distributed rates sites, and <br />   pBAD33 (circles), pBAD33 +VsmurE (triangles). The cells grown at <br />   30&#730;C 5 h shifted 42&#730;C additional &#65533;?ve hours (shaded area).   proportion invariant sites estimated <span id='am-582' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-583'  ></span><span id='am-584' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-585' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-586' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> set <br />   (C) SDS-PAGE analysis proteins extracted TKL-11 grown LB            (WAG + G + I). ME NJ tree searches performed using <br />   supplemented 0.2% arabinose (w/v). Lane (1) protein ladder (kDa),         parsimony-based JTT model (Jones et al., 1992) gamma <br />   Lane (2) 10 &#956;g soluble protein extract TKL-11 harboring pBAD33         distributed rates (JTT + G). Optimal models determined <br />   grown 30&#730;C, Lane (3) 10 &#956;g soluble protein extract harboring <br />   pBAD33 +VsmurE grown 42&#730;C. The proteins resolved 10% <br />                                                                                  MEGA 5.05 iteratively tests partition hierar- <br />   (w/v) acrylamide gel stained Coomassie blue                  chically nested models evolution obtain best &#65533;?t. Node <br />   visualization.                                                                 strength assessed using bootstrap technique 100 <br />                                                                                  pseudo-replicate data sets analysis type (ML, MP, ME, and <br />                                                                                  NJ). The accessions numbers following DapL proteins are: <br /> aminotransferases involved alanine biosynthesis E. coli                Arabidopsis thaliana (AEE86265), V. spinosum (ZP_02927470), <br /> (Yoneyama et al., 2011).                                                         C. reinhardtii (XP_001693061), C. trachomatis (NP_219900), <br />    Aminotransferases common genome V. spinosum                     <span id='am-587' about='obo:NCBITaxon_83551' typeof='obo:NCBITaxon_204428 owl:Thing obo:NCBITaxon_204429 obo:NCBITaxon_1 obo:NCBITaxon_92713 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_51290 obo:NCBITaxon_51291'><span id='am-588' property="rdfs:label" content="Parachlamydia" datatype="xsd:string"></span><span id='am-589' property="oboInOwl:hasDbXref" content="PMID:10319462" datatype="xsd:string"></span><span id='am-590' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-591' property="oboInOwl:hasDbXref" content="PMID:8979345" datatype="xsd:string"></span><span id='am-592' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-593'  ></span><span id='am-594' property="oboInOwl:hasRelatedSynonym" content="Candidatus Parachlamydia" datatype="xsd:string"></span>Parachlamydia</span> UWE25 (YP_007684), <span id='am-61' about='obo:NCBITaxon_1142' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_1301283 obo:NCBITaxon_1118 obo:NCBITaxon_1117'><span id='am-62' property="rdfs:label" content="Synechocystis" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Synechococystis" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-65'  ></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Synechocystis</span> sp. PCC 6803 <br /> given integral role multiple anabolic catabolic path-               (BAA10583), <span id='am-32' about='obo:NCBITaxon_146786' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_1301283 obo:NCBITaxon_146785 obo:NCBITaxon_1118 obo:NCBITaxon_1117'><span id='am-33' property="rdfs:label" content="Thermosynechococcus elongatus" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="PMID:11427679" datatype="xsd:string"></span>Thermosynechococcus elongatus</span> BP1 NP_682892), <br /> ways. If compound inhibits bacterial DapL vitro,                <span id='am-156' about='obo:NCBITaxon_272559' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_817 obo:NCBITaxon_200643 obo:NCBITaxon_815 obo:NCBITaxon_816 obo:NCBITaxon_171549 obo:NCBITaxon_976 obo:NCBITaxon_68336'><span id='am-157' property="rdfs:label" content="Bacteroides fragilis NCTC 9343" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasRelatedSynonym" content="Bacteroides fragilis NCTC9343" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasExactSynonym" content="Bacteroides fragilis str. NCTC 9343" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasExactSynonym" content="Bacteroides fragilis strain NCTC 9343" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasRelatedSynonym" content="Bacteroides fragilis ATCC 25285" datatype="xsd:string"></span>Bacteroides fragilis NCTC 9343</span> (YP_212286), Clostridium <br /> case Arabidopsis enzyme, question com-                thermocellum 27405 (YP_001039489), <span id='am-25' about='obo:NCBITaxon_145260' typeof='owl:Thing obo:NCBITaxon_2158 obo:NCBITaxon_2159 obo:NCBITaxon_1 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_183925 obo:NCBITaxon_28890'><span id='am-26'  ></span><span id='am-27' property="rdfs:label" content="Methanothermobacter" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Methanobacter" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="PMID:10826786" datatype="xsd:string"></span>Methanothermobacter</span> <br /> pound speci&#65533;?c DapL vivo investigated. <br /> One way answer question use vivo studies <br /> by exposing putative inhibitory compound dapL mutant                    2 http://www.ebi.ac.uk/Tools/msa/clustalw2/ <br />  <br />  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                              May 2012 | Volume 3 | Article 183 | 8 <br />  Nachar et al.                                                                        Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />   FIGURE 8 | Protein sequence alignment DapL V. spinosum, C.          loops line active site, asterisk refers loop <br />   trachomatis, A. thaliana, C. reinhardtii generated using ClustalW   contributing active site opposing monomer. The ClustalW scores <br />   server (http://www.ebi.ac.uk/Tools/msa/clustalw2/). Putative active-site    relative V. spinosum sequence were: vs. C. reinhardtii = 49 <br />   residues, based structural studies ligand bound At DapL   (length = 443 residues, identity); vs. A. thaliana = 45 (length = 426 residues); <br />   Cr DapL enzymes, shown red. The loop regions correspond key   vs. C. trachomatis = 39 (length = 394 residues). <br />  <br />  <br />  <br /> thermoautotrophicus (NP_275195), <span id='am-517' about='obo:NCBITaxon_28231' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_68525 obo:NCBITaxon_131567 obo:NCBITaxon_69541 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_213422 obo:NCBITaxon_28221'><span id='am-518' property="oboInOwl:hasAlternativeId" content="NCBITaxon:115782" datatype="xsd:string"></span><span id='am-519'  ></span><span id='am-520' property="oboInOwl:hasDbXref" content="PMID:8387263" datatype="xsd:string"></span><span id='am-521' property="oboInOwl:hasRelatedSynonym" content="Trichlorobacter" datatype="xsd:string"></span><span id='am-522' property="rdfs:label" content="Geobacter" datatype="xsd:string"></span><span id='am-523' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-524' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-525' property="oboInOwl:hasDbXref" content="PMID:17329769" datatype="xsd:string"></span><span id='am-526' property="oboInOwl:hasDbXref" content="PMID:10831402" datatype="xsd:string"></span>Geobacter</span> sulfurreducens                     (NP_927054), <span id='am-241' about='obo:NCBITaxon_2222' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_94695 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_224756 obo:NCBITaxon_143067 obo:NCBITaxon_28890'><span id='am-242'  ></span><span id='am-243' property="oboInOwl:hasRelatedSynonym" content="Methanotrix" datatype="xsd:string"></span><span id='am-244' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-245' property="rdfs:label" content="Methanosaeta" datatype="xsd:string"></span><span id='am-246' property="oboInOwl:hasDbXref" content="PMID:18599733" datatype="xsd:string"></span><span id='am-247' property="oboInOwl:hasDbXref" content="PMID:18599729" datatype="xsd:string"></span><span id='am-248' property="oboInOwl:hasRelatedSynonym" content="Methanothrix" datatype="xsd:string"></span><span id='am-249' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-250' property="oboInOwl:hasExactSynonym" content="Methanisaeta" datatype="xsd:string"></span>Methanosaeta</span> thermophila PT (YP_843230), <br /> (NP_951224), <span id='am-404' about='obo:NCBITaxon_171' typeof='owl:Thing obo:NCBITaxon_203691 obo:NCBITaxon_1 obo:NCBITaxon_203692 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_170 obo:NCBITaxon_136'><span id='am-405'  ></span><span id='am-406' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-407' property="oboInOwl:hasRelatedSynonym" content="Canela" datatype="xsd:string"></span><span id='am-408' property="oboInOwl:hasRelatedSynonym" content="Jequitaia" datatype="xsd:string"></span><span id='am-409' property="oboInOwl:hasAlternativeId" content="NCBITaxon:29514" datatype="xsd:string"></span><span id='am-410' property="oboInOwl:hasDbXref" content="PMID:10425768" datatype="xsd:string"></span><span id='am-411' property="oboInOwl:hasDbXref" content="PMID:19868121" datatype="xsd:string"></span><span id='am-412' property="oboInOwl:hasAlternativeId" content="NCBITaxon:29512" datatype="xsd:string"></span><span id='am-413' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-414' property="rdfs:label" content="Leptospira" datatype="xsd:string"></span><span id='am-415' property="oboInOwl:hasDbXref" content="PMID:9103653" datatype="xsd:string"></span><span id='am-416' property="oboInOwl:hasAlternativeId" content="NCBITaxon:29516" datatype="xsd:string"></span><span id='am-417' property="oboInOwl:hasRelatedSynonym" content="Ancona" datatype="xsd:string"></span><span id='am-418' property="oboInOwl:hasDbXref" content="PMID:1581182" datatype="xsd:string"></span>Leptospira</span> interrogans (YP_002757), <span id='am-82' about='obo:NCBITaxon_1177' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_1161 obo:NCBITaxon_1117 obo:NCBITaxon_1162'><span id='am-83' property="rdfs:label" content="Nostoc" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Amorphonostoc" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="PMID:1581185" datatype="xsd:string"></span><span id='am-87'  ></span><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Nostoc</span>                       <span id='am-226' about='obo:NCBITaxon_2203' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_2191 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_224756 obo:NCBITaxon_196137 obo:NCBITaxon_2202 obo:NCBITaxon_28890'><span id='am-227' property="oboInOwl:hasRelatedSynonym" content="Methanospirillum hungateii" datatype="xsd:string"></span><span id='am-228' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-229'  ></span><span id='am-230' property="oboInOwl:hasRelatedSynonym" content="Methanospirillum hungatii" datatype="xsd:string"></span><span id='am-231' property="rdfs:label" content="Methanospirillum hungatei" datatype="xsd:string"></span><span id='am-232' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-233' property="oboInOwl:hasDbXref" content="PMID:11540074" datatype="xsd:string"></span><span id='am-234' property="oboInOwl:hasDbXref" content="PMID:20008107" datatype="xsd:string"></span>Methanospirillum hungatei</span> JF1 (YP_504354), <span id='am-554' about='obo:NCBITaxon_61434' typeof='owl:Thing obo:NCBITaxon_1202465 obo:NCBITaxon_1202464 obo:NCBITaxon_1 obo:NCBITaxon_301297 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_200795'><span id='am-555' property="rdfs:label" content="Dehalococcoides" datatype="xsd:string"></span><span id='am-556' property="oboInOwl:hasDbXref" content="PMID:22544797" datatype="xsd:string"></span><span id='am-557' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-558' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-559'  ></span><span id='am-560' property="oboInOwl:hasDbXref" content="PMID:9171062" datatype="xsd:string"></span>Dehalococcoides</span> <br /> PCC 7120 (NP_488367), <span id='am-17' about='obo:NCBITaxon_251221' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_307596 obo:NCBITaxon_307595 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_33072 obo:NCBITaxon_33071 obo:NCBITaxon_1117'><span id='am-18' property="rdfs:label" content="Gloeobacter violaceus PCC 7421" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="Gloeobacter violaceus strain PCC 7421" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Gloeobacter violaceus ATCC 29082" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="PMID:14621292" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="Gloeobacter violaceus str. PCC 7421" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Gloeobacter violaceus PCC 7421</span>                          CBDB1 (YP_307791), <span id='am-251' about='obo:NCBITaxon_2225' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_2206 obo:NCBITaxon_94695 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_224756 obo:NCBITaxon_28890'><span id='am-252'  ></span><span id='am-253' property="oboInOwl:hasDbXref" content="PMID:16346215" datatype="xsd:string"></span><span id='am-254' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-255' property="oboInOwl:hasRelatedSynonym" content="Methylcoccoides" datatype="xsd:string"></span><span id='am-256' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-257' property="rdfs:label" content="Methanococcoides" datatype="xsd:string"></span>Methanococcoides</span> burtonii DSM 6242 <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                        May 2012 | Volume 3 | Article 183 | 9 <br />  Nachar et al.                                                                Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />   FIGURE 9 | VsDapL homology model generated                 N-terminal arm region compared Cr DapL At DapL. (B) Assuming <br />   Swiss-Model Protein Modeling Server (Arnold et al., 2006) using     dimeric structure, required place conserved active site <br />   Cr DapL apo structure template (PDB id: 3QGU Dobson        resides (highlighted red bold Figure 8), active sites the <br />   et al., 2011; Hudson et al., 2011a, BLAST identity = 51% (207/404   dimer shown. (C) A close view active site 1 showing the <br />   residues), QMEAN4 score = 0.673). (A) The domain structure   conserved active site residues loops lining active site (as <br />   monomer. Note that, seen Figure 8, VsDapL truncated     Figure 8). <br />  <br />  <br />  <br /> (YP_565702), <span id='am-210' about='obo:NCBITaxon_2207' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_2206 obo:NCBITaxon_94695 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_224756 obo:NCBITaxon_28890'><span id='am-211' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasDbXref" content="PMID:11536640" datatype="xsd:string"></span><span id='am-213' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-214' property="rdfs:label" content="Methanosarcina" datatype="xsd:string"></span><span id='am-215'  ></span>Methanosarcina</span> barkeri Fusaro (YP_306095),               appropriate geometry using visualization software COOT <br /> <span id='am-203' about='obo:NCBITaxon_2214' typeof='owl:Thing obo:NCBITaxon_2207 obo:NCBITaxon_1 obo:NCBITaxon_2206 obo:NCBITaxon_94695 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_224756 obo:NCBITaxon_28890'><span id='am-204' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-205' property="oboInOwl:hasRelatedSynonym" content="Methanosarcina acidovorans" datatype="xsd:string"></span><span id='am-206' property="oboInOwl:hasDbXref" content="PMID:16346552" datatype="xsd:string"></span><span id='am-207' property="rdfs:label" content="Methanosarcina acetivorans" datatype="xsd:string"></span><span id='am-208' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-209'  ></span>Methanosarcina acetivorans</span> C2A (NP_616639), Syntrophobac-             (Emsley Cowtan, 2004). <br /> ter fumaroxidans MPOB (YP_844192), <span id='am-391' about='obo:NCBITaxon_118' typeof='owl:Thing obo:NCBITaxon_112 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_203683 obo:NCBITaxon_203682 obo:NCBITaxon_126'><span id='am-392' property="oboInOwl:hasRelatedSynonym" content="Acinothrix" datatype="xsd:string"></span><span id='am-393'  ></span><span id='am-394' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-395' property="oboInOwl:hasDbXref" content="PMID:15653928" datatype="xsd:string"></span><span id='am-396' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-397' property="rdfs:label" content="Planctomyces" datatype="xsd:string"></span><span id='am-398' property="oboInOwl:hasRelatedSynonym" content="Blastocaulis" datatype="xsd:string"></span>Planctomyces</span> brasilien- <br /> sis DSM 5305 (YP_004269630), <span id='am-399' about='obo:NCBITaxon_120' typeof='owl:Thing obo:NCBITaxon_112 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_203683 obo:NCBITaxon_203682 obo:NCBITaxon_126 obo:NCBITaxon_118'><span id='am-400' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-401'  ></span><span id='am-402' property="rdfs:label" content="Planctomyces limnophilus" datatype="xsd:string"></span><span id='am-403' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Planctomyces limnophilus</span> DSM             PCR AMPLIFICATION AND CLONING OF THE V. SPINOSUM dapL AND <br /> 3776 (YP_003632005), <span id='am-9' about='obo:NCBITaxon_382464' typeof='owl:Thing obo:NCBITaxon_107707 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_74201 obo:NCBITaxon_388747 obo:NCBITaxon_51290 obo:NCBITaxon_48461 obo:NCBITaxon_203494'><span id='am-10' property="rdfs:label" content="Verrucomicrobiae bacterium DG1235" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Verrucomicrobiales bacterium DG1235" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="Verrucomicrobiae bacterium str. DG1235" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="Verrucomicrobiae bacterium strain DG1235" datatype="xsd:string"></span><span id='am-16'  ></span>Verrucomicrobiae bacterium DG1235</span>                murE OPEN READING FRAMES <br /> (ZP_05058547). The accession number Yersinia pseudo-          The length ORFs annotated locus tags VspiD_0101000 <br /> tuberculosis IP 32953 aspartate aminotransferase (AspC)            12510 (dapL, L,L-diaminopimelate aminotransferase) and <br /> (CAH20674).                                                           VspiD_010100019130 (murE, UDP-N -acetylmuramoylalanyl-d- <br />     A homology model VsDapL protein generated using        glutamyl-2,6-meso-diaminopimelate ligase) ampli&#65533;?ed by <br /> the Swiss-Model Protein Modeling Server (Arnold et al., 2006)         PCR. The ORFs ampli&#65533;?ed using: 12 pmol forward and <br /> using CrDapL apo structure template (PDB id: 3QGU;           reverse primers, 1 mM MgSO4 , 0.5 mM of four <br /> Dobson et al., 2011). The model examined hand clashes      deoxynucleotide triphosphates, 0.5 ng genomic DNA 1 unit <br />  <br />  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                May 2012 | Volume 3 | Article 183 | 10 <br />  Nachar et al.                                                                               Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br /> Table 2 | Activity VsDapL various amino donors amino                     Table 3 | Plasmids strains used study. <br /> acceptors. <br />                                                                                      Plasmid/strains                                 Vendor/Reference <br /> Amino donor                  Relative         Amino acceptor        Relative <br />                              activity (%)                           activity (%)     pET30A                                          Novagen, USA <br />                                                                                      pET100D/Topo                                    Invitrogen, USA <br /> L,L-diaminopimelate          100              2-Ketoglutarate       100              pBAD33                                          Guzman et al. (1995) <br /> meso-Diaminopimelate         1.6 &#177; 0.35       Pyruvate              8.1 &#177; 0.3        pBAD33 + AtdapL                                 Hudson et al. (2006) <br /> L-Lysine                     3.0 &#177; 0.2        Oxaloacetate          7.4 &#177; 0.15       pBAD33 + CrdapL                                 Dobson et al. (2011) <br /> L-Ornithine                  5.1 &#177; 0.2        Oxovalerate           7.9 &#177; 0.15       pET30A +VsdapL                                  This study <br />                                                                                      pET100D +VsmurE                                 This study <br /> The assay measures production dihydroquinazolium 440 nm using <br />                                                                                      pBAD33 +VsdapL                                  This study <br /> the ortho-aminobenzaldehyde assay. The assays pertaining amino <br />                                                                                      pBAD33 +VsmurE                                  This study <br /> donor/acceptor combination triplicates, relative activity based <br />                                                                                      Verrucomicrobium spinosum DSM 4136              Schlesner (1987) <br /> on activity enzyme L,L-diaminopimelate 2-ketoglutarate as <br />                                                                                      5-alpha competent cells                         New England Biolabs, USA <br /> substrates. <br />                                                                                      BL21 <span id='am-353' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-354' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-355' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-356' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-357'  ></span>codon</span> plus RIPL                            Agilent Technologies, USA <br /> of Platinum Pfx DNA polymerase (Invitrogen Corporation, Carls-                       AOH1 (&#916;dapD/dapE )                              Hudson et al. (2006) <br /> bad, CA, USA) using following PCR conditions: 1 cycle                         TKL-11 (murE )                                  Lugtenberg van <br /> 94&#730;C 2 min, followed 30 cycles 94&#730;C 15 s, 60&#730;C                                                                     Schijndel-van Dam (1972) <br /> 30 s 72&#730;C 2 min. The forward reverse primers used to <br /> amplify ORFs VsdapL F VsdapL R dapL ORF                         Table 4 | List Primers used PCR amplification, cloning and <br /> and VsmurE F VsmurE R murE ORF. The nucleotide                           nucleotide sequencing. <br /> sequences primers listed Table 4. The underlined <br /> sequence represents restriction enzyme sites used facilitate                  Primer Sequence (from 5  3  ) <br /> cloning ORF bolded italicized sequences repre- <br />                                                                                      VsdapL F         CCCCGAATTCATGGCCCTCATCAACGAAAACTTCCTCAAG <br /> sent initiation termination codons. For cloning, dapL PCR <br />                                                                                      VsdapL R         CCCCGTCGACCTACTTCAGCGCGGCGATACGGCGGCAGAC <br /> fragment digested EcoRI SalI ligated the <br />                                                                                      VsmurE F         CACCATGACCATTTTGCGCGATCTTATCGAGGGT <br /> plasmid pET30a (EMD Biosciences, Gibbstown, NJ, USA) pro- <br />                                                                                      VsmurE R         GTCGACTCACTGACGGTCATCCCTCCTTTGGCGTGC <br /> duce plasmid pET30a + VsdapL. The murE PCR fragment was <br />                                                                                      T7 promoter      TAATACGACTCACTATAGGG <br /> ligated plasmid pET100D/topo (Invitrogen Corporation, <br />                                                                                      T7 R             TAGTTATTGCTCAGCGGTGG <br /> Carlsbad, CA, USA) produce plasmid pE100D + VsmurE. <br />                                                                                      T7 terminator TATGCTAGTTATTGCTCAG <br /> The recombinant protein derived plasmid carries hexa- <br /> histidine tag derived pET30a pET100D plasmids                        The underlined sequences represent restriction enzyme sites.The bolded ital- <br /> amino terminus. To con&#65533;?rm &#65533;?delity PCR reactions,                      icized sequences denoted start stop codons open reading <br /> dapL ORF sequenced pET30a using T7 promoter                             frames (ORFs). <br /> and T7 terminator primer (Table 4). The murE ORF was <br /> sequenced pET100D using T7 promoter T7 R primer                         V. spinosum (pBAD33 + VsdapL), Arabidopsis thaliana <br /> (Table 4). Both dapL murE ORFs 100% identical                     (pBAD33 + AtdapL), C. reinhardtii (pBAD33 + CrdapL). <br /> sequences deposited Integrated Microbial Genomes (IMG)                        Transformants selected LB agar medium supplemented <br /> public database3 .                                                                   50 &#956;g mL&#8722;1 DAP 34 &#956;g mL&#8722;1 chloramphenicol. Ten <br />                                                                                      individual colonies vector recombinant <br /> FUNCTIONAL COMPLEMENTATION PLASMID CONSTRUCTS                                        plasmid replica-plated LB medium plus 0.2% <br /> The plasmids used functional complementation E.                           (w/v) arabinose 50 &#956;g mL&#8722;1 DAP. The cultures <br /> coli &#916;dapD/dapE double mutant E. coli murE mutant                            grown 30&#730;C 24 h. The E. coli murE mutant (TKL-11) <br /> were produced sub-cloning XbaI SalI fragment                         (thr-1, leuB6 (Am), murE1, fhuA21, codA1, lacY1, tsx-95, glnV44 <br /> pET30a + VsdapL pET100D + VsmurE plasmids                               (AS), &#955;&#8722; , pyrF101, his-108, thyA6, argG66, ilvA634, thi-1, deoC1) <br /> pBAD33 produce pBAD33 + VsdapL pBAD33 + VsmurE                                transformed pBAD33 pBAD33 + VsmurE (Table 3). <br /> (Guzman et al., 1995). The fusion proteins produced                         Transformants selected LB agar medium supplemented <br /> pBAD33 constructs identical proteins produced                        50 &#956;g mL&#8722;1 thymine 34 &#956;g mL&#8722;1 chloramphenicol at <br /> the pET30a pET100D constructs.                                                   30&#730;C. Ten individual colonies vector recombi- <br />                                                                                      nant plasmid transformation replica-plated LB medium <br /> FUNCTIONAL COMPLEMENTATION OF THE E. COLI dapD/E AND murE                            plus 0.2% (w/v) arabinose 50 &#956;g mL&#8722;1 thymine. The cul- <br /> MUTANTS                                                                              tures grown 30 42&#730;C 24 h assess functional <br /> The E. coli mutant AOH1 ( dapD::Kan2, dapE6 ) trans-                             complementation. <br /> formed pBAD33 plasmids harboring dapL orthologs <br />                                                                                      PROTEIN EXPRESSION AND PURIFICATION OF VsDapL <br />                                                                                      The E. coli BL21-CodonPlus-RIPL strain transformed with <br /> 3 http://img.jgi.doe.gov/cgi-bin/w/main.cgi                                          plasmid pET30a + VsdapL grown LB broth containing <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                            May 2012 | Volume 3 | Article 183 | 11 <br />  Nachar et al.                                                                                 Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br /> 50 &#956;g mL&#8722;1 kanamycin 34 &#956;g mL&#8722;1 chloramphenicol 37&#730;C                           MESO-DIAMINOPIMELATE DEHYDROGENASE ACTIVITY <br /> to OD600 0.5. Protein expression induced 1.0 L                        Ddh activity assessed using oxidative deamination. The assay <br /> culture using isopropyl &#946;-d-1-thiogalactopyranoside &#65533;?nal                         consisted 0.5 mM m-DAP, 0.5 mM NADP+ , 1 mg crude <br /> concentration 0.5 mM 6 h 20&#730;C. The cell pellet lysed                    soluble V. spinosum extract 10 &#956;g pure recombinant meso- <br /> by sonication solution 50 mM sodium phosphate (pH 8.0)                        diaminopimelate dehydrogenase C. thermocellum (Hudson <br /> and 300 mM NaCl. The soluble extract incubated 1 mL                          et al., 2011b) 100 mM <span id='am-271' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-272' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-273' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-274' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-275'  ></span>glycine</span>-KOH (pH 10.5) 100 mM <br /> bed volume Talon Metal Af&#65533;?nity Resin (Clontech, Mountain                           HEPES-KOH (pH 7.6) &#65533;?nal volume 0.5 mL. The produc- <br /> View, CA, USA) 30 min 4&#730;C. The resin washed &#65533;?ve                            tion NADPH measured continuously 340 nm 30&#730;C <br /> times 30 mL sonication buffer containing 10 mM imi-                           using Beckman DU640 spectrophotometer <br /> dazole (pH 8.0) 15 min each. The enzyme eluted with <br /> 10 mL sonication buffer containing 250 mM imidazole (pH                            ACKNOWLEDGMENTS <br /> 8.0). The pure protein concentrated Amicon Ultra                            This research supported United States National Science <br /> 10,000 molecular weight cutoff &#65533;?lter unit replacing elution                       Foundation (NSF) award AOH (MCB-1120541). Victoria R. <br /> buffer 100 mM HEPES-KOH containing 1 mM DTT, 2 mM                                Nachar supported Undergraduate Summer Research <br /> EDTA (pH 7.6). The recombinant protein stored 50%                              Fellowship College Science RIT. Victoria R. Nachar and <br /> glycerol. Protein concentration measured using Brad-                          Francisco C. Savka enrolled Biology Program SEM <br /> ford assay bovine serum albumin standard (Bradford,                       enrolled Bioinformatics Program RIT. Victoria R. Nachar, <br /> 1976).                                                                                Francisco C. Savka, Sean E. McGroty supported the <br />                                                                                       NSF award (MCB-1120541) Andr&#233; O. Hudson. Renwick C. J. <br /> L,L-DIAMINOPIMELATE AMINOTRANSFERASE ENZYME                                           Dobson acknowledges CR Roper Bequest Fellowship sup- <br /> ORTHO-AMINOBENZALDEHYDE ASSAY                                                         port, New Zealand Royal Society Marsden Fund (UOC1013) <br /> The ortho-aminobenzaldehyde assay measured formation                           funding support material based work sup- <br /> THDPA L,L-DAP using ortho-aminobenzaldehyde,                               ported U. S. Army Research Laboratory U. S. <br /> forms dihydroquinazolium adduct maximum absorbance                           Army Research Of&#65533;?ce contract/grant number W911NF-11- <br /> at 440 nm. The assay contained 100 mM HEPES-KOH (pH 7.6),                             1-0481. Katherine A. Donovan Rachel A. North acknowledge <br /> 0.5 mM amino donor, 2 mM amino acceptor 1.25 mM ortho-                            University Canterbury Summer Scholarship Scheme. Andr&#233; <br /> aminobenzaldehyde 10.0 &#956;g puri&#65533;?ed recombinant VsDapL                           O. Hudson thanks Naomi L. Ward form University Wyoming <br /> enzyme 0.5 mL reaction. The reactions incubated                          graciously providing V. spinosum strain/genomic DNA. The <br /> 30&#730;C change absorbance measured continuously                           authors thank Michael A. Savka (RIT) helpful discussions and <br /> at 440 nm using Beckman DU640 spectrophotometer.                                    revisions manuscript. <br />  <br /> REFERENCES                                    development. PLoS ONE 6, e20439.          novel diaminopimelate biosynthe-          Jones, D. T., Taylor,W. R., Thornton, <br /> Arnold, K., Bordoli, L., Kopp, J.,        doi:10.1371/journal.pone.0020439          sis pathway prokaryotes iden-             J. M. (1992). The rapidgeneration <br />    Schwede, T. (2006). The SWISS-          Domman, D. B., Steven, B. T.,            ti&#65533;?es diverged form L,L-                mutationdatamatrices pro- <br />    MODEL workspace: web-based               Ward, N. L. (2011). Random trans-         diaminopimelate aminotransferase.            teinsequences. Comput. Appl. Biosci. <br />    environment protein structure          poson mutagenesis Verrucomi-           J. Bacteriol. 190, 3256&#8211;3263.                8, 275&#8211;282. <br />    homology modeling. Bioinformatics          crobium spinosum DSM 4136(T).           Hudson, A. O., Gir&#243;n, I., Dobson,       Liu, Y., White, R. H., Whitman, <br />    22, 195&#8211;201.                               Arch. Microbiol. 193, 307&#8211;312.            R. C. J. (2011a). Crystallization        W. B. (2010). <span id='am-1' about='obo:NCBITaxon_183939' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_28890'><span id='am-2' property="oboInOwl:hasDbXref" content="PMID:11837318" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="rdfs:label" content="Methanococci" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Methanothermea" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="PMID:12054225" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Methanococci</span> use the <br /> Baizman, E. R., Branstrom, A. A., Lon-     Emsley, P., Cowtan, K. (2004). Coot:     preliminary X-ray diffraction analy-         diaminopimelate aminotransferase <br />    gley, C. B., Allanson, N., So&#65533;?a, M.        model-building tools molecular        sis L,L-diaminopimelate amino-            (DapL) pathway lysine biosyn- <br />    J., Gange, D., Goldman, R.             graphics. Acta Crystallogr. D Biol.       transferase (DapL) Chalmy-              thesis. J. Bacteriol. 192, 3304&#8211;3310. <br />    C. (2000). Antibacterial activity       Crystallogr. 60, 2126&#8211;2132.               domonas reinhardtii. Acta Crystal-        Lugtenberg, E. J. J., van Schijndel- <br />    synthetic analogues based        Fan, C., Clay, M. D., Dehyholos, M.          logr. Sect. F Struct. Biol. Cryst. Com-      van Dam, A. (1972). Temperature- <br />    disaccharide structure moeno-           K., Vederas, J. C. (2010).            mun. 67, 140&#8211;143.                            sensitive mutants <span id='am-500' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-501'  ></span><span id='am-502' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-503' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-504' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-505' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-506' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span>Escherichia</span> <br />    mycin, inhibitor bacterial           Exploration inhibitors           Hudson, A. O., Klartag, A., Gilvarg,           coli K-12 low activity of <br />    transglycosylase. Microbiology 146,        diaminopimelate aminotransferase.         C., Dobson, R. C. J., Marques, F.            diaminopimelic acid adding <br />    3129&#8211;3140.                                 Bioorg. Med. Chem. 18, 2141&#8211;2151.         G., Leustek, T. (2011b). Dual            enzyme. J. Bacteriol. 110, 41&#8211;46. <br /> Bradford, M. M. (1976). A rapid        Gu, W., Song, J., Bonner, C. A., Xie,        diaminopimelate biosynthesis path-        McCoy, A. J, Adams, N. E., Hud- <br />    sensitive method quanti&#65533;?ca-        G., Jensen, R. A. (1998). PhhC        ways <span id='am-472' about='obo:NCBITaxon_816' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_200643 obo:NCBITaxon_815 obo:NCBITaxon_171549 obo:NCBITaxon_976 obo:NCBITaxon_68336'><span id='am-473'  ></span><span id='am-474' property="oboInOwl:hasRelatedSynonym" content="Ristella" datatype="xsd:string"></span><span id='am-475' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-476' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-477' property="rdfs:label" content="Bacteroides" datatype="xsd:string"></span><span id='am-478' property="oboInOwl:hasRelatedSynonym" content="Capsularis" datatype="xsd:string"></span>Bacteroides</span> fragilis             son, A. O., Gilvarg, C., Leustek, T., <br />    tion microgram quantities pro-       essential aminotransferase      Clostridium thermocellum. Bochim.            Maurelli, A. T. (2006). L,L- <br />    tein utilizing principle pro-       aromatic amino acid metabolism         Biophys. Acta 1814, 1162&#8211;1168.               Diaminopimelate aminotransferase, <br />    tein dye binding. Anal. Biochem. 72,       <span id='am-375' about='obo:NCBITaxon_286' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621'><span id='am-376' property="oboInOwl:hasRelatedSynonym" content="Liquidomonas" datatype="xsd:string"></span><span id='am-377' property="oboInOwl:hasDbXref" content="PMID:7727274" datatype="xsd:string"></span><span id='am-378' property="oboInOwl:hasRelatedSynonym" content="Peudomonas" datatype="xsd:string"></span><span id='am-379' property="oboInOwl:hasDbXref" content="PMID:10758879" datatype="xsd:string"></span><span id='am-380' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-381' property="oboInOwl:hasDbXref" content="PMID:18048745" datatype="xsd:string"></span><span id='am-382' property="oboInOwl:hasDbXref" content="PMID:23918787" datatype="xsd:string"></span><span id='am-383' property="oboInOwl:hasDbXref" content="PMID:9103607" datatype="xsd:string"></span><span id='am-384'  ></span><span id='am-385' property="oboInOwl:hasRelatedSynonym" content="Loefflerella" datatype="xsd:string"></span><span id='am-386' property="oboInOwl:hasDbXref" content="PMID:10939664" datatype="xsd:string"></span><span id='am-387' property="oboInOwl:hasRelatedSynonym" content="RNA similarity group I" datatype="xsd:string"></span><span id='am-388' property="oboInOwl:hasDbXref" content="PMID:15950132" datatype="xsd:string"></span><span id='am-389' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-390' property="rdfs:label" content="Pseudomonas" datatype="xsd:string"></span>Pseudomonas</span> areuginosa. Micrbiol-       Hudson, A. O., Singh, B. K., Leustek,          trans-<span id='am-551' about='obo:NCBITaxon_kingdom' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-552' property="rdfs:label" content="kingdom" datatype="xsd:string"></span><span id='am-553' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>kingdom</span> enzyme shared <br />    248&#8211;254.                                   ogy 144, 3127&#8211;3134.                       T., Gilvarg, C. (2006). An L,L-          Chlamydia plants syn- <br /> Cox, R. J. (1996). The DAP pathway         Guzman, L. M., Belin, D., Carson, M. J.,     diaminopimelate aminotransferase             thesis diaminopimelate/lysine. <br />    lysine target antimi-          Beckwith, J. (1995). Tight reg-       de&#65533;?<span id='am-546' about='obo:NCBITaxon_203327' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_8220 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_8219 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489878 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_497223 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-547' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-548' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-549' property="rdfs:label" content="Nes" datatype="xsd:string"></span><span id='am-550'  ></span>nes</span> novel variant lysine         Proc. Natl. Acad. Sci. U.S.A. 103, <br />    crobial agents. Nat. Prod. Rep. 13,        ulation, modulation, high level       biosynthesis pathway plants. Plant        17909&#8211;17914. <br />    29&#8211;43.                                     expression vectors containing      Physiol. 140, 292&#8211;301.                    Mills, W. R., Wilson, K. G. <br /> Dobson, R. C. J., Gir&#243;n, I.,              arabinose pBAD promoter. J. Bacte-      Hutton, C. A., Perugini, M. A.,            (1978). Amino acid biosynthe- <br />    Hudson, A. O. (2011). L,L-                 riol. 177, 4121&#8211;4130.                     Gerrard, J. A. (2007). Inhibition            sis isolated pea chloroplast: <br />    diaminopimelate aminotransferase        Hudson, A. O., Gilvarg, C.,              lysine biosynthesis: evolving          metabolism labeled aspar- <br />          Chlamydomonas        rein-      Leustek, T. (2008). Biochemical       antibiotic strategy. Mol. Biosyst. 3,        tate sulfate. FEBS Lett. 92, <br />    hardtii: target algaecide            phylogenetic characterization        458&#8211;465.                                     129&#8211;132. <br />  <br />  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                                 May 2012 | Volume 3 | Article 183 | 12 <br />  Nachar et al.                                                                                   Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br /> Nishida, H. M., Nishiyama, N.,                  Sahm, H. (1991). A functionally            James, M. N. (2011). The struc-          conducted absence com- <br />     Kobayashi, T., Kosuge, T., Hoshimo,         split pathway lysine synthesis         ture L,L-diaminopimelate amino-       mercial &#65533;?nancial relationships that <br />     T., Yamane, H. (1999). A                <span id='am-134' about='obo:NCBITaxon_1716' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_1653'><span id='am-135' property="oboInOwl:hasDbXref" content="PMID:10425754" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasDbXref" content="PMID:7547293" datatype="xsd:string"></span><span id='am-139' property="rdfs:label" content="Corynebacterium" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasDbXref" content="PMID:7547284" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:hasDbXref" content="PMID:19661509" datatype="xsd:string"></span><span id='am-142'  ></span><span id='am-143' property="oboInOwl:hasDbXref" content="PMID:7547291" datatype="xsd:string"></span>Corynebacterium</span> glutamicum. J.          transferase Chlamydia tra-          construed potential con- <br />     prokaryotic gene cluster involved           Bacteriol. 173, 4510&#8211;4516.                 chomatis: implications broad     &#64258;ict interest. <br />     synthesis lysine           Tamura, K., Peterson, D., Peterson,           substrate speci&#65533;?city. J. Mol Biol. 19, <br />     amino adipate pathway:                N., Stecher, G., Nei, M.,              649&#8211;660.                                 Received: 14 January 2012; accepted: 02 <br />     key evolution amino               Kumar, S. (2011). MEGA5: mole-          Watanabe, N., Clay, M. D., van Belkum,      May 2012; published online: 25 May <br />     acid biosynthesis. Genome Res. 9,           cular evolutionary genetics analy-         M. J., Marcus, S. L., Flegel, M.         2012. <br />     1175&#8211;1183.                                  sis using maximum likelihood,              D., Clay, M. D., Deyholos, M.            Citation: Nachar VR, Savka FC, <br /> Rambaut, A. (2009). FigTree: Tree Figure        evolutionary distance, maxi-           K., Vederas, J. C., James, M.        McGroty SE, Donovan KA, North RA, <br />     Drawing Tool. Version 1.3.1, Insti-         mumparsimony. Mol. Biol. Evol. 28,         N. (2007). Crystal structure          Dobson RCJ, Buckley LJ Hudson <br />     tute Evolutionary Biology, Uni-          2731&#8211;2739.                                 L,L-diaminopimelate aminotrans-          AO (2012) Genomic biochemi- <br />     versity Edinburgh. Available at:      Velasco, A. M., Leguina, J. I., Laz-      ferase Arabidopsis thaliana:      cal analysis diaminopimelate <br />     http://tree.bio.ed.ac.uk/                   cano, A. (2002). Molecular evolution       recently discovered enzyme        lysine biosynthesis pathway in <br /> Sait, M., Kamneva, O. K., Fay, D. S.,           lysine biosynthetic pathways.       biosynthesis L-Lysine plants       Verrucomicrobium spinosum: identi&#65533;?- <br />     Kirienko, N. V., Polek, J., Shirasu-        J. Mol. Evol. 55, 445&#8211;459.                 Chlamydia. J. Mol. Biol. 371,        cation partial characterization of <br />     Hiza, M. M., Ward, N. L.             Wagner, M., Horn, M. (2006).              685&#8211;702.                                 L,L-diaminopimelate aminotransferase <br />     (2011). Genomic experimental            The <span id='am-532' about='obo:NCBITaxon_203682' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-533' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-534' property="oboInOwl:hasDbXref" content="PMID:15280311" datatype="xsd:string"></span><span id='am-535' property="rdfs:label" content="Planctomycetes" datatype="xsd:string"></span><span id='am-536' property="oboInOwl:hasDbXref" content="PMID:15143026" datatype="xsd:string"></span><span id='am-537' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-538'  ></span>Planctomycetes</span>, Verrucomi-          Whelan, S., Goldman, N. (2001).               UDP-N-acetylmuramoylalanyl- <br />     evidence suggests Verrucomi-           crobia, <span id='am-561' about='obo:NCBITaxon_204428' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_51290'><span id='am-562' property="oboInOwl:hasDbXref" content="PMID:24185849" datatype="xsd:string"></span><span id='am-563'  ></span><span id='am-564' property="oboInOwl:hasRelatedSynonym" content="&apos;Chlamydiae&apos;" datatype="xsd:string"></span><span id='am-565' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-566' property="rdfs:label" content="Chlamydiae" datatype="xsd:string"></span><span id='am-567' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Chlamydiae</span> sister              A general empirical model pro-        D-glutamyl-2,6-meso- <br />     crobium spinosum interacts             phyla comprise <span id='am-364' about='obo:NCBITaxon_superphylum' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-365' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-366' property="rdfs:label" content="superphylum" datatype="xsd:string"></span>superphylum</span>               tein evolution derived multiple     diaminopimelate ligase. Front. Microbio. <br />     eukaryotes. Front. Microbiol. 2:211.        biotechnological medical          protein families using maximum-        3:183. doi: 10.3389/fmicb.2012.00183 <br />     doi:10.33389/fmicb.2011.00211               relevance. Curr. Opin <br /> </body></html>