<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Younger Genes Are Less Likely Be Essential Older <br /> Genes, Duplicates Are Less Likely Be Essential than <br /> Singletons Same Age <br /> Wei-Hua Chen,1 Kalliopi Trachana,1 Martin J. Lercher,*,2 Peer Bork*,1 <br /> 1 <br />  Structural Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany <br /> 2 <br />  Institute Computer Science, Heinrich-Heine-University, Du&#168;sseldorf, Germany <br /> *Corresponding author: E-mail: bork@embl.de; lercher@cs.uni-duesseldorf.de. <br /> Associate editor: Arndt von Haeseler <br />  <br /> Abstract <br /> Recently duplicated genes believed overlap function expression. A priori, likely be <br /> essential. Although observed yeast, mouse singletons duplicates reported equally often <br /> essential. <span id='am-54' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-55' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-56'  ></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> contradiction partly explained experimental biases. We older genes <br /> (i.e., genes earlier phyletic origin) likely essential, regardless duplication status. At given <br /> phyletic gene age, duplicates likely essential compared singletons. The &#8216;&#8216;paradoxical&#8217;&#8217; high <br /> essentiality mouse gene duplicates caused different age profiles singletons duplicates, the <br /> latter tending derived older genes. <br /> Key words: gene essentiality, yeast, mouse, phyletic age, linking genotype phenotype. <br />  <br />  <br /> In model organisms mouse yeast, phenotypic                          cates statistically significantly lower in <br /> changes caused single-gene mutations assayed                         singletons (Su Gu 2008; Makino et al. 2009). <br /> a genome-wide scale (Kelly et al. 2001; Blake et al. 2011). Of                     However, authors did explore immediate <br /> particular essential genes, removal re-                      conclusions studies (Su Gu 2008; Makino <br /> sults death infertility. Many expressed genes perform-                    et al. 2009): 1) genes derived old duplications more <br /> ing important molecular functions nonessential. In                          likely essential singletons 2) developmental <br /> these cases, likely gene deletion partially               duplicates likely essential developmental <br /> compensated gene overlapping function                           singletons (or singletons whole). Both conclu- <br /> and expression.                                                                 sions hold true older latest versions <br />    Gene duplication believed important source                       mouse phenotype <span id='am-68' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72'  ></span>data</span> sets (table 1). This appears <br /> of functional redundancy (Ohno 1970). Accordingly,                         contradict duplication-functional redundancy <br /> the proportion essential genes (PE) duplicates                      concept, consider issue unresolved. <br /> much lower singletons yeast (Gu et al.                               What factors duplication status affect gene es- <br /> 2003). However, expected trend confirmed                           sentiality? Developmental genes likely essential <br /> in mouse, proportion essentials dupli-                       nondevelopmental genes (Makino et al. 2009), this <br /> cates comparable (Liao Zhang 2007) lower                         apply duplicates singletons alike. It also <br /> (table 1) singletons.                                                suggested hubs protein&#8211;protein interaction networks <br />    The contradicting results mouse initially inter-                     likely essential (Jeong et al. 2001); however, this <br /> preted evidence widespread functional redun-                         observation probably reflects biases proteins large <br />  <br />                                                                                                                                                      Letter <br /> dancy duplicates (Liao Zhang 2007);                                 essential protein complexes (Zotenko et al. 2008). <br /> interpretation hotly disputed (Su Gu 2008; Liang                           Previous studies indicated phyletic origin (age) <br /> and Li 2009; Makino et al. 2009). At time (Liao                        genes, defined evolutionarily distant <span id='am-59' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> <br /> Zhang 2007; Su Gu 2008; Liang Li 2009; Makino                           group homologs (Wolf et al. 2009), is <br /> et al. 2009), ;5,000 mouse genes tested                           correlated gene features (Hao et al. 2010). <br /> in knockout experiments. Biases expected                           Genes originated early tend conserved across <br /> subset mouse genes, genes known severe muta-                         species, highly broadly expressed, broadly useful <br /> tional phenotypes selected higher priority.                       (Hao et al. 2010). Thus, hypothesized knocking <br /> Two follow-up studies (Su Gu 2008; Makino et al. 2009)                      phyletically old genes likely severe phe- <br /> discovered knockout data enriched                      notypic effects: old genes essential. <br /> genes derived old duplications developmental                           To test <span id='am-42' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-43' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-46'  ></span>idea</span>, classified mouse yeast genes <br /> genes; correcting biases, overall PE dupli-                  different age groups according earliest phyletic <br />  <br /> &#169; The Author(s) 2012. Published Oxford University Press behalf Society Molecular Biology Evolution. <br /> This Open Access article distributed terms Creative Commons Attribution Non-Commercial License <br /> (http://creativecommons.org/licenses/by-nc/3.0), permits unrestricted non-commercial use, distribution, and <br /> reproduction medium, provided original work properly cited.                                                  Open Access <br /> Mol. Biol. Evol. 29(7):1703&#8211;1706. 2012 doi:10.1093/molbev/mss014 Advance Access publication January 19, 2012                                1703 <br />  Chen et al. &#183; doi:10.1093/molbev/mss014                                                                                                                                                                                                MBE <br /> Table 1. Proportions Essential Genes Different Gene                                                                                          true mouse yeast (fig. 1). Thus, duplicated <br /> Categories Two Phenotypical Data Sets Mouse.                                                                                            genes tend likely essential. Furthermore, <br />                                                                   Proportion Essential Genes (%)                                                singletons duplicated genes, fraction of <br />                                                                 Current                                 Data Makino                           essential genes increases increasing age; thus, older <br /> Categories                                                     Data Seta                                  et al. (2009)                            genes likely essential (fig. 1). The <br /> All genes                                                     43.3                                             42.07                               trends observed figure 1 reproduced restricting <br /> Duplicates                                                    43.9 (41.6b)                                     41.92                               analysis developmental genes nondeve- <br /> Singletons                                                    41.1                                             42.61                               lopmental genes (Supplementary figs. S1 S2, Supple- <br /> All developmental genes                                       62.53                                            59.51                               mentary Material online; raw data, see <br /> Developmental duplicates                                      64.75                                            60.9 <br /> Developmental singletons                                      53.1                                             53.36                               Supplementary table, Supplementary Material online). <br /> Old duplications (Ksc &#8225; 2)                                    47.31                                            44.94                                   Gene duplicates ages: age gene fam- <br /> a <br />   MGI 4.4 (October 2010). <br />                                                                                                                                                    ily (phyletic age; fig. 1) age recent du- <br /> b <br />   If genes valid phyletic ages used.                                                                                                 plication event (duplication age). The effect phyletic age <br /> c <br />   If gene multiple duplicates, pairwise Ks (the number synonymous                                                                     likely similar duplicates singletons. In addi- <br /> substitutions synonymous site) gene duplicates be <br /> calculated, lowest Ks value used. Synonymous substitutions most <br />                                                                                                                                                    tion, functional redundancy expected strongly af- <br /> genes Ks   2 reached saturation, hence, corresponding                                                                       fected age duplication event, duplicates <br /> genes tend older genes Ks , 2.                                                                                                derived ancient duplications likely es- <br />                                                                                                                                                    sential genes derived recent duplications (Su and <br />                                                                                                                                                    Gu 2008). In mouse gene duplicates, essentiality reaches <br /> origin (see Materials Methods). We classified genes                                                                                         plateau <span id='am-30' about='obo:NCBITaxon_4751' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-31' property="oboInOwl:hasExactSynonym" content="fungi" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="PMID:11062127" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Fungi" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="PMID:12684019" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="fungi" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Fungi</span>/<span id='am-73' about='obo:NCBITaxon_33208' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-74'  ></span><span id='am-75' property="oboInOwl:hasExactSynonym" content="multicellular animals" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasRelatedSynonym" content="animals" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasExactSynonym" content="metazoans" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Animalia" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Metazoa" datatype="xsd:string"></span>Metazoa</span> group does in- <br /> specific taxonomic groups yeast (fig. 1A)                                                                                       crease older age groups. According to <br /> and broad taxonomic groups mouse (fig. 1C).                                                                                                reasoning above, plateau caused com- <br /> Because large differences yeast mouse,                                                                                          parably young duplication age. We two <br /> we did attempt direct cross-species comparisons                                                                                            oldest groups contain higher fractions younger dupli- <br /> and did attempt map histories common                                                                                           cates &#8216;&#8216;Fungi/Metazoa&#8217;&#8217; group (Supplementary <br /> timescale.                                                                                                                                         fig. S3, Supplementary Material online). <br />    We age group, PE sin-                                                                                              In phyletic age group, duplicates likely to <br /> gletons higher duplicated genes;                                                                                         essential singletons (fig. 1). Why same <br />  <br />  <br />                              A                                                                                                                                                    B <br />                                       % essential genes <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                       10 20 30 40 <br />  <br />  <br />  <br />                                                                                                                                                              Singlets <br />                                                                   10 20 30 40 <br />  <br />  <br />  <br />  <br />                                                                                                                                                              Duplicates <br />                                                                   0 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                       0 <br />  <br />  <br />  <br />  <br />                                                                                   s                        ta                   oa                                        es                                  s                 es <br />                                                                                 sm                      yo                    az                ot                      al                                  et               at <br />                                                                                                   r                                         yc                       t                                gl <br />                                                                                               ka                         et              om                  y   ce                              Si <br />                                                                                                                                                                                                        n               pl <br />                                                                                                                                                                                                                           ic <br />                                                                rg                            Eu                          /M              c                                                                           Du <br />                                                              ro                                                    n   gi              As                   r om <br />                                                                                                                                                        ha <br />                                   lul                                                                           Fu <br />                                el                                                                                                                    cc <br />                              C                                                                                                                 Sa <br />  <br />                              C <br />                                       % essential genes <br />  <br />  <br />  <br />  <br />                                                                                                                                                              Singlets             D <br />                                                                                                                                                                                            40 <br />                                                                   50 <br />  <br />  <br />  <br />  <br />                                                                                                                                                              Duplicates <br />                                                                                                                                                                                            20 <br />                                                                   30 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                            0 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                  s                es <br />                                                                   10 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                              et                at <br />                                                                                                                                                                                                         n  gl                ic <br />                                                                                      s <br />                                                                                                 ot <br />                                                                                                                      oa              oa               at <br />                                                                                                                                                                            <span id='am-1' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-2'  ></span><span id='am-3' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>ia</span>                      Si                   pl <br />                                                                                 ism           ry                     az              az            rd                      al                                        Du <br />                                                                       n                                         et              et            o                     m <br />                                                                   rga                     uk                   i/M               M           Ch                  am <br />                                                              ro                          E                                                                      M <br />                                               la                                                          ng <br />                                  llu                                                               Fu <br />                              Ce <br />  <br /> FIG. 1. In yeast (A) mouse (C), genes recent phyletic origins likely essential, duplicated genes compared <br /> with singletons phyletic age. However, ignoring age, overall proportion essential genes singletons higher yeast (B) but <br /> lower mouse (D) compared duplications. Filled circles (C) indicate proportion essential genes corresponding <br /> duplication groups higher closer overall PE singletons (41.1%; dashed horizontal line); hollow circles indicate <br /> that PE lower. <br />  <br /> 1704 <br />  Younger Duplicated Genes Are Less Likely Be Essential &#183; doi:10.1093/molbev/mss014                                        MBE <br /> not true disregarding age, previous stud-              Materials Methods <br /> ies (Liao Zhang 2007; Su Gu 2008; Liang Li <br /> 2009; Makino et al. 2009)? This fact instance              We determined phyletic origins genes yeast, <br /> Simpson&#8217;s paradox (Simpson 1951), arise                      mouse, fly using method described Wolf et al. <br /> when dependence categorical variables (essen-               (2009) modifications (for details, Supple- <br />                                                                        mentary text, Supplementary Material online, for <br /> tiality duplication status) variable (phyletic <br />                                                                        results, Supplementary table, Supplementary Material <br /> age) disregarded. To illustrate mathematics behind <br />                                                                        online). We separated genes singletons duplicates <br /> this paradox, divided age groups duplicated <br />                                                                        previously described (Liao Zhang 2007; Makino et al. <br /> genes parts reflecting coarse <span id='am-67' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> of <br /> age: including age groups (the &#8216;&#8216;old part,&#8217;&#8217; filled           2009). We grouped duplicates gene families using <br /> circles fig. 1C) higher PEs               clustering-based method (Markov cluster algorithm <br /> overall singletons including remaining                   [MCL]; Enright et al. 2002) used ancient <br /> two groups lower PEs (the &#8216;&#8216;young part,&#8217;&#8217; open circles            origin members age corresponding <span id='am-39' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-40' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span>. <br />                                                                           We obtained phenotypic data species <br /> in fig. 1C). This partitioning results higher overall pro- <br />                                                                        online gene essentiality database (Chen et al. 2012), <br /> portion essential genes old duplicate part <br />                                                                        originally published <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-26'  ></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span> <br /> (Pold <br />    E 544:89%) higher compared overall sin- <br />                                                                        Genome Deletion Project (Cherry et al. 1997), Mouse <br /> gletons (Psingleton <br />              E       541:1%), corresponding <br /> proportion                young      duplicate             Genome Informatics (Blake et al. 2011), authors of <br /> (Pyoung 522:97%) lower. The overall PE duplicates re-            Chen et al. (2010), respectively. We restricted anal- <br />    E <br /> gardless age calculated weighted              yses genes tested phenotypic data <br /> average:                                                               sets. <br />  <br />             PE 5 fold   Pold <br />                          E   &#254; fyoung   Pyoung <br />                                          E     ;                       Supplementary Material <br /> where fold fyoung fraction duplicates contained         Supplementary text, Supplementary figures, Supple- <br /> in &#8216;&#8216;old&#8217;&#8217; &#8216;&#8216;young&#8217;&#8217; parts, respectively (with fold &#254; fyoung   mentary table available Molecular Biology and <br /> 5 1) (for details, Supplementary text Supple-             Evolution online (http://www.mbe.oxfordjournals.org/). <br /> mentary table, Supplementary Material online). In theory, <br /> the overall PE high 44.89% low as <br /> 22.97%, depending values fold fyoung. In             Acknowledgments <br /> study, vast majority duplicates de- <br /> rived old gene families (fold 5 84.66%), resulting          SysteMtb grant #241587 P.B. provided funding the <br /> overall PE 41.6% duplicates (see Supplementary table,           open access license. <br /> Supplementary Material online). Thus, surprising result <br /> of higher essentiality mouse duplicates compared               References <br /> with singletons caused different age profile single-        Blake JA, Bult CJ, Kadin JA, Richardson JE, Eppig JT. Mouse Genome <br /> tons duplicated gene families.                                         Database G. 2011. The Mouse Genome Database (MGD): <br />     Our results differ significantly recent publication             premier model organism resource mammalian genomics and <br /> on <span id='am-47' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-48' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-51'  ></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span>Drosophila melanogaster</span> (Chen et al. 2010). Based                    genetics. Nucleic Acids Res. 39:D842&#8211;D848. <br /> RNAi knockdowns ;440 genes, Chen et al.                  Chen S, Zhang YE, Long M. 2010. New genes Drosophila quickly <br /> ;30% young genes (,35 myr) essential compared                      essential. Science 330:1682&#8211;1685. <br />                                                                        Chen WH, Minguez P, Lercher MJ, Bork P. 2012. OGEE: online <br /> with ;35% old genes (.40 myr). The authors concluded <br />                                                                            gene essentiality database. Nucleic Acids Res. 40:D901&#8211;D906. <br /> that &#8216;&#8216;young genes essential old genes terms           Cherry JM, Ball C, Weng S, et al. (11 co-authors). 1997. Genetic <br /> viability&#8217;&#8217; (Chen et al. 2010). We reanalyzed data using             physical maps <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-21'  ></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>. Nature 387: <br /> our methods, differ Chen et al. age                 67&#8211;73. <br /> classification separate analysis duplicates          Enright AJ, Van Dongen S, Ouzounis CA. 2002. An efficient <br /> singletons (for raw data, Supplementary table,                     algorithm large-scale detection protein families. Nucleic <br /> Supplementary Material online). We                          Acids Res. 30:1575&#8211;1584. <br />                                                                        Gu Z, Steinmetz LM, Gu X, Scharfe C, Davis RW, Li WH. 2003. Role <br /> proportion essential genes singletons and <br />                                                                            duplicate genes genetic robustness null mutations. <br /> duplicates general increases increasing age                 Nature 421:63&#8211;66. <br /> five age groups, fluctuations (Supplementary                 Hao L, Ge X, Wan H, Hu S, Lercher M, Yu J, Chen W-H. 2010. Human <br /> fig. S4, Supplementary Material online). However, PE                   functional genetic studies biased medically most <br /> in duplicates lower singletons sim-               relevant primate-specific genes. BMC Evol Biol. 10:316. <br /> ilar age, differences statistically significant        Jeong H, Mason SP, Barabasi AL, Oltvai ZN. <br /> </body></html>