<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Mol Biochem Parasitol. 2013 July ; 190(1): 44&#8211;49. doi:10.1016/j.molbiopara.2013.05.002. <br />  <br />  <br />  <br />                            Identification Essential Non-essential Protein Kinases by <br />                            Fusion PCR Method Efficient Production Transgenic <br />                            <span id='am-7' about='obo:NCBITaxon_5690' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_2759 obo:NCBITaxon_5653'><span id='am-8' property="rdfs:label" content="Trypanosoma" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Trypanosoma</span> brucei <br />                            Christopher Merritt Kenneth Stuart1 <br />                            Seattle Biomedical Research Institute, Seattle, WA 98109, USA <br />  <br />                            Abstract <br />                                  Manipulation gene expression used elucidate gene function, explore fundamental <br />                                  biological processes identify potential drug targets <span id='am-12' about='obo:NCBITaxon_5691' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_39700 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_5690 obo:NCBITaxon_2759 obo:NCBITaxon_5653'><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Trypanosoma (Trypanozoon) brucei" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Trypanosoma brucei" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Trypanosoma brucei subgroup" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-18'  ></span>Trypanosoma brucei</span>. We in <br />                                  bloodstream forms CDC2-related kinase CRK12 (Tb11.01.4130) essential since <br />                                  transcriptional inactivation conditional null mutants lethal 19 protein kinases not <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  essential null mutants viable. We did using efficient methods generation null <br />                                  conditional null cell lines T. brucei approaches generate transfection constructs with <br />                                  large targeting sequences use reliable transfection selection conditions. These <br />                                  methods, described in supplementary material, employ multiple <br />                                  oligonucleotides PCR reactions transfections cost effective can <br />                                  simultaneously generate 24 transfectants shifting rate limiting experimental steps the <br />                                  production cell lines analysis. <br />  <br />  <br />                            Keywords <br />                                  Trypanosoma brucei; Protein kinases; Gene knockout; Conditional expression; Fusion PCR <br />                                  Method <br />  <br />                                             Manipulation gene expression powerful tool elucidating functions genes in <br />                                             context biological assessing potential gene products as <br />                                             drug targets. The functions numerous genes examined Trypanosoma brucei <br />                                             <span id='am-43' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-44' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> explore fundamental biological processes search drug targets. In <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             addition, diploid protozoan extracellular pathogen served experimental <br />                                             surrogate related intracellular kinetoplastid pathogens T. cruzi Leishmania <br />                                             highly homologous syntenic genomes experimental genetic <br />                                             manipulation developed T. brucei [1]. <br />  <br />                                             The robust homologous recombination naturally present T. brucei been <br />                                             exploited make gene knockouts (KOs) genome modifications. For gene KOs, <br />                                             cells transfected electroporation linear DNA construct contains an <br />                                             antibiotic resistance gene, i.e. selectable marker, flanking sequences target <br />                                             recombination results replacement endogenous gene selectable <br />                                             marker. The creation null mutant, i.e. cell alleles eliminated, a <br />                                             definitive way identify non-essential genes. Conditional RNA interference (RNAi) [1], by <br />                                             expression target gene repressed mRNA destruction tetracycline(tet)- <br />                                             dependant manner, commonly used T. brucei assess gene function and <br />  <br />  <br />                            1 <br />                             Corresponding author. Seattle Biomedical Research Institute, 307 Westlake Avenue North, Suite 500, Seattle, WA 98109-5219. <br />                            ken.stuart@seattlebiomed.org, Tel: 206-256-7302. <br />                             Merritt Stuart                                                                                            Page 2 <br />  <br />  <br />                                                 essentiality. RNAi cell line production entails cloning single RNAi construct and <br />                                                 single transfection step appropriate cell line. The ability efficiently generate <br />                                                 RNAi cell lines led systematic use assess genes chromosome [2] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 genome-wide analysis genes using deep sequencing population RNAi cell <br />                                                 lines [3]. An additional advantage RNAi approach mRNAs nearly <br />                                                 identical paralogs simultaneously targeted single construct. However, RNAi <br />                                                 disadvantages sufficiently reduce mRNA levels generate a <br />                                                 definitive phenotype off-target effects, i.e. degradation mRNA genes <br />                                                 intentionally targeted. Little known off-target RNAi effects T. <br />                                                 brucei, however, genetic approaches used verify RNAi acting on-target <br />                                                 [4]. We prepared null cell lines knocking endogenous alleles cells a <br />                                                 &#8220;conditional null&#65533;? cell line knocking endogenous alleles cells tet- <br />                                                 regulatable copy target gene&#8217;s coding DNA sequence (CDS) determine the <br />                                                 essentiality protein kinases. Expression knockdown robust conditional <br />                                                 nulls cell lines cell lines widely used preparation <br />                                                 requires multiple transfections cloning steps. However, improved efficiency of <br />                                                 production minimizing need prepare plasmid constructs improving <br />                                                 transfection selection conditions. <br />  <br />                                                 We employed efficient PCR approach make gene KO DNA constructs instead a <br />                                                 sequential PCR/cloning approach (e.g. [5&#8211;9]) time consuming has <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 pitfalls reflect limitations restriction enzyme digestion, DNA fragment <br />                                                 purification, ligation efficiency. A PCR approach used directly produce the <br />                                                 DNA constructs transfection single PCR reaction drug resistance markers with <br />                                                 long oligos (~100 nucleotides) contain targeting sequences gene-specific KOs (e.g. <br />                                                 [10] http://tryps.rockefeller.edu/trypsru2_genetics.html). We instead employed &#8220;fusion <br />                                                 PCR&#65533;? approach generate KO constructs sizeable homologous targeting sequences <br />                                                 (Fig. 1) use long oligos PCR adds significant cost effectively limits the <br />                                                 length homologous targeting region. <span id='am-33' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-34'  ></span><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> targeting region limit affect frequency <br />                                                 homologous recombination shown shorter targeting sequences result <br />                                                 lower recombination frequency T. brucei [11] seen organisms. <br />                                                 Hence, used long (~500 nt) targeting sequences potentially provide greater <br />                                                 recombination frequency, greater frequency transfection. These larger targeting <br />                                                 regions used fusion PCR approach enhance efficiency transfection <br />                                                 construct production. <br />  <br />                                                 The fusion PCR method used (Fig. 1) adapted used systems <br />                                                 (e.g. [12, 13]). Multiple PCR reactions used rapidly, inexpensively, effectively <br />                                                 generate DNA direct use transfections. Two separate PCR reactions used to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 amplify 5&#8242; 3&#8242; targeting sequences T. brucei genomic DNA gene. The inner <br />                                                 oligos contain sequence complementary outer sequences selectable <br />                                                 marker PCR amplified bulk plasmid DNA gel purified. The 5&#8242;, 3&#8242; <br />                                                 selectable marker PCR products combined removal PCR primers using <br />                                                 commercial spin column (Qiagen) amplified PCR reaction produces <br />                                                 fusion PCR product transfection. Six unique oligos needed create fusion <br />                                                 product KO allele (Fig 1). For 1st allele inner oligos (oligos B, D) <br />                                                 contain sequences complement 1st selectable marker, outer oligos <br />                                                 (oligos A, F) provide amplification ~500 nts targeting sequence flanking gene <br />                                                 CDS, oligos (G, H) nested (i.e. internal oligos A F) specificity <br />                                                 create fusion PCR product used transfection. Two additional <br />                                                 oligos (oligos C, E) needed contain sequences complementary 2nd <br />                                                 selectable marker generate PCR product used fusion PCR to <br />  <br />  <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                               Page 3 <br />  <br />  <br />                                                 generate construct KO 2nd allele. The original outer oligos the <br />                                                 nested PCR (i.e. oligos A, F, G, H). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 The fusion PCR transfection constructs alleles manner the <br />                                                 targeting sequences. This result construct 2nd allele KO recombining <br />                                                 site 1st allele KO. However use drugs combination select <br />                                                 cells making approach suitable high throughput studies. Nevertheless, <br />                                                 recombination 2nd allele KO construct site 1st allele KO (and does) <br />                                                 occur. This eliminate 2nd endogenous allele insert site 2nd <br />                                                 selectable marker retain replace 1st selectable marker. This be <br />                                                 averted detailed supplementary material using oligo pairs internal A, <br />                                                 B D, F PCR amplify targeting sequences 2nd selectable marker. This <br />                                                 results need additional oligos awareness possible consequences to <br />                                                 adjacent genes (e.g. deletion important regulatory coding sequences). Thus, minimally <br />                                                 eight, maximally ten, oligos needed KO alleles, the <br />                                                 number used long oligo KO approach. However, total cost ten <br />                                                 oligos fraction cost longer ones smaller size. Currently total <br />                                                 reagents costs fusion PCR 3&#8211;4 times single PCR method using <br />                                                 long oligos substantially greater cost long oligos. In addition, the <br />                                                 larger flanking gene targeting regions fusion PCR products provide higher <br />                                                 recombination transfection efficiencies. The oligos designed similar <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 annealing temperatures PCR products generated same <br />                                                 conditions. This PCR protocol scaled 96-well format enables <br />                                                 simultaneous production PCR products KO 12&#8211;24 genes (see supplementary <br />                                                 materials details). More PCR products produced time (e.g. 48&#8211;96 genes) <br />                                                 subsequent steps transfection, cell maintenance confirmation desired <br />                                                 transfections effectively limit production 24 gene KOs time person. <br />  <br />                                                 Fusion PCR products prepared used KO target genes T. brucei attempt to <br />                                                 create null mutants, i.e. eliminate endogenous alleles, 20 predicted protein kinases. <br />                                                 The creation null mutants definitively demonstrate eliminated gene not <br />                                                 essential viability. A conditional null mutant produced null mutants not <br />                                                 obtained. The protein kinases chosen primarily members Sterile (STE) and <br />                                                 Other protein kinase groups thoroughly studied genetically T. brucei. <br />                                                 Null mutants generated higher-throughput fusion PCR transfection method <br />                                                 using different selectable markers sequentially KO endogenous copies target <br />                                                 genes (see supplementary materials KO schematic). These constructs designed to <br />                                                 precisely delete target gene start stop <span id='am-38' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-39' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span>. One endogenous allele was <br />                                                 eliminated transfecting purified fusion PCR products electroporation bloodform <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 (BF) T. brucei single marker 427 cells (SM427) [14]. The SM427 cell line expresses T7 <br />                                                 polymerase Tet repressor proteins &#946; tubulin locus (maintained NEO <br />                                                 selectable marker) control tet conditional expression transgenes [14]. The <br />                                                 electroporation PCR fusion product using approximately 2&#215;107 mid log <br />                                                 phase SM427 cells outlined supplementary materials. Several conditions affect <br />                                                 transfection efficiency generation KO cell lines. These include amount <br />                                                 quality DNA used, length targeting sequences, number, growth <br />                                                 phase strain cells, transfection device settings used, selectable <br />                                                 markers, concentrations plating conditions used selection for <br />                                                 transfectants. Many variables examined previously ([11, 15]) few <br />                                                 addressed herein. We choice transfection device used greatly affects <br />                                                 transfection frequency ([16], http://tryps.rockefeller.edu/ <br />                                                 trypsru2_protocols_index.html). More importantly, drug resistance <br />  <br />  <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                              Page 4 <br />  <br />  <br />                                                 markers used concentrations drugs used selection critical reliably <br />                                                 obtaining KO, especially null, cell lines. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 An average 58 independent clones (determined limiting dilution) 1st allele <br />                                                 knocked obtained ~2&#215;107 cells using ~200&#8211;600ng PCR fusion product. <br />                                                 For 20 genes analyzed, range order magnitude. No correlation was <br />                                                 observed number clones obtained PCR product used <br />                                                 200ng, length target gene&#8217;s ORF kb (<span id='am-60' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62'  ></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> shown). <br />                                                 Transfections similar conditions using blasticidin, puromycin, phleomycin <br />                                                 selectable markers (BSD, PAC, BLE) resulted 1st allele KOs numbers of <br />                                                 clones obtained different genes different drug concentrations not <br />                                                 analyzed. Success rates 2nd allele KOs similar conditions 15&#8211;140 fold lower. <br />                                                 Success rates 2nd allele KOs genes lower tet-regulatable <br />                                                 copy expressed (data shown). Thus lower success rates 2nd allele KOs be <br />                                                 explained solely response elimination gene expression. Electroporation using <br />                                                 Amaxa Nucleofection device, higher transfection efficiency BTX <br />                                                 device ([16], http://tryps.rockefeller.edu/trypsru2_protocols_index.html), resulted a <br />                                                 greater number 2nd allele KOs (7&#8211;30 fold more). The lower number transfectants of <br />                                                 2nd versus 1st allele KOs increased selection consequence absence <br />                                                 allele, potential effects expression genes (e.g. cluster) of <br />                                                 gene KO, competition recombination 1st allele KO locus, and/or the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 effect using multiple antibiotics. Competition recombination reduced not <br />                                                 eliminated using sequences fusion PCR product 2nd allele been <br />                                                 eliminated 1st allele KO outlined (see supplementary material). <br />  <br />                                                 The choice selectable marker, drug concentration, conditions selection were <br />                                                 significant variables affecting frequency allele KO clones obtained, <br />                                                 especially 2nd allele KO (see supplementary material). Reducing drug concentrations <br />                                                 generally used selection T. brucei (5 &#956;g/ml hygromycin, 5 &#956;g/ml <br />                                                 blasticidin, 0.1 &#956;g/ml puromycin, 2.5 &#956;g/ml phelomycin) (http://tryps.rockefeller.edu/ <br />                                                 trypsru2_genetics.html) omitting drug selection time resulted a <br />                                                 greater frequency obtaining KO cell clones (see supplementary material). This suggests <br />                                                 expression levels drug resistance markers governed target genes <br />                                                 endogenous UTRs ideal selection depending selectable marker. In <br />                                                 support this, providing selectable marker gene UTRs support <br />                                                 higher levels expression provided wider window drug selection level of <br />                                                 expression endogenous gene UTRs limiting, lower levels of <br />                                                 expression (see supplementary material). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 To verify desired gene KOs, genomic DNA clones obtained 1st 2nd allele <br />                                                 transfection genotyped using diagnostic oligos qRT-PCR and, cases, by <br />                                                 PCR gel analysis. All clones obtained attempting KO 1st allele (with HYG) <br />                                                 of 20 protein kinases showed loss endogenous coding sequence (CDS) <br />                                                 copy qRT-PCR (Fig. 2). Attempts KO 2nd allele (with BSD PAC) 20 1st <br />                                                 allele KO cell lines resulted 46/47 clones (multiple clones genes) both <br />                                                 endogenous CDSs absent (Fig. 2, data shown). However, unable to <br />                                                 knockout second CRK12 allele (see below). We did analyze cell line <br />                                                 resistant selectable markers copy target gene CDS. <br />                                                 Several KO cell lines examined PCR using oligos outside targeting region <br />                                                 subsequent gel analysis showed presence bands consistent replacement of <br />                                                 target CDS selectable markers (Fig. 2B, data shown). These data suggest <br />                                                 approach described infrequently generate cell lines resistant <br />                                                 selectable marker contain desired 2nd allele KO events as <br />  <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                                Page 5 <br />  <br />  <br />                                                 mis-targeting gene duplication. This minimizes time needed confirm identity of <br />                                                 desired clones transfection. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Identification genes essential viability order determine their <br />                                                 function potential drug targets. Stable null cell lines obtained such <br />                                                 genes genetic compensation. Attempts knock 2nd endogenous <br />                                                 allele essential gene result viable stable cell lines multiple <br />                                                 attempts, repeatedly observed (data shown), result in <br />                                                 unintended genetic changes. Stable cell lines obtained attempts resistant <br />                                                 drugs encoded 1st 2nd KO selectable markers retain copy gene <br />                                                 CDS mis-integrating selectable marker off-target locus integrating the <br />                                                 targeted locus loss resident allele (e.g. result gene duplication <br />                                                 event). For example, CK1.2 (Tb927.5.800) strong growth defect RNAi BFs and <br />                                                 attempts create null cell lines unsuccessful suggesting essential <br />                                                 [8]. Our multiple attempts create null cell lines CK1.2 resulted cell lines that <br />                                                 contained resistance markers contained copy CK1.2 CDS, consistent <br />                                                 gene essential growth BF T. brucei (data shown). <br />  <br />                                                 Conditional null cell lines generated study genes essential viability. By <br />                                                 approach, tet-regulatable copy gene CDS inserted transcriptionally <br />                                                 silent rRNA intergenic locus SM427 T. brucei cells prior KO endogenous <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 alleles. For efficiency, tet-regulatable copy introduced cells 1st <br />                                                 allele knocked simultaneously parallel attempts generate null <br />                                                 mutant. The 2nd endogenous allele knocked fusion PCR product the <br />                                                 2nd allele KO selection performed media contains tet ensure expression the <br />                                                 regulatable copy rRNA intergenic region. Proper gene insertion assayed PCR as <br />                                                 indicated tet-regulation assayed monitoring cell growth presence <br />                                                 (gene expressed) absence tet (gene repressed) monitoring mRNA knockdown with <br />                                                 qPCR northern blot. Expression regulatable copy monitored western <br />                                                 analysis antibody available protein epitope tag has <br />                                                 fused regulatable copy. In addition, level mRNA protein knockdown <br />                                                 controlled moderate range adjusting level tet cell grown <br />                                                 media. <br />  <br />                                                 The tet-regulatable copy gene CDS cloned <span id='am-56' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> that <br />                                                 widely available (see http://www.zbh.uni-heidelberg.de/clayton/vectors.html, http:// <br />                                                 tryps.rockefeller.edu/trypsru2_plasmids.html references supplementary materials). <br />                                                 Insertion CDS appropriate tet-regulatable vector performed by <br />                                                 traditional ligation cloning restriction enzyme sites designed CDS PCR oligos or <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 efficiently vitro recombination using Gateway (Invitrogen) the <br />                                                 recombinase-specific sequence tails added oligos amplify CDS. To expedite <br />                                                 cloning regulatable transgenes simultaneously, modified commonly <br />                                                 used regulatable expression vector (pLEW100v5) gene CDS inserted via <br />                                                 vitro recombination (see supplementary materials). <br />  <br />                                                 Using scheme outlined supplementary materials, created conditional <br />                                                 null cell lines BF procyclic form (PF) T. brucei including CDC2-related <br />                                                 kinase CRK12 (Tb11.01.4130) BF cells (Fig. 3) previously shown to <br />                                                 growth defect RNAi [17]. This cell line created SM427 cells which <br />                                                 express tet repressor protein necessary regulation CDS allele the <br />                                                 rRNA locus. For cell line, fusion PCR products constructed described were <br />                                                 used KO endogenous alleles Gateway version pLEW100v5 used to <br />                                                 produce tet-regulatable copy (Fig. 3A). The genotype cell line verified by <br />  <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                                               Page 6 <br />  <br />  <br />                                                 qRT-PCR genomic DNA template (Fig. 3B). Growth cells monitored over <br />                                                 days media containing tet (regulatable CRK12 on) lacking tet (regulatable <br />                                                 CRK12 off) showed strong growth phenotype cells regulatable copy <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 longer expressed (Fig. 3C). Growth resumed days presumably loss <br />                                                 repression regulatable allele rRNA locus means have <br />                                                 resulted growth recovery. However, similar loss repression seen other <br />                                                 conditional null cell lines [18&#8211;20] shown correspond mutations that <br />                                                 result loss tet repressor coding sequences [19]. <br />  <br />                                                 The method described supplementary materials enable efficient analysis of <br />                                                 gene essentiality gene function T. brucei. It does substituting PCR gene <br />                                                 cloning substantial extent preparing numerous transfection constructs and <br />                                                 performing multiple transfections simultaneously. In way numerous gene KO cell lines <br />                                                 conditional cell lines rapidly generated. In addition, variations described that <br />                                                 increase utility conditional null cell lines. We selectable <br />                                                 drug markers used selection conditions important factors obtaining KO cells <br />                                                 lines (see supplementary materials). The number selectable drug markers used for <br />                                                 cell line construction reduced using KO method. One approach to <br />                                                 employ loss-of-heterozygosity increasing drug pressure select recombinants in <br />                                                 inserted drug marker single allele gene KO converts wild type allele <br />                                                 mitotic recombination eliminating it. This approach used infrequently <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 T. brucei gene conversion occurs frequently association antigenic <br />                                                 variation. Selectable markers excised integration allow reuse in <br />                                                 subsequent transfectants. This accomplished generating KO cell lines PCR <br />                                                 products contain positive selectable marker (i.e. HYG, PAC, BSD, BLE) fused a <br />                                                 negative selectable marker (HSVTK CDS) flanked loxP sites. Excision be <br />                                                 accomplished transient transfection Cre recombinase selection HSVTK <br />                                                 protein addition Ganciclovir media (see references details supplementary <br />                                                 materials). <br />  <br />                                                 Overall, used fusion PCR-based KO technique stringently assess gene essentiality in <br />                                                 BF T. brucei producing null conditional null cell lines. Obtaining 19 null cell lines for <br />                                                 protein kinases illustrates efficiency 6 nulls protein kinases BF T. brucei <br />                                                 reported date [5&#8211;9]. The methods used provide alternative greater <br />                                                 stringency conditional RNAi analysis essential genes. Additional <br />                                                 methodological details provided supplementary material. <br />  <br />                               Supplementary Material <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                                 We thank Ngoc Tran technical assistance Jef Boeke oligo ordering advice. This work supported by <br />                                                 National Institutes Health grant AI078962 studies initiated support National Institutes <br />                                                 Health grant AI075641. <br />  <br />  <br />                               References <br />                                                 1. Kolev NG, Tschudi C, Ullu E. RNA interference protozoan parasites: achievements and <br />                                                    challenges. Eukaryot Cell. 2011; 10:1156&#8211;63. [PubMed: 21764910] <br />                                                 2. Subramaniam C, Veazey P, Redmond S, Hayes-Sinclair J, Chambers E, Carrington M, et al. <br />                                                    Chromosome-wide analysis gene function RNA interference african trypanosome. <br />                                                    Eukaryot Cell. 2006; 5:1539&#8211;49. [PubMed: 16963636] <br />  <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                                  Page 7 <br />  <br />  <br />                                                 3. Alsford S, Turner DJ, Obado SO, Sanchez-Flores A, Glover L, Berriman M, et al. High-throughput <br />                                                    phenotyping using parallel sequencing RNA interference targets African trypanosome. <br />                                                    Genome Res. 2011; 21:915&#8211;24. [PubMed: 21363968] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 4. Ringpis GE, Lathrop RH, Aphasizhev R. iCODA: RNAi-based inducible knock-in in <br />                                                    Trypanosoma brucei. Methods Mol Biol. 2011; 718:23&#8211;37. [PubMed: 21370040] <br />                                                 5. Domenicali Pfister D, Burkard G, Morand S, Renggli CK, Roditi I, Vassella E. A Mitogen-activated <br />                                                    protein kinase controls differentiation bloodstream forms Trypanosoma brucei. Eukaryot Cell. <br />                                                    2006; 5:1126&#8211;35. [PubMed: 16835456] <br />                                                 6. Jensen BC, Kifer CT, Parsons M. Trypanosoma brucei: Two mitogen activated protein kinase <br />                                                    kinases dispensable growth virulence bloodstream form. Exp Parasitol. 2011; <br />                                                    128:250&#8211;5. [PubMed: 21396364] <br />                                                 7. Muller IB, Domenicali-Pfister D, Roditi I, Vassella E. Stage-specific requirement mitogen- <br />                                                    activated protein kinase Trypanosoma brucei. Mol Biol Cell. 2002; 13:3787&#8211;99. [PubMed: <br />                                                    12429824] <br />                                                 8. Urbaniak MD. Casein kinase 1 isoform 2 essential bloodstream form Trypanosoma brucei. <br />                                                    Mol Biochem Parasitol. 2009; 166:183&#8211;5. [PubMed: 19450734] <br />                                                 9. Vassella E, Kramer R, Turner CM, Wankell M, Modes C, van den Bogaard M, et al. Deletion a <br />                                                    novel protein kinase PX FYVE-related domains increases rate differentiation of <br />                                                    Trypanosoma brucei. Mol Microbiol. 2001; 41:33&#8211;46. [PubMed: 11454198] <br />                                                 10. Gaud A, Carrington M, Deshusses J, Schaller DR. Polymerase chain reaction-based gene <br />                                                      disruption Trypanosoma brucei. Mol Biochem Parasitol. 1997; 87:113&#8211;5. [PubMed: 9233680] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 11. Barnes RL, McCulloch R. Trypanosoma brucei homologous recombination dependent on <br />                                                      substrate length homology, displays differential dependence mismatch repair as <br />                                                      substrate length decreases. Nucleic Acids Res. 2007; 35:3478&#8211;93. [PubMed: 17478508] <br />                                                 12. Szewczyk E, Nayak T, Oakley CE, Edgerton H, Xiong Y, Taheri-Talesh N, et al. Fusion PCR and <br />                                                      gene targeting <span id='am-1' about='obo:NCBITaxon_5052' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5042 obo:NCBITaxon_33154 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-2' property="oboInOwl:hasAlternativeId" content="NCBITaxon:37503" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="rdfs:label" content="Aspergillus" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Aspergillus</span> nidulans. Nat Protoc. 2006; 1:3111&#8211;20. [PubMed: 17406574] <br />                                                 13. Noble SM, Johnson AD. Strains strategies large-scale gene deletion studies diploid <br />                                                      human fungal pathogen <span id='am-19' about='obo:NCBITaxon_5475' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Torulopsis" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Candida" datatype="xsd:string"></span><span id='am-23'  ></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Candida</span> albicans. Eukaryot Cell. 2005; 4:298&#8211;309. [PubMed: 15701792] <br />                                                 14. Wirtz E, Leal S, Ochatt C, Cross GA. A tightly regulated inducible expression for <br />                                                      conditional gene knock-outs dominant-negative genetics Trypanosoma brucei. Mol <br />                                                      Biochem Parasitol. 1999; 99:89&#8211;101. [PubMed: 10215027] <br />                                                 15. Wirtz E, Hartmann C, Clayton C. Gene expression mediated bacteriophage T3 T7 RNA <br />                                                      polymerases transgenic trypanosomes. Nucleic Acids Res. 1994; 22:3887&#8211;94. [PubMed: <br />                                                      7937108] <br />                                                 16. Burkard G, Fragoso CM, Roditi I. Highly efficient stable transformation bloodstream forms of <br />                                                      Trypanosoma brucei. Mol Biochem Parasitol. 2007; 153:220&#8211;3. [PubMed: 17408766] <br />                                                 17. Mackey ZB, Koupparis K, Nishino M, McKerrow JH. High-throughput analysis RNAi <br />                                                      library identifies novel kinase targets Trypanosoma brucei. Chem Biol Drug Des. 2011; 78:454&#8211; <br />                                                      63. [PubMed: 21668652] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 18. Martin KL, Smith TK. The glycosylphosphatidylinositol (GPI) biosynthetic pathway of <br />                                                      bloodstream-form Trypanosoma brucei dependent novo synthesis inositol. Mol <br />                                                      Microbiol. 2006; 61:89&#8211;105. [PubMed: 16824097] <br />                                                 19. Roper JR, Guther ML, Milne KG, Ferguson MA. Galactose metabolism essential African <br />                                                      sleeping sickness parasite Trypanosoma brucei. Proc Natl Acad Sci U S A. 2002; 99:5884&#8211;9. <br />                                                      [PubMed: 11983889] <br />                                                 20. Young SA, Smith TK. The essential neutral sphingomyelinase involved trafficking the <br />                                                      variant surface glycoprotein bloodstream form Trypanosoma brucei. Mol Microbiol. <br />                                                      2010; 76:1461&#8211;82. [PubMed: 20398210] <br />  <br />  <br />  <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                         Page 8 <br />  <br />  <br />                                                                                              Highlights <br />                                                     &#8226;     CRK12 essential protein kinase BF T. brucei. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     &#8226;     19 protein kinases essential bloodstream form T. brucei. <br />                                                     &#8226;     Efficient methods producing transgenic T. brucei cell lines. <br />                                                     &#8226;     Use fusion PCR, Gateway system, improved selection conditions. <br />                                                     &#8226;     Supplementary material methodological details protocols. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                          Page 9 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Figure 1. Schema generation transfection constructs <br />                                                 Fusion PCR method generates gene knockout (KO) constructs. Step 1: gene-specific 5&#8242; <br />                                                 3&#8242; targeting sequences alleles amplified separate PCRs using different <br />                                                 oligonucleotides (oligos). Step 2: The 5&#8242; 3&#8242; targeting fragments allele fused <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 PCR coding DNA sequence (CDS) respective selectable marker (SM) <br />                                                 amplified plasmid stock. Complementary sequences designed the <br />                                                 termini oligos illustrated nested oligos G H provide specific <br />                                                 amplification fusion PCR product. Approximately 25&#8211;50ng 5&#8242; 3&#8242; targeting <br />                                                 sequences drug resistance CDS PCR products combined fused PCR <br />                                                 amplification. The fusion PCR products verified purified agarose gel and <br />                                                 recovered Qiagen gel extraction kit. Fusion PCR protocols oligo sequences are <br />                                                 detailed supplementary material. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                         Page 10 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Figure 2. Genotyping putative KO cell lines <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 A. qRT-PCR analysis genomic DNA isolated putative 1st allele KO 2nd allele <br />                                                 KO (putative null) cell lines. For cell line, genomic DNA isolated from <br />                                                 approximately 1&#215;107 cells DNeasy Blood &amp; Tissue Kit (Qiagen). QRT-PCR was <br />                                                 performed triplicate SsoFast EvaGreen Supermix With Low ROX (BioRad) a <br />                                                 7500 Real-Time PCR System (Applied Biosystems). Oligos qRT-PCR designed <br />                                                 using Primer3Plus (sequences given supplementary material). Tert used internal <br />                                                 control. All 1st allele KO cell lines (n=19) showed loss copy qRT-PCR. Most 2nd <br />                                                 allele KO cell lines (putative nulls) showed loss copies qRT-PCR. Genotypically <br />                                                 null cell lines obtained 19 genes, lines Tb11.02.4860 <br />                                                 (CAMKK) retained copy target CDS (* RT &lt; 0.0001, ** RT &lt; 0.005, &#165; no <br />                                                 amplification, # target amplification). B. Example genotyping Tb927.10.14780 <br />                                                 null cell line PCR using oligos outside targeting regions subsequent gel <br />                                                 analysis showed presence novel bands consistent replacement target <br />                                                 CDS&#8217;s selectable marker CDSs. <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                         Page 11 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Figure 3. CRK12 conditional null cell line created fusion PCR Gateway approach <br />                                                 A. Fusion PCR products used KO CRK12 endogenous alleles HYG and <br />                                                 BSD. The Gateway cloning approach used generate tet-regulatable copy of <br />                                                 CRK12 transfection. To clone tet-regulatable copy CRK12, CRK12 CDS <br />                                                 (Tb11.01.4130) PCR amplified SM427 genomic DNA using Phusion polymerase <br />                                                 (NEB) oligos KO_078(CRK12)GWF KO_078(CRK12)GWR, contain attB <br />                                                 recombination sequences Gateway cloning. The product gel purified cloned <br />                                                 pDONR221 using BP Clonase II (Invitrogen). The proper clone verified by <br />                                                 restriction enzyme digestion DNA sequencing. The resultant Entry Clone (pCM013) <br />                                                 recombined Gateway compatible version pLEW100v5 vector <br />                                                 (pCM038) proper clone verified restriction enzyme digestion sequencing. <br />                                                 The resultant plasmid (pCM092) linearized NotI prior transfection T. brucei <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                        Page 12 <br />  <br />  <br />                                                 generate cell line tet-dependent CRK12 expression rRNA intergenic <br />                                                 region. B. QRT-PCR verification CRK12 conditional null cell line. Oligos amplifying a <br />                                                 region CRK12 CDS copy CDS present (tet-regulatable copy) and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 sets oligos amplify 5&#8242;/CDS 3&#8242;/CDS regions specific endogenous allele of <br />                                                 CRK12 absence endogenous CRK12 CDS (* RT &lt; 0.0005). The tet-regulatable <br />                                                 CRK12 different 5&#8242; 3&#8242; UTRs endogenous alleles. C. Growth curve <br />                                                 CRK12 conditional null cell line shows growth rate inhibition withdraw tet. <br />                                                 CRK12 (Tb11.01.4130) conditional null cells maintained sequential dilutions of <br />                                                 media containing tet maintain cells mid-log phase growth. CRK12 conditional null cell <br />                                                 cultures densities 7&#215;105 &#8211; 1&#215;106 parasites/ml used inocula growth curves. <br />                                                 The cells spun washed twice media lacking tet wash tet 5&#215;104 <br />                                                 parasites/ml diluted 20 ml split 10 ml cultures (+tet &#8722;tet). <br />                                                 Cultures conditional null copy maintained media 1 &#956;g/ml tet maintain <br />                                                 expression. Cells Incubat <br /> </body></html>