<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Sept. 1994, p. 6350-6360                                                                       Vol. 14, No. 9 <br /> 0270-7306/94/$04.00+0 <br /> Copyright &#169; 1994, American Society Microbiology <br />  <br />  <br />         DBF8, Essential Gene Required Efficient Chromosome <br />                   Segregation <span id='am-22' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-23' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br />                                          FARIBA HOUMANt AND CONNIE HOLM* <br />                                Department Cellular Developmental Biology, Harvard University, <br />                                                  Cambridge, Massachusetts 02138 <br />                          Received 13 January 1994/Returned modification 4 March 1994/Accepted 3 June 1994 <br />  <br />              To investigate chromosome segregation <span id='am-39' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44'  ></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae, examined collection temperature- <br />           sensitive mutants arrest large-budded cells restrictive temperatures (L. H. Johnston A. P. <br />           Thomas, Mol. Gen. Genet. 186:439-444, 1982). We characterized dbJ8, mutation causes cells arrest <br />           2c DNA content short spindle. DBF8 maps chromosome IX near centromere, encodes <br />           36-kDa protein essential viability temperatures. Mutational analysis reveals dbJ8 <br />           alleles nonsense mutations affecting carboxy-terminal encoded protein. Since these <br />           mutations confer temperature sensitivity, appears carboxyl-terminal protein essential <br />           restrictive temperature. In support conclusion, insertion URA3 position also <br />           confers temperature-sensitive phenotype. Although evidence DNA damage, dbJ8 mutants <br />           exhibit increased rates chromosome loss nondisjunction permissive temperature. Taken <br />           together, <span id='am-119' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-120' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-121' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-122'  ></span><span id='am-123' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> suggest Dbf8p plays essential role chromosome segregation. <br />  <br />    Genetic screens used successfully identify                   mutants arrest large-bud morphology defective <br /> proteins monitor chromosome stability ensure faithful               gene products involved mitosis (16, 36, 43, 56). <br /> segregation chromosomes. For example, screen                       However, mutants arrest large buds defec- <br /> monitors decreased fidelity chromosome transmission                   tive DNA replication mitosis; large- <br /> identified <span id='am-66' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-67' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> mutants called ctf (chromosome transfer                budded cells original cdc screen, approximately half <br /> fidelity) mutants (47, 48). Several CTF genes encode products                directly involved DNA replication (15). <br /> that bind cenitromeric DNA (9) monitored                      Thus, production large-budded cells common <br /> mitotic checkpoint mechanism (10). An overlapping class                   characteristic mitotic mutants, morphological screen for <br /> mutations (chl, chromosome loss) obtained screening                   mitotic mutants important differentiate mu- <br /> for chromosome loss disomic strain (31). In                     tants defective mitosis mutants are <br /> approach, cin (chromosome instability) mutants identi-                  defective DNA synthesis. <br /> fied virtue elevated rates chromosome loss                   Several years ago, morphological screen undertaken to <br /> sensitivity microtubule destabilizing drugs (24, 25). The                 isolate new mutants undergo arrest large-budded cells <br /> genes cin8 cin9 encode kinesin-like proteins function               restrictive temperature (27). <span id='am-69' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-73'  ></span>This</span> dbf screen (dumbbell <br /> in mitosis (24). Another gene encoding kinesin-like protein                formers) used identify new genes are <br /> was identified screen mutants defective nuclear                  involved regulation initiation DNA synthesis (26, <br /> fusion (7). Finally, components spindle pole                  29, 51). Although remaining dbf mutants tested <br /> body identified diploidization assay (3).                          disturbances mitotic function, good candidates <br />    Although mitotic mutants identified                    mitotic mutants appear normal DNA <br /> different types genetic screens, mutants common               replication exhibit elevated levels large-budded cells <br /> an elevated level cells large-bud morphology com-                  (26). Since likely mutants are <br /> pared wild type. The proportion large-                   defective mitotic processes, examined dbf collection <br /> budded cells varies mutant. Some mutants arrest                genes encode mitotic functions. <br /> over 80% large-budded cells restrictive temperature (38,                 We report characterization DBF8, novel gene <br /> 50), fits <span id='am-118' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> cdc phenotype (15). Other                involved chromosome segregation. We dbf8 <br /> tnutants exhibit dramatic alteration cell morphology,              mutants appear normal aspects DNA metabolism and <br /> accumulating 30 70% large-budded cells restric-               accumulate large-bud cells 2c DNA content. The <br /> tive temperature (25, 33, 47, 48). In addition,                 DBF8 gene essential temperatures, encodes a <br /> increase large-budded population permissive               36-kDa protein. dbf8 mutants exhibit high levels nondisjunc- <br /> temperature (10, 25, 58). Thus, large-bud morphology                tion chromosome loss, suggesting Dbf8p plays role <br /> common characteristic mutants defective mitosis.                       mitosis. <br />    Morphological screening provided rich source of <br /> mutants accumulate large-budded cells restrictive <br /> temperature (15). Among original cdc mutants,                                   MATERUILS AND METHODS <br />   * <br />     Corresponding author. Mailing address: Division Cellular             Strains media. Saccharomyces cerevisiae strains carrying <br /> Molecular Medicine, 0651, University <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-5'  ></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>California</span>, San Diego, 9500          dbf mutations obtained L. H. Johnston (27). Strains <br /> Gilman Dr., La Jolla, CA 92093-0651. Phone: (619) 534-6336.                  used study listed Table 1. The <span id='am-89' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-90' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-101'  ></span><span id='am-102' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span>Escherichia coli</span> <br />   t Present address: Cubist Pharmaceuticals, Inc., Cambridge, MA             strain used study MC1066 (trpC9830). Standard <br /> 02139.                                                                       genetic techniques used constructing yeast strains (44). <br />                                                                       6350 <br />  VOL. 14, 1994                                                            DBF8 IS REQUIRED FOR CHROMOSOME SEGREGATION                            6351 <br />  <br />                                                       TABLE 1. S. cerevisiae strains used study <br />  <br />  Strain                                                                Genotype                                                       Source or <br />                                                                                                                                       reference <br /> CH595                       AMITa leu2-3,112 ura3-52                                                                               D. Botstein <br /> CH898                       MATs leu2-3,112 lys2-801am ura3-52                                                                     This laboratory <br /> CH1731                      MAMTa adel his] trp2                                                                                   27 <br /> CH1732                      MAMTa adel hisl trp2 dbf6-1                                                                            27 <br /> CH1733                       MATea adel hisl trp2 dbf6-2                                                                           27 <br /> CH1734                      MATat adel hisl trp2 dbf7-1                                                                            27 <br /> CH1735                      MATot adel his] trp2 dbf8-1                                                                            27 <br /> CH1736                      AMTat adel hisl trp2 dbflO-l                                                                           27 <br /> CH1737                      MATat adel hisl trp2 dbfll-l                                                                           27 <br /> CH1738                      MATot adel hisl trp2 dbfl 1-2                                                                          27 <br /> CH1739                      MATot adel his] trp2 dbf]3-1                                                                           27 <br /> CH1740                      MATes adel his] trp2 dbf]4-1                                                                           27 <br /> CH1741                      MAToa adel his] trp2 dbfl4-2                                                                           27 <br /> CH1742                      MAToa adel his] trp2 dbfl5-1                                                                           27 <br /> CH1743                      MATas adel his] trp2 dbf]6-1                                                                           27 <br /> CH1744                      MATas adel hisl trp2 dbfl7-1                                                                           27 <br /> CH1745                      MAToa adel hisl trp2 dbf]8-2                                                                           27 <br /> CH1746                      MATas adel his] trp2 dbf2O-1                                                                           27 <br /> CH1747                      MATa gal2 trp5 ura]                                                                                    27 <br /> CH1748                      MATa ade6 gal2 his4 trp5 ura] dbJ9-1                                                                   27 <br /> CH1749                      MATa ade6 gal2 his4 trp5 ura] dbJ2-Tla                                                                 27 <br /> CH1750                      ALTa ade6 aral gal2 his4 trp5 ural dbf22-1                                                             27 <br /> CH1751                      AM Ta ade6 gal2 his4 trp5 ura] dbf23-1                                                                 27 <br /> CH1763                      MATa hisl leu2-3,112 trp2 ura3-52 dbf8-1                                                               This study <br /> CH1819                      MATa leu2-3.112 lvs2-801am hisl Lrp2 ura3-52 dbf8-1                                                    This study <br />                             MATa leu2-3,112 LYS2          HISI TRP2 ura3-52 DBF8 <br /> CH1863                      AM4Ta leu2-3.112 Ivs2-801am ura3-52                                                                    This study <br />                             AM Tot leu2-3,112 LYS2        ura3-52 <br /> CH1895                      MATa leu2AI trplA63 ura3-52 Adbf8::LEU2 (pCH1462)b                                                     This study <br /> YPH263                      MATas ade2-101 leu2-Al lys2-801 trpl-Al ura3-52                                                        45 <br /> YRS30                       MATa ade2-101 his3-A200 leu2-Al lys2-801 trpl-Al ura3-52 (CF&apos; TRPI SUP]])                              45 <br /> CH2046                      MATa leu2-3,112 lys2-801am ura3-52 DBF8. URA3d                                                         This study <br /> CH2052                      MATot leu2-3,112 lys2-801am ura3-52 DBF8. URA3d                                                        This study <br /> CH2057                      MATca leu2-3,112 lys2-801am ura3-52 dbJ8-5. URA3d                                                      This study <br /> CH2065                      MATa leu2-3,112 ura3-52 dbf8-5. URA3d                                                                  This study <br /> CH2091                      MALTa ade2-101 his3-A200 leu2-A1 Ivs2-801 trpl-Al ura3-52 db-5 (CF&apos; TRPI SUP]])                        This study <br />                             MATas ade2-101 HIS3         leu2-AI lys2-801 trpl-Al ura3-52 dbf8-5 <br /> CH2094                      MAITa ade2-101 his3-A200 leu2-Al Ivs2-801 trpl-Al ura3-52 DBF8 (CF&apos; TRPI SUP]])                        This study <br />                             MATas ade2-101 HIS3         Ieu2-Al lys2-801 trpl-Al ura3-52 DBF8 <br />   &quot;This <br />   b <br />         strain additional <span id='am-129' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-136'  ></span><span id='am-137' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-141' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span>unidentified</span> auxotrophies. <br />       The LEU2 marker strain disrupts DBF8. <br />   &apos; <br />     CF, chromosome fragment. <br />   d The URA3 marker strain adjacent DBF8 does disrupt it. <br />  <br />  <br />  <br />  <br /> The construction strains created integration                         positive clones (among 20,000 colonies) exactly the <br /> plasmid DNA verified Southern analysis (46).                               restriction map original clones. To confirm the <br />    Media described previously (22). 5-Fluoroorotic                      cloned DBF8 gene structure genomic <br /> acid (5-<span id='am-12' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-13' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15'  ></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>FOA</span>) plates selection yeast ura3 mutants (4)                   DBF8 gene, used PCR amplify chromosomal copy of <br /> were described (2).                                             DBF8 gene. Two primers flanking gene (DBF8-1, <br />    Cloning mapping. Strain CH1763 (dbf8-]) trans-                         5&apos;TGCGGCGAGTGGGAATTTC3&apos;, DBF8-9, 5&apos;GAC <br /> formed centromere-based YCp5O genomic library (37).                        GATGACCACTCACAAGTG3&apos;) used PCR. The <br /> After transformants selected synthetic complete me-                       size restriction fragment obtained PCR as <br /> dium lacking uracil, transformants replica plated                     predicted restriction map cloned DBF8 gene. <br /> 35&#176;C examined growth. Although approximately                        Thus, structure cloned gene accurately reflects the <br /> genomes worth library DNA (22,500 colonies) used,                         structure genomic DBF8 gene. To detect DBF8-specific <br /> only clones complemented temperature sensitivity                     mRNA, total yeast RNA probed internal DBF8 <br /> of dbf8 cells obtained. Restriction mapping revealed                    fragment (HindIII PvuII). The size mRNA was <br /> both clones restriction fragment. To ensure                     estimated size 25S (3,390 nucleotides) 18S <br /> we cloned entire DBF8 gene, took additional                        (1,800 nucleotides) rRNA bands (57). <br /> steps. To look clones carrying overlapping fragments,                      <span id='am-124' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-125' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-127' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Plasmids</span>. Plasmid pCH1334 (DBF8+) contains 5.5-kb <br /> E. coli cells containing plasmid library plated <span id='am-17' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-18' property="rdfs:label" content="Luria" datatype="xsd:string"></span><span id='am-19'  ></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Luria</span>                 genomic insert YCp5O library. A 2.0-kb EcoRI-BglII <br /> broth plus ampicillin, colonies transferred nitrocel-                   segment (see Fig. 3) plasmid cloned pBSKS <br /> lulose, filters hybridized small probe just                     (Stratagene) (pCH1401), YIp5 (pCH1375). Plasmid <br /> upstream DBF8 coding region (EcoRI-HindIII). The                           pCH1375 linearized Hindlll direct integration to <br />  6352     HOUMAN AND HOLM                                                                                           MOL. CELL. BIOL. <br />  <br /> the chromosome linkage analysis. URA3 insertions shown        (CH1750), 82% dbf23-1 (CH1751). Results the <br /> in Fig. 3 inserted plasmid pCH1401 (from left           dbf strains described Results. <br /> to right, respective plasmids pCH1505, pCH1507,              Plate assay. Strains CH1763 (dbf8) CH595 (DBF8) were <br /> pCH1509, pCH1411, pCH1510, pCH1412); source            grown overnight 25&#176;C, aliquots shifted 25 <br /> URA3 DNA 1.5-kb NruI-SmaI fragment YCp5O. To           35&#176;C generations (6 h). Aliquots were <br /> probe phage A library, plasmid pCH1403 (DBF8+)          sonicated, diluted, plated prewarmed YEPD plates at <br /> carries 2.7-kb HindlIl fragment plasmid pCH1334        0, 5, 17 h. Plates examined microscopically 0, 5, <br /> used. Plasmids pCH1410, pCH1461, pCH1462 carry              17 h. <br /> 1.5-kb HindIII-BglII fragment containing DBF8 com-          Canavanine resistance assay. Two independent cultures of <br /> plementing activity. The backbones plasmids         strains CH1763 (dbf8-1), CH2065 (dbJ8-5), CH2046 <br /> pRS315, pRS314, pRS316, respectively (45). Plasmids           (DBF8) grown 25&#176;C overnight, split into <br /> pCH1485 (dbf8-3), pCH1486 (dbf8-4), pCH1487 (dbf8-5),             aliquots incubation half generation <br /> pCH1488 (dbf8-6), pCH1489 (dbf8-7), pCH1490 (dbf8-8)          times 25 35&#176;C. Cultures sonicated diluted, and <br /> are derivatives pCH1461 mutagenesis. The Eco47III-      107 cells plated synthetic complete <br /> SpeI fragment plasmid pCH1510 (containing URA3           plates containing canavanine (60 mg/liter). Colonies were <br /> insert) ligated adjacent dbf8 gene plasmids         scored 4 5 days incubation 25&#176;C, number <br /> pCH1485 pCH1490, creating plasmids pCH1514             canavanine-resistant cells 106 viable cells calculated. <br /> pCH1519.                                                             Flow cytometry. Microfluorometric analysis performed <br />    DNA sequencing. The 1.5-kb HindIII-BglII fragment encod-       previously described (32). Haploid strains CH1763 (dbf8-1), <br /> ing DBF8 sequenced dideoxy method (41). Besides        CH595 (DBF8), CH2052 (DBF8), CH2057 (dbf8-5) were <br /> the universal primers, insert-specific primers used     grown early log phase YEPD 25&#176;C. The cultures were <br /> (sequences available request). Each strand se-           split, portions shifted YEPD medium 35&#176;C for <br /> quenced twice. The databases searched BLAST pro-           6 h. The samples fixed, treated RNase A (1 mg/ml), <br /> gram GenBank release 79 EMBL release 27. The             stained propidium iodide. Cells (106) each <br /> sequence original dbf8-1 mutation parental         culture prepared fluorescence-activated cell sorter <br /> strain determined single-stranded DNA amplified         (FACS) analysis. A total 40,000 cells counted each <br /> from strains CH1731 CH1735 asymmetric PCR (12).            sample. <br />    Essential gene. To determine DBF8 essential         CHEF gel assays. Yeast chromosomal DNA samples were <br /> gene, deletion Stu1212-Eco47111933 created         prepared strains CH1763 (dbf8-1), CH595 (DBF8), <br /> marked LEU2 gene plasmid pJJ250 (28). A             CH2052 (DBF8), CH2057 (dbf8-5) previously described <br /> linear EcoRI-Spel fragment ADBF8 gene containing       (18). Cultures grown early log phase 25&#176;C then <br /> LEU2 insertion integrated diploid strain CH1861,         split incubated 25 37&#176;C half and <br /> creating strain CH1865. After sporulation, tetrad analysis    half generation times. For control samples, hydroxyurea was <br /> used determine phenotype caused deletion. To        added 0.1 M final concentration time split, or <br /> make strain CH1895, plasmid pCH1462 introduced           cells allowed grow 23 h 25&#176;C reach stationary <br /> strain CH1865 prior sporulation dissection.                phase. The samples lysed previously described (18), <br />    In vitro mutagenesis. Mutations introduced       applied 1% agarose gel (Pulsed Field Certified Agarose <br /> cloned DBF8 gene transferred yeast plasmid        [Bio-Rad]), electrophoresed 0.5 x Tris-borate-EDTA <br /> shuffle technique (5, 23). Plasmid pCH1461 (DBF8 TRPI)        (TBE) 4&#176;C 15 h switching time 70 s, 12 h at <br /> mutagenized vitro 42-h incubation hydroxylamine        switching time 120 s field strength 7.6 V/cm. The <br /> (38). The rate mutagenesis assessed examining       DNA transferred nitrocellulose probed a <br /> frequency Trp- transformants E. coli (trpC). The mu-        TRPI fragment (Hindlll-SnaBI fragment plasmid <br /> tagenized plasmids introduced strain CH1895             pRS304 [45] labeled 32p using decaprobe kit [Ambion]). <br /> (Adbf8::LEU2 [pCH1462 {DBF8 URA3}]). After selecting          This probe chromosome IV chosen con- <br /> the loss resident URA3 plasmid (pCH1462) 5-FOA          tour-clamped homogeneous electric field (CHEF) assay is <br /> plates, transformants examined abilities    sensitive probes derived large chromosomes <br /> grow 35 14&#176;C. The nature mutation allele     used (18). <br /> determined DNA sequencing.                                        Colony sectoring assay. To examine chromosome stability, <br />    Microscopy. Strains CH595 (DBF8), CH1763 (dbf8-1),             constructed strains homozygous dbf8 DBF8, <br /> CH2065 (dbf8-5), CH2046 (DBF8) used examine           carries nonessential chromosome fragment <br /> nuclear spindle morphology. Exponentially growing cul-        (22, 45). Because colony color signaled SUPI I nonsense <br /> tures incubated yeast extract-peptone-dextrose            suppressor chromosome fragment, used missense <br /> (YEPD) 25&#176;C overnight; cultures split        allele DBF8 (dbf8-5) experiment. The strains were <br /> aliquots incubated 25 35&#176;C            constructed follows. Plasmids pCH1510 (DBF8.URA3) and <br /> generation times. The cells sonicated, fixed, stained    pCH1516 (dbf8-5.URA3) digested Spel HindIll, <br /> as previously described (23).                                     3.1-kb fragments bearing URA3-tagged DBF8 or <br />    Yeast cells CH1731 genetic background soni-        dbf8-5 alleles isolated. These fragments trans- <br /> cated ice 10 s output 6 Branson sonifier 185       formed strains CH1063 (YRS30) CH1060 (YPH263) <br /> equipped microtip. Other strains sonicated         (45), creating strains CH2080 CH2083. Diploids homozy- <br /> output 2 6 s. Percentages large-budded cells dbf   gous DBF8 (strain 2094) dbf8-5 (strain 2091) created <br /> strains determined, half generation times    mating strains CH2082 (MA Ta DBF8) CH2080 (MA To <br /> at restrictive temperature, 48% dbp7-1 (CH1734),    DBF8) strains CH2083 (AL Ta dbf8-5) CH2081 <br /> 6% dbfl3-1 (CH1739), 34% dbfl4-1 (CH1740), 27%        (MA To dbf8-5). <br /> dbfl4-2 (CH1741), 45% dbfl5-J (CH1742), 60%                  To perform colony sectoring assay, strains CH2094 <br /> dbfl8-2 (CH1745), 60% dbf20-1 (CH1746), 20% dbJ9-J        (DBF8/DBF8) CH2091 (dbf8-5/dbf8-5) struck for <br /> (CH1748), 48% dbf21-1 (CH1749), 83% dbf22-1               single colonies low-adenine medium allowed grow <br />  VOL. 14, 1994                                             DBF8 IS REQUIRED FOR CHROMOSOME SEGREGATION                      6353 <br />  <br />  <br />                       DAOPI                                 Anti-tubulin <br />  <br />  <br />  <br />                                                                                                               DBF8 <br />  <br />  <br />  <br />  <br />  stan-nBa 5. <br />           rn <br />  <br />  <br />  <br />  <br /> at 25&#176;C 4 5 days. Three pink colonies strain                              RESULTS <br /> were inoculated YEPD grown log phase. Each <br /> culture split transferred 25 35&#176;C       dbf8 mutants arrest large-budded cells several <br /> half generation times (4 h). Aliquots sonicated,     generations. To identify yeast mutants defects mitosis, <br /> diluted, plated low-adenine medium; samples          examined collection cell cycle mutants arrest as <br /> each aliquot fixed 3.7% formaldehyde scoring    large-budded cells restrictive temperature. These dumb- <br /> cell morphology. After incubation 25&#176;C 5 8 days,       bell formers (dbfs) originally isolated screening a <br /> half-sectored colonies counted. The rate nondisjunc-     library temperature-sensitive mutants large-bud phe- <br /> tion determined dividing number red white      notype 16 h 37&#176;C (27). The mutant strains all <br /> half-sectored colonies total number colonies. Simi-    shown single-gene defects, subset we <br /> larly, rate chromosome loss determined dividing    examined, incorporation studies showed DNA synthesis <br /> the number red pink half-sectored colonies total   continues 37&#176;C (25a). We performed complementation tests <br /> number colonies. Solid red white colonies excluded    mutants cdc mutants known arrest as <br /> from total count.                                            large-budded cells restrictive temperature. Our complemen- <br />    Nucleotide sequence accession number. The DNA sequence        tation tests revealed 2 16 mutants examined are <br /> described paper deposited GenBank      allelic CDC genes encoding products involved mitosis <br /> accession number U12437.                                         (DBF6 = CDC15 DBFJ1 = CDC16). In addition, two <br />  6354        HOUMAN AND HOLM                                                                                                            MOL. CELL. BIOL. <br />  <br />  TABLE 2. Phenotypes dbf8 DBF8 cells solid medium                        prewarmed YEPD plates, incubated 25 35&#176;C, and <br />             permissive restrictive temperatures&apos;                                  examined microscopically. Although 60% dbf8 cells <br />                                         No.   following morphology:              large-bud morphology 6 h liquid medium the <br /> Preincubation     Plating                                                             restrictive temperature, half large-budded cells <br />  temp (&quot;C)&quot;        temp <br />                    (OCY         Single          Doublet    4 cells      &gt;4 cells   divide plated restrictive temperature <br />                              small-bud cell <br />                                                                                       (Table 2). In contrast, large-budded cells cultures incu- <br />        25         Noned            91                 8          0            0       bated liquid 17 h fail divide plated at <br />                   25                0                 5          6           89       permissive restrictive temperature (data shown). <br />                   35                0                 2          4           94       Thus, appears dbf8 initially delays progression of <br />                                                                                       cells cell cycle restrictive temperature and <br />        35         None             34                64          0            0       ultimately leads terminal arrest generation <br />                   25                0                11         41           48       times (a multicycle arrest). <br />                   35                1                19         40           40          DBF8 encodes essential 36-kDa protein. To determine if <br />   &quot;This experiment repeated times similar results.                     Dbf8p sequence similarity known proteins, we <br />   &quot;Preincubation liquid medium 6 h.                                        cloned gene complementation temperature- <br />     Plates incubated 5 h indicated temperatures morphol-       sensitive phenotype dbf8 mutant. After transforming a <br /> ogy determination. <br />   &quot; None, morphology determined directly preincubation.                         genomic library strain CH1763 (dbJ8), recovered two <br />                                                                                       Ts&apos; transformants contained library plasmid <br />                                                                                       carrying 5.5-kb genomic fragment. An identical 5.5-kb cloned <br />                                                                                       fragment obtained times complementing the <br /> other dbf genes allelic genes encoding products involved                       temperature sensitivity dbf8 S. cerevisiae hybridizing <br /> in DNA replication (DBFJO = MCM3 DBF16 = CDC13).                                  single DBF8 clone genomic library. Since no <br />    To determine mutants cell division                          overlapping clones detected, possible the <br /> cycle (Cdc) phenotype, examined morphology                              sequences flanking DBF8 gene toxic E. coli, their <br /> remaining 12 strains restrictive temperature. To focus                      corresponding clones underrepresented. To confirm <br /> early effects loss function gene products,                      structure cloned DNA accurately represents <br /> examined cells incubated 35&#176;C half                         genomic DNA, amplified compared segments of <br /> generation times (approximately 3 h). After incubation,                          genomic plasmid DNA PCR; expected, amplified <br /> only dbf8, dbfl8, dbJ2O, dbf22, dbJ23 mutants produced                            fragments identical size (data shown). <br /> more 50% large-budded cells (range, 60 83%). Among                               To verify cloned DBF8 gene, examined <br /> these, chose dbf8 strain characterization.                         linkage cloned DNA dbf8 mutation. <br />    Cytological examination revealed phenotype dbf8                        We used homologous recombination integrate DBF8+ <br /> mutants restrictive temperature consistent                           plasmid (pCH1375) dbf8/DBF8 diploid strain (CH1819). <br /> defect DNA replication mitosis. To examine                           After sporulation, tetrad analysis revealed cloned gene <br /> cellular morphology dbf8 mutants, culture growing expo-                          DBF8 locus linked (six parental ditype <br /> nentially 25&#176;C split, aliquots shifted 25                       tetrads). We showed DBF8 gene maps chromosome <br /> 35&#176;C half generation times. Whereas DBF8+                               IX using cloned gene (2.7-kb HindlIl fragment) to <br /> cells 1 10% large-budded cells 35&#176;C, dbf8 mutants                          probe blot yeast chromosomes separated pulsed-field <br /> accumulate 55 70% large-budded cells. Thus, dbf8 mutants                           gel electrophoresis. The exact position gene ob- <br /> exhibit marked elevation proportion large-budded                          tained probing phage A library fragments yeast <br /> cells, lower standard cdc mutants                         genome (set 163 [25a]). Three overlapping phage X clones <br /> (15). In addition, dbf8 mutation affects cellular morphology                      (6675, 7071, 5165) detected probe. This result <br /> even permissive temperature (5 20% large-budded                             places DBF8 gene long arm chromosome IX <br /> cells 25&#176;C, depending strain background).                                   (MERG 0807), near centromere approximately 45 kb <br />    To examine spindle nuclear morphology dbf8                              away BAR]. Subcloning portions insert narrowed <br /> cells, cultures shifted 25 35&#176;C treated fluores-                       complementing activity 5.5 kb 1.5-kb region (data <br /> cence microscopy (Fig. 1). DBF8+ strains large-                              shown). <br /> budded cells temperature, large-budded cells                           DNA sequence analysis 1.5-kb HindIII-BglII region <br /> have variety nuclear morphologies depending                             revealed 957-bp open reading frame homology cut8 <br /> stage growth cycle. In contrast, 75% large-budded                           <span id='am-46' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48'  ></span><span id='am-49' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Schizosaccharomyces</span> pombe (Fig. 2) (40). The size the <br /> cells dbf8 culture restrictive temperature                       open reading frame consistent results Northern <br /> single undivided nucleus adjacent mother-daugh-                         (RNA) analysis, revealed hybridization HindlIl- <br /> ter neck threaded it. These dbf8 mutants                            PvuII fragment DBF8 approximately 1-kb mRNA <br /> short spindle distinct cytoplasmic microtubules. Thus,                        (data shown). The deduced protein sequence predicts that <br /> phenotype dbf8 mutants restrictive temperature                           Dbf8p 319-amino-acid (36-kda) protein neutral (7.5) <br /> similar observed cdc mutants defects DNA                           pKa. A putative promoter element (TATAA box) at <br /> replication mitosis (14).                                                          -263 (49), putative polyadenylation signal (53) lies <br />    The accumulation large-budded dbf8 cells                               approximately 40 nucleotides downstream TAA stop <br /> caused terminal arrest large-budded cells cell                     <span id='am-74' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-75'  ></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span>. The DBF8 gene product shows weak homology to <br /> cycle delay cells cells pass cell                   Pho2p carboxy terminus, 20% identity 43 amino <br /> cycle. To distinguish possibilities, used                          acids (42); significance homology doubtful, <br /> plate assay examine fate large-budded cells long                       however, segment DBF8 required for <br /> periods. Strain CH1763 (dbf8) grown early logarithmic                          viability (see section). Informative homology revealed <br /> phase permissive temperature shifted liquid                        comparison private data bank (l la); Dbf8p exhibits a <br /> medium permissive restrictive temperature                        striking similarity cut8 gene product (40). This similarity <br /> for generation times (6 h). Aliquots spread                          spans entire protein sequence consists 26% identity <br />  VOL. 14, 1994                                                                      DBF8 IS REQUIRED FOR CHROMOSOME SEGREGATION                                                              6355 <br />  <br />  <br /> -378       --+ -        +        --+                - +-         - 319                         463 -            +-               +-           +-        +-                 +-          + 522 <br />      CGTGTTCGACCTCCACAGTGGCACAATCTCCTGCTCAACTCTTCGGTAGTITATAA                                  155 F V         R M K            P Q I     L   E F L    N C L           V   D F     I   L 174 <br /> -318        -        +---         --+             -   +-         - 259 <br />      AC&#182;ETTGAGGTTCCACITACGCTrAATITG3GCTATFmG&#182;mGC3FI&apos;GTCTICA                                          GATAACATTCCACCACGrlIGGAAAATTIGCACGCATCATITAAAATETTAGACATlq C <br /> -258 - +-               +       +-        +-        +-         + -199                               +------+- <br />                                                                                                523 --                 +----+---           + 582 <br />      AAAGGCCTCATCTAAAGCCCAT,CGAATTAT&apos;CCTrrAGITATTCATAGTGAAAATT                                 175 D N I P P R L E N L H A S L K F L D I g 194 <br /> -198 -        +.+-              +-         +        +-         + -139 <br />        CCAGAAAAAGAAACGGCGGGCGGG&#182;ATAATGCGGCGAGTGGGAAETITACTTAAAAA                                   ACGGAATTAGTrATCAAA7MCCCAGATTiIAATrAGCTAGTAATAATrACTACTACGAT <br /> -138   -            +-              +-        +-          +-           +-        + -79         583 --       +-----++                      +- --+---- + 642 <br />        &#182;IATACTrAACCAAGTATCCTCACTrAGTGTGTTCGAAGGTACCIGGCAAAAGCAACAAAC                           195 T E L V I K L &#163; R F E L A S N N Y Y Y D 214 <br />  -78                                ----+-+       +     -         + -19 <br />        ACAGGACAGTATAGAGCATTATGA             <span id='am-112' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-113' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-116' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-117'  ></span>AA</span>   AA CCCAGCAGTAAAATA                                  AAA&#182;GCATmIAACAATITGTCACGTATGGTTACGCTIATAAAACATGTTGCCCGIGAC <br />  -18   -            +               --+-          +---+---+ 42                                 643 -               +-               +-    +-               +-              +-              + 702 <br />                                     M M G F E W G F K P S S K I      14                        215 K C         I    E Q         L   S H V W C T        L    I E        H   V A     R   D     234 <br />        ACACAGAGTACAGTCAGTrCACAAGGCACAGGCAATGTAATGATTCCAACTSCTGGAGTG                                  AGGATAATIATTAGCAGATAATAGCICCGTCIOGAAAAGTCATAT&#182;;ACAAGATrACAG <br />   43   -        +----++ +-                                        + 102                        703---------------------------------------------- <br />                                                                                                                                               -+------------+ 762 <br />   15    T Q S T V S S Q G T G N V M I P T A G V                      34 <br />                                                                                                235  R I I L L A D N S S V W K S H M T R L Q 254 <br />  <br />  103   -            +-              +-        +-       +-              +-         + 162              GTATACAA&#182;GAACATTCAAAG0GTrATAGGAACGCCCACTCCAG0CI.IAAATC1ITA <br />   35    K   Q   K    R R        Y   A N   E   E Q E   E E E        L   P R   N   K   54             +-----+-+ <br />                                                                                                763 --                                                  -+-             -       ---      + 822 <br />                                                                                                255 V Y N E H S N G L L E                           R   P L Q           L   F   K   S   L 274 <br />  163   ---          +-----++            +    -----   + 222                                           GATAIGGGTAGCCCTTCAGCACTCGTCATCAACATrATCACTTCAAGAAAGTATTATr <br />   55    N   V   M   K Y G G V S K R R P Q P G S L I R   74                                     823 -    +----+-+-                                              --          +---+ 882 <br />                                                                                                275 D M G S P S A A S S S T L                                   S   L   Q   E S I I 294 <br />        GGGCAGCCTCTGCCCTCAACGAGGCAT0GAACTAAIGAACAAGAACCAATTACAGCAG <br />  223                                 ----+-+      +      +-     + 282                                TACCACCACGATACCATIACAGCAAACGAAAACAATAACAATAGITAGCGCIGCTACT <br />   75    G   Q   P       L   P   L    Q R G M E L M N K N Q L Q Q   94                          883 --   +-----+-+------                   + 942 <br />                                                                                                295 Y H H D T M T A N E N N N N S G S A A T 314 <br />        TTGTI.GGGA[TIAIACAAAACA&#182;CCAGAAATACAACAAAGCGTACATACGAGAGTC <br />  283   -            +----++                                            +- + 342                  GATrCGCCCTrTAACTAAGGATAGAAATGTmCTI&apos;ITICAAATACATrAATATGT <br />   95    L   L   V   D L M T K H P E I Q Q S V                          H T R V 114             943--------------------+---+-                           + 1002 <br />                                                                                                315    D    S   P    F       N   * <br />        ATCGGATTAGACTTCAGTATACAAAAGGITCTGGACATGTTGAAACAAAAATCAGAAGCC <br />  343   -         +-              +-         +-         +-           +-          + 402                ATATATATGTATATATAAAIGTATCACTGT&#182;TAATGTrACATATGCAGAAGTATCCGATT <br />  115    I   G   L D F           S I Q     K C L D     M L K        Q K S     E A 134          1003   -     +-      +-      +-      +-      +-       + 1062 <br />                                                                                                          IIIICTliTrMTATAAAlCCAGTNrACT0GAAA0ATAAACTIAT <br />                                                                                                    ATrI <span id='am-7' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-8' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>IA</span> <br />        GrACCAGAGCATACCITACAATAGGTCTrATGAGAGTAATAAATTAGATSACTATGCT                             1063 -       +-      +-      +-      +-      +-       + 1122 <br />                                                   --+-+ 462 <br />  403 -------------------------------------------+----------+                                         CCCIC.AACAGCIGTSAGGCATrAATCACT?lGAG&#182;I0&#182;ItATICGTCCGITrCGCTIGA <br />  135 V Y Q S I P Y N R S Y E S N K L D D Y A 154                                              1123   -             +-               +-        +-       +-                  +-              + 1182 <br />  <br />    FIG. 2. Nucleotide sequence predicted amino acid sequence DBF8 gene. The nucleotide sequence 1.5-kb HindIII-BglII region <br /> is shown numbered beginning nucleotide ATG. The predicted amino acid sequence numbered beginning first <br /> methionine open reading frame. Underlined amino acids indicate mutation sites missense mutants; asterisks indicate mutation site of <br /> nonsense mutants (see Table 3 details). The PvuII site marked asterisks. <br />  <br />  <br /> and 50% similarity. Temperature-sensitive mutations cut8                                   807) dbf8IDBF8 diploid strain (CH1819), half trans- <br /> cause defects nuclear division S. pombe (21, 40).                                       formants temperature-sensitive phenotype. After sporu- <br />    To determine DBF8 essential viability S. cerevisiae,                          lation, dissection resulting tetrads revealed this <br /> we created disruptions DBF8 open reading                                       simple insertion mutation DBF8 gene creates temper- <br /> frame. When deletion strains URA3 insertions                                  ature-sensitive growth phenotype. In summary, DBF8 gene <br /> examined, resulted inviability (Fig. 3; data                                  essential viability temperatures, results a <br /> shown). Examination spores failed form viable                                      URA3 insertion suggest carboxy terminus be <br /> colonies indicated spores containing DBF8 deletions                                      essential viability restrictive temperature. <br /> germinate divide twice. Insertion URA3                                        The carboxy terminus Dbf8p required viability only <br /> cassette Stul PvuII sites outside DBF8 open                              restrictive temperature. To determine the <br /> reading frame produced inviable strains, inviabil-                              phenotype caused dbf8-1 mutation allele specific, we <br /> ity complemented cloned DBF8 gene. Thus,                                    used vitro mutagenesis generate new conditional alleles. <br /> other essential sequences appear lie near DBF8 gene.                                   Plasmid pCH1461 (DBF8+ TRP1+) mutagenized vitro <br /> When URA3 introduced PvuII site (at nucleotide                                     hydroxylamine, resulting dbf8 mutations were <br />  <br />  <br />                                                                                                                         5         6 <br />                                                                                                                    4 8          3 7 2 <br />  <br />  <br />  <br />                                 L                              I             I            I               DBF8              -I            I <br />                                 R                     H        S                          S                                         N P       E    N       P       B <br />  <br />  <br />                        p <br />   FIG. 3. Summary point mutations insertions DBF8 coding region. The line drawing represents 2.0-kb EcoRI-BglII region <br /> surrounding DBF8 gene; restriction sites used study shown. B, BglII; R, EcoRI; E, Eco47III; H, HindIII; N, NdeI; P, PvuII; S, StuI. <br /> URA3 insertions result inviability indicated downward arrows. The insertion NdeI site (outside coding region) does <br /> not alter viability insertion PvuII site causes Ts- phenotype (also called dbf8-2) indicated upward arrows. The <br /> approximate location point mutations clustering depicted line. Filled circles represent missense mutation sites <br /> (dbf8-4, -5, -8), small arrows represent nonsense mutation sites (dbf8-1, -2, -3, -6, -7). <br />  6356       HOUMAN AND HOLM                                                                                                                     MOL. CELL. BIOL. <br />  <br />                          TABLE 3. Phenotype various dbf8 alleles tabulation nucleotide amino acid changes <br />                                    Ts- phenotypea                                                                  Alteration(s) <br /> Allele <br />                       On plasmid                On chromosome                              Nucleotide                                        Amino acid <br /> dbf8-1                   ND                          Strong                     G737A                                              W246am (V2301)b <br /> dbf8-2                   ND                          Weak                                                                          Insertion 269 <br /> dbf8-3                  Weak                         Strong                     T712AC                                             L239och <br /> dbf8-4                  Strong                        ND                        G507C, G581A                                       L169F, C194Y <br /> dbf8-5                  Weak                         Strong                     C65T                                               P202S <br /> dbf8-6                  Weak                         Strong                     G737A, G738A                                       W246och <br /> dbf8-7                  Strong                        ND                        G592C, G728A, G733A, G737A                         V198L, S243N, V2451, W246am <br /> dbf8-8                  Strong                        ND                        C605T, C66T                                        P202F <br />   &quot; A strong temperature-sensitive phenotype defined producing haze growth spot cells grown restrictive temperature. A weak <br /> temperature sensitivity phenotype substantially growth produces growth DBF8 strain grown conditions. ND, <br /> not determined. <br />   b There additional changes dbf8-J allele differences strain CH1731 (Johnston laboratory strain) strain S288c (Holm <br /> laboratory strain) DBF8+ genes; differences G9A, C183T, G691A, T1084A. The resulting amino acid change V230L. <br />   &apos; There additional mutations stop codon: base deletion position 715 C-to-G transversion position 718. <br />  <br />  <br />  <br /> screened conditional phenotype plasmid-shuffle pro-                       open reading frame, bp 507 738 (Fig. 3). <br /> tocol (5). The mutagenized plasmids transformed                        Alleles dbf8-4, dbf8-5, dbf8-8 missense mutations that <br /> strain harboring chromosomal DBF8 deletion (CH1895)                         map 5&apos; region discovered mutations <br /> and plasmid carrying DBF8+ URA3 (pCH1462). The                               (bp 507 605). Alleles dbf8-1, dbf8-3, dbf8-6, dbf8-7 <br /> phenotype transformants determined selection                      cluster 3&apos; region (bp 592 738) result in <br /> for loss DBF8+ plasmid 5-FOA plates. Of 12,000                           nonsense mutations. In summary, independent mutations <br /> transformants, 6 exhibited temperature-sensitive phenotype                       (four point mutations URA3 insertion PvuII site <br /> that conferred plasmid-borne copy dbf8. Strains                      [see previous section]) lead truncation Dbf8p pro- <br /> carrying plasmid-borne dbf8 mutations exhibited various                        duce temperature sensitivity phenotype. Thus, carboxy <br /> degrees temperature sensitivity (Table 3). When spotted                         terminus Dbf8p appears dispensable viability at <br /> onto YEPD plates, strains showed tight temperature                           permissive temperature required viability restrictive <br /> sensitivity 35&#176;C; strains harbored plasmids pCH1486                       temperature. <br /> (dbf8-4), pCH1489 (dbf8-7), pCH1490 (dbf8-8). Three                               The DBF8 gene product plays role mitosis. To deter- <br /> other strains papillated readily restrictive temperature;                   temperature-sensitive defect dbf8 mutants <br /> these strains contained plasmids pCH1485 (dbf8-3), pCH1487                         likely caused perturbation mitosis DNA <br /> (dbf8-5), pCH1488 (dbf8-6).                                                    metabolism, looked disturbances DNA metab- <br />    To examine phenotype conferred new alleles                           olism. To determine bulk DNA synthesis is <br /> when integrated chromosome, alleles trans- <br /> placed genome. A URA3 cassette ligated adjacent <br /> to dbf8 gene NdeI site bp 1042. This insertion <br /> causes detectable phenotype integrated adjacent                                                   DBF8 25&apos;                                dbf8 25&apos; <br /> wild-type DBF8 gene (data shown). The URA3-tagged dbf8 <br /> alleles initially transplaced haploid strain (CH898). <br /> Surprisingly, alleles produced weak <br />                                                                                                               o(I? <br />                                                                                                               46   54     1 <br />                                                                                                                                                       o1   O <br />                                                                                                                                                       17 63 19 <br /> phenotypes carried plasmid readily recov- <br /> ered (dbf8-3, dbf8-5, dbJ8-6). Integration diploid <br /> strain (CH1863) followed sporulation dissection re-                           :: <br /> sulted recovery additional allele (dbf8-7). However, <br />                                                                                     z <br /> meiotic segregants harboring mutation grew slowly, <br /> and papillated heavily restreaking. The temperature                       =u <br />                                                                                      c <br />                                                                                                                DBF8 35&apos;            D                   dbf8   35&apos; <br /> sensitivity conferred integrated alleles (dbf8-3, dbf8-5, <br /> and dbf8-6) considerably tighter observed                                                  o(&amp;9 <br />                                                                                                               63   36     1 <br />                                                                                                                                                   A o9 <br />                                                                                                                                                     9 16 72 <br /> these mutant alleles carried plasmid (e.g., no <br /> papillation restrictive temperature) (Table 3). Thus, it <br /> appears likely alleles caused strong phenotype <br /> when carried plasmid caused inviability integrated <br /> into genome. The cellular morphologies conferred the <br />                                                                                                                                                   I           vl <br />  <br /> integrated alleles restrictive temperature revealed                                                              Fluorescence <br /> phenotype similar observed dbf8-1 (i.e., 50 65% <br /> large-budded cells half generation times).                             FIG. 4. DNA content dbJ8 mutant cells permissive and <br /> Thus, alteration cell morphology observed dbf8                            restrictive temperatures. Strains CH1763 (dbf8) CH595 (DBF8) <br />                                                                                     grown early log phase YEPD 25&apos;C. Cultures split, <br /> mutants gene specific special characteristic                    portion shifted 35&#176;C 6 h. After fixation, samples were <br /> dbf8-l allele.                                                                      stained nucleic acid-specific dye propidium iodide exam- <br />    DNA sequencing dbf8 alleles revealed nature                        ined flow cytometry. Cell morphology observed parallel <br /> the mutations lead temperature sensitivity. The muta-                       samples (insets). (A) DBF8 (25&#176;C); (B) DBF8 (35&#176;C); (C) dbf8 (25&#176;C); <br /> tions cluster 231-nucleotide region 3&apos; one-third                     (D) dbf8 (35&#176;C). <br />  VOL. 14, 1994                                                           DBF8 IS REQUIRED FOR CHROMOSOME SEGREGATION                                  6357 <br />  <br />                               Et             C;        Q                     TABLE 4. Analysis spontaneous mutation rate CAN1 <br />                                              O <br />                                                   v <br />  <br />                                                        O                                    locus dbJ8 DBF8 strains&apos; <br />                               xo    x        X.   oe   6.      ~0 <br />                                                                                                                   No. canavanine-resistant colonies <br />                               4     .0       k <br />                                                   4~   k <br />                                                                               Strain                                       (per 106 colonies) <br />              Wells      --   ai.    m                                                                           25&#176;C                               35&#176;C <br />                                                                              dbf8-1                           4.8 &#177; 1.3                         4.0 &#177; 1.5 <br />              chr   IV   -- <br />                               _0W        4         _ <br />                                                         _op <br />                                                         _  _        _ <br />                                                                              dbJ8-5 <br />                                                                              DBF8+ <br />                                                                                                               4.3 &#177; 3.2 <br />                                                                                                               4.8 &#177; 0.4 <br />                                                                                                                                                 5.0 &#177; 1.4 <br />                                                                                                                                                 4.3 &#177; 0.7 <br />                                                                                &quot; The results reported average experiments, involving two <br />                                                                              independent colonies strain. Standard deviations indi- <br />                                                                              cated. <br />    FIG. 5. Integrity DNA dbf8 mutants. Exponentially growing <br /> cultures strains CH595 (DBF8) CH1763 (dbf8) split into <br /> aliquots incubated 3 3.5 25&#176;C YEPD, 37&#176;C YEPD,          We examined spontaneous mutagenesis CAN] locus <br /> or 25&#176;C YEPD plus 0.1 M hydroxyurea. Another sample             determine DNA dbf8 strains minor <br /> dbf8 culture allowed grow saturation YEPD 25&#176;C. DNA          lesions undetected physical analysis (27, 32). <br /> was isolated sample run pulsed-field gel. After           Strains CH1763 (dbf8-1), CH2067 (dbf8-5), CH2046 <br /> transfer nylon membrane, DNA hybridized probe            (DBF8) incubated permissive restrictive tempera- <br /> to chromosome (chr) IV. DNA saturated culture (STAT)                ture half generation times plated onto <br /> serves control migration fully replicated DNA. The lane          medium containing canavanine. The levels spontaneous <br /> containing sample contained DNA lanes          mutation CAN1 locus indistinguishable DBF8 and <br /> in gel, 10-fold-longer exposure shown lane. Other <br /> gels amounts DNA uniform gave results              dbf8 strains incubated 25 35&#176;C (Table 4). This <br /> similar shown here. DNA sample incubated                result distinguishes dbf8 mutants defects in <br /> hydroxyurea (HU) serves control migration DNA                DNA replication, cdc9 (16). This observation is <br /> undergoing replication.                                                      consistent previous observations (27), suggests once <br />                                                                              dbf8 mutation unlikely cause DNA damage. <br />                                                                                 Since dbf8 mutants exhibit mitotic delay late cell <br /> complete dbf8 mutants restrictive temperature,                  cycle obvious defects DNA metabolism, used <br /> examined DNA content dbf8 cells. If dbf8 cultures                colony sectoring assay test possibility dbf8 causes <br /> enriched cells 2c DNA content                     defect chromosome segregation. Increases rates of <br /> restrictive temperature, suggest DBF8 gene prod-                nondisjunction chromosome loss observed with <br /> uct plays role DNA replication. We incubated exponen-                   mutations affecting chromosome segregation (10, 22, 36), but <br /> tially growing dbf8 DBF8 strains permissive (25&#176;C)               defects DNA metabolism typically lead large increase <br /> restrictive temperature (35&#176;C) generation times,                   rate chromosome loss (17, 31, 36). To examine <br /> stained cells propidium iodide, performed FACS                  rates nondisjunction chromosome loss dbf8 <br /> analysis. DBF8+ cells lc 2c peaks 25 35&#176;C               mutants, used colony sectoring assay diploid <br /> (Fig. 4). In contrast, dbf8 mutants predominantly 2c              strains homozygous ade2(ochre) mutation carry single <br /> either 25 35&#176;C. Thus, dbf8 mutants spend large fraction              copy centromere-linked ochre suppressor (SUP11) a <br /> time 2c DNA content, consistent presence                  dispensable chromosome fragment (20). Exponentially grow- <br /> some large-budded cells permissive temperature.                  ing cultures dbf8/dbf8 DBF8IDBF8 strains shifted <br /> This observation makes unlikely substantial failure             permissive restrictive temperatures half <br /> bulk DNA synthesis responsible temperature sensi-                 generation times, plated permissive temperature, and <br /> tivity dbf8 mutants.                                                      examined frequency nondisjunction chro- <br />    Although bulk DNA replicated dbf8 mutants,                   mosome loss division. While DBF8IDBF8 strains <br /> there small amounts unreplicated DNA sufficient                  exhibited rates nondisjunction chromosome loss similar <br /> to cause disruption cell cycle. To determine                previously reported wild-type strains (10, 22, 30), <br /> DNA fully replicated dbf8 cells, used CHEF gel                    dbf8/dbf8 strains exhibited significant increases the <br /> chromosome mobility assay (18). In assay fully                     frequency nondisjunction (50-fold increase) chro- <br /> replicated DNA enters gel; incompletely replicated DNA                   mosome loss (70-fold increase) permissive restric- <br /> (e.g., replication loops single-stranded regions) causes DNA              tive temperature (Table 5). Together cell cycle delay <br /> to retarded well. DNA prepared strains                    intact DNA, increases nondisjunction and <br /> CH595 (DBF8), CH1763 (dbf8-1), CH2065 (dbf8-5),                          chromosome loss suggest Dbf8p plays role chromo- <br /> CH2046 (DBF8). As controls, dbf strains treated                segregation S. cerevisiae. <br /> hydroxyurea allowed grow stationary phase before <br /> DNA isolation. As expected, DNA isolated hydroxy-                                            DISCUSSION <br /> urea-treated culture (the incompletely replicated control) re- <br /> mained wells electrophoresis, majority                 Results presented DBF8 essential gene <br /> the DNA stationary cultures (fully replicated con-                  encodes product likely involved chromosome <br /> trols) migrated gel (Fig. 5). DNA dbf8 cells            segregation. Mutations remove terminal quarter of <br /> restrictive temperature enters gel fully                  Dbf8p create temperature-sensitive phenotype, suggesting <br /> DNA DBF8+ cells temperature. Thus,                       portion protein dispensable permissive <br /> appears majority DNA fully replicated dbf8             temperature. Our data indicate dbf8 mutants spend a <br /> cells restrictive temperature. Therefore, appears             major cycle fully replicated, intact DNA, even <br /> the defect dbf8 cells unlikely incomplete DNA                    permissive temperature. Thus, dbf8 mutants not <br /> replication.                                                                 appear primary defect DNA metabolism. dbf8 <br />  6358        HOUMAN AND HOLM                                                                                                                  MOL. CELL. BIOL. <br />  <br />                             TABLE 5. Nondisjunction chromosome loss permissive restrictive temperatures <br /> Strain incubation                                                                    No. indicated phenotype (per 103 colonies)a <br />       temp (0C)                                                        Red                         White                        Red-white             Red-pink <br /> dbf8ldbf8 <br />   Before incubation                       5,395                      46 &#177; 20                      59 &#177; 14                       12.5 &#177; 2.0           5.2 &#177; 2.0 <br />   25                                      3,968                      38 &#177; 18                      54 &#177; 2                        12.6 &#177; 1.0           7.8 &#177; 3.5 <br />   35                                      2,659                      40 &#177; 10                      49 &#177; 10                       17.6 &#177; 4.0           9.9 &#177; 2.6 <br /> DBF8+IDBF8+ <br />   Before incubation                       3,609                     0.8 &#177; 0.8                     1.9 &#177; 0.9                      0.3 &#177; 0.3               0 <br />   25                                      8,509                     0.2 &#177; 0.2                     1.7 &#177; 1.1                          0               0.1 &#177; 0.1 <br />   35                                      6,179                     0.6 &#177; 0.1                     2.2 &#177; 1.6                          0               0.2 &#177; 0.2 <br />   For clone, independent cultures examined. Average values + standard deviations shown. Red-white half-sectored colonies represent <br /> nondisjunction events (2:0 segregation); red-pink half-sectored colonies represent chromosome loss events (1:0 segregation). <br />  <br />  <br />  <br /> mutants exhibit sizable chromosome instability phenotype,                       Like mitotic mutants, dbf8 mutants undamaged DNA, <br /> suggesting Dbf8p plays role mitosis.                                    accumulate large buds short spindle, signifi- <br />    Like cell cycle mutants, dbf8 mutants undergo arrest                      cantly reduced chromosome stability. This constellation of <br /> following multiple cell cycles restrictive temperature (1, 8,                  phenotypes closely shared mutations that <br /> 15, 35). It surprising multicycle arrest mutants                  recovered phenotypic screens chromosome <br /> found morphological screens cdc dbf screens,                    instability (9, 25, 47, 58). Like dbf8 mutants, class of <br /> because long incubation periods frequently occur anal-                     mutants exhibits high levels chromosome loss nondis- <br /> ysis. The significance multicycle arrest depends                      junction, cases permissive temperature (25, <br /> characteristics mutation. Since first-division cell cycle                  47). The accumulation large-bud cells variable, ranging <br /> arrests dependent checkpoint mechanisms (54),                      30 70%, cases accumulation appears <br /> multicycle arrest represent evasion checkpoints,                     caused delay mitosis. In summary, phenotype <br /> has proposed components spindle pole body                        dbf8 mutants similar ctf mutants, <br /> duplication (3, 6, 50, 55). Alternatively, multicycle arrest can <br /> be caused conditional defects synthesis                     especially ctf12, ctf13, ctf14, ctf16, exhibit 50 70% <br /> protein assembly complex (11, 15); cases,                    large-budded cells restrictive temperature (9, 47). Thus, <br /> the expression mutant phenotype delayed                       appears DBF8 gene likely encode mitotic <br /> the existing protein complex diluted degraded. Finally,                  function. <br /> a multicycle arrest caused leaky allele. This                          The carboxyl terminus Dbf8p appears essential at <br /> possibility appears likely genes CDC2, CDC14,                     restrictive temperature permissive temper- <br /> CDC16, different alleles exhibit single cycle                ature. It possible carboxyl terminus required for <br /> or multicycle arrests recovered (15). The fact                 stability proper folding protein restrictive <br /> the alleles dbf8 multicycle arrest makes                 temperature. Since phenotype dbfJ mutations partially <br /> likely mutations simply leaky,                       expressed permissive temperature, however, the <br /> cannot completely rule possibility.                                      carboxy terminal Dbf8 play significant nonessen- <br />    The multicycle arrest dbf mutants makes difficult                     tial role permissive temperature. <br /> perform potentially informative experiments. In partic-                         Dbf8p exhibits striking similarity cut8+ gene product <br /> ular, useful determine time action                   S. pombe (26% identity 50% similarity the <br /> DBF8 gene product using reciprocal shift analysis (19). In                     length proteins [40]). S. pombe cut genes originally <br /> reciprocal shift experiments, independent cell cycle blocks                   identified morphological screen conditionally lethal <br /> are imposed sequentially; ability cell continue                 mutations deregulate mitosis (21, 39). In cut mutants, cell <br /> cell cycle assessed. Unfortunately, impractical                  division continues absence nuclear division, the <br /> first arrest cells dbf8 block, dbf8 arrest                  catastrophic consequence loss genetic material. Several <br /> irreversible. In addition, cells arrested                    cut genes involved mitotic processes spindle pole <br /> another point cell cycle, multicycle nature dbf8 <br /> mutation preclude meaningful analysis. Thus, reciprocal                     body duplication, encodes putative microtubule <br /> shift experiments helpful determining func-                       motor (13, 52). Both cut8 mutants dbf8 mutants continue <br /> tion Dbf8p. Fortunately, however, informa-                        divide restrictive temperature, appear <br /> tive experiments provided useful hints function                      defects mitosis. In addition, cut8 mutants exhibit <br /> DBF8p.                                                                            cytologically visible defects chromosome segregation (21, <br />    Chromosome loss nondisjunction associated                      40). Interestingly, cut8-563 mutation maps serine a <br /> defects DNA synthesis mitosis (16, 36),                          position equivalent amino acid 261 Dbf8p, is <br /> additional information indication primary                      situated region discovered nonsense mutations <br /> defect. For example, DNA synthesis mutants usually exhibit                        DBF8. The extent similarity proteins <br /> increased levels recombination spontaneous mutation;                     diverges region. In light dispensability of <br /> addition, exhibit sensitivity radiation interact                   carboxy terminus Dbf8p permissive temperature, <br /> with checkpoint proteins sense damaged DNA (11, 16,                      interesting note shared homology <br /> 18, 23, 30, 34, 36). Mitotic mutants, hand, tend                  occurs amino-terminal two-thirds proteins. De- <br /> show greater increase chromosome nondisjunction events,                      termination Dbf8p Cut8p functional ho- <br /> and seldom exhibit damaged DNA (16, 22, 25, 56, 58).                         mologs awaits interspecies complementation analysis. <br />  VOL. 14, 1994                                                         DBF8 IS REQUIRED FOR CHROMOSOME SEGREGATION                                6359 <br />  <br />                        ACKNOWLEDGMENTS                                       19. Hereford, L. M., L. H. Hartwell. 1974. Sequential gene <br />                                                                                  function initiation Saccharomyces cerevisiae DNA synthe- <br />    We thank Kaaren Kemp Espenshade, Adrian Halme, Michelle                   sis. J. Mol. Biol. 84:445-461. <br /> Tuffo excellent technical assistance, Elizabeth Purswell assisting   20. Hieter, P., C. Mann, M. Snyder, R. Davis. 1985. Mitotic <br /> with graphics typing manuscript, Michael                    stability yeast chromosomes: colony color assay measures <br /> McAlear identification dbflO. We thank present members             nondisjunction chromosome loss. Cell 40:381-392. <br /> Holm laboratory, Elizabeth Amatruda Howell, Rachelle Spell           21. Hirano, T., S. Funahashi, T. Uemura, M. Yanagida. 1986. <br /> critical reading manuscript. Special thanks Leland             Isolation characterization <span id='am-51' about='obo:NCBITaxon_4896' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_1 obo:NCBITaxon_147554 obo:NCBITaxon_34346 obo:NCBITaxon_4890 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4895 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-52' property="oboInOwl:hasExactSynonym" content="fission yeast" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces pombeP" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-55'  ></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45042" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces malidevorans" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="Schizosaccharomyces pombe" datatype="xsd:string"></span>Schizosaccharomyces pombe</span> cut <br /> Johnston generously providing dbf strains, Mark Goebl                mutants block nuclear division cytokinesis. EMBO J. <br /> informing cut8 homology, Mitsuhiro Yanagida                 5:2973-2979. <br /> sharing information prior publication.                                    22. Holm, C., T. Stearns, D. Botstein. 1989. DNA topoisomerase <br />   F.H. fellow Jane Coffin Childs Memorial Fund                   II act mitosis prevent nondisjunction chromosome <br /> Medical Research. This work supported award C.H.               breakage. Mol. Cell. Biol. 9:159-168. <br /> the National Institutes Health (GM36510).                                 23. Howell, L, M. McAlear, D. Rose, C. Holm. 1994. CDC44: a <br />                                                                                  putative nucleotide-binding protein required cell cycle progres- <br />                              REFERENCES                                          sion homology subunits replication factor C. Mol. <br />  1. Araki, H., K. Awane, N. Ogawa, Y. Oshima. 1992. The CDC26                Cell. Biol. 14:255-267. <br />     gene Saccharomyces cerevisiae required cell growth at <br />                                                                              24. Hoyt, M. A., L. He, K. K. Loo, W. S. Saunders. 1992. Two <br />     high temperature. Mol. Gen. Genet. 231:329-331.                              Saccharomyces cerevisiae kinesin-related gene-products required <br />  2. Ausubel, F. M., R. Brent, R. E. Kingston, D. D. Moore, J. G.                 mitotic spindle assembly. J. Cell Biol. 118:109-120. <br />     Seidman, J. A. Smith, K. Struhl. 1989. Current protocols          25. Hoyt, M. A., T. Stearns, D. Botstein. 1990. Chromosome <br />     molecular biology. Greene, Brooklyn, N.Y.                                    instability mutants Saccharomyces cerevisiae defective in <br />  3. Baum, P., C. Yip, L. Goetsch, B. Byers. 1988. A yeast gene               microtubule-mediated processes. Mol. Cell. Biol. 10:223-234. <br />     essential regulation spindle pole duplication. Mol. Cell. Biol.   25a.Johnston, L. Personal communication. <br />     8:5386-5397.                                                             26. Johnston, L. H., S. L. Eberly, J. W. Chapman, H. Araki, A. <br />  4. Boeke, J. D., F. LaCroute, G. R. Finlk 1984. A positive                  Sugino. 1990. The product Saccharomyces cerevisiae cell <br />     selection mutants lacking orotidine-5&apos;-phosphate decarboxyl-             cycle gene DBF2 homology protein kinases is <br />     ate activity yeast: 5-fluoro-orotic acid resistance. Mol. Gen.            periodically expressed cell cycle. Mol. Cell. Biol. 10:1358- <br />     Genet. 197:345-347.                                                          1366. <br />  5. Budd, M., J. L. Campbell. 1987. Temperature-sensitive mu-            27. Johnston, L. H., A. P. Thomas. 1982. The isolation new <br />     tants yeast DNA polymerase I. Proc. Natl. Acad. Sci. USA                  DNA synthesis mutants yeast Saccharomyces cerevisiae. Mol. <br />     84:2838-2842.                                                                Gen. Genet. 186:439-444. <br />  6. Chan, C. S. M., D. Botstein. 1993. Isolation characteriza-       28. Jones, J. S., L. Prakash. 1990. Yeast Saccharomyces cerevisiae <br />     tion chromosome-gain increase-in-ploidy mutants yeast.             selectable markers pUC18 polylinkers. Yeast 6:363-366. <br />     Genetics 135:677-691.                                                    29. Kitada, K., L. H. Johnston, T. Sugino, A. Sugino. 1992. <br />  7. Conde, J., G. R. Fink. 1986. A mutant Saccharomyces                   Temperature-sensitive cdc7 mutations Saccharomyces cerevisiae <br />     cerevisiae defective nuclear fusion. Proc. Natl. Acad. Sci. USA          suppressed DBF4 gene, required G,/S <br />     73:3651-3655.                                                                cell cycle transition. Genetics 131:21-29. <br />  8. Dickinson, J. R. 1991. Biochemical genetic studies            30. Kouprina, N., E. Kroll, V. Bannikov, V. Bliskovsky, R. Gizatullin, <br />     function of, relationship between, PGII- CDC30-                  A. Kirillov, V.   Zakharyev, P. Hieter, F. Spencer, V. Larionov. <br />     encoded phosphoglucose isomerases Saccharomyces cerevisiae.               1992. CTF4 (CHLI5) mutants exhibit defective DNA metabolism <br />     J. Gen. Microbiol. 137:765-770.                                              yeast Saccharomyces cerevisiae. Mol. Cell. Biol. 12:5736- <br />  9. Doheny, K. F., P. K. Soger, A. A. Hyman, S. Tugendreich, F.                  5747. <br />     Spencer, P. Hieter. 1993. Identification essential compo-         31. Kouprina, N., A. Tsouladze, M. Koryabin, P. Hieter, F. Spencer, <br />     nents S. cerevisiae kinetochore. Cell 73:761-774.                     V. Larionov. 1993. Identification genetic mapping CHL <br /> 10. Gerring, S. L., F. Spencer, P. Hieter. 1990. The CHLJ (CTFJ)             genes controlling mitotic chromosome transmission yeast. Yeast <br />     gene product Saccharomyces cerevisiae important chro-              9:11-19. <br />     mosome transmission normal cell cycle progression GJM.            32. McAlear, M. A., E. A. Howell, K. K. Espenshade, C. Holm. <br />     EMBO J. 9:4347-4358.                                                         Proliferating cell nuclear antigen (pol30) mutations suppress cdc44 <br /> 11. Gibson, S. I., R. T. Surosky, B.-K. Tye. 1990. The phenotype             mutations identify potential regions interaction between <br />     minichromosome maintenance mutant mcm3 character-                  encoded proteins. Mol. Cell. Biol. 14:4390-4397. <br />     istic mutants defective DNA replication. Mol. Cell. Biol.          33. Meluh, P. B., M. D. Rose. 1990. KAR3, kinesin-related gene <br />     10:5707-5720.                                                                required yeast nuclear fusion. Cell 60:1029-1041. <br /> 11a.Goebl, M. Personal communication.                                        34. Miles, J., T. Formosoa. 1992. Evidence POB1, Saccha- <br /> 12. Gyllensten, U. B., H. A. Erlich. 1988. Generation single-             romyces cerevisiae protein binds DNA polymerase a, acts in <br />     stranded DNA polymerase chain reaction applica-               DNA metabolism vivo. Mol. Cell. Biol. 12:5724-5735. <br />     tion direct sequencing HLA-DQA locus. Proc. Natl.              35. Mirabito, P. M., N. R. Morris. 1993. BIMA, TPR-containing <br />     Acad. Sci. USA 85:7652-7656.                                                 protein required mitosis, localizes spindle pole body in <br /> 13. Hagan, I., M. Yanagida. 1992. Kinesin-related cut7 protein               <span id='am-60' about='obo:NCBITaxon_5052' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5042 obo:NCBITaxon_33154 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-61' property="rdfs:label" content="Aspergillus" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63'  ></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasAlternativeId" content="NCBITaxon:37503" datatype="xsd:string"></span>Aspergillus</span> nidulans. J. Cell Biol. 120:959-968. <br />     associates mitotic meiotic spindles fission yeast. Nature    36. Palmer, R. E., E. Hogan, D. Koshland. 1990. Mitotic trans- <br />     (London) 356:74-76.                                                          mission artificial clones cdc mutants yeast, Saccharo- <br /> 14. Hartwell, L. H. 1976. Sequential function gene products relative          myces cerevisiae. Genetics 125:763-774. <br />     DNA synthesis yeast cell cycle. J. Mol. Biol. 104:803-817.     37. Penn, M. D., G. Thireos, H. Greer. 1984. Temporal analysis of <br /> 15. Hartwell, L. H., R. K. Mortimer, J. Culotti, M. Culotti. 1973.           general control amino acid biosy <br /> </body></html>