<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Serum Resistome Globally Disseminated <br /> Multidrug Resistant Uropathogenic <span id='am-106' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-107' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-114' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-118'  ></span><span id='am-119' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span>Escherichia coli</span> <br /> Clone <br /> Minh-Duy Phan1,2, Kate M. Peters1,2, Sohinee Sarkar1,2, Samuel W. Lukowski3&#164;, Luke P. Allsopp1,2, <br /> Danilo Gomes Moriel1,2, Maud E. S. Achard1,2, Makrina Totsika1,2, Vikki M. Marshall3, Mathew Upton4, <br /> Scott A. Beatson1,2, Mark A. Schembri1,2* <br /> 1 Australian Infectious Diseases Research Centre, The University Queensland, Brisbane, Queensland, Australia, 2 School Chemistry Molecular Biosciences, The <br /> University Queensland, Brisbane, Queensland, Australia, 3 Queensland Brain Institute, The University Queensland, Brisbane, Queensland, Australia, 4 Faculty of <br /> Medicine Dentistry, University Plymouth, Plymouth, United <span id='am-238' about='obo:NCBITaxon_kingdom' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-239' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-240' property="rdfs:label" content="kingdom" datatype="xsd:string"></span>Kingdom</span> <br />  <br />  <br />  <br />      Abstract <br />      <span id='am-122' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-123' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-125' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-126'  ></span><span id='am-127' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span>Escherichia</span> coli ST131 globally disseminated, multidrug resistant clone responsible high proportion urinary tract <br />      bloodstream infections. The rapid emergence successful spread E. coli ST131 strongly associated with <br />      antibiotic resistance; however, phenotype unlikely explain dominance multidrug resistant <br />      uropathogens circulating worldwide hospitals community. Thus, greater understanding molecular <br />      mechanisms underpin fitness E. coli ST131 required. In study, employed hyper-saturated transposon <br />      mutagenesis combination multiplexed transposon directed insertion-site sequencing define essential genes <br />      required vitro growth serum resistome (i.e. genes required resistance human serum) E. coli EC958, a <br />      representative predominant E. coli ST131 clonal lineage. We identified 315 essential genes E. coli EC958, 231 (73%) <br />      essential E. coli K-12. The serum resistome comprised 56 genes, majority encode <br />      membrane proteins factors involved lipopolysaccharide (LPS) biosynthesis. Targeted mutagenesis confirmed role in <br />      serum resistance 46 (82%) genes. The murein lipoprotein Lpp, lipid A-core biosynthesis enzymes <br />      WaaP WaaG, strongly associated serum resistance. While LPS main resistance mechanism <br />      defined E. coli EC958 serum, enterobacterial common antigen colanic acid impacted phenotype. <br />      Our analysis identified novel function genes, hyxA hyxR, minor regulators O-antigen chain length. <br />      <span id='am-30' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-32'  ></span><span id='am-33' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> study offers novel insight genetic make-up E. coli ST131, provides framework future research E. <br />      coli Gram-negative pathogens define essential gene repertoire dissect molecular mechanisms <br />      enable survive bloodstream cause disease. <br />  <br />   Citation: Phan M-D, Peters KM, Sarkar S, Lukowski SW, Allsopp LP, et al. (2013) The Serum Resistome Globally Disseminated Multidrug Resistant <br />   Uropathogenic Escherichia coli Clone. PLoS Genet 9(10): e1003834. doi:10.1371/journal.pgen.1003834 <br />   Editor: Paul M. Richardson, MicroTrek Incorporated, United States America <br />   Received April 11, 2013; Accepted August 12, 2013; Published October 3, 2013 <br />   Copyright: &#223; 2013 Phan et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported grant Australian National Health Medical Research Council [APP1012076]. MAS supported an <br />   Australian Research Council (ARC) Future Fellowship [FT100100662]. SAB supported ARC Australian Research Fellowship [DP0881347]. MT was <br />   supported ARC Discovery Early Career Researcher Award [DE130101169]. The funders role study design, <span id='am-246' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-247' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-248' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-249'  ></span><span id='am-250' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> collection analysis, decision to <br />   publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: m.schembri@uq.edu.au <br />   &#164; Current address: Department Genetic Medicine Development, University Geneva, Geneva, Switzerland. <br />  <br />  <br />  <br />  <br /> Introduction                                                                         fluoroquinolones [3,10]. Indeed, fluoroquinolone-resistant E. <br />                                                                                      coli strains belong recently emerged dominant subgroup of <br />     <span id='am-234' about='obo:NCBITaxon_941280' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-235' property="rdfs:label" content="Escherichia coli O25b:H4" datatype="xsd:string"></span><span id='am-236' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-237' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia coli O25b:H4</span>-ST131 (E. coli ST131) recently                     ST131 strains [11]. Some E. coli ST131 strains been <br /> emerged, globally disseminated clone multidrug                         reported produce carbapenemases [12&#8211;14], severely <br /> resistant responsible high proportion community-                     limiting treatment options currently available this <br /> and nosocomially-acquired urinary tract bloodstream infec-                       clinically predominant clone [15]. <br /> tions [1&#8211;6]. E. coli ST131 strains capable causing                          E. coli ST131 strains, like uropathogenic E. coli <br /> complicated infections including acute pyelonephritis, osteomye-                     (UPEC) strains, derived phylogenetic group B2 [3]. <br /> litis, septic arthritis septic shock [7,8]. E. coli ST131                    Typically, UPEC strains possess large diverse range of <br /> commonly associated production CTX-M-15 enzyme,                          virulence factors contribute ability cause urinary <br /> currently widespread extended spectrum b-lactamase                          tract bloodstream infections, including adhesins, toxins, <br /> (ESBL) type world [1,9]. In addition resistance                     siderophores protectins [15,16]. Several studies demon- <br /> against oxyimino-cephalosporins (i.e. cefotaxime, ceftazidime),                  strated E. coli ST131 strains possess similar suite virulence <br /> monobactams, E. coli ST131 strains co-resistant                         factors cause invasive disease, leading hypothesis that <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                             1                          October 2013 | Volume 9 | Issue 10 | e1003834 <br />                                                                                                           Serum Resistome E. coli ST131 Strain <br />  <br />  <br />  <br />   Author Summary                                                              elements, little known combination factors that <br />                                                                               contribute resistance single strain. <br />   The emergence rapid dissemination new bacterial                         High-throughput transposon mutagenesis combined ge- <br />   pathogens presents multiple challenges healthcare                        nome-wide targeted sequencing used recently study the <br />   systems, including need rapid detection, precise                    essential genes <span id='am-155' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-156' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-167'  ></span>Salmonella</span> enterica serovar Typhi <span id='am-183' about='obo:NCBITaxon_75' typeof='owl:Thing obo:NCBITaxon_204458 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_28211 obo:NCBITaxon_76892'><span id='am-184' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-186' property="rdfs:label" content="Caulobacter" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasDbXref" content="PMID:10425763" datatype="xsd:string"></span><span id='am-188' property="oboInOwl:hasDbXref" content="PMID:10319468" datatype="xsd:string"></span><span id='am-189'  ></span>Caulobacter</span> <br />   diagnostics, effective transmission control effective                   crescentus [41,42]. Langridge et al. used transposon <br />   treatment. E. coli ST131 example recently                        directed insertion-site sequencing (TraDIS) method assay every <br />   emerged multidrug resistant pathogen capable                     gene role survival S. Typhi presence bile <br />   causing urinary tract bloodstream infections with <br />                                                                               salts [42]. Similar approaches (INSeq, HITS, Tn-seq) also <br />   limited available treatment options. In <span id='am-35' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> increase <br />                                                                               applied range organisms study gene requirements <br />   molecular understanding E. coli ST131, devel- <br />   oped high-throughput transposon mutagenesis                        survival particular niches [43&#8211;46]. Here, adapted <br />   combination generation sequencing test                      TraDIS designed multiplexing method define the <br />   gene essential role growth                     essential genes required vitro growth (i.e. <span id='am-1' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-2' property="rdfs:label" content="Luria" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Luria</span>-Bertani agar <br />   contribution serum resistance. We identified 315                         media supplemented 30 mg/ml Cm 37uC) serum <br />   essential genes, 270 conserved                      resistome (i.e. genes required resistance human serum, E. <br />   currently available complete E. coli genomes. Fifty-six                     coli EC958). We essential gene list E. coli EC958 <br />   genes define serum resistome E. coli ST131                      comprises 315 genes, 231 shared E. coli K-12. <br />   identified, including genes encoding membrane                          We define time comprehensive inventory of <br />   proteins, proteins involved LPS biosynthesis, regulators                 genes required resistance human serum. Our study provides <br />   novel proteins previously unknown                          molecular blueprint understanding mechanisms em- <br />   function. This study provides inventory                     ployed E. coli ST131 survive, grow bloodstream and <br />   essential serum resistance genes form                      cause disease. <br />   framework future development targeted <br />   therapeutics prevent disease caused multidrug- <br />   resistant E. coli ST131 strains.                                            Results <br />                                                                               Application multiplexed TraDIS E. coli ST131 <br /> the widespread pathogenic success E. coli ST131 strains be <br />                                                                               strain EC958 <br /> in enhanced virulence [7,8,17]. However,                       Approximately 1 million mutants generated E. coli <br /> become clear recent studies E. coli ST131 strains            ST131 strain EC958 [22] using in-house miniTn5 transposon <br /> possess heightened virulence potential compared UPEC               carrying chloramphenicol (Cm) resistance gene derived the <br /> or B2 E. coli strains causing invasive infections [18] infections       pKD3 plasmid [47]. A primer comprising functional regions <br /> in <span id='am-168' about='obo:NCBITaxon_333870' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_195229 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41087 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_71193 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-169' property="oboInOwl:hasRelatedSynonym" content="Nematodes Berthold, 1827" datatype="xsd:string"></span><span id='am-170' property="rdfs:label" content="Nematodes" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-172'  ></span><span id='am-173' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>nematodes</span> zebrafish embryos [19]. Thus, factors             designed facilitate specific sequencing transposon <br /> as enhanced metabolic capacity proposed contribute               insertion sites Illumina HiSeq 2000 platform allowing <br /> to fitness pathogenic success dominant clone                  intra-lane multiple sample indexing (Figure S1). This primer <br /> [20,21].                                                                      contained (59-39): (i) P5 sequence bind TruSeq flowcells, <br />    The genome sequence best-characterized E. coli               (ii) Illumina read 1 sequencing primer binding site, (iii) 6-bp <br /> ST131 strains, EC958, recently described [22]. E. coli EC958              index sequence multiple sample barcoding (iv) 25-bp <br /> is member pulsed-field gel electrophoresis (PFGE) defined            transposon specific sequence designed amplify 12 bp of <br /> UK epidemic strain A, represents major                       transposon adjacent genomic sequence. Using this <br /> pathogenic lineages (PFGE strains A&#8211;E) ESBL producing E. coli              custom primer, successfully sequenced transposon insertion <br /> causing urinary tract infections (UTI) UK [23]. E. coli            sites 6 samples TruSeq version 2 version 3 <br /> EC958 resistant antibiotic classes, including oxyimino-           flowcells (Figure 1A). Each sample yielded 6.8 million 15 <br /> cephalosporins, fluoroquinolones sulphonamides. E. coli                   million reads tagged transposon specific sequence, <br /> EC958 colonizes bladder mice type 1 fimbriae-                     71% reliably mapped EC958 draft chromosome <br /> dependent manner [22], invade bladder epithelial cells               (excluding unscaffolded contigs <span id='am-268' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-269' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-270' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-271' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-272' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span>) (Table 1). All <br /> and form intracellular bacterial communities, establish              experiments performed duplicate, correlation <br /> acute chronic UTI [24] inhibit contraction                 coefficient number insertions gene pair of <br /> ureters, vitro [25].                                                       samples close 1 (R2.0.99) demonstrating high level <br />    The ability resist bactericidal activity serum,         reproducibility experiment (Figure 1B). <br /> survive bloodstream, represents essential virulence trait <br /> for UPEC extra-intestinal E. coli (ExPEC) strains,                  Essential genes E. coli EC958 <br /> including E. coli ST131 [26&#8211;28]. In E. coli, mechanisms                  We initially used saturated random insertion mutant library <br /> have shown contribute serum resistance. The                        determine &#8216;essential genes EC958&#8217;, defined set of <br /> importance O-antigens K capsules resistance serum                genes required growth LB agar supplemented Cm <br /> has recognized 1960s 1980s, respectively [29&#8211;              30 mg/ml. We extracted genomic DNA (in duplicates: input A and <br /> 31]; multiple types, combinations length contribute             B) directly library pool sequenced using our <br /> differently serum resistance [32&#8211;35]. The major outer                      multiplexed TraDIS protocol. We combined reads input <br /> membrane protein OmpA [36], plasmid-encoded proteins TraT                     A B maximize coverage resulting 16 million <br /> [37,38] Iss [39], phage membrane protein Bor [40]                 transposon-tagged reads, 11 million uniquely mapped to <br /> have reported contribute serum resistance E. coli.         EC958 chromosome, resulting 502,068 unique insertion <br /> Notably, resistance mechanisms studied              sites. This equates average insertion site every <br /> isolation different strain backgrounds. Thus, serum              9.92 bp, low probability having 100 consecutive bp <br /> resistance clearly complex phenotype determined multiple              interruption chance (P = 4.261025). <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                      2                      October 2013 | Volume 9 | Issue 10 | e1003834 <br />                                                                                                            Serum Resistome E. coli ST131 Strain <br />  <br />  <br />  <br />  <br /> Figure 1. Summary sequence read data multiplexed TraDIS. (A) Number tagged reads obtained lane TruSeq version 2 <br /> and version 3 flowcells. (B) Correlation biological replicates. The number insertions gene Test A plotted against <br /> that Test B; correlation coefficient R2 indicates low variation replicates. <br /> doi:10.1371/journal.pgen.1003834.g001 <br />  <br />    The essential gene list identified using statistical analysis          investigated. An additional 17 proteins present more <br /> similar described Langridge et al., recognized             90% genomes. Only 6 proteins specific EC958 <br /> distinct distributions insertion indexes (number insertions               (not 50 genomes) (Table S3). <br /> divided gene length) non-essential genes (gamma) and <br /> essential genes (exponential) called insertion                   The serum resistome E. coli EC958 <br /> indexes equal intercept distributions               Saturated transposon mutagenesis combination next- <br /> as essential [42]. In data, insertion index cut-off 0.0158,           generation sequencing powerful tool genome, high- <br /> resulted identification 315 genes essential (Table S1).            throughput identification candidate genes involved a <br /> This cut-off equivalent log2-likelihood-ratio (LLR) 23.6,            particular phenotype. Here, used transposon mutant <br /> which means essential genes 12 times                 library combination TraDIS identify genes EC958 <br /> likely belong exponential distribution (essential)           involved resistance human serum, enabling define <br /> gamma (non-essential) distribution.                                             serum resistome EC958. We designed mutant selection <br />    The functional category gene identified based                 procedure 1 million mutants exposed pooled <br /> the COG (Clusters Orthologous Groups) numbers                       fresh human serum 90 minutes allowed grow LB <br /> EC958 annotation (accession number PRJEA61443). Figure 2                        broth 4 hours genomic DNA extraction. This procedure <br /> shows overview essential functions EC958 compared                 permitted growth serum resistant mutants eliminating <br /> the total number genes functional category. Genes                    inhibiting mutants sensitive serum (Figure 3A). The <br /> involved translation, ribosomal structure biogenesis account             procedure performed parallel control samples where <br /> for 25% total number essential genes EC958,               fresh serum replaced inactivated serum lacked <br /> 42% total number genes category. The second                   bactericidal activity (data shown). The genomic DNA from <br /> most abundant category essential gene list comprised genes               test control samples sequenced using modified <br /> involved cell wall/membrane/envelope biogenesis (12%),                       Illumina multiplexed TraDIS procedure (Figure 3B) generate <br /> followed genes involved coenzyme transport metabo-                    multiple datasets (Figure 1A) analysed the <br /> lism. There 23 essential genes functions identified            Bioconductor package edgeR filtering genes identified <br /> the COG database.                                                               essential [48]. <br />    To investigate conservation EC958 essential proteins                     The serum resistance genes identified genes have <br /> among different E. coli pathotypes, performed tfastx alignment               significant reduction read counts test samples compared <br /> (FASTA v36) essential protein sequences                         control samples (i.e. mutants survived serum <br /> translated DNA 50 E. coli complete genomes (Table S2).                     treatment) (Table S4). A stringent threshold log2 fold change of <br /> There 270 (86%) proteins conserved genomes                      read counts (logFC) 21 adjusted p-value than <br />  <br />  Table 1. Summary sequencing mapping results multiplexed TraDIS runs. <br />  <br />  <br />                                Tagged reads                Reads mapped EC958 chromosome (%)             Unique insertion sites EC958 <br />  <br />  Input A &amp; B                   16,281,107                  11,558,154 (70.99)                               502,068 <br />    Test A                      6,819,691                   4,879,098 (71.54)                                438,706 <br />    Test B                      7,572,730                   5,408,136 (71.41)                                436,858 <br />    Ctrl A                      8,350,687                   5,918,766 (70.87)                                394,521 <br />    Ctrl B                      15,006,139                  10,727,954 (71.49)                               485,459 <br />  <br />  doi:10.1371/journal.pgen.1003834.t001 <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                       3                       October 2013 | Volume 9 | Issue 10 | e1003834 <br />                                                                                                       Serum Resistome E. coli ST131 Strain <br />  <br />  <br />  <br />  <br /> Figure 2. Number essential genes COG functional category. <br /> doi:10.1371/journal.pgen.1003834.g002 <br />  <br />  <br /> 0.001 used identify significant genes involved          affect LPS rfaH (EC958_4322), encoding a <br /> serum resistance (Figure S2). Figure 4 shows names                 known regulator required LPS biosynthesis [51,52] and <br /> genomic locations 56 genes satisfied stringent           virulence pathogenic E. coli strains [53,54], whilst others <br /> criteria.                                                                  represent genes previously shown be <br />    Twenty-two (39.3%) genes belong operons                 associated serum resistance. The murein lipoprotein gene lpp <br /> responsible LPS biosynthesis (including O-antigen                 (EC958_1897) showed greatest difference test and <br /> biosynthesis lipid A-core biosynthesis) enterobacterial            control samples (logFC 210), followed pgm (EC958_0806), <br /> common antigen (ECA) biosynthesis. This result represents              encoding phosphoglucomutase. Four hypothetical proteins were <br /> first layer evidence demonstrating importance O25            identified, (EC958_0460 EC958_0461) <br /> antigen ECA E. coli ST131 background                  characterized study (see below). <br /> protection bactericidal activity human serum. <br /> Detailed characterisation O25 antigen gene cluster               Validation serum resistance genes <br /> discussed subsequent sections. ECA common                        As mentioned above, employed stringent threshold <br /> <span id='am-93' about='obo:NCBITaxon_543' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236'><span id='am-94' property="oboInOwl:hasDbXref" content="PMID:16166704" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasRelatedSynonym" content="Enterobacteraceae" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="PMID:10555334" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="Enterobacteriaceae" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasRelatedSynonym" content="gamma-3 proteobacteria" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="PMID:10555323" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="Enterobacteraceae (ex Lapage 1979) Lapage 1982, fam. nov., nom. rev." datatype="xsd:string"></span><span id='am-102'  ></span><span id='am-103' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="Enterobacteriaceae Rahn 1937" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasRelatedSynonym" content="Enterobacteriaceae (ex Rahn 1937) Ewing et al. 1980, fam. nov., nom. rev." datatype="xsd:string"></span>Enterobacteriaceae</span> expressed serum resistant (smooth)       combining fold change statistical significance define the <br /> and sensitive (rough) strains, rough strains           set 56 genes EC958 serum resistome. In order validate <br /> defective shared biosynthesis pathway affecting O-             findings, attempted test 56 genes independently for <br /> antigen ECA [49,50]. Seven 12 genes ECA                  role serum resistance. Using modified lambda red <br /> operon required serum resistance determined             mediated-homologous recombination approach [22,47] suc- <br /> TraDIS technique.                                                          cessfully generated defined knock-out mutants 54 genes <br />    The remaining 60.7% genes serum resistome identified          (96.4%) EC958. We unable obtain mutants the <br /> by TraDIS included genes encoding lipoprotein, membrane                    remaining 2 genes (acrA EC958_2373) despite multiple <br /> proteins, regulators hypothetical proteins (Table 2). Some          attempts (Table 2). <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                   4                      October 2013 | Volume 9 | Issue 10 | e1003834 <br />                                                                                                         Serum Resistome E. coli ST131 Strain <br />  <br />  <br />  <br />  <br /> Figure 3. Experimental design identify serum resistance genes EC958. (A) Selection steps employed using fresh serum test and <br /> inactivated serum control. (B) Schematic illustration Illumina sequencing procedure including use custom oligo indexing and <br /> enrichment insert sites. <br /> doi:10.1371/journal.pgen.1003834.g003 <br />  <br />  <br />  <br />  <br />    The 54 defined mutants subjected serum susceptibility            O25b antigen biosynthesis genes conferring resistance to <br /> testing, number surviving colonies 90-               serum <br /> minute exposure fresh pooled human serum compared                    It established O-antigen represents main <br /> the number colonies prior treatment (Table 2). A mutant           determinant serum resistance E. coli. However, are <br /> defined susceptible serum log difference        180 different O-antigens defined E. <br /> 1 (i.e. 10 fold reduction exposure serum). Forty-one genes         coli contribute differently serum resistance in <br /> (75.9%) contributed serum resistance EC958 using assay.          individual bacterial strains [32,33]. Furthermore, direct genetic <br /> In case remaining 13 mutants, possible lack       evidence linking O-antigen biosynthesis serum resistance only <br /> of susceptibility human serum reflection assay,             available small number specific O-antigen types. With this <br /> suggesting survival serum mixed population          mind, detailed characterization O25b biosynthesis <br /> million mutants different survival pure               genes performed using sequence comparison function <br /> population carrying defective mutation. Therefore,              prediction combination analysis LPS composition to <br /> better mimic condition TraDIS library serum selection,             deduce role gene resistance serum. <br /> competitive assay devised EC958 wild-type mixed                  Similar E. coli strains, O-antigen biosynthesis operon <br /> equally mutant exposure serum                      located galF gnd genes EC958. Figure 5A <br /> competitive index mutant measured. Using                    shows comparison EC958 O-antigen operon the <br /> competitive assay, mutants significantly            equivalent operon K-12 strain MG1655 O25 <br /> attenuated compared wild-type EC958 strain (Table 2).                serotype E. coli strain E47a [32,55]. <br /> Thus, overall number validated susceptible mutants 46               Nucleotide sugar biosynthesis genes. The genes <br /> out 54 tested (85.2%).                                                   encoding dTDP-a-L-rhamnose biosynthesis enzymes highly <br />                                                                             conserved strains, consistent presence a-L- <br /> Additional characteristics serum resistant mutants                       rhamnose O16 O25 antigen repeat unit expressed by <br />    We hypothesized mechanism associated enhanced              MG1655 EC958, respectively [56&#8211;58]. The dTDP-a-L- <br /> sensitivity human serum decreased membrane               rhamnose biosynthesis pathway starts a-D-glucose-1-<span id='am-273' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-274'  ></span><span id='am-275' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-276' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-277' property="rdfs:label" content="Phos" datatype="xsd:string"></span>phos</span>- <br /> integrity caused destabilization outer leaflet outer       phate produce dTDP-a-L-rhamnose catalysis RmlA <br /> membrane. In order test this, examined survival            (to make dTDP-a-D-glucose), RmlB (to make dTDP-4-dehydro-6- <br /> 54 mutants response outer membrane stresses (i.e. SDS)            deoxy-a-D-glucose), RmlC (to make dTDP-4-dehydro-6-deoxy-b- <br /> osmotic potential (i.e. NaCl). In total, 50.0% (27/54)               mannose) RmlD (to make dTDP-a-L-rhamnose), respectively. <br /> mutants displayed enhanced sensitivity SDS 11.1% (6/54)              LPS gel analysis EC958 mutants lacking four <br /> of mutants displayed enhanced sensitivity NaCl (Table 2). A          enzymes showed RmlC RmlD required the <br /> comparative analysis phenotypes context serum            biosynthesis O-antigen mutation gene resulted in <br /> sensitivity presented below.                                             cells lipid A core (Figure 5C). On contrary, the <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                    5                      October 2013 | Volume 9 | Issue 10 | e1003834 <br />                                                                                                          Serum Resistome E. coli ST131 Strain <br />  <br />  <br />  <br />  <br /> Figure 4. Overview E. coli EC958 serum resistance genes. The circular diagram depicts location 56 serum resistance genes the <br /> E. coli EC958 genome. The outer rings containing blue red arrows illustrate CDS serum resistance genes, respectively, forward <br /> and reverse strand genome. The inner red ring represents logFC control test samples CDS. The insets <br /> represent close-up look regions genome graphs showing location relative number mutant the <br /> control (blue) test (red) samples. <br /> doi:10.1371/journal.pgen.1003834.g004 <br />  <br /> LPS patterns rmlA rmlB mutants showed small                    polyhexamethylene biguanide suggested Wzx (also known as <br /> change intensity retained O-antigen chain                 RfbX) involved cellular stress responses [61]. In <br /> length distribution, long chain length band.             contrast previous studies, wzx gene defined as <br /> This explained existence RffH RffG,               essential study (Figure 4, Table S1). This be <br /> isozymes RmlA RmlB, EC958. RffH RffG, however,                 explained fact Cm used select transposon <br /> were able fully compensate RmlA RmlB resistance             mutants wzx mutants hypersensitive this <br /> to serum (Table 2). The inability produce long chain             antibiotic. The wzy O-antigen polymerase gene (EC958_2374) was <br /> length O-antigen rmlA rmlB mutants crucial                 defined essential EC958 further <br /> factor determining susceptibility human serum. Only              characterized. Mutation chain length regulator gene (wzz, <br /> the rmlC mutant displayed altered sensitivity SDS,                  EC958_2368) showed non-modal distribution O-antigen <br /> mutations rmlABCD did affect sensitivity NaCl (Table 2).           chain length LPS pattern (Figure 5C), observation <br />    The O-antigen operon E47a set nucleotide                consistent previous study describing role gene [62]. <br /> sugar biosynthesis genes (fnlABC) biosynthesis UDP-N-             The reduction long chain length O-antigen mutant is <br /> acetyl-a-L-fucosamine (UDP-FucNAc) UDP-N-acetyl-a-D-                    likely account serum sensitivity shown assay <br /> glucosamine (UDP-GlcNAc). Curiously, EC958 does                     (Table 2). <br /> these genes operon genome.                 Glycosyltransferases genes. Glycosyltransferases (GTs) are <br />    O-antigen processing genes. The O-antigen flippase (Wzx,                  required form glycosidic bonds sugars O- <br /> EC958_2377) O-antigen polymerase (Wzy, EC958_2374)                       antigen repeat unit. There 100,000 GTs within <br /> from EC958 highly similar corresponding genes                94 families (http://www.cazy.org). Based structure the <br /> E47a. In contrast, genes share low similarity               O25 repeat unit [63,64], predicted requirement 4 GTs in <br /> corresponding genes MG1655. Mutation wzx gene                 EC958. Bioinformatic analysis (blastp) confirmed are <br /> MG1655 results accumulation high levels UndPP-              4 GTs O-antigen operon: EC958_2371, <br /> O unit cytoplasm [59] hypersensitivity                 EC958_2372, EC958_2375 EC958_2376. Surprisingly, two <br /> antibiotics agents including nalidixic acid, tetracycline,         predicted GTs EC958 different in <br /> mitomycin C hydrogen peroxide [60]. Transcriptional                      E47a. Whether related different source a-L- <br /> analyses MG1655 responses broad-spectrum biocide                   FucNAc EC958 lack fnlABC remains be <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                     6                       October 2013 | Volume 9 | Issue 10 | e1003834 <br />                                                                                                                     Serum Resistome E. coli ST131 Strain <br />  <br />  <br />  <br />  Table 2. Genes required serum resistance identified multiplexed TraDIS. <br />  <br />  <br />                                                                                                                                      SDS     NaCl <br />  EC958        Gene      Matched                                                                                                   MICd    MICd    COG <br />                                                                                                                                c <br />  locus_tag    Name      MG1655         Product                                                             Serum           LPS       (%)     (M)     code <br />  <br />  EC958_0147   dnaJ      b0015          DnaJ chaperone                                                      1               WT        WT      WT      O <br />  EC958_0152   nhaA      b0019          Sodium/proton antiporter                                            0b              WT        WT      WT      P <br />  EC958_0260   acnB      b0118          aconitate hydratase 2/2-methylisocitrate dehydratase                1               WT        WT      WT      C <br />  EC958_0460   0460      -              -                                                                   3               X         WT      WT      - <br />  EC958_0461   0461      -              -                                                                   1b              X         WT      WT      TK <br />  EC958_0602   acrA      b0463          multidrug efflux                                             N/A             N/A       N/A     N/A     M <br />  EC958_0784   nagC      b0676          Transcriptional regulator, repressor N-acetylglucosamine         1               WT        WT      WT      KG <br />  EC958_0785   nagA      b0677          N-acetylglucosamine-6-phosphate deacetylase                         3               X         0.063   WT      G <br />  EC958_0806   pgm       b0688          Phosphoglucomutase                                                  4               X         0.016   WT      G <br />  EC958_0856   tolQ      b0737          membrane spanning protein Tol-Pal cell envelope complex          3               WT        0.031   WT      U <br />  EC958_0858   tolA      b0739          membrane anchored protein Tol-Pal cell envelope complex          3               WT        0.031   WT      M <br />  EC958_0859   tolB      b0740          periplasmic protein Tol-Pal cell envelope complex                0               WT        WT      WT      U <br />  EC958_0871   galE      b0759          UDP-galactose-4-epimerase                                           1               X         WT      WT      M <br />  EC958_1112   1112      -              -                                                                   1               X         WT      WT      - <br />  EC958_1114   1114      -              -                                                                   2               WT        WT      WT      - <br />  EC958_1897   lpp       b1677          murein lipoprotein                                                  6               WT        0.063   WT      M <br />  EC958_2368   wzz       b2027          regulator O-antigen chain length                                 1               X         WT      WT      M <br />  EC958_2371   2371      -              -                                                                   2               LA        0.063   WT      R <br />  EC958_2373   2373      -              -                                                                   N/A             N/A       N/A     N/A     E <br />  EC958_2378   rmlC      b2038          dTDP-4-deoxyrhamnose-3,5-epimerase                                  4               LA        0.063   WT      M <br />  EC958_2379   rmlA      b2039          glucose-1-phosphate thymidylyltransferase                           2               X         WT      WT      M <br />  EC958_2380   rmlD      b2040          dTDP-4-dehydrorhamnose reductase                                    4               LA        WT      WT      M <br />  EC958_2381   rmlB      b2041          dTDP-glucose 4,6 dehydratase                                        3               X         WT      WT      M <br />  EC958_2390   wcaI      b2050          glycosyl transferase involved colanic acid synthesis             0b              WT        WT      WT      M <br />                                                                                                                b <br />  EC958_2391   gmm       b2051          GDP-mannose mannosyl hydrolase                                      1               WT        WT      WT      F <br />  EC958_2394   wcaF      b2054          predicted acyl transferase                                          4               X         0.063   0.5     R <br />  EC958_2594   arnD      b2256          undecaprenyl phosphate-alpha-L-ara4FN deformylase                   2               WT        0.063   WT      G <br />  EC958_2596   arnE      b4544          ArnE/ArnF undecaprenyl-phosphate-a-L-Ara4N flippase                 1               WT        WT      WT      P <br />  EC958_2597   arnF      b2258          ArnE/ArnF undecaprenyl-phosphate-a-L-Ara4N flippase                 2               WT        WT      WT      - <br />  EC958_2652   dedA      b2317          conserved inner membrane protein                                    1b              WT        WT      WT      S <br />  EC958_2653   truA      b2318          pseudouridylate synthase I                                          0b              WT        WT      WT      J <br />  EC958_2822   bamB      b2512          lipoprotein required OM biogenesis, BamABCD complex          3               WT        0.063   WT      S <br />  EC958_3571   <span id='am-6' about='obo:NCBITaxon_468772' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_33154 obo:NCBITaxon_43733 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_50680 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50673 obo:NCBITaxon_50557 obo:NCBITaxon_50671'><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Nusa" datatype="xsd:string"></span>nusA</span>      b3169          transcription termination/antitermination L factor                  0b              WT        WT      WT      K <br />  EC958_3581   greA      b3181          transcript cleavage factor                                          2               WT        0.063   WT      K <br />  EC958_3621   sspA      b3229          stringent starvation protein A                                      2               WT        0.063   WT      O <br />  EC958_4029   waaL      b3622          O-antigen ligase                                                    3               LA        0.063   WT      M <br />  EC958_4030   waaK      b3623          Lipopolysaccharide core biosynthesis; heptosyl transferase IV       3               LA        0.063   WT      M <br />  EC958_4032   waaY      b3625          Lipopolysaccharide core heptose (II) kinase                         5               LA        WT      ,0.3    T <br />  EC958_4033   waaJ      b3626          UDP-glucose:(glucosyl)LPS a-1,2-glucosyltransferase                 2               LA        0.063   WT      M <br />  EC958_4034   waaI      b3627          UDP-D-glucose:(glucosyl)LPS a-1,3-glucosyltransferase               5               X         0.063   0.5     M <br />  EC958_4035   waaB      b3628          UDP-D-galactose:(glucosyl)LPS-1, 6-D-galactosyltransferase          4               X         WT      WT      M <br />  EC958_4037   waaP      b3630          Lipopolysaccharide core heptose (I) kinase                          6               X         0.016   WT      - <br />  EC958_4038   waaG      b3631          Lipopolysaccharide glucosyltransferase I                            6               X         0.016   0.5     M <br />  EC958_4050   pyrE      b3642          orotate phosphoribosyltransferase                                   2               WT        WT      WT      F <br />  EC958_4246   wecA      b3784          undecaprenyl-phosphate a-N-acetylglucosaminyl transferase           5               LA        0.063   WT      M <br />  EC958_4247   wzzE      b3785          ECA polysaccharide chain length modulation protein                  2               X         0.063   WT      M <br />  EC958_4252   wecD      b3790          TDP-fucosamine acetyltransferase                                    5               X         0.063   WT      - <br />  EC958_4253   wecE      b3791          TDP-4-oxo-6-deoxy-D-glucose transaminase                            2               WT        WT      WT      M <br />  <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                         7                         October 2013 | Volume 9 | Issue 10 | e1003834 <br />                                                                                                                              Serum Resistome E. coli ST131 Strain <br />  <br />  <br />  <br />  Table 2. Cont. <br />  <br />  <br />                                                                                                                                               SDS     NaCl <br />  EC958           Gene        Matched                                                                                                       MICd    MICd      COG <br />                                                                                                                                         c <br />  locus_tag       Name        MG1655           Product                                                               Serum           LPS       (%)     (M)       code <br />  <br />  EC958_4254      wzxE        b3792            lipid III flippase                                                    0b              WT        WT      WT        R <br />  EC958_4255      wecF        b4481            TDP-Fuc4NAc:lipidIIFuc4NAc transferase                                1b              WT        0.063   WT        - <br />  EC958_4256      wzyE        b3793            ECA polysaccharide chain elongation                                   0b              WT        0.063   WT        - <br />  EC958_4322      rfaH        b3842            DNA-binding transcriptional antiterminator                            5               LA        0.016   ,0.3      K <br />  EC958_4336      dsbA        b3860            Protein disulfide oxidoreductase                                      1               WT        0.031   0.6       O <br />  EC958_4480      pgi         b4025            glucosephosphate isomerase                                            1b              WT        0.063   WT        G <br />                                                                                                                         b <br />  EC958_4658      miaA        b4171            D(2)-isopentenylpyrophosphate tRNA-adenosine transferase              0               WT        0.063   WT        J <br />  EC958_4725      fbp         b4232            fructose-1,6-bisphosphatase I                                         3               WT        WT      WT        G <br />  <br />  a <br />    Serum sensitivity shown 2log10 viable count difference t = 90 min t = 0 min. The higher number sensitive mutant is. <br />  b <br />    Serum sensitivity determined competitive assays: 0 means difference wild-type mutant; 1 means mutant more <br />  attenuated wild-type. <br />  c <br />   LPS patterns coded followed: WT, similar wild-type strain, change observed; LA, band lipid A-core observed; LA+1, lipid A-core band only <br />  band it; X, changes observed, Supplementary Figure S3 band patterns. <br />  d <br />    The wild-type (WT) MIC 0.8 M NaCl 0.125% SDS. <br />  doi:10.1371/journal.pgen.1003834.t002 <br />  <br />  <br /> investigated. Since O25 antigen structure known,                             sensitive human serum SDS (except waaB and <br /> attempted predict glycosidic link formed GT using                     waaY mutant), waaG, waaI waaY mutants <br /> combination sequence similarity O-antigen structure                             displayed enhanced sensitivity NaCl (Table 2). <br /> comparison previously described [64]; results                         ECA biosynthesis genes. Seven serum resistant candidate <br /> shown Table 3 Figure 5B. We able mutate                            genes ECA biosynthesis wec operon identified by <br /> EC958_2371, EC958_2372 EC958_2375 defined                               TraDIS (Table 2). Two genes (wzxE wzyE) not <br /> as essential study. Analysis EC958_2371 mutant                          confirmed serum assays confer resistance changes <br /> confirmed EC958_2371 required O-antigen biosynthe-                         observed LPS patterns mutants (Table 2, <br /> sis (Figure 5C), serum resistance resistance SDS (Table 2).                     Figure S3). The wzyE mutant (but wzxE mutant) was, <br />                                                                                        however, sensitive SDS (Table 2). The LPS patterns the <br /> Other serum resistance mechanisms affecting LPS                                     wzzE mutant revealed changes lipid A-core the <br />                                                                                        modulation O-antigen chain length (Figure S3). WzxE has <br /> EC958                                                                                  previously shown preferentially form protein complex <br />    LPS gel analysis performed 54 defined mutants                         WzyE WzzE biosynthesis ECA Wzy and <br /> identify genes contribute serum resistance affecting LPS                    Wzz [67]. However, results suggested WzzE also <br /> (Figure S3). The normal LPS pattern EC958 consists 12                            contribute chain length regulation O-antigen EC958. <br /> bands including band representing lipid A-core                      WecA shown previously involved the <br /> and 11-band laddering pattern lipid A-core bound O-antigen                       biosynthesis O7, O18, O75, O111 antigen [68] our <br /> polymers, followed approximately 6 bands long O-                      results indicate true O25b, explaining <br /> antigen chain length (Figure 5C). In addition 6 genes                           role WecA serum resistance EC958. The wecD mutant <br /> involved O-antigen biosynthesis mentioned above, LPS                            highly sensitive serum, possessed LPS profile was <br /> patterns 20 mutants altered comparison wild-type                         altered lipid A-core O-antigen, and <br /> EC958; 6 mutants (waaLKYJ, wecA rfaH)                                displayed enhanced sensitivity SDS (Table 2, Figure S3). <br /> produced lipid A-core (Table 2 Figure S3).                                       Further investigation required understand involvement of <br />    LPS core biosynthesis genes. The genes involved                                  WecD lipid A-core O-antigen biosynthesis. Finally, of <br /> biosynthesis LPS outer core EC958 share strong                               ECA biosynthesis mutants altered sensitivity NaCl. <br /> similarity K-12 MG1655 (waaL waaGPBIJYK)                              Other genes. The remaining 8 mutants altered LPS <br /> [reviewed 65,66], suggesting LPS outer core structure                      patterns confirmed serum sensitive (Table 2). They <br /> in EC958 MG1655. As expected                           included mutant, rfaH, produced lipid A-core, and <br /> functions waaGBIJK MG1655, waaG mutant showed                            7 mutants different O-antigen patterns: hyxA (EC958_0460), <br /> smallest size lipid A-core, indicating outer core                    hyxR (EC958_0461), nagA (N-acetylglucosamine-6-phosphate dea- <br /> was linked lipid A-inner core. Mutations waaIJYK                    cetylase), pgm (phosphoglucomutase), galE (UDP-galactose-4-epim- <br /> produced expected LPS pattern lipid A-core only,                        erase), EC958_1112 wcaF (predicted acetyltransferase in- <br /> mutations prevent linkage O-antigen outer core. The                      volved colanic acid synthesis [69]). Some genes were <br /> LPS waaB waaP mutants produced O-antigen laddering                          associated resistance additional stresses; EC958 rfaH <br /> patterns containing abnormal lipidA-core band, indicating                      wcaF mutants sensitive SDS NaCl, nagA and <br /> although core structures changed, changes                         pgm mutants sensitive SDS (Table 2). <br /> allow linking O-antigen outer core. WaaL is <br /> responsible ligation O-antigen outer core                        Novel O-antigen chain length regulators <br /> mutation gene resulted synthesis lipid A-core                       Mutation hyxA hyxR genes EC958 resulted the <br /> without O-antigen. The EC958 waaL waaGPBIJYK mutants                               modulation O-antigen chain length (Figure 6B). The EC958 <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                               8                          October 2013 | Volume 9 | Issue 10 | e1003834 <br />                                                                                                             Serum Resistome E. coli ST131 Strain <br />  <br />  <br />  <br />  <br /> Figure 5. Characterization O25b antigen genes EC958. (A) Comparison O-antigen cluster EC958 MG1655 (NC_000913) and <br /> E47a (GU014554); genes green involved sugar biosynthesis, genes purple involved O-antigen processing genes orange <br /> encode glycosyl transferase enzymes; genes black outline shown required serum resistance genes red outline were <br /> defined essential. (B) Predicted function glycosyl transferases biosynthesis O25b repeat unit [57,58]. (C) LPS gels showing <br /> changes O-antigen structures corresponding mutations gene. <br /> doi:10.1371/journal.pgen.1003834.g005 <br />  <br />  <br />  <br /> hyxA mutant exhibited increased proportion O-antigen chain               Novel serum resistance mechanisms affecting LPS <br /> of 2 6 units maximum number 3&#8211;5 units reduction                    A major advantage genome approaches as <br /> in high chain length polymer. The EC958 hyxR mutant                   TraDIS lies power discovery. Out 46 genes that <br /> an increased proportion O-antigen polymer 2&#8211;4 units. The                 define serum resistome EC958, 21 (46%) genes were <br /> hyxA hyxR genes located pathogenicity island (PAI-X)              confirmed required serum resistance independent of <br /> consisting 4 genes (fimX hyxRAB) previously described             altered LPS patterns (Table 2; Figure S3). The function these <br /> the UPEC strain UTI89 [70]. The hyxB gene (EC958_0459)                     genes ranged 10 COG functional categories included <br /> also named upaB function autotransporter                 &#8216;Carbohydrate transport metabolism&#8217; (4 genes), &#8216;Cell wall/ <br /> [71], prefer maintain nomenclature. This island              membrane/envelope biogenesis&#8217; (3), &#8216;Posttranslational modifica- <br /> present (in variations) 24 50 E. coli genomes         tion, protein turnover, chaperones&#8217; (3) (Table 2). <br /> all pathotypes (Figure 6A) current sequence data suggest              Twelve genes associated enhanced <br /> it exclusive E. coli.                                                    sensitivity SDS enhanced NaCl sensitivity <br />    No functional prediction hyxA,               (Table 2) <br /> the report hyxA involvement serum resistance                       Of non-LPS genes required serum resistance, most <br /> regulating O-antigen chain length. The hyxR gene encodes                     notable lpp (logFC 210, log difference = 6) (Table 2). The lpp <br /> LuxR-like response regulator suppresses nitrosative stress            gene encodes abundant proteins E. coli is <br /> response contributes intracellular survival macrophages              responsible stabilisation integrity bacterial cell <br /> by regulating hmpA, encodes nitric oxide-detoxifying                   envelope [72]. Mutation lpp gene results formation of <br /> flavohaemoglobin [70]. The expression hyxR gene                      outer membrane blebs, leakage periplasmic enzyme ribonucle- <br /> regulated bidirectional phase inversion promoter                ase, decreased growth rate media low ionic strength low <br /> region upstream gene fimX, encodes tyrosine-like                osmolarity hypersensitive toxic compounds [73,74]. Indeed, <br /> recombinase [70]. It worth noting contribution             EC958 lpp mutant sensitive SDS, suggesting <br /> hyxA serum resistance greater hyxR, demonstrated                decreased membrane integrity (Table 2). To best our <br /> by 3-log reduction viability hyxA mutant compared                knowledge, study direct link Lpp <br /> to hyxR mutant. A serum sensitive phenotype hyxR                   serum resistance. <br /> mutant observed mixed competition assays,                    Another set genes notable TraDIS analysis include <br /> mutants did exhibit altered sensitivity SDS NaCl                     tolQAB, encode proteins (YbgC-YbgF-TolQ- <br /> (Table 2).                                                                     R-A-B-Pal) make Tol-Pal E. coli cell <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                       9                       October 2013 | Volume 9 | Issue 10 | e1003834 <br />                                                                                                                        Serum Resistome E. coli ST131 Strain <br />  <br />  <br />  <br />  Table 3. Summary E. coli EC958 O25b antigen biosynthesis genes. <br />  <br />  <br />                                                                                                   % identities/% <br />                         No.                                                                    positives/no. of <br />                         amino                                                                     amino acid <br />  Locus tag      Gene    acids     Similar protein                                                 overlap             predicted function <br />  <br />  EC958_2381     rmlB    361       dTDP-glucose 4,6-dehydratase [<span id='am-251' about='obo:NCBITaxon_1033813' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-252' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-253' property="oboInOwl:hasDbXref" content="PMID:21685291" datatype="xsd:string"></span><span id='am-254' property="oboInOwl:hasExactSynonym" content="Escherichia coli strain NA114" datatype="xsd:string"></span><span id='am-255' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-256' property="rdfs:label" content="Escherichia coli NA114" datatype="xsd:string"></span><span id='am-257' property="oboInOwl:hasExactSynonym" content="Escherichia coli str. NA114" datatype="xsd:string"></span>Escherichia coli NA114</span>]           99/100/361          dTDP-glucose 4,6-dehydratase <br />                                   (YP_006139244) <br />  EC958_2380     rmlD    299       dTDP-4-dehydrorhamnose reductase [Escherichia                   100/100/299         dTDP-4-dehydrorhamnose reductase <br />                                   coli NA114] (YP_006139241) <br />  EC958_2379     rmlA    292       glucose-1-phosphate thymidylyltransferase                       100/100/292         glucose-1-phosphate thymidylyltransferase <br />                                   [Escherichia coli NA114] (YP_006139242) <br />  EC958_2378     rmlC    180       dTDP-4-dehydrorhamnose 3,5-epimerase                            99/99/180           dTDP-4-dehydrorhamnose 3,5-epimerase <br />                                   [Escherichia coli NA114] (YP_006139241) <br />  EC958_2377     wzx     419       O-antigen flippase Wzx [Escherichia coli E47a]                  95/98/149           O-antigen flippase <br />                                   (ADI43260) <br />  EC958_2376     wekA    321       putative glycosyl transferase <span id='am-22' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-23' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> 2                          93/98/321           glycosyl transferase family 2; <br />                                   [Escherichia coli ABU83972] (ADN46883)                                              a-L-FucpNAc-(1-.3)-b-D-GlcpNAc <br />  EC958_2375     wekB    382       putative glycosyl transferase [Escherichia coli E47a]           93/97/382           glycosyl transferase family 4; <br />                                   (ADI43262)                                                                          b-D-Glcp-(1-.6)- a-D-Glcp <br />                                   glycosyl transferase WcmS [Escherichia                          39/59/372 <br />                                   coli E1020-72, O158] (ADN43874) <br />  EC958_2374     wzy     405       O-antigen polymerase Wzy [Escherichia coli E47a]                92/95/346           O-antigen polymerase <br />                                   (ADI43263) <br />  EC958_2373     wbbJ    201       predicted lipopolysaccharide biosynthesis                       68/80/194           predicted lipopolysaccharide <br />                                   O-acetyl transferase [Escherichia coli str. K-12                                    biosynthesis O-acetyl transferase <br />                                   substr. MG1655] (NP_416537) <br />  EC958_2372     wbuH    366       Putative glycosyltransferase WbuH                               29/49/372           glycosyl transferase family 4; <br />                                   [Escherichia coli O4:K3:H5] (AAT85654)                                              a-D-Glcp-(1-.3)- a-L-FucpNAc <br />  EC958_2371     wbbL    263       Rhamnosyltransferase [Escherichia coli str. K-12                60/76/260           glycosyl transferase family 2; <br />                                   substr. MG1655] (P36667)                                                            Rhamnosyltransferase:a-L-Rhap(1-.3)a-D-Glcp <br />  EC958_2370     gnd     468       6-phosphogluconate dehydrogenase                                100/100/468         6-phosphogluconate dehydrogenase <br />                                   [Escherichia coli NA114] (YP_006137974.1) <br />  EC958_2369     ugd     388       UDP-glucose-6-dehydrogenase [Escherichia coli NA114]            100/100/388         UDP-glucose-6-dehydrogenase <br />                                   (YP_006137975) <br />  EC958_2368     wzz     325       regulator length O-antigen component                   100/100/325         O-antigen chain length regulator <br />                                   lipopolysaccharide chain [Escherichia coli NA114] <br />                                   (YP_006137976) <br />  <br />  doi:10.1371/journal.pgen.1003834.t003 <br />  <br />  <br /> envelope. The Tol-Pal responsible maintaining                           mutagenesis (Table 2). This mechanism known confer <br /> integrity outer membrane. TolQRA form inner                                     resistance polymixin B preventing binding lipid A <br /> membrane complex TolQR necessary stability                            [Reviewed 78,79,80]. ArnD catalyzes deformylation step to <br /> [75]. TolB, periplasmic protein, connects inner membrane                            generate UDP-L-Ara4N transported inner <br /> complex peptidoglycan-associated lipoprotein, Pal,                         membrane ArnEF [79,81]. The requirement ArnDEF for <br /> is anchored outer membrane [76]. In study, mutation                         serum resistance indicates EC958 requires L-Ara4N modifi- <br /> the tolA tolQ genes caused sensitivity human serum                             cation evade antimicrobial activity cationic peptides <br /> increased sensitivity SDS, tolB mutant did                               present human serum. Interestingly, arnD mutation <br /> (Table 2). Our results demonstrate Tol-Pal                             conferred sensitivity SDS (Table 2), suggest role <br /> important resistance human serum,                              UDP-L-Ara4N maintaining membrane integrity. Further <br /> novel function important cell wall complex.                                      investigation needed understand ArnT, final enzyme <br />    BamB lipoprotein BamABCD complex.                             required transferring L-Ara4N residue 49-phosphate <br /> Mutation bamB results increased outer membrane perme-                               group lipid A-core, identified TraDIS-defined <br /> ability, enhancing sensitivity rifampin dramatically                          serum resistome. <br /> reducing growth SDS novobiocin [77]. Our data showed                                  We identified genes encoding catabolic enzymes that <br /> that mutation bamB gene EC958 resulted increased                             contributed serum resistance phenotype EC958 (gmm, pgi <br /> sensitivity human serum SDS (Table 2).                                        fbp) confirmed role mutagenesis (Table 2). Of these <br />    Our TraDIS experiment indicated modification                          genes, pgi mutant displayed enhanced sensitivity SDS <br /> lipid A L-Ara4N important serum resistance. Three                            (Table 2). Gmm GDP-mannose mannosyl hydrolase capable of <br /> (arnDEF) seven genes involved biosynthesis                              hydrolyzing GDP-mannose GDP-glucose [82]. This <br /> attachment L-Ara4N lipid A-core identified                          enzyme contributes biosynthesis GDP-fucose, compo- <br /> the serum resistome EC958 role confirmed                              nent colanic acid, possibly influencing concentration of <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                                 10                         October 2013 | Volume 9 | Issue 10 | e1003834 <br />                                                                                                       Serum Resistome E. coli ST131 Strain <br />  <br />  <br /> GDP-mannose GDP-glucose cell regulating cell            mutant library. In fact, OmpA inhibits serum-mediated killing <br /> wall biosynthesis [82]. Both Pgi Fbp catalyze production        binding C4b-binding protein (C4BP) prevent the <br /> D-fructose-6-phosphate b-D-glucose-6-phosphate fruc-              activation C3b classical complement pathway [85], <br /> tose-1,6-bisphosphate, respectively [83,84]. D-fructose-6-phosphate        OmpA known released E. coli cells treated <br /> is precursor biosynthesis UDP-GlcNAc, turn        serum [86]. In mutant library, ompA mutants only <br /> required peptidoglycan, lipid A ECA biosynthesis. Thus,            accounted approximately 0.02% total bacterial cells, and <br /> these enzymes catalyse key reactions that, disrupted,         hypothesized release OmpA 99.98% of <br /> could adversely affect downstream biosynthesis cell surface         cells, treated serum, provided OmpA trans to <br /> components including colanic acid, peptidoglycan, lipid A              complement ompA mutants. We tested hypothesis mixing <br /> ECA.                                                                       ompA mutant wild-type EC958 various ratios and <br />                                                                            showed ompA mutants protected serum <br /> An ompA mutant protected serum killing                        killing proportion ompA mutants 15% <br /> present low proportion mixed bacterial                           (Figure 7). This result strongly suggests trans complemen- <br /> population                                                                 tation OmpA prevents identification ompA serum <br />    Of chromosomal genes previously attributed serum             resistance gene assay. <br /> resistance, gene identified TraDIS <br /> screen ompA. To examine constructed                 Complementation selected mutants restores serum <br /> EC958 ompA mutant observed sensitive killing          resistance <br /> by human serum (Figure 7). One way explain discrepancy             To demonstrate function non-LPS genes serum <br /> that phenotype ompA mutant complemented              resistance, selected orphan genes (acnB, greA fbp) that <br /> trans ompA-intact bacteria mixed population          belong operon perform genetic complementation. <br />  <br />  <br />  <br />  <br /> Figure 6. Genetic context hyxA hyxR genes role O-antigen synthesis. (A) The conserved location PAI-X containing <br /> hyxAR genes E. coli genomes various pathotypes. (B) The LPS pattern hyxA hyxR mutants complemented strains, <br /> demonstrating role hyxA hyxR genes O-antigen chain length regulation. Red boxes highlight differences LPS patterns compared <br /> with wild-type. <br /> doi:10.1371/journal.pgen.1003834.g006 <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                  11                      October 2013 | Volume 9 | Issue 10 | e1003834 <br />                                                                                                             Serum Resistome E. coli ST131 Strain <br />  <br />  <br /> For experiments, selected gene amplified PCR,                 genes that, mutated transposon insertion, either <br /> cloned low copy number plasmid pSU2718G,                          prevent severely attenuate growth LB agar media <br /> transformed respective mutant strain complementa-                 supplemented 30 mg/ml Cm 37uC. The cut-off value to <br /> tion. In case, phenotype complemented strain                   determine gene essential defined intercept <br /> exactly matched wild-type EC958 (Table 4). Taken                       distributions insertion index gene: the <br /> together, results confirm role acnB, greA fbp              exponential distribution representing essential genes the <br /> serum resistance provide layer evidence support            gamma distribution representing non-essential genes [42]. This <br /> the use techniques TraDIS functional gene discovery.             means essential genes include genes can <br />                                                                                tolerate insertions severely attenuated input pool. <br /> Discussion                                                                     Out 315 essential genes, 64 genes transposon insertions, <br />                                                                                178 genes 1 5 transposon insertions 73 genes more <br />    The rapid advancement new sequencing technologies                    5 transposon insertions (Table S1). <br /> created novel opportunities interrogate biological systems                The high density insertion sites achieved study <br /> were previously possible. TraDIS described                  provided reliable data identification essential genes <br /> method combined high-density mutagenesis Illumina                    EC958 genomes minimal probability false <br /> next generation sequencing technology study essential genes             positive calls transposon insertion cold spots. The <br /> of S. Typhi conditional essential genes required                   identification essential genes previously performed <br /> survival bile [42]. The increased data output afforded              using approaches E. coli K-12 (strains MG1655 and <br /> generation DNA sequencing particularly useful cost-                     W3110) [96,97]. Baba et al. generated null mutations lambda- <br /> effective small bacterial genomes, presents technical <br />                                                                                red recombination 3985 E. coli W3110 genes (the Keio library), <br /> and bioinformatical challenges applications TraDIS <br />                                                                                unable mutate 303 candidate essential genes [97]. This <br /> that utilize low complexity DNA libraries. Here present the <br />                                                                                set essential genes consolidated manual literature <br /> application modified version TraDIS amenable to <br />                                                                                review EcoGene website (www.ecogene.org), which <br /> multiplexing using Illumina HiSeq 2000 platform, we <br />                                                                                reduced set 289 genes. Of 315 essential genes identified <br /> demonstrate effectiveness using define essential gene <br />                                                                                EC958 study, 231 genes (73%) matched previously <br /> repertoire serum resistome multidrug resistant strain <br />                                                                                described EcoGene list (Table S1). There 84 essential <br /> from globally disseminated E. coli ST131 lineage. <br />                                                                                genes specific EC958, twenty-four have <br />    The multiplexed TraDIS protocol utilizes newly designed <br />                                                                                homologs MG1655 genome. In contrast, 58 genes <br /> custom oligonucleotide library enrichment step the <br />                                                                                previously identified essential E. coli K-12 not <br /> Illumina library preparation protocol (Figure S1). This oligonu- <br />                                                                                present EC958 identified analysis. The majority of <br /> cleotide incorporates Illumina sequencing primer-binding site <br />                                                                                essential genes EC958 conserved 91% genes <br /> into transposon specific DNA fragments, enabling use the <br />                                                                                present 90% E. coli complete genomes available. <br /> standard Illumina sequencing primer eliminating need to <br />                                                                                   In study, provided layers evidence role of <br /> design optimize sequencing primer new <br />                                                                                serum resistance gene: simultaneously assaying large <br /> transposon sequence. The 6-bp barcode immediately the <br />                                                                                mutant library generation defined mutants for <br /> sequencing primer-binding site allows 12-sample multiplexing <br />                                                                                independent phenotypic testing. Indeed, using defined mutagen- <br /> within lane. The use 12 barcodes 6 nucleotides of <br />                                                                                esis able confirm role 46 56 genes identified <br /> read 1 increased complexity library compared the <br /> original TraDIS protocol, reducing data loss mis- <br /> identification clusters [87,88]. However, number useable <br /> reads sequencing runs low (15&#8211;20% total <br /> reads). We believe non-specific amplification enrichment <br /> step main cause, optimization the <br /> enrichment PCR conditions required. Similar approaches <br /> combining transposon mutagenesis high-throughput sequenc- <br /> ing (Tn-seq [45], INSeq [43], HITS [44]) used to <br /> address different scientific questions, including identification of <br /> essential genes genes associated enhanced fitness in <br /> specific growth conditions [41,42,45], determination niche- <br /> specific essential genes [43,44,89], identification genes associ- <br /> ated tolerance various agents/conditions [42,90] many <br /> other applications reviewed [91,92]. In terms of <br /> insertion density, achieved 502,068 independent insertion sites <br /> with density 1/10 bp (i.e. insertion 10 bp), is <br /> comparable work Christen et al. (1/8 bp C. crescentus) <br /> [41], Langridge et al. (1/13 bp Salmonella Typhi) [42]                  Figure 7. In trans complementation wild-type EC958 protects <br /> Barquist et al. (1/9 bp Salmonella Typhimurium) [46].                       ompA mutants serum killing. Wild-type EC958 EC958 <br />    The identification essential gene set single organism          ompA mutant mixed different ratios, exposed human serum <br /> is challenging factors, including presence               90 minutes, EC958 ompA mutant examined its <br /> transposon insertion cold spots (i.e. regions low transposon                sensitivity human serum (expressed 2log10 CFU difference <br />                                                                                time 90 time 0). The red line indicates threshold below <br /> insertion frequency), difficulty distinguishing mutations          strain considered resistant serum killing. The EC958 <br /> prevent growth severely reduce growth rate, pre-               ompA mutant protected killing present percentage <br /> existing gene duplications specific growth conditions used             15% mixed bacterial suspension. <br /> in experiment [93&#8211;95]. In study, define essential genes            doi:10.1371/journal.pgen.1003834.g007 <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                      12                       October 2013 | Volume 9 | Issue 10 | e1003834 <br />                                                                                                                   Serum Resistome E. coli ST131 Strain <br />  <br />  <br />  <br />  Table 4. Complementation selected mutants.                                        available sequences GenBank database deducing <br />                                                                                       K type silico possible. <br />                                                                                          The O25 antigen gene cluster characterized using <br />                            Phenotype                                                  sequence analysis, targeted mutation LPS profiling. All the <br />  Strains <br />                                                                                       dTDP-a-L-rhamnose biosynthesis genes (rmlCADB) required <br />                            Seruma           SDS (%)b         NaCl (M)b <br />                                                                                       serum resistance. However, EC958 lacks biosynthesis genes <br />  EC958 (wild-type)         0                0.125            0.8                      UDP-FucNAc, component O25 antigen unit. If, based on <br />  EC958DacnB                1                0.125            0.8                      cross reaction antiserum O25 antigen O25b <br />                                                                                       expressing cells, assume EC958 O-antigen <br />  EC958DacnB + pAcnB        0                0.125            0.8 <br />                                                                                       repeat unit O25 determined previous studies [57,58], <br />  EC958DgreA                2                0.063            0.8 <br />                                                                                       EC958 possess novel mechanism synthesis or <br />  EC958DgreA + pGreA        0                0.125            0.8                      uptake UDP-FucNAc. <br />  EC958Dfbp                 3                0.125            0.8                         Two additional surface antigens contribute serum <br />  EC958Dfbp + pFbp          0                0.125            0.8                      resistance EC958 enterobacterial common antigen and <br />                                                                                       colanic acid (M antigen). Mutations ECA biosynthesis genes <br />  a <br />   Serum sensitivity shown 2log10 viable count difference            rendered EC958 susceptible serum. While mutation of <br />  t = 90 min t = 0 min. The higher number sensitive mutant <br />  is. <br />                                                                                       genes (wecA, wzzE wecD) affected LPS sensitivity to <br />  b <br />    Minimal inhibitory concentration (MIC) SDS (%) NaCl (M).                    SDS, mutation wecE wecF did change LPS (although a <br />  doi:10.1371/journal.pgen.1003834.t004                                                wecF mutant sensitive SDS), suggesting ECA <br />                                                                                       involved serum resistance, indirectly role <br /> by TraDIS serum resistance. To best knowledge,                     membrane integrity. Our data suggest involvement of <br /> represents large scale follow-up TraDIS data                     colanic acid serum resistance. Three genes encoding colanic <br /> manner highlights effectiveness technique large-                    acid biosynthesis (wcaI, gmm wcaF) identified TraDIS. <br /> scale functional genomics. Our study revealed trans                         Gmm likely involved biosynthesis colanic <br /> complementation specific mutants occur large mutant                       acid [82], mutation wcaI gene did confer serum <br /> library population, demonstrated findings ompA                      resistance. The product wcaF predicted an <br /> mutant. We demonstrated complete complementation                              acetyltransferase [103] required colanic acid production [69]. <br /> mutants containing deletions acnB, greA fbp genes,                         The EC958 wcaF mutant possessed altered LPS pattern a <br /> corroborating novel role serum resistance independent                     reduced O-antigen (especially long chain length <br /> LPS alterations. Finally, provided insight                        O-antigen) sensitive SDS high osmolarity. <br /> mechanistic action serum resistance genes identified                        The enhanced sensitivity EC958 wcaF mutant could <br /> EC958 examining survival respective mutants                          explained number factors, including altered <br /> outer membrane stresses affect antimicrobial access                          LPS, altered colonic acid reduced overall membrane integrity. <br /> osmotic potential.                                                                       A number genes identified contributed to <br />    The search genetic determinants serum resistance                         serum resistance LPS-dependent manner. The gene nagA <br /> bacteria ongoing 1970s [98]. Our current                           encodes N-acetylglucosamine-6-phosphate deacetylase, enzyme <br /> understanding mechanisms promote bacterial resis-                         important metabolism N-acetyl-D-glucosamine [104]. It <br /> tance human serum include role surface structures                    catalyzes step producing UDP-GlcNAc, nucleotide <br /> O antigens, K antigens, outer membrane proteins (OmpA),                           sugar required ECA, lipid A peptidoglycan biosynthesis, <br /> plasmid-encoded proteins (TraT, Iss) [37&#8211;39]; notably, however,                       deacetylating N-acetylglucosamine-6-phosphate glucos- <br /> not mechanisms required resistance single                   amine-6-phosphate [Reviewed 105]. However, NagA not <br /> bacterial strain [32,33]. Our study represents report                    solely responsible production UDP-GlcNAc because <br /> simultaneously investigate entire serum resistome strain.                  glucosamine-6-phosphate obtained GlmS from <br /> Our results demonstrated lipid A-core O25                           fructose-6-phosphate taken environment by <br /> antigen crucial serum resistance EC958, K antigen                    ManXYZ [105]. Indeed, LPS banding pattern nagA <br /> does contribute serum resistance. Out 54 defined mutants                    mutant different parent strain (i.e. possessed <br /> investigated, half changes LPS gel patterns,                      thicker second bands the gel; Figure <br /> which resulted serum sensitivity. In contrast, K                       S3), suggesting enhanced sensitivity phenotype be <br /> capsular biosynthesis genes identified TraDIS screen.                     associated predominantly shorter O antigen. <br /> This result similar reported O75:K5 UPEC strain                       Pgm phosphoglucomutase catalyses reversible <br /> GR-12, alterations O75 LPS affected serum resistance                         conversion glucose-1-phosphate glucose-6-phosphate, an <br /> more K5 null mutation [34]. It likely, however,                  important step galactose maltose catabolism [106]. A pgm <br /> are strain-specific differences role O antigen K                       mutant phenotypes; defective swimming and <br /> capsule serum resistance, reflects differences                swarming mobility [107], possesses aberrant (shorter and <br /> make-up structures. For example, previous analysis                     wider) cell morphology, sensitive detergents [108] stains <br /> E. coli O4:K54:H5 blood isolate revealed K54 antigen                         blue iodine grown presence galactose [106]. <br /> contributes serum resistance O4 antigen [99].                        An EC958 pgm mutant produced little length O-antigen; the <br /> Other K antigens K1 K2 capsules                        majority LPS condensed band incomplete lipid <br /> shown play important role serum resistance [100,101].                        A-core clear band lipid A-core plus unit O- <br /> The K antigen expressed EC958 typed                               antigen. This feature consistent high serum SDS <br /> genomic analysis shows EC958 group 2 capsular gene                         sensitivity phenotypes observed mutant. <br /> cluster conforms conserved structure group                           GalE well-studied enzyme catalyzes interconversion <br /> [Reviewed 102]. However, region 2, encodes glycosyl-                         UDP-galactose UDP-glucose [Reviewed 109]. Both <br /> transferases specific K type, shares low similarity                     nucleotide sugars required colanic acid biosynthesis. <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                             13                       October 2013 | Volume 9 | Issue 10 | e1003834 <br />                                                                                                         Serum Resistome E. coli ST131 Strain <br />  <br />  <br /> Furthermore, UDP-glucose used steps synthesize            25 mF 200V. Cells resuspended 1 mL SOC medium <br /> LPS outer core (catalyzes WaaG, WaaI WaaJ). LPS                       incubated 37uC 2 hours, spread LB agar plates <br /> patterns galE mutant exhibited band lipid A-          supplemented Cm 30 mg/mL. After incubation overnight at <br /> core, suggesting lipid A-core strain multiple           37uC, total number colonies estimated counting a <br /> sizes. This explained limiting effect UDP-glucose           proportion multiple plates. Chloramphenicol resistant <br /> in steps involved incorporation outer core.        colonies resuspended sterilised LB broth using a <br /> UDP-glucose synthesized GalU glucose-1-                  bacteriological spreader adding sterile glycerol 15% <br /> phosphate [110], explain EC958 galE mutant                  total volume stored 280uC. Each batch mutants <br /> could make LPS (Figure S3). Despite able make LPS,            contained estimated 32,000 180,000 mutants. The final <br /> however, galE mutant sensitive human serum. Whether               library 1 million mutants created pooling 11 mutant <br /> this sensitivity attributed effect galE mutation         batches, resulting cell suspension 261011 CFU/ml. <br /> on LPS colanic acid remains determined. <br />    We demonstrated successful application multiplexed            Transposon library screening human serum <br /> TraDIS functional genomics study targeted E. coli EC958,               Freshly pooled human serum collected two <br /> a prototype strain globally disseminated multidrug              healthy individuals day experiment. Ten milliliters of <br /> resistant E. coli ST131 lineage. This approach enabled             blood collected person centrifuged at <br /> description essential gene set ExPEC strain. Our               4000 rpm 10 minutes collect serum. Approximately <br /> work defined serum resistome E. coli EC958. This             26108 viable mutants incubated 1 ml 50% freshly <br /> comprehensive inventory E. coli EC958 genes contribute            pooled human serum LB broth 37uC 90 minutes. The <br /> this phenotype provides framework future characteriza-             control samples prepared way incubated <br /> tion virulence genes ExPEC Gram-negative              inactivated serum (Millipore) instead fresh serum. Both <br /> pathogens cause systemic infection.                                     control test experiments performed duplicate. The <br />                                                                              cells washed twice sterile 16PBS remove serum, <br /> Materials Methods                                                        transferred 100 ml LB broth allowed grow 37uC with <br />                                                                              250 rpm shaking 4 hours. The genomic DNA then <br /> Ethics statement                                                             extracted 5 ml culture using Qiagen 100-G genomic <br />    Approval collection human blood obtained              tips. <br /> the University Queensland Medical Research Ethics Committee <br /> (2008001123). All subjects provided written informed consent.                Multiplexed TraDIS <br />                                                                                 Genomic DNA standardized 3.6 mg volume of <br /> Bacterial strains growth conditions                                      120 ml sheared Covaris S2 according the <br />    E. coli EC958 isolated urine patient presenting         Illumina TruSeq Enrichment gel-free method (TruSeq DNA <br /> with community UTI Northwest region England               sample preparation v2 guide). The subsequent steps DNA end <br /> representative member UK epidemic strain A (PFGE type),               repair, DNA end adenylation adapter ligation done <br /> one major pathogenic lineages causing UTI                  following Illumina TruSeq v2 instructions. The adapter-ligated <br /> United Kingdom [23]. EC958Dlac, contained mutation                fragments containing transposon insertion sites enriched <br /> the lac operon, used competitive assays. This strain           using transposon-specific indexing forward primer the <br /> identical growth rate wild-type EC958. Strains routinely             Illumina reverse primer Index 1 (Table S5) 500 nM per <br /> cultured 37uC solid liquid Luria Broth (LB) medium               reaction. This 89 bp forward primer binds specifically miniTn5- <br /> supplemented appropriate antibiotics (Cm 30 mg/ml                Cm transposon (25 bp) carries 64 bp overhang includes <br /> gentamicin 20 mg/ml) unless indicated otherwise.                             6 bp index sequence, 33 bp binding site Illumina read 1 <br />                                                                              sequencing primer 23 bp P5 sequence binding the <br /> Generation miniTn5-Cm mutant library                                      flowcell. The enrichment step using KAPA Library <br />    A miniTn5-Cm transposon containing Cm cassette flanked               Amplification kit (KAPA Biosystems) annealing temperature <br /> Tn5 mosaic ends (sequence Epicenter) PCR amplified                  60uC 22 cycles. The KAPA Library Quantification kit was <br /> from pKD3 plasmid DNA (NotI digested) using primers 2279 59-                 used measure concentration DNA fragments the <br /> CTGTCTCTTATACACATCTcacgtcttgagcgattgtgtagg-39                            enriched library. Twelve libraries 12 samples pooled to <br /> 2280      59-     CTGTCTCTTATACACATCTgacatgggaattagc-                        equimolar concentrations achieve cluster density 850 K/ <br /> catggtcc-39. The PCR reactions performed using Phusion                  mm2 10 nM library pool loaded flowcell. <br /> High-Fidelity DNA polymerase (New England BioLabs). The                      The 12-plex pool loaded 3 lanes TruSeq v2 3 lanes <br /> amplicon purified using QIAGEN MinElute PCR                          TruSeq v3 flowcells sequencing using 100 cycles, paired- <br /> purification kit phosphorylated using T4 polynucle-             end protocol access reproducibility read quantity <br /> otide kinase (New England BioLabs) subjected final                lanes flowcells. The data samples (Figure 1A) <br /> purification step. A total 800 ng miniTn5-Cm             presented study. The TraDIS sequence data this <br /> transposon DNA incubated 8 ml reaction containing 4 ml             study deposited Sequence Read Archive (SRA) under <br /> of EZ-Tn5 transposase (Epicenter Biotechnologies) 37uC 1 h            BioProject number PRJNA189704. <br /> then stored 220uC. <br />    Bacterial cells prepared electroporation previously           Analysis nucleotide sequence data <br /> described [42]. Briefly, cells grown 26TY broth                  Sequence reads FASTQ files split according to <br /> OD600 0.3&#8211;0.5, harvested washed times 0.56              6 bp index sequences combined 37 bp transposon- <br /> volume 10% cold glycerol resuspended 1/                 specific sequence using fastx_barcode_splitter.pl (total length of <br /> 10006 volume 10% cold glycerol kept ice. A volume               43 bp barcodes, allowing 2 mismatches) (FASTX-Toolkit <br /> 60 ml cells mixed 0.2 ml transposomes electropo-             version 0.0.13, http://hannonlab.cshl.edu/fastx_toolkit/index. <br /> rated 2 mm cuvette using BioRad GenePulser set 2.5 kV,             html). The barcode matching reads trimmed 43 bp <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                    14                      October 2013 | Volume 9 | Issue 10 | e1003834 <br />                                                                                                            Serum Resistome E. coli ST131 Strain <br />  <br />  <br /> barcode 59end 25 bp potential low quality 39               wild-type (EC958Dlac) mutant strains used. Viable <br /> end, resulting high quality sequence reads 31 bp length                counts performed MacConkey agar, allowed the <br /> that used map EC958 chromosome (PRJEA61443)                      differentiation EC958Dlac (non-lactose fermenter) the <br /> by Maq version 0.7.1 [111]. Subsequent analysis steps                      mutant strains. <br /> carried previously described [42] calculate number of <br /> sequence reads (raw read counts) number different                    SDS NaCl sensitivity assays (MIC) <br /> insertion sites gene, used estimate              The MICs SDS NaCl determined broth <br /> threshold identify essential genes. The read counts insertion            microdilution method previously described [120]. We used five <br /> sites visualized using Artemis version 13.0 [112]. The circular            concentrations SDS including 0.125%, 0.0625%, 0.031%, <br /> genome diagram generated CGView [113] linear                         0.016% 0.008% LB. For NaCl, range concentration <br /> genetic comparison illustrated using Easyfig version 2.1 [114].             0.8 M, 0.6 M, 0.5 M, 0.4 M 0.3 M. <br />  <br /> Statistical analyses                                                            LPS gel assay <br />    We identified genes required survival human serum                    LPS extracted bacterial strains LPS patterns were <br /> comparing differences read abundance gene                        determined Tricine-SDS Polyacrylamide gel electrophoresis <br /> between inactivated serum control active serum test                     (TSDS-PAGE) visualized silver staining previously <br /> samples using Bioconductor package edgeR (version 2.6.10)                   described [121,122]. <br /> [48]. The raw read counts biological replicates each <br /> treatment loaded edgeR package (version 2.6.12) <br />                                                                                 Supporting Information <br /> using R environment (version 2.15.1). Genes very <br /> low read counts samples (essential genes) removed               Figure S1 Diagram demonstrating use custom primer <br /> from analysis. The composition <span id='am-11' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-12'  ></span><span id='am-13' property="rdfs:label" content="Bias" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>bias</span> sequence                    functional regions PCR enrichment transposon <br /> library normalized using trimmed mean M value                        derived fragments. <br /> (TMM) method [115]. We used quantile-adjusted                          (PDF) <br /> conditional maximum likelihood (qCML) negative binomial <br /> models estimate dispersions (biological variation                Figure S2 Results edgeR analysis showing logFC genes <br /> replicates) carry exact tests determining genes              highlighting genes satisfy stringent cut-off serum <br /> with significantly lower read counts test samples compared               resistome. <br /> to control samples [116,117]. Stringent criteria log fold-               (PDF) <br /> change (logFC) #21 false discovery rate #0.001 chosen                  Figure S3 LPS patterns defined mutants generated this <br /> to define list significant genes investigation        study. The LPS pattern wild-type strain EC958 (WT) used <br /> by phenotypic assays.                                                           reference identify alteration LPS mutants. <br />                                                                                 (PDF) <br /> Molecular methods <br />                                                                                 Table S1 Summary transposon insertions CDS within <br />    Chromosomal DNA purification, PCR DNA sequencing of <br />                                                                                 EC958 chromosome. <br /> PCR products performed previously described [118]. <br />                                                                                 (XLSX) <br /> Defined mutations using l-Red recombinase <br /> method modifications [22,47]. In brief, final PCR                 Table S2 Sequence comparison EC958 essential genes and <br /> products fused amplified fragments contain-                 genes 50 E. coli complete genomes. <br /> ing 500-bp homologous regions flanking gene                 (XLSX) <br /> and Cm cassette pKD3 plasmid (see Table S5 list of <br />                                                                                 Table S3 Essential genes unique EC958. <br /> primers). The fused PCR products electroporated into <br /> EC958 harbouring gentamicin resistant plasmid carrying l-                 (XLSX) <br /> Red recombinase gene. Mutants selected confirmed                  Table S4 Results edgeR analysis comparing read counts <br /> by sequencing. Complementation cloning gene                  control test samples identify serum resistome. <br /> interest gentamicin resistant derivative pSU2718 [119]             (XLSX) <br /> BamHI-XbaI cut sites (primers listed Table S5). The construct <br /> was transformed respective mutant induced                     Table S5 Primers used study. <br /> using 1 mM IPTG phenotypic assays.                            (XLSX) <br />  <br /> Serum resistance assay                                                          Acknowledgments <br />    Overnight bacterial cultures washed phosphate buffered               We like thank <span id='am-241' about='obo:NCBITaxon_345460' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6580 obo:NCBITaxon_6592 obo:NCBITaxon_6544 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_6599 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_74489 obo:NCBITaxon_6072'><span id='am-242'  ></span><span id='am-243' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-244' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-245' property="rdfs:label" content="Gemma" datatype="xsd:string"></span>Gemma</span> Langridge, Keith Turner, Sabine Eckert, <br /> saline (PBS) standardized OD600 0.8. Equal                    Daniel Turner, Philip Hugenholtz, Nicholas West Brian Forde for <br /> volumes (50 mL) standardized cultures pooled human sera                  expert advice Leopold Parts statistical analysis R. <br /> were mixed incubated 90 min 37uC (in triplicates). <br /> Viable counts performed estimate number bacterial                Author Contributions <br /> cells prior serum treatment (t = 0 min) post serum <br />                                                                                 Conceived designed experiments: MDP SS MT SAB MAS. <br /> treatment (t = 90 min). E. coli MG1655 used control                 Performed experiments: MDP KMP SS SWL LPA MESA VMM. <br /> it completely killed serum. Serum PBS samples                    Analyzed data: MDP DGM SAB MAS. Contributed reagents/ <br /> served sterility controls. Competitive serum resistance assays               materials/analysis tools: VMM MU SAB. Wrote paper: MDP SS <br /> were performed manner, 50:50 mixture                  DGM MT MU SAB MAS. <br />  <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                       15                      October 2013 | Volume 9 | Issue 10 | e1003834 <br />                                                                                                                                Serum Resistome E. coli ST131 Strain <br />  <br />  <br />  <br />  <br /> References <br />  1. Nicolas-Chanoine MH, Blanco J, Leflon-Guibout V, Demarty R, Alonso MP,                        Caused Multidrug-Resistant Uropathogenic <span id='am-226' about='obo:NCBITaxon_1359206' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-227' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-228' property="oboInOwl:hasExactSynonym" content="Escherichia coli Sequence Type 131" datatype="xsd:string"></span><span id='am-229' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-230' property="oboInOwl:hasDbXref" content="PMID:18199778" datatype="xsd:string"></span><span id='am-231' property="oboInOwl:hasExactSynonym" content="Escherichia coli Multi Locus Sequence Type 131" datatype="xsd:string"></span><span id='am-232' property="oboInOwl:hasExactSynonym" content="Escherichia coli MLST 131" datatype="xsd:string"></span><span id='am-233' property="rdfs:label" content="Escherichia coli ST131" datatype="xsd:string"></span>Escherichia coli ST131</span>. J Infect <br />     et al. (2008) Intercontinental emergence Escherichia coli clone O25:H4-                    Dis 208(6):921&#8211;8. <br />     ST131 producing CTX-M-15. J Antimicrob Chemother 61: 273&#8211;281.                           25.   Floyd RV, Upton M, Hultgren SJ, Wray S, Burdyga TV, et al. (2012) <br />  2. Pitout JD, Gregson DB, Campbell L, Laupland KB (2009) Molecular                               Escherichia coli-mediated impairment ureteric contractility uropathogenic <br />     characteristics extended-spectrum-<span id='am-16' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>beta</span>-lactamase-producing Escherichia                     E. coli specific. J Infect Dis 206: 1589&#8211;1596. <br />     coli isolates causing bacteremia Calgary Health Region 2000              26.   McCabe WR, Kaijser B, Olling S, Uwaydah M, Hanson LA (1978) Escherichia <br />     2007: emergence clone ST131 cause community-acquired infections.                   coli bacteremia: K O antigens serum sensitivity strains from <br />     Antimicrob Agents Chemother 53: 2846&#8211;2851.                                                    adults neonates. J Infect Dis 138: 33&#8211;41. <br />  3. Johnson JR, Johnston B, Clabots C, Kuskowski MA, Castanheira M (2010)                   27.   Mart&#305;&#180;nez JL, Cercenado E, Pe&#180;rez-D&#305;&#180;az JC, Baquero F (1986) Multifactorial <br />     Escherichia coli sequence type ST131 major cause multidrug-                 determination systemic invasivity Escherichia coli. FEMS Microbiology <br />     resistant E. coli infections United States. Clin Infect Dis 51: 286&#8211;294.               Letters 37: 259&#8211;261. <br />  4. Peirano G, Pitout JD (2010) Molecular epidemiology Escherichia coli                  28.   Jacobson SH, O   &#168; Stenson C-G, Tullus K, Brauner A (1992) Serum resistance in <br />     producing CTX-M beta-lactamases: worldwide emergence clone ST131                       Escherichia coli strains causing acute pyelonephritis bacteraemia. APMIS <br />     O25:H4. Int J Antimicrob Agents 35: 316&#8211;321.                                                  100: 147&#8211;153. <br />  5. Sidjabat HE, Derrington P, Nimmo GR, Paterson DL (2010) Escherichia coli                29.   Opal S, Cross A, Gemski P (1982) K antigen serum sensitivity rough <br />     ST131 producing CTX-M-15 Australia. J Antimicrob Chemother 65: 1301&#8211;                       Escherichia coli. Infect Immun 37: 956&#8211;960. <br />     1303.                                                                                   30.   Feingold DS (1969) The serum bactericidal reaction. IV. Phenotypic <br />  6. Rogers BA, Sidjabat HE, Paterson DL (2011) Escherichia coli O25b-ST131:                     conversion Escherichia coli serum-resistance serum-sensitivity by <br />     pandemic, multiresistant, community-associated strain. J Antimicrob Che-                      diphenylamine. J Infect Dis 120: 437&#8211;444. <br />     mother 66: 1&#8211;14.                                                                        31.   Gemski P, Cross AS, Sadoff JC (1980) K1 antigen-associated resistance the <br />  7. Ender PT, Gajanana D, Johnston B, Clabots C, Tamarkin FJ, et al. (2009)                       bactericidal activity serum. FEMS Microbiology Letters 9: 193&#8211;197. <br />     Transmission extended-spectrum-beta-lactamase-producing Escherichia               32.   Stawski G, Nielsen L, &#216;Rskov F, &#216;Rskov <span id='am-174' about='obo:NCBITaxon_426865' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158332 obo:NCBITaxon_158416 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_158394 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-175' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-176'  ></span><span id='am-177' property="rdfs:label" content="Ida" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>IDA</span> (1990) Serum sensitivity a <br />     coli (sequence type ST131) strain father daughter resulting                  diversity Escherichia coli antigenic reference strains. APMIS 98: 828&#8211;838. <br />     septic shock Emphysematous pyelonephritis. J Clin Microbiol 47: 3780&#8211;               33.   Cross AS, Kim KS, Wright DC, Sadoff JC, Gemski P (1986) Role of <br />     3782.                                                                                         lipopolysaccharide capsule serum resistance bacteremic strains of <br />  8. Johnson JR, Anderson JT, Clabots C, Johnston B, Cooperstock M (2010)                          Escherichia coli. J Infect Dis 154: 497&#8211;503. <br />     Within-household sharing fluoroquinolone-resistant Escherichia coli                34.   Burns SM, Hull SI (1998) Comparison loss serum resistance defined <br />     sequence type ST131 strain causing pediatric osteoarticular infection. Pediatr                lipopolysaccharide mutants acapsular mutant uropathogenic <br />     Infect Dis J 29: 473&#8211;475.                                                                     Escherichia coli O75:K5. Infect Immun 66: 4244&#8211;4253. <br />  9. Coque TM, Novais A, Carattoli A, Poirel L, Pitout J, et al. (2008)                      35.   Porat R, Mosseri R, Kaplan E, Johns MA, Shibolet S (1992) Distribution of <br />     Dissemination clonally related Escherichia coli strains expressing extend-                 polysaccharide chains lipopolysaccharide determine resistance of <br />     ed-spectrum beta-lactamase CTX-M-15. Emerg Infect Dis 14: 195&#8211;200.                            Escherichia coli bactericidal activity serum. J Infect Dis 165: 953&#8211;956. <br /> 10. Johnson JR, Menard M, Johnston B, Kuskowski MA, Nichol K, et al. (2009)                 36.   Weiser JN, Gotschlich EC (1991) Outer membrane protein A (OmpA) <br />     Epidemic clonal groups Escherichia coli cause antimicrobial-resistant              contributes serum resistance pathogenicity Escherichia coli K-1. <br />     urinary tract infections Canada, 2002 2004. Antimicrob Agents                           Infect Immun 59: 2252&#8211;2258. <br />     Chemother 53: 2733&#8211;2739.                                                                37.   Moll A, Manning PA, Timmis KN (1980) Plasmid-determined resistance to <br /> 11. Johnson JR, Tchesnokova V, Johnston B, Clabots C, Roberts PL, et al. (2013)                   serum bactericidal activity: major outer membrane protein, traT gene <br />     Abrupt Emergence Single Dominant Multidrug-Resistant Strain                           product, responsible plasmid-specified serum resistance Escherichia <br />     Escherichia coli. J Infect Dis 207: 919&#8211;928.                                                  coli. Infect Immun 28: 359&#8211;367. <br /> 12. Peirano G, Schreckenberger PC, Pitout JD (2011) Characteristics NDM-1-               38.   Montenegro MA, Bitter-Suermann D, Timmis JK, Aguero ME, Cabello FC, et <br />     producing Escherichia coli isolates belong successful virulent                al. (1985) traT gene sequences, serum resistance pathogenicity-related <br />     clone ST131. Antimicrob Agents Chemother 55: 2986&#8211;2988.                                       factors clinical isolates Escherichia coli gram-negative bacteria. <br /> 13. Morris D, Boyle F, Ludden C, Condon I, Hale J, et al. (2011) Production                    J Gen Microbiol 131: 1511&#8211;1521. <br />     KPC-2 carbapenemase Escherichia coli clinical isolate belonging            39.   Smith HW (1974) A search transmissible pathogenic characters invasive <br />     international ST131 clone. Antimicrob Agents Chemother 55: 4935&#8211;4936.                         strains Escherichia coli: discovery plasmid-controlled toxin a <br /> 14. Morris D, McGarry E, Cotter M, Passet V, Lynch M, et al. (2012) Detection                  plasmid-controlled lethal character closely associated, identical, colicine <br />     OXA-48 carbapenemase pandemic clone Escherichia coli O25b:H4-                          V. J Gen Microbiol 83: 95&#8211;111. <br />     ST131 course investigation outbreak OXA-48-producing                 40.   Barondess JJ, Beckwith J (1995) bor gene phage lambda, involved serum <br />     <span id='am-129' about='obo:NCBITaxon_570' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="Klebsiella Trevisan 1885 (Approved Lists 1980) emend. Carter et al. 1999" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasRelatedSynonym" content="Hyalococcus" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasRelatedSynonym" content="&quot;Hyalococcus&quot; Schroeter 1886" datatype="xsd:string"></span><span id='am-135' property="rdfs:label" content="Klebsiella" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasDbXref" content="PMID:11411716" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasAlternativeId" content="NCBITaxon:39823" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasDbXref" content="PMID:10555350" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasRelatedSynonym" content="Klebsiella Trevisan 1885 (Approved Lists 1980) emend. Drancourt et al. 2001" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasDbXref" content="PMID:12635932" datatype="xsd:string"></span><span id='am-141'  ></span>Klebsiella</span> pneumoniae. Antimicrob Agents Chemother 56: 4030&#8211;4031.                             resistance, encodes widely conserved outer membrane lipoprotein. J Bacteriol <br /> 15. Totsika M, Moriel DG, <span id='am-263' about='obo:NCBITaxon_81085' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_237399 obo:NCBITaxon_1 obo:NCBITaxon_81084 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_32417 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_33317 obo:NCBITaxon_7399 obo:NCBITaxon_2759 obo:NCBITaxon_50557 obo:NCBITaxon_6960 obo:NCBITaxon_6072'><span id='am-264'  ></span><span id='am-265' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-266' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-267' property="rdfs:label" content="Idris" datatype="xsd:string"></span>Idris</span> A, Rogers BA, Wurpel DJ, et al. (2012)                            177: 1247&#8211;1253. <br />     Uropathogenic Escherichia coli mediated urinary tract infection. Curr Drug              41.   Christen B, Abeliuk E, Collier JM, Kalogeraki VS, Passarelli B, et al. (2011) <br />     Targets 13: 1386&#8211;1399.                                                                        The essential genome bacterium. Mol Syst Biol 7: 528. <br /> 16. Ulett GC, Totsika M, Schaale K, Carey AJ, Sweet MJ, et al. (2013)                       42.   Langridge GC, Phan MD, Turner DJ, Perkins TT, Parts L, et al. (2009) <br />     Uropathogenic Escherichia coli virulence innate immune responses                   Simultaneous assay Salmonella Typhi gene using million <br />     urinary tract infection. Curr Opin Microbiol 16: 100&#8211;107.                                     transposon mutants. Genome Res 19: 2308&#8211;2316. <br /> 17. Clermont O, Lavollay M, Vimont S, Deschamps C, Forestier C, et al. (2008)               43.   Goodman AL, McNulty NP, Zhao Y, Leip D, <span id='am-209' about='obo:NCBITaxon_69650' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_57612 obo:NCBITaxon_69649 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-210'  ></span><span id='am-211' property="rdfs:label" content="Mitra" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-213' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Mitra</span> RD, et al. (2009) <br />     The CTX-M-15-producing Escherichia coli diffusing clone belongs highly                   Identifying genetic determinants needed establish human gut symbiont in <br />     virulent B2 phylogenetic subgroup. J Antimicrob Chemother 61: 1024&#8211;1028.                      habitat. Cell Host Microbe 6: 279&#8211;289. <br /> 18. Johnson JR, Porter SB, Zhanel G, Kuskowski MA, Denamur E (2012)                         44.   Gawronski JD, Wong SM, Giannoukos G, Ward DV, Akerley BJ (2009) <br />     Virulence Escherichia coli clinical isolates murine <span id='am-190' about='obo:NCBITaxon_137507' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_137503 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-191' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-192' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-193' property="rdfs:label" content="Sepsis" datatype="xsd:string"></span><span id='am-194'  ></span>sepsis</span> model                   Tracking insertion mutants libraries deep sequencing genome- <br />     relation sequence type ST131 status, fluoroquinolone resistance,                       wide screen <span id='am-77' about='obo:NCBITaxon_724' typeof='owl:Thing obo:NCBITaxon_135625 obo:NCBITaxon_712 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236'><span id='am-78' property="oboInOwl:hasDbXref" content="PMID:1736960" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Haemophilus Winslow et al. 1917" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Haemophilus" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-83'  ></span>Haemophilus</span> genes required lung. Proc Natl Acad <br />     virulence genotype. Infect Immun 80: 1554&#8211;1562.                                               Sci U S A 106: 16422&#8211;16427. <br /> 19. Lavigne JP, Vergunst AC, Goret L, Sotto A, Combescure C, et al. (2012)                  45.   van Opijnen T, Bodi KL, Camilli A (2009) Tn-seq: high-throughput parallel <br />     Virulence potential genomic mapping worldwide clone Escherichia                    sequencing fitness genetic interaction studies microorganisms. Nat <br />     coli ST131. PLoS One 7: e34294.                                                               Methods 6: 767&#8211;772. <br /> 20. Gibreel TM, Dodgson AR, Cheesbrough J, Bolton FJ, Fox AJ, et al. (2012)                 46.   Barquist L, Langridge GC, Turner DJ, Phan MD, Turner AK, et al. (2013) A <br />     High metabolic potential contribute success ST131 uropatho-                     comparison dense transposon insertion libraries Salmonella serovars <br />     genic Escherichia coli. J Clin Microbiol 50: 3202&#8211;3207.                                       Typhi Typhimurium. Nucleic Acids Res 41(8):4549&#8211;64. <br /> 21. Vimont S, Boyd A, Bleibtreu A, Bens M, Goujon JM, et al. (2012) The CTX-                47.   Datsenko KA, Wanner BL (2000) One-step inactivation chromosomal genes <br />     M-15-producing Escherichia coli clone O25b: H4-ST131 high intestine                       <span id='am-258' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-259' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-260' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-261' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-262' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span>Escherichia coli K-12</span> using PCR products. Proc Natl Acad Sci U S A 97: <br />     colonization urinary tract infection abilities. PLoS One 7: e46547.                       6640&#8211;6645. <br /> 22. Totsika M, Beatson SA, Sarkar S, Phan MD, Petty NK, et al. (2011) Insights              48.   Robinson MD, McCarthy DJ, Smyth GK (2010) edgeR: Bioconductor <br />     Multidrug Resistant Escherichia coli Pathogen Globally                          package differential expression analysis digital gene expression data. <br />     Disseminated ST131 Lineage: Genome Analysis Virulence Mechanisms.                         Bioinformatics 26: 139&#8211;140. <br />     PLoS One 6: e26578.                                                                     49.   Whang HY, Neter E (1973) Production common enterobacterial antigen by <br /> 23. Lau SH, Kaufmann ME, Livermore DM, Woodford N, Willshaw GA, et al.                            members family Enterobacteriaceae. Experientia 29: 96&#8211;97. <br />     (2008) UK epidemic Escherichia coli strains A-E, CTX-M-15 beta-                    50.   Makela PH, Mayer H (1976) Enterobacterial common antigen. Bacteriol Rev <br />     lactamase, belong international O25:H4-ST131 clone. J Antimicrob                   40: 591&#8211;632. <br />     Chemother 62: 1241&#8211;1244.                                                                51.   Beutin L, Manning PA, Achtman M, Willetts N (1981) sfrA sfrB products <br /> 24. Totsika M, Kostakioti M, Hannan TJ, Upton M, Beatson SA, et al. (2013) A                      Escherichia coli K-12 transcriptional control factors. J Bacteriol 145: <br />     FimH Inhibitor Prevents Acute Bladder Infection Treats Chronic Cystitis                   840&#8211;844. <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                                   16                            October 2013 | Volume 9 | Issue 10 | e1003834 <br />                                                                                                                               Serum Resistome E. coli ST131 Strain <br />  <br />  <br /> 52. Bailey MJ, Hughes C, Koronakis V (1996) Increased distal gene transcription              82. Frick DN, Townsend BD, Bessman MJ (1995) A novel GDP-mannose <br />     elongation factor RfaH, specialized homologue NusG. Mol                          mannosyl hydrolase shares homology MutT family enzymes. J Biol <br />     Microbiol 22: 729&#8211;737.                                                                       Chem 270: 24086&#8211;24091. <br /> 53. Nagy G, Dobrindt U, Schneider G, Khan AS, Hacker J, et al. (2002) Loss                83. Fraenkel DG, Levisohn SR (1967) Glucose gluconate metabolism an <br />     regulatory protein RfaH attenuates virulence uropathogenic Escherichia                    Escherichia coli mutant lacking phosphoglucose isomerase. J Bacteriol 93: <br />     coli. Infect Immun 70: 4406&#8211;4413.                                                            1571&#8211;1578. <br /> 54. Nagy G, Dobrindt U, Grozdanov L, Hacker J, Emody L (2005) Transcriptional                84. Fraenkel DG, Horecker BL (1965) Fructose-1, 6-diphosphatase acid <br />     regulation RfaH contributes intestinal colonization Escherichia                hexose phosphatase Escherichia coli. J Bacteriol 90: 837&#8211;842. <br />     coli. FEMS Microbiol Lett 244: 173&#8211;180.                                                  85. Prasadarao NV, Blom AM, Villoutreix BO, Linsangan LC (2002) A novel <br /> 55. Wang Q, Wang S, Beutin L, Cao B, Feng L, et al. (2010) Development                      interaction outer membrane protein A C4b binding protein mediates <br />     DNA microarray detection serotyping enterotoxigenic Escherichia                   serum resistance <span id='am-179' about='obo:NCBITaxon_1392869' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-180' property="rdfs:label" content="Escherichia coli K1" datatype="xsd:string"></span><span id='am-181' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-182' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia coli K1</span>. J Immunol 169: 6352&#8211;6360. <br />     coli. J Clin Microbiol 48: 2066&#8211;2074.                                                    86. Hellman J, Loiselle PM, Tehan MM, Allaire JE, Boyle LA, et al. (2000) Outer <br /> 56. Stevenson G, Neal B, Liu D, Hobbs M, Packer NH, et al. (1994) Structure                   membrane protein A, peptidoglycan-associated lipoprotein, murein <br />     O antigen Escherichia coli K-12 sequence rfb gene cluster.             lipoprotein released Escherichia coli bacteria serum. Infect Immun <br />     J Bacteriol 176: 4144&#8211;4156.                                                                   68: 2566&#8211;2572. <br /> 57. Kenne L, Lindberg B, Madden JK, Lindberg <span id='am-214' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-215' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-216' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-217'  ></span><span id='am-218' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-219' property="rdfs:label" content="Aa" datatype="xsd:string"></span>AA</span>, Gemski P, Jr. (1983)                        87. Kircher M, Heyn P, Kelso J (2011) Addressing challenges production <br />     Structural studies Escherichia coli O-antigen 25. Carbohydr Res 122:                  analysis illumina sequencing data. BMC Genomics 12: 382. <br />     249&#8211;256.                                                                                 88. Krueger F, Andrews SR, Osborne CS (2011) Large scale loss data low- <br /> 58. Fundin J, Weintraub A, Xu J-G, Widmalm G (2003) NMR analysis O-                       diversity illumina sequencing libraries recovered deferred cluster <br />     antigen polysaccharide Escherichia coli strain F171. Magnetic Resonance                 calling. PLoS One 6: e16607. <br />     Chemistry 41: 202&#8211;205.                                                                89. Crimmins GT, Mohammadi S, Green ER, Bergman MA, Isberg RR, et al. <br /> 59. Liu D, Cole RA, Reeves PR (1996) An O-antigen processing function Wzx                    (2012) Identification MrtAB, ABC transporter specifically required for <br />     (RfbX): promising candidate O-unit flippase. J Bacteriol 178: 2102&#8211;2107.               <span id='am-84' about='obo:NCBITaxon_633' typeof='owl:Thing obo:NCBITaxon_629 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasRelatedSynonym" content="Bacillus pseudotuberkulosis" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasRelatedSynonym" content="Pasteurella pseudotuberculosis" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasRelatedSynonym" content="Shigella pseudotuberculosis" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Yersinia pseudotuberculosis" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="Bacterium pseudotuberculosis" datatype="xsd:string"></span><span id='am-92'  ></span>Yersinia pseudotuberculosis</span> colonize mesenteric lymph nodes. PLoS <br /> 60. Han X, Dorsey-Oresto A, Malik M, Wang JY, Drlica K, et al. (2010)                            Pathog 8: e1002828. <br />     Escherichia coli genes reduce lethal effects stress. BMC Microbiol           90. Gallagher LA, Shendure J, Manoil C (2011) Genome-scale identification of <br />     10: 35.                                                                                      resistance functions <span id='am-38' about='obo:NCBITaxon_286' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621'><span id='am-39' property="oboInOwl:hasDbXref" content="PMID:7727274" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="PMID:10758879" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="RNA similarity group I" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Loefflerella" datatype="xsd:string"></span><span id='am-43'  ></span><span id='am-44' property="oboInOwl:hasDbXref" content="PMID:10939664" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="PMID:23918787" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="PMID:18048745" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="PMID:9103607" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="PMID:15950132" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Liquidomonas" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Pseudomonas" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Peudomonas" datatype="xsd:string"></span>Pseudomonas</span> aeruginosa using Tn-seq. MBio 2: <br /> 61. Allen MJ, White GF, Morby AP (2006) The response Escherichia coli                      e00315&#8211;00310. <br />     exposure biocide polyhexamethylene biguanide. Microbiology 152: 989&#8211;              91. van Opijnen T, Camilli A (2013) Transposon insertion sequencing: new tool <br />     1000.                                                                                        systems-level analysis microorganisms. Nat Rev Microbiol 11: 435&#8211;442. <br /> 62. Franco AV, Liu D, Reeves PR (1998) The wzz (cld) protein Escherichia coli:            92. Barquist L, Boinett CJ, Cain AK (2013) Approaches querying bacterial <br />     amino acid sequence variation determines O-antigen chain length specificity.                 genomes transposon-insertion sequencing. RNA Biol 10: 1161&#8211;9. <br />     J Bacteriol 180: 2670&#8211;2675.                                                              93. Koonin EV (2003) Comparative genomics, minimal gene-sets last <br /> 63. Stenutz R, Weintraub A, Widmalm G (2006) The structures Escherichia coli                  universal common ancestor. Nat Rev Microbiol 1: 127&#8211;136. <br />     O-polysaccharide antigens. FEMS Microbiol Rev 30: 382&#8211;403.                               94. Fang G, Rocha E, Danchin A (2005) How essential nonessential genes? Mol <br /> 64. Lundborg M, Modhukur V, Widmalm G (2010) Glycosyltransferase functions                       Biol Evol 22: 2147&#8211;2156. <br />     E. coli O-antigens. Glycobiology 20: 366&#8211;368.                                         95. Gerdes S, Edwards R, Kubal M, Fonstein M, Stevens R, et al. (2006) Essential <br /> 65. Schnaitman CA, Klena JD (1993) Genetics lipopolysaccharide biosynthesis                   genes metabolic maps. Curr Opin Biotechnol 17: 448&#8211;456. <br />     enteric bacteria. Microbiol Rev 57: 655&#8211;682.                                          96. Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, et al. (2003) <br /> 66. Raetz CR, Whitfield C (2002) Lipopolysaccharide endotoxins. Annu Rev                         Experimental determination level analysis essential genes in <br />     Biochem 71: 635&#8211;700.                                                                         Escherichia coli MG1655. J Bacteriol 185: 5673&#8211;5684. <br /> 67. Marolda CL, Tatar LD, Alaimo C, Aebi M, Valvano MA (2006) Interplay                   97. Baba T, <span id='am-25' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Ara</span> T, Hasegawa M, Takai Y, Okumura Y, et al. (2006) Construction <br />     Wzx translocase corresponding polymerase chain length                        Escherichia coli K-12 in-frame, single-gene knockout mutants: Keio <br />     regulator proteins translocation periplasmic assembly lipopoly-                collection. Mol Syst Biol 2: 2006 0008. <br />     saccharide o antigen. J Bacteriol 188: 5124&#8211;5135.                                        98. Taylor PW (1975) Genetical studies serum resistance Escherichia coli. <br /> 68. Alexander DC, Valvano MA (1994) Role rfe gene biosynthesis                  J Gen Microbiol 89: 57&#8211;66. <br />     Escherichia coli O7-specific lipopolysaccharide O-specific                 99. Russo TA, Sharma G, Brown CR, Campagnari AA (1995) Loss O4 <br />     polysaccharides containing N-acetylglucosamine. J Bacteriol 176: 7079&#8211;7084.                  antigen moiety lipopolysaccharide extraintestinal isolate of <br /> 69. Danese PN, Pratt LA, Kolter R (2000) Exopolysaccharide production                         Escherichia coli minor effects serum sensitivity virulence in <br />     required development Escherichia coli K-12 biofilm architecture.                      vivo. Infect Immun 63: 1263&#8211;1269. <br />     J Bacteriol 182: 3593&#8211;3596.                                                             100. Buckles EL, Wang X, Lane MC, Lockatell CV, Johnson DE, et al. (2009) Role <br /> 70. Bateman SL, Seed PC (2012) Epigenetic regulation nitrosative stress                   K2 capsule Escherichia coli urinary tract infection serum <br />     response intracellular macrophage survival extraintestinal pathogenic                 resistance. J Infect Dis 199: 1689&#8211;1697. <br />     Escherichia coli. Mol Microbiol 83: 908&#8211;925.                                            101. Leying H, Suerbaum S, Kroll HP, Stahl D, Opferkuch W (1990) The capsular <br /> 71. Allsopp LP, Beloin C, Ulett GC, Valle J, Totsika M, et al. (2012) Molecular                  polysaccharide major determinant serum resistance K-1-positive blood <br />     characterization UpaB UpaC, new autotransporter proteins                       culture isolates Escherichia coli. Infect Immun 58: 222&#8211;227. <br />     uropathogenic <span id='am-220' about='obo:NCBITaxon_199310' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-221' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-222' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-223' property="oboInOwl:hasExactSynonym" content="Escherichia coli strain CFT073" datatype="xsd:string"></span><span id='am-224' property="oboInOwl:hasExactSynonym" content="Escherichia coli str. CFT073" datatype="xsd:string"></span><span id='am-225' property="rdfs:label" content="Escherichia coli CFT073" datatype="xsd:string"></span>Escherichia coli CFT073</span>. Infect Immun 80: 321&#8211;332.                        102. Whitfield C (2006) Biosynthesis assembly capsular polysaccharides in <br /> 72. Keseler IM, Collado-Vides J, Santos-Zavaleta A, Peralta-Gil M, Gama-Castro                   Escherichia coli. Annu Rev Biochem 75: 39&#8211;68. <br />     S, et al. (2011) EcoCyc: comprehensive database Escherichia coli biology.          103. Stevenson G, Andrianopoulos K, Hobbs M, Reeves PR (1996) Organization of <br />     Nucleic Acids Res 39: D583&#8211;590.                                                              Escherichia coli K-12 gene cluster responsible production the <br /> 73. Yem DW, Wu HC (1978) Physiological characterization Escherichia coli                   extracellular polysaccharide colanic acid. J Bacteriol 178: 4885&#8211;4893. <br />     mutant altered structure murein lipoprotein. J Bacteriol 133: 1419&#8211;           104. White RJ (1968) Control amino sugar metabolism Escherichia coli and <br />     1426.                                                                                        isolation mutants unable degrade amino sugars. Biochem J 106: 847&#8211;858. <br /> 74. Vadillo-Rodriguez V, Schooling SR, Dutcher JR(2009) In situ characterization            105. Park JT, Uehara T (2008) How bacteria consume exoskeletons <br />     differences viscoelastic response individual gram-negative                  (turnover recycling cell wall peptidoglycan). Microbiol Mol Biol Rev 72: <br />     gram-positive bacterial cells. J Bacteriol 191: 5518&#8211;5525.                                   211&#8211;227. <br /> 75. Cascales E, Lloubes R, Sturgis JN (2001) The TolQ-TolR proteins energize                106. Adhya S, Schwartz M (1971) Phosphoglucomutase mutants Escherichia coli <br />     TolA share homologies flagellar motor proteins MotA-MotB. Mol                   K-12. J Bacteriol 108: 621&#8211;626. <br />     Microbiol 42: 795&#8211;807.                                                                  107. Inoue T, Shingaki R, Hirose S, Waki K, Mori H, et al. (2007) Genome-wide <br /> 76. Walburger A, Lazdunski C, Corda Y (2002) The Tol/Pal function                         screening genes required swarming motility Escherichia coli K-12. <br />     requires interaction C-terminal domain TolA N-                     J Bacteriol 189: 950&#8211;957. <br />     terminal domain TolB. Mol Microbiol 44: 695&#8211;708.                                     108. Lu M, Kleckner N (1994) Molecular cloning characterization pgm <br /> 77. Onufryk C, Crouch ML, Fang FC, Gross CA (2005) Characterization                       gene encoding phosphoglucomutase Escherichia coli. J Bacteriol 176: 5847&#8211; <br />     lipoproteins sigmaE regulon. J Bacteriol 187: 4552&#8211;4561.                              5851. <br /> 78. Nikaido H, Vaara M (1985) Molecular basis bacterial outer membrane                   109. Frey PA (1996) The Leloir pathway: mechanistic imperative three <br />     permeability. Microbiol Rev 49: 1&#8211;32.                                                        enzymes change stereochemical configuration single carbon in <br /> 79. Yan A, Guan Z, Raetz CR (2007) An undecaprenyl phosphate-aminoarabinose                      galactose. FASEB J 10: 461&#8211;470. <br />     flippase required polymyxin resistance Escherichia coli. J Biol Chem 282:        110. Weissborn AC, Liu Q, Rumley MK, Kennedy EP (1994) UTP: alpha-D- <br />     36077&#8211;36089.                                                                                 glucose-1-phosphate uridylyltransferase Escherichia coli: isolation DNA <br /> 80. Trent MS, Ribeiro AA, Doerrler WT, Lin S, Cotter RJ, et al. (2001)                           sequence galU gene purification enzyme. J Bacteriol 176: <br />     Accumulation  <br /> </body></html>