<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Zomorrodi Maranas BMC Systems Biology 2010, 4:178 <br /> http://www.biomedcentral.com/1752-0509/4/178 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                             Open Access <br />  <br /> Improving iMM904 S. cerevisiae metabolic <br /> model using essentiality synthetic <br /> lethality <span id='am-257' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-258' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-259' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-260' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-261'  ></span>data</span> <br /> Ali R Zomorrodi, Costas D Maranas* <br />  <br />  <br />   Abstract <br />   Background: <span id='am-71' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-82'  ></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> eukaryotic organism multi-compartment genome- <br />   scale metabolic model constructed. Since sequence improved metabolic reconstructions yeast <br />   introduced. These metabolic models extensively used elucidate organizational <br />   principles yeast metabolism drive yeast strain engineering strategies targeted overproductions. They <br />   served starting point benchmark reconstruction genome-scale metabolic models for <br />   eukaryotic organisms. In spite successive improvements details described metabolic <br />   processes, recent yeast model (i.e., iMM904) remains significantly predictive latest E. coli <br />   model (i.e., iAF1260). <span id='am-113' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-114'  ></span><span id='am-115' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> manifested significantly lower specificity predicting outcome grow/no <br />   grow experiments comparison E. coli model. <br />   Results: In paper make use automated GrowMatch procedure restoring consistency single <br />   gene deletion experiments yeast extend procedure make use synthetic lethality data using the <br />   genome-scale model iMM904 basis. We identified vetted using literature sources 120 distinct model <br />   modifications including various regulatory constraints minimal YP media. The incorporation the <br />   suggested modifications led substantial increase fraction correctly predicted lethal knockouts (i.e., <br />   specificity) 38.84% (87 224) 53.57% (120 224) minimal medium 24.73% (45 <br />   182) 40.11% (73 182) YP medium. Synthetic lethality predictions improved 12.03% (16 <br />   133) 23.31% (31 133) minimal medium 6.96% (8 115) 13.04% (15 of <br />   115) YP medium. <br />   Conclusions: Overall, study provides roadmap computationally driven correction multi- <br />   compartment genome-scale metabolic models demonstrates value synthetic lethals curation agents. <br />  <br />  <br /> Background                                                                          compartments environment and <br /> <span id='am-88' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-89' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-91'  ></span><span id='am-92' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae eukaryote                               cytosol included model. Soon thereafter, the <br /> genome fully sequenced annotated [1]. It                                iND750 model introduced, included five <br /> since focus genome-scale recon-                               additional compartments (i.e., peroxisome, nucleus, golgi <br /> struction efforts. Forster et al [2] reconstructed                        apparatus, vacuole endoplasmic reticulum) re- <br /> multi-compartment genome-scale metabolic model                                  assessing localization gene products [3]. In par- <br /> yeast (i.e., iFF708). The model accounted 708 open                              allel study [4], predictive capability iFF708 was <br /> reading frames (ORFs) (~10.5% ORFs) 1,175                                improved number modifications bio- <br /> metabolic reactions. The metabolic reactions                               mass equation removal blocked reactions <br /> model compartmentalized cytosol                                    (iLL672 model). Subsequently, version the <br /> mitochondria. Transport mechanisms                                yeast metabolic model improved description of <br />                                                                                     lipid metabolism containing 800 genes 1,446 <br /> * Correspondence: costas@psu.edu                                                    reactions (i.e., iIN800) introduced Nookaew et al <br /> Department Chemical Engineering, The Pennsylvania State University, <br /> University Park, PA 16802, USA <br />                                                                                     [5]. These improvements yeast model culminated <br />  <br />                                       &#169; 2010 Zomorrodi Maranas; licensee BioMed Central Ltd. This Open Access article distributed terms Creative <br />                                       Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, and <br />                                       reproduction medium, provided original work properly cited. <br />  Zomorrodi Maranas BMC Systems Biology 2010, 4:178                                                                                                 Page 2 15 <br /> http://www.biomedcentral.com/1752-0509/4/178 <br />  <br />  <br />  <br />  <br /> with iMM904 model [6] increased size                               model experimental data imply growth <br /> iND750 904 genes 1,412 reactions. All                                 (G) growth (NG) mutant strain, NGG <br /> metabolic reconstruction efforts carried largely                          model predicts gene deletion lethal <br /> independently using different data                                  experiment shows viable, finally <br /> sources cases different interpretations                          GNG model predicts mutant strain <br /> same literature evidence. This lack consistency moti-                           viable vivo observations lethal <br /> vated yeast systems biology community consoli-                              effect. Reed et al [19] proposed systems analysis <br /> date available metabolic models single                                  approach restore growth NGGs variety of <br /> consensus reconstruction annotation model [7].                                 growth media addition appropriate <span id='am-233' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-234'  ></span><span id='am-235' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-236' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-237' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>meta</span>- <br /> This model updated regularly pub-                            bolic transport functions model. In another <br /> lished latest version (i.e., Yeast 4.0) contains,                       study, Satish Kumar et al [20] introduced GapFind and <br /> decompartmentalized form, 1102 reactions 924 pro-                              GapFill, identify metabolites pro- <br /> teins [8].                                                                         duced consumed model uptake con- <br />    Despite efforts exists gap                       ditions bridge gaps adding missing <br /> quality available metabolic reconstructions                            functionalities model. Subsequently, pro- <br /> for yeast corresponding models microbial meta-                             cedure termed GrowMatch [18] proposed recon- <br /> bolism. Table 1 summarizes quality metrics                             cile NGG GNG growth prediction <br /> E. coli [9], M. genitalium [10], B. subtilis [11], P. putida                       inconsistencies different substrates (see Orth and <br /> [12], H. pylori [13] <span id='am-220' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-221'  ></span><span id='am-222' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-223' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-224' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-225' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-226' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-227' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-228' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-229' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span><span id='am-230' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-231' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-232' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span>Salmonella</span> Typhimurium [14]                               Palsson [21] review). Notably, GrowMatch <br /> latest genome-scale models growth predictions                             procedure [18], GNG mismatches corrected by <br /> contrasted experimental data single gene                               modifying metabolic model convert to <br /> deletions. As shown table, accuracy (i.e.,                         NGNGs. Alternatively, Motter et al [22,23] suggested to <br /> fraction correctly predicted lethal knockouts)                           reconcile GNGs identifying candidate gene knockouts <br /> iMM904 metabolic model yeast significantly                               restore growth mutants were <br /> worse corresponding microbial models.                              initially non-viable vivo. <br /> This partly caused uncertainty enzyme localiza-                             Recent studies suggested making use not <br /> tion inter-compartment metabolite transport                                 single gene deletion information synthetic <br /> yeast [15]. This implies draft model reconstruc-                            lethal pairs (higher orders) provide additional <br /> tion followed detailed manual curation                               layer curation validation metabolic models. <br /> not sufficient strategy bring eukaryotic gen-                            Harrison et al [24] showed investigation fal- <br /> ome-scale model quality level microbial                           sely predicted synthetic lethals S. cerevisiae help <br /> one. Here explore extent model correction                                improve functional annotation. Furthermore, recent <br /> that brought systematically comparing                              research group [25] demonstrated mis- <br /> the model predictions single multiple gene dele-                           matches growth auxotrophy pheno- <br /> tions available experimental data.                                            types synthetic lethal predictions vivo <br />    The established standard [15] testing accuracy                          observations used provide 19 model correction <br /> of genome-scale metabolic models contrast                                hypotheses iAF1260 model E. coli. In this <br /> predicted growth phenotype single mutant strains                                paper, modify deploy automated approaches <br /> with available experimental data various                                 [18,20] resolving falsely predicted single multiple <br /> growth conditions [6,9,16,17]. These comparisons result                            gene deletions aerobic minimal complex <br /> in different outcomes [18]: GG NGNG                                   (YP) medium, generation appropriate <br />  <br />  <br /> Table 1 Accuracy microbial metabolic models vs iMM904 model. <br />            Microorganism                          Name metabolic model                         Specificity (%)                      Reference <br />        Saccharomyces cerevisiae                                 iMM904                                      38.8                        [6] study <br />             Escherichiae coli                                   iAF1260                                     73.4                                [9] <br />         <span id='am-34' about='obo:NCBITaxon_2093' typeof='owl:Thing obo:NCBITaxon_2092 obo:NCBITaxon_1 obo:NCBITaxon_2085 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_544448 obo:NCBITaxon_31969'><span id='am-35' property="oboInOwl:hasDbXref" content="PMID:16403858" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Asteromyces" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="PMID:13403276" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="PMID:15176735" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="PMID:11321109" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Bovimyces" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="PMID:8863441" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:4" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Asterococcus" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Pleuropneumonia" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="PMID:11411711" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasAlternativeId" content="NCBITaxon:29500" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Mycoplasma" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="PMID:8995799" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Borrelomyces" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Eperythrozoon" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasAlternativeId" content="NCBITaxon:57371" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="PMID:10826816" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="PMID:16350067" datatype="xsd:string"></span><span id='am-55'  ></span><span id='am-56' property="oboInOwl:hasDbXref" content="PMID:11931184" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasRelatedSynonym" content="Haemobartonella" datatype="xsd:string"></span>Mycoplasma</span> genitalium                                    iPS189                                     79.0                               [10] <br />            <span id='am-1' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-2' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span>Bacillus subtilis</span>                                   iBSU1103                                     89.3                               [11] <br />          <span id='am-126' about='obo:NCBITaxon_303' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_286 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621 obo:NCBITaxon_136845'><span id='am-127' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas striata" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasDbXref" content="PMID:1177780" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1416793" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas ovalis" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas rugosa" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasDbXref" content="PMID:11211254" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasDbXref" content="PMID:9734035" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasAlternativeId" content="NCBITaxon:29437" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas sp. PX-A-Ps.sp2" datatype="xsd:string"></span><span id='am-137'  ></span><span id='am-138' property="oboInOwl:hasRelatedSynonym" content="Pseudomanas putida" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas incognita" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasAlternativeId" content="NCBITaxon:691260" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:hasAlternativeId" content="NCBITaxon:126995" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas convexa" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931964" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasAlternativeId" content="NCBITaxon:668614" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931954" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas eisenbergii" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasRelatedSynonym" content="Arthrobacter siderocapsulatus" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasAlternativeId" content="NCBITaxon:105847" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasAlternativeId" content="NCBITaxon:79218" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas arvilla" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasRelatedSynonym" content="Bacillus putidus" datatype="xsd:string"></span><span id='am-153' property="rdfs:label" content="Pseudomonas putida" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasDbXref" content="PMID:17978216" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasAlternativeId" content="NCBITaxon:760256" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasRelatedSynonym" content="Bacillus fluorescens putidus" datatype="xsd:string"></span>Pseudomonas putida</span>                                         -                                       74.5                               [12] <br />            <span id='am-157' about='obo:NCBITaxon_210' typeof='owl:Thing obo:NCBITaxon_72293 obo:NCBITaxon_213849 obo:NCBITaxon_209 obo:NCBITaxon_1 obo:NCBITaxon_68525 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_29547'><span id='am-158' property="oboInOwl:hasRelatedSynonym" content="Campylobacter pylori subsp. pylori" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasRelatedSynonym" content="Campylobacter pylori" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasDbXref" content="PMID:8494747" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasRelatedSynonym" content="Helicobacter nemestrinae" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasDbXref" content="PMID:1995031" datatype="xsd:string"></span><span id='am-164'  ></span><span id='am-165' property="oboInOwl:hasDbXref" content="PMID:8186097" datatype="xsd:string"></span><span id='am-166' property="rdfs:label" content="Helicobacter pylori" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasAlternativeId" content="NCBITaxon:219" datatype="xsd:string"></span><span id='am-169' property="oboInOwl:hasDbXref" content="PMID:11931154" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:hasRelatedSynonym" content="Campylobacter pyloridis" datatype="xsd:string"></span>Helicobacter pylori</span>                             iIT341 GSM/GPR                                   73.0                               [13] <br />        Salmonella Typhimurium                                   iMA945                                      66.7                               [14] <br /> Comparison fraction correctly predicted lethal knockouts (i.e., specificities) microbial genome-scale metabolic models yeast iMM904 <br /> model single gene mutation experiments. <br />  Zomorrodi Maranas BMC Systems Biology 2010, 4:178                                                         Page 3 15 <br /> http://www.biomedcentral.com/1752-0509/4/178 <br />  <br />  <br />  <br />  <br /> model-correction hypotheses iMM904 model           carrying function absent the <br /> the yeast. We chose iMM904 study          model specific gene-protein-reaction (GPR) associa- <br /> contains biomass composition reaction reversibility       tions [18]. A bidirectional protein-protein BLAST (i.e., <br /> information making suitable performing FBA com-        BLASTp) search S. cerevisiae genome <br /> putations. Following workflow presented           revealed seven genes involved NGGs <br /> paper pinpointed 90 model corrections identified       minimal medium YP <br /> 30 missing regulatory constraints supporting litera-     medium share significant sequence similarity the <br /> ture evidence doubled prediction accu-        entire length protein (i.e., forward backward <br /> racy growth phenotype single double gene        BLASTp expectation value 10-13) other <br /> deletions aerobic minimal YP media          ORFs yeast. We however, rejected resolu- <br /> iMM904 model. Examples literature evidence include         tion strategies minimal medium the <br /> interaction proteins support modification      YP medium contradicting experimental evidence <br /> GPR associations, presence specific compound          (see Additional file 2 details). Alternative correction <br /> the cell wall corroborate inclusion biomass     strategies resolved NGGs explored. <br /> equation, gene expression data confirm suppres-         Two NGG inconsistencies minimal medium <br /> sion gene given condition. The majority       YP medium resolved relax- <br /> modifications proposed (i.e., 86% them) remain        ing irreversibility reactions. For exam- <br /> relevant latest update community          ple, deletion gene ADK1 (YDR226W), is <br /> yeast model (Yeast 4.0) [8].                                  involved nucleoside salvage pathway, non- <br />                                                               lethal treating reactions ADPT (Adenine <br /> Results discussion                                        phosphoribosyltransferase), HXPRT (Hypoxanthine <br /> Single gene perturbations                                     phosphoribosyltransferase (Hypoxanthine)), UPPRT <br /> Analysis impact single gene reaction dele-       (Uracil phosphoribosyltransferase) GUAPRT (Gua- <br /> tions growth phenotype (for iMM904 model)          phosphoribosyltransferase) reversible. The rever- <br /> revealed 106 essential genes 163 essential reactions      sibility reactions supported based <br /> for growth using minimal medium 57 essen-        value free Gibbs energy change, &#916;G, pre- <br /> tial genes 92 essential reactions YP medium.      vious reports literature [27-30] (also Additional <br /> We contrasted predicted essential genes           file 2). Notably, relaxing irreversibility constraint on <br /> media experimental single gene deletion data           reactions enables production prpp <br /> reported Kuepfer et al [4] pinpoint model           (5-phospho-alpha-D-ribose 1-diphosphate), serves <br /> inconsistencies. A summary model/experiment            precursor biomass components. <br /> (mis)matches given Figure 1A 1B (see Addi-             Upon incorporation global modifications <br /> tional file 1 details). We note metrics      model, pursued resolution additional <br /> specificity sensitivity figure represent      NGGs adding transport reactions reactions from <br /> fraction correctly predicted lethal viable mutants,    KEGG database [31] (see Methods). We able to <br /> respectively. Comparing accuracy iMM904            reconcile growth inconsistencies NGGs in <br /> model iFF708 (specificity = 68.2%) [2,26]            minimal medium method. For example, ADE8 <br /> growth using YP medium, signifi-      (YDR408C), codes reaction GARFTi (<span id='am-267' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-268' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-269'  ></span><span id='am-270' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-271' property="rdfs:label" content="Phos" datatype="xsd:string"></span>phos</span>- <br /> cant reduction specificity. This reduction primarily    phoribosylglycinamide formyltransferas) fixed by <br /> due uncertainties associated assignment        adding reaction R06974 (glycinamide ribonucleotide <br /> functionalities compartments expansion      transformylase) model. GARFTi serves a <br /> of model studied reactions, pointed      step novo purine nucleotide biosynthetic path- <br /> earlier [15]. We applied computational procedures         ways reaction model responsible for <br /> described Methods section reconcile false     producing essential metabolite fgam (N2-Formyl-N1- <br /> predictions experimental growth. We note        (5-phospho-D-ribosyl)glycinamide). Addition reaction <br /> all reaction metabolite abbreviations          R06974 model provides alternative way pro- <br /> this manuscript based iMM904 model.                ducing fgam compensate absence of <br /> Resolution NGG inconsistencies                             GARFTi. By performing BLAST bi-directional test we <br /> As denoted Figure 1A 1B, identified 19 NGGs         identified ORF yeast genome (i.e., ADE2) with <br /> for minimal 14 YP medium. A summary           high sequence similarity (forward E-value = 3 &#215; <br /> of pathways NGGs growth             10-24 backward E-value = 3 &#215; 10-24) enzyme <br /> media involved given Additional file 1. The   phosphoribosylglycinamide formyltransferase 2 (EC <br /> step resolution NGGs GrowMatch              2.1.2.-) catalyzing R06974 <span id='am-63' about='obo:NCBITaxon_2184' typeof='owl:Thing obo:NCBITaxon_183939 obo:NCBITaxon_2183 obo:NCBITaxon_1 obo:NCBITaxon_2182 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_28890'><span id='am-64' property="oboInOwl:hasDbXref" content="PMID:8782682" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Methanococcus" datatype="xsd:string"></span><span id='am-68'  ></span><span id='am-69' property="oboInOwl:hasDbXref" content="PMID:15653928" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="PMID:10319480" datatype="xsd:string"></span>Methanococcus</span> jannaschii, <br /> procedure look alternate genes yeast capable     supports presence reaction yeast. <br />  Zomorrodi Maranas BMC Systems Biology 2010, 4:178                                                                           Page 4 15 <br /> http://www.biomedcentral.com/1752-0509/4/178 <br />  <br />  <br />  <br />  <br />  Figure 1 Accuracy iMM904 model modifications using single gene perturbations. The number false and <br />  correct predictions iMM904 model (A B) (C D) applying modifications single gene mutations under <br />  minimal YP media, respectively. Note specificity = #NGNG/(#NGNG + #GNG), sensitivity true viable rate (TVR) = #GG/(#GG + #NGG) <br />  false viable rate (FVR) = #GNG/(#GNG + #NGNG). <br />  <br />  <br />   Except cases (i.e., RIB1 RIB4),                 silico YP medium. This renders deletion of <br /> NGGs identified YP medium present                      RIB1 RIB4 non-lethal. Notably, importance of <br /> and corrected considering mini-                      correctly describing medium composition in <br /> mal medium. Both RIB1 (YBL033C) RIB4                                growth phenotype predictions raised pre- <br /> (YOL143C) fixed modifying iMM904                         vious studies [26]. Overall, using GrowMatch <br /> model adding missing compounds                            procedure literature searches able <br /> in silico YP medium description. Both genes                    fix thirteen NGGs minimal medium and <br /> are involved riboflavin (vitamin B 2 ) biosynthesis,                 YP medium (see Figure 1). The <br /> which biomass precursor. Because yeast extract                  details analysis evidence(s) <br /> reported major source vitamin B 2 [32],                   support modification provided as <br /> decided add riboflavin list components                  Additional file 2. <br />  Zomorrodi Maranas BMC Systems Biology 2010, 4:178                                                         Page 5 15 <br /> http://www.biomedcentral.com/1752-0509/4/178 <br />  <br />  <br />  <br />  <br /> Resolution GNG inconsistencies                             GNGs code reactions blocked the <br /> As shown Figure 1A 1B, total 137 GNG              model mainly presence no-consumption <br /> inconsistencies minimal YP media iden-        metabolite. One resolve inconsistency by <br /> tified 128 jointly present. The distribution     exploring no-consumption metabolite a <br /> of genes different functional classes         component biomass equation. Interestingly, we <br /> metabolism revealed involved         21 GNGs code tRNA char- <br /> tRNA charging, oxidative phosphorylation extracel-        ging reactions converted NGNG consis- <br /> lular transport (see Additional file 1). Resolution        tencies including charged uncharged tRNA <br /> these GNGs GrowMatch generally involves suppres-           molecules place corresponding amino acids in <br /> sion incorrectly present alternative production routes     biomass equation. This strategy previously <br /> of biomass components model. The identified            used reconstruction Salmonella metabolic <br /> GNGs divided major categories based         model [14]. For example, GNG prediction WRS1 <br /> how affect flux distribution network [18].    (YOL097C), codes blocked reaction <br /> The category comprised genes coding <span id='am-251' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-252' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-253' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-254'  ></span><span id='am-255' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>iso</span>-      TRPTRS (Tryptophanyl tRNA synthetase), corrected <br /> zymes alluding deletion genes        including charged tRNA (trptrna) uncharged <br /> not affect reaction flux. A straightforward resolution    tRNA (trantrp) reactant product biomass <br /> strategy case suppress gene(s)        reaction, respectively. This modification simultaneously <br /> whose product serves isozyme coded reaction.       unblocks reaction TRPTRS renders essential. <br /> An alternative hypothesis modify corresponding         In cases, including no-consumption metabo- <br /> GPR relationship recast deletion         lite blocked reaction encoded GNG the <br /> gene lethal. These resolution strategies          biomass equation viable option con- <br /> viable coded reaction(s) essential form    verted GG consistencies NGG mismatches (i.e., <br /> synthetic lethal(s) according model. For example,      conditional modification). Using GapFind proce- <br /> gene ACO1 (YLR304C), identified GNG                dure [20] problem metabolites are <br /> minimal medium, isozyme (with ACO2) mito-           upstream no-consumption, implying be <br /> chondrial aconitase (ACONTm) independently           fixed corresponding <span id='am-118' about='obo:NCBITaxon_1' typeof='owl:Thing'><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="all" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-122' property="rdfs:label" content="root" datatype="xsd:string"></span>root</span> (or down- <br /> catalyzes cytosolic aconitaase (ACONT) according           stream) no-consumption metabolite(s) resolved. <br /> the iMM904 model. Notably, reactions, ACONT               Therefore, tested GNGs fixed <br /> ACONTm form synthetic lethal pair mini-           globally, root downstream no-con- <br /> mal medium according iMM904 model. We did              sumption metabolites included component the <br /> not evidence indicating ACO2 sup-            biomass equation. An additional seven GNGs corre- <br /> pressed aerobic glucose condition. Instead,          sponding blocked reactions fixed method <br /> found protein coded ACO2 (Aco2p)         added list proposed model modi- <br /> putative mitochondrial aconitase isozyme, func-         fications lack experimental validation (see <br /> tion assigned based sequence simi-       Additional file 2 uncorroborated model changes). <br /> larity Aco1p [33,34]. Therefore, decided           We note incomplete description biomass <br /> remove ACO2 GPR ACONTm ren-                 equation implicated important source of <br /> dering deletion ACO1 lethal. This modification      false model predictions earlier studies [3,4,26]. There- <br /> is consistent latest update community         fore, exercised great caution modifying the <br /> model [8]. Notably, possibility ACO2       biomass equation fix inconsistency. Corrections <br /> correctly assigned ACONTm, ACO1 involved            accepted solid corroborating literature evi- <br /> in unaccounted model essential functions         dence found. <br /> (e.g., mitochondrial DNA maintenance [35]) addition           Two GNGs identified correspond a <br /> to aconitase activity. We manage fix GNGs         reaction blocked minimal conditions: <br /> under minimal medium YP               Genes MPH2 (YDL247W) MPH3 (YJR160C) encode <br /> medium using procedure.                               isozymes catalyzing transport maltose the <br />    The second category GNGs contains genes            extra-cellular environment cytosol (reaction MALTt2 <br /> code blocked reactions, i.e., reactions   iMM904). This reaction blocked maltose <br /> carry flux. This implies dele-       present minimal medium. Therefore, we <br /> tion genes affect flux distribution     explored genes involved in <br /> in silico, knockout lethal vivo. Resolution    transport D-glucose (reaction GLCt1). A BLAST <br /> these inconsistencies involves unblocking coded     bi-directional analysis revealed genes <br /> reactions rendering deletion lethal        high sequence similarity glucose trans- <br /> the examined conditions. Notably, 52            port genes. Interestingly, previous studies <br />  Zomorrodi Maranas BMC Systems Biology 2010, 4:178                                                                       Page 6 15 <br /> http://www.biomedcentral.com/1752-0509/4/178 <br />  <br />  <br />  <br />  <br /> have reported involvement MPH2 MPH3 <br /> in mediating residual glucose uptake [36]. <br />    The category GNGs contains genes whose <br /> deletion affects flux distribution network. To <br /> rectify mismatches employed original <br /> GrowMatch procedure introduced [18] modi- <br /> fied version proposed study (see Methods) to <br /> identify minimal number genes/reactions whose <br /> suppression lower biomass formation pre- <br /> specified viability threshold. We performed analysis <br /> by allowing simultaneous gene/reaction <br /> suppressions GNG leading correction of <br /> 18 inconsistencies minimal 19 the <br /> YP medium. Twelve fixed GNGs minimal <br /> medium fourteen YP excluded the <br /> list corrected mismatches did sup- <br /> porting evidence literature. Overall, fixed 33 <br /> GNGs minimal 28 YP medium glo- <br /> bal modifications conflict correctly <br /> predicted growth phenotype single gene mutants.              Figure 2 Different types mismatches silico <br /> We note 27 GNGs appeared media                 predictions vivo observations double gene <br /> were corrected mechanisms (see Addi-             perturbations. The abbreviations G, ES SL figure refer to <br /> tional file 2 exhaustive list corrections).           Growth, Essential Synthetic Lethal, respectively. Here, &#8216;No Growth&#8217; <br />                                                                  essentiality synthetic lethality single or <br />    Upon incorporating model correc-            double gene deletions. <br /> tions NGGs GNGs litera- <br /> ture evidence, accuracy iMM904 model was <br /> substantially improved. The number correctly pre-            Synthetic Lethal). Similarly, ESG ESSL represent <br /> dicted lethal knockouts (i.e., specificity) total   mismatches single deletion the <br /> 224 increased 87 120 minimal med-           genes silico lethal (i.e., ESsential), however, their <br /> ium 45 73 (out total 182) YP        simultaneous deletion vivo results viable <br /> medium. The corresponding false viability rate (FVR)            strain (i.e., Growth) lethal phenotype (i.e., Synthetic <br /> was decreased 61.16% 46.43% minimal             Lethal), respectively. Finally, SLG SLES denote <br /> medium 75.27% 59.89% YP medium.             inconsistencies model implies the <br /> These results summarized Figure 1C 1D.               double gene mutation lethal (i.e., Synthetic Lethal) but <br />                                                                 experimental observations support growth (G) or <br /> Double gene perturbations                                       lethality of single gene deletions (i.e., <br /> We employed SL-Finder procedure [25] identify            ESsential), respectively. <br /> the set synthetic lethal (SL) gene pairs        In following restoration of <br /> minimal YP media based iMM9904 model              type inconsistencies used to <br /> yeast. This analysis led identification 97 SL pairs    improve predictive capability iMM904 model. <br /> the minimal medium 42 YP medium. Model               In analogy <span id='am-256' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> specificity single gene <br /> SL prediction inconsistencies identified con-           mutations, define specificity double gene per- <br /> trasting available experimentally identified        turbations fraction correctly predicted syn- <br /> SLs (see Additional file 1 complete list predicted     thetic lethals (SLSL), i.e., Specificity = #SLSL/(#SLSL + <br /> and experimentally identified synthetic lethal interac-         #ESSL +# GSL). We <span id='am-250' about='rdfs:comment' typeof='owl:Thing'>comment</span> resolution <br /> tions). As shown Figure 2, comparison reveals         GES ESG mismatches, completely <br /> number additional ways model experiment             equivalent GNG NGG inconsistencies. In addi- <br /> may differ predictions. Notably, &#8220;no growth&#65533;?       tion, note model corrections based <br /> phenotype case essentiality      currently available incomplete list experimen- <br /> (ES) synthetic lethality (SL) gene deletions. For     tally identified SLs. <br /> example, GES GSL inconsistencies refer cases             Resolution GSL inconsistencies <br /> where silico deletion gene pair lethal       The GSL inconsistencies denote cases model <br /> (i.e., Growth) vivo lethal gene          implies growth simultaneous deletion two <br /> essentiality synthetic lethality (i.e., ESsential         genes experimental results lethal effect. A <br />  Zomorrodi Maranas BMC Systems Biology 2010, 4:178                                                           Page 7 15 <br /> http://www.biomedcentral.com/1752-0509/4/178 <br />  <br />  <br />  <br />  <br /> total 104 GSLs minimal medium 98          GapFill procedure applied correct root <br /> YP medium identified. We note GSLs dou-          no-consumption metabolites adding consumption <br /> ble perturbations treated similarly GNG          export pathway model, unblocking the <br /> mismatches single perturbations differ-      coded reactions. With modification, GSLs can <br /> ence &#8220;no growth&#65533;? GNGs essentiality       treated deletion single <br /> of single genes GSLs result syn-       gene change flux distribution silico. We could <br /> thetic lethality.                                              fix GSL inconsistency modifying the <br />   The strategy fixing GSLs involved coding          biomass equation: KRE6 (YPR159W) SKN1 <br /> isozymes repress genes coding         (YGR143W) provide isozymes blocked reac- <br /> isozyme examined condition modify             tion 16GS (1,6 b-glucan synthase) fixed includ- <br /> GPR associations. When genes code isozymes            ing 16BDglcn (1,6 b-D-glucan) root no-consumption <br /> for reaction resolution strategy viable       metabolite biomass equation reactant. This <br /> only reaction essential silico. Alternatively,   modification corroborated previous reports <br /> when genes associated (or more) dif-          showing 1,6 b-D-glucan key component the <br /> ferent reactions form synthetic lethal. We         yeast cell wall [40-42]. <br /> note gene repressions group GSLs            Reconciling GSLs disruption single gene <br /> can inferred manually inspection           affects silico flux distribution, involves identifying <br /> GPR associations inconsistency (as did         missing regulatory constraints genes/reactions by <br /> GNGs), automatically, using modified version         using GrowMatch procedure modified ver- <br /> of GrowMatch (see Methods). As example,             sion. We followed analysis allowing to <br /> modified GrowMatch procedure suggests GSL             simultaneous gene/reaction suppressions each <br /> (ASN1,ASN2) fixed suppressing gene                   GSL considered global modifications. For <br /> YML096W. By reviewing GPR relationships                 example, genes ZWF1 (YNL241C) RPE1 (YJL121C) <br /> found genes ASN1 (YPR145W) ASN2                       involved pentose-phosphate pathway and <br /> (YGR124W) YML096W provide isozymes               form GSL (under minimal medium). The deletion of <br /> for catalyzing essential reaction ASNS1 (asparagine        ZWF1 RPE1 change flux distribution <br /> synthase (glutamine-hydrolysing)). Therefore, suppres-         silico. Application GrowMatch fix this <br /> sing gene YML096W render simultaneous dele-           inconsistency suggests suppressing reaction G6PDH2er <br /> tion ASN1 ASN2 lethal. Note, however,           (glucose-6-phosphate dehydrogenase [endoplasmic reti- <br /> did evidence support YML096W sup-           culum]) identical reaction catalyzed by <br /> pression aerobic minimal conditions,              Zwf1p cytosol (i.e., G6PDH2). This suggests that <br /> found gene coding putative protein       glucose-6-phosphate dehydrogenase activity endo- <br /> unknown function assigned ASN1            plasmic reticulum sufficient compensate its <br /> based sequence similarity [37-39] prompting          deficiency cytosol RPE1 mutant background. <br /> to remove YML096W GPR association                  Interestingly, shown [43] mammalian <br /> ASN1. This example demonstrates identified            cells sets enzymes catalyzing reactions <br /> gene suppressions GrowMatch allude incor-            pentose phosphate pathway (including glucose-6- <br /> rect assignment genes reactions.                         phosphate dehydrogenase) active set in <br />   Resolution GSL inconsistencies coding             cytoplasm active endoplasmic <br /> blocked reactions complicated GNGs.           reticulum. If holds true yeast could <br /> Different cases need examined: (1) If genes         resolve inconsistency. An alternative hypothesis for <br /> appear GPR relationship blocked            fixing inconsistency ZWF1 independently <br /> reaction revert method discussed fix-       codes glucose-6-phosphate dehydrogenase activity in <br /> ing GNGs. (2) If genes coding (or          endoplasmic reticulum (i.e., reaction G6PDH2er) as <br /> more) different reactions blocked      cytosol (i.e., reaction G6PDH2). In another <br /> the root no-consumption metabolite explor-           example, FUR1 (YHR128W) URA3 (YEL021W) <br /> ing addition (common) downstream,            form GSL. They catalyze reactions serve as <br /> corresponding root problem metabolite, biomass          alternative production routes ump (uridine mono- <br /> equation reactant fix inconsistency. (3) If       phosphate) biomass precursor. Application the <br /> the genes coding reactions blocked        GrowMatch procedure fix GSL suggests repres- <br /> because different root no-consumption metabolites        sing reaction PYNP2r (pyrimidine-nucleoside phosphor- <br /> one genes codes non-blocked reaction         ylase (uracil)). According iMM904 model, <br /> we reconcile growth inconsistency simply         reaction PYNP2r provides source uri (uridine), <br /> modifying biomass reaction. To resolve GSLs,          subsequently converted ump. Suppression <br />  Zomorrodi Maranas BMC Systems Biology 2010, 4:178                                                                            Page 8 15 <br /> http://www.biomedcentral.com/1752-0509/4/178 <br />  <br />  <br />  <br />  <br /> of PYNP2r render simultaneous deletion                  PRS5)) contain genes (i.e., <br /> FUR1 URA3 lethal blocking production routes                   (PRS1,PRS2,PRS4) (PRS3,PRS2,PRS4)). All these <br /> for biomass precursor ump (see Figure 3). Note                  genes provide isozymes essential reaction PRPPS <br /> since PYNP2r does gene association                   (phosphoribosylpyrophosphate synthetase) according to <br /> model, suppression fix GSL inconsistency                 iMM904 model. The inconsistency model <br /> raises possibility reaction erroneously                 predictions experimental observations be <br /> included model removed.                        resolved changing GPR relationship (PRS1 <br />    Not surprisingly identified suppressions cor-             OR PRS2 OR PRS3 OR PRS4 OR PRS5) (PRS1 AND <br /> recting GSLs minimal medium valid                      PRS3) OR [(PRS2 OR PRS4) AND PRS5)] render the <br /> for YP medium. We identified case                      simultaneous deletion of aforementioned <br /> gene suppression resolve GSL, valid                   gene pairs/triple combinations lethal (see Figure 4). <br /> only YP medium. Genes MET12 (YPL023C)                            Interestingly, previous study strong interaction <br /> MET13 (YGL125W), identified GSL mini-                       PRS1 PRS3 PRS5 and <br /> mal YP media provide isozymes reaction                           PRS2 PRS4 described [45]. In total, <br /> MTHFR3 (5,10-methylenetetrahydrofolatereductase                          fixed GSLs minimal medium five <br /> (NADPH)), involved folate metabolism.                        YP medium converting SLSL consis- <br /> Application modified GrowMatch resolve                    tencies relying literature vetted global modifica- <br /> inconsistency suggests suppression CYS3 (YAL012W)                     tions (see Additional file 2 details). <br /> in minimal YP media. However, suppressing                       Resolution ESSL inconsistencies <br /> CYS3 minimal medium change                           The ESSL inconsistencies refer cases model <br /> GGs NGGs (i.e., conditional modification)                     predicts genes essential even <br /> whereas, global suppression YP medium.                    experimental results form a <br /> Interestingly, support CYS3 suppression YP                  SL. Based available data literature, identi- <br /> medium, reported gene cystein,                fied 13 ESSLs containing seven silico essential genes <br /> methionine glutathione suppressed [44].                              minimal medium ESSLs containing five <br />    Finally, cases GSL mismatches allude                       silico essential genes YP medium (see Addi- <br /> incorrect GPR relationships model. For example,                   tional file 1). Not surprisingly, silico essential <br /> we GSLs (under minimal medium) involved                       genes previously identified NGG mis- <br /> in GPR implicates genes PRS1 5. The                       matches single gene mutations. Consequently, a <br /> first include genes (i.e., (PRS1,PRS5) (PRS3,                direct resolution strategy fix ESSLs resolving <br />  <br />  <br />  <br />  <br />  Figure 3 Resolution GSL reaction suppressions. (A) FUR1 URA3 form GSL based iMM904 model exists <br />  alternative pathways producing biomass precursor ump absence genes. (B) By suppressing (or removing) reaction <br />  PYNP2r, production routes biomass precursor ump blocked, rendering simultaneous deletion FUR1 URA3 lethal. <br />  Zomorrodi Maranas BMC Systems Biology 2010, 4:178                                                                     Page 9 15 <br /> http://www.biomedcentral.com/1752-0509/4/178 <br />  <br />  <br />  <br />  <br />                                                                          studies demonstrated OPI3 par- <br />                                                                          tially contribute isozyme catalyzing reaction <br />                                                                          PETOHM_SC [46] implying GPR reac- <br />                                                                          tion changed (CHO2 OR OPI3). There <br />                                                                          reports implicating CHO2 isozyme for <br />                                                                          catalyzing reactions MFAPS_SC PMETM_SC. How- <br />                                                                          ever, previous studies reported S. cerevisiae is <br />                                                                          flexible respect phospholipid composition and <br />                                                                          substitute pe_SC, ptdmeeta_SC (phosphatidyl- <br />                                                                          monomethylethanolamine) ptd2meeta_SC (phosphati- <br />                                                                          dyl-dimethylethanolamine) pc_SC substantial <br />                                                                          extent [46-49]. In <span id='am-123' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-124' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> capture lack specificity <br />                                                                          removed pc_SC biomass reaction and <br />                                                                          instead added proxy phospholipid compound the <br />                                                                          stoichiometric coefficient pc_SC. Subse- <br />                                                                          quently, added hypothetical reactions the <br />                                                                          model produce phospholipid of <br />                                                                          pc_SC, pe_SC, ptdmeeta_SC ptd2meeta_SC. Note <br />                                                                          modifications model global <br />                                                                          fixed inconsistencies OPI3 NGG well <br />                                                                          (CHO2, OPI3) ESSL mismatch. <br />                                                                          Resolution SLG inconsistencies <br />                                                                          The SLG inconsistencies represent mismatch where <br />                                                                          genes form silico SL, however, simul- <br />  Figure 4 Change GPR reaction PRPPS. The GSL                      taneous deletion results viable strain. SLG mis- <br />  inconsistencies (PRS1,PRS5), (PRS3,PRS5), (PRS1,PRS2,PRS4) (PRS3, <br />                                                                          matches double gene perturbations viewed as <br />  PRS2,PRS4) imply GPR reaction PRPPS be <br />  changed (PRS1 OR PRS2 OR PRS3 OR PRS4 OR PRS5) (PRS1            NGG mismatches single gene perturbations the <br />  AND PRS3) OR [(PRS2 OR PRS4) AND PRS5)].                                sense imply certain functionalities <br />                                                                          missing model. The difference between <br />                                                                          NGGs SLGs &#8220;no-growth&#65533;? NGGs is <br /> the NGGs single gene perturbations. Following                    essentiality single genes SLGs <br /> this route, ESSLs minimal                     result synthetic lethality gene pairs. There- <br /> medium YP medium automatically                       fore, simply adapt procedure used <br /> rectified converted SLSLs resolution                  fixing NGGs resolve SLG inconsistencies. <br /> of NGGs (see Additional file 2 details). However,                   We resolved case inconsistency <br /> rest resolved, instead con-                   minimal YP media. This SLG pertains genes <br /> verted new GSL mismatches. Except case,                     PGM1 (YKL127W) PGM2 (YMR105C) code for <br /> application methods discussed previous                     isozymes reaction PGMT (phosphoglucomutase) <br /> section resolving GSL mismatches did identify                    involved glycolysis/gluconeogenesis. The simultaneous <br /> any global literature supported correction strategies.                deletion PGM1 PGM2 lethal based the <br />   An interesting example ESSL mismatches                       iMM904 model glygogen, biomass precursor, can- <br /> gene pair CHO2 (YGR157W) OPI3 (YJR073C)                              produced absence genes. Appli- <br /> where gene essential vivo [4]. CHO2 catalyzes                 cation GrowMatch procedure fix SLG, did not <br /> the step conversion pe_SC (phosphatidyletha-                 lead literature-supported correction strategy. We <br /> nolamine) pc_SC (phosphatidylcholine) phospholi-                   mined literature possible isozyme this <br /> pid biosynthesis pathway (reaction PETOHM_SC                      reaction identified gene PGM3 (YMR278W) is <br /> iMM904 model), whereas, OPI3 catalyzes                      known catalyze interconversion glucose-1-pho- <br /> steps (reactions MFAPS_SC PMETM_SC). We                         phate glucose-6-phosphate [50], however, missing in <br /> not able fix CHO2 OPI3 separate                        iMM904 model. The addition PGM3 GPR for <br /> NGGs methods discussed previously.                         reaction PGMT (i.e., PGM1 OR PGM2 OR PGM3) renders <br /> However, fact genes form SL                      simultaneous deletion PGM1 PGM2 non-lethal. <br /> vivo suggests act isozymes reactions                Resolution SLES inconsistencies <br /> PETOHM_SC, MFAPS_SC PMETM_SC. Interest-                               The SLES mismatches denote cases model <br /> ingly, mining literature previous                   predicts simultaneous deletion both <br />  Zomorrodi Maranas BMC Systems Biology 2010, 4:178                                                       Page 10 15 <br /> http://www.biomedcentral.com/1752-0509/4/178 <br />  <br />  <br />  <br />  <br /> genes lethal genes (or both)        Auxotrophy inconsistencies <br /> essential vivo. This implies deletion      These mismatches refer cases essentiality <br /> these genes compensated             single gene deletions synthetic lethality double <br /> other gene experimental conditions. There-         gene knockouts agreements vivo observa- <br /> fore, SLES inconsistencies rectified suppressing      tions, however, model predictions supplementa- <br /> in model gene essential vivo.         tion rescue (i.e., auxotrophy) scenarios inconsistent <br /> Essential genes participating in silico SLs yield GNG     experimental data. We seven these <br /> inconsistencies. Therefore, resolution          inconsistencies minimal YP med- <br /> GNGs fixes SLES mismatches double           ium, respectively, correctly predicted essential genes, <br /> gene mutations. We identified 59 SLES mismatches         minimal YP media for <br /> the minimal medium 42 YP medium. Fifty           SLSL predictions (see Additional file 1 complete <br /> one SLESs minimal YP          list). Notably, mismatches, experi- <br /> medium fixed using global modifications          mental results single double gene <br /> found GNGs. For example, gene IPP1 (YBR011C)             mutant strains restore growth additional com- <br /> which codes PPA (inorganic diphosphatase                 pounds added growth medium, the <br /> involved oxidative phosphorylation) forms SL pair       model predictions imply genes remain essen- <br /> under minimal YP medium gene IPP2              tial synthetic lethal presence these <br /> (YMR267W) codes reaction mito-          compounds. These inconsistencies treated in <br /> chondria (reaction PPAm). However, IPP1             exactly way NGG SLG mismatches <br /> found essential vivo [4]. Therefore, resolve     treated, refer functionalities that <br /> this inconsistency, conditionally suppress gene IPP2      missing model present experimen- <br /> under aerobic conditions. Interestingly, investigating    tal conditions. <br /> the available expression data genes          As example, reported strain con- <br /> that expression level IPP1 aerobic condi-       taining FOL1 (YNL256W) deletion grow the <br /> tions glucose carbon source 37         medium supplemented folic acid [54]. Nonethe- <br /> times higher IPP2 [51],              less, folic acid included list metabolites <br /> explain IPP2 able compensate           iMM904 model. By adding folic acid as <br /> deletion IPP1.                                            exchange transport reactions (between cytosol and <br />   In example, gene PGK1 (YCR012W),          extracellular environment) iMM904 model FOL1 <br /> associated reaction PGK (phosphoglycerate kinase)       remains essential folic acid allowed be <br /> involved glycolysis/gluconeogenesis, participates   taken up. GrowMatch suggested addition of the <br /> many 39 silico SLs minimal medium. How-          reactions R00937 (5,6,7,8-tetrahydrofolate:NAD+ <br /> ever, essential vivo [4]. This    oxidoreductase) R02236 (dihydrofolate:NADP+ oxi- <br /> implies suppressing 39 genes        doreductase) KEGG database model so <br /> forming synthetic lethal PGK1 resolve       connect folic acid rest network. Interest- <br /> these SLESs. Although, did evidence          ingly, searching KEGG database that <br /> confirming resolution hypothesis       enzyme catalyzing reactions present in <br /> genes PCK1 (YKR097W), known           yeast gene coding enzyme (DFR1: <br /> be suppressed presence glucose [52,53]: gene      YOR236W) present iMM904 model. <br /> is involved gluconeogenesis, process allowing yeast        The gene pair HMG1 (YML075C) HMG2 <br /> to synthesize glucose non-carbohydrate precursors       (YLR450W), forms SLSL minimal <br /> such ethanol glycerol. Notably, suppression         YP media. example auxotrophy mismatches <br /> PCK1 iMM904 model block production            double gene perturbations: vivo observa- <br /> three biomass precursors, i.e., phe-L (L-Phenylalanine),     tions mutant strain lacking genes <br /> trp-L (L-Tryptophan) tyr-L (L-Tyrosine)           rescued addition mev-R (mevalonate) <br /> absence PGK1.                                             growth medium [55], silico predictions imply <br />   Overall, resolution model inconsistencies       double deletion lethal pre- <br /> double gene deletions improved specificity            sence mev-R. The addition exchange trans- <br /> iMM904 model 12.03% 23.31% (out total        port reaction cytosol extracellular <br /> 133) minimal medium 6.96% 13.04%         environment mev-R model resolves auxo- <br /> (out total 115) YP medium. It worth       trophic inconsistency. Notably, addition this <br /> noting corrections based          import pathway mev-R model fixed <br /> incomplete list SL data available literature.          auxotrophy inconsistency essential gene <br />  Zomorrodi Maranas BMC Systems Biology 2010, 4:178                                                                    Page 11 15 <br /> http://www.biomedcentral.com/1752-0509/4/178 <br />  <br />  <br />  <br />  <br />                                                                         cases fixing mismatch double gene deletions <br />                                                                         automatically fixes mismatch(s) single <br />                                                                         gene perturbations. This illustrated NGG, <br />                                                                         CHO2 mechanisms proposed to <br />                                                                         resolve NGGs successful. Approxi- <br />                                                                         mately, 20% total suggested corrections the <br />                                                                         iMM904 model use information single and <br />                                                                         double gene perturbations 17% were <br />                                                                         exclusively discerned double gene perturbations. A <br />                                                                         far larger contribution synthetic lethals providing <br />                                                                         model refinement strategies expected more <br />                                                                         synthetic lethality data available. <br />                                                                           The high number GNG GSL inconsistencies <br />                                                                         identified study runs contrary general per- <br />                                                                         ception predictive inaccuracy genome-scale <br />                                                                         metabolic models primarily missing metabolic <br />                                                                         capabilities. It appears presence properly <br />                                                                         restricted specific conditions functionalities the <br />                                                                         model largest contributor inconsistent predic- <br />  Figure 5 A summary modifications iMM904 model <br />  using single double gene perturbations. (A) Venn <br />                                                                         tions. Application GrowMatch eliminate and/or <br />  diagrams minimal YP media representing number           properly regulate functionalities led identifi- <br />  modifications involved fixing inconsistencies associated   cation 30 growth medium-specific regulatory con- <br />  single double gene deletions both. (B) A Bar chart               straints. These growth prediction inconsistency-based <br />  demonstrating number type modifications         constraints complement existing regulatory constraints <br />  iMM904 model. <br />                                                                         based gene expression data [44,56,57]. <br />                                                                           In study considered essentiality and <br />                                                                         synthetic lethality predictions disagreements in <br /> ERG10 (YPL028W), involved mevalonate                        auxotrophy complementation. We demonstrated that <br /> biosynthesis.                                                           identified growth-phenotype discrepancies some- <br />   Overall, including global modifications                 times incorrect incomplete silico descrip- <br /> for supporting evidence found, fix                 tion complex growth medium inaccuracy of <br /> three auxotrophy inconsistencies essential genes                 metabolic model. Overall, significantly improved <br /> well SLs minimal YP media                     predictive capability (i.e., specificity, sensitivity and <br /> (see Additional file 2 details). A summary               false viability rate) iMM904 model essentials <br /> suggested modifications iMM904 model using                   synthetic lethals incorporating minimum 112 <br /> all types inconsistencies single double gene                 (out 120) suggested corrections. All modifica- <br /> perturbations given Figure 5. The revised iMM904                  tions global invalidate correct <br /> model available Systems Biology Markup                   model predictions. The proposed corrections span wide <br /> Language (SBML) Additional file 3.                                   array changes model including relaxation the <br />                                                                         irreversibility constraints existing reactions the <br /> Conclusions                                                             model, adding new reactions, compounds genes to <br /> We identified 120 corrections supporting evidence                  model, modifying biomass equation, changing the <br /> to iMM904 metabolic model yeast using                         GPR associations medium-specific regulatory con- <br /> essentiality synthetic lethality data. Previous studies             straints. As independent corroborating evidence <br /> geared improving predictive ability meta-                proposed corrections, vast majority (i.e., 103) of <br /> bolic models used growth phenotype inconsisten-                    remain relevant latest update the <br /> cies single gene mutation experiments microbial                  community yeast model (Yeast 4.0) [8]. This includes 73 <br /> systems E. coli [18,19]. Here, step                     model refinement strategies 30 identified med- <br /> further demonstrating utility synthetic lethality             ium-specific regulatory constraints. Of remaining <br /> data improving accuracy multi-compartment                  modifications iMM904 model, also <br /> metabolic model eukaryotic organism. This                         independently incorporated Yeast 4.0 are <br /> revealed missing erroneously present metabolic func-                 different ones adopted Yeast 4.0. A compari- <br /> tions model captured                   son suggested corrections Yeast 4.0 given in <br /> single gene perturbations. In addition,                Additional file 2. <br />  Zomorrodi Maranas BMC Systems Biology 2010, 4:178                                                            Page 12 15 <br /> http://www.biomedcentral.com/1752-0509/4/178 <br />  <br />  <br />  <br />  <br />   In addition model refinement strategies sup-       [6]. The uptake rate source metabolites was <br /> porting evidence, identified 60 glo-      set 1000 mmol gDW -1 h -1 . The flux non- <br /> bal modifications conflicting    growth associated ATP maintenance fixed 1 <br /> nor supporting evidence (see Additional file 2         mmol gDW-1h-1 [6]. We employed GrowMatch pro- <br /> complete list). These modifications treated        cedure [18] reconcile growth phenotype discre- <br /> testable hypotheses experiments             pancies NGGs, modified fix GNGs (in <br /> designed prove/disprove validity. Overall,      addition using original form). All algorithms <br /> study demonstrates value bringing bear multi-      adapted resolving inconsistencies associated <br /> gene deletion data improve predictive         double gene perturbations. In following we <br /> capability genome-scale metabolic models. The avail-      provide brief overview GrowMatch in <br /> ability high-throughput experimental techniques           modified version. <br /> [58-61] efficient computational tools <br /> [25,62,63] elucidate synthetic lethal interactions        The Modified GrowMatch procedure <br /> opens door rapidly reveal additional model defi-      The GrowMatch procedure relies separate pro- <br /> ciencies. The model refinement approaches presented       cedures resolve NGG GNG growth prediction <br /> this study versatile employed         inconsistencies. The NGGs fixed one-by-one by <br /> wider range experimental conditions (e.g.,          minimally perturbing original metabolic model using <br /> growth media) synthetic lethal interactions increas-   mechanisms including (i) relaxation irrever- <br /> ing size (e.g., triples, quadruples, etc).                   sibility constraints reactions model, (ii) addi- <br />                                                              tion new reactions external databases as <br /> Methods                                                      KEGG [31] model (iii) allowing direct <br /> We applied SL Finder procedure developed              import/export metabolites into/out cell, and <br /> Suthers et al [25] iMM904 model S. cerevisiae      multi-compartment models [18], addition trans- <br /> [6] comprised 904 genes 1,412 reactions deter-     port reactions compartments cytosol. <br /> mine set essential genes/reactions dif-    Alternatively, GNG mismatches corrected by <br /> ferent orders synthetic lethals. All simulations     identifying minimal set suppression constraints <br /> performed aerobic growth different media          reactions transport mechanisms lower the <br /> with glucose sole carbon source: minimal medium       maximum biomass yield network pre- <br /> and complex (yeast-extract peptone YP) medium.          specified viability threshold. The suggested modifications <br /> The silico minimal medium contains ammonium, sul-         GrowMatch referred global not <br /> fate phosphate nitrogen, sulfur phosphor          conflict correct model predictions, they <br /> sources, respectively, necessary salts (such   called conditional otherwise. <br /> Na K). In addition components          Here, modified GrowMatch procedure iden- <br /> silico YP medium contains 20 amino acids         tify minimal number suppressed genes, rather <br /> four nucleotides [3]. All simulations performed     reactions, negate biomass formation below <br /> a strain auxotrophic methionine, leucine, histidine      viability threshold. This defining two <br /> and uracil closely mimic conditions used           sets binary variables genes reac- <br /> experimental studies [4]. This auxotrophy simulated      tions relating binary variables in <br /> by deleting genes (his3&#916; leu2&#916; met15&#916; ura3&#916;) sup-        way reflects specific GPR associations <br /> plementation growth medium missing           [25]. For reactions gene associations ficti- <br /> nutrients non-limiting low levels [6]. In addition,   tious gene coding reaction assumed. An <br /> trace amounts essential compounds          alternative way avoid introducing new set binary <br /> present experimental growth medium including          variables, result reduce computational bur- <br /> 4-aminobenzoate, biotin, inositol, nicotinate, panthothe-    den further, directly correlate binary variables <br /> nate thiamin added silico media [6].      genes reaction fluxes. To end, first, <br /> Consistent Suthers et al [25] chose 1%        define following sets: <br /> maximum theoretical biomass yield viability <br /> threshold computational identification lethal            I = {i | = 1, 2,..., N} = set metabolites <br /> knockouts. The upperbound reactions set          J = { j | j = 1, 2,..., M} = set reactions <br /> 1000, whereas, lowerbound set 0 irreversi-       K = {k | k = 1, 2,..., G} = set genes <br /> ble reactions -1000 reversible reactions. The <br /> maximum rate glucose uptake set 10               where, N, M G denote total number meta- <br /> mmol gDW-1h-1 aerobic condition modeled          bolites, reactions genes network, respectively. <br /> by limiting oxygen uptake rate 2 mmol gDW-1h-1        To simulate gene knockouts, binary variable wk, <br />  Zomorrodi Maranas BMC Systems Biology 2010, 4:178                                                                             Page 13 15 <br /> http://www.biomedcentral.com/1752-0509/4/178 <br />  <br />  <br />  <br />  <br /> representing gene k deleted defined         These cases mathematically enforced an <br /> following:                                                       appropriate combination equations (3) (4). As an <br />                                                                  example, genes k1, k2 k 3 correlated as <br />          &#9127; 0, gene k deleted                               (k1 AND k2 ) OR (k1 AND k3 ) code enzyme <br />     wk = &#9128;                                      &#8704; k&#8712;K      (1) <br />          &#9129; 1, gene k active                                catalyzing reaction j, constraints simulating <br />                                                                  relationship written following: <br />   The impact gene knockouts reactions through <br /> the GPR relationships mathematically described               &#9127;&#9130; w k1 .LB j &#8804; v j &#8804; w k1 .UB j <br /> by using appropriate constraints relating wk reac-            &#9128;                                                                     (5) <br /> tion fluxes, v j . Let LB j UB j represent lower-            &#9130;&#9129; (w k 2 + w k3 ).LB j &#8804; v j &#8804; (w k 2 + w k3 ).UB j <br /> bound upperbound reaction j, respectively. <br />                                                                   The modified GrowMatch formulation resolve the <br /> Different cases GPR relationships be <br />                                                                  GNGs (or GSLs) formulated following: <br /> considered: <br />   (i) A single gene k codes enzyme catalyzing             Minimi ze            v biomass <br /> reaction j. This easily incorporated                        Wj <br /> mathematical framework using following con-                  s.t . <br /> straint: <br />                                                                             &#9121; Maximize v biomass                         &#9124; <br />                                                                             &#9122;       vj                                   &#9125; <br />     w k .LB j &#8804; v j &#8804; w k .UB j                            (2)              &#9122; s.t .                                      &#9125; <br />                                                                             &#9122;                                            &#9125; <br />   (ii) Two genes k1 k2 form single multi-protein <br /> enzyme enable reaction j. This case, recast <br />                                                                             &#9122; <br />                                                                             &#9122;    &#8721;       s ij v j = 0            &#8704;i&#8712;I &#9125; <br />                                                                                                                          &#9125; <br />                                                                             &#9122;       j                                    &#9125; <br /> as logic AND relationship genes k1                        &#9122; Appropriate GRP eqns &#8704; j &#8712; J &#9125; <br /> k2, enforced following set constraints:                    &#9122;                                            &#9125; <br />                                                                             &#9122; v glu cos e &#8804; v glu   uptake limit <br />                                                                                                        cos e             &#9125; <br />                                                                             &#9122;                                            &#9125; <br />     &#9130;&#9127; w k1 .LB j &#8804; v j &#8804; w k1 .UB j                                        &#9122; v oxygen &#8804; v oxygen  uptake limit          &#9125; <br />      &#9128;                                                     (3)              &#9122;                                            &#9125; <br />      &#9130;&#9129; w k 2 .LB j &#8804; v j &#8804; w k 2 .UB j                                     &#9122; v ATPM = v ATPM <br />                                                                                                    maintenance <br />                                                                                                                          &#9125; <br />                                                                             &#9122; LB &#8804; v &#8804; .UB                       &#8704; j &#8712; J &#9125; <br />   Note based equations,                    &#9123;          j         j       j               &#9126; <br /> w k1 w k 2 zero, flux reaction j                  &#8721;(1 &#8722; w <br />                                                                              j <br />                                                                                          k   )&#8804;n <br /> forced zero. This set constraints easily gen- <br /> eralized multi-protein enzymes                       w k &#8712; { 0, 1}                          &#8704; k&#8712;K <br /> genes involved. <br />   (iii) The genes k1 k2 provide isozymes catalyzing          where, sij, represents stoichiometric coefficient of <br /> reaction j. This case, indicates logic OR relationship         metabolite i, reaction j, vbiomass denotes bio- <br /> between genes k1 k2 enable reaction j,            mass        flux              v uptake   limit <br />                                                                                                                       ,     uptake limit <br />                                                                                                                           v oxygen         and <br />                                                                                                        glu cos e <br /> and mathematically expressed following <br /> constraints:                                                      v maintenance <br />                                                                     ATPM        denote minimum required glucose and <br />                                                                  oxygen uptake rates non-growth associated ATP <br />     &#9127;&#9130; (w k1 + w k 2 ).LB j &#8804; v j &#8804; (w k1 + w k 2 ).UB j         maintenance, respectively. The parameter n, repre- <br />      &#9128;                                                     (4) <br />      &#9129;&#9130; LB j &#8804; v j &#8804; .UB j <br />                                                                  sents allowable number knock-outs. This bilevel <br />                                                                  optimization problem solved similarly Grow- <br />   Note according equations, vj forced        Match SL Finder [18,25] writing dual <br /> zero genes knocked out, i.e., w k1 =            inner problem. <br />  w k 2 = 0. For case, genes present <br /> ( w k1 = w k 2 = 1), second constraint bind-         Test suggested hypotheses <br /> ing restrict vj fall defined lower        All model correction strategies provided Grow- <br /> and upperbound. This approach readily general-            Match serve hypotheses need tested to <br /> ized GPRs containing genes related             confirm applicability. Different methods used <br /> with OR.                                                         test validity type modification. Similarly <br />   (iv) More genes combination AND             [18], relaxation irreversibility constraints on <br /> and OR relationships required enable reaction j.        existing reactions model checked using <br />  Zomorrodi Maranas BMC Systems Biology 2010, 4:178                                                                                            Page 14 15 <br /> http://www.biomedcentral.com/1752-0509/4/178 <br />  <br />  <br />  <br />  <br /> three independent methods. In step check                          2.    Forster J, Famili I, Fu P, Palsson BO, Nielsen J: Genome-scale reconstruction <br />                                                                                      Saccharomyces cerevisiae metabolic network. Genome Res 2003, <br /> the reversibility reaction iAF1260 model E.                            13(2):244-253. <br /> coli [9] metabolic models organisms. We                            3.    Duarte NC, Herrgard MJ, Palsson BO: Reconstruction validation of <br /> then query databases MetaCyc [64]                                Saccharomyces cerevisiae iND750, fully compartmentalized genome- <br />                                                                                      scale metabolic model. Genome Res 2004, 14(7):1298-1309. <br /> the reversibility desired reaction                 4.    Kuepfer L, Sauer U, Blank LM: Metabolic functions duplicate genes in <br /> reversible organisms. Finally, examine                               Saccharomyces cerevisiae. Genome Res 2005, 15(10):1421-1430. <br /> reversibility reactions computing value free                      5.    Nookaew I, Jewett MC, Meechai A, Thammarongtham C, Laoteng K, <br />                                                                                      Cheevadhanarak S, Nielsen J, Bhumiratana S: The genome-scale metabolic <br /> Gibbs energy change, &#916;G [65].                                                        model iIN800 Saccharomyces cerevisiae validation: scaffold <br />    The validity added transport reactions                              query lipid metabolism. BMC Syst Biol 2008, 2:71. <br /> model examined searching literature                         6.    Mo ML, Palsson BO, Herrgard MJ: Connecting extracellular metabolomic <br />                                                                                      measurements intracellular flux states yeast. BMC Syst Biol 2009, <br /> potential clues presence suggested                                  3:37. <br /> transport mechanisms querying databases                              7.    Herrgard MJ, Swainston N, Dobson P, Dunn WB, Arga KY, Arvas M, <br /> such MetaCyc possibility presence                                 Bluthgen N, Borger S, Costenoble R, Heinemann M, et al: A consensus <br />                                                                                      yeast metabolic network reconstruction obtained community <br /> mechanisms multi-<span id='am-245' about='obo:NCBITaxon_131567' typeof='owl:Thing obo:NCBITaxon_1'><span id='am-246' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-247' property="oboInOwl:hasRelatedSynonym" content="biota" datatype="xsd:string"></span><span id='am-248' property="rdfs:label" content="cellular organisms" datatype="xsd:string"></span><span id='am-249' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cellular organisms</span>. The                                    approach systems biology. Nat Biotechnol 2008, 26(10):1155-1160. <br /> hypotheses adding new reactions external data-                        8.    Dobson PD, Smallbone K, Jameson D, Simeonidis E, Lanthaler K, Pir P, Lu C, <br /> bases KEGG model tested per-                                   Swainston N, Dunn WB, Fisher P, et al: Further developments a <br />                                                                                      genome-scale metabolic model yeast. BMC Syst Biol 2010, 4:145. <br /> forming bi-directional BLAST enzymes                           9.    Feist AM, Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD, <br /> catalyzing reactions yeast genome. Consis-                             Broadbelt LJ, Hatzimanikatis V, Palsson BO: A genome-scale metabolic <br /> tent [18,20] assumed BLAST expectation                                     reconstruction <span id='am-197' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-198' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-199' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-201'  ></span><span id='am-202' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-203' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-205' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-206' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-207' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-208' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-209' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-210' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-211' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span>Escherichia coli</span> K-12 MG1655 accounts 1260 <br />                                                                                      ORFs thermodynamic information. Mol Syst Biol 2007, 3:121. <br /> value cutoff 10-13 basis define high sequence                     10.   Suthers PF, Dasika MS, Kumar VS, Denisov G, Glass JI, Maranas CD: A <br /> similarity. Finally, gene suppressions validated                          genome-scale metabolic reconstruction <span id='am-58' about='obo:NCBITaxon_2097' typeof='owl:Thing obo:NCBITaxon_2093 obo:NCBITaxon_2092 obo:NCBITaxon_1 obo:NCBITaxon_2085 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_544448 obo:NCBITaxon_31969'><span id='am-59'  ></span><span id='am-60' property="rdfs:label" content="Mycoplasma genitalium" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:4" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Mycoplasma genitalium</span>, <br /> analysis gene expression data searching                            iPS189. PLoS Comput Biol 2009, 5(2):e1000285. <br />                                                                                11.   Henry CS, Zinner JF, Cohoon MP, Stevens RL: iBsu1103: new genome- <br /> the literature available evidence. All global modi-                          scale metabolic model Bacillus subtilis based SEED annotations. <br /> fications did supporting evi-                              Genome Biol 2009, 10(6):R69. <br /> dence using methods mentioned above,                              12.   Puchalka J, Oberhardt MA, Godinho M, Bielecka A, Regenhardt D, <br />                                                                                      Timmis KN, Papin JA, Martins dos Santos VA: Genome-scale reconstruction <br /> incorporated model just added                               analysis <span id='am-181' about='obo:NCBITaxon_286' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621'><span id='am-182' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-183' property="oboInOwl:hasRelatedSynonym" content="Liquidomonas" datatype="xsd:string"></span><span id='am-184' property="oboInOwl:hasDbXref" content="PMID:9103607" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasRelatedSynonym" content="Loefflerella" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasDbXref" content="PMID:7727274" datatype="xsd:string"></span><span id='am-187'  ></span><span id='am-188' property="oboInOwl:hasDbXref" content="PMID:23918787" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasDbXref" content="PMID:15950132" datatype="xsd:string"></span><span id='am-191' property="rdfs:label" content="Pseudomonas" datatype="xsd:string"></span><span id='am-192' property="oboInOwl:hasDbXref" content="PMID:10758879" datatype="xsd:string"></span><span id='am-193' property="oboInOwl:hasRelatedSynonym" content="Peudomonas" datatype="xsd:string"></span><span id='am-194' property="oboInOwl:hasDbXref" content="PMID:10939664" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasRelatedSynonym" content="RNA similarity group I" datatype="xsd:string"></span><span id='am-196' property="oboInOwl:hasDbXref" content="PMID:18048745" datatype="xsd:string"></span>Pseudomonas</span> putida KT2440 metabolic network <br /> list modifications corroborating evidence.                                facilitates applications biotechnology. PLoS Comput Biol 2008, 4(10): <br />                                                                                      e1000210. <br />                                                                                13.   Thiele I, Vo TD, Price ND, Palsson BO: Expanded metabolic reconstruction <br /> Additional material                                                                  <span id='am-171' about='obo:NCBITaxon_209' typeof='owl:Thing obo:NCBITaxon_72293 obo:NCBITaxon_213849 obo:NCBITaxon_1 obo:NCBITaxon_68525 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_29547'><span id='am-172' property="oboInOwl:hasDbXref" content="PMID:17329766" datatype="xsd:string"></span><span id='am-173' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-174' property="oboInOwl:hasDbXref" content="PMID:11156001" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasDbXref" content="PMID:15143020" datatype="xsd:string"></span><span id='am-176' property="rdfs:label" content="Helicobacter" datatype="xsd:string"></span><span id='am-177' property="oboInOwl:hasAlternativeId" content="NCBITaxon:28203" datatype="xsd:string"></span><span id='am-178'  ></span><span id='am-179' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasDbXref" content="PMID:1704793" datatype="xsd:string"></span>Helicobacter</span> pylori (iIT341 GSM/GPR): silico genome-scale <br />                                                                                      characterization single- double-deletion mutants. J Bacteriol 2005, <br />  Additional file 1: The list model inconsistencies. The complete list             187(16):5818-5830. <br />  different types model inconsistencies single double gene        14.   AbuOun M, Suthers PF, Jones GI, Carter BR, Saunders MP, Maranas CD, <br />  perturbations literature citations.                                      Woodward MJ, Anjum MF: Genome scale reconstruction Salmonella <br />                                                                                      metabolic model: comparison similarity differences a <br />  Additional file 2: The list corrections model. The complete <br />                                                                                      commensal <span id='am-213' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-214'  ></span><span id='am-215' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-216' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-217' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-218' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-219' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia</span> coli strain. J Biol Chem 2009, 284(43):29480-29488. <br />  list suggested corrections mismatches associated single and <br />                                                                                15.   Thiele I, Palsson BO: A protocol generating high-quality genome- <br />  double gene perturbations supporting evidence. This file also <br />                                                                                      scale metabolic reconstruction. Nat Protoc 2009, 5(1):93-121. <br />  contains details BLAST analysis different cases other <br />                                                                                16.   Ibarra RU, Edwards JS, Palsson BO: <span id='am-262' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-263' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-264' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span><span id='am-265' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-266' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia coli K-12</span> undergoes <br />  computations. <br />                                                                                      adaptive evolution achieve silico predicted optimal growth. Nature <br />  Additional file 3: The revised yeast model SBML format.                          2002, 420(6912):186-189. <br />                                                                                17.   Joyce AR, Reed JL, White A, Edwards R, Osterman A, Baba T, Mori H, <br />                                                                                      Lesely SA, Palsson BO, Agarwalla S: Experimental computational <br />                                                                                      assessment conditionally essential genes Escherichia coli. J Bacteriol <br /> Acknowledgements                                                                     2006, 188(23):8259-8271. <br /> This work funded DOE grant DE-FG02-05ER25684         18.   Kumar VS, Maranas CD: GrowMatch: automated method for <br /> Network Science Initiative The Pennsylvania State University. The authors         reconciling silico/in vivo growth predictions. PLoS Comput Biol 2009, <br /> would like thank Dr. Anthony P. Burgard critical reading               5(3):e1000308. <br /> manuscript insightful suggestions, Dr. Patrick F. Suthers useful   19.   Reed JL, Patel TR, Chen KH, Joyce AR, Applebee MK, Herring CD, Bui OT, <br /> discussions.                                                                         Knight EM, Fong SS, Palsson BO: Systems approach refining genome <br />                                                                                      annotation. Proc Natl Acad Sci USA 2006, 103(46):17480-17484. <br /> Authors&#8217; contributions                                                         20.   Satish Kumar V, Dasika MS, Maranas CD: Optimization based automated <br /> CDM conceived coordinated study, participated design              curation metabolic reconstructions. BMC Bioinformatics 2007, 8:212. <br /> helped draft manuscript. ARZ performed simulations data         21.   Orth JD, Palsson BO: Systematizing generation missing metabolic <br /> analysis drafted manuscript. Both authors read approved              knowledge. Biotechnol Bioeng 2010, 107(3):403-412. <br /> final manuscript.                                                              22.   Motter AE, Gulbahce N, Almaas E, Barabasi AL: Predicting synthetic rescues <br />                                                                                      metabolic networks. Mol Syst Biol 2008, 4:168. <br /> Received: 16 September 2010 Accepted: 29 December 2010                         23.   Nishikawa T, Gulbahce N, Motter AE: Spontaneous reaction silencing in <br /> Published: 29 December 2010                                                          metabolic optimization. PLoS Comput Biol 2008, 4(12):e1000236. <br />                                                                                24.   Harrison R, Papp B, Pal C, Oliver SG, Delneri D: Plasticity genetic <br />                                                                                      interactions metabolic networks yeast. Proc Natl Acad Sci USA 2007, <br /> References                                                                           104(7):2307-2312. <br /> 1. Goffeau A: The yeast genome directory. Nature 1997, 387:5-6. <br />  Zomorrodi Maranas BMC Systems Biology 2010, 4:178                                                                                                 Page 15 15 <br /> http://www.biomedcentral.com/1752-0509/4/178 <br />  <br />  <br />  <br />  <br /> 25. Suthers PF, Zomorrodi A, Maranas CD: Genome-scale gene/reaction                   47. Kodaki T, Yamashita S: Yeast phosphatidylethanolamine methylation <br />     essentiality synthetic lethality analysis. Mol Syst Biol 2009, 5:301.             pathway. Cloning characterization distinct methyltransferase <br /> 26. Forster J, Famili I, Palsson BO, Nielsen J: Large-scale evaluation silico       genes. J Biol Chem 1987, 262(32):15428-15435. <br />     gene deletions Saccharomyces cerevisiae. OMICS 2003, 7(2):193-202.             48. Kodaki T, Yamashita S: Characterization methyltransferases the <br /> 27. Flaks JG, Erwin MJ, Buchanan JM: Biosynthesis purines. XVI. The                yeast phosphatidylethanolamine methylation pathway selective gene <br />     synthesis adenosine 5&#8217;-phosphate 5-amino-4-                                    disruption. Eur J Biochem 1989, 185(2):243-251. <br />     imidazolecarboxamide ribotide nucleotide pyrophosphorylase. J Biol           49. Kanipes MI, Henry SA: The phospholipid methyltransferases yeast. <br />     Chem 1957, 228(1):201-213.                                                            Biochim Biophys Acta 1997, 1348(1-2):134-141. <br /> 28. Shi W, Sarver AE, Wang CC, Tanaka KS, Almo SC, Schramm VL: Closed site            50. Tiwari A, Bhat JP: Molecular characterization reveals YMR278w <br />     complexes adenine phosphoribosyltransferase <span id='am-100' about='obo:NCBITaxon_5740' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_68459 obo:NCBITaxon_207245 obo:NCBITaxon_131567 obo:NCBITaxon_5738 obo:NCBITaxon_5739 obo:NCBITaxon_2759'><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-102'  ></span><span id='am-103' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-104' property="rdfs:label" content="Giardia" datatype="xsd:string"></span>Giardia</span> lamblia                   encoded protein environmental stress response homologue of <br />     reveal mechanism ribosyl migration. J Biol Chem 2002,                            Saccharomyces cerevisiae PGM2. Biochem Biophys Res Commun 2008, <br />     277(42):39981-39988.                                                                  366(2):340-345. <br /> 29. Smithers GW, O&#8217;Sullivan WJ: Hypoxanthine phosphoribosyltransferase:               51. Holstege FC, Jennings EG, Wyrick JJ, Lee TI, Hengartner CJ, Green MR, <br />     radiochemical assay procedures forward reverse reactions.                 Golub TR, Lander ES, Young RA: Dissecting regulatory circuitry a <br />     Anal Biochem 1985, 145(1):14-20.                                                      eukaryotic genome. Cell 1998, 95(5):717-728. <br /> 30. Caskey CT, Kruh GD: The HPRT locus. Cell 1979, 16(1):1-9.                         52. Haarasilta S, Oura E: On activity regulation anaplerotic and <br /> 31. Kanehisa M, Araki M, Goto S, Hattori M, Hirakawa M, Itoh M, Katayama T,               gluconeogenetic enzymes growth process baker&#8217;s yeast. <br />     Kawashima S, Okuda S, Tokimatsu T, et al: KEGG linking genomes                 The biphasic growth. Eur J Biochem 1975, 52(1):1-7. <br />     life environment. Nucleic Acids Res 2008, 36(Database):D480-484.          53. Yin Z, Smith RJ, Brown AJ: Multiple signalling pathways trigger the <br /> 32. Guha BC: Investigations vitamin B(2): The sources vitamin B(2). II.             exquisite sensitivity yeast gluconeogenic mRNAs glucose. Mol <br />     The stability vitamin B(2). III. The chemistry vitamin B(2). Biochem J          Microbiol 1996, 20(4):751-764. <br />     1931, 25(3):945-959.                                                              54. Iliades P, Meshnick SR, Macreadie IG: Dihydropteroate synthase mutations <br /> 33. Przybyla-Zawislak B, Gadde DM, Ducharme K, McCammon MT: Genetic                   <span id='am-95' about='obo:NCBITaxon_4753' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_44281 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_147553 obo:NCBITaxon_37987 obo:NCBITaxon_2759'><span id='am-96'  ></span><span id='am-97' property="rdfs:label" content="Pneumocystis" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Pneumocystis</span> jiroveci affect sulfamethoxazole resistance a <br />     biochemical interactions involving tricarboxylic acid cycle (TCA) function            Saccharomyces cerevisiae model. Antimicrob Agents Chemother 2004, <br />     using collection mutants defective TCA cycle genes. Genetics              48(7):2617-2623. <br />     1999, 152(1):153-166.                                                             55. Basson ME, Moore RL, O&#8217;Rear J, Rine J: Identifying mutations duplicated <br /> 34. Sickmann A, Reinders J, Wagner Y, Joppich C, Zahedi R, Meyer HE,                      functions Saccharomyces cerevisiae: recessive mutations HMG-CoA <br />     Schonfisch B, Perschil I, Chacinska A, Guiard B, et al: The proteome               reductase genes. Genetics 1987, 117(4):645-655. <br />     Saccharomyces cerevisiae mitochondria. Proc Natl Acad Sci USA 2003,               56. Covert MW, Schilling CH, Palsson B: Regulation gene expression flux <br />     100(23):13207-13212.                                                                  balance models metabolism. J Theor Biol 2001, 213(1):73-88. <br /> 35. Chen XJ, Wang X, Kaufman BA, Butow RA: Aconitase couples metabolic                57. Kim J, Reed JL: OptORF: Optimal metabolic regulatory perturbations <br />     regulation mitochondrial DNA maintenance. Science 2005,                            metabolic engineering microbial strains. BMC Syst Biol 2010, 4:53. <br />     307(5710):714-717.                                                                58. Typas A, Nichols RJ, Siegele DA, Shales M, Collins SR, Lim B, Braberg H, <br /> 36. Wieczorke R, Krampe S, Weierstall T, Freidel K, Hollenberg CP, Boles E:               Yamamoto N, Takeuchi R, Wanner BL, et al: High-throughput, quantitative <br />     Concurrent knock-out 20 transporter genes required                  analyses genetic interactions E. coli. Nat Methods 2008, 5(9):781-787. <br />     block uptake hexoses Saccharomyces cerevisiae. FEBS Lett 1999,              59. Tong AH, Evangelista M, Parsons AB, Xu H, Bader GD, Page N, Robinson M, <br />     464(3):123-128.                                                                       Raghibizadeh S, Hogue CW, Bussey H, et al: Systematic genetic analysis <br /> 37. Balakrishnan R, Christie KR, Costanzo MC, Dolinski K, Dwight SS, Engel SR,            ordered arrays yeast deletion mutants. Science 2001, <br />     Fisk DG, Hirschman JE, Hong EL, Nash R, et al: Fungal BLAST Model                 294(5550):2364-2368. <br />     Organism BLASTP Best Hits: new comparison resources                        60. Ooi SL, Shoemaker DD, Boeke JD: DNA helicase gene interaction network <br />     Saccharomyces Genome Database (SGD). Nucleic Acids Res 2005,                          defined using synthetic lethality analyzed microarray. Nat Genet 2003, <br />     33(Database):D374-377.                                                                35(3):277-286. <br /> 38. Giaever G, Chu AM, Ni L, Connelly C, Riles L, Veronneau S, Dow S, Lucau-          61. <span id='am-105' about='obo:NCBITaxon_9596' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-106' property="oboInOwl:hasAlternativeId" content="NCBITaxon:317028" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasAlternativeId" content="NCBITaxon:445518" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="Pan" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasExactSynonym" content="chimpanzees" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-112'  ></span>Pan</span> X, Yuan DS, Xiang D, Wang X, Sookhai-Mahadeo S, Bader JS, Hieter P, <br />     Danila A, Anderson K, Andre B, et al: Functional profiling                     Spencer F, Boeke JD: A robust toolkit functional profiling yeast <br />     Saccharomyces cerevisiae genome. Nature 2002, 418(6896):387-391.                      genome. Mol Cell 2004, 16(3):487-496. <br /> 39. Huh WK, Falvo JV, Gerke LC, Carroll AS, Howson RW, Weissman JS,                   62. Behre J, Wilhelm T, von Kamp A, Ruppin E, Schuster S: Structural <br />     O&#8217;Shea EK: Global analysis protein localization budding yeast.                  robustness metabolic networks respect multiple knockouts. J <br />     Nature 2003, 425(6959):686-691.                                                       Theor Biol 2008, 252(3):433-441. <br /> 40. Dallies N, Francois J, Paquet V: A new method quantitative                    63. Deutscher D, Meilijson I, Schuster S, Ruppin E: Can single knockouts <br />     determination  <br /> </body></html>