<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                 Journal Visualized Experiments                                                                                           www.jove.com <br />  <br />  <br /> Video Article <br /> Mapping Bacterial Functional Networks Pathways <span id='am-22' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-28'  ></span><span id='am-29' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span>Escherichia Coli</span> <br /> using Synthetic Genetic Arrays <br />                      1,*                  2,3,*                   1,2                1,2 <br /> Alla Gagarinova            , Mohan Babu           , Jack Greenblatt , Andrew Emili <br /> 1 <br />     Department Molecular Genetics, University Toronto <br /> 2 <br />     Banting Best Department Medical Research, Donnelly Centre, University Toronto <br /> 3 <br />     Department Biochemistry, Research Innovation Centre, University <span id='am-1' about='obo:NCBITaxon_183601' typeof='owl:Thing obo:NCBITaxon_1329950 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8504 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8509 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_1329912 obo:NCBITaxon_33208 obo:NCBITaxon_1329961 obo:NCBITaxon_1329911 obo:NCBITaxon_7711 obo:NCBITaxon_8570 obo:NCBITaxon_169862 obo:NCBITaxon_7776 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_8578 obo:NCBITaxon_34989'><span id='am-2' property="rdfs:label" content="Regina" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Regina</span> <br /> * <br />     These authors contributed equally <br />  <br /> Correspondence to: Andrew Emili andrew.emili@utoronto.ca <br />  <br /> URL: http://www.jove.com/video/4056 <br /> DOI: doi:10.3791/4056 <br /> Keywords: Genetics, Issue 69, Molecular Biology, Medicine, Biochemistry, Microbiology, Aggravating, alleviating, conjugation, double mutant, <br /> <span id='am-38' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-39'  ></span><span id='am-40' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span>Escherichia</span> coli, genetic interaction, Gram-negative bacteria, homologous recombination, network, synthetic lethality sickness, suppression <br /> Date Published: 11/12/2012 <br /> Citation: Gagarinova, A., Babu, M., Greenblatt, J., Emili, A. Mapping Bacterial Functional Networks Pathways Escherichia Coli using <br /> Synthetic Genetic Arrays. J. Vis. Exp. (69), e4056, doi:10.3791/4056 (2012). <br />  <br />  <br /> Abstract <br />                                                                                                                                                        1 <br />       Phenotypes determined complex series physical (e.g. protein-protein) functional (e.g. gene-gene genetic) interactions (GI) . <br />       While physical interactions indicate bacterial proteins associated complexes, necessarily reveal pathway-level <br />       functional relationships1. GI screens, growth double mutants bearing deleted inactivated genes measured compared <br />       corresponding single mutants, illuminate epistatic dependencies loci provide means query discover novel <br />                               2                                                                              3-7 <br />       functional relationships . Large-scale GI maps reported eukaryotic organisms like yeast , GI information remains sparse <br />                        8 <br />       prokaryotes , hinders functional annotation bacterial genomes. To end, developed high-throughput <br />                                                    9, 10 <br />       quantitative bacterial GI screening methods       . <br />  <br />       Here, present key steps required perform quantitative E. coli Synthetic Genetic Array (eSGA) screening procedure genome- <br />             9 <br />       scale , using natural bacterial conjugation homologous recombination systemically generate measure fitness large numbers <br />       double mutants colony array format. Briefly, robot used transfer, conjugation, chloramphenicol (Cm) - marked mutant <br />       alleles engineered Hfr (High frequency recombination) &apos;donor strains&apos; ordered array kanamycin (Kan) - marked F- recipient <br />                                                                                                                                      11 <br />       strains. Typically, use loss-of-function single mutants bearing non-essential gene deletions (e.g. &apos;Keio&apos; collection ) essential gene <br />                                                                                                        9, 12, 13 <br />       hypomorphic mutations (i.e. alleles conferring reduced protein expression, stability, activity         ) query functional associations of <br />       non-essential essential genes, respectively. After conjugation ensuing genetic exchange mediated homologous recombination, the <br />       resulting double mutants selected solid medium containing antibiotics. After outgrowth, plates digitally imaged colony <br />                                                                                                 14 <br />       sizes quantitatively scored using in-house automated image processing . GIs revealed growth rate double <br />                                                                    9 <br />       mutant significantly better worse expected . Aggravating (or negative) GIs result loss-of-function mutations in <br />                                                                                                    2 <br />       pairs genes compensatory pathways impinge essential process . Here, loss single gene buffered, that <br />       single mutant viable. However, loss pathways deleterious results synthetic lethality sickness (i.e. slow growth). <br />                                                                                                                                 2 <br />       Conversely, alleviating (or positive) interactions occur genes pathway protein complex deletion either <br />       gene sufficient perturb normal function pathway complex additional perturbations reduce activity, <br />       growth, further. Overall, systematically identifying analyzing GI networks provide unbiased, global maps functional <br />                                                                                                                                                 9 <br />       relationships large numbers genes, pathway-level information missed approaches inferred . <br />  <br />  <br /> Video Link <br />       The video component article http://www.jove.com/video/4056/ <br />  <br />  <br /> Protocol <br />  <br />                                                                                                                      15, 16 <br /> 1. Constructing Hfr Cavalli Donor Mutant Strains Recombineering <br />                                                                                                                                                        16 <br />       The steps constructing eSGA donor stains described below. Briefly, use targeted &#955;- Red mediated homologous recombination of <br />       amplified selectable DNA marker cassette fragments generated PCR create non-essential gene deletion mutants (section 1.1) essential <br />       gene hypomorphic mutant donor strains (section 1.2) used &apos;queries&apos; define GI networks. <br />  <br />  <br /> Copyright &#169; 2012 Journal Visualized Experiments                                                            November 2012 | 69 | e4056 | Page 1 12 <br />              Journal Visualized Experiments                                                                                             www.jove.com <br />  <br />  <br />   Note: During technology development process, assessed effectiveness using Hfr-mediated conjugative transfer combining <br />   mutations using well-defined Hfr donor strain (Hfr Cavalli, Hfr Hayes, Hfr 3000). We examined (i) ability efficiently make donor <br />                                                                                16 <br />   mutants using recombineering method pioneered Yu et al. (2000) , (ii) relative efficiencies conjugative DNA transfer different <br />   chromosomal markers, (iii) effects query gene position chromosomal orientation relative Hfr transfer locus, oriT. We <br />   &#955;-Red-mediated homologous recombination efficiency higher Hfr Cavalli Hfr Hayes Hfr3000. If required, P1 <br />                                      17 <br />   transduction Psuedo Hfr strain , used create mutant donors. We successfully adapted methods making and <br />   screening large numbers donors. Since trial conjugation experiments, overall efficiency transfer number ex-conjugants <br />   observed significantly greater Hfr Cavalli, particular strain background chosen donor construction large-scale eSGA. All <br />   procedures eSGA screens using Hfr Cavalli donors described. <br />  <br />   1. Amplifying DNA fragment subsequent recombineering delete non-essential gene <br />  <br />      Employ two-step nested PCR amplification (Figure 1) create gene deletion cassette replacing target open reading frame a <br />                                                                                         15                                          11 <br />      Cm resistance marker pKD3 plasmid (GI: 15554330; protein id: AAL02033.1) . The marker flanked FRT-sites allowing <br />      removal resistance gene, necessary, follow experiments. <br />  <br />      Note: We nested two-step PCR remove plasmid amplified product used subsequently transform bacterial <br />      cells. Removing plasmid (in PCR) creating gene knockout cassette second PCR necessary because <br />      transformation plasmid efficient recombineering, goal obtain - possible - strains that <br />      successfully replaced targeted genomic DNA selectable marker. An alternative nested PCR perform restriction <br />      digest plasmid outside region amplified second PCR step. Then product restriction digest purified used as <br />      template nested PCR step (1.1.2). <span id='am-11' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-15'  ></span>This</span> option simple, requires work time. <br />         1. Use 45 ng pKD3 template PCR forward F1 reverse R1 primers produce 1,070 bp product. Purify PCR <br />             fragment using Qiagen PCR purification protocol, dilute product sterile distilled water 5 ng/&#956;l concentration. <br />         2. Using purified PCR product template, set second PCR gene-specific (e.g. knockout) nested primers, F2 R2, <br />             (i) 45 nt gene-specific homology regions 5&apos;-ends allow homologous recombination immediately upstream <br />             downstream target gene replace Cm resistance marker (ii) 20 nt 3&apos;-ends prime synthesis of <br />             Cm marker. The size produced fragment 1,123 bp. The gene-specific homology regions designed remove target <br />             open reading frame leaving adjacent partially overlapping coding segments intact. After amplification, proceed step <br />             1.3. <br />  <br />   2. Amplifying tagging cassette creating essential gene hypomorphic mutation <br />       1. To construct hypomorphic donor mutant strains, add C-terminal sequential peptide affinity (SPA) fusion tag, slightly <br />            impacts essential protein function destabilizing transcript abundance (i.e. protein copy number) protein folding/activity. To <br />                                                                                                                                 18 <br />            end, create selectable SPA-tag (SPA-Cm) DNA template converting Kan resistance marker pJL148 Cm using <br />                                                                                                                                    15 <br />            recombineering identical nucleotide sequences flanking Kan marker pJL148 Cm marker pKD3 . The resulting <br />            template consists in-frame SPA-tag followed Cm resistance marker, suitable PCR amplification, recombineering and <br />                               12 <br />            subsequent eSGA . <br />  <br />             Note: Essential genes potentially interesting gene function studies. However, genes deleted, <br />             innovative methods developed investigating functional relationships essential genes. Several suitable highly <br />                                                                                                                  19 <br />             successful approaches developed initially yeast. For example, Davierwala et al. (2005) performed GI screens using <br />             panel 575 temperature-sensitive essential gene mutants, essential genes exhibited times number <br />                                                                            20 <br />             GIs nonessential genes. Likewise, Breslow et al. (2008) developed &apos;decreased abundance mRNA perturbation&apos; (DAmP) <br />             approach generate library hypomorphic alleles covering ~82% essential yeast genes study pathways complexes. The <br />                                                                                                                6 <br />             DAmP approach modifies 3&apos;end expressed mRNA, reducing level target gene , likely mRNA destabilization. <br />                                           6 <br />             Similar DAmP approach , strategy exploit subtle effects perturbing 3&apos; end expressed mRNA of <br />             essential proteins. <br />  <br />             As below, tag itself does likely affect protein folding function, subtle perturbation mRNA the <br />                                                                                                                                                        13 <br />             tag sufficient, combined environmental stressors mutations, reveal functionally informative gene-environment <br />                               9, 12 <br />             gene-gene           interactions. <br />  <br />                                                                                                              21-23 <br />             Firstly, date, used SPA-tags purify ~3,000 essential non-essential E. coli proteins       , 80% 307 E. coli <br />                                                                                    8 <br />             essential proteins represented SPA-tagged strains . Because tag successfully used isolate known protein <br />             complexes, reciprocally validated (i.e. complex isolated tagging purifying different proteins <br />             complex); isolated complexes biologically relevant, reason tags usually affect <br />                                        12 <br />             protein folding function . Similarly, unpublished observations revealed tag itself does usually affect strain <br />             growth. However, reasoned altering 3&apos;-UTR integrating tag marker cassette, reported DAmP <br />                     6 <br />             strains , destabilize certain transcripts RNA level or, rare occasions, impede proper folding function fusion <br />             proteins. Therefore, predicted combining perturbed alleles essential genes functionally relevant mutations <br />                                                                                                      9, 12 <br />             genome result informative GIs, observed            . Furthermore, SPA-tagged alleles essential <br />             genes exhibited gene-environment interactions (e.g. drug hypersensitivity) according phenotypic profiling work Nichols et <br />                         13 <br />             al. (2011) , subjected subset SPA-tagged essential strains variety growth conditions. The study confirmed that <br />                                                                                                                                    13 <br />             (54 58 tested) SPA-tagged essential strains showed impaired fitness culture conditions , supports <br />  <br />  <br /> Copyright &#169; 2012 Journal Visualized Experiments                                                        November 2012 | 69 | e4056 | Page 2 12 <br />               Journal Visualized Experiments                                                                                         www.jove.com <br />  <br />  <br />            notion SPA-tag creates mild hypomorphic alleles essential genes useful genome-wide GI screen <br />            applications including eSGA. <br />         2. Amplify tagging cassette SPA-Cm template using gene-specific primers S1 S2, containing (i) 45 nt gene- <br />            specific region homologous recombineering 5&apos; end, followed (ii) 27 bp SPA-Cm specific sequence 3&apos; end. <br />  <br />   3. Confirming purifying PCR product <br />        1. Confirm tagging cassette correctly amplified subjecting 2-5 &#956;l PCR product agarose gel electrophoresis and <br />            purify remaining DNA following manufacturer&apos;s standard PCR purification protocol (Qiagen). Elute PCR product 30 &#956;l of <br />            sterile distilled water dilute ~50 ng/&#956;l concentration. The purified product used immediately transformation (step 1.5) or <br />            stored -20 &#176;C 6 months use. <br />  <br />   4. Preparing Hfr Cavalli competent cells donor construction <br />        1. Inoculate ml saturated overnight culture Hfr Cavalli strain 70 ml fresh <span id='am-6' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-7' property="rdfs:label" content="Luria" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10'  ></span>Luria</span>-Bertani (LB) medium 35 &#956;l 50 <br />           &#956;g/ml ampicillin 250 ml flask. Incubate culture 32 &#176;C shaking 220 rpm optical density (OD) 600 nm ~0.5-0.6 <br />           obtained (~2 hr). <br />        2. Transfer culture water bath heat induction &#955;red recombination system16 42 &#176;C 15 min shaking 160 rpm. <br />           Stop induction transferring culture chilled ice-slurry water bath 10-20 min 160 rpm. Make sure cells <br />           cold point transformation; use tubes cuvettes chilled ice 15 min prior use. <br />        3. Divide culture equally 2 pre-chilled 50 ml polypropylene tubes (~35 ml culture tube) centrifuge 4,400 x g 6 min at <br />           4 &#176;C. <br />        4. Discard supernatant, re-suspend cell pellet gentle inversion ~50 ml ice-cold sterile distilled water. Centrifuge the <br />           cells 4,400 x g 6 min 4 &#176;C. Discard supernatant re-suspend cell pellet 20 ml ice-cold sterile 10% <br />           glycerol centrifuge described above. <br />        5. Decant supernatant re-suspend cell pellet 500 &#956;l ice-cold 10% glycerol. Aliquot prepared competent cells 50 &#956;l <br />           volumes individual, pre-chilled 1.5 ml micro-centrifuge tubes immediate transformation. The cells frozen dry ice in <br />           liquid nitrogen stored -80 &#176;C 3 months. However, transformation efficiency highest freshly prepared competent <br />           cells. <br />  <br />   5. Transformation <br />        1. Add 100 ng purified PCR product step 1.3.1 competent cells. Flick tube allow suspension sit ice 5 min. <br />        2. Transfer suspension ice-cold competent cells DNA pre-chilled electroporation cuvette. Electroporate cell mixture with <br />            2.5 kV, 25 &#956;F, 200 ohms setting 2 mm gap cuvette (i.e. using different cuvette, manufacturer&apos;s instructions <br />            electroporation settings), immediately add 1 ml room temperature SOC medium. Transfer electroporated cells SOC <br />            medium 15 ml culture tube incubate 32 &#176;C 1 hr orbital shaking 220 rpm. <br />        3. After incubation, centrifuge cells 4,400 x g 5 min room temperature. Remove approximately 850 &#956;l supernatant, and <br />            re-suspend cell pellet remaining liquid. <br />        4. Spread cells pre-warmed LB plates containing 17 &#956;g/ml Cm, incubate 32 &#176;C overnight. Pick individual transformant <br />            colonies streak LB-Cm plates mutant confirmation. We recommend avoiding largest transformant colonies <span id='am-19' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> to <br />            reduce possibility picking strains rare suppressor mutations. Also streak transformants LB-Kan confirm that <br />            strains Kan resistant. <br />  <br />             Note: Since Cm resistance marker used make donor strains, don&apos;t expect donor mutants Kan resistant. By <br />             streaking transformants LB-Kan LB-Cm plates, ensure donor mutation itself did cause Kan <br />             resistance. If step performed target mutations affect ability strain survive Kan, double <br />             mutants effectively using eSGA donor strains survive selection steps. This merely precautionary <br />             logical step catch unexpected phenotypes issues interfere completion eSGA screens. If this <br />             step skipped, controls identify failed screens. However, earlier detection donor resistant Kan <br />             eliminate series unproductive unnecessary steps. Other possible reasons strain grow Kan include <br />             example expired antibiotic use wrong PCR product strain transformation. All issues be <br />             identified dealt early procedure aid control, required, recommended eSGA. <br />  <br />   6. Confirming non-essential gene deletion (step 1.6.1) essential gene hypomorphic mutation (section 1.6.2) <br />  <br />       Note: When confirming gene deletion PCR, genomic DNA wild type strain used control. <br />        1. Confirming non-essential gene deletion <br />              1. To confirm gene deletion PCR, isolate genomic DNA transformed, putative knockout strains grown liquid LB- <br />                 Cm medium, following manufacturer&apos;s instructions (Promega). <br />              2. To confirm successful recombineering, amplify DNA transformant different sets knockout confirmation <br />                 primers. Perform reactions ~150 ng genomic DNA template, confirm correct fragment sizes running the <br />                 PCR-synthesized products agarose gel. <br />              3. The primer set consists 20-nt flanking primer, located 200 bp upstream targeted region (KOCO-F), Cm-R <br />                 primer, complementary Cm cassette sequence. This amplification expected produce 445 nt amplicon the <br />                 mutant, wild type strain (Figure 1). <br />              4. The second set includes forward primer (Cm-F), annealing Cm cassette sequence, 20-nt reverse flanking <br />                 confirmation primer (KOCO-R), designed anneal 200 bp downstream 3&apos; end deleted gene. This <br />                 amplification reaction expected produce 309 nt amplicon mutant, wild type strain (Figure 1). <br />              5. The PCR contains KOCO-F KOCO-R primers. This reaction required confirm selected strain a <br />                 merodiploid, gene locus having replaced cassette duplicated wild-type gene <br />                 copy present. This amplification correct deletion mutant expected produce 1.4 kb product (Figure 1). <br />  <br />  <br /> Copyright &#169; 2012 Journal Visualized Experiments                                                       November 2012 | 69 | e4056 | Page 3 12 <br />               Journal Visualized Experiments                                                                                           www.jove.com <br />  <br />  <br />                    Note: When target gene length base pairs gene deletion cassette, notice a <br />                    difference locus replaced cassette, locus not. For definitive confirmation in <br />                    cases, recommend using additional primers specifically amplify DNA segment deleted gene. In case, <br />                    gene deleted, amplification product observed wild type control strain mutant. <br />                    These primers manually computationally designed specifically amplify segment DNA deleted region. <br />                    We typically amplify 140 bp regions. <br />  <br />         2. Confirming essential gene hypomorphic mutation <br />              1. To confirm hypomorphic mutation PCR, use gene-specific 20-nt forward (KOCO-F) reverse primers (KOCO-R) <br />                  located 200 bp upstream downstream, respectively, SPA-tag insertion site (Figure 2). <br />  <br />                  Note: This amplification step used control ensure absence untagged copy target essential gene. When <br />                  tagged version present, single band 310 bp larger SPA-Cm amplicon observed. A 400 bp product would <br />                  signal presence untagged version gene. <br />               2. In parallel PCR confirmation, use Western blotting verify in-frame insertion SPA-tag using anti-FLAG M2 <br />                                                                          21 <br />                  antibody specific FLAG-epitopes SPA-tag . <br />  <br />  <br />   7. Storing confirmed donor strain <br />        1. Transfer overnight culture confirmed recombinant donor mutant strain labeled cryovials, supplement glycerol to <br />            15% final concentration, mix well. For long-term storage, cryovials -80 &#176;C. <br />  <br />  <br /> 2. Arraying E. coli F- Recipient Mutants eSGA Screens <br />   1. Replica-pin E. coli Keio single non-essential gene deletion mutant collection generated Mori et al. (3,968 strains11; F- BW25113; can <br />      obtained Open Biosystems) robotically twenty-four 384-well microplates containing 80 &#956;l liquid LB media well, supplemented <br />      50 &#956;g/ml Kan. <br />  <br />                                                                                                   11 <br />      Note: For systematically assessing GIs essential genes, recipient Keio collection supplemented F- Kan-marked <br />                                                                             22, 23 <br />      essential gene hypomorphic mutant strains C-terminal SPA-tags           . <br />   2. To make room border control strain, act positive control, aid plate normalizations in <br />      automated colony quantizations, remove inoculated media outermost wells plate step transfer new <br />      plates, leaving outermost wells empty. <br />  <br />                                                                         11 <br />      Note: The border control strain JW5028 Keio collection deleted small pseudo gene not, in <br />      whole-genome screens, exhibit GIs donors. Therefore, good positive control strain identify rare instances when <br />      conjugation, pinning, selections failed, resulting absent sporadic border strain growth. For instance, border control colonies <br />      grow double mutant selection, query strain capable conjugation, conjugation attempted <br />      condition prevented taking place (e.g. antibiotic-containing plate), wrong antibiotic added the <br />      selection plate. Similarly, few, sporadic colonies observed plate, including border, conjugation pinning may <br />      failed screen needs repeated. If sporadic border growth reproducible, indicate consistently poor donor <br />      conjugation instances double mutants observed possibly conjugation failure, true GIs. Such screens <br />      need discarded analysis. Furthermore, present plates, border control strain allows colony <br />      quantization software automatically determine colony positions double mutant plates colony positions be <br />      demarcated manually. Lastly, border control strain aids normalization colony sizes (e.g. different rows plate) and <br />      (e.g. different recipient plates pinned donor different times) plates. <br />   3. Likewise, create negative control spots removing strains different location plate (not border) transfer <br />      strains new plate. Fill wells LB media containing 17 &#956;g/ml Cm. These negative control spots expected to <br />      recipient double mutant plates help ensure processing plate handling errors when <br />      numbering, imaging, pinning plates. <br />  <br />      Note: These spots required negative controls. Negative controls help identify instances selection fails - negative <br />      control strains grow selection. We suggest select unique negative control spots recipient plates, also <br />      choose negative control spots quadrant plate (e.g. lower right). This way, pattern negative control spots would <br />      identify possible plate mishandling errors (e.g. plates mislabeled pinned upside down, upper left colony pinned in <br />      lower right position). <br />   4. Inoculate Keio strain JW502811, pseudogene deletion, 500 ml flask 200 ml LB, containing 50 &#956;g/ml Kan. Based our <br />      whole-genome eSGA screens, growth particular mutant closest wild type BW25113 used border control. <br />   5. Grow arrayed strains JW5028 culture overnight 32 &#176;C 190 rpm orbital shaking OD ~0.4-0.6 600 nm. After <br />      overnight growth, border wells approximately 80 &#956;l JW5028 overnight culture. <br />   6. The assembled plates used step 3.2. Alternatively, long-term storage, supplement recipient plates 15% <br />      glycerol, mix media glycerol, plates -80 &#176;C. <br />  <br /> 3. High-density Strain Mating Procedure Generating E. coli Double Mutants <br />   1. Grow Hfr donor strain, bearing query mutation, marked Cm, overnight 32 &#176;C rich LB-Cm liquid medium (17 &#956;g/ml Cm) with <br />      shaking 220 rpm. <br />  <br /> Copyright &#169; 2012 Journal Visualized Experiments                                                        November 2012 | 69 | e4056 | Page 4 12 <br />                Journal Visualized Experiments                                                                                           www.jove.com <br />  <br />  <br />   2. Pin ordered recipient mutant collection 384-colony density solid LB plates supplemented 50 &#956;g/ml Kan. In parallel, pin the <br />      donor query mutant strain 384-colony density number LB plates, supplemented 17 &#956;g/ml Cm. Incubate plates <br />      overnight 32 &#176;C. <br />  <br />      Note: The recipient plates used pinning require 36 hr obtaining sufficiently large colonies subsequent steps. When <br />      average border colony size 2 mm diameter (or greater), plates ready pinned. <br />   3. To conjugate strains, pin donor 384-colony overnight donor plate solid LB plate. Subsequently pin single 384-colony <br />      recipient plate freshly-pinned donor conjugation plate. Continue pinning donor-recipient plates conjugated <br />      pinning solid LB conjugation plates. Incubate pinned conjugation plates 32 &#176;C 16 24 hr. <br />  <br />      Note: Individual single mutant donor strains gene-to-gene variability mating efficiency potential effects the <br />      mutations conjugation, DNA transfer efficiency, fitness. Growth stopped time 16 24 hr, sufficiently <br />      large colonies obtained subsequent pinning steps. Conjugations lasting 16 24 hr produce sufficient numbers ex- <br />      conjugants reproducible eSGA results mutants late transferring genes mutants slightly lower fitness. <br />   4. Pin 384-density conjugation plate single solid LB plate, containing Cm (17 &#956;g/ml) Kan (50 &#956;g/ml) conjugation plates <br />      pinned. Incubate freshly-pinned selection plates 16-36 hr 32 &#176;C. <br />  <br />      Note: In regards selection steps, different amounts time necessary different donor screens. Since double mutants a <br />      given screen donor mutation, average, slower growing donor mutant gives rise slower growing double mutants. Thus, <br />      depending donor fitness, selection steps require 16 36 hr. The double mutant growth stopped the <br />      double mutants sufficiently large subsequent step pinning selection imaging second. This usually <br />      translates border control colonies average 2 mm diameters. <br />   5. Re-pin selection plate second, double drug (Cm Kan), selection plate 1,536-colony format, first <br />      selection colony represented colonies second selection plate. Incubate plates 16-36 hr 32 &#176;C. Photograph final <br />      double mutant selection plates quantitatively measure growth fitness mutant analyze interactions gene pairs <br />      (Figure 3). <br />  <br />                                                                                                                        24 <br />        Note: Although frequency merodiploidy (partial chromosomal duplications) low 1/1,000 cells , merodiploidy mask GI. <br />        Hence, recommend including biological replicates eSGA screens. Fortunately, commercial disposable plates plastic pinning <br />        pads reused times sterilizing materials follows: Agar plates discarded plates pads <br />        sterilized soaking 10% bleach overnight, followed rinse distilled water, washing 70% ethanol, air-drying the <br />        plasticware flow hood ultraviolet light. The sterile pads plates stored sealed sterile plastic bags use. <br />  <br /> 4. Processing <span id='am-45' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-49'  ></span>Data</span> Deriving GI Scores <br />   1. Measure colonies plate batch mode individually using specialized colony imaging software 3, 9. <br />  <br />        Note: A suitable Java-based high-throughput colony imaging scoring application freely available public access (http:// <br />        srcollins77.users.sourceforge.net/). The software works different platforms (i.e. Mac PC) launched executable <br />        terminal window. <br />   2.   Normalize raw colony size measurements correcting systematic biases plates plate edge effects, <br />        inter-plate variation effects, uneven image lighting, artifacts physical curvature agar surface, competition effects neighboring <br />                                                                                        9 <br />        colonies possible pinning defects, differences growth time . These systematic artifacts independent double mutant <br />        fitness corrected rise spurious growth fitness estimates. <br />   3.   The colonies edge rows columns tend larger lower competition nutrients center plate. Thus, to <br />        correct effect, scale sizes edge colonies median size row column equal median size the <br />        colonies center plate. <br />   4.   Analyze results statistically account reproducibility standard variance median sizes replicate colony <br />        measurements. At independent biological replicate experiments necessary evaluate experimental reproducibility. <br />   5.   Finally, use normalized median colony growth fitness sizes generate GI score (S) gene pair using following formula: <br />  <br />  <br />  <br />  <br />   Where, Svar = (varExp x (nExp-1) + varCont x (nCont -1)) / (nExp + nCont -2); varExp = maximum variance normalized colony sizes for <br />   double mutant; varCont = median variances normalized double mutant colony sizes reference set; nExp = number of <br />   measurements double mutant colony sizes; nCont = median number experimental replicates experiments; &#956;Exp = median <br />   normalized colony sizes double mutants; &#956;Cont = median normalized colony sizes double mutants arising single <br />   donor mutant strain. The S-score reflects statistical confidence putative digenic interaction biological strength the <br />                                                                                                                                               9 <br />   interaction. Strong positive S-scores indicate alleviating effects, suggesting interacting genes participate pathway , while <br />                                                                                                                                                    9 <br />   significant negative S-scores reflect synthetic sickness lethality, suggestive membership parallel redundant pathways . For <br />   determining pathway-level functional relationships, single S-score pair tested genes produced final dataset. <br />  <br />  <br />  <br /> Copyright &#169; 2012 Journal Visualized Experiments                                                         November 2012 | 69 | e4056 | Page 5 12 <br />               Journal Visualized Experiments                                                                                              www.jove.com <br />  <br />                                9 <br />   Note: In previous study , recombination tends occur frequently genes 30 kbp other. Therefore, <br />   downstream analysis, following normalization score generation, remove interactions genes 30 kbp other. In <br />   addition GIs themselves, functional relationships pairs genes investigated looking similar GI profiles the <br />   genes are. The GI profile gene set interactions gene genes genome outside gene&apos;s linkage <br />                                                                                                      12, 25 <br />   area. Functionally similar genes tend higher correlations genetic interaction profiles        . Thus, calculating correlation coefficients <br />   genes experimental dataset use eSGA investigating functional relationships genes lying close each <br />   chromosome. <br />  <br />  <br /> Representative Results <br />   GIs reveal functional relationships genes. Similarly, genes pathway display similar GI patterns GI profile <br />   similarity represents congruency phenotypes, group functionally related genes pathways clustering GI profiles. <br />   Integrating GI GI correlation networks physical interaction information association data, genomic context (GC) <br />   relationships reveal organization higher-order functional modules define core biological systems (and crosstalk) <br />                                           4, 6, 7, 26 <br />   bacteria. For example, yeast            , E. coli gene pairs exhibiting alleviating interactions highly correlated GI profiles <br />   tend encode proteins physically associated (e.g. form complex; Figure 4a) act coherently common biochemical <br />   pathway (e.g. components linear cascade). A representative example shown Figure 4b, components functionally <br />   redundant Isc Suf pathways, jointly participate essential Fe-S biosynthesis process, form distinct clusters linked <br />                                                                                                                                                      27 <br />   extensive aggravating interactions (i.e. synthetic lethality). A statistical measure (e.g. hypergeometric distribution function ) be <br />   used GI GI correlation data significant enrichment interactions pathways (Figure 4c). Cluster analysis can <br />                                                                                                                            5, 6, 9, 28-30 <br />   applied functional networks derived using eSGA predict functions genes lacking annotations                         (Figure 4d). Since <br />   clustering algorithms vary, however, putative functional assignments determined clustering require independent experimental verification. <br />  <br />  <br />  <br />  <br /> Copyright &#169; 2012 Journal Visualized Experiments                                                          November 2012 | 69 | e4056 | Page 6 12 <br />              Journal Visualized Experiments                                                                                            www.jove.com <br />  <br />  <br />  <br />  <br />                                                                                                                                             14 <br />   Figure 1. The construction confirmation Hfr Cavalli non-essential single gene deletion donor strains. The panels (adapted ) <br />   illustrate deletion E. coli chromosomal ORF (A) primer sets used confirming (B) correctly generated mutant Hfr Cavalli <br />   donor strain. Amplifications correctly constructed deletion donor strain produce (i) 445 bp, (ii) 309 bp, (iii) 1.4 kb products. See <br />   protocol text details. Click view larger figure. <br />  <br />  <br />  <br />  <br /> Copyright &#169; 2012 Journal Visualized Experiments                                                        November 2012 | 69 | e4056 | Page 7 12 <br />              Journal Visualized Experiments                                                                                     www.jove.com <br />  <br />  <br />  <br />  <br />   Figure 2. The construction confirmation strategies essential hypomorphic donor mutant strains Hfr Cavalli genetic background. The <br />   panels illustrate creation hypomorphic mutations essential genes recombineering technology (A) primer sets (B) used <br />   PCR confirmation essential gene hypomorphic mutations. See protocol text details. <br />  <br />  <br />  <br />  <br /> Copyright &#169; 2012 Journal Visualized Experiments                                                   November 2012 | 69 | e4056 | Page 8 12 <br />              Journal Visualized Experiments                                                                                       www.jove.com <br />  <br />  <br />  <br />  <br />                                                                                                                  R <br />   Figure 3. Schematic summary key eSGA steps. The Hfr donor (marked chloramphenicol resistance, Cm ) recipient F- (marked with <br />                               R <br />   kanamycin resistance, Kan ) mutant strains grown 384-colony density LB-Cm LB-Kan plates, respectively. The donor the <br />   recipient strains conjugated pinning LB plate, incubated overnight 32 &#176;C. Consequently, the <br />   conjugants pinned plates containing Kan Cm select double mutants. The double mutant plates digitally imaged the <br />   growth fitness colony sizes quantitatively scored identify aggravating alleviating interactions. <br />  <br />  <br />  <br />  <br /> Copyright &#169; 2012 Journal Visualized Experiments                                                    November 2012 | 69 | e4056 | Page 9 12 <br />              Journal Visualized Experiments                                                                                            www.jove.com <br />  <br />  <br />  <br />  <br />   Figure 4. Representative computational analyses genetic interaction (GI) scores determining pathway-level functional relationships. (A) <br />   Panel I, Distribution correlation coefficients GI profiles gene pairs encoding proteins linked protein-protein interactions (PPI) <br />   versus randomly drawn gene pairs. The p-value computed using two-sample Kolmogorov-Smirnov (KS) test; Panel II, Scatter plot of <br />   correlated genetic profiles rich media (RM) transporters (mdtI, mdtJ) form heterodimeric complex required spermidine <br />                                                                                                 9 <br />   excretion. (B) The hierarchical clustering GI sub-network adapted Butland et al. highlights functional connectivity between <br />   components previously known Isc Suf pathways similar GI patterns. Pink represents aggravating (negative S-score) interactions, <br />   green represents alleviating (positive S-score) interactions black represents absence GI. A predicted novel component SUF <br />   pathway, ydhD, displays GIs members Isc pathway. (C) Pathway cross-talk recorded cell envelope processes are <br />                                                                                                                  27 <br />   significantly enriched aggravating alleviating GI according hypergeometric enrichment analysis . (D) A GI sub-network predicts the <br />   role functionally unannotated genes, yceG yebA, peptidoglycan splitting based pattern strong alleviating interactions with <br />                                                                                                                                        12 <br />   well-known cell division peptidoglycan hydrolases. The figures shown panels A, C D adapted Babu et al. (2011) . Click to <br />   view larger figure. <br />  <br />  <br /> Discussion <br />   We outlined step-wise protocol using robotic eSGA screening investigate bacterial gene functions pathway level by <br />   interrogating GI. This approach used study individual genes entire biological systems E. coli. Carefully executing the <br />   experimental steps described above, including appropriate controls, rigorously analyzing independently validating GI data are <br />   key aspects success eSGA making new functional discoveries. In addition eSGA, conceptually similar method studying <br />                                    17 <br />   GI E. coli, termed GIANT-coli , used illuminate novel functional relationships missed approaches, such <br />                  23             13 <br />   proteomic phenotypic screens alone. Nevertheless, genome-wide approach, limitations exist applicability of <br />   eSGA GIANT-coli, example bacterial <span id='am-16' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>. This partly unavailability genome-wide single gene deletion <br />   mutant strain collections bacterial species, particularly species targeted mutagenesis hindered low natural frequency of <br />                                                                                                                                                  31 <br />   homologous recombination. In cases, gene disru <br /> </body></html>