<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />          Characterization of HIRJ and HIR2, Two Genes Required for <br />                   Regulation of Histone Gene Transcription in <br />                            <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br />  <br />              The products of the HIRI and HIR2 genes have been defined genetically as repressors of histone gene <br />            transcription in S. cerevisiae. A mutation in either gene affects cell cycle regulation of three of the four histone <br />            gene loci; transcription of these loci occurs throughout the cell cycle and is no longer repressed in response to <br />            the inhibition of DNA replication. The same mutations also eliminate autogenous regulation of the HTA1-HTB1 <br />            locus by histones H2A and H2B. The HIR1 and HIR2 genes have been isolated, and their roles in the <br />            transcriptional regulation of the HTA1-HTB1 locus have been characterized. Neither gene encodes an essential <br />            protein, and null alleles derepress HTA1-HTB1 transcription. Both HIR genes are expressed constitutively <br />            under conditions that lead to repression or derepression of the HTAI gene, and neither gene regulates the <br />            expression of the other. The sequence of the HIR1 gene predicts an 88-kDa protein with three repeats of a motif <br />            found in the G. subunit of retinal transducin and in a yeast transcriptional repressor, Tupl. The sequence of <br />            the HIR2 gene predicts a protein of 98 kDa. Both gene products contain nuclear targeting signals, and the Hir2 <br />            protein is localized in the nucleus. <br />  <br />    The core histone genes in <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-28'  ></span><span id='am-29' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae are                repressor, Tupl (40). Both Hirl and Hir2 contain a bipartite <br /> under two forms of transcriptional control. Temporal regu-               nuclear localization signal present in nucleoplasmin, SwiS, <br /> lation occurs during the cell cycle, where transcription of all          Cdc25, and a large number of other nuclear proteins (7, 28), <br /> four histone gene loci is restricted to late Gl-early S phase (5,        and we have demonstrated that Hir2 is a nuclear protein by <br /> 15, 17, 38). Autogenous regulation is a unique response of               indirect immunofluorescence. <br /> the HTA1-HTB1 locus to altered dosage of the genes encod-                  We have found that the Hirl and Hir2 proteins are not <br /> ing H2A and H2B. Transcription of <span id='am-38' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-42'  ></span>this</span> locus is repressed                limiting for their regulatory functions in the cell. Both HIR <br /> when extra copies of HTA-HTB genes are present and                       genes are constitutively expressed under a variety of condi- <br /> derepressed when the HTA-HTB copy number is reduced by                   tions in which transcription of the HTAl-HTBI locus is <br /> half (22). Both temporal and autogenous regulation depend                differentially regulated, and neither gene represses HTA1 <br /> on common elements. One element is a <span id='am-51' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-55'  ></span>cis</span>-acting negative                 transcription when overexpressed. Since the HIR1 and <br /> site in the HTAl-HTBI promoter. When the negative site is                HIR2 genes do not appear to encode site-specific DNA <br /> deleted from this locus, HTA1-HTB1 transcription is dere-                binding proteins, the Hir proteins may function as corepres- <br /> pressed throughout the cell cycle (21, 25) and not repressed             sors to assist a repressor that interacts with sequences at the <br /> by elevated levels of H2A and H2B (22). The second element               HTAI-HTBI negative site. <br /> is a set of trans-acting HIR (histone regulatory) gene prod- <br /> ucts. Mutations in the HIRI, HIR2, and HIR3 genes confer <br /> phenotypes similar to deletion of the HTA1-HTB1 negative                                MATERIALS AND METHODS <br /> site; the HTAl-HTBI locus becomes constitutively tran- <br /> scribed during the cell cycle (26), and it loses feedback                   Strains and genetic methods. Standard yeast genetic pro- <br /> repression by H2A and H2B (22). The concordance of these                 cedures and media were used (33). The yeast strains used in <br /> phenotypes is provided by the observation that the three                 this study are listed in Table 1. The his4-9128 and lys2-1288 <br /> HIR gene products have been characterized genetically as                 alleles cause histidine and lysine auxotrophies and have been <br /> repressors that act through the negative site in the HTAl-               characterized previously (3, 4, 8, 9, 36, 43). The Spt pheno- <br /> HTB1 promoter (22, 26).                                                  type of strains was determined by replica plating onto SC <br />    In this study, we have isolated the HIR1 and HIR2 genes               medium lacking histidine or lysine and then by incubation at <br /> to determine what roles their gene products play in transcrip-           23&#176;C for 3 to 5 days (43). Transformation of yeast cells was <br /> tional regulation. Neither gene is essential, and null alleles           by the lithium acetate method (19). <br /> confer the same regulatory defects as the original hirl and                 <span id='am-67' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-68' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>Plasmids</span>. pHIR1 and pHIR2 were isolated from a library <br /> hir2 mutations. Both genes encode large proteins that have               of yeast genomic DNA cloned into YCp5O (29) and contain <br /> not been identified previously. The predicted Hirl protein               10-kb (HIRI) or 12-kb (HIR2) DNA inserts. YCp5O-HIR1 is <br /> contains three copies of a motif first noted in the G,3 subunit          a 3.4-kb ClaI-SalI fragnent from pHIRl cloned into the <br /> of bovine retinal transducin (11) and also found in a several            ClaI-SalI sites of YCp5O. YCp5O-HIR2 is a 3.8-kb subclone <br /> other yeast proteins, including the general transcriptional              of pHIR2 constructed by deletion of an 8.2-kb ClaI fragment <br />                                                                          from pHIR2. YIpHIR1 is a 3.5-kb EcoRI-SalI fragment from <br />                                                                          pHIRl cloned into the EcoRI-SalI sites of plasmid YIp5, and <br />                                                                          YIpHIR2 is a 3.9-kb SalI-ClaI fragment from pHIR2 cloned <br />   * <br />       Corresponding author.                                              into the SalI-ClaI sites of YIp5. YEpHIR1 is a 3.5-kb <br />                                                                     28 <br />  VOL. 13, 1993                                                                   CHARACTERIZATION OF HIR1 AND HIR2                  29 <br />  <br />                                                    TABLE 1. S. cerevisiae strains <br />    Strain                                                     Genotype                                                    Source <br /> PS2138-16B         MATa hirl-1 his4-9128 lys2-1288 HTA1-HTB1::HTA1-lacZ::LEU2 ura3-52                                  34 <br /> PS2-7A             MATa hir2-1 his4-9128 lys2-1288 HTAl-HTB1::HTAl-lacZ::LEU2ura3-52                                   34 <br /> PS-118             MATt HIR1::YIpS-HIR1 his4-9128 lys2-1288 ura3-52                                                    This study <br /> PS4-27             MATa hir2::YIpS-HIR2 his4-9128 lys2-1288 HTA1-HTB1::HTA1-lacZ::LEU2 ura3-52                         This study <br /> PSFWA1-A           MATa Ahirl::URA3 his4-9128 lys2-1288 ura3-52                                                        This study <br /> PSFWA&amp;2-A          MATatAhir2::URA3 his4-912b lys2-1288 ura3-52                                                        This study <br /> W303A1             MATa Ahirl::LEU2 ade2-1 ura3-52 trpl canl his3 leu2-3, -112                                         This study <br /> W303-D             MATa/MATot leu2-3, -11211eu2-3, -112 ade2-1/ade2-1 ura3-52/ura52 trpl/trpl canl/canl his3/his3      This study <br /> PSD100             MA Ta/MA Ta HIR1::YIpS-HIR1/hirl ura3-52/ura3-52 his4-9128/his4-912i lys2-1288/lys2-1288            This study <br /> PSD200             MA Ta/MA Ta hir2::YIp5-HIR21HIR2 ura3-52/ura3-52 his4-9128/his4-9128 lys2-1288/1lys2-1288           This study <br /> 2C37-2A            MATax hir3-1 his4-9128 lys2-1288 ura3-52                                                            34 <br /> MSS12-2B           MA Ta HIR4-1 his4-9128 lys2-1288 ade2-1 ura3-52                                                     M. Spector <br /> FW1238             AMTa his4-9128 lys2-1288 ura3-52                                                                    F. Winston <br /> L210               MA4Ta sptlO-11 his4-917 lys2-1288 trpl Aleu2-3,112                                                  F. Winston <br /> FW1619             MATa spt21-1 his4-917 lys2-1288 leu2 ura3-52                                                        F. Winston <br /> DN105              MATca Ahta2-htb2::URA3 his3 ura3-52                                                                 D. Norris <br /> DN106              AMTa Ahtal-htbl::URA3 his3 ura3-52                                                                  D. Norris <br /> M90-11D            MATa cdc28 leu2-3,112 thrl met2 his3                                                                M. Whiteway <br /> 1078-2B            MATa cdc4-3 leu2-3,112 trpl ura3-52 Iys2 lysS his7 ade2 ade6                                        B. Beyers <br /> RM14-4B            MATa cdc7-1 barl his6 trpl ura3 leu2-3,112 adel                                                     L. Hartwell <br /> 2754-7-4           MAATa cdc8 his7 leu2-3,112                                                                          L. Hartwell <br /> MSS21-5A           AMTat cdelS ura3-52 trpl                                                                            M. Spector <br />  <br />  <br /> EcoRI-SalI fragment from pHIR1 cloned into the EcoRI-SalI            visiae and plasmid DNAs from Eschenichia coli were pre- <br /> polylinker sites of plasmid YEp352. YEpHIR2 was con-                 pared as described previously (2, 33). Standard procedures <br /> structed by ligation of a 2.3-kb AatII-ClaI fragment from            were followed for subcloning DNA fragments and for iden- <br /> YEp24 (containing the 2 ,um circle replication determinant)          tifying recombinant clones in bacteria (1, 30). Southern blot <br /> to an 8.0-kb AatII-ClaI fragment from YIpHIR2.                       analysis (37) utilized radiolabelled DNA hybridization <br />    A 9-amino-acid epitope derived from the influenza virus           probes prepared by the method of random priming (10), with <br /> hemagglutinin protein (ha [42]) was inserted into both HIRI          [a-32P]dATP (NEN). The nucleotide sequence of the HIR1 <br /> and HIR2 by oligonucleotide-directed mutagenesis. Plasmid            and HIR2 genes was determined by the dideoxy chain <br /> YEpHIR1-ha was constructed by inserting a double-                    termination method (31), with restriction enzyme fragments <br /> stranded oligonucleotide (5&apos;-CTAGTTAGCCATACGACGT                     subcloned into bacteriophage M13mpl8 or M13mpl9 as <br /> CCCAGACTACGCTA-3&apos;) into a unique XbaI site within the                templates. The complete nucleotide sequence of both <br /> putative open reading frame of HIR] in plasmid YEpHIR1.              strands of DNA was obtained for each gene, and the <br /> Plasmid YEpHIR2-ha was constructed in two steps. A                   sequences spanned all restriction fragment junctions used to <br /> double-stranded oligonucleotide (5&apos;-CTACCCATACGACG                   create M13 subclones, with the exception of the most <br /> TCCCAGACTACGCTGG-3&apos;) was first inserted into a unique                extreme 3&apos; end of the HIR2 SalI-ClaI subclone. <br /> PvuII site within the putative coding region of pHIR2. A               Histone gene regulation assay. The Hir phenotype of <br /> 3.9-kb SalI-ClaI fragment containing the HIR2 coding region          strains was determined as previously described by analyzing <br /> with the ha epitope was then substituted for the same                transcription of the HTAI gene in cells treated for 30 min <br /> SalI-ClaI fragment in YEpHIR2.                                       with hydroxyurea (26, 34). The levels of HTA1 RNA were <br />    RNA analysis. Total RNA was isolated as described pre-            determined by Northern blot analysis; a Hir- phenotype is <br /> viously (23), and 20 plg was subjected to formaldehyde-              manifested as the failure to repress HTAJ transcription <br /> agarose gel electrophoresis and Northern (RNA) blot analy-           following the inhibition of DNA replication. <br /> sis (30). A 2.4-kb Sacl fragment from the HTAl-HTB1 locus              Construction of null alleles and genetic analysis of HIRI and <br /> (16) was used to detected HTAJ transcripts. This fragment            HIR2 clones. Plasmid pHIR1::URA3, which contains a dele- <br /> contains the entire HTAI gene, as well as a region of the            tion-disruption allele of HIR1, was constructed by replacing <br /> constitutively transcribed AKY2 (PRTI) gene (15, 24). A              a 1.4-kb BglII-BglII fragment within the putative open read- <br /> 1.7-kb BamHI-SalI fragment from YCp5O-HIR1 was used to               ing frame of HIRI (nucleotides 1000 to 2410) in plasmid <br /> detect HIRJ transcripts, and a 1.5-kb EcoRI-SalI fragment            YIpHIR1 with a 1.1-kb BamHI fragment containing the <br /> from YCp5O-HIR2 was used to detect HIR2 transcripts.                 URA3 gene (Fig. 1). A 1.4-kb BamHI-SalI fragment was <br />    The 5&apos; ends of HIR1 and HIR2 mRNA were estimated by               isolated from pHIR1::URA3, transformed into diploid strain <br /> Si nuclease protection analysis with total RNA prepared              W303-D, and Ura+ prototrophs were selected. Replacement <br /> from cells transformed with YEpHIR1 or YEpHIR2. A                    of the HIRI + allele on one homolog was confirmed by <br /> 1.6-kb ClaI-AccI fragment from YCp5O-HIR1 was labelled at            Southern blot analysis. A null allele of HIR2 was con- <br /> the AccI site and a 2.6-kb EcoRI-EcoRV fragment from                 structed in two steps. First, plasmid pBS-HIR2 was con- <br /> YIpHIR2 was labelled at the EcoRI site with [_y32P]ATP, by           structed by ligating a 2.9-kb SalI-EcoRV fragment from <br /> using T4 polynucleotide kinase (22, 25). These fragments             pBluescript II KS+ (Stratagene) with a 2.3-kb SalI-SspI <br /> were hybridized overnight at 48&#176;C to 20 ,ug of RNA and               fragment from pHIR2. Second, a 1.2-kb HindIlI fragment <br /> analyzed as previously described on a 4% acrylamide-8 M              within the putative open reading frame of HIR2 (nucleotides <br /> urea gel (25).                                                       472 to 1738) in plasmid pBS-HIR2 was replaced with a 1.1-kb <br />   DNA analysis. Genomic and plasmid DNAs from S. cere-               HindIll fragment containing the URA3 gene to construct <br />  30     SHERWOOD ET AL.                                                                                                       MOL. CELL. BIOL. <br />  <br />              - E           %-F m                                                a Zeiss Axioplan microscope and a Zeiss Neofluar 100x <br />   HIR1                     A.,o         a         x    c   CD                   objective. Samples were photographed with TMAX-400 film. <br />  chrom. II     H-          .                        ll  I                       The exposure time for fluorescein micrographs was 40 s for <br />                                                                                 both experimental and control strains. <br />                                                     ..7/ <br />                                                        88 <span id='am-56' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-57' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-60'  ></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span>aa</span>                       Western blot (immunoblot) analysis. Total yeast proteins <br />                 i              I <br />                                    hirl::URA 43                                 were extracted from cells that had been transformed with <br />                                                                                 plasmids YEpHIR1, YEpHIR1-ha, YEpHIR2, or YEpHIR2- <br />                                                                                 ha. Cells (10 ) were collected by centrifugation, resuspended <br />                                                                                 in 110 ,ul of 33% glycerol-7% sodium dodecyl sulfate (SDS)- <br />                                                                                 0.3 M dithiothreitol-1 x protease inhibitor mix ([in micro- <br />                -a    S     &gt;       .C   8                        L         ,,   grams per milliliter] chymostatin, 0.1; aprotinin, 2; pepstatin <br />                Cl)   I     a.I <br />    HIR2              I             I        I                   l          l    A, 1; E-64, 7.2; leupeptin, 0.5), and then subjected to two <br /> chrom. XV                                                                       cycles of vortexing with glass beads for 1 min and heating at <br />                                                                      875 aa     95&#176;C for 5 min. After the second cycle of vortexing and <br />                I-I        hir2::URA3        I-                              l   heating, samples were centrifuged for 15 s to remove debris, <br />                                                                                 and 15 ,ul of the supernatant fraction was loaded directly <br />                                                                                 onto a 7.5% polyacrylamide-SDS gel. Following electro- <br />                                                            500 nt               phoresis, proteins were transferred to polyvinyl difluoride <br />    FIG. 1. Restriction enzyme maps of tthe HIRJ and HIR2 genes.                 membranes (Immobilon-P, Millipore) by electroblotting. The <br /> The 3.4-kb ClaI-SalI fragment containirig the HIR1 gene and the                 filters were blocked (30) and incubated for 12 h at room <br /> 3.8-kb SalI-ClaI fragment containing thle HIR2 gene are shown.                  temperature with a 1:250 dilution of primary antibody <br /> Arrows indicate the open reading frames for each gene. Amino acid               (12CA5 monoclonal antibody). Antibody binding was visu- <br /> (aa) motifs in the predicted HIR1 and HIIR2 gene products are also              alized by subsequent incubation with alkaline phosphatase- <br /> indicated. Stippled bars represent three <br /> in Hirl. Striped bars represent the region        g-transducin <br />                                                        repeats found            conjugated goat anti-mouse immunoglobulin G (30) pur- <br /> and Hir2. Solid bars represent the putat <span id='am-1' about='obo:NCBITaxon_1148155' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_72037 obo:NCBITaxon_131567 obo:NCBITaxon_197562 obo:NCBITaxon_1148154 obo:NCBITaxon_197563 obo:NCBITaxon_6830 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Ive" datatype="xsd:string"></span>ive</span> bipartite nuclear local-           chased from Organon Teknika. Western blots were <br /> ization signals found in Hirl and Hir2. Regions of the HIRI and                 photographed by using Polaroid type 55 film. <br /> HIR2 open reading frames that were               Releted <br />                                                   and replaced with               Nucleotide sequence accession numbers. The sequences for <br />                                                                                 HIRI and HIR2 have been assigned GenBank numbers <br /> URA3 sequences are indicated below the open reading frame maps. <br /> The yeast chromosomes (chrom.) on whiich the two genes reside are               L03838 and L03839, respectively. <br /> also indicated (34). nt, nucleotides. <br />  <br />                                                                                                           RESULTS <br /> plasmid pHIR2::URA3 (Fig. 1). A 2.3-kb XbaI-XhoI frag- <br /> ment was isolated from pHIR2: :URA3 and used to transform                         Isolation of HIRi and HIR2 genes. Mutations in either the <br /> diploid strain W303-D to uracil prototrophy. Southern blot                      HIR1 or the HIR2 gene prevent transcription of the HTAl- <br /> analysis showed that the HIR2+ gene on one homolog had                          HTBJ locus from being turned off when DNA replication is <br /> been replaced with the deletion-disruption allele.                              inhibited and thus confer a Hir- (histone regulatory) pheno- <br />    To confirm that the authentic HIRI and HIR2 genes had                        type (26, 34). We took advantage of our observation that the <br /> been isolated, we tested whether the cloned DNA directed                        same mutations also suppress the transcriptional effects of <br /> integration of a marker gene (URA3) to the HIRJ or HIR2                         the his4-9128 and lys2-1288 mutations (34) to isolate DNA <br /> locus. Plasmid YIpHIR1 was digested with XbaI, which cuts                       clones that contained the HIRJ or HIR2 gene. hirl and hir2 <br /> once within the HIRJ insert, and transformed into strain                        mutant strains containing the two 8 insertion alleles are <br /> FW1238 (HIR1 + his4-9128). Ura+ transformants were se-                          phenotypically His&apos; Lys&apos; (or Spt-), while HIR+ strains are <br /> lected, and Southern blot analysis was used to identify a                       His- Lys- (or Spt+). Using a library of yeast genomic DNA <br /> strain in which the plasmid had integrated at the putative                      cloned into centromere plasmid YCp5O (28), we screened for <br /> HIRJ + locus. This strain (PS-I18) was then crossed to strain                   the wild-type HIRI and HIR2 genes by their ability to <br /> PS2138-16B (hirl-1 his4-9128) to create heterozygous diploid                    complement the Spt- phenotype of a hirl or hir2 mutant. <br /> strain PSD100 (Table 1). Plasmid YIpHIR2 was partially                          One transformant with a stable Spt+ (His- Lys-) phenotype <br /> digested with PvuII, which cuts once within the HIR2 insert,                    was identified for each hir mutant, and in each case the Spt+ <br /> and transformed into strain PS2-7A (his2-1 his4-9128). Fol-                     phenotype was associated with the presence of a unique <br /> lowing selection for Ura+ transformants, a strain containing                    plasmid. When the pHIR1 or pHIR2 plasmid was lost by <br /> the plasmid integrated at the putative hir2 locus was identi-                   growth of the transformants in nonselective medium, the <br /> fied by Southern blot analysis. This strain (PS-127) was then                   hirl or hir2 mutant strain recovered its Spt- (His&apos; Lys&apos;) <br /> crossed with strain FW1238 (HIR2+ his4-9128) to create                          phenotype. In addition, when the pHIR1 or pHIR2 plasmid <br /> heterozygous diploid strain PSD200 (Table 1). Following                         was isolated from transformants and retransformed into hirl <br /> sporulation of both diploids, tetrads were analyzed for their                   or hir2 mutant strains, each reconferred an Spt+ (His- <br /> Spt (His) phenotype.                                                            Lys-) phenotype. <br />    Immunofluorescence. Strains W303A1 (Ahirl) and PS2-7A                          The minimal DNA sequences encoding HIR1 or HIR2 <br /> (hir2) were transformed to uracil prototrophy with plasmids                     were identified by subcloning fragments of pHIRl or pHIR2 <br /> YEpHIR1 and YEpHIR1-ha or YEpHIR2 and YEpHIR2-ha.                               into YCp5O (Fig. 1). A 3.4-kb ClaI-SalI fragment from <br /> Indirect immunofluorescence of fixed cells was performed as                     pHIR1 and a 3.9-kb SalI-ClaI fragment from pHIR2 were <br /> previously described (27), with monoclonal antibody 12CA5                       found to complement the Spt- phenotype of a hirl or hir2 <br /> (I. Wilson, Scripps Institute), which recognizes the influenza                  mutant (Fig. 2A). More importantly, the appropriate HIRJ <br /> virus hemagglutinin epitope as the primary antibody. DNA                        or HIR2 subclone also corrected the Hir- phenotype of the <br /> was stained with 4&apos;,6&apos;-diamidino-2-phenylindole dihydro-                        same hir mutants (Fig. 2B). Each subclone specifically <br /> chloride. Fluorescence microscopy was carried out by using                      complemented the two mutant phenotypes of hirl or hir2 <br />  VOL. 13, 1993                                                                  CHARACTERIZATION OF HIRJ AND HIR2                   31 <br />  <br />   AYCp5O            YCp5O-HIR 1            YCp5O-HIR2                segregation of both the Ura and Spt phenotypes was fol- <br />                                                                      lowed in tetrads. The Spt phenotype segregated 2+:2- in 20 <br />                                                                      tetrads resulting from the HIRJ cross, and every Spt+ <br />                                                                      segregant was Ura+. No Spt- segregants were observed <br />                                                                      among 21 tetrads analyzed from the HIR2 cross. These <br />                                                                      results demonstrated tight linkage of YIpHIR1 to the HIRJ <br />                                                                      locus and YIpHIR2 to the hir2 locus. Hybridization of <br />                                                                      restriction enzyme fragments to yeast chromosomal DNA <br />                                                                      blots localized HIRJ to chromosome II and HIR2 to chro- <br />                                                                      mosome XV (Fig. 1) (34). We have shown previously that <br />                                                                      HIR2 is linked to the centromere of chromosome XV (34). <br />                                                                         Sequence analysis of HIRI and HIR2. The nucleotide <br />                                                                      sequences of the HIRI and HIR2 genes revealed that these <br />                                                                      genes have not been identified previously. The HIRJ gene <br />                                                                      (Fig. 3A) contains a single long open reading frame (2,364 <br />                                                                      nucleotides) that is sufficient to encode a 788-amino-acid <br />                                                                      protein of 87,923 Da. We mapped the 5&apos; end of HIRJ RNA <br />                                                                      by an Si nuclease protection assay and identified a single <br />                                                                      major site of transcription initiation approximately 80 nucle- <br />                                                                      otides downstream of a sequence that could serve as a <br />                                                                      TATA box (nucleotides 441 to 447) and approximately 150 <br />                                                                      nucleotides upstream of the putative initiator ATG <span id='am-43' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-46'  ></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> at <br />                                                                      nucleotides 682 to 684. An epitope-tagged Hirl protein <br />                                                                      migrates with an apparent Mr of .-100,000 (see Fig. 8), larger <br />                                                                      than the protein predicted from the nucleotide sequence. <br />                                                                      Since translation initiated at a more TATA-proximal ATG <br />                                                                      (nucleotides 579 to 581) is terminated by a stop codon at <br />                                                                      nucleotides 662 to 664, this difference may reflect anomolous <br />                                                                      mobility of the Hirl gene product or modification of the Hirl <br />                                                                      protein. <br />                                                                         The predicted Hirl protein has three distinguishing char- <br />                                                                      acteristics (Fig. 4). First (Fig. 4A), it contains a region of 62 <br />                                                                      amino acids (residues 282 to 343) that is similar to a region in <br />                                                                      the predicted Hir2 protein (residues 300 to 361 [12]). In this <br />                                                                      region, 22 residues (35%) are identical, and 31 residues (50%) <br /> B          hirl-l                           hir2-1 <br />                                                                      are conserved between Hirl and Hir2. Second (Fig. 4B), <br />                                                                      residues 426 to 443 have the characteristics of the bipartite <br />                                                         IV           nuclear localization signal found in nucleoplasmin, Swi5, <br />      to)     4t~                   4 <br />                                        0                             Cdc25, and a large number of other predominantly nuclear <br />                      A,v <br />                                                                      proteins (7, 28). Third (Fig. 4C), the amino terminus contains <br />  A                                                                   three copies of an amino acid repeat that was first identified <br />                                                                      in the 1 subunit of bovine transducin, a heterotrimeric G <br />                                       ad&amp;co A-A <br />                                                                      protein (11), and subsequently identified in a number of yeast <br />        +        +    -                                               proteins, including Ste4, Cdc4, Cdc2O, Tupl, Prp4, and <br />                                 .4 <br />                                    111g <br />                                     . <br />                                    so,                               Makll (6, 11, 13, 18, 39, 40), as well as in the Drosophila <br />                                                   &quot;S                 enhancer of split gene product (14). <br />                                                                         The sequence of the HIR2 gene (Fig. 3B) contains a single <br />    FIG. 2. Complementation of Spt- and Hir- phenotypes by the <br /> cloned HIRI and HIR2 genes. Strains FW1238 (HIR+), PS2138-16B        long open reading frame (2,625 nucleotides) sufficient to <br /> (hirl-1), and PS2-7A (hir2-1) were transformed with plasmids         encode an 875-amino-acid protein of 98,517 Da. Two sites of <br /> YCp5O, YCp5O-HIR1, or YCp5O-HIR2. (A) Spt phenotypes. Top            transcription initiation have been mapped by an S1 nuclease <br /> panel, growth on SD plus His medium; bottom panel, growth on SD      protection assay to approximately nucleotides 50 and 100. <br /> without His medium. (B) Hir phenotypes. Northern blot analysis of    There is no recognizable TATA box sequence upstream of <br /> H2A1 and PRT1 (internal control) mRNAs from cells grown in the       the longer protected transcript; however, the latter initiation <br /> absence (-) or presence (+) of hydroxyurea for 30 min. A Spt- Hir-   site is approximately 30 nucleotides downstream of a poten- <br /> phenotype is manifested as growth on medium lacking histidine, and   tial TATA box sequence (nucleotides 64 to 70) and 40 <br /> the continued production of HTA1 mRNA in the presence of <br /> hydroxyurea.                                                         nucleotides upstream of the putative translation initiation <br />                                                                      codon. This spacing is typical of many yeast genes and <br />                                                                      makes it likely that the ATG at nucleotides 139 to 141 <br />                                                                      represents the authentic translation initiation codon. This <br /> mutants and had no effect on either the Spt+ (Fig. 2A) or the        conclusion is strengthened by Western blot analysis of an <br /> Hir+ phenotype of HIR+ strains (see Fig. 7).                         epitope-tagged Hir2 protein, which has identified a protein of <br />   We confirmed that the authentic HIRJ or HIR2 gene had              -98,000 Da (see Fig. 8). <br /> been isolated by testing whether the cloned DNA directed                The putative Hir2 protein contains two features of interest <br /> integration of a marker gene (URA3) to the HIRJ or hir2              (Fig. 4). The first is the region of homology to the Hirl <br /> locus as described in Materials and Methods. Diploid strains         protein described above (residues 300 to 361). The second is <br /> PSD100 and PSD200 (Table 1) were sporulated, and the                 the presence of a similar bipartite nuclear localization signal <br />  <br />  <br />      FIG. 3. Nucleotide sequences of the HII~1                         (A)      and HIR2        (B)    genes and                    predicted    amino acid residues in the                           Hirl   and Hir2         proteins. <br />  <br />  <br />  <br />  <br /> between residues 454 and 472. Neither                            Hirl nor Hir2 contain                                wild-tWrpe,              Ahirl, Ahir2, and congenic                                iAhirl       Ahir2 strains. <br /> any of the        typical          motifs that have been                    associated           with                 Each strain grows at                           th&apos;e    same         rate on      YPD        plates      incubated <br /> DNA binding-            proteins.                                                                                         at     temperatures ranging from 16                              to 370C and has an identical <br />  <br />      Phenoye                  I)and HIR2 null mutants. Null alleles of                                                generation                time        in       liquid        SD       medium           at       30&apos;C. Finally, <br /> HIRI                   I2wre              co&apos;nstructed &apos;in            vitro       (Fig. 1)        and                     microscopic             examination                      has     revealed          no   morphological <br /> i&apos;ntegrated   into         diploid       strain W303-D            des&apos;cribed in <br />                                                                  as                             Mate-                 differenc&apos;es               among           the        wild         ty&apos;pe    and        single- or double- <br /> rials and Methods. Diploid strains                           heterozygo&apos;us for                  either                deletion mutants. <br /> the HIRJ          or   the HIR2            null allele-          produced          four viable                                 Although          null alleles of               HIRI         or    HIR2 do not affect cell <br /> spores,    indicating        that neither HIRJ               nor    HIR2 is        an   essential                     viabilit.y,                              (see Fig. 6) And Spt- (<span id='am-62' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-63' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-64'  ></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> <br />                                                                                                                                               both alleles confer               Hir- <br /> gene. A    Ahirl <br />               Lihir2 double mutant is also viable. We have                                                            not           shown) phenotypes, suggesting that the original hirl-I <br /> noted no obvious phenotypic differences among isogenic                                                                and hir2-1               mutations represent loss of function alleles. <br />  <br />  <br />  <br />  <br />                                                                                                                                    We first examined whether the cell cycle-dependent tran- <br />                                                                                                                                                                                                     <br /> products have been identified genetically as repressors of                                                                       scription of the HTAI gene was correlated with changes in <br /> yeast histone gene transcription. They are required for the                                                                      the steady-state levels of HIRI or HIR2 mRNA (Fig. 5A). <br /> periodic repression of three of the four histone gene loci                                                                       During the mitotic cell cycle, transcription of the HTAI- <br /> during the cell cycle (26), as well as for autogenous regula-                                                                    HTBI locus is repressed until late G1, when it is activated by <br /> tion of the HTAI-HTBI locus by H2A and H2B (22). We                                                                              completion of the CDC4-dependent step (15, 38). Transcrip- <br /> asked whether the finction of Hirl or Hir2 in these two                                                                          tion is repressed in S-phase cells by two pathways. In <br /> processes is determined by the differential transcription of                                                                     normally cycling cells, it is repressed in early S phase shortly <br /> the HIRI or HIR2 gene. We therefore measured the levels of                                                                       after the execution of the CDC7-dependent step; in the <br /> HIRJ or HIR2 mRNA, each of which is an approximately                                                                             absence of CDC7 function, the histone genes are constitu- <br /> 2.6- to 2.7-kb-long polyadenylated RNA <span id='am-48' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-49' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>, under                                                                            tively transcnbed (15). Interruption of S phase with a drug <br /> conditions that lead to either repression or derepression of                                                                     such as hydroxyurea leads to premature repression of HTA1- <br /> the HTAI-HTBI locus.                                                                                                             HTBI transcription (21). Finally, the histone genes are not <br />  <br />   FIG. 4. Motifs present in the predicted Hirl and Hir2 proteins. <br /> (A) Homology between amino acid residues 282 to 343 of Hirl and                                                                                             HIR2 <br /> 300 to 361 of Hir2. Identical residues are indicated by vertical lines, <br /> and conserved residues are indicated by dots. (B) Comparison of the <br /> putative nuclear localization signal in Hirl and Hir2 to that identified                                                         =                          PRTI <br />                                                                                                                             ZW                          &quot; ~H2A1 <br /> in nucleoplasmin and Swi5. Numbers represent the position of the <br /> signal within the four proteins. (C) Comparison of the ,B-transducin                          FIG. 5. Regulation of HIR2 mRNA synthesis. (A) Cell division <br /> repeats in the Hirl protein to those in Tupl. Residues in Hirl                             cycle mutants M90-11D (cdc28), 1078-2B (cdc4), RM14-4B (cdc7), <br /> identical to the consensus sequence derived by Fong et al. (11) are                        2754-7-4 (cdc8), and MSS21-5A (cdclS) were grown at the permis- <br /> indicated by bold letters. Numbers represent the position of the                           sive (23&#176;C) or restrictive (37&#176;C) temperature for 3 h, and wild-type <br /> repeats in the two proteins.                                                               strain FW1238 (CDC&apos;) was arrested in S phase by treatment with <br />                                                                                            hydroxyurea for 30 min at 30&#176;C. (B) Wild-type strain FW1238 (lane <br />                                                                                            WT) was transformed with high-copy-number plasmids YEp24, a <br />                                                                                            vector control, or 2 ,u-HTA1-HTB1, which produces elevated levels <br /> expressed in G2-phase cells. We measured the levels of                                     of H2A and H2B. Strains DN106 (Ahta2-htb2) and DN105 (Ahtal- <br /> HTA1, HIR1, and HIR2 mRNAs in cdc mutants grown at the                                     htbl) have reduced HTA-HTB gene dosage. (C) Strains PS2138-16B <br /> permissive temperature or blocked at various points in G1,                                 (hirl), PS2-7A (hir2), 3137-2C (hir3), MSS12-2B (HIR4-), L210 <br /> S, and G2 by growth at the restrictive temperature. The                                    (sptlO), and FW1619 (spt2l) all confer a Hir- phenotype. Total RNA <br /> HTAJ gene was regulated as expected at each CDC control                                    was extracted from each strain, and 20 pug was subjected to <br /> point, but neither HIRI nor HIR2 mRNA synthesis was                                        Northern blot analysis, with hybridization probes that detect HTA1, <br /> regulated in a cell cycle-dependent manner.                                                PRTI, and HIR2 transcripts. The same blots were then stripped and <br />   We next examined the levels of HIRI and HIR2 mRNAs                                       rehybridized to a probe that detected HIRI mRNA, and identical <br /> under conditions in which transcription of the HTAI-HTB1                                   results were obtained (data not shown). <br /> locus is autogenously regulated (Fig. 5B). We measured the <br /> levels of HTA1, HIRI, and HIR2 mRNAs in strains in which <br /> HTAl-HTB1 transcription was either repressed by increased                                  Hir- phenotype (33, 37a). None of these mutations affected <br /> levels of H2A and H2B (2p1ITA1-HTB1 [22]) or derepressed                                   transcription of either HIR gene. Thus, the HIRJ and HIR2 <br /> by reduced levels of the same histones (Ahta2-htb2 and                                     genes do not appear to be targets in a sequential cascade of <br /> Ahtal-htbl [22]). We again found that the two HIR tran-                                    transcriptional regulation. <br /> scripts were present at invariant levels, independent of the                                 We also inserted the HIR1 and HIR2 genes into a high- <br /> effects of the levels of H2A and H2B on the expression of                                  copy-number plasmid to assess the effects of overproduction <br /> HTAI.                                                                                      of their gene products. Approximately 30 times more RNA <br />   We finally asked whether any gene that is known to                                       was made from each high-copy-number HIR gene relative to <br /> regulate the HTAI-HTBI locus affects the expression of                                     that of a single copy HIR1 or HIR2 gene (Fig. 6, compare <br /> HIRI or HIR2 (Fig. 5C). Mutations in at least eight genes                                  panels 1 and 3 to panels 5 and 7), and it resulted in elevated <br /> cause a Hir- phenotype (33, 37a). Some of these genes may                                  levels of Hir proteins (see Fig. 8) (33a). <br /> confer a Hir- phenotype because they regulate other HIR                                      We asked whether overexpression of either gene product <br /> genes that are primary transcriptional regulators. We exam-                                could suppress the mutant phenotypes of three strains that <br /> ined the levels of HIRI and HIR2 mRNAs in sptlO, spt2l,                                    were derepressed for transcription of HTA1. We initially <br /> hirl, hir2, hir3, or HIR4- mutants, each of which shows a                                  found that a mutant allele of one HIR gene could not be <br />  VOL. 13, 1993                                                                                   CHARACTERIZATION OF HIRJ AND HIR2                  35 <br />  <br />           1       2            3       4       5       6       7       8              HIR1 or HIR2 gene (lanes 1 and 3). The apparent molecular <br />                                                                                       mass of the tagged Hir2 protein agreed exactly with the size <br />  HIR1 <br />               +       + <br />  <br />  <br />  <br />  <br />                       4 <br />                                    +       +       +       +       +       + <br />  <br />  <br />  <br />  <br />                                                                                HIR2 <br />                                                                                       predicted (98 kDa) from the HIR2 open reading frame, but <br />                                                                                       the size of the Hirl protein (-100 kDa) was larger than <br />                                                                                       expected. As discussed above, this apparent discrepancy <br />  PRTI                                                                          PRT    may result from the electrophoretic behavior of the Hirl <br />  H2A1    -4 <br />           3               ky                           Xu2A1                          gene product. <br />                                                                                          We also utilized the epitope-tagged HIR2 gene to ask <br />   FIG. 6. Northern blot analysis of HIRI (lanes 1 to 4) and HIR2                      whether the levels of Hir2 protein were regulated during the <br /> (lanes 5 to 8) transcripts in strains grown in the absence (-) or                     cell cycle. We visualized the tagged Hir2 protein by Western <br /> presence (+) of hydroxyurea for 30 min. Lanes 1 to 3 and 5 to 7, <br /> wild-type strain FW1238 without plasmid (lanes 1 and 5) or trans-                     blot analysis of wild-type cells arrested with alpha factor or <br /> formed with plasmid YCp5O-HIR1 (lanes 2), YEpHIR1 (lanes 3),                          hydroxyurea and of cdc28, cdc4, cdc7, and cdclS mutants <br /> YCp5O-HIR2 (lanes 6), or YEpHIR2 (lanes 7); lanes 4, strain                           grown at the permissive temperature or shifted for one cell <br /> PSFWA1-A (Ahirl); lanes 8, strain PSFWA2-A (Ahir2). Twenty                            generation to the restrictive temperature. We found that the <br /> micrograms of total RNA extracted from each strain was subjected                      levels of Hir2 protein were unchanged in cells blocked at <br /> to Northern blot analysis. Hybridization probes that detect HIRI,                     these points in G1, S, or G2 phase (data not shown). In <br /> HIR2, HTA1, and PRT1 transcripts were used. The HIRI and HIR2                         addition, the levels of Hir2-ha protein were unchanged in <br /> blots in lanes 3 and 7 were exposed for 1/25 the time of the remaining                strains that contained a mutation in the HIR1 or HIR3 gene <br /> lanes to account for the presence of high-copy-number HIRI and                        (data not shown). The results suggest that the HIR2 gene is <br /> HIR2 genes.                                                                           not regulated at the level of either mRNA or protein synthe- <br />                                                                                       sis. <br /> suppressed by the overproduction of the wild-type product                                                      DISCUSSION <br /> of the second HIR gene (data not shown). We then deter- <br /> mined that the derepressed transcription of an HTAI-lacZ                                 The yeast histone genes show complex transcriptional <br /> reporter gene in a Ahtal-htbl strain (22) was not repressed                           regulation in response to distinct intracellular signals. In the <br /> by the overproduction of either Hirl or Hir2 (data not                                cell cycle, transcription of these genes is dependent upon <br /> shown). Finally, we observed that the derepressed transcrip-                          three different signals that are generated as cells progress <br /> tion of the HTAI gene in a cdc7 mutant could not be                                   from G1 through S phase. Transcription is activated in G, <br /> corrected by the presence of high-copy-number HIRJ or                                 following the completion of the CDC4-dependent step (15, <br /> HIR2 genes (data not shown).                                                          38), and repressed in early S phase subsequent to the <br />    The presence of high-copy-number HIRJ or HIR2 genes                                execution of the late G1, CDC7-dependent step (15). Tran- <br /> also produced no discernible phenotypes in wild-type cells:                           scription is prematurely repressed in S phase in response to <br /> neither gene product conferred a Hir- (Fig. 6, panels 3 and                           a third cell cycle signal that is generated by the interruption <br /> 7) or Spt- (data not shown) phenotype or repressed tran-                              of DNA replication (21). In addition to these temporal <br /> s ription of the HTAI gene when overproduced (33a). To-                               signals, the intracellular levels of H2A and H2B compose a <br /> gether, the data suggest that neither Hirl nor Hir2 is limiting                       fourth signal; transcription of the HTAl-HTBI locus is <br /> for its regulatory function in the cell.                                              repressed when H2A and H2B are in excess, and dere- <br />    Intracellular localization and expression of the Hirl and                          pressed when these histones are limiting (22). We have <br /> Hir2 proteins. To detect the Hir proteins, we tagged them                             identified four HIR genes whose products are involved in <br /> with an epitope derived from the hemagglutinin (ha) protein                           transmitting these signals (26, 37a). Mutations in these genes <br /> of influenza virus (42) as described in Materials and Meth-                           cause transcription of the HTAI-HTBI locus to be indepen- <br /> ods. The tagged Hir proteins were shown to be functional                              dent of each signal, suggesting that the HIR gene products <br /> since a HIRl-ha or HIR2-ha gene carried on a high-copy-                               function downstream in a pathway that links both cell cycle <br /> number plasmid complemented both the Hir- and the Spt-                                progression and intracellular levels of H2A and H2B to the <br /> phenotypes of a hirl or hir2 mutant (data not shown).                                 regulation of the HTAl-HTBI locus. <br /> Indirect immunofluorescence was used to determine the                                    In this study, we have isolated and characterized two HIR <br /> localization of the Hir proteins in cells transformed with a                          genes to learn how their gene products mediate such diverse <br /> high-copy-number HIRl-ha or HIR2-ha gene. The presence                                regulatory signals. We found that the HIR1 and HIR2 genes <br /> of a nuclear targeting signal in both Hir proteins would be                           encode nonessential proteins that are nonetheless required <br /> predicted to confer nuclear localization. No detectable im-                           to repress transcription of the HTAl-HTB1 locus; when <br /> munofluorescence above background was observed for the                                either HIR gene is deleted, transcription of the histone gene <br /> epitope-tagged Hirl protein (data not shown). This may                                pair is not repressed in response to appropriate regulatory <br /> reflect the position of the epitope in the native Hirl protein,                       signals. <br /> a masking of the epitope by the association of Hirl with                                 The differential transcription of the HTAl-HTB1 locus <br /> another protein, or low abundance of the Hirl protein.                                cannot be accounted for by the regulated transcription of <br /> Results for localization of the Hir2-ha protein are shown in                          either HIR gene. Both HIR genes are constitutively tran- <br /> Fig. 7. Although fluorescence was faint, the epitope-tagged                           scribed under a wide variety of conditions in which the <br /> protein was only detected in the nucleus (Fig. 7A and C),                             HTAI-HTBI locus is either repressed or derepressed, and <br /> while a strain expressing a high-copy-number untagged                                 the Hir2 protein is present during the G1, S, and G2 phases of <br /> HIR2 gene showed no nuclear staining (Fig. 7C and D).                                 the cell cycle. Moreover, the pattern of HIRJ and HIR2 <br />   To determine the size of the tagged Hir proteins, we                                mRNA synthesis is not altered by mutations in the SPT1O, <br /> performed Western blot analysis of strains carrying the                               SPT21, HIR3, or HIR4 gene, each of which confers a Hir- <br /> HIRI-ha or HIR2-ha gene (Fig. 8). In each strain, the                                 phenotype (33, 37a). Consistent with the view that the levels <br /> primary antibody recognized a single major antigen (lanes 2                           of the HIRI and HIR2 gene products are not limiting for their <br /> and 4) that was not present in strains carrying an untagged                           regulatory functions, transcription of the HTAI gene is not <br />  36     SHERWOOD ET AL.                                                                                                  MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 7. Nuclear localization of epitope-tagged Hir2 protein. Strain PS2-7A (hir2-1) was transformed with plasmid YEpHIR2-ha (A and B) <br /> or YEpHIR2 (C and D). (A and C) Cells stained with 4&apos;,6&apos;-diamidino-2-phenylindole dihydrochloride to visualize DNA; (B and D) fluorescein <br /> isothiocyanate fluorescence after incubation of cells with monoclonal antibody 12CA5 and fluorescein isothiocyanate-conjugated secondary <br /> antibody as described in Materials and Methods. <br />  <br />  <br /> repressed by overproduction of either the Hirl or the Hir2 <br /> protein. These results suggest that the functions of the Hir                  A.                              B. <br /> proteins may be regulated by posttranslational modification                                11.1   v <br />                                                                                                       -.,N <br />                                                                                                                                NO, <br /> and/or by their association with other proteins. Two poten-                              $+.                              S+ <br /> tial regulators of Hirl or Hir2 function could be the cell cycle                     I <br />  <br />                                                                                    --kO -4.111 <br /> proteins, Cdc4 (a ,B-transducin <span id='am-35' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> protein [see below]), <br /> which is required for activation of HTA1-HTB1 transcrip-                                          - 180                        - 1 80 <br /> tion, and Cdc7 (a protein-serine/threonine kinase), which is <br /> required for HTA1-HTB1 repression.                                                                - 1 16                       - 1 16 <br />    Since mutations in the HIRI or HIR2 gene confer identical <br /> regulatory phenotypes, it is formally possible that the func-                                     -    84                      -   84 <br /> tion of one HIR gene product is to regulate the expression of                                                                  -   S8 <br />                                                                                                   -    5 8 <br /> the second HIR gene. We eliminated this possibility by                                                                         -4 8. 5 <br /> showing that both HIRI and HIR2 mRNAs are produced at                                             -    48.5 <br /> invariant levels in wild-type, hirl, and hir2 strains. We have <br /> also noted that deletion of both the HIRJ and HIR2 genes <br /> does not confer additive effects on the derepressed transcrip- <br /> tion of HTA1 (33a). These data indicate that the Hirl and                FIG. 8. Western blot analysis of epitope-tagged Hirl and Hir2 <br /> Hir2 proteins may act in a common regulatory pathway.                  proteins. Strain W303A1 (Ahirl) was transformed with YEpHIR1 or <br />    The HIRI and HIR2 gene products have been character-                YEpHIR1-ha (A), and strain PS2-7A (hir2) was transformed with <br /> ized genetically as repressors that act through a negative site        YEpHIR2 or YEpHIR2-ha (B). Extracts were prepared, and total <br /> in the HTA1-HTB1 promoter (26). The notion that both                   proteins were separated on SDS-polyacrylamide gels and trans- <br /> proteins might function directly in transcription is supported         ferred to a filter. The filter was incubated with monoclonal antibody <br /> by the presence of a nuclear localization signal in Hirl and           12CA5 and developed as described in Materials and Methods. The <br />                                                                        hirl strain contained an antigen that reacted with the monoclonal <br /> Hir2 and the observation that a Hir2 protein tagged with the           antibody even in the absence of the HIRl-ha gene. Molecular mass <br /> influenza virus hemagglutinin epitope is localized in the              markers a2-macroglobulin (180 kDa), 3-galactosidase (116 kDa), <br /> nucleus. However, the amino acid sequences of Hirl and                 fructose-6-phosphate kinase (84 kDa), pyruvate kinase (58 kDa), and <br /> Hir2 do not contain any of the common motifs associated                fumarase (48.5 kDa) were included as size standards. <br />  <br /> with DNA binding, and HIRI or HIR2 do not encode one of                    <br /> several proteins that bind to specific sequences in the <br /> HTAI-HTB1 negative site or upstream activating sequence                                          <br /> element (22a). These observations raise the possibility that             <br /> Hirl or Hir2 do not confer repression by directly contacting              <br /> the HTAl-HTB1 promoter and suggest that they may func-                       <br />                                                                            <br /> tion as corepressors to assist a repressor that binds to         <br /> sequences in the negative site. In this regard, it is intriguing          <br /> that the predicted Hirl protein contains three copies of the             <br /> ,B-transducin repeat, a motif found in another transcriptional          <br /> repressor, Tupl (40). Tupl also does not bind directly to              <br /> DNA but has been shown to act with another protein, Ssn6                  <br /> (20, 23, 32), to represses specific groups of genes in response             <br /> to signals which determine either the mating type of the cell                <br /> or the utilization of carbon sources (20, 40). The Tupl/Ssn6        <br /> repressor is apparently targeted to particular promoters by         <br /> its association with additional, site-specific DNA binding                 <br /> proteins.                                                                 <br />    The Ssn6 protein is characterized by another motif, the             <br /> TPR snap <span id='am-30' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-31' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33'  ></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>helix</span> (13, 20, 35). Goebl and Yanagida have noted               <br /> that many genes encoding TPR proteins are functionally                <br /> related to those encoding ,B-transducin proteins and have                   <br /> suggested that particular pairs of TPR and 1-transducin                <br /> proteins interact to perform a common function (13). In the                 <br /> case of Tupl and Ssn6, whose biochemical (41) and genetic                <br /> (20) interactions have been demonstrated, one such function            <br /> is to act in conjunction with site-specific DNA binding                   <br /> proteins such as a2 and Mcml to repress transcription of               <br /> a-specific genes (20). It is possible that the Hirl protein also          <br /> acts with a TPR class protein to repress transcription of the           <br /> HTA-HBTBI locus by interacting with site-specific DNA                    <br /> binding proteins at the negative site of this locus. While the            <br /> TPR motif is not found in Hir2, it may occur in another Hir               <br /> protein (e.g., Hir3 or Hir4), since a mutation in such a TPR        <br /> class protein would be predicted to confer a Hir- phenotype.          <br /> Alternatively, one of the known TPR proteins of yeasts (e.g.,           <br /> Cdc16, Cdc23, Ski3, Prp6, or Ssn6) may act together with                  <br /> Hirl to regulate HTAl-HTBI transcription; however, muta-               <br /> tions in none of the genes that encode such proteins confer a            <br /> Hir- phenotype (33a).                                                        <br />    Mutations in HIRI and HIR2 produce pleiotropic effects                  <br /> on transcription. In addition to derepressing transcription of           <br /> the HTAJ-HTBI, HHTI-HHFI, and HH72-HHF2 loci, hir                  <br /> mutations affect the transcription of 8 insertion alleles of          <br /> HIS4 and LYS2 by altering the site of transcription initiation            <br /> (34). It is possible that Hirl and Hir2 represent another class          <br /> of general transcriptional repressors that act at diverse yeast         <br /> promoters. However, we have proposed that the effects of                <br /> mutant HIR gene products on transcription of nonhistone            <br /> gene loci are indirect and are mediated by altered chromatin             <br /> structure that results from nonstoichiometric expression of                <br /> the histone gene loci (34). Thus, some mutations that alter            <br /> transcription pleiotropically may not define general tran-                <br /> scriptional regulators but may act primarily by affecting the              <br /> expression of chromatin components such as histones.                       <br /> </body></html>