<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, July 1989, p. 3081-3087                                                                   Vol. 9. No. 7 <br /> 0270-7306/89/073081-07$02.00/0 <br /> Copyright &#169; 1989, American Society Microbiology <br />  <br />  <br /> A Single Essential Gene, PRI2, Encodes Large Subunit DNA <br />                Primase <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-21'  ></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br />            MARCO FOIANI, CORRADO SANTOCANALE, PAOLO PLEVANI. AND GIOVANNA LUCCHINI* <br />       Dipartirnento di Genetica e di Biologia dei Mlicrorgadisini, Univ&apos;ersitdi di Milano, Viai Celoria 26, 20133 Milani, Ital <br />                                           Received 28 December 1988/Accepted 2 March 1989 <br />  <br />              DNA primase activity yeast DNA polymerase-primase complex related polypeptides, p58 and <br />            p48. The reciprocal role protein <span id='am-59' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> clarified, participate in <br />            formation active center enzyme. The gene encoding p58 subunit cloned screening <br />            Xgtll yeast genomic DNA library, using specific anti-p58 antiserum. Antibodies inhibited DNA <br />            primase activity purified lysates <span id='am-62' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-74'  ></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span>Escherichia coli</span> cells infected recombinant bacteriophage <br />            containing entire gene, designate PRI2. The gene transcribed 1.7-kilobase <br />            mRNA level appeared fluctuate mitotic cell cycle. Nucleotide sequence determination <br />            indicated PRJ2 encodes 528-amino-acid polypeptide calculated molecular weight 62,262. The <br />            gene unique haploid yeast genome, product essential cell viability, shown for <br />            components yeast DNA polymerase-primase complex. <br />  <br />    A DNA polymerase-primase complex highly con-                  essential cell viability directly related DNA <br /> served polypeptide structure number              primase activity (20). Recently, used highly specific <br /> eucaryotic organisms phylogenetically distant yeasts              affinity-labeling procedure map catalytic site yeast <br /> and humans. A complete understanding functions                DNA primase p48 p58 participate in <br /> such complex achieved, <span id='am-93' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-94' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-96'  ></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span>              formation primase active center, ATP- <br /> suggest plays crucial role priming            binding site exclusively located p48 subunit (Sa). <br /> elongation Okazaki fragments initiation             In paper, report isolation characterization <br /> step DNA replication origin (reviewed references            yeast PRI2 gene, encoding p58 polypeptide. We <br /> 4 15). DNA polymerase activity associated               gene unique haploid yeast genome and <br /> highest-molecular-size polypeptide complex (170               transcribed 1.7-kb mRNA appears fluctuate in <br /> 180 kilodaltons [kDa]), smallest polypep-               cell cycle. The gene encodes protein <br /> tides (-60 -50 kDa, respectively) correlated               calculated molecular size 62,262 Da essential <br /> DNA primase activity. A fourth polypeptide, 70 80                 cell viability directly related DNA primase <br /> kDa, function unknown, present            activity, confirming previous biochemical findings. <br /> complex (4, 15). <br />    To better understand functions interactions                          MATERIALS AND METHODS <br /> protein complex essential DNA replication, under-                   Strains <span id='am-98' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-102' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span>. E. coli Y1090 (45) used for <br /> took cloning characterization genes encoding             plaque immunoscreening. E. c oli Y1089 (45) lysogenized <br /> the different protein species Saccharomnvces cerevisiae.             recombinant phages used protein analysis and <br /> The genes coding DNA polymerase I, POLI. (13. 19, 26),              phage DNA preparation. Growth media phage infection <br /> and small primase polypeptide, PRII (20, 30),             conditions described Maniatis et al. (22). Plasmid <br /> cloned sequenced. Sequence comparison revealed                    pGEM4 Promega Biotech, Cambridge, Mass. <br /> consistent amino acid sequence homology putative                    Plasmids pCS10 pCS11 obtained subcloning <br /> catalytic domain located carboxy-terminal half                X215 yeast DNA insert orientations EcoRI <br /> yeast DNA polymerase I viral DNA polym-                  site pGEM4. Plasmid pCS15 constructed inserting <br /> erases. Interestingly, yeast DNA polymerase I human                 HinidIII-ScaI fragment PRI2 5&apos; deletion, obtained <br /> DNA polymerase ox, sequenced polymer-                plasmid pCS11 (see below), HindIII-NruI sites <br /> ases tightly associated DNA primase,              plasmid YIp5 (34). Plasmid pCS1OA67, used synthesize <br /> a higher amino acid homology, extends                 vitro transcript RNase protection mapping 5&apos; <br /> amino-terminal half proteins (26, 43). Sequencing           endpoint PRI2 transcript, derived exonucle- <br /> yeast poll temperature-sensitive mutant altered stability            ase III-digested pCS10 (see below) contains PRI2 <br /> of polymerase-primase complex located mutation               DNA fragment spanning positions -520 + 141 (see Fig. 6). <br /> the conserved amino-terminal region (21, 26, 28),                  Plasmid pMF18 derivative plasmid YEp24 (3) carrying <br /> suggesting polypeptide structure               15-kb yeast DNA insert includes 7.5-kb EcoRI <br /> complex kinds interactions                  genomic PRI2 segment; identified screening a <br /> protein components conserved yeast human cells.              YEp24 yeast genomic library (5) colony hybridization, <br />    In S. cerev&apos;isiae, DNA primase correlates polypep-            using X215 yeast DNA insert probe. <br /> tides, 58 48 kDa (29), roles                    S. cerevisiae S288C (MATot prototroph) DES5 <br /> completely defined. Cloning characterization                 (MATaIMATot ura3-521/ar3-52) previously de- <br /> PRII gene demonstrated p48 polypeptide                  scribed (20). Strain CG378 (MATa ura3-52 leiu2-3,112 trpl- <br />                                                                         289) L. H. Johnston (M.R.C., London, United <br />   * <br />       Corresponding author.                                             <span id='am-90' about='obo:NCBITaxon_kingdom' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-91' property="rdfs:label" content="kingdom" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Kingdom</span>). <br />                                                                  3081 <br />  3082     FOIANI ET AL.                                                                                             MOL. CELL. BIOI- <br />  <br />    Enzyme assays, immunol6gicai reagents, plaque screen-                                A          B <br /> ing. Isolation DNA polymerase I-DNA primase com-                           Mr <br /> plex immunoaffinity chromatography physical sepa-                          (Xt)t 11 AE215       ACB 1 Ji.215; <br /> ration enzyme activities described (29. <br /> 31). DNA primase activity assayed poly(dT)                             116- <br /> template measuring incorporation [a.-32PIdAMP                            97 <br />  <br /> into acid-insoluble material presence 200 U E.                        66                      -   _ <br /> coli DNA polymerase I ml (29). <br />    Polyvalent rabbit antisera DNA polymerase-                               45 <br />  <br />  <br /> primase complex isolated protein species                         36-  - <br />  <br />  <br />  <br />                                                                                                 m <br /> described (20, 31). Antiserum p58 polypeptide                          29- <br />  <br /> was affinity purified absorption specific antibodies                     20- <br /> to p58 immobilized nitrocellulose strips (39). The recov- <br /> ered antibodies used confirm identities              FIG. 1. Analysis proteins encoded X215 genome. <br /> positive recombinant phages plaque lysates transferred         Crude extracts E. coli Y1089 cells lysogenized Xgt1l or <br /> nitrocellulose filters (19). Affinity purification p58 antibod-   X215 grown presence (+) absence (-) 5 mM IPTG <br /> ies plaque proteins phage X215 carried            prepared (45) analyzed 10% sodium dodecyl sulfate- <br />                                                                      polyacrylamide gel. (A) Coomassie blue-stained gel; (B) Western <br /> described previously (20).                                           blot analogous gel probed anti-p58 immunopurified <br />    Identification immunoreactive polypeptides. Lysates        antibodies. Arrows indicate 58- 50-kDa immunoreactive <br /> were prepared E. coli Y1089 cells lysogenized              polypeptides. <br /> Xgtl1 X215 described Young Davis (45) and <br /> were analyzed sodium dodecyl sulfate-polyacrylamide gel                                       RESULTS <br /> electrophoresis (17). Western blotting (immunoblotting)             Identification p58 recombinant phages Xgtl1 expres- <br /> detection immunoreactive polypeptides ap-            sion library. We previously described production of <br /> propriate antiserum performed described            specific antisera raised single protein species of <br /> (19).                                                                DNA polymerase-primase complex use a <br />    DNA sequencing. Plasmids pCSI0 pCS11 di-                 mixture anti-p74. anti-p58. anti-p48 antibodies to <br /> gested SplIl BainHI exonuclease III treat-           screen genomic yeast DNA library Xgtll expression <br /> ment construct set unidirectional deletions 150        vector (20. 27). The positive recombinant phages iden- <br /> base pairs (bp) apart procedure Henikoff (11). T7          tified retested specific antiserum. <span id='am-43' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47'  ></span>This</span> pro- <br /> phage promoter primer used sequence PRI2                  cedure allowed identification classes clones, <br /> region directions pCS10- pCS11-derived            designated p74. p58. p48 basis their <br /> deletion plasmids dideoxynucleotide chain termina-            immunological properties (20). The 10 recombinant phages <br /> tion method (33), using 5&apos;-[cc-35S]thio-dATP, denatured plas-        belonging <span id='am-40' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-41' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> p58 retested anti-p58 affinity- <br /> mid DNA template (10), Klenow fragment E. coli                purified antibodies. Seven clones positively reacted this <br /> DNA polymerase I. The sequences immediately adjacent              second test analyzed. One phages, <br /> the EcoRI linkers X215 yeast DNA insert (45)             X215, shown contain 2.6-kb yeast DNA fragment <br /> confirmed belong PR12 region using appropriate          inserted EcoRI site l(cZ (see Fig. 4). This fragment <br /> synthetic oligonucleotides primers plasmid pMF18.              purified used probe plaque hybridization test, <br /> Protein data base searches DNA sequence analysis            showed seven clones belonged same <br /> performed UWGCG program.                                    cross-hybridization group (data shown). Western analy- <br />    Hybridization procedures techniques. Conditions         sis crude extracts prepared E. (coli cells lysogenized <br /> for Southern blot hybridization described Roeder          seven recombinant phages revealed them <br /> and Fink (32). Northern (RNA) hybridization DNA                 constitutive polypeptides, 50 58 kDa, <br /> probes carried method Maniatis et al. (22)         respectively, specifically reacting anti-p58 antibod- <br />                                                                      ies (Fig. 1). Because extensive degradation foreign <br /> after electrophoresis 1% agarose gel containing 2.2 M           proteins E. (coli strains (19, 44, 45), minor 58-kDa <br /> formaldehyde hybridization 42&apos;C presence            protein species native product yeast DNA <br /> 50Y formamide. Northern blot hybridization RNA                  insert. contain entire p58-coding <br /> probes performed 60&#176;C conditions.              region. transcribed opposite direction <br />    In vitro transcription RNase protection mapping          respect la(cZ. To confirm recombinant phage <br /> carried described Melton et al. (24). Phage            genome encodes p58 primase subunit, we <br /> plasmid DNA preparation, DNA restriction analysis, agar-             determined X215 E. coli lysates used to <br /> ose gel electrophoresis, nick translation DNA probes          purifv anti-p58 antibodies antiserum raised against <br /> (specific activity, 108 cpm/[Lg) described             DNA polymerase-primase complex (31) therefore <br /> previously (22). Total yeast DNA prepared                 reactive polypeptides. X215 affinity-purified <br /> method Winston et al. (42).                                       antibodies reacted p58 protein species on <br />    Total yeast RNA (12) enriched poly(A)&apos; RNA            Western blots complex isolated DNA primase <br /> two-step purification oligo(dT)-cellulose. The filter <span id='am-85' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-86' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-87'  ></span><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>car</span>-      (Fig. 2). Moreover, antibodies inhibited enzyme <br /> rying DNA separated yeast chromosomal bands                 activity tested isolated DNA primase (Table 1), <br /> was kind gift P. Philippsen (Universitait Giessen.            confirming identity X215 yeast DNA insert <br /> Giessen, Federal Republic Germany). Synchronization            involvement p58 subunit DNA primase <br /> CG378 yeast cultures obtained oc-factor (Sigma              activity. <br /> Chemical Co.. St. Louis, Mo.) procedure White et            Characterization p58 gene (PRI2). The X215 yeast <br /> al. (41).                                                            DNA insert used probe Northern blot analysis of <br />  VOL. 9, 1989                                                                        LARGE SUBUNIT OF YEAST DNA PRIMASE                                   3083 <br />  <br />  <br />                        Mr <br />                        Mr                                                                                1        2 3 <br />                                   1      2 3   4   5 <br />                      (xlO0 3 <br />                          180-   bow <br />                          140- <br />  <br />  <br />                           74-         9- <br />                           58- <br />                                                                                                         %W-             NO   -   1.7 kb <br />                           48-            -- <br />                                           4        .&quot; <br />  <br />  <br />  <br />  <br />    FIG. 2. Immunoblot analysis using X215 affinity-purified antibod- <br /> ies. The DNA polymerase-primase complex electrophoresed on <br /> a sodium dodecyl sulfate-polyacrylamide gel stained with <br /> Coomassie blue (lane 1) blotted nitrocellulose filters and <br /> probed antibodies raised complex (lane 3) with <br /> X215 affinity-purified antibodies (lane 4). The analysis was <br /> performed isolated DNA primase: lane 2. stained gel: lane 5.                    FIG. 3. Identification determination direction the <br /> Western blot probed X215 affinity-purified antibodies.                     PRI2 transcript. Poly(A)+ RNA (5 ,ug) strain S288C was <br />                                                                                 probed Northern blots X215 EcoRI yeast DNA insert <br />                                                                                 (lane 1). riboprobe 1 (lane 2). riboprobe 2 (lane 3). <br /> poly(A)+ yeast RNA. This probe identified single 1.7-kb <br /> mRNA species large encode 58-kDa polypeptide                        p58 message opposite direction respect to <br /> (Fig. 3). The X215 yeast DNA insert subcloned                       IlaZ, previously observed Xgtll recombinant <br /> both orientations EcoRI site plasmid pGEM4                         phages (7, 19). Preliminary studies cell cultures synchro- <br /> generate plasmids pCS10 pCS11, transcripts                      nized ox-factor (Fig. 5) indicated steady-state <br /> strand synthesized vitro T7 RNA polymerase                         level 1.7-kb mRNA fluctuated mitotic cell <br /> after linearization plasmids HpaI NuoI,                         cycle reached peak G1-to-S-phase transition, as <br /> respectively (Fig. 4).                                                          previously observed transcripts genes <br />    The transcripts used probe sequentially                      involved DNA replication (14). The identified gene, which <br /> same poly(A)+ RNA blot. The direction transcription                      PR12, unique haploid yeast genome. In fact, <br /> the p58 gene BglII N(oI (Fig. 4),                        Southern blots total yeast DNA restricted E(oRI, <br /> riboprobe 2 hybridized 1.7-kb mRNA identified                       XloI, BamflHI, AvaI, KpnzI showed single bands when <br /> the X215 DNA insert, riboprobe 1 did recognize                      probed X215 DNA insert (data shown). Southern <br /> any mRNA species (Fig. 3). These data agreement                          blot analysis intact yeast chromosomes, fractionated by <br /> with observation X215 does express isopro-                     orthogonal-field gel electrophoresis (35) tested the <br /> py-13-D-thiogalactopyranoside (IPTG)-inducible ,B-galactosi-                    probe, located gene chromosome XI. <br /> dase-p58 fusion protein. In fact, orientation yeast <br /> DNA insert phage genome allows transcription                                   1Kb <br />  <br />  <br />  <br />       TABLE 1. Inhibition DNA primase activity X215 <br />                  affinity-purified antibodies&quot;                                                                -        &gt;~ <br />                                                                                                                        X <br />                                                                                                                                  O     cni&gt;40c     Cc <br />                                                                                          O    Z                    C             OM <br />                                                                                          o    0.   z&apos;         .                                    0 <br />                                           % DNA primase activity&quot; <br /> Antiserum dilution                                 Affinity-purified antibody                                                         * --mRNA <br />                                 Ab mix <br />                                                    Xgtll               X215 <br />                                                                                                                                            I 215 <br />       1:256                      100                100                 100 <br />       1:128                      108                106                  85                                                                RIBOPROBE <br />       1:64                        92                104                  78                                                                RIBOPROBE 2 <br />       1:32                        81                 97                  61 <br />       1:16                        71                 91                  57        FIG. 4. Restriction map PR12 genomic region DNA- <br />       1:8                         60                 85                  47     sequencing strategy. The figure depicts 7.5-kb EcoRI yeast ge- <br />       1:4                         41                 78                  30     nomic DNA fragment containing entire PRJ2 coding sequence <br />       1:2                         23                 66                  17     (C). The map determined restriction analysis X215 DNA <br />                                                                                 plasmid pMF18 (see Materials Methods) confirmed <br />      A 10-ng sample purified DNA primase (29) incubated wPith serial     Southern blot analysis total yeast DNA digested the <br /> dilutions antiserum raised DNA polymerase-primase com-        appropriate enzymes probed X215 2.6-kb EcoRI insert. <br /> plex (Ab mix) Xgt1l X215 affinity-purified antibodies. Dilutions    The size orientation insert indicated ( H-H ) an <br /> made 50 mM Tris hydrochloride (pH 8.0)-l(O mM NaCI-0.1 mg bovine          asterisk denoting position amino-terminal portion the <br /> serum albumin ml- 10% preimmune serum. saimples incubated       1a(cZ gene. The direction PRI2 transcription (the exact 3&apos; endpoint <br /> 0&#176;C 2.5 h microdilution plates precoated 1 mg bovine serum       determined) sizes directions riboprobes <br /> albumin ml. DNA primase activity remaining antiserum tr-eat- <br /> ment assayed described Materials Methods.                         generated vitro transcription X215 yeast DNA insert are <br />    &quot; 100% DNA primase activity corresponds 40 pmol [ot-32PldAMP           shown bottom. Arrows indicate extents and <br /> incorporated 30 min 3TC.                                                  directions sequenced fragments. <br />  3084           FOIANI ET AL.                                                                                                                                             MOL. CELL. BIOL. <br />  <br />                                                                                                              Nucleotide sequence PRI2. As pointed above, the <br />                                                 -e       1-:&apos; 11                4-2.5 Kb <br />                                                                                    - <br />                                                                                                            sizes imnmunoreactive polypeptides identified X215 <br />                                                                                4-1.7 Kt;                   lysates suggested phage genome contain the <br />                                                                                                            entire p58-coding region. Therefore, sequenced X215 <br />                     +                                                                                      yeast DNA insert. A set partially overlapping deletions <br />        BC                                                                                                  generated exonuclease III subcloning this <br />        -,o                                                                                                 DNA fragment orientations plasmid pGEM4 (see <br />        60                                                                                                  Materials Methods). Serial deletions sides, <br />        50                                                                                                  150 bp apart, sequenced dideoxynucle- <br />            ,, <br />        114 0 <br />        30 <br />                                                                                                -           otide chain termination method (33). The restriction sites <br />                                                                                                            deduced corresponding 2,645-nucle- <br />        20 <br />        10                                                                                                  otide sequence (Fig. 6) agreement restriction <br />         0                                                                                                  map established experimentally (Fig. 4). A 531-<span id='am-54' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56'  ></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> open <br />                                                                                                            reading frame terminated TAG stop codon found <br />                         30 60 90 120 150 180 210 240 270 300C                                              sequence (the next-longest reading frame en- <br />                                Time (miinutes&apos;                                                             coded 86 amino acids). The direction transcription <br />                                                                                                            identified previously Northern <br />    FIG. 5. Fluctuation level PR12 transcript                                      hybridization (see above). The ATG longest open <br /> cell cycle. x-Factor added time zero (+) cell cultures                                        reading frame located 239 bp upstream PvuI <br /> strain CG378 logarithmically growing synthetic medium                                           site, second ATG 244 bp downstream <br /> removed filtration 125 min (-). The percentage budded                                          site. We did invariant signal sequence <br /> cells indicated times determined light microscopy. At <br /> the Northern blot total RNA (5,ug lane) samples                                       TACTAAC, present introns yeast genes <br /> taken times probed X215 yeast DNA insert                                          transcribed RNA polymerase 11 (18). Given transla- <br /> and EcoRI-SallI 1.6-kb yeast DNA fragment hybridizing                                         tion starts ATG (position +1 Fig. 6) that <br /> 2.5-kb transcript, previously shown fluctuate                                             gene does contain introns, identified open reading <br /> during cell cycle (14). <br />  <br />                                                                                                                                        CGGGTGACGGGACACCGCTACGAAT            -496 <br />                ATAGTACGATACACTTATACATATATGGAAGCATTTATAGCGTATATTCATCGTACGGTACATTCTTTAAATAAATGATTGAATTTTTCTGAACCAACA                                                          -397 <br />                GTTATAAGCTGAGATCCAGTGCGTTCATGAAACGGAACATCTTACATCATCAAGAGCCCAGAAGCACGGCGTAGGATTGTTTCCGCTGCTACAGACAGA                                                          -298 <br />                CATGGAAACACTGTTTTTATCAGGCCAAAGGTGATACGACAGGTCTCTTCTTGTTTCAATATTGCCTTACATATCTGCTGCTCACTCTATTTGCTTTTT                                                          -199 <br />                CCTGCTCATTTATAGTTAACTTGATTTGAAATTACGCGTCGCGGGGAATCGCGTAAAACTCGATATTAAATCCTGTGATTTTGCAAAAGTAAATTTTTT                                                          -100 <br />                ACTATCTACATCTCTCCAATTTGCACAACTTTGACAGGACTGATAACTACCCAGGTGGAGCAACCAGTGACAGCAGGTAAAGCACAATAAGGAGAAGAA                                                            -1 <br />                 N F R Q S K R R         I A S R     K N F S S Y D D I V K S E L D V G N T N A                                                                                 33 <br />                ATGTTCAGGCAGTCAAAAAGAAGAATTGCCTCAAGGAAGAATTTTAGTTCATACGATGATATTGTCAAAAGCGAATTGGACGTAGGCAACACAAATGCA                                                           +99 <br />                 A N Q I I L S S S S S E E E K K L Y A R L Y E S K L S F Y D L P P                                                                                             66 <br />                GCTAACCAGATCATCTTAAGTTCAAGTTCATCTGAAGAAGAGAAAAAACTTTACGCTAGGCTGTATGAGTCTAAATTATCATTTTACGATTTACCACCA                                                          +198 <br />                 Q G E I T L E Q F E I W             A I D R L K I L L E I E S C L S R                   N K S I                                                               99 <br />                CAGGGAGAAATAACGTTGGAACAATTTGAGATCTGGGCAATCGATCGTCTCAAAATTCTGTTAGAGATCGAATCTTGCCTTTCAAGAAACAAATCGATC                                                          +297 <br />                 K E I E T I I K P Q F Q K L L P F N T E S L E D                         R   K K D Y Y S H F                                                                  132 <br />                AAAGAAATTGAAACCATTATCAAGCCGCAGTTCCAAAAACTACTGCCTTTCAACACAGAGAGTTTAGAAGACAGAAAAAAGGATTACTACTCTCATTTC                                                          +396 <br />                 I   L   R   L   C F     C R          S      K      E   L   R   E       K   F       V   R   A   E T     F   L   F   K     I   R   F   N   N   L   T   S       165 <br />                ATTTTAAGATTGTGCTTTTGTCGTTCAAAAGAACTGCGGGAAAAATTCGTCAGGGCTGAAACGTTTTTGTTCAAAATTAGATTTAACATGCTAACCTCC                                                          +495 <br />                 T   D   Q T K F V Q S L D L P L L Q F I S N E E K A E L S H Q L Y Q T                                                                                        198 <br />                ACCGATCAAACAAAATTTGTTCAATCATTGGACTTACCCTTACTACAGTTTATCTCAAATGAAGAAAAGGCCGAATTATCTCATCAGTTGTATCAAACA                                                          +594 <br />                 V S A S L Q F Q L N L N E E H Q R K Q Y F Q Q E K F I K L P F E N                                                                                            231 <br />                GTTTCTGCGTCTCTACAGTTCCAATTGAATTTAAACGAGGAACATCAAAGAAAACAGTATTTCCAACAGGAAAAATTCATAAAATTACCTTTCGAAAAT                                                          +693 <br />                 V   I   E   L   V G     N   R        L      V      F   L   K   D       G   Y       A   Y   L   P   Q   F   Q   Q   L     N   L   L   S   N   E   F   A       264 <br />                GTGATAGAACTGGTAGGAAACCGTTTAGTGTTTTTGAAGGACGGGTACGCATATTTACCACAATTCCAGCAATTGAATTTACTTTCTAATGAGTTTGCT                                                          +792 <br />                 S   K   L   N   Q   E   L   I        K T           Y   Q   Y   L       P R         L   N   E   D   D   R   L   L   P     I   L   N   H   L   S   S   G       297 <br />                AGCAAATTGAACCAGGAGTTAATAAAAACGTACCAGTACTTACCAAGACTGAATGAGGATGACCGGTTGTTACCAATTCTAAATCATCTTTCGTCGGGG                                                          +891 <br />                 Y   T   I   A   D   F   N   Q        Q      K      A   N   Q   F       S   E       N   V   D   D   E   I   N   A   Q     S   V W     S   E   E   I   S       330 <br />                TACACTATCGCGGATTTCAACCAGCAAAAGGCAAACCAATTCAGTGAGAACGTAGATGATGAGATTAATGCGCAAAGTGTCTGGTCTGAAGAGATTAGC                                                          +990 <br />                 S   N   Y   P   L   C   I   K        N       L     N   E G     L       K   K       N   H   H   L R     Y   Y   G R       Q   Q   L   S   L   F   L   K       363 <br />                TCAAACTATCCGCTATGTATCAAAAACCTGATGGAGGGTCTTAAAAAGAACCATCATTTGAGGTATTATGGGAGACAACAACTGAGTCTGTTTTTGAAA                                                         +1089 <br />                 G   I   G   L   S   A   D   E        A       L     K   F W     S       E   A       F   T   R   N   G   N   N   T   N     E   K   F   N   K   E   Y   R       396 <br />                GGTATCGGCCTGAGCGCTGATGAAGCGTTAAAATTTTGGTCTGAGGCATTTACAAGAAATGGGAACATGACAATGGAGAAGTTCAATAAAGAATACCGT                                                         +1188 <br />                 Y S F R         H   N   Y G   G      L E           P W D C H T I L S K P R P G R G <br />                                                                        N R     I       N   Y       K                                                                         429 <br />                TACAGCTTCAGGCATAATTACGGTCTTGAAGGTAACAGAATCAACTACAAACCATGGGACTGTCACACTATCCTTTCCAAGCCCAGACCTGGGCGCGGA                                                         +1287 <br />                 D Y H G C P F R D W S H E R L S A E L R S N K L T Q A Q I I S V L                                                                                            462 <br />                GATTATCATGGATGCCCTTTCCGTGATTGGAGTCACGAGAGACTATCTGCAGAATTGCGCTCTATGAAGTTGACCCAAGCGCAGATCATCAGTGTTCTG                                                         +1386 <br />                 D   S   C   Q   K   G   E   Y        T       I     A   C   T   K       V   F       E M     T   H   N   S   A   S   A     D   L   E   I   G   E   Q T         495 <br />                GATTCCTGCCAGAAAGGTGAATACACAATTGCTTGCACTAAAGTGTTTGAAATGACACACAACTCTGCATCAGCGGATTTGGAAATTGGCGAGCAAACG                                                         +1485 <br />                 H   I   A   H   P   N   L   Y        F       E     R   S   R   Q       L   Q       K   K   Q   Q   K   L   E   K   E     K   L   F   N   N G     N   H       528 <br />                CATATCGCGCATCCTAACCTATACTTCGAAAGATCAAGGCAACTGCAAAAGAAACAGCAGAAGCTGGAAAAGGAAAAACTATTCAATAATGGTAATCAT                                                         +1584 <br />                tAGTGCAGAAAGGGAAAAGGAAAAAAGCGAAGAGATAACTAAAAAGGATTAATAGTATACGTAATTTAGTCTACTGTAAGTATGGTCAAAAATTTTCTT +1683 <br />                TAGTGAGAAAAAAAAAAAAAAAGAGAAAGCATAAACGCTGTACAATACCAGTGACTGCACTGCTCCGAAAACGTCCGAGATATGTATATTGTCCGAATT +1782 <br />                TTTTCATACTTTAATAGCTGGCCAGAGAAACCTTATAATTTTGTGGCTGCAAGCTATCGCGCTGTTCC.TCTGTGACAGGCAACACAGAACAACTAACCG                                                        +1881 <br />                CCTGTCTTTTATGCCTGTACACTTCATTAAAATTTACCGTTTTCTTCCATTACAAGCGGGGACACTTTCCGTTTAAACCTGCAATATGTTGCAGGGCCT                                                         +1980 <br />                ATCCACGCCAATTTACTGTGGCAGAAGTTACTGCAGTGGAAGAACTACCTTTTCCAGCATAGCCTCCGCCACCAAAAAATGCAGGTGCTGCGTTTTCTG                                                         +2079 <br />                GATACGTGGCTGGCCTTTAAAACGCATGGGAGGGACAGTATCTCAC <br />    FIG. 6. Nucleotide sequence deduced amino acid sequence PR12. The nucleotide sequence numbered ATG initiation <br /> codon PR12 open reading frame (position + 1). The amino acid sequence translated DNA sequence; amino acids (in <br /> single-letter code) numbered methionine. Sequences 5&apos; noncoding region POL1 (see text) are <br /> underlined. -*, Putative positions major 5&apos; endpoints PR12 transcript. <br />  VOL. 9, 1989                                                             LARGE SUBUNIT OF YEAST DNA PRIMASE                        3085 <br />  <br />  <br />                                                                                         TI                        T2 <br />              C A A TG              1 2 3 <br />  <br />  <br />  <br />  <br />                                                                              A                         B <br />  <br />  <br />  <br />  <br />   FIG. 7. RNase protection mapping 5&apos; endpoint the <br /> mature PR12 mRNA. The 5&apos; termini PRI2 transcript were <br /> analyzed RNase protection assay (24) 32P-labeled RNA <br /> probe complementary PR12 mRNA spanning positions                                                        (I <br />  <br />  <br />  <br />  <br /> -520 +141. Lanes 1. 2. 3 results obtained 10. <br /> 20, 40 jig yeast poly(A)+ RNA. respectively. A ladder a <br /> known DNA sequence run parallel used size marker. The <br /> four major signals (*-) correspond positions -69 5. -60 + 5. <br />                                                     &#177; <br />  <br /> -52 5. -49 5 ATG. <br />                    &#177;                                                   FIG. 8. Spore lethality morphology disruption PRI2. <br />                                                                      The upper shows spore survival tetrads deriving from <br />                                                                      sporulation strain DESS heterozygous integration of <br /> frame codes 528-amino-acid polypeptide calcu-           plasmid pCS15 PR12 (Ti) URA3 (T2) locus. Tetrads were <br /> lated molecular weight 62,262, good agreement         dissected YPD plates (37) incubated 25&#176;C. Panels A <br /> with value 58,000 estimated electrophoretic          D pictures. taken light microscope 20 h after <br />                                                                      dissection. Ti tetrad. Spore survival scored 3 days. <br /> mobility p58 polypeptide polyacrylamide gel         viable spores tested uracil requirement. (A C) <br /> presence sodium dodecyl sulfate (29). The AUG context             Viable Ura- spores: (B D) inviable spores. <br /> does correspond consensus highly <br /> expressed yeast genes, 5&apos;-AAAAAAAUGUC-3&apos; (8). Fur- <br /> thermore, codon usage p58 gene reveals little, any.        The similarity POLl striking, the <br /> codon <span id='am-1' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-2' property="rdfs:label" content="Bias" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5'  ></span>bias</span>, usually seen poorly expressed yeast            -165 hexanucleotide immediately followed <br /> genes (36). Both observations agreement              direct repetition 5&apos;-CGCG-3&apos;, the <br /> finding p58 low abundance (29).                           POLI repeats. <br />    The p58 protein 35.8% nonpolar, 35.1% polar, 16.3%               Disruption PRI2 gene lethal yeast cells. Since we <br /> positively charged, 12.8% negatively charged amino               showed PR12 gene product related DNA <br /> acids, represents random distribution            primase activity, expect essential cell <br /> types amino acids. The hydrophilic profile p58             viability. Therefore, carried gene disruption exper- <br /> polypeptide (16) indicates protein rich          iment procedure Shortle et al. (38). <br /> hydrophilic regions. The secondary structure predicted               A DNA fragment internal p58-coding region (span- <br /> computer-assisted analysis (6) shows regions              ning segment positions +141 +829) sub- <br /> a-helical structure 10 amino acids. A <span id='am-35' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-36'  ></span><span id='am-37' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>helix</span>-              cloned yeast integrating vector YIp5 (34), the <br /> turn-helix motif, DNA-binding proteins              derivative plasmid pCS15 used disrupt PRI2 gene. <br /> (25), located residues 210 239.                             The plasmid linearized digestion BglII SinaI <br />    RNase-mapping experiments (24) revealed presence           direct integration PRI2 URA3 locus, respec- <br /> four major transcription starts, positions -49 5. -52&#177;       &#177;    tively, used transform diploid strain DES5 to <br /> 5, -60 5, -69 5 ATG (Fig. 7). No <br />          &#177;                                                           uracil prototrophy. Transformants heterozygous integra- <br /> convincing TATA box upstream major                tion single copy plasmid loci <br /> start sites, weakly expressed yeast genes         identified Southern blot analysis total yeast DNA <br /> not clearly recognizable TATA homology. The 5&apos;-               digested EcoRI probed X215 DNA insert. <br /> flanking region contained short repeated DNA se-             One transformant class sporulated, tetrad <br /> quences, relevant control              analysis performed. Integration pCS15 PR12 <br /> transcription PRI2 gene genes            locus lethal spores (Fig. 8). In fact, tetrads <br /> involved DNA metabolism. In particular, sequence              derived diploid heterozygous integration <br /> 5&apos;-ACGCGT-3&apos;, present twice POLI (26)            showed 2:2 lethality linked URA3 gene of <br /> CDC21 (40) 5&apos; noncoding regions CDC9 (1)             plasmid, viable spores Ura-. The <br /> CDC8 (2), PRI2 position -165, similar          plasmid did contain sequences harmful se the <br /> sequence, 5&apos;-TCGCGT-3&apos;, located 15 bp downstream.                cells, spore lethality caused integration at <br />  3086    FOIANI ET AL.                                                                                              MOL. CELL. BIOL. <br />  <br /> the URA3 locus. Therefore, spore lethality         PRII, similar kinetics fluctuation tran- <br /> the absence functional p58 polypeptide. All           scripts cell cycle (reviewed reference 14). <br /> inviable germinated spores appeared single, elongated             The 520-bp sequence upstream PRI2 coding region <br /> cells, eventually buds (Fig. 8). The         contains hexanucleotide 5&apos;-ACGCGT-3&apos;, pres- <br /> morphology, typical mutants impaired DNA           ent 5&apos; noncoding regions genes <br /> replication (9), observed germinated spores        listed (23, 26). Analysis 5&apos; noncoding region of <br /> containing disrupted copy PRII gene (20).                CDC21, contains 5&apos;-ACGCGT-3&apos; repeats, indi- <br />                                                                   cates 37-bp deletion, removing region between <br />                         DISCUSSION                                reconstituting single hexamer, severely affects the <br />                                                                   level CDC21 transcript possibly impairs fluctuation <br />    Eucaryotic DNA replication highly regulated process       cell cycle (23). We investigating whether <br /> requiring precise coordination protein-protein     6-bp sequence role mediating transcriptional <br /> and protein-DNA interactions (4, 15). One essential compo-        control POLl, PRII, PRI2. <br /> nent replicative apparatus DNA polymerase-             The PRI2 gene present single copy haploid <br /> primase complex, generally considered         yeast genome, localized chromosome XI. <br /> the enzymatic machinery involved synthesis          Disruption gene lethal cell; therefore, p58 <br /> Okazaki fragments lagging strand replication        polypeptide performs essential function yeast cells, as <br /> forks (4, 15). A prerequisite understanding physical       expected protein species integral component <br /> structure protein complex examining mecha-        yeast DNA primase. The production conditional lethal <br /> nisms controlling expression structural genes   mutants vitro mutagenesis PRII and <br /> cloning genes encoding DNA polymerase-            PRI2 genes eventually establish roles DNA <br /> primase polypeptides. Cloning sequencing yeast         primase vivo helpful identifying structural <br /> genes POLI (DNA polymerase I) PRII (48-kDa subunit            domains important functional interactions between <br /> of DNA primase) described (13, 19, 20, 26, 30). In      different protein components polymerase-primase <br /> this report, provide information genes       complex. Moreover, possibility obtaining large <br /> coding components yeast DNA polymerase-            amounts proteins overexpression cloned <br /> primase complex describing isolation partial           genes homologous heterologous systems will <br /> characterization gene encoding large subunit        allow integration genetic data detailed <br /> DNA primase (58 kDa), called PRI2.                       biochemical characterization system. <br />    The PRI2 gene identified using specific <br /> anti-p58 antibodies immunological screening yeast <br /> genomic DNA library Xgtll (45). The 2.6-kb yeast DNA                                ACKNOWLEDGMENTS <br /> insert positive recombinant phage X215 appears         We thank Marco Muzi Falconi assistance analysis <br /> to contain entire coding region gene,         Simonetta Piatti help synchronization yeast cell <br /> transcribed opposite direction respect lcZ. In     cultures. <br /> fact, crude extracts prepared E. coli cells lysogenized         M.F. supported fellowship Fondazione A. Buzzati <br /> with X215 contain 58-kDa immunoreactive polypeptide.            Traverso. <br /> which rapidly converted 50-kDa protein species; the <br /> concentrations proteins increased IPTG-                              LITERATURE CITED <br /> treated cells. Lysates X215-infected cells affinity         1. Barker, D. G., J. H. M. White, L. H. Johnston. 1985. The <br /> purify p58 antibodies specifically inhibit DNA primase           nLucleotide sequence DNA ligase gene (CDC9) Saccha- <br /> activity, confirming identity gene       roniv es cereviisiae: gene cell cycle regulated and <br /> also correlation p58 polypeptide DNA pri-             induced response DNA damage. Nucleic Acids Res. <br /> mase.                                                                 13:8323-8337. <br />                                                                    2. Birkenmeyer, L. G., J. C. Hill, L. B. Dumas. 1984. Saccha- <br />    The X215 yeast DNA insert identifies single 1.7-kb               romyiv(cs cerei&apos;isiae CDC8 gene product. Mol. Cell. Biol. <br /> mRNA species used probe Northern blots                  4:583-590. <br /> yeast poly(A)+ RNA. This size sufficient encode            3. Botstein, D., S. C. <span id='am-48' about='obo:NCBITaxon_8952' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_8949 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_8948 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51'  ></span><span id='am-52' property="oboInOwl:hasExactSynonym" content="falcons" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Falco" datatype="xsd:string"></span>Falco</span>, S. E. Steward, M. Brennen, S. Scherer, <br /> 58-kDa polypeptide, direction transcription             D. T. Stinchcomb, K. Struhl, R. W. Davis. 1979. Sterile host <br /> from BglII NcoI, expected results A215              yeast (SHY): eukaryotic biological containment for <br /> protein analysis.                                                     recombinant DNA experiments. Gene 8:17-24. <br />   Nucleotide sequence determination X215 DNA                4. Campbell, J. L. 1986. Eukaryotic DNA replication. Annu. Rev. <br /> insert confirmed contains entire PRI2 coding              Biochem. 55:733-771. <br />                                                                    5. Carlson, M., D. Botstein. 1982. Two differentially regulated <br /> region, translated 528-amino-acid polypeptide           mRNAs different 5&apos; ends encode secreted intracellular <br /> with calculated molecular weight 62,262. The protein          forms yeast invertase. Cell 28:145-154. <br /> very rich hydrophilic regions, predicted secondary      Sa.Foiani, M., A. J. Lindner, G. R. Hartmann, G. Lucchini, P. <br /> structure shows helix-turn-helix motif,         Plevani. 1989. Affinity labeling active center ribonu- <br /> DNA-binding proteins (6). Computer-assisted analysis did              cleoside triphosphate binding site yeast DNA primase. J. <br /> not significant homology p58 known             Biol. Chem. 264:2189-2194. <br /> proteins. Only procaryotic DNA primases com-              6. Garnier, J., D. J. Osguthorpe, B. Robson. 1978. Analysis of <br /> pared, PRI2 eucaryotic gene encoding           accuracy implication simple methods predicting <br /> large primase subunit cloned sequenced.             secondary structure globular proteins. J. Mol. Biol. <br />                                                                       120:97-120. <br />   Preliminary analysis steady-state level 1.7-kb     7. Goto, T., J. C. Wang. 1984. Yeast DNA topoisomerase II is <br /> mRNA synchronized yeast cells indicates level          encoded single-copy, essential gene. Cell 36:1073-1080. <br /> PRI2 transcript varies cell cycle, reaching peak      8. Hamilton, R., C. K. Watanabe, H. A. Boer. 1987. <br /> at G1-to-S-phase transition. Interestingly, DNA             Compilation comparison sequence context around <br /> synthesis genes, CDC8. CDC9, CDC21, POLI,                 AUG start codons SaccharoinNces cerev&apos;isiae mRNAs. <br />  VOL. 9, 1989                                                                  LARGE SUBUNIT OF YEAST DNA PRIMASE                           3087 <br />  <br />     Nucleic Acids Res. 15:3581-3593.                                          polymerase-DNA primase complex. p. 341-346. In T. Kelly and <br />  9. Hartwell, L. H. 1971. Genetic control cell division cycle       B. Stillman (ed.). <span id='am-30' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-32'  ></span><span id='am-33' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Cancer</span> cells, vol. 6. Cold Spring Harbor <br />     yeast. J. Mol. Biol. 59:183-194.                                          Laboratory. Cold Spring Harbor, N.Y. <br /> 10. Hattori, M., Y. Sasaki. 1986. Dideoxy sequencing method           28. Plevani, P., M. Foiani, M. Muzi Falconi, A. Pizzagalli, C. <br />     using denatured plasmid templates. Anal. Biochem. 152:232-                Santocanale, S. Francesconi, P. Valsasnini, A. Comedini, S. <br />     238.                                                                      Piatti, G. Lucchini. 1988. The yeast DNA polymerase- <br /> 11. Henikoff, S. 1984. Unidirectional digestion exonuclease III          primase complex: genes proteins. Biochim. Biophys. Acta <br />     creates targeted break points DNA sequencing. Gene 28:                951:268-273. <br />     351-359.                                                              29. Plevani, P., M. Foiani, P. Valsasnini, G. Badaracco, E. Cheria- <br /> 12. Hinnebusch, A. G., G. R. Fink. 1983. Repeated DNA                     thundam, L. M. S. Chang. 1985. Polypeptide structure of <br />     sequences upstream HISI occur                  DNA primase yeast DNA polymerase-primase complex. <br />     coregulated genes Saccharomvces cerei isiae. J. Biol. Chem.            J. Biol. Chem. 260:7102-7106. <br />     258:5238-5247.                                                        30. Plevani, P., S. Francesconi, G. Lucchini. 1987. The nucleo- <br /> 13. Johnson, L. M., M. Snyder, L. M. S. Chang, R. W. Davis,               tide sequence PRII gene related DNA primase in <br />     J. L. Campbell. 1985. Identification gene encoding yeast           Saccharonixvces cerei&apos;isiae. Nucleic Acids Res. 15:7975-7989. <br />     DNA polymerase I. Cell 43:369-377.                                    31. Plevani, P., G. Lucchini, M. Foiani, P. Valsasnini, A. Brandazza, <br /> 14. Johnston, L. H., J. H. M. White, A. L. Johnson, G. Lucchini,              M. Bianchi, G. Magni, G. Badaracco. 1987. Structure and <br />     P. Plevani. 1987. The yeast DNA polymerase I transcript            function yeast DNA polymerase I-DNA primase com- <br />     regulated mitotic cell cycle meiosis       plex. Mol. Genet. 6:53-60. <br />     induced DNA damage. Nucleic Acids Res. 15:5017-5030.            32. Roeder, G. S., G. R. Fink. 1980. DNA rearrangements <br /> 15. Kaguni, L. S., I. R. Lehman. 1988. Eukaryotic DNA                     associated transposable element yeast. Cell 21:239- <br />     polymerase-primase: structure, mechanism function. Bio-               249. <br />     chim. Biophys. Acta 950:87-101.                                       33. Sanger, F., S. Nicklen, A. R. Coulson. 1977. DNA sequenc- <br /> 16. Kyte, J., R. F. Doolittle. 1982. A simple method                  ing chain-terminating inhibitors. Proc. Natl. Acad. Sci. <br />     displaying hydropathic character protein. J. Mol. Biol.          USA 74:5463-5467. <br />     157:105-132.                                                          34. Scherer, S., R. W. Davis. 1979. Replacement chromo- <br /> 17. Laemmli, U. K. 1970. Cleavage structural proteins           segments altered DNA sequences constructed in <br />     assembly head bacteriophage T4. Nature (London)                 *itro. Proc. Natl. Acad. Sci. USA 76:4951-4955. <br />     227:680-685.                                                          35. Schwartz, D. C., C. R. Cantor. 1984. Separation yeast <br /> 18. Langford, C. J., F. J. Klinz, C. Donath, D. Gallwitz. 1984.           chromosome-sized DNAs pulsed field gradient gel electro- <br />     Point mutations identify conserved. intron-contained TAC              phoresis. Cell 37:67-75. <br />     TAAC box essential splicing sequence yeast. Cell             36. Sharp, P. M., T. M. F. Tuohy, K. R. Mosurski. 1986. Codon <br />     36:645-653.                                                               usage yeast: cluster analysis clearly differentiates highly and <br /> 19. Lucchini, G., A. Brandazza, G. Badaracco, M. Bianchi, P.              lowly expressed genes. Nucleic Acids Res. 14:5125-5143. <br />     Plevani. 1985. Identification yeast DNA polymerase I           37. Sherman, F., G. R. Fink, J. B. Hicks. 1986. Methods in <br />     gene antibody probes. Curr. Genet. 10:245-252.                       yeast genetics: laboratory course manual. Cold Spring Harbor <br /> 20. Lucchini, G., S. Francesconi, M. Foiani, G. Badaracco, P.             Laboratory, Cold Spring Harbor, N.Y. <br />     Plevani. 1987. The yeast DNA polymerase-primase complex:              38. Shortle, D., J. E. Haber, D. Botstein. 1982. Lethal disruption <br />     cloning PRII, single essential gene related DNA primase           yeast actine gene integrative DNA transformation. <br />     activity. EMBO J. 6:737-742.                                              Science 217:371-373. <br /> 21. Lucchini, G., C. Mazza, E. Scacheri, P. Plevani. 1988.            39. Smith, D. B., P. A. Fisher. 1984. Identification, develop- <br />     Genetic mapping Saccharonvyces cereviisiae DNA poly-               mental regulation, response heat shock antigeni- <br />     merase I gene characterization poll temperature-                 cally related forms major nuclear envelope protein in <br />     sensitive mu <br /> </body></html>