<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Feb. 1991, p. 721-730                                                                    Vol. 11, No. 2 <br /> 0270-7306/91/020721-10$02.00/0 <br /> Copyright C 1991, American Society Microbiology <br />  <br />  <br />       Characterization RPRI, Essential Gene Encoding RNA <br />          Component <span id='am-39' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-49'  ></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> Nuclear RNase P <br />            JAE-YONG LEE, CHRISTOPHER E. ROHLMAN, LISA A. MOLONY, AND DAVID R. ENGELKE* <br />              Department Biological Chemistry, The University Michigan, Ann Arbor, Michigan 48109-0606 <br />                                         Received 27 August 1990/Accepted 8 November 1990 <br />  <br />               RNA components identified preparations RNase P number eucaryotic sources, but <br />            final proof RNAs true RNase P subunits elusive eucaryotic RNAs, unlike <br />            procaryotic RNase P ribozymes, shown catalytic activity absence protein. <br />            We previously identified RNA component <span id='am-56' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-61'  ></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae nuclear RNase P preparations <br />            characterized corresponding, chromosomal gene, called RPRI (RNase P ribonucleoprotein <br />            1). Gene disruption experiments showed RPR1 single copy essential. Characterization gene <br />            region located RPRI 600 bp downstream URA3 coding region chromosome V. We sequenced 400 <br />            bp upstream 550 bp downstream region encoding major 369-nucleotide RPRI RNA. The <br />            presence abundant, potential precursor RNAs extra 84 nucleotides 5&apos; leader 30 <br />            nucleotides 3&apos; trailing sequences suggests primary RPRI transcript subjected multiple <br />            processing steps obtain 369-nucleotide form. Complementation RPRI-disrupted haploids one <br />            variant RPRI gave slow-growth temperature-sensitive phenotype. <span id='am-77' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-81'  ></span>This</span> strain accumulates tRNA <br />            precursors lack 5&apos; end maturation performed RNase P, providing direct evidence RPRI RNA <br />            essential component enzyme. <br />  <br />   RNase P procaryotic eucaryotic sources                essential gene encoding RNA identified charac- <br /> an endonuclease cleaves pre-tRNA substrates yield                terized. Disruption gene, termed RPRI (RNase P <br /> mature 5&apos; termini. In procaryotes, RNA protein              ribonucleoprotein 1), complementation plasmid- <br /> components ribonucleoprotein enzymes required                 borne variant defective growth leads accumulation of <br /> for vivo activity (15, 27, 29, 33, 43, 49, 57, 58),           tRNA precursors lacking 5&apos; RNase P cleavage. This <br /> RNA component capable efficiently catalyzing             observation confirms identity S. cerevisiae RPRI <br /> correct reaction conditions vitro (3, 18, 21, 22,          RNA essential component RNase P suggests <br /> 52). Extensive phylogenetic sequence comparisons                similar RNAs isolated eucaryotes also <br /> RNAs (30), combined folding energy calculations                 bona fide RNase P components. <br /> cleavage sensitivity studies (6, 19, 51, 52), suggest con- <br /> served, highly ordered secondary structure. Most of <br /> the key contacts substrates depend struc-                             MATERIALS AND METHODS <br /> ture, protein contributing efficiency                  Strains genetic methods. S. cerevisiae W3031A (MA Ta <br /> secondary effects charge shielding RNA                   ade2-1 his3-11,15 leu2-3,112 trpl-J ura3-1 cani -100) the <br /> chain phosphate backbones. The mechanism pre-                   isogenic strain W3031B (MATot ade2-1 his3-11,15 leu2-3,112 <br /> tRNA substrates recognized clearly understood,                trpl-l ura3-1 can1-100) obtained E. Esposito <br /> however, obvious regions required                  (originally R. Rothstein) used gene disruption, com- <br /> Watson-Crick base pairing enzyme sub-                    plementation analysis, Southern Northern (RNA) <br /> strate RNAs (20, 36).                                                    analyses. Strains BJ411 (MATTa his] prbl-1122 prel-126), <br />    Studies eucaryotic RNase Ps suggested               previously referred EJ101 (31; gift E. Jones), and <br /> also contain essential RNA subunits, role            PP1002 (MATTa ade2 leu2-3,112 pep4-3 rna3 rna82) (46; gift <br /> RNA components firmly established. RNAs                     P. <span id='am-1' about='obo:NCBITaxon_13215' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_16736 obo:NCBITaxon_131567 obo:NCBITaxon_16739 obo:NCBITaxon_3398 obo:NCBITaxon_232347 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-2' property="rdfs:label" content="Piper" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Piper</span>) used preparation RNase P end <br /> that copurify nuclear organelle RNase Ps              analysis RPRI RNA transcript. Strain YPH149 <br /> been characterized (1, 7, 9, 11, 12, 14, 24, 32, 34, 35, 37, 38,         (MATa ade his7 lys2 ura3-52 tri-di::TRPJ+) obtained <br /> 40, 42) shown sequence structural similar-               P. Hieter used chromosome mapping. Yeast <br /> ities to lesser degree procaryotic            transformation performed lithium acetate proce- <br /> RNAs (5, 37). The identification eucaryotic RNAs               dure (28). Sporulation tetrad analysis performed by <br /> essential RNase P subunits rigorously con-                  using standard genetic methods (59). Cells grown in <br /> firmed, however, reports eucaryotic            selection medium contained 0.67% yeast nitrogen <br /> RNAs having enzyme activity absence protein                base amino acids, 2% dextrose amino acids for <br /> some studies report RNase Ps essential RNA                       selection, YPD medium (2% dextrose, 2% peptone, 1% <br /> component discernible (9, 63).                                           yeast extract). <br />    We previously reported sequence 369-nucleotide                 Cloning RPRI gene. A yeast genomic DNA library in <br /> RNA copurifies nuclear RNase P activity                   centromere-containing <span id='am-118' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-119' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-122' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> (YCp5O library) ob- <br /> Saccharomyces cerevisiae. In report, single-copy,                 tained D. Thiele (54; originally D. Botstein G. <br />                                                                          Fink) screened colony hybridization (2). The <br />                                                                          probe used screening prepared vitro <br />   * <br />       Corresponding author.                                              transcription antisense RPRJ cDNA clone T7 <br />                                                                    721 <br />  722     LEE ET AL.                                                                                             MOL. CELL. BIOL. <br />  <br /> RNA polymerase presence [a-32P]UTP (10). Sev-            tween CUP] promoter CYCI terminator a <br /> eral clones chosen 23 positive clones           single-copy centromere vector 2,um-based multicopy <br /> subsequently used restriction mapping subcloning. A        vector. These plasmid constructs used test for <br /> 1.3-kb BglII-HindIII fragment subcloned pUC19,          complementation disrupted chromosomal RPRI gene <br /> both strands sequenced dideoxynucleotide chain         transformation diploid strain heterozygous for <br /> termination method, using modified T7 DNA polymerase               disruption tetrad analysis resulting transfor- <br /> (26) oligonucleotide primers hybridize intervals       mants. <br /> across RPRI region (37). T7 RNA polymerase (10)              Northern analysis RPRI RNA tRNA. The wild-type <br /> modified T7 DNA polymerase (62; gift J. Huibregtse)        haploid strain (W3031A), RPRI-disrupted haploid strain <br /> prepared described previously. Primers sequencing           complemented multiple copies RPRI cDNA con- <br /> were prepared Center Molecular Genetics Core            struct, RPRI-disrupted haploid strain complemented <br /> Facility University Michigan School Medicine.         multiple copies RPRI cDNA dimer construct were <br />    Gene disruption Southern analysis. A 1.75-kb BamHI          grown YPD medium overnight 30&#176;C. Fresh YPD <br /> fragment containing HIS3 gene isolated                medium inoculated overnight cultures to <br /> YCp407 (39) using agarose gel electrophoresis elec-         initial OD650 0.05 grown 30 38&#176;C. Cell <br /> troelution (41). The isolated fragment inserted      growth monitored spectrophotometrically 2 h to <br /> the BamHI site RPRI genomic clone. W303 diploid cells           determine doubling time. One milliliter (approximately 1.2 x <br /> were transformed linearized HIS3-interrupted              108 cells) cultures harvested OD650 1.0 <br /> RPRI genomic DNA clone, transformants selected            used preparation total RNA (23). All RNA <br /> on defined medium lacking histidine (55). The resulting            samples separated 6% denaturing acrylamide gel <br /> transformants sporulated dissected tetrad anal-       transferred nylon membrane, using semidry <br /> ysis. DNAs isolated haploids (W3031A             electroblotter (Gelman Sciences, Inc.) conditions <br /> and W3031B), parental diploid (W303), hetero-          specified manufacturer. The RNA cross- <br /> zygous diploid, single RPRJ disruption (59). The            linked membrane UV exposure (Fotodyne UV light <br /> DNAs digested StuI, separated 1% agarose            mode 3-3000) 3 min. The membrane probed with <br /> gel, transferred nylon membrane (41). The mem-            [a-32P]UTP-labeled antisense RPRI RNA labeled an- <br /> brane probed [a-32P]UTP-labeled antisense          tisense tRNALeU3 (37, 59). For vitro digestion RNA <br /> RPRI RNA transcript previously described (37, 59).              nuclear RNase P, highly purified fraction su- <br />    Mapping RPR1 gene. Strain YPH149 grown            crose gradient centrifugation used previously de- <br /> an optical density 650 nm (OD650) 0.4. Cells            scribed (37). <br /> harvested embedded agarose gel make chromo-               5&apos; end analysis. The 5&apos; end RPRI transcripts was <br /> somal DNA plug described Southern et al. (60).               determined using primer extension labeled an- <br /> Chromosomes separated 1.0% agarose gel 14&#176;C           tisense oligonucleotide hybridized nucleotides 33 to <br /> by pulsed-field electrophoresis, using Bio-Rad CHEF-             48 RPRI coding region (Fig. 1). Both total yeast RNA <br /> DRII apparatus. The switch time 70 s 15 h 120 s        purified RNase P holoenzyme RNA tested and <br /> for additional 12 h. Seventeen chromosomes visible         gave essentially results: major product corre- <br /> after staining ethidium bromide. Chromosomes             sponding mature 5&apos; end minor product -84 <br /> partially digested gel soaking twice 0.25 M HCl       corresponding potential precursor (<span id='am-113' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-114' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-116'  ></span><span id='am-117' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> shown). <br /> for 15 min transferred nylon membrane         This consistent previous results showing a <br /> standard capillary method (41). The membrane              small longer RPRI RNA copurifies the <br /> probed labeled antisense RPRJ RNA (37, 59). The               major 369-nucleotide RNA RNase P holoenzyme. Total <br /> RPRI probe hybridized chromosome V (data               yeast RNA (10 ,g) combined reverse transcriptase <br /> shown). It subsequently Riles Olson          buffer, extensions performed previously de- <br /> the RPRI gene localized restriction fragment            scribed (25). Both primer annealling extension <br /> containing URA3 gene chromosome V (52a). Com-               reactions carried 44&#176;C. Stop mix (4 ,ul 2% <br /> parison restriction map RPRI 8-kb genomic            sodium dodecyl sulfate, 0.1 M EDTA, 1 mg proteinase K <br /> DNA clone published URA3 region (53)         ml) added, mixture incubated 10 min <br /> sequencing RPRI clone downstream HindIII             44&#176;C. After precipitation, reaction resuspended in <br /> site confirmed relative positions genes (Fig. 1).   formamide loading buffer, heated 95 100&#176;C 2 min, <br />    Plasmid construction. The 400-nucleotide 5&apos;-flanking re-        separated 8% denaturing acrylamide gel. The <br /> gion 550-nucleotide 3&apos;-flanking region amplified       positions primer extension stops determined by <br /> the polymerase chain reaction using Taq polymerase (56)            direct comparison dideoxynucleotide chain termination <br /> and inserted BamHI HindlIl sites               reactions linearized RPRJ genomic clone, using the <br /> single-copy centromere vector YCpSO 2,m-based               labeled primer (26). <br /> multicopy vector YEp24. The resulting constructs contained            S1 analysis. A probe determination 3&apos; end the <br /> an EcoRI site junction flanking regions.         RPRI transcripts protection S1 nuclease was <br /> RPRI cDNA, <span id='am-85' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-86' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-95'  ></span><span id='am-96' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span>Escherichia coli</span> RNase P Ml cDNA (50),                  generated RPRI-containing RsaI fragment cloned <br /> human RNase P Hi cDNA (4; gift S. Altman), S.                   EcoRI polylinker site pUC19 use of <br />  cerevisiae mitochondrial RNase P RNA cDNA, RPRI            EcoRI linkers. The DNA linearized unique Styl <br /> cDNA dimer ligated EcoRI linker inserted        site coding region RPRI gene, 5&apos; <br />  EcoRI site. S. cerevisiae mitochondrial RNase P         overhangs end filled [t_-32P]dCTP, using standard <br />  RNA cDNA prepared amplification syn-              techniques (41). The 330-bp 3&apos; end probe containing the <br /> thetic primers Taq polymerase (56) genomic clone          antisense strand labeled StyI site removed <br />  (42; gift C. Wise N. Martin). In attempt obtain      pUC19 vector cleavage EcoRI. The radio- <br />  conditionally expressed clones, various RNase P RNA               labeled strand isolated electrophoresis a <br />  cDNAs inserted adjacent GALIO promoter be-           denaturing 6% polyacrylamide gel electroelution (41). S1 <br />  VOL. 11, 1991                                            RPRI ENCODES THE S. CEREVISIAE NUCLEAR RNase P RNA                             723 <br />  <br />  <br />                            A <br />                                                                       1.3 kb <br />                                                                         m <br />                                 K                 A P BgSt       Bg St B       HSStAPH           H         Bg <br />                                                    l11ii           II -_ W- <br />                                                                                  IIA I <br />                                                                       RPR1         URA3 <br />                                     1 kb <br />                    B <br />                          BglIl                                    SpeI <br />                     -407 agatctgcca attgaacata acatggtagt tacatatact agtaatatgg ttcggcacac <br />                     -347 attaaaagta taaaaactat ctgaattacg aattacatat attggtcata aaaatcaatc <br />                     -287 aatcatcgtg tgttttatat gtctcttatc taagtataag aatatccata gttaatattc <br />                                                                                                EpaI <br />                     -227 acttacgcta ccttttaacc tgtaatcatt gtcaacaggt atgttaacga cccacattga <br />                                                                                  NaeI <br />                    -167 taaacgctag tatttctttt tcctcttctt attggccggc tgtctctata ctcccctata <br />                                                                StuI        A Box <br />                    -107 gtctgtttct tttcgtttcg attGTTTTAC GTTTGAGGCC TCGTGGCGCA CATGGTACGC <br />                                                                            1 <br />                     -47 TGTGGTGCTC GCGGCTGGGA ACGAAACTCT GGGAGCTGCG ATTGGCAC.*_aC&amp;MGGT <br />                                                                                BamilI <br />                        14 AATTCCTaC ATTAAaAAAC! CTGTTTACAG                A&amp;QA=^ACCC      &apos;A   CTACGG    GTTATCAG <br />                        74 ATATTATCAQTGAkTsGTGAC                           CAGTGGA=GC TTG2TCCTCCG GGTTAATGTC <br />                      134 GCTTTTGGC-A TClCTGCTCCT                      AIAATATA (%-GGAAC AGTCTTTACC <br />                                                                      kA <br />                      194 <br />                      254 ACCIGTTGTTC CGTTACGCGCoCCT ACcGGCrTr.Ar CGTCAfGC GTaGC <br />                                    Aatll                                    B Box <br />                      314 AATCGTAGG CGTCATTAGT GGCGAACCCG ATACCGATTA CTGCTGCTGT TCCAGCCCAT <br />                      374 ATCCAACTTC CAATTTAATC TTTCTTtttt aattttcact tatttgcgat                          acagaaagaa <br />  <br />                      434 aaaagcgata gtaactattg aattttgttt ggatttggtt agattagata tggtttctct <br />                      494 ttatatttac atgctaaaaa tgggctacac cagagataca taattagata tatatacgcc <br />                                                          <span id='am-108' about='obo:NCBITaxon_328886' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_124343 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_124344 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-109'  ></span><span id='am-110' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-112' property="rdfs:label" content="Styx" datatype="xsd:string"></span>Styx</span> <br />                      554 agtacacctt atcggcccaa gccttgtccc aaggcagcgt tttgttcttg gaaacgctgc <br />                      614 cctacacgtt cgctatgctt caagaacttt tctgagcact tcatgatgca tgtttgttcc <br />                      674 ttattggtta gctttgatgt tgtgaagtca ttgacacagt ctgtgaaaca tctttctacc <br />                      734 agattagagt acaaacgcat gaaatccitc atttgctttt gttccactac tttttggaac <br />                      794 tcttgttgtt ctttcggagt tcaatgcgtc catctttaca gtcctgtctt attgttcttg <br />                      854 atttgtgccc cgtaaaatac tgttacttgg ttctggcgag gtattggata gttccttttt <br />                                       HindlIl <br />                      914 ataaaggcca tgaagctt <br />    FIG. 1. Location sequence RPRJ gene region. (A) Location RPR1 gene. The RPRI gene located chromosome <br /> V immediately adjacent URA3 described Materials Methods. A partial restriction endonuclease map region relative <br /> orientations RPRI URA3 coding regions shown. Restriction sites: A, ApaI; B, BamHI; Bg, BglII; H, HindIII; K, KpnI; P, PstI; <br /> S, SmaI; St, StuI. (B) Sequence RPRI region sense strand BglII Hindlll sites. Positions numbered relative the <br /> first nucleotide major RPR1 RNA (+ 1). The sequence corresponding major RNA underlined, sequence corresponding <br /> to longest prevalent longer RNAs (Fig. 6) indicated capital letters (Fig. 2). Sequences corresponding suspected RNA <br /> polymerase III A box B box internal promoters indicated overlining (see Discussion). The Styl site used prepare 3&apos; end S1 <br /> probe indicated. <br />  724    LEE ET AL.                                                                                                       MOL. CELL. BIOL. <br />  <br /> nuclease analysis performed previously de-             A            &lt; <br /> scribed (2). Size markers Si-protected fragments                   I-   CO g <br /> were unrelated dideoxynucleotide chain termination reac-                    m0:aG.X <br />                                                                             jm <br /> tions adjacent lanes (not shown). <br />   Nucleotide sequence accession number. The GenBank ac- <br /> cession number RPRI region sequence M27035. <br />                                                                                                    B <br />                                                                                                          T-        X-     0 <br />  <br />                            RESULTS                                                                       Vm <br />                                                                                              nt               m           CL~~~0 <br />    Antisense transcripts cDNA encoding S. cerevisiae                 _-         - -   84 <br /> RNase P-associated RNA (37) used screen S. <br /> cerevisiae genomic library YCp5O filter hybridization <br /> (2). Twenty-three positive clones encompassed the <br /> cDNA sequence identified. A partial restriction endo- <br /> nuclease map 6-kb region surrounding gene shown <br /> in Fig. 1A. In preparation obtaining genetic map                                                                                 nt <br /> location locus, referred subsequently RPRJ                                                 *         ]w               J 385-388 <br /> (RNase P ribonucleoprotein 1), cDNA mapped to <br /> chromosome V probing chromosomal blots pulsed- <br /> field gel electrophoresis (60; data shown). Before classi- <br /> cal genetic mapping clone accomplished,                                                 * <br /> however, localization hybridization chromosomal re-                                                          *        -3369 <br /> striction digest blots indicated RPRI close URA3 <br /> (52a). Comparison restriction map genomic <br /> RPRJ clones published map URA3 (53) locus <br /> suggested genes adjacent, possibility                                 +1 <br /> which confirmed extending sequence the <br /> RPRI region (Fig. 1B; below) downstream past the <br /> HindIII site encountering previously published se- <br /> quence downstream URA3 (53). <br />    The sequences strands DNA region <br /> shown Fig. 1B obtained direct analysis a <br /> pUC19 subclone containing 1.3-kb BglII-HindIII fragment,          FIG. 2. 5&apos; 3&apos; end analysis RPRI RNAs. (A) Primer <br /> using sequential oligonucleotide primers dideoxynucle-        extension analysis performed described Materials and <br /> otide chain termination methods double-stranded tem-          Methods 10 ,ug RNA indicated haploid yeast <br /> plates (26). The positions corresponding major RPRI        strains radiolabeled primer anneals positions 33 48 in <br /> RNA underlined. A larger RNA copurifies        RPRI sequence. (B) SI nuclease analysis performed a <br /> RNase P relatively small quantities hybridizes      DNA fragment labeled antisense strand Styl site as <br /> RPRI probes (37) characterized (Fig. 2),        described Materials Methods. The exact 3&apos; terminus the <br /> the extent larger RNA indicated capital letters   major RNA determined previously (37). (Low levels slightly <br /> Fig. 1B. Previous work indicated RPRI RNA            longer 3&apos; termini observed experiments [e.g., Fig. 6] <br /> DNA sequences colinear (37), suggesting             dependent concentration Si nuclease within <br /> larger RNA longer 5&apos; 3&apos; terminus           experiment.) nt, Nucleotides. <br /> 369-nucleotide major RNA precursor. The 5&apos; <br /> and 3&apos; termini larger RNA mapped primer <br /> extension Si nuclease analysis, respectively, total        sponding additional 28 30 nucleotides past the <br /> cellular RNA S. cerevisiae strains used      369-nucleotide signal seen experi- <br /> and previous study (37). Extension performed         ments. The data chosen Fig. 2, example, no <br /> radiolabeled primer anneals 32 nucleotides downstream        obvious longer RNA, data Fig. 6 do. <br /> of 5&apos; terminus smaller RNA (Fig. 1B). In        We able demonstrate longer <br /> strains, multiple extension stops corresponding        RNAs primary transcripts RPRI gene that <br /> to slightly heterogeneous 5&apos; end major 369-            precursors major form. If conversion <br /> nucleotide RNA significant extension         taking place, however, lack stable intermediates at <br /> product corresponding transcript 84 nucleotides      5&apos; end suggests single step involved, <br /> additional 5&apos; sequence (Fig. 2). S1 nuclease mapping 3&apos;    multiple 3&apos; reactions suggested results <br /> end RNA showed major product corresponding            PP1002 alternate constructs described (see <br /> the unique 3&apos; end 369-nucleotide RNA lesser       Fig. 6). This pattern interesting resembles the <br /> amounts RNAs ranging 16 30 nucleotides longer          pattern pre-tRNA processing, preliminary <br /> (Fig. 2). In strains BJ411 W3031A, longer products        evidence RNase P RNAs synthesized RNA <br /> appeared clusters roughly 16 19 nucleotides longer     polymerase III (4; Discussion). Given possibility, <br /> the major 3&apos; terminus 369-nucleotide RNA. We           potential A box B box intragenic promoter elements for <br /> unable tell appearance multiple bands           RNA polymerase III (16) indicated Fig. 1B and <br /> reflects RNA heterogeneity artifact Si assay.     discussed below. Each 80% sequence <br /> Strain PP1002, however, harbors rna82 lesion causes       similarity corresponding promoter consensus (16). <br /> aberrant 3&apos; end processing 5S tRNA RNA polymerase             Gene disruption performed strategy outlined in <br /> III transcripts (46-48). In strain, faint signal corre-    Fig. 3 confirm gene single copy, examine <br />  VOL. 11, 1991                                               RPR! ENCODES THE S. CEREVISIAE NUCLEAR RNase P RNA                                          725 <br />  <br />            RPR1                                                                                DNA Insert <br />  <br />                                                                     z              RPR15&apos;F7                 ,RPR 1F <br />                                                                     en     2&gt;I 408 bD           I                                           Complementation <br />        -i3-                                                                    B           E                E              H <br />                                                                     z     I        b |     RPR1 cDNA In single &amp; multi-copy vector                 + <br />                                                                40alO <br />                                                                           0 zzzzz4w1bE         coi RNase P MI cDNA In single &amp; multi-copy vector   - <br />  <br />  <br />  <br />  <br />                                                               IF    z <br />                                                                     0 <br />  <br />                                                                     z <br />                                                                     ui <br />                                                                           I <br />  <br />  <br />  <br />  <br />                                                                           I <br />                                                                                    * .j* <br />  <br />                                                                                    53 <br />  <br />                                                                               ====IF <br />                                                                                            Human RNase P HI cDNA In single &amp; multi-copy vector <br />  <br />                                                                                            S. cerevislae mitochondria RNase P RNA cDNA <br />                                                                                            In single &amp; multi-copy vector <br />  <br />                                                                                                     * RPRI cDNA dimer In single-copy vector <br />                                                                     -i <br />                                                                     O <br />                                      Sporulation &amp; <br />                                                                     all <br />                                                                     0 <br />                                                                           I         * *        ** RPRI cDNA dimer In multi-copy vector             + <br />                                      Tetrad Analysis           00 <br />                                                                             FIG. 5. Complementation disrupted chromosomal RPRI gene <br />                                                                           various RNase P RNA cDNAs. cDNA mature RNase P <br />   FIG. 3. Scheme RPRI gene disruption complementation.            RNA inserted replace 369-nucleotide major RPR1 RNA <br /> Procedural details described Materials Methods.                coding region addition EcoRI linker described Materials <br />                                                                           Methods. These plasmids used transform diploid <br />                                                                           heterozygous RPRI gene disruption. Spores obtained <br /> whether gene product essential, prepare rprl             transformants dissected determine complementation. <br />                                                                           +, All spores viable result complementation by <br /> mutant background phenotype defective                     plasmid-borne RNase P RNA gene; -, spores were <br /> RPRI gene constructs tested. HIS3 insertion                 viable result functional complementation the <br /> diploids selected growth medium lacking histi-                  plasmid. <br /> dine, followed sporulation tetrad dissection. Only <br /> two spores tetrad viable (Fig. 4A), viable <br /> spores His&apos; (not shown), indicating HIS3                    inserted essential locus. Evidence substitutive <br />                                                                           transformation site RPRJ gene provided by <br />                                                                           Southern analysis DNA parental diploid, a <br />             A                                                             diploid heterozygous disruption, viable haploids <br />                                                                           probed RPRI antisense RNA (Fig. 4B). Only case <br />                                                    1 <br />                                                                           heterozygous disrupted His&apos; diploid did StuI diges- <br />                                               - 2                         tion produce RPRJ fragments, normal size and <br />                                               -3                          second larger corresponding 1,750-bp <br />                                               -4                          HIS3 fragment. Taken together, results indicate that <br />                                                                           RPRI single copy essential. <br />              B                                                               Several experiments conducted test var- <br />                                                                           iants RPRI cDNA sequences heterologous RNase <br />                          b     c    d                                   P RNA component cDNAs complement disrupted <br />                                                                           RPRI gene. These experiments conducted two <br />                                               -8454                       reasons. First, precedent mixed RNase P protein <br />                                               -5686                       RNA components distantly related bacteria (21) or <br />                                               - 3675 <br />                                                                           bacteria human cells (17) able reconstitute <br />                                               - 2323                      functional holoenzyme vitro, to <br />                                               - 1929                      determine RPRJ product sufficiently related <br />                                                 1371                      analogous RNAs E. coli, human cells, yeast <br />                                                 1264                      mitochondria (42) reconstitute mixed holoenzymes in <br />                                                                           vivo. Second, repeated attempts produce partially defec- <br />    FIG. 4. Analyses diploids RPRI gene disruptions. Dip-          tive conditionally lethal mutations RPRI been <br /> loids selected URA3 plasmid acquisition RPRI gene              unsuccessful, hoped heterologous <br /> disruption scheme (Fig. 3) sporulated, tetrads              altered cDNA constructs defect a <br /> dissected. (A) Growth dissected spores rich medium showed           molecular phenotype identifiable RNase P lesion. At <br /> 2:2 segregation expected disruption essential gene. 1, 2,      first, heterologous cDNAs mutated RPRI cDNAs were <br /> 3, 4 refer positions spores       placed control inducible S. cerevisiae RNA <br /> eight tetrads shown. (B) Southern analyses performed         polymerase II promoters plasmids, construct <br /> antisense RPRI RNA probe DNAs wild-type haploid              including bona fide RPRI cDNA complement (not <br /> strains [lane a, W3031A(a); lane b, W3031B(a)], wild-type diploid       shown), suggesting RNase P RNA be <br /> strain (lane c, W303), diploid strain heterozygous RPRJ      properly synthesized RNA polymerase II. <br /> gene disruption (lane d) cleavage StuI. Sizes RPR1         The YCp5O YEp24 plasmid constructs described in <br /> fragments 1.5 kb wild-type haploid strains (lanes a <br /> and b) wild-type diploid strain (lane c) 1.5 3.25 kb   Materials Methods Fig. 5 devised <br /> strain heterozygous RPRJ gene disruption (lane d). Size            drive expression cDNAs RPRI 5&apos;- and <br /> markers (indicated nucleotides) lambda phage DNA di-              3&apos;-flanking regions. In case, cDNA corresponding <br /> gested BstEII stained ethidium bromide (not shown).         indicated mature RNase P RNA inserted to <br />  726      LEE ET AL.                                                                                                            MOL. CELL. BIOL. <br />  <br />              TABLE 1. Doubling time YPD medium                                 RNA, little smaller RNA 5&apos; end appeared <br />                                                      Doubling time (h) <br />                                                                                   (Fig. 6B). Longer heterogeneous primer extension products <br />         Strain                                                                    primer annealing second copy the <br />                                              300C                        38&#176;C     RNA dimeric transcripts. Similarly, 3&apos; termini the <br /> Wild type                                    1.27                        1.60     dimeric transcripts appeared roughly pattern <br /> RPRJ cDNA                                    1.36                        1.70     observed cDNA monomer insert (Fig. 6C) that <br /> RPRI cDNA dimer                              2.60                        5.80a    3&apos; signal corresponding small <br />                                                                                   RPRJ RNA. Thus, likely interpretation the <br />   3.2 h approximately generations temperature shift, followed   pattern dimer clone RNAs Fig. 6A transcripts are <br /> by 5.8 h.                                                                         initiating normal position upstream cDNA <br />                                                                                   copy. The heterogeneous pattern long RNAs arises either <br />                                                                                   multiple terminations past second copy or <br /> replace 369-nucleotide major RPRJ RNA coding region                           termination correct site dimer insert <br /> by addition EcoRI linkers. After use plasmids                     followed aberrant processing. In case, little RNA <br /> to transform heterozygous RPRI-disrupted diploid, spores                        normal size growth-defective strain at <br /> were obtained dissected determine complementation.                         30 38&#176;C. <br /> The mature RPRI cDNA served control defects                             By using RNA samples Fig. 6, steady- <br /> to linker insertions YCpSO YEp24 showed                         state levels mature precursor tRNA examined <br /> growth defects. However, heterologous cDNAs                           probing duplicate Northern blots antisense RNA <br /> from E. coli, human cells, yeast mitochondria gave                         probe tRNALeU3. This tRNA synthesized precursor <br /> detectable complementation. It clear time                          10-nucleotide leader, 32-nucleotide intron, 6 to <br /> whether finding defects transcription, RNA                      7 nucleotides 3&apos; trailing sequence (44, 61). The terminal <br /> processing, assembly transport, function. Several                          sequences normally removed intron (61), and <br /> variants RPRJ gene did complement (not                            vitro removal 3&apos; sequences yeast lysates quite <br /> shown) serendipitous insertion RPRJ cDNA                            slow 5&apos; leader removed RNase P <br /> direct tandem repeat tested. In single copy, con-                        (unpublished observations). The partial processing products <br /> struct did complement, diploids carrying                         Fig. 7, assigned direct size comparison sequenced in <br /> dimer multicopy plasmid gave normal spores                               vitro products (not shown), pattern was <br /> (nondisrupted chromosomal RPRI copies) slowly                             followed parental strain disrupted strain <br /> growing spores sporulated. Growth curves                             carrying single cDNA insert, somewhat pri- <br /> normal defective spores (Table 1) differ-                       mary transcript accumulating higher temperature in <br /> ence doubling twofold 30&#176;C increased                        both. Ordered removal 5&apos; end, 3&apos; end, and <br /> than threefold 38&#176;C.                                                           intron implied observed intermediates (61). <br />    To characterize defect caused direct cDNA                           In contrast, strain RPRI complemented by <br /> repeat RPRI dimer, expression RPRI RNA                                cDNA dimer accumulated near-primary-tran- <br /> tRNA examined 30 38&#176;C. Figure 6 shows North-                            script-size RNA, little intermediate appearing that <br /> ern blot analysis 5&apos; 3&apos; end analysis RPRI                       contains intron lacks 5&apos; 3&apos; termini. End <br /> parent strain, disrupted RPRI haploid strain comple-                            processing defective RPRJ-defective strain. <br /> mented multiple copies single RPRI cDNA                               Also, 38&#176;C strain accumulated intermediate in <br /> construct (Fig. 5), disrupted haploid complemented                           intron appears removed, the <br /> with multiple copies cDNA dimer construct. The                              5&apos; leader 3 4 nucleotides 3&apos; trailing sequence. <br /> wild-type RNA displayed normal pattern 369-nucleo-                          Smearing primary transcript apparently <br /> tide RNA, lesser amounts larger RNA containing                         spliced intermediate downward probably reflects partial 3&apos; <br /> longer 5&apos; 3&apos; termini. The cDNA monomer clone                                exonucleolytic processing slowed presence 5&apos; <br /> multiple copies accumulated considerably larger                        leader (12). An RNA slightly smaller mature-size <br /> RNA somewhat smaller 369-nucleotide                                tRNALeU, appeared 38&#176;C samples, not <br /> RNA correct 5&apos; 3&apos; termini. If longer RNA                       known lacks portions 3&apos; CCA some <br /> in fact precursor 369-nucleotide form, appears                    deficiency. It worth noting spliced prod- <br /> that correctly processed 5&apos; 3&apos; ends                      ucts containing terminal sequences normally <br /> without regard exact sequence flanking mature                          accumulate, observed appearance RNA size of <br /> termini. Accumulation presumptive precursor                           spliced RNase P-cut intermediates RPRI- <br /> indicate step processing unable                     defective strain, indicating normal ordering prob- <br /> with increased synthesis, overloading                       ably obligatory vivo. The occurrence tRNA splicing <br /> assembly processing machinery slowing                           end maturation recently observed by <br /> rate limiting step(s). Since 5&apos; termini larger                  (45). <br /> RNAs discrete extend wild type                                  Because size analyses tRNA suggest pre- <br /> approximately size added linker (Fig. 6B),                          cursors accumulate, purified total RNA defective <br />  multiple longer products seen cDNA Northern analy-                         strain grown 38&#176;C digested isolated RNase P (37) <br />  ses probably extended 3&apos; ends, consistent                     blotted samples shown Fig. 7. <br /> longer products seen S1 3&apos; analyses. It known                      The accumulated tRNALeU3 primary transcript con- <br /> whether longer products arise inefficient tran-                         verted size intron-containing precursor slight <br />  scription termination inappropriately processed longer                         size heterogeneity consistent variable amounts re- <br />  transcripts stable seen                        maining 3&apos; trailer. Identical size reductions observed <br />  wild-type gene.                                                                   lesser amounts primary transcripts accumulated by <br />    The RPRI cDNA dimer construct transcribed                           wild-type RPRI cDNA monomer strains (not <br /> give large percentage accurate 5&apos; termini longer                       shown). The spliced intermediate de- <br />  VOL . 1 l, 1991                                                           RPRI ENCODES THE S. CEREVISIAE NUCLEAR RNase P RNA                                             727 <br />  <br />  <br />                   A                     30&#176;C                38&#176;C             B          300C           380C           C                     30&#176;C             380C <br />                                    I&apos;        -      I                 I                                                                                1&quot;           A. <br />                                                                                                  0               0 <br />                                                                                                                                                    0                 0 <br />                                                0)                 0                              E               E                                 E                 E <br />                                              E                    E                                                                                X                 C <br />                                          4c .:               4: .:                          Z Z        Z Z <br />  <br />                                     2Za                 @ Za                           &gt;-                                              X, <br />                                                                                                                                        3     a:    a: <br />                                                                                                                                                    I&amp; <br />                                                                                                                                                    T <br />                                                                                                                                                               z <br />                                                                                                                                                               rr    z <br />                                                                                                                                                                     T- <br />                                                                                             1- I-w-   1~ <br />                                                                                                       I- <br />                                                                                                                 T-. <br />                                                                                                                                         = <br />                                                                                                                                             v           .v    m <br />                                                                                                                                                         = <br />                                                                                         =0. 0.             0.0.M <br />                                     = <br />                                     3   0.   a.         =    0. <br />                                                              ccX                       r   cr              Er   E <br />                                                                                                                       nt <br />                                                                                                                       397-399      [ <br />                                                                   w <br />  <br />                                                                                                                       385388so *4 <br />                         nt               &lt; <br />  <br />                       540    - <br />  <br />                                                                                                                           369 -- <br />                                                                            nt <br />                       450    - <br />  <br />  <br />  <br />                                                                            -84    -.        _              _ <br />  <br />                   369   nt <br />                major -             -     *                  1 <br />              RPR1 RNA <br />  <br />  <br />  <br />  <br />                                                                             .1- <br />                                                                                                       iS <br />  <br />  <br />  <br />  <br />    FIG. 6. RPRI expression monomer tandem dimer cDNA clones. Northern end analyses performed RNA isolated <br /> from parental haploid strain (&apos;Wild Type&apos;), W3031A, disrupted haploids complemented monomer tandem dimer <br /> RPRI cDNA insert RPRI flanking regions described legend Fig. 4 text. Analyses performed described <br /> in legend Fig. 2 text total cellular RNA cultures grown 30 38&#176;C. (A) Northern analysis using antisense <br /> RPRI RNA. The positions known 369-nucleotide (nt) RNA longer markers indicated. (B) 5&apos; end analysis primer extension. <br /> The cDNA construct termini corresponding 369-nucleotide RNA wild-type cells, RNAs initiating further <br /> upstream longer 5 bp corresponding added EcoRI linker. (C) 3&apos; end analysis S1 nuclease protection. The major 3&apos; termini <br /> are discussed text. Positions numbered according Fig. 1B. <br />  <br />  <br />  <br /> creased, presumably conversion mature-size                                   14, 24, 32, 34, 35, 37, 38, 40, 42). The sequence similarities <br /> tRNA removal 5&apos; terminus. Thus, major                                        nuclear RNase P RNAs potential <br /> accumulated pre-tRNAs RPRI-defective strain                                 structural similarities analogous procaryotic ri- <br /> not cleaved RNase P,                                 bozymes (5, 37) strongly suggested identification of <br /> appropriate substrates enzyme.                                                  RNAs RNase P components correct, formal <br />                                                                                         proof lacking. Unlike procaryotic ribozymes, <br />                                  DISCUSSION                                             eucaryotic RNAs shown catalytic <br />                                                                                         activity purified synthesized vitro, have <br />   RNAs copurify RNase P activity                                sequence-specific attacks RNase H shown to <br /> isolated number eucaryotic sources (1, 7, 9, 11, 12,                          destroy RNase P activity. As enzyme purified <br />  728     LEE ET AL.                                                                                                               MOL. CELL. BIOL. <br />  <br />              0                                                                    previous biochemical assignment assign- <br />              0 <br />              C&apos;) <br />                                       co) <br />                                       C&apos;)                                           ment analogous eucaryotic RNase P-associated <br />              jn <br />                          I                         i <br />                                                                                     RNAs well. <br />                                                                                        Analysis RPRI transcription unit preliminary. <br />                                                    00                               Two main RNAs appear wild-type cells, predominant <br />                      EEE                                                            369-nucleotide form (37) longer RNA containing <br />                                                                                     extra 84 nucleotides 5&apos; end 16 30 extra <br />                  Z   Z        z z zB                                                residues 3&apos; end. It determined <br />         I-       Ct       -   I             LD                                      RNAs primary transcript or <br />                                                                                     precursor-product relationship between <br />                               ca:C: a:                                              longer smaller forms. For reasons, however, <br />                                                                                     speculate longest observed RNA the <br />                                                                                     primary transcript RPRJ 369-nucleotide form <br />        wool                                              o-    primary transcript <br />                                                                (+5&apos; +3.+IVS) <br />                                                                                     results processing steps. First, RNAs <br />                                                                                     derive single-copy gene, use <br />                                                                                     probes detectable genes similar <br />                                                         -o*    +3&apos;,+IVS             sequence S. cerevisiae. Second, RNAs not <br />                          &quot;_01-1             -111 <br />                                                         -.,-   +IVS                 appear single RNase P holoenzyme. Although <br />                                                                                     copurify stages ion-exchange chro- <br />                                                                                     matography, suggesting similar protein complements, they <br />                                                                                     precisely comigrate velocity sedimentation (37). <br />                                                                                     Third, longer, shorter, sequence at- <br />                                        0                --- <br />                                                                +5,+3 <br />                                                                                     tributes RNA polymerase III transcription unit. The <br />                                                                                     gene encoding analogous human RNA shown to <br />                                                                                     transcribed RNA polymerase III vitro (4), and <br />                                                                                     preliminary work laboratory suggests RPRI is <br />                                                         -o     mature     tRNALeU <br />                                                                               3     transcribed RNA polymerase III vitro vivo <br />                                                                                     basis a-amanitin tagetin sensitivities the <br />    FIG. 7. Accumulation tRNA precursors RPRI-defective                     effects mutations expression (unpublished data). RPRI <br /> strain. Northern analysis tRNALCU3 RNAs performed                         contains A box B box internal promoters typical of <br /> described Materials Methods, using total cellular RNA                   tRNA-like yeast U6 RNA gene promoters (8). The two <br /> the experiment shown Fig. 6. The known sizes sequences                    best B box sequences marked Fig. 1 about <br /> the normal precursors (61) indicated right                      80% similarity consensus GGTTCGANTCC (16). <br /> confirmed electrophoresis radiolabeled tRNALCU3 gene tran-                    These sequences considerably farther downstream than <br /> scription processing products prepared vitro (not shown, 13,                 normal tRNA promoters, distance not <br /> 61). +5&apos;, +3&apos;, +IVS (intervening sequence) refer portions                 inconsistent functional B box far downstream the <br /> the primary transcript remaining indicated precursor. The far                yeast U6 transcription initiation site (8). The best A box <br /> right-hand lane shows products obtained sample of <br /> purified 38&#176;C RPR1 cDNA dimer construct strain RNA cleaved                      match marked 83% similarity the <br /> with S. cerevisiae nuclear RNase P vitro described Materials               TPuGCNNAGPyNGG consensus (16), perfect matches <br /> and Methods.                                                                        invariant residues. The position sequence 20 <br />                                                                                     residues downstream longer RNA 5&apos; end correct for <br />                                                                                     positions A boxes relative transcription initiation sites <br />                                                                                     (13). Although sequences with <br /> to absolute homogeneity, remained possible                        A box similarity RPRJ, correctly positioned <br /> the identified RNAs copurifying contaminants. To                               5&apos; end major 369-nucleotide RNA a <br /> address problem, characterized RPRJ gene                           transcription start. The observed 3&apos; ends appear heteroge- <br /> for yeast nuclear RNase P-associated RNA used                             neous represent multiple processing events, since <br /> a defective variant determine RNase P function                        longest extend T6 region downstream would <br /> impaired.                                                                           appropriate RNA polymerase III terminator. A rea- <br />    Disruption single RPRI gene chromosome V                            sonable hypothesis RPRI RNA production therefore <br /> lethal, number RPRI variants tested                         synthesis precursor (indicated underlining Fig. 1) <br /> complementation gave defective growth phenotype short                             followed processing 5&apos; 3&apos; ends the <br /> of complete lethality. This variant, tandem cDNA dimer                             369-nucleotide mature RNA. <br /> inserted RPRI flanking regions, little                                The disruption chromosomal RPRI gene opens the <br /> correct-size RPRI RNA accumulated tRNA precursors                                possibility examining effects incorporating variant <br /> that undergone cleavage RNase P. It formally                           RNase P RNAs holoenzyme vivo, circumventing <br /> possible defect RPRI actually RNase                         inactivity RNA inability recon- <br /> P subunit causes accumulation indirectly                             stitute significant activity separated RNA protein <br /> through undetermined interference stage                           components repeated experiments. In studies de- <br /> tRNA RNase P RNA maturation. This unlikely,                                 scribed here, attempted substitute heterolo- <br /> however, considering (i) RPRJ RNA cofractionates                           gous RNase P cDNAs mature RPRJ cDNA to <br /> highly purified S. cerevisiae RNase P (37), (ii) similar RNAs                        investigate function yeast holo- <br /> copurify different methods RNase Ps                               enzyme, demonstrated previously reconsti- <br /> other eucaryotes, (iii) RNase P cleavage appears                         tuted heterologous components. Unfortunately, such <br /> very early step tRNA maturation. The genetic evidence                             attempts lethal recessive, unambig- <br /> presented represents essential validation                       uously determined defect processing, <br />  VOL.   11, 1991                                            RPRI ENCODES THE S. CEREVISIAE NUCLEAR RNase P RNA                               729 <br />  <br /> assembly, stability, activity, transcription (e.g.,            17. Gold, H. A., S. Altman. 1986. Reconstitution RNase P <br /> lack internal promoters). It expected, however,                    activity using inactive subunits E. coli HeLa cells. Cell <br /> reconstitution RNase P holoenzyme vivo, future                    44:243-249. <br /> studies evaluating heterologous RNAs mutations               18. Guemer-Takada, C., S. Altman. 1984. Catalytic activity of <br /> RPRJ facilitated RPRI-disrupted -defec-                   RNA molecule prepared transcription vitro. Science <br /> tive strains.                                                                223:285-286. <br />                                                                          19. Guerrier-Takada, C., S. Altman. 1984. Structure solution <br />                                                                              Ml RNA, catalytic subunit ribonuclease P from <br />                        ACKNOWLEDGMENTS                                       <span id='am-101' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-102' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-105' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-107'  ></span>Escherichia</span> coli. Biochemistry 23:6327-6334. <br />                                                                          20. Guerrier-Takada, C., S. Altman. 1986. Ml RNA large <br />    We thank M. Hollingsworth, C. Wise, N. Martin, M. Baer, S.                terminal deletions retains catalytic activity. Cell 45:177-183. <br /> Altman, D. Waugh, N. Pace gifts plasmids N. Pace,         21. Guerrier-Takada, C., K. Gardiner, T. Marsh, N. Pace, S. <br /> A. Forster, S. Altman helpful discussions. We thank W.          Altman. 1983. The RNA moiety ribonuclease P cata- <br /> Studier T7 RNA polymerase clone, S. Tabor T7 DNA             lytic subunit enzyme. Cell 35:849-857. <br /> polymerase clone, E. Esposito, E. Jones, P. Hieter yeast         22. Guerrier-Takada, C., K. Haydock, A. Leland, S. Altman. <br /> strains, D. Thiele plasmids yeast genomic DNA                1986. Metal ion requirement aspects reaction <br /> library. We deeply indebted L. Riles M. Olson                 catalyzed Ml RNA, RNA subunit ribonuclease P in <br /> mapping gene.                                                            Escherichia coli. Biochemistry 25:1509-1515. <br />    This work supported National Institutes Health grant        23. Hanning, E. M., D. J. Thiele, M. J. Leibowitz. 1984. <br /> GM34869 National Science Foundation grant DMB-8901559                 Saccharomyces cerevisiae killer virus transcripts contain tem- <br /> D.R.E.                                                                       plate-coded polyadenylate tracts. Mol. Cell. Biol. 4:101-109. <br />                                                                          24. Hollingsworth, M., N. C. Martin. 1986. RNase P activity in <br />                             REFERENCES                                       mitochondria Saccharomyces cerevisiae depends both <br />  1. Akaboshi, E., C. Guerrier-Takada, S. Altman. 1980. Veal              mitochondrion nucleus-encoded components. Mol. Cell. <br />       heart ribonuclease P essential RNA component. Bio-              Biol. 6:1058-1064. <br />       chem. Biophys. Res. Commun. %:831-837.                             25. Huibregtse, J. M., D. R. Engelke. 1986. Direct identification <br />  2.   Altman, S., C. Guerrier-Takada. 1986. Ml RNA, RNA              small sequences changes chromosomal DNA. Gene 44: <br />       subunit Escherichia coli ribonuclease P, undergo              151-158. <br />       pH-sensitive conformational change. Biochemistry 25:1205-          26. Huibregtse, J. M., D. R. Engelke. 1990. Sequence and <br />       1208.                                                                  footprint analysis yeast DNA primer extension. Methods <br />  3.   Ausubel, F. M., R. Brent, R. E. Kington, D. D. Moore, J. G.            Enzymol. 194:550-562. <br />       Seidman, J. A. Smith, K. Struhl (ed.). 1987. Current proto-    27. Ikemura, T., Y. Shimura, H. Sakano, H. Ozeki. 1975. <br />       cols molecular biology. John Wiley &amp; Sons, Inc., New York.          Precursor molecules Escherichia coli transfer RNAs accumu- <br />  4.   Baer, M., T. W. Nilsen, C. Costigan, S. Altman. 1990.              lated temperature-sensitive mutant. J. Mol. Biol. 96:69-86. <br />       Structure transcription human gene Hi RNA:        28. Ito, H., Y. Fukuda, K. Murata, A. Kimura. 1983. Transfor- <br />       RNA component human RNase P. Nucleic Acids Res.                     mation intact yeast cells treated alkali cations. J. <br />       18:97-101.                                                             Bacteriol. 153:163-168. <br />  5.   Bartkiewicz, M., H. Gold, S. Altman. 1989. Identification   29. Jain, S. K., M. Gurevitz, D. Apirion. 1982. A small RNA <br />       RNA molecule copurifies RNase P activity             complements mutants RNA processing enzyme <br />       HeLa cells. Genes Dev. 3:488-499.                                      ribonuclease P. J. Mol. Biol. 162:515-533. <br />  6.   Boehm, S. 1987. Similarities predicted secondary         30. James, B. D., G. J. Olsen, J. Liu, N. R. Pace. 1988. The <br />       structure Ml RNA ribozyme tRNA binding                 secondary structure ribonuclease P RNA, catalytic ele- <br />       center 16S rRNA E. coli. FEBS Lett. 220:283-287.               ment ribonucleoprotein enzyme. Cell 52:19-26. <br />  7.   Bowman, E. J., S. Altman. 1980. Identification ribonu-      31. Jones, E. W. 1977. Proteinase mutants Saccharomyces cere- <br />       clease P activity chick embryos. Biochim. Biophys. Acta           visiae. Genetics 85:23-33. <br />       613:439-447.                                                       32. Kline, R., S. Nishikawa, D. Soll. 1981. Partial purification of <br />  8. Brow, D. A., C. Guthrie. 1990. Transcription yeast U6           RNase P <span id='am-63' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-64'  ></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span>Schizosaccharomyces</span> pombe. J. Biol. Chem. <br />     snRNA gene requires pol III promoter element novel                256:5058-5063. <br />     position. Genes Dev. 4:1345-1356.                                    33. Kole, R., M. F. Baer, B. C. Stark, S. Altman. 1982. <br />  9. Castano, J. G., R. Ornberg, J. G. Koster, J. A. Tobian, M.           Escherichia coli RNase P required RNA component in <br />     Zasloff. 1986. Eukaryotic pre-tRNA 5&apos; processing nuclease:               vivo. Cell 19:881-887. <br />     copurification complex cylindrical particle. Cell 46:377-     34. Koski, R., A. Bothwell, S. Altman. 1976. Identification a <br />     387.                                                                     ribonuclease P-like activity human KB cell. Cell 9:101- <br /> 10. Davanloo, P., A. H. Rosenberg, J. J. Dunn, W. Studier.               106. <br />     1984. Cloning expression gene bacteriophage T7        35. Krupp, G., B. Cheravil, D. Frendewey, S. Nishikawa, D. <br />     RNA polymerase. Proc. Natl. Acad. Sci. USA 81:2035-2039.                 Soll. 1986. Two RNA <span id='am-82' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-83' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> copurify RNase P the <br /> 11. Doersen, C., C. Guerrier-Takada, S. Altman, G. Attardi.              fission yeast <span id='am-68' about='obo:NCBITaxon_4896' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_1 obo:NCBITaxon_147554 obo:NCBITaxon_34346 obo:NCBITaxon_4890 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4895 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-69' property="oboInOwl:hasExactSynonym" content="fission yeast" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces pombeP" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces malidevorans" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="Schizosaccharomyces pombe" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45042" datatype="xsd:string"></span><span id='am-76'  ></span>Schizosaccharomyces pombe</span>. EMBO J. 5:1697- <br />       1985. Characterization RNase P activity HeLa cell           1703. <br />       mitochondria. J. Biol. Chem. 260:5942-5949.                        36. Lawrence, N. P., S. Altman. 1986. Site-directed mutagenesis <br /> 12.   Engelke, D. R., P. Gegenheimer, J. Abelson. 1985. Nucleo-          Ml RNA, RNA subunit Escherichia coli ribonuclease <br />       lytic processing tRNAAr-tRNAASP dimeric precursor            P. J. Mol. Biol. 191:163-175. <br />       homologous component Saccharomyces cerevisiae. J.             37. Lee, J.-Y., D. R. Engelke. 1989. Partial characterization of <br />       Biol. Chem. 260:1271-1279.                                             RNA component copurifies Saccharomyces cere- <br /> 13.   Evans, C. F., D. R. Engelke. 1990. Yeast extracts tRNA         visiae RNase P. Mol. Cell. Biol. 9:1058-1064. <br />       gene transcription processing. Methods Enzymol. 181:439-       38. Leontis, N., A. DaLio, M. Strobel, D. Engelke. 1988. Effects <br />       450.                                                                   tRNA-intron structure cleavage precursor tRNAs by <br /> 14.   Garber, R. L., S. Altman. 1979. In vitro processing B.          RNase P Saccharomyces cerevisiae. Nucleic Acids Res. <br />       mori transfer RNA precursor molecules. Cell 17:389-397.                16:2537-2552. <br /> 15.   Gardiner, K., N. R. Pace. 1980. RNase P <span id='am-6' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span>Bacillus subtilis</span>   39. Ma, H., S. Kunes, P. J. Schatz, D. Botstein. 1987. Plasmid <br />       RNA component. J. Biol. Chem. 255:7507-7509.                    construction homologous recombination yeast. Gene <br /> 16.   Geiduschek, E. P., G. P. Tocchini-Valentini. 1988. Tran-           58:201-216. <br />       scription RNA polymerase III. Annu. Rev. Biochem. 57:           40. Manam, S., G. C. Van Tuyle. 1987. Separation charac- <br />       873-914.                                                               terization 5&apos;- 3&apos;-tRNA processing nuclease rat liver <br />  730     LEE ET AL.                                                                                                         MOL. CELL. BIOL. <br />  <br />     mitochondria. J. Biol. Chem. 262:10272-10279.                        52. Reich, C., K. J. Gardiner, G. J. Olsen, B. Pace, T. L. Marsh, <br /> 41. Maniatis, T., E. F. Fritsch, J. Sambrook. 1982. Molecular            N. R. Pace. 1986. The RNA component Bacillus <br />     cloning: laboratory manual. Cold Spring Harbor Laboratory,             subtilis RNase P. Sequence, activity partial secondary <br />     Cold Spring Harbor, N.Y.                                                 structure. J. Biol. Chem. 261:7888-7893. <br /> 42. Mller, D. L., N. C. Martin. 1983. Characterization          52a.Riles, L., M. Olson. Personal communication. <br />     yeast mitochondrial locus necessary tRNA biosynthesis:           53. Rose, M., P. Grisafi, D. Botstein. 1984. Structure and <br />     DNA sequence analysis identification new transcript.            function yeast URA3 gene: expression Escherichia <br />     Cell 34:911-917.                                                         coli. Gene 29:113-124. <br /> 43. Motamedi, H., K. Lee, L. Nichols, F. J. Schmidt. 1982. An        54. Rose, M. D., P. Novick, J. H. Thomas, D. Botstein, G. Fink. <br />     RNA species involved Escherichia coli ribonuclease P activ-            1987. A Saccharomyces cerevisiae genomic plasmid bank based <br />     ity. J. Mol. Biol. 162:535-550.                                          centromere-containing shuttle vector. Gene 60:237-243. <br /> 44. Newman, A. J., R. C. Ogden, J. Abelson. 1983. tRNA gene          55. Rothstein, R. J. 1982. One-step disruption yeast. Methods <br />     transcription yeast: effects specified base substitutions        Enzymol. 101:202-211. <br />     intragenic promoter. Cell 35:117-125.                            56. Saiki, R. K., D. H. Gelfand, S. Stoffel, S. J. Scharf, R. Higuchi, <br /> 45. O&apos;Connor, J. P., C. L. Peebles. 1991. In vivo pre-tRNA               G. T. Horn, K. B. Mullis, H. A. Erlich. 1988. Primer- <br />     processing Saccharomyces cerevisiae. Mol. Cell. Biol. 11:             directed enzymatic amplification DNA thermostable <br />     425-439.                                                                  DNA polymerase. Science 239:487-491. <br /> 46. Piper, P. E., J <br /> </body></html>