<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 16 August 2012                                           Nucleic Acids Research, 2012, Vol. 40, No. 20 10567&#8211;10575 <br />                                                                                                                 doi:10.1093/nar/gks756 <br />  <br />  <br /> Assessing biocompatibility click-linked <br /> DNA <span id='am-63' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-64' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-68'  ></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span>Escherichia coli</span> <br /> A. Pia Sanzone1, Afaf H. El-Sagheer1,2, Tom Brown1 Ali Tavassoli1,3,* <br /> 1 <br />  Chemistry, University Southampton, Southampton, SO17 1BJ, UK, 2Department Science and <br /> Mathematics, Chemistry Branch, Faculty Petroleum Mining Engineering, Suez Canal University, Suez, <br /> 43721, Egypt 3Cancer Research UK Centre, Faculty Medicine, University Southampton, Southampton, <br /> SO16 6YD, UK <br />  <br /> Received May 10, 2012; Revised July 14, 2012; Accepted July 16, 2012 <br />  <br />  <br />  <br /> ABSTRACT                                                                        inability incorporate epigenetic information, or <br />                                                                                 modi&#65533;?ed bases genes genomes. An <br /> The biocompatibility triazole mimic DNA                             alternative approach DNA assembly envisaged <br /> phosphodiester linkage <span id='am-79' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-85'  ></span>Escherichia</span> coli                             instead enzymes, uses highly ef&#65533;?cient chemical re- <br /> evaluated. The requirement selective pressure                               actions ligation oligonucleotides (5&#8211;7). Such an <br /> on click-containing gene probed                                   approach eliminate need enzymatic <br /> plasmid containing click DNA backbone linkages                               ligation cloning gene synthesis enable the <br /> each strand gene encoding fluorescent                                automation large-scale gene synthesis, also <br /> protein mCherry. The effect proximity                                 readily allow incorporation modi&#65533;?ed bases into <br /> click linkers biocompatibility                                large DNA fragments. The resulting click-linked DNA <br /> probed placing click DNA linkers 4-bp apart                              however, contain unnatural triazole linkage on <br /> at region encoding fluorophore fluor-                            backbone sites ligation (in place canon- <br />                                                                                 ical phosphodiester linker). <span id='am-45' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47'  ></span><span id='am-48' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> enzyme-free approach to <br /> escent protein. The resulting click-containing <br />                                                                                 DNA ligation use unnatural <br /> plasmid encode mCherry E. coli                               backbone linkage does adversely affect rate or <br /> a similar level canonical equivalent. The ability                        &#65533;?delity DNA replication transcription, func- <br /> of cellular machinery read                                       tioning normally biological systems. <br /> click-linked DNA probed using                                   To end, recently reported (8) rapid, clean <br /> above click-linked plasmid express mCherry                                   ef&#65533;?cient chemical ligation DNA oligonucleotides <br /> using vitro transcription/translation system,                             using copper-catalysed alkyne&#8211;azide cycloaddition <br /> and similar canonical                             (CuAAC) reaction (Figure 1) (9,10). The DNA sequence <br /> DNA. The yield fluorescence recombinant                                  linker ampli&#65533;?ed PCR high &#65533;?delity, <br /> mCherry expressed click-linked plasmid                                 TEM-1 b-lactamase gene (BLA) encoding ampicillin <br /> was compared canonical                                    resistance containing click linkers (83-bp apart, one <br />                                                                                 strand), functional E. coli at <br /> equivalent, same. The biocom- <br />                                                                                 similar levels equivalent plasmid canonical <br /> patibility click DNA ligation sites close proxim-                         DNA backbone. The click DNA backbone linker also <br /> ity non-essential gene demonstrated E. coli                             shown functional UvrB de&#65533;?cient strain of <br /> suggests possibility using click DNA ligation                            E. coli, suggesting nucleotide excision <br /> for enzyme-free assembly chemically                                      repair does play key role observed biocom- <br /> modified genes genomes.                                                     patibility (UvrB central component nucleotide <br />                                                                                 excision repair) (8). Further work shown T7 <br />                                                                                 RNA polymerase reads linker vitro, <br /> INTRODUCTION                                                                    effect click-linker &#65533;?delity <br /> Current methods assembly large DNA mol-                              rate process thoroughly <br /> ecules (e.g. genes) utilize mixture oligonucleotide syn-                   evaluated (11). <br /> thesis, PCR ampli&#65533;?cation enzymatic ligation.(1,2)                              The selective pressure exerted organism ampi- <br /> Although extremely powerful (3,4), approach                            cillin previously reported experiment (8) have <br /> several weaknesses, including additional time                           pre-selected colonies correctly read through <br /> resources required enzymatic steps,                             click-linker, possibly masking true &#65533;?delity the <br />  <br />  <br /> *To correspondence addressed. Tel: +44 2380 592395; Fax: +44 2380 593131; Email: a.tavassoli@soton.ac.uk <br />  <br /> &#223; The Author(s) 2012. Published Oxford University Press. <br /> This Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ <br /> by-nc/3.0), permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />  10568 Nucleic Acids Research, 2012, Vol. 40, No. 20 <br />  <br />  <br />  <br />  <br /> Figure 1. (A) Click DNA ligation. The CuAAC reaction used combination alkyne azide-modi&#65533;?ed DNA bases join oligo- <br /> nucleotides together. (B) The structure equivalent canonical DNA comparison triazole backbone natural phosphodiester. <br />  <br />  <br />  <br /> process; errors replication transcription                    automated trityl cation conductivity monitoring <br /> click-modi&#65533;?ed BLA gene result non-functional                    facility cases &gt;98.0%. All b-cyanoethyl <br /> b-lactamase subsequently prevent survival                    phosphoramidite monomers dissolved anhydrous <br /> the host organism ampicillin containing media,                     acetonitrile concentration 0.1 M immediately prior <br /> therefore observed. This account                     use. The coupling time normal A, G, C T <br /> high rates biocompatibility previously observed                   monomers 25 s. Cleavage oligonucleotides <br /> the click-containing BLA gene. With view using                       solid support deprotection achieved by <br /> click DNA ligation synthesis wide variety                 exposure concentrated aqueous <span id='am-91' about='obo:NCBITaxon_29189' typeof='obo:NCBITaxon_29178 owl:Thing obo:NCBITaxon_203397 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_69034 obo:NCBITaxon_29185 obo:NCBITaxon_543769 obo:NCBITaxon_2759'><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93'  ></span><span id='am-94' property="rdfs:label" content="Ammonia" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>ammonia</span> solution for <br /> genes, sought re-assess biocompatibility click-              60 min room temperature followed heating a <br /> linked DNA incorporated non-essential                       sealed tube 5 h 55  C. The oligonucleotides were <br /> gene encoding red-&#64258;uorescent protein mCherry (12).                    puri&#65533;?ed reversed-phase HPLC Gilson system <br /> As encoded &#64258;uorescent protein does exert selective                using XBridgeTM BEH300 Prep C18 10 mM <br /> pressure bacterial host, true &#65533;?delity                       10   250 mm column (Waters) gradient acetoni- <br /> click-DNA replication transcription                           trile ammonium acetate (0% 50% buffer B over <br /> determined. We sought assess effect click                  30 min, &#64258;ow rate 4 ml/min), buffer A: 0.1 M ammonium <br /> DNA linker proximity biocompatibility; click                       acetate, pH 7.0, buffer B: 0.1 M ammonium acetate, pH <br /> linkers previous study 83-bp apart, potentially               7.0, 50% acetonitrile. Elution monitored UV <br /> minimizing distortive effects double <span id='am-40' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-43'  ></span><span id='am-44' property="rdfs:label" content="Helix" datatype="xsd:string"></span>helix</span>.                    absorption 295 nm. After HPLC puri&#65533;?cation, oligo- <br /> Recent structural <span id='am-101' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-102' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-103'  ></span><span id='am-104' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> shown joining                    nucleotides desalted using NAP-10 columns and <br /> thymine residues, click DNA backbone linker causes                    analysed gel electrophoresis. <br /> thermodynamic destabilization duplex, spread                     The sequences SDM primers shown in <br /> the 4&#8211;5 bp surrounding incorporated triazole (13).                    Figure 2A; unmodi&#65533;?ed primers synthesized by <br /> Here, utilize site-directed mutagenesis (SDM)                  Euro&#65533;?ns MWG Operon (Ebersberg, Germany) the <br /> construction plasmid containing click DNA                        click-linked primers synthesized outlined below. <br /> linkers, 4-bp apart, gene encodes &#64258;uorescent              The template plasmid SDM pRSET-mCherry, <br /> protein mCherry. This click-linked plasmid used                     kindly provided Tsien laboratory. <span id='am-106' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-107' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Plasmids</span> were <br /> demonstrate biocompatibility click ligated                    isolated puri&#65533;?ed using QIAprep Spin Minipreps Kit <br /> DNA non-essential gene vitro E. coli.                      (QIAgen, Cat No. 27 106) following manufacturer&#8217;s <br />                                                                           instructions. All DNA sequencing carried by <br /> MATERIALS AND METHODS                                                     Euro&#65533;?ns MWG Operon (Ebersberg, Germany). <br />                                                                           Antibodies purchased GE Healthcare. <br /> General method oligonucleotide synthesis and <br /> puri&#65533;?cation <br />                                                                           Synthesis 30 -alkyne 50 -azide oligonucleotides <br /> Standard DNA phosphoramidites, solid supports add- <br /> itional reagents purchased Link Technologies                    (i) Synthesis 30 -alkyne oligonucleotides ODN-1 <br /> and Applied Biosystems Ltd. All oligonucleotides                     ODN-2 <br /> synthesized Applied Biosystems 394 automated                        30 -Alkyne oligos synthesized attachment <br /> DNA/RNA synthesizer using standard 0.2 1.0 mmol                      50 -O-(4,40 -dimethoxytrityl)  30 -O-propargyl-5- <br /> phosphoramidite cycle acid-catalysed detritylation,                    methyldeoxycytidine solid support according the <br /> coupling, capping iodine oxidation. Stepwise                          previously published method (8) packing resin <br /> coupling ef&#65533;?ciencies overall yields determined                   twist column (Glen Research) assembling the <br />                                                                       Nucleic Acids Research, 2012, Vol. 40, No. 20 10569 <br />  <br />  <br /> required sequence 30 - 50 -direction normal       template gel puri&#65533;?cation 0.8% agarose using the <br /> standard phosphoramidite synthesis.                            QIAquick Gel Extraction Kit (QIAgen) following the <br />                                                                manufacturer&#8217;s instructions. <br /> (ii) Synthesis 50 -azide oligonucleotides ODN-3 <br /> and ODN-4                                                      Transformation SDM product <br /> Oligonucleotides assembled 1.0 0.2 mmol <br /> scale (trityl-off) described general           A 5 ml aliquot puri&#65533;?ed SDM product added a <br /> method normal 50 -hydroxy group dC. To               50 ml aliquot KRX chemical competent cells (Cat. No. <br /> convert 50 -hydroxyl 50 -iodo resin, protected   L3002, Promega), incubated 30 min ice. The <br /> oligonucleotide attached synthesis column           mixture heat-shocked 42  C 30 s placed <br /> treated 0.5 M solution methyltriphenoxy-             ice 2 min. An 450 ml SOC added <br /> phosphonium iodide DMF (1.0 ml) period-           mixture cells incubated shaking at <br /> ically passed column 1 ml syringes         37  C 1 h. An 100 ml aliquots cell <br /> over 15 min room temperature. The column           suspension spread LB agar plates containing <br /> washed times dry DMF (14). To convert         ampicillin (100 mg/ml) 0.1% rhamnose. The plates <br /> 50 -iodo dC 50 -azido dC, sodium azide (50 mg) <span id='am-55' about='obo:NCBITaxon_9822' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9821 obo:NCBITaxon_35497 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58'  ></span><span id='am-59' property="rdfs:label" content="Sus" datatype="xsd:string"></span>sus</span>-     incubated overnight 37  C. <br /> pended dry DMF (1 ml), heated 10 min 70  C then <br /> cooled supernatant taken 1 ml          Assessing biocompatibility click-linked DNA <br /> syringe passed forth column           The number visible colonies plate counted <br /> then left room temperature overnight. The column            compared click-SDM plates the <br /> was washed DMF acetonitrile dried            positive negative control plates (these experiments <br /> by passing stream <span id='am-86' about='obo:NCBITaxon_354775' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_40096 obo:NCBITaxon_40092 obo:NCBITaxon_33208 obo:NCBITaxon_40093 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-87'  ></span><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Argon" datatype="xsd:string"></span>argon</span> gas (15).              repeated 10 times generate statistically signi&#65533;?cant <br /> Resultant 50 -azide oligonucleotides cleaved         data). Twenty colonies click-SDM plates were <br /> the solid support, deprotected puri&#65533;?ed HPLC          selected random grow overnight 5 ml LB supple- <br /> explained general method oligonucleotide       mented 100 mg/ml ampicillin, 37  C. The plasmid <br /> synthesis puri&#65533;?cation.                                     isolated puri&#65533;?ed mCherry gene <br />                                                                sequenced. Ten random colonies picked and <br /> Templated click ligation                                       assessed presence BamHI watermark by <br /> The alkyne, azide splint oligonucleotides (30.0 nmol       restriction digestion using BamHI-HF restriction endo- <br /> of each) 0.2 M NaCl (400 ml) annealed heating       nuclease (New England Biolabs) according manu- <br /> at 80  C 5 min cooled slowly room tem-         facturer&#8217;s instructions. <br /> perature. A solution CuI click catalyst prepared <br /> from tris-hydroxypropyltriazole ligand (22) (2.1 mmol       In vitro mCherry expression <br /> 0.2 M NaCl, 91.0 ml), sodium ascorbate (3.0 mmol            The S30 T7 High Yield Protein Expression System <br /> 0.2 M NaCl, 6.0 ml) CuSO4.5H2O (0.3 mmol 0.2 M          (Promega, L1110) used vitro protein expression. <br /> NaCl, 3.0 ml). The CuI click catalyst solution added    Reactions set following manufacturer&#8217;s in- <br /> the annealed oligonucleotides reaction mixture         structions, 1 mg puri&#65533;?ed SDM product was <br /> was kept room temperature 2 h. Reagents            used instead plasmid DNA template. The template <br /> removed using NAP-10 gel-&#65533;?ltration column              dialysed distilled water 2 h using Millipore <br /> ligated products analysed puri&#65533;?ed HPLC             &#65533;?lter (Merk Millipore &#8216;V&#8217; Series Membranes, 0.025 mm) <br /> and denaturing 20% polyacrylamide gel electrophoresis.         prior use. The reaction mixtures incubated at <br /> Yield HPLC puri&#65533;?cation 49&#8211;50%.                       37  C 1 h, shaking 300 rpm, followed the <br />                                                                addition 30 ml slurry Ni&#8211;NTA resin <br /> Site-directed mutagenesis                                      (Ni SepharoseTM 6 Fast Flow, GE Healthcare) <br /> PCR ampli&#65533;?cation carried follows: typical         pre-washed Ni wash buffer [20 mM Tris base, <br /> reaction performed total volume 50 ml, con-        20 mM Imidazole, 500 mM NaCl, 10%(v/v) Glycerol, <br /> taining 10 ng plasmid DNA, 0.3 mM mutagenic              1% (v/v) Triton X-100]. The mixtures incubated at <br /> primers, 5 ml 10  KOD Hot Start reaction buffer, 4 ml          4  C shaking 1 h. The Ni&#8211;NTA resin washed <br /> 1.5 mM MgSO4, 5 ml 0.2 mM dNTPs, 0.02 U/ml KOD                 times 1 ml Ni wash buffer. The bound protein <br /> Hot Start Polymerase (Novagen, Cat. No. 71 086-3). The         eluted 25 ml elution buffer [Tris base 20 mM, <br /> reactions hot-started heating 95  C 2 min.      Imidazole 500 mM, NaCl 200 mM, 5%(v/v) Glycerol, <br /> Each reaction subjected cycles 95  C       0.1% (v/v) Triton-X 100]. The product analysed by <br /> 20 s, 54  C 10 s 70  C 2 min. The solution     western blot. <br /> allowed cool room temperature 1 h. <br />                                                                Western blot analysis <br /> DpnI digestion puri&#65533;?cation SDM products <br />                                                                Samples resolved SDS&#8211;PAGE (12% polyacryl- <br /> DpnI restriction endonuclease (NEB, Cat. No. R0176L)           amide), protein transferred PROTRAN nitro- <br /> was directly added product ampli&#65533;?ca-       cellulose membrane (Bio-Rad). Membranes blocked <br /> tion reaction, incubated room temperature 6 h.      5% non-fat dry milk PBS-Tween 1 h, followed by <br /> The SDM product separated remaining               incubation 5% milk PBS-Tween 1:1000 <br />  10570 Nucleic Acids Research, 2012, Vol. 40, No. 20 <br />  <br />  <br />  <br />  <br /> Figure 2. (A) The click-oligonucleotides used site directed mutagenesis contained silent C A mutation (shown blue), introduces a <br /> BamHI restriction site present native mCherry gene. The click-linked bases shown red. (B) Assembly click-linked <br /> pRSET-mCherry plasmid site directed mutagenesis, introducing BamHI watermark. (C) Gel electrophoresis (0.8% agarose gel) SDM <br /> products (expected size 3577 bp); Lane 1, 2-log DNA ladder (New England Biolabs); lane 2, pRSET-mCherry SDM normal primers; lane 3, <br /> pRSET-mCherry SDM normal primers followed DpnI digestion; lane 4, pRSET-mCherry SDM using click primers; lane 5, pRSET-mCherry <br /> SDM using click primers followed DpnI digestion; lane 6, negative control (pRSET-mCherry SDM using water instead primers); lane 7, <br /> negative control followed DpnI digestion; lane 8, pRSET-mCherry template plasmid. <br />  <br />  <br />  <br /> dilution anti-His antibody overnight 4  C. This                  RESULTS AND DISCUSSION <br /> followed incubation 5% milk PBS-Tween                           Construction click-pRSET-mCherry plasmid SDM <br /> containing 1:1000 dilution horseradish peroxid- <br /> ase-conjugated secondary antibody 1 h room tem-                     An optimized SDM protocol (16) used incorpor- <br /> perature. ImmunStar Western C (Bio-Rad) used                        ate click linkers strand gene encoding the <br /> detect horseradish peroxidase-labelled secondary                       &#64258;uorescent protein mCherry (12). Forward reverse <br /> antibody, followed imaging analysis                            mutagenic primers, containing pair click-linked <br /> ChemiDoc MP imaging (Bio-Rad), quanti&#65533;?-                         cytosine nucleosides designed overlap the <br /> cation Image Lab software (Bio-Rad).                                  region encoding &#64258;uorophore mCherry (Figure <br />                                                                            2A), formed tripeptide MYG motif (me- <br />                                                                            thionine 71, tyrosine 72 <span id='am-30' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-34'  ></span>glycine</span> 73) protein. <br /> mCherry expression E. coli experiments                                  The primers introduced click linkers, 4-bp apart, <br /> The product SDM click-linked primers trans-                    region critical &#64258;uorescence mCherry. <br /> formed heat-shock KRX competent cells                           Once transformed E. coli, deletions muta- <br /> detailed above. The volume recovery mixture                     tions arising presence click linkers <br /> increased 5 ml LB media, grown OD600                     expected result easily identi&#65533;?able, <br /> of 0.6 reached. Recombinant protein expression                     non-&#64258;uorescent phenotype. The click SDM primers <br /> induced 18  C 0.1 mM IPTG, shaken 18  C                     designed introduce silent BamHI restric- <br /> overnight. The cells pelleted frozen  80  C.                   tion site, allow progeny click-linked plasmids to <br /> The cell pellet re-suspended 2 ml MHW buffer                     readily distinguishable parent <br /> [20 mM Tris base, 10 mM imidazole, 400 mM NaCl, 1 mM                       pRSET-mCherry plasmid (Figure 2A). Introduction of <br /> Na3VO4, 10 mM NaF, 1 mM DTT, Protease inhibitor                            watermark SDM (Figure 2B) changes <span id='am-50' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-51'  ></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>codon</span> <br /> cocktail tablets (Roche), 20% (v/v) Glycerol, 1% (v/v)                     used glycine MYG &#64258;uorophore GGC <br /> Triton X-100] lysed sonication. The lysate                      GGT, codons similar relative abun- <br /> centrifuged 8000 rpm 20 min supernatant                     dance E. coli (3.0% 2.8%, respectively), this <br /> applied 100 ml slurry Ni&#8211;NTA resin                              change expected signi&#65533;?cantly affect protein <br /> (Ni SepharoseTM 6 Fast Flow, GE Healthcare)                                expression levels. <br /> pre-washed Ni wash buffer [20 mM Tris base,                              The click-linked forward reverse mutagenic primers <br /> 20 mM Imidazole, 500 mM NaCl, 10%(v/v) Glycerol,                           synthesized CuAAC reaction an <br /> 1% (v/v) Triton X-100]. After 1 h incubation 4  C                       oligonucleotide 30 -alkyne modi&#65533;?cation and <br /> shaking, beads bounded protein washed                        50 -azide group (ODN-1 ODN-3 or <br /> three times 1 ml Ni wash buffer eluted                       ODN-2 ODN-4, Table 1) presence splint <br /> 25 ml elution buffer [Tris base 20 mM, Imidazole                        (ODN-5 ODN-6 respectively, Table 1) CuI <br /> 500 mM, NaCl 200 mM, 5%(v/v) Glycerol, 0.1% (v/v)                          catalyst. In presence splint, conversion of <br /> Triton-X 100]. The puri&#65533;?ed protein analysed using                      CuAAC reaction quantitative. The <br /> 12% polyacrylamide SDS&#8211;PAGE Tecan M200pro                            click-linked oligonucleotides puri&#65533;?ed HPLC <br /> &#64258;uorescent plate reader.                                                   (to remove splint) used subsequent experiments. <br />                                                                                     Nucleic Acids Research, 2012, Vol. 40, No. 20 10571 <br />  <br />  <br />    The plasmid used study pRSET-                                puri&#65533;?ed gel electrophoresis ensure absence of <br /> mCherry,(12) high copy number plasmid pUC                          template subsequent reactions. <br /> origin encodes mCherry control bac- <br /> teriophage T7 promoter. The high copy number                         Biocompatibility pRSET-click-mCherry E. coli <br /> plasmid maximizes number times click-linked <br /> DNA replicated, propagating arising                        The puri&#65533;?ed products SDM reactions <br /> errors replication. The click-containing primers                      (click, positive control negative control) initially <br /> canonical equivalents (as positive control)                            transformed high-ef&#65533;?ciency E. coli DH5a assess the <br /> incorporated pRSET mCherry using pfu poly-                      yield SDM protocol primer type. As the <br />                                                                             strain DH5a used lacked chromosomal copy T7 <br /> merase, KOD polymerase. The procedure also <br />                                                                             RNA polymerase, red-&#64258;uorescent phenotype expected <br /> repeated water instead mutagenic primers as <br />                                                                             mCherry (whose expression control a <br /> a negative control. The yield SDM product found <br />                                                                             bacteriophage T7 promoter) observed. The <br /> to signi&#65533;?cantly higher KOD polymerase <br />                                                                             SDM product using click-linked primers KOD <br /> (Supplementary Data), line previous reports.(17) <br />                                                                             polymerase resulted average 179 colonies LB <br /> KOD polymerase used following ex- <br />                                                                             agar plate supplemented 100 mg/ml ampicillin, the <br /> periments. The products subsequently digested <br />                                                                             positive control SDM canonical DNA resulted in <br /> with DpnI restriction endonuclease remove the <br />                                                                             185 colonies, negative control SDM water <br /> methylated template plasmid; note absence                            place mutagenic primers resulted colonies <br /> plasmid template (band  2 kb, reference lane 8)                       (Supplementary Table 1). In <span id='am-60' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-61' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> utilize red- <br /> lanes 3, 5 7 compared presence lanes 2, 4                     &#64258;uorescent phenotype rapidly identify functional <br /> and 6 Figure 2C. As expected, SDM                           copies mCherry gene, SDM products <br /> product present negative control reaction (lanes 6                   transformed KRX strain E. coli <br /> and 7, Figure 2C). The incubation time DpnI                        (Promega), high transformation ef&#65533;?ciency derivative <br /> optimized (Supplementary Data), plasmid                            express T7 RNA polymerase control a <br /> template fully degraded 6 h. The SDM product                      rhamnose promoter. Transformation SDM <br />                                                                             product click-linked DNA primers resulted 78 <br />                                                                             colonies, compared 84 colonies positive <br /> Table 1. Oligonucleotides used synthesis click-linked        control using native mutagenic primers. All colonies <br /> mutagenic primers                                                           visibly pink colour displayed red &#64258;uorescence <br />                                                                             viewed UV light (Figure 3A). A single colony <br /> Code                                  Oligonucleotide sequences (50 &#8211;30 ) <br />                                                                             observed plate containing transformants the <br /> ODN-1                                 AGTTCATGTACGGATMeCk                   negative control SDM product, indicating number of <br /> ODN-2                                 CGTAGGCCTTGGATMeCk                    colonies arising background levels parent <br />                                       z <br /> ODN-3 <br />                                       z <br />                                        CAAGGCCTACG                          plasmid. These experiment repeated 10 times <br /> ODN-4                                  CGTACATGAACT                         (Supplementary Table 2) generate statistically signi&#65533;?- <br /> ODN-5                                 CATGTACGGATCCAAG <br /> ODN-6                                 AGGCCTTGGATCCGTA                      numbers; colonies containing triazole-containing <br />                                                                             plasmids red cases, contained 92.7 &#177; 8.3% <br /> Ck, alkyne-modi&#65533;?ed cytidine, Cz, azide-modi&#65533;?ed cytidine.                    colonies native plates, negative <br />  <br />  <br />  <br />  <br /> Figure 3. Biocompatability click DNA E. coli. (A) The plate left contains transformants undergone SDM native primers, <br /> and plate right shows transformants SDM triazole-containing primers. Ten replicates experiment performed, with <br /> all colonies showing expected mCherry &#64258;uorescence viewed UV light source. (B) Comparison number colonies the <br /> water control (W), native (N) triazole (T) plates. The triazole plates contained 92.7 &#177; 8.3% colonies native plates, the <br /> negative control plate contained &lt;1%. <br />  10572 Nucleic Acids Research, 2012, Vol. 40, No. 20 <br />  <br />  <br />  <br />  <br /> Figure 4. (A) (B) Representative sequencing data mCherry gene colonies triazole DNA plates. The forward strand shown <br /> in (A), complementary strand shown (B), location triazole backbone linkers shown strand. (C) Lane 1 2-Log <br /> DNA ladder (New England Biolabs), lane 2 BamHI restriction digestion native pRSET mCherry plasmid, lane 3 progeny the <br /> click-modi&#65533;?ed variant. The presence additional BamHI watermark incorporated click-SDM primers results highlighted 229-bp <br /> fragment lane 3. <br />  <br />  <br /> control plates containing &lt; 1% colonies                             transcription/translation used. As vitro <br /> (Figure 3B).                                                               contains T7 RNA polymerase S30 Extract <br />    To con&#65533;?rm origin plasmid colonies                 components required translation (18&#8211;20), in- <br /> transformed product SDM click-linked                      formation inferred effect click-linker <br /> primers, 20 colonies picked pRSET-                          &#65533;?delity rate transcription linker <br /> mCherry plasmid over-expressed, isolated sequenced.                    T7 RNA polymerase, re&#64258;ected quantity of <br /> In cases examined, forward reverse                        &#64258;uorescent mCherry produced system; protein <br /> strands contained BamHI watermark, errors                      quantity lower canonical DNA <br /> observed sequence mCherry gene                      T7 RNA polymerase slowed click linker, while <br /> (Figure 4A B). The parental origin additional 10                 errors translation skipping the <br /> colonies probed restriction digestion BamHI.                   click-linked bases polymerase result loss <br /> The native pRSET-mCherry plasmid contains BamHI                          &#64258;uorescent phenotype protein product. It <br /> restriction site multiple cloning site prior start               noted systems optimized use <br /> of mCherry; progeny click-containing plasmids                  plasmid DNA, products SDM typ- <br /> contain additional BamHI watermark introduced                       ically sub-cloned over-expressed E. coli prior use <br /> the mutagenic click-linked primers region encoding                  templates vitro protein production. In case of <br /> the MYG &#64258;uorophore. Restriction digestion                          click-pRSET-mCherry however, over-expression this <br /> plasmids BamHI result bands,                    plasmid sub-cloning result loss the <br /> one 229 bp 3348 bp, digestion                 click DNA-linker, DNA replication E. coli results <br /> the native plasmid lead single band 3577 bp.                  phosphodiester backbone. We sought to <br /> Two bands observed colonies examined                      directly use SDM product template for <br /> the click-SDM plate, demonstrating presence                     vitro transcription/translation. As quantity of <br /> BamHI watermark (Figure 4C), directly linking                          puri&#65533;?ed DNA isolated reaction typically <br /> their heritage click-containing SDM primers.                         0.1 mg, (10-fold lower 1 mg plasmid recom- <br />    The &#65533;?delity click-DNA replication                        mended manufacturer), 10 SDM reactions (of each) <br /> probed analysing progeny click-pRSET                         carried parallel click-linked primers, <br /> mCherry plasmids single colony. The trans-                        canonical primers (positive control) water (negative <br /> formed click-containing plasmid replicated multiple                     control). The products set pooled used as <br /> times bacterium, errors process                    template vitro transcription/translation of <br /> be observed isolating, re-transforming assessing                    mCherry. The SDM product used directly as <br /> plasmids single colony. Plasmids 10                            template vitro transcription/translation, treat- <br /> click-transformed colonies isolated re-trans-                     ment SDM product T4 DNA ligase did not <br /> formed KRX cells. All resulting colonies displayed                    affect yield reaction (Supplementary Figure S5). <br /> the red-&#64258;uorescent phenotype associated correct                       The mCherry protein produced reaction readily <br /> replication click-containing mCherry gene                           visualized quanti&#65533;?ed western blot; yield of <br /> (Supplementary Figure S4).                                                 vitro mCherry production click-SDM <br />                                                                            product comparable canonical-SDM <br />                                                                            product (Figure 5A), &#64258;uorescence emission <br /> Protein expression click-pRSET-mCherry <br />                                                                            associated mCherry 610 nm (Figure 5B), no <br /> To probe compatibility click-DNA linker                    product &#64258;uorescence observed negative control <br /> the transcription machinery, vitro coupled                           template. These results indicate transcription from <br />                                                                                       Nucleic Acids Research, 2012, Vol. 40, No. 20 10573 <br />  <br />  <br />  <br />  <br /> Figure 5. (A) Western blot product vitro transcription/translation using SDM products water (lane 1), canonical primers (lane 2) <br /> and click-linked primers (lane 3), template. The intensity band quanti&#65533;?ed blot (average repeats). (B) Fluorescent <br /> emission mCherry (at 610 nm) vitro transcription/translation pRSET-mCherry constructed SDM click-linked primers (red line), <br /> normal primers (blue line); &#64258;uorescence observed negative control (black line). <br />  <br />  <br />  <br /> the click linker vitro results functional mCherry                      primers (Figure 6B). As expected solution the <br /> protein, similar quantity equivalent                     negative control plasmid &#64258;uorescent. The rate of <br /> canonical DNA.                                                               mCherry expression normal click-linked <br />    The effect click DNA linker DNA replication                     SDM products assessed monitoring protein <br /> and transcription E. coli probed                           levels 1, 2, 3, 4 12 h IPTG induction, and <br /> comparing &#64258;uorescence mCherry protein                             comparable (Supplementary Figure S6). The <br /> produced click-linked DNA canon-                       similarity mCherry quantity quality produced from <br /> ical equivalent. Rather determine &#64258;uorescence                    click-linked plasmid equivalent canonical <br /> mCherry individual colonies,                            DNA, vitro vivo experiments <br /> expected stochastic variation (21), sought                     (Figures 5 6) demonstrates biocompatibility <br /> assess total mCherry levels population trans-                      product click DNA-backbone ligation and <br /> formants. Typically starter culture single                          con&#65533;?rms potential alternative enzymatic <br /> colony used protein expression (to ensure genomic                     DNA ligation. <br /> uniformity); case however, transform- <br /> ation recovery mixture used starter culture <br /> for protein expression. This allowed total mCherry expres-                   CONCLUSIONS <br /> sion population transformants assessed, <br /> averaging variations mCherry levels single                        An essential prerequisite use chemically <br /> cells. The product SDM click primers, native                         modi&#65533;?ed DNA molecular biology modi&#65533;?ca- <br /> primers water transformed KRX competent                          tion benign, modi&#65533;?ed DNA a <br /> cells, recovery 1 h, volume solution                functional mimic natural counterpart. The triazole <br /> was increased 5 ml LB media, cells grown                    linkage Figure 1a ful&#65533;?ls criterion surprisingly well, <br /> until OD600 0.6. IPTG added induce mCherry                            despite apparent dissimilarity normal phospho- <br /> expression, resulting protein af&#65533;?nity                            diester bridge; triazole linkage accurately ef&#65533;?- <br /> puri&#65533;?ed. As transformants placed                          ciently read DNA RNA polymerases <br /> selective pressure maintain plasmid encoding                          (8,11), making click DNA ligation promising alternative <br /> mCherry, &#64258;uorescent protein produced                           enzymatic ligation. Here demonstrated full <br /> will affected replication rate transformed                  biocompatibility click DNA-backbone linker a <br /> plasmid, informing effect click DNA                            non-essential gene, determining presence select- <br /> linker process. The isolated protein product                    <span id='am-1' about='obo:NCBITaxon_1148155' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_72037 obo:NCBITaxon_131567 obo:NCBITaxon_197562 obo:NCBITaxon_1148154 obo:NCBITaxon_197563 obo:NCBITaxon_6830 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Ive" datatype="xsd:string"></span><span id='am-5'  ></span>ive</span> pressure required correct function in <br /> the click-SDM native plasmids visibly pink                        E. coli. That click-linked mCherry gene replicated <br /> colour, negative control solution colour-                    transcribed high &#65533;?delity despite close <br /> less (Figure 6A, insert). Analysis puri&#65533;?ed protein                    proximity triazole backbone linkers (4-bp apart), <br /> by SDS&#8211;PAGE showed band 30 kDa, line                            demonstrates robustness click DNA <br /> the mass expected mCherry (Figure 6A). The function-                     backbone linker mimics canonical DNA. These &#65533;?ndings <br /> ality &#64258;uorescent protein produced                           suggest possibility alternative, fully synthetic <br /> plasmid assessed comparing &#64258;uorescence                         approach gene synthesis potential signi&#65533;?- <br /> 2 ml mCherry solution using &#64258;uorescence plate                      cance synthetic biology industrial biotechnology. <br /> reader. When excited 587 nm, protein                                  This approach particularly relevant synthesis <br /> expressed click-pRSET-mCherry showed similar                            large DNA strands containing site-speci&#65533;?c epigenetic <br /> levels &#64258;uorescent emission 610 nm                     modi&#65533;?cations, DNA constructs dif&#65533;?cult to <br /> pRSET-mCherry constructed SDM normal                                 make PCR-based methods. <br />  10574 Nucleic Acids Research, 2012, Vol. 40, No. 20 <br />  <br />  <br />  <br />  <br /> Figure 6. (A) SDS&#8211;PAGE Ni2+ af&#65533;?nity puri&#65533;?ed mCherry (30 kDa) colonies transformed SDM product water (negative control, <br /> lane 2), canonical (positive control, lane 3), click-linked primers (lane 4). Lane 1 NEB ColorPlus Prestained Protein Ladder insert shows <br /> the readily visible mCherry protein product 30 kDa using normal (left) click-linked primers (right) white light. (B) Fluorescent emission <br /> of recombinant mCherry (610 nm) expressed pRSET-mCherry constructed SDM click-linked primers (red line), normal primers (blue <br /> line) water control (black line). Insert shows puri&#65533;?ed mCherry pRSET-mCherry constructed SDM normal (left) click-linked <br /> (right) primers, UV light 365 nm. <br />  <br />  <br /> SUPPLEMENTARY DATA                                                                Brownley,A., Thomas,D.W., Algire,M.A. et al. (2008) Complete <br />                                                                                   chemical synthesis, assembly, cloning <span id='am-6' about='obo:NCBITaxon_2093' typeof='owl:Thing obo:NCBITaxon_2092 obo:NCBITaxon_1 obo:NCBITaxon_2085 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_544448 obo:NCBITaxon_31969'><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Haemobartonella" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:4" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="PMID:8863441" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Eperythrozoon" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Bovimyces" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Borrelomyces" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="PMID:15176735" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Asteromyces" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="NCBITaxon:57371" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="PMID:11411711" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="NCBITaxon:29500" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="PMID:11321109" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="PMID:16350067" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Pleuropneumonia" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="PMID:13403276" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="PMID:10826816" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Mycoplasma" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Asterococcus" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="PMID:16403858" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="PMID:11931184" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="PMID:8995799" datatype="xsd:string"></span>Mycoplasma</span> <br /> Supplementary Data available NAR Online:                                   genitalium genome. Science, 319, 1215&#8211;1220. <br /> Supplementary Tables 1&#8211;3, Supplementary Figures 1&#8211;6                            4. Gibson,D.G., Glass,J.I., Lartigue,C., Noskov,V.N., Chuang,R.Y., <br /> and Supplementary Methods.                                                        Algire,M.A., Benders,G.A., Montague,M.G., Ma,L., <br />                                                                                   Moodie,M.M. et al. (2010) Creation bacterial cell Controlled <br />                                                                                   chemically synthesized genome. Science, 329, 52&#8211;56. <br />                                                                                5. El-Sagheer,A.H. Brown,T. (2009) Synthesis polymerase <br /> ACKNOWLEDGEMENTS                                                                  chain reaction ampli&#65533;?cation DNA strands containing an <br />                                                                                   unnatural triazole linkage. J. Am. Chem. Soc., 131, 3958&#8211;3964. <br /> The authors thank Professor <span id='am-96' about='obo:NCBITaxon_1009818' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_263623 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_71526 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_7080 obo:NCBITaxon_50557 obo:NCBITaxon_115347 obo:NCBITaxon_7041'><span id='am-97' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-99'  ></span><span id='am-100' property="rdfs:label" content="Roger" datatype="xsd:string"></span>Roger</span> Tsien providing                          6. Herrlein,M.K., Nelson,J.S. Letsinger,R.L. (1995) A covalent <br /> pRSET-mCherry, Dr Patrick Duriez Dr Fedor                                     lock self-assembled oligonucleotide conjugates. J. Am. Chem. <br />                                                                                   Soc., 117, 10151&#8211;10152. <br /> Forafonov helpful discussions, Dr Paul                                 7. Xu,Y.Z. Kool,E.T. (1998) Chemical enzymatic properties <br /> Duckmanton taking photograph Figure 6B.                                bridging 5&#8217;-S-phosphorothioester linkages DNA. Nucleic <br />                                                                                   Acids Res., 26, 3159&#8211;3164. <br />                                                                                8. El-Sagheer,A.H., Sanzone,A.P., Gao,R., Tavassoli,A. and <br />                                                                                   Brown,T. (2011) Biocompatible arti&#65533;?cial DNA linker read <br /> FUNDING                                                                           DNA polymerases functional Escherichia <br />                                                                                   coli. Proc. Natl Acad. Sci. USA, 108, 11338&#8211;11343. <br /> <span id='am-35' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-39'  ></span>Cancer</span> Research UK [Career Establishment Award 10 263                          9. Rostovtsev,V.V., Green,L.G., Fokin,V.V. Sharpless,K.B. <br /> to A.T.]; Engineering Physical Sciences Research                              (2002) A stepwise huisgen cycloaddition process: copper(I)- <br /> Council [EP/H04986X/1 A.T.]; Southampton                                       catalyzed regioselective &#8216;&#8216;ligation&#8217;&#8217; azides terminal alkynes. <br /> University Life Sciences Interface [studentship A.P.S.                        Angew. Chem. Int. Ed. Engl., 41, 2596&#8211;2599. <br /> to A.T.]; Biotechnology Biological Sciences                           10. Tornoe,C.W., Christensen,C. Meldal,M. (2002) <br />                                                                                   Peptidotriazoles solid phase: [1,2,3]-triazoles regiospeci&#65533;?c <br /> Research Council [sLOLA grant BB/J001694/1                                     copper(i)-catalyzed 1,3-dipolar cycl <br /> </body></html>