<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Apr. 1992, p. 1879-1892                                                                   Vol. 12, No. 4 <br /> 0270-7306/92/041879-14$02.00/0 <br /> Copyright C) 1992, American Society Microbiology <br />  <br />  <br />         A Presumptive Helicase (MOTi Gene Product) Affects Gene <br />            Expression Is Required Viability Yeast <br />                         <span id='am-53' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-54'  ></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br />                          JENNIFER L. DAVIS,t RIYO KUNISAWA, AND JEREMY THORNER* <br />                 Division Biochemistry Molecular Biology, Department Molecular Cell Biology, <br />                          Room 401, Barker Hall, University <span id='am-32' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-37'  ></span>California</span>, Berkeley, California 94720 <br />                                         Received 25 November 1991/Accepted 8 January 1992 <br />  <br />             Exposure haploid yeast cell mating pheromone induces transcription set genes. Induction is <br />          mediated <span id='am-122' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-123' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-124' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-126'  ></span>cis</span>-acting DNA sequence upstream pheromone-responsive genes. Although the <br />          STE12 gene product binds specifically sequence element required maximum levels both <br />          basal induced transcription, pheromone-responsive genes regulated identical manner. To <br />          investigate additional factors play role transcription genes, genetic screen used <br />          identify mutants able express pheromone-responsive genes constitutively absence Stel2. In <span id='am-91' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-94'  ></span><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>this</span> <br />          way, identified recessive, single gene mutation (mot], modifier transcription) increases the <br />          basal level expression several, all, pheromone-responsive genes. The motl-l allele relaxes the <br />          requirement <span id='am-88' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-89' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> upstream activating sequence enhances expression another <br />          gene previously thought involved mating pathway. Cells carrying mot1-l grow slowly 30&#176;C <br />          inviable 38&#176;C. The MOT] gene cloned complementation temperature-sensitive <br />          lethality. Construction null allele confirmed MOTI essential gene. MOT] resides chromosome <br />          XVI encodes large protein 1,867 amino acids contains seven conserved domains found <br />          known putative helicases. The product MOT] strikingly homologous <span id='am-70' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-71' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-73'  ></span><span id='am-74' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae <br />          SNF2ISWI2 RAD54 gene products entire helicase region. <br />  <br />    Coordinate transcriptional regulation appropriate set            induced transcription pheromone-responsive genes (23, <br /> of genes essential feature response cells              24), requirement Stel2 basal transcription not <br /> extracellular signals. In eukaryotes, concurrent expres-             absolute. Fields et al. (23) demonstrated phero- <br /> sion gene set involves specific upstream DNA                   mone-responsive genes produced markedly lower, but <br /> sequence elements regulatory proteins bind              readily detectable, levels mRNA stel2 null strain than <br /> elements interact complex manner general                    isogenic STE12+ control, stel2 <br /> transcription factors. One useful investigating                strain, transcription pheromone-responsive genes <br /> such regulation mating response pathway                     undetectable. In addition, Errede Ammerer (19) <br /> unicellular eukaryote Saccharomyces cerevisiae. Haploid                   observed apparent composition Stel2-dependent <br /> cells yeast exist cell types (A 4Ta              DNA-binding complexes varied pheromone-respon- <br /> M Ta). Each cell type secretes small oligopeptide phero-                sive promoter examined. These findings suggest all <br /> mone (AL4Ta cells secrete a-factor; AM Ta cells secrete                   pheromone-responsive genes regulated identical <br /> a-factor) acts cells opposite mating type              manner additional factors exist differ- <br /> induce programmed set physiological changes, including               entially modulate transcription various members this <br /> arrest cell growth G1 phase cell cycle,                  set genes. <br /> synthesis cell surface agglutinins, characteristic alter-            To expand understanding components that <br /> ations cell morphology. Collectively, changes per-               govern expression pheromone-responsive genes, de- <br /> mit cell fusion diploid formation (for recent review,           vised genetic screen identify factors operate <br /> reference 68).                                                            downstream of, independently of, Stel2. Here de- <br />    Response mating pheromone involves transcrip-                   scribe isolation mutation increases basal-level <br /> tional activation (and, cases, repression) set            expression pheromone-responsive genes the <br /> genes, termed pheromone-responsive genes, ini-                  absence Stel2 absence pheromone. We have <br /> tiate mating program (16, 68, 69). The product                 isolated corresponding wild-type gene determined <br /> STE12 gene protein binds specifically (17, 19)              encodes unusually large, previously <span id='am-183' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-184'  ></span><span id='am-185' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-188' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-191' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-192' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-193' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-194' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-196' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-197' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-198' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-199' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span>unidentified</span> <br /> an 8-bp upstream activating sequence (UAS)                   protein required cell growth viability. <br /> pheromone-responsive promoters copies and <br /> in orientation (44, 74). This UAS, called phero- <br /> mone response element (PRE), consensus sequence                                 MATERIALS AND METHODS <br /> 5&apos;-ATGAAACA-3&apos; (44, 74). <br />    Although Stel2 protein required basal                 Strains, media genetic methods. The yeast strains used <br />                                                                           study listed Table 1. Genetic manipulations and <br />                                                                           culture conditions described Sherman et al. <br />   * <br />     Corresponding author.                                                 (63) twice recommended levels nutritional <br />   t Present address: Somatic Therapy Corp., 850 <span id='am-157' about='obo:NCBITaxon_104321' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_163718 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-158' property="oboInOwl:hasExactSynonym" content="false prairie-clover" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-160'  ></span><span id='am-161' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-162' property="rdfs:label" content="Marina" datatype="xsd:string"></span>Marina</span> Village            supplements used synthetic media. Plates con- <br /> Parkway, Alameda, CA 94501.                                               taining 0.02% 5-bromo-4-chloro-3-indolyl-,B-D-galactoside <br />                                                                    1879 <br />  1880        DAVIS ET AL.                                                                                                           MOL. CELL. BIOL. <br />  <br />                      TABLE 1. Yeast strains used                                                    TABLE 2. Gene probes used <br />    Strain                         Genotype                           Source        Gene                      Probe                       Reference(s) <br /> EG123            MA Ta leu2-3, 112 ura3-52 trpl-l his4-           S. Fields       CMD1          0.8-kb EcoRI-StyI fragment                 13 <br />                    519 canl-101                                                   DEDI          1.8-kb BamHI fragment                      70 <br /> SF167-5aa        EG123 stel2::LEU2                                S. Fields       <span id='am-152' about='obo:NCBITaxon_1055228' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_66535 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_144794 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_75543 obo:NCBITaxon_6656 obo:NCBITaxon_41085 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-153' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-154'  ></span><span id='am-155' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-156' property="rdfs:label" content="Fusi" datatype="xsd:string"></span>FUSI</span>          0.5-kb NheI-SphI fragment                  72 <br /> 1788             MATa/a leu2-3,112/leu2-3,112 ura3-               D. Levin        MFal          1.6-kb BamHI fragment                      6 <br />                    52/ura3-52 trpl-lltrpl-l his4-519/                             MFa2          1.1-kb HpaI fragment                       6 <br />                    his4-519 canl-101/canl-101                                     PKC1          23-base specific oligonucleotide           48 <br /> JD195b           MATa isogenic EG123                           This work       STE2          14-base specific oligonucleotide           9, 52 <br /> JD209C           JD195 stel2::LEU2                                This work       STE6          0.8-kb SpeI fragment                       45 <br /> JD100            SF167-5a motl-l                                  This work       SST2          1.3-kb ClaI-NheI fragment                  12, 15 <br /> JD102Cd          AL4Ta motl-l leu2-3, 112 ura3-52                 This work <br />                    trpl-1 his4-519 canl-101 <br /> JD22/23e         MATa/a motl-lImotl-l leu2-3, 112/                This work <br />                    leu2-3,112 ura3-52/ura3-52 trpl-1/                                Plasmid constructions gene probes. All recombinant <br />                    trpl-1 his4-519/his4-519 canl-1011                             DNA methods performed standard procedures (50). <br />                    canl-101                                                       Plasmid pLGA178 (29) digested URA3 marker <br /> JD215Af          MATa isogenic JD102C                          This work       ApaI, converted flush ends incubation T4 <br /> JD215Bf          ALTa isogenic JD102C                          This work <br /> JD227Af          MATa stel2::LEU2 isogenic                     This work       DNA polymerase, ligated BglII fragment containing <br />                    JD102C                                                         TRPI marker, excised plasmid p330 (73), and <br /> JD216Df          M4Ta isogenic JD227A                          This work       converted blunt ends incubation deoxyribonucle- <br /> YPH4999          MATa ura3-52 lys2-801a&apos; ade2-101&quot;c               P. Hieter       otides Klenow fragment E. coli DNA polymerase <br />                    trpl-A63 his3-A200 leu2-AI                                     I, generate plasmid pLGA178T. Plasmid pLGA&amp;178T con- <br /> YPH5019          MATa/ot ura3-52/ura3-52 lys2-801a,/              P. Hieter       fers tryptophan, uracil, prototrophy recipient trpl <br />                    lys2-801am ade2-101oclade2-101oc                               ura3 yeast cells. To construct pJD11, XhoI cassette <br />                    trpl-A63/trpl-A63 his3-A200/his3-                              containing tandem consensus PRE boxes (provided by <br />                    A200 leu2-Al/leu2-Al                                           F. Gimble) inserted XhoI site pLGA178T. <br />     Construction strain described (25). <br />   b Constructed by <br />                                                                                   (The pJD11 plasmid contains tandem head-to-tail inserts <br />                      DNA-mediated transformation EG123 plasmid-         PRE consensus sequences noncoding <br /> borne copy HO gene (35, 61).                                               strand.) Plasmid pSTE12(URA3) contains Sacl fragment <br />   &apos; <br />     Constructed one-step gene replacement method (60) using a <br /> stel2::LEU2 plasmid (pLB1367), generously provided G. Ammerer.                 bearing C-terminal 18 amino acids the <br />   d A <br />        haploid segregant derived second backcross strain JD100        STE12 coding sequence (19), isolated plasmid pLB1212 <br /> carrying pSTE12(URA3) EG123 (see Materials Methods).                       (generously provided G. Ammerer), inserted pRS316 <br />   e Two haploid segregants opposite mating type derived described        (65). Plasmid pSTE12(URA3) fully restores mating compe- <br /> footnote d mated. <br />   f Haploid segregants derived backcross JD102C SF167-Sa             tence stel2 strains (<span id='am-163' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-164' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-167'  ></span>data</span> shown). <br /> carrying pSTE12(URA3) (see Materials Methods). Prior sporulation           DNA probes used study listed Table 2. After <br /> the final resulting diploid, cells grown nonselectively plated        cleavage indicated restriction enzymes, DNA frag- <br /> medium containing 5-fluoroorotic acid (4) identify cells lost   ments purified gel electrophoresis, recovered by <br /> pSTE12(URA3). <br />   g Derivation strains described (65).            electroelution, labeled random-primer (hexanu- <br />                                                                                   cleotide primers; Pharmacia) labeling method (22) and <br />                                                                                   [al-32P]dCTP (Amersham). Oligonucleotide probes la- <br /> (X-Gal) (Bachem) prepared using synthetic medium                          beled T4 polynucleotide kinase [y_- 3P]ATP (Amer- <br /> buffered pH 7.0 50 mM potassium phosphate. DNA-                           sham). Excess unincorporated <span id='am-130' about='rdfs:label' typeof='owl:Thing'>label</span> removed the <br /> mediated transformation yeast cells performed                       probes using NICK columns (Pharmacia). Restriction <br /> lithium acetate procedure (39). Plasmid genomic DNAs                          enzymes DNA-modifying enzymes purchased <br /> were isolated yeast cells method Hoffman                       Stratagene, Pharmacia, Boehringer-Mannheim, New <br /> Winston (38), <span id='am-173' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-174' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-177' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> introduced <span id='am-115' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-116' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-120'  ></span><span id='am-121' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span>Escherichia</span>                       England Biolabs used according manufacturers&apos; <br /> coli method Chung Miller (10).                                      recommendations. <br />   E. coli HB101 (5) used propagation                             Isolation suppressors stel2. Multiple cultures strain <br /> plasmids. Strains DH5aF&apos; (33) JM101 (51) used                        SF167-5a carrying pJD11 grown selective condi- <br /> the propagation certain plasmids preparation                       tions (no Trp) -2 x 108 cells ml treated with <br /> of single-stranded double-stranded DNAs sequenc-                          various amounts ethyl methanesulfonate (Kodak) accord- <br /> ing. Standard methods culture conditions used                            ing standard protocols (63). Mutagenized cells plated <br /> (50).                                                                             synthetic complete medium lacking Trp cell densities <br />    Chromosomal assignment determined hybridiza-                            yield isolated single colonies. Plates incubated at <br /> tion MOTI probe intact yeast chromosomes                        17&#176;C (for 7 days), 30&#176;C (for 3 days), 37&#176;C (for 2 days) and <br /> separated pulsed-field gel electrophoresis (75) trans-                     replica plated X-Gal-containing plates. The indi- <br /> ferred nitrocellulose. Assignment chromosome XVI                            cator plates incubated 30&#176;C 1 2 days, and <br /> was confirmed (i) rehybridizing filter (after                         colonies visually inspected blue color. The motl-l <br /> stripping reexposing film ensure complete removal                       mutation described isolated culture cells <br /> of probe) gene probe (PEP4) known reside                         mutagenized 50% survival initially plated 30&#176;C. <br /> chromosome XVI (41) (ii) hybridizing MOTJ                              Plasmid pSTE12(URA3) introduced original <br /> probe ordered collection yeast genomic DNA                               motl-l mutant strain (JD100), backcrossed <br /> fragments (in bacteriophage X vector) (55), generously                          times EG123; time, slowly growing, STE+ <br /> provided L. Riles M. Olson (Washington University,                         spores selected examined temperature sensi- <br /> St. Louis, Mo.).                                                                  tivity blue color X-Gal medium. It observed that <br />  VOL. 12, 1992                                         PUTATIVE HELICASE AFFECTING GENE EXPRESSION AND GROWTH                                              1881 <br />  <br />                                                      TABLE 3. Effect motl-J gene expression <br />  <br />   Strain                              Relevant genotypea                              Plasmid (UAS)b&quot;-Galactosidase (U)&apos; <br />                                                                                                                             -a-MF                      +a-MF <br /> SF167-5a                        MATa stel2::LEU2 MOTI+                        pJDll (PRE)                             Und                        Und <br /> JD227A                          MATa stel2::LEU2 motl-1                       pJDll (PRE)                             1.1                        0.5 <br /> EG123                           MATa STE12+ MOT1+                             pJDll (PRE)                             21.3                       121.6 <br /> JD215B                          M4Ta STE12+ motl-l                            pJDll (PRE)                             48.5                       156.9 <br /> SF167-5a                        MATa stel2::LEU2 MOTI+                        pLGA&amp;178T (none)                        Und                        Und <br /> JD227A                          MA Ta stel2::LEU2 motl-l                      pLGA178T (none)                         0.3                        0.5 <br /> EG123                           MATa STE12+ MOT1+                             pLGA&amp;178T (none)                        Und                        Und <br /> JD215B                          MATa STE12+ motl-l                            pLGA178T (none)                         0.6                        0.5 <br />   All strains isogenic STE12 MOTI loci. <br />   b <br />     pLGA178T cycl promoter-lacZ fusion lacking normal UAS cycl; pJD11, synthetic PRE elements inserted place UAS cycl. <br />   c   Values averages assays performed duplicate (range, c30%). Und, undetectable (&lt;0.1 U 6-h incubation); a-MF, a-factor mating pheromone. <br />  <br />  <br /> the growth phenotypes (slow growth 30&#176;C lethal-                         stranded herring sperm DNA ml, 1 x Denhardt&apos;s solution <br /> ity 38&#176;C) transcriptional phenotype coseg-                       (50), 1% <span id='am-48' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-49' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-51'  ></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>glycine</span>, 1% sodium dodecyl sulfate (SDS) and <br /> regated. A final backcross SF167-Sa carrying pSTE12                           washed 60&#176;C 0.lx SSPE-0.1% SDS. For hybridization <br /> (URA3) performed generate strains used                          oligonucleotide probes, buffer described above, but <br /> majority experiments presented here.                                        formamide HEPES, used 42&#176;C; blots <br />    Procedures phenotypic characterizations. For p-galac-                       washed 2x SSPE-0.1% SDS 45&#176;C (STE2 probe) or <br /> tosidase assays, cultures grown early exponential                          55&#176;C (PKCJ probe). The filters used expose Kodak <br /> phase synthetic complete medium lacking Trp, buffered                        XAR-5 film, using DuPont Cronex Lightning-Plus intensi- <br /> pH 3.5, select maintenance lacZ reporter                             fying screen. When necessary remove probe <br /> plasmid split portions. To culture                        rehybridizing filter, filter stripped <br /> each set, a-factor (Peninsula Laboratories) added                         (50) exposed film, using intensifying screen at <br /> final concentration 2.5 ,M, treated                            20 h ensure removal radioactivity was <br /> untreated cultures incubated 30&#176;C 60 min. Cells                        complete prior reuse. <br /> were harvested assayed P-galactosidase described                           DNA sequence analysis. Restriction fragments the <br /> elsewhere (31, 58). Independent assays performed                           region containing MOTJ-complementing activity (8-kb <br /> duplicate transformants, data averaged;                           BglII segment) adjacent fragments subcloned into <br /> range observed usually 10% greater                         polylinker regions M13mpl8 (54), M13mpl9 (54), or <br /> than 30%.                                                                          pRS316 (65). Nested deletions fragments pRS316 were <br />    Bioassays a-factor production performed de-                          generated using exonuclease III-mung bean nuclease <br /> scribed Kuchler et al. (45).                                                    protocol supplied manufacturer enzymes (Strat- <br />    Quantitative mating assays performed mixing                             agene). Both single-stranded double-stranded templates <br /> -107 exponential-phase cells strain examined                          sequenced using Sequenase kit (United States <br /> with -10&apos; exponential-phase cells tester strain                        Biochemical). Specific oligonucleotide primers synthe- <br /> opposite mating type, DC17 (MATTa hisl) DC14                             sized used necessary close gaps sequence <br /> (A4Ta hisl), nitrocellulose filter. Filters washed                       obtained deletion constructions confirm the <br /> with YPD incubated YPD plate 30&#176;C 5                        sequence junctions restriction fragments. All <br /> 6 h. Cells washed filters resuspended                     nucleotide sequences confirmed region the <br /> water. Dilutions cells (in case STE12+ matings)                       DNA sequencing strands. [a-35S]dATP from <br /> all cells (in case stel2 matings) plated minimal                    Amersham. <br /> plates selective diploids. Mating frequency defined                        Nucleotide sequence accession number. The GenBank ac- <br /> the total number diploids produced divided total                         cession number MOTI M83224. <br /> number input haploids strain examined. <br />    Isolation poly(A)+ RNA hybridization analysis. Cul-                                                 RESULTS <br /> tures grown mid-exponential phase YPD (pH 4.0) <br /> and split equal portions. One half culture                          Isolation motl-] mutation. A genetic screen used <br /> was treated a-factor (Peninsula Laboratories) final                      identify mutants expressed pheromone-responsive <br /> concentration 2.5 p,M 30&#176;C 40 min; half                        genes absence Stel2 absence phero- <br /> was placed ice period time. Total RNA                        mone. A reporter plasmid (pJD11) 16 PREs were <br /> prepared, poly(A)+ RNA isolated using oli-                              inserted place UAS cycl promoter-lacZ gene <br /> go(dT)-cellulose (Pharmacia), described (63).                         fusion (see Materials Methods) introduced a <br /> Poly(A)+ RNA (4 p,g) separated electrophoresis                              strain lacking functional STE12 gene (SF167-5a) (25). It has <br /> through 1% agarose gels containing 2.2 M formaldehyde                          amply documented level 3-galactosidase <br /> transferred Hybond-extraC membranes (Amersham)                               activity expressed appropriate promoter-lacZ fusions <br /> 20x SSPE (lx SSPE 0.18 M NaCl, 10 mM NaPO4, 1                               directly reflects transcriptional activity gene (31, 58). In <br /> mM EDTA [pH 7.7]) (50). Molecular weight markers                              absence Stel2, expression lacZ driven PRE <br /> from Bethesda Research Laboratories. Filters <span id='am-168' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-169' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-170'  ></span><span id='am-171' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>hybrid</span>-                          elements extremely low absence the <br /> ized DNA probes 42&#176;C buffer containing 50%                              presence a-factor (Table 3). In agreement this <br /> formamide, 6x SSPE, 50 mM N-2-hydroxyethylpiperazine-                              finding, SF167-Sa[pJD11] white X-Gal-containing <br /> N&apos;-2-ethanesulfonic acid (HEPES; pH 7.4), 40 p,g single-                        plates absence presence pheromone (data <br />  1882     DAVIS ET AL.                                                                                                   MOL. CELL. BIOL. <br />  <br /> not shown). In contrast, pJD11 introduced                   TABLE 4. Effect motl-l mating efficiency <br /> isogenic STE12+ strain (EG123) (25), level ,B-galac- <br /> tosidase activity produced 3 orders magnitude higher         Strain                  Relevant genotypea                   Mating <br />                                                                                                                                frequency&quot; <br /> than stel2 strain absence pheromone <br /> (Table 3). Furthermore, activity increased nearly sixfold         SF167-5a           MATa stel2::LEU2 MOT1I                   &lt;2.6 x 10-7 <br /> when cells exposed a-factor (Table 3). Correspond-        JD227A             MATa stel2::LEU2 motl-l                  &lt;2.4 x 10-7 <br /> ingly, EG123[pJD11] pale blue absence phero-        JD209              MATa stel2::LEU2 MOTIJ                   &lt;1.7 x 10-7 <br />                                                                   JD216D             MATot stel2::LEU2 motl-l                 &lt;2.0 x 10-7 <br /> mone dark blue presence a-factor X-Gal-          EG123              MATa STE12+ MOT]+                        0.59 <br /> containing plates (data shown). Thus, expression           JD215B             AL4Ta STE12+ motl-l                      0.44 <br /> pJD11 strains regulated manner identical      JD195              MATa STE12+ MOT1+                        0.73 <br /> that observed natural pheromone-responsive genes           JD215A             MATa STE12+ motl-l                       0.40 <br /> that maximum basal induced levels transcription              All strains isogenic AL4T, STE12, MOTI loci. <br /> required functional STE12 gene (23, 24).                          bTotal diploids input number haploid cells strain examined. See <br />    SF167-5a carrying pJD11 mutagenized ethyl             Materials Methods experimental details. <br /> methanesulfonate (see Materials Methods), 50,000 <br /> surviving colonies screened blue color X-Gal- <br /> containing plates lacking a-factor. Candidate blue colonies <br /> (73 total) assayed vitro 3-galactosidase activity     motl mutation did rescue sterility stel2 null <br /> (see Materials Methods) identify mutants             mutants. <br /> specifically elevated expression PRE-driven lacZ gene         The effect motl mutation expression ,B-galac- <br /> and eliminate displayed blue color           tosidase pLGA178T, lacks UAS (see Materials <br /> increased permeability X-Gal nonspecific effects.     Methods), examined (Table 3). Interestingly, <br /> Of 73 candidates tested, displayed increased         motl-l strains displayed readily measurable level of <br /> vitro ,-galactosidase activity. To rule mutants      expression, isogenic MOT] + cells did not. This <br />                                                                   effect independent presence Stel2 not <br /> expression <span id='am-43' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-46'  ></span><span id='am-47' property="rdfs:label" content="Indicator" datatype="xsd:string"></span>indicator</span> gene elevated plas-        influenced pheromone treatment. This relaxation the <br /> mid-linked alterations, expression endogenous phero-           requirement UAS cycl-lacZ expression suggested <br /> mone-responsive genes examined remaining         motl alter expression genes in <br /> candidates.                                                       addition pheromone-responsive genes (see below). <br />    Both a-factor structural genes, MFal MFa2 (6),         To rule possibility, however, variations in <br /> and gene required a-factor secretion, STE6 (45),        plasmid copy number explain apparent effects <br /> induced a-factor (6, 45) require Stel2 basal      observed gene expression different strain back- <br /> and induced expression (23, 24). Mutations increase          grounds, plasmid DNA recovered the <br /> expression reporter gene generally            strains examined quantitative hybridization analysis <br /> elevate expression PRE-containing UAS elements               (67). Comparison plasmid copy number all <br /> should increase expression chromosomally located          various strains examined showed negligible differ- <br /> pheromone-responsive genes generate              ences (less twofold) (data shown), confirming that <br /> a-factor does parental strain. Pheromone produc-         different levels 3-galactosidase observed reflected <br /> tion judged a-factor bioassay measures      changes gene expression. <br /> amount a-factor synthesized secreted sur-            Effect mot] gene transcription. To determine how <br /> rounding medium (45). Of remaining candidates,          general effect motl gene expression, hybrid- <br /> one isolate reproducibly displayed elevated a-factor produc-      ization analysis poly(A)+ RNA variety chromo- <br /> tion (data shown) addition elevated ,-galactosi-    somal genes performed (Fig. 1). For pheromone-respon- <br /> dase activity. The level production a-factor de-        sive genes expression severely reduced, not <br /> tected size zone growth inhibition (halo)   completely eliminated, inactivation STE12 (23), includ- <br /> lawn A4 Tao cells AL4Ta mutant cells       ing MFa2 (6), STE2 (encoding a-factor receptor) (9, 52), <br /> spotted. This mutant called motl (for modifier tran-       STE6 (45), motl-l allele caused significantly in- <br /> scription), allele described designated          creased expression (at threefold) stel2 background <br /> motl-l.                                                           modest increase STE12- background (Fig. 1A <br />    Phenotypic characterization motl-1. In STE12+ back-       Table 3). In stel2 strains, a-factor induction <br /> ground, basal expression pJD11 twofold higher      detectable motl cells. In STE12+ strains, pheromone <br /> cells carrying motl-l allele (JD215B) wild-type       treatment elicited transcriptional induction motl-l <br /> cells (EG123) (Table 3). Following exposure pheromone,         MOTI + cells, levels motl cells appeared <br /> the induced level strains equivalent. As expected     slightly higher. The effect motl expression of <br /> on basis scheme isolation, stel2          chromosomally located genes qualitatively the <br /> background, motl-l strain (JD227A) carrying pJD11               effect PRE-driven reporter plasmid (Table <br /> showed readily detectable expression PRE-driven            3). To rule possibility transcriptional effects <br /> lacZ gene, MOTI + stel2 parent (SF167-Sa) did         motl-l a-cell specific, expression AL4Ta cells <br /> not (Table 3). Expression essentially unaffected           STE3, a-cell-specific gene encodes a-factor <br /> exposure a-factor, indicating motl-l does restore     receptor (31, 52), examined. STE3 expression similar <br /> response pheromone action does bypass require-      MFa2, STE2, STE6 sense detectable <br /> ment Stel2 maximum levels induction. This point        basal transcription observed stel2 mutant cells (23). It <br /> was supported quantitative mating assays (Table        STE3 expression increased three- <br /> 4). Regardless mating type, significant difference       fold ALTa motl-l cells compared isogenic ALTa <br /> mating efficiency observed MOT] + motl-l          MOTJ + cells (data shown), indicating transcrip- <br /> strains containing functional STE12 gene; conversely,       tional phenotype motl-l cell type specific. <br />                                                                                                                                         U, <br /> VOL. 12, 1992                                                  PUTATIVE HELICASE AFFECTING GENE EXPRESSION AND GROWTH                                       1883 <br />  <br />                               stel2::LEU2                          STE112+ <br />                                                                                                        Alf- T I       T   1   2+ <br />                                      +                +             +         + <br />                                vWE-  E--                      vi    E-i   v   Es4                  MnoTl7-            STEI2+ <br />                                0 0 0                      0 00 <br />                                fE3           E                E:          E <br />                                                                                          MOT1&quot;+ stee2::LEU2 <br />                     o-MF:      -     -       +        +       -     -     +   + <br />        A. <br />                     MFa 2     lI <br />                                s a <br />                                  ----                                               I   mort1-1 stSel 2::LEU2 <br />                     STE2          al AW ilO                                                                                             3v C3&apos; ::C <br />                                                                                           FIG. 2. Growth phenotypes motl-l cells. Four different <br />                     S TE6      AM    ,   , <br />                                                  ;W   _                                 spores identical genotype, derived cross JD102C (MA Ta <br />                                                                                         motl-1) SF167-5a {AM Ta stel12::LEU2 [pSTE12(URA3)]} were <br />        B.                                                                               streaked YPD plate allowed grow 30&#176;C 3 days. <br />                        J <br />                     FUS1                                  _                             Cells plates resuspended water equal densities, <br />                                                                                         placed multiwelled template dish, spotted fresh <br />                     SSlEa 1                               M               -             YPD plates using pronged inoculator. The plates incu- <br />                                                                                         bated 30&#176;C 3 days 38&#176;C 3 days, indicated. <br />                     3372214                                        z. <br />  <br />  <br />        c. <br />                                                                                         calmodulin) (13) (Fig. 1D; Fig. 5B) PKCJ <br />                     DED1       go. .   .e <br />                                        . <br />                                    .I..... <br />                                            .~~~~~k ,I..w: ::.... <br />                                                               (encoding..    sZprotein kinase C) (48) (see Fig. 5A). The <br />                                                                         yeast <br />                                                               level CMDJ PKCJ transcript used to <br />                                                                                         normalize differences RNA loading. <br />        D. <br />                       CMD 1         _        I            44E -                            Effect moti mutation growth. In addition the <br />                                                                                         transcriptional phenotypes described above, growth <br />    FIG. 1. Effect motl gene transcription. Poly(A)+ RNA (4 <br /> pug) strain loaded sample well. The relevant                    phenotypes observed motl mutants. Cells carrying <br /> genotype strain indicated lane; strains                       motl-l allele grew slowly 30&#176;C, doubling time of <br /> those listed Table 3. Gene probes (see Table 2) designated                    3.5 h rich medium (YPD), inviable 38&#176;C <br /> the left strip. The filter used hybridizations                 (Fig. 2). These growth phenotypes manifest a <br /> (see Materials Methods conditions). a-MF, a-mating factor;                      stei2 STE12+ background. A temperature shift experi- <br /> -, treated a-factor; +, treated a-factor. (A) Phero-                      ment performed early-exponential-phase <br /> mone-responsive genes display Ste12-independent basal tran-                        motl-l cells shifted 30 38&#176;C examined at <br /> scription; (B) pheromone-responsive genes require Stel2                        time points shift. No uniform terminal cell <br /> basal transcription; (C) DED1, gene product thought not <br /> to mating pathway; (D) CMD1, used control                       morphology observed (data shown). Cells held at <br /> the uniformity RNA loading.                                                          38&#176;C 4 h retained nearly 100% viability shifted <br />                                                                                         30&#176;C. However, longer periods 38&#176;C, viabil- <br />                                                                                         ity dropped rapidly progressively. For example, 24 <br />                                                                                         h 38&#176;C, viability 0.1% MOT] + cells <br />    For pheromone-responsive genes expression                                   subjected treatment. After MOTI gene was <br /> reduced undetectable levels inactivation STE12 (23),                           cloned (see below), expression MOTJ transcript in <br /> including FUSJ (encoding protein required cell fusion)                            motl-l mutants analyzed 4 8 h shift to <br /> (72), MFal (6), SST2 (encoding protein involved                                38&#176;C. Even 8 h, transcript present level <br /> recovery a-factor-induced G1 arrest) (12, 15),                                 nearly equivalent observed initially (data shown). <br /> motl-l mutation did lead detectable transcription                             Thus, inviability 38&#176;C likely lack of <br /> the stel2 background (Fig. 1B). These results account                               mRNA mutant cells defect the <br /> for fact despite elevated expression certain                            function motl-l product 38&#176;C. <br /> mating genes, motl stel2 strains mating deficient                                Isolation sequence MOT) gene. When motl-l <br /> (Table 4). Remarkably, STE12+ strains, motl-l                                haploid crossed normal haploid, resulting <br /> mutation detectable effect enhancing basal                               heterozygous diploid cells grew normally 30 38&#176;C and <br /> transcription genes; however, a-factor induction                             did display elevated expression pJD11 reporter <br /> of genes equivalent motl MOTI + cells.                                 plasmid; hence, motl-l allele recessive mutation. <br />    Expression gene thought unrelated mating <br /> pathway, DEDJ (for defines essential domain) (70), also <br /> increased motl-l mutation stel2                                  s&apos;                                                                      3. <br /> STE12+ backgrounds (Fig. 1C), way observed <br /> for MFa2, STE2, STE3, STE6 genes. DEDI tran- <br /> scription significantly affected a-factor treatment                                             II                          i <br />                                                                                                                                                    I0 <br /> in STE12+ cells, agreement fact 5&apos;-                                     I--                                                            _   9 <br /> flanking region DEDJ gene lacks canonical PRE                                    kb         0              1           2    3       4   5        6 <br />  <br /> sequences (70). The DEDJ gene product been <br /> implicated mating process previously. If does                               FIG. 3. Restriction map MOTI gene. Rectangular regions <br /> play role mating, data suggest expres-                          represent areas subjected DNA sequence analysis; <br />                                                                                         dashed lines represent regions sequenced. The open <br /> sion genes outside pheromone response                                    reading frame (ORF) represented hatched rectangle; <br /> network affected motl-l mutation.                                             noncoding regions represented open rectangles. All sites <br />   Two genes identified expression com-                                   enzyme indicated shown ClaI (a second <br /> pletely unaltered motl-l, namely, CMDJ (encoding yeast                               dam-sensitive site exists position 3435). <br />  1884         DAVIS ET AL.                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                       -221                         -191                          -161                       -130 <br />              GTCCCACTCGCCGGCCGCTGCACTTGATATAAAAGCTTATGATTCCACCAATTCCGTTCCCTTGGCGGTTGCGCGCCAmTT7TCATGCGCTCAGA^AAACCACCATTCTGCTATT <br />                                       -100                          -70                           -39                        -10 <br />              ATACCGAAAACCGGCGG                                   TTCAAACATAAATAAAGTACGATCGTATTCCAATATTAACGAACAAAAGTATAACACACACA <br />                     -1 <br />              GCACCTCAA <br />                                             30                            60                             90                             120 <br />              ATGACGTcAcGAGTTT      ;AGGCTGGAcCGG <br />                                                CAGGTATT   AAGAGACGGTTCCACCCAAGTATCGGAACTGTGCCG&amp;CATGAATGGCCA AACAGCACCCCGAA <br />                                                           I E T G S T 0 V V R N M A A D Q M G D L A K O H P E <br />         1    M T S R V S         R   L D R    V I L <br />                                                  0 <br />                                            150                           180                            210                             240 <br />              GACATTCTCGTTGCTGTCCAGAGTT1ACCC TTACTAGTCAGAGAATGGGAACGCGTGTTACTGCAGCAAGAGTTCi-CCATGCTCCixTTGGGATCCG <br />        41    D I L S L L S R V Y P F L L V K K W E T R V T A A R A V G G I V A H A P S W D P <br />                                            270                           300                            330                             360 <br />              AATGAATCTATGGTAGGTGG                                                                              TATAACATATGA <br />        81    N E S D L V G G T N E G S P L D N A                    Q    V K L E H E M K          I   K L E E A T O N N Q L N <br />                                            390                           420                            450                             480 <br />              CTCTTCGAGACTACTCAGlTTGTCTGGA5       <span id='am-26' about='obo:NCBITaxon_12956' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_36668 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_143999 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7434 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_34695 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_34725 obo:NCBITaxon_50557'><span id='am-27'  ></span><span id='am-28' property="rdfs:label" content="Atta" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="leaf cutting ants" datatype="xsd:string"></span>ATTA</span> AGTCAGGTAAAGCTCCTGGCATTATATCAG <br />                                                 0A                                     TGGTAAGGCT <br />    121       L L Q E D H H L S S L S D W K L N E I L K S G K V L L A S S M N D Y N V L G K A <br />                                            510                           540                            570                             600 <br />              GACGATAATATAAGAAAAAAGCTA_C                                        CGTTGATAAGG   CGAATGaAAAATAAGTAATGCACAAGAGCGAGGATG <br />    161       D   D N   I   R K   Q <br />                                  A K T D <br />                                 D I K  E            Q     T S M L N A S D K A N E N K S N A N|  KS A R M <br />                                       630                         660                       690        700 <br />              CTTGCTATGGCAAGGCG^AAAAAGAAAATGAGTGCTAAGAATACTCCCaAAATCCTGAATTACM u _AAAGATTCAG <br />    201       L A M A IR R K K K1M S A K N T P K H P V D I T E S S V S K T L L N G K N M T N S <br />                                        730                        760                       790        820 <br />              GCTGCATCGCTAGCCACTCTCCCACCAGTAACCAGCTAAATCCCAAACTGGAGATTACTGAACAAGCCGTGAAGTAAACTAATAAATCACAGTAAGGCCATTGCTAGAACAA <br />    241       A A S L A T S P T S N O L N P K L E I T E                  Q A D E S K L M I E S T V R P L L E Q <br />                                850                 880                                         910                         940 <br />              CACGAAATTGTTGCTGGCCTAGTTTGGCAA CCAAGGCATTTATGAACTACTTTAAAACTAGAGAAAATTGGGAAATAAGGCATGGAGCTGCGTTGGGTCTTCGAGAATTG <br />    281       H E I V A G L V W Q F Q G I Y E L L L D N L M S E N W E I R H G A A L G L R E L <br />                                       970                       1000                       1030                       1060 <br />              GTAAAAAAACTGCTTATGGAGTCAGCAGGGTTAAGGGGAATACCCGAGGACAACCTGAGGAATTCTAGAAGCTTAGAAGACTTGGCGTCCCGTCTACTTCGTTTGAT <br />     321      V K K H A Y G V S R V K G N T R E E N N L R N S R S L E D L A S R L L T V F A L <br />                                      1090                       1120                       1150                       1180 <br />     361 <br />              GTAGT     GTATAGIAGTCAGTTGTGGCCCCTGTACGAGAGTCAG TTGCGCAAACTCTAGCT GCGTTATTAA TTCACCTAGACAGTACCTTATCAATTAAAATATTTAAT <br />              D R F G D Y V Y D T V V A P V R E S V A Q T L A A L L I H L D S T L S I K I F N <br />                                1210                1240                1270                1300 <br />                                                                   LCTTTTAGGGA RVKGTTACTTCGTCAGTAGACTC <br />              TGLTTCANTGGTTCTACEAGATCCACTACAAACAGGTCTACCGATTCAT GGAG <br />     401      C L E Q L V L Q D P L Q T G L P N K I W E A T H G G L L G I R Y F V S I K T N F <br />                                        1330                          1360                       1390                      1410 <br />              CTATTTGCCCATGGCCTTTAGAAAACGTTGTCCGTATAGTGCTATATGGTTAACAATATAGC=IGTTQAAAGCGTAGCAGCCTCAA=AACCCAATAACCAGCAATT <br />     441      L F A H G L L E N V V R I V L Y G L N Q S D D D V Q S V A A S I L T P I T S E F <br />                                        1440                          1470                       1500                      1530 <br />                  ;CGBTACgkTGcATGTAT                                GCTIG     GAIGATGAT AIC GCGTAGGTTCT   TAATG  ATIGGCAAAG <br />     481      Q)K L N(N)ST (I)E I L 9 TT)W S LUA R(9D D DGS                                       AKVG0  I0  D               L <br />                                        1560                          191620                                                10 <br />              TGCGACCCAG1SATA _CTAGAGTGGTCATTCAAGTCACTG;TCGA TTP-GPTATSTTCCTTTTTCGCCATTCGTC <br />     521      C D H Q E V L D I L K N K A L E H P S E W S F K S L V P K L Y P F L R H S I S S <br />                                        1680                          1710                       1740                       1770 <br />              GTGAGGAGGGC          T        CATTTATC   AATAAATGGATGATCTAAAA1ACTGGTTAAACGGCAAGGTATTCCG            _      A <br />        561   V R R A V L N L L I A F L S I K D D S T K N W L N G K V F R L V F Q N I L L E Q <br />                                        1800                          1830                       1860                       1890 <br />              AACCCGGAACTATTGCAACTGTCTu  CGATGTTATGTTGCGTTATTGGAGCATTATAAAGTTAAGCATACCGAAAA_C TAGATCATGTTTCGACTTGCAGCCAATACTA <br />        601   N P E L L Q L S F D V Y V A L L E H Y K V K H T E K T L D H V F S K H L Q P I L <br />                                        1920                          1950                       1980                       2010 <br />        641 <br />              CATCTTCTAAATACGCCAGTAGGGGAA.AAGGGAAAGAATTATGCCATGAT=CACATTCTAAAGCCTTCCCACCATAAT <br />              H L L N T P V G E K G K N Y A M E S Q Y I L K P S Q H Y Q L H P E K K R                                         S <br />                                                                                                                               TATATCTGAA <br />                                                                                                                                   I   S E <br />                                            2040                          2070                          2100                             2130 <br />              ACCACAACAGATTCTGACATTCCACCM                     TCAATATCG     ACCTATIGCTGGAGA TTA           AGGATIGGAC TAATA        CACTAGAATA <br />        681   T     T T D S D I P            P6KIN <br />                                               ((A <br />                                              NNE  D P M()A GD()T L(L)G                                        L D    V)IL()T          R I <br />                                            2160                          2T0                    ~2250 <br />              &amp;TG A=Cm        ACATTGTCAATG TTCAAGATTCCACACTTCAATCCTT TCA <br />                                                                      T   AAACGTTT GTTGlGTTTAGAGCTGCCATTCTCAACTCCACGTATGTTG <br />        721      G A K A F A L T L S M F Q D S T L Q S F F T N V L V R C L E L P F S T P R M L <br />                                      2280                           2310                     2340                        2370 <br />              GCAGGCATTATTGTTCACAGTTTTGCTCCTCGTGGTTACAAAAGCATCCAGAAGGGGAAATC.L%-TTACTGCTTATCA AGAmTTTTCATC TGTGATGAATAACGTCGA <br />        761   A G I I V S O F C S S W L O K H P E G E K L P S F V S E I F S P V M N KO                               L L N <br />                                      2400                           2430                     2460                        2490 <br />              CGCGATGAATTCCTGTATTAGGGAGCTAGTACCTAGCTTAAAAGCTCTGAGGACTATTAACCTACTGGTACCTTTGTGGATGTAGGTATGTTACCTATCACAC    C <br />        801   R D E F P V F R E L V P S L K A L R T Q C Q S L L A T F V D V G M L P Q Y K L P <br />                                      2520                           2550                     2580                        2610 <br />        841 <br />              AACGTAGCAATTGTGGTTCAAGGCGAAACAGAAGCCGGTCCACATGCGTTTGGTTTAAACAGAGAATTACGGCGAATACThG <br />              N V A I V V 0 G E T E A G P H A F G V E T A E K V Y G E Y Y D K M F K S M N N S <br />                                                                                                                _AAGTCAACCTGAACAATCT <br />                                2640                2670                2700               2730 <br />               YTACAKACLATKKPEAAGA           GTATG <br />                              AGKHATTAGRVAGMATCTGC   INSATC AAAGAAT C <span id='am-131' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-132' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-133' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-136'  ></span>AA</span> K RCTTCGTGGSCITTCTCGCTAATTACGSTC <br />                                                                            L <br />        881    Y K L L A K K P L E D S K H R V L M A I N S A K E S A K L R T G S I L A N Y A S <br />   FIG. 4. Nucleotide sequence predicted amino acid sequence MOTI. Nucleotide positions shown line; amino acid <br /> positions shown left line. Amino acid sequence one-letter code nucleotide sequence. A potential nuclear <br /> targeting signal boxed. The 568-residue domain (amino acids 1257 1825) homology helicases SNF2 RAD54 underlined. <br /> A number segments amino-terminal domain, example, residues 481 520 689 721 (circled), fit 4-3 hydrophobic repeat <br /> that diagnostic amphipathic helices capable forming coiled coils (11). <br />   921 <br />  <br />  961 <br />  <br /> 1001 <br />         TCCATrC <br />         S I L L <br />  <br />  <br />         H <br />  <br />         GMT1ATACAGAAAMTATAGAAT <br />         E <br />                 L <br />  <br />  <br />                 Y <br />                      I <br />  <br />  <br />                      K E <br />                          = _~ ~ T A TG TC G TA GAT GTCA A TC G A= <br />                          Q <br />                               F <br />  <br />  <br />                               Q <br />  <br />  <br />                               K <br />                            3120 <br />                                 3000 <br />                                       D <br />  <br />  <br />                                       L <br />  <br />  <br />                                       I <br />                                           AATACCAAACACCCATTATCAGGTCGCTAATGGACAGTGTTAAAGAGAGGAATGAAAGTACAAACGATGGCCGGAGAACGGTCGTA <br />                                           G <br />  <br />  <br />                                           L <br />  <br />  <br />                                           L <br />                                            TCTTA <br />                                                L <br />  <br />  <br />                                                E <br />  <br />                                                T <br />                                                 3150 <br />         ACGAAATACTATGAGTCTTGCGCTTAAT TAAGAAATCTCAGAT <br />                                                     P <br />  <br />  <br />                                                     N N <br />  <br />  <br />                                                     L <br />                                                          2760 <br />                                                           L K <br />                                                          2880 <br />  <br />                                                           3030 <br />                                                           I     K E <br />                                                                      L N <br />  <br />  <br />                                                                 K V N V S <br />  <br />  <br />  <br />  <br />                                                                      T <br />                                                                              P <br />  <br />  <br />  <br />  <br />                                                                          S N S <br />                                                                            A F <br />                                                                                  I <br />  <br />  <br />  <br />                                                                                  3060 <br />                                                                                          I <br />  <br />  <br />                                                                                          G <br />  <br />  <br />                                                                                          I <br />                                                                                                  R <br />  <br />  <br />                                                                                                  K <br />                                                                                                      S <br />  <br />  <br />                                                                                                      I <br />                                                                                                            2790 <br />                                                                                                           L M D <br />                                                                                                            2910 <br />                                                                                                           V K N <br />                                                                                                          3090 <br />                           AAAGAATCTAATTCGATTGCCGCACAAGATGACATTAATC GCGCAGATGTCGt;AAGAGGCTCAACTAAAAGAGGGGTGT <br />                                                                                           E A Q L K R K G G L I <br />                                                                                          3180 <br />                                                                                                  A A Q D          D <br />                                                                                                      GAAAGCAAG3AAGTT <br />                                                                                                                       S <br />  <br />  <br />                                                                                                                       L <br />  <br />  <br />                                                                                                                       I <br />                                                                                                                           V K <br />  <br />  <br />                                                                                                                           C <br />  <br />  <br />                                                                                                                           N <br />                                                                                                                               G <br />  <br />  <br />                                                                                                                               L A K M <br />                                                                                                                              3210 <br />                                                                                                                                   E E <br />  <br />  <br />                                                                                                                                   F   L <br />                                                                                                                                            R N <br />  <br />  <br />                                                                                                                                            C    V D <br />  <br />  <br />                                                                                                                                                 S <br />                                                                                                                                                     E <br />  <br />                                                                                                                                            ATCTGAATCAGAGTCATrCGTAATC <br />  <br />  <br />                                                                                                                                                     E <br />                                                                                                                                                         2820 <br />                                                                                                                                                          K L Q T M A G <br />                                                                                                                                                         2940 <br />  <br />                                                                                                                                                          T S       E    V P     D   F <br />                                                                                                                                                                                         E <br />  <br />  <br />                                                                                                                                                                                         S <br />                                                                                                                                                                                             S V V <br />  <br />  <br />                                                                                                                                                                                             V N A <br />                                                                                                                                                                                                  2850 <br />  <br />  <br />                                                                                                                                                                                                  2970 <br />  <br />  <br />  <br />  <br /> 1041    T L K I L F E V L G P S I L Q K L P Q L R S I L F D S L S D H E N E E A S K V D <br />                                  3240                        3270                        3300                                3330 <br />         AACGAGCAAGGTC:AAAAATAGTCGACTCATTGGGACTAAGAGCTCTATTCCCATTTATGTCTGATCCTACGTrCTAGTGAGGT lACCCGlCCCXToTMGT <br /> 1081    N E Q G Q K I V D S F G V L R A L F                P  F  M S  D  S  L R   S S  E  V   F   T   R    F   P   V L  L  T  F <br />                                  3360                        3390                        3420                                3450 <br />                            R=GTTCARTTAAVCTCCGTGGAGGTAATCACCATACG                                                          ATG <br /> 1121    L R S N L S V F R Y S A A R T F A D L A K I S S V E V M A Y T I R E I L P L M N <br />                                  3480                        3510                        3540                                3570 <br />         AGCGCTGGTT&apos;CTCTATCCGATCG             _                 GAAACCGATGTTCTACCTTACGC           ATGTCAl                    GC <br /> 1161    S A G S L S D R Q G S T E L I Y H L S L S M E T D V L P Y V I F L I V P L L G R <br />                                  3600                        3630                        3690                                3720 <br />         ATGAGTGATTCTLCG^AAGTTAGATTTOAGT           ATMGTCT               TAG     AAGTGGATCGCCGATCCAAAGACTCCCAGAAGAACAGTG <br /> 1201    M S D S N E D V R N L A T T T F A S I I K L V P L E A G I A D P K G L P E E L V <br />                                  3750                        3780                        3810                                3840 <br />         GCAGAAAGAGGGAGCGTGATTTTwATCCAACQAATGTGATCCTTCTAAAGCTAAGCCATTTAAATTACCCATTGCCA           lcA^GGCGACTTAAGAAAATATCGAGAGTT <br /> 1241    A S R E R E R D F I Q0            M M D P S K A K P F K L P I A I K A T L R K Y Q 0 D G V <br />                                  3870                        39ou                        3930                                3960 <br />         AACTGGTTGGCATTCCTGAAATCCAAGATATTGTTGACGATATGGGTTGGTAACATTACAACCATGCTATCGCACACAATGGAGA <br /> 1281    N        W   L A F            L   N    K     Y    HL G K T L Q T I C I I A S D Q Y L R K E <br />                                                                 L    H   G   I   L C             D   D    M G <br />                                                       4020 <br />                                                          3990                     4050                          4080 <br />         GATTATGAAAAACGC     CGGAGGCTAQ               CCCCCATCTTTAATGG CCCATGG      GATTGACAGTATGCTCCT AAAGTT <br /> 1321    D Y E K T R S V E S R A L P S L I I C P P S L T G H W E N E F D Q Y A P F L K V <br />                             4110                      4140                        41-10                         4200 <br />         GTTGTGTATGCCGGTGGCCCAACT                     GGTAA        T G T       AT    ACC           CAGTCCTAACAAAC <br /> 1361    V V Y A G G P T V R L T L R P Q L        SD    A   DII   VT   S  Y    DV   A    RN   DL    AV   L N   K  T <br />                             4230                      4260                        4290                          4320 <br />              TATAATTATTGNHRL <br />                     GAA                                                                 CTGTTAALTTTAACCGGACT CCCAT C <br /> 1401    E Y N Y C V L D E G H I I K N S Q S K L A K A V K E I T -A N H R L I L T G T P I <br />                             4350                      4380                        4410                          4440 <br />         CAAAACAACGTTTTAGAATTGTGGTCTGTA CTTCTTAATGCCGGGATTTGGAGGAAA   CTCCAGGAAGGTTCGCAA GCCATGGCCAAAATAGTAAA <br /> 1441    QN N V L E L W S L F D F L M P G F L G T E K M F Q               ER     F  AK      PI   AA   S  RN    SK <br />                              4470.                    4500                        4530                          4560 <br />         ACACATCTAAGGAGC=GGAAGCGTGTACTGGCTTlTGGAGGCACTACATAAG;CAAGTTCTACCTTTCA7CTGAlAAGATGAGAGACGTTCTATCTGATTACCACCTAAAATT <br /> 1481 <br />                                                          4590                                               4620                                        4650                                     4680 <br />         ATTCAGGATTACTACTGTGAATTAGGTGATTTACAAAAACACTATACTGGAAG <br /> 1521    I        Q D     Y     Y      CL Q K Q L Y M D F T K K 0 K N V V E K D I E N S E I A D G K <br />                                           E    L G D <br />                                   4710                        4740                       4770                      4800 <br />         CAACATATTTTCCAAG TACAATACATGAGAAAATTGTGTAATCATCCAGCTTTGGT&apos;CCTTTCACCAAT ACCCGCAATTAGCGCAAGTAAGCTATTAAGCAACTGTCTA <br /> 1561    Q H I F Q A L Q Y M R KLCNHPALVLSPNHPLAVDYLKTGL <br />                                   4830                        4860                       4890                       4920 <br />         GATTTACACGATATTATCAACGCTCCAAAACTAGCGCATTGAGAACATTACTC=GAGGGTATAGGTGAAGAAGATATCGACA AAAA MGCAAGCCAAGTAA7CTATT <br /> 1601    D        L   H D       I      I   N    A P        K L        S   A L R T                 LC G I G E E D I D K K A S Q D Q N F P I <br />                                                                                                      L    F E <br />                                                          4950                                4980                     5010                    5040 <br />         CAATTAACCACA&gt;CAG AGCCT A          t                                                AGACAGGTAAGC <br />                                                                              CATAAGATGCTACOT            TAAAAAGlTATATGCCCTCCGTAACCTATATGAGGCTAGAT <br /> 1641    Q N V I S QHRALIFCLKDMLDMVENDLFKKYMPSVTYMRLD <br />                                  5070                        5100                          5130                           5160 <br />         GGAAGCATGCCCAAAGACAGAACAAGTTGTTCGG <br />                                          AAATTTAACGAAGATC CTCTA   TTGATTGCCTACTGTTGACCACCAAGGTCGGAGGGCTGGGTCTG;AATTAACTGGTGCA <br /> 1681    G S I D P R D R Q K V V R K F N E D P S I D C L L L T T K V G G L G L N L T G A <br />                                                           5190                          5220                   5250                  5280 <br />         GACACC                TAT=&gt;GA                                                                             TTTACAGAATATTACG1AAAG <br />                                                                      CAATGAATGATCTACAGGAAGACAGAGCCATAATGTAAAAGTTAATG <br /> 1721    D        T V       W N P M N D L <br />                              I F      V   E     H    D                           Q A M D R A H R IG O K K V V N V Y R         I IT K G <br />                           5310                                                                              5340           5370                5400 <br />           ACV7TTGAGAAAAA7CTGG                                                                                             CATCCATGGATAAACGTCTACT_ <br />  1761   T L E E K I M G L Q K F K M N I                                                          A S T V V N Q Q N S G L A S M D T H Q L L D L F <br />                                  5430                                                                  5460                5490                5520 <br />         GACCCCGATAACGTTACCTCACGGACAATGAGGAAAAGAATAATGGCGATTCCCAAGCAG;CCAAGGATGGAAGATATTGCCAACGAGACTGGTTAACTGGCAAAGCCAAAAGGC <br />  1801   D        P   D N V T                  S Q D N            E   E   K N N               G   D       S Q A A K G M E D                  I   A N      E     T   G    L   T   G K A K E A <br />                                                           5550                                              5580 <br />         CTCGGTGAATGAAAGGTATGGGACCCATCTCAATACGAGGAGGAGTATAATTTAGACACCTTCATCAAAACTTTACGA <br />  1841       L    G E     L        K   E   L W D           P      S   Q   Y E         E   E       Y N      L D T F         I   K T      L    R <br />                                                                                                                                                                                                   5721 <br />         TAAACAAATTCTTGCATTGGAAGAlGCGTATACAAGG                                                    GCTTGTTATTTAACCmTGTATTATCAACTTG                                       TATTAAGTTT <br />                                                                                                                                                                                                   5841 <br />             C                                       TTGTTAAAATTCAG;GTTATTAGCTGTCTATTGATTATGGCTTGG                                                                  AT       TTT             GT      A <br />  <br />                                                                                                          5961 <br />             GTTGCGCAAAGGGAAACTGACGTTTAGTCGAACTCTGCA GAGCTTCATCTAAGAATTTCTrGCTATCGAAACGTTTGGAA AAAATCAAACCTCG <br />                                                                                                          6081 <br />             TTCTTC GCAATMGCATTCGCTGGAAACGZATAGACACATCTAACTVTCCGC <br />                                                                                                                                                                                                   6201 <br />                                C__C <br />             CTGACCTCGTTGAAATGCTGCT                                                                                    CAAAG           GCTAI 7TTCAAGTC                         l <br />                                                                                                                                                                             TCATA ICATAGCTG <br />                                                                                                                                                                                                   6321 <br />             TTTGATACCATACGTAATAAGAGGCACAACAGTAAAATC7GAGT=AAGAGTCCTATAGTTCAT ATCAACGTACTTAAACCC                                                                              AAAGGCAGC=ATGCAAACGA <br />                                                                                                                                                                                                   6441 <br />             TGGGAAGGTGTID AAGCGAA                        CTCTGGCllTTGATGAGTmAATTATAGATTA <br />                                                                                       TTA GAAGCAGGCTCTAA                                                                                      MAAGTA <br />                                                                                                                                                                                                   6561 <br />             AGCACGAAACAAGATTATACTCAGTATGGAGGATTTGTATTCGlT <br />                                         6596 <br />                 CAGCA ICAAGAAGCAGCAGTCAAGACTTCTTT <br />                                                                                              FIG. 4-Continued. <br />                                                                                                           1885 <br />  1886     DAVIS ET AL.                                                                                           MOL. CELL. BIOL. <br />  <br /> The growth phenotypes associated original motl-l         simply indication MOTI transcript more <br /> isolate segregated 2:2 cosegregated transcrip-       unstable G1-arrested cells growing cultures. <br /> tional phenotypes associated original motl-l muta-          MOT] essential cell growth. A MOTI gene disruption <br /> tion; therefore, motl-l allele represented alteration   constructed recently devised y transformation <br /> a single nuclear gene. The MOTJ gene cloned            procedure (65). The motlA1 allele prepared subclon- <br /> YCpSO yeast genomic library (59) complementation            ing 474-bp SacI-BamHI fragment (nucleotide positions <br /> the slow-growth/temperature-sensitive phenotypes. Eleven          3539 4013) 1,058-bp HindIII-ClaI fragment (nu- <br /> complementing plasmids isolated fell        cleotide positions 1286 2344) (Fig. 3 4) tandem but <br /> distinct overlapping classes based restriction map-        reverse <span id='am-96' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-98' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> integrating plasmid pRS306 (URA3) <br /> ping. The complementing region localized approx-        (65). This construct linearized unique restriction site <br /> imately 8-kb BglII fragment (Fig. 3). To demonstrate     fragments (HindIII) introduced into <br /> cloned DNA corresponded MOTI locus, following          unrelated diploid strains (YPH501 1788) DNA- <br /> experiment performed. A SacI fragment (Fig. 3) con-           mediated transformation. The resulting disrupted gene is <br /> taining complementing region plus flanking se-        predicted 1,194 bp (398 amino acids) the <br /> quences subcloned integrating plasmid pRS306         ClaI Sacl sites (Fig. 3 4) deleted replaced by <br /> (URA3) (65). This plasmid linearized unique     plasmid sequences. A BamHI fragment spans site <br /> HpaI site unique BglII site subcloned           deletion/insertion mutation (Fig. 3) used a <br /> fragment (Fig. 3), target integration homologous        probe detect wild-type disrupted MOT1 <br /> genomic locus, introduced AL4 Ta strain (YPH499)       alleles Southern blot analysis (67) genomic DNA <br /> (65) DNA-mediated transformation. Ura+ transformants           isolated recipients Ura+ transformants had <br /> were selected. These transformants exhibited phenotype,        digested BamHI (Fig. 6). The wild-type allele <br /> and representative crossed AL4Ta motl-l strain       migrated 3.5 kb larger deletion/insertion allele <br /> (JD102C). Tetrad analysis resulting diploids showed        migrated 6.5 kb, confirming diploid strains <br /> that site integration allelic MOT] locus;    heterozygous motlAl mutation. Both heterozy- <br /> all 22 tetrads examined, slowly growing, Ura+       gous diploids subjected tetrad analysis. For 11 12 <br /> or normal-growing, Ura- recombinants. Hybridization          10 11 tetrads strain, respectively, 2:2 <br /> BamHI fragment MOTI coding region (Fig. 3)            segregation live spores:dead spores observed (Fig. 6). <br /> chromosome blot (see Fig. SC), ordered set           All live spores Ura-. The spores did not <br /> cloned yeast DNA fragments bacteriophage X vector            form visible colonies examined microscopically. In all <br /> (see Materials Methods), revealed MOT] resides        cases, spores germinated yielded microcol- <br /> chromosome XVI. Two genes affect transcription,        onies consisting 10 20 normal-looking cells. Thus, <br /> SIT3 (2) SPT14 (21), reside chromosome XVI.           MOTI essential gene required cell growth. This <br /> The motl-l mutation allelic loci,   result supports conclusion Motl affect <br /> as indicated following observations. First, plasmid-     expression genes outside mating response <br /> borne copy SIT3 (provided K. Arndt) does com-           pathway pheromone-responsive genes <br /> plement motl-l, cloned SIT3 MOTI DNAs                  described date nonessential. Of course, Motl also <br /> cross-hybridize (data shown). Second, SPT14 encodes         involved additional cellular processes critical <br /> small transcript tightly linked MFWl (21),      cell viability. <br /> MOT1 encodes large transcript 6.2 kb (see Fig. 5A)          Analysis MOTI coding sequence. The putative pro- <br /> is unlinked MFoxl (data shown).                            tein sequence Motl compared sequences the <br />    Nearly 7 kb complementing surrounding             GenBank data base, using TFASTA algorithm (56). This <br /> regions subjected DNA sequence analysis. The               search revealed (Fig. 7) Motl 40% identical a <br /> nucleotide predicted amino acid sequences shown        stretch 530 amino acids S. cerevisiae Snf2 <br /> Fig. 4. One large open reading frame 5,601 nucleotides      protein (47), involved transcriptional regula- <br /> 1,867 amino acids (predicted molecular weight 209,989)         tion glucose-repressible invertase gene, SUC2. Cells <br /> was identified. There significant open reading frames      carrying snf2 mutation unable derepress SUC2 the <br /> on opposite strand. An analysis deduced primary        transcriptional level grown carbon source as <br /> sequence presented later.                                      sucrose raffinose. In addition, like motl-1, snf2 mutations <br />    MOT] expressed cell types. To examine expression     pleiotropic, indicated defects galactose and <br /> of MOTI cell type, internal 3.5-kb BamHI               glycerol utilization effects expression acid <span id='am-178' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-179' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-180' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-181'  ></span><span id='am-182' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>phos</span>- <br /> fragment (Fig. 3) used hybridization probe blot     phatase, non-glucose-repressible gene. Furthermore, the <br /> containing poly(A)+ RNA isogenic MATa, MA To,            SNF2 message present cells grown <br /> MATa/at strains MOT1+ motl-l. The             repressing nonrepressing conditions. However, <br /> probe detected single transcript 6.2 kb       unlike MOT1, snJ2 null mutation lethal is <br /> present cell types level (Fig.     identical phenotype point mutations. Recent evi- <br /> 5A). The size transcript good agreement    dence suggests Snf2 nuclear protein (47), is <br /> size predicted open reading frame. The           consistent proposed role transcription. The Motl <br /> MOT1 message motl-l cells higher        protein sequence motif (residues 195 209) <br /> the corresponding isogenic wild-type strains (Fig. SA), sug-      serve nuclear targeting signal (57). <br /> gesting autoregulatory mechanism involved             Motl 33% identical 556-amino-acid domain <br /> in MOTI expression.                                               S. cerevisiae RAD54 gene product (18), exactly <br />    Expression MOTI transcript response ao-factor     region Motl aligns Snf2 (Fig. 7A). <br /> was examined. There decrease MOTI mRNA                 Examination alignment shown Fig. 7A revealed <br /> after a-factor treatment MOTI+ cells motl-l           regions amino acid identity clustered, with <br /> cells (Fig. SB). It clear experiment     significantly identity intervening sequences. The <br /> the decrease observed direct effect ax-factor action     clusters matches exactly A site the <br />  VOL. 12, 1992 <br />  <br />              rT, <br />                    .a .            Ii <br />                                                PUTATIVE HELICASE AFFECTING GENE EXPRESSION AND GROWTH <br />  <br />                                                                        A.          kb <br />                                                                                    6. 6. <br />                                                                                              1    2 <br />                                                                                                                                          1887 <br />  <br />  <br />  <br />  <br />                                                      _m                            4 4. <br />                                                                                      . <br />  <br />  <br />  <br />  <br />                                                                                    2 3 <br />                                                                                      . <br />  <br />  <br />  <br />  <br />                                                                        B.          kb <br />                                                                                    6. 6. <br />                                                                                    4 4D <br />                                                                                      . <br />  <br />  <br />  <br />  <br />                                                                                    2 3- <br />                                                                                      . <br />  <br />  <br />  <br />  <br />                                                                           FIG. 6. Tetrad analysis MOTJ/motlAl strains. (A) Hybridiza- <br />                    ,,,   -7 <br />  <br />                               ---7 <br />                                      -   7 <br />                                                                        tion analysis (left) tetrad analysis (right) heterozygous <br />         (I -&apos;-17                                                       MOTI/motlAl derivative strain YPH501. (B) Hybridization anal- <br />                                                                        ysis (left) tetrad analysis (right) heterozygous MOTI! <br />              Al-                         4m <br />                                                                        motlWl derivative strain 1788. For Southern blots, genomic DNA <br />                                                                        isolated wild-type (MOTI/MOTI) parent (lane 1) or <br />                                                                        heterozygous disruption strain (MOTJ/motlAl) (lane 2), <br />         *:.::.-,- <br />             _E                                                         digested BamHI, probed MOTI BamHI fragment <br />                                                                        (Fig. 3). The wild-type allele migrates 3.5 kb; motiAl allele <br />    FIG. 5. Expression MOTI mRNA. An internal 3.5-kb BamHI           migrates 6.5 kb. Positions migration HindIll markers are <br /> fragment MOTJ (Fig. 3) used probe. (A) Poly(A)&apos;          shown left blot. For tetrad dissections, spores <br /> RNA MA Ta, MA4Ta, MA Ta/a cells. The relevant genotype        tetrad placed rows left right <br /> is indicated lane; strains STE12&apos; isogenic   center streak growth. One tetrad dissection yielded <br /> at ALT MOTI loci. Migration molecular weight markers        1:3 segregation live-to-dead spores, presumably result of <br /> is shown left. The PKCI probe (see Table 2) used       gene conversion event. <br /> control uniformity RNA loading. (B) Effect pheromone <br /> treatment. Left lanes, RNA AL4 Ta MOTI + cells; right two <br /> lanes, RNA MATa motl-l cells. The CMDI probe (see Table <br /> 2) used control RNA loading. (C) Chromosome blot          vitro (71). Although sequence signatures seven <br /> showing hybridization MOTI chromosome XVI.                       domains distinct SFI SFII, clearly <br />                                                                        related (28). The composite consensus domain (28) <br />                                                                        shown Fig. 7B. Among known putative heli- <br />  nucleotide-binding sequence diverse enzymes             cases described date, domains I, II, VI most <br />  utilize ATP GTP (76) (domain I; Fig. 7B). Further-            highly conserved, domains <span id='am-38' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-41'  ></span><span id='am-42' property="rdfs:label" content="Ia" datatype="xsd:string"></span>Ia</span>, III, IV, V less <br />  more, B site nucleotide-binding domain (14)                conserved. In cases, degenerate forms of <br />  (domain II; Fig. 7B) present. Domain II thought         domains identified (28). <br /> be Mg2e-binding domain interact Mg2+-                       The homologies MOT], SNF2, RAD54 <br /> nucleoside triphosphate conserved Asp residue              coding sequences cluster regions perfect, or <br />  (14). Thus, likely Motl, Snf2, Rad54        near-perfect, matches seven diagnostic helicase <br /> nucleotide-binding proteins.                                           domains, correct order the <br />     Motl, Snf2, Rad54 putative helicases. Further              appropriate spacing. In cases (for example, domain <br /> examination Motl, Snf2, Rad54 alignments re-                III), sequences Motl, Snf2, Rad54 match the <br /> vealed nucleotide-binding site matched best con-            consensus better does Rad3 sequence. Domain <br />  sensus kind site helicases (27). In fact,    IV Motl, Snf2, Rad54 interesting stand- <br />  Motl, Snf2, Rad54 sequences possess seven              point intermediate two <br /> consecutively placed signature domains diagnostic             previously recognized superfamilies. The sequence each <br /> of RNA DNA helicases (27, 28, 37, 49). The primary                 proteins begins F residue defines <br /> sequence proteins begins A B              domain IV SFII, rest sequence more <br /> sites nucleotide-binding domain, followed               similar proteins belonging SFI. In particular, domain IV <br /> additional motifs. These seven helicase domains              Motl Snf2 quite similar domain IV the <br /> delineated comparing sequences 25 known                 helicases <span id='am-9' about='obo:NCBITaxon_11033' typeof='owl:Thing obo:NCBITaxon_11018 obo:NCBITaxon_1 obo:NCBITaxon_10239 obo:NCBITaxon_439488 obo:NCBITaxon_11019 obo:NCBITaxon_177875 obo:NCBITaxon_35278'><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Semliki forest virus" datatype="xsd:string"></span><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Semliki Forest virus</span>, <span id='am-14' about='obo:NCBITaxon_11034' typeof='owl:Thing obo:NCBITaxon_11018 obo:NCBITaxon_1 obo:NCBITaxon_10239 obo:NCBITaxon_439488 obo:NCBITaxon_11019 obo:NCBITaxon_35278 obo:NCBITaxon_177874'><span id='am-15' property="rdfs:label" content="Sindbis virus" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Sindbis virus dsSIN" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19'  ></span>Sindbis virus</span>, and <br /> and putative RNA DNA helicases variety                   <span id='am-20' about='obo:NCBITaxon_12302' typeof='owl:Thing obo:NCBITaxon_12300 obo:NCBITaxon_1 obo:NCBITaxon_10239 obo:NCBITaxon_439488 obo:NCBITaxon_35278 obo:NCBITaxon_39740'><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasBroadSynonym" content="BMV" datatype="xsd:string"></span><span id='am-23'  ></span><span id='am-24' property="rdfs:label" content="Brome mosaic virus" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>brome mosaic virus</span> (all members SFI) the <br /> different organisms, including <span id='am-99' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-100'  ></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia coli</span>, various               sequence R-R-X-X-X-D-V (28, 37). <br /> RNA DNA <span id='am-1' about='obo:NCBITaxon_10239' typeof='owl:Thing obo:NCBITaxon_1'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="viruses" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Viridae" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Vira" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-7'  ></span><span id='am-8' property="rdfs:label" content="Viruses" datatype="xsd:string"></span>viruses</span>, S. cerevisiae, Drosophila melano-                    To align domain V Motl, Snf2, Rad54 domain <br /> gaster, humans (27, 28, 37, 49). These helicases              V Rad3, gaps need introduced (Fig. 7B). <br /> been split superfamilies, SFI SFII, based          Gorbalenya et al. (28) chose delete corresponding <br /> amino acid sequences seven domains (28).                residues Rad3 means maximize alignment of <br /> Proteins belonging SFI include viral proteins           Rad3 putative helicases. Apparently, spacing <br /> the yeast mitochondrial protein Pifl (37),              conserved residues domain V somewhat <br /> shown DNA helicase activity vitro (46). Proteins            variable. <br /> belonging SFII exemplified Rad3 (28),                 Finally, domain VI, Motl, Snf2, Rad54 <br /> also demonstrated DNA helicase activity                lack G residue normally proteins belonging to <br />  1888   DAVIS ET AL.                                                                                                                        MOL. CELL. BIOL. <br />  <br />                 A <br />                 YSCSNF2 (751) <br />                 YSCMOT1 (1257) <br />                                              IMDIKIJQ   LVG6G6KDfIDIUL5VSEFNNUI <br />                                              SIAKP FEL-AI KA   3RKUQDUVNELAFKFHKYI <br />                                                                                                                          M5liuWS N----- <br />                                                                                                                        CEDEUEC I ----- <br />                 YSCRAD54 (294)               YLEAEA FNTUSEDPLKSDEKALTESKTEQNAJRGAYG                                    DNEf3CjOA3&quot;WTLL <br />                 YSCNSF2 (806)                ------------------LT           IYENKhIRGPYEVIVPUSTuSNWSSEFAKW.-------- <br />                 YSCMOT1 (1311)               ---------------.IASDQRKEDYKTRSVESRAEPSUIICPPSLTGHWEIEFDQYI <br />                 YSCRAD54 (354)               RQGPQ6KRLIDKCIIVCPSSLVNNWA                LIKVUGPNTETPUAVDGKKSSNGIJTTVSQU <br />  <br />                 YSCSNF2 (840)                                             VFT FVEY I IKFRALELSffVKR VHI I <br />                                              OTERT IS FKSIINEIKAKAKI RAGEFfI <br />                 YSCMOT1 (1356)               *FEKVVVYAUGUTV1LTLRPQLSDADIIETSYDVARNDL--AVLNETEYNYCVLEIUfII <br />                 YSCRAD54 (414)               IHAWAQAQGRNIVKPVLIISYETLRRNVD                   ------------. KNCNVGLULA               IL <br />  <br />                 YSCSNF2 (900)                  DAEaSLTINTHYYDHELEEINMVLEKI FNEVMSEDEWUNTE <br />                 YSCMOT1 (1414)               MS~&#167;A IDV K- E I TON H        Vl ISFDILNMFTETEMQERUAKI <br />                 YSCRAD54 (461)               QIGDELT FTMUDS I SCPR-IIVMS <br />                                                                       JDlDSUY FMSUSNMLRAEORKNDENE <br />                 YSCSNF2 (960)                FUNTGGQKI ELSEIETULVIRRIERVLRLEEIIKEMENDUVEUVEKUKNSAU <br />                 YSCNOT1 (1473)               uASuNSKTSSIQAUVEA!E Q        LM     fErLSDMPFI IQDYYBEUGDU <br />                 YSCRAD54 (520)               DLRGMDAUATDIITIQEAQUQIJSTIUSKI IIETNDILAUYCEYEHUI FVNJKPU <br />                 YSCSNF2 (1020)               QImJQQMYMRL FIG-I LN6NVNLRGFNN PIGVF-1IE   DFV FEEEEDQINITRE <br />                 YSCMOT1 (1533)               KQIUDFTICQKNVVEIIEUSEIADGKQHIFEALIYNREEAIAj---ILSPNHIQLA <br />                 YSCRAD54 (580)               NEINKLIESIEV---J-KVVEGVGUSQPL-RAUGI- ENNIjLNFEDEFDDEDDL <br />                 YSCSNF2 (1078)               TNDJIWRVAUKFELLDRILPKLUATGHVIUFQTIQIENMEDFLRUIMIKY .------ <br />                 YSCMOT1 (1590)               QVQUYLKQTULLHDI IN---APjLEAEDTC6ECGIGEEMIDKKAEQDEDFPIQNVISQ <br />                 YSCRAD54 (634)               ELPFDYNNPUSKARDVQTKYSAUFEIIERFHKIKfESDEKIVLINUDQTLD---UEK <br />                 YSCSNF2 (1132)                                                 I RC UCobJLULhAh3 I L1l <br />                                                                           an^vvcj3 <br />                                              ------------------------------- <br />                 YSCMOT1 (1647)               HRALIFCQLKDNLDNVENDLFKKYNPSVTYIEflSIDPRDWVEIcK-EIPSIDILE <br />                 YSCRAD54 (691)               MCRYKHYSAV     ---------------U!lTSINKI133DREDLEGQEFIM <br />                 YSCSNF2 (1161) <br />                 YSCMOT1 (1706) <br />                 YSCRAD54 (730)               M5CCMIMIMN R LULIPlPAAsL AllVWMDlD C FIFHSEII <br />                 YSCSNF2 (1221)               *LEUAYKIQLDEDGaIQUGKFDNKSTMEEEEALERSLLDAEEERRKKRESfEEEEELKD <br />                 YSCMOT1 (1766)               PMGLDKFENDAUTUNQQNSGLASNDTHDLDDDPDNVOSQDNEIKNNEDSQAUKqE <br />                 YSCRAD54 (790)               BFQIS?jSLSECCDIKEDVERLFESDNERQKQKNENI CETHETYHCKRCNIQEKQ <br />  <br />                B                                                                                     Ia <br />                YSCRAD3     (39)      MEMPSETVSLL                              (71)     fCSRTNSIIEEAM <br />                YSCRAD54 (332)                                                 (374)     LVNNWAMLIIWEG <br />                YSCSNF2     (789)                                              (808)    *YENJI R6PYM <br />                YSCMOT1     (1294)    mcc                                      (1316)    MRKEDYIKTRSVE <br />                consensus             O                G GKT             0               00       Z        0       ZO <br />                                                           S                                   T <br />  <br />                                                        II                                                 III <br />                YSCRAD3     (226)     VSUDSIVUFMAWNE                           (457)     I1TISPLNYPRNLNU <br />                YSCRAD54 (446)        NCNVGLMLAINWCL                           (480)          VHNESEY FM <br />                YSCSNF2     (885)     EVUWVHMIEIIIN                            (920) <br />                YSCMOT1     (1399)    ETEYNYCVLEIH                             (1433) <br />                consensus                     Z       OOOODE H                           0 OTAT                Z        OZ <br />                                                             D                              S6S <br />                                                        IV                                                               V <br />                YSCRAD3     (539)     VDFPSYEYNES IV                           (590)     CSNGRGAIVARDCVS EDFDHUYGRL <br />                YSCRAD54 (549)        KUII*TNDILAE                             (721)     EEGQE F IS--UCID--IENRLI <br />                YSCSNF2     (989)     MLINKMIEE                                (1152)    MSEYLMIR--ULEl--U ED <br />                YSCMOT1      (1502)       MEEIIEL S                            (1697)    EUPSIDELUTENVE--ELUEII-T-T <br />                consensus                 F           S Z        Z                       Z    Z        OOO SZO               G ZO    Z OO <br />                                          Y           T                                                    T                 S <br />  <br />                                                            VI <br />                YSCRAD3      (658)    ANRHAAUCEGULUGKD <br />                YSCRAD54 (755)        MAAMQMVMD <br />                YSCSNF2      (1186)   MHR_ <br />                 YSCMOT1     (1731) <br />                 consensus            Z           Z    Q     GR      R <br />                                                       H <br />  VOL. 12, 1992                                 PUTATIVE HELICASE AFFECTING GENE EXPRESSION AND GROWTH                                      1889 <br />  <br />   FIG. 7. Homology MOTI SNF2 RAD54. (A) Alignment C terminus Motl Snf2 Rad54. The alignment <br /> was generated using FASTA algorithm (56) optimized visual inspection. Gene designation far left, followed <br /> by amino acid position parentheses. Amino acids depicted one-letter code; identities proteins are <br /> presented white-on-black letters. (B) Comparison helicase domains Rad3, Rad54, Snf2, Motl seven consensus <br /> helicase domains. Gene designation far left, amino acid positions parentheses. Amino acids depicted one-letter <br /> code; identities proteins given white-on-black letters. The consensus sequence domain described <br /> by Gorbalenya et al. (28). 0, hydrophobic residues (I, L, V, M, F, Y, W); Z, charged/polar residues (S, T, D, E, N, Q, K, R). <br />  <br />  <br />  <br /> SFII, rest domain perfect match                 consistent data normal Motl operates in <br /> consensus. Interestingly, members SFI lack G          conjunction general transcription apparatus. Fur- <br /> residue. Thus Motl, Snf2, Rad54, domain VI, like                thermore, motl-l recessive allele likely, <br /> domain IV, appears hybrid SFI SFII                  therefore, represent loss-of-function mutation, nor- <br /> superfamilies.                                                         mal role Motl remove basal transcription <br />                                                                        factors DNA. The persistent occupancy the <br />                           DISCUSSION                                   promoter region gene factors motl-l cell <br />                                                                        explain enhancement gene expression observed <br />    The genetic screen described designed identify          Stel2 absent. The higher level expression these <br />  factors operate absence Stel2 protein,        genes supported presence specific transcriptional <br />  DNA-binding protein required transcriptional             activator, like Stel2, obscure modest effect the <br />  activity pheromone-responsive genes (16, 17, 19, 23, 24).          motl-l mutation. <br />  Such screen uncover mutations specific factors               The MOTJ gene contains open reading frame is <br />  directly downstream Stel2, possibly belonging               capable encoding protein molecular weight of <br /> newly proposed class coactivator proteins (3, 42),          210,000. The C-terminal 568 amino acids Motl homol- <br /> as factors operating independently Ste12              ogous corresponding amino acids S. cerevisiae <br />  affecting general transcriptional machinery, chromatin            Snf2 Rad54 proteins. All proteins the <br />  structure, nontranscriptional aspects gene                seven signature sequences RNA, RNA/DNA, DNA <br /> expression mRNA transport stability. Cells <span id='am-142' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-143'  ></span><span id='am-144' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-145' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>car</span>-             helicases (27, 28, 37, 49). On basis following <br /> rying motl-l allele increased expression              observations, propose Motl, Snf2, Rad54 define <br /> those pheromone-responsive genes exhibit low basal                new <span id='am-80' about='obo:NCBITaxon_subfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="subfamily" datatype="xsd:string"></span>subfamily</span> helicases. First, degree of <br /> transcription absence Stel2. Alteration MOTJ              sequence identity helicase domains greater <br /> had discernible effect expression phero-            Motl, Snf2, Rad54 proteins <br /> mone-responsive genes completely Stel2-                   previously described member helicase <br /> dependent mode transcription. Because motl-l does               <span id='am-77' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-78' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span>. Second, proteins share additional <br /> affect pheromone-responsive genes equivalently, seems <br /> probable effect transcription indirect than <br /> specific pheromone-regulated genes. <br />    Additional evidence presented supports view.                                    Doiain A                  Domain B <br /> First, cycl-lacZ plasmid lacking UAS                  TPR Consensus   AEAIFGEHIIEKLGDLEK|LDAEQK|LLLDINN <br /> exhibits higher level expression motl-l strains                             U,.                    I         m      Ia <br /> isogenic wild-type controls. Other mutations appear               MOTI     75-    APSEDPRESDEVGGTNEGSPLD   VKLEHEMKI 4/8 <br /> to alter chromatin structure vivo shown                    131-    LSDEKLNEILKSGKVLLASNMNDINVIGKA D&#174;NI 6/8 <br /> suppress UAS-less promoter; example, cells                       286-    GLVEQF(SIY&#174;LLLDNLMSINWR HGAAL     E 7/8 <br />                                                                                 417-    NKIEEATHGGELGIRYFVSIKTN LFMHGLLUNV 5/8 <br /> depleted histone H2B H4, elevated transcription                     493-    TTIESL RLDDDISSSVGEIMDELAKLCD     V 6/8 <br /> a PH05 promoter lacking UAS observed re-                           580-    TKNELN   M <br />                                                                                                  FRLVFQNILLEQNPELLQLSF DYV 4/8 <br /> pressing derepressing conditions (32). Second,             FIG. 8. Potential TPR motifs amino-terminal domain the <br /> have identified gene (DED1) (70) previously implicated           MOTJ gene product. The consensus sequence derived com- <br /> in way mating response expression                  parison 47 34-amino-acid TPR repeats different yeast <br /> increased motl-l background, raising possibility            nuclear proteins, defined Sikorski et al. (64) Hirano et al. <br /> that numerous cellular genes affected alteration           (36), indicated line. The amino acids indicated this <br /> of MOT1. It interesting note DNA sequence                   line white-on-black letters frequently (in at <br /> analysis DEDI gene product reveals            40% repeats) positions; amino acids <br /> presumptive helicase, like Motl, furthermore, appears           indicated line frequently observed <br />                                                                        substitutions positions, amino acids indicated below <br /> to role mRNA splicing (40). Finally, demon-               line frequently observed replacements these <br /> strated MOTJ gene essential cell growth            positions. Six segments predicted primary sequence the <br /> viability, consistent conclusion function         MOTI gene product, beginning residue indicated the <br /> Motl includes vital cellular processes exclusively          number left, shown. Amino acids indicated white-on- <br /> confined expression genes mating pathway.                 black letters perfect matches highly conserved <br />    For Stel2-dependent genes expression                residues TPR consensus; amino acids circled match <br /> elevated motl-l mutation, relative effect            residues identical position the <br /> motl-l allele pronounced stel2 background                TPR repeats used compile consensus. The number matches <br /> than STE12 background. In sense, presence             putative TPR repeats Motl highly <br />                                                                        conserved positions TPR consensus given right. It <br /> Stel2 appears counteract influence               noted 47 TPR repeats used derive the <br /> motl-l defect, absolute level expression            consensus sequence, distribution, terms degree match at <br /> of Stel2-dependent genes was, course, higher STE12               conserved positions, 1/8, 2; 2/8, 2; 3/8, 6; 4/8, 6; <br /> cells. If Motl DNA helicase, hypothesis               5/8, 6; 6/8, 9; 7/8, 10; 8/8, 6 (36, 64). <br />  1890    DAVIS ET AL.                                                                                                MOL. CELL. BIOL. <br />  <br /> blocks amino acid identity (1565 1579 1678 1696     4-3 hydrophobic repeat characteristic amphipathic (x <br /> in Motl) helicases. Third,           helices potential form intramolecular <br /> although helicase domains Motl, Snf2,         intermolecular coiled coils (11) (Fig. 4). Thus, is <br /> Rad54 match SFII subfamily,              tempting speculate Motl nucleoskeletal protein <br /> domains, IV VI, somewhat unique       capable interacting DNA C-ter- <br /> hybrid motifs present SFI SFII subclasses.     minal helicase domain associating part <br /> Based sequence comparisons, additional potential          nuclear matrix TPR units through <br /> members newly recognized subfamily             formation helical bundles proteins. These latter <br /> identified S. cerevisiae, RAD16 (61a) STH1, ho-         structural features protein serve correctly <br /> molog SNF2 (47a). The fact proteins         position C-terminal helicase domain carries out <br /> similar respective helicase domains       function properly. <br /> suggests involved similar areas nucleic <br /> acid metabolism, interact similar substrates                                ACKNOWLEDGMENTS <br /> accessory molecules, and/or regulated similar <br /> manner.                                                            We thank David Schild Marian Carlson communication of <br />    Helicases implicated numerous cellular       results prior publication helpful discussions; Stan Fields, <br />                                                                  Gustav Ammerer, Tony Brake, Bob Sikorski, Phil Hieter for <br /> processes require DNA RNA unwinding, including          providing strains, plasmids, both; Gary Anderson yeast <br /> DNA recombination repair (43, 71). However, cells            chromosome blot; L. Riles M. Olson ordered yeast <br /> carrying motl-l mutation sensitive UV      contig set. <br /> X-ray irradiation isogenic normal cells (data             This work supported special grant research (PF-3308) <br /> shown), making unlikely Motl involved major      American <span id='am-83' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-85'  ></span><span id='am-86' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Cancer</span> Society, California Division (to J.L.D.), <br /> DNA repair pathway. Other cellular processes            NIH research grant GM21841, funds Lucille P. <br /> helicases implicated initiation progres-   Markey Charitable Trust, facilities provided Cancer <br /> sion DNA replication (7),. cell proliferation (26, 34), RNA   Research Laboratory University California, Berkeley (to <br /> splicing (30), ribosome assembly (53), initiation transla-    J.T.). <br /> tion (1), transcription (66). Because mutation MOTI                               REFERENCES <br /> has shown elevate transcription certain genes,     1. Abramson, R. D., T. E. Dever, T. G. Lawson, B. K. Ray, R. E. <br /> it tempting speculate helicase involved        Tach, W. C. Merriclk 1987. The ATP-dependent interaction <br /> transcription. The motl-l allele recessive increases          eukaryotic initiation factors mRNA. J. Biol. Chem. <br /> the basal level expression classes genes.           262:3826-3832. <br /> Taken together, observations suggest wild-         2. Arndt, K. T., C. A. Styles, G. R. Fink. 1989. A suppressor <br /> type Motl protein functions, formally, negative regula-         HIS4 transcriptional defect encodes protein homol- <br /> <span id='am-137' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-138' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-140' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-141'  ></span>tor</span> transcription. Perhaps function Motl normally       ogy catalytic subunit protein phosphatases. Cell <br /> to strip expel resident DNA-binding proteins           56:527-537. <br /> cognate sites melting duplex DNA regions            3. Berger, S. L., W. D. Cress, A. Cress, S. J. Triezenberg, L. <br />                                                                      Guarente. 1990. Selective inhibition activated basal <br /> where transcriptional activators initiation factors         transcription acidic activation domain VP16: evidence <br /> would bind. On hand, putative helicase         transcriptional adaptors. Cell 61:1199-1208. <br /> pivotally involved different nuclear process,        4. Boeke, J. D., F. LaCroute, G. R. Fink. 1984. A positive <br /> transcriptional effects observed motl-l cells may, there-         selection mutants lacking orotidine-5&apos;-phosphate decarbox- <br /> fore, indirect. Such situation account            ylase activity yeast: 5-fluoro-orotic acid resistance. Mol. Gen. <br /> relatively modest transcriptional effects (threefold) com-           Genet. 197:345-346. <br /> pared dramatic phenotypic effects slow-growth         5. Boyer, H., D. Roilland-Dussoix. 1969. A complementation <br /> and temperature-sensitive lethality conferred motl-l          analysis restriction modification DNA Esche- <br /> mutation. It important determine              <span id='am-147' about='obo:NCBITaxon_1430853' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_95179 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_7100 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-148' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-149' property="rdfs:label" content="Richia" datatype="xsd:string"></span><span id='am-150'  ></span><span id='am-151' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>richia</span> coli. J. Mol. Biol. 41:459-472. <br />                                                                   6. Brake, A. J., C. Brenner, R. Najarian, P. Laybourn, J. <br /> primary role Motl transcription lies       Merryweather. 1985. Structure genes encoding precursors of <br /> some aspect cellular physiology.                            yeast peptide mating pheromone a-factor, p. 103-108. In <br />    There additional features MOTI coding          M.-J. Gething (ed.), Protein transport secretion. Cold <br /> sequence worth noting. First, numerous           Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. <br /> potential phosphorylation sites different types    7. Brill, S. J., B. Stillman. 1989. Yeast replication factor-A <br /> protein kinases, revealed search protein sequence         functions unwinding SV40 origin DNA replica- <br /> motifs Keybank7 data base (8). Hence, Motl             tion. Nature (London) 342:92-95. <br /> a phosphoprotein activity regulated phos-        8. Brutlag, D. L., J. P. Dautricourt, S. Maulik, J. Relph. 1990. <br /> phorylation.                                                         Improved sensitivity biological sequence database searches. <br />                                                                      Comp. Appl. Biol. Sci. 6:237-245. <br />    Second, entire protein lacks contiguous blocks      9. Burkholder, A. C., L. H. Hartwell. 1985. The yeast a-factor <br /> strikingly hydrophobic segments significant length,           receptor: structural properties deduced sequence of <br /> making unlikely Motl transmembrane protein.             STE2 gene. Nucleic Acids Res. 13:8463-8475. <br />    Finally, N-terminal region precedes       10. Chung, C. T., R. H. Miller. 1988. A rapid convenient <br /> helicase domain, motifs similar            method preparation storage competent bacterial <br /> so-called tetratricopeptide (TPR) units proteins        cells. Nucleic Acids Res. 16:3580. <br /> which modulate RNA synthesis, encoded SSN6                11. Cohen, C., D. A. D. Parry. 1986. a-Helical coiled coils-a <br /> SKI3, proteins important mitosis,          widespread motif proteins. Trends Biochem. Sci. 11:245-248. <br /> encoded CDC23, nuc2+, CDC16, bimA (36, 64) (Fig.          12. Courschesne, W. E., J. Thorner. 1986. Control phero- <br />                                                                      mone response Saccharomyces cerevisiae: isolation the <br /> 8). In case nuc2+ gene product, nuclear scaffold        SS72 gene. Yeast 2:S74. <br /> protein, Ssn6, nuclear phosphoprotein, repeats       13. Davis, T. N., M. S. Urdea, F. R. Masiarz, J. Thorner. 1986. <br /> have shown essential function (36, 62). In            Isolation yeast calmodulin gene: calmodulin essen- <br /> addition, long segments Motl          tial protein. Cell 47:423-431. <br />  VOL. 12, 1992                                    PUTATIVE HELICASE AFFECTING GENE EXPRESSION AND GROWTH                                  1891 <br />  <br />  14. Vos, A. M., L. M. Tong, M. V. Milburn, P. M. Natias, J.               nuclear protein nuc2+. Cell 60:319-328. <br />      Jancarik, S. Noguchi, S. Nishimura, K. Miura, E. Ohtsuka,        37. Hodgman, T. C. 1988. A new <span id='am-127' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-128' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-129' property="rdfs:label" content="superfamily" datatype="xsd:string"></span>superfamily</span> replicative pro- <br />      S. H. Kim. 1988. Three dimensional structure oncogene              teins. Nature (London) 333:23. (Erratum, 333:578.) <br />      protein: catalytic domain human c-H-ras p21. Science 239:         38. Hoffman, C. S., F. Winston. 1987. A ten-minute DNA <br />      888-893.                                                                 preparation yeast efficiently releases autonomous plasmids <br /> 15. Dietzel, C., J. Kurjan. 1987. Pheromonal regulation               transformation Escherichia coli. Gene 57:267-272. <br />      sequence Saccharomyces cerevisiae SST2 gene: model          39. Ito, H., Y. Fukuda, K. Murata, A. Kimura. 1983. Transfor- <br />      desensitization pheromone. Mol. Cell. Biol. 7:4169-4177.          mation intact yeast cells treated alkali cations. J. <br /> 16. Dolan, J. W., S. Fields. 1990. Overproduction yeast            Bacteriol. 153:163-168. <br />      Stel2 protein leads constitutive transcriptional induction.       40. Jamieson, D. J., B. Rahe, J. Pringle, J. D. Beggs. 1991. A <br />      Genes Dev. 4:492-502.                                                    suppressor yeast splicing mutation (prp8-1) encodes a <br /> 17. Dolan, J. W., C. Kirkman, S. Fields. 1989. The yeast Stel2            putative ATP-dependent RNA helicase. Nature (London) 349: <br />      protein binds DNA sequence mediating pheromone                    715-717. <br />      induction. Proc. Natl. Acad. Sci. USA 86:5703-5707.                  41. Jones, E. W. 1977. Proteinase mutants Saccharomyces cere- <br />  18. Emery, H. S., D. Schild, D. E. Kellogg, R. K. Mortimer.              visiae. Genetics 85:23-33. <br />      1991. Sequence RAD54, Saccharomyces cerevisiae gene             42. Kelleher, R. J., III, P. M. Flanagan, R. D. Kornberg. 1990. <br />      involved recombination repair. Gene 104:103-106.                  A novel mediator activator proteins RNA <br />  19. Errede, B., G. Ammerer. 1989. STE12, protein involved           polymerase II transcription apparatus. Cell 61:1209-1215. <br />      cell-type-specific transcription signal transduction yeast,   43. Kodahek, T., B. Alberts. 1987. Stimulation protein strand <br />      protein-DNA complexes. Genes Dev. 3:1349-1361.                exchange DNA helicase. Nature (London) 326:312-314. <br /> 20. Estruch, F., M. Carlson. 1990. SNF6 encodes nuclear             44. Kronstad, J. W., J. A. Holly, V. L. MacKay. 1987. A yeast <br />      protein required expression genes Sac-            operator overlaps upstream activation site. Cell 50:369-377. <br />      charomyces cerevisiae. Mol. Cell. Biol. 10:2544-2553.                45. Kuchler, K., R. E. Sterne, J. Thorner. 1989. Saccharomyces <br /> 21. Fassler, J. S., W. Gray, J. P. Lee, G. Yu, G. Gingerich.              cerevisiae STE6 gene product: novel pathway protein <br />      1991. The Saccharomyces cerevisiae SPT14 gene essential           export eukaryotic cells. EMBO J. 8:3973-3984. <br />      normal expression yeast transposon, Ty,        46. Lahaye, A., H. Stahl, D. Thines-Sempoux, F. Foury. 1991. <br />      expression HIS4 gene genes mating              PIFl: DNA helicase yeast mitochondria. EMBO J. 10:997- <br />      pathway. Mol. Gen. Genet. 230:310-320.                                   1007. <br /> 22. Feinberg, A. P., B. Vogelstein. 1983. A technique             47. Laurent, B. C., M. A. Treitel, M. Carlson. 1991. Functional <br />      radiolabeling DNA restriction endonuclease fragments high             interdependence yeast SNF2, SNF5 SNF6 proteins in <br />      specific activity. Anal. Biochem. 132:6-13.                              transcriptional activation. Proc. Nati. Acad. Sci. USA 88:2687- <br /> 23. Fields, S., D. T. Chaleff, G. F. Sprague, Jr. 1988. Yeast             2691. <br />      STE7, STE11, STE12 genes required expression          47a.Laurent, B. C., X. Yang, M. Carlson. Personal communi- <br />      cell-type-specific genes. Mol. Cell. Biol. 8:551-556.                    cation. <br /> 24. Fields, S., I. Herskowitz. 1985. The yeast STE12 product       48. Levin, D. E., F. 0. Fields, R. Kunisawa, J. M. Bishop, J. <br />      required expression sets cell-type-specific genes.         Thorner. 1990. A candidate protein kinase C gene, PKC1, is <br />      Cell 42:923-930.                                                         required S. cerevisiae cell cycle. Cell 62:213-224. <br /> 25. Fields, S., I. Herskowitz. 1987. Regulation yeast          49. Linder, P., P. F. Lasko, P. J. Nielsen, K. Nishi, P. P. <br />      mating-type locus STE12, gene required cell-type-               Slonimski. 1989. Birth D-E-A-D  <br /> </body></html>