<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Strategy Survival Critical High <br /> Temperature Eschierichia coli <br /> Masayuki Murata1., Hiroko Fujimoto1., Kaori Nishimura1, Kannikar Charoensuk1, Hiroshi Nagamitsu1, <br /> Satish Raina2, Tomoyuki Kosaka3, <span id='am-217' about='obo:NCBITaxon_85148' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_77689 obo:NCBITaxon_131567 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_75389 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6656 obo:NCBITaxon_6681 obo:NCBITaxon_75390 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_77690 obo:NCBITaxon_85304 obo:NCBITaxon_6072'><span id='am-218' property="rdfs:label" content="Taku" datatype="xsd:string"></span><span id='am-219' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-220'  ></span><span id='am-221' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Taku</span> Oshima4, Naotake Ogasawara4, Mamoru Yamada1,3* <br /> 1 Applied Molecular Bioscience, Graduate School Medicine, Yamaguchi University, Ube, Japan, 2 Division Medical Biochemical Microbiology, Research Center <br /> Borstel, Borstel, Germany, 3 Department Biological Chemistry, Faculty Agriculture, Yamaguchi University, Yamaguchi, Japan, 4 Graduate School Information <br /> Science, Nara Institute Science Technology, Ikoma, Nara, Japan <br />  <br />  <br />  <br />      Abstract <br />      The molecular mechanism supporting survival critical high temperature (CHT) <span id='am-141' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-142' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-146' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-150'  ></span><span id='am-151' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span>Escherichia coli</span> investigated. <br />      Genome-wide screening single-gene knockout library provided list genes indispensable growth 47uC, <br />      called thermotolerant genes. Genes expression affected exposure CHT identified DNA chip <br />      analysis. Unexpectedly, contents did overlap dnaJ dnaK, indicating a <br />      specific set non-heat shock genes required organism survive severe condition. More half of <br />      mutants thermotolerant genes sensitive H2O2 30uC, suggesting mechanism of <br />      thermotolerance partially overlaps oxidative stress resistance. Their encoded enzymes proteins related to <br />      outer membrane organization, DNA double-strand break repair, tRNA modification, protein quality control, translation <br />      control cell division. DNA chip analyses essential genes suggest genes encoding ribosomal proteins <br />      down-regulated CHT. Bioinformatics analysis comparison genomic information microbes <br />      suggest E. coli possesses systems survival CHT. <span id='am-67' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-70'  ></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> analysis allows speculate a <br />      lipopolysaccharide biosynthesis outer membrane organization sulfur-relay tRNA modification <br />      acquired horizontal gene transfer. <br />  <br />   Citation: Murata M, Fujimoto H, Nishimura K, Charoensuk K, Nagamitsu H, et al. (2011) Molecular Strategy Survival Critical High Temperature in <br />   Eschierichia coli. PLoS ONE 6(6): e20063. doi:10.1371/journal.pone.0020063 <br />   Editor: Mick F. Tuite, University Kent, United <span id='am-194' about='obo:NCBITaxon_kingdom' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-195' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-196' property="rdfs:label" content="kingdom" datatype="xsd:string"></span>Kingdom</span> <br />   Received January 6, 2011; Accepted April 12, 2011; Published June 10, 2011 <br />   This open-access article, free copyright, freely reproduced, distributed, transmitted, modified, built upon, used for <br />   lawful purpose. The work available Creative Commons CC0 public domain dedication. <br />   Funding: This work supported Program Promotion Basic Research Activities Innovative Biosciences Special Coordination Funds for <br />   Promoting Science Technology, Ministry Education, Culture, Sports, Science Technology. This work performed collaborative research the <br />   Asian Core Program Yamaguchi University Khon Kaen University, supported Scientific Cooperation Program agreed the <br />   Japan Society Promotion Science (JSPS) National Research Council Thailand (NRCT). These funders role study design, <span id='am-202' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-203' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-205'  ></span><span id='am-206' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> collection <br />   analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * E-mail: m-yamada@yamaguchi-u.ac.jp <br />   . These authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                            folding, refolding, quality control degradation [6]. HSPs are <br />                                                                                         complex regulations divided regulatory <br />    Responses <span id='am-157' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-158' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-161'  ></span><span id='am-162' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-163' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span>Escherichia</span> coli high temperatures                         groups major stimulons [7]. The control their <br /> extensively investigated, previous studies mainly                           expression, however, highly variable organisms and <br /> focused response temperature up-shift 42uC,                        various bacteria [8]. <br /> response known heat shock response (HSR) induce                                One control elements Gram-negative bacteria is <br /> expression set proteins, heat-shock proteins (HSPs) [1]. The                    heat shock sigma factor regulates transcription major <br /> fact HSPs conserved <span id='am-77' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-78' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> indicates                          HSPs. HSR E. coli generally mediated alternative sigma <br /> the actions HSRs fundamentally physiologically                           factors, sigma 32 sigma 24 [4,7,8]. Transcription rpoH <br /> important mechanisms living organisms [2,3]. HSPs play crucial                       gene sigma 32 induced elevated temperature action <br /> roles rescue removal proteins damaged                          sigma 24 [7]. Sigma 24, inactive non-stress <br /> environmental stresses, including heat stress salt stress,                      conditions interaction anti-sigma factor, activated by <br /> also intrinsic folding proteins normal growth                           misfolding outer membrane periplasmic proteins by <br /> conditions [4].                                                                         stresses including heat shock [9]. Both sigma factors further <br />    It shown 384 genes up-regulated short-time                      regulated translation level posttranslational <br /> exposure temperature 43uC heat shock E. coli [5],                       level. The factor sigma 24 regulated cognate small <br /> and genes directly indirectly induced                            RNA, sigma 32 synthesis regulated structural change of <br /> treatment. The directly induced genes encode HSPs, including                        mRNA molecules serving cellular thermometer its <br /> main cellular chaperone machineries GroEL DnaK, ATP-                             activity modulated phosphorylation [10,11]. Other microor- <br /> dependent proteases Lon, HslUV, Clp FtsH (HflB),                                 ganisms, hand, appear possess diverged regulatory <br /> periplasmic protease DegP, proteins involved protein                       mechanisms [12]. <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                   1                                   June 2011 | Volume 6 | Issue 6 | e20063 <br />                                                                                                        E. coli Survival Critical High Temperature <br />  <br />  <br />    There information molecular mechanisms                     expressional alteration 4 genes nearly both <br /> response survival critical high temperature (CHT)              temperatures, growth corresponding mutant strains 37uC <br /> organisms, probably limited experimental procedures.               significantly changed parental strain. <br /> Developments single-gene knockout library DNA chip                   Taken together, results suggest thermotolerant <br /> analysis encouraged perform genome-wide investi-                 phenotype 51 mutants disruption targeted <br /> gation responses organisms extreme conditions. Since              gene polar effect downstream genes. Out of <br /> several mesophilic bacteria including E. coli grow survive            51 thermotolerant genes, 8 genes, cydB, degP, dnaJ, dnaK, dnaQ, <br /> at high temperatures compared mesophilic bacteria,              nlpI, rfaD rfaC, reported genes supporting growth <br /> are assumed acquired potential thermotolerance                high temperature [16&#8211;22], newly identified 43 <br /> during evolution. In study, utilized new procedures             thermotolerant genes organism. <br /> for time obtain information molecular <br /> mechanisms related thermotolerance E. coli CHT.                      Effects supplements oxidative stress growth of <br /> Screening thermosensitive mutants CHT informatics                   thermosensitive mutant strains <br /> analysis corresponding genes revealed pathways factors                 Since LB utilized medium screening of <br /> indispensable survival CHT. For essential genes,                 thermosensitive mutants, limitation carbon source cause <br /> possible involvement response CHT examined                   sensitiveness CHT. We examined effect glucose a <br /> DNA chip analysis. Based results, propose novel                     supplement growth thermosensitive mutant strains <br /> molecular mechanisms survival CHT E. coli.                          (Table 1, Table 2 Figure S4). We tested effect of <br />                                                                               MgCl2 Mg2+ protects cell damage under <br /> Results Discussion                                                        stress conditions [23,24]. The growth 20 37 mutants was <br />                                                                               improved CHT addition 0.5% glucose 20 mM <br /> Thermosensitive mutants thermotolerant genes <br />                                                                               MgCl2, respectively. The growth sixteen mutant strains was <br />    In <span id='am-80' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-81' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> identify genes required survival CHT E. coli, <br />                                                                               improved supplementation glucose MgCl2. <br /> we screened thermosensitive mutants single-gene <br />                                                                                  Next, effect exogenous oxidative stress thermo- <br /> knockout library [13], constructed according to <br />                                                                               sensitive mutant strains tested higher temperature <br /> the one-step gene disruption method aph cassette [14] and <br />                                                                               causes oxidative stress (Noor et al, 2009; unpublished data). <br /> for construct confirmed extensively [15]. In <br />                                                                               We exposed thermosensitive mutant strains 0.5 mM H2O2 in <br /> the disrupted gene mutant strain, region the <br />                                                                               LB liquid medium 30uC. Twenty-nine mutants to <br /> 1st <span id='am-72' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75'  ></span><span id='am-76' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> 6 codons displaced aph cassette, <br />                                                                               sensitive H2O2 (Table 1, Table 2 Figure S4), <br /> so coding region gene deleted. Our <br />                                                                               corresponding approximately 60% thermosensitive <br /> experiments indicated parental strain used construc- <br />                                                                               mutants. Moreover, 10 thermosensitive mutants for <br /> tion disrupted library able grow temperatures to <br />                                                                               glucose MgCl2 supplementation effect, 9 <br /> 47uC, temperature CHT. <br />                                                                               mutants showed sensitivity H2O2. These results suggest the <br />    After successive screening steps library, including <br />                                                                               mechanism thermotolerance CHT partially overlaps with <br /> 3,908 disrupted-mutant strains, 51 strains be <br />                                                                               oxidative stress resistance. <br /> sensitive CHT. Their growth curves 37uC, 45uC 46uC <br /> were compared parental strain (Figure S1). <br /> The growth profiles suggest mutants selected                    Bioinformatics analysis classification of <br /> significantly sensitive 46uC 45uC. Such                 thermotolerant genes <br /> disrupted gene responsible thermosensitive phenotype                 To understand molecular mechanism E. coli survival at <br /> designated thermotolerant gene (Table 1 Table 2).                    CHT, bioinformatics analysis various public databases <br />    The gene organization generated construction                     including KEGG PATHWAY database performed. Out <br /> disrupted mutants rise polar effect inserted           51 thermotolerant genes, 29 genes successfully mapped <br /> aph gene transcription downstream genes                        E. coli pathways KEGG PATHWAY database. <br /> intrinsically transcribed read-through promoter                Interestingly, genes involved same <br /> the region upstream disrupted gene. Such organization               metabolic pathway, suggesting organism possesses <br /> was 42 51 mutants. Sensitivity               indispensable pathways CHT. The remaining 19 genes except <br /> polar effect 29 42 mutants disruption genes            3 unknown genes extensively analyzed using DDBJ <br /> just downstream disrupted gene method                    GenBank database. On basis results analyses and <br /> caused thermosensitive phenotype. The remaining 13 mutants                 effects supplements, 51 thermotolerant genes were <br /> have essential gene thermotolerant gene                  classified 7 groups (Table 1 Table 2). <br /> immediate downstream gene (Figure S2). Their possible polar                      Group A consists genes concerned energy metabolism <br /> effects tested RT-PCR total RNA prepared                    production ATP. The gene products aceE, aceF, lpd and <br /> from cells exposed temperature 37uC 47uC (Figure S3).              ackA mapped pyruvate metabolism pathway from <br /> The results suggest transcription level immediate             pyruvate acetyl CoA [25&#8211;28] rpe located the <br /> downstream gene mutant the              pentose phosphate pathway. cydB yhcB encode subunits of <br /> parent cases cases mutants aceF, tolQ, dnaK       cytochrome d terminal oxidase, generates membrane <br /> and rpsF. Most downstream genes                potential responsible ATP synthesis [16,29]. lipA, which <br /> own promoters transcription level read-through                encodes LipA produce lipoate required pyruvate dehydro- <br /> be nearly the aph promoter. However,                  genase reaction, contributes pyruvate metabolism [30]. <br /> transcription levels lpd, tolR dnaJ located downstream              Based results showing disrupted mutations these <br /> aceF, tolQ dnaK, respectively, increased level            genes caused thermosensitive phenotype, assumed the <br /> rpsR located downstream rpsF decreased compared               cells require ATP higher temperature. This assumption <br /> of parental strain temperatures. Although                     supported finding phenotype mutants <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         2                            June 2011 | Volume 6 | Issue 6 | e20063 <br />                                                                                                                         E. coli Survival Critical High Temperature <br />  <br />  <br />  <br />  Table 1. Thermotolerant genes identified study. <br />  <br />  <br />  Classification        Sub-classification                       Gene     Function                                                                     Glca Mg2+b H2O2c <br />  <br />  Energy metabolism Pyruvate metabolism                          aceE     pyruvate dehydrogenase, decarboxylase component E1                           ++              S <br />  (Group A)             Pyruvate metabolism                      aceF     pyruvate dehydrogenase, dihydrolipoyltransacetylase component E2             ++     ++ <br />                        Pyruvate metabolism                      lpd      lipoamide dehydrogenase, E3 component enzyme                ++ <br />                                                                          complexes <br />                        Pyruvate metabolism                      lipA     lipoate synthase                                                             ++     + <br />                        Propanate metabolism                     ackA     acetate kinase A propionate kinase 2                                            ++       S <br />                        Pentose phosphate pathway                rpe      D-ribulose-5-phosphate 3-epimerase                                           ++              S <br />                        Respiratory chain                        cydB     cytochrome d terminal oxidase, subunit II                                    +      ++ <br />                        Respiratory chain                        yhcB     cytochrome d terminal oxidase, subunit III                                          ++       S <br />  Outer membrane        Lipopolysaccharide biosynthesis          gmhB     D,D-heptose 1,7-bisphosphate phosphatase                                            + <br />  stabilization         Lipopolysaccharide biosynthesis          lpcA     D-sedoheptulose 7-phosphate isomerase                                        +      ++ <br />  (Group B)             Lipopolysaccharide biosynthesis          rfaC     ADP-heptose:LPS heptosyl transferase I                                              ++ <br />                        Lipopolysaccharide biosynthesis          rfaD     ADP-L-glycero-D-mannoheptose-6-epimerase, NAD(P)-binding                            ++ <br />                        Lipopolysaccharide biosynthesis          rfaE     fused heptose 7-phosphate kinase heptose 1-phosphate                            + <br />                                                                          adenyltransferase <br />                        Lipopolysaccharide biosynthesis          rfaF     ADP-heptose:LPS heptosyltransferase II                                              +        S <br />                        Lipopolysaccharide biosynthesis          rfaG     glucosyltransferase I                                                               + <br />                        Peptidoglycan-associated lipoprotein     ydcL     predicted lipoprotein                                                        +      +        S <br />                        Peptidoglycan-associated lipoprotein     yfgL     protein assembly complex, lipoprotein component                                     ++       S <br />                        Peptidoglycan-associated lipoprotein     ynbE     predicted lipoprotein                                                        +      + <br />                        Peptidoglycan-associated lipoprotein     nlpI     conserved protein                                                            +      + <br />                        Peptidoglycan-associated lipoprotein     ycdO     conserved protein                                                            +      + <br />                        Outer membrane integrity                 pal      peptidoglycan-associated outer membrane lipoprotein                                 ++ <br />                        Outer membrane integrity                 tolQ     membrane spanning protein TolA-TolQ-TolR complex                                 ++ <br />                        Outer membrane integrity                 tolR     membrane spanning protein TolA-TolQ-TolR complex                                 ++       S <br />                        Outer membrane integrity                 yciM     conserved hypothetical protein                                                      + <br />  <br />  a <br />    According data Figure S4, ratios growth presence glucose absence glucose 46uC wer estimated. &#8216;&#8216;++&#8217;&#8217; &#8216;&#8216;+&#8217;&#8217; represent more <br />    2.0 1.5&#8211;2.0, respectively. <br />  b <br />    According data Figure S4, ratios growth presence MgCl2 absence MgCl2 46uC estimated. &#8216;&#8216;++&#8217;&#8217; &#8216;&#8216;+&#8217;&#8217; represent than <br />    2.0 1.5&#8211;2.0, respectively. <br />  c <br />   According data Figure S4, ratios growth presence H2O2 absence H2O2 30uC estimated. &#8216;&#8216;S&#8217;&#8217; represents 0.5. <br />  doi:10.1371/journal.pone.0020063.t001 <br />  <br />  <br /> in group partially suppressed addition glucose                        maintain outer membrane integrity, especially high temper- <br /> (Table 1 Figure S4).                                                                 ature. Similarly, data suggest Mg2+ able stabilize the <br />    Group B consists genes related biosynthesis cell wall                    outer membrane structure LPS biosynthesis pathway <br /> or organization outer membrane. The products gmhB,                             defective. <br /> lpcA(gmhA), rfaC (waaC), rfaD (waaD/htrM), rfaE (gmhC), rfaF (waaF)                         Group C consists dnaQ, holC, <span id='am-54' about='obo:NCBITaxon_558101' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_577241 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_71526 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_116151 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-55' property="rdfs:label" content="Pria" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58'  ></span>priA</span>, ruvA ruvC DNA <br /> and rfaG (waaG) mapped lipopolysaccharide (LPS)                            double-strand break repair (DSBR) [37]. DnaQ HolC are <br /> biosynthesis pathway [17,18,31,32]. The products genes                          epsilon chi subunits, respectively, DNA polymerase III <br /> are involved synthesis heptose unit ADP-L-glycero-D-                        [38,39], required homologous recombination in <br /> manno-heptose sedoheptulose-7phosphate encode early                              DSBR [19]. RuvA RuvC act DNA helicase and <br /> heptosyl transferases KDO-lipid A (rfaC rfaF)                             endonuclease, respectively [19,40], replication restart <br /> further extend inner core LPS glcosyltransferase (rfaG).                     DSBR process, PriA functions DNA helicase the <br /> ydcL, yfgL (bamB), ynbE, nlpI ycdO encode peptidoglycan-                             replication restart [41]. The requirement DSBR survival at <br /> associated outer membrane lipoproteins, products pal,                         CHT suggests DNA molecules subjected double- <br /> tolQ tolR components complex structure forming                           strand breaks higher temperature. Interestingly, mutants all <br /> biopolymer transporter [33,34]. yciM encodes protein possibly                          members group exhibited sensitivity oxidative stress at <br /> required integrity outer membrane [35]. The                                   30uC. Therefore, thought strong connection <br /> thermosensitive phenotype caused disrupted mutants of                          oxidative stress DNA double-strand breaks. <br /> these genes significantly suppressed addition Mg2+                            Group D includes genes tRNA modification. Products of <br /> (Table 1 Figure S4). Since Mg2+ known stabilize outer                      iscS, yheL (tusB), yheM (tusC), yheN (tusD) yhhP (tusA) been <br /> membrane structure binding extracellularly [36], assumed                        demonstrated compose sulfur-relay [42&#8211;44]. IscS a <br /> that YdcL, YfgL, YnbE, NlpI, YcdO, Pal, TolQ, TolR YciM                              widely distributed cysteine desulfurase catalyzes desulfuration <br /> act components scaffold proteins membrane                                L-cysteine transfer sulfur active-site cysteine to <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                    3                                   June 2011 | Volume 6 | Issue 6 | e20063 <br />                                                                                                                         E. coli Survival Critical High Temperature <br />  <br />  <br />  <br />  Table 2. Thermotolerant genes identified study. <br />  <br />  <br />  Classification          Sub-classification                  Gene Function                                                                            Glca Mg2+b H2O2c <br />  <br />  DNA repair              DNA replication &amp; repair, DSBR      dnaQ DNA polymerase III subunit, epsilon                                                        +        S <br />  (Group C)               DNA replication &amp; repair, DSBR      holC   DNA polymerase III subunit, chi                                                          ++       S <br />                          DNA replication &amp; repair, DSBR      priA   primosome factor n&#8217;                                                               ++     +        S <br />                          DNA repair, DSBR                    ruvA   component RuvABC resolvasome, endonuclease                                     ++     ++       S <br />                          DNA repair, DSBR                    ruvC   conserved protein required cell growth                                        +      +        S <br />  tRNA modification       tRNA modification                   iscS   sulfer relay system, cysteine desulfurase                                                         S <br />  (Group D)               tRNA modification                   yheL   sulfer relay system, predicted intracellular sulfur oxidation protein                             S <br />                          tRNA modification                   yheM sulfer relay system, predicted intracellular sulfur oxidation protein                               S <br />                          tRNA modification                   yheN   sulfer relay system, predicted intracellular sulfur oxidation protein                             S <br />                          tRNA modification                   yhhP   conserved protein required cell growth                                                        S <br />                          tRNA modification                   miaA   <span id='am-197' about='obo:NCBITaxon_998453' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_50638 obo:NCBITaxon_7438 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7434 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_34725 obo:NCBITaxon_50557'><span id='am-198' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-199'  ></span><span id='am-200' property="rdfs:label" content="Delta" datatype="xsd:string"></span><span id='am-201' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>delta</span>(2)-isopentenylpyrophosphate tRNA-adenosine transferase                             +        S <br />                          tRNA modification                   trmU   tRNA (5-methylaminomethyl-2-thiouridylate)- methyltransferase                                     S <br />                          tRNA modification                   truA   pseudouridylate synthase I                                                        + <br />  Chaperone/protease      Chaperon                     dnaJ   chaperone Hsp40, co-chaperone DnaK                                                  ++       S <br />  (Group E)               Chaperon                     dnaK   chaperone Hsp70, co-chaperone DnaJ                                                           S <br />                          Chaperon                     degP   chaperone/serine endoprotease <br />                          Chaperon regulator                  rseA   anti-sigma factor                                                                        ++       S <br />  Translation control     Translation control                 rpmJ   50S ribosomal subunit L36, related secY expression                                             S <br />  (Group F)               Translation control                 rpsF   30S ribosomal subunit S6, supecifically modified glutamic acid                            S <br />                                                                     phosphate <br />                          Translation control                 dksA   DNA-binding transcriptional regulator rRNA transcription DnaK suppressor +            +        S <br />                          Translation control                 smpB component trans-translation process                                              +      +        S <br />  Cell division           Related cell division            xerC   site-specific tyrosine recombinse involved chromosome dimer resolution                ++       S <br />  (Group G)               Related cell division            dedD   membrane-anchored periplasmic protein involved separation                             ++ <br />                          Related cell division            envC   regulator cell wall hydrolases responsible cell separation                        + <br />  Others                  Membrane transport                  zntA   zinc/cadmium/mercury/lead-exporting ATPase                                               +        + <br />                          Membrane transport                  ybgH   predicted proton-dependent oligopeptide Transporter, POT <span id='am-59' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-60' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span>                   +      ++ <br />                          Membrane transpor                   ybhH   conserved hypothetical protein                                                    ++     ++       S <br />  <br />  a <br />    According data Figure S4, ratios growth presence glucose absence glucose 46uC wer estimated. &#8216;&#8216;++&#8217;&#8217; &#8216;&#8216;+&#8217;&#8217; represent more <br />    2.0 1.5&#8211;2.0, respectively. <br />  b <br />    According data Figure S4, ratios growth presence MgCl2 absence MgCl2 46uC estimated. &#8216;&#8216;++&#8217;&#8217; &#8216;&#8216;+&#8217;&#8217; represent than <br />    2.0 1.5&#8211;2.0, respectively. <br />  c <br />   According data Figure S4, ratios growth presence H2O2 absence H2O2 30uC estimated. &#8216;&#8216;S&#8217;&#8217; represents 0.5. <br />  doi:10.1371/journal.pone.0020063.t002 <br />  <br />  <br /> form persulfide group (-SSH), responsible                          dnaK dnaJ encode chaperone co-chaperone, respectively, <br /> YheL, YheM, YheN YhhP biosynthesis 2-thio                                 maturation protein folding refolding unfolded proteins <br /> modification 5-methylaminomethyl-2-thiouridine (mnm5s2U)                              [20,50], degP encodes chaperone/serine protease located in <br /> [43] different thio modifications bacterial tRNAs [45].                      periplasm [21]. The indispensability genes CHT <br /> IscS works general sulfur donor various metabolic                           suggests DnaK/DnaJ play crucial role dealing with <br /> pathways [46] including biosynthesis iron-sulfur (Fe-S) cluster                       unfolded proteins caused CHT DegP plays an <br /> [47], thiamine [48], nicotinic acid branched-chain amino                             important role removal damaged proteins have <br /> acids [49]. Additionally, miaA, trmU truA group                          accumulated temperature. rseA group encodes an <br /> involved tRNA modification. The mutations genes related                         anti-sigma factor sigma 24 inactive non-stress <br /> sulfer modification cause phenotype sensitivity anti-                          conditions. The thermosensitivity caused rseA disrupted <br /> oxidation stress [44]. Consistently, study provided evidence                         mutation suggests fine tuning intracellular level of <br /> that mutants group exhibited hypersensitivity oxidative                       active sigma 24 regulates expression chaperone protease <br /> stress. YheL, YheM, YheN YhhP, mainly function                              genes crucial adaptation CHT condition. <br /> tRNA modification [45], conserved thermotolerant bacteria                         Alternatively, defective mutant rseA increased sigma 24 <br /> in mesophiles (see Table S3), iscS, general sulfur donor,                      activity, turn decreased production outer <br /> is widely conserved mesophiles.                                                       membrane proteins MicA RybB sigma 24 regulon <br />    These findings suggest tRNA modifications presented                         gene [51], resulting membrane unstability thermosensi- <br /> are indispensable growth CHT.                                                     tiveness CHT. <br />    Group E genes encode chaperones protease                                  Genes group F belong translation control apparatus. <br /> contribute cellular process regulating heat shock response:                    S6 encoded rpsF interacts central domain 16S rRNA <br />  <br />  <br />        PLoS ONE | www.plosone.org                                                    4                                   June 2011 | Volume 6 | Issue 6 | e20063 <br />                                                                                                            E. coli Survival Critical High Temperature <br />  <br />  <br /> and demonstrated play regulatory                  Expressional change caused heat shock CHT <br /> structural role ribosome [52]. L36 encoded rpmJ                   None thermotolerant genes identified study were <br /> component 50S subunit ribosome, disruption              encode HSPs previously identified E. coli for <br /> decreases expression secY [53], encodes protein-               dnaJ, dnaK, degP dnaQ. To examine thermotolerant <br /> conducting channel cytoplasmic membrane. DksA encoded                  genes up-regulated CHT not, examined transient <br /> by dksA functions negative regulator rRNA genes [54].                change expression genomic genes CHT DNA chip <br /> Overexpression DksA shown suppressor                analysis. The results showed 42 genes 111 genes were <br /> dnaK deletion mutation [55] ensures replication completion             significantly up-regulated down-regulated, respectively (Table <br /> removing transcription roadblocks [56]. SmpB encoded smpB               S4). The up-regulated genes classified mainly genes <br /> a component trans-translation process performs rescue              involved cellular process, down-regulated genes <br /> of stalled ribosomes binding partner, transfer-messenger             classified genes involved energy metabolism, <br /> RNA [57]. These lines evidence suggest constituents           transport/binding protein translation. However, the <br /> in translation pathways crucial survival CHT.                      thermotolerant genes including degP dnaQ heat-shock gene <br />    Finally, genes group G related cell division. A xerC-            identified up-regulated genes dnaJ dnaK. <br /> encoded protein site-specific recombinase [58] essential          Taken data shown above, possible the <br /> for conversion chromosome dimers monomers cell                   chaperone systems DnaJ/DnaK GroEL/GroES <br /> division. envC encodes component cell division machinery             neccesarily involved thermotolerant mechanisms <br /> that direct regulator cell wall hydrolase responsible         acquired CHT. Therefore, likely products of <br /> cell separation required cell division [59]. DedD                 thermotolerant genes HSPs organism <br /> encoded dedD membrane-anchored periplasmic protein                    possesses specific set genes required survival CHT. <br /> involved septation [60] shown participate                  It possible essential genes crucial for <br /> cytokinesis [61].                                                             growth CHT. Such genes, however, examined in <br />    The functions remaining genes, ybgH, yciM yhhH,             analysis disrupted mutants genes are <br /> unknown. Notably, thermosensitiveness mutations              available conditional mutants. We listed <br /> partially suppressed addition Mg2+. It likely          essential genes significant fluctuation expression CHT <br /> their gene products related cellular activities similar       (Table 3). groEL (groL) encoding HSP up-regulated, indicating <br /> in group B, C G.                                                           possibility gene product contributes survival at <br />                                                                               CHT. Consistently, reported GroEL appears a <br /> Possible acquisition thermotolerant genes                          mediator evolution extremely heat-resistant E. coli cells [63]. <br /> horizontal gene transfer                                                      On hand, 90% down-regulated genes were <br />    Two groups outer membrane integrity tRNA                           mapped translation pathway (Figure S5), encoding for <br /> modification completely conserved limited bacterial             components ribosomal proteins. It possible down- <br /> species optimal growth relatively high temperature                  regulation ribosomal genes strategies survival at <br /> (Table S1). Of group members, genes LPS                         CHT E. coli. Noteworthily, Alix et al. reported ribosome <br /> biosynthesis pathway, lipoproteins sulfur-relay                  biogenesis E. coli high temperature-sensitive DnaK- <br /> system distributed limited bacterial species including            dependant predicted high temperature causes severe <br /> <span id='am-128' about='obo:NCBITaxon_543' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236'><span id='am-129'  ></span><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="Enterobacteriaceae Rahn 1937" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasDbXref" content="PMID:10555323" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasRelatedSynonym" content="Enterobacteraceae (ex Lapage 1979) Lapage 1982, fam. nov., nom. rev." datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasRelatedSynonym" content="gamma-3 proteobacteria" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasDbXref" content="PMID:10555334" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasDbXref" content="PMID:16166704" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasRelatedSynonym" content="Enterobacteriaceae (ex Rahn 1937) Ewing et al. 1980, fam. nov., nom. rev." datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-138' property="rdfs:label" content="Enterobacteriaceae" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasRelatedSynonym" content="Enterobacteraceae" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Enterobacteriaceae</span> (Tables S2 Table S3). The sulfur-relay          limitation DnaK/DnaJ hamper ribosome assembly because <br /> classified tRNA modification demonstrated modify <br /> a nucleotides tRNA molecules, contributing stabilization <br /> of structure, required survival extremely <br />                                                                                Table 3. Essential genes significantly up-regulated and <br /> high temperature Thermous thermophilus [62] also <br />                                                                                down-regulated CHT. <br /> conserved <span id='am-7' about='obo:NCBITaxon_1754' typeof='owl:Thing obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_186801 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_68295 obo:NCBITaxon_186814'><span id='am-8' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2324" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="PMID:7981107" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="Acetigenium" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Clostridium cluster V" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="Thermoanaerobacter" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="PMID:17625170" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Thermobacteroides" datatype="xsd:string"></span><span id='am-17'  ></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Thermoanaerobium" datatype="xsd:string"></span>Thermoanaerobacter</span> tengcongensis (Table S3). The mature <br /> LPS biosynthesis pathway assembly outer membrane <br /> consists enzyme reactions,                       Classificationa       Pathwaya                      Gene <br /> dispensable lower temperature. Interestingly, pathway is <br /> mostly conserved <span id='am-170' about='obo:NCBITaxon_28262' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_203693 obo:NCBITaxon_131567 obo:NCBITaxon_40117 obo:NCBITaxon_2 obo:NCBITaxon_189778 obo:NCBITaxon_28261 obo:NCBITaxon_189779'><span id='am-171' property="oboInOwl:hasExactSynonym" content="Thermodesulfurivibrio yellowstonii" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasRelatedSynonym" content="Thermodesulfovibrio yellowstonensis" datatype="xsd:string"></span><span id='am-173'  ></span><span id='am-174' property="oboInOwl:hasRelatedSynonym" content="Thermodesulfovibrio yellowstonii Henry et al. 1994" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-176' property="rdfs:label" content="Thermodesulfovibrio yellowstonii" datatype="xsd:string"></span><span id='am-177' property="oboInOwl:hasDbXref" content="PMID:8897428" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasDbXref" content="PMID:11541228" datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasExactSynonym" content="Thermodesulfurivibrio yellowstonensis" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Thermodesulfovibrio yellowstonii</span> Thermanaer-           Up-regulated <br /> ovibrio acidaminovorans (Table S2). Enzymes LPS biosynthesis            Cpn60 chaperonine     RNA degradation               groEL <br /> and sulfur-relay E. coli share 40% sequence                    tRNA-Leu              Transfer RNA                  leuU <br /> identity 50% sequence similarity corresponding                Down-regulated <br /> enzymes thermophilic bacteria. E. coli closely related              Lipid metabolism      Fatty acid biosynthesis       fabG <br /> bacteria acquired genes groups <br />                                                                                Transcription         RNA polymerase                rpoA <br /> presumably horizontal gene transfer evolution. <br /> Since groups widely conserved                   Translation           16S rRNA processing protein   rim <br /> thermotolerant mesophilic bacteria mesophilic                Translation           Translation factors           fusA <br /> bacteria, intrinsically present E. coli. This              Translation           Ribosome                      rpsP, rplQ, rpsD, rpsK, <br /> consistent conserved nature essentiality lipid A                                  rpsM, rpmC, rplP, rpsC, <br /> part LPS essentiality synthesis lipid <span id='am-83' about='obo:NCBITaxon_114188' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_4210 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_102804 obo:NCBITaxon_91827 obo:NCBITaxon_911341 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_4209 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_35493 obo:NCBITaxon_2759 obo:NCBITaxon_71274 obo:NCBITaxon_102814 obo:NCBITaxon_3193'><span id='am-84' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-85'  ></span><span id='am-86' property="rdfs:label" content="Iva" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasExactSynonym" content="marsh elder" datatype="xsd:string"></span>IVA</span> but <br />                                                                                                      rplV, rpsS, rplB, rplW, <br /> dispensability enzymes involved extension Kdo2-lipid A <br />                                                                                                      rplD, rplC, rpsJ, rplJ, <br /> by various glycosyltransferases. This draws support Re (rfaC) <br /> mutants tetraacylated lipid A exhibiting narrow                                     rplL, rpsR <br /> growth range ability grow slow growth                       a <br />                                                                                 Classification Pathway accoding KEGG PATHWAY shown. (http:// <br /> conditions minimal medium 23uC [18], suggesting                       www.genome.jp/kegg). <br /> overall importance outer membrane integrity CHT.                         doi:10.1371/journal.pone.0020063.t003 <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                         5                                 June 2011 | Volume 6 | Issue 6 | e20063 <br />                                                                                                       E. coli Survival Critical High Temperature <br />  <br />  <br /> heat-induced misfolded proteins titrate free               Screening thermosensitive mutants <br /> DnaK/DnaJ [64,65].                                                              The Keio collection consisting 3,908 mutant strains used <br />                                                                              screening. In 1st screening, mutant strains grown on <br /> Further consideration mechanisms survival CHT                      LB plates 30uC overnight. A colony strain patched <br />    Two groups DNA double-strand repair chaperone/                     LB plates incubated 47uC 48 h sensitive <br /> proteinase genes contribute endurance oxidative               strains. The sensitive strains subjected 2nd screening of <br /> stress addition CHT. Evidence higher temperature                spotting tests plates. Cells cultured LB medium 18 h <br /> results accumulation oxidative stress [24]                diluted LB medium adjust turbidity OD600 of <br /> finding mutants members groups exhibited                 0.5, 0.05 0.005. The diluted samples (10 ml) spotted on <br /> sensitivity oxidative stress allow speculate oxidative         LB plates incubated 47uC 48 h. The thermosensitive <br /> stress main cause DNA double-strand breaks damage             strains selected 2nd screening subjected 3rd <br /> to proteins CHT. Interestingly, oxidative stress involved           screening liquid culture. After 8-h preculture, cells diluted <br /> heat-induced cell death <span id='am-30' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-31' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> [66],           turbidity corresponding OD600 0.1 inoculated into <br /> supported findings overexpression catalase                LB medium final OD600 0.001. Samples then <br /> superoxide dismutase genes increase degree thermo-              incubated 47uC 18 h shaking condition. Thermo- <br /> tolerance thermotolerance increased                    sensitivity determined measuring OD600. Thermosensitive <br /> anaerobic conditions. We assume CHT causes                 strains defined ,0.1 OD600. The experiments were <br /> intracellular oxidative stress elicit harmful effects cells       performed times, results confirmed be <br /> secondary stress.                                                            reproducible. <br />    Significant suppression thermosensitive phenotype a <br /> defect group energy metabolism (Group A)                    Effects glucose MgCl2 sensitivity H2O2 <br /> addition glucose suggests limitation energy level CHT              To examine effects supplements, glucose (0.5% (w/v)) or <br /> in organism. The limitation resolved alternative          MgCl2 (20 mM) added LB liquid culture. After 8-h <br /> pathways generate ATP glucose assimilation. The                  preculture, cells diluted turbidity corresponding to <br /> requirement ATP CHT consistent expression               OD600 0.1 inoculated LB medium the <br /> ribosomal genes. Many genes ribosomal proteins             supplement final OD600 0.001. Samples then <br /> be down-regulated exposure CHT, disrupted mutant               incubated 47uC 18 h shaking condition. After 18 h, <br /> of dksA encodes negative regulator rRNA genes              turbidity OD600 measured. To test sensitivity to <br /> thermosensitive CHT. These findings evidence                     oxidative stress, H2O2 added culture medium the <br /> translation ribosomal activity utilizes energy,              final concentration 0.5 mM. After 8-h preculture, cells were <br /> about 90% energy consumed cells [67], suggest cells               diluted turbidity corresponding OD600 0.1 and <br /> manage reduce energy consumption severe condition              inoculated LB medium H2O2 final <br /> CHT. Such saved energy utilized crucial                   OD600 0.001. Samples incubated 30uC 8 h <br /> activities repair degradation damaged DNA                   shaking condition. The experiments performed three <br /> protein molecules. A smooth translational process CHT               times, results confirmed reproducible. <br /> also save energy, S6 L36 ribosomal proteins in <br /> addition SmpB important functions. <br />    Several strategies E. coli survive CHT discovered.         RT-PCR analysis <br /> Most responsible stresses                 Cultures grown LB medium 37uC the <br /> conserved mesophilic bacteria. Early glycosyltransferases            exponential phase, temperature up-shifted to <br /> for LPS core biosynthesis proper outer membrane assembly                 47uC incubation continued 8 min. Total RNA was <br /> and permeability barrier function sulfur-relay            immediately prepared heat-stressed cells hot <br /> tRNA modification acquired organism               phenol method [68]. RT-PCR analysis performed using an <br /> perform main task survive CHT. Considering genetic               mRNA-selective RT-PCR kit (TAKARA BIO Inc, Otsu, Japan) to <br /> conversion non-thermotolerant thermotolerant bacteria,             examine expression immediate downstream genes of <br /> two strategies applicable.                                          disrupted genes described previously [69]. The primer set used <br />                                                                              gene shown Table S5. The RT reaction carried <br />                                                                              42uC 15 min, 85uC 1 min, 45uC 1 min and <br /> Materials Methods <br />                                                                              extension 72uC 2 min using specific primers each <br /> Materials                                                                    gene. After completion 15, 20, 25 30 cycles, PCR <br />   Oligonucleotide primers polymerase chain reaction (PCR)                products analyzed 0.9% agarose gel electrophoresis and <br /> were purchased FASMAC Co, Ltd (Atsugi, Japan). Other                    stained ethidium bromide. The relative amounts RT-PCR <br /> chemicals analytical grade.                                      products gel compared measuring band density <br />                                                                              color image taken reversed using model <br /> Bacterial strains growth conditions                                      GS-700 Imaging Densitometer (Bio-Rad Laboratories, Inc, <br />    Strains used study derivatives E. coli K-12.              Tokyo, Japan) [70]. <br /> W3110 (IN (rrnD-rrnE), rph-1), BW25113 (rrnB3, D(lacZ)4787, <br /> hsdR514, D(araBAD)567, D(rhaBAD)568, rph-1) [14] mutants              Bioinformatics phylogenetic analyses <br /> BW25113 Keio collection single-gene knockout library                Bioinformatics analysis mainly performed according the <br /> [13] grown plates liquid modified <span id='am-19' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Luria" datatype="xsd:string"></span>Luria</span>-Bertani             instructions KEGG site (http://www.genome.jp/kegg/). <br /> (LB) medium (1% Bactotryptone, 0.5% yeast extract, 0.5%                  Databases DDBJ (http://www.ddbj.nig.ac.jp) GenBank <br /> NaCl) 37uC, 45uC 47uC appropriate times.                           (http://www.ncbi.nlm.nih.gov/genbank/) used. <br />  <br />  <br />  <br />        PLoS ONE | www.plosone.org                                        6                             June 2011 | Volume 6 | Issue 6 | e20063 <br />                                                                                                                            E. coli Survival Critical High Temperature <br />  <br />  <br />  <br /> DNA chip analysis                                                                           RT-PCR performed primers specific just <br />    W3110 cultures grown LB medium 37uC                                 downstream gene thermotolerant gene amplify about <br /> exponential phase, temperature up-shifted                               500-bp DNA fragments. (A B) After RT reaction, PCR was <br /> 47uC incubation continued 8 min. A control culture                              performed 15, 20, 25 30 cycles products were <br /> was incubated parallel 37uC 8 min. Total RNA                                  analyzed. (C D) As control, total RNA (10 mg) was <br /> immediately prepared heat-stressed cells hot                                submitted 1.2% agarose gel electrophoresis staining with <br /> phenol method [69]. Preparation cDNA, fragmentation                              ethidium bromide. <br /> end-labeling DNA fragments performed according                               (TIF) <br /> instruction manual Affymetrix. The ENZO Bioarray <br />                                                                                             Figure S4 Effects addition glucose MgCl2 and <br /> terminal labeling kit (Enzo Life Sciences, Inc, New York, USA) <br />                                                                                             sensitivity H2O2. Thermosensitive mutant strains shown <br /> was used end-<span id='am-169' about='rdfs:label' typeof='owl:Thing'>label</span> DNA fragments. DNA hybridization, data <br />                                                                                             gene names. Growth conditions described Materials and <br /> capture analyses performed described protocol <br />                                                                                             Methods. Black white columns represent turbidity the <br /> supplied Affymetrix GCOS software (Affymetrix, Inc, <br /> <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="California" datatype="xsd:string"></span>California</span>, USA). Two independent experiments performed                                conditions supplements (0.5% glucose (A) or <br /> and data sets (two data sets 37uC: 37uC-1 37uC-2,                           20 mM MgCl2 (B)) 0.5 mM H2O2 (C). <br /> data sets 47uC: 47uC-1 47uC-2) gene obtained.                               (TIF) <br /> The expression ratio used indicates average ratios                          Figure S5 Down-regulated genes ribosomal pro- <br /> obtained independent experiments. Spots                                   teins. Systematic analysis gene function performed a <br /> significantly lower (,0.50; i.e., negative fold difference) higher                     database KEGG PATHWAY. Down-regulated genes for <br /> (2.; i.e., positive fold difference) fluorescence ratio heated                     ribosomal proteins mapped 6 operons. <br /> sample control sample considered represent real                            (TIF) <br /> significant difference. Physiological function functional <br /> classification genes derived Genobase                                  Table S1 Distribution thermotolerant genes in <br /> database (http://ecoli.aist-nara.ac.jp/). Array data accessible                         various bacteria. <br /> through ArrayExpress accession number E-MEXP-3191.                                          (DOC) <br />                                                                                             Table S2 Distribution thermotolerant genes group <br /> Supporting Information                                                                      B various bacteria. <br /> Figure S1 Growth thermosensitive mutants LB                                           (DOC) <br /> liquid culture different temperatures. Each 51 thermo-                                   Table S3 Distribution thermotolerant genes group <br /> sensitive mutant strain (opened symbols) parental strain,                           D various bacteria. <br /> BW25113 (closed symbols), grown 30 ml LB medium                                  (DOC) <br /> 37uC (circles), 45uC (squares), 46uC (triangles). At times <br /> indicated, turbidity OD600 measured. A, group A; B, group                            Table S4 Genes significantly up-regulated down- <br /> B; C, group C; D, group D; E, group E; F, group F; G, group G;                              regulated CHT. <br /> H, others.                                                                                  (DOC) <br /> (TIF)                                                                                       Table S5 RT-PCR primers used study. <br /> Figure S2 Gene organizations genes having                                            (DOC) <br /> either essential gene thermotolerant gene a <br /> just downstream gene. Gene organizations 13                                          Acknowledgments <br /> thermotolerant genes essential gene a <br />                                                                                             We thank K. Matsushita T. Yakushi helpful discussion, J. <br /> thermotolerant gene just downstream gene depicted. Black                           Makino A. Ishii assisting E. coli growth experiments. We thank <br /> boxes represent identified 13 thermotolerant genes. Grey boxes                              National BioResource Project (NIG, Japan) providing 3908 mutants <br /> represent essential thermotolerant genes. The direction boxes                         KEIO Collection. <br /> shows direction transcription. <br /> (TIF)                                                                                       Author Contributions <br /> Figure S3 Testing possible polar effects aph                                      Conceived designed experiments: MM HF SR NO MY. <br /> insertion. Total RNA prepared cells cultured 37uC                               Performed experiments: MM HF KN KC HN. Analyzed data: <br /> (A, C) 47uC (B, D) described Materials Methods.                               MM HF TK TO MY. Wrote paper: MM HF SR TO MY. <br />  <br /> References <br />  1. Morimoto RI, Kroeger PE, Cotto JJ (1996) The transcriptional regulation               6. Narberhaus F, Kaser R, Nocker A, Hennecke H (1998) A novel DNA element <br />     heat shock genes. plethora heat shock factors regulatory conditions. In:           controls bacterial heat-shock gene expression. Mol Microbiol 28: 315&#8211;323. <br />     Feige U, Morimoto RI, Yahara I, <span id='am-164' about='obo:NCBITaxon_1137717' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_82596 obo:NCBITaxon_82593 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_82592 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-165' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-166'  ></span><span id='am-167' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-168' property="rdfs:label" content="Polla" datatype="xsd:string"></span>Polla</span> BS, eds. The transcriptional regulation            7. Erickson JW, Gross CA (1989) Identification sigma E subunit Escherichia <br />     heat shock genes: A plethora heat shock factors regulatory conditions.            coli RNA polymerase: A second alternate sigma factor involved high- <br />     Basle: Birkha&#168;user-Verlag. pp 139&#8211;163.                                                      temperature gene expression. Genes Dev 3: 1462&#8211;1471. <br />  2. Boorstein WR, Ziegelhoffer T, Craig EA (1994) Molecular evolution                 8. Raina S, Missiakas D, Georgopoulos C (1995) The rpoE gene encoding sigma <br />     HSP70 multigene family. J Mol Evol 38: 1&#8211;17.                                                E (sigma 24) heat-shock sigma factor Escherichia coli. EMBO J 14: 1043&#8211;1055. <br />  3. Gupta RS (1995) Evolution chaperonin families (Hsp60, Hsp10, Tcp-             9. Alba BM, Leeds JA, Onufryk C, Lu CZ, Gross CA (2002) DegS YaeL <br />     1) proteins origin eukaryotic cells. Mol Microbiol 15: 1&#8211;11.                  participate sequentially cleavage RseA activate sigma(E)- <br />  4. Yura T, Kanemori M, Morita MT (2000) The heat shock response: regulation                    dependent extracytoplasmic stress response. Genes Dev 16: 2156&#8211;2168. <br />     function. In: Storz G, Hennecke H, eds. Bacterial stress responses.                 10. Morita M, Kanemori M, Yanagi H, Yura T (1999) Heat-induced synthesis of <br />     Washington, D.C.: ASM Press. pp 3&#8211;18.                                                       sigma32 Escherichia coli: structural functional dissection rpoH mRNA <br />  5. Gunasekera TS, Gsonka LN, Pally O (2008) Genome-wide transcriptional                        secondary structure. J Bacteriol 181: 401&#8211;410. <br />     responses <span id='am-207' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-208' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-209' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-210' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-211' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span>Escherichia coli K-12</span> continuous osmotic heat stresses.             11. Klein G, Dartigalongue C, Raina S (2003) Phosphorylation-mediated regulation <br />     J Bacteriol 190: 3712&#8211;3720.                                                                 heat shock response Escherichia coli. Mol Microbiol 48: 269&#8211;285. <br />  <br />  <br />  <br />         PLoS ONE | www.plosone.org                                                      7                                   June 2011 | Volume 6 | Issue 6 | e20063 <br />                                                                                                                                    E. coli Survival Critical High Temperature <br />  <br />  <br /> 12. Helmann JD, Chamberlin MJ (1988) Structure function bacterial sigma                     40. Bennett RJ, West SC (1996) Resolution Holliday junctions genetic <br />     factors. Annu Rev Biochem 57: 839&#8211;872.                                                             recombination: RuvC protein nicks DNA point strand exchange. Proc <br /> 13. Baba T, <span id='am-62' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-63' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-66'  ></span>Ara</span> T, Hasegawa M, Takai Y, Okumura Y, et al. (2006) Construction                          Natl Acad Sci USA 93: 12217&#8211;12222. <br />     Escherichia coli K-12 in-frame, single-gene knockout mutants: Keio                      41. Al-Deib <span id='am-188' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-189' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-191' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-192'  ></span><span id='am-193' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>AA</span>, Mahdi AA, Lloyd RG (1996) Modulation recombination and <br />     collection. Mol Syst Biol 2: 1&#8211;8.                                                                  DNA repair RecG PriA helicases Escherichia coli K-12. J Bacteriol <br /> 14. Datsenko KA, Wanner BL (2000) One-step inactivation chromosomal genes                           178: 6782&#8211;6789. <br />     Escherichia coli K-12 using PCR products. Proc Natl Acad Sci USA 97:                        42. Rojas DM, Va&#180;squez CC (2005) Sensitivity potassium tellurite Escherichia coli <br />     6640&#8211;6645.                                                                                         cells deficient CSD, CsdB IscS cysteine desulfurases. Res Microbiol 156: <br /> 15. Yamamoto N, Nakahigashi K, Nakamichi T, Yoshino M, Takai Y, et al. (2009)                          465&#8211;471. <br />     Update Keio collection Escherichia coli single-gene deletion mutants.                43. Ikeuchi Y, Shigi N, Kato J, Nishimura A, Suzuki T (2006) Mechanistic insights <br />     Mol Syst Biol 5: 335&#8211;337.                                                                          sulfur relay multiple sulfur mediators involved thiouridine biosynthesis <br /> 16. Wall D, Delaney JM, Fayet O, Lipinska B, Yamamoto T, et al. (1992) arc-                            tRNA wobble positions. Mol Cell 21: 97&#8211;108. <br />     dependent thermal regulation extragenic suppression Escherichia coli                44. Dahl C, Schulte A, Stockdreher Y, Hong C, Grimm F, et al. (2008) Structural <br />     cytochromed operon. J Bacteriol 174: 6554&#8211;6562.                                                    molecular genetic insight widespread sulfur oxidation pathway. J Mol <br /> 17. Raina S, Georgopoulos C (1991) The htrM gene, product essential                       Biol 384: 1287&#8211;1300. <br />     Escherichia coli viability elevated temperatures, identical rfaD gene.       45. Lauhon CT (2002) Requirement IscS biosynthesis thio-nucleosides in <br />     Nucleic Acids Res 19: 3811&#8211;3819.                                                                   Escherichia coli. J Bacteriol 184: 6820&#8211;6829. <br /> 18. Klein G, Lindner B, Brabetz W, Brade H, Raina S (2009) Escherichia coli K-12                   46. Mihara H, Esaki N (2002) Bacterial cysteine desulfurases: function and <br />     Suppressor-free Mutants Lacking Early Glycosyltransferases Late Acyl-                          mechanisms. Appl Microbiol Biotechnol 60: 12&#8211;30. <br />     transferases: minimal lipopolysaccharide structure induction envelope                   47. Frazzon J, Dean DR (2003) Formation iron-sulfur clusters bacteria: an <br />     stress response. J Biol Chem 284: 15369&#8211;15389.                                                     emerging field bioinorganic chemistry. Curr Opin Chem Biol 7: 166&#8211;173. <br /> 19. Adams DE, West SC (1996) Bypass DNA heterologies RuvAB-                              48. Taylor SV, Kelleher NL, Kinsland C, Chiu HJ, Costello CA, et al. (1998) <br />     mediated three- four-strand branch migration. J Mol Biol 263: 582&#8211;596.                         Thiamin biosynthesis Escherichia coli. Identification thiocarboxylate as <br /> 20. Tomoyasu T, Ogura T, Tatsuta T, Bukau B (1998) Levels DnaK DnaJ                             immediate sulfur donor thiazole formation. J Biol Chem 273: <br />     provide tight control heat shock gene expression protein repair                          16555&#8211;16560. <br />     Escherichia coli. Mol Microbiol 30: 567&#8211;581.                                                   49. Lauhon CT, Kambampati R (2000) The iscS gene Escherichia coli required <br /> 21. Lipinska B, Zylicz M, Georgopoulos C (1990) The HtrA (DegP) protein,                               biothynthesis 4-thiouridine, thiamine, NAD. J Biol Chem 280: <br />     essential Escherichia coli survival high temperatures, endopeptidase.                 26760&#8211;26769. <br />     J Bacteriol 172: 1791&#8211;1797.                                                                    50. Ben-Zvi AP, Goloubinoff P (2001) Review: mechanisms disaggregation and <br /> 22. Ohara M, Wu HC, Sankaran K, Rick PD (1999) Identification                                      refolding stable protein aggregates molecular chaperones. J Struct Biol 135: <br />     characterization new lipoprotein, NlpI, Escherichia coli K-12. J Bacteriol                 84&#8211;93. <br />     181: 4318&#8211;4325.                                                                                51. Valentin-Hansen P, Johansen J, Rasmussen AA (2007) Small RNAs controlling <br /> 23. Kabir MS, Yamashita D, Koyama S, Oshima T, Kurokawa K, et al. (2005) Cell                          outer membrane porins. Curr Opin Microbiol 10: 152&#8211;155. <br />     lysis directed sigmaE early stationary phase effect induction              52. Britton RA, Lupski JR (1997) Isolation characterization suppressors of <br />     rpoE gene global gene expression Escherichia coli. Microbiology 151: <br />                                                                                                        Escherichia coli dnaG mutations, dnaG2903 parB. Genetics 145: 867&#8211;875. <br />     2721&#8211;2735. <br />                                                                                                    53. Ikegami A, Nishiyama K, Matsuyama S, Tokuda H (2005) Disruption rpmJ <br /> 24. Noor R, Murata M, Yamada M (2009) Oxidative stress trigger growth <br />                                                                                                        encoding ribosomal protein L36 decreases expression secY upstream the <br />     phase-specific sigmaE-dependent cell lysis Escherichia coli. J Mol Microbiol <br />                                                                                                        spc operon inhibits protein translocation Escherichia coli. Biosci Biotechnol <br />     Biotechnol 17: 177&#8211;187. <br />                                                                                                        Biochem 69: 1595&#8211;1602. <br /> 25. Haydon DJ, Quail MA, Guest JR (1993) A mutation causing constitutive <br />                                                                                                    54. Perron K, Comte R, van Delden C (2005) DksA represses ribosomal gene <br />     synthesis pyruvate dehydrogenase complex Escherichia coli located <br />                                                                                                        transcription <span id='am-89' about='obo:NCBITaxon_286' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621'><span id='am-90' property="oboInOwl:hasDbXref" content="PMID:18048745" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="Peudomonas" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Pseudomonas" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasRelatedSynonym" content="Loefflerella" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="PMID:10939664" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="PMID:9103607" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="PMID:10758879" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="PMID:7727274" datatype="xsd:string"></span><span id='am-100'  ></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="Liquidomonas" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="PMID:15950132" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasRelatedSynonym" content="RNA similarity group I" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasDbXref" content="PMID:23918787" datatype="xsd:string"></span>Pseudomonas</span> aeruginosa interacting RNA polymerase on <br />     pdhR gene. FEBS Lett 336: 43&#8211;47. <br />                                                                                                        ribosomal promoters. Mol Microbiol 56: 1087&#8211;1102. <br /> 26. Lyngstadaas A, L&#248;bner-Olesen A, Boye E (1995) Characterization genes <br />                                                                                                    55. Kang PJ, Craig EA (1990) Identification characterization new Escherichia <br />     dam-containing operon Escherichia coli. Mol Gen Genet 247: 546&#8211;554. <br />                                                                                                        coli gene dosage-dependent suppressor dnaK deletion mutation. <br /> 27. Cassey B, Guest JR, Attwood MM (1988) Environmental control pyruvate <br />                                                                                                        J Bacteriol 172: 2055&#8211;2064. <br />     dehydrogenase complex expression Escherichia coli. FEMS Microbiol Lett <br />     1159: 325&#8211;329.                                                                                 56. Tehranchi AK, Blankschien MD, Zhang Y, Halliday JA, Srivatsan A, et al. <br />                                                                                                        (2010) The transcription factor DksA prevents conflicts DNA <br /> 28. Barak R, Abouhamad WN, Eisenbach M (1998) Both acetate kinase acetyl <br />                                                                                                        replication transcription machinery. Cell 141: 595&#8211;605. <br />     coenzyme A synthetase involved acetate-stimulated change the <br />     direction flagellar rotation Escherichia coli. J Bacteriol 180: 985&#8211;988.                 57. Watts T, Cazier D, Healey D, Buskirk A (2009) SmpB contributes reading <br /> 29. Mogi T, Mizuochi-Asai E, Endou S, Akimoto S, Nakamura H (2006) Role                           frame selection translation transfer-messenger RNA. J Mol Biol 391: <br />     putative subunit YhcB assembly function cytochrome bd-                         275&#8211;281. <br />     type ubiquinol oxidase Escherichia coli. Biochim Biophys Acta 1757:                       58. Bloor AE, Cranenburgh RM (2006) An efficient method selectable marker <br />     860&#8211;864.                                                                                           gene excision Xer recombination gene replacement bacterial <br /> 30. Reed KE, Cronan JE, Jr. (1993) Lipoic acid metabolism Escherichia coli:                         chromosomes. Appl Environ Microbiol 72: 2520&#8211;2525. <br />     sequencing functional characterization lipA lipB genes. J Bacteriol             59. Uehara T, Parzych KR, Dinh T, Bernhardt TG (2010) Daughter cell separation <br />     175: 1325&#8211;1336.                                                                                    controlled cytokinetic ring-activated cell wall hydrolysis. EMBO J 29: <br /> 31. Kneidinger B, Marolda C, Graninger M, Zamyatina A, McArthur F, et al.                              1412&#8211;1422. <br />     (2002) Biosynthesis pathway ADP-L-glycero-<span id='am-24' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29'  ></span>beta</span>-D-manno-heptose                           60. Gerding MA, Liu B, Bendezu&#180; FO, Hale CA, Bernhardt TG, et al. (2009) Self- <br />     Escherichia coli. J Bacteriol 184: 363&#8211;369.                                                        enhanced accumulation FtsN Division Sites Roles Other Proteins <br /> 32. Roncero C, Casadaban MJ (1992) Genetic analysis genes involved                           SPOR domain (DamX, DedD, RlpA) Escherichia coli cell <br />     synthesis lipopolysaccharide core Escherichia coli K-12: operons                constriction. J Bacteriol 191: 7383&#8211;7401. <br />     rfa locus. J Bacteriol 174: 3250&#8211;3260.                                                     61. Arends SJ, Williams K, Scott RJ, Rolong S, Popham DL, et al. (2010) Discovery <br /> 33. Gerding MA, Ogata Y, <span id='am-212' about='obo:NCBITaxon_35500' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-213' property="rdfs:label" content="Pecora" datatype="xsd:string"></span><span id='am-214' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-215'  ></span><span id='am-216' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Pecora</span> ND, Niki H, Boer PA (2007) The trans-                               characterization new Escherichia coli septal ring proteins contain <br />     envelope Tol-Pal complex cell division machinery required                   SPOR domain: DamX, DedD, RlpA. J Bacteriol 192: 242&#8211;255. <br />     proper outer-membrane invagination cell constriction E. coli. Mol                    62. Shigi N, Sakaguchi Y, Asai S, Suzuki T, Watanabe K (2008) Common thiolation <br />     Microbiol 63: 1008&#8211;1025.                                                                           mechanism biosynthesis tRNA thiouridine sulphur-containing <br /> 34. Kampfenkel K, Braun V (1993) Membrane topologies TolQ TolR                              cofactors. EMBO J 7: 3267&#8211;3278. <br />     proteins Escherichia coli: inactivation TolQ missense mutation               63. Rudolph B, Gebendorfer KM, Buchner J, Winter J (2010) Evolution of <br />     proposed transmembrane segment. J Bacteriol 75: 4485&#8211;4491.                                   Escherichia coli growth high temperatures. J Biol Chem 285: 19029&#8211;19034. <br /> 35. Niba ET, Naka Y, Nagase M, Mori H, Kitakawa M (2007) A genome-wide                             64. Al Refaii A, Alix JH (2009) Ribosome biogenesis temperature-dependent and <br />     approach identify genes involved biofilm formation E. coli. DNA Res                   delayed Escherichia coli lacking chaperones DnaK DnaJ. Mol Microbiol <br />     14: 237&#8211;246.                                                                                       71: 748&#8211;762. <br /> 36. Nikaido H (2003) Molecular basis bacterial outer membrane permeability                      65. Rene&#180; O, Alix JH (2010) Late steps ribosome assembly E. coli sensitive to <br />     revisited. Microbiol Mol Biol Rev 67: 593&#8211;656.                                                     severe heat stress assisted HSP70 chaperone machine. Nucleic <br /> 37. Motamedi MR, Szigety SK, Rosenberg SM (1999) Double-strand-break repair                            Acids Res 39: 1855&#8211;1867. <br />    <br /> </body></html>