<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Sept. 1992, p. 3843-3856                                                                   Vol. 12, No. 9 <br /> 0270-7306/92/093843-14$02.00/0 <br /> Copyright &#169; 1992, American Society Microbiology <br />  <br />  <br />                PTAJ, Essential Gene <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br />                           Affecting Pre-tRNA Processing <br />                                    J. PATRICK O&apos;CONNORt AND CRAIG L. PEEBLES* <br />                  Department Biological Sciences, University Pittsburgh, Pittsburgh, Pennsylvania 15260 <br />                     Received 19 December 1991/Returned modification 11 February 1992/Accepted 17 June 1992 <br />  <br />             We identified essential <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae gene, PTA], affects pre-tRNA processing. <br />           PTA] initially defined UV-induced mutation, ptal-1, causes accumulation 10 <br />           end-trimmed, intron-containing pre-tRNAs temperature-sensitive osmotic-remedial growth. ptal-) <br />           does appear allele known gene affecting pre-tRNA processing. Extracts prepared from <br />           ptal-) strains normal pre-tRNA splicing endonuclease activity. ptal-) suppressed ochre <br />           suppressor tRNA gene SUPII, indicating ptal-I mutation creates termination <span id='am-43' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-46'  ></span><span id='am-47' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> protein <br />           reading frame. The PTAI gene isolated genomic library complementation ptal-) growth <br />           defect. Episome-borne PTA) directs recombination ptal-) locus. PTAI mapped left arm <br />           chromosome I near CDC24; gene sequenced encode protein 785 amino acids a <br />           molecular weight 88,417. No protein sequences similar predicted PTAI gene product have <br />           identified EMBL GenBank <span id='am-82' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-84'  ></span><span id='am-85' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> base. Disnrption PTA) near carboxy terminus the <br />           putative open reading frame lethal. Possible functions PTA) gene product discussed. <br />  <br />    The pre-tRNA processing pathway consists steps                   endonuclease subunits (51). Three mutant alleles SEN2 <br /> catalyzed large number enzymes (15). A fairly                       identified. The sen2-1 allele causes temperature- <br /> complete picture pre-tRNA processing reactions Sac-                   sensitive (Ts) pre-tRNA splicing endonuclease activity but <br /> charomyces cerevisiae eukaryotes emerged                      pre-tRNA accumulation (62), sen2-3 allele <br /> from experiments crude extracts purified activities                exhibits aberrant pre-tRNA splicing endonuclease activity <br /> (9, 21, 35, 38, 48), microinjection Xenopus oocytes                 causes accumulation splicing intermediate <br /> cloned tRNA genes (14, 39, 41), analysis processing                  composed 5&apos; exon intron (2/3 molecule [25]). Two <br /> intermediates S. cerevisiae cells (33, 44). It clear            genes control activities required end maturation. <br /> that primary transcripts tRNA genes 5&apos;                       RPR1 encodes RNA component nuclear RNase P <br /> leader 3&apos; trailer sequences, lack -CCAOH tails                      (35), CCA1 encodes tRNA nucleotidyltransferase (1). <br /> nucleoside modifications, contain introns transcribed                The 10 remaining genes (RNA1, LOS), SEN1, SPLI, STPI, <br /> from genes intervening sequences. Processing                  TPD1 TPD5) represented mutant alleles that <br /> transcripts requires nucleolytic reactions                 exhibit various effects pre-tRNA processing. Mutant <br /> nucleoside modifications. Some processing reactions                  alleles RNAJ LOS1 lead accumulation end- <br /> occur dependent <span id='am-51' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span>; example, 5&apos; end trimming                 trimmed unspliced pre-tRNAs 10 tRNA gene <br /> RNase P normally occurs 3&apos; end trimming eukary-                   families intervening sequences (27, 28). A mutant allele <br /> otes (35, 44); likewise, -CCAoH tail addition occur              SENI displays reduced pre-tRNA splicing endonuclease <br /> 3&apos; end trimming (16). The steps splicing cleavage             activity accumulation end-trimmed unspliced <br /> 3&apos; 5&apos; splice sites, respectively (44, 48), exon ligation (21),          pre-tRNAs (62). Null alleles STPJ lead accumulation of <br /> and finally removal 2&apos; phosphate (38). However,                 end-trimmed unspliced pre-tRNAs subset the <br /> processing reactions occur dependent order,                     gene families (61). The TPD1 TPD5 genes defined by <br /> since splicing place end                    mutations tpdl-l, tpd2-1, tpd3-1, tpd4-1, tpd5-1; <br /> trimming (44).                                                              mutations identified modified loss-of-sup- <br />    Genetic approaches important complement di-                    pression assay (59). A mutant allele SPLI dominant <br /> rect biochemical dissection complex cellular functions. To               restoration-of-splicing phenotype, activating splicing the <br /> date, 18 S. cerevisiae genes affect pre-tRNA processing                mutant suppressor tRNA sup3-e A39 (34, 46). These 18 <br /> have identified variety assay procedures. Four                 genes grouped classes; known <br /> genes (TRM1, TRM2, MU), MOD5) encode nucleoside                         biochemical defects (CCAI, MM1, MOD5, RLG1, RPR1, <br /> modification activities (26). The order tRNA nucleoside                  SEN2, TRMI, TRM2) unknown modes of <br /> modifications does appear obligatory. However,                    action (LOS), RNAI, SEN1, SPLI, STPJ, TPDI to <br /> some nucleoside modifications important function                 TPD5). <span id='am-38' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-39' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> simple distinction suggests pre-tRNA <br /> certain tRNAs added unspliced pre-tRNAs (30,                    processing pathway substantially complex than <br /> 58). Two genes encode enzymatic activities previously im-                   current model emphasizes (12, 44). Direct screening of <br /> plicated pre-tRNA splicing. RLGI tRNA ligase gene                 mutagenized strains recovered multiple mutant alleles <br /> (49), SEN2 encodes pre-tRNA splicing                         just genes, LOS) SEN2 (25, 28, 62). This <br />                                                                             implies additional genes affecting pre-tRNA processing <br />                                                                             remain identified. <br />   * <br />       Corresponding author.                                                    We know pre-tRNA splicing essential cell <br />   t Present address: Howard Hughes Medical Institute, University            growth, known member 10 different S. cere- <br /> of Pennsylvania Medical School, Philadelphia, PA 19104.                     visiae tRNA gene families contains intervening sequences <br />                                                                      3843 <br />  3844          O&apos;CONNOR AND PEEBLES                                                                                       MOL. CELL. BIOL. <br />  <br />                                                       TABLE 1. S. cerevisiae haploid strains <br />      Strain                                                         Genotype                                                         Source <br /> A364A                          MA4Ta adel ade2 gall his7 lys2 tyrl ural                                                      YGSCa <br /> BJ3341                         MATa gal2 leu2-Al lys2-801 mal mel trpl-AIOI ura3-52 CUPI SUC2                                E. W. Jones <br /> BJ3342                         AL4Ta gal2 leu2-Al lys2-801 mal mel trpl-AJOl ura3-52 CUP1 SUC2                               E. W. Jones <br /> BJ4065                         AL4Ta his3-A200                                                                               E. W. Jones <br /> BLAY5-2c                       M4Ta leu2-Al lys2-801 trpl-AlOJ ura3-52 PTA1::LEU2 (YCpPTAI-HindIIIB)                         This study <br /> BUB-24c                        M4Ta FUN40::URA3 leu2-Al ura3-52                                                              This study <br /> CLP1-la                        MATa ade2-1 his3-A200 trpl-AlJO ptal-l                                                        This study <br /> DK365-28B                      AL4Ta ura3 cdc24                                                                              D. Kaback <br /> EElb                           MATa ade mal-l tyrl ura3-52                                                                   A. K. Hopper <br /> LA4-22                         UV-mutagenized isolate A364A                                                             This study <br /> LABJ3-S17c                     ALATa leu2-Al lys2 trpl-AlOJ tyrl ura3-52                                                     This study <br /> LABJ3-S17d                     ATcTa ade2-1 his7 leu2-AJ lys2 ptal-l trpl-A101 tyrl                                          This study <br /> POC1-13a                       MATa ade2-1 leu2-Al lys2ptal-l trpl-AIOI                                                      This study <br /> POC8-23c                       M4Ta ade2-1 leu2-Al lys2 trpl-AJOJ                                                            This study <br /> POC8-23d                       MATa ade2-1 leu2-Al lys2ptal-l trpl-AJOJ ura3-52                                              This study <br /> POC8-32c                       MATa leu2-Al lys2-801 ptal-l t7pl-AlOl ura3-52                                                This study <br /> POC8-32d                       MATa ade2-1 leu2-Al lys2 trpl-AJOJ ura3-52                                                    This study <br /> POC8-36a                       MATa ade2-1 leu2-Al lys2-801 ptal-l trpl-A101 ura3-52                                         This study <br /> POC9-15d                       AMTa gal2 leu2-Al mal mel trpl-AlOJ ura3-52 CUPJ SUC2                                         This study <br /> POC9-9c                        MATa gal2 leu2-Al lys2-801 mal mel ura3-52 CUPJ SUC2                                          This study <br /> POC18-ld                       ptal-J sen2-1                                                                                 This study <br /> SS2A-3C                        MA Ta ade2-101 his3-A2000 lys2-801 sen2-3 ura3-52 GAL SUC2                                    J. Abelson <br /> tsxl7-6b                       MATa leu2-3,112 pep4-2 sen2-1 ura3-52                                                         M. Culbertson <br /> tsx24-5b                       AL4Ta leu2-3,112 pep4-3 senl-l ura3-52                                                        M. Culbertson <br /> IIId-2c                        MATa ade2-1 canl-100 losi-A VSUP-4o ura3-1                                                    A. K. Hopper <br />  a <br />      YGSC, Yeast Genetic Stock Center, Berkeley, Calif. <br />  <br />  <br />  <br /> (44). Increased steady-state levels intron-containing pre-                             MATERIALS AND METHODS <br /> tRNAs, referred pre-tRNA accumulation, one <br /> phenotype characteristic mutations affecting pre-tRNA                     S. cerevisiae strains, media, cultivation, genetic meth- <br /> splicing pre-tRNA processing general. Accordingly,                  ods. S. cerevisiae strains listed Tables 1 2. YPD <br /> we chose look pre-tRNA accumulation strains                      contained 2% peptone, 2% glucose, 1% yeast extract. <br /> harboring conditional growth mutations, believing               YPDS YPD supplemented 1 M sorbitol. Synthetic <br /> two phenotypes identify additional pre-tRNA proc-                    complete (SC) medium prepared according recipe <br /> essing mutants defects essential genes. Here                    E. W. Jones, Carnegie-Mellon University (63). SCS was <br /> describe new mutation leading pre-tRNA accumulation,                  SC supplemented 1 M sorbitol; SCS-Leu and <br /> ptal-1. We analyzed pre-tRNA processing ptal-1                     SCS-Ura SCS leucine uracil, respectively. <br /> cells, isolated sequenced PTAJ structural gene,                5-Fluoro-orotic acid (5-<span id='am-1' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>FOA</span>) medium prepared de- <br /> shown PTAI essential gene.                                      scribed (7). Solid media contained 2% agar. Cul- <br />  <br />  <br />                                                       TABLE 2. S. cerevisiae diploid strains <br />  Strain                                                   Cross                                                  Relevant genotype <br /> BLAY5                                POC17::p2.3-BL14(YCpPTAI-HindIIIB)                              +/PTAI::LEU2 (YCpPTAI-HindIIIB) <br /> BUB                                  POC17::pCB2-U17                                                 +IFUN40::URA3 <br /> CLP6                                 CLPl-la x BUB-24c                                               FUN40::URA3/ptal-I <br /> CLP11b                               POC8-36a x BLAY5-2c                                             ptal-1/PTAJ::LEU2 <br /> LABJ3                                LA4-22 x BJ3341                                                 ptal-lI+ <br /> POCi                                 LABJ3-S17d x BJ3342                                             ptal-1/+ <br /> POC4                                 LABJ3-S17d x tsx24-5b                                           ptal-l/senl-l <br /> POC5                                 LABJ3-S17d x tsxl7-6b                                           ptal-l/sen2-1 <br /> POC6                                 LABJ3-S17d x LA4-22                                             ptal-l/ptal-l <br /> POC7                                 LABJ3-S17d x SS2A-3C                                            ptal-l/sen2-3 <br /> POC8                                 POCl-13a x BJ3341                                               ptal-l/+ <br /> POC1O                                LABJ3-S17c x BJ3342                                              +/+ <br /> POCll                                LABJ3-S17c x LA4-22                                              +Iptal-l <br /> POC14                                LABJ3-Sl7c x tsx24-5b                                            +Isenl-l <br /> POC16                                LABJ3-S17c x SS2A-3C                                             +/sen2-3 <br /> POC17                                POC9-15d x POC9-9c                                              leu2-AJ/leu2-Al ura3-52/ura3-52 <br /> POC31                                POC8-32c x POC8-23d (YCpPTA1)                                   ptal-l/ptal-l (YCpPTA1) <br /> POC34                                POC8-32c x POC8-23d::YIpPTA1(SnaBI)                             ptal-1/ptal-1::PTA1 <br /> POC40                                BJ4065 x POC8-23d::YIpPTA1(SnaBI)                                +/ptal-1:YPTAl <br />  VOL. 12, 1992                                                                                       S. CEREVISIE PTAI GENE        3845 <br />  <br />                                                        TABLE 3. <span id='am-92' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-93' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Plasmids</span> <br />           Plasmid                                 S. cerevisiae gene(s) origin replication                    Source (reference) <br /> YIp5                                URA3                                                                             J. Woolford <br /> YCp5O                               URA3 CEN4 ARS1                                                                   J. Woolford <br /> YEp24                               URA3 2pm                                                                         J. Woolford <br /> pSEYC58                             URA3 CEN4ARSI                                                                    E. Phizicky <br /> pUN50                               URA3 CEN4 ARS1                                                                   J. Woolford (20) <br /> pUN60                               URA3 CEN4ARSI SUPJJ                                                              J. Woolford (20) <br /> YEpRNAI                             RNAI YEp24                                                                    A. Hopper (2) <br /> YCpLOSI                             LOSI YCp5O                                                                    A. Hopper (29) <br /> pYEP11                              RLGI pSEYC58                                                                  E. Phizicky (49) <br /> pYEP99                              RLG1 YEp24                                                                    E. Phizicky (49) <br /> pC6                                 STPI YEp24                                                                    A. Hopper (61) <br /> YCpPTAI                             PTAI YCp5O                                                                    This study <br /> YCpPTAl-HindIIIB                    5.3-kbp HindIII fragment YCpPTAI YCp5O                                     This study <br /> YCpPTAI-HindIII-SaII                2.5-kbp HindIII-SalI fragment YCpPTAI YCp5O                                This study <br /> YCpPTAI-HindIII-ClaI                3.8-kbp HindIII-ClaI fragment YCpPTAI YCp5O                                This study <br /> YCpPTAI-BamHI-HindIII               4.2-kbp BamHI-HindIII fragment YCpPTAI YCp5O                               This study <br /> YCpPTAI-HindIII-BamHI               1.1-kbp HindIII-BamHI fragment YCpPTAI YCp5O                               This study <br /> YIp5-XhoIA                          9.5-kbp XhoI fragment YCpPTAI YIp5                                         This study <br /> YIpPTAI                             5.3-kbp HindIll fragment YCpPTAI YIp5                                      This study <br /> YEp13-G                             4.8-kbp EcoRI fragment spanning FUN40 YEp13                                   J. Pringle (19) <br /> pUN-PTAI                            4.0-kbp SacI-SalI fragment YEp13-G pUN50                                   This study <br /> pUN-BamHI                           1.7-kbp fragment YEp13-G pUN50                                             This study <br /> pCB2                                4.0-kbp Sacl-Sall fragment YEp13-G pUC119                                  This study <br /> pCB2-U17                            1.2-kbp KprnI-BamHI fragment pBLUE-URA3 pCB2                               This study <br /> pBLUE-URA3                          1.2-kbp HindIII fragment containing URA3 pBluescribe                          J. Pringle <br /> p2.3                                2.5-kbp HindIII-EcoRI fragment YCpPTAI pUC119                              This study <br /> p2.3-BL14                           3.1-kbp BgAII fragment YEpl3 containing LEU2 p2.3                          This study <br />  <br />  <br />  <br /> tivation, plating, tetrad analysis performed              C. Guthrie [45]). Pre-tRNA, tRNA, oligonucleotide <br /> described previously (44, 56). Diploid strains identified        probe nomenclature described previously (44). <br /> by selection prototrophs, microdissection zygotes,             DNA sequencing data analysis. The chain termination <br /> screening sporulation ability. Sporulation induced            method sequencing used Sequenase (U.S. Biochemical, <br /> with PSP medium (1% potassium acetate, 0.8% nutrient                  Cleveland, Ohio) according manufacturer&apos;s instruc- <br /> broth, 1% yeast extract, 2% agar).                                    tions. Templates prepared plasmids single- <br />    Transformation, nucleic acid isolation, manipulation.          stranded DNA libraries DNA fragments by <br /> S. cerevisiae cultures transformed plasmid DNA              construction specific subclones convenient restric- <br /> for complementation analysis linear DNA gene             tion endonuclease sites (4). Sequencing primers pur- <br /> disruption targeted integration (54), using lithium            chased commercial sources (M13 universal reverse <br /> acetate method (56) followed selection SCS-Ura               primers; New England Biolabs) prepared Univer- <br /> SCS-Leu. Plasmid DNA isolated transformed S.                 sity Pittsburgh DNA Synthesis Facility. Sequence data <br /> cerevisiae strains described previously (56). <span id='am-54' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-58'  ></span><span id='am-59' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span>Escherichia</span>          compiled checked assistance the <br /> coli DH5a JM101 transformed protocol               program DNA Strider (37). Protein sequence comparisons, <br /> Hanahan (23). Plasmid DNA isolated transformed               pattern searching, structure prediction, figure prepara- <br /> E. coli strains modified alkaline lysis procedure (4)        tion relied GCG package, version 7.0, implemented <br /> further purified CL-4B gel filtration chromatography.              Pittsburgh Supercomputing Center (17). <br /> Plasmids listed Table 3. Restriction endonucleases                Identification pre-tRNA-accumulating strains. S. cerevi- <br /> were obtained various manufacturers used                  siae A364A mutagenized UV light 2 5% <br /> recommended New England Biolabs (Beverly, Mass.).                  survival plated 23&#176;C YPD. Individual Ts isolates <br /> Other DNA manipulations described else-                  identified lack growth 37&#176;C. Each Ts strain was <br /> where (4).                                                            grown 12 16 h 23&#176;C YPD shifted 37&#176;C <br />    S. cerevisiae total RNA isolated described previ-           30 60 min. Cells collected 1-ml cultures by <br /> ously (44). Northern (RNA) blots separating              centrifugation, supernatant carefully removed. <br /> 10 p,g total RNA lane partially denaturing 7%               The cell pellet placed ice suspended 200 ,l of <br /> polyacrylamide gels followed electrophoretic transfer           1% sodium dodecyl sulfate-1% 2-mercaptoethanol-3.7% <br /> GeneScreen described previously (44). Northern blots               formaldehyde-50 mM EDTA. The suspension incubated <br /> were prehybridized, hybridized, stripped described             65&#176;C 15 min placed ice addition 50 ,ul <br /> previously (44). Northern blots hybridized               0.5 M KCl. The mixture vortexed, held ice 10 <br /> oligonucleotide complementary S. cerevisiae pre-                min, centrifuged 5 min 12,000 x g. Samples <br /> tRNA intron (for pILE, pSERcga, pSERgcu, pTRP                     containing 100 ,ul supernatant applied Gene- <br /> probe sequences; reference 44), stripped,                Screen, using Schleicher &amp; Schuell Mini-Fold apparatus. <br /> hybridized oligonucleotide complementary               The RNA covalently cross-linked GeneScreen <br /> 214- 179-base US RNA (snR7) <span id='am-48' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-49' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> (probe 5E7;                   exposure UV light (10) hybridized 32P-labeled <br /> 5&apos;-CCT CCG CCA TTG ATC TGT A-3&apos;; gift D. Brow                  oligonucleotide probes complementary pre-tRNA introns <br />  3846    O&apos;CONNOR AND PEEBLES <br />  <br /> as described previously (44). After autoradiography, pre- <br /> tRNA accumulation compared wild-type <br /> and rnal-1 strains (42, 43). <br />    Pre-tRNA splicing endonuclease assays. S. cerevisiae pre- <br /> tRNA splicing endonuclease fractions equivalent fraction <br /> II Peebles et al. prepared essentially described <br /> previously (48). Cultures grown overnight 2 liters of <br /> YPDS 30&#176;C. Cells harvested centrifugation, <br /> washed, broken single pass French press <br /> at 20,000 lb/in2. The 2,000 x g supernatant centrifuged <br />                                                                    Shtamin <br />                                                                    Grown   3 <br />                                                                    Shifted t 3o!ew -. . . . . - . . . <br />                                                                                 &apos;3 <br />  <br />  <br />                                                                              _E&quot;V <br />                                                                                        A37 <br />  <br />                                                                                     _A_OK <br />                                                                               _ Ol-_ __ <br />                                                                                          A364A <br />                                                                                              3t7 <br />                                                                                                33 <br />                                                                                                3 <br />                                                                                                                7_ <br />                                                                                                                  L A4-2 2 <br />  <br />  <br />  <br />  <br />                                                                                                               *_qw   .A*, <br />                                                                                                                          MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                                                                                                              ..    ..... <br />  <br />  <br />  <br />  <br /> at 250,000 x g 2 h 4&#176;C. The membrane pellet was <br /> suspended final protein concentration 10 mg/ml.                                        Pre-tRNA Ile-UAU Intron <br /> Pre-tRNA splicing endonuclease assays de-                   B <br /> scribed previously (48), using 1 pmol pre-tRNA PheGAA                                   AiIHIML. <br /> (52) 20 ,ug pre-tRNA splicing endonuclease fraction in <br /> 10-pl reactions. For pre-tRNA splicing endonuclease ther- <br /> mal inactivation, nuclear extracts prepared as- <br /> sayed described previously (62) pre-tRNA <br /> PheGAA used substrate. <br />    Nucleotide sequence accession number. The PTAI se- <br /> quence submitted GenBank (6) available <br /> under accession number M95673.                                                                Pre-tRNA SER-GCU Intron <br />  <br />  <br />                            RESULTS <br />                                                                                                       -inUwm&quot;. ImaI               u . t. i <br />    Identification characterization pre-tRNA-accumu- <br /> lating strain. Ts strains UV-mutagenized A364A cells <br /> were assayed pre-tRNA accumulation. RNA <span id='am-77' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-78'  ></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Iso" datatype="xsd:string"></span>iso</span>- <br /> lated Ts strain analyzed filter hybridization <br /> with 32P-labeled oligonucleotides complementary pre- <br /> tRNA introns. From approximately 500 strains, one <br /> designated LA4-22 displayed particularly strong pre-tRNA                                         U5 (SNR7) RNA 5&apos;End <br /> accumulation (data shown) chosen           FIG. 1. Accumulation unspliced pre-tRNAs inptal-1. North- <br /> study. As demonstrate, Ts growth pre-tRNA            ern blot analysis total RNA A364A (wild-type) LA4-22 <br /> accumulation LA4-22 phenotypes ptal-1,          (ptal-1) cells grown different temperatures YPD. Cultures were <br /> single recessive mutation.                                       grown indicated temperatures. Some cultures tempera- <br />    We determined LA4-22 accumulated just            ture shifted 30 min indicated temperature prior to <br /> one, some, 10 intron-containing pre-tRNAs. If LA4-22      harvesting. Lanes: 1 6, A364A total RNA; 7 11, LA4-22 total <br /> accumulated single pre-tRNA, accumulation         RNA. (A) pILE probe; (B) pSERgcu probe; (C) 5E7, snR7 (U5 <br /> attributed tRNA gene mutation. If ptal-1 mutation       snRNA)-specific probe. Arrows panels A B indicate end- <br />                                                                  mature pre-tRNAs. <br /> present LA4-22 acts trans affect pre-tRNA process- <br /> ing, pre-tRNAs accumulate. Total RNA <br /> was isolated LA4-22 A364A cells cultured at <br /> various temperatures. No change overall pattern        LA4-22. Twelve independent Ts+ revertants selected <br /> RNA evident ethidium bromide staining sepa-         LA4-22 tested pre-tRNA accumulation by <br /> ration polyacrylamide agarose gels (data shown).       Northern analysis, using different pre-tRNA intron <br /> The RNA preparations subsequently assayed pre-          probes. The pre-tRNA accumulation phenotype lost <br /> tRNA accumulation Northern analysis (Fig. 1). Substan-        12 Ts+ revertants pre-tRNA tested (43). <br /> tial accumulation end-trimmed pre-tRNAs IleUAU            These results indicate mutation controls both <br /> SerGCU clearly evident LA4-22 cells inde-         pre-tRNA accumulation Ts growth. <br /> pendent growth temperature. The amounts pre-tRNAs             All ptal-JIPTAI heterozygotes grew 37&#176;C complete <br /> IleUAU (Fig. 1A) SerGCU (Fig. 1B) consistently          media (data shown); heterozygotes tested by <br /> much lower A364A cells grown condi-            Northern analysis failed accumulate pre-tRNA (Fig. 2, <br /> tions. Similarly, intron-containing       lanes 1, 6, 11; however, theptal-1lsen2-3 heterozygote <br /> pre-tRNAs accumulated LA4-22 cells compared              exception [see below]). The small pre-tRNA <br /> A364A cells (43). The amounts U5 snRNA (Fig. 1C)          accumulation seen diploid strains (lanes 1, 6, 11) <br /> end-extended pre-tRNAs IleUAU SerGCU ap-                appears general property diploid strains not <br /> proximately A364A LA4-22. We           indication ptal-1 semidominance (43). In contrast, a <br /> examined pattern pre-tRNAs accumulated              ptal-l/ptal-I homozygous diploid Ts accumulates <br /> outcrossed strain harboringptal-I (POC8-23d) metabolic        pre-tRNAs (for example, Fig. 7, lane 3). These observa- <br /> labeling 32Pi (43). The ptal-1 strain          tions RNA processing Ts <br /> accumulate labeled pre-tRNAs did mal-1            phenotypes ptal-1 recessive. <br /> culture shifted 37&#176;C (27, 32). We conclude PTA1         The ptal-I mutation LA4-22 outcrossed three <br /> gene product acts trans affect processing       times succession strain BJ3341 BJ3342 yield <br /> intron-containing pre-tRNAs.                                     diploids LABJ3, POC1, POC8 (Tables 1 2). Since <br />    A single recessive mutation controls phenotypes       BJ3341 BJ3342 wild type accumulate <br />  VOL. 12, 1992                                                                                                                S. CEREVISIE PTA1 GENE   3847 <br />  <br />                           LANE                1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 <br />                            Strain             I    1A   IB   IC   1D   2     2A   2B     2C    3 <br />                                                                                                2D   3 l 3A   3B   3C   13D WTIIWT21pta1WlVT <br />                            Growvn          37   30   30   30   30   37    30        30 30   30            30   30    30   3   30   3   30 <br />                            Accumulation       - l _I 1+ _ l+ _I 1+ -_ T _ _ [-1                                              -        + _ <br />                            Growth at37&#176;C      + 1_-I+ + + 1_ l+ 1+ 1_ + 1_ 1+ l+ -                                           -+ I+ <br />  <br />  <br />  <br />  <br />                                                                             Pre-tRNA TRP-CCA Intron <br />  <br />                                           B <br />  <br />  <br />  <br />  <br />                                                                     U5 (SNR7) RNA 5&apos;End <br />   FIG. 2. Tetrad analysis ptal-1 pre-tRNA accumulation phenotype. Total RNA spore clones tetrad three <br /> successive outcrosses LA4-22, parentalptal-1 heterozygotes, wild-type strains used outcrosses, LA4-22, A364A <br /> (the strain LA4-22 derived) assayed pre-tRNA accumulation Northern analysis. Each lane partially <br /> denaturing 7% polyacrylamide gel loaded 2 ,ug total RNA. The gel run 1.5 h 200 V. The blot hybridized the <br /> pTRP probe (A), stripped, hybridized 5E7 probe (specific snR7) (B). Lanes: 1, LABJ3 (+/ptal-l); 2 5, a, b, c, <br /> and d spore clones, respectively, tetrad LABJ3-S17; 6, POC1 (+Iptal-l); 7 10, a, b, c, d spore clones, respectively, tetrad <br /> POC1-13; 11, POC8 (+/ptal-l); 12 15, a, b, c, d spore clones, respectively, tetrad POC8-23; 16, BJ3341 (wild type [WT1]); 17, <br /> BJ3342 (wild type [WT2]); 18, LA4-22 (ptal-1); 19, A364A (wild type [WT]; LA4-22 parental strain). The Ts growth pre-tRNA <br /> accumulation phenotypes strain indicated. The temperature strain cultured prior harvesting RNA <br /> extraction indicated. <br />  <br />  <br /> pre-tRNAs (43), pre-tRNA accumulation displayed                             suppressed conditions (43). The ptal-1 growth <br /> the meiotic products diploids attributed                      phenotype best described temperature sensitive, <br /> the mutation(s) originally present LA4-22. Several asci                         osmotic remedial (Tsor [55]). <br /> from diploids cross sen2-1 strain                              ptal-) defines gene affecting pre-tRNA processing. Com- <br /> (POCS) dissected YPDS. The spore clones                               plementation analysis performed determine whether <br /> tested Ts genetic markers. In 34 36                           ptal-1 represented allele known pre-tRNA proc- <br /> tetrads examined, Ts segregated 2:2 pattern yield                        essing mutation. An outcrossedptal-1 strain (LABJ3-S17d) <br /> total 70 Ts- 74 Ts&apos; spore clones. In                                wild-type sibling (LABJ3-S17c) mated individ- <br /> exceptional tetrads, spores Ts&apos; Ts-.                        ual strains harboring pre-tRNA processing mutations <br /> We suspect extra Ts+ spore clones arose                          (mal-1, losl-AV, senl-1, sen2-1, sen2-3). Each diploid <br /> either gene conversion second-site suppression ptal-1                        assayed growth 37&#176;C YPD pre-tRNA <br /> (see below). All genetic markers segregated normally.                        accumulation Northern analysis. In instance, the <br /> Spore clones tetrads sen2-1 cross (POC5),                        heterozygotes grew 37&#176;C. The heterozygotes failed to <br /> three tetrads second outcrosses,                         accumulate pre-tRNA (representative data shown Fig. <br /> tetrads outcross ptal-1 assayed                           3 summarized Table 4; reference 43). Since <br /> pre-tRNA accumulation (Fig. 2). In 18 tetrads, pre-                          sen2-3 bona fide pre-tRNA splicing endonuclease mu- <br /> tRNA accumulation segregated 2:2 cosegregated                           tant, concerned accumulation 2/3 mole- <br /> with Ts growth, demonstrating ptal-1 single,                             cules ptal-l/sen2-3 strain (Fig. 3B, lane 7) might <br /> recessive, nuclear mutation controlling phenotypes. In                        indicate functional interaction Ptalp Sen2p (the <br /> the course genetic analysis, ptal-1                          protein products PTAI SEN2 genes, respective- <br /> strains grow better grow 37&#176;C media supple-                          ly). However, +Isen2-3 heterozygote accumulated <br /> mented 1 M sorbitol osmotic supplements.                             equivalent pre-tRNA SerCGA 2/3 molecules <br /> However, pre-tRNA accumulation phenotype                               (Fig. 3B, lane 9), demonstrating sen2-3 allele is <br />  3848 <br />  <br />  <br />  <br />         r.4 <br />             O&apos;CONNOR AND PEEBLES <br />  <br />  <br />  <br />  <br />        Str&apos;ain1 <br />        C-row n at <br />        Sliifted to <br />  <br />  <br />  <br />  <br />                      t,.&apos;:M&lt;.ie?EA;&apos;Sa.M,=&quot;.;?g&apos;j*..;e -_:&apos;4.,.;st <br />                      123 123 T23 130 <br />  <br />  <br />  <br />  <br />                         v&#167; <br />                              1 <br />  <br />  <br />  <br />  <br />                        : ::.;,::,c::. ,zSr <br />  <br />  <br />                       &apos;:. <br />                        &apos;.?:.: :&apos; <br />  <br />  <br />  <br />  <br />                         &apos;: <br />                              .: <br />                                    :. <br />                                         2 <br />  <br />  <br />  <br />  <br />                                         30 <br />  <br />  <br />  <br />  <br />                                              ;r &gt;&apos;S&apos;Ss&apos; Ss- <br />  <br />  <br />  <br />                                                 :. <br />  <br />  <br />  <br />  <br />                                   U5 (SNR7) RNA 5END <br />                                                      3 <br />  <br />  <br />  <br />  <br />                                                      37 <br />  <br />  <br />  <br />  <br />                                                      . <br />  <br />  <br />  <br />  <br /> interaction Ptalp Sen2p. Several different <br />                                                          _~~~PetN <br />  <br />  <br />  <br />  <br />                                                          &apos;, <br />  <br />  <br />  <br />  <br /> semidominant evidence functional <br />                                                               :: <br />                                                                     4 <br />  <br />  <br />  <br />  <br />                                                                    :.-!r: <br />                                                                                30 <br />                                                                                37 <br />  <br />  <br />  <br />  <br />                                                                             &apos;??, <br />                                                                                    5 <br />  <br />  <br />  <br />  <br />                                                                                    S=-G <br />  <br />  <br />  <br />  <br />                                                                                    :. <br />  <br />                                                                                    .: <br />                                                                                         A   6       7 <br />  <br />  <br />  <br />                                                                                                 137 130} <br />  <br />  <br />  <br />  <br />                                                                                                 &quot;&apos; &apos;, <br />                                                                                                             8 <br />  <br />  <br />  <br />  <br />                                                                                                    .................. <br />  <br />  <br />  <br />  <br />                                                                                                                 :&apos;&apos;   &apos;.?&apos; <br />  <br />  <br />  <br />  <br />                                                                                                     U5 (SN171 SHNA 5iEnd <br />                                                                                                                              9 <br />  <br />  <br />  <br />                                                                                                                              37 <br />  <br />  <br />  <br />  <br />                                                                                                                              :: <br />  <br />  <br />  <br />  <br />    FIG. 3. Complementation ptal-1 pre-tRNA accumulation phenotype. Northern analysis performed total RNA extracted <br />                                                                                                                                   10 <br />  <br />  <br />  <br />                                                                                                                                   30aU <br />  <br />  <br />  <br />  <br /> from tsx24-5b (senl-l) aptal-1 heterozygote tsx24-5b (POC4) (A) total RNA extracted SS2A-3C (sen2-3) a <br /> ptal-1 heterozygote SS2A-3C (POC7) (B). The blots hybridized pSERcga probe (top panels), stripped, then <br />                                                                                                                                     :: ::&apos; <br />  <br />  <br />  <br />  <br /> hybridized 5E7 probe ensure intact equivalent amounts RNA present lane (bottom panels). (A) Lanes: <br />  1 5, tsx24-5b; 6, blank; 7, POC4 (senl-llptal-1); 8, LABJ3-S17d (ptal-1); 9, POC14 (+Isenl-1); 10, LABJ3-S17c (wild type [WT]). (B) <br /> Lanes: 1 6, SS2A-3C; 7, POC7 (sen2-3Iptal-1); 8, LABJ3-S17d (ptal-1); 9, POC16 (+/sen2-3); 10, LABJ3-S17c (wild type [WT]). Cultures <br />                                                                                                                                        :. <br />  <br />  <br />  <br />  <br /> were grown indicated temperatures. Some cultures temperature shifted 30 min indicated temperature prior harvesting. <br />                                                                                                                                              LANE <br />                                                                                                                                              Str.ailn <br />                                                                                                                                              Crow.n1 <br />                                                                                                                                              Shifted <br />                                                                                                                                                         at <br />                                                                                                                                                         to <br />  <br />                                                                                                                                                              .*_lX.je.a Sm;_.. ;* :..;G:J <br />                                                                                                                                                              1 <br />                                                                                                                                                              r~ <br />                                                                                                                                                               ~ <br />                                                                                                                                                              23 <br />                                                                                                                                                                ~~~~ stsr&gt;-&gt; <br />  <br />  <br />  <br />  <br />                                                                                                                                                               ,* <br />                                                                                                                                                                 23 <br />                                                                                                                                                              E 30 <br />  <br />  <br />                                                                                                                                                              f&apos; _:;. <br />  <br />  <br />  <br />  <br />                                                                                                                                                              2.041. <br />                                                                                                                                                                       2   3 <br />  <br />                                                                                                                                                                            3 <br />                                                                                                                                                                           37 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                  _ <br />  <br />  <br />  <br />  <br />                                                                                                                                                                           ......... <br />                                                                                                                                                                                       4 <br />  <br />                                                                                                                                                                                       30 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                Pre-tRNA <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                        : <br />                                                                                                                                                                                             3I <br />                                                                                                                                                                                            1371 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                            .! <br />                                                                                                                                                                                                 5 <br />  <br />                                                                                                                                                                                                 30 <br />                                                                                                                                                                                                      B6 <br />                                                                                                                                                                                                        37 <br />  <br />  <br />  <br />  <br />                                                                                                                                              pre-tRNA allele SEN2. We prepared extracts <br />                                                                                                                                              ptal-1, senl-1, sen2-1, wild-type strains as- <br />                                                                                                                                                                                                                  7 <br />                                                                                                                                                                                                               J.2t-f <br />                                                                                                                                                                                                                 37 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  SEtt-CCG.A lilti-oni <br />                                                                                                                                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br />                                                                                                                                                                                                                             8           9 <br />  <br />                                                                                                                                                                                                                                            FN <br />                                                                                                                                                                                                                                                   10 <br />                                                                                                                                                                                                             l~~~~~~&gt;,Itl+}lt;&apos;7s-l)&apos;l&apos;&apos;^$ixjZl-&apos;&apos;&apos;w&apos;-&apos;]\;r <br />                                                                                                                                                                                                                                           &apos;i <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                         &apos;+-- -     30 <br />                                                                                                                                                                                                                                                      V &quot; <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                         . <br />  <br />  <br />  <br />  <br /> intron-specific pre-tRNA hybridization probes used,                                                                                     sayed pre-tRNA splicing endonuclease activity. The <br /> and gave similar results (data shown). Since pre-                                                                                    levels activity ptal-1 wild-type strains were <br /> tRNA accumulation result deficiency pre-                                                                                       identical (Fig. 5A). In contrast, extract prepared a <br /> tRNA splicing endonuclease, showed senl-l                                                                                    senl-l strain reduced pre-tRNA splicing endonuclease <br /> sen2-1 mutations assort independently ptal-1, establish-                                                                                  activity, expected (62). We examined thermal <br /> ing ptal-1 represents gene distinct SENI                                                                                  inactivation kinetics pre-tRNA splicing endonuclease <br /> or SEN2 (43, 49b).                                                                                                                           activity prepared fromptal-1 strains rate <br />    Complementation performed episomal cop-                                                                                     inactivation indistinguishable wild-type rate <br /> ies genes involved pre-tRNA processing. POC8-32c                                                                                       (Fig. SB), extracts prepared sen2-1 strains <br /> (relevant genotype, ptal-1 ura3-52) transformed                                                                                     heat sensitive, consistent described <br /> either RNA1, LOS1, RLG1, STPJ plasmid (2, 29, 49,                                                                                      phenotype mutation (62). We conclude theptal-1 <br /> 61). In case, transformed strains remained Tsor                                                                                 mutation does directly affect pre-tRNA splicing endonu- <br /> accumulated pre-tRNAs (Fig. 4). Complementation analysis                                                                                     clease activity assayed vitro PTAJ probably <br /> of tpdl tpdS strains ptal-1 reported van Zyl                                                                                  does encode pre-tRNA splicing endonuclease <br /> et al. (59), andptal-1 allele TPDJ                                                                               subunits. <br /> to TPDS genes. SPLI RPRI mapped                                                                                                Nature ptal-l lesion. Genetic analysis ptal-1 <br /> chromosomes III V, respectively (34, 35), PTAJ                                                                                     mutation hampered high reversion frequency. <br /> maps chromosome I (see below). Therefore, PTA1, SPL1,                                                                                     Approximately 2% progeny derived ptal-1 single <br /> and RPRI distinct genes. Comparison restriction maps                                                                                  colony revertants strain grown YPD at <br /> and sequences (see Fig. 8 PTA1) shows PTAJ                                                                                       30&#176;C 10 generations. The frequent reversion ptal-1 <br /> different CCA1, gene tRNA nucleotidyltrans-                                                                                     suggested second-site mutations suppress ptal-1. Most <br /> ferase (1). These results (summarized Table 4) clearly                                                                                    ptal-1 strains carry ade2-1, derived parental <br /> indicate ptal-I defines PTA1, previously undescribed                                                                                  A364A strain. The ade2-1 allele ochre mutation, and <br /> gene affecting pre-tRNA processing.                                                                                                          ade2-1 colonies appear red. Almost Ts&apos; revertant <br />    Pre-tRNA splicing endonuclease activity normal ptal-l                                                                               colonies white Ade+ (data shown). Simulta- <br /> extracts. Pre-tRNA splicing endonuclease purified                                                                                   neous suppression ade2-1 indicates second-site <br /> as complex distinct protein subunits (51). The                                                                                     suppressors ptal-1 ochre suppressor tRNAs. To show <br /> SEN2 gene encodes subunit pre-tRNA splicing                                                                                        ptal-1 lesion ochre mutation, POC8-23d <br /> endonuclease (25), genes remaining                                                                                        (relevant genotype, ade2-1 ptal-1 ura3-52) transformed <br /> subunits identified. We wanted test                                                                                 pUN50 pUN60, low-copy-number plas- <br /> PTAI encoded pre-tRNA splicing endonuclease                                                                                        mids (20). These plasmids identical pUN60 <br /> subunits, ptal-I results accumulation unspliced                                                                                   carries SUPJ1, tyrosine-inserting ochre suppressor tRNA <br />  VOL. 12, 1992                                                                                             S. CEREVISIAE PTA1 GENE                   3849 <br />  <br />                                                  TABLE 4. ptal-1 complementation analysis <br />                                                    Classic genetic complementation                                   Episomal comple- <br />                                                                                                                      mentation    (trans- <br />                                                                                                                        formed a <br />  Gene    Chromo-                      Mutant phenotype                               ptal-I diploid phenotype          ptal-l   strain)      Is gene <br />                                                                                                                                        essential? <br />                                      Growth  Splicing          Pre-tRNA      Growth       Pre-tRNA                  Growth      Pre-tRNA <br />                        Mutation      defects endonu- <br />                                        defects <br />                                                                 accumu-      defects       accumu-     Recombines   defects <br />                                                                                                                     defects <br />                                                                                                                                  accumu- <br />                                               clease             lation                     lation                                lation <br />  <br />                      W1l             None      WT            No              None          No <br /> RNAI       XIII      rnal-1          Ts        WT            Yes             None          No                       Tsor           Yes          Yes <br /> LOSI       XI        losl-A          None      WT            Yes             None          No                       Tsor           Yes          No <br /> RLGI                                                                                                                Tsor           Yes          Yes <br /> SENI       XII       senl-l          Ts        Reduced       Yes             None          No              Yes                                  Yes <br /> SEN2                 sen2-1          None      Ts             No             None           No             Yes                                  ? <br />                      sen2-3          Noneb     Abnormal       Yes            None           Slight <br /> STPI       IV        stpl::URA3      None                     Yesc                                                  Tsor           Yes          No <br /> TPD1                 tpdl_ld         Ts        Wr            Yes             None                                                               ? <br /> TPD2                 tpd2-1          Ts        WT                            None                                                               ? <br /> TPD3                 tpd3-1          Ts        WT                            None                                                               ? <br /> TPD4                 tpd4-1          Ts        WT                            None                                                               ? <br /> TPD5                 tpd5-1          Ts        WT             Yes            None                                                               ? <br /> CCAlf                ccal-I          Ts                       Abnormalg                                                                         ? <br /> RPR1       V                                                                                                                                    Yes <br /> SPL1       III                                                                                                                                  ? <br /> PTAI       I         ptal-1          Tsor       WT            Yes            Tsor           Yes                     None           No           Yes <br />  a <br />  b <br />    WT, wild type. <br />    sen2-3 causes cold-sensitive spore germination defect. <br />  c The STPI disruption causes accumulation pre-tRNA families. <br />  d ptal-1 complementation analysis tpd strains performed van Zyl et al. (59). <br />    Pre-tRNA splicing endonuclease activity tpd strains assayed Pippert (49a). <br />  f The PTAI CCAI genes different sequences, loci different restriction endonuclease maps. <br />  g The ccal-I mutation causes accumulation tRNAs -CCAOH tail aberrant mRNA synthesis nonpermissive temperature. <br />  <br />  <br />  <br /> gene. Both plasmids complement ura3-52 (Fig. 6C). The                       quency. Restriction endonuclease analysis demonstrated <br /> ade2-1 mutation suppressed pUN60                          independently isolated plasmids identical; <br /> pUN50 (data shown). Only pUN60 permitted POC8-23d                       plasmid designated YCpPTAJ (see Fig. 8). <br /> to grow 37&#176;C YPD SC-Ura (Fig. 6). Identical                 Growth ptal-1 strain transformed YCpPTAI is <br /> results obtained secondptal-1 strain, POC8-32c                  illustrated Fig. 6. YCpPTAJ complements pre- <br /> (data shown). POC8-23d strains lost pUN60                      tRNA accumulation phenotype ofptal-1 (Fig. 7). Total RNA <br /> were selected resistance 5-FOA; resulting plasmid-                prepared wild-type diploid, ptal-1 heterozy- <br /> loss strain capable growth 30&#176;C YPDS (Fig.                gote, aptal-I homozygote, aptal-1 homozygote trans- <br /> 6). This result shows ptal-J reverted                   formed YCpPTAJ. Pre-tRNA accumulation the <br /> pUN60-transformed strain. SUPJJ suppressed pre-                        transformedptal-1 homozygote dramatically reduced to <br /> tRNA accumulation ptal-I (Fig. 4, lanes 1 2). Accu-                  near wild-type levels. Three different pre-tRNAs ex- <br /> mulation pre-tRNA TrpCCA occurred POC8-32c trans-                     amined, gave similar results. The result was <br /> formed pUN50 pUN60. Identical results                     obtained haploid ptal-1 strains transformed with <br /> were obtained pre-tRNA LysUUU transformed                      YCpPTAI (data shown). <br /> POC8-23d strains (data shown). These results indicate                      Several subclones YCpPTAI independent clone, <br /> that ptal-1 lesion point mutation disrupts                YEp13-G (18), assayed PTA1 function (Fig. 8B). <br /> translational reading frame PTAJ ochre stop                      The smallestptal-l-complementing region a <br /> codon.                                                                      HindIII-SalI fragment YCpPTAI (YCpPTA1-HindIII- <br />    Isolation mapping PTA] gene. The PTA1 gene                    SalI; Fig. 8). <br /> was isolated complementation ptal-1 mutation. We                    To DNA sequences present YCpPTAI map <br /> used available library prepared partial Sau3AI diges-                 ptal-1 locus, plasmid YIpPTAI constructed by <br /> tion S. cerevisiae genomic DNA ligated BamHI                    subcloning ptal-l-complementing HindIII fragment of <br /> site YCp50 (53). This vector maintained                 YCpPTAl YIp5 (the HindIII fragment present in <br /> copies S. cerevisiae cell URA3 selectable                     YCpPTAI-HindIIIB; Fig. 8). This integration vector con- <br /> marker. Approximately 64,000 transformants POC8-23d                      tains URA3 selectable marker. YIpPTAJ cut with <br /> were selected SCS-Ura 30&#176;C screened. Two                          SnaBI transformed POC8-23d, integrative trans- <br /> characteristics used identify transformants                   formant selected SCS-Ura, transformant was <br /> that contained PTAJ gene: growth 37&#176;C low                         mated POC8-32c (relevant genotype, ptal-1 ura3-52) and <br /> osmolarity pink colony color. Only transformants                    BJ4065 (relevant genotype, PTA1 URA3) produce <br /> met criteria. Plasmid DNA isolated                       diploids POC34 POC40, respectively. We tested 20 22 <br /> positive strains transformed E. coli DH5a. Plasmid                 tetrads POC34 produced viable spore clones. <br /> DNA prepared single-colony isolates trans-                  The outside genetic markers segregated normal 2:2 <br /> formed E. coli strains complemented ptal-1 high fre-                     pattern. Ts&apos; Ura+ cosegregated 2:2 17 tetrads (Table <br />  3850         O&apos;CONNOR AND PEEBLES                                                                                                                                  MOL. CELL. BIOL. <br />  <br /> LANE               1   2       3    4        5     6      7     8     9    10 11                          A. <br /> Plastmidt1     IPtN pL.NI     YC   YEE,24    rpR.&quot;:A1 i.f 11 pYL E 1&apos;9 |Y 9 C6 <br />                                             p&apos;pC6.                                I          I <br />  <br /> Gelie         Al       UP17                       RA <br />                                                          0, I    I; W      STP1       IN&apos;T                90. <br />                                                                                                           g. <br />  <br />  <br />              Al                       K..                                                                 80 <br />                                                                                                      O 70 <br />                                                                                                      S <br />                                                                                                      X 60- <br />                                                                                                      1    50- <br />                                                                                                      T40- <br />                                                                                                      I                                  _A <br />                                                                                                      a. 30- <br />                                      P&apos;re-tRNA TRP-CCA Intron                                           20- <br />                                                                                                           10- <br />                                                                                                                 0              10            20         30        40        50   60 <br />                                                                                                                                                   minutes 300C <br />                                                                                                      B. <br />  <br />  <br />                                                                                                      1.000 <br />  <br />                                                                                                  a <br />                                                                                                                     1,qf VO,        - <br />  <br />  <br />  <br />  <br />   FIG. 4. Episomal complementation ptal-1 pre-tRNA ac-                                    15 0.100- <br /> cumulation phenotype. POC8-32c (ptal-1 ura3-52) transformed <br /> with different plasmids. Total RNA extracted                                 2                                 v~~~~~~~ <br /> transformed strains assayed pre-tRNA accumulation by <br /> Northern analysis. The strains grown 30&#176;C SCS-Ura                                     .a0.010- <br /> medium. The blot hybridized 32P-labeled pTRP probe <br /> (A), stripped, hybridized 32P-labeled 5E7 probe (specific <br /> for snR7), (B). Lanes: 1, pUN50 (vector) 2, pUN60 (SUPJl); 3, <br /> YCp5O (vector); 4, YEp24 (vector); 5, pSEYC58 (vector); 6,                                           n   rni. <br /> YEpRNA1; 7, YCpLOSJ; 8, pYEP11 (RLG1 [also called LIG1]); 9,                                                    0                       1o              20             30        40 <br /> pYEP99 (RLG1); 10, pC6 (STP1); 11, untransformed POC8-32d <br /> (wild type) grown SCS medium 30&#176;C.                                                                                         minutes 370C <br />                                                                                                    FIG. 5. Splicing endonuclease activity ptal-I extracts. Ex- <br />                                                                                                  tracts prepared wild-type mutant strains S. <br />                                                                                                  cerevisiae assayed pre-tRNA splicing endonuclease activity. <br /> 5); exceptional tetrads contained Ura+ colonies                                     32P-labeled pre-tRNA PheGAA used substrate. All <br /> had (one tetrad) (two tetrads) Ts&apos; clones                                        cultures grown YPDS 30&#176;C extract preparation. (A) 0, <br /> probably represent URA3 plasmid excision events left                                        Wild type (average independent A364A extracts one <br />                                                                                                  POC8-23c extract); 0, ptal-I (average independent POC8- <br /> behind PTAI allele (Table 5). These results                                      23d extracts POC18-ld extract); A, senl-l (results one <br /> sequence derived YCpPTAl URA3 marker                                            tsx24-5b extract); A,ptal-l/senl-l (equal amounts aptal-1 a <br /> linked YIpPTAJ transformant. We analyzed 24 tetrads                                       senl-l extract mixed assayed); V, mock (wild-type <br /> from POC40. The outside genetic markers segregated                                          extract heat inactivated 65&#176;C 10 min assayed). <br /> normal 2:2 pattern. None 96 spore clones Ts                                       The extracts assayed 30&#176;C. (B) Crude nuclear extracts were <br /> Tsor growth, establishing integrated sequences                                      prepared wild-type (POC8-23c; 0),ptal-l (POC8-23d; 0), and <br /> derived YCpPTAl tightly linked toptal-1 (Table 5).                                      sen2-1 (tsx17-6b; Y) strains. Each extract preincubated 37&#176;C <br /> Hybridization analysis genomic DNA extracted                                         prior assay 25&#176;C 15 min. Data normalized eacb <br />                                                                                                  extract 100% activity time zero preincubation presented <br /> parental strains colonies tetrads                                        natural logarithm fraction product relative total <br /> demonstrated single copy YIpPTAl integrated                                        remaining substrate plus product. <br /> at intended locus segregated expected (data not <br /> shown). These results indicate sequences derived from <br /> YCpPTAl direct integration ptal-I locus. Thus, <br /> YCpPTAJ contains bona fide PTAJ gene.                                                        YCpPTAJ complemented Ts growth defect cdc24. <br />   The PTAI gene mapped chromosome I <span id='am-87' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-88'  ></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91' property="rdfs:label" content="hybrid" datatype="xsd:string"></span>hybrid</span>-                                            This finding demonstrates close linkage PTA1 and <br /> ization analysis S. cerevisiae chromosomal DNAs re-                                           CDC24. <br /> solved pulsed-field gel electrophoresis probed                                         A transcript map available region chromo- <br /> PTAJ DNA (data shown). A similar analysis using                                              1 (18, 19) (Fig. 8). Three transcripts overlap HindIII <br /> genomic DNAs S. cerevisiae strains carrying fragments                                       fragment YCpPTAJ-HindIIIB; transcripts have <br /> of chromosome I refined location PTAJ left arm                                     designated FUN9 (1.4 kb), FUN39 (3.1 kb), and <br /> (CDC24 side) chromosome I. Comparison                                                  FUN40 (0.6 kb). (The <span id='am-66' about='ncbitaxon:acronym' typeof='owl:Thing'>acronym</span> FUN stands function <br /> YCpPTAI CDC24 restriction maps suggested                                                unknown.) While direction transcription not <br /> YCpPTAJ overlapped CDC24 (Fig. 8) (19, 40). To confirm                                           established empirically RNAs (18, 19), partial <br /> this assignment, strain DK365-28b (relevant genotype, ura3                                       sequence analysis suggests FUN9 transcribed toward <br /> cdc24) transformed YCpPTAI. Several Ura+ trans-                                         centromere FUN39 FUN40 transcribed <br /> formants tested, grew 37&#176;C, indicating                                      away centromere. The smallest fragment comple- <br />  VOL. 12, 1992                                                                                                                                 S. CEREVISLAE PTA1 GENE                            3851 <br />  <br /> RLASMID      pUN3SO     pUNSO   |   p   UUN6       &apos;CpS <br />                                     L~Lost { O(1A IL <br />                                                              SCpI      I|   SC-iflF <br />                                                                                  FOA <br />                                                                                                                            LANE                 1           2           3            4 <br />  GE:NE <br />  <br />  <br />          A <br />                tfU II <br />               c}<span id='am-72' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-73' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-76'  ></span>tor</span>   lolie -cct,1r I|                            AI    |              | <br />                                                                                                                            GENOTYPE           I+/+ I+/- 1-/ I-/-I <br />                                                                                            YPDS(300C)                      YCpPTAl              I       I       I           I        +   I <br />          B                                                                                 YPD (37CC)                                          z... <br />  <br />                                                                                            SCS-ura (30CC) <br />          D <br />                                                                                            SC-ura (37CC) <br />  <br />   FIG. 6. Episomal complementation ptal-1 growth defect.                                                                             Pre-tRNA TRP-CCA Intron <br /> POC8-23d (ade2-1 ptal-l ura3-52) transformed pUN50,                                                         FIG. 7. Complementation YCpPTAI ptal-1 pre-tRNA <br /> pUN60 (SUP11), YCp5O, YCpPTAI. The transformed strains                                                        accumulation phenotype. Northern analysis performed total <br /> were grown 2 days different media 30 37&#176;C then <br />                                                                                                                   RNA extracted POC10 (+/+; lane 1), POC11 (+/ptal-1; lane <br /> photographed. The pUN60- YCpPTAJ-transformed strains                                                          2), POC6 (ptal-lIptal-l; lane 3), POC31 (ptal-l/ptal-1) trans- <br /> were induced lose plasmid DNA 5-FOA selection. The <br />                                                                                                                   formed YCpPTAI (lane 4). Five micrograms RNA was <br /> plasmid loss strains assayed growth described                                                    applied lane. The blot hybridized pTRP probe <br /> above. Growth conditions: (A) YPDS, 30&#176;C; (B) YPD, 37&#176;C; (C)                                                      autoradiographed. The blot subsequently stripped then <br /> SCS - Ura, 30&#176;C; (D) SC - Ura, 37&#176;C.                                                                              hybridized 5E7 probe ensure intact equivalent <br />                                                                                                                   amounts RNA present lane (data shown). <br />  <br /> menting ptal-1 2.5-kbp HindIII-SalI fragment (YCp- <br /> PTAI-HindIII-SalI; Fig. 8), contains large C-termi-                                                       URA3 allelic ptal-1. The resulting Ura+ diploid <br /> nal portion putative FUN39 open reading frame,                                                             (CLP6) grew wild type temperatures, <br /> lacking promoter translation initiation sites, large                                                    indicating ptal-1 Tsor defect comple- <br /> N-terminal segment putative FUN40 open reading                                                             mented. Therefore, PTAI FUN40. <br /> frame promoter intact. Since FUN9 does overlap                                                         FUN39 disrupted inserting 3.1-kbp BglII DNA <br /> this fragment, FUN9 PTA1.                                                                                  fragment containing LEU2 BamHI site FUN39 to <br />    Gene disruptions identify FUN39 PTAI. We constructed                                                        make plasmid p2.3-BL14 (Table 3 Fig. 8). p2.3-BL14 <br /> gene disruptions assign PTAJ unambiguously                                                       digested BglII plus SnaBI transformed into <br /> transcribed regions theptal-l-complementing region.                                                        diploid POC17 obtain FUN39::LEU2 disruption a <br /> A FUN40::URA3 disruption constructed replacing                                                             heterozygous condition. Asci dissected Leu&apos; <br /> the Kjpn1-BglH fragment FUN40 plasmid pCB2                                                           transformants POC17 (strains BILA BLB) either <br /> 1.2-kbp fragment containing URA3 make plasmid pCB2-                                                            YPD YPDS. Most asci produced just viable colonies <br /> U17 (Table 3 Fig. 8). Diploid POC17 transformed                                                           (Table 6). All survivors Leu-, the <br /> with BamHI-digested pCB2-U17 obtain FUN40 dis-                                                             genetic markers segregated 50%+ 50%-. <br /> ruption heterozygous condition. Asci dissected                                                          The single, exceptional Leu+ spore resulted from <br /> from Ura+ transformants (strains BUB, PUA,                                                              ectopic recombination LEU2 allele integrated the <br /> PUB). Four viable spores obtained asci                                                         PTA1 locus leu2 locus (36) nondisjunction <br /> Ura+ segregating 2:2 tetrads (data shown). Hy-                                                         leading chromosome I disomy (2, 3). Microscopic exam- <br /> bridization analysis performed genomic DNAs ex-                                                            ination nongrowing spore clones revealed germi- <br /> tracted diploid clone                                                          nation occurred growth cell division failed <br /> tetrads. The pattern hybridizing fragments confirmed                                                      cycles. The microcolonies contained 5 <br /> the intended replacement recovered                                                          89 cells (36 colonies examined). All cells <br /> restriction fragments diagnostic gene disruption al-                                                      microcolonies extremely large, arrested <br /> ways cosegregated Ura+ (data shown). Thus,                                                               unbudded cells large cells two <br /> FUN40 essential gene growth complete                                                             equally large buds. No cell division detected <br /> medium. One Ura+ haploid strain, BUB-24c (relevant gen-                                                           18 24 h 30&#176;C, cells continued to <br /> otype, FUN40::URA3), mated CLP1-la (relevant                                                               increase size. These observations FUN39 not <br /> genotype, ura3-52 ptal-1) determine FUN40::                                                            required germination essential normal growth. <br />  <br />                                                                                 TABLE 5. Tetrad analysis: YIpPTAI integration <br />                                                           Asci given no.                        No.                            Tetrads given no. of: <br /> Diploid               dissected                               viable spores                             tetrads           Ts&apos; spores/tetrad                      Ura+ spores/tetrad <br />                                                                                                         tested <br />                                                4             3              2              1    0                   4      3      2       1         0       4       3            2           1      0 <br /> POC34                    44                22                14             5              2    1          20       0      0      18      2         0       0        0          17a          0     3b <br /> POC40                    36                24                 5             4              3    0          24      24      0       0      0         0       2       19           3           0     OC <br />   a <br />       All Ura+ spores Ts&apos;. <br />   b   Two tetrads Ura+ spores Ts&apos; spore. Two outside markers segregated 2+:2- tetrads. <br />   c   One tetrad Leu- spores Leu+ spore. Five outside markers segregated 2+:2- tetrads. <br />  3852     O&apos;CONNOR AND PEEBLES                                                                                                                                      MOL. CELL. BIOL. <br />  <br />            A.                                        Left Arm Chromosome I <br />                   E Sc BH HKG BX                    S EXb CE             HEPEEP        G   HPE   H      xssc               E <br />            ,M-.r -I         )I     s irI:                                    1)Ar          II                              -*CEN <br />                  FUN42 FtN41 FUJN40                 PTA1          FUN9         CDC24             WI-              CYC3      Transcripts <br />                                                 (9J9) <br />                                                                                                                                                      Complementaton <br />            B.                     S3                                                                          S3         Plasmid                     ptal-1    PTAI.::LEU2 <br />                           1hOp                                                                               J&apos;          YCpPrAl                       +           + <br />  <br />                                  HS3                                     H <br />                                                                                                                          YCpPTAl-HindIIIB             +            +1 <br />                                  H S3                         C <br />                                    &apos;a                         ~~~~~~~~~~~~~~~~~~~~~~I                                    YCpPTAl-HindIII-ClaI         +            ND <br />  <br />  <br />  <br />  <br />                                                                                                                          YCpPTAl-HindfI-SalI          +            ND <br />                                  HS3    B <br />  <br />                                                                                                                          YCpPTAl-HindHII-BamHI        -            ND <br />                                         B                                H <br />                                                                                                                          YCpPTAl-BamHI-HindIII        -            ND <br />                       Sc                             S <br />  <br />                       I                                                                                                  pUNPTAl                      +            _ <br />  <br />  <br />  <br />                   E                                       E <br />                                                 _    _.                                                                  YEpl3-G                                   ND <br />  <br />                                             x                                                           x <br />  <br />                                                                                                                          YIp5-XhoLA                                ND <br />                       Sc B H H          B           S <br />  <br />                            I11 )                                                                                         pCB2-U17                     +            ND <br />  <br />                                  H S3   B           SE <br />                                                                                                                          p2.3-BL14                    -            ND <br />  <br />  <br />   FIG. 8. Restriction map YCpPTAI subclones. (A) The restriction map PTAI locus diagrammed scale. Restriction sites: <br /> B, BamHI; C, CiaI; E, EcoRI; G, BglII; H, HindIII; Hp, HpaI; K, KpnI; P, PvuII; S, SalI; Sc, SacI; S3, Sau3AI; X, XhoI; Xb, XbaI. (B) <br /> The original PTAI clone, YCpPTA1, partially digested Sau3AI fragment inserted BamHI site YCp5O. Subclones YCpPTAI <br /> and clones locus diagrammed. The complementation ability plasmid ptal-1 PTAI::LEU2 strains is <br /> indicated. ND, determined. <br />  <br />  <br />    To recover haploid strains carrying FUN39::LEU2, plas-                                                Sequence analysis PTA]. We determined se- <br /> mids FUN39 sequences introduced diploids                                                quence 2,760 bp spanning PTAI (Fig. 9). This segment <br /> BLA BLB, asci dissected. Only plasmids                                               includes entire region mapped PTAJ transcript <br /> containing complete FUN39 gene, YCpPTA1,                                                   (18). There single large open reading frame could <br /> rescued lethal defect FUN39::LEU2 allowed                                                   encode protein 785 amino acids, assuming transla- <br /> complete tetrads recovered, Leu+ segregating 2:2                                            tion initiates available methionine codon. The <br /> (Table 6). In Leu+ colonies, complementing plasmid <br /> was retained, plasmids frequently lost <br /> from Leu- colonies. This segregation pattern expected <br /> for centromeric plasmid complementing disruption                                                         TABLE 6. Tetrad analysis: PTA1::LEU2 disruption <br /> essential gene. Furthermore, Leu+ colonies failed                                                                                                                Tetrads given <br /> produce 5-FOA-resistant colonies, Leu- colonies                                                                                    No.      No.            no. viable <br /> did readily. The Leu+ phenotype linked                                                            Diploid              4-spore asci complete            spores/tetrad <br /> expected, altered pattern genomic DNA restriction endo-                                                                                 dissected    tetradsa <br />                                                                                                                                                                         4   3    2   1 0 <br /> nuclease fragments (data shown). <br />    To determine FUN39::LEU2 allelic ptal-1,                                              BLA(YPDS, 30&#176;C)                          23            18        0 0     18 <br />                                                                                                                                                                                  0       0 <br /> the haploid BLAY5-2c [relevant genotype, ura3 leu2                                                     BLB(YPDS, 30&#176;C)                          22            19        0 1b 16C 2       0 <br /> FUN39::LEU2 (YCpPTA1)J mated POC8-36a (rele-                                                    BLB(YPD, 23&#176;C)                           12            10        0 0   9 0        1 <br /> vant genotype, leu2 ura3 ptal-1). The resulting diploid                                                BLA(pUN5O)                                6             4        0 0   4 0        0 <br /> CLP11 readily lost YCpPTAI Ura- diploid                                                    BLA(pUNPTAI)                              6             6        0 0   6 0        0 <br /> CLP11b, Tsor growth. Transformation                                                   BLA(YCp5O)                               30            30        0 0 29 1         0 <br />                                                                                                        BLA(YCpPTAI)                             18            18        4 7   7 0        0 <br /> CLP11b (relevant genotype, FUN39::LEU2Iptal-1)                                                    BLA(YCpPTAI-HmndIIIB)                    22            22        7 9   5 1        0 <br /> YCpPTA1, pUNPTAI, pUN60 restored growth high                                                     BLB(YCp5O)                               12            12        0 0 12 0         0 <br /> temperature (data shown). These results confirm                                           BLB(YCpPTAI)                             12            11        2 7   2 0        0 <br /> observed Tsor growth CLP11b results expression of <br /> the ochre-suppressible ptal-J mutation confirms                                                 All spores successfully germinated judged microscopic visual <br />                                                                                                        inspection. <br /> FUN39::LEU2 (hereafter referred PTA1::LEU2)                                                     b   All spore clones Leu- tetrad. <br /> allelic ptal-1.                                                                                       c   A single Leu+ spore recovered group. <br />  VOL. 12, 1992                                                                                                                                S. CEREVISIAE PTA1 GENE                          3853 <br />  <br />  -240   ------------------------------------------------------------ <br />         TGTGTTATATCACTGTCCTACTCCGCATGTATACGAGACATAAAACTAGCTTTCCTTTAT                           -181   1141   CCTGTGTCCGCACCTGCAACGGGCTCTTCAACCGAAAACATGCTTGATCAACTGAAGATA                     1200 <br />                                                                                                        381   P V S A P A T G S S T E N M L D Q L K I                                          400 <br />  -180   AATGGCGTGGTTTCCACTCCATCTTGATGGAAAATCGCCGTTCTGATGTTCAAGTCGTAT                           -121 <br />                                                                                                       1201   TTGCAAAAATACACCCTCAACAAGGCTTCACACCAGGGCAATACTTTTTTCAACAACTCA                     1260 <br />  -120   ATAGTGACTTGATCTAAAAGTGAAAAGTTGCCATATTTCGATACCTTTTAACCGTTAGTA                           -61     401   L   Q   K   Y   T   L   N   K   A    S   H   Q   G   N   T   F   F   N   N   S   420 <br />   -60   CTTGACTGAACAAATGGTTAAATCGTGGCAAGAAGCATATAACTACCACTAAGCAGCGCA                           -1     1261   CCCAAACCAATCAGCAACACCTACTCATCTGTGTACTCATTGATGAACAGTTCGAACTCC                     1320 <br />                                                                                                        421   P K P I S N T Y S S V Y S L M N S S N S                                          440 <br />     1   ATGTCATCTGCAGAGATGGAACAATTGTTACAGGCCAAGACACTGGCCATGCACAACAAT                           60 <br />     1   M   S   S   A   E   M   EQ      L   L   Q   A   K   T   L   A   M   H   N   N          20     1321   AACCAGGATGTGACCCAGCTACCCAATGACATACTTATCAAGCTGTCCACAGAGGCCATC                     1380 <br />                                                                                                        441   N   Q   D   V   T   Q   L   P   N    D   I   L   I   K   L   S   T   E   A   I   460 <br />    61   CCAACGGAGATGCTGCCCAAGGTGCTCGAAACTACGGCATCCATGTACCACAACGGTAAT                           120 <br />         P   T   E   M   L   P   K   V   L   E   T   T   A   S   M   Y   H   N   G   N          40     1381   TTGCAAATGGACAGCACGAAACTGATCACCGGATTGTCTATCGTTGCTTCGAGGTACACG                     1440 <br />    21                                                                                                  461   L   Q   M   D   S   T   K   L    I   T   G   L   S   I   V   A   S   R   Y   T   480 <br />   121   CTCAGCAAGCTGAAGTTGCCTTTGGCCAAGTTTTTTACACAGTTAGTTCTAGACGTGGTG                           180 <br />    41   L   S   K   L   K   L   P   L   A   K   F   F   T   Q   L   V   L   D   V   V          60     1441   GATTTAATGAATACGTACATCAATTCTGTACCGTCCTCGTCATCATCAAAGAGGAAATCC                     1500 <br />                                                                                                        481   D   L   M   N   T   Y   I   N   S    V   P   S   S   S   S   S   K   R   K   S   500 <br />   181   TCGATGGACTCTCCAATTGCGAATACTGAGAGACCGTTTATTGCTGCTCAATATCTGCCA                           240 <br />    61   S M D S P I A N T E R P F I A A Q Y L P                                                80     1501   GACGATGATGACGACGGCAACGACAATGAAGAAGTTGGAAACGATGGCCCAACGGCTAAT                     1560 <br />                                                                                                        501   D   D   D   D   D   G   N   D   N    E   E   V   G   N   D   G   P   T   A   N   520 <br />   241   CTACTTCTTGCTATGGCGCAATCCACCGCGGACGTACTAGTGTACAAGAATATCGTGCTT                           300 <br />    81   L L L A M A QS T A D V L V Y K N I V L                                                 100    1561   AGCAAGAAAATCAAAATGGAAACAGAACCACTAGCGGAGGAACCAGAGGAGCCCGAAGAC                     1620 <br />                                                                                                        521   S   K   K   I   K   M   E   T   E    P   L   A   E   E   P   E   E   P   E   D   540 <br />   301   ATTATGTGCGCTTCATACCCGCTGGTGTTGGATCTGGTTGCTAAGACATCAAACCAGGAA                           360 <br />   101   I   M   C   A   S   Y   P   L   V   L   D   L   V   A   K   T   S   N   Q   E          120    1621   GATGACCGAATGCAGAAGATGCTTCAAGAAGAGGAAAGCGCCCAAGAAATCTCAGGAGAT                     1680 <br />                                                                                                        541   D   D   R   M   Q   K   M   L   Q    E   E   E   S   A   Q   E   I   S   G   D   560 <br />   361   ATGTTTGATCAGTTGTGTATGCTGAAGAAGTTCGTGCTCTCGCACTGGAGAACTGCATAT                           420 <br />   121   M   F   D   Q   L   C   M   L   K   K   F   V   L   S   H   W   R   T   A   Y          140    1681   GCCAACAAATCAACTTCTGCCATTAAGGAGATCGCACCCCCCTTTGAACCTGACTCATTG                     1740 <br />                                                                                                        561   A   N   K   S   T   S   A   I   K    E   I   A   P   P   F   E   P   D   S   L   580 <br />   421   CCTTTGCGTGCCACCGTTGACGATGAAACGGATGTCGAACAATGGCTGGCGCAGATTGAC                           480 <br />   141   P L R A T V D D E T D V E Q W L A Q I D                                                160    1741   ACGCAGGATGAAAAACTAAAGTACCTCTCAAAGCTGACCAAGAAACTGTTTGAATTATCC                     1800 <br />                                                                                                        581   T Q D E K L K Y L S K L T K K *L F E L S                                         600 <br />   481   CAAAATATCGGCGTGAAATTAGCGACCATCAAGTTCATATCTGAGGTCGTGCTGTCGCAA                           540 <br />   161   QN I G V K L A T I K F I S E V V L SQ                                                  180    1801   GGTCGCCAGGATACTACCCGGGCCAAATCTTCGTCTTCCTCCTCCATATTACTGGACGAT                     1860 <br />                                                                                                        601   G   R Q     D   T   T   R   A   K    S   S   S   S   S   S   I   L   L   D   D   620 <br />   541   ACTAAATCACCCAGCGGCAACGAGATTAATTCATCTACCATCCCGGATAACCACCCTGTG                           600 <br />   181   T   K   S   P   S   G   N   E   I   N   S   S   T   I   P   D   N   H   P   V          200    1861   GACGACTCCTCGTCATGGTTACACGTCTTAATCAGATTGGTTACGAGAGGAATCGAAGCA                     1920 <br />                                                                                                        621   D   D   S   S   S   W   L   H   V    L   I   R   L   V   T   R   G   I   E   A   640 <br />   601   TTGAACAAACCGGCTTTGGAGAGCGAGGCTAAGAGGCTTCTTGATATGTTGCTAAACTAC                           660 <br />   201   L   N   K   P   A   L   E   S   E   A   K   R   L   L   D   M   L   L   N   Y          220    1921   CAAGAGGCCAGTGACCTGATTCGTGAAGAACTGCTTGGCTTCTTCATCCAGGATTTCGAG                     1980 <br />                                                                                                        641   Q   E   A   S   D   L   I   R   E    E   L   L   G   F   F   I   Q   D   F   E   660 <br />   661   CTAATTGAGGAACAGTACATGGTCTCGTCCGTTTTCATTGGTATCATCAATTCTTTATCC                           720 <br />   221   L   I   E   E   Q   Y   M   V   S   S   V   F   I   G   I   I   N   S   L   S          240    1981   CAACGTGTCAGTCTGATCATTGAATGGCTCAATGAAGAATGGTTCTTCCAAACCTCGCTG                     2040 <br />                                                                                                        661   Q   R   V   S   L   I   I   E   W    L   N   E   E   W   F   F   Q   T   S   L   680 <br />   721   TTCGTCATCAAAAGAAGGCCGCAGACAACAATAAGAATTCTTTCCGGGCTGTTGCGTTTC                           780 <br />   241   F   V   I   K   R   R   P   Q   T   T   I   R   I   L   S   G   L   L   R   F          260    2041   CATCAAGATCCCTCTAACTACAAAAAATGGTCCTTAAGAGTTCTCGAGTCTCTGGGTCCA                     2100 <br />                                                                                                        681   H   Q D     P   S   N   Y   K   K    W   S   L   R   V   L   E   S   L   G   P   700 <br />   781   AACGTCGACGCCAAGTTTCCCCTAGAGGGCAAGTCTGACTTGAACTACAAACTATCCAAG                           840 <br />   261   N   V   D   A   K   F   P   L   E   G   K   S   D   L   N   Y   K   L   S   K          280    2101   TTCCTTGAAAACAAACACAGACGATTCTTCATCAGACTTATGAGCGAACTGCCCAGTCTT                     2160 <br />                                                                                                        701   F   L   E   N   K   H   R   R   F    F   I   R   L   M   S   E   L   P   S   L   720 <br />   841                                                                                   rT I   900 <br />   281   R   F   V   E   R   A   Y   K   N   F   V   Q   F   G   L   K   N   Q   I   I          300    2161   CAAAGCGATCATCTTGAGGCACTGAAGCCTATCTGCCTGGATCCGGCAAGAAGTTCCCTT                     2220 <br />                                                                                                        721   Q  S D H L E A L K P I C L D P A R S S L                                         740 <br />   901   ACAAAATCCCTCTCATCCGGATCAGGGTCATCGATCTACTCCAAGCTGACCAAGATTTCT                           960 <br />   301   T K S L S S G S G S S I Y S K L T K I S                                                320    2221   GGTTTCCAAACGCTAAAGTTTCTCATTATGTTTAGACCCCCAGTGCAGGACACTGTTCGC                     2280 <br />                                                                                                        741   G   F   Q   T   L   K   F   L   I    M   F   R   P   P   V   Q   D   T   V   R   760 <br />   961   CAAACTTTACACGTTATTGGCGAAGAGACCAAGAGCAAGGGAATTTTGAACTTCGACCCT                           1020 <br />   321   Q  T L H V I G E E T K S K G I L N F D P                                               340    2281   GACCTGCTGCATCAGCTAAAGCAAGAAGATGAAGGCTTACACAAGCAGTGCGATTCACTG                     2340 <br />                                                                                                        761   D L L H Q L K Q E D E G L H K Q C D S L                                          780 <br />  1021   TCCAAGGGCAATAGCAAGAAAACGTTGTCCAGGCAGGACAAACTAAAATACATCTCACTA                           1080 <br />   341   S K G N S K K T L S R Q D K L K Y I S L                                                360    2341   CTTGACAGGCTAAAATGATCACAACCATTACATACATCATATATACGCATGTGTAGGGTC                     2400 <br />                                                                                                        781   L   D   R   L   K <br />  1081   TGGAAAAGGCAATTATCCGCGTTATTGTCTACTCTAGGGGTGTCCACAAAGACCCCCACG                           1140 <br />   361   W K R Q L S A L L S T L G V S T K T P T                                                380    2401   TTCCTTCAACTTGCAAATCAATGTTTTATCTATCATTCTGTGCTATAGGGACCTCGCAAT                     2460 <br />                                                                                                       2461   TTTGACACTTCCGTAAGGAGTCATTCAGCGGCGGAGTCACTTTCGCTAACTCTTTCTTCT                     2520 <br />   FIG. 9. DNA sequence PTAI gene. The sequence entire PTAI gene shown flanking regions corresponding <br /> approximately ClaI-SphI fragment. The DNA sequence numbered 1 ATG open reading frame; negative <br /> numbers indicate sequences upstream ATG. The predicted protein sequence shown DNA sequence single-letter <br /> code, asterisk termination codon. <br />  <br />  <br />  <br /> calculated molecular weight 88,417 isoelectric                                                                      DISCUSSION <br /> point 6.17. The insertion LEU2 BamHI removes                                                 We identified recessive nuclear mutation called <br /> replaces C-terminal 51 amino acids (P735 K785). No                                             ptal-I causes accumulation end-trimmed, intron- <br /> significant sequence similarities known protein sequences                                          containing pre-tRNAs growth conditions (Fig. 1) <br /> in GenBank (6) EMBL (8) data base identified                                           exhibits temperature-sensitive osmotic-remedial <br /> FASTA TFASTA (47) sequence comparison analysis,                                                    growth. The set 10 pre-tRNA families accumu- <br /> using entire PTAI protein-coding region query                                              lated pre-tRNA processing-defective mu- <br /> sequence. For example, best score FASTA                                                    tants, including rnal-1, losl-l, senl-l mutants (27, 28, <br /> search represented 16 identities spanning segment 58                                             32, 43, 62). The ptal-I mutation causes slight accu- <br /> amino acids included just tripeptides, dipep-                                             mulation 2/3 molecules cells grown low osmolarity <br /> tides, single amino acid matches. Shuffled Ptalp                                              (Fig. 1) accumulation 5&apos;-2/3 (a 2/3 molecule <br /> sequences frequently produced better scores, seg-                                              5&apos; leader) 2/3 molecules cells grown high <br /> ment Ptalp repeatedly match                                             osmolarity (43). We did detect accumulation 35S <br /> proteins. No motifs identified pattern-matching                                           pre-rRNA actin pre-mRNA inptal-1 strains Northern <br /> program PROFILESCAN (22). Only short ambiguous                                                     blots (data shown), indicating ptal-I allele <br /> patterns identified program MOTIFS, using                                              specifically affects pre-tRNA processing. ptal-I segregates <br /> PROSITE data base (5). As FASTA searches,                                                     simple Mendelian character (Fig. 2). Complementation <br /> shuffled Ptalp sequences produced equal higher numbers                                              tests, gene mapping, sequence comparisons estab- <br /> of matches. Self-comparisons dot matrix methods re-                                                 lished PTAI different previously described <br /> vealed internal repeated amino acid sequences. The                                                  gene affects pre-tRNA processing splicing. The <br /> simplest interpretation results PTAJ                                          ptal-1 lesion apparently ochre stop codon, indicated <br /> closely related previously sequenced gene.                                                      suppression SUP11 (Fig. 4 6). The recessive <br />  3854     O&apos;CONNOR AND PEEBLES                                                                                 MOL. CELL. BIOL. <br />  <br /> character ptal-1 indicates absence truncation      form active complex. Alternatively, C-terminal <br /> of Ptalp inptal-I strains results observed phenotypes.     Ptalp fragment ochre fragment Ptalp be <br /> The ptal-l-derived ochre fragment probably deleteri-       separate functional domains independently carry <br /> ous se. ptal-1 apparently affects cleavage step        different functions. Neither explanations are <br /> pre-tRNA splicing, known role pre-tRNA splicing            completely satisfactory, disruption allele PTA]:: <br /> endonuclease. However, extracts prepared fromptal-1 cells         LEU2 make N-terminal segment 734 amino <br /> contain normal amounts pre-tRNA splicing endonuclease          acids, allele fails complement eitherptal-] the <br /> activity normal heat stability (Fig. 5). This observation    C-terminal segment PTA] present pUNPTAl. The <br /> was somewhat unexpected, pre-tRNA accumulation           precise mechanism complementation ptal-] the <br /> ptal-I cells drastic. These results suggest Ptalp      large C-terminal gene fragment pUNPTAI remains <br /> may responsible unknown step pre-tRNA                obscure now. <br /> processing control activity known pre-tRNA           Although original ptal-] isolate, LA4-22, grows on <br /> processing activity.                                              YPD 23 30&#176;C, outcrossed strains harborptal-] are <br />     The PTA] gene isolated plasmid YCpPTA1,          difficult grow media 1 M sorbitol (43). Growth <br /> was identified complementation ptal-1. YCpPTAI              slow, revertants arise frequently ptal-] <br /> restores wild-type growth dramatically reduces pre-           strains grown low osmolarity (data shown). All <br /> tRNA accumulation transformed ptal-I strains            ptal-] strains routinely cultured high osmolarity <br />  (Fig. 7). Sequences present YCpPTAI direct recombina-         using 1 M sorbitol, alleviates severe growth <br /> tion ptal-1 locus, tightly linked CDC24     defect ptal-] selects newly arising <br /> chromosome I (Table 5, Fig. 8, data shown). The           nonsense suppressor mutations (57). However, growth 1 <br /> complete sequence PTA] determined,                M sorbitol does prevent pre-tRNA accumulation in <br />  inspection sequence reveals single large open          ptal-] strains (43). The observation thatptal-] suppressed <br />  reading frame extends 785 amino acids        ochre suppressor tRNA gene SUP]] suggested that <br />  methionine (Fig. 9). The predicted molecular weight Ptalp     ptal-] represent null phenotype. However, is <br />  88,417 isoelectric point 6.17. The amino acid      case, PTA]::LEU2 spores fail grow even <br />  composition does appear remarkable,      1 M sorbitol medium (Table 5). Theptal-] ochre <br />  stretches unusual monotonous amino acid compo-          fragment apparently performs essential function. Al- <br />  sition. There protein sequences data base     ternatively, occasional translational read-through the <br />  significantly like PTA]. Moreover, unable         ochre stop codon generate low level full-length <br />  identify familiar motifs anticipated     gene product sufficient essential activity Ptalp. <br />  protein essential role pre-tRNA processing           Identification pta]-1 demonstrates screening of <br />  pathway. In particular, evident membrane-            mutagenized Ts collections accumulation precur- <br />  spanning, single-stranded RNA-binding, zinc finger, nu-        sor RNAs filter hybridization analysis remains feasible <br />  cleotide-binding motifs. The predicted size Ptalp       approach identifying essential genes affecting pre- <br />  large proposed pre-tRNA splicing             RNA processing. One version approach orig- <br />  endonuclease subunits (51). Further investigation          inally described 1986 (42), similar methods been <br />  physiological role PTAI progress.                        successfully used identify S. cerevisiae pre-mRNA and <br />     The smallest subclone complements ptal-1 2.3          pre-tRNA processing mutants (1, 25, 60). Since 2 (LOS] <br />  kbp YCpPTAl insert includes large N-terminal        SEN2) 18 known genes affecting pre-tRNA <br />  portion FUN40 C-terminal FUN39 sufficient        processing represented multiple alleles were <br />  524 amino acids (18) (YCpPTAI-HindIII-SalI; Table 3           identified direct screening, probably S. <br />  Fig. 8). Since FUN39 information separated            cerevisiae genes affecting pre-tRNA processing remain <br />  promoter, initially thought FUN39          (26). Recovery analysis mutant alleles of <br />  expressed 2.5-kbp fragment FUN39         genes help define pre-tRNA processing <br />  PTA]. After disruption FUN40 established         pathway completely. <br />  PTA] FUN40, tested possibility         There 100 transcribed regions chromosome I <br />  C-terminal half FUN39 complementsptal-1. FUN39            S. cerevisiae. Of genes, 15 identified by <br />  disrupted LEU2 make PTAI::LEU2 allele (Fig.          classical genetic methods. This disparity termed <br />  8). We PTA1::LEU2 recessive lethal mutation      gene-number paradox (31, 50). Recently, transcribed <br />  fails complement ptal-]. Curiously, 4.0-kbp          regions chromosome I mapped systematically <br />  fragment pUNPTA] complements ptal-] fails             disrupted targeted recombination determine which <br />  complement PTA1::LEU2 (Fig. 8). At different         transcripts represent essential functions (18, 19). In one <br />  restriction fragments cloned different plasmids serve   interval, transcribed regions (FUN]], FUN12, FUN]9, <br />  complement ptal-] providing similar-size C-terminal        FUN20, FUN21, FUN22) disrupted; three <br />  gene fragments (Fig. 8). This situation reminiscent    (FUN12, FUN]9, FUN20) required growth on <br />  complementation pattern seen sen]-]; case,         YPD. In complementary approach, extensive mutagen- <br />  large C-terminal gene fragment expressed promoter      esis chromosome I conducted (24). Of 21 independent <br />  internal gene normally used.           Ts mutations analyzed, 17 mapped just previously <br />   Nonetheless, protein translated abbreviated        identified complementation groups. Of remaining Ts <br />   transcript complements point mutations disrup-         mutations, mapped known gene involved cysteine <br />   tions SEN] (13). We fully explain observed        metabolism, leaky map, last <br />   complementation pattern PTA] locus, suggest      mapped FUN24, essential gene unknown function. <br />   example intragenic complementation.      PTA] represents eighth gene identified Ts allele <br />   It possible episome-encoded C-terminal fragment      chromosome I. These studies. chromosome I genes <br />   Ptalp associates N-terminal ochre      demonstrate essential genes S. <br />   fragment Ptalp synthesized genomicptal-] allele     cerevisiae difficult identify conventional <br />  VOL. 12, 1992                                                                                    S. CEREVISIAE PTAI GENE               3855 <br />  <br /> Ts strain collections. Our experience Tsor pheno-                 protocols molecular biology. John Wiley &amp; Sons, New York. <br /> type ofptal-1 suggests additional essential genes          5. Bairoch, A. PROSITE: dictionary protein sites pat- <br /> found Tsor osmotic-remedial collections.                             terns, 7th release. University Geneva, Geneva, Switzerland. <br />    There possible roles PTAJ gene                  6. Bilofsky, H. S., C. Burks. 1988. The GenBanks genetic <br />                                                                            sequence data bank. Nucleic Acids Res. 16:1861-1864. <br /> product cell. (i) Ptalp normally functions           7. Boeke, J. D., J. Trueheart, G. Natsonlis, G. R. Fink. 1987. <br /> the turnover excess pre-tRNA. Accumulation occurs                    5-Fluoro-orotic acid selective agent yeast molecular <br /> ptal-I transcription routinely produces pre-                  genetics. Methods Enzymol. 154:164-175. <br /> tRNA required, excess destined turnover.            8. Cameron, G. N. 1988. The EMBL data library. Nucleic Acids <br /> (ii) Ptalp functions negative regulator tRNA gene                  Res. 16:1865-1867. <br /> transcription responds unknown signal. For                 9. Chen, J.-Y., G. Kirchner, M. Aebi, N. C. Martin. 1990. <br /> instance, Ptalp sense level charged un-                    Purification properties yeast ATP (CTP):tRNA nucleoti- <br /> charged cytoplasmic tRNA relative translational need                dyltransferase wild type overproducing cells. J. Biol. <br /> inhibit tRNA gene transcription excess                      Chem. 265:16221-16224. <br />                                                                        10. Church, G. M., W. Gilbert. 1984. Genomic sequencing. <br /> tRNA develops. Lack Ptalp function leads uncon-                      Proc. Natl. Acad. Sci. USA 81:1991-1995. <br /> trolled tRNA gene transcription pre-tRNA accumula-                 11. Clark, M. W., J. Abelson. 1987. The subnuclear localization <br /> tion. (iii) Ptalp functions retain pre-tRNA nucleus.             tRNA ligase yeast. J. Cell Biol. 105:1515-1526. <br /> Lack Ptalp allows inappropriate nuclear export un-               12. Culbertson, M. R., M. Winey. 1989. Split tRNA genes and <br />  spliced pre-tRNA, accumulation results separating                products: paradigm study cell function and <br /> unspliced pre-tRNAs splicing activities re-                  evolution. Yeast 5:405-427. <br />  tained nucleus (11, 39). (iv) Ptalp functions             13. DeMarini, D. J., M. Winey, D. Ursic, F. Webb, M. R. <br />  positive effector pre-tRNA splicing activity. Pre-tRNA                 Culbertson. 1992. SEN1, positive effector tRNA-splicing <br />  accumulation ptal-I strains reflect compen-               endonuclease Saccharomyces cerevisiae. Mol. Cell. Biol. <br />                                                                            12:2154-2164. <br />  sation mechanism increasing pre-tRNA substrate concen-             14. DeRobertis, E. M., M. V. Olson. 1979. Transcription and <br />  tration down-regulated pre-tRNA splicing activ-               processing cloned yeast tyrosine tRNA genes microinjected <br />  ities supply adequate amounts mature tRNA.                   frog oocytes. Nature (London) 278:137-143. <br />  Additional studies needed distinguish              15. Deutscher, M. P. 1984. Processing tRNA prokaryotes and <br />  models Ptalp function.                                                eukaryotes. Crit. Rev. Biochem. 17:45-71. <br />                                                                        16. Deutscher, M. P. 1990. Ribonucleases, tRNA nucleotidyltrans- <br />                       ACKNOWLEDGMENTS                                      ferase, 3&apos; processing tRNA. Prog. Nucleic Acids Res. <br />                                                                            39:209-240. <br />   We thank Shirley Wang, <span id='am-61' about='obo:NCBITaxon_735482' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37571 obo:NCBITaxon_37570 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_688638 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-62' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-63'  ></span><span id='am-64' property="rdfs:label" content="Anita" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Anita</span> Hopper, Eric Phizicky, Calvin Ho,       17. Devereux, J., P. Haeberli, 0. Smithies. 1984. A comprehen- <br /> John Abelson, Mark Winey, Mike Culbertson, Emil van Zyl,               sive set sequence analysis programs VAX. Nucleic <br /> James Broach providing yeast strains plasmids              Acids Res. 12:387-395. <br /> communication unpublished results. We grateful John          18. Diehl, B. E. 1990. The gene-number paradox. Ph.D. thesis. <br /> Woolford, Elizabeth Jones, members laboratories           University Michigan, Ann Arbor. <br /> advice, yeast strains, plasmid DNAs used          19. Diehl, B. E., J. R. Pringle. 1991. Molecular analysis of <br /> studies. We express appreciation Graham Hatfull material        Saccharomyces cerevisiae chromosome I: identification ad- <br /> assistance advice sequencing PTAI. We thank Leslie Lyons,           ditional transcribed regions demonstration en- <br /> Bonnie Diehl, David Kaback, John Pringle providing yeast           code essential functions. Genetics 127:287-298. <br /> chromosomal DNAs hybridization filters, providing yeast strains     20. Eliedge, S. J., R. W. Davis. 1988. A <span id='am-30' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> versatile <br /> and cloned DNAs pertaining chromosome I, communication              centomeric <span id='am-68' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> designed use sector-shuffle muta- <br /> of unpublished results. We specifically thank David Kaback             genesis assay Saccharomyces cerevisiae. Gene 70:303-312. <br /> identifying map location PTAI comparing restriction maps.    21. Greer, C. L., C. L. Peebles, P. Gegenheimer, J. Abelson. <br /> We wish emphasize appreciation open communication             1983. Mechanism action yeast RNA ligase tRNA <br /> and sharing materials yeast pre-tRNA processing                splicing. Cell 32:537-546. <br /> laboratories. We grateful Robert Duda assistance         22. Gribskov, M., M. Homyak, J. Edenfield, D. Eisenberg. 1988. <br /> sequence comparison analysis Pittsburgh Supercomput-             Proffle scanning dimensional structural patterns in <br /> ing Center use facilities. We thank Lisa Altomari,        protein sequences. CABIOS 4:61-66. <br /> James Brayer, Holly Anderson, Todd Pippert, Viktor Stolc, Chadd        23. Hanahan, D. 1983. Studies transformation Eschenichia coli <br /> Nesbitt, Mincheng Zhang, Xiaodong Zhu technical assis-             plasmids. J. Mol. Biol. 166:557-580. <br /> tance.                                                                 24. Harris, S. D., J. R. Pringle. 1991. Genetic analysis of <br />   This work supported grant GM37166 NIH             Saccharomyces cerevisiae chromosome I: role muta- <br /> by grant NP-729 American <span id='am-33' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Cancer" datatype="xsd:string"></span>Cancer</span> Society.                           gen specificity delimiting set genes identifiable using <br />                                                                             temperature-sensitive-lethal mutations. Genetics 127:279-285. <br />                           REFERENCES                                   25. Ho, C. K., R. Rauhut, U. Vijayraghavan, J. Abelson. 1990. <br />  1. Aebi, M., G. Kirchner, J.-Y. Chen, U. Vjayraghavan, A.                  Accumulation pre-tRNA splicing &apos;2/3&apos; intermediates a <br />     Jacobson, N. C. Martin, J. Abelson. 1990. Isolation            Saccharomyces cerevisiae mutant. EMBO J. 9:1245-1252. <br />     temperature-sensitive mutant altered tRNA nucleotidyl-     26. Hopper, A. K. 1990. Genetic methods study trans- <br />     transferase cloning gene encoding tRNA nucleotidyl-          acting genes involved processing precursors yeast <br />     transferase yeast Saccharomyces cerevisiae. J. Biol.             cytoplasmic tRNAs. Meth <br /> </body></html>