<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, May 1991, p. 2629-2640                                                                    Vol. 11, No. 5 <br /> 0270-7306/91/052629-12$02.00/0 <br /> Copyright &#169; 1991, American Society Microbiology <br />  <br />  <br />                          The Yeast Homolog Mouse Tcp-1 Affects <br />                               Microtubule-Mediated Processest <br />                                    <span id='am-16' about='obo:NCBITaxon_146270' typeof='owl:Thing obo:NCBITaxon_6514 obo:NCBITaxon_33208 obo:NCBITaxon_216331 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_146272 obo:NCBITaxon_33154 obo:NCBITaxon_680346 obo:NCBITaxon_1206795 obo:NCBITaxon_70849 obo:NCBITaxon_216305 obo:NCBITaxon_216307 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_6072'><span id='am-17' property="rdfs:label" content="Doris" datatype="xsd:string"></span><span id='am-18'  ></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>DORIS</span> URSIC1* AND MICHAEL R. CULBERTSON&apos; 2 <br />                           Laboratories Molecular Biology&apos; Genetics,2 University Wisconsin, <br />                                         1525 Linden Drive, Madison, Wisconsin 53706 <br />                                         Received 17 September 1990/Accepted 25 January 1991 <br />  <br />               A <span id='am-21' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-22' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-24'  ></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> homolog <span id='am-77' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-78'  ></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Drosophila melanogaster</span> mouse Tcp-1 encoding taiDless complex <br />            polypeptide 1 (TCP1) identified, sequenced, mapped. The mouse t complex under <br />            scrutiny decades effects embryogenesis sperm differentiation function. TCPI <br />            essential gene yeast cells located chromosome 4R, linked petl4. The TCPI-encoded proteins <br />            yeast, Drosophia, mouse cels share 61 72% amino acid sequence identities, suggesting a <br />            primordial function TCPI gene product. To assess function, constructed cold-impaired recessive <br />            mutation (tcpl-1) yeast gene. CeDls carrying tcpl-l mutation grew linearly exponentialy <br />            restrictive temperature 15&#176;C generation time approximately 32 h minimal medium. Both <br />            multinucleate anucleate cells accumulated time, suggesting linear growth kinetics be <br />            explained generation anucleate buds incapable cell division. In addition, multinucleate <br />            anucleate cells contained morphologically abnormal structures detected anti-a-tubulin antibodies. The <br />            kinetics appearance abnormalities suggest direct consequence loss function of <br />            TCP1 protein delayed, indirect consequence cell death. We observed strains carrying <br />            tcpl-l hypersensitive antimitotic compounds. Taken together, observations imply TCP1 <br />            protein affects microtubule-mediated processes. <br />  <br />    The mouse t complex unusual properties                 spermiogenesis mouse (62464). It exists elec- <br /> have attracted attention geneticists nearly 60 years           trophoretic forms, TCP-1A TCP-1B (5961, 77). TCP-1A <br /> (11, 12, 20, 60). t alleles discovered                mice carrying t chromosome, all <br /> interaction dominant T-locus mutation produce                 wild-type mice TCP-1B isozyme. Immunofluores- <br /> tailless phenotype double heterozygous T/t animals. Het-               cent analysis distribution mouse protein in <br /> erozygous T/+ animals short tail, homozygous                   tissue culture cells shown TCP-1 be <br /> T/T animals embryonic lethals. Homozygous tx/tY males                 peripherally associated cytoplasmic face mem- <br /> are sterile (37, 38), heterozygous +/t animals tails             branes trans-Golgi network (77). <br /> of normal length visually indistinguishable                     In recombinant derivatives t complex, been <br /> normal wild-type +/+ animals. Compared wild-type                     shown Tcp-l locus maps chromosomal <br /> laboratory chromosome 17, principal variation t                    interval contains Tcd4 locus. Tcp-J Tcd4 may <br /> chromosomes consists inversions 1%              locus (63), identity yet <br /> of mouse genome (30 Mbs DNA) (21, 23). The t                       proven involvement TCP-1A transmission <br /> complex given rearranged region. It                    ratio distortion uncertain (77). <br /> contains 100 genes (39), including genes ex-                   The following TCPI DNA sequences known: complete <br /> pressed testes products t-allele-specific               mouse Tcp-lb, partial mouse Tcp-la, human Tcp-l (75, <br /> alterations.                                                              76); complete Drosophila <span id='am-155' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-156'  ></span><span id='am-157' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-159' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span>melanogaster</span> Tcp-J (72) and <br />    t chromosomes confer number effects, including                    yeast TCPI (72; report). The genes share extensive <br /> disturbances embryonic development, alterations                     nucleic acid sequence identity greater identity in <br /> sperm differentiation function, transmission ratio                deduced amino acid sequence, suggesting stringent evolu- <br /> distortion. While transmission t complex                   tionary constraints. <br /> females normal, heterozygous t/+ males transmit                       On basis findings TCPJ essential gene <br /> t-bearing chromosome progeny excess                    <span id='am-38' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-39' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-41'  ></span><span id='am-42' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae, isolated conditional <br /> Mendelian expectation. Genetic analysis transmission                   lethal mutation TCPJ confers cold-impaired growth. <br /> ratio distortion suggests depends action          By using fluorescence microscopy analyze mu- <br /> four distorter loci (Tcd-J Tcd4) responder                tant (tcpl-J) wild-type (TCPI) alleles, a <br /> locus (Tcr) (2, 36-38, 56, 63). The molecular basis trans-             fraction cold-impaired tcpl-l cells displayed abnormal <br /> mission ratio distortion remains unknown.                                 nuclear distribution contained multinucleate aploid <br />    A number candidate genes distorter re-                  cells shifted nonpermissive temperature. Fur- <br /> sponder loci, including Tcp-1, isolated t              thermore, cells sensitive wild type <br /> complex (75). Tailless complex polypeptide 1 (TCP-1),                   antimitotic drugs inhibit microtubule polymerization. <br /> protein 57 kDa, expressed large amounts                   Overexpression wild-type TCPI gene led to <br />                                                                           increased sensitivity antimitotic drugs. Indirect immuno- <br />                                                                           fluorescence analysis tcpl-l mutant cells showed that <br />   * <br />       Corresponding author.                                               cells abnormal microtubular structures when <br />   t Laboratory Genetics paper no. 3103.                                shifted nonpermissive temperature. <br />                                                                    2629 <br />  2630     URSIC AND CULBERTSON                                                                                  MOL. CELL. BIOL. <br />  <br />               MATERIALS AND METHODS                                  Gene disruption TCPI. The 3.5-kb EcoRl fragment, <br />                                                                   containing TCPJ gene present clone y22, sub- <br />   Nomenclature. We adopted standard yeast nomencla-           cloned EcoRI site vector YCp5O form clone <br /> ture yeast TCPI homolog: wild-type gene, TCPI;            pDU22. The XhoI HpaI sites vector deleted <br /> protein, TCP1; cold-sensitive mutant allele, tcpl-J.          cutting pDU22 XhoI filling Klenow and <br />   Chemicals. Enzymes chemicals obtained             deoxynucleoside triphosphates, resulting clone pDU30. <br />                                                                   The internal PvuII fragment TCP1 (open reading frame <br /> Promega-Biotec,New England BioLabs, Sigma Chemical                [ORF] sequences bp 236 1361) deleted cutting with <br /> Co., Boehringer-Mannheim, GIBCO-BRL. <span id='am-142' about='obo:NCBITaxon_586402' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3646 obo:NCBITaxon_1 obo:NCBITaxon_4004 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-143' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-145' property="rdfs:label" content="Radiola" datatype="xsd:string"></span><span id='am-146'  ></span>Radiola</span>-                 PvuII adding XhoI linker. The resulting clone, <br /> beled nucleotides ([a-32P]CTP [a-355]ATP) pur-           pDU40, digested XhoI HpaI ligated to <br /> chased Amersham New England Nuclear. The                 YEp13, cut SalI HpaI. <span id='am-89' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-90' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-91'  ></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> clone, <br /> oligomer RD20 (5&apos;-CGACGGCCAGTGAATTCCCC-3&apos;)                    pDU70, carries LEU2 gene coding region of <br /> prepared R. Niece University Wisconsin DNA           TCPI prototrophic leucine uracil. <br /> synthesis facility. Sequenase purchased Stratagene          Growth arrest. Cells grown early log phase 30&#176;C <br /> Cloning Systems. Biotinylated anti-rabbit immunoglobulin G        liquid minimal medium shifted nonpermissive <br /> (IgG) goat obtained Vector Laboratories,         temperature placement rotary water bath 15&#176;C. <br /> Inc. Benomyl gift Du Pont Co. Rat mono-            Cells kept exponential phase growth diluting a <br /> clonal antibodies yeast a-tubulin (YL 1/2; 31) fluores-    fraction fresh 15&#176;C minimal medium. The <br /> cein isothiocyanate-labeled rabbit antiserum rat IgG      number duplicating cells determined plating ap- <br /> a gift G. Borisy purchased Sera-Lab.           propriate dilutions culture YEPD plates 30&#176;C. <br /> Rat monoclonal antibodies mouse TCP-1 gift         Consistently tcpl-l mutant cells visible colonies <br /> K. Willison.                                                      4 days growth, compared 2 days wild <br />    Strains, <span id='am-165' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-166' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-167' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-169' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span>, media. Media growth selec-      type. To count total number cells milliliter well <br /> tion yeast (18) bacterial (73) cells, M13 phage prepa-     nonviable cells, microscopic observations made <br /> ration (41) 5-fluoro-orotic acid (5-<span id='am-11' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-15'  ></span>FOA</span>) medium (3)      hemacytometer. The <span id='am-160' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-161' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-162'  ></span><span id='am-163' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-164' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> total number cells <br /> as described previously. Standard yeast genetic techniques        presented Fig. 6 sampling error + 10 to <br /> were used (58). Thiabendazole (TBZ; 40 mg/ml dimethyl          20%. Absorption measurements optical density 600 <br /> sulfoxide) benomyl (20 mg/ml dimethyl sulfoxide)        nm (OD6.) approximately parallel total numbers <br /> added indicated concentrations appropriate          cells (data shown). Methylene blue (0.05%) added <br /> media. For filter disk assay, log-phase cells overlaid   visualize cells unable metabolize dye. The <br /> with soft agar proper medium plates. The drugs           wild-type strain generation time 16 h, the <br /> were applied filter disks (Schleicher &amp; Schuell)       number mutant cells increased linearly 32 h. The <br /> placed plates. Incubation temperatures           wild type 0.8% cell death analyzed these <br /> indicated.                                                        conditions. <br />    Plasmid DNA manipulations. Plasmid DNA isolation              Construction TCPI null mutation. The homozygous <br /> and transformation <span id='am-114' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-115' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-127'  ></span><span id='am-128' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span>Escherichia coli</span> previ-        diploid strain M10/M11 (Table 1) transformed the <br /> ously described (73). Yeast strains transformed       HindIII fragment (29 bp 5&apos; end vector derived) of <br /> method Ito et al. (29), yeast DNA prepared          pDU70 transplace genomic TCPI gene using <br /> described (58).                                         site-directed integration homologous genomic region <br />    DNA cloning methods. We isolated yeast TCPJ            (50). Leucine prototrophs selected. Two transformants <br /> gene screening genomic centromere-containing library         sporulated, tetrads dissected. <br /> (49), using D. melanogaster Tcp-J cDNA clone                Immunoblotting. Strains DUY1 DUYT1 grown <br /> probe. The hybridization carried low stringency:       minimal medium lacking uracil OD6. 0.7. Soluble <br /> hybridization 55&#176;C 5x SSPE wash 55&#176;C          membranous protein fractions prepared de- <br /> 2x SSC-0.1% sodium dodecyl sulfate (SDS). We obtained             scribed previously (5) 1% deoxycholate used <br /> four independent overlapping clones, clone        instead Triton X-100 isolation membrane- <br /> y22. Consistent previous results (72), 3.5-kb EcoRI      associated proteins. The proteins separated 10% <br /> fragment clone y22 homologous D. melanogaster        SDS-polyacrylamide gels (34) transferred nitrocellu- <br /> Tcp-J gene. This restriction fragment subcloned      lose (70), using Bio-Rad electrophoretic transfer appara- <br /> EcoRI site M13 mpl9 orientations (41). A sequen-       tus 4&#176;C 2 h 200-mA constant current. After transfer, <br /> tial series deletions clones prepared (13).       proteins visualized Ponceau S stain. Mono- <br /> Sequence analysis performed method Sanger           clonal antibody 91a (corresponding amino acids 315 501 <br /> et al. (53). We sequenced overlapping clones orien-       mouse Tcp-J gene; 77) used dilution of <br /> tations.                                                          1/2,000. Immunoblotting performed described by <br />    Genetic mapping TCPI. TCPI assigned chromo-          Bethesda Research Laboratories (immunoselect instruction <br /> some 4 Southern blot hybridization yeast DNA re-            manual catalog no. 9590SA), using biotinylated anti-rat IgG <br /> solved orthogonal field alternation gel electrophoresis        goat streptavidin-alkaline phosphatase, nitro- <br /> (OFAGE). For genetic mapping, strain DUY2 (containing             blue tetrazolium, 5-bromo-4-chloro-3-indolyl phosphate <br /> LEU2 integrated adjacent TCPI gene) crossed         visualize protein. <br /> strain A343-1A (petl4) (19). A total 140 tetrads             Generation TCPI cold-sensitive mutant. We isolated a <br /> analyzed leucine auxotrophy, yielding 101 parental di-        mutation TCPI gene vitro mutagenesis (48), <br /> types, 39 tetratypes, nonparental ditypes map       introduced yeast cells mutant allele <br /> distance calculated 13.9 centimorgans TCPJ          replaced wild-type allele, screened conditional <br /> and petl4. A three-point cross performed DUY2             lethality (6). Strain DUY4a (Table 1), carries chro- <br /> (containing LEU2 integrated adjacent TCPI gene)        mosomal copy tcpl ::LEU2 null allele TCPI and <br /> DUY3 (cdc 37-1 petl4) determined TCPI distal           URA3 centromeric plasmid, transformed the <br /> petl4. <br />  VOL. 11, 1991                                                                                    YEAST HOMOLOG TO MOUSE Tcp-l                                                                      2631 <br />  <br />                                                                   TABLE 1. Strains <br />   Strain                                                             Genotype                                                                                                           Source <br /> M1o                        MATa leu2-3 leu 2-112 ura3-52 trpl-7                                                                                                                W. Boorstein <br /> M10                        MATa leu2-3 leu2-112 ura3-52 trpl-7                                                                                                                 W. Boorstein <br /> DUY4                       MATala ura3-521ura3-52 trpl-71trpl-7 leu2-3 leu2-1121leu2-3 leu2-112                                                                                This study <br />                              TCPl/tcpl::LEU2a <br /> DUY4a                      MATct ura3-52 trpl-7 leu2-3 leu2-112 tcpl::LEU2 {TCPI URA3 yCp5o}b                                                                                  This study <br /> DUY4al                     MATa ura3-52 trpl-7 leu2-3 leu2-112 tcpl::LEU2 {tcpl-l(Cs) TRPI YCpMS38}                                                                            This study <br /> DUY4alr                    MATa ura3-52 leu2-3 leu2-112 trpl-7 tcpl::LEU2 {TCPI URA3 YCp50} {tcpl-l(Cs)                                                                        This study <br />                              TRPI YCpMS38} <br /> PL5                        MATa leu2-3 ura3-52                                                                                                                                 P. Leeds <br /> DUY1                       MATa leu2-3 ura3-52 {URA3 YEp352}                                                                                                                   This study <br /> DUYT1                      MATa leu2-3 ura3-52 {TCPI URA3 YEp352}                                                                                                              This study <br /> LL20                       MATa his3-11 his3-15 leu2-3 leu2-112 CAN]                                                                                                           G. Fink <br /> SRC672-1                   MATa cdc37-1 cyd2 ural                                                                                                                              S. Reed <br /> DUY2                       MATa his3-11 his3-15 CAN) leu2-3 leu2-112 TCPIITCPI::LEU2::YIp26                                                                                    This study <br /> DUY3                       MATa cdc37-1 leu2-3 leu2-112 ural petl4                                                                                                             This study <br /> A343-1A                    MATa leu2-3 petl4 argl Iys7 ural ade3 met6                                                                                                          R. Gaber <br /> MC1066                     leuB600 trpC9830 pyrF74::TnS Kanr <span id='am-84' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="Ara" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-88'  ></span>ara</span> rK- mK-                                                                                                       M. Casadaban <br /> MC1066a                    leuB600 trpC9830 pyrF74::TnS Kanr ara rK- mKj recA13                                                                                                M. Sandbaken <br />   The symbol:: separates locus integrated gene and/or plasmid. <br />   b The braces enclose autonomously replicating plasmid relevant genes. <br />  <br />  <br />  <br />  <br /> in vitro-mutagenized plasmid pDU25-4, centromeric plas-                         The 5&apos; region contains A+T-rich region dyad sym- <br /> mid carrying TRPI TCPI. The URA3 plasmid carrying                           metry consisting 16-nucleotide inverted repeat separated <br /> the wild-type TCPI gene evicted presence                          29-bp spacer region. Poly(dA-dT) upstream promoter <br /> 5-FOA. We isolated cold-sensitive mutant strain DUY4al                      elements type shown necessary and <br /> after screening approximately 2,500 transformants. The                          sufficient constitutive expression (67). In addition, the <br /> screening procedure consisted conditional lethality ei-                   TCPI upstream region contains close matches (6 7 bp) <br /> ther 15 37&#176;C.                                                                (positions 161 227) pheromone response element <br />    Fluorescence microscopy. About 108 logarithmic-phase                         sequences (TGAAACA). These sequences present in <br /> cells washed twice H20, fixed 15 min room                      copies orientation 5&apos; noncoding <br /> temperature 70% ethanol, washed                         regions known pheromone-inducible genes (32, 40). The <br /> H20. The DNA stained 2.5 0.025 ,ug 4&apos;,6&apos;-                        importance elements TCPI induction yet <br /> diamidino-2-phenylindole (DAPI) phosphate-buffered sa-                       analyzed. The 3&apos; untranslated region includes possi- <br /> line (150 mM NaCl, 10 mM KHPO4, pH 7.2) 15 min. The                         ble polyadenylation signal (AATAAA) (47) yeast tran- <br /> DAPI concentration varied factor 10 either <br /> direction stain mitochondria. The DAPI was <br /> washed twice H20, cells suspended                                             A <br /> 50% glycerol. Preparations viewed photographed <br /> on microscope equipped epifluorescence microscopy. <br /> The cells visualized Nomarski optics. <br /> Indirect immunofluorescence using fixed yeast cells                                                                         g                                 I <br /> performed described previously (1, 31)                                                                                Mt4. <br /> cells fixed 5% formaldehyde.                                                                      Ty                            /     TCP1          \ <br />    Nucleotide sequence accession numbers. The GenBank <br /> DNA sequence accession numbers M21159 Droso-                                   B                                            // <br />                                                                                                                                      XC &lt;8 <br />                                                                                                                                                    pu 31          \\ \               Hind m <br />                                                                                                                                                                                           Eco RI <br /> phila Tcp-l M21160 yeast TCPI.                                                  Eco RI                              //v      1aI              I <br />                                                                                                                                                                                .HKICOR <br />  <br />  <br />  <br />                                                                                                                        I                                                       .. <br />                                                                                                                                 I                                  x <br />                                                                                                                       ATG   /                                          \       TGA <br />                                RESULTS                                          C                                      /                                                   \ <br />                                                                                                                                               4 LEU2 <br />   Isolation analysis yeast S. cerevisiae TCPI gene                    \ Etc RI                        s/                       Eco RI                       HPI \/B..HI <br /> and product. The yeast TCPI gene isolated                              .L                         I <br />                                                                                                                                          ., --!!P&apos; -, -, -, <br />  <br />  <br /> centromere-containing genomic shuttle vector library (49),                                            ATG                                                                                TGA <br />  <br /> using D. melanogaster Tcp-J cDNA probe (72).                             FIG. 1. Yeast TCPI gene, flanking regions, construction <br /> Clone y22 (Fig. 1A) contains entire ORF yeast TCPI                       tcpl::LEU2 insertion-deletion. (A) The yeast S. cerevisiae <br /> as flanking regions, including segment 2.2                genomic clone y22, contains TCPI gene flanking <br /> kb upstream TCPI gene, homologous Tyl                        regions. Symbols: _, ORF TCPI; M, region cross- <br /> and 8 element (10; data shown).                                           hybridization Ty element; Ei, possible extent Ty <br />   The cloned segment contains ORF corresponding                           element (extending striped area adjacent EcoRI site). <br />                                                                                 (B) Expanded view 3.5-kb EcoRI fragment containing the <br /> TCPI codes polypeptide 559 amino acids                    TCPI gene. (C) Deletion disruption TCPI gene. The <br /> unmodified molecular size 60,437 kDa (Fig. 2A). The                          1.1-kb region TCPI gene deleted replaced a <br /> promoter region contains TATAA-like sequences                        2.0-kb sequence encoding LEU2 gene. The resulting clone <br /> possible transcription initiation capping sequences (CAAG                       contains bp 1 235 1389 1677 ORF TCPI. Symbols: <br /> and GAAG) (35).                                                                  _, ORF TCPI; M, LEU2 gene. <br />  2632     URSIC AND CULBERTSON                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />    0 tccacgcagtaccctgcataatcatgtattaccataaaaaggctttaatatgagaaaaaggtgagcgctatatagatttagtaatttacatgttacagaagcaaaaagcaaatatatata <br />  120 tattttttatcatattagtttttatcctcagtagtactactgaaataataatctcagatttttttttttcacttgccgagggaccttgtcctaagtggcaa_gaactgaaaaaaaaaaaa <br />  240 attgaaatgcggtggtatcacatgaagagtaaataacttaagcaaaagagaggccactgagagagta_agaaa_gggacaataacaatagacagaagacttttatatttgtaataactta <br />  360 gcgATGTCCCMTTATTCMTMCTCGCGCTCAGATACCCTTTTCCTAGGAGGTGAAAATAGTGGTGATGATATTCGTATCAGAATGTTTTGGCTACTATGGCTGTGGCTACGTC <br />  480 GTCMGTCATCTTTAGGCCCCGTTGGACTGGACAAAATGCTTGTTGATGACATTGGTGATTTTACCGTCACCMCGATGGTGCCACTATTTTGTCTTTGTTAGATGTTCACATCCAGCT <br />  600 GGAAAAATTTTAGTAGMTTGGCTCAACAACAGGACAGAGMATTGGTGACGGTACTACTAGTGTTGTTATTATTGCTTCTGAGTTGTTGMAAGAGCTMTGAATTAGTCMAAATMG <br />  720 ATTCATCCMCTACCATCATAACTGGTTTCAGGGTTGCATTAGAGAGGCTATTCGTTTTATAATGAAGTTCTTTCCACAGTGTTGACACTTTAGGGAAGAGACGTTATTATATA <br />  840 GCCAGACTAGTATGTCCTCTAGATCATTGGTGCTGACTCTGATTTTTTCAGCATATGGTTGTAGATGCTCTTTTGGCTGTTAGACGCAAAATTCCAAAGGAGAATCAGTATCCC <br />  960 GTAAAGGCTGTCMTGTGTTGAAAGCTCATGGTAAATCTGCCACTGAATCACTGTTGGTGCCAGGTTATGCTTTGMTTGTACAGTTGCTTCTCAGCATGCCTAACGTATCGCCGGT <br /> 1080 GGCMTGTGAAAATCGCATGTTTGGATTTGAACTTGCAGAAAGCTCGTATGGCATGGGTGTTCAAATCACATTGATGATCCGGAACAATTGGAGCAATCCGTAACGTGAAGCTGGT <br /> 1200 ATTGTTTTGGAAAGAGTGMGMGATCATTGATGCTGGTGCTCMGTTGTGTTMCCACCAAAGGTATTGATGATTTATGCTTAAAGGAGTTCGTTGAAGCTMGATMTGGGTGTAAGA <br /> 1320 CGCTGTAGAAGGAAGATTTGAGAAGAATTGCACGCGCTACAGGCGCCACTTTGGTTAGTTCTATGTCTACTTAGAGGTGAGGAAACTTTCGAAAGCAGTTATTTAGGTCTTTGTGAT <br /> 1440 GAGTAGTTCAGCCAAATTTTCTGATGATGAGTGTATCTTATTAGGGTACTTCTAAACATTCATCTTCTTCGATCATTTTAAGAGGTGCCACGATTATTCTTTAGATGAGATGGAA <br /> 1560 CGTTCTTTGCACGATTCTCTTTCTGTTGTCAAAAGACGCTTGAAAGTGGAAATGTTGTCCCCGGTGGAGGTTGCGTTGAGCCGCATTGATATCTATTTGGACATTTTGCCACCACA <br /> 1680 GTTGGTTCTCGTGACAGTTAGCTATTGCTGAGTTTGCAGCTGCATTATTATCATTCCAAAAACTTTAGCTGTTACGCTGCCAGGATTCTAGTGAATTAGTTGCAAAATTAGGTCC <br /> 1800 TACCATGCCGCTAGTCAAATGGCCAGCCAGAGGACGTTAAGAGGAGAGCTACAGAAATTACGGCCTAGATTTGATTAGAGGTAAAATTGTGGACGAGATTCACGCTGGTGTTCTTGAG <br /> 1920 CCMCCATMGCAAAGTCAAATCCTTGAAATCAGCATTGGAGGCTTGTGTAGCMTTCTMGAATTGATACMTGATTACAGTGGATCCAGAACCACCCMAGMGATCCGCACGATCAT <br /> 2040 TGAttct t tgcgtaatccgaacgagattctgtatctgcatgtaaactaataaacctatataaaatt taat tatattt tt ttcagcattctat ttt tgaatttgatcgctaactctggg <br /> 2160 cagttgataagcgcttcccagcttttcacgtgcgaggtggactaatacgtaataacgcgtacctattaggacttcgttgagttgtttttcttgtccaaatgcagaagcgggtccctacaa <br /> 2280 actattgtgtttgttcgtaaacgagcaggaaccctgttaaaggaaaaaagcttaacgctagcgttacattttagtaaggcggtaataaaaaagaatgcggagtgtattccatgaattc <br />  <br /> B                              :0 .              .         .    :30 .                     .    :60 . <br />  Saccharomyces cerevi si ae      MSQLFNNSRSDTLFLGGEKI SGDDIRNQNVLATMAVANWKSSLGPVGLDKMLVDDIGDFTVTNDGATI LSLLDVQHPAGKI LVELAQQ <br />  Drosophila melanogaster              MSTLASP <span id='am-150' about='obo:NCBITaxon_208766' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_62771 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6994 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_70905 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_6993 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_33341 obo:NCBITaxon_50557 obo:NCBITaxon_6072 obo:NCBITaxon_6960'><span id='am-151' property="rdfs:label" content="Sia" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-154'  ></span>SIA</span> TRQ RRIV T M ALSIS I                         V          R E E A V        L <br />  Mus muscutus                            MEGP SVF DRST EAV S M AASI I        F                 V I        K E E A V C DL <br />  :90       .         :     .    :120.                           :150           .   :      .    :180 <br />  QDREIGDGTTSWI IASELLKRANELVKNKIHPTTI ITGFRVALREAIRFINEVLSTSVDTLGKETLINIAKTSMSSKI IGADSDFFSNMVVDALLAVKTQNSKGEIKYPVKAVNVLKAH <br />    E V         L A     N D   Q      S  S Y  I CK   CKY S H TAP  E  RDS                 AE    A      AQS ITDPR QAV SI <br />    K V           A     N D   Q      SV S Y L CK V Y      N IINT E RDC T              ING Y A        V   YTDAR QPR NS I <br />  :210 .    :    .    :     .    :240 .     :    .     :     .    :270 .      :     .    :    .    :300.       :    .    : <br />  GKSATESLLVPGYALNCTVASQAMPKR IAGGNVK IACLDLNLQKARMAMGVQI N IDDPEQLEQ IRKREAGI VLERVKKI IDAGAQVVLTTKG IDDLCLKE FVEAKIMGVRRCKKEDLRR I <br />      R V I         I   Q K --V A        H FS TK K       VL N DK A A LD TK INM LGT VN VSG V                MY      GA A V S KI <br />   R QI M IN       V G G       --V A       FS TK KL W T K D Q SD TK IQ LAT N I                     G     MY Y      GA A VL R KHV <br />  :330.      :    .    :    .    :360.     :    .    :     .     :390.     :    .    :    .    :410.     : <br />  ARATGATLVSSMSNLEGEETFESSYLGLCDEVVQAKFSDDECI LIKGTSKHSSSSI I LRGANDYSLDEMERSLHDSLSWKRTLESGNVVPGGGCVEAALNIYLDNFATTVGSREQLAIA <br />   K    AFIT LT MD S DA MV EAA A ERIC L               KARAAA      P FYC        V A C    V KK R          S E      SLA <br />   K S SIL TLA           VTM QAE     ERIC L         N KART A          FMC         A C   V LKS      A    S E Y NM <br />  :450.      :    .    :    .    :480.      :     .     :    .     :510.    :     .   :   .    :540.    : <br />  EFAAALLI IPKTLAVNAAKDSSELVAKLRSYHAASQMAKPEDVKRRSYRNYGLDLIRGKIVDEIHAGVLEPTISKVKSLKSALEACVAI LRIDTMITVDPEPPKEDPHDH* <br />     KS V       S      ATD         NS T-      --- SDLKWT       E WR NKK       AM I    F T AIT     D KLN DKSGKSYADACAAGELDG* <br />     RS V N          Q TD       AF NEA V-N --- KNLKWI        VH PR NKQ    F     V     F T AIT F DL KLH SKDDKHGSYENAVHSGALDD* <br />    FIG. 2. Nucleotide amino acid sequences TCPI. (A) Sequence yeast TCPI DNA flanking regions. The sequence <br /> encoding ORF capital letters beginning presumed translation start site nucleotide position 364 termination <span id='am-94' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-95' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-96'  ></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> <br /> at nucleotide position 2041. The 5&apos; 3&apos; flanking regions indicated small letters. The poly(dA-dT) inverted repeat nucleotide <br /> positions 179 194 224 239, TATA-like sequences, CAAG, closely related sequences near 5&apos; end transcripts underlined. <br /> Transcription termination sequences TAG.. . TATGT.. . TTT poly(A) addition sequence underlined. (B) Comparisons of <br /> the deduced amino acid sequences S. cerevisiae, D. melanogaster, <span id='am-1' about='obo:NCBITaxon_10090' typeof='owl:Thing obo:NCBITaxon_314147 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_337687 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_862507 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_39107 obo:NCBITaxon_10088 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_10066 obo:NCBITaxon_6072 obo:NCBITaxon_33553 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9989 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-2' property="oboInOwl:hasExactSynonym" content="mouse" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Mus musculus" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="mice C57BL/6xCBA/CaJ hybrid" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasAlternativeId" content="NCBITaxon:85055" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Mus muscaris" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="house mouse" datatype="xsd:string"></span>Mus musculus</span> TCP1 proteins. The differences amino acid <br /> sequences relative yeast polypeptide shown. Termination codons indicated asterisks. To maximize number matches, <br /> gaps (-) inserted Drosophila mouse TCP-1 sequences. <br />  <br />  <br /> scription termination sequences (TAG... .TATGT. . TTT)                  genomic DNA digested number restriction <br /> (81).                                                                   endonucleases analyzed Southern blotting. Analo- <br />    Hydropathy profiles (33) yeast,                gous findings Drosophila Tcp-J (72), results <br /> Drosophila, mouse TCP1 homolog contains extended                    showed yeast TCPI present copy haploid <br /> hydrophobic domains, suggesting integral              genome (Fig. 3). To determine TCPI codes an <br /> membrane proteins (data shown). In addition, TCP1                   essential gene, null allele constructed replacing the <br /> polypeptides lack apparent signal sequence (44, 74).                 internal PvuII fragment TCPJ coding region the <br />    The nucleotide deduced amino acid sequences                 LEU2 gene (Fig. 1) transplacing diploid strain <br /> yeast, Drosophila, mouse TCP1 share extensive                    (leu2-3 leu2-11211eu2-3 leu2-112; strain M10/M11; Table 1). <br /> overall identity. The yeast polypeptide identity              The growth sporulation efficiency tcpl::LEU21 <br /> 61% Drosophila mouse polypeptides, respec-                 TCPI heterozygote comparable parental <br /> tively, amino acid sequence identity                diploid. Each tetrad independent Leu+ diploids <br /> the mouse Drosophila proteins 72% (Fig. 2B). The                 produced nonviable spores terminated cell division <br /> nucleotide sequences homologs share                spore germination (13 19), (5 of <br /> 58 66% identity.                                                    19), (1 19) cells. The viable spores each <br />     TCPI single-copy essential gene chromosome 4. To            tetrad Leu- phenotype; thus, lethal phe- <br /> establish copy number TCPI gene, wild-type yeast             notype genetically linked tcpl ::LEU2. <br />  VOL . 1 l, 1991                                                                               YEAST HOMOLOG TO MOUSE Tcp-l                      2633 <br />  <br />                            0-                                                                             membrane       soluble <br />                            E                                                                               + <br />                                                                                                                          4- <br />  <br />                                                                                                     kd, <br />                        C   U,                 c <br />  <br />                        ?        Tetrods   ~             Tetrodsl                                    10-                  - <br />     Leu Phenotype      -   +    -         _             _ <br />  <br />  <br />  <br />  <br />               kb                                                         kb <br />  <br />                                                                                                    60- -_ <br />                                                                          6.6 <br />                                  s0 <br />  <br />                                                                                                    42-                        _ <br />               354. <br />               2.6           _                                            2.6 <br />                                               v   -,*   40M     i-   - <br />                                                                          2 0 <br />  <br />                   .7 <br />  <br />                                                                           1.1                                      2     3         4 <br />                                                                                                           Anti   mouse  TCP-I <br />                                                                                    FIG. 4. Western blot analysis wild-type TCPI-overex- <br />                                                                                 pressing cells. Yeast wild-type TCPI multicopy plasmid strains <br />                                                                                 differentially extracted yield soluble membranous <br />                                                                                 fractions. The proteins separated 10o SDS-polyacryl- <br />                                                                                 amide gel. Lanes contain protein extracts cells wild type for <br />                                                                                 TCPI (DUY1; -) overexpressing wild-type gene a <br />                                                                                 high-copy-number plasmid vector (DUYT1; +). Immunoblotting <br />                                                                                 probing rat anti-mouse TCP-1 monoclonal anti- <br />                                                                                 bodies biotinylated goat anti-rat IgG streptavidin coupled to <br />                             2 3 4         5       6         7        8          alkaline phosphatase. The 42- 110-kDa bands represent binding <br />                             Eco RI            Pvu        II <br />                                                                                 secondary antibodies present absence primary <br />                                                                                 antibodies. <br />   FIG. 3. Southern analysis yeast DNA strains trans- <br /> formed tcpl::LEU2 deletion-insertion construction. Yeast <br /> DNA isolated untransformed parent (MlO/M11), a <br /> diploid transformant (DUY4), viable progeny spores                  TCPJ chromosome 4. To map TCP1 locus, copy <br /> from diploid. The DNA digested EcoRI PvuII,                 LEU2 gene integrated TCPI provide a <br /> Southern blots analyzed probe 3.5-kb                  convenient marker genetic mapping. Genetic linkage was <br /> EcoRI fragment containing entire TCPI sequence.                             detected cross heterozygous LEU2 chromo- <br />                                                                                 4 marker petl4. Tetrad analysis revealed 13.9- <br />                                                                                 centimorgan linkage TCPI ::LEU2 petl4 a <br />    Southern blot analysis DNA isolated parental                     32.6-centimorgan linkage TCP1::LEU2 cdc37. <br /> diploid, tcpl ::LEU21TCPI transformant, viable spore                      Analysis double crossovers intervals confirmed <br /> colonies indicated wild-type null allele                  TCPI distal cdc37. The map position indicates that <br /> present heterozygote, wild-type spores                     TCPI identical previously known gene. <br /> recovered tetrads (Fig. 3). Correct transplacement                     The 60-kDa yeast TCP1 protein shares epitope with <br /> inferred Southern blot analysis PvuII-digested                      mouse TCP-1. To determine cellular distribution of <br /> DNA, yielded predicted restriction endonuclease                       TCP1, soluble membrane fractions analyzed by <br /> pattern transformant (Fig. 3).                                          Western immunoblotting strains overexpress <br />    We showed TCPI required vegetative                          TCP1 wild-type strains. Monoclonal antibodies to <br /> growth. First, tcpl: :LEU21TCP1 heterozygote (strain                          mouse TCP-1 antigenic epitope spanning amino acids 315 <br /> DUY4; Table 1) transformed pDU22, centro-                            501 (77) cross-react yeast 60-kDa protein detected <br /> meric plasmid carrying URA3 TCPI. Following sporu-                          strain overexpresses TCP1. In assay, 90 to <br /> lation, tetrads segregated 0:2, 1:2, 2:2 Leu+                   95% antigen detected membranous fraction, <br /> Leu- phenotypes. No Leu+ Ura- colonies observed.                           remainder nonsedimentable cytosolic fraction <br /> This result indicates viability cells carrying                      (Fig. 4). When membranes overexpressing cells <br /> tcpl::LEU2 allele maintained pDU22. During re-                           washed 1 M NaCl 0.1 M borate <br /> peated subculturing nonselective medium, strains carrying                    buffer (pH 11), TCP1 remained membrane bound (data not <br /> the tcpl null allele lost plasmid, strains                    shown). We unable detect equivalent 60-kDa <br /> carrying TCPI allele readily lost plasmid. In addition,                 protein wild-type strain. <br /> strains carrying tcpl null allele viable                       These results indicate TCP1 probably membrane <br /> absence wild-type extrachromosomal plasmid carry-                        associated provided overexpression does cause <br /> ing TCPI gene, indicated sensitivity 5-FOA                   mislocalization. The lack detection TCP1 wild-type <br /> (3). Thus, attribute lethality tcpl ::LEU2 allele                  cells reflect low affinity mouse antibodies <br /> loss function essential vegetative growth.                             yeast protein indicate TCP1 present low <br />    Data OFAGE hybridization mapping suggested                         abundance conditions. <br />  2634     URSIC AND CULBERTSON                                                                                              MOL. CELL. BIOL. <br />  <br />                 37      30         23             ;5 <br />            A <br />                                                        Tf p <br />  <br />  <br />  <br />                        ~        m                                           50 <br />  <br />  <br />                                                        -, /-,./   r <br />                                                                        Z- <br />  <br />                 * <br />                                                                        1- <br />  <br />  <br />                      Benorr.y                L- <br />                                                                        C) <br />            B <br />                                 :~~o                                   E0   20 <br />  <br />                          . IDIt.&apos;-&apos;il   :1 <br />  <br />                                                                             10 <br />  <br />                        Be&apos;cr&gt;           T&apos; Z <br />            C <br />                                                                                         20         40        60         S21 In <br />                                                                                                                          1&apos;.t <br />                         (O&quot;Y{                                                                    hours shift t   (1 <br />                                                                                                                        ! C <br />                                                                           FIG. 6. Area graph showing tcpl-l mutant cells form buds <br />                                                                        unable duplicate. tcpl-l mutant cells (DUY4al) grown <br />                                                                        exponentially minimal medium 30&#176;C shifted time zero to <br />    FIG. 5. Growth presence absence TBZ               15&#176;C. At various time points, samples taken analyzed for <br /> benomyl. (A) Logarithmic-phase cells phenotypic wild-type       cell viability, cell inviability, ability undergo cell duplication <br /> strains DUY4a {TCPI} (see Table 1, footnote b, explanation      30&#176;C incubation 15&apos;C (see Materials Methods). The <br /> braces) DUY4ar {TCPI} {tcpl -l} mutant strain DUY4al       black area shows numbers duplicating cells judged their <br /> {tcpl-l} (Table 1), grown YEP medium 30&#176;C spotted         ability form colonies YEPD. The gray area shows numbers <br />                                                                        viable cells (cells able metabolize methylene blue) <br /> OD6w 0.4 YEP plates. Incubation 37, 30, 23, 15&apos;C       aside duplicating cells. The striped area depicts the <br /> as indicated. (B) Overnight cultures strains DUY4a {TCPI} and <br /> DUY4al {tcpl-1} (Table 1), grown YEP medium 30&#176;C             numbers inviable cells judged methylene blue staining. <br /> overlaid soft agar YEP plates. Incubation 2 days <br /> at 23&#176;C. (C) Wild-type (DUY1) TCPJ-overexpressing (DUYT1) <br /> cells, grown selective medium 30&#176;C overlaid soft agar <br /> onto selective medium plates. Incubation 2 days 37&#176;C.       sensitivity (see below) cosegregate tcpl-l mu- <br /> The drug disks panels B C contained 1.6 mg TBZ 0.4 mg     tation recessive. <br /> of benomyl.                                                               The tcpl-l mutation affects cell duplication, nuclear migra- <br />                                                                        tion, sensitivity antimitotic drugs, microtubular struc- <br />                                                                        tures. Cells carrying tcpl-l allele arrest a <br />   The tcpl-l mutation confers cold-sensitive growth. On            morphologically homogeneous population shift to <br /> basis experiments showing TCPI essential             restrictive conditions. Budded cells variable size were <br /> gene, isolated allele TCPI confers cold-                 observed different stages cell cycle. <br /> sensitive growth (see Materials Methods). To confirm                  To precisely determine effects tcpl-l the <br /> that mutation, designated tcpl-J, TCPI              viability cells, examined tcpl-l cells (DUY4al) and <br /> gene, separate tests made: complementation,                 phenotypic wild-type strain (DUY4a) various times after <br /> coinstability, plasmid rescue. To test complementation,            shift restrictive temperature 15&#176;C (see Materials <br /> the cold-sensitive mutant strain DUY4al (Table 1)                  Methods). The mutant cells linearly increased cell <br /> transformed centromere-containing URA3 vector                   number, generation time 32 h minimal <br /> carrying entire wild-type TCPI gene (pDU22). In              medium (Fig. 6). However, number duplicating cells <br /> transformant analyzed, wild-type (cold-resistant) pheno-           remained constant 100 h decreased. The <br /> type restored (Fig. 5A). To test coinstability,        number inviable cells, judged inability to <br /> transformants screened loss URA3-contain-              metabolize methylene blue, increased linearly 3.5 to <br /> ing plasmid selecting resistance 5-FOA. In               30% termination experiment 119 h. The <br /> cases, resistance coincided restoration cold-              remainder cell population metabolically viable as <br /> sensitive phenotype. Finally, recovered plasmid                 judged methylene blue staining genetically inviable, <br /> encoding tcpl-l mutation cold-sensitive mutant            cells unable form colonies shifted <br /> strain DUY4a1 (pDU5-4). This plasmid reintroduced                  permissive temperature. These results suggest the <br /> into diploid strain DUY4 (Table 1). pDU5-4 restored            tcpl-l mutation causes cells form buds unable <br /> mutant phenotypes, exemplified meiotic segregants             undergo cell duplication. <br /> this transformant. Only Trp+ Leu+ prototrophic spores                 To observe cell morphology nonpermissive temper- <br /> showed cold-sensitive growth phenotype addition              ature, examined asynchronous cultures wild-type <br /> benomyl sensitivity (see below).                                       strain (M10), phenotypically wild-type strain (DUY4a), <br />    These results demonstrate wild-type TCPJ gene              tcpl-l mutant strain (DUY4al) fluorescence <br /> can supply function lacking mutant. Furthermore,            microscopy DAPI staining. Exponentially growing cells <br /> these findings indicate cold sensitivity benomyl              shifted permissive temperature (30&#176;C) the <br />  VOL . 11, 1991                                                                                      YEAST HOMOLOG TO MOUSE Tcp-l                            2635 <br />  <br />   TABLE 2. Percentage cell types wild-type (M10), phenotypic wild-type (DUY4a), tcpl-l mutant (DUY4al) cells grown at <br />                                                 30&#176;C time zero shifted 15&#176;C <br />                                                                                                Nuclei                Nuclei in <br />                                                            Morphologyb                     large-budded                 unbudded <br />                                                                                                cellsc <br />                                                                                                                            celd- <br />  <br />                            Genotype OC Hoursk <br />                                                          (D (           \-../    \M/ \.j \.,/ &apos;z.Y %Z. <br />                                                                                                                         eo <br />                               TCP1        30      0       33      37     30        0      23      77       0    0         0      0 <br />                                           15     48       41      29     30        0      18      82       0    0         0      0 <br />                          tCpl {TCP1} 30           0       38      28     35        2      19      77       0    1         1      1 <br />                                           15     48       36      29     35        6      27      65       2    0         2      2 <br />                          tcpl {tCpl-1} 30         0       48      20     32       15      15      67       0     3        1      1 <br />                                           15     4        33      18     48       50       7      29       2    12        3      3 <br />                                           15      6       27      19     54       47       8      37       5     3        4      3 <br />                                           15      8       31      20     49       52       7      29       5     7        3      2 <br />                                           15     17       25      18     57       49       5      27       10    9        4       6 <br />                                           15     46       24      24     52       51       7      23       9    11        9      7 <br />                                           15     70       31      17     52       52       9      18       6    15       13      8 <br />                                           15     96       49      16     35       58      4       13       8    17       21      18 <br />   Hours incubation shift 30 15&#176;C. The cell generation time minimal medium 16 h wild type 32 h mutant. <br />   b Percentages unbudded, small-budded, large-budded cells. <br />   I <br />     Percentages relative large-budded cells. <br />   d Percentages relative total population. <br />  <br />  <br />  <br /> nonpermissive temperature (15&#176;C). At time points                         proportion cells containing multiple nuclei increased with <br /> shift, cell types position appearance nuclear                       length incubation nonpermissive temperature. <br /> DNA examined (Table 2). For wild-type strains,                        Furthermore, 10% tcpl-l mother cells two <br /> distribution morphological <span id='am-74' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-75' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> unbudded,                             buds, cases lacked nuclei (Fig. 7C F). <br /> small-budded, large-budded cells approximately                         After generation times 15&#176;C, cells <br /> same. This contrast increase proportion                     large misshaped. They large amounts of <br /> large-budded cells tcpl-l mutant strain early 4 h                     DAPI-staining particles, making impossible unambig- <br /> after shift, time one-half culture                       uously score various cell types. Despite that, were <br /> consisted large-budded cells appeared larger                          able discern cells having large DAPI bodies sugges- <br /> than wild type (Fig. 7). In S. cerevisiae, mutations                           tive nuclei cell body. <br /> affecting genes involved DNA synthesis nuclear migra-                           We observed multinucleate cells wild type. <br /> tion cells treated nocodazole, inhibitor micro-                      However, 1 4% phenotypic wild-type <br /> tubular assembly, arrest similar manner predomi-                         cells (DUY4a) multiple nuclei. This strain carries a <br /> nantly large-budded cells (25, 27, 28, 30).                                        chromosomal tcpl null mutation complemented centro- <br />    We determined number position nuclear chro-                          meric plasmid harboring wild-type TCPJ gene. It is <br /> mosomal DNA large-budded cells. Comparison                             possible differences nuclear morphology between <br /> the tcpl-l mutant wild-type strains revealed prominent                         wild-type phenotypic wild-type strains to <br /> differences (Table 2). At 30 15&#176;C,                            differences levels expression TCP1 the <br /> wild-type large-budded cells divided segregated                            plasmid versus chromosome. <br /> their chromosomes mother daughter cells. In                          Mutations genes affect microtubule function some- <br /> (wild type) (phenotypic wild type) remaining                        times confer multinucleate phenotype (e.g., spal-l cin <br /> large-budded cells, chromosomal DNA localized                           mutations) (25, 65, 66). For reason, tested effects <br /> the mother-bud neck. In contrast, half large-                         drugs, benomyl TBZ, strains carrying wild- <br /> budded tcpl-l mutant cells mononucleate 15C                            type tcpl-l alleles. These drugs belong class of <br /> contained chromosomal DNA localized random positions                            benzimidazole fungicides specifically inhibit assem- <br /> in cells. Even permissive temperature                        bly cytoskeletal structures <span id='am-45' about='obo:NCBITaxon_4751' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-46'  ></span><span id='am-47' property="rdfs:label" content="Fungi" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="fungi" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="fungi" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="PMID:11062127" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="PMID:12684019" datatype="xsd:string"></span>fungi</span> (14, 79). <br /> more mononucleate tcpl-l cells cells wild-type                         Benzimidazole fungicide sensitivity tested the <br /> strains.                                                                           semipermissive temperature tcpl-l 23&#176;C, using both <br />    In addition, high percentage mutant cells                         colony formation (not shown) filter disk inhibition as- <br /> unbudded, DAPI-staining center                           says (Fig. SB). The phenotypic wild-type strains <br /> or visible nuclear DNA staining. The                            (DUY4a DUY4a1r; Table 1) failed form colonies at <br /> class presumably multinucleate. The class presum-                        TBZ concentrations excess 30 ,ug/ml, the <br /> ably arose cytokinesis large-budded mononucle-                         tcpl-l strain sensitive failed form colonies <br /> ate multinucleate cells lacking chromosomal DNA                         TBZ concentrations excess 12.5 ,ug/ml. Similar <br /> bud. The numbers nuclei cell body                                results obtained benomyl, colony <span id='am-147' about='obo:NCBITaxon_forma' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-148' property="rdfs:label" content="forma" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>forma</span>- <br />  2636     URSIC AND CULBERTSON                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                                                     I <br />  <br />  <br />  <br />  <br />   FIG. 7. Abnormal nuclear distribution tcpl-l cells, shown microscopic analysis phenotypic wild-type (DUY4a) mutant <br /> (DUY4al) cells shifted fromn 30 15&#176;C 22 h (about 1.3 generations wild type). The photographs cell morphology as <br /> demonstrated Nomarski optics (A, C, E) cellular DNA revealed DAPI fluorescence (B, D, F). (A B) Typical <br /> large-budded wild-type cells; (C D) mononucleate tcpl-l cells large bud (left) buds (right); (E F) binucleate tcpl-l <br /> mother cell anuclear buds. <br />  <br />  <br /> tion inhibited concentrations excess 25 ,ug/ml         increased steady-state level TCP1 protein determined <br /> the wild-type strains 12.5 ,ug/ml tcpl-l strain.            Western blotting, suggest increased intracel- <br /> These results confirmed using filter disk assays             lular concentration protein overexpression of <br /> which radius growth inhibition surrounding drug-             gene likely cause hypersensitivity. The <br /> containing disk reflects degree sensitivity drug.         fact overexpressing wild-type TCPI causes increased <br /> The tcpl-l strain consistently gave greatest radius              sensitivity benomyl indicates drug sensitivity <br /> inhibition, indicating increased sensitivity (Fig. SB). We              conferred tcpl-l simply aberration mutant <br /> showed tcpl-l mutation causes altered drug sensi-              probably reflects role wild-type protein the <br /> tivity demonstrating drug-sensitive cold-               drug-sensitive assembly microtubular components. <br /> sensitive phenotypes cosegregate crosses (see above).                   Microtubules present nucleus the <br />    We examined effects temperature TCPI                 cytoplasm cell cycle. They nucleate from <br /> gene copy number drug sensitivity using colony               spindle pole bodies, cylindrical multilaminate <br /> formation disk sensitivity assays. Strains carrying               organelles, embedded nuclear envelope (30, 51). The <br /> single wild-type copy TCPI chromosome 4 typi-                 intranuclear microtubules implicated essential in <br /> cally insensitive TBZ benomyl 23 30&#176;C              mitosis, nuclear elongation, division, cyto- <br /> drug concentrations analyzed. However, growth                  plasmic microtubules required migration the <br /> strains slightly inhibited 15 37&#176;C concentrations          dividing nucleus neck mother daughter <br /> in excess 1.6 mg TBZ disk 0.4 mg benomyl               cell bodies (9, 27, 28, 30, 46). It suggested that <br /> per disk filter disk assays (Fig. 5C). By contrast,                spindle pole body separation dependent intranuclear <br /> wild-type strain carrying TCPJ multicopy plasmid               microtubules, cytoplasmic microtubules, (30). <br /> found hypersensitive 37&#176;C benomyl                 We used indirect immunofluorescence a-tubulin- <br /> to TBZ drug concentrations (Fig. SC). The differ-              specific antibodies visualize cellular microtubules the <br /> ence drug sensitivity reflect differences site       phenotypic wild-type (DUY4a) tCpl-l mutant (DUY4al) <br /> action benomyl TBZ (14). Since strain contains           cells (31). The microtubules wild-type mutant cells <br />  VOL . 1 l, 1991                                                                        YEAST HOMOLOG TO MOUSE Tcp-l                      2637 <br />  <br />  <br />  <br />  <br />   FIG. 8. Abnormal nuclear distribution aberrant microtubular structures tcpl-l mutant cells. Photographs indirect immuno- <br /> fluorescence fluorescence microscopy a-tubulin nuclear-stained phenotypic wild-type (DUY4a) tcpl-l mutant (DUY4al) cells <br /> shifted 30 15&#176;C. Each pair photographs shows cell stained DAPI (left set) anti-ax-tubulin antibody (right of <br /> each set). (A D) Normal proliferating phenotypic wild-type cells; (E P) tcpl-l mutant cells. (A B) A large-budded cell short <br /> spindle chromosomal DNA mother-bud neck. (C D) The cell completed nuclear division, cytokinesis yet <br /> occurred microtubules intact. (E F) The cell nuclei aploid bud fanned tubulin-staining <br /> arrays extend. (G H) One nuclei partitioned mother bud. The connecting antitubulin-stained bundles extend <br /> through nuclei bud. (I J) An unbudded cell sizable microtubular bundles. (K L) The upper cell has <br /> equal numbers nuclei tubulin-staining structures suggestive spindles. (M N) The cell large DAPI-staining bodies on <br /> opposite poles cell extensive microtubular bundles. (O P) Nuclei randomly positioned binucleate mother cell an <br /> aploid bud. We overstained cells DAPI visualize DAPI-staining particles, presumably mitochondria. Microtubules connect the <br /> nuclei cell extend bud. <br />  <br />  <br /> grown 30&#176;C indistinguishable (not shown),                extensive, frayed, extending cell. <br /> the microtubules phenotypic wild-type cells grown 15&#176;C             In addition, multinucleate cells numbers antitu- <br /> (Fig. 8A D). After shift 15&#176;C, number mutant            bulin-staining structures, suggestive spindles, equal or <br /> cells having morphologically abnormal microtubular struc-                number nuclei. Figures 8E F <br /> tures increased time exposure cells                multinucleate cell sets fanned <br /> restrictive temperature. The increase occurred parallel               microtubular arrays expand anucleate bud. Figures <br /> with increase numbers multinucleate anucleate              8M P abnormal microtubular bundles connecting the <br /> cells described previously. Figures 8E P range            nuclei cell body. In tcpl-l mutant cells, <br /> of different types microtubular structures               microtubular structures positioned randomly budded <br /> mutant cells. About 5 10% cells strikingly                 cells oriented budding site (Fig. <br /> aberrant microtubular structures appeared                80 P). After generation times, approxi- <br />  2638     URSIC AND CULBERTSON                                                                                  MOL. CELL. BIOL. <br />  <br /> mately half cells abnormal microtubular              wild-type cells benomyl TBZ semipermissive <br /> distribution. These phenotypes likely            temperatures. Benzimidazole compounds believed to <br /> consequence cell death, 80% cells           function binding tubulin, shifting assembly <br /> viable judged methylene blue staining. The numbers        monomer form (14, 79). These drugs block <br /> tcpl-l mutant cells grown 15&#176;C antitubulin             processes thought dependent or <br /> binding structures comparable numbers phe-          mediated microtubules, including nuclear migration (43, <br /> notypic wild-type cells. Thus, TCP1 does appear             46), chromosome segregation (80), karyogamy (16). <br /> affect microtubular stability.                                        Mutations fungi confer resistance high concen- <br />                                                                    trations benomyl map genes encoding p-tubulin (24, <br />                         DISCUSSION                                 42, 57, 69). By contrast, mutations confer benzimidazole <br />                                                                    hypersensitivity map genes encoding a- and <br />   Evolutionary conservation TCP1 structure. TCP1,            P-tubulin genes, CIN1, CIN2, and <br /> abundant mouse protein spermatogenesis,             CIN4 (27, 66, 71). TBZ hypersensitivity conferred by <br /> present human, Drosophila, yeast cells. We isolated         mutation <span id='am-54' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-58'  ></span>Schizosaccharomyces</span> pombe cdc13 gene, <br /> the yeast TCPI homolog utilizing sequence identity         product required entry mitosis (4). Our <br /> the D. melanogaster Tcp-J gene similar       results genetic mapping, DNA sequencing, drug <br /> DNA amino acid sequence length           sensitivity studies TCPJ represents newly iden- <br /> gene protein. Monoclonal antibodies mouse               tified gene functionally related distinct the <br /> protein cross-react yeast polypeptide. Evolutionary       tubulin, CIN, cdc genes. <br /> conservation TCP1 structure result func-              Also, overproduction wild-type TCPJ gene product <br /> tional requirement specific protein-protein interactions       causes benomyl hypersensitivity 37&#176;C does affect <br /> that place constraints variation possible      growth presence TBZ temperature. The <br /> sequence.                                                          benomyl hypersensitivity phenotype conferred increasing <br />    Our cell fractionation experiments shown        intracellular concentration wild-type product <br /> conditions overexpression TCP1 membrane associated,          indicates drug sensitivity associated tcpl-J <br /> yet deduced amino acid sequences genes          mutation phenotype peculiar mutant tcpl-J <br /> three organisms contain membrane-spanning               product indicative function wild- <br /> regions sequences conserved homologs known         type product. The differential sensitivity benomyl ob- <br /> involved modifications N-linked glycosylation (26)      served strains overexpress wild-type gene product <br /> or covalent bonding palmitic acid anchor protein      result differences cell permeability metab- <br /> the membranes (8, 78). If membrane association true      olism drugs. Alternatively, results indicate a <br /> reflection TCP1 protein, mislocalization        direct effect interaction drugs microtu- <br /> due overexpression, protein anchor      bules. <br /> membranes binding molecular component.               Deleterious phenotypes associated overexpres- <br />    Strains carrying cold-sensitive tcpl-l mutation         sion yeast cytoskeleton-related gene products. In <br /> sensitive benomyl TBZ grown semiper-            extreme case, overexpression causes lethality (7, 17, 25, <br /> missive temperature. Since biochemical genetic experi-         48). These results indicate cytoskeletal protein function <br /> ments led conclusion tubulin          sensitive concentration proteins. The TCP1 <br /> intracellular target benzimidazole compounds (14, 15, 24,      gene product behaves similar manner. It conceivable <br /> 57, 69), TCPI encode protein functionally            effects antimitotic drugs TCPJ overexpres- <br /> structurally interacts microtubules spindle pole bod-      sion synergistic: benomyl decreases tubulin polymer <br /> ies, nucleating point microtubules. The result        concentration, elevated levels TCPI product may <br /> overexpression TCPJ causes hypersensitivity             exclude essential components nonfunctional assem- <br /> benomyl suggest stoichiometry TCPI gene            blies. <br /> product critical function.                                  We shown tcpl-J mutant cells produce metabol- <br />    The isolation mutation TCPI confers cold-          ically active buds judged methylene blue staining, yet <br /> impaired growth indicate TCPI encodes protein         buds unable undergo cell division. We <br /> that participates assembly oligomeric structures.        presume unbudded aploid cells large- <br /> Cold-sensitive mutations generally destabilize hydrophobic         budded cells lacking nuclei buds account of <br /> interactions result failure mutant protein   genetically inviable cells. In addition, possibility <br /> to assemble multimeric structures.                            ruled fraction daughter cells did <br />    Possible function TCP1. Several lines evidence sug-       receive complement chromosomes contained <br /> gest TCP1 affect microtubules. By examining           abnormal microtubular structures, prevented normal <br /> interactions nuclei microtubules tcpl-l mutant cells     cell duplication. <br /> with use fluorescence indirect immunofluorescence              About one-third tcpl-J mutant cell contained aber- <br /> microscopy, noted half cells           rant nuclear distribution anti-a-tubulin binding struc- <br /> population contained DAPI-staining bodies              tures, presumed microtubules, early hours <br /> presumed multiple nuclei, lacked            shift restrictive temperature. The abnormal- <br /> nucleus restrictive temperature 15&#176;C. In addition,       ities increased approximately one-half cell popula- <br /> some cells contained multiple antitubulin-staining bundles,        tion cell generations. In comparison, propor- <br /> suggestive multiple spindles. Often microtubular            tion viable cells cell cultures ranged 96 to <br /> structures appeared located randomly, deformed          80% cell generation times. Thus, kinetics of <br /> in structure, developed fanned arrays, looked             appearance mutant phenotypes suggest a <br /> expansive microtubules cells grown            direct consequence cold-impaired tcpl-J-encoded <br /> permissive temperature wild-type cells.                      protein function delayed, indirect progression of <br />    Furthermore, tcpl-l mutant cells sensitive       cell death. <br />  VOL.   11, 1991                                                                        YEAST HOMOLOG TO MOUSE Tcp-l                      2639 <br />  <br />   The simplest model provides explanation                12. Chesley, P., L. C. Dunn. 1936. The inheritance tailless- <br /> findings TCP1 affects microtubules involved               ness (anury) house mouse. Genetics 21:525-536. <br /> correct positioning spindle pole bodies. This               13. Dale, R. M. K., B. A. McClure, J. P. Houchins. 1985. A <br /> account random position nuclei microtubules,                  rapid single-stranded cloning strategy producing sequential <br />                                                                              series overlapping clones use DNA sequencing: appli- <br /> for abnormal nuclear distribution microtubular struc-                cation sequencing corn mitochondrial 18S rDNA. Plasmid <br /> tures, lack terminal phenotype                       13:31-40. <br /> restrictive temperature. Consistent model,             14. Davidse, L. C. 1986. Benzimidazole fungicides: mechanisms of <br /> note nocodazole-treated cells characteristics sim-                 action biological impact. Annu. Rev. Phytopathol. 24:43- <br /> ilar tcpl-l mutant cells. Depletion microtubules                65. <br /> this drug inhibits orientation nucleus spindle            15. Davidse, L. C., W. Flach. 1977. Differential binding of <br /> pole bodies bud migration (30).                 methyl benzimidazole-2-yl carbamate fungal tubulin a <br />    A number cell cycle mutations necessary nuclear                    mechanism resistance antimitotic agent mutant <br /> division (cdc14 cdclS), spindle pole duplication (cdc3l),                strains <span id='am-68' about='obo:NCBITaxon_5052' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5042 obo:NCBITaxon_33154 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-69' property="oboInOwl:hasAlternativeId" content="NCBITaxon:37503" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-71'  ></span><span id='am-72' property="rdfs:label" content="Aspergillus" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Aspergillus</span> nidulans. J. Cell Biol. 72:174-193. <br />                                                                          16. Delgado, M. A., J. Conde. 1984. Benomyl prevents nuclear <br /> and spindle elongation (cdcl3, cdcl6, cdc17, cdc2O,                      fusion Saccharomyces cerevisiae. Mol. Gen. Genet. 193:188- <br /> cdc23) isolated phenotypes                   189. <br /> that showed tcpl-l mutation (9, 27, 45). Yet              17. Drubin, D. G., K. G. Miler, D. Botstein. 1988. Yeast <br /> primary functions gene products                       actin-binding proteins: evidence role morphogenesis. J. <br /> determined, known alter structural                       Cell Biol. 107:2551-2561. <br /> components spindle apparatus. Finally, YPTI, gene               18. Gaber, R. F., M. R. Culbertson. 1982. Frameshift suppres- <br /> shown involved secretion, affects nuclear migration                 sion Saccharomyces cerevisiae. IV. New suppressors among <br /> and microtubular structures (22, 52, 54, 55). Further studies                spontaneous corevertants group II his4-206 leu2-3 <br /> will necessary determine TCP1 plays direct                   frameshift mutations. Genetics 101:345-367. <br /> role cytoskeletal function.                                           19. Gaber, R. F., L. Mathison, I. Edelman, M. R. Culbertson. <br />                                                                              1983. Frameshift suppression Saccharomyces cerevisiae. VI. <br />                                                                              Complete genetic map twenty-five suppressor genes. Genet- <br />                      ACKNOWLEDGMENTS                                         ics 103:389-407. <br />   This research supported College Agricultural         20. Gluecksohn-Waelsch, S. 1989. In praise complexity. Genetics <br /> Life Sciences, University Wisconsin, Madison, PHS grants              122:721-725. <br /> GM40310 GM26217 (M.R.C.).                                            21. Hammer, M., J. Schimenti, L. Silver. 1989. Evolution of <br />   We thank B. Ganetzky, laboratory yeast TCP1-                  mouse chromosome 17 origin inversions associated <br /> containing clone isolated DNA sequence analysis carried          t haplotypes. Proc. Nat. Acad. Sci. USA 86:3261-3265. <br /> out. Monoclonal antibodies mouse TCP-1 considerately             22. Haubruck, H., C. Disela, P. Wagner, D. Gallwitz. 1987. The <br /> given K. Willison, S. Leong D. Skinner generously                 ras-related YPT protein ubiquitous eukaryotic protein: <br /> provided yeast OFAGE blots. We thank L. C. Robinson,                isolation sequence analysis mouse cDNA clones highly <br /> M. Winey, P. Leeds, K. Willson, P. Olds-Clarke helpful               homologous yeast YPTI gene. EMBO J. 6:4049-4053. <br /> discussions L. Olds illustrations.                               23. Herrmann, B. G., D. P. Barlow, H. Lehrbach. 1987. A large <br />                                                                              inverted duplication allows homologous recombination between <br />                            REFERENCES                                        chromosomes heterozygous proximal t complex inver- <br />  1. Adams, A. E. M., J. R. Pringle. 1984. Relationship actin          sion. Cell 48:813-825. <br />     tubulin distribution bud growth wild-type <span id='am-137' about='obo:NCBITaxon_64446' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_442183 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_127263 obo:NCBITaxon_33415 obo:NCBITaxon_42282 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-138' property="rdfs:label" content="Morpho" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-141'  ></span>morpho</span>-      24. Hiraoka, Y., T. Toda, M. Yanagida. 1984. The NDA3 gene <br />     genic-mutant Saccharomyces cerevisiae. J. Cell Biol. 98:934-             fission yeast encodes ,B-tubulin: cold-sensitive nda3 muta- <br />     945.                                                                     tion reversibly blocks spindle formation chromosome <br />  2. Bennett, D., A. K. Alton, K. Arzt. 1983. Genetic analysis         movement mitosis. Cell 39:349-358. <br />     transmission ratio distortion t-haplotypes mouse.          25. Hoyt, M. A., T. Stearns, D. Botstein. 1990. Chromosome <br />     Genet. Res. (Cambridge) 41:29-45.                                        instability mutants Saccharomyces cerevisiae defec- <br />  3. Boeke, J. D., F. LaCroute, G. R. Fink. 1984. A phosphate             tive microtubule-mediated process. Mol. Cell. Biol. 10:223- <br />     decarboxylase activity yeast: 5-fluoro-orotic acid resistance.        234. <br />     Mol. Gen. Genet. 197:345-346.                                        26. Hubbard, S. C., R. J. Ivatt. 1981. Synthesis processing <br />  4. Booher, R., D. Beach. 1988. Involvement cdc13+                 asparagine-linked oligosaccharides. Annu. Rev. Biochem. <br />     mitotic control <span id='am-59' about='obo:NCBITaxon_4896' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_1 obo:NCBITaxon_147554 obo:NCBITaxon_34346 obo:NCBITaxon_4890 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4895 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-61'  ></span><span id='am-62' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45042" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="Schizosaccharomyces pombe" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces pombeP" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces malidevorans" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasExactSynonym" content="fission yeast" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Schizosaccharomyces pombe</span>: possible inter-            50:555-583. <br />     action gene product microtubules. EMBO J. 7:2321-        27. Huffaker, T. C., M. A. Hoyt, D. Botstein. 1987. Genetic <br />     2327.                                                                    analysis yeast cytoskeleton. Annu. Rev. Genet. 21:259- <br />  5. Bostian, K. A., S. Jayachandran, D. J. Tipper. 1983. A               284. <br />     glycosylated prototoxin killer yeast: models structure    28. Huffaker, T. C., J. H. Thomas, D. Botstein. 1988. Diverse <br />     maturation. Cell 32:169-180.                                         effects 1-tubulin mutations microtubule formation and <br />  6. Botstein, D., D. Shortie. 1985. Strategies applications       function. J. Cell Biol. 106:1997-2010. <br />     vitro mutagenesis. Science 229:1193-2001.                         29. Ito, H., Y. Fukuda, K. Murata, A. Kinura. 1983. Transfor- <br />  7. Burke, D., P. Gasdaska, L. Hartwell. 1989. Dominant effects          mation intact yeast cells treated alkali cations. J. <br />     tubulin overexpression Saccharomyces cerevisiae. Mol.              Bacteriol. 153:163-168. <br />     Cell. Biol. 9:1049-1059.                                             30. Jacobs, C. W., A. E. M. Adams, P. J. Szaniszlo, J. R. <br />  8. Buss, J. E., B. M. Sefton. 1986. Direct identification            Pringle. 1988. Functions microtubules Saccharomyces <br />     palinitic acid lipid attached p2lras. Mol. Cell. Biol.         cerevisiae cell cycle. J. Cell Biol. 107:1409-1426. <br />     6:116-122.                                                           31. Kilmartin, J. V., A. E. M. Adams. 1984. Structural rear- <br />  9. Byers, B., L. Goetsh. 1975. Behavior spindles spindle         rangements tubulin actin cell cycle yeast <br />     plaques cell cycle conjugation Saccharomyces               Saccharomyces. J. Cell Biol. 98:922-933. <br />     cerevisiae. J. Bacteriol. 124:511-523.                               32. Kronstad, J. W., J. A. Holly, V. L. MacKay. 1987. A yeast <br /> 10. Cameron, J. R., E. Y. Loh, R. W. Davis. 1979. Evidence           operator overlaps upstream activation site. Cell 50:369-377. <br />     transposition dispersed repetitive DNA families yeast. Cell    33. Kyte, J., R. F. Doolittle. 1982. A simple method for <br />     16:739-751.                                                              displaying hydropathic character protein. J. Mol. Biol. <br /> 11. Chesley, P. 1932. Lethal action short-tailed mutation           157:105-132. <br />     house mouse. Proc. Soc. Exp. Biol. Med. 29:437-438.              34. Laemmli, U. K. 1970. Cleavage structural proteins the <br />  2640       URSIC AND CULBERTSON                                                                                              MOL. CELL. BIOL. <br />  <br />       assembly head bacteriophage T4. Nature (London)                complex separable effects tail length lethality but <br />       227:680-685.                                                             associated effects spermatogenesis. Genet. Res. <br /> 35.   Laughon, A., R. F. Gesteland. 1984. Primary structure         (Cambridge) 38:115-123. <br />       Saccharomyces cerevisiae GAL4 gene. Mol. Cell. Biol. 4:260-          60. Silver, L. M. 1985. Mouse t-haplotypes. Annu. Rev. Genet. <br />       267.                                                                     19:179-208. <br /> 36.   Lyon, M. F. 1984. Transmission ratio distortion mouse             61. Silver, L. M., K. Artzt, D. Bennett. 1979. A major cell <br />       t-haplotypes multiple distorter genes acting              protein specified mouse T/t complex gene. Cell 17:275-284. <br />       responder locus. Cell 37:621-628.                                    62. Silver, L. M., K. C. Kleene, R. J. Distel, N. B. Hecht. 1987. <br /> 37.   Lyon, M. F. 1986. Male sterility mouse t-complex           Synthesis mouse t complex proteins haploid stages of <br />       homozygosity distorter genes. Cell 44:357-363.                 spermatogenesis. Dev. Biol. 119:605-608. <br /> 38.   Lyon, M. F. 1987. Distorter genes mouse t-complex             63. Silver, L. M., D. Remis. 1987. Five genetically <br />       impair fertility he <br /> </body></html>