<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Eukaryotic translation initiation factor (eIF) <br /> 5A stimulates protein synthesis in <br /> <span id='am-40' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-45'  ></span><span id='am-46' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br />  <br /> Within the field of eukaryotic protein synthesis, one factor re-              have normal polysome profiles (11), suggesting that eIF5A is not <br /> mained putative for decades: eukaryotic translation initiation                required for general protein synthesis. <br /> factor (eIF) 5A. Because eIF5A is an essential protein required for              We chose to reexamine the possibility that eIF5A stimulates <br /> cell proliferation, and one easily targeted by inhibitors, identifying        translation for several reasons. First, the degree to which eIF5A <br /> its role in the cell remains important and urgent. Recent reports             consistently stimulated met-puro formation did not substantially <br /> support early findings that eIF5A stimulates protein synthesis                differ from what was observed when depleting UBR5A from <br /> and newly assign the factor a role in elongation rather than initia-          yeast cells. Second, the primary physiological activity of eIF5A <br /> tion. Here we show that eIF5A directly stimulates protein synthesis           had remained elusive. Finally, we discovered potential artifacts <br /> on native mRNAs, that rapid depletion of eIF5A in vivo immediately            in the eIF5A depletion experiments that were a basis for doubting <br /> leads to a 2-fold inhibition of protein synthesis, and that both the          that eIF5A plays a role in general translation initiation. By <br />  <br />  <br />  <br />  <br />                                                                                                                                                                        BIOCHEMISTRY <br /> immediate and lasting effects of eIF5A depletion are a reduction              optimizing the depletion experiments we find that loss of eIF5A <br /> in polysome size concomitant with eIF5A depletion. Addition of                correlates with a substantial inhibition of protein synthesis in vivo, <br /> purified eIF5A to a depleted lysate results in a roughly 2-fold sti-          UBR5A depletion results in a pronounced effect on polysome <br /> mulation of protein synthesis in vitro, a result consistent with both         distribution, and supplementing depleted lysates with eIF5A <br /> older methionyl&#8211;puromycin synthesis <span id='am-88' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-91'  ></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> and more recently                    stimulates protein synthesis in vitro. <br /> published findings. We find that although eIF5A is not required for              While <span id='am-69' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71'  ></span><span id='am-72' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>this</span> manuscript was being prepared, other laboratories <br /> protein synthesis, it stimulates the process by about 2- to 3-fold.           reported quantitatively similar effects of eIF5A depletion on <br /> Our data, along with other published results, reinforce the conclu-           global translation (12, 13). However, although these reports <br /> sion that eIF5A stimulates protein synthesis with one important               conclude that eIF5A stimulates elongation, the polysome profiles <br /> difference: Polysome profiles observed immediately after eIF5A                we observe during and after eIF5A depletion are diagnostic for a <br /> depletion are diagnostic for a role in initiation. This discrepancy <br />                                                                               role in initiation. Possible explanations for this discrepancy are <br /> is discussed. <br />                                                                               discussed, as is a recently published structure suggesting that <br />                                                                               EF-P stimulates only the formation of the first peptide bond (14). <br /> hypusine &#8739; ribosome <br />                                                                               Results <br />  <br /> E   ukaryotic translation initiation factor (eIF) 5A is a small <br />     (16&#8211;18 kDa), essential protein that is ubiquitous and highly <br /> conserved among eukaryotes and <span id='am-1' about='obo:NCBITaxon_2157' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567'><span id='am-2' property="oboInOwl:hasDbXref" content="PMID:10425797" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="PMID:270744" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="PMID:11542149" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="PMID:9103655" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Prokaryota" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="PMID:11321083" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="PMID:11211268" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="PMID:10425795" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="PMID:12054223" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Metabacteria" datatype="xsd:string"></span><span id='am-13'  ></span><span id='am-14' property="oboInOwl:hasDbXref" content="PMID:10939673" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Archaebacteria" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="PMID:11540071" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="PMID:10939651" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="PMID:10939677" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="PMID:11411719" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="prokaryotes" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Mendosicutes" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Monera" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="PMID:11321113" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="PMID:10490293" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Prokaryotae" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="prokaryote" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="PMID:8590690" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="PMID:9336922" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="PMID:8123559" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="PMID:2112744" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="archaea" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="PMID:10425796" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Archaea" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="PMID:11541975" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="PMID:11542064" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Procaryotae" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="PMID:10843050" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="PMID:11760965" datatype="xsd:string"></span>archaea</span> (1, 2). Several crystal <br />                                                                               Control Experiments Suggest That eIF5A Acts in Translation. We first <br />                                                                               asked whether previously published data, obtained with strain <br />                                                                               UBHY-R, may have been subject to an artifact. Some degrada- <br /> structures of archaeal aIF5A have been published within the last              tion of UBR5A, the rapidly depleted protein expressed in strain <br /> decade (2&#8211;4). The less-conserved C-terminal portion is classified             UBHY-R, may have occurred during lysate preparation, inaccu- <br /> as an oligonucleotide binding motif (4). The N-terminal portion is            rately representing in vivo UBR5A levels. We found this effect to <br /> more highly conserved and bears the absolutely unique feature of              be negligible, although a striking media-dependent effect was <br /> a/eIF5A: a specific, posttranslationally modified lysine known as             observed, indicating that UBR5A depletion in SD glucose is <br /> hypusine [N&#1013; -(4-amino-2-hydroxybutyl)lysine] (5). This residue               impaired (Fig. S1). Depletion of UBR5A was previously <br /> lies at the tip of an extended loop in the crystal structures, the <br />                                                                               demonstrated by blotting extracts of cells grown in rich media. <br /> sequence of which (STSKTG-hypusine-HGHAK) is universally <br />                                                                               If UBR5A depletion in minimal media is inefficient, then in vivo <br /> conserved in eukaryotes (6). Remarkably, a crystal structure of <br />                                                                               assays of [35 S]methionine incorporation into protein by UBHY-R <br /> the prokaryotic eIF5A homolog, called elongation factor P <br />                                                                               grown in minimal media could have been misleading. We also <br /> (EF-P), shows that both proteins structurally resemble tRNA (7). <br />    The essential function of eIF5A remains obscure. The factor                surmised that, because the TIF51A coding region was not com- <br /> was originally isolated from ribosomal high-salt washes (8, 9),               pletely deleted in UBHY-R, reversion to wild type at the TIF51A <br /> suggesting a role in translation. Purified eIF5A stimulates a                 locus by homologous recombination with the plasmid-borne <br /> model assay of translation initiation, methionyl&#8211;puromycin (met-              UBR5A construct was possible, absent selection. Reversion does <br /> puro) synthesis (9, 10). Although eIF5A does not affect any step              occur during growth in rich media (Fig. S2). Growth in selective <br /> prior to 80S initiation complex assembly (10), the factor increases           media prior to all experiments prevents contamination with re- <br /> met-puro formation by 2- to 3-fold (9, 10). Whether eIF5A sti- <br /> mulates translation in vivo was tested by depleting eIF5A in yeast            Author contributions: A.H. and J.W.H. designed research; A.H. performed research; A.H. <br /> (11). These depletion experiments used a &#8220;rapid-depletion&#8221; yeast              contributed new reagents/analytic tools; A.H. and J.W.H. analyzed data; and A.H. and <br /> strain, UBHY-R, engineered to express an unstable form of                     J.W.H. wrote the paper. <br /> eIF5A (UBR5A). By rendering UBR5A expression dependent                        The authors declare no conflict of interest. <br /> on the GAL1 promoter in a null genetic background, eIF5A is                   This article is a PNAS Direct Submission. <br /> rapidly depleted by shifting UBHY-R cells into glucose-contain-               1 <br />                                                                               To whom correspondence should be addressed. E-mail: jwhershey@ucdavis.edu. <br /> ing media. UBHY-R cells grown in glucose exhibited only a mild                This article contains supporting information online at www.pnas.org/lookup/suppl/ <br /> (30%) reduction in the rate of protein synthesis and appeared to              doi:10.1073/pnas.1008150108/-/DCSupplemental. <br />  <br />  <br />  <br />     www.pnas.org/cgi/doi/10.1073/pnas.1008150108                                                              PNAS &#8739; April 19, 2011 &#8739; vol. 108 &#8739; no. 16 &#8739; 6415&#8211;6419 <br />  vertants as determined by periodic selection tests and prolonged                 The effect of growth medium on protein synthesis rates is also <br /> growth suppression in glucose-containing media. These results sug-            readily seen when polysome profiles are determined. In the case <br /> gest that a role for eIF5A in translation may have been masked                of YP galactose (Fig. 2C), heavy polysomes are much more pre- <br /> by UBHY-R-specific effects. Therefore we asked whether, as ex-                dominant compared to those from cells grown in SD galactose <br /> pected for a translation factor (9), eIF5A associates with poly-              (Fig. 2A), indicative of more efficient translation initiation in rich <br /> somes. As since reported by others (15), we found that it does                media. Following depletion, polysomes are severely reduced in <br /> bind to ribosomes (Fig. S3), although not strongly. Finally, we rea-          YP-glucose medium (Fig. 2D), whereas the effect in SD-glucose <br /> soned that because eIF5A is not absolutely required for the                   medium (Fig. 2B) is more modest, although polysome distribu- <br /> formation of met-puro in vitro, a general reduction in translation            tion changes similarly in both cases. These observations are <br /> activity (expected in cells grown in minimal media; see below)                consistent with the protein synthesis rate data (Fig. 1; ref. 11). <br /> could mask a potential effect of UBR5A depletion on general                   The results obtained in YP media indicate that eIF5A stimulates <br /> translation. This possibility is addressed in the next section.               the rate of protein synthesis by at least twofold. We conclude that <br />                                                                               the growth conditions used in earlier depletion studies masked <br /> The Effect of eIF5A Depletion on Translation Is More Pronounced in            important effects of eIF5A depletion on protein synthesis. In <br /> Rich Media. The growth medium used to propagate UBHY-R in-                    this experiment, UBHY-R cells were incubated in glucose-con- <br /> fluences both the rate and extent of protein synthesis inhibition             taining medium for 5&#8211;6 h in <span id='am-84' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-85' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> to replicate conditions used <br /> upon UBR5A depletion. By comparing the rate of [35 S]methio-                  previously (11). <br /> nine incorporation into total protein by UBHY-R cells before <br /> and during depletion, we observe a media-dependent effect on                  eIF5A Directly Stimulates Protein Synthesis in Vitro. The demonstra- <br /> protein synthesis (Fig. 1 and Fig. S4). Extended depletion (for               tions above that eIF5A depletion reduces global translation in <br /> approximately 5 h) inhibits translation initiation when either rich           yeast involve polysome profile analysis and in vivo protein synth- <br /> [yeast extract peptone (YP)] or minimal [synthetic defined (SD)]              esis assays. Both techniques show an effect of eIF5A depletion in <br /> media are employed, but the effect in rich media is more rapid                intact cells, but the time required to reduce eIF5A levels means <br /> and larger. Protein synthesis is inhibited by 2-fold after 1 h of             that we cannot rule out possible indirect effects of eIF5A deple- <br /> UBR5A depletion in rich media (Fig. 1 and Fig. S4C), whereas                  tion. The difference between the rates of UBR5A depletion <br /> a similar degree of inhibition is not observed until 2&#8211;3 h of incu-           (Fig. S1) and the reduction of [35 S]methionine incorporation <br /> bation in minimal media (Fig. S4 A and B). Extended depletion                 (Fig. 1) also suggests that secondary effects might contribute <br /> for approximately 5 h results in a 4-fold inhibition of protein               to translational repression. We therefore sought a more direct <br /> synthesis in rich media; in minimal media a lower inhibitory effect           measure of eIF5A stimulation of protein synthesis and asked <br /> (between 2.5- and 3-fold) is observed after similar depletion                 whether the addition of purified eIF5A to depleted lysates would <br /> times. In both minimal and rich media we observe a greater in-                stimulate protein synthesis in vitro. <br /> hibition of protein synthesis than was initially reported in minimal             Toward this end, we prepared translationally active extracts <br /> media (about 1.4-fold; see ref. 11); based on our present findings            from UBHY-R cells incubated in YP glucose for 2.5 h. The <br /> we conclude that this difference arises from media effects and the            UBR5A-depleted extracts were then supplemented with purified <br /> artifacts described in the previous section. The effects of eIF5A             eIF5A(lys or dhp) or eIF5A(hyp) purified from yeast. Protein <br /> depletion are seen as early as 1 h of depletion, when protein                 synthesis on endogenous messages was measured by following <br /> synthesis rates already have fallen by 2-fold in YP media. These              the incorporation of [35 S]methionine into acid-insoluble protein. <br /> differences are likely due to more efficient translation initiation           It is important to recognize that such stimulation by eIF5A <br /> by cells in YP media, although this is difficult to demonstrate by            depends on having a sufficient level of methionine in the assay, <br /> this method without quantifying intracellular methionine pools.               and that the concentration of total methionine dramatically af- <br />                                                                               fects activity (Fig. S5). The use of recombinant eIF5A expressed <br />                                                                               in and prepared from bacteria, rather than eIF5A purified from <br />                                                                               yeast, eliminates possible contaminating yeast proteins that might <br />                                                                               affect protein synthesis. We also tested whether yeast-derived <br />                                                                               FLAG-tagged eIF5A(hyp) would differ in activity from the bac- <br />                                                                               terially derived eIF5A preparations. Although eIF5A(hyp) shows <br />                                                                               greater stimulatory activity than eIF5A(dhp) during met-puro <br />                                                                               synthesis, both forms are active and specific; eIF5A(lys) or a <br />                                                                               homodeoxyhypusine-modified analog of eIF5A shows no stimu- <br />                                                                               latory activity (16). We demonstrated that protein synthesis in <br />                                                                               depleted lysates depends on new initiation events by using a cap <br />                                                                               analog (m7 GTP) to sequester eIF4E and block cap-dependent <br />                                                                               initiation. The result, shown in Fig. 3A, reveals an initial rapid <br />                                                                               incorporation of [35 S]methionine resistant to inhibition, presum- <br />                                                                               ably due to elongation of nascent proteins on preexisting <br />                                                                               polysomes. After 10&#8211;20 min, protein synthesis in the presence <br />                                                                               of m7 GTP virtually ceases, whereas uninhibited lysates show a <br />                                                                               near-linear increase in [35 S]methionine incorporation. We inter- <br />                                                                               pret the latter to mean that [35 S]methionine incorporation after <br /> Fig. 1. The effect of UBR5A depletion on protein synthesis rates is pro-      20 min depends on new initiation events. Thus the slope after <br /> nounced when cells are grown in rich media. W303-1A and UBHY-R cells          20 min is a measure of the rate of initiation. The two slopes <br /> were grown in YP galactose, then resuspended either in YP glucose (&#8220;De-       in Fig. 3A differ by a factor of 10, indicating that m7 GTP inhibits <br /> pleted&#8221; UBHY-R and &#8220;W303-1A/Glucose&#8221;) or galactose (&#8220;Induced&#8221; UBHY-R) <br />                                                                               initiation by 90%. That protein synthesis and new initiation per- <br /> as indicated. Duplicate samples were processed independently. [35 S]methio- <br /> nine incorporation and protein concentrations were determined as described <br />                                                                               sist in severely depleted lysates is expected, given that polysomes <br /> in Materials and Methods. Protein synthesis rates were measured as cpm        also persist in depleted cells, albeit reduced in size (Fig. 2), <br /> incorporated per &#956;g total protein per minute of labeling, then expressed as   and that methionyl-puromycin formation in vitro proceeds in the <br /> a percentage of the respective value at time zero.                            absence of added eIF5A (9, 10). <br />  <br /> 6416 &#8739;   www.pnas.org/cgi/doi/10.1073/pnas.1008150108                                                                          Henderson and Hershey <br />                                                                                                                                                                      BIOCHEMISTRY <br /> Fig. 2. The effect of UBR5A depletion on polysomes is more pronounced in rich media. UBHY-R cells were cultured in the indicated galactose-containing media <br /> (A and C) until reaching mid-log phase. Half of the culture was treated immediately with CHX and used to prepare a nondepleted extract. The other half was <br /> resuspended in the indicated media (B and D) with glucose and incubated at 30 &#176;C for 5 (SD) or 6 (YP) h. After glucose incubation, extracts were prepared in the <br /> same manner. Approximately three OD units of each extract were loaded onto 10&#8211;60% (SD) or 10&#8211;50% (YP) sucrose gradients and centrifuged. Absorbance <br /> scans at 254 nm were obtained for each gradient. The ratio of polysomes to total ribosomes was calculated by numerical integration and is given for each <br /> profile (Upper Right). The profiles are representative of three independent analyses. <br />  <br />     The effect of adding various forms of eIF5A to depleted lysates                  Our data indicate that eIF5A stimulates translation directly. <br /> is shown in Fig. 3B. Distinct stimulation of [35 S]methionine incor-              Depletion of eIF5A from yeast cells results in a rapid decrease <br /> poration is apparent and depends upon the presence of deoxyhy-                    in ribosomal loading on messages (Fig. 2) that is concomitant <br /> pusine or hypusine (Fig. 3B and Fig. S6). This specificity was seen               with the factor&#8217;s disappearance (Fig. S1). Under optimal growth <br /> previously with the methionyl&#8211;puromycin assay, although unlike                    conditions, the immediate effect of depleting eIF5A is a twofold <br /> our current result, the quantitative activity of eIF5A(dhp) and                   inhibition of protein synthesis (Fig. 1 and Fig. S4C). Although <br /> eIF5A(hyp) then differed. This may reflect a disparity between                    translation initiation proceeds in the absence of eIF5A (Fig. 1 <br /> mammalian and fungal translation (10). The average difference                     and Fig. S1), supplementing the factor in vitro (Fig. 3B and <br /> in slopes we observed, comparing added eIF5A(dhp) or eIF5A                        Fig. S6) directly restores translation activity on endogenous <br /> (hyp) with buffer alone at points after 20 min, was 1.8 &#177; 0.1                     messages to a degree that is surprisingly consistent with multiple <br /> among nine experiments and multiple depleted lysates. Therefore                   met-puro synthesis assay results (stimulation factor: 2.8 &#177; 0.6) <br /> exogenous eIF5A stimulates protein synthesis, likely at the initia-               (9, 10). The parity between our in vivo and in vitro data is parti- <br /> tion phase (see Discussion), by about 2-fold in this system. A                    cularly important. When cells are depleted of eIF5A for 2.5 h, <br /> slightly larger degree of stimulation of protein synthesis (2.5-fold)             protein synthesis rates fall by a factor that is essentially identical <br /> is seen if one simply subtracts the amount of [35 S]methionine                    to the degree of stimulation observed when supplementing <br /> incorporated in the presence of m7 GTP from each of the corre-                    extracts prepared from these cells. This result suggests that <br /> sponding time points to give initiation-dependent protein synth-                  any immediate secondary effects of depleting eIF5A activity are <br /> esis. Also noteworthy is the fact that the rate of [35 S]methionine               nominal, although such secondary effects may play a role during <br /> incorporation during the first 10 min of incubation does not sig-                 later phases of depletion. <br /> nificantly differ between reactions stimulated and not stimulated                    The question of whether eIF5A plays a role in protein synth- <br /> by eIF5A (Fig. 3B), suggesting that eIF5A does not immediately                    esis has been analyzed in detail by other groups recently (12, 13, <br /> affect the elongation rate (see Discussion). We conclude that                     17). Their results, and our data, demonstrate that eIF5A plays a <br /> eIF5A is not essential for global protein synthesis, but is required              direct role in translation. A recent analysis of mutant forms of <br /> for optimal translation of a substantial number of endogen-                       eIF5A also supports this conclusion (18). Therefore, the numer- <br /> ous mRNAs.                                                                        ous pleiotropic phenotypes seen with mutant forms of eIF5A <br />                                                                                   may be due to changes in the translational efficiencies of specific <br /> Discussion                                                                        mRNAs. <br /> Our findings lead to three important conclusions. First, eIF5A                       That eIF5A acts in translation is now well-supported. How- <br /> stimulates protein synthesis directly and globally, presumably                    ever, our data must be reconciled with other reports that indicate <br /> on all or most yeast mRNAs, although further study must deter-                    distinctly different roles of eIF5A in protein synthesis. While this <br /> mine its precise mechanism of action. Second, assay optimization                  work was in preparation, two groups reported data supporting a <br /> is required when characterizing eIF activities that have only a                   role for eIF5A in translational elongation (12, 13); a third group <br /> modest effect on translation rates. Third, and related, suboptimal                reported a crystal structure, EF-P bound to 70S, that suggests a <br /> experimental conditions contributed to the erroneous and widely                   role for the eIF5A homolog in the formation of the first peptide <br /> cited conclusion that eIF5A does not act in general translation                   bond only (14). Our data agree with the interpretation that eIF5A <br /> based on work employing strain UBHY-R. These conclusions                          acts as an elongation factor, but only with respect to formation of <br /> are addressed in detail below.                                                    the first peptide bond. The polysome profiles we observe upon <br />  <br /> Henderson and Hershey                                                                                   PNAS &#8739;    April 19, 2011 &#8739;   vol. 108 &#8739;   no. 16 &#8739;   6417 <br />                                                                                 parallel lines, suggesting that elongation rates decreased through- <br />                                                                                out their experiment. Finally, the dipeptide release experiment <br />                                                                                required isolation of the substrate, during which time the tRNA <br />                                                                                in the ribosomal E site may have been lost. EF-P binding to the <br />                                                                                E site, and by homology eIF5A binding, is incompatible with <br />                                                                                simultaneous occupancy by stripped tRNA at the same site (14). <br />                                                                                Because E-site occupancy by tRNAi Met is observed or expected <br />                                                                                following formation of the first peptide bond and translocation <br />                                                                                (and by other tRNAs in all subsequent elongation steps), it is <br />                                                                                difficult to rationalize a general elongation activity for eIF5A <br />                                                                                after formation of the first peptide bond. <br />                                                                                   It is important to note that robust quantification of eIF5A <br />                                                                                activity in translation was observed only after optimizing experi- <br />                                                                                mental conditions in vivo and in vitro. First, artifacts unique to <br />                                                                                strain UBHY-R were determined (Figs. S1 and S2). Second, we <br />                                                                                observed a strong effect of UBR5A depletion only after maximiz- <br />                                                                                ing in vivo protein synthesis (Ref. 11, Figs. 1 and 2 and Fig. S4). <br />                                                                                Third, efficient protein synthesis on endogenous mRNAs in vitro <br />                                                                                depends on raising the molar concentration of radioactive methi- <br />                                                                                onine by cold methionine supplementation (Fig. S5). Although <br />                                                                                we have not determined the step(s) inhibited by low methionine <br />                                                                                concentrations in vitro, relieving such inhibition is a sensible and <br />                                                                                likely required step to take when examining the function of <br />                                                                                weakly stimulating factors such as eIF5A (9, 10). <br />                                                                                   We favor a model in which the primary physiological activity <br />                                                                                of eIF5A is to stimulate formation of the first peptide bond. In <br />                                                                                support of this <span id='am-64' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-65' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67'  ></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>idea</span> we note the following: (i) eIF5A was initially <br />                                                                                found not to stimulate elongation on polyU in vitro in the <br />                                                                                presence of saturating levels of eEF1A and eEF2 (8); (ii) eIF5A <br />                                                                                was more recently found to stimulate only the formation of the <br />                                                                                first two peptide bonds on polyU, but not other templates, in the <br />                                                                                presence of eEF activity (13); (iii) most compelling, the eIF5A <br />                                                                                homolog EF-P makes contacts with L1 in the large ribosomal <br />                                                                                subunit that are incompatible with simultaneous stripped tRNA <br />                                                                                E-site occupancy (14). Considering our experiments showing <br />                                                                                diminished polysome profiles upon depletion, a failure to stimu- <br />                                                                                late elongation on preexisting mRNAs in vitro, and the above <br />                                                                                considerations, we conclude that eIF5A stimulates the formation <br />                                                                                of the first peptide bond. However, we cannot rule out the <br />                                                                                possibility that differences in genetic backgrounds, effects of <br />                                                                                prolonged eIF5A inactivation, or experimental procedures may <br />                                                                                account for other interpretations of eIF5A activity. <br />                                                                                Materials and Methods <br /> Fig. 3. Although not required for initiation, eIF5A directly stimulates pro-   <span id='am-93' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>Plasmids</span> and Proteins. Plasmids pYAH-F5A and pYAH-F5A(K51R) were con- <br /> tein synthesis in vitro in a hypusine-dependent manner. Translation extracts   structed by PCR (19) from pBM-TIF51A (20) and YRp7-5A(K51R) (21). The am- <br /> depleted of UBR5A for 2.5 h were prepared and assayed in vitro for protein     plified fragment was inserted into the BamH1 and EcoR1 sites of p414Gal1 <br /> synthesis as described in Materials and Methods. To identical reaction         (22) and used to express FLAG-tagged eIF5A and its K51R mutant form in <br /> aliquots was added m7 GTP to inhibit initiation, or a buffer control (A). In   strains YAH2 and YAH60. His6 -eIF5A(lys) was expressed in strain BAH6 <br /> B, extracts were supplemented with buffer alone, or at five molecules          [BL21(DE3) carrying the eIF5A cDNA in pET28c]. To generate eIF5A(lys), <br /> per 80S ribosome of eIF5A(lys), eIF5A(dhp), or FLAG-tagged eIF5A(hyp),         His6 -eIF5A(lys) was purified from strain BAH6 and treated with thrombin <br /> the latter protein being purified from yeast. Ribosome concentration in the    to remove the tag. Similarly, eIF5A(dhp) was obtained from BAH6 by treating <br /> extract was determined by using the molar extinction coefficient of 80S as     purified eIF5A(lys) with His6-yDHS (23, 24). Native FLAG-tagged eIF5A [F5A <br /> 5 &#215; 107 M&#8722;1 cm&#8722;1 at 260 nm and by assuming that ribosomes are 80% of           (hyp)] was purified from yeast strain YAH2 with an anti-FLAG resin. Details of <br /> total RNA (25). Experiments shown are representative of multiple indepen-      these procedures are provided in SI Materials and Methods. <br /> dent experiments. <br />                                                                                Extract Preparation and Western Blotting. Cells were propagated at 30 &#176;C in YP <br /> eIF5A depletion indicate that eIF5A is required for efficient in-              medium containing either 2% glucose or 2% galactose. For depletion studies, <br /> itiation; we see no other interpretation that explains a decrease in           cells were washed once with sterile room-temperature water containing <br /> efficiently loaded mRNAs after depletion of eIF5A. However,                    2% glucose, then resuspended in either YP or SD glucose or galactose as <br />                                                                                indicated. Two lysis methods were employed. The nondenaturing method in- <br /> the polysome profiles obtained by Dever and coworkers (13) lead <br />                                                                                volved glass bead vortexing and was described previously (11). For the dena- <br /> to the conclusion that eIF5A promotes translation elongation. It               turing method, a similar glass bead procedure was used, except that the <br /> is unclear how to reconcile our polysome profiles with theirs.                 buffer was 40 mM Tris-HCl, pH 6.8, 4% SDS, 4% glycerol, and 0.1 M DTT, <br /> However, their other data suggesting a role in elongation are                  and samples were boiled for 10 min. Lysate proteins were resolved and <br /> more difficult to interpret. In particular, some of the results were           blotted with rabbit anti-yeast eIF5A (11). UBR5A bands were quantified with <br />                                                                                TotalLab TL100 (Nonlinear Dynamics, version 2006b). <br /> obtained after a number of hours of eIF5A depletion, when sec- <br /> ondary effects of eIF5A depletion likely occur. The ribosome                   In Vivo Protein Synthesis Measurements. W303-1A and UBHY-R cells (11) were <br /> transit time experiments were not done under steady-state con-                 pulse-labeled with [35 S]methionine at various times following the shift from <br /> ditions; total and released incorporation data do not generate                 YP galactose to YP glucose. About three OD595 units of cells (processed in- <br />  <br />  <br /> 6418 &#8739;   www.pnas.org/cgi/doi/10.1073/pnas.1008150108                                                                                 Henderson and Hershey <br />  dependently in duplicate) were washed in water/carbon source, resuspended                        rotor for approximately 1 h. Polysome profiles were generated by following <br /> in 300 &#956;L YP (glucose or galactose), and incubated with 100 &#956;Ci Trans-35 S-                      absorbance at 254 nm and fractions collected using an ISCO fractionator. <br /> <span id='am-87' about='rdfs:label' typeof='owl:Thing'>Label</span> (MP Biomedicals) at 30 &#176;C for 15 min with shaking. Labeling was termi- <br /> nated by adding 1-mL ice-cold stop solution [1.2 mg&#8725;mL methionine,                               In Vitro Translation Assays. Cells were grown in galactose media, shifted to <br /> 0.1 mg&#8725;mL cycloheximide (CHX)] and incubating on ice briefly. Cells were                         glucose to deplete eIF5A, and lysed as described in detail in SI Materials <br /> collected and snap-frozen in liquid nitrogen, then resuspended in 15% tri-                       and Methods. Translation assay reactions contained 60 &#177; 10% extract and <br /> chloroacetic acid (TCA) (1 mL), incubated on ice for 15 min, and boiled at                       eIF5A proteins as indicated, with final reaction conditions of 30 mM Hepes, <br /> 95 &#176;C for 15 min to deacylate tRNAs. Denatured protein was pelleted, <span id='am-79' about='obo:NCBITaxon_9822' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9821 obo:NCBITaxon_35497 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="Sus" datatype="xsd:string"></span><span id='am-83'  ></span>sus</span>-                        pH 7.5, 3.8 &#177; 0.1 mM total MgCl2 , 2 mM ATP, 0.5 mM GTP, 0.2 mg&#8725;mL creatine <br /> pended in 1 mL acetone at &#8722;20 &#176;C, stored 30 min at &#8722;20 &#176;C, and centrifuged                       kinase, 12 mM creatine phosphate, 165 &#177; 10 mM KOAc/KCl, 100 &#956;M amino <br /> again. Precipitated proteins were dissolved in 400 &#956;L 1% SDS, boiled for                         acids (lacking methionine), 30&#8211;40 &#956;M unlabeled methionine, 0.17 &#956;M trans- <br /> 10 min, and clarified by centrifugation. The protein content in 50-&#956;L aliquots                   lation grade [35 S]methionine (1175 Ci&#8725;mmol; MP Biomedicals), 1 mM AEBSF <br /> was determined with a BCA kit (Pierce) against a bovine serum albumin                            (added fresh), and 0.3 units&#8725;&#956;L RNAprime (added fresh). Reactions were <br /> control. To measure radiolabel incorporation, 25-&#956;L aliquots were counted                        initiated by adding extract and were performed at 20 &#177; 2 &#176;C. At the indicated <br /> in a Beckman LS 6000 IC scintillation counter. The rate of protein synthesis                     times, 5-&#956;L aliquots were withdrawn, spotted on Whatman #1 filters, soaked <br /> was calculated for each sample as cpm&#8725;&#956;g protein&#8725; min.                                           in fixative (15% TCA, 20 mM cold methionine), swirled, and stored at 4 &#176;C <br />                                                                                                  until assay completion. The fixative was decanted and replaced, boiled <br /> Polysome Profile Analysis. Cells were grown as above, brought to 0.1 mg&#8725;mL                       briefly, and allowed to cool to room temperature. Filters were washed twice <br /> CHX and rapidly cooled on ice. Pellets were washed, resuspended in poly-                         with methanol, air-dried, and then counted in a Beckman LS 6000 IC <span id='am-74' about='obo:NCBITaxon_1177105' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_52932 obo:NCBITaxon_131567 obo:NCBITaxon_6580 obo:NCBITaxon_6544 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_106228 obo:NCBITaxon_6599 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Scintilla" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78'  ></span>scintilla</span>- <br /> some lysis buffer [20 mM Hepes, pH 7.5, 140 mM KCl, 5 mM Mg&#240;OAc&#222;2 ,                              tion counter. <br /> 0.5 mM DTT, 1% Triton X-100, 1 mg&#8725;mL heparin, 100 U&#8725;mL RNAprime <br /> (Eppendorf), 1 mM 4-(2-aminoethyl)benzenesulfonyl fluoride (AEBSF) and <br />                                                                                            <br /> complete protease inhibitors (Roche)] at approximately 1-mL buffer per gram                     <br /> of cells and lysed by vortexing with glass beads. Lysates were centrifugation         <br /> briefly at 1;000 &#215; g, and three A260 units were loaded onto sucrose gradients                  <br /> and centrifuged at 54,000 rpm in a Beckman TL100 centrifuge in a TLS55                        <br /> </body></html>