<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Lejeune et al. BMC Genomics 2012, 13:91 <br /> http://www.biomedcentral.com/1471-2164/13/91 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                              Open Access <br />  <br /> Large-scale functional RNAi screen C. elegans <br /> identifies genes regulate dysfunction of <br /> mutant polyglutamine neurons <br /> Fran&#231;ois-Xavier Lejeune1&#8224;, Lilia Mesrob1&#8224;, Fr&#233;d&#233;ric Parmentier1&#8224;, Cedric Bicep1,2, Rafael P Vazquez-Manrique1,2, <br /> J Alex Parker1,2,3, Jean-Philippe Vert4,5,6, Cendrine Tourette1,2,7 Christian Neri1,2* <br />  <br />  <br />   Abstract <br />   Background: A central goal Huntington&#8217;s disease (HD) research identify prioritize candidate targets for <br />   neuroprotective intervention, requires genome-scale information modifiers early-stage neuron <br />   injury HD. <br />   Results: Here, performed large-scale RNA interference screen C. elegans strains express N-terminal <br />   huntingtin (htt) touch receptor neurons. These neurons control response light touch. Their function is <br />   strongly impaired expanded polyglutamines (128Q) shown nearly complete loss touch response in <br />   adult animals, providing vivo model manipulate early phases expanded-polyQ <br />   neurotoxicity. In total, 6034 genes examined, revealing 662 gene inactivations reduce aggravate <br />   defective touch response 128Q animals. Several genes previously implicated HD neurodegenerative <br />   disease, suggesting screen effectively identified candidate targets HD. Network-based analysis <br />   emphasized subset high-confidence modifier genes pathways HD including metabolic, <br />   neurodevelopmental pro-survival pathways. Finally, 49 modifiers 128Q-neuron dysfunction are <br />   dysregulated striatum R/2 CHL2 HD mice, both, identified. <br />   Conclusions: Collectively, results highlight relevance HD pathogenesis, providing novel information <br />   potential therapeutic targets neuroprotection HD. <br />  <br />  <br /> Background                                                                           biological processes essential cellular <br /> Huntington&#8217;s disease (HD) dominantly-inherited dis-                             homeostasis neuron survival. Consistent <span id='am-53' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-54' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57'  ></span>this</span> <br /> <span id='am-66' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-67' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> caused expanded polyglutamine (polyQ) tracts                                possibility, transcriptomic analyses revealed that <br /> in N-terminal portion huntingtin (htt) char-                              mutant/polyQ-expanded htt expression change the <br /> acterized striatal cortical degeneration [1]. While                           expression large number genes caudate <br /> HD pathogenesis involve gain toxic proper-                                nucleus HD subjects [9] brain fragment, <br /> ties mutant htt loss normal htt func-                             knock-in full-length mouse models HD [10]. <br /> tion, studies emphasized critical role                             Given complexity alterations cells tissues <br /> misfolded N-terminal fragments mutant htt [2,3]                              expressing expanded polyQ/htt, gene perturbation <br /> are natural products htt processing [4]. Huntingtin                            screens expected shed light mechanisms <br /> thought large number partner proteins                                   critical cell response mutant polyQ <br /> involved variety biological processes [5-8], sug-                            expression. Additionally, integration gene pertur- <br /> gesting polyglutamine expansion htt alter                                bation <span id='am-102' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-103' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-104' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-105'  ></span><span id='am-106' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> types &#8216;omics data for <br />                                                                                      example gene expression profiles HD mice [10] might <br /> * Correspondence: christian.neri@inserm.fr                                           fulfill need target profiling prioritisation in <br /> &#8224; Contributed equally                                                                HD. This particularly important compar- <br /> 1 <br />  INSERM, Unit 894, Laboratory Neuronal Cell Biology Pathology,                ing target gene activity regulation neuronal <br /> 75014 <span id='am-80' about='obo:NCBITaxon_49669' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_50362 obo:NCBITaxon_4447 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_4667 obo:NCBITaxon_3193'><span id='am-81' property="rdfs:label" content="Paris" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Paris L." datatype="xsd:string"></span><span id='am-84'  ></span><span id='am-85' property="oboInOwl:hasAlternativeId" content="NCBITaxon:49665" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Paris</span>, France <br /> Full list author information available end article               dysfunction versus neuronal death. Before the <br />  <br />                                        &#169; 2012 Lejeune et al; licensee BioMed Central Ltd. This Open Access article distributed terms Creative Commons <br />                                        Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, reproduction in <br />                                        medium, provided original work properly cited. <br />  Lejeune et al. BMC Genomics 2012, 13:91                                                                       Page 2 14 <br /> http://www.biomedcentral.com/1471-2164/13/91 <br />  <br />  <br />  <br />  <br /> occurrence neuronal cell loss, neuron dysfunction       Methods <br /> indeed thought early-stage component          C. elegans strains <br /> pathogenic process HD [3,11]. However, neuronal cell       Nematode strains handled using standard methods <br /> dysfunction easily tractable vivo. Interestingly,   [16]. Integrated polyQ strains used study were <br /> the genetic regulation response light touch          previously described [17,18]. Briefly, ID245 (igIs245[mec- <br /> mechanosensory neurons C. elegans             3p::htt57-19Q::CFP; lin-15(+); mec-7p::YFP]) ID1 <br /> subject detailed analysis [12]. By targeting polyQ         (igIs1[mec-3p::htt57-128Q::CFP; lin-15(+); mec-7p::YFP]) <br /> expression C. elegans touch cells, previously gen-      stably co-express 57 amino acids human <br /> erated model recapitulates early phase           huntingtin (htt) fused CFP control the <br /> expanded polyQ neurotoxicity, neuronal dys-            promoter mec-3 gene YFP control <br /> function absence cell death [13,14]. In       promoter mec-7 gene. We previously <br /> animals, exon 1 like htt fused CFP expressed        reported ID245 animals moderate loss of <br /> touch receptor neurons control promo-        response light touch tail ID1 animals <br /> ter mec-3 gene (mec-3p) [14]. This promoter        strong loss response light touch tail, <br /> active early stages embryonic development         phenotype accompanied axonal swelling <br /> through adulthood [15]. Young adults expressing normal        [14]. The enhanced RNAi strain YY13 carries the <br /> polyQs (19Q) moderate loss touch response           rrf-3(mg373) mutation [19] provided Caenor- <br /> with little change touch cell morphology,            habditis Genetics Center, funded NIH <br /> whereas young adults expressing expanded polyQs               National Center Research Resources (NCRR). To <br /> (128Q) dramatic loss touch response, beha-        construct polyQ strains enhanced RNAi sensitivity <br /> vioral phenotype accompanied strong level        neurons, crossed YY13 animals ID245 <br /> of axonal dystrophy [14]. Axonal dystrophy followed        ID1 animals generate strain ID448 (igIs245[mec- <br /> by late-onset axonal degeneration observed older        3p::htt57-19Q::CFP;lin-15(+);mec-7p::YFP];rrf-3(pk1426)) <br /> adult animals (unpublished data). The loss touch           strain ID447 (igIs1[mec-3p::htt57-128Q::CFP;lin-15 <br /> response expanded-polyQ <span id='am-74' about='obo:NCBITaxon_333870' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_195229 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41087 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_71193 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-75' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="Nematodes Berthold, 1827" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="rdfs:label" content="Nematodes" datatype="xsd:string"></span>nematodes</span> manipu-           (+);mec-7p::YFP];rrf-3(pk1426)), respectively. The ID448 <br /> lated genetic means, exemplified protective      ID447 strains tested sensitivity touch <br /> effect sirtuin overexpression aggravating effect    receptor neurons RNAi subsequently used to <br /> loss-of-function sir-2.1/SIRT1&#8211;daf-16/FOXO cell        screen C. elegans ORF-RNAi library (Open Biosys- <br /> survival pathway [14]. These animals provide       tems, AL) includes clones C. elegans <br /> vivo model screen modifiers neuron         ORFeome library v1.1 [20] detailed below. <br /> dysfunction induced expanded polyQs. Here, used <br /> 128Q nematodes large-scale functional RNA inter-         RNAi screen <br /> ference (RNAi) screen modulation response           Bacterial clones C. elegans ORF-RNAi library <br /> touch. Additionally, used 19Q nematodes test          grown 500 &#956;l LB culture medium supplemen- <br /> subset 128Q-confirmed hits modulation           ted ampicillin (100 &#956;g/ml) tetracyclin (12.5 &#956;g/ <br /> response touch 19Q nematodes. To end,           ml), induced adding 10 &#956;l IPTG <br /> generated polyQ strains enhanced response             (12.5 mg/ml) 4 hours prior incubation nema- <br /> touch receptor neurons double-standed RNA deliv-           todes. Synchronized L1 larvae obtained hypo- <br /> ered feeding. We undertook semi-directed            chlorite extraction using standard methods [16] and <br /> screening approach test total 6034 genes includ-      grown 20&#176;C Petri dishes they <br /> ing 5128 genes tested random 906 genes            reach young adult stage. The day incubation in <br /> belong major signaling pathways (16-19). This led       96-well plates (D0), young adult animals incu- <br /> to identify 662 genes reduce aggravate     bated well. Each wells contained 33.6 &#956;l of <br /> loss response light touch knocked-down          bacterial culture 16.6 &#956;l M9 media supplemented <br /> RNAi 128Q nematodes. Screening 19Q nematodes            cholesterol, ampicillin, tetracyclin, fungizone and <br /> suggested greater 90% genes specifi-       IPTG. Incubation performed 20&#176;C 4 days <br /> cally modify 128Q cytotoxicity. In-depth data analysis        animals reached adulthood produce late <br /> emphasized role cell differentiation/survival, <span id='am-69' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-70' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-72'  ></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>meta</span>-    L4 larvae. To ensure appropriate feeding nema- <br /> bolic neurodegenerative disease pathways           todes well, 16.6 &#956;l bacterial cultures expressing <br /> modulation expanded-polyQ cytotoxicity. Addition-          RNAi clones added day 1. At day 4, animals <br /> ally, data analysis comparisons gene expression        transferred agar plates allowed recover <br /> data HD mice emphasized potential value           15 minutes. RNAi clones producing lethality, larval <br /> this RNAi dataset target discovery HD.                 arrest, egg-laying defect, developmental delay any <br />  Lejeune et al. BMC Genomics 2012, 13:91                                                                     Page 3 14 <br /> http://www.biomedcentral.com/1471-2164/13/91 <br />  <br />  <br />  <br />  <br /> other abnormalities morphological abnormalities     methods. All nematode experiments per- <br /> (2243 RNAi clones) excluded analysis.     formed times. Student&#8217;s t-tests used <br /> For RNAi clones did alter nematode growth          statistical analysis YFP mRNA levels YFP <br /> and morphology, P0 animals (old dark            expression levels. <br /> nematodes) initially incubated manually <br /> removed plate young adults (F1 progeny)        Data analysis <br /> were assayed light touch response previously         The final screening results initially classified using <br /> described [13]. Briefly, touch tests involved scoring   Gene Ontology (GO) annotations Biological Process, <br /> the response light touch tail using fine     Molecular Function Cellular Component http:// <br /> hair. Touch test performed scoring 5 touches     www.geneontology.org/[21]. GO enrichment tests were <br /> the tail animal 70-90 animals/RNAi clone/test.   performed using Ontologizer v 2.0 [22]. Additionally, we <br /> Ordinarily, wild-type animals respond backing       used hypergeometric tests test gene set enrich- <br /> away touch. The responses recorded        ment [23,24]. The gene sets used enrichment analy- <br /> every animal that, example, 4 responses     sis established using core components (i) <br /> 5 tail given 80% responsiveness,       canonical pathways based Wormbook http://www. <br /> mean values responsiveness retained    wormbook.org, KEGG http://www.genome.ad.jp/kegg/ <br /> analysis data. A score change touch           [25] Panther http://www.pantherdb.org/pathway/. <br /> response (S) calculated [(Percent response -           To analyze biological content RNAi <br /> mean baseline)/mean baseline]. Using formula,      clones confirmed modulate 128Q toxicity, per- <br /> maximally-achievable score 4.55 (maximal suppression     formed network-boosted data analysis. To end, we <br /> of 128Q cytotoxicity, i.e. 100% responsiveness)     subjected RNAi data spectral decomposition ana- <br /> minimally-achievable score -1 (maximal aggravation       lysis using integrated C. elegans network Wormnet <br /> of 128Q cytotoxicity). The baseline assay        v.2 [26]. This large-coverage network contains gene <br /> monitored entire time screening      interaction data established according sev- <br /> the incubation animals treated        eral types evidences derived multiple organisms, <br /> vector L4440 96-well plates assayed. The      including types gene interactions based on <br /> cumulated baseline entire time screening    gene co-expression (microarray measurements the <br /> was observed stable (20.6 &#177; 3%) cumu-         expression C. elegans mRNAs), protein-protein inter- <br /> lated baseline/month used calculate S scores. Sta-   actions including genome-wide yeast two-<span id='am-107' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-108' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-111'  ></span>hybrid</span> interac- <br /> tistical analysis performed using one-way ANOVA         tions C. elegans proteins protein-protein <br /> and Tukey&#8217;s Multiple Comparison Test. To decrease       interactions Worm Interactome database [27], <br /> the risk false positives, RNAi clones scored    genetic interactions WormBase [28], human and <br /> positive hits (suppressors enhancers) S &#177; SD      fly &#8216;interologs&#8217; i.e. human fly protein interactions <br /> value outside baseline &#177; (SD*2.6). Initial hits     [29,30], Yeast &#8216;associalogs&#8217; i.e. functional interactions <br /> (suppressors, enhancers) tested confirmation    transferred yeast functional gene network [31], <br /> invalidation effect touch response result-    phylogenic profiles (comparative genomics linkages <br /> ing data retained biological content analysis.     derived analysis bacterial archaeal gen- <br /> All screening assays performed investiga-    omes), gene neighbors co-citation (literature-mined <br /> tors trained score touch response 128Q      C. elegans gene associations). To increase power of <br /> animals similar way using blind assays.                spectral decomposition analysis detecting pathways, <br />    Prior performing RNAi screen, efficiency   used homogenous network containing <br /> gene inactivation touch receptor neurons tested      high confidence interactions, Core Wormnet <br /> by incubating ID447 L1 larvae 96-well plates (1 lar-     [26] omitted co-expression co-citation <br /> vae/well) bacteria expressing RNAi      links. To perform spectral decomposition analysis, we <br /> GFP (plasmid pL4417) vector (plasmid               developed enhanced version previously <br /> pL4440). Incubation performed described        described [32] Fourier-like analysis. This approach is <br /> until F1 young adults produced. At end incu-     based spectral decomposition gene activity <br /> bation, 70-90 animals/genotype moved agar           profiles respect connectivity reference <br /> plates mounted slides, YFP levels touch       network. S scores RNAi screen considered as <br /> cells (mec-7p target cells) measured high (63X)     &#8216;signal&#8217; Fourier transformation decomposes <br /> magnification using fluorescence microscope (Zeiss        signal superposition signals different fre- <br /> Axioplan Imaging II) image analysis software            quencies, 40 successive attenuations performed <br /> (Image J). Additionally, YFP mRNA levels <span id='am-58' about='obo:NCBITaxon_1178312' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_572719 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41011 obo:NCBITaxon_41197 obo:NCBITaxon_41012 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-59' property="rdfs:label" content="Mea" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-62'  ></span>mea</span>-          [32]. The aim transformations the <br /> sured qRT-PCR animals using standard          low-frequency signals discard high frequency <br />  Lejeune et al. BMC Genomics 2012, 13:91                                                                       Page 4 14 <br /> http://www.biomedcentral.com/1471-2164/13/91 <br />  <br />  <br />  <br />  <br /> signals, considered noise, context gene con-     YFP expression levels mec7-p target cells illustrated <br /> nectivity. The logic procedure highly     Figure 1A/B. The level YFP mRNA expression was <br /> inter-connected genes implicated           strongly reduced (about 80%) RNAi GFP <br /> pathway fluctuate direction [32].         compared vector (Figure 1C). Additionally, the <br /> Starting raw RNAi screen data, iteratively        level YFP expression mechanosensory neurons <br /> removed high-frequency components 2.5% time,         control response touch tail (PLM neu- <br /> resulting 40 increasingly smoothed versions         rons) reduced RNAi GFP compared to <br /> data. Given smoothed gene activity data, consid-         vector (Figure 1D). Collectively, results <br /> ered modules groups connected genes signal       indicated touch receptor neurons F1 young <br /> &gt; mean S-score + SD &lt; mean S-score-SD. Addition-           adults sensitive RNAi. Consistent previous <br /> ally, generate relevant signal        reports [37], little reduction YFP levels was <br /> robust information, selected modules         detected F0 young adults (data shown). <br /> present 4 th attenuation showed a <br /> stable gene content (&#8805; 40% genes value          A large-scale functional RNAi screen identifies 662 genes <br /> over threshold) 2 successive attenuations.   modulate neuron dysfunction expanded- <br /> The resulting modules filtered redundant gene        polyglutamine nematodes <br /> content functional content remaining mod-          Using 128Q nematodes rrf-3 sensitized back- <br /> ules analyzed using Ontologizer v 2.0 [22]. Additionally,     ground, conducted primary RNAi screen for <br /> we used hypergeometric tests annotate signaling        modulation neuronal dysfunction. This screen <br /> pathways (gene lists extracted Wormbook, KEGG            based manual quantification response to <br /> and Panther), druggable genes [33], htt partners [7,34]       light touch tail F1 young adult animals. We <br /> and genes involved autophagy, mitochondrial function       screened 6260 RNAi clones equaling 6034 genes the <br /> and synaptic activity (gene lists kindly provided      C. elegans genome (Figure 2). To decrease risk for <br /> CHDI Foundation). Information conservation              false positive, applied stringent threshold where <br /> humans derived Inparanoid clusters [35]. In          modification touch response considered sig- <br /> addition tables included herein, results net-       nificant score touch response outside the <br /> work-boosted data analysis displayed freely-         mean baseline &#177; SD*2.6 (Additional file 1: Figure S1). <br /> accessible database (see http://www.broca.inserm.fr/          This screen identified 2243 RNAi clones (2211 genes) <br /> EHDN2/RNAiscreen) vizualisation modules.           caused toxicity phenotypes lethality, larval <br /> This database allows users browse results          arrest, egg-laying defect, developmental delay other <br /> using GO/KEGG/Panther annotations annota-           abnormalities morphological abnormalities, 3101 <br /> tions interest, uses VisANT 3.5 graph           RNAi clones (3054 genes) showed effect and <br /> visualization [36]. We used confidence graphs dis-        916 RNAi clones (769 genes) modulated neuronal <br /> play modules normalizing statistical weights        dysfunction (Figure 2, Additional file 2: Table S1; Addi- <br /> for functional interactions available Wormnet v 2.0     tional file 3: Table S2). The <span id='am-50' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-51' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> RNAi clones <br /> [26].                                                         subjected second screen 128Q nematodes, <br />                                                               led identification 664 RNAi clones con- <br /> Results                                                       firmed modulate neuronal dysfunction. These clones <br /> The touch receptor neurons F1 128Q;rrf-3 animals       corresponded 662 genes, consisting 399 genes that <br /> sensitive RNAi                                             reduced neuronal dysfunction knocked-down by <br /> To construct polyQ strains enhanced RNAi sensi-          RNAi 263 genes enhanced neuronal dysfunc- <br /> tivity neurons, crossed enhanced RNAi strain        tion knocked-down RNAi (Additional file 4: <br /> YY13 carries rrf-3(mg373)II mutation [19]           Table S3). As previously reported, 128Q nematodes <br /> with integrated strains co-express N-terminal            80% decreased touch response tail <br /> (amino acids 1-57) htt control mec3-p        strong level axonal dystrophy, 19Q <br /> YFP control mec7-p [14]. This resulted        nematodes 45-50% reduction touch response <br /> strain ID448 (19Q;rrf-3) strain ID447(128Q;rrf-3).        tail significant axonal dystrophy [14]. <br /> The efficiency gene inactivation touch receptor         Since screen based manual quantifica- <br /> neurons tested incubating strain ID447 L1 larvae       tion touch response 70-90 animals/RNAi clone, <br /> in 96-well plates bacteria expressing RNAi        considerably labour-intensive time- <br /> against GFP (plasmid L4417) vector (plasmid          consuming compared faster RNAi screens as <br /> L4440) F1 young adults produced. Treatment          looking modifiers mutant htt aggregation <br /> with RNAi GFP induced overall reduction         [38,39], evaluated modifiers 128Q- <br />  Lejeune et al. BMC Genomics 2012, 13:91                                                                                               Page 5 14 <br /> http://www.biomedcentral.com/1471-2164/13/91 <br />  <br />  <br />  <br />  <br />  Figure 1 Reduction YFP expression 128Q;rff-3 nematodes treated RNAi GFP. Following incubation ID447 L1 larvae <br />  bacteria expressing vector (pL4440) RNAi GFP (pL4417) F1 young adults produced, YFP levels PLM <br />  touch receptor cells measured image analysis, YFP mRNA levels measured qRT-PCR animals. (A) Representative <br />  image showing YFP expression (pseudocolored green) animals treated vector. The mec-7p target cells indicated. <br />  The insert shows closer view left (PLMl) right (PLMr) PLM cell. (B) Representative image showing reduced YFP expression animals <br />  treated RNAi GFP. The yellow arrow indicates natural auto-fluorescence gut granules. This fluorescence present the <br />  control animals works internal control light intensity, showing pictures collected confocal microscope the <br />  conditions. The insert shows closer view PLM cell left (PLMl) right (PLMr). (C) YFP mRNA levels reduced treatment with <br />  pL4417. Data means &#177; SE (n = 5). ***P &lt; 0.001 compared pL4440. (D) YFP expression levels PLM cells reduced treatment with <br />  pL4417. Data means &#177; SEM (n = 3) compiled greater 70 animals/experiment. ***P &lt; 0.001 compared pL4440. Scale bar 100 <br />  &#956;m panels. <br />  <br />  <br /> neuron dysfunction modify neuronal dys-                         In-depth data analysis emphasizes modifiers 128Q- <br /> function 19Q;rrf-3 nematodes testing subset                     neuron dysfunction pathways associated with <br /> the 662 genes (245 genes choosen random). Only 15/                      metabolism, development, survival and <br /> 245 genes (6.1%) modulate neuronal dys-                      neurodegenerative disease <br /> function 19Q;rrf-3 nematodes knocked-down                          The 662 genes associated modification of <br /> by RNAi (Additional file 5: Table S4), suggesting                     128Q-neuron dysfunction appeared encompass vari- <br /> most 662 genes emphasized screens 128Q;                       ety biological processes (cell death, protein folding, <br /> rrf-3 nematodes specifically modify 128Q cytotoxicity.                     intracellular transport, metabolic processes, response to <br /> Genome-wide RNAi screens previously aimed identi-                       stress, stress-activated pathways) role in <br /> fying modifiers polyQ aggregation C. elegans                         neurodegenerative disease pathogenesis suggested by <br /> [40,41], Consistent notion aggregation                   functional classification using GO annotations (Fig- <br /> cytotoxicity genetically uncoupled [41], over-                  ure 3, Additional file 7: Tables S6; Additional file 8: <br /> lap RNAi screen small 12 modifiers                       Table S7). To assess biological content cor- <br /> of 128Q-neuron dysfunction previously                             responding modifiers 128Q-neuron dysfunction, <br /> enhance aggregation 35Q-peptides                               performed types analysis. First, used gene <br /> knocked-down RNAi [40] modifiers                        lists describing major signaling pathways (see Materials <br /> of 128Q-neuron dysfunction previously sup-                        Methods) perform gene set enrichment analyses. <br /> press aggregation 35Q-peptides knocked-                        This analysis performed genes aggra- <br /> down RNAi [41] (Additional file 6: Table S5).                           vate (herein referred &#8216;protection-associated&#8217; PA <br />  Lejeune et al. BMC Genomics 2012, 13:91                                                                         Page 6 14 <br /> http://www.biomedcentral.com/1471-2164/13/91 <br />  <br />  <br />  <br />  <br />  Figure 2 Overview screening data analysis strategy. <br />  <br />  <br /> genes) suppress (herein referred &#8216;pathogenic             Toll receptor signaling [44]. These observations <br /> process-associated&#8217; PPA genes) 128Q-neuron dys-                illustrated revelance RNAi dataset neuro- <br /> function knocked-down RNAi. Pathway analy-                degeneration. Given PPA genes constitute <br /> sis PA genes identified metabolic pathways              preferred targets therapeutic intervention, these <br /> such example involved glycolysis              results suggested targets early- <br /> amino acid metabolism (Table 1). Consistent              stage neuroprotection HD lie pathways <br /> notion mutant htt expression dysregulate               important neuronal cell differentiation and <br /> large number genes alter variety biological           survival. <br /> processes [3,10], pathway analysis PPA genes identi-          In contrast metabolic pathways a <br /> fied main pathway categories, including pathways            simple structure, pathways respond exter- <br /> primarily involved metabolism (glycolysis, amino acid          nal stimuli regulate gene expression highly <br /> metabolism), cytokine signaling/immune (inter-             inter-connected, highly-branched <br /> feron-gamma signaling, interleukin signaling, Toll recep-         structures biological outcomes. Additionally, <br /> <span id='am-92' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-93'  ></span><span id='am-94' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>tor</span> signaling) cellular differentiation                   RNAi-based screens generate noise off-tar- <br /> homeostasis (IGF pathway, angiogenesis, integrin signal-          effects variations degree knockdown of <br /> ing, p53 pathway glucose deprivation, FGF signaling,           target genes. Therefore, map RNAi activity func- <br /> EGF signaling, Toll receptor signaling) (Table 2). Inter-         tional clusters identify reliable set of <br /> estingly, category included Alzheimer&#8217;s dis-       genes RNAi hits, used network-based <br /> ease (AD)-amyloid secretase pathway, pathways                     approach. To end, used Fourier-like analysis <br /> previously associated neuroprotection HD            [32] based reference network Wormnet [26]. To <br /> IGF signaling [14] FGF signaling [42], pathways          enhance discriminative power reliability this <br /> could play role neurodegenerative disease pathogen-          analysis, developed applied optimized version <br /> esis EGF signaling [43], pathways having            Fourier-like analysis (see Materials Methods). <br /> dual role neurodegeneration neuroprotection                We analyzed gene set score activity <br />  Lejeune et al. BMC Genomics 2012, 13:91                                                                                                  Page 7 14 <br /> http://www.biomedcentral.com/1471-2164/13/91 <br />  <br />  <br />  <br />  <br />  Figure 3 Gene Ontology classification genes modulate neuronal dysfunction induced expanded polyQ expression. Genes <br />  classified based functional annotations GO categories &#8216;Biological Process&#8217;, &#8216;Molecular Function&#8217; &#8216;Cellular component&#8217;. Pie- <br />  charts frequent &#8216;Biological process&#8217; categories (corresponding 18 genes) shown gene inactivations reduced <br />  128Q-neuron dysfunction (A) aggravated neuronal dysfunction (B). The number genes indicated. See indicated <br />  supplementary tables (Supplementary Tables 4-5) complete lists genes GO category. <br />  Lejeune et al. BMC Genomics 2012, 13:91                                                                                                    Page 8 14 <br /> http://www.biomedcentral.com/1471-2164/13/91 <br />  <br />  <br />  <br />  <br /> Table 1 Pathway analysis 263 genes enhanced 128Q-neuron dysfunction knocked-down RNAi <br /> Pathway                              P value Gene <br /> Ascorbate aldarate               0.00000 B0218.8/clec-52, C01G6.1/aqp-2, C08B11.8, C12C8.2, C13B9.1, C36E8.3/pxd-1, C50D2.8, C52B9.1/cka-2, <br /> metabolism                                   C52E4.1/cpr-1, C53B7.3, C54C8.9/nlp-39, D1025.4/nspc-20, D1086.3, F08B1.1/vhp-1, F29G6.2, F38E11.1/hsp- <br />                                              12.3, F45C12.7/btb-6, F46H5.3, F49E12.1, F54B11.3/unc-84, F54F7.3, F56D6.2/clec-67, K07E3.3/dao-3, <br />                                              K08D8.3, K09C4.5, M02F4.7/clec-265, M03F4.7/calu-1, T07H3.3/math-38, T16H12.1, T23B3.2, T23H2.2/snt-4, <br />                                              W08D2.4/fat-3, Y38H6C.3/dct-14, ZC410.5, ZC416.6, ZK6.11 <br /> Starch sucrose metabolism        0.00119 F57B10.7/tre-1, R05F9.6, T22F3.3, Y87G2A.8/gpi-1 <br /> Tyrosine metabolism                  0.00535 B0334.11/ooc-3, F46C3.1/pek-1, M110.5/dab-1 <br /> Amino sugar nucleotide           0.01885 F22B3.4, R05F9.6, Y87G2A.8/gpi-1 <br /> sugar metabolism <br /> Glycolysis                           0.03209 T03F1.3/pgk-1, Y87G2A.8/gpi-1 <br /> Glycolysis/Gluconeogenesis           0.03332 R05F9.6, T03F1.3/pgk-1, Y87G2A.8/gpi-1 <br /> Fatty acid biosynthesis              0.03415 W09B6.1/pod-2 <br /> Pentose phosphate pathway            0.04375 R05F9.6, Y87G2A.8/gpi-1 <br /> Pathway analysis based hypergeomeric tests using pathways Wormbook, KEGG Panther. P &lt; 0.05 considered significant results are <br /> shown increasing P values. <br />  <br />  <br /> Table 2 Pathway analysis 399 genes reduced 128Q-neuron dysfunction knocked-down RNAi <br /> Pathway                                 P value Gene <br /> Ascorbate aldarate metabolism       0.00000 B0281.5, B0286.3, B0478.1/jnk-1, C06B3.4/stdh-1, C08E3.4/fbxa-161, C08F8.5/fbxb-9, C09G1.1/pqn-11, <br />                                                 C10C5.4, C10C5.5, C10G11.5/pnk-1, C17G10.5/lys-8, C17H12.8, C39D10.7, C45B11.3/dhs-18, C46E10.7/ <br />                                                 srh-99, C54D10.3, D1054.10, D2045.6/cul-1, F07F6.1, F07F6.5/dct-5, F09B12.3, F17E5.1/lin-2, F28D1.5/ <br />                                                 thn-2, F29G9.1, F31C3.6, F36F2.1, F38E11.2/hsp-12.6, F38H4.8/ech-2, F41B5.4/cyp-33C3, F42G10.1, <br />                                                 F43G9.5, F44D12.8, F47C10.2/btb-21, F52H3.5, F53C3.5, F56D5.5, F56G4.3/pes-2.2, F57C9.4, F57H12.7/ <br />                                                 mec-17, H04M03.1, H10D18.2/scl-12, K02B12.2, K02E11.5, K10B3.8/gpd-2, M01E11.2, M02D8.4, R03E9.1/ <br />                                                 mdl-1, R05F9.10/sgt-1, R12B2.5/mdt-15, R12H7.2/asp-4, T03F6.1/qdpr-1, T17H7.1, T20B5.3/oga-1, <br />                                                 T21C9.8/ttr-23, T22G5.2/lbp-7, T24B8.5, W02D3.1, W03D2.6, W05H9.1, Y106G6H.7/sec-8, Y106G6H.9, <br />                                                 Y38H6C.1/dct-16, Y43C5B.2, Y47D3B.7/sbp-1, ZK1127.10, ZK1290.6/rnh-1.1, ZK384.2/scl-20, ZK520.4/ <br />                                                 cul-2, ZK550.6, ZK637.8/unc-32, ZK829.6/tgt-1 <br /> Sulfate assimilation                    0.00001 B0218.3/pmk-1, B0412.2/daf-7, B0478.1/jnk-1, C47G2.2/unc-130, C53D6.2/unc-129, F38A6.1/pha-4, <br />                                                 F43C1.2/mpk-1, T23H2.5/rab-10, Y11D7A.4/rab-28 <br /> Pyrimidine metabolism                   0.00113 B0304.1/hlh-1, B0478.1/jnk-1, B0547.1/csn-5, C04E7.2/sor-3, C07B5.5/nuc-1, C25D7.3/sdc-3, C34E10.7/ <br />                                                 cnd-1, C37A2.4/cye-1, C47G2.2/unc-130, F14F3.1/vab-3, F38A6.1/pha-4, F56A8.7/unc-64, K01G5.2/hpl- <br />                                                 2, Y47D3A.6/tra-1, ZK520.4/cul-2 <br /> Pyridoxal phosphate salvage             0.00176 B0218.3/pmk-1, B0478.1/jnk-1, F43C1.2/mpk-1, Y18D10A.5/gsk-3, Y38F1A.10/max-2 <br /> pathway <br /> Oxytocin receptor mediated              0.00236 B0478.1/jnk-1, B0547.1/csn-5, C44H4.6, F43C1.2/mpk-1, Y11D7A.4/rab-28, Y18D10A.5/gsk-3 <br /> signaling <br /> Interferon-gamma signaling              0.00251 B0218.3/pmk-1, B0478.1/jnk-1, F43C1.2/mpk-1 <br /> Glycosphingolipid biosynthesis -        0.00297 T14F9.3/hex-1, T26C5.3 <br /> ganglio series <br /> Cyanoamino acid metabolism              0.00340 B0035.8/his-48, B0304.1/hlh-1, B0478.1/jnk-1, B0547.1/csn-5, C02B8.4/hlh-8, C04E7.2/sor-3, C07B5.5/ <br />                                                 nuc-1, C09G9.7, C18A3.1, C25A1.11/aha-1, C25D7.3/sdc-3, C29F9.5, C34E10.7/cnd-1, C37A2.4/cye-1, <br />                                                 C47G2.2/unc-130, F11A10.1/lex-1, F14F3.1/vab-3, F22D3.1/ceh-38, F22F1.1/hil-3, F38A6.1/pha-4, <br />                                                 F45F2.4/his-7, F54C8.2/cpar-1, F56A8.7/unc-64, K01G5.2/hpl-2, K02B9.4/elt-3, R03E9.1/mdl-1, R05F9.10/ <br />                                                 sgt-1, R07B1.1/vab-15, R13H8.1/daf-16, T05A6.1/cki-1, T19B10.11/mxl-1, T27F2.1/skp-1, W06E11.1, <br />                                                 Y39B6A.2/pph-5, Y47D3A.6/tra-1, Y47D3B.7/sbp-1, Y49E10.1/rpt-6, Y57E12AL.5/mdt-6, ZC204.2, <br />                                                 ZK131.7/his-13, ZK520.4/cul-2, ZK652.5/ceh-23 <br /> Taurine/hypotaurine metabolism          0.00348 F56A8.7/unc-64, K02D10.5, T10H9.3, Y22F5A.3/ric-4 <br /> Glycolysis/Gluconeogenesis              0.00365 AC3.7/ugt-1, C05C8.3/fkb-3, C06B3.4/stdh-1, C07B5.5/nuc-1, F11A5.12/stdh-2, F38E11.2/hsp-12.6, <br />                                                 K10B3.8/gpd-2, R03E9.1/mdl-1, R12B2.5/mdt-15, R13H8.1/daf-16, T22G5.2/lbp-7, T28B8.2/ins-18 <br /> Selenocompound metabolism               0.00365 AC3.7/ugt-1, C05C8.3/fkb-3, C06B3.4/stdh-1, C07B5.5/nuc-1, F11A5.12/stdh-2, F38E11.2/hsp-12.6, <br />                                                 K10B3.8/gpd-2, R03E9.1/mdl-1, R12B2.5/mdt-15, R13H8.1/daf-16, T22G5.2/lbp-7, T28B8.2/ins-18 <br /> Insulin_IGF pathway-protein kinase B 0.00404 C44H4.6, C47G2.2/unc-130, F38A6.1/pha-4, Y18D10A.5/gsk-3 <br /> signaling cascade <br /> Salvage pyrimidine ribonucleotides      0.00417 F42G9.7/snt-2, K02D10.5, Y22F5A.3/ric-4 <br /> Tetrahydrofolate biosynthesis           0.00417 B0218.3/pmk-1, B0478.1/jnk-1, F43C1.2/mpk-1 <br /> Apoptosis signaling                     0.00503 B0218.3/pmk-1, B0478.1/jnk-1, F43C1.2/mpk-1, Y18D10A.5/gsk-3, Y38F1A.10/max-2 <br /> Glycosaminoglycan degradation           0.00520 K09E4.4, T14F9.3/hex-1 <br />  Lejeune et al. BMC Genomics 2012, 13:91                                                                                                    Page 9 14 <br /> http://www.biomedcentral.com/1471-2164/13/91 <br />  <br />  <br />  <br />  <br /> Table 2 Pathway analysis 399 genes reduced 128Q-neuron dysfunction knocked-down RNAi <br /> (Continued) <br /> Angiogenesis                            0.00520 C06B3.4/stdh-1, F11A5.12/stdh-2, K04A8.5 <br /> mRNA splicing                           0.00534 F56A8.7/unc-64, K02D10.5, Y22F5A.3/ric-4, Y48B6A.8/ace-3 <br /> Muscarinic acetylcholine receptor 1     0.00608 F56A8.7/unc-64, K02D10.5, Y22F5A.3/ric-4, Y48B6A.8/ace-3 <br /> and 3 signaling <br /> Alanine biosynthesis                    0.00769 K02D10.5, K08F8.4/pah-1, Y22F5A.3/ric-4 <br /> Butanoate metabolism                    0.00777 C06B3.4/stdh-1, F11A5.12/stdh-2, K03A1.5/sur-5, T02G5.7 <br /> 5-Hydroxytryptamine biosynthesis        0.00798 K02D10.5, Y22F5A.3/ric-4 <br /> Interleukin signaling                   0.00974 C47G2.2/unc-130, F38A6.1/pha-4, F43C1.2/mpk-1, Y18D10A.5/gsk-3 <br /> Amino sugar nucleotide sugar        0.01084 C01F1.3, C53B4.7/bre-1, K08E3.5, T14F9.3/hex-1 <br /> metabolism <br /> Metabotropic glutamate receptor         0.01254 F56A8.7/unc-64, K02D10.5, Y22F5A.3/ric-4 <br /> group II pathway <br /> Linoleic acid metabolism                0.01511 C06B3.4/stdh-1, F11A5.12/stdh-2 <br /> Beta3 adrenergic receptor signaling     0.01511 K02D10.5, Y22F5A.3/ric-4 <br /> Integrin signaling                      0.01742 B0218.3/pmk-1, B0478.1/jnk-1, C27B7.8/rap-1, C36B1.1/cle-1, F43C1.2/mpk-1 <br /> p53 pathway glucose deprivation      0.01908 C47G2.2/unc-130, F38A6.1/pha-4, F48E8.1/lon-1, Y18D10A.5/gsk-3 <br /> 5-Hydroxytryptamine degredation         0.01939 K02D10.5, Y22F5A.3/ric-4 <br /> Ornithine degradation                   0.02121 B0218.3/pmk-1, B0478.1/jnk-1, T19B10.11/mxl-1 <br /> Cortocotropin releasing factor          0.02412 K02D10.5, Y22F5A.3/ric-4 <br /> receptor signaling <br /> FGF signaling                           0.02627 B0218.3/pmk-1, B0478.1/jnk-1, F26E4.1/sur-6, F43C1.2/mpk-1 <br /> N-acetylglucosamine metabolism          0.02712 F56A8.7/unc-64, K02D10.5, T02C12.1/hum-5, Y22F5A.3/ric-4, Y48B6A.8/ace-3 <br /> 5HT3 type receptor mediated             0.02928 K02D10.5, Y22F5A.3/ric-4 <br /> signaling <br /> B cell activation                       0.02939 B0218.3/pmk-1, B0478.1/jnk-1, F43C1.2/mpk-1 <br /> Fructose mannose metabolism         0.03565 C06B3.4/stdh-1, C53B4.7/bre-1, F11A5.12/stdh-2 <br /> Pantothenate CoA biosynthesis       0.04074 F25H9.6, T04G9.4 <br /> Alzheimer disease-amyloid secretase     0.04074 K02D10.5, Y22F5A.3/ric-4 <br /> pathway <br /> Beta1 adrenergic receptor signaling     0.04074 K02D10.5, Y22F5A.3/ric-4 <br /> Beta2 adrenergic receptor signaling     0.04074 K02D10.5, Y22F5A.3/ric-4 <br /> EGF receptor signaling                  0.04229 B0218.3/pmk-1, B0478.1/jnk-1, C27B7.8/rap-1, F43C1.2/mpk-1 <br /> Toll receptor signaling                 0.04494 C08B6.9/aos-1, F11A10.1/lex-1, F40G9.3/ubc-20, Y49E10.1/rpt-6 <br /> Alanine, aspartate glutamate        0.04494 C04E7.2/sor-3, C34E10.7/cnd-1, C47G2.2/unc-130, F57H12.7/mec-17 <br /> metabolism <br /> Serine <span id='am-6' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="rdfs:label" content="Glycine" datatype="xsd:string"></span>glycine</span> biosynthesis             0.04620 B0478.1/jnk-1, F43C1.2/mpk-1, Y38F1A.10/max-2 <br /> Lysine biosynthesis                     0.04869 C25A11.4/ajm-1, C36B1.1/cle-1, F17E5.1/lin-2, F22B5.1/evl-20, F43C9.4/mig-13, F56A8.7/unc-64, <br />                                                 Y22F5A.3/ric-4 <br /> Pathway analysis based hypergeomeric tests using pathways Wormbook, KEGG Panther. P &lt; 0.05 considered significant results are <br /> shown increasing P values. <br />  <br />  <br /> (S score) available (genes modulate                           Additional file 10: Table S9). Modules variable sizes <br /> 128Q-neuron dysfunction produce effect                             obtained (2-50 genes) suppression or <br /> knocked-down RNAi). The modules resulting                             aggravation 128Q-neuron dysfunction. Of note, small <br /> this analysis annotated using Gene Ontology,                          modules equally significant compared large mod- <br /> KEGG Panther pathways gene lists                          ules size resulting modules dri- <br /> (conserved genes, druggable genes, htt partners, genes                        ven stable best balance gene <br /> involved autophagy, mitochondrial function                             connectivity gene activity spectral <br /> synaptic activity). These analyses identified reli-                    decomposition process (see Materials Methods). <br /> able set 286 modifier genes distributed 137                          The modules content visualized using a <br /> modules suppression 128Q-neuron dysfunction                         freely-accessible database (see http://www.broca.inserm. <br /> RNAi 105 modules aggravation 128Q-neuron                           fr/EHDN2/RNAiscreen) contains query page and <br /> dysfunction RNAi (Additional file 9: Table S8;                             interactive graphical interface [36]. Our network-based <br />  Lejeune et al. BMC Genomics 2012, 13:91                                                                     Page 10 14 <br /> http://www.biomedcentral.com/1471-2164/13/91 <br />  <br />  <br />  <br />  <br /> approach confirmed trends observed gene set            intersection C. elegans RNAi data <br /> enrichment analysis, detection pathways         gene dysregulation data HD mice (Figure 4). Six <br /> involved metabolism, immune response develop-          49 genes including Acy1, Sh3gl3, Snw1, Aars, <br /> ment. Additionally, provided insights possible    Ppp5 Gpi1 oppositely dysregulated the <br /> mechanisms RNAi hits potentially interact           CHL2 mice compared R6/2 mice (Figure 4), illustrat- <br /> modify 128Q-neuron dysfunction. We notably detected           ing differences exist two <br /> modules seeded RNAi hits annotated           models molecular level. In contrast, genes that <br /> &#8216;Huntington&#8217;s disease&#8217;, &#8216;Parkinson&#8217;s disease&#8217; &#8216;pantho-    involved variety biological processes that <br /> tenate kinase activity&#8217;, previously associated     modulate 128Q-neuron dysfunction knocked- <br /> with neurodegeneration [45]. More largely, detected        RNAi dysregulated CHL2 or <br /> modules annotated biological processes established        R6/2 mice (Figure 4), illustrating data comparison <br /> or strongly suspected (see [3] review HD mechan-       using substractive analysis useful target <br /> isms) role neurodegenerative disease patho-      prioritisation. Notably, 12 genes suppress 128Q- <br /> genesis &#8216;oxidative phosphorylation&#8217;, &#8216;cholesterol     neuron dysfunction knocked-down RNAi C. <br /> biosynthesis&#8217; &#8216;cell cycle&#8217;. Interestingly, mod-   elegans observed up-regulated the <br /> ules annotated cell differentiation survival     CHL2 R6/2 mice, suggesting inhibiting these <br /> pathways &#8216;Wnt signaling&#8217;, pathway suspected         genes elicit neuroprotection HD. One them, <br /> to role HD pathogenesis [46], &#8216;apoptosis sig-       phenylalanine-4-hydroxylase Pah, appeared be <br /> naling&#8217; &#8216;integrin signaling&#8217;, suggesting        particularly interesting druggable gene that <br /> pathways targeted therapeutic intervention       acts upstream tyrosine-dopamine biosynthesis and <br /> in HD. Annotations metabolic pathways             increased dopamine levels previously asso- <br /> &#8216;sucrose metabolism&#8217;, &#8216;pentose phosphate pathway&#8217;         ciated early HD phenotypes hyperkinesia <br /> &#8216;cholesterol biosynthesis&#8217; detected, consistent     [49]. <br /> with notion metabolic pathways tar- <br /> geted therapeutic intervention HD [3,47]. Addi-        Discussion <br /> tionally, modules annotated genes         The aim study collect information the <br /> interest HD druggable genes (as notably            genes pathways act modifiers the <br /> observed modules suppression 128Q-neu-          early stage mutant htt toxicity HD neuron <br /> ron dysfunction RNAi; Additional file 9: Table         dysfunction. To end, used vivo model to <br /> S8), htt partners, genes involved autophagy, mito-     screen gene inactivations modulate neuronal <br /> chondrial function synaptic activity,       dysfunction induced expanded-polyQ expression in <br /> illustrated relevance dataset neuroprotec-      C. elegans, used network-boosted analysis to <br /> tive target discovery HD.                                  enhance identification candidate pathways that <br />                                                               evaluated mammalian systems. We <br /> Modifiers 128Q-neuron dysfunction C. elegans        identified 662 hits evaluation 6034 genes. <br /> dysregulated striatum CHL2 knock-in mice     Compared RNAi screens based biochemical pro- <br /> or R6/2 transgenic mice                                       cesses protein aggregation [38-40], relatively <br /> To assess relevance RNAi data can-      high number hits study likely reflect the <br /> didate target profiling discovery HD,          multigenic nature behavioral phenotypes as <br /> compared dataset striatal gene expression data         response touch changes polyQ- <br /> collected mouse models HD. Given 128Q            expanded htt expression levels. Previously-identified <br /> nematodes recapitulate progressive neuron dysfunction,        modulators neuron dysfunction expanded-polyQ <br /> which occur HD cell loss, compared           nematodes change transgene expression levels <br /> our RNAi data striatal gene expression data collected      [14,18], suggesting RNAi effects not <br /> from CHL2(Q150/Q150) knock-in mice, model               changes htt expression levels. <br /> that recapitulate progressive neuronal injury similar       In contrast 128Q-neuron dysfunction, neuronal <br /> to early course HD [48]. To reach      dysfunction produced normal polyQ expression in <br /> more specific level comparison, compared           trangenic nematodes directly relevant HD <br /> our data striatal gene expression data collected      pathogenesis. Microarray analysis gene expression in <br /> the R6/2 transgenic mice, model exhibits rapid         touch receptor cells isolated fluorescence-activated <br /> phenotypic changes induced short N-terminal           cell sorting indicated 18 2000 <br /> fragment mutant htt [10]. Among 662 modifiers          genes dysregulated 128Q expression dysregu- <br /> of 128Q-neuron dysfunction C. elegans, 239 genes           lated 19Q expression (submitted elsewhere), sev- <br /> were conserved mouse. Forty-nine genes            eral expanded-polyQ cytotoxicity we <br />  Lejeune et al. BMC Genomics 2012, 13:91                                                                                        Page 11 14 <br /> http://www.biomedcentral.com/1471-2164/13/91 <br />  <br />  <br />  <br />  <br />  Figure 4 Overlap 239 conserved modifiers 128Q-neuron dysfunction C-elegans genes dysregulated striatum <br />  CHL2 knock-in mice R6/2 transgenic mice. (A) Overlap dysregulated genes mouse striatum suppressors of <br />  128Q-neuron dysfunction mouse orthologs. Numbers genes indicated. The lists genes intersection nematode <br />  mouse datasets indicated graphs. (B) Overlap dysregulated genes mouse striatum enhancers 128Q- <br />  neuron dysfunction mouse orthologs. The numbers genes indicated. The lists genes intersection nematode and <br />  mouse datasets indicated graphs. Some genes oppositely dysregulated CHL2 mice compared R6/2 mice (yellow <br />  tags). In panels, red circles indicate C. elegans genes modify 128Q-neuron toxicity. *Druggable genes. <br />  <br />  <br /> previously identified showed effect normal polyQ                  screening perspective. This example illustrated <br /> cytotoxicity [14,18]. Additionally, observed                    N-acetylgalactosaminyltransferases (gly-9 and <br /> greater 90% modifiers 128Q-neuron dys-                   gly-3), <span id='am-35' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-36' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> enzymes involved neuron differen- <br /> function showed effect 19Q nematodes using                      tiation survival [50]. <br /> subset 243 genes identified RNAi screening                        A major feature network-boosted analysis to <br /> 128Q nematodes. Collectively, observations sug-                   reduce data complexicity highlighting individual <br /> gest mechanisms neuron dysfunction 19Q                   pathways possibly inter-connected pathways that <br /> nematodes [14] different underlying                  embedded modules seeded gene activity. <br /> strong level neuron dysfunction 128Q nematodes.                   In respect, network-boosted data analysis our <br /> While specific modifiers expanded-polyQ cytotoxicity                 RNAi dataset instructive compared sole <br /> might constitute preferred HD targets,                       use GO annotations gene set enrichment analysis. <br /> exclude possibility non-specific modifiers                  While role variety pathways suggested by <br /> neuron dysfunction polyQ nematodes belong                     gene set enrichment analysis, network-boosted analysis <br /> the core machinery neuron differentiation survi-                emphasized role pathways more <br /> val, potential interest,                  comprehensive fashion, providing new insights how <br />  Lejeune et al. BMC Genomics 2012, 13:91                                                                                     Page 12 14 <br /> http://www.biomedcentral.com/1471-2164/13/91 <br />  <br />  <br />  <br />  <br /> the genes associated pathways interact            chaperone [51], aggravates neuron dysfunction when <br /> within modules. Furthermore, network-boosted analysis          knocked-down RNAi down-regulated the <br /> identified pathways detected gene set enrichment        CHL2 mice, (ii) sel-8, C. elegans gene encodes a <br /> analysis. As indicated pathways embedded         nuclear protein required lin-12/NOTCH signaling, <br /> modules resulting network-boosted analysis (see           neuron survival activities [52], that <br /> the on-line database http://www.broca.inserm.fr/            aggravates 128Q-neuron dysfunction knocked- <br /> EHDN2/RNAiscreen), role neurodegenerative dis-           RNAi, (iii) jnk-1/JNK, gene promote HD <br /> ease pathways neurodevelopmental/cell survival             pathology rat model disease [53] sup- <br /> pathways modulation expanded-polyQ cyto-             presses 128Q-neuron dysfunction knocked-down <br /> toxicity emphasized, provide guidance         RNAi, (iv) pnk-1/PanK4, gene encodes a <br /> the selection candidate targets neuroprotection      panthotenate kinase suppresses 128Q-neuron dys- <br /> HD. The annotation modules resulting net-          function knocked-down RNAi, consis- <br /> work-boosted analysis identified clusters enriched          tent pathological role panthotenate kinases <br /> genes druggable genes, htt partners        neurodegeneration [45]. Additionally, comparison <br /> and genes involved autophagy, mitochondrial function        data gene dysregulation striatum of <br /> and synaptic activity (all gene classes quer-     CHL2 knock-in mice versus R6/2 transgenic <br /> ied using on-line database), providing additional gui-     mice indicated dataset contains infor- <br /> dance neuroprotective target discovery HD. Here,         mation relevant HD target selection illustrated by <br /> it important note hypothesis underlying         proposed inhibition phenylalanine-4-hydroxylase <br /> network-based approaches highly-interconnected         candidate strategy reduce early HD phenotypes <br /> genes similar biological activities. In        associated transient excess dopamine <br /> respect, network-boosted analysis data pre-      [49]. <br /> dictive genes decrease increase <br /> 128Q-neuron dysfunction knocked-down                   Conclusions <br /> RNAi, showed effect borderline          In conclusion, results provide new insights the <br /> effects RNAi screen. While increase        genes pathways regulate dysfunction <br /> number candidate targets considered neuro-        death neurons expressing mutant polyQs, <br /> protective intervention, decreases risk noise       emphasizing role specific metabolic, developmental <br /> associated RNAi screens favors inclusion          pro-survival pathways. Several models HD are <br /> of potentially-interesting information multi-model         used screen modifiers mutant polyQ <br /> target discovery.                                              toxicity genes dysregulated mutant htt. In <br />    Multi-model target discovery strongly expected        context, use cross-<span id='am-63' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-64' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> data integration for <br /> enhance drug discovery process neuroprotection         analysis RNAi dataset datasets is <br /> in HD. A standard method validation modi-           anticipated boost prediction prioritisation of <br /> fiers neuronal dysfunction instructed simple          neuroprotective targets HD, investi- <br /> models HD pathogenesis relies target validation       gated additional studies. <br /> the mouse. Because feasible test the <br /> mouse genes supported modifiers neuronal             Additional material <br /> decline simple models, outcome functional <br /> screen screens analyzed using data            Additional file 1: Figure S1. Distribution S scores primary <br /> integration procedures selection best       screen 128Q;rrf-3 nematodes. The graph shows S scores 4017 <br />                                                                 RNAi clones did elicit lethality developmental abnormalities. <br /> targets validation rely target expression levels    The S score calculated [(Percent response - mean baseline)/mean <br /> and activity multiple models HD. One               baseline]. The maximally-achievable score 4.55 (100% response to <br /> objectives data integration assess relevance       touch) smallest score -1 (complete loss touch response). <br />                                                                 The blue line indicate mean baseline (S = 0). Dooted blue lines show <br /> of simple-model data mouse models, lar-             mean baseline &#177; SD*2.6. Clones S scores falling outside the <br /> gely human neurodegenerative disease. In                mean baseline &#177; SD*2.6 interval retained secondary screen. <br /> respect, notable modifiers 128Q-neu-      Additional file 2: Table S1. List 2211 genes caused lethality <br /> ron dysfunction screen previously             developmental abnormalities knocked-down RNAi. <br /> associated pathogenesis neurodegenerative           Additional file 3: Table S2. List 3823 genes showed <br />                                                                 effect modified 128Q-neuron dysfunction knocked-down by <br /> diseases HD AD, suggesting C. ele-          RNAi primary screen. <br /> gans dataset relevant HD pathogenesis higher           Additional file 4: Table S3. List 662 genes modified 128Q- <br /> order models disease. This example            neuron dysfunction knocked-down RNAi secondary <br /> case genes retained network-boosted analysis        screen. <br /> including (i) dnj-1/DNAJ, gene encodes DNAJ <br />  Lejeune et al. BMC Genomics 2012, 13:91                                                                                                           Page 13 14 <br /> http://www.biomedcentral.com/1471-2164/13/91 <br />  <br />  <br />  <br />  <br />                                                                                         Neuropathologic genetic evidence role Huntington&#8217;s <br />     Additional file 5: Table S4. List 15 genes modified 19Q-                disease pathogenesis. Proc Natl Acad Sci USA 2001, 98(4):1811-1816. <br />     neuron dysfunction knocked-down RNAi.                                 6.    Goehler H, Lalowski M, Stelzl U, Waelter S, Stroedicke M, Worm U, <br />     Additional file 6: Table S5. List 15 genes modified 128Q-               Droege A, Lindenberg KS, Knoblich M, Haenig C, et al: A protein <br />     neuron dysfunction secondary screen previously                      interaction network links GIT1, enhancer huntingtin aggregation, <br />     reported modify polyQ aggregation knocked-down RNAi.                     Huntington&#8217;s disease. Mol Cell 2004, 15(6):853-865. <br />     Additional file 7: Table S6. Gene Ontology classification genes       7.    Kaltenbach LS, Romero E, Becklin RR, Chettier R, Bell R, Phansalkar A, <br />     suppressed 128Q-neuron dysfunction knocked-down RNAi.                       Strand A, Torcassi C, Savage J, Hurlburt A, et al: Huntingtin interacting <br />                                                                                         proteins genetic modifiers neurodegeneration. PLoS Genet 2007, <br />     Additional file 8: Table S7. Gene Ontology classification genes             3(5):e82. <br />     aggravated 128Q-neuron dysfunction knocked-down RNAi.                 8.    Holbert S, Dedeoglu A, Humbert S, Saudou F, Ferrante RJ, Neri C: Cdc42- <br />     Additional file 9: Table S8. Modules (n = 137) generated network-                interacting protein 4 binds huntingtin: neuropathologic and <br />     boosted analysis suppression 128Q-neuron dysfunction RNAi.                biological evidence role Huntington&#8217;s disease. Proc Natl Acad Sci <br />     Additional file 10: Table S9. Modules (n = 105) generated network-               USA 2003, 100(5):2712-2717. <br />     boosted analysis aggravation 128Q-neuron dysfunction RNAi.          9.    Hodges A, Strand AD, Aragaki AK, Kuhn A, Sengstag T, Hughes G, <br />                                                                                         Elliston LA, Hartog C, Goldstein DR, Thu D, et al: Regional cellular <br />                                                                                         gene expression changes human Huntington&#8217;s disease brain. Hum Mol <br />                                                                                         Genet 2006, 15(6):965-977. <br />                                                                                   10.   Kuhn A, Goldstein DR, Hodges A, Strand AD, Sengstag T, Kooperberg C, <br /> Acknowledgements                                                                        Becanovic K, Pouladi MA, Sathasivam K, Cha JH, et al: Mutant huntingtin&#8217;s <br /> We grateful Cheryl Wellington Michael Hayden (University                  effects striatal gene expression mice recapitulate changes <br /> British Columbia, Canada) providing constructs encoding htt 128                observed human Huntington&#8217;s disease brain differ with <br /> Glns Jean Philippe Vert help bioinformatics. We thank Hazare               mutant huntingtin length wild-type huntingtin dosage. Hum Mol <br /> Nouari, David Bragado, Salima Abderrahmane, Sabine Cleofax, Sidonie                 Genet 2007, 16(15):1845-1861. <br /> Vivequin technical assistance, Jim Wang George Yohrling (CHDI         11.   Smith R, Brundin P, Li JY: Synaptic dysfunction Huntington&#8217;s disease: a <br /> Foundation) lists genes involved autophagy, mitochondrial activity,           new perspective. Cell Mol Life Sci 2005, 62(17):1901-1912. <br /> and synaptic activity. J.A.P. supported Young Researcher Award           12.   Chalfie M: Neurosensory mechanotransduction. Nat Rev Mol Cell Biol 2009, <br /> from Inserm. R.V. supported Poste Vert fellowship Inserm. This             10(1):44-52. <br /> work supported Inserm, Agence Nationale la Recherche                13.   Parker JA, Connolly JB, Wellington C, Hayden M, Dausset J, Neri C: <br /> (ANR), Fondation pour la Recherche M&#233;dicale (FRM), Paris, France,               Expanded polyglutamines <span id='am-38' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-39' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-41'  ></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Caenorhabditis elegans</span> cause axonal <br /> Hereditary Disease Foundation (USA) European Huntington Disease                 abnormalities severe dysfunction PLM mechanosensory neurons <br /> Network (Euro-HD, Germany).                                                             cell death. Proc Natl Acad Sci USA 2001, 98(23):13318-13323. <br />                                                                                   14.   Parker JA, Arango M, Abderrahmane S, Lambert E, Tourette C, Catoire H, <br /> Author details                                                                          Neri C: Resveratrol rescues mutant polyglutamine cytotoxicity in <br /> 1 <br />  INSERM, Unit 894, Laboratory Neuronal Cell Biology Pathology,                   nematode mammalian neurons. Nat Genet 2005, 37(4):349-350. <br /> 75014 Paris, France. 2University Paris Descartes, EA 4059, 75014 Paris,        15.   Du H, Chalfie M: Genes regulating touch cell development in <br /> France. 3Universit&#233; Montreal, CRCHUM Centre d&#8217;excellence en                          <span id='am-45' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-46' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-48'  ></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Caenorhabditis</span> elegans. Genetics 2001, 158(1):197-207. <br /> neuromique, H&#244;pital Notre-Dame, Montreal, QC,H2W 1T8 Canada. 4Mines               16.   Brenner S: The genetics Caenorhabditis elegans. Genetics 1974, <br /> ParisTech, CBIO, Fontainebleau, 75006 Paris, France. 5Curie Institute, Research         77(1):71-94. <br /> Center, 75005 Paris, France. 6INSERM, Unit 900, Paris, 75005 France. 7Buck        17.   Parker JA, Connolly JB, Wellington C, Hayden M, Dausset J, Neri C: <br /> Institute, Novato, CA 94945, USA.                                                       Expanded polyglutamines Caenorhabditis elegans cause axonal <br />                                                                                         abnormalities severe dysfunction PLM mechanosensory neurons <br /> Authors&#8217; contributions                                                                  cell death. Proc Natl Acad Sci USA 2001, 98(23):13318-13323. <br /> FXL, LM FP designed performed data analysis developed                 18.   Parker JA, Metzler M, Georgiou J, Mage M, Roder JC, Rose AM, Hayden MR, <br /> database tools. CB contributed database developments. JPV contributed                Neri C: Huntingtin-interacting protein 1 influences worm mouse <br /> essential tools data analysis. RVM, JAP CT designed performed C.            presynaptic function protects Caenorhabditis elegans neurons <br /> elegans experiments. CN conceived supervised experiments. FXL CN                mutant polyglutamine toxicity. J Neurosci 2007, <br /> wrote manuscript. All authors read approved final manuscript.               27(41):11056-11064. <br />                                                                                   19.   Pavelec DM, Lachowiec J, Duchaine TF, Smith HE, Kennedy S: Requirement <br /> Competing interests                                                                     ERI/DICER complex endogenous RNA interference sperm <br /> The authors declare competing interests.                              development Caenorhabditis elegans. Genetics 2009, 183(4):1283-1295. <br />                                                                                   20.   Reboul J, Vaglio P, Rual JF, Lamesch P, Martinez M, Armstrong CM, Li S, <br /> Received: 20 October 2011 Accepted: 13 March 2012                                       Jacotot L, Bertin N, Janky R, et al: C. elegans ORFeome version 1.1: <br /> Published: 13 March 2012                                                                experimental verification genome annotation resource for <br />                                                                                         proteome-scale protein expression. Nat Genet 2003, 34(1):35-41. <br /> References                                                                        21.   Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, Davis AP, <br /> 1. Gil JM, Rego AC: Mechanisms neurodegeneration Huntington&#8217;s                     Dolinski K, Dwight SS, Eppig JT, et al: Gene ontology: tool the <br />     disease. Eur J Neurosci 2008, 27(11):2803-2820.                                     unification biology. The Gene Ontology Consortium. Nat Genet 2000, <br /> 2. Graham RK, Deng Y, Slow EJ, Haigh B, Bissada N, Lu G, Pearson J,                     25(1):25-29. <br />     Shehadeh J, Bertram L, Murphy Z, et al: Cleavage caspase-6 site     22.   Bauer S, Grossmann S, Vingron M, Robinson PN: Ontologizer 2.0&#8211;a <br />     required neuronal dysfunction degeneration mutant                    multifunctional tool GO term enrichment analysis data <br />     huntingtin. Cell 2006, 125(6):1179-1191.                                            exploration. Bioinformatics 2008, 24(14):1650-1651. <br /> 3. Zuccato C, Valenza M, Cattaneo E: Molecular mechanisms potential           23.   Mootha VK, Lindgren CM, Eriksson KF, Subramanian A, Sihag S, Lehar J, <br />     therapeutical targets Huntington&#8217;s disease. Physiol Rev 2010,                    Puigserver P, Carlsson E, Ridderstrale M, Laurila E, et al: PGC-1alpha- <br />     90(3):905-981.                                                                      responsive genes involved oxidative phosphorylation coordinately <br /> 4. Sun B, Fan W, Balciunas A, Cooper JK, Bitan G, Steavenson S, Denis PE,               downregulated human diabetes. Nat Genet 2003, 34(3):267-273. <br />     Young Y, Adler B, Daugherty L, et al: Polyglutamine repeat length-            24.   Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, Gillette MA, <br />     dependent proteolysis huntingtin. Neurobiol Dis 2002, 11(1):111-122.             Paulovich A, Pomeroy SL, Golub TR, Lander ES, et al: Gene set enrichment <br /> 5. Holbert S, Denghien I, Kiechle T, Rosenblatt A, Wellington C, Hayden MR,             analysis: knowledge-based approach interpreting genome-wide <br />     Margolis RL, Ross CA, Dausset J, Ferrante RJ, et al: The Gln-<span id='am-87' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-88' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="Ala" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-91'  ></span>Ala</span> repeat             expression profiles. Proc Natl Acad Sci USA 2005, 102(43):15545-15550. <br />     transcriptional activator CA150 interacts huntingtin: <br />  Lejeune et al. BMC Genomics 2012, 13:91                                                                                                                  Page 14 14 <br /> http://www.biomedcentral.com/1471-2164/13/91 <br />  <br />  <br />  <br />  <br /> 25. Kanehisa M, Goto S: KEGG: kyoto encyclopedia genes genomes.                     monooxygenase inhibition blood ameliorates neurodegeneration. Cell <br />     Nucleic Acids Res 2000, 28(1):27-30.                                                   2011, 145(6):863-874. <br /> 26. Lee I, Lehner B, Crombie C, Wong W, Fraser AG, Marcotte EM: A single             48.   Lin CH, Tallaksen-Greene S, Chien WM, Cearley JA, Jackson WS, Crouse AB, <br />     gene network accurately predicts phenotypic effects gene                            Ren S, Li XJ, Albin RL, Detloff PJ: Neurological abnormalities knock-in <br />     perturbation Caenorhabditis elegans. Nat Genet 2008, 40(2):181-188.                 mouse model Huntington&#8217;s disease. Hum Mol Genet 2001, <br /> 27. Li S, Armstrong CM, Bertin N, Ge H, Milstein S, Boxem M, Vidalain PO,                  10(2):137-144. <br />     Han JD, Chesneau A, Hao T, et al: A map interactome network            49.   Raymond LA, Andre VM, Cepeda C, Gladding CM, Milnerwood AJ, <br />     metazoan C. elegans. Science 2004, 303(5657):540-543.                              Levine MS: Pathophysiology Huntington&#8217;s disease: time-dependent <br /> 28. Chen N, Harris TW, Antoshechkin I, Bastiani C, Bieri T, Blasiar D, Bradnam K,          alterations synaptic receptor function. Neuroscience 2011, <br />     Canaran P, Chan J, Chen CK, et al: WormBase: comprehensive data                      198:252-273. <br />     resource Caenorhabditis biology genomics. Nucleic Acids Res              50.   Yamashita T, Wu YP, Sandhoff R, Werth N, Mizukami H, Ellis JM, Dupree JL, <br />     2005, , 33 Database issue: D383-D389.                                                  Geyer R, Sandhoff K, Proia RL: Interruption ganglioside synthesis <br /> 29. Rual JF, Venkatesan K, Hao T, Hirozane-Kishikawa T, Dricot A, Li N, Berriz GF,         produces central nervous degeneration altered axon-glial <br />     Gibbons FD, Dreze M, Ayivi-Guedehoussou N, et al: Towards proteome-                  interactions. Proc Natl Acad Sci USA 2005, 102(8):2725-2730. <br />     scale map human protein-protein interaction network. Nature               51.   Hageman J, van Waarde MA, Zylicz A, Walerych D, Kampinga HH: The <br />     2005, 437(7062):1173-1178.                                                             diverse members mammalian HSP70 machine distinct <br /> 30. Giot L, Bader JS, Brouwer C, Chaudhuri A, Kuang B, Li Y, Hao YL, Ooi CE,               chaperone-like activities. Biochem J 2011, 435(1):127-142. <br />     Godwin B, Vitols E, et al: A protein interaction map Drosophila               52.   Ables JL, Breunig JJ, Eisch AJ, Rakic P: Not(ch) just development: Notch <br />     <span id='am-97' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-98' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-101'  ></span>melanogaster</span>. Science 2003, 302(5651):1727-1736.                                       signalling adult brain. Nat Rev Neurosci 2011, 12(5):269-283. <br /> 31. Lee I, Li Z, Marcotte EM: An improved, <span id='am-1' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="rdfs:label" content="Bias" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>bias</span>-reduced probabilistic                53.   Perrin V, Dufour N, Raoul C, Hassig R, Brouillet E, Aebischer P, Luthi-Carter R, <br />     functional gene network baker&#8217;s yeast, <span id='am-11' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21'  ></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>.                    Deglon N: Implication JNK pathway rat model of <br />     PLoS One 2007, 2(10):e988.                                                             Huntington&#8217;s disease. Exp Neurol 2009, 215(1):191-200. <br /> 32. Rapaport F, Zinovyev A, Dutreix M, Barillot E, Vert JP: Classification of <br />     microarray data using gene networks. BMC Bioinforma 2007, 8:35.                    doi:10.1186/1471-2164-13-91 <br /> 33. Russ AP, Lampel S: The druggable genome: update. Drug Discov Today              Cite article as: Lejeune et al.: Large-scale functional RNAi screen in <br />     2005, 10(23-24):1607-1610.                                                         C. elegans identifies genes regulate dysfuncti <br /> </body></html>