<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body><span id='am-14' about='obo:NCBITaxon_5690' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_2759 obo:NCBITaxon_5653'><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Trypanosoma" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-18'  ></span>Trypanosoma</span> brucei Tb927.2.6100 Is Essential Protein Associated <br /> with Kinetoplast DNA <br /> Kirsten Beck,a Nathalie Acestor,a Anjelique Schulfer,a Atashi Anupama,a Jason Carnes,a Aswini K. Panigrahi,b Ken Stuarta <br /> a <br /> Seattle Biomedical Research Institute, Seattle, Washington, USAa; King Abdullah University Science Technology, Thuwal, Saudi Arabiab <br />  <br /> The mitochondrial DNA trypanosomatid protozoa consists complex, intercatenated network tens maxicircles and <br /> thousands minicircles. <span id='am-40' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44'  ></span>This</span> structure, called kinetoplast DNA (kDNA), requires numerous proteins multiprotein com- <br /> plexes replication, segregation, transcription. In study, used proteomic approach identify proteins are <br /> associated kDNA network. We identified novel protein encoded Tb927.2.6100 present fraction en- <br /> riched kDNA colocalized protein kDNA fluorescence microscopy. RNA interference (RNAi) knockdown its <br /> expression resulted growth defect changes proportion kinetoplasts nuclei cell population. RNAi also <br /> resulted shrinkage loss kinetoplasts, loss maxicircle minicircle components kDNA similar rates, and <br /> (perhaps secondarily) loss edited pre-edited mRNA. These results indicate Tb927.2.6100 protein essential for <br /> the maintenance kDNA. <br />  <br />  <br />  <br /> T    mitochondrial DNA kinetoplastid protozoa consists of <br />      tens maxicircles thousands minicircles, which <br /> are intercatenated single network resides the <br />                                                                                      lication, repair, segregation, processing sub- <br />                                                                                      organellar localization. <br />                                                                                          In study, proteins kDNA enriched fractions iden- <br /> cell&#8217;s large single mitochondrion flagellar basal                    tified mass spectrometry (MS), localization candidate <br /> body (1&#8211;3). Early cytologists named highly visible region                     proteins determined immunofluorescence. One novel pro- <br /> cells stained DNA kinetoplast virtue association                   tein demonstrated colocalize kDNA vivo, its <br /> with base motile flagellum. Subsequently, mito-                      functional role assessed using RNA interference (RNAi). This <br /> chondrial DNA came called kinetoplast DNA (kDNA) (4).                          protein, Tb927.2.6100, associated kDNA is <br />     The kDNA maxicircles encode mitochondrial rRNAs                          essential maintenance. <br /> 17 proteins, RNAs undergo posttranscriptional se- <br /> quence remodeling called RNA editing (5&#8211;7). The minicircles en-                      MATERIALS AND METHODS <br /> code small guide RNAs (gRNAs) specify edited sequences. <br />                                                                                      Trypanosome cell growth. Procyclic-form (PF) T. brucei 164 clone IsTaR <br /> Maxicircle minicircle sizes vary different kinetoplas-                     1.7a grown densities 1&#11003; 107 2 &#11003; 107 cells/ml vitro 27&#176;C <br /> tids, <span id='am-55' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-56' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> maxicircle sequences <span id='am-45' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47'  ></span><span id='am-48' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>homo</span>-                   SDM-79 medium containing hemin (7.5 mg/ml) 10% (vol/vol) <br /> geneous, minicircle sequences heterogeneous (8,                      fetal bovine serum (FBS). PF T. brucei strain 29.13 (20), contains <br /> 9). The specific mRNAs undergo editing extent                        integrated genes T7 polymerase tetracycline repressor, was <br /> their editing vary different kinetoplastids,                    grown presence G418 (15 &#65533;?&#65533;g/ml) hygromycin (25 &#65533;?&#65533;g/ml). <br /> reflection minicircle sequence diversity (10).                                The cells harvested centrifugation 6,000 &#11003; g 10 min 4&#176;C. <br />     Transcription maxicircles minicircles polycis-                        Plasmid constructs, transfections, induction. To create con- <br /> tronic, primary transcripts processed cleavage (11,                   structs inducible expression c-myc epitope-tagged proteins T. <br /> 12). In addition, transcripts undergo editing                      brucei, open reading frames (ORFs) PCR amplified <br /> prior cleavage step (13). Both maxicircles minicircles                    genomic DNA T. brucei strain Lister 427. A detailed list all <br />                                                                                      tagged proteins primers used amplify selected ORFs provided <br /> appear transcribed single mitochondrial RNA polymer- <br />                                                                                      supplemental material. The PCR products cloned into <br /> ase, characteristics viral polymerase (14, 15). It                pGEM-T Easy vector (Promega), digested BamHI BglII and <br /> is unclear RNA polymerases involved mito-                      HindIII enzymes, ligated pLEW79-MHTAP vector (21, 22). <br /> chondrial transcription.                                                             The <span id='am-100' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-101' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span> linearized NotI enzyme transfected the <br />     Although classes DNA molecules,                   PF T. brucei 29.13 cell line; phleomycin-resistant cell lines selected <br /> different mitochondrial DNA polymerases, specific functions                      checked tetracycline (Tet)-regulated expression. The transgenic <br /> of completely elucidated (16&#8211;18). Replication <br /> of kDNA network studied detail, especially the <br /> Englund laboratory (19). These studies revealed somewhat differ-                        Received 20 December 2012 Accepted 2 May 2012 <br /> ent mechanisms different kinetoplastids. In Trypanosoma bru-                         Published ahead print 6 May 2013 <br /> cei, multiple proteins functionally associate kDNA                         Address correspondence Ken Stuart, ken.stuart@seattlebiomed.org. <br /> ensure proper positioning replication, repair                       K.B. N.A. contributed equally article. <br /> and segregation cell division transcription.                      Supplemental material article http://dx.doi.org/10.1128 <br /> The processing primary transcripts coordinated                       /EC.00352-12. <br /> with transcription, implied editing preprocessed tran-                     Copyright &#169; 2013, American Society Microbiology. All Rights Reserved. <br /> scripts. Hence, likely set proteins functionally                     doi:10.1128/EC.00352-12 <br /> interact kDNA, directly indirectly, accomplish rep- <br />  <br />  <br /> 970   ec.asm.org                                            Eukaryotic Cell   p. 970 &#8211;978                                                    July 2013 Volume 12 Number 7 <br />                                                                                                                                A Novel kDNA-Associated Protein <br />  <br />  <br />  <br /> PF cell lines expressing tandem affinity purification (TAP)-tagged pro-         coupled Texas Red-X-conjugated secondary antibody. For 4=,6-di- <br /> tein supplemented 2.5 &#65533;?&#65533;g/ml phleomycin.                              amidino-2-phenylindole dihydrochloride (DAPI) staining, &#11011;5 &#11003; 105 <br />     To create construct RNAi Tb927.2.6100 transcript,            noninduced RNAi-induced cells pelleted centrifugation, <br /> fragment 439 bp PCR amplified genomic DNA T. brucei                washed, fixed 4% formaldehyde, treated 50 &#65533;?&#65533;l 1-&#65533;?&#65533;g/ml <br /> strain Lister 427 using sense primer 5=-ATACTCGAGGAGGCTCTA                    DAPI visualize DNA. Phase-contrast images cells fluo- <br /> GCAGCAGAGGA-3= antisense primer 5=-ATAAAGCTTGAGTAAC                           rescence captured Nikon fluorescence microscope equipped <br /> TGGGGCTGCTACG-3=. The resulting PCR product cloned                       <span id='am-50' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-51' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54'  ></span>camera</span> appropriate filters. <br /> pZJM plasmid (23, 24) XhoI HindIII restriction sites. RNAi cell               RNA isolation quantitative real-time PCR. Total RNA har- <br /> lines generated transfection 10 &#65533;?&#65533;g NotI-linearized pZJM           vested cell lines using TRIzol reagent according man- <br /> construct described previously (25). Three independent cell lines         ufacturer&#8217;s instructions. Ten micrograms RNA treated DNase <br /> selected, growth curves generated absence presence          I using DNA-free kit (Ambion) used templates reverse <br /> 1 &#65533;?&#65533;g/ml tetracycline, induces expression double-stranded              transcription-PCR (RT-PCR) analysis. Real-time RT-PCR measure the <br /> RNA (dsRNA). The induced noninduced cultures maintained               relative abundance mRNAs performed previously described <br /> between 2 &#11003; 106 2 &#11003; 107 cells/ml maintain log-phase growth,            (35). The primer pairs used amplify 18S rRNA &#65533;?&#65533;-tubulin cDNAs, <br /> cell density monitored daily using particle counter (Beckman).              used reference genes, A6 (pre-edited edited), cy- <br />     kDNA isolation. Mitochondrial (mt) vesicles isolated hypo-            tochrome oxidase I (COI), 9S/ND8, RPS12/ND5 described <br /> tonic lysis enriched density gradient flotation 20 35% linear        previously (35, 36). The sequences primers maxicircles fol- <br /> Percoll gradients described previously (26). The enriched vesicles        lows: forward, GGTTCATAGAGGAGAATGGTTCAAG, reverse, CC <br /> solubilized adding n-dodecyl-&#65533;?&#65533;-D-maltoside (DDM) corresponding              TCCACTTACACAATCTTTTGCTT. Those minicircles follows: <br /> a DDM/protein ratio 2 (g/g) incubated ice 30 min. Following         forward, GGCGTGCAAAAATACACATACAC, reverse, GTACCTC <br /> centrifugation 12,000 &#11003; g 20 min 4&#176;C, pellet washed thrice      GGACCTCAAAAATGC. The average cycle threshold (CT) values <br /> with buffer containing 0.5 M sucrose, 20 mM Tris-HCl (pH 7.6), 0.1 mM             target used calculations. The relative standard devia- <br /> EDTA, 0.8 mM spermidine, 1 &#65533;?&#65533;g/ml pepstatin, 2 &#65533;?&#65533;g/ml leupeptin, 1            tion measured CT values quantitative PCR (qPCR) assays be- <br /> mM Pefabloc, 0.5% Nonidet P-40 (NP-40). The centrifugation               tween 0.03% 1.77%. Analysis carried using Pfaffl method, <br /> was 15,000 &#11003; g 20 min 4&#176;C, pellet used kDNA          PCR efficiencies calculated linear regression LinRegPCR <br /> fraction.                                                                         software (37, 38). <span id='am-95' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-96' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-98'  ></span><span id='am-99' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Data</span> normalized 18S rRNA &#65533;?&#65533;-tubulin, and <br />     Tandem affinity purification. Expression TAP-tagged Tb927.2.6100           relative changes mRNA abundance RNAi induction ex- <br /> was induced tetracycline (100 ng/ml culture) 48 h, 500 ml      pressed fold changes relative noninduced control cells. <br /> cells harvested centrifugation density &#11011;9 &#11003; 106 cells/ml. Tagged       For quantification minicircles maxicircles, total DNA <br /> complex purified cell lysate prepared two-step lysis Nonidet     isolated cells collected RNAi, dyskinetoplastic <br />                                                                                   EATRO164 blood form (DK164) cells (which lack kDNA) used a <br /> P-40 detergent described previously (27, 28). Briefly, cells lysed <br />                                                                                   control (39, 40). The cells harvested washed <span id='am-105' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-106' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-108'  ></span><span id='am-109' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>phos</span>- <br /> with 0.25% Nonidet P-40 cleared low-speed centrifugation, the <br />                                                                                   phate-buffered saline (PBS) lysed PBS containing 0.5% SDS <br /> supernatant lysed 1.5% Nonidet P-40 cleared high- <br />                                                                                   0.2 mg/ml proteinase K 4 h 56&#176;C. The DNA treated with <br /> speed centrifugation. The tagged complexes isolated sequential IgG <br />                                                                                   0.1 mg/ml RNase 37&#176;C 30 min extracted phenol- <br /> affinity calmodulin affinity columns (29). <br />                                                                                   chloroform-isoamyl alcohol (25:24:1) precipitated ethanol. The <br />     Protein identification LC-MS-MS. Proteins analyzed liq- <br />                                                                                   pellets resuspended water stored &#11002;20&#176;C were <br /> uid chromatography-tandem mass spectrometry (LC-MS-MS) de- <br />                                                                                   analyzed. For control mtDNA isolated wild-type <br /> scribed previously (30). Briefly, proteins enriched kDNA frac- <br />                                                                                   (WT) PF 29.13 cells. <br /> tion separated 10% SDS-PAGE gels visualized SYPRO <br />                                                                                       The quantitative PCR minicircles maxicircles performed <br /> ruby staining. The entire gel lane divided 9 pieces, pro- <br />                                                                                   essentially described cDNAs, differences the <br /> teins digested in-gel sequencing-grade modified trypsin; the <br />                                                                                   use EvaGreen (Bio-Rad) PCR mix experimental deter- <br /> resulting peptides extracted 50% acetonitrile&#8211;5% formic acid            mination optimal concentration DNA preparation; 7.5 pg <br /> and dried SpeedVac. For proteins isolated TAP-tagged                    DNA used samples Fig. 3B C. As reference genes, the <br /> Tb927.2.6100, 200 &#65533;?&#65533;l calmodulin eluate digested solution,           &#65533;?&#65533;-tubulin gene 18S rRNA gene tested. The reference <br /> peptides extracted described above. The peptides fraction-           genes gave similar results, thus, &#65533;?&#65533;-tubulin gene used <br /> ated nanoflow C18 liquid chromatography analyzed using LTQ               reference gene. The kDNA content quantified relative gene <br /> linear ion trap mass spectrometer. The collision-induced dissociation             using PCR efficiency Pfaffl method described above. <br /> spectra compared Tb927, version 4.0, T. brucei protein <br /> database downloaded GeneDB using TurboSEQUEST software, and <br /> protein matches determined using PeptideProphet Pro-                     RESULTS <br /> teinProphet software (31, 32).                                                    Protein identification. The kDNA isolated Percoll gra- <br />     Sequence analysis identified proteins. The probable functions           dient-enriched mitochondria, proteins fraction <br /> the proteins assigned based GeneDB annotation, proteins           identified mass spectrometry. A total 2,573 unique <br /> with unknown function, possible motifs and/or domains searched            peptides identified LC-MS-MS analyses. Five hundred <br /> InterPro (http://www.ebi.ac.uk/Tools/InterProScan/), Pfam (http://pfam            twenty-three proteins matched pep- <br /> .sanger.ac.uk/), NCBI CDD (http://www.ncbi.nlm.nih.gov/Structure              tide hits, 338 proteins identified single peptide hit, <br /> /cdd/wrpsb.cgi) databases. Alignment Tb927.2.6100 orthologs per-           175 (52%) protein identification probability of <br /> formed using MUSCLE (version 3.8.31) (33). Mitochondrial targeting <br />                                                                                   &#11350;0.99, 109 (32%) 0.98, rest 0.91 and <br /> signal Tb927.2.6100 predicted using MitoProt II (version 1.101) <br /> (34). <br />                                                                                   0.97; 510 (98%) proteins identified pep- <br />     Immunofluorescence microscopy. Subcellular localizations               tide matches protein identification probability &#11350;0.99, 7 <br /> expressed tagged proteins cell determined immuno-              0.98, rest 0.93 0.97. The identified <br /> fluorescence analysis (IFA) using anti-c-myc antibody (Invitrogen) de-         peptide sequences associated probability values presented <br /> scribed previously (27). Colocalization analysis performed using              Table S1 supplemental material. Of 861 proteins <br /> monoclonal antibody (MAb) 78 (anti-mt heat shock protein 70) (27)                 identified study, 44 previously detected our <br />  <br />  <br /> July 2013 Volume 12 Number 7                                                                                                                 ec.asm.org 971 <br />  Beck et al. <br />  <br />  <br />  <br />                                                                                         43), DNA topoisomerase II (Tb09.160.4090) (44), kDNA-associated <br />                                                                                         proteins (Tb10.6k15.1460 Tb927.8.7260), components of <br />                                                                                         HslVU protease complex (Tb11.01.2000, Tb11.01.4050, and <br />                                                                                         Tb927.5.1520) (45). <br />                                                                                             Subcellular location proteins. In <span id='am-58' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-59' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> determine <br />                                                                                         identified proteins colocalize kDNA <br /> FIG 1 Tb927.2.6100 associates kDNA. The tagged protein visu-               T. brucei PF cells, protein candidates selected based <br /> alized fluorescence microscopy using polyclonal anti-c myc antiserum cou- <br /> pled fluorescein isothiocyanate (FITC)-conjugated secondary antibody.              following criteria: (i) identification tryptic <br /> Panels phase-contrast light microscopy PF T. brucei cells, anti-c-myc           peptides, (ii) absence known function (&#8220;hypothetical con- <br /> antibody coupled FITC-conjugated secondary antibody showing local-                 served&#65533;? proteins T. brucei genome database), (iii) pI of <br /> ization target protein, DAPI staining nucleus kDNA, merge.            &#11022;9.5, (iv) domains motifs, (v) mt localization sequence <br />                                                                                         predicted available software MitoProt (http://ihg.gsf.de/ihg <br />                                                                                         /mitoprot.html) and/or SignalP (http://www.cbs.dtu.dk/services <br /> prior proteomic analyses T. brucei mitochondrion (30, 41, <br />                                                                                         /SignalP/). These criteria led set 64 hypothetical proteins, of <br /> 42) (see Table S2 supplemental material). <br />                                                                                         34 assigned mt protein complex <br />     Not 861 different proteins identified kDNA <br />                                                                                         and/or mt membranes (30, 36, 41, 46). Of remaining 30 pro- <br /> fraction mitochondrial, finding reflects fact <br /> that large, structurally complex mitochondrion disrupted                         teins, 18 pI &#11022;9.5 TAP tagged, subcellular <br /> and reseals vesicles isolation, contributing cross con-                    localization assessed immunofluorescence analysis (IFA) <br /> tamination. Based available GeneDB annotation, keyword                               induction expression PF T. brucei cells (Fig. 1; also <br /> searches, literature searches (although genes                           Fig. S1 supplemental material). However, been <br /> misannotated), 424 proteins (&#11011;49%) mitochondrial 105                            shown protein p166, acidic pI 5.32, <br /> proteins (&#11011;12%) likely contaminants enriched kDNA                            localizes kDNA (47). Therefore, second round of <br /> fraction, assignable non-mt locations, including                      selection, protein candidates selected pI of <br /> glycosomes, endoplasmic reticulum, nucleus, Golgi, lyso-                        &#65533;?9.5 (see Fig. S1). <br /> somes (see Table S3 supplemental material). The remaining                            Immunofluorescence assay using anti-myc MAb (anti-tag) <br /> 332 proteins (&#11011;39%) annotated localization;                            showed 24 tagged successfully expressed pro- <br /> likely mitochondrial proteins location dem-                    teins (Table 1), Tb927.2.6100 exclusively co- <br /> onstrated, nonmitochondrial contaminants,                       localized kDNA (Fig. 1). As remaining proteins, <br /> been observed previous analyses (30). As expected, proteins                  IFA showed 15 evenly distributed throughout <br /> known associated kDNA identified sample,                        reticulated mitochondrion did intensely <span id='am-75' about='rdfs:label' typeof='owl:Thing'>label</span> kin- <br /> including DNA polymerases (Tb927.5.2780 Tb927.5.2790) (17,                          etoplast area, 3 localized nucleus, 5 appeared <br />  <br />  <br /> TABLE 1 Subcellular localization tagged proteins <br />                                                                                                                              Previous mt <br />                                Theoretical pI/molecular                                                                      assignment        New assignment <br /> Protein <span id='am-1' about='obo:NCBITaxon_389777' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_105691 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_6894 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Namea" datatype="xsd:string"></span>namea</span>                  mass (kDa)                         Motif domain                                            confidenceb       (IFA based) <br /> Tb927.2.6100TAP233             10.77/53.28                                                                                   Low               mt-kDNA <br /> Tb927.10.2970TAP245            7.74/13.79                                                                                    Low               mt <br /> Tb927.5.4360TAP246             6.95/36.43                         S-Adenosylmethionine-dependent methyltransferase           Low               mt <br /> Tb927.4.3080TAP247             8.58/18.44                         DUF167                                                     Low               mt <br /> Tb09.211.0530TAP234            10.07/30.41                                                                                   Low               mt <br /> Tb927.3.1810TAP235             9.75/36.1                                                                                     Low               mt <br /> Tb927.10.1860 TAP273           9.61/39.1                                                                                     Low               Nuclear <br /> Tb11.01.3970TAP231             10.24/23.49                                                                                   Moderate          mt <br /> Tb927.5.740 TAP269             10.11/23.91                                                                                   Moderate          mt <br /> Tb11.01.8500 TAP270            9.91/25.87                                                                                    Moderate          mt <br /> Tb11.02.1860TAP268             5.52/51.88                         Putative transcriptional regulator                         Moderate          mt <br /> Tb11.01.2740TAP267             5.99/71.77                         AlkB, alkylated DNA repair protein                         Moderate          mt <br /> Tb09.160.2380TAP241            9.89/9.23                          Uncharacterized protein <span id='am-6' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-7' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> (UPF0041)                   Moderate          mt <br /> Tb927.10.1660 TAP271           9.73/33.2                                                                                     Moderate          mt <br /> Tb927.5.1130 TAP272            9.59/34.3                                                                                     Moderate          Non-mt <br /> Tb927.3.1720TAP248             5.67/11.17                                                                                    Moderate          Non-mt <br /> Tb927.7.630 TAP274             10.73/28.19                                                                                   None              mt <br /> Tb927.4.1370TAP237             9.66/17.64                                                                                    None              mt <br /> Tb11.02.5230 TAP275            10.25/23.28                                                                                   None              Nuclear <br /> Tb09.211.1150 TAP277           10.38/20.69                                                                                   None              Nuclear <br /> Tb09.211.2620TAP238            10.31/22.75                                                                                   None              Non-mt <br /> Tb927.8.2490TAP250             8.52/35.23                         SET domain                                                 Not detected      Non-mt <br /> a <br />   GeneDB accession number (http://www.genedb.org) superscript indicating TAP-tagged identifier. <br /> b <br />   From reference 30. <br />  <br />  <br />  <br /> 972   ec.asm.org                                                                                                                                  Eukaryotic Cell <br />                                                                                                                                    A Novel kDNA-Associated Protein <br />  <br />  <br />  <br />  <br /> FIG 2 Effects Tb927.2.6100 RNAi. (A) Cell growth curves representative RNAi cell line Tb927.2.6100 grown presence (open circles) absence <br /> (filled squares) Tet. The cumulative cell numbers calculated multiplying cell densities dilution factor. (B) qPCR analysis total RNA isolated <br /> from cells, expression Tb927.2.6100 repressed 4 days RNAi. The relative change Tb927.2.6100 mRNA abundance determined by <br /> using &#65533;?&#65533;-tubulin mRNA (right bar) 18S rRNA (left bar) internal control presented log scale. (C) Effect RNAi kinetoplast size. DAPI <br /> staining shows kinetoplasts cells RNAi 4 days RNAi. n, nucleus; k, kinetoplast. (D) Quantitation different cell cycle stages during <br /> RNAi Tb927.2.6100 showing proportion 200 randomly chosen cells having kinetoplast/one nucleus (1K1N) kinetoplasts/one nucleus <br /> (2K1N) reduced RNAi Tb927.2.6100, proportion cells having kinetoplast/one nucleus (0K/1N) increased dramatically, as <br /> is proportion having reduced-size kinetoplast/one nucleus (1ksmall1N). <br />  <br />  <br /> to nonmitochondrial (see Fig. S1 supplemental mate-                      encoded single-copy gene, predicted product 487 <br /> rial). However, definitive localization assigned                amino acid residues pI 10.77. Bioinformatic analyses of <br /> the proteins indistinct speckled fluorescence; per-               Tb927.2.6100 predict N-terminal mitochondrial targeting sig- <br /> haps expressed proteins folded way prevented                    nal known motif domain. Homologous genes pres- <br /> the tag recognized MAb.                                          ent related Trypanosoma species Leishmania species. <br />     Based IFA results, seven proteins pre-                 A sequence alignment orthologs T. brucei gambiense, T. <br /> viously assigned mitochondrion moderate confidence                     cruzi, T. congolense, T. vivax showed conservation puta- <br /> and low confidence (30) clearly localized mito-                tive mitochondrial import sequence regions near N C <br /> chondrion, protein (Tb927.10.1860) earlier assigned                    termini divergence regions (see Fig. S2 the <br /> to mitochondrion low confidence localized nucleus                  supplemental material). To evaluate Tb927.2.6100 es- <br /> (see Fig. S1 supplemental material).                                        sential T. brucei, expression knocked PF cells by <br />     Tandem affinity purification tagged Tb927.2.6100. To de-                    RNAi. Growth inhibition apparent day 5 lasted <br /> termine proteins associate Tb927.2.6100, tan-                      10 days compared noninduced cell lines (Fig. <br /> dem affinity purification performed cells expressing                      2A). Induction RNAi expression resulted &#11011;80% knock- <br /> Tb927.2.6100TAP, purified proteins trypsin digested                       Tb927.2.6100 mRNA abundance 4 days compared <br /> and analyzed mass spectrometry (see Table S4 sup-                        noninduced cells (Fig. 2B). Growth resumed &#11011;10 <br /> plemental material). Among proteins identified mass                         days, indicating cells increasingly resistant to <br /> spectrometry, proteins previously identified associating                   RNAi knockdown, previously reported T. brucei system <br /> with mitochondrial ribosome found: Tb10.70.7960,                          (50). Tb927.2.6100 protein levels directly assessed in <br /> Tb11.01.1840, Tb11.02.2250, Tb11.02.3180, Tb927.1.2990,                            RNAi cells lack antibody specific for <br /> Tb927.3.5610, Tb927.6.3930, Tb927.7.3430, Tb927.7.3510                         Tb927.2.6100. The growth defects resulting repression of <br /> (42). Among these, Tb11.02.3180 Tb10.70.7960                         Tb927.2.6100 imply essential protein PF T. brucei. <br /> also previously identified MRB1                            Effect Tb927.2.6100 RNAi mitochondrial DNA. RNAi <br /> complex, involved mitochondrial RNA processing                         proteins closely associated kDNA shown to <br /> and thought mediate exchange gRNAs RNA                             cause kDNA loss (16, 47, 51&#8211;53), maintenance kDNA after <br /> editing (36, 48, 49).                                                              Tb927.2.6100 RNAi examined. DAPI staining revealed that <br />     Effect Tb927.2.6100 RNAi cell growth. Tb927.2.6100                    Tb927.2.6100 RNAi caused shrinking disappearance of <br />  <br />  <br /> July 2013 Volume 12 Number 7                                                                                                                       ec.asm.org 973 <br />  Beck et al. <br />  <br />  <br />  <br /> the kDNA network. In Fig. 2C, image shows DAPI-stained           maxicircle gave similar results, confirming the <br /> cells (noninduced) normal nucleus kinetoplast. In con-        measured reduction correct. The qPCR minicircle DNA <br /> trast, image (4 days RNAi induction) shows cells        showed similar pattern: 4 days changes were <br /> that lost majority kinetoplast staining. In RNAi         small, variation higher maxicircle, <br /> cells retain kDNA, DAPI staining showed obvious de-           2.4-fold increase day 4 (Fig. 3B). But days 5 6, the <br /> crease size kinetoplasts, point barely   minicircle DNA decreased 30% levels non- <br /> detectable (Fig. 2C; Fig. S3 supplemental material).     induced cells. This shows decrease kDNA RNAi <br />     The effects cell division kinetoplast segregation        Tb927.2.6100 concurrent loss maxicircles and <br /> also monitored DAPI staining (Fig. 2D). The cells G1 S         minicircles. <br /> phases nucleus kinetoplast (1N1K). Since              As additional control, mitochondrial DNA isolated on <br /> mt division precedes nuclear division, cells G2/M phase          days 2, 4, 6 growth <span id='am-19' about='obo:NCBITaxon_5691' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_39700 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_5690 obo:NCBITaxon_2759 obo:NCBITaxon_5653'><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Trypanosoma (Trypanozoon) brucei" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23'  ></span><span id='am-24' property="rdfs:label" content="Trypanosoma brucei" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Trypanosoma brucei subgroup" datatype="xsd:string"></span>Trypanosoma brucei</span> PF 29.13 cells <br /> one nucleus kinetoplasts (1N2K), cells undergoing            editing exonuclease KREX2 repressed RNAi <br /> cytokinesis nuclei kinetoplasts (2N2K). We              (54). Because KREX2 essential (55), cells suitable <br /> evaluated proportion cells nuclear mitochondrial         control mitochondrial protein lacking, a <br /> DNA 200 cells sample randomly chosen. In            decrease maxicircle minicircle DNA expected. <br /> control Tb927.2.6100 cell line (noninduced cells), &#11011;90%           The qPCR showed change abundance the <br /> cells normal-size single nucleus single       mitochondrial DNA, small increase, than <br /> kinetoplast (1N1K); rest cells (&#11011;10%) 1N2K        2-fold, minicircle 6 days (Fig. 3C). This shows the <br /> or 2N2K. However, RNAi-induced cells, reduction              observed decrease mitochondrial DNA RNAi of <br /> proportion cells G1/S phase (1N1K) proportional in-          Tb927.2.6100 general reaction cell the <br /> crease cells lacking kinetoplast (1N0K) observed. The        depletion mitochondrial protein specific response. <br /> percentage 1N0K cells increased time: 55% day 2, 60%       Southern blot analysis showed minicircles max- <br /> day 4, 75% day 6 (Fig. 2D). Moreover, percentage           icircles diminished course RNAi knockdown in <br /> 1N1K cells harboring smaller kinetoplast remained constant             Tb927.2.6100 (results shown). <br /> over time, &#11011;20%. These results suggest Tb927.2.6100               We decrease mitochondrial DNA accom- <br /> be involved controlling replication and/or segregation         panied reduction abundance pre-edited and <br /> kDNA network.                                                            edited kDNA transcripts. This assessed using qPCR RNA <br />     To assess decrease total kDNA specific        isolated cells RNAi Tb927.2.6100 induced. <br /> reduction minicircle and/or maxicircle component           The transcripts examined pre-edited edited A6, COI that <br /> mitochondrial DNA, quantitative PCR (qPCR) performed                 does edited, regions 9S/ND8 RPS12/ND5 that <br /> using primers binding maxicircle minicircle.        span site cleaved processing polycis- <br /> DNA DK164, does contain mitochondrial DNA,                tronic primary transcript. All transcripts showed slight <br /> was used control, testing 7.5 pg DNA reaction;           reduction 2 days dramatic reduction, 90% <br /> mitochondrial DNA tested 3 dilutions 75, 7.5, 0.75         (apart edited A6, reduction 75%), compared <br /> pg reaction. The analysis threshold cycle shows          levels noninduced cells day 6 (Fig. 4). These results <br /> there amplification using DK164 DNA tem-            expected serve additional control that <br /> plate (CT &#11022; 30) maxicircle primer pairs. This shows         depletion Tb927.2.6100 leads massive decrease mito- <br /> there nonspecific binding primers nuclear DNA,         chondrial DNA subsequently mitochondrial transcripts. <br /> making suitable quantify mitochondrial DNA (Fig.                 Together, results confirmed loss kinetoplast <br /> 3A). The qPCR serial dilutions mitochondrial DNA pro-              inhibition Tb927.2.6100 showed maxicircle and <br /> duced CT values reflecting robust amplification (14 19       minicircle DNA disappear comparable rates same <br /> concentration range), demonstrating specific amplification        time period result RNAi knockdown. Thus, possible <br /> the mitochondrial targets. The linearity CT value plot-      determine protein involved replication and/or <br /> ted DNA concentration (R2 &#11022; 0.9 primer           maintenance other; results point to <br /> pairs) shows primers used quantify mito-        Tb927.2.6100 having general function regulation of <br /> chondrial DNA. To ensure primers amplify         kinetoplast copy numbers. <br /> their intended target, qPCR amplicons ligated the <br /> pGEM-T Easy vector sequenced, confirmed speci-             DISCUSSION <br /> ficity primers.                                                   A multitude proteins required replication, segre- <br />     Quantitative PCR performed using total DNA purified              gation, transcription complex trypanosome mito- <br /> from cells RNAi Tb927.2.6100 noninduced               chondrial DNA, known kDNA, consists intercat- <br /> cells controls time points. The DNA isolated          enated network tens maxicircles thousands of <br /> every day time course 6 days, 6 days corresponds       minicircles (19). We identified novel protein, Tb927.2.6100, <br /> with development phenotype (Fig. 2A); point,       MS analysis proteins associated enriched <br /> growth stalls depletion Tb927.2.6100. The qPCR             kDNA. Although sequence analysis did identify known <br /> shows 4 days levels maxicircle DNA changed      motifs, alignment orthologs trypanosomes re- <br /> only slightly (to maximum increase 1.5-fold),          vealed regions sequence conservation near N C ter- <br /> days 5 6 RNAi maxicircle DNA decreased dramatically,          mini involved protein function. We showed that <br /> to eventually 20% compared noninduced cells (Fig. 3B).            protein localizes exclusively kDNA area mito- <br /> Using different primer pairs targeting different parts        chondrion. Furthermore, demonstrated RNAi its <br />  <br />  <br /> 974   ec.asm.org                                                                                                                  Eukaryotic Cell <br />                                                                                                                                A Novel kDNA-Associated Protein <br />  <br />  <br />  <br />  <br /> FIG 3 Quantitative PCR kDNA. (A) Threshold cycles using different amounts DNA DK164 cells (no mitochondrial DNA) kDNA PF 29.13 <br /> cells. Primers target maxicircle (max1, A6 pre) minicircle (min2) DNA showed significant amplification mitochondrial DNA (DK164), <br /> whereas good amplification obtained using kDNA template. Error bars represent standard deviations measured CT replicates. (B) RNAi of <br /> Tb927.2.6100 results decreases maxicircle minicircle DNAs, maxicircle decreasing larger extent (&#11011;20%) minicircle (&#11011;30%). <br /> (C) Maxicircle minicircle abundance altered KREX2 repression RNAi, control indicates effects Tb927.2.6100 RNAi specific. <br /> &#65533;?&#65533;-Tubulin used internal reference relative comparisons. <br />  <br />  <br /> presence essential kDNA maintenance rep-                     DAPI staining loss kDNA sequence qPCR. However, the <br /> lication. Hence, association kDNA functional                loss kDNA DAPI staining loss maxicircle tran- <br /> as physical. The results RNAi studies indicate                  scripts qPCR occurred day 2, loss kDNA by <br /> the function protein exclusive max-                 qPCR occurred day 5. This difference probably reflects fact <br /> icircle minicircle components, knockdown resulted                   maxicircle DNA abundance measured qPCR, which <br /> the loss molecules similar rates.                                    does require maxicircles remain intact that <br />     Loss kDNA following RNAi observed loss                template DNA present. However, kDNA transcripts rap- <br />  <br />  <br /> July 2013 Volume 12 Number 7                                                                                                                  ec.asm.org 975 <br />  Beck et al. <br />  <br />  <br />  <br />                                                                                  Tb927.2.6100 itself consisted identified peptides, <br />                                                                                  consequence location abundance its <br />                                                                                  lysine arginine residues affected susceptibility <br />                                                                                  protease generation optimally sized peptides mass <br />                                                                                  spectrometry. Coverage individual proteins mitochon- <br />                                                                                  drial ribosome limited, ribosomal proteins <br />                                                                                  observed peptides. However, observation of <br />                                                                                  distinct ribosomal proteins implies association between <br />                                                                                  Tb927.2.6100 mitochondrial ribosomes. Two pro- <br />                                                                                  teins isolations MRB1 com- <br />                                                                                  plex, involved aspect processing kDNA <br />                                                                                  transcripts. Immunofluorescence studies revealed ribo- <br />                                                                                  somal proteins proteins function RNA editing lo- <br /> FIG 4 Quantitative PCR mtRNA. RNAi Tb927.2.6100 results             cated proximal kDNA (57; L. Simpson, personal communica- <br /> strong decrease examined mt transcripts course 6 days. RNAs <br /> examined included pre-edited edited ATPase subunit 6 (A6 pre        tion). Since Tb927.2.6100 associated kDNA essential <br /> A6 ed, respectively), never-edited COI, polycistronic junction regions   maintenance, simple explanation proteins that <br /> of primary transcripts corresponding RPS12 ND5 9S ND8.             function maintenance kDNA, transcription RNA <br />                                                                                  processing translation, generally colocalized region <br />                                                                                  kDNA resides, i.e., kinetoplast. <br /> idly turned disruption transcription, DAPI                          The specific role Tb927.2.6100 uncertain. It does ap- <br /> staining affected kDNA structure supercoiling.                         pear function directly replication specific either <br />     Immunofluorescence analysis 13 additional TAP-tagged pro-                 maxicircle minicircle DNA, molecules lost at <br /> teins enriched kDNA fraction showed              similar rates. Its restriction submitochondrial location <br /> localize mitochondria restricted kDNA re-              kDNA proteins involved kDNA replication, <br /> gion. This study identified proteins localize nu-         transcription, RNA processing translation located and <br /> cleus. It surprising proteins associated         apparent loss kDNA structure transcripts sug- <br /> kDNA, contains tens identical &#11011;20-kb maxicircles hun-               gest role associated structural organization kDNA <br /> dreds heterogeneous &#11011;1-kb minicircles intercatenated            integration macromolecular processes. The absence <br /> single complex DNA network located mitochon-                  protein Leishmania T. cruzi implies role <br /> drion flagellar basal body. These proteins include               assumed protein, histone- <br /> DNA polymerases, DNA helicases, topoisomerases, DNA ligases                  like protein hypothetical protein encoded the <br /> that function replication reassembly network            syntenic position Leishmania. Alternatively, absence may <br /> kDNA, observed kDNA preparation <span id='am-90' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-94'  ></span>iso</span>-                   reflect functional differences trypanosomes Leishma- <br /> lated work. They include p166, links kDNA              <span id='am-82' about='obo:NCBITaxon_155002' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_5338 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_155000 obo:NCBITaxon_2759 obo:NCBITaxon_155619'><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-85'  ></span><span id='am-86' property="rdfs:label" content="Nia" datatype="xsd:string"></span>nia</span> associated expression segregation of <br /> flagellar basal body functions analogously centriole segre-          kDNA (19). <br /> gating kDNA cell division. In addition, protease                 The proteins associated kDNA reflect the <br /> functions regulating level kDNA. Furthermore, pro-             complexity unusual mitochondrial genome, differ- <br /> teins function polygenic transcription kDNA             ences kinetoplastid species reflect evolutionary diver- <br /> subsequent RNA processing, includes transcript cleavage, ad-               sification. Differences kDNA replication segregation that <br /> dition 3= oligo(U) tails gRNAs rRNA, extensive                 evolved species (9, 19, 58) reflect how <br /> editing mRNAs. The apparent functions proteins               genetic information differentially utilized regulated in <br /> consistent localized kDNA.                                 diverse kinetoplastid species, substantial differences their <br />     Our previous proteomics studies identified total 3,043 T.               host target tissues, <span id='am-61' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-62' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>vectors</span>, metabolic (and other) <br /> brucei proteins assigned 979 mitochondrion                   processes. <br /> with progressively diminishing stringent criteria 459 high- <br /> confidence, 218 moderate-confidence, 302 low-confidence                      ACKNOWLEDGMENTS <br /> proteins (30, 41, 42). This study identified 44 additional proteins <br />                                                                                  This work primarily supported National Institutes Health grant <br /> that previously detected MS. Analysis TAP- <br />                                                                                  AI065935 received support grant AI014102. <br /> tagged proteins identified 2 additional mitochondrial proteins,                     We thank Yuko Ogata mass spectrometry analysis. <br /> enhanced confidence mitochondrial location 12 pro- <br /> teins, demonstrated nuclear localization 3 (Table <br />                                                                                  REFERENCES <br /> 1). These results update list 978 proteins assigned <br /> to mitoproteome, 474 high-confidence, 209 moderate-                      1. Riou G, Delain E. 1969. <span id='am-65' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-66' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-69'  ></span>Electron</span> microscopy circular kinetoplastic <br />                                                                                      DNA <span id='am-9' about='obo:NCBITaxon_5693' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_5690 obo:NCBITaxon_2759 obo:NCBITaxon_47570 obo:NCBITaxon_5653'><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-12'  ></span><span id='am-13' property="rdfs:label" content="Trypanosoma cruzi" datatype="xsd:string"></span>Trypanosoma cruzi</span>: occurrence catenated forms. Proc. Natl. <br /> confidence, 295 low-confidence mitochondrial proteins (see                       Acad. Sci. U. S. A. 62:210 &#8211;217. <br /> Table S5 supplemental material). These probably                     2. Kleisen CM, Weislogel PO, Fonk K, Borst P. 1976. The structure of <br /> represent mitochondrial proteins, dual                         kinetoplast DNA. 2. Characterization novel component high com- <br /> localization (i.e., isoleucyl-tRNA synthetase) (56).                                 plexity present kinetoplast DNA network <span id='am-26' about='obo:NCBITaxon_5655' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33682 obo:NCBITaxon_1286322 obo:NCBITaxon_5654 obo:NCBITaxon_2759 obo:NCBITaxon_5653'><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Crithidia" datatype="xsd:string"></span><span id='am-30'  ></span>Crithidia</span> luciliae. Eur. J. <br />                                                                                      Biochem. 64:153&#8211;160. <br />     Analysis proteins associated TAP-tagged Tb927.2.6100                  3. Shapiro TA, Englund PT. 1995. The structure replication kineto- <br /> suggests interact directly indirectly mitochon-                  plast DNA. Annu. Rev. Microbiol. 49:117&#8211;143. <br /> drial ribosomes. The mass spectrometry coverage tagged                         4. Vickerman K. 1976. The diversity kinetoplastid flagellates, p 1&#8211;34. <br />  <br />  <br />  <br /> 976   ec.asm.org                                                                                                                                   Eukaryotic Cell <br />                                                                                                                                   A Novel kDNA-Associated Protein <br />  <br />  <br />  <br />       In Lumsden WHR, Evans DA (ed), Biology <span id='am-31' about='obo:NCBITaxon_5653' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33682 obo:NCBITaxon_2759'><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="kinetoplastids" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Kinetoplastea" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Protomonadida" datatype="xsd:string"></span><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Kinetoplastida" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="kinetoplasts" datatype="xsd:string"></span>kinetoplastida</span>. Academic          28. Gavin AC, Aloy P, Grandi P, Krause R, Boesche M, Marzioch M, Rau <br />       Press, London, United <span id='am-87' about='obo:NCBITaxon_kingdom' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-88' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="kingdom" datatype="xsd:string"></span>Kingdom</span>.                                                  C, Jensen LJ, Bastuck S, Dumpelfeld B, Edelmann A, Heurtier MA, <br />  5.   Stuart KD, Schnaufer A, Ernst NL, Panigrahi AK. 2005. Complex                   Hoffman V, Hoefert C, Klein K, Hudak M, Michon AM, Schelder M, <br />       management: RNA editing trypanosomes. Trends Biochem. Sci. 30:97&#8211;            Schirle M, Remor M, Rudi T, Hooper S, Bauer A, Bouwmeester T, <br />       105.                                                                            Casari G, Drewes G, Neubauer G, Rick JM, Kuster B, Bork P, Russell <br />  6.   Aphasizhev R, Aphasizheva I. 2011. Mitochondrial RNA processing              RB, Superti-Furga G. 2006. Proteome survey reveals modularity the <br />       trypanosomes. Res. Microbiol. 162:655&#8211; 663.                                     yeast cell machinery. Nature 440:631&#8211; 636. <br />  7.   Hajduk S, Ochsenreiter T. 2010. RNA editing kinetoplastids. RNA          29. Rigaut G, Shevchenko A, Rutz B, Wilm M, Mann M, Seraphin B. 1999. <br />       Biol. 7:229 &#8211;236.                                                               A generic protein purification method protein complex characteriza- <br />  8.   Kleisen CM, Borst P, Weijers PJ. 1976. The structure kinetoplast             tion proteome exploration. Nat. Biotechnol. 17:1030 &#8211;1032. <br />       DNA. 1. The mini-circles <span id='am-70' about='obo:NCBITaxon_46909' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33682 obo:NCBITaxon_1286322 obo:NCBITaxon_5654 obo:NCBITaxon_2759 obo:NCBITaxon_5655 obo:NCBITaxon_5653'><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Crithidia luciliae" datatype="xsd:string"></span><span id='am-74'  ></span>Crithidia luciliae</span> heterogeneous base    30. Panigrahi AK, Ogata Y, Zikova A, Anupama A, Dalley RA, Acestor N, <br />       sequence. Eur. J. Biochem. 64:141&#8211;151.                                          Myler PJ, Stuart KD. 2009. A comprehensive analysis Trypanosoma <br />  9.   Lukes J, Guilbride DL, Votypka J, Zikova A, Benne R, Englund PT.                brucei mitochondrial proteome. Proteomics 9:434 &#8211; 450. <br />       2002. Kinetoplast DNA network: evolution improbable structure.        31. Keller A, Nesvizhskii AI, Kolker E, Aebersold R. 2002. Empirical statis- <br />       Eukaryot. Cell 1:495&#8211;502.                                                       tical model estimate accuracy peptide identifications by <br /> 10.   Simpson L, Maslov DA. 1999. Evolution U-insertion/deletion RNA           MS/MS database search. Anal. Chem. 74:5383&#8211;5392. <br />       editing mitochondria kinetoplastid protozoa. Ann. N. Y. Acad. Sci.    32. Nesvizhskii AI, Keller A, Kolker E, Aebersold R. 2003. A statistical model <br />       870:190 &#8211;205.                                                                   identifying proteins tandem mass spectrometry. Anal. Chem. 75: <br /> 11.   Grams J, McManus MT, Hajduk SL. 2000. Processing polycistronic               4646 &#8211; 4658. <br />       guide RNAs associated RNA editing complexes Trypanosoma          33. Edgar RC. 2004. MUSCLE: multiple sequence alignment method with <br />       brucei. EMBO J. 19:5525&#8211;5532.                                                   reduced time space complexity. BMC Bioinformatics 5:113. doi:10 <br /> 12.   Madina BR, Kuppan G, Vashisht <span id='am-76' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-77' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>AA</span>, Liang YH, Downey KM,                          .1186/1471-2105-5-113. <br />       Wohlschlegel JA, Ji X, Sze SH, Sacchettini JC, Read LK, Cruz-Reyes J.       34. Claros MG, Vincens P. 1996. Computational method predict mito- <br />       2011. Guide RNA biogenesis involves novel RNase III family endoribo-          chondrially imported proteins targeting sequences. Eur. J. <br />       nuclease Trypanosoma brucei. RNA 17:1821&#8211;1830.                               Biochem. 241:779 &#8211;786. <br /> 13.   Koslowsky DJ, Yahampath G. 1997. Mitochondrial mRNA 3= cleavage/            35. Carnes J, Trotter JR, Ernst NL, Steinberg AG, Stuart K. 2005. An <br />       polyadenylation RNA editing Trypanosoma brucei independent           essential RNase III insertion editing endonuclease Trypanosoma brucei. <br />       events. Mol. Biochem. Parasitol. 90:81&#8211;94.                                      Proc. Natl. Acad. Sci. U. S. A. 102:16614 &#8211;16619. <br /> 14.   Hashimi H, Cicova Z, Novotna L, Wen YZ, Lukes J. 2009. Kinetoplastid        36. Acestor N, Panigrahi AK, Carnes J, Zikova A, Stuart KD. 2009. The <br />       guide RNA biogenesis dependent subunits mitochondrial              MRB1 complex functions kinetoplastid RNA processing. RNA 15:277&#8211; <br />       RNA binding complex 1 mitochondrial RNA polymerase. RNA 15:                 286. <br />       588 &#8211;599.                                                                   37. Pfaffl MW. 2001. A new mathematical model relative quantification in <br /> 15.   Grams J, Morris JC, Drew ME, Wang Z, Englund PT, Hajduk SL. 2002.               real-time RT-PCR. Nucleic Acids Res. 29:e45. doi:10.1093/nar/29.9.e45. <br />       A trypanosome mitochondrial RNA polymerase required transcrip-       3 <br /> </body></html>