<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A Genome-Wide Screen Yeast Identifies Specific <br /> Oxidative Stress Genes Required Maintenance of <br /> Sub-Cellular Redox Homeostasis <br /> <span id='am-66' about='obo:NCBITaxon_735482' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37571 obo:NCBITaxon_37570 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_688638 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-67'  ></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="Anita" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Anita</span> Ayer1, Sina Fellermeier1, Christopher Fife1, Simone S. Li1, Gertien Smits2, Andreas J. Meyer3, <br /> Ian W. Dawes1*, Gabriel G. Perrone4 <br /> 1 University New South Wales, Sydney, Australia, 2 Swammerdam Institute Life Sciences, University Amsterdam, Amsterdam, The Netherlands, 3 INRES- Chemical <br /> Signalling, University Bonn, Bonn, Germany, 4 University Western Sydney, Penrith, Australia <br />  <br />  <br />  <br />      Abstract <br />      Maintenance optimal redox environment critical appropriate functioning cellular processes cell survival. <br />      Despite importance maintaining redox homeostasis, clear optimal redox potential sensed set, <br />      processes impact redox cellular/organellar level poorly understood. The genetic bases cellular redox <br />      homeostasis investigated using green fluorescent protein (GFP) based redox probe, roGFP2 pH sensitive GFP- <br />      based probe, pHluorin. The use roGFP2, conjunction pHluorin, enabled determination pH-adjusted sub-cellular <br />      redox potential non-invasive real-time manner. A genome-wide screen using non-essential essential <br />      gene collections carried <span id='am-28' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-29' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> using cytosolic-roGFP2 identify factors essential for <br />      maintenance cytosolic redox state steady-state conditions. 102 genes diverse function identified are <br />      required maintenance cytosolic redox state. Mutations genes led shifts half-cell glutathione redox <br />      potential 75-10 mV. Interestingly, specific oxidative stress-response processes identified over-represented <br />      <span id='am-95' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-96'  ></span><span id='am-97' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> set. Further investigation role oxidative stress-responsive systems sub-cellular redox homeostasis was <br />      conducted using roGFP2 constructs targeted mitochondrial matrix peroxisome EGSH measured cells in <br />      exponential stationary phase. Analyses allowed identification key redox systems sub-cellular level and <br />      identification novel genes involved regulation cellular redox homeostasis. <br />  <br />   Citation: Ayer A, Fellermeier S, Fife C, Li SS, Smits G, et al. (2012) A Genome-Wide Screen Yeast Identifies Specific Oxidative Stress Genes Required the <br />   Maintenance Sub-Cellular Redox Homeostasis. PLoS ONE 7(9): e44278. doi:10.1371/journal.pone.0044278 <br />   Editor: Mick F. Tuite, University Kent, United <span id='am-82' about='obo:NCBITaxon_kingdom' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="kingdom" datatype="xsd:string"></span>Kingdom</span> <br />   Received May 31, 2012; Accepted July 31, 2012; Published September 6, 2012 <br />   Copyright: &#223; 2012 Ayer et al. <span id='am-55' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57'  ></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This research funded Australian Postgraduate Award (to <span id='am-76' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>AA</span>) Discovery grant Australian Research Council (to IWD GGP). <br />   Work GJS&#8217;s lab partially funded Unilever work AJM&#8217;s laboratory funded Deutsche Forschungsgemeinschaft (grant ME1567/5-1). The <br />   funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors read journal&#8217;s policy following interests: One authors (GJS) received funding Unilever. <br />   There patents, products development marketed products declare. This does alter authors&#8217; adherence PLoS ONE policies on <br />   sharing data materials. <br />   * E-mail: i.dawes@unsw.edu.au <br />  <br />  <br />  <br /> Introduction                                                                          erative disorders including Alzheimer&#8217;s Parkinson&#8217;s diseases <br />                                                                                       [10] metabolic syndromes diabetes [11]. <br />    The maintenance optimal redox environment critical                         Despite importance redox homeostasis fundamental <br /> the functioning biological processes. The majority                         cell biology disease states, current understanding <br /> intracellular functions require strongly reducing environment,                      limited available assay methods relying whole-cell <br /> which maintains key sulfhydryl groups reduced active                         lysate analysis use altered resistance oxidants a <br /> form key elements proteins enzymes remain                         phenotypic <span id='am-6' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Indicator" datatype="xsd:string"></span><span id='am-10'  ></span>indicator</span> redox imbalance dysfunction. While the <br /> functional. In eukaryotic cells, highly reducing environments                     thioredoxin (TrxSS/Trx(SH)2) system, glutathione (GSSG/ <br /> found compartments including cytosol, mitochondrial                            2GSH), NADP+/NADPH couple, protein thiols antiox- <br /> matrix peroxisome. Conversely, optimal oxidative protein                          idant defence systems observed impact redox <br /> folding maturation requires disulfide bond formation                        homeostasis cell-wide level [9,12,13], exact genetic <br /> more oxidative redox environment, endoplasmic                            mechanisms place maintenance redox homeostasis are <br /> reticulum [1,2] mitochondrial inter-membrane space [3&#8211;5].                         unknown. Studying cells regulate redox state sub- <br /> Maintenance appropriate cellular sub-cellular redox                            cellular level lead better understanding aberrant <br /> environments crucial alterations redox adversely                      redox biology impact cellular health variety of <br /> affect biological processes signal transduction [6], protein                  settings. <br /> structure function, DNA RNA synthesis [7], enzyme                                  In <span id='am-63' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-64' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> gain better insight genetic organization <br /> activation regulation cell cycle [8,9]. Moreover, altered                  underpinning cellular redox homeostasis, genome-wide screen of <br /> cellular redox homeostasis associated neurodegen-                       cytosolic redox state carried using <span id='am-45' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-46' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-51'  ></span>Saccharomyces</span> <br />                                                                                       cerevisiae tetO7 promoter replacement collection (essential genes) <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                        1                           September 2012 | Volume 7 | Issue 9 | e44278 <br />                                                                                                          Factors Affecting Sub-Cellular Redox State <br />  <br />  <br /> [14] homozygous diploid non-essential gene deletion                  roGFP2 probe completely reduced untreated <br /> collection [15]. Due relatively high concentration (in            conditions. The roGFP2 protein extracted untreated cells <br /> millimolar range) low half-cell reduction potential pH 7              mobility identical protein extracted cells treated <br /> (2240 mV), glutathione couple considered primary                  10 mM DTT. Treatment diamide resulted shift in <br /> cellular redox buffer redox state state (EGSH) used         protein migration corresponding oxidized <br /> indicator state cellular and/or sub-cellular redox            roGFP2 protein. Conversely, wild-type GFP (non-redox sensitive <br /> environment. Cytosolic redox potential (EGSH) systematically             GFP) did respond treatment diamide DTT. <br /> analyzed identify genes affecting cytosolic redox homeostasis.            Together data indicate organelle-targeted roGFP2 <br /> Redox measurements determined using roGFP2 probe,                   probes sense changes intracellular redox environment <br /> which predominantly equilibrates GSSG/2GSH redox                    responsive changes redox changes. <br /> couple [16,17] acts vivo sensor couple. 102 <br /> genes identified essential maintenance cytosolic              High Throughput Screen Yeast Essential Non- <br /> redox state, majority previously known           essential Deletion Libraries Identifies 102 Genes Required <br /> important redox homeostasis. Specific oxidative stress genes <br /> were over-represented genes identified taking <br />                                                                              Cytosolic Redox State Maintenance Steady- <br /> advantage ability roGFP2 targeted individual              state Conditions <br /> sub-cellular compartments, targeted screen 42 mutants                      A genome-wide screen carried identify mutants <br /> affected oxidative stress pathways cytosol, mitochondrial          affected cytosolic redox environment using redox-sensitive <br /> matrix peroxisome conducted better understand                 green fluorescent protein (roGFP2). The collection 4,800 <br /> systems required maintain redox homeostasis sub-cellular             homozygous diploid mutants, deleted non-essential <br /> level.                                                                       gene [15] transformed roGFP2 construct. Cells were <br />                                                                              grown exponential phase 25uC SCURA 96-well plate <br /> Results                                                                      format ratio intensity emission excitation at <br />                                                                              405 nm 488 nm (R405/488) determined mutant. <br /> Verification Redox-responsiveness roGFP2-based                         The primary round screening conducted using 96-well <br /> Probes                                                                       plate method fluorescent spectrophotometer estimate the <br />    Probes used measure redox state GSSG/2GSH                   emission ratio. All potential mutants showing shift R405/488 <br /> couple based roGFP2 probe targeted                   decrease emission excitation 488 nm and/or <br /> cytosol mitochondrial matrix generated verified              increase emission excitation 405 nm were <br /> described [18] localization peroxisome-targeted roGFP2             subsequently rescreened using flow cytometry, found <br /> was analyzed confocal microscopy study (Figure S1). In            sensitive. An increase R405/488 indicated shift <br /> yeast cells expressing roGFP2-SKL punctate fluorescent pattern             cytosolic redox state oxidized environment. <br /> consistent peroxisomal localization observed. Addition-             Mutants displayed statistically different R405/488 values <br /> ally, expressed roGFP2-SKL pex5 cells,                  compared wild type using p-value 0.001 then <br /> defective import peroxisomal proteins peroxisomal           identified. To rule false positives, mutants treated with <br /> targeting signal 1 (PTS1) -SKL. In pex5 cells, cytosolic           10 mM dithiothreitol (DTT) strong reductant. If change <br /> fluorescent pattern observed indicative roGFP2-SKL                  R405/488 redox dependent, example the <br /> construct correctly targeted imported PTS1-               accumulation autofluorescent metabolites cell-wall compo- <br /> mediated import pathway peroxisome.                                 nents, treatment DTT significantly alter R405/ <br />    The roGFP2 probes response redox state calibrated             488 pronounced way. However, change R405/488 was <br /> in vitro (18). Additionally, responsiveness vivo redox state       redox-dependent, DTT treatment restore R405/488 to <br /> changes. To verify roGFP2 organelle-targeted <span id='am-100' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-101' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-102' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span>            wild type. After stringent testing process, 89 <br /> redox responsive vivo, wild-type cells expressing organelle-              mutants significantly higher R405/488 compared wild type <br /> targeted roGFP2 grown exponential phase SCURA                 identified. To allow pH-corrected EGSH values, the <br /> treated various concentrations dithiothreitol, strong thiol-       cytosolic pH wild type determined pH 7.5 using <br /> containing reductant diamide known GSH oxidizing agent.                 cytosolic pHluorin probe. This value used the <br /> Wild-type cells treated various concentrations DTT,                  determination cytosolic redox potential (EGSH) cytosolic <br /> resulted small change EGSH indicating               pH identified mutants verified similar wild type. <br /> cytosol, peroxisome mitochondrial matrix roGFP2 probe                Table 1 lists 15 mutants showing significantly more <br /> was entirely fully reduced state (Figure S2A).                   oxidized cytosol wild type list mutants <br /> Conversely, diamide-treated cells displayed progressively               identified given Dataset S1. <br /> oxidized EGSH values reached plateau reporter                   The screen essential gene mutations carried using the <br /> was oxidized limit (Figure S2A). This verified               tetO7 promoter replacement (tetO7) collection [14], which <br /> organelle-targeted roGFP2 probes able respond dynam-                 contains , 800 mutants, mutant KanR-tetO7- <br /> ically reduced oxidized state intracellular         TATACYC1 cassette integrated promoter essential <br /> redox state changes. To verify changes observed flow                gene tetracycline transactivator (tTA) integrated the <br /> cytometry changes redox-state roGFP2               URA3 locus form &#8216;&#8216;tet-off&#8217;&#8217; system. Addition doxycycline to <br /> protein vivo, redox western blots carried determine           medium represses expression genes control the <br /> the redox status roGFP2 probe untreated cells cells            tetO7-CYC1 promoter. Mutants tetO7 collection were <br /> treated diamide (10 mM; 20 min) DTT (10 mM; 20 min).                 separately transformed cytosolic roGFP2 construct <br /> Proteins harvested trichloroactetic acid precipitation           (leucine-based selection) grown 25uC SCLEU 96-well <br /> protein concentration determined detergent-compatible                   plate format presence doxycycline. The ratio the <br /> Bradford protein assay (Figure S2B). The western blots indicate              intensity emission excitation 405 nm 488 nm (R405/ <br /> that cytosol, mitochondrial matrix peroxisome                 488) determined mutant grown 24 hours and <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               2                       September 2012 | Volume 7 | Issue 9 | e44278 <br />                                                                                                                      Factors Affecting Sub-Cellular Redox State <br />  <br />  <br />  <br />  Table 1. List non-essential S. cerevisiae genes                            Functional Analysis Genes Identified <br />  deleted significantly affect cytosolic redox state.                                        The mutants identified screen analyzed identify <br />                                                                                          significantly enriched (p,0.01) cellular processes functions <br />                                                                                          using Gene Ontology Munich Information Center for <br />                                                                   EGSH&#177;SD                Protein Sequences (MIPS) databases FunSpec [19] as <br />  Mutant        Function                                           (2mV)                  described Materials Methods. Importantly, Funspec <br />  Wild type &#8211;                                                      34963                  analysis showed enrichment slow growing mutants, <br />                                                                                          indicating altered growth does affect cytosolic <br />  glr1          Cytosolic/mitochondrial glutathione reductase      27564 <br />                                                                                          redox state. As seen Figure 1, significantly enriched terms <br />  yap1          Oxidative stress responsive transcription factor   31964 <br />                                                                                          generated databases represent range cellular <br />  skn7          Oxidative stress responsive transcription factor   32065                  processes functions databases categorizing the <br />  dog2          2-deoxyglucose-6-phosphate phosphatase             32564                  data appreciably different ways. From analysis, genes <br />  dig1          MAP kinase-responsive inhibitor Ste12p      32766                  involved endoplasmic reticulum protein transport translo- <br />  dgr2          Protein unknown function                        33064 <br />                                                                                          cation including components signal recognition particle <br />                                                                                          (SRP) complex enriched data set. While not <br />  dit1          Sporulation-specific enzyme                        33063 <br />                                                                                          study redox state endoplasmic reticulum too <br />  tkl1          Transketolase                                      33064                  oxidizing measured roGFP2, evident defects in <br />  ipk1          Inositol 1,3,4,5,6-pentakisphosphate 2-kinase      33263                  endoplasmic reticulum protein translocation affect cytosolic redox <br />  ysp1          Mitochondrial protein                              33264                  state. Mutations SRP downstream effects through <br />  nvj1          Nuclear envelope protein                           33263                  increased cytosolic accumulation secretory protein precursor <br />                                                                                          chaperones accumulation misfolded proteins <br />  rpl26b        Component large (60S) ribosomal             33266 <br />                subunit                                                                   endoplasmic reticulum (as suggested [20]) leading to <br />                                                                                          oxidative stress rounds operation protein folding <br />  inp2          Peroxisome-specific receptor                       33266 <br />                                                                                          machinery. <br />  yju3          Monoglyceride lipase                               33265 <br />                                                                                             Many genes identified possibly represent new functions <br />  ubi4          Ubiquitin                                          33264                  important redox homeostasis. DOG2, encodes 2- <br />                                                                                          deoxyglucose-6-phosphatase, induced oxidative osmotic <br />  Mutants listed oxidized. <br />  doi:10.1371/journal.pone.0044278.t001                                                   stress [20] results point having significant role in <br />                                                                                          regulating redox homeostasis, current functional <br />                                                                                          description indicates. Additionally, identification genes that <br /> R405/488 calculated. As non-essential homozygous diploid <br />                                                                                          near identical duplicates (for example HTA1 RPL26) or <br /> collection, mutants tetO7 collection having R405/488 <br />                                                                                          greater protein complex (LEA1; required U2 snRNP) <br /> significantly greater wild type treated DTT to <br />                                                                                          suggest sub-set genes moonlight redox <br /> identify redox-dependent changes cytosolic pH measured. <br />                                                                                          sensors. There genes encoding RPL26 S. cerevisiae, <br /> 13 genes identified significantly higher R405/488 <br />                                                                                          RPL26A RPL26B nearly identical HTA2, only <br /> compared wild type cytosolic redox potential (EGSH) <br />                                                                                          mutation RPL26B HTA1 caused changes cytosolic redox <br /> was calculated (Table 2). <br />                                                                                          state. The fact mutation profound effect <br />                                                                                          redox homeostasis indicates genes RPL26B and <br />                                                                                          HTA1 specific function redox homeostasis aside <br />  <br />  Table 2. Mutants tetO7 promoter replacement collection exhibited significantly oxidized cytosol wild <br />  type treatment doxycycline. <br />  <br />  <br />  Gene                                      Function                                                                                        EGSH&#177;SD (2mV) <br />  <br />  Wild type                                                                                                                                 322 <br />  tet-O7-STT4                               Phosphotidylinositol-4-kinase                                                                   27266 <br />  tet-O7-SRP14                              Signal recognition particle complex subunit                                                     282616 <br />  tet-O7-yor146w                            Dubious opening reading frame                                                                   28466 <br />  tet-O7-CDC34                              Ubiquitin-conjugating enzyme                                                                    29167 <br />  tet-O7-YPT1                               Rab <span id='am-52' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-53' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> GTPase                                                                               298614 <br />  tet-O7-SPC98                              Component microtubule-nucleating Tub4p complex                                           29863 <br />  tet-O7-SAH1                               S-adenosyl-L-homocysteine hydrolase                                                             29866 <br />  tet-O7-SEC53                              Phosphomannomutase                                                                              29965 <br />  tet-O7-RIB5                               Riboflavin synthase                                                                             29966 <br />  tet-O7-TRR1                               Cytoplasmic thioredoxin reductase                                                               30064 <br />  tet-O7-GPI16                              Transmembrane protein subunit glycosylphosphatidylinositol transamidase complex          30766 <br />  tet-O7-NUP159                             Nucleoporin subunit nuclear pore complex                                                 310610 <br />  tet-O7-NAB3                               Single-stranded RNA binding protein                                                             31063 <br />  <br />  doi:10.1371/journal.pone.0044278.t002 <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                         3                         September 2012 | Volume 7 | Issue 9 | e44278 <br />                                                                                                            Factors Affecting Sub-Cellular Redox State <br />  <br />  <br />  <br />  <br /> Figure 1. Characterization mutants displaying significantly oxidized cytosol relative wild type. All mutants exhibited <br /> a significantly oxidized cytosol compared wild type analyzed enriched terms using Gene Ontology (GO) Munich Information <br /> Center Protein Sequences (MIPS) databases FunSpec [18]. p-values (Boneferroni corrected) represent probability intersection a <br /> given list functional category occurs chance. The terms identified related oxidative stress highlighted bold. <br /> doi:10.1371/journal.pone.0044278.g001 <br />  <br /> from main function. Alternatively, gene identified          matrix redox state observed ccp1 mutant compared the <br /> expressed highly homologue leading deletion             skn7 mutant. Deletion CCP1 cause increased levels of <br /> having greater effect redox homeostasis. In vein,          hydrogen peroxide accumulate decreased activation of <br /> identification gene protein complex LEA1            Skn7p doubly affecting matrix redox state. Taken together, is <br /> (component U2 snRNP complex) PXA1 (subunit                  apparent connection mitochondrial <br /> peroxisomal ATP-binding cassette transporter complex; Pxa1p-                 function redox homeostasis. The link mitochondrial <br /> Pxa2p), indicate specialized role genes redox             function redox homeostasis reasons. <br /> homeostasis.                                                                 Many mitochondrial processes integral generation of <br />    While functional categorization gene lists using GO terms           redox-active <span id='am-60' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> NADH NADPH, involve <br /> especially useful working large amounts data,             redox-dependent <span id='am-71' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-72' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-73'  ></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>electron</span> transfer reactions produce reactive <br /> such terms sufficiently fine descriptive identify              oxygen species. Consequently, redox-association many <br /> functional groups allocate genes products                      mitochondrial functions indicate potential role in <br /> pleiotropic effects studied, established groups.            setting, sensing maintenance cellular redox <br /> Closer inspection data revealed processes affecting              environment. <br /> cytosolic redox homeostasis mitochondrial function (YPS1,               Four mutants (ubi4, uip4, ubp13 cdc34-tetO7) involved in <br /> NDE1, MRPL25, MRPS16, MRPL35, MRPL24, ICP55, MNE1,                           ubiquitin-dependent protein degradation identified having <br /> MSF1, MRPL40, COQ10, MSW1, TIM8, AIM38) ubiquitin                    oxidized cytosol wild type. Moreover, decreased <br /> system (UBI4, UIP4, UBP13, CDC34) detected                expression CDC34, essential gene encodes ubiquitin- <br /> the GO MIPS databases. These represent new pathways                       conjugating enzyme [23], led oxidized cytosol. Mutants <br /> somehow involved redox homeostasis. Respiratory-incompetent               ubiquitin previously identified screen for <br /> cells increased levels GSSG stationary phase relative          mutants over-excrete glutathione [24] CDC34 was <br /> exponential phase mitochondrial function required                 identified high-copy suppressor poor growth phenotype <br /> resistance oxidative stress [21]. Ccp1p involved               gsh2 cells, lack second step glutathione synthesis <br /> oxidative stress signaling leading activation Skn7p                [25]). CDC34 shown activate GSH1 (encoding c- <br /> transcription factor [22]. This explain larger shift            glutamylcysteine synthetase) expression modulating activity <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               4                        September 2012 | Volume 7 | Issue 9 | e44278 <br />                                                                                                          Factors Affecting Sub-Cellular Redox State <br />  <br />  <br /> of Met4p, transcriptional activator sulphur assimilation         tional response oxidative stress, NADPH generation, the <br /> pathway [25], providing link GSH ubiquitin                thioredoxin glutathione systems antioxidant defences. A <br /> systems. Given overexpression CDC34 increases GSH levels            list pathways mutants given Table 3. <br /> and repression CDC34 leads oxidized cytosolic redox            Compartmental EGSH subsequently estimated mutants <br /> state, tight control Cdc34p required optimal intracellular        experiment carried triplicate. For calculation of <br /> glutathione concentrations, alterations activity              EGSH pH values cytosol, mitochondrial matrix and <br /> Cdc43p affect glutathione homeostasis. Disruption              peroxisome using targeted pHluorin probes calculated 7.5, <br /> ubiquitin protein degradation processes broad            7.7 8 respectively used EGSH determination. <br /> level Cdc34p, affect GSH1 gene expression              Data obtained cells exponential phase (A600 = ,0.5) <br /> regulation glutathione couple redox state.                       stationary phase (A600 = 5&#8211;6) presented Dataset S2. <br />                                                                                From targeted screen, genes affecting redox state all <br /> Specific Oxidative Stress Responsive Genes Essential                    organelles tested genes unique maintenance redox state <br /> for Cytosolic Redox State Homeostasis                                       particular compartment identified exponential <br />    Interestingly, databases highlighted enrichment              stationary phase. These outlined Figure 2A 2B. <br /> genes involved oxidative stress response reactive oxygen          Only 12 mutants (exponential phase) 10 mutants (stationary <br /> species shown bold Figure 1. Some oxidative-stress                 phase) 42 tested altered redox the <br /> responsive genes identified showed extensive shift              compartments. Given fact large number genes and <br /> EGSH compared wild type (DEGSH), glr1, yap1, skn7           systems known deal ROS major perturbations of <br /> and tkl1 mutants 10 mutants largest               redox homeostasis, significant mutations these <br />                                                                             systems influence redox state measured using roGFP2. <br /> DEGSH values. Genes involved detoxification reactive <br />                                                                             This indicates cellular redox homeostasis tightly regulated <br /> oxygen species conventionally regarded response <br />                                                                             subset key oxidative stress proteins modulate <br /> systems maintenance systems. However results of <br />                                                                             EGSH. <br /> the screen indicate key oxidative stress responsive <br /> systems serve dual role. This challenges paradigm                   The prominent result requirement glutathione <br />                                                                             reductase (Glr1p), NADPH regeneration systems tran- <br /> maintenance response systems distinct elements redox <br />                                                                             scription factors Yap1p Skn7p maintenance redox <br /> regulation supports view systems are <br />                                                                             state compartments, highlighting fundamental role of <br /> needed response counteract deleterious oxidative changes <br />                                                                             systems redox homeostasis. The requirement for <br /> are required maintain redox homeostasis steady state <br />                                                                             glutathione reductase independent growth phase unlike <br /> conditions. Clearly, genes identified screen will <br />                                                                             Yap1p Skn7p, appeared important during <br /> indirectly affect redox environment direct <br />                                                                             exponential phase compartments. The identification of <br /> modulators cytosolic redox state. For instance possible that <br />                                                                             Yap1p Skn7p essential cytosolic redox response to <br /> mutants increased levels oxidative stress, could <br />                                                                             stress indicate transcription factors sit boundary of <br /> affect roGFP2 readout directly affecting the <br />                                                                             maintenance-response systems. Basal levels expression the <br /> glutathione couple. However, delineating primary and <br />                                                                             genes regulate important steady-state redox homeo- <br /> secondary redox effects complex focus present <br />                                                                             stasis, induction activation target genes appears <br /> study. Additionally, interesting ,10% genes <br />                                                                             important mounting appropriate responses oxidative stress. <br /> identified screen known putative function. These <br />                                                                                From data appears cytosol mitochon- <br /> now seen involvement, directly indirectly <br />                                                                             drial matrix contain systems required meet unique redox <br /> in modulating cytosolic redox state. <br />                                                                             metabolic requirements organelle. The mitochondrial <br />                                                                             redox state affected deletion oxidative stress <br /> Targeted Cross-compartmental Screen Understand                       transcription factors mitochondrial NADPH generating <br /> Role Oxidative-stress Systems Redox Homeostasis                       enzymes NADH kinase encoded POS5 the <br />    From mutants identified screen, (glr1, yap1,        NADP+-dependent isocitrate dehydrogenase encoded IDP1 or <br /> skn7, tkl1 trr1; [12,26]) previously reported           mitochondrial ROS detoxifiers Mn-superoxide dismutase and <br /> involved redox homeostasis. Some genes previously identified          cytochrome c peroxidase encoded SOD2 CCP1 respective- <br /> affecting cellular redox state identified             ly. The ccp1 mutant altered matrix redox state both <br /> screen possibly differences methods used estimate          growth phases, sod2 mutants exhibited altered matrix <br /> EGSH redundancy known antioxidant functions.                    redox stationary phase, probably increased <br /> Given genes involved antioxidant function resistance         requirement Sod2p stationary phase [27]. The significant <br /> oxidative stress identified important cytosolic redox            role genes play maintaining mitochondrial redox <br /> maintenance genome-wide screen, looked                  state reflects higher burden compartment faces terms of <br /> more closely known pathways involved antioxidant function             endogenous ROS production subsequent detoxification as <br /> and oxidative stress responses understand better role           normal metabolism compared compartments. The <br /> setting, sensing maintenance redox state. Moreover,            effect deleting CCP1 particularly interesting no <br /> analyses carried compartmental level (in cytosol,         compartment did deletion peroxide-detoxifying gene <br /> mitochondrial matrix peroxisome) understand role              result alteration redox state. Given mitochondrial <br /> these functions organelles nature systems               matrix peroxide-detoxifying enzymes, example Prx1p <br /> maintaining redox state sub-cellular level.                            [28] Cta1p [29], data suggest separate significant <br />    For study, cells transformed compartment-targeted              role Ccp1p mitochondrial redox regulation. <br /> roGFP2 constructs grown SCURA exponential                           In contrast, observed peroxisomal redox highly <br /> (A600 = ,0.5) stationary phase (A600 = 5&#8211;6) analyzed            reliant cytosolic peroxisomal redox systems, with <br /> flow cytometry. EGSH measured cytosol, mitochondrial             glutathione reductase activity, operation pentose <br /> matrix peroxisome 42 mutants affected transcrip-              phosphate pathway Cu,Zn superoxide dismutase critical <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              5                       September 2012 | Volume 7 | Issue 9 | e44278 <br />                                                                                                                      Factors Affecting Sub-Cellular Redox State <br />  <br />  <br />  <br />  Table 3. Genes known pathways important antioxidant function resistance oxidative stress [53]. <br />  <br />  <br />  Process                                                               Genes involved <br />  <br />  Transcription factors                                                 msn2, msn4, yap1, skn7 gnd1, gnd2, tal1, <br />  NADPH regeneration                                                    sol3, sol4,rpe1, tkl1, ald6, idp1, pos5, idp2, idp3 <br />  GSH                                                            glr1, gpx1, gp2, gpx3, grx1, grx2, grx3, grx4, grx5, grx6, grx7, grx8 <br />  Thioredoxin                                                    trx1, trx2, trx3, trr2 <br />  Peroxiredoxins                                                        prx1, tsa1, tsa2, ahp1, dot5 <br />  Antioxidants                                                          ctt1, cta1, sod1, sod2, ccp1 <br />  <br />  Description gene information obtained Saccharomyces Genome Database (SGD). <br />  doi:10.1371/journal.pone.0044278.t003 <br />  <br />  <br /> for maintenance redox environment peroxisome.                      regulation previously thought, represent a <br /> Since little known redox systems place               transcriptional acting concert YAP1/SKN7 tran- <br /> peroxisome, data help understand manner                  scription factors regulate redox state. <br /> the peroxisome regulates redox homeostasis relationship <br /> with compartments. The dependence peroxisome                     Discussion <br /> cytosolic pathways indicates clear inter-relationship between <br /> peroxisomal cytosolic redox homeostasis.                                        The present study undertaken aim better <br />    One outcome study identification                    understanding genetic basis setting, sensing and <br /> function genes annotated Saccharomyces Genome                maintenance cellular redox homeostasis. The development of <br /> Database &#8216;&#8216;of unknown putative function&#8217;&#8217;. This led              redox-sensitive fluorescent probes roGFP2 enabled the <br /> to new improved annotations 10 genes (Table 4) basis                high throughput study redox state in vivo real-time <br /> of involvement redox homeostasis potentially                     manner distinct cellular compartments, circumventing <br /> oxidative stress responses. A good example HMS2,                   issues associated traditional techniques for <br /> paralogue SKN7 arising ancient genome duplication                    estimating redox state, mixing compartmental <br /> [30]. HMS2 putative transcription factor annotated               glutathione pools cell volume estimation. This study <br /> SGD &#8216;&#8216;protein similarity heat shock transcription                   represents systematic genome-wide study analyzing <br /> factors&#8217;&#8217; involved pseudohyphal growth nitrogen deple-                  cellular redox state use genome-wide redox <br /> tion. Despite rapid protein divergence HMS2 SKN7,                    potential data comprehensively understand bases redox <br /> HMS2 shares heat-shock factor DNA-binding domains                     homeostasis compartmental level. The combination a <br /> Skn7p predicted basis microarray data                  genome-wide screen targeted sub-screen identify factors <br /> regulate subset responses reducing conditions, heat shock                contributing sub-cellular redox homeostasis, highlights value <br /> and hydrogen peroxide [30]. Moreover, Hms2p structurally                 vivo redox probes examining redox homeostasis and <br /> similar Skn7p Sfl1p, transcriptional                   broader cellular issues involving redox perturbations. While <br /> activator repressor involved activation stress-            roGFP2 predominantly equilibrates glutathione couple, <br /> response genes including Hsp30, strongly induced                     redox state acts reflection protein- <br /> response various environmental stress conditions [31]. HMS2                   sulfhydryl redox buffering system. Protein-thiols contribute signif- <br /> therefore, greater role redox oxidative stress                      icantly cellular redox state [13] measurements using <br />  <br />  <br />  <br />  <br /> Figure 2. Functional overlap mutants oxidizing cytosol and/or mitochondrial matrix and/or peroxisome. <br /> Mutants known pathways involved antioxidant defence oxidative stress responses analyzed exponential (A600 = ,0.5) stationary <br /> phase (A600 = 5&#8211;6) cytosol, mitochondrial matrix peroxisome. Analysis carried using flow cytometry EGSH estimated. 10,000 <br /> cells counted conditions experiment conducted triplicate. Mutants DEGSH compared wild type 10 mV or <br /> more A) exponential B) stationary phase compartment tested shown. Data taken Dataset S2. <br /> doi:10.1371/journal.pone.0044278.g002 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                  6                              September 2012 | Volume 7 | Issue 9 | e44278 <br />                                                                                                                     Factors Affecting Sub-Cellular Redox State <br />  <br />  <br />  <br />  Table 4. Genes annotated &#8216;&#8216;putative unknown function&#8217;&#8217; identified contributing cytosolic redox homeostasis this <br />  study oxidative stress phenotype(s). <br />  <br />  <br />  Gene                   Phenotype                                                                                                   Reference <br />  <br />  DOG2                   Induced oxidative stress                                                                                 [20] <br />  DGR2                   Implicated stress response signalling                                                                    [54] <br />  RPI1                   Inhibitor Ras-cAMP pathway                                                                               [55] <br />  PMP3                   Increased glutathione excretion                                                                             [24] <br />  HMS2                   Predicted regulated responses reducing conditions, heat shock hydrogen peroxide.             [56] <br />                         Homolog SKN7 <br />  SVF1                   Inhibits reactive oxygen species generation promotes cell survival oxidative stress conditions    [57] <br />  ECM25                  Increased glutathione secretion; decreased oxidative stress resistance                                      [24,58] <br />  ISF1                   Similar Mbr1p, protein involved mitochondrial function stress responses                         [59] <br />  <br />  doi:10.1371/journal.pone.0044278.t004 <br />  <br />  <br /> roGFP2 likely reflect glutathione redox state                   activity Glr1p predominant. Similarly, NADPH <br /> the protein-thiol network allowing reporter multi-                      regeneration systems important maintaining redox <br /> dimensional read-out redox conditions cell. Since                     state exponential phase multiple NADPH regeneration <br /> roGFP2 probe wild-type strain fully reduced                  mutants identified (cytosol - tkl1, gnd1; matrix &#8211; tkl1, gnd1, idp1, <br /> state compartments studied, used                pos5; peroxisome - rpe1, gnd1, tkl1). <br /> identify factors resulted reducing cellular conditions.                   The finding key oxidative stress response elements as <br /> Re-engineered roGFP2 probes various mid-point potentials                       transcription factors Yap1p Skn7p, GSH regeneration <br /> [16,32] used study hyper-reducing conditions                         essential redox maintenance steady state conditions <br /> investigate redox state endoplasmic reticulum,                  alter concept systems involved maintenance of <br /> too oxidizing studied using roGFP2. It kept mind,                  redox homeostasis systems response ROS distinct <br /> however definitive cellular redox state.                      other. These data support view oxidative <br /> Glutathione present high concentration,                         stress response systems evolved mechanisms were <br /> important feature buffer. Many important redox couples                   important maintenance redox homeostasis. It evident <br /> exist cells varying interactions NADP+/                         study redox homeostasis tightly regulated, with <br /> NADPH TrxOX/TrxRED couples. An increasing body                               redundant systems place ensure maintenance the <br /> evidence pointing complex relationship                     appropriate redox environment compartment. For the <br /> glutathione thioredoxin systems functional overlap                       part, mutations needed cause large shifts <br /> between them. The thioredoxin important preventing                     redox environment, highlighting robustness cell in <br /> over-accumulation GSSG cells lacking glutathione reductase                    terms redox control. The phenotype single mutations does <br /> [33,34]. Additionally, thioredoxin                       reflect complete network cell functions. We did <br /> system specifically involved regulating glutathione redox                     observe significant redox shift trx1 trx2 mutants. <br /> potential nucleus presence glutathione                        However, Drakulic colleagues (2005) showed the <br /> reductase [35]. Future studies greatly enhanced                     presence TRX1 TRX2 required maintenance <br /> use reporter constructs equilibrate thioredoxin                  EGSH [12], despite redundancy cytosolic <br /> system NADP+/NADPH couple broader under-                               thioredoxins glutaredoxins cell viability [12]. In that <br /> standing redox cellular dynamics.                                                study, greatest shift EGSH wild-type exponential <br />    From data obtained, redox homeostasis cytosol,                        phase seen trx1 trx2 double mutant. Therefore, while <br /> mitochondrial matrix peroxisome primarily controlled                      cellular redox systems redundant robust, complex redox <br /> regulated glutathione cycling growth-phase-independent                         relationships exist. Multiple mutations additive <br /> manner glr1 mutant exhibited highest DEGSH. In                        effects likely represent pathways acting concert correct <br /> exponential phase Yap1p Skn7p transcription factors                    sensing regulating particular redox couples redox <br /> affected sub-cellular redox. The lack redox shift msn2                homeostasis general, example glutathione and <br /> msn4 mutants indicates Msn2p MSn4p                               thioredoxin systems. As state previously, <span id='am-1' about='obo:NCBITaxon_905814' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_7401 obo:NCBITaxon_7402 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_65231 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_33317 obo:NCBITaxon_7399 obo:NCBITaxon_2759 obo:NCBITaxon_50557 obo:NCBITaxon_6960 obo:NCBITaxon_6072'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Iare" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5'  ></span>iare</span> numerous <br /> transcription factors important general stress response                 connections thioredoxin glutathione systems. The <br /> [36], individually play little role maintenance                    thioredoxin reductase-glutathione reductase double mutant (trr1 <br /> redox state. Therefore, unlikely genes basal levels                glr1) inviable [39] trr1 glr1 double mutant inviable [39] <br /> are highly regulated Msn2p/Msn4p GRX2 [37]                         single thioredoxin glutaredoxin essential for <br /> be required steady state redox homeostasis reflected                viability S. cerevisiae [40]. Deletion thioredoxin <br /> in results. The reduced need stationary phase                        reductases results increased intracellular concentrations of <br /> Yap1p Skn7p compartments probably fact                     oxidized reduced glutathione [41,42] recently, was <br /> that, exponential phase, cells actively growing                      thioredoxin/thioredoxins reductase could <br /> require transcription redox maintenance genes                   function alternative reduce oxidized glutathione in <br /> diminishes stationary phase active metabolism                           vivo S. cerevisiae [33] <span id='am-11' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15'  ></span><span id='am-16' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span>Arabidopsis</span> thaliana [34]. <br /> protein synthesis [38] decline. In stationary phase, need                       There strong connection mitochondrial function <br /> regenerate existing reduced glutathione important                   redox homeostasis. Deletion CCP1 encoding cytochrome <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                      7                          September 2012 | Volume 7 | Issue 9 | e44278 <br />                                                                                                        Factors Affecting Sub-Cellular Redox State <br />  <br />  <br /> c peroxidase affected matrix redox state growth phases.            Materials Methods <br /> As Ccp1p detoxifies hydrogen peroxide peroxides, it <br /> appears detoxification endogenous hydrogen peroxide             Strains, Media Growth Conditions <br /> required maintain mitochondrial matrix redox state. Inter-                 For genome-wide screening, homozygous diploid knock-out <br /> estingly, Ccp1p involved oxidative stress signaling             mutant collection (Euroscarf) [15] (MATa/MATa his3D1/his3D1 <br /> leading activation Skn7p transcription factor [43]. This         leu2D0/leu2D0 met15D0/MET15 LYS2/lysD0 ura3D0/ura3D0, <br /> could explain larger shift matrix redox state observed           KanMX4::target gene) tetO7 promoter replacement <br /> the ccp1 mutant compared skn7 mutant. Deletion CCP1              collection (OpenBiosystems) [14] (MATa; his3D1; leu2D0; met15D0; <br /> may simultaneously cause increased hydrogen peroxide accu-                 URA::CMV-tTA, kanR-tetO7-TATA::target gene) used. <br /> mulation decreased activation Skn7p, doubly affecting               Strains grown synthetic complete (SC) medium (0.17% <br /> matrix redox state. The signaling function Ccp1p provides             yeast nitrogen base amino acids, 0.5% ammonium sulfate, <br /> clear example existence mitochondrial signaling                  2% w/v glucose) supplemented appropriate amino acids and <br /> molecules cross-compartmental redox signaling. Despite              bases (Table S1). Dropout medium appropriate <br /> mitochondrial function absent functions                   supplement omitted provide plasmid selection denoted a <br /> significantly enriched dataset using GO MIPS                subscript, example SC medium lacking leucine denoted as <br />                                                                            SCLEU. Media solidified addition 2% (w/v) agar. <br /> databases, clear relationship exists some <br />                                                                            Strains transformed plasmids using 96-well plate <br /> mitochondrial functions maintenance cytosolic redox <br />                                                                            transformation protocol [47]. <br /> state mutations affecting mitochondria identified as <br /> affecting cytosolic redox state. From previous work known <br /> that respiratory-incompetent cells increased levels GSSG           Organelle-targeted roGFP2 Construction <br /> in stationary phase relative exponential phase                    Details organellar-targeted roGFP2 constructs - pAG416- <br /> mitochondrial function required resistance oxidative             roGFP2 pAG416-COX4roGFP2 described [18]. The <br /> stress [44]. Key mitochondrial proteins act cellular            peroxisome-targeted roGFP2 probe, pAG416-roGFP2-SKL was <br /> signaling sensing redox change retrograde redox             generated using PCR amplification Invitrogen Gateway <br />                                                                            SystemH. Briefly roGFP2 PCR amplified plasmid <br /> signaling pathway(s). Since mitochondria largest site of <br />                                                                            pBinCM-TKTP-GRX1roGFP2 peroxisomal C-terminal <br /> endogenous ROS production, redox state changes mitochon- <br />                                                                            targeting sequence seryl-lysyl-leucine (SKL) added the <br /> dria initial indicator overall cellular redox <br />                                                                            primer used amplification. attb1 attb2 sites added to <br /> dysfunction. <br />                                                                            primer appropriate resulting PCR products <br />    Little known peroxisomal redox despite the <br />                                                                            flanked 59 attb1 site 39 attb2 site. The resulting <br /> peroxisome significant site endogenous ROS <br />                                                                            PCR fragments used GatewayH BP clonaseH reaction <br /> production. Despite this, peroxisome exhibited distinctive             using destination vector pDonr221. The pDonr221-roGFP2- <br /> redox characteristics, curious dependence cytosolic              SKL plasmid generated using Invitrogen GatewayH system <br /> proteins glutathione reductase (Glr1p), enzymes involved           according manufacturer&#8217;s instructions resulting plasmid <br /> in pentose phosphate pathway (Tkl1p, Rpe1p) Cu, Zn                 pDonr221roGFP2-SKL used Gateway LRH clonaseH <br /> superoxide dismutase (Sod1p) peroxisomal proteins              reaction using destination vector pAG416GPD-ccdB [48]. <br /> for maintenance redox state exponential             pHluorin organelle-targeted probes identical <br /> stationary phase. These results raise interesting possibilities       manner roGFP2 based probes using pCB901YpHc the <br /> regarding nature peroxisomal redox maintenance              starting vector encoding pHluorin kind gift R. Rao (Johns <br /> relationship compartments, cytosol particular.           Hopkins University School Medicine, Baltimore). <br /> Free diffusion small molecules cytosol and <br /> peroxisome hypothesized account pool                Organelle-targeted roGFP2 Localization Redox- <br /> GSH GSSG redox state peroxisome <br />                                                                            sensitivity Verification <br /> [45] results suggestive peroxisomal redox state <br />                                                                               Correct localization mitochondrial matrix-targeted roGFP2 <br /> being reliant cytosolic redox maintenance systems. However, <br />                                                                            confirmed confocal co-localization [18]. Correct localiza- <br /> recently, isoform glutathione reductase to <br />                                                                            tion plasmids used study confocal <br /> be dually targeted cytosol peroxisome Arabidopsis            microscopy (Figure S1). The redox responsiveness cells <br /> thaliana novel peroxisomal targeting sequence [46] raising            expressing roGFP2-based plasmids validated flow cytom- <br /> the possibility glutathione reductase localized         etry western blot. Cells grown in SCURA 48 h <br /> peroxisome organisms peroxisome                  (30uC; 600 rpm) inoculated SCURA (A600 = 0.001) and <br /> itself contain discrete glutathione system.                              grown exponential phase. Cells treated various <br />    This study identified approximately 100 genes involved               concentrations strong reductant dithiothreitol the <br /> setting cytosolic redox state using roGFP2 redox reporter              oxidant diamide 30 min redox state roGFP2 <br /> construct. The processes identified highlight certain redox-          determined (data shown). <br /> active systems traditionally thought stress response systems <br /> are critical steady state redox homeostasis, indicate         Analysis Compartmental Redox Potential <br /> redox maintenance systems evolved important               Cells transformed roGFP2 constructs grown in <br /> for responding reactive oxygen species. The study          SCURA 2 d (30uC; 700 rpm), inoculated SCURA medium <br /> phenotype redox dysfunction 12 genes unknown                   grown shaking 25uC appropriate cell yield had <br /> putative function provides rich source analysis       reached. All analyses carried 96-well plate <br /> their cellular role function redox homeostasis.                     format. For preliminary analyses, cytosolic redox state cells <br />                                                                            analyzed fluorescent spectrophotometer (SpectraMax M2, <br />                                                                            Molecular Devices) excitation 405 nm 488 nm. The <br />                                                                            emission excitation wavelength measured and <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             8                       September 2012 | Volume 7 | Issue 9 | e44278 <br />                                                                                                           Factors Affecting Sub-Cellular Redox State <br />  <br />  <br /> then R405/488 ratio calculated, R405/488 representing the <br /> ratio emission excitation 405 nm divided the <br /> emission excitation 488 nm. Given                      0 <br />                                                                                         pH           0             2:303RT <br /> round screening lenient cut-off used identify mutants                    E0(roGFP2) ~E0(roGFP2) {             &#240;pH{7&#222; <br />                                                                                                                       zF <br /> with altered cytosolic EGSH. In preliminary screening we <br /> identified mutants change R405/488                 mid-point redox potential roGFP2 (E0&#8217;roGFP2) <br /> change emission excitation 405 nm           estimated 2272 mV. <br /> and emission excitation 488 nm. These                  For analysis significance differences, t-tests conducted <br /> further screened. Subsequent sub-screening organellar redox            samples using = 0.001. <br /> analyses conducted using FACS Canto II flow cytometer <br /> (Becton Dickinson, Australia). For non-essential mutants, cells            Analysis Compartmental pH <br /> transformed roGFP2 grown exponential phase (A600                 Wild-type (BY4743) cells transformed pHluorin <br /> 0.5&#8211;2.0) SCURA. For screening tetO7 collection, cells            constructs grown SCURA 2 d (30uC; 700 rpm), <br /> transformed roGFP2 (leucine selection) grown SCLEU            inoculated SCURA medium grown shaking 25uC <br /> for 2 d (25uC; 700 rpm) inoculated SCLEU+10 mg/ml                 appropriate cell yield reached. Cells analyzed <br /> doxycycline grown 24 36 h (25uC; 700 rpm). Cells           using FACS Canto II (BD Bioscience) excitation at <br /> then analyzed using FACS Canto II (BD Bioscience) equipped             405 nm 488 nm emission detected 512 nm. The <br /> with <span id='am-90' about='obo:NCBITaxon_62990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_241799 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_241789 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-92'  ></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Laser" datatype="xsd:string"></span>laser</span> excites 405 nm 488 nm emission            ratio emission excitation 405 nm 488 nm was <br /> detected 512 nm. The ratio emission excitation          calculated used pH estimation. 10,000 cells were <br /> 405 488 nm calculated. The R405/488 EGSH                  counted sample. Data obtained analyzed with <br /> calculated previously reported [18,49]. For sample 10,000          FlowJo version 7.4 software. To obtain pH values using the <br /> cells analyzed biological replicates. For           pHluorin probe compartment, standard curve was <br /> analysis significance differences R405/488 EGSH values t-        established allow estimation intracellular pH determining <br /> tests conducted EGSH values reported significant           fluorescence digitonin-permeabilized cells incubated in <br /> were calculated using p-value 0.001. In case entire          buffers pH values 5 8 [50&#8211;52]. For calibration the <br /> experiment repeated twice. Data obtained                 cytosolic-, mitochondrial matrix-, peroxisomal-localized <br /> analyzed FloJo version 7.4 (Tree Star Inc.).                          pHluorin, cells suspended 0.65 M sorbitol containing <br />                                                                            100 mg/ml digitonin incubated (10 min; 600 rpm; 24uC). <br />                                                                            Cells pelleted, washed resuspended calibration buffer <br /> EGSH Calculation <br />                                                                            containing 50 mM 2-(N-morpholino)ethanesulfonic acid (MES), <br />    The following modified Nernst equation [49] used to <br />                                                                            50 mM N-(2-hydroxyethyl)piperazine-N-(2-ethanesulfonic acid) <br /> determine EGSH roGFP2 405/488 ratio. Cells also <br />                                                                            (HEPES), 50 mM KCl, 50 mM NaCl, 200 mM ammonium <br /> treated 20 min DTT (10 mM) diamide (10 mM) to <br />                                                                            acetate, 10 mM NaN3, 10 mM 2-deoxyglucose, 75 mM monensin <br /> obtain measurements fully reduced oxidized (respectively)        10 mM nigericin, titrated different pH values using NaOH <br /> forms roGFP2 probe necessary calculate EGSH             range pH 6.0&#8211;8.0. A calibration curve plotted <br /> using following formulae derived Nernst equation.             using ratio fluorescence intensity 405 nm 488 nm <br />    Briefly, degree oxidation (OxD) roGFP2 sensor         (after background subtraction) pH. <br /> calculated from <br />                                                                            Analysis Genome-wide Data <br />                                    R{Rred                                    Cellular processes over-represented genome-wide screen <br />               OxD~ I488ox <br />                       I488red   &#240;Rox {R&#222;z&#240;R{Rred &#222;                         data identified using Gene Ontology (GO) Munich <br />                                                                            Information Centre Protein Sequences (MIPS) databases via <br /> where R ratio emission excitation 405/488 nm,           FunSpec [19] p-value threshold set 0.01. FunSpec <br /> Rred emission ratio fully reduced form following             calculated p-values using hypergeometric distribution, p- <br /> incubation 10 mM DTT (20 min), Rox emission ratio           value represents probability intersection given list <br /> the fully oxidized form following incubation 10 mM diamide            given functional category occurs chance. The p-values <br /> (20 min) I488ox I488red respective emission                Boneferroni corrected account spurious significance <br /> intensities 488 nm fully oxidized fully reduced forms       multiple testing categories database. <br /> of given sample. The redox potential estimated using: <br />                                                                            Supporting Information <br />                                                                            Figure S1 Confocal microscopy images yeast cells <br />               0 pH        2:303RT           1{OxDroGFP2                    expressing peroxisomal roGFP2. <br />        EGSH ~E0(roGFP2) {             log10 <br />                                  zF          OxDroGFP2                     (EPS) <br />  <br /> where R gas constant (8.315 J K21 mol21), T absolute         Figure S2 Verification organelle-targeted roGFP2 <br /> temperature (298.15 K), z number transferred electrons           responsiveness redox state. <br /> (2), F Faraday constant (9.648 104 C mol21)                     (EPS) <br /> E&#8217;pH0(roGFP2) mid-point redox potential based                Table S1      Supplements synthetic complete (SC) <br /> standard mid-point potential E0 30uC pH 7, adjusted             medium. <br /> the estimated compartment pH experimental temperature                  (DOC) <br /> (20&#8211;25uC) according to:                                                    Dataset S1 Non-essential mutants identified having a <br />                                                                            significantly oxidized cytosol wild type. <br />                                                                            (XLS) <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             9                       September 2012 | Volume 7 | Issue 9 | e44278 <br />                                                                                                                                      Factors Affecting Sub-Cellular Redox State <br />  <br />  <br /> Dataset S2 EGSH cytosol, mitochondrial matrix                                             Author Contributions <br /> and peroxisome cells lacking genes involved redox <br />                                                                                                  Conceived designed experiments: AA GGP IWD. Performed the <br /> homeostasis antioxidant defence exponential                                           experiments: AA SF CF SSL GJS. Analyzed data: AA GGP IWD. <br /> stationary phase.                                                                                Contributed reagents/materials/analysis tools: AJM. Wrote paper: AA <br /> (DOC)                                                                                            GGP IWD. <br />  <br /> References <br />  1. Tu BP, Ho-Schleyer SC, Travers KJ, Weissman JS (2000) Biochemical basis                   27. Flattery-O&#8217;Brien JA, Grant CM, Dawes IW (1997) Stationary-phase regulation <br />     oxidative protein folding endoplasmic reticulum. Science 290: 1571&#8211;1574.                  Saccharomyces cerevisiae SOD2 gene dependent additive effects of <br />  2. Tu BP, Weissman JS (2004) Oxidative protein folding eukaryotes:                               HAP2/3/4/5-and STRE-binding elements. Mol Micro 23: 303&#8211;312. <br />     mechanisms consequences. J Cell Biol 164: 341&#8211;346.                                       28. Pedrajas JR, Miranda-Vizuete A, Javanmardy N, Gustafsson J, Spyrou G (2000) <br />  3. Riemer J, Bulleid N, Herrmann JM (2009) Disulfide formation ER                        Mitochondria Saccharomyces cerevisiae contain one-conserved cysteine type <br />     mitochondria: solutions common process. Science 324: 1284&#8211;1287.                         peroxiredoxin thioredoxin peroxidase activity. J Biol Chem 275: 16296&#8211; <br />  4. Tokatlidis K (2005) A disulfide relay mitochondria. Cell 121: 965&#8211;967.                 16301. <br />  5. Mesecke N, Terziyska N, Kozany C, Baumann F, Neupert W, et al. (2005) A                      29. <span id='am-85' about='obo:NCBITaxon_692029' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_37568 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_81687 obo:NCBITaxon_85604 obo:NCBITaxon_104430 obo:NCBITaxon_33154 obo:NCBITaxon_581385 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7139 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-86' property="rdfs:label" content="Petrova" datatype="xsd:string"></span><span id='am-87'  ></span><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Petrova</span> VY, Drescher D, Kujumdzieva AV, Schmitt MJ (2004) Dual targeting <br />     disulfide relay intermembrane space mitochondria mediates                  yeast catalase A peroxisomes mitochondria. Biochem J 380: 393&#8211;400. <br />     protein import. Cell 121: 1059&#8211;1069.                                                         30. Kellis M, Birren BW, Lander ES (2004) Proof evolutionary analysis of <br />  6. Janssen-Heininger YM, Mossman BT, Heintz NH, Forman HJ, Kalyanaraman                             ancient genome duplication yeast Saccharomyces cerevisiae. Nature 428: 617&#8211; <br />     B, et al. (2008) Redox-based regulation signal transduction: principles, pitfalls,            624. <br />     promises. Free Radic Biol Med 45: 1&#8211;17.                                                  31. Ansanay Galeote V, Alexandre H, Bach B, Delobel P, Dequin S, et al. (2007) <br />  7. Muller EG (1995) A redox-dependent function thioredoxin necessary                       Sfl1p acts activator HSP30 gene Saccharomyces cerevisiae. Curr Genet <br />     sustain rapid rate DNA synthesis yeast. Arch Biochem Biophys 318: 356&#8211;                   52: 55&#8211;63. <br />     361.                                                                                         32. Lohman JR, Remington SJ (2008) Development family redox-sensitive <br />  8. Burhans WC, Heintz NH (2009) The cell cycle redox cycle: linking phase-                     green fluorescent protein indicators use relatively oxidizing subcellular <br />     specific targets cell fate. Free Radic Biol Med 47: 1282&#8211;1293.                                environments. Biochem 47: 8678&#8211;8688. <br />  9. Schafer FQ, Buettner GR (2001) Redox environment cell viewed                       33. Tan SX, Greetham D, Raeth S, Grant CM, Dawes IW, et al. (2010) The <br />     redox state glutathione disulfide/glutathione couple. Free                    thioredoxin-thioredoxin reductase function vivo alternative <br />     Radic Biol Med 30: 1191&#8211;1212.                                                                    reduce oxidized glutathione Saccharomyces cerevisiae. J Biol Chem 285: <br /> 10. Barnham KJ, Masters CL, Bush AI (2004) Neurodegenerative diseases                            6118&#8211;6126. <br />     oxidative stress. Nat Rev Drug Discov 3: 205&#8211;214.                                            34. Marty L, Siala W, Schwarzlander M, Fricker MD, Wirtz M, et al. (2009) The <br /> 11. Kaneto H, Katakami N, Kawamori D, Miyatsuka T, Sakamoto K, et al. (2007)                         NADPH-dependent thioredoxin constitutes functional backup for <br />     Involvement oxidative stress pathogenesis diabetes. Antioxid Redox                  cytosolic glutathione reductase Arabidopsis. Proc Natl Acad Sci USA 106: <br />     Signal 9: 355&#8211;366.                                                                               9109&#8211;9114. <br /> 12. Drakulic T, Temple MD, Guido R, Jarolim S, Breitenbach M, et al. (2005)                      35. Dardalhon M, Kumar C, Iraqui I, Vernis L, Kienda G, et al. (2012) Redox- <br />     Involvement oxidative stress response genes redox homeostasis, level                sensitive YFP sensors monitor dynamic nuclear cytosolic glutathione redox <br />     reactive oxygen species, ageing Saccharomyces cerevisiae. FEMS Yeast Res 5:               changes. Free Radic Biol Med 52: 2254&#8211;2265. <br />     1215&#8211;1228.                                                                                   36. Gasch AP, Spellman PT, Kao CM, Carmel-Harel O, Eisen MB, et al. (2000) <br /> 13. Hansen RE, Roth D, Winther JR (2009) Quantifying global cellular thiol-                      Genomic expression programs response yeast cells environmental <br />     disulfide status. Proc Natl Acad Sci USA 106: 422&#8211;427.                                           changes. Mol Biol Cell 11: 4241&#8211;4257. <br /> 14. Mnaimneh S, Davierwala AP, Haynes J, Moffat J, Peng WT, et al. (2004)                        37. Grant CM, Luikenhuis S, Beckhouse A, Soderbergh M, Dawes IW (2000) <br />     Exploration essential gene functions titratable promoter alleles. Cell 118:               Differential regulation glutaredoxin gene expression response stress <br />     31&#8211;44.                                                                                           conditions yeast Saccharomyces cerevisiae. Biochim Biophys Acta-Gene <br /> 15. Winzeler EA, Shoemaker DD, Astromoff A, Liang H, Anderson K, et al. (1999)                       Structure Expression 1490: 33&#8211;42. <br />     Functional characterization S. cerevisiae genome gene deletion                 38. Fuge EK, Braun EL, Werner-Washburne M (1994) Protein synthesis long- <br />     parallel analysis. Science 285: 901&#8211;906. <br />                                                                                                      term stationary-phase cultures Saccharomyces cerevisiae. J Bacteriol 176: 5802&#8211; <br /> 16. Meyer AJ, Brach T, Marty L, Kreye S, Rouhier N, et al. (2007) Redox-sensitive                    5813. <br />     GFP <span id='am-18' about='obo:NCBITaxon_3702' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_3701 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-19' property="oboInOwl:hasExactSynonym" content="thale cress" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="thale-cress" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Arabidopsis thaliana (thale cress)" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-24'  ></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Arbisopsis thaliana" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="mouse-ear cress" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Arabidopsis thaliana" datatype="xsd:string"></span>Arabidopsis thaliana</span> quantitative biosensor redox potential of <br />                                                                                                  39. Trotter EW, Grant CM (2003) Non-reciprocal regulation redox state of <br />     cellular glutathione redox buffer. Plant J 52: 973&#8211;986. <br />                                                                                                      glutathione, glutaredoxin thioredoxin systems. EMBO reports 4: 184&#8211; <br /> 17. Gutscher M, Pauleau AL, Marty L, Brach T, Wabnitz GH, et al. (2008) Real- <br />                                                                                                      188. <br />     time imaging intracellular glutathione redox potential. Nat Meth 5: 553&#8211; <br />                                                                                                  40. Draculic T, Dawes IW, Grant CM (2000) A single glutaredoxin thioredoxin <br />     559. <br />                                                                                                      gene essential viability yeast Saccharomyces cerevisiae. Mol Micro 36: <br /> 18. Ayer A, Tan SX, Grant CM, Meyer AJ, Dawes IW, et al. (2010) The critical role <br />                                                                                                      1167&#8211;1174. <br />     glutathione maintenance mitochondrial genome. Free Radic Biol <br />                                                                                                  41. Muller E (1996) A glutathione reductase mutant yeast accumulates high levels <br />     Med 49: 1956&#8211;1968. <br />                                                                                                      oxidized glutathione requires thioredoxin growth. Mol Biol Cell 7: <br /> 19. Robinson M, Grigull J, Mohammad N, Hughes T (2002) FunSpec: web-based <br />                                                                                                      1805&#8211;1813. <br />     cluster interpreter yeast. BMC bioinformatics 3: 35. <br />                                                                                                  42. Garrido EO, Grant CM (2002) Role thioredoxins response of <br /> 20. Tsujimoto Y, Izawa S, Inoue Y (2000) Cooperative regulation DOG2, <br />     encoding 2-deoxyglucose-6-phosphate phosphatase, Snf1 kinase                          Saccharomyces cerevisiae oxidative stress induced hydroperoxides. Mol Micro <br />     high-osmolarity glycerol-mitogen-activated protein kinase cascade stress                      43: 993&#8211;1003. <br />     responses Saccharomyces cerevisiae. J Bact 182: 5121&#8211;30418.                               43. Charizanis C, Juhnke H, Krems B, Entian KD (1999) The mitochondrial <br /> 21. Grant C, MacIver FH, Dawes IW (1997) Glutathione synthetase dispensable                       cytochrome c peroxidase Ccp1 Saccharomyces cerevisiae involved conveying <br />     growth normal oxidative stress conditions yeast                        oxidative stress signal transcription factor Pos9 (Skn7). Mol Gen Genet <br />     Sacchromyces cerevisiae accumulation dipeptide gamma-                           262: 437&#8211;447. <br />     glutamylcysteine. Mol Biol Cell 8: 1699&#8211;1707.                                                44. Grant CM, MacIver FH, Dawes IW (1997) Mitochondrial function required <br /> 22. Charizanis C, Juhnke H, Krems B, Entian KD (1999) The mitochondrial                              resistance oxidative stress yeast Saccharomyces cerevisiae. FEBS Lett <br />     cytochrome c peroxidase Ccp1 Saccharomyces cerevisiae involved conveying                410: 219&#8211;222. <br />     oxidative stress signal transcription factor Pos9 (Skn7). Mol Gen Genet            45. Antonenkov VD, Grunau S, Ohlmeier S, Hiltunen JK (2010) Peroxisomes are <br />     MGG 262: 437&#8211;447.                                                                                oxidative organelles. Antioxid Redox Signal 13: 525&#8211;537. <br /> 23. Goebl MG, Yochem J, Jentsch S, McGrath JP, Varshavsky A, et al. (1988) The                   46. Kataya AR, Reumann S (2010) Arabidopsis glutathione reductase 1 dually <br />     yeast cell cycle gene CDC34 encodes ubiquitin-conjugating enzyme. Science                      targeted peroxisomes cytosol. Plant Signal Behav 5: 171&#8211;175. <br />     241: 1331&#8211;1335.                                                                              47. Gietz RD, Schiestl RH (2007) Microtiter plate transformation using LiAc/SS <br /> 24. Perrone GG, Grant CM, Dawes IW (2005) Genetic environmental factors                          carrier DNA/PEG method. Nature Protoc 2: 5&#8211;8. <br />     influencing glutathione homeostasis Saccharomyces cerevisiae. Mol Biol Cell 16:           48. Alberti S, Gitler AD, Lindquist S (2007) A su <br /> </body></html>