<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Bacterial Resistance Antisense Peptide Phosphorodiamidate <br /> Morpholino Oligomers <br /> Susan E. Puckett,a* Kaleb A. Reese,a Georgi M. Mitev,a Valerie Mullen,a Rudd C. Johnson,a Kyle R. Pomraning,b Brett L. Mellbye,c* <br /> Lucas D. Tilley,c* Patrick L. Iversen,c Michael Freitag,b Bruce L. Gellera,c <br /> Department Microbiologya Department Biochemistry Biophysics,b Oregon State University, Corvallis, Oregon, USA, AVI BioPharma, Inc., Corvallis, <br /> Oregon, USAc <br />  <br /> Peptide phosphorodiamidate morpholino oligomers (PPMOs) synthetic DNA mimics bind cRNA inhibit bacterial <br /> gene expression. The PPMO (RFF)3RXB-AcpP (where R arginine, F, phenylalanine, X 6-aminohexanoic acid, B &#65533;?&#65533;-alanine, <br /> and AcpP acyl carrier protein) complementary 11 bases essential gene acpP (which encodes acyl carrier protein). <br /> The MIC (RFF)3RXB-AcpP 2.5 &#65533;?&#65533;M (14 &#65533;?&#65533;g/ml) <span id='am-90' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-91' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-97'  ></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span>Escherichia coli</span> W3110. The rate spontaneous resistance E. coli to <br /> (RFF)3RXB-AcpP 4 &#11547; 10&#11546;7 mutations/cell division. A spontaneous (RFF)3RXB-AcpP-resistant mutant (PR200.1) <span id='am-144' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-145' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-146'  ></span><span id='am-147' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>iso</span>- <br /> lated. The MIC (RFF)3RXB-AcpP 40 &#65533;?&#65533;M (224 &#65533;?&#65533;g/ml) PR200.1. The MICs standard antibiotics PR200.1 and <br /> W3110 identical. The sequence acpP identical PR200.1 W3110. PR200.1 resistant PPMOs <br /> conjugated (RFF)3RXB peptides similar composition pattern cationic nonpolar residues. Genomic se- <br /> quencing PR200.1 identified mutation sbmA, encodes active transport protein. In separate experiments, a <br /> (RFF)3RXB-AcpP-resistant isolate (RR3) selected transposome library, insertion mapped sbmA. Ge- <br /> netic complementation PR200.1 RR3 sbmA restored susceptibility (RFF)3RXB-AcpP. Deletion sbmA caused resis- <br /> tance (RFF)3RXB-AcpP. We conclude resistance (RFF)3RXB-AcpP linked peptide phosphoro- <br /> diamidate morpholino oligomer, dependent composition repeating pattern amino acids, caused mutations in <br /> sbmA. The <span id='am-149' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-150' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-152'  ></span><span id='am-153' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> suggest (RFF)3R-XB PPMOs transported plasma membrane SbmA. <br />  <br />  <br />  <br /> A    ntibiotic resistance bacteria continues prob- <br />      lem. The number antibiotic-resistant pathogens increas- <br /> ing, level resistance standard antibiotics increasing, and <br />                                                                                      cationic amphipathic. Previous investigations sug- <br />                                                                                      gest repeated peptide motif cationic residue fol- <br />                                                                                      lowed hydrophobic residues im- <br /> the percentage isolates resistance multiple antibiotics                   portant feature efficient membrane penetration (39). More <br /> has risen dramatically recent years (3, 37). At time,                recently, compared variety membrane-penetrating <br /> number antibiotics developed decreased sig-                    peptides abilities enhance efficacy peptide <span id='am-166' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-167' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-169'  ></span><span id='am-170' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>phos</span>- <br /> nificantly, particularly targeting Gram-negative bacteria.                     phorodiamidate morpholino oligomers (PPMOs) dif- <br /> Most new antibiotics approved use                   ferences peptides vary pattern alternating <br /> United States past 40 years new classes antibiotics                cationic nonpolar residues amino acid composi- <br /> but simply chemical derivatives antibiotic classes                   tions (27). <br /> that discovered mid-20th century (8). There                            Despite progress improving efficacy potency of <br /> urgent need new antibiotics, particularly novel                    antisense oligomers made, little known about <br /> innovative strategies targeting bacterial pathogens cause                    bacterial resistance compounds. Some naturally occur- <br /> serious diseases (3, 22).                                                            ring antimicrobial peptides, characteristics sim- <br />     Genomics created attractive potential developing                      ilar synthetic peptides used make peptide oli- <br /> innovative strategies address problem antibiotic resis-                  gomers, appear cause resistance bacteria (38). One <br /> tance. Synthetic antisense oligomers, peptide nucleic acids                  report resistance antisense morpholino oligomer a <br /> (14), phosphorothioates (16), phosphorodiamidate <span id='am-136' about='obo:NCBITaxon_64446' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_442183 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_127263 obo:NCBITaxon_33415 obo:NCBITaxon_42282 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-137'  ></span><span id='am-138' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-140' property="rdfs:label" content="Morpho" datatype="xsd:string"></span>morpho</span>-                         mutation region virus genome targeted oligomer <br /> lino oligomers (PMOs) (11, 15), silence expression bacterial <br /> genes. Gene-silencing oligomers decrease expression reporter <br /> genes luciferase, activate endogenous genes &#65533;?&#65533;-ga-                       Received 24 April 2012 Returned modification 21 May 2012 <br />                                                                                         Accepted 25 June 2012 <br /> lactosidase, inhibit growth kill bacteria targeting es- <br />                                                                                         Published ahead print 17 September 2012 <br /> sential genes (10). Antisense oligomers targeted specific, essen- <br />                                                                                         Address correspondence Bruce L. Geller, gellerb@orst.edu. <br /> tial bacterial genes reduce infections increase survival in <br />                                                                                         * Present address: Susan E. Puckett, Department Microbiology and <br /> mouse models infection (12, 15, 40).                                                 Immunology, Weill Cornell Medical College, New York, New York, USA; Brett L. <br />     Antisense oligomers require assistance cross outer mem-                      Mellbye, Department Microbiology, Oregon State University, Corvallis, Oregon, <br /> brane Gram-negative bacteria molecular                              USA; Lucas D. Tilley, Department Microbiology Molecular Genetics, <br /> weight polar characteristics. Short amphipathic peptides                       University Vermont, Burlington, Vermont, USA. <br /> been attached antisense oligomers, greatly im-                          S.E.P. K.A.R. contributed equally article. <br /> proved entry Gram-negative bacteria increased                            Copyright &#169; 2012, American Society Microbiology. All Rights Reserved. <br /> their potency (11, 13, 27).                                                             doi:10.1128/AAC.00850-12 <br />     Membrane-penetrating peptides diverse sequences, but <br />  <br />  <br /> December 2012 Volume 56 Number 12                                 Antimicrobial Agents Chemotherapy         p. 6147&#8211; 6153                           aac.asm.org   6147 <br />  Puckett et al. <br />  <br />  <br />  <br /> (28). Resistance antibiotic important charac-           spread 20 agar plates (60 mm 15 mm) LB plus 20 &#65533;?&#65533;M peptide <br /> teristic determined drug development. The frequency            PMO. The cultures grown overnight 37&#176;C aeration, and <br /> of antibiotic resistance ultimately manifest itself clinic      colonies enumerated. <br /> and play role use particular indication.                  Screening transposome mutants. Transposome EZ-Tn5&#65533;?R6K&#65533;?&#65533;ori/ <br />     In report, characterize spontaneous resistance             Kan-2&#11022;Tnp (Epicentre, Madison, WI) electroporated E. coli <br />                                                                             XL1-Blue MRF=, 3 &#11003; 103 transductants selected LB-kanamy- <br /> PPMO compare cross-resistance antibiotics, PPMOs <br />                                                                             cin (15 &#65533;?&#65533;g/ml) plates. The transductants pooled stored phos- <br /> with different peptides PMO, PPMOs                phate-buffered saline 15% glycerol &#11002;75&#176;C. The pooled transduc- <br /> same peptide targeted different genes. Furthermore,              tants thawed, 1 &#11003; 104 CFU spread LB plate that <br /> same gene causes PPMO resistance identified isolates             included 20 &#65533;?&#65533;M (RFF)3RXB-AcpP. Insertion mutations PPMO- <br /> from independent strategies selection.                               resistant mutants sequenced rescue cloning described the <br />                                                                             manufacturer (Epicentre). Insertions sbmA confirmed using PCR <br /> MATERIALS AND METHODS                                                       described previously (18). Briefly, PCR mixtures contained chromo- <br /> Bacterial strains. Wild-type <span id='am-106' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-110'  ></span><span id='am-111' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span>Escherichia</span> coli K-12 strain W3110 used    somal DNA extracted bacteria using commercial kit (DNeasy; <br /> for selecting spontaneous mutants resistant PPMO            Qiagen, <span id='am-131' about='obo:NCBITaxon_195854' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_8087 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_28738 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_7898 obo:NCBITaxon_195853 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-132' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-134' property="rdfs:label" content="Valencia" datatype="xsd:string"></span><span id='am-135'  ></span>Valencia</span>, CA) primers (IDT Technologies, Coralville, <span id='am-1' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-2'  ></span><span id='am-3' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>IA</span>) <br /> (RFF)3RXB-AcpP (where R arginine, F, phenylalanine, X 6-amino-        flank insertion site: 5=-GATTGCCGTTATCTTCTGGC 5=-G <br /> hexanoic acid, B &#65533;?&#65533;-alanine, AcpP acyl carrier protein). Sponta-   CTCAAGGTATGGGTTACTTCC. Thirty PCR cycles denaturation at <br /> neous mutants resistant (RFF)3RXB-AcpP selected         95&#176;C 0 s, annealing 45&#176;C 0 s, extension 72&#176;C 1 min were <br /> growth Mueller-Hinton II broth supplemented 8&#11003; MIC               carried out. PCRs run 1605 air thermocycler (Idaho Technol- <br /> (RFF)3RXB-AcpP. Liquid cultures grown Mueller-Hinton         ogy, Idaho Falls, ID). <br /> II LB broth. LB agar used growth solid medium. Transfor-          Sequencing acpP. The acpP allele strain analyzed am- <br /> mants pSE380myc-luc (11) grown LB medium supple-               plified PCR (18), using template single bacterial colony picked <br /> mented 50 &#65533;?&#65533;g/ml ampicillin (LBA; Sigma-Aldrich, St. Louis, MO).        growth plate, Promega Taq polymerase (Madison, WI), the <br />     Oligopeptide transport mutants PA0183 (opp), PA0333 (opp dpp),          following primers (Invitrogen, Carlsbad, CA): 5=-AACGTAAAATCGTG <br /> PA0410 (opp tpp), PA0643 (opp dpp tpp), PA0610 (opp dpp tpp),           GTAAGACC-3= 5=-TAACGCCTGGTGGCCGTTGATG-3=. The <br /> which derived parent strain Morse 2034 [trpE9851 leu 277          PCR products gel purified using Qiagen MinElute PCR purifica- <br /> F&#11002;IN(rrnD-rrnE)], described previously (36) gifts        tion kit (Valencia, CA) sequenced using primers shown <br /> from J. W. Payne (University Wales, Bangor, United <span id='am-141' about='obo:NCBITaxon_kingdom' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-142' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-143' property="rdfs:label" content="kingdom" datatype="xsd:string"></span>Kingdom</span>).             core laboratory Center Genome Research, Oregon <br />     In-frame, nonpolar knockout strains E. coli JW3496 (dctA knockout       State University. <br /> mutant), JW5730 (eptA knockout mutant), JW0368 (sbmA knockout               Genomic sequencing. Genomic DNA W3110 wild type and <br /> mutant) isogenic parent strain, BW25113 (2), provided        PR200.1 generated standard procedures (1). DNA sheared by <br /> from Keio collection National BioResource Project (NIG; Ja-      sonication processed Illumina high-throughput sequencing as <br /> <span id='am-21' about='obo:NCBITaxon_9596' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-22' property="oboInOwl:hasAlternativeId" content="NCBITaxon:317028" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="chimpanzees" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Pan" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-26'  ></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasAlternativeId" content="NCBITaxon:445518" datatype="xsd:string"></span>pan</span>). The knockout strains grown LB broth 50 &#65533;?&#65533;g/ml kana-       previously described (31, 32). Data analyses individual point mu- <br /> mycin (Sigma-Aldrich). The IPTG (isopropyl-&#65533;?&#65533;-D-thiogalactopyrano-           tations carried described previously (32). <br /> side)-inducible sbmA expression plasmid (which pSbmA, from <br /> strain b0377) vector control (pNTR-SD) (34) pro-        RESULTS <br /> vided National BioResource Project (NIG; Japan) grown LB      Spontaneous mutants resistant peptide PMOs. Spontaneous <br /> broth 20 &#65533;?&#65533;g/ml ampicillin. <br />                                                                             resistance apparent growth occasionally occurred in <br />     PPMOs. PPMOs synthesized AVI BioPharma, Inc. (Corvallis, <br /> OR), described previously (39). The base sequence PPMOs tar-      cultures included (RFF)3RXB-AcpP (X 6-amino- <br /> geted acpP (AcpP) 5=-CTTCGATAGTG-3=, PPMOs tar-           hexanoic acid B &#65533;?&#65533;-alanine) concentrations the <br /> geted ftsZ (FtsZ) 5=-TCCATTGGTTC-3=, PPMOs            MIC. Growth MIC observed cultures that <br /> targeted luc (Luc) 5=-AACGTTGAGIG. Inosine place guanine      included AcpP PPMOs different peptides attached to <br /> in Luc PPMO necessary make oligomer soluble aqueous       PMO, (RX)6B-AcpP (RXR)4XB-AcpP. The <br /> solutions avoiding guanine quartet structure. The scrambled-base-    rate spontaneous resistance (RFF)3RXB-AcpP measured <br /> sequence (Scr) control 5=-TCTCAGATGGT-3=.                                4 &#11003; 10&#11002;7 mutations/cell generation. <br />     Antibiotics. All antibiotics bleomycin purchased           Susceptibility antibiotics growth rate. Colonies were <br /> Sigma-Aldrich. Bleomycin purchased Enzo Life Science (Farm-        isolated single liquid culture W3110 grown 8&#11003; <br /> ingdale, NY). <br />                                                                             MIC (20 &#65533;?&#65533;M, 112 &#65533;?&#65533;g/ml) (RFF)3RXB-AcpP. One colony <br />     MICs. MICs determined microdilution method (5) in <br /> Mueller-Hinton II broth. For determination MICs strains XL1-Blue     (PR200.1) picked random characterized. <br /> MRF= RR3, Mueller-Hinton II broth supplemented 1%              PR200.1 equally susceptible parent strain W3110 each <br /> tryptone.                                                                   antibiotic tested (MICs, 4 &#65533;?&#65533;g/ml, 1.25 &#65533;?&#65533;g/ml, 1.25 &#65533;?&#65533;g/ml, 0.125 <br />     Luciferase expression. Spontaneous (RFF)3RXB-AcpP-resistant mu-         &#65533;?&#65533;g/ml, 10 &#65533;?&#65533;g/ml ampicillin, tetracycline, kanamycin, <br /> tants chemically competent transformed described pre-      polymyxin B, rifampin, respectively). These results indicate <br /> viously (29) pSE380myc-Luc (11). Overnight cultures grown         particular PPMO-resistant isolate resistant to <br /> aerobically 37&#176;C LBA diluted 2 &#11003; 10&#11002;2 LBA       antibiotics general. <br /> without various concentrations (8, 20, 50 &#65533;?&#65533;M) (RFF)3RXB-Luc           The doubling times strains PR200.1 W3110 iden- <br /> (RFF)3RXB-Scr grown aerobically 37&#176;C 7 h. Samples an-       tical, difference growth rate observed liquid or <br /> alyzed luciferase expression luminometry described previously <br />                                                                             solid medium. <br /> (11). <br />     Rate spontaneous resistance. The rate spontaneous resistance       Sequences acpP alleles. The target PMO, acpP, was <br /> peptide PMO measured method <span id='am-6' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-7'  ></span><span id='am-8' property="rdfs:label" content="Luria" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Luria</span> Delbruck (23)        sequenced PR200.1 W3110, sequences found <br /> described previously (33). An overnight culture diluted 1 &#11003; 104      identical (data shown). <br /> CFU/ml LB medium divided 20 1-ml aliquots. Each aliquot             MICs AcpP PPMOs attached various peptides. MICs for <br /> was grown overnight 37&#176;C aeration, 1 &#65533;?&#65533;l 50 &#65533;?&#65533;l   different AcpP PPMOs measured using strains PR200.1 and <br />  <br />  <br /> 6148    aac.asm.org                                                                                               Antimicrobial Agents Chemotherapy <br />                                                                                                                                            Morpholino Oligomer Resistance <br />  <br />  <br />  <br /> TABLE 1 MIC AcpP PPMOs pure cultures E. coli <br />                                                                                                                                MIC (&#65533;?&#65533;M [&#65533;?&#65533;g/ml]) <br />                                                                                b <br /> Motif, PPMO no.                                            Conjugated peptide                                                  W3110                            PR200.1 <br /> Motif 1 (C-N-N) <br />  NG-05-0200                                                RFFRFFRFFRXB                                                        2.5 (14)                         40 (222) <br />  NG-05-0653                                                DRDFDFDRDFDFDRDFDFDRXB                                              2.5 (14)                         40 (222) <br />  NG-23-248                                                 RXXRXXRXXRXB                                                        20 (102)                         80 (204) <br />  NG-06-0199                                                KFFKFFKFFKXB                                                        10 (54)                          80 (435) <br />  <br /> Motif 2 (C-N), NG-06-0073                                  RXRXRXRXRXRXB                                                       1.25 (7)                         1.25 (7) <br />  <br /> Motif 3 (C-N-C) <br />  NG-06-0076                                                RXRRXRRXRRXRXB                                                      1.25 (7)                         1.25 (7) <br />  NG-07-0795                                                RFRRFRRFRRFRXB                                                      1 (6)                            16 (94) <br />  <br /> No motif, NG-05-0246                                       RTRTRFLRRTXB                                                        20 (111)                         40 (111) <br /> a <br />   Motif 1 cationic-nonpolar-nonpolar (C-N-N). Motif 2 cationic-nonpolar (C-N). Motif 3 cationic-nonpolar-cationic (C-N-C). <br /> b <br />   X 6-aminohexanoic acid, B &#65533;?&#65533;-alanine, O ornithine, D indicates isomeric form residue follows. <br />  <br />  <br />  <br /> W3110 indicators (Table 1). All AcpP PPMOs tested                              Exponential cultures grown 18 h (RFF)3RXB- <br /> the base sequence different peptides attached. The                          FtsZ, targeted ftsZ, scrambled-base-sequence (Scr) <br /> attached peptides differed amino acid composi-                         control. Samples culture plated, viable <br /> tions pattern repeating sequences cationic                     cells counted. PR200.1 grew normal cell density, whereas <br /> nonpolar residues. Repeating patterns amino acids, in-                          viable cell count parent strain W3110 reduced 2 <br /> cluding cationic nonpolar residues, important features                        orders magnitude presence (RFF)3RXB-FtsZ (Fig. 1). <br /> membrane-penetrating peptides (17, 41, 42). Four AcpP                             The scrambled-base-sequence control effect growth <br /> PPMOs, including (RFF)3RXB-AcpP, peptides repeat-                             W3110 PR200.1. <br /> ing amino acid motif cationic-nonpolar-nonpolar (C-N-N),                                  In experiments, exponential cultures W3110 and <br /> and composed D-amino acids instead                            PR200.1 grown 7 h various concentrations of <br /> usual L-amino acids. One AcpP PPMO conjugated (RX)6B,                             (RFF)3RXB-Luc, targeted luciferase reporter gene, or <br /> which repeating motif cationic-nonpolar (C-N). Two                              scrambled-base-sequence control, (RFF)3RXB-Scr. A plasmid <br /> other AcpP PPMOs conjugated peptides repeating                            expresses luciferase transferred PR200.1 prior <br /> motif cationic-nonpolar-cationic (C-N-C): (RXR)4XB                                experiment. Samples culture analyzed by <br /> (RFR)4XB. Another AcpP PMO conjugated RTRTRFLRR                                   luminometry luciferase activity. The results that <br /> TXB, repeat pattern does conform                          (RFF)3RXB-Luc did inhibit luciferase PR200.1 the <br /> the repeat patterns. All PPMOs various pep-                          3 concentrations tested (Fig. 2). In comparison, W3110 showed <br /> tides attached AcpP PMO previously char-                           inhibition luciferase proportional concentration <br /> acterized effective inhibiting growth E. coli                      PPMO added. The scrambled-base-sequence control did not <br /> (27).                                                                                    inhibit luciferase strain. There differences the <br />     The results PR200.1 resistant AcpP                            growth (optical density) of cultures (data shown). <br /> PMO C-N-N peptide motif tested fully suscepti-                              Peptide transport mutants. The results presented indi- <br /> ble (RX)6B-AcpP PMO (RXR)4XB-AcpP PMO (Table <br /> 1). However, PR200.1 resistant (RFR)4XB-AcpP, which <br /> shares C-N-C motif (RXR)4XB-AcpP but, like <br /> (RFF)3RXB-AcpP, contains phenylalanine instead 6-amino- <br /> hexaonoic acid. Compared susceptible parent strain <br /> W3110, resistance (RFF)3RXB-AcpP PR200.1 increased the <br /> MIC 16-fold. PR200.1 resistant D-isomeric form of <br /> (RFF)3RXB. The MIC RTRTRFLRRTXB-AcpP, lacks a <br /> repeating amino acid motif includes phenylalanine, in- <br /> creased 2-fold PR200.1 used <span id='am-11' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-12' property="rdfs:label" content="Indicator" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15'  ></span>indicator</span> com- <br /> pared obtained W3110 used indicator. The <br /> MICs scrambled-base-sequence PPMOs composed each <br /> of peptides used AcpP PPMOs undetectable                           FIG 1 Viable cell count 18-h cultures. Stationary cultures W3110 or <br /> (&#11022;80 &#65533;?&#65533;M) case.                                                                  PR200.1 diluted 5 &#11003; 105 CFU/ml Mueller-Hinton broth divided <br />     (RFF)3RXB-PPMOs targeted various genes. PR200.1                               three. (RFF)3RXB-FtsZ (FtsZ) scrambled-base-sequence (Scr) control <br /> tested susceptibility PPMOs, (RFF)3 pep-                        PPMO (160 &#65533;?&#65533;M) PPMO added. Cultures grown aerobically at <br />                                                                                          37&#176;C 18 h, samples diluted plated determine <br /> tide motif different base sequences. One PPMO comple-                             viable cells. Error bars indicate standard deviations. *, highly significant (P &#65533;? <br /> mentary ftsZ, essential gene involved cell division.                   0.01) difference compared no-PPMO scrambled-base-se- <br /> The PPMO targeted luciferase reporter gene (luc).                          quence (Scr) control-treated culture. <br />  <br />  <br />  <br /> December 2012 Volume 56 Number 12                                                                                                                      aac.asm.org 6149 <br />  Puckett et al. <br />  <br />  <br />  <br />                                                                                  <span id='am-16' about='obo:NCBITaxon_8186' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_1489905 obo:NCBITaxon_123366 obo:NCBITaxon_1489904 obo:NCBITaxon_1489872 obo:NCBITaxon_8184 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-17' property="rdfs:label" content="Lates" datatype="xsd:string"></span><span id='am-18'  ></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>lates</span> indicated transposome inserted exact <br />                                                                                  position sbmA isolate, suggesting two <br />                                                                                  colonies clones. The isolates named RR3. <br />                                                                                      RR3 characterized measuring MICs various stan- <br />                                                                                  dard antibiotics (Table 3). All standard antibiotics tested the <br />                                                                                  MIC using XL1-Blue MRF= RR3 indicator, <br />                                                                                  including peptide antibiotics, colistin polymyxin B. How- <br />                                                                                  ever, RR3 4-fold resistant peptide antibi- <br />                                                                                  otics bleomycin phleomycin. PR200.1 4-fold resis- <br /> FIG 2 Luciferase activity cultures treated PPMOs. Growing cultures       tant bleomycin (MIC &#11005; 5.6 &#65533;?&#65533;M [8 &#65533;?&#65533;g/ml]) phleomycin <br /> were treated 7 h PPMO (no PPMO) 3 concentrations        (MIC &#11005; 5.2 &#65533;?&#65533;M [8 &#65533;?&#65533;g/ml]) W3110 (bleomycin MIC &#11005; 1.4 <br /> PPMO [(RFF)3RXB-Luc] targeted luciferase reporter gene scrambled-      &#65533;?&#65533;M [2 &#65533;?&#65533;g/ml]; phleomycin MIC &#11005; 1.3 &#65533;?&#65533;M [2 &#65533;?&#65533;g/ml]). <br /> base-sequence control [(RFF)3RXB-Scr]. After 7 h, samples culture <br /> were measured luciferase activity luminometry. The experiment             The MICs (RFF)3R-AcpP (RXR)4XB-AcpP were <br /> repeated 3 times, error bars indicate standard deviations. *, highly   measured using RR3 XL1-Blue MRF= indicator strains. <br /> significant (P &#65533;? 0.01) difference compared cultures PR200.1       The results RR3 32-fold resistant to <br /> same concentrations (RFF)3RXB-Luc, cultures W3110        (RFF)3RXB-AcpP 8-fold resistant (RXR)4XB- <br /> concentrations (RFF)3RXB-Scr, culture PPMO. <br />                                                                                  AcpP (Table 3). RR3 resistant PPMO with <br />                                                                                  D-amino acids ([D-(RFF)3R]XB-AcpP). Scrambled-base-se- <br />                                                                                  quence control PPMOs did inhibit growth either <br /> cate resistance PPMOs linked peptide moiety. We                RR3 XL1-Blue MRF=. <br /> hypothesized PPMO resistance caused mutation                      RR3 XL1-Blue MRF= genetically complemented with <br /> in known oligopeptide transporters. To test <span id='am-29' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-30' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32'  ></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>this</span>,               pSbmA used measure MIC (RFF)3RXB-AcpP. <br /> MIC measured using various strains mutations one,                    pSbma fully restored susceptibility RR3 PPMO, when <br /> two, oligopeptide transporters (Table 2). The results               induced IPTG (Table 3). Interestingly, MIC signifi- <br /> show (RFF)3RXB-AcpP MIC PR200.1                     cantly complemented strains used indicators <br /> the parent (nonmutant) strain. The scrambled-base-sequence                       strains pSbmA used indicators. <br /> control (RFF)3RXB-Scr showed detectable MIC (&#11022;160 &#65533;?&#65533;M).                        Complementation pSbmA restored susceptibility to <br />     Genomic sequencing. The genomes PR200.1 parent                    bleomycin phleomycin. <br /> strain, W3110, sequence compared. The results indi- <br /> cated total 3 genes mutations PR200.1 compared               DISCUSSION <br /> the W3110 sequence: dctA, eptA, sbmA. In dctA, 2                  This study aware characterize bacterial <br /> transition mutations bases 3958154 (T &#161; A) 395153 (A &#161;                    resistance antisense antibacterial compound. Initially, <br /> G), <span id='am-34' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> 396, caused missense                  growth occasionally unexpectedly observed rou- <br /> Ile <span id='am-126' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-127' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-128' property="rdfs:label" content="Ala" datatype="xsd:string"></span><span id='am-129'  ></span><span id='am-130' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Ala</span>. In eptA, transversion mutation base                 tine MIC assays (RFF)3XB-AcpP present cultures levels <br /> 4339795 (T &#161; A) affected codon 259 caused missense                    4- 8-fold MIC. Similar growth observed <br /> from Ser Thr. In sbmA, transversion mutation                 MIC assays (RXR)4XB-AcpP (RX)6B-AcpP. We <br /> base 396121 (T &#161; G) changed codon 87 (Ser Ala). No                       speculate greater number X (6-aminohexanoic acid) <br /> deletions insertions detected gene.                               residues lack F PPMOs respon- <br />     Characterization deletion mutants. Mutants in-frame                  sible apparent lack spontaneous resistance these <br /> knockout mutations dctA, eptA, sbmA tested <span id='am-39' about='obo:NCBITaxon_9822' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9821 obo:NCBITaxon_35497 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="rdfs:label" content="Sus" datatype="xsd:string"></span>sus</span>-                  PPMOs conditions used MIC assay. Alterna- <br /> ceptibility (RFF)3RXB-AcpP, susceptibilities com-                    tively, genetic loci involved resistance to <br /> pared parent strain (BW25113). The MIC                        (RFF)3RXB PPMOs (putative) resistance PPMOs <br /> (RFF)3RXB-AcpP (2 &#65533;?&#65533;M, 11 &#65533;?&#65533;g/ml) using                     conjugated peptides, (RXR)4XB (RX)6B. How- <br /> the dctA eptA knockout strain parent strain. The MIC ob- <br /> tained using sbmA knockout strain 32 &#65533;?&#65533;M (179 &#65533;?&#65533;g/ml). <br />     The MIC (RXR)4XB-AcpP measured using sbmA-                        TABLE 2 MIC (RFF)3R-AcpP (NG-05-0200) using oligopeptide <br /> knockout strain indicator 2 &#65533;?&#65533;M (11 &#65533;?&#65533;g/ml),                 transport mutants <br /> the isogenic parent strain.                                                                                      MIC <br />     Complementation pSbmA. PR200.1 genetically                          E. coli strain     Mutation/phenotype                &#65533;?&#65533;M          &#65533;?&#65533;g/ml <br /> complemented IPTG-inducible expression plasmid that <br />                                                                                  Morse 2034         Wild-type oligopeptide            5           28 <br /> encodes sbmA (pSbma) control. The complemented                                           transport <br /> strain grown IPTG used measure MIC                        PA0183             opp knockout, oligopeptide        5           28 <br /> (RFF)3RXB-AcpP. The MICs 1 &#65533;?&#65533;M 32 &#65533;?&#65533;M in-                                              permease deletion <br /> duced, sbmA complemented strain vector control                     PA0333             dpp opp knockout,             5           28 <br /> strain, respectively.                                                                                 PA0183 plus dipeptide <br />     Transposome mutants. E. coli XL1-Blue MRF= mu-                                                permease deletion <br /> tagenized transposome EZ-Tn5, library 1 &#11003; 104                  PA0643             tpp, dpp, opp                 5           28 <br /> mutants spread selection plates included 20 &#65533;?&#65533;M                                            knockout, PA0333 plus <br /> (RFF)3R-AcpP. Two colonies grew selection plate,                                       tripeptide permease <br />                                                                                                       mutant <br /> mutated gene sequenced. The sequences iso- <br />  <br />  <br /> 6150    aac.asm.org                                                                                                 Antimicrobial Agents Chemotherapy <br />                                                                                                                   Morpholino Oligomer Resistance <br />  <br />  <br />  <br /> TABLE 3 MICs standard antibiotics PPMOs using transposome mutant RR3, isogenic parent strain XL1-Blue MRF=, sbmA <br /> complemented strains <br />                                                       MIC (&#65533;?&#65533;M [&#65533;?&#65533;g/ml]) <br />                                                                                                       XL1-Blue MRF= <br /> PPMO no.              Antibiotic PPMO              XL1-Blue MRF=            RR3                    (pSbmA)                    RR3(pSbmA) <br />                       Polymyxin B                     0.8 (1)                  0.8 (1)                0.8 (1)                    0.8 (1) <br />                       Colistin                        0.9 (1)                  0.9 (1)                0.9 (1)                    0.9 (1) <br />                       Erythromycin                    34 (25)                  34 (25)                34 (25)                    34 (25) <br />                       Rifampin                        6 (5)                    6 (5)                  6 (5)                      6 (5) <br />                       Bleomycin                       0.2 (0.25)               0.7 (1)                0.01 (0.016)               0.01 (0.016) <br />                       Phleomycin                      0.3 (0.5)                1.3 (2)                0.02 (0.03)                0.02 (0.03) <br /> NG-05-0200            (RFF)3RXB-AcpP                  2 (11)                   64 (355)               2 (11)                     2 (11) <br /> NG-05-0653            [D-(RFF)3R]XB-AcpP              2 (11)                   64 (355)               2 (11)                     2 (11) <br /> NG-06-0076            (RXR)4XB-AcpP                   1 (6)                    8 (48)                 0.1 (0.6)                  0.1 (0.6) <br /> NG-05-0655            (RFF)3RXB-Scr                   &#11022;128 (&#11022;714)              &#11022;128 (&#11022;714)            &#11022;128 (&#11022;714)                &#11022;128 (&#11022;714) <br /> NG-06-0078            (RXR)4XB-Scr                    &#11022;128 (&#11022;714)              &#11022;128 (&#11022;714)            &#11022;128 (&#11022;714)                &#11022;128 (&#11022;714) <br />  <br />  <br />  <br /> ever, rigorously pursued resistance        However, say target-site mutations do <br /> two PPMOs, certainly possible spontaneous resis-          occur uncharacterized mutants. Neverthe- <br /> tance occur appropriate conditions.                            less, target-site mutations statistically improbable con- <br />     The rate spontaneous resistance (RFF)3RXB-AcpP             sidering 4 wobble bases target region of <br /> similar rate spontaneous mutation individual genes      acpP possibly lead decrease efficacy without <br /> E. coli, typically 10&#11002;6 10&#11002;7 mutations/gene/        changing amino acid sequence targeted protein. <br /> generation (6, 24). This suggests genes which,            Another hypothesis resistance PR200.1 caused <br /> when mutated, rise PPMO-resistant phenotype.             mutation oligopeptide transporter. However, oligopep- <br /> However, rate mutation vary widely, measure-          tide transport mutants just susceptible (RFF)3RXB- <br /> ment rate mutation function variables (25),       AcpP isogenic parent strain. This showed resistance in <br /> including concentration antibiotic used selection         PR200.1 caused mutation known oligopeptide <br /> the number genes loci capable causing resistance pheno-       transporters tested. <br /> type. Ultimately, rate resistance PPMOs vivo              In effort identify mutation PR200.1 is <br /> conditions specific infections meaningful           responsible resistance (RFF)3RXB-AcpP, genome of <br /> measure usefulness clinic.                               PR200.1 sequenced. The results showed missense mutations <br />     The spontaneous mutant PR200.1 susceptible small-         3 genes compared sequence PPMO-suscepti- <br /> molecule antibiotics tested. This shows resistance           ble strain: dctA, eptA, sbmA. In-frame, nonpolar deletion mu- <br /> PPMO caused change physiology result        tations gene showed three, strain with <br /> resistance antibiotics general. Such general changes           sbmA deletion resistant (RFF)3RXB-AcpP. Further- <br /> known occur include reduction net negative charge        more, complementation PR200.1 sbmA restored suscepti- <br /> of lipopolysaccharide Gram-negative bacteria (7, 30),             bility PPMO. These results mutations sbmA <br /> changes capsule polysaccharide (4), changes expression          cause resistance (RFF)3RXB-AcpP. <br /> outer membrane porins (30), alterations outer membrane lipid              The MIC (RFF)3RXB-AcpP slightly lower the <br /> composition result decreased membrane permeability               strain sbmA deletion PR200.1 used the <br /> (30), activation overexpression multidrug efflux pumps         indicator. However, strains originated different parent <br /> (21). We tested variety antibiotics, hydro-       strains, probably accounts difference suscepti- <br /> philic (ampicillin, kanamycin), hydrophobic (rifampin, tetracy-          bility. The parent strain strain sbmA deletion was <br /> cline), amphiphilic (polymyxin B) enter             slightly susceptible PPMO parent strain <br /> Gram-negative bacteria outer membrane porins (am-            PR200.1. <br /> picillin, tetracycline) outer membrane lipid bilayer          The transposome mutant RR3 resistant both <br /> (rifampin, polymyxin B). The results suggested mutation         (RFF)3RXB-AcpP (RXR)4XB-AcpP. This differs re- <br /> in PR200.1 specific (RFF)3RXB-AcpP PPMOs simi-            sult spontaneous mutant PR200.1 strain the <br /> lar peptide moieties. Later, following identification mu-     sbmA deletion, resistant (RFF)3RXB-AcpP not <br /> tation sbmA, PR200.1 mildly resistant (4-fold)        (RXR)4XB-AcpP. This suggests RR3 polar effect the <br /> to peptide antibiotics bleomycin phleomycin.                     gene downstream sbmA (yaiW) responsible re- <br />     We hypothesized resistance caused mutation          sistance (RXR)4XB-AcpP. yaiW predicted DNA-binding <br /> the sequence acpP targeted PPMO. We previously            transcriptional regulator. A mutation sbmA apparently suffi- <br /> shown one-base mismatch near 3=end PPMO tar-             cient cause resistance (RFF)3RXB-AcpP, (RXR)4- <br /> geted acpP <span id='am-76' about='obo:NCBITaxon_292' typeof='owl:Thing obo:NCBITaxon_80840 obo:NCBITaxon_28216 obo:NCBITaxon_87882 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_119060 obo:NCBITaxon_2 obo:NCBITaxon_32008'><span id='am-77'  ></span><span id='am-78' property="oboInOwl:hasDbXref" content="PMID:5963505" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Burkholderia capacia" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Burkholderia cepacia" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas multivorans" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="PMID:1283774" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasRelatedSynonym" content="Burgholderia cepacia" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas kingii" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas cepacia" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasDbXref" content="PMID:7526797" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasExactSynonym" content="Burkholderia cepacia genomovar I" datatype="xsd:string"></span>Burkholderia cepacia</span> complex raised MIC         AcpP. <br /> by factor 8- 32-fold (15). However, present study       PR200.1 susceptible (RXR)4XB-AcpP resistant to <br /> found mutation target region acpP resistant     (RFR)4XB-AcpP, PPMOs share C- <br /> mutant. Therefore, hypothesis case disproven.            N-C repeat motif. This indicates amino acid composition <br />  <br />  <br /> December 2012 Volume 56 Number 12                                                                                            aac.asm.org 6151 <br />  Puckett et al. <br />  <br />  <br />  <br /> of PPMOs important repeating pattern            sbmA deletion somewhat susceptible to <br /> of amino acids determining resistance PR200.1. The similar-        (RFF)3RXB-AcpP, albeit high concentrations. We speculate <br /> ity resistance PR200.1 (RFF)3RXB-AcpP             PPMOs able cross plasma membrane passing <br /> (RFR)4XB-AcpP complete susceptibility (RXR)4XB-AcpP               lipid bilayer manner cross the <br /> may suggest X (6-aminohexanoic acid) accounts dif-          outer membrane. There additional active transporters <br /> ference. This supported result (Table 1) level     specificities nucleic acid oligomers. The possibility <br /> resistance (RXX)3RXB-AcpP 4-fold level re-        suggested ability PMOs (not conjugated peptide) <br /> sistance (RFF)3RXB-AcpP,               inhibit gene expression strains porous outer membranes <br /> repeating pattern cationic nonpolar amino acids           allow passage large oligomers (11, 12). <br /> former contains X. Perhaps unusual 6-carbon backbone                In summary, results suggest bacterial resistance a <br /> of X causes conformational change disallows interaction           PPMO determined peptide PMO. The <br /> with SbmA. Alternatively, sbmA mutants resis-            rate occurrence spontaneous resistance (RFF)3RXB-AcpP <br /> tant peptides F (phenylalanine). There positive trend       similar spontaneous changes bacterial phe- <br /> between number F residues peptide resistance.          notypes. In PR200.1 RR3, resistance caused mutations in <br /> This supported results higher resistance          sbmA. Our results, combination known role SbmA <br /> PPMOs F [such (RFF)3RXB-AcpP (RFR)4XB-                  peptide antibiotic uptake, suggest SbmA acts trans- <br /> AcpP], resistance PPMOs fewer F residues (such           porter (RFF)3RXB-AcpP periplasm cytoplasm. <br /> RTRTRFLRRTXB-AcpP), resistance PPMOs F <br /> [such (RXR)4XB-AcpP (RXR)6XB-AcpP],                      ACKNOWLEDGMENTS <br /> (RXX)3RXB-AcpP exception trend.                            This work supported AVI BioPharma, Inc., Howard <br />     sbmA encodes active transporter bleomycin           Hughes Medical Institute (through undergraduate student research fel- <br /> peptide antibiotics (19, 26, 35, 43). Our results consistent    lowships Susan E. Puckett Valerie Mullen). Preparation Illumina <br /> SbmA acting active transporter (RFF)3RXB-AcpP. sbmA           sequencing libraries data analyses supported start-up funds <br /> homologs widely conserved bacteria (9). The homolog            OSU Computational Genome Biology Initiative Michael <br /> of sbmA <span id='am-44' about='obo:NCBITaxon_379' typeof='owl:Thing obo:NCBITaxon_356 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_82115 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_28211 obo:NCBITaxon_227290'><span id='am-45' property="oboInOwl:hasDbXref" content="PMID:13130068" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasAlternativeId" content="NCBITaxon:78526" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="PMID:8240956" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="PMID:9828428" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Rhizobium" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="PMID:8573502" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="PMID:8494742" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Rhizobacterium" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="PMID:1371066" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="PMID:10758890" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="PMID:13130069" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="PMID:11211278" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="Phytomyxa" datatype="xsd:string"></span><span id='am-60'  ></span>Rhizobium</span> meliloti, bacA, required symbiosis      Freitag. <br /> alfalfa (9). The homolog <span id='am-61' about='obo:NCBITaxon_235' typeof='owl:Thing obo:NCBITaxon_118882 obo:NCBITaxon_356 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_28211 obo:NCBITaxon_234'><span id='am-62' property="rdfs:label" content="Brucella abortus" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Brucella melitensis biovar Abortus" datatype="xsd:string"></span><span id='am-66'  ></span><span id='am-67' property="oboInOwl:hasExactSynonym" content="Brucella melitensis bv. Abortus" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Bacterium abortus" datatype="xsd:string"></span>Brucella abortus</span> virulence factor          We thank Andrew Karplus critical discussion. <br /> important intracellular survival macrophages (20). It            Bruce L. Geller employed AVI BioPharma, Inc., and <br />                                                                          Oregon State University. <br /> been proposed physiological substrates SbmA or- <br /> ganic signaling molecules (43). An assay measure uptake of <br />                                                                          REFERENCES <br /> PPMOs currently available used define fur- <br />                                                                           1. Ausubel FM, et al. 1998. Current protocols molecular biology. John <br /> ther role SbmA resistance PPMO.                                 Wiley &amp; Sons, Inc, New York, NY. <br />     The substrate specificity SbmA investigated           2. Baba T, et al. 2006. Construction <span id='am-154' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-155' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-158' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span>Escherichia coli K-12</span> in-frame, single- <br /> found quite flexible. Initially, specificity proposed       gene knockout mutants: Keio collection. Mol. Syst. Biol. 2:2006.0008. <br /> be associated thiazole oxazole structural motif (43).              doi:10.1038/msb4100050. <br />                                                                           3. Boucher HW, et al. 2009. Bad bugs, drugs: ESKAPE! An update <br /> Later, proline-rich antimicrobial peptides shown trans- <br />                                                                              Infectious Diseases Society America. Clin. Infect. Dis. 48:1&#8211;12. <br /> ported SbmA (26). However, (RFF)3RXB-AcpP                  4. Campos MA, et al. 2004. Capsule polysaccharide mediates bacterial re- <br /> these structural features. If SbmA transporter                     sistance antimicrobial peptides. Infect. Immun. 72:7107&#8211;7114. <br /> (RFF)3RXB-AcpP, specificity SbmA apparently lim-            5. Clinical Laboratory Standards Institute. 2006. Methods dilution <br /> ited thiazole- oxazole-containing compounds proline-             antimicrobial susceptibility tests bacteria grow aerobically, p 10.2&#8211; <br />                                                                              10.3. Approved standard, 7th ed. Clinical Laboratory Standards Insti- <br /> rich peptides. Our results suggest substrate specificity         tute, Wayne, PA. <br /> SbmA flexible accommodate polypeptides               6. Drake J, Charlesworth WB, Charlesworth D, Crow JF. 1998. Rates of <br /> thiazole, oxazole, proline. With peptides used         spontaneous mutation. Genetics 148:1667&#8211;1686. <br /> conjugates, specificity appears linked spacing        7. Ernst RK, Guina T, Miller SI. 2001. <span id='am-113' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-114' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-121'  ></span><span id='am-122' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-124' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span>Salmonella</span> typhimurium outer <br />                                                                              membrane remodeling: role resistance host immunity. Microbes <br /> cationic nonpolar amino acid residues context of <br />                                                                              Infect. 3:1327&#8211;1334. <br /> the peptide. It noteworthy all-D-enantiomer con-         8. Fischback MA, Walsh CT. 2009. Antibiotics emerging pathogens. <br /> jugate (NG-05-0653) MIC values all-L-enan-               Science 325:1089 &#8211;1093. <br /> tiomer conjugate (NG-05-0200) parental resistant strains.         9. Gazebrook J, Ichige A, Walker GC. 1993. A Rhizobium meliloti homolog <br /> This contrast results shown all-D-isomer              Escherichia coli peptide antibiotic-transport protein SbmA essen- <br />                                                                              tial bacteroid development. Genes Dev. 7:1485&#8211;1497. <br /> proline-rich antimicrobial peptide Bac7(1-35), ineffec-        10. Geller BL. 2005. Antisense antibiotics. Curr. Opin. Mol. Ther. 7:109 &#8211;113. <br /> tive compared all-L-isomer form (26). It suggested       11. Geller BL, et al. 2003. Inhibition gene expression Escherichia coli by <br /> th <br /> </body></html>